<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2002 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2002.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2002</h1><p>Number of papers published in 2002: <strong>102</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Induced mitotic recombination: a switch in time.</h4>
<p class="authors"><span class="author">Adams DJ</span> and <span class="author">Bradley A</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">30</span>;<span class="number">1</span>;<span class="pagerange">6-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11753379">11753379</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0102-6">10.1038/ng0102-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The MHC haplotype project: a resource for HLA-linked association studies.</h4>
<p class="authors"><span class="author">Allcock RJ</span>, <span class="author">Atrazhev AM</span>, <span class="author">Beck S</span>, <span class="author">de Jong PJ</span>, <span class="author">Elliott JF</span>, <span class="author">Forbes S</span>, <span class="author">Halls K</span>, <span class="author">Horton R</span>, <span class="author">Osoegawa K</span>, <span class="author">Rogers J</span>, <span class="author">Sawcer S</span>, <span class="author">Todd JA</span>, <span class="author">Trowsdale J</span>, <span class="author">Wang Y</span> and <span class="author">Williams S</span></p>
<p class="affiliation">University of Cambridge, UK.</p>
<p class="publication"><span class="title">Tissue antigens</span>&nbsp;<span class="year">2002</span>;<span class="volume">59</span>;<span class="number">6</span>;<span class="pagerange">520-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12445322" rel="external">12445322</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From genomes to vaccines: Leishmania as a model.</h4>
<p class="authors"><span class="author">Almeida R</span>, <span class="author">Norrish A</span>, <span class="author">Levick M</span>, <span class="author">Vetrie D</span>, <span class="author">Freeman T</span>, <span class="author">Vilo J</span>, <span class="author">Ivens A</span>, <span class="author">Lange U</span>, <span class="author">Stober C</span>, <span class="author">McCann S</span> and <span class="author">Blackwell JM</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, University of Cambridge School of Clinical Medicine, Wellcome Trust/MRC Building, Addenbrooke&#39;s Hospital, Hills Road, Cambridge CB2 2XY.</p><p class="abstract">The 35 Mb genome of Leishmania should be sequenced by late 2002. It contains approximately 8500 genes that will probably translate into more than 10 000 proteins. In the laboratory we have been piloting strategies to try to harness the power of the genome-proteome for rapid screening of new vaccine candidate. To this end, microarray analysis of 1094 unique genes identified using an EST analysis of 2091 cDNA clones from spliced leader libraries prepared from different developmental stages of Leishmania has been employed. The plan was to identify amastigote-expressed genes that could be used in high-throughput DNA-vaccine screens to identify potential new vaccine candidates. Despite the lack of transcriptional regulation that polycistronic transcription in Leishmania dictates, the data provide evidence for a high level of post-transcriptional regulation of RNA abundance during the developmental cycle of promastigotes in culture and in lesion-derived amastigotes of Leishmania major. This has provided 147 candidates from the 1094 unique genes that are specifically upregulated in amastigotes and are being used in vaccine studies. Using DNA vaccination, it was demonstrated that pooling strategies can work to identify protective vaccines, but it was found that some potentially protective antigens are masked by other disease-exacerbatory antigens in the pool. A total of 100 new vaccine candidates are currently being tested separately and in pools to extend this analysis, and to facilitate retrospective bioinformatic analysis to develop predictive algorithms for sequences that constitute potentially protective antigens. We are also working with other members of the Leishmania Genome Network to determine whether RNA expression determined by microarray analyses parallels expression at the protein level. We believe we are making good progress in developing strategies that will allow rapid translation of the sequence of Leishmania into potential interventions for disease control in humans.</p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2002</span>;<span class="volume">357</span>;<span class="number">1417</span>;<span class="pagerange">5-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11839176" rel="external">11839176</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1692919" rel="external">1692919</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2001.0985" rel="external">10.1098/rstb.2001.0985</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary analyses of ABC transporters of Dictyostelium discoideum.</h4>
<p class="authors"><span class="author">Anjard C</span>, <span class="author">Loomis WF</span> and <span class="author">Dictyostelium Sequencing Consortium</span></p>
<p class="affiliation">Center for Molecular Genetics, Division of Biology, University of California--San Diego, La Jolla, California 92093-0368, USA.</p><p class="abstract">The ABC superfamily of genes is one of the largest in the genomes of both bacteria and eukaryotes. The proteins encoded by these genes all carry a characteristic 200- to 250-amino-acid ATP-binding cassette that gives them their family name. In bacteria they are mostly involved in nutrient import, while in eukaryotes many are involved in export. Seven different families have been defined in eukaryotes based on sequence homology, domain topology, and function. While only 6 ABC genes in Dictyostelium discoideum have been studied in detail previously, sequences from the well-advanced Dictyostelium genome project have allowed us to recognize 68 members of this superfamily. They have been classified and compared to animal, plant, and fungal orthologs in order to gain some insight into the evolution of this superfamily. It appears that many of the genes inferred to have been present in the ancestor of the crown organisms duplicated extensively in some but not all phyla, while others were lost in one lineage or the other.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM60447</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2002</span>;<span class="volume">1</span>;<span class="number">4</span>;<span class="pagerange">643-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12456012" rel="external">12456012</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC117992" rel="external">117992</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comprehensive model for familial breast cancer incorporating BRCA1, BRCA2 and other genes.</h4>
<p class="authors"><span class="author">Antoniou AC</span>, <span class="author">Pharoah PD</span>, <span class="author">McMullan G</span>, <span class="author">Day NE</span>, <span class="author">Stratton MR</span>, <span class="author">Peto J</span>, <span class="author">Ponder BJ</span> and <span class="author">Easton DF</span></p>
<p class="affiliation">CRC Genetic Epidemiology Unit, Institute of Public Health, Strangeways Research Laboratory, Worts Causeway, University of Cambridge, Cambridge CB1 8RN, UK.</p><p class="abstract">In computing the probability that a woman is a BRCA1 or BRCA2 carrier for genetic counselling purposes, it is important to allow for the fact that other breast cancer susceptibility genes may exist. We used data from both a population based series of breast cancer cases and high risk families in the UK, with information on BRCA1 and BRCA2 mutation status, to investigate the genetic models that can best explain familial breast cancer outside BRCA1 and BRCA2 families. We also evaluated the evidence for risk modifiers in BRCA1 and BRCA2 carriers. We estimated the simultaneous effects of BRCA1, BRCA2, a third hypothetical gene &#39;BRCA3&#39;, and a polygenic effect using segregation analysis. The hypergeometric polygenic model was used to approximate polygenic inheritance and the effect of risk modifiers. BRCA1 and BRCA2 could not explain all the observed familial clustering. The best fitting model for the residual familial breast cancer was the polygenic, although a model with a single recessive allele produced a similar fit. There was also significant evidence for a modifying effect of other genes on the risks of breast cancer in BRCA1 and BRCA2 mutation carriers. Under this model, the frequency of BRCA1 was estimated to be 0.051% (95% CI: 0.021-0.125%) and of BRCA2 0.068% (95% CI: 0.033-0.141%). The breast cancer risk by age 70 years, based on the average incidence over all modifiers was estimated to be 35.3% for BRCA1 and 50.3% for BRCA2. The corresponding ovarian cancer risks were 25.9% for BRCA1 and 9.1% for BRCA2. The findings suggest that several common, low penetrance genes with multiplicative effects on risk may account for the residual non-BRCA1/2 familial aggregation of breast cancer. The modifying effect may explain the previously reported differences between population based estimates for BRCA1/2 penetrance and estimates based on high-risk families.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">1R01 CA81203-01A1</span></p>
<p class="publication"><span class="title">British journal of cancer</span>&nbsp;<span class="year">2002</span>;<span class="volume">86</span>;<span class="number">1</span>;<span class="pagerange">76-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11857015" rel="external">11857015</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2746531" rel="external">2746531</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.bjc.6600008" rel="external">10.1038/sj.bjc.6600008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical and genetic characterization reveals a pseudogene, an evolutionary junction, and unstable loci in distal Xq28.</h4>
<p class="authors"><span class="author">Aradhya S</span>, <span class="author">Woffendin H</span>, <span class="author">Bonnen P</span>, <span class="author">Heiss NS</span>, <span class="author">Yamagata T</span>, <span class="author">Esposito T</span>, <span class="author">Bardaro T</span>, <span class="author">Poustka A</span>, <span class="author">D&#39;Urso M</span>, <span class="author">Kenwrick S</span> and <span class="author">Nelson DL</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza 902E, Houston, TX 77030, USA.</p><p class="abstract">A large portion of human Xq28 has been completely characterized but the interval between G6PD and Xqter has remained poorly understood. Because of a lack of stable, high-density clone coverage in this region, we constructed a 1.6-Mb bacterial and P1 artificial chromosome (BAC and PAC, respectively) contig to expedite mapping, structural and evolutionary analysis, and sequencing. The contig helped to reposition previously mismapped genes and to characterize the XAP135 pseudogene near the int22h-2 repeat. BAC clones containing the distal int22h repeats also demonstrated spontaneous rearrangements and sparse coverage, which suggested that they were unstable. Because the int22h repeats are involved in genetic diseases, we examined them in great apes to see if they have always been unstable. Differences in copy number among the apes, due to duplications and deletions, indicated that they have been unstable throughout their evolution. Taking another approach toward understanding the genomic nature of distal Xq28, we examined the homologous mouse region and found an evolutionary junction near the distal int22h loci that separated the human distal Xq28 region into two segments on the mouse X chromosome. Finally, haplotype analysis showed that a segment within Xq28 has resisted excessive interchromosomal exchange through great ape evolution, potentially accounting for the linkage disequilibrium recently reported in this region. Collectively, these data highlight some interesting features of the genomic sequence in Xq28 and will be useful for positional cloning efforts, mouse mutagenesis studies, and further evolutionary analyses.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">2 P30 HD24064</span>, <span class="grant">5 R01 HD35617</span></p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2002</span>;<span class="volume">79</span>;<span class="number">1</span>;<span class="pagerange">31-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11827455" rel="external">11827455</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2001.6680" rel="external">10.1006/geno.2001.6680</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The mei3 region of the Schizosaccharomyces pombe genome.</h4>
<p class="authors"><span class="author">Aves SJ</span>, <span class="author">Hunt C</span>, <span class="author">Xiang Z</span>, <span class="author">Lyne MH</span>, <span class="author">Wood V</span>, <span class="author">Rajandream MA</span>, <span class="author">Skelton J</span>, <span class="author">Churcher CM</span>, <span class="author">Warren T</span>, <span class="author">Harris D</span>, <span class="author">Gwilliam R</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">School of Biological Sciences, University of Exeter, Washington Singer Laboratories, Perry Road, Exeter EX4 4QG, UK. <a href="mailto:%53.%4A.%41%76%65%73%40%65%78%65%74%65%72.%61%63.%75%6B">&#83;&#46;&#74;&#46;&#65;&#118;&#101;&#115;&#64;&#101;&#120;&#101;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Expression of the mei3 gene is sufficient to induce meiosis in the fission yeast Schizosaccharomyces pombe. The mei3 gene is located 0.64 Mb from the telomere of the left arm of Sz. pombe chromosome II. We have sequenced and analysed 107 kb of DNA from the mei3 genomic region. The sequence includes 14 known genes (bag1-B, csh3, dps1, gpt1, mei3, mfm3, pac1, prp31, rpl38-1, rpn3, rti1, spa1, spm1 and ubc4) and 26 other open reading frames (ORFs) longer than 100 codons: a density of one protein-coding gene per 2.7 kb. Twenty-one of the 40 ORFs (53%) have introns. In addition there is one lone Tf1 transposon long terminal repeat (LTR), tRNA(Trp) and tRNA(Ser) genes and a 5S rRNA gene. 14 of the novel ORFs show sequence similarities which suggest functions of their products, including a coatomer alpha-subunit, a catechol O-methyltransferase, protein kinase, asparagine synthetase, zinc metalloprotease, acetyltransferase, phosphatidylinositol 4-kinase, inositol polyphosphate phosphatase, GTPase-activating protein, permease, pre-mRNA splicing factor, 20S proteasome component and a thioredoxin-like protein. One predicted protein has similarity to the human Cockayne syndrome protein CSA and one with human GTPase XPA binding protein XAB1. Three ORFs are likely to code for proteins because they have sequence similarity with hypothetical proteins, three encode predicted coiled-coil proteins and four are sequence orphans.</p>
<p class="publication"><span class="title">Yeast (Chichester, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">19</span>;<span class="number">6</span>;<span class="pagerange">521-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11921100" rel="external">11921100</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fyea.851" rel="external">10.1002/yea.851</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The SGS3 protein involved in PTGS finds a family.</h4>
<p class="authors"><span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Post transcriptional gene silencing (PTGS) is a recently discovered phenomenon that is an area of intense research interest. Components of the PTGS machinery are being discovered by genetic and bioinformatics approaches, but the picture is not yet complete.</p><p class="abstract"><strong>Results</strong>: The gene for the PTGS impaired Arabidopsis mutant sgs3 was recently cloned and was not found to have similarity to any other known protein. By a detailed analysis of the sequence of SGS3 we have defined three new protein domains: the XH domain, the XS domain and the zf-XS domain, that are shared with a large family of uncharacterised plant proteins. This work implicates these plant proteins in PTGS.</p><p class="abstract"><strong>Conclusion</strong>: The enigmatic SGS3 protein has been found to contain two predicted domains in common with a family of plant proteins. The other members of this family have been predicted to be transcription factors, however this function seems unlikely based on this analysis. A bioinformatics approach has implicated a new family of plant proteins related to SGS3 as potential candidates for PTGS related functions.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="pagerange">21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12162795" rel="external">12162795</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC119857" rel="external">119857</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HMM-based databases in InterPro.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Haft DH</span></p>
<p class="affiliation">Pfam Group, The Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">Protein family databases are an important resource for protein annotation and understanding protein evolution and function. In recent years hidden Markov models (HMMs) have become one of the key technologies used for detection of members of these families. This paper reviews the Pfam, TIGRFAMs and SMART databases that use the profile-HMMs provided by the HMMER package.</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">236-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12230032" rel="external">12230032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Pfam protein families database.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Birney E</span>, <span class="author">Cerruti L</span>, <span class="author">Durbin R</span>, <span class="author">Etwiller L</span>, <span class="author">Eddy SR</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Howe KL</span>, <span class="author">Marshall M</span> and <span class="author">Sonnhammer EL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Pfam is a large collection of protein multiple sequence alignments and profile hidden Markov models. Pfam is available on the World Wide Web in the UK at http://www.sanger.ac.uk/Software/Pfam/, in Sweden at http://www.cgb.ki.se/Pfam/, in France at http://pfam.jouy.inra.fr/ and in the US at http://pfam.wustl.edu/. The latest version (6.6) of Pfam contains 3071 families, which match 69% of proteins in SWISS-PROT 39 and TrEMBL 14. Structural data, where available, have been utilised to ensure that Pfam families correspond with structural domains, and to improve domain-based annotation. Predictions of non-domain regions are now also included. In addition to secondary structure, Pfam multiple sequence alignments now contain active site residue mark-up. New search tools, including taxonomy search and domain query, greatly add to the functionality and usability of the Pfam resource.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2002</span>;<span class="volume">30</span>;<span class="number">1</span>;<span class="pagerange">276-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11752314" rel="external">11752314</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC99071" rel="external">99071</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Armed to the teeth.</h4>
<p class="authors"><span class="author">Bentley S</span>, <span class="author">Holden M</span>, <span class="author">Thomson N</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">4</span>;<span class="pagerange">163-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11912016" rel="external">11912016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence of the model actinomycete Streptomyces coelicolor A3(2).</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Chater KF</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Challis GL</span>, <span class="author">Thomson NR</span>, <span class="author">James KD</span>, <span class="author">Harris DE</span>, <span class="author">Quail MA</span>, <span class="author">Kieser H</span>, <span class="author">Harper D</span>, <span class="author">Bateman A</span>, <span class="author">Brown S</span>, <span class="author">Chandra G</span>, <span class="author">Chen CW</span>, <span class="author">Collins M</span>, <span class="author">Cronin A</span>, <span class="author">Fraser A</span>, <span class="author">Goble A</span>, <span class="author">Hidalgo J</span>, <span class="author">Hornsby T</span>, <span class="author">Howarth S</span>, <span class="author">Huang CH</span>, <span class="author">Kieser T</span>, <span class="author">Larke L</span>, <span class="author">Murphy L</span>, <span class="author">Oliver K</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rajandream MA</span>, <span class="author">Rutherford K</span>, <span class="author">Rutter S</span>, <span class="author">Seeger K</span>, <span class="author">Saunders D</span>, <span class="author">Sharp S</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Taylor K</span>, <span class="author">Warren T</span>, <span class="author">Wietzorrek A</span>, <span class="author">Woodward J</span>, <span class="author">Barrell BG</span>, <span class="author">Parkhill J</span> and <span class="author">Hopwood DA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%64%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#100;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Streptomyces coelicolor is a representative of the group of soil-dwelling, filamentous bacteria responsible for producing most natural antibiotics used in human and veterinary medicine. Here we report the 8,667,507 base pair linear chromosome of this organism, containing the largest number of genes so far discovered in a bacterium. The 7,825 predicted genes include more than 20 clusters coding for known or predicted secondary metabolites. The genome contains an unprecedented proportion of regulatory genes, predominantly those likely to be involved in responses to external stimuli and stresses, and many duplicated gene sets that may represent &#39;tissue-specific&#39; isoforms operating in different phases of colonial development, a unique situation for a bacterium. An ancient synteny was revealed between the central &#39;core&#39; of the chromosome and the whole chromosome of pathogens Mycobacterium tuberculosis and Corynebacterium diphtheriae. The genome sequence will greatly increase our understanding of microbial life in the soil as well as aiding the generation of new drug candidates by genetic engineering.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">417</span>;<span class="number">6885</span>;<span class="pagerange">141-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12000953">12000953</a>; DOI: <a href="http://dx.doi.org/10.1038%2F417141a">10.1038/417141a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The architecture of variant surface glycoprotein gene expression sites in Trypanosoma brucei.</h4>
<p class="authors"><span class="author">Berriman M</span>, <span class="author">Hall N</span>, <span class="author">Sheader K</span>, <span class="author">Bringaud F</span>, <span class="author">Tiwari B</span>, <span class="author">Isobe T</span>, <span class="author">Bowman S</span>, <span class="author">Corton C</span>, <span class="author">Clark L</span>, <span class="author">Cross GA</span>, <span class="author">Hoek M</span>, <span class="author">Zanders T</span>, <span class="author">Berberof M</span>, <span class="author">Borst P</span> and <span class="author">Rudenko G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Trypanosoma brucei evades the immune system by switching between Variant Surface Glycoprotein (VSG) genes. The active VSG gene is transcribed in one of approximately 20 telomeric expression sites (ESs). It has been postulated that ES polymorphism plays a role in host adaptation. To gain more insight into ES architecture, we have determined the complete sequence of Bacterial Artificial Chromosomes (BACs) containing DNA from three ESs and their flanking regions. There was variation in the order and number of ES-associated genes (ESAGs). ESAGs 6 and 7, encoding transferrin receptor subunits, are the only ESAGs with functional copies in every ES that has been sequenced until now. A BAC clone containing the VO2 ES sequences comprised approximately half of a 330 kb &#39;intermediate&#39; chromosome. The extensive similarity between this intermediate chromosome and the left telomere of T. brucei 927 chromosome I, suggests that this previously uncharacterised intermediate size class of chromosomes could have arisen from breakage of megabase chromosomes. Unexpected conservation of sequences, including pseudogenes, indicates that the multiple ESs could have arisen through a relatively recent amplification of a single ES.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI21729</span>; Wellcome Trust: <span class="grant">095161</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2002</span>;<span class="volume">122</span>;<span class="number">2</span>;<span class="pagerange">131-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12106867">12106867</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete sequence and organization of pBtoxis, the toxin-coding plasmid of Bacillus thuringiensis subsp. israelensis.</h4>
<p class="authors"><span class="author">Berry C</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Ben-Dov E</span>, <span class="author">Jones AF</span>, <span class="author">Murphy L</span>, <span class="author">Quail MA</span>, <span class="author">Holden MT</span>, <span class="author">Harris D</span>, <span class="author">Zaritsky A</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Cardiff School of Biosciences, Cardiff University, Cardiff, United Kingdom. <a href="mailto:%42%65%72%72%79%40%63%66.%61%63.%75%6B">&#66;&#101;&#114;&#114;&#121;&#64;&#99;&#102;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The entire 127,923-bp sequence of the toxin-encoding plasmid pBtoxis from Bacillus thuringiensis subsp. israelensis is presented and analyzed. In addition to the four known Cry and two known Cyt toxins, a third Cyt-type sequence was found with an additional C-terminal domain previously unseen in such proteins. Many plasmid-encoded genes could be involved in several functions other than toxin production. The most striking of these are several genes potentially affecting host sporulation and germination and a set of genes for the production and export of a peptide antibiotic.</p>
<p class="publication"><span class="title">Applied and environmental microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">68</span>;<span class="number">10</span>;<span class="pagerange">5082-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12324359" rel="external">12324359</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC126441" rel="external">126441</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Databases and tools for browsing genomes.</h4>
<p class="authors"><span class="author">Birney E</span>, <span class="author">Clamp M</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom. <a href="mailto:%62%69%72%6E%65%79%40%65%62%69.%61%63.%75%6B">&#98;&#105;&#114;&#110;&#101;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">To maximize the value of genome sequences they need to be integrated with other types of biological data and with each other. The entire collection of data then needs to be made available in a way that is easy to view and mine for complex relationships. The recently determined vertebrate genome sequences of human and mouse are so large that building the infrastructure to manage these datasets is a major challenge. This article reviews the database systems and tools for analysis that have so far been developed to address this.</p>
<p class="publication"><span class="title">Annual review of genomics and human genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="pagerange">293-310</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12194990">12194990</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev.genom.3.030502.101529">10.1146/annurev.genom.3.030502.101529</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A global analysis of Caenorhabditis elegans operons.</h4>
<p class="authors"><span class="author">Blumenthal T</span>, <span class="author">Evans D</span>, <span class="author">Link CD</span>, <span class="author">Guffanti A</span>, <span class="author">Lawson D</span>, <span class="author">Thierry-Mieg J</span>, <span class="author">Thierry-Mieg D</span>, <span class="author">Chiu WL</span>, <span class="author">Duke K</span>, <span class="author">Kiraly M</span> and <span class="author">Kim SK</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Box B121, 4200 E. 9th Avenue, Denver, Colorado 80262, USA. <a href="mailto:%74%6F%6D.%62%6C%75%6D%65%6E%74%68%61%6C%40%75%63%68%73%63.%65%64%75">&#116;&#111;&#109;&#46;&#98;&#108;&#117;&#109;&#101;&#110;&#116;&#104;&#97;&#108;&#64;&#117;&#99;&#104;&#115;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The nematode worm Caenorhabditis elegans and its relatives are unique among animals in having operons. Operons are regulated multigene transcription units, in which polycistronic pre-messenger RNA (pre-mRNA coding for multiple peptides) is processed to monocistronic mRNAs. This occurs by 3&#39; end formation and trans-splicing using the specialized SL2 small nuclear ribonucleoprotein particle for downstream mRNAs. Previously, the correlation between downstream location in an operon and SL2 trans-splicing has been strong, but anecdotal. Although only 28 operons have been reported, the complete sequence of the C. elegans genome reveals numerous gene clusters. To determine how many of these clusters represent operons, we probed full-genome microarrays for SL2-containing mRNAs. We found significant enrichment for about 1,200 genes, including most of a group of several hundred genes represented by complementary DNAs that contain SL2 sequence. Analysis of their genomic arrangements indicates that &gt;90% are downstream genes, falling in 790 distinct operons. Our evidence indicates that the genome contains at least 1,000 operons, 2 8 genes long, that contain about 15% of all C. elegans genes. Numerous examples of co-transcription of genes encoding functionally related proteins are evident. Inspection of the operon list should reveal previously unknown functional relationships.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">417</span>;<span class="number">6891</span>;<span class="pagerange">851-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12075352">12075352</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature00831">10.1038/nature00831</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mining the mouse genome.</h4>
<p class="authors"><span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">420</span>;<span class="number">6915</span>;<span class="pagerange">512-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12466846">12466846</a>; DOI: <a href="http://dx.doi.org/10.1038%2F420512a">10.1038/420512a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new, expressed multigene family containing a hot spot for insertion of retroelements is associated with polymorphic subtelomeric regions of Trypanosoma brucei.</h4>
<p class="authors"><span class="author">Bringaud F</span>, <span class="author">Biteau N</span>, <span class="author">Melville SE</span>, <span class="author">Hez S</span>, <span class="author">El-Sayed NM</span>, <span class="author">Leech V</span>, <span class="author">Berriman M</span>, <span class="author">Hall N</span>, <span class="author">Donelson JE</span> and <span class="author">Baltz T</span></p>
<p class="affiliation">Laboratoire de Parasitologie Mol&eacute;culaire, Universit&eacute; Victor Segalen Bordeaux II, UMR-5016 CNRS, 33076 Bordeaux, France. <a href="mailto:%62%72%69%6E%67%61%75%64%40%75-%62%6F%72%64%65%61%75%78%32.%66%72">&#98;&#114;&#105;&#110;&#103;&#97;&#117;&#100;&#64;&#117;&#45;&#98;&#111;&#114;&#100;&#101;&#97;&#117;&#120;&#50;&#46;&#102;&#114;</a></p><p class="abstract">We describe a novel gene family that forms clusters in subtelomeric regions of Trypanosoma brucei chromosomes and partially accounts for the observed clustering of retrotransposons. The ingi and ribosomal inserted mobile element (RIME) non-LTR retrotransposons share 250 bp at both extremities and are the most abundant putatively mobile elements, with about 500 copies per haploid genome. From cDNA clones and subsequently in the T. brucei genomic DNA databases, we identified 52 homologous gene and pseudogene sequences, 16 of which contain a RIME and/or ingi retrotransposon inserted at exactly the same relative position. Here these genes are called the RHS family, for retrotransposon hot spot. Comparison of the protein sequences encoded by RHS genes (21 copies) and pseudogenes (24 copies) revealed a conserved central region containing an ATP/GTP-binding motif and the RIME/ingi insertion site. The RHS proteins share between 13 and 96% identity, and six subfamilies, RHS1 to RHS6, can be defined on the basis of their divergent C-terminal domains. Immunofluorescence and Western blot analyses using RHS subfamily-specific immune sera show that RHS proteins are constitutively expressed and occur mainly in the nucleus. Analysis of Genome Survey Sequence databases indicated that the Trypanosoma brucei diploid genome contains about 280 RHS (pseudo)genes. Among the 52 identified RHS (pseudo)genes, 48 copies are in three RHS clusters located in subtelomeric regions of chromosomes Ia and II and adjacent to the active bloodstream form expression site in T. brucei strain TREU927/4 GUTat10.1. RHS genes comprise the remaining sequence of the size-polymorphic &quot;repetitive region&quot; described for T. brucei chromosome I, and a homologous gene family is present in the Trypanosoma cruzi genome.</p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2002</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">137-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12455980" rel="external">12455980</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC118050" rel="external">118050</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BRAF and RAS mutations in human lung cancer and melanoma.</h4>
<p class="authors"><span class="author">Brose MS</span>, <span class="author">Volpe P</span>, <span class="author">Feldman M</span>, <span class="author">Kumar M</span>, <span class="author">Rishi I</span>, <span class="author">Gerrero R</span>, <span class="author">Einhorn E</span>, <span class="author">Herlyn M</span>, <span class="author">Minna J</span>, <span class="author">Nicholson A</span>, <span class="author">Roth JA</span>, <span class="author">Albelda SM</span>, <span class="author">Davies H</span>, <span class="author">Cox C</span>, <span class="author">Brignell G</span>, <span class="author">Stephens P</span>, <span class="author">Futreal PA</span>, <span class="author">Wooster R</span>, <span class="author">Stratton MR</span> and <span class="author">Weber BL</span></p>
<p class="affiliation">Department of Medicine, Abramson Family Cancer Research Institute, University of Pennsylvania Cancer Center, Philadelphia 19104, USA.</p><p class="abstract">BRAF encodes a RAS-regulated kinase that mediates cell growth and malignant transformation kinase pathway activation. Recently, we have identified activating BRAF mutations in 66% of melanomas and a smaller percentage of many other human cancers. To determine whether BRAF mutations account for the MAP kinase pathway activation common in non-small cell lung carcinomas (NSCLCs) and to extend the initial findings in melanoma, we screened DNA from 179 NSCLCs and 35 melanomas for BRAF mutations (exons 11 and 15). We identified BRAF mutations in 5 NSCLCs (3%; one V599 and four non-V599) and 22 melanomas (63%; 21 V599 and 1 non-V599). Three BRAF mutations identified in this study are novel, altering residues important in AKT-mediated BRAF phosphorylation and suggesting that disruption of AKT-induced BRAF inhibition can play a role in malignant transformation. To our knowledge, this is the first report of mutations documenting this interaction in human cancers. Although &gt;90% of BRAF mutations in melanoma involve codon 599 (57 of 60), 8 of 9 BRAF mutations reported to date in NSCLC are non-V599 (89%; P &lt; 10(-7)), strongly suggesting that BRAF mutations in NSCLC are qualitatively different from those in melanoma; thus, there may be therapeutic differences between lung cancer and melanoma in response to RAF inhibitors. Although uncommon, BRAF mutations in human lung cancers may identify a subset of tumors sensitive to targeted therapy.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P50 CA70907</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2002</span>;<span class="volume">62</span>;<span class="number">23</span>;<span class="pagerange">6997-7000</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12460918" rel="external">12460918</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation analysis of EP300 in colon, breast and ovarian carcinomas.</h4>
<p class="authors"><span class="author">Bryan EJ</span>, <span class="author">Jokubaitis VJ</span>, <span class="author">Chamberlain NL</span>, <span class="author">Baxter SW</span>, <span class="author">Dawson E</span>, <span class="author">Choong DY</span> and <span class="author">Campbell IG</span></p>
<p class="affiliation">VBCRC Cancer Genetics Laboratory, Peter MacCallum Cancer Institute, Locked Bag No. 1 A&#39;Beckett Street, East Melbourne, Victoria, Australia.</p><p class="abstract">The putative tumour suppressor gene EP300 is located on chromosome 22q13 which is a region showing frequent loss of heterozygosity (LOH) in colon, breast and ovarian cancers. We analysed 203 human breast, colon and ovarian primary tumours and cell lines for somatic mutations in EP300. LOH across the EP300 locus was detected in 38% of colon, 36% of breast, and 49% of ovarian primary tumours but no somatic mutations in EP300 were identified in any primary tumour. Analysis of 17 colon, 11 breast, and 11 ovarian cancer cell lines identified truncating mutations in 4 colon cancer cell lines (HCT116, HT29, LIM2405 and LIM2412). We confirmed the presence of a previously reported frameshift mutation in HCT116 at codon 1699 and identified a second frameshift mutation at codon 1468. Bi-allelic inactivation of EP300 was also detected in LIM2405 that harbours an insC mutation at codon 927 as well an insA mutation at codon 1468. An insA mutation at codon 1468 was identified in HT29 and a CGA&gt;TGA mutation at codon 86 was identified in LIM2412. Both these lines were heterozygous across the EP300 locus and western blot analysis confirmed the presence of an apparently wild-type protein. Our study has established that genetic inactivation of EP300 is rare in primary colorectal, breast and ovarian cancers. In contrast, mutations are common among colorectal cancer cell lines with 4/17 harbouring homozygous or heterozygous mutations. The rarity of EP300 mutations among these tumour types that show a high frequency of LOH across 22q13 may indicate that another gene is the target of the loss. It is possible that bi-allelic inactivation of EP300 is not necessary and that haploinsufficiency is sufficient to promote tumorigenesis. Alternatively, silencing of EP300 may be achieved by epigenetic mechanisms such as promoter methylation.</p>
<p class="publication"><span class="title">International journal of cancer. Journal international du cancer</span>&nbsp;<span class="year">2002</span>;<span class="volume">102</span>;<span class="number">2</span>;<span class="pagerange">137-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12385008" rel="external">12385008</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fijc.10682" rel="external">10.1002/ijc.10682</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A full-coverage, high-resolution human chromosome 22 genomic microarray for clinical and research applications.</h4>
<p class="authors"><span class="author">Buckley PG</span>, <span class="author">Mantripragada KK</span>, <span class="author">Benetkiewicz M</span>, <span class="author">Tapia-P&aacute;ez I</span>, <span class="author">Diaz De St&aring;hl T</span>, <span class="author">Rosenquist M</span>, <span class="author">Ali H</span>, <span class="author">Jarbo C</span>, <span class="author">De Bustos C</span>, <span class="author">Hirvel&auml; C</span>, <span class="author">Sinder Wil&eacute;n B</span>, <span class="author">Fransson I</span>, <span class="author">Thyr C</span>, <span class="author">Johnsson BI</span>, <span class="author">Bruder CE</span>, <span class="author">Menzel U</span>, <span class="author">Hergersberg M</span>, <span class="author">Mandahl N</span>, <span class="author">Blennow E</span>, <span class="author">Wedell A</span>, <span class="author">Beare DM</span>, <span class="author">Collins JE</span>, <span class="author">Dunham I</span>, <span class="author">Albertson D</span>, <span class="author">Pinkel D</span>, <span class="author">Bastian BC</span>, <span class="author">Faruqi AF</span>, <span class="author">Lasken RS</span>, <span class="author">Ichimura K</span>, <span class="author">Collins VP</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Genetics and Pathology, Rudbeck laboratory, Uppsala University, 751 85 Uppsala, Sweden</p><p class="abstract">We have constructed the first comprehensive microarray representing a human chromosome for analysis of DNA copy number variation. This chromosome 22 array covers 34.7 Mb, representing 1.1% of the genome, with an average resolution of 75 kb. To demonstrate the utility of the array, we have applied it to profile acral melanoma, dermatofibrosarcoma, DiGeorge syndrome and neurofibromatosis 2. We accurately diagnosed homozygous/heterozygous deletions, amplifications/gains, IGLV/IGLC locus instability, and breakpoints of an imbalanced translocation. We further identified the 14-3-3 eta isoform as a candidate tumor suppressor in glioblastoma. Two significant methodological advances in array construction were also developed and validated. These include a strictly sequence defined, repeat-free, and non-redundant strategy for array preparation. This approach allows an increase in array resolution and analysis of any locus; disregarding common repeats, genomic clone availability and sequence redundancy. In addition, we report that the application of phi29 DNA polymerase is advantageous in microarray preparation. A broad spectrum of issues in medical research and diagnostics can be approached using the array. This well annotated and gene-rich autosome contains numerous uncharacterized disease genes. It is therefore crucial to associate these genes to specific 22q-related conditions and this array will be instrumental towards this goal. Furthermore, comprehensive epigenetic profiling of 22q-located genes and high-resolution analysis of replication timing across the entire chromosome can be studied using our array.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">11</span>;<span class="number">25</span>;<span class="pagerange">3221-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12444106" rel="external">12444106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome analysis of an inducible prophage and prophage remnants integrated in the Streptococcus pyogenes strain SF370.</h4>
<p class="authors"><span class="author">Canchaya C</span>, <span class="author">Desiere F</span>, <span class="author">McShan WM</span>, <span class="author">Ferretti JJ</span>, <span class="author">Parkhill J</span> and <span class="author">Br&uuml;ssow H</span></p>
<p class="affiliation">Nestl&eacute; Research Center, Nestec Ltd. Vers-chez-les-Blanc, CH Lausanne 26, Switzerland.</p><p class="abstract">The mitomycin C inducible prophage SF370.1 from the highly pathogenic M1 serotype Streptococcus pyogenes isolate SF370 showed a 41-kb-long genome whose genetic organization resembled that of SF11-like pac-site Siphoviridae. Its closest relative was prophage NIH1.1 from an M3 serotype S. pyogenes strain, followed by S. pneumoniae phage MM1 and Lactobacillus phage phig1e, Listeria phage A118, and Bacillus phage SPP1 in a gradient of relatedness. Sequence similarity with the previously described prophages SF370.2 and SF370.3 from the same polylysogenic SF370 strain were mainly limited to the tail fiber genes. As in these two other prophages, SF370.1 encoded likely lysogenic conversion genes between the phage lysin and the right attachment site. The genes encoded the pyrogenic exotoxin C of S. pyogenes and a protein sharing sequence similarity with both DNases and mitogenic factors. The screening of the SF370 genome revealed further prophage-like elements. A 13-kb-long phage remnant SF370.4 encoded lysogeny and DNA replication genes. A closely related prophage remnant was identified in S. pyogenes strain Manfredo at a corresponding genome position. The two prophages differed by internal indels and gene replacements. Four phage-like integrases were detected; three were still accompanied by likely repressor genes. All prophage elements were integrated into coding sequences. The phage sequences complemented the coding sequences in all cases. The DNA repair genes mutL and mutS were separated by the prophage remnant SF370.4; prophage SF370.1 and S. pneumoniae phage MM1 integrated into homologous chromosomal locations. The prophage sequences were interpreted with a hypothesis that predicts elements of cooperation and an arms race between phage and host genomes.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI19034</span></p>
<p class="publication"><span class="title">Virology</span>&nbsp;<span class="year">2002</span>;<span class="volume">302</span>;<span class="number">2</span>;<span class="pagerange">245-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12441069" rel="external">12441069</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mice deficient for the wild-type p53-induced phosphatase gene (Wip1) exhibit defects in reproductive organs, immune function, and cell cycle control.</h4>
<p class="authors"><span class="author">Choi J</span>, <span class="author">Nannenga B</span>, <span class="author">Demidov ON</span>, <span class="author">Bulavin DV</span>, <span class="author">Cooney A</span>, <span class="author">Brayton C</span>, <span class="author">Zhang Y</span>, <span class="author">Mbawuike IN</span>, <span class="author">Bradley A</span>, <span class="author">Appella E</span> and <span class="author">Donehower LA</span></p>
<p class="affiliation">Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas 77030, USA.</p><p class="abstract">The Wip1 gene is a serine/threonine phosphatase that is induced in a p53-dependent manner by DNA-damaging agents. We show here that Wip1 message is expressed in moderate levels in all organs, but is present at very high levels in the testes, particularly in the postmeiotic round spermatid compartment of the seminiferous tubules. We have confirmed that Wip1 mRNA is induced by ionizing radiation in mouse tissues in a p53-dependent manner. To further determine the normal biological function of Wip1 in mammalian organisms, we have generated Wip1-deficient mice. Wip1 null mice are viable but show a variety of postnatal abnormalities, including variable male runting, male reproductive organ atrophy, reduced male fertility, and reduced male longevity. Mice lacking Wip1 show increased susceptibility to pathogens and diminished T- and B-cell function. Fibroblasts derived from Wip1 null embryos have decreased proliferation rates and appear to be compromised in entering mitosis. The data are consistent with an important role for Wip1 in spermatogenesis, lymphoid cell function, and cell cycle regulation.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA54897</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2002</span>;<span class="volume">22</span>;<span class="number">4</span>;<span class="pagerange">1094-105</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11809801" rel="external">11809801</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC134641" rel="external">134641</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of amplified and expressed genes in breast cancer by comparative hybridization onto microarrays of randomly selected cDNA clones.</h4>
<p class="authors"><span class="author">Clark J</span>, <span class="author">Edwards S</span>, <span class="author">John M</span>, <span class="author">Flohr P</span>, <span class="author">Gordon T</span>, <span class="author">Maillard K</span>, <span class="author">Giddings I</span>, <span class="author">Brown C</span>, <span class="author">Bagherzadeh A</span>, <span class="author">Campbell C</span>, <span class="author">Shipley J</span>, <span class="author">Wooster R</span> and <span class="author">Cooper CS</span></p>
<p class="affiliation">Section of Molecular Carcinogenesis, The Haddow Laboratories, Institute of Cancer Research, Sutton, Surrey, United Kingdom.</p><p class="abstract">Microarray analysis using sets of known human genes provides a powerful platform for identifying candidate oncogenes involved in DNA amplification events but suffers from the disadvantage that information can be gained only on genes that have been preselected for inclusion on the array. To address this issue, we have performed comparative genome hybridization (CGH) and expression analyses on microarrays of clones, randomly selected from a cDNA library, prepared from a cancer containing the DNA amplicon under investigation. Application of this approach to the BT474 breast carcinoma cell line, which contains amplicons at 20q13, 17q11-21, and 17q22-23, identified 50 amplified and expressed genes, including genes from these regions previously proposed as candidate oncogenes. When considered together with data from microarray expression profiles and Northern analyses, we were able to propose five genes as new candidate oncogenes where amplification in breast cancer cell lines was consistently associated with higher levels of RNA expression. These included the HB01 histone acetyl transferase gene at 17q22-23 and the TRAP100 gene, which encodes a thyroid hormone receptor-associated protein coactivator, at 17q11-21. The results demonstrate the utility of this microarray-based CGH approach in hunting for candidate oncogenes within DNA amplicons.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2002</span>;<span class="volume">34</span>;<span class="number">1</span>;<span class="pagerange">104-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11921288" rel="external">11921288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human genome. HapMap launched with pledges of $100 million.</h4>
<p class="authors"><span class="author">Couzin J</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2002</span>;<span class="volume">298</span>;<span class="number">5595</span>;<span class="pagerange">941-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12411675">12411675</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.298.5595.941a">10.1126/science.298.5595.941a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations of the BRAF gene in human cancer.</h4>
<p class="authors"><span class="author">Davies H</span>, <span class="author">Bignell GR</span>, <span class="author">Cox C</span>, <span class="author">Stephens P</span>, <span class="author">Edkins S</span>, <span class="author">Clegg S</span>, <span class="author">Teague J</span>, <span class="author">Woffendin H</span>, <span class="author">Garnett MJ</span>, <span class="author">Bottomley W</span>, <span class="author">Davis N</span>, <span class="author">Dicks E</span>, <span class="author">Ewing R</span>, <span class="author">Floyd Y</span>, <span class="author">Gray K</span>, <span class="author">Hall S</span>, <span class="author">Hawes R</span>, <span class="author">Hughes J</span>, <span class="author">Kosmidou V</span>, <span class="author">Menzies A</span>, <span class="author">Mould C</span>, <span class="author">Parker A</span>, <span class="author">Stevens C</span>, <span class="author">Watt S</span>, <span class="author">Hooper S</span>, <span class="author">Wilson R</span>, <span class="author">Jayatilake H</span>, <span class="author">Gusterson BA</span>, <span class="author">Cooper C</span>, <span class="author">Shipley J</span>, <span class="author">Hargrave D</span>, <span class="author">Pritchard-Jones K</span>, <span class="author">Maitland N</span>, <span class="author">Chenevix-Trench G</span>, <span class="author">Riggins GJ</span>, <span class="author">Bigner DD</span>, <span class="author">Palmieri G</span>, <span class="author">Cossu A</span>, <span class="author">Flanagan A</span>, <span class="author">Nicholson A</span>, <span class="author">Ho JW</span>, <span class="author">Leung SY</span>, <span class="author">Yuen ST</span>, <span class="author">Weber BL</span>, <span class="author">Seigler HF</span>, <span class="author">Darrow TL</span>, <span class="author">Paterson H</span>, <span class="author">Marais R</span>, <span class="author">Marshall CJ</span>, <span class="author">Wooster R</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p><p class="abstract">Cancers arise owing to the accumulation of mutations in critical genes that alter normal programmes of cell proliferation, differentiation and death. As the first stage of a systematic genome-wide screen for these genes, we have prioritized for analysis signalling pathways in which at least one gene is mutated in human cancer. The RAS RAF MEK ERK MAP kinase pathway mediates cellular responses to growth signals. RAS is mutated to an oncogenic form in about 15% of human cancer. The three RAF genes code for cytoplasmic serine/threonine kinases that are regulated by binding RAS. Here we report BRAF somatic missense mutations in 66% of malignant melanomas and at lower frequency in a wide range of human cancers. All mutations are within the kinase domain, with a single substitution (V599E) accounting for 80%. Mutated BRAF proteins have elevated kinase activity and are transforming in NIH3T3 cells. Furthermore, RAS function is not required for the growth of cancer cell lines with the V599E mutation. As BRAF is a serine/threonine kinase that is commonly activated by somatic point mutation in human cancer, it may provide new therapeutic opportunities in malignant melanoma.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">417</span>;<span class="number">6892</span>;<span class="pagerange">949-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12068308">12068308</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature00766">10.1038/nature00766</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A first-generation linkage disequilibrium map of human chromosome 22.</h4>
<p class="authors"><span class="author">Dawson E</span>, <span class="author">Abecasis GR</span>, <span class="author">Bumpstead S</span>, <span class="author">Chen Y</span>, <span class="author">Hunt S</span>, <span class="author">Beare DM</span>, <span class="author">Pabial J</span>, <span class="author">Dibling T</span>, <span class="author">Tinsley E</span>, <span class="author">Kirby S</span>, <span class="author">Carter D</span>, <span class="author">Papaspyridonos M</span>, <span class="author">Livingstone S</span>, <span class="author">Ganske R</span>, <span class="author">L&otilde;hmussaar E</span>, <span class="author">Zernant J</span>, <span class="author">T&otilde;nisson N</span>, <span class="author">Remm M</span>, <span class="author">M&auml;gi R</span>, <span class="author">Puurand T</span>, <span class="author">Vilo J</span>, <span class="author">Kurg A</span>, <span class="author">Rice K</span>, <span class="author">Deloukas P</span>, <span class="author">Mott R</span>, <span class="author">Metspalu A</span>, <span class="author">Bentley DR</span>, <span class="author">Cardon LR</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">DNA sequence variants in specific genes or regions of the human genome are responsible for a variety of phenotypes such as disease risk or variable drug response. These variants can be investigated directly, or through their non-random associations with neighbouring markers (called linkage disequilibrium (LD)). Here we report measurement of LD along the complete sequence of human chromosome 22. Duplicate genotyping and analysis of 1,504 markers in Centre d&#39;Etude du Polymorphisme Humain (CEPH) reference families at a median spacing of 15 kilobases (kb) reveals a highly variable pattern of LD along the chromosome, in which extensive regions of nearly complete LD up to 804 kb in length are interspersed with regions of little or no detectable LD. The LD patterns are replicated in a panel of unrelated UK Caucasians. There is a strong correlation between high LD and low recombination frequency in the extant genetic map, suggesting that historical and contemporary recombination rates are similar. This study demonstrates the feasibility of developing genome-wide maps of LD.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">418</span>;<span class="number">6897</span>;<span class="pagerange">544-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12110843">12110843</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature00864">10.1038/nature00864</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Computational detection and location of transcription start sites in mammalian genomic DNA.</h4>
<p class="authors"><span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom. <a href="mailto:%74%64%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#100;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Transcription, the process whereby RNA copies are made from sections of the DNA genome, is directed by promoter regions. These define the transcription start site, and also the set of cellular conditions under which the promoter is active. At least in more complex species, it appears to be common for genes to have several different transcription start sites, which may be active under different conditions. Eukaryotic promoters are complex and fairly diffuse structures, which have proven hard to detect in silico. We show that a novel hybrid machine-learning method is able to build useful models of promoters for &gt;50% of human transcription start sites. We estimate specificity to be &gt;70%, and demonstrate good positional accuracy. Based on the structure of our learned models, we conclude that a signal resembling the well known TATA box, together with flanking regions of C-G enrichment, are the most important sequence-based signals marking sites of transcriptional initiation at a large class of typical promoters.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2002</span>;<span class="volume">12</span>;<span class="number">3</span>;<span class="pagerange">458-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11875034">11875034</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC155284">155284</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.216102">10.1101/gr.216102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human genome sequences: enigmatic variations.</h4>
<p class="authors"><span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%69%64%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#100;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The sequence of the human genome should be completed in 2003. The next steps are to obtain accurate annotation of the genes within the sequence and to begin to define the sequences of multiple human genomes.</p>
<p class="publication"><span class="title">Mutagenesis</span>&nbsp;<span class="year">2002</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">457-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12435842" rel="external">12435842</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Linkage and association with type 1 diabetes on chromosome 1q42.</h4>
<p class="authors"><span class="author">Ewens KG</span>, <span class="author">Johnson LN</span>, <span class="author">Wapelhorst B</span>, <span class="author">O&#39;Brien K</span>, <span class="author">Gutin S</span>, <span class="author">Morrison VA</span>, <span class="author">Street C</span>, <span class="author">Gregory SG</span>, <span class="author">Spielman RS</span> and <span class="author">Concannon P</span></p>
<p class="affiliation">Department of Genetics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania 19104, USA.</p><p class="abstract">Type 1 diabetes is a complex disorder with multiple genetic loci and environmental factors contributing to disease etiology. In the current study, a human type 1 diabetes candidate region on chromosome 1q42 was mapped at high marker density in a panel of 616 multiplex type 1 diabetic families. To facilitate the identification and evaluation of candidate genes, a physical map of the 7-cM region surrounding the maximum logarithm of odds (LOD) score (2.46, P = 0.0004) was constructed. Genes were identified in the 500-kb region surrounding the marker yielding the peak LOD score and evaluated for polymorphism by resequencing. Single-nucleotide polymorphisms (SNPs) identified in these genes as well as other anonymous markers were tested for allelic association with type 1 diabetes by both family-based and case-control methods. A haplotype formed by common alleles at three adjacent markers (D1S225, D1S2383, and D1S251) was preferentially transmitted to affected offspring in type 1 diabetic families (nominal P = 0.006). These findings extend the evidence supporting the existence of a type 1 diabetes susceptibility locus on chromosome 1q42 and identify a candidate region amenable to positional cloning efforts.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK46618</span>, <span class="grant">DK46635</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2002</span>;<span class="volume">51</span>;<span class="number">11</span>;<span class="pagerange">3318-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12401725" rel="external">12401725</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptional regulation of the stem cell leukemia gene (SCL)--comparative analysis of five vertebrate SCL loci.</h4>
<p class="authors"><span class="author">G&ouml;ttgens B</span>, <span class="author">Barton LM</span>, <span class="author">Chapman MA</span>, <span class="author">Sinclair AM</span>, <span class="author">Knudsen B</span>, <span class="author">Grafham D</span>, <span class="author">Gilbert JG</span>, <span class="author">Rogers J</span>, <span class="author">Bentley DR</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Cambridge University, Cambridge, CB2 2XY, United Kingdom. <a href="mailto:%62%67%32%30%30%40%63%61%6D.%61%63.%75%6B">&#98;&#103;&#50;&#48;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The stem cell leukemia (SCL) gene encodes a bHLH transcription factor with a pivotal role in hematopoiesis and vasculogenesis and a pattern of expression that is highly conserved between mammals and zebrafish. Here we report the isolation and characterization of the zebrafish SCL locus together with the identification of three neighboring genes, IER5, MAP17, and MUPP1. This region spans 68 kb and comprises the longest zebrafish genomic sequence currently available for comparison with mammalian, chicken, and pufferfish sequences. Our data show conserved synteny between zebrafish and mammalian SCL and MAP17 loci, thus suggesting the likely genomic domain necessary for the conserved pattern of SCL expression. Long-range comparative sequence analysis/phylogenetic footprinting was used to identify noncoding conserved sequences representing candidate transcriptional regulatory elements. The SCL promoter/enhancer, exon 1, and the poly(A) region were highly conserved, but no homology to other known mouse SCL enhancers was detected in the zebrafish sequence. A combined homology/structure analysis of the poly(A) region predicted consistent structural features, suggesting a conserved functional role in mRNA regulation. Analysis of the SCL promoter/enhancer revealed five motifs, which were conserved from zebrafish to mammals, and each of which is essential for the appropriate pattern or level of SCL transcription.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2002</span>;<span class="volume">12</span>;<span class="number">5</span>;<span class="pagerange">749-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11997341" rel="external">11997341</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC186570" rel="external">186570</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.45502" rel="external">10.1101/gr.45502</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of the human malaria parasite Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Gardner MJ</span>, <span class="author">Hall N</span>, <span class="author">Fung E</span>, <span class="author">White O</span>, <span class="author">Berriman M</span>, <span class="author">Hyman RW</span>, <span class="author">Carlton JM</span>, <span class="author">Pain A</span>, <span class="author">Nelson KE</span>, <span class="author">Bowman S</span>, <span class="author">Paulsen IT</span>, <span class="author">James K</span>, <span class="author">Eisen JA</span>, <span class="author">Rutherford K</span>, <span class="author">Salzberg SL</span>, <span class="author">Craig A</span>, <span class="author">Kyes S</span>, <span class="author">Chan MS</span>, <span class="author">Nene V</span>, <span class="author">Shallom SJ</span>, <span class="author">Suh B</span>, <span class="author">Peterson J</span>, <span class="author">Angiuoli S</span>, <span class="author">Pertea M</span>, <span class="author">Allen J</span>, <span class="author">Selengut J</span>, <span class="author">Haft D</span>, <span class="author">Mather MW</span>, <span class="author">Vaidya AB</span>, <span class="author">Martin DM</span>, <span class="author">Fairlamb AH</span>, <span class="author">Fraunholz MJ</span>, <span class="author">Roos DS</span>, <span class="author">Ralph SA</span>, <span class="author">McFadden GI</span>, <span class="author">Cummings LM</span>, <span class="author">Subramanian GM</span>, <span class="author">Mungall C</span>, <span class="author">Venter JC</span>, <span class="author">Carucci DJ</span>, <span class="author">Hoffman SL</span>, <span class="author">Newbold C</span>, <span class="author">Davis RW</span>, <span class="author">Fraser CM</span> and <span class="author">Barrell B</span></p>
<p class="affiliation">The Institute for Genomic Research, 9712 Medical Center Drive, Rockville, Maryland 20850, USA. <a href="mailto:%67%61%72%64%6E%65%72%40%74%69%67%72.%6F%72%67">&#103;&#97;&#114;&#100;&#110;&#101;&#114;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The parasite Plasmodium falciparum is responsible for hundreds of millions of cases of malaria, and kills more than one million African children annually. Here we report an analysis of the genome sequence of P. falciparum clone 3D7. The 23-megabase nuclear genome consists of 14 chromosomes, encodes about 5,300 genes, and is the most (A + T)-rich genome sequenced to date. Genes involved in antigenic variation are concentrated in the subtelomeric regions of the chromosomes. Compared to the genomes of free-living eukaryotic microbes, the genome of this intracellular parasite encodes fewer enzymes and transporters, but a large proportion of genes are devoted to immune evasion and host-parasite interactions. Many nuclear-encoded proteins are targeted to the apicoplast, an organelle involved in fatty-acid and isoprenoid metabolism. The genome sequence provides the foundation for future studies of this organism, and is being exploited in the search for new drugs and vaccines to fight malaria.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI028398</span>; Wellcome Trust: <span class="grant">061524</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">419</span>;<span class="number">6906</span>;<span class="pagerange">498-511</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12368864">12368864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836256">3836256</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01097">10.1038/nature01097</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence and analysis of chromosome 2 of Dictyostelium discoideum.</h4>
<p class="authors"><span class="author">Gl&ouml;ckner G</span>, <span class="author">Eichinger L</span>, <span class="author">Szafranski K</span>, <span class="author">Pachebat JA</span>, <span class="author">Bankier AT</span>, <span class="author">Dear PH</span>, <span class="author">Lehmann R</span>, <span class="author">Baumgart C</span>, <span class="author">Parra G</span>, <span class="author">Abril JF</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Kumpf K</span>, <span class="author">Tunggal B</span>, <span class="author">Cox E</span>, <span class="author">Quail MA</span>, <span class="author">Platzer M</span>, <span class="author">Rosenthal A</span>, <span class="author">Noegel AA</span> and <span class="author">Dictyostelium Genome Sequencing Consortium</span></p>
<p class="affiliation">IMB Jena, Department of Genome Analysis, Beutenbergstr. 11, 07745 Jena, Germany. <a href="mailto:%67%65%72%6E%6F%74%40%69%6D%62-%6A%65%6E%61.%64%65">&#103;&#101;&#114;&#110;&#111;&#116;&#64;&#105;&#109;&#98;&#45;&#106;&#101;&#110;&#97;&#46;&#100;&#101;</a></p><p class="abstract">The genome of the lower eukaryote Dictyostelium discoideum comprises six chromosomes. Here we report the sequence of the largest, chromosome 2, which at 8 megabases (Mb) represents about 25% of the genome. Despite an A + T content of nearly 80%, the chromosome codes for 2,799 predicted protein coding genes and 73 transfer RNA genes. This gene density, about 1 gene per 2.6 kilobases (kb), is surpassed only by Saccharomyces cerevisiae (one per 2 kb) and is similar to that of Schizosaccharomyces pombe (one per 2.5 kb). If we assume that the other chromosomes have a similar gene density, we can expect around 11,000 genes in the D. discoideum genome. A significant number of the genes show higher similarities to genes of vertebrates than to those of other fully sequenced eukaryotes. This analysis strengthens the view that the evolutionary position of D. discoideum is located before the branching of metazoa and fungi but after the divergence of the plant kingdom, placing it close to the base of metazoan evolution.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">418</span>;<span class="number">6893</span>;<span class="pagerange">79-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12097910">12097910</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature00847">10.1038/nature00847</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A physical map of the mouse genome.</h4>
<p class="authors"><span class="author">Gregory SG</span>, <span class="author">Sekhon M</span>, <span class="author">Schein J</span>, <span class="author">Zhao S</span>, <span class="author">Osoegawa K</span>, <span class="author">Scott CE</span>, <span class="author">Evans RS</span>, <span class="author">Burridge PW</span>, <span class="author">Cox TV</span>, <span class="author">Fox CA</span>, <span class="author">Hutton RD</span>, <span class="author">Mullenger IR</span>, <span class="author">Phillips KJ</span>, <span class="author">Smith J</span>, <span class="author">Stalker J</span>, <span class="author">Threadgold GJ</span>, <span class="author">Birney E</span>, <span class="author">Wylie K</span>, <span class="author">Chinwalla A</span>, <span class="author">Wallis J</span>, <span class="author">Hillier L</span>, <span class="author">Carter J</span>, <span class="author">Gaige T</span>, <span class="author">Jaeger S</span>, <span class="author">Kremitzki C</span>, <span class="author">Layman D</span>, <span class="author">Maas J</span>, <span class="author">McGrane R</span>, <span class="author">Mead K</span>, <span class="author">Walker R</span>, <span class="author">Jones S</span>, <span class="author">Smith M</span>, <span class="author">Asano J</span>, <span class="author">Bosdet I</span>, <span class="author">Chan S</span>, <span class="author">Chittaranjan S</span>, <span class="author">Chiu R</span>, <span class="author">Fjell C</span>, <span class="author">Fuhrmann D</span>, <span class="author">Girn N</span>, <span class="author">Gray C</span>, <span class="author">Guin R</span>, <span class="author">Hsiao L</span>, <span class="author">Krzywinski M</span>, <span class="author">Kutsche R</span>, <span class="author">Lee SS</span>, <span class="author">Mathewson C</span>, <span class="author">McLeavy C</span>, <span class="author">Messervier S</span>, <span class="author">Ness S</span>, <span class="author">Pandoh P</span>, <span class="author">Prabhu AL</span>, <span class="author">Saeedi P</span>, <span class="author">Smailus D</span>, <span class="author">Spence L</span>, <span class="author">Stott J</span>, <span class="author">Taylor S</span>, <span class="author">Terpstra W</span>, <span class="author">Tsai M</span>, <span class="author">Vardy J</span>, <span class="author">Wye N</span>, <span class="author">Yang G</span>, <span class="author">Shatsman S</span>, <span class="author">Ayodeji B</span>, <span class="author">Geer K</span>, <span class="author">Tsegaye G</span>, <span class="author">Shvartsbeyn A</span>, <span class="author">Gebregeorgis E</span>, <span class="author">Krol M</span>, <span class="author">Russell D</span>, <span class="author">Overton L</span>, <span class="author">Malek JA</span>, <span class="author">Holmes M</span>, <span class="author">Heaney M</span>, <span class="author">Shetty J</span>, <span class="author">Feldblyum T</span>, <span class="author">Nierman WC</span>, <span class="author">Catanese JJ</span>, <span class="author">Hubbard T</span>, <span class="author">Waterston RH</span>, <span class="author">Rogers J</span>, <span class="author">de Jong PJ</span>, <span class="author">Fraser CM</span>, <span class="author">Marra M</span>, <span class="author">McPherson JD</span> and <span class="author">Bentley DR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">A physical map of a genome is an essential guide for navigation, allowing the location of any gene or other landmark in the chromosomal DNA. We have constructed a physical map of the mouse genome that contains 296 contigs of overlapping bacterial clones and 16,992 unique markers. The mouse contigs were aligned to the human genome sequence on the basis of 51,486 homology matches, thus enabling use of the conserved synteny (correspondence between chromosome blocks) of the two genomes to accelerate construction of the mouse map. The map provides a framework for assembly of whole-genome shotgun sequence data, and a tile path of clones for generation of the reference sequence. Definition of the human-mouse alignment at this level of resolution enables identification of a mouse clone that corresponds to almost any position in the human genome. The human sequence may be used to facilitate construction of other mammalian genome maps using the same strategy.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01 HG002137-03</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">418</span>;<span class="number">6899</span>;<span class="pagerange">743-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12181558">12181558</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature00957">10.1038/nature00957</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The use of structure information to increase alignment accuracy does not aid homologue detection with profile HMMs.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: The best quality multiple sequence alignments are generally considered to derive from structural superposition. However, no previous work has studied the relative performance of profile hidden Markov models (HMMs) derived from such alignments. Therefore several alignment methods have been used to generate multiple sequence alignments from 348 structurally aligned families in the HOMSTRAD database. The performance of profile HMMs derived from the structural and sequence-based alignments has been assessed for homologue detection.</p><p class="abstract"><strong>Results</strong>: The best alignment methods studied here correctly align nearly 80% of residues with respect to structure alignments. Alignment quality and model sensitivity are found to be dependent on average number, length, and identity of sequences in the alignment. The striking conclusion is that, although structural data may improve the quality of multiple sequence alignments, this does not add to the ability of the derived profile HMMs to find sequence homologues.</p><p class="abstract">A list of HOMSTRAD families used in this study and the corresponding Pfam families is available at http://www.sanger.ac.uk/Users/sgj/alignments/map.html Contact: <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">18</span>;<span class="number">9</span>;<span class="pagerange">1243-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12217916" rel="external">12217916</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conservation of long-range synteny and microsynteny between the genomes of two distantly related nematodes.</h4>
<p class="authors"><span class="author">Guiliano DB</span>, <span class="author">Hall N</span>, <span class="author">Jones SJ</span>, <span class="author">Clark LN</span>, <span class="author">Corton CH</span>, <span class="author">Barrell BG</span> and <span class="author">Blaxter ML</span></p>
<p class="affiliation">Institute of Cell, Animal and Population Biology, University of Edinburgh, Edinburgh EH9 3JT, UK.</p><p class="abstract"><strong>Background</strong>: Comparisons between the genomes of the closely related nematodes Caenorhabditis elegans and Caenorhabditis briggsae reveal high rates of rearrangement, with a bias towards within-chromosome events. To assess whether this pattern is true of nematodes in general, we have used genome sequence to compare two nematode species that last shared a common ancestor approximately 300 million years ago: the model C. elegans and the filarial parasite Brugia malayi.</p><p class="abstract"><strong>Results</strong>: An 83 kb region flanking the gene for Bm-mif-1 (macrophage migration inhibitory factor, a B. malayi homolog of a human cytokine) was sequenced. When compared to the complete genome of C. elegans, evidence for conservation of long-range synteny and microsynteny was found. Potential C. elegans orthologs for II of the 12 protein-coding genes predicted in the B. malayi sequence were identified. Ten of these orthologs were located on chromosome I, with eight clustered in a 2.3 Mb region. While several, relatively local, intrachromosomal rearrangements have occurred, the order, composition, and configuration of two gene clusters, each containing three genes, was conserved. Comparison of B. malayi BAC-end genome survey sequence to C. elegans also revealed a bias towards intrachromosome rearrangements.</p><p class="abstract"><strong>Conclusions</strong>: We suggest that intrachromosomal rearrangement is a major force driving chromosomal organization in nematodes, but is constrained by the interdigitation of functional elements of neighboring genes.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="number">10</span>;<span class="pagerange">RESEARCH0057</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12372145" rel="external">12372145</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC134624" rel="external">134624</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic organization of human CDS2 and evaluation as a candidate gene for corneal hereditary endothelial dystrophy 2 on chromosome 20p13.</h4>
<p class="authors"><span class="author">Halford S</span>, <span class="author">Inglis S</span>, <span class="author">Gwilliam R</span>, <span class="author">Spencer P</span>, <span class="author">Mohamed M</span>, <span class="author">Ebenezer ND</span> and <span class="author">Hunt DM</span></p>
<p class="publication"><span class="title">Experimental eye research</span>&nbsp;<span class="year">2002</span>;<span class="volume">75</span>;<span class="number">5</span>;<span class="pagerange">619-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12457874" rel="external">12457874</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence of Plasmodium falciparum chromosomes 1, 3-9 and 13.</h4>
<p class="authors"><span class="author">Hall N</span>, <span class="author">Pain A</span>, <span class="author">Berriman M</span>, <span class="author">Churcher C</span>, <span class="author">Harris B</span>, <span class="author">Harris D</span>, <span class="author">Mungall K</span>, <span class="author">Bowman S</span>, <span class="author">Atkin R</span>, <span class="author">Baker S</span>, <span class="author">Barron A</span>, <span class="author">Brooks K</span>, <span class="author">Buckee CO</span>, <span class="author">Burrows C</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth C</span>, <span class="author">Chillingworth T</span>, <span class="author">Christodoulou Z</span>, <span class="author">Clark L</span>, <span class="author">Clark R</span>, <span class="author">Corton C</span>, <span class="author">Cronin A</span>, <span class="author">Davies R</span>, <span class="author">Davis P</span>, <span class="author">Dear P</span>, <span class="author">Dearden F</span>, <span class="author">Doggett J</span>, <span class="author">Feltwell T</span>, <span class="author">Goble A</span>, <span class="author">Goodhead I</span>, <span class="author">Gwilliam R</span>, <span class="author">Hamlin N</span>, <span class="author">Hance Z</span>, <span class="author">Harper D</span>, <span class="author">Hauser H</span>, <span class="author">Hornsby T</span>, <span class="author">Holroyd S</span>, <span class="author">Horrocks P</span>, <span class="author">Humphray S</span>, <span class="author">Jagels K</span>, <span class="author">James KD</span>, <span class="author">Johnson D</span>, <span class="author">Kerhornou A</span>, <span class="author">Knights A</span>, <span class="author">Konfortov B</span>, <span class="author">Kyes S</span>, <span class="author">Larke N</span>, <span class="author">Lawson D</span>, <span class="author">Lennard N</span>, <span class="author">Line A</span>, <span class="author">Maddison M</span>, <span class="author">McLean J</span>, <span class="author">Mooney P</span>, <span class="author">Moule S</span>, <span class="author">Murphy L</span>, <span class="author">Oliver K</span>, <span class="author">Ormond D</span>, <span class="author">Price C</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rajandream MA</span>, <span class="author">Rutter S</span>, <span class="author">Rutherford KM</span>, <span class="author">Sanders M</span>, <span class="author">Simmonds M</span>, <span class="author">Seeger K</span>, <span class="author">Sharp S</span>, <span class="author">Smith R</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Stevens K</span>, <span class="author">Taylor K</span>, <span class="author">Tivey A</span>, <span class="author">Unwin L</span>, <span class="author">Whitehead S</span>, <span class="author">Woodward J</span>, <span class="author">Sulston JE</span>, <span class="author">Craig A</span>, <span class="author">Newbold C</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6E%68%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#104;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Since the sequencing of the first two chromosomes of the malaria parasite, Plasmodium falciparum, there has been a concerted effort to sequence and assemble the entire genome of this organism. Here we report the sequence of chromosomes 1, 3-9 and 13 of P. falciparum clone 3D7--these chromosomes account for approximately 55% of the total genome. We describe the methods used to map, sequence and annotate these chromosomes. By comparing our assemblies with the optical map, we indicate the completeness of the resulting sequence. During annotation, we assign Gene Ontology terms to the predicted gene products, and observe clustering of some malaria-specific terms to specific chromosomes. We identify a highly conserved sequence element found in the intergenic region of internal var genes that is not associated with their telomeric counterparts.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">419</span>;<span class="number">6906</span>;<span class="pagerange">527-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12368867">12368867</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01095">10.1038/nature01095</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Searching for clues.</h4>
<p class="authors"><span class="author">Holden M</span>, <span class="author">Sebaihia M</span>, <span class="author">Bentley S</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">8</span>;<span class="pagerange">354-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12160625" rel="external">12160625</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Split personalities.</h4>
<p class="authors"><span class="author">Holden M</span>, <span class="author">Sebaihia M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">3</span>;<span class="pagerange">115</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11864813" rel="external">11864813</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of the malaria mosquito Anopheles gambiae.</h4>
<p class="authors"><span class="author">Holt RA</span>, <span class="author">Subramanian GM</span>, <span class="author">Halpern A</span>, <span class="author">Sutton GG</span>, <span class="author">Charlab R</span>, <span class="author">Nusskern DR</span>, <span class="author">Wincker P</span>, <span class="author">Clark AG</span>, <span class="author">Ribeiro JM</span>, <span class="author">Wides R</span>, <span class="author">Salzberg SL</span>, <span class="author">Loftus B</span>, <span class="author">Yandell M</span>, <span class="author">Majoros WH</span>, <span class="author">Rusch DB</span>, <span class="author">Lai Z</span>, <span class="author">Kraft CL</span>, <span class="author">Abril JF</span>, <span class="author">Anthouard V</span>, <span class="author">Arensburger P</span>, <span class="author">Atkinson PW</span>, <span class="author">Baden H</span>, <span class="author">de Berardinis V</span>, <span class="author">Baldwin D</span>, <span class="author">Benes V</span>, <span class="author">Biedler J</span>, <span class="author">Blass C</span>, <span class="author">Bolanos R</span>, <span class="author">Boscus D</span>, <span class="author">Barnstead M</span>, <span class="author">Cai S</span>, <span class="author">Center A</span>, <span class="author">Chaturverdi K</span>, <span class="author">Christophides GK</span>, <span class="author">Chrystal MA</span>, <span class="author">Clamp M</span>, <span class="author">Cravchik A</span>, <span class="author">Curwen V</span>, <span class="author">Dana A</span>, <span class="author">Delcher A</span>, <span class="author">Dew I</span>, <span class="author">Evans CA</span>, <span class="author">Flanigan M</span>, <span class="author">Grundschober-Freimoser A</span>, <span class="author">Friedli L</span>, <span class="author">Gu Z</span>, <span class="author">Guan P</span>, <span class="author">Guigo R</span>, <span class="author">Hillenmeyer ME</span>, <span class="author">Hladun SL</span>, <span class="author">Hogan JR</span>, <span class="author">Hong YS</span>, <span class="author">Hoover J</span>, <span class="author">Jaillon O</span>, <span class="author">Ke Z</span>, <span class="author">Kodira C</span>, <span class="author">Kokoza E</span>, <span class="author">Koutsos A</span>, <span class="author">Letunic I</span>, <span class="author">Levitsky A</span>, <span class="author">Liang Y</span>, <span class="author">Lin JJ</span>, <span class="author">Lobo NF</span>, <span class="author">Lopez JR</span>, <span class="author">Malek JA</span>, <span class="author">McIntosh TC</span>, <span class="author">Meister S</span>, <span class="author">Miller J</span>, <span class="author">Mobarry C</span>, <span class="author">Mongin E</span>, <span class="author">Murphy SD</span>, <span class="author">O&#39;Brochta DA</span>, <span class="author">Pfannkoch C</span>, <span class="author">Qi R</span>, <span class="author">Regier MA</span>, <span class="author">Remington K</span>, <span class="author">Shao H</span>, <span class="author">Sharakhova MV</span>, <span class="author">Sitter CD</span>, <span class="author">Shetty J</span>, <span class="author">Smith TJ</span>, <span class="author">Strong R</span>, <span class="author">Sun J</span>, <span class="author">Thomasova D</span>, <span class="author">Ton LQ</span>, <span class="author">Topalis P</span>, <span class="author">Tu Z</span>, <span class="author">Unger MF</span>, <span class="author">Walenz B</span>, <span class="author">Wang A</span>, <span class="author">Wang J</span>, <span class="author">Wang M</span>, <span class="author">Wang X</span>, <span class="author">Woodford KJ</span>, <span class="author">Wortman JR</span>, <span class="author">Wu M</span>, <span class="author">Yao A</span>, <span class="author">Zdobnov EM</span>, <span class="author">Zhang H</span>, <span class="author">Zhao Q</span>, <span class="author">Zhao S</span>, <span class="author">Zhu SC</span>, <span class="author">Zhimulev I</span>, <span class="author">Coluzzi M</span>, <span class="author">della Torre A</span>, <span class="author">Roth CW</span>, <span class="author">Louis C</span>, <span class="author">Kalush F</span>, <span class="author">Mural RJ</span>, <span class="author">Myers EW</span>, <span class="author">Adams MD</span>, <span class="author">Smith HO</span>, <span class="author">Broder S</span>, <span class="author">Gardner MJ</span>, <span class="author">Fraser CM</span>, <span class="author">Birney E</span>, <span class="author">Bork P</span>, <span class="author">Brey PT</span>, <span class="author">Venter JC</span>, <span class="author">Weissenbach J</span>, <span class="author">Kafatos FC</span>, <span class="author">Collins FH</span> and <span class="author">Hoffman SL</span></p>
<p class="affiliation">Celera Genomics, 45 West Gude Drive, Rockville, MD 20850, USA. <a href="mailto:%72%6F%62%65%72%74.%68%6F%6C%74%40%63%65%6C%65%72%61.%63%6F%6D">&#114;&#111;&#98;&#101;&#114;&#116;&#46;&#104;&#111;&#108;&#116;&#64;&#99;&#101;&#108;&#101;&#114;&#97;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Anopheles gambiae is the principal vector of malaria, a disease that afflicts more than 500 million people and causes more than 1 million deaths each year. Tenfold shotgun sequence coverage was obtained from the PEST strain of A. gambiae and assembled into scaffolds that span 278 million base pairs. A total of 91% of the genome was organized in 303 scaffolds; the largest scaffold was 23.1 million base pairs. There was substantial genetic variation within this strain, and the apparent existence of two haplotypes of approximately equal frequency (&quot;dual haplotypes&quot;) in a substantial fraction of the genome likely reflects the outbred nature of the PEST strain. The sequence produced a conservative inference of more than 400,000 single-nucleotide polymorphisms that showed a markedly bimodal density distribution. Analysis of the genome sequence revealed strong evidence for about 14,000 protein-encoding transcripts. Prominent expansions in specific families of proteins likely involved in cell adhesion and immunity were noted. An expressed sequence tag analysis of genes regulated by blood feeding provided insights into the physiological adaptations of a hematophagous insect.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01AI44273</span>, <span class="grant">U01AI48846</span>, <span class="grant">U01AI50687</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2002</span>;<span class="volume">298</span>;<span class="number">5591</span>;<span class="pagerange">129-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12364791">12364791</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1076181">10.1126/science.1076181</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">QuickTree: building huge Neighbour-Joining trees of protein sequences.</h4>
<p class="authors"><span class="author">Howe K</span>, <span class="author">Bateman A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%6B%6C%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#107;&#108;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have written a fast implementation of the popular Neighbor-Joining tree building algorithm. QuickTree allows the reconstruction of phylogenies for very large protein families (including the largest Pfam alignment containing 27000 HIV GP120 glycoprotein sequences) that would be infeasible using other popular methods.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">18</span>;<span class="number">11</span>;<span class="pagerange">1546-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12424131">12424131</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GAZE: a generic framework for the integration of gene-prediction data by dynamic programming.</h4>
<p class="authors"><span class="author">Howe KL</span>, <span class="author">Chothia T</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">We describe a method (implemented in a program, GAZE) for assembling arbitrary evidence for individual gene components (features) into predictions of complete gene structures. Our system is generic in that both the features themselves, and the model of gene structure against which potential assemblies are validated and scored, are external to the system and supplied by the user. GAZE uses a dynamic programming algorithm to obtain the highest scoring gene structure according to the model and posterior probabilities that each input feature is part of a gene. A novel pruning strategy ensures that the algorithm has a run-time effectively linear in sequence length. To demonstrate the flexibility of our system in the incorporation of additional evidence into the gene prediction process, we show how it can be used to both represent nonstandard gene structures (in the form of trans-spliced genes in Caenorhabditis elegans), and make use of similarity information (in the form of Expressed Sequence Tag alignments), while requiring no change to the underlying software. GAZE is available at http://www.sanger.ac.uk/Software/analysis/GAZE.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2002</span>;<span class="volume">12</span>;<span class="number">9</span>;<span class="pagerange">1418-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12213779">12213779</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC186661">186661</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.149502">10.1101/gr.149502</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biological information: making it accessible and integrated (and trying to make sense of it).</h4>
<p class="authors"><span class="author">Hubbard T</span></p>
<p class="affiliation">Sanger Institute, Cambridgeshire, UK.</p><p class="abstract">The availability of the genome sequences of human and mouse, human sequence variation data and other large genetic data sets will lead to a revolution in understanding of the human machine and the treatment of its diseases. The success of the international genome sequencing consortiums shows what can be achieved by well coordinated large scale public domain projects and the benefits of data access to all. It is already clear that the availability of this sequence is having a huge impact on research worldwide. Complete genome sequences provide a framework to pull all biological data together such that each piece has the potential to say something about biology as a whole. Biology is too complex for any organisation to have a monopoly of ideas or data, so the collection, analysis and access to this data can be contributed to by research institutes around the world. However, although it is possible for all this data to be accessible to all through the internet, the more organisations provide data or analysis separately, the harder it becomes for anyone to collect and integrate the results. To address these problems of intergration of data, open standards for biological data exchange, such as the &#39;Distributed Annotation System&#39; (DAS) are being developed and bioinformatics (Dowell et al., 2001) as a whole is now being strongly driven by the open source software (OSS) model for collaborative software development (Hubbard and Birney, 1999). The leading provider of human genome annotation, the Ensembl project (http://www.ensembl.org), is entirely an OSS project and has been widely adopted by academic and commerical organisations alike (Hubbard et al., 2002). Accurate automatic annotation of features such as genes in vertebrate genomes currently relies on supporting evidence in the form of homologies to mRNAs, ESTs or protein. However, it appears that sufficient high quality experimentally curated annotation now exists to be used as a substrate for machine learning algorithms to create effective models of biological signal sequences (Down and Hubbard, 2002). Is there hope for ab initio prediction methods after all?</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">18 Suppl 2</span>;<span class="pagerange">S140</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12385995" rel="external">12385995</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ensembl genome database project.</h4>
<p class="authors"><span class="author">Hubbard T</span>, <span class="author">Barker D</span>, <span class="author">Birney E</span>, <span class="author">Cameron G</span>, <span class="author">Chen Y</span>, <span class="author">Clark L</span>, <span class="author">Cox T</span>, <span class="author">Cuff J</span>, <span class="author">Curwen V</span>, <span class="author">Down T</span>, <span class="author">Durbin R</span>, <span class="author">Eyras E</span>, <span class="author">Gilbert J</span>, <span class="author">Hammond M</span>, <span class="author">Huminiecki L</span>, <span class="author">Kasprzyk A</span>, <span class="author">Lehvaslaiho H</span>, <span class="author">Lijnzaad P</span>, <span class="author">Melsopp C</span>, <span class="author">Mongin E</span>, <span class="author">Pettett R</span>, <span class="author">Pocock M</span>, <span class="author">Potter S</span>, <span class="author">Rust A</span>, <span class="author">Schmidt E</span>, <span class="author">Searle S</span>, <span class="author">Slater G</span>, <span class="author">Smith J</span>, <span class="author">Spooner W</span>, <span class="author">Stabenau A</span>, <span class="author">Stalker J</span>, <span class="author">Stupka E</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Vastrik I</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute and European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The Ensembl (http://www.ensembl.org/) database project provides a bioinformatics framework to organise biology around the sequences of large genomes. It is a comprehensive source of stable automatic annotation of the human genome sequence, with confirmed gene predictions that have been integrated with external data sources, and is available as either an interactive web site or as flat files. It is also an open source software engineering project to develop a portable system able to handle very large genomes and associated requirements from sequence analysis to data storage and visualisation. The Ensembl site is one of the leading sources of human genome sequence annotation and provided much of the analysis for publication by the international human genome project of the draft genome. The Ensembl system is being installed around the world in both companies and academic sites on machines ranging from supercomputers to laptops.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2002</span>;<span class="volume">30</span>;<span class="number">1</span>;<span class="pagerange">38-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11752248">11752248</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC99161">99161</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An anthropoid-specific locus of orphan C to U RNA-editing enzymes on chromosome 22.</h4>
<p class="authors"><span class="author">Jarmuz A</span>, <span class="author">Chester A</span>, <span class="author">Bayliss J</span>, <span class="author">Gisbourne J</span>, <span class="author">Dunham I</span>, <span class="author">Scott J</span> and <span class="author">Navaratnam N</span></p>
<p class="affiliation">RNA Editing Group, MRC Clinical Sciences Centre, Imperial College, Hammersmith Hospital Campus, Du Cane Road, London, W12 ONN, UK.</p><p class="abstract">The cytidine (C) to uridine (U) editing of apolipoprotein (apo) B mRNA is mediated by tissue-specific, RNA-binding cytidine deaminase APOBEC1. APOBEC1 is structurally homologous to Escherichia coli cytidine deaminase (ECCDA), but has evolved specific features required for RNA substrate binding and editing. A signature sequence for APOBEC1 has been used to identify other members of this family. One of these genes, designated APOBEC2, is found on chromosome 6. Another gene corresponds to the activation-induced deaminase (AID) gene, which is located adjacent to APOBEC1 on chromosome 12. Seven additional genes, or pseudogenes (designated APOBEC3A to 3G), are arrayed in tandem on chromosome 22. Not present in rodents, this locus is apparently an anthropoid-specific expansion of the APOBEC family. The conclusion that these new genes encode orphan C to U RNA-editing enzymes of the APOBEC family comes from similarity in amino acid sequence with APOBEC1, conserved intron/exon organization, tissue-specific expression, homodimerization, and zinc and RNA binding similar to APOBEC1. Tissue-specific expression of these genes in a variety of cell lines, along with other evidence, suggests a role for these enzymes in growth or cell cycle control.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2002</span>;<span class="volume">79</span>;<span class="number">3</span>;<span class="pagerange">285-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11863358" rel="external">11863358</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2002.6718" rel="external">10.1006/geno.2002.6718</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterisation and distribution of a cryptic Salmonella typhi plasmid pHCM2.</h4>
<p class="authors"><span class="author">Kidgell C</span>, <span class="author">Pickard D</span>, <span class="author">Wain J</span>, <span class="author">James K</span>, <span class="author">Diem Nga LT</span>, <span class="author">Diep TS</span>, <span class="author">Levine MM</span>, <span class="author">O&#39;Gaora P</span>, <span class="author">Prentice MB</span>, <span class="author">Parkhill J</span>, <span class="author">Day N</span>, <span class="author">Farrar J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Centre for Molecular Microbiology and Infection, Department of Biological Sciences, Imperial College of Science, Technology and Medicine, London SW7 2AY, UK.</p><p class="abstract">pHCM2 is a 106 kbp cryptic plasmid harboured by Salmonella typhi CT18, originally isolated from a typhoid patient in Vietnam. The genome of S. typhi CT18, including pHCM2, has recently been completely sequenced and annotated. Bioinformatic analysis revealed that 57% of the coding sequences (CDSs) encoded on pHCM2 display over 97% DNA sequence identity to the virulence-associated plasmid of Yersinia pestis, pFra. pHCM2 encodes no obvious virulence-associated determinants or antibiotic resistance genes but does encode a wide array of putative genes directly related to DNA metabolism and replication. PCR analysis of a series of S. typhi isolates from Vietnam detected pHCM2-related DNA sequences in some S. typhi isolated before, but not after, 1994. Similar pHCM2-related sequences were also detected in S. typhi isolated from other regions of South East Asia and Pakistan but not elsewhere in the world.</p>
<p class="publication"><span class="title">Plasmid</span>&nbsp;<span class="year">2002</span>;<span class="volume">47</span>;<span class="number">3</span>;<span class="pagerange">159-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12151231" rel="external">12151231</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The sterol-sensing domain: multiple families, a unique role?</h4>
<p class="authors"><span class="author">Kuwabara PE</span> and <span class="author">Labouesse M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%70%65%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#101;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The &quot;sterol-sensing domain&quot; (SSD) is conserved across phyla and is present in several membrane proteins, such as Patched (a Hedgehog receptor) and NPC-1 (the protein defective in Niemann-Pick type C1 disease). The role of the SSD is perhaps best understood from the standpoint of its involvement in cholesterol homeostasis. This article discusses how the SSD appears to function as a regulatory domain involved in linking vesicle trafficking and protein localization with such varied processes as cholesterol homeostasis, cell signalling and cytokinesis.</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2002</span>;<span class="volume">18</span>;<span class="number">4</span>;<span class="pagerange">193-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11932020" rel="external">11932020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MaxBench: evaluation of sequence and structure comparison methods.</h4>
<p class="authors"><span class="author">Leplae R</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6C%70%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#112;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Summary</strong>: MaxBench is a web-based system available for evaluating the results of sequence and structure comparison methods, based on the SCOP protein domain classification. The system makes it easy for developers to both compare the overall performance of their methods to standard algorithms and investigate the results of individual comparisons.</p><p class="abstract"><strong>Availability</strong>: http://www.sanger.ac.uk/Users/lp1/MaxBench/</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">18</span>;<span class="number">3</span>;<span class="pagerange">494-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11934754">11934754</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human homologue of unc-93 maps to chromosome 6q27 - characterisation and analysis in sporadic epithelial ovarian cancer.</h4>
<p class="authors"><span class="author">Liu Y</span>, <span class="author">Dodds P</span>, <span class="author">Emilion G</span>, <span class="author">Mungall AJ</span>, <span class="author">Dunham I</span>, <span class="author">Beck S</span>, <span class="author">Wells RS</span>, <span class="author">Charnock FM</span> and <span class="author">Ganesan TS</span></p>
<p class="affiliation">Cancer Research UK Molecular Oncology Laboratories, Weatherall Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford, OX3 9DS, UK. <a href="mailto:%79%69%6E%67.%6C%69%75%40%63%61%6E%63%65%72.%6F%72%67.%75%6B">&#121;&#105;&#110;&#103;&#46;&#108;&#105;&#117;&#64;&#99;&#97;&#110;&#99;&#101;&#114;&#46;&#111;&#114;&#103;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: In sporadic ovarian cancer, we have previously reported allele loss at D6S193 (62%) on chromosome 6q27, which suggested the presence of a putative tumour suppressor gene. Based on our data and that from another group, the minimal region of allele loss was between D6S264 and D6S149 (7.4 cM). To identify the putative tumour suppressor gene, we established a physical map initially with YACs and subsequently with PACs/BACs from D6S264 to D6S149. To accelerate the identification of genes, we sequenced the entire contig of approximately 1.1 Mb. Seven genes were identified within the region of allele loss between D6S264 and D6S149.</p><p class="abstract"><strong>Results</strong>: The human homologue of unc-93 (UNC93A) in C. elegans was identified to be within the interval of allele loss centromeric to D6S149. This gene is 24.5 kb and comprises of 8 exons. There are two transcripts with the shorter one due to splicing out of exon 4. It is expressed in testis, small intestine, spleen, prostate, and ovary. In a panel of 8 ovarian cancer cell lines, UNC93A expression was detected by RT-PCR which identified the two transcripts in 2/8 cell lines. The entire coding sequence was examined for mutations in a panel of ovarian tumours and ovarian cancer cell lines. Mutations were identified in exons 1, 3, 4, 5, 6 and 8. Only 3 mutations were identified specifically in the tumour. These included a c.452G&gt;A (W151X) mutation in exon 3, c.676C&gt;T (R226X) in exon 5 and c.1225G&gt;A(V409I) mutation in exon 8. However, the mutations in exon 3 and 5 were also present in 6% and 2% of the normal population respectively. The UNC93A cDNA was shown to express at the cell membrane and encodes for a protein of 60 kDa.</p><p class="abstract"><strong>Conclusions</strong>: These results suggest that no evidence for UNC93A as a tumour suppressor gene in sporadic ovarian cancer has been identified and further research is required to evaluate its normal function and role in the pathogenesis of ovarian cancer.</p>
<p class="publication"><span class="title">BMC genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="pagerange">20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12381271" rel="external">12381271</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC134458" rel="external">134458</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical and transcript map of the region between D6S264 and D6S149 on chromosome 6q27, the minimal region of allele loss in sporadic epithelial ovarian cancer.</h4>
<p class="authors"><span class="author">Liu Y</span>, <span class="author">Emilion G</span>, <span class="author">Mungall AJ</span>, <span class="author">Dunham I</span>, <span class="author">Beck S</span>, <span class="author">Le Meuth-Metzinger VG</span>, <span class="author">Shelling AN</span>, <span class="author">Charnock FM</span> and <span class="author">Ganesan TS</span></p>
<p class="affiliation">ICRF Molecular Oncology Laboratories, John Radcliffe Hospital, Headington, Oxford, OX3 9DS, UK.</p><p class="abstract">We have previously shown a high frequency of allele loss at D6S193 (62%) on chromosomal arm 6q27 in ovarian tumours and mapped the minimal region of allele loss between D6S297 and D6S264 (3 cM). We isolated and mapped a single non-chimaeric YAC (17IA12, 260-280 kb) containing D6S193 and D6S297. A further extended bacterial contig (between D6S264 and D6S149) has been established using PACs and BACs and a transcript map has been established. We have mapped six new markers to the YAC; three of them are ESTs (WI-15078, WI-8751, and TCP10). We have isolated three cDNA clones of EST WI-15078 and one clone contains a complete open reading frame. The sequence shows homology to a new member of the ribonuclease family. The other two clones are splice variants of this new gene. The gene is expressed ubiquitously in normal tissues. It is expressed in 4/8 ovarian cancer cell lines by Northern analysis. The gene encodes for a 40 kDa protein. Direct sequencing of the gene in all the eight ovarian cancer cell lines did not identify any mutations. Clonogenic assays were performed by transfecting the full-length gene in to ovarian cancer cell lines and no suppression of growth was observed.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2002</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">387-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11821951" rel="external">11821951</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1205067" rel="external">10.1038/sj.onc.1205067</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCOP database in 2002: refinements accommodate structural genomics.</h4>
<p class="authors"><span class="author">Lo Conte L</span>, <span class="author">Brenner SE</span>, <span class="author">Hubbard TJ</span>, <span class="author">Chothia C</span> and <span class="author">Murzin AG</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge CB2 2QH, UK. <a href="mailto:%6C%6F%72%65%64%61%6E%61%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#108;&#111;&#114;&#101;&#100;&#97;&#110;&#97;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The SCOP (Structural Classification of Proteins) database is a comprehensive ordering of all proteins of known structure, according to their evolutionary and structural relationships. Protein domains in SCOP are grouped into species and hierarchically classified into families, superfamilies, folds and classes. Recently, we introduced a new set of features with the aim of standardizing access to the database, and providing a solid basis to manage the increasing number of experimental structures expected from structural genomics projects. These features include: a new set of identifiers, which uniquely identify each entry in the hierarchy; a compact representation of protein domain classification; a new set of parseable files, which fully describe all domains in SCOP and the hierarchy itself. These new features are reflected in the ASTRAL compendium. The SCOP search engine has also been updated, and a set of links to external resources added at the level of domain entries. SCOP can be accessed at http://scop.mrc-lmb.cam.ac.uk/scop.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2002</span>;<span class="volume">30</span>;<span class="number">1</span>;<span class="pagerange">264-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11752311">11752311</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC99154">99154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The transcriptional program of meiosis and sporulation in fission yeast.</h4>
<p class="authors"><span class="author">Mata J</span>, <span class="author">Lyne R</span>, <span class="author">Burns G</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK.</p><p class="abstract">Sexual reproduction requires meiosis to produce haploid gametes, which in turn can fuse to regenerate a diploid organism. We have studied the transcriptional program that drives this developmental process in Schizosaccharomyces pombe using DNA microarrays. Here we show that hundreds of genes are regulated in successive waves of transcription that correlate with major biological events of meiosis and sporulation. Each wave is associated with specific promoter motifs. Clusters of neighboring genes (mostly close to telomeres) are co-expressed early in the process, which reflects a more global control of these genes. We find that two Atf-like transcription factors are essential for the expression of late genes and formation of spores, and identify dozens of potential Atf target genes. Comparison with the meiotic program of the distantly related Saccharomyces cerevisiae reveals an unexpectedly small shared meiotic transcriptome, suggesting that the transcriptional regulation of meiosis evolved independently in both species.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">32</span>;<span class="number">1</span>;<span class="pagerange">143-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12161753">12161753</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng951">10.1038/ng951</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A candidate gene for congenital bilateral isolated ptosis identified by molecular analysis of a de novo balanced translocation.</h4>
<p class="authors"><span class="author">McMullan TW</span>, <span class="author">Crolla JA</span>, <span class="author">Gregory SG</span>, <span class="author">Carter NP</span>, <span class="author">Cooper RA</span>, <span class="author">Howell GR</span> and <span class="author">Robinson DO</span></p>
<p class="affiliation">Southampton University School of Medicine, Human Genetics Division, Southampton General Hospital, Southampton SO16 6YD, UK.</p><p class="abstract">Ptosis is defined as drooping of the upper eyelid and can impair full visual acuity. It occurs in a number of forms including congenital bilateral isolated ptosis, which may be familial and for which two linkage groups are known on chromosomes 1p32-34.1 and Xq24-27.1. We describe the analysis of the chromosome breakpoints in a patient with congenital bilateral isolated ptosis and a de novo balanced translocation 46,XY,t(1;8)(p34.3;q21.12). Both breakpoints were localized by fluorescence in situ hybridisation with yeast artificial chromosomes, bacterial artificial chromosomes and P1 artificial chromosomes. The derived chromosomes were isolated by flow-sorting, amplified by degenerate oligonucleotide-primed polymerase chain reaction and analyzed by sequence tagged sites amplification to map the breakpoints at a resolution that enabled molecular characterization by DNA sequencing. The 1p breakpoint lies ~13 Mb distal to the previously reported linkage locus at 1p32-1p34.1 and does not disrupt a coding sequence, whereas the chromosome 8 breakpoint disrupts a gene homologous to the mouse zfh-4gene. Murine zfh-4 codes for a zinc finger homeodomain protein and is a transcription factor expressed in both muscle and nerve tissue. Human ZFH-4 is therefore a candidate gene for congenital bilateral isolated ptosis.</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">110</span>;<span class="number">3</span>;<span class="pagerange">244-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11935336" rel="external">11935336</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-002-0679-5" rel="external">10.1007/s00439-002-0679-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Low-penetrance susceptibility to breast cancer due to CHEK2(*)1100delC in noncarriers of BRCA1 or BRCA2 mutations.</h4>
<p class="authors"><span class="author">Meijers-Heijboer H</span>, <span class="author">van den Ouweland A</span>, <span class="author">Klijn J</span>, <span class="author">Wasielewski M</span>, <span class="author">de Snoo A</span>, <span class="author">Oldenburg R</span>, <span class="author">Hollestelle A</span>, <span class="author">Houben M</span>, <span class="author">Crepin E</span>, <span class="author">van Veghel-Plandsoen M</span>, <span class="author">Elstrodt F</span>, <span class="author">van Duijn C</span>, <span class="author">Bartels C</span>, <span class="author">Meijers C</span>, <span class="author">Schutte M</span>, <span class="author">McGuffog L</span>, <span class="author">Thompson D</span>, <span class="author">Easton D</span>, <span class="author">Sodha N</span>, <span class="author">Seal S</span>, <span class="author">Barfoot R</span>, <span class="author">Mangion J</span>, <span class="author">Chang-Claude J</span>, <span class="author">Eccles D</span>, <span class="author">Eeles R</span>, <span class="author">Evans DG</span>, <span class="author">Houlston R</span>, <span class="author">Murday V</span>, <span class="author">Narod S</span>, <span class="author">Peretz T</span>, <span class="author">Peto J</span>, <span class="author">Phelan C</span>, <span class="author">Zhang HX</span>, <span class="author">Szabo C</span>, <span class="author">Devilee P</span>, <span class="author">Goldgar D</span>, <span class="author">Futreal PA</span>, <span class="author">Nathanson KL</span>, <span class="author">Weber B</span>, <span class="author">Rahman N</span>, <span class="author">Stratton MR</span> and <span class="author">CHEK2-Breast Cancer Consortium</span></p>
<p class="affiliation">Department of Clinical Genetics, Erasmus Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">Mutations in BRCA1 and BRCA2 confer a high risk of breast and ovarian cancer, but account for only a small fraction of breast cancer susceptibility. To find additional genes conferring susceptibility to breast cancer, we analyzed CHEK2 (also known as CHK2), which encodes a cell-cycle checkpoint kinase that is implicated in DNA repair processes involving BRCA1 and p53 (refs 3,4,5). We show that CHEK2(*)1100delC, a truncating variant that abrogates the kinase activity, has a frequency of 1.1% in healthy individuals. However, this variant is present in 5.1% of individuals with breast cancer from 718 families that do not carry mutations in BRCA1 or BRCA2 (P = 0.00000003), including 13.5% of individuals from families with male breast cancer (P = 0.00015). We estimate that the CHEK2(*)1100delC variant results in an approximately twofold increase of breast cancer risk in women and a tenfold increase of risk in men. By contrast, the variant confers no increased cancer risk in carriers of BRCA1 or BRCA2 mutations. This suggests that the biological mechanisms underlying the elevated risk of breast cancer in CHEK2 mutation carriers are already subverted in carriers of BRCA1 or BRCA2 mutations, which is consistent with participation of the encoded proteins in the same pathway.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">31</span>;<span class="number">1</span>;<span class="pagerange">55-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11967536">11967536</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng879">10.1038/ng879</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative ab initio prediction of gene structures using pair HMMs.</h4>
<p class="authors"><span class="author">Meyer IM</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%69%6D%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#109;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present a novel comparative method for the ab initio prediction of protein coding genes in eukaryotic genomes. The method simultaneously predicts the gene structures of two un-annotated input DNA sequences which are homologous to each other and retrieves the subsequences which are conserved between the two DNA sequences. It is capable of predicting partial, complete and multiple genes and can align pairs of genes which differ by events of exon-fusion or exon-splitting. The method employs a probabilistic pair hidden Markov model. We generate annotations using our model with two different algorithms: the Viterbi algorithm in its linear memory implementation and a new heuristic algorithm, called the stepping stone, for which both memory and time requirements scale linearly with the sequence length. We have implemented the model in a computer program called DOUBLESCAN. In this article, we introduce the method and confirm the validity of the approach on a test set of 80 pairs of orthologous DNA sequences from mouse and human. More information can be found at: http://www.sanger.ac.uk/Software/analysis/doublescan/</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">18</span>;<span class="number">10</span>;<span class="pagerange">1309-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12376375">12376375</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Membrane-bound progesterone receptors contain a cytochrome b5-like ligand-binding domain.</h4>
<p class="authors"><span class="author">Mifsud W</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract"><strong>Background</strong>: Membrane-associated progesterone receptors (MAPRs) are thought to mediate a number of rapid cellular effects not involving changes in gene expression. They do not show sequence similarity to any of the classical steroid receptors. We were interested in identifying distant homologs of MAPR better to understand their biological roles.</p><p class="abstract"><strong>Results</strong>: We have identified MAPRs as distant homologs of cytochrome b5. We have also found regions homologous to cytochrome b5 in the mammalian HERC2 ubiquitin transferase proteins and a number of fungal chitin synthases.</p><p class="abstract"><strong>Conclusions</strong>: In view of these findings, we propose that the heme-binding cytochrome b5 domain served as a template for the evolution of membrane-associated binding pockets for non-heme ligands.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="number">12</span>;<span class="pagerange">RESEARCH0068</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12537557" rel="external">12537557</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC151170" rel="external">151170</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conditional inactivation of p63 by Cre-mediated excision.</h4>
<p class="authors"><span class="author">Mills AA</span>, <span class="author">Qi Y</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, One Bungtown Road, Cold Spring Harbor, New York 11724, USA. <a href="mailto:%6D%69%6C%6C%73%40%63%73%68%6C.%6F%72%67">&#109;&#105;&#108;&#108;&#115;&#64;&#99;&#115;&#104;&#108;&#46;&#111;&#114;&#103;</a></p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2002</span>;<span class="volume">32</span>;<span class="number">2</span>;<span class="pagerange">138-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11857801" rel="external">11857801</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the LGI1/Epitempin gene on 10q24 cause autosomal dominant lateral temporal epilepsy.</h4>
<p class="authors"><span class="author">Morante-Redolat JM</span>, <span class="author">Gorostidi-Pagola A</span>, <span class="author">Piquer-Sirerol S</span>, <span class="author">S&aacute;enz A</span>, <span class="author">Poza JJ</span>, <span class="author">Gal&aacute;n J</span>, <span class="author">Gesk S</span>, <span class="author">Sarafidou T</span>, <span class="author">Mautner VF</span>, <span class="author">Binelli S</span>, <span class="author">Staub E</span>, <span class="author">Hinzmann B</span>, <span class="author">French L</span>, <span class="author">Prud&#39;homme JF</span>, <span class="author">Passarelli D</span>, <span class="author">Scannapieco P</span>, <span class="author">Tassinari CA</span>, <span class="author">Avanzini G</span>, <span class="author">Mart&iacute;-Mass&oacute; JF</span>, <span class="author">Kluwe L</span>, <span class="author">Deloukas P</span>, <span class="author">Moschonas NK</span>, <span class="author">Michelucci R</span>, <span class="author">Siebert R</span>, <span class="author">Nobile C</span>, <span class="author">P&eacute;rez-Tur J</span> and <span class="author">L&oacute;pez de Munain A</span></p>
<p class="affiliation">Unitat de Gen&egrave;tica Molecular, Institut de Biomedicina de Val&egrave;ncia-CSIC, Jaume Roig 11, E-46010 Val&egrave;ncia, Spain.</p><p class="abstract">Autosomal dominant lateral temporal epilepsy (EPT; OMIM 600512) is a form of epilepsy characterized by partial seizures, usually preceded by auditory signs. The gene for this disorder has been mapped by linkage studies to chromosomal region 10q24. Here we show that mutations in the LGI1 gene segregate with EPT in two families affected by this disorder. Both mutations introduce premature stop codons and thus prevent the production of the full-length protein from the affected allele. By immunohistochemical studies, we demonstrate that the LGI1 protein, which contains several leucine-rich repeats, is expressed ubiquitously in the neuronal cell compartment of the brain. Moreover, we provide evidence for genetic heterogeneity within this disorder, since several other families with a phenotype consistent with this type of epilepsy lack mutations in the LGI1 gene.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">11</span>;<span class="number">9</span>;<span class="pagerange">1119-28</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11978770" rel="external">11978770</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cloning and characterization of the common fragile site FRA6F harboring a replicative senescence gene and frequently deleted in human tumors.</h4>
<p class="authors"><span class="author">Morelli C</span>, <span class="author">Karayianni E</span>, <span class="author">Magnanini C</span>, <span class="author">Mungall AJ</span>, <span class="author">Thorland E</span>, <span class="author">Negrini M</span>, <span class="author">Smith DI</span> and <span class="author">Barbanti-Brodano G</span></p>
<p class="affiliation">Department of Experimental and Diagnostic Medicine, Section of Microbiology and Center for Biotechnology, University of Ferrara, I-44100 Ferrara, Italy.</p><p class="abstract">The common fragile site FRA6F, located at 6q21, is an extended region of about 1200 kb, with two hot spots of breakage each spanning about 200 kb. Transcription mapping of the FRA6F region identified 19 known genes, 10 within the FRA6F interval and nine in a proximal or distal position. The nucleotide sequence of FRA6F is rich in repetitive elements (LINE1 and LINE2, Alu, MIR, MER and endogenous retroviral sequences) as well as in matrix attachment regions (MARs), and shows several DNA segments with increased helix flexibility. We found that tight clusters of stem-loop structures were localized exclusively in the two regions with greater frequency of breakage. Chromosomal instability at FRA6F probably depends on a complex interaction of different factors, involving regions of greater DNA flexibility and MARs. We propose an additional mechanism of fragility at FRA6F, based on stem-loop structures which may cause delay or arrest in DNA replication. A senescence gene likely maps within FRA6F, as suggested by detection of deletion and translocation breakpoints involving this fragile site in immortal human-mouse cell hybrids and in SV40-immortalized human fibroblasts containing a human chromosome 6 deleted at q21. Deletion breakpoints within FRA6F are common in several types of human leukemias and solid tumors, suggesting the presence of a tumor suppressor gene in the region. Moreover, a gene associated to hereditary schizophrenia maps within FRA6F. Therefore, FRA6F may represent a landmark for the identification and cloning of genes involved in senescence, leukemia, cancer and schizophrenia.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2002</span>;<span class="volume">21</span>;<span class="number">47</span>;<span class="pagerange">7266-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12370818" rel="external">12370818</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1205573" rel="external">10.1038/sj.onc.1205573</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The significance of performance ranking in CASP--response to Marti-Renom et al.</h4>
<p class="authors"><span class="author">Moult J</span>, <span class="author">Fidelis K</span>, <span class="author">Zemla A</span>, <span class="author">Hubbard T</span> and <span class="author">Tramontano A</span></p>
<p class="affiliation">Center for Advanced Research in Biotechnology, University of Maryland Biotechnology Institute, Rockville 20850, USA. <a href="mailto:%6A%6D%6F%75%6C%74%40%74%75%6E%63.%6F%72%67">&#106;&#109;&#111;&#117;&#108;&#116;&#64;&#116;&#117;&#110;&#99;&#46;&#111;&#114;&#103;</a></p>
<p class="publication"><span class="title">Structure (London, England : 1993)</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">3</span>;<span class="pagerange">291-2; discussion 292-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12005428">12005428</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Initial sequencing and comparative analysis of the mouse genome.</h4>
<p class="authors"><span class="author">Mouse Genome Sequencing Consortium</span>, <span class="author">Waterston RH</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Birney E</span>, <span class="author">Rogers J</span>, <span class="author">Abril JF</span>, <span class="author">Agarwal P</span>, <span class="author">Agarwala R</span>, <span class="author">Ainscough R</span>, <span class="author">Alexandersson M</span>, <span class="author">An P</span>, <span class="author">Antonarakis SE</span>, <span class="author">Attwood J</span>, <span class="author">Baertsch R</span>, <span class="author">Bailey J</span>, <span class="author">Barlow K</span>, <span class="author">Beck S</span>, <span class="author">Berry E</span>, <span class="author">Birren B</span>, <span class="author">Bloom T</span>, <span class="author">Bork P</span>, <span class="author">Botcherby M</span>, <span class="author">Bray N</span>, <span class="author">Brent MR</span>, <span class="author">Brown DG</span>, <span class="author">Brown SD</span>, <span class="author">Bult C</span>, <span class="author">Burton J</span>, <span class="author">Butler J</span>, <span class="author">Campbell RD</span>, <span class="author">Carninci P</span>, <span class="author">Cawley S</span>, <span class="author">Chiaromonte F</span>, <span class="author">Chinwalla AT</span>, <span class="author">Church DM</span>, <span class="author">Clamp M</span>, <span class="author">Clee C</span>, <span class="author">Collins FS</span>, <span class="author">Cook LL</span>, <span class="author">Copley RR</span>, <span class="author">Coulson A</span>, <span class="author">Couronne O</span>, <span class="author">Cuff J</span>, <span class="author">Curwen V</span>, <span class="author">Cutts T</span>, <span class="author">Daly M</span>, <span class="author">David R</span>, <span class="author">Davies J</span>, <span class="author">Delehaunty KD</span>, <span class="author">Deri J</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Dewey C</span>, <span class="author">Dickens NJ</span>, <span class="author">Diekhans M</span>, <span class="author">Dodge S</span>, <span class="author">Dubchak I</span>, <span class="author">Dunn DM</span>, <span class="author">Eddy SR</span>, <span class="author">Elnitski L</span>, <span class="author">Emes RD</span>, <span class="author">Eswara P</span>, <span class="author">Eyras E</span>, <span class="author">Felsenfeld A</span>, <span class="author">Fewell GA</span>, <span class="author">Flicek P</span>, <span class="author">Foley K</span>, <span class="author">Frankel WN</span>, <span class="author">Fulton LA</span>, <span class="author">Fulton RS</span>, <span class="author">Furey TS</span>, <span class="author">Gage D</span>, <span class="author">Gibbs RA</span>, <span class="author">Glusman G</span>, <span class="author">Gnerre S</span>, <span class="author">Goldman N</span>, <span class="author">Goodstadt L</span>, <span class="author">Grafham D</span>, <span class="author">Graves TA</span>, <span class="author">Green ED</span>, <span class="author">Gregory S</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Guyer M</span>, <span class="author">Hardison RC</span>, <span class="author">Haussler D</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Hillier LW</span>, <span class="author">Hinrichs A</span>, <span class="author">Hlavina W</span>, <span class="author">Holzer T</span>, <span class="author">Hsu F</span>, <span class="author">Hua A</span>, <span class="author">Hubbard T</span>, <span class="author">Hunt A</span>, <span class="author">Jackson I</span>, <span class="author">Jaffe DB</span>, <span class="author">Johnson LS</span>, <span class="author">Jones M</span>, <span class="author">Jones TA</span>, <span class="author">Joy A</span>, <span class="author">Kamal M</span>, <span class="author">Karlsson EK</span>, <span class="author">Karolchik D</span>, <span class="author">Kasprzyk A</span>, <span class="author">Kawai J</span>, <span class="author">Keibler E</span>, <span class="author">Kells C</span>, <span class="author">Kent WJ</span>, <span class="author">Kirby A</span>, <span class="author">Kolbe DL</span>, <span class="author">Korf I</span>, <span class="author">Kucherlapati RS</span>, <span class="author">Kulbokas EJ</span>, <span class="author">Kulp D</span>, <span class="author">Landers T</span>, <span class="author">Leger JP</span>, <span class="author">Leonard S</span>, <span class="author">Letunic I</span>, <span class="author">Levine R</span>, <span class="author">Li J</span>, <span class="author">Li M</span>, <span class="author">Lloyd C</span>, <span class="author">Lucas S</span>, <span class="author">Ma B</span>, <span class="author">Maglott DR</span>, <span class="author">Mardis ER</span>, <span class="author">Matthews L</span>, <span class="author">Mauceli E</span>, <span class="author">Mayer JH</span>, <span class="author">McCarthy M</span>, <span class="author">McCombie WR</span>, <span class="author">McLaren S</span>, <span class="author">McLay K</span>, <span class="author">McPherson JD</span>, <span class="author">Meldrim J</span>, <span class="author">Meredith B</span>, <span class="author">Mesirov JP</span>, <span class="author">Miller W</span>, <span class="author">Miner TL</span>, <span class="author">Mongin E</span>, <span class="author">Montgomery KT</span>, <span class="author">Morgan M</span>, <span class="author">Mott R</span>, <span class="author">Mullikin JC</span>, <span class="author">Muzny DM</span>, <span class="author">Nash WE</span>, <span class="author">Nelson JO</span>, <span class="author">Nhan MN</span>, <span class="author">Nicol R</span>, <span class="author">Ning Z</span>, <span class="author">Nusbaum C</span>, <span class="author">O&#39;Connor MJ</span>, <span class="author">Okazaki Y</span>, <span class="author">Oliver K</span>, <span class="author">Overton-Larty E</span>, <span class="author">Pachter L</span>, <span class="author">Parra G</span>, <span class="author">Pepin KH</span>, <span class="author">Peterson J</span>, <span class="author">Pevzner P</span>, <span class="author">Plumb R</span>, <span class="author">Pohl CS</span>, <span class="author">Poliakov A</span>, <span class="author">Ponce TC</span>, <span class="author">Ponting CP</span>, <span class="author">Potter S</span>, <span class="author">Quail M</span>, <span class="author">Reymond A</span>, <span class="author">Roe BA</span>, <span class="author">Roskin KM</span>, <span class="author">Rubin EM</span>, <span class="author">Rust AG</span>, <span class="author">Santos R</span>, <span class="author">Sapojnikov V</span>, <span class="author">Schultz B</span>, <span class="author">Schultz J</span>, <span class="author">Schwartz MS</span>, <span class="author">Schwartz S</span>, <span class="author">Scott C</span>, <span class="author">Seaman S</span>, <span class="author">Searle S</span>, <span class="author">Sharpe T</span>, <span class="author">Sheridan A</span>, <span class="author">Shownkeen R</span>, <span class="author">Sims S</span>, <span class="author">Singer JB</span>, <span class="author">Slater G</span>, <span class="author">Smit A</span>, <span class="author">Smith DR</span>, <span class="author">Spencer B</span>, <span class="author">Stabenau A</span>, <span class="author">Stange-Thomann N</span>, <span class="author">Sugnet C</span>, <span class="author">Suyama M</span>, <span class="author">Tesler G</span>, <span class="author">Thompson J</span>, <span class="author">Torrents D</span>, <span class="author">Trevaskis E</span>, <span class="author">Tromp J</span>, <span class="author">Ucla C</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Vinson JP</span>, <span class="author">Von Niederhausern AC</span>, <span class="author">Wade CM</span>, <span class="author">Wall M</span>, <span class="author">Weber RJ</span>, <span class="author">Weiss RB</span>, <span class="author">Wendl MC</span>, <span class="author">West AP</span>, <span class="author">Wetterstrand K</span>, <span class="author">Wheeler R</span>, <span class="author">Whelan S</span>, <span class="author">Wierzbowski J</span>, <span class="author">Willey D</span>, <span class="author">Williams S</span>, <span class="author">Wilson RK</span>, <span class="author">Winter E</span>, <span class="author">Worley KC</span>, <span class="author">Wyman D</span>, <span class="author">Yang S</span>, <span class="author">Yang SP</span>, <span class="author">Zdobnov EM</span>, <span class="author">Zody MC</span> and <span class="author">Lander ES</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University School of Medicine, Campus Box 8501, 4444 Forest Park Avenue, St Louis, Missouri 63108, USA. <a href="mailto:%77%61%74%65%72%73%74%6F%6E%40%67%73.%77%61%73%68%69%6E%67%74%6F%6E.%65%64%75">&#119;&#97;&#116;&#101;&#114;&#115;&#116;&#111;&#110;&#64;&#103;&#115;&#46;&#119;&#97;&#115;&#104;&#105;&#110;&#103;&#116;&#111;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The sequence of the mouse genome is a key informational tool for understanding the contents of the human genome and a key experimental tool for biomedical research. Here, we report the results of an international collaboration to produce a high-quality draft sequence of the mouse genome. We also present an initial comparative analysis of the mouse and human genomes, describing some of the insights that can be gleaned from the two sequences. We discuss topics including the analysis of the evolutionary forces shaping the size, structure and sequence of the genomes; the conservation of large-scale synteny across most of the genomes; the much lower extent of sequence orthology covering less than half of the genomes; the proportions of the genomes under selection; the number of protein-coding genes; the expansion of gene families related to reproduction and immunity; the evolution of proteins; and the identification of intraspecies polymorphism.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">420</span>;<span class="number">6915</span>;<span class="pagerange">520-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12466850">12466850</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01262">10.1038/nature01262</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">InterPro: an integrated documentation resource for protein families, domains and functional sites.</h4>
<p class="authors"><span class="author">Mulder NJ</span>, <span class="author">Apweiler R</span>, <span class="author">Attwood TK</span>, <span class="author">Bairoch A</span>, <span class="author">Bateman A</span>, <span class="author">Binns D</span>, <span class="author">Biswas M</span>, <span class="author">Bradley P</span>, <span class="author">Bork P</span>, <span class="author">Bucher P</span>, <span class="author">Copley R</span>, <span class="author">Courcelle E</span>, <span class="author">Durbin R</span>, <span class="author">Falquet L</span>, <span class="author">Fleischmann W</span>, <span class="author">Gouzy J</span>, <span class="author">Griffith-Jones S</span>, <span class="author">Haft D</span>, <span class="author">Hermjakob H</span>, <span class="author">Hulo N</span>, <span class="author">Kahn D</span>, <span class="author">Kanapin A</span>, <span class="author">Krestyaninova M</span>, <span class="author">Lopez R</span>, <span class="author">Letunic I</span>, <span class="author">Orchard S</span>, <span class="author">Pagni M</span>, <span class="author">Peyruc D</span>, <span class="author">Ponting CP</span>, <span class="author">Servant F</span>, <span class="author">Sigrist CJ</span> and <span class="author">InterPro Consortium</span></p>
<p class="affiliation">EMBL Outstation, European Bioinformatics Institute, Hinxton, Cambridge, UK. <a href="mailto:%6D%75%6C%64%65%72%40%65%62%69.%61%63.%75%6B">&#109;&#117;&#108;&#100;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The exponential increase in the submission of nucleotide sequences to the nucleotide sequence database by genome sequencing centres has resulted in a need for rapid, automatic methods for classification of the resulting protein sequences. There are several signature and sequence cluster-based methods for protein classification, each resource having distinct areas of optimum application owing to the differences in the underlying analysis methods. In recognition of this, InterPro was developed as an integrated documentation resource for protein families, domains and functional sites, to rationalise the complementary efforts of the individual protein signature database projects. The member databases - PRINTS, PROSITE, Pfam, ProDom, SMART and TIGRFAMs - form the InterPro core. Related signatures from each member database are unified into single InterPro entries. Each InterPro entry includes a unique accession number, functional descriptions and literature references, and links are made back to the relevant member database(s). Release 4.0 of InterPro (November 2001) contains 4,691 entries, representing 3,532 families, 1,068 domains, 74 repeats and 15 sites of post-translational modification (PTMs) encoded by different regular expressions, profiles, fingerprints and hidden Markov models (HMMs). Each InterPro entry lists all the matches against SWISS-PROT and TrEMBL (2,141,621 InterPro hits from 586,124 SWISS-PROT and TrEMBL protein sequences). The database is freely accessible for text- and sequence-based searches.</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">225-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12230031" rel="external">12230031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and characterization of a novel human brain-specific gene, homologous to S. scrofa tmp83.5, in the chromosome 10q24 critical region for temporal lobe epilepsy and spastic paraplegia.</h4>
<p class="authors"><span class="author">Nobile C</span>, <span class="author">Hinzmann B</span>, <span class="author">Scannapieco P</span>, <span class="author">Siebert R</span>, <span class="author">Zimbello R</span>, <span class="author">Perez-Tur J</span>, <span class="author">Sarafidou T</span>, <span class="author">Moschonas NK</span>, <span class="author">French L</span>, <span class="author">Deloukas P</span>, <span class="author">Ciccodicola A</span>, <span class="author">Gesk S</span>, <span class="author">Poza JJ</span>, <span class="author">Lo Nigro C</span>, <span class="author">Seri M</span>, <span class="author">Schlegelberger B</span>, <span class="author">Rosenthal A</span>, <span class="author">Valle G</span>, <span class="author">Lopez de Munain A</span>, <span class="author">Tassinari CA</span> and <span class="author">Michelucci R</span></p>
<p class="affiliation">CNR-Centro di Studio per la Biologia e Fisiopatologia Muscolare, Dipartimento di Scienze Biomediche Sperimentali, Viale G. Colombo 3, 35121 Padova, Italy. <a href="mailto:%63%61%72%6C%6F%40%74%65%6C%65%74%68%6F%6E.%62%69%6F.%75%6E%69%70%64.%69%74">&#99;&#97;&#114;&#108;&#111;&#64;&#116;&#101;&#108;&#101;&#116;&#104;&#111;&#110;&#46;&#98;&#105;&#111;&#46;&#117;&#110;&#105;&#112;&#100;&#46;&#105;&#116;</a></p><p class="abstract">We describe the structure, genomic organization, and some transcription features of a human brain-specific gene previously localized to the genomic region involved in temporal lobe epilepsy and spastic paraplegia on chromosome 10q24. The gene, which consists of six exons disseminated over 16 kb of genomic DNA, is highly homologous to the porcine tmp83.5 gene and encodes a putative transmembrane protein of 141 amino acids. Unlike its porcine homolog, from which two mRNAs with different 5&#39;-sequences are transcribed, the human gene apparently encodes three mRNA species with 3&#39;-untranslated regions of different sizes. Mutation analysis of its coding sequence in families affected with temporal lobe epilepsy or spastic paraplegia linked to 10q24 do not support the involvement of this gene in either diseases.</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">2002</span>;<span class="volume">282</span>;<span class="number">1-2</span>;<span class="pagerange">87-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11814680" rel="external">11814680</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenomics: genome-wide study of methylation phenomena.</h4>
<p class="authors"><span class="author">Novik KL</span>, <span class="author">Nimmrich I</span>, <span class="author">Genc B</span>, <span class="author">Maier S</span>, <span class="author">Piepenbrock C</span>, <span class="author">Olek A</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%6B%6C%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#107;&#108;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Epigenetics is one of the key areas of future research that can elucidate how genomes work. It combines genetics and the environment to address complex biological systems such as the plasticity of our genome. While all nucleated human cells carry the same genome, they express different genes at different times. Much of this is governed by epigenetic changes resulting in differential methylation of our genome--or different epigenomes. Individual studies over the past decades have already established the involvement of DNA methylation in imprinting, gene regulation, chromatin structure, genome stability and disease, especially cancer. Now, in the wake of the Human Genome Project (HGP), epigenetic phenomena can be studied genome-wide and are giving rise to a new field, epigenomics. Here, we review the current and future potential of this field and introduce the pilot study towards the Human Epigenome Project (HEP).</p>
<p class="publication"><span class="title">Current issues in molecular biology</span>&nbsp;<span class="year">2002</span>;<span class="volume">4</span>;<span class="number">4</span>;<span class="pagerange">111-28</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12432963" rel="external">12432963</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs.</h4>
<p class="authors"><span class="author">Okazaki Y</span>, <span class="author">Furuno M</span>, <span class="author">Kasukawa T</span>, <span class="author">Adachi J</span>, <span class="author">Bono H</span>, <span class="author">Kondo S</span>, <span class="author">Nikaido I</span>, <span class="author">Osato N</span>, <span class="author">Saito R</span>, <span class="author">Suzuki H</span>, <span class="author">Yamanaka I</span>, <span class="author">Kiyosawa H</span>, <span class="author">Yagi K</span>, <span class="author">Tomaru Y</span>, <span class="author">Hasegawa Y</span>, <span class="author">Nogami A</span>, <span class="author">Sch&ouml;nbach C</span>, <span class="author">Gojobori T</span>, <span class="author">Baldarelli R</span>, <span class="author">Hill DP</span>, <span class="author">Bult C</span>, <span class="author">Hume DA</span>, <span class="author">Quackenbush J</span>, <span class="author">Schriml LM</span>, <span class="author">Kanapin A</span>, <span class="author">Matsuda H</span>, <span class="author">Batalov S</span>, <span class="author">Beisel KW</span>, <span class="author">Blake JA</span>, <span class="author">Bradt D</span>, <span class="author">Brusic V</span>, <span class="author">Chothia C</span>, <span class="author">Corbani LE</span>, <span class="author">Cousins S</span>, <span class="author">Dalla E</span>, <span class="author">Dragani TA</span>, <span class="author">Fletcher CF</span>, <span class="author">Forrest A</span>, <span class="author">Frazer KS</span>, <span class="author">Gaasterland T</span>, <span class="author">Gariboldi M</span>, <span class="author">Gissi C</span>, <span class="author">Godzik A</span>, <span class="author">Gough J</span>, <span class="author">Grimmond S</span>, <span class="author">Gustincich S</span>, <span class="author">Hirokawa N</span>, <span class="author">Jackson IJ</span>, <span class="author">Jarvis ED</span>, <span class="author">Kanai A</span>, <span class="author">Kawaji H</span>, <span class="author">Kawasawa Y</span>, <span class="author">Kedzierski RM</span>, <span class="author">King BL</span>, <span class="author">Konagaya A</span>, <span class="author">Kurochkin IV</span>, <span class="author">Lee Y</span>, <span class="author">Lenhard B</span>, <span class="author">Lyons PA</span>, <span class="author">Maglott DR</span>, <span class="author">Maltais L</span>, <span class="author">Marchionni L</span>, <span class="author">McKenzie L</span>, <span class="author">Miki H</span>, <span class="author">Nagashima T</span>, <span class="author">Numata K</span>, <span class="author">Okido T</span>, <span class="author">Pavan WJ</span>, <span class="author">Pertea G</span>, <span class="author">Pesole G</span>, <span class="author">Petrovsky N</span>, <span class="author">Pillai R</span>, <span class="author">Pontius JU</span>, <span class="author">Qi D</span>, <span class="author">Ramachandran S</span>, <span class="author">Ravasi T</span>, <span class="author">Reed JC</span>, <span class="author">Reed DJ</span>, <span class="author">Reid J</span>, <span class="author">Ring BZ</span>, <span class="author">Ringwald M</span>, <span class="author">Sandelin A</span>, <span class="author">Schneider C</span>, <span class="author">Semple CA</span>, <span class="author">Setou M</span>, <span class="author">Shimada K</span>, <span class="author">Sultana R</span>, <span class="author">Takenaka Y</span>, <span class="author">Taylor MS</span>, <span class="author">Teasdale RD</span>, <span class="author">Tomita M</span>, <span class="author">Verardo R</span>, <span class="author">Wagner L</span>, <span class="author">Wahlestedt C</span>, <span class="author">Wang Y</span>, <span class="author">Watanabe Y</span>, <span class="author">Wells C</span>, <span class="author">Wilming LG</span>, <span class="author">Wynshaw-Boris A</span>, <span class="author">Yanagisawa M</span>, <span class="author">Yang I</span>, <span class="author">Yang L</span>, <span class="author">Yuan Z</span>, <span class="author">Zavolan M</span>, <span class="author">Zhu Y</span>, <span class="author">Zimmer A</span>, <span class="author">Carninci P</span>, <span class="author">Hayatsu N</span>, <span class="author">Hirozane-Kishikawa T</span>, <span class="author">Konno H</span>, <span class="author">Nakamura M</span>, <span class="author">Sakazume N</span>, <span class="author">Sato K</span>, <span class="author">Shiraki T</span>, <span class="author">Waki K</span>, <span class="author">Kawai J</span>, <span class="author">Aizawa K</span>, <span class="author">Arakawa T</span>, <span class="author">Fukuda S</span>, <span class="author">Hara A</span>, <span class="author">Hashizume W</span>, <span class="author">Imotani K</span>, <span class="author">Ishii Y</span>, <span class="author">Itoh M</span>, <span class="author">Kagawa I</span>, <span class="author">Miyazaki A</span>, <span class="author">Sakai K</span>, <span class="author">Sasaki D</span>, <span class="author">Shibata K</span>, <span class="author">Shinagawa A</span>, <span class="author">Yasunishi A</span>, <span class="author">Yoshino M</span>, <span class="author">Waterston R</span>, <span class="author">Lander ES</span>, <span class="author">Rogers J</span>, <span class="author">Birney E</span>, <span class="author">Hayashizaki Y</span>, <span class="author">FANTOM Consortium</span> and <span class="author">RIKEN Genome Exploration Research Group Phase I &amp;amp; II Team</span></p>
<p class="affiliation">[1] Laboratory for Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa, 230-0045, Japan.</p><p class="abstract">Only a small proportion of the mouse genome is transcribed into mature messenger RNA transcripts. There is an international collaborative effort to identify all full-length mRNA transcripts from the mouse, and to ensure that each is represented in a physical collection of clones. Here we report the manual annotation of 60,770 full-length mouse complementary DNA sequences. These are clustered into 33,409 &#39;transcriptional units&#39;, contributing 90.1% of a newly established mouse transcriptome database. Of these transcriptional units, 4,258 are new protein-coding and 11,665 are new non-coding messages, indicating that non-coding RNA is a major component of the transcriptome. 41% of all transcriptional units showed evidence of alternative splicing. In protein-coding transcripts, 79% of splice variations altered the protein product. Whole-transcriptome analyses resulted in the identification of 2,431 sense-antisense pairs. The present work, completely supported by physical clones, provides the most comprehensive survey of a mammalian transcriptome so far, and is a valuable resource for functional genomics.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">420</span>;<span class="number">6915</span>;<span class="pagerange">563-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12466851">12466851</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01266">10.1038/nature01266</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Human Genome Project: a player&#39;s perspective.</h4>
<p class="authors"><span class="author">Olson MV</span></p>
<p class="affiliation">Department of Medicine, UW Genome Center, University of Washington, Box 352145, Seattle, WA 98195, USA.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">2002</span>;<span class="volume">319</span>;<span class="number">4</span>;<span class="pagerange">931-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12079320" rel="external">12079320</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0022-2836(02)00333-9" rel="external">10.1016/S0022-2836(02)00333-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BRCA1 and BRCA2 germline mutations in Sardinian breast cancer families and their implications for genetic counseling.</h4>
<p class="authors"><span class="author">Palmieri G</span>, <span class="author">Palomba G</span>, <span class="author">Cossu A</span>, <span class="author">Pisano M</span>, <span class="author">Dedola MF</span>, <span class="author">Sarobba MG</span>, <span class="author">Farris A</span>, <span class="author">Olmeo N</span>, <span class="author">Contu A</span>, <span class="author">Pasca A</span>, <span class="author">Satta MP</span>, <span class="author">Persico I</span>, <span class="author">Carboni AA</span>, <span class="author">Cossu-Rocca P</span>, <span class="author">Contini M</span>, <span class="author">Mangion J</span>, <span class="author">Stratton MR</span> and <span class="author">Tanda F</span></p>
<p class="affiliation">Institute of Molecular Genetics, Consiglio Nazionale Ricerche, Alghero, Italy. <a href="mailto:%67%70%61%6C%6D%69%65%72%69%40%79%61%68%6F%6F.%63%6F%6D">&#103;&#112;&#97;&#108;&#109;&#105;&#101;&#114;&#105;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: The Sardinian population is genetically homogeneous and could be useful in understanding better the genetics of a complex disease like breast cancer (BC).</p><p class="abstract">Using a screening assay based on a combination of single-strand conformation polymorphism, denaturing high-performance liquid chromatography and sequence analysis, 47 Sardinian families with three or more BC cases were screened for germline mutations in BRCA1 and BRCA2 genes.</p><p class="abstract"><strong>Results</strong>: Three BRCA1/2 germline sequence variants were identified. While BRCA2-Ile3412Val is a missense variant with unknown functional significance, BRCA2-8765delAG and BRCA1-Lys505ter are two deleterious mutations (due to their predicted effects on protein truncation), which were found in seven families (15%). BRCA2-8765delAG was found in six of eight (75%) BRCA1/2-positive families and seven of 501 (1.4%) unselected and consecutively collected BC patients. Prevalence of BRCA1/2 mutations in BC families was significantly correlated with the total number of female BCs (P &lt;0.01) and increased by the presence of (i) at least one case of ovarian or male BC, or (ii) three generations affected, or (iii) bilateral BC.</p><p class="abstract"><strong>Conclusions</strong>: Identification of such features should address BC patients and their families to genetic counseling and BRCA1/2 mutational analysis. In addition, this is the first report of a detailed BRCA1/2 mutation screening in Sardinia, having immediate implications for the clinical management of BC families.</p>
<p class="publication"><span class="title">Annals of oncology : official journal of the European Society for Medical Oncology / ESMO</span>&nbsp;<span class="year">2002</span>;<span class="volume">13</span>;<span class="number">12</span>;<span class="pagerange">1899-907</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12453858" rel="external">12453858</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The importance of complete genome sequences.</h4>
<p class="authors"><span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">5</span>;<span class="pagerange">219-20; author reply 220</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11973153" rel="external">11973153</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation of TBCE causes hypoparathyroidism-retardation-dysmorphism and autosomal recessive Kenny-Caffey syndrome.</h4>
<p class="authors"><span class="author">Parvari R</span>, <span class="author">Hershkovitz E</span>, <span class="author">Grossman N</span>, <span class="author">Gorodischer R</span>, <span class="author">Loeys B</span>, <span class="author">Zecic A</span>, <span class="author">Mortier G</span>, <span class="author">Gregory S</span>, <span class="author">Sharony R</span>, <span class="author">Kambouris M</span>, <span class="author">Sakati N</span>, <span class="author">Meyer BF</span>, <span class="author">Al Aqeel AI</span>, <span class="author">Al Humaidan AK</span>, <span class="author">Al Zanhrani F</span>, <span class="author">Al Swaid A</span>, <span class="author">Al Othman J</span>, <span class="author">Diaz GA</span>, <span class="author">Weiner R</span>, <span class="author">Khan KT</span>, <span class="author">Gordon R</span>, <span class="author">Gelb BD</span> and <span class="author">HRD/Autosomal Recessive Kenny-Caffey Syndrome Consortium</span></p>
<p class="affiliation">Department of Developmental Molecular Genetics, Soroka Medical Center and Faculty of Health Sciences, Ben Gurion University of the Negev, Beer Sheva 84105, Israel.</p><p class="abstract">The syndrome of congenital hypoparathyroidism, mental retardation, facial dysmorphism and extreme growth failure (HRD or Sanjad-Sakati syndrome; OMIM 241410) is an autosomal recessive disorder reported almost exclusively in Middle Eastern populations. A similar syndrome with the additional features of osteosclerosis and recurrent bacterial infections has been classified as autosomal recessive Kenny-Caffey syndrome (AR-KCS; OMIM 244460). Both traits have previously been mapped to chromosome 1q43-44 (refs 5,6) and, despite the observed clinical variability, share an ancestral haplotype, suggesting a common founder mutation. We describe refinement of the critical region to an interval of roughly 230 kb and identification of deletion and truncation mutations of TBCE in affected individuals. The gene TBCE encodes one of several chaperone proteins required for the proper folding of alpha-tubulin subunits and the formation of alpha-beta-tubulin heterodimers. Analysis of diseased fibroblasts and lymphoblastoid cells showed lower microtubule density at the microtubule-organizing center (MTOC) and perturbed microtubule polarity in diseased cells. Immunofluorescence and ultrastructural studies showed disturbances in subcellular organelles that require microtubules for membrane trafficking, such as the Golgi and late endosomal compartments. These findings demonstrate that HRD and AR-KCS are chaperone diseases caused by a genetic defect in the tubulin assembly pathway, and establish a potential connection between tubulin physiology and the development of the parathyroid.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2002</span>;<span class="volume">32</span>;<span class="number">3</span>;<span class="pagerange">448-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12389028">12389028</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1012">10.1038/ng1012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping and identification of essential gene functions on the X chromosome of Drosophila.</h4>
<p class="authors"><span class="author">Peter A</span>, <span class="author">Sch&ouml;ttler P</span>, <span class="author">Werner M</span>, <span class="author">Beinert N</span>, <span class="author">Dowe G</span>, <span class="author">Burkert P</span>, <span class="author">Mourkioti F</span>, <span class="author">Dentzer L</span>, <span class="author">He Y</span>, <span class="author">Deak P</span>, <span class="author">Benos PV</span>, <span class="author">Gatt MK</span>, <span class="author">Murphy L</span>, <span class="author">Harris D</span>, <span class="author">Barrell B</span>, <span class="author">Ferraz C</span>, <span class="author">Vidal S</span>, <span class="author">Brun C</span>, <span class="author">Demaille J</span>, <span class="author">Cadieu E</span>, <span class="author">Dreano S</span>, <span class="author">Gloux S</span>, <span class="author">Lelaure V</span>, <span class="author">Mottier S</span>, <span class="author">Galibert F</span>, <span class="author">Borkova D</span>, <span class="author">Mi&ntilde;ana B</span>, <span class="author">Kafatos FC</span>, <span class="author">Bolshakov S</span>, <span class="author">Sid&eacute;n-Kiamos I</span>, <span class="author">Papagiannakis G</span>, <span class="author">Spanos L</span>, <span class="author">Louis C</span>, <span class="author">Madue&ntilde;o E</span>, <span class="author">de Pablos B</span>, <span class="author">Modolell J</span>, <span class="author">Bucheton A</span>, <span class="author">Callister D</span>, <span class="author">Campbell L</span>, <span class="author">Henderson NS</span>, <span class="author">McMillan PJ</span>, <span class="author">Salles C</span>, <span class="author">Tait E</span>, <span class="author">Valenti P</span>, <span class="author">Saunders RD</span>, <span class="author">Billaud A</span>, <span class="author">Pachter L</span>, <span class="author">Klapper R</span>, <span class="author">Janning W</span>, <span class="author">Glover DM</span>, <span class="author">Ashburner M</span>, <span class="author">Bellen HJ</span>, <span class="author">J&auml;ckle H</span> and <span class="author">Sch&auml;fer U</span></p>
<p class="affiliation">Max-Planck-Institut f&uuml;r Biophysikalische Chemie, Abt. Molekulare Entwicklungsbiologie, Am Fassberg, 37077 G&ouml;ttingen, Germany.</p><p class="abstract">The Drosophila melanogaster genome consists of four chromosomes that contain 165 Mb of DNA, 120 Mb of which are euchromatic. The two Drosophila Genome Projects, in collaboration with Celera Genomics Systems, have sequenced the genome, complementing the previously established physical and genetic maps. In addition, the Berkeley Drosophila Genome Project has undertaken large-scale functional analysis based on mutagenesis by transposable P element insertions into autosomes. Here, we present a large-scale P element insertion screen for vital gene functions and a BAC tiling map for the X chromosome. A collection of 501 X-chromosomal P element insertion lines was used to map essential genes cytogenetically and to establish short sequence tags (STSs) linking the insertion sites to the genome. The distribution of the P element integration sites, the identified genes and transcription units as well as the expression patterns of the P-element-tagged enhancers is described and discussed.</p>
<p class="publication"><span class="title">EMBO reports</span>&nbsp;<span class="year">2002</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">34-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11751581" rel="external">11751581</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1083931" rel="external">1083931</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fembo-reports%2Fkvf012" rel="external">10.1093/embo-reports/kvf012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Restricting genome data won&#39;t stop bioterrorism.</h4>
<p class="authors"><span class="author">Read TD</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">417</span>;<span class="number">6887</span>;<span class="pagerange">379</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12024189">12024189</a>; DOI: <a href="http://dx.doi.org/10.1038%2F417379a">10.1038/417379a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nucleotide sequence, genome organisation and phylogenetic analysis of Indian citrus ringspot virus. Brief report.</h4>
<p class="authors"><span class="author">Rustici G</span>, <span class="author">Milne RG</span> and <span class="author">Accotto GP</span></p>
<p class="affiliation">Istituto di Virologia Vegetale, Consiglio Nazionale delle Ricerche, Torino, Italy.</p><p class="abstract">The sequence of the single-stranded RNA genome of Indian citrus ringspot virus (ICRSV) consists of 7560 nucleotides. It contains six open reading frames (ORFs) which encode putative proteins of 187.3, 25, 12, 6.4, 34 and 23 kDa respectively. ORF1 encodes a polypeptide that contains all the elements of a replicase; ORFs 2, 3 and 4 compose a triple-gene block; ORF5 encodes the capsid protein; the function of ORF6 is unknown. Phylogenetic analysis of the complete genome and each ORF separately, and database searches indicate that ICRSV, though showing some similarities to potexviruses, is significantly different, as in the presence of ORF6, the genome and CP sizes, and particle morphology. These differences favour its inclusion in a new virus genus.</p>
<p class="publication"><span class="title">Archives of virology</span>&nbsp;<span class="year">2002</span>;<span class="volume">147</span>;<span class="number">11</span>;<span class="pagerange">2215-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12417955" rel="external">12417955</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00705-002-0875-6" rel="external">10.1007/s00705-002-0875-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterisation of transcripts from the human cytomegalovirus genes TRL7, UL20a, UL36, UL65, UL94, US3 and US34.</h4>
<p class="authors"><span class="author">Scott GM</span>, <span class="author">Barrell BG</span>, <span class="author">Oram J</span> and <span class="author">Rawlinson WD</span></p>
<p class="affiliation">Virology Division, Department of Microbiology SEALS, Prince of Wales Hospital, Randwick NSW, Australia.</p><p class="abstract">The genome of human cytomegalovirus (HCMV) has been studied extensively in some regions, but not others. In this study, transcripts of the genome were further characterised for open reading frames (ORFs) TRL7, UL36, UL65, UL94, US3 and US34, and for the previously unrecognised ORF, UL20a. Reverse transcription-PCR demonstrated the presence of spliced transcripts from the putative glycoprotein gene, UL20a, at early and late times post-infection. US3 full-length and spliced transcripts, including a previously unidentified transcript (US3ii), were described at immediate early times. Sequencing of the complete ORFs of UL20a and US3 from 21 clinical isolates showed that US3 is well conserved in all isolates (97-100% identity), whereas UL20a shows more variation at the nucleotide level, with 90-100% identity. The limits of transcription, and splice donor and acceptor sequences for UL20a and US3 were conserved in all isolates, indicating likely conservation of mRNA splicing patterns. Sequencing a late cDNA library identified the limits of transcription for ORFs TRL7, UL94 and US34 and transcription from the TRL7 ORF was confirmed by northern blotting. Transcripts were found that were congruent with UL36 and UL65, but these differed in the limits previously predicted for these ORFs. These findings show the variation between predicted and actual transcription and indicate the complex nature of transcription from HCMV ORFs.</p>
<p class="publication"><span class="title">Virus genes</span>&nbsp;<span class="year">2002</span>;<span class="volume">24</span>;<span class="number">1</span>;<span class="pagerange">39-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11928987" rel="external">11928987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome giants.</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Bentley S</span>, <span class="author">Thomson N</span>, <span class="author">Holden M</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">7</span>;<span class="pagerange">309-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12110204" rel="external">12110204</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tales of the unexpected.</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Bentley S</span>, <span class="author">Thomson N</span>, <span class="author">Holden M</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">6</span>;<span class="pagerange">261-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12088653" rel="external">12088653</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plant protein families and their relationships to food allergy.</h4>
<p class="authors"><span class="author">Shewry PR</span>, <span class="author">Beaudoin F</span>, <span class="author">Jenkins J</span>, <span class="author">Griffiths-Jones S</span> and <span class="author">Mills EN</span></p>
<p class="affiliation">Long Ashton Research Station, Department of Agricultural Sciences, University of Bristol, UK. <a href="mailto:%70%65%74%65%72.%73%68%65%77%72%79%40%62%62%73%72%63.%61%63.%75%6B">&#112;&#101;&#116;&#101;&#114;&#46;&#115;&#104;&#101;&#119;&#114;&#121;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The analysis of plant proteins has a long and distinguished history, with work dating back over 250 years. Much of the work has focused on seed proteins, which are important in animal nutrition and food processing. Early studies classified plant proteins into groups based on solubility (&#39;Osborne fractions&#39;) or protein function. More recently, families have been defined based on stuctural and evolutionary relationships. One of the most widespread groups of plant proteins is the prolaminin superfamily, which comprises cereal seed storage proteins, a range of low-molecular-mass sulphur-rich proteins (many of which are located in seeds) and some cell wall glycoproteins. This superfamily includes several major types of plant allergen: non-specific lipid transfer proteins, cereal seed inhibitors of alpha-amylase and/or trypsin, and 2 S albumin storage proteins of dicotyledonous seeds.</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2002</span>;<span class="volume">30</span>;<span class="number">Pt 6</span>;<span class="pagerange">906-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12440943" rel="external">12440943</a>; DOI: <a href="http://dx.doi.org/10.1042%2F" rel="external">10.1042/</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BRCA1 and BRCA2 mutation frequency in women evaluated in a breast cancer risk evaluation clinic.</h4>
<p class="authors"><span class="author">Shih HA</span>, <span class="author">Couch FJ</span>, <span class="author">Nathanson KL</span>, <span class="author">Blackwood MA</span>, <span class="author">Rebbeck TR</span>, <span class="author">Armstrong KA</span>, <span class="author">Calzone K</span>, <span class="author">Stopfer J</span>, <span class="author">Seal S</span>, <span class="author">Stratton MR</span> and <span class="author">Weber BL</span></p>
<p class="affiliation">Abramson Family Cancer Research Institute and Department of Biostatistics, University of Pennsylvania School of Medicine, Philadelphia, PA, USA.</p><p class="abstract"><strong>Purpose</strong>: To determine the prevalence of BRCA1 and BRCA2 mutations in families identified in a breast cancer risk evaluation clinic.</p><p class="abstract">One hundred sixty-four families seeking breast cancer risk evaluation were screened for coding region mutations in BRCA1 and BRCA2 by conformation-sensitive gel electrophoresis and DNA sequencing.</p><p class="abstract"><strong>Results</strong>: Mutations were identified in 37 families (22.6%); 28 (17.1%) had BRCA1 mutations and nine (5.5%) had BRCA2 mutations. The Ashkenazi Jewish founder mutations 185delAG and 5382insC (BRCA1) were found in 10 families (6.1%). However, 6174delT (BRCA2) was found in only one family (0.6%) despite estimates of equal frequency in the Ashkenazi population. In contrast to other series, the average age of breast cancer diagnosis was earlier in BRCA2 mutation carriers (32.1 years) than in women with BRCA1 mutations (37.6 years, P =.028). BRCA1 mutations were detected in 20 (45.5%) of 44 families with ovarian cancer and 12 (75%) of 16 families with both breast and ovarian cancer in a single individual. Significantly fewer BRCA2 mutations (two [4.5%] of 44) were detected in families with ovarian cancer (P =.01). Eight families had male breast cancer; one had a BRCA1 mutation and three had BRCA2 mutations.</p><p class="abstract"><strong>Conclusion</strong>: BRCA1 mutations were three times more prevalent than BRCA2 mutations. Breast cancer diagnosis before 50 years of age, ovarian cancer, breast and ovarian cancer in a single individual, and male breast cancer were all significantly more common in families with BRCA1 and BRCA2 mutations, but none of these factors distinguished between BRCA1 and BRCA2 mutations. Evidence for reduced breast cancer penetrance associated with the BRCA2 mutation 6174delT was noted.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA57601</span>, <span class="grant">CA84030</span></p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2002</span>;<span class="volume">20</span>;<span class="number">4</span>;<span class="pagerange">994-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11844822" rel="external">11844822</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pre-counseling education materials for BRCA testing: does tailoring make a difference?</h4>
<p class="authors"><span class="author">Skinner CS</span>, <span class="author">Schildkraut JM</span>, <span class="author">Berry D</span>, <span class="author">Calingaert B</span>, <span class="author">Marcom PK</span>, <span class="author">Sugarman J</span>, <span class="author">Winer EP</span>, <span class="author">Iglehart JD</span>, <span class="author">Futreal PA</span> and <span class="author">Rimer BK</span></p>
<p class="affiliation">Duke University Medical Center, Department of Community and Family Medicine and the Duke Comprehensive Cancer Center, Durham, NC 27710, USA. <a href="mailto:%63%65%6C%65%74%74%65.%73%6B%69%6E%6E%65%72%40%64%75%6B%65.%65%64%75">&#99;&#101;&#108;&#101;&#116;&#116;&#101;&#46;&#115;&#107;&#105;&#110;&#110;&#101;&#114;&#64;&#100;&#117;&#107;&#101;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Although tailored print materials (TPMs) have been assessed for a variety of behavioral targets, their effectiveness as decision aids for genetic testing had not been evaluated at the time this study began. We compared TPMs and non-tailored print material (NPMs) that included similar content about genetic testing for breast and ovarian cancer susceptibility. TPMs were prepared especially for an individual based on information from and about her. We mailed baseline surveys to 461 women referred by physicians or identified through a tumor registry. All had personal and family histories of breast and/or ovarian cancer and, on the basis of these histories, an estimated &gt; or =10% probability of carrying a mutation in the breast/ovarian cancer genes BRCA1 or BRCA2. The 325 (70%) who responded were randomly assigned to receive TPM or NPM. Followup surveys, mailed 2 weeks following receipt of print materials, were returned by 262 women (81% of baseline responders). Participants were predominately white (94%) and well-educated (50% college graduates). The mean age was 49 years. At follow-up, TPM recipients exhibited significantly greater improvement in percent of correct responses for the 13-item true/false measure of knowledge (24% increase for TPM vs. 16% for NPM; p &lt; 0.0001) and significantly less over-estimation of risk of being a mutation carrier (40% TPM group overestimated vs. 70% NPM; p &lt; 0.0001). Anxiety did not differ significantly between groups. Reactions to materials differed on two items: &quot;seemed to be prepared just for me&quot; (76% TPM vs. 52% NPM; p &lt; 0.001) and &quot;told me what I wanted to know about BRCA1 and 2 testing&quot; (98% TPM vs. 91% NPM; p &lt; 0.05). TPMs showed an advantage in increasing knowledge and enhancing accuracy of perceived risk. Both are critical components of informed decision making.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">2P50CA68438</span></p>
<p class="publication"><span class="title">Genetic testing</span>&nbsp;<span class="year">2002</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">93-105</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12215248" rel="external">12215248</a>; DOI: <a href="http://dx.doi.org/10.1089%2F10906570260199348" rel="external">10.1089/10906570260199348</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Srk1 protein kinase is a target for the Sty1 stress-activated MAPK in fission yeast.</h4>
<p class="authors"><span class="author">Smith DA</span>, <span class="author">Toone WM</span>, <span class="author">Chen D</span>, <span class="author">Bahler J</span>, <span class="author">Jones N</span>, <span class="author">Morgan BA</span> and <span class="author">Quinn J</span></p>
<p class="affiliation">School of Biochemistry and Genetics, Medical School, University of Newcastle upon Tyne, Newcastle upon Tyne NE2 4HH, United Kingdom.</p><p class="abstract">The fission yeast stress-activated Sty1/Spc1 MAPK pathway responds to a similar range of stresses as do the mammalian p38 and SAPK/JNK MAPK pathways. In addition, sty1(-) cells are sterile and exhibit a G(2) cell cycle delay, indicating additional roles of Sty1 in meiosis and cell cycle progression. To identify novel proteins involved in stress responses, a microarray analysis of the Schizosaccharomyces pombe genome was performed to find genes that are up-regulated following exposure to stress in a Sty1-dependent manner. One such gene identified, srk1(+) (Sty1-regulated kinase 1), encodes a putative serine/threonine kinase homologous to mammalian calmodulin kinases. At the C terminus of Srk1 is a putative MAPK binding motif similar to that in the p38 substrates, MAPK-activated protein kinases 2 and 3. Indeed, we find that Srk1 is present in a complex with the Sty1 MAPK and is directly phosphorylated by Sty1. Furthermore, upon stress, Srk1 translocates from the cytoplasm to the nucleus in a process that is dependent on the Sty1 MAPK. Finally, we show that Srk1 has a role in regulating meiosis in fission yeast; following nitrogen limitation, srk1(-) cells enter meiosis significantly faster than wild-type cells and overexpression of srk1(+) inhibits the nitrogen starvation-induced arrest in G(1).</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2002</span>;<span class="volume">277</span>;<span class="number">36</span>;<span class="pagerange">33411-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12080074" rel="external">12080074</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M204593200" rel="external">10.1074/jbc.M204593200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Bioperl toolkit: Perl modules for the life sciences.</h4>
<p class="authors"><span class="author">Stajich JE</span>, <span class="author">Block D</span>, <span class="author">Boulez K</span>, <span class="author">Brenner SE</span>, <span class="author">Chervitz SA</span>, <span class="author">Dagdigian C</span>, <span class="author">Fuellen G</span>, <span class="author">Gilbert JG</span>, <span class="author">Korf I</span>, <span class="author">Lapp H</span>, <span class="author">Lehv&auml;slaiho H</span>, <span class="author">Matsalla C</span>, <span class="author">Mungall CJ</span>, <span class="author">Osborne BI</span>, <span class="author">Pocock MR</span>, <span class="author">Schattner P</span>, <span class="author">Senger M</span>, <span class="author">Stein LD</span>, <span class="author">Stupka E</span>, <span class="author">Wilkinson MD</span> and <span class="author">Birney E</span></p>
<p class="affiliation">University Program in Genetics, Duke University, Durham, North Carolina 27710, USA. <a href="mailto:%6A%61%73%6F%6E.%73%74%61%6A%69%63%68%40%64%75%6B%65.%65%64%75">&#106;&#97;&#115;&#111;&#110;&#46;&#115;&#116;&#97;&#106;&#105;&#99;&#104;&#64;&#100;&#117;&#107;&#101;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The Bioperl project is an international open-source collaboration of biologists, bioinformaticians, and computer scientists that has evolved over the past 7 yr into the most comprehensive library of Perl modules available for managing and manipulating life-science information. Bioperl provides an easy-to-use, stable, and consistent programming interface for bioinformatics application programmers. The Bioperl modules have been successfully and repeatedly used to reduce otherwise complex tasks to only a few lines of code. The Bioperl object model has been proven to be flexible enough to support enterprise-level applications such as EnsEMBL, while maintaining an easy learning curve for novice Perl programmers. Bioperl is capable of executing analyses and processing results from programs such as BLAST, ClustalW, or the EMBOSS suite. Interoperation with modules written in Python and Java is supported through the evolving BioCORBA bridge. Bioperl provides access to data stores such as GenBank and SwissProt via a flexible series of sequence input/output modules, and to the emerging common sequence data storage format of the Open Bioinformatics Database Access project. This study describes the overall architecture of the toolkit, the problem domains that it addresses, and gives specific examples of how the toolkit can be used to solve common life-sciences problems. We conclude with a discussion of how the open-source nature of the project has contributed to the development effort.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1 K32 HG00056</span>, <span class="grant">HG00739</span>, <span class="grant">K22 HG-00064-01</span>, <span class="grant">P41HG02223</span>; NIGMS NIH HHS: <span class="grant">T32 GM07754-22</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2002</span>;<span class="volume">12</span>;<span class="number">10</span>;<span class="pagerange">1611-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12368254" rel="external">12368254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC187536" rel="external">187536</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.361602" rel="external">10.1101/gr.361602</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The novel EPTP repeat defines a superfamily of proteins implicated in epileptic disorders.</h4>
<p class="authors"><span class="author">Staub E</span>, <span class="author">P&eacute;rez-Tur J</span>, <span class="author">Siebert R</span>, <span class="author">Nobile C</span>, <span class="author">Moschonas NK</span>, <span class="author">Deloukas P</span> and <span class="author">Hinzmann B</span></p>
<p class="affiliation">metaGen Pharmaceuticals GmbH, Oudenarder Strasse 16, D-13347 Berlin, Germany. <a href="mailto:%65%69%6B%65.%73%74%61%75%62%40%6D%65%74%61%67%65%6E.%64%65">&#101;&#105;&#107;&#101;&#46;&#115;&#116;&#97;&#117;&#98;&#64;&#109;&#101;&#116;&#97;&#103;&#101;&#110;&#46;&#100;&#101;</a></p><p class="abstract">Recent studies suggest that mutations in the LGI1/Epitempin gene cause autosomal dominant lateral temporal epilepsy. This gene encodes a protein of unknown function, which we postulate is secreted. The LGI1 protein has leucine-rich repeats in the N-terminal sequence and a tandem repeat (which we named EPTP) in its C-terminal region. A redefinition of the C-terminal repeat and the application of sensitive sequence analysis methods enabled us to define a new superfamily of proteins carrying varying numbers of the novel EPTP repeats in combination with various extracellular domains. Genes encoding proteins of this family are located in genomic regions associated with epilepsy and other neurological disorders.</p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2002</span>;<span class="volume">27</span>;<span class="number">9</span>;<span class="pagerange">441-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12217514" rel="external">12217514</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical and transcript map of the hereditary prostate cancer region at xq27.</h4>
<p class="authors"><span class="author">Stephan DA</span>, <span class="author">Howell GR</span>, <span class="author">Teslovich TM</span>, <span class="author">Coffey AJ</span>, <span class="author">Smith L</span>, <span class="author">Bailey-Wilson JE</span>, <span class="author">Malechek L</span>, <span class="author">Gildea D</span>, <span class="author">Smith JR</span>, <span class="author">Gillanders EM</span>, <span class="author">Schleutker J</span>, <span class="author">Hu P</span>, <span class="author">Steingruber HE</span>, <span class="author">Dhami P</span>, <span class="author">Robbins CM</span>, <span class="author">Makalowska I</span>, <span class="author">Carpten JD</span>, <span class="author">Sood R</span>, <span class="author">Mumm S</span>, <span class="author">Reinbold R</span>, <span class="author">Bonner TI</span>, <span class="author">Baffoe-Bonnie A</span>, <span class="author">Bubendorf L</span>, <span class="author">Heiskanen M</span>, <span class="author">Kallioneimi OP</span>, <span class="author">Baxevanis AD</span>, <span class="author">Joseph SS</span>, <span class="author">Zucchi I</span>, <span class="author">Burk RD</span>, <span class="author">Isaacs W</span>, <span class="author">Ross MT</span> and <span class="author">Trent JM</span></p>
<p class="affiliation">Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA. <a href="mailto:%64%73%74%65%70%68%61%6E%40%63%6E%6D%63%72%65%73%65%61%72%63%68.%6F%72%67">&#100;&#115;&#116;&#101;&#112;&#104;&#97;&#110;&#64;&#99;&#110;&#109;&#99;&#114;&#101;&#115;&#101;&#97;&#114;&#99;&#104;&#46;&#111;&#114;&#103;</a></p><p class="abstract">We have recently mapped a locus for hereditary prostate cancer (termed HPCX) to the long arm of the X chromosome (Xq25-q27) through a genome-wide linkage study. Here we report the construction of an approximately 9-Mb sequence-ready bacterial clone contig map of Xq26.3-q27.3. The contig was constructed by screening BAC/PAC libraries with markers spaced at approximately 85-kb intervals. We identified overlapping clones by end-sequencing framework clones to generate 407 new sequence-tagged sites, followed by PCR verification of overlaps. Contig assembly was based on clone restriction fingerprinting and the landmark information. We identified a minimal overlap contig for genomic sequencing, which has yielded 7.7 Mb of finished sequence and 1.5 Mb of draft sequence. The transcriptional mapping effort localized 57 known and predicted genes by database searching, STS content mapping, and sequencing, followed by sequence annotation. These transcriptional units represent candidate genes for HPCX and multiple other hereditary diseases at Xq26.3-q27.3.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2002</span>;<span class="volume">79</span>;<span class="number">1</span>;<span class="pagerange">41-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11827456" rel="external">11827456</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2001.6681" rel="external">10.1006/geno.2001.6681</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A targeted X-linked CMV-Cre line.</h4>
<p class="authors"><span class="author">Su H</span>, <span class="author">Mills AA</span>, <span class="author">Wang X</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, USA.</p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2002</span>;<span class="volume">32</span>;<span class="number">2</span>;<span class="pagerange">187-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11857817" rel="external">11857817</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Escherichia coli aconitases and oxidative stress: post-transcriptional regulation of sodA expression.</h4>
<p class="authors"><span class="author">Tang Y</span>, <span class="author">Quail MA</span>, <span class="author">Artymiuk PJ</span>, <span class="author">Guest JR</span> and <span class="author">Green J</span></p>
<p class="affiliation">The Krebs Institute for Biomolecular Research, Department of Molecular Biology and Biotechnology, University of Sheffield, Western Bank, Sheffield S10 2TN, UK.</p><p class="abstract">Escherichia coli possesses two aconitases, a stationary-phase enzyme (AcnA), which is induced by iron and oxidative stress, and a major but less stable enzyme (AcnB), synthesized during exponential growth. In addition to the catalytic activities of the holo-proteins, the apo-proteins function as post-transcriptional regulators by site-specific binding to acn mRNAs. Thus, it has been suggested that inactivation of the enzymes could mediate a rapidly reacting post-transcriptional component of the bacterial oxidative stress response. Here it is shown that E. coli acn mutants are hypersensitive to the redox-stress reagents H(2)O(2) and methyl viologen. Proteomic analyses further revealed that the level of superoxide dismutase (SodA) is enhanced in acnB and acnAB mutants, and by exposure to methyl viologen. The amounts of other proteins, including thioredoxin reductase, 2-oxoglutarate dehydrogenase, succinyl-CoA synthetase and chaperone proteins, were also affected in the acn mutants. The altered patterns of sodA expression were confirmed in studies with sodA-lacZ reporter strains. Quantitative Northern blotting indicated that AcnA enhances the stability of the sodA transcript, whereas AcnB lowers its stability. Direct evidence that the apo-proteins have positive (AcnA) and negative (AcnB) effects on SodA synthesis was obtained from in vitro transcription-translation experiments. It is suggested that the aconitase proteins of E. coli serve as a protective buffer against the basal level of oxidative stress that accompanies aerobic growth by acting as a sink for reactive oxygen species and by modulating translation of the sodA transcript.</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">148</span>;<span class="number">Pt 4</span>;<span class="pagerange">1027-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11932448" rel="external">11932448</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A human TAPBP (TAPASIN)-related gene, TAPBP-R.</h4>
<p class="authors"><span class="author">Teng MS</span>, <span class="author">Stephens R</span>, <span class="author">Du Pasquier L</span>, <span class="author">Freeman T</span>, <span class="author">Lindquist JA</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Immunology Division, Department of Pathology, Cambridge, GB.</p><p class="abstract">TAPASIN, a V-C1 (variable-constant) immunoglobulin superfamily (IgSF) molecule that links MHC class I molecules to the transporter associated with antigen processing (TAP) in the endoplasmic reticulum (ER) is encoded by the TAPBP gene, located near to the MHC at 6p21.3. A related gene was identified at chromosome position 12p13.3 between the CD27 and VAMP1 genes near a group of MHC-paralogous loci. The gene, which we have called TAPBP-R (R for related), also encodes a member of the IgSF, TAPASIN-R. Its putative product contains similar structural motifs to TAPASIN, with some marked differences, especially in the V domain, transmembrane and cytoplasmic regions. By using the mouse ortholog to screen tissue, we revealed that the TAPBP-R gene was broadly expressed. Sub-cellular localization showed that the bulk of TAPASIN-R is located within the ER but biotinylation experiments were consistent with some expression at thecell surface. TAPASIN-R lacks an obvious ER retention signal. The function of TAPASIN-R will be of interest in regards to the evolution of the immune system as well as antigen processing.</p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">2002</span>;<span class="volume">32</span>;<span class="number">4</span>;<span class="pagerange">1059-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11920573" rel="external">11920573</a>; DOI: <a href="http://dx.doi.org/10.1002%2F1521-4141(200204)32%3A4%26%2360%3B1059%3A%3AAID-IMMU1059%26%2362%3B3.0.CO%3B2-G" rel="external">10.1002/1521-4141(200204)32:4&amp;#60;1059::AID-IMMU1059&amp;#62;3.0.CO;2-G</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Requirement for downregulation of kreisler during late patterning of the hindbrain.</h4>
<p class="authors"><span class="author">Theil T</span>, <span class="author">Ariza-McNaughton L</span>, <span class="author">Manzanares M</span>, <span class="author">Brodie J</span>, <span class="author">Krumlauf R</span> and <span class="author">Wilkinson DG</span></p>
<p class="affiliation">Division of Developmental Neurobiology, National Institute for Medical Research, The Ridgeway, London NW7 1AA, UK.</p><p class="abstract">Pattern formation in the hindbrain is governed by a segmentation process that provides the basis for the organisation of cranial motor nerves. A cascade of transcriptional activators, including the bZIP transcription factor encoded by the kreisler gene controls this segmentation process. In kreisler mutants, r5 fails to form and this correlates with abnormalities in the neuroanatomical organisation of the hindbrain. Studies of Hox gene regulation suggest that kreisler may regulate the identity as well as the formation of r5, but such a role cannot be detected in kreisler mutants since r5 is absent. To gain further insights into the function of kreisler we have generated transgenic mice in which kreisler is ectopically expressed in r3 and for an extended period in r5. In these transgenic mice, the Fgf3, Krox20, Hoxa3 and Hoxb3 genes have ectopic or prolonged expression domains in r3, indicating that it acquires molecular characteristics of r5. Prolonged kreisler expression subsequently causes morphological alterations of r3/r5 that are due to an inhibition of neuronal differentiation and migration from the ventricular zone to form the mantle layer. We find that these alterations in r5 correlate with an arrest of facial branchiomotor neurone migration from r4 into the caudal hindbrain, which is possibly due to the deficiency in the mantle layer through which they normally migrate. We propose that the requirement for the downregulation of segmental kreisler expression prior to neuronal differentiation reflects the stage-specific roles of this gene and its targets.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2002</span>;<span class="volume">129</span>;<span class="number">6</span>;<span class="pagerange">1477-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11880356" rel="external">11880356</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated, functionally annotated gene map of the DXS8026-ELK1 interval on human Xp11.3-Xp11.23: potential hotspot for neurogenetic disorders.</h4>
<p class="authors"><span class="author">Thiselton DL</span>, <span class="author">McDowall J</span>, <span class="author">Brandau O</span>, <span class="author">Ramser J</span>, <span class="author">d&#39;Esposito F</span>, <span class="author">Bhattacharya SS</span>, <span class="author">Ross MT</span>, <span class="author">Hardcastle AJ</span> and <span class="author">Meindl A</span></p>
<p class="affiliation">Department of Molecular Genetics, Institute of Ophthalmology, University College, London, EC1V 9EL, UK. <a href="mailto:%64%74%68%69%73%65%6C%74%40%68%67%6D%70.%6D%72%63.%61%63.%75%6B">&#100;&#116;&#104;&#105;&#115;&#101;&#108;&#116;&#64;&#104;&#103;&#109;&#112;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Human chromosome Xp11.3-Xp11.23 encompasses the map location for a growing number of diseases with a genetic basis or genetic component. These include several eye disorders, syndromic and nonsyndromic forms of X-linked mental retardation (XLMR), X-linked neuromuscular diseases and susceptibility loci for schizophrenia, type 1 diabetes, and Graves&#39; disease. We have constructed an approximately 2.7-Mb high-resolution physical map extending from DXS8026 to ELK1, corresponding to a genetic distance of approximately 5.5 cM. A combination of chromosome walking and sequence-tagged site (STS)-content mapping resulted in an integrated framework and transcript map, precisely positioning 10 polymorphic microsatellites (one of which is novel), 16 ESTs, and 12 known genes (RP2, PCTK1, UHX1, UBE1, RBM10, ZNF157, SYN1, ARAF1, TIMP1, PFC, ELK1, UXT). The composite map is currently anchored with 89 STSs to give an average resolution of approximately 1 STS every 30 kb. By a combination of EST database searches and in silico detection of UniGene clusters within genomic sequence generated from this template map, we have mapped several novel genes within this interval: a Na+/H+ exchanger (SLC9A7), at least two zincfinger transcription factors (KIAA0215 and Hs.68318), carbohydrate sulfotransferase-7 (CHST7), regucalcin (RGN), inactivation-escape-1 (INE1), the human ortholog of mouse neuronal protein 15.6, and four putative novel genes. Further genomic analysis enabled annotation of the sequence interval with 20 predicted pseudogenes and 21 UniGene clusters of unknown function. The combined PAC/BAC transcript map and YAC scaffold presented here clarifies previously conflicting data for markers and genes within the Xp11.3-Xp11.23 interval and provides a powerful integrated resource for functional characterization of this clonally unstable, yet gene-rich and clinically significant region of proximal Xp.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2002</span>;<span class="volume">79</span>;<span class="number">4</span>;<span class="pagerange">560-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11944989" rel="external">11944989</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2002.6733" rel="external">10.1006/geno.2002.6733</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluation of linkage of breast cancer to the putative BRCA3 locus on chromosome 13q21 in 128 multiple case families from the Breast Cancer Linkage Consortium.</h4>
<p class="authors"><span class="author">Thompson D</span>, <span class="author">Szabo CI</span>, <span class="author">Mangion J</span>, <span class="author">Oldenburg RA</span>, <span class="author">Odefrey F</span>, <span class="author">Seal S</span>, <span class="author">Barfoot R</span>, <span class="author">Kroeze-Jansema K</span>, <span class="author">Teare D</span>, <span class="author">Rahman N</span>, <span class="author">Renard H</span>, <span class="author">Mann G</span>, <span class="author">Hopper JL</span>, <span class="author">Buys SS</span>, <span class="author">Andrulis IL</span>, <span class="author">Senie R</span>, <span class="author">Daly MB</span>, <span class="author">West D</span>, <span class="author">Ostrander EA</span>, <span class="author">Offit K</span>, <span class="author">Peretz T</span>, <span class="author">Osorio A</span>, <span class="author">Benitez J</span>, <span class="author">Nathanson KL</span>, <span class="author">Sinilnikova OM</span>, <span class="author">Ol&agrave;h E</span>, <span class="author">Bignon YJ</span>, <span class="author">Ruiz P</span>, <span class="author">Badzioch MD</span>, <span class="author">Vasen HF</span>, <span class="author">Futreal AP</span>, <span class="author">Phelan CM</span>, <span class="author">Narod SA</span>, <span class="author">Lynch HT</span>, <span class="author">Ponder BA</span>, <span class="author">Eeles RA</span>, <span class="author">Meijers-Heijboer H</span>, <span class="author">Stoppa-Lyonnet D</span>, <span class="author">Couch FJ</span>, <span class="author">Eccles DM</span>, <span class="author">Evans DG</span>, <span class="author">Chang-Claude J</span>, <span class="author">Lenoir G</span>, <span class="author">Weber BL</span>, <span class="author">Devilee P</span>, <span class="author">Easton DF</span>, <span class="author">Goldgar DE</span>, <span class="author">Stratton MR</span> and <span class="author">KConFab Consortium</span></p>
<p class="affiliation">CRC Genetic Epidemiology Unit, Strangeways Research Laboratories, University of Cambridge, Cambridge CB1 4RN, United Kingdom.</p><p class="abstract">The known susceptibility genes for breast cancer, including BRCA1 and BRCA2, only account for a minority of the familial aggregation of the disease. A recent study of 77 multiple case breast cancer families from Scandinavia found evidence of linkage between the disease and polymorphic markers on chromosome 13q21. We have evaluated the contribution of this candidate &quot;BRCA3&quot; locus to breast cancer susceptibility in 128 high-risk breast cancer families of Western European ancestry with no identified BRCA1 or BRCA2 mutations. No evidence of linkage was found. The estimated proportion (alpha) of families linked to a susceptibility locus at D13S1308, the location estimated by Kainu et al. [(2000) Proc. Natl. Acad. Sci. USA 97, 9603-9608], was 0 (upper 95% confidence limit 0.13). Adjustment for possible bias due to selection of families on the basis of linkage evidence at BRCA2 did not materially alter this result (alpha = 0, upper 95% confidence limit 0.18). The proportion of linked families reported by Kainu et al. (0.65) is excluded with a high degree of confidence in our dataset [heterogeneity logarithm of odds (HLOD) at alpha = 0.65 was -11.0]. We conclude that, if a susceptibility gene does exist at this locus, it can only account for a small proportion of non-BRCA1/2 families with multiple cases of early-onset breast cancer.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA69398</span>, <span class="grant">CA69417</span>, <span class="grant">CA69446</span>, <span class="grant">CA69467</span>, <span class="grant">CA69631</span>, <span class="grant">CA69638</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2002</span>;<span class="volume">99</span>;<span class="number">2</span>;<span class="pagerange">827-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11792833" rel="external">11792833</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC117390" rel="external">117390</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.012584499" rel="external">10.1073/pnas.012584499</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sibling rivalry.</h4>
<p class="authors"><span class="author">Thomson N</span>, <span class="author">Sebaihia M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">9</span>;<span class="pagerange">396-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12217499" rel="external">12217499</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microbial genomics: spot the difference. . .</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Sebaihia M</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2002</span>;<span class="volume">10</span>;<span class="number">11</span>;<span class="pagerange">489-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12419606" rel="external">12419606</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">p53 mutant mice that display early ageing-associated phenotypes.</h4>
<p class="authors"><span class="author">Tyner SD</span>, <span class="author">Venkatachalam S</span>, <span class="author">Choi J</span>, <span class="author">Jones S</span>, <span class="author">Ghebranious N</span>, <span class="author">Igelmann H</span>, <span class="author">Lu X</span>, <span class="author">Soron G</span>, <span class="author">Cooper B</span>, <span class="author">Brayton C</span>, <span class="author">Park SH</span>, <span class="author">Thompson T</span>, <span class="author">Karsenty G</span>, <span class="author">Bradley A</span> and <span class="author">Donehower LA</span></p>
<p class="affiliation">Cell and Molecular Biology Program, Baylor College of Medicine, Houston, TX 77030, USA.</p><p class="abstract">The p53 tumour suppressor is activated by numerous stressors to induce apoptosis, cell cycle arrest, or senescence. To study the biological effects of altered p53 function, we generated mice with a deletion mutation in the first six exons of the p53 gene that express a truncated RNA capable of encoding a carboxy-terminal p53 fragment. This mutation confers phenotypes consistent with activated p53 rather than inactivated p53. Mutant (p53+/m) mice exhibit enhanced resistance to spontaneous tumours compared with wild-type (p53+/+) littermates. As p53+/m mice age, they display an early onset of phenotypes associated with ageing. These include reduced longevity, osteoporosis, generalized organ atrophy and a diminished stress tolerance. A second line of transgenic mice containing a temperature-sensitive mutant allele of p53 also exhibits early ageing phenotypes. These data suggest that p53 has a role in regulating organismal ageing.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">415</span>;<span class="number">6867</span>;<span class="pagerange">45-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11780111">11780111</a>; DOI: <a href="http://dx.doi.org/10.1038%2F415045a">10.1038/415045a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tools for targeted manipulation of the mouse genome.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Adams DJ</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambs CB10 1SA, United Kingdom.</p><p class="abstract">In the postgenomic era the mouse will be central to the challenge of ascribing a function to the 40,000 or so genes that constitute our genome. In this review, we summarize some of the classic and modern approaches that have fueled the recent dramatic explosion in mouse genetics. Together with the sequencing of the mouse genome, these tools will have a profound effect on our ability to generate new and more accurate mouse models and thus provide a powerful insight into the function of human genes during the processes of both normal development and disease.</p>
<p class="publication"><span class="title">Physiological genomics</span>&nbsp;<span class="year">2002</span>;<span class="volume">11</span>;<span class="number">3</span>;<span class="pagerange">133-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12464689" rel="external">12464689</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fphysiolgenomics.00074.2002" rel="external">10.1152/physiolgenomics.00074.2002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer: stuck at first base.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Jonkers J</span> and <span class="author">Bradley A</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">419</span>;<span class="number">6903</span>;<span class="pagerange">127-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12226650">12226650</a>; DOI: <a href="http://dx.doi.org/10.1038%2F419127a">10.1038/419127a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The mosaic structure of variation in the laboratory mouse genome.</h4>
<p class="authors"><span class="author">Wade CM</span>, <span class="author">Kulbokas EJ</span>, <span class="author">Kirby AW</span>, <span class="author">Zody MC</span>, <span class="author">Mullikin JC</span>, <span class="author">Lander ES</span>, <span class="author">Lindblad-Toh K</span> and <span class="author">Daly MJ</span></p>
<p class="affiliation">Whitehead Institute for Biomedical Research and Whitehead/MIT Center for Genome Research, 9 Cambridge Center, Cambridge, Massachusetts 02139, USA.</p><p class="abstract">Most inbred laboratory mouse strains are known to have originated from a mixed but limited founder population in a few laboratories. However, the effect of this breeding history on patterns of genetic variation among these strains and the implications for their use are not well understood. Here we present an analysis of the fine structure of variation in the mouse genome, using single nucleotide polymorphisms (SNPs). When the recently assembled genome sequence from the C57BL/6J strain is aligned with sample sequence from other strains, we observe long segments of either extremely high (approximately 40 SNPs per 10 kb) or extremely low (approximately 0.5 SNPs per 10 kb) polymorphism rates. In all strain-to-strain comparisons examined, only one-third of the genome falls into long regions (averaging &gt;1 Mb) of a high SNP rate, consistent with estimated divergence rates between Mus musculus domesticus and either M. m. musculus or M. m. castaneus. These data suggest that the genomes of these inbred strains are mosaics with the vast majority of segments derived from domesticus and musculus sources. These observations have important implications for the design and interpretation of positional cloning experiments.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">420</span>;<span class="number">6915</span>;<span class="pagerange">574-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12466852">12466852</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01252">10.1038/nature01252</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular mechanisms governing Pcdh-gamma gene expression: evidence for a multiple promoter and cis-alternative splicing model.</h4>
<p class="authors"><span class="author">Wang X</span>, <span class="author">Su H</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas 77030, USA.</p><p class="abstract">The genomic architecture of protocadherin (Pcdh) gene clusters is remarkably similar to that of the immunoglobulin and T cell receptor gene clusters, and can potentially provide significant molecular diversity. Pcdh genes are abundantly expressed in the central nervous system. These molecules are primary candidates for establishing specific neuronal connectivity. Despite the extensive analyses of the genomic structure of both human and mouse Pcdh gene clusters, the definitive molecular mechanisms that control Pcdh gene expression are still unknown. Four theories have been proposed, including (1) DNA recombination followed by cis-splicing, (2) single promoter and cis-alternative splicing, (3) multiple promoters and cis-alternative splicing, and (4) multiple promoters and trans-splicing. Using a combination of molecular and genetic analyses, we evaluated the four models at the Pcdh-gamma locus. Our analysis provides evidence that the transcription of individual Pcdh-gamma genes is under the control of a distinct but related promoter upstream of each Pcdh-gamma variable exon, and posttranscriptional processing of each Pcdh-gamma transcript is predominantly mediated through cis-alternative splicing.</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2002</span>;<span class="volume">16</span>;<span class="number">15</span>;<span class="pagerange">1890-905</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12154121" rel="external">12154121</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC186422" rel="external">186422</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.1004802" rel="external">10.1101/gad.1004802</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gamma protocadherins are required for survival of spinal interneurons.</h4>
<p class="authors"><span class="author">Wang X</span>, <span class="author">Weiner JA</span>, <span class="author">Levi S</span>, <span class="author">Craig AM</span>, <span class="author">Bradley A</span> and <span class="author">Sanes JR</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA.</p><p class="abstract">The murine genome contains approximately 70 protocadherin (Pcdh) genes. Many are expressed in the nervous system, suggesting that Pcdhs may specify neuronal connectivity. Here, we analyze the 22 contiguous genes of the Pcdh-gamma cluster. Individual neurons express subsets of Pcdh-gamma genes. Pcdh-gamma proteins are present in most neurons and associated with, but not confined to, synapses. Early steps in neuronal migration, axon outgrowth, and synapse formation proceed in mutant mice lacking all 22 Pcdh-gamma genes. At late embryonic stages, however, dramatic neurodegeneration leads to neonatal death. In mutant spinal cord, many interneurons are lost, but sensory and motor neurons are relatively spared. In cultures from mutant spinal cord, neurons differentiate and form synapses but then die. Thus, Pcdh-gamma genes are dispensable for at least some aspects of connectivity but required for survival of specific neuronal types.</p>
<p class="publication"><span class="title">Neuron</span>&nbsp;<span class="year">2002</span>;<span class="volume">36</span>;<span class="number">5</span>;<span class="pagerange">843-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12467588" rel="external">12467588</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">On the sequencing of the human genome.</h4>
<p class="authors"><span class="author">Waterston RH</span>, <span class="author">Lander ES</span> and <span class="author">Sulston JE</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University, Saint Louis, MO 63108, USA. <a href="mailto:%62%77%61%74%65%72%73%74%40%77%61%74%73%6F%6E.%77%75%73%74%6C.%65%64%75">&#98;&#119;&#97;&#116;&#101;&#114;&#115;&#116;&#64;&#119;&#97;&#116;&#115;&#111;&#110;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Two recent papers using different approaches reported draft sequences of the human genome. The international Human Genome Project (HGP) used the hierarchical shotgun approach, whereas Celera Genomics adopted the whole-genome shotgun (WGS) approach. Here, we analyze whether the latter paper provides a meaningful test of the WGS approach on a mammalian genome. In the Celera paper, the authors did not analyze their own WGS data. Instead, they decomposed the HGP&#39;s assembled sequence into a &quot;perfect tiling path&quot;, combined it with their WGS data, and assembled the merged data set. To study the implications of this approach, we perform computational analysis and find that a perfect tiling path with 2-fold coverage is sufficient to recover virtually the entirety of a genome assembly. We also examine the manner in which the assembly was anchored to the human genome and conclude that the process primarily depended on the HGP&#39;s sequence-tagged site maps, BAC maps, and clone-based sequences. Our analysis indicates that the Celera paper provides neither a meaningful test of the WGS approach nor an independent sequence of the human genome. Our analysis does not imply that a WGS approach could not be successfully applied to assemble a draft sequence of a large mammalian genome, but merely that the Celera paper does not provide such evidence.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2002</span>;<span class="volume">99</span>;<span class="number">6</span>;<span class="pagerange">3712-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11880605" rel="external">11880605</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC122589" rel="external">122589</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.042692499" rel="external">10.1073/pnas.042692499</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of Schizosaccharomyces pombe.</h4>
<p class="authors"><span class="author">Wood V</span>, <span class="author">Gwilliam R</span>, <span class="author">Rajandream MA</span>, <span class="author">Lyne M</span>, <span class="author">Lyne R</span>, <span class="author">Stewart A</span>, <span class="author">Sgouros J</span>, <span class="author">Peat N</span>, <span class="author">Hayles J</span>, <span class="author">Baker S</span>, <span class="author">Basham D</span>, <span class="author">Bowman S</span>, <span class="author">Brooks K</span>, <span class="author">Brown D</span>, <span class="author">Brown S</span>, <span class="author">Chillingworth T</span>, <span class="author">Churcher C</span>, <span class="author">Collins M</span>, <span class="author">Connor R</span>, <span class="author">Cronin A</span>, <span class="author">Davis P</span>, <span class="author">Feltwell T</span>, <span class="author">Fraser A</span>, <span class="author">Gentles S</span>, <span class="author">Goble A</span>, <span class="author">Hamlin N</span>, <span class="author">Harris D</span>, <span class="author">Hidalgo J</span>, <span class="author">Hodgson G</span>, <span class="author">Holroyd S</span>, <span class="author">Hornsby T</span>, <span class="author">Howarth S</span>, <span class="author">Huckle EJ</span>, <span class="author">Hunt S</span>, <span class="author">Jagels K</span>, <span class="author">James K</span>, <span class="author">Jones L</span>, <span class="author">Jones M</span>, <span class="author">Leather S</span>, <span class="author">McDonald S</span>, <span class="author">McLean J</span>, <span class="author">Mooney P</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Murphy L</span>, <span class="author">Niblett D</span>, <span class="author">Odell C</span>, <span class="author">Oliver K</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Pearson D</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rutherford K</span>, <span class="author">Rutter S</span>, <span class="author">Saunders D</span>, <span class="author">Seeger K</span>, <span class="author">Sharp S</span>, <span class="author">Skelton J</span>, <span class="author">Simmonds M</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Stevens K</span>, <span class="author">Taylor K</span>, <span class="author">Taylor RG</span>, <span class="author">Tivey A</span>, <span class="author">Walsh S</span>, <span class="author">Warren T</span>, <span class="author">Whitehead S</span>, <span class="author">Woodward J</span>, <span class="author">Volckaert G</span>, <span class="author">Aert R</span>, <span class="author">Robben J</span>, <span class="author">Grymonprez B</span>, <span class="author">Weltjens I</span>, <span class="author">Vanstreels E</span>, <span class="author">Rieger M</span>, <span class="author">Sch&auml;fer M</span>, <span class="author">M&uuml;ller-Auer S</span>, <span class="author">Gabel C</span>, <span class="author">Fuchs M</span>, <span class="author">D&uuml;sterh&ouml;ft A</span>, <span class="author">Fritzc C</span>, <span class="author">Holzer E</span>, <span class="author">Moestl D</span>, <span class="author">Hilbert H</span>, <span class="author">Borzym K</span>, <span class="author">Langer I</span>, <span class="author">Beck A</span>, <span class="author">Lehrach H</span>, <span class="author">Reinhardt R</span>, <span class="author">Pohl TM</span>, <span class="author">Eger P</span>, <span class="author">Zimmermann W</span>, <span class="author">Wedler H</span>, <span class="author">Wambutt R</span>, <span class="author">Purnelle B</span>, <span class="author">Goffeau A</span>, <span class="author">Cadieu E</span>, <span class="author">Dr&eacute;ano S</span>, <span class="author">Gloux S</span>, <span class="author">Lelaure V</span>, <span class="author">Mottier S</span>, <span class="author">Galibert F</span>, <span class="author">Aves SJ</span>, <span class="author">Xiang Z</span>, <span class="author">Hunt C</span>, <span class="author">Moore K</span>, <span class="author">Hurst SM</span>, <span class="author">Lucas M</span>, <span class="author">Rochet M</span>, <span class="author">Gaillardin C</span>, <span class="author">Tallada VA</span>, <span class="author">Garzon A</span>, <span class="author">Thode G</span>, <span class="author">Daga RR</span>, <span class="author">Cruzado L</span>, <span class="author">Jimenez J</span>, <span class="author">S&aacute;nchez M</span>, <span class="author">del Rey F</span>, <span class="author">Benito J</span>, <span class="author">Dom&iacute;nguez A</span>, <span class="author">Revuelta JL</span>, <span class="author">Moreno S</span>, <span class="author">Armstrong J</span>, <span class="author">Forsburg SL</span>, <span class="author">Cerutti L</span>, <span class="author">Lowe T</span>, <span class="author">McCombie WR</span>, <span class="author">Paulsen I</span>, <span class="author">Potashkin J</span>, <span class="author">Shpakovski GV</span>, <span class="author">Ussery D</span>, <span class="author">Barrell BG</span>, <span class="author">Nurse P</span> and <span class="author">Cerrutti L</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We have sequenced and annotated the genome of fission yeast (Schizosaccharomyces pombe), which contains the smallest number of protein-coding genes yet recorded for a eukaryote: 4,824. The centromeres are between 35 and 110 kilobases (kb) and contain related repeats including a highly conserved 1.8-kb element. Regions upstream of genes are longer than in budding yeast (Saccharomyces cerevisiae), possibly reflecting more-extended control regions. Some 43% of the genes contain introns, of which there are 4,730. Fifty genes have significant similarity with human disease genes; half of these are cancer related. We identify highly conserved genes important for eukaryotic cell organization including those required for the cytoskeleton, compartmentation, cell-cycle control, proteolysis, protein phosphorylation and RNA splicing. These genes may have originated with the appearance of eukaryotic life. Few similarly conserved genes that are important for multicellular organization were identified, suggesting that the transition from prokaryotes to eukaryotes required more new genes than did the transition from unicellular to multicellular organization.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2002</span>;<span class="volume">415</span>;<span class="number">6874</span>;<span class="pagerange">871-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11859360">11859360</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature724">10.1038/nature724</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The PASTA domain: a beta-lactam-binding domain.</h4>
<p class="authors"><span class="author">Yeats C</span>, <span class="author">Finn RD</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK CB10 1SA. <a href="mailto:%63%61%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The PASTA domain (for penicillin-binding protein and serine/threonine kinase associated domain) is found in the high molecular weight penicillin-binding proteins and eukaryotic-like serine/threonine kinases of a range of pathogens. We describe this previously uncharacterized domain and infer that it binds beta-lactam antibiotics and their peptidoglycan analogues. We postulate that PknB-like kinases are key regulators of cell-wall biosynthesis. The essential function of these enzymes suggests an additional pathway for the action of beta-lactam antibiotics.</p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2002</span>;<span class="volume">27</span>;<span class="number">9</span>;<span class="pagerange">438</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12217513">12217513</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Similarity of the phenotypic patterns associated with BRAF and KRAS mutations in colorectal neoplasia.</h4>
<p class="authors"><span class="author">Yuen ST</span>, <span class="author">Davies H</span>, <span class="author">Chan TL</span>, <span class="author">Ho JW</span>, <span class="author">Bignell GR</span>, <span class="author">Cox C</span>, <span class="author">Stephens P</span>, <span class="author">Edkins S</span>, <span class="author">Tsui WW</span>, <span class="author">Chan AS</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Wooster R</span> and <span class="author">Leung SY</span></p>
<p class="affiliation">Department of Pathology, University of Hong Kong, Queen Mary Hospital, Hong Kong.</p><p class="abstract">Activation of the RAS/RAF/extracellular signal-regulated kinase-mitogen-activated protein kinase/extracellular signal-regulated kinase/mitogen-activated protein kinase pathway by RAS mutations is commonly found in human cancers. Recently, we reported that mutation of BRAF provides an alternative route for activation of this signaling pathway and can be found in melanomas, colorectal cancers, and ovarian tumors. Here we perform an extensive characterization of BRAF mutations in a large series of colorectal tumors in various stages of neoplastic transformation. BRAF mutations were found in 11 of 215 (5.1%) colorectal adenocarcinomas, 3 of 108 (2.8%) sporadic adenomas, 1 of 63 (1.6%) adenomas from familial adenomatous polyposis (FAP) patients, and 1 of 3 (33%) hyperplastic polyps. KRAS mutations were detected in 34% of carcinomas, 31% of sporadic adenomas, 9% of FAP adenomas, and no hyperplastic polyps. Eight of 16 BRAF mutations were V599E, the previously described hotspot, and none of these was associated with a KRAS mutation in the same lesion. The remaining eight mutations involve other conserved amino acids in the kinase domain, and 62.5% have a KRAS mutation in the same tumor. Our data suggest that BRAF mutations are, to some extent, biologically similar to RAS mutations in colorectal cancer because both occur at approximately the same stage of the adenoma-carcinoma sequence, both are associated with villous morphology, and both are less common in adenomas from FAP cases. By contrast, colorectal adenocarcinomas with BRAF mutations are associated with early Dukes&#39; tumor stages (P = 0.006) and no such relationship was observed for KRAS mutations. The presence in some colorectal neoplasms of mutations in both BRAF and KRAS suggests that modulation of the RAS-RAF-extracellular signal-regulated kinase-mitogen-activated protein kinase/extracellular signal-regulated kinase/mitogen-activated protein kinase signaling pathway may occur by mutation of multiple components.</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2002</span>;<span class="volume">62</span>;<span class="number">22</span>;<span class="pagerange">6451-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12438234" rel="external">12438234</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Visual genotyping of a coat color tagged p53 mutant mouse line.</h4>
<p class="authors"><span class="author">Zheng B</span>, <span class="author">Vogel H</span>, <span class="author">Donehower LA</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas 77030, USA.</p><p class="abstract">The p53 knockout mouse has been widely used as a model in cancer research and other applications. Because neither homozygous nor heterozygous mutant p53 mice exhibit an overt phenotype, each animal requires laborious molecular genotyping. Here we describe a new p53 mutant mouse that is tagged with a tyrosinase coat color minigene. On an albino background, heterozygous tyrosinase-tagged p53 mutant mice exhibit a light tan coat color, while homozygous mutants display a darker brown coat color. Thus, by 8-10 days of age, mice with two, one, or no mutant p53 alleles are immediately distinguishable by their coat color, eliminating the time, costs, and errors associated with molecular genotyping. Moreover, the homozygous mutant p53-tyrosinase mice display a tumor incidence and spectrum virtually identical to previous p53 null mouse lines. Thus, tagging targeted mutations with such coat color markers provides a generally applicable genotyping method for embryonic stem cell-derived mice.</p>
<p class="publication"><span class="title">Cancer biology &amp; therapy</span>&nbsp;<span class="year">2002</span>;<span class="volume">1</span>;<span class="number">4</span>;<span class="pagerange">433-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12432262" rel="external">12432262</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/q8icgcnu"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/q8icgcnu" src="/qr/q8icgcnu.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:47 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
