<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2011 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2011.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2011</h1><p>Number of papers published in 2011: <strong>413</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics in 2011: challenges and opportunities.</h4>
<p class="authors"><span class="author">Adams DJ</span>, <span class="author">Berger B</span>, <span class="author">Harismendy O</span>, <span class="author">Huttenhower C</span>, <span class="author">Liu XS</span>, <span class="author">Myers CL</span>, <span class="author">Oshlack A</span>, <span class="author">Rinn JL</span> and <span class="author">Walhout AJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute.</p>
<p class="abstract">As we come to the end of 2011, Genome Biology has asked some members of our Editorial Board for their views on the state of play in genomics. What was their favorite paper of 2011? What are the challenges in their particular research area? Who has had the biggest influence on their careers? What advice would they give to young researchers embarking on a career in research?</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">12</span>;<span class="pagerange">137</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22204305" rel="external">22204305</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3334607" rel="external">3334607</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-12-137" rel="external">10.1186/gb-2011-12-12-137</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The future of model organisms in human disease research.</h4>
<p class="authors"><span class="author">Aitman TJ</span>, <span class="author">Boone C</span>, <span class="author">Churchill GA</span>, <span class="author">Hengartner MO</span>, <span class="author">Mackay TF</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Medical Research Council Clinical Sciences Centre, Imperial College, London, UK. <a href="mailto:%74.%61%69%74%6D%61%6E%40%63%73%63.%6D%72%63.%61%63.%75%6B">&#116;&#46;&#97;&#105;&#116;&#109;&#97;&#110;&#64;&#99;&#115;&#99;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Model organisms have played a huge part in the history of studies of human genetic disease, both in identifying disease genes and characterizing their normal and abnormal functions. But is the importance of model organisms diminishing? The direct discovery of disease genes and variants in humans has been revolutionized, first by genome-wide association studies and now by whole-genome sequencing. Not only is it now much easier to directly identify potential disease genes in humans, but the genetic architecture that is being revealed in many cases is hard to replicate in model organisms. Furthermore, disease modelling can be done with increasing effectiveness using human cells. Where does this leave non-human models of disease?</p>
<p class="grants">Funded by: Medical Research Council; NIGMS NIH HHS: <span class="grant">GM076468</span>, <span class="grant">GM45146</span>, <span class="grant">R01 GM045146-20</span>, <span class="grant">R01 GM070683</span></p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">8</span>;<span class="pagerange">575-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21765459" rel="external">21765459</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg3047" rel="external">10.1038/nrg3047</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing identifies NBEAL2 as the causative gene for gray platelet syndrome.</h4>
<p class="authors"><span class="author">Albers CA</span>, <span class="author">Cvejic A</span>, <span class="author">Favier R</span>, <span class="author">Bouwmans EE</span>, <span class="author">Alessi MC</span>, <span class="author">Bertone P</span>, <span class="author">Jordan G</span>, <span class="author">Kettleborough RN</span>, <span class="author">Kiddle G</span>, <span class="author">Kostadima M</span>, <span class="author">Read RJ</span>, <span class="author">Sipos B</span>, <span class="author">Sivapalaratnam S</span>, <span class="author">Smethurst PA</span>, <span class="author">Stephens J</span>, <span class="author">Voss K</span>, <span class="author">Nurden A</span>, <span class="author">Rendon A</span>, <span class="author">Nurden P</span> and <span class="author">Ouwehand WH</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%63%61%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Gray platelet syndrome (GPS) is a predominantly recessive platelet disorder that is characterized by mild thrombocytopenia with large platelets and a paucity of &alpha;-granules; these abnormalities cause mostly moderate but in rare cases severe bleeding. We sequenced the exomes of four unrelated individuals and identified NBEAL2 as the causative gene; it has no previously known function but is a member of a gene family that is involved in granule development. Silencing of nbeal2 in zebrafish abrogated thrombocyte formation.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Medical Research Council: <span class="grant">MC_U105260799</span>; Wellcome Trust: <span class="grant">082597</span>, <span class="grant">082961</span>, <span class="grant">084183</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">8</span>;<span class="pagerange">735-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21765411">21765411</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428934">3428934</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.885">10.1038/ng.885</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dindel: accurate indel calls from short-read data.</h4>
<p class="authors"><span class="author">Albers CA</span>, <span class="author">Lunter G</span>, <span class="author">MacArthur DG</span>, <span class="author">McVean G</span>, <span class="author">Ouwehand WH</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1HH, United Kingdom. <a href="mailto:%63%61%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Small insertions and deletions (indels) are a common and functionally important type of sequence polymorphism. Most of the focus of studies of sequence variation is on single nucleotide variants (SNVs) and large structural variants. In principle, high-throughput sequencing studies should allow identification of indels just as SNVs. However, inference of indels from next-generation sequence data is challenging, and so far methods for identifying indels lag behind methods for calling SNVs in terms of sensitivity and specificity. We propose a Bayesian method to call indels from short-read sequence data in individuals and populations by realigning reads to candidate haplotypes that represent alternative sequence to the reference. The candidate haplotypes are formed by combining candidate indels and SNVs identified by the read mapper, while allowing for known sequence variants or candidates from other methods to be included. In our probabilistic realignment model we account for base-calling errors, mapping errors, and also, importantly, for increased sequencing error indel rates in long homopolymer runs. We show that our method is sensitive and achieves low false discovery rates on simulated and real data sets, although challenges remain. The algorithm is implemented in the program Dindel, which has been used in the 1000 Genomes Project call sets.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Wellcome Trust: <span class="grant">086084</span>, <span class="grant">090532</span>, <span class="grant">WT089088/Z/09/Z</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">6</span>;<span class="pagerange">961-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20980555">20980555</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3106329">3106329</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.112326.110">10.1101/gr.112326.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcription factor Bcl11b controls selection of invariant natural killer T-cells by regulating glycolipid presentation in double-positive thymocytes.</h4>
<p class="authors"><span class="author">Albu DI</span>, <span class="author">VanValkenburgh J</span>, <span class="author">Morin N</span>, <span class="author">Califano D</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Liu P</span> and <span class="author">Avram D</span></p>
<p class="affiliation">Center for Cell Biology and Cancer Research, Albany Medical College, Albany, NY 12208, USA.</p><p class="abstract">Invariant natural killer T cells (iNKT cells) are innate-like T cells important in immune regulation, antimicrobial protection, and anti-tumor responses. They express semi-invariant T cell receptors, which recognize glycolipid antigens. Their positive selection is mediated by double-positive (DP) thymocytes, which present glycolipid self-antigens through the noncanonical MHC class I-like molecule CD1d. Here we provide genetic and biochemical evidence that removal of the transcription factor Bcl11b in DP thymocytes leads to an early block in iNKT cell development, caused by both iNKT cell extrinsic and intrinsic defects. Specifically, Bcl11b-deficient DP thymocytes failed to support Bcl11b-sufficient iNKT precursor development due to defective glycolipid self-antigen presentation, and showed enlarged lysosomes and accumulation of glycosphingolipids. Expression of genes encoding lysosomal proteins with roles in sphingolipid metabolism and glycolipid presentation was found to be altered in Bcl11b-deficient DP thymocytes. These include cathepsins and Niemann-Pick disease type A, B, and C genes. Thus, Bcl11b plays a central role in presentation of glycolipid self-antigens by DP thymocytes, and regulates directly or indirectly expression of lysosomal genes, exerting a critical extrinsic role in development of iNKT lineage, in addition to the intrinsic role in iNKT precursors. These studies demonstrate a unique and previously undescribed role of Bcl11b in DP thymocytes, in addition to the critical function in positive selection of conventional CD4 and CD8 single-positive thymocytes.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI067846</span>, <span class="grant">AI078273</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">15</span>;<span class="pagerange">6211-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21444811" rel="external">21444811</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3076841" rel="external">3076841</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1014304108" rel="external">10.1073/pnas.1014304108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the green anole lizard and a comparative analysis with birds and mammals.</h4>
<p class="authors"><span class="author">Alf&ouml;ldi J</span>, <span class="author">Di Palma F</span>, <span class="author">Grabherr M</span>, <span class="author">Williams C</span>, <span class="author">Kong L</span>, <span class="author">Mauceli E</span>, <span class="author">Russell P</span>, <span class="author">Lowe CB</span>, <span class="author">Glor RE</span>, <span class="author">Jaffe JD</span>, <span class="author">Ray DA</span>, <span class="author">Boissinot S</span>, <span class="author">Shedlock AM</span>, <span class="author">Botka C</span>, <span class="author">Castoe TA</span>, <span class="author">Colbourne JK</span>, <span class="author">Fujita MK</span>, <span class="author">Moreno RG</span>, <span class="author">ten Hallers BF</span>, <span class="author">Haussler D</span>, <span class="author">Heger A</span>, <span class="author">Heiman D</span>, <span class="author">Janes DE</span>, <span class="author">Johnson J</span>, <span class="author">de Jong PJ</span>, <span class="author">Koriabine MY</span>, <span class="author">Lara M</span>, <span class="author">Novick PA</span>, <span class="author">Organ CL</span>, <span class="author">Peach SE</span>, <span class="author">Poe S</span>, <span class="author">Pollock DD</span>, <span class="author">de Queiroz K</span>, <span class="author">Sanger T</span>, <span class="author">Searle S</span>, <span class="author">Smith JD</span>, <span class="author">Smith Z</span>, <span class="author">Swofford R</span>, <span class="author">Turner-Maier J</span>, <span class="author">Wade J</span>, <span class="author">Young S</span>, <span class="author">Zadissa A</span>, <span class="author">Edwards SV</span>, <span class="author">Glenn TC</span>, <span class="author">Schneider CJ</span>, <span class="author">Losos JB</span>, <span class="author">Lander ES</span>, <span class="author">Breen M</span>, <span class="author">Ponting CP</span> and <span class="author">Lindblad-Toh K</span></p>
<p class="affiliation">Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA. <a href="mailto:%6A%61%6C%66%6F%6C%64%69%40%62%72%6F%61%64%69%6E%73%74%69%74%75%74%65.%6F%72%67">&#106;&#97;&#108;&#102;&#111;&#108;&#100;&#105;&#64;&#98;&#114;&#111;&#97;&#100;&#105;&#110;&#115;&#116;&#105;&#116;&#117;&#116;&#101;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The evolution of the amniotic egg was one of the great evolutionary innovations in the history of life, freeing vertebrates from an obligatory connection to water and thus permitting the conquest of terrestrial environments. Among amniotes, genome sequences are available for mammals and birds, but not for non-avian reptiles. Here we report the genome sequence of the North American green anole lizard, Anolis carolinensis. We find that A. carolinensis microchromosomes are highly syntenic with chicken microchromosomes, yet do not exhibit the high GC and low repeat content that are characteristic of avian microchromosomes. Also, A. carolinensis mobile elements are very young and diverse-more so than in any other sequenced amniote genome. The GC content of this lizard genome is also unusual in its homogeneity, unlike the regionally variable GC content found in mammals and birds. We describe and assign sequence to the previously unknown A. carolinensis X chromosome. Comparative gene analysis shows that amniote egg proteins have evolved significantly more rapidly than other proteins. An anole phylogeny resolves basal branches to illuminate the history of their repeated adaptive radiations.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG003067-08</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7366</span>;<span class="pagerange">587-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21881562" rel="external">21881562</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3184186" rel="external">3184186</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10390" rel="external">10.1038/nature10390</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput phenotyping using parallel sequencing of RNA interference targets in the African trypanosome.</h4>
<p class="authors"><span class="author">Alsford S</span>, <span class="author">Turner DJ</span>, <span class="author">Obado SO</span>, <span class="author">Sanchez-Flores A</span>, <span class="author">Glover L</span>, <span class="author">Berriman M</span>, <span class="author">Hertz-Fowler C</span> and <span class="author">Horn D</span></p>
<p class="affiliation">London School of Hygiene &amp; Tropical Medicine, London WC1E 7HT, United Kingdom.</p><p class="abstract">African trypanosomes are major pathogens of humans and livestock and represent a model for studies of unusual protozoal biology. We describe a high-throughput phenotyping approach termed RNA interference (RNAi) target sequencing, or RIT-seq that, using Illumina sequencing, maps fitness-costs associated with RNAi. We scored the abundance of &gt;90,000 integrated RNAi targets recovered from trypanosome libraries before and after induction of RNAi. Data are presented for 7435 protein coding sequences, &gt;99% of a non-redundant set in the Trypanosoma brucei genome. Analysis of bloodstream and insect life-cycle stages and differentiated libraries revealed genome-scale knockdown profiles of growth and development, linking thousands of previously uncharacterized and &quot;hypothetical&quot; genes to essential functions. Genes underlying prominent features of trypanosome biology are highlighted, including the constitutive emphasis on post-transcriptional gene expression control, the importance of flagellar motility and glycolysis in the bloodstream, and of carboxylic acid metabolism and phosphorylation during differentiation from the bloodstream to the insect stage. The current data set also provides much needed genetic validation to identify new drug targets. RIT-seq represents a versatile new tool for genome-scale functional analyses and for the exploitation of genome sequence data.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span>, <span class="grant">083648</span>, <span class="grant">085775/Z/08/Z</span>, <span class="grant">089172</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">6</span>;<span class="pagerange">915-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21363968" rel="external">21363968</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3106324" rel="external">3106324</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.115089.110" rel="external">10.1101/gr.115089.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">IDH1 and IDH2 mutations are frequent events in central chondrosarcoma and central and periosteal chondromas but not in other mesenchymal tumours.</h4>
<p class="authors"><span class="author">Amary MF</span>, <span class="author">Bacsi K</span>, <span class="author">Maggiani F</span>, <span class="author">Damato S</span>, <span class="author">Halai D</span>, <span class="author">Berisha F</span>, <span class="author">Pollock R</span>, <span class="author">O&#39;Donnell P</span>, <span class="author">Grigoriadis A</span>, <span class="author">Diss T</span>, <span class="author">Eskandarpour M</span>, <span class="author">Presneau N</span>, <span class="author">Hogendoorn PC</span>, <span class="author">Futreal A</span>, <span class="author">Tirabosco R</span> and <span class="author">Flanagan AM</span></p>
<p class="affiliation">Department of Histopathology, Royal National Orthopaedic Hospital NHS Trust, Stanmore, Middlesex HA7 4LP, UK.</p><p class="abstract">Somatic mutations in isocitrate dehydrogenase 1 (IDH1) and IDH2 occur in gliomas and acute myeloid leukaemia (AML). Since patients with multiple enchondromas have occasionally been reported to have these conditions, we hypothesized that the same mutations would occur in cartilaginous neoplasms. Approximately 1200 mesenchymal tumours, including 220 cartilaginous tumours, 222 osteosarcomas and another &sim;750 bone and soft tissue tumours, were screened for IDH1 R132 mutations, using Sequenom(&reg;) mass spectrometry. Cartilaginous tumours and chondroblastic osteosarcomas, wild-type for IDH1 R132, were analysed for IDH2 (R172, R140) mutations. Validation was performed by capillary sequencing and restriction enzyme digestion. Heterozygous somatic IDH1/IDH2 mutations, which result in the production of a potential oncometabolite, 2-hydroxyglutarate, were only detected in central and periosteal cartilaginous tumours, and were found in at least 56% of these, &sim;40% of which were represented by R132C. IDH1 R132H mutations were confirmed by immunoreactivity for this mutant allele. The ratio of IDH1:IDH2 mutation was 10.6 : 1. No IDH2 R140 mutations were detected. Mutations were detected in enchondromas through to conventional central and dedifferentiated chondrosarcomas, in patients with both solitary and multiple neoplasms. No germline mutations were detected. No mutations were detected in peripheral chondrosarcomas and osteochondromas. In conclusion, IDH1 and IDH2 mutations represent the first common genetic abnormalities to be identified in conventional central and periosteal cartilaginous tumours. As in gliomas and AML, the mutations appear to occur early in tumourigenesis. We speculate that a mosaic pattern of IDH-mutation-bearing cells explains the reports of diverse tumours (gliomas, AML, multiple cartilaginous neoplasms, haemangiomas) occurring in the same patient.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077012</span></p>
<p class="publication"><span class="title">The Journal of pathology</span>&nbsp;<span class="year">2011</span>;<span class="volume">224</span>;<span class="number">3</span>;<span class="pagerange">334-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21598255">21598255</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpath.2913">10.1002/path.2913</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ollier disease and Maffucci syndrome are caused by somatic mosaic mutations of IDH1 and IDH2.</h4>
<p class="authors"><span class="author">Amary MF</span>, <span class="author">Damato S</span>, <span class="author">Halai D</span>, <span class="author">Eskandarpour M</span>, <span class="author">Berisha F</span>, <span class="author">Bonar F</span>, <span class="author">McCarthy S</span>, <span class="author">Fantin VR</span>, <span class="author">Straley KS</span>, <span class="author">Lobo S</span>, <span class="author">Aston W</span>, <span class="author">Green CL</span>, <span class="author">Gale RE</span>, <span class="author">Tirabosco R</span>, <span class="author">Futreal A</span>, <span class="author">Campbell P</span>, <span class="author">Presneau N</span> and <span class="author">Flanagan AM</span></p>
<p class="affiliation">Histopathology Unit, Royal National Orthopaedic Hospital National Health Service Trust, Stanmore, UK. <a href="mailto:%61.%66%6C%61%6E%61%67%61%6E%40%75%63%6C.%61%63.%75%6B">&#97;&#46;&#102;&#108;&#97;&#110;&#97;&#103;&#97;&#110;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ollier disease and Maffucci syndrome are characterized by multiple central cartilaginous tumors that are accompanied by soft tissue hemangiomas in Maffucci syndrome. We show that in 37 of 40 individuals with these syndromes, at least one tumor has a mutation in isocitrate dehydrogenase 1 (IDH1) or in IDH2, 65% of which result in a R132C substitution in the protein. In 18 of 19 individuals with more than one tumor analyzed, all tumors from a given individual shared the same IDH1 mutation affecting Arg132. In 2 of 12 subjects, a low level of mutated DNA was identified in non-neoplastic tissue. The levels of the metabolite 2HG were measured in a series of central cartilaginous and vascular tumors, including samples from syndromic and nonsyndromic subjects, and these levels correlated strongly with the presence of IDH1 mutations. The findings are compatible with a model in which IDH1 or IDH2 mutations represent early post-zygotic occurrences in individuals with these syndromes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077012</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">12</span>;<span class="pagerange">1262-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22057236">22057236</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.994">10.1038/ng.994</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Synthetic associations are unlikely to account for many common disease genome-wide association signals.</h4>
<p class="authors"><span class="author">Anderson CA</span>, <span class="author">Soranzo N</span>, <span class="author">Zeggini E</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB101HH, United Kingdom. <a href="mailto:%73%74%61%74%67%65%6E-%66%61%63%75%6C%74%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#116;&#97;&#116;&#103;&#101;&#110;&#45;&#102;&#97;&#99;&#117;&#108;&#116;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT089120/Z/09/Z</span>, <span class="grant">WT091745/Z/10/Z</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">e1000580</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21267062">21267062</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3022527">3022527</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1000580">10.1371/journal.pbio.1000580</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Towards an understanding of genetic predisposition to migraine.</h4>
<p class="authors"><span class="author">Anttila V</span>, <span class="author">Wessman M</span>, <span class="author">Kallela M</span> and <span class="author">Palotie A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, CB10 1SA, Hinxton, UK. <a href="mailto:%61%61%72%6E%6F.%70%61%6C%6F%74%69%65%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#97;&#114;&#110;&#111;&#46;&#112;&#97;&#108;&#111;&#116;&#105;&#101;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">Plausible genome-wide associations for episodic neurological diseases (such as migraine, epilepsy and ataxias) have been slow to emerge. The first such association was reported in a recent genome-wide association study of migraine, with quantitative expression analysis linking the variant to a nearby regulatory gene, MTDH/AEG-1. This putative mechanism, regulating the expression of the primary glutamate transporter in the brain, EAAT2/GLT-1, has interesting implications bridging the gap between Mendelian and common forms in this key group of disorders.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21457514" rel="external">21457514</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3092102" rel="external">3092102</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm231" rel="external">10.1186/gm231</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative whole genome sequence analysis of the carcinogenic bacterial model pathogen Helicobacter felis.</h4>
<p class="authors"><span class="author">Arnold IC</span>, <span class="author">Zigova Z</span>, <span class="author">Holden M</span>, <span class="author">Lawley TD</span>, <span class="author">Rad R</span>, <span class="author">Dougan G</span>, <span class="author">Falkow S</span>, <span class="author">Bentley SD</span> and <span class="author">M&uuml;ller A</span></p>
<p class="affiliation">Institute of Molecular Cancer Research, University of Z&uuml;rich, Switzerland.</p><p class="abstract">The gram-negative bacterium Helicobacter felis naturally colonizes the gastric mucosa of dogs and cats. Due to its ability to persistently infect laboratory mice, H. felis has been used extensively to experimentally model gastric disorders induced in humans by H. pylori. We determined the 1.67 Mb genome sequence of H. felis using combined Solexa and 454 pyrosequencing, annotated the genome, and compared it with multiple previously published Helicobacter genomes. About 1,063 (63.6%) of the 1,671 genes identified in the H. felis genome have orthologues in H. pylori, its closest relative among the fully sequenced Helicobacter species. Many H. pylori virulence factors are shared by H. felis: these include the gamma-glutamyl transpeptidase GGT, the immunomodulator NapA, and the secreted enzymes collagenase and HtrA. Helicobacter felis lacks a Cag pathogenicity island and the vacuolating cytotoxin VacA but possesses a complete comB system conferring natural competence. Remarkable features of the H. felis genome include its paucity of transcriptional regulators and an extraordinary abundance of chemotaxis sensors and restriction/modification systems. Helicobacter felis possesses an episomally replicating 6.7-kb plasmid and harbors three chromosomal regions with deviating GC content. These putative horizontally acquired regions show homology and synteny with the recently isolated H. pylori plasmid pHPPC4 and homology to Campylobacter bacteriophage genes (transposases, structural, and lytic genes), respectively. In summary, the H. felis genome harbors a variety of putative mobile elements that are unique among Helicobacter species and may contribute to this pathogen&#39;s carcinogenic properties.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span>, <span class="grant">076964</span></p>
<p class="publication"><span class="title">Genome biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">3</span>;<span class="pagerange">302-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21402865">21402865</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4197744">4197744</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fgbe%2Fevr022">10.1093/gbe/evr022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence-based analysis uncovers an abundance of non-coding RNA in the total transcriptome of Mycobacterium tuberculosis.</h4>
<p class="authors"><span class="author">Arnvig KB</span>, <span class="author">Comas I</span>, <span class="author">Thomson NR</span>, <span class="author">Houghton J</span>, <span class="author">Boshoff HI</span>, <span class="author">Croucher NJ</span>, <span class="author">Rose G</span>, <span class="author">Perkins TT</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">Young DB</span></p>
<p class="affiliation">Division of Mycobacterial Research, MRC National Institute for Medical Research, London, United Kingdom. <a href="mailto:%6B%61%72%6E%76%69%67%40%6E%69%6D%72.%6D%72%63.%61%63.%75%6B">&#107;&#97;&#114;&#110;&#118;&#105;&#103;&#64;&#110;&#105;&#109;&#114;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">RNA sequencing provides a new perspective on the genome of Mycobacterium tuberculosis by revealing an extensive presence of non-coding RNA, including long 5&#39; and 3&#39; untranslated regions, antisense transcripts, and intergenic small RNA (sRNA) molecules. More than a quarter of all sequence reads mapping outside of ribosomal RNA genes represent non-coding RNA, and the density of reads mapping to intergenic regions was more than two-fold higher than that mapping to annotated coding sequences. Selected sRNAs were found at increased abundance in stationary phase cultures and accumulated to remarkably high levels in the lungs of chronically infected mice, indicating a potential contribution to pathogenesis. The ability of tubercle bacilli to adapt to changing environments within the host is critical to their ability to cause disease and to persist during drug treatment; it is likely that novel post-transcriptional regulatory networks will play an important role in these adaptive responses.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">U117581288</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">11</span>;<span class="pagerange">e1002342</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22072964" rel="external">22072964</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3207917" rel="external">3207917</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002342" rel="external">10.1371/journal.ppat.1002342</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enterotypes of the human gut microbiome.</h4>
<p class="authors"><span class="author">Arumugam M</span>, <span class="author">Raes J</span>, <span class="author">Pelletier E</span>, <span class="author">Le Paslier D</span>, <span class="author">Yamada T</span>, <span class="author">Mende DR</span>, <span class="author">Fernandes GR</span>, <span class="author">Tap J</span>, <span class="author">Bruls T</span>, <span class="author">Batto JM</span>, <span class="author">Bertalan M</span>, <span class="author">Borruel N</span>, <span class="author">Casellas F</span>, <span class="author">Fernandez L</span>, <span class="author">Gautier L</span>, <span class="author">Hansen T</span>, <span class="author">Hattori M</span>, <span class="author">Hayashi T</span>, <span class="author">Kleerebezem M</span>, <span class="author">Kurokawa K</span>, <span class="author">Leclerc M</span>, <span class="author">Levenez F</span>, <span class="author">Manichanh C</span>, <span class="author">Nielsen HB</span>, <span class="author">Nielsen T</span>, <span class="author">Pons N</span>, <span class="author">Poulain J</span>, <span class="author">Qin J</span>, <span class="author">Sicheritz-Ponten T</span>, <span class="author">Tims S</span>, <span class="author">Torrents D</span>, <span class="author">Ugarte E</span>, <span class="author">Zoetendal EG</span>, <span class="author">Wang J</span>, <span class="author">Guarner F</span>, <span class="author">Pedersen O</span>, <span class="author">de Vos WM</span>, <span class="author">Brunak S</span>, <span class="author">Dor&eacute; J</span>, <span class="author">MetaHIT Consortium</span>, <span class="author">Antol&iacute;n M</span>, <span class="author">Artiguenave F</span>, <span class="author">Blottiere HM</span>, <span class="author">Almeida M</span>, <span class="author">Brechot C</span>, <span class="author">Cara C</span>, <span class="author">Chervaux C</span>, <span class="author">Cultrone A</span>, <span class="author">Delorme C</span>, <span class="author">Denariaz G</span>, <span class="author">Dervyn R</span>, <span class="author">Foerstner KU</span>, <span class="author">Friss C</span>, <span class="author">van de Guchte M</span>, <span class="author">Guedon E</span>, <span class="author">Haimet F</span>, <span class="author">Huber W</span>, <span class="author">van Hylckama-Vlieg J</span>, <span class="author">Jamet A</span>, <span class="author">Juste C</span>, <span class="author">Kaci G</span>, <span class="author">Knol J</span>, <span class="author">Lakhdari O</span>, <span class="author">Layec S</span>, <span class="author">Le Roux K</span>, <span class="author">Maguin E</span>, <span class="author">M&eacute;rieux A</span>, <span class="author">Melo Minardi R</span>, <span class="author">M&#39;rini C</span>, <span class="author">Muller J</span>, <span class="author">Oozeer R</span>, <span class="author">Parkhill J</span>, <span class="author">Renault P</span>, <span class="author">Rescigno M</span>, <span class="author">Sanchez N</span>, <span class="author">Sunagawa S</span>, <span class="author">Torrejon A</span>, <span class="author">Turner K</span>, <span class="author">Vandemeulebrouck G</span>, <span class="author">Varela E</span>, <span class="author">Winogradsky Y</span>, <span class="author">Zeller G</span>, <span class="author">Weissenbach J</span>, <span class="author">Ehrlich SD</span> and <span class="author">Bork P</span></p>
<p class="affiliation">European Molecular Biology Laboratory, Meyerhofstrasse 1, 69117 Heidelberg, Germany.</p><p class="abstract">Our knowledge of species and functional composition of the human gut microbiome is rapidly increasing, but it is still based on very few cohorts and little is known about variation across the world. By combining 22 newly sequenced faecal metagenomes of individuals from four countries with previously published data sets, here we identify three robust clusters (referred to as enterotypes hereafter) that are not nation or continent specific. We also confirmed the enterotypes in two published, larger cohorts, indicating that intestinal microbiota variation is generally stratified, not continuous. This indicates further the existence of a limited number of well-balanced host-microbial symbiotic states that might respond differently to diet and drug intake. The enterotypes are mostly driven by species composition, but abundant molecular functions are not necessarily provided by abundant species, highlighting the importance of a functional analysis to understand microbial communities. Although individual host properties such as body mass index, age, or gender cannot explain the observed enterotypes, data-driven marker genes or functional modules can be identified for each of these host properties. For example, twelve genes significantly correlate with age and three functional modules with the body mass index, hinting at a diagnostic potential of microbial markers.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span>, <span class="grant">082372</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">473</span>;<span class="number">7346</span>;<span class="pagerange">174-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21508958">21508958</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3728647">3728647</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09944">10.1038/nature09944</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comprehensive comparison of three commercial human whole-exome capture platforms.</h4>
<p class="authors"><span class="author">Asan</span>, <span class="author">Xu Y</span>, <span class="author">Jiang H</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Xue Y</span>, <span class="author">Jiang T</span>, <span class="author">Wang J</span>, <span class="author">Wu M</span>, <span class="author">Liu X</span>, <span class="author">Tian G</span>, <span class="author">Wang J</span>, <span class="author">Wang J</span>, <span class="author">Yang H</span> and <span class="author">Zhang X</span></p>
<p class="affiliation">Beijing Institute of Genomics, Chinese Academy of Sciences, No.7 Beitucheng West Road, Chaoyang District, Beijing 100029, China. <a href="mailto:%61%73%61%6E%40%67%65%6E%6F%6D%69%63%73.%6F%72%67.%63%6E">&#97;&#115;&#97;&#110;&#64;&#103;&#101;&#110;&#111;&#109;&#105;&#99;&#115;&#46;&#111;&#114;&#103;&#46;&#99;&#110;</a></p><p class="abstract"><strong>Background</strong>: Exome sequencing, which allows the global analysis of protein coding sequences in the human genome, has become an effective and affordable approach to detecting causative genetic mutations in diseases. Currently, there are several commercial human exome capture platforms; however, the relative performances of these have not been characterized sufficiently to know which is best for a particular study.</p><p class="abstract"><strong>Results</strong>: We comprehensively compared three platforms: NimbleGen&#39;s Sequence Capture Array and SeqCap EZ, and Agilent&#39;s SureSelect. We assessed their performance in a variety of ways, including number of genes covered and capture efficacy. Differences that may impact on the choice of platform were that Agilent SureSelect covered approximately 1,100 more genes, while NimbleGen provided better flanking sequence capture. Although all three platforms achieved similar capture specificity of targeted regions, the NimbleGen platforms showed better uniformity of coverage and greater genotype sensitivity at 30- to 100-fold sequencing depth. All three platforms showed similar power in exome SNP calling, including medically relevant SNPs. Compared with genotyping and whole-genome sequencing data, the three platforms achieved a similar accuracy of genotype assignment and SNP detection. Importantly, all three platforms showed similar levels of reproducibility, GC bias and reference allele bias.</p><p class="abstract"><strong>Conclusions</strong>: We demonstrate key differences between the three platforms, particularly advantages of solutions over array capture and the importance of a large gene target set.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">9</span>;<span class="pagerange">R95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21955857">21955857</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308058">3308058</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-9-r95">10.1186/gb-2011-12-9-r95</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An effective method to purify Plasmodium falciparum DNA directly from clinical blood samples for whole genome high-throughput sequencing.</h4>
<p class="authors"><span class="author">Auburn S</span>, <span class="author">Campino S</span>, <span class="author">Clark TG</span>, <span class="author">Djimde AA</span>, <span class="author">Zongo I</span>, <span class="author">Pinches R</span>, <span class="author">Manske M</span>, <span class="author">Mangano V</span>, <span class="author">Alcock D</span>, <span class="author">Anastasi E</span>, <span class="author">Maslen G</span>, <span class="author">Macinnis B</span>, <span class="author">Rockett K</span>, <span class="author">Modiano D</span>, <span class="author">Newbold CI</span>, <span class="author">Doumbo OK</span>, <span class="author">Ou&eacute;draogo JB</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom. <a href="mailto:%73%61%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#97;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Highly parallel sequencing technologies permit cost-effective whole genome sequencing of hundreds of Plasmodium parasites. The ability to sequence clinical Plasmodium samples, extracted directly from patient blood without a culture step, presents a unique opportunity to sample the diversity of &quot;natural&quot; parasite populations in high resolution clinical and epidemiological studies. A major challenge to sequencing clinical Plasmodium samples is the abundance of human DNA, which may substantially reduce the yield of Plasmodium sequence. We tested a range of human white blood cell (WBC) depletion methods on P. falciparum-infected patient samples in search of a method displaying an optimal balance of WBC-removal efficacy, cost, simplicity, and applicability to low resource settings. In the first of a two-part study, combinations of three different WBC depletion methods were tested on 43 patient blood samples in Mali. A two-step combination of Lymphoprep plus Plasmodipur best fitted our requirements, although moderate variability was observed in human DNA quantity. This approach was further assessed in a larger sample of 76 patients from Burkina Faso. WBC-removal efficacy remained high (&lt;30% human DNA in &gt;70% samples) and lower variation was observed in human DNA quantities. In order to assess the Plasmodium sequence yield at different human DNA proportions, 59 samples with up to 60% human DNA contamination were sequenced on the Illumina Genome Analyzer platform. An average ~40-fold coverage of the genome was observed per lane for samples with &le; 30% human DNA. Even in low resource settings, using a simple two-step combination of Lymphoprep plus Plasmodipur, over 70% of clinical sample preparations should exhibit sufficiently low human DNA quantities to enable ~40-fold sequence coverage of the P. falciparum genome using a single lane on the Illumina Genome Analyzer platform. This approach should greatly facilitate large-scale clinical and epidemiologic studies of P. falciparum.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute: <span class="grant">55005502</span>; Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">090770</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">e22213</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21789235">21789235</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3138765">3138765</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0022213">10.1371/journal.pone.0022213</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Male lineages in the Himalayan foothills: a commentary on Y-chromosome haplogroup diversity in the sub-Himalayan Terai and Duars populations of East India.</h4>
<p class="authors"><span class="author">Ayub Q</span></p>
<p class="publication"><span class="title">Journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">56</span>;<span class="number">12</span>;<span class="pagerange">813-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21975650">21975650</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fjhg.2011.114">10.1038/jhg.2011.114</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Combined high-resolution genotyping and geospatial analysis reveals modes of endemic urban typhoid fever transmission.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Holt KE</span>, <span class="author">Clements AC</span>, <span class="author">Karkey A</span>, <span class="author">Arjyal A</span>, <span class="author">Boni MF</span>, <span class="author">Dongol S</span>, <span class="author">Hammond N</span>, <span class="author">Koirala S</span>, <span class="author">Duy PT</span>, <span class="author">Nga TV</span>, <span class="author">Campbell JI</span>, <span class="author">Dolecek C</span>, <span class="author">Basnyat B</span>, <span class="author">Dougan G</span> and <span class="author">Farrar JJ</span></p>
<p class="affiliation">The Hospital for Tropical Diseases, Wellcome Trust Major Overseas Programme , Oxford University Clinical Research Unit , 190 Ben Ham Tu, Quan 5, Ho Chi Minh City , Vietnam.</p>
<p class="abstract">Typhoid is a systemic infection caused by Salmonella Typhi and Salmonella Paratyphi A, human-restricted bacteria that are transmitted faeco-orally. Salmonella Typhi and S. Paratyphi A are clonal, and their limited genetic diversity has precluded the identification of long-term transmission networks in areas with a high disease burden. To improve our understanding of typhoid transmission we have taken a novel approach, performing a longitudinal spatial case-control study for typhoid in Nepal, combining single-nucleotide polymorphism genotyping and case localization via global positioning. We show extensive clustering of typhoid occurring independent of population size and density. For the first time, we demonstrate an extensive range of genotypes existing within typhoid clusters, and even within individual households, including some resulting from clonal expansion. Furthermore, although the data provide evidence for direct human-to-human transmission, we demonstrate an overwhelming contribution of indirect transmission, potentially via contaminated water. Consistent with this, we detected S. Typhi and S. Paratyphi A in water supplies and found that typhoid was spatially associated with public water sources and low elevation. These findings have implications for typhoid-control strategies, and our innovative approach may be applied to other diseases caused by other monophyletic or emerging pathogens.</p>
<p class="publication"><span class="title">Open biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">1</span>;<span class="number">2</span>;<span class="pagerange">110008</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22645647" rel="external">22645647</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3352080" rel="external">3352080</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frsob.110008" rel="external">10.1098/rsob.110008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parallel evolution of genes and languages in the Caucasus region.</h4>
<p class="authors"><span class="author">Balanovsky O</span>, <span class="author">Dibirova K</span>, <span class="author">Dybo A</span>, <span class="author">Mudrak O</span>, <span class="author">Frolova S</span>, <span class="author">Pocheshkhova E</span>, <span class="author">Haber M</span>, <span class="author">Platt D</span>, <span class="author">Schurr T</span>, <span class="author">Haak W</span>, <span class="author">Kuznetsova M</span>, <span class="author">Radzhabov M</span>, <span class="author">Balaganskaya O</span>, <span class="author">Romanov A</span>, <span class="author">Zakharova T</span>, <span class="author">Soria Hernanz DF</span>, <span class="author">Zalloua P</span>, <span class="author">Koshel S</span>, <span class="author">Ruhlen M</span>, <span class="author">Renfrew C</span>, <span class="author">Wells RS</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Balanovska E</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Research Centre for Medical Genetics, Russian Academy of Medical Sciences, Moscow, Russia. <a href="mailto:%62%61%6C%61%6E%6F%76%73%6B%79%40%69%6E%62%6F%78.%72%75">&#98;&#97;&#108;&#97;&#110;&#111;&#118;&#115;&#107;&#121;&#64;&#105;&#110;&#98;&#111;&#120;&#46;&#114;&#117;</a></p><p class="abstract">We analyzed 40 single nucleotide polymorphism and 19 short tandem repeat Y-chromosomal markers in a large sample of 1,525 indigenous individuals from 14 populations in the Caucasus and 254 additional individuals representing potential source populations. We also employed a lexicostatistical approach to reconstruct the history of the languages of the North Caucasian family spoken by the Caucasus populations. We found a different major haplogroup to be prevalent in each of four sets of populations that occupy distinct geographic regions and belong to different linguistic branches. The haplogroup frequencies correlated with geography and, even more strongly, with language. Within haplogroups, a number of haplotype clusters were shown to be specific to individual populations and languages. The data suggested a direct origin of Caucasus male lineages from the Near East, followed by high levels of isolation, differentiation, and genetic drift in situ. Comparison of genetic and linguistic reconstructions covering the last few millennia showed striking correspondences between the topology and dates of the respective gene and language trees and with documented historical events. Overall, in the Caucasus region, unmatched levels of gene-language coevolution occurred within geographically isolated populations, probably due to its mountainous terrain.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">28</span>;<span class="number">10</span>;<span class="pagerange">2905-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21571925">21571925</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3355373">3355373</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsr126">10.1093/molbev/msr126</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene inactivation and its implications for annotation in the era of personal genomics.</h4>
<p class="authors"><span class="author">Balasubramanian S</span>, <span class="author">Habegger L</span>, <span class="author">Frankish A</span>, <span class="author">MacArthur DG</span>, <span class="author">Harte R</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Harrow J</span> and <span class="author">Gerstein M</span></p>
<p class="affiliation">Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06520, USA.</p><p class="abstract">The first wave of personal genomes documents how no single individual genome contains the full complement of functional genes. Here, we describe the extent of variation in gene and pseudogene numbers between individuals arising from inactivation events such as premature termination or aberrant splicing due to single-nucleotide polymorphisms. This highlights the inadequacy of the current reference sequence and gene set. We present a proposal to define a reference gene set that will remain stable as more individuals are sequenced. In particular, we recommend that the ancestral allele be used to define the reference sequence from which a core human reference gene annotation set can be derived. In addition, we call for the development of an expanded gene set to include human-specific genes that have arisen recently and are absent from the ancestral set.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2011</span>;<span class="volume">25</span>;<span class="number">1</span>;<span class="pagerange">1-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21205862">21205862</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3012931">3012931</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.1968411">10.1101/gad.1968411</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CCR4-associated factor 1 coordinates the expression of Plasmodium falciparum egress and invasion proteins.</h4>
<p class="authors"><span class="author">Balu B</span>, <span class="author">Maher SP</span>, <span class="author">Pance A</span>, <span class="author">Chauhan C</span>, <span class="author">Naumov AV</span>, <span class="author">Andrews RM</span>, <span class="author">Ellis PD</span>, <span class="author">Khan SM</span>, <span class="author">Lin JW</span>, <span class="author">Janse CJ</span>, <span class="author">Rayner JC</span> and <span class="author">Adams JH</span></p>
<p class="affiliation">Department of Global Health, College of Public Health, University of South Florida, College of Public Health, 3720 Spectrum Blvd., Suite 304, Tampa, FL, USA.</p><p class="abstract">Coordinated regulation of gene expression is a hallmark of the Plasmodium falciparum asexual blood-stage development cycle. We report that carbon catabolite repressor protein 4 (CCR4)-associated factor 1 (CAF1) is critical in regulating more than 1,000 genes during malaria parasites&#39; intraerythrocytic stages, especially egress and invasion proteins. CAF1 knockout results in mistimed expression, aberrant accumulation and localization of proteins involved in parasite egress, and invasion of new host cells, leading to premature release of predominantly half-finished merozoites, drastically reducing the intraerythrocytic growth rate of the parasite. This study demonstrates that CAF1 of the CCR4-Not complex is a significant gene regulatory mechanism needed for Plasmodium development within the human host.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI094973</span>, <span class="grant">R01 AI094973-01</span>, <span class="grant">R01AI033656</span>, <span class="grant">R01AI094973</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">10</span>;<span class="number">9</span>;<span class="pagerange">1257-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21803864">21803864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3187058">3187058</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.05099-11">10.1128/EC.05099-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of genetic Loci with glucose levels in childhood and adolescence: a meta-analysis of over 6,000 children.</h4>
<p class="authors"><span class="author">Barker A</span>, <span class="author">Sharp SJ</span>, <span class="author">Timpson NJ</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Warrington NM</span>, <span class="author">Kanoni S</span>, <span class="author">Beilin LJ</span>, <span class="author">Brage S</span>, <span class="author">Deloukas P</span>, <span class="author">Evans DM</span>, <span class="author">Grontved A</span>, <span class="author">Hassanali N</span>, <span class="author">Lawlor DA</span>, <span class="author">Lecoeur C</span>, <span class="author">Loos RJ</span>, <span class="author">Lye SJ</span>, <span class="author">McCarthy MI</span>, <span class="author">Mori TA</span>, <span class="author">Ndiaye NC</span>, <span class="author">Newnham JP</span>, <span class="author">Ntalla I</span>, <span class="author">Pennell CE</span>, <span class="author">St Pourcain B</span>, <span class="author">Prokopenko I</span>, <span class="author">Ring SM</span>, <span class="author">Sattar N</span>, <span class="author">Visvikis-Siest S</span>, <span class="author">Dedoussis GV</span>, <span class="author">Palmer LJ</span>, <span class="author">Froguel P</span>, <span class="author">Smith GD</span>, <span class="author">Ekelund U</span>, <span class="author">Wareham NJ</span> and <span class="author">Langenberg C</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Addenbrooke&rsquo;s Hospital,Institute of Metabolic Science, Cambridge, U.K.</p><p class="abstract"><strong>Objective</strong>: To investigate whether associations of common genetic variants recently identified for fasting glucose or insulin levels in nondiabetic adults are detectable in healthy children and adolescents.</p><p class="abstract">A total of 16 single nucleotide polymorphisms (SNPs) associated with fasting glucose were genotyped in six studies of children and adolescents of European origin, including over 6,000 boys and girls aged 9-16 years. We performed meta-analyses to test associations of individual SNPs and a weighted risk score of the 16 loci with fasting glucose.</p><p class="abstract"><strong>Results</strong>: Nine loci were associated with glucose levels in healthy children and adolescents, with four of these associations reported in previous studies and five reported here for the first time (GLIS3, PROX1, SLC2A2, ADCY5, and CRY2). Effect sizes were similar to those in adults, suggesting age-independent effects of these fasting glucose loci. Children and adolescents carrying glucose-raising alleles of G6PC2, MTNR1B, GCK, and GLIS3 also showed reduced &beta;-cell function, as indicated by homeostasis model assessment of &beta;-cell function. Analysis using a weighted risk score showed an increase [&beta; (95% CI)] in fasting glucose level of 0.026 mmol/L (0.021-0.031) for each unit increase in the score.</p><p class="abstract"><strong>Conclusions</strong>: Novel fasting glucose loci identified in genome-wide association studies of adults are associated with altered fasting glucose levels in healthy children and adolescents with effect sizes comparable to adults. In nondiabetic adults, fasting glucose changes little over time, and our results suggest that age-independent effects of fasting glucose loci contribute to long-term interindividual differences in glucose levels from childhood onwards.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">R01-DK-077659</span>; Wellcome Trust: <span class="grant">076467</span>, <span class="grant">74882</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">6</span>;<span class="pagerange">1805-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21515849" rel="external">21515849</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3114379" rel="external">3114379</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb10-1575" rel="external">10.2337/db10-1575</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNAcentral: A vision for an international database of RNA sequences.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Agrawal S</span>, <span class="author">Birney E</span>, <span class="author">Bruford EA</span>, <span class="author">Bujnicki JM</span>, <span class="author">Cochrane G</span>, <span class="author">Cole JR</span>, <span class="author">Dinger ME</span>, <span class="author">Enright AJ</span>, <span class="author">Gardner PP</span>, <span class="author">Gautheret D</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Harrow J</span>, <span class="author">Herrero J</span>, <span class="author">Holmes IH</span>, <span class="author">Huang HD</span>, <span class="author">Kelly KA</span>, <span class="author">Kersey P</span>, <span class="author">Kozomara A</span>, <span class="author">Lowe TM</span>, <span class="author">Marz M</span>, <span class="author">Moxon S</span>, <span class="author">Pruitt KD</span>, <span class="author">Samuelsson T</span>, <span class="author">Stadler PF</span>, <span class="author">Vilella AJ</span>, <span class="author">Vogel JH</span>, <span class="author">Williams KP</span>, <span class="author">Wright MW</span> and <span class="author">Zwieb C</span></p><p class="abstract">During the last decade there has been a great increase in the number of noncoding RNA genes identified, including new classes such as microRNAs and piRNAs. There is also a large growth in the amount of experimental characterization of these RNA components. Despite this growth in information, it is still difficult for researchers to access RNA data, because key data resources for noncoding RNAs have not yet been created. The most pressing omission is the lack of a comprehensive RNA sequence database, much like UniProt, which provides a comprehensive set of protein knowledge. In this article we propose the creation of a new open public resource that we term RNAcentral, which will contain a comprehensive collection of RNA sequences and fill an important gap in the provision of biomedical databases. We envision RNA researchers from all over the world joining a federated RNAcentral network, contributing specialized knowledge and databases. RNAcentral would centralize key data that are currently held across a variety of databases, allowing researchers instant access to a single, unified resource. This resource would facilitate the next generation of RNA research and help drive further discoveries, including those that improve food production and human and animal health. We encourage additional RNA database resources and research groups to join this effort. We aim to obtain international network funding to further this endeavor.</p>
<p class="publication"><span class="title">RNA (New York, N.Y.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">17</span>;<span class="number">11</span>;<span class="pagerange">1941-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21940779">21940779</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3198587">3198587</a>; DOI: <a href="http://dx.doi.org/10.1261%2Frna.2750811">10.1261/rna.2750811</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the proteome, diseases and evolution of the human postsynaptic density.</h4>
<p class="authors"><span class="author">Bay&eacute;s A</span>, <span class="author">van de Lagemaat LN</span>, <span class="author">Collins MO</span>, <span class="author">Croning MD</span>, <span class="author">Whittle IR</span>, <span class="author">Choudhary JS</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire, UK.</p><p class="abstract">We isolated the postsynaptic density from human neocortex (hPSD) and identified 1,461 proteins. hPSD mutations cause 133 neurological and psychiatric diseases and were enriched in cognitive, affective and motor phenotypes underpinned by sets of genes. Strong protein sequence conservation in mammalian lineages, particularly in hub proteins, indicates conserved function and organization in primate and rodent models. The hPSD is an important structure for nervous system disease and behavior.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/486</span>; Medical Research Council: <span class="grant">G0802238</span>, <span class="grant">G0802238(89569)</span>; Wellcome Trust: <span class="grant">066717</span>, <span class="grant">077155</span></p>
<p class="publication"><span class="title">Nature neuroscience</span>&nbsp;<span class="year">2011</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">19-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21170055">21170055</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3040565">3040565</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnn.2719">10.1038/nn.2719</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">How I treat essential thrombocythemia.</h4>
<p class="authors"><span class="author">Beer PA</span>, <span class="author">Erber WN</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Cambridge Institute for Medical Research and Department of Haematology, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">In the past 5 years we have witnessed significant advances in both the diagnostic process and optimal therapy for patients with essential thrombocythemia (ET). Insights into the underlying molecular mechanisms have been accompanied by the development of new diagnostic tests and by an improved understanding of the relationship between ET and other related myeloproliferative neoplasms, such as polycythemia vera and primary myelofibrosis. In the first part of this review, we describe how recent molecular and histologic studies can be integrated into a streamlined diagnostic process that is applicable to everyday clinical practice. We also address areas of current diagnostic controversy, including heterogeneity within ET and the phenotypic overlap between ET, polycythemia vera, and primary myelofibrosis. In the second part, we provide an overview of our current approach to the treatment of ET, including risk stratification, choice of cytoreductive agent, and a consideration of special situations such as the pregnant or perioperative patient. Areas of controversy discussed include the identification of those at high risk of complications and therapeutic decisions in the younger patient.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2011</span>;<span class="volume">117</span>;<span class="number">5</span>;<span class="pagerange">1472-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21106990" rel="external">21106990</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145107" rel="external">3145107</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2010-08-270033" rel="external">10.1182/blood-2010-08-270033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Essential thrombocythemia: seeing the wood for the trees Response</h4>
<p class="authors"><span class="author">Beer PA</span>, <span class="author">Erber WN</span>, <span class="author">CAMPBELL PJ</span>, <span class="author">Green AR</span></p>
<p class="publication"><span class="title">Blood.</span> <span class="year">2011</span>;<span class="volume">118</span>;<span class="pagerange">1180-1</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association scan allowing for epistasis in type 2 diabetes.</h4>
<p class="authors"><span class="author">Bell JT</span>, <span class="author">Timpson NJ</span>, <span class="author">Rayner NW</span>, <span class="author">Zeggini E</span>, <span class="author">Frayling TM</span>, <span class="author">Hattersley AT</span>, <span class="author">Morris AP</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, UK. <a href="mailto:%6A%6F%72%64%61%6E%61%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#106;&#111;&#114;&#100;&#97;&#110;&#97;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="abstract">In the presence of epistasis multilocus association tests of human complex traits can provide powerful methods to detect susceptibility variants. We undertook multilocus analyses in 1924 type 2 diabetes cases and 2938 controls from the Wellcome Trust Case Control Consortium (WTCCC). We performed a two-dimensional genome-wide association (GWA) scan using joint two-locus tests of association including main and epistatic effects in 70,236 markers tagging common variants. We found two-locus association at 79 SNP-pairs at a Bonferroni-corrected P-value = 0.05 (uncorrected P-value = 2.14 Ã— 10â»Â¹Â¹). The 79 pair-wise results always contained rs11196205 in TCF7L2 paired with 79 variants including confirmed variants in FTO, TSPAN8, and CDKAL1, which are associated in the absence of epistasis. However, the majority (82%) of the 79 variants did not have compelling single-locus association signals (P-value = 5 Ã— 10â»â´). Analyses conditional on the single-locus effects at TCF7L2 established that the joint two-locus results could be attributed to single-locus association at TCF7L2 alone. Interaction analyses among the peak 80 regions and among 23 previously established diabetes candidate genes identified five SNP-pairs with case-control and case-only epistatic signals. Our results demonstrate the feasibility of systematic scans in GWA data, but confirm that single-locus association can underlie and obscure multilocus findings.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600705</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">088885</span>, <span class="grant">WT081682/Z/06/Z</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Annals of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">75</span>;<span class="number">1</span>;<span class="pagerange">10-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21133856" rel="external">21133856</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3430851" rel="external">3430851</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1469-1809.2010.00629.x" rel="external">10.1111/j.1469-1809.2010.00629.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metagenomics and the molecular identification of novel viruses.</h4>
<p class="authors"><span class="author">Bexfield N</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, UK. <a href="mailto:%6E%62%32%38%39%40%63%61%6D.%61%63.%75%6B">&#110;&#98;&#50;&#56;&#57;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">There have been rapid recent developments in establishing methods for identifying and characterising viruses associated with animal and human diseases. These methodologies, commonly based on hybridisation or PCR techniques, are combined with advanced sequencing techniques termed &#39;next generation sequencing&#39;. Allied advances in data analysis, including the use of computational transcriptome subtraction, have also impacted the field of viral pathogen discovery. This review details these molecular detection techniques, discusses their application in viral discovery, and provides an overview of some of the novel viruses discovered. The problems encountered in attributing disease causality to a newly identified virus are also considered.</p>
<p class="publication"><span class="title">Veterinary journal (London, England : 1997)</span>&nbsp;<span class="year">2011</span>;<span class="volume">190</span>;<span class="number">2</span>;<span class="pagerange">191-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21111643">21111643</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tvjl.2010.10.014">10.1016/j.tvjl.2010.10.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genomic rate of molecular adaptation of the human influenza A virus.</h4>
<p class="authors"><span class="author">Bhatt S</span>, <span class="author">Holmes EC</span> and <span class="author">Pybus OG</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">Quantifying adaptive evolution at the genomic scale is an essential yet challenging aspect of evolutionary biology. Here, we develop a method that extends and generalizes previous approaches to estimate the rate of genomic adaptation in rapidly evolving populations and apply it to a large data set of complete human influenza A virus genome sequences. In accord with previous studies, we observe particularly high rates of adaptive evolution in domain 1 of the viral hemagglutinin (HA1). However, our novel approach also reveals previously unseen adaptation in other viral genes. Notably, we find that the rate of adaptation (per codon per year) is higher in surface residues of the viral neuraminidase than in HA1, indicating strong antibody-mediated selection on the former. We also observed high rates of adaptive evolution in several nonstructural proteins, which may relate to viral evasion of T-cell and innate immune responses. Furthermore, our analysis provides strong quantitative support for the hypothesis that human H1N1 influenza experiences weaker antigenic selection than H3N2. As well as shedding new light on the dynamics and determinants of positive Darwinian selection in influenza viruses, the approach introduced here is applicable to other pathogens for which densely sampled genome sequences are available, and hence is ideally suited to the interpretation of next-generation genome sequencing data.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM080533</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">28</span>;<span class="number">9</span>;<span class="pagerange">2443-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21415025" rel="external">21415025</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3163432" rel="external">3163432</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsr044" rel="external">10.1093/molbev/msr044</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting report of the RNA Ontology Consortium January 8-9, 2011.</h4>
<p class="authors"><span class="author">Birmingham A</span>, <span class="author">Clemente JC</span>, <span class="author">Desai N</span>, <span class="author">Gilbert J</span>, <span class="author">Gonzalez A</span>, <span class="author">Kyrpides N</span>, <span class="author">Meyer F</span>, <span class="author">Nawrocki E</span>, <span class="author">Sterk P</span>, <span class="author">Stombaugh J</span>, <span class="author">Weinberg Z</span>, <span class="author">Wendel D</span>, <span class="author">Leontis NB</span>, <span class="author">Zirbel C</span>, <span class="author">Knight R</span> and <span class="author">Laederach A</span></p><p class="abstract">This report summarizes the proceedings of the structure mapping working group meeting of the RNA Ontology Consortium (ROC), held in Kona, Hawaii on January 8-9, 2011. The ROC hosted this workshop to facilitate collaborations among those researchers formalizing concepts in RNA, those developing RNA-related software, and those performing genome annotation and standardization. The workshop included three software presentations, extended round-table discussions, and the constitution of two new working groups, the first to address the need for better software integration and the second to discuss standardization and benchmarking of existing RNA annotation pipelines. These working groups have subsequently pursued concrete implementation of actions suggested during the discussion. Further information about the ROC and its activities can be found at http://roc.bgsu.edu/.</p>
<p class="publication"><span class="title">Standards in genomic sciences</span>&nbsp;<span class="year">2011</span>;<span class="volume">4</span>;<span class="number">2</span>;<span class="pagerange">252-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21677862" rel="external">21677862</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3111981" rel="external">3111981</a>; DOI: <a href="http://dx.doi.org/10.4056%2Fsigs.1724282" rel="external">10.4056/sigs.1724282</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies from the CHARGE consortium identifies common variants associated with carotid intima media thickness and plaque.</h4>
<p class="authors"><span class="author">Bis JC</span>, <span class="author">Kavousi M</span>, <span class="author">Franceschini N</span>, <span class="author">Isaacs A</span>, <span class="author">Abecasis GR</span>, <span class="author">Schminke U</span>, <span class="author">Post WS</span>, <span class="author">Smith AV</span>, <span class="author">Cupples LA</span>, <span class="author">Markus HS</span>, <span class="author">Schmidt R</span>, <span class="author">Huffman JE</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Baumert J</span>, <span class="author">M&uuml;nzel T</span>, <span class="author">Heckbert SR</span>, <span class="author">Dehghan A</span>, <span class="author">North K</span>, <span class="author">Oostra B</span>, <span class="author">Bevan S</span>, <span class="author">Stoegerer EM</span>, <span class="author">Hayward C</span>, <span class="author">Raitakari O</span>, <span class="author">Meisinger C</span>, <span class="author">Schillert A</span>, <span class="author">Sanna S</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Cheng YC</span>, <span class="author">Thorsson B</span>, <span class="author">Fox CS</span>, <span class="author">Rice K</span>, <span class="author">Rivadeneira F</span>, <span class="author">Nambi V</span>, <span class="author">Halperin E</span>, <span class="author">Petrovic KE</span>, <span class="author">Peltonen L</span>, <span class="author">Wichmann HE</span>, <span class="author">Schnabel RB</span>, <span class="author">D&ouml;rr M</span>, <span class="author">Parsa A</span>, <span class="author">Aspelund T</span>, <span class="author">Demissie S</span>, <span class="author">Kathiresan S</span>, <span class="author">Reilly MP</span>, <span class="author">Taylor K</span>, <span class="author">Uitterlinden A</span>, <span class="author">Couper DJ</span>, <span class="author">Sitzer M</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Illig T</span>, <span class="author">Wild PS</span>, <span class="author">Orru M</span>, <span class="author">L&uuml;demann J</span>, <span class="author">Shuldiner AR</span>, <span class="author">Eiriksdottir G</span>, <span class="author">White CC</span>, <span class="author">Rotter JI</span>, <span class="author">Hofman A</span>, <span class="author">Seissler J</span>, <span class="author">Zeller T</span>, <span class="author">Usala G</span>, <span class="author">Ernst F</span>, <span class="author">Launer LJ</span>, <span class="author">D&#39;Agostino RB</span>, <span class="author">O&#39;Leary DH</span>, <span class="author">Ballantyne C</span>, <span class="author">Thiery J</span>, <span class="author">Ziegler A</span>, <span class="author">Lakatta EG</span>, <span class="author">Chilukoti RK</span>, <span class="author">Harris TB</span>, <span class="author">Wolf PA</span>, <span class="author">Psaty BM</span>, <span class="author">Polak JF</span>, <span class="author">Li X</span>, <span class="author">Rathmann W</span>, <span class="author">Uda M</span>, <span class="author">Boerwinkle E</span>, <span class="author">Klopp N</span>, <span class="author">Schmidt H</span>, <span class="author">Wilson JF</span>, <span class="author">Viikari J</span>, <span class="author">Koenig W</span>, <span class="author">Blankenberg S</span>, <span class="author">Newman AB</span>, <span class="author">Witteman J</span>, <span class="author">Heiss G</span>, <span class="author">Duijn Cv</span>, <span class="author">Scuteri A</span>, <span class="author">Homuth G</span>, <span class="author">Mitchell BD</span>, <span class="author">Gudnason V</span>, <span class="author">O&#39;Donnell CJ</span> and <span class="author">CARDIoGRAM Consortium</span></p>
<p class="affiliation">Cardiovascular Health Research Unit and Department of Medicine, University of Washington, Seattle, Washington, USA. <a href="mailto:%6A%6F%73%68%62%69%73%40%75%77.%65%64%75">&#106;&#111;&#115;&#104;&#98;&#105;&#115;&#64;&#117;&#119;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Carotid intima media thickness (cIMT) and plaque determined by ultrasonography are established measures of subclinical atherosclerosis that each predicts future cardiovascular disease events. We conducted a meta-analysis of genome-wide association data in 31,211 participants of European ancestry from nine large studies in the setting of the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium. We then sought additional evidence to support our findings among 11,273 individuals using data from seven additional studies. In the combined meta-analysis, we identified three genomic regions associated with common carotid intima media thickness and two different regions associated with the presence of carotid plaque (P &lt; 5 &times; 10(-8)). The associated SNPs mapped in or near genes related to cellular signaling, lipid metabolism and blood pressure homeostasis, and two of the regions were associated with coronary artery disease (P &lt; 0.006) in the Coronary Artery Disease Genome-Wide Replication and Meta-Analysis (CARDIoGRAM) consortium. Our findings may provide new insight into pathways leading to subclinical atherosclerosis and subsequent cardiovascular events.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">MC_U127561128</span>; NCRR NIH HHS: <span class="grant">M01 RR 16500</span>, <span class="grant">M01RR00069</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">HG005581</span>, <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">HL075366</span>, <span class="grant">HL080295</span>, <span class="grant">HL084729</span>, <span class="grant">HL087652</span>, <span class="grant">HL105756</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-85239</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">Z01 HL006002-01</span>, <span class="grant">Z99 HL999999</span>; NIA NIH HHS: <span class="grant">AG-023629</span>, <span class="grant">AG-027058</span>, <span class="grant">AG-15928</span>, <span class="grant">AG-20098</span>, <span class="grant">AG033193</span>, <span class="grant">AG08122</span>, <span class="grant">AG16495</span>, <span class="grant">N01-AG-1-2109</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">R01 AG18728</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">P30 DK072488</span>; NIGMS NIH HHS: <span class="grant">U01 GM074518-04</span>; NINDS NIH HHS: <span class="grant">NS17950</span>; PHS HHS: <span class="grant">268200625226C</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">10</span>;<span class="pagerange">940-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21909108">21909108</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3257519">3257519</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.920">10.1038/ng.920</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic mapping identifies novel highly protective antigens for an apicomplexan parasite.</h4>
<p class="authors"><span class="author">Blake DP</span>, <span class="author">Billington KJ</span>, <span class="author">Copestake SL</span>, <span class="author">Oakes RD</span>, <span class="author">Quail MA</span>, <span class="author">Wan KL</span>, <span class="author">Shirley MW</span> and <span class="author">Smith AL</span></p>
<p class="affiliation">Institute for Animal Health, Compton, Berkshire, United Kingdom. <a href="mailto:%64%62%6C%61%6B%65%40%72%76%63.%61%63.%75%6B">&#100;&#98;&#108;&#97;&#107;&#101;&#64;&#114;&#118;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Apicomplexan parasites are responsible for a myriad of diseases in humans and livestock; yet despite intensive effort, development of effective sub-unit vaccines remains a long-term goal. Antigenic complexity and our inability to identify protective antigens from the pool that induce response are serious challenges in the development of new vaccines. Using a combination of parasite genetics and selective barriers with population-based genetic fingerprinting, we have identified that immunity against the most important apicomplexan parasite of livestock (Eimeria spp.) was targeted against a few discrete regions of the genome. Herein we report the identification of six genomic regions and, within two of those loci, the identification of true protective antigens that confer immunity as sub-unit vaccines. The first of these is an Eimeria maxima homologue of apical membrane antigen-1 (AMA-1) and the second is a previously uncharacterised gene that we have termed &#39;immune mapped protein-1&#39; (IMP-1). Significantly, homologues of the AMA-1 antigen are protective with a range of apicomplexan parasites including Plasmodium spp., which suggest that there may be some characteristic(s) of protective antigens shared across this diverse group of parasites. Interestingly, homologues of the IMP-1 antigen, which is protective against E. maxima infection, can be identified in Toxoplasma gondii and Neospora caninum. Overall, this study documents the discovery of novel protective antigens using a population-based genetic mapping approach allied with a protection-based screen of candidate genes. The identification of AMA-1 and IMP-1 represents a substantial step towards development of an effective anti-eimerian sub-unit vaccine and raises the possibility of identification of novel antigens for other apicomplexan parasites. Moreover, validation of the parasite genetics approach to identify effective antigens supports its adoption in other parasite systems where legitimate protective antigen identification is difficult.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBE01089X1</span>, <span class="grant">S20198</span>; Wellcome Trust: <span class="grant">085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e1001279</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21347348" rel="external">21347348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3037358" rel="external">3037358</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1001279" rel="external">10.1371/journal.ppat.1001279</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Soluble flagellin, FliC, induces an Ag-specific Th2 response, yet promotes T-bet-regulated Th1 clearance of Salmonella typhimurium infection.</h4>
<p class="authors"><span class="author">Bobat S</span>, <span class="author">Flores-Langarica A</span>, <span class="author">Hitchcock J</span>, <span class="author">Marshall JL</span>, <span class="author">Kingsley RA</span>, <span class="author">Goodall M</span>, <span class="author">Gil-Cruz C</span>, <span class="author">Serre K</span>, <span class="author">Leyton DL</span>, <span class="author">Letran SE</span>, <span class="author">Gaspal F</span>, <span class="author">Chester R</span>, <span class="author">Chamberlain JL</span>, <span class="author">Dougan G</span>, <span class="author">L&oacute;pez-Mac&iacute;as C</span>, <span class="author">Henderson IR</span>, <span class="author">Alexander J</span>, <span class="author">MacLennan IC</span> and <span class="author">Cunningham AF</span></p>
<p class="affiliation">MRC Centre for Immune Regulation, University of Birmingham, Birmingham, UK.</p><p class="abstract">Clearance of disseminated Salmonella infection requires bacterial-specific Th1 cells and IFN-&gamma; production, and Th1-promoting vaccines are likely to help control these infections. Consequently, vaccine design has focused on developing Th1-polarizing adjuvants or Ag that naturally induce Th1 responses. In this study, we show that, in mice, immunization with soluble, recombinant FliC protein flagellin (sFliC) induces Th2 responses as evidenced by Ag-specific GATA-3, IL-4 mRNA, and protein induction in CD62L(lo) CD4(+) T cells without associated IFN-&gamma; production. Despite these Th2 features, sFliC immunization can enhance the development of protective Th1 immunity during subsequent Salmonella infection in an Ab-independent, T-cell-dependent manner. Salmonella infection in sFliC-immunized mice resulted in augmented Th1 responses, with greater bacterial clearance and increased numbers of IFN-&gamma;-producing CD4(+) T cells, despite the early induction of Th2 features to sFliC. The augmented Th1 immunity after sFliC immunization was regulated by T-bet although T-bet is dispensable for primary responses to sFliC. These findings show that there can be flexibility in T-cell responses to some subunit vaccines. These vaccines may induce Th2-type immunity during primary immunization yet promote Th1-dependent responses during later infection. This suggests that designing Th1-inducing subunit vaccines may not always be necessary since this can occur naturally during subsequent infection.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">2011</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">1606-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21469112" rel="external">21469112</a>; DOI: <a href="http://dx.doi.org/10.1002%2Feji.201041089" rel="external">10.1002/eji.201041089</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Abdominal aortic aneurysm is associated with a variant in low-density lipoprotein receptor-related protein 1.</h4>
<p class="authors"><span class="author">Bown MJ</span>, <span class="author">Jones GT</span>, <span class="author">Harrison SC</span>, <span class="author">Wright BJ</span>, <span class="author">Bumpstead S</span>, <span class="author">Baas AF</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Badger SA</span>, <span class="author">Bradley DT</span>, <span class="author">Burnand K</span>, <span class="author">Child AH</span>, <span class="author">Clough RE</span>, <span class="author">Cockerill G</span>, <span class="author">Hafez H</span>, <span class="author">Scott DJ</span>, <span class="author">Futers S</span>, <span class="author">Johnson A</span>, <span class="author">Sohrabi S</span>, <span class="author">Smith A</span>, <span class="author">Thompson MM</span>, <span class="author">van Bockxmeer FM</span>, <span class="author">Waltham M</span>, <span class="author">Matthiasson SE</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Blankensteijn JD</span>, <span class="author">Teijink JA</span>, <span class="author">Wijmenga C</span>, <span class="author">de Graaf J</span>, <span class="author">Kiemeney LA</span>, <span class="author">Assimes TL</span>, <span class="author">McPherson R</span>, <span class="author">CARDIoGRAM Consortium</span>, <span class="author">Global BPgen Consortium</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">VRCNZ Consortium</span>, <span class="author">Folkersen L</span>, <span class="author">Franco-Cereceda A</span>, <span class="author">Palmen J</span>, <span class="author">Smith AJ</span>, <span class="author">Sylvius N</span>, <span class="author">Wild JB</span>, <span class="author">Refstrup M</span>, <span class="author">Edkins S</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt SE</span>, <span class="author">Potter S</span>, <span class="author">Lindholt JS</span>, <span class="author">Frikke-Schmidt R</span>, <span class="author">Tybj&aelig;rg-Hansen A</span>, <span class="author">Hughes AE</span>, <span class="author">Golledge J</span>, <span class="author">Norman PE</span>, <span class="author">van Rij A</span>, <span class="author">Powell JT</span>, <span class="author">Eriksson P</span>, <span class="author">Stefansson K</span>, <span class="author">Thompson JR</span>, <span class="author">Humphries SE</span>, <span class="author">Sayers RD</span>, <span class="author">Deloukas P</span> and <span class="author">Samani NJ</span></p>
<p class="affiliation">Department of Cardiovascular Sciences, University of Leicester, Leicester LE2 7LX, UK. <a href="mailto:%6D.%62%6F%77%6E%40%6C%65.%61%63.%75%6B">&#109;&#46;&#98;&#111;&#119;&#110;&#64;&#108;&#101;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Abdominal aortic aneurysm (AAA) is a common cause of morbidity and mortality and has a significant heritability. We carried out a genome-wide association discovery study of 1866 patients with AAA and 5435 controls and replication of promising signals (lead SNP with a p value &lt; 1&nbsp;&times; 10(-5)) in 2871 additional cases and 32,687 controls and performed further follow-up in 1491 AAA and 11,060 controls. In the discovery study, nine loci demonstrated association with AAA (p &lt; 1&nbsp;&times; 10(-5)). In the replication sample, the lead SNP at one of these loci, rs1466535, located within intron 1 of low-density-lipoprotein receptor-related protein 1 (LRP1) demonstrated significant association (p = 0.0042). We confirmed the association of rs1466535 and AAA in our follow-up study (p = 0.035). In a combined analysis (6228 AAA and 49182 controls), rs1466535 had a consistent effect size and direction in all sample sets (combined p = 4.52&nbsp;&times; 10(-10), odds ratio 1.15 [1.10-1.21]). No associations were seen for either rs1466535 or the 12q13.3 locus in independent association studies of coronary artery disease, blood pressure, diabetes, or hyperlipidaemia, suggesting that this locus is specific to AAA. Gene-expression studies demonstrated a trend toward increased LRP1 expression for the rs1466535 CC genotype in arterial tissues; there was a significant (p = 0.029) 1.19-fold (1.04-1.36) increase in LRP1 expression in CC homozygotes compared to TT homozygotes in aortic adventitia. Functional studies demonstrated that rs1466535 might alter a SREBP-1 binding site and influence enhancer activity at the locus. In conclusion, this study has identified a biologically plausible genetic variant associated specifically with AAA, and we suggest that this variant has a possible functional role in LRP1 expression.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/11/16/28696</span>, <span class="grant">PG/10/001/28098</span>, <span class="grant">RG2008/08</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">084695</span>, <span class="grant">085475</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">89</span>;<span class="number">5</span>;<span class="pagerange">619-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22055160">22055160</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3213391">3213391</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2011.10.002">10.1016/j.ajhg.2011.10.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Joint thrombus and vessel segmentation using dynamic texture likelihoods and shape prior.</h4>
<p class="authors"><span class="author">Brieu N</span>, <span class="author">Groher M</span>, <span class="author">Serbanovic-Canic J</span>, <span class="author">Cvejic A</span>, <span class="author">Ouwehand W</span> and <span class="author">Navab N</span></p>
<p class="affiliation">Computer Aided Medical Procedures, Technische Universit&auml;t M&uuml;nchen, Germany. <a href="mailto:%62%72%69%65%75%40%69%6E.%74%75%6D.%64%65">&#98;&#114;&#105;&#101;&#117;&#64;&#105;&#110;&#46;&#116;&#117;&#109;&#46;&#100;&#101;</a></p><p class="abstract">The segmentation of thrombus and vessel in microscopic image sequences is of high interest for identifying genes linked to cardiovascular diseases. This task is however challenging because of the low contrast and the highly dynamic conditions observed in time-lapse DIC in-vivo microscopic scenes. In this work, we introduce a probabilistic framework for the joint segmentation of thrombus and vessel regions. Modeling the scene with dynamic textures, we derive two likelihood functions to account for both spatial and temporal discrepancies of the motion patterns. A tubular shape prior is moreover introduced to constrain the aortic region. Extensive experiments on microscopic sequences quantitatively show the good performance of our approach.</p>
<p class="publication"><span class="title">Medical image computing and computer-assisted intervention : MICCAI ... International Conference on Medical Image Computing and Computer-Assisted Intervention</span>&nbsp;<span class="year">2011</span>;<span class="volume">14</span>;<span class="number">Pt 3</span>;<span class="pagerange">579-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22003746" rel="external">22003746</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TSIDER1, a short and non-autonomous Salivarian trypanosome-specific retroposon related to the ingi6 subclade.</h4>
<p class="authors"><span class="author">Bringaud F</span>, <span class="author">Berriman M</span> and <span class="author">Hertz-Fowler C</span></p>
<p class="affiliation">Centre de R&eacute;sonance Magn&eacute;tique des Syst&egrave;mes Biologiques, UMR 5536, Universit&eacute; Bordeaux Segalen, CNRS, 146 rue L&eacute;o Saignat, 33076 Bordeaux, France. <a href="mailto:%62%72%69%6E%67%61%75%64%40%72%6D%73%62.%75-%62%6F%72%64%65%61%75%78%32.%66%72">&#98;&#114;&#105;&#110;&#103;&#97;&#117;&#100;&#64;&#114;&#109;&#115;&#98;&#46;&#117;&#45;&#98;&#111;&#114;&#100;&#101;&#97;&#117;&#120;&#50;&#46;&#102;&#114;</a></p><p class="abstract">Retroposons of the ingi clade are the most abundant transposable elements identified in the trypanosomatid genomes. Some are long autonomous elements (ingi, L1Tc) while others, such as RIME and NARTc, are short non-coding elements that parasitize the retrotransposition machinery of the active autonomous ones for their own mobilization. Here, we identified a new family of short non-autonomous retroposons of the ingi clade, called TSIDER1, which are present in the genome of Salivarian (African) trypanosomes, Trypanosoma brucei, T. congolense and T. vivax, but absent in the T. cruzi and Leishmania spp. genomes and, as such, TSIDER1 is the only retroposon subfamily conserved at the nucleotide level between African trypanosome species. We identified three TvSIDER1 families within the genome of T. vivax and the high level of sequence conservation within the TvSIDER1a and TvSIDER1b groups suggests that they are still active. We propose that TvSIDER1a/b elements are using the Tvingi retrotransposition machinery, as they are preceded by the same conserved pattern characteristic of the ingi6 subclade, which corresponds to the retroposon-encoded endonuclease binding site. In contrast, TcoSIDER1, TbSIDER1 and TvSIDER1c are too divergent to be considered as active retroposons. The relatively low number of SIDER elements identified in the T. congolense (70 copies), T. vivax (32 copies) and T. brucei (22 copies) genomes confirms that trypanosomes have not expanded short transposable elements, which is in contrast to Leishmania spp. (&sim;2000 copies), where SIDER play a role in the regulation of gene expression.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z//08/Z</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2011</span>;<span class="volume">179</span>;<span class="number">1</span>;<span class="pagerange">30-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21664383">21664383</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2011.05.007">10.1016/j.molbiopara.2011.05.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shotgun proteomics aids discovery of novel protein-coding genes, alternative splicing, and &quot;resurrected&quot; pseudogenes in the mouse genome.</h4>
<p class="authors"><span class="author">Brosch M</span>, <span class="author">Saunders GI</span>, <span class="author">Frankish A</span>, <span class="author">Collins MO</span>, <span class="author">Yu L</span>, <span class="author">Wright J</span>, <span class="author">Verstraten R</span>, <span class="author">Adams DJ</span>, <span class="author">Harrow J</span>, <span class="author">Choudhary JS</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Recent advances in proteomic mass spectrometry (MS) offer the chance to marry high-throughput peptide sequencing to transcript models, allowing the validation, refinement, and identification of new protein-coding loci. We present a novel pipeline that integrates highly sensitive and statistically robust peptide spectrum matching with genome-wide protein-coding predictions to perform large-scale gene validation and discovery in the mouse genome for the first time. In searching an excess of 10 million spectra, we have been able to validate 32%, 17%, and 7% of all protein-coding genes, exons, and splice boundaries, respectively. Moreover, we present strong evidence for the identification of multiple alternatively spliced translations from 53 genes and have uncovered 10 entirely novel protein-coding genes, which are not covered in any mouse annotation data sources. One such novel protein-coding gene is a fusion protein that spans the Ins2 and Igf2 loci to produce a transcript encoding the insulin II and the insulin-like growth factor 2-derived peptides. We also report nine processed pseudogenes that have unique peptide hits, demonstrating, for the first time, that they are not just transcribed but are translated and are therefore resurrected into new coding loci. This work not only highlights an important utility for MS data in genome annotation but also provides unique insights into the gene structure and propagation in the mouse genome. All these data have been subsequently used to improve the publicly available mouse annotation available in both the Vega and Ensembl genome browsers (http://vega.sanger.ac.uk).</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">5</span>;<span class="pagerange">756-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21460061">21460061</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3083093">3083093</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.114272.110">10.1101/gr.114272.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A missense mutation in Fgfr1 causes ear and skull defects in hush puppy mice.</h4>
<p class="authors"><span class="author">Calvert JA</span>, <span class="author">Dedos SG</span>, <span class="author">Hawker K</span>, <span class="author">Fleming M</span>, <span class="author">Lewis MA</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The hush puppy mouse mutant has been shown previously to have skull and outer, middle, and inner ear defects, and an increase in hearing threshold. The fibroblast growth factor receptor 1 (Fgfr1) gene is located in the region of chromosome 8 containing the mutation. Sequencing of the gene in hush puppy heterozygotes revealed a missense mutation in the kinase domain of the protein (W691R). Homozygotes were found to die during development, at approximately embryonic day 8.5, and displayed a phenotype similar to null mutants. Reverse transcription PCR indicated a decrease in Fgfr1 transcript in heterozygotes and homozygotes. Generation of a construct containing the mutation allowed the function of the mutated receptor to be studied. Immunocytochemistry showed that the mutant receptor protein was present at the cell membrane, suggesting normal expression and trafficking. Measurements of changes in intracellular calcium concentration showed that the mutated receptor could not activate the IP(3) pathway, in contrast to the wild-type receptor, nor could it initiate activation of the Ras/MAP kinase pathway. Thus, the hush puppy mutation in fibroblast growth factor receptor 1 appears to cause a loss of receptor function. The mutant protein appears to have a dominant negative effect, which could be due to it dimerising with the wild-type protein and inhibiting its activity, thus further reducing the levels of functional protein. A dominant modifier, Mhspy, which reduces the effect of the hush puppy mutation on pinna and stapes development, has been mapped to the distal end of chromosome 7 and may show imprinting.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">072084</span>, <span class="grant">077189</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2011</span>;<span class="volume">22</span>;<span class="number">5-6</span>;<span class="pagerange">290-305</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21479780" rel="external">21479780</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3099004" rel="external">3099004</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-011-9324-8" rel="external">10.1007/s00335-011-9324-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel role for PSD-95 in mediating ethanol intoxication, drinking and place preference.</h4>
<p class="authors"><span class="author">Camp MC</span>, <span class="author">Feyder M</span>, <span class="author">Ihne J</span>, <span class="author">Palachick B</span>, <span class="author">Hurd B</span>, <span class="author">Karlsson RM</span>, <span class="author">Noronha B</span>, <span class="author">Chen YC</span>, <span class="author">Coba MP</span>, <span class="author">Grant SG</span> and <span class="author">Holmes A</span></p>
<p class="affiliation">Section on Behavioral Science and Genetics, Laboratory for Integrative Neuroscience, National Institute on Alcohol Abuse and Alcoholism/NIH, 5625 Fishers Ln., Rockville, MD 20852-1798, USA. <a href="mailto:%63%61%6D%70%6D%63%40%6D%61%69%6C.%6E%69%68.%67%6F%76">&#99;&#97;&#109;&#112;&#109;&#99;&#64;&#109;&#97;&#105;&#108;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">The synaptic signaling mechanisms mediating the behavioral effects of ethanol (EtOH) remain poorly understood. Post-synaptic density 95 (PSD-95, SAP-90, Dlg4) is a key orchestrator of N-methyl-D-aspartate receptors (NMDAR) and glutamatergic synapses, which are known to be major sites of EtOH&#39;s behavioral actions. However, the potential contribution of PSD-95 to EtOH-related behaviors has not been established. Here, we evaluated knockout (KO) mice lacking PSD-95 for multiple measures of sensitivity to the acute intoxicating effects of EtOH (ataxia, hypothermia, sedation/hypnosis), EtOH drinking under conditions of free access and following deprivation, acquisition and long-term retention of EtOH conditioned place preference (CPP) (and lithium chloride-induced conditioned taste aversion), and intoxication-potentiating responses to NMDAR antagonism. PSD-95 KO exhibited increased sensitivity to the sedative/hypnotic, but not ataxic or hypothermic, effects of acute EtOH relative to wild-type controls (WT). PSD-95 KO consumed less EtOH than WT, particularly at higher EtOH concentrations, although increases in KO drinking could be induced by concentration-fading and deprivation. PSD-95 KO showed normal EtOH CPP 1 day after conditioning, but showed significant aversion 2 weeks later. Lithium chloride-induced taste aversion was impaired in PSD-95 KO at both time points. Finally, the EtOH-potentiating effects of the NMDAR antagonist MK-801 were intact in PSD-95 KO at the dose tested. These data reveal a major, novel role for PSD-95 in mediating EtOH behaviors, and add to growing evidence that PSD-95 is a key mediator of the effects of multiple abused drugs.</p>
<p class="grants">Funded by: NIAAA NIH HHS: <span class="grant">Z01-AA000411</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Addiction biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">16</span>;<span class="number">3</span>;<span class="pagerange">428-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21309945" rel="external">21309945</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3150485" rel="external">3150485</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1369-1600.2010.00282.x" rel="external">10.1111/j.1369-1600.2010.00282.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genetic analysis of Plasmodium falciparum parasites using a customized Illumina GoldenGate genotyping assay.</h4>
<p class="authors"><span class="author">Campino S</span>, <span class="author">Auburn S</span>, <span class="author">Kivinen K</span>, <span class="author">Zongo I</span>, <span class="author">Ouedraogo JB</span>, <span class="author">Mangano V</span>, <span class="author">Djimde A</span>, <span class="author">Doumbo OK</span>, <span class="author">Kiara SM</span>, <span class="author">Nzila A</span>, <span class="author">Borrmann S</span>, <span class="author">Marsh K</span>, <span class="author">Michon P</span>, <span class="author">Mueller I</span>, <span class="author">Siba P</span>, <span class="author">Jiang H</span>, <span class="author">Su XZ</span>, <span class="author">Amaratunga C</span>, <span class="author">Socheat D</span>, <span class="author">Fairhurst RM</span>, <span class="author">Imwong M</span>, <span class="author">Anderson T</span>, <span class="author">Nosten F</span>, <span class="author">White NJ</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">MacInnis B</span>, <span class="author">Newbold CI</span>, <span class="author">Rockett K</span>, <span class="author">Clark TG</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom. <a href="mailto:%73%63%31%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#99;&#49;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The diversity in the Plasmodium falciparum genome can be used to explore parasite population dynamics, with practical applications to malaria control. The ability to identify the geographic origin and trace the migratory patterns of parasites with clinically important phenotypes such as drug resistance is particularly relevant. With increasing single-nucleotide polymorphism (SNP) discovery from ongoing Plasmodium genome sequencing projects, a demand for high SNP and sample throughput genotyping platforms for large-scale population genetic studies is required. Low parasitaemias and multiple clone infections present a number of challenges to genotyping P. falciparum. We addressed some of these issues using a custom 384-SNP Illumina GoldenGate assay on P. falciparum DNA from laboratory clones (long-term cultured adapted parasite clones), short-term cultured parasite isolates and clinical (non-cultured isolates) samples from East and West Africa, Southeast Asia and Oceania. Eighty percent of the SNPs (n&#x200A;=&#x200A;306) produced reliable genotype calls on samples containing as little as 2 ng of total genomic DNA and on whole genome amplified DNA. Analysis of artificial mixtures of laboratory clones demonstrated high genotype calling specificity and moderate sensitivity to call minor frequency alleles. Clear resolution of geographically distinct populations was demonstrated using Principal Components Analysis (PCA), and global patterns of population genetic diversity were consistent with previous reports. These results validate the utility of the platform in performing population genetic studies of P. falciparum.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; NIAID NIH HHS: <span class="grant">R37 AI048071</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">093956</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">e20251</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21673999">21673999</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3108946">3108946</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0020251">10.1371/journal.pone.0020251</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differences between Trypanosoma brucei gambiense groups 1 and 2 in their resistance to killing by trypanolytic factor 1.</h4>
<p class="authors"><span class="author">Capewell P</span>, <span class="author">Veitch NJ</span>, <span class="author">Turner CM</span>, <span class="author">Raper J</span>, <span class="author">Berriman M</span>, <span class="author">Hajduk SL</span> and <span class="author">MacLeod A</span></p>
<p class="affiliation">College of Medical, Veterinary and Biological Sciences, Wellcome Trust Centre for Molecular Parasitology, University of Glasgow, Glasgow, United Kingdom.</p><p class="abstract"><strong>Background</strong>: The three sub-species of Trypanosoma brucei are important pathogens of sub-Saharan Africa. T. b. brucei is unable to infect humans due to sensitivity to trypanosome lytic factors (TLF) 1 and 2 found in human serum. T. b. rhodesiense and T. b. gambiense are able to resist lysis by TLF. There are two distinct sub-groups of T. b. gambiense that differ genetically and by human serum resistance phenotypes. Group 1 T. b. gambiense have an invariant phenotype whereas group 2 show variable resistance. Previous data indicated that group 1 T. b. gambiense are resistant to TLF-1 due in-part to reduced uptake of TLF-1 mediated by reduced expression of the TLF-1 receptor (the haptoglobin-hemoglobin receptor (HpHbR)) gene. Here we investigate if this is also true in group 2 parasites.</p><p class="abstract"><strong>Methodology</strong>: Isogenic resistant and sensitive group 2 T. b. gambiense were derived and compared to other T. brucei parasites. Both resistant and sensitive lines express the HpHbR gene at similar levels and internalized fluorescently labeled TLF-1 similar fashion to T. b. brucei. Both resistant and sensitive group 2, as well as group 1 T. b. gambiense, internalize recombinant APOL1, but only sensitive group 2 parasites are lysed.</p><p class="abstract"><strong>Conclusions</strong>: Our data indicate that, despite group 1 T. b. gambiense avoiding TLF-1, it is resistant to the main lytic component, APOL1. Similarly group 2 T. b. gambiense is innately resistant to APOL1, which could be based on the same mechanism. However, group 2 T. b. gambiense variably displays this phenotype and expression does not appear to correlate with a change in expression site or expression of HpHbR. Thus there are differences in the mechanism of human serum resistance between T. b. gambiense groups 1 and 2.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; NIAID NIH HHS: <span class="grant">AI039033</span>, <span class="grant">AI041233</span>; Wellcome Trust: <span class="grant">079703</span>, <span class="grant">095201</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">9</span>;<span class="pagerange">e1287</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21909441" rel="external">21909441</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3167774" rel="external">3167774</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001287" rel="external">10.1371/journal.pntd.0001287</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Determinants of bluetongue virus virulence in murine models of disease.</h4>
<p class="authors"><span class="author">Caporale M</span>, <span class="author">Wash R</span>, <span class="author">Pini A</span>, <span class="author">Savini G</span>, <span class="author">Franchi P</span>, <span class="author">Golder M</span>, <span class="author">Patterson-Kane J</span>, <span class="author">Mertens P</span>, <span class="author">Di Gialleonardo L</span>, <span class="author">Armillotta G</span>, <span class="author">Lelli R</span>, <span class="author">Kellam P</span> and <span class="author">Palmarini M</span></p>
<p class="affiliation">Medical Research Council-University of Glasgow Centre for Virus Research, Institute of Infection, Inflammation and Immunity, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, United Kingdom.</p><p class="abstract">Bluetongue is a major infectious disease of ruminants that is caused by bluetongue virus (BTV). In this study, we analyzed virulence and genetic differences of (i) three BTV field strains from Italy maintained at either a low (L strains) or high (H strains) passage number in cell culture and (ii) three South African &quot;reference&quot; wild-type strains and their corresponding live attenuated vaccine strains. The Italian BTV L strains, in general, were lethal for both newborn NIH-Swiss mice inoculated intracerebrally and adult type I interferon receptor-deficient (IFNAR(-/-)) mice, while the virulence of the H strains was attenuated significantly in both experimental models. Similarly, the South African vaccine strains were not pathogenic for IFNAR(-/-) mice, while the corresponding wild-type strains were virulent. Thus, attenuation of the virulence of the BTV strains used in this study is not mediated by the presence of an intact interferon system. No clear distinction in virulence was observed for the South African BTV strains in newborn NIH-Swiss mice. Full genomic sequencing revealed relatively few amino acid substitutions, scattered in several different viral proteins, for the strains found to be attenuated in mice compared to the pathogenic related strains. However, only the genome segments encoding VP1, VP2, and NS2 consistently showed nonsynonymous changes between all virulent and attenuated strain pairs. This study established an experimental platform for investigating the determinants of BTV virulence. Future studies using reverse genetics will allow researchers to precisely map and &quot;weight&quot; the relative influences of the various genome segments and viral proteins on BTV virulence.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801822</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2011</span>;<span class="volume">85</span>;<span class="number">21</span>;<span class="pagerange">11479-89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21865388">21865388</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3194974">3194974</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.05226-11">10.1128/JVI.05226-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microbial sequences benefit health now.</h4>
<p class="authors"><span class="author">Cartwright EJ</span>, <span class="author">K&ouml;ser CU</span> and <span class="author">Peacock SJ</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">471</span>;<span class="number">7340</span>;<span class="pagerange">578</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21455159" rel="external">21455159</a>; DOI: <a href="http://dx.doi.org/10.1038%2F471578d" rel="external">10.1038/471578d</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A modified vaccinia Ankara virus (MVA) vaccine expressing African horse sickness virus (AHSV) VP2 protects against AHSV challenge in an IFNAR -/- mouse model.</h4>
<p class="authors"><span class="author">Castillo-Olivares J</span>, <span class="author">Calvo-Pinilla E</span>, <span class="author">Casanova I</span>, <span class="author">Bachanek-Bankowska K</span>, <span class="author">Chiam R</span>, <span class="author">Maan S</span>, <span class="author">Nieto JM</span>, <span class="author">Ortego J</span> and <span class="author">Mertens PP</span></p>
<p class="affiliation">Institute for Animal Health, Pirbright, Woking, Surrey, United Kingdom. <a href="mailto:%6A%61%76%69%65%72.%63%61%73%74%69%6C%6C%6F-%6F%6C%69%76%61%72%65%73%40%62%62%73%72%63.%61%63.%75%6B">&#106;&#97;&#118;&#105;&#101;&#114;&#46;&#99;&#97;&#115;&#116;&#105;&#108;&#108;&#111;&#45;&#111;&#108;&#105;&#118;&#97;&#114;&#101;&#115;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">African horse sickness (AHS) is a lethal viral disease of equids, which is transmitted by Culicoides midges that become infected after biting a viraemic host. The use of live attenuated vaccines has been vital for the control of this disease in endemic regions. However, there are safety concerns over their use in non-endemic countries. Research efforts over the last two decades have therefore focused on developing alternative vaccines based on recombinant baculovirus or live viral vectors expressing structural components of the AHS virion. However, ethical and financial considerations, relating to the use of infected horses in high biosecurity installations, have made progress very slow. We have therefore assessed the potential of an experimental mouse-model for AHSV infection for vaccine and immunology research. We initially characterised AHSV infection in this model, then tested the protective efficacy of a recombinant vaccine based on modified vaccinia Ankara expressing AHS-4 VP2 (MVA-VP2).</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/B/00654</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">e16503</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21298069">21298069</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3027694">3027694</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0016503">10.1371/journal.pone.0016503</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The impact of recombination on dN/dS within recently emerged bacterial clones.</h4>
<p class="authors"><span class="author">Castillo-Ram&iacute;rez S</span>, <span class="author">Harris SR</span>, <span class="author">Holden MT</span>, <span class="author">He M</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span> and <span class="author">Feil EJ</span></p>
<p class="affiliation">Department of Biology and Biochemistry, University of Bath, Claverton Down, Bath, United Kingdom.</p><p class="abstract">The development of next-generation sequencing platforms is set to reveal an unprecedented level of detail on short-term molecular evolutionary processes in bacteria. Here we re-analyse genome-wide single nucleotide polymorphism (SNP) datasets for recently emerged clones of methicillin resistant Staphylococcus aureus (MRSA) and Clostridium difficile. We note a highly significant enrichment of synonymous SNPs in those genes which have been affected by recombination, i.e. those genes on mobile elements designated &quot;non-core&quot; (in the case of S. aureus), or those core genes which have been affected by homologous replacements (S. aureus and C. difficile). This observation suggests that the previously documented decrease in dN/dS over time in bacteria applies not only to genomes of differing levels of divergence overall, but also to horizontally acquired genes of differing levels of divergence within a single genome. We also consider the role of increased drift acting on recently emerged, highly specialised clones, and the impact of recombination on selection at linked sites. This work has implications for a wide range of genomic analyses.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e1002129</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21779170">21779170</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3136474">3136474</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002129">10.1371/journal.ppat.1002129</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies loci influencing concentrations of liver enzymes in plasma.</h4>
<p class="authors"><span class="author">Chambers JC</span>, <span class="author">Zhang W</span>, <span class="author">Sehmi J</span>, <span class="author">Li X</span>, <span class="author">Wass MN</span>, <span class="author">Van der Harst P</span>, <span class="author">Holm H</span>, <span class="author">Sanna S</span>, <span class="author">Kavousi M</span>, <span class="author">Baumeister SE</span>, <span class="author">Coin LJ</span>, <span class="author">Deng G</span>, <span class="author">Gieger C</span>, <span class="author">Heard-Costa NL</span>, <span class="author">Hottenga JJ</span>, <span class="author">K&uuml;hnel B</span>, <span class="author">Kumar V</span>, <span class="author">Lagou V</span>, <span class="author">Liang L</span>, <span class="author">Luan J</span>, <span class="author">Vidal PM</span>, <span class="author">Mateo Leach I</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Peden JF</span>, <span class="author">Rahmioglu N</span>, <span class="author">Soininen P</span>, <span class="author">Speliotes EK</span>, <span class="author">Yuan X</span>, <span class="author">Thorleifsson G</span>, <span class="author">Alizadeh BZ</span>, <span class="author">Atwood LD</span>, <span class="author">Borecki IB</span>, <span class="author">Brown MJ</span>, <span class="author">Charoen P</span>, <span class="author">Cucca F</span>, <span class="author">Das D</span>, <span class="author">de Geus EJ</span>, <span class="author">Dixon AL</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Ehret G</span>, <span class="author">Eyjolfsson GI</span>, <span class="author">Farrall M</span>, <span class="author">Forouhi NG</span>, <span class="author">Friedrich N</span>, <span class="author">Goessling W</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Harris TB</span>, <span class="author">Hartikainen AL</span>, <span class="author">Heath S</span>, <span class="author">Hirschfield GM</span>, <span class="author">Hofman A</span>, <span class="author">Homuth G</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Janssen HL</span>, <span class="author">Johnson T</span>, <span class="author">Kangas AJ</span>, <span class="author">Kema IP</span>, <span class="author">K&uuml;hn JP</span>, <span class="author">Lai S</span>, <span class="author">Lathrop M</span>, <span class="author">Lerch MM</span>, <span class="author">Li Y</span>, <span class="author">Liang TJ</span>, <span class="author">Lin JP</span>, <span class="author">Loos RJ</span>, <span class="author">Martin NG</span>, <span class="author">Moffatt MF</span>, <span class="author">Montgomery GW</span>, <span class="author">Munroe PB</span>, <span class="author">Musunuru K</span>, <span class="author">Nakamura Y</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Olafsson I</span>, <span class="author">Penninx BW</span>, <span class="author">Pouta A</span>, <span class="author">Prins BP</span>, <span class="author">Prokopenko I</span>, <span class="author">Puls R</span>, <span class="author">Ruokonen A</span>, <span class="author">Savolainen MJ</span>, <span class="author">Schlessinger D</span>, <span class="author">Schouten JN</span>, <span class="author">Seedorf U</span>, <span class="author">Sen-Chowdhry S</span>, <span class="author">Siminovitch KA</span>, <span class="author">Smit JH</span>, <span class="author">Spector TD</span>, <span class="author">Tan W</span>, <span class="author">Teslovich TM</span>, <span class="author">Tukiainen T</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Van der Klauw MM</span>, <span class="author">Vasan RS</span>, <span class="author">Wallace C</span>, <span class="author">Wallaschofski H</span>, <span class="author">Wichmann HE</span>, <span class="author">Willemsen G</span>, <span class="author">W&uuml;rtz P</span>, <span class="author">Xu C</span>, <span class="author">Yerges-Armstrong LM</span>, <span class="author">Alcohol Genome-wide Association (AlcGen) Consortium</span>, <span class="author">Diabetes Genetics Replication and Meta-analyses (DIAGRAM+) Study</span>, <span class="author">Genetic Investigation of Anthropometric Traits (GIANT) Consortium</span>, <span class="author">Global Lipids Genetics Consortium</span>, <span class="author">Genetics of Liver Disease (GOLD) Consortium</span>, <span class="author">International Consortium for Blood Pressure (ICBP-GWAS)</span>, <span class="author">Meta-analyses of Glucose and Insulin-Related Traits Consortium (MAGIC)</span>, <span class="author">Abecasis GR</span>, <span class="author">Ahmadi KR</span>, <span class="author">Boomsma DI</span>, <span class="author">Caulfield M</span>, <span class="author">Cookson WO</span>, <span class="author">van Duijn CM</span>, <span class="author">Froguel P</span>, <span class="author">Matsuda K</span>, <span class="author">McCarthy MI</span>, <span class="author">Meisinger C</span>, <span class="author">Mooser V</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Schumann G</span>, <span class="author">Snieder H</span>, <span class="author">Sternberg MJ</span>, <span class="author">Stolk RP</span>, <span class="author">Thomas HC</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Uda M</span>, <span class="author">Waeber G</span>, <span class="author">Wareham NJ</span>, <span class="author">Waterworth DM</span>, <span class="author">Watkins H</span>, <span class="author">Whitfield JB</span>, <span class="author">Witteman JC</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Fox CS</span>, <span class="author">Ala-Korpela M</span>, <span class="author">Stefansson K</span>, <span class="author">Vollenweider P</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Schadt EE</span>, <span class="author">Scott J</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Elliott P</span> and <span class="author">Kooner JS</span></p>
<p class="affiliation">Epidemiology and Biostatistics, Imperial College London, Norfolk Place, London, UK. <a href="mailto:%6A%6F%68%6E.%63%68%61%6D%62%65%72%73%40%69%63.%61%63.%75%6B">&#106;&#111;&#104;&#110;&#46;&#99;&#104;&#97;&#109;&#98;&#101;&#114;&#115;&#64;&#105;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Concentrations of liver enzymes in plasma are widely used as indicators of liver disease. We carried out a genome-wide association study in 61,089 individuals, identifying 42 loci associated with concentrations of liver enzymes in plasma, of which 32 are new associations (P = 10(-8) to P = 10(-190)). We used functional genomic approaches including metabonomic profiling and gene expression analyses to identify probable candidate genes at these regions. We identified 69 candidate genes, including genes involved in biliary transport (ATP8B1 and ABCB11), glucose, carbohydrate and lipid metabolism (FADS1, FADS2, GCKR, JMJD1C, HNF1A, MLXIPL, PNPLA3, PPP1R3B, SLC2A2 and TRIB1), glycoprotein biosynthesis and cell surface glycobiology (ABO, ASGR1, FUT2, GPLD1 and ST3GAL4), inflammation and immunity (CD276, CDH6, GCKR, HNF1A, HPR, ITGA1, RORA and STAT4) and glutathione metabolism (GSTT1, GSTT2 and GGT), as well as several genes of uncertain or unknown function (including ABHD12, EFHD1, EFNA1, EPHA2, MICAL3 and ZNF827). Our results provide new insight into genetic mechanisms and pathways influencing markers of liver function.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/10/011/27881</span>, <span class="grant">PG/09/002/26056</span>, <span class="grant">PG/09/023/26806</span>, <span class="grant">RG/07/008/23674</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0601653</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">G9521010</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1015</span>; NHLBI NIH HHS: <span class="grant">R01 HL087647</span>; NIAAA NIH HHS: <span class="grant">K05 AA017688</span>; NIDDK NIH HHS: <span class="grant">Z99 DK999999</span>, <span class="grant">ZIA DK075013-05</span>, <span class="grant">ZIA DK075013-07</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">11</span>;<span class="pagerange">1131-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22001757">22001757</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3482372">3482372</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.970">10.1038/ng.970</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants show predicted polygenic effects on height in the tails of the distribution, except in extremely short individuals.</h4>
<p class="authors"><span class="author">Chan Y</span>, <span class="author">Holmen OL</span>, <span class="author">Dauber A</span>, <span class="author">Vatten L</span>, <span class="author">Havulinna AS</span>, <span class="author">Skorpen F</span>, <span class="author">Kval&oslash;y K</span>, <span class="author">Silander K</span>, <span class="author">Nguyen TT</span>, <span class="author">Willer C</span>, <span class="author">Boehnke M</span>, <span class="author">Perola M</span>, <span class="author">Palotie A</span>, <span class="author">Salomaa V</span>, <span class="author">Hveem K</span>, <span class="author">Frayling TM</span>, <span class="author">Hirschhorn JN</span> and <span class="author">Weedon MN</span></p>
<p class="affiliation">Department of Genetics, Harvard Medical School, Boston, Massachusetts, United States of America.</p><p class="abstract">Common genetic variants have been shown to explain a fraction of the inherited variation for many common diseases and quantitative traits, including height, a classic polygenic trait. The extent to which common variation determines the phenotype of highly heritable traits such as height is uncertain, as is the extent to which common variation is relevant to individuals with more extreme phenotypes. To address these questions, we studied 1,214 individuals from the top and bottom extremes of the height distribution (tallest and shortest &sim;1.5%), drawn from &sim;78,000 individuals from the HUNT and FINRISK cohorts. We found that common variants still influence height at the extremes of the distribution: common variants (49/141) were nominally associated with height in the expected direction more often than is expected by chance (p&lt;5&times;10&#x207B;&sup2;&#x2078;), and the odds ratios in the extreme samples were consistent with the effects estimated previously in population-based data. To examine more closely whether the common variants have the expected effects, we calculated a weighted allele score (WAS), which is a weighted prediction of height for each individual based on the previously estimated effect sizes of the common variants in the overall population. The average WAS is consistent with expectation in the tall individuals, but was not as extreme as expected in the shortest individuals (p&lt;0.006), indicating that some of the short stature is explained by factors other than common genetic variation. The discrepancy was more pronounced (p&lt;10&#x207B;&#x2076;) in the most extreme individuals (height&lt;0.25 percentile). The results at the extreme short tails are consistent with a large number of models incorporating either rare genetic non-additive or rare non-genetic factors that decrease height. We conclude that common genetic variants are associated with height at the extremes as well as across the population, but that additional factors become more prominent at the shorter extreme.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">1R01DK075787</span>; Wellcome Trust: <span class="grant">085301/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e1002439</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22242009" rel="external">22242009</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3248463" rel="external">3248463</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002439" rel="external">10.1371/journal.pgen.1002439</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antimicrobial resistance to ceftazidime involving loss of penicillin-binding protein 3 in Burkholderia pseudomallei.</h4>
<p class="authors"><span class="author">Chantratita N</span>, <span class="author">Rholl DA</span>, <span class="author">Sim B</span>, <span class="author">Wuthiekanun V</span>, <span class="author">Limmathurotsakul D</span>, <span class="author">Amornchai P</span>, <span class="author">Thanwisai A</span>, <span class="author">Chua HH</span>, <span class="author">Ooi WF</span>, <span class="author">Holden MT</span>, <span class="author">Day NP</span>, <span class="author">Tan P</span>, <span class="author">Schweizer HP</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Department of Microbiology and Immunology, Faculty of Tropical Medicine, Mahidol University, Bangkok 10400, Thailand.</p><p class="abstract">Known mechanisms of resistance to &beta;-lactam antibiotics include &beta;-lactamase expression, altered drug target, decreased bacterial permeability, and increased drug efflux. Here, we describe a unique mechanism of &beta;-lactam resistance in the biothreat organism Burkholderia pseudomallei (the cause of melioidosis), associated with treatment failure during prolonged ceftazidime therapy of natural infection. Detailed comparisons of the initial ceftazidime-susceptible infecting isolate and subsequent ceftazidime-resistant variants from six patients led us to identify a common, large-scale genomic loss involving a minimum of 49 genes in all six resistant strains. Mutational analysis of wild-type B. pseudomallei demonstrated that ceftazidime resistance was due to deletion of a gene encoding a penicillin-binding protein 3 (BPSS1219) present within the region of genomic loss. The clinical ceftazidime-resistant variants failed to grow using commonly used laboratory culture media, including commercial blood cultures, rendering the variants almost undetectable in the diagnostic laboratory. Melioidosis is notoriously difficult to cure and clinical treatment failure is common in patients treated with ceftazidime, the drug of first choice across most of Southeast Asia where the majority of cases are reported. The mechanism described here represents an explanation for ceftazidime treatment failure, and may be a frequent but undetected resistance event.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI065357</span>; Wellcome Trust: <span class="grant">087769/Z/08/</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">41</span>;<span class="pagerange">17165-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21969582" rel="external">21969582</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3193241" rel="external">3193241</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1111020108" rel="external">10.1073/pnas.1111020108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Defining the power limits of genome-wide association scan meta-analyses.</h4>
<p class="authors"><span class="author">Chapman K</span>, <span class="author">Ferreira T</span>, <span class="author">Morris A</span>, <span class="author">Asimit J</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Roosevelt Drive, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">Large-scale meta-analyses of genome-wide association scans (GWAS) have been successful in discovering common risk variants with modest and small effects. The detection of lower frequency signals will undoubtedly require concerted efforts of at least similar scale. We investigate the sample size-dictated power limits of GWAS meta-analyses, in the presence and absence of modest levels of heterogeneity and across a range of different allelic architectures. We find that data combination through large-scale collaboration is vital in the quest for complex trait susceptibility loci, but that effect size heterogeneity across meta-analyzed studies drawn from similar populations does not appear to have a profound effect on sample size requirements.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088885</span>, <span class="grant">090532</span>, <span class="grant">WT079557MA</span>, <span class="grant">WT081682/Z/06/Z</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">35</span>;<span class="number">8</span>;<span class="pagerange">781-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21922540">21922540</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428938">3428938</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20627">10.1002/gepi.20627</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expressions of individuality.</h4>
<p class="authors"><span class="author">Chappell L</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">10</span>;<span class="pagerange">701</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21921932">21921932</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2662">10.1038/nrmicro2662</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study reveals three susceptibility loci for common migraine in the general population.</h4>
<p class="authors"><span class="author">Chasman DI</span>, <span class="author">Sch&uuml;rks M</span>, <span class="author">Anttila V</span>, <span class="author">de Vries B</span>, <span class="author">Schminke U</span>, <span class="author">Launer LJ</span>, <span class="author">Terwindt GM</span>, <span class="author">van den Maagdenberg AM</span>, <span class="author">Fendrich K</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Ernst F</span>, <span class="author">Griffiths LR</span>, <span class="author">Buring JE</span>, <span class="author">Kallela M</span>, <span class="author">Freilinger T</span>, <span class="author">Kubisch C</span>, <span class="author">Ridker PM</span>, <span class="author">Palotie A</span>, <span class="author">Ferrari MD</span>, <span class="author">Hoffmann W</span>, <span class="author">Zee RY</span> and <span class="author">Kurth T</span></p>
<p class="affiliation">Division of Preventive Medicine, Department of Medicine, Brigham and Women&#39;s Hospital, Harvard Medical School, Boston, Massachusetts, USA.</p><p class="abstract">Migraine is a common, heterogeneous and heritable neurological disorder. Its pathophysiology is incompletely understood, and its genetic influences at the population level are unknown. In a population-based genome-wide analysis including 5,122 migraineurs and 18,108 non-migraineurs, rs2651899 (1p36.32, PRDM16), rs10166942 (2q37.1, TRPM8) and rs11172113 (12q13.3, LRP1) were among the top seven associations (P &lt; 5 &times; 10(-6)) with migraine. These SNPs were significant in a meta-analysis among three replication cohorts and met genome-wide significance in a meta-analysis combining the discovery and replication cohorts (rs2651899, odds ratio (OR) = 1.11, P = 3.8 &times; 10(-9); rs10166942, OR = 0.85, P = 5.5 &times; 10(-12); and rs11172113, OR = 0.90, P = 4.3 &times; 10(-9)). The associations at rs2651899 and rs10166942 were specific for migraine compared with non-migraine headache. None of the three SNP associations was preferential for migraine with aura or without aura, nor were any associations specific for migraine features. TRPM8 has been the focus of neuropathic pain models, whereas LRP1 modulates neuronal glutamate signaling, plausibly linking both genes to migraine pathophysiology.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA-47988</span>, <span class="grant">R01 CA047988</span>, <span class="grant">R01 CA047988-21</span>; NHLBI NIH HHS: <span class="grant">HL-043851</span>, <span class="grant">HL-080467</span>, <span class="grant">HL-099355</span>, <span class="grant">R01 HL043851</span>, <span class="grant">R01 HL043851-10</span>, <span class="grant">R01 HL080467</span>, <span class="grant">R01 HL080467-05</span>, <span class="grant">RC1 HL099355</span>, <span class="grant">RC1 HL099355-02</span>; NINDS NIH HHS: <span class="grant">NS-061836</span>, <span class="grant">R01 NS061836</span>, <span class="grant">R01 NS061836-03</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">7</span>;<span class="pagerange">695-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21666692">21666692</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3125402">3125402</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.856">10.1038/ng.856</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genetic structure in Indian Austroasiatic speakers: the role of landscape barriers and sex-specific admixture.</h4>
<p class="authors"><span class="author">Chaubey G</span>, <span class="author">Metspalu M</span>, <span class="author">Choi Y</span>, <span class="author">M&auml;gi R</span>, <span class="author">Romero IG</span>, <span class="author">Soares P</span>, <span class="author">van Oven M</span>, <span class="author">Behar DM</span>, <span class="author">Rootsi S</span>, <span class="author">Hudjashov G</span>, <span class="author">Mallick CB</span>, <span class="author">Karmin M</span>, <span class="author">Nelis M</span>, <span class="author">Parik J</span>, <span class="author">Reddy AG</span>, <span class="author">Metspalu E</span>, <span class="author">van Driem G</span>, <span class="author">Xue Y</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Thangaraj K</span>, <span class="author">Singh L</span>, <span class="author">Remm M</span>, <span class="author">Richards MB</span>, <span class="author">Lahr MM</span>, <span class="author">Kayser M</span>, <span class="author">Villems R</span> and <span class="author">Kivisild T</span></p>
<p class="affiliation">Department of Evolutionary Biology, Institute of Molecular and Cell Biology, University of Tartu and Estonian Biocentre, Tartu, Estonia.</p><p class="abstract">The geographic origin and time of dispersal of Austroasiatic (AA) speakers, presently settled in south and southeast Asia, remains disputed. Two rival hypotheses, both assuming a demic component to the language dispersal, have been proposed. The first of these places the origin of Austroasiatic speakers in southeast Asia with a later dispersal to south Asia during the Neolithic, whereas the second hypothesis advocates pre-Neolithic origins and dispersal of this language family from south Asia. To test the two alternative models, this study combines the analysis of uniparentally inherited markers with 610,000 common single nucleotide polymorphism loci from the nuclear genome. Indian AA speakers have high frequencies of Y chromosome haplogroup O2a; our results show that this haplogroup has significantly higher diversity and coalescent time (17-28 thousand years ago) in southeast Asia, strongly supporting the first of the two hypotheses. Nevertheless, the results of principal component and &quot;structure-like&quot; analyses on autosomal loci also show that the population history of AA speakers in India is more complex, being characterized by two ancestral components-one represented in the pattern of Y chromosomal and EDAR results and the other by mitochondrial DNA diversity and genomic structure. We propose that AA speakers in India today are derived from dispersal from southeast Asia, followed by extensive sex-specific admixture with local Indian populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">28</span>;<span class="number">2</span>;<span class="pagerange">1013-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20978040">20978040</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3355372">3355372</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsq288">10.1093/molbev/msq288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quantitative RNA-seq analysis of the Campylobacter jejuni transcriptome.</h4>
<p class="authors"><span class="author">Chaudhuri RR</span>, <span class="author">Yu L</span>, <span class="author">Kanji A</span>, <span class="author">Perkins TT</span>, <span class="author">Gardner PP</span>, <span class="author">Choudhary J</span>, <span class="author">Maskell DJ</span> and <span class="author">Grant AJ</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, UK.</p><p class="abstract">Campylobacter jejuni is the most common bacterial cause of foodborne disease in the developed world. Its general physiology and biochemistry, as well as the mechanisms enabling it to colonize and cause disease in various hosts, are not well understood, and new approaches are required to understand its basic biology. High-throughput sequencing technologies provide unprecedented opportunities for functional genomic research. Recent studies have shown that direct Illumina sequencing of cDNA (RNA-seq) is a useful technique for the quantitative and qualitative examination of transcriptomes. In this study we report RNA-seq analyses of the transcriptomes of C. jejuni (NCTC11168) and its rpoN mutant. This has allowed the identification of hitherto unknown transcriptional units, and further defines the regulon that is dependent on rpoN for expression. The analysis of the NCTC11168 transcriptome was supplemented by additional proteomic analysis using liquid chromatography-MS. The transcriptomic and proteomic datasets represent an important resource for the Campylobacter research community.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801161</span>; Wellcome Trust: <span class="grant">079643/Z/06/Z</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2011</span>;<span class="volume">157</span>;<span class="number">Pt 10</span>;<span class="pagerange">2922-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21816880">21816880</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3353397">3353397</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.050278-0">10.1099/mic.0.050278-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic screens using the piggyBac transposon.</h4>
<p class="authors"><span class="author">Chew SK</span>, <span class="author">Rad R</span>, <span class="author">Futreal PA</span>, <span class="author">Bradley A</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Transposons are an attractive system to use in genetic screens as they are molecularly tractable and the disrupted loci that give rise to the desired phenotype are easily mapped. We consider herein the characteristics of the piggyBac transposon system in complementing existing mammalian screen strategies, including the Sleeping Beauty transposon system. We also describe the design of the piggyBac resources that we have developed for both forward and reverse genetic screens, and the protocols we use in these experiments.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Methods (San Diego, Calif.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">53</span>;<span class="number">4</span>;<span class="pagerange">366-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21185377">21185377</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3650578">3650578</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ymeth.2010.12.022">10.1016/j.ymeth.2010.12.022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Implementing a genomic data management system using iRODS in the Wellcome Trust Sanger Institute.</h4>
<p class="authors"><span class="author">Chiang GT</span>, <span class="author">Clapham P</span>, <span class="author">Qi G</span>, <span class="author">Sale K</span> and <span class="author">Coates G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Informatics System Group, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK. <a href="mailto:%67%74%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#116;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Increasingly large amounts of DNA sequencing data are being generated within the Wellcome Trust Sanger Institute (WTSI). The traditional file system struggles to handle these increasing amounts of sequence data. A good data management system therefore needs to be implemented and integrated into the current WTSI infrastructure. Such a system enables good management of the IT infrastructure of the sequencing pipeline and allows biologists to track their data.</p><p class="abstract"><strong>Results</strong>: We have chosen a data grid system, iRODS (Rule-Oriented Data management systems), to act as the data management system for the WTSI. iRODS provides a rule-based system management approach which makes data replication much easier and provides extra data protection. Unlike the metadata provided by traditional file systems, the metadata system of iRODS is comprehensive and allows users to customize their own application level metadata. Users and IT experts in the WTSI can then query the metadata to find and track data.The aim of this paper is to describe how we designed and used (from both system and user viewpoints) iRODS as a data management system. Details are given about the problems faced and the solutions found when iRODS was implemented. A simple use case describing how users within the WTSI use iRODS is also introduced.</p><p class="abstract"><strong>Conclusions</strong>: iRODS has been implemented and works as the production system for the sequencing pipeline of the WTSI. Both biologists and IT experts can now track and manage data, which could not previously be achieved. This novel approach allows biologists to define their own metadata and query the genomic data using those metadata.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="pagerange">361</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21906284" rel="external">21906284</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3228552" rel="external">3228552</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-12-361" rel="external">10.1186/1471-2105-12-361</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer genomics: from discovery science to personalized medicine.</h4>
<p class="authors"><span class="author">Chin L</span>, <span class="author">Andersen JN</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Belfer Institute for Applied Cancer Science, Dana-Farber Cancer Institute, Boston, Massachusetts, USA. <a href="mailto:%6C%79%6E%64%61%5F%63%68%69%6E%40%64%66%63%69.%68%61%72%76%61%72%64.%65%64%75">&#108;&#121;&#110;&#100;&#97;&#95;&#99;&#104;&#105;&#110;&#64;&#100;&#102;&#99;&#105;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Recent advances in genome technologies and the ensuing outpouring of genomic information related to cancer have accelerated the convergence of discovery science and clinical medicine. Successful examples of translating cancer genomics into therapeutics and diagnostics reinforce its potential to make possible personalized cancer medicine. However, the bottlenecks along the path of converting a genome discovery into a tangible clinical endpoint are numerous and formidable. In this Perspective, we emphasize the importance of establishing the biological relevance of a cancer genomic discovery in realizing its clinical potential and discuss some of the major obstacles to moving from the bench to the bedside.</p>
<p class="publication"><span class="title">Nature medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">297-303</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21383744" rel="external">21383744</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnm.2323" rel="external">10.1038/nm.2323</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A deep sequencing approach to comparatively analyze the transcriptome of lifecycle stages of the filarial worm, Brugia malayi.</h4>
<p class="authors"><span class="author">Choi YJ</span>, <span class="author">Ghedin E</span>, <span class="author">Berriman M</span>, <span class="author">McQuillan J</span>, <span class="author">Holroyd N</span>, <span class="author">Mayhew GF</span>, <span class="author">Christensen BM</span> and <span class="author">Michalski ML</span></p>
<p class="affiliation">Department of Pathobiological Sciences, University of Wisconsin-Madison, Madison, Wisconsin, USA.</p><p class="abstract"><strong>Background</strong>: Developing intervention strategies for the control of parasitic nematodes continues to be a significant challenge. Genomic and post-genomic approaches play an increasingly important role for providing fundamental molecular information about these parasites, thus enhancing basic as well as translational research. Here we report a comprehensive genome-wide survey of the developmental transcriptome of the human filarial parasite Brugia malayi.</p><p class="abstract">Using deep sequencing, we profiled the transcriptome of eggs and embryos, immature (&le;3 days of age) and mature microfilariae (MF), third- and fourth-stage larvae (L3 and L4), and adult male and female worms. Comparative analysis across these stages provided a detailed overview of the molecular repertoires that define and differentiate distinct lifecycle stages of the parasite. Genome-wide assessment of the overall transcriptional variability indicated that the cuticle collagen family and those implicated in molting exhibit noticeably dynamic stage-dependent patterns. Of particular interest was the identification of genes displaying sex-biased or germline-enriched profiles due to their potential involvement in reproductive processes. The study also revealed discrete transcriptional changes during larval development, namely those accompanying the maturation of MF and the L3 to L4 transition that are vital in establishing successful infection in mosquito vectors and vertebrate hosts, respectively.</p><p class="abstract">Characterization of the transcriptional program of the parasite&#39;s lifecycle is an important step toward understanding the developmental processes required for the infectious cycle. We find that the transcriptional program has a number of stage-specific pathways activated during worm development. In addition to advancing our understanding of transcriptome dynamics, these data will aid in the study of genome structure and organization by facilitating the identification of novel transcribed elements and splice variants.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI019769</span>, <span class="grant">AI067295</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">12</span>;<span class="pagerange">e1409</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22180794" rel="external">22180794</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3236722" rel="external">3236722</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001409" rel="external">10.1371/journal.pntd.0001409</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modernizing reference genome assemblies.</h4>
<p class="authors"><span class="author">Church DM</span>, <span class="author">Schneider VA</span>, <span class="author">Graves T</span>, <span class="author">Auger K</span>, <span class="author">Cunningham F</span>, <span class="author">Bouk N</span>, <span class="author">Chen HC</span>, <span class="author">Agarwala R</span>, <span class="author">McLaren WM</span>, <span class="author">Ritchie GR</span>, <span class="author">Albracht D</span>, <span class="author">Kremitzki M</span>, <span class="author">Rock S</span>, <span class="author">Kotkiewicz H</span>, <span class="author">Kremitzki C</span>, <span class="author">Wollam A</span>, <span class="author">Trani L</span>, <span class="author">Fulton L</span>, <span class="author">Fulton R</span>, <span class="author">Matthews L</span>, <span class="author">Whitehead S</span>, <span class="author">Chow W</span>, <span class="author">Torrance J</span>, <span class="author">Dunn M</span>, <span class="author">Harden G</span>, <span class="author">Threadgold G</span>, <span class="author">Wood J</span>, <span class="author">Collins J</span>, <span class="author">Heath P</span>, <span class="author">Griffiths G</span>, <span class="author">Pelan S</span>, <span class="author">Grafham D</span>, <span class="author">Eichler EE</span>, <span class="author">Weinstock G</span>, <span class="author">Mardis ER</span>, <span class="author">Wilson RK</span>, <span class="author">Howe K</span>, <span class="author">Flicek P</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America. <a href="mailto:%63%68%75%72%63%68%40%6E%63%62%69.%6E%6C%6D.%6E%69%68.%67%6F%76">&#99;&#104;&#117;&#114;&#99;&#104;&#64;&#110;&#99;&#98;&#105;&#46;&#110;&#108;&#109;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span>, <span class="grant">095908</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">7</span>;<span class="pagerange">e1001091</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21750661">21750661</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3130012">3130012</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1001091">10.1371/journal.pbio.1001091</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single nucleotide polymorphism (SNP) panels for rapid positional cloning in zebrafish.</h4>
<p class="authors"><span class="author">Clark MD</span>, <span class="author">Guryev V</span>, <span class="author">Bruijn Ed</span>, <span class="author">Nijman IJ</span>, <span class="author">Tada M</span>, <span class="author">Wilson C</span>, <span class="author">Deloukas P</span>, <span class="author">Postlethwait JH</span>, <span class="author">Cuppen E</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p>
<p class="abstract">Despite considerable genetic and genomic resources the positional cloning of forward mutations remains a slow and manually intensive task, typically using gel based genotyping and sequential rounds of mapping. We have used the latest genetic resources and genotyping technologies to develop two commercially available SNP panels of thousands of markers that can be used to speed up positional cloning.</p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">104</span>;<span class="pagerange">219-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21924166">21924166</a>; DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-374814-0.00013-6">10.1016/B978-0-12-374814-0.00013-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Basic statistical analysis in genetic case-control studies.</h4>
<p class="authors"><span class="author">Clarke GM</span>, <span class="author">Anderson CA</span>, <span class="author">Pettersson FH</span>, <span class="author">Cardon LR</span>, <span class="author">Morris AP</span> and <span class="author">Zondervan KT</span></p>
<p class="affiliation">Genetic and Genomic Epidemiology Unit, Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%67%63%6C%61%72%6B%65%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#103;&#99;&#108;&#97;&#114;&#107;&#101;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This protocol describes how to perform basic statistical analysis in a population-based genetic association case-control study. The steps described involve the (i) appropriate selection of measures of association and relevance of disease models; (ii) appropriate selection of tests of association; (iii) visualization and interpretation of results; (iv) consideration of appropriate methods to control for multiple testing; and (v) replication strategies. Assuming no previous experience with software such as PLINK, R or Haploview, we describe how to use these popular tools for handling single-nucleotide polymorphism data in order to carry out tests of association and visualize and interpret results. This protocol assumes that data quality assessment and control has been performed, as described in a previous protocol, so that samples and markers deemed to have the potential to introduce bias to the study have been identified and removed. Study design, marker selection and quality control of case-control studies have also been discussed in earlier protocols. The protocol should take ~1 h to complete.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">081682</span>, <span class="grant">085235</span>, <span class="grant">WT91745/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">121-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21293453" rel="external">21293453</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3154648" rel="external">3154648</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2010.182" rel="external">10.1038/nprot.2010.182</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The GENCODE exome: sequencing the complete human exome.</h4>
<p class="authors"><span class="author">Coffey AJ</span>, <span class="author">Kokocinski F</span>, <span class="author">Calafato MS</span>, <span class="author">Scott CE</span>, <span class="author">Palta P</span>, <span class="author">Drury E</span>, <span class="author">Joyce CJ</span>, <span class="author">Leproust EM</span>, <span class="author">Harrow J</span>, <span class="author">Hunt S</span>, <span class="author">Lehesjoki AE</span>, <span class="author">Turner DJ</span>, <span class="author">Hubbard TJ</span> and <span class="author">Palotie A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p><p class="abstract">Sequencing the coding regions, the exome, of the human genome is one of the major current strategies to identify low frequency and rare variants associated with human disease traits. So far, the most widely used commercial exome capture reagents have mainly targeted the consensus coding sequence (CCDS) database. We report the design of an extended set of targets for capturing the complete human exome, based on annotation from the GENCODE consortium. The extended set covers an additional 5594 genes and 10.3 Mb compared with the current CCDS-based sets. The additional regions include potential disease genes previously inaccessible to exome resequencing studies, such as 43 genes linked to ion channel activity and 70 genes linked to protein kinase activity. In total, the new GENCODE exome set developed here covers 47.9 Mb and performed well in sequence capture experiments. In the sample set used in this study, we identified over 5000 SNP variants more in the GENCODE exome target (24%) than in the CCDS-based exome sequencing.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5U54HG004555</span>; Wellcome Trust: <span class="grant">077198</span>, <span class="grant">WT062023</span>, <span class="grant">WT077198</span>, <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">7</span>;<span class="pagerange">827-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21364695">21364695</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3137498">3137498</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2011.28">10.1038/ejhg.2011.28</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A world in a grain of sand: human history from genetic data.</h4>
<p class="authors"><span class="author">Colonna V</span>, <span class="author">Pagani L</span>, <span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK.</p><p class="abstract">Genome-wide genotypes and sequences are enriching our understanding of the past 50,000 years of human history and providing insights into earlier periods largely inaccessible to mitochondrial DNA and Y-chromosomal studies.To see a world in a grain of sand ...William Blake, Auguries of Innocence.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">11</span>;<span class="pagerange">234</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22104725">22104725</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3334592">3334592</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-11-234">10.1186/gb-2011-12-11-234</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation in genome-wide mutation rates within and between human families.</h4>
<p class="authors"><span class="author">Conrad DF</span>, <span class="author">Keebler JE</span>, <span class="author">DePristo MA</span>, <span class="author">Lindsay SJ</span>, <span class="author">Zhang Y</span>, <span class="author">Casals F</span>, <span class="author">Idaghdour Y</span>, <span class="author">Hartl CL</span>, <span class="author">Torroja C</span>, <span class="author">Garimella KV</span>, <span class="author">Zilversmit M</span>, <span class="author">Cartwright R</span>, <span class="author">Rouleau GA</span>, <span class="author">Daly M</span>, <span class="author">Stone EA</span>, <span class="author">Hurles ME</span>, <span class="author">Awadalla P</span> and <span class="author">1000 Genomes Project</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">J.B.S. Haldane proposed in 1947 that the male germline may be more mutagenic than the female germline. Diverse studies have supported Haldane&#39;s contention of a higher average mutation rate in the male germline in a variety of mammals, including humans. Here we present, to our knowledge, the first direct comparative analysis of male and female germline mutation rates from the complete genome sequences of two parent-offspring trios. Through extensive validation, we identified 49 and 35 germline de novo mutations (DNMs) in two trio offspring, as well as 1,586 non-germline DNMs arising either somatically or in the cell lines from which the DNA was derived. Most strikingly, in one family, we observed that 92% of germline DNMs were from the paternal germline, whereas, in contrast, in the other family, 64% of DNMs were from the maternal germline. These observations suggest considerable variation in mutation rates within and between families.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG004960</span>; NIGMS NIH HHS: <span class="grant">R01 GM070806</span>; Wellcome Trust: <span class="grant">077014</span>, <span class="grant">077014/Z/05/Z</span>, <span class="grant">085532</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">7</span>;<span class="pagerange">712-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21666693">21666693</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322360">3322360</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.862">10.1038/ng.862</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inherited variation in vitamin D genes is associated with predisposition to autoimmune disease type 1 diabetes.</h4>
<p class="authors"><span class="author">Cooper JD</span>, <span class="author">Smyth DJ</span>, <span class="author">Walker NM</span>, <span class="author">Stevens H</span>, <span class="author">Burren OS</span>, <span class="author">Wallace C</span>, <span class="author">Greissl C</span>, <span class="author">Ramos-Lopez E</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Dunger DB</span>, <span class="author">Spector TD</span>, <span class="author">Ouwehand WH</span>, <span class="author">Wang TJ</span>, <span class="author">Badenhoop K</span> and <span class="author">Todd JA</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, UK.</p><p class="abstract"><strong>Objective</strong>: Vitamin D deficiency (25-hydroxyvitamin D [25(OH)D] &lt;50 nmol/L) is commonly reported in both children and adults worldwide, and growing evidence indicates that vitamin D deficiency is associated with many extraskeletal chronic disorders, including the autoimmune diseases type 1 diabetes and multiple sclerosis.</p><p class="abstract">We measured 25(OH)D concentrations in 720 case and 2,610 control plasma samples and genotyped single nucleotide polymorphisms from seven vitamin D metabolism genes in 8,517 case, 10,438 control, and 1,933 family samples. We tested genetic variants influencing 25(OH)D metabolism for an association with both circulating 25(OH)D concentrations and disease status.</p><p class="abstract"><strong>Results</strong>: Type 1 diabetic patients have lower circulating levels of 25(OH)D than similarly aged subjects from the British population. Only 4.3 and 18.6% of type 1 diabetic patients reached optimal levels (&ge;75 nmol/L) of 25(OH)D for bone health in the winter and summer, respectively. We replicated the associations of four vitamin D metabolism genes (GC, DHCR7, CYP2R1, and CYP24A1) with 25(OH)D in control subjects. In addition to the previously reported association between type 1 diabetes and CYP27B1 (P = 1.4 &times; 10(-4)), we obtained consistent evidence of type 1 diabetes being associated with DHCR7 (P = 1.2 &times; 10(-3)) and CYP2R1 (P = 3.0 &times; 10(-3)).</p><p class="abstract"><strong>Conclusions</strong>: Circulating levels of 25(OH)D in children and adolescents with type 1 diabetes vary seasonally and are under the same genetic control as in the general population but are much lower. Three key 25(OH)D metabolism genes show consistent evidence of association with type 1 diabetes risk, indicating a genetic etiological role for vitamin D deficiency in type 1 diabetes.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0601653</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076113</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">079895</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">5</span>;<span class="pagerange">1624-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21441443" rel="external">21441443</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3292339" rel="external">3292339</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb10-1656" rel="external">10.2337/db10-1656</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study in Europeans and South Asians identifies five new loci for coronary artery disease.</h4>
<p class="authors"><span class="author">Coronary Artery Disease (C4D) Genetics Consortium</span></p><p class="abstract">Genome-wide association studies have identified 11 common variants convincingly associated with coronary artery disease (CAD)&sup1;&#x207B;&#x2077;, a modest number considering the apparent heritability of CAD&#x2078;. All of these variants have been discovered in European populations. We report a meta-analysis of four large genome-wide association studies of CAD, with &sim;575,000 genotyped SNPs in a discovery dataset comprising 15,420 individuals with CAD (cases) (8,424 Europeans and 6,996 South Asians) and 15,062 controls. There was little evidence for ancestry-specific associations, supporting the use of combined analyses. Replication in an independent sample of 21,408 cases and 19,185 controls identified five loci newly associated with CAD (P &lt; 5 &times; 10&#x207B;&#x2078; in the combined discovery and replication analysis): LIPA on 10q23, PDGFD on 11q22, ADAMTS7-MORF4L1 on 15q25, a gene rich locus on 7q22 and KIAA1462 on 10p11. The CAD-associated SNP in the PDGFD locus showed tissue-specific cis expression quantitative trait locus effects. These findings implicate new pathways for CAD susceptibility.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/08/014/24067</span>, <span class="grant">SP/08/010/25939</span>; Cancer Research UK: <span class="grant">10293</span>; Medical Research Council: <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0801056</span>, <span class="grant">G9521010</span>, <span class="grant">MC_U137686854</span>, <span class="grant">MC_U137686857</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">4</span>;<span class="pagerange">339-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21378988">21378988</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.782">10.1038/ng.782</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bamako 2009 conference on the bioinformatics of infectious diseases.</h4>
<p class="authors"><span class="author">Corpas M</span>, <span class="author">Doumbia S</span>, <span class="author">Gascuel O</span> and <span class="author">Mulder N</span></p>
<p class="publication"><span class="title">Infection, genetics and evolution : journal of molecular epidemiology and evolutionary genetics in infectious diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="number">4</span>;<span class="pagerange">695-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21352953" rel="external">21352953</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.meegid.2011.02.012" rel="external">10.1016/j.meegid.2011.02.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pervasive sharing of genetic effects in autoimmune disease.</h4>
<p class="authors"><span class="author">Cotsapas C</span>, <span class="author">Voight BF</span>, <span class="author">Rossin E</span>, <span class="author">Lage K</span>, <span class="author">Neale BM</span>, <span class="author">Wallace C</span>, <span class="author">Abecasis GR</span>, <span class="author">Barrett JC</span>, <span class="author">Behrens T</span>, <span class="author">Cho J</span>, <span class="author">De Jager PL</span>, <span class="author">Elder JT</span>, <span class="author">Graham RR</span>, <span class="author">Gregersen P</span>, <span class="author">Klareskog L</span>, <span class="author">Siminovitch KA</span>, <span class="author">van Heel DA</span>, <span class="author">Wijmenga C</span>, <span class="author">Worthington J</span>, <span class="author">Todd JA</span>, <span class="author">Hafler DA</span>, <span class="author">Rich SS</span>, <span class="author">Daly MJ</span> and <span class="author">FOCiS Network of Consortia</span></p>
<p class="affiliation">Center For Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, United States of America.</p><p class="abstract">Genome-wide association (GWA) studies have identified numerous, replicable, genetic associations between common single nucleotide polymorphisms (SNPs) and risk of common autoimmune and inflammatory (immune-mediated) diseases, some of which are shared between two diseases. Along with epidemiological and clinical evidence, this suggests that some genetic risk factors may be shared across diseases-as is the case with alleles in the Major Histocompatibility Locus. In this work we evaluate the extent of this sharing for 107 immune disease-risk SNPs in seven diseases: celiac disease, Crohn&#39;s disease, multiple sclerosis, psoriasis, rheumatoid arthritis, systemic lupus erythematosus, and type 1 diabetes. We have developed a novel statistic for Cross Phenotype Meta-Analysis (CPMA) which detects association of a SNP to multiple, but not necessarily all, phenotypes. With it, we find evidence that 47/107 (44%) immune-mediated disease risk SNPs are associated to multiple-but not all-immune-mediated diseases (SNP-wise P(CPMA)&lt;0.01). We also show that distinct groups of interacting proteins are encoded near SNPs which predispose to the same subsets of diseases; we propose these as the mechanistic basis of shared disease risk. We are thus able to leverage genetic data across diseases to construct biological hypotheses about the underlying mechanism of pathogenesis.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">8</span>;<span class="pagerange">e1002254</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21852963" rel="external">21852963</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3154137" rel="external">3154137</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002254" rel="external">10.1371/journal.pgen.1002254</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete sequence and molecular epidemiology of IncK epidemic plasmid encoding blaCTX-M-14.</h4>
<p class="authors"><span class="author">Cottell JL</span>, <span class="author">Webber MA</span>, <span class="author">Coldham NG</span>, <span class="author">Taylor DL</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Hauser H</span>, <span class="author">Thomson NR</span>, <span class="author">Woodward MJ</span> and <span class="author">Piddock LJ</span></p>
<p class="affiliation">The University of Birmingham, Birmingham, UK.</p><p class="abstract">Antimicrobial drug resistance is a global challenge for the 21st century with the emergence of resistant bacterial strains worldwide. Transferable resistance to &beta;-lactam antimicrobial drugs, mediated by production of extended-spectrum &beta;-lactamases (ESBLs), is of particular concern. In 2004, an ESBL-carrying IncK plasmid (pCT) was isolated from cattle in the United Kingdom. The sequence was a 93,629-bp plasmid encoding a single antimicrobial drug resistance gene, blaCTX-M-14. From this information, PCRs identifying novel features of pCT were designed and applied to isolates from several countries, showing that the plasmid has disseminated worldwide in bacteria from humans and animals. Complete DNA sequences can be used as a platform to develop rapid epidemiologic tools to identify and trace the spread of plasmids in clinically relevant pathogens, thus facilitating a better understanding of their distribution and ability to transfer between bacteria of humans and animals.</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">17</span>;<span class="number">4</span>;<span class="pagerange">645-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21470454" rel="external">21470454</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3377399" rel="external">3377399</a>; DOI: <a href="http://dx.doi.org/10.3201%2Feid1704.101009" rel="external">10.3201/eid1704.101009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Basigin is a receptor essential for erythrocyte invasion by Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Crosnier C</span>, <span class="author">Bustamante LY</span>, <span class="author">Bartholdson SJ</span>, <span class="author">Bei AK</span>, <span class="author">Theron M</span>, <span class="author">Uchikawa M</span>, <span class="author">Mboup S</span>, <span class="author">Ndir O</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Duraisingh MT</span>, <span class="author">Rayner JC</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Erythrocyte invasion by Plasmodium falciparum is central to the pathogenesis of malaria. Invasion requires a series of extracellular recognition events between erythrocyte receptors and ligands on the merozoite, the invasive form of the parasite. None of the few known receptor-ligand interactions involved are required in all parasite strains, indicating that the parasite is able to access multiple redundant invasion pathways. Here, we show that we have identified a receptor-ligand pair that is essential for erythrocyte invasion in all tested P. falciparum strains. By systematically screening a library of erythrocyte proteins, we have found that the Ok blood group antigen, basigin, is a receptor for PfRh5, a parasite ligand that is essential for blood stage growth. Erythrocyte invasion was potently inhibited by soluble basigin or by basigin knockdown, and invasion could be completely blocked using low concentrations of anti-basigin antibodies; importantly, these effects were observed across all laboratory-adapted and field strains tested. Furthermore, Ok(a-) erythrocytes, which express a basigin variant that has a weaker binding affinity for PfRh5, had reduced invasion efficiencies. Our discovery of a cross-strain dependency on a single extracellular receptor-ligand pair for erythrocyte invasion by P. falciparum provides a focus for new anti-malarial therapies.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G19/9</span>; NCEZID CDC HHS: <span class="grant">R36 CK000119-01</span>; NIAID NIH HHS: <span class="grant">2T32 AI007535-12</span>, <span class="grant">R01 AI057919</span>, <span class="grant">R01 AI057919-05</span>, <span class="grant">R01AI057919</span>; Wellcome Trust: <span class="grant">077108</span>, <span class="grant">089084</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">480</span>;<span class="number">7378</span>;<span class="pagerange">534-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22080952">22080952</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245779">3245779</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10606">10.1038/nature10606</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Disruption of mouse Slx4, a regulator of structure-specific nucleases, phenocopies Fanconi anemia.</h4>
<p class="authors"><span class="author">Crossan GP</span>, <span class="author">van der Weyden L</span>, <span class="author">Rosado IV</span>, <span class="author">Langevin F</span>, <span class="author">Gaillard PH</span>, <span class="author">McIntyre RE</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Gallagher F</span>, <span class="author">Kettunen MI</span>, <span class="author">Lewis DY</span>, <span class="author">Brindle K</span>, <span class="author">Arends MJ</span>, <span class="author">Adams DJ</span> and <span class="author">Patel KJ</span></p>
<p class="affiliation">Medical Research Council, Laboratory of Molecular Biology, Cambridge, UK.</p><p class="abstract">The evolutionarily conserved SLX4 protein, a key regulator of nucleases, is critical for DNA damage response. SLX4 nuclease complexes mediate repair during replication and can also resolve Holliday junctions formed during homologous recombination. Here we describe the phenotype of the Btbd12 knockout mouse, the mouse ortholog of SLX4, which recapitulates many key features of the human genetic illness Fanconi anemia. Btbd12-deficient animals are born at sub-Mendelian ratios, have greatly reduced fertility, are developmentally compromised and are prone to blood cytopenias. Btbd12(-/-) cells prematurely senesce, spontaneously accumulate damaged chromosomes and are particularly sensitive to DNA crosslinking agents. Genetic complementation reveals a crucial requirement for Btbd12 (also known as Slx4) to interact with the structure-specific endonuclease Xpf-Ercc1 to promote crosslink repair. The Btbd12 knockout mouse therefore establishes a disease model for Fanconi anemia and genetically links a regulator of nuclease incision complexes to the Fanconi anemia DNA crosslink repair pathway.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">12401</span>, <span class="grant">A11073</span>, <span class="grant">A11376</span>, <span class="grant">A12401</span>, <span class="grant">A8449</span>; Medical Research Council: <span class="grant">MC_U105178811</span>, <span class="grant">U.1051.03.009(78811)</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">2</span>;<span class="pagerange">147-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21240276">21240276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3624090">3624090</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.752">10.1038/ng.752</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid pneumococcal evolution in response to clinical interventions.</h4>
<p class="authors"><span class="author">Croucher NJ</span>, <span class="author">Harris SR</span>, <span class="author">Fraser C</span>, <span class="author">Quail MA</span>, <span class="author">Burton J</span>, <span class="author">van der Linden M</span>, <span class="author">McGee L</span>, <span class="author">von Gottberg A</span>, <span class="author">Song JH</span>, <span class="author">Ko KS</span>, <span class="author">Pichon B</span>, <span class="author">Baker S</span>, <span class="author">Parry CM</span>, <span class="author">Lambertsen LM</span>, <span class="author">Shahinas D</span>, <span class="author">Pillai DR</span>, <span class="author">Mitchell TJ</span>, <span class="author">Dougan G</span>, <span class="author">Tomasz A</span>, <span class="author">Klugman KP</span>, <span class="author">Parkhill J</span>, <span class="author">Hanage WP</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Epidemiological studies of the naturally transformable bacterial pathogen Streptococcus pneumoniae have previously been confounded by high rates of recombination. Sequencing 240 isolates of the PMEN1 (Spain(23F)-1) multidrug-resistant lineage enabled base substitutions to be distinguished from polymorphisms arising through horizontal sequence transfer. More than 700 recombinations were detected, with genes encoding major antigens frequently affected. Among these were 10 capsule-switching events, one of which accompanied a population shift as vaccine-escape serotype 19A isolates emerged in the USA after the introduction of the conjugate polysaccharide vaccine. The evolution of resistance to fluoroquinolones, rifampicin, and macrolides was observed to occur on multiple occasions. This study details how genomic plasticity within lineages of recombinogenic bacteria can permit adaptation to clinical interventions over remarkably short time scales.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800596</span>; Wellcome Trust: <span class="grant">076962</span>, <span class="grant">076964</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">331</span>;<span class="number">6016</span>;<span class="pagerange">430-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21273480">21273480</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3648787">3648787</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1198545">10.1126/science.1198545</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification, variation and transcription of pneumococcal repeat sequences.</h4>
<p class="authors"><span class="author">Croucher NJ</span>, <span class="author">Vernikos GS</span>, <span class="author">Parkhill J</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Pathogen Genomics, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6E%63%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#99;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Small interspersed repeats are commonly found in many bacterial chromosomes. Two families of repeats (BOX and RUP) have previously been identified in the genome of Streptococcus pneumoniae, a nasopharyngeal commensal and respiratory pathogen of humans. However, little is known about the role they play in pneumococcal genetics.</p><p class="abstract"><strong>Results</strong>: Analysis of the genome of S. pneumoniae ATCC 700669 revealed the presence of a third repeat family, which we have named SPRITE. All three repeats are present at a reduced density in the genome of the closely related species S. mitis. However, they are almost entirely absent from all other streptococci, although a set of elements related to the pneumococcal BOX repeat was identified in the zoonotic pathogen S. suis. In conjunction with information regarding their distribution within the pneumococcal chromosome, this suggests that it is unlikely that these repeats are specialised sequences performing a particular role for the host, but rather that they constitute parasitic elements. However, comparing insertion sites between pneumococcal sequences indicates that they appear to transpose at a much lower rate than IS elements. Some large BOX elements in S. pneumoniae were found to encode open reading frames on both strands of the genome, whilst another was found to form a composite RNA structure with two T box riboswitches. In multiple cases, such BOX elements were demonstrated as being expressed using directional RNA-seq and RT-PCR.</p><p class="abstract"><strong>Conclusions</strong>: BOX, RUP and SPRITE repeats appear to have proliferated extensively throughout the pneumococcal chromosome during the species&#39; past, but novel insertions are currently occurring at a relatively slow rate. Through their extensive secondary structures, they seem likely to affect the expression of genes with which they are co-transcribed. Software for annotation of these repeats is freely available from ftp://ftp.sanger.ac.uk/pub/pathogens/strep_repeats/.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="pagerange">120</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21333003" rel="external">21333003</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3049150" rel="external">3049150</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-12-120" rel="external">10.1186/1471-2164-12-120</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessing the complex architecture of polygenic traits in diverged yeast populations.</h4>
<p class="authors"><span class="author">Cubillos FA</span>, <span class="author">Billi E</span>, <span class="author">Z&ouml;rg&ouml; E</span>, <span class="author">Parts L</span>, <span class="author">Fargier P</span>, <span class="author">Omholt S</span>, <span class="author">Blomberg A</span>, <span class="author">Warringer J</span>, <span class="author">Louis EJ</span> and <span class="author">Liti G</span></p>
<p class="affiliation">Centre for Genetics and Genomics, Queen&#39;s Medical Centre, University of Nottingham, Nottingham, UK.</p><p class="abstract">Phenotypic variation arising from populations adapting to different niches has a complex underlying genetic architecture. A major challenge in modern biology is to identify the causative variants driving phenotypic variation. Recently, the baker&#39;s yeast, Saccharomyces cerevisiae has emerged as a powerful model for dissecting complex traits. However, past studies using a laboratory strain were unable to reveal the complete architecture of polygenic traits. Here, we present a linkage study using 576 recombinant strains obtained from crosses of isolates representative of the major lineages. The meiotic recombinational landscape appears largely conserved between populations; however, strain-specific hotspots were also detected. Quantitative measurements of growth in 23 distinct ecologically relevant environments show that our recombinant population recapitulates most of the standing phenotypic variation described in the species. Linkage analysis detected an average of 6.3 distinct QTLs for each condition tested in all crosses, explaining on average 39% of the phenotypic variation. The QTLs detected are not constrained to a small number of loci, and the majority are specific to a single cross-combination and to a specific environment. Moreover, crosses between strains of similar phenotypes generate greater variation in the offspring, suggesting the presence of many antagonistic alleles and epistatic interactions. We found that subtelomeric regions play a key role in defining individual quantitative variation, emphasizing the importance of the adaptive nature of these regions in natural populations. This set of recombinant strains is a powerful tool for investigating the complex architecture of polygenic traits.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F015216/1</span>, <span class="grant">BB/G01616X/1</span>, <span class="grant">BBF0152161</span>; Wellcome Trust: <span class="grant">WT 084507MA</span>, <span class="grant">WT077192 &frasl; Z &frasl; 05 &frasl; Z</span></p>
<p class="publication"><span class="title">Molecular ecology</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">7</span>;<span class="pagerange">1401-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21261765">21261765</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-294X.2011.05005.x">10.1111/j.1365-294X.2011.05005.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of meis1 in primitive and definitive hematopoiesis during zebrafish development.</h4>
<p class="authors"><span class="author">Cvejic A</span>, <span class="author">Serbanovic-Canic J</span>, <span class="author">Stemple DL</span> and <span class="author">Ouwehand WH</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, Long Road, Cambridge CB2 0PT, UK.</p><p class="abstract"><strong>Background</strong>: The Meis1 protein represents an important cofactor for Hox and Pbx1 and is implicated in human and murine leukemias. Though much is known about the role of meis1 in leukemogenesis, its function in normal hematopoiesis remains largely unclear. Here we characterized the role of the proto-oncogene, meis1, during zebrafish primitive and definitive hematopoiesis.</p><p class="abstract">Zebrafish embryos were stained with o-dianisidine to detect hemoglobin-containing cells and Sudan black to quantify neutrophils. The numbers of other cells (scl-, gata1- and alas2-positive cells) were also quantified by measuring the corresponding stained areas of the embryos. We used anti-Meis1 antibody and whole mount immunohistochemistry to determine the pattern of expression of Meis1 during zebrafish development and then analyzed the functional role of Meis1 by knocking-down the meis1 gene.</p><p class="abstract"><strong>Results</strong>: Using antisense morpholino oligomers to interrupt meis1 expression we found that, although primitive macrophage development could occur unhampered, posterior erythroid differentiation required meis1, and its absence resulted in a severe decrease in the number of mature erythrocytes. Furthermore a picture emerged that meis1 exerts important effects on later stages of erythrocyte maturation and that these effects are independent of gata1, but under the control of scl. In addition, meis1 morpholino knock-down led to dramatic single arteriovenous tube formation. We also found that knock-down of pbx1 resulted in a phenotype that was strikingly similar to that of meis1 knock-down zebrafish.</p><p class="abstract"><strong>Conclusions</strong>: These results imply that meis1, jointly with pbx1, regulates primitive hematopoiesis as well as vascular development.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077037/Z/05/Z</span>, <span class="grant">077047/Z/05/Z</span>, <span class="grant">082597/Z/07/Z</span></p>
<p class="publication"><span class="title">Haematologica</span>&nbsp;<span class="year">2011</span>;<span class="volume">96</span>;<span class="number">2</span>;<span class="pagerange">190-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21048033" rel="external">21048033</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3031685" rel="external">3031685</a>; DOI: <a href="http://dx.doi.org/10.3324%2Fhaematol.2010.027698" rel="external">10.3324/haematol.2010.027698</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A viral discovery methodology for clinical biopsy samples utilising massively parallel next generation sequencing.</h4>
<p class="authors"><span class="author">Daly GM</span>, <span class="author">Bexfield N</span>, <span class="author">Heaney J</span>, <span class="author">Stubbs S</span>, <span class="author">Mayer AP</span>, <span class="author">Palser A</span>, <span class="author">Kellam P</span>, <span class="author">Drou N</span>, <span class="author">Caccamo M</span>, <span class="author">Tiley L</span>, <span class="author">Alexander GJ</span>, <span class="author">Bernal W</span> and <span class="author">Heeney JL</span></p>
<p class="affiliation">Department of Veterinary Medicine, The University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Here we describe a virus discovery protocol for a range of different virus genera, that can be applied to biopsy-sized tissue samples. Our viral enrichment procedure, validated using canine and human liver samples, significantly improves viral read copy number and increases the length of viral contigs that can be generated by de novo assembly. This in turn enables the Illumina next generation sequencing (NGS) platform to be used as an effective tool for viral discovery from tissue samples.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">12</span>;<span class="pagerange">e28879</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22216131">22216131</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3244418">3244418</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0028879">10.1371/journal.pone.0028879</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The variant call format and VCFtools.</h4>
<p class="authors"><span class="author">Danecek P</span>, <span class="author">Auton A</span>, <span class="author">Abecasis G</span>, <span class="author">Albers CA</span>, <span class="author">Banks E</span>, <span class="author">DePristo MA</span>, <span class="author">Handsaker RE</span>, <span class="author">Lunter G</span>, <span class="author">Marth GT</span>, <span class="author">Sherry ST</span>, <span class="author">McVean G</span>, <span class="author">Durbin R</span> and <span class="author">1000 Genomes Project Analysis Group</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK.</p><p class="abstract">SUMMARY: The variant call format (VCF) is a generic format for storing DNA polymorphism data such as SNPs, insertions, deletions and structural variants, together with rich annotations. VCF is usually stored in a compressed manner and can be indexed for fast data retrieval of variants from a range of positions on the reference genome. The format was developed for the 1000 Genomes Project, and has also been adopted by other projects such as UK10K, dbSNP and the NHLBI Exome Project. VCFtools is a software suite that implements various utilities for processing VCF files, including validation, merging, comparing and also provides a general Perl API. AVAILABILITY: http://vcftools.sourceforge.net</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; NHGRI NIH HHS: <span class="grant">54 HG003067</span>, <span class="grant">R01 HG004719</span>, <span class="grant">U01 HG005208</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">RG/09/012/28096</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">15</span>;<span class="pagerange">2156-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21653522" rel="external">21653522</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3137218" rel="external">3137218</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr330" rel="external">10.1093/bioinformatics/btr330</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The effect of next-generation sequencing technology on complex trait research.</h4>
<p class="authors"><span class="author">Day-Williams AG</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Advances in the understanding of complex trait genetics have always been enabled by advances in genomic technology. Next-generation sequencing (NGS) is set to revolutionize the way complex trait genetics research is carried out.</p><p class="abstract"><strong>Results</strong>: NGS has multiple applications in the field of human genetics, but is accompanied by substantial study design, analysis and interpretation challenges. This review discusses key aspects of study design considerations, data handling issues and required analytical developments. We also highlight early successes in mapping genetic traits using NGS.</p><p class="abstract"><strong>Conclusion</strong>: NGS opens the entire spectrum of genomic alterations for the genetic analysis of complex traits and there are early publications illustrating its power. Continuing development in analytical tools will allow the promise of NGS to be realized.</p>
<p class="publication"><span class="title">European journal of clinical investigation</span>&nbsp;<span class="year">2011</span>;<span class="volume">41</span>;<span class="number">5</span>;<span class="pagerange">561-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21155765">21155765</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3107422">3107422</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2362.2010.02437.x">10.1111/j.1365-2362.2010.02437.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Linkage analysis without defined pedigrees.</h4>
<p class="authors"><span class="author">Day-Williams AG</span>, <span class="author">Blangero J</span>, <span class="author">Dyer TD</span>, <span class="author">Lange K</span> and <span class="author">Sobel EM</span></p>
<p class="affiliation">Department of Human Genetics, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, CA 90095-7088, USA.</p><p class="abstract">The need to collect accurate and complete pedigree information has been a drawback of family-based linkage and association studies. Even in case-control studies, investigators should be aware of, and condition on, familial relationships. In single nucleotide polymorphism (SNP) genome scans, relatedness can be directly inferred from the genetic data rather than determined through interviews. Various methods of estimating relatedness have previously been implemented, most notably in PLINK. We present new fast and accurate algorithms for estimating global and local kinship coefficients from dense SNP genotypes. These algorithms require only a single pass through the SNP genotype data. We also show that these estimates can be used to cluster individuals into pedigrees. With these estimates in hand, quantitative trait locus linkage analysis proceeds via traditional variance components methods without any prior relationship information. We demonstrate the success of our algorithms on simulated and real data sets. Our procedures make linkage analysis as easy as a typical genomewide association study.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG006139</span>; NHLBI NIH HHS: <span class="grant">P01 HL045522-18</span>; NIGMS NIH HHS: <span class="grant">GM053275</span>, <span class="grant">R01 GM053275</span>, <span class="grant">R01 GM053275-15</span>; NIMH NIH HHS: <span class="grant">MH059490</span>, <span class="grant">R37 MH059490-12</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">35</span>;<span class="number">5</span>;<span class="pagerange">360-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21465549">21465549</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3290352">3290352</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20584">10.1002/gepi.20584</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evaluation of different target enrichment methods in pooled sequencing designs for complex disease association studies.</h4>
<p class="authors"><span class="author">Day-Williams AG</span>, <span class="author">McLay K</span>, <span class="author">Drury E</span>, <span class="author">Edkins S</span>, <span class="author">Coffey AJ</span>, <span class="author">Palotie A</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">Pooled sequencing can be a cost-effective approach to disease variant discovery, but its applicability in association studies remains unclear. We compare sequence enrichment methods coupled to next-generation sequencing in non-indexed pools of 1, 2, 10, 20 and 50 individuals and assess their ability to discover variants and to estimate their allele frequencies. We find that pooled resequencing is most usefully applied as a variant discovery tool due to limitations in estimating allele frequency with high enough accuracy for association studies, and that in-solution hybrid-capture performs best among the enrichment methods examined regardless of pool size.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e26279</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22069447">22069447</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3206031">3206031</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0026279">10.1371/journal.pone.0026279</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A variant in MCF2L is associated with osteoarthritis.</h4>
<p class="authors"><span class="author">Day-Williams AG</span>, <span class="author">Southam L</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Rayner NW</span>, <span class="author">Esko T</span>, <span class="author">Estrada K</span>, <span class="author">Helgadottir HT</span>, <span class="author">Hofman A</span>, <span class="author">Ingvarsson T</span>, <span class="author">Jonsson H</span>, <span class="author">Keis A</span>, <span class="author">Kerkhof HJ</span>, <span class="author">Thorleifsson G</span>, <span class="author">Arden NK</span>, <span class="author">Carr A</span>, <span class="author">Chapman K</span>, <span class="author">Deloukas P</span>, <span class="author">Loughlin J</span>, <span class="author">McCaskie A</span>, <span class="author">Ollier WE</span>, <span class="author">Ralston SH</span>, <span class="author">Spector TD</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson JM</span>, <span class="author">Aslam N</span>, <span class="author">Birell F</span>, <span class="author">Carluke I</span>, <span class="author">Joseph J</span>, <span class="author">Rai A</span>, <span class="author">Reed M</span>, <span class="author">Walker K</span>, <span class="author">arcOGEN Consortium</span>, <span class="author">Doherty SA</span>, <span class="author">Jonsdottir I</span>, <span class="author">Maciewicz RA</span>, <span class="author">Muir KR</span>, <span class="author">Metspalu A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Stefansson K</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Meurs JB</span>, <span class="author">Zhang W</span>, <span class="author">Valdes AM</span>, <span class="author">Doherty M</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">Osteoarthritis (OA) is a prevalent, heritable degenerative joint disease with a substantial public health impact. We used a 1000-Genomes-Project-based imputation in a genome-wide association scan for osteoarthritis (3177 OA cases and 4894 controls) to detect a previously unidentified risk locus. We discovered a small disease-associated set of variants on chromosome 13. Through large-scale replication, we establish a robust association with SNPs in MCF2L (rs11842874, combined odds ratio [95% confidence interval] 1.17 [1.11-1.23], p&nbsp;=&nbsp;2.1&nbsp;&times; 10(-8)) across a total of 19,041 OA cases and 24,504 controls of European descent. This risk locus represents the third established signal for OA overall. MCF2L regulates a nerve growth factor (NGF), and treatment with a humanized monoclonal antibody against NGF is associated with reduction in pain and improvement in function for knee OA patients.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0100594</span>, <span class="grant">G0901461</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">89</span>;<span class="number">3</span>;<span class="pagerange">446-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21871595">21871595</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3169824">3169824</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2011.08.001">10.1016/j.ajhg.2011.08.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Contrasting signals of positive selection in genes involved in human skin-color variation from tests based on SNP scans and resequencing.</h4>
<p class="authors"><span class="author">de Gruijter JM</span>, <span class="author">Lao O</span>, <span class="author">Vermeulen M</span>, <span class="author">Xue Y</span>, <span class="author">Woodwark C</span>, <span class="author">Gillson CJ</span>, <span class="author">Coffey AJ</span>, <span class="author">Ayub Q</span>, <span class="author">Mehdi SQ</span>, <span class="author">Kayser M</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Department of Forensic Molecular Biology, Erasmus MC University Medical Center, PO Box 2040, Rotterdam, 3000 CA, The Netherlands. <a href="mailto:%6F.%6C%61%6F%67%72%75%65%73%6F%40%65%72%61%73%6D%75%73%6D%63.%6E%6C">&#111;&#46;&#108;&#97;&#111;&#103;&#114;&#117;&#101;&#115;&#111;&#64;&#101;&#114;&#97;&#115;&#109;&#117;&#115;&#109;&#99;&#46;&#110;&#108;</a>.</p><p class="abstract"><strong>Background</strong>: Numerous genome-wide scans conducted by genotyping previously ascertained single-nucleotide polymorphisms (SNPs) have provided candidate signatures for positive selection in various regions of the human genome, including in genes involved in pigmentation traits. However, it is unclear how well the signatures discovered by such haplotype-based test statistics can be reproduced in tests based on full resequencing data. Four genes (oculocutaneous albinism II (OCA2), tyrosinase-related protein 1 (TYRP1), dopachrome tautomerase (DCT), and KIT ligand (KITLG)) implicated in human skin-color variation, have shown evidence for positive selection in Europeans and East Asians in previous SNP-scan data. In the current study, we resequenced 4.7 to 6.7 kb of DNA from each of these genes in Africans, Europeans, East Asians, and South Asians.</p><p class="abstract"><strong>Results</strong>: Applying all commonly used neutrality-test statistics for allele frequency distribution to the newly generated sequence data provided conflicting results regarding evidence for positive selection. Previous haplotype-based findings could not be clearly confirmed. Although some tests were marginally significant for some populations and genes, none of them were significant after multiple-testing correction. Combined P values for each gene-population pair did not improve these results. Application of Approximate Bayesian Computation Markov chain Monte Carlo based to these sequence data using a simple forward simulator revealed broad posterior distributions of the selective parameters for all four genes, providing no support for positive selection. However, when we applied this approach to published sequence data on SLC45A2, another human pigmentation candidate gene, we could readily confirm evidence for positive selection, as previously detected with sequence-based and some haplotype-based tests.</p><p class="abstract"><strong>Conclusions</strong>: Overall, our data indicate that even genes that are strong biological candidates for positive selection and show reproducible signatures of positive selection in SNP scans do not always show the same replicability of selection signals in other tests, which should be considered in future studies on detecting positive selection in genetic data.</p>
<p class="publication"><span class="title">Investigative genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">2</span>;<span class="number">1</span>;<span class="pagerange">24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22133426">22133426</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3287149">3287149</a>; DOI: <a href="http://dx.doi.org/10.1186%2F2041-2223-2-24">10.1186/2041-2223-2-24</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Computational identification of insertional mutagenesis targets for cancer gene discovery.</h4>
<p class="authors"><span class="author">de Jong J</span>, <span class="author">de Ridder J</span>, <span class="author">van der Weyden L</span>, <span class="author">Sun N</span>, <span class="author">van Uitert M</span>, <span class="author">Berns A</span>, <span class="author">van Lohuizen M</span>, <span class="author">Jonkers J</span>, <span class="author">Adams DJ</span> and <span class="author">Wessels LF</span></p>
<p class="affiliation">Bioinformatics and Statistics, The Netherlands Cancer Institute, Plesmanlaan 121, 1066CX Amsterdam, The Netherlands.</p><p class="abstract">Insertional mutagenesis is a potent forward genetic screening technique used to identify candidate cancer genes in mouse model systems. An important, yet unresolved issue in the analysis of these screens, is the identification of the genes affected by the insertions. To address this, we developed Kernel Convolved Rule Based Mapping (KC-RBM). KC-RBM exploits distance, orientation and insertion density across tumors to automatically map integration sites to target genes. We perform the first genome-wide evaluation of the association of insertion occurrences with aberrant gene expression of the predicted targets in both retroviral and transposon data sets. We demonstrate the efficiency of KC-RBM by showing its superior performance over existing approaches in recovering true positives from a list of independently, manually curated cancer genes. The results of this work will significantly enhance the accuracy and speed of cancer gene discovery in forward genetic screens. KC-RBM is available as R-package.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">15</span>;<span class="pagerange">e105</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21652642">21652642</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3159484">3159484</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr447">10.1093/nar/gkr447</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cadm1 expression and function in the mouse lens.</h4>
<p class="authors"><span class="author">De Maria A</span>, <span class="author">Shi Y</span>, <span class="author">Luo X</span>, <span class="author">Van Der Weyden L</span> and <span class="author">Bassnett S</span></p>
<p class="affiliation">Department of Ophthalmology and Visual Sciences, Washington University School of Medicine, St. Louis, Missouri, USA.</p><p class="abstract"><strong>Purpose</strong>: The immunoglobulin superfamily member Cadm1 is a single-pass, type 1 membrane protein that mediates calcium-independent, cell-cell adhesion. Cadm1 has been implicated in tumor formation and synaptogenesis. A recent analysis of mouse lens cell membranes identified Cadm1 as a major constituent of the fiber cell membrane proteome. Here the authors examined the expression and function of Cadm1 in the mouse lens.</p><p class="abstract"><strong>Methods</strong>: Cadm1 expression was analyzed by Western blotting and immunofluorescence. The morphology of individual wild-type and Cadm1-null lens cells was visualized by confocal microscopy.</p><p class="abstract"><strong>Results</strong>: Cadm1 was present in epithelial and superficial fiber cells as a heavily glycosylated protein with an apparent molecular mass of &asymp;80 kDa. Analysis of proteins extracted from various strata of the lens indicated that Cadm1 was degraded during fiber cell differentiation, at approximately the same time as the lens organelles, an observation confirmed by confocal microscopy. In epithelial cells, Cadm1 was enriched in basolateral membranes, whereas, in fiber cells, expression was restricted to the lateral membranes. Lenses from Cadm1-null mice were of normal size and transparency. The three-dimensional morphology of the cells in the epithelial layer was unaltered in the absence of Cadm1. However, in contrast to wild-type lens fiber cells, Cadm1-null fiber cells had an irregular, highly undulating morphology.</p><p class="abstract"><strong>Conclusions</strong>: Cadm1 is an abundant component of the lens fiber cell membrane. Although not essential for lens transparency, Cadm1 has an indispensable role in establishing and maintaining the characteristic three-dimensional architecture of the lens fiber cell mass.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY009852</span>, <span class="grant">P30EY002687</span>, <span class="grant">R01EY018185</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Investigative ophthalmology &amp; visual science</span>&nbsp;<span class="year">2011</span>;<span class="volume">52</span>;<span class="number">5</span>;<span class="pagerange">2293-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21217103" rel="external">21217103</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3080734" rel="external">3080734</a>; DOI: <a href="http://dx.doi.org/10.1167%2Fiovs.10-6677" rel="external">10.1167/iovs.10-6677</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic risk reclassification for type 2 diabetes by age below or above 50 years using 40 type 2 diabetes risk single nucleotide polymorphisms.</h4>
<p class="authors"><span class="author">de Miguel-Yanes JM</span>, <span class="author">Shrader P</span>, <span class="author">Pencina MJ</span>, <span class="author">Fox CS</span>, <span class="author">Manning AK</span>, <span class="author">Grant RW</span>, <span class="author">Dupuis J</span>, <span class="author">Florez JC</span>, <span class="author">D&#39;Agostino RB</span>, <span class="author">Cupples LA</span>, <span class="author">Meigs JB</span>, <span class="author">MAGIC Investigators</span> and <span class="author">DIAGRAM+ Investigators</span></p>
<p class="affiliation">General Medicine Division, Massachusetts General Hospital, Boston, Massachusetts, USA.</p><p class="abstract"><strong>Objective</strong>: To test if knowledge of type 2 diabetes genetic variants improves disease prediction.</p><p class="abstract">We tested 40 single nucleotide polymorphisms (SNPs) associated with diabetes in 3,471 Framingham Offspring Study subjects followed over 34 years using pooled logistic regression models stratified by age (&lt;50 years, diabetes cases = 144; or &ge;50 years, diabetes cases = 302). Models included clinical risk factors and a 40-SNP weighted genetic risk score.</p><p class="abstract"><strong>Results</strong>: In people &lt;50 years of age, the clinical risk factors model C-statistic was 0.908; the 40-SNP score increased it to 0.911 (P = 0.3; net reclassification improvement (NRI): 10.2%, P = 0.001). In people &ge;50 years of age, the C-statistics without and with the score were 0.883 and 0.884 (P = 0.2; NRI: 0.4%). The risk per risk allele was higher in people &lt;50 than &ge;50 years of age (24 vs. 11%; P value for age interaction = 0.02).</p><p class="abstract"><strong>Conclusions</strong>: Knowledge of common genetic variation appropriately reclassifies younger people for type 2 diabetes risk beyond clinical risk factors but not older people.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179474</span>; NCRR NIH HHS: <span class="grant">1S10RR163736-01A1</span>; NHLBI NIH HHS: <span class="grant">N01-HC- 25195</span>; NIDDK NIH HHS: <span class="grant">K23 DK65978</span>, <span class="grant">K24 DK080140</span>, <span class="grant">R01 DK078616</span>, <span class="grant">R21 DK084527</span>, <span class="grant">R21 DK084527-01</span></p>
<p class="publication"><span class="title">Diabetes care</span>&nbsp;<span class="year">2011</span>;<span class="volume">34</span>;<span class="number">1</span>;<span class="pagerange">121-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20889853">20889853</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3005447">3005447</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdc10-1265">10.2337/dc10-1265</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ethical issues in human genomics research in developing countries.</h4>
<p class="authors"><span class="author">de Vries J</span>, <span class="author">Bull SJ</span>, <span class="author">Doumbo O</span>, <span class="author">Ibrahim M</span>, <span class="author">Mercereau-Puijalon O</span>, <span class="author">Kwiatkowski D</span> and <span class="author">Parker M</span></p>
<p class="affiliation">The Ethox Centre, Department of Public Health and Primary Care, University of Oxford, Old Road Campus, Headington, Oxford, OX3 7LF, UK. <a href="mailto:%6A%61%6E%74%69%6E%61.%64%65%76%72%69%65%73%40%65%74%68%6F%78.%6F%78.%61%63.%75%6B">&#106;&#97;&#110;&#116;&#105;&#110;&#97;&#46;&#100;&#101;&#118;&#114;&#105;&#101;&#115;&#64;&#101;&#116;&#104;&#111;&#120;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Genome-wide association studies (GWAS) provide a powerful means of identifying genetic variants that play a role in common diseases. Such studies present important ethical challenges. An increasing number of GWAS is taking place in lower income countries and there is a pressing need to identify the particular ethical challenges arising in such contexts. In this paper, we draw upon the experiences of the MalariaGEN Consortium to identify specific ethical issues raised by such research in Africa, Asia and Oceania.</p><p class="abstract"><strong>Discussion</strong>: We explore ethical issues in three key areas: protecting the interests of research participants, regulation of international collaborative genomics research and protecting the interests of scientists in low income countries. With regard to participants, important challenges are raised about community consultation and consent. Genomics research raises ethical and governance issues about sample export and ownership, about the use of archived samples and about the complexity of reviewing such large international projects. In the context of protecting the interests of researchers in low income countries, we discuss aspects of data sharing and capacity building that need to be considered for sustainable and mutually beneficial collaborations.</p><p class="abstract"><strong>Summary</strong>: Many ethical issues are raised when genomics research is conducted on populations that are characterised by lower average income and literacy levels, such as the populations included in MalariaGEN. It is important that such issues are appropriately addressed in such research. Our experience suggests that the ethical issues in genomics research can best be identified, analysed and addressed where ethics is embedded in the design and implementation of such research projects.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">077383/Z/05/Z</span>, <span class="grant">087285/Z/08/Z</span>, <span class="grant">WT 083326/Z/07/Z</span></p>
<p class="publication"><span class="title">BMC medical ethics</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="pagerange">5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21418562">21418562</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3076260">3076260</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1472-6939-12-5">10.1186/1472-6939-12-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cell type-specific DNA methylation at intragenic CpG islands in the immune system.</h4>
<p class="authors"><span class="author">Deaton AM</span>, <span class="author">Webb S</span>, <span class="author">Kerr AR</span>, <span class="author">Illingworth RS</span>, <span class="author">Guy J</span>, <span class="author">Andrews R</span> and <span class="author">Bird A</span></p>
<p class="affiliation">Wellcome Trust Centre for Cell Biology, University of Edinburgh, Edinburgh, United Kingdom</p><p class="abstract">Human and mouse genomes contain a similar number of CpG islands (CGIs), which are discrete CpG-rich DNA sequences associated with transcription start sites. In both species, &sim;50% of all CGIs are remote from annotated promoters but, nevertheless, often have promoter-like features. To determine the role of CGI methylation in cell differentiation, we analyzed DNA methylation at a comprehensive CGI set in cells of the mouse hematopoietic lineage. Using a method that potentially detects &sim;33% of genomic CpGs in the methylated state, we found that large differences in gene expression were accompanied by surprisingly few DNA methylation changes. There were, however, many DNA methylation differences between hematopoietic cells and a distantly related tissue, brain. Altered DNA methylation in the immune system occurred predominantly at CGIs within gene bodies, which have the properties of cell type-restricted promoters, but infrequently at annotated gene promoters or CGI flanking sequences (CGI &quot;shores&quot;). Unexpectedly, elevated intragenic CGI methylation correlated with silencing of the associated gene. Differentially methylated intragenic CGIs tended to lack H3K4me3 and associate with a transcriptionally repressive environment regardless of methylation state. Our results indicate that DNA methylation changes play a relatively minor role in the late stages of differentiation and suggest that intragenic CGIs represent regulatory sites of differential gene expression during the early stages of lineage specification.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">7</span>;<span class="pagerange">1074-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21628449">21628449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3129250">3129250</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.118703.110">10.1101/gr.118703.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Does a short breastfeeding period protect from FTO-induced adiposity in children?</h4>
<p class="authors"><span class="author">Dedoussis GV</span>, <span class="author">Yannakoulia M</span>, <span class="author">Timpson NJ</span>, <span class="author">Manios Y</span>, <span class="author">Kanoni S</span>, <span class="author">Scott RA</span>, <span class="author">Papoutsakis C</span>, <span class="author">Deloukas P</span>, <span class="author">Pitsiladis YP</span>, <span class="author">Davey-Smith G</span>, <span class="author">Hirschhorn JN</span> and <span class="author">Lyon HN</span></p>
<p class="affiliation">Department of Dietetics and Nutrition, Harokopio University, Athens, Greece. <a href="mailto:%64%65%64%6F%75%73%69%40%68%75%61.%67%72">&#100;&#101;&#100;&#111;&#117;&#115;&#105;&#64;&#104;&#117;&#97;&#46;&#103;&#114;</a></p><p class="abstract"><strong>Context</strong>: A number of studies have reported replicable associations between common genetic loci and obesity indices. One of these loci is the fat mass and obesity associated locus (FTO). We aimed to assess whether breastfeeding mediated the known association between FTO and indices of body fatness.</p><p class="abstract"><strong>Methods</strong>: This study includes three independent pediatric cohorts, two of Greek origin (the Gene-Diet Attica Investigation: GENDAI, n=1 138 and the &quot;Growth, Exercise and Nutrition Epidemiological Study In preschoolers&quot;: the GENESIS study, n=2 374) and one British (the Avon Longitudinal Study of Parents and Children:ALSPAC, n=4 325). Among other information, breastfeeding history was recorded. A DNA sample was ascertained by either blood or saliva. Genotyping for FTO variants was performed in GENDAI and ALSPAC for the rs9939609, while in GENESIS, for the rs17817449 variant.</p><p class="abstract"><strong>Results</strong>: In all cohorts, multivariate analysis showed that the association between FTO:rs9939609 and measures of obesity was consistent across newly presented cohorts (GENDAI: Body mass index [BMI], &beta;=0.43, p=0.009; Waist Circumference, &beta;=1.067, p=0.019; triceps skinfold, &beta;=0.972, p=0.003; subscapular skinfold, &beta;=0.593, p=0.023; GENESIS: Waist Circumference, &beta;=0.473, p=0.008 and subscapular skinfold, &beta;=0.227, p=0.014). Inclusion of one month of breastfeeding as an interaction term effectively removed these associations with indices of obesity (BMI, Waist-Hip-Ratio and subscapular skinfold). No evidence of such interaction was observed for the independent cohort of British children.</p><p class="abstract"><strong>Conclusions</strong>: Our findings indicate that in two moderately sized Greek samples, breastfeeding may exert a modifying effect on the relationship between variants at the FTO locus and indices of adiposity. These findings were not replicated in a larger British collection.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600705</span>, <span class="grant">G9815508</span>; NIDDK NIH HHS: <span class="grant">K23 DK067288</span>; Wellcome Trust</p>
<p class="publication"><span class="title">International journal of pediatric obesity : IJPO : an official journal of the International Association for the Study of Obesity</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">2-2</span>;<span class="pagerange">e326-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20863147">20863147</a>; DOI: <a href="http://dx.doi.org/10.3109%2F17477166.2010.490269">10.3109/17477166.2010.490269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies in &gt;80 000 subjects identifies multiple loci for C-reactive protein levels.</h4>
<p class="authors"><span class="author">Dehghan A</span>, <span class="author">Dupuis J</span>, <span class="author">Barbalic M</span>, <span class="author">Bis JC</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Lu C</span>, <span class="author">Pellikka N</span>, <span class="author">Wallaschofski H</span>, <span class="author">Kettunen J</span>, <span class="author">Henneman P</span>, <span class="author">Baumert J</span>, <span class="author">Strachan DP</span>, <span class="author">Fuchsberger C</span>, <span class="author">Vitart V</span>, <span class="author">Wilson JF</span>, <span class="author">Par&eacute; G</span>, <span class="author">Naitza S</span>, <span class="author">Rudock ME</span>, <span class="author">Surakka I</span>, <span class="author">de Geus EJ</span>, <span class="author">Alizadeh BZ</span>, <span class="author">Guralnik J</span>, <span class="author">Shuldiner A</span>, <span class="author">Tanaka T</span>, <span class="author">Zee RY</span>, <span class="author">Schnabel RB</span>, <span class="author">Nambi V</span>, <span class="author">Kavousi M</span>, <span class="author">Ripatti S</span>, <span class="author">Nauck M</span>, <span class="author">Smith NL</span>, <span class="author">Smith AV</span>, <span class="author">Sundvall J</span>, <span class="author">Scheet P</span>, <span class="author">Liu Y</span>, <span class="author">Ruokonen A</span>, <span class="author">Rose LM</span>, <span class="author">Larson MG</span>, <span class="author">Hoogeveen RC</span>, <span class="author">Freimer NB</span>, <span class="author">Teumer A</span>, <span class="author">Tracy RP</span>, <span class="author">Launer LJ</span>, <span class="author">Buring JE</span>, <span class="author">Yamamoto JF</span>, <span class="author">Folsom AR</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Pankow J</span>, <span class="author">Elliott P</span>, <span class="author">Keaney JF</span>, <span class="author">Sun W</span>, <span class="author">Sarin AP</span>, <span class="author">Fontes JD</span>, <span class="author">Badola S</span>, <span class="author">Astor BC</span>, <span class="author">Hofman A</span>, <span class="author">Pouta A</span>, <span class="author">Werdan K</span>, <span class="author">Greiser KH</span>, <span class="author">Kuss O</span>, <span class="author">Meyer zu Schwabedissen HE</span>, <span class="author">Thiery J</span>, <span class="author">Jamshidi Y</span>, <span class="author">Nolte IM</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Parker AN</span>, <span class="author">Aspelund T</span>, <span class="author">Bates D</span>, <span class="author">Young L</span>, <span class="author">Tsui K</span>, <span class="author">Siscovick DS</span>, <span class="author">Guo X</span>, <span class="author">Rotter JI</span>, <span class="author">Uda M</span>, <span class="author">Schlessinger D</span>, <span class="author">Rudan I</span>, <span class="author">Hicks AA</span>, <span class="author">Penninx BW</span>, <span class="author">Thorand B</span>, <span class="author">Gieger C</span>, <span class="author">Coresh J</span>, <span class="author">Willemsen G</span>, <span class="author">Harris TB</span>, <span class="author">Uitterlinden AG</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Rice K</span>, <span class="author">Radke D</span>, <span class="author">Salomaa V</span>, <span class="author">Willems van Dijk K</span>, <span class="author">Boerwinkle E</span>, <span class="author">Vasan RS</span>, <span class="author">Ferrucci L</span>, <span class="author">Gibson QD</span>, <span class="author">Bandinelli S</span>, <span class="author">Snieder H</span>, <span class="author">Boomsma DI</span>, <span class="author">Xiao X</span>, <span class="author">Campbell H</span>, <span class="author">Hayward C</span>, <span class="author">Pramstaller PP</span>, <span class="author">van Duijn CM</span>, <span class="author">Peltonen L</span>, <span class="author">Psaty BM</span>, <span class="author">Gudnason V</span>, <span class="author">Ridker PM</span>, <span class="author">Homuth G</span>, <span class="author">Koenig W</span>, <span class="author">Ballantyne CM</span>, <span class="author">Witteman JC</span>, <span class="author">Benjamin EJ</span>, <span class="author">Perola M</span> and <span class="author">Chasman DI</span></p>
<p class="affiliation">Erasmus Medical Center, Dr Molewaterplein 50, Rotterdam, Netherlands.</p><p class="abstract"><strong>Background</strong>: C-reactive protein (CRP) is a heritable marker of chronic inflammation that is strongly associated with cardiovascular disease. We sought to identify genetic variants that are associated with CRP levels.</p><p class="abstract">We performed a genome-wide association analysis of CRP in 66 185 participants from 15 population-based studies. We sought replication for the genome-wide significant and suggestive loci in a replication panel comprising 16 540 individuals from 10 independent studies. We found 18 genome-wide significant loci, and we provided evidence of replication for 8 of them. Our results confirm 7 previously known loci and introduce 11 novel loci that are implicated in pathways related to the metabolic syndrome (APOC1, HNF1A, LEPR, GCKR, HNF4A, and PTPN2) or the immune system (CRP, IL6R, NLRP3, IL1F10, and IRF1) or that reside in regions previously not known to play a role in chronic inflammation (PPP1R3B, SALL1, PABPC4, ASCL1, RORA, and BCL7B). We found a significant interaction of body mass index with LEPR (P&lt;2.9&times;10(-6)). A weighted genetic risk score that was developed to summarize the effect of risk alleles was strongly associated with CRP levels and explained &asymp;5% of the trait variance; however, there was no evidence for these genetic variants explaining the association of CRP with coronary heart disease.</p><p class="abstract"><strong>Conclusions</strong>: We identified 18 loci that were associated with CRP levels. Our study highlights immune response and metabolic regulatory pathways involved in the regulation of chronic inflammation.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">PG/05/117</span>; Chief Scientist Office; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0500539</span>, <span class="grant">G0600705</span>; NCI NIH HHS: <span class="grant">CA047988</span>; NCRR NIH HHS: <span class="grant">1S10 RR163736-01A1</span>, <span class="grant">M01RR00425</span>; NHLBI NIH HHS: <span class="grant">5R01 HL087679-02</span>, <span class="grant">HL043851</span>, <span class="grant">HL064753</span>, <span class="grant">HL076784</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01 HC-75150</span>, <span class="grant">N01-HC 25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N02-HL-64278</span>, <span class="grant">R01 HL087652</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL080295</span>; NIA NIH HHS: <span class="grant">1R01 AG032098-01A1</span>, <span class="grant">AG028321</span>, <span class="grant">N01 AG62103</span>, <span class="grant">N01 AG62106</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">N01-AG-12109</span>, <span class="grant">Z01 AG000015-50</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">P30 DK072488</span>; NIMH NIH HHS: <span class="grant">1RL1 MH083268-01</span>, <span class="grant">5R01 MH63706:02</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2011</span>;<span class="volume">123</span>;<span class="number">7</span>;<span class="pagerange">731-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21300955" rel="external">21300955</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3147232" rel="external">3147232</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.110.948570" rel="external">10.1161/CIRCULATIONAHA.110.948570</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Y-chromosome landscape of the Philippines: extensive heterogeneity and varying genetic affinities of Negrito and non-Negrito groups.</h4>
<p class="authors"><span class="author">Delfin F</span>, <span class="author">Salvador JM</span>, <span class="author">Calacal GC</span>, <span class="author">Perdigon HB</span>, <span class="author">Tabbada KA</span>, <span class="author">Villamor LP</span>, <span class="author">Halos SC</span>, <span class="author">Gunnarsd&oacute;ttir E</span>, <span class="author">Myles S</span>, <span class="author">Hughes DA</span>, <span class="author">Xu S</span>, <span class="author">Jin L</span>, <span class="author">Lao O</span>, <span class="author">Kayser M</span>, <span class="author">Hurles ME</span>, <span class="author">Stoneking M</span> and <span class="author">De Ungria MC</span></p>
<p class="affiliation">DNA Analysis Laboratory, Natural Sciences Research Institute, University of the Philippines, Diliman, Quezon City, Philippines.</p><p class="abstract">The Philippines exhibits a rich diversity of people, languages, and culture, including so-called &#39;Negrito&#39; groups that have for long fascinated anthropologists, yet little is known about their genetic diversity. We report here, a survey of Y-chromosome variation in 390 individuals from 16 Filipino ethnolinguistic groups, including six Negrito groups, from across the archipelago. We find extreme diversity in the Y-chromosome lineages of Filipino groups with heterogeneity seen in both Negrito and non-Negrito groups, which does not support a simple dichotomy of Filipino groups as Negrito vs non-Negrito. Filipino non-recombining region of the human Y chromosome lineages reflect a chronology that extends from after the initial colonization of the Asia-Pacific region, to the time frame of the Austronesian expansion. Filipino groups appear to have diverse genetic affinities with different populations in the Asia-Pacific region. In particular, some Negrito groups are associated with indigenous Australians, with a potential time for the association ranging from the initial colonization of the region to more recent (after colonization) times. Overall, our results indicate extensive heterogeneity contributing to a complex genetic history for Filipino groups, with varying roles for migrations from outside the Philippines, genetic drift, and admixture among neighboring groups.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">2</span>;<span class="pagerange">224-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20877414" rel="external">20877414</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3025791" rel="external">3025791</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2010.162" rel="external">10.1038/ejhg.2010.162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic architecture of circulating lipid levels.</h4>
<p class="authors"><span class="author">Demirkan A</span>, <span class="author">Amin N</span>, <span class="author">Isaacs A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Whitfield JB</span>, <span class="author">Wichmann HE</span>, <span class="author">Kyvik KO</span>, <span class="author">Rudan I</span>, <span class="author">Gieger C</span>, <span class="author">Hicks AA</span>, <span class="author">Johansson &Aring;</span>, <span class="author">Hottenga JJ</span>, <span class="author">Smith JJ</span>, <span class="author">Wild SH</span>, <span class="author">Pedersen NL</span>, <span class="author">Willemsen G</span>, <span class="author">Mangino M</span>, <span class="author">Hayward C</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">Witteman J</span>, <span class="author">Montgomery GW</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Rantanen T</span>, <span class="author">Kaprio J</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Gyllensten U</span>, <span class="author">de Geus EJ</span>, <span class="author">Penninx BW</span>, <span class="author">Wilson JF</span>, <span class="author">Rivadeneria F</span>, <span class="author">Magnusson PK</span>, <span class="author">Boomsma DI</span>, <span class="author">Spector T</span>, <span class="author">Campbell H</span>, <span class="author">Hoehne B</span>, <span class="author">Martin NG</span>, <span class="author">Oostra BA</span>, <span class="author">McCarthy M</span>, <span class="author">Peltonen-Palotie L</span>, <span class="author">Aulchenko Y</span>, <span class="author">Visscher PM</span>, <span class="author">Ripatti S</span>, <span class="author">Janssens AC</span>, <span class="author">van Duijn CM</span> and <span class="author">ENGAGE CONSORTIUM</span></p>
<p class="affiliation">Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics, Erasmus University Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">Serum concentrations of low-density lipoprotein cholesterol (LDL-C), high-density lipoprotein cholesterol (HDL-C), triglycerides (TGs) and total cholesterol (TC) are important heritable risk factors for cardiovascular disease. Although genome-wide association studies (GWASs) of circulating lipid levels have identified numerous loci, a substantial portion of the heritability of these traits remains unexplained. Evidence of unexplained genetic variance can be detected by combining multiple independent markers into additive genetic risk scores. Such polygenic scores, constructed using results from the ENGAGE Consortium GWAS on serum lipids, were applied to predict lipid levels in an independent population-based study, the Rotterdam Study-II (RS-II). We additionally tested for evidence of a shared genetic basis for different lipid phenotypes. Finally, the polygenic score approach was used to identify an alternative genome-wide significance threshold before pathway analysis and those results were compared with those based on the classical genome-wide significance threshold. Our study provides evidence suggesting that many loci influencing circulating lipid levels remain undiscovered. Cross-prediction models suggested a small overlap between the polygenic backgrounds involved in determining LDL-C, HDL-C and TG levels. Pathway analysis utilizing the best polygenic score for TC uncovered extra information compared with using only genome-wide significant loci. These results suggest that the genetic architecture of circulating lipids involves a number of undiscovered variants with very small effects, and that increasing GWAS sample sizes will enable the identification of novel variants that regulate lipid levels.</p>
<p class="grants">Funded by: Chief Scientist Office; Wellcome Trust</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">7</span>;<span class="pagerange">813-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21448234" rel="external">21448234</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3137496" rel="external">3137496</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2011.21" rel="external">10.1038/ejhg.2011.21</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Specific capture and whole-genome sequencing of viruses from clinical samples.</h4>
<p class="authors"><span class="author">Depledge DP</span>, <span class="author">Palser AL</span>, <span class="author">Watson SJ</span>, <span class="author">Lai IY</span>, <span class="author">Gray ER</span>, <span class="author">Grant P</span>, <span class="author">Kanda RK</span>, <span class="author">Leproust E</span>, <span class="author">Kellam P</span> and <span class="author">Breuer J</span></p>
<p class="affiliation">Division of Infection and Immunity, University College London, London, United Kingdom. <a href="mailto:%64.%64%65%70%6C%65%64%67%65%40%75%63%6C.%61%63.%75%6B">&#100;&#46;&#100;&#101;&#112;&#108;&#101;&#100;&#103;&#101;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Whole genome sequencing of viruses directly from clinical samples is integral for understanding the genetics of host-virus interactions. Here, we report the use of sample sparing target enrichment (by hybridisation) for viral nucleic acid separation and deep-sequencing of herpesvirus genomes directly from a range of clinical samples including saliva, blood, virus vesicles, cerebrospinal fluid, and tumour cell lines. We demonstrate the effectiveness of the method by deep-sequencing 13 highly cell-associated human herpesvirus genomes and generating full length genome alignments at high read depth. Moreover, we show the specificity of the method enables the study of viral population structures and their diversity within a range of clinical samples types.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G07008</span>, <span class="grant">G0700814</span>, <span class="grant">G0900950</span>; Wellcome Trust: <span class="grant">081703MA</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e27805</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22125625">22125625</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3220689">3220689</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0027805">10.1371/journal.pone.0027805</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Glucocorticoid receptor (NR3C1) gene polymorphisms and onset of alcohol abuse in adolescents.</h4>
<p class="authors"><span class="author">Desrivi&egrave;res S</span>, <span class="author">Lourdusamy A</span>, <span class="author">M&uuml;ller C</span>, <span class="author">Ducci F</span>, <span class="author">Wong CP</span>, <span class="author">Kaakinen M</span>, <span class="author">Pouta A</span>, <span class="author">Hartikainen AL</span>, <span class="author">Isohanni M</span>, <span class="author">Charoen P</span>, <span class="author">Peltonen L</span>, <span class="author">Freimer N</span>, <span class="author">Elliott P</span>, <span class="author">Jarvelin MR</span> and <span class="author">Schumann G</span></p>
<p class="affiliation">MRC-SGDP Centre, Institute of Psychiatry, Kings College, UK.</p>
<p class="abstract">Onset of alcohol use at an early age increases the risk for later alcohol dependence. We investigated the role of the glucocorticoid receptor (GR) gene (NR3C1) in onset of alcohol use and abuse in 14-year-old adolescents (n=4534). Several NR3C1 polymorphisms were associated with onset of alcohol drinking or drunkenness at this age. Strongest associations were observed in females, with one marker (rs244465) remaining significant after correction for multiple testing (P(adj) =0.0067; odds ratio=1.7, for drunkenness). Our data provide the first evidence that GR modulates initiation of alcohol abuse and reveal a polymorphism that might contribute to susceptibility to addiction.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600705</span>; NHLBI NIH HHS: <span class="grant">1-R01HL087679-01</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5RL1MH083268</span>; Wellcome Trust: <span class="grant">069224</span>, <span class="grant">089061</span>, <span class="grant">GR06922</span></p>
<p class="publication"><span class="title">Addiction biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">16</span>;<span class="number">3</span>;<span class="pagerange">510-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20731635" rel="external">20731635</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428936" rel="external">3428936</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1369-1600.2010.00239.x" rel="external">10.1111/j.1369-1600.2010.00239.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Host candidate gene polymorphisms and clearance of drug-resistant Plasmodium falciparum parasites.</h4>
<p class="authors"><span class="author">Diakite M</span>, <span class="author">Achidi EA</span>, <span class="author">Achonduh O</span>, <span class="author">Craik R</span>, <span class="author">Djimde AA</span>, <span class="author">Evehe MS</span>, <span class="author">Green A</span>, <span class="author">Hubbart C</span>, <span class="author">Ibrahim M</span>, <span class="author">Jeffreys A</span>, <span class="author">Khan BK</span>, <span class="author">Kimani F</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Mbacham WF</span>, <span class="author">Jezan SO</span>, <span class="author">Ouedraogo JB</span>, <span class="author">Rockett K</span>, <span class="author">Rowlands K</span>, <span class="author">Tagelsir N</span>, <span class="author">Tekete MM</span>, <span class="author">Zongo I</span> and <span class="author">Ranford-Cartwright LC</span></p>
<p class="affiliation">Malaria Research and Training Centre, Faculty of Medicine, Pharmacy and Odontostomatology, University of Bamako, Mali. <a href="mailto:%6D%64%69%61%6B%69%74%65%40%69%63%65%72%6D%61%6C%69.%6F%72%67">&#109;&#100;&#105;&#97;&#107;&#105;&#116;&#101;&#64;&#105;&#99;&#101;&#114;&#109;&#97;&#108;&#105;&#46;&#111;&#114;&#103;</a></p><p class="abstract"><strong>Background</strong>: Resistance to anti-malarial drugs is a widespread problem for control programmes for this devastating disease. Molecular tests are available for many anti-malarial drugs and are useful tools for the surveillance of drug resistance. However, the correlation of treatment outcome and molecular tests with particular parasite markers is not perfect, due in part to individuals who are able to clear genotypically drug-resistant parasites. This study aimed to identify molecular markers in the human genome that correlate with the clearance of malaria parasites after drug treatment, despite the drug resistance profile of the protozoan as predicted by molecular approaches.</p><p class="abstract"><strong>Methods</strong>: 3721 samples from five African countries, which were known to contain genotypically drug resistant parasites, were analysed. These parasites were collected from patients who subsequently failed to clear their infection following drug treatment, as expected, but also from patients who successfully cleared their infections with drug-resistant parasites. 67 human polymorphisms (SNPs) on 17 chromosomes were analysed using Sequenom&#39;s mass spectrometry iPLEX gold platform, to identify regions of the human genome, which contribute to enhanced clearance of drug resistant parasites.</p><p class="abstract"><strong>Results</strong>: An analysis of all data from the five countries revealed significant associations between the phenotype of ability to clear drug-resistant Plasmodium falciparum infection and human immune response loci common to all populations. Overall, three SNPs showed a significant association with clearance of drug-resistant parasites with odds ratios of 0.76 for SNP rs2706384 (95% CI 0.71-0.92, P = 0.005), 0.66 for SNP rs1805015 (95% CI 0.45-0.97, P = 0.03), and 0.67 for SNP rs1128127 (95% CI 0.45-0.99, P = 0.05), after adjustment for possible confounding factors. The first two SNPs (rs2706384 and rs1805015) are within loci involved in pro-inflammatory (interferon-gamma) and anti-inflammatory (IL-4) cytokine responses. The third locus encodes a protein involved in the degradation of misfolded proteins within the endoplasmic reticulum, and its role, if any, in the clearance phenotype is unclear.</p><p class="abstract"><strong>Conclusions</strong>: The study showed significant association of three loci in the human genome with the ability of parasite to clear drug-resistant P. falciparum in samples taken from five countries distributed across sub-Saharan Africa. Both SNP rs2706384 and SNP1805015 have previously been reported to be associated with risk of malaria infection in African populations. The loci are involved in the Th1/Th2 balance, and the association of SNPs within these genes suggests a key role for antibody in the clearance of drug-resistant parasites. It is possible that patients able to clear drug-resistant infections have an enhanced ability to control parasite growth.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">075491/Z/04</span></p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2011</span>;<span class="volume">10</span>;<span class="pagerange">250</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21867552" rel="external">21867552</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3177816" rel="external">3177816</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-10-250" rel="external">10.1186/1475-2875-10-250</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Live vaccines and their role in modern vaccinology</h4>
<p class="authors"><span class="author">DOUGAN G</span>, <span class="author">GOULDING D</span>, <span class="author">Hall LJ</span></p>
<p class="publication"><span class="title">Replicating Vaccines.</span> <span class="year">2011</span>;<span class="volume">Part 1</span>;<span class="pagerange">3-14</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Immunity to salmonellosis.</h4>
<p class="authors"><span class="author">Dougan G</span>, <span class="author">John V</span>, <span class="author">Palmer S</span> and <span class="author">Mastroeni P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%67%64%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#100;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Salmonella enterica is a genetically broad species harboring isolates that display considerable antigenic heterogeneity and significant differences in virulence potential. Salmonella generally exhibit an invasive potential and they can survive for extended periods within cells of the immune system. They cause acute or chronic infections that can be local (e.g. gastroenteritis) or systemic (e.g. typhoid). In vivo Salmonella infections are complex with multiple arms of the immune system being engaged. Both humoral and cellular responses can be detected and characterized, but full protective immunity is not always induced, even following natural infection. The murine model has proven to be a fertile ground for exploring immune mechanisms and observations in the mouse have often, although not always, correlated with those in other infectable species, including humans. Host genetic studies have identified a number of mammalian genes that are central to controlling infection, operating both in innate and acquired immune pathways. Vaccines, both oral and parenteral, are available or under development, and these have been used with some success to explore immunity in both model systems and clinically in humans.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Immunological reviews</span>&nbsp;<span class="year">2011</span>;<span class="volume">240</span>;<span class="number">1</span>;<span class="pagerange">196-210</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21349095" rel="external">21349095</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1600-065X.2010.00999.x" rel="external">10.1111/j.1600-065X.2010.00999.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dalliance: interactive genome viewing on the web.</h4>
<p class="authors"><span class="author">Down TA</span>, <span class="author">Piipari M</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust/CRUK Gurdon Institute, Cambridge CB2 1QN, UK. <a href="mailto:%74%68%6F%6D%61%73%40%62%69%6F%64%61%6C%6C%69%61%6E%63%65.%6F%72%67">&#116;&#104;&#111;&#109;&#97;&#115;&#64;&#98;&#105;&#111;&#100;&#97;&#108;&#108;&#105;&#97;&#110;&#99;&#101;&#46;&#111;&#114;&#103;</a></p><p class="abstract"><strong>Summary</strong>: Dalliance is a new genome viewer which offers a high level of interactivity while running within a web browser. All data is fetched using the established distributed annotation system (DAS) protocol, making it easy to customize the browser and add extra data.</p><p class="abstract">Dalliance runs entirely within your web browser, and relies on existing DAS server infrastructure. Browsers for several mammalian genomes are available at http://www.biodalliance.org/, and the use of DAS means you can add your own data to these browsers. In addition, the source code (Javascript) is available under the BSD license, and is straightforward to install on your own web server and embed within other documents.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span>, <span class="grant">083563</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">6</span>;<span class="pagerange">889-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21252075">21252075</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3051325">3051325</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr020">10.1093/bioinformatics/btr020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome sequencing of multiple Leishmania donovani clinical isolates provides insights into population structure and mechanisms of drug resistance.</h4>
<p class="authors"><span class="author">Downing T</span>, <span class="author">Imamura H</span>, <span class="author">Decuypere S</span>, <span class="author">Clark TG</span>, <span class="author">Coombs GH</span>, <span class="author">Cotton JA</span>, <span class="author">Hilley JD</span>, <span class="author">de Doncker S</span>, <span class="author">Maes I</span>, <span class="author">Mottram JC</span>, <span class="author">Quail MA</span>, <span class="author">Rijal S</span>, <span class="author">Sanders M</span>, <span class="author">Sch&ouml;nian G</span>, <span class="author">Stark O</span>, <span class="author">Sundar S</span>, <span class="author">Vanaerschot M</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Dujardin JC</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, United Kingdom.</p><p class="abstract">Visceral leishmaniasis is a potentially fatal disease endemic to large parts of Asia and Africa, primarily caused by the protozoan parasite Leishmania donovani. Here, we report a high-quality reference genome sequence for a strain of L. donovani from Nepal, and use this sequence to study variation in a set of 16 related clinical lines, isolated from visceral leishmaniasis patients from the same region, which also differ in their response to in vitro drug susceptibility. We show that whole-genome sequence data reveals genetic structure within these lines not shown by multilocus typing, and suggests that drug resistance has emerged multiple times in this closely related set of lines. Sequence comparisons with other Leishmania species and analysis of single-nucleotide diversity within our sample showed evidence of selection acting in a range of surface- and transport-related genes, including genes associated with drug resistance. Against a background of relative genetic homogeneity, we found extensive variation in chromosome copy number between our lines. Other forms of structural variation were significantly associated with drug resistance, notably including gene dosage and the copy number of an experimentally verified circular episome present in all lines and described here for the first time. This study provides a basis for more powerful molecular profiling of visceral leishmaniasis, providing additional power to track the drug resistance and epidemiology of an important human pathogen.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076355</span>, <span class="grant">085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2143-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22038251">22038251</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3227103">3227103</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.123430.111">10.1101/gr.123430.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TTC12-ANKK1-DRD2 and CHRNA5-CHRNA3-CHRNB4 influence different pathways leading to smoking behavior from adolescence to mid-adulthood.</h4>
<p class="authors"><span class="author">Ducci F</span>, <span class="author">Kaakinen M</span>, <span class="author">Pouta A</span>, <span class="author">Hartikainen AL</span>, <span class="author">Veijola J</span>, <span class="author">Isohanni M</span>, <span class="author">Charoen P</span>, <span class="author">Coin L</span>, <span class="author">Hoggart C</span>, <span class="author">Ekelund J</span>, <span class="author">Peltonen L</span>, <span class="author">Freimer N</span>, <span class="author">Elliott P</span>, <span class="author">Schumann G</span> and <span class="author">J&auml;rvelin MR</span></p>
<p class="affiliation">Medical Research Council-Social Genetics and Developmental Psychiatry Centre, Institute of Psychiatry, King&#39;s College, London, United Kingdom. <a href="mailto:%46%72%61%6E%63%65%73%63%61.%44%75%63%63%69%40%6B%63%6C.%61%63.%75%6B">&#70;&#114;&#97;&#110;&#99;&#101;&#115;&#99;&#97;&#46;&#68;&#117;&#99;&#99;&#105;&#64;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: CHRNA5-CHRNA3-CHRNB4 and TTC12-ANKK1-DRD2 gene-clusters influence smoking behavior. Our aim was to test developmental changes in their effects as well as the interplays between them and with nongenetic factors.</p><p class="abstract"><strong>Methods</strong>: Participants included 4762 subjects from a general population-based, prospective Northern Finland 1966 Birth Cohort (NFBC 1966). Smoking behavior was collected at age 14 and 31 years. Information on maternal smoking, socioeconomic status, and novelty seeking were also collected. Structural equation modeling was used to construct an integrative etiologic model including genetic and nongenetic factors.</p><p class="abstract"><strong>Results</strong>: Several single nucleotide polymorphisms in both gene-clusters were significantly associated with smoking. The most significant were in CHRNA3 (rs1051730, p = 1.1 &times; 10(-5)) and in TTC12 (rs10502172, p = 9.1 &times; 10(-6)). CHRNA3-rs1051730[A] was more common among heavy/regular smokers than nonsmokers with similar effect-sizes at age 14 years (odds ratio [95% CI]: 1.27 [1.06-1.52]) and 31 years (1.28 [1.13-1.44]). TTC12-rs10502172[G] was more common among smokers than nonsmokers with stronger association at 14 years (1.33 [1.11-1.60]) than 31 years (1.14 [1.02-1.28]). In adolescence, carriers of three-four risk alleles at either CHRNA3-rs1051730 or TTC12-rs10502172 had almost threefold odds of smoking regularly than subjects with no risk alleles. TTC12-rs10502172 effect on smoking in adulthood was mediated by its effect on smoking in adolescence and via novelty seeking. Effect of CHRNA3-rs1051730 on smoking in adulthood was direct.</p><p class="abstract"><strong>Conclusions</strong>: TTC12-ANKK1-DRD2s seemed to influence smoking behavior mainly in adolescence, and its effect is partially mediated by personality characteristics promoting drug-seeking behavior. In contrast, CHRNA5-CHRNA3-CHRNB4 is involved in the transition toward heavy smoking in mid-adulthood and in smoking persistence. Factors related to familial and social disadvantages were strong independent predictors of smoking.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">93558</span>, <span class="grant">G0500539</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">RL1 MH083268-01</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">69</span>;<span class="number">7</span>;<span class="pagerange">650-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21168125" rel="external">21168125</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3058144" rel="external">3058144</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2010.09.055" rel="external">10.1016/j.biopsych.2010.09.055</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Developing and implementing an institute-wide data sharing policy.</h4>
<p class="authors"><span class="author">Dyke SO</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%74%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">The Wellcome Trust Sanger Institute has a strong reputation for prepublication data sharing as a result of its policy of rapid release of genome sequence data and particularly through its contribution to the Human Genome Project. The practicalities of broad data sharing remain largely uncharted, especially to cover the wide range of data types currently produced by genomic studies and to adequately address ethical issues. This paper describes the processes and challenges involved in implementing a data sharing policy on an institute-wide scale. This includes questions of governance, practical aspects of applying principles to diverse experimental contexts, building enabling systems and infrastructure, incentives and collaborative issues.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">3</span>;<span class="number">9</span>;<span class="pagerange">60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21955348">21955348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3239235">3239235</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm276">10.1186/gm276</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assemblathon 1: a competitive assessment of de novo short read assembly methods.</h4>
<p class="authors"><span class="author">Earl D</span>, <span class="author">Bradnam K</span>, <span class="author">St John J</span>, <span class="author">Darling A</span>, <span class="author">Lin D</span>, <span class="author">Fass J</span>, <span class="author">Yu HO</span>, <span class="author">Buffalo V</span>, <span class="author">Zerbino DR</span>, <span class="author">Diekhans M</span>, <span class="author">Nguyen N</span>, <span class="author">Ariyaratne PN</span>, <span class="author">Sung WK</span>, <span class="author">Ning Z</span>, <span class="author">Haimel M</span>, <span class="author">Simpson JT</span>, <span class="author">Fonseca NA</span>, <span class="author">Birol &#x130;</span>, <span class="author">Docking TR</span>, <span class="author">Ho IY</span>, <span class="author">Rokhsar DS</span>, <span class="author">Chikhi R</span>, <span class="author">Lavenier D</span>, <span class="author">Chapuis G</span>, <span class="author">Naquin D</span>, <span class="author">Maillet N</span>, <span class="author">Schatz MC</span>, <span class="author">Kelley DR</span>, <span class="author">Phillippy AM</span>, <span class="author">Koren S</span>, <span class="author">Yang SP</span>, <span class="author">Wu W</span>, <span class="author">Chou WC</span>, <span class="author">Srivastava A</span>, <span class="author">Shaw TI</span>, <span class="author">Ruby JG</span>, <span class="author">Skewes-Cox P</span>, <span class="author">Betegon M</span>, <span class="author">Dimon MT</span>, <span class="author">Solovyev V</span>, <span class="author">Seledtsov I</span>, <span class="author">Kosarev P</span>, <span class="author">Vorobyev D</span>, <span class="author">Ramirez-Gonzalez R</span>, <span class="author">Leggett R</span>, <span class="author">MacLean D</span>, <span class="author">Xia F</span>, <span class="author">Luo R</span>, <span class="author">Li Z</span>, <span class="author">Xie Y</span>, <span class="author">Liu B</span>, <span class="author">Gnerre S</span>, <span class="author">MacCallum I</span>, <span class="author">Przybylski D</span>, <span class="author">Ribeiro FJ</span>, <span class="author">Yin S</span>, <span class="author">Sharpe T</span>, <span class="author">Hall G</span>, <span class="author">Kersey PJ</span>, <span class="author">Durbin R</span>, <span class="author">Jackman SD</span>, <span class="author">Chapman JA</span>, <span class="author">Huang X</span>, <span class="author">DeRisi JL</span>, <span class="author">Caccamo M</span>, <span class="author">Li Y</span>, <span class="author">Jaffe DB</span>, <span class="author">Green RE</span>, <span class="author">Haussler D</span>, <span class="author">Korf I</span> and <span class="author">Paten B</span></p>
<p class="affiliation">Center for Biomolecular Science and Engineering, University of California, Santa Cruz, California 95064, USA.</p><p class="abstract">Low-cost short read sequencing technology has revolutionized genomics, though it is only just becoming practical for the high-quality de novo assembly of a novel large genome. We describe the Assemblathon 1 competition, which aimed to comprehensively assess the state of the art in de novo assembly methods when applied to current sequencing technologies. In a collaborative effort, teams were asked to assemble a simulated Illumina HiSeq data set of an unknown, simulated diploid genome. A total of 41 assemblies from 17 different groups were received. Novel haplotype aware assessments of coverage, contiguity, structure, base calling, and copy number were made. We establish that within this benchmark: (1) It is possible to assemble the genome to a high level of coverage and accuracy, and that (2) large differences exist between the assemblies, suggesting room for further improvements in current methods. The simulated benchmark, including the correct answer, the assemblies, and the code that was used to evaluate the assemblies is now public and freely available from http://www.assemblathon.org/.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCI NIH HHS: <span class="grant">1U24CA143858-01</span>; NHGRI NIH HHS: <span class="grant">HG00064</span>, <span class="grant">P41HG002371</span>, <span class="grant">U01HG004695</span>, <span class="grant">U41HG004568</span>, <span class="grant">U54HG004555</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2224-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21926179" rel="external">21926179</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3227110" rel="external">3227110</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.126599.111" rel="external">10.1101/gr.126599.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Retrospective application of transposon-directed insertion site sequencing to a library of signature-tagged mini-Tn5Km2 mutants of Escherichia coli O157:H7 screened in cattle.</h4>
<p class="authors"><span class="author">Eckert SE</span>, <span class="author">Dziva F</span>, <span class="author">Chaudhuri RR</span>, <span class="author">Langridge GC</span>, <span class="author">Turner DJ</span>, <span class="author">Pickard DJ</span>, <span class="author">Maskell DJ</span>, <span class="author">Thomson NR</span> and <span class="author">Stevens MP</span></p>
<p class="affiliation">Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Bush Farm Road, Roslin, Midlothian EH25 9RG, United Kingdom.</p><p class="abstract">Massively parallel sequencing of transposon-flanking regions assigned the genotype and fitness score to 91% of Escherichia coli O157:H7 mutants previously screened in cattle by signature-tagged mutagenesis (STM). The method obviates the limitations of STM and markedly extended the functional annotation of the prototype E. coli O157:H7 genome without further animal use.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">D017556</span>, <span class="grant">D017947</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2011</span>;<span class="volume">193</span>;<span class="number">7</span>;<span class="pagerange">1771-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21278291" rel="external">21278291</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3067669" rel="external">3067669</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01292-10" rel="external">10.1128/JB.01292-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human postsynaptic density shares conserved elements with proteomes of unicellular eukaryotes and prokaryotes.</h4>
<p class="authors"><span class="author">Emes RD</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">School of Veterinary Medicine and Science, University of Nottingham Leicestershire, UK.</p><p class="abstract">The animal nervous system processes information from the environment and mediates learning and memory using molecular signaling pathways in the postsynaptic terminal of synapses. Postsynaptic neurotransmitter receptors assemble to form multiprotein complexes that drive signal transduction pathways to downstream cell biological processes. Studies of mouse and Drosophila postsynaptic proteins have identified key roles in synaptic physiology and behavior for a wide range of proteins including receptors, scaffolds, enzymes, structural, translational, and transcriptional regulators. Comparative proteomic and genomic studies identified components of the postsynaptic proteome conserved in eukaryotes and early metazoans. We extend these studies, and examine the conservation of genes and domains found in the human postsynaptic density with those across the three superkingdoms, archaeal, bacteria, and eukaryota. A conserved set of proteins essential for basic cellular functions were conserved across the three superkingdoms, whereas synaptic structural and many signaling molecules were specific to the eukaryote lineage. Genes involved with metabolism and environmental signaling in Escherichia coli including the chemotactic and ArcAB Two-Component signal transduction systems shared homologous genes in the mammalian postsynaptic proteome. These data suggest conservation between prokaryotes and mammalian synapses of signaling mechanisms from receptors to transcriptional responses, a process essential to learning and memory in vertebrates. A number of human postsynaptic proteins with homologs in prokaryotes are mutated in human genetic diseases with nervous system pathology. These data also indicate that structural and signaling proteins characteristic of postsynaptic complexes arose in the eukaryotic lineage and rapidly expanded following the emergence of the metazoa, and provide an insight into the early evolution of synaptic mechanisms and conserved mechanisms of learning and memory.</p>
<p class="publication"><span class="title">Frontiers in neuroscience</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="pagerange">44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21503141" rel="external">21503141</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3071500" rel="external">3071500</a>; DOI: <a href="http://dx.doi.org/10.3389%2Ffnins.2011.00044" rel="external">10.3389/fnins.2011.00044</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A user&#39;s guide to the encyclopedia of DNA elements (ENCODE).</h4>
<p class="authors"><span class="author">ENCODE Project Consortium</span></p>
<p class="affiliation">HudsonAlpha Institute for Biotechnology, Huntsville, Alabama, United States of America. <a href="mailto:%72%6D%79%65%72%73%40%68%75%64%73%6F%6E%61%6C%70%68%61.%6F%72%67">&#114;&#109;&#121;&#101;&#114;&#115;&#64;&#104;&#117;&#100;&#115;&#111;&#110;&#97;&#108;&#112;&#104;&#97;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The mission of the Encyclopedia of DNA Elements (ENCODE) Project is to enable the scientific and medical communities to interpret the human genome sequence and apply it to understand human biology and improve health. The ENCODE Consortium is integrating multiple technologies and approaches in a collective effort to discover and define the functional elements encoded in the human genome, including genes, transcripts, and transcriptional regulatory regions, together with their attendant chromatin states and DNA methylation patterns. In the process, standards to ensure high-quality data have been implemented, and novel algorithms have been developed to facilitate analysis. Data and derived results are made available through a freely accessible database. Here we provide an overview of the project and the resources it is generating and illustrate the application of ENCODE data to interpret the human genome.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG003143</span>, <span class="grant">R01 HG004037</span>, <span class="grant">RC2 HG005573</span>; NIDDK NIH HHS: <span class="grant">R01 DK054369</span>, <span class="grant">R01 DK065806</span>; Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">4</span>;<span class="pagerange">e1001046</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21526222">21526222</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3079585">3079585</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1001046">10.1371/journal.pbio.1001046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies confirms a susceptibility locus for knee osteoarthritis on chromosome 7q22.</h4>
<p class="authors"><span class="author">Evangelou E</span>, <span class="author">Valdes AM</span>, <span class="author">Kerkhof HJ</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Zhu Y</span>, <span class="author">Meulenbelt I</span>, <span class="author">Lories RJ</span>, <span class="author">Karassa FB</span>, <span class="author">Tylzanowski P</span>, <span class="author">Bos SD</span>, <span class="author">arcOGEN Consortium</span>, <span class="author">Akune T</span>, <span class="author">Arden NK</span>, <span class="author">Carr A</span>, <span class="author">Chapman K</span>, <span class="author">Cupples LA</span>, <span class="author">Dai J</span>, <span class="author">Deloukas P</span>, <span class="author">Doherty M</span>, <span class="author">Doherty S</span>, <span class="author">Engstrom G</span>, <span class="author">Gonzalez A</span>, <span class="author">Halldorsson BV</span>, <span class="author">Hammond CL</span>, <span class="author">Hart DJ</span>, <span class="author">Helgadottir H</span>, <span class="author">Hofman A</span>, <span class="author">Ikegawa S</span>, <span class="author">Ingvarsson T</span>, <span class="author">Jiang Q</span>, <span class="author">Jonsson H</span>, <span class="author">Kaprio J</span>, <span class="author">Kawaguchi H</span>, <span class="author">Kisand K</span>, <span class="author">Kloppenburg M</span>, <span class="author">Kujala UM</span>, <span class="author">Lohmander LS</span>, <span class="author">Loughlin J</span>, <span class="author">Luyten FP</span>, <span class="author">Mabuchi A</span>, <span class="author">McCaskie A</span>, <span class="author">Nakajima M</span>, <span class="author">Nilsson PM</span>, <span class="author">Nishida N</span>, <span class="author">Ollier WE</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">van de Putte T</span>, <span class="author">Ralston SH</span>, <span class="author">Rivadeneira F</span>, <span class="author">Saarela J</span>, <span class="author">Schulte-Merker S</span>, <span class="author">Shi D</span>, <span class="author">Slagboom PE</span>, <span class="author">Sudo A</span>, <span class="author">Tamm A</span>, <span class="author">Tamm A</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Tsezou A</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson JM</span>, <span class="author">Yoshimura N</span>, <span class="author">Zeggini E</span>, <span class="author">Zhai G</span>, <span class="author">Zhang F</span>, <span class="author">Jonsdottir I</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Felson DT</span>, <span class="author">van Meurs JB</span>, <span class="author">Stefansson K</span>, <span class="author">Ioannidis JP</span>, <span class="author">Spector TD</span> and <span class="author">Translation Research in Europe Applied Technologies for Osteoarthritis (TreatOA)</span></p>
<p class="affiliation">Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece.</p><p class="abstract"><strong>Objectives</strong>: Osteoarthritis (OA) is the most prevalent form of arthritis and accounts for substantial morbidity and disability, particularly in older people. It is characterised by changes in joint structure, including degeneration of the articular cartilage, and its aetiology is multifactorial with a strong postulated genetic component.</p><p class="abstract"><strong>Methods</strong>: A meta-analysis was performed of four genome-wide association (GWA) studies of 2371 cases of knee OA and 35 909 controls in Caucasian populations. Replication of the top hits was attempted with data from 10 additional replication datasets.</p><p class="abstract"><strong>Results</strong>: With a cumulative sample size of 6709 cases and 44 439 controls, one genome-wide significant locus was identified on chromosome 7q22 for knee OA (rs4730250, p=9.2 &times; 10&#x207B;&#x2079;), thereby confirming its role as a susceptibility locus for OA.</p><p class="abstract"><strong>Conclusion</strong>: The associated signal is located within a large (500 kb) linkage disequilibrium block that contains six genes: PRKAR2B (protein kinase, cAMP-dependent, regulatory, type II, &beta;), HPB1 (HMG-box transcription factor 1), COG5 (component of oligomeric golgi complex 5), GPR22 (G protein-coupled receptor 22), DUS4L (dihydrouridine synthase 4-like) and BCAP29 (B cell receptor-associated protein 29). Gene expression analyses of the (six) genes in primary cells derived from different joint tissues confirmed expression of all the genes in the joint environment.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17489</span>, <span class="grant">18030</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0100594</span>, <span class="grant">G0901461</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">083948</span>, <span class="grant">088785</span>, <span class="grant">WT079557MA</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Annals of the rheumatic diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">70</span>;<span class="number">2</span>;<span class="pagerange">349-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21068099">21068099</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3615180">3615180</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fard.2010.132787">10.1136/ard.2010.132787</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction between ERAP1 and HLA-B27 in ankylosing spondylitis implicates peptide handling in the mechanism for HLA-B27 in disease susceptibility.</h4>
<p class="authors"><span class="author">Evans DM</span>, <span class="author">Spencer CC</span>, <span class="author">Pointon JJ</span>, <span class="author">Su Z</span>, <span class="author">Harvey D</span>, <span class="author">Kochan G</span>, <span class="author">Oppermann U</span>, <span class="author">Opperman U</span>, <span class="author">Dilthey A</span>, <span class="author">Pirinen M</span>, <span class="author">Stone MA</span>, <span class="author">Appleton L</span>, <span class="author">Moutsianas L</span>, <span class="author">Moutsianis L</span>, <span class="author">Leslie S</span>, <span class="author">Wordsworth T</span>, <span class="author">Kenna TJ</span>, <span class="author">Karaderi T</span>, <span class="author">Thomas GP</span>, <span class="author">Ward MM</span>, <span class="author">Weisman MH</span>, <span class="author">Farrar C</span>, <span class="author">Bradbury LA</span>, <span class="author">Danoy P</span>, <span class="author">Inman RD</span>, <span class="author">Maksymowych W</span>, <span class="author">Gladman D</span>, <span class="author">Rahman P</span>, <span class="author">Spondyloarthritis Research Consortium of Canada (SPARCC)</span>, <span class="author">Morgan A</span>, <span class="author">Marzo-Ortega H</span>, <span class="author">Bowness P</span>, <span class="author">Gaffney K</span>, <span class="author">Gaston JS</span>, <span class="author">Smith M</span>, <span class="author">Bruges-Armas J</span>, <span class="author">Couto AR</span>, <span class="author">Sorrentino R</span>, <span class="author">Paladini F</span>, <span class="author">Ferreira MA</span>, <span class="author">Xu H</span>, <span class="author">Liu Y</span>, <span class="author">Jiang L</span>, <span class="author">Lopez-Larrea C</span>, <span class="author">D&iacute;az-Pe&ntilde;a R</span>, <span class="author">L&oacute;pez-V&aacute;zquez A</span>, <span class="author">Zayats T</span>, <span class="author">Band G</span>, <span class="author">Bellenguez C</span>, <span class="author">Blackburn H</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Casas JP</span>, <span class="author">Corvin A</span>, <span class="author">Craddock N</span>, <span class="author">Deloukas P</span>, <span class="author">Dronov S</span>, <span class="author">Duncanson A</span>, <span class="author">Edkins S</span>, <span class="author">Freeman C</span>, <span class="author">Gillman M</span>, <span class="author">Gray E</span>, <span class="author">Gwilliam R</span>, <span class="author">Hammond N</span>, <span class="author">Hunt SE</span>, <span class="author">Jankowski J</span>, <span class="author">Jayakumar A</span>, <span class="author">Langford C</span>, <span class="author">Liddle J</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">McCann OT</span>, <span class="author">McCarthy MI</span>, <span class="author">Palmer CN</span>, <span class="author">Peltonen L</span>, <span class="author">Plomin R</span>, <span class="author">Potter SC</span>, <span class="author">Rautanen A</span>, <span class="author">Ravindrarajah R</span>, <span class="author">Ricketts M</span>, <span class="author">Samani N</span>, <span class="author">Sawcer SJ</span>, <span class="author">Strange A</span>, <span class="author">Trembath RC</span>, <span class="author">Viswanathan AC</span>, <span class="author">Waller M</span>, <span class="author">Weston P</span>, <span class="author">Whittaker P</span>, <span class="author">Widaa S</span>, <span class="author">Wood NW</span>, <span class="author">McVean G</span>, <span class="author">Reveille JD</span>, <span class="author">Wordsworth BP</span>, <span class="author">Brown MA</span>, <span class="author">Donnelly P</span>, <span class="author">Australo-Anglo-American Spondyloarthritis Consortium (TASC)</span> and <span class="author">Wellcome Trust Case Control Consortium 2 (WTCCC2)</span></p>
<p class="affiliation">Medical Research Council (MRC) Centre for Causal Analyses in Translational Epidemiology, School of Social and Community Medicine, University of Bristol, Bristol, UK.</p><p class="abstract">Ankylosing spondylitis is a common form of inflammatory arthritis predominantly affecting the spine and pelvis that occurs in approximately 5 out of 1,000 adults of European descent. Here we report the identification of three variants in the RUNX3, LTBR-TNFRSF1A and IL12B regions convincingly associated with ankylosing spondylitis (P &lt; 5 &times; 10(-8) in the combined discovery and replication datasets) and a further four loci at PTGER4, TBKBP1, ANTXR2 and CARD9 that show strong association across all our datasets (P &lt; 5 &times; 10(-6) overall, with support in each of the three datasets studied). We also show that polymorphisms of ERAP1, which encodes an endoplasmic reticulum aminopeptidase involved in peptide trimming before HLA class I presentation, only affect ankylosing spondylitis risk in HLA-B27-positive individuals. These findings provide strong evidence that HLA-B27 operates in ankylosing spondylitis through a mechanism involving aberrant processing of antigenic peptides.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">18797</span>, <span class="grant">19536</span>; Canadian Institutes of Health Research; Medical Research Council: <span class="grant">G0000934</span>; NCRR NIH HHS: <span class="grant">MO1-RR00425</span>, <span class="grant">UL1RR024188</span>; NIAMS NIH HHS: <span class="grant">R01-AR046208</span>; PHS HHS: <span class="grant">P01-052915</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113</span>, <span class="grant">083948/Z/07/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">8</span>;<span class="pagerange">761-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21743469">21743469</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.873">10.1038/ng.873</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential protein expression throughout the life cycle of Trypanosoma congolense, a major parasite of cattle in Africa.</h4>
<p class="authors"><span class="author">Eyford BA</span>, <span class="author">Sakurai T</span>, <span class="author">Smith D</span>, <span class="author">Loveless B</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Donelson JE</span>, <span class="author">Inoue N</span> and <span class="author">Pearson TW</span></p>
<p class="affiliation">Department of Biochemistry and Microbiology, University of Victoria, Victoria, British Columbia, Canada.</p><p class="abstract">Trypanosoma congolense is an important pathogen of livestock in Africa. To study protein expression throughout the T. congolense life cycle, we used culture-derived parasites of each of the three main insect stages and bloodstream stage parasites isolated from infected mice, to perform differential protein expression analysis. Three complete biological replicates of all four life cycle stages were produced from T. congolense IL3000, a cloned parasite that is amenable to culture of major life cycle stages in vitro. Cellular proteins from each life cycle stage were trypsin digested and the resulting peptides were labeled with isobaric tags for relative and absolute quantification (iTRAQ). The peptides were then analyzed by tandem mass spectrometry (MS/MS). This method was used to identify and relatively quantify proteins from the different life cycle stages in the same experiment. A search of the Wellcome Trust&#39;s Sanger Institute&#39;s semi-annotated T. congolense database was performed using the MS/MS fragmentation data to identify the corresponding source proteins. A total of 2088 unique protein sequences were identified, representing 23% of the &sim;9000 proteins predicted for the T. congolense proteome. The 1291 most confidently identified proteins were prioritized for further study. Of these, 784 yielded annotated hits while 501 were described as &quot;hypothetical proteins&quot;. Six proteins showed no significant sequence similarity to any known proteins (from any species) and thus represent new, previously uncharacterized T. congolense proteins. Of particular interest among the remainder are several membrane molecules that showed drastic differential expression, including, not surprisingly, the well-studied variant surface glycoproteins (VSGs), invariant surface glycoproteins (ISGs) 65 and 75, congolense epimastigote specific protein (CESP), the surface protease GP63, an amino acid transporter, a pteridine transporter and a haptoglobin-hemoglobin receptor. Several of these surface disposed proteins are of functional interest as they are necessary for survival of the parasites.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2011</span>;<span class="volume">177</span>;<span class="number">2</span>;<span class="pagerange">116-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21354217">21354217</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2011.02.009">10.1016/j.molbiopara.2011.02.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Examining the overlap between genome-wide rare variant association signals and linkage peaks in rheumatoid arthritis.</h4>
<p class="authors"><span class="author">Eyre S</span>, <span class="author">Ke X</span>, <span class="author">Lawrence R</span>, <span class="author">Bowes J</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Barton A</span>, <span class="author">Thomson W</span>, <span class="author">Worthington J</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">University of Manchester, Manchester, UK.</p><p class="abstract"><strong>Objective</strong>: With the exception of the major histocompatibility complex (MHC) and STAT4, no other rheumatoid arthritis (RA) linkage peak has been successfully fine-mapped to date. This apparent failure to identify association under peaks of linkage could be ascribed to the examination of common variation, when linkage is likely to be driven by rare variants. The purpose of this study was to investigate the overlap between genome-wide rare variant RA association signals observed in the Wellcome Trust Case Control Consortium (WTCCC) study and 11 replicating RA linkage peaks, defined as regions with evidence for linkage in &gt;1 study.</p><p class="abstract"><strong>Methods</strong>: The WTCCC data set contained 40,482 variants with minor allele frequency of &le;0.05 in 1,860 RA patients and 2,938 controls. Genotypes of all rare variants within a given gene region were collapsed into a single locus and a global P value was calculated per gene.</p><p class="abstract"><strong>Results</strong>: The distribution of rare variant signals (association P&le;10(-5)) was found to differ significantly between regions with and without linkage evidence (P=2&times;10(-17) by Fisher&#39;s exact test). No significant difference was observed after data from the MHC region were removed or when the effect of the HLA-DRB1 locus was accounted for.</p><p class="abstract"><strong>Conclusion</strong>: The results suggest that rare variant association signals are significantly overrepresented under linkage peaks in RA, but the effect is driven by the MHC. This is the first study to examine the overlap between linkage peaks and rare variant association signals genome-wide in a complex disease.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>, <span class="grant">18030</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">079557MA</span>, <span class="grant">088885</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Arthritis and rheumatism</span>&nbsp;<span class="year">2011</span>;<span class="volume">63</span>;<span class="number">6</span>;<span class="pagerange">1522-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21370227">21370227</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428937">3428937</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fart.30315">10.1002/art.30315</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Troponin T is essential for sarcomere assembly in zebrafish skeletal muscle.</h4>
<p class="authors"><span class="author">Ferrante MI</span>, <span class="author">Kiff RM</span>, <span class="author">Goulding DA</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">In striated muscle, the basic contractile unit is the sarcomere, which comprises myosin-rich thick filaments intercalated with thin filaments made of actin, tropomyosin and troponin. Troponin is required to regulate Ca(2+)-dependent contraction, and mutant forms of troponins are associated with muscle diseases. We have disrupted several genes simultaneously in zebrafish embryos and have followed the progression of muscle degeneration in the absence of troponin. Complete loss of troponin T activity leads to loss of sarcomere structure, in part owing to the destructive nature of deregulated actin-myosin activity. When troponin T and myosin activity are simultaneously disrupted, immature sarcomeres are rescued. However, tropomyosin fails to localise to sarcomeres, and intercalating thin filaments are missing from electron microscopic cross-sections, indicating that loss of troponin T affects thin filament composition. If troponin activity is only partially disrupted, myofibrils are formed but eventually disintegrate owing to deregulated actin-myosin activity. We conclude that the troponin complex has at least two distinct activities: regulation of actin-myosin activity and, independently, a role in the proper assembly of thin filaments. Our results also indicate that sarcomere assembly can occur in the absence of normal thin filaments.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 077037/Z/05/Z</span>, <span class="grant">WT 077047/Z/05/Z</span></p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2011</span>;<span class="volume">124</span>;<span class="number">Pt 4</span>;<span class="pagerange">565-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21245197">21245197</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3031369">3031369</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fjcs.071274">10.1242/jcs.071274</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Genomic Standards Consortium.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Amaral-Zettler L</span>, <span class="author">Cochrane G</span>, <span class="author">Cole JR</span>, <span class="author">Dawyndt P</span>, <span class="author">Garrity GM</span>, <span class="author">Gilbert J</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Hirschman L</span>, <span class="author">Karsch-Mizrachi I</span>, <span class="author">Klenk HP</span>, <span class="author">Knight R</span>, <span class="author">Kottmann R</span>, <span class="author">Kyrpides N</span>, <span class="author">Meyer F</span>, <span class="author">San Gil I</span>, <span class="author">Sansone SA</span>, <span class="author">Schriml LM</span>, <span class="author">Sterk P</span>, <span class="author">Tatusova T</span>, <span class="author">Ussery DW</span>, <span class="author">White O</span> and <span class="author">Wooley J</span></p>
<p class="affiliation">Centre for Ecology &amp; Hydrology, Maclean Building, Crowmarsh Gifford, Wallingford, Oxfordshire, United Kingdom. <a href="mailto:%64%66%69%65%6C%64%40%63%65%68.%61%63.%75%6B">&#100;&#102;&#105;&#101;&#108;&#100;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A vast and rich body of information has grown up as a result of the world&#39;s enthusiasm for &#39;omics technologies. Finding ways to describe and make available this information that maximise its usefulness has become a major effort across the &#39;omics world. At the heart of this effort is the Genomic Standards Consortium (GSC), an open-membership organization that drives community-based standardization activities, Here we provide a short history of the GSC, provide an overview of its range of current activities, and make a call for the scientific community to join forces to improve the quality and quantity of contextual information about our public collections of genomes, metagenomes, and marker gene sequences.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">6</span>;<span class="pagerange">e1001088</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21713030">21713030</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3119656">3119656</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1001088">10.1371/journal.pbio.1001088</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A call for papers for the second special issue of SIGS from the genomic standards consortium</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">STERK P</span>, <span class="author">Kottmann R</span></p>
<p class="publication"><span class="title">Standards in Genomic Sciences</span> <span class="year">2011</span>;<span class="volume">4</span>;<span class="pagerange">111-2</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Deciphering Developmental Disorders (DDD) study.</h4>
<p class="authors"><span class="author">Firth HV</span>, <span class="author">Wright CF</span> and <span class="author">DDD Study</span></p>
<p class="affiliation">Department of Medical Genetics, Cambridge University Hospitals Foundation Trust, Cambridge, UK.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Developmental medicine and child neurology</span>&nbsp;<span class="year">2011</span>;<span class="volume">53</span>;<span class="number">8</span>;<span class="pagerange">702-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21679367">21679367</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1469-8749.2011.04032.x">10.1111/j.1469-8749.2011.04032.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Germline fitness-based scoring of cancer mutations.</h4>
<p class="authors"><span class="author">Fischer A</span>, <span class="author">Greenman C</span> and <span class="author">Mustonen V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">A key goal in cancer research is to find the genomic alterations that underlie malignant cells. Genomics has proved successful in identifying somatic variants at a large scale. However, it has become evident that a typical cancer exhibits a heterogenous mutation pattern across samples. Cases where the same alteration is observed repeatedly seem to be the exception rather than the norm. Thus, pinpointing the key alterations (driver mutations) from a background of variations with no direct causal link to cancer (passenger mutations) is difficult. Here we analyze somatic missense mutations from cancer samples and their healthy tissue counterparts (germline mutations) from the viewpoint of germline fitness. We calibrate a scoring system from protein domain alignments to score mutations and their target loci. We show first that this score predicts to a good degree the rate of polymorphism of the observed germline variation. The scoring is then applied to somatic mutations. We show that candidate cancer genes prone to copy number loss harbor mutations with germline fitness effects that are significantly more deleterious than expected by chance. This suggests that missense mutations play a driving role in tumor suppressor genes. Furthermore, these mutations fall preferably onto loci in sequence neighborhoods that are high scoring in terms of germline fitness. In contrast, for somatic mutations in candidate onco genes we do not observe a statistically significant effect. These results help to inform how to exploit germline fitness predictions in discovering new genes and mutations responsible for cancer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">091747</span></p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">188</span>;<span class="number">2</span>;<span class="pagerange">383-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21441214">21441214</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3122307">3122307</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.111.127480">10.1534/genetics.111.127480</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">aCGH.Spline--an R package for aCGH dye bias normalization.</h4>
<p class="authors"><span class="author">Fitzgerald TW</span>, <span class="author">Larcombe LD</span>, <span class="author">Le Scouarnec S</span>, <span class="author">Clayton S</span>, <span class="author">Rajan D</span>, <span class="author">Carter NP</span> and <span class="author">Redon R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%74%66%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#102;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: The careful normalization of array-based comparative genomic hybridization (aCGH) data is of critical importance for the accurate detection of copy number changes. The difference in labelling affinity between the two fluorophores used in aCGH-usually Cy5 and Cy3-can be observed as a bias within the intensity distributions. If left unchecked, this bias is likely to skew data interpretation during downstream analysis and lead to an increased number of false discoveries.</p><p class="abstract"><strong>Results</strong>: In this study, we have developed aCGH.Spline, a natural cubic spline interpolation method followed by linear interpolation of outlier values, which is able to remove a large portion of the dye bias from large aCGH datasets in a quick and efficient manner.</p><p class="abstract"><strong>Conclusions</strong>: We have shown that removing this bias and reducing the experimental noise has a strong positive impact on the ability to detect accurately both copy number variation (CNV) and copy number alterations (CNA).</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">9</span>;<span class="pagerange">1195-200</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21357574">21357574</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3077069">3077069</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr107">10.1093/bioinformatics/btr107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2011.</h4>
<p class="authors"><span class="author">Flicek P</span>, <span class="author">Amode MR</span>, <span class="author">Barrell D</span>, <span class="author">Beal K</span>, <span class="author">Brent S</span>, <span class="author">Chen Y</span>, <span class="author">Clapham P</span>, <span class="author">Coates G</span>, <span class="author">Fairley S</span>, <span class="author">Fitzgerald S</span>, <span class="author">Gordon L</span>, <span class="author">Hendrix M</span>, <span class="author">Hourlier T</span>, <span class="author">Johnson N</span>, <span class="author">K&auml;h&auml;ri A</span>, <span class="author">Keefe D</span>, <span class="author">Keenan S</span>, <span class="author">Kinsella R</span>, <span class="author">Kokocinski F</span>, <span class="author">Kulesha E</span>, <span class="author">Larsson P</span>, <span class="author">Longden I</span>, <span class="author">McLaren W</span>, <span class="author">Overduin B</span>, <span class="author">Pritchard B</span>, <span class="author">Riat HS</span>, <span class="author">Rios D</span>, <span class="author">Ritchie GR</span>, <span class="author">Ruffier M</span>, <span class="author">Schuster M</span>, <span class="author">Sobral D</span>, <span class="author">Spudich G</span>, <span class="author">Tang YA</span>, <span class="author">Trevanion S</span>, <span class="author">Vandrovcova J</span>, <span class="author">Vilella AJ</span>, <span class="author">White S</span>, <span class="author">Wilder SP</span>, <span class="author">Zadissa A</span>, <span class="author">Zamora J</span>, <span class="author">Aken BL</span>, <span class="author">Birney E</span>, <span class="author">Cunningham F</span>, <span class="author">Dunham I</span>, <span class="author">Durbin R</span>, <span class="author">Fern&aacute;ndez-Suarez XM</span>, <span class="author">Herrero J</span>, <span class="author">Hubbard TJ</span>, <span class="author">Parker A</span>, <span class="author">Proctor G</span>, <span class="author">Vogel J</span> and <span class="author">Searle SM</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%66%6C%69%63%65%6B%40%65%62%69.%61%63.%75%6B">&#102;&#108;&#105;&#99;&#101;&#107;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl project (http://www.ensembl.org) seeks to enable genomic science by providing high quality, integrated annotation on chordate and selected eukaryotic genomes within a consistent and accessible infrastructure. All supported species include comprehensive, evidence-based gene annotations and a selected set of genomes includes additional data focused on variation, comparative, evolutionary, functional and regulatory annotation. The most advanced resources are provided for key species including human, mouse, rat and zebrafish reflecting the popularity and importance of these species in biomedical research. As of Ensembl release 59 (August 2010), 56 species are supported of which 5 have been added in the past year. Since our previous report, we have substantially improved the presentation and integration of both data of disease relevance and the regulatory state of different cell types.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/B/13438</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">Database issue</span>;<span class="pagerange">D800-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21045057">21045057</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3013672">3013672</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq1064">10.1093/nar/gkq1064</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Salmonella bongori provides insights into the evolution of the Salmonellae.</h4>
<p class="authors"><span class="author">Fookes M</span>, <span class="author">Schroeder GN</span>, <span class="author">Langridge GC</span>, <span class="author">Blondel CJ</span>, <span class="author">Mammina C</span>, <span class="author">Connor TR</span>, <span class="author">Seth-Smith H</span>, <span class="author">Vernikos GS</span>, <span class="author">Robinson KS</span>, <span class="author">Sanders M</span>, <span class="author">Petty NK</span>, <span class="author">Kingsley RA</span>, <span class="author">B&auml;umler AJ</span>, <span class="author">Nuccio SP</span>, <span class="author">Contreras I</span>, <span class="author">Santiviago CA</span>, <span class="author">Maskell D</span>, <span class="author">Barrow P</span>, <span class="author">Humphrey T</span>, <span class="author">Nastasi A</span>, <span class="author">Roberts M</span>, <span class="author">Frankel G</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The genus Salmonella contains two species, S. bongori and S. enterica. Compared to the well-studied S. enterica there is a marked lack of information regarding the genetic makeup and diversity of S. bongori. S. bongori has been found predominantly associated with cold-blooded animals, but it can infect humans. To define the phylogeny of this species, and compare it to S. enterica, we have sequenced 28 isolates representing most of the known diversity of S. bongori. This cross-species analysis allowed us to confidently differentiate ancestral functions from those acquired following speciation, which include both metabolic and virulence-associated capacities. We show that, although S. bongori inherited a basic set of Salmonella common virulence functions, it has subsequently elaborated on this in a different direction to S. enterica. It is an established feature of S. enterica evolution that the acquisition of the type III secretion systems (T3SS-1 and T3SS-2) has been followed by the sequential acquisition of genes encoding secreted targets, termed effectors proteins. We show that this is also true of S. bongori, which has acquired an array of novel effector proteins (sboA-L). All but two of these effectors have no significant S. enterica homologues and instead are highly similar to those found in enteropathogenic Escherichia coli (EPEC). Remarkably, SboH is found to be a chimeric effector protein, encoded by a fusion of the T3SS-1 effector gene sopA and a gene highly similar to the EPEC effector nleH from enteropathogenic E. coli. We demonstrate that representatives of these new effectors are translocated and that SboH, similarly to NleH, blocks intrinsic apoptotic pathways while being targeted to the mitochondria by the SopA part of the fusion. This work suggests that S. bongori has inherited the ancestral Salmonella virulence gene set, but has adapted by incorporating virulence determinants that resemble those employed by EPEC.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">076964</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">8</span>;<span class="pagerange">e1002191</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21876672">21876672</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3158058">3158058</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002191">10.1371/journal.ppat.1002191</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">COSMIC: mining complete cancer genomes in the Catalogue of Somatic Mutations in Cancer.</h4>
<p class="authors"><span class="author">Forbes SA</span>, <span class="author">Bindal N</span>, <span class="author">Bamford S</span>, <span class="author">Cole C</span>, <span class="author">Kok CY</span>, <span class="author">Beare D</span>, <span class="author">Jia M</span>, <span class="author">Shepherd R</span>, <span class="author">Leung K</span>, <span class="author">Menzies A</span>, <span class="author">Teague JW</span>, <span class="author">Campbell PJ</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA Cambridge, UK.</p><p class="abstract">COSMIC (http://www.sanger.ac.uk/cosmic) curates comprehensive information on somatic mutations in human cancer. Release v48 (July 2010) describes over 136,000 coding mutations in almost 542,000 tumour samples; of the 18,490 genes documented, 4803 (26%) have one or more mutations. Full scientific literature curations are available on 83 major cancer genes and 49 fusion gene pairs (19 new cancer genes and 30 new fusion pairs this year) and this number is continually increasing. Key amongst these is TP53, now available through a collaboration with the IARC p53 database. In addition to data from the Cancer Genome Project (CGP) at the Sanger Institute, UK, and The Cancer Genome Atlas project (TCGA), large systematic screens are also now curated. Major website upgrades now make these data much more mineable, with many new selection filters and graphics. A Biomart is now available allowing more automated data mining and integration with other biological databases. Annotation of genomic features has become a significant focus; COSMIC has begun curating full-genome resequencing experiments, developing new web pages, export formats and graphics styles. With all genomic information recently updated to GRCh37, COSMIC integrates many diverse types of mutation information and is making much closer links with Ensembl and other data resources.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">Database issue</span>;<span class="pagerange">D945-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20952405" rel="external">20952405</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3013785" rel="external">3013785</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq929" rel="external">10.1093/nar/gkq929</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessment of a 44 gene classifier for the evaluation of chronic fatigue syndrome from peripheral blood mononuclear cell gene expression.</h4>
<p class="authors"><span class="author">Frampton D</span>, <span class="author">Kerr J</span>, <span class="author">Harrison TJ</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Department of Infection, Division of Infection and Immunity, University College London, London, United Kingdom.</p><p class="abstract">Chronic fatigue syndrome (CFS) is a clinically defined illness estimated to affect millions of people worldwide causing significant morbidity and an annual cost of billions of dollars. Currently there are no laboratory-based diagnostic methods for CFS. However, differences in gene expression profiles between CFS patients and healthy persons have been reported in the literature. Using mRNA relative quantities for 44 previously identified reporter genes taken from a large dataset comprising both CFS patients and healthy volunteers, we derived a gene profile scoring metric to accurately classify CFS and healthy samples. This metric out-performed any of the reporter genes used individually as a classifier of CFS.To determine whether the reporter genes were robust across populations, we applied this metric to classify a separate blind dataset of mRNA relative quantities from a new population of CFS patients and healthy persons with limited success. Although the metric was able to successfully classify roughly two-thirds of both CFS and healthy samples correctly, the level of misclassification was high. We conclude many of the previously identified reporter genes are study-specific and thus cannot be used as a broad CFS diagnostic.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">e16872</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21479222">21479222</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3068152">3068152</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0016872">10.1371/journal.pone.0016872</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Endogenous ion channel complexes: the NMDA receptor.</h4>
<p class="authors"><span class="author">Frank RA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Cambridge U.K. <a href="mailto:%72%66%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#102;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ionotropic receptors, including the NMDAR (N-methyl-D-aspartate receptor) mediate fast neurotransmission, neurodevelopment, neuronal excitability and learning. In the present article, the structure and function of the NMDAR is reviewed with the aim to condense our current understanding and highlight frontiers where important questions regarding the biology of this receptor remain unanswered. In the second part of the present review, new biochemical and genetic approaches for the investigation of ion channel receptor complexes will be discussed.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">3</span>;<span class="pagerange">707-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21599639" rel="external">21599639</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0390707" rel="external">10.1042/BST0390707</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clustered coding variants in the glutamate receptor complexes of individuals with schizophrenia and bipolar disorder.</h4>
<p class="authors"><span class="author">Frank RA</span>, <span class="author">McRae AF</span>, <span class="author">Pocklington AJ</span>, <span class="author">van de Lagemaat LN</span>, <span class="author">Navarro P</span>, <span class="author">Croning MD</span>, <span class="author">Komiyama NH</span>, <span class="author">Bradley SJ</span>, <span class="author">Challiss RA</span>, <span class="author">Armstrong JD</span>, <span class="author">Finn RD</span>, <span class="author">Malloy MP</span>, <span class="author">MacLean AW</span>, <span class="author">Harris SE</span>, <span class="author">Starr JM</span>, <span class="author">Bhaskar SS</span>, <span class="author">Howard EK</span>, <span class="author">Hunt SE</span>, <span class="author">Coffey AJ</span>, <span class="author">Ranganath V</span>, <span class="author">Deloukas P</span>, <span class="author">Rogers J</span>, <span class="author">Muir WJ</span>, <span class="author">Deary IJ</span>, <span class="author">Blackwood DH</span>, <span class="author">Visscher PM</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">Current models of schizophrenia and bipolar disorder implicate multiple genes, however their biological relationships remain elusive. To test the genetic role of glutamate receptors and their interacting scaffold proteins, the exons of ten glutamatergic &#39;hub&#39; genes in 1304 individuals were re-sequenced in case and control samples. No significant difference in the overall number of non-synonymous single nucleotide polymorphisms (nsSNPs) was observed between cases and controls. However, cluster analysis of nsSNPs identified two exons encoding the cysteine-rich domain and first transmembrane helix of GRM1 as a risk locus with five mutations highly enriched within these domains. A new splice variant lacking the transmembrane GPCR domain of GRM1 was discovered in the human brain and the GRM1 mutation cluster could perturb the regulation of this variant. The predicted effect on individuals harbouring multiple mutations distributed in their ten hub genes was also examined. Diseased individuals possessed an increased load of deleteriousness from multiple concurrent rare and common coding variants. Together, these data suggest a disease model in which the interplay of compound genetic coding variants, distributed among glutamate receptors and their interacting proteins, contribute to the pathogenesis of schizophrenia and bipolar disorders.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/505</span>, <span class="grant">ETM/55</span>; Medical Research Council: <span class="grant">MC_U127592696</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">e19011</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21559497">21559497</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3084736">3084736</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0019011">10.1371/journal.pone.0019011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maternally inherited partial monosomy 9p (pterâ€‰â†’â€‰p24.1) and partial trisomy 20p (pterâ€‰â†’â€‰p12.1) characterized by microarray comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Freitas &Eacute;L</span>, <span class="author">Gribble SM</span>, <span class="author">Simioni M</span>, <span class="author">Vieira TP</span>, <span class="author">Silva-Grecco RL</span>, <span class="author">Balarin MA</span>, <span class="author">Prigmore E</span>, <span class="author">Krepischi-Santos AC</span>, <span class="author">Rosenberg C</span>, <span class="author">Szuhai K</span>, <span class="author">van Haeringen A</span>, <span class="author">Carter NP</span> and <span class="author">Gil-da-Silva-Lopes VL</span></p>
<p class="affiliation">Faculty of Medical Sciences, Department of Medical Genetics, University of Campinas, Campinas, São Paulo, Brazil.</p>
<p class="abstract">We report on a 17-year-old patient with midline defects, ocular hypertelorism, neuropsychomotor development delay, neonatal macrosomy, and dental anomalies. DNA copy number investigations using a Whole Genome TilePath array consisting, of 30K BAC/PAC clones showed a 6.36 Mb deletion in the 9p24.1-p24.3 region and a 14.83 Mb duplication in the 20p12.1-p13 region, which derived from a maternal balanced t(9;20)(p24.1;p12.1) as shown by FISH studies. Monosomy 9p is a well-delineated chromosomal syndrome with characteristic clinical features, while chromosome 20p duplication is a rare genetic condition. Only a handful of cases of monosomy 9/trisomy 20 have been previously described. In this report, we compare the phenotype of our patient with those already reported in the literature, and discuss the role of DMRT, DOCK8, FOXD4, VLDLR, RSPO4, AVP, RASSF2, PROKR2, BMP2, MKKS, and JAG1, all genes mapping to the deleted and duplicated regions.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span>, <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2011</span>;<span class="volume">155A</span>;<span class="number">11</span>;<span class="pagerange">2754-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21948691" rel="external">21948691</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428835" rel="external">3428835</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.34168" rel="external">10.1002/ajmg.a.34168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Perilipin deficiency and autosomal dominant partial lipodystrophy.</h4>
<p class="authors"><span class="author">Gandotra S</span>, <span class="author">Le Dour C</span>, <span class="author">Bottomley W</span>, <span class="author">Cervera P</span>, <span class="author">Giral P</span>, <span class="author">Reznik Y</span>, <span class="author">Charpentier G</span>, <span class="author">Auclair M</span>, <span class="author">Del&eacute;pine M</span>, <span class="author">Barroso I</span>, <span class="author">Semple RK</span>, <span class="author">Lathrop M</span>, <span class="author">Lascols O</span>, <span class="author">Capeau J</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Magr&eacute; J</span>, <span class="author">Savage DB</span> and <span class="author">Vigouroux C</span></p>
<p class="affiliation">University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom.</p><p class="abstract">Perilipin is the most abundant adipocyte-specific protein that coats lipid droplets, and it is required for optimal lipid incorporation and release from the droplet. We identified two heterozygous frameshift mutations in the perilipin gene (PLIN1) in three families with partial lipodystrophy, severe dyslipidemia, and insulin-resistant diabetes. Subcutaneous fat from the patients was characterized by smaller-than-normal adipocytes, macrophage infiltration, and fibrosis. In contrast to wild-type perilipin, mutant forms of the protein failed to increase triglyceride accumulation when expressed heterologously in preadipocytes. These findings define a novel dominant form of inherited lipodystrophy and highlight the serious metabolic consequences of a primary defect in the formation of lipid droplets in adipose tissue.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">091551</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">364</span>;<span class="number">8</span>;<span class="pagerange">740-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21345103">21345103</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3773916">3773916</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1007487">10.1056/NEJMoa1007487</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meticillin-resistant Staphylococcus aureus with a novel mecA homologue in human and bovine populations in the UK and Denmark: a descriptive study.</h4>
<p class="authors"><span class="author">Garc&iacute;a-&Aacute;lvarez L</span>, <span class="author">Holden MT</span>, <span class="author">Lindsay H</span>, <span class="author">Webb CR</span>, <span class="author">Brown DF</span>, <span class="author">Curran MD</span>, <span class="author">Walpole E</span>, <span class="author">Brooks K</span>, <span class="author">Pickard DJ</span>, <span class="author">Teale C</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span>, <span class="author">Edwards GF</span>, <span class="author">Girvan EK</span>, <span class="author">Kearns AM</span>, <span class="author">Pichon B</span>, <span class="author">Hill RL</span>, <span class="author">Larsen AR</span>, <span class="author">Skov RL</span>, <span class="author">Peacock SJ</span>, <span class="author">Maskell DJ</span> and <span class="author">Holmes MA</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Animals can act as a reservoir and source for the emergence of novel meticillin-resistant Staphylococcus aureus (MRSA) clones in human beings. Here, we report the discovery of a strain of S aureus (LGA251) isolated from bulk milk that was phenotypically resistant to meticillin but tested negative for the mecA gene and a preliminary investigation of the extent to which such strains are present in bovine and human populations.</p><p class="abstract"><strong>Methods</strong>: Isolates of bovine MRSA were obtained from the Veterinary Laboratories Agency in the UK, and isolates of human MRSA were obtained from diagnostic or reference laboratories (two in the UK and one in Denmark). From these collections, we searched for mecA PCR-negative bovine and human S aureus isolates showing phenotypic meticillin resistance. We used whole-genome sequencing to establish the genetic basis for the observed antibiotic resistance.</p><p class="abstract"><strong>Findings</strong>: A divergent mecA homologue (mecA(LGA251)) was discovered in the LGA251 genome located in a novel staphylococcal cassette chromosome mec element, designated type-XI SCCmec. The mecA(LGA251) was 70% identical to S aureus mecA homologues and was initially detected in 15 S aureus isolates from dairy cattle in England. These isolates were from three different multilocus sequence type lineages (CC130, CC705, and ST425); spa type t843 (associated with CC130) was identified in 60% of bovine isolates. When human mecA-negative MRSA isolates were tested, the mecA(LGA251) homologue was identified in 12 of 16 isolates from Scotland, 15 of 26 from England, and 24 of 32 from Denmark. As in cows, t843 was the most common spa type detected in human beings.</p><p class="abstract"><strong>Interpretation</strong>: Although routine culture and antimicrobial susceptibility testing will identify S aureus isolates with this novel mecA homologue as meticillin resistant, present confirmatory methods will not identify them as MRSA. New diagnostic guidelines for the detection of MRSA should consider the inclusion of tests for mecA(LGA251).</p><p class="abstract"><strong>Funding</strong>: Department for Environment, Food and Rural Affairs, Higher Education Funding Council for England, Isaac Newton Trust (University of Cambridge), and the Wellcome Trust.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Lancet. Infectious diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="number">8</span>;<span class="pagerange">595-603</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21641281">21641281</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3829197">3829197</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS1473-3099%2811%2970126-8">10.1016/S1473-3099(11)70126-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Protein-based signatures of functional evolution in Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Gardner KB</span>, <span class="author">Sinha I</span>, <span class="author">Bustamante LY</span>, <span class="author">Day NP</span>, <span class="author">White NJ</span> and <span class="author">Woodrow CJ</span></p>
<p class="affiliation">Wellcome Trust Mahidol University-Oxford Tropical Medicine Research Unit (MORU), 420/6 Rajwithi Road, Bangkok, 10400 Thailand.</p>
<p class="abstract">Background: It has been known for over a decade that Plasmodium falciparum proteins are enriched in non-globular domains of unknown function. The potential for these regions of protein sequence to undergo high levels of genetic drift provides a fundamental challenge to attempts to identify the molecular basis of adaptive change in malaria parasites.

Results: Evolutionary comparisons were undertaken using a set of forty P. falciparum metabolic enzyme genes, both within the hominid malaria clade (P. reichenowi) and across the genus (P. chabaudi). All genes contained coding elements highly conserved across the genus, but there were also a large number of regions of weakly or non-aligning coding sequence. These displayed remarkable levels of non-synonymous fixed differences within the hominid malaria clade indicating near complete release from purifying selection (dN/dS ratio at residues non-aligning across genus: 0.64, dN/dS ratio at residues identical across genus: 0.03). Regions of low conservation also possessed high levels of hydrophilicity, a marker of non-globularity. The propensity for such regions to act as potent sources of non-synonymous genetic drift within extant P. falciparum isolates was confirmed at chromosomal regions containing genes known to mediate drug resistance in field isolates, where 150 of 153 amino acid variants were located in poorly conserved regions. In contrast, all 22 amino acid variants associated with drug resistance were restricted to highly conserved regions. Additional mutations associated with laboratory-selected drug resistance, such as those in PfATPase4 selected by spiroindolone, were similarly restricted while mutations in another calcium ATPase (PfSERCA, a gene proposed to mediate artemisinin resistance) that reach significant frequencies in field isolates were located exclusively in poorly conserved regions consistent with genetic drift.

Conclusion: Coding sequences of malaria parasites contain prospectively definable domains subject to neutral or nearly neutral evolution on a scale that appears unrivalled in biology. This distinct evolutionary landscape has potential to confound analytical methods developed for other genera. Against this tide of genetic drift, polymorphisms mediating functional change stand out to such an extent that evolutionary context provides a useful signal for identifying the molecular basis of drug resistance in malaria parasites, a finding that is of relevance to both genome-wide and candidate gene studies in this genus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="pagerange">257</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21917172" rel="external">21917172</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3197514" rel="external">3197514</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-11-257" rel="external">10.1186/1471-2148-11-257</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNIE: genome-wide prediction of bacterial intrinsic terminators.</h4>
<p class="authors"><span class="author">Gardner PP</span>, <span class="author">Barquist L</span>, <span class="author">Bateman A</span>, <span class="author">Nawrocki EP</span> and <span class="author">Weinberg Z</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA0, UK. <a href="mailto:%70%67%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#103;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Bacterial Rho-independent terminators (RITs) are important genomic landmarks involved in gene regulation and terminating gene expression. In this investigation we present RNIE, a probabilistic approach for predicting RITs. The method is based upon covariance models which have been known for many years to be the most accurate computational tools for predicting homology in structural non-coding RNAs. We show that RNIE has superior performance in model species from a spectrum of bacterial phyla. Further analysis of species where a low number of RITs were predicted revealed a highly conserved structural sequence motif enriched near the genic termini of the pathogenic Actinobacteria, Mycobacterium tuberculosis. This motif, together with classical RITs, account for up to 90% of all the significantly structured regions from the termini of M. tuberculosis genic elements. The software, predictions and alignments described below are available from http://github.com/ppgardne/RNIE.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">14</span>;<span class="pagerange">5845-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21478170">21478170</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3152330">3152330</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr168">10.1093/nar/gkr168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rfam: Wikipedia, clans and the &quot;decimal&quot; release.</h4>
<p class="authors"><span class="author">Gardner PP</span>, <span class="author">Daub J</span>, <span class="author">Tate J</span>, <span class="author">Moore BL</span>, <span class="author">Osuch IH</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Finn RD</span>, <span class="author">Nawrocki EP</span>, <span class="author">Kolbe DL</span>, <span class="author">Eddy SR</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA0, USA. <a href="mailto:%70%67%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#103;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Rfam database aims to catalogue non-coding RNAs through the use of sequence alignments and statistical profile models known as covariance models. In this contribution, we discuss the pros and cons of using the online encyclopedia, Wikipedia, as a source of community-derived annotation. We discuss the addition of groupings of related RNA families into clans and new developments to the website. Rfam is available on the Web at http://rfam.sanger.ac.uk.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">077044</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">Database issue</span>;<span class="pagerange">D141-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21062808" rel="external">21062808</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3013711" rel="external">3013711</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq1129" rel="external">10.1093/nar/gkq1129</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of XMRV integration sites from human prostate cancer tissues suggests PCR contamination rather than genuine human infection.</h4>
<p class="authors"><span class="author">Garson JA</span>, <span class="author">Kellam P</span> and <span class="author">Towers GJ</span></p>
<p class="affiliation">MRC Centre for Medical Molecular Virology, Division of Infection and Immunity, University College London, 46 Cleveland St, London W1T 4JF, UK.</p><p class="abstract">XMRV is a gammaretrovirus associated in some studies with human prostate cancer and chronic fatigue syndrome. Central to the hypothesis of XMRV as a human pathogen is the description of integration sites in DNA from prostate tumour tissues. Here we demonstrate that 2 of 14 patient-derived sites are identical to sites cloned in the same laboratory from experimentally infected DU145 cells. Identical integration sites have never previously been described in any retrovirus infection. We propose that the patient-derived sites are the result of PCR contamination. This observation further undermines the notion that XMRV is a genuine human pathogen.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801172</span>, <span class="grant">G9721629</span>; Wellcome Trust: <span class="grant">090940</span>, <span class="grant">WT076608</span>, <span class="grant">WT090940</span></p>
<p class="publication"><span class="title">Retrovirology</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="pagerange">13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21352548">21352548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3050814">3050814</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1742-4690-8-13">10.1186/1742-4690-8-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Towards BioDBcore: a community-defined information specification for biological databases.</h4>
<p class="authors"><span class="author">Gaudet P</span>, <span class="author">Bairoch A</span>, <span class="author">Field D</span>, <span class="author">Sansone SA</span>, <span class="author">Taylor C</span>, <span class="author">Attwood TK</span>, <span class="author">Bateman A</span>, <span class="author">Blake JA</span>, <span class="author">Bult CJ</span>, <span class="author">Cherry JM</span>, <span class="author">Chisholm RL</span>, <span class="author">Cochrane G</span>, <span class="author">Cook CE</span>, <span class="author">Eppig JT</span>, <span class="author">Galperin MY</span>, <span class="author">Gentleman R</span>, <span class="author">Goble CA</span>, <span class="author">Gojobori T</span>, <span class="author">Hancock JM</span>, <span class="author">Howe DG</span>, <span class="author">Imanishi T</span>, <span class="author">Kelso J</span>, <span class="author">Landsman D</span>, <span class="author">Lewis SE</span>, <span class="author">Mizrachi IK</span>, <span class="author">Orchard S</span>, <span class="author">Ouellette BF</span>, <span class="author">Ranganathan S</span>, <span class="author">Richardson L</span>, <span class="author">Rocca-Serra P</span>, <span class="author">Schofield PN</span>, <span class="author">Smedley D</span>, <span class="author">Southan C</span>, <span class="author">Tan TW</span>, <span class="author">Tatusova T</span>, <span class="author">Whetzel PL</span>, <span class="author">White O</span>, <span class="author">Yamasaki C</span> and <span class="author">BioDBCore Working Group</span></p><p class="abstract">The present article proposes the adoption of a community-defined, uniform, generic description of the core attributes of biological databases, BioDBCore. The goals of these attributes are to provide a general overview of the database landscape, to encourage consistency and interoperability between resources and to promote the use of semantic and syntactic standards. BioDBCore will make it easier for users to evaluate the scope and relevance of available resources. This new resource will increase the collective impact of the information present in biological databases.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM064426-10</span>, <span class="grant">R01 GM087371-03</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">Database issue</span>;<span class="pagerange">D7-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21097465" rel="external">21097465</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3013734" rel="external">3013734</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq1173" rel="external">10.1093/nar/gkq1173</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New gene functions in megakaryopoiesis and platelet formation.</h4>
<p class="authors"><span class="author">Gieger C</span>, <span class="author">Radhakrishnan A</span>, <span class="author">Cvejic A</span>, <span class="author">Tang W</span>, <span class="author">Porcu E</span>, <span class="author">Pistis G</span>, <span class="author">Serbanovic-Canic J</span>, <span class="author">Elling U</span>, <span class="author">Goodall AH</span>, <span class="author">Labrune Y</span>, <span class="author">Lopez LM</span>, <span class="author">M&auml;gi R</span>, <span class="author">Meacham S</span>, <span class="author">Okada Y</span>, <span class="author">Pirastu N</span>, <span class="author">Sorice R</span>, <span class="author">Teumer A</span>, <span class="author">Voss K</span>, <span class="author">Zhang W</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Bis JC</span>, <span class="author">Ellinghaus D</span>, <span class="author">G&ouml;gele M</span>, <span class="author">Hottenga JJ</span>, <span class="author">Langenberg C</span>, <span class="author">Kovacs P</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Shin SY</span>, <span class="author">Esko T</span>, <span class="author">Hartiala J</span>, <span class="author">Kanoni S</span>, <span class="author">Murgia F</span>, <span class="author">Parsa A</span>, <span class="author">Stephens J</span>, <span class="author">van der Harst P</span>, <span class="author">Ellen van der Schoot C</span>, <span class="author">Allayee H</span>, <span class="author">Attwood A</span>, <span class="author">Balkau B</span>, <span class="author">Bastardot F</span>, <span class="author">Basu S</span>, <span class="author">Baumeister SE</span>, <span class="author">Biino G</span>, <span class="author">Bomba L</span>, <span class="author">Bonnefond A</span>, <span class="author">Cambien F</span>, <span class="author">Chambers JC</span>, <span class="author">Cucca F</span>, <span class="author">D&#39;Adamo P</span>, <span class="author">Davies G</span>, <span class="author">de Boer RA</span>, <span class="author">de Geus EJ</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Elliott P</span>, <span class="author">Erdmann J</span>, <span class="author">Evans DM</span>, <span class="author">Falchi M</span>, <span class="author">Feng W</span>, <span class="author">Folsom AR</span>, <span class="author">Frazer IH</span>, <span class="author">Gibson QD</span>, <span class="author">Glazer NL</span>, <span class="author">Hammond C</span>, <span class="author">Hartikainen AL</span>, <span class="author">Heckbert SR</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hersch M</span>, <span class="author">Illig T</span>, <span class="author">Loos RJ</span>, <span class="author">Jolley J</span>, <span class="author">Khaw KT</span>, <span class="author">K&uuml;hnel B</span>, <span class="author">Kyrtsonis MC</span>, <span class="author">Lagou V</span>, <span class="author">Lloyd-Jones H</span>, <span class="author">Lumley T</span>, <span class="author">Mangino M</span>, <span class="author">Maschio A</span>, <span class="author">Mateo Leach I</span>, <span class="author">McKnight B</span>, <span class="author">Memari Y</span>, <span class="author">Mitchell BD</span>, <span class="author">Montgomery GW</span>, <span class="author">Nakamura Y</span>, <span class="author">Nauck M</span>, <span class="author">Navis G</span>, <span class="author">N&ouml;thlings U</span>, <span class="author">Nolte IM</span>, <span class="author">Porteous DJ</span>, <span class="author">Pouta A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Pullat J</span>, <span class="author">Ring SM</span>, <span class="author">Rotter JI</span>, <span class="author">Ruggiero D</span>, <span class="author">Ruokonen A</span>, <span class="author">Sala C</span>, <span class="author">Samani NJ</span>, <span class="author">Sambrook J</span>, <span class="author">Schlessinger D</span>, <span class="author">Schreiber S</span>, <span class="author">Schunkert H</span>, <span class="author">Scott J</span>, <span class="author">Smith NL</span>, <span class="author">Snieder H</span>, <span class="author">Starr JM</span>, <span class="author">Stumvoll M</span>, <span class="author">Takahashi A</span>, <span class="author">Tang WH</span>, <span class="author">Taylor K</span>, <span class="author">Tenesa A</span>, <span class="author">Lay Thein S</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Uda M</span>, <span class="author">Ulivi S</span>, <span class="author">van Veldhuisen DJ</span>, <span class="author">Visscher PM</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Wichmann HE</span>, <span class="author">Wiggins KL</span>, <span class="author">Willemsen G</span>, <span class="author">Yang TP</span>, <span class="author">Hua Zhao J</span>, <span class="author">Zitting P</span>, <span class="author">Bradley JR</span>, <span class="author">Dedoussis GV</span>, <span class="author">Gasparini P</span>, <span class="author">Hazen SL</span>, <span class="author">Metspalu A</span>, <span class="author">Pirastu M</span>, <span class="author">Shuldiner AR</span>, <span class="author">Joost van Pelt L</span>, <span class="author">Zwaginga JJ</span>, <span class="author">Boomsma DI</span>, <span class="author">Deary IJ</span>, <span class="author">Franke A</span>, <span class="author">Froguel P</span>, <span class="author">Ganesh SK</span>, <span class="author">Jarvelin MR</span>, <span class="author">Martin NG</span>, <span class="author">Meisinger C</span>, <span class="author">Psaty BM</span>, <span class="author">Spector TD</span>, <span class="author">Wareham NJ</span>, <span class="author">Akkerman JW</span>, <span class="author">Ciullo M</span>, <span class="author">Deloukas P</span>, <span class="author">Greinacher A</span>, <span class="author">Jupe S</span>, <span class="author">Kamatani N</span>, <span class="author">Khadake J</span>, <span class="author">Kooner JS</span>, <span class="author">Penninger J</span>, <span class="author">Prokopenko I</span>, <span class="author">Stemple D</span>, <span class="author">Toniolo D</span>, <span class="author">Wernisch L</span>, <span class="author">Sanna S</span>, <span class="author">Hicks AA</span>, <span class="author">Rendon A</span>, <span class="author">Ferreira MA</span>, <span class="author">Ouwehand WH</span> and <span class="author">Soranzo N</span></p>
<p class="affiliation">Institute of Genetic Epidemiology, Helmholtz Zentrum M&uuml;nchen, German Research Center for Environmental Health, Ingolst&auml;dter Landstr 1, 85764 Neuherberg, Germany. <a href="mailto:%63%68%72%69%73%74%69%61%6E.%67%69%65%67%65%72%40%68%65%6C%6D%68%6F%6C%74%7A-%6D%75%65%6E%63%68%65%6E.%64%65">&#99;&#104;&#114;&#105;&#115;&#116;&#105;&#97;&#110;&#46;&#103;&#105;&#101;&#103;&#101;&#114;&#64;&#104;&#101;&#108;&#109;&#104;&#111;&#108;&#116;&#122;&#45;&#109;&#117;&#101;&#110;&#99;&#104;&#101;&#110;&#46;&#100;&#101;</a></p><p class="abstract">Platelets are the second most abundant cell type in blood and are essential for maintaining haemostasis. Their count and volume are tightly controlled within narrow physiological ranges, but there is only limited understanding of the molecular processes controlling both traits. Here we carried out a high-powered meta-analysis of genome-wide association studies (GWAS) in up to 66,867 individuals of European ancestry, followed by extensive biological and functional assessment. We identified 68 genomic loci reliably associated with platelet count and volume mapping to established and putative novel regulators of megakaryopoiesis and platelet formation. These genes show megakaryocyte-specific gene expression patterns and extensive network connectivity. Using gene silencing in Danio rerio and Drosophila melanogaster, we identified 11 of the genes as novel regulators of blood cell formation. Taken together, our findings advance understanding of novel gene functions controlling fate-determining events during megakaryopoiesis and platelet formation, providing a new example of successful translation of GWAS to function.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F019394/1</span>; British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Chief Scientist Office: <span class="grant">CZB/4/505</span>, <span class="grant">ETM/55</span>; Medical Research Council: <span class="grant">G0601966</span>, <span class="grant">G0700704</span>, <span class="grant">G0700931</span>, <span class="grant">G0701120</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G1000143</span>, <span class="grant">MC_U105260799</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NCRR NIH HHS: <span class="grant">K12 RR023250</span>, <span class="grant">K12 RR023250-05</span>, <span class="grant">M01 RR016500-08</span>, <span class="grant">U54 RR020278-06</span>, <span class="grant">UL1 RR025005</span>, <span class="grant">UL1 RR025005-05</span>; NHGRI NIH HHS: <span class="grant">P41 HG003751</span>, <span class="grant">T32 HG002536</span>; NHLBI NIH HHS: <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC055016</span>, <span class="grant">N01 HC055018</span>, <span class="grant">N01 HC055019</span>, <span class="grant">N01 HC055020</span>, <span class="grant">N01 HC055021</span>, <span class="grant">N01 HC055022</span>, <span class="grant">N01 HC085079</span>, <span class="grant">P01 HL076491</span>, <span class="grant">P01 HL076491-09</span>, <span class="grant">P01 HL098055</span>, <span class="grant">P01 HL098055-03</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL059367-11</span>, <span class="grant">R01 HL068986</span>, <span class="grant">R01 HL068986-06</span>, <span class="grant">R01 HL073410-08</span>, <span class="grant">R01 HL085251</span>, <span class="grant">R01 HL085251-04</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL086694-05</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087641-03</span>, <span class="grant">R01 HL087679-03</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL088119-04</span>, <span class="grant">R01 HL103866</span>, <span class="grant">R01 HL103866-03</span>, <span class="grant">R01 HL105756</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U01 HL084756-03</span>; NIA NIH HHS: <span class="grant">R01 AG018728</span>, <span class="grant">R01 AG018728-05S1</span>; NICHD NIH HHS: <span class="grant">R01 HD042157-01A1</span>; NIDDK NIH HHS: <span class="grant">P30 DK072488</span>, <span class="grant">P30 DK072488-08</span>; NIGMS NIH HHS: <span class="grant">R01 GM053275</span>, <span class="grant">R01 GM053275-14</span>, <span class="grant">U01 GM074518</span>, <span class="grant">U01 GM074518-04</span>; NIMH NIH HHS: <span class="grant">RL1 MH083268</span>, <span class="grant">RL1 MH083268-05</span>; Wellcome Trust: <span class="grant">092731</span>, <span class="grant">098051</span>, <span class="grant">WT077037/Z/05/Z</span>, <span class="grant">WT077047/Z/05/Z</span>, <span class="grant">WT082597/Z/07/Z</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">480</span>;<span class="number">7376</span>;<span class="pagerange">201-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22139419">22139419</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3335296">3335296</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10659">10.1038/nature10659</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants near ATM are associated with glycemic response to metformin in type 2 diabetes.</h4>
<p class="authors"><span class="author">GoDARTS and UKPDS Diabetes Pharmacogenetics Study Group</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Zhou K</span>, <span class="author">Bellenguez C</span>, <span class="author">Spencer CC</span>, <span class="author">Bennett AJ</span>, <span class="author">Coleman RL</span>, <span class="author">Tavendale R</span>, <span class="author">Hawley SA</span>, <span class="author">Donnelly LA</span>, <span class="author">Schofield C</span>, <span class="author">Groves CJ</span>, <span class="author">Burch L</span>, <span class="author">Carr F</span>, <span class="author">Strange A</span>, <span class="author">Freeman C</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Casas JP</span>, <span class="author">Corvin A</span>, <span class="author">Craddock N</span>, <span class="author">Deloukas P</span>, <span class="author">Dronov S</span>, <span class="author">Duncanson A</span>, <span class="author">Edkins S</span>, <span class="author">Gray E</span>, <span class="author">Hunt S</span>, <span class="author">Jankowski J</span>, <span class="author">Langford C</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">Plomin R</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Samani NJ</span>, <span class="author">Trembath R</span>, <span class="author">Viswanathan AC</span>, <span class="author">Wood NW</span>, <span class="author">MAGIC investigators</span>, <span class="author">Harries LW</span>, <span class="author">Hattersley AT</span>, <span class="author">Doney AS</span>, <span class="author">Colhoun H</span>, <span class="author">Morris AD</span>, <span class="author">Sutherland C</span>, <span class="author">Hardie DG</span>, <span class="author">Peltonen L</span>, <span class="author">McCarthy MI</span>, <span class="author">Holman RR</span>, <span class="author">Palmer CN</span>, <span class="author">Donnelly P</span> and <span class="author">Pearson ER</span></p>
<p class="affiliation">Biomedical Research Institute, University of Dundee, Dundee, UK.</p><p class="abstract">Metformin is the most commonly used pharmacological therapy for type 2 diabetes. We report a genome-wide association study for glycemic response to metformin in 1,024 Scottish individuals with type 2 diabetes with replication in two cohorts including 1,783 Scottish individuals and 1,113 individuals from the UK Prospective Diabetes Study. In a combined meta-analysis, we identified a SNP, rs11212617, associated with treatment success (n = 3,920, P = 2.9 &times; 10(-9), odds ratio = 1.35, 95% CI 1.22-1.49) at a locus containing ATM, the ataxia telangiectasia mutated gene. In a rat hepatoma cell line, inhibition of ATM with KU-55933 attenuated the phosphorylation and activation of AMP-activated protein kinase in response to metformin. We conclude that ATM, a gene known to be involved in DNA repair and cell cycle control, plays a role in the effect of metformin upstream of AMP-activated protein kinase, and variation in this gene alters glycemic response to metformin.</p>
<p class="grants">Funded by: Chief Scientist Office; Department of Health: <span class="grant">PDA/02/06/016</span>; Medical Research Council: <span class="grant">G0601261</span>, <span class="grant">G0901310</span>, <span class="grant">G19/2</span>; Wellcome Trust: <span class="grant">084726</span>, <span class="grant">084726/Z/08/Z</span>, <span class="grant">085475/B/08/Z</span>, <span class="grant">085475/Z/08/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">2</span>;<span class="pagerange">117-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21186350">21186350</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3030919">3030919</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.735">10.1038/ng.735</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In situ thrombolysis for cerebral venous thrombosis complicating anti-leukemic therapy.</h4>
<p class="authors"><span class="author">Godfrey AL</span>, <span class="author">Higgins JN</span>, <span class="author">Beer PA</span>, <span class="author">Craig JI</span> and <span class="author">Vassiliou GS</span></p>
<p class="affiliation">Department of Haematology, Addenbrooke&#39;s Hospital, Cambridge Institute for Medical Research, Wellcome Trust/MRC Building, Cambridge, UK. <a href="mailto:%61%6C%70%35%32%40%63%61%6D.%61%63.%75%6B">&#97;&#108;&#112;&#53;&#50;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Leukemia research</span>&nbsp;<span class="year">2011</span>;<span class="volume">35</span>;<span class="number">8</span>;<span class="pagerange">1127-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21444111" rel="external">21444111</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.leukres.2011.02.014" rel="external">10.1016/j.leukres.2011.02.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transition of Plasmodium sporozoites into liver stage-like forms is regulated by the RNA binding protein Pumilio.</h4>
<p class="authors"><span class="author">Gomes-Santos CS</span>, <span class="author">Braks J</span>, <span class="author">Prud&ecirc;ncio M</span>, <span class="author">Carret C</span>, <span class="author">Gomes AR</span>, <span class="author">Pain A</span>, <span class="author">Feltwell T</span>, <span class="author">Khan S</span>, <span class="author">Waters A</span>, <span class="author">Janse C</span>, <span class="author">Mair GR</span> and <span class="author">Mota MM</span></p>
<p class="affiliation">Malaria Unit, Instituto de Medicina Molecular, Lisboa, Portugal.</p><p class="abstract">Many eukaryotic developmental and cell fate decisions that are effected post-transcriptionally involve RNA binding proteins as regulators of translation of key mRNAs. In malaria parasites (Plasmodium spp.), the development of round, non-motile and replicating exo-erythrocytic liver stage forms from slender, motile and cell-cycle arrested sporozoites is believed to depend on environmental changes experienced during the transmission of the parasite from the mosquito vector to the vertebrate host. Here we identify a Plasmodium member of the RNA binding protein family PUF as a key regulator of this transformation. In the absence of Pumilio-2 (Puf2) sporozoites initiate EEF development inside mosquito salivary glands independently of the normal transmission-associated environmental cues. Puf2- sporozoites exhibit genome-wide transcriptional changes that result in loss of gliding motility, cell traversal ability and reduction in infectivity, and, moreover, trigger metamorphosis typical of early Plasmodium intra-hepatic development. These data demonstrate that Puf2 is a key player in regulating sporozoite developmental control, and imply that transformation of salivary gland-resident sporozoites into liver stage-like parasites is regulated by a post-transcriptional mechanism.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">083811</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">e1002046</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21625527">21625527</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3098293">3098293</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002046">10.1371/journal.ppat.1002046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">No evidence of XMRV or related retroviruses in a London HIV-1-positive patient cohort.</h4>
<p class="authors"><span class="author">Gray ER</span>, <span class="author">Garson JA</span>, <span class="author">Breuer J</span>, <span class="author">Edwards S</span>, <span class="author">Kellam P</span>, <span class="author">Pillay D</span> and <span class="author">Towers GJ</span></p>
<p class="affiliation">Department of Infection and Immunity, University College London, London, United Kingdom. <a href="mailto:%65.%67%72%61%79%40%75%63%6C.%61%63.%75%6B">&#101;&#46;&#103;&#114;&#97;&#121;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Several studies have implicated a recently discovered gammaretrovirus, XMRV (Xenotropic murine leukaemia virus-related virus), in chronic fatigue syndrome and prostate cancer, though whether as causative agent or opportunistic infection is unclear. It has also been suggested that the virus can be found circulating amongst the general population. The discovery has been controversial, with conflicting results from attempts to reproduce the original studies.</p><p class="abstract">We extracted peripheral blood DNA from a cohort of 540 HIV-1-positive patients (approximately 20% of whom have never been on anti-retroviral treatment) and determined the presence of XMRV and related viruses using TaqMan PCR. While we were able to amplify as few as 5 copies of positive control DNA, we did not find any positive samples in the patient cohort.</p><p class="abstract">In view of these negative findings in this highly susceptible group, we conclude that it is unlikely that XMRV or related viruses are circulating at a significant level, if at all, in HIV-1-positive patients in London or in the general population.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0801172</span>, <span class="grant">G9721629</span>; Wellcome Trust: <span class="grant">090940</span>, <span class="grant">WT090940</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">e18096</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21448291">21448291</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3063244">3063244</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0018096">10.1371/journal.pone.0018096</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Binding of more than one Tva800 molecule is required for ASLV-A entry.</h4>
<p class="authors"><span class="author">Gray ER</span>, <span class="author">Illingworth CJ</span>, <span class="author">Coffin JM</span> and <span class="author">Stoye JP</span></p>
<p class="affiliation">Division of Virology, MRC National Institute for Medical Research, The Ridgeway, Mill Hill, London NW7 1AA, UK.</p><p class="abstract"><strong>Background</strong>: Understanding the mechanism by which viruses enter their target cell is an essential part of understanding their infectious cycle. Previous studies have focussed on the multiplicity of viral envelope proteins that need to bind to their cognate receptor to initiate entry. Avian sarcoma and leukosis virus Envelope protein (ASLV Env) mediates entry via a receptor, Tva, which can be attached to the cell surface either by a phospholipid anchor (Tva800) or a transmembrane domain (Tva950). In these studies, we have now investigated the number of target receptors necessary for entry of ASLV Env-pseudotyped virions.</p><p class="abstract"><strong>Results</strong>: Using titration and modelling experiments we provide evidence that binding of more than one receptor, probably two, is needed for entry of virions via Tva800. However, binding of just one Tva950 receptor is sufficient for successful entry.</p><p class="abstract"><strong>Conclusions</strong>: The different modes of attachment of Tva800 and Tva950 to the cell membrane have important implications for the utilisation of these proteins as receptors for viral binding and/or uptake.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">U117512710</span>; NCI NIH HHS: <span class="grant">R37 CA 089441</span>, <span class="grant">R37 CA089441-12</span>; Wellcome Trust: <span class="grant">091747</span></p>
<p class="publication"><span class="title">Retrovirology</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="pagerange">96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22099981" rel="external">22099981</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3267798" rel="external">3267798</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1742-4690-8-96" rel="external">10.1186/1742-4690-8-96</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targets downstream of Cdk8 in Dictyostelium development.</h4>
<p class="authors"><span class="author">Greene DM</span>, <span class="author">Bloomfield G</span>, <span class="author">Skelton J</span>, <span class="author">Ivens A</span> and <span class="author">Pears CJ</span></p>
<p class="affiliation">Biochemistry Department, Oxford University, South Parks Road, Oxford OX1 3QU UK. <a href="mailto:%63%61%74%68%65%72%69%6E%65.%70%65%61%72%73%40%62%69%6F%63%68.%6F%78.%61%63.%75%6B">&#99;&#97;&#116;&#104;&#101;&#114;&#105;&#110;&#101;&#46;&#112;&#101;&#97;&#114;&#115;&#64;&#98;&#105;&#111;&#99;&#104;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Cdk8 is a component of the mediator complex which facilitates transcription by RNA polymerase II and has been shown to play an important role in development of Dictyostelium discoideum. This eukaryote feeds as single cells but starvation triggers the formation of a multicellular organism in response to extracellular pulses of cAMP and the eventual generation of spores. Strains in which the gene encoding Cdk8 have been disrupted fail to form multicellular aggregates unless supplied with exogenous pulses of cAMP and later in development, cdk8- cells show a defect in spore production.</p><p class="abstract"><strong>Results</strong>: Microarray analysis revealed that the cdk8- strain previously described (cdk8-HL) contained genome duplications. Regeneration of the strain in a background lacking detectable gene duplication generated strains (cdk8-2) with identical defects in growth and early development, but a milder defect in spore generation, suggesting that the severity of this defect depends on the genetic background. The failure of cdk8- cells to aggregate unless rescued by exogenous pulses of cAMP is consistent with a failure to express the catalytic subunit of protein kinase A. However, overexpression of the gene encoding this protein was not sufficient to rescue the defect, suggesting that this is not the only important target for Cdk8 at this stage of development. Proteomic analysis revealed two potential targets for Cdk8 regulation, one regulated post-transcriptionally (4-hydroxyphenylpyruvate dioxygenase (HPD)) and one transcriptionally (short chain dehydrogenase/reductase (SDR1)).</p><p class="abstract"><strong>Conclusions</strong>: This analysis has confirmed the importance of Cdk8 at multiple stages of Dictyostelium development, although the severity of the defect in spore production depends on the genetic background. Potential targets of Cdk8-mediated gene regulation have been identified in Dictyostelium which will allow the mechanism of Cdk8 action and its role in development to be determined.</p>
<p class="publication"><span class="title">BMC developmental biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="pagerange">2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21255384" rel="external">21255384</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3037916" rel="external">3037916</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-213X-11-2" rel="external">10.1186/1471-213X-11-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">LIF-independent JAK signalling to chromatin in embryonic stem cells uncovered from an adult stem cell disease.</h4>
<p class="authors"><span class="author">Griffiths DS</span>, <span class="author">Li J</span>, <span class="author">Dawson MA</span>, <span class="author">Trotter MW</span>, <span class="author">Cheng YH</span>, <span class="author">Smith AM</span>, <span class="author">Mansfield W</span>, <span class="author">Liu P</span>, <span class="author">Kouzarides T</span>, <span class="author">Nichols J</span>, <span class="author">Bannister AJ</span>, <span class="author">Green AR</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Department of Haematology and Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK. <a href="mailto:%64%73%67%32%39%40%63%61%6D.%61%63.%75%6B">&#100;&#115;&#103;&#50;&#57;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Activating mutations in the tyrosine kinase Janus kinase 2 (JAK2) cause myeloproliferative neoplasms, clonal blood stem cell disorders with a propensity for leukaemic transformation. Leukaemia inhibitory factor (LIF) signalling through the JAK-signal transducer and activator of transcription (STAT) pathway enables self-renewal of embryonic stem (ES) cells. Here we show that mouse ES cells carrying the human JAK2V617F mutation were able to self-renew in chemically defined conditions without cytokines or small-molecule inhibitors, independently of JAK signalling through the STAT3 or phosphatidylinositol-3-OH kinase pathways. Phosphorylation of histone H3 tyrosine 41 (H3Y41) by JAK2 was recently shown to interfere with binding of heterochromatin protein 1&alpha; (HP1&alpha;). Levels of chromatin-bound HP1&alpha; were lower in JAK2V617F ES cells but increased following inhibition of JAK2, coincident with a global reduction in histone H3Y41 phosphorylation. JAK2 inhibition reduced levels of the pluripotency regulator Nanog, with a reduction in H3Y41 phosphorylation and concomitant increase in HP1&alpha; levels at the Nanog promoter. Furthermore, Nanog was required for factor independence of JAK2V617F ES cells. Taken together, these results uncover a previously unrecognized role for direct signalling to chromatin by JAK2 as an important mediator of ES cell self-renewal.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A8043</span>; Medical Research Council</p>
<p class="publication"><span class="title">Nature cell biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">13-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21151131" rel="external">21151131</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3008749" rel="external">3008749</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncb2135" rel="external">10.1038/ncb2135</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BioMart Central Portal: an open database network for the biological community.</h4>
<p class="authors"><span class="author">Guberman JM</span>, <span class="author">Ai J</span>, <span class="author">Arnaiz O</span>, <span class="author">Baran J</span>, <span class="author">Blake A</span>, <span class="author">Baldock R</span>, <span class="author">Chelala C</span>, <span class="author">Croft D</span>, <span class="author">Cros A</span>, <span class="author">Cutts RJ</span>, <span class="author">Di G&eacute;nova A</span>, <span class="author">Forbes S</span>, <span class="author">Fujisawa T</span>, <span class="author">Gadaleta E</span>, <span class="author">Goodstein DM</span>, <span class="author">Gundem G</span>, <span class="author">Haggarty B</span>, <span class="author">Haider S</span>, <span class="author">Hall M</span>, <span class="author">Harris T</span>, <span class="author">Haw R</span>, <span class="author">Hu S</span>, <span class="author">Hubbard S</span>, <span class="author">Hsu J</span>, <span class="author">Iyer V</span>, <span class="author">Jones P</span>, <span class="author">Katayama T</span>, <span class="author">Kinsella R</span>, <span class="author">Kong L</span>, <span class="author">Lawson D</span>, <span class="author">Liang Y</span>, <span class="author">Lopez-Bigas N</span>, <span class="author">Luo J</span>, <span class="author">Lush M</span>, <span class="author">Mason J</span>, <span class="author">Moreews F</span>, <span class="author">Ndegwa N</span>, <span class="author">Oakley D</span>, <span class="author">Perez-Llamas C</span>, <span class="author">Primig M</span>, <span class="author">Rivkin E</span>, <span class="author">Rosanoff S</span>, <span class="author">Shepherd R</span>, <span class="author">Simon R</span>, <span class="author">Skarnes B</span>, <span class="author">Smedley D</span>, <span class="author">Sperling L</span>, <span class="author">Spooner W</span>, <span class="author">Stevenson P</span>, <span class="author">Stone K</span>, <span class="author">Teague J</span>, <span class="author">Wang J</span>, <span class="author">Wang J</span>, <span class="author">Whitty B</span>, <span class="author">Wong DT</span>, <span class="author">Wong-Erasmus M</span>, <span class="author">Yao L</span>, <span class="author">Youens-Clark K</span>, <span class="author">Yung C</span>, <span class="author">Zhang J</span> and <span class="author">Kasprzyk A</span></p>
<p class="affiliation">Ontario Institute for Cancer Research, Toronto, M5G 0A3, Canada.</p><p class="abstract">BioMart Central Portal is a first of its kind, community-driven effort to provide unified access to dozens of biological databases spanning genomics, proteomics, model organisms, cancer data, ontology information and more. Anybody can contribute an independently maintained resource to the Central Portal, allowing it to be exposed to and shared with the research community, and linking it with the other resources in the portal. Users can take advantage of the common interface to quickly utilize different sources without learning a new system for each. The system also simplifies cross-database searches that might otherwise require several complicated steps. Several integrated tools streamline common tasks, such as converting between ID formats and retrieving sequences. The combination of a wide variety of databases, an easy-to-use interface, robust programmatic access and the array of tools make Central Portal a one-stop shop for biological data querying. Here, we describe the structure of Central Portal and show example queries to demonstrate its capabilities.</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2011</span>;<span class="volume">2011</span>;<span class="pagerange">bar041</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21930507" rel="external">21930507</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3263598" rel="external">3263598</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbar041" rel="external">10.1093/database/bar041</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using RNA-seq to determine the transcriptional landscape and the hypoxic response of the pathogenic yeast Candida parapsilosis.</h4>
<p class="authors"><span class="author">Guida A</span>, <span class="author">Lindst&auml;dt C</span>, <span class="author">Maguire SL</span>, <span class="author">Ding C</span>, <span class="author">Higgins DG</span>, <span class="author">Corton NJ</span>, <span class="author">Berriman M</span> and <span class="author">Butler G</span></p>
<p class="affiliation">School of Medicine and Medical Science, Conway Institute, UniversityCollege Dublin, Belfield, Dublin 4, Ireland.</p><p class="abstract"><strong>Background</strong>: Candida parapsilosis is one of the most common causes of Candida infection worldwide. However, the genome sequence annotation was made without experimental validation and little is known about the transcriptional landscape. The transcriptional response of C. parapsilosis to hypoxic (low oxygen) conditions, such as those encountered in the host, is also relatively unexplored.</p><p class="abstract"><strong>Results</strong>: We used next generation sequencing (RNA-seq) to determine the transcriptional profile of C. parapsilosis growing in several conditions including different media, temperatures and oxygen concentrations. We identified 395 novel protein-coding sequences that had not previously been annotated. We removed &gt; 300 unsupported gene models, and corrected approximately 900. We mapped the 5&#39; and 3&#39; UTR for thousands of genes. We also identified 422 introns, including two introns in the 3&#39; UTR of one gene. This is the first report of 3&#39; UTR introns in the Saccharomycotina. Comparing the introns in coding sequences with other species shows that small numbers have been gained and lost throughout evolution. Our analysis also identified a number of novel transcriptional active regions (nTARs). We used both RNA-seq and microarray analysis to determine the transcriptional profile of cells grown in normoxic and hypoxic conditions in rich media, and we showed that there was a high correlation between the approaches. We also generated a knockout of the UPC2 transcriptional regulator, and we found that similar to C. albicans, Upc2 is required for conferring resistance to azole drugs, and for regulation of expression of the ergosterol pathway in hypoxia.</p><p class="abstract"><strong>Conclusion</strong>: We provide the first detailed annotation of the C. parapsilosis genome, based on gene predictions and transcriptional analysis. We identified a number of novel ORFs and other transcribed regions, and detected transcripts from approximately 90% of the annotated protein coding genes. We found that the transcription factor Upc2 role has a conserved role as a major regulator of the hypoxic response in C. parapsilosis and C. albicans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="pagerange">628</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22192698" rel="external">22192698</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3287387" rel="external">3287387</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-12-628" rel="external">10.1186/1471-2164-12-628</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A PiggyBac-based recessive screening method to identify pluripotency regulators.</h4>
<p class="authors"><span class="author">Guo G</span>, <span class="author">Huang Y</span>, <span class="author">Humphreys P</span>, <span class="author">Wang X</span> and <span class="author">Smith A</span></p>
<p class="affiliation">Wellcome Trust Centre for Stem Cell Research, Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Phenotype driven genetic screens allow unbiased exploration of the genome to discover new biological regulators. Bloom syndrome gene (Blm) deficient embryonic stem (ES) cells provide an opportunity for recessive screening due to frequent loss of heterozygosity. We describe a strategy for isolating regulators of mammalian pluripotency based on conversion to homozygosity of PiggyBac gene trap insertions combined with stringent selection for differentiation resistance. From a screen of 2000 mutants we obtained a disruptive integration in the Tcf3 gene. Homozygous Tcf3 mutants showed impaired differentiation and enhanced self-renewal. This phenotype was reverted in a dosage sensitive manner by excision of one or both copies of the gene trap. These results provide new evidence confirming that Tcf3 is a potent negative regulator of pluripotency and validate a forward screening methodology to identify modulators of pluripotent stem cell biology.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">e18189</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21533166" rel="external">21533166</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3078921" rel="external">3078921</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0018189" rel="external">10.1371/journal.pone.0018189</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping a new spontaneous preterm birth susceptibility gene, IGF1R, using linkage, haplotype sharing, and association analysis.</h4>
<p class="authors"><span class="author">Haataja R</span>, <span class="author">Karjalainen MK</span>, <span class="author">Luukkonen A</span>, <span class="author">Teramo K</span>, <span class="author">Puttonen H</span>, <span class="author">Ojaniemi M</span>, <span class="author">Varilo T</span>, <span class="author">Chaudhari BP</span>, <span class="author">Plunkett J</span>, <span class="author">Murray JC</span>, <span class="author">McCarroll SA</span>, <span class="author">Peltonen L</span>, <span class="author">Muglia LJ</span>, <span class="author">Palotie A</span> and <span class="author">Hallman M</span></p>
<p class="affiliation">Department of Pediatrics, Institute of Clinical Medicine, University of Oulu, Oulu, Finland.</p><p class="abstract">Preterm birth is the major cause of neonatal death and serious morbidity. Most preterm births are due to spontaneous onset of labor without a known cause or effective prevention. Both maternal and fetal genomes influence the predisposition to spontaneous preterm birth (SPTB), but the susceptibility loci remain to be defined. We utilized a combination of unique population structures, family-based linkage analysis, and subsequent case-control association to identify a susceptibility haplotype for SPTB. Clinically well-characterized SPTB families from northern Finland, a subisolate founded by a relatively small founder population that has subsequently experienced a number of bottlenecks, were selected for the initial discovery sample. Genome-wide linkage analysis using a high-density single-nucleotide polymorphism (SNP) array in seven large northern Finnish non-consanginous families identified a locus on 15q26.3 (HLOD 4.68). This region contains the IGF1R gene, which encodes the type 1 insulin-like growth factor receptor IGF-1R. Haplotype segregation analysis revealed that a 55 kb 12-SNP core segment within the IGF1R gene was shared identical-by-state (IBS) in five families. A follow-up case-control study in an independent sample representing the more general Finnish population showed an association of a 6-SNP IGF1R haplotype with SPTB in the fetuses, providing further evidence for IGF1R as a SPTB predisposition gene (frequency in cases versus controls 0.11 versus 0.05, P = 0.001, odds ratio 2.3). This study demonstrates the identification of a predisposing, low-frequency haplotype in a multifactorial trait using a well-characterized population and a combination of family and case-control designs. Our findings support the identification of the novel susceptibility gene IGF1R for predisposition by the fetal genome to being born preterm.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">U54 RR020278</span>; NICHD NIH HHS: <span class="grant">R01 HD057192-03</span>; Wellcome Trust: <span class="grant">WTO89062</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e1001293</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21304894" rel="external">21304894</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3033387" rel="external">3033387</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001293" rel="external">10.1371/journal.pgen.1001293</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Influences of history, geography, and religion on genetic structure: the Maronites in Lebanon.</h4>
<p class="authors"><span class="author">Haber M</span>, <span class="author">Platt DE</span>, <span class="author">Badro DA</span>, <span class="author">Xue Y</span>, <span class="author">El-Sibai M</span>, <span class="author">Bonab MA</span>, <span class="author">Youhanna SC</span>, <span class="author">Saade S</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Royyuru A</span>, <span class="author">Wells RS</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Zalloua PA</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">The Lebanese American University, Chouran, Beirut, Lebanon.</p><p class="abstract">Cultural expansions, including of religions, frequently leave genetic traces of differentiation and in-migration. These expansions may be driven by complex doctrinal differentiation, together with major population migrations and gene flow. The aim of this study was to explore the genetic signature of the establishment of religious communities in a region where some of the most influential religions originated, using the Y chromosome as an informative male-lineage marker. A total of 3139 samples were analyzed, including 647 Lebanese and Iranian samples newly genotyped for 28 binary markers and 19 short tandem repeats on the non-recombinant segment of the Y chromosome. Genetic organization was identified by geography and religion across Lebanon in the context of surrounding populations important in the expansions of the major sects of Lebanon, including Italy, Turkey, the Balkans, Syria, and Iran by employing principal component analysis, multidimensional scaling, and AMOVA. Timing of population differentiations was estimated using BATWING, in comparison with dates of historical religious events to determine if these differentiations could be caused by religious conversion, or rather, whether religious conversion was facilitated within already differentiated populations. Our analysis shows that the great religions in Lebanon were adopted within already distinguishable communities. Once religious affiliations were established, subsequent genetic signatures of the older differentiations were reinforced. Post-establishment differentiations are most plausibly explained by migrations of peoples seeking refuge to avoid the turmoil of major historical events.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">3</span>;<span class="pagerange">334-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21119711">21119711</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3062011">3062011</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2010.177">10.1038/ejhg.2010.177</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y-chromosome R-M343 African lineages and sickle cell disease reveal structured assimilation in Lebanon.</h4>
<p class="authors"><span class="author">Haber M</span>, <span class="author">Platt DE</span>, <span class="author">Khoury S</span>, <span class="author">Badro DA</span>, <span class="author">Abboud M</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Zalloua PA</span></p>
<p class="affiliation">Medical School, The Lebanese American University, Beirut, Lebanon.</p><p class="abstract">We have sought to identify signals of assimilation of African male lines in Lebanon by exploring the association of sickle cell disease (SCD) in Lebanon with Y-chromosome haplogroups that are informative of the disease origin and its exclusivity to the Muslim community. A total of 732 samples were analyzed, including 33 SCD patients from Lebanon genotyped for 28 binary markers and 19 short tandem repeats on the non-recombinant segment of the Y chromosome. Genetic organization was identified using populations known to have influenced the genetic structure of the Lebanese population, in addition to African populations with high incidence of SCD. Y-chromosome haplogroup R-M343 sub-lineages distinguish between sub-Saharan African and Lebanese Y chromosomes. We detected a limited penetration of SCD into Lebanese R-M343 carriers, restricted to Lebanese Muslims. We suggest that this penetration brought the sickle cell gene along with the African R-M343, probably with the Saharan caravan slave trade.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">56</span>;<span class="number">1</span>;<span class="pagerange">29-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20981037">20981037</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3312574">3312574</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fjhg.2010.131">10.1038/jhg.2010.131</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">At-risk variant in TCF7L2 for type II diabetes increases risk of schizophrenia.</h4>
<p class="authors"><span class="author">Hansen T</span>, <span class="author">Ingason A</span>, <span class="author">Djurovic S</span>, <span class="author">Melle I</span>, <span class="author">Fenger M</span>, <span class="author">Gustafsson O</span>, <span class="author">Jakobsen KD</span>, <span class="author">Rasmussen HB</span>, <span class="author">Tosato S</span>, <span class="author">Rietschel M</span>, <span class="author">Frank J</span>, <span class="author">Owen M</span>, <span class="author">Bonetto C</span>, <span class="author">Suvisaari J</span>, <span class="author">Thygesen JH</span>, <span class="author">P&eacute;tursson H</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Sigurdsson E</span>, <span class="author">Giegling I</span>, <span class="author">Craddock N</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Ruggeri M</span>, <span class="author">Cichon S</span>, <span class="author">Ophoff RA</span>, <span class="author">Pietil&auml;inen O</span>, <span class="author">Peltonen L</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Rujescu D</span>, <span class="author">St Clair D</span>, <span class="author">Collier DA</span>, <span class="author">Andreassen OA</span> and <span class="author">Werge T</span></p>
<p class="affiliation">Mental Health Centre Sct. Hans, Copenhagen University Hospital, Research Institute of Biological Psychiatry, Roskilde, Denmark; Copenhagen University, Center for Pharmacogenomics, Copenhagen, Denmark.</p><p class="abstract"><strong>Background</strong>: Schizophrenia is associated with increased risk of type II diabetes and metabolic disorders. However, it is unclear whether this comorbidity reflects shared genetic risk factors, at-risk lifestyle, or side effects of antipsychotic medication.</p><p class="abstract"><strong>Methods</strong>: Eleven known risk variants of type II diabetes were genotyped in patients with schizophrenia in a sample of 410 Danish patients, each matched with two healthy control subjects on sex, birth year, and month. Replication was carried out in a large multinational European sample of 4089 patients with schizophrenia and 17,597 controls (SGENE+) using Mantel-Haenszel test.</p><p class="abstract"><strong>Results</strong>: One type II diabetes at-risk allele located in TCF7L2, rs7903146 [T], was associated with schizophrenia in the discovery sample (p = .0052) and in the replication with an odds ratio of 1.07 (95% confidence interval 1.01-1.14, p = .033).</p><p class="abstract"><strong>Conclusion</strong>: The association reported here with a well-known diabetes variant suggests that the observed comorbidity is partially caused by genetic risk variants. This study also demonstrates how genetic studies can successfully examine an epidemiologically derived hypothesis of comorbidity.</p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">70</span>;<span class="number">1</span>;<span class="pagerange">59-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21414605">21414605</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2011.01.031">10.1016/j.biopsych.2011.01.031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A worldwide analysis of beta-defensin copy number variation suggests recent selection of a high-expressing DEFB103 gene copy in East Asia.</h4>
<p class="authors"><span class="author">Hardwick RJ</span>, <span class="author">Machado LR</span>, <span class="author">Zuccherato LW</span>, <span class="author">Antolinos S</span>, <span class="author">Xue Y</span>, <span class="author">Shawa N</span>, <span class="author">Gilman RH</span>, <span class="author">Cabrera L</span>, <span class="author">Berg DE</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Kelly P</span>, <span class="author">Tarazona-Santos E</span> and <span class="author">Hollox EJ</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, University Road, Leicester, United Kingdom.</p><p class="abstract">Beta-defensins are a family of multifunctional genes with roles in defense against pathogens, reproduction, and pigmentation. In humans, six beta-defensin genes are clustered in a repeated region which is copy-number variable (CNV) as a block, with a diploid copy number between 1 and 12. The role in host defense makes the evolutionary history of this CNV particularly interesting, because morbidity due to infectious disease is likely to have been an important selective force in human evolution, and to have varied between geographical locations. Here, we show CNV of the beta-defensin region in chimpanzees, and identify a beta-defensin block in the human lineage that contains rapidly evolving noncoding regulatory sequences. We also show that variation at one of these rapidly evolving sequences affects expression levels and cytokine responsiveness of DEFB103, a key inhibitor of influenza virus fusion at the cell surface. A worldwide analysis of beta-defensin CNV in 67 populations shows an unusually high frequency of high-DEFB103-expressing copies in East Asia, the geographical origin of historical and modern influenza epidemics, possibly as a result of selection for increased resistance to influenza in this region.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801123</span>, <span class="grant">GO801123</span>; Wellcome Trust: <span class="grant">067948</span>, <span class="grant">077009</span>, <span class="grant">087663</span></p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2011</span>;<span class="volume">32</span>;<span class="number">7</span>;<span class="pagerange">743-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21387465">21387465</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3263423">3263423</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.21491">10.1002/humu.21491</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The transcriptional repressor Blimp1/Prdm1 regulates postnatal reprogramming of intestinal enterocytes.</h4>
<p class="authors"><span class="author">Harper J</span>, <span class="author">Mould A</span>, <span class="author">Andrews RM</span>, <span class="author">Bikoff EK</span> and <span class="author">Robertson EJ</span></p>
<p class="affiliation">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, United Kingdom.</p><p class="abstract">Female mammals produce milk to feed their newborn offspring before teeth develop and permit the consumption of solid food. Intestinal enterocytes dramatically alter their biochemical signature during the suckling-to-weaning transition. The transcriptional repressor Blimp1 is strongly expressed in immature enterocytes in utero, but these are gradually replaced by Blimp1(-) crypt-derived adult enterocytes. Here we used a conditional inactivation strategy to eliminate Blimp1 function in the developing intestinal epithelium. There was no noticeable effect on gross morphology or formation of mature cell types before birth. However, survival of mutant neonates was severely compromised. Transcriptional profiling experiments reveal global changes in gene expression patterns. Key components of the adult enterocyte biochemical signature were substantially and prematurely activated. In contrast, those required for processing maternal milk were markedly reduced. Thus, we conclude Blimp1 governs the developmental switch responsible for postnatal intestinal maturation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">26</span>;<span class="pagerange">10585-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21670299" rel="external">21670299</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3127883" rel="external">3127883</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1105852108" rel="external">10.1073/pnas.1105852108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic Analysis of Hepatitis B Virus Reveals Antigen State and Genotype as Sources of Evolutionary Rate Variation</h4>
<p class="authors"><span class="author">Harrison A</span>, <span class="author">Lemey P</span>, <span class="author">HURLES ME</span>, <span class="author">Moyes C</span>, <span class="author">Horn S</span>, <span class="author">Pryor J</span>, <span class="author">Malani J</span>, <span class="author">Supuri M</span>, <span class="author">Masta A</span>, <span class="author">Teriboriki B</span>, <span class="author">Toatu T</span>, <span class="author">Penny D</span>, <span class="author">Rambaut A</span>, <span class="author">Shapiro B</span></p>
<p class="publication"><span class="title">Viruses-Basel.</span> <span class="year">2011</span>;<span class="volume">3</span>;<span class="pagerange">83-101</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EpiChIP: gene-by-gene quantification of epigenetic modification levels.</h4>
<p class="authors"><span class="author">Hebenstreit D</span>, <span class="author">Gu M</span>, <span class="author">Haider S</span>, <span class="author">Turner DJ</span>, <span class="author">Li&ograve; P</span> and <span class="author">Teichmann SA</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Rd, CB2 0QH Cambridge, UK. <a href="mailto:%64%61%6E%69%65%6C%68%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#100;&#97;&#110;&#105;&#101;&#108;&#104;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The combination of chromatin immunoprecipitation with next-generation sequencing technology (ChIP-seq) is a powerful and increasingly popular method for mapping protein-DNA interactions in a genome-wide fashion. The conventional way of analyzing this data is to identify sequencing peaks along the chromosomes that are significantly higher than the read background. For histone modifications and other epigenetic marks, it is often preferable to find a characteristic region of enrichment in sequencing reads relative to gene annotations. For instance, many histone modifications are typically enriched around transcription start sites. Calculating the optimal window that describes this enrichment allows one to quantify modification levels for each individual gene. Using data sets for the H3K9/14ac histone modification in Th cells and an accompanying IgG control, we present an analysis strategy that alternates between single gene and global data distribution levels and allows a clear distinction between experimental background and signal. Curve fitting permits false discovery rate-based classification of genes as modified versus unmodified. We have developed a software package called EpiChIP that carries out this type of analysis, including integration with and visualization of gene expression data.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105161047</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">5</span>;<span class="pagerange">e27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21131282">21131282</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3061070">3061070</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq1226">10.1093/nar/gkq1226</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of Staphylococcus lugdunensis N920143 allows identification of putative colonization and virulence factors.</h4>
<p class="authors"><span class="author">Heilbronner S</span>, <span class="author">Holden MT</span>, <span class="author">van Tonder A</span>, <span class="author">Geoghegan JA</span>, <span class="author">Foster TJ</span>, <span class="author">Parkhill J</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Microbiology Department, Trinity College, Dublin, Ireland.</p><p class="abstract">Staphylococcus lugdunensis is an opportunistic pathogen related to Staphylococcus aureus and Staphylococcus epidermidis. The genome sequence of S. lugdunensis strain N920143 has been compared with other staphylococci, and genes were identified that could promote survival of S. lugdunensis on human skin and pathogenesis of infections. Staphylococcus lugdunensis lacks virulence factors that characterize S. aureus and harbours a smaller number of genes encoding surface proteins. It is the only staphylococcal species other than S. aureus that possesses a locus encoding iron-regulated surface determinant (Isd) proteins involved in iron acquisition from haemoglobin.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">FEMS microbiology letters</span>&nbsp;<span class="year">2011</span>;<span class="volume">322</span>;<span class="number">1</span>;<span class="pagerange">60-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21682763">21682763</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1574-6968.2011.02339.x">10.1111/j.1574-6968.2011.02339.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing identifies a missense mutation in Isl1 associated with low penetrance otitis media in dearisch mice.</h4>
<p class="authors"><span class="author">Hilton JM</span>, <span class="author">Lewis MA</span>, <span class="author">Grati M</span>, <span class="author">Ingham N</span>, <span class="author">Pearson S</span>, <span class="author">Laskowski RA</span>, <span class="author">Adams DJ</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Background</strong>: Inflammation of the middle ear (otitis media) is very common and can lead to serious complications if not resolved. Genetic studies suggest an inherited component, but few of the genes that contribute to this condition are known. Mouse mutants have contributed significantly to the identification of genes predisposing to otitis media</p><p class="abstract"><strong>Results</strong>: The dearisch mouse mutant is an ENU-induced mutant detected by its impaired Preyer reflex (ear flick in response to sound). Auditory brainstem responses revealed raised thresholds from as early as three weeks old. Pedigree analysis suggested a dominant but partially penetrant mode of inheritance. The middle ear of dearisch mutants shows a thickened mucosa and cellular effusion suggesting chronic otitis media with effusion with superimposed acute infection. The inner ear, including the sensory hair cells, appears normal. Due to the low penetrance of the phenotype, normal backcross mapping of the mutation was not possible. Exome sequencing was therefore employed to identify a non-conservative tyrosine to cysteine (Y71C) missense mutation in the Islet1 gene, Isl1(Drsh). Isl1 is expressed in the normal middle ear mucosa. The findings suggest the Isl1(Drsh) mutation is likely to predispose carriers to otitis media.</p><p class="abstract"><strong>Conclusions</strong>: Dearisch, Isl1(Drsh), represents the first point mutation in the mouse Isl1 gene and suggests a previously unrecognized role for this gene. It is also the first recorded exome sequencing of the C3HeB/FeJ background relevant to many ENU-induced mutants. Most importantly, the power of exome resequencing to identify ENU-induced mutations without a mapped gene locus is illustrated.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">G0800024</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust: <span class="grant">077189</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">9</span>;<span class="pagerange">R90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21936904">21936904</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308053">3308053</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-9-r90">10.1186/gb-2011-12-9-r90</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at ABCA7, MS4A6A/MS4A4E, EPHA1, CD33 and CD2AP are associated with Alzheimer&#39;s disease.</h4>
<p class="authors"><span class="author">Hollingworth P</span>, <span class="author">Harold D</span>, <span class="author">Sims R</span>, <span class="author">Gerrish A</span>, <span class="author">Lambert JC</span>, <span class="author">Carrasquillo MM</span>, <span class="author">Abraham R</span>, <span class="author">Hamshere ML</span>, <span class="author">Pahwa JS</span>, <span class="author">Moskvina V</span>, <span class="author">Dowzell K</span>, <span class="author">Jones N</span>, <span class="author">Stretton A</span>, <span class="author">Thomas C</span>, <span class="author">Richards A</span>, <span class="author">Ivanov D</span>, <span class="author">Widdowson C</span>, <span class="author">Chapman J</span>, <span class="author">Lovestone S</span>, <span class="author">Powell J</span>, <span class="author">Proitsi P</span>, <span class="author">Lupton MK</span>, <span class="author">Brayne C</span>, <span class="author">Rubinsztein DC</span>, <span class="author">Gill M</span>, <span class="author">Lawlor B</span>, <span class="author">Lynch A</span>, <span class="author">Brown KS</span>, <span class="author">Passmore PA</span>, <span class="author">Craig D</span>, <span class="author">McGuinness B</span>, <span class="author">Todd S</span>, <span class="author">Holmes C</span>, <span class="author">Mann D</span>, <span class="author">Smith AD</span>, <span class="author">Beaumont H</span>, <span class="author">Warden D</span>, <span class="author">Wilcock G</span>, <span class="author">Love S</span>, <span class="author">Kehoe PG</span>, <span class="author">Hooper NM</span>, <span class="author">Vardy ER</span>, <span class="author">Hardy J</span>, <span class="author">Mead S</span>, <span class="author">Fox NC</span>, <span class="author">Rossor M</span>, <span class="author">Collinge J</span>, <span class="author">Maier W</span>, <span class="author">Jessen F</span>, <span class="author">R&uuml;ther E</span>, <span class="author">Sch&uuml;rmann B</span>, <span class="author">Heun R</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">van den Bussche H</span>, <span class="author">Heuser I</span>, <span class="author">Kornhuber J</span>, <span class="author">Wiltfang J</span>, <span class="author">Dichgans M</span>, <span class="author">Fr&ouml;lich L</span>, <span class="author">Hampel H</span>, <span class="author">Gallacher J</span>, <span class="author">H&uuml;ll M</span>, <span class="author">Rujescu D</span>, <span class="author">Giegling I</span>, <span class="author">Goate AM</span>, <span class="author">Kauwe JS</span>, <span class="author">Cruchaga C</span>, <span class="author">Nowotny P</span>, <span class="author">Morris JC</span>, <span class="author">Mayo K</span>, <span class="author">Sleegers K</span>, <span class="author">Bettens K</span>, <span class="author">Engelborghs S</span>, <span class="author">De Deyn PP</span>, <span class="author">Van Broeckhoven C</span>, <span class="author">Livingston G</span>, <span class="author">Bass NJ</span>, <span class="author">Gurling H</span>, <span class="author">McQuillin A</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">Al-Chalabi A</span>, <span class="author">Shaw CE</span>, <span class="author">Tsolaki M</span>, <span class="author">Singleton AB</span>, <span class="author">Guerreiro R</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Moebus S</span>, <span class="author">J&ouml;ckel KH</span>, <span class="author">Klopp N</span>, <span class="author">Wichmann HE</span>, <span class="author">Pankratz VS</span>, <span class="author">Sando SB</span>, <span class="author">Aasly JO</span>, <span class="author">Barcikowska M</span>, <span class="author">Wszolek ZK</span>, <span class="author">Dickson DW</span>, <span class="author">Graff-Radford NR</span>, <span class="author">Petersen RC</span>, <span class="author">Alzheimer&#39;s Disease Neuroimaging Initiative</span>, <span class="author">van Duijn CM</span>, <span class="author">Breteler MM</span>, <span class="author">Ikram MA</span>, <span class="author">DeStefano AL</span>, <span class="author">Fitzpatrick AL</span>, <span class="author">Lopez O</span>, <span class="author">Launer LJ</span>, <span class="author">Seshadri S</span>, <span class="author">CHARGE consortium</span>, <span class="author">Berr C</span>, <span class="author">Campion D</span>, <span class="author">Epelbaum J</span>, <span class="author">Dartigues JF</span>, <span class="author">Tzourio C</span>, <span class="author">Alp&eacute;rovitch A</span>, <span class="author">Lathrop M</span>, <span class="author">EADI1 consortium</span>, <span class="author">Feulner TM</span>, <span class="author">Friedrich P</span>, <span class="author">Riehle C</span>, <span class="author">Krawczak M</span>, <span class="author">Schreiber S</span>, <span class="author">Mayhaus M</span>, <span class="author">Nicolhaus S</span>, <span class="author">Wagenpfeil S</span>, <span class="author">Steinberg S</span>, <span class="author">Stefansson H</span>, <span class="author">Stefansson K</span>, <span class="author">Snaedal J</span>, <span class="author">Bj&ouml;rnsson S</span>, <span class="author">Jonsson PV</span>, <span class="author">Chouraki V</span>, <span class="author">Genier-Boley B</span>, <span class="author">Hiltunen M</span>, <span class="author">Soininen H</span>, <span class="author">Combarros O</span>, <span class="author">Zelenika D</span>, <span class="author">Delepine M</span>, <span class="author">Bullido MJ</span>, <span class="author">Pasquier F</span>, <span class="author">Mateo I</span>, <span class="author">Frank-Garcia A</span>, <span class="author">Porcellini E</span>, <span class="author">Hanon O</span>, <span class="author">Coto E</span>, <span class="author">Alvarez V</span>, <span class="author">Bosco P</span>, <span class="author">Siciliano G</span>, <span class="author">Mancuso M</span>, <span class="author">Panza F</span>, <span class="author">Solfrizzi V</span>, <span class="author">Nacmias B</span>, <span class="author">Sorbi S</span>, <span class="author">Boss&ugrave; P</span>, <span class="author">Piccardi P</span>, <span class="author">Arosio B</span>, <span class="author">Annoni G</span>, <span class="author">Seripa D</span>, <span class="author">Pilotto A</span>, <span class="author">Scarpini E</span>, <span class="author">Galimberti D</span>, <span class="author">Brice A</span>, <span class="author">Hannequin D</span>, <span class="author">Licastro F</span>, <span class="author">Jones L</span>, <span class="author">Holmans PA</span>, <span class="author">Jonsson T</span>, <span class="author">Riemenschneider M</span>, <span class="author">Morgan K</span>, <span class="author">Younkin SG</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan M</span>, <span class="author">Amouyel P</span> and <span class="author">Williams J</span></p>
<p class="affiliation">Medical Research Council Centre for Neuropsychiatric Genetics and Genomics, Neurosciences and Mental Health Research Institute, Department of Psychological Medicine and Neurology, School of Medicine, Cardiff University, Cardiff, UK.</p><p class="abstract">We sought to identify new susceptibility loci for Alzheimer&#39;s disease through a staged association study (GERAD+) and by testing suggestive loci reported by the Alzheimer&#39;s Disease Genetic Consortium (ADGC) in a companion paper. We undertook a combined analysis of four genome-wide association datasets (stage 1) and identified ten newly associated variants with P &le; 1 &times; 10(-5). We tested these variants for association in an independent sample (stage 2). Three SNPs at two loci replicated and showed evidence for association in a further sample (stage 3). Meta-analyses of all data provided compelling evidence that ABCA7 (rs3764650, meta P = 4.5 &times; 10(-17); including ADGC data, meta P = 5.0 &times; 10(-21)) and the MS4A gene cluster (rs610932, meta P = 1.8 &times; 10(-14); including ADGC data, meta P = 1.2 &times; 10(-16)) are new Alzheimer&#39;s disease susceptibility loci. We also found independent evidence for association for three loci reported by the ADGC, which, when combined, showed genome-wide significance: CD2AP (GERAD+, P = 8.0 &times; 10(-4); including ADGC data, meta P = 8.6 &times; 10(-9)), CD33 (GERAD+, P = 2.2 &times; 10(-4); including ADGC data, meta P = 1.6 &times; 10(-9)) and EPHA1 (GERAD+, P = 3.4 &times; 10(-4); including ADGC data, meta P = 6.0 &times; 10(-10)).</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300429(66813)</span>, <span class="grant">G9810900(63319)</span>; Wellcome Trust: <span class="grant">082604</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">5</span>;<span class="pagerange">429-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21460840" rel="external">21460840</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3084173" rel="external">3084173</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.803" rel="external">10.1038/ng.803</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effect modification by population dietary folate on the association between MTHFR genotype, homocysteine, and stroke risk: a meta-analysis of genetic studies and randomised trials.</h4>
<p class="authors"><span class="author">Holmes MV</span>, <span class="author">Newcombe P</span>, <span class="author">Hubacek JA</span>, <span class="author">Sofat R</span>, <span class="author">Ricketts SL</span>, <span class="author">Cooper J</span>, <span class="author">Breteler MM</span>, <span class="author">Bautista LE</span>, <span class="author">Sharma P</span>, <span class="author">Whittaker JC</span>, <span class="author">Smeeth L</span>, <span class="author">Fowkes FG</span>, <span class="author">Algra A</span>, <span class="author">Shmeleva V</span>, <span class="author">Szolnoki Z</span>, <span class="author">Roest M</span>, <span class="author">Linnebank M</span>, <span class="author">Zacho J</span>, <span class="author">Nalls MA</span>, <span class="author">Singleton AB</span>, <span class="author">Ferrucci L</span>, <span class="author">Hardy J</span>, <span class="author">Worrall BB</span>, <span class="author">Rich SS</span>, <span class="author">Matarin M</span>, <span class="author">Norman PE</span>, <span class="author">Flicker L</span>, <span class="author">Almeida OP</span>, <span class="author">van Bockxmeer FM</span>, <span class="author">Shimokata H</span>, <span class="author">Khaw KT</span>, <span class="author">Wareham NJ</span>, <span class="author">Bobak M</span>, <span class="author">Sterne JA</span>, <span class="author">Smith GD</span>, <span class="author">Talmud PJ</span>, <span class="author">van Duijn C</span>, <span class="author">Humphries SE</span>, <span class="author">Price JF</span>, <span class="author">Ebrahim S</span>, <span class="author">Lawlor DA</span>, <span class="author">Hankey GJ</span>, <span class="author">Meschia JF</span>, <span class="author">Sandhu MS</span>, <span class="author">Hingorani AD</span> and <span class="author">Casas JP</span></p>
<p class="affiliation">Research Department of Epidemiology and Public Health, University College London, London, UK.</p><p class="abstract"><strong>Background</strong>: The MTHFR 677C&rarr;T polymorphism has been associated with raised homocysteine concentration and increased risk of stroke. A previous overview showed that the effects were greatest in regions with low dietary folate consumption, but differentiation between the effect of folate and small-study bias was difficult. A meta-analysis of randomised trials of homocysteine-lowering interventions showed no reduction in coronary heart disease events or stroke, but the trials were generally set in populations with high folate consumption. We aimed to reduce the effect of small-study bias and investigate whether folate status modifies the association between MTHFR 677C&rarr;T and stroke in a genetic analysis and meta-analysis of randomised controlled trials.</p><p class="abstract"><strong>Methods</strong>: We established a collaboration of genetic studies consisting of 237 datasets including 59,995 individuals with data for homocysteine and 20,885 stroke events. We compared the genetic findings with a meta-analysis of 13 randomised trials of homocysteine-lowering treatments and stroke risk (45,549 individuals, 2314 stroke events, 269 transient ischaemic attacks).</p><p class="abstract"><strong>Findings</strong>: The effect of the MTHFR 677C&rarr;T variant on homocysteine concentration was larger in low folate regions (Asia; difference between individuals with TT versus CC genotype, 3&middot;12 &mu;mol/L, 95% CI 2&middot;23 to 4&middot;01) than in areas with folate fortification (America, Australia, and New Zealand, high; 0&middot;13 &mu;mol/L, -0&middot;85 to 1&middot;11). The odds ratio (OR) for stroke was also higher in Asia (1&middot;68, 95% CI 1&middot;44 to 1&middot;97) than in America, Australia, and New Zealand, high (1&middot;03, 0&middot;84 to 1&middot;25). Most randomised trials took place in regions with high or increasing population folate concentrations. The summary relative risk (RR) of stroke in trials of homocysteine-lowering interventions (0&middot;94, 95% CI 0&middot;85 to 1&middot;04) was similar to that predicted for the same extent of homocysteine reduction in large genetic studies in populations with similar folate status (predicted RR 1&middot;00, 95% CI 0&middot;90 to 1&middot;11). Although the predicted effect of homocysteine reduction from large genetic studies in low folate regions (Asia) was larger (RR 0&middot;78, 95% CI 0&middot;68 to 0&middot;90), no trial has evaluated the effect of lowering of homocysteine on stroke risk exclusively in a low folate region.</p><p class="abstract"><strong>Interpretation</strong>: In regions with increasing levels or established policies of population folate supplementation, evidence from genetic studies and randomised trials is concordant in suggesting an absence of benefit from lowering of homocysteine for prevention of stroke. Further large-scale genetic studies of the association between MTHFR 677C&rarr;T and stroke in low folate settings are needed to distinguish effect modification by folate from small-study bias. If future randomised trials of homocysteine-lowering interventions for stroke prevention are undertaken, they should take place in regions with low folate consumption.</p><p class="abstract"><strong>Funding</strong>: Full funding sources listed at end of paper (see Acknowledgments).</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/07/01</span>, <span class="grant">FS05/125</span>; Department of Health; Medical Research Council: <span class="grant">G0600580</span>, <span class="grant">G0600705</span>, <span class="grant">G0802432</span>; NINDS NIH HHS: <span class="grant">R01 NS39987</span>, <span class="grant">R01 NS42733</span>; Wellcome Trust: <span class="grant">082178</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2011</span>;<span class="volume">378</span>;<span class="number">9791</span>;<span class="pagerange">584-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21803414" rel="external">21803414</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3156981" rel="external">3156981</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(11)60872-6" rel="external">10.1016/S0140-6736(11)60872-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A very early-branching Staphylococcus aureus lineage lacking the carotenoid pigment staphyloxanthin.</h4>
<p class="authors"><span class="author">Holt DC</span>, <span class="author">Holden MT</span>, <span class="author">Tong SY</span>, <span class="author">Castillo-Ramirez S</span>, <span class="author">Clarke L</span>, <span class="author">Quail MA</span>, <span class="author">Currie BJ</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span>, <span class="author">Feil EJ</span> and <span class="author">Giffard PM</span></p>
<p class="affiliation">Menzies School of Health Research, Charles Darwin University, Darwin, Northern Territory, Australia.</p><p class="abstract">Here we discuss the evolution of the northern Australian Staphylococcus aureus isolate MSHR1132 genome. MSHR1132 belongs to the divergent clonal complex 75 lineage. The average nucleotide divergence between orthologous genes in MSHR1132 and typical S. aureus is approximately sevenfold greater than the maximum divergence observed in this species to date. MSHR1132 has a small accessory genome, which includes the well-characterized genomic islands, &nu;SA&alpha; and &nu;Sa&beta;, suggesting that these elements were acquired well before the expansion of the typical S. aureus population. Other mobile elements show mosaic structure (the prophage &#x3D5;Sa3) or evidence of recent acquisition from a typical S. aureus lineage (SCCmec, ICE6013 and plasmid pMSHR1132). There are two differences in gene repertoire compared with typical S. aureus that may be significant clues as to the genetic basis underlying the successful emergence of S. aureus as a pathogen. First, MSHR1132 lacks the genes for production of staphyloxanthin, the carotenoid pigment that confers upon S. aureus its characteristic golden color and protects against oxidative stress. The lack of pigment was demonstrated in 126 of 126 CC75 isolates. Second, a mobile clustered regularly interspaced short palindromic repeat (CRISPR) element is inserted into orfX of MSHR1132. Although common in other staphylococcal species, these elements are very rare within S. aureus and may impact accessory genome acquisition. The CRISPR spacer sequences reveal a history of attempted invasion by known S. aureus mobile elements. There is a case for the creation of a new taxon to accommodate this and related isolates.</p>
<p class="publication"><span class="title">Genome biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">3</span>;<span class="pagerange">881-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21813488">21813488</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3175761">3175761</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fgbe%2Fevr078">10.1093/gbe/evr078</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Temporal fluctuation of multidrug resistant salmonella typhi haplotypes in the mekong river delta region of Vietnam.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Dolecek C</span>, <span class="author">Chau TT</span>, <span class="author">Duy PT</span>, <span class="author">La TT</span>, <span class="author">Hoang NV</span>, <span class="author">Nga TV</span>, <span class="author">Campbell JI</span>, <span class="author">Manh BH</span>, <span class="author">Vinh Chau NV</span>, <span class="author">Hien TT</span>, <span class="author">Farrar J</span>, <span class="author">Dougan G</span> and <span class="author">Baker S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom. <a href="mailto:%6B%68%6F%6C%74%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#107;&#104;&#111;&#108;&#116;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract"><strong>Background</strong>: typhoid fever remains a public health problem in Vietnam, with a significant burden in the Mekong River delta region. Typhoid fever is caused by the bacterial pathogen Salmonella enterica serovar Typhi (S. Typhi), which is frequently multidrug resistant with reduced susceptibility to fluoroquinolone-based drugs, the first choice for the treatment of typhoid fever. We used a GoldenGate (Illumina) assay to type 1,500 single nucleotide polymorphisms (SNPs) and analyse the genetic variation of S. Typhi isolated from 267 typhoid fever patients in the Mekong delta region participating in a randomized trial conducted between 2004 and 2005.</p><p class="abstract">the population of S. Typhi circulating during the study was highly clonal, with 91% of isolates belonging to a single clonal complex of the S. Typhi H58 haplogroup. The patterns of disease were consistent with the presence of an endemic haplotype H58-C and a localised outbreak of S. Typhi haplotype H58-E2 in 2004. H58-E2-associated typhoid fever cases exhibited evidence of significant geo-spatial clustering along the S&ocirc;ng H u branch of the Mekong River. Multidrug resistance was common in the established clone H58-C but not in the outbreak clone H58-E2, however all H58 S. Typhi were nalidixic acid resistant and carried a Ser83Phe amino acid substitution in the gyrA gene.</p><p class="abstract"><strong>Significance</strong>: the H58 haplogroup dominates S. Typhi populations in other endemic areas, but the population described here was more homogeneous than previously examined populations, and the dominant clonal complex (H58-C, -E1, -E2) observed in this study has not been detected outside Vietnam. IncHI1 plasmid-bearing S. Typhi H58-C was endemic during the study period whilst H58-E2, which rarely carried the plasmid, was only transient, suggesting a selective advantage for the plasmid. These data add insight into the outbreak dynamics and local molecular epidemiology of S. Typhi in southern Vietnam.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">e929</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21245916" rel="external">21245916</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3014949" rel="external">3014949</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000929" rel="external">10.1371/journal.pntd.0000929</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Emergence of a globally dominant IncHI1 plasmid type associated with multiple drug resistant typhoid.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Phan MD</span>, <span class="author">Baker S</span>, <span class="author">Duy PT</span>, <span class="author">Nga TV</span>, <span class="author">Nair S</span>, <span class="author">Turner AK</span>, <span class="author">Walsh C</span>, <span class="author">Fanning S</span>, <span class="author">Farrell-Ward S</span>, <span class="author">Dutta S</span>, <span class="author">Kariuki S</span>, <span class="author">Weill FX</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">Wain J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%6B%68%6F%6C%74%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#107;&#104;&#111;&#108;&#116;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Typhoid fever, caused by Salmonella enterica serovar Typhi (S. Typhi), remains a serious global health concern. Since their emergence in the mid-1970s multi-drug resistant (MDR) S. Typhi now dominate drug sensitive equivalents in many regions. MDR in S. Typhi is almost exclusively conferred by self-transmissible IncHI1 plasmids carrying a suite of antimicrobial resistance genes. We identified over 300 single nucleotide polymorphisms (SNPs) within conserved regions of the IncHI1 plasmid, and genotyped both plasmid and chromosomal SNPs in over 450 S. Typhi dating back to 1958. Prior to 1995, a variety of IncHI1 plasmid types were detected in distinct S. Typhi haplotypes. Highly similar plasmids were detected in co-circulating S. Typhi haplotypes, indicative of plasmid transfer. In contrast, from 1995 onwards, 98% of MDR S. Typhi were plasmid sequence type 6 (PST6) and S. Typhi haplotype H58, indicating recent global spread of a dominant MDR clone. To investigate whether PST6 conferred a selective advantage compared to other IncHI1 plasmids, we used a phenotyping array to compare the impact of IncHI1 PST6 and PST1 plasmids in a common S. Typhi host. The PST6 plasmid conferred the ability to grow in high salt medium (4.7% NaCl), which we demonstrate is due to the presence in PST6 of the Tn6062 transposon encoding BetU.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">e1245</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21811646" rel="external">21811646</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3139670" rel="external">3139670</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001245" rel="external">10.1371/journal.pntd.0001245</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A homozygous mutant embryonic stem cell bank applicable for phenotype-driven genetic screening.</h4>
<p class="authors"><span class="author">Horie K</span>, <span class="author">Kokubu C</span>, <span class="author">Yoshida J</span>, <span class="author">Akagi K</span>, <span class="author">Isotani A</span>, <span class="author">Oshitani A</span>, <span class="author">Yusa K</span>, <span class="author">Ikeda R</span>, <span class="author">Huang Y</span>, <span class="author">Bradley A</span> and <span class="author">Takeda J</span></p>
<p class="affiliation">Department of Social and Environmental Medicine, Graduate School of Medicine, Osaka University, Suita, Osaka, Japan. <a href="mailto:%68%6F%72%69%65%40%6D%72-%65%6E%76%69.%6D%65%64.%6F%73%61%6B%61-%75.%61%63.%6A%70">&#104;&#111;&#114;&#105;&#101;&#64;&#109;&#114;&#45;&#101;&#110;&#118;&#105;&#46;&#109;&#101;&#100;&#46;&#111;&#115;&#97;&#107;&#97;&#45;&#117;&#46;&#97;&#99;&#46;&#106;&#112;</a></p><p class="abstract">Genome-wide mutagenesis in mouse embryonic stem cells (ESCs) is a powerful tool, but the diploid nature of the mammalian genome hampers its application for recessive genetic screening. We have previously reported a method to induce homozygous mutant ESCs from heterozygous mutants by tetracycline-dependent transient disruption of the Bloom&#39;s syndrome gene. However, we could not purify homozygous mutants from a large population of heterozygous mutant cells, limiting the applications. Here we developed a strategy for rapid enrichment of homozygous mutant mouse ESCs and demonstrated its feasibility for cell-based phenotypic analysis. The method uses G418-plus-puromycin double selection to enrich for homozygotes and single-nucleotide polymorphism analysis for identification of homozygosity. We combined this simple approach with gene-trap mutagenesis to construct a homozygous mutant ESC bank with 138 mutant lines and demonstrate its use in phenotype-driven genetic screening.</p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">1071-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22020066">22020066</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1739">10.1038/nmeth.1739</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large duplications at reciprocal translocation breakpoints that might be the counterpart of large deletions and could arise from stalled replication bubbles.</h4>
<p class="authors"><span class="author">Howarth KD</span>, <span class="author">Pole JC</span>, <span class="author">Beavis JC</span>, <span class="author">Batty EM</span>, <span class="author">Newman S</span>, <span class="author">Bignell GR</span> and <span class="author">Edwards PA</span></p>
<p class="affiliation">Hutchison/MRC Research Centre and Department of Pathology, University of Cambridge, Cambridge, UK. <a href="mailto:%6B%64%68%32%39%40%63%61%6D.%61%63.%75%6B">&#107;&#100;&#104;&#50;&#57;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Reciprocal chromosome translocations are often not exactly reciprocal. Most familiar are deletions at the breakpoints, up to megabases in extent. We describe here the opposite phenomenon-duplication of tens or hundreds of kilobases at the breakpoint junction, so that the same sequence is present on both products of a translocation. When the products of the translocation are mapped on the genome, they overlap. We report several of these &quot;overlapping-breakpoint&quot; duplications in breast cancer cell lines HCC1187, HCC1806, and DU4475. These lines also had deletions and essentially balanced translocations. In HCC1187 and HCC1806, we identified five cases of duplication ranging between 46 kb and 200 kb, with the partner chromosome showing deletions between 29 bp and 31 Mb. DU4475 had a duplication of at least 200 kb. Breakpoints were mapped using array painting, i.e., hybridization of chromosomes isolated by flow cytometry to custom oligonucleotide microarrays. Duplications were verified by fluorescent in situ hybridization (FISH), PCR on isolated chromosomes, and cloning of breakpoints. We propose that these duplications are the counterpart of deletions and that they are produced at a replication bubble, comprising two replication forks with the duplicated sequence in between. Both copies of the duplicated sequence would go to one daughter cell, on different products of the translocation, while the other daughter cell would show deletion. These duplications may have been overlooked because they may be missed by FISH and array-CGH and may be interpreted as insertions by paired-end sequencing. Such duplications may therefore be quite frequent.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">4</span>;<span class="pagerange">525-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21252201" rel="external">21252201</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3065700" rel="external">3065700</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.114116.110" rel="external">10.1101/gr.114116.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genetic structure of the Swedish population.</h4>
<p class="authors"><span class="author">Humphreys K</span>, <span class="author">Grankvist A</span>, <span class="author">Leu M</span>, <span class="author">Hall P</span>, <span class="author">Liu J</span>, <span class="author">Ripatti S</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Groop L</span>, <span class="author">Klareskog L</span>, <span class="author">Ding B</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Xu J</span>, <span class="author">Pedersen NL</span>, <span class="author">Lichtenstein P</span>, <span class="author">Mattingsdal M</span>, <span class="author">Andreassen OA</span>, <span class="author">O&#39;Dushlaine C</span>, <span class="author">Purcell SM</span>, <span class="author">Sklar P</span>, <span class="author">Sullivan PF</span>, <span class="author">Hultman CM</span>, <span class="author">Palmgren J</span> and <span class="author">Magnusson PK</span></p>
<p class="affiliation">Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden.</p><p class="abstract">Patterns of genetic diversity have previously been shown to mirror geography on a global scale and within continents and individual countries. Using genome-wide SNP data on 5174 Swedes with extensive geographical coverage, we analyzed the genetic structure of the Swedish population. We observed strong differences between the far northern counties and the remaining counties. The population of Dalarna county, in north middle Sweden, which borders southern Norway, also appears to differ markedly from other counties, possibly due to this county having more individuals with remote Finnish or Norwegian ancestry than other counties. An analysis of genetic differentiation (based on pairwise F(st)) indicated that the population of Sweden&#39;s southernmost counties are genetically closer to the HapMap CEU samples of Northern European ancestry than to the populations of Sweden&#39;s northernmost counties. In a comparison of extended homozygous segments, we detected a clear divide between southern and northern Sweden with small differences between the southern counties and considerably more segments in northern Sweden. Both the increased degree of homozygosity in the north and the large genetic differences between the south and the north may have arisen due to a small population in the north and the vast geographical distances between towns and villages in the north, in contrast to the more densely settled southern parts of Sweden. Our findings have implications for future genome-wide association studies (GWAS) with respect to the matching of cases and controls and the need for within-county matching. We have shown that genetic differences within a single country may be substantial, even when viewed on a European scale. Thus, population stratification needs to be accounted for, even within a country like Sweden, which is often perceived to be relatively homogenous and a favourable resource for genetic mapping, otherwise inferences based on genetic data may lead to false conclusions.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01 CA58427</span>; NIMH NIH HHS: <span class="grant">MH077139</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">e22547</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21829632" rel="external">21829632</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3150368" rel="external">3150368</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0022547" rel="external">10.1371/journal.pone.0022547</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An activating mutation of AKT2 and human hypoglycemia.</h4>
<p class="authors"><span class="author">Hussain K</span>, <span class="author">Challis B</span>, <span class="author">Rocha N</span>, <span class="author">Payne F</span>, <span class="author">Minic M</span>, <span class="author">Thompson A</span>, <span class="author">Daly A</span>, <span class="author">Scott C</span>, <span class="author">Harris J</span>, <span class="author">Smillie BJ</span>, <span class="author">Savage DB</span>, <span class="author">Ramaswami U</span>, <span class="author">De Lonlay P</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Barroso I</span> and <span class="author">Semple RK</span></p>
<p class="affiliation">Clinical and Molecular Genetics Unit, Developmental Endocrinology Research Group, Institute of Child Health, University College London, London WC1N 1EH, UK.</p><p class="abstract">Pathological fasting hypoglycemia in humans is usually explained by excessive circulating insulin or insulin-like molecules or by inborn errors of metabolism impairing liver glucose production. We studied three unrelated children with unexplained, recurrent, and severe fasting hypoglycemia and asymmetrical growth. All were found to carry the same de novo mutation, p.Glu17Lys, in the serine/threonine kinase AKT2, in two cases as heterozygotes and in one case in mosaic form. In heterologous cells, the mutant AKT2 was constitutively recruited to the plasma membrane, leading to insulin-independent activation of downstream signaling. Thus, systemic metabolic disease can result from constitutive, cell-autonomous activation of signaling pathways normally controlled by insulin.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0502115</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">078986</span>, <span class="grant">078986/Z/06/Z</span>, <span class="grant">080952</span>, <span class="grant">080952/Z/06/Z</span>, <span class="grant">091551</span>, <span class="grant">091551/Z/10/Z</span>, <span class="grant">095515</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">334</span>;<span class="number">6055</span>;<span class="pagerange">474</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21979934">21979934</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3204221">3204221</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1210878">10.1126/science.1210878</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale gene-centric analysis identifies novel variants for coronary artery disease.</h4>
<p class="authors"><span class="author">IBC 50K CAD Consortium</span></p><p class="abstract">Coronary artery disease (CAD) has a significant genetic contribution that is incompletely characterized. To complement genome-wide association (GWA) studies, we conducted a large and systematic candidate gene study of CAD susceptibility, including analysis of many uncommon and functional variants. We examined 49,094 genetic variants in &sim;2,100 genes of cardiovascular relevance, using a customised gene array in 15,596 CAD cases and 34,992 controls (11,202 cases and 30,733 controls of European descent; 4,394 cases and 4,259 controls of South Asian origin). We attempted to replicate putative novel associations in an additional 17,121 CAD cases and 40,473 controls. Potential mechanisms through which the novel variants could affect CAD risk were explored through association tests with vascular risk factors and gene expression. We confirmed associations of several previously known CAD susceptibility loci (eg, 9p21.3:p&lt;10(-33); LPA:p&lt;10(-19); 1p13.3:p&lt;10(-17)) as well as three recently discovered loci (COL4A1/COL4A2, ZC3HC1, CYP17A1:p&lt;5&times;10(-7)). However, we found essentially null results for most previously suggested CAD candidate genes. In our replication study of 24 promising common variants, we identified novel associations of variants in or near LIPA, IL5, TRIB1, and ABCG5/ABCG8, with per-allele odds ratios for CAD risk with each of the novel variants ranging from 1.06-1.09. Associations with variants at LIPA, TRIB1, and ABCG5/ABCG8 were supported by gene expression data or effects on lipid levels. Apart from the previously reported variants in LPA, none of the other &sim;4,500 low frequency and functional variants showed a strong effect. Associations in South Asians did not differ appreciably from those in Europeans, except for 9p21.3 (per-allele odds ratio: 1.14 versus 1.27 respectively; P for heterogeneity&#x200A;=&#x200A;0.003). This large-scale gene-centric analysis has identified several novel genes for CAD that relate to diverse biochemical and cellular functions and clarified the literature with regard to many previously suggested genes.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/08/014/24067</span>, <span class="grant">RG/09/12/28096</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G1000143</span>, <span class="grant">MC_U105260792</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U137686857</span>; NHLBI NIH HHS: <span class="grant">R01 HL087647</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e1002260</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21966275">21966275</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3178591">3178591</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002260">10.1371/journal.pgen.1002260</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinguishing driver and passenger mutations in an evolutionary history categorized by interference.</h4>
<p class="authors"><span class="author">Illingworth CJ</span> and <span class="author">Mustonen V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">In many biological scenarios, from the development of drug resistance in pathogens to the progression of healthy cells toward cancer, quantifying the selection acting on observed mutations is a central question. One difficulty in answering this question is the complexity of the background upon which mutations can arise, with multiple potential interactions between genetic loci. We here present a method for discerning selection from a population history that accounts for interference between mutations. Given sequences sampled from multiple time points in the history of a population, we infer selection at each locus by maximizing a likelihood function derived from a multilocus evolution model. We apply the method to the question of distinguishing between loci where new mutations are under positive selection (drivers) and loci that emit neutral mutations (passengers) in a Wright-Fisher model of evolution. Relative to an otherwise equivalent method in which the genetic background of mutations was ignored, our method inferred selection coefficients more accurately for both driver mutations evolving under clonal interference and passenger mutations reaching fixation in the population through genetic drift or hitchhiking. In a population history recorded by 750 sets of sequences of 100 individuals taken at intervals of 100 generations, a set of 50 loci were divided into drivers and passengers with a mean accuracy of &gt;0.95 across a range of numbers of driver loci. The potential application of our model, either in full or in part, to a range of biological systems, is discussed.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">091747</span></p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">189</span>;<span class="number">3</span>;<span class="pagerange">989-1000</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21900272">21900272</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3213378">3213378</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.111.133975">10.1534/genetics.111.133975</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maternally derived microduplications at 15q11-q13: implication of imprinted genes in psychotic illness.</h4>
<p class="authors"><span class="author">Ingason A</span>, <span class="author">Kirov G</span>, <span class="author">Giegling I</span>, <span class="author">Hansen T</span>, <span class="author">Isles AR</span>, <span class="author">Jakobsen KD</span>, <span class="author">Kristinsson KT</span>, <span class="author">le Roux L</span>, <span class="author">Gustafsson O</span>, <span class="author">Craddock N</span>, <span class="author">M&ouml;ller HJ</span>, <span class="author">McQuillin A</span>, <span class="author">Muglia P</span>, <span class="author">Cichon S</span>, <span class="author">Rietschel M</span>, <span class="author">Ophoff RA</span>, <span class="author">Djurovic S</span>, <span class="author">Andreassen OA</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Peltonen L</span>, <span class="author">Dempster E</span>, <span class="author">Collier DA</span>, <span class="author">St Clair D</span>, <span class="author">Rasmussen HB</span>, <span class="author">Glenth&oslash;j BY</span>, <span class="author">Kiemeney LA</span>, <span class="author">Franke B</span>, <span class="author">Tosato S</span>, <span class="author">Bonetto C</span>, <span class="author">Saemundsen E</span>, <span class="author">Hreidarsson SJ</span>, <span class="author">GROUP Investigators</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Gurling H</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Owen MJ</span>, <span class="author">Sigurdsson E</span>, <span class="author">Petursson H</span>, <span class="author">Stefansson H</span>, <span class="author">Rujescu D</span>, <span class="author">Stefansson K</span> and <span class="author">Werge T</span></p>
<p class="affiliation">Research Institute of Biological Psychiatry, Mental Health Center Sct. Hans, Copenhagen University Hospital, Roskilde, Denmark.</p>
<p class="abstract">Objective: Rare copy number variants have been implicated in different neurodevelopmental disorders, with the same copy number variants often increasing risk of more than one of these phenotypes. In a discovery sample of 22 schizophrenia patients with an early onset of illness (10-15 years of age), the authors observed in one patient a maternally derived 15q11-q13 duplication overlapping the Prader-Willi/Angelman syndrome critical region. This prompted investigation of the role of 15q11-q13 duplications in psychotic illness.

Method: The authors scanned 7,582 patients with schizophrenia or schizoaffective disorder and 41,370 comparison subjects without known psychiatric illness for copy number variants at 15q11-q13 and determined the parental origin of duplications using methylation-sensitive Southern hybridization analysis.

Results: Duplications were found in four case patients and five comparison subjects. All four case patients had maternally derived duplications (0.05%), while only three of the five comparison duplications were maternally derived (0.007%), resulting in a significant excess of maternally derived duplications in case patients (odds ratio=7.3). This excess is compatible with earlier observations that risk for psychosis in people with Prader-Willi syndrome caused by maternal uniparental disomy is much higher than in those caused by deletion of the paternal chromosome.

Conclusions: These findings suggest that the presence of two maternal copies of a fragment of chromosome 15q11.2-q13.1 that overlaps with the Prader-Willi/Angelman syndrome critical region may be a rare risk factor for schizophrenia and other psychoses. Given that maternal duplications of this region are among the most consistent cytogenetic observations in autism, the findings provide further support for a shared genetic etiology between autism and psychosis.</p>
<p class="grants">Funded by: Medical Research Council; NIMH NIH HHS: <span class="grant">MH071425</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">The American journal of psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">168</span>;<span class="number">4</span>;<span class="pagerange">408-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21324950" rel="external">21324950</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428917" rel="external">3428917</a>; DOI: <a href="http://dx.doi.org/10.1176%2Fappi.ajp.2010.09111660" rel="external">10.1176/appi.ajp.2010.09111660</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genotype Calling</h4>
<p class="authors"><span class="author">Inouye,M.</span> and <span class="author">Teo,Y.Y.</span></p>
<p class="publication"><span class="title">Analysis of Complex Disease Association Studies</span>&nbsp;<span class="year">2011</span>;<span class="number">Chapter 5</span>;<span class="pagerange">69-86</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/I10.1016%2FB978-0-12-375142-3.10005-7">I10.1016/B978-0-12-375142-3.10005-7</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123751423100057" href="http://www.sciencedirect.com/science/article/pii/B9780123751423100057">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123751423</span>100057</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Design and cohort description of the InterAct Project: an examination of the interaction of genetic and lifestyle factors on the incidence of type 2 diabetes in the EPIC Study.</h4>
<p class="authors"><span class="author">InterAct Consortium</span>, <span class="author">Langenberg C</span>, <span class="author">Sharp S</span>, <span class="author">Forouhi NG</span>, <span class="author">Franks PW</span>, <span class="author">Schulze MB</span>, <span class="author">Kerrison N</span>, <span class="author">Ekelund U</span>, <span class="author">Barroso I</span>, <span class="author">Panico S</span>, <span class="author">Tormo MJ</span>, <span class="author">Spranger J</span>, <span class="author">Griffin S</span>, <span class="author">van der Schouw YT</span>, <span class="author">Amiano P</span>, <span class="author">Ardanaz E</span>, <span class="author">Arriola L</span>, <span class="author">Balkau B</span>, <span class="author">Barricarte A</span>, <span class="author">Beulens JW</span>, <span class="author">Boeing H</span>, <span class="author">Bueno-de-Mesquita HB</span>, <span class="author">Buijsse B</span>, <span class="author">Chirlaque Lopez MD</span>, <span class="author">Clavel-Chapelon F</span>, <span class="author">Crowe FL</span>, <span class="author">de Lauzon-Guillan B</span>, <span class="author">Deloukas P</span>, <span class="author">Dorronsoro M</span>, <span class="author">Drogan D</span>, <span class="author">Froguel P</span>, <span class="author">Gonzalez C</span>, <span class="author">Grioni S</span>, <span class="author">Groop L</span>, <span class="author">Groves C</span>, <span class="author">Hainaut P</span>, <span class="author">Halkjaer J</span>, <span class="author">Hallmans G</span>, <span class="author">Hansen T</span>, <span class="author">Huerta Casta&ntilde;o JM</span>, <span class="author">Kaaks R</span>, <span class="author">Key TJ</span>, <span class="author">Khaw KT</span>, <span class="author">Koulman A</span>, <span class="author">Mattiello A</span>, <span class="author">Navarro C</span>, <span class="author">Nilsson P</span>, <span class="author">Norat T</span>, <span class="author">Overvad K</span>, <span class="author">Palla L</span>, <span class="author">Palli D</span>, <span class="author">Pedersen O</span>, <span class="author">Peeters PH</span>, <span class="author">Quir&oacute;s JR</span>, <span class="author">Ramachandran A</span>, <span class="author">Rodriguez-Suarez L</span>, <span class="author">Rolandsson O</span>, <span class="author">Romaguera D</span>, <span class="author">Romieu I</span>, <span class="author">Sacerdote C</span>, <span class="author">S&aacute;nchez MJ</span>, <span class="author">Sandbaek A</span>, <span class="author">Slimani N</span>, <span class="author">Sluijs I</span>, <span class="author">Spijkerman AM</span>, <span class="author">Teucher B</span>, <span class="author">Tjonneland A</span>, <span class="author">Tumino R</span>, <span class="author">van der A DL</span>, <span class="author">Verschuren WM</span>, <span class="author">Tuomilehto J</span>, <span class="author">Feskens E</span>, <span class="author">McCarthy M</span>, <span class="author">Riboli E</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Institute of Metabolic Science, Addenbrooke&rsquo;s Hospital, Box 285, Cambridge CB2 0QQ, UK e-mail: <a href="mailto:%63%6C%61%75%64%69%61.%6C%61%6E%67%65%6E%62%65%72%67%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#99;&#108;&#97;&#117;&#100;&#105;&#97;&#46;&#108;&#97;&#110;&#103;&#101;&#110;&#98;&#101;&#114;&#103;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Studying gene-lifestyle interaction may help to identify lifestyle factors that modify genetic susceptibility and uncover genetic loci exerting important subgroup effects. Adequately powered studies with prospective, unbiased, standardised assessment of key behavioural factors for gene-lifestyle studies are lacking. This case-cohort study aims to investigate how genetic and potentially modifiable lifestyle and behavioural factors, particularly diet and physical activity, interact in their influence on the risk of developing type 2 diabetes.</p><p class="abstract"><strong>Methods</strong>: Incident cases of type 2 diabetes occurring in European Prospective Investigation into Cancer and Nutrition (EPIC) cohorts between 1991 and 2007 from eight of the ten EPIC countries were ascertained and verified. Prentice-weighted Cox regression and random-effects meta-analyses were used to investigate differences in diabetes incidence by age and sex.</p><p class="abstract"><strong>Results</strong>: A total of 12,403 verified incident cases of type 2 diabetes occurred during 3.99 million person-years of follow-up of 340,234 EPIC participants eligible for InterAct. We defined a centre-stratified subcohort of 16,154 individuals for comparative analyses. Individuals with incident diabetes who were randomly selected into the subcohort (n = 778) were included as cases in the analyses. All prevalent diabetes cases were excluded from the study. InterAct cases were followed-up for an average of 6.9 years; 49.7% were men. Mean baseline age and age at diagnosis were 55.6 and 62.5 years, mean BMI and waist circumference values were 29.4 kg/m(2) and 102.7 cm in men, and 30.1 kg/m(2) and 92.8 cm in women, respectively. Risk of type 2 diabetes increased linearly with age, with an overall HR of 1.56 (95% CI 1.48-1.64) for a 10 year age difference, adjusted for sex. A male excess in the risk of incident diabetes was consistently observed across all countries, with a pooled HR of 1.51 (95% CI 1.39-1.64), adjusted for age.</p><p class="abstract">InterAct is a large, well-powered, prospective study that will inform our understanding of the interplay between genes and lifestyle factors on the risk of type 2 diabetes development.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">G0601261</span>; Cancer Research UK: <span class="grant">11692</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0601261</span>, <span class="grant">G1000143</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179473</span>, <span class="grant">MC_U106179474</span>, <span class="grant">MC_UP_A090_1006</span>, <span class="grant">MC_UP_A100_1003</span>; Wellcome Trust: <span class="grant">083270/083270/z</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2011</span>;<span class="volume">54</span>;<span class="number">9</span>;<span class="pagerange">2272-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21717116">21717116</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4222062">4222062</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-011-2182-9">10.1007/s00125-011-2182-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variants in novel pathways influence blood pressure and cardiovascular disease risk.</h4>
<p class="authors"><span class="author">International Consortium for Blood Pressure Genome-Wide Association Studies</span>, <span class="author">Ehret GB</span>, <span class="author">Munroe PB</span>, <span class="author">Rice KM</span>, <span class="author">Bochud M</span>, <span class="author">Johnson AD</span>, <span class="author">Chasman DI</span>, <span class="author">Smith AV</span>, <span class="author">Tobin MD</span>, <span class="author">Verwoert GC</span>, <span class="author">Hwang SJ</span>, <span class="author">Pihur V</span>, <span class="author">Vollenweider P</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Amin N</span>, <span class="author">Bragg-Gresham JL</span>, <span class="author">Teumer A</span>, <span class="author">Glazer NL</span>, <span class="author">Launer L</span>, <span class="author">Zhao JH</span>, <span class="author">Aulchenko Y</span>, <span class="author">Heath S</span>, <span class="author">S&otilde;ber S</span>, <span class="author">Parsa A</span>, <span class="author">Luan J</span>, <span class="author">Arora P</span>, <span class="author">Dehghan A</span>, <span class="author">Zhang F</span>, <span class="author">Lucas G</span>, <span class="author">Hicks AA</span>, <span class="author">Jackson AU</span>, <span class="author">Peden JF</span>, <span class="author">Tanaka T</span>, <span class="author">Wild SH</span>, <span class="author">Rudan I</span>, <span class="author">Igl W</span>, <span class="author">Milaneschi Y</span>, <span class="author">Parker AN</span>, <span class="author">Fava C</span>, <span class="author">Chambers JC</span>, <span class="author">Fox ER</span>, <span class="author">Kumari M</span>, <span class="author">Go MJ</span>, <span class="author">van der Harst P</span>, <span class="author">Kao WH</span>, <span class="author">Sj&ouml;gren M</span>, <span class="author">Vinay DG</span>, <span class="author">Alexander M</span>, <span class="author">Tabara Y</span>, <span class="author">Shaw-Hawkins S</span>, <span class="author">Whincup PH</span>, <span class="author">Liu Y</span>, <span class="author">Shi G</span>, <span class="author">Kuusisto J</span>, <span class="author">Tayo B</span>, <span class="author">Seielstad M</span>, <span class="author">Sim X</span>, <span class="author">Nguyen KD</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Matullo G</span>, <span class="author">Wu Y</span>, <span class="author">Gaunt TR</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Cooper MN</span>, <span class="author">Platou CG</span>, <span class="author">Org E</span>, <span class="author">Hardy R</span>, <span class="author">Dahgam S</span>, <span class="author">Palmen J</span>, <span class="author">Vitart V</span>, <span class="author">Braund PS</span>, <span class="author">Kuznetsova T</span>, <span class="author">Uiterwaal CS</span>, <span class="author">Adeyemo A</span>, <span class="author">Palmas W</span>, <span class="author">Campbell H</span>, <span class="author">Ludwig B</span>, <span class="author">Tomaszewski M</span>, <span class="author">Tzoulaki I</span>, <span class="author">Palmer ND</span>, <span class="author">CARDIoGRAM consortium</span>, <span class="author">CKDGen Consortium</span>, <span class="author">KidneyGen Consortium</span>, <span class="author">EchoGen consortium</span>, <span class="author">CHARGE-HF consortium</span>, <span class="author">Aspelund T</span>, <span class="author">Garcia M</span>, <span class="author">Chang YP</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Steinle NI</span>, <span class="author">Grobbee DE</span>, <span class="author">Arking DE</span>, <span class="author">Kardia SL</span>, <span class="author">Morrison AC</span>, <span class="author">Hernandez D</span>, <span class="author">Najjar S</span>, <span class="author">McArdle WL</span>, <span class="author">Hadley D</span>, <span class="author">Brown MJ</span>, <span class="author">Connell JM</span>, <span class="author">Hingorani AD</span>, <span class="author">Day IN</span>, <span class="author">Lawlor DA</span>, <span class="author">Beilby JP</span>, <span class="author">Lawrence RW</span>, <span class="author">Clarke R</span>, <span class="author">Hopewell JC</span>, <span class="author">Ongen H</span>, <span class="author">Dreisbach AW</span>, <span class="author">Li Y</span>, <span class="author">Young JH</span>, <span class="author">Bis JC</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Viikari J</span>, <span class="author">Adair LS</span>, <span class="author">Lee NR</span>, <span class="author">Chen MH</span>, <span class="author">Olden M</span>, <span class="author">Pattaro C</span>, <span class="author">Bolton JA</span>, <span class="author">K&ouml;ttgen A</span>, <span class="author">Bergmann S</span>, <span class="author">Mooser V</span>, <span class="author">Chaturvedi N</span>, <span class="author">Frayling TM</span>, <span class="author">Islam M</span>, <span class="author">Jafar TH</span>, <span class="author">Erdmann J</span>, <span class="author">Kulkarni SR</span>, <span class="author">Bornstein SR</span>, <span class="author">Gr&auml;ssler J</span>, <span class="author">Groop L</span>, <span class="author">Voight BF</span>, <span class="author">Kettunen J</span>, <span class="author">Howard P</span>, <span class="author">Taylor A</span>, <span class="author">Guarrera S</span>, <span class="author">Ricceri F</span>, <span class="author">Emilsson V</span>, <span class="author">Plump A</span>, <span class="author">Barroso I</span>, <span class="author">Khaw KT</span>, <span class="author">Weder AB</span>, <span class="author">Hunt SC</span>, <span class="author">Sun YV</span>, <span class="author">Bergman RN</span>, <span class="author">Collins FS</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Scott LJ</span>, <span class="author">Stringham HM</span>, <span class="author">Peltonen L</span>, <span class="author">Perola M</span>, <span class="author">Vartiainen E</span>, <span class="author">Brand SM</span>, <span class="author">Staessen JA</span>, <span class="author">Wang TJ</span>, <span class="author">Burton PR</span>, <span class="author">Soler Artigas M</span>, <span class="author">Dong Y</span>, <span class="author">Snieder H</span>, <span class="author">Wang X</span>, <span class="author">Zhu H</span>, <span class="author">Lohman KK</span>, <span class="author">Rudock ME</span>, <span class="author">Heckbert SR</span>, <span class="author">Smith NL</span>, <span class="author">Wiggins KL</span>, <span class="author">Doumatey A</span>, <span class="author">Shriner D</span>, <span class="author">Veldre G</span>, <span class="author">Viigimaa M</span>, <span class="author">Kinra S</span>, <span class="author">Prabhakaran D</span>, <span class="author">Tripathy V</span>, <span class="author">Langefeld CD</span>, <span class="author">Rosengren A</span>, <span class="author">Thelle DS</span>, <span class="author">Corsi AM</span>, <span class="author">Singleton A</span>, <span class="author">Forrester T</span>, <span class="author">Hilton G</span>, <span class="author">McKenzie CA</span>, <span class="author">Salako T</span>, <span class="author">Iwai N</span>, <span class="author">Kita Y</span>, <span class="author">Ogihara T</span>, <span class="author">Ohkubo T</span>, <span class="author">Okamura T</span>, <span class="author">Ueshima H</span>, <span class="author">Umemura S</span>, <span class="author">Eyheramendy S</span>, <span class="author">Meitinger T</span>, <span class="author">Wichmann HE</span>, <span class="author">Cho YS</span>, <span class="author">Kim HL</span>, <span class="author">Lee JY</span>, <span class="author">Scott J</span>, <span class="author">Sehmi JS</span>, <span class="author">Zhang W</span>, <span class="author">Hedblad B</span>, <span class="author">Nilsson P</span>, <span class="author">Smith GD</span>, <span class="author">Wong A</span>, <span class="author">Narisu N</span>, <span class="author">Stan&#x10D;&aacute;kov&aacute; A</span>, <span class="author">Raffel LJ</span>, <span class="author">Yao J</span>, <span class="author">Kathiresan S</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Schwartz SM</span>, <span class="author">Ikram MA</span>, <span class="author">Longstreth WT</span>, <span class="author">Mosley TH</span>, <span class="author">Seshadri S</span>, <span class="author">Shrine NR</span>, <span class="author">Wain LV</span>, <span class="author">Morken MA</span>, <span class="author">Swift AJ</span>, <span class="author">Laitinen J</span>, <span class="author">Prokopenko I</span>, <span class="author">Zitting P</span>, <span class="author">Cooper JA</span>, <span class="author">Humphries SE</span>, <span class="author">Danesh J</span>, <span class="author">Rasheed A</span>, <span class="author">Goel A</span>, <span class="author">Hamsten A</span>, <span class="author">Watkins H</span>, <span class="author">Bakker SJ</span>, <span class="author">van Gilst WH</span>, <span class="author">Janipalli CS</span>, <span class="author">Mani KR</span>, <span class="author">Yajnik CS</span>, <span class="author">Hofman A</span>, <span class="author">Mattace-Raso FU</span>, <span class="author">Oostra BA</span>, <span class="author">Demirkan A</span>, <span class="author">Isaacs A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Lakatta EG</span>, <span class="author">Orru M</span>, <span class="author">Scuteri A</span>, <span class="author">Ala-Korpela M</span>, <span class="author">Kangas AJ</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Soininen P</span>, <span class="author">Tukiainen T</span>, <span class="author">W&uuml;rtz P</span>, <span class="author">Ong RT</span>, <span class="author">D&ouml;rr M</span>, <span class="author">Kroemer HK</span>, <span class="author">V&ouml;lker U</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Galan P</span>, <span class="author">Hercberg S</span>, <span class="author">Lathrop M</span>, <span class="author">Zelenika D</span>, <span class="author">Deloukas P</span>, <span class="author">Mangino M</span>, <span class="author">Spector TD</span>, <span class="author">Zhai G</span>, <span class="author">Meschia JF</span>, <span class="author">Nalls MA</span>, <span class="author">Sharma P</span>, <span class="author">Terzic J</span>, <span class="author">Kumar MV</span>, <span class="author">Denniff M</span>, <span class="author">Zukowska-Szczechowska E</span>, <span class="author">Wagenknecht LE</span>, <span class="author">Fowkes FG</span>, <span class="author">Charchar FJ</span>, <span class="author">Schwarz PE</span>, <span class="author">Hayward C</span>, <span class="author">Guo X</span>, <span class="author">Rotimi C</span>, <span class="author">Bots ML</span>, <span class="author">Brand E</span>, <span class="author">Samani NJ</span>, <span class="author">Polasek O</span>, <span class="author">Talmud PJ</span>, <span class="author">Nyberg F</span>, <span class="author">Kuh D</span>, <span class="author">Laan M</span>, <span class="author">Hveem K</span>, <span class="author">Palmer LJ</span>, <span class="author">van der Schouw YT</span>, <span class="author">Casas JP</span>, <span class="author">Mohlke KL</span>, <span class="author">Vineis P</span>, <span class="author">Raitakari O</span>, <span class="author">Ganesh SK</span>, <span class="author">Wong TY</span>, <span class="author">Tai ES</span>, <span class="author">Cooper RS</span>, <span class="author">Laakso M</span>, <span class="author">Rao DC</span>, <span class="author">Harris TB</span>, <span class="author">Morris RW</span>, <span class="author">Dominiczak AF</span>, <span class="author">Kivimaki M</span>, <span class="author">Marmot MG</span>, <span class="author">Miki T</span>, <span class="author">Saleheen D</span>, <span class="author">Chandak GR</span>, <span class="author">Coresh J</span>, <span class="author">Navis G</span>, <span class="author">Salomaa V</span>, <span class="author">Han BG</span>, <span class="author">Zhu X</span>, <span class="author">Kooner JS</span>, <span class="author">Melander O</span>, <span class="author">Ridker PM</span>, <span class="author">Bandinelli S</span>, <span class="author">Gyllensten UB</span>, <span class="author">Wright AF</span>, <span class="author">Wilson JF</span>, <span class="author">Ferrucci L</span>, <span class="author">Farrall M</span>, <span class="author">Tuomilehto J</span>, <span class="author">Pramstaller PP</span>, <span class="author">Elosua R</span>, <span class="author">Soranzo N</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Altshuler D</span>, <span class="author">Loos RJ</span>, <span class="author">Shuldiner AR</span>, <span class="author">Gieger C</span>, <span class="author">Meneton P</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Wareham NJ</span>, <span class="author">Gudnason V</span>, <span class="author">Rotter JI</span>, <span class="author">Rettig R</span>, <span class="author">Uda M</span>, <span class="author">Strachan DP</span>, <span class="author">Witteman JC</span>, <span class="author">Hartikainen AL</span>, <span class="author">Beckmann JS</span>, <span class="author">Boerwinkle E</span>, <span class="author">Vasan RS</span>, <span class="author">Boehnke M</span>, <span class="author">Larson MG</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Psaty BM</span>, <span class="author">Abecasis GR</span>, <span class="author">Chakravarti A</span>, <span class="author">Elliott P</span>, <span class="author">van Duijn CM</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Levy D</span>, <span class="author">Caulfield MJ</span> and <span class="author">Johnson T</span></p><p class="abstract">Blood pressure is a heritable trait influenced by several biological pathways and responsive to environmental stimuli. Over one billion people worldwide have hypertension (&ge;140&thinsp;mm&thinsp;Hg systolic blood pressure or&thinsp; &ge;90&thinsp;mm&thinsp;Hg diastolic blood pressure). Even small increments in blood pressure are associated with an increased risk of cardiovascular events. This genome-wide association study of systolic and diastolic blood pressure, which used a multi-stage design in 200,000 individuals of European descent, identified sixteen novel loci: six of these loci contain genes previously known or suspected to regulate blood pressure (GUCY1A3-GUCY1B3, NPR3-C5orf23, ADM, FURIN-FES, GOSR2, GNAS-EDN3); the other ten provide new clues to blood pressure physiology. A genetic risk score based on 29 genome-wide significant variants was associated with hypertension, left ventricular wall thickness, stroke and coronary artery disease, but not kidney disease or kidney function. We also observed associations with blood pressure in East Asian, South Asian and African ancestry individuals. Our findings provide new insights into the genetics and biology of blood pressure, and suggest potential novel therapeutic pathways for cardiovascular disease prevention.</p>
<p class="grants">Funded by: AHRQ HHS: <span class="grant">HS06516</span>; Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">CH/03/001</span>, <span class="grant">FS05/125</span>, <span class="grant">G0501942</span>, <span class="grant">PG/02/128</span>, <span class="grant">PG97012</span>, <span class="grant">PG97027</span>, <span class="grant">RG/07/005/23633</span>, <span class="grant">RG/08/008/25291</span>, <span class="grant">RG/08/013/25942</span>, <span class="grant">RG/08/014/24067</span>, <span class="grant">RG/98002</span>, <span class="grant">RG08/01</span>, <span class="grant">SP/04/002</span>, <span class="grant">SP/08/005/25115</span>; Canadian Institutes of Health Research: <span class="grant">MOP-82810</span>, <span class="grant">MOP172605</span>, <span class="grant">MOP77682</span>; Chief Scientist Office: <span class="grant">CZB/4/276</span>, <span class="grant">CZB/4/710</span>; FIC NIH HHS: <span class="grant">R03 TW007165</span>, <span class="grant">TW008288</span>, <span class="grant">TW05596</span>; Howard Hughes Medical Institute: <span class="grant">55005617</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0400874</span>, <span class="grant">G0401527</span>, <span class="grant">G0500539</span>, <span class="grant">G0501942</span>, <span class="grant">G0600331</span>, <span class="grant">G0600705</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0902037</span>, <span class="grant">G0902313</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U123092723</span>, <span class="grant">MC_U137686857</span>, <span class="grant">MC_UP_A100_1003</span>; NCI NIH HHS: <span class="grant">5U01CA086308</span>, <span class="grant">P01CA055075</span>, <span class="grant">P01CA087969</span>; NCRR NIH HHS: <span class="grant">2M01RR010284</span>, <span class="grant">K12RR023250</span>, <span class="grant">M01 RR16500</span>, <span class="grant">M01-RR00425</span>, <span class="grant">RR-024156</span>, <span class="grant">RR20649</span>, <span class="grant">U54 RR020278</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">HG003054</span>, <span class="grant">HG005581</span>, <span class="grant">U01HG004399</span>, <span class="grant">U01HG004402</span>, <span class="grant">U01HG004415</span>, <span class="grant">U01HG004422</span>, <span class="grant">U01HG004423</span>, <span class="grant">U01HG004436</span>, <span class="grant">U01HG004438</span>, <span class="grant">U01HG004446</span>, <span class="grant">U01HG004726</span>, <span class="grant">U01HG004728</span>, <span class="grant">U01HG004729</span>, <span class="grant">U01HG004735</span>, <span class="grant">U01HG004738</span>; NHLBI NIH HHS: <span class="grant">5R01HL086694-03</span>, <span class="grant">5R01HL087679-02</span>, <span class="grant">5R01HL08770002</span>, <span class="grant">HL 54512</span>, <span class="grant">HL-87660</span>, <span class="grant">HL043851</span>, <span class="grant">HL080025</span>, <span class="grant">HL084729</span>, <span class="grant">HL085144</span>, <span class="grant">HL086718</span>, <span class="grant">HL087647</span>, <span class="grant">HL098283</span>, <span class="grant">HL36310</span>, <span class="grant">HL45508</span>, <span class="grant">HL53353</span>, <span class="grant">HL54512</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01 HC-95159</span>, <span class="grant">N01 HC-95169</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-95160</span>, <span class="grant">N01-HC-95161</span>, <span class="grant">N01-HC-95162</span>, <span class="grant">N01-HC-95163</span>, <span class="grant">N01-HC-95164</span>, <span class="grant">N01-HC-95165</span>, <span class="grant">N01-HC-95166</span>, <span class="grant">N01-HC-95167</span>, <span class="grant">N01-HC-95168</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL073410</span>, <span class="grant">R01 HL085251</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL086694-03</span>, <span class="grant">R01 HL086694-04A1</span>, <span class="grant">R01 HL086694-05</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01HL056931</span>, <span class="grant">R01HL060894</span>, <span class="grant">R01HL060919</span>, <span class="grant">R01HL06094</span>, <span class="grant">R01HL061019</span>, <span class="grant">R01HL071051</span>, <span class="grant">R01HL071205</span>, <span class="grant">R01HL071250</span>, <span class="grant">R01HL071251</span>, <span class="grant">R01HL071252</span>, <span class="grant">R01HL071258</span>, <span class="grant">R01HL071259</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL089650-02</span>, <span class="grant">R01HL59367</span>, <span class="grant">R37HL051021</span>, <span class="grant">U01 HL054466</span>, <span class="grant">U01 HL054466-11</span>, <span class="grant">U01 HL054471</span>, <span class="grant">U01 HL054473</span>, <span class="grant">U01 HL054527</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL080295</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U10 HL054512</span>, <span class="grant">U10HL054512</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A</span>, <span class="grant">AG13196</span>, <span class="grant">N01-AG-1-2109</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">N01AG6210</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">R01 AG017644-09S1</span>, <span class="grant">R01 AG18728</span>; NICHD NIH HHS: <span class="grant">N01-HD-1-3107</span>; NIDCR NIH HHS: <span class="grant">U01DE018903</span>, <span class="grant">U01DE01899</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK063491</span>, <span class="grant">DK072193</span>, <span class="grant">DK075787</span>, <span class="grant">DK078150</span>, <span class="grant">DK56350</span>, <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK078150</span>, <span class="grant">R01DK058845</span>, <span class="grant">R01DK066574</span>, <span class="grant">U01 DK062418</span>; NIEHS NIH HHS: <span class="grant">ES10126</span>, <span class="grant">P30 ES010126</span>, <span class="grant">P30ES007033</span>; NIGMS NIH HHS: <span class="grant">S06GM008016-320107</span>, <span class="grant">S06GM008016-380111</span>, <span class="grant">U01 GM074518-04</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>; NINDS NIH HHS: <span class="grant">R01 NS39987</span>, <span class="grant">R01 NS42733</span>, <span class="grant">U01 NS069208</span>, <span class="grant">U01 NS069208-01</span>; PHS HHS: <span class="grant">263-MA-410953</span>, <span class="grant">33014</span>, <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096</span>, <span class="grant">HHSN268200782096C</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">070191/Z/03/Z</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span>, <span class="grant">080747/Z/06/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">478</span>;<span class="number">7367</span>;<span class="pagerange">103-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21909115">21909115</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3340926">3340926</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10405">10.1038/nature10405</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genetic association of variants in CD6, TNFRSF1A and IRF8 to multiple sclerosis: a multicenter case-control study.</h4>
<p class="authors"><span class="author">International Multiple Sclerosis Genetics Consortium</span></p><p class="abstract"><strong>Background</strong>: In the recently published meta-analysis of multiple sclerosis genome-wide association studies De Jager et al. identified three single nucleotide polymorphisms associated to MS: rs17824933 (CD6), rs1800693 (TNFRSF1A) and rs17445836 (61.5 kb from IRF8). To refine our understanding of these associations we sought to replicate these findings in a large more extensive independent sample set of 11 populations of European origin.</p><p class="abstract">We calculated individual and combined associations using a meta-analysis method by Kazeem and Farral (2005). We confirmed the association of rs1800693 in TNFRSF1A (p 4.19 &times; 10-7, OR 1.12, 7,665 cases, 8,051 controls) and rs17445836 near IRF8 (p 5.35 &times; 10-10, OR 0.84, 6,895 cases, 7,580 controls and 596 case-parent trios) The SNP rs17824933 in CD6 also showed nominally significant evidence for association (p 2.19 &times; 10-5, OR 1.11, 8,047 cases, 9,174 controls, 604 case-parent trios).</p><p class="abstract"><strong>Conclusions</strong>: Variants in TNFRSF1A and in the vicinity of IRF8 were confirmed to be associated in these independent cohorts, which supports the role of these loci in etiology of multiple sclerosis. The variant in CD6 reached genome-wide significance after combining the data with the original meta-analysis. Fine mapping is required to identify the predisposing variants in the loci and future functional studies will refine their molecular role in MS pathogenesis.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">R01 NS 43559</span>, <span class="grant">R01 NS049477</span>, <span class="grant">R01 NS067305</span>; Wellcome Trust: <span class="grant">089061/Z/09/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">e18813</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21552549" rel="external">21552549</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3084233" rel="external">3084233</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0018813" rel="external">10.1371/journal.pone.0018813</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Imputation of sequence variants for identification of genetic risks for Parkinson&#39;s disease: a meta-analysis of genome-wide association studies.</h4>
<p class="authors"><span class="author">International Parkinson Disease Genomics Consortium</span>, <span class="author">Nalls MA</span>, <span class="author">Plagnol V</span>, <span class="author">Hernandez DG</span>, <span class="author">Sharma M</span>, <span class="author">Sheerin UM</span>, <span class="author">Saad M</span>, <span class="author">Sim&oacute;n-S&aacute;nchez J</span>, <span class="author">Schulte C</span>, <span class="author">Lesage S</span>, <span class="author">Sveinbj&ouml;rnsd&oacute;ttir S</span>, <span class="author">Stef&aacute;nsson K</span>, <span class="author">Martinez M</span>, <span class="author">Hardy J</span>, <span class="author">Heutink P</span>, <span class="author">Brice A</span>, <span class="author">Gasser T</span>, <span class="author">Singleton AB</span> and <span class="author">Wood NW</span></p><p class="abstract"><strong>Background</strong>: Genome-wide association studies (GWAS) for Parkinson&#39;s disease have linked two loci (MAPT and SNCA) to risk of Parkinson&#39;s disease. We aimed to identify novel risk loci for Parkinson&#39;s disease.</p><p class="abstract"><strong>Methods</strong>: We did a meta-analysis of datasets from five Parkinson&#39;s disease GWAS from the USA and Europe to identify loci associated with Parkinson&#39;s disease (discovery phase). We then did replication analyses of significantly associated loci in an independent sample series. Estimates of population-attributable risk were calculated from estimates from the discovery and replication phases combined, and risk-profile estimates for loci identified in the discovery phase were calculated.</p><p class="abstract"><strong>Findings</strong>: The discovery phase consisted of 5333 case and 12&#x2008;019 control samples, with genotyped and imputed data at 7&#x2008;689&#x2008;524 SNPs. The replication phase consisted of 7053 case and 9007 control samples. We identified 11 loci that surpassed the threshold for genome-wide significance (p&lt;5&times;10(-8)). Six were previously identified loci (MAPT, SNCA, HLA-DRB5, BST1, GAK and LRRK2) and five were newly identified loci (ACMSD, STK39, MCCC1/LAMP3, SYT11, and CCDC62/HIP1R). The combined population-attributable risk was 60&middot;3% (95% CI 43&middot;7-69&middot;3). In the risk-profile analysis, the odds ratio in the highest quintile of disease risk was 2&middot;51 (95% CI 2&middot;23-2&middot;83) compared with 1&middot;00 in the lowest quintile of disease risk.</p><p class="abstract"><strong>Interpretation</strong>: These data provide an insight into the genetics of Parkinson&#39;s disease and the molecular cause of the disease and could provide future targets for therapies.</p><p class="abstract"><strong>Funding</strong>: Wellcome Trust, National Institute on Aging, and US Department of Defense.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700943</span>; NCRR NIH HHS: <span class="grant">RR024992</span>; NIA NIH HHS: <span class="grant">Z01-AG000949-02</span>; NIEHS NIH HHS: <span class="grant">Z01-ES101986</span>; NINDS NIH HHS: <span class="grant">NS057105</span>, <span class="grant">R01 NS060722-04</span>; Parkinson&#39;s UK: <span class="grant">J-0804</span>, <span class="grant">J-0901</span>; Wellcome Trust: <span class="grant">083948/Z/07/Z</span>, <span class="grant">WT089698</span>, <span class="grant">WT089698/Z/09/Z</span>, <span class="grant">WTCCC1</span>, <span class="grant">WTCCC2</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2011</span>;<span class="volume">377</span>;<span class="number">9766</span>;<span class="pagerange">641-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21292315" rel="external">21292315</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(10)62345-8" rel="external">10.1016/S0140-6736(10)62345-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A two-stage meta-analysis identifies several new loci for Parkinson&#39;s disease.</h4>
<p class="authors"><span class="author">International Parkinson&#39;s Disease Genomics Consortium (IPDGC)</span> and <span class="author">Wellcome Trust Case Control Consortium 2 (WTCCC2)</span></p><p class="abstract">A previous genome-wide association (GWA) meta-analysis of 12,386 PD cases and 21,026 controls conducted by the International Parkinson&#39;s Disease Genomics Consortium (IPDGC) discovered or confirmed 11 Parkinson&#39;s disease (PD) loci. This first analysis of the two-stage IPDGC study focused on the set of loci that passed genome-wide significance in the first stage GWA scan. However, the second stage genotyping array, the ImmunoChip, included a larger set of 1,920 SNPs selected on the basis of the GWA analysis. Here, we analyzed this set of 1,920 SNPs, and we identified five additional PD risk loci (combined p&lt;5&times;10(-10), PARK16/1q32, STX1B/16p11, FGF20/8p22, STBD1/4q21, and GPNMB/7p15). Two of these five loci have been suggested by previous association studies (PARK16/1q32, FGF20/8p22), and this study provides further support for these findings. Using a dataset of post-mortem brain samples assayed for gene expression (n&#x200A;=&#x200A;399) and methylation (n&#x200A;=&#x200A;292), we identified methylation and expression changes associated with PD risk variants in PARK16/1q32, GPNMB/7p15, and STX1B/16p11 loci, hence suggesting potential molecular mechanisms and candidate genes at these risk loci.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700943</span>, <span class="grant">G0901254</span>; NHGRI NIH HHS: <span class="grant">ZIA HG200336-05</span>; NIA NIH HHS: <span class="grant">Z01 AG000949-02</span>; NIEHS NIH HHS: <span class="grant">Z01-ES101986</span>; Parkinson&#39;s UK: <span class="grant">J-0804</span>, <span class="grant">J-0901</span>; Wellcome Trust: <span class="grant">085475/B/08/Z</span>, <span class="grant">085475/Z/08/Z</span>, <span class="grant">WT089698</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e1002142</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21738488" rel="external">21738488</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3128098" rel="external">3128098</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002142" rel="external">10.1371/journal.pgen.1002142</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Salmonella Typhimurium-Typhi genomic chimera: a model to study Vi polysaccharide capsule function in vivo.</h4>
<p class="authors"><span class="author">Jansen AM</span>, <span class="author">Hall LJ</span>, <span class="author">Clare S</span>, <span class="author">Goulding D</span>, <span class="author">Holt KE</span>, <span class="author">Grant AJ</span>, <span class="author">Mastroeni P</span>, <span class="author">Dougan G</span> and <span class="author">Kingsley RA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The Vi capsular polysaccharide is a virulence-associated factor expressed by Salmonella enterica serotype Typhi but absent from virtually all other Salmonella serotypes. In order to study this determinant in vivo, we characterised a Vi-positive S. Typhimurium (C5.507 Vi(+)), harbouring the Salmonella pathogenicity island (SPI)-7, which encodes the Vi locus. S. Typhimurium C5.507 Vi(+) colonised and persisted in mice at similar levels compared to the parent strain, S. Typhimurium C5. However, the innate immune response to infection with C5.507 Vi(+) and SGB1, an isogenic derivative not expressing Vi, differed markedly. Infection with C5.507 Vi(+) resulted in a significant reduction in cellular trafficking of innate immune cells, including PMN and NK cells, compared to SGB1 Vi(-) infected animals. C5.507 Vi(+) infection stimulated reduced numbers of TNF-&alpha;, MIP-2 and perforin producing cells compared to SGB1 Vi(-). The modulating effect associated with Vi was not observed in MyD88(-/-) and was reduced in TLR4(-/-) mice. The presence of the Vi capsule also correlated with induction of the anti-inflammatory cytokine IL-10 in vivo, a factor that impacted on chemotaxis and the activation of immune cells in vitro.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e1002131</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21829346" rel="external">21829346</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145788" rel="external">3145788</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002131" rel="external">10.1371/journal.ppat.1002131</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The rise and fall of supervised machine learning techniques.</h4>
<p class="authors"><span class="author">Jensen LJ</span> and <span class="author">Bateman A</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">24</span>;<span class="pagerange">3331-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22101152" rel="external">22101152</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3232371" rel="external">3232371</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr585" rel="external">10.1093/bioinformatics/btr585</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">myKaryoView: a light-weight client for visualization of genomic data.</h4>
<p class="authors"><span class="author">Jimenez RC</span>, <span class="author">Salazar GA</span>, <span class="author">Gel B</span>, <span class="author">Dopazo J</span>, <span class="author">Mulder N</span> and <span class="author">Corpas M</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The Distributed Annotation System (DAS) is a protocol for easy sharing and integration of biological annotations. In order to visualize feature annotations in a genomic context a client is required. Here we present myKaryoView, a simple light-weight DAS tool for visualization of genomic annotation. myKaryoView has been specifically configured to help analyse data derived from personal genomics, although it can also be used as a generic genome browser visualization. Several well-known data sources are provided to facilitate comparison of known genes and normal variation regions. The navigation experience is enhanced by simultaneous rendering of different levels of detail across chromosomes. A simple interface is provided to allow searches for any SNP, gene or chromosomal region. User-defined DAS data sources may also be added when querying the system. We demonstrate myKaryoView capabilities for adding user-defined sources with a set of genetic profiles of family-related individuals downloaded directly from 23andMe. myKaryoView is a web tool for visualization of genomic data specifically designed for direct-to-consumer genomic data that uses publicly available data distributed throughout the Internet. It does not require data to be held locally and it is capable of rendering any feature as long as it conforms to DAS specifications. Configuration and addition of sources to myKaryoView can be done through the interface. Here we show a proof of principle of myKaryoView&#39;s ability to display personal genomics data with 23andMe genome data sources. The tool is available at: http://mykaryoview.com.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">e26345</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22046276" rel="external">22046276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3202530" rel="external">3202530</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0026345" rel="external">10.1371/journal.pone.0026345</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Blood pressure loci identified with a gene-centric array.</h4>
<p class="authors"><span class="author">Johnson T</span>, <span class="author">Gaunt TR</span>, <span class="author">Newhouse SJ</span>, <span class="author">Padmanabhan S</span>, <span class="author">Tomaszewski M</span>, <span class="author">Kumari M</span>, <span class="author">Morris RW</span>, <span class="author">Tzoulaki I</span>, <span class="author">O&#39;Brien ET</span>, <span class="author">Poulter NR</span>, <span class="author">Sever P</span>, <span class="author">Shields DC</span>, <span class="author">Thom S</span>, <span class="author">Wannamethee SG</span>, <span class="author">Whincup PH</span>, <span class="author">Brown MJ</span>, <span class="author">Connell JM</span>, <span class="author">Dobson RJ</span>, <span class="author">Howard PJ</span>, <span class="author">Mein CA</span>, <span class="author">Onipinla A</span>, <span class="author">Shaw-Hawkins S</span>, <span class="author">Zhang Y</span>, <span class="author">Davey Smith G</span>, <span class="author">Day IN</span>, <span class="author">Lawlor DA</span>, <span class="author">Goodall AH</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">Fowkes FG</span>, <span class="author">Abecasis GR</span>, <span class="author">Elliott P</span>, <span class="author">Gateva V</span>, <span class="author">Global BPgen Consortium</span>, <span class="author">Braund PS</span>, <span class="author">Burton PR</span>, <span class="author">Nelson CP</span>, <span class="author">Tobin MD</span>, <span class="author">van der Harst P</span>, <span class="author">Glorioso N</span>, <span class="author">Neuvrith H</span>, <span class="author">Salvi E</span>, <span class="author">Staessen JA</span>, <span class="author">Stucchi A</span>, <span class="author">Devos N</span>, <span class="author">Jeunemaitre X</span>, <span class="author">Plouin PF</span>, <span class="author">Tichet J</span>, <span class="author">Juhanson P</span>, <span class="author">Org E</span>, <span class="author">Putku M</span>, <span class="author">S&otilde;ber S</span>, <span class="author">Veldre G</span>, <span class="author">Viigimaa M</span>, <span class="author">Levinsson A</span>, <span class="author">Rosengren A</span>, <span class="author">Thelle DS</span>, <span class="author">Hastie CE</span>, <span class="author">Hedner T</span>, <span class="author">Lee WK</span>, <span class="author">Melander O</span>, <span class="author">Wahlstrand B</span>, <span class="author">Hardy R</span>, <span class="author">Wong A</span>, <span class="author">Cooper JA</span>, <span class="author">Palmen J</span>, <span class="author">Chen L</span>, <span class="author">Stewart AF</span>, <span class="author">Wells GA</span>, <span class="author">Westra HJ</span>, <span class="author">Wolfs MG</span>, <span class="author">Clarke R</span>, <span class="author">Franzosi MG</span>, <span class="author">Goel A</span>, <span class="author">Hamsten A</span>, <span class="author">Lathrop M</span>, <span class="author">Peden JF</span>, <span class="author">Seedorf U</span>, <span class="author">Watkins H</span>, <span class="author">Ouwehand WH</span>, <span class="author">Sambrook J</span>, <span class="author">Stephens J</span>, <span class="author">Casas JP</span>, <span class="author">Drenos F</span>, <span class="author">Holmes MV</span>, <span class="author">Kivimaki M</span>, <span class="author">Shah S</span>, <span class="author">Shah T</span>, <span class="author">Talmud PJ</span>, <span class="author">Whittaker J</span>, <span class="author">Wallace C</span>, <span class="author">Delles C</span>, <span class="author">Laan M</span>, <span class="author">Kuh D</span>, <span class="author">Humphries SE</span>, <span class="author">Nyberg F</span>, <span class="author">Cusi D</span>, <span class="author">Roberts R</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Franke L</span>, <span class="author">Stanton AV</span>, <span class="author">Dominiczak AF</span>, <span class="author">Farrall M</span>, <span class="author">Hingorani AD</span>, <span class="author">Samani NJ</span>, <span class="author">Caulfield MJ</span> and <span class="author">Munroe PB</span></p>
<p class="affiliation">Clinical Pharmacology and Barts and The London Genome Centre, William Harvey Research Institute, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, Charterhouse Square, London, UK. <a href="mailto:%74.%6A%6F%68%6E%73%6F%6E%40%71%6D%75%6C.%61%63.%75%6B">&#116;&#46;&#106;&#111;&#104;&#110;&#115;&#111;&#110;&#64;&#113;&#109;&#117;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Raised blood pressure (BP) is a major risk factor for cardiovascular disease. Previous studies have identified 47 distinct genetic variants robustly associated with BP, but collectively these explain only a few percent of the heritability for BP phenotypes. To find additional BP loci, we used a bespoke gene-centric array to genotype an independent discovery sample of 25,118 individuals that combined hypertensive case-control and general population samples. We followed up four SNPs associated with BP at our p &lt; 8.56&nbsp;&times; 10(-7) study-specific significance threshold and six suggestively associated SNPs in a further 59,349 individuals. We identified and replicated a SNP at LSP1/TNNT3, a SNP at MTHFR-NPPB independent (r(2) = 0.33) of previous reports, and replicated SNPs at AGT and ATP2B1 reported previously. An analysis of combined discovery and follow-up data identified SNPs significantly associated with BP at p &lt; 8.56&nbsp;&times; 10(-7) at four further loci (NPR3, HFE, NOS3, and SOX6). The high number of discoveries made with modest genotyping effort can be attributed to using a large-scale yet targeted genotyping array and to the development of a weighting scheme that maximized power when meta-analyzing results from samples ascertained with extreme phenotypes, in combination with results from nonascertained or population samples. Chromatin immunoprecipitation and transcript expression data highlight potential gene regulatory mechanisms at the MTHFR and NOS3 loci. These results provide candidates for further study to help dissect mechanisms affecting BP and highlight the utility of studying SNPs and samples that are independent of those studied previously even when the sample size is smaller than that in previous studies.</p>
<p class="grants">Funded by: AHRQ HHS: <span class="grant">HS06516</span>; British Heart Foundation: <span class="grant">CH/98001</span>, <span class="grant">FS05/125</span>, <span class="grant">PG/07/131/24254</span>, <span class="grant">PG/07/132/24256</span>, <span class="grant">PG/07/133/24260</span>, <span class="grant">PG/97012</span>, <span class="grant">RG/07/005/23633</span>, <span class="grant">RG/07/008/23674</span>, <span class="grant">RG/08/008</span>, <span class="grant">RG/08/008/25291</span>, <span class="grant">RG/08/013/25942</span>, <span class="grant">RG/2001004</span>, <span class="grant">SP/07/007/2367</span>, <span class="grant">SP/08/005/25115</span>; Canadian Institutes of Health Research: <span class="grant">MOP172605</span>, <span class="grant">MOP77682</span>, <span class="grant">MOP82810</span>; Department of Health; Medical Research Council: <span class="grant">G0400874</span>, <span class="grant">G0401527</span>, <span class="grant">G0501942</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0802432</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U123092723</span>, <span class="grant">MC_U137686857</span>, <span class="grant">MC_UP_A100_1003</span>; NIA NIH HHS: <span class="grant">AG13196</span>, <span class="grant">R01 AG017644-09S1</span>; Wellcome Trust: <span class="grant">070191/Z/03/A</span>, <span class="grant">070191/Z/03/Z</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">090532</span>, <span class="grant">093078/Z/10/Z</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">89</span>;<span class="number">6</span>;<span class="pagerange">688-700</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22100073">22100073</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3234370">3234370</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2011.10.013">10.1016/j.ajhg.2011.10.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal rearrangements maintain a polymorphic supergene controlling butterfly mimicry.</h4>
<p class="authors"><span class="author">Joron M</span>, <span class="author">Frezal L</span>, <span class="author">Jones RT</span>, <span class="author">Chamberlain NL</span>, <span class="author">Lee SF</span>, <span class="author">Haag CR</span>, <span class="author">Whibley A</span>, <span class="author">Becuwe M</span>, <span class="author">Baxter SW</span>, <span class="author">Ferguson L</span>, <span class="author">Wilkinson PA</span>, <span class="author">Salazar C</span>, <span class="author">Davidson C</span>, <span class="author">Clark R</span>, <span class="author">Quail MA</span>, <span class="author">Beasley H</span>, <span class="author">Glithero R</span>, <span class="author">Lloyd C</span>, <span class="author">Sims S</span>, <span class="author">Jones MC</span>, <span class="author">Rogers J</span>, <span class="author">Jiggins CD</span> and <span class="author">ffrench-Constant RH</span></p>
<p class="affiliation">CNRS UMR 7205, Mus&eacute;um National d&#39;Histoire Naturelle, CP50, 45 Rue Buffon, 75005 Paris, France. <a href="mailto:%6A%6F%72%6F%6E%40%6D%6E%68%6E.%66%72">&#106;&#111;&#114;&#111;&#110;&#64;&#109;&#110;&#104;&#110;&#46;&#102;&#114;</a></p><p class="abstract">Supergenes are tight clusters of loci that facilitate the co-segregation of adaptive variation, providing integrated control of complex adaptive phenotypes. Polymorphic supergenes, in which specific combinations of traits are maintained within a single population, were first described for &#39;pin&#39; and &#39;thrum&#39; floral types in Primula and Fagopyrum, but classic examples are also found in insect mimicry and snail morphology. Understanding the evolutionary mechanisms that generate these co-adapted gene sets, as well as the mode of limiting the production of unfit recombinant forms, remains a substantial challenge. Here we show that individual wing-pattern morphs in the polymorphic mimetic butterfly Heliconius numata are associated with different genomic rearrangements at the supergene locus P. These rearrangements tighten the genetic linkage between at least two colour-pattern loci that are known to recombine in closely related species, with complete suppression of recombination being observed in experimental crosses across a 400-kilobase interval containing at least 18 genes. In natural populations, notable patterns of linkage disequilibrium (LD) are observed across the entire P region. The resulting divergent haplotype clades and inversion breakpoints are found in complete association with wing-pattern morphs. Our results indicate that allelic combinations at known wing-patterning loci have become locked together in a polymorphic rearrangement at the P locus, forming a supergene that acts as a simple switch between complex adaptive phenotypes found in sympatry. These findings highlight how genomic rearrangements can have a central role in the coexistence of adaptive phenotypes involving several genes acting in concert, by locally limiting recombination and gene flow.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBE0118451</span>; Medical Research Council: <span class="grant">G0900740</span>; Wellcome Trust: <span class="grant">079643</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7363</span>;<span class="pagerange">203-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21841803">21841803</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3717454">3717454</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10341">10.1038/nature10341</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic risk prediction in complex disease.</h4>
<p class="authors"><span class="author">Jostins L</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Statistical and Computational Genetics, Wellcome Trust Sanger Institute, Cambs, UK</p><p class="abstract">Attempting to classify patients into high or low risk for disease onset or outcomes is one of the cornerstones of epidemiology. For some (but by no means all) diseases, clinically usable risk prediction can be performed using classical risk factors such as body mass index, lipid levels, smoking status, family history and, under certain circumstances, genetics (e.g. BRCA1/2 in breast cancer). The advent of genome-wide association studies (GWAS) has led to the discovery of common risk loci for the majority of common diseases. These discoveries raise the possibility of using these variants for risk prediction in a clinical setting. We discuss the different ways in which the predictive accuracy of these loci can be measured, and survey the predictive accuracy of GWAS variants for 18 common diseases. We show that predictive accuracy from genetic models varies greatly across diseases, but that the range is similar to that of non-genetic risk-prediction models. We discuss what factors drive differences in predictive accuracy, and how much value these predictions add over classical predictive tests. We also review the uses and pitfalls of idealized models of risk prediction. Finally, we look forward towards possible future clinical implementation of genetic risk prediction, and discuss realistic expectations for future utility.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT089120/Z/09/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">R2</span>;<span class="pagerange">R182-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21873261">21873261</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3179379">3179379</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr378">10.1093/hmg/ddr378</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Imputation of low-frequency variants using the HapMap3 benefits from large, diverse reference sets.</h4>
<p class="authors"><span class="author">Jostins L</span>, <span class="author">Morley KI</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Statistical and Computational Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Imputation allows the inference of unobserved genotypes in low-density data sets, and is often used to test for disease association at variants that are poorly captured by standard genotyping chips (such as low-frequency variants). Although much effort has gone into developing the best imputation algorithms, less is known about the effects of reference set choice on imputation accuracy. We assess the improvements afforded by increases in reference size and diversity, specifically comparing the HapMap2 data set, which has been used to date for imputation, and the new HapMap3 data set, which contains more samples from a more diverse range of populations. We find that, for imputation into Western European samples, the HapMap3 reference provides more accurate imputation with better-calibrated quality scores than HapMap2, and that increasing the number of HapMap3 populations included in the reference set grant further improvements. Improvements are most pronounced for low-frequency variants (frequency &lt;5%), with the largest and most diverse reference sets bringing the accuracy of imputation of low-frequency variants close to that of common ones. For low-frequency variants, reference set diversity can improve the accuracy of imputation, independent of reference sample size. HapMap3 reference sets provide significant increases in imputation accuracy relative to HapMap2, and are of particular use if highly accurate imputation of low-frequency variants is required. Our results suggest that, although the sample sizes from the 1000 Genomes Pilot Project will not allow reliable imputation of low-frequency variants, the larger sample sizes of the main project will allow.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT089120/Z/09/Z</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">6</span>;<span class="pagerange">662-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21364697">21364697</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3110048">3110048</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2011.10">10.1038/ejhg.2011.10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate gene analysis of the human natural killer-1 carbohydrate pathway and perineuronal nets in schizophrenia: B3GAT2 is associated with disease risk and cortical surface area.</h4>
<p class="authors"><span class="author">K&auml;hler AK</span>, <span class="author">Djurovic S</span>, <span class="author">Rimol LM</span>, <span class="author">Brown AA</span>, <span class="author">Athanasiu L</span>, <span class="author">J&ouml;nsson EG</span>, <span class="author">Hansen T</span>, <span class="author">G&uacute;stafsson O</span>, <span class="author">Hall H</span>, <span class="author">Giegling I</span>, <span class="author">Muglia P</span>, <span class="author">Cichon S</span>, <span class="author">Rietschel M</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Peltonen L</span>, <span class="author">Bramon E</span>, <span class="author">Collier D</span>, <span class="author">St Clair D</span>, <span class="author">Sigurdsson E</span>, <span class="author">Petursson H</span>, <span class="author">Rujescu D</span>, <span class="author">Melle I</span>, <span class="author">Werge T</span>, <span class="author">Steen VM</span>, <span class="author">Dale AM</span>, <span class="author">Matthews RT</span>, <span class="author">Agartz I</span> and <span class="author">Andreassen OA</span></p>
<p class="affiliation">Institute of Psychiatry, University of Oslo, Oslo University Hospital-Ulleval, Norway. <a href="mailto:%61.%6B.%6B%61%68%6C%65%72%40%6D%65%64%69%73%69%6E.%75%69%6F.%6E%6F">&#97;&#46;&#107;&#46;&#107;&#97;&#104;&#108;&#101;&#114;&#64;&#109;&#101;&#100;&#105;&#115;&#105;&#110;&#46;&#117;&#105;&#111;&#46;&#110;&#111;</a></p><p class="abstract"><strong>Background</strong>: The Human Natural Killer-1 carbohydrate (HNK-1) is involved in neurodevelopment and synaptic plasticity. Extracellular matrix structures called perineuronal nets, condensed around subsets of neurons and proximal dendrites during brain maturation, regulate synaptic transmission and plasticity.</p><p class="abstract"><strong>Methods</strong>: Ten genes of importance for HNK-1 biosynthesis (B3GAT1, B3GAT2, and CHST10) or for the formation of perineuronal nets (TNR, BCAN, NCAN, HAPLN1, HAPLN2, HAPLN3, and HAPLN4) were investigated for potential involvement in schizophrenia (SCZ) susceptibility, by genotyping 104 tagSNPs in the Scandinavian Collaboration on Psychiatric Etiology sample (849 cases; 1602 control subjects). Genome-wide association study imputation data from the European SGENE-plus sample (2663 cases; 13,498 control subjects) were used for comparison. The effect of SCZ risk alleles on brain structure was investigated in a Norwegian subset (98 cases; 177 control subjects) with structural magnetic resonance imaging data.</p><p class="abstract"><strong>Results</strong>: Five single nucleotide polymorphisms (SNPs), located in two adjacent estimated linkage disequilibrium blocks in the first intron of &beta;-1,3-glucuronyltransferase 2 (B3GAT2), were nominally associated with SCZ (.004 &le; P(empirical) &le; .05). The rs2460691 was significantly associated in the comparison sample and in the meta-analysis after correction for all 121 SNP/haplotype tests (P(raw) = 1 &times; 10(-4); P(corrected) = .018). Increased dosage of the rs2460691 SCZ risk allele was associated with decreased cortical area (p = .002) but not thickness or hippocampal volume. A second SNP (r(2) = .24 with rs10945275), which conferred the highest SCZ risk effect in the Norwegian subset, was also associated with cortical area.</p><p class="abstract"><strong>Conclusions</strong>: The present results suggest that effects on biosynthesis of the neuronal epitope HNK-1, through common B3GAT2 variation, could increase the risk of SCZ, possibly by decreasing cortical area.</p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">69</span>;<span class="number">1</span>;<span class="pagerange">90-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20950796">20950796</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2010.07.035">10.1016/j.biopsych.2010.07.035</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CEP152 is a genome maintenance protein disrupted in Seckel syndrome.</h4>
<p class="authors"><span class="author">Kalay E</span>, <span class="author">Yigit G</span>, <span class="author">Aslan Y</span>, <span class="author">Brown KE</span>, <span class="author">Pohl E</span>, <span class="author">Bicknell LS</span>, <span class="author">Kayserili H</span>, <span class="author">Li Y</span>, <span class="author">T&uuml;ys&uuml;z B</span>, <span class="author">N&uuml;rnberg G</span>, <span class="author">Kiess W</span>, <span class="author">Koegl M</span>, <span class="author">Baessmann I</span>, <span class="author">Buruk K</span>, <span class="author">Toraman B</span>, <span class="author">Kayipmaz S</span>, <span class="author">Kul S</span>, <span class="author">Ikbal M</span>, <span class="author">Turner DJ</span>, <span class="author">Taylor MS</span>, <span class="author">Aerts J</span>, <span class="author">Scott C</span>, <span class="author">Milstein K</span>, <span class="author">Dollfus H</span>, <span class="author">Wieczorek D</span>, <span class="author">Brunner HG</span>, <span class="author">Hurles M</span>, <span class="author">Jackson AP</span>, <span class="author">Rauch A</span>, <span class="author">N&uuml;rnberg P</span>, <span class="author">Karag&uuml;zel A</span> and <span class="author">Wollnik B</span></p>
<p class="affiliation">Department of Medical Biology, Faculty of Medicine, Karadeniz Technical University, Trabzon, Turkey. <a href="mailto:%65%72%73%61%6E%6B%61%6C%61%79%40%68%6F%74%6D%61%69%6C.%63%6F%6D">&#101;&#114;&#115;&#97;&#110;&#107;&#97;&#108;&#97;&#121;&#64;&#104;&#111;&#116;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Functional impairment of DNA damage response pathways leads to increased genomic instability. Here we describe the centrosomal protein CEP152 as a new regulator of genomic integrity and cellular response to DNA damage. Using homozygosity mapping and exome sequencing, we identified CEP152 mutations in Seckel syndrome and showed that impaired CEP152 function leads to accumulation of genomic defects resulting from replicative stress through enhanced activation of ATM signaling and increased H2AX phosphorylation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U120081295</span>, <span class="grant">MC_U127580972</span>, <span class="grant">MC_U127597124</span>; Wellcome Trust: <span class="grant">077014</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">1</span>;<span class="pagerange">23-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21131973">21131973</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3430850">3430850</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.725">10.1038/ng.725</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Total zinc intake may modify the glucose-raising effect of a zinc transporter (SLC30A8) variant: a 14-cohort meta-analysis.</h4>
<p class="authors"><span class="author">Kanoni S</span>, <span class="author">Nettleton JA</span>, <span class="author">Hivert MF</span>, <span class="author">Ye Z</span>, <span class="author">van Rooij FJ</span>, <span class="author">Shungin D</span>, <span class="author">Sonestedt E</span>, <span class="author">Ngwa JS</span>, <span class="author">Wojczynski MK</span>, <span class="author">Lemaitre RN</span>, <span class="author">Gustafsson S</span>, <span class="author">Anderson JS</span>, <span class="author">Tanaka T</span>, <span class="author">Hindy G</span>, <span class="author">Saylor G</span>, <span class="author">Renstrom F</span>, <span class="author">Bennett AJ</span>, <span class="author">van Duijn CM</span>, <span class="author">Florez JC</span>, <span class="author">Fox CS</span>, <span class="author">Hofman A</span>, <span class="author">Hoogeveen RC</span>, <span class="author">Houston DK</span>, <span class="author">Hu FB</span>, <span class="author">Jacques PF</span>, <span class="author">Johansson I</span>, <span class="author">Lind L</span>, <span class="author">Liu Y</span>, <span class="author">McKeown N</span>, <span class="author">Ordovas J</span>, <span class="author">Pankow JS</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Yannakoulia M</span>, <span class="author">Zillikens MC</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Wareham NJ</span>, <span class="author">Prokopenko I</span>, <span class="author">Bandinelli S</span>, <span class="author">Forouhi NG</span>, <span class="author">Cupples LA</span>, <span class="author">Loos RJ</span>, <span class="author">Hallmans G</span>, <span class="author">Dupuis J</span>, <span class="author">Langenberg C</span>, <span class="author">Ferrucci L</span>, <span class="author">Kritchevsky SB</span>, <span class="author">McCarthy MI</span>, <span class="author">Ingelsson E</span>, <span class="author">Borecki IB</span>, <span class="author">Witteman JC</span>, <span class="author">Orho-Melander M</span>, <span class="author">Siscovick DS</span>, <span class="author">Meigs JB</span>, <span class="author">Franks PW</span> and <span class="author">Dedoussis GV</span></p>
<p class="affiliation">Department of Nutrition-Dietetics, Harokopio University, Athens, Greece. <a href="mailto:%73%74%61%76%72%6F%75%6C%61.%6B%61%6E%6F%6E%69%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#116;&#97;&#118;&#114;&#111;&#117;&#108;&#97;&#46;&#107;&#97;&#110;&#111;&#110;&#105;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Objective</strong>: Many genetic variants have been associated with glucose homeostasis and type 2 diabetes in genome-wide association studies. Zinc is an essential micronutrient that is important for &beta;-cell function and glucose homeostasis. We tested the hypothesis that zinc intake could influence the glucose-raising effect of specific variants.</p><p class="abstract">We conducted a 14-cohort meta-analysis to assess the interaction of 20 genetic variants known to be related to glycemic traits and zinc metabolism with dietary zinc intake (food sources) and a 5-cohort meta-analysis to assess the interaction with total zinc intake (food sources and supplements) on fasting glucose levels among individuals of European ancestry without diabetes.</p><p class="abstract"><strong>Results</strong>: We observed a significant association of total zinc intake with lower fasting glucose levels (&beta;-coefficient &plusmn; SE per 1 mg/day of zinc intake: -0.0012 &plusmn; 0.0003 mmol/L, summary P value = 0.0003), while the association of dietary zinc intake was not significant. We identified a nominally significant interaction between total zinc intake and the SLC30A8 rs11558471 variant on fasting glucose levels (&beta;-coefficient &plusmn; SE per A allele for 1 mg/day of greater total zinc intake: -0.0017 &plusmn; 0.0006 mmol/L, summary interaction P value = 0.005); this result suggests a stronger inverse association between total zinc intake and fasting glucose in individuals carrying the glucose-raising A allele compared with individuals who do not carry it. None of the other interaction tests were statistically significant.</p><p class="abstract"><strong>Conclusions</strong>: Our results suggest that higher total zinc intake may attenuate the glucose-raising effect of the rs11558471 SLC30A8 (zinc transporter) variant. Our findings also support evidence for the association of higher total zinc intake with lower fasting glucose levels.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701863</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_UP_A100_1003</span>; NIGMS NIH HHS: <span class="grant">T32 GM074905</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">9</span>;<span class="pagerange">2407-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21810599">21810599</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3161318">3161318</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb11-0176">10.2337/db11-0176</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Salmonella Typhi sense host neuroendocrine stress hormones and release the toxin haemolysin E.</h4>
<p class="authors"><span class="author">Karavolos MH</span>, <span class="author">Bulmer DM</span>, <span class="author">Spencer H</span>, <span class="author">Rampioni G</span>, <span class="author">Schmalen I</span>, <span class="author">Baker S</span>, <span class="author">Pickard D</span>, <span class="author">Gray J</span>, <span class="author">Fookes M</span>, <span class="author">Winzer K</span>, <span class="author">Ivens A</span>, <span class="author">Dougan G</span>, <span class="author">Williams P</span> and <span class="author">Khan CM</span></p>
<p class="affiliation">Institute for Cell and Molecular Biosciences, The Medical School, Newcastle University, Newcastle NE2 4HH, UK.</p><p class="abstract">Salmonella enterica serovar Typhi (S. typhi) causes typhoid fever. We show that exposure of S. typhi to neuroendocrine stress hormones results in haemolysis, which is associated with the release of haemolysin E in membrane vesicles. This effect is attributed to increased expression of the small RNA micA and RNA chaperone Hfq, with concomitant downregulation of outer membrane protein A. Deletion of micA or the two-component signal-transduction system, CpxAR, abolishes the phenotype. The hormone response is inhibited by the &beta;-blocker propranolol. We provide mechanistic insights into the basis of neuroendocrine hormone-mediated haemolysis by S. typhi, increasing our understanding of inter-kingdom signalling.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">EMBO reports</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">3</span>;<span class="pagerange">252-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21331094" rel="external">21331094</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3059909" rel="external">3059909</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fembor.2011.4" rel="external">10.1038/embor.2011.4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In vivo identification of tumor- suppressive PTEN ceRNAs in an oncogenic BRAF-induced mouse model of melanoma.</h4>
<p class="authors"><span class="author">Karreth FA</span>, <span class="author">Tay Y</span>, <span class="author">Perna D</span>, <span class="author">Ala U</span>, <span class="author">Tan SM</span>, <span class="author">Rust AG</span>, <span class="author">DeNicola G</span>, <span class="author">Webster KA</span>, <span class="author">Weiss D</span>, <span class="author">Perez-Mancera PA</span>, <span class="author">Krauthammer M</span>, <span class="author">Halaban R</span>, <span class="author">Provero P</span>, <span class="author">Adams DJ</span>, <span class="author">Tuveson DA</span> and <span class="author">Pandolfi PP</span></p>
<p class="affiliation">Cancer Genetics Program, Division of Genetics, Beth Israel Deaconess Cancer Center, Department of Medicine and Pathology, Harvard Medical School, Boston, MA 02215, USA.</p><p class="abstract">We recently proposed that competitive endogenous RNAs (ceRNAs) sequester microRNAs to regulate mRNA transcripts containing common microRNA recognition elements (MREs). However, the functional role of ceRNAs in cancer remains unknown. Loss of PTEN, a tumor suppressor regulated by ceRNA activity, frequently occurs in melanoma. Here, we report the discovery of significant enrichment of putative PTEN ceRNAs among genes whose loss accelerates tumorigenesis following Sleeping Beauty insertional mutagenesis in a mouse model of melanoma. We validated several putative PTEN ceRNAs and further characterized one, the ZEB2 transcript. We show that ZEB2 modulates PTEN protein levels in a microRNA-dependent, protein coding-independent manner. Attenuation of ZEB2 expression activates the PI3K/AKT pathway, enhances cell transformation, and commonly occurs in human melanomas and other cancers expressing low PTEN levels. Our study genetically identifies multiple putative microRNA decoys for PTEN, validates ZEB2 mRNA as a bona fide PTEN ceRNA, and demonstrates that abrogated ZEB2 expression cooperates with BRAF(V600E) to promote melanomagenesis.</p>
<p class="grants">Funded by: Cancer Research UK; NCI NIH HHS: <span class="grant">1P50 CA121974</span>, <span class="grant">P50 CA121974</span>, <span class="grant">P50 CA121974-01</span>, <span class="grant">R01 CA-82328-09</span>, <span class="grant">R01 CA082328</span>, <span class="grant">R01 CA082328-09</span>; NCRR NIH HHS: <span class="grant">UL1 RR025758</span>, <span class="grant">UL1 RR025758-04</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">147</span>;<span class="number">2</span>;<span class="pagerange">382-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22000016">22000016</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3236086">3236086</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2011.09.032">10.1016/j.cell.2011.09.032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulation of bone mass by serotonin: molecular biology and therapeutic implications.</h4>
<p class="authors"><span class="author">Karsenty G</span> and <span class="author">Yadav VK</span></p>
<p class="affiliation">Department of Genetics &amp; Development, Columbia University Medical Center, New York, New York 10032, USA. <a href="mailto:%6A%70%62%32%40%63%6F%6C%75%6D%62%69%61.%65%64%75">&#106;&#112;&#98;&#50;&#64;&#99;&#111;&#108;&#117;&#109;&#98;&#105;&#97;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The molecular elucidation of two human skeletal dysplasias revealed that they are caused by an increase or a decrease in the synthesis of serotonin by enterochromaffin cells of the gut. This observation revealed a novel and powerful endocrine means to regulate bone mass. Exploiting these findings in the pharmacological arena led to the demonstration that inhibiting synthesis of gut-derived serotonin could be an effective means to treat low-bone-mass diseases such as osteoporosis.</p>
<p class="publication"><span class="title">Annual review of medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">62</span>;<span class="pagerange">323-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21073335" rel="external">21073335</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev-med-090710-133426" rel="external">10.1146/annurev-med-090710-133426</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phylogenetic analysis of murine leukemia virus sequences from longitudinally sampled chronic fatigue syndrome patients suggests PCR contamination rather than viral evolution.</h4>
<p class="authors"><span class="author">Katzourakis A</span>, <span class="author">Hu&eacute; S</span>, <span class="author">Kellam P</span> and <span class="author">Towers GJ</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, South Parks Road, Oxford OX13PS, United Kingdom.</p><p class="abstract">Xenotropic murine leukemia virus (MLV)-related virus (XMRV) has been amplified from human prostate cancer and chronic fatigue syndrome (CFS) patient samples. Other studies failed to replicate these findings and suggested PCR contamination with a prostate cancer cell line, 22Rv1, as a likely source. MLV-like sequences have also been detected in CFS patients in longitudinal samples 15 years apart. Here, we tested whether sequence data from these samples are consistent with viral evolution. Our phylogenetic analyses strongly reject a model of within-patient evolution and demonstrate that the sequences from the first and second time points represent distinct endogenous murine retroviruses, suggesting contamination.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801172</span>, <span class="grant">G9721629</span>; Wellcome Trust: <span class="grant">090940</span>, <span class="grant">WT090940</span></p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2011</span>;<span class="volume">85</span>;<span class="number">20</span>;<span class="pagerange">10909-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21849443">21849443</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3187511">3187511</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.00827-11">10.1128/JVI.00827-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mouse genomic variation and its effect on phenotypes and gene regulation.</h4>
<p class="authors"><span class="author">Keane TM</span>, <span class="author">Goodstadt L</span>, <span class="author">Danecek P</span>, <span class="author">White MA</span>, <span class="author">Wong K</span>, <span class="author">Yalcin B</span>, <span class="author">Heger A</span>, <span class="author">Agam A</span>, <span class="author">Slater G</span>, <span class="author">Goodson M</span>, <span class="author">Furlotte NA</span>, <span class="author">Eskin E</span>, <span class="author">Nell&aring;ker C</span>, <span class="author">Whitley H</span>, <span class="author">Cleak J</span>, <span class="author">Janowitz D</span>, <span class="author">Hernandez-Pliego P</span>, <span class="author">Edwards A</span>, <span class="author">Belgard TG</span>, <span class="author">Oliver PL</span>, <span class="author">McIntyre RE</span>, <span class="author">Bhomra A</span>, <span class="author">Nicod J</span>, <span class="author">Gan X</span>, <span class="author">Yuan W</span>, <span class="author">van der Weyden L</span>, <span class="author">Steward CA</span>, <span class="author">Bala S</span>, <span class="author">Stalker J</span>, <span class="author">Mott R</span>, <span class="author">Durbin R</span>, <span class="author">Jackson IJ</span>, <span class="author">Czechanski A</span>, <span class="author">Guerra-Assun&ccedil;&atilde;o JA</span>, <span class="author">Donahue LR</span>, <span class="author">Reinholdt LG</span>, <span class="author">Payseur BA</span>, <span class="author">Ponting CP</span>, <span class="author">Birney E</span>, <span class="author">Flint J</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">We report genome sequences of 17 inbred strains of laboratory mice and identify almost ten times more variants than previously known. We use these genomes to explore the phylogenetic history of the laboratory mouse and to examine the functional consequences of allele-specific variation on transcript abundance, revealing that at least 12% of transcripts show a significant tissue-specific expression bias. By identifying candidate functional variants at 718 quantitative trait loci we show that the molecular nature of functional variants and their position relative to genes vary according to the effect size of the locus. These sequences provide a starting point for a new era in the functional analysis of a key model organism.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F022697/1</span>; Cancer Research UK: <span class="grant">A6997</span>; Medical Research Council: <span class="grant">G0800024</span>, <span class="grant">MC_U127561112</span>, <span class="grant">MC_U137761446</span>; NHLBI NIH HHS: <span class="grant">K25 HL080079</span>; NLM NIH HHS: <span class="grant">2T15LM007359</span>; Wellcome Trust: <span class="grant">077192</span>, <span class="grant">079912</span>, <span class="grant">082356</span>, <span class="grant">083573</span>, <span class="grant">083573/Z/07/Z</span>, <span class="grant">085906</span>, <span class="grant">085906/Z/08/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7364</span>;<span class="pagerange">289-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21921910">21921910</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3276836">3276836</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10413">10.1038/nature10413</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional analysis of conserved non-coding regions around the short stature hox gene (shox) in whole zebrafish embryos.</h4>
<p class="authors"><span class="author">Kenyon EJ</span>, <span class="author">McEwen GK</span>, <span class="author">Callaway H</span> and <span class="author">Elgar G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, Cambridgeshire, United Kingdom. <a href="mailto:%65%6B%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#107;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Mutations in the SHOX gene are responsible for Leri-Weill Dyschondrosteosis, a disorder characterised by mesomelic limb shortening. Recent investigations into regulatory elements surrounding SHOX have shown that deletions of conserved non-coding elements (CNEs) downstream of the SHOX gene produce a phenotype indistinguishable from Leri-Weill Dyschondrosteosis. As this gene is not found in rodents, we used zebrafish as a model to characterise the expression pattern of the shox gene across the whole embryo and characterise the enhancer domains of different CNEs associated with this gene.</p><p class="abstract">Expression of the shox gene in zebrafish was identified using in situ hybridization, with embryos showing expression in the blood, putative heart, hatching gland, brain pharyngeal arch, olfactory epithelium, and fin bud apical ectodermal ridge. By identifying sequences showing 65% identity over at least 40 nucleotides between Fugu, human, dog and opossum we uncovered 35 CNEs around the shox gene. These CNEs were compared with CNEs previously discovered by Sabherwal et al., resulting in the identification of smaller more deeply conserved sub-sequence. Sabherwal et al.&#39;s CNEs were assayed for regulatory function in whole zebrafish embryos resulting in the identification of additional tissues under the regulatory control of these CNEs.</p><p class="abstract">Our results using whole zebrafish embryos have provided a more comprehensive picture of the expression pattern of the shox gene, and a better understanding of its regulation via deeply conserved noncoding elements. In particular, we identify additional tissues under the regulatory control of previously identified SHOX CNEs. We also demonstrate the importance of these CNEs in evolution by identifying duplicated shox CNEs and more deeply conserved sub-sequences within already identified CNEs.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0401138</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">e21498</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21731768" rel="external">21731768</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3123344" rel="external">3123344</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0021498" rel="external">10.1371/journal.pone.0021498</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput target-selected gene inactivation in zebrafish.</h4>
<p class="authors"><span class="author">Kettleborough RN</span>, <span class="author">Bruijn Ed</span>, <span class="author">Eeden Fv</span>, <span class="author">Cuppen E</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p>
<p class="abstract">There is an increasing requirement for efficient reverse genetics in the zebrafish, Here we describe a method that takes advantage of conventional mutagenized libraries (identical to ones used in forward screens) and re-sequencing to identify ENU-induced mutations in genes of interest. The efficiency of TILLING (Targeting Induced Local Legions IN Genomes) depends on the rate of mutagenesis in the library being screened, the amount of base pairs screened, and the ability to effectively identify and retrieve mutations on interest. Here we show that by improving the mutagenesis protocol, using in silico methods to predict codon changes for target selection, efficient PCR and re-sequencing, and accurate mutation detection we can vastly improve current TILLING protocols. Importantly it is also possible to use this method for screening for splice and mis-sense mutations, and with even a relatively small library, there is a high chance of identifying mutations across any given gene.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">104</span>;<span class="pagerange">121-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21924159">21924159</a>; DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-374814-0.00006-9">10.1016/B978-0-12-374814-0.00006-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic insights into the origin of parasitism in the emerging plant pathogen Bursaphelenchus xylophilus.</h4>
<p class="authors"><span class="author">Kikuchi T</span>, <span class="author">Cotton JA</span>, <span class="author">Dalzell JJ</span>, <span class="author">Hasegawa K</span>, <span class="author">Kanzaki N</span>, <span class="author">McVeigh P</span>, <span class="author">Takanashi T</span>, <span class="author">Tsai IJ</span>, <span class="author">Assefa SA</span>, <span class="author">Cock PJ</span>, <span class="author">Otto TD</span>, <span class="author">Hunt M</span>, <span class="author">Reid AJ</span>, <span class="author">Sanchez-Flores A</span>, <span class="author">Tsuchihara K</span>, <span class="author">Yokoi T</span>, <span class="author">Larsson MC</span>, <span class="author">Miwa J</span>, <span class="author">Maule AG</span>, <span class="author">Sahashi N</span>, <span class="author">Jones JT</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Forestry and Forest Products Research Institute, Tsukuba, Japan. <a href="mailto:%6B%69%6B%75%63%68%69%74%40%61%66%66%72%63.%67%6F.%6A%70">&#107;&#105;&#107;&#117;&#99;&#104;&#105;&#116;&#64;&#97;&#102;&#102;&#114;&#99;&#46;&#103;&#111;&#46;&#106;&#112;</a></p><p class="abstract">Bursaphelenchus xylophilus is the nematode responsible for a devastating epidemic of pine wilt disease in Asia and Europe, and represents a recent, independent origin of plant parasitism in nematodes, ecologically and taxonomically distinct from other nematodes for which genomic data is available. As well as being an important pathogen, the B. xylophilus genome thus provides a unique opportunity to study the evolution and mechanism of plant parasitism. Here, we present a high-quality draft genome sequence from an inbred line of B. xylophilus, and use this to investigate the biological basis of its complex ecology which combines fungal feeding, plant parasitic and insect-associated stages. We focus particularly on putative parasitism genes as well as those linked to other key biological processes and demonstrate that B. xylophilus is well endowed with RNA interference effectors, peptidergic neurotransmitters (including the first description of ins genes in a parasite) stress response and developmental genes and has a contracted set of chemosensory receptors. B. xylophilus has the largest number of digestive proteases known for any nematode and displays expanded families of lysosome pathway genes, ABC transporters and cytochrome P450 pathway genes. This expansion in digestive and detoxification proteins may reflect the unusual diversity in foods it exploits and environments it encounters during its life cycle. In addition, B. xylophilus possesses a unique complement of plant cell wall modifying proteins acquired by horizontal gene transfer, underscoring the impact of this process on the evolution of plant parasitism by nematodes. Together with the lack of proteins homologous to effectors from other plant parasitic nematodes, this confirms the distinctive molecular basis of plant parasitism in the Bursaphelenchus lineage. The genome sequence of B. xylophilus adds to the diversity of genomic data for nematodes, and will be an important resource in understanding the biology of this unusual parasite.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e1002219</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21909270">21909270</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3164644">3164644</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002219">10.1371/journal.ppat.1002219</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variation near IRS1 associates with reduced adiposity and an impaired metabolic profile.</h4>
<p class="authors"><span class="author">Kilpel&auml;inen TO</span>, <span class="author">Zillikens MC</span>, <span class="author">Stan&#x10D;&aacute;kova A</span>, <span class="author">Finucane FM</span>, <span class="author">Ried JS</span>, <span class="author">Langenberg C</span>, <span class="author">Zhang W</span>, <span class="author">Beckmann JS</span>, <span class="author">Luan J</span>, <span class="author">Vandenput L</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Zhou Y</span>, <span class="author">Smith AV</span>, <span class="author">Zhao JH</span>, <span class="author">Amin N</span>, <span class="author">Vedantam S</span>, <span class="author">Shin SY</span>, <span class="author">Haritunians T</span>, <span class="author">Fu M</span>, <span class="author">Feitosa MF</span>, <span class="author">Kumari M</span>, <span class="author">Halldorsson BV</span>, <span class="author">Tikkanen E</span>, <span class="author">Mangino M</span>, <span class="author">Hayward C</span>, <span class="author">Song C</span>, <span class="author">Arnold AM</span>, <span class="author">Aulchenko YS</span>, <span class="author">Oostra BA</span>, <span class="author">Campbell H</span>, <span class="author">Cupples LA</span>, <span class="author">Davis KE</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Estrada K</span>, <span class="author">Fern&aacute;ndez-Real JM</span>, <span class="author">Garcia M</span>, <span class="author">Gieger C</span>, <span class="author">Glazer NL</span>, <span class="author">Guiducci C</span>, <span class="author">Hofman A</span>, <span class="author">Humphries SE</span>, <span class="author">Isomaa B</span>, <span class="author">Jacobs LC</span>, <span class="author">Jula A</span>, <span class="author">Karasik D</span>, <span class="author">Karlsson MK</span>, <span class="author">Khaw KT</span>, <span class="author">Kim LJ</span>, <span class="author">Kivim&auml;ki M</span>, <span class="author">Klopp N</span>, <span class="author">K&uuml;hnel B</span>, <span class="author">Kuusisto J</span>, <span class="author">Liu Y</span>, <span class="author">Ljunggren O</span>, <span class="author">Lorentzon M</span>, <span class="author">Luben RN</span>, <span class="author">McKnight B</span>, <span class="author">Mellstr&ouml;m D</span>, <span class="author">Mitchell BD</span>, <span class="author">Mooser V</span>, <span class="author">Moreno JM</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Pascoe L</span>, <span class="author">Peltonen L</span>, <span class="author">Peral B</span>, <span class="author">Perola M</span>, <span class="author">Psaty BM</span>, <span class="author">Salomaa V</span>, <span class="author">Savage DB</span>, <span class="author">Semple RK</span>, <span class="author">Skaric-Juric T</span>, <span class="author">Sigurdsson G</span>, <span class="author">Song KS</span>, <span class="author">Spector TD</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Talmud PJ</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">Vidal-Puig A</span>, <span class="author">Wild SH</span>, <span class="author">Wright AF</span>, <span class="author">Clegg DJ</span>, <span class="author">Schadt E</span>, <span class="author">Wilson JF</span>, <span class="author">Rudan I</span>, <span class="author">Ripatti S</span>, <span class="author">Borecki IB</span>, <span class="author">Shuldiner AR</span>, <span class="author">Ingelsson E</span>, <span class="author">Jansson JO</span>, <span class="author">Kaplan RC</span>, <span class="author">Gudnason V</span>, <span class="author">Harris TB</span>, <span class="author">Groop L</span>, <span class="author">Kiel DP</span>, <span class="author">Rivadeneira F</span>, <span class="author">Walker M</span>, <span class="author">Barroso I</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Chambers JC</span>, <span class="author">Kooner JS</span>, <span class="author">Soranzo N</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Stefansson K</span>, <span class="author">Wichmann HE</span>, <span class="author">Ohlsson C</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Wareham NJ</span>, <span class="author">Speliotes EK</span>, <span class="author">Fox CS</span>, <span class="author">Laakso M</span> and <span class="author">Loos RJ</span></p>
<p class="affiliation">Medical Research Council (MRC) Epidemiology Unit, Institute of Metabolic Science, Cambridge, UK.</p><p class="abstract">Genome-wide association studies have identified 32 loci influencing body mass index, but this measure does not distinguish lean from fat mass. To identify adiposity loci, we meta-analyzed associations between &sim;2.5 million SNPs and body fat percentage from 36,626 individuals and followed up the 14 most significant (P &lt; 10(-6)) independent loci in 39,576 individuals. We confirmed a previously established adiposity locus in FTO (P = 3 &times; 10(-26)) and identified two new loci associated with body fat percentage, one near IRS1 (P = 4 &times; 10(-11)) and one near SPRY2 (P = 3 &times; 10(-8)). Both loci contain genes with potential links to adipocyte physiology. Notably, the body-fat-decreasing allele near IRS1 is associated with decreased IRS1 expression and with an impaired metabolic profile, including an increased visceral to subcutaneous fat ratio, insulin resistance, dyslipidemia, risk of diabetes and coronary artery disease and decreased adiponectin levels. Our findings provide new insights into adiposity and insulin resistance.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">PG/07/133/24260</span>, <span class="grant">RG/07/008/23674</span>, <span class="grant">RG/08/008</span>, <span class="grant">RG/08/008/25291</span>, <span class="grant">SP/04/002</span>, <span class="grant">SP/07/007/23671</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Department of Health; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0802051</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>; NCRR NIH HHS: <span class="grant">M01 RR 16500</span>, <span class="grant">M01 RR000425-36</span>, <span class="grant">M01 RR016500-04</span>, <span class="grant">M01-RR00425</span>; NHLBI NIH HHS: <span class="grant">N01 HC015103</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC045133</span>, <span class="grant">N01 HC055222</span>, <span class="grant">N01 HC075150</span>, <span class="grant">N01 HC085079</span>, <span class="grant">N01 HC085086</span>, <span class="grant">N01-HC15103</span>, <span class="grant">N01-HC25195</span>, <span class="grant">N01-HC35129</span>, <span class="grant">N01-HC45133</span>, <span class="grant">N01-HC55222</span>, <span class="grant">N01-HC75150</span>, <span class="grant">N01-HC85079-86</span>, <span class="grant">N01HC25195</span>, <span class="grant">N02 HL64278</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL087652-03</span>, <span class="grant">R01 HL087700</span>, <span class="grant">R01 HL087700-03</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL088119-04</span>, <span class="grant">R01-HL036310-20A2</span>, <span class="grant">R01-HL087652</span>, <span class="grant">R01-HL08770003</span>, <span class="grant">R01-HL088119</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL080295</span>, <span class="grant">U01 HL080295-04</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U01 HL084756-03</span>, <span class="grant">U01-HL080295</span>, <span class="grant">U01-HL72515</span>, <span class="grant">U01-HL84756</span>; NIA NIH HHS: <span class="grant">AG13196</span>, <span class="grant">N01-AG12100</span>, <span class="grant">N01-AG62101</span>, <span class="grant">N01-AG62103</span>, <span class="grant">N01-AG62106</span>, <span class="grant">N01AG12100</span>, <span class="grant">N1AG62101A</span>, <span class="grant">N1AG62103A</span>, <span class="grant">N1AG62106A</span>, <span class="grant">R01 AG018728</span>, <span class="grant">R01 AG018728-05S1</span>, <span class="grant">R01 AG032098</span>, <span class="grant">R01 AG032098-01A1</span>, <span class="grant">R01-AG031890-01</span>, <span class="grant">R01-AG032098-01A1</span>, <span class="grant">R01-AG18728</span>, <span class="grant">R01-AR/AG41398</span>; NIAMS NIH HHS: <span class="grant">R01 AR041398</span>, <span class="grant">R01 AR041398-19</span>, <span class="grant">R01 AR046838</span>, <span class="grant">R01 AR046838-05</span>, <span class="grant">R01-AR046838</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">K23 DK080145</span>, <span class="grant">K23 DK080145-05</span>, <span class="grant">K23-DK080145</span>, <span class="grant">P30 DK063491-03</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30 DK072488-04S1</span>, <span class="grant">P30-DK072488</span>, <span class="grant">R01 DK068336</span>, <span class="grant">R01 DK068336-03</span>, <span class="grant">R01 DK075681</span>, <span class="grant">R01 DK075681-04</span>, <span class="grant">R01 DK075787</span>, <span class="grant">R01 DK075787-05</span>, <span class="grant">R01 DK089256</span>, <span class="grant">R01-DK06833603</span>, <span class="grant">R01-DK07568102</span>, <span class="grant">R01-DK075787</span>; Wellcome Trust: <span class="grant">077016/Z/05/Z</span>, <span class="grant">084723/Z/08/Z</span>, <span class="grant">091551</span>, <span class="grant">091746/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">8</span>;<span class="pagerange">753-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21706003">21706003</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3262230">3262230</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.866">10.1038/ng.866</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conventional and Mendelian randomization analyses suggest no association between lipoprotein(a) and early atherosclerosis: the Young Finns Study.</h4>
<p class="authors"><span class="author">Kivim&auml;ki M</span>, <span class="author">Magnussen CG</span>, <span class="author">Juonala M</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kettunen J</span>, <span class="author">Loo BM</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Viikari J</span> and <span class="author">Raitakari OT</span></p>
<p class="affiliation">Department of Epidemiology, University College London, London, UK. <a href="mailto:%6D.%6B%69%76%69%6D%61%6B%69%40%75%63%6C.%61%63.%75%6B">&#109;&#46;&#107;&#105;&#118;&#105;&#109;&#97;&#107;&#105;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Lipoprotein(a) [Lp(a)] is an established risk factor for coronary disease and stroke, but mechanisms underlying this association are unknown. We examined the association of Lp(a) with early atherosclerosis by using conventional epidemiologic analysis and a Mendelian randomization analysis. The latter utilized genetic variants that are associated with Lp(a) to estimate causal effect.</p><p class="abstract"><strong>Methods</strong>: A prospective population-based cohort study of 939 men and 1141 women was conducted. Lp(a) was measured repeatedly at mean ages 17 and 38 years. Measurements of carotid intima-media thickness (IMT) and brachial flow-mediated dilation (FMD) at mean ages 32 and 38 years were used to determine the level and 6-year progression of subclinical atherosclerosis. Lp(a)-related genetic variant, rs783147, was identified by a genome wide association analysis (P = 3.1 &times; 10&#x207B;&#x2075;&#x2078;), and a genetic score was constructed based on 10 Lp(a)-related variants. Mendelian randomization test was performed using a two-stage instrumental variables analysis.</p><p class="abstract"><strong>Results</strong>: rs783147 and the genetic score were strong instruments for nonconfounded Lp(a) levels (F-statistics 269.6 and 446.0 in the first-stage instrumental variable analysis). However, Lp(a) levels were not associated with the levels of or change in IMT or FMD in any of the conventional and instrumental variables tests. The null finding was observed both with rs783147 and the genetic score as instruments and remained unchanged after adjustment for clinical characteristics, such as age, sex, HDL and LDL cholesterol, ApoB, systolic and diastolic blood pressure, diabetes and smoking.</p><p class="abstract"><strong>Conclusions</strong>: Data from conventional and Mendelian randomization analyses provide no support for early atherogenic effects of increased Lp(a) levels.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">AG034454</span></p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">40</span>;<span class="number">2</span>;<span class="pagerange">470-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21078622" rel="external">21078622</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3106966" rel="external">3106966</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdyq205" rel="external">10.1093/ije/dyq205</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Runx1-Smad6 rheostat controls Runx1 activity during embryonic hematopoiesis.</h4>
<p class="authors"><span class="author">Knezevic K</span>, <span class="author">Bee T</span>, <span class="author">Wilson NK</span>, <span class="author">Janes ME</span>, <span class="author">Kinston S</span>, <span class="author">Polderdijk S</span>, <span class="author">Kolb-Kokocinski A</span>, <span class="author">Ottersbach K</span>, <span class="author">Pencovich N</span>, <span class="author">Groner Y</span>, <span class="author">de Bruijn M</span>, <span class="author">G&ouml;ttgens B</span> and <span class="author">Pimanda JE</span></p>
<p class="affiliation">Lowy Cancer Research Centre and the Prince of Wales Clinical School, University of New South Wales, Sydney, New South Wales 2052, Australia.</p><p class="abstract">The oncogenic transcription factor Runx1 is required for the specification of definitive hematopoietic stem cells (HSC) in the developing embryo. The activity of this master regulator is tightly controlled during development. The transcription factors that upregulate the expression of Runx1 also upregulate the expression of Smad6, the inhibitory Smad, which controls Runx1 activity by targeting it to the proteasome. Here we show that Runx1, in conjunction with Fli1, Gata2, and Scl, directly regulates the expression of Smad6 in the aorta-gonad-mesonephros (AGM) region in the developing embryo, where HSCs originate. Runx1 regulates Smad6 activity via a novel upstream enhancer, and Runx1 null embryos show reduced Smad6 transcripts in the yolk-sac and c-Kit-positive fetal liver cells. By directly regulating the expression of Smad6, Runx1 sets up a functional rheostat to control its own activity. The perturbation of this rheostat, using a proteasomal inhibitor, results in an increase in Runx1 and Smad6 levels that can be directly attributed to increased Runx1 binding to tissue-specific regulatory elements of these genes. Taken together, we describe a scenario in which a key hematopoietic transcription factor controls its own expression levels by transcriptionally controlling its controller.</p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">31</span>;<span class="number">14</span>;<span class="pagerange">2817-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21576367" rel="external">21576367</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3133398" rel="external">3133398</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.01305-10" rel="external">10.1128/MCB.01305-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Glyburide is anti-inflammatory and associated with reduced mortality in melioidosis.</h4>
<p class="authors"><span class="author">Koh GC</span>, <span class="author">Maude RR</span>, <span class="author">Schreiber MF</span>, <span class="author">Limmathurotsakul D</span>, <span class="author">Wiersinga WJ</span>, <span class="author">Wuthiekanun V</span>, <span class="author">Lee SJ</span>, <span class="author">Mahavanakul W</span>, <span class="author">Chaowagul W</span>, <span class="author">Chierakul W</span>, <span class="author">White NJ</span>, <span class="author">van der Poll T</span>, <span class="author">Day NP</span>, <span class="author">Dougan G</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, UK. <a href="mailto:%67%61%76%69%6E.%6B%6F%68%40%67%6D%61%69%6C.%63%6F%6D">&#103;&#97;&#118;&#105;&#110;&#46;&#107;&#111;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: Patients with diabetes mellitus are more prone to bacterial sepsis, but there are conflicting data on whether outcomes are worse in diabetics after presentation with sepsis. Glyburide is an oral hypoglycemic agent used to treat diabetes mellitus. This K(ATP)-channel blocker and broad-spectrum ATP-binding cassette (ABC) transporter inhibitor has broad-ranging effects on the immune system, including inhibition of inflammasome assembly and would be predicted to influence the host response to infection.</p><p class="abstract"><strong>Methods</strong>: We studied a cohort of 1160 patients with gram-negative sepsis caused by a single pathogen (Burkholderia pseudomallei), 410 (35%) of whom were known to have diabetes. We subsequently studied prospectively diabetics with B. pseudomallei infection (n = 20) to compare the gene expression profile of peripheral whole blood leukocytes in patients who were taking glyburide against those not taking any sulfonylurea.</p><p class="abstract"><strong>Results</strong>: Survival was greater in diabetics than in nondiabetics (38% vs 45%, respectively, P = .04), but the survival benefit was confined to the patient group taking glyburide (adjusted odds ratio .47, 95% confidence interval .28-.74, P = .005). We identified differential expression of 63 immune-related genes (P = .001) in patients taking glyburide, the sum effect of which we predict to be antiinflammatory in the glyburide group.</p><p class="abstract"><strong>Conclusions</strong>: We present observational evidence for a glyburide-associated benefit during human melioidosis and correlate this with an anti-inflammatory effect of glyburide on the immune system.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093956</span></p>
<p class="publication"><span class="title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">52</span>;<span class="number">6</span>;<span class="pagerange">717-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21293047">21293047</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3049341">3049341</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fcid%2Fciq192">10.1093/cid/ciq192</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diabetes does not influence activation of coagulation, fibrinolysis or anticoagulant pathways in Gram-negative sepsis (melioidosis).</h4>
<p class="authors"><span class="author">Koh GC</span>, <span class="author">Meijers JC</span>, <span class="author">Maude RR</span>, <span class="author">Limmathurotsakul D</span>, <span class="author">Day NP</span>, <span class="author">Peacock SJ</span>, <span class="author">van der Poll T</span> and <span class="author">Wiersinga WJ</span></p>
<p class="affiliation">Center for Experimental and Molecular Medicine, Department of Infectious Diseases, Tropical Medicine &amp; AIDS, Academic Medical Center, Amsterdam, The Netherlands. <a href="mailto:%67%61%76%69%6E.%6B%6F%68%40%67%6D%61%69%6C.%63%6F%6D">&#103;&#97;&#118;&#105;&#110;&#46;&#107;&#111;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Diabetes is associated with a disturbance of the haemostatic balance and is an important risk factor for sepsis, but the influence of diabetes on the pathogenesis of sepsis remains unclear. Melioidosis ( Burkholderia pseudomallei infection) is a common cause of community-acquired sepsis in Southeast Asia and northern Australia. We sought to investigate the impact of pre-existing diabetes on the coagulation and fibrinolytic systems during sepsis caused by B.pseudomallei . We recruited a cohort of 44 patients (34 with diabetes and 10 without diabetes) with culture-proven melioidosis. Diabetes was defined as a pre-admission diagnosis of diabetes or an HbA&#x2081;c&gt;7.8% at enrolment. Thirty healthy blood donors and 52 otherwise healthy diabetes patients served as controls. Citrated plasma was collected from all subjects; additionally in melioidosis patients follow-up specimens were collected seven and &ge; 28 days after enrolment where possible. Relative to uninfected healthy controls, diabetes per se (i.e. in the absence of infection) was characterised by a procoagulant effect. Melioidosis was associated with activation of coagulation (thrombin-antithrombin complexes (TAT), prothrombin fragment F&#x2081;+&#x2082; and fibrinogen concentrations were elevated; PT and PTT prolonged), suppression of anti-coagulation (antithrombin, protein C, total and free protein S levels were depressed) and abnormalities of fibrinolysis (D-dimer and plasmin-antiplasmin complex [PAP] were elevated). Remarkably, none of these haemostatic alterations were influenced by pre-existing diabetes. In conclusion, although diabetes is associated with multiple abnormalities of coagulation, anticoagulation and fibrinolysis, these changes are not detectable when superimposed on the background of larger abnormalities attributable to B. pseudomallei sepsis.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Thrombosis and haemostasis</span>&nbsp;<span class="year">2011</span>;<span class="volume">106</span>;<span class="number">6</span>;<span class="pagerange">1139-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21979378">21979378</a>; DOI: <a href="http://dx.doi.org/10.1160%2FTH11-07-0504">10.1160/TH11-07-0504</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">iCLIP--transcriptome-wide mapping of protein-RNA interactions with individual nucleotide resolution.</h4>
<p class="authors"><span class="author">Konig J</span>, <span class="author">Zarnack K</span>, <span class="author">Rot G</span>, <span class="author">Curk T</span>, <span class="author">Kayikci M</span>, <span class="author">Zupan B</span>, <span class="author">Turner DJ</span>, <span class="author">Luscombe NM</span> and <span class="author">Ule J</span></p>
<p class="affiliation">Laboratory of Molecular Biology, Medical Research Council - MRC.</p><p class="abstract">The unique composition and spatial arrangement of RNA-binding proteins (RBPs) on a transcript guide the diverse aspects of post-transcriptional regulation. Therefore, an essential step towards understanding transcript regulation at the molecular level is to gain positional information on the binding sites of RBPs. Protein-RNA interactions can be studied using biochemical methods, but these approaches do not address RNA binding in its native cellular context. Initial attempts to study protein-RNA complexes in their cellular environment employed affinity purification or immunoprecipitation combined with differential display or microarray analysis (RIP-CHIP). These approaches were prone to identifying indirect or non-physiological interactions. In order to increase the specificity and positional resolution, a strategy referred to as CLIP (UV cross-linking and immunoprecipitation) was introduced. CLIP combines UV cross-linking of proteins and RNA molecules with rigorous purification schemes including denaturing polyacrylamide gel electrophoresis. In combination with high-throughput sequencing technologies, CLIP has proven as a powerful tool to study protein-RNA interactions on a genome-wide scale (referred to as HITS-CLIP or CLIP-seq). Recently, PAR-CLIP was introduced that uses photoreactive ribonucleoside analogs for cross-linking. Despite the high specificity of the obtained data, CLIP experiments often generate cDNA libraries of limited sequence complexity. This is partly due to the restricted amount of co-purified RNA and the two inefficient RNA ligation reactions required for library preparation. In addition, primer extension assays indicated that many cDNAs truncate prematurely at the crosslinked nucleotide. Such truncated cDNAs are lost during the standard CLIP library preparation protocol. We recently developed iCLIP (individual-nucleotide resolution CLIP), which captures the truncated cDNAs by replacing one of the inefficient intermolecular RNA ligation steps with a more efficient intramolecular cDNA circularization (Figure 1). Importantly, sequencing the truncated cDNAs provides insights into the position of the cross-link site at nucleotide resolution. We successfully applied iCLIP to study hnRNP C particle organization on a genome-wide scale and assess its role in splicing regulation.</p>
<p class="publication"><span class="title">Journal of visualized experiments : JoVE</span>&nbsp;<span class="year">2011</span>;<span class="number">50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21559008" rel="external">21559008</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3169244" rel="external">3169244</a>; DOI: <a href="http://dx.doi.org/10.3791%2F2638" rel="external">10.3791/2638</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study in individuals of South Asian ancestry identifies six new type 2 diabetes susceptibility loci.</h4>
<p class="authors"><span class="author">Kooner JS</span>, <span class="author">Saleheen D</span>, <span class="author">Sim X</span>, <span class="author">Sehmi J</span>, <span class="author">Zhang W</span>, <span class="author">Frossard P</span>, <span class="author">Been LF</span>, <span class="author">Chia KS</span>, <span class="author">Dimas AS</span>, <span class="author">Hassanali N</span>, <span class="author">Jafar T</span>, <span class="author">Jowett JB</span>, <span class="author">Li X</span>, <span class="author">Radha V</span>, <span class="author">Rees SD</span>, <span class="author">Takeuchi F</span>, <span class="author">Young R</span>, <span class="author">Aung T</span>, <span class="author">Basit A</span>, <span class="author">Chidambaram M</span>, <span class="author">Das D</span>, <span class="author">Grundberg E</span>, <span class="author">Hedman AK</span>, <span class="author">Hydrie ZI</span>, <span class="author">Islam M</span>, <span class="author">Khor CC</span>, <span class="author">Kowlessur S</span>, <span class="author">Kristensen MM</span>, <span class="author">Liju S</span>, <span class="author">Lim WY</span>, <span class="author">Matthews DR</span>, <span class="author">Liu J</span>, <span class="author">Morris AP</span>, <span class="author">Nica AC</span>, <span class="author">Pinidiyapathirage JM</span>, <span class="author">Prokopenko I</span>, <span class="author">Rasheed A</span>, <span class="author">Samuel M</span>, <span class="author">Shah N</span>, <span class="author">Shera AS</span>, <span class="author">Small KS</span>, <span class="author">Suo C</span>, <span class="author">Wickremasinghe AR</span>, <span class="author">Wong TY</span>, <span class="author">Yang M</span>, <span class="author">Zhang F</span>, <span class="author">DIAGRAM</span>, <span class="author">MuTHER</span>, <span class="author">Abecasis GR</span>, <span class="author">Barnett AH</span>, <span class="author">Caulfield M</span>, <span class="author">Deloukas P</span>, <span class="author">Frayling TM</span>, <span class="author">Froguel P</span>, <span class="author">Kato N</span>, <span class="author">Katulanda P</span>, <span class="author">Kelly MA</span>, <span class="author">Liang J</span>, <span class="author">Mohan V</span>, <span class="author">Sanghera DK</span>, <span class="author">Scott J</span>, <span class="author">Seielstad M</span>, <span class="author">Zimmet PZ</span>, <span class="author">Elliott P</span>, <span class="author">Teo YY</span>, <span class="author">McCarthy MI</span>, <span class="author">Danesh J</span>, <span class="author">Tai ES</span> and <span class="author">Chambers JC</span></p>
<p class="affiliation">National Heart and Lung Institute (NHLI), Imperial College London, Hammersmith Hospital, London, UK. <a href="mailto:%6A.%6B%6F%6F%6E%65%72%40%69%6D%70%65%72%69%61%6C.%61%63.%75%6B">&#106;&#46;&#107;&#111;&#111;&#110;&#101;&#114;&#64;&#105;&#109;&#112;&#101;&#114;&#105;&#97;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We carried out a genome-wide association study of type-2 diabetes (T2D) in individuals of South Asian ancestry. Our discovery set included 5,561 individuals with T2D (cases) and 14,458 controls drawn from studies in London, Pakistan and Singapore. We identified 20 independent SNPs associated with T2D at P &lt; 10(-4) for testing in a replication sample of 13,170 cases and 25,398 controls, also all of South Asian ancestry. In the combined analysis, we identified common genetic variants at six loci (GRB14, ST6GAL1, VPS26A, HMG20A, AP3S2 and HNF4A) newly associated with T2D (P = 4.1 &times; 10(-8) to P = 1.9 &times; 10(-11)). SNPs at GRB14 were also associated with insulin sensitivity (P = 5.0 &times; 10(-4)), and SNPs at ST6GAL1 and HNF4A were also associated with pancreatic beta-cell function (P = 0.02 and P = 0.001, respectively). Our findings provide additional insight into mechanisms underlying T2D and show the potential for new discovery from genetic association studies in South Asians, a population with increased susceptibility to T2D.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">SP/04/002</span>; FIC NIH HHS: <span class="grant">KO1TW006087</span>; Medical Research Council: <span class="grant">G0700931</span>; NIDDK NIH HHS: <span class="grant">DK-25446</span>, <span class="grant">R01DK082766</span>; Wellcome Trust: <span class="grant">070854/Z/03/Z</span>, <span class="grant">080747/Z/06/Z</span>, <span class="grant">083270/Z/07/Z</span>, <span class="grant">084723/Z/08/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">10</span>;<span class="pagerange">984-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21874001">21874001</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.921">10.1038/ng.921</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput semiquantitative analysis of insertional mutations in heterogeneous tumors.</h4>
<p class="authors"><span class="author">Koudijs MJ</span>, <span class="author">Klijn C</span>, <span class="author">van der Weyden L</span>, <span class="author">Kool J</span>, <span class="author">ten Hoeve J</span>, <span class="author">Sie D</span>, <span class="author">Prasetyanti PR</span>, <span class="author">Schut E</span>, <span class="author">Kas S</span>, <span class="author">Whipp T</span>, <span class="author">Cuppen E</span>, <span class="author">Wessels L</span>, <span class="author">Adams DJ</span> and <span class="author">Jonkers J</span></p>
<p class="affiliation">Division of Molecular Biology and Cancer Systems Biology Center, Netherlands Cancer Institute, 1066 CX Amsterdam, The Netherlands.</p><p class="abstract">Retroviral and transposon-based insertional mutagenesis (IM) screens are widely used for cancer gene discovery in mice. Exploiting the full potential of IM screens requires methods for high-throughput sequencing and mapping of transposon and retroviral insertion sites. Current protocols are based on ligation-mediated PCR amplification of junction fragments from restriction endonuclease-digested genomic DNA, resulting in amplification biases due to uneven genomic distribution of restriction enzyme recognition sites. Consequently, sequence coverage cannot be used to assess the clonality of individual insertions. We have developed a novel method, called shear-splink, for the semiquantitative high-throughput analysis of insertional mutations. Shear-splink employs random fragmentation of genomic DNA, which reduces unwanted amplification biases. Additionally, shear-splink enables us to assess clonality of individual insertions by determining the number of unique ligation points (LPs) between the adapter and genomic DNA. This parameter serves as a semiquantitative measure of the relative clonality of individual insertions within heterogeneous tumors. Mixing experiments with clonal cell lines derived from mouse mammary tumor virus (MMTV)-induced tumors showed that shear-splink enables the semiquantitative assessment of the clonality of MMTV insertions. Further, shear-splink analysis of 16 MMTV- and 127 Sleeping Beauty (SB)-induced tumors showed enrichment for cancer-relevant insertions by exclusion of irrelevant background insertions marked by single LPs, thereby facilitating the discovery of candidate cancer genes. To fully exploit the use of the shear-splink method, we set up the Insertional Mutagenesis Database (iMDB), offering a publicly available web-based application to analyze both retroviral- and transposon-based insertional mutagenesis data.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2181-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21852388">21852388</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3227106">3227106</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.112763.110">10.1101/gr.112763.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FoSTeS, MMBIR and NAHR at the human proximal Xp region and the mechanisms of human Xq isochromosome formation.</h4>
<p class="authors"><span class="author">Koumbaris G</span>, <span class="author">Hatzisevastou-Loukidou H</span>, <span class="author">Alexandrou A</span>, <span class="author">Ioannides M</span>, <span class="author">Christodoulou C</span>, <span class="author">Fitzgerald T</span>, <span class="author">Rajan D</span>, <span class="author">Clayton S</span>, <span class="author">Kitsiou-Tzeli S</span>, <span class="author">Vermeesch JR</span>, <span class="author">Skordis N</span>, <span class="author">Antoniou P</span>, <span class="author">Kurg A</span>, <span class="author">Georgiou I</span>, <span class="author">Carter NP</span> and <span class="author">Patsalis PC</span></p>
<p class="affiliation">Department of Medical Genetics, University Medical Center Utrecht, Universiteitsweg 100, 3584 CG Utrecht, The Netherlands.</p><p class="abstract">The recently described DNA replication-based mechanisms of fork stalling and template switching (FoSTeS) and microhomology-mediated break-induced replication (MMBIR) were previously shown to catalyze complex exonic, genic and genomic rearrangements. By analyzing a large number of isochromosomes of the long arm of chromosome X (i(Xq)), using whole-genome tiling path array comparative genomic hybridization (aCGH), ultra-high resolution targeted aCGH and sequencing, we provide evidence that the FoSTeS and MMBIR mechanisms can generate large-scale gross chromosomal rearrangements leading to the deletion and duplication of entire chromosome arms, thus suggesting an important role for DNA replication-based mechanisms in both the development of genomic disorders and cancer. Furthermore, we elucidate the mechanisms of dicentric i(Xq) (idic(Xq)) formation and show that most idic(Xq) chromosomes result from non-allelic homologous recombination between palindromic low copy repeats and highly homologous palindromic LINE elements. We also show that non-recurrent-breakpoint idic(Xq) chromosomes have microhomology-associated breakpoint junctions and are likely catalyzed by microhomology-mediated replication-dependent recombination mechanisms such as FoSTeS and MMBIR. Finally, we stress the role of the proximal Xp region as a chromosomal rearrangement hotspot.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">10</span>;<span class="pagerange">1925-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21349920">21349920</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428953">3428953</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr074">10.1093/hmg/ddr074</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">96-plex molecular barcoding for the Illumina Genome Analyzer.</h4>
<p class="authors"><span class="author">Kozarewa I</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Next-generation sequencing technologies have a massive throughput, which dramatically reduces the cost of sequencing per gigabase, compared to standard Sanger sequencing. To make the most efficient use of this throughput when sequencing small regions or genomes, we developed a barcoding method, which allows multiplexing of 96 or more samples per lane. The method employs 8 bp tags, incorporated into each sequencing library during the library preparation enrichment polymerase chain reaction (PCR), pooling bar-coded libraries in equimolar ratios based on quantitative PCR, and sequencing using the three-read Illumina method.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">733</span>;<span class="pagerange">279-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21431778">21431778</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-61779-089-8_20">10.1007/978-1-61779-089-8_20</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Amplification-free library preparation for paired-end Illumina sequencing.</h4>
<p class="authors"><span class="author">Kozarewa I</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">The library preparation step is of critical importance for the quality of next-generation sequencing data. The use of the polymerase chain reaction (PCR) as a part of the standard Illumina library preparation protocol causes an appreciable proportion of the obtained sequences to be duplicates, making the sequencing run less efficient. Also, amplification introduces biases, particularly for genomes with high or low GC content, which reduces the complexity of the resulting library. To overcome these difficulties, we developed an amplification-free library preparation. By the use of custom adapters, unamplified, ligated samples can hybridize directly to the oligonucleotides on the flowcell surface.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">733</span>;<span class="pagerange">257-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21431776">21431776</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-61779-089-8_18">10.1007/978-1-61779-089-8_18</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A bivariate genome-wide approach to metabolic syndrome: STAMPEED consortium.</h4>
<p class="authors"><span class="author">Kraja AT</span>, <span class="author">Vaidya D</span>, <span class="author">Pankow JS</span>, <span class="author">Goodarzi MO</span>, <span class="author">Assimes TL</span>, <span class="author">Kullo IJ</span>, <span class="author">Sovio U</span>, <span class="author">Mathias RA</span>, <span class="author">Sun YV</span>, <span class="author">Franceschini N</span>, <span class="author">Absher D</span>, <span class="author">Li G</span>, <span class="author">Zhang Q</span>, <span class="author">Feitosa MF</span>, <span class="author">Glazer NL</span>, <span class="author">Haritunians T</span>, <span class="author">Hartikainen AL</span>, <span class="author">Knowles JW</span>, <span class="author">North KE</span>, <span class="author">Iribarren C</span>, <span class="author">Kral B</span>, <span class="author">Yanek L</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">McCarthy MI</span>, <span class="author">Jaquish C</span>, <span class="author">Couper DJ</span>, <span class="author">Chakravarti A</span>, <span class="author">Psaty BM</span>, <span class="author">Becker LC</span>, <span class="author">Province MA</span>, <span class="author">Boerwinkle E</span>, <span class="author">Quertermous T</span>, <span class="author">Palotie L</span>, <span class="author">Jarvelin MR</span>, <span class="author">Becker DM</span>, <span class="author">Kardia SL</span>, <span class="author">Rotter JI</span>, <span class="author">Chen YD</span> and <span class="author">Borecki IB</span></p>
<p class="affiliation">Division of Statistical Genomics, Washington University School of Medicine, Saint Louis, Missouri, USA. <a href="mailto:%61%6C%64%69%40%77%75%73%74%6C.%65%64%75">&#97;&#108;&#100;&#105;&#64;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">OBJECTIVE The metabolic syndrome (MetS) is defined as concomitant disorders of lipid and glucose metabolism, central obesity, and high blood pressure, with an increased risk of type 2 diabetes and cardiovascular disease. This study tests whether common genetic variants with pleiotropic effects account for some of the correlated architecture among five metabolic phenotypes that define MetS. RESEARCH DESIGN AND METHODS Seven studies of the STAMPEED consortium, comprising 22,161 participants of European ancestry, underwent genome-wide association analyses of metabolic traits using a panel of &sim;2.5 million imputed single nucleotide polymorphisms (SNPs). Phenotypes were defined by the National Cholesterol Education Program (NCEP) criteria for MetS in pairwise combinations. Individuals exceeding the NCEP thresholds for both traits of a pair were considered affected. RESULTS Twenty-nine common variants were associated with MetS or a pair of traits. Variants in the genes LPL, CETP, APOA5 (and its cluster), GCKR (and its cluster), LIPC, TRIB1, LOC100128354/MTNR1B, ABCB11, and LOC100129150 were further tested for their association with individual qualitative and quantitative traits. None of the 16 top SNPs (one per gene) associated simultaneously with more than two individual traits. Of them 11 variants showed nominal associations with MetS per se. The effects of 16 top SNPs on the quantitative traits were relatively small, together explaining from &sim;9% of the variance in triglycerides, 5.8% of high-density lipoprotein cholesterol, 3.6% of fasting glucose, and 1.4% of systolic blood pressure. CONCLUSIONS Qualitative and quantitative pleiotropic tests on pairs of traits indicate that a small portion of the covariation in these traits can be explained by the reported common genetic variants.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">L1-RR-025005</span>, <span class="grant">M01RR00425</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">5R01-HL-087647</span>, <span class="grant">5R01-HL-087679-02</span>, <span class="grant">5R01-HL-087698</span>, <span class="grant">HL-087652</span>, <span class="grant">HL-087660</span>, <span class="grant">HL-087700</span>, <span class="grant">HL-0877700</span>, <span class="grant">N01-HC-15103</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-55222</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">NC01-HC-55021</span>, <span class="grant">R01-HL-086694</span>, <span class="grant">R01-HL-087641</span>, <span class="grant">R01-HL-087652</span>, <span class="grant">R01-HL-59367</span>, <span class="grant">U01-HL-075572</span>, <span class="grant">U01-HL-080295</span>; NIDDK NIH HHS: <span class="grant">DK-063491</span>; NIMH NIH HHS: <span class="grant">1RL1-MH-083268-01</span>, <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01-MH-63706:0</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">4</span>;<span class="pagerange">1329-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21386085" rel="external">21386085</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3064107" rel="external">3064107</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb10-1011" rel="external">10.2337/db10-1011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">miR-96 regulates the progression of differentiation in mammalian cochlear inner and outer hair cells.</h4>
<p class="authors"><span class="author">Kuhn S</span>, <span class="author">Johnson SL</span>, <span class="author">Furness DN</span>, <span class="author">Chen J</span>, <span class="author">Ingham N</span>, <span class="author">Hilton JM</span>, <span class="author">Steffes G</span>, <span class="author">Lewis MA</span>, <span class="author">Zampini V</span>, <span class="author">Hackney CM</span>, <span class="author">Masetto S</span>, <span class="author">Holley MC</span>, <span class="author">Steel KP</span> and <span class="author">Marcotti W</span></p>
<p class="affiliation">Department of Biomedical Science, University of Sheffield, Sheffield S10 2TN, United Kingdom.</p><p class="abstract">MicroRNAs (miRNAs) are small noncoding RNAs able to regulate a broad range of protein-coding genes involved in many biological processes. miR-96 is a sensory organ-specific miRNA expressed in the mammalian cochlea during development. Mutations in miR-96 cause nonsyndromic progressive hearing loss in humans and mice. The mouse mutant diminuendo has a single base change in the seed region of the Mir96 gene leading to widespread changes in the expression of many genes. We have used this mutant to explore the role of miR-96 in the maturation of the auditory organ. We found that the physiological development of mutant sensory hair cells is arrested at around the day of birth, before their biophysical differentiation into inner and outer hair cells. Moreover, maturation of the hair cell stereocilia bundle and remodelling of auditory nerve connections within the cochlea fail to occur in miR-96 mutants. We conclude that miR-96 regulates the progression of the physiological and morphological differentiation of cochlear hair cells and, as such, coordinates one of the most distinctive functional refinements of the mammalian auditory system.</p>
<p class="grants">Funded by: Action on Hearing Loss: <span class="grant">G41</span>; Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust: <span class="grant">077189</span>, <span class="grant">088719</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">6</span>;<span class="pagerange">2355-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21245307">21245307</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3038748">3038748</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1016646108">10.1073/pnas.1016646108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a ViI-like phage specific to Escherichia coli O157:H7.</h4>
<p class="authors"><span class="author">Kutter EM</span>, <span class="author">Skutt-Kakaria K</span>, <span class="author">Blasdel B</span>, <span class="author">El-Shibiny A</span>, <span class="author">Castano A</span>, <span class="author">Bryan D</span>, <span class="author">Kropinski AM</span>, <span class="author">Villegas A</span>, <span class="author">Ackermann HW</span>, <span class="author">Toribio AL</span>, <span class="author">Pickard D</span>, <span class="author">Anany H</span>, <span class="author">Callaway T</span> and <span class="author">Brabban AD</span></p>
<p class="affiliation">The Evergreen State College, Olympia, WA, USA. <a href="mailto:%6B%75%74%74%65%72%62%40%65%76%65%72%67%72%65%65%6E.%65%64%75">&#107;&#117;&#116;&#116;&#101;&#114;&#98;&#64;&#101;&#118;&#101;&#114;&#103;&#114;&#101;&#101;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Phage vB_EcoM_CBA120 (CBA120), isolated against Escherichia coli O157:H7 from a cattle feedlot, is morphologically very similar to the classic phage ViI of Salmonella enterica serovar Typhi. Until recently, little was known genetically or physiologically about the ViI-like phages, and none targeting E. coli have been described in the literature. The genome of CBA120 has been fully sequenced and is highly similar to those of both ViI and the Shigella phage AG3. The core set of structural and replication-related proteins of CBA120 are homologous to those from T-even phages, but generally are more closely related to those from T4-like phages of Vibrio, Aeromonas and cyanobacteria than those of the Enterobacteriaceae. The baseplate and method of adhesion to the host are, however, very different from those of either T4 or the cyanophages. None of the outer baseplate proteins are conserved. Instead of T4&#39;s long and short tail fibers, CBA120, like ViI, encodes tail spikes related to those normally seen on podoviruses. The 158 kb genome, like that of T4, is circularly permuted and terminally redundant, but unlike T4 CBA120 does not substitute hmdCyt for cytosine in its DNA. However, in contrast to other coliphages, CBA120 and related coliphages we have isolated cannot incorporate 3H-thymidine (3H-dThd) into their DNA. Protein sequence comparisons cluster the putative &quot;thymidylate synthase&quot; of CBA120, ViI and AG3 much more closely with those of Delftia phage &phi;W-14, Bacillus subtilis phage SPO1, and Pseudomonas phage YuA, all known to produce and incorporate hydroxymethyluracil (hmdUra).</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">2-R15GM063637-02</span>, <span class="grant">2R15GM63637-3A1.</span></p>
<p class="publication"><span class="title">Virology journal</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="pagerange">430</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21899740" rel="external">21899740</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3184105" rel="external">3184105</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1743-422X-8-430" rel="external">10.1186/1743-422X-8-430</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population-prevalent desmosomal mutations predisposing to arrhythmogenic right ventricular cardiomyopathy.</h4>
<p class="authors"><span class="author">Lahtinen AM</span>, <span class="author">Lehtonen E</span>, <span class="author">Marjamaa A</span>, <span class="author">Kaartinen M</span>, <span class="author">Heli&ouml; T</span>, <span class="author">Porthan K</span>, <span class="author">Oikarinen L</span>, <span class="author">Toivonen L</span>, <span class="author">Swan H</span>, <span class="author">Jula A</span>, <span class="author">Peltonen L</span>, <span class="author">Palotie A</span>, <span class="author">Salomaa V</span> and <span class="author">Kontula K</span></p>
<p class="affiliation">Research Program for Molecular Medicine, Biomedicum Helsinki, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: Arrhythmogenic right ventricular cardiomyopathy (ARVC) is a progressive myocardial disorder caused by mutations of desmosomal cell adhesion proteins. The prevalence of these variants in the general population is unknown.</p><p class="abstract"><strong>Objective</strong>: This study examined the spectrum and population prevalence of desmosomal mutations predisposing to ARVC in Finland.</p><p class="abstract"><strong>Methods</strong>: We screened 29 Finnish ARVC probands for mutations in the DSP, DSG2, and DSC2 genes. All Finnish-type ARVC-associated mutations, including those 3 previously identified in PKP2 in the same patient group, were analyzed in the population-based Health 2000 cohort of 6,334 individuals and tested for association with electrocardiographic variables.</p><p class="abstract"><strong>Results</strong>: We detected 2 novel mutations: DSG2 3059_3062delAGAG and DSP T1373A. DSG2 3059_3062delAGAG was present in a family with 5 mutation carriers. The endomyocardial samples of the DSG2 deletion carrier showed reduced immunoreactive signal for desmoglein-2, plakophilin-2, plakoglobin, and desmoplakin. DSP T1373A was found in 1 proband with typical right ventricular disease and exercise-related ventricular tachycardia. In the population sample, the collective prevalence of all 5 mutations identified in the 29 ARVC patients (PKP2 Q62K, Q59L, N613K, DSG2 3059_3062delAGAG, and DSP T1373A) was 31 of 6,334 individuals, or 0.5%. The apparent founder mutation PKP2 Q59L is present in 0.3% of Finns and was previously shown to have an approximately 20% disease penetrance.</p><p class="abstract"><strong>Conclusion</strong>: One of 200 Finns carries a desmosomal mutation that may predispose to ARVC and its clinical sequelae. ARVC-associated mutations may thus be more prevalent in the population than expected based on the published ARVC prevalence data.</p>
<p class="publication"><span class="title">Heart rhythm : the official journal of the Heart Rhythm Society</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">1214-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21397041">21397041</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.hrthm.2011.03.015">10.1016/j.hrthm.2011.03.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">X-box binding protein 1 induces the expression of the lytic cycle transactivator of Kaposi&#39;s sarcoma-associated herpesvirus but not Epstein-Barr virus in co-infected primary effusion lymphoma.</h4>
<p class="authors"><span class="author">Lai IY</span>, <span class="author">Farrell PJ</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">University College London, MRC Centre for Molecular Virology, Department of Infection, Division of Infection and Immunity, Windeyer Institute of Medical Science, 46 Cleveland Street, London W1T 4JF, UK.</p><p class="abstract">Cells of primary effusion lymphoma (PEL), a B-cell non-Hodgkin&#39;s lymphoma, are latently infected by Kaposi&#39;s sarcoma-associated herpesvirus (KSHV), with about 80&#x200A;% of PEL also co-infected with Epstein-Barr virus (EBV). Both viruses can be reactivated into their lytic replication cycle in PEL by chemical inducers. However, simultaneous activation of both lytic cascades leads to mutual lytic cycle co-repression. The plasma cell-differentiation factor X-box binding protein 1 (XBP-1) transactivates the KSHV immediate-early promoter leading to the production of the replication and transcription activator protein (RTA), and reactivation of KSHV from latency. XBP-1 has been reported to act similarly on the EBV immediate-early promoter Zp, leading to the production of the lytic-cycle transactivator protein BZLF1. Here we show that activated B-cell terminal-differentiation transcription factor X-box binding protein 1 (XBP-1s) does not induce EBV BZLF1 and BRLF1 expression in PEL and BL cell lines, despite inducing lytic reactivation of KSHV in PEL. We show that XBP-1s transactivates the KSHV RTA promoter but does not transactivate the EBV BZLF1 promoter in non-B-cells by using a luciferase assay. Co-expression of activated protein kinase D, which can phosphorylate and inactivate class II histone deacetylases (HDACs), does not rescue XBP-1 activity on Zp nor does it induce BZLF1 and BRLF1 expression in PEL. Finally, chemical inducers of KSHV and EBV lytic replication in PEL, including HDAC inhibitors, do not lead to XBP-1 activation. We conclude that XBP-1 specifically reactivates the KSHV lytic cycle in dually infected PELs.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of general virology</span>&nbsp;<span class="year">2011</span>;<span class="volume">92</span>;<span class="number">Pt 2</span>;<span class="pagerange">421-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20980528">20980528</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3081082">3081082</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fvir.0.025494-0">10.1099/vir.0.025494-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Annotation of two large contiguous regions from the Haemonchus contortus genome using RNA-seq and comparative analysis with Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Laing R</span>, <span class="author">Hunt M</span>, <span class="author">Protasio AV</span>, <span class="author">Saunders G</span>, <span class="author">Mungall K</span>, <span class="author">Laing S</span>, <span class="author">Jackson F</span>, <span class="author">Quail M</span>, <span class="author">Beech R</span>, <span class="author">Berriman M</span> and <span class="author">Gilleard JS</span></p>
<p class="affiliation">Welcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The genomes of numerous parasitic nematodes are currently being sequenced, but their complexity and size, together with high levels of intra-specific sequence variation and a lack of reference genomes, makes their assembly and annotation a challenging task. Haemonchus contortus is an economically significant parasite of livestock that is widely used for basic research as well as for vaccine development and drug discovery. It is one of many medically and economically important parasites within the strongylid nematode group. This group of parasites has the closest phylogenetic relationship with the model organism Caenorhabditis elegans, making comparative analysis a potentially powerful tool for genome annotation and functional studies. To investigate this hypothesis, we sequenced two contiguous fragments from the H. contortus genome and undertook detailed annotation and comparative analysis with C. elegans. The adult H. contortus transcriptome was sequenced using an Illumina platform and RNA-seq was used to annotate a 409 kb overlapping BAC tiling path relating to the X chromosome and a 181 kb BAC insert relating to chromosome I. In total, 40 genes and 12 putative transposable elements were identified. 97.5% of the annotated genes had detectable homologues in C. elegans of which 60% had putative orthologues, significantly higher than previous analyses based on EST analysis. Gene density appears to be less in H. contortus than in C. elegans, with annotated H. contortus genes being an average of two-to-three times larger than their putative C. elegans orthologues due to a greater intron number and size. Synteny appears high but gene order is generally poorly conserved, although areas of conserved microsynteny are apparent. C. elegans operons appear to be partially conserved in H. contortus. Our findings suggest that a combination of RNA-seq and comparative analysis with C. elegans is a powerful approach for the annotation and analysis of strongylid nematode genomes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">e23216</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21858033">21858033</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3156134">3156134</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0023216">10.1371/journal.pone.0023216</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Review article: faecal transplantation therapy for gastrointestinal disease.</h4>
<p class="authors"><span class="author">Landy J</span>, <span class="author">Al-Hassi HO</span>, <span class="author">McLaughlin SD</span>, <span class="author">Walker AW</span>, <span class="author">Ciclitira PJ</span>, <span class="author">Nicholls RJ</span>, <span class="author">Clark SK</span> and <span class="author">Hart AL</span></p>
<p class="affiliation">IBD Unit, St Mark&#39;s Hospital, Harrow, London, UK.</p><p class="abstract"><strong>Background</strong>: Evidence is emerging regarding the relationship between a dysbiosis of the human gut microbiota and a number of gastrointestinal diseases as well as diseases beyond the gut. Probiotics have been investigated in many gastrointestinal disease states, with variable and often modest outcomes. Faecal transplantation is an alternative approach to manipulate the gut microbiota.</p><p class="abstract"><strong>Aim</strong>: To review the use of faecal transplantation therapy for the management of gastrointestinal disorders.</p><p class="abstract"><strong>Methods</strong>: Available articles on faecal transplantation in the management of gastrointestinal disorders were identified using a Pubmed search and bibliographies of review articles on the subject were collated.</p><p class="abstract"><strong>Results</strong>: A total of 239 patients who had undergone faecal transplantation were reported. Seventeen of 22 studies of faecal transplantation were in fulminant or refractory Clostridium difficile. Studies of faecal transplantation are heterogeneous regarding the patients, donors, screening, methods of administration and definition of response. Faecal transplantation for C. difficile has been demonstrated to be effective in 145/166 (87%) patients. Small numbers of patients are reported to have undergone successful faecal transplantation for irritable bowel syndrome and inflammatory bowel disease.</p><p class="abstract"><strong>Conclusions</strong>: Faecal transplantation has been reported with good outcomes for fulminant and refractory C. difficile. No adverse effects of faecal transplantation have been reported. However, there are no level 1 data of faecal transplantation and reports to date may suffer from reporting bias of positive outcomes and under-reporting of adverse effects. This therapy holds great promise, where a dysbiosis of the gut microbiota is responsible for disease and further studies are necessary to explore this potential.</p>
<p class="publication"><span class="title">Alimentary pharmacology &amp; therapeutics</span>&nbsp;<span class="year">2011</span>;<span class="volume">34</span>;<span class="number">4</span>;<span class="pagerange">409-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21682755" rel="external">21682755</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2036.2011.04737.x" rel="external">10.1111/j.1365-2036.2011.04737.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plant power: converting a kingdom.</h4>
<p class="authors"><span class="author">Langridge G</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This month, Genome Watch looks at the potential for bacterially derived enzymes to degrade lignocellulose from plant biomass and thus provide an efficient way of producing biofuels.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">316</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21494277" rel="external">21494277</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2566" rel="external">10.1038/nrmicro2566</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of Dense Genecentric Association Studies Reveals Common and Uncommon Variants Associated with Height.</h4>
<p class="authors"><span class="author">Lanktree MB</span>, <span class="author">Guo Y</span>, <span class="author">Murtaza M</span>, <span class="author">Glessner JT</span>, <span class="author">Bailey SD</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Lettre G</span>, <span class="author">Ongen H</span>, <span class="author">Rajagopalan R</span>, <span class="author">Johnson T</span>, <span class="author">Shen H</span>, <span class="author">Nelson CP</span>, <span class="author">Klopp N</span>, <span class="author">Baumert J</span>, <span class="author">Padmanabhan S</span>, <span class="author">Pankratz N</span>, <span class="author">Pankow JS</span>, <span class="author">Shah S</span>, <span class="author">Taylor K</span>, <span class="author">Barnard J</span>, <span class="author">Peters BJ</span>, <span class="author">Maloney CM</span>, <span class="author">Lobmeyer MT</span>, <span class="author">Stanton A</span>, <span class="author">Zafarmand MH</span>, <span class="author">Romaine SP</span>, <span class="author">Mehta A</span>, <span class="author">van Iperen EP</span>, <span class="author">Gong Y</span>, <span class="author">Price TS</span>, <span class="author">Smith EN</span>, <span class="author">Kim CE</span>, <span class="author">Li YR</span>, <span class="author">Asselbergs FW</span>, <span class="author">Atwood LD</span>, <span class="author">Bailey KM</span>, <span class="author">Bhatt D</span>, <span class="author">Bauer F</span>, <span class="author">Behr ER</span>, <span class="author">Bhangale T</span>, <span class="author">Boer JM</span>, <span class="author">Boehm BO</span>, <span class="author">Bradfield JP</span>, <span class="author">Brown M</span>, <span class="author">Braund PS</span>, <span class="author">Burton PR</span>, <span class="author">Carty C</span>, <span class="author">Chandrupatla HR</span>, <span class="author">Chen W</span>, <span class="author">Connell J</span>, <span class="author">Dalgeorgou C</span>, <span class="author">Boer A</span>, <span class="author">Drenos F</span>, <span class="author">Elbers CC</span>, <span class="author">Fang JC</span>, <span class="author">Fox CS</span>, <span class="author">Frackelton EC</span>, <span class="author">Fuchs B</span>, <span class="author">Furlong CE</span>, <span class="author">Gibson Q</span>, <span class="author">Gieger C</span>, <span class="author">Goel A</span>, <span class="author">Grobbee DE</span>, <span class="author">Hastie C</span>, <span class="author">Howard PJ</span>, <span class="author">Huang GH</span>, <span class="author">Johnson WC</span>, <span class="author">Li Q</span>, <span class="author">Kleber ME</span>, <span class="author">Klein BE</span>, <span class="author">Klein R</span>, <span class="author">Kooperberg C</span>, <span class="author">Ky B</span>, <span class="author">Lacroix A</span>, <span class="author">Lanken P</span>, <span class="author">Lathrop M</span>, <span class="author">Li M</span>, <span class="author">Marshall V</span>, <span class="author">Melander O</span>, <span class="author">Mentch FD</span>, <span class="author">Meyer NJ</span>, <span class="author">Monda KL</span>, <span class="author">Montpetit A</span>, <span class="author">Murugesan G</span>, <span class="author">Nakayama K</span>, <span class="author">Nondahl D</span>, <span class="author">Onipinla A</span>, <span class="author">Rafelt S</span>, <span class="author">Newhouse SJ</span>, <span class="author">Otieno FG</span>, <span class="author">Patel SR</span>, <span class="author">Putt ME</span>, <span class="author">Rodriguez S</span>, <span class="author">Safa RN</span>, <span class="author">Sawyer DB</span>, <span class="author">Schreiner PJ</span>, <span class="author">Simpson C</span>, <span class="author">Sivapalaratnam S</span>, <span class="author">Srinivasan SR</span>, <span class="author">Suver C</span>, <span class="author">Swergold G</span>, <span class="author">Sweitzer NK</span>, <span class="author">Thomas KA</span>, <span class="author">Thorand B</span>, <span class="author">Timpson NJ</span>, <span class="author">Tischfield S</span>, <span class="author">Tobin M</span>, <span class="author">Tomaszweski M</span>, <span class="author">Verschuren WM</span>, <span class="author">Wallace C</span>, <span class="author">Winkelmann B</span>, <span class="author">Zhang H</span>, <span class="author">Zheng D</span>, <span class="author">Zhang L</span>, <span class="author">Zmuda JM</span>, <span class="author">Clarke R</span>, <span class="author">Balmforth AJ</span>, <span class="author">Danesh J</span>, <span class="author">Day IN</span>, <span class="author">Schork NJ</span>, <span class="author">de Bakker PI</span>, <span class="author">Delles C</span>, <span class="author">Duggan D</span>, <span class="author">Hingorani AD</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Hofker MH</span>, <span class="author">Humphries SE</span>, <span class="author">Kivimaki M</span>, <span class="author">Lawlor DA</span>, <span class="author">Kottke-Marchant K</span>, <span class="author">Mega JL</span>, <span class="author">Mitchell BD</span>, <span class="author">Morrow DA</span>, <span class="author">Palmen J</span>, <span class="author">Redline S</span>, <span class="author">Shields DC</span>, <span class="author">Shuldiner AR</span>, <span class="author">Sleiman PM</span>, <span class="author">Smith GD</span>, <span class="author">Farrall M</span>, <span class="author">Jamshidi Y</span>, <span class="author">Christiani DC</span>, <span class="author">Casas JP</span>, <span class="author">Hall AS</span>, <span class="author">Doevendans PA</span>, <span class="author">Christie JD</span>, <span class="author">Berenson GS</span>, <span class="author">Murray SS</span>, <span class="author">Illig T</span>, <span class="author">Dorn GW</span>, <span class="author">Cappola TP</span>, <span class="author">Boerwinkle E</span>, <span class="author">Sever P</span>, <span class="author">Rader DJ</span>, <span class="author">Reilly MP</span>, <span class="author">Caulfield M</span>, <span class="author">Talmud PJ</span>, <span class="author">Topol E</span>, <span class="author">Engert JC</span>, <span class="author">Wang K</span>, <span class="author">Dominiczak A</span>, <span class="author">Hamsten A</span>, <span class="author">Curtis SP</span>, <span class="author">Silverstein RL</span>, <span class="author">Lange LA</span>, <span class="author">Sabatine MS</span>, <span class="author">Trip M</span>, <span class="author">Saleheen D</span>, <span class="author">Peden JF</span>, <span class="author">Cruickshanks KJ</span>, <span class="author">M&auml;rz W</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Klungel OH</span>, <span class="author">Wijmenga C</span>, <span class="author">Maitland-van der Zee AH</span>, <span class="author">Schadt EE</span>, <span class="author">Johnson JA</span>, <span class="author">Jarvik GP</span>, <span class="author">Papanicolaou GJ</span>, <span class="author">Hugh Watkins on behalf of PROCARDIS</span>, <span class="author">Grant SF</span>, <span class="author">Munroe PB</span>, <span class="author">North KE</span>, <span class="author">Samani NJ</span>, <span class="author">Koenig W</span>, <span class="author">Gaunt TR</span>, <span class="author">Anand SS</span>, <span class="author">van der Schouw YT</span>, <span class="author">Meena Kumari on behalf of the Whitehall II Study and the WHII 50K Group</span>, <span class="author">Soranzo N</span>, <span class="author">Fitzgerald GA</span>, <span class="author">Reiner A</span>, <span class="author">Hegele RA</span>, <span class="author">Hakonarson H</span> and <span class="author">Keating BJ</span></p>
<p class="affiliation">Department of Medicine and Biochemistry, University of Western Ontario, London, Ontario, N6A 5C1, Canada.</p><p class="abstract">Height is a classic complex trait with common variants in a growing list of genes known to contribute to the phenotype. Using a genecentric genotyping array targeted toward cardiovascular-related loci, comprising 49,320 SNPs across approximately 2000 loci, we evaluated the association of common and uncommon SNPs with adult height in 114,223 individuals from 47 studies and six ethnicities. A&nbsp;total of 64 loci contained a SNP associated with height at array-wide significance (p &lt; 2.4&nbsp;&times; 10(-6)), with 42 loci surpassing the conventional genome-wide significance threshold (p &lt; 5&nbsp;&times; 10(-8)). Common variants with minor allele frequencies greater than 5% were observed to be associated with height in 37 previously reported loci. In individuals of European ancestry, uncommon SNPs in IL11 and SMAD3, which would not be genotyped with the use of standard genome-wide genotyping arrays, were strongly associated with height (p &lt; 3&nbsp;&times; 10(-11)). Conditional analysis within associated regions revealed five additional variants associated with height independent of lead SNPs within the locus, suggesting allelic heterogeneity. Although underpowered to replicate findings from individuals of European ancestry, the direction of effect of associated variants was largely consistent in African American, South Asian, and Hispanic populations. Overall, we show that dense coverage of genes for uncommon SNPs, coupled with large-scale meta-analysis, can successfully identify additional variants associated with a common complex trait.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">79533</span>; NEI NIH HHS: <span class="grant">U10 EY006594-24</span>; NHLBI NIH HHS: <span class="grant">K23 HL095661-02</span>; NIA NIH HHS: <span class="grant">R01 AG011099-07</span>, <span class="grant">R01 AG011099-08</span>, <span class="grant">R01 AG011099-09</span>, <span class="grant">R01 AG011099-10</span>, <span class="grant">R01 AG011099-11</span>, <span class="grant">R01 AG011099-12</span>, <span class="grant">R01 AG011099-13</span>, <span class="grant">R01 AG011099-14</span>, <span class="grant">R01 AG011099-15</span>, <span class="grant">R01 AG011099-15S1</span>, <span class="grant">R01 AG021917-08</span>, <span class="grant">R37 AG011099-17</span>, <span class="grant">R37 AG011099-19</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">88</span>;<span class="number">1</span>;<span class="pagerange">6-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21194676" rel="external">21194676</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3014369" rel="external">3014369</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.11.007" rel="external">10.1016/j.ajhg.2010.11.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SNP detection and genotyping from low-coverage sequencing data on multiple diploid samples.</h4>
<p class="authors"><span class="author">Le SQ</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Reductions in the cost of sequencing have enabled whole-genome sequencing to identify sequence variants segregating in a population. An efficient approach is to sequence many samples at low coverage, then to combine data across samples to detect shared variants. Here, we present methods to discover and genotype single-nucleotide polymorphism (SNP) sites from low-coverage sequencing data, making use of shared haplotype (linkage disequilibrium) information. For each population, we first collect SNP candidates based on independent sequence calls per site. We then use MARGARITA with genotype or phased haplotype data from the same samples to collect 20 ancestral recombination graphs (ARGs). We refine the posterior probability of SNP candidates by considering possible mutations at internal branches of the 40 marginal ancestral trees inferred from the 20 ARGs at the left and right flanking genotype sites. Using a population genetic prior distribution on tree-branch length and Bayesian inference, we determine a posterior probability of the SNP being real and also the most probable phased genotype call for each individual. We present experiments on both simulation data and real data from the 1000 Genomes Project to prove the applicability of the methods. We also explore the relative tradeoff between sequencing depth and the number of sequenced samples.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT089088/Z/09/Z</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">6</span>;<span class="pagerange">952-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20980557" rel="external">20980557</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3106328" rel="external">3106328</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.113084.110" rel="external">10.1101/gr.113084.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal instability confers intrinsic multidrug resistance.</h4>
<p class="authors"><span class="author">Lee AJ</span>, <span class="author">Endesfelder D</span>, <span class="author">Rowan AJ</span>, <span class="author">Walther A</span>, <span class="author">Birkbak NJ</span>, <span class="author">Futreal PA</span>, <span class="author">Downward J</span>, <span class="author">Szallasi Z</span>, <span class="author">Tomlinson IP</span>, <span class="author">Howell M</span>, <span class="author">Kschischo M</span> and <span class="author">Swanton C</span></p>
<p class="affiliation">Translational Cancer Therapeutics Laboratory, Cancer Research UK London Research Institute, London, United Kingdom.</p><p class="abstract">Aneuploidy is associated with poor prognosis in solid tumors. Spontaneous chromosome missegregation events in aneuploid cells promote chromosomal instability (CIN) that may contribute to the acquisition of multidrug resistance in vitro and heighten risk for tumor relapse in animal models. Identification of distinct therapeutic agents that target tumor karyotypic complexity has important clinical implications. To identify distinct therapeutic approaches to specifically limit the growth of CIN tumors, we focused on a panel of colorectal cancer (CRC) cell lines, previously classified as either chromosomally unstable (CIN(+)) or diploid/near-diploid (CIN(-)), and treated them individually with a library of kinase inhibitors targeting components of signal transduction, cell cycle, and transmembrane receptor signaling pathways. CIN(+) cell lines displayed significant intrinsic multidrug resistance compared with CIN(-) cancer cell lines, and this seemed to be independent of somatic mutation status and proliferation rate. Confirming the association of CIN rather than ploidy status with multidrug resistance, tetraploid isogenic cells that had arisen from diploid cell lines displayed lower drug sensitivity than their diploid parental cells only with increasing chromosomal heterogeneity and isogenic cell line models of CIN(+) displayed multidrug resistance relative to their CIN(-) parental cancer cell line derivatives. In a meta-analysis of CRC outcome following cytotoxic treatment, CIN(+) predicted worse progression-free or disease-free survival relative to patients with CIN(-) disease. Our results suggest that stratifying tumor responses according to CIN status should be considered within the context of clinical trials to minimize the confounding effects of tumor CIN status on drug sensitivity.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0701935(86985)</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2011</span>;<span class="volume">71</span>;<span class="number">5</span>;<span class="pagerange">1858-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21363922" rel="external">21363922</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3059493" rel="external">3059493</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-10-3604" rel="external">10.1158/0008-5472.CAN-10-3604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Induction of stable drug resistance in human breast cancer cells using a combinatorial zinc finger transcription factor library.</h4>
<p class="authors"><span class="author">Lee J</span>, <span class="author">Hirsh AS</span>, <span class="author">Wittner BS</span>, <span class="author">Maeder ML</span>, <span class="author">Singavarapu R</span>, <span class="author">Lang M</span>, <span class="author">Janarthanan S</span>, <span class="author">McDermott U</span>, <span class="author">Yajnik V</span>, <span class="author">Ramaswamy S</span>, <span class="author">Joung JK</span> and <span class="author">Sgroi DC</span></p>
<p class="affiliation">Molecular Pathology Unit, Massachusetts General Hospital, Charlestown, Massachusetts, United States of America.</p><p class="abstract">Combinatorial libraries of artificial zinc-finger transcription factors (ZF-TFs) provide a robust tool for inducing and understanding various functional components of the cancer phenotype. Herein, we utilized combinatorial ZF-TF library technology to better understand how breast cancer cells acquire resistance to fulvestrant, a clinically important anti-endocrine therapeutic agent. From a diverse collection of nearly 400,000 different ZF-TFs, we isolated six ZF-TF library members capable of inducing stable, long-term anti-endocrine drug-resistance in two independent estrogen receptor-positive breast cancer cell lines. Comparative gene expression profile analysis of the six different ZF-TF-transduced breast cancer cell lines revealed five distinct clusters of differentially expressed genes. One cluster was shared among all 6 ZF-TF-transduced cell lines and therefore constituted a common fulvestrant-resistant gene expression signature. Pathway enrichment-analysis of this common fulvestrant resistant signature also revealed significant overlap with gene sets associated with an estrogen receptor-negative-like state and with gene sets associated with drug resistance to different classes of breast cancer anti-endocrine therapeutic agents. Enrichment-analysis of the four remaining unique gene clusters revealed overlap with myb-regulated genes. Finally, we also demonstrated that the common fulvestrant-resistant signature is associated with poor prognosis by interrogating five independent, publicly available human breast cancer gene expression datasets. Our results demonstrate that artificial ZF-TF libraries can be used successfully to induce stable drug-resistance in human cancer cell lines and to identify a gene expression signature that is associated with a clinically relevant drug-resistance phenotype.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01-CA112021</span>, <span class="grant">T32 CA009216</span>; NIGMS NIH HHS: <span class="grant">R01 GM069906</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">e21112</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21818254" rel="external">21818254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3139592" rel="external">3139592</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0021112" rel="external">10.1371/journal.pone.0021112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New IBD genetics: common pathways with other diseases.</h4>
<p class="authors"><span class="author">Lees CW</span>, <span class="author">Barrett JC</span>, <span class="author">Parkes M</span> and <span class="author">Satsangi J</span></p>
<p class="affiliation">Gastrointestinal Unit, Molecular Medicine Centre, University of Edinburgh, Edinburgh, UK. <a href="mailto:%63%68%61%72%6C%69%65.%6C%65%65%73%40%65%64.%61%63.%75%6B">&#99;&#104;&#97;&#114;&#108;&#105;&#101;&#46;&#108;&#101;&#101;&#115;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Complex disease genetics has been revolutionised in recent years by the advent of genome-wide association (GWA) studies. The chronic inflammatory bowel diseases (IBDs), Crohn&#39;s disease and ulcerative colitis have seen notable successes culminating in the discovery of 99 published susceptibility loci/genes (71 Crohn&#39;s disease; 47 ulcerative colitis) to date. Approximately one-third of loci described confer susceptibility to both Crohn&#39;s disease and ulcerative colitis. Amongst these are multiple genes involved in IL23/Th17 signalling (IL23R, IL12B, JAK2, TYK2 and STAT3), IL10, IL1R2, REL, CARD9, NKX2.3, ICOSLG, PRDM1, SMAD3 and ORMDL3. The evolving genetic architecture of IBD has furthered our understanding of disease pathogenesis. For Crohn&#39;s disease, defective processing of intracellular bacteria has become a central theme, following gene discoveries in autophagy and innate immunity (associations with NOD2, IRGM, ATG16L1 are specific to Crohn&#39;s disease). Genetic evidence has also demonstrated the importance of barrier function to the development of ulcerative colitis (HNF4A, LAMB1, CDH1 and GNA12). However, when the data are analysed in more detail, deeper themes emerge including the shared susceptibility seen with other diseases. Many immune-mediated diseases overlap in this respect, paralleling the reported epidemiological evidence. However, in several cases the reported shared susceptibility appears at odds with the clinical picture. Examples include both type 1 and type 2 diabetes mellitus. In this review we will detail the presently available data on the genetic overlap between IBD and other diseases. The discussion will be informed by the epidemiological data in the published literature and the implications for pathogenesis and therapy will be outlined. This arena will move forwards very quickly in the next few years. Ultimately, we anticipate that these genetic insights will transform the landscape of common complex diseases such as IBD.</p>
<p class="publication"><span class="title">Gut</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">12</span>;<span class="pagerange">1739-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21300624" rel="external">21300624</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fgut.2009.199679" rel="external">10.1136/gut.2009.199679</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ITFoM - The IT future of medicine</h4>
<p class="authors"><span class="author">Lehrach,H.</span>, <span class="author">Subrak,R.</span>, <span class="author">Boyle,P.</span>, <span class="author">Pasterk,M.</span>, <span class="author">Zatloukal,K.</span>, <span class="author">Muller, H.</span>, <span class="author">HUBBARD,T.</span>, <span class="author">Brand,A.</span>, <span class="author">Girolami,M.</span>, <span class="author">Jameson,D.</span>, <span class="author">Bruggeman,F.J.</span> and <span class="author">Westerhoff,H.V.</span></p>
<p class="publication"><span class="title">Procedia Computer Science</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="pagerange">26-9</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2Fj.procs.2011.12.012" rel="external">10.1016/j.procs.2011.12.012</a>; URL: <a href="http%3A%2F%2Fdx.doi.org%2F10.1016%2Fj.procs.2011.12.012" rel="external">http://dx.doi.org/10.1016/j.procs.2011.12.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Q8IYL2 is a candidate gene for the familial epilepsy syndrome of Partial Epilepsy with Pericentral Spikes (PEPS).</h4>
<p class="authors"><span class="author">Leschziner GD</span>, <span class="author">Coffey AJ</span>, <span class="author">Andrew T</span>, <span class="author">Gregorio SP</span>, <span class="author">Dias-Neto E</span>, <span class="author">Calafato M</span>, <span class="author">Bentley DR</span>, <span class="author">Kinton L</span>, <span class="author">Sander JW</span> and <span class="author">Johnson MR</span></p>
<p class="affiliation">Division of Neuroscience, Imperial College London, UK; Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%67%75%79%6C%65%73%63%68%40%67%6D%61%69%6C.%63%6F%6D">&#103;&#117;&#121;&#108;&#101;&#115;&#99;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Purpose</strong>: Partial Epilepsy with Pericentral Spikes (PEPS) is a novel Mendelian idiopathic epilepsy with evidence of linkage to Chromosome 4p15. Our aim was to identify the causative mutation in this epilepsy syndrome.</p><p class="abstract"><strong>Methods</strong>: We re-annotated all 42 genes in the linked chromosomal region and sequenced all genes within the linked interval. All exons, intron-exon boundaries and untranslated regions were sequenced in the original pedigree, and novel changes segregating correctly were subjected to bioinformatic analysis. Quantitative polymerase chain reaction was performed to examine for potential copy number variation (CNV).</p><p class="abstract"><strong>Results</strong>: 29 previously undescribed variants correctly segregating with the linked haplotype were identified. Bioinformatic analysis demonstrated that six variants were non-synonymous coding sequence polymorphisms, one of which, in Q8IYL2 (Gly400Ala), was found in neither Caucasian (n=243) and ancestry-matched Brazilian (n=180) control samples, nor subjects from the 1000 Genome Project. No gene duplications or deletions were identified in the linked region.</p><p class="abstract"><strong>Discussion</strong>: We postulate that Q8IYL2 is a causative gene for PEPS, after exhaustive resequencing and bioinformatic analysis. The function of this gene is unknown, but it is expressed in brain tissue.</p>
<p class="publication"><span class="title">Epilepsy research</span>&nbsp;<span class="year">2011</span>;<span class="volume">96</span>;<span class="number">1-2</span>;<span class="pagerange">109-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21658913">21658913</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.eplepsyres.2011.05.010">10.1016/j.eplepsyres.2011.05.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SET nuclear oncogene associates with microcephalin/MCPH1 and regulates chromosome condensation.</h4>
<p class="authors"><span class="author">Leung JW</span>, <span class="author">Leitch A</span>, <span class="author">Wood JL</span>, <span class="author">Shaw-Smith C</span>, <span class="author">Metcalfe K</span>, <span class="author">Bicknell LS</span>, <span class="author">Jackson AP</span> and <span class="author">Chen J</span></p>
<p class="affiliation">Department of Experimental Radiation Oncology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><p class="abstract">Primary microcephaly is an autosomal recessive disorder characterized by marked reduction in human brain size. Microcephalin (MCPH1), one of the genes mutated in primary microcephaly, plays an important role in DNA damage checkpoint control and mitotic entry. Additionally, MCPH1 ensures the proper temporal activation of chromosome condensation during mitosis, by acting as a negative regulator of the condensin II complex. We previously found that deletion of the of the MCPH1 N terminus leads to the premature chromosome condensation (PCC) phenotype. In the present study, we unexpectedly observed that a truncated form of MCPH1 appears to be expressed in MCPH1(S25X/S25X) patient cells. This likely results from utilization of an alternative translational start codon, which would produce a mutant MCPH1 protein with a small deletion of its N-terminal BRCT domain. Furthermore, missense mutations in the MCPH1 cluster at its N terminus, suggesting that intact function of this BRCT protein-interaction domain is required both for coordinating chromosome condensation and human brain development. Subsequently, we identified the SET nuclear oncogene as a direct binding partner of the MCPH1 N-terminal BRCT domain. Cells with SET knockdown exhibited abnormal condensed chromosomes similar to those observed in MCPH1-deficient mouse embryonic fibroblasts. Condensin II knockdown rescued the abnormal chromosome condensation phenotype in SET-depleted cells. In addition, MCPH1 V50G/I51V missense mutations, impair binding to SET and fail to fully rescue the abnormal chromosome condensation phenotype in Mcph1(-/-) mouse embryonic fibroblasts. Collectively, our findings suggest that SET is an important regulator of chromosome condensation/decondensation and that disruption of the MCPH1-SET interaction might be important for the pathogenesis of primary microcephaly.</p>
<p class="grants">Funded by: Medical Research Council; NCI NIH HHS: <span class="grant">CA089239</span>, <span class="grant">CA092312</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2011</span>;<span class="volume">286</span>;<span class="number">24</span>;<span class="pagerange">21393-400</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21515671" rel="external">21515671</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3122199" rel="external">3122199</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M110.208793" rel="external">10.1074/jbc.M110.208793</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inference of human population history from individual whole-genome sequences.</h4>
<p class="authors"><span class="author">Li H</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The history of human population size is important for understanding human evolution. Various studies have found evidence for a founder event (bottleneck) in East Asian and European populations, associated with the human dispersal out-of-Africa event around 60 thousand years (kyr) ago. However, these studies have had to assume simplified demographic models with few parameters, and they do not provide a precise date for the start and stop times of the bottleneck. Here, with fewer assumptions on population size changes, we present a more detailed history of human population sizes between approximately ten thousand and a million years ago, using the pairwise sequentially Markovian coalescent model applied to the complete diploid genome sequences of a Chinese male (YH), a Korean male (SJK), three European individuals (J. C. Venter, NA12891 and NA12878 (ref. 9)) and two Yoruba males (NA18507 (ref. 10) and NA19239). We infer that European and Chinese populations had very similar population-size histories before 10-20&thinsp;kyr ago. Both populations experienced a severe bottleneck 10-60&thinsp;kyr ago, whereas African populations experienced a milder bottleneck from which they recovered earlier. All three populations have an elevated effective population size between 60 and 250&thinsp;kyr ago, possibly due to population substructure. We also infer that the differentiation of genetically modern humans may have started as early as 100-120&thinsp;kyr ago, but considerable genetic exchanges may still have occurred until 20-40&thinsp;kyr ago.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077192</span>, <span class="grant">WT077192</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">475</span>;<span class="number">7357</span>;<span class="pagerange">493-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21753753">21753753</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3154645">3154645</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10231">10.1038/nature10231</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A combined analysis of genome-wide association studies in breast cancer.</h4>
<p class="authors"><span class="author">Li J</span>, <span class="author">Humphreys K</span>, <span class="author">Heikkinen T</span>, <span class="author">Aittom&auml;ki K</span>, <span class="author">Blomqvist C</span>, <span class="author">Pharoah PD</span>, <span class="author">Dunning AM</span>, <span class="author">Ahmed S</span>, <span class="author">Hooning MJ</span>, <span class="author">Martens JW</span>, <span class="author">van den Ouweland AM</span>, <span class="author">Alfredsson L</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen-Palotie L</span>, <span class="author">Irwanto A</span>, <span class="author">Low HQ</span>, <span class="author">Teoh GH</span>, <span class="author">Thalamuthu A</span>, <span class="author">Easton DF</span>, <span class="author">Nevanlinna H</span>, <span class="author">Liu J</span>, <span class="author">Czene K</span> and <span class="author">Hall P</span></p>
<p class="affiliation">Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, PO Box 281, 17177 Stockholm, Sweden.</p><p class="abstract">In an attempt to identify common disease susceptibility alleles for breast cancer, we performed a combined analysis of three genome-wide association studies (GWAS), involving 2,702 women of European ancestry with invasive breast cancer and 5,726 controls. Tests for association were performed for 285,984 SNPs. Evidence for association with SNPs in genes in specific pathways was assessed using a permutation-based approach. We confirmed associations with loci reported by previous GWAS on 1p11.2, 2q35, 3p, 5p12, 8q24, 10q23.13, 14q24.1 and 16q. Six SNPs with the strongest signals of association with breast cancer, and which have not been reported previously, were typed in two further studies; however, none of the associations could be confirmed. Suggestive evidence for an excess of associations was found for genes involved in the regulation of actin cytoskeleton, glycan degradation, alpha-linolenic acid metabolism, circadian rhythm, hematopoietic cell lineage and drug metabolism. Androgen and oestrogen metabolism, a pathway previously found to be associated with the development of postmenopausal breast cancer, was marginally significant (P&nbsp;=&nbsp;0.051 [unadjusted]). These results suggest that further analysis of SNPs in these pathways may identify associations that would be difficult to detect through agnostic single SNP analyses. More effort focused in these aspects of oncology can potentially open up promising avenues for the understanding of breast cancer and its prevention.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A10119</span>, <span class="grant">A10124</span>; NCI NIH HHS: <span class="grant">R01 CA58427</span></p>
<p class="publication"><span class="title">Breast cancer research and treatment</span>&nbsp;<span class="year">2011</span>;<span class="volume">126</span>;<span class="number">3</span>;<span class="pagerange">717-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20872241">20872241</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10549-010-1172-9">10.1007/s10549-010-1172-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Crafting rat genomes with zinc fingers.</h4>
<p class="authors"><span class="author">Li MA</span> and <span class="author">Bradley A</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077187</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2011</span>;<span class="volume">29</span>;<span class="number">1</span>;<span class="pagerange">39-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21221099" rel="external">21221099</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3024535" rel="external">3024535</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.1749" rel="external">10.1038/nbt.1749</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mobilization of giant piggyBac transposons in the mouse genome.</h4>
<p class="authors"><span class="author">Li MA</span>, <span class="author">Turner DJ</span>, <span class="author">Ning Z</span>, <span class="author">Yusa K</span>, <span class="author">Liang Q</span>, <span class="author">Eckert S</span>, <span class="author">Rad L</span>, <span class="author">Fitzgerald TW</span>, <span class="author">Craig NL</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, CB10 1SA.</p><p class="abstract">The development of technologies that allow the stable delivery of large genomic DNA fragments in mammalian systems is important for genetic studies as well as for applications in gene therapy. DNA transposons have emerged as flexible and efficient molecular vehicles to mediate stable cargo transfer. However, the ability to carry DNA fragments &gt;10 kb is limited in most DNA transposons. Here, we show that the DNA transposon piggyBac can mobilize 100-kb DNA fragments in mouse embryonic stem (ES) cells, making it the only known transposon with such a large cargo capacity. The integrity of the cargo is maintained during transposition, the copy number can be controlled and the inserted giant transposons express the genomic cargo. Furthermore, these 100-kb transposons can also be excised from the genome without leaving a footprint. The development of piggyBac as a large cargo vector will facilitate a wider range of genetic and genomic applications.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">WT077187</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">22</span>;<span class="pagerange">e148</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21948799">21948799</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3239208">3239208</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr764">10.1093/nar/gkr764</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Zebrafish Fukutin family proteins link the unfolded protein response with dystroglycanopathies.</h4>
<p class="authors"><span class="author">Lin YY</span>, <span class="author">White RJ</span>, <span class="author">Torelli S</span>, <span class="author">Cirak S</span>, <span class="author">Muntoni F</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Allelic mutations in putative glycosyltransferase genes, fukutin and fukutin-related protein (fkrp), lead to a wide range of muscular dystrophies associated with hypoglycosylation of &alpha;-dystroglycan, commonly referred to as dystroglycanopathies. Defective glycosylation affecting dystroglycan-ligand interactions is considered to underlie the disease pathogenesis. We have modelled dystroglycanopathies in zebrafish using a novel loss-of-function dystroglycan allele and by inhibition of Fukutin family protein activities. We show that muscle pathology in embryos lacking Fukutin or FKRP is different from loss of dystroglycan. In addition to hypoglycosylated &alpha;-dystroglycan, knockdown of Fukutin or FKRP leads to a notochord defect and a perturbation of laminin expression before muscle degeneration. These are a consequence of endoplasmic reticulum stress and activation of the unfolded protein response (UPR), preceding loss of dystroglycan-ligand interactions. Together, our results suggest that Fukutin family proteins may play important roles in protein secretion and that the UPR may contribute to the phenotypic spectrum of some dystroglycanopathies in humans.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0502130</span>, <span class="grant">G0601943</span>; Wellcome Trust: <span class="grant">077037/Z/05/Z</span>, <span class="grant">077047/Z/05/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">9</span>;<span class="pagerange">1763-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21317159">21317159</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3071672">3071672</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr059">10.1093/hmg/ddr059</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Knime4Bio: a set of custom nodes for the interpretation of next-generation sequencing data with KNIME.</h4>
<p class="authors"><span class="author">Lindenbaum P</span>, <span class="author">Le Scouarnec S</span>, <span class="author">Portero V</span> and <span class="author">Redon R</span></p>
<p class="affiliation">Institut du thorax, Inserm UMR 915, Centre Hospitalier Universitaire de Nantes, 44000 Nantes, France.</p><p class="abstract">SUMMARY: Analysing large amounts of data generated by next-generation sequencing (NGS) technologies is difficult for researchers or clinicians without computational skills. They are often compelled to delegate this task to computer biologists working with command line utilities. The availability of easy-to-use tools will become essential with the generalization of NGS in research and diagnosis. It will enable investigators to handle much more of the analysis. Here, we describe Knime4Bio, a set of custom nodes for the KNIME (The Konstanz Information Miner) interactive graphical workbench, for the interpretation of large biological datasets. We demonstrate that this tool can be utilized to quickly retrieve previously published scientific findings.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">22</span>;<span class="pagerange">3200-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21984761" rel="external">21984761</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3208396" rel="external">3208396</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr554" rel="external">10.1093/bioinformatics/btr554</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Stella-Cre mice are highly efficient Cre deleters.</h4>
<p class="authors"><span class="author">Liu H</span>, <span class="author">Wang W</span>, <span class="author">Chew SK</span>, <span class="author">Lee SC</span>, <span class="author">Li J</span>, <span class="author">Vassiliou GS</span>, <span class="author">Green T</span>, <span class="author">Futreal PA</span>, <span class="author">Bradley A</span>, <span class="author">Zhang S</span> and <span class="author">Liu P</span></p>
<p class="affiliation">College of Animal Science and Technology, Huazhong Agriculture University, Wuhan, China.</p><p class="abstract">Cre-loxP recombination is widely used for genetic manipulation of the mouse genome. Here, we report generation and characterization of a new Cre line, Stella-Cre, where Cre expression cassette was targeted to the 3&#39; UTR of the Stella locus. Stella is specifically expressed in preimplantation embryos and in the germline. Cre-loxP recombination efficiency in Stella-Cre mice was investigated at several genomic loci including Rosa26, Jak2, and Npm1. At all the loci examined, we observed 100% Cre-loxP recombination efficiency in the embryos and in the germline. Thus, Stella-Cre mice serve as a very efficient deleter line.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2011</span>;<span class="volume">49</span>;<span class="number">8</span>;<span class="pagerange">689-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21786403">21786403</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3674523">3674523</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvg.20741">10.1002/dvg.20741</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative and demographic analysis of orang-utan genomes.</h4>
<p class="authors"><span class="author">Locke DP</span>, <span class="author">Hillier LW</span>, <span class="author">Warren WC</span>, <span class="author">Worley KC</span>, <span class="author">Nazareth LV</span>, <span class="author">Muzny DM</span>, <span class="author">Yang SP</span>, <span class="author">Wang Z</span>, <span class="author">Chinwalla AT</span>, <span class="author">Minx P</span>, <span class="author">Mitreva M</span>, <span class="author">Cook L</span>, <span class="author">Delehaunty KD</span>, <span class="author">Fronick C</span>, <span class="author">Schmidt H</span>, <span class="author">Fulton LA</span>, <span class="author">Fulton RS</span>, <span class="author">Nelson JO</span>, <span class="author">Magrini V</span>, <span class="author">Pohl C</span>, <span class="author">Graves TA</span>, <span class="author">Markovic C</span>, <span class="author">Cree A</span>, <span class="author">Dinh HH</span>, <span class="author">Hume J</span>, <span class="author">Kovar CL</span>, <span class="author">Fowler GR</span>, <span class="author">Lunter G</span>, <span class="author">Meader S</span>, <span class="author">Heger A</span>, <span class="author">Ponting CP</span>, <span class="author">Marques-Bonet T</span>, <span class="author">Alkan C</span>, <span class="author">Chen L</span>, <span class="author">Cheng Z</span>, <span class="author">Kidd JM</span>, <span class="author">Eichler EE</span>, <span class="author">White S</span>, <span class="author">Searle S</span>, <span class="author">Vilella AJ</span>, <span class="author">Chen Y</span>, <span class="author">Flicek P</span>, <span class="author">Ma J</span>, <span class="author">Raney B</span>, <span class="author">Suh B</span>, <span class="author">Burhans R</span>, <span class="author">Herrero J</span>, <span class="author">Haussler D</span>, <span class="author">Faria R</span>, <span class="author">Fernando O</span>, <span class="author">Darr&eacute; F</span>, <span class="author">Farr&eacute; D</span>, <span class="author">Gazave E</span>, <span class="author">Oliva M</span>, <span class="author">Navarro A</span>, <span class="author">Roberto R</span>, <span class="author">Capozzi O</span>, <span class="author">Archidiacono N</span>, <span class="author">Della Valle G</span>, <span class="author">Purgato S</span>, <span class="author">Rocchi M</span>, <span class="author">Konkel MK</span>, <span class="author">Walker JA</span>, <span class="author">Ullmer B</span>, <span class="author">Batzer MA</span>, <span class="author">Smit AF</span>, <span class="author">Hubley R</span>, <span class="author">Casola C</span>, <span class="author">Schrider DR</span>, <span class="author">Hahn MW</span>, <span class="author">Quesada V</span>, <span class="author">Puente XS</span>, <span class="author">Ordo&ntilde;ez GR</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span>, <span class="author">Vinar T</span>, <span class="author">Brejova B</span>, <span class="author">Ratan A</span>, <span class="author">Harris RS</span>, <span class="author">Miller W</span>, <span class="author">Kosiol C</span>, <span class="author">Lawson HA</span>, <span class="author">Taliwal V</span>, <span class="author">Martins AL</span>, <span class="author">Siepel A</span>, <span class="author">Roychoudhury A</span>, <span class="author">Ma X</span>, <span class="author">Degenhardt J</span>, <span class="author">Bustamante CD</span>, <span class="author">Gutenkunst RN</span>, <span class="author">Mailund T</span>, <span class="author">Dutheil JY</span>, <span class="author">Hobolth A</span>, <span class="author">Schierup MH</span>, <span class="author">Ryder OA</span>, <span class="author">Yoshinaga Y</span>, <span class="author">de Jong PJ</span>, <span class="author">Weinstock GM</span>, <span class="author">Rogers J</span>, <span class="author">Mardis ER</span>, <span class="author">Gibbs RA</span> and <span class="author">Wilson RK</span></p>
<p class="affiliation">The Genome Center at Washington University, Washington University School of Medicine, 4444 Forest Park Avenue, Saint Louis, Missouri 63108, USA. <a href="mailto:%64%6C%6F%63%6B%65%40%77%75%73%74%6C.%65%64%75">&#100;&#108;&#111;&#99;&#107;&#101;&#64;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">&#39;Orang-utan&#39; is derived from a Malay term meaning &#39;man of the forest&#39; and aptly describes the southeast Asian great apes native to Sumatra and Borneo. The orang-utan species, Pongo abelii (Sumatran) and Pongo pygmaeus (Bornean), are the most phylogenetically distant great apes from humans, thereby providing an informative perspective on hominid evolution. Here we present a Sumatran orang-utan draft genome assembly and short read sequence data from five Sumatran and five Bornean orang-utan genomes. Our analyses reveal that, compared to other primates, the orang-utan genome has many unique features. Structural evolution of the orang-utan genome has proceeded much more slowly than other great apes, evidenced by fewer rearrangements, less segmental duplication, a lower rate of gene family turnover and surprisingly quiescent Alu repeats, which have played a major role in restructuring other primate genomes. We also describe a primate polymorphic neocentromere, found in both Pongo species, emphasizing the gradual evolution of orang-utan genome structure. Orang-utans have extremely low energy usage for a eutherian mammal, far lower than their hominid relatives. Adding their genome to the repertoire of sequenced primates illuminates new signals of positive selection in several pathways including glycolipid metabolism. From the population perspective, both Pongo species are deeply diverse; however, Sumatran individuals possess greater diversity than their Bornean counterparts, and more species-specific variation. Our estimate of Bornean/Sumatran speciation time, 400,000&thinsp;years ago, is more recent than most previous studies and underscores the complexity of the orang-utan speciation process. Despite a smaller modern census population size, the Sumatran effective population size (N(e)) expanded exponentially relative to the ancestral N(e) after the split, while Bornean N(e) declined over the same period. Overall, the resources and analyses presented here offer new opportunities in evolutionary genomics, insights into hominid biology, and an extensive database of variation for conservation efforts.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501331</span>, <span class="grant">MC_U137761446</span>; NHGRI NIH HHS: <span class="grant">HG002238</span>, <span class="grant">HG002385</span>, <span class="grant">U54 HG003079</span>, <span class="grant">U54 HG003079-08</span>, <span class="grant">U54 HG003273</span>; NHLBI NIH HHS: <span class="grant">T32 HL091823</span>; NIA NIH HHS: <span class="grant">P01 AG022064</span>; NIGMS NIH HHS: <span class="grant">R01 GM059290</span>, <span class="grant">R01 GM59290</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">469</span>;<span class="number">7331</span>;<span class="pagerange">529-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21270892">21270892</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3060778">3060778</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09687">10.1038/nature09687</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ATMIN is required for maintenance of genomic stability and suppression of B cell lymphoma.</h4>
<p class="authors"><span class="author">Loizou JI</span>, <span class="author">Sancho R</span>, <span class="author">Kanu N</span>, <span class="author">Bolland DJ</span>, <span class="author">Yang F</span>, <span class="author">Rada C</span>, <span class="author">Corcoran AE</span> and <span class="author">Behrens A</span></p>
<p class="affiliation">Mammalian Genetics Lab, Cancer Research UK, London Research Institute, 44, Lincoln&#39;s Inn Fields, London WC2A 3LY, UK.</p><p class="abstract">Defective V(D)J rearrangement of immunoglobulin heavy or light chain (IgH or IgL) or class switch recombination (CSR) can initiate chromosomal translocations. The DNA-damage kinase ATM is required for the suppression of chromosomal translocations but ATM regulation is incompletely understood. Here, we show that mice lacking the ATM cofactor ATMIN in B cells (ATMIN(&Delta;B/&Delta;B)) have impaired ATM signaling and develop B cell lymphomas. Notably, ATMIN(&Delta;B/&Delta;B) cells exhibited defective peripheral V(D)J rearrangement and CSR, resulting in translocations involving the Igh and Igl loci, indicating that ATMIN is required for efficient repair of DNA breaks generated during somatic recombination. Thus, our results identify a role for ATMIN in regulating the maintenance of genomic stability and tumor suppression in B cells.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/E/B/0000C163</span>; Cancer Research UK; Medical Research Council: <span class="grant">MC_U105178806</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">5</span>;<span class="pagerange">587-600</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21575860">21575860</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ccr.2011.03.022">10.1016/j.ccr.2011.03.022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PoolHap: inferring haplotype frequencies from pooled samples by next generation sequencing.</h4>
<p class="authors"><span class="author">Long Q</span>, <span class="author">Jeffares DC</span>, <span class="author">Zhang Q</span>, <span class="author">Ye K</span>, <span class="author">Nizhynska V</span>, <span class="author">Ning Z</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Nordborg M</span></p>
<p class="affiliation">Gregor Mendel Institute, Vienna, Austria. <a href="mailto:%71%75%61%6E.%6C%6F%6E%67%40%67%6D%69.%6F%65%61%77.%61%63.%61%74">&#113;&#117;&#97;&#110;&#46;&#108;&#111;&#110;&#103;&#64;&#103;&#109;&#105;&#46;&#111;&#101;&#97;&#119;&#46;&#97;&#99;&#46;&#97;&#116;</a></p><p class="abstract">With the advance of next-generation sequencing (NGS) technologies, increasingly ambitious applications are becoming feasible. A particularly powerful one is the sequencing of polymorphic, pooled samples. The pool can be naturally occurring, as in the case of multiple pathogen strains in a blood sample, multiple types of cells in a cancerous tissue sample, or multiple isoforms of mRNA in a cell. In these cases, it&#39;s difficult or impossible to partition the subtypes experimentally before sequencing, and those subtype frequencies must hence be inferred. In addition, investigators may occasionally want to artificially pool the sample of a large number of individuals for reasons of cost-efficiency, e.g., when carrying out genetic mapping using bulked segregant analysis. Here we describe PoolHap, a computational tool for inferring haplotype frequencies from pooled samples when haplotypes are known. The key insight into why PoolHap works is that the large number of SNPs that come with genome-wide coverage can compensate for the uneven coverage across the genome. The performance of PoolHap is illustrated and discussed using simulated and real data. We show that PoolHap is able to accurately estimate the proportions of haplotypes with less than 2% error for 34-strain mixtures with 2X total coverage Arabidopsis thaliana whole genome polymorphism data. This method should facilitate greater biological insight into heterogeneous samples that are difficult or impossible to isolate experimentally. Software and users manual are freely available at http://arabidopsis.gmi.oeaw.ac.at/quan/poolhap/.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">e15292</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21264334">21264334</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3016441">3016441</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0015292">10.1371/journal.pone.0015292</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Usher 1B protein, MYO7A, is required for normal localization and function of the visual retinoid cycle enzyme, RPE65.</h4>
<p class="authors"><span class="author">Lopes VS</span>, <span class="author">Gibbs D</span>, <span class="author">Libby RT</span>, <span class="author">Aleman TS</span>, <span class="author">Welch DL</span>, <span class="author">Lillo C</span>, <span class="author">Jacobson SG</span>, <span class="author">Radu RA</span>, <span class="author">Steel KP</span> and <span class="author">Williams DS</span></p>
<p class="affiliation">Jules Stein Eye Institute and Department of Neurobiology, UCLA School of Medicine, University of California-Los Angeles, 200 Stein Plaza, Los Angeles, CA 90095, USA.</p><p class="abstract">Mutations in the MYO7A gene cause a deaf-blindness disorder, known as Usher syndrome 1B.&nbsp; In the retina, the majority of MYO7A is in the retinal pigmented epithelium (RPE), where many of the reactions of the visual retinoid cycle take place.&nbsp; We have observed that the retinas of Myo7a-mutant mice are resistant to acute light damage. In exploring the basis of this resistance, we found that Myo7a-mutant mice have lower levels of RPE65, the RPE isomerase that has a key role in the retinoid cycle.&nbsp; We show for the first time that RPE65 normally undergoes a light-dependent translocation to become more concentrated in the central region of the RPE cells.&nbsp; This translocation requires MYO7A, so that, in Myo7a-mutant mice, RPE65 is partly mislocalized in the light.&nbsp; RPE65 is degraded more quickly in Myo7a-mutant mice, perhaps due to its mislocalization, providing a plausible explanation for its lower levels.&nbsp; Following a 50-60% photobleach, Myo7a-mutant retinas exhibited increased all-trans-retinyl ester levels during the initial stages of dark recovery, consistent with a deficiency in RPE65 activity.&nbsp; Lastly, MYO7A and RPE65 were co-immunoprecipitated from RPE cell lysate by antibodies against either of the proteins, and the two proteins were partly colocalized, suggesting a direct or indirect interaction.&nbsp; Together, the results support a role for MYO7A in the translocation of RPE65, illustrating the involvement of a molecular motor in the spatiotemporal organization of the retinoid cycle in vision.</p>
<p class="grants">Funded by: Medical Research Council; NEI NIH HHS: <span class="grant">EY00331</span>, <span class="grant">EY07042</span>, <span class="grant">EY13203</span>, <span class="grant">P30 EY000331-45</span>, <span class="grant">R01 EY007042</span>, <span class="grant">R01 EY007042-25</span>; Wellcome Trust: <span class="grant">077189</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">13</span>;<span class="pagerange">2560-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21493626" rel="external">21493626</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3110002" rel="external">3110002</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr155" rel="external">10.1093/hmg/ddr155</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ADAM-15 disintegrin-like domain structure and function</h4>
<p class="authors"><span class="author">LU D</span>, <span class="author">Scully M</span>, <span class="author">Kakkar V</span>, <span class="author">Lu X</span></p>
<p class="publication"><span class="title">Toxins.</span> <span class="year">2011</span>;<span class="volume">2</span>;<span class="pagerange">2411-27</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome association scan for genetic polymorphisms influencing information processing speed.</h4>
<p class="authors"><span class="author">Luciano M</span>, <span class="author">Hansell NK</span>, <span class="author">Lahti J</span>, <span class="author">Davies G</span>, <span class="author">Medland SE</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Tenesa A</span>, <span class="author">Widen E</span>, <span class="author">McGhee KA</span>, <span class="author">Palotie A</span>, <span class="author">Liewald D</span>, <span class="author">Porteous DJ</span>, <span class="author">Starr JM</span>, <span class="author">Montgomery GW</span>, <span class="author">Martin NG</span>, <span class="author">Eriksson JG</span>, <span class="author">Wright MJ</span> and <span class="author">Deary IJ</span></p>
<p class="affiliation">Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK. <a href="mailto:%6D%69%63%68%65%6C%6C%65.%6C%75%63%69%61%6E%6F%40%65%64.%61%63.%75%6B">&#109;&#105;&#99;&#104;&#101;&#108;&#108;&#101;&#46;&#108;&#117;&#99;&#105;&#97;&#110;&#111;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Processing speed is an important cognitive function that is compromised in psychiatric illness (e.g., schizophrenia, depression) and old age; it shares genetic background with complex cognition (e.g., working memory, reasoning). To find genes influencing speed we performed a genome-wide association scan in up to three cohorts: Brisbane (mean age 16 years; N = 1659); LBC1936 (mean age 70 years, N = 992); LBC1921 (mean age 82 years, N = 307), and; HBCS (mean age 64 years, N =1080). Meta-analysis of the common measures highlighted various suggestively significant (p &lt; 1.21 &times; 10&#x207B;&#x2075;) SNPs and plausible candidate genes (e.g., TRIB3). A biological pathways analysis of the speed factor identified two common pathways from the KEGG database (cell junction, focal adhesion) in two cohorts, while a pathway analysis linked to the GO database revealed common pathways across pairs of speed measures (e.g., receptor binding, cellular metabolic process). These highlighted genes and pathways will be able to inform future research, including results for psychiatric disease.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust: <span class="grant">089062</span>, <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">Biological psychology</span>&nbsp;<span class="year">2011</span>;<span class="volume">86</span>;<span class="number">3</span>;<span class="pagerange">193-202</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21130836" rel="external">21130836</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3319015" rel="external">3319015</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsycho.2010.11.008" rel="external">10.1016/j.biopsycho.2010.11.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A large palindrome with interchromosomal gene duplications in the pericentromeric region of the D. melanogaster Y chromosome.</h4>
<p class="authors"><span class="author">M&eacute;ndez-Lago M</span>, <span class="author">Bergman CM</span>, <span class="author">de Pablos B</span>, <span class="author">Tracey A</span>, <span class="author">Whitehead SL</span> and <span class="author">Villasante A</span></p>
<p class="affiliation">Centro de Biolog&iacute;a Molecular Severo Ochoa (CSIC-UAM), Universidad Aut&oacute;noma de Madrid, Cantoblanco, Madrid, Spain.</p><p class="abstract">The non-recombining Y chromosome is expected to degenerate over evolutionary time, however, gene gain is a common feature of Y chromosomes of mammals and Drosophila. Here, we report that a large palindrome containing interchromosomal segmental duplications is located in the vicinity of the first amplicon detected in the Y chromosome of D. melanogaster. The recent appearance of such amplicons suggests that duplications to the Y chromosome, followed by the amplification of the segmental duplications, are a mechanism for the continuing evolution of Drosophila Y chromosomes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">28</span>;<span class="number">7</span>;<span class="pagerange">1967-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21297157">21297157</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4202384">4202384</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsr034">10.1093/molbev/msr034</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Knockdown of mental disorder susceptibility genes disrupts neuronal network physiology in vitro.</h4>
<p class="authors"><span class="author">MacLaren EJ</span>, <span class="author">Charlesworth P</span>, <span class="author">Coba MP</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB101SA, UK.</p><p class="abstract">Schizophrenia and bipolar disorder are common diseases caused by multiple genes that disrupt brain circuits. While great progress has been made in identifying schizophrenia susceptibility genes, these studies have left two major unanswered mechanistic questions: is there a core biochemical mechanism that these genes regulate, and what are the electrophysiological consequences of the altered gene expression? Because clinical studies implicate abnormalities in neuronal networks, we developed a system for studying the neurophysiology of neuronal networks in vitro where the role of candidate disease genes can be rapidly assayed. Using this system we focused on three postsynaptic proteins DISC1, TNIK and PSD-93/DLG2 each of which is encoded by a schizophrenia susceptibility gene. We also examined the utility of this assay system in bipolar disorder (BD), which has a strong genetic overlap with schizophrenia, by examining the bipolar disorder susceptibility gene Dctn5. The global neuronal network firing behavior of primary cultures of mouse hippocampus neurons was examined on multi-electrode arrays (MEAs) and genes of interest were knocked down using RNAi interference. Measurement of multiple neural network parameters demonstrated phenotypes for these genes compared with controls. Moreover, the different genes disrupted network properties and showed distinct and overlapping effects. These data show multiple susceptibility genes for complex psychiatric disorders, regulate neural network physiology and demonstrate a new assay system with wide application.</p>
<p class="publication"><span class="title">Molecular and cellular neurosciences</span>&nbsp;<span class="year">2011</span>;<span class="volume">47</span>;<span class="number">2</span>;<span class="pagerange">93-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21440632" rel="external">21440632</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3105225" rel="external">3105225</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mcn.2010.12.014" rel="external">10.1016/j.mcn.2010.12.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clinical significance of SF3B1 mutations in myelodysplastic syndromes and myelodysplastic/myeloproliferative neoplasms.</h4>
<p class="authors"><span class="author">Malcovati L</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Bowen DT</span>, <span class="author">Boultwood J</span>, <span class="author">Della Porta MG</span>, <span class="author">Pascutto C</span>, <span class="author">Travaglino E</span>, <span class="author">Groves MJ</span>, <span class="author">Godfrey AL</span>, <span class="author">Ambaglio I</span>, <span class="author">Gall&igrave; A</span>, <span class="author">Da Vi&agrave; MC</span>, <span class="author">Conte S</span>, <span class="author">Tauro S</span>, <span class="author">Keenan N</span>, <span class="author">Hyslop A</span>, <span class="author">Hinton J</span>, <span class="author">Mudie LJ</span>, <span class="author">Wainscoat JS</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span>, <span class="author">Hellstr&ouml;m-Lindberg E</span>, <span class="author">Cazzola M</span> and <span class="author">Chronic Myeloid Disorders Working Group of the International Cancer Genome Consortium and of the Associazione Italiana per la Ricerca sul Cancro Gruppo Italiano Malattie Mieloproliferative</span></p>
<p class="affiliation">Department of Hematology Oncology, Fondazione Istituto di Ricovero e Cura a Carattere Scientifico Policlinico San Matteo and University of Pavia, Pavia, Italy.</p><p class="abstract">In a previous study, we identified somatic mutations of SF3B1, a gene encoding a core component of RNA splicing machinery, in patients with myelodysplastic syndrome (MDS). Here, we define the clinical significance of these mutations in MDS and myelodysplastic/myeloproliferative neoplasms (MDS/MPN). The coding exons of SF3B1 were screened using massively parallel pyrosequencing in patients with MDS, MDS/MPN, or acute myeloid leukemia (AML) evolving from MDS. Somatic mutations of SF3B1 were found in 150 of 533 (28.1%) patients with MDS, 16 of 83 (19.3%) with MDS/MPN, and 2 of 38 (5.3%) with AML. There was a significant association of SF3B1 mutations with the presence of ring sideroblasts (P &lt; .001) and of mutant allele burden with their proportion (P = .002). The mutant gene had a positive predictive value for ring sideroblasts of 97.7% (95% confidence interval, 93.5%-99.5%). In multivariate analysis including established risk factors, SF3B1 mutations were found to be independently associated with better overall survival (hazard ratio = 0.15, P = .025) and lower risk of evolution into AML (hazard ratio = 0.33, P = .049). The close association between SF3B1 mutations and disease phenotype with ring sideroblasts across MDS and MDS/MPN is consistent with a causal relationship. Furthermore, SF3B1 mutations are independent predictors of favorable clinical outcome, and their incorporation into stratification systems might improve risk assessment in MDS.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2011</span>;<span class="volume">118</span>;<span class="number">24</span>;<span class="pagerange">6239-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21998214" rel="external">21998214</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3236114" rel="external">3236114</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2011-09-377275" rel="external">10.1182/blood-2011-09-377275</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A research agenda for malaria eradication: basic science and enabling technologies.</h4>
<p class="authors"><span class="author">malERA Consultative Group on Basic Science and Enabling Technologies</span></p><p class="abstract">Today&#39;s malaria control efforts are limited by our incomplete understanding of the biology of Plasmodium and of the complex relationships between human populations and the multiple species of mosquito and parasite. Research priorities include the development of in vitro culture systems for the complete life cycle of P. falciparum and P. vivax and the development of an appropriate liver culture system to study hepatic stages. In addition, genetic technologies for the manipulation of Plasmodium need to be improved, the entire parasite metabolome needs to be characterized to identify new druggable targets, and improved information systems for monitoring the changes in epidemiology, pathology, and host-parasite-vector interactions as a result of intensified control need to be established to bridge the gap between bench, preclinical, clinical, and population-based sciences.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span></p>
<p class="publication"><span class="title">PLoS medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">e1000399</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21311584">21311584</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3026698">3026698</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pmed.1000399">10.1371/journal.pmed.1000399</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Low-bias, strand-specific transcriptome Illumina sequencing by on-flowcell reverse transcription (FRT-seq).</h4>
<p class="authors"><span class="author">Mamanova L</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%6C%6D%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#109;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The unifying feature of second-generation sequencing technologies is that single template strands are amplified clonally onto a solid surface prior to the sequencing reaction. To convert template strands into a compatible state for attachment to this surface, a multistep library preparation is required, which typically culminates in amplification by the PCR. PCR is an inherently biased process, which decreases the efficiency of data acquisition. Flowcell reverse transcription sequencing is a method of transcriptome sequencing for Illumina sequencers in which the reverse transcription reaction is performed on the flowcell by using unamplified, adapter-ligated mRNA as a template. This approach removes PCR biases and duplicates, generates strand-specific paired-end data and is highly reproducible. The procedure can be performed quickly, taking 2 d to generate clusters from mRNA.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">1736-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22015844">22015844</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2011.399">10.1038/nprot.2011.399</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">APC15 drives the turnover of MCC-CDC20 to make the spindle assembly checkpoint responsive to kinetochore attachment.</h4>
<p class="authors"><span class="author">Mansfeld J</span>, <span class="author">Collin P</span>, <span class="author">Collins MO</span>, <span class="author">Choudhary JS</span> and <span class="author">Pines J</span></p>
<p class="affiliation">The Gurdon Institute and Department of Zoology, Tennis Court Road, Cambridge CB2 1QN, UK.</p><p class="abstract">Faithful chromosome segregation during mitosis depends on the spindle assembly checkpoint (SAC), which monitors kinetochore attachment to the mitotic spindle. Unattached kinetochores generate mitotic checkpoint proteins complexes (MCCs) that bind and inhibit the anaphase-promoting complex, or cyclosome (APC/C). How the SAC proficiently inhibits the APC/C but still allows its rapid activation when the last kinetochore attaches to the spindle is important for the understanding of how cells maintain genomic stability. We show that the APC/C subunit APC15 is required for the turnover of the APC/C co-activator CDC20 and release of MCCs during SAC signalling but not for APC/C activity per se. In the absence of APC15, MCCs and ubiquitylated CDC20 remain &#39;locked&#39; onto the APC/C, which prevents the ubiquitylation and degradation of cyclin B1 when the SAC is satisfied. We conclude that APC15 mediates the constant turnover of CDC20 and MCCs on the APC/C to allow the SAC to respond to the attachment state of kinetochores.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/G001537/1</span>; Cancer Research UK: <span class="grant">A3211</span>; Wellcome Trust: <span class="grant">079643/Z/06/Z</span></p>
<p class="publication"><span class="title">Nature cell biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">13</span>;<span class="number">10</span>;<span class="pagerange">1234-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21926987">21926987</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3188299">3188299</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncb2347">10.1038/ncb2347</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insertional mutagenesis identifies multiple networks of cooperating genes driving intestinal tumorigenesis.</h4>
<p class="authors"><span class="author">March HN</span>, <span class="author">Rust AG</span>, <span class="author">Wright NA</span>, <span class="author">ten Hoeve J</span>, <span class="author">de Ridder J</span>, <span class="author">Eldridge M</span>, <span class="author">van der Weyden L</span>, <span class="author">Berns A</span>, <span class="author">Gadiot J</span>, <span class="author">Uren A</span>, <span class="author">Kemp R</span>, <span class="author">Arends MJ</span>, <span class="author">Wessels LF</span>, <span class="author">Winton DJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre, UK.</p><p class="abstract">The evolution of colorectal cancer suggests the involvement of many genes. To identify new drivers of intestinal cancer, we performed insertional mutagenesis using the Sleeping Beauty transposon system in mice carrying germline or somatic Apc mutations. By analyzing common insertion sites (CISs) isolated from 446 tumors, we identified many hundreds of candidate cancer drivers. Comparison to human data sets suggested that 234 CIS-targeted genes are also dysregulated in human colorectal cancers. In addition, we found 183 CIS-containing genes that are candidate Wnt targets and showed that 20 CISs-containing genes are newly discovered modifiers of canonical Wnt signaling. We also identified mutations associated with a subset of tumors containing an expanded number of Paneth cells, a hallmark of deregulated Wnt signaling, and genes associated with more severe dysplasia included those encoding members of the FGF signaling cascade. Some 70 genes had co-occurrence of CIS pairs, clustering into 38 sub-networks that may regulate tumor development.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>, <span class="grant">A6997</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">12</span>;<span class="pagerange">1202-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22057237">22057237</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3233530">3233530</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.990">10.1038/ng.990</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introducing the Human Brain Project</h4>
<p class="authors"><span class="author">Markram,H.</span>, <span class="author">Meier,K.</span>, <span class="author">Lippert,T.</span>, <span class="author">Grillner,S.</span>, <span class="author">Frackowiak,R.</span>, <span class="author">Dehaene,S.</span>, <span class="author">Knoll,A.</span>, <span class="author">Sompolinsky,H.</span>, <span class="author">Verstreken,K.</span>, <span class="author">DeFelipe,J.</span>, <span class="author">GRANT,S.</span>, <span class="author">Changeux,J.-P.</span> and <span class="author">Sariam,A.</span></p>
<p class="publication"><span class="title">Procedia Computer Science</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="pagerange">39-42</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2Fj.procs.2011.12.015" rel="external">10.1016/j.procs.2011.12.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced virulence of Salmonella enterica serovar typhimurium after passage through mice.</h4>
<p class="authors"><span class="author">Mastroeni P</span>, <span class="author">Morgan FJ</span>, <span class="author">McKinley TJ</span>, <span class="author">Shawcroft E</span>, <span class="author">Clare S</span>, <span class="author">Maskell DJ</span> and <span class="author">Grant AJ</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Madingley Road, Cambridge CB3 0ES, United Kingdom.</p><p class="abstract">The interaction between Salmonella enterica and the host immune system is complex. The outcome of an infection is the result of a balance between the in vivo environment where the bacteria survive and grow and the regulation of fitness genes at a level sufficient for the bacteria to retain their characteristic rate of growth in a given host. Using bacteriological counts from tissue homogenates and fluorescence microscopy to determine the spread, localization, and distribution of S. enterica in the tissues, we show that, during a systemic infection, S. enterica adapts to the in vivo environment. The adaptation becomes a measurable phenotype when bacteria that have resided in a donor animal are introduced into a recipient na&iuml;ve animal. This adaptation does not confer increased resistance to early host killing mechanisms but can be detected as an enhancement in the bacterial net growth rate later in the infection. The enhanced growth rate is lost upon a single passage in vitro, and it is therefore transient and not due to selection of mutants. The adapted bacteria on average reach higher intracellular numbers in individual infected cells and therefore have patterns of organ spread different from those of nonadapted bacteria. These experiments help in developing an understanding of the influence of passage in a host on the fitness and virulence of S. enterica.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801161</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2011</span>;<span class="volume">79</span>;<span class="number">2</span>;<span class="pagerange">636-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21098099" rel="external">21098099</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3028859" rel="external">3028859</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00954-10" rel="external">10.1128/IAI.00954-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative cytogenetic mapping of Sox2 and Sox14 in cichlid fishes and inferences on the genomic organization of both genes in vertebrates.</h4>
<p class="authors"><span class="author">Mazzuchelli J</span>, <span class="author">Yang F</span>, <span class="author">Kocher TD</span> and <span class="author">Martins C</span></p>
<p class="affiliation">Department of Morphology, Bioscience Institute, UNESP, S&atilde;o Paulo State University, 18618-000 Botucatu, S&atilde;o Paulo, Brazil.</p><p class="abstract">To better understand the genomic organization and evolution of Sox genes in vertebrates, we cytogenetically mapped Sox2 and Sox14 genes in cichlid fishes and performed comparative analyses of their orthologs in several vertebrate species. The genomic regions neighboring Sox2 and Sox14 have been conserved during vertebrate diversification. Although cichlids seem to have undergone high rates of genomic rearrangements, Sox2 and Sox14 are linked in the same chromosome in the Etroplinae Etroplus maculatus that represents the sister group of all remaining cichlids. However, these genes are located on different chromosomes in several species of the sister group Pseudocrenilabrinae. Similarly, the ancestral synteny of Sox2 and Sox14 has been maintained in several vertebrates, but this synteny has been broken independently in all major groups as a consequence of karyotype rearrangements that took place during the vertebrate evolution.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R01 HD058635-04</span></p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">5</span>;<span class="pagerange">657-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21691861" rel="external">21691861</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3190197" rel="external">3190197</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-011-9225-4" rel="external">10.1007/s10577-011-9225-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HLA-A*3101 and carbamazepine-induced hypersensitivity reactions in Europeans.</h4>
<p class="authors"><span class="author">McCormack M</span>, <span class="author">Alfirevic A</span>, <span class="author">Bourgeois S</span>, <span class="author">Farrell JJ</span>, <span class="author">Kasperavi&#x10D;i&#x16B;t&#x117; D</span>, <span class="author">Carrington M</span>, <span class="author">Sills GJ</span>, <span class="author">Marson T</span>, <span class="author">Jia X</span>, <span class="author">de Bakker PI</span>, <span class="author">Chinthapalli K</span>, <span class="author">Molokhia M</span>, <span class="author">Johnson MR</span>, <span class="author">O&#39;Connor GD</span>, <span class="author">Chaila E</span>, <span class="author">Alhusaini S</span>, <span class="author">Shianna KV</span>, <span class="author">Radtke RA</span>, <span class="author">Heinzen EL</span>, <span class="author">Walley N</span>, <span class="author">Pandolfo M</span>, <span class="author">Pichler W</span>, <span class="author">Park BK</span>, <span class="author">Depondt C</span>, <span class="author">Sisodiya SM</span>, <span class="author">Goldstein DB</span>, <span class="author">Deloukas P</span>, <span class="author">Delanty N</span>, <span class="author">Cavalleri GL</span> and <span class="author">Pirmohamed M</span></p>
<p class="affiliation">Molecular and Cellular Therapeutics, the Royal College of Surgeons in Ireland, Dublin, Ireland.</p><p class="abstract"><strong>Background</strong>: Carbamazepine causes various forms of hypersensitivity reactions, ranging from maculopapular exanthema to severe blistering reactions. The HLA-B*1502 allele has been shown to be strongly correlated with carbamazepine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis (SJS-TEN) in the Han Chinese and other Asian populations but not in European populations.</p><p class="abstract"><strong>Methods</strong>: We performed a genomewide association study of samples obtained from 22 subjects with carbamazepine-induced hypersensitivity syndrome, 43 subjects with carbamazepine-induced maculopapular exanthema, and 3987 control subjects, all of European descent. We tested for an association between disease and HLA alleles through proxy single-nucleotide polymorphisms and imputation, confirming associations by high-resolution sequence-based HLA typing. We replicated the associations in samples from 145 subjects with carbamazepine-induced hypersensitivity reactions.</p><p class="abstract"><strong>Results</strong>: The HLA-A*3101 allele, which has a prevalence of 2 to 5% in Northern European populations, was significantly associated with the hypersensitivity syndrome (P=3.5&times;10(-8)). An independent genomewide association study of samples from subjects with maculopapular exanthema also showed an association with the HLA-A*3101 allele (P=1.1&times;10(-6)). Follow-up genotyping confirmed the variant as a risk factor for the hypersensitivity syndrome (odds ratio, 12.41; 95% confidence interval [CI], 1.27 to 121.03), maculopapular exanthema (odds ratio, 8.33; 95% CI, 3.59 to 19.36), and SJS-TEN (odds ratio, 25.93; 95% CI, 4.93 to 116.18).</p><p class="abstract"><strong>Conclusions</strong>: The presence of the HLA-A*3101 allele was associated with carbamazepine-induced hypersensitivity reactions among subjects of Northern European ancestry. The presence of the allele increased the risk from 5.0% to 26.0%, whereas its absence reduced the risk from 5.0% to 3.8%. (Funded by the U.K. Department of Health and others.).</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0400126</span>; PHS HHS: <span class="grant">HHS-N261200800001E</span>, <span class="grant">HHSN261200800001E</span>; Wellcome Trust: <span class="grant">084730</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">364</span>;<span class="number">12</span>;<span class="pagerange">1134-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21428769">21428769</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3113609">3113609</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1013297">10.1056/NEJMoa1013297</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics and the continuum of cancer care.</h4>
<p class="authors"><span class="author">McDermott U</span>, <span class="author">Downing JR</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute,Hinxton, Cambridge, United Kingdom.</p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">364</span>;<span class="number">4</span>;<span class="pagerange">340-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21268726" rel="external">21268726</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMra0907178" rel="external">10.1056/NEJMra0907178</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reply: Ileal pouch microbial diversity</h4>
<p class="authors"><span class="author">McLaughlin SD</span>, <span class="author">Walker AW</span>, <span class="author">CHURCHER C</span>, <span class="author">CLARK SK</span>, <span class="author">Tekkis PP</span>, <span class="author">Johnson MW</span>, <span class="author">PARKHILL J</span>, <span class="author">CICLITIRA PJ</span>, <span class="author">DOUGAN G</span>, <span class="author">Nicholls RJ</span>, <span class="author">Petrovska L</span></p>
<p class="publication"><span class="title">Annals of Surgery.</span> <span class="year">2011</span>;<span class="volume">254</span>;<span class="pagerange">669-70</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies 12 new susceptibility loci for primary biliary cirrhosis.</h4>
<p class="authors"><span class="author">Mells GF</span>, <span class="author">Floyd JA</span>, <span class="author">Morley KI</span>, <span class="author">Cordell HJ</span>, <span class="author">Franklin CS</span>, <span class="author">Shin SY</span>, <span class="author">Heneghan MA</span>, <span class="author">Neuberger JM</span>, <span class="author">Donaldson PT</span>, <span class="author">Day DB</span>, <span class="author">Ducker SJ</span>, <span class="author">Muriithi AW</span>, <span class="author">Wheater EF</span>, <span class="author">Hammond CJ</span>, <span class="author">Dawwas MF</span>, <span class="author">UK PBC Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium 3</span>, <span class="author">Jones DE</span>, <span class="author">Peltonen L</span>, <span class="author">Alexander GJ</span>, <span class="author">Sandford RN</span> and <span class="author">Anderson CA</span></p>
<p class="affiliation">Academic Department of Medical Genetics, Cambridge University, Cambridge, UK; Department of Hepatology, Cambridge University Hospitals National Health Service (NHS) Foundation Trust, Cambridge, UK.</p><p class="abstract">In addition to the HLA locus, six genetic risk factors for primary biliary cirrhosis (PBC) have been identified in recent genome-wide association studies (GWAS). To identify additional loci, we carried out a GWAS using 1,840 cases from the UK PBC Consortium and 5,163 UK population controls as part of the Wellcome Trust Case Control Consortium 3 (WTCCC3). We followed up 28 loci in an additional UK cohort of 620 PBC cases and 2,514 population controls. We identified 12 new susceptibility loci (at a genome-wide significance level of P &lt; 5 &times; 10&#x207B;&#x2078;) and replicated all previously associated loci. We identified three further new loci in a meta-analysis of data from our study and previously published GWAS results. New candidate genes include STAT4, DENND1B, CD80, IL7R, CXCR5, TNFRSF1A, CLEC16A and NFKB1. This study has considerably expanded our knowledge of the genetic architecture of PBC.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500020</span>, <span class="grant">G0800460</span>, <span class="grant">G0802068</span>; PHS HHS: <span class="grant">1R01LEY018246</span>; Wellcome Trust: <span class="grant">085925/Z/08/Z</span>, <span class="grant">091745</span>, <span class="grant">WT090355/B/09/Z</span>, <span class="grant">WT09355A/09/Z</span>, <span class="grant">WT91745/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">4</span>;<span class="pagerange">329-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21399635">21399635</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3071550">3071550</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.789">10.1038/ng.789</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hostility in adolescents and adults: a genome-wide association study of the Young Finns.</h4>
<p class="authors"><span class="author">Merjonen P</span>, <span class="author">Keltikangas-J&auml;rvinen L</span>, <span class="author">Jokela M</span>, <span class="author">Sepp&auml;l&auml; I</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Pulkki-R&aring;back L</span>, <span class="author">Kivim&auml;ki M</span>, <span class="author">Elovainio M</span>, <span class="author">Kettunen J</span>, <span class="author">Ripatti S</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Viikari J</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen L</span>, <span class="author">Raitakari OT</span> and <span class="author">Lehtim&auml;ki T</span></p>
<p class="affiliation">IBS, Unit of Personality, Work and Health Psychology, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Hostility is a multidimensional personality trait with changing expression over the life course. We performed a genome-wide association study (GWAS) of the components of hostility in a population-based sample of Finnish men and women for whom a total of 2.5 million single-nucleotide polymorphisms (SNPs) were available through direct or in silico genotyping. Hostility dimensions (anger, cynicism and paranoia) were assessed at four time points over a 15-year interval (age range 15-30 years at phase 1 and 30-45 years at phase 4) in 982-1780 participants depending on the hostility measure. Few promising areas from chromosome 14 at 99&thinsp;cM (top SNPs rs3783337, rs7158754, rs3783332, rs2181102, rs7159195, rs11160570, rs941898, P values &lt;3.9 &times; 10(-8) with nearest gene Enah/Vasp-like (EVL)) were found suggestively to be related to paranoia and from chromosome 7 at 86&thinsp;cM (top SNPs rs802047, rs802028, rs802030, rs802026, rs802036, rs802025, rs802024, rs802032, rs802049, rs802051, P values &lt;6.9 &times; 10(-7) with nearest gene CROT (carnitine O-octanoyltransferase)) to cynicism, respectively. Some shared suggestive genetic influence for both paranoia and cynicism was also found from chromosome 17 at 2.8&thinsp;cM (SNPs rs12936442, rs894664, rs6502671, rs7216028) and chromosome 22 at 43&thinsp;cM (SNPs rs7510759, rs7510924, rs7290560), with nearest genes RAP1 GTPase activating protein 2 (RAP1GAP2) and KIAA1644, respectively. These suggestive associations did not replicate across all measurement times, which warrants further study on these SNPs in other populations.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01HL36310</span>; NIA NIH HHS: <span class="grant">R01AG013196</span>, <span class="grant">R01AG034454</span></p>
<p class="publication"><span class="title">Translational psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">1</span>;<span class="pagerange">e11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22832427">22832427</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3309462">3309462</a>; DOI: <a href="http://dx.doi.org/10.1038%2Ftp.2011.13">10.1038/tp.2011.13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genetic association between personality and major depression or bipolar disorder. A polygenic score analysis using genome-wide association data.</h4>
<p class="authors"><span class="author">Middeldorp CM</span>, <span class="author">de Moor MH</span>, <span class="author">McGrath LM</span>, <span class="author">Gordon SD</span>, <span class="author">Blackwood DH</span>, <span class="author">Costa PT</span>, <span class="author">Terracciano A</span>, <span class="author">Krueger RF</span>, <span class="author">de Geus EJ</span>, <span class="author">Nyholt DR</span>, <span class="author">Tanaka T</span>, <span class="author">Esko T</span>, <span class="author">Madden PA</span>, <span class="author">Derringer J</span>, <span class="author">Amin N</span>, <span class="author">Willemsen G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Distel MA</span>, <span class="author">Uda M</span>, <span class="author">Sanna S</span>, <span class="author">Spinhoven P</span>, <span class="author">Hartman CA</span>, <span class="author">Ripke S</span>, <span class="author">Sullivan PF</span>, <span class="author">Realo A</span>, <span class="author">Allik J</span>, <span class="author">Heath AC</span>, <span class="author">Pergadia ML</span>, <span class="author">Agrawal A</span>, <span class="author">Lin P</span>, <span class="author">Grucza RA</span>, <span class="author">Widen E</span>, <span class="author">Cousminer DL</span>, <span class="author">Eriksson JG</span>, <span class="author">Palotie A</span>, <span class="author">Barnett JH</span>, <span class="author">Lee PH</span>, <span class="author">Luciano M</span>, <span class="author">Tenesa A</span>, <span class="author">Davies G</span>, <span class="author">Lopez LM</span>, <span class="author">Hansell NK</span>, <span class="author">Medland SE</span>, <span class="author">Ferrucci L</span>, <span class="author">Schlessinger D</span>, <span class="author">Montgomery GW</span>, <span class="author">Wright MJ</span>, <span class="author">Aulchenko YS</span>, <span class="author">Janssens AC</span>, <span class="author">Oostra BA</span>, <span class="author">Metspalu A</span>, <span class="author">Abecasis GR</span>, <span class="author">Deary IJ</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Bierut LJ</span>, <span class="author">Martin NG</span>, <span class="author">Wray NR</span>, <span class="author">van Duijn CM</span>, <span class="author">Smoller JW</span>, <span class="author">Penninx BW</span> and <span class="author">Boomsma DI</span></p>
<p class="affiliation">Department of Biological Psychology, VU University Amsterdam, Amsterdam, The Netherlands. <a href="mailto:%63%6D.%6D%69%64%64%65%6C%64%6F%72%70%40%70%73%79.%76%75.%6E%6C">&#99;&#109;&#46;&#109;&#105;&#100;&#100;&#101;&#108;&#100;&#111;&#114;&#112;&#64;&#112;&#115;&#121;&#46;&#118;&#117;&#46;&#110;&#108;</a></p>
<p class="abstract">The relationship between major depressive disorder (MDD) and bipolar disorder (BD) remains controversial. Previous research has reported differences and similarities in risk factors for MDD and BD, such as predisposing personality traits. For example, high neuroticism is related to both disorders, whereas openness to experience is specific for BD. This study examined the genetic association between personality and MDD and BD by applying polygenic scores for neuroticism, extraversion, openness to experience, agreeableness and conscientiousness to both disorders. Polygenic scores reflect the weighted sum of multiple single-nucleotide polymorphism alleles associated with the trait for an individual and were based on a meta-analysis of genome-wide association studies for personality traits including 13,835 subjects. Polygenic scores were tested for MDD in the combined Genetic Association Information Network (GAIN-MDD) and MDD2000+ samples (N=8921) and for BD in the combined Systematic Treatment Enhancement Program for Bipolar Disorder and Wellcome Trust Case-Control Consortium samples (N=6329) using logistic regression analyses. At the phenotypic level, personality dimensions were associated with MDD and BD. Polygenic neuroticism scores were significantly positively associated with MDD, whereas polygenic extraversion scores were significantly positively associated with BD. The explained variance of MDD and BD, âˆ¼0.1%, was highly comparable to the variance explained by the polygenic personality scores in the corresponding personality traits themselves (between 0.1 and 0.4%). This indicates that the proportions of variance explained in mood disorders are at the upper limit of what could have been expected. This study suggests shared genetic risk factors for neuroticism and MDD on the one hand and for extraversion and BD on the other.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; NCI NIH HHS: <span class="grant">P01 CA089392</span>, <span class="grant">P01CA89392</span>; NHGRI NIH HHS: <span class="grant">U01 HG004422</span>, <span class="grant">U01 HG004446</span>, <span class="grant">U01HG004438</span>; NIA NIH HHS: <span class="grant">N01-AG-1-2109</span>; NIAAA NIH HHS: <span class="grant">AA07535</span>, <span class="grant">AA07580</span>, <span class="grant">AA07728</span>, <span class="grant">AA10248</span>, <span class="grant">AA11998</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">AA14041</span>, <span class="grant">U10 AA008401</span>, <span class="grant">U10AA008401</span>; NIDA NIH HHS: <span class="grant">DA019951</span>, <span class="grant">DA12854</span>, <span class="grant">R01 DA013423</span>, <span class="grant">R01 DA019963</span>; NIMH NIH HHS: <span class="grant">MH66206</span>, <span class="grant">R01 MH059160</span>; PHS HHS: <span class="grant">HHSN268200782096C</span>, <span class="grant">R01 079799</span>; Wellcome Trust: <span class="grant">076113</span></p>
<p class="publication"><span class="title">Translational psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">1</span>;<span class="pagerange">e50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22833196" rel="external">22833196</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3309491" rel="external">3309491</a>; DOI: <a href="http://dx.doi.org/10.1038%2Ftp.2011.45" rel="external">10.1038/tp.2011.45</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identifying schizophrenia and other psychoses with psychological scales in the general population.</h4>
<p class="authors"><span class="author">Miettunen J</span>, <span class="author">Veijola J</span>, <span class="author">Isohanni M</span>, <span class="author">Paunio T</span>, <span class="author">Freimer N</span>, <span class="author">J&auml;&auml;skel&auml;inen E</span>, <span class="author">Taanila A</span>, <span class="author">Ekelund J</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Peltonen L</span>, <span class="author">Joukamaa M</span> and <span class="author">Lichtermann D</span></p>
<p class="affiliation">Department of Psychiatry, Oulu University and Oulu University Hospital, Oulu, Finland. <a href="mailto:%6A%6F%75%6B%6F.%6D%69%65%74%74%75%6E%65%6E%40%6F%75%6C%75.%66%69">&#106;&#111;&#117;&#107;&#111;&#46;&#109;&#105;&#101;&#116;&#116;&#117;&#110;&#101;&#110;&#64;&#111;&#117;&#108;&#117;&#46;&#102;&#105;</a></p><p class="abstract">We study the predictive power and associations of several psychopathology and temperament scales with respect to schizophrenia and other psychotic disorders. Measures of psychopathology (Physical and Social Anhedonia Scales, Perceptual Aberration Scale, Hypomanic Personality Scale, Bipolar II Scale, and Schizoidia Scale) and the Temperament and Character Inventory were included in the 31-year follow-up of the prospective Northern Finland 1966 birth cohort (N = 4926). The Perceptual Aberration Scale was the best scale for concurrent validity in psychoses, and also the best psychopathology scale in terms of discriminant validity. Participants scoring high in hypomanic personality were at the highest risk for developing psychosis during the 11-year follow-up. Harm avoidance was a dominant temperament dimension in individuals with psychosis compared with participants without psychiatric diagnoses. These scales are useful as vulnerability markers in studying psychoses.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">5R01MH63706:02</span></p>
<p class="publication"><span class="title">The Journal of nervous and mental disease</span>&nbsp;<span class="year">2011</span>;<span class="volume">199</span>;<span class="number">4</span>;<span class="pagerange">230-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21451346">21451346</a>; DOI: <a href="http://dx.doi.org/10.1097%2FNMD.0b013e3182125d2c">10.1097/NMD.0b013e3182125d2c</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cop1 constitutively regulates c-Jun protein stability and functions as a tumor suppressor in mice.</h4>
<p class="authors"><span class="author">Migliorini D</span>, <span class="author">Bogaerts S</span>, <span class="author">Defever D</span>, <span class="author">Vyas R</span>, <span class="author">Denecker G</span>, <span class="author">Radaelli E</span>, <span class="author">Zwolinska A</span>, <span class="author">Depaepe V</span>, <span class="author">Hochepied T</span>, <span class="author">Skarnes WC</span> and <span class="author">Marine JC</span></p>
<p class="affiliation">Laboratory for Molecular Cancer Biology, Department of Molecular and Developmental Genetics, VIB-K.U.Leuven, Leuven, Belgium.</p><p class="abstract">Biochemical studies have suggested conflicting roles for the E3 ubiquitin ligase constitutive photomorphogenesis protein 1 (Cop1; also known as Rfwd2) in tumorigenesis, providing evidence for both the oncoprotein c-Jun and the tumor suppressor p53 as its targets. Here we present what we believe to be the first in vivo investigation of the role of Cop1 in cancer etiology. Using an innovative genetic approach to generate an allelic series of Cop1, we found that Cop1 hypomorphic mice spontaneously developed malignancy at a high frequency in the first year of life and were highly susceptible to radiation-induced lymphomagenesis. Further analysis revealed that c-Jun was a key physiological target for Cop1 and that Cop1 constitutively kept c-Jun at low levels in vivo and thereby modulated c-Jun/AP-1 transcriptional activity. Importantly, Cop1 deficiency stimulated cell proliferation in a c-Jun-dependent manner. Focal deletions of COP1 were observed at significant frequency across several cancer types, and COP1 loss was determined to be one of the mechanisms leading to c-Jun upregulation in human cancer. We therefore conclude that Cop1 is a tumor suppressor that functions, at least in part, by antagonizing c-Jun oncogenic activity. In the absence of evidence for a genetic interaction between Cop1 and p53, our data strongly argue against the use of Cop1-inhibitory drugs for cancer therapy.</p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2011</span>;<span class="volume">121</span>;<span class="number">4</span>;<span class="pagerange">1329-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21403399" rel="external">21403399</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3070608" rel="external">3070608</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI45784" rel="external">10.1172/JCI45784</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The use of genome-wide eQTL associations in lymphoblastoid cell lines to identify novel genetic pathways involved in complex traits.</h4>
<p class="authors"><span class="author">Min JL</span>, <span class="author">Taylor JM</span>, <span class="author">Richards JB</span>, <span class="author">Watts T</span>, <span class="author">Pettersson FH</span>, <span class="author">Broxholme J</span>, <span class="author">Ahmadi KR</span>, <span class="author">Surdulescu GL</span>, <span class="author">Lowy E</span>, <span class="author">Gieger C</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Perola M</span>, <span class="author">Soranzo N</span>, <span class="author">Surakka I</span>, <span class="author">Lindgren CM</span>, <span class="author">Ragoussis J</span>, <span class="author">Morris AP</span>, <span class="author">Cardon LR</span>, <span class="author">Spector TD</span> and <span class="author">Zondervan KT</span></p>
<p class="affiliation">Genetic and Genomic Epidemiology Unit, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom. <a href="mailto:%6A%6F%73%69%6E%65.%6D%69%6E%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#106;&#111;&#115;&#105;&#110;&#101;&#46;&#109;&#105;&#110;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The integrated analysis of genotypic and expression data for association with complex traits could identify novel genetic pathways involved in complex traits. We profiled 19,573 expression probes in Epstein-Barr virus-transformed lymphoblastoid cell lines (LCLs) from 299 twins and correlated these with 44 quantitative traits (QTs). For 939 expressed probes correlating with more than one QT, we investigated the presence of eQTL associations in three datasets of 57 CEU HapMap founders and 86 unrelated twins. Genome-wide association analysis of these probes with 2.2 m SNPs revealed 131 potential eQTLs (1,989 eQTL SNPs) overlapping between the HapMap datasets, five of which were in cis (58 eQTL SNPs). We then tested 535 SNPs tagging the eQTL SNPs, for association with the relevant QT in 2,905 twins. We identified nine potential SNP-QT associations (P&lt;0.01) but none significantly replicated in five large consortia of 1,097-16,129 subjects. We also failed to replicate previous reported eQTL associations with body mass index, plasma low-density lipoprotein cholesterol, high-density lipoprotein cholesterol and triglycerides levels derived from lymphocytes, adipose and liver tissue. Our results and additional power calculations suggest that proponents may have been overoptimistic in the power of LCLs in eQTL approaches to elucidate regulatory genetic effects on complex traits using the small datasets generated to date. Nevertheless, larger tissue-specific expression data sets relevant to specific traits are becoming available, and should enable the adoption of similar integrated analyses in the near future.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085235</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">e22070</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21789213" rel="external">21789213</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3137612" rel="external">3137612</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0022070" rel="external">10.1371/journal.pone.0022070</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel chromosomal rearrangements and break points at the t(6;9) in salivary adenoid cystic carcinoma: association with MYB-NFIB chimeric fusion, MYB expression, and clinical outcome.</h4>
<p class="authors"><span class="author">Mitani Y</span>, <span class="author">Rao PH</span>, <span class="author">Futreal PA</span>, <span class="author">Roberts DB</span>, <span class="author">Stephens PJ</span>, <span class="author">Zhao YJ</span>, <span class="author">Zhang L</span>, <span class="author">Mitani M</span>, <span class="author">Weber RS</span>, <span class="author">Lippman SM</span>, <span class="author">Caulin C</span> and <span class="author">El-Naggar AK</span></p>
<p class="affiliation">Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, USA.</p><p class="abstract"><strong>Objective</strong>: To investigate the molecular genetic heterogeneity associated with the t(6:9) in adenoid cystic carcinoma (ACC) and correlate the findings with patient clinical outcome.</p><p class="abstract">Multimolecular and genetic techniques complemented with massive pair-ended sequencing and single-nucleotide polymorphism array analyses were used on tumor specimens from 30 new and 52 previously analyzed fusion transcript-negative ACCs by reverse transcriptase PCR (RT-PCR). MYB mRNA expression level was determined by quantitative RT-PCR. The results of 102 tumors (30 new and 72 previously reported cases) were correlated with the clinicopathologic factors and patients&#39; survival.</p><p class="abstract"><strong>Results</strong>: The FISH analysis showed 34 of 82 (41.5%) fusion-positive tumors and molecular techniques identified fusion transcripts in 21 of the 82 (25.6%) tumors. Detailed FISH analysis of 11 out the 15 tumors with gene fusion without transcript formation showed translocation of NFIB sequences to proximal or distal sites of the MYB gene. Massive pair-end sequencing of a subset of tumors confirmed the proximal translocation to an NFIB sequence and led to the identification of a new fusion gene (NFIB-AIG1) in one of the tumors. Overall, MYB-NFIB gene fusion rate by FISH was in 52.9% whereas fusion transcript forming incidence was 38.2%. Significant statistical association between the 5&#39; MYB transcript expression and patient survival was found.</p><p class="abstract"><strong>Conclusions</strong>: We conclude that: (i) t(6;9) results in complex genetic and molecular alterations in ACC, (ii) MYB-NFIB gene fusion may not always be associated with chimeric transcript formation, (iii) noncanonical MYB-NFIB gene fusions occur in a subset of tumors, (iv) high MYB expression correlates with worse patient survival.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA-16672</span>, <span class="grant">P50 CA097007</span>; NIDCR NIH HHS: <span class="grant">U01DE019765</span>; Wellcome Trust: <span class="grant">077012</span>, <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Clinical cancer research : an official journal of the American Association for Cancer Research</span>&nbsp;<span class="year">2011</span>;<span class="volume">17</span>;<span class="number">22</span>;<span class="pagerange">7003-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21976542" rel="external">21976542</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3225955" rel="external">3225955</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1078-0432.CCR-11-1870" rel="external">10.1158/1078-0432.CCR-11-1870</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beta-adrenergic receptor activation rescues theta frequency stimulation-induced LTP deficits in mice expressing C-terminally truncated NMDA receptor GluN2A subunits.</h4>
<p class="authors"><span class="author">Moody TD</span>, <span class="author">Watabe AM</span>, <span class="author">Indersmitten T</span>, <span class="author">Komiyama NH</span>, <span class="author">Grant SG</span> and <span class="author">O&#39;Dell TJ</span></p>
<p class="affiliation">Interdepartmental PhD Program for Neuroscience, University of California Los Angeles, Los Angeles, California 90024, USA.</p><p class="abstract">Through protein interactions mediated by their cytoplasmic C termini the GluN2A and GluN2B subunits of NMDA receptors (NMDARs) have a key role in the formation of NMDAR signaling complexes at excitatory synapses. Although these signaling complexes are thought to have a crucial role in NMDAR-dependent forms of synaptic plasticity such as long-term potentiation (LTP), the role of the C terminus of GluN2A in coupling NMDARs to LTP enhancing and/or suppressing signaling pathways is unclear. To address this issue we examined the induction of LTP in the hippocampal CA1 region in mice lacking the C terminus of endogenous GluN2A subunits (GluN2A&Delta;C/&Delta;C). Our results show that truncation of GluN2A subunits produces robust, but highly frequency-dependent, deficits in LTP and a reduction in basal levels of extracellular signal regulated kinase 2 (ERK2) activation and phosphorylation of AMPA receptor GluA1 subunits at a protein kinase A site (serine 845). Consistent with the notion that these signaling deficits contribute to the deficits in LTP in GluN2A&Delta;C/&Delta;C mice, activating ERK2 and increasing GluA1 S845 phosphorylation through activation of &beta;-adrenergic receptors rescued the induction of LTP in these mutants. Together, our results indicate that the capacity of excitatory synapses to undergo plasticity in response to different patterns of activity is dependent on the coupling of specific signaling pathways to the intracellular domains of the NMDARs and that abnormal plasticity resulting from mutations in NMDARs can be reduced by activation of key neuromodulatory transmitter receptors that engage converging signaling pathways.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">MH609197</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Learning &amp; memory (Cold Spring Harbor, N.Y.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">18</span>;<span class="number">2</span>;<span class="pagerange">118-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21257779" rel="external">21257779</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3032578" rel="external">3032578</a>; DOI: <a href="http://dx.doi.org/10.1101%2Flm.2045311" rel="external">10.1101/lm.2045311</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The origins, evolution, and functional potential of alternative splicing in vertebrates.</h4>
<p class="authors"><span class="author">Mudge JM</span>, <span class="author">Frankish A</span>, <span class="author">Fernandez-Banet J</span>, <span class="author">Alioto T</span>, <span class="author">Derrien T</span>, <span class="author">Howald C</span>, <span class="author">Reymond A</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Hubbard T</span> and <span class="author">Harrow J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6A%6D%31%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#109;&#49;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Alternative splicing (AS) has the potential to greatly expand the functional repertoire of mammalian transcriptomes. However, few variant transcripts have been characterized functionally, making it difficult to assess the contribution of AS to the generation of phenotypic complexity and to study the evolution of splicing patterns. We have compared the AS of 309 protein-coding genes in the human ENCODE pilot regions against their mouse orthologs in unprecedented detail, utilizing traditional transcriptomic and RNAseq data. The conservation status of every transcript has been investigated, and each functionally categorized as coding (separated into coding sequence [CDS] or nonsense-mediated decay [NMD] linked) or noncoding. In total, 36.7% of human and 19.3% of mouse coding transcripts are species specific, and we observe a 3.6 times excess of human NMD transcripts compared with mouse; in contrast to previous studies, the majority of species-specific AS is unlinked to transposable elements. We observe one conserved CDS variant and one conserved NMD variant per 2.3 and 11.4 genes, respectively. Subsequently, we identify and characterize equivalent AS patterns for 22.9% of these CDS or NMD-linked events in nonmammalian vertebrate genomes, and our data indicate that functional NMD-linked AS is more widespread and ancient than previously thought. Furthermore, although we observe an association between conserved AS and elevated sequence conservation, as previously reported, we emphasize that 30% of conserved AS exons display sequence conservation below the average score for constitutive exons. In conclusion, we demonstrate the value of detailed comparative annotation in generating a comprehensive set of AS transcripts, increasing our understanding of AS evolution in vertebrates. Our data supports a model whereby the acquisition of functional AS has occurred throughout vertebrate evolution and is considered alongside amino acid change as a key mechanism in gene evolution.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5U54HG004555</span>; Wellcome Trust: <span class="grant">077198</span>, <span class="grant">WT077198/Z/05/Z</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2011</span>;<span class="volume">28</span>;<span class="number">10</span>;<span class="pagerange">2949-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21551269">21551269</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3176834">3176834</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsr127">10.1093/molbev/msr127</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing skippy: the genome sequence of an Australian kangaroo, Macropus eugenii.</h4>
<p class="authors"><span class="author">Murchison EP</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Sequencing of the tammar wallaby (Macropus eugenii) reveals insights into genome evolution, and mammalian reproduction and development.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">8</span>;<span class="pagerange">123</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21861852">21861852</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245606">3245606</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-8-123">10.1186/gb-2011-12-8-123</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Obstetric and perinatal outcomes in pregnancies complicated by Type 1 and Type 2 diabetes: influences of glycaemic control, obesity and social disadvantage.</h4>
<p class="authors"><span class="author">Murphy HR</span>, <span class="author">Steel SA</span>, <span class="author">Roland JM</span>, <span class="author">Morris D</span>, <span class="author">Ball V</span>, <span class="author">Campbell PJ</span>, <span class="author">Temple RC</span> and <span class="author">East Anglia Study Group for Improving Pregnancy Outcomes in Women with Diabetes (EASIPOD)</span></p>
<p class="affiliation">Metabolic Research Laboratories, Institute of Metabolic Science, University of Cambridge, Cambridge, UK. <a href="mailto:%68%6D%33%38%36%40%6D%65%64%73%63%68%6C.%63%61%6D.%61%63.%75%6B">&#104;&#109;&#51;&#56;&#54;&#64;&#109;&#101;&#100;&#115;&#99;&#104;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Aims</strong>: To compare obstetric and perinatal outcomes in women with Type 1 and Type 2 diabetes and relate these to maternal risk factors.</p><p class="abstract"><strong>Methods</strong>: Prospective cohort study of 682 consecutive diabetic pregnancies in East Anglia during 2006-2009. Relationships between congenital malformation, perinatal mortality and perinatal morbidity (large for gestational age, preterm delivery, neonatal care) with maternal age, parity, ethnicity, glycaemic control, obesity and social disadvantage were examined using bivariable and multivariate models.</p><p class="abstract"><strong>Results</strong>: There were 408 (59.8%) Type 1 and 274 (40.2%) Type 2 diabetes pregnancies. Women with Type 2 diabetes were older (P &lt; 0.001), heavier (P &lt; 0.0001), more frequently multiparous (P &lt; 0.001), more ethnically diverse (p &lt; 0.0001) and more socially disadvantaged (P = 0.0004). Although women with Type 2 diabetes had shorter duration of diabetes (P &lt; 0.0001) and better pre-conception glycaemic control [HbA(1c) 52 mmol/mol (6.9%) Type 2 diabetes vs. 63 mmol/l (7.9%) Type 1 diabetes; p &lt; 0.0001), rates of congenital malformation and perinatal mortality were comparable. Women with Type 2 diabetes had fewer large-for-gestational-age infants (37.6 vs. 52.9%, P &lt; 0.0008), fewer preterm deliveries (17.5 vs. 37.1%, P &lt; 0.0001) and their offspring had fewer neonatal care admissions (29.8 vs. 43.2%, P = 0.001). Third trimester HbA(1c) (OR 1.35, 95% CI 1.09-1.67, P = 0.006) and social disadvantage (OR 0.80, 95% CI 0.67-0.98; P = 0.03) were risk factors for large for gestational age.</p><p class="abstract"><strong>Conclusions</strong>: Despite increased age, parity, obesity and social disadvantage, women with Type 2 diabetes had better glycaemic control, fewer large-for-gestational-age infants, fewer preterm deliveries and fewer neonatal care admissions. Better tools are needed to improve glycaemic control and reduce the rates of large for gestational age, particularly in Type 1 diabetes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093867</span></p>
<p class="publication"><span class="title">Diabetic medicine : a journal of the British Diabetic Association</span>&nbsp;<span class="year">2011</span>;<span class="volume">28</span>;<span class="number">9</span>;<span class="pagerange">1060-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21843303" rel="external">21843303</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322333" rel="external">3322333</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1464-5491.2011.03333.x" rel="external">10.1111/j.1464-5491.2011.03333.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evidence for several waves of global transmission in the seventh cholera pandemic.</h4>
<p class="authors"><span class="author">Mutreja A</span>, <span class="author">Kim DW</span>, <span class="author">Thomson NR</span>, <span class="author">Connor TR</span>, <span class="author">Lee JH</span>, <span class="author">Kariuki S</span>, <span class="author">Croucher NJ</span>, <span class="author">Choi SY</span>, <span class="author">Harris SR</span>, <span class="author">Lebens M</span>, <span class="author">Niyogi SK</span>, <span class="author">Kim EJ</span>, <span class="author">Ramamurthy T</span>, <span class="author">Chun J</span>, <span class="author">Wood JL</span>, <span class="author">Clemens JD</span>, <span class="author">Czerkinsky C</span>, <span class="author">Nair GB</span>, <span class="author">Holmgren J</span>, <span class="author">Parkhill J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Vibrio cholerae is a globally important pathogen that is endemic in many areas of the world and causes 3-5 million reported cases of cholera every year. Historically, there have been seven acknowledged cholera pandemics; recent outbreaks in Zimbabwe and Haiti are included in the seventh and ongoing pandemic. Only isolates in serogroup O1 (consisting of two biotypes known as &#39;classical&#39; and &#39;El Tor&#39;) and the derivative O139 can cause epidemic cholera. It is believed that the first six cholera pandemics were caused by the classical biotype, but El Tor has subsequently spread globally and replaced the classical biotype in the current pandemic. Detailed molecular epidemiological mapping of cholera has been compromised by a reliance on sub-genomic regions such as mobile elements to infer relationships, making El Tor isolates associated with the seventh pandemic seem superficially diverse. To understand the underlying phylogeny of the lineage responsible for the current pandemic, we identified high-resolution markers (single nucleotide polymorphisms; SNPs) in 154 whole-genome sequences of globally and temporally representative V. cholerae isolates. Using this phylogeny, we show here that the seventh pandemic has spread from the Bay of Bengal in at least three independent but overlapping waves with a common ancestor in the 1950s, and identify several transcontinental transmission events. Additionally, we show how the acquisition of the SXT family of antibiotic resistance elements has shaped pandemic spread, and show that this family was first acquired at least ten years before its discovery in V. cholerae.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span>, <span class="grant">076964</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7365</span>;<span class="pagerange">462-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21866102">21866102</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3736323">3736323</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10392">10.1038/nature10392</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate gene association study of magnetic resonance imaging-based hip osteoarthritis (OA): evidence for COL9A2 gene as a common predisposing factor for hip OA and lumbar disc degeneration.</h4>
<p class="authors"><span class="author">N&auml;kki A</span>, <span class="author">Videman T</span>, <span class="author">Kujala UM</span>, <span class="author">Suhonen M</span>, <span class="author">M&auml;nnikk&ouml; M</span>, <span class="author">Peltonen L</span>, <span class="author">Batti&eacute; MC</span>, <span class="author">Kaprio J</span> and <span class="author">Saarela J</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland (FIMM), Biomedicum Helsinki 2U, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Objective</strong>: To study whether gene variants associated with lumbar disc degeneration (LDD) phenotypes are also associated with hip osteoarthritis (OA).</p><p class="abstract"><strong>Methods</strong>: Magnetic resonance imaging (MRI)-based hip OA changes for 345 twins were assessed and 99 single-nucleotide polymorphisms (SNP) were analyzed.</p><p class="abstract"><strong>Results</strong>: Variants in the COL9A2 (rs7533552, p = 0.0025) and COL10A1 (rs568725, p = 0.002) genes showed association with hip OA.</p><p class="abstract"><strong>Conclusion</strong>: The associating G allele in COL9A2 changes a glutamine to arginine or to tryptophan and may predispose to both hip OA and LDD, making it a candidate for degenerative connective tissue diseases.</p>
<p class="publication"><span class="title">The Journal of rheumatology</span>&nbsp;<span class="year">2011</span>;<span class="volume">38</span>;<span class="number">4</span>;<span class="pagerange">747-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21159828">21159828</a>; DOI: <a href="http://dx.doi.org/10.3899%2Fjrheum.100080">10.3899/jrheum.100080</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Activation of K-RAS by co-mutation of codons 19 and 20 is transforming.</h4>
<p class="authors"><span class="author">Naguib A</span>, <span class="author">Wilson CH</span>, <span class="author">Adams DJ</span> and <span class="author">Arends MJ</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, UK, CB2 0QQ, UK. <a href="mailto:%6D%6A%61%34%30%40%63%61%6D.%61%63.%75%6B">&#109;&#106;&#97;&#52;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">The K-RAS oncogene is widely mutated in human cancers. Activating mutations in K-RAS give rise to constitutive signalling through the MAPK/ERK and PI3K/AKT pathways promoting increased cell division, reduced apoptosis and transformation. The majority of activating mutations in K-RAS are located in codons 12 and 13. In a human colorectal cancer we identified a novel K-RAS co-mutation that altered codons 19 and 20 resulting in transitions at both codons (L19F/T20A) in the same allele. Using focus forming transformation assays in vitro , we showed that co-mutation of L19F/T20A in K-RAS demonstrated intermediate transforming ability that was greater than that of individual L19F and T20A mutants, but less than that of G12D and G12V K-RAS mutants. This demonstrated the synergistic effects of co-mutation of codons 19 and 20 and illustrated that co-mutation of these codons is functionally significant.</p>
<p class="publication"><span class="title">Journal of molecular signaling</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="pagerange">2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21371307">21371307</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3056876">3056876</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1750-2187-6-2">10.1186/1750-2187-6-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple loci are associated with white blood cell phenotypes.</h4>
<p class="authors"><span class="author">Nalls MA</span>, <span class="author">Couper DJ</span>, <span class="author">Tanaka T</span>, <span class="author">van Rooij FJ</span>, <span class="author">Chen MH</span>, <span class="author">Smith AV</span>, <span class="author">Toniolo D</span>, <span class="author">Zakai NA</span>, <span class="author">Yang Q</span>, <span class="author">Greinacher A</span>, <span class="author">Wood AR</span>, <span class="author">Garcia M</span>, <span class="author">Gasparini P</span>, <span class="author">Liu Y</span>, <span class="author">Lumley T</span>, <span class="author">Folsom AR</span>, <span class="author">Reiner AP</span>, <span class="author">Gieger C</span>, <span class="author">Lagou V</span>, <span class="author">Felix JF</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Gouskova NA</span>, <span class="author">Biffi A</span>, <span class="author">D&ouml;ring A</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Chong S</span>, <span class="author">Wiggins KL</span>, <span class="author">Rendon A</span>, <span class="author">Dehghan A</span>, <span class="author">Moore M</span>, <span class="author">Taylor K</span>, <span class="author">Wilson JG</span>, <span class="author">Lettre G</span>, <span class="author">Hofman A</span>, <span class="author">Bis JC</span>, <span class="author">Pirastu N</span>, <span class="author">Fox CS</span>, <span class="author">Meisinger C</span>, <span class="author">Sambrook J</span>, <span class="author">Arepalli S</span>, <span class="author">Nauck M</span>, <span class="author">Prokisch H</span>, <span class="author">Stephens J</span>, <span class="author">Glazer NL</span>, <span class="author">Cupples LA</span>, <span class="author">Okada Y</span>, <span class="author">Takahashi A</span>, <span class="author">Kamatani Y</span>, <span class="author">Matsuda K</span>, <span class="author">Tsunoda T</span>, <span class="author">Tanaka T</span>, <span class="author">Kubo M</span>, <span class="author">Nakamura Y</span>, <span class="author">Yamamoto K</span>, <span class="author">Kamatani N</span>, <span class="author">Stumvoll M</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Prokopenko I</span>, <span class="author">Illig T</span>, <span class="author">Patel KV</span>, <span class="author">Garner SF</span>, <span class="author">Kuhnel B</span>, <span class="author">Mangino M</span>, <span class="author">Oostra BA</span>, <span class="author">Thein SL</span>, <span class="author">Coresh J</span>, <span class="author">Wichmann HE</span>, <span class="author">Menzel S</span>, <span class="author">Lin J</span>, <span class="author">Pistis G</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Spector TD</span>, <span class="author">Teumer A</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Gudnason V</span>, <span class="author">Bandinelli S</span>, <span class="author">Frayling TM</span>, <span class="author">Chakravarti A</span>, <span class="author">van Duijn CM</span>, <span class="author">Melzer D</span>, <span class="author">Ouwehand WH</span>, <span class="author">Levy D</span>, <span class="author">Boerwinkle E</span>, <span class="author">Singleton AB</span>, <span class="author">Hernandez DG</span>, <span class="author">Longo DL</span>, <span class="author">Soranzo N</span>, <span class="author">Witteman JC</span>, <span class="author">Psaty BM</span>, <span class="author">Ferrucci L</span>, <span class="author">Harris TB</span>, <span class="author">O&#39;Donnell CJ</span> and <span class="author">Ganesh SK</span></p>
<p class="affiliation">Laboratory of Neurogenetics, Intramural Research Program, National Institute on Aging, National Institutes of Health, Bethesda, Maryland, United States of America. <a href="mailto:%6E%61%6C%6C%73%6D%40%6D%61%69%6C.%6E%69%68.%67%6F%76">&#110;&#97;&#108;&#108;&#115;&#109;&#64;&#109;&#97;&#105;&#108;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">White blood cell (WBC) count is a common clinical measure from complete blood count assays, and it varies widely among healthy individuals. Total WBC count and its constituent subtypes have been shown to be moderately heritable, with the heritability estimates varying across cell types. We studied 19,509 subjects from seven cohorts in a discovery analysis, and 11,823 subjects from ten cohorts for replication analyses, to determine genetic factors influencing variability within the normal hematological range for total WBC count and five WBC subtype measures. Cohort specific data was supplied by the CHARGE, HeamGen, and INGI consortia, as well as independent collaborative studies. We identified and replicated ten associations with total WBC count and five WBC subtypes at seven different genomic loci (total WBC count-6p21 in the HLA region, 17q21 near ORMDL3, and CSF3; neutrophil count-17q21; basophil count- 3p21 near RPN1 and C3orf27; lymphocyte count-6p21, 19p13 at EPS15L1; monocyte count-2q31 at ITGA4, 3q21, 8q24 an intergenic region, 9q31 near EDG2), including three previously reported associations and seven novel associations. To investigate functional relationships among variants contributing to variability in the six WBC traits, we utilized gene expression- and pathways-based analyses. We implemented gene-clustering algorithms to evaluate functional connectivity among implicated loci and showed functional relationships across cell types. Gene expression data from whole blood was utilized to show that significant biological consequences can be extracted from our genome-wide analyses, with effect estimates for significant loci from the meta-analyses being highly corellated with the proximal gene expression. In addition, collaborative efforts between the groups contributing to this study and related studies conducted by the COGENT and RIKEN groups allowed for the examination of effect homogeneity for genome-wide significant associations across populations of diverse ancestral backgrounds.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; NCRR NIH HHS: <span class="grant">M01-RR000052</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">HL085251</span>, <span class="grant">HL087652</span>, <span class="grant">N01-HC-05187</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-45134</span>, <span class="grant">N01-HC-45204</span>, <span class="grant">N01-HC-45205</span>, <span class="grant">N01-HC-48047</span>, <span class="grant">N01-HC-48048</span>, <span class="grant">N01-HC-48049</span>, <span class="grant">N01-HC-48050</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55017</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-65226</span>, <span class="grant">N01-HC-95095</span>, <span class="grant">N01-HC-95100</span>, <span class="grant">N01-HC-95170</span>, <span class="grant">N01-HC-95171</span>, <span class="grant">N01-HC-95172</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL-071862</span>, <span class="grant">R01 HL087698-01</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">R01HL86694</span>, <span class="grant">U01 HL72518</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">AG000932-02</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">N01-AG-821336</span>, <span class="grant">N01-AG-916413</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>, <span class="grant">R01 AG032098-04</span>, <span class="grant">R01 AG24233-01</span>; NINR NIH HHS: <span class="grant">R01 NR08153</span>; PHS HHS: <span class="grant">24152</span>, <span class="grant">32100-2</span>, <span class="grant">32105-6</span>, <span class="grant">32108-9</span>, <span class="grant">32111-13</span>, <span class="grant">32115</span>, <span class="grant">32118-32119</span>, <span class="grant">32122</span>, <span class="grant">42107-26</span>, <span class="grant">42129-32</span>, <span class="grant">44221</span>, <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>; WHI NIH HHS: <span class="grant">N01WH22110</span>; Wellcome Trust: <span class="grant">076113/C/04/Z</span>, <span class="grant">091746/Z/10/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e1002113</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21738480" rel="external">21738480</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3128114" rel="external">3128114</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002113" rel="external">10.1371/journal.pgen.1002113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High incidence of recurrent copy number variants in patients with isolated and syndromic M&uuml;llerian aplasia.</h4>
<p class="authors"><span class="author">Nik-Zainal S</span>, <span class="author">Strick R</span>, <span class="author">Storer M</span>, <span class="author">Huang N</span>, <span class="author">Rad R</span>, <span class="author">Willatt L</span>, <span class="author">Fitzgerald T</span>, <span class="author">Martin V</span>, <span class="author">Sandford R</span>, <span class="author">Carter NP</span>, <span class="author">Janecke AR</span>, <span class="author">Renner SP</span>, <span class="author">Oppelt PG</span>, <span class="author">Oppelt P</span>, <span class="author">Schulze C</span>, <span class="author">Brucker S</span>, <span class="author">Hurles M</span>, <span class="author">Beckmann MW</span>, <span class="author">Strissel PL</span> and <span class="author">Shaw-Smith C</span></p>
<p class="affiliation">Department of Obstetrics and Gynecology, University-Clinic Erlangen, Erlangen, Germany.</p><p class="abstract"><strong>Background</strong>: Congenital malformations involving the M&uuml;llerian ducts are observed in around 5% of infertile women. Complete aplasia of the uterus, cervix, and upper vagina, also termed M&uuml;llerian aplasia or Mayer-Rokitansky-Kuster-Hauser (MRKH) syndrome, occurs with an incidence of around 1 in 4500 female births, and occurs in both isolated and syndromic forms. Previous reports have suggested that a proportion of cases, especially syndromic cases, are caused by variation in copy number at different genomic loci.</p><p class="abstract"><strong>Methods</strong>: In order to obtain an overview of the contribution of copy number variation to both isolated and syndromic forms of M&uuml;llerian aplasia, copy number assays were performed in a series of 63 cases, of which 25 were syndromic and 38 isolated.</p><p class="abstract"><strong>Results</strong>: A high incidence (9/63, 14%) of recurrent copy number variants in this cohort is reported here. These comprised four cases of microdeletion at 16p11.2, an autism susceptibility locus not previously associated with M&uuml;llerian aplasia, four cases of microdeletion at 17q12, and one case of a distal 22q11.2 microdeletion. Microdeletions at 16p11.2 and 17q12 were found in 4/38 (10.5%) cases with isolated M&uuml;llerian aplasia, and at 16p11.2, 17q12 and 22q11.2 (distal) in 5/25 cases (20%) with syndromic M&uuml;llerian aplasia.</p><p class="abstract"><strong>Conclusion</strong>: The finding of microdeletion at 16p11.2 in 2/38 (5%) of isolated and 2/25 (8%) of syndromic cases suggests a significant contribution of this copy number variant alone to the pathogenesis of M&uuml;llerian aplasia. Overall, the high incidence of recurrent copy number variants in all forms of M&uuml;llerian aplasia has implications for the understanding of the aetiopathogenesis of the condition, and for genetic counselling in families affected by it.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077014</span>, <span class="grant">079973</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">48</span>;<span class="number">3</span>;<span class="pagerange">197-204</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21278390">21278390</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322361">3322361</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2010.082412">10.1136/jmg.2010.082412</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mechanisms mediating brain and cognitive reserve: experience-dependent neuroprotection and functional compensation in animal models of neurodegenerative diseases.</h4>
<p class="authors"><span class="author">Nithianantharajah J</span> and <span class="author">Hannan AJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%6A%6E%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#110;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">&#39;Brain and cognitive reserve&#39; (BCR) refers here to the accumulated neuroprotective reserve and capacity for functional compensation induced by the chronic enhancement of mental and physical activity. BCR is thought to protect against, and compensate for, a range of different neurodegenerative diseases, as well as other neurological and psychiatric disorders. In this review we will discuss BCR, and its potential mechanisms, in neurodegenerative disorders, with a focus on Huntington&#39;s disease (HD) and Alzheimer&#39;s disease (AD). Epidemiological studies of AD, and other forms of dementia, provided early evidence for BCR. The first evidence for the beneficial effects of enhanced mental and physical activity, and associated mechanistic insights, in an animal model of neurodegenerative disease was provided by experiments using HD transgenic mice. More recently, experiments on animal models of HD, AD and various other brain disorders have suggested potential molecular and cellular mechanisms underpinning BCR. We propose that sophisticated insight into the processes underlying BCR, and identification of key molecules mediating these beneficial effects, will pave the way for therapeutic advances targeting these currently incurable neurodegenerative diseases.</p>
<p class="publication"><span class="title">Progress in neuro-psychopharmacology &amp; biological psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">35</span>;<span class="number">2</span>;<span class="pagerange">331-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21112312">21112312</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pnpbp.2010.10.026">10.1016/j.pnpbp.2010.10.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Impact of temperament on depression and anxiety symptoms and depressive disorder in a population-based birth cohort.</h4>
<p class="authors"><span class="author">Nyman E</span>, <span class="author">Miettunen J</span>, <span class="author">Freimer N</span>, <span class="author">Joukamaa M</span>, <span class="author">M&auml;ki P</span>, <span class="author">Ekelund J</span>, <span class="author">Peltonen L</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Veijola J</span> and <span class="author">Paunio T</span></p>
<p class="affiliation">Public Health Genomics Unit, Institute for Molecular Medicine Finland FIMM, University of Helsinki and National Institute for Health and Welfare, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: The aim of this study was to characterize at the population level how innate features of temperament relate to experience of depressive mood and anxiety, and whether these symptoms have separable temperamental backgrounds.</p><p class="abstract"><strong>Methods</strong>: The study subjects were 4773 members of the population-based Northern Finland Birth Cohort 1966, a culturally and genetically homogeneous study sample. Temperament was measured at age 31 using the temperament items of the Temperament and Character Inventory and a separate Pessimism score. Depressive mood was assessed based on a previous diagnosis of depressive disorder or symptoms of depression according to the Hopkins Symptom Check List - 25. Anxiety was assessed analogously.</p><p class="abstract"><strong>Results</strong>: High levels of Harm avoidance and Pessimism were related to both depressive mood (effect sizes; d=0.84 and d=1.25, respectively) and depressive disorder (d=0.68 and d=0.68, respectively). Of the dimensions of Harm avoidance, Anticipatory worry and Fatigability had the strongest effects. Symptoms of depression and anxiety showed very similar underlying temperament patterns.</p><p class="abstract"><strong>Limitations</strong>: Although Harm avoidance and Pessimism appear to be important endophenotype candidates for depression and anxiety, their potential usefulness as endophenotypes, and whether they meet all the suggested criteria for endophenotypes will remain to be confirmed in future studies.</p><p class="abstract"><strong>Conclusions</strong>: Personality characteristics of Pessimism and Harm avoidance, in particular its dimensions Anticipatory worry and Fatigability, are strongly related to symptoms of depression and anxiety as well as to depressive disorder in this population. These temperamental features may be used as dimensional susceptibility factors in etiological studies of depression, which may aid in the development of improved clinical practice.</p>
<p class="publication"><span class="title">Journal of affective disorders</span>&nbsp;<span class="year">2011</span>;<span class="volume">131</span>;<span class="number">1-3</span>;<span class="pagerange">393-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21570580">21570580</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jad.2010.12.008">10.1016/j.jad.2010.12.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BioMart as an integration solution for the International Knockout Mouse Consortium.</h4>
<p class="authors"><span class="author">Oakley DJ</span>, <span class="author">Iyer V</span>, <span class="author">Skarnes WC</span> and <span class="author">Smedley D</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1HH.</p><p class="abstract">In this article, we describe the use of the BioMart data management system to provide integrated access to International Knockout Mouse Consortium (IKMC) data and other related mouse resources. The IKMC is currently mutating all mouse protein-coding genes in embryonic stem (ES) cells using gene targeting and gene trapping approaches. The BioMart portal allows researchers to identify and obtain IKMC knockout vectors, ES cells and mice for genes of interest. Gene annotation, expression, phenotype and disease data is also integrated from external BioMarts, allowing selection of IKMC products by a wide variety of criteria. These products are invaluable for researchers involved in the elucidation of gene function and the role of individual genes in human disease. Here, we describe these datasets in more detail and illustrate the functionality of the portal using several examples.</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2011</span>;<span class="volume">2011</span>;<span class="pagerange">bar028</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21930503" rel="external">21930503</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3263594" rel="external">3263594</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbar028" rel="external">10.1093/database/bar028</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comprehensive evaluation of potential lung function associated genes in the SpiroMeta general population sample.</h4>
<p class="authors"><span class="author">Obeidat M</span>, <span class="author">Wain LV</span>, <span class="author">Shrine N</span>, <span class="author">Kalsheker N</span>, <span class="author">Soler Artigas M</span>, <span class="author">Repapi E</span>, <span class="author">Burton PR</span>, <span class="author">Johnson T</span>, <span class="author">Ramasamy A</span>, <span class="author">Zhao JH</span>, <span class="author">Zhai G</span>, <span class="author">Huffman JE</span>, <span class="author">Vitart V</span>, <span class="author">Albrecht E</span>, <span class="author">Igl W</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">Cadby G</span>, <span class="author">Hui J</span>, <span class="author">Palmer LJ</span>, <span class="author">Hadley D</span>, <span class="author">McArdle WL</span>, <span class="author">Rudnicka AR</span>, <span class="author">Barroso I</span>, <span class="author">Loos RJ</span>, <span class="author">Wareham NJ</span>, <span class="author">Mangino M</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Gl&auml;ser S</span>, <span class="author">Homuth G</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Deloukas P</span>, <span class="author">Granell R</span>, <span class="author">Henderson J</span>, <span class="author">Grkovic I</span>, <span class="author">Jankovic S</span>, <span class="author">Zgaga L</span>, <span class="author">Pola&scaron;ek O</span>, <span class="author">Rudan I</span>, <span class="author">Wright AF</span>, <span class="author">Campbell H</span>, <span class="author">Wild SH</span>, <span class="author">Wilson JF</span>, <span class="author">Heinrich J</span>, <span class="author">Imboden M</span>, <span class="author">Probst-Hensch NM</span>, <span class="author">Gyllensten U</span>, <span class="author">Johansson &Aring;</span>, <span class="author">Zaboli G</span>, <span class="author">Mustelin L</span>, <span class="author">Rantanen T</span>, <span class="author">Surakka I</span>, <span class="author">Kaprio J</span>, <span class="author">Jarvelin MR</span>, <span class="author">Hayward C</span>, <span class="author">Evans DM</span>, <span class="author">Koch B</span>, <span class="author">Musk AW</span>, <span class="author">Elliott P</span>, <span class="author">Strachan DP</span>, <span class="author">Tobin MD</span>, <span class="author">Sayers I</span>, <span class="author">Hall IP</span> and <span class="author">SpiroMeta Consortium</span></p>
<p class="affiliation">Nottingham Respiratory Biomedical Research Unit, Division of Therapeutics and Molecular Medicine, University Hospital of Nottingham, Nottingham, United Kingdom.</p><p class="abstract"><strong>Rationale</strong>: Lung function measures are heritable traits that predict population morbidity and mortality and are essential for the diagnosis of chronic obstructive pulmonary disease (COPD). Variations in many genes have been reported to affect these traits, but attempts at replication have provided conflicting results. Recently, we undertook a meta-analysis of Genome Wide Association Study (GWAS) results for lung function measures in 20,288 individuals from the general population (the SpiroMeta consortium).</p><p class="abstract"><strong>Objectives</strong>: To comprehensively analyse previously reported genetic associations with lung function measures, and to investigate whether single nucleotide polymorphisms (SNPs) in these genomic regions are associated with lung function in a large population sample.</p><p class="abstract"><strong>Methods</strong>: We analysed association for SNPs tagging 130 genes and 48 intergenic regions (+/-10 kb), after conducting a systematic review of the literature in the PubMed database for genetic association studies reporting lung function associations.</p><p class="abstract"><strong>Results</strong>: The analysis included 16,936 genotyped and imputed SNPs. No loci showed overall significant association for FEV(1) or FEV(1)/FVC traits using a carefully defined significance threshold of 1.3&times;10(-5). The most significant loci associated with FEV(1) include SNPs tagging MACROD2 (P&#x200A;=&#x200A;6.81&times;10(-5)), CNTN5 (P&#x200A;=&#x200A;4.37&times;10(-4)), and TRPV4 (P&#x200A;=&#x200A;1.58&times;10(-3)). Among ever-smokers, SERPINA1 showed the most significant association with FEV(1) (P&#x200A;=&#x200A;8.41&times;10(-5)), followed by PDE4D (P&#x200A;=&#x200A;1.22&times;10(-4)). The strongest association with FEV(1)/FVC ratio was observed with ABCC1 (P&#x200A;=&#x200A;4.38&times;10(-4)), and ESR1 (P&#x200A;=&#x200A;5.42&times;10(-4)) among ever-smokers.</p><p class="abstract"><strong>Conclusions</strong>: Polymorphisms spanning previously associated lung function genes did not show strong evidence for association with lung function measures in the SpiroMeta consortium population. Common SERPINA1 polymorphisms may affect FEV(1) among smokers in the general population.</p>
<p class="grants">Funded by: Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401540</span>, <span class="grant">G0600705</span>, <span class="grant">G0701863</span>, <span class="grant">G0800582</span>, <span class="grant">G0801056</span>, <span class="grant">G0902125</span>, <span class="grant">G0902313</span>, <span class="grant">G9815508</span>, <span class="grant">G990146</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIDDK NIH HHS: <span class="grant">U01 DK062418</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span>, <span class="grant">092731</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">e19382</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21625484">21625484</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3098839">3098839</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0019382">10.1371/journal.pone.0019382</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeted disruption of py235ebp-1: invasion of erythrocytes by Plasmodium yoelii using an alternative Py235 erythrocyte binding protein.</h4>
<p class="authors"><span class="author">Ogun SA</span>, <span class="author">Tewari R</span>, <span class="author">Otto TD</span>, <span class="author">Howell SA</span>, <span class="author">Knuepfer E</span>, <span class="author">Cunningham DA</span>, <span class="author">Xu Z</span>, <span class="author">Pain A</span> and <span class="author">Holder AA</span></p>
<p class="affiliation">Division of Parasitology, MRC National Institute for Medical Research, London, UK. <a href="mailto:%73%6F%67%75%6E%40%6E%69%6D%72.%6D%72%63.%61%63.%75%6B">&#115;&#111;&#103;&#117;&#110;&#64;&#110;&#105;&#109;&#114;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Plasmodium yoelii YM asexual blood stage parasites express multiple members of the py235 gene family, part of the super-family of genes including those coding for Plasmodium vivax reticulocyte binding proteins and Plasmodium falciparum RH proteins. We previously identified a Py235 erythrocyte binding protein (Py235EBP-1, encoded by the PY01365 gene) that is recognized by protective mAb 25.77. Proteins recognized by a second protective mAb 25.37 have been identified by mass spectrometry and are encoded by two genes, PY01185 and PY05995/PY03534. We deleted the PY01365 gene and examined the phenotype. The expression of the members of the py235 family in both the WT and gene deletion parasites was measured by quantitative RT-PCR and RNA-Seq. py235ebp-1 expression was undetectable in the knockout parasite, but transcription of other members of the family was essentially unaffected. The knockout parasites continued to react with mAb 25.77; and the 25.77-binding proteins in these parasites were the PY01185 and PY05995/PY03534 products. The PY01185 product was also identified as erythrocyte binding. There was no clear change in erythrocyte invasion profile suggesting that the PY01185 gene product (designated PY235EBP-2) is able to fulfill the role of EBP-1 by serving as an invasion ligand although the molecular details of its interaction with erythrocytes have not been examined. The PY01365, PY01185, and PY05995/PY03534 genes are part of a distinct subset of the py235 family. In P. falciparum, the RH protein genes are under epigenetic control and expression correlates with binding to distinct erythrocyte receptors and specific invasion pathways, whereas in P. yoelii YM all the genes are expressed and deletion of one does not result in upregulation of another. We propose that simultaneous expression of multiple Py235 ligands enables invasion of a wide range of host erythrocytes even in the presence of antibodies to one or more of the proteins and that this functional redundancy at the protein level gives the parasite phenotypic plasticity in the absence of differences in gene expression.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">U117532067</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e1001288</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21379566" rel="external">21379566</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3040676" rel="external">3040676</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1001288" rel="external">10.1371/journal.ppat.1001288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variants and blood pressure in a population-based cohort: the Cardiovascular Risk in Young Finns study.</h4>
<p class="authors"><span class="author">Oikonen M</span>, <span class="author">Tikkanen E</span>, <span class="author">Juhola J</span>, <span class="author">Tuovinen T</span>, <span class="author">Sepp&auml;l&auml; I</span>, <span class="author">Juonala M</span>, <span class="author">Taittonen L</span>, <span class="author">Mikkil&auml; V</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Ripatti S</span>, <span class="author">Viikari J</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Havulinna AS</span>, <span class="author">Kee F</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Peltonen L</span>, <span class="author">Schork NJ</span>, <span class="author">Murray SS</span>, <span class="author">Berenson GS</span>, <span class="author">Chen W</span>, <span class="author">Srinivasan SR</span>, <span class="author">Salomaa V</span> and <span class="author">Raitakari OT</span></p>
<p class="affiliation">Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, PO Box 52, FI-20521 Turku, Finland. <a href="mailto:%6D%65%72%76%69.%6F%69%6B%6F%6E%65%6E%40%75%74%75.%66%69">&#109;&#101;&#114;&#118;&#105;&#46;&#111;&#105;&#107;&#111;&#110;&#101;&#110;&#64;&#117;&#116;&#117;&#46;&#102;&#105;</a></p><p class="abstract">Clinical relevance of a genetic predisposition to elevated blood pressure was quantified during the transition from childhood to adulthood in a population-based Finnish cohort (N=2357). Blood pressure was measured at baseline in 1980 (age 3-18 years) and in follow-ups in 1983, 1986, 2001, and 2007. Thirteen single nucleotide polymorphisms associated with blood pressure were genotyped, and 3 genetic risk scores associated with systolic and diastolic blood pressures and their combination were derived for all of the participants. Effects of the genetic risk score were 0.47 mm Hg for systolic and 0.53 mm Hg for diastolic blood pressures (both P&lt;0.01). The combination genetic risk score was associated with diastolic blood pressure from age 9 years onward (&beta;=0.68 mm Hg; P=0.015). Replications in 1194 participants of the Bogalusa Heart Study showed essentially similar results. The participants in the highest quintile of the combination genetic risk score had a 1.82-fold risk of hypertension in adulthood (P&lt;0.0001) compared with the lowest quintile, independent of a family history of premature hypertension. These findings show that genetic variants are associated with preclinical blood pressure traits in childhood; individuals with several susceptibility alleles have, on average, a 0.5-mm Hg higher blood pressure, and this trajectory continues from childhood to adulthood.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">AG-16592</span>; NICHD NIH HHS: <span class="grant">HD-061437</span>, <span class="grant">HD-062783</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Hypertension</span>&nbsp;<span class="year">2011</span>;<span class="volume">58</span>;<span class="number">6</span>;<span class="pagerange">1079-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22025373" rel="external">22025373</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3247907" rel="external">3247907</a>; DOI: <a href="http://dx.doi.org/10.1161%2FHYPERTENSIONAHA.111.179291" rel="external">10.1161/HYPERTENSIONAHA.111.179291</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genotypic and phenotypic modifications of Neisseria meningitidis after an accidental human passage.</h4>
<p class="authors"><span class="author">Omer H</span>, <span class="author">Rose G</span>, <span class="author">Jolley KA</span>, <span class="author">Frapy E</span>, <span class="author">Zahar JR</span>, <span class="author">Maiden MC</span>, <span class="author">Bentley SD</span>, <span class="author">Tinsley CR</span>, <span class="author">Nassif X</span> and <span class="author">Bille E</span></p>
<p class="affiliation">INSERM U1002, Paris, France.</p><p class="abstract">A scientist in our laboratory was accidentally infected while working with Z5463, a Neisseria meningitidis serogroup A strain. She developed severe symptoms (fever, meningism, purpuric lesions) that fortunately evolved with antibiotic treatment to complete recovery. Pulse-field gel electrophoresis confirmed that the isolate obtained from the blood culture (Z5463BC) was identical to Z5463, more precisely to a fourth subculture of this strain used the week before the contamination (Z5463PI). In order to get some insights into genomic modifications that can occur in vivo, we sequenced these three isolates. All the strains contained a mutated mutS allele and therefore displayed an hypermutator phenotype, consistent with the high number of mutations (SNP, Single Nucleotide Polymorphism) detected in the three strains. By comparing the number of SNP in all three isolates and knowing the number of passages between Z5463 and Z5463PI, we concluded that around 25 bacterial divisions occurred in the human body. As expected, the in vivo passage is responsible for several modifications of phase variable genes. This genomic study has been completed by transcriptomic and phenotypic studies, showing that the blood strain used a different haemoglobin-linked iron receptor (HpuA/B) than the parental strains (HmbR). Different pilin variants were found after the in vivo passage, which expressed different properties of adhesion. Furthermore the deletion of one gene involved in LOS biosynthesis (lgtB) results in Z5463BC expressing a different LOS than the L9 immunotype of Z2491. The in vivo passage, despite the small numbers of divisions, permits the selection of numerous genomic modifications that may account for the high capacity of the strain to disseminate.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087622</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">e17145</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21386889" rel="external">21386889</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3046118" rel="external">3046118</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0017145" rel="external">10.1371/journal.pone.0017145</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of mitosis and antimitotic drug responses in tumors by in vivo microscopy and single-cell pharmacodynamics.</h4>
<p class="authors"><span class="author">Orth JD</span>, <span class="author">Kohler RH</span>, <span class="author">Foijer F</span>, <span class="author">Sorger PK</span>, <span class="author">Weissleder R</span> and <span class="author">Mitchison TJ</span></p>
<p class="affiliation">Department of Systems Biology, Harvard Medical School, Center for Systems Biology, Massachusetts General Hospital, Boston, Massachusetts 02115, USA.</p><p class="abstract">Cancer relies upon frequent or abnormal cell division, but how the tumor microenvironment affects mitotic processes in vivo remains unclear, largely due to the technical challenges of optical access, spatial resolution, and motion. We developed high-resolution in vivo microscopy methods to visualize mitosis in a murine xenograft model of human cancer. Using these methods, we determined whether the single-cell response to the antimitotic drug paclitaxel (Ptx) was the same in tumors as in cell culture, observed the impact of Ptx on the tumor response as a whole, and evaluated the single-cell pharmacodynamics (PD) of Ptx (by in vivo PD microscopy). Mitotic initiation was generally less frequent in tumors than in cell culture, but subsequently it proceeded normally. Ptx treatment caused spindle assembly defects and mitotic arrest, followed by slippage from mitotic arrest, multinucleation, and apoptosis. Compared with cell culture, the peak mitotic index in tumors exposed to Ptx was lower and the tumor cells survived longer after mitotic arrest, becoming multinucleated rather than dying directly from mitotic arrest. Thus, the tumor microenvironment was much less proapoptotic than cell culture. The morphologies associated with mitotic arrest were dose and time dependent, thereby providing a semiquantitative, single-cell measure of PD. Although many tumor cells did not progress through Ptx-induced mitotic arrest, tumor significantly regressed in the model. Our findings show that in vivo microscopy offers a useful tool to visualize mitosis during tumor progression, drug responses, and cell fate at the single-cell level.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA084179</span>, <span class="grant">CA092782</span>, <span class="grant">CA139980</span>, <span class="grant">CA86355</span>, <span class="grant">P01 CA139980-01A1</span>, <span class="grant">R01 CA084179-11</span>; NIGMS NIH HHS: <span class="grant">R01 GM039565-25</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2011</span>;<span class="volume">71</span>;<span class="number">13</span>;<span class="pagerange">4608-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21712408" rel="external">21712408</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3129392" rel="external">3129392</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-11-0412" rel="external">10.1158/0008-5472.CAN-11-0412</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Real-time sequencing.</h4>
<p class="authors"><span class="author">Otto TD</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">9</span>;<span class="pagerange">633</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21836624">21836624</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2638">10.1038/nrmicro2638</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RATT: Rapid Annotation Transfer Tool.</h4>
<p class="authors"><span class="author">Otto TD</span>, <span class="author">Dillon GP</span>, <span class="author">Degrave WS</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK. <a href="mailto:%74%64%6F%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#100;&#111;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Second-generation sequencing technologies have made large-scale sequencing projects commonplace. However, making use of these datasets often requires gene function to be ascribed genome wide. Although tool development has kept pace with the changes in sequence production, for tasks such as mapping, de novo assembly or visualization, genome annotation remains a challenge. We have developed a method to rapidly provide accurate annotation for new genomes using previously annotated genomes as a reference. The method, implemented in a tool called RATT (Rapid Annotation Transfer Tool), transfers annotations from a high-quality reference to a new genome on the basis of conserved synteny. We demonstrate that a Mycobacterium tuberculosis genome or a single 2.5 Mb chromosome from a malaria parasite can be annotated in less than five minutes with only modest computational resources. RATT is available at http://ratt.sourceforge.net.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">9</span>;<span class="pagerange">e57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21306991">21306991</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3089447">3089447</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq1268">10.1093/nar/gkq1268</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Coordinating cell cycle progression via cyclin specificity.</h4>
<p class="authors"><span class="author">Pagliuca FW</span>, <span class="author">Collins MO</span> and <span class="author">Choudhary JS</span></p>
<p class="publication"><span class="title">Cell cycle (Georgetown, Tex.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">10</span>;<span class="number">24</span>;<span class="pagerange">4195-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22156915">22156915</a>; DOI: <a href="http://dx.doi.org/10.4161%2Fcc.10.24.18395">10.4161/cc.10.24.18395</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quantitative proteomics reveals the basis for the biochemical specificity of the cell-cycle machinery.</h4>
<p class="authors"><span class="author">Pagliuca FW</span>, <span class="author">Collins MO</span>, <span class="author">Lichawska A</span>, <span class="author">Zegerman P</span>, <span class="author">Choudhary JS</span> and <span class="author">Pines J</span></p>
<p class="affiliation">The Gurdon Institute, University of Cambridge, Cambridge, UK.</p><p class="abstract">Cyclin-dependent kinases comprise the conserved machinery that drives progress through the cell cycle, but how they do this in mammalian cells is still unclear. To identify the mechanisms by which cyclin-cdks control the cell cycle, we performed a time-resolved analysis of the in vivo interactors of cyclins E1, A2, and B1 by quantitative mass spectrometry. This global analysis of context-dependent protein interactions reveals the temporal dynamics of cyclin function in which networks of cyclin-cdk interactions vary according to the type of cyclin and cell-cycle stage. Our results explain the temporal specificity of the cell-cycle machinery, thereby providing a biochemical mechanism for the genetic requirement for multiple cyclins in vivo and reveal how the actions of specific cyclins are coordinated to control the cell cycle. Furthermore, we identify key substrates (Wee1 and c15orf42/Sld3) that reveal how cyclin A is able to promote both DNA replication and mitosis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A7397</span>; Wellcome Trust: <span class="grant">079643/Z/06/Z</span></p>
<p class="publication"><span class="title">Molecular cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">3</span>;<span class="pagerange">406-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21816347">21816347</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3332305">3332305</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molcel.2011.05.031">10.1016/j.molcel.2011.05.031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies a locus at 7p15.2 associated with endometriosis.</h4>
<p class="authors"><span class="author">Painter JN</span>, <span class="author">Anderson CA</span>, <span class="author">Nyholt DR</span>, <span class="author">Macgregor S</span>, <span class="author">Lin J</span>, <span class="author">Lee SH</span>, <span class="author">Lambert A</span>, <span class="author">Zhao ZZ</span>, <span class="author">Roseman F</span>, <span class="author">Guo Q</span>, <span class="author">Gordon SD</span>, <span class="author">Wallace L</span>, <span class="author">Henders AK</span>, <span class="author">Visscher PM</span>, <span class="author">Kraft P</span>, <span class="author">Martin NG</span>, <span class="author">Morris AP</span>, <span class="author">Treloar SA</span>, <span class="author">Kennedy SH</span>, <span class="author">Missmer SA</span>, <span class="author">Montgomery GW</span> and <span class="author">Zondervan KT</span></p>
<p class="affiliation">Molecular Epidemiology, Queensland Institute of Medical Research, Herston, Queensland, Australia. <a href="mailto:%6A%6F%64%69%65.%70%61%69%6E%74%65%72%40%71%69%6D%72.%65%64%75.%61%75">&#106;&#111;&#100;&#105;&#101;&#46;&#112;&#97;&#105;&#110;&#116;&#101;&#114;&#64;&#113;&#105;&#109;&#114;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Endometriosis is a common gynecological disease associated with pelvic pain and subfertility. We conducted a genome-wide association study (GWAS) in 3,194 individuals with surgically confirmed endometriosis (cases) and 7,060 controls from Australia and the UK. Polygenic predictive modeling showed significantly increased genetic loading among 1,364 cases with moderate to severe endometriosis. The strongest association signal was on 7p15.2 (rs12700667) for &#39;all&#39; endometriosis (P = 2.6 &times; 10&#x207B;&#x2077;, odds ratio (OR) = 1.22, 95% CI 1.13-1.32) and for moderate to severe disease (P = 1.5 &times; 10&#x207B;&#x2079;, OR = 1.38, 95% CI 1.24-1.53). We replicated rs12700667 in an independent cohort from the United States of 2,392 self-reported, surgically confirmed endometriosis cases and 2,271 controls (P = 1.2 &times; 10&#x207B;&sup3;, OR = 1.17, 95% CI 1.06-1.28), resulting in a genome-wide significant P value of 1.4 &times; 10&#x207B;&#x2079; (OR = 1.20, 95% CI 1.13-1.27) for &#39;all&#39; endometriosis in our combined datasets of 5,586 cases and 9,331 controls. rs12700667 is located in an intergenic region upstream of the plausible candidate genes NFE2L3 and HOXA10.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCI NIH HHS: <span class="grant">P01 CA087969</span>, <span class="grant">R01 CA049449</span>, <span class="grant">R01 CA050385</span>, <span class="grant">R01 CA067262</span>, <span class="grant">U01 CA098233</span>; NICHD NIH HHS: <span class="grant">R01 HD052473</span>, <span class="grant">R01 HD057210</span>; NIDDK NIH HHS: <span class="grant">P01 DK070756</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">081682</span>, <span class="grant">084766</span>, <span class="grant">085235</span>, <span class="grant">WT084766/Z/08/Z</span>, <span class="grant">WT085235/Z/08/Z</span>, <span class="grant">WT91745/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">1</span>;<span class="pagerange">51-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21151130">21151130</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3019124">3019124</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.731">10.1038/ng.731</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identity-by-descent-based phasing and imputation in founder populations using graphical models.</h4>
<p class="authors"><span class="author">Palin K</span>, <span class="author">Campbell H</span>, <span class="author">Wright AF</span>, <span class="author">Wilson JF</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Accurate knowledge of haplotypes, the combination of alleles co-residing on a single copy of a chromosome, enables powerful gene mapping and sequence imputation methods. Since humans are diploid, haplotypes must be derived from genotypes by a phasing process. In this study, we present a new computational model for haplotype phasing based on pairwise sharing of haplotypes inferred to be Identical-By-Descent (IBD). We apply the Bayesian network based model in a new phasing algorithm, called systematic long-range phasing (SLRP), that can capitalize on the close genetic relationships in isolated founder populations, and show with simulated and real genome-wide genotype data that SLRP substantially reduces the rate of phasing errors compared to previous phasing algorithms. Furthermore, the method accurately identifies regions of IBD, enabling linkage-like studies without pedigrees, and can be used to impute most genotypes with very low error rate.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">MC_U127561128</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077192</span>, <span class="grant">085475</span>, <span class="grant">WT077192</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">35</span>;<span class="number">8</span>;<span class="pagerange">853-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22006673">22006673</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3368215">3368215</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20635">10.1002/gepi.20635</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insights into the genetic architecture of osteoarthritis from stage 1 of the arcOGEN study.</h4>
<p class="authors"><span class="author">Panoutsopoulou K</span>, <span class="author">Southam L</span>, <span class="author">Elliott KS</span>, <span class="author">Wrayner N</span>, <span class="author">Zhai G</span>, <span class="author">Beazley C</span>, <span class="author">Thorleifsson G</span>, <span class="author">Arden NK</span>, <span class="author">Carr A</span>, <span class="author">Chapman K</span>, <span class="author">Deloukas P</span>, <span class="author">Doherty M</span>, <span class="author">McCaskie A</span>, <span class="author">Ollier WE</span>, <span class="author">Ralston SH</span>, <span class="author">Spector TD</span>, <span class="author">Valdes AM</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson JM</span>, <span class="author">Arden E</span>, <span class="author">Battley K</span>, <span class="author">Blackburn H</span>, <span class="author">Blanco FJ</span>, <span class="author">Bumpstead S</span>, <span class="author">Cupples LA</span>, <span class="author">Day-Williams AG</span>, <span class="author">Dixon K</span>, <span class="author">Doherty SA</span>, <span class="author">Esko T</span>, <span class="author">Evangelou E</span>, <span class="author">Felson D</span>, <span class="author">Gomez-Reino JJ</span>, <span class="author">Gonzalez A</span>, <span class="author">Gordon A</span>, <span class="author">Gwilliam R</span>, <span class="author">Halldorsson BV</span>, <span class="author">Hauksson VB</span>, <span class="author">Hofman A</span>, <span class="author">Hunt SE</span>, <span class="author">Ioannidis JP</span>, <span class="author">Ingvarsson T</span>, <span class="author">Jonsdottir I</span>, <span class="author">Jonsson H</span>, <span class="author">Keen R</span>, <span class="author">Kerkhof HJ</span>, <span class="author">Kloppenburg MG</span>, <span class="author">Koller N</span>, <span class="author">Lakenberg N</span>, <span class="author">Lane NE</span>, <span class="author">Lee AT</span>, <span class="author">Metspalu A</span>, <span class="author">Meulenbelt I</span>, <span class="author">Nevitt MC</span>, <span class="author">O&#39;Neill F</span>, <span class="author">Parimi N</span>, <span class="author">Potter SC</span>, <span class="author">Rego-Perez I</span>, <span class="author">Riancho JA</span>, <span class="author">Sherburn K</span>, <span class="author">Slagboom PE</span>, <span class="author">Stefansson K</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Sumillera M</span>, <span class="author">Swift D</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Tsezou A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Meurs JB</span>, <span class="author">Watkins B</span>, <span class="author">Wheeler M</span>, <span class="author">Mitchell S</span>, <span class="author">Zhu Y</span>, <span class="author">Zmuda JM</span>, <span class="author">arcOGEN Consortium</span>, <span class="author">Zeggini E</span> and <span class="author">Loughlin J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.</p><p class="abstract"><strong>Objectives</strong>: The genetic aetiology of osteoarthritis has not yet been elucidated. To enable a well-powered genome-wide association study (GWAS) for osteoarthritis, the authors have formed the arcOGEN Consortium, a UK-wide collaborative effort aiming to scan genome-wide over 7500 osteoarthritis cases in a two-stage genome-wide association scan. Here the authors report the findings of the stage 1 interim analysis.</p><p class="abstract"><strong>Methods</strong>: The authors have performed a genome-wide association scan for knee and hip osteoarthritis in 3177 cases and 4894 population-based controls from the UK. Replication of promising signals was carried out in silico in five further scans (44,449 individuals), and de novo in 14 534 independent samples, all of European descent.</p><p class="abstract"><strong>Results</strong>: None of the association signals the authors identified reach genome-wide levels of statistical significance, therefore stressing the need for corroboration in sample sets of a larger size. Application of analytical approaches to examine the allelic architecture of disease to the stage 1 genome-wide association scan data suggests that osteoarthritis is a highly polygenic disease with multiple risk variants conferring small effects.</p><p class="abstract"><strong>Conclusions</strong>: Identifying loci conferring susceptibility to osteoarthritis will require large-scale sample sizes and well-defined phenotypes to minimise heterogeneity.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17489</span>; Medical Research Council: <span class="grant">G0901461</span>; NIAMS NIH HHS: <span class="grant">K24 AR048841</span>, <span class="grant">R01 AR052000</span></p>
<p class="publication"><span class="title">Annals of the rheumatic diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">70</span>;<span class="number">5</span>;<span class="pagerange">864-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21177295">21177295</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3070286">3070286</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fard.2010.141473">10.1136/ard.2010.141473</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic SF3B1 mutation in myelodysplasia with ring sideroblasts.</h4>
<p class="authors"><span class="author">Papaemmanuil E</span>, <span class="author">Cazzola M</span>, <span class="author">Boultwood J</span>, <span class="author">Malcovati L</span>, <span class="author">Vyas P</span>, <span class="author">Bowen D</span>, <span class="author">Pellagatti A</span>, <span class="author">Wainscoat JS</span>, <span class="author">Hellstrom-Lindberg E</span>, <span class="author">Gambacorti-Passerini C</span>, <span class="author">Godfrey AL</span>, <span class="author">Rapado I</span>, <span class="author">Cvejic A</span>, <span class="author">Rance R</span>, <span class="author">McGee C</span>, <span class="author">Ellis P</span>, <span class="author">Mudie LJ</span>, <span class="author">Stephens PJ</span>, <span class="author">McLaren S</span>, <span class="author">Massie CE</span>, <span class="author">Tarpey PS</span>, <span class="author">Varela I</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Davies HR</span>, <span class="author">Shlien A</span>, <span class="author">Jones D</span>, <span class="author">Raine K</span>, <span class="author">Hinton J</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">Baxter EJ</span>, <span class="author">Score J</span>, <span class="author">Galli A</span>, <span class="author">Della Porta MG</span>, <span class="author">Travaglino E</span>, <span class="author">Groves M</span>, <span class="author">Tauro S</span>, <span class="author">Munshi NC</span>, <span class="author">Anderson KC</span>, <span class="author">El-Naggar A</span>, <span class="author">Fischer A</span>, <span class="author">Mustonen V</span>, <span class="author">Warren AJ</span>, <span class="author">Cross NC</span>, <span class="author">Green AR</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span> and <span class="author">Chronic Myeloid Disorders Working Group of the International Cancer Genome Consortium</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, United Kingdom</p><p class="abstract"><strong>Background</strong>: Myelodysplastic syndromes are a diverse and common group of chronic hematologic cancers. The identification of new genetic lesions could facilitate new diagnostic and therapeutic strategies.</p><p class="abstract"><strong>Methods</strong>: We used massively parallel sequencing technology to identify somatically acquired point mutations across all protein-coding exons in the genome in 9 patients with low-grade myelodysplasia. Targeted resequencing of the gene encoding RNA splicing factor 3B, subunit 1 (SF3B1), was also performed in a cohort of 2087 patients with myeloid or other cancers.</p><p class="abstract"><strong>Results</strong>: We identified 64 point mutations in the 9 patients. Recurrent somatically acquired mutations were identified in SF3B1. Follow-up revealed SF3B1 mutations in 72 of 354 patients (20%) with myelodysplastic syndromes, with particularly high frequency among patients whose disease was characterized by ring sideroblasts (53 of 82 [65%]). The gene was also mutated in 1 to 5% of patients with a variety of other tumor types. The observed mutations were less deleterious than was expected on the basis of chance, suggesting that the mutated protein retains structural integrity with altered function. SF3B1 mutations were associated with down-regulation of key gene networks, including core mitochondrial pathways. Clinically, patients with SF3B1 mutations had fewer cytopenias and longer event-free survival than patients without SF3B1 mutations.</p><p class="abstract"><strong>Conclusions</strong>: Mutations in SF3B1 implicate abnormalities of messenger RNA splicing in the pathogenesis of myelodysplastic syndromes. (Funded by the Wellcome Trust and others.).</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800784</span>, <span class="grant">G1000729</span>, <span class="grant">MC_U105161083</span>, <span class="grant">MR/L003368/1</span>; NCI NIH HHS: <span class="grant">P01 CA078378</span>, <span class="grant">P01 CA078378-10</span>, <span class="grant">R01 CA124929</span>, <span class="grant">R01 CA124929-05</span>; PHS HHS: <span class="grant">P01-155249</span>, <span class="grant">P01-78378</span>, <span class="grant">P50-100007</span>, <span class="grant">R01-124929</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">093867</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">365</span>;<span class="number">15</span>;<span class="pagerange">1384-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21995386">21995386</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322589">3322589</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1103283">10.1056/NEJMoa1103283</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fetal-specific DNA methylation ratio permits noninvasive prenatal diagnosis of trisomy 21.</h4>
<p class="authors"><span class="author">Papageorgiou EA</span>, <span class="author">Karagrigoriou A</span>, <span class="author">Tsaliki E</span>, <span class="author">Velissariou V</span>, <span class="author">Carter NP</span> and <span class="author">Patsalis PC</span></p>
<p class="affiliation">Cytogenetics and Genomics Department, The Cyprus Institute of Neurology and Genetics, Nicosia, Cyprus.</p><p class="abstract">The trials performed worldwide toward noninvasive prenatal diagnosis (NIPD) of Down&#39;s syndrome (or trisomy 21) have shown the commercial and medical potential of NIPD compared to the currently used invasive prenatal diagnostic procedures. Extensive investigation of methylation differences between the mother and the fetus has led to the identification of differentially methylated regions (DMRs). In this study, we present a strategy using the methylated DNA immunoprecipitation (MeDiP) methodology in combination with real-time quantitative PCR (qPCR) to achieve fetal chromosome dosage assessment, which can be performed noninvasively through the analysis of fetal-specific DMRs. We achieved noninvasive prenatal detection of trisomy 21 by determining the methylation ratio of normal and trisomy 21 cases for each tested fetal-specific DMR present in maternal peripheral blood, followed by further statistical analysis. The application of this fetal-specific methylation ratio approach provided correct diagnosis of 14 trisomy 21 and 26 normal cases.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Nature medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">17</span>;<span class="number">4</span>;<span class="pagerange">510-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21378977">21378977</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3977039">3977039</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnm.2312">10.1038/nm.2312</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fetal-specific DNA methylation ratio permits noninvasive prenatal diagnosis of trisomy 21</h4>
<p class="authors"><span class="author">Papageorgiou EA</span>, <span class="author">Karagrigoriou A</span>, <span class="author">Tsaliki E</span>, <span class="author">Velissariou V</span>, <span class="author">CARTER NP</span>, <span class="author">Patsalis PC</span></p>
<p class="publication"><span class="title">Obstetrical and Gynecological Survey</span> <span class="year">2011</span>;<span class="volume">66</span>;<span class="pagerange">419</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacterial epidemiology and biology--lessons from genome sequencing.</h4>
<p class="authors"><span class="author">Parkhill J</span> and <span class="author">Wren BW</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Next-generation sequencing has ushered in a new era of microbial genomics, enabling the detailed historical and geographical tracing of bacteria. This is helping to shape our understanding of bacterial evolution.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">10</span>;<span class="pagerange">230</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22027015">22027015</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3333767">3333767</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-10-230">10.1186/gb-2011-12-10-230</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identity crisis? The need for systematic gene IDs.</h4>
<p class="authors"><span class="author">Parsons M</span>, <span class="author">Myler PJ</span>, <span class="author">Berriman M</span>, <span class="author">Roos DS</span> and <span class="author">Stuart KD</span></p><p class="abstract">Recent years have seen an explosion in the availability of protozoan pathogen genome sequences. Although data regarding the underlying genome sequence remain relatively stable after the initial draft, understanding of specific gene function is increasing rapidly. This dichotomy is reflected in the relative stability of systematic gene identifiers (SysIDs(*)) in genome sequence databases, as compared to evolving and/or conflicting gene and gene product names. GenBank/EMBL/DDBJ accession numbers are important, but most protozoan parasite researchers use organism-based databases such as EuPathDB or GeneDB as their immediate resource for gene-based information because they not only provide sequence information but also functional information and links to references. Reference to SysIDs therefore provides a valuable bridge to this repository of information.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">5U01AI075641</span></p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">5</span>;<span class="pagerange">183-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21474380" rel="external">21474380</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2011.02.007" rel="external">10.1016/j.pt.2011.02.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Revealing the genetic structure of a trait by sequencing a population under selection.</h4>
<p class="authors"><span class="author">Parts L</span>, <span class="author">Cubillos FA</span>, <span class="author">Warringer J</span>, <span class="author">Jain K</span>, <span class="author">Salinas F</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Molin M</span>, <span class="author">Zia A</span>, <span class="author">Simpson JT</span>, <span class="author">Quail MA</span>, <span class="author">Moses A</span>, <span class="author">Louis EJ</span>, <span class="author">Durbin R</span> and <span class="author">Liti G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, United Kingdom. <a href="mailto:%6C%65%6F%70%6F%6C%64.%70%61%72%74%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#101;&#111;&#112;&#111;&#108;&#100;&#46;&#112;&#97;&#114;&#116;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">One approach to understanding the genetic basis of traits is to study their pattern of inheritance among offspring of phenotypically different parents. Previously, such analysis has been limited by low mapping resolution, high labor costs, and large sample size requirements for detecting modest effects. Here, we present a novel approach to map trait loci using artificial selection. First, we generated populations of 10-100 million haploid and diploid segregants by crossing two budding yeast strains of different heat tolerance for up to 12 generations. We then subjected these large segregant pools to heat stress for up to 12 d, enriching for beneficial alleles. Finally, we sequenced total DNA from the pools before and during selection to measure the changes in parental allele frequency. We mapped 21 intervals with significant changes in genetic background in response to selection, which is several times more than found with traditional linkage methods. Nine of these regions contained two or fewer genes, yielding much higher resolution than previous genomic linkage studies. Multiple members of the RAS/cAMP signaling pathway were implicated, along with genes previously not annotated with heat stress response function. Surprisingly, at most selected loci, allele frequencies stopped changing before the end of the selection experiment, but alleles did not become fixed. Furthermore, we were able to detect the same set of trait loci in a population of diploid individuals with similar power and resolution, and observed primarily additive effects, similar to what is seen for complex trait genetics in other diploid organisms such as humans.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBF0152161</span>; Canadian Institutes of Health Research: <span class="grant">202372</span>; Wellcome Trust: <span class="grant">WT077192/Z/05/Z</span>, <span class="grant">WT084507MA</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">7</span>;<span class="pagerange">1131-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21422276" rel="external">21422276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3129255" rel="external">3129255</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.116731.110" rel="external">10.1101/gr.116731.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Joint genetic analysis of gene expression data with inferred cellular phenotypes.</h4>
<p class="authors"><span class="author">Parts L</span>, <span class="author">Stegle O</span>, <span class="author">Winn J</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%6C%65%6F%70%6F%6C%64.%70%61%72%74%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#101;&#111;&#112;&#111;&#108;&#100;&#46;&#112;&#97;&#114;&#116;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Even within a defined cell type, the expression level of a gene differs in individual samples. The effects of genotype, measured factors such as environmental conditions, and their interactions have been explored in recent studies. Methods have also been developed to identify unmeasured intermediate factors that coherently influence transcript levels of multiple genes. Here, we show how to bring these two approaches together and analyse genetic effects in the context of inferred determinants of gene expression. We use a sparse factor analysis model to infer hidden factors, which we treat as intermediate cellular phenotypes that in turn affect gene expression in a yeast dataset. We find that the inferred phenotypes are associated with locus genotypes and environmental conditions and can explain genetic associations to genes in trans. For the first time, we consider and find interactions between genotype and intermediate phenotypes inferred from gene expression levels, complementing and extending established results.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077192/Z/05/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">e1001276</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21283789">21283789</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3024309">3024309</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001276">10.1371/journal.pgen.1001276</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maps of open chromatin guide the functional follow-up of genome-wide association signals: application to hematological traits.</h4>
<p class="authors"><span class="author">Paul DS</span>, <span class="author">Nisbet JP</span>, <span class="author">Yang TP</span>, <span class="author">Meacham S</span>, <span class="author">Rendon A</span>, <span class="author">Hautaviita K</span>, <span class="author">Tallila J</span>, <span class="author">White J</span>, <span class="author">Tijssen MR</span>, <span class="author">Sivapalaratnam S</span>, <span class="author">Basart H</span>, <span class="author">Trip MD</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">MuTHER Consortium</span>, <span class="author">G&ouml;ttgens B</span>, <span class="author">Soranzo N</span>, <span class="author">Ouwehand WH</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom. <a href="mailto:%64%70%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#112;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Turning genetic discoveries identified in genome-wide association (GWA) studies into biological mechanisms is an important challenge in human genetics. Many GWA signals map outside exons, suggesting that the associated variants may lie within regulatory regions. We applied the formaldehyde-assisted isolation of regulatory elements (FAIRE) method in a megakaryocytic and an erythroblastoid cell line to map active regulatory elements at known loci associated with hematological quantitative traits, coronary artery disease, and myocardial infarction. We showed that the two cell types exhibit distinct patterns of open chromatin and that cell-specific open chromatin can guide the finding of functional variants. We identified an open chromatin region at chromosome 7q22.3 in megakaryocytes but not erythroblasts, which harbors the common non-coding sequence variant rs342293 known to be associated with platelet volume and function. Resequencing of this open chromatin region in 643 individuals provided strong evidence that rs342293 is the only putative causative variant in this region. We demonstrated that the C- and G-alleles differentially bind the transcription factor EVI1 affecting PIK3CG gene expression in platelets and macrophages. A protein-protein interaction network including up- and down-regulated genes in Pik3cg knockout mice indicated that PIK3CG is associated with gene pathways with an established role in platelet membrane biogenesis and thrombus formation. Thus, rs342293 is the functional common variant at this locus; to the best of our knowledge this is the first such variant to be elucidated among the known platelet quantitative trait loci (QTLs). Our data suggested a molecular mechanism by which a non-coding GWA index SNP modulates platelet phenotype.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/12/28096</span>; Medical Research Council: <span class="grant">G0800784</span>, <span class="grant">G0900339</span>, <span class="grant">MC_U105260799</span>; Wellcome Trust: <span class="grant">081917/Z/07/Z</span>, <span class="grant">091746/Z/10/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e1002139</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21738486">21738486</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3128100">3128100</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002139">10.1371/journal.pgen.1002139</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Acquired bleeding disorders</h4>
<p class="authors"><span class="author">Perry,D.J</span> and <span class="author">GROVE,C.</span></p>
<p class="publication"><span class="title">Blood and Bone Marrow Pathology</span>&nbsp;<span class="year">2011</span>;<span class="pagerange">565-82</span></p>
<p class="links">URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780702031472000353" href="http://www.sciencedirect.com/science/article/pii/B9780702031472000353">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780702031472</span>000353</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Citrobacter rodentium is an unstable pathogen showing evidence of significant genomic flux.</h4>
<p class="authors"><span class="author">Petty NK</span>, <span class="author">Feltwell T</span>, <span class="author">Pickard D</span>, <span class="author">Clare S</span>, <span class="author">Toribio AL</span>, <span class="author">Fookes M</span>, <span class="author">Roberts K</span>, <span class="author">Monson R</span>, <span class="author">Nair S</span>, <span class="author">Kingsley RA</span>, <span class="author">Bulgin R</span>, <span class="author">Wiles S</span>, <span class="author">Goulding D</span>, <span class="author">Keane T</span>, <span class="author">Corton C</span>, <span class="author">Lennard N</span>, <span class="author">Harris D</span>, <span class="author">Willey D</span>, <span class="author">Rance R</span>, <span class="author">Yu L</span>, <span class="author">Choudhary JS</span>, <span class="author">Churcher C</span>, <span class="author">Quail MA</span>, <span class="author">Parkhill J</span>, <span class="author">Frankel G</span>, <span class="author">Dougan G</span>, <span class="author">Salmond GP</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Citrobacter rodentium is a natural mouse pathogen that causes attaching and effacing (A/E) lesions. It shares a common virulence strategy with the clinically significant human A/E pathogens enteropathogenic E. coli (EPEC) and enterohaemorrhagic E. coli (EHEC) and is widely used to model this route of pathogenesis. We previously reported the complete genome sequence of C. rodentium ICC168, where we found that the genome displayed many characteristics of a newly evolved pathogen. In this study, through PFGE, sequencing of isolates showing variation, whole genome transcriptome analysis and examination of the mobile genetic elements, we found that, consistent with our previous hypothesis, the genome of C. rodentium is unstable as a result of repeat-mediated, large-scale genome recombination and because of active transposition of mobile genetic elements such as the prophages. We sequenced an additional C. rodentium strain, EX-33, to reveal that the reference strain ICC168 is representative of the species and that most of the inactivating mutations were common to both isolates and likely to have occurred early on in the evolution of this pathogen. We draw parallels with the evolution of other bacterial pathogens and conclude that C. rodentium is a recently evolved pathogen that may have emerged alongside the development of inbred mice as a model for human disease.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">4</span>;<span class="pagerange">e1002018</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21490962">21490962</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3072379">3072379</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002018">10.1371/journal.ppat.1002018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A scalable pipeline for highly effective genetic modification of a malaria parasite.</h4>
<p class="authors"><span class="author">Pfander C</span>, <span class="author">Anar B</span>, <span class="author">Schwach F</span>, <span class="author">Otto TD</span>, <span class="author">Brochet M</span>, <span class="author">Volkmann K</span>, <span class="author">Quail MA</span>, <span class="author">Pain A</span>, <span class="author">Rosen B</span>, <span class="author">Skarnes W</span>, <span class="author">Rayner JC</span> and <span class="author">Billker O</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">In malaria parasites, the systematic experimental validation of drug and vaccine targets by reverse genetics is constrained by the inefficiency of homologous recombination and by the difficulty of manipulating adenine and thymine (A+T)-rich DNA of most Plasmodium species in Escherichia coli. We overcame these roadblocks by creating a high-integrity library of Plasmodium berghei genomic DNA (&gt;77% A+T content) in a bacteriophage N15-based vector that can be modified efficiently using the lambda Red method of recombineering. We built a pipeline for generating P. berghei genetic modification vectors at genome scale in serial liquid cultures on 96-well plates. Vectors have long homology arms, which increase recombination frequency up to tenfold over conventional designs. The feasibility of efficient genetic modification at scale will stimulate collaborative, genome-wide knockout and tagging programs for P. berghei.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>, <span class="grant">G0501670(76331)</span>; Wellcome Trust: <span class="grant">089085</span>, <span class="grant">WT089085/Z/09/Z</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">1078-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22020067">22020067</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3431185">3431185</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1742">10.1038/nmeth.1742</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mendelian randomization study of B-type natriuretic peptide and type 2 diabetes: evidence of causal association from population studies.</h4>
<p class="authors"><span class="author">Pfister R</span>, <span class="author">Sharp S</span>, <span class="author">Luben R</span>, <span class="author">Welsh P</span>, <span class="author">Barroso I</span>, <span class="author">Salomaa V</span>, <span class="author">Meirhaeghe A</span>, <span class="author">Khaw KT</span>, <span class="author">Sattar N</span>, <span class="author">Langenberg C</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Institute of Metabolic Science, University of Cambridge, Cambridge, United Kingdom. <a href="mailto:%72%70%34%31%35%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#114;&#112;&#52;&#49;&#53;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Genetic and epidemiological evidence suggests an inverse association between B-type natriuretic peptide (BNP) levels in blood and risk of type 2 diabetes (T2D), but the prospective association of BNP with T2D is uncertain, and it is unclear whether the association is confounded.</p><p class="abstract">We analysed the association between levels of the N-terminal fragment of pro-BNP (NT-pro-BNP) in blood and risk of incident T2D in a prospective case-cohort study and genotyped the variant rs198389 within the BNP locus in three T2D case-control studies. We combined our results with existing data in a meta-analysis of 11 case-control studies. Using a Mendelian randomization approach, we compared the observed association between rs198389 and T2D to that expected from the NT-pro-BNP level to T2D association and the NT-pro-BNP difference per C allele of rs198389. In participants of our case-cohort study who were free of T2D and cardiovascular disease at baseline, we observed a 21% (95% CI 3%-36%) decreased risk of incident T2D per one standard deviation (SD) higher log-transformed NT-pro-BNP levels in analysis adjusted for age, sex, body mass index, systolic blood pressure, smoking, family history of T2D, history of hypertension, and levels of triglycerides, high-density lipoprotein cholesterol, and low-density lipoprotein cholesterol. The association between rs198389 and T2D observed in case-control studies (odds ratio&#x200A;=&#x200A;0.94 per C allele, 95% CI 0.91-0.97) was similar to that expected (0.96, 0.93-0.98) based on the pooled estimate for the log-NT-pro-BNP level to T2D association derived from a meta-analysis of our study and published data (hazard ratio&#x200A;=&#x200A;0.82 per SD, 0.74-0.90) and the difference in NT-pro-BNP levels (0.22 SD, 0.15-0.29) per C allele of rs198389. No significant associations were observed between the rs198389 genotype and potential confounders.</p><p class="abstract"><strong>Conclusions</strong>: Our results provide evidence for a potential causal role of the BNP system in the aetiology of T2D. Further studies are needed to investigate the mechanisms underlying this association and possibilities for preventive interventions. Please see later in the article for the Editors&#39; Summary.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/10/005/28147</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0601463</span>, <span class="grant">G1000143</span>; Wellcome Trust: <span class="grant">077016/Z/05/Z</span></p>
<p class="publication"><span class="title">PLoS medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e1001112</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22039354">22039354</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3201934">3201934</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pmed.1001112">10.1371/journal.pmed.1001112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phenotype mining in CNV carriers from a population cohort.</h4>
<p class="authors"><span class="author">Pietil&auml;inen OP</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Jakkula E</span>, <span class="author">Service SK</span>, <span class="author">Congdon E</span>, <span class="author">Tilgmann C</span>, <span class="author">Hartikainen AL</span>, <span class="author">Taanila A</span>, <span class="author">Heikura U</span>, <span class="author">Paunio T</span>, <span class="author">Ripatti S</span>, <span class="author">Jarvelin MR</span>, <span class="author">Isohanni M</span>, <span class="author">Sabatti C</span>, <span class="author">Palotie A</span>, <span class="author">Freimer NB</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, and Department of Medical Genetics, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Phenotype mining is a novel approach for elucidating the genetic basis of complex phenotypic variation. It involves a search of rich phenotype databases for measures correlated with genetic variation, as identified in genome-wide genotyping or sequencing studies. An initial implementation of phenotype mining in a prospective unselected population cohort, the Northern Finland 1966 Birth Cohort (NFBC1966), identifies neurodevelopment-related traits-intellectual deficits, poor school performance and hearing abnormalities-which are more frequent among individuals with large (&gt;500 kb) deletions than among other cohort members. Observation of extensive shared single nucleotide polymorphism haplotypes around deletions suggests an opportunity to expand phenotype mining from cohort samples to the populations from which they derive.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; NINDS NIH HHS: <span class="grant">P30NS062691</span>, <span class="grant">PL1NS062410</span>; Wellcome Trust: <span class="grant">WT089061</span>, <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">13</span>;<span class="pagerange">2686-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21505072" rel="external">21505072</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3110003" rel="external">3110003</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr162" rel="external">10.1093/hmg/ddr162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comprehensive assessment of array-based platforms and calling algorithms for detection of copy number variants.</h4>
<p class="authors"><span class="author">Pinto D</span>, <span class="author">Darvishi K</span>, <span class="author">Shi X</span>, <span class="author">Rajan D</span>, <span class="author">Rigler D</span>, <span class="author">Fitzgerald T</span>, <span class="author">Lionel AC</span>, <span class="author">Thiruvahindrapuram B</span>, <span class="author">Macdonald JR</span>, <span class="author">Mills R</span>, <span class="author">Prasad A</span>, <span class="author">Noonan K</span>, <span class="author">Gribble S</span>, <span class="author">Prigmore E</span>, <span class="author">Donahoe PK</span>, <span class="author">Smith RS</span>, <span class="author">Park JH</span>, <span class="author">Hurles ME</span>, <span class="author">Carter NP</span>, <span class="author">Lee C</span>, <span class="author">Scherer SW</span> and <span class="author">Feuk L</span></p>
<p class="affiliation">The Centre for Applied Genomics, The Hospital for Sick Children, Toronto, Ontario, Canada.</p><p class="abstract">We have systematically compared copy number variant (CNV) detection on eleven microarrays to evaluate data quality and CNV calling, reproducibility, concordance across array platforms and laboratory sites, breakpoint accuracy and analysis tool variability. Different analytic tools applied to the same raw data typically yield CNV calls with &lt;50% concordance. Moreover, reproducibility in replicate experiments is &lt;70% for most platforms. Nevertheless, these findings should not preclude detection of large CNVs for clinical diagnostic purposes because large CNVs with poor reproducibility are found primarily in complex genomic regions and would typically be removed by standard clinical data curation. The striking differences between CNV calls from different platforms and analytic tools highlight the importance of careful assessment of experimental design in discovery and association studies and of strict data curation and filtering in diagnostics. The CNV resource presented here allows independent data evaluation and provides a means to benchmark new algorithms.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">213997</span>; NCI NIH HHS: <span class="grant">CA111560</span>, <span class="grant">R01 CA111560-05</span>; NHGRI NIH HHS: <span class="grant">HG004221</span>, <span class="grant">HG005209</span>, <span class="grant">P41 HG004221-03</span>, <span class="grant">U01 HG005209-02</span>; NICHD NIH HHS: <span class="grant">HD007396</span>, <span class="grant">HD055150</span>; NIGMS NIH HHS: <span class="grant">T32 GM007748-33</span>; Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077014</span>, <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2011</span>;<span class="volume">29</span>;<span class="number">6</span>;<span class="pagerange">512-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21552272" rel="external">21552272</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3270583" rel="external">3270583</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.1852" rel="external">10.1038/nbt.1852</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evolutionary genomic approach to identify genes involved in human birth timing.</h4>
<p class="authors"><span class="author">Plunkett J</span>, <span class="author">Doniger S</span>, <span class="author">Orabona G</span>, <span class="author">Morgan T</span>, <span class="author">Haataja R</span>, <span class="author">Hallman M</span>, <span class="author">Puttonen H</span>, <span class="author">Menon R</span>, <span class="author">Kuczynski E</span>, <span class="author">Norwitz E</span>, <span class="author">Snegovskikh V</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen L</span>, <span class="author">Fellman V</span>, <span class="author">DeFranco EA</span>, <span class="author">Chaudhari BP</span>, <span class="author">McGregor TL</span>, <span class="author">McElroy JJ</span>, <span class="author">Oetjens MT</span>, <span class="author">Teramo K</span>, <span class="author">Borecki I</span>, <span class="author">Fay J</span> and <span class="author">Muglia L</span></p>
<p class="affiliation">Department of Pediatrics, Vanderbilt University School of Medicine and Monroe Carell Jr. Children&#39;s Hospital at Vanderbilt, Nashville, Tennessee, United States of America.</p><p class="abstract">Coordination of fetal maturation with birth timing is essential for mammalian reproduction. In humans, preterm birth is a disorder of profound global health significance. The signals initiating parturition in humans have remained elusive, due to divergence in physiological mechanisms between humans and model organisms typically studied. Because of relatively large human head size and narrow birth canal cross-sectional area compared to other primates, we hypothesized that genes involved in parturition would display accelerated evolution along the human and/or higher primate phylogenetic lineages to decrease the length of gestation and promote delivery of a smaller fetus that transits the birth canal more readily. Further, we tested whether current variation in such accelerated genes contributes to preterm birth risk. Evidence from allometric scaling of gestational age suggests human gestation has been shortened relative to other primates. Consistent with our hypothesis, many genes involved in reproduction show human acceleration in their coding or adjacent noncoding regions. We screened &gt;8,400 SNPs in 150 human accelerated genes in 165 Finnish preterm and 163 control mothers for association with preterm birth. In this cohort, the most significant association was in FSHR, and 8 of the 10 most significant SNPs were in this gene. Further evidence for association of a linkage disequilibrium block of SNPs in FSHR, rs11686474, rs11680730, rs12473870, and rs1247381 was found in African Americans. By considering human acceleration, we identified a novel gene that may be associated with preterm birth, FSHR. We anticipate other human accelerated genes will similarly be associated with preterm birth risk and elucidate essential pathways for human parturition.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">T32 GM081739</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">4</span>;<span class="pagerange">e1001365</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21533219" rel="external">21533219</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3077368" rel="external">3077368</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001365" rel="external">10.1371/journal.pgen.1001365</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Jamb and jamc are essential for vertebrate myocyte fusion.</h4>
<p class="authors"><span class="author">Powell GT</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Cellular fusion is required in the development of several tissues, including skeletal muscle. In vertebrates, this process is poorly understood and lacks an in vivo-validated cell surface heterophilic receptor pair that is necessary for fusion. Identification of essential cell surface interactions between fusing cells is an important step in elucidating the molecular mechanism of cellular fusion. We show here that the zebrafish orthologues of JAM-B and JAM-C receptors are essential for fusion of myocyte precursors to form syncytial muscle fibres. Both jamb and jamc are dynamically co-expressed in developing muscles and encode receptors that physically interact. Heritable mutations in either gene prevent myocyte fusion in vivo, resulting in an overabundance of mononuclear, but otherwise overtly normal, functional fast-twitch muscle fibres. Transplantation experiments show that the Jamb and Jamc receptors must interact between neighbouring cells (in trans) for fusion to occur. We also show that jamc is ectopically expressed in prdm1a mutant slow muscle precursors, which inappropriately fuse with other myocytes, suggesting that control of myocyte fusion through regulation of jamc expression has important implications for the growth and patterning of muscles. Our discovery of a receptor-ligand pair critical for fusion in vivo has important implications for understanding the molecular mechanisms responsible for myocyte fusion and its regulation in vertebrate myogenesis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077047/Z/05/Z</span>, <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">e1001216</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22180726">22180726</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3236736">3236736</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1001216">10.1371/journal.pbio.1001216</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A resource of vectors and ES cells for targeted deletion of microRNAs in mice.</h4>
<p class="authors"><span class="author">Prosser HM</span>, <span class="author">Koike-Yusa H</span>, <span class="author">Cooper JD</span>, <span class="author">Law FC</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%68%6D%70%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#109;&#112;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The 21-23 nucleotide, single-stranded RNAs classified as microRNAs (miRNA) perform fundamental roles in diverse cellular and developmental processes. In contrast to the situation for protein-coding genes, no public resource of miRNA mouse mutant alleles exists. Here we describe a collection of 428 miRNA targeting vectors covering 476 of the miRNA genes annotated in the miRBase registry. Using these vectors, we generated a library of highly germline-transmissible C57BL/6N mouse embryonic stem (ES) cell clones harboring targeted deletions for 392 miRNA genes. For most of these targeted clones, chimerism and germline transmission can be scored through a coat color marker. The targeted alleles have been designed to be adaptable research tools that can be efficiently altered by recombinase-mediated cassette exchange to create reporter, conditional and other allelic variants. This miRNA knockout (mirKO) resource can be searched electronically and is available from ES cell repositories for distribution to the scientific community.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span>, <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2011</span>;<span class="volume">29</span>;<span class="number">9</span>;<span class="pagerange">840-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21822254">21822254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3242032">3242032</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.1929">10.1038/nbt.1929</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing identifies recurrent mutations in chronic lymphocytic leukaemia.</h4>
<p class="authors"><span class="author">Puente XS</span>, <span class="author">Pinyol M</span>, <span class="author">Quesada V</span>, <span class="author">Conde L</span>, <span class="author">Ord&oacute;&ntilde;ez GR</span>, <span class="author">Villamor N</span>, <span class="author">Escaramis G</span>, <span class="author">Jares P</span>, <span class="author">Be&agrave; S</span>, <span class="author">Gonz&aacute;lez-D&iacute;az M</span>, <span class="author">Bassaganyas L</span>, <span class="author">Baumann T</span>, <span class="author">Juan M</span>, <span class="author">L&oacute;pez-Guerra M</span>, <span class="author">Colomer D</span>, <span class="author">Tub&iacute;o JM</span>, <span class="author">L&oacute;pez C</span>, <span class="author">Navarro A</span>, <span class="author">Tornador C</span>, <span class="author">Aymerich M</span>, <span class="author">Rozman M</span>, <span class="author">Hern&aacute;ndez JM</span>, <span class="author">Puente DA</span>, <span class="author">Freije JM</span>, <span class="author">Velasco G</span>, <span class="author">Guti&eacute;rrez-Fern&aacute;ndez A</span>, <span class="author">Costa D</span>, <span class="author">Carri&oacute; A</span>, <span class="author">Guijarro S</span>, <span class="author">Enjuanes A</span>, <span class="author">Hern&aacute;ndez L</span>, <span class="author">Yag&uuml;e J</span>, <span class="author">Nicol&aacute;s P</span>, <span class="author">Romeo-Casabona CM</span>, <span class="author">Himmelbauer H</span>, <span class="author">Castillo E</span>, <span class="author">Dohm JC</span>, <span class="author">de Sanjos&eacute; S</span>, <span class="author">Piris MA</span>, <span class="author">de Alava E</span>, <span class="author">San Miguel J</span>, <span class="author">Royo R</span>, <span class="author">Gelp&iacute; JL</span>, <span class="author">Torrents D</span>, <span class="author">Orozco M</span>, <span class="author">Pisano DG</span>, <span class="author">Valencia A</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Bay&eacute;s M</span>, <span class="author">Heath S</span>, <span class="author">Gut M</span>, <span class="author">Klatt P</span>, <span class="author">Marshall J</span>, <span class="author">Raine K</span>, <span class="author">Stebbings LA</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span>, <span class="author">Gut I</span>, <span class="author">L&oacute;pez-Guillermo A</span>, <span class="author">Estivill X</span>, <span class="author">Montserrat E</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span> and <span class="author">Campo E</span></p>
<p class="affiliation">Departamento de Bioqu&iacute;mica y Biolog&iacute;a Molecular, Instituto Universitario de Oncolog&iacute;a, Universidad de Oviedo, 33006 Oviedo, Spain.</p><p class="abstract">Chronic lymphocytic leukaemia (CLL), the most frequent leukaemia in adults in Western countries, is a heterogeneous disease with variable clinical presentation and evolution. Two major molecular subtypes can be distinguished, characterized respectively by a high or low number of somatic hypermutations in the variable region of immunoglobulin genes. The molecular changes leading to the pathogenesis of the disease are still poorly understood. Here we performed whole-genome sequencing of four cases of CLL and identified 46 somatic mutations that potentially affect gene function. Further analysis of these mutations in 363 patients with CLL identified four genes that are recurrently mutated: notch 1 (NOTCH1), exportin 1 (XPO1), myeloid differentiation primary response gene 88 (MYD88) and kelch-like 6 (KLHL6). Mutations in MYD88 and KLHL6 are predominant in cases of CLL with mutated immunoglobulin genes, whereas NOTCH1 and XPO1 mutations are mainly detected in patients with unmutated immunoglobulins. The patterns of somatic mutation, supported by functional and clinical analyses, strongly indicate that the recurrent NOTCH1, MYD88 and XPO1 mutations are oncogenic changes that contribute to the clinical evolution of the disease. To our knowledge, this is the first comprehensive analysis of CLL combining whole-genome sequencing with clinical characteristics and clinical outcomes. It highlights the usefulness of this approach for the identification of clinically relevant mutations in cancer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">475</span>;<span class="number">7354</span>;<span class="pagerange">101-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21642962" rel="external">21642962</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322590" rel="external">3322590</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10113" rel="external">10.1038/nature10113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic libraries: I. Construction and screening of fosmid genomic libraries.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Matthews L</span>, <span class="author">Sims S</span>, <span class="author">Lloyd C</span>, <span class="author">Beasley H</span> and <span class="author">Baxter SW</span></p>
<p class="affiliation">Sequencing Research and Development, Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">Large insert genome libraries have been a core resource required to sequence genomes, analyze haplotypes, and aid gene discovery. While next generation sequencing technologies are revolutionizing the field of genomics, traditional genome libraries will still be required for accurate genome assembly. Their utility is also being extended to functional studies for understanding DNA regulatory elements. Here, we present a detailed method for constructing genomic fosmid libraries, testing for common contaminants, gridding the library to nylon membranes, then hybridizing the library membranes with a radiolabeled probe to identify corresponding genomic clones. While this chapter focuses on fosmid libraries, many of these steps can also be applied to bacterial artificial chromosome libraries.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">772</span>;<span class="pagerange">37-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22065431">22065431</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-61779-228-1_3">10.1007/978-1-61779-228-1_3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic libraries: II. Subcloning, sequencing, and assembling large-insert genomic DNA clones.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Matthews L</span>, <span class="author">Sims S</span>, <span class="author">Lloyd C</span>, <span class="author">Beasley H</span> and <span class="author">Baxter SW</span></p>
<p class="affiliation">Sequencing Research and Development, Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">Sequencing large insert clones to completion is useful for characterizing specific genomic regions, identifying haplotypes, and closing gaps in whole genome sequencing projects. Despite being a standard technique in molecular laboratories, DNA sequencing using the Sanger method can be highly problematic when complex secondary structures or sequence repeats are encountered in genomic clones. Here, we describe methods to isolate DNA from a large insert clone (fosmid or BAC), subclone the sample, and sequence the region to the highest industry standard. Troubleshooting solutions for sequencing difficult templates are discussed.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">772</span>;<span class="pagerange">59-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22065432">22065432</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-61779-228-1_4">10.1007/978-1-61779-228-1_4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic and transcriptional analysis of phosphoinositide-specific phospholipase C in Plasmodium.</h4>
<p class="authors"><span class="author">Raabe A</span>, <span class="author">Berry L</span>, <span class="author">Sollelis L</span>, <span class="author">Cerdan R</span>, <span class="author">Tawk L</span>, <span class="author">Vial HJ</span>, <span class="author">Billker O</span> and <span class="author">Wengelnik K</span></p>
<p class="affiliation">UMR5235, CNRS-Universit&eacute; Montpellier 2, 34095 Montpellier, France.</p><p class="abstract">Phosphoinositide-specific phospholipase C (PI-PLC) is a major regulator of calcium-dependent signal transduction, which has been shown to be important in various processes of the malaria parasite Plasmodium. PI-PLC is generally implicated in calcium liberation from intracellular stores through the action of its product, inositol-(1,4,5)-trisphosphate, and is itself dependent on calcium for its activation. Here we describe the plc genes from Plasmodium species. The encoded proteins contain all domains typically found in PI-PLCs of the &delta; class but are almost twice as long as their orthologues in mammals. Transcriptional analysis by qRT-PCR of plc during the erythrocytic cycle of P. falciparum revealed steady expression levels that increased at the late schizont stages. Genetic analysis in the P. berghei model revealed that the plc locus was targetable but that plc gene knock-outs could not be obtained, thereby strongly indicating that the gene is essential during blood stage development. Alternatively, we attempted to modify plc expression through a promoter exchange approach but found the gene to be refractory to over-expression indicating that plc expression levels might additionally be tightly controlled.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; Wellcome Trust: <span class="grant">WT089085/Z/09/Z</span></p>
<p class="publication"><span class="title">Experimental parasitology</span>&nbsp;<span class="year">2011</span>;<span class="volume">129</span>;<span class="number">1</span>;<span class="pagerange">75-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21651909" rel="external">21651909</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.exppara.2011.05.023" rel="external">10.1016/j.exppara.2011.05.023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple roles for Plasmodium berghei phosphoinositide-specific phospholipase C in regulating gametocyte activation and differentiation.</h4>
<p class="authors"><span class="author">Raabe AC</span>, <span class="author">Wengelnik K</span>, <span class="author">Billker O</span> and <span class="author">Vial HJ</span></p>
<p class="affiliation">UMR5235, CNRS-Universit&eacute; Montpellier 2, Place Eug&egrave;ne Bataillon, Montpellier cedex 5, France.</p><p class="abstract">Critical events in the life cycle of malaria parasites are controlled by calcium-dependent signalling cascades, yet the molecular mechanisms of calcium release remain poorly understood. The synchronized development of Plasmodium berghei gametocytes relies on rapid calcium release from internal stores within 10&emsp;s of gametocytes being exposed to mosquito-derived xanthurenic acid (XA). Here we addressed the function of phosphoinositide-specific phospholipase C (PI-PLC) for regulating gametocyte activation. XA triggered the hydrolysis of PIP(2) and the production of the secondary messenger IP(3) in gametocytes. Both processes were selectively blocked by a PI-PLC inhibitor, which also reduced the early Ca(2+) signal. However, microgametocyte differentiation into microgametes was blocked even when the inhibitor was added up to 5&emsp;min after activation, suggesting a requirement for PI-PLC beyond the early mobilization of calcium. In contrast, inhibitors of calcium release through ryanodine receptor channels were active only during the first minute of gametocyte activation. Biochemical determination of PI-PLC activity was confirmed using transgenic parasites expressing a fluorescent PIP(2) /IP(3) probe that translocates from the parasite plasmalemma to the cytosol upon cell activation. Our study revealed a complex interdependency of Ca(2+) and PI-PLC activity, with PI-PLC being essential throughout gamete formation, possibly explaining the irreversibility of this process.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; Wellcome Trust: <span class="grant">WT089085/Z/09/Z</span></p>
<p class="publication"><span class="title">Cellular microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">13</span>;<span class="number">7</span>;<span class="pagerange">955-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21518218" rel="external">21518218</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3132445" rel="external">3132445</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1462-5822.2011.01591.x" rel="external">10.1111/j.1462-5822.2011.01591.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Early Diagnosis of Werner&#39;s Syndrome Using Exome-Wide Sequencing in a Single, Atypical Patient.</h4>
<p class="authors"><span class="author">Raffan E</span>, <span class="author">Hurst LA</span>, <span class="author">Turki SA</span>, <span class="author">Carpenter G</span>, <span class="author">Scott C</span>, <span class="author">Daly A</span>, <span class="author">Coffey A</span>, <span class="author">Bhaskar S</span>, <span class="author">Howard E</span>, <span class="author">Khan N</span>, <span class="author">Kingston H</span>, <span class="author">Palotie A</span>, <span class="author">Savage DB</span>, <span class="author">O&#39;Driscoll M</span>, <span class="author">Smith C</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Barroso I</span> and <span class="author">Semple RK</span></p>
<p class="affiliation">Institute of Metabolic Science, University of Cambridge Metabolic Research Laboratories Cambridge, UK.</p><p class="abstract">Genetic diagnosis of inherited metabolic disease is conventionally achieved through syndrome recognition and targeted gene sequencing, but many patients receive no specific diagnosis. Next-generation sequencing allied to capture of expressed sequences from genomic DNA now offers a powerful new diagnostic approach. Barriers to routine diagnostic use include cost, and the complexity of interpreting results arising from simultaneous identification of large numbers of variants. We applied exome-wide sequencing to an individual, 16-year-old daughter of consanguineous parents with a novel syndrome of short stature, severe insulin resistance, ptosis, and microcephaly. Pulldown of expressed sequences from genomic DNA followed by massively parallel sequencing was undertaken. Single nucleotide variants were called using SAMtools prior to filtering based on sequence quality and existence in control genomes and exomes. Of 485 genetic variants predicted to alter protein sequence and absent from control data, 24 were homozygous in the patient. One mutation - the p.Arg732X mutation in the WRN gene - has previously been reported in Werner&#39;s syndrome (WS). On re-evaluation of the patient several early features of WS were detected including loss of fat from the extremities and frontal hair thinning. Lymphoblastoid cells from the proband exhibited a defective decatenation checkpoint, consistent with loss of WRN activity. We have thus diagnosed WS some 15&thinsp;years earlier than average, permitting aggressive prophylactic therapy and screening for WS complications, illustrating the potential of exome-wide sequencing to achieve early diagnosis and change management of rare autosomal recessive disease, even in individual patients of consanguineous parentage with apparently novel syndromes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">8300</span>; Medical Research Council: <span class="grant">G0700733</span></p>
<p class="publication"><span class="title">Frontiers in endocrinology</span>&nbsp;<span class="year">2011</span>;<span class="volume">2</span>;<span class="pagerange">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22654791">22654791</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3356119">3356119</a>; DOI: <a href="http://dx.doi.org/10.3389%2Ffendo.2011.00008">10.3389/fendo.2011.00008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Founder effect in the Horn of Africa for an insulin receptor mutation that may impair receptor recycling.</h4>
<p class="authors"><span class="author">Raffan E</span>, <span class="author">Soos MA</span>, <span class="author">Rocha N</span>, <span class="author">Tuthill A</span>, <span class="author">Thomsen AR</span>, <span class="author">Hyden CS</span>, <span class="author">Gregory JW</span>, <span class="author">Hindmarsh P</span>, <span class="author">Dattani M</span>, <span class="author">Cochran E</span>, <span class="author">Al Kaabi J</span>, <span class="author">Gorden P</span>, <span class="author">Barroso I</span>, <span class="author">Morling N</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Semple RK</span></p>
<p class="affiliation">University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, University of Cambridge, Addenbrooke&#39;s Hospital B289, Cambridge, CB2 0QR, UK.</p><p class="abstract">Genetic insulin receptoropathies are a rare cause of severe insulin resistance. We identified the Ile119Met missense mutation in the insulin receptor INSR gene, previously reported in a Yemeni kindred, in four unrelated patients with Somali ancestry. We aimed to investigate a possible genetic founder effect, and to study the mechanism of loss of function of the mutant receptor.</p><p class="abstract"><strong>Methods</strong>: Biochemical profiling and DNA haplotype analysis of affected patients were performed. Insulin receptor expression in lymphoblastoid cells from a homozygous p.Ile119Met INSR patient, and in cells heterologously expressing the mutant receptor, was examined. Insulin binding, insulin-stimulated receptor autophosphorylation, and cooperativity and pH dependency of insulin dissociation were also assessed.</p><p class="abstract"><strong>Results</strong>: All patients had biochemical profiles pathognomonic of insulin receptoropathy, while haplotype analysis revealed the putative shared region around the INSR mutant to be no larger than 28 kb. An increased insulin proreceptor to &beta; subunit ratio was seen in patient-derived cells. Steady state insulin binding and insulin-stimulated autophosphorylation of the mutant receptor was normal; however it exhibited decreased insulin dissociation rates with preserved cooperativity, a difference accentuated at low pH.</p><p class="abstract">The p.Ile119Met INSR appears to have arisen around the Horn of Africa, and should be sought first in severely insulin resistant patients with ancestry from this region. Despite collectively compelling genetic, clinical and biochemical evidence for its pathogenicity, loss of function in conventional in vitro assays is subtle, suggesting mildly impaired receptor recycling only.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">077016/Z/05/Z</span>, <span class="grant">078986/Z/06/Z</span>, <span class="grant">080952/Z/06/Z</span>, <span class="grant">087678/Z/08/Z</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2011</span>;<span class="volume">54</span>;<span class="number">5</span>;<span class="pagerange">1057-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21318406">21318406</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3071941">3071941</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-011-2066-z">10.1007/s00125-011-2066-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evidence that Cd101 is an autoimmune diabetes gene in nonobese diabetic mice.</h4>
<p class="authors"><span class="author">Rainbow DB</span>, <span class="author">Moule C</span>, <span class="author">Fraser HI</span>, <span class="author">Clark J</span>, <span class="author">Howlett SK</span>, <span class="author">Burren O</span>, <span class="author">Christensen M</span>, <span class="author">Moody V</span>, <span class="author">Steward CA</span>, <span class="author">Mohammed JP</span>, <span class="author">Fusakio ME</span>, <span class="author">Masteller EL</span>, <span class="author">Finger EB</span>, <span class="author">Houchins JP</span>, <span class="author">Naf D</span>, <span class="author">Koentgen F</span>, <span class="author">Ridgway WM</span>, <span class="author">Todd JA</span>, <span class="author">Bluestone JA</span>, <span class="author">Peterson LB</span>, <span class="author">Mattner J</span> and <span class="author">Wicker LS</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, United Kingdom.</p><p class="abstract">We have previously proposed that sequence variation of the CD101 gene between NOD and C57BL/6 mice accounts for the protection from type 1 diabetes (T1D) provided by the insulin-dependent diabetes susceptibility region 10 (Idd10), a &lt;1 Mb region on mouse chromosome 3. In this study, we provide further support for the hypothesis that Cd101 is Idd10 using haplotype and expression analyses of novel Idd10 congenic strains coupled to the development of a CD101 knockout mouse. Susceptibility to T1D was correlated with genotype-dependent CD101 expression on multiple cell subsets, including Foxp3(+) regulatory CD4(+) T cells, CD11c(+) dendritic cells, and Gr1(+) myeloid cells. The correlation of CD101 expression on immune cells from four independent Idd10 haplotypes with the development of T1D supports the identity of Cd101 as Idd10. Because CD101 has been associated with regulatory T and Ag presentation cell functions, our results provide a further link between immune regulation and susceptibility to T1D.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 15416</span>, <span class="grant">N01 AI015416</span>, <span class="grant">P01 AI039671</span>, <span class="grant">P01 AI039671-16</span>, <span class="grant">P01AI039671</span>; NIDDK NIH HHS: <span class="grant">P30 DK078392</span>, <span class="grant">P30 DK078392-01</span>, <span class="grant">R01 DK084054</span>, <span class="grant">R01 DK084054-03</span>, <span class="grant">R01DK084054</span>; Wellcome Trust: <span class="grant">079895</span>, <span class="grant">091157</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2011</span>;<span class="volume">187</span>;<span class="number">1</span>;<span class="pagerange">325-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21613616">21613616</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3128927">3128927</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1003523">10.4049/jimmunol.1003523</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cutting edge: the membrane attack complex of complement is required for the development of murine experimental cerebral malaria.</h4>
<p class="authors"><span class="author">Ramos TN</span>, <span class="author">Darley MM</span>, <span class="author">Hu X</span>, <span class="author">Billker O</span>, <span class="author">Rayner JC</span>, <span class="author">Ahras M</span>, <span class="author">Wohler JE</span> and <span class="author">Barnum SR</span></p>
<p class="affiliation">Department of Microbiology, University of Alabama at Birmingham, Birmingham, AL 35294, USA.</p><p class="abstract">Cerebral malaria is the most severe complication of Plasmodium falciparum infection and accounts for a large number of malaria fatalities worldwide. Recent studies demonstrated that C5(-/-) mice are resistant to experimental cerebral malaria (ECM) and suggested that protection was due to loss of C5a-induced inflammation. Surprisingly, we observed that C5aR(-/-) mice were fully susceptible to disease, indicating that C5a is not required for ECM. C3aR(-/-) and C3aR(-/-) &times; C5aR(-/-) mice were equally susceptible to ECM as were wild-type mice, indicating that neither complement anaphylatoxin receptor is critical for ECM development. In contrast, C9 deposition in the brains of mice with ECM suggested an important role for the terminal complement pathway. Treatment with anti-C9 Ab significantly increased survival time and reduced mortality in ECM. Our data indicate that protection from ECM in C5(-/-) mice is mediated through inhibition of membrane attack complex formation and not through C5a-induced inflammation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; NIAID NIH HHS: <span class="grant">AI08382</span>, <span class="grant">R03 AI083820</span>, <span class="grant">R03 AI083820-02</span>, <span class="grant">T32 AI007051</span>, <span class="grant">T32 AI007051-35</span>, <span class="grant">T32 AI07051</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2011</span>;<span class="volume">186</span>;<span class="number">12</span>;<span class="pagerange">6657-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21572031">21572031</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3110530">3110530</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1100603">10.4049/jimmunol.1100603</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Kino: A generic document management system for biologists using SA-REST and faceted search</h4>
<p class="authors"><span class="author">Ranabahu A</span>, <span class="author">Parikh P</span>, <span class="author">Panahiazar M</span>, <span class="author">Sheth A</span>, <span class="author">LOGAN-KLUMPLER F</span></p>
<p class="publication"><span class="title">Proceedings - 5th IEEE International Conference on Semantic Computing, ICSC</span> <span class="year">2011</span>;<span class="pagerange">205-8</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Asparagine peptide lyases: a seventh catalytic type of proteolytic enzymes.</h4>
<p class="authors"><span class="author">Rawlings ND</span>, <span class="author">Barrett AJ</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Genome Campus, The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, United Kingdom. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The terms &quot;proteolytic enzyme&quot; and &quot;peptidase&quot; have been treated as synonymous, and all proteolytic enzymes have been considered to be hydrolases (EC 3.4). However, the recent discovery of proteins that cleave themselves at asparagine residues indicates that not all peptide bond cleavage occurs by hydrolysis. These self-cleaving proteins include the Tsh protein precursor of Escherichia coli, in which the large C-terminal propeptide acts as an autotransporter; certain viral coat proteins; and proteins containing inteins. Proteolysis is the action of an amidine lyase (EC 4.3.2). These proteolytic enzymes are also the first in which the nucleophile is an asparagine, defining the seventh proteolytic catalytic type and the first to be discovered since 2004. We have assembled ten families based on sequence similarity in which cleavage is thought to be catalyzed by an asparagine.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2011</span>;<span class="volume">286</span>;<span class="number">44</span>;<span class="pagerange">38321-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21832066" rel="external">21832066</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3207474" rel="external">3207474</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M111.260026" rel="external">10.1074/jbc.M111.260026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A plethora of Plasmodium species in wild apes: a source of human infection?</h4>
<p class="authors"><span class="author">Rayner JC</span>, <span class="author">Liu W</span>, <span class="author">Peeters M</span>, <span class="author">Sharp PM</span> and <span class="author">Hahn BH</span></p>
<p class="affiliation">Malaria Programme, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6A%75%6C%69%61%6E.%72%61%79%6E%65%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#117;&#108;&#105;&#97;&#110;&#46;&#114;&#97;&#121;&#110;&#101;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Recent studies of captive and wild-living apes in Africa have uncovered evidence of numerous new Plasmodium species, one of which was identified as the immediate precursor of human Plasmodium falciparum. These findings raise the question whether wild apes could be a recurrent source of Plasmodium infections in humans. This question is not new, but was the subject of intense investigation by researchers in the first half of the last century. Re-examination of their work in the context of recent molecular findings provides a new framework to understand the diversity of Plasmodium species and to assess the risk of future cross-species transmissions to humans in the context of proposed malaria eradication programs.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">P30 AI 27767</span>, <span class="grant">R01 AI091595</span>, <span class="grant">R01 AI50529</span>, <span class="grant">R01 AI58715</span>, <span class="grant">R03 AI074778</span>, <span class="grant">R37 AI050529</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2011</span>;<span class="volume">27</span>;<span class="number">5</span>;<span class="pagerange">222-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21354860">21354860</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3087880">3087880</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2011.01.006">10.1016/j.pt.2011.01.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Retrotransposon-induced heterochromatin spreading in the mouse revealed by insertional polymorphisms.</h4>
<p class="authors"><span class="author">Rebollo R</span>, <span class="author">Karimi MM</span>, <span class="author">Bilenky M</span>, <span class="author">Gagnier L</span>, <span class="author">Miceli-Royer K</span>, <span class="author">Zhang Y</span>, <span class="author">Goyal P</span>, <span class="author">Keane TM</span>, <span class="author">Jones S</span>, <span class="author">Hirst M</span>, <span class="author">Lorincz MC</span> and <span class="author">Mager DL</span></p>
<p class="affiliation">Terry Fox Laboratory, British Columbia Cancer Agency, Vancouver, Canada.</p><p class="abstract">The &quot;arms race&quot; relationship between transposable elements (TEs) and their host has promoted a series of epigenetic silencing mechanisms directed against TEs. Retrotransposons, a class of TEs, are often located in repressed regions and are thought to induce heterochromatin formation and spreading. However, direct evidence for TE-induced local heterochromatin in mammals is surprisingly scarce. To examine this phenomenon, we chose two mouse embryonic stem (ES) cell lines that possess insertionally polymorphic retrotransposons (IAP, ETn/MusD, and LINE elements) at specific loci in one cell line but not the other. Employing ChIP-seq data for these cell lines, we show that IAP elements robustly induce H3K9me3 and H4K20me3 marks in flanking genomic DNA. In contrast, such heterochromatin is not induced by LINE copies and only by a minority of polymorphic ETn/MusD copies. DNA methylation is independent of the presence of IAP copies, since it is present in flanking regions of both full and empty sites. Finally, such spreading into genes appears to be rare, since the transcriptional start sites of very few genes are less than one Kb from an IAP. However, the B3galtl gene is subject to transcriptional silencing via IAP-induced heterochromatin. Hence, although rare, IAP-induced local heterochromatin spreading into nearby genes may influence expression and, in turn, host fitness.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">10825</span>, <span class="grant">92090</span>, <span class="grant">92093</span>; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e1002301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21980304" rel="external">21980304</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3183085" rel="external">3183085</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002301" rel="external">10.1371/journal.pgen.1002301</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Determinants of R-loop formation at convergent bidirectionally transcribed trinucleotide repeats.</h4>
<p class="authors"><span class="author">Reddy K</span>, <span class="author">Tam M</span>, <span class="author">Bowater RP</span>, <span class="author">Barber M</span>, <span class="author">Tomlinson M</span>, <span class="author">Nichol Edamura K</span>, <span class="author">Wang YH</span> and <span class="author">Pearson CE</span></p>
<p class="affiliation">Program of Genetics &amp; Genome Biology, The Hospital for Sick Children, 101 College Street, East Tower, 15-312 TMDT, Toronto, Ontario M5G 1L7, Canada.</p><p class="abstract">R-loops have been described at immunoglobulin class switch sequences, prokaryotic and mitochondrial replication origins, and disease-associated (CAG)n and (GAA)n trinucleotide repeats. The determinants of trinucleotide R-loop formation are unclear. Trinucleotide repeat expansions cause diseases including DM1 (CTG)n, SCA1 (CAG)n, FRAXA (CGG)n, FRAXE (CCG)n and FRDA (GAA)n. Bidirectional convergent transcription across these disease repeats can occur. We find R-loops formed when CTG or CGG and their complementary strands CAG or CCG were transcribed; GAA transcription, but not TTC, yielded R-loops. R-loop formation was sensitive to DNA supercoiling, repeat length, insensitive to repeat interruptions, and formed by extension of RNA:DNA hybrids in the RNA polymerase. R-loops arose by transcription in one direction followed by transcription in the opposite direction, and during simultaneous convergent bidirectional transcription of the same repeat forming double R-loop structures. Since each transcribed disease repeat formed R-loops suggests they may have biological functions.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">MOP-94966</span>; NCI NIH HHS: <span class="grant">R01 CA085826</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">5</span>;<span class="pagerange">1749-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21051337" rel="external">21051337</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3061079" rel="external">3061079</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq935" rel="external">10.1093/nar/gkq935</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequencing gets func-y.</h4>
<p class="authors"><span class="author">Reid AJ</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">6</span>;<span class="pagerange">401</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21572456">21572456</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2583">10.1038/nrmicro2583</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effects of chronic ascariasis and trichuriasis on cytokine production and gene expression in human blood: a cross-sectional study.</h4>
<p class="authors"><span class="author">Reina Ortiz M</span>, <span class="author">Schreiber F</span>, <span class="author">Benitez S</span>, <span class="author">Broncano N</span>, <span class="author">Chico ME</span>, <span class="author">Vaca M</span>, <span class="author">Alexander N</span>, <span class="author">Lewis DJ</span>, <span class="author">Dougan G</span> and <span class="author">Cooper PJ</span></p>
<p class="affiliation">Centro de Investigaciones, Fundaci&oacute;n Ecuatoriana Para Investigaci&oacute;n en Salud, Quinind&eacute;, Ecuador.</p><p class="abstract"><strong>Background</strong>: Chronic soil-transmitted helminth (STH) infections are associated with effects on systemic immune responses that could be caused by alterations in immune homeostasis. To investigate this, we measured the impact in children of STH infections on cytokine responses and gene expression in unstimulated blood.</p><p class="abstract">Sixty children were classified as having chronic, light, or no STH infections. Peripheral blood mononuclear cells were cultured in medium for 5 days to measure cytokine accumulation. RNA was isolated from peripheral blood and gene expression analysed using microarrays. Different infection groups were compared for the purpose of analysis: STH infection (combined chronic and light vs. uninfected groups) and chronic STH infection (chronic vs. combined light and uninfected groups). The chronic STH infection effect was associated with elevated production of GM-CSF (P=0.007), IL-2 (P=0.03), IL-5 (P=0.01), and IL-10 (P=0.01). Data reduction suggested that chronic infections were primarily associated with an immune phenotype characterized by elevated IL-5 and IL-10, typical of a modified Th2-like response. Chronic STH infections were associated with the up-regulation of genes associated with immune homeostasis (IDO, P=0.03; CCL23, P=0.008, HRK, P=0.005), down-regulation of microRNA hsa-let-7d (P=0.01) and differential regulation of several genes associated with granulocyte-mediated inflammation (IL-8, down-regulated, P=0.0002; RNASE2, up-regulated, P=0.009; RNASE3, up-regulated, p=0.03).</p><p class="abstract">Chronic STH infections were associated with a cytokine response indicative of a modified Th2 response. There was evidence that STH infections were associated with a pattern of gene expression suggestive of the induction of homeostatic mechanisms, the differential expression of several inflammatory genes and the down-regulation of microRNA has-let-7d. Effects on immune homeostasis and the development of a modified Th2 immune response during chronic STH infections could explain the systemic immunologic effects that have been associated with these infections such as impaired immune responses to vaccines and the suppression of inflammatory diseases.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">074679/Z/04/Z</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e1157</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21666788" rel="external">21666788</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3110165" rel="external">3110165</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001157" rel="external">10.1371/journal.pntd.0001157</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of an Australian kangaroo, Macropus eugenii, provides insight into the evolution of mammalian reproduction and development.</h4>
<p class="authors"><span class="author">Renfree MB</span>, <span class="author">Papenfuss AT</span>, <span class="author">Deakin JE</span>, <span class="author">Lindsay J</span>, <span class="author">Heider T</span>, <span class="author">Belov K</span>, <span class="author">Rens W</span>, <span class="author">Waters PD</span>, <span class="author">Pharo EA</span>, <span class="author">Shaw G</span>, <span class="author">Wong ES</span>, <span class="author">Lef&egrave;vre CM</span>, <span class="author">Nicholas KR</span>, <span class="author">Kuroki Y</span>, <span class="author">Wakefield MJ</span>, <span class="author">Zenger KR</span>, <span class="author">Wang C</span>, <span class="author">Ferguson-Smith M</span>, <span class="author">Nicholas FW</span>, <span class="author">Hickford D</span>, <span class="author">Yu H</span>, <span class="author">Short KR</span>, <span class="author">Siddle HV</span>, <span class="author">Frankenberg SR</span>, <span class="author">Chew KY</span>, <span class="author">Menzies BR</span>, <span class="author">Stringer JM</span>, <span class="author">Suzuki S</span>, <span class="author">Hore TA</span>, <span class="author">Delbridge ML</span>, <span class="author">Mohammadi A</span>, <span class="author">Schneider NY</span>, <span class="author">Hu Y</span>, <span class="author">O&#39;Hara W</span>, <span class="author">Al Nadaf S</span>, <span class="author">Wu C</span>, <span class="author">Feng ZP</span>, <span class="author">Cocks BG</span>, <span class="author">Wang J</span>, <span class="author">Flicek P</span>, <span class="author">Searle SM</span>, <span class="author">Fairley S</span>, <span class="author">Beal K</span>, <span class="author">Herrero J</span>, <span class="author">Carone DM</span>, <span class="author">Suzuki Y</span>, <span class="author">Sugano S</span>, <span class="author">Toyoda A</span>, <span class="author">Sakaki Y</span>, <span class="author">Kondo S</span>, <span class="author">Nishida Y</span>, <span class="author">Tatsumoto S</span>, <span class="author">Mandiou I</span>, <span class="author">Hsu A</span>, <span class="author">McColl KA</span>, <span class="author">Lansdell B</span>, <span class="author">Weinstock G</span>, <span class="author">Kuczek E</span>, <span class="author">McGrath A</span>, <span class="author">Wilson P</span>, <span class="author">Men A</span>, <span class="author">Hazar-Rethinam M</span>, <span class="author">Hall A</span>, <span class="author">Davis J</span>, <span class="author">Wood D</span>, <span class="author">Williams S</span>, <span class="author">Sundaravadanam Y</span>, <span class="author">Muzny DM</span>, <span class="author">Jhangiani SN</span>, <span class="author">Lewis LR</span>, <span class="author">Morgan MB</span>, <span class="author">Okwuonu GO</span>, <span class="author">Ruiz SJ</span>, <span class="author">Santibanez J</span>, <span class="author">Nazareth L</span>, <span class="author">Cree A</span>, <span class="author">Fowler G</span>, <span class="author">Kovar CL</span>, <span class="author">Dinh HH</span>, <span class="author">Joshi V</span>, <span class="author">Jing C</span>, <span class="author">Lara F</span>, <span class="author">Thornton R</span>, <span class="author">Chen L</span>, <span class="author">Deng J</span>, <span class="author">Liu Y</span>, <span class="author">Shen JY</span>, <span class="author">Song XZ</span>, <span class="author">Edson J</span>, <span class="author">Troon C</span>, <span class="author">Thomas D</span>, <span class="author">Stephens A</span>, <span class="author">Yapa L</span>, <span class="author">Levchenko T</span>, <span class="author">Gibbs RA</span>, <span class="author">Cooper DW</span>, <span class="author">Speed TP</span>, <span class="author">Fujiyama A</span>, <span class="author">Graves JA</span>, <span class="author">O&#39;Neill RJ</span>, <span class="author">Pask AJ</span>, <span class="author">Forrest SM</span> and <span class="author">Worley KC</span></p>
<p class="affiliation">The Australian Research Council Centre of Excellence in Kangaroo Genomics, Australia. <a href="mailto:%6D.%72%65%6E%66%72%65%65%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#109;&#46;&#114;&#101;&#110;&#102;&#114;&#101;&#101;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract"><strong>Background</strong>: We present the genome sequence of the tammar wallaby, Macropus eugenii, which is a member of the kangaroo family and the first representative of the iconic hopping mammals that symbolize Australia to be sequenced. The tammar has many unusual biological characteristics, including the longest period of embryonic diapause of any mammal, extremely synchronized seasonal breeding and prolonged and sophisticated lactation within a well-defined pouch. Like other marsupials, it gives birth to highly altricial young, and has a small number of very large chromosomes, making it a valuable model for genomics, reproduction and development.</p><p class="abstract"><strong>Results</strong>: The genome has been sequenced to 2 &times; coverage using Sanger sequencing, enhanced with additional next generation sequencing and the integration of extensive physical and linkage maps to build the genome assembly. We also sequenced the tammar transcriptome across many tissues and developmental time points. Our analyses of these data shed light on mammalian reproduction, development and genome evolution: there is innovation in reproductive and lactational genes, rapid evolution of germ cell genes, and incomplete, locus-specific X inactivation. We also observe novel retrotransposons and a highly rearranged major histocompatibility complex, with many class I genes located outside the complex. Novel microRNAs in the tammar HOX clusters uncover new potential mammalian HOX regulatory elements.</p><p class="abstract"><strong>Conclusions</strong>: Analyses of these resources enhance our understanding of marsupial gene evolution, identify marsupial-specific conserved non-coding elements and critical genes across a range of biological systems, including reproduction, development and immunity, and provide new insight into marsupial and mammalian biology and genome evolution.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54-HG003273</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">8</span>;<span class="pagerange">R81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21854559" rel="external">21854559</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3277949" rel="external">3277949</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-8-r81" rel="external">10.1186/gb-2011-12-8-r81</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic predisposition to long-term nondiabetic deteriorations in glucose homeostasis: Ten-year follow-up of the GLACIER study.</h4>
<p class="authors"><span class="author">Renstr&ouml;m F</span>, <span class="author">Shungin D</span>, <span class="author">Johansson I</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Florez JC</span>, <span class="author">Hallmans G</span>, <span class="author">Hu FB</span> and <span class="author">Franks PW</span></p>
<p class="affiliation">Department of Public Health and Clinical Medicine, Ume&aring; University Hospital, Sweden.</p><p class="abstract"><strong>Objective</strong>: To assess whether recently discovered genetic loci associated with hyperglycemia also predict long-term changes in glycemic traits.</p><p class="abstract">Sixteen fasting glucose-raising loci were genotyped in middle-aged adults from the Gene x Lifestyle interactions And Complex traits Involved in Elevated disease Risk (GLACIER) Study, a population-based prospective cohort study from northern Sweden. Genotypes were tested for association with baseline fasting and 2-h postchallenge glycemia (N = 16,330), and for changes in these glycemic traits during a 10-year follow-up period (N = 4,059).</p><p class="abstract"><strong>Results</strong>: Cross-sectional directionally consistent replication with fasting glucose concentrations was achieved for 12 of 16 variants; 10 variants were also associated with impaired fasting glucose (IFG) and 7 were independently associated with 2-h postchallenge glucose concentrations. In prospective analyses, the effect alleles at four loci (GCK rs4607517, ADRA2A rs10885122, DGKB-TMEM195 rs2191349, and G6PC2 rs560887) were nominally associated with worsening fasting glucose concentrations during 10-years of follow-up. MTNR1B rs10830963, which was predictive of elevated fasting glucose concentrations in cross-sectional analyses, was associated with a protective effect on postchallenge glucose concentrations during follow-up; however, this was only when baseline fasting and 2-h glucoses were adjusted for. An additive effect of multiple risk alleles on glycemic traits was observed: a weighted genetic risk score (80th vs. 20th centiles) was associated with a 0.16 mmol/l (P = 2.4 &times; 10&#x207B;&#x2076;) greater elevation in fasting glucose and a 64% (95% CI: 33-201%) higher risk of developing IFG during 10 years of follow-up.</p><p class="abstract"><strong>Conclusions</strong>: Our findings imply that genetic profiling might facilitate the early detection of persons who are genetically susceptible to deteriorating glucose control; studies of incident type 2 diabetes and discrete cardiovascular end points will help establish whether the magnitude of these changes is clinically relevant.</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">1</span>;<span class="pagerange">345-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20870969">20870969</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3012192">3012192</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb10-0933">10.2337/db10-0933</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effect of using varying negative examples in transcription factor binding site predictions</h4>
<p class="authors"><span class="author">Rezwan F</span>, <span class="author">Sun Y</span>, <span class="author">Davey N</span>, <span class="author">Adams R</span>, <span class="author">RUST AG</span>, <span class="author">Robinson M</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science</span> <span class="year">2011</span>;<span class="volume">6623 LNCS</span>;<span class="pagerange">1-12</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequences of Salmonella enterica serovar typhimurium, Choleraesuis, Dublin, and Gallinarum strains of well- defined virulence in food-producing animals.</h4>
<p class="authors"><span class="author">Richardson EJ</span>, <span class="author">Limaye B</span>, <span class="author">Inamdar H</span>, <span class="author">Datta A</span>, <span class="author">Manjari KS</span>, <span class="author">Pullinger GD</span>, <span class="author">Thomson NR</span>, <span class="author">Joshi RR</span>, <span class="author">Watson M</span> and <span class="author">Stevens MP</span></p>
<p class="affiliation">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush, Midlothian, EH25 9RG, Scotland, UK.</p><p class="abstract">Salmonella enterica is an animal and zoonotic pathogen of worldwide importance and may be classified into serovars differing in virulence and host range. We sequenced and annotated the genomes of serovar Typhimurium, Choleraesuis, Dublin, and Gallinarum strains of defined virulence in each of three food-producing animal hosts. This provides valuable measures of intraserovar diversity and opportunities to formally link genotypes to phenotypes in target animals.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2011</span>;<span class="volume">193</span>;<span class="number">12</span>;<span class="pagerange">3162-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21478351" rel="external">21478351</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3133203" rel="external">3133203</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00394-11" rel="external">10.1128/JB.00394-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prospective study of insulin-like growth factor-I, insulin-like growth factor-binding protein 3, genetic variants in the IGF1 and IGFBP3 genes and risk of coronary artery disease.</h4>
<p class="authors"><span class="author">Ricketts SL</span>, <span class="author">Rensing KL</span>, <span class="author">Holly JM</span>, <span class="author">Chen L</span>, <span class="author">Young EH</span>, <span class="author">Luben R</span>, <span class="author">Ashford S</span>, <span class="author">Song K</span>, <span class="author">Yuan X</span>, <span class="author">Dehghan A</span>, <span class="author">Wright BJ</span>, <span class="author">Waterworth DM</span>, <span class="author">Mooser V</span>, <span class="author">GEMS Investigators</span>, <span class="author">Waeber G</span>, <span class="author">Vollenweider P</span>, <span class="author">Epstein SE</span>, <span class="author">Burnett MS</span>, <span class="author">Devaney JM</span>, <span class="author">Hakonarson HH</span>, <span class="author">Rader DJ</span>, <span class="author">Reilly MP</span>, <span class="author">Danesh J</span>, <span class="author">Thompson SG</span>, <span class="author">Dunning AM</span>, <span class="author">van Duijn CM</span>, <span class="author">Samani NJ</span>, <span class="author">McPherson R</span>, <span class="author">Wareham NJ</span>, <span class="author">Khaw KT</span>, <span class="author">Boekholdt SM</span> and <span class="author">Sandhu MS</span></p><p class="abstract">Although experimental studies have suggested that insulin-like growth factor I (IGF-I) and its binding protein IGFBP-3 might have a role in the aetiology of coronary artery disease (CAD), the relevance of circulating IGFs and their binding proteins in the development of CAD in human populations is unclear. We conducted a nested case-control study, with a mean follow-up of six years, within the EPIC-Norfolk cohort to assess the association between circulating levels of IGF-I and IGFBP-3 and risk of CAD in up to 1,013 cases and 2,055 controls matched for age, sex and study enrolment date. After adjustment for cardiovascular risk factors, we found no association between circulating levels of IGF-I or IGFBP-3 and risk of CAD (odds ratio: 0.98 (95% Cl 0.90-1.06) per 1 SD increase in circulating IGF-I; odds ratio: 1.02 (95% Cl 0.94-1.12) for IGFBP-3). We examined associations between tagging single nucleotide polymorphisms (tSNPs) at the IGF1 and IGFBP3 loci and circulating IGF-I and IGFBP-3 levels in up to 1,133 cases and 2,223 controls and identified three tSNPs (rs1520220, rs3730204, rs2132571) that showed independent association with either circulating IGF-I or IGFBP-3 levels. In an assessment of 31 SNPs spanning the IGF1 or IGFBP3 loci, none were associated with risk of CAD in a meta-analysis that included EPIC-Norfolk and eight additional studies comprising up to 9,319 cases and 19,964 controls. Our results indicate that IGF-I and IGFBP-3 are unlikely to be importantly involved in the aetiology of CAD in human populations.</p>
<p class="publication"><span class="title">International journal of molecular epidemiology and genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">261-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21915365" rel="external">21915365</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3166154" rel="external">3166154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The IKMC web portal: a central point of entry to data and resources from the International Knockout Mouse Consortium.</h4>
<p class="authors"><span class="author">Ringwald M</span>, <span class="author">Iyer V</span>, <span class="author">Mason JC</span>, <span class="author">Stone KR</span>, <span class="author">Tadepally HD</span>, <span class="author">Kadin JA</span>, <span class="author">Bult CJ</span>, <span class="author">Eppig JT</span>, <span class="author">Oakley DJ</span>, <span class="author">Briois S</span>, <span class="author">Stupka E</span>, <span class="author">Maselli V</span>, <span class="author">Smedley D</span>, <span class="author">Liu S</span>, <span class="author">Hansen J</span>, <span class="author">Baldock R</span>, <span class="author">Hicks GG</span> and <span class="author">Skarnes WC</span></p>
<p class="affiliation">The Jackson Laboratory, 600 Main Street, Bar Harbor, ME 04609, USA. <a href="mailto:%72%69%6E%67%77%61%6C%64%40%69%6E%66%6F%72%6D%61%74%69%63%73.%6A%61%78.%6F%72%67">&#114;&#105;&#110;&#103;&#119;&#97;&#108;&#100;&#64;&#105;&#110;&#102;&#111;&#114;&#109;&#97;&#116;&#105;&#99;&#115;&#46;&#106;&#97;&#120;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The International Knockout Mouse Consortium (IKMC) aims to mutate all protein-coding genes in the mouse using a combination of gene targeting and gene trapping in mouse embryonic stem (ES) cells and to make the generated resources readily available to the research community. The IKMC database and web portal (www.knockoutmouse.org) serves as the central public web site for IKMC data and facilitates the coordination and prioritization of work within the consortium. Researchers can access up-to-date information on IKMC knockout vectors, ES cells and mice for specific genes, and follow links to the respective repositories from which corresponding IKMC products can be ordered. Researchers can also use the web site to nominate genes for targeting, or to indicate that targeting of a gene should receive high priority. The IKMC database provides data to, and features extensive interconnections with, other community databases.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U127527203</span>; NHGRI NIH HHS: <span class="grant">HG004074</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2011</span>;<span class="volume">39</span>;<span class="number">Database issue</span>;<span class="pagerange">D849-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20929875">20929875</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3013768">3013768</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq879">10.1093/nar/gkq879</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies five new schizophrenia loci.</h4>
<p class="authors"><span class="author">Ripke S</span>, <span class="author">Sanders AR</span>, <span class="author">Kendler KS</span>, <span class="author">Levinson DF</span>, <span class="author">Sklar P</span>, <span class="author">Holmans PA</span>, <span class="author">Lin DY</span>, <span class="author">Duan J</span>, <span class="author">Ophoff RA</span>, <span class="author">Andreassen OA</span>, <span class="author">Scolnick E</span>, <span class="author">Cichon S</span>, <span class="author">St Clair D</span>, <span class="author">Corvin A</span>, <span class="author">Gurling H</span>, <span class="author">Werge T</span>, <span class="author">Rujescu D</span>, <span class="author">Blackwood DH</span>, <span class="author">Pato CN</span>, <span class="author">Malhotra AK</span>, <span class="author">Purcell S</span>, <span class="author">Dudbridge F</span>, <span class="author">Neale BM</span>, <span class="author">Rossin L</span>, <span class="author">Visscher PM</span>, <span class="author">Posthuma D</span>, <span class="author">Ruderfer DM</span>, <span class="author">Fanous A</span>, <span class="author">Stefansson H</span>, <span class="author">Steinberg S</span>, <span class="author">Mowry BJ</span>, <span class="author">Golimbet V</span>, <span class="author">De Hert M</span>, <span class="author">J&ouml;nsson EG</span>, <span class="author">Bitter I</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Collier DA</span>, <span class="author">Tosato S</span>, <span class="author">Agartz I</span>, <span class="author">Albus M</span>, <span class="author">Alexander M</span>, <span class="author">Amdur RL</span>, <span class="author">Amin F</span>, <span class="author">Bass N</span>, <span class="author">Bergen SE</span>, <span class="author">Black DW</span>, <span class="author">B&oslash;rglum AD</span>, <span class="author">Brown MA</span>, <span class="author">Bruggeman R</span>, <span class="author">Buccola NG</span>, <span class="author">Byerley WF</span>, <span class="author">Cahn W</span>, <span class="author">Cantor RM</span>, <span class="author">Carr VJ</span>, <span class="author">Catts SV</span>, <span class="author">Choudhury K</span>, <span class="author">Cloninger CR</span>, <span class="author">Cormican P</span>, <span class="author">Craddock N</span>, <span class="author">Danoy PA</span>, <span class="author">Datta S</span>, <span class="author">de Haan L</span>, <span class="author">Demontis D</span>, <span class="author">Dikeos D</span>, <span class="author">Djurovic S</span>, <span class="author">Donnelly P</span>, <span class="author">Donohoe G</span>, <span class="author">Duong L</span>, <span class="author">Dwyer S</span>, <span class="author">Fink-Jensen A</span>, <span class="author">Freedman R</span>, <span class="author">Freimer NB</span>, <span class="author">Friedl M</span>, <span class="author">Georgieva L</span>, <span class="author">Giegling I</span>, <span class="author">Gill M</span>, <span class="author">Glenth&oslash;j B</span>, <span class="author">Godard S</span>, <span class="author">Hamshere M</span>, <span class="author">Hansen M</span>, <span class="author">Hansen T</span>, <span class="author">Hartmann AM</span>, <span class="author">Henskens FA</span>, <span class="author">Hougaard DM</span>, <span class="author">Hultman CM</span>, <span class="author">Ingason A</span>, <span class="author">Jablensky AV</span>, <span class="author">Jakobsen KD</span>, <span class="author">Jay M</span>, <span class="author">J&uuml;rgens G</span>, <span class="author">Kahn RS</span>, <span class="author">Keller MC</span>, <span class="author">Kenis G</span>, <span class="author">Kenny E</span>, <span class="author">Kim Y</span>, <span class="author">Kirov GK</span>, <span class="author">Konnerth H</span>, <span class="author">Konte B</span>, <span class="author">Krabbendam L</span>, <span class="author">Krasucki R</span>, <span class="author">Lasseter VK</span>, <span class="author">Laurent C</span>, <span class="author">Lawrence J</span>, <span class="author">Lencz T</span>, <span class="author">Lerer FB</span>, <span class="author">Liang KY</span>, <span class="author">Lichtenstein P</span>, <span class="author">Lieberman JA</span>, <span class="author">Linszen DH</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Loughland CM</span>, <span class="author">Maclean AW</span>, <span class="author">Maher BS</span>, <span class="author">Maier W</span>, <span class="author">Mallet J</span>, <span class="author">Malloy P</span>, <span class="author">Mattheisen M</span>, <span class="author">Mattingsdal M</span>, <span class="author">McGhee KA</span>, <span class="author">McGrath JJ</span>, <span class="author">McIntosh A</span>, <span class="author">McLean DE</span>, <span class="author">McQuillin A</span>, <span class="author">Melle I</span>, <span class="author">Michie PT</span>, <span class="author">Milanova V</span>, <span class="author">Morris DW</span>, <span class="author">Mors O</span>, <span class="author">Mortensen PB</span>, <span class="author">Moskvina V</span>, <span class="author">Muglia P</span>, <span class="author">Myin-Germeys I</span>, <span class="author">Nertney DA</span>, <span class="author">Nestadt G</span>, <span class="author">Nielsen J</span>, <span class="author">Nikolov I</span>, <span class="author">Nordentoft M</span>, <span class="author">Norton N</span>, <span class="author">N&ouml;then MM</span>, <span class="author">O&#39;Dushlaine CT</span>, <span class="author">Olincy A</span>, <span class="author">Olsen L</span>, <span class="author">O&#39;Neill FA</span>, <span class="author">Orntoft TF</span>, <span class="author">Owen MJ</span>, <span class="author">Pantelis C</span>, <span class="author">Papadimitriou G</span>, <span class="author">Pato MT</span>, <span class="author">Peltonen L</span>, <span class="author">Petursson H</span>, <span class="author">Pickard B</span>, <span class="author">Pimm J</span>, <span class="author">Pulver AE</span>, <span class="author">Puri V</span>, <span class="author">Quested D</span>, <span class="author">Quinn EM</span>, <span class="author">Rasmussen HB</span>, <span class="author">R&eacute;thelyi JM</span>, <span class="author">Ribble R</span>, <span class="author">Rietschel M</span>, <span class="author">Riley BP</span>, <span class="author">Ruggeri M</span>, <span class="author">Schall U</span>, <span class="author">Schulze TG</span>, <span class="author">Schwab SG</span>, <span class="author">Scott RJ</span>, <span class="author">Shi J</span>, <span class="author">Sigurdsson E</span>, <span class="author">Silverman JM</span>, <span class="author">Spencer CC</span>, <span class="author">Stefansson K</span>, <span class="author">Strange A</span>, <span class="author">Strengman E</span>, <span class="author">Stroup TS</span>, <span class="author">Suvisaari J</span>, <span class="author">Terenius L</span>, <span class="author">Thirumalai S</span>, <span class="author">Thygesen JH</span>, <span class="author">Timm S</span>, <span class="author">Toncheva D</span>, <span class="author">van den Oord E</span>, <span class="author">van Os J</span>, <span class="author">van Winkel R</span>, <span class="author">Veldink J</span>, <span class="author">Walsh D</span>, <span class="author">Wang AG</span>, <span class="author">Wiersma D</span>, <span class="author">Wildenauer DB</span>, <span class="author">Williams HJ</span>, <span class="author">Williams NM</span>, <span class="author">Wormley B</span>, <span class="author">Zammit S</span>, <span class="author">Sullivan PF</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Daly MJ</span>, <span class="author">Gejman PV</span> and <span class="author">Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium</span></p>
<p class="affiliation">Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA.</p><p class="abstract">We examined the role of common genetic variation in schizophrenia in a genome-wide association study of substantial size: a stage 1 discovery sample of 21,856 individuals of European ancestry and a stage 2 replication sample of 29,839 independent subjects. The combined stage 1 and 2 analysis yielded genome-wide significant associations with schizophrenia for seven loci, five of which are new (1p21.3, 2q32.3, 8p23.2, 8q21.3 and 10q24.32-q24.33) and two of which have been previously implicated (6p21.32-p22.1 and 18q21.2). The strongest new finding (P = 1.6 &times; 10(-11)) was with rs1625579 within an intron of a putative primary transcript for MIR137 (microRNA 137), a known regulator of neuronal development. Four other schizophrenia loci achieving genome-wide significance contain predicted targets of MIR137, suggesting MIR137-mediated dysregulation as a previously unknown etiologic mechanism in schizophrenia. In a joint analysis with a bipolar disorder sample (16,374 affected individuals and 14,044 controls), three loci reached genome-wide significance: CACNA1C (rs4765905, P = 7.0 &times; 10(-9)), ANK3 (rs10994359, P = 2.5 &times; 10(-8)) and the ITIH3-ITIH4 region (rs2239547, P = 7.8 &times; 10(-9)).</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01 CA082659-12</span>; NIGMS NIH HHS: <span class="grant">R37 GM047845-21</span>; NIMH NIH HHS: <span class="grant">K01 MH085812-03</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">10</span>;<span class="pagerange">969-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21926974" rel="external">21926974</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3303194" rel="external">3303194</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.940" rel="external">10.1038/ng.940</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Drug-resistant genotypes and multi-clonality in Plasmodium falciparum analysed by direct genome sequencing from peripheral blood of malaria patients.</h4>
<p class="authors"><span class="author">Robinson T</span>, <span class="author">Campino SG</span>, <span class="author">Auburn S</span>, <span class="author">Assefa SA</span>, <span class="author">Polley SD</span>, <span class="author">Manske M</span>, <span class="author">MacInnis B</span>, <span class="author">Rockett KA</span>, <span class="author">Maslen GL</span>, <span class="author">Sanders M</span>, <span class="author">Quail MA</span>, <span class="author">Chiodini PL</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Clark TG</span> and <span class="author">Sutherland CJ</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">Naturally acquired blood-stage infections of the malaria parasite Plasmodium falciparum typically harbour multiple haploid clones. The apparent number of clones observed in any single infection depends on the diversity of the polymorphic markers used for the analysis, and the relative abundance of rare clones, which frequently fail to be detected among PCR products derived from numerically dominant clones. However, minority clones are of clinical interest as they may harbour genes conferring drug resistance, leading to enhanced survival after treatment and the possibility of subsequent therapeutic failure. We deployed new generation sequencing to derive genome data for five non-propagated parasite isolates taken directly from 4 different patients treated for clinical malaria in a UK hospital. Analysis of depth of coverage and length of sequence intervals between paired reads identified both previously described and novel gene deletions and amplifications. Full-length sequence data was extracted for 6 loci considered to be under selection by antimalarial drugs, and both known and previously unknown amino acid substitutions were identified. Full mitochondrial genomes were extracted from the sequencing data for each isolate, and these are compared against a panel of polymorphic sites derived from published or unpublished but publicly available data. Finally, genome-wide analysis of clone multiplicity was performed, and the number of infecting parasite clones estimated for each isolate. Each patient harboured at least 3 clones of P. falciparum by this analysis, consistent with results obtained with conventional PCR analysis of polymorphic merozoite antigen loci. We conclude that genome sequencing of peripheral blood P. falciparum taken directly from malaria patients provides high quality data useful for drug resistance studies, genomic structural analyses and population genetics, and also robustly represents clonal multiplicity.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">e23204</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21853089">21853089</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3154926">3154926</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0023204">10.1371/journal.pone.0023204</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome and gene copy number variation allow major structural change between species and strains of Leishmania.</h4>
<p class="authors"><span class="author">Rogers MB</span>, <span class="author">Hilley JD</span>, <span class="author">Dickens NJ</span>, <span class="author">Wilkes J</span>, <span class="author">Bates PA</span>, <span class="author">Depledge DP</span>, <span class="author">Harris D</span>, <span class="author">Her Y</span>, <span class="author">Herzyk P</span>, <span class="author">Imamura H</span>, <span class="author">Otto TD</span>, <span class="author">Sanders M</span>, <span class="author">Seeger K</span>, <span class="author">Dujardin JC</span>, <span class="author">Berriman M</span>, <span class="author">Smith DF</span>, <span class="author">Hertz-Fowler C</span> and <span class="author">Mottram JC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Leishmania parasites cause a spectrum of clinical pathology in humans ranging from disfiguring cutaneous lesions to fatal visceral leishmaniasis. We have generated a reference genome for Leishmania mexicana and refined the reference genomes for Leishmania major, Leishmania infantum, and Leishmania braziliensis. This has allowed the identification of a remarkably low number of genes or paralog groups (2, 14, 19, and 67, respectively) unique to one species. These were found to be conserved in additional isolates of the same species. We have predicted allelic variation and find that in these isolates, L. major and L. infantum have a surprisingly low number of predicted heterozygous SNPs compared with L. braziliensis and L. mexicana. We used short read coverage to infer ploidy and gene copy numbers, identifying large copy number variations between species, with 200 tandem gene arrays in L. major and 132 in L. mexicana. Chromosome copy number also varied significantly between species, with nine supernumerary chromosomes in L. infantum, four in L. mexicana, two in L. braziliensis, and one in L. major. A significant bias against gene arrays on supernumerary chromosomes was shown to exist, indicating that duplication events occur more frequently on disomic chromosomes. Taken together, our data demonstrate that there is little variation in unique gene content across Leishmania species, but large-scale genetic heterogeneity can result through gene amplification on disomic chromosomes and variation in chromosome number. Increased gene copy number due to chromosome amplification may contribute to alterations in gene expression in response to environmental conditions in the host, providing a genetic basis for disease tropism.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076355</span>, <span class="grant">085775</span>, <span class="grant">085822</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2129-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22038252">22038252</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3227102">3227102</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.122945.111">10.1101/gr.122945.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluation of the immunogenicity and biological activity of the Citrobacter freundii Vi-CRM197 conjugate as a vaccine for Salmonella enterica serovar Typhi.</h4>
<p class="authors"><span class="author">Rondini S</span>, <span class="author">Micoli F</span>, <span class="author">Lanzilao L</span>, <span class="author">Hale C</span>, <span class="author">Saul AJ</span> and <span class="author">Martin LB</span></p>
<p class="affiliation">Novartis Vaccines Institute for Global Health, Via Fiorentina 1, 53100 Siena, Italy. <a href="mailto:%73%69%6D%6F%6E%61.%72%6F%6E%64%69%6E%69%40%6E%6F%76%61%72%74%69%73.%63%6F%6D">&#115;&#105;&#109;&#111;&#110;&#97;&#46;&#114;&#111;&#110;&#100;&#105;&#110;&#105;&#64;&#110;&#111;&#118;&#97;&#114;&#116;&#105;&#115;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Typhoid fever remains a major health problem in developing countries. Young children are at high risk, and a vaccine effective for this age group is urgently needed. Purified capsular polysaccharide from Salmonella enterica serovar Typhi (Vi) is licensed as a vaccine, providing 50 to 70% protection in individuals older than 5 years. However, this vaccine is ineffective in infants. Vi conjugated to a carrier protein (i.e., an exoprotein A mutant from Pseudomonas aeruginosa [rEPA]) is highly immunogenic, provides long-term protection, and shows more than 90% protective efficacy in children 2 to 5 years old. Here, we describe an alternative glycoconjugate vaccine for S. Typhi, Vi-CRM(197), where Vi was obtained from Citrobacter freundii WR7011 and CRM(197), the mutant diphtheria toxin protein, was used as the carrier. We investigated the optimization of growth conditions for Vi production from C. freundii WR7011 and the immunogenicity of Vi-CRM(197) conjugates in mice. The optimal saccharide/protein ratio of the glycoconjugates was identified for the best antibody production. We also demonstrated the ability of this new vaccine to protect mice against challenge with Vi-positive Salmonella enterica serovar Typhimurium.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Clinical and vaccine immunology : CVI</span>&nbsp;<span class="year">2011</span>;<span class="volume">18</span>;<span class="number">3</span>;<span class="pagerange">460-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21248155" rel="external">21248155</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3067394" rel="external">3067394</a>; DOI: <a href="http://dx.doi.org/10.1128%2FCVI.00387-10" rel="external">10.1128/CVI.00387-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrating genome-wide genetic variations and monocyte expression data reveals trans-regulated gene modules in humans.</h4>
<p class="authors"><span class="author">Rotival M</span>, <span class="author">Zeller T</span>, <span class="author">Wild PS</span>, <span class="author">Maouche S</span>, <span class="author">Szymczak S</span>, <span class="author">Schillert A</span>, <span class="author">Castagn&eacute; R</span>, <span class="author">Deiseroth A</span>, <span class="author">Proust C</span>, <span class="author">Brocheton J</span>, <span class="author">Godefroy T</span>, <span class="author">Perret C</span>, <span class="author">Germain M</span>, <span class="author">Eleftheriadis M</span>, <span class="author">Sinning CR</span>, <span class="author">Schnabel RB</span>, <span class="author">Lubos E</span>, <span class="author">Lackner KJ</span>, <span class="author">Rossmann H</span>, <span class="author">M&uuml;nzel T</span>, <span class="author">Rendon A</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">Erdmann J</span>, <span class="author">Deloukas P</span>, <span class="author">Hengstenberg C</span>, <span class="author">Diemert P</span>, <span class="author">Montalescot G</span>, <span class="author">Ouwehand WH</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Tregouet DA</span>, <span class="author">Ziegler A</span>, <span class="author">Goodall AH</span>, <span class="author">Cambien F</span>, <span class="author">Tiret L</span> and <span class="author">Blankenberg S</span></p>
<p class="affiliation">INSERM UMRS 937, Pierre and Marie Curie University (UPMC, Paris 6) and Medical School, Paris, France.</p><p class="abstract">One major expectation from the transcriptome in humans is to characterize the biological basis of associations identified by genome-wide association studies. So far, few cis expression quantitative trait loci (eQTLs) have been reliably related to disease susceptibility. Trans-regulating mechanisms may play a more prominent role in disease susceptibility. We analyzed 12,808 genes detected in at least 5% of circulating monocyte samples from a population-based sample of 1,490 European unrelated subjects. We applied a method of extraction of expression patterns-independent component analysis-to identify sets of co-regulated genes. These patterns were then related to 675,350 SNPs to identify major trans-acting regulators. We detected three genomic regions significantly associated with co-regulated gene modules. Association of these loci with multiple expression traits was replicated in Cardiogenics, an independent study in which expression profiles of monocytes were available in 758 subjects. The locus 12q13 (lead SNP rs11171739), previously identified as a type 1 diabetes locus, was associated with a pattern including two cis eQTLs, RPS26 and SUOX, and 5 trans eQTLs, one of which (MADCAM1) is a potential candidate for mediating T1D susceptibility. The locus 12q24 (lead SNP rs653178), which has demonstrated extensive disease pleiotropy, including type 1 diabetes, hypertension, and celiac disease, was associated to a pattern strongly correlating to blood pressure level. The strongest trans eQTL in this pattern was CRIP1, a known marker of cellular proliferation in cancer. The locus 12q15 (lead SNP rs11177644) was associated with a pattern driven by two cis eQTLs, LYZ and YEATS4, and including 34 trans eQTLs, several of them tumor-related genes. This study shows that a method exploiting the structure of co-expressions among genes can help identify genomic regions involved in trans regulation of sets of genes and can provide clues for understanding the mechanisms linking genome-wide association loci to disease.</p>
<p class="grants">Funded by: British Heart Foundation</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e1002367</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22144904" rel="external">22144904</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3228821" rel="external">3228821</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002367" rel="external">10.1371/journal.pgen.1002367</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The explosive-degrading cytochrome P450 XplA: biochemistry, structural features and prospects for bioremediation.</h4>
<p class="authors"><span class="author">Rylott EL</span>, <span class="author">Jackson RG</span>, <span class="author">Sabbadin F</span>, <span class="author">Seth-Smith HM</span>, <span class="author">Edwards J</span>, <span class="author">Chong CS</span>, <span class="author">Strand SE</span>, <span class="author">Grogan G</span> and <span class="author">Bruce NC</span></p>
<p class="affiliation">CNAP, Department of Biology, University of York, PO Box 373, York YO105YW, UK.</p><p class="abstract">XplA is a cytochrome P450 that mediates the microbial metabolism of the military explosive hexahydro-1,3,5-trinitro-1,3,5-triazine (RDX). It has an unusual structural organisation comprising a heme domain that is fused to its flavodoxin redox partner. XplA along with its partnering reductase XplB are plasmid encoded and the gene xplA has now been found in divergent genera across the globe with near sequence identity. Importantly, it has only been detected at explosives contaminated sites suggesting rapid dissemination of this novel catabolic activity, possibly within the 50-year period since the introduction of RDX into the environment. The X-ray structure of XplA-heme has been solved, providing fundamental information on the heme binding site. Interestingly, oxygen is not required for the degradation of RDX, but its presence determines the final degradation products, demonstrating that the degradation chemistry is flexible with both anaerobic and aerobic pathways resulting in the release of nitrite from the substrate. Transgenic plants expressing xplA are able to remove saturating levels of RDX from soil leachate and may provide a low cost sustainable remediation strategy for contaminated military sites.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Biochimica et biophysica acta</span>&nbsp;<span class="year">2011</span>;<span class="volume">1814</span>;<span class="number">1</span>;<span class="pagerange">230-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20624490" rel="external">20624490</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.bbapap.2010.07.004" rel="external">10.1016/j.bbapap.2010.07.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">You cannot B. cereus.</h4>
<p class="authors"><span class="author">Salter SJ</span></p><p class="abstract">This month&#39;s Genome Watch looks at the different Bacillus species that can cause anthrax.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21189478" rel="external">21189478</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2506" rel="external">10.1038/nrmicro2506</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A &#39;clap&#39; for in silico studies.</h4>
<p class="authors"><span class="author">Sanchez-Flores A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21164533" rel="external">21164533</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2496" rel="external">10.1038/nrmicro2496</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new piece of the eukaryotic puzzle.</h4>
<p class="authors"><span class="author">Sanchez-Flores A</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">11</span>;<span class="pagerange">769</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21997841" rel="external">21997841</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2680" rel="external">10.1038/nrmicro2680</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Emergent neutrality in adaptive asexual evolution.</h4>
<p class="authors"><span class="author">Schiffels S</span>, <span class="author">Sz&ouml;llosi GJ</span>, <span class="author">Mustonen V</span> and <span class="author">L&auml;ssig M</span></p>
<p class="affiliation">Institut f&uuml;r Theoretische Physik, Universit&auml;t zu K&ouml;ln, 50937 K&ouml;ln, Germany.</p><p class="abstract">In nonrecombining genomes, genetic linkage can be an important evolutionary force. Linkage generates interference interactions, by which simultaneously occurring mutations affect each other&#39;s chance of fixation. Here, we develop a comprehensive model of adaptive evolution in linked genomes, which integrates interference interactions between multiple beneficial and deleterious mutations into a unified framework. By an approximate analytical solution, we predict the fixation rates of these mutations, as well as the probabilities of beneficial and deleterious alleles at fixed genomic sites. We find that interference interactions generate a regime of emergent neutrality: all genomic sites with selection coefficients smaller in magnitude than a characteristic threshold have nearly random fixed alleles, and both beneficial and deleterious mutations at these sites have nearly neutral fixation rates. We show that this dynamic limits not only the speed of adaptation, but also a population&#39;s degree of adaptation in its current environment. We apply the model to different scenarios: stationary adaptation in a time-dependent environment and approach to equilibrium in a fixed environment. In both cases, the analytical predictions are in good agreement with numerical simulations. Our results suggest that interference can severely compromise biological functions in an adapting population, which sets viability limits on adaptive evolution under linkage.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">091747</span></p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">189</span>;<span class="number">4</span>;<span class="pagerange">1361-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21926305">21926305</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3241435">3241435</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.111.132027">10.1534/genetics.111.132027</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human transcriptome during nontyphoid Salmonella and HIV coinfection reveals attenuated NFkappaB-mediated inflammation and persistent cell cycle disruption.</h4>
<p class="authors"><span class="author">Schreiber F</span>, <span class="author">Lynn DJ</span>, <span class="author">Houston A</span>, <span class="author">Peters J</span>, <span class="author">Mwafulirwa G</span>, <span class="author">Finlay BB</span>, <span class="author">Brinkman FS</span>, <span class="author">Hancock RE</span>, <span class="author">Heyderman RS</span>, <span class="author">Dougan G</span> and <span class="author">Gordon MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Invasive nontyphoid Salmonella (iNTS) disease is common and severe in adults with human immunodeficiency virus (HIV) infection in Africa. We previously observed that ex vivo macrophages from HIV-infected subjects challenged with Salmonella Typhimurium exhibit dysregulated proinflammatory cytokine responses.</p><p class="abstract"><strong>Methods</strong>: We studied the transcriptional response in whole blood from HIV-positive patients during acute and convalescent iNTS disease compared to other invasive bacterial diseases, and to HIV-positive and -negative controls.</p><p class="abstract"><strong>Results</strong>: During iNTS disease, there was a remarkable lack of a coordinated inflammatory or innate immune signaling response. Few interferon &gamma; (IFN&gamma;)-induced genes or Toll-like receptor/transcription factor nuclear factor &kappa;B (TLR/NF&kappa;B) gene pathways were upregulated in expression. Ex vivo lipopolysacharide (LPS) or flagellin stimulation of whole blood, however, showed that convalescent iNTS subjects and controls were competent to mount prominent TLR/NF&kappa;B-associated patterns of mRNA expression. In contrast, HIV-positive patients with other invasive bacterial infections (Escherichia coli and Streptococcus pneumoniae) displayed a pronounced proinflammatory innate immune transcriptional response. There was also upregulated mRNA expression in cell cycle, DNA replication, translation and repair, and viral replication pathways during iNTS. These patterns persisted for up to 2 months into convalescence.</p><p class="abstract"><strong>Conclusions</strong>: Attenuation of NF&kappa;B-mediated inflammation and dysregulation of cell cycle and DNA-function gene pathway expression are key features of the interplay between iNTS and HIV.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">419</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2011</span>;<span class="volume">204</span>;<span class="number">8</span>;<span class="pagerange">1237-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21917897" rel="external">21917897</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3173506" rel="external">3173506</a>; DOI: <a href="http://dx.doi.org/10.1093%2Finfdis%2Fjir512" rel="external">10.1093/infdis/jir512</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association and genetic functional studies identify autism susceptibility candidate 2 gene (AUTS2) in the regulation of alcohol consumption.</h4>
<p class="authors"><span class="author">Schumann G</span>, <span class="author">Coin LJ</span>, <span class="author">Lourdusamy A</span>, <span class="author">Charoen P</span>, <span class="author">Berger KH</span>, <span class="author">Stacey D</span>, <span class="author">Desrivi&egrave;res S</span>, <span class="author">Aliev FA</span>, <span class="author">Khan AA</span>, <span class="author">Amin N</span>, <span class="author">Aulchenko YS</span>, <span class="author">Bakalkin G</span>, <span class="author">Bakker SJ</span>, <span class="author">Balkau B</span>, <span class="author">Beulens JW</span>, <span class="author">Bilbao A</span>, <span class="author">de Boer RA</span>, <span class="author">Beury D</span>, <span class="author">Bots ML</span>, <span class="author">Breetvelt EJ</span>, <span class="author">Cauchi S</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Chambers JC</span>, <span class="author">Clarke TK</span>, <span class="author">Dahmen N</span>, <span class="author">de Geus EJ</span>, <span class="author">Dick D</span>, <span class="author">Ducci F</span>, <span class="author">Easton A</span>, <span class="author">Edenberg HJ</span>, <span class="author">Esko T</span>, <span class="author">Esk T</span>, <span class="author">Fern&aacute;ndez-Medarde A</span>, <span class="author">Foroud T</span>, <span class="author">Freimer NB</span>, <span class="author">Girault JA</span>, <span class="author">Grobbee DE</span>, <span class="author">Guarrera S</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Hartikainen AL</span>, <span class="author">Heath AC</span>, <span class="author">Hesselbrock V</span>, <span class="author">Hofman A</span>, <span class="author">Hottenga JJ</span>, <span class="author">Isohanni MK</span>, <span class="author">Kaprio J</span>, <span class="author">Khaw KT</span>, <span class="author">Kuehnel B</span>, <span class="author">Laitinen J</span>, <span class="author">Lobbens S</span>, <span class="author">Luan J</span>, <span class="author">Mangino M</span>, <span class="author">Maroteaux M</span>, <span class="author">Matullo G</span>, <span class="author">McCarthy MI</span>, <span class="author">Mueller C</span>, <span class="author">Navis G</span>, <span class="author">Numans ME</span>, <span class="author">N&uacute;&ntilde;ez A</span>, <span class="author">Nyholt DR</span>, <span class="author">Onland-Moret CN</span>, <span class="author">Oostra BA</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Palkovits M</span>, <span class="author">Penninx BW</span>, <span class="author">Polidoro S</span>, <span class="author">Pouta A</span>, <span class="author">Prokopenko I</span>, <span class="author">Ricceri F</span>, <span class="author">Santos E</span>, <span class="author">Smit JH</span>, <span class="author">Soranzo N</span>, <span class="author">Song K</span>, <span class="author">Sovio U</span>, <span class="author">Stumvoll M</span>, <span class="author">Surakk I</span>, <span class="author">Thorgeirsson TE</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Troakes C</span>, <span class="author">Tyrfingsson T</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Uiterwaal CS</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van der Harst P</span>, <span class="author">van der Schouw YT</span>, <span class="author">Staehlin O</span>, <span class="author">Vogelzangs N</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Wareham NJ</span>, <span class="author">Waterworth DM</span>, <span class="author">Whitfield JB</span>, <span class="author">Wichmann EH</span>, <span class="author">Willemsen G</span>, <span class="author">Witteman JC</span>, <span class="author">Yuan X</span>, <span class="author">Zhai G</span>, <span class="author">Zhao JH</span>, <span class="author">Zhang W</span>, <span class="author">Martin NG</span>, <span class="author">Metspalu A</span>, <span class="author">Doering A</span>, <span class="author">Scott J</span>, <span class="author">Spector TD</span>, <span class="author">Loos RJ</span>, <span class="author">Boomsma DI</span>, <span class="author">Mooser V</span>, <span class="author">Peltonen L</span>, <span class="author">Stefansson K</span>, <span class="author">van Duijn CM</span>, <span class="author">Vineis P</span>, <span class="author">Sommer WH</span>, <span class="author">Kooner JS</span>, <span class="author">Spanagel R</span>, <span class="author">Heberlein UA</span>, <span class="author">Jarvelin MR</span> and <span class="author">Elliott P</span></p>
<p class="affiliation">Medical Research Council-Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King&#39;s College, London SE5 8AF, United Kingdom. <a href="mailto:%67%75%6E%74%65%72.%73%63%68%75%6D%61%6E%6E%40%6B%63%6C.%61%63.%75%6B">&#103;&#117;&#110;&#116;&#101;&#114;&#46;&#115;&#99;&#104;&#117;&#109;&#97;&#110;&#110;&#64;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Alcohol consumption is a moderately heritable trait, but the genetic basis in humans is largely unknown, despite its clinical and societal importance. We report a genome-wide association study meta-analysis of &sim;2.5 million directly genotyped or imputed SNPs with alcohol consumption (gram per day per kilogram body weight) among 12 population-based samples of European ancestry, comprising 26,316 individuals, with replication genotyping in an additional 21,185 individuals. SNP rs6943555 in autism susceptibility candidate 2 gene (AUTS2) was associated with alcohol consumption at genome-wide significance (P = 4 &times; 10(-8) to P = 4 &times; 10(-9)). We found a genotype-specific expression of AUTS2 in 96 human prefrontal cortex samples (P = 0.026) and significant (P &lt; 0.017) differences in expression of AUTS2 in whole-brain extracts of mice selected for differences in voluntary alcohol consumption. Down-regulation of an AUTS2 homolog caused reduced alcohol sensitivity in Drosophila (P &lt; 0.001). Our finding of a regulator of alcohol consumption adds knowledge to our understanding of genetic mechanisms influencing alcohol drinking behavior.</p>
<p class="grants">Funded by: NIAAA NIH HHS: <span class="grant">K05 AA017688-04</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">17</span>;<span class="pagerange">7119-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21471458" rel="external">21471458</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3084048" rel="external">3084048</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1017288108" rel="external">10.1073/pnas.1017288108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease.</h4>
<p class="authors"><span class="author">Schunkert H</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Kathiresan S</span>, <span class="author">Reilly MP</span>, <span class="author">Assimes TL</span>, <span class="author">Holm H</span>, <span class="author">Preuss M</span>, <span class="author">Stewart AF</span>, <span class="author">Barbalic M</span>, <span class="author">Gieger C</span>, <span class="author">Absher D</span>, <span class="author">Aherrahrou Z</span>, <span class="author">Allayee H</span>, <span class="author">Altshuler D</span>, <span class="author">Anand SS</span>, <span class="author">Andersen K</span>, <span class="author">Anderson JL</span>, <span class="author">Ardissino D</span>, <span class="author">Ball SG</span>, <span class="author">Balmforth AJ</span>, <span class="author">Barnes TA</span>, <span class="author">Becker DM</span>, <span class="author">Becker LC</span>, <span class="author">Berger K</span>, <span class="author">Bis JC</span>, <span class="author">Boekholdt SM</span>, <span class="author">Boerwinkle E</span>, <span class="author">Braund PS</span>, <span class="author">Brown MJ</span>, <span class="author">Burnett MS</span>, <span class="author">Buysschaert I</span>, <span class="author">Cardiogenics</span>, <span class="author">Carlquist JF</span>, <span class="author">Chen L</span>, <span class="author">Cichon S</span>, <span class="author">Codd V</span>, <span class="author">Davies RW</span>, <span class="author">Dedoussis G</span>, <span class="author">Dehghan A</span>, <span class="author">Demissie S</span>, <span class="author">Devaney JM</span>, <span class="author">Diemert P</span>, <span class="author">Do R</span>, <span class="author">Doering A</span>, <span class="author">Eifert S</span>, <span class="author">Mokhtari NE</span>, <span class="author">Ellis SG</span>, <span class="author">Elosua R</span>, <span class="author">Engert JC</span>, <span class="author">Epstein SE</span>, <span class="author">de Faire U</span>, <span class="author">Fischer M</span>, <span class="author">Folsom AR</span>, <span class="author">Freyer J</span>, <span class="author">Gigante B</span>, <span class="author">Girelli D</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Gudnason V</span>, <span class="author">Gulcher JR</span>, <span class="author">Halperin E</span>, <span class="author">Hammond N</span>, <span class="author">Hazen SL</span>, <span class="author">Hofman A</span>, <span class="author">Horne BD</span>, <span class="author">Illig T</span>, <span class="author">Iribarren C</span>, <span class="author">Jones GT</span>, <span class="author">Jukema JW</span>, <span class="author">Kaiser MA</span>, <span class="author">Kaplan LM</span>, <span class="author">Kastelein JJ</span>, <span class="author">Khaw KT</span>, <span class="author">Knowles JW</span>, <span class="author">Kolovou G</span>, <span class="author">Kong A</span>, <span class="author">Laaksonen R</span>, <span class="author">Lambrechts D</span>, <span class="author">Leander K</span>, <span class="author">Lettre G</span>, <span class="author">Li M</span>, <span class="author">Lieb W</span>, <span class="author">Loley C</span>, <span class="author">Lotery AJ</span>, <span class="author">Mannucci PM</span>, <span class="author">Maouche S</span>, <span class="author">Martinelli N</span>, <span class="author">McKeown PP</span>, <span class="author">Meisinger C</span>, <span class="author">Meitinger T</span>, <span class="author">Melander O</span>, <span class="author">Merlini PA</span>, <span class="author">Mooser V</span>, <span class="author">Morgan T</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">Muhlestein JB</span>, <span class="author">M&uuml;nzel T</span>, <span class="author">Musunuru K</span>, <span class="author">Nahrstaedt J</span>, <span class="author">Nelson CP</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Olivieri O</span>, <span class="author">Patel RS</span>, <span class="author">Patterson CC</span>, <span class="author">Peters A</span>, <span class="author">Peyvandi F</span>, <span class="author">Qu L</span>, <span class="author">Quyyumi AA</span>, <span class="author">Rader DJ</span>, <span class="author">Rallidis LS</span>, <span class="author">Rice C</span>, <span class="author">Rosendaal FR</span>, <span class="author">Rubin D</span>, <span class="author">Salomaa V</span>, <span class="author">Sampietro ML</span>, <span class="author">Sandhu MS</span>, <span class="author">Schadt E</span>, <span class="author">Sch&auml;fer A</span>, <span class="author">Schillert A</span>, <span class="author">Schreiber S</span>, <span class="author">Schrezenmeir J</span>, <span class="author">Schwartz SM</span>, <span class="author">Siscovick DS</span>, <span class="author">Sivananthan M</span>, <span class="author">Sivapalaratnam S</span>, <span class="author">Smith A</span>, <span class="author">Smith TB</span>, <span class="author">Snoep JD</span>, <span class="author">Soranzo N</span>, <span class="author">Spertus JA</span>, <span class="author">Stark K</span>, <span class="author">Stirrups K</span>, <span class="author">Stoll M</span>, <span class="author">Tang WH</span>, <span class="author">Tennstedt S</span>, <span class="author">Thorgeirsson G</span>, <span class="author">Thorleifsson G</span>, <span class="author">Tomaszewski M</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Rij AM</span>, <span class="author">Voight BF</span>, <span class="author">Wareham NJ</span>, <span class="author">Wells GA</span>, <span class="author">Wichmann HE</span>, <span class="author">Wild PS</span>, <span class="author">Willenborg C</span>, <span class="author">Witteman JC</span>, <span class="author">Wright BJ</span>, <span class="author">Ye S</span>, <span class="author">Zeller T</span>, <span class="author">Ziegler A</span>, <span class="author">Cambien F</span>, <span class="author">Goodall AH</span>, <span class="author">Cupples LA</span>, <span class="author">Quertermous T</span>, <span class="author">M&auml;rz W</span>, <span class="author">Hengstenberg C</span>, <span class="author">Blankenberg S</span>, <span class="author">Ouwehand WH</span>, <span class="author">Hall AS</span>, <span class="author">Deloukas P</span>, <span class="author">Thompson JR</span>, <span class="author">Stefansson K</span>, <span class="author">Roberts R</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">McPherson R</span>, <span class="author">Erdmann J</span>, <span class="author">CARDIoGRAM Consortium</span> and <span class="author">Samani NJ</span></p>
<p class="affiliation">Universit&auml;t zu L&uuml;beck, Medizinische Klinik II, L&uuml;beck, Germany.</p><p class="abstract">We performed a meta-analysis of 14 genome-wide association studies of coronary artery disease (CAD) comprising 22,233 individuals with CAD (cases) and 64,762 controls of European descent followed by genotyping of top association signals in 56,682 additional individuals. This analysis identified 13 loci newly associated with CAD at P &lt; 5 &times; 10&#x207B;&#x2078; and confirmed the association of 10 of 12 previously reported CAD loci. The 13 new loci showed risk allele frequencies ranging from 0.13 to 0.91 and were associated with a 6% to 17% increase in the risk of CAD per allele. Notably, only three of the new loci showed significant association with traditional CAD risk factors and the majority lie in gene regions not previously implicated in the pathogenesis of CAD. Finally, five of the new CAD risk loci appear to have pleiotropic effects, showing strong association with various other human diseases or traits.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/08/094/26019</span>, <span class="grant">RG/08/014/24067</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0801566</span>, <span class="grant">G1000143</span>, <span class="grant">MC_U106179471</span>; NHLBI NIH HHS: <span class="grant">HL087647</span>, <span class="grant">N01 HC025195</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01HL089650-02</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">4</span>;<span class="pagerange">333-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21378990">21378990</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3119261">3119261</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.784">10.1038/ng.784</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A role for cohesin in T-cell-receptor rearrangement and thymocyte differentiation.</h4>
<p class="authors"><span class="author">Seitan VC</span>, <span class="author">Hao B</span>, <span class="author">Tachibana-Konwalski K</span>, <span class="author">Lavagnolli T</span>, <span class="author">Mira-Bontenbal H</span>, <span class="author">Brown KE</span>, <span class="author">Teng G</span>, <span class="author">Carroll T</span>, <span class="author">Terry A</span>, <span class="author">Horan K</span>, <span class="author">Marks H</span>, <span class="author">Adams DJ</span>, <span class="author">Schatz DG</span>, <span class="author">Aragon L</span>, <span class="author">Fisher AG</span>, <span class="author">Krangel MS</span>, <span class="author">Nasmyth K</span> and <span class="author">Merkenschlager M</span></p>
<p class="affiliation">Lymphocyte Development Group, MRC Clinical Sciences Centre, Imperial College London, Du Cane Road, London W12 0NN, UK.</p><p class="abstract">Cohesin enables post-replicative DNA repair and chromosome segregation by holding sister chromatids together from the time of DNA replication in S phase until mitosis. There is growing evidence that cohesin also forms long-range chromosomal cis-interactions and may regulate gene expression in association with CTCF, mediator or tissue-specific transcription factors. Human cohesinopathies such as Cornelia de Lange syndrome are thought to result from impaired non-canonical cohesin functions, but a clear distinction between the cell-division-related and cell-division-independent functions of cohesion--as exemplified in Drosophila--has not been demonstrated in vertebrate systems. To address this, here we deleted the cohesin locus Rad21 in mouse thymocytes at a time in development when these cells stop cycling and rearrange their T-cell receptor (TCR) &alpha; locus (Tcra). Rad21-deficient thymocytes had a normal lifespan and retained the ability to differentiate, albeit with reduced efficiency. Loss of Rad21 led to defective chromatin architecture at the Tcra locus, where cohesion-binding sites flank the TEA promoter and the E&alpha; enhancer, and demarcate Tcra from interspersed Tcrd elements and neighbouring housekeeping genes. Cohesin was required for long-range promoter-enhancer interactions, Tcra transcription, H3K4me3 histone modifications that recruit the recombination machinery and Tcra rearrangement. Provision of pre-rearranged TCR transgenes largely rescued thymocyte differentiation, demonstrating that among thousands of potential target genes across the genome, defective Tcra rearrangement was limiting for the differentiation of cohesin-deficient thymocytes. These findings firmly establish a cell-division-independent role for cohesin in Tcra locus rearrangement and provide a comprehensive account of the mechanisms by which cohesin enables cellular differentiation in a well-characterized mammalian system.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">MC_U120027516</span>, <span class="grant">MC_U120081295</span>; NIAID NIH HHS: <span class="grant">R37 AI032524</span>, <span class="grant">R37 AI032524-20</span>; NIGMS NIH HHS: <span class="grant">R37 GM041052</span>, <span class="grant">R37 GM041052-22</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">476</span>;<span class="number">7361</span>;<span class="pagerange">467-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21832993">21832993</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3179485">3179485</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10312">10.1038/nature10312</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Silencing of RhoA nucleotide exchange factor, ARHGEF3, reveals its unexpected role in iron uptake.</h4>
<p class="authors"><span class="author">Serbanovic-Canic J</span>, <span class="author">Cvejic A</span>, <span class="author">Soranzo N</span>, <span class="author">Stemple DL</span>, <span class="author">Ouwehand WH</span> and <span class="author">Freson K</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge and NHS Blood and Transplant, Cambridge, UK.</p><p class="abstract">Genomewide association meta-analysis studies have identified &gt; 100 independent genetic loci associated with blood cell indices, including volume and count of platelets and erythrocytes. Although several of these loci encode known regulators of hematopoiesis, the mechanism by which most sequence variants exert their effect on blood cell formation remains elusive. An example is the Rho guanine nucleotide exchange factor, ARHGEF3, which was previously implicated by genomewide association meta-analysis studies in bone cell biology. Here, we report on the unexpected role of ARHGEF3 in regulation of iron uptake and erythroid cell maturation. Although early erythroid differentiation progressed normally, silencing of arhgef3 in Danio rerio resulted in microcytic and hypochromic anemia. This was rescued by intracellular supplementation of iron, showing that arhgef3-depleted erythroid cells are fully capable of hemoglobinization. Disruption of the arhgef3 target, RhoA, also produced severe anemia, which was, again, corrected by iron injection. Moreover, silencing of ARHGEF3 in erythromyeloblastoid cells K562 showed that the uptake of transferrin was severely impaired. Taken together, this is the first study to provide evidence for ARHGEF3 being a regulator of transferrin uptake in erythroid cells, through activation of RHOA.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 077037/Z/05/Z</span>, <span class="grant">WT077047/Z/05/Z</span>, <span class="grant">WT082597/Z/07/Z</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2011</span>;<span class="volume">118</span>;<span class="number">18</span>;<span class="pagerange">4967-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21715309">21715309</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3208301">3208301</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2011-02-337295">10.1182/blood-2011-02-337295</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pneu tricks.</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p><p class="abstract">This month&#39;s Genome Watch looks at how recombination has provided Streptococcus pneumoniae with the adaptability to overcome challenges.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">4</span>;<span class="pagerange">230</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21407240" rel="external">21407240</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2547" rel="external">10.1038/nrmicro2547</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of the zoonotic pathogen Chlamydophila psittaci.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span>, <span class="author">Harris SR</span>, <span class="author">Rance R</span>, <span class="author">West AP</span>, <span class="author">Severin JA</span>, <span class="author">Ossewaarde JM</span>, <span class="author">Cutcliffe LT</span>, <span class="author">Skilton RJ</span>, <span class="author">Marsh P</span>, <span class="author">Parkhill J</span>, <span class="author">Clarke IN</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Pathogen Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK.</p><p class="abstract">We present the first genome sequence of Chlamydophila psittaci, an intracellular pathogen of birds and a human zoonotic pathogen. A comparison with previously sequenced Chlamydophila genomes shows that, as in other chlamydiae, most of the genome diversity is restricted to the plasticity zone. The C. psittaci plasmid was also sequenced.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT076964</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2011</span>;<span class="volume">193</span>;<span class="number">5</span>;<span class="pagerange">1282-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21183672" rel="external">21183672</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3067587" rel="external">3067587</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01435-10" rel="external">10.1128/JB.01435-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Indian Siddis: African descendants with Indian admixture.</h4>
<p class="authors"><span class="author">Shah AM</span>, <span class="author">Tamang R</span>, <span class="author">Moorjani P</span>, <span class="author">Rani DS</span>, <span class="author">Govindaraj P</span>, <span class="author">Kulkarni G</span>, <span class="author">Bhattacharya T</span>, <span class="author">Mustak MS</span>, <span class="author">Bhaskar LV</span>, <span class="author">Reddy AG</span>, <span class="author">Gadhvi D</span>, <span class="author">Gai PB</span>, <span class="author">Chaubey G</span>, <span class="author">Patterson N</span>, <span class="author">Reich D</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Singh L</span> and <span class="author">Thangaraj K</span></p>
<p class="affiliation">Centre for Cellular and Molecular Biology, Council of Scientific and Industrial Research, Hyderabad, India.</p><p class="abstract">The Siddis (Afro-Indians) are a tribal population whose members live in coastal Karnataka, Gujarat, and in some parts of Andhra Pradesh. Historical records indicate that the Portuguese brought the Siddis to India from Africa about 300-500 years ago; however, there is little information about their more precise ancestral origins. Here, we perform a genome-wide survey to understand the population history of the Siddis. Using hundreds of thousands of autosomal markers, we show that they have inherited ancestry from Africans, Indians, and possibly Europeans (Portuguese). Additionally, analyses of the uniparental (Y-chromosomal and mitochondrial DNA) markers indicate that the Siddis trace their ancestry to Bantu speakers from sub-Saharan Africa. We estimate that the admixture between the African ancestors of the Siddis and neighboring South Asian groups probably occurred in the past eight generations (&sim;200 years ago), consistent with historical records.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">89</span>;<span class="number">1</span>;<span class="pagerange">154-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21741027">21741027</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3135801">3135801</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2011.05.030">10.1016/j.ajhg.2011.05.030</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Source of the human malaria parasite Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Sharp PM</span>, <span class="author">Liu W</span>, <span class="author">Learn GH</span>, <span class="author">Rayner JC</span>, <span class="author">Peeters M</span> and <span class="author">Hahn BH</span></p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R37 AI050529-11</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">38</span>;<span class="pagerange">E744-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21903922" rel="external">21903922</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3179099" rel="external">3179099</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1112134108" rel="external">10.1073/pnas.1112134108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data mining using the Catalogue of Somatic Mutations in Cancer BioMart.</h4>
<p class="authors"><span class="author">Shepherd R</span>, <span class="author">Forbes SA</span>, <span class="author">Beare D</span>, <span class="author">Bamford S</span>, <span class="author">Cole CG</span>, <span class="author">Ward S</span>, <span class="author">Bindal N</span>, <span class="author">Gunasekaran P</span>, <span class="author">Jia M</span>, <span class="author">Kok CY</span>, <span class="author">Leung K</span>, <span class="author">Menzies A</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">Campbell PJ</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.</p><p class="abstract">Catalogue of Somatic Mutations in Cancer (COSMIC) (http://www.sanger.ac.uk/cosmic) is a publicly available resource providing information on somatic mutations implicated in human cancer. Release v51 (January 2011) includes data from just over 19,000 genes, 161,787 coding mutations and 5573 gene fusions, described in more than 577,000 tumour samples. COSMICMart (COSMIC BioMart) provides a flexible way to mine these data and combine somatic mutations with other biological relevant data sets. This article describes the data available in COSMIC along with examples of how to successfully mine and integrate data sets using COSMICMart. DATABASE URL: http://www.sanger.ac.uk/genetics/CGP/cosmic/biomart/martview/.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2011</span>;<span class="volume">2011</span>;<span class="pagerange">bar018</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21609966" rel="external">21609966</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3263736" rel="external">3263736</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbar018" rel="external">10.1093/database/bar018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants on 8p12 and 1q24.2 confer risk of schizophrenia.</h4>
<p class="authors"><span class="author">Shi Y</span>, <span class="author">Li Z</span>, <span class="author">Xu Q</span>, <span class="author">Wang T</span>, <span class="author">Li T</span>, <span class="author">Shen J</span>, <span class="author">Zhang F</span>, <span class="author">Chen J</span>, <span class="author">Zhou G</span>, <span class="author">Ji W</span>, <span class="author">Li B</span>, <span class="author">Xu Y</span>, <span class="author">Liu D</span>, <span class="author">Wang P</span>, <span class="author">Yang P</span>, <span class="author">Liu B</span>, <span class="author">Sun W</span>, <span class="author">Wan C</span>, <span class="author">Qin S</span>, <span class="author">He G</span>, <span class="author">Steinberg S</span>, <span class="author">Cichon S</span>, <span class="author">Werge T</span>, <span class="author">Sigurdsson E</span>, <span class="author">Tosato S</span>, <span class="author">Palotie A</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Rietschel M</span>, <span class="author">Ophoff RA</span>, <span class="author">Collier DA</span>, <span class="author">Rujescu D</span>, <span class="author">Clair DS</span>, <span class="author">Stefansson H</span>, <span class="author">Stefansson K</span>, <span class="author">Ji J</span>, <span class="author">Wang Q</span>, <span class="author">Li W</span>, <span class="author">Zheng L</span>, <span class="author">Zhang H</span>, <span class="author">Feng G</span> and <span class="author">He L</span></p>
<p class="affiliation">Bio-X Institutes, Key Laboratory for the Genetics of Developmental and Neuropsychiatric Disorders, Ministry of Education, Shanghai Jiao Tong University, China. <a href="mailto:%73%68%69%79%6F%6E%67%79%6F%6E%67%40%67%6D%61%69%6C.%63%6F%6D">&#115;&#104;&#105;&#121;&#111;&#110;&#103;&#121;&#111;&#110;&#103;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Schizophrenia is a severe mental disorder affecting &sim;1% of the world population, with heritability of up to 80%. To identify new common genetic risk factors, we performed a genome-wide association study (GWAS) in the Han Chinese population. The discovery sample set consisted of 3,750 individuals with schizophrenia and 6,468 healthy controls (1,578 cases and 1,592 controls from northern Han Chinese, 1,238 cases and 2,856 controls from central Han Chinese, and 934 cases and 2,020 controls from the southern Han Chinese). We further analyzed the strongest association signals in an additional independent cohort of 4,383 cases and 4,539 controls from the Han Chinese population. Meta-analysis identified common SNPs that associated with schizophrenia with genome-wide significance on 8p12 (rs16887244, P = 1.27 &times; 10(-10)) and 1q24.2 (rs10489202, P = 9.50 &times; 10(-9)). Our findings provide new insights into the pathogenesis of schizophrenia.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">12</span>;<span class="pagerange">1224-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22037555">22037555</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.980">10.1038/ng.980</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comparative survey of the frequency and distribution of polymorphism in the genome of Xenopus tropicalis.</h4>
<p class="authors"><span class="author">Showell C</span>, <span class="author">Carruthers S</span>, <span class="author">Hall A</span>, <span class="author">Pardo-Manuel de Villena F</span>, <span class="author">Stemple D</span> and <span class="author">Conlon FL</span></p>
<p class="affiliation">UNC McAllister Heart Institute, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, United States of America.</p><p class="abstract">Naturally occurring DNA sequence variation within a species underlies evolutionary adaptation and can give rise to phenotypic changes that provide novel insight into biological questions. This variation exists in laboratory populations just as in wild populations and, in addition to being a source of useful alleles for genetic studies, can impact efforts to identify induced mutations in sequence-based genetic screens. The Western clawed frog Xenopus tropicalis (X. tropicalis) has been adopted as a model system for studying the genetic control of embryonic development and a variety of other areas of research. Its diploid genome has been extensively sequenced and efforts are underway to isolate mutants by phenotype- and genotype-based approaches. Here, we describe a study of genetic polymorphism in laboratory strains of X. tropicalis. Polymorphism was detected in the coding and non-coding regions of developmental genes distributed widely across the genome. Laboratory strains exhibit unexpectedly high frequencies of genetic polymorphism, with alleles carrying a variety of synonymous and non-synonymous codon substitutions and nucleotide insertions/deletions. Inter-strain comparisons of polymorphism uncover a high proportion of shared alleles between Nigerian and Ivory Coast strains, in spite of their distinct geographical origins. These observations will likely influence the design of future sequence-based mutation screens, particularly those using DNA mismatch-based detection methods which can be disrupted by the presence of naturally occurring sequence variants. The existence of a significant reservoir of alleles also suggests that existing laboratory stocks may be a useful source of novel alleles for mapping and functional studies.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL089641</span>; NICHD NIH HHS: <span class="grant">HD054354</span>; PHS HHS: <span class="grant">DEO18825</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">e22392</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21829622" rel="external">21829622</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3150332" rel="external">3150332</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0022392" rel="external">10.1371/journal.pone.0022392</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The tammar wallaby major histocompatibility complex shows evidence of past genomic instability.</h4>
<p class="authors"><span class="author">Siddle HV</span>, <span class="author">Deakin JE</span>, <span class="author">Coggill P</span>, <span class="author">Whilming LG</span>, <span class="author">Harrow J</span>, <span class="author">Kaufman J</span>, <span class="author">Beck S</span> and <span class="author">Belov K</span></p>
<p class="affiliation">Faculty of Veterinary Science, University of Sydney, NSW 2006, Australia.</p><p class="abstract"><strong>Background</strong>: The major histocompatibility complex (MHC) is a group of genes with a variety of roles in the innate and adaptive immune responses. MHC genes form a genetically linked cluster in eutherian mammals, an organization that is thought to confer functional and evolutionary advantages to the immune system. The tammar wallaby (Macropus eugenii), an Australian marsupial, provides a unique model for understanding MHC gene evolution, as many of its antigen presenting genes are not linked to the MHC, but are scattered around the genome.</p><p class="abstract"><strong>Results</strong>: Here we describe the &#39;core&#39; tammar wallaby MHC region on chromosome 2q by ordering and sequencing 33 BAC clones, covering over 4.5 MB and containing 129 genes. When compared to the MHC region of the South American opossum, eutherian mammals and non-mammals, the wallaby MHC has a novel gene organization. The wallaby has undergone an expansion of MHC class II genes, which are separated into two clusters by the class III genes. The antigen processing genes have undergone duplication, resulting in two copies of TAP1 and three copies of TAP2. Notably, Kangaroo Endogenous Retroviral Elements are present within the region and may have contributed to the genomic instability.</p><p class="abstract"><strong>Conclusions</strong>: The wallaby MHC has been extensively remodeled since the American and Australian marsupials last shared a common ancestor. The instability is characterized by the movement of antigen presenting genes away from the core MHC, most likely via the presence and activity of retroviral elements. We propose that the movement of class II genes away from the ancestral class II region has allowed this gene family to expand and diversify in the wallaby. The duplication of TAP genes in the wallaby MHC makes this species a unique model organism for studying the relationship between MHC gene organization and function.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">084071</span>, <span class="grant">089305</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="pagerange">421</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21854592">21854592</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3179965">3179965</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-12-421">10.1186/1471-2164-12-421</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of candidate genes linking systemic inflammation to atherosclerosis; results of a human in vivo LPS infusion study.</h4>
<p class="authors"><span class="author">Sivapalaratnam S</span>, <span class="author">Farrugia R</span>, <span class="author">Nieuwdorp M</span>, <span class="author">Langford CF</span>, <span class="author">van Beem RT</span>, <span class="author">Maiwald S</span>, <span class="author">Zwaginga JJ</span>, <span class="author">Gusnanto A</span>, <span class="author">Watkins NA</span>, <span class="author">Trip MD</span> and <span class="author">Ouwehand WH</span></p>
<p class="affiliation">Department of Vascular Medicine, Academic Medical Center, Meibergdreef 15, 1105 AZ, Amsterdam, The Netherlands. <a href="mailto:%73.%73%69%76%61%70%61%6C%61%72%61%74%6E%61%6D%40%61%6D%63.%75%76%61.%6E%6C">&#115;&#46;&#115;&#105;&#118;&#97;&#112;&#97;&#108;&#97;&#114;&#97;&#116;&#110;&#97;&#109;&#64;&#97;&#109;&#99;&#46;&#117;&#118;&#97;&#46;&#110;&#108;</a></p><p class="abstract"><strong>Background</strong>: It is widely accepted that atherosclerosis and inflammation are intimately linked. Monocytes play a key role in both of these processes and we hypothesized that activation of inflammatory pathways in monocytes would lead to, among others, proatherogenic changes in the monocyte transcriptome. Such differentially expressed genes in circulating monocytes would be strong candidates for further investigation in disease association studies.</p><p class="abstract"><strong>Methods</strong>: Endotoxin, lipopolysaccharide (LPS), or saline control was infused in healthy volunteers. Monocyte RNA was isolated, processed and hybridized to Hver 2.1.1 spotted cDNA microarrays. Differential expression of key genes was confirmed by RT-PCR and results were compared to in vitro data obtained by our group to identify candidate genes.</p><p class="abstract"><strong>Results</strong>: All subjects who received LPS experienced the anticipated clinical response indicating successful stimulation. One hour after LPS infusion, 11 genes were identified as being differentially expressed; 1 down regulated and 10 up regulated. Four hours after LPS infusion, 28 genes were identified as being differentially expressed; 3 being down regulated and 25 up regulated. No genes were significantly differentially expressed following saline infusion. Comparison with results obtained in in vitro experiments lead to the identification of 6 strong candidate genes (BATF, BID, C3aR1, IL1RN, SEC61B and SLC43A3)</p><p class="abstract"><strong>Conclusion</strong>: In vivo endotoxin exposure of healthy individuals resulted in the identification of several candidate genes through which systemic inflammation links to atherosclerosis.</p>
<p class="publication"><span class="title">BMC medical genomics</span>&nbsp;<span class="year">2011</span>;<span class="volume">4</span>;<span class="pagerange">64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21827714" rel="external">21827714</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3174875" rel="external">3174875</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1755-8794-4-64" rel="external">10.1186/1755-8794-4-64</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A conditional knockout resource for the genome-wide study of mouse gene function.</h4>
<p class="authors"><span class="author">Skarnes WC</span>, <span class="author">Rosen B</span>, <span class="author">West AP</span>, <span class="author">Koutsourakis M</span>, <span class="author">Bushell W</span>, <span class="author">Iyer V</span>, <span class="author">Mujica AO</span>, <span class="author">Thomas M</span>, <span class="author">Harrow J</span>, <span class="author">Cox T</span>, <span class="author">Jackson D</span>, <span class="author">Severin J</span>, <span class="author">Biggs P</span>, <span class="author">Fu J</span>, <span class="author">Nefedov M</span>, <span class="author">de Jong PJ</span>, <span class="author">Stewart AF</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%6B%61%72%6E%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#107;&#97;&#114;&#110;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Gene targeting in embryonic stem cells has become the principal technology for manipulation of the mouse genome, offering unrivalled accuracy in allele design and access to conditional mutagenesis. To bring these advantages to the wider research community, large-scale mouse knockout programmes are producing a permanent resource of targeted mutations in all protein-coding genes. Here we report the establishment of a high-throughput gene-targeting pipeline for the generation of reporter-tagged, conditional alleles. Computational allele design, 96-well modular vector construction and high-efficiency gene-targeting strategies have been combined to mutate genes on an unprecedented scale. So far, more than 12,000 vectors and 9,000 conditional targeted alleles have been produced in highly germline-competent C57BL/6N embryonic stem cells. High-throughput genome engineering highlighted by this study is broadly applicable to rat and human stem cells and provides a foundation for future genome-wide efforts aimed at deciphering the function of all genes encoded by the mammalian genome.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01-HG004080</span>; Wellcome Trust: <span class="grant">077188</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">474</span>;<span class="number">7351</span>;<span class="pagerange">337-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21677750">21677750</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3572410">3572410</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10163">10.1038/nature10163</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of an imprinted master trans regulator at the KLF14 locus related to multiple metabolic phenotypes.</h4>
<p class="authors"><span class="author">Small KS</span>, <span class="author">Hedman AK</span>, <span class="author">Grundberg E</span>, <span class="author">Nica AC</span>, <span class="author">Thorleifsson G</span>, <span class="author">Kong A</span>, <span class="author">Thorsteindottir U</span>, <span class="author">Shin SY</span>, <span class="author">Richards HB</span>, <span class="author">GIANT Consortium</span>, <span class="author">MAGIC Investigators</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Soranzo N</span>, <span class="author">Ahmadi KR</span>, <span class="author">Lindgren CM</span>, <span class="author">Stefansson K</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Deloukas P</span>, <span class="author">Spector TD</span>, <span class="author">McCarthy MI</span> and <span class="author">MuTHER Consortium</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King&#39;s College London, London, UK.</p><p class="abstract">Genome-wide association studies have identified many genetic variants associated with complex traits. However, at only a minority of loci have the molecular mechanisms mediating these associations been characterized. In parallel, whereas cis regulatory patterns of gene expression have been extensively explored, the identification of trans regulatory effects in humans has attracted less attention. Here we show that the type 2 diabetes and high-density lipoprotein cholesterol-associated cis-acting expression quantitative trait locus (eQTL) of the maternally expressed transcription factor KLF14 acts as a master trans regulator of adipose gene expression. Expression levels of genes regulated by this trans-eQTL are highly correlated with concurrently measured metabolic traits, and a subset of the trans-regulated genes harbor variants directly associated with metabolic phenotypes. This trans-eQTL network provides a mechanistic understanding of the effect of the KLF14 locus on metabolic disease risk and offers a potential model for other complex traits.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900339</span>; NIMH NIH HHS: <span class="grant">R01 MH090941</span>; Wellcome Trust: <span class="grant">079771</span>, <span class="grant">081878</span>, <span class="grant">081917</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">6</span>;<span class="pagerange">561-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21572415">21572415</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3192952">3192952</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.833">10.1038/ng.833</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic predictors of fibrin D-dimer levels in healthy adults.</h4>
<p class="authors"><span class="author">Smith NL</span>, <span class="author">Huffman JE</span>, <span class="author">Strachan DP</span>, <span class="author">Huang J</span>, <span class="author">Dehghan A</span>, <span class="author">Trompet S</span>, <span class="author">Lopez LM</span>, <span class="author">Shin SY</span>, <span class="author">Baumert J</span>, <span class="author">Vitart V</span>, <span class="author">Bis JC</span>, <span class="author">Wild SH</span>, <span class="author">Rumley A</span>, <span class="author">Yang Q</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Stott DJ</span>, <span class="author">Davies G</span>, <span class="author">Carter AM</span>, <span class="author">Thorand B</span>, <span class="author">Pola&scaron;ek O</span>, <span class="author">McKnight B</span>, <span class="author">Campbell H</span>, <span class="author">Rudnicka AR</span>, <span class="author">Chen MH</span>, <span class="author">Buckley BM</span>, <span class="author">Harris SE</span>, <span class="author">Peters A</span>, <span class="author">Pulanic D</span>, <span class="author">Lumley T</span>, <span class="author">de Craen AJ</span>, <span class="author">Liewald DC</span>, <span class="author">Gieger C</span>, <span class="author">Campbell S</span>, <span class="author">Ford I</span>, <span class="author">Gow AJ</span>, <span class="author">Luciano M</span>, <span class="author">Porteous DJ</span>, <span class="author">Guo X</span>, <span class="author">Sattar N</span>, <span class="author">Tenesa A</span>, <span class="author">Cushman M</span>, <span class="author">Slagboom PE</span>, <span class="author">Visscher PM</span>, <span class="author">Spector TD</span>, <span class="author">Illig T</span>, <span class="author">Rudan I</span>, <span class="author">Bovill EG</span>, <span class="author">Wright AF</span>, <span class="author">McArdle WL</span>, <span class="author">Tofler G</span>, <span class="author">Hofman A</span>, <span class="author">Westendorp RG</span>, <span class="author">Starr JM</span>, <span class="author">Grant PJ</span>, <span class="author">Karakas M</span>, <span class="author">Hastie ND</span>, <span class="author">Psaty BM</span>, <span class="author">Wilson JF</span>, <span class="author">Lowe GD</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Witteman JC</span>, <span class="author">Jukema JW</span>, <span class="author">Deary IJ</span>, <span class="author">Soranzo N</span>, <span class="author">Koenig W</span> and <span class="author">Hayward C</span></p>
<p class="affiliation">Department of Epidemiology, University of Washington, Seattle 98101, USA. <a href="mailto:%6E%6C%73%6D%69%74%68%40%75.%77%61%73%68%69%6E%67%74%6F%6E.%65%64%75">&#110;&#108;&#115;&#109;&#105;&#116;&#104;&#64;&#117;&#46;&#119;&#97;&#115;&#104;&#105;&#110;&#103;&#116;&#111;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Fibrin fragment D-dimer, one of several peptides produced when crosslinked fibrin is degraded by plasmin, is the most widely used clinical marker of activated blood coagulation. To identity genetic loci influencing D-dimer levels, we performed the first large-scale, genome-wide association search.</p><p class="abstract">A genome-wide investigation of the genomic correlates of plasma D-dimer levels was conducted among 21 052 European-ancestry adults. Plasma levels of D-dimer were measured independently in each of 13 cohorts. Each study analyzed the association between &asymp;2.6 million genotyped and imputed variants across the 22 autosomal chromosomes and natural-log&ndash;transformed D-dimer levels using linear regression in additive genetic models adjusted for age and sex. Among all variants, 74 exceeded the genome-wide significance threshold and marked 3 regions. At 1p22, rs12029080 (P=6.4&times;10(-52)) was 46.0 kb upstream from F3, coagulation factor III (tissue factor). At 1q24, rs6687813 (P=2.4&times;10(-14)) was 79.7 kb downstream of F5, coagulation factor V. At 4q32, rs13109457 (P=2.9&times;10(-18)) was located between 2 fibrinogen genes: 10.4 kb downstream from FGG and 3.0 kb upstream from FGA. Variants were associated with a 0.099-, 0.096-, and 0.061-unit difference, respectively, in natural-log&ndash;transformed D-dimer and together accounted for 1.8% of the total variance. When adjusted for nonsynonymous substitutions in F5 and FGA loci known to be associated with D-dimer levels, there was no evidence of an additional association at either locus.</p><p class="abstract"><strong>Conclusions</strong>: Three genes were associated with fibrin D-dimer levels. Of these 3, the F3 association was the strongest, and has not been previously reported.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F019394/1</span>, <span class="grant">SAG09977</span>; Chief Scientist Office: <span class="grant">CZB/4/276</span>, <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">U.1275.00.001(61128)</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>; NHLBI NIH HHS: <span class="grant">HL073410</span>, <span class="grant">HL095080</span>, <span class="grant">N01-HC-15103</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55222</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL 087652</span>, <span class="grant">U01 HL080295</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">K24 DK080140</span>, <span class="grant">U01 DK062418</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">079895</span>, <span class="grant">079996</span>, <span class="grant">091746</span>, <span class="grant">091746/Z/</span></p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2011</span>;<span class="volume">123</span>;<span class="number">17</span>;<span class="pagerange">1864-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21502573" rel="external">21502573</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3095913" rel="external">3095913</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.110.009480" rel="external">10.1161/CIRCULATIONAHA.110.009480</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate gene association study for diabetic retinopathy in persons with type 2 diabetes: the Candidate gene Association Resource (CARe).</h4>
<p class="authors"><span class="author">Sobrin L</span>, <span class="author">Green T</span>, <span class="author">Sim X</span>, <span class="author">Jensen RA</span>, <span class="author">Tai ES</span>, <span class="author">Tay WT</span>, <span class="author">Wang JJ</span>, <span class="author">Mitchell P</span>, <span class="author">Sandholm N</span>, <span class="author">Liu Y</span>, <span class="author">Hietala K</span>, <span class="author">Iyengar SK</span>, <span class="author">Family Investigation of Nephropathy and Diabetes-Eye Research Group</span>, <span class="author">Brooks M</span>, <span class="author">Buraczynska M</span>, <span class="author">Van Zuydam N</span>, <span class="author">Smith AV</span>, <span class="author">Gudnason V</span>, <span class="author">Doney AS</span>, <span class="author">Morris AD</span>, <span class="author">Leese GP</span>, <span class="author">Palmer CN</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Swaroop A</span>, <span class="author">Taylor HA</span>, <span class="author">Wilson JG</span>, <span class="author">Penman A</span>, <span class="author">Chen CJ</span>, <span class="author">Groop PH</span>, <span class="author">Saw SM</span>, <span class="author">Aung T</span>, <span class="author">Klein BE</span>, <span class="author">Rotter JI</span>, <span class="author">Siscovick DS</span>, <span class="author">Cotch MF</span>, <span class="author">Klein R</span>, <span class="author">Daly MJ</span> and <span class="author">Wong TY</span></p>
<p class="affiliation">Department of Ophthalmology, Harvard Medical School, Massachusetts Eye and Ear Infirmary, Boston, Massachusetts 02114, USA. <a href="mailto:%6C%75%63%69%61%5F%73%6F%62%72%69%6E%40%6D%65%65%69.%68%61%72%76%61%72%64.%65%64%75">&#108;&#117;&#99;&#105;&#97;&#95;&#115;&#111;&#98;&#114;&#105;&#110;&#64;&#109;&#101;&#101;&#105;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Purpose</strong>: To investigate whether variants in cardiovascular candidate genes, some of which have been previously associated with type 2 diabetes (T2D), diabetic retinopathy (DR), and diabetic nephropathy (DN), are associated with DR in the Candidate gene Association Resource (CARe).</p><p class="abstract"><strong>Methods</strong>: Persons with T2D who were enrolled in the study (n = 2691) had fundus photography and genotyping of single nucleotide polymorphisms (SNPs) in 2000 candidate genes. Two case definitions were investigated: Early Treatment Diabetic Retinopathy Study (ETDRS) grades &ge; 14 and &ge; 30. The &chi;&sup2; analyses for each CARe cohort were combined by Cochran-Mantel-Haenszel (CMH) pooling of odds ratios (ORs) and corrected for multiple hypothesis testing. Logistic regression was performed with adjustment for other DR risk factors. Results from replication in independent cohorts were analyzed with CMH meta-analysis methods.</p><p class="abstract"><strong>Results</strong>: Among 39 genes previously associated with DR, DN, or T2D, three SNPs in P-selectin (SELP) were associated with DR. The strongest association was to rs6128 (OR = 0.43, P = 0.0001, after Bonferroni correction). These associations remained significant after adjustment for DR risk factors. Among other genes examined, several variants were associated with DR with significant P values, including rs6856425 tagging &alpha;-l-iduronidase (IDUA) (P = 2.1 &times; 10(-5), after Bonferroni correction). However, replication in independent cohorts did not reveal study-wide significant effects. The P values after replication were 0.55 and 0.10 for rs6128 and rs6856425, respectively.</p><p class="abstract"><strong>Conclusions</strong>: Genes associated with DN, T2D, and vascular diseases do not appear to be consistently associated with DR. A few genetic variants associated with DR, particularly those in SELP and near IDUA, should be investigated in additional DR cohorts.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">UL1 RR 025758</span>; NEI NIH HHS: <span class="grant">K12-EY16335</span>, <span class="grant">Z01 EY000401-06</span>, <span class="grant">Z01 EY000401-07</span>, <span class="grant">Z01 EY000403-06</span>, <span class="grant">Z01 EY000403-07</span>, <span class="grant">Z01 EY000425-04</span>, <span class="grant">Z99 EY999999</span>, <span class="grant">ZIA EY000401-08</span>, <span class="grant">ZIA EY000401-09</span>, <span class="grant">ZIA EY000401-10</span>, <span class="grant">ZIA EY000403-08</span>, <span class="grant">ZIA EY000403-09</span>, <span class="grant">ZIA EY000403-10</span>, <span class="grant">ZIA EY000425-06</span>; NHLBI NIH HHS: <span class="grant">N01-HC-65226</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Investigative ophthalmology &amp; visual science</span>&nbsp;<span class="year">2011</span>;<span class="volume">52</span>;<span class="number">10</span>;<span class="pagerange">7593-602</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21873659">21873659</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3183981">3183981</a>; DOI: <a href="http://dx.doi.org/10.1167%2Fiovs.11-7510">10.1167/iovs.11-7510</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association and large-scale follow up identifies 16 new loci influencing lung function.</h4>
<p class="authors"><span class="author">Soler Artigas M</span>, <span class="author">Loth DW</span>, <span class="author">Wain LV</span>, <span class="author">Gharib SA</span>, <span class="author">Obeidat M</span>, <span class="author">Tang W</span>, <span class="author">Zhai G</span>, <span class="author">Zhao JH</span>, <span class="author">Smith AV</span>, <span class="author">Huffman JE</span>, <span class="author">Albrecht E</span>, <span class="author">Jackson CM</span>, <span class="author">Evans DM</span>, <span class="author">Cadby G</span>, <span class="author">Fornage M</span>, <span class="author">Manichaikul A</span>, <span class="author">Lopez LM</span>, <span class="author">Johnson T</span>, <span class="author">Aldrich MC</span>, <span class="author">Aspelund T</span>, <span class="author">Barroso I</span>, <span class="author">Campbell H</span>, <span class="author">Cassano PA</span>, <span class="author">Couper DJ</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Franceschini N</span>, <span class="author">Garcia M</span>, <span class="author">Gieger C</span>, <span class="author">Gislason GK</span>, <span class="author">Grkovic I</span>, <span class="author">Hammond CJ</span>, <span class="author">Hancock DB</span>, <span class="author">Harris TB</span>, <span class="author">Ramasamy A</span>, <span class="author">Heckbert SR</span>, <span class="author">Heli&ouml;vaara M</span>, <span class="author">Homuth G</span>, <span class="author">Hysi PG</span>, <span class="author">James AL</span>, <span class="author">Jankovic S</span>, <span class="author">Joubert BR</span>, <span class="author">Karrasch S</span>, <span class="author">Klopp N</span>, <span class="author">Koch B</span>, <span class="author">Kritchevsky SB</span>, <span class="author">Launer LJ</span>, <span class="author">Liu Y</span>, <span class="author">Loehr LR</span>, <span class="author">Lohman K</span>, <span class="author">Loos RJ</span>, <span class="author">Lumley T</span>, <span class="author">Al Balushi KA</span>, <span class="author">Ang WQ</span>, <span class="author">Barr RG</span>, <span class="author">Beilby J</span>, <span class="author">Blakey JD</span>, <span class="author">Boban M</span>, <span class="author">Boraska V</span>, <span class="author">Brisman J</span>, <span class="author">Britton JR</span>, <span class="author">Brusselle GG</span>, <span class="author">Cooper C</span>, <span class="author">Curjuric I</span>, <span class="author">Dahgam S</span>, <span class="author">Deary IJ</span>, <span class="author">Ebrahim S</span>, <span class="author">Eijgelsheim M</span>, <span class="author">Francks C</span>, <span class="author">Gaysina D</span>, <span class="author">Granell R</span>, <span class="author">Gu X</span>, <span class="author">Hankinson JL</span>, <span class="author">Hardy R</span>, <span class="author">Harris SE</span>, <span class="author">Henderson J</span>, <span class="author">Henry A</span>, <span class="author">Hingorani AD</span>, <span class="author">Hofman A</span>, <span class="author">Holt PG</span>, <span class="author">Hui J</span>, <span class="author">Hunter ML</span>, <span class="author">Imboden M</span>, <span class="author">Jameson KA</span>, <span class="author">Kerr SM</span>, <span class="author">Kolcic I</span>, <span class="author">Kronenberg F</span>, <span class="author">Liu JZ</span>, <span class="author">Marchini J</span>, <span class="author">McKeever T</span>, <span class="author">Morris AD</span>, <span class="author">Olin AC</span>, <span class="author">Porteous DJ</span>, <span class="author">Postma DS</span>, <span class="author">Rich SS</span>, <span class="author">Ring SM</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rochat T</span>, <span class="author">Sayer AA</span>, <span class="author">Sayers I</span>, <span class="author">Sly PD</span>, <span class="author">Smith GD</span>, <span class="author">Sood A</span>, <span class="author">Starr JM</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Vonk JM</span>, <span class="author">Wannamethee SG</span>, <span class="author">Whincup PH</span>, <span class="author">Wijmenga C</span>, <span class="author">Williams OD</span>, <span class="author">Wong A</span>, <span class="author">Mangino M</span>, <span class="author">Marciante KD</span>, <span class="author">McArdle WL</span>, <span class="author">Meibohm B</span>, <span class="author">Morrison AC</span>, <span class="author">North KE</span>, <span class="author">Omenaas E</span>, <span class="author">Palmer LJ</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Pin I</span>, <span class="author">Pola Sbreve Ek O</span>, <span class="author">Pouta A</span>, <span class="author">Psaty BM</span>, <span class="author">Hartikainen AL</span>, <span class="author">Rantanen T</span>, <span class="author">Ripatti S</span>, <span class="author">Rotter JI</span>, <span class="author">Rudan I</span>, <span class="author">Rudnicka AR</span>, <span class="author">Schulz H</span>, <span class="author">Shin SY</span>, <span class="author">Spector TD</span>, <span class="author">Surakka I</span>, <span class="author">Vitart V</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wareham NJ</span>, <span class="author">Warrington NM</span>, <span class="author">Wichmann HE</span>, <span class="author">Wild SH</span>, <span class="author">Wilk JB</span>, <span class="author">Wjst M</span>, <span class="author">Wright AF</span>, <span class="author">Zgaga L</span>, <span class="author">Zemunik T</span>, <span class="author">Pennell CE</span>, <span class="author">Nyberg F</span>, <span class="author">Kuh D</span>, <span class="author">Holloway JW</span>, <span class="author">Boezen HM</span>, <span class="author">Lawlor DA</span>, <span class="author">Morris RW</span>, <span class="author">Probst-Hensch N</span>, <span class="author">International Lung Cancer Consortium</span>, <span class="author">GIANT consortium</span>, <span class="author">Kaprio J</span>, <span class="author">Wilson JF</span>, <span class="author">Hayward C</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Heinrich J</span>, <span class="author">Musk AW</span>, <span class="author">Jarvis DL</span>, <span class="author">Gl&auml;ser S</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Ch Stricker BH</span>, <span class="author">Elliott P</span>, <span class="author">O&#39;Connor GT</span>, <span class="author">Strachan DP</span>, <span class="author">London SJ</span>, <span class="author">Hall IP</span>, <span class="author">Gudnason V</span> and <span class="author">Tobin MD</span></p>
<p class="affiliation">Department of Health Sciences, University of Leicester, Leicester, UK.</p><p class="abstract">Pulmonary function measures reflect respiratory health and are used in the diagnosis of chronic obstructive pulmonary disease. We tested genome-wide association with forced expiratory volume in 1 second and the ratio of forced expiratory volume in 1 second to forced vital capacity in 48,201 individuals of European ancestry with follow up of the top associations in up to an additional 46,411 individuals. We identified new regions showing association (combined P &lt; 5 &times; 10(-8)) with pulmonary function in or near MFAP2, TGFB2, HDAC4, RARB, MECOM (also known as EVI1), SPATA9, ARMC2, NCR3, ZKSCAN3, CDC123, C10orf11, LRP1, CCDC38, MMP15, CFDP1 and KCNE2. Identification of these 16 new loci may provide insight into the molecular mechanisms regulating pulmonary function and into molecular targets for future therapy to alleviate reduced lung function.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F019394/1</span>, <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">FS05/125</span>, <span class="grant">PG/06/154/22043</span>, <span class="grant">PG/97012</span>, <span class="grant">RG/08/013/25942</span>; Canadian Institutes of Health Research: <span class="grant">MOP-82893</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/710</span>, <span class="grant">CZD/16/6</span>, <span class="grant">CZD/16/6/2</span>, <span class="grant">CZD/16/6/4</span>; Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401540</span>, <span class="grant">G0500539</span>, <span class="grant">G0501942</span>, <span class="grant">G0600705</span>, <span class="grant">G0701863</span>, <span class="grant">G0800582</span>, <span class="grant">G0801056</span>, <span class="grant">G0902125</span>, <span class="grant">G0902313</span>, <span class="grant">G1000861</span>, <span class="grant">G9815508</span>, <span class="grant">G9901462</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U123092721</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NCI NIH HHS: <span class="grant">1P50 CA70907</span>, <span class="grant">CA127219</span>, <span class="grant">CA55769</span>, <span class="grant">R01 CA121197</span>, <span class="grant">R01CA111703</span>, <span class="grant">U19 CA148127</span>; NCRR NIH HHS: <span class="grant">5M01 RR00997</span>, <span class="grant">M01-RR00425</span>, <span class="grant">RR-024156</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01-HG-004402</span>, <span class="grant">U01-HG-004729</span>; NHLBI NIH HHS: <span class="grant">1K23HL094531-01</span>, <span class="grant">5R01HL087679-02</span>, <span class="grant">HL075336</span>, <span class="grant">HL080295</span>, <span class="grant">HL087652</span>, <span class="grant">HL088133</span>, <span class="grant">HL105756</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-25195</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-05187</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-45134</span>, <span class="grant">N01-HC-45204</span>, <span class="grant">N01-HC-45205</span>, <span class="grant">N01-HC-48047</span>, <span class="grant">N01-HC-48048</span>, <span class="grant">N01-HC-48049</span>, <span class="grant">N01-HC-48050</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-95095</span>, <span class="grant">N01-HC-95159</span>, <span class="grant">N01-HC-95160</span>, <span class="grant">N01-HC-95161</span>, <span class="grant">N01-HC-95162</span>, <span class="grant">N01-HC-95163</span>, <span class="grant">N01-HC-95164</span>, <span class="grant">N01-HC-95165</span>, <span class="grant">N01-HC-95166</span>, <span class="grant">N01-HC-95167</span>, <span class="grant">N01-HC-95168</span>, <span class="grant">N01-HC-95169</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL-071022</span>, <span class="grant">R01 HL-074104</span>, <span class="grant">R01 HL-077612</span>, <span class="grant">R01 HL075476</span>, <span class="grant">R01 HL077612</span>, <span class="grant">R01-HL-084099</span>, <span class="grant">R01-HL084099</span>, <span class="grant">R01HL071051</span>, <span class="grant">R01HL071205</span>, <span class="grant">R01HL071250</span>, <span class="grant">R01HL071251</span>, <span class="grant">R01HL071252</span>, <span class="grant">R01HL071258</span>, <span class="grant">R01HL071259</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">RC1 HL100543</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">AG-023269</span>, <span class="grant">AG-027058</span>, <span class="grant">AG-15928</span>, <span class="grant">AG-20098</span>, <span class="grant">AG035835</span>, <span class="grant">N01AG12100</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>, <span class="grant">R01 AG032098</span>, <span class="grant">RC1 AG035835</span>, <span class="grant">RC1 AG035835-01</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>; NIEHS NIH HHS: <span class="grant">ES015794</span>, <span class="grant">ZO1 ES49019</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span>; PHS HHS: <span class="grant">268200625226C</span>, <span class="grant">268200782096C</span>, <span class="grant">268201100005C</span>, <span class="grant">268201100006C</span>, <span class="grant">268201100007C</span>, <span class="grant">268201100008C</span>, <span class="grant">268201100009C</span>, <span class="grant">268201100010C</span>, <span class="grant">268201100011C</span>, <span class="grant">268201100012C</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span>, <span class="grant">090532</span>, <span class="grant">092731</span>, <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">11</span>;<span class="pagerange">1082-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21946350">21946350</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3267376">3267376</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.941">10.1038/ng.941</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic determinants of variability in glycated hemoglobin (HbA(1c)) in humans: review of recent progress and prospects for use in diabetes care.</h4>
<p class="authors"><span class="author">Soranzo N</span></p>
<p class="affiliation">Human Genetics, Wellcome Trust Sanger Institute, Genome Campus, Hinxton CB10 1HH, UK. <a href="mailto:%6E%69%63%6F%6C%65.%73%6F%72%61%6E%7A%6F%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#105;&#99;&#111;&#108;&#101;&#46;&#115;&#111;&#114;&#97;&#110;&#122;&#111;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Glycated hemoglobin A(1c) (HbA(1c)) indicates the percentage of total hemoglobin that is bound by glucose, produced from the nonenzymatic chemical modification by glucose of hemoglobin molecules carried in erythrocytes. HbA(1c) represents a surrogate marker of average blood glucose concentration over the previous 8 to 12 weeks, or the average lifespan of the erythrocyte, and thus represents a more stable indicator of glycemic status compared with fasting glucose. HbA(1c) levels are genetically determined, with heritability of 47% to 59%. Over the past few years, inroads into understanding genetic predisposition by glycemic and nonglycemic factors have been achieved through genomewide analyses. Here I review current research aimed at discovering genetic determinants of HbA(1c) levels, discussing insights into biologic factors influencing variability in the general and diabetic population, and across different ethnicities. Furthermore, I discuss briefly the relevance of findings for diabetes monitoring and diagnosis.</p>
<p class="publication"><span class="title">Current diabetes reports</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="number">6</span>;<span class="pagerange">562-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21975967" rel="external">21975967</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3207128" rel="external">3207128</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs11892-011-0232-9" rel="external">10.1007/s11892-011-0232-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interpreting Association Signals</h4>
<p class="authors"><span class="author">Soranzo,N</span> and <span class="author">Dudbridge,F.;</span></p>
<p class="publication"><span class="title">Analysis of Complex Disease Association Studies</span>&nbsp;<span class="year">2011</span>;<span class="number">Chapter 16</span>;<span class="pagerange">261-76</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-375142-3.10016-1">10.1016/B978-0-12-375142-3.10016-1</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123751423100161" href="http://www.sciencedirect.com/science/article/pii/B9780123751423100161">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123751423</span>100161</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The effect of genome-wide association scan quality control on imputation outcome for common variants.</h4>
<p class="authors"><span class="author">Southam L</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Rayner NW</span>, <span class="author">Chapman K</span>, <span class="author">Durrant C</span>, <span class="author">Ferreira T</span>, <span class="author">Arden N</span>, <span class="author">Carr A</span>, <span class="author">Deloukas P</span>, <span class="author">Doherty M</span>, <span class="author">Loughlin J</span>, <span class="author">McCaskie A</span>, <span class="author">Ollier WE</span>, <span class="author">Ralston S</span>, <span class="author">Spector TD</span>, <span class="author">Valdes AM</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson JM</span>, <span class="author">arcOGEN consortium</span>, <span class="author">Marchini J</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, University of Oxford, Oxford, UK.</p><p class="abstract">Imputation is an extremely valuable tool in conducting and synthesising genome-wide association studies (GWASs). Directly typed SNP quality control (QC) is thought to affect imputation quality. It is, therefore, common practise to use quality-controlled (QCed) data as an input for imputing genotypes. This study aims to determine the effect of commonly applied QC steps on imputation outcomes. We performed several iterations of imputing SNPs across chromosome 22 in a dataset consisting of 3177 samples with Illumina 610 k (Illumina, San Diego, CA, USA) GWAS data, applying different QC steps each time. The imputed genotypes were compared with the directly typed genotypes. In addition, we investigated the correlation between alternatively QCed data. We also applied a series of post-imputation QC steps balancing elimination of poorly imputed SNPs and information loss. We found that the difference between the unQCed data and the fully QCed data on imputation outcome was minimal. Our study shows that imputation of common variants is generally very accurate and robust to GWAS QC, which is not a major factor affecting imputation outcome. A minority of common-frequency SNPs with particular properties cannot be accurately imputed regardless of QC stringency. These findings may not generalise to the imputation of low frequency and rare variants.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">18030</span>; Medical Research Council: <span class="grant">G0100594</span>, <span class="grant">G0901461</span>; Wellcome Trust: <span class="grant">079557</span>, <span class="grant">088885</span>, <span class="grant">090532</span>, <span class="grant">WT079557MA</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2011</span>;<span class="volume">19</span>;<span class="number">5</span>;<span class="pagerange">610-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21267008">21267008</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3083623">3083623</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2010.242">10.1038/ejhg.2010.242</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis identifies variants associated with nonalcoholic fatty liver disease that have distinct effects on metabolic traits.</h4>
<p class="authors"><span class="author">Speliotes EK</span>, <span class="author">Yerges-Armstrong LM</span>, <span class="author">Wu J</span>, <span class="author">Hernaez R</span>, <span class="author">Kim LJ</span>, <span class="author">Palmer CD</span>, <span class="author">Gudnason V</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Garcia ME</span>, <span class="author">Launer LJ</span>, <span class="author">Nalls MA</span>, <span class="author">Clark JM</span>, <span class="author">Mitchell BD</span>, <span class="author">Shuldiner AR</span>, <span class="author">Butler JL</span>, <span class="author">Tomas M</span>, <span class="author">Hoffmann U</span>, <span class="author">Hwang SJ</span>, <span class="author">Massaro JM</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Sahani DV</span>, <span class="author">Salomaa V</span>, <span class="author">Schadt EE</span>, <span class="author">Schwartz SM</span>, <span class="author">Siscovick DS</span>, <span class="author">NASH CRN</span>, <span class="author">GIANT Consortium</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Voight BF</span>, <span class="author">Carr JJ</span>, <span class="author">Feitosa MF</span>, <span class="author">Harris TB</span>, <span class="author">Fox CS</span>, <span class="author">Smith AV</span>, <span class="author">Kao WH</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Borecki IB</span> and <span class="author">GOLD Consortium</span></p>
<p class="affiliation">Department of Internal Medicine, Division of Gastroenterology, University of Michigan, Ann Arbor, Michigan, United States of America. <a href="mailto:%65%73%70%65%6C%69%6F%74%40%6D%65%64.%75%6D%69%63%68.%65%64%75">&#101;&#115;&#112;&#101;&#108;&#105;&#111;&#116;&#64;&#109;&#101;&#100;&#46;&#117;&#109;&#105;&#99;&#104;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Nonalcoholic fatty liver disease (NAFLD) clusters in families, but the only known common genetic variants influencing risk are near PNPLA3. We sought to identify additional genetic variants influencing NAFLD using genome-wide association (GWA) analysis of computed tomography (CT) measured hepatic steatosis, a non-invasive measure of NAFLD, in large population based samples. Using variance components methods, we show that CT hepatic steatosis is heritable (&sim;26%-27%) in family-based Amish, Family Heart, and Framingham Heart Studies (n&#x200A;=&#x200A;880 to 3,070). By carrying out a fixed-effects meta-analysis of genome-wide association (GWA) results between CT hepatic steatosis and &sim;2.4 million imputed or genotyped SNPs in 7,176 individuals from the Old Order Amish, Age, Gene/Environment Susceptibility-Reykjavik study (AGES), Family Heart, and Framingham Heart Studies, we identify variants associated at genome-wide significant levels (p&lt;5&times;10(-8)) in or near PNPLA3, NCAN, and PPP1R3B. We genotype these and 42 other top CT hepatic steatosis-associated SNPs in 592 subjects with biopsy-proven NAFLD from the NASH Clinical Research Network (NASH CRN). In comparisons with 1,405 healthy controls from the Myocardial Genetics Consortium (MIGen), we observe significant associations with histologic NAFLD at variants in or near NCAN, GCKR, LYPLAL1, and PNPLA3, but not PPP1R3B. Variants at these five loci exhibit distinct patterns of association with serum lipids, as well as glycemic and anthropometric traits. We identify common genetic variants influencing CT-assessed steatosis and risk of NAFLD. Hepatic steatosis associated variants are not uniformly associated with NASH/fibrosis or result in abnormalities in serum lipids or glycemic and anthropometric traits, suggesting genetic heterogeneity in the pathways influencing these traits.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/09/002/26056</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NCRR NIH HHS: <span class="grant">M01RR000065</span>, <span class="grant">M01RR000750</span>, <span class="grant">M01RR000827</span>, <span class="grant">M01RR00188</span>, <span class="grant">M01RR020359</span>, <span class="grant">UL1 RR024989</span>, <span class="grant">UL1RR024989</span>, <span class="grant">UL1RR02501401</span>; NHLBI NIH HHS: <span class="grant">N01-HC-25195</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01HL087700</span>, <span class="grant">R01HL088119</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U01 HL72515</span>; NIA NIH HHS: <span class="grant">N01-AG-12100</span>, <span class="grant">R01 AG18728</span>, <span class="grant">T32AG000262</span>; NIAMS NIH HHS: <span class="grant">F32AR059469</span>; NIDDK NIH HHS: <span class="grant">F32 DK079466-01</span>, <span class="grant">K01 DK067207</span>, <span class="grant">K23DK080145-01</span>, <span class="grant">K24 DK002957</span>, <span class="grant">P30DK072488</span>, <span class="grant">P60 DK079637</span>, <span class="grant">R01DK075681</span>, <span class="grant">R01DK075787</span>, <span class="grant">T32 DK07191-32</span>, <span class="grant">U01 DK061728</span>, <span class="grant">U01DK061713</span>, <span class="grant">U01DK061718</span>, <span class="grant">U01DK061728</span>, <span class="grant">U01DK061730</span>, <span class="grant">U01DK061731</span>, <span class="grant">U01DK061732</span>, <span class="grant">U01DK061734</span>, <span class="grant">U01DK061737</span>, <span class="grant">U01DK061738</span>; NIGMS NIH HHS: <span class="grant">T32 GM074905</span>; PHS HHS: <span class="grant">ULRR02413101</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">e1001324</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21423719">21423719</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3053321">3053321</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001324">10.1371/journal.pgen.1001324</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of Atlantic cod reveals a unique immune system.</h4>
<p class="authors"><span class="author">Star B</span>, <span class="author">Nederbragt AJ</span>, <span class="author">Jentoft S</span>, <span class="author">Grimholt U</span>, <span class="author">Malmstr&oslash;m M</span>, <span class="author">Gregers TF</span>, <span class="author">Rounge TB</span>, <span class="author">Paulsen J</span>, <span class="author">Solbakken MH</span>, <span class="author">Sharma A</span>, <span class="author">Wetten OF</span>, <span class="author">Lanz&eacute;n A</span>, <span class="author">Winer R</span>, <span class="author">Knight J</span>, <span class="author">Vogel JH</span>, <span class="author">Aken B</span>, <span class="author">Andersen O</span>, <span class="author">Lagesen K</span>, <span class="author">Tooming-Klunderud A</span>, <span class="author">Edvardsen RB</span>, <span class="author">Tina KG</span>, <span class="author">Espelund M</span>, <span class="author">Nepal C</span>, <span class="author">Previti C</span>, <span class="author">Karlsen BO</span>, <span class="author">Moum T</span>, <span class="author">Skage M</span>, <span class="author">Berg PR</span>, <span class="author">Gj&oslash;en T</span>, <span class="author">Kuhl H</span>, <span class="author">Thorsen J</span>, <span class="author">Malde K</span>, <span class="author">Reinhardt R</span>, <span class="author">Du L</span>, <span class="author">Johansen SD</span>, <span class="author">Searle S</span>, <span class="author">Lien S</span>, <span class="author">Nilsen F</span>, <span class="author">Jonassen I</span>, <span class="author">Omholt SW</span>, <span class="author">Stenseth NC</span> and <span class="author">Jakobsen KS</span></p>
<p class="affiliation">Centre for Ecological and Evolutionary Synthesis, Department of Biology, University of Oslo, PO Box 1066, Blindern, N-0316 Oslo, Norway.</p><p class="abstract">Atlantic cod (Gadus morhua) is a large, cold-adapted teleost that sustains long-standing commercial fisheries and incipient aquaculture. Here we present the genome sequence of Atlantic cod, showing evidence for complex thermal adaptations in its haemoglobin gene cluster and an unusual immune architecture compared to other sequenced vertebrates. The genome assembly was obtained exclusively by 454 sequencing of shotgun and paired-end libraries, and automated annotation identified 22,154 genes. The major histocompatibility complex (MHC)&thinsp;II is a conserved feature of the adaptive immune system of jawed vertebrates, but we show that Atlantic cod has lost the genes for MHC&thinsp;II, CD4 and invariant chain (Ii) that are essential for the function of this pathway. Nevertheless, Atlantic cod is not exceptionally susceptible to disease under natural conditions. We find a highly expanded number of MHC&thinsp;I genes and a unique composition of its Toll-like receptor (TLR) families. This indicates how the Atlantic cod immune system has evolved compensatory mechanisms in both adaptive and innate immunity in the absence of MHC&thinsp;II. These observations affect fundamental assumptions about the evolution of the adaptive immune system and its components in vertebrates.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7363</span>;<span class="pagerange">207-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21832995" rel="external">21832995</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10342" rel="external">10.1038/nature10342</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of vitamin D receptor gene polymorphisms in the bone mineral density of Greek postmenopausal women with low calcium intake.</h4>
<p class="authors"><span class="author">Stathopoulou MG</span>, <span class="author">Dedoussis GV</span>, <span class="author">Trovas G</span>, <span class="author">Theodoraki EV</span>, <span class="author">Katsalira A</span>, <span class="author">Dontas IA</span>, <span class="author">Hammond N</span>, <span class="author">Deloukas P</span> and <span class="author">Lyritis GP</span></p>
<p class="affiliation">Department of Nutrition - Dietetics, Harokopio University, 17671 Athens, Greece.</p><p class="abstract">The aim of this study was to investigate the effect of common vitamin D receptor (VDR) gene polymorphisms on the bone mineral density (BMD) of Greek postmenopausal women. Healthy postmenopausal women (n=578) were recruited for the study. The BMD of the lumbar spine and hip was measured using dual-energy X-ray absorptiometry with the Lunar DPX-MD device. Assessment of dietary calcium intake was performed with multiple 24-h recalls. Genotyping was performed for the BsmI, TaqI and Cdx-2 polymorphisms of the VDR gene. The selected polymorphisms were not associated with BMD, osteoporosis or osteoporotic fractures. Stratification by calcium intake revealed that in the low calcium intake group (&lt;680 mg/day), all polymorphisms were associated with the BMD of the lumbar spine (P&lt;.05). After adjustment for potential covariates, BsmI and TaqI polymorphisms were associated with the presence of osteoporosis (P&lt;.05), while the presence of the minor A allele of Cdx-2 polymorphism was associated with a lower spine BMD (P=.025). In the higher calcium intake group (&gt;680 mg/day), no significant differences were observed within the genotypes for all polymorphisms. The VDR gene is shown to affect BMD in women with low calcium intake, while its effect is masked in women with higher calcium intake. This result underlines the significance of adequate calcium intake in postmenopausal women, given that it exerts a positive effect on BMD even in the presence of negative genetic predisposition.</p>
<p class="publication"><span class="title">The Journal of nutritional biochemistry</span>&nbsp;<span class="year">2011</span>;<span class="volume">22</span>;<span class="number">8</span>;<span class="pagerange">752-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21115334">21115334</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jnutbio.2010.06.007">10.1016/j.jnutbio.2010.06.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at VRK2 and TCF4 conferring risk of schizophrenia.</h4>
<p class="authors"><span class="author">Steinberg S</span>, <span class="author">de Jong S</span>, <span class="author">Irish Schizophrenia Genomics Consortium</span>, <span class="author">Andreassen OA</span>, <span class="author">Werge T</span>, <span class="author">B&oslash;rglum AD</span>, <span class="author">Mors O</span>, <span class="author">Mortensen PB</span>, <span class="author">Gustafsson O</span>, <span class="author">Costas J</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Demontis D</span>, <span class="author">Papiol S</span>, <span class="author">Huttenlocher J</span>, <span class="author">Mattheisen M</span>, <span class="author">Breuer R</span>, <span class="author">Vassos E</span>, <span class="author">Giegling I</span>, <span class="author">Fraser G</span>, <span class="author">Walker N</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Suvisaari J</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Paunio T</span>, <span class="author">Agartz I</span>, <span class="author">Melle I</span>, <span class="author">Djurovic S</span>, <span class="author">Strengman E</span>, <span class="author">GROUP</span>, <span class="author">J&uuml;rgens G</span>, <span class="author">Glenth&oslash;j B</span>, <span class="author">Terenius L</span>, <span class="author">Hougaard DM</span>, <span class="author">&Oslash;rntoft T</span>, <span class="author">Wiuf C</span>, <span class="author">Didriksen M</span>, <span class="author">Hollegaard MV</span>, <span class="author">Nordentoft M</span>, <span class="author">van Winkel R</span>, <span class="author">Kenis G</span>, <span class="author">Abramova L</span>, <span class="author">Kaleda V</span>, <span class="author">Arrojo M</span>, <span class="author">Sanju&aacute;n J</span>, <span class="author">Arango C</span>, <span class="author">Sperling S</span>, <span class="author">Rossner M</span>, <span class="author">Ribolsi M</span>, <span class="author">Magni V</span>, <span class="author">Siracusano A</span>, <span class="author">Christiansen C</span>, <span class="author">Kiemeney LA</span>, <span class="author">Veldink J</span>, <span class="author">van den Berg L</span>, <span class="author">Ingason A</span>, <span class="author">Muglia P</span>, <span class="author">Murray R</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Sigurdsson E</span>, <span class="author">Petursson H</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Kong A</span>, <span class="author">Rubino IA</span>, <span class="author">De Hert M</span>, <span class="author">R&eacute;thelyi JM</span>, <span class="author">Bitter I</span>, <span class="author">J&ouml;nsson EG</span>, <span class="author">Golimbet V</span>, <span class="author">Carracedo A</span>, <span class="author">Ehrenreich H</span>, <span class="author">Craddock N</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Ruggeri M</span>, <span class="author">Tosato S</span>, <span class="author">Peltonen L</span>, <span class="author">Ophoff RA</span>, <span class="author">Collier DA</span>, <span class="author">St Clair D</span>, <span class="author">Rietschel M</span>, <span class="author">Cichon S</span>, <span class="author">Stefansson H</span>, <span class="author">Rujescu D</span> and <span class="author">Stefansson K</span></p>
<p class="affiliation">deCODE Genetics, IS-101 Reykjavik, Iceland.</p><p class="abstract">Common sequence variants have recently joined rare structural polymorphisms as genetic factors with strong evidence for association with schizophrenia. Here we extend our previous genome-wide association study and meta-analysis (totalling 7 946 cases and 19 036 controls) by examining an expanded set of variants using an enlarged follow-up sample (up to 10 260 cases and 23 500 controls). In addition to previously reported alleles in the major histocompatibility complex region, near neurogranin (NRGN) and in an intron of transcription factor 4 (TCF4), we find two novel variants showing genome-wide significant association: rs2312147[C], upstream of vaccinia-related kinase 2 (VRK2) [odds ratio (OR) = 1.09, P = 1.9 &times; 10(-9)] and rs4309482[A], between coiled-coiled domain containing 68 (CCDC68) and TCF4, about 400 kb from the previously described risk allele, but not accounted for by its association (OR = 1.09, P = 7.8 &times; 10(-9)).</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">MH074027</span>, <span class="grant">N01 MH900001</span>, <span class="grant">R01 MH078075</span>; NINDS NIH HHS: <span class="grant">R01 NS058980</span>; Wellcome Trust: <span class="grant">083948/Z/07/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">20</span>;<span class="pagerange">4076-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21791550" rel="external">21791550</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3298077" rel="external">3298077</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr325" rel="external">10.1093/hmg/ddr325</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expanding the range of ZNF804A variants conferring risk of psychosis.</h4>
<p class="authors"><span class="author">Steinberg S</span>, <span class="author">Mors O</span>, <span class="author">B&oslash;rglum AD</span>, <span class="author">Gustafsson O</span>, <span class="author">Werge T</span>, <span class="author">Mortensen PB</span>, <span class="author">Andreassen OA</span>, <span class="author">Sigurdsson E</span>, <span class="author">Thorgeirsson TE</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Olason P</span>, <span class="author">Ophoff RA</span>, <span class="author">Cichon S</span>, <span class="author">Gudjonsdottir IH</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Nyegaard M</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Ingason A</span>, <span class="author">Hansen T</span>, <span class="author">Athanasiu L</span>, <span class="author">Suvisaari J</span>, <span class="author">Lonnqvist J</span>, <span class="author">Paunio T</span>, <span class="author">Hartmann A</span>, <span class="author">J&uuml;rgens G</span>, <span class="author">Nordentoft M</span>, <span class="author">Hougaard D</span>, <span class="author">Norgaard-Pedersen B</span>, <span class="author">Breuer R</span>, <span class="author">M&ouml;ller HJ</span>, <span class="author">Giegling I</span>, <span class="author">Glenth&oslash;j B</span>, <span class="author">Rasmussen HB</span>, <span class="author">Mattheisen M</span>, <span class="author">Bitter I</span>, <span class="author">R&eacute;thelyi JM</span>, <span class="author">Sigmundsson T</span>, <span class="author">Fossdal R</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Ruggeri M</span>, <span class="author">Tosato S</span>, <span class="author">Strengman E</span>, <span class="author">Genetic Risk and Outcome in Psychosis</span>, <span class="author">Kiemeney LA</span>, <span class="author">Melle I</span>, <span class="author">Djurovic S</span>, <span class="author">Abramova L</span>, <span class="author">Kaleda V</span>, <span class="author">Walshe M</span>, <span class="author">Bramon E</span>, <span class="author">Vassos E</span>, <span class="author">Li T</span>, <span class="author">Fraser G</span>, <span class="author">Walker N</span>, <span class="author">Toulopoulou T</span>, <span class="author">Yoon J</span>, <span class="author">Freimer NB</span>, <span class="author">Cantor RM</span>, <span class="author">Murray R</span>, <span class="author">Kong A</span>, <span class="author">Golimbet V</span>, <span class="author">J&ouml;nsson EG</span>, <span class="author">Terenius L</span>, <span class="author">Agartz I</span>, <span class="author">Petursson H</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Rietschel M</span>, <span class="author">Peltonen L</span>, <span class="author">Rujescu D</span>, <span class="author">Collier DA</span>, <span class="author">Stefansson H</span>, <span class="author">St Clair D</span> and <span class="author">Stefansson K</span></p>
<p class="affiliation">deCODE genetics, Reykjavik, Iceland.</p><p class="abstract">A trio of genome-wide association studies recently reported sequence variants at three loci to be significantly associated with schizophrenia. No sequence polymorphism had been unequivocally (P&lt;5 &times; 10(-8)) associated with schizophrenia earlier. However, one variant, rs1344706[T], had come very close. This polymorphism, located in an intron of ZNF804A, was reported to associate with schizophrenia with a P-value of 1.6 &times; 10(-7), and with psychosis (schizophrenia plus bipolar disorder) with a P-value of 1.0 &times; 10(-8). In this study, using 5164 schizophrenia cases and 20,709 controls, we replicated the association with schizophrenia (odds ratio OR = 1.08, P = 0.0029) and, by adding bipolar disorder patients, we also confirmed the association with psychosis (added N = 609, OR = 1.09, P = 0.00065). Furthermore, as it has been proposed that variants such as rs1344706[T]-common and with low relative risk-may also serve to identify regions harboring less common, higher-risk susceptibility alleles, we searched ZNF804A for large copy number variants (CNVs) in 4235 psychosis patients, 1173 patients with other psychiatric disorders and 39,481 controls. We identified two CNVs including at least part of ZNF804A in psychosis patients and no ZNF804A CNVs in controls (P = 0.013 for association with psychosis). In addition, we found a ZNF804A CNV in an anxiety patient (P = 0.0016 for association with the larger set of psychiatric disorders).</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">R01MH078075</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">59-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20048749" rel="external">20048749</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3242031" rel="external">3242031</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2009.149" rel="external">10.1038/mp.2009.149</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Massive genomic rearrangement acquired in a single catastrophic event during cancer development.</h4>
<p class="authors"><span class="author">Stephens PJ</span>, <span class="author">Greenman CD</span>, <span class="author">Fu B</span>, <span class="author">Yang F</span>, <span class="author">Bignell GR</span>, <span class="author">Mudie LJ</span>, <span class="author">Pleasance ED</span>, <span class="author">Lau KW</span>, <span class="author">Beare D</span>, <span class="author">Stebbings LA</span>, <span class="author">McLaren S</span>, <span class="author">Lin ML</span>, <span class="author">McBride DJ</span>, <span class="author">Varela I</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Leroy C</span>, <span class="author">Jia M</span>, <span class="author">Menzies A</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">Quail MA</span>, <span class="author">Burton J</span>, <span class="author">Swerdlow H</span>, <span class="author">Carter NP</span>, <span class="author">Morsberger LA</span>, <span class="author">Iacobuzio-Donahue C</span>, <span class="author">Follows GA</span>, <span class="author">Green AR</span>, <span class="author">Flanagan AM</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Cancer is driven by somatically acquired point mutations and chromosomal rearrangements, conventionally thought to accumulate gradually over time. Using next-generation sequencing, we characterize a phenomenon, which we term chromothripsis, whereby tens to hundreds of genomic rearrangements occur in a one-off cellular crisis. Rearrangements involving one or a few chromosomes crisscross back and forth across involved regions, generating frequent oscillations between two copy number states. These genomic hallmarks are highly improbable if rearrangements accumulate over time and instead imply that nearly all occur during a single cellular catastrophe. The stamp of chromothripsis can be seen in at least 2%-3% of all cancers, across many subtypes, and is present in &sim;25% of bone cancers. We find that one, or indeed more than one, cancer-causing lesion can emerge out of the genomic crisis. This phenomenon has important implications for the origins of genomic remodeling and temporal emergence of cancer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">144</span>;<span class="number">1</span>;<span class="pagerange">27-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21215367">21215367</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3065307">3065307</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2010.11.055">10.1016/j.cell.2010.11.055</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two covariance models for iron-responsive elements.</h4>
<p class="authors"><span class="author">Stevens SG</span>, <span class="author">Gardner PP</span> and <span class="author">Brown C</span></p>
<p class="affiliation">Biochemistry and Genetics Otago, University of Otago, Dunedin, New Zealand.</p><p class="abstract">Iron-responsive elements (IREs) function in the 5&#39; or 3&#39; untranslated regions (UTRs) of mRNAs as post-transcriptional structured cis-acting RNA regulatory elements. One known functional mechanism is the binding of Iron Regulatory Proteins (IRPs) to 5&#39; UTR IREs, reducing translation rates at low iron levels. Another known mechanism is IRPs binding to 3&#39; UTR IREs in other mRNAs, increasing RNA stability. Experimentally proven elements are quite small, have some diversity of sequence and structure, and functional genes have similar pseudogenes in the genome. This paper presents two new IRE covariance models, comprising a new IRE clan in the RFAM database to encompass this variation without over-generalisation. Two IRE models rather than a single model is consistent with experimentally proven structures and predictions. All of the IREs with experimental support are modelled. These two new models show a marked increase in the sensitivity and specificity in detection of known iron-responsive elements and ability to predict novel IREs.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">RNA biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">792-801</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21881407">21881407</a>; DOI: <a href="http://dx.doi.org/10.4161%2Frna.8.5.16037">10.4161/rna.8.5.16037</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exploring the genomes of cancer cells: progress and promise.</h4>
<p class="authors"><span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%6D%72%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#114;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The description and interpretation of genomic abnormalities in cancer cells have been at the heart of cancer research for more than a century. With exhaustive sequencing of cancer genomes across a wide range of human tumors well under way, we are now entering the end game of this mission. In the forthcoming decade, essentially complete catalogs of somatic mutations will be generated for tens of thousands of human cancers. Here, I provide an overview of what these efforts have revealed to date about the origin and behavioral features of cancer cells and how this genomic information is being exploited to improve diagnosis and therapy of the disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2011</span>;<span class="volume">331</span>;<span class="number">6024</span>;<span class="pagerange">1553-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21436442" rel="external">21436442</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1204040" rel="external">10.1126/science.1204040</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association identifies nine common variants associated with fasting proinsulin levels and provides new insights into the pathophysiology of type 2 diabetes.</h4>
<p class="authors"><span class="author">Strawbridge RJ</span>, <span class="author">Dupuis J</span>, <span class="author">Prokopenko I</span>, <span class="author">Barker A</span>, <span class="author">Ahlqvist E</span>, <span class="author">Rybin D</span>, <span class="author">Petrie JR</span>, <span class="author">Travers ME</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Dimas AS</span>, <span class="author">Nica A</span>, <span class="author">Wheeler E</span>, <span class="author">Chen H</span>, <span class="author">Voight BF</span>, <span class="author">Taneera J</span>, <span class="author">Kanoni S</span>, <span class="author">Peden JF</span>, <span class="author">Turrini F</span>, <span class="author">Gustafsson S</span>, <span class="author">Zabena C</span>, <span class="author">Almgren P</span>, <span class="author">Barker DJ</span>, <span class="author">Barnes D</span>, <span class="author">Dennison EM</span>, <span class="author">Eriksson JG</span>, <span class="author">Eriksson P</span>, <span class="author">Eury E</span>, <span class="author">Folkersen L</span>, <span class="author">Fox CS</span>, <span class="author">Frayling TM</span>, <span class="author">Goel A</span>, <span class="author">Gu HF</span>, <span class="author">Horikoshi M</span>, <span class="author">Isomaa B</span>, <span class="author">Jackson AU</span>, <span class="author">Jameson KA</span>, <span class="author">Kajantie E</span>, <span class="author">Kerr-Conte J</span>, <span class="author">Kuulasmaa T</span>, <span class="author">Kuusisto J</span>, <span class="author">Loos RJ</span>, <span class="author">Luan J</span>, <span class="author">Makrilakis K</span>, <span class="author">Manning AK</span>, <span class="author">Mart&iacute;nez-Larrad MT</span>, <span class="author">Narisu N</span>, <span class="author">Nastase Mannila M</span>, <span class="author">Ohrvik J</span>, <span class="author">Osmond C</span>, <span class="author">Pascoe L</span>, <span class="author">Payne F</span>, <span class="author">Sayer AA</span>, <span class="author">Sennblad B</span>, <span class="author">Silveira A</span>, <span class="author">Stanc&aacute;kov&aacute; A</span>, <span class="author">Stirrups K</span>, <span class="author">Swift AJ</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Tuomi T</span>, <span class="author">van &#39;t Hooft FM</span>, <span class="author">Walker M</span>, <span class="author">Weedon MN</span>, <span class="author">Xie W</span>, <span class="author">Zethelius B</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">GIANT Consortium</span>, <span class="author">MuTHER Consortium</span>, <span class="author">CARDIoGRAM Consortium</span>, <span class="author">C4D Consortium</span>, <span class="author">Ongen H</span>, <span class="author">M&auml;larstig A</span>, <span class="author">Hopewell JC</span>, <span class="author">Saleheen D</span>, <span class="author">Chambers J</span>, <span class="author">Parish S</span>, <span class="author">Danesh J</span>, <span class="author">Kooner J</span>, <span class="author">Ostenson CG</span>, <span class="author">Lind L</span>, <span class="author">Cooper CC</span>, <span class="author">Serrano-R&iacute;os M</span>, <span class="author">Ferrannini E</span>, <span class="author">Forsen TJ</span>, <span class="author">Clarke R</span>, <span class="author">Franzosi MG</span>, <span class="author">Seedorf U</span>, <span class="author">Watkins H</span>, <span class="author">Froguel P</span>, <span class="author">Johnson P</span>, <span class="author">Deloukas P</span>, <span class="author">Collins FS</span>, <span class="author">Laakso M</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span>, <span class="author">Wareham NJ</span>, <span class="author">Groop L</span>, <span class="author">Pattou F</span>, <span class="author">Gloyn AL</span>, <span class="author">Dedoussis GV</span>, <span class="author">Lyssenko V</span>, <span class="author">Meigs JB</span>, <span class="author">Barroso I</span>, <span class="author">Watanabe RM</span>, <span class="author">Ingelsson E</span>, <span class="author">Langenberg C</span>, <span class="author">Hamsten A</span> and <span class="author">Florez JC</span></p>
<p class="affiliation">Atherosclerosis Research Unit, Department of Medicine Solna, Karolinska Institutet, Karolinska University Hospital Solna, Stockholm, Sweden.</p><p class="abstract"><strong>Objective</strong>: Proinsulin is a precursor of mature insulin and C-peptide. Higher circulating proinsulin levels are associated with impaired &beta;-cell function, raised glucose levels, insulin resistance, and type 2 diabetes (T2D). Studies of the insulin processing pathway could provide new insights about T2D pathophysiology.</p><p class="abstract">We have conducted a meta-analysis of genome-wide association tests of &sim;2.5 million genotyped or imputed single nucleotide polymorphisms (SNPs) and fasting proinsulin levels in 10,701 nondiabetic adults of European ancestry, with follow-up of 23 loci in up to 16,378 individuals, using additive genetic models adjusted for age, sex, fasting insulin, and study-specific covariates.</p><p class="abstract"><strong>Results</strong>: Nine SNPs at eight loci were associated with proinsulin levels (P &lt; 5 &times; 10(-8)). Two loci (LARP6 and SGSM2) have not been previously related to metabolic traits, one (MADD) has been associated with fasting glucose, one (PCSK1) has been implicated in obesity, and four (TCF7L2, SLC30A8, VPS13C/C2CD4A/B, and ARAP1, formerly CENTD2) increase T2D risk. The proinsulin-raising allele of ARAP1 was associated with a lower fasting glucose (P = 1.7 &times; 10(-4)), improved &beta;-cell function (P = 1.1 &times; 10(-5)), and lower risk of T2D (odds ratio 0.88; P = 7.8 &times; 10(-6)). Notably, PCSK1 encodes the protein prohormone convertase 1/3, the first enzyme in the insulin processing pathway. A genotype score composed of the nine proinsulin-raising alleles was not associated with coronary disease in two large case-control datasets.</p><p class="abstract"><strong>Conclusions</strong>: We have identified nine genetic variants associated with fasting proinsulin. Our findings illuminate the biology underlying glucose homeostasis and T2D development in humans and argue against a direct role of proinsulin in coronary artery disease pathogenesis.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/08/014/24067</span>; Medical Research Council: <span class="grant">81696</span>, <span class="grant">G0601261</span>, <span class="grant">G0601966</span>, <span class="grant">G0700222</span>, <span class="grant">G0700222(81696)</span>, <span class="grant">G0700931</span>, <span class="grant">G0801056</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U137686857</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NHLBI NIH HHS: <span class="grant">R01 HL087647</span>, <span class="grant">U01 HL054527</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">K24 DK080140</span>, <span class="grant">R01 DK078616</span>; NIGMS NIH HHS: <span class="grant">T32 GM074905</span>; Wellcome Trust: <span class="grant">077016/Z/05/Z</span>, <span class="grant">083270/Z/07/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2011</span>;<span class="volume">60</span>;<span class="number">10</span>;<span class="pagerange">2624-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21873549">21873549</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3178302">3178302</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb11-0415">10.2337/db11-0415</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Dictyostelium SH2 adaptor protein required for correct DIF-1 signaling and pattern formation.</h4>
<p class="authors"><span class="author">Sugden C</span>, <span class="author">Ross S</span>, <span class="author">Annesley SJ</span>, <span class="author">Cole C</span>, <span class="author">Bloomfield G</span>, <span class="author">Ivens A</span>, <span class="author">Skelton J</span>, <span class="author">Fisher PR</span>, <span class="author">Barton G</span> and <span class="author">Williams JG</span></p>
<p class="affiliation">School of Life Sciences, University of Dundee, Dow St., Dundee, DD1 5EH, UK.</p><p class="abstract">Dictyostelium is the only non-metazoan with functionally analyzed SH2 domains and studying them can give insights into their evolution and wider potential. LrrB has a novel domain configuration with leucine-rich repeat, 14-3-3 and SH2 protein-protein interaction modules. It is required for the correct expression of several specific genes in early development and here we characterize its role in later, multicellular development. During development in the light, slug formation in LrrB null (lrrB-) mutants is delayed relative to the parental strain, and the slugs are highly defective in phototaxis and thermotaxis. In the dark the mutant arrests development as an elongated mound, in a hitherto unreported process we term dark stalling. The developmental and phototaxis defects are cell autonomous and marker analysis shows that the pstO prestalk sub-region of the slug is aberrant in the lrrB- mutant. Expression profiling, by parallel micro-array and deep RNA sequence analyses, reveals many other alterations in prestalk-specific gene expression in lrrB- slugs, including reduced expression of the ecmB gene and elevated expression of ampA. During culmination ampA is ectopically expressed in the stalk, there is no expression of ampA and ecmB in the lower cup and the mutant fruiting bodies lack a basal disc. The basal disc cup derives from the pstB cells and this population is greatly reduced in the lrrB- mutant. This anatomical feature is a hallmark of mutants aberrant in signaling by DIF-1, the polyketide that induces prestalk and stalk cell differentiation. In a DIF-1 induction assay the lrrB- mutant is profoundly defective in ecmB activation but only marginally defective in ecmA induction. Thus the mutation partially uncouples these two inductive events. In early development LrrB interacts physically and functionally with CldA, another SH2 domain containing protein. However, the CldA null mutant does not phenocopy the lrrB- in its aberrant multicellular development or phototaxis defect, implying that the early and late functions of LrrB are affected in different ways. These observations, coupled with its domain structure, suggest that LrrB is an SH2 adaptor protein active in diverse developmental signaling pathways.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">053640/Z</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">353</span>;<span class="number">2</span>;<span class="pagerange">290-301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21396932" rel="external">21396932</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3085826" rel="external">3085826</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2011.03.003" rel="external">10.1016/j.ydbio.2011.03.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human metabolic individuality in biomedical and pharmaceutical research.</h4>
<p class="authors"><span class="author">Suhre K</span>, <span class="author">Shin SY</span>, <span class="author">Petersen AK</span>, <span class="author">Mohney RP</span>, <span class="author">Meredith D</span>, <span class="author">W&auml;gele B</span>, <span class="author">Altmaier E</span>, <span class="author">CARDIoGRAM</span>, <span class="author">Deloukas P</span>, <span class="author">Erdmann J</span>, <span class="author">Grundberg E</span>, <span class="author">Hammond CJ</span>, <span class="author">de Angelis MH</span>, <span class="author">Kastenm&uuml;ller G</span>, <span class="author">K&ouml;ttgen A</span>, <span class="author">Kronenberg F</span>, <span class="author">Mangino M</span>, <span class="author">Meisinger C</span>, <span class="author">Meitinger T</span>, <span class="author">Mewes HW</span>, <span class="author">Milburn MV</span>, <span class="author">Prehn C</span>, <span class="author">Raffler J</span>, <span class="author">Ried JS</span>, <span class="author">R&ouml;misch-Margl W</span>, <span class="author">Samani NJ</span>, <span class="author">Small KS</span>, <span class="author">Wichmann HE</span>, <span class="author">Zhai G</span>, <span class="author">Illig T</span>, <span class="author">Spector TD</span>, <span class="author">Adamski J</span>, <span class="author">Soranzo N</span> and <span class="author">Gieger C</span></p>
<p class="affiliation">Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum M&uuml;nchen, German Research Center for Environmental Health, Ingolst&auml;dter Landstra&szlig;e 1, 85764 Neuherberg, Germany. <a href="mailto:%6B%61%72%73%74%65%6E%40%73%75%68%72%65.%66%72">&#107;&#97;&#114;&#115;&#116;&#101;&#110;&#64;&#115;&#117;&#104;&#114;&#101;&#46;&#102;&#114;</a></p><p class="abstract">Genome-wide association studies (GWAS) have identified many risk loci for complex diseases, but effect sizes are typically small and information on the underlying biological processes is often lacking. Associations with metabolic traits as functional intermediates can overcome these problems and potentially inform individualized therapy. Here we report a comprehensive analysis of genotype-dependent metabolic phenotypes using a GWAS with non-targeted metabolomics. We identified 37 genetic loci associated with blood metabolite concentrations, of which 25 show effect sizes that are unusually high for GWAS and account for 10-60% differences in metabolite levels per allele copy. Our associations provide new functional insights for many disease-related associations that have been reported in previous studies, including those for cardiovascular and kidney disorders, type 2 diabetes, cancer, gout, venous thromboembolism and Crohn&#39;s disease. The study advances our knowledge of the genetic basis of metabolic individuality in humans and generates many new hypotheses for biomedical and pharmaceutical research.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; British Heart Foundation; Canadian Institutes of Health Research: <span class="grant">MOP172605</span>, <span class="grant">MOP77682</span>, <span class="grant">MOP&#x2010;82810</span>; Cancer Research UK; Medical Research Council; NHLBI NIH HHS: <span class="grant">1R01HL103931&#x2010;01</span>, <span class="grant">HL087647</span>, <span class="grant">N01&#x2010;HC&#x2010;55015</span>, <span class="grant">N01&#x2010;HC&#x2010;55016</span>, <span class="grant">N01&#x2010;HC&#x2010;55018</span>, <span class="grant">N01&#x2010;HC&#x2010;55019</span>, <span class="grant">N01&#x2010;HC&#x2010;55020</span>, <span class="grant">N01&#x2010;HC&#x2010;55021</span>, <span class="grant">N01&#x2010;HC&#x2010;55022</span>, <span class="grant">P01 HL098055</span>, <span class="grant">P01HL076491&#x2010;06</span>, <span class="grant">P01HL087018</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL087676</span>, <span class="grant">R01HL089650&#x2010;02</span>; NIA NIH HHS: <span class="grant">N01&#x2010;AG&#x2010;12100</span>; NIDDK NIH HHS: <span class="grant">R01DK080732</span>; Wellcome Trust: <span class="grant">091746</span>, <span class="grant">091746/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7362</span>;<span class="pagerange">54-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21886157">21886157</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3832838">3832838</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10354">10.1038/nature10354</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel missense mutations in MYO7A underlying postlingual high- or low-frequency non-syndromic hearing impairment in two large families from China.</h4>
<p class="authors"><span class="author">Sun Y</span>, <span class="author">Chen J</span>, <span class="author">Sun H</span>, <span class="author">Cheng J</span>, <span class="author">Li J</span>, <span class="author">Lu Y</span>, <span class="author">Lu Y</span>, <span class="author">Jin Z</span>, <span class="author">Zhu Y</span>, <span class="author">Ouyang X</span>, <span class="author">Yan D</span>, <span class="author">Dai P</span>, <span class="author">Han D</span>, <span class="author">Yang W</span>, <span class="author">Wang R</span>, <span class="author">Liu X</span> and <span class="author">Yuan H</span></p>
<p class="affiliation">Department of Otolaryngology, Head and Neck Surgery, Institute of Otolaryngology, Chinese PLA General Hospital, Beijing, China.</p><p class="abstract">The myosin VIIA (MYO7A) gene encodes a protein classified as an unconventional myosin. Mutations within MYO7A can lead to both syndromic and non-syndromic hearing impairment in humans. Among different mutations reported in MYO7A, only five led to non-syndromic sensorineural deafness autosomal dominant type 11 (DFNA11). Here, we present the clinical, genetic and molecular characteristics of two large Chinese DFNA11 families with either high- or low-frequency hearing loss. Affected individuals of family DX-J033 have a sloping audiogram at young ages with high frequency are most affected. With increasing age, all test frequencies are affected. Affected members of family HB-S037 present with an ascending audiogram affecting low frequencies at young ages, and then all frequencies are involved with increasing age. Genome-wide linkage analysis mapped the disease loci within the DFNA11 interval in both families. DNA sequencing of MYO7A revealed two novel nucleotide variations, c.652G &gt; A (p.D218N) and c.2011G &gt; A (p.G671S), in the two families. It is for the first time that the mutations identified in MYO7A in the present study are being implicated in DFNA11 in a Chinese population. For the first time, we tested electrocochleography (ECochG) in a DFNA11 family with low-frequency hearing loss. We speculate that the low-frequency sensorineural hearing loss in this DFNA11 family was not associated with endolymphatic hydrops.</p>
<p class="publication"><span class="title">Journal of human genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">56</span>;<span class="number">1</span>;<span class="pagerange">64-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21150918" rel="external">21150918</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fjhg.2010.147" rel="external">10.1038/jhg.2010.147</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide screen for interactions reveals a new locus on 4p15 modifying the effect of waist-to-hip ratio on total cholesterol.</h4>
<p class="authors"><span class="author">Surakka I</span>, <span class="author">Isaacs A</span>, <span class="author">Karssen LC</span>, <span class="author">Laurila PP</span>, <span class="author">Middelberg RP</span>, <span class="author">Tikkanen E</span>, <span class="author">Ried JS</span>, <span class="author">Lamina C</span>, <span class="author">Mangino M</span>, <span class="author">Igl W</span>, <span class="author">Hottenga JJ</span>, <span class="author">Lagou V</span>, <span class="author">van der Harst P</span>, <span class="author">Mateo Leach I</span>, <span class="author">Esko T</span>, <span class="author">Kutalik Z</span>, <span class="author">Wainwright NW</span>, <span class="author">Struchalin MV</span>, <span class="author">Sarin AP</span>, <span class="author">Kangas AJ</span>, <span class="author">Viikari JS</span>, <span class="author">Perola M</span>, <span class="author">Rantanen T</span>, <span class="author">Petersen AK</span>, <span class="author">Soininen P</span>, <span class="author">Johansson A</span>, <span class="author">Soranzo N</span>, <span class="author">Heath AC</span>, <span class="author">Papamarkou T</span>, <span class="author">Prokopenko I</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Kronenberg F</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Montgomery GW</span>, <span class="author">Whitfield JB</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Freimer NB</span>, <span class="author">Willemsen G</span>, <span class="author">de Geus EJ</span>, <span class="author">Palotie A</span>, <span class="author">Sandhu MS</span>, <span class="author">Waterworth DM</span>, <span class="author">Metspalu A</span>, <span class="author">Stumvoll M</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Jula A</span>, <span class="author">Navis G</span>, <span class="author">Wijmenga C</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Taskinen MR</span>, <span class="author">Ala-Korpela M</span>, <span class="author">Kaprio J</span>, <span class="author">Kyvik KO</span>, <span class="author">Boomsma DI</span>, <span class="author">Pedersen NL</span>, <span class="author">Gyllensten U</span>, <span class="author">Wilson JF</span>, <span class="author">Rudan I</span>, <span class="author">Campbell H</span>, <span class="author">Pramstaller PP</span>, <span class="author">Spector TD</span>, <span class="author">Witteman JC</span>, <span class="author">Eriksson JG</span>, <span class="author">Salomaa V</span>, <span class="author">Oostra BA</span>, <span class="author">Raitakari OT</span>, <span class="author">Wichmann HE</span>, <span class="author">Gieger C</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Martin NG</span>, <span class="author">Hofman A</span>, <span class="author">McCarthy MI</span>, <span class="author">Peltonen L</span>, <span class="author">van Duijn CM</span>, <span class="author">Aulchenko YS</span>, <span class="author">Ripatti S</span> and <span class="author">ENGAGE Consortium</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland.</p><p class="abstract">Recent genome-wide association (GWA) studies described 95 loci controlling serum lipid levels. These common variants explain &sim;25% of the heritability of the phenotypes. To date, no unbiased screen for gene-environment interactions for circulating lipids has been reported. We screened for variants that modify the relationship between known epidemiological risk factors and circulating lipid levels in a meta-analysis of genome-wide association (GWA) data from 18 population-based cohorts with European ancestry (maximum N&#x200A;=&#x200A;32,225). We collected 8 further cohorts (N&#x200A;=&#x200A;17,102) for replication, and rs6448771 on 4p15 demonstrated genome-wide significant interaction with waist-to-hip-ratio (WHR) on total cholesterol (TC) with a combined P-value of 4.79&times;10(-9). There were two potential candidate genes in the region, PCDH7 and CCKAR, with differential expression levels for rs6448771 genotypes in adipose tissue. The effect of WHR on TC was strongest for individuals carrying two copies of G allele, for whom a one standard deviation (sd) difference in WHR corresponds to 0.19 sd difference in TC concentration, while for A allele homozygous the difference was 0.12 sd. Our findings may open up possibilities for targeted intervention strategies for people characterized by specific genomic profiles. However, more refined measures of both body-fat distribution and metabolic measures are needed to understand how their joint dynamics are modified by the newly found locus.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/08/094/26019</span>; Cancer Research UK: <span class="grant">C865/A2883</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0300128</span>, <span class="grant">G0801566</span>, <span class="grant">G9502233</span>, <span class="grant">g0500539</span>, <span class="grant">g600705</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679</span>; NIAAA NIH HHS: <span class="grant">AA07535</span>, <span class="grant">AA10248</span>, <span class="grant">AA11998</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">AA14041</span>, <span class="grant">AA17688</span>, <span class="grant">K05 AA017688</span>; NIDA NIH HHS: <span class="grant">DA12854</span>; NIMH NIH HHS: <span class="grant">1R01MH083268-01</span>, <span class="grant">MH66206</span>, <span class="grant">U24 MH068457-06</span>; PHS HHS: <span class="grant">R01D0042157-01A</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">gr069224</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">e1002333</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22028671">22028671</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3197672">3197672</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002333">10.1371/journal.pgen.1002333</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Breast cancer genome heterogeneity: a challenge to personalised medicine?</h4>
<p class="authors"><span class="author">Swanton C</span>, <span class="author">Burrell RA</span> and <span class="author">Futreal PA</span></p><p class="abstract">Implementation of high-throughput genomics sequencing approaches into routine laboratory practice has raised the potential for the identification of multiple breast cancer targets suitable for future therapeutic intervention in order to improve cancer outcomes. Results from these studies have revealed bewildering breast cancer genome complexity with very few aberrations occurring in common between breast cancers. In addition, such complexity is compounded by evidence of genomic heterogeneity occurring within individual breast cancers. Such inter-tumoural and intratumoural heterogeneity is likely to present a challenge to personalised therapeutic approaches that might be circumvented through the definition of genome instability mechanisms governing such diversity and their exploitation using synthetic lethal approaches.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Breast cancer research : BCR</span>&nbsp;<span class="year">2011</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">104</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21345264" rel="external">21345264</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3109569" rel="external">3109569</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fbcr2807" rel="external">10.1186/bcr2807</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An optimized microarray platform for assaying genomic variation in Plasmodium falciparum field populations.</h4>
<p class="authors"><span class="author">Tan JC</span>, <span class="author">Miller BA</span>, <span class="author">Tan A</span>, <span class="author">Patel JJ</span>, <span class="author">Cheeseman IH</span>, <span class="author">Anderson TJ</span>, <span class="author">Manske M</span>, <span class="author">Maslen G</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Ferdig MT</span></p>
<p class="affiliation">The Eck Institute for Global Health, University of Notre Dame, 100 Galvin Life Sciences, Notre Dame, IN 46556, USA.</p><p class="abstract">We present an optimized probe design for copy number variation (CNV) and SNP genotyping in the Plasmodium falciparum genome. We demonstrate that variable length and isothermal probes are superior to static length probes. We show that sample preparation and hybridization conditions mitigate the effects of host DNA contamination in field samples. The microarray and workflow presented can be used to identify CNVs and SNPs with 95% accuracy in a single hybridization, in field samples containing up to 92% human DNA contamination.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G19/9</span>; NCRR NIH HHS: <span class="grant">RR013556</span>; NIAID NIH HHS: <span class="grant">AI072517</span>, <span class="grant">AI075145</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">4</span>;<span class="pagerange">R35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21477297">21477297</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3218861">3218861</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-4-r35">10.1186/gb-2011-12-4-r35</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies identifies common variants in CTNNA2 associated with excitement-seeking.</h4>
<p class="authors"><span class="author">Terracciano A</span>, <span class="author">Esko T</span>, <span class="author">Sutin AR</span>, <span class="author">de Moor MH</span>, <span class="author">Meirelles O</span>, <span class="author">Zhu G</span>, <span class="author">Tanaka T</span>, <span class="author">Giegling I</span>, <span class="author">Nutile T</span>, <span class="author">Realo A</span>, <span class="author">Allik J</span>, <span class="author">Hansell NK</span>, <span class="author">Wright MJ</span>, <span class="author">Montgomery GW</span>, <span class="author">Willemsen G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Friedl M</span>, <span class="author">Ruggiero D</span>, <span class="author">Sorice R</span>, <span class="author">Sanna S</span>, <span class="author">Cannas A</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Widen E</span>, <span class="author">Palotie A</span>, <span class="author">Eriksson JG</span>, <span class="author">Cucca F</span>, <span class="author">Krueger RF</span>, <span class="author">Lahti J</span>, <span class="author">Luciano M</span>, <span class="author">Smoller JW</span>, <span class="author">van Duijn CM</span>, <span class="author">Abecasis GR</span>, <span class="author">Boomsma DI</span>, <span class="author">Ciullo M</span>, <span class="author">Costa PT</span>, <span class="author">Ferrucci L</span>, <span class="author">Martin NG</span>, <span class="author">Metspalu A</span>, <span class="author">Rujescu D</span>, <span class="author">Schlessinger D</span> and <span class="author">Uda M</span></p>
<p class="affiliation">National Institute on Aging, NIH, DHHS, Baltimore, MD 21224, USA. <a href="mailto:%54%65%72%72%61%63%63%69%61%6E%6F%61%40%6D%61%69%6C.%6E%69%68.%67%6F%76">&#84;&#101;&#114;&#114;&#97;&#99;&#99;&#105;&#97;&#110;&#111;&#97;&#64;&#109;&#97;&#105;&#108;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p>
<p class="abstract">The tendency to seek stimulating activities and intense sensations define excitement-seeking, a personality trait akin to some aspects of sensation-seeking. This trait is a central feature of extraversion and is a component of the multifaceted impulsivity construct. Those who score high on measures of excitement-seeking are more likely to smoke, use other drugs, gamble, drive recklessly, have unsafe/unprotected sex and engage in other risky behaviors of clinical and social relevance. To identify common genetic variants associated with the Excitement-Seeking scale of the Revised NEO Personality Inventory, we performed genome-wide association studies in six samples of European ancestry (N=7860), and combined the results in a meta-analysis. We identified a genome-wide significant association between the Excitement-Seeking scale and rs7600563 (P=2 Ã— 10(-8)). This single-nucleotide polymorphism maps within the catenin cadherin-associated protein, alpha 2 (CTNNA2) gene, which encodes for a brain-expressed Î±-catenin critical for synaptic contact. The effect of rs7600563 was in the same direction in all six samples, but did not replicate in additional samples (N=5105). The results provide insight into the genetics of excitement-seeking and risk-taking, and are relevant to hyperactivity, substance use, antisocial and bipolar disorders.</p>
<p class="publication"><span class="title">Translational psychiatry</span>&nbsp;<span class="year">2011</span>;<span class="volume">1</span>;<span class="pagerange">e49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22833195" rel="external">22833195</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3309493" rel="external">3309493</a>; DOI: <a href="http://dx.doi.org/10.1038%2Ftp.2011.42" rel="external">10.1038/tp.2011.42</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The clinical and molecular genetic features of idiopathic infantile periodic alternating nystagmus.</h4>
<p class="authors"><span class="author">Thomas MG</span>, <span class="author">Crosier M</span>, <span class="author">Lindsay S</span>, <span class="author">Kumar A</span>, <span class="author">Thomas S</span>, <span class="author">Araki M</span>, <span class="author">Talbot CJ</span>, <span class="author">McLean RJ</span>, <span class="author">Surendran M</span>, <span class="author">Taylor K</span>, <span class="author">Leroy BP</span>, <span class="author">Moore AT</span>, <span class="author">Hunter DG</span>, <span class="author">Hertle RW</span>, <span class="author">Tarpey P</span>, <span class="author">Langmann A</span>, <span class="author">Lindner S</span>, <span class="author">Brandner M</span> and <span class="author">Gottlob I</span></p>
<p class="affiliation">Ophthalmology Group, School of Medicine, University of Leicester, RKCSB, PO Box 65, Leicester LE2 7LX, UK.</p><p class="abstract">Periodic alternating nystagmus consists of involuntary oscillations of the eyes with cyclical changes of nystagmus direction. It can occur during infancy (e.g. idiopathic infantile periodic alternating nystagmus) or later in life. Acquired forms are often associated with cerebellar dysfunction arising due to instability of the optokinetic-vestibular systems. Idiopathic infantile periodic alternating nystagmus can be familial or occur in isolation; however, very little is known about the clinical characteristics, genetic aetiology and neural substrates involved. Five loci (NYS1-5) have been identified for idiopathic infantile nystagmus; three are autosomal (NYS2, NYS3 and NYS4) and two are X-chromosomal (NYS1 and NYS5). We previously identified the FRMD7 gene on chromosome Xq26 (NYS1 locus); mutations of FRMD7 are causative of idiopathic infantile nystagmus influencing neuronal outgrowth and development. It is unclear whether the periodic alternating nystagmus phenotype is linked to NYS1, NYS5 (Xp11.4-p11.3) or a separate locus. From a cohort of 31 X-linked families and 14 singletons (70 patients) with idiopathic infantile nystagmus we identified 10 families and one singleton (21 patients) with periodic alternating nystagmus of which we describe clinical phenotype, genetic aetiology and neural substrates involved. Periodic alternating nystagmus was not detected clinically but only on eye movement recordings. The cycle duration varied from 90 to 280&thinsp;s. Optokinetic reflex was not detectable horizontally. Mutations of the FRMD7 gene were found in all 10 families and the singleton (including three novel mutations). Periodic alternating nystagmus was predominantly associated with missense mutations within the FERM domain. There was significant sibship clustering of the phenotype although in some families not all affected members had periodic alternating nystagmus. In situ hybridization studies during mid-late human embryonic stages in normal tissue showed restricted FRMD7 expression in neuronal tissue with strong hybridization signals within the afferent arms of the vestibulo-ocular reflex consisting of the otic vesicle, cranial nerve VIII and vestibular ganglia. Similarly within the afferent arm of the optokinetic reflex we showed expression in the developing neural retina and ventricular zone of the optic stalk. Strong FRMD7 expression was seen in rhombomeres 1 to 4, which give rise to the cerebellum and the common integrator site for both these reflexes (vestibular nuclei). Based on the expression and phenotypic data, we hypothesize that periodic alternating nystagmus arises from instability of the optokinetic-vestibular systems. This study shows for the first time that mutations in FRMD7 can cause idiopathic infantile periodic alternating nystagmus and may affect neuronal circuits that have been implicated in acquired forms.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G9900837</span></p>
<p class="publication"><span class="title">Brain : a journal of neurology</span>&nbsp;<span class="year">2011</span>;<span class="volume">134</span>;<span class="number">Pt 3</span>;<span class="pagerange">892-902</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21303855">21303855</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4125620">4125620</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbrain%2Fawq373">10.1093/brain/awq373</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide analysis of simultaneous GATA1/2, RUNX1, FLI1, and SCL binding in megakaryocytes identifies hematopoietic regulators.</h4>
<p class="authors"><span class="author">Tijssen MR</span>, <span class="author">Cvejic A</span>, <span class="author">Joshi A</span>, <span class="author">Hannah RL</span>, <span class="author">Ferreira R</span>, <span class="author">Forrai A</span>, <span class="author">Bellissimo DC</span>, <span class="author">Oram SH</span>, <span class="author">Smethurst PA</span>, <span class="author">Wilson NK</span>, <span class="author">Wang X</span>, <span class="author">Ottersbach K</span>, <span class="author">Stemple DL</span>, <span class="author">Green AR</span>, <span class="author">Ouwehand WH</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK.</p><p class="abstract">Hematopoietic differentiation critically depends on combinations of transcriptional regulators controlling the development of individual lineages. Here, we report the genome-wide binding sites for the five key hematopoietic transcription factors--GATA1, GATA2, RUNX1, FLI1, and TAL1/SCL--in primary human megakaryocytes. Statistical analysis of the 17,263 regions bound by at least one factor demonstrated that simultaneous binding by all five factors was the most enriched pattern and often occurred near known hematopoietic regulators. Eight genes not previously appreciated to function in hematopoiesis that were bound by all five factors were shown to be essential for thrombocyte and/or erythroid development in zebrafish. Moreover, one of these genes encoding the PDZK1IP1 protein shared transcriptional enhancer elements with the blood stem cell regulator TAL1/SCL. Multifactor ChIP-Seq analysis in primary human cells coupled with a high-throughput in vivo perturbation screen therefore offers a powerful strategy to identify essential regulators of complex mammalian differentiation processes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800784</span>, <span class="grant">G0900951</span>, <span class="grant">G0900951(91754)</span>; Wellcome Trust: <span class="grant">077037/Z/05/Z</span>, <span class="grant">077047/Z/05/Z</span>, <span class="grant">082597/Z/07/Z</span></p>
<p class="publication"><span class="title">Developmental cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">5</span>;<span class="pagerange">597-609</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21571218">21571218</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145975">3145975</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.devcel.2011.04.008">10.1016/j.devcel.2011.04.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of known loci with lipid levels among children and prediction of dyslipidemia in adults.</h4>
<p class="authors"><span class="author">Tikkanen E</span>, <span class="author">Tuovinen T</span>, <span class="author">Wid&eacute;n E</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Viikari J</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Peltonen L</span>, <span class="author">Raitakari OT</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">Institute for Molecular Medicine, Finland FIMM, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: Recent genome-wide association studies have found 95 distinct genetic loci associated with high-density (HDL-C) and low-density (LDL-C) lipoprotein cholesterol, total cholesterol (TC), and triglycerides (TG), using adult samples. It is not known if these variants are associated with lipid levels in children and adolescents and if the genetic risk score (GRS), based on these variants, could improve adulthood dyslipidemia prediction over the childhood lipid measurements.</p><p class="abstract">We used 2443 participants of the Cardiovascular Risk in Young Finns study cohort with up to 5 measurements of serum lipids taken between ages 3 and 45 years to estimate the effect of individual single-nucleotide polymorphisms and the GRS on lipids. The GRSs were strongly associated with lipids in all age groups (1.5 &times; 10(-20)&lt;P&lt;8.7 &times; 10(-12) for HDL-C, 3.5 &times; 10(-27)&lt;P&lt;5.6 &times; 10(-09) for LDL-C, 2.0 &times; 10(-25)&lt;P&lt;5.2 &times; 10(-09) for TC, and 4.1 &times; 10(-20)&lt;P&lt;8.4 &times; 10(-05) for TG). Jointly, the lipid loci explained 11.8-26.7% of the total variance in lipids among 3- to 6-year-old children, and the proportion dropped over age, except for TG. The discrimination of adult hypertriglyceridemia improved when GRS was added to childhood lipid measurement (C statistic=0.04, P=0.01).</p><p class="abstract"><strong>Conclusions</strong>: Previously identified lipid loci are associated with lipid levels in children and adolescents and explain up to more than 2 times of the lipid variation in children compared with adults. The TG-GRS improves the risk discrimination over childhood lipid measurement for adult hypertriglyceridemia.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Circulation. Cardiovascular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">4</span>;<span class="number">6</span>;<span class="pagerange">673-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21984478">21984478</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCGENETICS.111.960369">10.1161/CIRCGENETICS.111.960369</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tumor-specific diagnostic marker for transmissible facial tumors of Tasmanian devils: immunohistochemistry studies.</h4>
<p class="authors"><span class="author">Tovar C</span>, <span class="author">Obendorf D</span>, <span class="author">Murchison EP</span>, <span class="author">Papenfuss AT</span>, <span class="author">Kreiss A</span> and <span class="author">Woods GM</span></p>
<p class="affiliation">Menzies Research Institute, University of Tasmania, Hobart, Tasmania, Australia.</p><p class="abstract">Devil facial tumor disease (DFTD) is a transmissible neoplasm that is threatening the survival of the Tasmanian devil. Genetic analyses have indicated that the disease is a peripheral nerve sheath neoplasm of Schwann cell origin. DFTD cells express genes characteristic of myelinating Schwann cells, and periaxin, a Schwann cell protein, has been proposed as a marker for the disease. Diagnosis of DFTD is currently based on histopathology, cytogenetics, and clinical appearance of the disease in affected animals. As devils are susceptible to a variety of neoplastic processes, a specific diagnostic test is required to differentiate DFTD from cancers of similar morphological appearance. This study presents a thorough examination of the expression of a set of Schwann cell and other neural crest markers in DFTD tumors and normal devil tissues. Samples from 20 primary DFTD tumors and 10 DFTD metastases were evaluated by immunohistochemistry for the expression of periaxin, S100 protein, peripheral myelin protein 22, nerve growth factor receptor, nestin, neuron specific enolase, chromogranin A, and myelin basic protein. Of these, periaxin was confirmed as the most sensitive and specific marker, labeling the majority of DFTD cells in 100% of primary DFTD tumors and DFTD metastases. In normal tissues, periaxin showed specificity for Schwann cells in peripheral nerve bundles. This marker was then evaluated in cultured devil Schwann cells, DFTD cell lines, and xenografted DFTD tumors. Periaxin expression was maintained in all these models, validating its utility as a diagnostic marker for the disease.</p>
<p class="publication"><span class="title">Veterinary pathology</span>&nbsp;<span class="year">2011</span>;<span class="volume">48</span>;<span class="number">6</span>;<span class="pagerange">1195-203</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21383118">21383118</a>; DOI: <a href="http://dx.doi.org/10.1177%2F0300985811400447">10.1177/0300985811400447</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Messenger RNA and microRNA profiling during early mouse EB formation.</h4>
<p class="authors"><span class="author">Tripathi R</span>, <span class="author">Saini HK</span>, <span class="author">Rad R</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">van Dongen S</span> and <span class="author">Enright AJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Embryonic stem (ES) cells can be induced to differentiate into embryoid bodies (EBs) in a synchronised manner when plated at a fixed density in hanging drops. This differentiation procedure mimics post-implantation development in mouse embryos and also serves as the starting point of protocols used in differentiation of stem cells into various lineages. Currently, little is known about the potential influence of microRNAs (miRNAs) on mRNA expression patterns during EB formation. We have measured mRNA and miRNA expression in developing EBs plated in hanging drops until day 3, when discrete structural changes occur involving their differentiation into three germ layers. We observe significant alterations in mRNA and miRNA expression profiles during this early developmental time frame, in particular of genes involved in germ layer formation, stem cell pluripotency and nervous system development. Computational target prediction using Pictar, TargetScan and miRBase Targets reveals an enrichment of binding sites corresponding to differentially and highly expressed miRNAs in stem cell pluripotency genes and a neuroectodermal marker, Nes. We also find that members of let-7 family are significantly down-regulated at day 3 and the corresponding up-regulated genes are enriched in let-7 seed sequences. These results depict how miRNA expression changes may affect the expression of mRNAs involved in EB formation on a genome-wide scale. Understanding the regulatory effects of miRNAs during EB formation may enable more efficient derivation of different cell types in culture.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Gene expression patterns : GEP</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="number">5-6</span>;<span class="pagerange">334-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21440681">21440681</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gep.2011.03.004">10.1016/j.gep.2011.03.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dense genotyping identifies and localizes multiple common and rare variant association signals in celiac disease.</h4>
<p class="authors"><span class="author">Trynka G</span>, <span class="author">Hunt KA</span>, <span class="author">Bockett NA</span>, <span class="author">Romanos J</span>, <span class="author">Mistry V</span>, <span class="author">Szperl A</span>, <span class="author">Bakker SF</span>, <span class="author">Bardella MT</span>, <span class="author">Bhaw-Rosun L</span>, <span class="author">Castillejo G</span>, <span class="author">de la Concha EG</span>, <span class="author">de Almeida RC</span>, <span class="author">Dias KR</span>, <span class="author">van Diemen CC</span>, <span class="author">Dubois PC</span>, <span class="author">Duerr RH</span>, <span class="author">Edkins S</span>, <span class="author">Franke L</span>, <span class="author">Fransen K</span>, <span class="author">Gutierrez J</span>, <span class="author">Heap GA</span>, <span class="author">Hrdlickova B</span>, <span class="author">Hunt S</span>, <span class="author">Plaza Izurieta L</span>, <span class="author">Izzo V</span>, <span class="author">Joosten LA</span>, <span class="author">Langford C</span>, <span class="author">Mazzilli MC</span>, <span class="author">Mein CA</span>, <span class="author">Midah V</span>, <span class="author">Mitrovic M</span>, <span class="author">Mora B</span>, <span class="author">Morelli M</span>, <span class="author">Nutland S</span>, <span class="author">N&uacute;&ntilde;ez C</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Pearce K</span>, <span class="author">Platteel M</span>, <span class="author">Polanco I</span>, <span class="author">Potter S</span>, <span class="author">Ribes-Koninckx C</span>, <span class="author">Rica&ntilde;o-Ponce I</span>, <span class="author">Rich SS</span>, <span class="author">Rybak A</span>, <span class="author">Santiago JL</span>, <span class="author">Senapati S</span>, <span class="author">Sood A</span>, <span class="author">Szajewska H</span>, <span class="author">Troncone R</span>, <span class="author">Varad&eacute; J</span>, <span class="author">Wallace C</span>, <span class="author">Wolters VM</span>, <span class="author">Zhernakova A</span>, <span class="author">Spanish Consortium on the Genetics of Coeliac Disease (CEGEC)</span>, <span class="author">PreventCD Study Group</span>, <span class="author">Wellcome Trust Case Control Consortium (WTCCC)</span>, <span class="author">Thelma BK</span>, <span class="author">Cukrowska B</span>, <span class="author">Urcelay E</span>, <span class="author">Bilbao JR</span>, <span class="author">Mearin ML</span>, <span class="author">Barisani D</span>, <span class="author">Barrett JC</span>, <span class="author">Plagnol V</span>, <span class="author">Deloukas P</span>, <span class="author">Wijmenga C</span> and <span class="author">van Heel DA</span></p>
<p class="affiliation">Genetics Department, University Medical Center and University of Groningen, The Netherlands.</p><p class="abstract">Using variants from the 1000 Genomes Project pilot European CEU dataset and data from additional resequencing studies, we densely genotyped 183 non-HLA risk loci previously associated with immune-mediated diseases in 12,041 individuals with celiac disease (cases) and 12,228 controls. We identified 13 new celiac disease risk loci reaching genome-wide significance, bringing the number of known loci (including the HLA locus) to 40. We found multiple independent association signals at over one-third of these loci, a finding that is attributable to a combination of common, low-frequency and rare genetic variants. Compared to previously available data such as those from HapMap3, our dense genotyping in a large sample collection provided a higher resolution of the pattern of linkage disequilibrium and suggested localization of many signals to finer scale regions. In particular, 29 of the 54 fine-mapped signals seemed to be localized to single genes and, in some instances, to gene regulatory elements. Altogether, we define the complex genetic architecture of the risk regions of and refine the risk signals for celiac disease, providing the next step toward uncovering the causal mechanisms of the disease.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G1001158</span>, <span class="grant">G1001158(95979)</span>, <span class="grant">G1001799</span>; NCI NIH HHS: <span class="grant">1R01CA141743</span>; NIDDK NIH HHS: <span class="grant">U01-DK062418</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">084743</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">12</span>;<span class="pagerange">1193-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22057235">22057235</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3242065">3242065</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.998">10.1038/ng.998</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome watch: Honey, I shrunk the mimiviral genome.</h4>
<p class="authors"><span class="author">Tsai IJ</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This month&#39;s Genome Watch describes how the large size of the mimiviral genome is a result of the sympatric lifestyle of mimivirus in host amoebae.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">8</span>;<span class="pagerange">563</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21747393">21747393</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2623">10.1038/nrmicro2623</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Toward clinical proteomics on a next-generation sequencing platform.</h4>
<p class="authors"><span class="author">Turner DJ</span>, <span class="author">Tuytten R</span>, <span class="author">Janssen KP</span>, <span class="author">Lammertyn J</span>, <span class="author">Wuyts J</span>, <span class="author">Pollet J</span>, <span class="author">Eyckerman S</span>, <span class="author">Brown C</span> and <span class="author">Kas K</span></p><p class="abstract">We report the first next generation sequencing (NGS) application to identify and quantify proteins. Customization of protein specific aptamers enabled direct conversion of serum protein information into NGS read outs. The intrinsic ability of aptamer sequencing to highly multiplex protein detection and quantification, together with the prospect of DNA sequencing further evolving into a commodity technology, could constitute the core of a novel, universal diagnostics paradigm.</p>
<p class="publication"><span class="title">Analytical chemistry</span>&nbsp;<span class="year">2011</span>;<span class="volume">83</span>;<span class="number">3</span>;<span class="pagerange">666-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21142014" rel="external">21142014</a>; DOI: <a href="http://dx.doi.org/10.1021%2Fac102666n" rel="external">10.1021/ac102666n</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improved detection of nasopharyngeal cocolonization by multiple pneumococcal serotypes by use of latex agglutination or molecular serotyping by microarray.</h4>
<p class="authors"><span class="author">Turner P</span>, <span class="author">Hinds J</span>, <span class="author">Turner C</span>, <span class="author">Jankhot A</span>, <span class="author">Gould K</span>, <span class="author">Bentley SD</span>, <span class="author">Nosten F</span> and <span class="author">Goldblatt D</span></p>
<p class="affiliation">Shoklo Malaria Research Unit, Mae Sot, Thailand. <a href="mailto:%70%61%75%6C%74%40%74%72%6F%70%6D%65%64%72%65%73.%61%63">&#112;&#97;&#117;&#108;&#116;&#64;&#116;&#114;&#111;&#112;&#109;&#101;&#100;&#114;&#101;&#115;&#46;&#97;&#99;</a></p><p class="abstract">Identification of Streptococcus pneumoniae in the nasopharynx is critical for an understanding of transmission, estimates of vaccine efficacy, and possible replacement disease. Conventional nasopharyngeal swab (NPS) culture and serotyping (the WHO protocol) is likely to underestimate multiple-serotype carriage. We compared the WHO protocol with methods aimed at improving cocolonization detection. One hundred twenty-five NPSs from an infant pneumococcal-carriage study, containing &ge; 1 serotype by WHO culture, were recultured in duplicate. A sweep of colonies from one plate culture was serotyped by latex agglutination. DNA extracted from the second plate was analyzed by S. pneumoniae molecular-serotyping microarray. Multiple serotypes were detected in 11.2% of the swabs by WHO culture, 43.2% by sweep serotyping, and 48.8% by microarray. Sweep and microarray were more likely to detect multiple serotypes than WHO culture (P &lt; 0.0001). Cocolonization detection rates were similar between microarray and sweep, but the microarray identified the greatest number of serotypes. A common serogroup type was identified in 95.2% of swabs by all methods. WHO methodology significantly underestimates multiple-serotype carriage compared to these alternate methods. Sweep serotyping is cost-effective and field deployable but may fail to detect serotypes at low abundance, whereas microarray serotyping is more costly and technology dependent but may detect these additional minor carried serotypes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077166/Z/05</span>, <span class="grant">083735</span>, <span class="grant">086547</span></p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">49</span>;<span class="number">5</span>;<span class="pagerange">1784-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21411589" rel="external">21411589</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3122683" rel="external">3122683</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.00157-11" rel="external">10.1128/JCM.00157-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dissection of the genetics of Parkinson&#39;s disease identifies an additional association 5&#39; of SNCA and multiple associated haplotypes at 17q21.</h4>
<p class="authors"><span class="author">UK Parkinson&#39;s Disease Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Spencer CC</span>, <span class="author">Plagnol V</span>, <span class="author">Strange A</span>, <span class="author">Gardner M</span>, <span class="author">Paisan-Ruiz C</span>, <span class="author">Band G</span>, <span class="author">Barker RA</span>, <span class="author">Bellenguez C</span>, <span class="author">Bhatia K</span>, <span class="author">Blackburn H</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Brown MA</span>, <span class="author">Burn D</span>, <span class="author">Casas JP</span>, <span class="author">Chinnery PF</span>, <span class="author">Clarke CE</span>, <span class="author">Corvin A</span>, <span class="author">Craddock N</span>, <span class="author">Deloukas P</span>, <span class="author">Edkins S</span>, <span class="author">Evans J</span>, <span class="author">Freeman C</span>, <span class="author">Gray E</span>, <span class="author">Hardy J</span>, <span class="author">Hudson G</span>, <span class="author">Hunt S</span>, <span class="author">Jankowski J</span>, <span class="author">Langford C</span>, <span class="author">Lees AJ</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">McCarthy MI</span>, <span class="author">Morrison KE</span>, <span class="author">Palmer CN</span>, <span class="author">Pearson JP</span>, <span class="author">Peltonen L</span>, <span class="author">Pirinen M</span>, <span class="author">Plomin R</span>, <span class="author">Potter S</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Su Z</span>, <span class="author">Trembath RC</span>, <span class="author">Viswanathan AC</span>, <span class="author">Williams NW</span>, <span class="author">Morris HR</span>, <span class="author">Donnelly P</span> and <span class="author">Wood NW</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Oxford, UK.</p><p class="abstract">We performed a genome-wide association study (GWAS) in 1705 Parkinson&#39;s disease (PD) UK patients and 5175 UK controls, the largest sample size so far for a PD GWAS. Replication was attempted in an additional cohort of 1039 French PD cases and 1984 controls for the 27 regions showing the strongest evidence of association (P&lt; 10(-4)). We replicated published associations in the 4q22/SNCA and 17q21/MAPT chromosome regions (P&lt; 10(-10)) and found evidence for an additional independent association in 4q22/SNCA. A detailed analysis of the haplotype structure at 17q21 showed that there are three separate risk groups within this region. We found weak but consistent evidence of association for common variants located in three previously published associated regions (4p15/BST1, 4p16/GAK and 1q32/PARK16). We found no support for the previously reported SNP association in 12q12/LRRK2. We also found an association of the two SNPs in 4q22/SNCA with the age of onset of the disease.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0700943</span>; Parkinson&#39;s UK: <span class="grant">J-0804</span>, <span class="grant">K-0905</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">083948</span>, <span class="grant">084747</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">20</span>;<span class="number">2</span>;<span class="pagerange">345-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21044948" rel="external">21044948</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3005904" rel="external">3005904</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq469" rel="external">10.1093/hmg/ddq469</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dlg3 trafficking and apical tight junction formation is regulated by nedd4 and nedd4-2 e3&nbsp;ubiquitin ligases.</h4>
<p class="authors"><span class="author">Van Campenhout CA</span>, <span class="author">Eitelhuber A</span>, <span class="author">Gloeckner CJ</span>, <span class="author">Giallonardo P</span>, <span class="author">Gegg M</span>, <span class="author">Oller H</span>, <span class="author">Grant SG</span>, <span class="author">Krappmann D</span>, <span class="author">Ueffing M</span> and <span class="author">Lickert H</span></p>
<p class="affiliation">Institute of Stem Cell Research, Helmholtz Zentrum M&uuml;nchen, 85764 Neuherberg, Germany.</p><p class="abstract">The Drosophila Discs large (Dlg) scaffolding protein acts as a tumor suppressor regulating basolateral epithelial polarity and proliferation. In mammals, four Dlg homologs have been identified; however, their functions in cell polarity remain poorly understood. Here, we demonstrate that the X-linked mental retardation gene product Dlg3 contributes to apical-basal polarity and epithelial junction formation in mouse organizer tissues, as well as to planar cell polarity in the inner ear. We purified complexes associated with Dlg3 in polarized epithelial cells, including proteins regulating directed trafficking and tight junction formation. Remarkably, of the four Dlg family members, Dlg3 exerts a distinct function by recruiting the ubiquitin ligases Nedd4 and Nedd4-2 through its PPxY motifs. We found that these interactions are required for Dlg3 monoubiquitination, apical membrane recruitment, and tight junction consolidation. Our findings reveal an unexpected evolutionary diversification of the vertebrate Dlg family in basolateral epithelium formation.</p>
<p class="grants">Funded by: European Research Council: <span class="grant">242807</span></p>
<p class="publication"><span class="title">Developmental cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">479-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21920314">21920314</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.devcel.2011.08.003">10.1016/j.devcel.2011.08.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Partitioning core and satellite taxa from within cystic fibrosis lung bacterial communities.</h4>
<p class="authors"><span class="author">van der Gast CJ</span>, <span class="author">Walker AW</span>, <span class="author">Stressmann FA</span>, <span class="author">Rogers GB</span>, <span class="author">Scott P</span>, <span class="author">Daniels TW</span>, <span class="author">Carroll MP</span>, <span class="author">Parkhill J</span> and <span class="author">Bruce KD</span></p>
<p class="affiliation">NERC Centre for Ecology and Hydrology, Wallingford, UK. <a href="mailto:%63%6A%76%64%67%40%63%65%68.%61%63.%75%6B">&#99;&#106;&#118;&#100;&#103;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Cystic fibrosis (CF) patients suffer from chronic bacterial lung infections that lead to death in the majority of cases. The need to maintain lung function in these patients means that characterising these infections is vital. Increasingly, culture-independent analyses are expanding the number of bacterial species associated with CF respiratory samples; however, the potential significance of these species is not known. Here, we applied ecological statistical tools to such culture-independent data, in a novel manner, to partition taxa within the metacommunity into core and satellite species. Sputa and clinical data were obtained from 14 clinically stable adult CF patients. Fourteen rRNA gene libraries were constructed with 35 genera and 82 taxa, identified in 2139 bacterial clones. Shannon-Wiener and taxa-richness analyses confirmed no undersampling of bacterial diversity. By decomposing the distribution using the ratio of variance to the mean taxon abundance, we partitioned objectively the species abundance distribution into core and satellite species. The satellite group comprised 67 bacterial taxa from 33 genera and the core group, 15 taxa from 7 genera (including Pseudomonas (1 taxon), Streptococcus (2), Neisseria (2), Catonella (1), Porphyromonas (1), Prevotella (5) and Veillonella (3)], the last four being anaerobes). The core group was dominated by Pseudomonas aeruginosa. Other recognised CF pathogens were rare. Mantel and partial Mantel tests assessed which clinical factors influenced the composition observed. CF transmembrane conductance regulator genotype and antibiotic treatment correlated with all core taxa. Lung function correlated with richness. The clinical significance of these core and satellite species findings in the CF lung is discussed. GenBank accession numbers: FM995625&ndash;FM997761</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT076964</span></p>
<p class="publication"><span class="title">The ISME journal</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">780-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21151003">21151003</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3105771">3105771</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fismej.2010.175">10.1038/ismej.2010.175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Acute sensitivity of the oral mucosa to oncogenic K-ras.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Alcolea MP</span>, <span class="author">Jones PH</span>, <span class="author">Rust AG</span>, <span class="author">Arends MJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1HH, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Mouse models of cancer represent powerful tools for analysing the role of genetic alterations in carcinogenesis. Using a mouse model that allows tamoxifen-inducible somatic activation (by Cre-mediated recombination) of oncogenic K-ras(G12D) in a wide range of tissues, we observed hyperplasia of squamous epithelium located in moist or frequently abraded mucosa, with the most dramatic effects in the oral mucosa. This epithelium showed a sequence of squamous hyperplasia followed by squamous papilloma with dysplasia, in which some areas progressed to early invasive squamous cell carcinoma, within 14 days of widespread oncogenic K-ras activation. The marked proliferative response of the oral mucosa to K-ras(G12D) was most evident in the basal layers of the squamous epithelium of the outer lip with hair follicles and wet mucosal surface, with these cells staining positively for pAKT and cyclin D1, showing Ras/AKT pathway activation and increased proliferation with Ki-67 and EdU positivity. The stromal cells also showed gene activation by recombination and immunopositivity for pERK indicating K-Ras/ERK pathway activation, but without Ki-67 positivity or increase in stromal proliferation. The oral neoplasms showed changes in the expression pattern of cytokeratins (CK6 and CK13), similar to those observed in human oral tumours. Sporadic activation of the K-ras(G12D) allele (due to background spontaneous recombination in occasional cells) resulted in the development of benign oral squamous papillomas only showing a mild degree of dysplasia with no invasion. In summary, we show that oral mucosa is acutely sensitive to oncogenic K-ras, as widespread expression of activated K-ras in the murine oral mucosal squamous epithelium and underlying stroma can drive the oral squamous papilloma-carcinoma sequence.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">MC_U105370181</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of pathology</span>&nbsp;<span class="year">2011</span>;<span class="volume">224</span>;<span class="number">1</span>;<span class="pagerange">22-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21381032">21381032</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3627303">3627303</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpath.2853">10.1002/path.2853</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modeling the evolution of ETV6-RUNX1-induced B-cell precursor acute lymphoblastic leukemia in mice.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Giotopoulos G</span>, <span class="author">Rust AG</span>, <span class="author">Matheson LS</span>, <span class="author">van Delft FW</span>, <span class="author">Kong J</span>, <span class="author">Corcoran AE</span>, <span class="author">Greaves MF</span>, <span class="author">Mullighan CG</span>, <span class="author">Huntly BJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The t(12;21) translocation that generates the ETV6-RUNX1 (TEL-AML1) fusion gene, is the most common chromosomal rearrangement in childhood cancer and is exclusively associated with B-cell precursor acute lymphoblastic leukemia (BCP-ALL). The translocation arises in utero and is necessary but insufficient for the development of leukemia. Single-nucleotide polymorphism array analysis of ETV6-RUNX1 patient samples has identified multiple additional genetic alterations; however, the role of these lesions in leukemogenesis remains undetermined. Moreover, murine models of ETV6-RUNX1 ALL that faithfully recapitulate the human disease are lacking. To identify novel genes that cooperate with ETV6-RUNX1 in leukemogenesis, we generated a mouse model that uses the endogenous Etv6 locus to coexpress the Etv6-RUNX1 fusion and Sleeping Beauty transposase. An insertional mutagenesis screen was performed by intercrossing these mice with those carrying a Sleeping Beauty transposon array. In contrast to previous models, a substantial proportion (20%) of the offspring developed BCP-ALL. Isolation of the transposon insertion sites identified genes known to be associated with BCP-ALL, including Ebf1 and Epor, in addition to other novel candidates. This is the first mouse model of ETV6-RUNX1 to develop BCP-ALL and provides important insight into the cooperating genetic alterations in ETV6-RUNX1 leukemia.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Cancer Research UK: <span class="grant">13031</span>, <span class="grant">A12401</span>; Medical Research Council: <span class="grant">G116/187</span>; Wellcome Trust: <span class="grant">082356</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2011</span>;<span class="volume">118</span>;<span class="number">4</span>;<span class="pagerange">1041-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21628403">21628403</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3622520">3622520</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2011-02-338848">10.1182/blood-2011-02-338848</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The mouse genetics toolkit: revealing function and mechanism.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">White JK</span>, <span class="author">Adams DJ</span> and <span class="author">Logan DW</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Large-scale projects are providing rapid global access to a wealth of mouse genetic resources to help discover disease genes and to manipulate their function.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0800024</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">224</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21722353">21722353</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3218837">3218837</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2011-12-6-224">10.1186/gb-2011-12-6-224</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical role of Bcl11b in suppressor function of T regulatory cells and prevention of inflammatory bowel disease.</h4>
<p class="authors"><span class="author">Vanvalkenburgh J</span>, <span class="author">Albu DI</span>, <span class="author">Bapanpally C</span>, <span class="author">Casanova S</span>, <span class="author">Califano D</span>, <span class="author">Jones DM</span>, <span class="author">Ignatowicz L</span>, <span class="author">Kawamoto S</span>, <span class="author">Fagarasan S</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Liu P</span> and <span class="author">Avram D</span></p>
<p class="affiliation">Center for Cell Biology and Cancer Research, Albany Medical College, Albany, NY, USA.</p><p class="abstract">Dysregulated CD4(+) T cell responses and alterations in T regulatory cells (T(reg) cells) play a critical role in autoimmune diseases, including inflammatory bowel disease (IBD). The current study demonstrates that removal of Bcl11b at the double-positive stage of T cell development or only in T(reg) cells causes IBD because of proinflammatory cytokine-producing CD4(+) T cells infiltrating the colon. Provision of WT T(reg) cells prevented IBD, demonstrating that alterations in T(reg) cells are responsible for the disease. Furthermore, Bcl11b-deficient T(reg) cells had reduced suppressor activity with altered gene expression profiles, including reduced expression of the genes encoding Foxp3 and IL-10, and up-regulation of genes encoding proinflammatory cytokines. Additionally, the absence of Bcl11b altered the induction of Foxp3 expression and reduced the generation of induced T(reg) cells (iT(reg) cells) after Tgf-&beta; treatment of conventional CD4(+) T cells. Bcl11b bound to Foxp3 and IL-10 promoters, as well as to critical conserved noncoding sequences within the Foxp3 and IL-10 loci, and mutating the Bcl11b binding site in the Foxp3 promoter reduced expression of a luciferase reporter gene. These experiments demonstrate that Bcl11b is indispensable for T(reg) suppressor function and for maintenance of optimal Foxp3 and IL-10 gene expression, as well as for the induction of Foxp3 expression in conventional CD4(+) T cells in response to Tgf-&beta; and generation of iT(reg) cells.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01AI078273</span></p>
<p class="publication"><span class="title">The Journal of experimental medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">208</span>;<span class="number">10</span>;<span class="pagerange">2069-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21875956" rel="external">21875956</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3182057" rel="external">3182057</a>; DOI: <a href="http://dx.doi.org/10.1084%2Fjem.20102683" rel="external">10.1084/jem.20102683</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma.</h4>
<p class="authors"><span class="author">Varela I</span>, <span class="author">Tarpey P</span>, <span class="author">Raine K</span>, <span class="author">Huang D</span>, <span class="author">Ong CK</span>, <span class="author">Stephens P</span>, <span class="author">Davies H</span>, <span class="author">Jones D</span>, <span class="author">Lin ML</span>, <span class="author">Teague J</span>, <span class="author">Bignell G</span>, <span class="author">Butler A</span>, <span class="author">Cho J</span>, <span class="author">Dalgliesh GL</span>, <span class="author">Galappaththige D</span>, <span class="author">Greenman C</span>, <span class="author">Hardy C</span>, <span class="author">Jia M</span>, <span class="author">Latimer C</span>, <span class="author">Lau KW</span>, <span class="author">Marshall J</span>, <span class="author">McLaren S</span>, <span class="author">Menzies A</span>, <span class="author">Mudie L</span>, <span class="author">Stebbings L</span>, <span class="author">Largaespada DA</span>, <span class="author">Wessels LF</span>, <span class="author">Richard S</span>, <span class="author">Kahnoski RJ</span>, <span class="author">Anema J</span>, <span class="author">Tuveson DA</span>, <span class="author">Perez-Mancera PA</span>, <span class="author">Mustonen V</span>, <span class="author">Fischer A</span>, <span class="author">Adams DJ</span>, <span class="author">Rust A</span>, <span class="author">Chan-on W</span>, <span class="author">Subimerb C</span>, <span class="author">Dykema K</span>, <span class="author">Furge K</span>, <span class="author">Campbell PJ</span>, <span class="author">Teh BT</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">The genetics of renal cancer is dominated by inactivation of the VHL tumour suppressor gene in clear cell carcinoma (ccRCC), the commonest histological subtype. A recent large-scale screen of &sim;3,500 genes by PCR-based exon re-sequencing identified several new cancer genes in ccRCC including UTX (also known as KDM6A), JARID1C (also known as KDM5C) and SETD2 (ref. 2). These genes encode enzymes that demethylate (UTX, JARID1C) or methylate (SETD2) key lysine residues of histone H3. Modification of the methylation state of these lysine residues of histone H3 regulates chromatin structure and is implicated in transcriptional control. However, together these mutations are present in fewer than 15% of ccRCC, suggesting the existence of additional, currently unidentified cancer genes. Here, we have sequenced the protein coding exome in a series of primary ccRCC and report the identification of the SWI/SNF chromatin remodelling complex gene PBRM1 (ref. 4) as a second major ccRCC cancer gene, with truncating mutations in 41% (92/227) of cases. These data further elucidate the somatic genetic architecture of ccRCC and emphasize the marked contribution of aberrant chromatin biology.</p>
<p class="grants">Funded by: Cancer Research UK; NCI NIH HHS: <span class="grant">R01 CA113636</span>, <span class="grant">R01 CA134759</span>; Wellcome Trust: <span class="grant">077012</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">469</span>;<span class="number">7331</span>;<span class="pagerange">539-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21248752">21248752</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3030920">3030920</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09639">10.1038/nature09639</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutant nucleophosmin and cooperating pathways drive leukemia initiation and progression in mice.</h4>
<p class="authors"><span class="author">Vassiliou GS</span>, <span class="author">Cooper JL</span>, <span class="author">Rad R</span>, <span class="author">Li J</span>, <span class="author">Rice S</span>, <span class="author">Uren A</span>, <span class="author">Rad L</span>, <span class="author">Ellis P</span>, <span class="author">Andrews R</span>, <span class="author">Banerjee R</span>, <span class="author">Grove C</span>, <span class="author">Wang W</span>, <span class="author">Liu P</span>, <span class="author">Wright P</span>, <span class="author">Arends M</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Mouse Genomics Team, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%67%73%76%32%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#115;&#118;&#50;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Acute myeloid leukemia (AML) is a molecularly diverse malignancy with a poor prognosis whose largest subgroup is characterized by somatic mutations in NPM1, which encodes nucleophosmin. These mutations, termed NPM1c, result in cytoplasmic dislocation of nucleophosmin and are associated with distinctive transcriptional signatures, yet their role in leukemogenesis remains obscure. Here we report that activation of a humanized Npm1c knock-in allele in mouse hemopoietic stem cells causes Hox gene overexpression, enhanced self renewal and expanded myelopoiesis. One third of mice developed delayed-onset AML, suggesting a requirement for cooperating mutations. We identified such mutations using a Sleeping Beauty transposon, which caused rapid-onset AML in 80% of mice with Npm1c, associated with mutually exclusive integrations in Csf2, Flt3 or Rasgrp1 in 55 of 70 leukemias. We also identified recurrent integrations in known and newly discovered leukemia genes including Nf1, Bach2, Dleu2 and Nup98. Our results provide new pathogenetic insights and identify possible therapeutic targets in NPM1c+ AML.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A7273</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">5</span>;<span class="pagerange">470-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21441929">21441929</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3084174">3084174</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.796">10.1038/ng.796</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Y-encoded gene zfy2 acts to remove cells with unpaired chromosomes at the first meiotic metaphase in male mice.</h4>
<p class="authors"><span class="author">Vernet N</span>, <span class="author">Mahadevaiah SK</span>, <span class="author">Ojarikre OA</span>, <span class="author">Longepied G</span>, <span class="author">Prosser HM</span>, <span class="author">Bradley A</span>, <span class="author">Mitchell MJ</span> and <span class="author">Burgoyne PS</span></p>
<p class="affiliation">Division of Stem Cell Biology and Developmental Genetics, Medical Research Council National Institute for Medical Research, Mill Hill, London NW7 1AA, UK.</p><p class="abstract">During male but not female mammalian meiosis, there is efficient apoptotic elimination of cells with unpaired (univalent) chromosomes at the first meiotic metaphase (MI) [1]. Apoptotic elimination of MI spermatocytes is seen in response to the univalent X chromosome of XSxr(a)O male mice [2], in which the X chromosome carries Sxr(a) [3, 4], the Y-chromosome-derived sex-reversal factor that includes the testis determinant Sry. Sxr(b) is an Sxr(a)-derived variant in which a deletion has removed six Y short-arm genes and created a Zfy2/Zfy1 fusion gene spanning the deletion breakpoint [4, 5]. XSxr(b)O males have spermatogonial arrest that can be overcome by the re-addition of Eif2s3y from the deletion as a transgene; however, XSxr(b)OEif2s3y transgenic males do not show the expected elimination of MI spermatocytes in response to the univalent [6]. Here we show that these XSxr(b)OEif2s3y males have an impaired apoptotic response with completion of the first meiotic division, but there is no second meiotic division. We then show that Zfy2 (but not the closely related Zfy1) is sufficient to reinstate the apoptotic response to the X univalent. These findings provide further insight into the basis for the much lower transmission of chromosomal errors originating at the first meiotic division in men than in women [7].</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">U117532009</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2011</span>;<span class="volume">21</span>;<span class="number">9</span>;<span class="pagerange">787-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21530259" rel="external">21530259</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3176893" rel="external">3176893</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2011.03.057" rel="external">10.1016/j.cub.2011.03.057</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A barcode screen for epigenetic regulators reveals a role for the NuB4/HAT-B histone acetyltransferase complex in histone turnover.</h4>
<p class="authors"><span class="author">Verzijlbergen KF</span>, <span class="author">van Welsem T</span>, <span class="author">Sie D</span>, <span class="author">Lenstra TL</span>, <span class="author">Turner DJ</span>, <span class="author">Holstege FC</span>, <span class="author">Kerkhoven RM</span> and <span class="author">van Leeuwen F</span></p>
<p class="affiliation">Department of Gene Regulation, Netherlands Cancer Institute, Amsterdam, The Netherlands.</p><p class="abstract">Dynamic modification of histone proteins plays a key role in regulating gene expression. However, histones themselves can also be dynamic, which potentially affects the stability of histone modifications. To determine the molecular mechanisms of histone turnover, we developed a parallel screening method for epigenetic regulators by analyzing chromatin states on DNA barcodes. Histone turnover was quantified by employing a genetic pulse-chase technique called RITE, which was combined with chromatin immunoprecipitation and high-throughput sequencing. In this screen, the NuB4/HAT-B complex, containing the conserved type B histone acetyltransferase Hat1, was found to promote histone turnover. Unexpectedly, the three members of this complex could be functionally separated from each other as well as from the known interacting factor and histone chaperone Asf1. Thus, systematic and direct interrogation of chromatin structure on DNA barcodes can lead to the discovery of genes and pathways involved in chromatin modification and dynamics.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">e1002284</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21998594" rel="external">21998594</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3188528" rel="external">3188528</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002284" rel="external">10.1371/journal.pgen.1002284</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential phenotypic and genotypic characteristics of qnrS1-harboring plasmids carried by hospital and community commensal enterobacteria.</h4>
<p class="authors"><span class="author">Vien le TM</span>, <span class="author">Abuoun M</span>, <span class="author">Morrison V</span>, <span class="author">Thomson N</span>, <span class="author">Campbell JI</span>, <span class="author">Woodward MJ</span>, <span class="author">Van Vinh Chau N</span>, <span class="author">Farrar J</span>, <span class="author">Schultsz C</span> and <span class="author">Baker S</span></p>
<p class="affiliation">The Hospital for Tropical Diseases, Wellcome Trust Major Overseas Programme, Oxford University Clinical Research Unit, Ho Chi Minh City, Vietnam.</p><p class="abstract">The qnrS1 gene induces reduced susceptibility to fluoroquinolones in enterobacteria. We investigated the structure, antimicrobial susceptibility phenotype, and antimicrobial resistance gene characteristics of qnrS1 plasmids from hospitalized patients and community controls in southern Vietnam. We found that the antimicrobial susceptibilities, resistance gene characteristics, and plasmid structures of qnrS1 plasmids from the hospital differed from those from the community. Our data imply that the characteristics of the two plasmid groups are indicative of distinct selective pressures in the differing environments.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2011</span>;<span class="volume">55</span>;<span class="number">4</span>;<span class="pagerange">1798-802</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21282449" rel="external">21282449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3067134" rel="external">3067134</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.01200-10" rel="external">10.1128/AAC.01200-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies six new loci influencing pulse pressure and mean arterial pressure.</h4>
<p class="authors"><span class="author">Wain LV</span>, <span class="author">Verwoert GC</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Shi G</span>, <span class="author">Johnson T</span>, <span class="author">Johnson AD</span>, <span class="author">Bochud M</span>, <span class="author">Rice KM</span>, <span class="author">Henneman P</span>, <span class="author">Smith AV</span>, <span class="author">Ehret GB</span>, <span class="author">Amin N</span>, <span class="author">Larson MG</span>, <span class="author">Mooser V</span>, <span class="author">Hadley D</span>, <span class="author">D&ouml;rr M</span>, <span class="author">Bis JC</span>, <span class="author">Aspelund T</span>, <span class="author">Esko T</span>, <span class="author">Janssens AC</span>, <span class="author">Zhao JH</span>, <span class="author">Heath S</span>, <span class="author">Laan M</span>, <span class="author">Fu J</span>, <span class="author">Pistis G</span>, <span class="author">Luan J</span>, <span class="author">Arora P</span>, <span class="author">Lucas G</span>, <span class="author">Pirastu N</span>, <span class="author">Pichler I</span>, <span class="author">Jackson AU</span>, <span class="author">Webster RJ</span>, <span class="author">Zhang F</span>, <span class="author">Peden JF</span>, <span class="author">Schmidt H</span>, <span class="author">Tanaka T</span>, <span class="author">Campbell H</span>, <span class="author">Igl W</span>, <span class="author">Milaneschi Y</span>, <span class="author">Hottenga JJ</span>, <span class="author">Vitart V</span>, <span class="author">Chasman DI</span>, <span class="author">Trompet S</span>, <span class="author">Bragg-Gresham JL</span>, <span class="author">Alizadeh BZ</span>, <span class="author">Chambers JC</span>, <span class="author">Guo X</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">K&uuml;hnel B</span>, <span class="author">Lopez LM</span>, <span class="author">Pola&scaron;ek O</span>, <span class="author">Boban M</span>, <span class="author">Nelson CP</span>, <span class="author">Morrison AC</span>, <span class="author">Pihur V</span>, <span class="author">Ganesh SK</span>, <span class="author">Hofman A</span>, <span class="author">Kundu S</span>, <span class="author">Mattace-Raso FU</span>, <span class="author">Rivadeneira F</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hwang SJ</span>, <span class="author">Vasan RS</span>, <span class="author">Wang TJ</span>, <span class="author">Bergmann S</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Laitinen J</span>, <span class="author">Pouta A</span>, <span class="author">Zitting P</span>, <span class="author">McArdle WL</span>, <span class="author">Kroemer HK</span>, <span class="author">V&ouml;lker U</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Glazer NL</span>, <span class="author">Taylor KD</span>, <span class="author">Harris TB</span>, <span class="author">Alavere H</span>, <span class="author">Haller T</span>, <span class="author">Keis A</span>, <span class="author">Tammesoo ML</span>, <span class="author">Aulchenko Y</span>, <span class="author">Barroso I</span>, <span class="author">Khaw KT</span>, <span class="author">Galan P</span>, <span class="author">Hercberg S</span>, <span class="author">Lathrop M</span>, <span class="author">Eyheramendy S</span>, <span class="author">Org E</span>, <span class="author">S&otilde;ber S</span>, <span class="author">Lu X</span>, <span class="author">Nolte IM</span>, <span class="author">Penninx BW</span>, <span class="author">Corre T</span>, <span class="author">Masciullo C</span>, <span class="author">Sala C</span>, <span class="author">Groop L</span>, <span class="author">Voight BF</span>, <span class="author">Melander O</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Salomaa V</span>, <span class="author">d&#39;Adamo AP</span>, <span class="author">Fabretto A</span>, <span class="author">Faletra F</span>, <span class="author">Ulivi S</span>, <span class="author">Del Greco F</span>, <span class="author">Facheris M</span>, <span class="author">Collins FS</span>, <span class="author">Bergman RN</span>, <span class="author">Beilby JP</span>, <span class="author">Hung J</span>, <span class="author">Musk AW</span>, <span class="author">Mangino M</span>, <span class="author">Shin SY</span>, <span class="author">Soranzo N</span>, <span class="author">Watkins H</span>, <span class="author">Goel A</span>, <span class="author">Hamsten A</span>, <span class="author">Gider P</span>, <span class="author">Loitfelder M</span>, <span class="author">Zeginigg M</span>, <span class="author">Hernandez D</span>, <span class="author">Najjar SS</span>, <span class="author">Navarro P</span>, <span class="author">Wild SH</span>, <span class="author">Corsi AM</span>, <span class="author">Singleton A</span>, <span class="author">de Geus EJ</span>, <span class="author">Willemsen G</span>, <span class="author">Parker AN</span>, <span class="author">Rose LM</span>, <span class="author">Buckley B</span>, <span class="author">Stott D</span>, <span class="author">Orru M</span>, <span class="author">Uda M</span>, <span class="author">LifeLines Cohort Study</span>, <span class="author">van der Klauw MM</span>, <span class="author">Zhang W</span>, <span class="author">Li X</span>, <span class="author">Scott J</span>, <span class="author">Chen YD</span>, <span class="author">Burke GL</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Viikari J</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Meitinger T</span>, <span class="author">Davies G</span>, <span class="author">Starr JM</span>, <span class="author">Emilsson V</span>, <span class="author">Plump A</span>, <span class="author">Lindeman JH</span>, <span class="author">Hoen PA</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">EchoGen consortium</span>, <span class="author">Felix JF</span>, <span class="author">Clarke R</span>, <span class="author">Hopewell JC</span>, <span class="author">Ongen H</span>, <span class="author">Breteler M</span>, <span class="author">Debette S</span>, <span class="author">Destefano AL</span>, <span class="author">Fornage M</span>, <span class="author">AortaGen Consortium</span>, <span class="author">Mitchell GF</span>, <span class="author">CHARGE Consortium Heart Failure Working Group</span>, <span class="author">Smith NL</span>, <span class="author">KidneyGen consortium</span>, <span class="author">Holm H</span>, <span class="author">Stefansson K</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">CKDGen consortium</span>, <span class="author">Cardiogenics consortium</span>, <span class="author">CardioGram</span>, <span class="author">Samani NJ</span>, <span class="author">Preuss M</span>, <span class="author">Rudan I</span>, <span class="author">Hayward C</span>, <span class="author">Deary IJ</span>, <span class="author">Wichmann HE</span>, <span class="author">Raitakari OT</span>, <span class="author">Palmas W</span>, <span class="author">Kooner JS</span>, <span class="author">Stolk RP</span>, <span class="author">Jukema JW</span>, <span class="author">Wright AF</span>, <span class="author">Boomsma DI</span>, <span class="author">Bandinelli S</span>, <span class="author">Gyllensten UB</span>, <span class="author">Wilson JF</span>, <span class="author">Ferrucci L</span>, <span class="author">Schmidt R</span>, <span class="author">Farrall M</span>, <span class="author">Spector TD</span>, <span class="author">Palmer LJ</span>, <span class="author">Tuomilehto J</span>, <span class="author">Pfeufer A</span>, <span class="author">Gasparini P</span>, <span class="author">Siscovick D</span>, <span class="author">Altshuler D</span>, <span class="author">Loos RJ</span>, <span class="author">Toniolo D</span>, <span class="author">Snieder H</span>, <span class="author">Gieger C</span>, <span class="author">Meneton P</span>, <span class="author">Wareham NJ</span>, <span class="author">Oostra BA</span>, <span class="author">Metspalu A</span>, <span class="author">Launer L</span>, <span class="author">Rettig R</span>, <span class="author">Strachan DP</span>, <span class="author">Beckmann JS</span>, <span class="author">Witteman JC</span>, <span class="author">Erdmann J</span>, <span class="author">van Dijk KW</span>, <span class="author">Boerwinkle E</span>, <span class="author">Boehnke M</span>, <span class="author">Ridker PM</span>, <span class="author">Jarvelin MR</span>, <span class="author">Chakravarti A</span>, <span class="author">Abecasis GR</span>, <span class="author">Gudnason V</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Levy D</span>, <span class="author">Munroe PB</span>, <span class="author">Psaty BM</span>, <span class="author">Caulfield MJ</span>, <span class="author">Rao DC</span>, <span class="author">Tobin MD</span>, <span class="author">Elliott P</span> and <span class="author">van Duijn CM</span></p>
<p class="affiliation">Department of Health Sciences, University of Leicester, Leicester, UK.</p><p class="abstract">Numerous genetic loci have been associated with systolic blood pressure (SBP) and diastolic blood pressure (DBP) in Europeans. We now report genome-wide association studies of pulse pressure (PP) and mean arterial pressure (MAP). In discovery (N = 74,064) and follow-up studies (N = 48,607), we identified at genome-wide significance (P = 2.7 &times; 10(-8) to P = 2.3 &times; 10(-13)) four new PP loci (at 4q12 near CHIC2, 7q22.3 near PIK3CG, 8q24.12 in NOV and 11q24.3 near ADAMTS8), two new MAP loci (3p21.31 in MAP4 and 10q25.3 near ADRB1) and one locus associated with both of these traits (2q24.3 near FIGN) that has also recently been associated with SBP in east Asians. For three of the new PP loci, the estimated effect for SBP was opposite of that for DBP, in contrast to the majority of common SBP- and DBP-associated variants, which show concordant effects on both traits. These findings suggest new genetic pathways underlying blood pressure variation, some of which may differentially influence SBP and DBP.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F019394/1</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0902313</span>, <span class="grant">G1000143</span>, <span class="grant">G9521010</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_U137686857</span>; NHGRI NIH HHS: <span class="grant">Z01 HG000024-13</span>; NHLBI NIH HHS: <span class="grant">K23 HL080025</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC085079</span>, <span class="grant">N01 HC095159</span>, <span class="grant">R01 HL043851</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL105756</span>, <span class="grant">U10 HL054512</span>; NIA NIH HHS: <span class="grant">N01 AG012109</span>; NIMHD NIH HHS: <span class="grant">263 MD9164 13</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">10</span>;<span class="pagerange">1005-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21909110">21909110</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3445021">3445021</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.922">10.1038/ng.922</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome watch: Singled out.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">7</span>;<span class="pagerange">485</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21677683" rel="external">21677683</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2603" rel="external">10.1038/nrmicro2603</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dominant and diet-responsive groups of bacteria within the human colonic microbiota.</h4>
<p class="authors"><span class="author">Walker AW</span>, <span class="author">Ince J</span>, <span class="author">Duncan SH</span>, <span class="author">Webster LM</span>, <span class="author">Holtrop G</span>, <span class="author">Ze X</span>, <span class="author">Brown D</span>, <span class="author">Stares MD</span>, <span class="author">Scott P</span>, <span class="author">Bergerat A</span>, <span class="author">Louis P</span>, <span class="author">McIntosh F</span>, <span class="author">Johnstone AM</span>, <span class="author">Lobley GE</span>, <span class="author">Parkhill J</span> and <span class="author">Flint HJ</span></p>
<p class="affiliation">Pathogen Genomics, Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">The populations of dominant species within the human colonic microbiota can potentially be modified by dietary intake with consequences for health. Here we examined the influence of precisely controlled diets in 14 overweight men. Volunteers were provided successively with a control diet, diets high in resistant starch (RS) or non-starch polysaccharides (NSPs) and a reduced carbohydrate weight loss (WL) diet, over 10 weeks. Analysis of 16S rRNA sequences in stool samples of six volunteers detected 320 phylotypes (defined at &gt;98% identity) of which 26, including 19 cultured species, each accounted for &gt;1% of sequences. Although samples clustered more strongly by individual than by diet, time courses obtained by targeted qPCR revealed that &#39;blooms&#39; in specific bacterial groups occurred rapidly after a dietary change. These were rapidly reversed by the subsequent diet. Relatives of Ruminococcus bromii (R-ruminococci) increased in most volunteers on the RS diet, accounting for a mean of 17% of total bacteria compared with 3.8% on the NSP diet, whereas the uncultured Oscillibacter group increased on the RS and WL diets. Relatives of Eubacterium rectale increased on RS (to mean 10.1%) but decreased, along with Collinsella aerofaciens, on WL. Inter-individual variation was marked, however, with &gt;60% of RS remaining unfermented in two volunteers on the RS diet, compared to &lt;4% in the other 12 volunteers; these two individuals also showed low numbers of R-ruminococci (&lt;1%). Dietary non-digestible carbohydrate can produce marked changes in the gut microbiota, but these depend on the initial composition of an individual&#39;s gut microbiota.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span>, <span class="grant">WT 76964</span></p>
<p class="publication"><span class="title">The ISME journal</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">2</span>;<span class="pagerange">220-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20686513">20686513</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3105703">3105703</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fismej.2010.118">10.1038/ismej.2010.118</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput clone library analysis of the mucosa-associated microbiota reveals dysbiosis and differences between inflamed and non-inflamed regions of the intestine in inflammatory bowel disease.</h4>
<p class="authors"><span class="author">Walker AW</span>, <span class="author">Sanderson JD</span>, <span class="author">Churcher C</span>, <span class="author">Parkes GC</span>, <span class="author">Hudspith BN</span>, <span class="author">Rayment N</span>, <span class="author">Brostoff J</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">Petrovska L</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%61%77%36%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#119;&#54;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The gut microbiota is thought to play a key role in the development of the inflammatory bowel diseases Crohn&#39;s disease (CD) and ulcerative colitis (UC). Shifts in the composition of resident bacteria have been postulated to drive the chronic inflammation seen in both diseases (the &quot;dysbiosis&quot; hypothesis). We therefore specifically sought to compare the mucosa-associated microbiota from both inflamed and non-inflamed sites of the colon in CD and UC patients to that from non-IBD controls and to detect disease-specific profiles.</p><p class="abstract"><strong>Results</strong>: Paired mucosal biopsies of inflamed and non-inflamed intestinal tissue from 6 CD (n = 12) and 6 UC (n = 12) patients were compared to biopsies from 5 healthy controls (n = 5) by in-depth sequencing of over 10,000 near full-length bacterial 16S rRNA genes. The results indicate that mucosal microbial diversity is reduced in IBD, particularly in CD, and that the species composition is disturbed. Firmicutes were reduced in IBD samples and there were concurrent increases in Bacteroidetes, and in CD only, Enterobacteriaceae. There were also significant differences in microbial community structure between inflamed and non-inflamed mucosal sites. However, these differences varied greatly between individuals, meaning there was no obvious bacterial signature that was positively associated with the inflamed gut.</p><p class="abstract"><strong>Conclusions</strong>: These results may support the hypothesis that the overall dysbiosis observed in inflammatory bowel disease patients relative to non-IBD controls might to some extent be a result of the disturbed gut environment rather than the direct cause of disease. Nonetheless, the observed shifts in microbiota composition may be important factors in disease maintenance and severity.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT076964</span></p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">11</span>;<span class="pagerange">7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21219646">21219646</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3032643">3032643</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-11-7">10.1186/1471-2180-11-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mannosidase 2, alpha 1 deficiency is associated with ricin resistance in embryonic stem (ES) cells.</h4>
<p class="authors"><span class="author">Wang W</span>, <span class="author">Hale C</span>, <span class="author">Goulding D</span>, <span class="author">Haslam SM</span>, <span class="author">Tissot B</span>, <span class="author">Lindsay C</span>, <span class="author">Michell S</span>, <span class="author">Titball R</span>, <span class="author">Yu J</span>, <span class="author">Toribio AL</span>, <span class="author">Rossi R</span>, <span class="author">Dell A</span>, <span class="author">Bradley A</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">Host gene products required for mediating the action of toxins are potential targets for reversing or controlling their pathogenic impact following exposure. To identify such targets libraries of insertional gene-trap mutations generated with a PiggyBac transposon in Blm-deficient embryonic stem cells were exposed to the plant toxin, ricin. Resistant clones were isolated and genetically characterised and one was found to be a homozygous mutant of the mannosidase 2, alpha 1 (Man2&alpha;1) locus with a matching defect in the homologous allele. The causality of the molecular lesion was confirmed by removal of the transposon following expression of PB-transposase. Comparative glycomic and lectin binding analysis of the Man2&alpha;1 (-/-) ricin resistant cells revealed an increase in the levels of hybrid glycan structures and a reduction in terminal &beta;-galactose moieties, potential target receptors for ricin. Furthermore, na&iuml;ve ES cells treated with inhibitors of the N-linked glycosylation pathway at the mannosidase 2, alpha 1 step exhibited either full or partial resistance to ricin. Therefore, we conclusively identified mannosidase 2, alpha 1 deficiency to be associated with ricin resistance.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBF0083091</span>; Wellcome Trust: <span class="grant">076962</span>, <span class="grant">077187</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">e22993</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21886775" rel="external">21886775</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3160287" rel="external">3160287</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0022993" rel="external">10.1371/journal.pone.0022993</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid and efficient reprogramming of somatic cells to induced pluripotent stem cells by retinoic acid receptor gamma and liver receptor homolog 1.</h4>
<p class="authors"><span class="author">Wang W</span>, <span class="author">Yang J</span>, <span class="author">Liu H</span>, <span class="author">Lu D</span>, <span class="author">Chen X</span>, <span class="author">Zenonos Z</span>, <span class="author">Campos LS</span>, <span class="author">Rad R</span>, <span class="author">Guo G</span>, <span class="author">Zhang S</span>, <span class="author">Bradley A</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1HH, United Kingdom.</p><p class="abstract">Somatic cells can be reprogrammed to induced pluripotent stem cells (iPSCs) by expressing four transcription factors: Oct4, Sox2, Klf4, and c-Myc. Here we report that enhancing RA signaling by expressing RA receptors (RARs) or by RA agonists profoundly promoted reprogramming, but inhibiting it using a RAR-&alpha; dominant-negative form completely blocked it. Coexpressing Rarg (RAR-&gamma;) and Lrh-1 (liver receptor homologue 1; Nr5a2) with the four factors greatly accelerated reprogramming so that reprogramming of mouse embryonic fibroblast cells to ground-state iPSCs requires only 4 d induction of these six factors. The six-factor combination readily reprogrammed primary human neonatal and adult fibroblast cells to exogenous factor-independent iPSCs, which resembled ground-state mouse ES cells in growth properties, gene expression, and signaling dependency. Our findings demonstrate that signaling through RARs has critical roles in molecular reprogramming and that the synergistic interaction between Rarg and Lrh1 directs reprogramming toward ground-state pluripotency. The human iPSCs described here should facilitate functional analysis of the human genome.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700665</span>; Wellcome Trust: <span class="grant">077186/Z/05/Z</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">45</span>;<span class="pagerange">18283-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21990348">21990348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3215025">3215025</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1100893108">10.1073/pnas.1100893108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Trait variation in yeast is defined by population history.</h4>
<p class="authors"><span class="author">Warringer J</span>, <span class="author">Z&ouml;rg&ouml; E</span>, <span class="author">Cubillos FA</span>, <span class="author">Zia A</span>, <span class="author">Gjuvsland A</span>, <span class="author">Simpson JT</span>, <span class="author">Forsmark A</span>, <span class="author">Durbin R</span>, <span class="author">Omholt SW</span>, <span class="author">Louis EJ</span>, <span class="author">Liti G</span>, <span class="author">Moses A</span> and <span class="author">Blomberg A</span></p>
<p class="affiliation">Department of Cell and Molecular Biology, University of Gothenburg, Gothenburg, Sweden. <a href="mailto:%4A%6F%6E%61%73.%57%61%72%72%69%6E%67%65%72%40%63%6D%62.%67%75.%73%65">&#74;&#111;&#110;&#97;&#115;&#46;&#87;&#97;&#114;&#114;&#105;&#110;&#103;&#101;&#114;&#64;&#99;&#109;&#98;&#46;&#103;&#117;&#46;&#115;&#101;</a></p><p class="abstract">A fundamental goal in biology is to achieve a mechanistic understanding of how and to what extent ecological variation imposes selection for distinct traits and favors the fixation of specific genetic variants. Key to such an understanding is the detailed mapping of the natural genomic and phenomic space and a bridging of the gap that separates these worlds. Here we chart a high-resolution map of natural trait variation in one of the most important genetic model organisms, the budding yeast Saccharomyces cerevisiae, and its closest wild relatives and trace the genetic basis and timing of major phenotype changing events in its recent history. We show that natural trait variation in S. cerevisiae exceeds that of its relatives, despite limited genetic variation, and follows the population history rather than the source environment. In particular, the West African population is phenotypically unique, with an extreme abundance of low-performance alleles, notably a premature translational termination signal in GAL3 that cause inability to utilize galactose. Our observations suggest that many S. cerevisiae traits may be the consequence of genetic drift rather than selection, in line with the assumption that natural yeast lineages are remnants of recent population bottlenecks. Disconcertingly, the universal type strain S288C was found to be highly atypical, highlighting the danger of extrapolating gene-trait connections obtained in mosaic, lab-domesticated lineages to the species as a whole. Overall, this study represents a step towards an in-depth understanding of the causal relationship between co-variation in ecology, selection pressure, natural traits, molecular mechanism, and alleles in a key model organism.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e1002111</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21698134" rel="external">21698134</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3116910" rel="external">3116910</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002111" rel="external">10.1371/journal.pgen.1002111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cytokine signalling in mammary gland development.</h4>
<p class="authors"><span class="author">Watson CJ</span>, <span class="author">Oliver CH</span> and <span class="author">Khaled WT</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Tennis Court Road, Cambridge CB2 1QP, UK. <a href="mailto:%63%6A%77%35%33%40%63%61%6D.%61%63.%75%6B">&#99;&#106;&#119;&#53;&#51;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Mammary gland development occurs in three distinct stages during the lifetime of the female mammal: in embryonic, pubertal and reproductive life. At each of these developmental stages, different signalling molecules induce changes in both the epithelium and the surrounding stroma. However, it is during pregnancy that the most dramatic changes occur, resulting in a massive increase in the number of epithelial cells and in their function. Pregnancy initiates the development of a new epithelial lineage, the alveolar cells, which form the milk-producing lobuloalveolar structures. These cells become redundant at the end of lactation and are removed in an exquisitely controlled process of tissue remodelling coupled with extensive cell death. All of these events require not only steroid hormones but also sequential signalling by cytokines. A recent surprising discovery was that the signalling pathways and cytokines that regulate lineage determination in T helper cells are also involved in mammary gland development during pregnancy.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of reproductive immunology</span>&nbsp;<span class="year">2011</span>;<span class="volume">88</span>;<span class="number">2</span>;<span class="pagerange">124-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21255846" rel="external">21255846</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jri.2010.11.006" rel="external">10.1016/j.jri.2010.11.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genetic map of Xenopus tropicalis.</h4>
<p class="authors"><span class="author">Wells DE</span>, <span class="author">Gutierrez L</span>, <span class="author">Xu Z</span>, <span class="author">Krylov V</span>, <span class="author">Macha J</span>, <span class="author">Blankenburg KP</span>, <span class="author">Hitchens M</span>, <span class="author">Bellot LJ</span>, <span class="author">Spivey M</span>, <span class="author">Stemple DL</span>, <span class="author">Kowis A</span>, <span class="author">Ye Y</span>, <span class="author">Pasternak S</span>, <span class="author">Owen J</span>, <span class="author">Tran T</span>, <span class="author">Slavikova R</span>, <span class="author">Tumova L</span>, <span class="author">Tlapakova T</span>, <span class="author">Seifertova E</span>, <span class="author">Scherer SE</span> and <span class="author">Sater AK</span></p>
<p class="affiliation">Department of Biology and Biochemistry, University of Houston, 4800 Calhoun Rd., Houston TX 77204-5001, USA.</p><p class="abstract">We present a genetic map for Xenopus tropicalis, consisting of 2886 Simple Sequence Length Polymorphism (SSLP) markers. Using a bioinformatics-based strategy, we identified unique SSLPs within the X. tropicalis genome. Scaffolds from X. tropicalis genome assembly 2.0 (JGI) were scanned for Simple Sequence Repeats (SSRs); unique SSRs were then tested for amplification and polymorphisms using DNA from inbred Nigerian and Ivory Coast individuals. Thus identified, the SSLPs were genotyped against a mapping cross panel of DNA samples from 190 F2 individuals. Nearly 4000 SSLPs were genotyped, yielding a 2886-marker genetic map consisting of 10 major linkage groups between 73 and 132cM in length, and 4 smaller linkage groups between 7 and 40cM. The total effective size of the map is 1658cM, and the average intermarker distance for each linkage group ranged from 0.27 to 0.75cM. Fluorescence In Situ Hybridization (FISH) was carried out using probes for genes located on mapped scaffolds to assign linkage groups to chromosomes. Comparisons of this map with the X. tropicalis genome Assembly 4.1 (JGI) indicate that the map provides representation of a minimum of 66% of the X. tropicalis genome, incorporating 758 of the approximately 1300 scaffolds over 100,000bp. The genetic map and SSLP marker database constitute an essential resource for genetic and genomic analyses in X. tropicalis.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R01 HD046661</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2011</span>;<span class="volume">354</span>;<span class="number">1</span>;<span class="pagerange">1-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21458440" rel="external">21458440</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3098391" rel="external">3098391</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2011.03.022" rel="external">10.1016/j.ydbio.2011.03.022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association studies and type 2 diabetes.</h4>
<p class="authors"><span class="author">Wheeler E</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">In recent years, the search for genetic determinants of type 2 diabetes (T2D) has changed dramatically. Although linkage and small-scale candidate gene studies were highly successful in the identification of genes, which, when mutated, caused monogenic forms of T2D, they were largely unsuccessful when applied to the more common forms of the disease. To date, these approaches have only identified two loci (PPARG, KCNJ11) robustly implicated in T2D susceptibility. The ability to perform large-scale association analysis, including genome-wide association studies (GWAS) in many thousands of samples from different populations, and subsequently, the shift to form large international collaborations to perform meta-analyses across many studies has taken the number of independent loci showing genome-wide significant associations with T2D to 44. This number includes six loci identified initially through the analysis of quantitative glycaemic phenotypes, illustrating the usefulness of this approach both to identify new disease genes and gain insight into the mechanisms leading to disease. Combined, these loci still only account for &sim;10% of the observed familial clustering in Europeans, leaving much of the variance unexplained. In this review, we will describe what GWAS have taught us about the genetic basis of T2D and discuss possible next steps to uncover the remaining heritability.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077016/Z/05/Z</span></p>
<p class="publication"><span class="title">Briefings in functional genomics</span>&nbsp;<span class="year">2011</span>;<span class="volume">10</span>;<span class="number">2</span>;<span class="pagerange">52-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21436302">21436302</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbfgp%2Felr008">10.1093/bfgp/elr008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genomics of wild and laboratory zebrafish (Danio rerio).</h4>
<p class="authors"><span class="author">Whiteley AR</span>, <span class="author">Bhat A</span>, <span class="author">Martins EP</span>, <span class="author">Mayden RL</span>, <span class="author">Arunachalam M</span>, <span class="author">Uusi-Heikkil&auml; S</span>, <span class="author">Ahmed AT</span>, <span class="author">Shrestha J</span>, <span class="author">Clark M</span>, <span class="author">Stemple D</span> and <span class="author">Bernatchez L</span></p>
<p class="affiliation">Department of Environmental Conservation, University of Massachusetts, Amherst, MA 01003, USA Institut de Biologie Int&eacute;grative et des Syst&egrave;mes (IBIS), Universit&eacute; Laval, Qu&eacute;bec, QC, Canada G1V 0A6 Department of Biology, Indiana University, Bloomington IN, USA 47405 Department of Biology, Saint Louis University, St. Louis, MO 63103, USA Sri Paramakalyani Centre for Environmental Sciences, Manonmaniam Sundaranar University, Alwarkurichi, Tamil Nadu, India Department of Ecology and Biology of Fishes, Leibniz-Institute of Freshwater Ecology and Inland Fisheries, 12587 Berlin, Germany Department of Zoology, University of Dhaka, Dhaka, Bangladesh Department of Zoology, Tribhuvan University, Kathmandu, Nepal Wellcome Trust Sanger Institute, Cambridge, UK The Genome Analysis Centre, Norwich, UK.</p><p class="abstract">Understanding a wider range of genotype-phenotype associations can be achieved through ecological and evolutionary studies of traditional laboratory models. Here, we conducted the first large-scale geographic analysis of genetic variation within and among wild zebrafish (Danio rerio) populations occurring in Nepal, India, and Bangladesh, and we genetically compared wild populations to several commonly used lab strains. We examined genetic variation at 1832 polymorphic EST-based single nucleotide polymorphisms (SNPs) and the cytb mitochondrial gene in 13 wild populations and three lab strains. Natural populations were subdivided into three major mitochondrial DNA clades with an average among-clade sequence divergence of 5.8%. SNPs revealed five major evolutionarily and genetically distinct groups with an overall F(ST) of 0.170 (95% CI 0.105-0.254). These genetic groups corresponded to discrete geographic regions and appear to reflect isolation in refugia during past climate cycles. We detected 71 significantly divergent outlier loci (3.4%) and nine loci (0.5%) with significantly low F(ST) values. Valleys of reduced heterozygosity, consistent with selective sweeps, surrounded six of the 71 outliers (8.5%). The lab strains formed two additional groups that were genetically distinct from all wild populations. An additional subset of outlier loci was consistent with domestication selection within lab strains. Substantial genetic variation that exists in zebrafish as a whole is missing from lab strains that we analysed. A combination of laboratory and field studies that incorporates genetic variation from divergent wild populations along with the wealth of molecular information available for this model organism provides an opportunity to advance our understanding of genetic influences on phenotypic variation for a vertebrate species.</p>
<p class="publication"><span class="title">Molecular ecology</span>&nbsp;<span class="year">2011</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21923777">21923777</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-294X.2011.05272.x">10.1111/j.1365-294X.2011.05272.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Methods and strategies for gene structure curation in WormBase.</h4>
<p class="authors"><span class="author">Williams GW</span>, <span class="author">Davis PA</span>, <span class="author">Rogers AS</span>, <span class="author">Bieri T</span>, <span class="author">Ozersky P</span> and <span class="author">Spieth J</span></p>
<p class="affiliation">WormBase Group, The Wellcome Trust Sanger Institute, Hinxton, Cambs, UK. <a href="mailto:%67%77%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#119;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Caenorhabditis elegans genome sequence was published over a decade ago; this was the first published genome of a multi-cellular organism and now the WormBase project has had a decade of experience in curating this genome&#39;s sequence and gene structures. In one of its roles as a central repository for nematode biology, WormBase continues to refine the gene structure annotations using sequence similarity and other computational methods, as well as information from the literature- and community-submitted annotations. We describe the various methods of gene structure curation that have been tried by WormBase and the problems associated with each of them. We also describe the current strategy for gene structure curation, and introduce the WormBase &#39;curation tool&#39;, which integrates different data sources in order to identify new and correct gene structures. Database URL: http://www.wormbase.org/.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701197</span>; NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2011</span>;<span class="volume">2011</span>;<span class="pagerange">baq039</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21543339" rel="external">21543339</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3092607" rel="external">3092607</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbaq039" rel="external">10.1093/database/baq039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The characterisation of three types of genes that overlie copy number variable regions.</h4>
<p class="authors"><span class="author">Woodwark C</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom. <a href="mailto:%63%61%72%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#114;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Due to the increased accuracy of Copy Number Variable region (CNV) break point mapping, it is now possible to say with a reasonable degree of confidence whether a gene (i) falls entirely within a CNV; (ii) overlaps the CNV or (iii) actually contains the CNV. We classify these as type I, II and III CNV genes respectively.</p><p class="abstract">Here we show that although type I genes vary in copy number along with the CNV, most of these type I genes have the same expression levels as wild type copy numbers of the gene. These genes must, therefore, be under homeostatic dosage compensation control. Looking into possible mechanisms for the regulation of gene expression we found that type I genes have a significant paucity of genes regulated by miRNAs and are not significantly enriched for monoallelically expressed genes. Type III genes, on the other hand, have a significant excess of genes regulated by miRNAs and are enriched for genes that are monoallelically expressed.</p><p class="abstract"><strong>Significance</strong>: Many diseases and genomic disorders are associated with CNVs so a better understanding of the different ways genes are associated with normal CNVs will help focus on candidate genes in genome wide association studies.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">e14814</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21637334">21637334</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3102654">3102654</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0014814">10.1371/journal.pone.0014814</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Community Corner: Opening the Pandora&#39;s box of prenatal genetic testing.</h4>
<p class="authors"><span class="author">Wright C</span>, <span class="author">Quake SR</span>, <span class="author">Bianchi D</span> and <span class="author">Wald NJ</span></p>
<p class="affiliation">PHG Foundation, Cambridge, UK.</p>
<p class="publication"><span class="title">Nature medicine</span>&nbsp;<span class="year">2011</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">250-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21383717" rel="external">21383717</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnm0311-250" rel="external">10.1038/nm0311-250</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neurexins, neuroligins and LRRTMs: synaptic adhesion getting fishy.</h4>
<p class="authors"><span class="author">Wright GJ</span> and <span class="author">Washbourne P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">Recent studies have identified the leucine rich repeat protein LRRTM2 as a post-synaptic ligand of Neurexins. Neurexins also bind the post-synaptic adhesion molecules, Neuroligins. All three families of genes have been implicated in the etiologies of neurodevelopmental disorders, specifically autism spectrum disorders and schizophrenia. Does the binding promiscuity of Neurexins now suggest complex cooperativity or redundancy at the synapse? While recent studies in primary neuronal cultures and also systematic extracellular protein interaction screens suggest summative effects of these systems, we propose that studying these interactions in the developing zebrafish embryo or larvae may shed more light on their functions during synaptogenesis in vivo. These gene families have recently been extensively characterized in zebrafish, demonstrating high sequence conservation with the human genes. The simpler circuitry of the zebrafish, together with the characterization of the expression patterns down to single, identifiable neurons and the ability to knock-down or over-express multiple genes in a rapid way lend themselves to dissecting complex interaction pathways. Furthermore, the capability of performing high-throughput drug screens suggests that these small vertebrates may prove extremely useful in identifying pharmacological approaches to treating autism spectrum disorders.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">R01 NS065795-01A2</span>, <span class="grant">R01NS063400</span>, <span class="grant">R01NS065795</span>; Wellcome Trust: <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">Journal of neurochemistry</span>&nbsp;<span class="year">2011</span>;<span class="volume">117</span>;<span class="number">5</span>;<span class="pagerange">765-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21155806" rel="external">21155806</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3066302" rel="external">3066302</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1471-4159.2010.07141.x" rel="external">10.1111/j.1471-4159.2010.07141.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural analysis of papain-like NlpC/P60 superfamily enzymes with a circularly permuted topology reveals potential lipid binding sites.</h4>
<p class="authors"><span class="author">Xu Q</span>, <span class="author">Rawlings ND</span>, <span class="author">Chiu HJ</span>, <span class="author">Jaroszewski L</span>, <span class="author">Klock HE</span>, <span class="author">Knuth MW</span>, <span class="author">Miller MD</span>, <span class="author">Elsliger MA</span>, <span class="author">Deacon AM</span>, <span class="author">Godzik A</span>, <span class="author">Lesley SA</span> and <span class="author">Wilson IA</span></p>
<p class="affiliation">Joint Center for Structural Genomics, La Jolla, California, United States of America.</p><p class="abstract">NlpC/P60 superfamily papain-like enzymes play important roles in all kingdoms of life. Two members of this superfamily, LRAT-like and YaeF/YiiX-like families, were predicted to contain a catalytic domain that is circularly permuted such that the catalytic cysteine is located near the C-terminus, instead of at the N-terminus. These permuted enzymes are widespread in virus, pathogenic bacteria, and eukaryotes. We determined the crystal structure of a member of the YaeF/YiiX-like family from Bacillus cereus in complex with lysine. The structure, which adopts a ligand-induced, &quot;closed&quot; conformation, confirms the circular permutation of catalytic residues. A comparative analysis of other related protein structures within the NlpC/P60 superfamily is presented. Permutated NlpC/P60 enzymes contain a similar conserved core and arrangement of catalytic residues, including a Cys/His-containing triad and an additional conserved tyrosine. More surprisingly, permuted enzymes have a hydrophobic S1 binding pocket that is distinct from previously characterized enzymes in the family, indicative of novel substrate specificity. Further analysis of a structural homolog, YiiX (PDB 2if6) identified a fatty acid in the conserved hydrophobic pocket, thus providing additional insights into possible function of these novel enzymes.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">Y1-CO-1020</span>; NCRR NIH HHS: <span class="grant">P41RR001209</span>; NIGMS NIH HHS: <span class="grant">U54 GM074898</span>, <span class="grant">U54 GM094586</span>, <span class="grant">Y1-GM-1104</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">e22013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21799766" rel="external">21799766</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3142127" rel="external">3142127</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0022013" rel="external">10.1371/journal.pone.0022013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural and sequence analysis of imelysin-like proteins implicated in bacterial iron uptake.</h4>
<p class="authors"><span class="author">Xu Q</span>, <span class="author">Rawlings ND</span>, <span class="author">Farr CL</span>, <span class="author">Chiu HJ</span>, <span class="author">Grant JC</span>, <span class="author">Jaroszewski L</span>, <span class="author">Klock HE</span>, <span class="author">Knuth MW</span>, <span class="author">Miller MD</span>, <span class="author">Weekes D</span>, <span class="author">Elsliger MA</span>, <span class="author">Deacon AM</span>, <span class="author">Godzik A</span>, <span class="author">Lesley SA</span> and <span class="author">Wilson IA</span></p>
<p class="affiliation">Joint Center for Structural Genomics, La Jolla, California, United States of America.</p><p class="abstract">Imelysin-like proteins define a superfamily of bacterial proteins that are likely involved in iron uptake. Members of this superfamily were previously thought to be peptidases and were included in the MEROPS family M75. We determined the first crystal structures of two remotely related, imelysin-like proteins. The Psychrobacter arcticus structure was determined at 2.15 &Aring; resolution and contains the canonical imelysin fold, while higher resolution structures from the gut bacteria Bacteroides ovatus, in two crystal forms (at 1.25 &Aring; and 1.44 &Aring; resolution), have a circularly permuted topology. Both structures are highly similar to each other despite low sequence similarity and circular permutation. The all-helical structure can be divided into two similar four-helix bundle domains. The overall structure and the GxHxxE motif region differ from known HxxE metallopeptidases, suggesting that imelysin-like proteins are not peptidases. A putative functional site is located at the domain interface. We have now organized the known homologous proteins into a superfamily, which can be separated into four families. These families share a similar functional site, but each has family-specific structural and sequence features. These results indicate that imelysin-like proteins have evolved from a common ancestor, and likely have a conserved function.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54 GM074898</span>, <span class="grant">U54 GM094586</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">e21875</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21799754" rel="external">21799754</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3143127" rel="external">3143127</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0021875" rel="external">10.1371/journal.pone.0021875</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An Exceptional Gene: Evolution of the TSPY Gene Family in Humans and Other Great Apes.</h4>
<p class="authors"><span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs. CB10 1SA, UK. <a href="mailto:%79%6C%78%40%73%61%6E%67%65%72.%61%63.%75%6B">&#121;&#108;&#120;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">The TSPY gene stands out from all other human protein-coding genes because of its high copy number and tandemly-repeated organization. Here, we review its evolutionary history in great apes in order to assess whether these unusual properties are more likely to result from a relaxation of constraint or an unusual functional role. Detailed comparisons with chimpanzee are possible because a finished sequence of the chimpanzee Y chromosome is available, together with more limited data from other apes. These comparisons suggest that the human-chimpanzee ancestral Y chromosome carried a tandem array of TSPY genes which expanded on the human lineage while undergoing multiple duplication events followed by pseudogene formation on the chimpanzee lineage. The protein coding region is the most highly conserved of the multi-copy Y genes in human-chimpanzee comparisons, and the analysis of the dN/dS ratio indicates that TSPY is evolutionarily highly constrained, but may have experienced positive selection after the human-chimpanzee split. We therefore conclude that the exceptionally high copy number in humans is most likely due to a human-specific but unknown functional role, possibly involving rapid production of a large amount of TSPY protein at some stage during&nbsp;spermatogenesis.</p>
<p class="publication"><span class="title">Genes</span>&nbsp;<span class="year">2011</span>;<span class="volume">2</span>;<span class="number">1</span>;<span class="pagerange">36-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24710137">24710137</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3924835">3924835</a>; DOI: <a href="http://dx.doi.org/10.3390%2Fgenes2010036">10.3390/genes2010036</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Response to the comment on &quot;The hare and the tortoise: One small step for four SNPs, one giant leap for SNP-kind&quot;.</h4>
<p class="authors"><span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs. CB10 1SA, UK.</p><p class="abstract">The possibility of introducing new sequencing technologies into forensic genetics raises questions that go beyond the choice between SNPs and STRs as the preferred genetic markers. We suggest that many of the novel methodological and technical issues could be incorporated into the likelihood ratio frameworks currently used by forensic scientists. However, changes to ethical and legal structures may be needed before the new information could be used.</p>
<p class="publication"><span class="title">Forensic science international. Genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">361-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20452849" rel="external">20452849</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.fsigen.2010.04.003" rel="external">10.1016/j.fsigen.2010.04.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An exceptional gene: Evolution of the TSPY gene family in humans and other great apes</h4>
<p class="authors"><span class="author">XUE Y</span>, <span class="author">TYLER-SMITH C</span></p>
<p class="publication"><span class="title">Genes.</span> <span class="year">2011</span>;<span class="volume">2</span>;<span class="pagerange">36-47</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence-based characterization of structural variation in the mouse genome.</h4>
<p class="authors"><span class="author">Yalcin B</span>, <span class="author">Wong K</span>, <span class="author">Agam A</span>, <span class="author">Goodson M</span>, <span class="author">Keane TM</span>, <span class="author">Gan X</span>, <span class="author">Nell&aring;ker C</span>, <span class="author">Goodstadt L</span>, <span class="author">Nicod J</span>, <span class="author">Bhomra A</span>, <span class="author">Hernandez-Pliego P</span>, <span class="author">Whitley H</span>, <span class="author">Cleak J</span>, <span class="author">Dutton R</span>, <span class="author">Janowitz D</span>, <span class="author">Mott R</span>, <span class="author">Adams DJ</span> and <span class="author">Flint J</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7BN, UK.</p><p class="abstract">Structural variation is widespread in mammalian genomes and is an important cause of disease, but just how abundant and important structural variants (SVs) are in shaping phenotypic variation remains unclear. Without knowing how many SVs there are, and how they arise, it is difficult to discover what they do. Combining experimental with automated analyses, we identified 711,920 SVs at 281,243 sites in the genomes of thirteen classical and four wild-derived inbred mouse strains. The majority of SVs are less than 1&thinsp;kilobase in size and 98% are deletions or insertions. The breakpoints of 160,000 SVs were mapped to base pair resolution, allowing us to infer that insertion of retrotransposons causes more than half of SVs. Yet, despite their prevalence, SVs are less likely than other sequence variants to cause gene expression or quantitative phenotypic variation. We identified 24 SVs that disrupt coding exons, acting as rare variants of large effect on gene function. One-third of the genes so affected have immunological functions.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0800024</span>, <span class="grant">MC_U137761446</span>; Wellcome Trust: <span class="grant">079912</span>, <span class="grant">082356</span>, <span class="grant">090532</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">477</span>;<span class="number">7364</span>;<span class="pagerange">326-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21921916">21921916</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428933">3428933</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10432">10.1038/nature10432</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phase I trial of a selective c-MET inhibitor ARQ 197 incorporating proof of mechanism pharmacodynamic studies.</h4>
<p class="authors"><span class="author">Yap TA</span>, <span class="author">Olmos D</span>, <span class="author">Brunetto AT</span>, <span class="author">Tunariu N</span>, <span class="author">Barriuso J</span>, <span class="author">Riisnaes R</span>, <span class="author">Pope L</span>, <span class="author">Clark J</span>, <span class="author">Futreal A</span>, <span class="author">Germuska M</span>, <span class="author">Collins D</span>, <span class="author">deSouza NM</span>, <span class="author">Leach MO</span>, <span class="author">Savage RE</span>, <span class="author">Waghorne C</span>, <span class="author">Chai F</span>, <span class="author">Garmey E</span>, <span class="author">Schwartz B</span>, <span class="author">Kaye SB</span> and <span class="author">de Bono JS</span></p>
<p class="affiliation">Royal Marsden National Health Service Foundation Trust, The Institute of Cancer Research, Sutton, Surrey, UK.</p><p class="abstract"><strong>Purpose</strong>: The hepatocyte growth factor/c-MET axis is implicated in tumor cell proliferation, survival, and angiogenesis. ARQ 197 is an oral, selective, non-adenosine triphosphate competitive c-MET inhibitor. A phase I trial of ARQ 197 was conducted to assess safety, tolerability, and target inhibition, including intratumoral c-MET signaling, apoptosis, and angiogenesis.</p><p class="abstract">Patients with solid tumors amenable to pharmacokinetic and pharmacodynamic studies using serial biopsies, dynamic contrast-enhanced magnetic resonance imaging (DCE-MRI), and circulating endothelial cell (CEC) and circulating tumor cell (CTC) enumeration were enrolled.</p><p class="abstract"><strong>Results</strong>: Fifty-one patients received ARQ 197 at 100 to 400 mg twice per day. ARQ 197 was well tolerated, with the most common toxicities being grade 1 to 2 fatigue, nausea, and vomiting. Dose-limiting toxicities included grade 3 fatigue (200 mg twice per day; n = 1); grade 3 mucositis, palmar-plantar erythrodysesthesia, and hypokalemia (400 mg twice per day; n = 1); and grade 3 to 4 febrile neutropenia (400 mg twice per day, n = 2; 360 mg twice per day, n = 1). The recommended phase II dose was 360 mg twice per day. ARQ 197 systemic exposure was dose dependent and supported twice per day oral dosing. ARQ 197 decreased phosphorylated c-MET, total c-MET, and phosphorylated focal adhesion kinase and increased terminal deoxynucleotidyl transferase-mediated deoxyuridine triphosphate-biotin nick-end labeling (TUNEL) staining in tumor biopsies (n = 15). CECs decreased in 25 (58.1%) of 43 patients, but no significant changes in DCE-MRI parameters were observed after ARQ 197 treatment. Of 15 patients with detectable CTCs, eight (53.3%) had &ge; 30% decline in CTCs after treatment. Stable disease, as defined by Response Evaluation Criteria in Solid Tumors (RECIST), &ge; 4 months was observed in 14 patients, with minor regressions in gastric and Merkel cell cancers.</p><p class="abstract"><strong>Conclusion</strong>: ARQ 197 safely inhibited intratumoral c-MET signaling. Further clinical evaluation focusing on combination approaches, including an erlotinib combination in non-small-cell lung cancer, is ongoing.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10334</span>, <span class="grant">C1060/A10334</span>; Department of Health; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2011</span>;<span class="volume">29</span>;<span class="number">10</span>;<span class="pagerange">1271-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21383285">21383285</a>; DOI: <a href="http://dx.doi.org/10.1200%2FJCO.2010.31.0367">10.1200/JCO.2010.31.0367</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variant near the endothelin receptor type A (EDNRA) gene is associated with intracranial aneurysm risk.</h4>
<p class="authors"><span class="author">Yasuno K</span>, <span class="author">Bak&#x131;rc&#x131;o&#x11F;lu M</span>, <span class="author">Low SK</span>, <span class="author">Bilg&uuml;var K</span>, <span class="author">Ga&aacute;l E</span>, <span class="author">Ruigrok YM</span>, <span class="author">Niemel&auml; M</span>, <span class="author">Hata A</span>, <span class="author">Bijlenga P</span>, <span class="author">Kasuya H</span>, <span class="author">J&auml;&auml;skel&auml;inen JE</span>, <span class="author">Krex D</span>, <span class="author">Auburger G</span>, <span class="author">Simon M</span>, <span class="author">Krischek B</span>, <span class="author">Ozturk AK</span>, <span class="author">Mane S</span>, <span class="author">Rinkel GJ</span>, <span class="author">Steinmetz H</span>, <span class="author">Hernesniemi J</span>, <span class="author">Schaller K</span>, <span class="author">Zembutsu H</span>, <span class="author">Inoue I</span>, <span class="author">Palotie A</span>, <span class="author">Cambien F</span>, <span class="author">Nakamura Y</span>, <span class="author">Lifton RP</span> and <span class="author">G&uuml;nel M</span></p>
<p class="affiliation">Department of Neurosurgery, Yale University School of Medicine, New Haven, CT 06510, USA.</p><p class="abstract">The pathogenesis of intracranial aneurysm (IA) formation and rupture is complex, with significant contribution from genetic factors. We previously reported genome-wide association studies based on European discovery and Japanese replication cohorts of 5,891 cases and 14,181 controls that identified five disease-related loci. These studies were based on testing replication of genomic regions that contained SNPs with posterior probability of association (PPA) greater than 0.5 in the discovery cohort. To identify additional IA risk loci, we pursued 14 loci with PPAs in the discovery cohort between 0.1 and 0.5. Twenty-five SNPs from these loci were genotyped using two independent Japanese cohorts, and the results from discovery and replication cohorts were combined by meta-analysis. The results demonstrated significant association of IA with rs6841581 on chromosome 4q31.23, immediately 5&#39; of the endothelin receptor type A with P = 2.2 &times; 10(-8) [odds ratio (OR) = 1.22, PPA = 0.986]. We also observed substantially increased evidence of association for two other regions on chromosomes 12q22 (OR = 1.16, P = 1.1 &times; 10(-7), PPA = 0.934) and 20p12.1 (OR = 1.20, P = 6.9 &times; 10(-7), PPA = 0.728). Although endothelin signaling has been hypothesized to play a role in various cardiovascular disorders for over two decades, our results are unique in providing genetic evidence for a significant association with IA and suggest that manipulation of the endothelin pathway may have important implications for the prevention and treatment of IA.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NINDS NIH HHS: <span class="grant">R01NS057756</span>, <span class="grant">R01NS067023</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">49</span>;<span class="pagerange">19707-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22106312" rel="external">22106312</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3241810" rel="external">3241810</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1117137108" rel="external">10.1073/pnas.1117137108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Medicago genome provides insight into the evolution of rhizobial symbioses.</h4>
<p class="authors"><span class="author">Young ND</span>, <span class="author">Debell&eacute; F</span>, <span class="author">Oldroyd GE</span>, <span class="author">Geurts R</span>, <span class="author">Cannon SB</span>, <span class="author">Udvardi MK</span>, <span class="author">Benedito VA</span>, <span class="author">Mayer KF</span>, <span class="author">Gouzy J</span>, <span class="author">Schoof H</span>, <span class="author">Van de Peer Y</span>, <span class="author">Proost S</span>, <span class="author">Cook DR</span>, <span class="author">Meyers BC</span>, <span class="author">Spannagl M</span>, <span class="author">Cheung F</span>, <span class="author">De Mita S</span>, <span class="author">Krishnakumar V</span>, <span class="author">Gundlach H</span>, <span class="author">Zhou S</span>, <span class="author">Mudge J</span>, <span class="author">Bharti AK</span>, <span class="author">Murray JD</span>, <span class="author">Naoumkina MA</span>, <span class="author">Rosen B</span>, <span class="author">Silverstein KA</span>, <span class="author">Tang H</span>, <span class="author">Rombauts S</span>, <span class="author">Zhao PX</span>, <span class="author">Zhou P</span>, <span class="author">Barbe V</span>, <span class="author">Bardou P</span>, <span class="author">Bechner M</span>, <span class="author">Bellec A</span>, <span class="author">Berger A</span>, <span class="author">Berg&egrave;s H</span>, <span class="author">Bidwell S</span>, <span class="author">Bisseling T</span>, <span class="author">Choisne N</span>, <span class="author">Couloux A</span>, <span class="author">Denny R</span>, <span class="author">Deshpande S</span>, <span class="author">Dai X</span>, <span class="author">Doyle JJ</span>, <span class="author">Dudez AM</span>, <span class="author">Farmer AD</span>, <span class="author">Fouteau S</span>, <span class="author">Franken C</span>, <span class="author">Gibelin C</span>, <span class="author">Gish J</span>, <span class="author">Goldstein S</span>, <span class="author">Gonz&aacute;lez AJ</span>, <span class="author">Green PJ</span>, <span class="author">Hallab A</span>, <span class="author">Hartog M</span>, <span class="author">Hua A</span>, <span class="author">Humphray SJ</span>, <span class="author">Jeong DH</span>, <span class="author">Jing Y</span>, <span class="author">J&ouml;cker A</span>, <span class="author">Kenton SM</span>, <span class="author">Kim DJ</span>, <span class="author">Klee K</span>, <span class="author">Lai H</span>, <span class="author">Lang C</span>, <span class="author">Lin S</span>, <span class="author">Macmil SL</span>, <span class="author">Magdelenat G</span>, <span class="author">Matthews L</span>, <span class="author">McCorrison J</span>, <span class="author">Monaghan EL</span>, <span class="author">Mun JH</span>, <span class="author">Najar FZ</span>, <span class="author">Nicholson C</span>, <span class="author">Noirot C</span>, <span class="author">O&#39;Bleness M</span>, <span class="author">Paule CR</span>, <span class="author">Poulain J</span>, <span class="author">Prion F</span>, <span class="author">Qin B</span>, <span class="author">Qu C</span>, <span class="author">Retzel EF</span>, <span class="author">Riddle C</span>, <span class="author">Sallet E</span>, <span class="author">Samain S</span>, <span class="author">Samson N</span>, <span class="author">Sanders I</span>, <span class="author">Saurat O</span>, <span class="author">Scarpelli C</span>, <span class="author">Schiex T</span>, <span class="author">Segurens B</span>, <span class="author">Severin AJ</span>, <span class="author">Sherrier DJ</span>, <span class="author">Shi R</span>, <span class="author">Sims S</span>, <span class="author">Singer SR</span>, <span class="author">Sinharoy S</span>, <span class="author">Sterck L</span>, <span class="author">Viollet A</span>, <span class="author">Wang BB</span>, <span class="author">Wang K</span>, <span class="author">Wang M</span>, <span class="author">Wang X</span>, <span class="author">Warfsmann J</span>, <span class="author">Weissenbach J</span>, <span class="author">White DD</span>, <span class="author">White JD</span>, <span class="author">Wiley GB</span>, <span class="author">Wincker P</span>, <span class="author">Xing Y</span>, <span class="author">Yang L</span>, <span class="author">Yao Z</span>, <span class="author">Ying F</span>, <span class="author">Zhai J</span>, <span class="author">Zhou L</span>, <span class="author">Zuber A</span>, <span class="author">D&eacute;nari&eacute; J</span>, <span class="author">Dixon RA</span>, <span class="author">May GD</span>, <span class="author">Schwartz DC</span>, <span class="author">Rogers J</span>, <span class="author">Qu&eacute;tier F</span>, <span class="author">Town CD</span> and <span class="author">Roe BA</span></p>
<p class="affiliation">Department of Plant Pathology, University of Minnesota, St Paul, Minnesota 55108, USA. <a href="mailto:%6E%65%76%69%6E%79%40%75%6D%6E.%65%64%75">&#110;&#101;&#118;&#105;&#110;&#121;&#64;&#117;&#109;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Legumes (Fabaceae or Leguminosae) are unique among cultivated plants for their ability to carry out endosymbiotic nitrogen fixation with rhizobial bacteria, a process that takes place in a specialized structure known as the nodule. Legumes belong to one of the two main groups of eurosids, the Fabidae, which includes most species capable of endosymbiotic nitrogen fixation. Legumes comprise several evolutionary lineages derived from a common ancestor 60 million years ago (Myr ago). Papilionoids are the largest clade, dating nearly to the origin of legumes and containing most cultivated species. Medicago truncatula is a long-established model for the study of legume biology. Here we describe the draft sequence of the M. truncatula euchromatin based on a recently completed BAC assembly supplemented with Illumina shotgun sequence, together capturing &sim;94% of all M. truncatula genes. A whole-genome duplication (WGD) approximately 58 Myr ago had a major role in shaping the M. truncatula genome and thereby contributed to the evolution of endosymbiotic nitrogen fixation. Subsequent to the WGD, the M. truncatula genome experienced higher levels of rearrangement than two other sequenced legumes, Glycine max and Lotus japonicus. M. truncatula is a close relative of alfalfa (Medicago sativa), a widely cultivated crop with limited genomics tools and complex autotetraploid genetics. As such, the M. truncatula genome sequence provides significant opportunities to expand alfalfa&#39;s genomic toolbox.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/B/11524</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">480</span>;<span class="number">7378</span>;<span class="pagerange">520-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22089132" rel="external">22089132</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3272368" rel="external">3272368</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10625" rel="external">10.1038/nature10625</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeted gene correction of &alpha;1-antitrypsin deficiency in induced pluripotent stem cells.</h4>
<p class="authors"><span class="author">Yusa K</span>, <span class="author">Rashid ST</span>, <span class="author">Strick-Marchand H</span>, <span class="author">Varela I</span>, <span class="author">Liu PQ</span>, <span class="author">Paschon DE</span>, <span class="author">Miranda E</span>, <span class="author">Ord&oacute;&ntilde;ez A</span>, <span class="author">Hannan NR</span>, <span class="author">Rouhani FJ</span>, <span class="author">Darche S</span>, <span class="author">Alexander G</span>, <span class="author">Marciniak SJ</span>, <span class="author">Fusaki N</span>, <span class="author">Hasegawa M</span>, <span class="author">Holmes MC</span>, <span class="author">Di Santo JP</span>, <span class="author">Lomas DA</span>, <span class="author">Bradley A</span> and <span class="author">Vallier L</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Human induced pluripotent stem cells (iPSCs) represent a unique opportunity for regenerative medicine because they offer the prospect of generating unlimited quantities of cells for autologous transplantation, with potential application in treatments for a broad range of disorders. However, the use of human iPSCs in the context of genetically inherited human disease will require the correction of disease-causing mutations in a manner that is fully compatible with clinical applications. The methods currently available, such as homologous recombination, lack the necessary efficiency and also leave residual sequences in the targeted genome. Therefore, the development of new approaches to edit the mammalian genome is a prerequisite to delivering the clinical promise of human iPSCs. Here we show that a combination of zinc finger nucleases (ZFNs) and piggyBac technology in human iPSCs can achieve biallelic correction of a point mutation (Glu342Lys) in the &alpha;(1)-antitrypsin (A1AT, also known as SERPINA1) gene that is responsible for &alpha;(1)-antitrypsin deficiency. Genetic correction of human iPSCs restored the structure and function of A1AT in subsequently derived liver cells in vitro and in vivo. This approach is significantly more efficient than any other gene-targeting technology that is currently available and crucially prevents contamination of the host genome with residual non-human sequences. Our results provide the first proof of principle, to our knowledge, for the potential of combining human iPSCs with genetic correction to generate clinically relevant cells for autologous cell-based therapies.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0601840</span>, <span class="grant">G0701448</span>, <span class="grant">G0800784</span>, <span class="grant">G0901786</span>, <span class="grant">G1000847</span>; Wellcome Trust: <span class="grant">077187</span>, <span class="grant">WT077187</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2011</span>;<span class="volume">478</span>;<span class="number">7369</span>;<span class="pagerange">391-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21993621">21993621</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3198846">3198846</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10424">10.1038/nature10424</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A hyperactive piggyBac transposase for mammalian applications.</h4>
<p class="authors"><span class="author">Yusa K</span>, <span class="author">Zhou L</span>, <span class="author">Li MA</span>, <span class="author">Bradley A</span> and <span class="author">Craig NL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">DNA transposons have been widely used for transgenesis and insertional mutagenesis in various organisms. Among the transposons active in mammalian cells, the moth-derived transposon piggyBac is most promising with its highly efficient transposition, large cargo capacity, and precise repair of the donor site. Here we report the generation of a hyperactive piggyBac transposase. The active transposition of piggyBac in multiple organisms allowed us to screen a transposase mutant library in yeast for hyperactive mutants and then to test candidates in mouse ES cells. We isolated 18 hyperactive mutants in yeast, among which five were also hyperactive in mammalian cells. By combining all mutations, a total of 7 aa substitutions, into a single reading frame, we generated a unique hyperactive piggyBac transposase with 17-fold and ninefold increases in excision and integration, respectively. We showed its applicability by demonstrating an increased efficiency of generation of transgene-free mouse induced pluripotent stem cells. We also analyzed whether this hyperactive piggyBac transposase affects the genomic integrity of the host cells. The frequency of footprints left by the hyperactive piggyBac transposase was as low as WT transposase (~1%) and we found no evidence that the expression of the transposase affects genomic integrity. This hyperactive piggyBac transposase expands the utility of the piggyBac transposon for applications in mammalian genetics and gene therapy.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">WT077187</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2011</span>;<span class="volume">108</span>;<span class="number">4</span>;<span class="pagerange">1531-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21205896">21205896</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3029773">3029773</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1008322108">10.1073/pnas.1008322108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta analysis of candidate gene variants outside the LPA locus with Lp(a) plasma levels in 14,500 participants of six White European cohorts.</h4>
<p class="authors"><span class="author">Zabaneh D</span>, <span class="author">Kumari M</span>, <span class="author">Sandhu M</span>, <span class="author">Wareham N</span>, <span class="author">Wainwright N</span>, <span class="author">Papamarkou T</span>, <span class="author">Hopewell J</span>, <span class="author">Clarke R</span>, <span class="author">Li K</span>, <span class="author">Palmen J</span>, <span class="author">Talmud PJ</span>, <span class="author">Kronenberg F</span>, <span class="author">Lamina C</span>, <span class="author">Summerer M</span>, <span class="author">Paulweber B</span>, <span class="author">Price J</span>, <span class="author">Fowkes G</span>, <span class="author">Stewart M</span>, <span class="author">Drenos F</span>, <span class="author">Shah S</span>, <span class="author">Shah T</span>, <span class="author">Casas JP</span>, <span class="author">Kivimaki M</span>, <span class="author">Whittaker J</span>, <span class="author">Hingorani AD</span> and <span class="author">Humphries SE</span></p>
<p class="affiliation">University College London Genetics Institute, Department of Genetics, Environment and Evolution, Gower St, London WC1E 6BT, UK.</p><p class="abstract"><strong>Background</strong>: Both genome-wide association studies and candidate gene studies have reported that the major determinant of plasma levels of the Lipoprotein (a) [Lp(a)] reside within the LPA locus on chromosome 6. We have used data from the HumanCVD BeadChip to explore the contribution of other candidate genes determining Lp(a) levels.</p><p class="abstract"><strong>Methods</strong>: 48,032 single nucleotide polymorphisms (SNPs) from the Illumina HumanCVD BeadChip were genotyped in 5059 participants of the Whitehall II study (WHII) of randomly ascertained healthy men and women. SNPs showing association with Lp(a) levels of p&lt;10(-4) outside the LPA locus were selected for replication in a total of an additional 9463 participants of five European based studies (EAS, EPIC-Norfolk, NPHSII, PROCARDIS, and SAPHIR).</p><p class="abstract"><strong>Results</strong>: In Whitehall II, apart from the LPA locus (where p values for several SNPs were &lt;10(-30)) there was significant association at four loci GALNT2, FABP1, PPARGC1A and TNFRSFF11A. However, a meta-analysis of the six studies did not confirm any of these findings.</p><p class="abstract"><strong>Conclusion</strong>: Results from this meta analysis of 14,522 participants revealed no candidate genes from the HumanCVD BeadChip outside the LPA locus to have an effect on Lp(a) levels. Further studies with genome-wide and denser SNP coverage are required to confirm or refute this finding.</p>
<p class="grants">Funded by: AHRQ HHS: <span class="grant">HS06516</span>; British Heart Foundation: <span class="grant">PG/07/133/24260</span>, <span class="grant">RG/08/008</span>, <span class="grant">SP/07/007/23671</span>; Cancer Research UK; Department of Health; Medical Research Council; NHLBI NIH HHS: <span class="grant">HL36310</span>; NIA NIH HHS: <span class="grant">AG13196</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Atherosclerosis</span>&nbsp;<span class="year">2011</span>;<span class="volume">217</span>;<span class="number">2</span>;<span class="pagerange">447-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21592478">21592478</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.atherosclerosis.2011.04.015">10.1016/j.atherosclerosis.2011.04.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic and structural variation in the gastric cancer kinome revealed through targeted deep sequencing.</h4>
<p class="authors"><span class="author">Zang ZJ</span>, <span class="author">Ong CK</span>, <span class="author">Cutcutache I</span>, <span class="author">Yu W</span>, <span class="author">Zhang SL</span>, <span class="author">Huang D</span>, <span class="author">Ler LD</span>, <span class="author">Dykema K</span>, <span class="author">Gan A</span>, <span class="author">Tao J</span>, <span class="author">Lim S</span>, <span class="author">Liu Y</span>, <span class="author">Futreal PA</span>, <span class="author">Grabsch H</span>, <span class="author">Furge KA</span>, <span class="author">Goh LK</span>, <span class="author">Rozen S</span>, <span class="author">Teh BT</span> and <span class="author">Tan P</span></p>
<p class="affiliation">Cellular and Molecular Research, National Cancer Centre, Singapore.</p><p class="abstract">Genetic alterations in kinases have been linked to multiple human pathologies. To explore the landscape of kinase genetic variation in gastric cancer (GC), we used targeted, paired-end deep sequencing to analyze 532 protein and phosphoinositide kinases in 14 GC cell lines. We identified 10,604 single-nucleotide variants (SNV) in kinase exons including greater than 300 novel nonsynonymous SNVs. Family-wise analysis of the nonsynonymous SNVs revealed a significant enrichment in mitogen-activated protein kinase (MAPK)-related genes (P &lt; 0.01), suggesting a preferential involvement of this kinase family in GC. A potential antioncogenic role for MAP2K4, a gene exhibiting recurrent alterations in 2 lines, was functionally supported by siRNA knockdown and overexpression studies in wild-type and MAP2K4 variant lines. The deep sequencing data also revealed novel, large-scale structural rearrangement events involving kinases including gene fusions involving CDK12 and the ERBB2 receptor tyrosine kinase in MKN7 cells. Integrating SNVs and copy number alterations, we identified Hs746T as a cell line exhibiting both splice-site mutations and genomic amplification of MET, resulting in MET protein overexpression. When applied to primary GCs, we identified somatic mutations in 8 kinases, 4 of which were recurrently altered in both primary tumors and cell lines (MAP3K6, STK31, FER, and CDKL5). These results demonstrate that how targeted deep sequencing approaches can deliver unprecedented multilevel characterization of a medically and pharmacologically relevant gene family. The catalog of kinome genetic variants assembled here may broaden our knowledge on kinases and provide useful information on genetic alterations in GC.</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2011</span>;<span class="volume">71</span>;<span class="number">1</span>;<span class="pagerange">29-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21097718">21097718</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3719377">3719377</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-10-1749">10.1158/0008-5472.CAN-10-1749</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Animals learn new tricks from microorganisms.</h4>
<p class="authors"><span class="author">Zarowiecki M</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2011</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">836</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22085859">22085859</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2694">10.1038/nrmicro2694</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation association studies for complex traits.</h4>
<p class="authors"><span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%65%6C%65%66%74%68%65%72%69%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#108;&#101;&#102;&#116;&#104;&#101;&#114;&#105;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A new study successfully applies complementary whole-genome sequencing and imputation approaches to establish robust disease associations in an isolated population. This strategy is poised to help elucidate the role of variants at the low end of the allele frequency spectrum in the genetic architecture of complex traits.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088885</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">43</span>;<span class="number">4</span>;<span class="pagerange">287-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21445070">21445070</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3435533">3435533</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0411-287">10.1038/ng0411-287</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evaluation of power to detect low-frequency variant associations using allele-matching tests that account for uncertainty.</h4>
<p class="authors"><span class="author">Zeggini E</span> and <span class="author">Asimit JL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1HH, UK. <a href="mailto:%45%6C%65%66%74%68%65%72%69%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#69;&#108;&#101;&#102;&#116;&#104;&#101;&#114;&#105;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">There is growing interest in the role of rare variants in multifactorial disease etiology, and increasing evidence that rare variants are associated with complex traits. Single SNP tests are underpowered in rare variant association analyses, so locus-based tests must be used. Quality scores at both the SNP and genotype level are available for sequencing data and they are rarely accounted for. A locus-based method that has high power in the presence of rare variants is extended to incorporate such quality scores as weights, and its power is compared with the original method via a simulation study. Preliminary results suggest that taking uncertainty into account does not improve the power.</p>
<p class="publication"><span class="title">Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</span>&nbsp;<span class="year">2011</span>;<span class="pagerange">100-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21121037">21121037</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3337640">3337640</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of complex disease association studies</h4>
<p class="authors"><span class="author">ZEGGINI,E.</span> and <span class="author">Morris,A.</span></p>
<p class="publication"><span class="title">Analysis of complex disease association studies</span>&nbsp;<span class="year">2011</span></p>
<p class="links">URL: <a title="http://www.sciencedirect.com/science/book/9780123751423" href="http://www.sciencedirect.com/science/book/9780123751423">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/book/9780123</span>751423</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Eight common genetic variants associated with serum DHEAS levels suggest a key role in ageing mechanisms.</h4>
<p class="authors"><span class="author">Zhai G</span>, <span class="author">Teumer A</span>, <span class="author">Stolk L</span>, <span class="author">Perry JR</span>, <span class="author">Vandenput L</span>, <span class="author">Coviello AD</span>, <span class="author">Koster A</span>, <span class="author">Bell JT</span>, <span class="author">Bhasin S</span>, <span class="author">Eriksson J</span>, <span class="author">Eriksson A</span>, <span class="author">Ernst F</span>, <span class="author">Ferrucci L</span>, <span class="author">Frayling TM</span>, <span class="author">Glass D</span>, <span class="author">Grundberg E</span>, <span class="author">Haring R</span>, <span class="author">Hedman AK</span>, <span class="author">Hofman A</span>, <span class="author">Kiel DP</span>, <span class="author">Kroemer HK</span>, <span class="author">Liu Y</span>, <span class="author">Lunetta KL</span>, <span class="author">Maggio M</span>, <span class="author">Lorentzon M</span>, <span class="author">Mangino M</span>, <span class="author">Melzer D</span>, <span class="author">Miljkovic I</span>, <span class="author">MuTHER Consortium</span>, <span class="author">Nica A</span>, <span class="author">Penninx BW</span>, <span class="author">Vasan RS</span>, <span class="author">Rivadeneira F</span>, <span class="author">Small KS</span>, <span class="author">Soranzo N</span>, <span class="author">Uitterlinden AG</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wilson SG</span>, <span class="author">Xi L</span>, <span class="author">Zhuang WV</span>, <span class="author">Harris TB</span>, <span class="author">Murabito JM</span>, <span class="author">Ohlsson C</span>, <span class="author">Murray A</span>, <span class="author">de Jong FH</span>, <span class="author">Spector TD</span> and <span class="author">Wallaschofski H</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King&#39;s College London, London, United Kingdom.</p><p class="abstract">Dehydroepiandrosterone sulphate (DHEAS) is the most abundant circulating steroid secreted by adrenal glands--yet its function is unknown. Its serum concentration declines significantly with increasing age, which has led to speculation that a relative DHEAS deficiency may contribute to the development of common age-related diseases or diminished longevity. We conducted a meta-analysis of genome-wide association data with 14,846 individuals and identified eight independent common SNPs associated with serum DHEAS concentrations. Genes at or near the identified loci include ZKSCAN5 (rs11761528; p = 3.15 &times; 10(-36)), SULT2A1 (rs2637125; p =&#x200A; 2.61 &times; 10(-19)), ARPC1A (rs740160; p =&#x200A; 1.56 &times; 10(-16)), TRIM4 (rs17277546; p =&#x200A; 4.50 &times; 10(-11)), BMF (rs7181230; p = 5.44 &times; 10(-11)), HHEX (rs2497306; p =&#x200A; 4.64 &times; 10(-9)), BCL2L11 (rs6738028; p = 1.72 &times; 10(-8)), and CYP2C9 (rs2185570; p = 2.29 &times; 10(-8)). These genes are associated with type 2 diabetes, lymphoma, actin filament assembly, drug and xenobiotic metabolism, and zinc finger proteins. Several SNPs were associated with changes in gene expression levels, and the related genes are connected to biological pathways linking DHEAS with ageing. This study provides much needed insight into the function of DHEAS.</p>
<p class="grants">Funded by: Arthritis Research UK; Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; Department of Health; NHLBI NIH HHS: <span class="grant">N01-HC-25195</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL094755-04</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">N.1-AG-1-1</span>, <span class="grant">N.1-AG-1-2111</span>, <span class="grant">N01-AG-5-0002</span>, <span class="grant">N01-AG-6-2101</span>, <span class="grant">N01-AG-6-2103</span>, <span class="grant">N01-AG-6-2106</span>, <span class="grant">R01 AG032098-04</span>, <span class="grant">R01 AR/AG 41398</span>, <span class="grant">R21AG032598</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2011</span>;<span class="volume">7</span>;<span class="number">4</span>;<span class="pagerange">e1002025</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21533175" rel="external">21533175</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3077384" rel="external">3077384</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002025" rel="external">10.1371/journal.pgen.1002025</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative transcriptome analyses indicate molecular homology of zebrafish swimbladder and mammalian lung.</h4>
<p class="authors"><span class="author">Zheng W</span>, <span class="author">Wang Z</span>, <span class="author">Collins JE</span>, <span class="author">Andrews RM</span>, <span class="author">Stemple D</span> and <span class="author">Gong Z</span></p>
<p class="affiliation">Department of Biological Sciences, National University of Singapore, Singapore, Singapore.</p><p class="abstract">The fish swimbladder is a unique organ in vertebrate evolution and it functions for regulating buoyancy in most teleost species. It has long been postulated as a homolog of the tetrapod lung, but the molecular evidence is scarce. In order to understand the molecular function of swimbladder as well as its relationship with lungs in tetrapods, transcriptomic analyses of zebrafish swimbladder were carried out by RNA-seq. Gene ontology classification showed that genes in cytoskeleton and endoplasmic reticulum were enriched in the swimbladder. Further analyses depicted gene sets and pathways closely related to cytoskeleton constitution and regulation, cell adhesion, and extracellular matrix. Several prominent transcription factor genes in the swimbladder including hoxc4a, hoxc6a, hoxc8a and foxf1 were identified and their expressions in developing swimbladder during embryogenesis were confirmed. By comparison of enriched transcripts in the swimbladder with those in human and mouse lungs, we established the resemblance of transcriptome of the zebrafish swimbladder and mammalian lungs. Based on the transcriptomic data of zebrafish swimbladder, the predominant functions of swimbladder are in its epithelial and muscular tissues. Our comparative analyses also provide molecular evidence of the relatedness of the fish swimbladder and mammalian lung.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">e24019</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21887364" rel="external">21887364</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3162596" rel="external">3162596</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0024019" rel="external">10.1371/journal.pone.0024019</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of the association of a MET variant with autism in a Chinese Han population.</h4>
<p class="authors"><span class="author">Zhou X</span>, <span class="author">Xu Y</span>, <span class="author">Wang J</span>, <span class="author">Zhou H</span>, <span class="author">Liu X</span>, <span class="author">Ayub Q</span>, <span class="author">Wang X</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Wu L</span> and <span class="author">Xue Y</span></p>
<p class="affiliation">Department of Children&#39;s and Adolescent Health, Public Health College of Harbin Medical University, Harbin, Heilongjiang, People&#39;s Republic of China.</p><p class="abstract"><strong>Background</strong>: Autism is a common, severe and highly heritable neurodevelopmental disorder in children, affecting up to 100 children per 10,000. The MET gene has been regarded as a promising candidate gene for this disorder because it is located within a replicated linkage interval, is involved in pathways affecting the development of the cerebral cortex and cerebellum in ways relevant to autism patients, and has shown significant association signals in previous studies.</p><p class="abstract">Here, we present new ASD patient and control samples from Heilongjiang, China and use them in a case-control and family-based replication study of two MET variants. One SNP, rs38845, was successfully replicated in a case-control association study, but failed to replicate in a family-based study, possibly due to small sample size. The other SNP, rs1858830, failed to replicate in both case-control and family-based studies.</p><p class="abstract"><strong>Conclusions</strong>: This is the first attempt to replicate associations in Chinese autism samples, and our result provides evidence that MET variants may be relevant to autism susceptibility in the Chinese Han population.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2011</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e27428</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22110649">22110649</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3217055">3217055</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0027428">10.1371/journal.pone.0027428</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Lin28/let-7 axis regulates glucose metabolism.</h4>
<p class="authors"><span class="author">Zhu H</span>, <span class="author">Shyh-Chang N</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">Shinoda G</span>, <span class="author">Shah SP</span>, <span class="author">Einhorn WS</span>, <span class="author">Takeuchi A</span>, <span class="author">Engreitz JM</span>, <span class="author">Hagan JP</span>, <span class="author">Kharas MG</span>, <span class="author">Urbach A</span>, <span class="author">Thornton JE</span>, <span class="author">Triboulet R</span>, <span class="author">Gregory RI</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Altshuler D</span> and <span class="author">Daley GQ</span></p>
<p class="affiliation">Stem Cell Transplantation Program, Division of Pediatric Hematology/Oncology, Children&#39;s Hospital Boston and Dana Farber Cancer Institute, Boston, MA, USA.</p><p class="abstract">The let-7 tumor suppressor microRNAs are known for their regulation of oncogenes, while the RNA-binding proteins Lin28a/b promote malignancy by inhibiting let-7 biogenesis. We have uncovered unexpected roles for the Lin28/let-7 pathway in regulating metabolism. When overexpressed in mice, both Lin28a and LIN28B promote an insulin-sensitized state that resists high-fat-diet induced diabetes. Conversely, muscle-specific loss of Lin28a or overexpression of let-7 results in insulin resistance and impaired glucose tolerance. These phenomena occur, in part, through the let-7-mediated repression of multiple components of the insulin-PI3K-mTOR pathway, including IGF1R, INSR, and IRS2. In addition, the mTOR inhibitor, rapamycin, abrogates Lin28a-mediated insulin sensitivity and enhanced glucose uptake. Moreover, let-7 targets are enriched for genes containing SNPs associated with type 2 diabetes and control of fasting glucose in human genome-wide association studies. These data establish the Lin28/let-7 pathway as a central regulator of mammalian glucose metabolism.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G19/35</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1015</span>; NCI NIH HHS: <span class="grant">K08 CA157727</span>, <span class="grant">T32 CA009172</span>; NIDDK NIH HHS: <span class="grant">R01 DK070055</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2011</span>;<span class="volume">147</span>;<span class="number">1</span>;<span class="pagerange">81-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21962509">21962509</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3353524">3353524</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2011.08.033">10.1016/j.cell.2011.08.033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">16q24.1 microdeletion in a premature newborn: usefulness of array-based comparative genomic hybridization in persistent pulmonary hypertension of the newborn.</h4>
<p class="authors"><span class="author">Zufferey F</span>, <span class="author">Martinet D</span>, <span class="author">Osterheld MC</span>, <span class="author">Niel-B&uuml;tschi F</span>, <span class="author">Giannoni E</span>, <span class="author">Schmutz NB</span>, <span class="author">Xia Z</span>, <span class="author">Beckmann JS</span>, <span class="author">Shaw-Smith C</span>, <span class="author">Stankiewicz P</span>, <span class="author">Langston C</span> and <span class="author">Fellmann F</span></p>
<p class="affiliation">University Institute of Pathology, Centre Hospitalier Universitaire Vaudois, Lausanne, Switzerland.</p><p class="abstract"><strong>Objective</strong>: Report of a 16q24.1 deletion in a premature newborn, demonstrating the usefulness of array-based comparative genomic hybridization in persistent pulmonary hypertension of the newborn and multiple congenital malformations.</p><p class="abstract"><strong>Design</strong>: Descriptive case report.</p><p class="abstract"><strong>Setting</strong>: Genetic department and neonatal intensive care unit of a tertiary care children&#39;s hospital.</p><p class="abstract"><strong>Interventions</strong>: None.</p><p class="abstract"><strong>Patient</strong>: We report the case of a preterm male infant, born at 26 wks of gestation. A cardiac malformation and bilateral hydronephrosis were diagnosed at 19 wks of gestation. Karyotype analysis was normal, and a 22q11.2 microdeletion was excluded by fluorescence in situ hybridization analysis. A cesarean section was performed due to fetal distress. The patient developed persistent pulmonary hypertension unresponsive to mechanical ventilation and nitric oxide treatment and expired at 16 hrs of life.</p><p class="abstract">An autopsy revealed partial atrioventricular canal malformation and showed bilateral dilation of the renal pelvocaliceal system with bilateral ureteral stenosis and annular pancreas. Array-based comparative genomic hybridization analysis (Agilent oligoNT 44K, Agilent Technologies, Santa Clara, CA) showed an interstitial microdeletion encompassing the forkhead box gene cluster in 16q24.1. Review of the pulmonary microscopic examination showed the characteristic features of alveolar capillary dysplasia with misalignment of pulmonary veins. Some features were less prominent due to the gestational age.</p><p class="abstract"><strong>Conclusions</strong>: Our review of the literature shows that alveolar capillary dysplasia with misalignment of pulmonary veins is rare but probably underreported. Prematurity is not a usual presentation, and histologic features are difficult to interpret. In our case, array-based comparative genomic hybridization revealed a 16q24.1 deletion, leading to the final diagnosis of alveolar capillary dysplasia with misalignment of pulmonary veins. It emphasizes the usefulness of array-based comparative genomic hybridization analysis as a diagnostic tool with implications for both prognosis and management decisions in newborns with refractory persistent pulmonary hypertension and multiple congenital malformations.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">1 R01 HL101975-01</span></p>
<p class="publication"><span class="title">Pediatric critical care medicine : a journal of the Society of Critical Care Medicine and the World Federation of Pediatric Intensive and Critical Care Societies</span>&nbsp;<span class="year">2011</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">e427-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21572369" rel="external">21572369</a>; DOI: <a href="http://dx.doi.org/10.1097%2FPCC.0b013e3182192c96" rel="external">10.1097/PCC.0b013e3182192c96</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/s1muvacw"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/s1muvacw" src="/qr/s1muvacw.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:47 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
