<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2005 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2005.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2005</h1><p>Number of papers published in 2005: <strong>240</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WebACT--an online companion for the Artemis Comparison Tool.</h4>
<p class="authors"><span class="author">Abbott JC</span>, <span class="author">Aanensen DM</span>, <span class="author">Rutherford K</span>, <span class="author">Butcher S</span> and <span class="author">Spratt BG</span></p>
<p class="affiliation">Centre for Bioinformatics, Division of Molecular Biosciences, Imperial College London, London SW7 2AZ, UK. <a href="mailto:%61%64%6D%69%6E%40%77%65%62%61%63%74.%6F%72%67">&#97;&#100;&#109;&#105;&#110;&#64;&#119;&#101;&#98;&#97;&#99;&#116;&#46;&#111;&#114;&#103;</a></p><p class="abstract">WebACT is an online resource which enables the rapid provision of simultaneous BLAST comparisons between up to five genomic sequences in a format amenable for visualization with the well-known Artemis Comparison Tool (ACT). Comparisons can be generated on-the-fly using sequences directly retrieved via EMBL database queries, or by entering or uploading user sequences. Furthermore, pre-computed comparisons are available between all publicly available, completed prokaryotic genomes and plasmids currently contained within the Genome Reviews database (372 sequences, representing 175 different species). The system is designed to minimize the volume of downloaded data and maximize performance. Genome sequences, annotation and pre-computed comparisons are stored in a relational database allowing flexible querying based on user-defined sequence regions, from whole genome to a defined region flanking a specified gene. Comparison and sequence files, whether computed online or retrieved from the database of pre-computed genome comparisons, can be viewed online using ACT and are available for download. AVAILABILITY: Freely accessible at http://www.webact.org. SUPPLEMENTARY INFORMATION: User guide and worked examples are available at http://www.webact.org/WebACT/docs.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">18</span>;<span class="pagerange">3665-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16076890" rel="external">16076890</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti601" rel="external">10.1093/bioinformatics/bti601</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide, end-sequenced 129Sv BAC library resource for targeting vector construction.</h4>
<p class="authors"><span class="author">Adams DJ</span>, <span class="author">Quail MA</span>, <span class="author">Cox T</span>, <span class="author">van der Weyden L</span>, <span class="author">Gorick BD</span>, <span class="author">Su Q</span>, <span class="author">Chan WI</span>, <span class="author">Davies R</span>, <span class="author">Bonfield JK</span>, <span class="author">Law F</span>, <span class="author">Humphray S</span>, <span class="author">Plumb B</span>, <span class="author">Liu P</span>, <span class="author">Rogers J</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The majority of gene-targeting experiments in mice are performed in 129Sv-derived embryonic stem (ES) cell lines, which are generally considered to be more reliable at colonizing the germ line than ES cells derived from other strains. Gene targeting is reliant on homologous recombination of a targeting vector with the host ES cell genome. The efficiency of recombination is affected by many factors, including the isogenicity (H. te Riele et al., 1992, Proc. Natl. Acad. Sci. USA 89, 5128-5132) and the length of homologous sequence of the targeting vector and the location of the target locus. Here we describe the double-end sequencing and mapping of 84,507 bacterial artificial chromosomes (BACs) generated from AB2.2 ES cell DNA (129S7/SvEvBrd-Hprtb-m2). We have aligned these BACs against the mouse genome and displayed them on the Ensembl genome browser, DAS: 129S7/AB2.2. This library has an average insert size of 110.68 kb and average depth of genome coverage of 3.63- and 1.24-fold across the autosomes and sex chromosomes, respectively. Over 97% of the mouse genome and 99.1% of Ensembl genes are covered by clones from this library. This publicly available BAC resource can be used for the rapid construction of targeting vectors via recombineering. Furthermore, we show that targeting vectors containing DNA recombineered from this BAC library can be used to target genes efficiently in several 129-derived ES cell lines.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">86</span>;<span class="number">6</span>;<span class="pagerange">753-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16257172" rel="external">16257172</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ygeno.2005.08.003" rel="external">10.1016/j.ygeno.2005.08.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BRCTx is a novel, highly conserved RAD18-interacting protein.</h4>
<p class="authors"><span class="author">Adams DJ</span>, <span class="author">van der Weyden L</span>, <span class="author">Gergely FV</span>, <span class="author">Arends MJ</span>, <span class="author">Ng BL</span>, <span class="author">Tannahill D</span>, <span class="author">Kanaar R</span>, <span class="author">Markus A</span>, <span class="author">Morris BJ</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute Hinxton, Cambs CB10 1SA, United Kingdom.</p><p class="abstract">The BRCT domain is a highly conserved module found in many proteins that participate in DNA damage checkpoint regulation, DNA repair, and cell cycle control. Here we describe the cloning, characterization, and targeted mutagenesis of Brctx, a novel gene with a BRCT motif. Brctx was found to be expressed ubiquitously in adult tissues and during development, with the highest levels found in testis. Brctx-deficient mice develop normally, show no pathological abnormalities, and are fertile. BRCTx binds to the C terminus of hRAD18 in yeast two-hybrid and immunoprecipitation assays and colocalizes with this protein in the nucleus. Despite this, Brctx-deficient murine embryonic fibroblasts (MEFs) do not show overt sensitivity to DNA-damaging agents. MEFs from Brctx-deficient embryos grow at a similar rate to wild-type MEF CD4/CD8 expressions, and the cell cycle parameters of thymocytes from wild-type and Brctx knockout animals are indistinguishable. Intriguingly, the BRCT domain of BRCTx is responsible for mediating its localization to the nucleus and centrosome in interphase cells. We conclude that, although highly conserved, Brctx is not essential for the above-mentioned processes and may be redundant.</p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">2</span>;<span class="pagerange">779-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15632077">15632077</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC543416">543416</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.25.2.779-788.2005">10.1128/MCB.25.2.779-788.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A case for a Glossina genome project.</h4>
<p class="authors"><span class="author">Aksoy S</span>, <span class="author">Berriman M</span>, <span class="author">Hall N</span>, <span class="author">Hattori M</span>, <span class="author">Hide W</span> and <span class="author">Lehane MJ</span></p>
<p class="affiliation">Yale University School of Medicine, 60 College Street, 606 LEPH, New Haven, CT 06520, USA. <a href="mailto:%73%65%72%61%70.%61%6B%73%6F%79%40%79%61%6C%65.%65%64%75">&#115;&#101;&#114;&#97;&#112;&#46;&#97;&#107;&#115;&#111;&#121;&#64;&#121;&#97;&#108;&#101;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Given the medical and agricultural significance of Glossina, knowledge of the genomic aspects of the vector and vector-pathogen interactions are a high priority. In preparation for a full genome sequence initiative, an extensive set of expressed sequence tags (ESTs) has been generated from tissue-specific normalized libraries. In addition, bacterial artificial chromosome (BAC) libraries are being constructed, and information on the genome structure and size from different species has been obtained. An international consortium is now in place to further efforts to lead to a full genome project.</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">107-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15734656" rel="external">15734656</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2005.01.006" rel="external">10.1016/j.pt.2005.01.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations of the catalytic subunit of RAB3GAP cause Warburg Micro syndrome.</h4>
<p class="authors"><span class="author">Aligianis IA</span>, <span class="author">Johnson CA</span>, <span class="author">Gissen P</span>, <span class="author">Chen D</span>, <span class="author">Hampshire D</span>, <span class="author">Hoffmann K</span>, <span class="author">Maina EN</span>, <span class="author">Morgan NV</span>, <span class="author">Tee L</span>, <span class="author">Morton J</span>, <span class="author">Ainsworth JR</span>, <span class="author">Horn D</span>, <span class="author">Rosser E</span>, <span class="author">Cole TR</span>, <span class="author">Stolte-Dijkstra I</span>, <span class="author">Fieggen K</span>, <span class="author">Clayton-Smith J</span>, <span class="author">M&eacute;garban&eacute; A</span>, <span class="author">Shield JP</span>, <span class="author">Newbury-Ecob R</span>, <span class="author">Dobyns WB</span>, <span class="author">Graham JM</span>, <span class="author">Kjaer KW</span>, <span class="author">Warburg M</span>, <span class="author">Bond J</span>, <span class="author">Trembath RC</span>, <span class="author">Harris LW</span>, <span class="author">Takai Y</span>, <span class="author">Mundlos S</span>, <span class="author">Tannahill D</span>, <span class="author">Woods CG</span> and <span class="author">Maher ER</span></p>
<p class="affiliation">Section of Medical and Molecular Genetics, University of Birmingham, Birmingham, B15 2TT, UK.</p><p class="abstract">Warburg Micro syndrome (WARBM1) is a severe autosomal recessive disorder characterized by developmental abnormalities of the eye and central nervous system and by microgenitalia. We identified homozygous inactivating mutations in RAB3GAP, encoding RAB3 GTPase activating protein, a key regulator of the Rab3 pathway implicated in exocytic release of neurotransmitters and hormones, in 12 families with Micro syndrome. We hypothesize that the underlying pathogenesis of Micro syndrome is a failure of exocytic release of ocular and neurodevelopmental trophic factors.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">37</span>;<span class="number">3</span>;<span class="pagerange">221-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15696165">15696165</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1517">10.1038/ng1517</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of core and variable components of the Salmonella enterica subspecies I genome by microarray.</h4>
<p class="authors"><span class="author">Anjum MF</span>, <span class="author">Marooney C</span>, <span class="author">Fookes M</span>, <span class="author">Baker S</span>, <span class="author">Dougan G</span>, <span class="author">Ivens A</span> and <span class="author">Woodward MJ</span></p>
<p class="affiliation">Department of Food and Environmental Safety, Veterinary Laboratories Agency-Weybridge, New Haw, Addlestone, Surrey KT15 3NB, United Kingdom. <a href="mailto:%6D.%61%6E%6A%75%6D%40%76%6C%61.%64%65%66%72%61.%67%73%69.%67%6F%76.%75%6B">&#109;&#46;&#97;&#110;&#106;&#117;&#109;&#64;&#118;&#108;&#97;&#46;&#100;&#101;&#102;&#114;&#97;&#46;&#103;&#115;&#105;&#46;&#103;&#111;&#118;&#46;&#117;&#107;</a></p><p class="abstract">We have performed microarray hybridization studies on 40 clinical isolates from 12 common serovars within Salmonella enterica subspecies I to identify the conserved chromosomal gene pool. We were able to separate the core invariant portion of the genome by a novel mathematical approach using a decision tree based on genes ranked by increasing variance. All genes within the core component were confirmed using available sequence and microarray information for S. enterica subspecies I strains. The majority of genes within the core component had conserved homologues in Escherichia coli K-12 strain MG1655. However, many genes present in the conserved set which were absent or highly divergent in K-12 had close homologues in pathogenic bacteria such as Shigella flexneri and Pseudomonas aeruginosa. Genes within previously established virulence determinants such as SPI1 to SPI5 were conserved. In addition several genes within SPI6, all of SPI9, and three fimbrial operons (fim, bcf, and stb) were conserved within all S. enterica strains included in this study. Although many phage and insertion sequence elements were missing from the core component, approximately half the pseudogenes present in S. enterica serovar Typhi were conserved. Furthermore, approximately half the genes conserved in the core set encoded hypothetical proteins. Separation of the core and variant gene sets within S.enterica subspecies I has offered fundamental biological insight into the genetic basis of phenotypic similarity and diversity across S. enterica subspecies I and shown how the core genome of these pathogens differs from the closely related E. coli K-12 laboratory strain.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2005</span>;<span class="volume">73</span>;<span class="number">12</span>;<span class="pagerange">7894-905</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16299280" rel="external">16299280</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1307019" rel="external">1307019</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.73.12.7894-7905.2005" rel="external">10.1128/IAI.73.12.7894-7905.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Vertebrate Genome Annotation (Vega) database.</h4>
<p class="authors"><span class="author">Ashurst JL</span>, <span class="author">Chen CK</span>, <span class="author">Gilbert JG</span>, <span class="author">Jekosch K</span>, <span class="author">Keenan S</span>, <span class="author">Meidl P</span>, <span class="author">Searle SM</span>, <span class="author">Stalker J</span>, <span class="author">Storey R</span>, <span class="author">Trevanion S</span>, <span class="author">Wilming L</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6A%6C%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#108;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Vertebrate Genome Annotation (Vega) database (http://vega.sanger.ac.uk) has been designed to be a community resource for browsing manual annotation of finished sequences from a variety of vertebrate genomes. Its core database is based on an Ensembl-style schema, extended to incorporate curation-specific metadata. In collaboration with the genome sequencing centres, Vega attempts to present consistent high-quality annotation of the published human chromosome sequences. In addition, it is also possible to view various finished regions from other vertebrates, including mouse and zebrafish. Vega displays only manually annotated gene structures built using transcriptional evidence, which can be examined in the browser. Attempts have been made to standardize the annotation procedure across each vertebrate genome, which should aid comparative analysis of orthologues across the different finished regions.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Database issue</span>;<span class="pagerange">D459-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15608237">15608237</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540089">540089</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki135">10.1093/nar/gki135</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integration of tools and resources for display and analysis of genomic data for protozoan parasites.</h4>
<p class="authors"><span class="author">Aslett M</span>, <span class="author">Mooney P</span>, <span class="author">Adlem E</span>, <span class="author">Berriman M</span>, <span class="author">Berry A</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Ivens AC</span>, <span class="author">Kerhornou A</span>, <span class="author">Parkhill J</span>, <span class="author">Peacock CS</span>, <span class="author">Wood V</span>, <span class="author">Rajandream MA</span>, <span class="author">Barrell B</span> and <span class="author">Tivey A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%61%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#97;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Centralisation of tools for analysis of genomic data is paramount in ensuring that research is always carried out on the latest currently available data. As such, World Wide Web sites providing a range of online analyses and displays of data can play a crucial role in guaranteeing consistency of in silico work. In this respect, the protozoan parasite research community is served by several resources, either focussing on data and tools for one species or taking a broader view and providing tools for analysis of data from many species, thereby facilitating comparative studies. In this paper, we give a broad overview of the online resources available. We then focus on the GeneDB project, detailing the features and tools currently available through it. Finally, we discuss data curation and its importance in keeping genomic data &#39;relevant&#39; to the research community.</p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2005</span>;<span class="volume">35</span>;<span class="number">5</span>;<span class="pagerange">481-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15826641" rel="external">15826641</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijpara.2005.01.011" rel="external">10.1016/j.ijpara.2005.01.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A transcriptional pathway for cell separation in fission yeast.</h4>
<p class="authors"><span class="author">B&auml;hler J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6A%75%72%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#117;&#114;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Numerous genes are transcriptionally activated and repressed in a cell cycle-dependent manner. We have recently reported the global gene expression program during the cell cycle in fission yeast (S. pombe). Among the periodically expressed fission yeast genes, a large proportion shows peak transcript levels during mitosis. Many of these genes are regulated by a transcriptional cascade involving two transcription factors: the forkhead protein Sep1p which activates the zinc finger protein Ace2p. A main function of the Sep1p-Ace2p transcriptional pathway is to trigger the separation of daughter cells after cytokinesis. Absence of Sep1p, Ace2p, or some of their target genes leads to a hyphal-like growth pattern with chains of connected cells. Yeast cells probably evolved from filamentous fungi. It is possible that the Sep1p-Ace2p pathway contributed to the emergence of proliferation through single cells, and that this regulatory pathway can still be modulated to adjust growth modes depending on environmental conditions. Here, various properties of the Sep1p-Ace2p transcriptional pathway and mechanisms for cell separation are discussed in the context of recent findings.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Cell cycle (Georgetown, Tex.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">4</span>;<span class="number">1</span>;<span class="pagerange">39-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15611619" rel="external">15611619</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A logical circuit for the regulation of fission yeast growth modes.</h4>
<p class="authors"><span class="author">B&auml;hler J</span> and <span class="author">Svetina S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6A%75%72%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#117;&#114;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Growth of fission yeast at the ends of its cylindrical cells switches from a monopolar to a bipolar mode, before it ceases during mitosis and cell division. Here we assume that these growth modes correspond to three stable states of an underlying regulatory circuit, which is a relatively simple and to a large degree autonomous subsystem of an otherwise complex cellular control system. We develop a switch-like logical circuit based on three elements defined as binary variables. Effects of circuit variables on each other are expressed in terms of logical operations. We analyse this circuit for its behavior (&quot;phenotypes&quot;) after removing single or multiple operations (&quot;mutants&quot;). Known fission yeast polarity mutants such as those defective in the switch to bipolar growth can be classified based on these predicted &#39;phenotypes&#39;. Differences in growth patterns between daughter cells in different bipolar growth mutants are also predicted by the circuit model. The model presented here should provide a useful framework to guide future experiments into mechanisms of cellular polarity. This paper illustrates the usefulness of simple logical circuits to describe and dissect features of complex regulatory processes such as the fission yeast growth patterns in both wild type and mutant cells.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Journal of theoretical biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">237</span>;<span class="number">2</span>;<span class="pagerange">210-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15949817" rel="external">15949817</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jtbi.2005.04.008" rel="external">10.1016/j.jtbi.2005.04.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of Vi-negative Salmonella enterica serovar typhi in the peripheral blood of patients with typhoid fever in the Faisalabad region of Pakistan.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Sarwar Y</span>, <span class="author">Aziz H</span>, <span class="author">Haque A</span>, <span class="author">Ali A</span>, <span class="author">Dougan G</span>, <span class="author">Wain J</span> and <span class="author">Haque A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK. <a href="mailto:%73%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The synthesis and transportation proteins of the Vi capsular polysaccharide of Salmonella enterica serovar Typhi (serovar Typhi) are encoded by the viaB operon, which resides on a 134-kb pathogenicity island known as SPI-7. In recent years, Vi-negative strains of serovar Typhi have been reported in regions where typhoid fever is endemic. However, because Vi negativity can arise during in vitro passage, the clinical significance of Vi-negative serovar Typhi is not clear. To investigate the loss of Vi expression at the genetic level, 60 stored strains of serovar Typhi from the Faisalabad region of Pakistan were analyzed by PCR for the presence of SPI-7 and two genes essential for Vi production: tviA and tviB. Nine of the sixty strains analyzed (15%) tested negative for both tviA and tviB; only two of these strains lacked SPI-7. In order to investigate whether this phenomenon occurred in vivo, blood samples from patients with the clinical symptoms of typhoid fever were also investigated. Of 48 blood samples tested, 42 tested positive by fliC PCR for serovar Typhi; 4 of these were negative for tviA and tviB. Three of these samples tested positive for SPI-7. These results demonstrate that viaB-negative, SPI-7-positive serovar Typhi is naturally occurring and can be detected by PCR in the peripheral blood of typhoid patients in this region. The method described here can be used to monitor the incidence of Vi-negative serovar Typhi in regions where the Vi vaccine is used.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">43</span>;<span class="number">9</span>;<span class="pagerange">4418-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16145086" rel="external">16145086</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1234127" rel="external">1234127</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.43.9.4418-4425.2005" rel="external">10.1128/JCM.43.9.4418-4425.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular cytogenetic analyses of breakpoints in apparently balanced reciprocal translocations carried by phenotypically normal individuals.</h4>
<p class="authors"><span class="author">Baptista J</span>, <span class="author">Prigmore E</span>, <span class="author">Gribble SM</span>, <span class="author">Jacobs PA</span>, <span class="author">Carter NP</span> and <span class="author">Crolla JA</span></p>
<p class="affiliation">Wessex Regional Genetics Laboratory, Salisbury District Hospital, Salisbury, Wiltshire, UK. <a href="mailto:%4A%75%6C%69%61.%62%61%70%74%69%73%74%61%40%73%61%6C%69%73%62%75%72%79.%6E%68%73.%75%6B">&#74;&#117;&#108;&#105;&#97;&#46;&#98;&#97;&#112;&#116;&#105;&#115;&#116;&#97;&#64;&#115;&#97;&#108;&#105;&#115;&#98;&#117;&#114;&#121;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract">To test the hypothesis that translocation breakpoints in normal individuals are simple and do not disrupt genes, we characterised the breakpoints in 13 phenotypically normal individuals incidentally ascertained with an apparently balanced reciprocal translocation. Cases were karyotyped, and the breakpoints were refined by fluorescence in situ hybridisation until breakpoint-spanning clones were identified. 1 Mb array-CGH was performed as a whole genome analysis tool to detect any imbalances in chromatin not directly involved in the breakpoints. Breakpoint-associated imbalances were not found in any of the patients analysed in this study. However, breakpoints which disrupted known genes were identified in two patients, with RYR2 disrupted in one patient and COL13A1 in the other. In a further eight patients, Ensembl mapping data suggested that a gene might be disrupted by a breakpoint. In one further patient, the translocation was shown to be nonreciprocal. This study shows that apparently balanced reciprocal translocations in phenotypically normal patients do not have imbalances at the breakpoints, in contrast to phenotypically abnormal patients where the translocation breakpoints are often associated with cryptic imbalances. However, phenotypically normal individuals, and phenotypically abnormal individuals may have genes disrupted and therefore inactivated by one of the breakpoints. The significance of these disruptions remains to be determined.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">11</span>;<span class="pagerange">1205-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16118644" rel="external">16118644</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5201488" rel="external">10.1038/sj.ejhg.5201488</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complex disease: pleiotropic gene effects in obesity and type 2 diabetes.</h4>
<p class="authors"><span class="author">Barroso I</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">12</span>;<span class="pagerange">1243-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16251891">16251891</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5201514">10.1038/sj.ejhg.5201514</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetics of Type 2 diabetes.</h4>
<p class="authors"><span class="author">Barroso I</span></p>
<p class="affiliation">Metabolic Disease Group, The Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%69%62%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#98;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Type 2 diabetes (T2D) has become a health-care problem worldwide, with the rise in disease prevalence being all the more worrying as it not only affects the developed world but also developing nations with fewer resources to cope with yet another major disease burden. Furthermore, the problem is no longer restricted to the ageing population, as young adults and children are also being diagnosed with T2D. In recent years, there has been a surge in the number of genetic studies of T2D in attempts to identify some of the underlying risk factors. In this review, I highlight the main genes known to cause uncommon monogenic forms of diabetes (e.g. maturity-onset diabetes of the young--MODY--and insulin resistance syndromes), as well as describe some of the main approaches used to identify genes involved in the more common forms of T2D that result from the interaction between environmental risk factors and predisposing genotypes. Linkage and candidate gene studies have been highly successful in the identification of genes that cause the monogenic variants of diabetes and, although progress in the more common forms of T2D has been slow, a number of genes have now been reproducibly associated with T2D risk in multiple studies. These are discussed, as well as the main implications that the diabetes gene discoveries will have in diabetes treatment and prevention.</p>
<p class="publication"><span class="title">Diabetic medicine : a journal of the British Diabetic Association</span>&nbsp;<span class="year">2005</span>;<span class="volume">22</span>;<span class="number">5</span>;<span class="pagerange">517-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15842505">15842505</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1464-5491.2005.01550.x">10.1111/j.1464-5491.2005.01550.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">What the genome sequence is revealing about trypanosome antigenic variation.</h4>
<p class="authors"><span class="author">Barry JD</span>, <span class="author">Marcello L</span>, <span class="author">Morrison LJ</span>, <span class="author">Read AF</span>, <span class="author">Lythgoe K</span>, <span class="author">Jones N</span>, <span class="author">Carrington M</span>, <span class="author">Blandin G</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Caler E</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Renauld H</span>, <span class="author">El-Sayed N</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">University of Glasgow, Glasgow G12 8QQ, Scotland, UK. <a href="mailto:%6A.%64.%62%61%72%72%79%40%62%69%6F.%67%6C%61.%61%63.%75%6B">&#106;&#46;&#100;&#46;&#98;&#97;&#114;&#114;&#121;&#64;&#98;&#105;&#111;&#46;&#103;&#108;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">African trypanosomes evade humoral immunity through antigenic variation, whereby they switch expression of the gene encoding their VSG (variant surface glycoprotein) coat. Switching proceeds by duplication of silent VSG genes into a transcriptionally active locus. The genome project has revealed that most of the silent archive consists of hundreds of subtelomeric VSG tandem arrays, and that most of these are not functional genes. Precedent suggests that they can contribute combinatorially to the formation of expressed, functional genes through segmental gene conversion. These findings from the genome project have major implications for evolution of the VSG archive and for transmission of the parasite in the field.</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Pt 5</span>;<span class="pagerange">986-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16246028" rel="external">16246028</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST20050986" rel="external">10.1042/BST20050986</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The G5 domain: a potential N-acetylglucosamine recognition domain involved in biofilm formation.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Holden MT</span> and <span class="author">Yeats C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Summary</strong>: Biofilms are complex microbial communities found at surfaces that are often associated with extracellular polysaccharides. Biofilm formation is a complex process that is being understood at the molecular level only recently. We have identified a novel domain that we call the G5 domain (named after its conserved glycine residues), which is found in a variety of enzymes such as Streptococcal IgA peptidases and various glycosyl hydrolases in bacteria. The G5 domain is found in the Accumulation Associated Protein (AAP), which is an important component in biofilm formation in Staphylococcus aureus. A common feature of the proteins containing G5 domains is N-acetylglucosamine binding, and we attribute this function to the G5 domain.</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">8</span>;<span class="pagerange">1301-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15598841">15598841</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti206">10.1093/bioinformatics/bti206</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Endoplasmic reticulum stress induction of the Grp78/BiP promoter: activating mechanisms mediated by YY1 and its interactive chromatin modifiers.</h4>
<p class="authors"><span class="author">Baumeister P</span>, <span class="author">Luo S</span>, <span class="author">Skarnes WC</span>, <span class="author">Sui G</span>, <span class="author">Seto E</span>, <span class="author">Shi Y</span> and <span class="author">Lee AS</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology, USC/Norris Comprehensive Cancer Center, Keck School of Medicine, 1441 Eastlake Ave., Room 5308, MC-9176, Los Angeles, CA 90089-9176, USA.</p><p class="abstract">The unfolded protein response is an evolutionarily conserved mechanism whereby cells respond to stress conditions that target the endoplasmic reticulum (ER). The transcriptional activation of the promoter of GRP78/BiP, a prosurvival ER chaperone, has been used extensively as an indicator of the onset of the UPR. YY1, a constitutively expressed multifunctional transcription factor, activates the Grp78 promoter only under ER stress conditions. Previously, in vivo footprinting analysis revealed that the YY1 binding site of the ER stress response element of the Grp78 promoter exhibits ER stress-induced changes in occupancy. Toward understanding the underlying mechanisms of these unique phenomena, we performed chromatin immunoprecipitation analyses, revealing that YY1 only occupies the Grp78 promoter upon ER stress and is mediated in part by the nuclear form of ATF6. We show that YY1 is an essential coactivator of ATF6 and uncover their specific interactive domains. Using small interfering RNA against YY1 and insertional mutation of the gene encoding ATF6alpha, we provide direct evidence that YY1 and ATF6 are required for optimal stress induction of Grp78. We also discovered enhancement of the ER-stressed induction of the Grp78 promoter through the interaction of YY1 with the arginine methyltransferase PRMT1 and evidence of its action through methylation of the arginine 3 residue on histone H4. Furthermore, we detected ER stress-induced binding of the histone acetyltransferase p300 to the Grp78 promoter and histone H4 acetylation. A model for the ER stress-mediated transcription factor binding and chromatin modifications at the Grp78 promoter leading to its activation is proposed.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA 27607</span>; NIGMS NIH HHS: <span class="grant">GM 53874</span>, <span class="grant">GM 58486</span>, <span class="grant">GM 64850</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">11</span>;<span class="pagerange">4529-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15899857" rel="external">15899857</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1140640" rel="external">1140640</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.25.11.4529-4540.2005" rel="external">10.1128/MCB.25.11.4529-4540.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders.</h4>
<p class="authors"><span class="author">Baxter EJ</span>, <span class="author">Scott LM</span>, <span class="author">Campbell PJ</span>, <span class="author">East C</span>, <span class="author">Fourouclas N</span>, <span class="author">Swanton S</span>, <span class="author">Vassiliou GS</span>, <span class="author">Bench AJ</span>, <span class="author">Boyd EM</span>, <span class="author">Curtin N</span>, <span class="author">Scott MA</span>, <span class="author">Erber WN</span>, <span class="author">Green AR</span> and <span class="author">Cancer Genome Project</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, University of Cambridge, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Human myeloproliferative disorders form a range of clonal haematological malignant diseases, the main members of which are polycythaemia vera, essential thrombocythaemia, and idiopathic myelofibrosis. The molecular pathogenesis of these disorders is unknown, but tyrosine kinases have been implicated in several related disorders. We investigated the role of the cytoplasmic tyrosine kinase JAK2 in patients with a myeloproliferative disorder.</p><p class="abstract"><strong>Methods</strong>: We obtained DNA samples from patients with polycythaemia vera, essential thrombocythaemia, or idiopathic myelofibrosis. The coding exons of JAK2 were bidirectionally sequenced from peripheral-blood granulocytes, T cells, or both. Allele-specific PCR, molecular cytogenetic studies, microsatellite PCR, Affymetrix single nucleotide polymorphism array analyses, and colony assays were undertaken on subgroups of patients.</p><p class="abstract"><strong>Findings</strong>: A single point mutation (Val617Phe) was identified in JAK2 in 71 (97%) of 73 patients with polycythaemia vera, 29 (57%) of 51 with essential thrombocythaemia, and eight (50%) of 16 with idiopathic myelofibrosis. The mutation is acquired, is present in a variable proportion of granulocytes, alters a highly conserved valine present in the negative regulatory JH2 domain, and is predicted to dysregulate kinase activity. It was heterozygous in most patients, homozygous in a subset as a result of mitotic recombination, and arose in a multipotent progenitor capable of giving rise to erythroid and myeloid cells. The mutation was present in all erythropoietin-independent erythroid colonies.</p><p class="abstract"><strong>Interpretation</strong>: A single acquired mutation of JAK2 was noted in more than half of patients with a myeloproliferative disorder. Its presence in all erythropoietin-independent erythroid colonies demonstrates a link with growth factor hypersensitivity, a key biological feature of these disorders.</p><p class="abstract">Identification of the Val617Phe JAK2 mutation lays the foundation for new approaches to the diagnosis, classification, and treatment of myeloproliferative disorders.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2005</span>;<span class="volume">365</span>;<span class="number">9464</span>;<span class="pagerange">1054-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15781101">15781101</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2805%2971142-9">10.1016/S0140-6736(05)71142-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The interferon-inducible p47 (IRG) GTPases in vertebrates: loss of the cell autonomous resistance mechanism in the human lineage.</h4>
<p class="authors"><span class="author">Bekpen C</span>, <span class="author">Hunn JP</span>, <span class="author">Rohde C</span>, <span class="author">Parvanova I</span>, <span class="author">Guethlein L</span>, <span class="author">Dunn DM</span>, <span class="author">Glowalla E</span>, <span class="author">Leptin M</span> and <span class="author">Howard JC</span></p>
<p class="affiliation">Institute for Genetics, University of Cologne, Z&uuml;lpicher Strasse 47, 50674 Cologne, Germany. <a href="mailto:%63.%62%65%6B%70%65%6E%40%75%6E%69-%6B%6F%65%6C%6E.%64%65">&#99;&#46;&#98;&#101;&#107;&#112;&#101;&#110;&#64;&#117;&#110;&#105;&#45;&#107;&#111;&#101;&#108;&#110;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: Members of the p47 (immunity-related GTPases (IRG) family) GTPases are essential, interferon-inducible resistance factors in mice that are active against a broad spectrum of important intracellular pathogens. Surprisingly, there are no reports of p47 function in humans.</p><p class="abstract"><strong>Results</strong>: Here we show that the p47 GTPases are represented by 23 genes in the mouse, whereas humans have only a single full-length p47 GTPase and an expressed, truncated presumed pseudo-gene. The human full-length gene is orthologous to an isolated mouse p47 GTPase that carries no interferon-inducible elements in the promoter of either species and is expressed constitutively in the mature testis of both species. Thus, there is no evidence for a p47 GTPase-based resistance system in humans. Dogs have several interferon-inducible p47s, and so the primate lineage that led to humans appears to have lost an ancient function. Multiple p47 GTPases are also present in the zebrafish, but there is only a tandem p47 gene pair in pufferfish.</p><p class="abstract"><strong>Conclusion</strong>: Mice and humans must deploy their immune resources against vacuolar pathogens in radically different ways. This carries significant implications for the use of the mouse as a model of human infectious disease. The absence of the p47 resistance system in humans suggests that possession of this resistance system carries significant costs that, in the primate lineage that led to humans, are not outweighed by the benefits. The origin of the vertebrate p47 system is obscure.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">R92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16277747" rel="external">16277747</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1297648" rel="external">1297648</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2005-6-11-r92" rel="external">10.1186/gb-2005-6-11-r92</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analyses of murine postsynaptic density-95 identify novel isoforms and potential translational control elements.</h4>
<p class="authors"><span class="author">Bence M</span>, <span class="author">Arbuckle MI</span>, <span class="author">Dickson KS</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Division of Neuroscience, University of Edinburgh, Edinburgh EH8 9JZ, UK.</p><p class="abstract">Postsynaptic density-95 (PSD-95) is an evolutionarily conserved synaptic adaptor protein that is known to bind many proteins including the NMDA receptor. This observation has implicated it in many NMDA receptor-dependent processes including spatial learning and synaptic plasticity. We have cloned and characterised the murine PSD-95 gene. In addition, we have identified two previously uncharacterised splice variants of the major murine PSD-95 transcript (PSD-95alpha): PSD-95alpha-2b results from an extension of exon 2 and PSD-95alpha-Delta18 from the temporal exclusion of exon 18. The presence of PSD-95alpha-2b sequences in other PSD-95 family members implicates this peptide stretch as functionally significant. Another potential transcript (PSD-95gamma) was also identified based on examination of EST databases. Immunoprecipitation assays demonstrate that proteins corresponding in size to PSD-95alpha-Delta18 and PSD-95gamma interact with the NMDA receptor, suggesting an important biological role for these isoforms. Finally, we have performed bioinformatics analyses of the PSD-95 mRNA untranslated regions, identifying multiple translational control elements that suggest protein production could be regulated post-transcriptionally. The variety of mRNA isoforms and regulatory elements identified provides for a high degree of diversity in the structure and function of PSD-95 proteins.</p>
<p class="publication"><span class="title">Brain research. Molecular brain research</span>&nbsp;<span class="year">2005</span>;<span class="volume">133</span>;<span class="number">1</span>;<span class="pagerange">143-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15661374" rel="external">15661374</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbrainres.2004.09.024" rel="external">10.1016/j.molbrainres.2004.09.024</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enemies within.</h4>
<p class="authors"><span class="author">Bentley S</span>, <span class="author">Sebaihia M</span> and <span class="author">Crossman L</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">8-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15635776">15635776</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1071">10.1038/nrmicro1071</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Binding specificity and mRNA targets of a C. elegans PUF protein, FBF-1.</h4>
<p class="authors"><span class="author">Bernstein D</span>, <span class="author">Hook B</span>, <span class="author">Hajarnavis A</span>, <span class="author">Opperman L</span> and <span class="author">Wickens M</span></p>
<p class="affiliation">Department of Biochemistry, University of Wisconsin, 433 Babcock Drive, Madison, WI 53706, USA.</p><p class="abstract">Sequence-specific RNA-protein interactions underlie regulation of many mRNAs. Here we analyze the RNA sequence specificity of Caenorhabditis elegans FBF-1, a founding member of the PUF protein family. Like other PUF proteins, FBF-1 binds to the 3&#39; UTR of target mRNAs and decreases expression of those target genes. Here, we show that FBF-1 and its close relative, FBF-2, bind with similar affinity to multiple RNA sites. We use mutagenesis and in vivo selection experiments to identify nucleotides that are essential for FBF-1 binding. The binding elements comprise a &quot;core&quot; central region and flanking sequences. The core region is similar but distinct from the binding sites of other PUF proteins. We combine the identification of binding elements with informatics to predict new FBF-1 binding sites in a C. elegans 3&#39; UTR database. These data identify a set of new candidate mRNA targets of FBF-1 and FBF-2.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM31892</span>, <span class="grant">GM50942</span></p>
<p class="publication"><span class="title">RNA (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">11</span>;<span class="number">4</span>;<span class="pagerange">447-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15769874" rel="external">15769874</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1370734" rel="external">1370734</a>; DOI: <a href="http://dx.doi.org/10.1261%2Frna.7255805" rel="external">10.1261/rna.7255805</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the African trypanosome Trypanosoma brucei.</h4>
<p class="authors"><span class="author">Berriman M</span>, <span class="author">Ghedin E</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Blandin G</span>, <span class="author">Renauld H</span>, <span class="author">Bartholomeu DC</span>, <span class="author">Lennard NJ</span>, <span class="author">Caler E</span>, <span class="author">Hamlin NE</span>, <span class="author">Haas B</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Hannick L</span>, <span class="author">Aslett MA</span>, <span class="author">Shallom J</span>, <span class="author">Marcello L</span>, <span class="author">Hou L</span>, <span class="author">Wickstead B</span>, <span class="author">Alsmark UC</span>, <span class="author">Arrowsmith C</span>, <span class="author">Atkin RJ</span>, <span class="author">Barron AJ</span>, <span class="author">Bringaud F</span>, <span class="author">Brooks K</span>, <span class="author">Carrington M</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth TJ</span>, <span class="author">Churcher C</span>, <span class="author">Clark LN</span>, <span class="author">Corton CH</span>, <span class="author">Cronin A</span>, <span class="author">Davies RM</span>, <span class="author">Doggett J</span>, <span class="author">Djikeng A</span>, <span class="author">Feldblyum T</span>, <span class="author">Field MC</span>, <span class="author">Fraser A</span>, <span class="author">Goodhead I</span>, <span class="author">Hance Z</span>, <span class="author">Harper D</span>, <span class="author">Harris BR</span>, <span class="author">Hauser H</span>, <span class="author">Hostetler J</span>, <span class="author">Ivens A</span>, <span class="author">Jagels K</span>, <span class="author">Johnson D</span>, <span class="author">Johnson J</span>, <span class="author">Jones K</span>, <span class="author">Kerhornou AX</span>, <span class="author">Koo H</span>, <span class="author">Larke N</span>, <span class="author">Landfear S</span>, <span class="author">Larkin C</span>, <span class="author">Leech V</span>, <span class="author">Line A</span>, <span class="author">Lord A</span>, <span class="author">Macleod A</span>, <span class="author">Mooney PJ</span>, <span class="author">Moule S</span>, <span class="author">Martin DM</span>, <span class="author">Morgan GW</span>, <span class="author">Mungall K</span>, <span class="author">Norbertczak H</span>, <span class="author">Ormond D</span>, <span class="author">Pai G</span>, <span class="author">Peacock CS</span>, <span class="author">Peterson J</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rajandream MA</span>, <span class="author">Reitter C</span>, <span class="author">Salzberg SL</span>, <span class="author">Sanders M</span>, <span class="author">Schobel S</span>, <span class="author">Sharp S</span>, <span class="author">Simmonds M</span>, <span class="author">Simpson AJ</span>, <span class="author">Tallon L</span>, <span class="author">Turner CM</span>, <span class="author">Tait A</span>, <span class="author">Tivey AR</span>, <span class="author">Van Aken S</span>, <span class="author">Walker D</span>, <span class="author">Wanless D</span>, <span class="author">Wang S</span>, <span class="author">White B</span>, <span class="author">White O</span>, <span class="author">Whitehead S</span>, <span class="author">Woodward J</span>, <span class="author">Wortman J</span>, <span class="author">Adams MD</span>, <span class="author">Embley TM</span>, <span class="author">Gull K</span>, <span class="author">Ullu E</span>, <span class="author">Barry JD</span>, <span class="author">Fairlamb AH</span>, <span class="author">Opperdoes F</span>, <span class="author">Barrell BG</span>, <span class="author">Donelson JE</span>, <span class="author">Hall N</span>, <span class="author">Fraser CM</span>, <span class="author">Melville SE</span> and <span class="author">El-Sayed NM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%6D%62%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#98;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">African trypanosomes cause human sleeping sickness and livestock trypanosomiasis in sub-Saharan Africa. We present the sequence and analysis of the 11 megabase-sized chromosomes of Trypanosoma brucei. The 26-megabase genome contains 9068 predicted genes, including approximately 900 pseudogenes and approximately 1700 T. brucei-specific genes. Large subtelomeric arrays contain an archive of 806 variant surface glycoprotein (VSG) genes used by the parasite to evade the mammalian immune system. Most VSG genes are pseudogenes, which may be used to generate expressed mosaic genes by ectopic recombination. Comparisons of the cytoskeleton and endocytic trafficking systems with those of humans and other eukaryotic organisms reveal major differences. A comparison of metabolic pathways encoded by the genomes of T. brucei, T. cruzi, and Leishmania major reveals the least overall metabolic capability in T. brucei and the greatest in L. major. Horizontal transfer of genes of bacterial origin has contributed to some of the metabolic differences in these parasites, and a number of novel potential drug targets have been identified.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI43062</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5733</span>;<span class="pagerange">416-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16020726">16020726</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1112642">10.1126/science.1112642</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">XPACE4 is a localized pro-protein convertase required for mesoderm induction and the cleavage of specific TGFbeta proteins in Xenopus development.</h4>
<p class="authors"><span class="author">Birsoy B</span>, <span class="author">Berg L</span>, <span class="author">Williams PH</span>, <span class="author">Smith JC</span>, <span class="author">Wylie CC</span>, <span class="author">Christian JL</span> and <span class="author">Heasman J</span></p>
<p class="affiliation">Division of Developmental Biology, Cincinnati Children&#39;s Hospital Research Foundation, 3333 Burnet Avenue, Cincinnati, OH 45229, USA.</p><p class="abstract">XPACE4 is a member of the subtilisin/kexin family of pro-protein convertases. It cleaves many pro-proteins to release their active proteins, including members of the TGFbeta family of signaling molecules. Studies in mouse suggest it may have important roles in regulating embryonic tissue specification. Here, we examine the role of XPACE4 in Xenopus development and make three novel observations: first, XPACE4 is stored as maternal mRNA localized to the mitochondrial cloud and vegetal hemisphere of the oocyte; second, it is required for the endogenous mesoderm inducing activity of vegetal cells before gastrulation; and third, it has substrate-specific activity, cleaving Xnr1, Xnr2, Xnr3 and Vg1, but not Xnr5, Derriere or ActivinB pro-proteins. We conclude that maternal XPACE4 plays an important role in embryonic patterning by regulating the production of a subset of active mature TGFbeta proteins in specific sites.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD038272</span>, <span class="grant">HD042598</span>, <span class="grant">HD37976</span></p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">132</span>;<span class="number">3</span>;<span class="pagerange">591-602</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15634697" rel="external">15634697</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.01599" rel="external">10.1242/dev.01599</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the hypervariable region of the Salmonella enterica genome associated with tRNA(leuX).</h4>
<p class="authors"><span class="author">Bishop AL</span>, <span class="author">Baker S</span>, <span class="author">Jenks S</span>, <span class="author">Fookes M</span>, <span class="author">Gaora PO</span>, <span class="author">Pickard D</span>, <span class="author">Anjum M</span>, <span class="author">Farrar J</span>, <span class="author">Hien TT</span>, <span class="author">Ivens A</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%67%64%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#100;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">The divergence of Salmonella enterica and Escherichia coli is estimated to have occurred approximately 140 million years ago. Despite this evolutionary distance, the genomes of these two species still share extensive synteny and homology. However, there are significant differences between the two species in terms of genes putatively acquired via various horizontal transfer events. Here we report on the composition and distribution across the Salmonella genus of a chromosomal region designated SPI-10 in Salmonella enterica serovar Typhi and located adjacent to tRNA(leuX). We find that across the Salmonella genus the tRNA(leuX) region is a hypervariable hot spot for horizontal gene transfer; different isolates from the same S. enterica serovar can exhibit significant variation in this region. Many P4 phage, plasmid, and transposable element-associated genes are found adjacent to tRNA(leuX) in both Salmonella and E. coli, suggesting that these mobile genetic elements have played a major role in driving the variability of this region.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2005</span>;<span class="volume">187</span>;<span class="number">7</span>;<span class="pagerange">2469-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15774890" rel="external">15774890</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1065210" rel="external">1065210</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.187.7.2469-2482.2005" rel="external">10.1128/JB.187.7.2469-2482.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary comparison provides evidence for pathogenicity of RMRP mutations.</h4>
<p class="authors"><span class="author">Bonaf&eacute; L</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Unger S</span>, <span class="author">Greenberg CR</span>, <span class="author">Campos-Xavier BA</span>, <span class="author">Zankl A</span>, <span class="author">Ucla C</span>, <span class="author">Antonarakis SE</span>, <span class="author">Superti-Furga A</span> and <span class="author">Reymond A</span></p>
<p class="affiliation">Division of Molecular Pediatrics, Centre Hospitalier Universitaire Vaudois, Lausanne, Switzerland. <a href="mailto:%4C%75%69%73%61.%42%6F%6E%61%66%65%40%63%68%75%76.%63%68">&#76;&#117;&#105;&#115;&#97;&#46;&#66;&#111;&#110;&#97;&#102;&#101;&#64;&#99;&#104;&#117;&#118;&#46;&#99;&#104;</a></p><p class="abstract">Cartilage-hair hypoplasia (CHH) is a pleiotropic disease caused by recessive mutations in the RMRP gene that result in a wide spectrum of manifestations including short stature, sparse hair, metaphyseal dysplasia, anemia, immune deficiency, and increased incidence of cancer. Molecular diagnosis of CHH has implications for management, prognosis, follow-up, and genetic counseling of affected patients and their families. We report 20 novel mutations in 36 patients with CHH and describe the associated phenotypic spectrum. Given the high mutational heterogeneity (62 mutations reported to date), the high frequency of variations in the region (eight single nucleotide polymorphisms in and around RMRP), and the fact that RMRP is not translated into protein, prediction of mutation pathogenicity is difficult. We addressed this issue by a comparative genomic approach and aligned the genomic sequences of RMRP gene in the entire class of mammals. We found that putative pathogenic mutations are located in highly conserved nucleotides, whereas polymorphisms are located in non-conserved positions. We conclude that the abundance of variations in this small gene is remarkable and at odds with its high conservation through species; it is unclear whether these variations are caused by a high local mutation rate, a failure of repair mechanisms, or a relaxed selective pressure. The marked diversity of mutations in RMRP and the low homozygosity rate in our patient population indicate that CHH is more common than previously estimated, but may go unrecognized because of its variable clinical presentation. Thus, RMRP molecular testing may be indicated in individuals with isolated metaphyseal dysplasia, anemia, or immune dysregulation.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">1</span>;<span class="number">4</span>;<span class="pagerange">e47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16244706" rel="external">16244706</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1262189" rel="external">1262189</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0010047" rel="external">10.1371/journal.pgen.0010047</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple mutations in mouse Chd7 provide models for CHARGE syndrome.</h4>
<p class="authors"><span class="author">Bosman EA</span>, <span class="author">Penn AC</span>, <span class="author">Ambrose JC</span>, <span class="author">Kettleborough R</span>, <span class="author">Stemple DL</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Mouse ENU mutagenesis programmes have yielded a series of independent mutations on proximal chromosome 4 leading to dominant head-bobbing and circling behaviour due to truncations of the lateral semicircular canal of the inner ear. Here, we report the identification of mutations in the Chd7 gene in nine of these mutant alleles including six nonsense and three splice site mutations. The human CHD7 gene is known to be involved in CHARGE syndrome, which also shows inner ear malformations and a variety of other features with varying penetrance and appears to be due to frequent de novo mutation. We found widespread expression of Chd7 in early development of the mouse in organs affected in CHARGE syndrome including eye, olfactory epithelium, inner ear and vascular system. Closer inspection of heterozygous mutant mice revealed a range of defects with reduced penetrance, such as cleft palate, choanal atresia, septal defects of the heart, haemorrhages, prenatal death, vulva and clitoral defects and keratoconjunctivitis sicca. Many of these defects mimic the features of CHARGE syndrome. There were no obvious features of the gene that might make it more mutable than other genes. We conclude that the large number of mouse mutants and human de novo mutations may be due to the combination of the Chd7 gene being a large target and the fact that many heterozygous carriers of the mutations are viable individuals with a readily detectable phenotype.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14</span>;<span class="number">22</span>;<span class="pagerange">3463-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16207732">16207732</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi375">10.1093/hmg/ddi375</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A human-curated annotation of the Candida albicans genome.</h4>
<p class="authors"><span class="author">Braun BR</span>, <span class="author">van Het Hoog M</span>, <span class="author">d&#39;Enfert C</span>, <span class="author">Martchenko M</span>, <span class="author">Dungan J</span>, <span class="author">Kuo A</span>, <span class="author">Inglis DO</span>, <span class="author">Uhl MA</span>, <span class="author">Hogues H</span>, <span class="author">Berriman M</span>, <span class="author">Lorenz M</span>, <span class="author">Levitin A</span>, <span class="author">Oberholzer U</span>, <span class="author">Bachewich C</span>, <span class="author">Harcus D</span>, <span class="author">Marcil A</span>, <span class="author">Dignard D</span>, <span class="author">Iouk T</span>, <span class="author">Zito R</span>, <span class="author">Frangeul L</span>, <span class="author">Tekaia F</span>, <span class="author">Rutherford K</span>, <span class="author">Wang E</span>, <span class="author">Munro CA</span>, <span class="author">Bates S</span>, <span class="author">Gow NA</span>, <span class="author">Hoyer LL</span>, <span class="author">K&ouml;hler G</span>, <span class="author">Morschh&auml;user J</span>, <span class="author">Newport G</span>, <span class="author">Znaidi S</span>, <span class="author">Raymond M</span>, <span class="author">Turcotte B</span>, <span class="author">Sherlock G</span>, <span class="author">Costanzo M</span>, <span class="author">Ihmels J</span>, <span class="author">Berman J</span>, <span class="author">Sanglard D</span>, <span class="author">Agabian N</span>, <span class="author">Mitchell AP</span>, <span class="author">Johnson AD</span>, <span class="author">Whiteway M</span> and <span class="author">Nantel A</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of California, San Francisco, California, USA.</p><p class="abstract">Recent sequencing and assembly of the genome for the fungal pathogen Candida albicans used simple automated procedures for the identification of putative genes. We have reviewed the entire assembly, both by hand and with additional bioinformatic resources, to accurately map and describe 6,354 genes and to identify 246 genes whose original database entries contained sequencing errors (or possibly mutations) that affect their reading frame. Comparison with other fungal genomes permitted the identification of numerous fungus-specific genes that might be targeted for antifungal therapy. We also observed that, compared to other fungi, the protein-coding sequences in the C. albicans genome are especially rich in short sequence repeats. Finally, our improved annotation permitted a detailed analysis of several multigene families, and comparative genomic studies showed that C. albicans has a far greater catabolic range, encoding respiratory Complex 1, several novel oxidoreductases and ketone body degrading enzymes, malonyl-CoA and enoyl-CoA carriers, several novel amino acid degrading enzymes, a variety of secreted catabolic lipases and proteases, and numerous transporters to assimilate the resulting nutrients. The results of these efforts will ensure that the Candida research community has uniform and comprehensive genomic information for medical research as well as for future diagnostic and therapeutic applications.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">36-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16103911" rel="external">16103911</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1183520" rel="external">1183520</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0010001" rel="external">10.1371/journal.pgen.0010001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comprehensive DNA copy number profiling of meningioma using a chromosome 1 tiling path microarray identifies novel candidate tumor suppressor loci.</h4>
<p class="authors"><span class="author">Buckley PG</span>, <span class="author">Jarbo C</span>, <span class="author">Menzel U</span>, <span class="author">Mathiesen T</span>, <span class="author">Scott C</span>, <span class="author">Gregory SG</span>, <span class="author">Langford CF</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Rudbeck Laboratory, Department of Genetics and Pathology, Uppsala University, Uppsala, Sweden.</p><p class="abstract">Meningiomas are common neoplasms of the meninges lining of the central nervous system. Deletions of 1p have been established as important for the initiation and/or progression of meningioma. The rationale of this array-CGH study was to characterize copy number imbalances of chromosome 1 in meningioma, using a full-coverage genomic microarray containing 2,118 distinct measurement points. In total, 82 meningiomas were analyzed, making this the most detailed analysis of chromosome 1 in a comprehensive series of tumors. We detected a broad range of aberrations, such as deletions and/or gains of various sizes. Deletions were the predominant finding and ranged from monosomy to a 3.5-Mb terminal 1p homozygous deletion. Although multiple aberrations were observed across chromosome 1, every meningioma in which imbalances were detected harbored 1p deletions. Tumor heterogeneity was also observed in three recurrent meningiomas, which most likely reflects a progressive loss of chromosomal segments at different stages of tumor development. The distribution of aberrations supports the existence of at least four candidate loci on chromosome 1, which are important for meningioma tumorigenesis. In one of these regions, our results already allow the analysis of a number of candidate genes. In a large series of cases, we observed an association between the presence of segmental duplications and deletion breakpoints, which suggests their role in the generation of these tumor-specific aberrations. As 1p is the site of the genome most frequently affected by tumor-specific aberrations, our results indicate loci of general importance for cancer development and progression.</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2005</span>;<span class="volume">65</span>;<span class="number">7</span>;<span class="pagerange">2653-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15805262" rel="external">15805262</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-04-3651" rel="external">10.1158/0008-5472.CAN-04-3651</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of genetic aberrations on chromosome 22 outside the NF2 locus in schwannomatosis and neurofibromatosis type 2.</h4>
<p class="authors"><span class="author">Buckley PG</span>, <span class="author">Mantripragada KK</span>, <span class="author">D&iacute;az de St&aring;hl T</span>, <span class="author">Piotrowski A</span>, <span class="author">Hansson CM</span>, <span class="author">Kiss H</span>, <span class="author">Vetrie D</span>, <span class="author">Ernberg IT</span>, <span class="author">Nordenskj&ouml;ld M</span>, <span class="author">Bolund L</span>, <span class="author">Sainio M</span>, <span class="author">Rouleau GA</span>, <span class="author">Niimura M</span>, <span class="author">Wallace AJ</span>, <span class="author">Evans DG</span>, <span class="author">Grigelionis G</span>, <span class="author">Menzel U</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, Uppsala, Sweden.</p><p class="abstract">Schwannomatosis is characterized by multiple peripheral and cranial nerve schwannomas that occur in the absence of bilateral 8th cranial nerve schwannomas. The latter is the main diagnostic criterion of neurofibromatosis type 2 (NF2), which is a related but distinct disorder. The genetic factors underlying the differences between schwannomatosis and NF2 are poorly understood, although available evidence implicates chromosome 22 as the primary location of the gene(s) of interest. To investigate this, we comprehensively profiled the DNA copy number in samples from sporadic and familial schwannomatosis, NF2, and a large cohort of normal controls. Using a tiling-path chromosome 22 genomic array, we identified two candidate regions of copy number variation, which were further characterized by a PCR-based array with higher resolution. The latter approach allows the detection of minute alterations in total genomic DNA, with as little as 1.5 kb per measurement point of nonredundant sequence on the array. In DNA derived from peripheral blood from a schwannomatosis patient and a sporadic schwannoma sample, we detected rearrangements of the immunoglobulin lambda (IGL) locus, which is unlikely to be due to a B-cell specific somatic recombination of IGL. Analysis of normal controls indicated that these IGL rearrangements were restricted to schwannomatosis/schwannoma samples. In the second candidate region spanning GSTT1 and CABIN1 genes, we observed a frequent copy number polymorphism at the GSTT1 locus. We further describe missense mutations in the CABIN1 gene that are specific to samples from schwannomatosis and NF2 and make this gene a plausible candidate for contributing to the pathogenesis of these disorders.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2005</span>;<span class="volume">26</span>;<span class="number">6</span>;<span class="pagerange">540-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16287142" rel="external">16287142</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.20255" rel="external">10.1002/humu.20255</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plasmodium falciparum variant surface antigen expression patterns during malaria.</h4>
<p class="authors"><span class="author">Bull PC</span>, <span class="author">Berriman M</span>, <span class="author">Kyes S</span>, <span class="author">Quail MA</span>, <span class="author">Hall N</span>, <span class="author">Kortok MM</span>, <span class="author">Marsh K</span> and <span class="author">Newbold CI</span></p>
<p class="affiliation">Nuffield Department of Clinical Medicine, John Radcliffe Hospital, University of Oxford, Oxford, United Kingdom. <a href="mailto:%70%62%75%6C%6C%40%6B%69%6C%69%66%69.%6D%69%6D%63%6F%6D.%6E%65%74">&#112;&#98;&#117;&#108;&#108;&#64;&#107;&#105;&#108;&#105;&#102;&#105;&#46;&#109;&#105;&#109;&#99;&#111;&#109;&#46;&#110;&#101;&#116;</a></p><p class="abstract">The variant surface antigens expressed on Plasmodium falciparum-infected erythrocytes are potentially important targets of immunity to malaria and are encoded, at least in part, by a family of var genes, about 60 of which are present within every parasite genome. Here we use semi-conserved regions within short var gene sequence &quot;tags&quot; to make direct comparisons of var gene expression in 12 clinical parasite isolates from Kenyan children. A total of 1,746 var clones were sequenced from genomic and cDNA and assigned to one of six sequence groups using specific sequence features. The results show the following. (1) The relative numbers of genomic clones falling in each of the sequence groups was similar between parasite isolates and corresponded well with the numbers of genes found in the genome of a single, fully sequenced parasite isolate. In contrast, the relative numbers of cDNA clones falling in each group varied considerably between isolates. (2) Expression of sequences belonging to a relatively conserved group was negatively associated with the repertoire of variant surface antigen antibodies carried by the infected child at the time of disease, whereas expression of sequences belonging to another group was associated with the parasite &quot;rosetting&quot; phenotype, a well established virulence determinant. Our results suggest that information on the state of the host-parasite relationship in vivo can be provided by measurements of the differential expression of different var groups, and need only be defined by short stretches of sequence data.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">060678</span>, <span class="grant">631342</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2005</span>;<span class="volume">1</span>;<span class="number">3</span>;<span class="pagerange">e26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16304608">16304608</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1287908">1287908</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.0010026">10.1371/journal.ppat.0010026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plasmodium falciparum antigenic variation: relationships between in vivo selection, acquired antibody response, and disease severity.</h4>
<p class="authors"><span class="author">Bull PC</span>, <span class="author">Pain A</span>, <span class="author">Ndungu FM</span>, <span class="author">Kinyanjui SM</span>, <span class="author">Roberts DJ</span>, <span class="author">Newbold CI</span> and <span class="author">Marsh K</span></p>
<p class="affiliation">Nuffield Department of Clinical Medicine, John Radcliffe Hospital, University of Oxford, Oxford, United Kingdom. <a href="mailto:%70%62%75%6C%6C%40%6B%69%6C%69%66%69.%6D%69%6D%63%6F%6D.%6E%65%74">&#112;&#98;&#117;&#108;&#108;&#64;&#107;&#105;&#108;&#105;&#102;&#105;&#46;&#109;&#105;&#109;&#99;&#111;&#109;&#46;&#110;&#101;&#116;</a></p><p class="abstract"><strong>Background</strong>: Variant surface antigens (VSA) on Plasmodium falciparum-infected erythrocytes are potentially important targets of immunity to malaria. We previously identified a VSA phenotype--VSA with a high frequency of antibody recognition (VSA(FoRH))--that is associated with young host age and severe malaria. We hypothesized that VSA(FoRH) are positively selected by host molecules such as intercellular adhesion molecule 1 (ICAM1) and CD36 and dominate in the absence of an effective immune response. Here, we assessed, in 115 Kenyan children, the potential role played by in vivo selection pressures in either favoring or selecting against VSA(FoRH) among parasites that cause malaria.</p><p class="abstract"><strong>Methods</strong>: We tested for associations between VSA(FoRH) and (1) the repertoire of VSA antibodies carried by children at the time of acute malaria and (2) polymorphisms in ICAM1 (K29M) and CD36 (T188G) that could potentially reduce the positive selection of VSA(FoRH).</p><p class="abstract"><strong>Results</strong>: An expected negative association between VSA antibody repertoire and VSA(FoRH) was observed in children with nonsevere malaria. However, this association did not extend to children with severe malaria, many of whom apparently had well-developed VSA antibody responses despite being infected by parasites expressing VSA(FoRH). There was no evidence for involvement of CD36 or ICAM1 in positive selection of VSA(FoRH). On the contrary, a weak positive association between carriage of the CD36 (T188G) allele and VSA(FoRH) was observed in children with severe malaria.</p><p class="abstract"><strong>Conclusion</strong>: The association between the VSA(FoRH) parasite phenotype and severe malaria cannot be explained simply in terms of the total repertoire of VSA antibodies carried at the time of acute disease.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2005</span>;<span class="volume">192</span>;<span class="number">6</span>;<span class="pagerange">1119-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16107968" rel="external">16107968</a>; DOI: <a href="http://dx.doi.org/10.1086%2F432761" rel="external">10.1086/432761</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Universal DNA primers amplify bacterial DNA from human fetal membranes and link Fusobacterium nucleatum with prolonged preterm membrane rupture.</h4>
<p class="authors"><span class="author">Cahill RJ</span>, <span class="author">Tan S</span>, <span class="author">Dougan G</span>, <span class="author">O&#39;Gaora P</span>, <span class="author">Pickard D</span>, <span class="author">Kennea N</span>, <span class="author">Sullivan MH</span>, <span class="author">Feldman RG</span> and <span class="author">Edwards AD</span></p>
<p class="affiliation">Department of Biological Sciences, Centre for Molecular Microbiology and Infection, Flowers Building, London, UK.</p><p class="abstract">A large number of bacterial species have been identified in fetal membranes after preterm labour (PTL) associated with intrauterine infection by microbiological culture. In this study, we have investigated a molecular and bioinformatic approach to organism identification which surmounts the need for specific and diverse microbiological culture conditions required by conventional methods. Samples of fetal membranes were taken from 37 preterm infants, and 6 normal term controls delivered by caesarean section, in which bacteria had been detected by in situ hybridization of 16S ribosomal RNA using a generic probe. Degenerate primers were designed to amplify bacterial 16S ribosomal DNA by PCR and used to amplify bacterial DNA from human fetal membranes. Amplicons were cloned, sequenced and bacteria were identified bioinformatically by comparison of sequences with known bacterial DNA genomes. In situ hybridization using an organism specific probe was then used to confirm the presence of the commonest identified organism in tissue samples. Bacterial DNA amplified from 15/43 samples, all from preterm deliveries, and the bioinformatic approach identified organisms in all cases. Multiple bacteria were identified including Mycoplasma hominis, Pasturella multocida, Pseudomonas PH1, Escherichia coli and Prevotella bivia. The commonest organism Fusobacterium nucleatum was found in 9/15 (60%) of samples. Ten of the 12 samples obtained after prolonged membrane rupture were positive for bacterial DNA, and 7 of these (70%) contained DNA from F. nucleatum. Bacteria from fetal membranes may be identified by molecular and bioinformatic methods. Further work is warranted to investigate the apparent linkage between F. nucleatum, fetal membrane rupture and preterm delivery.</p>
<p class="publication"><span class="title">Molecular human reproduction</span>&nbsp;<span class="year">2005</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">761-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16254004" rel="external">16254004</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolehr%2Fgah234" rel="external">10.1093/molehr/gah234</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of model malaria parasites, and comparative genomics.</h4>
<p class="authors"><span class="author">Carlton J</span>, <span class="author">Silva J</span> and <span class="author">Hall N</span></p>
<p class="affiliation">The Institute for Genomic Research, Rockville, MD 20850, USA. <a href="mailto:%63%61%72%6C%74%6F%6E%40%74%69%67%72.%6F%72%67">&#99;&#97;&#114;&#108;&#116;&#111;&#110;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The field of comparative genomics of malaria parasites has recently come of age with the completion of the whole genome sequences of the human malaria parasite Plasmodium falciparum and a rodent malaria model, Plasmodium yoelii yoelii. With several other genome sequencing projects of different model and human malaria parasite species underway, comparing genomes from multiple species has necessitated the development of improved informatics tools and analyses. Results from initial comparative analyses reveal striking conservation of gene synteny between malaria species within conserved chromosome cores, in contrast to reduced homology within subtelomeric regions, in line with previous findings on a smaller scale. Genes that elicit a host immune response are frequently found to be species-specific, although a large variant multigene family is common to many rodent malaria species and Plasmodium vivax. Sequence alignment of syntenic regions from multiple species has revealed the similarity between species in coding regions to be high relative to non-coding regions, and phylogenetic footprinting studies promise to reveal conserved motifs in the latter. Comparison of non-synonymous substitution rates between orthologous genes is proving a powerful technique for identifying genes under selection pressure, and may be useful for vaccine design. This is a stimulating time for comparative genomics of model and human malaria parasites, which promises to produce useful results for the development of antimalarial drugs and vaccines.</p>
<p class="publication"><span class="title">Current issues in molecular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">23-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15580778" rel="external">15580778</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The transcriptional landscape of the mammalian genome.</h4>
<p class="authors"><span class="author">Carninci P</span>, <span class="author">Kasukawa T</span>, <span class="author">Katayama S</span>, <span class="author">Gough J</span>, <span class="author">Frith MC</span>, <span class="author">Maeda N</span>, <span class="author">Oyama R</span>, <span class="author">Ravasi T</span>, <span class="author">Lenhard B</span>, <span class="author">Wells C</span>, <span class="author">Kodzius R</span>, <span class="author">Shimokawa K</span>, <span class="author">Bajic VB</span>, <span class="author">Brenner SE</span>, <span class="author">Batalov S</span>, <span class="author">Forrest AR</span>, <span class="author">Zavolan M</span>, <span class="author">Davis MJ</span>, <span class="author">Wilming LG</span>, <span class="author">Aidinis V</span>, <span class="author">Allen JE</span>, <span class="author">Ambesi-Impiombato A</span>, <span class="author">Apweiler R</span>, <span class="author">Aturaliya RN</span>, <span class="author">Bailey TL</span>, <span class="author">Bansal M</span>, <span class="author">Baxter L</span>, <span class="author">Beisel KW</span>, <span class="author">Bersano T</span>, <span class="author">Bono H</span>, <span class="author">Chalk AM</span>, <span class="author">Chiu KP</span>, <span class="author">Choudhary V</span>, <span class="author">Christoffels A</span>, <span class="author">Clutterbuck DR</span>, <span class="author">Crowe ML</span>, <span class="author">Dalla E</span>, <span class="author">Dalrymple BP</span>, <span class="author">de Bono B</span>, <span class="author">Della Gatta G</span>, <span class="author">di Bernardo D</span>, <span class="author">Down T</span>, <span class="author">Engstrom P</span>, <span class="author">Fagiolini M</span>, <span class="author">Faulkner G</span>, <span class="author">Fletcher CF</span>, <span class="author">Fukushima T</span>, <span class="author">Furuno M</span>, <span class="author">Futaki S</span>, <span class="author">Gariboldi M</span>, <span class="author">Georgii-Hemming P</span>, <span class="author">Gingeras TR</span>, <span class="author">Gojobori T</span>, <span class="author">Green RE</span>, <span class="author">Gustincich S</span>, <span class="author">Harbers M</span>, <span class="author">Hayashi Y</span>, <span class="author">Hensch TK</span>, <span class="author">Hirokawa N</span>, <span class="author">Hill D</span>, <span class="author">Huminiecki L</span>, <span class="author">Iacono M</span>, <span class="author">Ikeo K</span>, <span class="author">Iwama A</span>, <span class="author">Ishikawa T</span>, <span class="author">Jakt M</span>, <span class="author">Kanapin A</span>, <span class="author">Katoh M</span>, <span class="author">Kawasawa Y</span>, <span class="author">Kelso J</span>, <span class="author">Kitamura H</span>, <span class="author">Kitano H</span>, <span class="author">Kollias G</span>, <span class="author">Krishnan SP</span>, <span class="author">Kruger A</span>, <span class="author">Kummerfeld SK</span>, <span class="author">Kurochkin IV</span>, <span class="author">Lareau LF</span>, <span class="author">Lazarevic D</span>, <span class="author">Lipovich L</span>, <span class="author">Liu J</span>, <span class="author">Liuni S</span>, <span class="author">McWilliam S</span>, <span class="author">Madan Babu M</span>, <span class="author">Madera M</span>, <span class="author">Marchionni L</span>, <span class="author">Matsuda H</span>, <span class="author">Matsuzawa S</span>, <span class="author">Miki H</span>, <span class="author">Mignone F</span>, <span class="author">Miyake S</span>, <span class="author">Morris K</span>, <span class="author">Mottagui-Tabar S</span>, <span class="author">Mulder N</span>, <span class="author">Nakano N</span>, <span class="author">Nakauchi H</span>, <span class="author">Ng P</span>, <span class="author">Nilsson R</span>, <span class="author">Nishiguchi S</span>, <span class="author">Nishikawa S</span>, <span class="author">Nori F</span>, <span class="author">Ohara O</span>, <span class="author">Okazaki Y</span>, <span class="author">Orlando V</span>, <span class="author">Pang KC</span>, <span class="author">Pavan WJ</span>, <span class="author">Pavesi G</span>, <span class="author">Pesole G</span>, <span class="author">Petrovsky N</span>, <span class="author">Piazza S</span>, <span class="author">Reed J</span>, <span class="author">Reid JF</span>, <span class="author">Ring BZ</span>, <span class="author">Ringwald M</span>, <span class="author">Rost B</span>, <span class="author">Ruan Y</span>, <span class="author">Salzberg SL</span>, <span class="author">Sandelin A</span>, <span class="author">Schneider C</span>, <span class="author">Sch&ouml;nbach C</span>, <span class="author">Sekiguchi K</span>, <span class="author">Semple CA</span>, <span class="author">Seno S</span>, <span class="author">Sessa L</span>, <span class="author">Sheng Y</span>, <span class="author">Shibata Y</span>, <span class="author">Shimada H</span>, <span class="author">Shimada K</span>, <span class="author">Silva D</span>, <span class="author">Sinclair B</span>, <span class="author">Sperling S</span>, <span class="author">Stupka E</span>, <span class="author">Sugiura K</span>, <span class="author">Sultana R</span>, <span class="author">Takenaka Y</span>, <span class="author">Taki K</span>, <span class="author">Tammoja K</span>, <span class="author">Tan SL</span>, <span class="author">Tang S</span>, <span class="author">Taylor MS</span>, <span class="author">Tegner J</span>, <span class="author">Teichmann SA</span>, <span class="author">Ueda HR</span>, <span class="author">van Nimwegen E</span>, <span class="author">Verardo R</span>, <span class="author">Wei CL</span>, <span class="author">Yagi K</span>, <span class="author">Yamanishi H</span>, <span class="author">Zabarovsky E</span>, <span class="author">Zhu S</span>, <span class="author">Zimmer A</span>, <span class="author">Hide W</span>, <span class="author">Bult C</span>, <span class="author">Grimmond SM</span>, <span class="author">Teasdale RD</span>, <span class="author">Liu ET</span>, <span class="author">Brusic V</span>, <span class="author">Quackenbush J</span>, <span class="author">Wahlestedt C</span>, <span class="author">Mattick JS</span>, <span class="author">Hume DA</span>, <span class="author">Kai C</span>, <span class="author">Sasaki D</span>, <span class="author">Tomaru Y</span>, <span class="author">Fukuda S</span>, <span class="author">Kanamori-Katayama M</span>, <span class="author">Suzuki M</span>, <span class="author">Aoki J</span>, <span class="author">Arakawa T</span>, <span class="author">Iida J</span>, <span class="author">Imamura K</span>, <span class="author">Itoh M</span>, <span class="author">Kato T</span>, <span class="author">Kawaji H</span>, <span class="author">Kawagashira N</span>, <span class="author">Kawashima T</span>, <span class="author">Kojima M</span>, <span class="author">Kondo S</span>, <span class="author">Konno H</span>, <span class="author">Nakano K</span>, <span class="author">Ninomiya N</span>, <span class="author">Nishio T</span>, <span class="author">Okada M</span>, <span class="author">Plessy C</span>, <span class="author">Shibata K</span>, <span class="author">Shiraki T</span>, <span class="author">Suzuki S</span>, <span class="author">Tagami M</span>, <span class="author">Waki K</span>, <span class="author">Watahiki A</span>, <span class="author">Okamura-Oho Y</span>, <span class="author">Suzuki H</span>, <span class="author">Kawai J</span>, <span class="author">Hayashizaki Y</span>, <span class="author">FANTOM Consortium</span> and <span class="author">RIKEN Genome Exploration Research Group and Genome Science Group (Genome Network Project Core Group)</span></p><p class="abstract">This study describes comprehensive polling of transcription start and termination sites and analysis of previously unidentified full-length complementary DNAs derived from the mouse genome. We identify the 5&#39; and 3&#39; boundaries of 181,047 transcripts with extensive variation in transcripts arising from alternative promoter usage, splicing, and polyadenylation. There are 16,247 new mouse protein-coding transcripts, including 5154 encoding previously unidentified proteins. Genomic mapping of the transcriptome reveals transcriptional forests, with overlapping transcription on both strands, separated by deserts in which few transcripts are observed. The data provide a comprehensive platform for the comparative analysis of mammalian transcriptional regulation in differentiation and development.</p>
<p class="grants">Funded by: Telethon: <span class="grant">TGM03P17</span>, <span class="grant">TGM06S01</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5740</span>;<span class="pagerange">1559-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16141072">16141072</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1112014">10.1126/science.1112014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microarray-based comparative genomic analyses of the human malaria parasite Plasmodium falciparum using Affymetrix arrays.</h4>
<p class="authors"><span class="author">Carret CK</span>, <span class="author">Horrocks P</span>, <span class="author">Konfortov B</span>, <span class="author">Winzeler E</span>, <span class="author">Qureshi M</span>, <span class="author">Newbold C</span> and <span class="author">Ivens A</span></p>
<p class="affiliation">Pathogen Microarrays Group, The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%63%6B%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#107;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Microarray-based comparative genomic hybridization (CGH) provides a powerful tool for whole genome analyses and the rapid detection of genomic variation that underlies virulence and disease. In the field of Plasmodium research, many of the parasite genomes that one might wish to study in a high throughput manner are not laboratory clones, but clinical isolates. One of the key limitations to the use of clinical samples in CGH, however, is the miniscule amounts of genomic DNA available. Here we describe the successful application of multiple displacement amplification (MDA), a non-PCR-based amplification method that exhibits clear advantages over all other currently available methods. Using MDA, CGH was performed on a panel of NF54 and IT/FCR3 clones, identifying previously published deletions on chromosomes 2 and 9 as well as polymorphism in genes associated with disease pathology.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2005</span>;<span class="volume">144</span>;<span class="number">2</span>;<span class="pagerange">177-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16174539" rel="external">16174539</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2005.08.010" rel="external">10.1016/j.molbiopara.2005.08.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">JAE: Jemboss Alignment Editor.</h4>
<p class="authors"><span class="author">Carver TJ</span> and <span class="author">Mullan LJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%74%6A%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#106;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">One of the most basic methods of understanding the biological significance of a sequence is to produce an alignment with related sequences. A vital aspect of correctly aligning sequences is to apply biological intuition through manual editing of an alignment produced by multiple-sequence alignment software. As part of the European Molecular Biology Open Source Software Suite (EMBOSS), a new alignment editor in the Jemboss package is freely available for download. The Jemboss Alignment Editor (JAE) incorporates standard methods of editing, and colouring residues and nucleotides to highlight important regions of interest. JAE also makes use of scoring matrices (PAM and BLOSUM), selected by the user, to display regions of high degrees of similarity and identity. Other tools include the ability to calculate a consensus, a consensus plot (using a selected scoring matrix) and pairwise identities. AVAILABILITY: The JAE can be launched from the webpage (http://emboss.sourceforge.net/Jemboss/).</p>
<p class="publication"><span class="title">Applied bioinformatics</span>&nbsp;<span class="year">2005</span>;<span class="volume">4</span>;<span class="number">2</span>;<span class="pagerange">151-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16128618" rel="external">16128618</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ACT: the Artemis Comparison Tool.</h4>
<p class="authors"><span class="author">Carver TJ</span>, <span class="author">Rutherford KM</span>, <span class="author">Berriman M</span>, <span class="author">Rajandream MA</span>, <span class="author">Barrell BG</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%72%74%65%6D%69%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#116;&#101;&#109;&#105;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Artemis Comparison Tool (ACT) allows an interactive visualisation of comparisons between complete genome sequences and associated annotations. The comparison data can be generated with several different programs; BLASTN, TBLASTX or Mummer comparisons between genomic DNA sequences, or orthologue tables generated by reciprocal FASTA comparison between protein sets. It is possible to identify regions of similarity, insertions and rearrangements at any level from the whole genome to base-pair differences. ACT uses Artemis components to display the sequences and so inherits powerful searching and analysis tools. ACT is part of the Artemis distribution and is similarly open source, written in Java and can run on any Java enabled platform, including UNIX, Macintosh and Windows.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">16</span>;<span class="pagerange">3422-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15976072">15976072</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti553">10.1093/bioinformatics/bti553</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GI genomes.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Claesson MJ</span>, <span class="author">Sebaihia M</span> and <span class="author">Thomson NR</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">368-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15871171">15871171</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1153">10.1038/nrmicro1153</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Extensive DNA inversions in the B. fragilis genome control variable gene expression.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Patrick S</span>, <span class="author">Crossman LC</span>, <span class="author">Blakely G</span>, <span class="author">Abratt V</span>, <span class="author">Lennard N</span>, <span class="author">Poxton I</span>, <span class="author">Duerden B</span>, <span class="author">Harris B</span>, <span class="author">Quail MA</span>, <span class="author">Barron A</span>, <span class="author">Clark L</span>, <span class="author">Corton C</span>, <span class="author">Doggett J</span>, <span class="author">Holden MT</span>, <span class="author">Larke N</span>, <span class="author">Line A</span>, <span class="author">Lord A</span>, <span class="author">Norbertczak H</span>, <span class="author">Ormond D</span>, <span class="author">Price C</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Woodward J</span>, <span class="author">Barrell B</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">The obligately anaerobic bacterium Bacteroides fragilis, an opportunistic pathogen and inhabitant of the normal human colonic microbiota, exhibits considerable within-strain phase and antigenic variation of surface components. The complete genome sequence has revealed an unusual breadth (in number and in effect) of DNA inversion events that potentially control expression of many different components, including surface and secreted components, regulatory molecules, and restriction-modification proteins. Invertible promoters of two different types (12 group 1 and 11 group 2) were identified. One group has inversion crossover (fix) sites similar to the hix sites of Salmonella typhimurium. There are also four independent intergenic shufflons that potentially alter the expression and function of varied genes. The composition of the 10 different polysaccharide biosynthesis gene clusters identified (7 with associated invertible promoters) suggests a mechanism of synthesis similar to the O-antigen capsules of Escherichia coli.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">307</span>;<span class="number">5714</span>;<span class="pagerange">1463-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15746427">15746427</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1107008">10.1126/science.1107008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">VFDB: a reference database for bacterial virulence factors.</h4>
<p class="authors"><span class="author">Chen L</span>, <span class="author">Yang J</span>, <span class="author">Yu J</span>, <span class="author">Yao Z</span>, <span class="author">Sun L</span>, <span class="author">Shen Y</span> and <span class="author">Jin Q</span></p>
<p class="affiliation">State Key Laboratory for Molecular Virology and Genetic Engineering, Beijing 100052, China.</p><p class="abstract">Bacterial pathogens continue to impose a major threat to public health worldwide in the 21st century. Intensified studies on bacterial pathogenesis have greatly expanded our knowledge about the mechanisms of the disease processes at the molecular level over the last decades. To facilitate future research, it becomes necessary to form a database collectively presenting the virulence factors (VFs) of various medical significant bacterial pathogens. The aim of virulence factor database (VFDB) (http://www.mgc.ac.cn/VFs/) is to provide such a source for scientists to rapidly access to current knowledge about VFs from various bacterial pathogens. VFDB is comprehensive and user-friendly. One can search VFDB by browsing each genus or by typing keywords. Furthermore, a BLAST search tool against all known VF-related genes is also available. VFDB provides a unified gateway to store, search, retrieve and update information about VFs from various bacterial pathogens.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Database issue</span>;<span class="pagerange">D325-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15608208" rel="external">15608208</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC539962" rel="external">539962</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki008" rel="external">10.1093/nar/gki008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gamma-glutamyl carboxylase (GGCX) microsatellite and warfarin dosing.</h4>
<p class="authors"><span class="author">Chen LY</span>, <span class="author">Eriksson N</span>, <span class="author">Gwilliam R</span>, <span class="author">Bentley D</span>, <span class="author">Deloukas P</span> and <span class="author">Wadelius M</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2005</span>;<span class="volume">106</span>;<span class="number">10</span>;<span class="pagerange">3673-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16267263" rel="external">16267263</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2005-04-1711" rel="external">10.1182/blood-2005-04-1711</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: a comprehensive data resource for Caenorhabditis biology and genomics.</h4>
<p class="authors"><span class="author">Chen N</span>, <span class="author">Harris TW</span>, <span class="author">Antoshechkin I</span>, <span class="author">Bastiani C</span>, <span class="author">Bieri T</span>, <span class="author">Blasiar D</span>, <span class="author">Bradnam K</span>, <span class="author">Canaran P</span>, <span class="author">Chan J</span>, <span class="author">Chen CK</span>, <span class="author">Chen WJ</span>, <span class="author">Cunningham F</span>, <span class="author">Davis P</span>, <span class="author">Kenny E</span>, <span class="author">Kishore R</span>, <span class="author">Lawson D</span>, <span class="author">Lee R</span>, <span class="author">Muller HM</span>, <span class="author">Nakamura C</span>, <span class="author">Pai S</span>, <span class="author">Ozersky P</span>, <span class="author">Petcherski A</span>, <span class="author">Rogers A</span>, <span class="author">Sabo A</span>, <span class="author">Schwarz EM</span>, <span class="author">Van Auken K</span>, <span class="author">Wang Q</span>, <span class="author">Durbin R</span>, <span class="author">Spieth J</span>, <span class="author">Sternberg PW</span> and <span class="author">Stein LD</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY 11724, USA. <a href="mailto:%63%68%65%6E%6E%40%63%73%68%6C.%6F%72%67">&#99;&#104;&#101;&#110;&#110;&#64;&#99;&#115;&#104;&#108;&#46;&#111;&#114;&#103;</a></p><p class="abstract">WormBase (http://www.wormbase.org), the model organism database for information about Caenorhabditis elegans and related nematodes, continues to expand in breadth and depth. Over the past year, WormBase has added multiple large-scale datasets including SAGE, interactome, 3D protein structure datasets and NCBI KOGs. To accommodate this growth, the International WormBase Consortium has improved the user interface by adding new features to aid in navigation, visualization of large-scale datasets, advanced searching and data mining. Internally, we have restructured the database models to rationalize the representation of genes and to prepare the system to accept the genome sequences of three additional Caenorhabditis species over the coming year.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Database issue</span>;<span class="pagerange">D383-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15608221" rel="external">15608221</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540020" rel="external">540020</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki066" rel="external">10.1093/nar/gki066</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Defining the orientation of the tandem fusions that occurred during the evolution of Indian muntjac chromosomes by BAC mapping.</h4>
<p class="authors"><span class="author">Chi JX</span>, <span class="author">Huang L</span>, <span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Su B</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Key Laboratory of Cellular and Molecular Evolution, Kunming Institute of Zoology, The Chinese Academy of Sciences, Kunming, Yunnan 650223, People&#39;s Republic of China.</p><p class="abstract">The Indian muntjac (Muntiacus muntjak vaginalis) has a karyotype of 2n=6 in the female and 7 in the male, the karyotypic evolution of which through extensive tandem fusions and several centric fusions has been well-documented by recent molecular cytogenetic studies. In an attempt to define the fusion orientations of conserved chromosomal segments and the molecular mechanisms underlying the tandem fusions, we have constructed a highly redundant (more than six times of whole genome coverage) bacterial artificial chromosome (BAC) library of Indian muntjac. The BAC library contains 124,800 clones with no chromosome bias and has an average insert DNA size of 120 kb. A total of 223 clones have been mapped by fluorescent in situ hybridization onto the chromosomes of both Indian muntjac and Chinese muntjac and a high-resolution comparative map has been established. Our mapping results demonstrate that all tandem fusions that occurred during the evolution of Indian muntjac karyotype from the acrocentric 2n=70 hypothetical ancestral karyotype are centromere-telomere (head-tail) fusions.</p>
<p class="publication"><span class="title">Chromosoma</span>&nbsp;<span class="year">2005</span>;<span class="volume">114</span>;<span class="number">3</span>;<span class="pagerange">167-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16010580" rel="external">16010580</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00412-005-0004-x" rel="external">10.1007/s00412-005-0004-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nematode genome evolution.</h4>
<p class="authors"><span class="author">Coghlan A</span></p>
<p class="affiliation">Conway Institute of Biomolecular and Biomedical Research, University College Dublin, Dublin 4, Ireland. <a href="mailto:%61%6C%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Nematodes are the most abundant type of animal on earth, and live in hot springs, polar ice, soil, fresh and salt water, and as parasites of plants, vertebrates, insects, and other nematodes. This extraordinary ability to adapt, which hints at an underlying genetic plasticity, has long fascinated biologists. The fully sequenced genomes of Caenorhabditis elegans and Caenorhabditis briggsae, and ongoing sequencing projects for eight other nematodes, provide an exciting opportunity to investigate the genomic changes that have enabled nematodes to invade many different habitats. Analyses of the C. elegans and C. briggsae genomes suggest that these include major changes in gene content; as well as in chromosome number, structure and size. Here I discuss how the data set of ten genomes will be ideal for tackling questions about nematode evolution, as well as questions relevant to all eukaryotes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">WormBook : the online review of C. elegans biology</span>&nbsp;<span class="year">2005</span>;<span class="pagerange">1-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18050393" rel="external">18050393</a>; DOI: <a href="http://dx.doi.org/10.1895%2Fwormbook.1.15.1" rel="external">10.1895/wormbook.1.15.1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome evolution in eukaryotes: a multi-kingdom perspective.</h4>
<p class="authors"><span class="author">Coghlan A</span>, <span class="author">Eichler EE</span>, <span class="author">Oliver SG</span>, <span class="author">Paterson AH</span> and <span class="author">Stein L</span></p>
<p class="affiliation">Conway Institute, University College Dublin, Belfield, Dublin 4, Ireland.</p><p class="abstract">In eukaryotes, chromosomal rearrangements, such as inversions, translocations and duplications, are common and range from part of a gene to hundreds of genes. Lineage-specific patterns are also seen: translocations are rare in dipteran flies, and angiosperm genomes seem prone to polyploidization. In most eukaryotes, there is a strong association between rearrangement breakpoints and repeat sequences. Current data suggest that some repeats promoted rearrangements via non-allelic homologous recombination, for others the association might not be causal but reflects the instability of particular genomic regions. Rearrangement polymorphisms in eukaryotes are correlated with phenotypic differences, so are thought to confer varying fitness in different habitats. Some seem to be under positive selection because they either trap favorable allele combinations together or alter the expression of nearby genes. There is little evidence that chromosomal rearrangements cause speciation, but they probably intensify reproductive isolation between species that have formed by another route.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG02639</span>; NIGMS NIH HHS: <span class="grant">GM58815</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">673-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16242204">16242204</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2005.09.009">10.1016/j.tig.2005.09.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proteomic analysis of in vivo phosphorylated synaptic proteins.</h4>
<p class="authors"><span class="author">Collins MO</span>, <span class="author">Yu L</span>, <span class="author">Coba MP</span>, <span class="author">Husi H</span>, <span class="author">Campuzano I</span>, <span class="author">Blackstock WP</span>, <span class="author">Choudhary JS</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Division of Neuroscience, University of Edinburgh, Edinburgh EH8 9JZ, UK.</p><p class="abstract">In the nervous system, protein phosphorylation is an essential feature of synaptic function. Although protein phosphorylation is known to be important for many synaptic processes and in disease, little is known about global phosphorylation of synaptic proteins. Heterogeneity and low abundance make protein phosphorylation analysis difficult, particularly for mammalian tissue samples. Using a new approach, combining both protein and peptide immobilized metal affinity chromatography and mass spectrometry data acquisition strategies, we have produced the first large scale map of the mouse synapse phosphoproteome. We report over 650 phosphorylation events corresponding to 331 sites (289 have been unambiguously assigned), 92% of which are novel. These represent 79 proteins, half of which are novel phosphoproteins, and include several highly phosphorylated proteins such as MAP1B (33 sites) and Bassoon (30 sites). An additional 149 candidate phosphoproteins were identified by profiling the composition of the protein immobilized metal affinity chromatography enrichment. All major synaptic protein classes were observed, including components of important pre- and postsynaptic complexes as well as low abundance signaling proteins. Bioinformatic and in vitro phosphorylation assays of peptide arrays suggest that a small number of kinases phosphorylate many proteins and that each substrate is phosphorylated by many kinases. These data substantially increase existing knowledge of synapse protein phosphorylation and support a model where the synapse phosphoproteome is functionally organized into a highly interconnected signaling network.</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2005</span>;<span class="volume">280</span>;<span class="number">7</span>;<span class="pagerange">5972-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15572359">15572359</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M411220200">10.1074/jbc.M411220200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Robust enrichment of phosphorylated species in complex mixtures by sequential protein and peptide metal-affinity chromatography and analysis by tandem mass spectrometry.</h4>
<p class="authors"><span class="author">Collins MO</span>, <span class="author">Yu L</span>, <span class="author">Husi H</span>, <span class="author">Blackstock WP</span>, <span class="author">Choudhary JS</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK. <a href="mailto:%6D%6F%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#111;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Reversible protein phosphorylation mediated by kinases, phosphatases, and regulatory molecules is an essential mechanism of signal transduction in living cells. Although phosphorylation is the most intensively studied of the several hundred known posttranslational modifications on proteins, until recently the rate of identification of phosphorylation sites has remained low. The use of tandem mass spectrometry has greatly accelerated the identification of phosphorylation sites, although progress was limited by difficulties in phosphoresidue enrichment techniques. We have improved upon existing immobilized metal-affinity chromatography (IMAC) techniques for capturing phosphopeptides, to selectively purify phosphoproteins from complex mixtures. Combinations of phosphoprotein and phosphopeptide enrichment were more effective than current single phosphopeptide purification approaches. We have also implemented iterative mass spectrometry-based scanning techniques to improve detection of phosphorylated peptides in these enriched samples. Here, we provide detailed instructions for implementing and validating these methods together with analysis by tandem mass spectrometry for the study of phosphorylation at the mammalian synapse. This strategy should be widely applicable to the characterization of protein phosphorylation in diverse tissues, organelles, and in cell culture.</p>
<p class="publication"><span class="title">Science&#39;s STKE : signal transduction knowledge environment</span>&nbsp;<span class="year">2005</span>;<span class="volume">2005</span>;<span class="number">298</span>;<span class="pagerange">pl6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16118397" rel="external">16118397</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fstke.2982005pl6" rel="external">10.1126/stke.2982005pl6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the heartwater agent Ehrlichia ruminantium contains multiple tandem repeats of actively variable copy number.</h4>
<p class="authors"><span class="author">Collins NE</span>, <span class="author">Liebenberg J</span>, <span class="author">de Villiers EP</span>, <span class="author">Brayton KA</span>, <span class="author">Louw E</span>, <span class="author">Pretorius A</span>, <span class="author">Faber FE</span>, <span class="author">van Heerden H</span>, <span class="author">Josemans A</span>, <span class="author">van Kleef M</span>, <span class="author">Steyn HC</span>, <span class="author">van Strijp MF</span>, <span class="author">Zweygarth E</span>, <span class="author">Jongejan F</span>, <span class="author">Maillard JC</span>, <span class="author">Berthier D</span>, <span class="author">Botha M</span>, <span class="author">Joubert F</span>, <span class="author">Corton CH</span>, <span class="author">Thomson NR</span>, <span class="author">Allsopp MT</span> and <span class="author">Allsopp BA</span></p>
<p class="affiliation">Department of Veterinary Tropical Diseases, Faculty of Veterinary Science, University of Pretoria, Private Bag X04, Onderstepoort 0110, South Africa.</p><p class="abstract">Heartwater, a tick-borne disease of domestic and wild ruminants, is caused by the intracellular rickettsia Ehrlichia ruminantium (previously known as Cowdria ruminantium). It is a major constraint to livestock production throughout subSaharan Africa, and it threatens to invade the Americas, yet there is no immediate prospect of an effective vaccine. A shotgun genome sequencing project was undertaken in the expectation that access to the complete protein coding repertoire of the organism will facilitate the search for vaccine candidate genes. We report here the complete 1,516,355-bp sequence of the type strain, the stock derived from the South African Welgevonden isolate. Only 62% of the genome is predicted to be coding sequence, encoding 888 proteins and 41 stable RNA species. The most striking feature is the large number of tandemly repeated and duplicated sequences, some of continuously variable copy number, which contributes to the low proportion of coding sequence. These repeats have mediated numerous translocation and inversion events that have resulted in the duplication and truncation of some genes and have also given rise to new genes. There are 32 predicted pseudogenes, most of which are truncated fragments of genes associated with repeats. Rather then being the result of the reductive evolution seen in other intracellular bacteria, these pseudogenes appear to be the product of ongoing sequence duplication events.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI47885</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2005</span>;<span class="volume">102</span>;<span class="number">3</span>;<span class="pagerange">838-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15637156">15637156</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC545511">545511</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0406633102">10.1073/pnas.0406633102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Postgraduate training in infectious diseases: investigating the current status in the international community.</h4>
<p class="authors"><span class="author">Cooke FJ</span>, <span class="author">Choubina P</span> and <span class="author">Holmes AH</span></p>
<p class="affiliation">Department of Infectious Diseases and Microbiology, Hammersmith Hospital, London, UK. <a href="mailto:%66%69%6F%6E%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#105;&#111;&#110;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">International collaboration and understanding is becoming increasingly important as we face a soaring number of emerging and re-emerging infectious diseases. Management of these conditions calls for a cohesive international effort, with contributions from many infectious disease specialists. To optimise collaborative efforts, an international understanding of training, capabilities, and skills would be valuable. An investigation of postgraduate training programmes in the infectious disease specialties around the world was done. 33 countries contributed information. 26 of these countries had established training programmes--one of which was changing its duration and research component; three were in the process of setting up programmes, two provided specialist training that had no official recognition, and two had no specialist training. In addition to promoting international understanding and collaboration, this article should catalyse a global assessment of postgraduate training programmes within the field of infectious diseases.</p>
<p class="publication"><span class="title">The Lancet infectious diseases</span>&nbsp;<span class="year">2005</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">440-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15978530" rel="external">15978530</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS1473-3099(05)70165-1" rel="external">10.1016/S1473-3099(05)70165-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The lrp gene and its role in type I fimbriation in Citrobacter rodentium.</h4>
<p class="authors"><span class="author">Cordone A</span>, <span class="author">Mauriello EM</span>, <span class="author">Pickard DJ</span>, <span class="author">Dougan G</span>, <span class="author">De Felice M</span> and <span class="author">Ricca E</span></p>
<p class="affiliation">Dipartimento di Biologia Strutturale e Funzionale, Universit&agrave; Federico II, via Cinthia, Complesso Monte S. Angelo, 80126, Naples, Italy.</p><p class="abstract">Citrobacter rodentium is a murine pathogen that is now widely used as an in vivo model for gastrointestinal infections due to its similarities with human enteropathogens, such as the possession of a locus for enterocyte effacement (the LEE island). We studied the lrp gene of C. rodentium and found that it encodes a product highly similar to members of the Lrp (leucine-responsive regulatory protein) family of transcriptional regulators, able to recognize leucine as an effector and to repress the expression of its own structural gene. In enterobacteria, Lrp is a global regulator of gene expression, as it controls a large variety of genes, including those coding for cell appendages and other potential virulence factors. Based on the well-established role of Lrp on the expression of pilus genes in Escherichia coli, we also studied the role of Lrp in controlling the formation of the type I pilus in C. rodentium. Type I pili, produced by the fim system, are virulence factors of uropathogens, involved in mediating bacterial adhesion to bladder epithelial cells. Yeast agglutination assays showed that Lrp is needed for type I pilus formation and real-time PCR experiments indicated that Lrp has a strong leucine-mediated effect on the expression of the fimAICDFGH operon. Mutant studies indicated that this positive action is exerted mainly through a positive control of Lrp on the phase variation mechanism that regulates fimAICDFGH expression. A quantitative analysis of its expression suggested that this operon may also be negatively regulated at the level of transcription.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2005</span>;<span class="volume">187</span>;<span class="number">20</span>;<span class="pagerange">7009-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16199571" rel="external">16199571</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1251604" rel="external">1251604</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.187.20.7009-7017.2005" rel="external">10.1128/JB.187.20.7009-7017.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The mSin3A chromatin-modifying complex is essential for embryogenesis and T-cell development.</h4>
<p class="authors"><span class="author">Cowley SM</span>, <span class="author">Iritani BM</span>, <span class="author">Mendrysa SM</span>, <span class="author">Xu T</span>, <span class="author">Cheng PF</span>, <span class="author">Yada J</span>, <span class="author">Liggitt HD</span> and <span class="author">Eisenman RN</span></p>
<p class="affiliation">Division of Basic Sciences, Fred Hutchinson Cancer Research Center, Seattle WA 98109-1024, USA.</p><p class="abstract">The corepressor mSin3A is the core component of a chromatin-modifying complex that is recruited by multiple gene-specific transcriptional repressors. In order to understand the role of mSin3A during development, we generated constitutive germ line as well as conditional msin3A deletions. msin3A deletion in the developing mouse embryo results in lethality at the postimplantation stage, demonstrating that it is an essential gene. Blastocysts derived from preimplantation msin3A null embryos and mouse embryo fibroblasts (MEFs) lacking msin3A display a significant reduction in cell division. msin3A null MEFs also show mislocalization of the heterochromatin protein, HP1alpha, without alterations in global histone acetylation. Heterozygous msin3A(+/-) mice with a systemic twofold decrease in mSin3A protein develop splenomegaly as well as kidney disease indicative of a disruption of lymphocyte homeostasis. Conditional deletion of msin3A from developing T cells results in reduced thymic cellularity and a fivefold decrease in the number of cytotoxic (CD8) T cells, while helper (CD4) T cells are unaffected. We show that CD8 development is dependent on mSin3A at a step downstream of T-cell receptor signaling and that loss of mSin3A specifically decreases survival of double-positive and CD8 T cells. Thus, msin3A is a pleiotropic gene which, in addition to its role in cell cycle progression, is required for the development and homeostasis of cells in the lymphoid lineage.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01CA57138</span>; NIAID NIH HHS: <span class="grant">R01 AI053568-01A1</span>, <span class="grant">R01 AI053568-02</span>, <span class="grant">R01 AI053568-03</span>, <span class="grant">R01 AI053568-04</span>, <span class="grant">R01 AI053568-05</span>, <span class="grant">R01AI0535468</span>; NICHD NIH HHS: <span class="grant">5R011HD18184</span>; PHS HHS: <span class="grant">5R01JL65898</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">16</span>;<span class="pagerange">6990-7004</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16055712" rel="external">16055712</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1190252" rel="external">1190252</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.25.16.6990-7004.2005" rel="external">10.1128/MCB.25.16.6990-7004.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A survey of homozygous deletions in human cancer genomes.</h4>
<p class="authors"><span class="author">Cox C</span>, <span class="author">Bignell G</span>, <span class="author">Greenman C</span>, <span class="author">Stabenau A</span>, <span class="author">Warren W</span>, <span class="author">Stephens P</span>, <span class="author">Davies H</span>, <span class="author">Watt S</span>, <span class="author">Teague J</span>, <span class="author">Edkins S</span>, <span class="author">Birney E</span>, <span class="author">Easton DF</span>, <span class="author">Wooster R</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">Homozygous deletions of recessive cancer genes and fragile sites are known to occur in human cancers. We identified 281 homozygous deletions in 636 cancer cell lines. Of these deletions, 86 were homozygous deletions of known recessive cancer genes, 17 were of sequenced common fragile sites, and 178 were in genomic regions that do not overlap known recessive oncogenes or fragile sites (&quot;unexplained&quot; homozygous deletions). Some cancer cell lines have multiple homozygous deletions whereas others have none, suggesting intrinsic variation in the tendency to develop this type of genetic abnormality (P &lt; 0.001). The 178 unexplained homozygous deletions clustered into 131 genomic regions, 27 of which exhibit homozygous deletions in more than one cancer cell line. This degree of clustering indicates that the genomic positions of the unexplained homozygous deletions are not randomly determined (P &lt; 0.001). Many homozygous deletions, including those that are in multiple clusters, do not overlap known genes and appear to be in intergenic DNA. Therefore, to elucidate further the pathogenesis of homozygous deletions in cancer, we investigated the genome landscape within unexplained homozygous deletions. The gene count within homozygous deletions is low compared with the rest of the genome. There are also fewer short interspersed nuclear elements (SINEs), long interspersed nuclear elements (LINEs), and low-copy-number repeats (LCRs). However, DNA within homozygous deletions has higher flexibility. These features may signal the presence of currently unrecognized zones of susceptibility to DNA rearrangement. They may also reflect a tendency to reduce the adverse effects of homozygous deletions by minimizing the number of genes removed.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2005</span>;<span class="volume">102</span>;<span class="number">12</span>;<span class="pagerange">4542-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15761058" rel="external">15761058</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC555487" rel="external">555487</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0408593102" rel="external">10.1073/pnas.0408593102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Treating genetic disease through RNA interference.</h4>
<p class="authors"><span class="author">Crombie C</span> and <span class="author">Fraser A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2005</span>;<span class="volume">365</span>;<span class="number">9467</span>;<span class="pagerange">1288-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15823365" rel="external">15823365</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(05)61006-9" rel="external">10.1016/S0140-6736(05)61006-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the genomic region containing the tammar wallaby (Macropus eugenii) orthologues of MHC class III genes.</h4>
<p class="authors"><span class="author">Cross JG</span>, <span class="author">Harrison GA</span>, <span class="author">Coggill P</span>, <span class="author">Sims S</span>, <span class="author">Beck S</span>, <span class="author">Deakin JE</span> and <span class="author">Graves JA</span></p>
<p class="affiliation">Comparative Genomics Unit, ARC Centre for Kangaroo Genomics, Research School of Biological Sciences, Australian National University, Canberra, Australia. <a href="mailto:%63%72%6F%73%73%40%72%73%62%73.%61%6E%75.%65%64%75.%61%75">&#99;&#114;&#111;&#115;&#115;&#64;&#114;&#115;&#98;&#115;&#46;&#97;&#110;&#117;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Major histocompatibility complex (MHC) molecules are central to development and regulation of the immune system in all jawed vertebrates. MHC class III cytokine genes from the tumor necrosis factor core family, including tumor necrosis factor (TNF) and lymphotoxin alpha and beta (LTA, LTB), are well studied in human and mouse. Orthologues have been identified in several other eutherian species and the cDNA sequences have been reported for a model marsupial, the tammar wallaby. Comparative genomics can help to determine gene function, to understand the evolution of a gene or gene family, and to identify potential regulatory regions. We therefore cloned the genomic region containing the tammar LTB, TNF, and LTA orthologues by &quot;genome walking&quot;, using primers designed from known tammar sequences and regions conserved in other species. We isolated two tammar BAC clones containing all three genes. These tammar genes show similar intergenic distances and the same transcriptional orientation as in human and mouse. Gene structures and sequences are also very conserved. By comparing the tammar, human and mouse genomic sequences we were able to identify candidate regulatory regions for these genes in mammals. Full length sequencing of BACs containing the three genes has been partially completed, and reveals the presence of a number of other tammar MHC III orthologues in this region.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">111</span>;<span class="number">2</span>;<span class="pagerange">110-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16103651" rel="external">16103651</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000086379" rel="external">10.1159/000086379</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diversity at every level.</h4>
<p class="authors"><span class="author">Crossman L</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">196-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15742523">15742523</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1479">10.1038/nrmicro1479</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plasmid replicons of Rhizobium.</h4>
<p class="authors"><span class="author">Crossman LC</span></p>
<p class="affiliation">Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6C%63%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#99;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Rhizobium spp. are found in soil. They are both free-living and found symbiotically associated with the nodules of leguminous plants. Traditionally, studies have focused on the association of these organisms with plants in nitrogen-fixing nodules, since this is regarded as the most important role of these bacteria in the environment. Rhizobium sp. are known to possess several replicons. Some, like the Rhizobium etli symbiotic plasmid p42d and the plasmid pNGR234b of Rhizobium NGR234, have been sequenced and characterized. The plasmids from these organisms are the focus of this short review.</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Pt 1</span>;<span class="pagerange">157-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15667292" rel="external">15667292</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0330157" rel="external">10.1042/BST0330157</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The immunoglobulin heavy-chain locus in zebrafish: identification and expression of a previously unknown isotype, immunoglobulin Z.</h4>
<p class="authors"><span class="author">Danilova N</span>, <span class="author">Bussmann J</span>, <span class="author">Jekosch K</span> and <span class="author">Steiner LA</span></p>
<p class="affiliation">Department of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts, 02139, USA.</p><p class="abstract">The only immunoglobulin heavy-chain classes known so far in teleosts have been mu and delta. We identify here a previously unknown class, immunoglobulin zeta, expressed in zebrafish and other teleosts. In the zebrafish heavy-chain locus, variable (V) gene segments lie upstream of two tandem diversity, joining and constant (DJC) clusters, resembling the mouse T cell receptor alpha (Tcra) and delta (Tcrd) locus. V genes rearrange to (DJC)(zeta) or to (DJC)(mu) without evidence of switch rearrangement. The zebrafish immunoglobulin zeta gene (ighz) and mouse Tcrd, which are proximal to the V gene array, are expressed earlier in development. In adults, ighz was expressed only in kidney and thymus, which are primary lymphoid organs in teleosts. This additional class adds complexity to the immunoglobulin repertoire and raises questions concerning the evolution of immunoglobulins and the regulation of the differential expression of ighz and ighm.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI08054</span></p>
<p class="publication"><span class="title">Nature immunology</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">295-302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15685175">15685175</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fni1166">10.1038/ni1166</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic mutations of the protein kinase gene family in human lung cancer.</h4>
<p class="authors"><span class="author">Davies H</span>, <span class="author">Hunter C</span>, <span class="author">Smith R</span>, <span class="author">Stephens P</span>, <span class="author">Greenman C</span>, <span class="author">Bignell G</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Edkins S</span>, <span class="author">Stevens C</span>, <span class="author">Parker A</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Avis T</span>, <span class="author">Barthorpe S</span>, <span class="author">Brackenbury L</span>, <span class="author">Buck G</span>, <span class="author">Clements J</span>, <span class="author">Cole J</span>, <span class="author">Dicks E</span>, <span class="author">Edwards K</span>, <span class="author">Forbes S</span>, <span class="author">Gorton M</span>, <span class="author">Gray K</span>, <span class="author">Halliday K</span>, <span class="author">Harrison R</span>, <span class="author">Hills K</span>, <span class="author">Hinton J</span>, <span class="author">Jones D</span>, <span class="author">Kosmidou V</span>, <span class="author">Laman R</span>, <span class="author">Lugg R</span>, <span class="author">Menzies A</span>, <span class="author">Perry J</span>, <span class="author">Petty R</span>, <span class="author">Raine K</span>, <span class="author">Shepherd R</span>, <span class="author">Small A</span>, <span class="author">Solomon H</span>, <span class="author">Stephens Y</span>, <span class="author">Tofts C</span>, <span class="author">Varian J</span>, <span class="author">Webb A</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Yates A</span>, <span class="author">Brasseur F</span>, <span class="author">Cooper CS</span>, <span class="author">Flanagan AM</span>, <span class="author">Green A</span>, <span class="author">Knowles M</span>, <span class="author">Leung SY</span>, <span class="author">Looijenga LH</span>, <span class="author">Malkowicz B</span>, <span class="author">Pierotti MA</span>, <span class="author">Teh BT</span>, <span class="author">Yuen ST</span>, <span class="author">Lakhani SR</span>, <span class="author">Easton DF</span>, <span class="author">Weber BL</span>, <span class="author">Goldstraw P</span>, <span class="author">Nicholson AG</span>, <span class="author">Wooster R</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, United Kingdom.</p><p class="abstract">Protein kinases are frequently mutated in human cancer and inhibitors of mutant protein kinases have proven to be effective anticancer drugs. We screened the coding sequences of 518 protein kinases (approximately 1.3 Mb of DNA per sample) for somatic mutations in 26 primary lung neoplasms and seven lung cancer cell lines. One hundred eighty-eight somatic mutations were detected in 141 genes. Of these, 35 were synonymous (silent) changes. This result indicates that most of the 188 mutations were &quot;passenger&quot; mutations that are not causally implicated in oncogenesis. However, an excess of approximately 40 nonsynonymous substitutions compared with that expected by chance (P = 0.07) suggests that some nonsynonymous mutations have been selected and are contributing to oncogenesis. There was considerable variation between individual lung cancers in the number of mutations observed and no mutations were found in lung carcinoids. The mutational spectra of most lung cancers were characterized by a high proportion of C:G &gt; A:T transversions, compatible with the mutagenic effects of tobacco carcinogens. However, one neuroendocrine cancer cell line had a distinctive mutational spectrum reminiscent of UV-induced DNA damage. The results suggest that several mutated protein kinases may be contributing to lung cancer development, but that mutations in each one are infrequent.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2005</span>;<span class="volume">65</span>;<span class="number">17</span>;<span class="pagerange">7591-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16140923">16140923</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-05-1855">10.1158/0008-5472.CAN-05-1855</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deletion at chromosome band 20p12.1 in colorectal cancer revealed by high resolution array comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Davison EJ</span>, <span class="author">Tarpey PS</span>, <span class="author">Fiegler H</span>, <span class="author">Tomlinson IP</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%65.%6A.%64%61%76%69%73%6F%6E%40%6C%65%65%64%73.%61%63.%75%6B">&#101;&#46;&#106;&#46;&#100;&#97;&#118;&#105;&#115;&#111;&#110;&#64;&#108;&#101;&#101;&#100;&#115;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Array comparative genomic hybridization (Array CGH) with tiling path resolution for a approximately 4.61 Mb region of chromosome band 20p12.1 has been used to investigate copy number loss in 48 colorectal cancer cell lines and 37 primary colorectal cancers. A recurrent deletion was detected in 55% of cell lines and 23% of primary cancers and the consensus minimum region of loss was identified as a approximately 190 kb section from 14.85 Mb to 15.04 Mb of chromosome 20. Two noncoding RNA genes located in the region, BA318C17.1 and DJ974N19.1, were investigated by mutation analysis and real-time PCR in colorectal cancer cell lines. Sequence changes in BA318C17.1 and reduced expression of both genes was detected, suggesting that the abrogation of these genes may play a role in colorectal tumorigenesis.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2005</span>;<span class="volume">44</span>;<span class="number">4</span>;<span class="pagerange">384-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16110499" rel="external">16110499</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20252" rel="external">10.1002/gcc.20252</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Positional and functional mapping of a neuroblastoma differentiation gene on chromosome 11.</h4>
<p class="authors"><span class="author">De Preter K</span>, <span class="author">Vandesompele J</span>, <span class="author">Menten B</span>, <span class="author">Carr P</span>, <span class="author">Fiegler H</span>, <span class="author">Edsj&ouml; A</span>, <span class="author">Carter NP</span>, <span class="author">Yigit N</span>, <span class="author">Waelput W</span>, <span class="author">Van Roy N</span>, <span class="author">Bader S</span>, <span class="author">P&aring;hlman S</span> and <span class="author">Speleman F</span></p>
<p class="affiliation">Center for Medical Genetics, Ghent University Hospital, MRB 2nd floor, De Pintelaan 185, B-9000 Ghent, Belgium. <a href="mailto:%6B%61%74%6C%65%65%6E.%64%65%70%72%65%74%65%72%40%75%67%65%6E%74.%62%65">&#107;&#97;&#116;&#108;&#101;&#101;&#110;&#46;&#100;&#101;&#112;&#114;&#101;&#116;&#101;&#114;&#64;&#117;&#103;&#101;&#110;&#116;&#46;&#98;&#101;</a></p><p class="abstract"><strong>Background</strong>: Loss of chromosome 11q defines a subset of high-stage aggressive neuroblastomas. Deletions are typically large and mapping efforts have thus far not lead to a well defined consensus region, which hampers the identification of positional candidate tumour suppressor genes. In a previous study, functional evidence for a neuroblastoma suppressor gene on chromosome 11 was obtained through microcell mediated chromosome transfer, indicated by differentiation of neuroblastoma cells with loss of distal 11q upon introduction of chromosome 11. Interestingly, some of these microcell hybrid clones were shown to harbour deletions in the transferred chromosome 11. We decided to further exploit this model system as a means to identify candidate tumour suppressor or differentiation genes located on chromosome 11.</p><p class="abstract"><strong>Results</strong>: In a first step, we performed high-resolution array CGH DNA copy-number analysis in order to evaluate the chromosome 11 status in the hybrids. Several deletions in both parental and transferred chromosomes in the investigated microcell hybrids were observed. Subsequent correlation of these deletion events with the observed morphological changes lead to the delineation of three putative regions on chromosome 11: 11q25, 11p13--&gt;11p15.1 and 11p15.3, that may harbour the responsible differentiation gene.</p><p class="abstract"><strong>Conclusion</strong>: Using an available model system, we were able to put forward some candidate regions that may be involved in neuroblastoma. Additional studies will be required to clarify the putative role of the genes located in these chromosomal segments in the observed differentiation phenotype specifically or in neuroblastoma pathogenesis in general.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="pagerange">97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16000168" rel="external">16000168</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1185534" rel="external">1185534</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-6-97" rel="external">10.1186/1471-2164-6-97</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic sequence of the class II region of the canine MHC: comparison with the MHC of other mammalian species.</h4>
<p class="authors"><span class="author">Debenham SL</span>, <span class="author">Hart EA</span>, <span class="author">Ashurst JL</span>, <span class="author">Howe KL</span>, <span class="author">Quail MA</span>, <span class="author">Ollier WE</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Genetics Section, Animal Health Trust, Lanwades Park, Kentford, Newmarket, Suffolk CB8 7UU, UK. <a href="mailto:%73%61%6C%6C%79.%64%65%62%65%6E%68%61%6D%40%61%68%74.%6F%72%67.%75%6B">&#115;&#97;&#108;&#108;&#121;&#46;&#100;&#101;&#98;&#101;&#110;&#104;&#97;&#109;&#64;&#97;&#104;&#116;&#46;&#111;&#114;&#103;&#46;&#117;&#107;</a></p><p class="abstract">The domestic dog, Canis familiaris, is an excellent model species in which to study complex inherited diseases, having over 200 recognized breeds, each of which represents a closed gene pool. Overlapping canine genomic BAC clones were sequenced to obtain 711,521 bp of the canine classical and extended MHC class II regions. Analysis and annotation of this sequence reveals that it contains 45 loci, of which 29 are predicted to be functionally expressed. Comparison of the DLA class II sequence with those of the cat, human, and mouse highlights regions of syntenic conservation and species-specific gene rearrangement and duplication and gives an insight into the evolution of the DR region in the order Carnivora. Elucidation of functionally important dog class II genes and the identification of 23 microsatellite markers spanning this region will contribute significantly to the study of canine diseases that have an immune component.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">85</span>;<span class="number">1</span>;<span class="pagerange">48-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15607421">15607421</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ygeno.2004.09.009">10.1016/j.ygeno.2004.09.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conserved non-genic sequences - an unexpected feature of mammalian genomes.</h4>
<p class="authors"><span class="author">Dermitzakis ET</span>, <span class="author">Reymond A</span> and <span class="author">Antonarakis SE</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%64%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#100;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Mammalian genomes contain highly conserved sequences that are not functionally transcribed. These sequences are single copy and comprise approximately 1-2% of the human genome. Evolutionary analysis strongly supports their functional conservation, although their potentially diverse, functional attributes remain unknown. It is likely that genomic variation in conserved non-genic sequences is associated with phenotypic variability and human disorders. So how might their function and contribution to human disorders be examined?</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">151-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15716910">15716910</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1527">10.1038/nrg1527</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression variation and expression quantitative trait mapping of human chromosome 21 genes.</h4>
<p class="authors"><span class="author">Deutsch S</span>, <span class="author">Lyle R</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Attar H</span>, <span class="author">Subrahmanyan L</span>, <span class="author">Gehrig C</span>, <span class="author">Parand L</span>, <span class="author">Gagnebin M</span>, <span class="author">Rougemont J</span>, <span class="author">Jongeneel CV</span> and <span class="author">Antonarakis SE</span></p>
<p class="affiliation">Department of Genetic Medicine and Development, Geneva University Medical School, Geneva, Switzerland.</p><p class="abstract">Inter-individual differences in gene expression are likely to account for an important fraction of phenotypic differences, including susceptibility to common disorders. Recent studies have shown extensive variation in gene expression levels in humans and other organisms, and that a fraction of this variation is under genetic control. We investigated the patterns of gene expression variation in a 25 Mb region of human chromosome 21, which has been associated with many Down syndrome (DS) phenotypes. Taqman real-time PCR was used to measure expression variation of 41 genes in lymphoblastoid cells of 40 unrelated individuals. For 25 genes found to be differentially expressed, additional analysis was performed in 10 CEPH families to determine heritabilities and map loci harboring regulatory variation. Seventy-six percent of the differentially expressed genes had significant heritabilities, and genomewide linkage analysis led to the identification of significant eQTLs for nine genes. Most eQTLs were in trans, with the best result (P=7.46 x 10(-8)) obtained for TMEM1 on chromosome 12q24.33. A cis-eQTL identified for CCT8 was validated by performing an association study in 60 individuals from the HapMap project. SNP rs965951 located within CCT8 was found to be significantly associated with its expression levels (P=2.5 x 10(-5)) confirming cis-regulatory variation. The results of our study provide a representative view of expression variation of chromosome 21 genes, identify loci involved in their regulation and suggest that genes, for which expression differences are significantly larger than 1.5-fold in control samples, are unlikely to be involved in DS-phenotypes present in all affected individuals.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14</span>;<span class="number">23</span>;<span class="pagerange">3741-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16251198" rel="external">16251198</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi404" rel="external">10.1093/hmg/ddi404</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exon array CGH: detection of copy-number changes at the resolution of individual exons in the human genome.</h4>
<p class="authors"><span class="author">Dhami P</span>, <span class="author">Coffey AJ</span>, <span class="author">Abbs S</span>, <span class="author">Vermeesch JR</span>, <span class="author">Dumanski JP</span>, <span class="author">Woodward KJ</span>, <span class="author">Andrews RM</span>, <span class="author">Langford C</span> and <span class="author">Vetrie D</span></p>
<p class="affiliation">Human Genetics, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">The development of high-throughput screening methods such as array-based comparative genome hybridization (array CGH) allows screening of the human genome for copy-number changes. Current array CGH strategies have limits of resolution that make detection of small (less than a few tens of kilobases) gains or losses of genomic DNA difficult to identify. We report here a significant improvement in the resolution of array CGH, with the development of an array platform that utilizes single-stranded DNA array elements to accurately measure copy-number changes of individual exons in the human genome. Using this technology, we screened 31 patient samples across an array containing a total of 162 exons for five disease genes and detected copy-number changes, ranging from whole-gene deletions and duplications to single-exon deletions and duplications, in 100% of the cases. Our data demonstrate that it is possible to screen the human genome for copy-number changes with array CGH at a resolution that is 2 orders of magnitude higher than that previously reported.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">76</span>;<span class="number">5</span>;<span class="pagerange">750-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15756638">15756638</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1199365">1199365</a>; DOI: <a href="http://dx.doi.org/10.1086%2F429588">10.1086/429588</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The SET-domain protein superfamily: protein lysine methyltransferases.</h4>
<p class="authors"><span class="author">Dillon SC</span>, <span class="author">Zhang X</span>, <span class="author">Trievel RC</span> and <span class="author">Cheng X</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%63%64%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#99;&#100;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The SET-domain protein methyltransferase superfamily includes all but one of the proteins known to methylate histones on lysine. Histone methylation is important in the regulation of chromatin and gene expression.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM49245</span>, <span class="grant">GM61355</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">227</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16086857" rel="external">16086857</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1273623" rel="external">1273623</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2005-6-8-227" rel="external">10.1186/gb-2005-6-8-227</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NestedMICA: sensitive inference of over-represented motifs in nucleic acid sequence.</h4>
<p class="authors"><span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton Cambridge, CB10 1SA, UK. <a href="mailto:%74%64%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#100;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">NestedMICA is a new, scalable, pattern-discovery system for finding transcription factor binding sites and similar motifs in biological sequences. Like several previous methods, NestedMICA tackles this problem by optimizing a probabilistic mixture model to fit a set of sequences. However, the use of a newly developed inference strategy called Nested Sampling means NestedMICA is able to find optimal solutions without the need for a problematic initialization or seeding step. We investigate the performance of NestedMICA in a range scenario, on synthetic data and a well-characterized set of muscle regulatory regions, and compare it with the popular MEME program. We show that the new method is significantly more sensitive than MEME: in one case, it successfully extracted a target motif from background sequence four times longer than could be handled by the existing program. It also performs robustly on synthetic sequences containing multiple significant motifs. When tested on a real set of regulatory sequences, NestedMICA produced motifs which were good predictors for all five abundant classes of annotated binding sites.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">5</span>;<span class="pagerange">1445-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15760844">15760844</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1064142">1064142</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki282">10.1093/nar/gki282</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The use of edge-betweenness clustering to investigate biological function in protein interaction networks.</h4>
<p class="authors"><span class="author">Dunn R</span>, <span class="author">Dudbridge F</span> and <span class="author">Sanderson CM</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%72%64%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#100;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a> &lt;<a href="mailto:%72%64%33%40%73%61%6E%67%65%72.%61%63.%75%6B&%67%74">&#114;&#100;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: This paper describes an automated method for finding clusters of interconnected proteins in protein interaction networks and retrieving protein annotations associated with these clusters.</p><p class="abstract"><strong>Results</strong>: Protein interaction graphs were separated into subgraphs of interconnected proteins, using the JUNG implementation of Girvan and Newman&#39;s Edge-Betweenness algorithm. Functions were sought for these subgraphs by detecting significant correlations with the distribution of Gene Ontology terms which had been used to annotate the proteins within each cluster. The method was implemented using freely available software (JUNG and the R statistical package). Protein clusters with significant correlations to functional annotations could be identified and included groups of proteins know to cooperate in cell metabolism. The method appears to be resilient against the presence of false positive interactions.</p><p class="abstract"><strong>Conclusion</strong>: This method provides a useful tool for rapid screening of small to medium size protein interaction datasets.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="pagerange">39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15740614" rel="external">15740614</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC555937" rel="external">555937</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-6-39" rel="external">10.1186/1471-2105-6-39</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Host susceptibility and clinical outcomes in toll-like receptor 5-deficient patients with typhoid fever in Vietnam.</h4>
<p class="authors"><span class="author">Dunstan SJ</span>, <span class="author">Hawn TR</span>, <span class="author">Hue NT</span>, <span class="author">Parry CP</span>, <span class="author">Ho VA</span>, <span class="author">Vinh H</span>, <span class="author">Diep TS</span>, <span class="author">House D</span>, <span class="author">Wain J</span>, <span class="author">Aderem A</span>, <span class="author">Hien TT</span> and <span class="author">Farrar JJ</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Ho Chi Minh City, Vietnam. <a href="mailto:%73%64%75%6E%73%74%61%6E%40%68%63%6D.%76%6E%6E.%76%6E">&#115;&#100;&#117;&#110;&#115;&#116;&#97;&#110;&#64;&#104;&#99;&#109;&#46;&#118;&#110;&#110;&#46;&#118;&#110;</a></p><p class="abstract">Toll-like receptor 5 (TLR5) mediates innate immune responses to bacterial pathogens by binding to flagellin. A polymorphism in the TLR5 gene introduces a premature stop codon (TLR5(392STOP)) that is associated with susceptibility to legionnaires disease. Here we investigated whether TLR5(392STOP) was associated with typhoid fever. The frequency of TLR5(392STOP) was not significantly different in 565 patients with typhoid fever and 281 ethnically matched control subjects. Furthermore, TLR5 deficiency had no measurable effect on a number of clinical parameters associated with typhoid fever, including fever clearance time, pathogen burden, disease severity, or age at acquisition of disease. TLR5 may not play an important role in TLR-stimulated innate immune responses to human infection with Salmonella enterica serovar Typhi. Initiation of these responses may rely on other TLRs that recognize different bacterial ligands.</p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2005</span>;<span class="volume">191</span>;<span class="number">7</span>;<span class="pagerange">1068-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15747241" rel="external">15747241</a>; DOI: <a href="http://dx.doi.org/10.1086%2F428593" rel="external">10.1086/428593</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the social amoeba Dictyostelium discoideum.</h4>
<p class="authors"><span class="author">Eichinger L</span>, <span class="author">Pachebat JA</span>, <span class="author">Gl&ouml;ckner G</span>, <span class="author">Rajandream MA</span>, <span class="author">Sucgang R</span>, <span class="author">Berriman M</span>, <span class="author">Song J</span>, <span class="author">Olsen R</span>, <span class="author">Szafranski K</span>, <span class="author">Xu Q</span>, <span class="author">Tunggal B</span>, <span class="author">Kummerfeld S</span>, <span class="author">Madera M</span>, <span class="author">Konfortov BA</span>, <span class="author">Rivero F</span>, <span class="author">Bankier AT</span>, <span class="author">Lehmann R</span>, <span class="author">Hamlin N</span>, <span class="author">Davies R</span>, <span class="author">Gaudet P</span>, <span class="author">Fey P</span>, <span class="author">Pilcher K</span>, <span class="author">Chen G</span>, <span class="author">Saunders D</span>, <span class="author">Sodergren E</span>, <span class="author">Davis P</span>, <span class="author">Kerhornou A</span>, <span class="author">Nie X</span>, <span class="author">Hall N</span>, <span class="author">Anjard C</span>, <span class="author">Hemphill L</span>, <span class="author">Bason N</span>, <span class="author">Farbrother P</span>, <span class="author">Desany B</span>, <span class="author">Just E</span>, <span class="author">Morio T</span>, <span class="author">Rost R</span>, <span class="author">Churcher C</span>, <span class="author">Cooper J</span>, <span class="author">Haydock S</span>, <span class="author">van Driessche N</span>, <span class="author">Cronin A</span>, <span class="author">Goodhead I</span>, <span class="author">Muzny D</span>, <span class="author">Mourier T</span>, <span class="author">Pain A</span>, <span class="author">Lu M</span>, <span class="author">Harper D</span>, <span class="author">Lindsay R</span>, <span class="author">Hauser H</span>, <span class="author">James K</span>, <span class="author">Quiles M</span>, <span class="author">Madan Babu M</span>, <span class="author">Saito T</span>, <span class="author">Buchrieser C</span>, <span class="author">Wardroper A</span>, <span class="author">Felder M</span>, <span class="author">Thangavelu M</span>, <span class="author">Johnson D</span>, <span class="author">Knights A</span>, <span class="author">Loulseged H</span>, <span class="author">Mungall K</span>, <span class="author">Oliver K</span>, <span class="author">Price C</span>, <span class="author">Quail MA</span>, <span class="author">Urushihara H</span>, <span class="author">Hernandez J</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Steffen D</span>, <span class="author">Sanders M</span>, <span class="author">Ma J</span>, <span class="author">Kohara Y</span>, <span class="author">Sharp S</span>, <span class="author">Simmonds M</span>, <span class="author">Spiegler S</span>, <span class="author">Tivey A</span>, <span class="author">Sugano S</span>, <span class="author">White B</span>, <span class="author">Walker D</span>, <span class="author">Woodward J</span>, <span class="author">Winckler T</span>, <span class="author">Tanaka Y</span>, <span class="author">Shaulsky G</span>, <span class="author">Schleicher M</span>, <span class="author">Weinstock G</span>, <span class="author">Rosenthal A</span>, <span class="author">Cox EC</span>, <span class="author">Chisholm RL</span>, <span class="author">Gibbs R</span>, <span class="author">Loomis WF</span>, <span class="author">Platzer M</span>, <span class="author">Kay RR</span>, <span class="author">Williams J</span>, <span class="author">Dear PH</span>, <span class="author">Noegel AA</span>, <span class="author">Barrell B</span> and <span class="author">Kuspa A</span></p>
<p class="affiliation">Center for Biochemistry and Center for Molecular Medicine Cologne, University of Cologne, Joseph-Stelzmann-Str. 52, 50931 Cologne, Germany.</p><p class="abstract">The social amoebae are exceptional in their ability to alternate between unicellular and multicellular forms. Here we describe the genome of the best-studied member of this group, Dictyostelium discoideum. The gene-dense chromosomes of this organism encode approximately 12,500 predicted proteins, a high proportion of which have long, repetitive amino acid tracts. There are many genes for polyketide synthases and ABC transporters, suggesting an extensive secondary metabolism for producing and exporting small molecules. The genome is rich in complex repeats, one class of which is clustered and may serve as centromeres. Partial copies of the extrachromosomal ribosomal DNA (rDNA) element are found at the ends of each chromosome, suggesting a novel telomere structure and the use of a common mechanism to maintain both the rDNA and chromosomal termini. A proteome-based phylogeny shows that the amoebozoa diverged from the animal-fungal lineage after the plant-animal split, but Dictyostelium seems to have retained more of the diversity of the ancestral genome than have plants, animals or fungi.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R01 HD035925</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">435</span>;<span class="number">7038</span>;<span class="pagerange">43-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15875012">15875012</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1352341">1352341</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03481">10.1038/nature03481</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Sequence Ontology: a tool for the unification of genome annotations.</h4>
<p class="authors"><span class="author">Eilbeck K</span>, <span class="author">Lewis SE</span>, <span class="author">Mungall CJ</span>, <span class="author">Yandell M</span>, <span class="author">Stein L</span>, <span class="author">Durbin R</span> and <span class="author">Ashburner M</span></p>
<p class="affiliation">Department of Molecular and Cellular Biology, Life Sciences Addition, University of California, Berkeley, CA 94729-3200, USA. <a href="mailto:%6B%65%69%6C%62%65%63%6B%40%66%72%75%69%74%66%6C%79.%6F%72%67">&#107;&#101;&#105;&#108;&#98;&#101;&#99;&#107;&#64;&#102;&#114;&#117;&#105;&#116;&#102;&#108;&#121;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The Sequence Ontology (SO) is a structured controlled vocabulary for the parts of a genomic annotation. SO provides a common set of terms and definitions that will facilitate the exchange, analysis and management of genomic data. Because SO treats part-whole relationships rigorously, data described with it can become substrates for automated reasoning, and instances of sequence features described by the SO can be subjected to a group of logical operations termed extensional mereology operators.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">R44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15892872" rel="external">15892872</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1175956" rel="external">1175956</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2005-6-5-r44" rel="external">10.1186/gb-2005-6-5-r44</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of trypanosomatid parasitic protozoa.</h4>
<p class="authors"><span class="author">El-Sayed NM</span>, <span class="author">Myler PJ</span>, <span class="author">Blandin G</span>, <span class="author">Berriman M</span>, <span class="author">Crabtree J</span>, <span class="author">Aggarwal G</span>, <span class="author">Caler E</span>, <span class="author">Renauld H</span>, <span class="author">Worthey EA</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Ghedin E</span>, <span class="author">Peacock C</span>, <span class="author">Bartholomeu DC</span>, <span class="author">Haas BJ</span>, <span class="author">Tran AN</span>, <span class="author">Wortman JR</span>, <span class="author">Alsmark UC</span>, <span class="author">Angiuoli S</span>, <span class="author">Anupama A</span>, <span class="author">Badger J</span>, <span class="author">Bringaud F</span>, <span class="author">Cadag E</span>, <span class="author">Carlton JM</span>, <span class="author">Cerqueira GC</span>, <span class="author">Creasy T</span>, <span class="author">Delcher AL</span>, <span class="author">Djikeng A</span>, <span class="author">Embley TM</span>, <span class="author">Hauser C</span>, <span class="author">Ivens AC</span>, <span class="author">Kummerfeld SK</span>, <span class="author">Pereira-Leal JB</span>, <span class="author">Nilsson D</span>, <span class="author">Peterson J</span>, <span class="author">Salzberg SL</span>, <span class="author">Shallom J</span>, <span class="author">Silva JC</span>, <span class="author">Sundaram J</span>, <span class="author">Westenberger S</span>, <span class="author">White O</span>, <span class="author">Melville SE</span>, <span class="author">Donelson JE</span>, <span class="author">Andersson B</span>, <span class="author">Stuart KD</span> and <span class="author">Hall N</span></p>
<p class="affiliation">Institute for Genomic Research, 9712 Medical Center Drive, Rockville, MD 20850, USA. <a href="mailto:%6E%65%6C%73%61%79%65%64%40%74%69%67%72.%6F%72%67">&#110;&#101;&#108;&#115;&#97;&#121;&#101;&#100;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">A comparison of gene content and genome architecture of Trypanosoma brucei, Trypanosoma cruzi, and Leishmania major, three related pathogens with different life cycles and disease pathology, revealed a conserved core proteome of about 6200 genes in large syntenic polycistronic gene clusters. Many species-specific genes, especially large surface antigen families, occur at nonsyntenic chromosome-internal and subtelomeric regions. Retroelements, structural RNAs, and gene family expansion are often associated with syntenic discontinuities that-along with gene divergence, acquisition and loss, and rearrangement within the syntenic regions-have shaped the genomes of each parasite. Contrary to recent reports, our analyses reveal no evidence that these species are descended from an ancestor that contained a photosynthetic endosymbiont.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI045039</span>, <span class="grant">AI45038</span>, <span class="grant">AI45061</span>, <span class="grant">U01 AI040599</span>, <span class="grant">U01 AI045039</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5733</span>;<span class="pagerange">404-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16020724">16020724</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1112181">10.1126/science.1112181</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations of C-RAF are rare in human cancer because C-RAF has a low basal kinase activity compared with B-RAF.</h4>
<p class="authors"><span class="author">Emuss V</span>, <span class="author">Garnett M</span>, <span class="author">Mason C</span> and <span class="author">Marais R</span></p>
<p class="affiliation">The Institute of Cancer Research, Signal Transduction Team, Cancer Research UK Centre of Cell and Molecular Biology, London, United Kingdom.</p><p class="abstract">The protein kinase B-RAF is mutated in approximately 8% of human cancers. Here we show that presumptive mutants of the closely related kinase, C-RAF, were detected in only 4 of 545 (0.7%) cancer cell lines. The activity of two of the mutated proteins is not significantly different from that of wild-type C-RAF and these variants may represent rare human polymorphisms. The basal and B-RAF-stimulated kinase activities of a third variant are unaltered but its activation by RAS is significantly reduced, suggesting that it may act in a dominant-negative manner to modulate pathway signaling. The fourth variant has elevated basal kinase activity and is hypersensitive to activation by RAS but does not transform mammalian cells. Furthermore, when we introduce the equivalent of the most common cancer mutation in B-RAF (V600E) into C-RAF, it only has a weak effect on kinase activity and does not convert C-RAF into an oncogene. This lack of activation occurs because C-RAF lacks a constitutive charge within a motif in the kinase domain called the N-region. This fundamental difference in RAF isoform regulation explains why B-RAF is frequently mutated in cancer whereas C-RAF mutations are rare.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2005</span>;<span class="volume">65</span>;<span class="number">21</span>;<span class="pagerange">9719-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16266992">16266992</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-05-1683">10.1158/0008-5472.CAN-05-1683</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene finding in the chicken genome.</h4>
<p class="authors"><span class="author">Eyras E</span>, <span class="author">Reymond A</span>, <span class="author">Castelo R</span>, <span class="author">Bye JM</span>, <span class="author">Camara F</span>, <span class="author">Flicek P</span>, <span class="author">Huckle EJ</span>, <span class="author">Parra G</span>, <span class="author">Shteynberg DD</span>, <span class="author">Wyss C</span>, <span class="author">Rogers J</span>, <span class="author">Antonarakis SE</span>, <span class="author">Birney E</span>, <span class="author">Guigo R</span> and <span class="author">Brent MR</span></p>
<p class="affiliation">Institut Municipal d&#39;Investigacio Medica, Universitat Pompeu Fabra, Centre de Regulacio Genomica, E08003 Barcelona, Catalonia, Spain. <a href="mailto:%65%65%79%72%61%73%40%69%6D%69%6D.%65%73">&#101;&#101;&#121;&#114;&#97;&#115;&#64;&#105;&#109;&#105;&#109;&#46;&#101;&#115;</a></p><p class="abstract"><strong>Background</strong>: Despite the continuous production of genome sequence for a number of organisms, reliable, comprehensive, and cost effective gene prediction remains problematic. This is particularly true for genomes for which there is not a large collection of known gene sequences, such as the recently published chicken genome. We used the chicken sequence to test comparative and homology-based gene-finding methods followed by experimental validation as an effective genome annotation method.</p><p class="abstract"><strong>Results</strong>: We performed experimental evaluation by RT-PCR of three different computational gene finders, Ensembl, SGP2 and TWINSCAN, applied to the chicken genome. A Venn diagram was computed and each component of it was evaluated. The results showed that de novo comparative methods can identify up to about 700 chicken genes with no previous evidence of expression, and can correctly extend about 40% of homology-based predictions at the 5&#39; end.</p><p class="abstract"><strong>Conclusions</strong>: De novo comparative gene prediction followed by experimental verification is effective at enhancing the annotation of the newly sequenced genomes provided by standard homology-based methods.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG003150-01</span>, <span class="grant">R01 HG02278</span>; Wellcome Trust</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="pagerange">131</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15924626">15924626</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1174864">1174864</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-6-131">10.1186/1471-2105-6-131</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Uncover genetic interactions in Caenorhabditis elegans by RNA interference.</h4>
<p class="authors"><span class="author">Fortunato A</span> and <span class="author">Fraser AG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, CB10 1SA, Cambridge, Hinxton, UK. <a href="mailto:%61%66%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#102;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">RNA-mediated interference (RNAi) has emerged recently as one of the most powerful functional genomics tools. RNAi has been particularly effective in the nematode worm C. elegans where RNAi has been used to analyse the loss-of-function phenotypes of almost all predicted genes. In this review, we illustrate how RNAi has been used to analyse gene function in C. elegans as well as pointing to some future directions for using RNAi to examine genetic interactions in a systematic manner.</p>
<p class="publication"><span class="title">Bioscience reports</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">5-6</span>;<span class="pagerange">299-307</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16307378" rel="external">16307378</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10540-005-2892-7" rel="external">10.1007/s10540-005-2892-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation in the eNOS gene modifies the association between total energy expenditure and glucose intolerance.</h4>
<p class="authors"><span class="author">Franks PW</span>, <span class="author">Luan J</span>, <span class="author">Barroso I</span>, <span class="author">Brage S</span>, <span class="author">Gonzalez Sanchez JL</span>, <span class="author">Ekelund U</span>, <span class="author">R&iacute;os MS</span>, <span class="author">Schafer AJ</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">National Institute of DiabetesDigestiveKidney Diseases, 1550 E. Indian School Rd., Phoenix, AZ 85014, USA. <a href="mailto:%70%66%72%61%6E%6B%73%40%6E%69%64%64%6B.%6E%69%68.%67%6F%76">&#112;&#102;&#114;&#97;&#110;&#107;&#115;&#64;&#110;&#105;&#100;&#100;&#107;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">Endothelium-derived nitric oxide (NO) facilitates skeletal muscle glucose uptake. Energy expenditure induces the endothelial NO synthase (eNOS) gene, providing a mechanism for insulin-independent glucose disposal. The object was to test 1) the association of genetic variation in eNOS, as assessed by haplotype-tagging single nucleotide polymorphisms (htSNPs) with type 2 diabetes, and 2) the interaction between eNOS haplotypes and total energy expenditure on glucose intolerance. Using multivariate models, we tested associations between eNOS htSNPs and diabetes (n = 461 and 474 case and control subjects, respectively) and glucose intolerance (two cohorts of n = 706 and 738 U.K. and Spanish Caucasians, respectively), and we tested eNOS x total energy expenditure interactions on glucose intolerance. An overall association between eNOS haplotype and diabetes was observed (P = 0.004). Relative to the most common haplotype (111), two haplotypes (121 and 212) tended to increase diabetes risk (OR 1.22, 95% CI 0.96-1.55), and one (122) was associated with decreased risk (0.58, 0.39-0.86). In the cohort studies, no association was observed between haplotypes and 2-h glucose (P &gt; 0.10). However, we observed a significant total energy expenditure-haplotype interaction (P = 0.007). Genetic variation at the eNOS locus is associated with diabetes, which may be attributable to an enhanced effect of total energy expenditure on glucose disposal in individuals with specific eNOS haplotypes. Gene-environment interactions such as this may help explain why replication of genetic association frequently fails.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2005</span>;<span class="volume">54</span>;<span class="number">9</span>;<span class="pagerange">2795-801</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16123371">16123371</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The pCoo plasmid of enterotoxigenic Escherichia coli is a mosaic cointegrate.</h4>
<p class="authors"><span class="author">Froehlich B</span>, <span class="author">Parkhill J</span>, <span class="author">Sanders M</span>, <span class="author">Quail MA</span> and <span class="author">Scott JR</span></p>
<p class="affiliation">Department of Microbiology and Immunology, Emory University School of Medicine, Atlanta, Georgia 30322, USA.</p><p class="abstract">CS1 is the prototype of a class of pili of enterotoxigenic Escherichia coli (ETEC) associated with diarrheal disease in humans. The genes encoding this pilus are carried on a large plasmid, pCoo. We report the sequence of the complete 98,396-bp plasmid. Like many other virulence plasmids, pCoo is a mosaic consisting of regions derived from multiple sources. Complete and fragmented insertion sequences (IS) make up 24% of the total DNA and are scattered throughout the plasmid. The pCoo DNA between these IS elements has a wide range of G+C content (35 to 57%), suggesting that these regions have different ancestries. We find that the pCoo plasmid is a cointegrate of two functional replicons, related to R64 and R100, which are joined at a 1,953-bp direct repeat of IS100. Recombination between these repeats in the cointegrate generates the two smaller replicons which coexist with the cointegrate in the culture. Both of the smaller replicons have plasmid stability genes as well as genes that may be important in pathogenesis. Examination by PCR of 17 other unrelated CS1 ETEC strains with a variety of serotypes demonstrated that all contained at least parts of both replicons of pCoo and that strains of the O6 genotype appear to contain a cointegrate very similar to pCoo. The results suggest that this family of CS1-encoding plasmids is evolving rapidly.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI24870</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2005</span>;<span class="volume">187</span>;<span class="number">18</span>;<span class="pagerange">6509-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16159784" rel="external">16159784</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1236633" rel="external">1236633</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.187.18.6509-6516.2005" rel="external">10.1128/JB.187.18.6509-6516.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The molecular clock mediates leptin-regulated bone formation.</h4>
<p class="authors"><span class="author">Fu L</span>, <span class="author">Patel MS</span>, <span class="author">Bradley A</span>, <span class="author">Wagner EF</span> and <span class="author">Karsenty G</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas 77030; Bone Disease Program of Texas, Baylor College of Medicine, Houston, Texas 77030, USA.</p><p class="abstract">The hormone leptin is a regulator of bone remodeling, a homeostatic function maintaining bone mass constant. Mice lacking molecular-clock components (Per and Cry), or lacking Per genes in osteoblasts, display high bone mass, suggesting that bone remodeling may also be subject to circadian regulation. Moreover, Per-deficient mice experience a paradoxical increase in bone mass following leptin intracerebroventricular infusion. Thus, clock genes may mediate the leptin-dependent sympathetic regulation of bone formation. We show that expression of clock genes in osteoblasts is regulated by the sympathetic nervous system and leptin. Clock genes mediate the antiproliferative function of sympathetic signaling by inhibiting G1 cyclin expression. Partially antagonizing this inhibitory loop, leptin also upregulates AP-1 gene expression, which promotes cyclin D1 expression, osteoblast proliferation, and bone formation. Thus, leptin determines the extent of bone formation by modulating, via sympathetic signaling, osteoblast proliferation through two antagonistic pathways, one of which involves the molecular clock.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2005</span>;<span class="volume">122</span>;<span class="number">5</span>;<span class="pagerange">803-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16143109">16143109</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2005.06.028">10.1016/j.cell.2005.06.028</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic mutations in human cancer: insights from resequencing the protein kinase gene family.</h4>
<p class="authors"><span class="author">Futreal PA</span>, <span class="author">Wooster R</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">All cancers arise due to the accumulation of mutations in critical target genes that, when altered, give rise to selective advantage in the cell and its progeny that harbor them. Knowledge of these mutations is key in understanding the biology of cancer initiation and progression, as well as the development of more targeted therapeutic strategies. We have undertaken a systematic screen of all annotated protein kinases in the human genome for mutations in a series of cancers including breast, non-small-cell lung, and testicular cancer. Our results show a wide diversity in mutation prevalence within and between tumor types. We have identified a mutator phenotype in human breast previously undescribed. The results presented from sequencing the same 1.3 million base pairs through several tumor types suggest that most of the observed mutations are likely to be passenger events rather than causally implicated in oncogenesis. However, this work does provide evidence for the likely existence of multiple, infrequently mutated kinases.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">70</span>;<span class="pagerange">43-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16869737" rel="external">16869737</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fsqb.2005.70.015" rel="external">10.1101/sqb.2005.70.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing of Aspergillus nidulans and comparative analysis with A. fumigatus and A. oryzae.</h4>
<p class="authors"><span class="author">Galagan JE</span>, <span class="author">Calvo SE</span>, <span class="author">Cuomo C</span>, <span class="author">Ma LJ</span>, <span class="author">Wortman JR</span>, <span class="author">Batzoglou S</span>, <span class="author">Lee SI</span>, <span class="author">Ba&#x15F;t&uuml;rkmen M</span>, <span class="author">Spevak CC</span>, <span class="author">Clutterbuck J</span>, <span class="author">Kapitonov V</span>, <span class="author">Jurka J</span>, <span class="author">Scazzocchio C</span>, <span class="author">Farman M</span>, <span class="author">Butler J</span>, <span class="author">Purcell S</span>, <span class="author">Harris S</span>, <span class="author">Braus GH</span>, <span class="author">Draht O</span>, <span class="author">Busch S</span>, <span class="author">D&#39;Enfert C</span>, <span class="author">Bouchier C</span>, <span class="author">Goldman GH</span>, <span class="author">Bell-Pedersen D</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Doonan JH</span>, <span class="author">Yu J</span>, <span class="author">Vienken K</span>, <span class="author">Pain A</span>, <span class="author">Freitag M</span>, <span class="author">Selker EU</span>, <span class="author">Archer DB</span>, <span class="author">Pe&ntilde;alva MA</span>, <span class="author">Oakley BR</span>, <span class="author">Momany M</span>, <span class="author">Tanaka T</span>, <span class="author">Kumagai T</span>, <span class="author">Asai K</span>, <span class="author">Machida M</span>, <span class="author">Nierman WC</span>, <span class="author">Denning DW</span>, <span class="author">Caddick M</span>, <span class="author">Hynes M</span>, <span class="author">Paoletti M</span>, <span class="author">Fischer R</span>, <span class="author">Miller B</span>, <span class="author">Dyer P</span>, <span class="author">Sachs MS</span>, <span class="author">Osmani SA</span> and <span class="author">Birren BW</span></p>
<p class="affiliation">The Broad Institute of MIT and Harvard, 320 Charles Street, Cambridge, Massachusetts 02142, USA.</p><p class="abstract">The aspergilli comprise a diverse group of filamentous fungi spanning over 200 million years of evolution. Here we report the genome sequence of the model organism Aspergillus nidulans, and a comparative study with Aspergillus fumigatus, a serious human pathogen, and Aspergillus oryzae, used in the production of sake, miso and soy sauce. Our analysis of genome structure provided a quantitative evaluation of forces driving long-term eukaryotic genome evolution. It also led to an experimentally validated model of mating-type locus evolution, suggesting the potential for sexual reproduction in A. fumigatus and A. oryzae. Our analysis of sequence conservation revealed over 5,000 non-coding regions actively conserved across all three species. Within these regions, we identified potential functional elements including a previously uncharacterized TPP riboswitch and motifs suggesting regulation in filamentous fungi by Puf family genes. We further obtained comparative and experimental evidence indicating widespread translational regulation by upstream open reading frames. These results enhance our understanding of these widely studied fungi as well as provide new insight into eukaryotic genome evolution and gene regulation.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">CFB17726</span>; NIGMS NIH HHS: <span class="grant">R01 GM058529</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">438</span>;<span class="number">7071</span>;<span class="pagerange">1105-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16372000">16372000</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature04341">10.1038/nature04341</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of Theileria parva, a bovine pathogen that transforms lymphocytes.</h4>
<p class="authors"><span class="author">Gardner MJ</span>, <span class="author">Bishop R</span>, <span class="author">Shah T</span>, <span class="author">de Villiers EP</span>, <span class="author">Carlton JM</span>, <span class="author">Hall N</span>, <span class="author">Ren Q</span>, <span class="author">Paulsen IT</span>, <span class="author">Pain A</span>, <span class="author">Berriman M</span>, <span class="author">Wilson RJ</span>, <span class="author">Sato S</span>, <span class="author">Ralph SA</span>, <span class="author">Mann DJ</span>, <span class="author">Xiong Z</span>, <span class="author">Shallom SJ</span>, <span class="author">Weidman J</span>, <span class="author">Jiang L</span>, <span class="author">Lynn J</span>, <span class="author">Weaver B</span>, <span class="author">Shoaibi A</span>, <span class="author">Domingo AR</span>, <span class="author">Wasawo D</span>, <span class="author">Crabtree J</span>, <span class="author">Wortman JR</span>, <span class="author">Haas B</span>, <span class="author">Angiuoli SV</span>, <span class="author">Creasy TH</span>, <span class="author">Lu C</span>, <span class="author">Suh B</span>, <span class="author">Silva JC</span>, <span class="author">Utterback TR</span>, <span class="author">Feldblyum TV</span>, <span class="author">Pertea M</span>, <span class="author">Allen J</span>, <span class="author">Nierman WC</span>, <span class="author">Taracha EL</span>, <span class="author">Salzberg SL</span>, <span class="author">White OR</span>, <span class="author">Fitzhugh HA</span>, <span class="author">Morzaria S</span>, <span class="author">Venter JC</span>, <span class="author">Fraser CM</span> and <span class="author">Nene V</span></p>
<p class="affiliation">Institute for Genomic Research (TIGR), 9712 Medical Center Drive, Rockville, MD 20850, USA. <a href="mailto:%67%61%72%64%6E%65%72%40%74%69%67%72.%6F%72%67">&#103;&#97;&#114;&#100;&#110;&#101;&#114;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">We report the genome sequence of Theileria parva, an apicomplexan pathogen causing economic losses to smallholder farmers in Africa. The parasite chromosomes exhibit limited conservation of gene synteny with Plasmodium falciparum, and its plastid-like genome represents the first example where all apicoplast genes are encoded on one DNA strand. We tentatively identify proteins that facilitate parasite segregation during host cell cytokinesis and contribute to persistent infection of transformed host cells. Several biosynthetic pathways are incomplete or absent, suggesting substantial metabolic dependence on the host cell. One protein family that may generate parasite antigenic diversity is not telomere-associated.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5731</span>;<span class="pagerange">134-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15994558">15994558</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1110439">10.1126/science.1110439</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MicroRNAs regulate brain morphogenesis in zebrafish.</h4>
<p class="authors"><span class="author">Giraldez AJ</span>, <span class="author">Cinalli RM</span>, <span class="author">Glasner ME</span>, <span class="author">Enright AJ</span>, <span class="author">Thomson JM</span>, <span class="author">Baskerville S</span>, <span class="author">Hammond SM</span>, <span class="author">Bartel DP</span> and <span class="author">Schier AF</span></p>
<p class="affiliation">Developmental Genetics Program, Skirball Institute of Biomolecular Medicine and Department of Cell Biology, New York University School of Medicine, New York, NY 10016, USA. <a href="mailto:%67%69%72%61%6C%64%65%7A%40%73%61%74%75%72%6E.%6D%65%64.%6E%79%75.%65%64%75">&#103;&#105;&#114;&#97;&#108;&#100;&#101;&#122;&#64;&#115;&#97;&#116;&#117;&#114;&#110;&#46;&#109;&#101;&#100;&#46;&#110;&#121;&#117;&#46;&#101;&#100;&#117;</a></p><p class="abstract">MicroRNAs (miRNAs) are small RNAs that regulate gene expression posttranscriptionally. To block all miRNA formation in zebrafish, we generated maternal-zygotic dicer (MZdicer) mutants that disrupt the Dicer ribonuclease III and double-stranded RNA-binding domains. Mutant embryos do not process precursor miRNAs into mature miRNAs, but injection of preprocessed miRNAs restores gene silencing, indicating that the disrupted domains are dispensable for later steps in silencing. MZdicer mutants undergo axis formation and differentiate multiple cell types but display abnormal morphogenesis during gastrulation, brain formation, somitogenesis, and heart development. Injection of miR-430 miRNAs rescues the brain defects in MZdicer mutants, revealing essential roles for miRNAs during morphogenesis.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">308</span>;<span class="number">5723</span>;<span class="pagerange">833-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15774722">15774722</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1109020">10.1126/science.1109020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CoGenT++: an extensive and extensible data environment for computational genomics.</h4>
<p class="authors"><span class="author">Goldovsky L</span>, <span class="author">Janssen P</span>, <span class="author">Ahr&eacute;n D</span>, <span class="author">Audit B</span>, <span class="author">Cases I</span>, <span class="author">Darzentas N</span>, <span class="author">Enright AJ</span>, <span class="author">L&oacute;pez-Bigas N</span>, <span class="author">Peregrin-Alvarez JM</span>, <span class="author">Smith M</span>, <span class="author">Tsoka S</span>, <span class="author">Kunin V</span> and <span class="author">Ouzounis CA</span></p>
<p class="affiliation">Computational Genomics Group, The European Bioinformatics Institute EMBL, Cambridge Outstation, Cambridge CB10 1SD, UK.</p><p class="abstract"><strong>Motivation</strong>: CoGenT++ is a data environment for computational research in comparative and functional genomics, designed to address issues of consistency, reproducibility, scalability and accessibility.</p><p class="abstract"><strong>Description</strong>: CoGenT++ facilitates the re-distribution of all fully sequenced and published genomes, storing information about species, gene names and protein sequences. We describe our scalable implementation of ProXSim, a continually updated all-against-all similarity database, which stores pairwise relationships between all genome sequences. Based on these similarities, derived databases are generated for gene fusions--AllFuse, putative orthologs--OFAM, protein families--TRIBES, phylogenetic profiles--ProfUse and phylogenetic trees. Extensions based on the CoGenT++ environment include disease gene prediction, pattern discovery, automated domain detection, genome annotation and ancestral reconstruction.</p><p class="abstract"><strong>Conclusion</strong>: CoGenT++ provides a comprehensive environment for computational genomics, accessible primarily for large-scale analyses as well as manual browsing.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">19</span>;<span class="pagerange">3806-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16216832" rel="external">16216832</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti579" rel="external">10.1093/bioinformatics/bti579</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insights into the P. y. yoelii hepatic stage transcriptome reveal complex transcriptional patterns.</h4>
<p class="authors"><span class="author">Gr&uuml;ner AC</span>, <span class="author">Hez-Deroubaix S</span>, <span class="author">Snounou G</span>, <span class="author">Hall N</span>, <span class="author">Bouchier C</span>, <span class="author">Letourneur F</span>, <span class="author">Landau I</span> and <span class="author">Druilhe P</span></p>
<p class="affiliation">Unit&eacute; de Parasitologie Bio-M&eacute;dicale, Institut Pasteur, 25 Rue du Dr Roux, 75731 Paris Cedex 15, France.</p><p class="abstract">During their complex life cycle, malaria parasites adopt morphologically, biochemically and immunologically distinct forms. The intra-hepatic form is the least known, yet of established value in the induction of sterile immunity and as a target for chemoprophylaxis. Using Plasmodium yoelii as a model we present here a novel approach to the elucidation of the transcriptome of this poorly studied stage. Sequences from Plasmodium were obtained in 388 of the 3533 inserts (11%) isolated from liver stages cDNA obtained from optimized cultures with high yields. These corresponded to a total of 88 putative P. yoelii genes. The majority of the transcribed genes identified, code for predicted proteins of as yet unknown function. The RT-PCR analysis carried out for 29 of these genes, confirmed expression at the hepatic stage and provided evidence for complex patterns of genes transcription in the distinct stages found in the mosquito and vertebrate host. The results demonstrate the efficacy of the approach that can now be applied to further detailed analysis of the hepatic stage transcriptome of Plasmodium.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2005</span>;<span class="volume">142</span>;<span class="number">2</span>;<span class="pagerange">184-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15913805" rel="external">15913805</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2005.03.019" rel="external">10.1016/j.molbiopara.2005.03.019</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">AMPA receptor trafficking and GluR1.</h4>
<p class="authors"><span class="author">Grant SG</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">310</span>;<span class="number">5746</span>;<span class="pagerange">234-5; author reply 234-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16224003">16224003</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.310.5746.234">10.1126/science.310.5746.234</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Synapse proteomics of multiprotein complexes: en route from genes to nervous system diseases.</h4>
<p class="authors"><span class="author">Grant SG</span>, <span class="author">Marshall MC</span>, <span class="author">Page KL</span>, <span class="author">Cumiskey MA</span> and <span class="author">Armstrong JD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, UK <a href="mailto:%73%67%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Proteomic experiments have produced a draft profile of the overall molecular composition of the mammalian neuronal synapse. It appears that synapses have over 1000 protein components and the mapping of their interactions, organization and functions will lead to a global view of the role of synapses in physiology and disease. A major functional subcomponent of the synaptic machinery is a multiprotein complex of glutamate receptors and adhesion proteins with associated adaptor and signalling enzymes totally 185 proteins known as the N-methyl-d-aspartate receptor complex/MAGUK associated signalling complex (NRC/MASC). Here, we review the proteomic studies and functions of NRC/MASC and specifically report on the role of its component genes in human diseases. Using a systematic literature search protocol, we identified reports of mutations or polymorphisms in 47 genes associated with 183 disorders, of which 54 were nervous system disorders. A similar number of genes are important in mouse synaptic plasticity and behaviour, where the NRC/MASC acts as a signalling complex with multiple functions provided by its individual protein components and their interactions. The individual gene mutations suggest not only an important role for the NRC/MASC in human diseases but that these diseases may be functionally connected by their common link to the NRC/MASC. The NRC/MASC is a rich source of genetic variation and provides a platform for understanding relationships of disease phenotype amenable to systematic studies such as the Genes to Cognition research consortium (www.genes2cognition.org) that links human and mouse genetics with proteomic studies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14 Spec No. 2</span>;<span class="pagerange">R225-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16150739" rel="external">16150739</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi330" rel="external">10.1093/hmg/ddi330</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complex nature of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes.</h4>
<p class="authors"><span class="author">Gribble SM</span>, <span class="author">Prigmore E</span>, <span class="author">Burford DC</span>, <span class="author">Porter KM</span>, <span class="author">Ng BL</span>, <span class="author">Douglas EJ</span>, <span class="author">Fiegler H</span>, <span class="author">Carr P</span>, <span class="author">Kalaitzopoulos D</span>, <span class="author">Clegg S</span>, <span class="author">Sandstrom R</span>, <span class="author">Temple IK</span>, <span class="author">Youings SA</span>, <span class="author">Thomas NS</span>, <span class="author">Dennis NR</span>, <span class="author">Jacobs PA</span>, <span class="author">Crolla JA</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Objective</strong>: To describe the systematic analysis of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes, characterise the structural chromosome rearrangements, map the translocation breakpoints, and report detectable genomic imbalances.</p><p class="abstract"><strong>Methods</strong>: DNA microarrays were used with a resolution of 1 Mb for the detailed genome-wide analysis of the patients. Array CGH was used to screen for genomic imbalance and array painting to map chromosome breakpoints rapidly. These two methods facilitate rapid analysis of translocation breakpoints and screening for cryptic chromosome imbalance. Breakpoints of rearrangements were further refined (to the level of spanning clones) using fluorescence in situ hybridisation where appropriate.</p><p class="abstract"><strong>Results</strong>: Unexpected additional complexity or genome imbalance was found in six of 10 patients studied. The patients could be grouped according to the general nature of the karyotype rearrangement as follows: (A) three cases with complex multiple rearrangements including deletions, inversions, and insertions at or near one or both breakpoints; (B) three cases in which, while the translocations appeared to be balanced, microarray analysis identified previously unrecognised imbalance on chromosomes unrelated to the translocation; (C) four cases in which the translocation breakpoints appeared simple and balanced at the resolution used.</p><p class="abstract"><strong>Conclusions</strong>: This high level of unexpected rearrangement complexity, if generally confirmed in the study of further patients, will have an impact on current diagnostic investigations of this type and provides an argument for the more widespread adoption of microarray analysis or other high resolution genome-wide screens for chromosome imbalance and rearrangement.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">42</span>;<span class="number">1</span>;<span class="pagerange">8-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15635069" rel="external">15635069</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735914" rel="external">1735914</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2004.024141" rel="external">10.1136/jmg.2004.024141</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Annotating non-coding RNAs with Rfam.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">Non-coding RNA (ncRNA) genes produce a functional RNA product, rather than a translated protein. The range and importance of such genes is only recently apparent, with known ncRNAs participating in a wide range of structural, regulatory, and catalytic roles within the cell. Like protein-coding genes, multiple sequence alignments of families of ncRNAs tell us much about their structure and function, and enable the formulation of statistical models for the detection of related sequences. Rfam is a database of families of ncRNAs, represented by structure-annotated multiple sequence alignments and covariance models.</p>
<p class="publication"><span class="title">Current protocols in bioinformatics / editoral board, Andreas D. Baxevanis ... [et al.]</span>&nbsp;<span class="year">2005</span>;<span class="volume">Chapter 12</span>;<span class="pagerange">Unit 12.5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18428745" rel="external">18428745</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471250953.bi1205s9" rel="external">10.1002/0471250953.bi1205s9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RALEE--RNA ALignment editor in Emacs.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute Wellcome Trust Genome Campus, Hinxton, CAMBS, CB10 1SA, UK. <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Production of high quality multiple sequence alignments of structured RNAs relies on an iterative combination of manual editing and structure prediction. An essential feature of an RNA alignment editor is the facility to mark-up the alignment based on how it matches a given secondary structure prediction, but few available alignment editors offer such a feature. The RALEE (RNA ALignment Editor in Emacs) tool provides a simple environment for RNA multiple sequence alignment editing, including structure-specific colour schemes, utilizing helper applications for structure prediction and many more conventional editing functions. This is accomplished by extending the commonly used text editor, Emacs, which is available for Linux, most UNIX systems, Windows and Mac OS. AVAILABILITY: The ELISP source code for RALEE is freely available from http://www.sanger.ac.uk/Users/sgj/ralee/ along with documentation and examples. CONTACT: <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">2</span>;<span class="pagerange">257-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15377506" rel="external">15377506</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbth489" rel="external">10.1093/bioinformatics/bth489</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rfam: annotating non-coding RNAs in complete genomes.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span>, <span class="author">Moxon S</span>, <span class="author">Marshall M</span>, <span class="author">Khanna A</span>, <span class="author">Eddy SR</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Rfam is a comprehensive collection of non-coding RNA (ncRNA) families, represented by multiple sequence alignments and profile stochastic context-free grammars. Rfam aims to facilitate the identification and classification of new members of known sequence families, and distributes annotation of ncRNAs in over 200 complete genome sequences. The data provide the first glimpses of conservation of multiple ncRNA families across a wide taxonomic range. A small number of large families are essential in all three kingdoms of life, with large numbers of smaller families specific to certain taxa. Recent improvements in the database are discussed, together with challenges for the future. Rfam is available on the Web at http://www.sanger.ac.uk/Software/Rfam/ and http://rfam.wustl.edu/.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Database issue</span>;<span class="pagerange">D121-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15608160">15608160</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540035">540035</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki081">10.1093/nar/gki081</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Posterior polymorphous corneal dystrophy in Czech families maps to chromosome 20 and excludes the VSX1 gene.</h4>
<p class="authors"><span class="author">Gwilliam R</span>, <span class="author">Liskova P</span>, <span class="author">Filipec M</span>, <span class="author">Kmoch S</span>, <span class="author">Jirsova K</span>, <span class="author">Huckle EJ</span>, <span class="author">Stables CL</span>, <span class="author">Bhattacharya SS</span>, <span class="author">Hardcastle AJ</span>, <span class="author">Deloukas P</span> and <span class="author">Ebenezer ND</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract"><strong>Purpose</strong>: Posterior polymorphous corneal dystrophy (PPCD) is an autosomal dominant disorder, affecting both the corneal endothelium and Descemet&#39;s membrane. In the Czech Republic, PPCD is one of the most prevalent corneal dystrophies. The purpose of this study was to determine the chromosomal locus of PPCD in two large Czech families, by using linkage analysis.</p><p class="abstract"><strong>Methods</strong>: Linkage analysis was performed on 52 members of two Czech families with PPCD and polymorphic microsatellite markers and lod scores were calculated. The candidate gene VSX1 was also screened for mutations.</p><p class="abstract"><strong>Results</strong>: Significant lod scores were obtained with microsatellite markers on chromosome 20. Linkage analysis delineated the Czech PPCD locus to a 2.7-cM locus on chromosome 20, region p11.2, between flanking markers D20S48 and D20S139, which excluded VSX1 as the disease-causing gene in both families. In addition, the exclusion of VSX1 was confirmed by sequence analysis.</p><p class="abstract"><strong>Conclusions</strong>: This study reports the localization of PPCD in patients of Czech origin to chromosome 20 at p11.2. Linkage data and sequence analysis exclude VSX1 as causative of PPCD in two Czech families. This refined locus for PPCD overlaps the congenital hereditary endothelial dystrophy (CHED1) disease interval, and it is possible that these corneal dystrophies are allelic.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Investigative ophthalmology &amp; visual science</span>&nbsp;<span class="year">2005</span>;<span class="volume">46</span>;<span class="number">12</span>;<span class="pagerange">4480-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16303937" rel="external">16303937</a>; DOI: <a href="http://dx.doi.org/10.1167%2Fiovs.05-0269" rel="external">10.1167/iovs.05-0269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comprehensive survey of the Plasmodium life cycle by genomic, transcriptomic, and proteomic analyses.</h4>
<p class="authors"><span class="author">Hall N</span>, <span class="author">Karras M</span>, <span class="author">Raine JD</span>, <span class="author">Carlton JM</span>, <span class="author">Kooij TW</span>, <span class="author">Berriman M</span>, <span class="author">Florens L</span>, <span class="author">Janssen CS</span>, <span class="author">Pain A</span>, <span class="author">Christophides GK</span>, <span class="author">James K</span>, <span class="author">Rutherford K</span>, <span class="author">Harris B</span>, <span class="author">Harris D</span>, <span class="author">Churcher C</span>, <span class="author">Quail MA</span>, <span class="author">Ormond D</span>, <span class="author">Doggett J</span>, <span class="author">Trueman HE</span>, <span class="author">Mendoza J</span>, <span class="author">Bidwell SL</span>, <span class="author">Rajandream MA</span>, <span class="author">Carucci DJ</span>, <span class="author">Yates JR</span>, <span class="author">Kafatos FC</span>, <span class="author">Janse CJ</span>, <span class="author">Barrell B</span>, <span class="author">Turner CM</span>, <span class="author">Waters AP</span> and <span class="author">Sinden RE</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6E%68%61%6C%6C%40%74%69%67%72.%6F%72%67">&#110;&#104;&#97;&#108;&#108;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Plasmodium berghei and Plasmodium chabaudi are widely used model malaria species. Comparison of their genomes, integrated with proteomic and microarray data, with the genomes of Plasmodium falciparum and Plasmodium yoelii revealed a conserved core of 4500 Plasmodium genes in the central regions of the 14 chromosomes and highlighted genes evolving rapidly because of stage-specific selective pressures. Four strategies for gene expression are apparent during the parasites&#39; life cycle: (i) housekeeping; (ii) host-related; (iii) strategy-specific related to invasion, asexual replication, and sexual development; and (iv) stage-specific. We observed posttranscriptional gene silencing through translational repression of messenger RNA during sexual development, and a 47-base 3&#39; untranslated region motif is implicated in this process.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">307</span>;<span class="number">5706</span>;<span class="pagerange">82-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15637271">15637271</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1103717">10.1126/science.1103717</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global effects on gene expression in fission yeast by silencing and RNA interference machineries.</h4>
<p class="authors"><span class="author">Hansen KR</span>, <span class="author">Burns G</span>, <span class="author">Mata J</span>, <span class="author">Volpe TA</span>, <span class="author">Martienssen RA</span>, <span class="author">B&auml;hler J</span> and <span class="author">Thon G</span></p>
<p class="affiliation">Department of Genetics, Institute of Molecular Biology, University of Copenhagen, &Oslash;ster Farimagsgade 2A, Copenhagen 1353 K, Denmark.</p><p class="abstract">Histone modifications influence gene expression in complex ways. The RNA interference (RNAi) machinery can repress transcription by recruiting histone-modifying enzymes to chromatin, although it is not clear whether this is a general mechanism for gene silencing or whether it requires repeated sequences such as long terminal repeats (LTRs). We analyzed the global effects of the Clr3 and Clr6 histone deacetylases, the Clr4 methyltransferase, the zinc finger protein Clr1, and the RNAi proteins Dicer, RdRP, and Argonaute on the transcriptome of Schizosaccharomyces pombe (fission yeast). The clr mutants derepressed similar subsets of genes, many of which also became transcriptionally activated in cells that were exposed to environmental stresses such as nitrogen starvation. Many genes that were repressed by the Clr proteins clustered in extended regions close to the telomeres. Surprisingly few genes were repressed by both the silencing and RNAi machineries, with transcripts from centromeric repeats and Tf2 retrotransposons being notable exceptions. We found no correlation between repression by RNAi and proximity to LTRs, and the wtf family of repeated sequences seems to be repressed by histone deacetylation independent of RNAi. Our data indicate that the RNAi and Clr proteins show only a limited functional overlap and that the Clr proteins play more global roles in gene silencing.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; NIGMS NIH HHS: <span class="grant">GM067014</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">2</span>;<span class="pagerange">590-601</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15632061" rel="external">15632061</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC543407" rel="external">543407</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.25.2.590-601.2005" rel="external">10.1128/MCB.25.2.590-601.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCF(Pof1)-ubiquitin and its target Zip1 transcription factor mediate cadmium response in fission yeast.</h4>
<p class="authors"><span class="author">Harrison C</span>, <span class="author">Katayama S</span>, <span class="author">Dhut S</span>, <span class="author">Chen D</span>, <span class="author">Jones N</span>, <span class="author">B&auml;hler J</span> and <span class="author">Toda T</span></p>
<p class="affiliation">Laboratory of Cell Regulation, Lincoln&#39;s Inn Fields Laboratories, Cancer Research UK, London Research Institute, London, UK.</p><p class="abstract">Ubiquitin-dependent proteolysis regulates gene expression in many eukaryotic systems. Pof1 is an essential fission yeast F-box protein that is homologous to budding yeast Met30. Temperature-sensitive pof1 mutants display acute growth arrest with small cell size. Extragenic suppressor analysis identified Zip1, a bZIP (basic leucine zipper) transcription factor, as a target for Pof1. We show Zip1 is stabilized in pof1 mutants, Pof1 binds only phosphorylated forms of Zip1, and Zip1 is ubiquitylated in vivo, indicating that Zip1 is a substrate of SCF(Pof1). Genome-wide DNA microarray assay shows that many cadmium-induced genes are under the control of Zip1, suggesting Zip1 plays a role in cadmium response. Consistently, zip1 mutants are hypersensitive to cadmium and unlike wild type, lose cell viability under this stress. Intriguingly, cadmium exposure results in upregulation of Zip1 levels and leads wild-type cells to growth arrest with reduced cell size, reminiscent of pof1 phenotypes. Our results indicate that Zip1 mediates growth arrest in cadmium response, which is essential to maintain viability. Normally growing cells prevent this response through constitutive ubiquitylation and degradation of Zip1 via SCF(Pof1).</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2005</span>;<span class="volume">24</span>;<span class="number">3</span>;<span class="pagerange">599-610</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15660136" rel="external">15660136</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC548646" rel="external">548646</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.emboj.7600536" rel="external">10.1038/sj.emboj.7600536</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Typhoid and paratyphoid fever.</h4>
<p class="authors"><span class="author">Hasan R</span>, <span class="author">Cooke FJ</span>, <span class="author">Nair S</span>, <span class="author">Harish BN</span> and <span class="author">Wain J</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2005</span>;<span class="volume">366</span>;<span class="number">9497</span>;<span class="pagerange">1603-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16271635" rel="external">16271635</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(05)67654-4" rel="external">10.1016/S0140-6736(05)67654-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two new mouse mutants with vestibular defects that map to the highly mutable locus on chromosome 4.</h4>
<p class="authors"><span class="author">Hawker K</span>, <span class="author">Fuchs H</span>, <span class="author">Angelis MH</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">MRC Institute of Hearing Research, Nottingham, UK.</p><p class="abstract">The purpose of this study was to characterise two new mouse mutants, carousel, and whirligig. Both were derived from a large-scale mutagenesis programme which screened for dominantly inherited mutations that cause hearing impairments and balance defects. Genetic mapping placed both mutations on the proximal region of chromosome 4. Paint-filling and clearing techniques revealed abnormalities of the lateral semicircular canal. Scanning electron microscopy showed increased numbers of outer and inner hair cells in the apical region of the organ of Corti. The behavioural, genetic, and morphological characteristics lead us to the conclusion that both mutants are probably alleles of seven previously identified mutants which all map to proximal chromosome 4 and share similar defects of the lateral semicircular canal. We suggest that this region may be particularly susceptible to ENU mutagenesis independent of genetic background.</p>
<p class="publication"><span class="title">International journal of audiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">171-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15916118" rel="external">15916118</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beta1,3-N-acetylglucosaminyltransferase 1 glycosylation is required for axon pathfinding by olfactory sensory neurons.</h4>
<p class="authors"><span class="author">Henion TR</span>, <span class="author">Raitcheva D</span>, <span class="author">Grosholz R</span>, <span class="author">Biellmann F</span>, <span class="author">Skarnes WC</span>, <span class="author">Hennet T</span> and <span class="author">Schwarting GA</span></p>
<p class="affiliation">Shriver Center, Waltham, Massachusetts 02452, USA.</p><p class="abstract">During embryonic development, axons from sensory neurons in the olfactory epithelium (OE) extend into the olfactory bulb (OB) where they synapse with projection neurons and form glomerular structures. To determine whether glycans play a role in these processes, we analyzed mice deficient for the glycosyltransferase beta1,3-N-acetylglucosaminyltransferase 1 (beta3GnT1), a key enzyme in lactosamine glycan synthesis. Terminal lactosamine expression, as shown by immunoreactivity with the monoclonal antibody 1B2, is dramatically reduced in the neonatal null OE. Postnatal beta3GnT1-/- mice exhibit severely disorganized OB innervation and defective glomerular formation. Beginning in embryonic development, specific subsets of odorant receptor-expressing neurons are progressively lost from the OE of null mice, which exhibit a postnatal smell perception deficit. Axon guidance errors and increased neuronal cell death result in an absence of P2, I7, and M72 glomeruli, indicating a reduction in the repertoire of odorant receptor-specific glomeruli. By approximately 2 weeks of age, lactosamine is unexpectedly reexpressed in sensory neurons of null mice through a secondary pathway, which is accompanied by the regrowth of axons into the OB glomerular layer and the return of smell perception. Thus, both neonatal OE degeneration and the postnatal regeneration are lactosamine dependent. Lactosamine expression in beta3GnT1-/- mice is also reduced in pheromone-receptive vomeronasal neurons and dorsal root ganglion cells, suggesting that beta3GnT1 may perform a conserved function in multiple sensory systems. These results reveal an essential role for lactosamine in sensory axon pathfinding and in the formation of OB synaptic connections.</p>
<p class="grants">Funded by: NIDCD NIH HHS: <span class="grant">DC00953</span>, <span class="grant">DC06496</span></p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">8</span>;<span class="pagerange">1894-903</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15728829" rel="external">15728829</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.4654-04.2005" rel="external">10.1523/JNEUROSCI.4654-04.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A feast of protozoan genomes.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span>, <span class="author">Berriman M</span> and <span class="author">Pain A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">9</span>;<span class="pagerange">670-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16145753" rel="external">16145753</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1237" rel="external">10.1038/nrmicro1237</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics in C. elegans: so many genes, such a little worm.</h4>
<p class="authors"><span class="author">Hillier LW</span>, <span class="author">Coulson A</span>, <span class="author">Murray JI</span>, <span class="author">Bao Z</span>, <span class="author">Sulston JE</span> and <span class="author">Waterston RH</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University School of Medicine, St. Louis, Missouri 63108, USA.</p><p class="abstract">The Caenorhabditis elegans genome sequence is now complete, fully contiguous telomere to telomere and totaling 100,291,840 bp. The sequence has catalyzed the collection of systematic data sets and analyses, including a curated set of 19,735 protein-coding genes--with &gt;90% directly supported by experimental evidence--and &gt;1300 noncoding RNA genes. High-throughput efforts are under way to complete the gene sets, along with studies to characterize gene expression, function, and regulation on a genome-wide scale. The success of the worm project has had a profound effect on genome sequencing and on genomics more broadly. We now have a solid platform on which to build toward the lofty goal of a true molecular understanding of worm biology with all its implications including those for human health.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">12</span>;<span class="pagerange">1651-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16339362" rel="external">16339362</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.3729105" rel="external">10.1101/gr.3729105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Facilitating genome navigation: survey sequencing and dense radiation-hybrid gene mapping.</h4>
<p class="authors"><span class="author">Hitte C</span>, <span class="author">Madeoy J</span>, <span class="author">Kirkness EF</span>, <span class="author">Priat C</span>, <span class="author">Lorentzen TD</span>, <span class="author">Senger F</span>, <span class="author">Thomas D</span>, <span class="author">Derrien T</span>, <span class="author">Ramirez C</span>, <span class="author">Scott C</span>, <span class="author">Evanno G</span>, <span class="author">Pullar B</span>, <span class="author">Cadieu E</span>, <span class="author">Oza V</span>, <span class="author">Lourgant K</span>, <span class="author">Jaffe DB</span>, <span class="author">Tacher S</span>, <span class="author">Dr&eacute;ano S</span>, <span class="author">Berkova N</span>, <span class="author">Andr&eacute; C</span>, <span class="author">Deloukas P</span>, <span class="author">Fraser C</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Ostrander EA</span> and <span class="author">Galibert F</span></p>
<p class="affiliation">CNRS, UMR 6061, G&eacute;n&eacute;tique et d&eacute;veloppement, Faculte de M&eacute;decine, Rennes, France.</p><p class="abstract">Accurate and comprehensive sequence coverage for large genomes has been restricted to only a few species of specific interest. Lower sequence coverage (survey sequencing) of related species can yield a wealth of information about gene content and putative regulatory elements. But survey sequences lack long-range continuity and provide only a fragmented view of a genome. Here we show the usefulness of combining survey sequencing with dense radiation-hybrid (RH) maps for extracting maximum comparative genome information from model organisms. Based on results from the canine system, we propose that from now on all low-pass sequencing projects should be accompanied by a dense, gene-based RH map-construction effort to extract maximum information from the genome with a marginal extra cost.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">643-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16012527">16012527</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1658">10.1038/nrg1658</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Food for thought.</h4>
<p class="authors"><span class="author">Holden M</span>, <span class="author">Rajandream MA</span> and <span class="author">Bentley S</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">12</span>;<span class="pagerange">912-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16350281">16350281</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1292">10.1038/nrmicro1292</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microbial mariners.</h4>
<p class="authors"><span class="author">Holden M</span>, <span class="author">Thomson N</span> and <span class="author">Bentley S</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">10</span>;<span class="pagerange">748-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16231859">16231859</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1262">10.1038/nrmicro1262</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of paired serum samples for serodiagnosis of typhoid fever.</h4>
<p class="authors"><span class="author">House D</span>, <span class="author">Chinh NT</span>, <span class="author">Diep TS</span>, <span class="author">Parry CM</span>, <span class="author">Wain J</span>, <span class="author">Dougan G</span>, <span class="author">White NJ</span>, <span class="author">Hien TT</span> and <span class="author">Farrar JJ</span></p>
<p class="affiliation">Centre for Molecular Microbiology and Infection, Department of Biological Sciences, Imperial College London, South Kensington, UK. <a href="mailto:%64.%68%6F%75%73%65%40%69%63.%61%63.%75%6B">&#100;&#46;&#104;&#111;&#117;&#115;&#101;&#64;&#105;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Using an enzyme-linked immunosorbent assay we demonstrate that, in adult patients with typhoid fever, the sensitivity of a serological test based on the detection of anti-lipopolysaccharide immunoglobulin G is increased when used with paired serum samples taken 1 week apart.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">43</span>;<span class="number">9</span>;<span class="pagerange">4889-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16145168" rel="external">16145168</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1234064" rel="external">1234064</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.43.9.4889-4890.2005" rel="external">10.1128/JCM.43.9.4889-4890.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Resin tissue microarrays: a universal format for immunohistochemistry.</h4>
<p class="authors"><span class="author">Howat WJ</span>, <span class="author">Warford A</span>, <span class="author">Mitchell JN</span>, <span class="author">Clarke KF</span>, <span class="author">Conquer JS</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">Atlas of Protein Expression Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK. <a href="mailto:%77%6A%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#119;&#106;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Tissue microarray (TMA) technology allows the miniaturization and characterization of multiple tissue samples on a single slide and commonly uses formalin-fixed paraffin-embedded (FFPE) tissue or acetone-fixed frozen tissue. The former provides good morphology but can compromise antigenicity, whereas the latter provides compromised morphology with good antigenicity. Here, we report the development of TMAs in glycol methacrylate resin, which combine the advantages of both methods in one embedding format. Freshly collected tissue fixed in -20C acetone or 10% neutral buffered formaldehyde were cored and arrayed into an intermediary medium of 2% agarose before infiltration of the agarose array with glycol methacrylate resin. Acetone-fixed resin TMA demonstrated improved morphology over acetone-fixed frozen TMA, with no loss of antigenicity. Staining for extracellular, cell surface, and nuclear antigens could be realized with monoclonal and polyclonal antibodies as well as with monomeric single-chain Fv preparations. In addition, when compared with FFPE TMA, formalin-fixed tissue in a resin TMA gave enhanced morphology and subcellular detail. Therefore, resin provides a universal format for the construction of TMAs, providing improved tissue morphology while retaining antigenicity, allows thin-section preparation, and could be used to replace preparation of frozen and FFPE TMAs for freshly collected tissue.</p>
<p class="publication"><span class="title">The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</span>&nbsp;<span class="year">2005</span>;<span class="volume">53</span>;<span class="number">10</span>;<span class="pagerange">1189-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15983117" rel="external">15983117</a>; DOI: <a href="http://dx.doi.org/10.1369%2Fjhc.5C6659.2005" rel="external">10.1369/jhc.5C6659.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phylogenomic study of the subfamily Caprinae by cross-species chromosome painting with Chinese muntjac paints.</h4>
<p class="authors"><span class="author">Huang L</span>, <span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Su W</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Key Laboratory of Cellular and Molecular Evolution, Kunming Institute of Zoology, The Chinese Academy of Sciences, Kunming, Yunnan, 650223, PR China.</p><p class="abstract">Chromosomal homologies have been established between the Chinese muntjac (Muntiacus reevesi, MRE, 2n = 46) and five ovine species: wild goat (Capra aegagrus, CAE, 2n = 60), argali (Ovis ammon, OAM, 2n = 56), snow sheep (Ovis nivicola, ONI, 2n = 52), red goral (Naemorhedus cranbrooki, NCR, 2n = 56) and Sumatra serow (Capricornis sumatraensis, CSU, 2n = 48) by chromosome painting with a set of chromosome-specific probes of the Chinese muntjac. In total, twenty-two Chinese muntjac autosomal painting probes detected thirty-five homologous segments in the genome of each species. The chromosome X probe hybridized to the whole X chromosomes of all ovine species while the chromosome Y probe gave no signal. Our results demonstrate that almost all homologous segments defined by comparative painting show a high degree of conservation in G-banding patterns and that each speciation event is accompanied by specific chromosomal rearrangements. The combined analysis of our results and previous cytogenetic and molecular systematic results enables us to map the chromosomal rearrangements onto a phylogenetic tree, thus providing new insights into the karyotypic evolution of these species.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">389-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15973503" rel="external">15973503</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-005-0977-6" rel="external">10.1007/s10577-005-0977-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptome analysis for the chicken based on 19,626 finished cDNA sequences and 485,337 expressed sequence tags.</h4>
<p class="authors"><span class="author">Hubbard SJ</span>, <span class="author">Grafham DV</span>, <span class="author">Beattie KJ</span>, <span class="author">Overton IM</span>, <span class="author">McLaren SR</span>, <span class="author">Croning MD</span>, <span class="author">Boardman PE</span>, <span class="author">Bonfield JK</span>, <span class="author">Burnside J</span>, <span class="author">Davies RM</span>, <span class="author">Farrell ER</span>, <span class="author">Francis MD</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Humphray SJ</span>, <span class="author">Hyland C</span>, <span class="author">Scott CE</span>, <span class="author">Tang H</span>, <span class="author">Taylor RG</span>, <span class="author">Tickle C</span>, <span class="author">Brown WR</span>, <span class="author">Birney E</span>, <span class="author">Rogers J</span> and <span class="author">Wilson SA</span></p>
<p class="affiliation">Faculty of Life Sciences, The University of Manchester, Manchester, M60 1QD, United Kingdom.</p><p class="abstract">We present an analysis of the chicken (Gallus gallus) transcriptome based on the full insert sequences for 19,626 cDNAs, combined with 485,337 EST sequences. The cDNA data set has been functionally annotated and describes a minimum of 11,929 chicken coding genes, including the sequence for 2260 full-length cDNAs together with a collection of noncoding (nc) cDNAs that have been stringently filtered to remove untranslated regions of coding mRNAs. The combined collection of cDNAs and ESTs describe 62,546 clustered transcripts and provide transcriptional evidence for a total of 18,989 chicken genes, including 88% of the annotated Ensembl gene set. Analysis of the ncRNAs reveals a set that is highly conserved in chickens and mammals, including sequences for 14 pri-miRNAs encoding 23 different miRNAs. The data sets described here provide a transcriptome toolkit linked to physical clones for bioinformaticians and experimental biologists who wish to use chicken systems as a low-cost, accessible alternative to mammals for the analysis of vertebrate development, immunology, and cell biology.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">1</span>;<span class="pagerange">174-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15590942" rel="external">15590942</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540287" rel="external">540287</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.3011405" rel="external">10.1101/gr.3011405</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2005.</h4>
<p class="authors"><span class="author">Hubbard T</span>, <span class="author">Andrews D</span>, <span class="author">Caccamo M</span>, <span class="author">Cameron G</span>, <span class="author">Chen Y</span>, <span class="author">Clamp M</span>, <span class="author">Clarke L</span>, <span class="author">Coates G</span>, <span class="author">Cox T</span>, <span class="author">Cunningham F</span>, <span class="author">Curwen V</span>, <span class="author">Cutts T</span>, <span class="author">Down T</span>, <span class="author">Durbin R</span>, <span class="author">Fernandez-Suarez XM</span>, <span class="author">Gilbert J</span>, <span class="author">Hammond M</span>, <span class="author">Herrero J</span>, <span class="author">Hotz H</span>, <span class="author">Howe K</span>, <span class="author">Iyer V</span>, <span class="author">Jekosch K</span>, <span class="author">Kahari A</span>, <span class="author">Kasprzyk A</span>, <span class="author">Keefe D</span>, <span class="author">Keenan S</span>, <span class="author">Kokocinsci F</span>, <span class="author">London D</span>, <span class="author">Longden I</span>, <span class="author">McVicker G</span>, <span class="author">Melsopp C</span>, <span class="author">Meidl P</span>, <span class="author">Potter S</span>, <span class="author">Proctor G</span>, <span class="author">Rae M</span>, <span class="author">Rios D</span>, <span class="author">Schuster M</span>, <span class="author">Searle S</span>, <span class="author">Severin J</span>, <span class="author">Slater G</span>, <span class="author">Smedley D</span>, <span class="author">Smith J</span>, <span class="author">Spooner W</span>, <span class="author">Stabenau A</span>, <span class="author">Stalker J</span>, <span class="author">Storey R</span>, <span class="author">Trevanion S</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Vogel J</span>, <span class="author">White S</span>, <span class="author">Woodwark C</span> and <span class="author">Birney E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The Ensembl (http://www.ensembl.org/) project provides a comprehensive and integrated source of annotation of large genome sequences. Over the last year the number of genomes available from the Ensembl site has increased by 7 to 16, with the addition of the six vertebrate genomes of chimpanzee, dog, cow, chicken, tetraodon and frog and the insect genome of honeybee. The majority have been annotated automatically using the Ensembl gene build system, showing its flexibility to reliably annotate a wide variety of genomes. With the increased number of vertebrate genomes, the comparative analysis provided to users has been greatly improved, with new website interfaces allowing annotation of different genomes to be directly compared. The Ensembl software system is being increasingly widely reused in different projects showing the benefits of a completely open approach to software development and distribution.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Database issue</span>;<span class="pagerange">D447-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15608235">15608235</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540092">540092</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki138">10.1093/nar/gki138</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">How homologous recombination generates a mutable genome.</h4>
<p class="authors"><span class="author">Hurles M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%65%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#101;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Recombination and mutation have traditionally been regarded as independent evolutionary processes: the latter generates variation, which the former reshuffles. Recent studies, however, have suggested that allelic recombination influences the underlying mutation rate, as high mutation rates are inferred in regions of high recombination. Furthermore, recombination between duplicated sequences introduces structural variation into the human genome and facilitates the formation of clustered gene families. Comparisons of whole-genome sequences reveal the expansion of gene family clusters to be an important mode of genome evolution. The negative aspect of this genomic dynamism is the contribution of these rearrangements to genetic diseases.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">179-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16197735">16197735</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3525126">3525126</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The dual origin of the Malagasy in Island Southeast Asia and East Africa: evidence from maternal and paternal lineages.</h4>
<p class="authors"><span class="author">Hurles ME</span>, <span class="author">Sykes BC</span>, <span class="author">Jobling MA</span> and <span class="author">Forster P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom. <a href="mailto:%6D%65%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#101;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Linguistic and archaeological evidence about the origins of the Malagasy, the indigenous peoples of Madagascar, points to mixed African and Indonesian ancestry. By contrast, genetic evidence about the origins of the Malagasy has hitherto remained partial and imprecise. We defined 26 Y-chromosomal lineages by typing 44 Y-chromosomal polymorphisms in 362 males from four different ethnic groups from Madagascar and 10 potential ancestral populations in Island Southeast Asia and the Pacific. We also compared mitochondrial sequence diversity in the Malagasy with a manually curated database of 19,371 hypervariable segment I sequences, incorporating both published and unpublished data. We could attribute every maternal and paternal lineage found in the Malagasy to a likely geographic origin. Here, we demonstrate approximately equal African and Indonesian contributions to both paternal and maternal Malagasy lineages. The most likely origin of the Asia-derived paternal lineages found in the Malagasy is Borneo. This agrees strikingly with the linguistic evidence that the languages spoken around the Barito River in southern Borneo are the closest extant relatives of Malagasy languages. As a result of their equally balanced admixed ancestry, the Malagasy may represent an ideal population in which to identify loci underlying complex traits of both anthropological and medical interest.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">76</span>;<span class="number">5</span>;<span class="pagerange">894-901</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15793703">15793703</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1199379">1199379</a>; DOI: <a href="http://dx.doi.org/10.1086%2F430051">10.1086/430051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Isolation and characterization of murine Cds (CDP-diacylglycerol synthase) 1 and 2.</h4>
<p class="authors"><span class="author">Inglis-Broadgate SL</span>, <span class="author">Ocaka L</span>, <span class="author">Banerjee R</span>, <span class="author">Gaasenbeek M</span>, <span class="author">Chapple JP</span>, <span class="author">Cheetham ME</span>, <span class="author">Clark BJ</span>, <span class="author">Hunt DM</span> and <span class="author">Halford S</span></p>
<p class="affiliation">Institute of Ophthalmology, University College London, 11-43 Bath Street, London, EC1V 9EL, UK.</p><p class="abstract">Phototransduction in Drosophila is a phosphoinositide-mediated signalling pathway. Phosphatidylinositol 4,5-bisphosphate (PIP2) plays a central role in this process, and its levels are tightly regulated. A photoreceptor-specific form of the enzyme CDP-diacylglycerol synthase (CDS), which catalyzes the formation of CDP-diacylglycerol from phosphatidic acid, is a key regulator of the amount of PIP2 available for signalling. cds mutants develop light-induced retinal degeneration. We report here the isolation and characterization of two murine genes encoding this enzyme, Cds1 and Cds2. The genes encode proteins that are 73% identical and 92% similar but exhibit very different expression patterns. Cds1 shows a very restricted expression pattern but is expressed in the inner segments of the photoreceptors whilst Cds2 shows a ubiquitous pattern of expression. Using fluorescent in situ hybridization we have mapped Cds1 and Cds2 to chromosomes 5E3 and 2G1 respectively. These are regions of synteny with the corresponding human gene localization (4q21 and 20p13). Transient transfection experiments with epitope tagged proteins have also demonstrated that both are associated with the endoplasmic reticulum.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">2005</span>;<span class="volume">356</span>;<span class="pagerange">19-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16023307" rel="external">16023307</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gene.2005.04.037" rel="external">10.1016/j.gene.2005.04.037</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A haplotype map of the human genome.</h4>
<p class="authors"><span class="author">International HapMap Consortium</span></p><p class="abstract">Inherited genetic variation has a critical but as yet largely uncharacterized role in human disease. Here we report a public database of common variation in the human genome: more than one million single nucleotide polymorphisms (SNPs) for which accurate and complete genotypes have been obtained in 269 DNA samples from four populations, including ten 500-kilobase regions in which essentially all information about common DNA variation has been extracted. These data document the generality of recombination hotspots, a block-like structure of linkage disequilibrium and low haplotype diversity, leading to substantial correlations of SNPs with many of their neighbours. We show how the HapMap resource can guide the design and analysis of genetic association studies, shed light on structural variation and recombination, and identify loci that may have been subject to natural selection during human evolution.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG001720</span>, <span class="grant">R01 HG001720-06</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">437</span>;<span class="number">7063</span>;<span class="pagerange">1299-320</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16255080">16255080</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1880871">1880871</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature04226">10.1038/nature04226</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the kinetoplastid parasite, Leishmania major.</h4>
<p class="authors"><span class="author">Ivens AC</span>, <span class="author">Peacock CS</span>, <span class="author">Worthey EA</span>, <span class="author">Murphy L</span>, <span class="author">Aggarwal G</span>, <span class="author">Berriman M</span>, <span class="author">Sisk E</span>, <span class="author">Rajandream MA</span>, <span class="author">Adlem E</span>, <span class="author">Aert R</span>, <span class="author">Anupama A</span>, <span class="author">Apostolou Z</span>, <span class="author">Attipoe P</span>, <span class="author">Bason N</span>, <span class="author">Bauser C</span>, <span class="author">Beck A</span>, <span class="author">Beverley SM</span>, <span class="author">Bianchettin G</span>, <span class="author">Borzym K</span>, <span class="author">Bothe G</span>, <span class="author">Bruschi CV</span>, <span class="author">Collins M</span>, <span class="author">Cadag E</span>, <span class="author">Ciarloni L</span>, <span class="author">Clayton C</span>, <span class="author">Coulson RM</span>, <span class="author">Cronin A</span>, <span class="author">Cruz AK</span>, <span class="author">Davies RM</span>, <span class="author">De Gaudenzi J</span>, <span class="author">Dobson DE</span>, <span class="author">Duesterhoeft A</span>, <span class="author">Fazelina G</span>, <span class="author">Fosker N</span>, <span class="author">Frasch AC</span>, <span class="author">Fraser A</span>, <span class="author">Fuchs M</span>, <span class="author">Gabel C</span>, <span class="author">Goble A</span>, <span class="author">Goffeau A</span>, <span class="author">Harris D</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Hilbert H</span>, <span class="author">Horn D</span>, <span class="author">Huang Y</span>, <span class="author">Klages S</span>, <span class="author">Knights A</span>, <span class="author">Kube M</span>, <span class="author">Larke N</span>, <span class="author">Litvin L</span>, <span class="author">Lord A</span>, <span class="author">Louie T</span>, <span class="author">Marra M</span>, <span class="author">Masuy D</span>, <span class="author">Matthews K</span>, <span class="author">Michaeli S</span>, <span class="author">Mottram JC</span>, <span class="author">M&uuml;ller-Auer S</span>, <span class="author">Munden H</span>, <span class="author">Nelson S</span>, <span class="author">Norbertczak H</span>, <span class="author">Oliver K</span>, <span class="author">O&#39;neil S</span>, <span class="author">Pentony M</span>, <span class="author">Pohl TM</span>, <span class="author">Price C</span>, <span class="author">Purnelle B</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Reinhardt R</span>, <span class="author">Rieger M</span>, <span class="author">Rinta J</span>, <span class="author">Robben J</span>, <span class="author">Robertson L</span>, <span class="author">Ruiz JC</span>, <span class="author">Rutter S</span>, <span class="author">Saunders D</span>, <span class="author">Sch&auml;fer M</span>, <span class="author">Schein J</span>, <span class="author">Schwartz DC</span>, <span class="author">Seeger K</span>, <span class="author">Seyler A</span>, <span class="author">Sharp S</span>, <span class="author">Shin H</span>, <span class="author">Sivam D</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Tosato V</span>, <span class="author">Vogt C</span>, <span class="author">Volckaert G</span>, <span class="author">Wambutt R</span>, <span class="author">Warren T</span>, <span class="author">Wedler H</span>, <span class="author">Woodward J</span>, <span class="author">Zhou S</span>, <span class="author">Zimmermann W</span>, <span class="author">Smith DF</span>, <span class="author">Blackwell JM</span>, <span class="author">Stuart KD</span>, <span class="author">Barrell B</span> and <span class="author">Myler PJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%61%6C%69%63%61%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#105;&#99;&#97;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Leishmania species cause a spectrum of human diseases in tropical and subtropical regions of the world. We have sequenced the 36 chromosomes of the 32.8-megabase haploid genome of Leishmania major (Friedlin strain) and predict 911 RNA genes, 39 pseudogenes, and 8272 protein-coding genes, of which 36% can be ascribed a putative function. These include genes involved in host-pathogen interactions, such as proteolytic enzymes, and extensive machinery for synthesis of complex surface glycoconjugates. The organization of protein-coding genes into long, strand-specific, polycistronic clusters and lack of general transcription factors in the L. major, Trypanosoma brucei, and Trypanosoma cruzi (Tritryp) genomes suggest that the mechanisms regulating RNA polymerase II-directed transcription are distinct from those operating in other eukaryotes, although the trypanosomatids appear capable of chromatin remodeling. Abundant RNA-binding proteins are encoded in the Tritryp genomes, consistent with active posttranscriptional regulation of gene expression.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI040599</span>, <span class="grant">R01 AI053667</span>, <span class="grant">U01 AI040599</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5733</span>;<span class="pagerange">436-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16020728">16020728</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1470643">1470643</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1112680">10.1126/science.1112680</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evidence for widespread reticulate evolution within human duplicons.</h4>
<p class="authors"><span class="author">Jackson MS</span>, <span class="author">Oliver K</span>, <span class="author">Loveland J</span>, <span class="author">Humphray S</span>, <span class="author">Dunham I</span>, <span class="author">Rocchi M</span>, <span class="author">Viggiano L</span>, <span class="author">Park JP</span>, <span class="author">Hurles ME</span> and <span class="author">Santibanez-Koref M</span></p>
<p class="affiliation">Institute of Human Genetics, University of Newcastle upon Tyne, International Centre for Life, Newcastle upon Tyne, United Kingdom. <a href="mailto:%6D.%73.%6A%61%63%6B%73%6F%6E%40%6E%63%6C.%61%63.%75%6B">&#109;&#46;&#115;&#46;&#106;&#97;&#99;&#107;&#115;&#111;&#110;&#64;&#110;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Approximately 5% of the human genome consists of segmental duplications that can cause genomic mutations and may play a role in gene innovation. Reticulate evolutionary processes, such as unequal crossing-over and gene conversion, are known to occur within specific duplicon families, but the broader contribution of these processes to the evolution of human duplications remains poorly characterized. Here, we use phylogenetic profiling to analyze multiple alignments of 24 human duplicon families that span &gt;8 Mb of DNA. Our results indicate that none of them are evolving independently, with all alignments showing sharp discontinuities in phylogenetic signal consistent with reticulation. To analyze these results in more detail, we have developed a quartet method that estimates the relative contribution of nucleotide substitution and reticulate processes to sequence evolution. Our data indicate that most of the duplications show a highly significant excess of sites consistent with reticulate evolution, compared with the number expected by nucleotide substitution alone, with 15 of 30 alignments showing a &gt;20-fold excess over that expected. Using permutation tests, we also show that at least 5% of the total sequence shares 100% sequence identity because of reticulation, a figure that includes 74 independent tracts of perfect identity &gt;2 kb in length. Furthermore, analysis of a subset of alignments indicates that the density of reticulation events is as high as 1 every 4 kb. These results indicate that phylogenetic relationships within recently duplicated human DNA can be rapidly disrupted by reticulate evolution. This finding has important implications for efforts to finish the human genome sequence, complicates comparative sequence analysis of duplicon families, and could profoundly influence the tempo of gene-family evolution.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">77</span>;<span class="number">5</span>;<span class="pagerange">824-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16252241" rel="external">16252241</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1271390" rel="external">1271390</a>; DOI: <a href="http://dx.doi.org/10.1086%2F497704" rel="external">10.1086/497704</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Activation of AP-1-dependent transcription by a truncated translation initiation factor.</h4>
<p class="authors"><span class="author">Jenkins CC</span>, <span class="author">Mata J</span>, <span class="author">Crane RF</span>, <span class="author">Thomas B</span>, <span class="author">Akoulitchev A</span>, <span class="author">B&auml;hler J</span> and <span class="author">Norbury CJ</span></p>
<p class="affiliation">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, United Kingdom.</p><p class="abstract">Int6/eIF3e is a highly conserved subunit of eukaryotic translation initiation factor 3 (eIF3) that has also been reported to interact with subunits of the proteasome and the COP9 signalosome. Overexpression of full-length Int6 or a 13-kDa C-terminal fragment, Int6CT, in the fission yeast Schizosaccharomyces pombe causes multidrug resistance that requires the otherwise inessential AP-1 transcription factor Pap1. Here we show for the first time that Int6CT acts to increase the transcriptional activity of Pap1. Microarray hybridization data indicate that Int6CT overexpression resulted in the up-regulation of 67 genes; this expression profile closely matched that of cells overexpressing Pap1. Analysis of the upstream regulatory sequences of these genes showed that the majority contained AP-1 consensus binding sites. Partial defects in ubiquitin-dependent proteolysis have been suggested to confer Pap1-dependent multidrug resistance, but no such defect was seen on Int6CT overexpression. Indeed, none of the previously identified interactions of endogenous Int6 was required for the activation of Pap1 transcription described here. Moreover, Int6CT-induced activation of Pap1-responsive gene expression was independent of the ability of Pap1 to undergo a redox-regulated conformational change which mediates its relocalization to the nucleus and expression of oxidative stress response genes. Int6CT therefore activates Pap1-dependent transcription by a novel mechanism.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2005</span>;<span class="volume">4</span>;<span class="number">11</span>;<span class="pagerange">1840-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16278451" rel="external">16278451</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1287857" rel="external">1287857</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.4.11.1840-1850.2005" rel="external">10.1128/EC.4.11.1840-1850.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural relatedness of plant food allergens with specific reference to cross-reactive allergens: an in silico analysis.</h4>
<p class="authors"><span class="author">Jenkins JA</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Shewry PR</span>, <span class="author">Breiteneder H</span> and <span class="author">Mills EN</span></p>
<p class="affiliation">Institute of Food Research, Norwich, Cambridge, United Kingdom.</p><p class="abstract"><strong>Background</strong>: The body of sequence and structural information on allergens and the sequence analysis of whole plant genomes are facilitating the application of bioinformatic approaches to identifying and defining plant allergens.</p><p class="abstract"><strong>Objective</strong>: An in silico approach was used to quantify the distribution of plant food allergen sequences across protein families and to develop and apply a novel means of assessing conserved surface features important for IgE cross-reactivity.</p><p class="abstract"><strong>Methods</strong>: Plant food allergen sequences were classified into Pfam families on the basis of sequence homology. Contact surface areas of selected proteins were calculated with MOLMOL by using a 1.4-A probe, corrected by removing contributions from IgE inaccessible main chains and side chains forming the ligand binding sites.</p><p class="abstract"><strong>Results</strong>: A set of 129 food allergen sequences were classified into only 20 of 3849 possible Pfam families, with 4 families accounting for more than 65% of food allergens. Structural bioinformatic analysis of conserved exterior main chains and amino acid side chains in cross-reactive homologues of Bet v 1 and nonspecific lipid transfer proteins showed higher levels of similarity than shown by simple sequence comparisons. Thus, 75% of the Mal d 1 surface is likely to bind anti-Bet v 1 antibodies, compared with a sequence identity of approximately 56%.</p><p class="abstract"><strong>Conclusion</strong>: Most plant food allergens belong to only 4 structural families, indicating that conserved structures and biological activities may play a role in determining or promoting allergenic properties. Structural bioinformatic analysis shows that conservation of 3-dimensional structure should be included in any assessment of potential IgE cross-reactivity in, for example, novel proteins.</p>
<p class="publication"><span class="title">The Journal of allergy and clinical immunology</span>&nbsp;<span class="year">2005</span>;<span class="volume">115</span>;<span class="number">1</span>;<span class="pagerange">163-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15637564" rel="external">15637564</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jaci.2004.10.026" rel="external">10.1016/j.jaci.2004.10.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Does structural and chemical divergence play a role in precluding undesirable protein interactions?</h4>
<p class="authors"><span class="author">Jim&eacute;nez JL</span></p>
<p class="affiliation">Biomolecular Modelling Laboratory, Cancer Research UK London Research Institute, London, United Kingdom. <a href="mailto:%6A%6C%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#108;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">To understand the evolutionary forces establishing, maintaining, breaking, or precluding protein-protein interactions, a comprehensive data set of protein complexes has been analyzed to examine the overlap between protein interfaces and the most conserved or divergent protein surface areas. The most divergent areas tend to be found predominantly away from protein interfaces, although when found at interfaces, they are associated with specific lack of cross-reactivity between close homologues, like in antibody-antigen complexes. Moreover, the amino acid composition of highly variable regions is significantly different from any other protein surfaces. The variable regions present higher structural plasticity as a result of insertions and deletions, and favor charged over hydrophobic residues, a known strategy to minimize aggregation. This suggests that (1) a rapid rate of mutations at these regions might be continuously altering their properties, making difficult the coadaptation, in shape and chemical complementarity, to potential interacting partners; and (2) the existence of some form of selective pressure for variable areas away from interfaces to accumulate charged residues, perhaps as an evolutionary mechanism to increase solubility and minimize undesirable interactions within the crowded cellular environment. Finally, these results are placed into the context of the aberrant oligomerization of sickle-cell anemia hemoglobin and prion proteins.</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2005</span>;<span class="volume">59</span>;<span class="number">4</span>;<span class="pagerange">757-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15822102" rel="external">15822102</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fprot.20448" rel="external">10.1002/prot.20448</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antibodies to the iron uptake ABC transporter lipoproteins PiaA and PiuA promote opsonophagocytosis of Streptococcus pneumoniae.</h4>
<p class="authors"><span class="author">Jomaa M</span>, <span class="author">Yuste J</span>, <span class="author">Paton JC</span>, <span class="author">Jones C</span>, <span class="author">Dougan G</span> and <span class="author">Brown JS</span></p>
<p class="affiliation">Centre for Biological Sciences, Imperial College London, UK.</p><p class="abstract">PiaA and PiuA are the lipoprotein components of the Pia and Piu Streptococcus pneumoniae iron uptake ABC transporters and are required for full virulence in mouse models of infection. Active or passive vaccination with recombinant PiuA and PiaA protects mice against invasive S. pneumoniae disease. In this study we have analyzed the antibody responses and mechanism of protection induced by PiuA and PiaA in more detail. For both proteins, two booster vaccinations induced stronger antibody responses in mice than a single or no booster vaccinations, and 5 mug of protein induced similar levels of antibody responses as 20 mug. Immunoglobulin G (IgG) subclass-specific enzyme-linked immunosorbent assays demonstrated that the antibody response to PiuA and PiaA was predominantly IgG1, with induction of only low levels of IgG2a. Anti-PiaA and anti-PiuA polyclonal rabbit antibodies bound to the surface of live S. pneumoniae when assessed by flow cytometry but did not inhibit growth of S. pneumoniae in cation-depleted medium or bacterial susceptibility to the iron-dependent antibiotic streptonigrin. However, anti-PiaA and anti-PiuA did increase complement-independent and -dependent opsonophagocytosis of different serotypes of S. pneumoniae by the human neutrophil cell line HL60. Hence, vaccination with PiaA and PiuA protects against S. pneumoniae infection by inducing antibodies that promote bacterial opsonophagocytosis rather than inhibiting iron transport.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2005</span>;<span class="volume">73</span>;<span class="number">10</span>;<span class="pagerange">6852-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16177364" rel="external">16177364</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1230898" rel="external">1230898</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.73.10.6852-6859.2005" rel="external">10.1128/IAI.73.10.6852-6859.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Array-CGH analysis of microsatellite-stable, near-diploid bowel cancers and comparison with other types of colorectal carcinoma.</h4>
<p class="authors"><span class="author">Jones AM</span>, <span class="author">Douglas EJ</span>, <span class="author">Halford SE</span>, <span class="author">Fiegler H</span>, <span class="author">Gorman PA</span>, <span class="author">Roylance RR</span>, <span class="author">Carter NP</span> and <span class="author">Tomlinson IP</span></p>
<p class="affiliation">Molecular and Population Genetics Laboratory, Cancer Research UK, 44 Lincoln&#39;s Inn Fields, London WC2A 3PX, UK.</p><p class="abstract">Microsatellite-stable, near-diploid (MSI-CIN-) colorectal carcinomas have been reported, but it is not clear as to whether these tumours form a discrete group or represent one end of the distribution of MSI-CIN+ cancers. In order to address this question, we screened 23 MSI-CIN- colorectal cancers for gains and losses using array-based comparative genomic hybridization (aCGH) based on large-insert clones at about 1 Mb density. We compared our findings with those from a small set of MSI+CIN+ cancers, and with our reported data from MSI-CIN+ and MSI+CIN- cancers. We found no evidence of any form of genomic instability in MSI-CIN- cancers. At the level of the chromosome arm, the MSI-CIN- cancers had significantly fewer gains and losses than MSI-CIN+ tumours, but more than the MSI+CIN- and MSI+CIN+ lesions. The chromosomal-scale changes found in MSI-CIN- cancers generally involved the same sites as those in MSI-CIN+ tumours, and in both cancer groups, the best predictor of a specific change was the total number of such changes in that tumour. A few chromosomal-scale changes did, however, differ between the MSI-CIN- and MSI-CIN+ pathways. MSI-CIN- cancers showed: low frequencies of gain of 9p and 19p; infrequent loss of 5q and a high frequency of 20p gain. Overall, our data suggested that the MSI-CIN- group is heterogeneous, one type of MSI-CIN- cancer having few (&lt; or =6) chromosomal-scale changes and the other with more (&gt; or =10) changes resembling MSI-CIN+ cancers. At the level of individual clones, frequent and/or discrete gains or losses were generally located within regions of chromosomal-scale changes in both MSI-CIN- and MSI-CIN+ cancers, and fewer losses and gains were present in MSI-CIN- than MSI-CIN+ tumours. No changes by clone, which were specific to the MSI-CIN- cancers, were found. In addition to indicating differences among the cancer groups, our results also detected over 50 sites (amplifications, potential homozygous deletion and gains or losses which extended over only a few megabases) which might harbour uncharacterized oncogenes or tumour suppressor loci. In conclusion, our data support the suggestion that some MSI-CIN- carcinomas form a qualitatively different group from the other cancer types, and also suggest that the MSI-CIN- group is itself heterogeneous.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2005</span>;<span class="volume">24</span>;<span class="number">1</span>;<span class="pagerange">118-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15531920" rel="external">15531920</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1208194" rel="external">10.1038/sj.onc.1208194</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nodulation signaling in legumes requires NSP2, a member of the GRAS family of transcriptional regulators.</h4>
<p class="authors"><span class="author">Kal&oacute; P</span>, <span class="author">Gleason C</span>, <span class="author">Edwards A</span>, <span class="author">Marsh J</span>, <span class="author">Mitra RM</span>, <span class="author">Hirsch S</span>, <span class="author">Jakab J</span>, <span class="author">Sims S</span>, <span class="author">Long SR</span>, <span class="author">Rogers J</span>, <span class="author">Kiss GB</span>, <span class="author">Downie JA</span> and <span class="author">Oldroyd GE</span></p>
<p class="affiliation">Departments of Disease and Stress Biology and Molecular Microbiology, John Innes Centre, Norwich NR4 7UH, UK.</p><p class="abstract">Rhizobial bacteria enter a symbiotic interaction with legumes, activating diverse responses in roots through the lipochito oligosaccharide signaling molecule Nod factor. Here, we show that NSP2 from Medicago truncatula encodes a GRAS protein essential for Nod-factor signaling. NSP2 functions downstream of Nod-factor-induced calcium spiking and a calcium/calmodulin-dependent protein kinase. We show that NSP2-GFP expressed from a constitutive promoter is localized to the endoplasmic reticulum/nuclear envelope and relocalizes to the nucleus after Nod-factor elicitation. This work provides evidence that a GRAS protein transduces calcium signals in plants and provides a possible regulator of Nod-factor-inducible gene expression.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">308</span>;<span class="number">5729</span>;<span class="pagerange">1786-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15961668">15961668</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1110951">10.1126/science.1110951</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of Campylobacter jejuni capsular loci reveals multiple mechanisms for the generation of structural diversity and the ability to form complex heptoses.</h4>
<p class="authors"><span class="author">Karlyshev AV</span>, <span class="author">Champion OL</span>, <span class="author">Churcher C</span>, <span class="author">Brisson JR</span>, <span class="author">Jarrell HC</span>, <span class="author">Gilbert M</span>, <span class="author">Brochu D</span>, <span class="author">St Michael F</span>, <span class="author">Li J</span>, <span class="author">Wakarchuk WW</span>, <span class="author">Goodhead I</span>, <span class="author">Sanders M</span>, <span class="author">Stevens K</span>, <span class="author">White B</span>, <span class="author">Parkhill J</span>, <span class="author">Wren BW</span> and <span class="author">Szymanski CM</span></p>
<p class="affiliation">Department of Infectious and Tropical Diseases, London School of Hygiene and Tropical Medicine, London WCIE 7HT, UK.</p><p class="abstract">We recently demonstrated that Campylobacter jejuni produces a capsular polysaccharide (CPS) that is the major antigenic component of the classical Penner serotyping system distinguishing Campylobacter into &gt;60 groups. Although the wide variety of C. jejuni serotypes are suggestive of structural differences in CPS, the genetic mechanisms of such differences are unknown. In this study we sequenced biosynthetic cps regions, ranging in size from 15 to 34 kb, from selected C. jejuni strains of HS:1, HS:19, HS:23, HS:36, HS:23/36 and HS:41 serotypes. Comparison of the determined cps sequences of the HS:1, HS:19 and HS:41 strains with the sequenced strain, NCTC11168 (HS:2), provides evidence for multiple mechanisms of structural variation including exchange of capsular genes and entire clusters by horizontal transfer, gene duplication, deletion, fusion and contingency gene variation. In contrast, the HS:23, HS:36 and HS:23/36 cps sequences were highly conserved. We report the first detailed structural analysis of 81-176 (HS:23/36) and G1 (HS:1) and refine the previous structural interpretations of the HS:19, HS:23, HS:36 and HS:41 serostrains. For the first time, we demonstrate the commonality and function of a second heptose biosynthetic pathway for Campylobacter CPS independent of the pathway for lipooligosaccharide (LOS) biosynthesis and identify a novel heptosyltransferase utilized by this alternate pathway. Furthermore, we show the retention of two functional heptose isomerases in Campylobacter and the sharing of a phosphatase for both LOS and CPS heptose biosynthesis.</p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">55</span>;<span class="number">1</span>;<span class="pagerange">90-103</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15612919" rel="external">15612919</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2004.04374.x" rel="external">10.1111/j.1365-2958.2004.04374.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comparison of tagging methods and their tagging space.</h4>
<p class="authors"><span class="author">Ke X</span>, <span class="author">Miretti MM</span>, <span class="author">Broxholme J</span>, <span class="author">Hunt S</span>, <span class="author">Beck S</span>, <span class="author">Bentley DR</span>, <span class="author">Deloukas P</span> and <span class="author">Cardon LR</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, UK. <a href="mailto:%78%69%61%79%69%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#120;&#105;&#97;&#121;&#105;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Single-nucleotide polymorphism (SNP) tagging is widely used as a way of saving genotyping costs in association studies. A number of different tagging methods have been developed to reduce the number of markers to be genotyped while maintaining power for detecting effects on non-assayed SNPs. How the different methods perform in different settings, the degree to which they overlap and share common tags and how they differ are important questions. We investigated these questions by comparing three widely used tagging methods/algorithms--one haplotype r2-based method, one pair-wise r2-based method and one method which was based on haplotype diversity but focused on major haplotypes. Tagging efficiency was defined as the number of genotyped markers divided by the number of tagging SNPs. Tagging effectiveness was defined as the proportion of un-genotyped or &#39;hidden&#39; SNPs being detected (having a pair-wise or haplotype r2 with a set of tagging SNPs over a threshold, e.g. haplotype r2&gt; or =0.80). The ENCODE regions genotyped on the HapMap CEPH individuals were examined in this study. Tagging effectiveness was generally poor for rare SNPs than for common SNPs, for all three tagging methods. Inclusion of rare SNPs into initial HapMap scheme could enhance the performance of tags on rare hidden SNPs at the expense of increased genotyping cost. At a moderate tagging efficiency, more than 90% of hidden SNPs detected by tagging SNPs selected by one method were also detected by tagging SNPs selected by another method, and this figure could be increased to 100% if tagging efficiency was allowed to drop. These results indicate that the tagging space is highly concordant between different tagging methods, despite the fact that they often involve different sets of tagging SNPs.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY-15652</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14</span>;<span class="number">18</span>;<span class="pagerange">2757-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16103130" rel="external">16103130</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi309" rel="external">10.1093/hmg/ddi309</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene arrays at Pneumocystis carinii telomeres.</h4>
<p class="authors"><span class="author">Keely SP</span>, <span class="author">Renauld H</span>, <span class="author">Wakefield AE</span>, <span class="author">Cushion MT</span>, <span class="author">Smulian AG</span>, <span class="author">Fosker N</span>, <span class="author">Fraser A</span>, <span class="author">Harris D</span>, <span class="author">Murphy L</span>, <span class="author">Price C</span>, <span class="author">Quail MA</span>, <span class="author">Seeger K</span>, <span class="author">Sharp S</span>, <span class="author">Tindal CJ</span>, <span class="author">Warren T</span>, <span class="author">Zuiderwijk E</span>, <span class="author">Barrell BG</span>, <span class="author">Stringer JR</span> and <span class="author">Hall N</span></p>
<p class="affiliation">Department of Molecular Genetics, Biochemistry and Microbiology, University of Cincinnati, Cincinnati, Ohio 45267, USA.</p><p class="abstract">In the fungus Pneumocystis carinii, at least three gene families (PRT1, MSR, and MSG) have the potential to generate high-frequency antigenic variation, which is likely to be a strategy by which this parasitic fungus is able to prolong its survival in the rat lung. Members of these gene families are clustered at chromosome termini, a location that fosters recombination, which has been implicated in selective expression of MSG genes. To gain insight into the architecture, evolution, and regulation of these gene clusters, six telomeric segments of the genome were sequenced. Each of the segments began with one or more unique genes, after which were members of different gene families, arranged in a head-to-tail array. The three-gene repeat PRT1-MSR-MSG was common, suggesting that duplications of these repeats have contributed to expansion of all three families. However, members of a gene family in an array were no more similar to one another than to members in other arrays, indicating rapid divergence after duplication. The intergenic spacers were more conserved than the genes and contained sequence motifs also present in subtelomeres, which in other species have been implicated in gene expression and recombination. Long mononucleotide tracts were present in some MSR genes. These unstable sequences can be expected to suffer frequent frameshift mutations, providing P. carinii with another mechanism to generate antigen variation.</p>
<p class="grants">Funded by: FIC NIH HHS: <span class="grant">TW01200-02</span>; NIAID NIH HHS: <span class="grant">R01AI36701</span>, <span class="grant">R01AI44651</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">170</span>;<span class="number">4</span>;<span class="pagerange">1589-600</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15965256" rel="external">15965256</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1449779" rel="external">1449779</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.105.040733" rel="external">10.1534/genetics.105.040733</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sox2 is required for sensory organ development in the mammalian inner ear.</h4>
<p class="authors"><span class="author">Kiernan AE</span>, <span class="author">Pelling AL</span>, <span class="author">Leung KK</span>, <span class="author">Tang AS</span>, <span class="author">Bell DM</span>, <span class="author">Tease C</span>, <span class="author">Lovell-Badge R</span>, <span class="author">Steel KP</span> and <span class="author">Cheah KS</span></p>
<p class="affiliation">MRC Institute of Hearing Research, University of Nottingham, Nottingham NG7 2RD, UK.</p><p class="abstract">Sensory hair cells and their associated non-sensory supporting cells in the inner ear are fundamental for hearing and balance. They arise from a common progenitor, but little is known about the molecular events specifying this cell lineage. We recently identified two allelic mouse mutants, light coat and circling (Lcc) and yellow submarine (Ysb), that show hearing and balance impairment. Lcc/Lcc mice are completely deaf, whereas Ysb/Ysb mice are severely hearing impaired. We report here that inner ears of Lcc/Lcc mice fail to establish a prosensory domain and neither hair cells nor supporting cells differentiate, resulting in a severe inner ear malformation, whereas the sensory epithelium of Ysb/Ysb mice shows abnormal development with disorganized and fewer hair cells. These phenotypes are due to the absence (in Lcc mutants) or reduced expression (in Ysb mutants) of the transcription factor SOX2, specifically within the developing inner ear. SOX2 continues to be expressed in the inner ears of mice lacking Math1 (also known as Atoh1 and HATH1), a gene essential for hair cell differentiation, whereas Math1 expression is absent in Lcc mutants, suggesting that Sox2 acts upstream of Math1.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U117562207</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">434</span>;<span class="number">7036</span>;<span class="pagerange">1031-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15846349">15846349</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03487">10.1038/nature03487</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide screening of genomic alterations and their clinicopathologic implications in non-small cell lung cancers.</h4>
<p class="authors"><span class="author">Kim TM</span>, <span class="author">Yim SH</span>, <span class="author">Lee JS</span>, <span class="author">Kwon MS</span>, <span class="author">Ryu JW</span>, <span class="author">Kang HM</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span> and <span class="author">Chung YJ</span></p>
<p class="affiliation">Department of Microbiology, College of Medicine, Catholic University of Korea, Socho-gu, Seoul, Korea.</p><p class="abstract"><strong>Purpose</strong>: Although many genomic alterations have been observed in lung cancer, their clinicopathologic significance has not been thoroughly investigated. This study screened the genomic aberrations across the whole genome of non-small cell lung cancer cells with high-resolution and investigated their clinicopathologic implications.</p><p class="abstract">One-megabase resolution array comparative genomic hybridization was applied to 29 squamous cell carcinomas and 21 adenocarcinomas of the lung. Tumor and normal tissues were microdissected and the extracted DNA was used directly for hybridization without genomic amplification. The recurrent genomic alterations were analyzed for their association with the clinicopathologic features of lung cancer.</p><p class="abstract"><strong>Results</strong>: Overall, 36 amplicons, 3 homozygous deletions, and 17 minimally altered regions common to many lung cancers were identified. Among them, genomic changes on 13q21, 1p32, Xq, and Yp were found to be significantly associated with clinical features such as age, stage, and disease recurrence. Kaplan-Meier survival analysis revealed that genomic changes on 10p, 16q, 9p, 13q, 6p21, and 19q13 were associated with poor survival. Multivariate analysis showed that alterations on 6p21, 7p, 9q, and 9p remained as independent predictors of poor outcome. In addition, significant correlations were observed for three pairs of minimally altered regions (19q13 and 6p21, 19p13 and 19q13, and 8p12 and 8q11), which indicated their possible collaborative roles.</p><p class="abstract"><strong>Conclusions</strong>: These results show that our approach is robust for high-resolution mapping of genomic alterations. The novel genomic alterations identified in this study, along with their clinicopathologic implications, would be useful to elucidate the molecular mechanisms of lung cancer and to identify reliable biomarkers for clinical application.</p>
<p class="publication"><span class="title">Clinical cancer research : an official journal of the American Association for Cancer Research</span>&nbsp;<span class="year">2005</span>;<span class="volume">11</span>;<span class="number">23</span>;<span class="pagerange">8235-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16322280" rel="external">16322280</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1078-0432.CCR-05-1157" rel="external">10.1158/1078-0432.CCR-05-1157</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intercalated cell H+/OH- transporter expression is reduced in Slc26a4 null mice.</h4>
<p class="authors"><span class="author">Kim YH</span>, <span class="author">Verlander JW</span>, <span class="author">Matthews SW</span>, <span class="author">Kurtz I</span>, <span class="author">Shin W</span>, <span class="author">Weiner ID</span>, <span class="author">Everett LA</span>, <span class="author">Green ED</span>, <span class="author">Nielsen S</span> and <span class="author">Wall SM</span></p>
<p class="affiliation">Department of Medicine, Emory University, Atlanta, Georgia, USA.</p><p class="abstract">Slc26a4 (Pds) encodes pendrin, a Cl(-)/HCO(3)(-) exchanger expressed in the apical region of type B and non-A, non-B cells, which mediates secretion of OH(-) equivalents. Thus genetic disruption of Slc26a4 leads to systemic alkalosis in some treatment models. However, humans and mice with genetic disruption of Slc26a4 have normal acid-base balance under basal conditions. Thus we asked: 1) Is net acid excretion altered in Slc26a4 (-/-) mice under basal conditions? 2) In the absence of pendrin-mediated OH(-) secretion, are increases in intracellular and systemic pH minimized through changes in intercalated cell subtype abundance or intercalated cell H(+)/OH(-) transporter expression? To answer these questions, net acid excretion and H(+)/OH(-) transporter expression were examined in Slc26a4 (-/-) and Slc26a4 (+/+) mice using balance studies, immunolocalization, and immunoblotting. Excretion of ammonium, titratable acid, and citrate were the same in Slc26a4 null and wild-type mice. However, urinary pH and Pco(2) were much lower in Slc26a4 null relative to wild-type mice due to reduced urinary buffering of secreted H(+) by HCO(3)(-). Abundance of non-A, but not type A intercalated cells, was reduced within the cortical collecting ducts of Slc26a4 null mice. Moreover, kidneys from Slc26a4 null mice had reduced H(+)-ATPase, NBC3 and RhBG total protein expression, particularly within type B and non-A, non-B intercalated cells, although RhCG protein expression was unchanged. Reduced intercalated cell H(+)/OH(-) transporter expression is observed in Slc26a4 null mice, which likely attenuates the rise in intracellular and systemic pH expected with genetic disruption of Slc26a4.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK-52935</span></p>
<p class="publication"><span class="title">American journal of physiology. Renal physiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">289</span>;<span class="number">6</span>;<span class="pagerange">F1262-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16144965" rel="external">16144965</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fajprenal.00206.2005" rel="external">10.1152/ajprenal.00206.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Myosin VI is required for normal retinal function.</h4>
<p class="authors"><span class="author">Kitamoto J</span>, <span class="author">Libby RT</span>, <span class="author">Gibbs D</span>, <span class="author">Steel KP</span> and <span class="author">Williams DS</span></p>
<p class="affiliation">Department of Pharmacology, UCSD School of Medicine, La Jolla, CA 92093-0912, USA.</p><p class="abstract">Different unconventional myosins have been shown to play important roles in sensory function, including vision. We investigated the role of myosin VI by examining the retinas of mice carrying a null mutation in the myosin VI gene. Myosin VI was found to be present in the photoreceptor and RPE cells of normal retinas. In the absence of myosin VI, the amplitudes of the a- and b-waves of the electroretinogram were reduced, although there was not photoreceptor cell loss and retinal anatomy appeared normal. Our results indicate that myosin VI is required in photoreceptor cells for normal retinal electrophysiology.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY12598</span></p>
<p class="publication"><span class="title">Experimental eye research</span>&nbsp;<span class="year">2005</span>;<span class="volume">81</span>;<span class="number">1</span>;<span class="pagerange">116-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15978262">15978262</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.exer.2005.02.014">10.1016/j.exer.2005.02.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Isolation, X location and activity of the marsupial homologue of SLC16A2, an XIST-flanking gene in eutherian mammals.</h4>
<p class="authors"><span class="author">Koina E</span>, <span class="author">Wakefield MJ</span>, <span class="author">Walcher C</span>, <span class="author">Disteche CM</span>, <span class="author">Whitehead S</span>, <span class="author">Ross M</span> and <span class="author">Marshall Graves JA</span></p>
<p class="affiliation">ARC Centre for Kangaroo Genomics, Research School of Biological Sciences, The Australian National University, Canberra, ACT 0200, Australia. <a href="mailto:%65%64%64%61.%6B%6F%69%6E%61%40%61%6E%75.%65%64%75.%61%75">&#101;&#100;&#100;&#97;&#46;&#107;&#111;&#105;&#110;&#97;&#64;&#97;&#110;&#117;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">X chromosome inactivation (XCI) achieves dosage compensation between males and females for most X-linked genes in eutherian mammals. It is a whole-chromosome effect under the control of the XIST locus, although some genes escape inactivation. Marsupial XCI differs from the eutherian process, implying fundamental changes in the XCI mechanism during the evolution of the two lineages. There is no direct evidence for the existence of a marsupial XIST homologue. XCI has been studied for only a handful of genes in any marsupial, and none in the model kangaroo Macropus eugenii (the tammar wallaby). We have therefore studied the sequence, location and activity of a gene SLC16A2 (solute carrier, family 16, class A, member 2) that flanks XIST on the human and mouse X chromosomes. A BAC clone containing the marsupial SLC16A2 was mapped to the end of the long arm of the tammar X chromosome and used in RNA FISH experiments to determine whether one or both loci are transcribed in female cells. In male and female cells, only a single signal was found, indicating that the marsupial SLC16A2 gene is silenced on the inactivated X.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM 61948</span>, <span class="grant">R01 GM061948-01</span></p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">7</span>;<span class="pagerange">687-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16235118" rel="external">16235118</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2819140" rel="external">2819140</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-005-1006-5" rel="external">10.1007/s10577-005-1006-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FACT--a framework for the functional interpretation of high-throughput experiments.</h4>
<p class="authors"><span class="author">Kokocinski F</span>, <span class="author">Delhomme N</span>, <span class="author">Wrobel G</span>, <span class="author">Hummerich L</span>, <span class="author">Toedt G</span> and <span class="author">Lichter P</span></p>
<p class="affiliation">Molecular Genetics, Deutsches Krebsforschungszentrum, 69115 Heidelberg, Germany. <a href="mailto:%46.%4B%6F%6B%6F%63%69%6E%73%6B%69%40%66%61%63%74%77%65%62.%64%65">&#70;&#46;&#75;&#111;&#107;&#111;&#99;&#105;&#110;&#115;&#107;&#105;&#64;&#102;&#97;&#99;&#116;&#119;&#101;&#98;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: Interpreting the results of high-throughput experiments, such as those obtained from DNA-microarrays, is an often time-consuming task due to the high number of data-points that need to be analyzed in parallel. It is usually a matter of extensive testing and unknown beforehand, which of the possible approaches for the functional analysis will be the most informative.</p><p class="abstract"><strong>Results</strong>: To address this problem, we have developed the Flexible Annotation and Correlation Tool (FACT). FACT allows for detection of important patterns in large data sets by simplifying the integration of heterogeneous data sources and the subsequent application of different algorithms for statistical evaluation or visualization of the annotated data. The system is constantly extended to include additional annotation data and comparison methods.</p><p class="abstract"><strong>Conclusion</strong>: FACT serves as a highly flexible framework for the explorative analysis of large genomic and proteomic result sets. The program can be used online; open source code and supplementary information are available at http://www.factweb.de.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="pagerange">161</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15985174" rel="external">15985174</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1189078" rel="external">1189078</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-6-161" rel="external">10.1186/1471-2105-6-161</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y-chromosomal STR haplotypes and their applications to forensic and population studies in east Asia.</h4>
<p class="authors"><span class="author">Kwak KD</span>, <span class="author">Jin HJ</span>, <span class="author">Shin DJ</span>, <span class="author">Kim JM</span>, <span class="author">Roewer L</span>, <span class="author">Krawczak M</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Kim W</span></p>
<p class="affiliation">Department of Biological Sciences, Dankook University, Cheonan, 330-714, South Korea.</p><p class="abstract">We have analyzed 11 Y-STR loci (DYS19, the two DYS385 loci, DYS388, DYS389I/II, DYS390, DYS391, DYS392, DYS393, DXYS156Y) in 700 males from ten ethnic groups in east Asia in order to evaluate their usefulness for forensic and population genetic studies. A total of 644 different haplotypes were identified, among which 603 (86.14%) were individual-specific. The haplotype diversity averaged over all populations was 0.9997; using only the nine Y-STRs comprising the &quot;minimal haplotype&quot; (excluding DYS388 and DXYS156Y) it was 0.9996, a value similar to that found in 1924 samples from other Asian populations (0.9996; Lessig et al. Legal Medicine 5(2003) 160-163), and slightly higher than in European populations (0.9976; n=11,610; Roewer et al. For Sci International (2001) 118:103-111). All of the individual east Asian populations examined here had high haplotype diversity (&gt; or =0.997), except for the Mongolians (0.992) and Manchurians (0.960). The most frequent haplotype identified by the nine markers was present at only 1% (7/700). Population comparisons based on Phi(ST) or rho genetic distance measures revealed clustering according to the traditional northeast-southeast distinction, but with exceptions. For example, the Yunnan population from southern China lay among the northern populations, possibly reflecting recent migration, while the Korean population, traditionally considered northern, lay at the boundary between northern and southern populations. An admixture estimate suggested 55(51-59)% northern, 45(41-49)% southern contribution to the Koreans, illustrating the complexity of the genetic history of this region.</p>
<p class="publication"><span class="title">International journal of legal medicine</span>&nbsp;<span class="year">2005</span>;<span class="volume">119</span>;<span class="number">4</span>;<span class="pagerange">195-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15856270" rel="external">15856270</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00414-004-0518-4" rel="external">10.1007/s00414-004-0518-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of ovarian cancer cell lines using array-based comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Lambros MB</span>, <span class="author">Fiegler H</span>, <span class="author">Jones A</span>, <span class="author">Gorman P</span>, <span class="author">Roylance RR</span>, <span class="author">Carter NP</span> and <span class="author">Tomlinson IP</span></p>
<p class="affiliation">Molecular and Population Genetics Laboratory, Cancer Research UK, London, UK.</p><p class="abstract">In this study, 23 ovarian cancer cell lines were screened using array-comparative genomic hybridization (aCGH) based on large-insert clones at about 1 Mb density from throughout the genome. The most frequent recurrent changes at the level of the chromosome arm were loss of chromosome 4 or 4q, loss of 18q and gain of 20 or 20q; other recurrent changes included losses of 6q, 8p, 9p, 11p, 15q, 16q, 17p, and 22q, and gain of 7q. Losses of 4q and 18q occurred together more often than expected. Evidence was found for two types of ovarian cancer, one typically near-triploid and characterized by a generally higher frequency of chromosomal changes (especially losses of 4p, 4q, 13q, 15q, 16p, 16q, 18p and 18q), and the other typically near-diploid/tetraploid and with fewer changes overall, but with relatively high frequencies of 9p loss, 9q gain, and 20p gain. Multiple novel changes (amplifications, homozygous deletions, discrete regions of gain or loss, small overlapping regions of change and frequently changed clones) were also detected, each of which might indicate the locations of oncogenes or tumour suppressor loci. For example, at least two regions of amplification on chromosome 11q13, one including cyclin D1 and the other the candidate oncogene PAK1, were found. Amplification on 11q22 near the progesterone receptor gene and a cluster of matrix metalloproteinase loci was also detected. Other potential oncogenes, which mapped to regions found by this study, included cyclin E and PIK3C2G. Candidate tumour suppressor genes in regions of loss included CDKN2C, SMAD4-interacting protein and RASSF2.</p>
<p class="publication"><span class="title">The Journal of pathology</span>&nbsp;<span class="year">2005</span>;<span class="volume">205</span>;<span class="number">1</span>;<span class="pagerange">29-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15586366" rel="external">15586366</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpath.1681" rel="external">10.1002/path.1681</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complete genome sequence of Francisella tularensis, the causative agent of tularemia.</h4>
<p class="authors"><span class="author">Larsson P</span>, <span class="author">Oyston PC</span>, <span class="author">Chain P</span>, <span class="author">Chu MC</span>, <span class="author">Duffield M</span>, <span class="author">Fuxelius HH</span>, <span class="author">Garcia E</span>, <span class="author">H&auml;lltorp G</span>, <span class="author">Johansson D</span>, <span class="author">Isherwood KE</span>, <span class="author">Karp PD</span>, <span class="author">Larsson E</span>, <span class="author">Liu Y</span>, <span class="author">Michell S</span>, <span class="author">Prior J</span>, <span class="author">Prior R</span>, <span class="author">Malfatti S</span>, <span class="author">Sj&ouml;stedt A</span>, <span class="author">Svensson K</span>, <span class="author">Thompson N</span>, <span class="author">Vergez L</span>, <span class="author">Wagg JK</span>, <span class="author">Wren BW</span>, <span class="author">Lindler LE</span>, <span class="author">Andersson SG</span>, <span class="author">Forsman M</span> and <span class="author">Titball RW</span></p>
<p class="affiliation">Swedish Defence Research Agency, SE-901 82 Ume&aring;, Sweden.</p><p class="abstract">Francisella tularensis is one of the most infectious human pathogens known. In the past, both the former Soviet Union and the US had programs to develop weapons containing the bacterium. We report the complete genome sequence of a highly virulent isolate of F. tularensis (1,892,819 bp). The sequence uncovers previously uncharacterized genes encoding type IV pili, a surface polysaccharide and iron-acquisition systems. Several virulence-associated genes were located in a putative pathogenicity island, which was duplicated in the genome. More than 10% of the putative coding sequences contained insertion-deletion or substitution mutations and seemed to be deteriorating. The genome is rich in IS elements, including IS630 Tc-1 mariner family transposons, which are not expected in a prokaryote. We used a computational method for predicting metabolic pathways and found an unexpectedly high proportion of disrupted pathways, explaining the fastidious nutritional requirements of the bacterium. The loss of biosynthetic pathways indicates that F. tularensis is an obligate host-dependent bacterium in its natural life cycle. Our results have implications for our understanding of how highly virulent human pathogens evolve and will expedite strategies to combat them.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">37</span>;<span class="number">2</span>;<span class="pagerange">153-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15640799">15640799</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1499">10.1038/ng1499</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetically indistinguishable SNPs and their influence on inferring the location of disease-associated variants.</h4>
<p class="authors"><span class="author">Lawrence R</span>, <span class="author">Evans DM</span>, <span class="author">Morris AP</span>, <span class="author">Ke X</span>, <span class="author">Hunt S</span>, <span class="author">Paolucci M</span>, <span class="author">Ragoussis J</span>, <span class="author">Deloukas P</span>, <span class="author">Bentley D</span> and <span class="author">Cardon LR</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">As part of a recent high-density linkage disequilibrium (LD) study of chromosome 20, we obtained genotypes for approximately 30,000 SNPs at a density of 1 SNP/2 kb on four different population samples (47 CEPH founders; 91 UK unrelateds [unrelated white individuals of western European ancestry]; 97 African Americans; 42 East Asians). We observed that approximately 50% of SNPs had at least one genetically indistinguishable partner; i.e., for every individual considered, their genotype at the first locus was identical to their genotype at the second locus, or in LD terms, the SNPs were in &quot;perfect&quot; LD (r2 = 1.0). These &quot;genetically indistinguishable SNPs&quot; (giSNPs) formed into clusters of varying size. The larger the cluster, the greater the tendency to be located within genes and to overlap with giSNP clusters in other population samples. As might be expected for this map density, many giSNPs were located close to one another, thus reflecting local regions of undetected recombination or haplotype blocks. However, approximately 1/3 of giSNP clusters had intermingled, non-indistinguishable SNPs with incomplete LD (D&#39; and r2 &lt;1), sometimes spanning hundreds of kilobases, comprising up to 70 indistinguishable markers and overlapping multiple haplotype blocks. These long-range, nonconsecutive giSNPs have implications for disease gene localization by allelic association as evidence for association at one locus will be indistinguishable from that at another locus, even though both loci may be situated far apart. We describe the distribution of giSNPs on this map of chromosome 20 and illustrate the potential impact they can have on association mapping.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">11</span>;<span class="pagerange">1503-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16251460" rel="external">16251460</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1310638" rel="external">1310638</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.4217605" rel="external">10.1101/gr.4217605</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CIC, a gene involved in cerebellar development and ErbB signaling, is significantly expressed in medulloblastomas.</h4>
<p class="authors"><span class="author">Lee CJ</span>, <span class="author">Chan WI</span> and <span class="author">Scotting PJ</span></p>
<p class="affiliation">EMBL-EBI, Wellcome Trust Genome Campus, Hinxton, UK.</p><p class="abstract">In children, the majority of brain tumors arise in the cerebellum. Medulloblastomas, the most common of these, are believed to originate from the granule cell lineage. We have recently identified a mammalian gene, capicua (Cic), the ortholog of a Drosophila gene implicated in c-erbB (Egfr) signaling, which is predominantly expressed during mouse granule cell development. Its expression in medulloblastoma is therefore of particular interest. In the present study the expression of human CIC in medulloblastoma was analyzed. In silico SAGE analysis demonstrated that medulloblastomas exhibited the highest level of CIC expression and expression was most common in tumors of the CNS in general. RT-PCR and in situ hybridization verified the expression of CIC in tumor cells, although the level of expression varied between different medulloblastoma subtypes. The expression of CIC did not correlate with other markers, such as neurofilament, GFAP and Mib-1. In postnatally developing cerebellum, in silico analysis and in situ hybridization both indicated a strong correlation between Cic expression and the maturation profile of cerebellar granule cell precursors. Expression of CIC is therefore a feature shared between immature granule cells and the tumors derived from them. Cic has been implicated as a mediator of ErbB signaling and this pathway has been associated with a poor prognosis for medulloblastomas. Therefore, further analysis of the role of Cic is likely to provide valuable insight into the biology of these tumors. Additionally, study of genes such as CIC should provide objective criteria by which, in combination with other markers and clinical data, to categorize these tumors into subgroups that might allow better allocation into specific treatment regimes.</p>
<p class="publication"><span class="title">Journal of neuro-oncology</span>&nbsp;<span class="year">2005</span>;<span class="volume">73</span>;<span class="number">2</span>;<span class="pagerange">101-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15981098" rel="external">15981098</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs11060-004-4598-2" rel="external">10.1007/s11060-004-4598-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Impairment of the TFIIH-associated CDK-activating kinase selectively affects cell cycle-regulated gene expression in fission yeast.</h4>
<p class="authors"><span class="author">Lee KM</span>, <span class="author">Miklos I</span>, <span class="author">Du H</span>, <span class="author">Watt S</span>, <span class="author">Szilagyi Z</span>, <span class="author">Saiz JE</span>, <span class="author">Madabhushi R</span>, <span class="author">Penkett CJ</span>, <span class="author">Sipiczki M</span>, <span class="author">B&auml;hler J</span> and <span class="author">Fisher RP</span></p>
<p class="affiliation">Molecular Biology Program, Memorial Sloan-Kettering Cancer Center, New York, NY 10021, USA.</p><p class="abstract">The fission yeast Mcs6-Mcs2-Pmh1 complex, homologous to metazoan Cdk7-cyclin H-Mat1, has dual functions in cell division and transcription: as a partially redundant cyclin-dependent kinase (CDK)-activating kinase (CAK) that phosphorylates the major cell cycle CDK, Cdc2, on Thr-167; and as the RNA polymerase (Pol) II carboxyl-terminal domain (CTD) kinase associated with transcription factor (TF) IIH. We analyzed conditional mutants of mcs6 and pmh1, which activate Cdc2 normally but cannot complete cell division at restrictive temperature and arrest with decreased CTD phosphorylation. Transcriptional profiling by microarray hybridization revealed only modest effects on global gene expression: a one-third reduction in a severe mcs6 mutant after prolonged incubation at 36 degrees C. In contrast, a small subset of transcripts ( approximately 5%) decreased by more than twofold after Mcs6 complex function was compromised. The signature of repressed genes overlapped significantly with those of cell separation mutants sep10 and sep15. Sep10, a component of the Pol II Mediator complex, becomes essential in mcs6 or pmh1 mutant backgrounds. Moreover, transcripts dependent on the forkhead transcription factor Sep1, which are expressed coordinately during mitosis, were repressed in Mcs6 complex mutants, and Mcs6 also interacts genetically with Sep1. Thus, the Mcs6 complex, a direct activator of Cdc2, also influences the cell cycle transcriptional program, possibly through its TFIIH-associated kinase function.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular biology of the cell</span>&nbsp;<span class="year">2005</span>;<span class="volume">16</span>;<span class="number">6</span>;<span class="pagerange">2734-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15829570" rel="external">15829570</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1142420" rel="external">1142420</a>; DOI: <a href="http://dx.doi.org/10.1091%2Fmbc.E04-11-0982" rel="external">10.1091/mbc.E04-11-0982</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systems biology: where it&#39;s at in 2005.</h4>
<p class="authors"><span class="author">Lehner B</span>, <span class="author">Tischler J</span> and <span class="author">Fraser AG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">A report on the joint Keystone Symposia on Systems and Biology and Proteomics and Bioinformatics, Keystone, USA, 8-13 April 2005.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">338</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16086862" rel="external">16086862</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1273629" rel="external">1273629</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2005-6-8-338" rel="external">10.1186/gb-2005-6-8-338</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulation of neural progenitor proliferation and survival by beta1 integrins.</h4>
<p class="authors"><span class="author">Leone DP</span>, <span class="author">Relvas JB</span>, <span class="author">Campos LS</span>, <span class="author">Hemmi S</span>, <span class="author">Brakebusch C</span>, <span class="author">F&auml;ssler R</span>, <span class="author">Ffrench-Constant C</span> and <span class="author">Suter U</span></p>
<p class="affiliation">Institute of Cell Biology, Department of Biology, Swiss Federal Institute of Technology, ETH H&ouml;nggerberg, Z&uuml;rich.</p><p class="abstract">Neural stem cells give rise to undifferentiated nestin-positive progenitors that undergo extensive cell division before differentiating into neuronal and glial cells. The precise control of this process is likely to be, at least in part, controlled by instructive cues originating from the extracellular environment. Some of these cues are interpreted by the integrin family of extracellular matrix receptors. Using neurosphere cell cultures as a model system, we show that beta1-integrin signalling plays a crucial role in the regulation of progenitor cell proliferation, survival and migration. Following conditional genetic ablation of the beta1-integrin allele, and consequent loss of beta1-integrin cell surface protein, mutant nestin-positive progenitor cells proliferate less and die in higher numbers than their wild-type counterparts. Mutant progenitor cell migration on different ECM substrates is also impaired. These effects can be partially compensated by the addition of exogenous growth factors. Thus, beta1-integrin signalling and growth factor signalling tightly interact to control the number and migratory capacity of nestin-positive progenitor cells.</p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2005</span>;<span class="volume">118</span>;<span class="number">Pt 12</span>;<span class="pagerange">2589-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15928047" rel="external">15928047</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fjcs.02396" rel="external">10.1242/jcs.02396</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of DNA markers for a transmissible Pseudomonas aeruginosa cystic fibrosis strain.</h4>
<p class="authors"><span class="author">Lewis DA</span>, <span class="author">Jones A</span>, <span class="author">Parkhill J</span>, <span class="author">Speert DP</span>, <span class="author">Govan JR</span>, <span class="author">Lipuma JJ</span>, <span class="author">Lory S</span>, <span class="author">Webb AK</span> and <span class="author">Mahenthiralingam E</span></p>
<p class="affiliation">Cardiff School of Biosciences, Cardiff University, UK.</p><p class="abstract">A number of transmissible Pseudomonas aeruginosa strains have been identified which potentially constitute an emerging threat to patients with cystic fibrosis (CF). We sought to identify DNA markers that were specific to a transmissible P. aeruginosa CF clone and evaluate these probes on a large collection of genotypically distinct P. aeruginosa strains. Using subtractive DNA hybridization, in combination with analysis using the P. aeruginosa PAO1 genome chip, DNA markers specific for or absent from the Manchester transmissible CF strain (MA) were identified. Five subtractive DNA hybridization markers (MA15, MA18, MA21, MA22, and MA30) were found to be specific to strain MA and were located within a novel 13,318-bp genomic island, designated the MA island. The MA island encoded 18 genes and consisted of two bacteriophage-like regions; one region encoded the MA-specific subtractive hybridization markers, while the other bacteriophage-like region contained a Vibrio cholera-like toxin gene. Probes MA15, MA18, MA21, MA22, and MA30 were all found to be specific to strain MA when a collection of 141 P. aeruginosa strains was examined by hybridization with each DNA marker. In contrast, a previously isolated DNA marker for the Liverpool transmissible CF strain, PS21, was not found to be specific, detecting two additional strain types in the collection screened. Both the Manchester and Liverpool strain types were not encountered in CF populations outside the United Kingdom. The MA genomic island and Vibrio cholera-like toxin gene within it constitute novel genetic factors associated with a transmissible P. aeruginosa strain and their role in pathogenesis remains to be determined.</p>
<p class="publication"><span class="title">American journal of respiratory cell and molecular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">1</span>;<span class="pagerange">56-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15834046" rel="external">15834046</a>; DOI: <a href="http://dx.doi.org/10.1165%2Frcmb.2004-0352OC" rel="external">10.1165/rcmb.2004-0352OC</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence, comparative analysis and haplotype structure of the domestic dog.</h4>
<p class="authors"><span class="author">Lindblad-Toh K</span>, <span class="author">Wade CM</span>, <span class="author">Mikkelsen TS</span>, <span class="author">Karlsson EK</span>, <span class="author">Jaffe DB</span>, <span class="author">Kamal M</span>, <span class="author">Clamp M</span>, <span class="author">Chang JL</span>, <span class="author">Kulbokas EJ</span>, <span class="author">Zody MC</span>, <span class="author">Mauceli E</span>, <span class="author">Xie X</span>, <span class="author">Breen M</span>, <span class="author">Wayne RK</span>, <span class="author">Ostrander EA</span>, <span class="author">Ponting CP</span>, <span class="author">Galibert F</span>, <span class="author">Smith DR</span>, <span class="author">DeJong PJ</span>, <span class="author">Kirkness E</span>, <span class="author">Alvarez P</span>, <span class="author">Biagi T</span>, <span class="author">Brockman W</span>, <span class="author">Butler J</span>, <span class="author">Chin CW</span>, <span class="author">Cook A</span>, <span class="author">Cuff J</span>, <span class="author">Daly MJ</span>, <span class="author">DeCaprio D</span>, <span class="author">Gnerre S</span>, <span class="author">Grabherr M</span>, <span class="author">Kellis M</span>, <span class="author">Kleber M</span>, <span class="author">Bardeleben C</span>, <span class="author">Goodstadt L</span>, <span class="author">Heger A</span>, <span class="author">Hitte C</span>, <span class="author">Kim L</span>, <span class="author">Koepfli KP</span>, <span class="author">Parker HG</span>, <span class="author">Pollinger JP</span>, <span class="author">Searle SM</span>, <span class="author">Sutter NB</span>, <span class="author">Thomas R</span>, <span class="author">Webber C</span>, <span class="author">Baldwin J</span>, <span class="author">Abebe A</span>, <span class="author">Abouelleil A</span>, <span class="author">Aftuck L</span>, <span class="author">Ait-Zahra M</span>, <span class="author">Aldredge T</span>, <span class="author">Allen N</span>, <span class="author">An P</span>, <span class="author">Anderson S</span>, <span class="author">Antoine C</span>, <span class="author">Arachchi H</span>, <span class="author">Aslam A</span>, <span class="author">Ayotte L</span>, <span class="author">Bachantsang P</span>, <span class="author">Barry A</span>, <span class="author">Bayul T</span>, <span class="author">Benamara M</span>, <span class="author">Berlin A</span>, <span class="author">Bessette D</span>, <span class="author">Blitshteyn B</span>, <span class="author">Bloom T</span>, <span class="author">Blye J</span>, <span class="author">Boguslavskiy L</span>, <span class="author">Bonnet C</span>, <span class="author">Boukhgalter B</span>, <span class="author">Brown A</span>, <span class="author">Cahill P</span>, <span class="author">Calixte N</span>, <span class="author">Camarata J</span>, <span class="author">Cheshatsang Y</span>, <span class="author">Chu J</span>, <span class="author">Citroen M</span>, <span class="author">Collymore A</span>, <span class="author">Cooke P</span>, <span class="author">Dawoe T</span>, <span class="author">Daza R</span>, <span class="author">Decktor K</span>, <span class="author">DeGray S</span>, <span class="author">Dhargay N</span>, <span class="author">Dooley K</span>, <span class="author">Dooley K</span>, <span class="author">Dorje P</span>, <span class="author">Dorjee K</span>, <span class="author">Dorris L</span>, <span class="author">Duffey N</span>, <span class="author">Dupes A</span>, <span class="author">Egbiremolen O</span>, <span class="author">Elong R</span>, <span class="author">Falk J</span>, <span class="author">Farina A</span>, <span class="author">Faro S</span>, <span class="author">Ferguson D</span>, <span class="author">Ferreira P</span>, <span class="author">Fisher S</span>, <span class="author">FitzGerald M</span>, <span class="author">Foley K</span>, <span class="author">Foley C</span>, <span class="author">Franke A</span>, <span class="author">Friedrich D</span>, <span class="author">Gage D</span>, <span class="author">Garber M</span>, <span class="author">Gearin G</span>, <span class="author">Giannoukos G</span>, <span class="author">Goode T</span>, <span class="author">Goyette A</span>, <span class="author">Graham J</span>, <span class="author">Grandbois E</span>, <span class="author">Gyaltsen K</span>, <span class="author">Hafez N</span>, <span class="author">Hagopian D</span>, <span class="author">Hagos B</span>, <span class="author">Hall J</span>, <span class="author">Healy C</span>, <span class="author">Hegarty R</span>, <span class="author">Honan T</span>, <span class="author">Horn A</span>, <span class="author">Houde N</span>, <span class="author">Hughes L</span>, <span class="author">Hunnicutt L</span>, <span class="author">Husby M</span>, <span class="author">Jester B</span>, <span class="author">Jones C</span>, <span class="author">Kamat A</span>, <span class="author">Kanga B</span>, <span class="author">Kells C</span>, <span class="author">Khazanovich D</span>, <span class="author">Kieu AC</span>, <span class="author">Kisner P</span>, <span class="author">Kumar M</span>, <span class="author">Lance K</span>, <span class="author">Landers T</span>, <span class="author">Lara M</span>, <span class="author">Lee W</span>, <span class="author">Leger JP</span>, <span class="author">Lennon N</span>, <span class="author">Leuper L</span>, <span class="author">LeVine S</span>, <span class="author">Liu J</span>, <span class="author">Liu X</span>, <span class="author">Lokyitsang Y</span>, <span class="author">Lokyitsang T</span>, <span class="author">Lui A</span>, <span class="author">Macdonald J</span>, <span class="author">Major J</span>, <span class="author">Marabella R</span>, <span class="author">Maru K</span>, <span class="author">Matthews C</span>, <span class="author">McDonough S</span>, <span class="author">Mehta T</span>, <span class="author">Meldrim J</span>, <span class="author">Melnikov A</span>, <span class="author">Meneus L</span>, <span class="author">Mihalev A</span>, <span class="author">Mihova T</span>, <span class="author">Miller K</span>, <span class="author">Mittelman R</span>, <span class="author">Mlenga V</span>, <span class="author">Mulrain L</span>, <span class="author">Munson G</span>, <span class="author">Navidi A</span>, <span class="author">Naylor J</span>, <span class="author">Nguyen T</span>, <span class="author">Nguyen N</span>, <span class="author">Nguyen C</span>, <span class="author">Nguyen T</span>, <span class="author">Nicol R</span>, <span class="author">Norbu N</span>, <span class="author">Norbu C</span>, <span class="author">Novod N</span>, <span class="author">Nyima T</span>, <span class="author">Olandt P</span>, <span class="author">O&#39;Neill B</span>, <span class="author">O&#39;Neill K</span>, <span class="author">Osman S</span>, <span class="author">Oyono L</span>, <span class="author">Patti C</span>, <span class="author">Perrin D</span>, <span class="author">Phunkhang P</span>, <span class="author">Pierre F</span>, <span class="author">Priest M</span>, <span class="author">Rachupka A</span>, <span class="author">Raghuraman S</span>, <span class="author">Rameau R</span>, <span class="author">Ray V</span>, <span class="author">Raymond C</span>, <span class="author">Rege F</span>, <span class="author">Rise C</span>, <span class="author">Rogers J</span>, <span class="author">Rogov P</span>, <span class="author">Sahalie J</span>, <span class="author">Settipalli S</span>, <span class="author">Sharpe T</span>, <span class="author">Shea T</span>, <span class="author">Sheehan M</span>, <span class="author">Sherpa N</span>, <span class="author">Shi J</span>, <span class="author">Shih D</span>, <span class="author">Sloan J</span>, <span class="author">Smith C</span>, <span class="author">Sparrow T</span>, <span class="author">Stalker J</span>, <span class="author">Stange-Thomann N</span>, <span class="author">Stavropoulos S</span>, <span class="author">Stone C</span>, <span class="author">Stone S</span>, <span class="author">Sykes S</span>, <span class="author">Tchuinga P</span>, <span class="author">Tenzing P</span>, <span class="author">Tesfaye S</span>, <span class="author">Thoulutsang D</span>, <span class="author">Thoulutsang Y</span>, <span class="author">Topham K</span>, <span class="author">Topping I</span>, <span class="author">Tsamla T</span>, <span class="author">Vassiliev H</span>, <span class="author">Venkataraman V</span>, <span class="author">Vo A</span>, <span class="author">Wangchuk T</span>, <span class="author">Wangdi T</span>, <span class="author">Weiand M</span>, <span class="author">Wilkinson J</span>, <span class="author">Wilson A</span>, <span class="author">Yadav S</span>, <span class="author">Yang S</span>, <span class="author">Yang X</span>, <span class="author">Young G</span>, <span class="author">Yu Q</span>, <span class="author">Zainoun J</span>, <span class="author">Zembek L</span>, <span class="author">Zimmer A</span> and <span class="author">Lander ES</span></p>
<p class="affiliation">Broad Institute of Harvard and MIT, 320 Charles Street, Cambridge, Massachusetts 02141, USA. <a href="mailto:%6B%65%72%73%6C%69%40%62%72%6F%61%64.%6D%69%74.%65%64%75">&#107;&#101;&#114;&#115;&#108;&#105;&#64;&#98;&#114;&#111;&#97;&#100;&#46;&#109;&#105;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Here we report a high-quality draft genome sequence of the domestic dog (Canis familiaris), together with a dense map of single nucleotide polymorphisms (SNPs) across breeds. The dog is of particular interest because it provides important evolutionary information and because existing breeds show great phenotypic diversity for morphological, physiological and behavioural traits. We use sequence comparison with the primate and rodent lineages to shed light on the structure and evolution of genomes and genes. Notably, the majority of the most highly conserved non-coding sequences in mammalian genomes are clustered near a small subset of genes with important roles in development. Analysis of SNPs reveals long-range haplotypes across the entire dog genome, and defines the nature of genetic diversity within and across breeds. The current SNP map now makes it possible for genome-wide association studies to identify genes responsible for diseases and traits, with important consequences for human and companion animal health.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">438</span>;<span class="number">7069</span>;<span class="pagerange">803-19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16341006">16341006</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature04338">10.1038/nature04338</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shotgun haplotyping: a novel method for surveying allelic sequence variation.</h4>
<p class="authors"><span class="author">Lindsay SJ</span>, <span class="author">Bonfield JK</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Haplotypic sequences contain significantly more information than genotypes of genetic markers and are critical for studying disease association and genome evolution. Current methods for obtaining haplotypic sequences require the physical separation of alleles before sequencing, are time consuming and are not scaleable for large surveys of genetic variation. We have developed a novel method for acquiring haplotypic sequences from long PCR products using simple, high-throughput techniques. This method applies modified shotgun sequencing protocols to sequence both alleles concurrently, with read-pair information allowing the two alleles to be separated during sequence assembly. Although the haplotypic sequences can be assembled manually from the resultant data using pre-existing sequence assembly software, we have devised a novel heuristic algorithm to automate assembly and remove human error. We validated the approach on two long PCR products amplified from the human genome and confirmed the accuracy of our sequences against full-length clones of the same alleles. This method presents a simple high-throughput means to obtain full haplotypic sequences potentially up to 20 kb in length and is suitable for surveying genetic variation even in poorly-characterized genomes as it requires no prior information on sequence variation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">18</span>;<span class="pagerange">e152</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16221968">16221968</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1253838">1253838</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgni152">10.1093/nar/gni152</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the protist parasite Entamoeba histolytica.</h4>
<p class="authors"><span class="author">Loftus B</span>, <span class="author">Anderson I</span>, <span class="author">Davies R</span>, <span class="author">Alsmark UC</span>, <span class="author">Samuelson J</span>, <span class="author">Amedeo P</span>, <span class="author">Roncaglia P</span>, <span class="author">Berriman M</span>, <span class="author">Hirt RP</span>, <span class="author">Mann BJ</span>, <span class="author">Nozaki T</span>, <span class="author">Suh B</span>, <span class="author">Pop M</span>, <span class="author">Duchene M</span>, <span class="author">Ackers J</span>, <span class="author">Tannich E</span>, <span class="author">Leippe M</span>, <span class="author">Hofer M</span>, <span class="author">Bruchhaus I</span>, <span class="author">Willhoeft U</span>, <span class="author">Bhattacharya A</span>, <span class="author">Chillingworth T</span>, <span class="author">Churcher C</span>, <span class="author">Hance Z</span>, <span class="author">Harris B</span>, <span class="author">Harris D</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Ormond D</span>, <span class="author">Squares R</span>, <span class="author">Whitehead S</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Norbertczak H</span>, <span class="author">Price C</span>, <span class="author">Wang Z</span>, <span class="author">Guill&eacute;n N</span>, <span class="author">Gilchrist C</span>, <span class="author">Stroup SE</span>, <span class="author">Bhattacharya S</span>, <span class="author">Lohia A</span>, <span class="author">Foster PG</span>, <span class="author">Sicheritz-Ponten T</span>, <span class="author">Weber C</span>, <span class="author">Singh U</span>, <span class="author">Mukherjee C</span>, <span class="author">El-Sayed NM</span>, <span class="author">Petri WA</span>, <span class="author">Clark CG</span>, <span class="author">Embley TM</span>, <span class="author">Barrell B</span>, <span class="author">Fraser CM</span> and <span class="author">Hall N</span></p>
<p class="affiliation">TIGR, 9712 Medical Center Drive, Rockville, Maryland 20850, USA. <a href="mailto:%62%6A%6C%6F%66%74%75%73%40%74%69%67%72.%6F%72%67">&#98;&#106;&#108;&#111;&#102;&#116;&#117;&#115;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Entamoeba histolytica is an intestinal parasite and the causative agent of amoebiasis, which is a significant source of morbidity and mortality in developing countries. Here we present the genome of E. histolytica, which reveals a variety of metabolic adaptations shared with two other amitochondrial protist pathogens: Giardia lamblia and Trichomonas vaginalis. These adaptations include reduction or elimination of most mitochondrial metabolic pathways and the use of oxidative stress enzymes generally associated with anaerobic prokaryotes. Phylogenomic analysis identifies evidence for lateral gene transfer of bacterial genes into the E. histolytica genome, the effects of which centre on expanding aspects of E. histolytica&#39;s metabolic repertoire. The presence of these genes and the potential for novel metabolic pathways in E. histolytica may allow for the development of new chemotherapeutic agents. The genome encodes a large number of novel receptor kinases and contains expansions of a variety of gene families, including those associated with virulence. Additional genome features include an abundance of tandemly repeated transfer-RNA-containing arrays, which may have a structural function in the genome. Analysis of the genome provides new insights into the workings and genome evolution of a major human pathogen.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">433</span>;<span class="number">7028</span>;<span class="pagerange">865-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15729342">15729342</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03291">10.1038/nature03291</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">VEGA, the genome browser with a difference.</h4>
<p class="authors"><span class="author">Loveland J</span></p>
<p class="affiliation">HAVANA Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%6A%65%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#101;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Vertebrate Genome Annotation (Vega) database is a community resource for browsing manual annotation from a variety of vertebrate genomes of finished sequence (http://vega.sanger.ac.uk). Vega is different from other genome browsers as it has a standardised classification of genes which encompasses pseudogenes and non-coding transcripts. The data is manually curated, which is more accurate at identifying splice variants, pseudogenes poly(A) features, non-coding and complex gene structures and arrangements than current automated methods. The database also contains annotation from regions, not just whole genomes, and displays multiple species annotation (human, mouse, dog and zebrafish) for comparative analysis. Vega encourages community feedback that results in annotation updates and manual annotation of finished vertebrate sequence.</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">189-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15975227">15975227</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptional adaptation of Shigella flexneri during infection of macrophages and epithelial cells: insights into the strategies of a cytosolic bacterial pathogen.</h4>
<p class="authors"><span class="author">Lucchini S</span>, <span class="author">Liu H</span>, <span class="author">Jin Q</span>, <span class="author">Hinton JC</span> and <span class="author">Yu J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Shigella flexneri, the etiologic agent of bacillary dysentery, invades epithelial cells as well as macrophages and dendritic cells and escapes into the cytosol soon after invasion. Dissection of the global gene expression profile of the bacterium in its intracellular niche is essential to fully understand the biology of Shigella infection. We have determined the complete gene expression profiles for S. flexneri infecting human epithelial HeLa cells and human macrophage-like U937 cells. Approximately one quarter of the S. flexneri genes showed significant transcriptional adaptation during infection; 929 and 1,060 genes were up- or down-regulated within HeLa cells and U937 cells, respectively. The key S. flexneri virulence genes, ipa-mxi-spa and icsA, were drastically down-regulated during intracellular growth. This theme seems to be common in bacterial infection, because the Ipa-Mxi-Spa-like type III secretion systems were also down-regulated during mammalian cell infection by Salmonella enterica serovar Typhimurium and Escherichia coli O157. The bacteria experienced restricted levels of iron, magnesium, and phosphate in both host cell types, as shown by up-regulation of the sitABCD system, the mgtA gene, and genes of the phoBR regulon. Interestingly, ydeO and other acid-induced genes were up-regulated only in U937 cells and not in HeLa cells, suggesting that the cytosol of U937 cells is acidic. Comparison with the gene expression of intracellular Salmonella serovar Typhimurium, which resides within the Salmonella-containing vacuole, indicated that S. flexneri is exposed to oxidative stress in U937 cells. This work will facilitate functional studies of hundreds of novel intracellularly regulated genes that may be important for the survival and growth strategies of Shigella in the human host.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2005</span>;<span class="volume">73</span>;<span class="number">1</span>;<span class="pagerange">88-102</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15618144">15618144</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC538992">538992</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.73.1.88-102.2005">10.1128/IAI.73.1.88-102.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PPARGC1A genotype (Gly482Ser) predicts exceptional endurance capacity in European men.</h4>
<p class="authors"><span class="author">Lucia A</span>, <span class="author">G&oacute;mez-Gallego F</span>, <span class="author">Barroso I</span>, <span class="author">Rabad&aacute;n M</span>, <span class="author">Bandr&eacute;s F</span>, <span class="author">San Juan AF</span>, <span class="author">Chicharro JL</span>, <span class="author">Ekelund U</span>, <span class="author">Brage S</span>, <span class="author">Earnest CP</span>, <span class="author">Wareham NJ</span> and <span class="author">Franks PW</span></p>
<p class="affiliation">European University of Madrid, Spain.</p><p class="abstract">Animal and human data indicate a role for the peroxisome proliferator-activated receptor-gamma coactivator 1alpha (PPARGC1A) gene product in the development of maximal oxygen uptake (V(O2 max)), a determinant of endurance capacity, diabetes, and early death. We tested the hypothesis that the frequency of the minor Ser482 allele at the PPARGC1A locus is lower in World-class Spanish male endurance athletes (cases) [n = 104; mean (SD) age: 26.8 (3.8) yr] than in unfit United Kingdom (UK) Caucasian male controls [n = 100; mean (SD) age: 49.3 (8.1) yr]. In cases and controls, the Gly482Ser genotype met Hardy-Weinberg expectations (P &gt; 0.05 in both groups tested separately). Cases had significantly higher V(O2 max) [73.4 (5.7) vs. 29.4 ml x kg(-1) x min(-1) (3.8); P &lt; 0.0001] and were leaner [body mass index: 20.6 (1.5) vs. 27.6 kg/m2 (3.9); P &lt; 0.0001] than controls. In unadjusted chi2 analyses, the frequency of the minor Ser482 allele was significantly lower in cases than in controls (29.1 vs. 40.0%; P = 0.01). To assess the possibility that genetic stratification could confound these observations, we also compared Gly482Ser genotype frequencies in Spanish (n = 164) and UK Caucasian men (n = 381) who were unselected for their level of fitness. In these analyses, Ser482 allele frequencies were very similar (36.9% in Spanish vs. 37.5% in UK Caucasians, P = 0.83), suggesting that confounding by genetic stratification is unlikely to explain the association between Gly482Ser genotype and endurance capacity. In summary, our data indicate a role for the Gly482Ser genotype in determining aerobic fitness. This finding has relevance from the perspective of physical performance, but it may also be informative for the targeted prevention of diseases associated with low fitness such as Type 2 diabetes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of applied physiology (Bethesda, Md. : 1985)</span>&nbsp;<span class="year">2005</span>;<span class="volume">99</span>;<span class="number">1</span>;<span class="pagerange">344-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15705733">15705733</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fjapplphysiol.00037.2005">10.1152/japplphysiol.00037.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genetic map and comparative analysis with the physical map of Trypanosoma brucei.</h4>
<p class="authors"><span class="author">MacLeod A</span>, <span class="author">Tweedie A</span>, <span class="author">McLellan S</span>, <span class="author">Taylor S</span>, <span class="author">Hall N</span>, <span class="author">Berriman M</span>, <span class="author">El-Sayed NM</span>, <span class="author">Hope M</span>, <span class="author">Turner CM</span> and <span class="author">Tait A</span></p>
<p class="affiliation">Wellcome Centre for Molecular Parasitology, Anderson College Complex, University of Glasgow, 56 Dumbarton Road, Glasgow G11 6NU, UK. <a href="mailto:%67%76%77%61%30%38%40%75%64%63%66.%67%6C%61.%61%63.%75%6B">&#103;&#118;&#119;&#97;&#48;&#56;&#64;&#117;&#100;&#99;&#102;&#46;&#103;&#108;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Trypanosoma brucei is the causative agent of African sleeping sickness in humans and contributes to the debilitating disease &#39;Nagana&#39; in cattle. To date we know little about the genes that determine drug resistance, host specificity, pathogenesis and virulence in these parasites. The availability of the complete genome sequence and the ability of the parasite to undergo genetic exchange have allowed genetic investigations into this parasite and here we report the first genetic map of T.brucei for the genome reference stock TREU 927, comprising of 182 markers and 11 major linkage groups, that correspond to the 11 previously identified chromosomes. The genetic map provides 90% probability of a marker being 11 cM from any given locus. Its comparison to the available physical map has revealed the average physical size of a recombination unit to be 15.6 Kb/cM. The genetic map coupled with the genome sequence and the ability to undertake crosses presents a new approach to identifying genes relevant to the disease and its prevention in this important pathogen through forward genetic analysis and positional cloning.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">21</span>;<span class="pagerange">6688-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16314301" rel="external">16314301</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1297707" rel="external">1297707</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki980" rel="external">10.1093/nar/gki980</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global expression changes resulting from loss of telomeric DNA in fission yeast.</h4>
<p class="authors"><span class="author">Mandell JG</span>, <span class="author">B&auml;hler J</span>, <span class="author">Volpe TA</span>, <span class="author">Martienssen RA</span> and <span class="author">Cech TR</span></p>
<p class="affiliation">Department of Chemistry and Biochemistry and Howard Hughes Medical Institute, University of Colorado, Boulder, CO 80309-0215, USA. <a href="mailto:%6A%6D%61%6E%64%65%6C%6C%40%63%6F%6C%6F%72%61%64%6F.%65%64%75">&#106;&#109;&#97;&#110;&#100;&#101;&#108;&#108;&#64;&#99;&#111;&#108;&#111;&#114;&#97;&#100;&#111;&#46;&#101;&#100;&#117;</a> &lt;<a href="mailto:%6A%6D%61%6E%64%65%6C%6C%40%63%6F%6C%6F%72%61%64%6F.%65%64%75&%67%74">&#106;&#109;&#97;&#110;&#100;&#101;&#108;&#108;&#64;&#99;&#111;&#108;&#111;&#114;&#97;&#100;&#111;&#46;&#101;&#100;&#117;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: Schizosaccharomyces pombe cells lacking the catalytic subunit of telomerase (encoded by trt1+) lose telomeric DNA and enter crisis, but rare survivors arise with either circular or linear chromosomes. Survivors with linear chromosomes have normal growth rates and morphology, but those with circular chromosomes have growth defects and are enlarged. We report the global gene-expression response of S. pombe to loss of trt1+.</p><p class="abstract"><strong>Results</strong>: Survivors with linear chromosomes had expression profiles similar to cells with native telomeres, whereas survivors with circular chromosomes showed continued upregulation of core environmental stress response (CESR) genes. In addition, survivors with circular chromosomes had altered expression of 51 genes compared to survivors with linear chromosomes, providing an expression signature. S. pombe progressing through crisis displayed two waves of altered gene expression. One coincided with crisis and consisted of around 110 genes, 44% of which overlapped with the CESR. The second was synchronized with the emergence of survivors and consisted of a single class of open reading frames (ORFs) with homology both to RecQ helicases and to dh repeats at centromeres targeted for heterochromatin formation via an RNA interference (RNAi) mechanism. Accumulation of transcript from the ORF was found not only in trt1- cells, but also in dcr1- and ago1- RNAi mutants, suggesting that RNAi may control its expression.</p><p class="abstract"><strong>Conclusions</strong>: These results demonstrate a correlation between a state of cellular stress, short telomeres and growth defects in cells with circular chromosomes. A putative new RecQ helicase was expressed as survivors emerged and appears to be transcriptionally regulated by RNAi, suggesting that this mechanism operates at telomeres.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; NIGMS NIH HHS: <span class="grant">R01GM067014.</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">R1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15642092" rel="external">15642092</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC549061" rel="external">549061</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2004-6-1-r1" rel="external">10.1186/gb-2004-6-1-r1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression of a RecQ helicase homolog affects progression through crisis in fission yeast lacking telomerase.</h4>
<p class="authors"><span class="author">Mandell JG</span>, <span class="author">Goodrich KJ</span>, <span class="author">B&auml;hler J</span> and <span class="author">Cech TR</span></p>
<p class="affiliation">Department of Chemistry and Biochemistry and Howard Hughes Medical Institute, University of Colorado, Boulder, CO 80309-0215, USA.</p><p class="abstract">RecQ helicases play roles in telomere maintenance in cancerous human cells using the alternative lengthening of telomeres mechanism and in budding yeast lacking telomerase. Fission yeast lacking the catalytic subunit of telomerase (trt1(+)) up-regulate the expression of a previously uncharacterized sub-telomeric open reading frame as survivors emerge from crisis. Here we show that this open reading frame encodes a protein with homology to RecQ helicases such as the human Bloom&#39;s and Werner&#39;s syndrome proteins and that copies of the helicase gene are present on multiple chromosome ends. Characterization of the helicase transcript revealed a 7.6-kilobase RNA that was associated with polyribosomes, suggesting it is translated. A 3.6-kilobase domain of the helicase gene predicted to encode the region with catalytic activity was cloned, and both native and mutant forms of this domain were overexpressed in trt1(-) cells as they progressed through crisis. Overexpression of the native form caused cells to recover from crisis earlier than cells with a vector-only control, whereas overexpression of the mutant form caused delayed recovery from crisis. Taken together, the sequence homology, functional analysis, and site-directed mutagenesis indicate that the protein is likely a second fission yeast RecQ helicase (in addition to Rqh1) that participates in telomere metabolism during crisis. These results strengthen the notion that in multiple organisms RecQ helicases contribute to survival after telomere damage.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; NIGMS NIH HHS: <span class="grant">GM28039</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2005</span>;<span class="volume">280</span>;<span class="number">7</span>;<span class="pagerange">5249-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15591066" rel="external">15591066</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M412756200" rel="external">10.1074/jbc.M412756200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CGHAnalyzer: a stand-alone software package for cancer genome analysis using array-based DNA copy number data.</h4>
<p class="authors"><span class="author">Margolin AA</span>, <span class="author">Greshock J</span>, <span class="author">Naylor TL</span>, <span class="author">Mosse Y</span>, <span class="author">Maris JM</span>, <span class="author">Bignell G</span>, <span class="author">Saeed AI</span>, <span class="author">Quackenbush J</span> and <span class="author">Weber BL</span></p>
<p class="affiliation">Abramson Family Cancer Research Institute, University of Pennsylvania, Philadelphia, PA 19104, USA.</p><p class="abstract">SUMMARY: This synopsis provides an overview of array-based comparative genomic hybridization data display, abstraction and analysis using CGHAnalyzer, a software suite, designed specifically for this purpose. CGHAnalyzer can be used to simultaneously load copy number data from multiple platforms, query and describe large, heterogeneous datasets and export results. Additionally, CGHAnalyzer employs a host of algorithms for microarray analysis that include hierarchical clustering and class differentiation. AVAILABILITY: CGHAnalyzer, the accompanying manual, documentation and sample data are available for download at http://acgh.afcri.upenn.edu. This is a Java-based application built in the framework of the TIGR MeV that can run on Microsoft Windows, Macintosh OSX and a variety of Unix-based platforms. It requires the installation of the free Java Runtime Environment 1.4.1 (or more recent) (http://www.java.sun.com).</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">15</span>;<span class="pagerange">3308-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15905276" rel="external">15905276</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti500" rel="external">10.1093/bioinformatics/bti500</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A large-scale screen in S. pombe identifies seven novel genes required for critical meiotic events.</h4>
<p class="authors"><span class="author">Mart&iacute;n-Castellanos C</span>, <span class="author">Blanco M</span>, <span class="author">Rozal&eacute;n AE</span>, <span class="author">P&eacute;rez-Hidalgo L</span>, <span class="author">Garc&iacute;a AI</span>, <span class="author">Conde F</span>, <span class="author">Mata J</span>, <span class="author">Ellermeier C</span>, <span class="author">Davis L</span>, <span class="author">San-Segundo P</span>, <span class="author">Smith GR</span> and <span class="author">Moreno S</span></p>
<p class="affiliation">Instituto de Biolog&iacute;a Molecular y Celular del C&aacute;ncer, CSIC/Universidad de Salamanca, Campus Miguel de Unamuno, 37007 Salamanca, Spain.</p><p class="abstract">Meiosis is a specialized form of cell division by which sexually reproducing diploid organisms generate haploid gametes. During a long prophase, telomeres cluster into the bouquet configuration to aid chromosome pairing, and DNA replication is followed by high levels of recombination between homologous chromosomes (homologs). This recombination is important for the reductional segregation of homologs at the first meiotic division; without further replication, a second meiotic division yields haploid nuclei. In the fission yeast Schizosaccharomyces pombe, we have deleted 175 meiotically upregulated genes and found seven genes not previously reported to be critical for meiotic events. Three mutants (rec24, rec25, and rec27) had strongly reduced meiosis-specific DNA double-strand breakage and recombination. One mutant (tht2) was deficient in karyogamy, and two (bqt1 and bqt2) were deficient in telomere clustering, explaining their defects in recombination and segregation. The moa1 mutant was delayed in premeiotic S phase progression and nuclear divisions. Further analysis of these mutants will help elucidate the complex machinery governing the special behavior of meiotic chromosomes.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM32194</span>, <span class="grant">R01 GM032194-21</span>, <span class="grant">R01 GM032194-22</span>, <span class="grant">R01 GM032194-23</span></p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">22</span>;<span class="pagerange">2056-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16303567" rel="external">16303567</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2721798" rel="external">2721798</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2005.10.038" rel="external">10.1016/j.cub.2005.10.038</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Post-transcriptional control of gene expression: a genome-wide perspective.</h4>
<p class="authors"><span class="author">Mata J</span>, <span class="author">Marguerat S</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Gene expression is regulated at multiple levels, and cells need to integrate and coordinate different layers of control to implement the information in the genome. Post-transcriptional levels of regulation such as transcript turnover and translational control are an integral part of gene expression and might rival the sophistication and importance of transcriptional control. Microarray-based methods are increasingly used to study not only transcription but also global patterns of transcript decay and translation rates in addition to comprehensively identify targets of RNA-binding proteins. Such large-scale analyses have recently provided supplementary and unique insights into gene expression programs. Integration of several different datasets will ultimately lead to a system-wide understanding of the varied and complex mechanisms for gene expression control.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2005</span>;<span class="volume">30</span>;<span class="number">9</span>;<span class="pagerange">506-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16054366" rel="external">16054366</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tibs.2005.07.005" rel="external">10.1016/j.tibs.2005.07.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and protein expression profiling identifies CDK6 as novel independent prognostic marker in medulloblastoma.</h4>
<p class="authors"><span class="author">Mendrzyk F</span>, <span class="author">Radlwimmer B</span>, <span class="author">Joos S</span>, <span class="author">Kokocinski F</span>, <span class="author">Benner A</span>, <span class="author">Stange DE</span>, <span class="author">Neben K</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span>, <span class="author">Reifenberger G</span>, <span class="author">Korshunov A</span> and <span class="author">Lichter P</span></p>
<p class="affiliation">Division of Molecular Genetics (B060), German Cancer Research Center, Im Neuenheimer Feld 580, 69120 Heidelberg, Germany.</p><p class="abstract"><strong>Purpose</strong>: Medulloblastoma is the most common malignant brain tumor in children. Despite multimodal aggressive treatment, nearly half of the patients die as a result of this tumor. Identification of molecular markers for prognosis and development of novel pathogenesis-based therapies depends crucially on a better understanding of medulloblastoma pathomechanisms.</p><p class="abstract">We performed genome-wide analysis of DNA copy number imbalances in 47 medulloblastomas using comparative genomic hybridization to large insert DNA microarrays (matrix-CGH). The expression of selected candidate genes identified by matrix-CGH was analyzed immunohistochemically on tissue microarrays representing medulloblastomas from 189 clinically well-documented patients. To identify novel prognostic markers, genomic findings and protein expression data were correlated to patient survival.</p><p class="abstract"><strong>Results</strong>: Matrix-CGH analysis revealed frequent DNA copy number alterations of several novel candidate regions. Among these, gains at 17q23.2-qter (P &lt; .01) and losses at 17p13.1 to 17p13.3 (P = .04) were significantly correlated to poor prognosis. Within 17q23.2-qter and 7q21.2, two of the most frequently gained chromosomal regions, confined amplicons were identified that contained the PPM1D and CDK6 genes, respectively. Immunohistochemistry revealed strong expression of PPM1D in 148 (88%) of 168 and CDK6 in 50 (30%) of 169 medulloblastomas. Overexpression of CDK6 correlated significantly with poor prognosis (P &lt; .01) and represented an independent prognostic marker of overall survival on multivariate analysis (P = .02).</p><p class="abstract"><strong>Conclusion</strong>: We identified CDK6 as a novel molecular marker that can be determined by immunohistochemistry on routinely processed tissue specimens and may facilitate the prognostic assessment of medulloblastoma patients. Furthermore, increased protein-levels of PPM1D and CDK6 may link the TP53 and RB1 tumor suppressor pathways to medulloblastoma pathomechanisms.</p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2005</span>;<span class="volume">23</span>;<span class="number">34</span>;<span class="pagerange">8853-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16314645" rel="external">16314645</a>; DOI: <a href="http://dx.doi.org/10.1200%2FJCO.2005.02.8589" rel="external">10.1200/JCO.2005.02.8589</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high-resolution linkage-disequilibrium map of the human major histocompatibility complex and first generation of tag single-nucleotide polymorphisms.</h4>
<p class="authors"><span class="author">Miretti MM</span>, <span class="author">Walsh EC</span>, <span class="author">Ke X</span>, <span class="author">Delgado M</span>, <span class="author">Griffiths M</span>, <span class="author">Hunt S</span>, <span class="author">Morrison J</span>, <span class="author">Whittaker P</span>, <span class="author">Lander ES</span>, <span class="author">Cardon LR</span>, <span class="author">Bentley DR</span>, <span class="author">Rioux JD</span>, <span class="author">Beck S</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">Autoimmune, inflammatory, and infectious diseases present a major burden to human health and are frequently associated with loci in the human major histocompatibility complex (MHC). Here, we report a high-resolution (1.9 kb) linkage-disequilibrium (LD) map of a 4.46-Mb fragment containing the MHC in U.S. pedigrees with northern and western European ancestry collected by the Centre d&#39;Etude du Polymorphisme Humain (CEPH) and the first generation of haplotype tag single-nucleotide polymorphisms (tagSNPs) that provide up to a fivefold increase in genotyping efficiency for all future MHC-linked disease-association studies. The data confirm previously identified recombination hotspots in the class II region and allow the prediction of numerous novel hotspots in the class I and class III regions. The region of longest LD maps outside the classic MHC to the extended class I region spanning the MHC-linked olfactory-receptor gene cluster. The extended haplotype homozygosity analysis for recent positive selection shows that all 14 outlying haplotype variants map to a single extended haplotype, which most commonly bears HLA-DRB1*1501. The SNP data, haplotype blocks, and tagSNPs analysis reported here have been entered into a multidimensional Web-based database (GLOVAR), where they can be accessed and viewed in the context of relevant genome annotation. This LD map allowed us to give coordinates for the extremely variable LD structure underlying the MHC.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK64869</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">76</span>;<span class="number">4</span>;<span class="pagerange">634-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15747258" rel="external">15747258</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1199300" rel="external">1199300</a>; DOI: <a href="http://dx.doi.org/10.1086%2F429393" rel="external">10.1086/429393</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical assessment of methods of protein structure prediction (CASP)--round 6.</h4>
<p class="authors"><span class="author">Moult J</span>, <span class="author">Fidelis K</span>, <span class="author">Rost B</span>, <span class="author">Hubbard T</span> and <span class="author">Tramontano A</span></p>
<p class="affiliation">Center for Advanced Research in Biotechnology, University of Maryland Biotechnology Institute, Rockville, Maryland 20850, USA. <a href="mailto:%6D%6F%75%6C%74%40%75%6D%62%69.%75%6D%64.%65%64%75">&#109;&#111;&#117;&#108;&#116;&#64;&#117;&#109;&#98;&#105;&#46;&#117;&#109;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">This article is an introduction to the special issue of the journal Proteins, dedicated to the sixth CASP experiment to assess the state of the art in protein structure prediction. The article describes the conduct of the experiment and the categories of prediction included, and outlines the evaluation and assessment procedures. A brief summary of progress over the decade of CASP experiments is also provided.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM072354</span>; NLM NIH HHS: <span class="grant">LM07085</span></p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2005</span>;<span class="volume">61 Suppl 7</span>;<span class="pagerange">3-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16187341">16187341</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fprot.20716">10.1002/prot.20716</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A selenocysteine tRNA and SECIS element in Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Mourier T</span>, <span class="author">Pain A</span>, <span class="author">Barrell B</span> and <span class="author">Griffiths-Jones S</span></p><p class="abstract">The molecular machinery for incorporating selenocysteine into proteins is present in both prokaryotes and eukaryotes. Although selenocysteine insertion has been reported in animals, plants, and protozoans, known eukaryotic selenocysteine tRNA sequences and selenocysteine insertion sequences are limited to animals and plants. Here we present clear indications of the presence of selenocysteine-tRNA and a selenocysteine insertion sequence in Plasmodium falciparum. To our knowledge, this is the first report of an identification of protozoan selenocysteine insertion machinery at the sequence level.</p>
<p class="publication"><span class="title">RNA (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">11</span>;<span class="number">2</span>;<span class="pagerange">119-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15659354" rel="external">15659354</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1370700" rel="external">1370700</a>; DOI: <a href="http://dx.doi.org/10.1261%2Frna.7185605" rel="external">10.1261/rna.7185605</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">InterPro, progress and status in 2005.</h4>
<p class="authors"><span class="author">Mulder NJ</span>, <span class="author">Apweiler R</span>, <span class="author">Attwood TK</span>, <span class="author">Bairoch A</span>, <span class="author">Bateman A</span>, <span class="author">Binns D</span>, <span class="author">Bradley P</span>, <span class="author">Bork P</span>, <span class="author">Bucher P</span>, <span class="author">Cerutti L</span>, <span class="author">Copley R</span>, <span class="author">Courcelle E</span>, <span class="author">Das U</span>, <span class="author">Durbin R</span>, <span class="author">Fleischmann W</span>, <span class="author">Gough J</span>, <span class="author">Haft D</span>, <span class="author">Harte N</span>, <span class="author">Hulo N</span>, <span class="author">Kahn D</span>, <span class="author">Kanapin A</span>, <span class="author">Krestyaninova M</span>, <span class="author">Lonsdale D</span>, <span class="author">Lopez R</span>, <span class="author">Letunic I</span>, <span class="author">Madera M</span>, <span class="author">Maslen J</span>, <span class="author">McDowall J</span>, <span class="author">Mitchell A</span>, <span class="author">Nikolskaya AN</span>, <span class="author">Orchard S</span>, <span class="author">Pagni M</span>, <span class="author">Ponting CP</span>, <span class="author">Quevillon E</span>, <span class="author">Selengut J</span>, <span class="author">Sigrist CJ</span>, <span class="author">Silventoinen V</span>, <span class="author">Studholme DJ</span>, <span class="author">Vaughan R</span> and <span class="author">Wu CH</span></p>
<p class="affiliation">EMBL Outstation-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%75%6C%64%65%72%40%65%62%69.%61%63.%75%6B">&#109;&#117;&#108;&#100;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">InterPro, an integrated documentation resource of protein families, domains and functional sites, was created to integrate the major protein signature databases. Currently, it includes PROSITE, Pfam, PRINTS, ProDom, SMART, TIGRFAMs, PIRSF and SUPERFAMILY. Signatures are manually integrated into InterPro entries that are curated to provide biological and functional information. Annotation is provided in an abstract, Gene Ontology mapping and links to specialized databases. New features of InterPro include extended protein match views, taxonomic range information and protein 3D structure data. One of the new match views is the InterPro Domain Architecture view, which shows the domain composition of protein matches. Two new entry types were introduced to better describe InterPro entries: these are active site and binding site. PIRSF and the structure-based SUPERFAMILY are the latest member databases to join InterPro, and CATH and PANTHER are soon to be integrated. InterPro release 8.0 contains 11 007 entries, representing 2573 domains, 8166 families, 201 repeats, 26 active sites, 21 binding sites and 20 post-translational modification sites. InterPro covers over 78% of all proteins in the Swiss-Prot and TrEMBL components of UniProt. The database is available for text- and sequence-based searches via a webserver (http://www.ebi.ac.uk/interpro), and for download by anonymous FTP (ftp://ftp.ebi.ac.uk/pub/databases/interpro).</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">Database issue</span>;<span class="pagerange">D201-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15608177" rel="external">15608177</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540060" rel="external">540060</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki106" rel="external">10.1093/nar/gki106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Citrobacter rodentium of mice and man.</h4>
<p class="authors"><span class="author">Mundy R</span>, <span class="author">MacDonald TT</span>, <span class="author">Dougan G</span>, <span class="author">Frankel G</span> and <span class="author">Wiles S</span></p>
<p class="affiliation">Centre for Molecular Microbiology and Infection, Division of Cell and Molecular Biology, Imperial College London SW7 2AZ, UK.</p><p class="abstract">The major classes of enteric bacteria harbour a conserved core genomic structure, common to both commensal and pathogenic strains, that is most likely optimized to a life style involving colonization of the host intestine and transmission via the environment. In pathogenic bacteria this core genome framework is decorated with novel genetic islands that are often associated with adaptive phenotypes such as virulence. This classical genome organization is well illustrated by a group of extracellular enteric pathogens, which includes enteropathogenic Escherichia coli (EPEC), enterohaemorrhagic E. coli (EHEC) and Citrobacter rodentium, all of which use attaching and effacing (A/E) lesion formation as a major mechanism of tissue targeting and infection. Both EHEC and EPEC are poorly pathogenic in mice but infect humans and domestic animals. In contrast, C. rodentium is a natural mouse pathogen that is related to E. coli, hence providing an excellent in vivo model for A/E lesion forming pathogens. C. rodentium also provides a model of infections that are mainly restricted to the lumen of the intestine. The mechanism&#39;s by which the immune system deals with such infections has become a topic of great interest in recent years. Here we review the literature of C. rodentium from its emergence in the mid-1960s to the most contemporary reports of colonization, pathogenesis, transmission and immunity.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cellular microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">1697-706</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16309456" rel="external">16309456</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1462-5822.2005.00625.x" rel="external">10.1111/j.1462-5822.2005.00625.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From genome to epigenome.</h4>
<p class="authors"><span class="author">Murrell A</span>, <span class="author">Rakyan VK</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%61%6D%6D%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#109;&#109;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The success of the human genome sequencing project has created wide-spread interest in exploring the human epigenome in order to elucidate how the genome executes the information it holds. Although all (nucleated) human cells effectively contain the same genome, they contain very different epigenomes depending upon cell type, developmental stage, sex, age and various other parameters. This complexity makes it intrinsically difficult to precisely define &#39;an&#39; epigenome, let alone &#39;the&#39; epigenome. What is clear, however, is that in order to unravel any epigenome, existing and novel high-throughput approaches on the DNA, RNA and protein levels need to be harnessed and integrated. Here, we review the current thinking and progress on how to get from the genome to the epigenome(s) and discuss some potential applications.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14 Spec No 1</span>;<span class="pagerange">R3-R10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15809270" rel="external">15809270</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi110" rel="external">10.1093/hmg/ddi110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of noncoding and silent coding sites in the Plasmodium falciparum and Plasmodium reichenowi genomes.</h4>
<p class="authors"><span class="author">Neafsey DE</span>, <span class="author">Hartl DL</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, MA, USA. <a href="mailto:%6E%65%61%66%73%65%79%40%62%72%6F%61%64.%6D%69%74.%65%64%75">&#110;&#101;&#97;&#102;&#115;&#101;&#121;&#64;&#98;&#114;&#111;&#97;&#100;&#46;&#109;&#105;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">We compared levels of sequence divergence between fourfold synonymous coding sites and noncoding sites from the intergenic and intronic regions of the Plasmodium falciparum and Plasmodium reichenowi genomes. We observed significant differences in the level of divergence between these classes of silent sites. Fourfold synonymous coding sites exhibited the highest level of sequence divergence, followed by introns, and then intergenic sequences. This pattern of relative divergence rates has been observed in primate genomes but was unexpected in Plasmodium due to a paucity of variation at silent sites in P. falciparum and the corollary hypothesis that silent sites in this genome may be subject to atypical selective constraints. Exclusion of hypermutable CpG dinucleotides reduces the divergence level of synonymous coding sites to that of intergenic sites but does not diminish the significantly higher divergence level of introns relative to intergenic sites. A greater than expected incidence of CpG dinucleotides in intergenic regions less than 500 bp from genes may indicate selective maintenance of regulatory motifs containing CpGs. Divergence rates of different classes of silent sites in these Plasmodium genomes are determined by a combination of mutational and selective pressures.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM61351</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2005</span>;<span class="volume">22</span>;<span class="number">7</span>;<span class="pagerange">1621-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15858207" rel="external">15858207</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsi154" rel="external">10.1093/molbev/msi154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic sequence of the pathogenic and allergenic filamentous fungus Aspergillus fumigatus.</h4>
<p class="authors"><span class="author">Nierman WC</span>, <span class="author">Pain A</span>, <span class="author">Anderson MJ</span>, <span class="author">Wortman JR</span>, <span class="author">Kim HS</span>, <span class="author">Arroyo J</span>, <span class="author">Berriman M</span>, <span class="author">Abe K</span>, <span class="author">Archer DB</span>, <span class="author">Bermejo C</span>, <span class="author">Bennett J</span>, <span class="author">Bowyer P</span>, <span class="author">Chen D</span>, <span class="author">Collins M</span>, <span class="author">Coulsen R</span>, <span class="author">Davies R</span>, <span class="author">Dyer PS</span>, <span class="author">Farman M</span>, <span class="author">Fedorova N</span>, <span class="author">Fedorova N</span>, <span class="author">Feldblyum TV</span>, <span class="author">Fischer R</span>, <span class="author">Fosker N</span>, <span class="author">Fraser A</span>, <span class="author">Garc&iacute;a JL</span>, <span class="author">Garc&iacute;a MJ</span>, <span class="author">Goble A</span>, <span class="author">Goldman GH</span>, <span class="author">Gomi K</span>, <span class="author">Griffith-Jones S</span>, <span class="author">Gwilliam R</span>, <span class="author">Haas B</span>, <span class="author">Haas H</span>, <span class="author">Harris D</span>, <span class="author">Horiuchi H</span>, <span class="author">Huang J</span>, <span class="author">Humphray S</span>, <span class="author">Jim&eacute;nez J</span>, <span class="author">Keller N</span>, <span class="author">Khouri H</span>, <span class="author">Kitamoto K</span>, <span class="author">Kobayashi T</span>, <span class="author">Konzack S</span>, <span class="author">Kulkarni R</span>, <span class="author">Kumagai T</span>, <span class="author">Lafon A</span>, <span class="author">Lafton A</span>, <span class="author">Latg&eacute; JP</span>, <span class="author">Li W</span>, <span class="author">Lord A</span>, <span class="author">Lu C</span>, <span class="author">Majoros WH</span>, <span class="author">May GS</span>, <span class="author">Miller BL</span>, <span class="author">Mohamoud Y</span>, <span class="author">Molina M</span>, <span class="author">Monod M</span>, <span class="author">Mouyna I</span>, <span class="author">Mulligan S</span>, <span class="author">Murphy L</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Paulsen I</span>, <span class="author">Pe&ntilde;alva MA</span>, <span class="author">Pertea M</span>, <span class="author">Price C</span>, <span class="author">Pritchard BL</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rawlins N</span>, <span class="author">Rajandream MA</span>, <span class="author">Reichard U</span>, <span class="author">Renauld H</span>, <span class="author">Robson GD</span>, <span class="author">Rodriguez de C&oacute;rdoba S</span>, <span class="author">Rodr&iacute;guez-Pe&ntilde;a JM</span>, <span class="author">Ronning CM</span>, <span class="author">Rutter S</span>, <span class="author">Salzberg SL</span>, <span class="author">Sanchez M</span>, <span class="author">S&aacute;nchez-Ferrero JC</span>, <span class="author">Saunders D</span>, <span class="author">Seeger K</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Takeuchi M</span>, <span class="author">Tekaia F</span>, <span class="author">Turner G</span>, <span class="author">Vazquez de Aldana CR</span>, <span class="author">Weidman J</span>, <span class="author">White O</span>, <span class="author">Woodward J</span>, <span class="author">Yu JH</span>, <span class="author">Fraser C</span>, <span class="author">Galagan JE</span>, <span class="author">Asai K</span>, <span class="author">Machida M</span>, <span class="author">Hall N</span>, <span class="author">Barrell B</span> and <span class="author">Denning DW</span></p>
<p class="affiliation">The Institute for Genomic Research, Rockville, Maryland 20850, USA. <a href="mailto:%77%6E%69%65%72%6D%61%6E%40%74%69%67%72.%6F%72%67">&#119;&#110;&#105;&#101;&#114;&#109;&#97;&#110;&#64;&#116;&#105;&#103;&#114;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Aspergillus fumigatus is exceptional among microorganisms in being both a primary and opportunistic pathogen as well as a major allergen. Its conidia production is prolific, and so human respiratory tract exposure is almost constant. A. fumigatus is isolated from human habitats and vegetable compost heaps. In immunocompromised individuals, the incidence of invasive infection can be as high as 50% and the mortality rate is often about 50% (ref. 2). The interaction of A. fumigatus and other airborne fungi with the immune system is increasingly linked to severe asthma and sinusitis. Although the burden of invasive disease caused by A. fumigatus is substantial, the basic biology of the organism is mostly obscure. Here we show the complete 29.4-megabase genome sequence of the clinical isolate Af293, which consists of eight chromosomes containing 9,926 predicted genes. Microarray analysis revealed temperature-dependent expression of distinct sets of genes, as well as 700 A. fumigatus genes not present or significantly diverged in the closely related sexual species Neosartorya fischeri, many of which may have roles in the pathogenicity phenotype. The Af293 genome sequence provides an unparalleled resource for the future understanding of this remarkable fungus.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">CFB17726</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">438</span>;<span class="number">7071</span>;<span class="pagerange">1151-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16372009">16372009</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature04332">10.1038/nature04332</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic factors in type 2 diabetes: the end of the beginning?</h4>
<p class="authors"><span class="author">O&#39;Rahilly S</span>, <span class="author">Barroso I</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">University of Cambridge, Department of Clinical Biochemistry, Addenbrooke&#39;s Hospital, Cambridge CB2 2QQ, UK. <a href="mailto:%73%6F%31%30%34%40%6D%65%64%73%63%68%6C.%63%61%6D.%61%63.%75%6B">&#115;&#111;&#49;&#48;&#52;&#64;&#109;&#101;&#100;&#115;&#99;&#104;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The intensive search for genetic variants that predispose to type 2 diabetes was launched with optimism, but progress has been slower than was hoped. Even so, major advances have been made in the understanding of monogenic forms of the disease which together represent a substantial health burden, and a few common gene variants that influence susceptibility have now been unequivocally identified. Armed with a better understanding of the tools needed to detect such genes, it seems inevitable that the rate of progress will increase and the relevance of genetic information to the diagnosis, treatment, and prevention of diabetes will become increasingly tangible.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">307</span>;<span class="number">5708</span>;<span class="pagerange">370-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15662000">15662000</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1104346">10.1126/science.1104346</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Salmonella paratyphi A rates, Asia.</h4>
<p class="authors"><span class="author">Ochiai RL</span>, <span class="author">Wang X</span>, <span class="author">von Seidlein L</span>, <span class="author">Yang J</span>, <span class="author">Bhutta ZA</span>, <span class="author">Bhattacharya SK</span>, <span class="author">Agtini M</span>, <span class="author">Deen JL</span>, <span class="author">Wain J</span>, <span class="author">Kim DR</span>, <span class="author">Ali M</span>, <span class="author">Acosta CJ</span>, <span class="author">Jodar L</span> and <span class="author">Clemens JD</span></p>
<p class="affiliation">International Vaccine Institute, Kwanak PO Box 14, Seoul, South Korea 151-600. <a href="mailto:%72%6C%6F%63%68%69%61%69%40%69%76%69.%69%6E%74">&#114;&#108;&#111;&#99;&#104;&#105;&#97;&#105;&#64;&#105;&#118;&#105;&#46;&#105;&#110;&#116;</a></p><p class="abstract">Little is known about the causes of enteric fever in Asia. Most cases are believed to be caused by Salmonella enterica serovar Typhi and the remainder by S. Paratyphi A. We compared their incidences by using standardized methods from population-based studies in China, Indonesia, India, and Pakistan.</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2005</span>;<span class="volume">11</span>;<span class="number">11</span>;<span class="pagerange">1764-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16318734" rel="external">16318734</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3367370" rel="external">3367370</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fungi behaving badly.</h4>
<p class="authors"><span class="author">Pain A</span></p>
<p class="affiliation">Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">11</span>;<span class="pagerange">832-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16273665">16273665</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1270">10.1038/nrmicro1270</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative apicomplexan genomics.</h4>
<p class="authors"><span class="author">Pain A</span>, <span class="author">Crossman L</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">6</span>;<span class="pagerange">454-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15934186">15934186</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1174">10.1038/nrmicro1174</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome of the host-cell transforming parasite Theileria annulata compared with T. parva.</h4>
<p class="authors"><span class="author">Pain A</span>, <span class="author">Renauld H</span>, <span class="author">Berriman M</span>, <span class="author">Murphy L</span>, <span class="author">Yeats CA</span>, <span class="author">Weir W</span>, <span class="author">Kerhornou A</span>, <span class="author">Aslett M</span>, <span class="author">Bishop R</span>, <span class="author">Bouchier C</span>, <span class="author">Cochet M</span>, <span class="author">Coulson RM</span>, <span class="author">Cronin A</span>, <span class="author">de Villiers EP</span>, <span class="author">Fraser A</span>, <span class="author">Fosker N</span>, <span class="author">Gardner M</span>, <span class="author">Goble A</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Harris DE</span>, <span class="author">Katzer F</span>, <span class="author">Larke N</span>, <span class="author">Lord A</span>, <span class="author">Maser P</span>, <span class="author">McKellar S</span>, <span class="author">Mooney P</span>, <span class="author">Morton F</span>, <span class="author">Nene V</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Price C</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rawlings ND</span>, <span class="author">Rutter S</span>, <span class="author">Saunders D</span>, <span class="author">Seeger K</span>, <span class="author">Shah T</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Tivey A</span>, <span class="author">Walker AR</span>, <span class="author">Woodward J</span>, <span class="author">Dobbelaere DA</span>, <span class="author">Langsley G</span>, <span class="author">Rajandream MA</span>, <span class="author">McKeever D</span>, <span class="author">Shiels B</span>, <span class="author">Tait A</span>, <span class="author">Barrell B</span> and <span class="author">Hall N</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%70%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#112;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Theileria annulata and T. parva are closely related protozoan parasites that cause lymphoproliferative diseases of cattle. We sequenced the genome of T. annulata and compared it with that of T. parva to understand the mechanisms underlying transformation and tropism. Despite high conservation of gene sequences and synteny, the analysis reveals unequally expanded gene families and species-specific genes. We also identify divergent families of putative secreted polypeptides that may reduce immune recognition, candidate regulators of host-cell transformation, and a Theileria-specific protein domain [frequently associated in Theileria (FAINT)] present in a large number of secreted proteins.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5731</span>;<span class="pagerange">131-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15994557">15994557</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1110418">10.1126/science.1110418</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HbO-Arab mutation originated in the Pomak population of Greek Thrace.</h4>
<p class="authors"><span class="author">Papadopoulos V</span>, <span class="author">Dermitzakis E</span>, <span class="author">Konstantinidou D</span>, <span class="author">Petridis D</span>, <span class="author">Xanthopoulidis G</span> and <span class="author">Loukopoulos D</span></p><p class="abstract">HbO-Arab emerged about 2,000 years on a rare haplotype, characteristic of the Greek Pomaks. Its frequency increased as a consequence of high genetic drift within this population, and it was dispersed throughout the Mediterranean basin and Middle East with minor variations of its haplotypic pattern.</p>
<p class="publication"><span class="title">Haematologica</span>&nbsp;<span class="year">2005</span>;<span class="volume">90</span>;<span class="number">2</span>;<span class="pagerange">255-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15710581" rel="external">15710581</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The nuclear rim protein Amo1 is required for proper microtubule cytoskeleton organisation in fission yeast.</h4>
<p class="authors"><span class="author">Pardo M</span> and <span class="author">Nurse P</span></p>
<p class="affiliation">Cell Cycle Laboratory, Cancer Research UK, 44 Lincoln&#39;s Inn Fields, London, WC2A 3PX, UK. <a href="mailto:%6D%70%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#112;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Microtubules have a central role in cell division and cell polarity in eukaryotic cells. The fission yeast is a useful organism for studying microtubule regulation owing to the highly organised nature of its microtubular arrays. To better understand microtubule dynamics and organisation we carried out a screen that identified over 30 genes whose overexpression resulted in microtubule cytoskeleton abnormalities. Here we describe a novel nucleoporin-like protein, Amo1, identified in this screen. Amo1 localises to the nuclear rim in a punctate pattern that does not overlap with nuclear pore complex components. Amo1Delta cells are bent, and they have fewer microtubule bundles that curl around the cell ends. The microtubules in amo1Delta cells have longer dwelling times at the cell tips, and grow in an uncoordinated fashion. Lack of Amo1 also causes a polarity defect. Amo1 is not required for the microtubule loading of several factors affecting microtubule dynamics, and does not seem to be required for nuclear pore function.</p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2005</span>;<span class="volume">118</span>;<span class="number">Pt 8</span>;<span class="pagerange">1705-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15797925">15797925</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fjcs.02305">10.1242/jcs.02305</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">JAM-A expression during embryonic development.</h4>
<p class="authors"><span class="author">Parris JJ</span>, <span class="author">Cooke VG</span>, <span class="author">Skarnes WC</span>, <span class="author">Duncan MK</span> and <span class="author">Naik UP</span></p>
<p class="affiliation">Department of Biological Sciences, University of Delaware, Newark, Delaware 19716, USA.</p><p class="abstract">Cell adhesion molecules of the immunoglobulin superfamily play an important role in embryonic development. We have shown recently that JAM-A, a member of this family expressed at endothelial and epithelial tight junctions, is involved in platelet activation, leukocyte transmigration, and angiogenesis. Here, we determine the expression pattern of the JAM-A gene during embryogenesis using transgenic mice expressing lacZ under the control of the endogenous JAM-A promoter. Histochemical staining for beta-galactosidase in heterozygous mouse embryos was first seen in the inner cell mass and trophectoderm of the blastocyst. By 8.5 days post coitum (dpc), JAM-A gene activity was detected in the endoderm and part of the surface ectoderm. At 9.5 dpc, JAM-A expression began to localize to certain organ systems, most notably the developing inner ear and early vasculature. Localization of JAM-A to embryonic vasculature was confirmed by double-staining with antibodies against JAM-A and platelet endothelial cell adhesion molecule-1, a known endothelial cell marker. As organogenesis progressed, high levels of JAM-A expression continued in the epithelial component of the inner ear as well as the epithelium of the developing skin, olfactory system, lungs, and kidneys. In addition, JAM-A gene activity was found in the developing liver, choroid plexuses, and gut tubes. Immunofluorescent staining with a JAM-A antibody was performed to confirm that expression of the JAM-A-beta-galactosidase fusion protein accurately represented endogenous JAM-A protein. Thus, JAM-A is prominently expressed in embryonic vasculature and the epithelial components of several organ systems and may have an important role in their development.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P20 RR16472</span>; NEI NIH HHS: <span class="grant">EY012221</span>, <span class="grant">EY015279</span>; NHLBI NIH HHS: <span class="grant">HL63960</span></p>
<p class="publication"><span class="title">Developmental dynamics : an official publication of the American Association of Anatomists</span>&nbsp;<span class="year">2005</span>;<span class="volume">233</span>;<span class="number">4</span>;<span class="pagerange">1517-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15977176" rel="external">15977176</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvdy.20481" rel="external">10.1002/dvdy.20481</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hush puppy: a new mouse mutant with pinna, ossicle, and inner ear defects.</h4>
<p class="authors"><span class="author">Pau H</span>, <span class="author">Fuchs H</span>, <span class="author">de Angelis MH</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">MRC Institute of Hearing Research, University Park, Nottingham, UK.</p><p class="abstract">Deafness can be associated with abnormalities of the pinna, ossicles, and cochlea. The authors studied a newly generated mouse mutant with pinna defects and asked whether these defects are associated with peripheral auditory or facial skeletal abnormalities, or both. Furthermore, the authors investigated where the mutation responsible for these defects was located in the mouse genome.</p><p class="abstract"><strong>Methods</strong>: The hearing of hush puppy mutants was assessed by Preyer reflex and electrophysiological measurement. The morphological features of their middle and inner ears were investigated by microdissection, paint-filling of the labyrinth, and scanning electron microscopy. Skeletal staining of skulls was performed to assess the craniofacial dimensions. Genome scanning was performed using microsatellite markers to localize the mutation to a chromosomal region.</p><p class="abstract"><strong>Results</strong>: Some hush puppy mutants showed early onset of hearing impairment. They had small, bat-like pinnae and normal malleus but abnormal incus and stapes. Some mutants had asymmetrical defects and showed reduced penetrance of the ear abnormalities. Paint-filling of newborns&#39; inner ears revealed no morphological abnormality, although half of the mice studied were expected to carry the mutation. Reduced numbers of outer hair cells were demonstrated in mutants&#39; cochlea on scanning electron microscopy. Skeletal staining showed that the mutants have significantly shorter snouts and mandibles. Genome scan revealed that the mutation lies on chromosome 8 between markers D8Mit58 and D8Mit289.</p><p class="abstract"><strong>Conclusion</strong>: The study results indicate developmental problems of the first and second branchial arches and otocyst as a result of a single gene mutation. Similar defects are found in humans, and hush puppy provides a mouse model for investigation of such defects.</p>
<p class="publication"><span class="title">The Laryngoscope</span>&nbsp;<span class="year">2005</span>;<span class="volume">115</span>;<span class="number">1</span>;<span class="pagerange">116-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15630379" rel="external">15630379</a>; DOI: <a href="http://dx.doi.org/10.1097%2F01.mlg.0000150693.31130.a0" rel="external">10.1097/01.mlg.0000150693.31130.a0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evidence for a dispersed Hox gene cluster in the platyhelminth parasite Schistosoma mansoni.</h4>
<p class="authors"><span class="author">Pierce RJ</span>, <span class="author">Wu W</span>, <span class="author">Hirai H</span>, <span class="author">Ivens A</span>, <span class="author">Murphy LD</span>, <span class="author">No&euml;l C</span>, <span class="author">Johnston DA</span>, <span class="author">Artiguenave F</span>, <span class="author">Adams M</span>, <span class="author">Cornette J</span>, <span class="author">Viscogliosi E</span>, <span class="author">Capron M</span> and <span class="author">Balavoine G</span></p>
<p class="affiliation">Inserm U 547, Institut Pasteur de Lille, France. <a href="mailto:%72%61%79%6D%6F%6E%64.%70%69%65%72%63%65%40%70%61%73%74%65%75%72-%6C%69%6C%6C%65.%66%72">&#114;&#97;&#121;&#109;&#111;&#110;&#100;&#46;&#112;&#105;&#101;&#114;&#99;&#101;&#64;&#112;&#97;&#115;&#116;&#101;&#117;&#114;&#45;&#108;&#105;&#108;&#108;&#101;&#46;&#102;&#114;</a></p><p class="abstract">In most bilaterian organisms so far studied, Hox genes are organized in genomic clusters and determine development along the anteroposterior axis. It has been suggested that this clustering, together with spatial and temporal colinearity of gene expression, represents the ancestral condition. However, in organisms with derived modes of embryogenesis and lineage-dependent mechanisms for the determination of cell fate, temporal colinearity of expression can be lost and Hox cluster organization disrupted, as is the case for the ecdysozoans Drosophila melanogaster and Caenorhabditis elegans and the urochordates Ciona intestinalis and Oikopleura dioica. We sought to determine whether a lophotrochozoan, the platyhelminth parasite Schistosoma mansoni, possesses a conserved or disrupted Hox cluster. Using a polymerase chain reaction (PCR)-based strategy, we have cloned and characterized three novel S. mansoni genes encoding orthologues of Drosophila labial (SmHox1), deformed (SmHox4), and abdominal A (SmHox8), as well as the full-length coding sequence of the previously described Smox1, which we identify as an orthologue of fushi tarazu. Quantitative reverse transcriptase-PCR showed that the four genes were expressed at all life-cycle stages but that levels of expression were differentially regulated. Phylogenetic analysis and the conservation of &quot;parapeptide&quot; sequences C-terminal to the homeodomains of SmHox8 and Smox1 support the grouping of platyhelminths within the lophotrochozoan clade. However, Bacterial Artificial Chromosome (BAC) library screening followed by genome walking failed to reconstitute a cluster. The BAC clones containing Hox genes were sequenced, and in no case were other Hox genes found on the same clone. Moreover, the SmHox4 and SmHox8 genes contained single very large introns (&gt;40 kbp) further indicating that the schistosome Hox cluster is highly extended. Localization of the Hox genes to chromosomes using fluorescence in situ hybridization showed that SmHox4 and SmHox8 are on the long arm of chromosome 4, whereas SmHox1 and Smox1 are on chromosome 3. In silico screening of the available genome sequences corroborated results of Southern blotting and BAC library screening that indicate that there are no paralogues of SmHox1, SmHox4, or SmHox8. The schistosome Hox cluster is therefore not duplicated, but is both dispersed and disintegrated in the genome.</p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2005</span>;<span class="volume">22</span>;<span class="number">12</span>;<span class="pagerange">2491-503</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16120809" rel="external">16120809</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsi239" rel="external">10.1093/molbev/msi239</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential expression of two NMDA receptor interacting proteins, PSD-95 and SynGAP during mouse development.</h4>
<p class="authors"><span class="author">Porter K</span>, <span class="author">Komiyama NH</span>, <span class="author">Vitalis T</span>, <span class="author">Kind PC</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Centre for Neuroscience Research, University of Edinburgh, Edinburgh UK.</p><p class="abstract">Patterns of neural activity mediated by N-methyl-D-aspartate (NMDA) receptors are known to play important roles in development of the central nervous system. However, the signalling pathways downstream from NMDA receptors that are critical for normal neuronal development are not yet clearly understood. NMDA receptors interact with various signalling proteins via scaffolding proteins, which are important in adult neuronal and behavioural plasticity. For example, the NR2B subunits of the NMDA receptor interact with postsynaptic density 95 (PSD-95), which in turn binds to synaptic ras GTPase-activating protein (SynGAP). Interestingly, the developmental phenotype of mice carrying null mutations in these genes differ. NR2B and SynGAP homozygote mice die within the first week of birth whereas PSD-95 homozygote mice survive to adulthood. We therefore examined the expression patterns of PSD-95 and SynGAP genes from embryonic stages to adult using lacZ (beta-galactosidase) marker gene knock-in mice. Dramatic changes of expression were observed throughout development in brain and other tissues. Although SynGAP binds PSD-95, both genes had distinct, as well as overlapping expression. SynGAP expression peaked at times of synaptogenesis and developmental plasticity in contrast to PSD-95, which was expressed throughout the brain from early embryonic stages. Furthermore, SynGAP showed a more spatially restricted pattern as illustrated by its restriction to forebrain in contrast to PSD-95, which was also found in mid- and hindbrain. These data support the model that synaptic signalling complexes are heterogeneous and individual components show temporal and spatial specificity during development.</p>
<p class="publication"><span class="title">The European journal of neuroscience</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">2</span>;<span class="pagerange">351-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15673435" rel="external">15673435</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1460-9568.2005.03874.x" rel="external">10.1111/j.1460-9568.2005.03874.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromatin regulation and sumoylation in the inhibition of Ras-induced vulval development in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Poulin G</span>, <span class="author">Dong Y</span>, <span class="author">Fraser AG</span>, <span class="author">Hopper NA</span> and <span class="author">Ahringer J</span></p>
<p class="affiliation">Wellcome Trust/Cancer Research UK Gurdon Institute, University of Cambridge, Cambridge, UK.</p><p class="abstract">In Caenorhabditis elegans, numerous &#39;synMuv&#39; (synthetic multivulval) genes encode for chromatin-associated proteins involved in transcriptional repression, including an orthologue of Rb and components of the NuRD histone deacetylase complex. These genes antagonize Ras signalling to prevent erroneous adoption of vulval fate. To identify new components of this mechanism, we performed a genome-wide RNA interference (RNAi) screen. After RNAi of 16 757 genes, we found nine new synMuv genes. Based on predicted functions and genetic epistasis experiments, we propose that at least four post-translational modifications converge to inhibit Ras-stimulated vulval development: sumoylation, histone tail deacetylation, methylation, and acetylation. In addition, we demonstrate a novel role for sumoylation in inhibiting LIN-12/Notch signalling in the vulva. We further show that many of the synMuv genes are involved in gene regulation outside the vulva, negatively regulating the expression of the Delta homologue lag-2. As most of the genes identified in this screen are conserved in humans, we suggest that similar interactions may be relevant in mammals for control of Ras and Notch signalling, crosstalk between these pathways, and cell proliferation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">054523</span></p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2005</span>;<span class="volume">24</span>;<span class="number">14</span>;<span class="pagerange">2613-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15990876" rel="external">15990876</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1176455" rel="external">1176455</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.emboj.7600726" rel="external">10.1038/sj.emboj.7600726</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel 5q11.2 deletion detected by microarray comparative genomic hybridisation in a child referred as a case of suspected 22q11 deletion syndrome.</h4>
<p class="authors"><span class="author">Prescott K</span>, <span class="author">Woodfine K</span>, <span class="author">Stubbs P</span>, <span class="author">Super M</span>, <span class="author">Kerr B</span>, <span class="author">Palmer R</span>, <span class="author">Carter NP</span> and <span class="author">Scambler P</span></p>
<p class="affiliation">Molecular Medicine Unit, Institute of Child Health, University College London, 30, Guilford Street, London WC1N 1EH, UK.</p><p class="abstract">The 22q11 deletion syndrome (22q11DS) is a developmental syndrome comprising of heart, palate, thymus and parathyroid glands defects. Individuals with 22q11DS usually carry a 1.5- to 3-Mb heterozygous deletion on chromosome 22q11.2. However, there are many patients with features of 22q11DS without a known cause from conventional karyotype and FISH analysis. Six patients with features of 22q11DS, a normal chromosomal and FISH 22q11 analysis, were selected for investigation by microarray genomic comparative hybridisation (array CGH). Array-CGH is a powerful technology enabling detection of submicroscopic chromosome duplications and deletions by comparing a differentially labelled test sample to a control. The samples are co-hybridised to a microarray containing genomic clones and the resulting ratio of fluorescence intensities on each array element is proportional to the DNA copy number difference. No chromosomal changes were detected by hybridisation to a high resolution array representing chromosome 22q. However, one patient was found to have a 6-Mb deletion on 5q11.2 detected by a whole genome 1-Mb array. This deletion was confirmed with fluorescence in-situ hybridisation (FISH) and microsatellite marker analysis. It is the first deletion described in this region. The patient had tetralogy of Fallot, a bifid uvula and velopharyngeal insufficiency, short stature, learning and behavioural difficulties. This case shows the increased sensitivity of array CGH over detailed karyotype analysis for detection of chromosomal changes. It is anticipated that array CGH will improve the clinician&#39;s capacity to diagnose congenital syndromes with an unknown aetiology.</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">116</span>;<span class="number">1-2</span>;<span class="pagerange">83-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15549396" rel="external">15549396</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-004-1195-6" rel="external">10.1007/s00439-004-1195-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adding some SPICE to DAS.</h4>
<p class="authors"><span class="author">Prli&#x107; A</span>, <span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus Hinxton, Cambridge, UK. <a href="mailto:%61%70%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#112;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Unlabelled</strong>: The distributed annotation system (DAS) defines a communication protocol used to exchange biological annotations. It is motivated by the idea that annotations should not be provided by single centralized databases but instead be spread over multiple sites. Data distribution, performed by DAS servers, is separated from visualization, which is carried out by DAS clients. The original DAS protocol was designed to serve annotation of genomic sequences. We have extended the protocol to be applicable to macromolecular structures. Here we present SPICE, a new DAS client that can be used to visualize protein sequence and structure annotations.</p><p class="abstract"><strong>Availability</strong>: http://www.efamily.org.uk/software/dasclients/spice/</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21 Suppl 2</span>;<span class="pagerange">ii40-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16204122">16204122</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656757">2656757</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti1106">10.1093/bioinformatics/bti1106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Binding sites for metabolic disease related transcription factors inferred at base pair resolution by chromatin immunoprecipitation and genomic microarrays.</h4>
<p class="authors"><span class="author">Rada-Iglesias A</span>, <span class="author">Wallerman O</span>, <span class="author">Koch C</span>, <span class="author">Ameur A</span>, <span class="author">Enroth S</span>, <span class="author">Clelland G</span>, <span class="author">Wester K</span>, <span class="author">Wilcox S</span>, <span class="author">Dovey OM</span>, <span class="author">Ellis PD</span>, <span class="author">Wraight VL</span>, <span class="author">James K</span>, <span class="author">Andrews R</span>, <span class="author">Langford C</span>, <span class="author">Dhami P</span>, <span class="author">Carter N</span>, <span class="author">Vetrie D</span>, <span class="author">Pont&eacute;n F</span>, <span class="author">Komorowski J</span>, <span class="author">Dunham I</span> and <span class="author">Wadelius C</span></p>
<p class="affiliation">Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, Sweden.</p><p class="abstract">We present a detailed in vivo characterization of hepatocyte transcriptional regulation in HepG2 cells, using chromatin immunoprecipitation and detection on PCR fragment-based genomic tiling path arrays covering the encyclopedia of DNA element (ENCODE) regions. Our data suggest that HNF-4alpha and HNF-3beta, which were commonly bound to distal regulatory elements, may cooperate in the regulation of a large fraction of the liver transcriptome and that both HNF-4alpha and USF1 may promote H3 acetylation to many of their targets. Importantly, bioinformatic analysis of the sequences bound by each transcription factor (TF) shows an over-representation of motifs highly similar to the in vitro established consensus sequences. On the basis of these data, we have inferred tentative binding sites at base pair resolution. Some of these sites have been previously found by in vitro analysis and some were verified in vitro in this study. Our data suggests that a similar approach could be used for the in vivo characterization of all predicted/uncharacterized TF and that the analysis could be scaled to the whole genome.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5 U01 HG003168</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14</span>;<span class="number">22</span>;<span class="pagerange">3435-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16221759" rel="external">16221759</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi378" rel="external">10.1093/hmg/ddi378</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conformational changes of Escherichia coli sigma54-RNA-polymerase upon closed-promoter complex formation.</h4>
<p class="authors"><span class="author">Ray P</span>, <span class="author">Hall RJ</span>, <span class="author">Finn RD</span>, <span class="author">Chen S</span>, <span class="author">Patwardhan A</span>, <span class="author">Buck M</span> and <span class="author">van Heel M</span></p>
<p class="affiliation">Department of Biological Sciences, Imperial College London, South Kensington Campus, London SW7 2AZ, UK.</p><p class="abstract">RNA polymerase from the mesophile Escherichia coli exists in two forms, the core enzyme and the holoenzyme. Using cryo-electron microscopy and single-particle analysis, we have obtained the structure of the complete RNA polymerase from E.coli containing the sigma54 factor within the closed-promoter complex. Comparisons with earlier reconstructions of the core enzyme and the sigma54 holoenzyme reveal the behaviour of this major variant RNA polymerase in defined functional states. The binding of DNA leads to significant conformational changes in the enzyme&#39;s catalytic subunits, apparently a necessity for the initiation of enhancer-dependent promoter-specific transcription.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">354</span>;<span class="number">2</span>;<span class="pagerange">201-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16246367" rel="external">16246367</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jmb.2005.09.057" rel="external">10.1016/j.jmb.2005.09.057</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tiling path resolution mapping of constitutional 1p36 deletions by array-CGH: contiguous gene deletion or &quot;deletion with positional effect&quot; syndrome?</h4>
<p class="authors"><span class="author">Redon R</span>, <span class="author">Rio M</span>, <span class="author">Gregory SG</span>, <span class="author">Cooper RA</span>, <span class="author">Fiegler H</span>, <span class="author">Sanlaville D</span>, <span class="author">Banerjee R</span>, <span class="author">Scott C</span>, <span class="author">Carr P</span>, <span class="author">Langford C</span>, <span class="author">Cormier-Daire V</span>, <span class="author">Munnich A</span>, <span class="author">Carter NP</span> and <span class="author">Colleaux L</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">42</span>;<span class="number">2</span>;<span class="pagerange">166-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15689456" rel="external">15689456</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735995" rel="external">1735995</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2004.023861" rel="external">10.1136/jmg.2004.023861</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SmartCapture and the frontiers of FISH technology: report of the Digital Scientific UK SmartCapture User&#39;s Meeting, Peterhouse College Cambridge, UK, 2nd September 2005.</h4>
<p class="authors"><span class="author">Reid AG</span> and <span class="author">Gribble SM</span></p>
<p class="affiliation">Digital Scientific UK, Sheraton House, Cambridge. <a href="mailto:%61%6C%69%73%74%61%69%72.%72%65%69%64%40%64%69%67%69%74%61%6C%73%63%69%65%6E%74%69%66%69%63.%63%6F.%75%6B">&#97;&#108;&#105;&#115;&#116;&#97;&#105;&#114;&#46;&#114;&#101;&#105;&#100;&#64;&#100;&#105;&#103;&#105;&#116;&#97;&#108;&#115;&#99;&#105;&#101;&#110;&#116;&#105;&#102;&#105;&#99;&#46;&#99;&#111;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">8</span>;<span class="pagerange">835-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16331415" rel="external">16331415</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-005-1014-5" rel="external">10.1007/s10577-005-1014-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Flag-2 locus, an ancestral gene cluster, is potentially associated with a novel flagellar system from Escherichia coli.</h4>
<p class="authors"><span class="author">Ren CP</span>, <span class="author">Beatson SA</span>, <span class="author">Parkhill J</span> and <span class="author">Pallen MJ</span></p>
<p class="affiliation">Bacterial Pathogenesis and Genomics Unit, Division of Immunity and Infection, Medical School, University of Birmingham, Birmingham, England B15 2TT.</p><p class="abstract">Escherichia coli K-12 possesses two adjacent, divergent, promoterless flagellar genes, fhiA-mbhA, that are absent from Salmonella enterica. Through bioinformatics analysis, we found that these genes are remnants of an ancestral 44-gene cluster and are capable of encoding a novel flagellar system, Flag-2. In enteroaggregative E. coli strain 042, there is a frameshift in lfgC that is likely to have inactivated the system in this strain. Tiling path PCR studies showed that the Flag-2 cluster is present in 15 of 72 of the well-characterized ECOR strains. The Flag-2 system resembles the lateral flagellar systems of Aeromonas and Vibrio, particularly in its apparent dependence on RpoN. Unlike the conventional Flag-1 flagellin, the Flag-2 flagellin shows a remarkable lack of sequence polymorphism. The Flag-2 gene cluster encodes a flagellar type III secretion system (including a dedicated flagellar sigma-antisigma combination), thus raising the number of distinct type III secretion systems in Escherichia/Shigella to five. The presence of the Flag-2 cluster at identical sites in E. coli and its close relative Citrobacter rodentium, combined with its absence from S. enterica, suggests that it was acquired by horizontal gene transfer after the former two species diverged from Salmonella. The presence of Flag-2-like gene clusters in Yersinia pestis, Yersinia pseudotuberculosis, and Chromobacterium violaceum suggests that coexistence of two flagellar systems within the same species is more common than previously suspected. The fact that the Flag-2 gene cluster was not discovered in the first 10 Escherichia/Shigella genome sequences studied emphasizes the importance of maintaining an energetic program of genome sequencing for this important taxonomic group.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2005</span>;<span class="volume">187</span>;<span class="number">4</span>;<span class="pagerange">1430-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15687208" rel="external">15687208</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC545627" rel="external">545627</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.187.4.1430-1440.2005" rel="external">10.1128/JB.187.4.1430-1440.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Birc6 (Bruce) gene regulates p53 and the mitochondrial pathway of apoptosis and is essential for mouse embryonic development.</h4>
<p class="authors"><span class="author">Ren J</span>, <span class="author">Shi M</span>, <span class="author">Liu R</span>, <span class="author">Yang QH</span>, <span class="author">Johnson T</span>, <span class="author">Skarnes WC</span> and <span class="author">Du C</span></p>
<p class="affiliation">Stowers Institute for Medical Research, Kansas City, MO 64110, USA.</p><p class="abstract">Baculoviral inhibitor of apoptosis repeat-containing (Birc)6 gene/BIRC6 (Bruce/APOLLON) encodes an inhibitor of apoptosis and a chimeric E2/E3 ubiquitin ligase in mammals. The physiological role of Bruce in antiapoptosis is unknown. Here, we show that deletion of the C-terminal half of Bruce, including the UBC domain, causes activation of caspases and apoptosis in the placenta and yolk sac, leading to embryonic lethality. This apoptosis is associated with up-regulation and nuclear localization of the tumor suppressor p53 and activation of mitochondrial apoptosis, which includes up-regulation of Bax, Bak, and Pidd, translocation of Bax and caspase-2 onto mitochondria, release of cytochrome c and apoptosis-inducing factor, and activation of caspase-9 and caspase-3. Mutant mouse embryonic fibroblasts are sensitive to multiple mitochondrial death stimuli but resistant to TNF. In addition, eliminating p53 by RNA interference rescues cell viability induced by Bruce ablation in human cell line H460. This viability preservation results from reduced expression of proapoptotic factors Bax, Bak, and Pidd and from prevention of activation of caspase-2, -9, and -3. The amount of second mitochondrial-derived activator of caspase and Omi does not change. We conclude that p53 is a downstream effector of Bruce, and, in response to loss of Bruce function, p53 activates Pidd/caspase-2 and Bax/Bak, leading to mitochondrial apoptosis.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2005</span>;<span class="volume">102</span>;<span class="number">3</span>;<span class="pagerange">565-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15640352" rel="external">15640352</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC543482" rel="external">543482</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0408744102" rel="external">10.1073/pnas.0408744102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prenatal diagnosis by array-CGH.</h4>
<p class="authors"><span class="author">Rickman L</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span> and <span class="author">Bobrow M</span></p>
<p class="affiliation">University of Cambridge Department of Medical Genetics, Addenbrooke&#39;s Hospital, Hills Road, Cambridge, CB2 2QQ, United Kingdom. <a href="mailto:%6C%72%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#114;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Microscopic karyotype analysis of cultured cells has been regarded as the gold standard for prenatal diagnosis for over 30 years. Since the first application of this technique to prenatal testing in the early 1970&#39;s, this procedure has proved to be highly reliable for identifying chromosome copy number abnormalities (aneuploidy) and large structural rearrangements in foetal cells obtained invasively by either amniocentesis or chorionic villus sampling (CVS). Recognising the need for more rapid testing methods which do not require cell culture, fluorescence in situ hybridisation (FISH) and quantitative fluorescence PCR (QF-PCR) have been introduced to this field in order to answer specific diagnostic questions. However, both FISH and QF-PCR suffer the disadvantage in that they are difficult to scale to a comprehensive, genome-wide screen. Array-comparative genomic hybridisation (array-CGH) in contrast is a comprehensive, genome-wide screening strategy for detecting DNA copy number imbalances which can be rapid, less labour-intensive than karyotype banding analysis and is highly amenable to automation. Array-CGH has the potential to be used for prenatal diagnosis and may address many of the limitations of both conventional microscopic cytogenetic analyses and the more recently employed rapid-screening strategies.</p>
<p class="publication"><span class="title">European journal of medical genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">48</span>;<span class="number">3</span>;<span class="pagerange">232-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16179219" rel="external">16179219</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ejmg.2005.03.003" rel="external">10.1016/j.ejmg.2005.03.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A member of the cAMP receptor protein family of transcription regulators in Mycobacterium tuberculosis is required for virulence in mice and controls transcription of the rpfA gene coding for a resuscitation promoting factor.</h4>
<p class="authors"><span class="author">Rickman L</span>, <span class="author">Scott C</span>, <span class="author">Hunt DM</span>, <span class="author">Hutchinson T</span>, <span class="author">Men&eacute;ndez MC</span>, <span class="author">Whalan R</span>, <span class="author">Hinds J</span>, <span class="author">Colston MJ</span>, <span class="author">Green J</span> and <span class="author">Buxton RS</span></p>
<p class="affiliation">Division of Mycobacterial Research, National Institute for Medical Research, Mill Hill, London NW7 1AA, UK.</p><p class="abstract">Deletion of gene Rv3676 in Mycobacterium tuberculosis coding for a transcription factor belonging to the cAMP receptor protein (CRP) family caused growth defects in laboratory medium, in bone marrow-derived macrophages and in a mouse model of tuberculosis. Transcript profiling of M. tuberculosis grown in vitro identified 16 genes with significantly altered expression in the mutant compared with the wild type. Analysis of the DNA sequences upstream of the corresponding open reading frames revealed that 12 possessed sequences related to a consensus CRP binding site that could represent the sites of action of Rv3676. These included rpfA, lprQ, whiB1 and ahpC among genes with enhanced expression in the wild type, and Rv3616c-Rv3613c, Rv0188 and lipQ among genes exhibiting enhanced expression in the mutant. The activity of an rpfA::lacZ promoter fusion was lowered in the Rv3676 mutant and by mutation of the predicted Rv3676 binding site. Moreover, the product of Rv3676 (isolated as a TrxA fusion protein) interacted specifically with the rpfA promoter, and binding was inhibited by mutation of the Rv3676 site. Although Rv3676 retains four of the six amino acid residues that bind cAMP in Escherichia coli CRP addition of cAMP did not enhance Rv3676 binding at the rpfA promoter in vitro. In summary, it has been shown that Rv3676 is a direct regulator of rpfA expression, and because rpfA codes for a resuscitation promoting factor this may implicate Rv3676 in reactivation of dormant M. tuberculosis infections.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">U.1175.02.002.00013 (85867)</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">56</span>;<span class="number">5</span>;<span class="pagerange">1274-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15882420" rel="external">15882420</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2964915" rel="external">2964915</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2005.04609.x" rel="external">10.1111/j.1365-2958.2005.04609.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Signature of recent historical events in the European Y-chromosomal STR haplotype distribution.</h4>
<p class="authors"><span class="author">Roewer L</span>, <span class="author">Croucher PJ</span>, <span class="author">Willuweit S</span>, <span class="author">Lu TT</span>, <span class="author">Kayser M</span>, <span class="author">Lessig R</span>, <span class="author">de Knijff P</span>, <span class="author">Jobling MA</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Krawczak M</span></p>
<p class="affiliation">Institute of Legal Medicine, Humboldt-University, Berlin, Germany.</p><p class="abstract">Previous studies of human Y-chromosomal single-nucleotide polymorphisms (Y-SNPs) established a link between the extant Y-SNP haplogroup distribution and the prehistoric demography of Europe. By contrast, our analysis of seven rapidly evolving Y-chromosomal short tandem repeat loci (Y-STRs) in over 12,700 samples from 91 different locations in Europe reveals a signature of more recent historic events, not previously detected by other genetic markers. Cluster analysis based upon molecular variance yields two clearly identifiable sub-clusters of Western and Eastern European Y-STR haplotypes, and a diverse transition zone in central Europe, where haplotype spectra change more rapidly with longitude than with latitude. This and other observed patterns of Y-STR similarity may plausibly be related to particular historical incidents, including, for example, the expansion of the Franconian and Ottoman Empires. We conclude that Y-STRs may be capable of resolving male genealogies to an unparalleled degree and could therefore provide a useful means to study local population structure and recent demographic history.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">116</span>;<span class="number">4</span>;<span class="pagerange">279-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15660227" rel="external">15660227</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-004-1201-z" rel="external">10.1007/s00439-004-1201-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the chicken C-type lectin-like receptors B-NK and B-lec suggests that the NK complex and the MHC share a common ancestral region.</h4>
<p class="authors"><span class="author">Rogers SL</span>, <span class="author">G&ouml;bel TW</span>, <span class="author">Viertlboeck BC</span>, <span class="author">Milne S</span>, <span class="author">Beck S</span> and <span class="author">Kaufman J</span></p>
<p class="affiliation">Institute for Animal Health, Compton, Berkshire, United Kingdom.</p><p class="abstract">The sequencing of the chicken MHC led to the identification of two open reading frames, designated B-NK and B-lec, that were predicted to encode C-type lectin domains. C-type lectin domains are not encoded in the MHC of any animal described to date; therefore, this observation was completely unexpected, particularly given that the chicken has a &quot;minimal essential MHC.&quot; In this study, we describe the initial characterization of the B-NK and B-lec genes, and show that they share greatest homology with C-type lectin-like receptors encoded in the human NK complex (NKC), in particular NKR-P1 and lectin-like transcript 1 (LLT1), respectively. In common with NKR-P1 and LLT1, B-NK and B-lec are located next to each other and transcribed in opposite orientation. Like human NKR-P1, B-NK has a functional inhibitory signaling motif in the cytoplasmic tail and is expressed in NK cells. In contrast, B-lec contains an endocytosis motif in the cytoplasmic tail, and like LLT1, is an early activation Ag. Further analysis leads us to propose that there are four subgroups of C-type lectin-like receptors in the NKC, which arose as a result of duplication events. Moreover, this analysis suggests that the NKC may be considered a fifth paralogous region, and therefore shares an ancient common origin with the MHC. This provides evidence that C-type lectin-like receptors were present in the preduplication, primordial MHC region, and suggests that an original function of MHC molecules was for recognition by NK cell receptors encoded nearby.</p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2005</span>;<span class="volume">174</span>;<span class="number">6</span>;<span class="pagerange">3475-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15749883" rel="external">15749883</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">More on: polymorphism and hemophilia A causing inversions in distal Xq28: a complex picture.</h4>
<p class="authors"><span class="author">Ross MT</span> and <span class="author">Bentley DR</span></p>
<p class="publication"><span class="title">Journal of thrombosis and haemostasis : JTH</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">11</span>;<span class="pagerange">2600-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16241968" rel="external">16241968</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1538-7836.2005.01622.x" rel="external">10.1111/j.1538-7836.2005.01622.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DNA sequence of the human X chromosome.</h4>
<p class="authors"><span class="author">Ross MT</span>, <span class="author">Grafham DV</span>, <span class="author">Coffey AJ</span>, <span class="author">Scherer S</span>, <span class="author">McLay K</span>, <span class="author">Muzny D</span>, <span class="author">Platzer M</span>, <span class="author">Howell GR</span>, <span class="author">Burrows C</span>, <span class="author">Bird CP</span>, <span class="author">Frankish A</span>, <span class="author">Lovell FL</span>, <span class="author">Howe KL</span>, <span class="author">Ashurst JL</span>, <span class="author">Fulton RS</span>, <span class="author">Sudbrak R</span>, <span class="author">Wen G</span>, <span class="author">Jones MC</span>, <span class="author">Hurles ME</span>, <span class="author">Andrews TD</span>, <span class="author">Scott CE</span>, <span class="author">Searle S</span>, <span class="author">Ramser J</span>, <span class="author">Whittaker A</span>, <span class="author">Deadman R</span>, <span class="author">Carter NP</span>, <span class="author">Hunt SE</span>, <span class="author">Chen R</span>, <span class="author">Cree A</span>, <span class="author">Gunaratne P</span>, <span class="author">Havlak P</span>, <span class="author">Hodgson A</span>, <span class="author">Metzker ML</span>, <span class="author">Richards S</span>, <span class="author">Scott G</span>, <span class="author">Steffen D</span>, <span class="author">Sodergren E</span>, <span class="author">Wheeler DA</span>, <span class="author">Worley KC</span>, <span class="author">Ainscough R</span>, <span class="author">Ambrose KD</span>, <span class="author">Ansari-Lari MA</span>, <span class="author">Aradhya S</span>, <span class="author">Ashwell RI</span>, <span class="author">Babbage AK</span>, <span class="author">Bagguley CL</span>, <span class="author">Ballabio A</span>, <span class="author">Banerjee R</span>, <span class="author">Barker GE</span>, <span class="author">Barlow KF</span>, <span class="author">Barrett IP</span>, <span class="author">Bates KN</span>, <span class="author">Beare DM</span>, <span class="author">Beasley H</span>, <span class="author">Beasley O</span>, <span class="author">Beck A</span>, <span class="author">Bethel G</span>, <span class="author">Blechschmidt K</span>, <span class="author">Brady N</span>, <span class="author">Bray-Allen S</span>, <span class="author">Bridgeman AM</span>, <span class="author">Brown AJ</span>, <span class="author">Brown MJ</span>, <span class="author">Bonnin D</span>, <span class="author">Bruford EA</span>, <span class="author">Buhay C</span>, <span class="author">Burch P</span>, <span class="author">Burford D</span>, <span class="author">Burgess J</span>, <span class="author">Burrill W</span>, <span class="author">Burton J</span>, <span class="author">Bye JM</span>, <span class="author">Carder C</span>, <span class="author">Carrel L</span>, <span class="author">Chako J</span>, <span class="author">Chapman JC</span>, <span class="author">Chavez D</span>, <span class="author">Chen E</span>, <span class="author">Chen G</span>, <span class="author">Chen Y</span>, <span class="author">Chen Z</span>, <span class="author">Chinault C</span>, <span class="author">Ciccodicola A</span>, <span class="author">Clark SY</span>, <span class="author">Clarke G</span>, <span class="author">Clee CM</span>, <span class="author">Clegg S</span>, <span class="author">Clerc-Blankenburg K</span>, <span class="author">Clifford K</span>, <span class="author">Cobley V</span>, <span class="author">Cole CG</span>, <span class="author">Conquer JS</span>, <span class="author">Corby N</span>, <span class="author">Connor RE</span>, <span class="author">David R</span>, <span class="author">Davies J</span>, <span class="author">Davis C</span>, <span class="author">Davis J</span>, <span class="author">Delgado O</span>, <span class="author">Deshazo D</span>, <span class="author">Dhami P</span>, <span class="author">Ding Y</span>, <span class="author">Dinh H</span>, <span class="author">Dodsworth S</span>, <span class="author">Draper H</span>, <span class="author">Dugan-Rocha S</span>, <span class="author">Dunham A</span>, <span class="author">Dunn M</span>, <span class="author">Durbin KJ</span>, <span class="author">Dutta I</span>, <span class="author">Eades T</span>, <span class="author">Ellwood M</span>, <span class="author">Emery-Cohen A</span>, <span class="author">Errington H</span>, <span class="author">Evans KL</span>, <span class="author">Faulkner L</span>, <span class="author">Francis F</span>, <span class="author">Frankland J</span>, <span class="author">Fraser AE</span>, <span class="author">Galgoczy P</span>, <span class="author">Gilbert J</span>, <span class="author">Gill R</span>, <span class="author">Gl&ouml;ckner G</span>, <span class="author">Gregory SG</span>, <span class="author">Gribble S</span>, <span class="author">Griffiths C</span>, <span class="author">Grocock R</span>, <span class="author">Gu Y</span>, <span class="author">Gwilliam R</span>, <span class="author">Hamilton C</span>, <span class="author">Hart EA</span>, <span class="author">Hawes A</span>, <span class="author">Heath PD</span>, <span class="author">Heitmann K</span>, <span class="author">Hennig S</span>, <span class="author">Hernandez J</span>, <span class="author">Hinzmann B</span>, <span class="author">Ho S</span>, <span class="author">Hoffs M</span>, <span class="author">Howden PJ</span>, <span class="author">Huckle EJ</span>, <span class="author">Hume J</span>, <span class="author">Hunt PJ</span>, <span class="author">Hunt AR</span>, <span class="author">Isherwood J</span>, <span class="author">Jacob L</span>, <span class="author">Johnson D</span>, <span class="author">Jones S</span>, <span class="author">de Jong PJ</span>, <span class="author">Joseph SS</span>, <span class="author">Keenan S</span>, <span class="author">Kelly S</span>, <span class="author">Kershaw JK</span>, <span class="author">Khan Z</span>, <span class="author">Kioschis P</span>, <span class="author">Klages S</span>, <span class="author">Knights AJ</span>, <span class="author">Kosiura A</span>, <span class="author">Kovar-Smith C</span>, <span class="author">Laird GK</span>, <span class="author">Langford C</span>, <span class="author">Lawlor S</span>, <span class="author">Leversha M</span>, <span class="author">Lewis L</span>, <span class="author">Liu W</span>, <span class="author">Lloyd C</span>, <span class="author">Lloyd DM</span>, <span class="author">Loulseged H</span>, <span class="author">Loveland JE</span>, <span class="author">Lovell JD</span>, <span class="author">Lozado R</span>, <span class="author">Lu J</span>, <span class="author">Lyne R</span>, <span class="author">Ma J</span>, <span class="author">Maheshwari M</span>, <span class="author">Matthews LH</span>, <span class="author">McDowall J</span>, <span class="author">McLaren S</span>, <span class="author">McMurray A</span>, <span class="author">Meidl P</span>, <span class="author">Meitinger T</span>, <span class="author">Milne S</span>, <span class="author">Miner G</span>, <span class="author">Mistry SL</span>, <span class="author">Morgan M</span>, <span class="author">Morris S</span>, <span class="author">M&uuml;ller I</span>, <span class="author">Mullikin JC</span>, <span class="author">Nguyen N</span>, <span class="author">Nordsiek G</span>, <span class="author">Nyakatura G</span>, <span class="author">O&#39;Dell CN</span>, <span class="author">Okwuonu G</span>, <span class="author">Palmer S</span>, <span class="author">Pandian R</span>, <span class="author">Parker D</span>, <span class="author">Parrish J</span>, <span class="author">Pasternak S</span>, <span class="author">Patel D</span>, <span class="author">Pearce AV</span>, <span class="author">Pearson DM</span>, <span class="author">Pelan SE</span>, <span class="author">Perez L</span>, <span class="author">Porter KM</span>, <span class="author">Ramsey Y</span>, <span class="author">Reichwald K</span>, <span class="author">Rhodes S</span>, <span class="author">Ridler KA</span>, <span class="author">Schlessinger D</span>, <span class="author">Schueler MG</span>, <span class="author">Sehra HK</span>, <span class="author">Shaw-Smith C</span>, <span class="author">Shen H</span>, <span class="author">Sheridan EM</span>, <span class="author">Shownkeen R</span>, <span class="author">Skuce CD</span>, <span class="author">Smith ML</span>, <span class="author">Sotheran EC</span>, <span class="author">Steingruber HE</span>, <span class="author">Steward CA</span>, <span class="author">Storey R</span>, <span class="author">Swann RM</span>, <span class="author">Swarbreck D</span>, <span class="author">Tabor PE</span>, <span class="author">Taudien S</span>, <span class="author">Taylor T</span>, <span class="author">Teague B</span>, <span class="author">Thomas K</span>, <span class="author">Thorpe A</span>, <span class="author">Timms K</span>, <span class="author">Tracey A</span>, <span class="author">Trevanion S</span>, <span class="author">Tromans AC</span>, <span class="author">d&#39;Urso M</span>, <span class="author">Verduzco D</span>, <span class="author">Villasana D</span>, <span class="author">Waldron L</span>, <span class="author">Wall M</span>, <span class="author">Wang Q</span>, <span class="author">Warren J</span>, <span class="author">Warry GL</span>, <span class="author">Wei X</span>, <span class="author">West A</span>, <span class="author">Whitehead SL</span>, <span class="author">Whiteley MN</span>, <span class="author">Wilkinson JE</span>, <span class="author">Willey DL</span>, <span class="author">Williams G</span>, <span class="author">Williams L</span>, <span class="author">Williamson A</span>, <span class="author">Williamson H</span>, <span class="author">Wilming L</span>, <span class="author">Woodmansey RL</span>, <span class="author">Wray PW</span>, <span class="author">Yen J</span>, <span class="author">Zhang J</span>, <span class="author">Zhou J</span>, <span class="author">Zoghbi H</span>, <span class="author">Zorilla S</span>, <span class="author">Buck D</span>, <span class="author">Reinhardt R</span>, <span class="author">Poustka A</span>, <span class="author">Rosenthal A</span>, <span class="author">Lehrach H</span>, <span class="author">Meindl A</span>, <span class="author">Minx PJ</span>, <span class="author">Hillier LW</span>, <span class="author">Willard HF</span>, <span class="author">Wilson RK</span>, <span class="author">Waterston RH</span>, <span class="author">Rice CM</span>, <span class="author">Vaudin M</span>, <span class="author">Coulson A</span>, <span class="author">Nelson DL</span>, <span class="author">Weinstock G</span>, <span class="author">Sulston JE</span>, <span class="author">Durbin R</span>, <span class="author">Hubbard T</span>, <span class="author">Gibbs RA</span>, <span class="author">Beck S</span>, <span class="author">Rogers J</span> and <span class="author">Bentley DR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%74%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#116;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The human X chromosome has a unique biology that was shaped by its evolution as the sex chromosome shared by males and females. We have determined 99.3% of the euchromatic sequence of the X chromosome. Our analysis illustrates the autosomal origin of the mammalian sex chromosomes, the stepwise process that led to the progressive loss of recombination between X and Y, and the extent of subsequent degradation of the Y chromosome. LINE1 repeat elements cover one-third of the X chromosome, with a distribution that is consistent with their proposed role as way stations in the process of X-chromosome inactivation. We found 1,098 genes in the sequence, of which 99 encode proteins expressed in testis and in various tumour types. A disproportionately high number of mendelian diseases are documented for the X chromosome. Of this number, 168 have been explained by mutations in 113 X-linked genes, which in many cases were characterized with the aid of the DNA sequence.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">434</span>;<span class="number">7031</span>;<span class="pagerange">325-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15772651">15772651</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2665286">2665286</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03440">10.1038/nature03440</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Refining molecular analysis in the pathways of colorectal carcinogenesis.</h4>
<p class="authors"><span class="author">Rowan A</span>, <span class="author">Halford S</span>, <span class="author">Gaasenbeek M</span>, <span class="author">Kemp Z</span>, <span class="author">Sieber O</span>, <span class="author">Volikos E</span>, <span class="author">Douglas E</span>, <span class="author">Fiegler H</span>, <span class="author">Carter N</span>, <span class="author">Talbot I</span>, <span class="author">Silver A</span> and <span class="author">Tomlinson I</span></p>
<p class="affiliation">Molecular and Population Genetics Laboratory, London Research Institute, Cancer Research UK, 44 Lincoln&#39;s Inn Fields, London WC2A 3PX, UK.</p><p class="abstract">In the stepwise model, specific genetic and epigenetic changes accumulate as colorectal adenomas progress to carcinomas (CRCs). CRCs also acquire global phenotypes, particularly microsatellite instability (MSI) and aneuploidy/polyploidy (chromosomal instability, CIN). Few changes specific to MSI-low or CIN+ cancers have been established.</p><p class="abstract"><strong>Methods</strong>: We investigated 100 CRCs for: mutations and loss of heterozygosity (LOH) where appropriate, of APC, K-ras, BRAF, SMAD4, and p53; deletion on 5q around APC and 18q around SMAD4; total chromosomal-scale losses and gains; MSI; and CIN.</p><p class="abstract"><strong>Results</strong>: As expected, CIN- cancers had fewer chromosomal changes overall than CIN+ lesions, but after correcting for this, 5q deletions alone predicted CIN+ status. 5q deletions were not, however, significantly associated with APC mutations, which were equally frequent in CIN+ and CIN- tumors. We therefore found no evidence to show that mutant APC promotes CIN. p53 mutations/LOH were more common in CIN+ than CIN- lesions, and all chromosomal amplifications were in CIN+ tumors. CIN- cancers could be subdivided according to the total number of chromosomal-scale changes into CIN-low and CIN-stable groups; 18q deletion was the best predictor, being present in nearly all CIN-low lesions and almost no CIN-stable tumors. MSI-low was not associated with CIN, any specific mutation, a mutational signature, or clinicopathologic characteristic.</p><p class="abstract"><strong>Conclusions</strong>: Overall, the components of the stepwise model (APC, K-ras, and p53 mutations, plus 18q LOH) tended to co-occur randomly. We propose an updated version of this model comprising 4 pathways of CRC pathogenesis, on the basis of 5q/18q deletions, MSI (high/low), and CIN (high/low/stable).</p>
<p class="publication"><span class="title">Clinical gastroenterology and hepatology : the official clinical practice journal of the American Gastroenterological Association</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">11</span>;<span class="pagerange">1115-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16271343" rel="external">16271343</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sustained cadherin 23 expression in young and adult cochlea of normal and hearing-impaired mice.</h4>
<p class="authors"><span class="author">Rzadzinska AK</span>, <span class="author">Derr A</span>, <span class="author">Kachar B</span> and <span class="author">Noben-Trauth K</span></p>
<p class="affiliation">Section on Structural Cell Biology, Laboratory of Cellular Biology, National Institute on Deafness and Other Communication Disorders, National Institutes of Health, Bethesda, MD 20892, USA.</p><p class="abstract">Cadherin 23 encodes a single-pass transmembrane protein with 27 extracellular cadherin-domains and localizes to stereocilia where it functions as an inter-stereocilia link. Cadherin 23-deficient mice show congenital deafness in combination with circling behavior as a result of organizational defects in the stereocilia hair bundle; common inbred mouse strains carrying the hypomorphic Cdh23(753A) allele are highly susceptible to sensorineural hearing loss. Here, we show that an antibody (N1086) directed against the intracellular carboxyterminus reacts specifically with cadherin 23 and detects with high sensitivity the isoform devoid of the peptide encoded by exon 68 (CDH23Delta68). Cochlea, vestibule, eye, brain and testis produce the CDH23Delta68 isoform in abundance and form moieties with different molecular weight due to variations in glycosylation content. In the cochlea, CDH23Delta68 expression is highest at postnatal day 1 (P1) and P7; expression is down regulated through P14 and P21 and persists at a low steady-state level throughout adulthood (P160). Furthermore, CDH23Delta68 expression levels in young and adult cochlea are similar among normal and hearing deficient strains (C3HeB/FeJ, C57BL/6J and BUB/BnJ). Finally, by immunofluorescence using an antibody (Pb240) specific for ectodomain 14, we show that cadherin 23 localizes to stereocilia during hair bundle development in late gestation and early postnatal days. Cadherin 23-specific labeling becomes weaker as the hair bundle matures but faint labeling concentrated near the top of stereocilia is still detectable at P35. No labeling of cochlea stereocilia was observed with N1086. In conclusion, our data describe a cadherin 23-specific antibody with high affinity to the CDH23Delta68 isoform, reveal a dynamic cochlea expression and localization profile and show sustained cadherin 23 levels in adult cochlea of normal and hearing-impaired mice.</p>
<p class="publication"><span class="title">Hearing research</span>&nbsp;<span class="year">2005</span>;<span class="volume">208</span>;<span class="number">1-2</span>;<span class="pagerange">114-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16005171" rel="external">16005171</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.heares.2005.05.008" rel="external">10.1016/j.heares.2005.05.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Full-genome RNAi profiling of early embryogenesis in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">S&ouml;nnichsen B</span>, <span class="author">Koski LB</span>, <span class="author">Walsh A</span>, <span class="author">Marschall P</span>, <span class="author">Neumann B</span>, <span class="author">Brehm M</span>, <span class="author">Alleaume AM</span>, <span class="author">Artelt J</span>, <span class="author">Bettencourt P</span>, <span class="author">Cassin E</span>, <span class="author">Hewitson M</span>, <span class="author">Holz C</span>, <span class="author">Khan M</span>, <span class="author">Lazik S</span>, <span class="author">Martin C</span>, <span class="author">Nitzsche B</span>, <span class="author">Ruer M</span>, <span class="author">Stamford J</span>, <span class="author">Winzi M</span>, <span class="author">Heinkel R</span>, <span class="author">R&ouml;der M</span>, <span class="author">Finell J</span>, <span class="author">H&auml;ntsch H</span>, <span class="author">Jones SJ</span>, <span class="author">Jones M</span>, <span class="author">Piano F</span>, <span class="author">Gunsalus KC</span>, <span class="author">Oegema K</span>, <span class="author">G&ouml;nczy P</span>, <span class="author">Coulson A</span>, <span class="author">Hyman AA</span> and <span class="author">Echeverri CJ</span></p>
<p class="affiliation">Cenix BioScience GmbH, Tatzberg 47-51, D-01307 Dresden, Germany. <a href="mailto:%73%6F%65%6E%6E%69%63%68%73%65%6E%40%63%65%6E%69%78-%62%69%6F%73%63%69%65%6E%63%65.%63%6F%6D">&#115;&#111;&#101;&#110;&#110;&#105;&#99;&#104;&#115;&#101;&#110;&#64;&#99;&#101;&#110;&#105;&#120;&#45;&#98;&#105;&#111;&#115;&#99;&#105;&#101;&#110;&#99;&#101;&#46;&#99;&#111;&#109;</a></p><p class="abstract">A key challenge of functional genomics today is to generate well-annotated data sets that can be interpreted across different platforms and technologies. Large-scale functional genomics data often fail to connect to standard experimental approaches of gene characterization in individual laboratories. Furthermore, a lack of universal annotation standards for phenotypic data sets makes it difficult to compare different screening approaches. Here we address this problem in a screen designed to identify all genes required for the first two rounds of cell division in the Caenorhabditis elegans embryo. We used RNA-mediated interference to target 98% of all genes predicted in the C. elegans genome in combination with differential interference contrast time-lapse microscopy. Through systematic annotation of the resulting movies, we developed a phenotypic profiling system, which shows high correlation with cellular processes and biochemical pathways, thus enabling us to predict new functions for previously uncharacterized genes.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">434</span>;<span class="number">7032</span>;<span class="pagerange">462-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15791247">15791247</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03353">10.1038/nature03353</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single haplotype analysis demonstrates rapid evolution of the killer immunoglobulin-like receptor (KIR) loci in primates.</h4>
<p class="authors"><span class="author">Sambrook JG</span>, <span class="author">Bashirova A</span>, <span class="author">Palmer S</span>, <span class="author">Sims S</span>, <span class="author">Trowsdale J</span>, <span class="author">Abi-Rached L</span>, <span class="author">Parham P</span>, <span class="author">Carrington M</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kindgom.</p><p class="abstract">The human killer immunoglobulin-like receptors (KIR) are encoded within the Leukocyte Receptor Complex (LRC) on chromosome 19q13.4. Here we report the comparative genomic analysis of single KIR haplotypes in two other primates. In the common chimpanzee (Pan troglodytes), seven KIR genes (ptKIRnewI, ptKIRnewII, ptKIR2DL5, ptKIRnewIII, ptKIR3DP1, ptKIR2DL4, ptKIR3DL1/2) have been identified, and five KIR genes (mmKIRnewI, mmKIR1D, mmKIR2DL4, mmKIR3DL10, mmKIR3DL1) are present in the haplotype sequenced for the rhesus macaque (Macaca mulatta). Additional cDNA analysis confirms the genes predicted from the genomic sequence and reveals the presence of a fifth novel KIR gene (mmKIRnewII) in the second haplotype of the rhesus macaque. While all known human haplotypes contain both activating and inhibitory KIR genes, only inhibitory KIR genes (characterized by long cytoplasmic tails) were found by in silico and cDNA analyses in the two primate haplotypes studied here. Comparison of the two human and the two non-human primate haplotypes demonstrates rapid diversification of the KIR gene family members, many of which have diverged in a species-specific manner. An analysis of the intronic regions of the two non-human primates reveals the presence of ancient repeat elements, which are indicative of the duplication events that have taken place since the last common ancestor.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">N01-CO-12400</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">1</span>;<span class="pagerange">25-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15632087" rel="external">15632087</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC540275" rel="external">540275</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2381205" rel="external">10.1101/gr.2381205</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide survey of Major Histocompatibility Complex (MHC) genes and their paralogues in zebrafish.</h4>
<p class="authors"><span class="author">Sambrook JG</span>, <span class="author">Figueroa F</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 ISA, UK. <a href="mailto:%6A%73%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#115;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The genomic organisation of the Major Histocompatibility Complex (MHC) varies greatly between different vertebrates. In mammals, the classical MHC consists of a large number of linked genes (e.g. greater than 200 in humans) with predominantly immune function. In some birds, it consists of only a small number of linked MHC core genes (e.g. smaller than 20 in chickens) forming a minimal essential MHC and, in fish, the MHC consists of a so far unknown number of genes including non-linked MHC core genes. Here we report a survey of MHC genes and their paralogues in the zebrafish genome.</p><p class="abstract"><strong>Results</strong>: Using sequence similarity searches against the zebrafish draft genome assembly (Zv4, September 2004), 149 putative MHC gene loci and their paralogues have been identified. Of these, 41 map to chromosome 19 while the remaining loci are spread across essentially all chromosomes. Despite the fragmentation, a set of MHC core genes involved in peptide transport, loading and presentation are still found in a single linkage group.</p><p class="abstract"><strong>Conclusion</strong>: The results extend the linkage information of MHC core genes on zebrafish chromosome 19 and show the distribution of the remaining MHC genes and their paralogues to be genome-wide. Although based on a draft genome assembly, this survey demonstrates an essentially fragmented MHC in zebrafish.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="pagerange">152</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16271140" rel="external">16271140</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1309616" rel="external">1309616</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-6-152" rel="external">10.1186/1471-2164-6-152</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GeneMCL in microarray analysis.</h4>
<p class="authors"><span class="author">Samuel Lattimore B</span>, <span class="author">van Dongen S</span> and <span class="author">Crabbe MJ</span></p>
<p class="affiliation">School of Animal and Microbial Sciences, University of Reading, Whiteknights, Reading RG6 6AJ, UK.</p><p class="abstract">Accurately and reliably identifying the actual number of clusters present with a dataset of gene expression profiles, when no additional information on cluster structure is available, is a problem addressed by few algorithms. GeneMCL transforms microarray analysis data into a graph consisting of nodes connected by edges, where the nodes represent genes, and the edges represent the similarity in expression of those genes, as given by a proximity measurement. This measurement is taken to be the Pearson correlation coefficient combined with a local non-linear rescaling step. The resulting graph is input to the Markov Cluster (MCL) algorithm, which is an elegant, deterministic, non-specific and scalable method, which models stochastic flow through the graph. The algorithm is inherently affected by any cluster structure present, and rapidly decomposes a graph into cohesive clusters. The potential of the GeneMCL algorithm is demonstrated with a 5,730 gene subset (IGS) of the Van&#39;t Veer breast cancer database, for which the clusterings are shown to reflect underlying biological mechanisms.</p>
<p class="publication"><span class="title">Computational biology and chemistry</span>&nbsp;<span class="year">2005</span>;<span class="volume">29</span>;<span class="number">5</span>;<span class="pagerange">354-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16172020" rel="external">16172020</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.compbiolchem.2005.07.002" rel="external">10.1016/j.compbiolchem.2005.07.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inactivation of Drosophila Apaf-1 related killer suppresses formation of polyglutamine aggregates and blocks polyglutamine pathogenesis.</h4>
<p class="authors"><span class="author">Sang TK</span>, <span class="author">Li C</span>, <span class="author">Liu W</span>, <span class="author">Rodriguez A</span>, <span class="author">Abrams JM</span>, <span class="author">Zipursky SL</span> and <span class="author">Jackson GR</span></p>
<p class="affiliation">Neurogenetics Program, Department of Neurology, Neuropsychiatric Institute, David Geffen School of Medicine at UCLA, Los Angeles, CA 90095, USA.</p><p class="abstract">Huntington&#39;s disease (HD) is caused by expansion of a polyglutamine tract near the N-terminal of huntingtin. Mutant huntingtin forms aggregates in striatum and cortex, where extensive cell death occurs. We used a Drosophila polyglutamine peptide model to assess the role of specific cell death regulators in polyglutamine-induced cell death. Here, we report that polyglutamine-induced cell death was dramatically suppressed in flies lacking Dark, the fly homolog of human Apaf-1, a key regulator of apoptosis. Dark appeared to play a role in the accumulation of polyglutamine-containing aggregates. Suppression of cell death, caspase activation and aggregate formation were also observed when mutant huntingtin exon 1 was expressed in homozygous dark mutant animals. Expanded polyglutamine induced a marked increase in expression of Dark, and Dark was observed to colocalize with ubiquitinated protein aggregates. Apaf-1 also was found to colocalize with huntingtin-containing aggregates in a murine model and HD brain, suggesting a common role for Dark/Apaf-1 in polyglutamine pathogenesis in invertebrates, mice and man. These findings suggest that limiting Apaf-1 activity may alleviate both pathological protein aggregation and neuronal cell death in HD.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">AG012466</span>; NIEHS NIH HHS: <span class="grant">V45 ES012078</span>; NIGMS NIH HHS: <span class="grant">R01 GM072124-14A1</span>; NINDS NIH HHS: <span class="grant">NS002116</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">14</span>;<span class="number">3</span>;<span class="pagerange">357-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15590702" rel="external">15590702</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddi032" rel="external">10.1093/hmg/ddi032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation in the transcriptional coactivator EYA4 causes dilated cardiomyopathy and sensorineural hearing loss.</h4>
<p class="authors"><span class="author">Sch&ouml;nberger J</span>, <span class="author">Wang L</span>, <span class="author">Shin JT</span>, <span class="author">Kim SD</span>, <span class="author">Depreux FF</span>, <span class="author">Zhu H</span>, <span class="author">Zon L</span>, <span class="author">Pizard A</span>, <span class="author">Kim JB</span>, <span class="author">Macrae CA</span>, <span class="author">Mungall AJ</span>, <span class="author">Seidman JG</span> and <span class="author">Seidman CE</span></p>
<p class="affiliation">Harvard Medical School, Department of Genetics, 77 Avenue Louis Pasteur, Boston, Massachusetts 02115, USA.</p><p class="abstract">We identified a human mutation that causes dilated cardiomyopathy and heart failure preceded by sensorineural hearing loss (SNHL). Unlike previously described mutations causing dilated cardiomyopathy that affect structural proteins, this mutation deletes 4,846 bp of the human transcriptional coactivator gene EYA4. To elucidate the roles of eya4 in heart function, we studied zebrafish embryos injected with antisense morpholino oligonucleotides. Attenuated eya4 transcript levels produced morphologic and hemodynamic features of heart failure. To determine why previously described mutated EYA4 alleles cause SNHL without heart disease, we examined biochemical interactions of mutant Eya4 peptides. Eya4 peptides associated with SNHL, but not the shortened 193-amino acid peptide associated with dilated cardiomyopathy and SNHL, bound wild-type Eya4 and associated with Six proteins. These data define unrecognized and crucial roles for Eya4-Six-mediated transcriptional regulation in normal heart function.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">37</span>;<span class="number">4</span>;<span class="pagerange">418-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15735644">15735644</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1527">10.1038/ng1527</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Progressive proximal expansion of the primate X chromosome centromere.</h4>
<p class="authors"><span class="author">Schueler MG</span>, <span class="author">Dunn JM</span>, <span class="author">Bird CP</span>, <span class="author">Ross MT</span>, <span class="author">Viggiano L</span>, <span class="author">NISC Comparative Sequencing Program</span>, <span class="author">Rocchi M</span>, <span class="author">Willard HF</span> and <span class="author">Green ED</span></p>
<p class="affiliation">Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA.</p><p class="abstract">Previous studies of the pericentromeric region of the human X chromosome short arm (Xp) revealed an age gradient from ancient DNA that contains expressed genes to recent human-specific DNA at the functional centromere. We analyzed the finished sequence of this human genomic region to investigate its evolutionary history. Phylogenetic analysis of &gt;1,500 alpha-satellite monomers from the region revealed the presence of five physical domains, each containing monomers from a distinct phylogenetic clade. The most distal domain contains long interspersed nucleotide element repeats that were active &gt;35 million years ago, whereas the four proximal domains contain more recently active long interspersed nucleotide element repeats. An out-of-register, unequal recombination (i.e., crossover) detected at the edge of the X chromosome-specific alpha-satellite array (DXZ1) may reflect the most recent of a series of punctuating events during evolution that resulted in a proximal physical expansion of the X centromere. The first 18 kb of this array has 97-99% pairwise identity among all 2-kb repeat units. To perform more detailed evolutionary comparisons, we sequenced the junction between the ancient DNA of Xp and the primate-specific alpha satellite in chimpanzee, gorilla, orangutan, vervet, macaque, and baboon. The striking conservation found in all cases supports the ancestral nature of the alpha satellite at this location. These studies demonstrate that the primate X centromere appears to have evolved through repeated expansion events occurring within the central, active region of centromeric DNA, with the newly added sequences then conferring centromere function.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2005</span>;<span class="volume">102</span>;<span class="number">30</span>;<span class="pagerange">10563-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16030148" rel="external">16030148</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1180780" rel="external">1180780</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0503346102" rel="external">10.1073/pnas.0503346102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Visualizing profile-profile alignment: pairwise HMM logos.</h4>
<p class="authors"><span class="author">Schuster-B&ouml;ckler B</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%62%73%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#115;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Unlabelled</strong>: The availability of advanced profile-profile comparison tools, such as PRC or HHsearch demands sophisticated visualization tools not presently available. We introduce an approach built upon the concept of HMM logos. The method illustrates the similarities of pairs of protein family profiles in an intuitive way. Two HMM logos, one for each profile, are drawn one upon the other. The aligned states are then highlighted and connected.</p><p class="abstract"><strong>Availability</strong>: A web interface offering online creation of pairwise HMM logos is available at http://www.sanger.ac.uk/Software/analysis/logomat-p. Furthermore, software developers may download a Perl package that includes methods for creation of pairwise HMM logos locally.</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%62%73%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#115;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2912-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15827079">15827079</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti434">10.1093/bioinformatics/bti434</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Zebrafish notochordal basement membrane: signaling and structure.</h4>
<p class="authors"><span class="author">Scott A</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p>
<p class="publication"><span class="title">Current topics in developmental biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">65</span>;<span class="pagerange">229-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15642386" rel="external">15642386</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0070-2153(04)65009-5" rel="external">10.1016/S0070-2153(04)65009-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The V617F JAK2 mutation is uncommon in cancers and in myeloid malignancies other than the classic myeloproliferative disorders.</h4>
<p class="authors"><span class="author">Scott LM</span>, <span class="author">Campbell PJ</span>, <span class="author">Baxter EJ</span>, <span class="author">Todd T</span>, <span class="author">Stephens P</span>, <span class="author">Edkins S</span>, <span class="author">Wooster R</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span> and <span class="author">Green AR</span></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2005</span>;<span class="volume">106</span>;<span class="number">8</span>;<span class="pagerange">2920-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16204151" rel="external">16204151</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2005-05-2087" rel="external">10.1182/blood-2005-05-2087</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Livelihood hazards.</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Thomson NR</span>, <span class="author">Crossman L</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">4</span>;<span class="pagerange">278-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15818892">15818892</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1132">10.1038/nrmicro1132</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Wolbachia variability and host effects on crossing type in Culex mosquitoes.</h4>
<p class="authors"><span class="author">Sinkins SP</span>, <span class="author">Walker T</span>, <span class="author">Lynd AR</span>, <span class="author">Steven AR</span>, <span class="author">Makepeace BL</span>, <span class="author">Godfray HC</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Peter Medawar Building, South Parks Road, Oxford OX1 3SY, UK. <a href="mailto:%73%74%65%76%65%6E.%73%69%6E%6B%69%6E%73%40%7A%6F%6F.%6F%78.%61%63.%75%6B">&#115;&#116;&#101;&#118;&#101;&#110;&#46;&#115;&#105;&#110;&#107;&#105;&#110;&#115;&#64;&#122;&#111;&#111;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Wolbachia is a common maternally inherited bacterial symbiont able to induce crossing sterilities known as cytoplasmic incompatibility (CI) in insects. Wolbachia-modified sperm are unable to complete fertilization of uninfected ova, but a rescue function allows infected eggs to develop normally. By providing a reproductive advantage to infected females, Wolbachia can rapidly invade uninfected populations, and this could provide a mechanism for driving transgenes through pest populations. CI can also occur between Wolbachia-infected populations and is usually associated with the presence of different Wolbachia strains. In the Culex pipiens mosquito group (including the filariasis vector C. quinquefasciatus) a very unusual degree of complexity of Wolbachia-induced crossing-types has been reported, with partial or complete CI that can be unidirectional or bidirectional, yet no Wolbachia strain variation was found. Here we show variation between incompatible Culex strains in two Wolbachia ankyrin repeat-encoding genes associated with a prophage region, one of which is sex-specifically expressed in some strains, and also a direct effect of the host nuclear genome on CI rescue.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">436</span>;<span class="number">7048</span>;<span class="pagerange">257-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16015330">16015330</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03629">10.1038/nature03629</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two ways to trap a gene in mice.</h4>
<p class="authors"><span class="author">Skarnes WC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%73%6B%61%72%6E%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#107;&#97;&#114;&#110;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2005</span>;<span class="volume">102</span>;<span class="number">37</span>;<span class="pagerange">13001-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16150712">16150712</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1201614">1201614</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0506279102">10.1073/pnas.0506279102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MagicMatch--cross-referencing sequence identifiers across databases.</h4>
<p class="authors"><span class="author">Smith M</span>, <span class="author">Kunin V</span>, <span class="author">Goldovsky L</span>, <span class="author">Enright AJ</span> and <span class="author">Ouzounis CA</span></p>
<p class="affiliation">Computational Genomics Group, The European Bioinformatics Institute, EMBL Cambridge Outstation, Cambridge CB10 1SD, UK.</p><p class="abstract">MOTIVATION: At present, mapping of sequence identifiers across databases is a daunting, time-consuming and computationally expensive process, usually achieved by sequence similarity searches with strict threshold values. SUMMARY: We present a rapid and efficient method to map sequence identifiers across databases. The method uses the MD5 checksum algorithm for message integrity to generate sequence fingerprints and uses these fingerprints as hash strings to map sequences across databases. The program, called MagicMatch, is able to cross-link any of the major sequence databases within a few seconds on a modest desktop computer.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">21</span>;<span class="number">16</span>;<span class="pagerange">3429-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15961438" rel="external">15961438</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbti548" rel="external">10.1093/bioinformatics/bti548</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The new cytogenetics: blurring the boundaries with molecular biology.</h4>
<p class="authors"><span class="author">Speicher MR</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Institut f&uuml;r Humangenetik, Technische Universit&auml;t M&uuml;nchen, Germany. <a href="mailto:%73%70%65%69%63%68%65%72%40%68%75%6D%61%6E%67%65%6E%65%74%69%6B.%6D%65%64.%74%75-%6D%75%65%6E%63%68%65%6E.%64%65">&#115;&#112;&#101;&#105;&#99;&#104;&#101;&#114;&#64;&#104;&#117;&#109;&#97;&#110;&#103;&#101;&#110;&#101;&#116;&#105;&#107;&#46;&#109;&#101;&#100;&#46;&#116;&#117;&#45;&#109;&#117;&#101;&#110;&#99;&#104;&#101;&#110;&#46;&#100;&#101;</a></p><p class="abstract">Exciting advances in fluorescence in situ hybridization and array-based techniques are changing the nature of cytogenetics, in both basic research and molecular diagnostics. Cytogenetic analysis now extends beyond the simple description of the chromosomal status of a genome and allows the study of fundamental biological questions, such as the nature of inherited syndromes, the genomic changes that are involved in tumorigenesis and the three-dimensional organization of the human genome. The high resolution that is achieved by these techniques, particularly by microarray technologies such as array comparative genomic hybridization, is blurring the traditional distinction between cytogenetics and molecular biology.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">782-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16145555">16145555</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1692">10.1038/nrg1692</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of pathogen-specific genes through microarray analysis of pathogenic and commensal Neisseria species.</h4>
<p class="authors"><span class="author">Stabler RA</span>, <span class="author">Marsden GL</span>, <span class="author">Witney AA</span>, <span class="author">Li Y</span>, <span class="author">Bentley SD</span>, <span class="author">Tang CM</span> and <span class="author">Hinds J</span></p>
<p class="affiliation">Bacterial Microarray Group, St George&#39;s Hospital Medical School, London SW7 0RE, UK. <a href="mailto:%52%69%63%68%61%72%64.%53%74%61%62%6C%65%72%40%6C%73%68%74%6D.%61%63.%75%6B">&#82;&#105;&#99;&#104;&#97;&#114;&#100;&#46;&#83;&#116;&#97;&#98;&#108;&#101;&#114;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The release of the complete genome sequences of Neisseria meningitidis MC58 and Z2491 along with access to the sequences of N. meningitidis FAM18 and Neisseria gonorrhoeae FA1090 allowed the construction of a pan-Neisseria microarray, with every gene in all four genomes represented. The microarray was used to analyse a selection of strains including all N. meningitidis serogroups and commensal Neisseria species. For each strain, genes were defined as present, divergent or absent using gack analysis software. Comparison of the strains identified genes that were conserved within N. meningitidis serogroup B strains but absent from all commensal strains tested, consisting of mainly virulence-associated genes and transmissible elements. The microarray was able to distinguish between pilin genes, pilC orthologues and serogroup-specific capsule biosynthetic genes, and to identify dam and drg genotypes. Previously described N. meningitidis genes involved in iron response, adherence to epithelial cells, and pathogenicity were compared to the microarray analysis. The microarray data correlated with other genetic typing methods and were able to predict genotypes for uncharacterized strains and thus offer the potential for a rapid typing method. The subset of pathogen-specific genes identified represents potential drug or vaccine targets that would not eliminate commensal neisseriae and the associated naturally acquired immunity.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">151</span>;<span class="number">Pt 9</span>;<span class="pagerange">2907-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16151203" rel="external">16151203</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.28099-0" rel="external">10.1099/mic.0.28099-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structure and function of the notochord: an essential organ for chordate development.</h4>
<p class="authors"><span class="author">Stemple DL</span></p>
<p class="affiliation">Vertebrate Development and Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%73%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The notochord is the defining structure of the chordates, and has essential roles in vertebrate development. It serves as a source of midline signals that pattern surrounding tissues and as a major skeletal element of the developing embryo. Genetic and embryological studies over the past decade have informed us about the development and function of the notochord. In this review, I discuss the embryonic origin, signalling roles and ultimate fate of the notochord, with an emphasis on structural aspects of notochord biology.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2005</span>;<span class="volume">132</span>;<span class="number">11</span>;<span class="pagerange">2503-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15890825">15890825</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.01812">10.1242/dev.01812</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A screen of the complete protein kinase gene family identifies diverse patterns of somatic mutations in human breast cancer.</h4>
<p class="authors"><span class="author">Stephens P</span>, <span class="author">Edkins S</span>, <span class="author">Davies H</span>, <span class="author">Greenman C</span>, <span class="author">Cox C</span>, <span class="author">Hunter C</span>, <span class="author">Bignell G</span>, <span class="author">Teague J</span>, <span class="author">Smith R</span>, <span class="author">Stevens C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Parker A</span>, <span class="author">Tarpey P</span>, <span class="author">Avis T</span>, <span class="author">Barthorpe A</span>, <span class="author">Brackenbury L</span>, <span class="author">Buck G</span>, <span class="author">Butler A</span>, <span class="author">Clements J</span>, <span class="author">Cole J</span>, <span class="author">Dicks E</span>, <span class="author">Edwards K</span>, <span class="author">Forbes S</span>, <span class="author">Gorton M</span>, <span class="author">Gray K</span>, <span class="author">Halliday K</span>, <span class="author">Harrison R</span>, <span class="author">Hills K</span>, <span class="author">Hinton J</span>, <span class="author">Jones D</span>, <span class="author">Kosmidou V</span>, <span class="author">Laman R</span>, <span class="author">Lugg R</span>, <span class="author">Menzies A</span>, <span class="author">Perry J</span>, <span class="author">Petty R</span>, <span class="author">Raine K</span>, <span class="author">Shepherd R</span>, <span class="author">Small A</span>, <span class="author">Solomon H</span>, <span class="author">Stephens Y</span>, <span class="author">Tofts C</span>, <span class="author">Varian J</span>, <span class="author">Webb A</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Yates A</span>, <span class="author">Brasseur F</span>, <span class="author">Cooper CS</span>, <span class="author">Flanagan AM</span>, <span class="author">Green A</span>, <span class="author">Knowles M</span>, <span class="author">Leung SY</span>, <span class="author">Looijenga LH</span>, <span class="author">Malkowicz B</span>, <span class="author">Pierotti MA</span>, <span class="author">Teh B</span>, <span class="author">Yuen ST</span>, <span class="author">Nicholson AG</span>, <span class="author">Lakhani S</span>, <span class="author">Easton DF</span>, <span class="author">Weber BL</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span> and <span class="author">Wooster R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">We examined the coding sequence of 518 protein kinases, approximately 1.3 Mb of DNA per sample, in 25 breast cancers. In many tumors, we detected no somatic mutations. But a few had numerous somatic mutations with distinctive patterns indicative of either a mutator phenotype or a past exposure.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">37</span>;<span class="number">6</span>;<span class="pagerange">590-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15908952">15908952</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1571">10.1038/ng1571</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effect of deletion or overexpression of the 19-kilodalton lipoprotein Rv3763 on the innate response to Mycobacterium tuberculosis.</h4>
<p class="authors"><span class="author">Stewart GR</span>, <span class="author">Wilkinson KA</span>, <span class="author">Newton SM</span>, <span class="author">Sullivan SM</span>, <span class="author">Neyrolles O</span>, <span class="author">Wain JR</span>, <span class="author">Patel J</span>, <span class="author">Pool KL</span>, <span class="author">Young DB</span> and <span class="author">Wilkinson RJ</span></p>
<p class="affiliation">Center for Molecular Microbiology and Infection and Wellcome Trust Center for Research in Clinical Tropical Medicine, Imperial College London, UK.</p><p class="abstract">The 19-kDa lipoprotein of Mycobacterium tuberculosis is an important target of the innate immune response. To investigate the immune biology of this antigen in the context of the whole bacillus, we derived a recombinant M. tuberculosis H37Rv that lacked the 19-kDa-lipoprotein gene (Delta19) and complemented this strain by reintroduction of the 19-kDa-lipoprotein gene on a multicopy vector to produce Delta19::pSMT181. The Delta19 strain multiplied less well than Delta19::pSMT181 in human monocyte-derived macrophages (MDM) (P = 0.039). Surface expression of major histocompatibility complex class II molecules was reduced in phagocytes infected with M. tuberculosis; this effect was not seen in cells infected with Delta19. Delta19 induced lower interleukin 1beta (IL-1beta) secretion from monocytes and MDM. Overexpression of the 19-kDa protein increased IL-1beta, IL-12p40, and tumor necrosis factor alpha secretion irrespective of phagocyte maturity. These data support reports that the 19-kDa lipoprotein has pleiotropic effects on the interaction of M. tuberculosis with phagocytes. However, this analysis indicates that in the context of the whole bacillus, the 19-kDa lipoprotein is only one of a number of molecules that mediate the innate response to M. tuberculosis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">072070</span></p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2005</span>;<span class="volume">73</span>;<span class="number">10</span>;<span class="pagerange">6831-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16177361" rel="external">16177361</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1230982" rel="external">1230982</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.73.10.6831-6837.2005" rel="external">10.1128/IAI.73.10.6831-6837.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genetics of regulatory variation in the human genome.</h4>
<p class="authors"><span class="author">Stranger BE</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The regulation of gene expression plays an important role in complex phenotypes, including disease in humans. For some genes, the genetic mechanisms influencing gene expression are well elucidated; however, it is unclear how applicable these results are to gene expression on a genome-wide level. Studies in model organisms and humans have clearly documented gene expression variation among individuals and shown that a significant proportion of this variation has a genetic basis. Recent studies combine microarray surveys of gene expression for thousands of genes with dense marker maps, and are beginning to identify regions in the human genome that have functional effects on gene expression. This paper reviews recent developments and methodologies in this field, and discusses implications and future directions of this research in the context of understanding the influence of human genomic variation on the regulation of gene expression.</p>
<p class="publication"><span class="title">Human genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">2</span>;<span class="number">2</span>;<span class="pagerange">126-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16004727" rel="external">16004727</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nucleotide variation at the myrosinase-encoding locus, TGG1, and quantitative myrosinase enzyme activity variation in Arabidopsis thaliana.</h4>
<p class="authors"><span class="author">Stranger BE</span> and <span class="author">Mitchell-Olds T</span></p>
<p class="affiliation">Department of Genetics and Evolution, Max Planck Institute of Chemical Ecology, Jena, Germany. <a href="mailto:%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Arabidopsis thaliana TGG1 gene encodes thioglucoside glucohydrolase (myrosinase), an enzyme catalysing the hydrolysis of glucosinolate compounds. The enzyme is involved in plant defence against some insect herbivores, and is present in species of the order Capparales (Brassicales). Nucleotide variation was surveyed by sequencing c. 2.4 kb of the TGG1 locus in a sample of 28 worldwide A. thaliana accessions, and one Arabidopsis lyrata ssp. lyrata individual. Myrosinase activity was quantified for 27 of these same A. thaliana accessions, plus five additional others. Overall, estimated nucleotide diversity in A. thaliana was low compared to other published A. thaliana surveys, and the frequency distribution was skewed toward an excess of low-frequency variants. Furthermore, comparison to the outgroup species A. lyrata demonstrated that A. thaliana exhibited an excess of high-frequency derived variants relative to a neutral equilibrium model, suggesting a selective sweep. A. thaliana accessions differed significantly in total myrosinase activity, but analysis of variance detected no statistical evidence for an association between quantitative enzyme activity and alleles at the TGG1 myrosinase-encoding locus. We thus conclude that other, unsurveyed factors primarily affect the observed myrosinase activity levels in this species. The pattern of nucleotide variation was consistent with a model of positive selection but might also be compatible with a completely neutral model that takes into account the metapopulation behaviour of this highly inbreeding species which experienced a relatively recent worldwide expansion.</p>
<p class="publication"><span class="title">Molecular ecology</span>&nbsp;<span class="year">2005</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">295-309</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15643972" rel="external">15643972</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-294X.2004.02403.x" rel="external">10.1111/j.1365-294X.2004.02403.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide associations of gene expression variation in humans.</h4>
<p class="authors"><span class="author">Stranger BE</span>, <span class="author">Forrest MS</span>, <span class="author">Clark AG</span>, <span class="author">Minichiello MJ</span>, <span class="author">Deutsch S</span>, <span class="author">Lyle R</span>, <span class="author">Hunt S</span>, <span class="author">Kahl B</span>, <span class="author">Antonarakis SE</span>, <span class="author">Tavar&eacute; S</span>, <span class="author">Deloukas P</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">The exploration of quantitative variation in human populations has become one of the major priorities for medical genetics. The successful identification of variants that contribute to complex traits is highly dependent on reliable assays and genetic maps. We have performed a genome-wide quantitative trait analysis of 630 genes in 60 unrelated Utah residents with ancestry from Northern and Western Europe using the publicly available phase I data of the International HapMap project. The genes are located in regions of the human genome with elevated functional annotation and disease interest including the ENCODE regions spanning 1% of the genome, Chromosome 21 and Chromosome 20q12-13.2. We apply three different methods of multiple test correction, including Bonferroni, false discovery rate, and permutations. For the 374 expressed genes, we find many regions with statistically significant association of single nucleotide polymorphisms (SNPs) with expression variation in lymphoblastoid cell lines after correcting for multiple tests. Based on our analyses, the signal proximal (cis-) to the genes of interest is more abundant and more stable than distal and trans across statistical methodologies. Our results suggest that regulatory polymorphism is widespread in the human genome and show that the 5-kb (phase I) HapMap has sufficient density to enable linkage disequilibrium mapping in humans. Such studies will significantly enhance our ability to annotate the non-coding part of the genome and interpret functional variation. In addition, we demonstrate that the HapMap cell lines themselves may serve as a useful resource for quantitative measurements at the cellular level.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG02790</span>, <span class="grant">HG03229</span>; NIGMS NIH HHS: <span class="grant">GM065509</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">1</span>;<span class="number">6</span>;<span class="pagerange">e78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16362079">16362079</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1315281">1315281</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0010078">10.1371/journal.pgen.0010078</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Role of morphogens in brain growth.</h4>
<p class="authors"><span class="author">Tannahill D</span>, <span class="author">Harris LW</span> and <span class="author">Keynes R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">During the last century, experiments on the chick embryo established that the ballooning expansion of the early forebrain and midbrain vesicles is dependent on the underlying axial (notochordal) mesoderm. Transient separation of the early midbrain primordium from the notochord causes subsequent collapse of both midbrain and forebrain (telencephalic) vesicles, accompanied by pronounced folding of the neural epithelium. More recent experiments have shown that vesicle collapse is caused by defective Sonic Hedgehog (Shh) signaling from the notochord and floor plate. Separation of the notochord from the brain causes loss of ventral Shh expression, resulting in reduced cell proliferation and increased cell death in the expanding neural epithelium, and culminating in vesicle collapse. These experiments are reviewed here, and set in the context of other studies illustrating the wide range of molecular and cellular processes that cause abnormal brain morphogenesis when perturbed. We also speculate that variation in the regulation of signaling pathways such as Hedgehog may have played a significant part in generating rapid morphogenetic changes during the evolution of the vertebrate brain.</p>
<p class="publication"><span class="title">Journal of neurobiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">64</span>;<span class="number">4</span>;<span class="pagerange">367-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16041753" rel="external">16041753</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fneu.20163" rel="external">10.1002/neu.20163</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction of a 2-Mb resolution BAC microarray for CGH analysis of canine tumors.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Scott A</span>, <span class="author">Langford CF</span>, <span class="author">Fosmire SP</span>, <span class="author">Jubala CM</span>, <span class="author">Lorentzen TD</span>, <span class="author">Hitte C</span>, <span class="author">Karlsson EK</span>, <span class="author">Kirkness E</span>, <span class="author">Ostrander EA</span>, <span class="author">Galibert F</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Modiano JF</span> and <span class="author">Breen M</span></p>
<p class="affiliation">Department of Molecular Biomedical Sciences, College of Veterinary Medicine, North Carolina State University, Raleigh, North Carolina 27606, USA.</p><p class="abstract">Recognition of the domestic dog as a model for the comparative study of human genetic traits has led to major advances in canine genomics. The pathophysiological similarities shared between many human and dog diseases extend to a range of cancers. Human tumors frequently display recurrent chromosome aberrations, many of which are hallmarks of particular tumor subtypes. Using a range of molecular cytogenetic techniques we have generated evidence indicating that this is also true of canine tumors. Detailed knowledge of these genomic abnormalities has the potential to aid diagnosis, prognosis, and the selection of appropriate therapy in both species. We recently improved the efficiency and resolution of canine cancer cytogenetics studies by developing a small-scale genomic microarray comprising a panel of canine BAC clones representing subgenomic regions of particular interest. We have now extended these studies to generate a comprehensive canine comparative genomic hybridization (CGH) array that comprises 1158 canine BAC clones ordered throughout the genome with an average interval of 2 Mb. Most of the clones (84.3%) have been assigned to a precise cytogenetic location by fluorescence in situ hybridization (FISH), and 98.5% are also directly anchored within the current canine genome assembly, permitting direct translation from cytogenetic aberration to DNA sequence. We are now using this resource routinely for high-throughput array CGH and single-locus probe analysis of a range of canine cancers. Here we provide examples of the varied applications of this resource to tumor cytogenetics, in combination with other molecular cytogenetic techniques.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">12</span>;<span class="pagerange">1831-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16339382" rel="external">16339382</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1356122" rel="external">1356122</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.3825705" rel="external">10.1101/gr.3825705</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Massive attack.</h4>
<p class="authors"><span class="author">Thomson N</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Bentley S</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">8</span>;<span class="pagerange">586-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16094740">16094740</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1212">10.1038/nrmicro1212</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Brothers in arms.</h4>
<p class="authors"><span class="author">Thomson N</span>, <span class="author">Holden M</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">3</span>;<span class="number">2</span>;<span class="pagerange">100-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15693175">15693175</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1092">10.1038/nrmicro1092</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Chlamydophila abortus genome sequence reveals an array of variable proteins that contribute to interspecies variation.</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Yeats C</span>, <span class="author">Bell K</span>, <span class="author">Holden MT</span>, <span class="author">Bentley SD</span>, <span class="author">Livingstone M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Harris B</span>, <span class="author">Doggett J</span>, <span class="author">Ormond D</span>, <span class="author">Mungall K</span>, <span class="author">Clarke K</span>, <span class="author">Feltwell T</span>, <span class="author">Hance Z</span>, <span class="author">Sanders M</span>, <span class="author">Quail MA</span>, <span class="author">Price C</span>, <span class="author">Barrell BG</span>, <span class="author">Parkhill J</span> and <span class="author">Longbottom D</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6E%72%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#114;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The obligate intracellular bacterial pathogen Chlamydophila abortus strain S26/3 (formerly the abortion subtype of Chlamydia psittaci) is an important cause of late gestation abortions in ruminants and pigs. Furthermore, although relatively rare, zoonotic infection can result in acute illness and miscarriage in pregnant women. The complete genome sequence was determined and shows a high level of conservation in both sequence and overall gene content in comparison to other Chlamydiaceae. The 1,144,377-bp genome contains 961 predicted coding sequences, 842 of which are conserved with those of Chlamydophila caviae and Chlamydophila pneumoniae. Within this conserved Cp. abortus core genome we have identified the major regions of variation and have focused our analysis on these loci, several of which were found to encode highly variable protein families, such as TMH/Inc and Pmp families, which are strong candidates for the source of diversity in host tropism and disease causation in this group of organisms. Significantly, Cp. abortus lacks any toxin genes, and also lacks genes involved in tryptophan metabolism and nucleotide salvaging (guaB is present as a pseudogene), suggesting that the genetic basis of niche adaptation of this species is distinct from those previously proposed for other chlamydial species.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">5</span>;<span class="pagerange">629-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15837807">15837807</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1088291">1088291</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.3684805">10.1101/gr.3684805</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamic distribution of endoplasmic reticulum in hippocampal neuron dendritic spines.</h4>
<p class="authors"><span class="author">Toresson H</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Division of Neuroscience, College of Medicine and Veterinary Medicine, University of Edinburgh, 1 George Square, Edinburgh, UK. <a href="mailto:%68%61%6B%61%6E.%74%6F%72%65%73%73%6F%6E%40%6D%65%64.%6C%75.%73%65">&#104;&#97;&#107;&#97;&#110;&#46;&#116;&#111;&#114;&#101;&#115;&#115;&#111;&#110;&#64;&#109;&#101;&#100;&#46;&#108;&#117;&#46;&#115;&#101;</a></p><p class="abstract">The role of the endoplasmic reticulum (ER) localized in dendritic spines has become a subject of intense interest because of its potential functions in local protein synthesis and signal transduction. Although it is recognized from electron microscopic studies that not all spines contain ER, little is know of its dynamic regulation or turnover. Here, we report a surprising degree of turnover of ER within spines. Using confocal microscopy imaging we observed continuity of spine-ER with dendritic ER in hippocampal primary neurons. Over 24 h, less than 50% of spine ER was stable. Despite this high degree of turn over, we identified a significant subset of spines that maintained ER for at least 4 days. These results indicate that within a single neuron, the organelle composition of a spine is unexpectedly dynamic and may explain aspects of the spine-to-spine variation in calcium spike magnitude and localized protein synthesis and trafficking.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The European journal of neuroscience</span>&nbsp;<span class="year">2005</span>;<span class="volume">22</span>;<span class="number">7</span>;<span class="pagerange">1793-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16197520" rel="external">16197520</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1460-9568.2005.04342.x" rel="external">10.1111/j.1460-9568.2005.04342.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mechanisms of Hedgehog gradient formation and interpretation.</h4>
<p class="authors"><span class="author">Torroja C</span>, <span class="author">Gorfinkiel N</span> and <span class="author">Guerrero I</span></p>
<p class="affiliation">Centro de Biolog&iacute;a Molecular Severo Ochoa, CSIC, Universidad Aut&oacute;noma de Madrid, Cantoblanco, E-28049 Madrid, Spain.</p><p class="abstract">Morphogens are molecules that spread from localized sites of production, specifying distinct cell outcomes at different concentrations. Members of the Hedgehog (Hh) family of signaling molecules act as morphogens in different developmental systems. If we are to understand how Hh elicits multiple responses in a temporally and spatially specific manner, the molecular mechanism of Hh gradient formation needs to be established. Moreover, understanding the mechanisms of Hh signaling is a central issue in biology, not only because of the role of Hh in morphogenesis, but also because of its involvement in a wide range of human diseases. Here, we review the mechanisms affecting the dynamics of Hh gradient formation, mostly in the context of Drosophila wing development, although parallel findings in vertebrate systems are also discussed.</p>
<p class="publication"><span class="title">Journal of neurobiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">64</span>;<span class="number">4</span>;<span class="pagerange">334-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16041759" rel="external">16041759</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fneu.20168" rel="external">10.1002/neu.20168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Abridged 5S rDNA units in sea beet (Beta vulgaris subsp. maritima).</h4>
<p class="authors"><span class="author">Turner DJ</span> and <span class="author">Brown TA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Amplification by polymerase chain reaction of the 5S rDNA repeat units of Beta vulgaris subsp. maritima resulted in a 350-bp product corresponding to the full-length 5S unit, but also revealed 4 abridged unit classes, each with a deletion that removed most of the spacer and 12-76 bp of the coding sequence. Each abridged type lacks at least 1 of the conserved elements involved in transcription of the 5S gene, and so appear to be nonfunctional. Network analysis revealed that the abridged units are evolving in the same manner as the full-length versions.</p>
<p class="publication"><span class="title">Genome / National Research Council Canada = G&eacute;nome / Conseil national de recherches Canada</span>&nbsp;<span class="year">2005</span>;<span class="volume">48</span>;<span class="number">2</span>;<span class="pagerange">352-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15838559" rel="external">15838559</a>; DOI: <a href="http://dx.doi.org/10.1139%2Fg04-107" rel="external">10.1139/g04-107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Don&#39;t mix radiocarbon and calendar years.</h4>
<p class="authors"><span class="author">Tyler-Smith C</span>, <span class="author">Hurles ME</span> and <span class="author">Jobling MA</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2005</span>;<span class="volume">434</span>;<span class="number">7034</span>;<span class="pagerange">697</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15815601">15815601</a>; DOI: <a href="http://dx.doi.org/10.1038%2F434697c">10.1038/434697c</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Segmental trisomy of chromosome 17: a mouse model of human aneuploidy syndromes.</h4>
<p class="authors"><span class="author">Vac&iacute;k T</span>, <span class="author">Ort M</span>, <span class="author">Gregorov&aacute; S</span>, <span class="author">Strnad P</span>, <span class="author">Blatny R</span>, <span class="author">Conte N</span>, <span class="author">Bradley A</span>, <span class="author">Bures J</span> and <span class="author">Forejt J</span></p>
<p class="affiliation">Institutes of Molecular Genetics and Physiology, Academy of Sciences of the Czech Republic, 14220 Prague, Czech Republic.</p><p class="abstract">Triplication of whole autosomes or large autosomal segments is detrimental to the development of a mammalian embryo. The trisomy of human chromosome (Chr) 21, known as Down&#39;s syndrome, is regularly associated with mental retardation and a variable set of other developmental anomalies. Several mouse models of Down&#39;s syndrome, triplicating 33-104 genes of Chr16, were designed in an attempt to analyze the contribution of specific orthologous genes to particular developmental features. However, a recent study challenged the concept of dosage-sensitive genes as a primary cause of an abnormal phenotype. To distinguish between the specific effects of dosage-sensitive genes and nonspecific effects of a large number of arbitrary genes, we revisited the mouse Ts43H/Ph segmental trisomy. It encompasses &gt;310 known genes triplicated within the proximal 30 megabases (Mb) of Chr17. We refined the distal border of the trisomic segment to the interval bounded by bacterial artificial chromosomes RP23-277B13 (location 29.0 Mb) and Cbs gene (location 30.2 Mb). The Ts43H mice, viable on a mixed genetic background, exhibited spatial learning deficits analogous to those observed in Ts65Dn mice with unrelated trisomy. Quantitative analysis of the brain expression of 20 genes inside the trisomic interval and 12 genes lying outside on Chr17 revealed 1.2-fold average increase of mRNA steady-state levels of triplicated genes and 0.9-fold average down-regulation of genes beyond the border of trisomy. We propose that systemic comparisons of unrelated segmental trisomies, such as Ts65Dn and Ts43H, will elucidate the pathways leading from the triplicated sequences to the complex developmental traits.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2005</span>;<span class="volume">102</span>;<span class="number">12</span>;<span class="pagerange">4500-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15755806" rel="external">15755806</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC552979" rel="external">552979</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0500802102" rel="external">10.1073/pnas.0500802102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Null and conditional semaphorin 3B alleles using a flexible puroDeltatk loxP/FRT vector.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Adams DJ</span>, <span class="author">Harris LW</span>, <span class="author">Tannahill D</span>, <span class="author">Arends MJ</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">In neural development, Semaphorin 3B (SEMA3B) is thought to play a role in guiding axons by repulsion. In nonneuronal tissue, SEMA3B has been postulated to be a tumor suppressor gene of lung and breast cancer. Much of the understanding of the function of members of the SEMA3 family has come from targeted deletion of these genes in mice (Sema3A, Sema3C, and Sema3F). Thus, targeted deletion of Sema3B in mice would prove invaluable in dissecting out its functions. To allow for maximum gene-targeting flexibility, we developed a generic targeting vector, pFlexible, containing the positive/negative selectable marker puroDeltatk and loxP and FRT recombination sites, and used it to target Sema3B in ES cells. Flpe- and Cre-mediated recombination in vitro generated ES cell lines that contained a conditional or null Sema3B allele, respectively, which were established as homozygous alleles in mice. Analysis of Sema3B null mice showed they were viable, fertile, and displayed no overt pathological abnormalities, suggesting an inherent correction mechanism or level of redundancy between the class 3 semaphorins. This targeting vector system has broad applicability in any knockout experiment and provides a flexible approach for the generation of modified alleles in mice.</p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2005</span>;<span class="volume">41</span>;<span class="number">4</span>;<span class="pagerange">171-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15789413" rel="external">15789413</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgene.20111" rel="external">10.1002/gene.20111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The RASSF1A isoform of RASSF1 promotes microtubule stability and suppresses tumorigenesis.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Tachibana KK</span>, <span class="author">Gonzalez MA</span>, <span class="author">Adams DJ</span>, <span class="author">Ng BL</span>, <span class="author">Petty R</span>, <span class="author">Venkitaraman AR</span>, <span class="author">Arends MJ</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The RASSF1A isoform of RASSF1 is frequently inactivated by epigenetic alterations in human cancers, but it remains unclear if and how it acts as a tumor suppressor. RASSF1A overexpression reduces in vitro colony formation and the tumorigenicity of cancer cell lines in vivo. Conversely, RASSF1A knockdown causes multiple mitotic defects that may promote genomic instability. Here, we have used a genetic approach to address the function of RASSF1A as a tumor suppressor in vivo by targeted deletion of Rassf1A in the mouse. Rassf1A null mice were viable and fertile and displayed no pathological abnormalities. Rassf1A null embryonic fibroblasts displayed an increased sensitivity to microtubule depolymerizing agents. No overtly altered cell cycle parameters or aberrations in centrosome number were detected in Rassf1A null fibroblasts. Rassf1A null fibroblasts did not show increased sensitivity to microtubule poisons or DNA-damaging agents and showed no evidence of gross genomic instability, suggesting that cellular responses to genotoxins were unaffected. Rassf1A null mice showed an increased incidence of spontaneous tumorigenesis and decreased survival rate compared with wild-type mice. Irradiated Rassf1A null mice also showed increased tumor susceptibility, particularly to tumors associated with the gastrointestinal tract, compared with wild-type mice. Thus, our results demonstrate that Rassf1A acts as a tumor suppressor gene.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">18</span>;<span class="pagerange">8356-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16135822" rel="external">16135822</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1234312" rel="external">1234312</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.25.18.8356-8367.2005" rel="external">10.1128/MCB.25.18.8356-8367.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common VKORC1 and GGCX polymorphisms associated with warfarin dose.</h4>
<p class="authors"><span class="author">Wadelius M</span>, <span class="author">Chen LY</span>, <span class="author">Downes K</span>, <span class="author">Ghori J</span>, <span class="author">Hunt S</span>, <span class="author">Eriksson N</span>, <span class="author">Wallerman O</span>, <span class="author">Melhus H</span>, <span class="author">Wadelius C</span>, <span class="author">Bentley D</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Department of Medical Sciences, Clinical Pharmacology, University Hospital, Uppsala, Sweden. <a href="mailto:%6D%69%61.%77%61%64%65%6C%69%75%73%40%6D%65%64%73%63%69.%75%75.%73%65">&#109;&#105;&#97;&#46;&#119;&#97;&#100;&#101;&#108;&#105;&#117;&#115;&#64;&#109;&#101;&#100;&#115;&#99;&#105;&#46;&#117;&#117;&#46;&#115;&#101;</a></p><p class="abstract">We report a novel combination of factors that explains almost 60% of variable response to warfarin. Warfarin is a widely used anticoagulant, which acts through interference with vitamin K epoxide reductase that is encoded by VKORC1. In the next step of the vitamin K cycle, gamma-glutamyl carboxylase encoded by GGCX uses reduced vitamin K to activate clotting factors. We genotyped 201 warfarin-treated patients for common polymorphisms in VKORC1 and GGCX. All the five VKORC1 single-nucleotide polymorphisms covary significantly with warfarin dose, and explain 29-30% of variance in dose. Thus, VKORC1 has a larger impact than cytochrome P450 2C9, which explains 12% of variance in dose. In addition, one GGCX SNP showed a small but significant effect on warfarin dose. Incorrect dosage, especially during the initial phase of treatment, carries a high risk of either severe bleeding or failure to prevent thromboembolism. Genotype-based dose predictions may in future enable personalised drug treatment from the start of warfarin therapy.</p>
<p class="publication"><span class="title">The pharmacogenomics journal</span>&nbsp;<span class="year">2005</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">262-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15883587" rel="external">15883587</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.tpj.6500313" rel="external">10.1038/sj.tpj.6500313</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Vi antigen expression in Salmonella enterica serovar Typhi clinical isolates from Pakistan.</h4>
<p class="authors"><span class="author">Wain J</span>, <span class="author">House D</span>, <span class="author">Zafar A</span>, <span class="author">Baker S</span>, <span class="author">Nair S</span>, <span class="author">Kidgell C</span>, <span class="author">Bhutta Z</span>, <span class="author">Dougan G</span> and <span class="author">Hasan R</span></p>
<p class="affiliation">Investigative Sciences, Centre for Molecular Microbiology and Infection, Imperial College, London, United Kingdom. <a href="mailto:%6A%77%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#119;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The accurate identification of Salmonella enterica subsp. enterica serovar Typhi variants that fail to express the capsular polysaccharide, Vi, is an important and much discussed issue for medical microbiology. We have tested a multiplex PCR method which shows the presence or absence of the genetic locus required for Vi expression. Of 2,222 Salmonella serovar Typhi clinical isolates collected from patients&#39; blood over a 4-year period in a region of Pakistan where typhoid is endemic, 12 tested negative for Vi expression by serological agglutination. However, only 1 of these 12 was Vi negative by the multiplex PCR method. This result was confirmed by immunofluorescence, the most sensitive method for Vi characterization in Salmonella serovar Typhi. The multiplex PCR described therefore represents a simple and accurate method for surveillance for Vi-negative variants of Salmonella serovar Typhi in Pakistan. Testing of clinical isolates of Salmonella serovar Typhi, before subculture, from other regions where Vi-negative Salmonella serovar Typhi has been described should be carried out so that the impact of vaccination with purified Vi antigen on the levels of Vi-negative Salmonella serovar Typhi in bacterial populations can be assessed.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">43</span>;<span class="number">3</span>;<span class="pagerange">1158-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15750077" rel="external">15750077</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1081282" rel="external">1081282</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.43.3.1158-1165.2005" rel="external">10.1128/JCM.43.3.1158-1165.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chicken TAP genes differ from their human orthologues in locus organisation, size, sequence features and polymorphism.</h4>
<p class="authors"><span class="author">Walker BA</span>, <span class="author">van Hateren A</span>, <span class="author">Milne S</span>, <span class="author">Beck S</span> and <span class="author">Kaufman J</span></p>
<p class="affiliation">Institute For Animal Health, Compton, Nr Newbury, Berkshire, RG20 7NN, UK.</p><p class="abstract">We have previously shown that in the chicken major histocompatibility complex, the two transporters associated with antigen processing genes (TAP1 and TAP2) are located head to head between two classical class I genes. Here we show that the region between these two TAP genes has transcription factor-binding sites in common with class I gene promoters. The TAP genes are also up-regulated by interferon-gamma in a similar way to mammalian TAP genes and in a way that suggests they are both transcribed from a bi-directional promoter. The gene structures of TAP1 and TAP2 differ from that of human TAPs in that TAP1 has a truncated exon 1 and TAP2 has fused exons, resulting in a much smaller gene size. The truncation of TAP1 results in the loss of approximately 150 amino acids, which are thought to be involved in endoplasmic reticulum retention, heterodimer formation and tapasin binding, compared to human TAP1. Most of the protein sequence features involved in binding ATP are conserved, with two exceptions: chicken TAP1 has a glycine in the switch region where other TAPs have glutamine or histidine, and both chicken TAP genes have serines in the C motif where mammalian TAP2 has an alanine. Lastly, the chicken TAP genes are highly polymorphic, with at least as many TAP alleles as there are class I alleles, as seen by investigating nine inbred lines of chicken. The close proximity of the TAP genes to the class I genes and the high level of polymorphism may allow co-evolution of the genes, allowing TAP molecules to transport peptides specifically for the class I molecules of that haplotype.</p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">57</span>;<span class="number">3-4</span>;<span class="pagerange">232-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15900495" rel="external">15900495</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00251-005-0786-2" rel="external">10.1007/s00251-005-0786-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hirudo medicinalis: a platform for investigating genes in neural repair.</h4>
<p class="authors"><span class="author">Wang WZ</span>, <span class="author">Emes RD</span>, <span class="author">Christoffers K</span>, <span class="author">Verrall J</span> and <span class="author">Blackshaw SE</span></p>
<p class="affiliation">Department of Human Anatomy &amp; Genetics, South Parks Road, University of Oxford OX1 3QX, UK.</p><p class="abstract">We have used the nervous system of the medicinal leech as a preparation to study the molecular basis of neural repair. The leech central nervous system, unlike mammalian CNS, can regenerate to restore function, and contains identified nerve cells of known function and connectivity. We have constructed subtractive cDNA probes from whole and regenerating ganglia of the ventral nerve cord and have used these to screen a serotonergic Retzius neuron library. This identifies genes that are regulated as a result of axotomy, and are expressed by the Retzius cell. This approach identifies many genes, both novel and known. Many of the known genes identified have homologues in vertebrates, including man. For example, genes encoding thioredoxin (TRX), Rough Endoplasmic Reticulum Protein 1 (RER-1) and ATP synthase are upregulated at 24 h postinjury in leech nerve cord. To investigate the functional role of regulated genes in neuron regrowth we are using microinjection of antisense oligonucleotides in combination with horseradish peroxidase to knock down expression of a chosen gene and to assess regeneration in single neurons in 3-D ganglion culture. As an example of this approach we describe experiments to microinject antisense oligonucleotide to a leech isoform of the structural protein, Protein 4.1. Our approach thus identifies genes regulated at different times after injury that may underpin the intrinsic ability of leech neurons to survive damage, to initiate regrowth programs and to remake functional connections. It enables us to determine the time course of gene expression in the regenerating nerve cord, and to study the effects of gene knockdown in identified neurons regenerating in defined conditions in culture.</p>
<p class="publication"><span class="title">Cellular and molecular neurobiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">25</span>;<span class="number">2</span>;<span class="pagerange">427-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16047550" rel="external">16047550</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Autosomal location of genes from the conserved mammalian X in the platypus (Ornithorhynchus anatinus): implications for mammalian sex chromosome evolution.</h4>
<p class="authors"><span class="author">Waters PD</span>, <span class="author">Delbridge ML</span>, <span class="author">Deakin JE</span>, <span class="author">El-Mogharbel N</span>, <span class="author">Kirby PJ</span>, <span class="author">Carvalho-Silva DR</span> and <span class="author">Graves JA</span></p>
<p class="affiliation">Comparative Genomics Group, Research Group of Biological Science, The Australian National University, Canberra, ACT, 2601, Australia. <a href="mailto:%70%77%61%74%65%72%73%40%73%75%6E.%61%63.%7A%61">&#112;&#119;&#97;&#116;&#101;&#114;&#115;&#64;&#115;&#117;&#110;&#46;&#97;&#99;&#46;&#122;&#97;</a></p><p class="abstract">Mammalian sex chromosomes evolved from an ancient autosomal pair. Mapping of human X- and Y-borne genes in distantly related mammals and non-mammalian vertebrates has proved valuable to help deduce the evolution of this unique part of the genome. The platypus, a monotreme mammal distantly related to eutherians and marsupials, has an extraordinary sex chromosome system comprising five X and five Y chromosomes that form a translocation chain at male meiosis. The largest X chromosome (X1), which lies at one end of the chain, has considerable homology to the human X. Using comparative mapping and the emerging chicken database, we demonstrate that part of the therian X chromosome, previously thought to be conserved across all mammals, was lost from the platypus X1 to an autosome. This region included genes flanking the XIST locus, and also genes with Y-linked homologues that are important to male reproduction in therians. Since these genes lie on the X in marsupials and eutherians, and also on the homologous region of chicken chromosome 4, this represents a loss from the monotreme X rather than an additional evolutionary stratum of the human X.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">401-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15973504" rel="external">15973504</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-005-0978-5" rel="external">10.1007/s10577-005-0978-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differentiating campomelic dysplasia from Cumming syndrome.</h4>
<p class="authors"><span class="author">Watiker V</span>, <span class="author">Lachman RS</span>, <span class="author">Wilcox WR</span>, <span class="author">Barroso I</span>, <span class="author">Schafer AJ</span> and <span class="author">Scherer G</span></p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">5P01 HD 22657</span></p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2005</span>;<span class="volume">135</span>;<span class="number">1</span>;<span class="pagerange">110-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15754354">15754354</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.30650">10.1002/ajmg.a.30650</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Validation of mRNA/EST-based gene predictions in human Xp11.4 revealed differences to the organization of the orthologous mouse locus.</h4>
<p class="authors"><span class="author">Wen G</span>, <span class="author">Ramser J</span>, <span class="author">Taudien S</span>, <span class="author">Gausmann U</span>, <span class="author">Blechschmidt K</span>, <span class="author">Frankish A</span>, <span class="author">Ashurst J</span>, <span class="author">Meindl A</span> and <span class="author">Platzer M</span></p>
<p class="affiliation">Genome Analysis, Institute of Molecular Biotechnology, Beutenbergstr. 11, 07745, Jena, Germany.</p><p class="abstract">Careful manual annotation of the human reference sequence provides a solid basis for the identification of disease-associated genes. Toward this end, we focused on a medically relevant 2.6-Mb region of the human chromosome Xp11.4 between markers DXS9851 and DXS9751 and identified 16 transcription units according to the Vertebrate Genome Annotation (Vega) rules. In order to validate these annotations, we performed a comprehensive RT-PCR expression analysis and a human-mouse comparison. This revealed, despite the high overall genomic conservation of the region, remarkable differences of the gene content between human and mouse. Whereas 12 of 16 annotations were confirmed by RT-PCR in human tissues, for only seven genes mouse orthologs could be identified and found to be expressed. This indicates that a comprehensive and experimentally supported annotation effort of the human genome simultaneously highlights regions with striking differences in gene organization to other species and may indicate evolutionary events specific to the human lineage demanding further functional analyses.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2005</span>;<span class="volume">16</span>;<span class="number">12</span>;<span class="pagerange">934-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16341673">16341673</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-005-0090-3">10.1007/s00335-005-0090-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Slc11a1-mediated resistance to Salmonella enterica serovar Typhimurium and Leishmania donovani infections does not require functional inducible nitric oxide synthase or phagocyte oxidase activity.</h4>
<p class="authors"><span class="author">White JK</span>, <span class="author">Mastroeni P</span>, <span class="author">Popoff JF</span>, <span class="author">Evans CA</span> and <span class="author">Blackwell JM</span></p>
<p class="affiliation">Wellcome Trust/MRC Building, University of Cambridge School of Clinical Medicine, Addenbrookes Hospital, Hills Road, Cambridge CB2 2XY, UK.</p><p class="abstract">Solute carrier family 11a member 1 (Slc11a1; formerly natural resistance-associated macrophage protein 1) encodes a late endosomal/lysosomal protein/divalent cation transporter, which regulates iron homeostasis in macrophages. During macrophage activation, Slc11a1 exerts pleiotropic effects on gene regulation and function, including generation of nitric oxide (NO) via inducible NO synthase (iNOS; encoded by Nos2A) and of reactive oxygen intermediates (ROI) via the phagocyte oxidase complex. As NO and ROI have potent antimicrobial activity in macrophages, it was assumed that their activities would contribute to Slc11a1-regulated innate resistance to Salmonella enterica serovar Typhimurium and Leishmania donovani. By intercrossing mice with gene disruptions at Nos2A and Cybb (encoding gp91phox, the heavy chain subunit of cytochrome b-245 and an essential component of phagocyte NADPH oxidase) onto equivalent Slc11a1 wild-type and mutant genetic backgrounds, we demonstrate that neither iNOS nor gp91phox activity is required for Slc11a1-mediated innate resistance to either infection. Functional gp91phox and iNOS are required to control S. enterica serovar Typhimurium in non-Slc11a1-regulated phases of infection. For L. donovani, an organ-specific requirement for iNOS to clear parasites from the spleen was observed at 50 days post-infection, but neither iNOS nor gp91phox influenced late-phase infection in the liver. This contrasted with Leishmania major infection, which caused rapid lesion growth and death in iNOS knockout mice and some exacerbation of disease with gp91phox deficiency. This highlights the adaptive differences in tissue and cellular tropisms between L. donovani and L. major and the different genes and mechanisms that regulate visceral versus cutaneous forms of the disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">065642</span></p>
<p class="publication"><span class="title">Journal of leukocyte biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">77</span>;<span class="number">3</span>;<span class="pagerange">311-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15601666" rel="external">15601666</a>; DOI: <a href="http://dx.doi.org/10.1189%2Fjlb.0904546" rel="external">10.1189/jlb.0904546</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Seasonally hibernating phenotype assessed through transcript screening.</h4>
<p class="authors"><span class="author">Williams DR</span>, <span class="author">Epperson LE</span>, <span class="author">Li W</span>, <span class="author">Hughes MA</span>, <span class="author">Taylor R</span>, <span class="author">Rogers J</span>, <span class="author">Martin SL</span>, <span class="author">Cossins AR</span> and <span class="author">Gracey AY</span></p>
<p class="affiliation">School of Biological Sciences, University of Liverpool, United Kingdom.</p><p class="abstract">Hibernation is a seasonally entrained and profound phenotypic transition to conserve energy in winter. It involves significant biochemical reprogramming, although our understanding of the underpinning molecular events is fragmentary and selective. We have conducted a large-scale gene expression screen of the golden-mantled ground squirrel, Spermophilus lateralis, to identify transcriptional responses associated specifically with the summer-winter transition and the torpid-arousal transition in winter. We used 112 cDNA microarrays comprising 12,288 probes that cover at least 5,109 genes. In liver, the profiles of torpid and active states in the winter were almost identical, although we identified 102 cDNAs that were differentially expressed between winter and summer, 90% of which were downregulated in the winter states. By contrast, in cardiac tissue, 59 and 115 cDNAs were elevated in interbout arousal and torpor, respectively, relative to the summer active condition, but only 7 were common to both winter states, and during arousal none was downregulated. In brain, 78 cDNAs were found to change in winter, 44 of which were upregulated. Thus transcriptional changes associated with hibernation are qualitatively modest and, since these changes are generally less than twofold, also quantitatively modest. Unbiased Gene Ontology profiling of the transcripts suggests a winter switch to beta-oxidation of lipids in liver and heart, a reduction in metabolism of toxic compounds and the urea cycle in liver, and downregulated electron transport in the brain. We identified just one strongly winter-induced transcript common to all tissues, namely an RNA-binding protein, RBM3. This analysis clearly differentiates responses of the principal tissues, identifies a large number of new genes undergoing regulation, and broadens our understanding of affected cellular processes that, in part, account for the winter-adaptive hibernating phenotype.</p>
<p class="publication"><span class="title">Physiological genomics</span>&nbsp;<span class="year">2005</span>;<span class="volume">24</span>;<span class="number">1</span>;<span class="pagerange">13-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16249311" rel="external">16249311</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fphysiolgenomics.00301.2004" rel="external">10.1152/physiolgenomics.00301.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence and lytic phase transcription profile of a Coccolithovirus.</h4>
<p class="authors"><span class="author">Wilson WH</span>, <span class="author">Schroeder DC</span>, <span class="author">Allen MJ</span>, <span class="author">Holden MT</span>, <span class="author">Parkhill J</span>, <span class="author">Barrell BG</span>, <span class="author">Churcher C</span>, <span class="author">Hamlin N</span>, <span class="author">Mungall K</span>, <span class="author">Norbertczak H</span>, <span class="author">Quail MA</span>, <span class="author">Price C</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Walker D</span>, <span class="author">Craigon M</span>, <span class="author">Roy D</span> and <span class="author">Ghazal P</span></p>
<p class="affiliation">Plymouth Marine Laboratory, Prospect Place, The Hoe, Plymouth, PL1 3DH, UK. <a href="mailto:%77%68%77%40%70%6D%6C.%61%63.%75%6B">&#119;&#104;&#119;&#64;&#112;&#109;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The genus Coccolithovirus is a recently discovered group of viruses that infect the globally important marine calcifying microalga Emiliania huxleyi. Among the 472 predicted genes of the 407,339-base pair genome are a variety of unexpected genes, most notably those involved in biosynthesis of ceramide, a sphingolipid known to induce apoptosis. Uniquely for algal viruses, it also contains six RNA polymerase subunits and a novel promoter, suggesting this virus encodes its own transcription machinery. Microarray transcriptomic analysis reveals that 65% of the predicted virus-encoded genes are expressed during lytic infection of E. huxleyi.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">309</span>;<span class="number">5737</span>;<span class="pagerange">1090-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16099989">16099989</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1113109">10.1126/science.1113109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Design, validation, and application of a seven-strain Staphylococcus aureus PCR product microarray for comparative genomics.</h4>
<p class="authors"><span class="author">Witney AA</span>, <span class="author">Marsden GL</span>, <span class="author">Holden MT</span>, <span class="author">Stabler RA</span>, <span class="author">Husain SE</span>, <span class="author">Vass JK</span>, <span class="author">Butcher PD</span>, <span class="author">Hinds J</span> and <span class="author">Lindsay JA</span></p>
<p class="affiliation">Department of Cellular and Molecular Medicine, St. George&#39;s Hospital Medical School, London SW17 0RE, United Kingdom.</p><p class="abstract">Bacterial comparative genomics has been revolutionized by microarrays, but the power of any microarray is dependent on the number and diversity of gene reporters it contains. Staphylococcus aureus is an important human pathogen causing a wide range of invasive and toxin-mediated diseases, and more than 20% of the genome of any isolate consists of variable genes. Seven whole-genome sequences of S. aureus are available, and we exploited this rare opportunity to design, build, and validate a comprehensive, nonredundant PCR product microarray carrying reporters that represent every predicted open reading frame (3,623 probes). Such a comprehensive microarray necessitated a novel design strategy. Validation with the seven sequenced strains showed correct identification of 93.9% of genes present or absent/divergent but was dependent on the method of analysis chosen. Microarray data were highly reproducible, reducing the need for many replicate slides. Interpretation of microarray data was enhanced by focusing on the major areas of variation--the presence or absence of mobile genetic elements (MGEs). We compiled &quot;composite genomes&quot; of every individual MGE and visualized their distribution. This allowed the sensitive discrimination of related isolates, including the first clear description of how isolates of the same clone of epidemic methicillin-resistant S. aureus differ substantially in their carriage of MGEs. These MGEs carry virulence and resistance genes, suggesting differences in pathogenic potential. The novel methods of design and interpretation of data generated from this microarray will enable further studies of S. aureus evolution, epidemiology, and pathogenesis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062511</span></p>
<p class="publication"><span class="title">Applied and environmental microbiology</span>&nbsp;<span class="year">2005</span>;<span class="volume">71</span>;<span class="number">11</span>;<span class="pagerange">7504-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16269792" rel="external">16269792</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1287626" rel="external">1287626</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAEM.71.11.7504-7514.2005" rel="external">10.1128/AEM.71.11.7504-7514.2005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BRAF and NRAS mutations are uncommon in melanomas arising in diverse internal organs.</h4>
<p class="authors"><span class="author">Wong CW</span>, <span class="author">Fan YS</span>, <span class="author">Chan TL</span>, <span class="author">Chan AS</span>, <span class="author">Ho LC</span>, <span class="author">Ma TK</span>, <span class="author">Yuen ST</span>, <span class="author">Leung SY</span> and <span class="author">Cancer Genome Project</span></p>
<p class="affiliation">Department of Pathology, University of Hong Kong, Queen Mary Hospital, Hong Kong.</p><p class="abstract"><strong>Background</strong>: Malignant melanoma arising from different body compartments may be associated with differing aetiological factors and clinical behaviour, and may manifest diverse molecular genetic profiles. Although many studies have focused on cutaneous melanoma, little is known of mucosal and other types of melanoma. In particular, malignant melanoma of soft parts is different from other melanomas in many respects, yet manifests a common melanocytic differentiation. Mutation of BRAF is now known to be common in cutaneous melanomas, and raises possible new therapeutic options of anti-RAF treatment for these patients. Few data are available for non-cutaneous melanomas.</p><p class="abstract"><strong>Aims</strong>: To study the incidence of BRAF and NRAS mutations in melanomas arising in diverse internal organs.</p><p class="abstract"><strong>Methods</strong>: Fifty one melanomas from various internal organs were investigated for BRAF and NRAS mutation by direct DNA sequencing.</p><p class="abstract"><strong>Results</strong>: BRAF and NRAS mutations were found in two and five mucosal melanomas arising from the aerodigestive and female genital tracts (n = 36). Their occurrence is mutually exclusive, giving a combined mutation incidence rate of 19.4% in mucosal melanomas. Both BRAF and NRAS mutations were absent in malignant melanoma of soft parts (n = 7). BRAF mutation was also absent in uveal melanoma (n = 6), but was seen in two of five cutaneous melanomas. The incidence of BRAF or combined BRAF/NRAS mutations in all non-cutaneous groups was significantly lower than published rates for cutaneous melanomas.</p><p class="abstract"><strong>Conclusion</strong>: Each melanoma subtype may have a unique oncogenetic pathway of tumour development, and only a small fraction of non-cutaneous melanomas may benefit from anti-RAF treatment.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of clinical pathology</span>&nbsp;<span class="year">2005</span>;<span class="volume">58</span>;<span class="number">6</span>;<span class="pagerange">640-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15917418" rel="external">15917418</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1770697" rel="external">1770697</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjcp.2004.022509" rel="external">10.1136/jcp.2004.022509</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication timing of human chromosome 6.</h4>
<p class="authors"><span class="author">Woodfine K</span>, <span class="author">Beare DM</span>, <span class="author">Ichimura K</span>, <span class="author">Debernardi S</span>, <span class="author">Mungall AJ</span>, <span class="author">Fiegler H</span>, <span class="author">Collins VP</span>, <span class="author">Carter NP</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Genomic microarrays have been used to assess DNA replication timing in a variety of eukaryotic organisms. A replication timing map of the human genome has already been published at a 1Mb resolution. Here we describe how the same method can be used to assess the replication timing of chromosome 6 with a greater resolution using an array of overlapping tile path clones. We report the replication timing map of the whole of chromosome 6 in general, and the MHC region in particular. Positive correlations are observed between replication timing and a number of genomic features including GC content, repeat content and transcriptional activity.</p>
<p class="publication"><span class="title">Cell cycle (Georgetown, Tex.)</span>&nbsp;<span class="year">2005</span>;<span class="volume">4</span>;<span class="number">1</span>;<span class="pagerange">172-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15611667" rel="external">15611667</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Investigating chromosome organization with genomic microarrays.</h4>
<p class="authors"><span class="author">Woodfine K</span>, <span class="author">Carter NP</span>, <span class="author">Dunham I</span> and <span class="author">Fiegler H</span></p>
<p class="affiliation">Department of Medical and Molecular Genetics, GKT School of Medicine, King&#39;s College London, London, SE1 9RT, UK.</p><p class="abstract">DNA microarrays are increasingly being used to investigate the functional role of chromatin. These studies are enhanced by the development of high-resolution arrays covering either the whole genome or specific regions of selected chromosomes with large insert clones, PCR products or oligonucleotides of around 100 bp or less. In combination with chromatin immunoprecipitation, this approach allows identification of protein binding for transcription factors, proteins involved in DNA replication and repair as well as sites of chromatin modification. Furthermore, by application of S phase fractions to genomic microarrays, replication timing can be estimated. Thus, microarrays can provide new information about chromosome structure and gene regulation.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2005</span>;<span class="volume">13</span>;<span class="number">3</span>;<span class="pagerange">249-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15868419" rel="external">15868419</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-005-1504-5" rel="external">10.1007/s10577-005-1504-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Heterogeneous duplications in patients with Pelizaeus-Merzbacher disease suggest a mechanism of coupled homologous and nonhomologous recombination.</h4>
<p class="authors"><span class="author">Woodward KJ</span>, <span class="author">Cundall M</span>, <span class="author">Sperle K</span>, <span class="author">Sistermans EA</span>, <span class="author">Ross M</span>, <span class="author">Howell G</span>, <span class="author">Gribble SM</span>, <span class="author">Burford DC</span>, <span class="author">Carter NP</span>, <span class="author">Hobson DL</span>, <span class="author">Garbern JY</span>, <span class="author">Kamholz J</span>, <span class="author">Heng H</span>, <span class="author">Hodes ME</span>, <span class="author">Malcolm S</span> and <span class="author">Hobson GM</span></p>
<p class="affiliation">Clinical and Molecular Genetics, Institute of Child Health, London.</p><p class="abstract">We describe genomic structures of 59 X-chromosome segmental duplications that include the proteolipid protein 1 gene (PLP1) in patients with Pelizaeus-Merzbacher disease. We provide the first report of 13 junction sequences, which gives insight into underlying mechanisms. Although proximal breakpoints were highly variable, distal breakpoints tended to cluster around low-copy repeats (LCRs) (50% of distal breakpoints), and each duplication event appeared to be unique (100 kb to 4.6 Mb in size). Sequence analysis of the junctions revealed no large homologous regions between proximal and distal breakpoints. Most junctions had microhomology of 1-6 bases, and one had a 2-base insertion. Boundaries between single-copy and duplicated DNA were identical to the reference genomic sequence in all patients investigated. Taken together, these data suggest that the tandem duplications are formed by a coupled homologous and nonhomologous recombination mechanism. We suggest repair of a double-stranded break (DSB) by one-sided homologous strand invasion of a sister chromatid, followed by DNA synthesis and nonhomologous end joining with the other end of the break. This is in contrast to other genomic disorders that have recurrent rearrangements formed by nonallelic homologous recombination between LCRs. Interspersed repetitive elements (Alu elements, long interspersed nuclear elements, and long terminal repeats) were found at 18 of the 26 breakpoint sequences studied. No specific motif that may predispose to DSBs was revealed, but single or alternating tracts of purines and pyrimidines that may cause secondary structures were common. Analysis of the 2-Mb region susceptible to duplications identified proximal-specific repeats and distal LCRs in addition to the previously reported ones, suggesting that the unique genomic architecture may have a role in nonrecurrent rearrangements by promoting instability.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P20 RR-020173-01</span>; NINDS NIH HHS: <span class="grant">NS043783</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">77</span>;<span class="number">6</span>;<span class="pagerange">966-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16380909" rel="external">16380909</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1285180" rel="external">1285180</a>; DOI: <a href="http://dx.doi.org/10.1086%2F498048" rel="external">10.1086/498048</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recent spread of a Y-chromosomal lineage in northern China and Mongolia.</h4>
<p class="authors"><span class="author">Xue Y</span>, <span class="author">Zerjal T</span>, <span class="author">Bao W</span>, <span class="author">Zhu S</span>, <span class="author">Lim SK</span>, <span class="author">Shu Q</span>, <span class="author">Xu J</span>, <span class="author">Du R</span>, <span class="author">Fu S</span>, <span class="author">Li P</span>, <span class="author">Yang H</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">We have identified a Y-chromosomal lineage that is unusually frequent in northeastern China and Mongolia, in which a haplotype cluster defined by 15 Y short tandem repeats was carried by approximately 3.3% of the males sampled from East Asia. The most recent common ancestor of this lineage lived 590 +/- 340 years ago (mean +/- SD), and it was detected in Mongolians and six Chinese minority populations. We suggest that the lineage was spread by Qing Dynasty (1644-1912) nobility, who were a privileged elite sharing patrilineal descent from Giocangga (died 1582), the grandfather of Manchu leader Nurhaci, and whose documented members formed approximately 0.4% of the minority population by the end of the dynasty.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">77</span>;<span class="number">6</span>;<span class="pagerange">1112-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16380921">16380921</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1285168">1285168</a>; DOI: <a href="http://dx.doi.org/10.1086%2F498583">10.1086/498583</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome dynamics and diversity of Shigella species, the etiologic agents of bacillary dysentery.</h4>
<p class="authors"><span class="author">Yang F</span>, <span class="author">Yang J</span>, <span class="author">Zhang X</span>, <span class="author">Chen L</span>, <span class="author">Jiang Y</span>, <span class="author">Yan Y</span>, <span class="author">Tang X</span>, <span class="author">Wang J</span>, <span class="author">Xiong Z</span>, <span class="author">Dong J</span>, <span class="author">Xue Y</span>, <span class="author">Zhu Y</span>, <span class="author">Xu X</span>, <span class="author">Sun L</span>, <span class="author">Chen S</span>, <span class="author">Nie H</span>, <span class="author">Peng J</span>, <span class="author">Xu J</span>, <span class="author">Wang Y</span>, <span class="author">Yuan Z</span>, <span class="author">Wen Y</span>, <span class="author">Yao Z</span>, <span class="author">Shen Y</span>, <span class="author">Qiang B</span>, <span class="author">Hou Y</span>, <span class="author">Yu J</span> and <span class="author">Jin Q</span></p>
<p class="affiliation">State Key Laboratory for Molecular Virology and Genetic Engineering, Chinese Ministry of Public Health, Beijing 100052, China.</p><p class="abstract">The Shigella bacteria cause bacillary dysentery, which remains a significant threat to public health. The genus status and species classification appear no longer valid, as compelling evidence indicates that Shigella, as well as enteroinvasive Escherichia coli, are derived from multiple origins of E.coli and form a single pathovar. Nevertheless, Shigella dysenteriae serotype 1 causes deadly epidemics but Shigella boydii is restricted to the Indian subcontinent, while Shigella flexneri and Shigella sonnei are prevalent in developing and developed countries respectively. To begin to explain these distinctive epidemiological and pathological features at the genome level, we have carried out comparative genomics on four representative strains. Each of the Shigella genomes includes a virulence plasmid that encodes conserved primary virulence determinants. The Shigella chromosomes share most of their genes with that of E.coli K12 strain MG1655, but each has over 200 pseudogenes, 300 approximately 700 copies of insertion sequence (IS) elements, and numerous deletions, insertions, translocations and inversions. There is extensive diversity of putative virulence genes, mostly acquired via bacteriophage-mediated lateral gene transfer. Hence, via convergent evolution involving gain and loss of functions, through bacteriophage-mediated gene acquisition, IS-mediated DNA rearrangements and formation of pseudogenes, the Shigella spp. became highly specific human pathogens with variable epidemiological and pathological features.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2005</span>;<span class="volume">33</span>;<span class="number">19</span>;<span class="pagerange">6445-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16275786">16275786</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1278947">1278947</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgki954">10.1093/nar/gki954</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evaluation of HapMap sample size and tagging SNP performance in large-scale empirical and simulated data sets.</h4>
<p class="authors"><span class="author">Zeggini E</span>, <span class="author">Rayner W</span>, <span class="author">Morris AP</span>, <span class="author">Hattersley AT</span>, <span class="author">Walker M</span>, <span class="author">Hitman GA</span>, <span class="author">Deloukas P</span>, <span class="author">Cardon LR</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%65%6C%65%7A%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#101;&#108;&#101;&#122;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A substantial investment has been made in the generation of large public resources designed to enable the identification of tag SNP sets, but data establishing the adequacy of the sample sizes used are limited. Using large-scale empirical and simulated data sets, we found that the sample sizes used in the HapMap project are sufficient to capture common variation, but that performance declines substantially for variants with minor allele frequencies of &lt;5%.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">37</span>;<span class="number">12</span>;<span class="pagerange">1320-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16258542">16258542</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1670">10.1038/ng1670</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The function and expansion of the Patched- and Hedgehog-related homologs in C. elegans.</h4>
<p class="authors"><span class="author">Zugasti O</span>, <span class="author">Rajan J</span> and <span class="author">Kuwabara PE</span></p>
<p class="affiliation">University of Bristol, Department of Biochemistry, School of Medical Sciences, Bristol BS8 1TD, United Kingdom.</p><p class="abstract">The Hedgehog (Hh) signaling pathway promotes pattern formation and cell proliferation in Drosophila and vertebrates. Hh is a ligand that binds and represses the Patched (Ptc) receptor and thereby releases the latent activity of the multipass membrane protein Smoothened (Smo), which is essential for transducing the Hh signal. In Caenorhabditis elegans, the Hh signaling pathway has undergone considerable divergence. Surprisingly, obvious Smo and Hh homologs are absent whereas PTC, PTC-related (PTR), and a large family of nematode Hh-related (Hh-r) proteins are present. We find that the number of PTC-related and Hh-r proteins has expanded in C. elegans, and that this expansion occurred early in Nematoda. Moreover, the function of these proteins appears to be conserved in Caenorhabditis briggsae. Given our present understanding of the Hh signaling pathway, the absence of Hh and Smo raises many questions about the evolution and the function of the PTC, PTR, and Hh-r proteins in C. elegans. To gain insights into their roles, we performed a global survey of the phenotypes produced by RNA-mediated interference (RNAi). Our study reveals that these genes do not require Smo for activity and that they function in multiple aspects of C. elegans development, including molting, cytokinesis, growth, and pattern formation. Moreover, a subset of the PTC, PTR, and Hh-r proteins have the same RNAi phenotypes, indicating that they have the potential to participate in the same processes.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2005</span>;<span class="volume">15</span>;<span class="number">10</span>;<span class="pagerange">1402-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16204193" rel="external">16204193</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1240083" rel="external">1240083</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.3935405" rel="external">10.1101/gr.3935405</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global hypomethylation of the genome in XX embryonic stem cells.</h4>
<p class="authors"><span class="author">Zvetkova I</span>, <span class="author">Apedaile A</span>, <span class="author">Ramsahoye B</span>, <span class="author">Mermoud JE</span>, <span class="author">Crompton LA</span>, <span class="author">John R</span>, <span class="author">Feil R</span> and <span class="author">Brockdorff N</span></p>
<p class="affiliation">MRC Clinical Sciences Centre, ICFM, Hammersmith Hospital, DuCane Road, London, W12 ONN, UK.</p><p class="abstract">Embryonic stem (ES) cells are important tools in the study of gene function and may also become important in cell therapy applications. Establishment of stable XX ES cell lines from mouse blastocysts is relatively problematic owing to frequent loss of one of the two X chromosomes. Here we show that DNA methylation is globally reduced in XX ES cell lines and that this is attributable to the presence of two active X chromosomes. Hypomethylation affects both repetitive and unique sequences, the latter including differentially methylated regions that regulate expression of parentally imprinted genes. Methylation of differentially methylated regions can be restored coincident with elimination of an X chromosome in early-passage parthenogenetic ES cells, suggesting that selection against loss of methylation may provide the basis for X-chromosome instability. Finally, we show that hypomethylation is associated with reduced levels of the de novo DNA methyltransferases Dnmt3a and Dnmt3b and that ectopic expression of these factors restores global methylation levels.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2005</span>;<span class="volume">37</span>;<span class="number">11</span>;<span class="pagerange">1274-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16244654">16244654</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1663">10.1038/ng1663</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/bhcwcxb2"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/bhcwcxb2" src="/qr/bhcwcxb2.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
