<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2009 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2009.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2009</h1><p>Number of papers published in 2009: <strong>239</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PSD-95 is essential for hallucinogen and atypical antipsychotic drug actions at serotonin receptors.</h4>
<p class="authors"><span class="author">Abbas AI</span>, <span class="author">Yadav PN</span>, <span class="author">Yao WD</span>, <span class="author">Arbuckle MI</span>, <span class="author">Grant SG</span>, <span class="author">Caron MG</span> and <span class="author">Roth BL</span></p>
<p class="affiliation">Department of Biochemistry, Case Western Reserve University School of Medicine, Cleveland, Ohio 44106, USA.</p><p class="abstract">Here, we report that postsynaptic density protein of 95 kDa (PSD-95), a postsynaptic density scaffolding protein, classically conceptualized as being essential for the regulation of ionotropic glutamatergic signaling at the postsynaptic membrane, plays an unanticipated and essential role in mediating the actions of hallucinogens and atypical antipsychotic drugs at 5-HT(2A) and 5-HT(2C) serotonergic G-protein-coupled receptors. We show that PSD-95 is crucial for normal 5-HT(2A) and 5-HT(2C) expression in vivo and that PSD-95 maintains normal receptor expression by promoting apical dendritic targeting and stabilizing receptor turnover in vivo. Significantly, 5-HT(2A)- and 5-HT(2C)-mediated downstream signaling is impaired in PSD-95(null) mice, and the 5-HT(2A)-mediated head-twitch response is abnormal. Furthermore, the ability of 5-HT(2A) inverse agonists to normalize behavioral changes induced by glutamate receptor antagonists is abolished in the absence of PSD-95 in vivo. These results demonstrate that PSD-95, in addition to the well known role it plays in scaffolding macromolecular glutamatergic signaling complexes, profoundly modulates metabotropic 5-HT(2A) and 5-HT(2C) receptor function.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">RR00168</span>; NIDA NIH HHS: <span class="grant">DA021420</span>; NIGMS NIH HHS: <span class="grant">T32 GM007250</span>; NIMH NIH HHS: <span class="grant">MH-73853</span>, <span class="grant">MH61887</span>, <span class="grant">R01 MH061887-07</span>, <span class="grant">R01 MH061887-08</span>, <span class="grant">R01 MH061887-09</span>, <span class="grant">R01 MH061887-10</span>, <span class="grant">R01 MH061887-11</span>, <span class="grant">R01 MH061887-12</span>, <span class="grant">U19 MH082441-01</span>, <span class="grant">U19 MH082441-010001</span>, <span class="grant">U19 MH082441-019003</span>, <span class="grant">U19 MH082441-02</span>, <span class="grant">U19 MH082441-020001</span>, <span class="grant">U19 MH082441-03S1</span>, <span class="grant">U19 MH082441-04</span>, <span class="grant">U19 MH082441-040001</span>, <span class="grant">U19 MH082441-05</span>, <span class="grant">U19MH82441</span>; NINDS NIH HHS: <span class="grant">NS-19576</span>, <span class="grant">NS057311</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2009</span>;<span class="volume">29</span>;<span class="number">22</span>;<span class="pagerange">7124-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19494135" rel="external">19494135</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2836830" rel="external">2836830</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.1090-09.2009" rel="external">10.1523/JNEUROSCI.1090-09.2009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Newly discovered breast cancer susceptibility loci on 3p24 and 17q23.2.</h4>
<p class="authors"><span class="author">Ahmed S</span>, <span class="author">Thomas G</span>, <span class="author">Ghoussaini M</span>, <span class="author">Healey CS</span>, <span class="author">Humphreys MK</span>, <span class="author">Platte R</span>, <span class="author">Morrison J</span>, <span class="author">Maranian M</span>, <span class="author">Pooley KA</span>, <span class="author">Luben R</span>, <span class="author">Eccles D</span>, <span class="author">Evans DG</span>, <span class="author">Fletcher O</span>, <span class="author">Johnson N</span>, <span class="author">dos Santos Silva I</span>, <span class="author">Peto J</span>, <span class="author">Stratton MR</span>, <span class="author">Rahman N</span>, <span class="author">Jacobs K</span>, <span class="author">Prentice R</span>, <span class="author">Anderson GL</span>, <span class="author">Rajkovic A</span>, <span class="author">Curb JD</span>, <span class="author">Ziegler RG</span>, <span class="author">Berg CD</span>, <span class="author">Buys SS</span>, <span class="author">McCarty CA</span>, <span class="author">Feigelson HS</span>, <span class="author">Calle EE</span>, <span class="author">Thun MJ</span>, <span class="author">Diver WR</span>, <span class="author">Bojesen S</span>, <span class="author">Nordestgaard BG</span>, <span class="author">Flyger H</span>, <span class="author">D&ouml;rk T</span>, <span class="author">Sch&uuml;rmann P</span>, <span class="author">Hillemanns P</span>, <span class="author">Karstens JH</span>, <span class="author">Bogdanova NV</span>, <span class="author">Antonenkova NN</span>, <span class="author">Zalutsky IV</span>, <span class="author">Bermisheva M</span>, <span class="author">Fedorova S</span>, <span class="author">Khusnutdinova E</span>, <span class="author">SEARCH</span>, <span class="author">Kang D</span>, <span class="author">Yoo KY</span>, <span class="author">Noh DY</span>, <span class="author">Ahn SH</span>, <span class="author">Devilee P</span>, <span class="author">van Asperen CJ</span>, <span class="author">Tollenaar RA</span>, <span class="author">Seynaeve C</span>, <span class="author">Garcia-Closas M</span>, <span class="author">Lissowska J</span>, <span class="author">Brinton L</span>, <span class="author">Peplonska B</span>, <span class="author">Nevanlinna H</span>, <span class="author">Heikkinen T</span>, <span class="author">Aittom&auml;ki K</span>, <span class="author">Blomqvist C</span>, <span class="author">Hopper JL</span>, <span class="author">Southey MC</span>, <span class="author">Smith L</span>, <span class="author">Spurdle AB</span>, <span class="author">Schmidt MK</span>, <span class="author">Broeks A</span>, <span class="author">van Hien RR</span>, <span class="author">Cornelissen S</span>, <span class="author">Milne RL</span>, <span class="author">Ribas G</span>, <span class="author">Gonz&aacute;lez-Neira A</span>, <span class="author">Benitez J</span>, <span class="author">Schmutzler RK</span>, <span class="author">Burwinkel B</span>, <span class="author">Bartram CR</span>, <span class="author">Meindl A</span>, <span class="author">Brauch H</span>, <span class="author">Justenhoven C</span>, <span class="author">Hamann U</span>, <span class="author">GENICA Consortium</span>, <span class="author">Chang-Claude J</span>, <span class="author">Hein R</span>, <span class="author">Wang-Gohrke S</span>, <span class="author">Lindblom A</span>, <span class="author">Margolin S</span>, <span class="author">Mannermaa A</span>, <span class="author">Kosma VM</span>, <span class="author">Kataja V</span>, <span class="author">Olson JE</span>, <span class="author">Wang X</span>, <span class="author">Fredericksen Z</span>, <span class="author">Giles GG</span>, <span class="author">Severi G</span>, <span class="author">Baglietto L</span>, <span class="author">English DR</span>, <span class="author">Hankinson SE</span>, <span class="author">Cox DG</span>, <span class="author">Kraft P</span>, <span class="author">Vatten LJ</span>, <span class="author">Hveem K</span>, <span class="author">Kumle M</span>, <span class="author">Sigurdson A</span>, <span class="author">Doody M</span>, <span class="author">Bhatti P</span>, <span class="author">Alexander BH</span>, <span class="author">Hooning MJ</span>, <span class="author">van den Ouweland AM</span>, <span class="author">Oldenburg RA</span>, <span class="author">Schutte M</span>, <span class="author">Hall P</span>, <span class="author">Czene K</span>, <span class="author">Liu J</span>, <span class="author">Li Y</span>, <span class="author">Cox A</span>, <span class="author">Elliott G</span>, <span class="author">Brock I</span>, <span class="author">Reed MW</span>, <span class="author">Shen CY</span>, <span class="author">Yu JC</span>, <span class="author">Hsu GC</span>, <span class="author">Chen ST</span>, <span class="author">Anton-Culver H</span>, <span class="author">Ziogas A</span>, <span class="author">Andrulis IL</span>, <span class="author">Knight JA</span>, <span class="author">kConFab</span>, <span class="author">Australian Ovarian Cancer Study Group</span>, <span class="author">Beesley J</span>, <span class="author">Goode EL</span>, <span class="author">Couch F</span>, <span class="author">Chenevix-Trench G</span>, <span class="author">Hoover RN</span>, <span class="author">Ponder BA</span>, <span class="author">Hunter DJ</span>, <span class="author">Pharoah PD</span>, <span class="author">Dunning AM</span>, <span class="author">Chanock SJ</span> and <span class="author">Easton DF</span></p>
<p class="affiliation">Department of Oncology, University of Cambridge, UK.</p><p class="abstract">Genome-wide association studies (GWAS) have identified seven breast cancer susceptibility loci, but these explain only a small fraction of the familial risk of the disease. Five of these loci were identified through a two-stage GWAS involving 390 familial cases and 364 controls in the first stage, and 3,990 cases and 3,916 controls in the second stage. To identify additional loci, we tested over 800 promising associations from this GWAS in a further two stages involving 37,012 cases and 40,069 controls from 33 studies in the CGEMS collaboration and Breast Cancer Association Consortium. We found strong evidence for additional susceptibility loci on 3p (rs4973768: per-allele OR = 1.11, 95% CI = 1.08-1.13, P = 4.1 x 10(-23)) and 17q (rs6504950: per-allele OR = 0.95, 95% CI = 0.92-0.97, P = 1.4 x 10(-8)). Potential causative genes include SLC4A7 and NEK10 on 3p and COX11 on 17q.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11021</span>, <span class="grant">A10123</span>, <span class="grant">C1287/A10118</span>, <span class="grant">C1287/A5260</span>, <span class="grant">C1287/A7497</span>, <span class="grant">C490/A11021</span>; NCI NIH HHS: <span class="grant">5UO1CA098233</span>, <span class="grant">CA-06-503</span>, <span class="grant">CA-58860</span>, <span class="grant">CA-92044</span>, <span class="grant">CA-95-011</span>, <span class="grant">CA49449</span>, <span class="grant">CA50385</span>, <span class="grant">CA65725</span>, <span class="grant">CA67262</span>, <span class="grant">CA87969</span>, <span class="grant">P30 CA062203</span>, <span class="grant">P50 CA116201</span>, <span class="grant">R01 CA102740-01A2</span>, <span class="grant">R01 CA104021-04</span>, <span class="grant">R01 CA122340</span>, <span class="grant">U01 CA69398</span>, <span class="grant">U01 CA69417</span>, <span class="grant">U01 CA69446</span>, <span class="grant">U01 CA69467</span>, <span class="grant">U01 CA69631</span>, <span class="grant">U01 CA69638</span>, <span class="grant">UO1 CA098710</span>, <span class="grant">UO1 CA69467</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">5</span>;<span class="pagerange">585-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19330027">19330027</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2748125">2748125</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.354">10.1038/ng.354</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic diversity amongst isolates of Neospora caninum, and the development of a multiplex assay for the detection of distinct strains.</h4>
<p class="authors"><span class="author">Al-Qassab S</span>, <span class="author">Reichel MP</span>, <span class="author">Ivens A</span> and <span class="author">Ellis JT</span></p>
<p class="affiliation">Department of Medical and Molecular Biosciences, University of Technology, Sydney, P.O. Box 123, Broadway, New South Wales 2007, Australia.</p><p class="abstract">Infection with Neospora caninum is regarded as a significant cause of abortion in cattle. Despite the economic impact of this infection, relatively little is known about the biology of this parasite. In this study, mini and microsatellite DNAs were detected in the genome of N. caninum and eight loci were identified that each contained repetitive DNA which was polymorphic among different isolates of this parasite. A multiplex PCR assay was developed for the detection of genetic variation within N. caninum based on length polymorphism associated with three different repetitive markers. The utility of the multiplex PCR was demonstrated in that it was able to distinguish amongst strains of N. caninum used as either vaccine or challenge strains in animal vaccination experiments and that it could genotype N. caninum associated with naturally acquired infections of animals. The multiplex PCR is simple, rapid, informative and sensitive and should provide a valuable tool for further studies on the epidemiology of N. caninum in different host species.</p>
<p class="publication"><span class="title">Molecular and cellular probes</span>&nbsp;<span class="year">2009</span>;<span class="volume">23</span>;<span class="number">3-4</span>;<span class="pagerange">132-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19496247">19496247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SnoopCGH: software for visualizing comparative genomic hybridization data.</h4>
<p class="authors"><span class="author">Almagro-Garcia J</span>, <span class="author">Manske M</span>, <span class="author">Carret C</span>, <span class="author">Campino S</span>, <span class="author">Auburn S</span>, <span class="author">Macinnis BL</span>, <span class="author">Maslen G</span>, <span class="author">Pain A</span>, <span class="author">Newbold CI</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Clark TG</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, The Weatherall Institute of Molecular Medicine and Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%6A%67%31%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#103;&#49;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Unlabelled</strong>: Array-based comparative genomic hybridization (CGH) technology is used to discover and validate genomic structural variation, including copy number variants, insertions, deletions and other structural variants (SVs). The visualization and summarization of the array CGH data outputs, potentially across many samples, is an important process in the identification and analysis of SVs. We have developed a software tool for SV analysis using data from array CGH technologies, which is also amenable to short-read sequence data.</p><p class="abstract">SnoopCGH is written in java and is available from http://snoopcgh.sourceforge.net/</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">20</span>;<span class="pagerange">2732-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19687029">19687029</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2759554">2759554</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp488">10.1093/bioinformatics/btp488</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Manual annotation and analysis of the defensin gene cluster in the C57BL/6J mouse reference genome.</h4>
<p class="authors"><span class="author">Amid C</span>, <span class="author">Rehaume LM</span>, <span class="author">Brown KL</span>, <span class="author">Gilbert JG</span>, <span class="author">Dougan G</span>, <span class="author">Hancock RE</span> and <span class="author">Harrow JL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%63%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Host defense peptides are a critical component of the innate immune system. Human alpha- and beta-defensin genes are subject to copy number variation (CNV) and historically the organization of mouse alpha-defensin genes has been poorly defined. Here we present the first full manual genomic annotation of the mouse defensin region on Chromosome 8 of the reference strain C57BL/6J, and the analysis of the orthologous regions of the human and rat genomes. Problems were identified with the reference assemblies of all three genomes. Defensins have been studied for over two decades and their naming has become a critical issue due to incorrect identification of defensin genes derived from different mouse strains and the duplicated nature of this region.</p><p class="abstract"><strong>Results</strong>: The defensin gene cluster region on mouse Chromosome 8 A2 contains 98 gene loci: 53 are likely active defensin genes and 22 defensin pseudogenes. Several TATA box motifs were found for human and mouse defensin genes that likely impact gene expression. Three novel defensin genes belonging to the Cryptdin Related Sequences (CRS) family were identified. All additional mouse defensin loci on Chromosomes 1, 2 and 14 were annotated and unusual splice variants identified. Comparison of the mouse alpha-defensins in the three main mouse reference gene sets Ensembl, Mouse Genome Informatics (MGI), and NCBI RefSeq reveals significant inconsistencies in annotation and nomenclature. We are collaborating with the Mouse Genome Nomenclature Committee (MGNC) to establish a standardized naming scheme for alpha-defensins.</p><p class="abstract"><strong>Conclusions</strong>: Prior to this analysis, there was no reliable reference gene set available for the mouse strain C57BL/6J defensin genes, demonstrating that manual intervention is still critical for the annotation of complex gene families and heavily duplicated regions. Accurate gene annotation is facilitated by the annotation of pseudogenes and regulatory elements. Manually curated gene models will be incorporated into the Ensembl and Consensus Coding Sequence (CCDS) reference sets. Elucidation of the genomic structure of this complex gene cluster on the mouse reference sequence, and adoption of a clear and unambiguous naming scheme, will provide a valuable tool to support studies on the evolution, regulatory mechanisms and biological functions of defensins in vivo.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">606</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20003482">20003482</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2807441">2807441</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-10-606">10.1186/1471-2164-10-606</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Testing for rare variant associations in complex diseases.</h4>
<p class="authors"><span class="author">Asimit J</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1HH, UK. <a href="mailto:%6A%61%31%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#97;&#49;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">The study of rare variants holds the promise of accounting for some of the missing heritability in complex traits. Next-generation sequencing technologies enable probing of variation across the full spectrum of allele frequencies. Multiple methods for the analysis of rare variants have been proposed and, recently, Ionita-Laza et al. have presented an approach with the theoretical capacity to detect risk and protective variants. The identification of rare risk variants could have major implications in understanding complex disease etiopathogenesis.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">1</span>;<span class="number">11</span>;<span class="pagerange">24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21542888">21542888</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3129640">3129640</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm238">10.1186/gm238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ABACAS: algorithm-based automatic contiguation of assembled sequences.</h4>
<p class="authors"><span class="author">Assefa S</span>, <span class="author">Keane TM</span>, <span class="author">Otto TD</span>, <span class="author">Newbold C</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK. <a href="mailto:%73%61%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#97;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Summary</strong>: Due to the availability of new sequencing technologies, we are now increasingly interested in sequencing closely related strains of existing finished genomes. Recently a number of de novo and mapping-based assemblers have been developed to produce high quality draft genomes from new sequencing technology reads. New tools are necessary to take contigs from a draft assembly through to a fully contiguated genome sequence. ABACAS is intended as a tool to rapidly contiguate (align, order, orientate), visualize and design primers to close gaps on shotgun assembled contigs based on a reference sequence. The input to ABACAS is a set of contigs which will be aligned to the reference genome, ordered and orientated, visualized in the ACT comparative browser, and optimal primer sequences are automatically generated.</p><p class="abstract">ABACAS is implemented in Perl and is freely available for download from http://abacas.sourceforge.net.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">15</span>;<span class="pagerange">1968-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19497936">19497936</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2712343">2712343</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp347">10.1093/bioinformatics/btp347</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts.</h4>
<p class="authors"><span class="author">Aulchenko YS</span>, <span class="author">Ripatti S</span>, <span class="author">Lindqvist I</span>, <span class="author">Boomsma D</span>, <span class="author">Heid IM</span>, <span class="author">Pramstaller PP</span>, <span class="author">Penninx BW</span>, <span class="author">Janssens AC</span>, <span class="author">Wilson JF</span>, <span class="author">Spector T</span>, <span class="author">Martin NG</span>, <span class="author">Pedersen NL</span>, <span class="author">Kyvik KO</span>, <span class="author">Kaprio J</span>, <span class="author">Hofman A</span>, <span class="author">Freimer NB</span>, <span class="author">Jarvelin MR</span>, <span class="author">Gyllensten U</span>, <span class="author">Campbell H</span>, <span class="author">Rudan I</span>, <span class="author">Johansson A</span>, <span class="author">Marroni F</span>, <span class="author">Hayward C</span>, <span class="author">Vitart V</span>, <span class="author">Jonasson I</span>, <span class="author">Pattaro C</span>, <span class="author">Wright A</span>, <span class="author">Hastie N</span>, <span class="author">Pichler I</span>, <span class="author">Hicks AA</span>, <span class="author">Falchi M</span>, <span class="author">Willemsen G</span>, <span class="author">Hottenga JJ</span>, <span class="author">de Geus EJ</span>, <span class="author">Montgomery GW</span>, <span class="author">Whitfield J</span>, <span class="author">Magnusson P</span>, <span class="author">Saharinen J</span>, <span class="author">Perola M</span>, <span class="author">Silander K</span>, <span class="author">Isaacs A</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Witteman JC</span>, <span class="author">Oostra BA</span>, <span class="author">Elliott P</span>, <span class="author">Ruokonen A</span>, <span class="author">Sabatti C</span>, <span class="author">Gieger C</span>, <span class="author">Meitinger T</span>, <span class="author">Kronenberg F</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Wichmann HE</span>, <span class="author">Smit JH</span>, <span class="author">McCarthy MI</span>, <span class="author">van Duijn CM</span>, <span class="author">Peltonen L</span> and <span class="author">ENGAGE Consortium</span></p>
<p class="affiliation">[1] Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands. [2] These authors contributed equally to this work.</p><p class="abstract">Recent genome-wide association (GWA) studies of lipids have been conducted in samples ascertained for other phenotypes, particularly diabetes. Here we report the first GWA analysis of loci affecting total cholesterol (TC), low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol and triglycerides sampled randomly from 16 population-based cohorts and genotyped using mainly the Illumina HumanHap300-Duo platform. Our study included a total of 17,797-22,562 persons, aged 18-104 years and from geographic regions spanning from the Nordic countries to Southern Europe. We established 22 loci associated with serum lipid levels at a genome-wide significance level (P &lt; 5 x 10(-8)), including 16 loci that were identified by previous GWA studies. The six newly identified loci in our cohort samples are ABCG5 (TC, P = 1.5 x 10(-11); LDL, P = 2.6 x 10(-10)), TMEM57 (TC, P = 5.4 x 10(-10)), CTCF-PRMT8 region (HDL, P = 8.3 x 10(-16)), DNAH11 (LDL, P = 6.1 x 10(-9)), FADS3-FADS2 (TC, P = 1.5 x 10(-10); LDL, P = 4.4 x 10(-13)) and MADD-FOLH1 region (HDL, P = 6 x 10(-11)). For three loci, effect sizes differed significantly by sex. Genetic risk scores based on lipid loci explain up to 4.8% of variation in lipids and were also associated with increased intima media thickness (P = 0.001) and coronary heart disease incidence (P = 0.04). The genetic risk score improves the screening of high-risk groups of dyslipidemia over classical risk factors.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">1</span>;<span class="pagerange">47-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19060911">19060911</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687074">2687074</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.269">10.1038/ng.269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel system of polymorphic and diverse NK cell receptors in primates.</h4>
<p class="authors"><span class="author">Averdam A</span>, <span class="author">Petersen B</span>, <span class="author">Rosner C</span>, <span class="author">Neff J</span>, <span class="author">Roos C</span>, <span class="author">Eberle M</span>, <span class="author">Aujard F</span>, <span class="author">M&uuml;nch C</span>, <span class="author">Schempp W</span>, <span class="author">Carrington M</span>, <span class="author">Shiina T</span>, <span class="author">Inoko H</span>, <span class="author">Knaust F</span>, <span class="author">Coggill P</span>, <span class="author">Sehra H</span>, <span class="author">Beck S</span>, <span class="author">Abi-Rached L</span>, <span class="author">Reinhardt R</span> and <span class="author">Walter L</span></p>
<p class="affiliation">Department of Primate Genetics, German Primate Centre, G&ouml;ttingen, Germany.</p><p class="abstract">There are two main classes of natural killer (NK) cell receptors in mammals, the killer cell immunoglobulin-like receptors (KIR) and the structurally unrelated killer cell lectin-like receptors (KLR). While KIR represent the most diverse group of NK receptors in all primates studied to date, including humans, apes, and Old and New World monkeys, KLR represent the functional equivalent in rodents. Here, we report a first digression from this rule in lemurs, where the KLR (CD94/NKG2) rather than KIR constitute the most diverse group of NK cell receptors. We demonstrate that natural selection contributed to such diversification in lemurs and particularly targeted KLR residues interacting with the peptide presented by MHC class I ligands. We further show that lemurs lack a strict ortholog or functional equivalent of MHC-E, the ligands of non-polymorphic KLR in &quot;higher&quot; primates. Our data support the existence of a hitherto unknown system of polymorphic and diverse NK cell receptors in primates and of combinatorial diversity as a novel mechanism to increase NK cell receptor repertoire.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 31168</span>, <span class="grant">R01 AI031168</span>; PHS HHS: <span class="grant">HHSN261200800001E</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">e1000688</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19834558">19834558</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2757895">2757895</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000688">10.1371/journal.pgen.1000688</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene body methylation of the dimethylarginine dimethylamino-hydrolase 2 (Ddah2) gene is an epigenetic biomarker for neural stem cell differentiation.</h4>
<p class="authors"><span class="author">B&auml;ckdahl L</span>, <span class="author">Herberth M</span>, <span class="author">Wilson G</span>, <span class="author">Tate P</span>, <span class="author">Campos LS</span>, <span class="author">Cortese R</span>, <span class="author">Eckhardt F</span> and <span class="author">Beck S</span></p>
<p class="affiliation">UCL Cancer Institute, University College London, London WC1E 6BT, UK. <a href="mailto:%6C.%62%61%63%6B%64%61%68%6C%40%75%63%6C.%61%63.%75%6B">&#108;&#46;&#98;&#97;&#99;&#107;&#100;&#97;&#104;&#108;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">DNA methylation is an important epigenetic mark that is involved in the regulation of many cellular processes such as gene expression, genomic imprinting and silencing of repetitive elements. Because of their ability to cause and capture phenotypic plasticity, epigenetic marks such as DNA methylation represent potential biomarkers to distinguish between different types of tissues and stages of differentiation. Here, we have identified differential DNA methylation in the gene body of the nitric oxide inhibitor Ddah2 that discriminates embryonic stem cells from neural stem cells and is positively correlated with differential gene expression.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT-084071</span></p>
<p class="publication"><span class="title">Epigenetics : official journal of the DNA Methylation Society</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">4</span>;<span class="pagerange">248-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19535899">19535899</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence of Macrococcus caseolyticus strain JCSCS5402, [corrected] reflecting the ancestral genome of the human-pathogenic staphylococci.</h4>
<p class="authors"><span class="author">Baba T</span>, <span class="author">Kuwahara-Arai K</span>, <span class="author">Uchiyama I</span>, <span class="author">Takeuchi F</span>, <span class="author">Ito T</span> and <span class="author">Hiramatsu K</span></p>
<p class="affiliation">Department of Microbiology and Infection Control Science, Juntendo University, 2-1-1 Hongo, Bunkyo, Tokyo 113-8421, Japan. <a href="mailto:%74%62%61%62%61%40%6A%75%6E%74%65%6E%64%6F.%61%63.%6A%70">&#116;&#98;&#97;&#98;&#97;&#64;&#106;&#117;&#110;&#116;&#101;&#110;&#100;&#111;&#46;&#97;&#99;&#46;&#106;&#112;</a></p><p class="abstract">We isolated the methicillin-resistant Macrococcus caseolyticus strain JCSC5402 from animal meat in a supermarket and determined its whole-genome nucleotide sequence. This is the first report on the genome analysis of a macrococcal species that is evolutionarily closely related to the human pathogens Staphylococcus aureus and Bacillus anthracis. The essential biological pathways of M. caseolyticus are similar to those of staphylococci. However, the species has a small chromosome (2.1 MB) and lacks many sugar and amino acid metabolism pathways and a plethora of virulence genes that are present in S. aureus. On the other hand, M. caseolyticus possesses a series of oxidative phosphorylation machineries that are closely related to those in the family Bacillaceae. We also discovered a probable primordial form of a Macrococcus methicillin resistance gene complex, mecIRAm, on one of the eight plasmids harbored by the M. caseolyticus strain. This is the first finding of a plasmid-encoding methicillin resistance gene. Macrococcus is considered to reflect the genome of ancestral bacteria before the speciation of staphylococcal species and may be closely associated with the origin of the methicillin resistance gene complex of the notorious human pathogen methicillin-resistant S. aureus.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">4</span>;<span class="pagerange">1180-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19074389" rel="external">19074389</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2632007" rel="external">2632007</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01058-08" rel="external">10.1128/JB.01058-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic complexity of the Y-STR DYS19: inversions, deletions and founder lineages carrying duplications.</h4>
<p class="authors"><span class="author">Balaresque P</span>, <span class="author">Parkin EJ</span>, <span class="author">Roewer L</span>, <span class="author">Carvalho-Silva DR</span>, <span class="author">Mitchell RJ</span>, <span class="author">van Oorschot RA</span>, <span class="author">Henke J</span>, <span class="author">Stoneking M</span>, <span class="author">Nasidze I</span>, <span class="author">Wetton J</span>, <span class="author">de Knijff P</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Jobling MA</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, University Road, Leicester, LE1 7RH, UK.</p><p class="abstract">The Y-STR DYS19 is firmly established in the repertoire of Y-chromosomal markers used in forensic analysis yet is poorly understood at the molecular level, lying in a complex genomic environment and exhibiting null alleles, as well as duplications and occasional triplications in population samples. Here, we analyse three null alleles and 51 duplications and show that DYS19 can also be involved in inversion events, so that even its location within the short arm of the Y chromosome is uncertain. Deletion mapping in the three chromosomes carrying null alleles shows that their deletions are less than approximately 300 kb in size. Haplotypic analysis with binary markers shows that they belong to three different haplogroups and so represent independent events. In contrast, a collection of 51 DYS19 duplication chromosomes belong to only four haplogroups: two are singletons and may represent somatic mutation in lymphoblastoid cell lines, but two, in haplogroups G and C3c, represent founder lineages that have spread widely in Central Europe/West Asia and East Asia, respectively. Consideration of candidate mechanisms underlying both deletions and duplications provides no evidence for the involvement of non-allelic homologous recombination, and they are likely to represent sporadic events with low mutation rates. Understanding the basis and population distribution of these DYS19 alleles will aid in the utilisation and interpretation of profiles that contain them.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span>, <span class="grant">077009</span></p>
<p class="publication"><span class="title">International journal of legal medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">123</span>;<span class="number">1</span>;<span class="pagerange">15-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18553096">18553096</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680205">2680205</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00414-008-0253-3">10.1007/s00414-008-0253-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication analysis identifies TYK2 as a multiple sclerosis susceptibility factor.</h4>
<p class="authors"><span class="author">Ban M</span>, <span class="author">Goris A</span>, <span class="author">Lorentzen AR</span>, <span class="author">Baker A</span>, <span class="author">Mihalova T</span>, <span class="author">Ingram G</span>, <span class="author">Booth DR</span>, <span class="author">Heard RN</span>, <span class="author">Stewart GJ</span>, <span class="author">Bogaert E</span>, <span class="author">Dubois B</span>, <span class="author">Harbo HF</span>, <span class="author">Celius EG</span>, <span class="author">Spurkland A</span>, <span class="author">Strange R</span>, <span class="author">Hawkins C</span>, <span class="author">Robertson NP</span>, <span class="author">Dudbridge F</span>, <span class="author">Wason J</span>, <span class="author">De Jager PL</span>, <span class="author">Hafler D</span>, <span class="author">Rioux JD</span>, <span class="author">Ivinson AJ</span>, <span class="author">McCauley JL</span>, <span class="author">Pericak-Vance M</span>, <span class="author">Oksenberg JR</span>, <span class="author">Hauser SL</span>, <span class="author">Sexton D</span>, <span class="author">Haines J</span>, <span class="author">Sawcer S</span>, <span class="author">Wellcome Trust Case-Control Consortium (WTCCC)</span> and <span class="author">Compston A</span></p>
<p class="affiliation">Department of Clinical Neuroscience, Addenbrooke&#39;s, Hospital, University of Cambridge, Cambridge, UK. <a href="mailto:%6D%62%35%33%31%40%6D%65%64%73%63%68%6C.%63%61%6D.%61%63.%75%6B">&#109;&#98;&#53;&#51;&#49;&#64;&#109;&#101;&#100;&#115;&#99;&#104;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In a recent genome-wide association study (GWAS) based on 12,374 non-synonymous single nucleotide polymorphisms we identified a number of candidate multiple sclerosis susceptibility genes. Here, we describe the extended analysis of 17 of these loci undertaken using an additional 4234 patients, 2983 controls and 2053 trio families. In the final analysis combining all available data, we found that evidence for association was substantially increased for one of the 17 loci, rs34536443 from the tyrosine kinase 2 (TYK2) gene (P=2.7 x 10(-6), odds ratio=1.32 (1.17-1.47)). This single nucleotide polymorphism results in an amino acid substitution (proline to alanine) in the kinase domain of TYK2, which is predicted to influence the levels of phosphorylation and therefore activity of the protein and so is likely to have a functional role in multiple sclerosis.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0600329</span>, <span class="grant">G0700061</span>, <span class="grant">MC_U105292688</span>; NINDS NIH HHS: <span class="grant">NS 049477-01A1</span>, <span class="grant">R01 NS049477</span>, <span class="grant">R01 NS049477-01A1</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076113</span>, <span class="grant">085475</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">10</span>;<span class="pagerange">1309-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19293837">19293837</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2782567">2782567</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2009.41">10.1038/ejhg.2009.41</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes.</h4>
<p class="authors"><span class="author">Barrett JC</span>, <span class="author">Clayton DG</span>, <span class="author">Concannon P</span>, <span class="author">Akolkar B</span>, <span class="author">Cooper JD</span>, <span class="author">Erlich HA</span>, <span class="author">Julier C</span>, <span class="author">Morahan G</span>, <span class="author">Nerup J</span>, <span class="author">Nierras C</span>, <span class="author">Plagnol V</span>, <span class="author">Pociot F</span>, <span class="author">Schuilenburg H</span>, <span class="author">Smyth DJ</span>, <span class="author">Stevens H</span>, <span class="author">Todd JA</span>, <span class="author">Walker NM</span>, <span class="author">Rich SS</span> and <span class="author">Type 1 Diabetes Genetics Consortium</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, UK.</p><p class="abstract">Type 1 diabetes (T1D) is a common autoimmune disorder that arises from the action of multiple genetic and environmental risk factors. We report the findings of a genome-wide association study of T1D, combined in a meta-analysis with two previously published studies. The total sample set included 7,514 cases and 9,045 reference samples. Forty-one distinct genomic locations provided evidence for association with T1D in the meta-analysis (P &lt; 10(-6)). After excluding previously reported associations, we further tested 27 regions in an independent set of 4,267 cases, 4,463 controls and 2,319 affected sib-pair (ASP) families. Of these, 18 regions were replicated (P &lt; 0.01; overall P &lt; 5 &times; 10(-8)) and 4 additional regions provided nominal evidence of replication (P &lt; 0.05). The many new candidate genes suggested by these results include IL10, IL19, IL20, GLIS3, CD69 and IL27.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; NIDDK NIH HHS: <span class="grant">DK46635</span>, <span class="grant">K08 DK002876</span>, <span class="grant">K08 DK002876-06</span>, <span class="grant">R01 DK046635</span>, <span class="grant">R01 DK046635-15</span>, <span class="grant">U01 DK062418</span>, <span class="grant">U01 DK062418-06</span>; NIMH NIH HHS: <span class="grant">MH 63420</span>, <span class="grant">MH059565</span>, <span class="grant">MH059571</span>, <span class="grant">MH059588</span>, <span class="grant">MH060879</span>, <span class="grant">MH061675</span>, <span class="grant">MH067257</span>, <span class="grant">MH59566</span>, <span class="grant">MH59586</span>, <span class="grant">MH59587</span>, <span class="grant">MH60870</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">076113</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">703-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19430480">19430480</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2889014">2889014</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.381">10.1038/ng.381</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cloud computing.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Wood M</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">12</span>;<span class="pagerange">1475</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19435745" rel="external">19435745</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp274" rel="external">10.1093/bioinformatics/btp274</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phospholipid scramblases and Tubby-like proteins belong to a new superfamily of membrane tethered transcription factors.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Finn RD</span>, <span class="author">Sims PJ</span>, <span class="author">Wiedmer T</span>, <span class="author">Biegert A</span> and <span class="author">S&ouml;ding J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: Phospholipid scramblases (PLSCRs) constitute a family of cytoplasmic membrane-associated proteins that were identified based upon their capacity to mediate a Ca(2+)-dependent bidirectional movement of phospholipids across membrane bilayers, thereby collapsing the normally asymmetric distribution of such lipids in cell membranes. The exact function and mechanism(s) of these proteins nevertheless remains obscure: data from several laboratories now suggest that in addition to their putative role in mediating transbilayer flip/flop of membrane lipids, the PLSCRs may also function to regulate diverse processes including signaling, apoptosis, cell proliferation and transcription. A major impediment to deducing the molecular details underlying the seemingly disparate biology of these proteins is the current absence of any representative molecular structures to provide guidance to the experimental investigation of their function.</p><p class="abstract"><strong>Results</strong>: Here, we show that the enigmatic PLSCR family of proteins is directly related to another family of cellular proteins with a known structure. The Arabidopsis protein At5g01750 from the DUF567 family was solved by X-ray crystallography and provides the first structural model for this family. This model identifies that the presumed C-terminal transmembrane helix is buried within the core of the PLSCR structure, suggesting that palmitoylation may represent the principal membrane anchorage for these proteins. The fold of the PLSCR family is also shared by Tubby-like proteins. A search of the PDB with the HHpred server suggests a common evolutionary ancestry. Common functional features also suggest that tubby and PLSCR share a functional origin as membrane tethered transcription factors with capacity to modulate phosphoinositide-based signaling.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL036946</span>, <span class="grant">HL063819</span>, <span class="grant">HL076215</span>; Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">2</span>;<span class="pagerange">159-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19010806">19010806</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2639001">2639001</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtn595">10.1093/bioinformatics/btn595</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression screening and annotation of a zebrafish myoblast cDNA library.</h4>
<p class="authors"><span class="author">Baxendale S</span>, <span class="author">Chen CK</span>, <span class="author">Tang H</span>, <span class="author">Davison C</span>, <span class="author">Hateren LV</span>, <span class="author">Croning MD</span>, <span class="author">Humphray SJ</span>, <span class="author">Hubbard SJ</span> and <span class="author">Ingham PW</span></p>
<p class="affiliation">MRC Centre for Developmental and Biomedical Genetics, University of Sheffield, Sheffield S10 2TN, UK.</p><p class="abstract">To analyse the myogenic transcriptome and identify novel genes involved in muscle development in an in vivo context, we have constructed a muscle specific cDNA library from GFP-expressing myoblasts purified by fluorescent activated cell sorting of transgenic zebrafish embryos. We have generated 153,428 EST sequences from this library that have been clustered into consensi, mapped to the genome assembly Zv6 and analysed for protein homology. Expression analysis of a randomly picked sample of clones using whole mount in situ hybridisation, identified 30 genes that are expressed specifically within the myotome, one third of which represent novel sequences. These genes have been assigned to syn-expression groups. The sequencing of the myoblast enriched cDNA library has significantly increased the number of zebrafish ESTs, facilitating the prediction of new spliced transcripts in the genome assembly and providing a transcriptome of an in vivo myoblast cell.</p>
<p class="publication"><span class="title">Gene expression patterns : GEP</span>&nbsp;<span class="year">2009</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">73-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19007914" rel="external">19007914</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gep.2008.10.003" rel="external">10.1016/j.gep.2008.10.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neuroproteomics: understanding the molecular organization and complexity of the brain.</h4>
<p class="authors"><span class="author">Bay&eacute;s A</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Advances in technology have equipped the field of neuroproteomics with refined tools for the study of the expression, interaction and function of proteins in the nervous system. In combination with bioinformatics, neuroproteomics can address the organization of dynamic, functional protein networks and macromolecular structures that underlie physiological, anatomical and behavioural processes. Furthermore, neuroproteomics is contributing to the elucidation of disease mechanisms and is a powerful tool for the identification of biomarkers.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Nature reviews. Neuroscience</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="number">9</span>;<span class="pagerange">635-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19693028">19693028</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrn2701">10.1038/nrn2701</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction of Salmonella enterica with basil and other salad leaves.</h4>
<p class="authors"><span class="author">Berger CN</span>, <span class="author">Shaw RK</span>, <span class="author">Brown DJ</span>, <span class="author">Mather H</span>, <span class="author">Clare S</span>, <span class="author">Dougan G</span>, <span class="author">Pallen MJ</span> and <span class="author">Frankel G</span></p>
<p class="affiliation">Department of Life Science, Division of Cell and Molecular Biology, Imperial College London, London, UK.</p>
<p class="abstract">Contaminated salad leaves have emerged as important vehicles for the transmission of enteric pathogens to humans. A recent outbreak of Salmonella enterica serovar Senftenberg (S. Senftenberg) in the United Kingdom has been traced to the consumption of contaminated basil. Using the outbreak strain of S. Senftenberg, we found that it binds to basil, lettuce, rocket and spinach leaves showing a pattern of diffuse adhesion. Flagella were seen linking S. Senftenberg to the leaf epidermis, and the deletion of fliC (encoding phase-1 flagella) resulted in a significantly reduced level of adhesion. In contrast, although flagella linking S. enterica serovar Typhimurium to the basil leaf epidermis were widespread, deletion of fliC did not affect leaf attachment levels. These results implicate the role of flagella in Salmonella leaf attachment and suggest that different Salmonella serovars use strain-specific mechanisms to attach to salad leaves.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council</p>
<p class="publication"><span class="title">The ISME journal</span>&nbsp;<span class="year">2009</span>;<span class="volume">3</span>;<span class="number">2</span>;<span class="pagerange">261-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18830276" rel="external">18830276</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fismej.2008.95" rel="external">10.1038/ismej.2008.95</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the blood fluke Schistosoma mansoni.</h4>
<p class="authors"><span class="author">Berriman M</span>, <span class="author">Haas BJ</span>, <span class="author">LoVerde PT</span>, <span class="author">Wilson RA</span>, <span class="author">Dillon GP</span>, <span class="author">Cerqueira GC</span>, <span class="author">Mashiyama ST</span>, <span class="author">Al-Lazikani B</span>, <span class="author">Andrade LF</span>, <span class="author">Ashton PD</span>, <span class="author">Aslett MA</span>, <span class="author">Bartholomeu DC</span>, <span class="author">Blandin G</span>, <span class="author">Caffrey CR</span>, <span class="author">Coghlan A</span>, <span class="author">Coulson R</span>, <span class="author">Day TA</span>, <span class="author">Delcher A</span>, <span class="author">DeMarco R</span>, <span class="author">Djikeng A</span>, <span class="author">Eyre T</span>, <span class="author">Gamble JA</span>, <span class="author">Ghedin E</span>, <span class="author">Gu Y</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Hirai H</span>, <span class="author">Hirai Y</span>, <span class="author">Houston R</span>, <span class="author">Ivens A</span>, <span class="author">Johnston DA</span>, <span class="author">Lacerda D</span>, <span class="author">Macedo CD</span>, <span class="author">McVeigh P</span>, <span class="author">Ning Z</span>, <span class="author">Oliveira G</span>, <span class="author">Overington JP</span>, <span class="author">Parkhill J</span>, <span class="author">Pertea M</span>, <span class="author">Pierce RJ</span>, <span class="author">Protasio AV</span>, <span class="author">Quail MA</span>, <span class="author">Rajandream MA</span>, <span class="author">Rogers J</span>, <span class="author">Sajid M</span>, <span class="author">Salzberg SL</span>, <span class="author">Stanke M</span>, <span class="author">Tivey AR</span>, <span class="author">White O</span>, <span class="author">Williams DL</span>, <span class="author">Wortman J</span>, <span class="author">Wu W</span>, <span class="author">Zamanian M</span>, <span class="author">Zerlotini A</span>, <span class="author">Fraser-Liggett CM</span>, <span class="author">Barrell BG</span> and <span class="author">El-Sayed NM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge CB10 1SD, UK. <a href="mailto:%6D%62%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#98;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Schistosoma mansoni is responsible for the neglected tropical disease schistosomiasis that affects 210 million people in 76 countries. Here we present analysis of the 363 megabase nuclear genome of the blood fluke. It encodes at least 11,809 genes, with an unusual intron size distribution, and new families of micro-exon genes that undergo frequent alternative splicing. As the first sequenced flatworm, and a representative of the Lophotrochozoa, it offers insights into early events in the evolution of the animals, including the development of a body pattern with bilateral symmetry, and the development of tissues into organs. Our analysis has been informed by the need to find new drug targets. The deficits in lipid metabolism that make schistosomes dependent on the host are revealed, and the identification of membrane receptors, ion channels and more than 300 proteases provide new insights into the biology of the life cycle and new targets. Bioinformatics approaches have identified metabolic chokepoints, and a chemogenomic screen has pinpointed schistosome proteins for which existing drugs may be active. The information generated provides an invaluable resource for the research community to develop much needed new control tools for the treatment and eradication of this important and neglected disease.</p>
<p class="grants">Funded by: FIC NIH HHS: <span class="grant">5D43TW006580</span>, <span class="grant">5D43TW007012-03</span>; NIAID NIH HHS: <span class="grant">AI054711-01A2</span>, <span class="grant">AI48828</span>, <span class="grant">U01 AI048828</span>, <span class="grant">U01 AI048828-01</span>, <span class="grant">U01 AI048828-02</span>; NIGMS NIH HHS: <span class="grant">R01 GM083873</span>, <span class="grant">R01 GM083873-07</span>, <span class="grant">R01 GM083873-08</span>; NLM NIH HHS: <span class="grant">R01 LM006845</span>, <span class="grant">R01 LM006845-08</span>, <span class="grant">R01 LM006845-09</span>; Wellcome Trust: <span class="grant">086151</span>, <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2009</span>;<span class="volume">460</span>;<span class="number">7253</span>;<span class="pagerange">352-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19606141">19606141</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2756445">2756445</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08160">10.1038/nature08160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and phenotypic variation in epidemic-spanning Salmonella enterica serovar Enteritidis isolates.</h4>
<p class="authors"><span class="author">Betancor L</span>, <span class="author">Yim L</span>, <span class="author">Fookes M</span>, <span class="author">Martinez A</span>, <span class="author">Thomson NR</span>, <span class="author">Ivens A</span>, <span class="author">Peters S</span>, <span class="author">Bryant C</span>, <span class="author">Algorta G</span>, <span class="author">Kariuki S</span>, <span class="author">Schelotto F</span>, <span class="author">Maskell D</span>, <span class="author">Dougan G</span> and <span class="author">Chabalgoity JA</span></p>
<p class="affiliation">Departamento de Desarrollo Biotecnol&oacute;gico, Instituto de Higiene, Universidad de la Rep&uacute;blica, Montevideo, Uruguay. <a href="mailto:%6C%61%75%72%61%62%65%74%40%68%69%67%69%65%6E%65.%65%64%75.%75%79">&#108;&#97;&#117;&#114;&#97;&#98;&#101;&#116;&#64;&#104;&#105;&#103;&#105;&#101;&#110;&#101;&#46;&#101;&#100;&#117;&#46;&#117;&#121;</a></p><p class="abstract"><strong>Background</strong>: Salmonella enterica serovar Enteritidis (S. Enteritidis) has caused major epidemics of gastrointestinal infection in many different countries. In this study we investigate genome divergence and pathogenic potential in S. Enteritidis isolated before, during and after an epidemic in Uruguay.</p><p class="abstract"><strong>Results</strong>: 266 S. Enteritidis isolates were genotyped using RAPD-PCR and a selection were subjected to PFGE analysis. From these, 29 isolates spanning different periods, genetic profiles and sources of isolation were assayed for their ability to infect human epithelial cells and subjected to comparative genomic hybridization using a Salmonella pan-array and the sequenced strain S. Enteritidis PT4 P125109 as reference. Six other isolates from distant countries were included as external comparators.Two hundred and thirty three chromosomal genes as well as the virulence plasmid were found as variable among S. Enteritidis isolates. Ten out of the 16 chromosomal regions that varied between different isolates correspond to phage-like regions. The 2 oldest pre-epidemic isolates lack phage SE20 and harbour other phage encoded genes that are absent in the sequenced strain. Besides variation in prophage, we found variation in genes involved in metabolism and bacterial fitness. Five epidemic strains lack the complete Salmonella virulence plasmid. Significantly, strains with indistinguishable genetic patterns still showed major differences in their ability to infect epithelial cells, indicating that the approach used was insufficient to detect the genetic basis of this differential behaviour.</p><p class="abstract"><strong>Conclusion</strong>: The recent epidemic of S. Enteritidis infection in Uruguay has been driven by the introduction of closely related strains of phage type 4 lineage. Our results confirm previous reports demonstrating a high degree of genetic homogeneity among S. Enteritidis isolates. However, 10 of the regions of variability described here are for the first time reported as being variable in S. Enteritidis. In particular, the oldest pre-epidemic isolates carry phage-associated genetic regions not previously reported in S. Enteritidis. Overall, our results support the view that phages play a crucial role in the generation of genetic diversity in S. Enteritidis and that phage SE20 may be a key marker for the emergence of particular isolates capable of causing epidemics.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">078168/Z/05/Z</span></p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">9</span>;<span class="pagerange">237</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19922635" rel="external">19922635</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2784474" rel="external">2784474</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-9-237" rel="external">10.1186/1471-2180-9-237</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Public health. The cholera crisis in Africa.</h4>
<p class="authors"><span class="author">Bhattacharya S</span>, <span class="author">Black R</span>, <span class="author">Bourgeois L</span>, <span class="author">Clemens J</span>, <span class="author">Cravioto A</span>, <span class="author">Deen JL</span>, <span class="author">Dougan G</span>, <span class="author">Glass R</span>, <span class="author">Grais RF</span>, <span class="author">Greco M</span>, <span class="author">Gust I</span>, <span class="author">Holmgren J</span>, <span class="author">Kariuki S</span>, <span class="author">Lambert PH</span>, <span class="author">Liu MA</span>, <span class="author">Longini I</span>, <span class="author">Nair GB</span>, <span class="author">Norrby R</span>, <span class="author">Nossal GJ</span>, <span class="author">Ogra P</span>, <span class="author">Sansonetti P</span>, <span class="author">von Seidlein L</span>, <span class="author">Songane F</span>, <span class="author">Svennerholm AM</span>, <span class="author">Steele D</span> and <span class="author">Walker R</span></p>
<p class="affiliation">Indian Council of Medical Research, Ansari Nagore, New Delhi, 110029, India.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">324</span>;<span class="number">5929</span>;<span class="pagerange">885</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19443768">19443768</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1173890">10.1126/science.1173890</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Calcium-dependent signaling and kinases in apicomplexan parasites.</h4>
<p class="authors"><span class="author">Billker O</span>, <span class="author">Lourido S</span> and <span class="author">Sibley LD</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Calcium controls many critical events in the complex life cycles of apicomplexan parasites including protein secretion, motility, and development. Calcium levels are normally tightly regulated and rapid release of calcium into the cytosol activates a family of calcium-dependent protein kinases (CDPKs), which are normally characteristic of plants. CDPKs present in apicomplexans have acquired a number of unique domain structures likely reflecting their diverse functions. Calcium regulation in parasites is closely linked to signaling by cyclic nucleotides and their associated kinases. This Review summarizes the pivotal roles that calcium- and cyclic nucleotide-dependent kinases play in unique aspects of parasite biology.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; NIAID NIH HHS: <span class="grant">AI34036</span>, <span class="grant">R01 AI034036</span>, <span class="grant">R01 AI034036-17</span>, <span class="grant">R01 AI082423</span>, <span class="grant">R01 AI082423-01</span>, <span class="grant">R21 AI067051</span></p>
<p class="publication"><span class="title">Cell host &amp; microbe</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">612-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19527888">19527888</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2718762">2718762</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.chom.2009.05.017">10.1016/j.chom.2009.05.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A mouse chromosome 4 balancer ENU-mutagenesis screen isolates eleven lethal lines.</h4>
<p class="authors"><span class="author">Boles MK</span>, <span class="author">Wilkinson BM</span>, <span class="author">Maxwell A</span>, <span class="author">Lai L</span>, <span class="author">Mills AA</span>, <span class="author">Nishijima I</span>, <span class="author">Salinger AP</span>, <span class="author">Moskowitz I</span>, <span class="author">Hirschi KK</span>, <span class="author">Liu B</span>, <span class="author">Bradley A</span> and <span class="author">Justice MJ</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA. <a href="mailto:%6D%62%31%34%34%30%37%30%40%62%63%6D.%74%6D%63.%65%64%75">&#109;&#98;&#49;&#52;&#52;&#48;&#55;&#48;&#64;&#98;&#99;&#109;&#46;&#116;&#109;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: ENU-mutagenesis is a powerful technique to identify genes regulating mammalian development. To functionally annotate the distal region of mouse chromosome 4, we performed an ENU-mutagenesis screen using a balancer chromosome targeted to this region of the genome.</p><p class="abstract"><strong>Results</strong>: We isolated 11 lethal lines that map to the region of chromosome 4 between D4Mit117 and D4Mit281. These lines form 10 complementation groups. The majority of lines die during embryonic development between E5.5 and E12.5 and display defects in gastrulation, cardiac development, and craniofacial development. One line displayed postnatal lethality and neurological defects, including ataxia and seizures.</p><p class="abstract"><strong>Conclusion</strong>: These eleven mutants allow us to query gene function within the distal region of mouse chromosome 4 and demonstrate that new mouse models of mammalian developmental defects can easily and quickly be generated and mapped with the use of ENU-mutagenesis in combination with balancer chromosomes. The low number of mutations isolated in this screen compared with other balancer chromosome screens indicates that the functions of genes in different regions of the genome vary widely.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01 CA115503</span>, <span class="grant">R01 CA115503-01A1</span>; NHLBI NIH HHS: <span class="grant">R01 HL76260</span>; NICHD NIH HHS: <span class="grant">U01 HD39372</span></p>
<p class="publication"><span class="title">BMC genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19267930" rel="external">19267930</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2670824" rel="external">2670824</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2156-10-12" rel="external">10.1186/1471-2156-10-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mosaic 22q13 deletions: evidence for concurrent mosaic segmental isodisomy and gene conversion.</h4>
<p class="authors"><span class="author">Bonaglia MC</span>, <span class="author">Giorda R</span>, <span class="author">Beri S</span>, <span class="author">Bigoni S</span>, <span class="author">Sensi A</span>, <span class="author">Baroncini A</span>, <span class="author">Capucci A</span>, <span class="author">De Agostini C</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">Dunham I</span> and <span class="author">Zuffardi O</span></p>
<p class="affiliation">Eugenio Medea Scientific Institute, Bosisio Parini, Lecco, Italy. <a href="mailto:%63%6C%61%72%61.%62%6F%6E%61%67%6C%69%61%40%62%70.%6C%6E%66.%69%74">&#99;&#108;&#97;&#114;&#97;&#46;&#98;&#111;&#110;&#97;&#103;&#108;&#105;&#97;&#64;&#98;&#112;&#46;&#108;&#110;&#102;&#46;&#105;&#116;</a></p><p class="abstract">Although 22q terminal deletions are well documented, very few patients with mosaicism have been reported. We describe two new cases with mosaic 22q13.2-qter deletion, detected by karyotype analysis, showing the neurological phenotype of 22q13.3 deletion syndrome. Case 1 represents an exceptional case of mosaicism for maternal 22q13.2-qter deletion (45% of cells) and 22q13.2-qter paternal segmental isodisomy (55% of cells). This complex situation was suspected because cytogenetic, FISH and array-CGH analyses showed the presence of an 8.8 Mb mosaic 22q13.2-qter deletion, whereas microsatellite marker analysis was consistent with maternal deletion without any evidence of mosaic deletion. Molecular analysis led to the definition of very close, but not coincident, deletion and uniparental disomy (UPD) break points. Furthermore, we demonstrated that the segmental UPD arose by gene conversion in the same region. In Case 2, mosaicism for a paternal 8.9 Mb 22q13.2-qter deletion (73% of cells) was detected. In both patients, the level of mosaicism was also verified in saliva samples. We propose possible causative mechanisms for both rearrangements. Although the size of the deletions was quite similar, the phenotype was more severe in Case 2 than in Case 1. As maternal UPD 22 has not been generally associated with any defects and as the size of the deletion is very similar in the two cases, phenotype severity is likely to depend entirely on the degree of mosaicism in each individual.</p>
<p class="grants">Funded by: Telethon: <span class="grant">GGP06208</span>; Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">4</span>;<span class="pagerange">426-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18854866" rel="external">18854866</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682770" rel="external">2682770</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2008.195" rel="external">10.1038/ejhg.2008.195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Family-based analysis of tumor necrosis factor and lymphotoxin-alpha tag polymorphisms with type 1 diabetes in the population of South Croatia.</h4>
<p class="authors"><span class="author">Boraska V</span>, <span class="author">Zeggini E</span>, <span class="author">Groves CJ</span>, <span class="author">Rayner NW</span>, <span class="author">Skrabi&#x107; V</span>, <span class="author">Diakite M</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski D</span>, <span class="author">McCarthy MI</span> and <span class="author">Zemunik T</span></p>
<p class="affiliation">Department of Medical Biology, Medical School, University of Split, Split, Croatia. <a href="mailto:%76%62%6F%72%61%73%6B%61%40%6D%65%66%73%74.%68%72">&#118;&#98;&#111;&#114;&#97;&#115;&#107;&#97;&#64;&#109;&#101;&#102;&#115;&#116;&#46;&#104;&#114;</a></p><p class="abstract">Tumor necrosis factor (TNF) and lymphotoxin-alpha (LTA) are cytokines with a wide range of inflammatory and immunomodulatory activities. Type 1 diabetes is an autoimmune disease characterized by destruction of insulin-producing pancreatic beta cells. The aim of the present study was to evaluate the association of polymorphisms in the TNF/LTA gene region with susceptibility to type 1 diabetes. We investigated 11 TNF/LTA tag polymorphisms, designed to capture the majority of common variation in the region, in 160 trio families from South Croatia. We observed overtransmission of alleles from parents to affected child at five variants: (rs909253, allele C, p = 1.2x10(-4); rs1041981, allele A, p = 1.1x10(-4); rs1800629 (G-308A), allele A, p = 1.2x10(-4); rs361525 (G-238A), allele G, p = 8.2x10(-3) and rs3093668, allele G, p = 0.014). We also identified overtransmission of the rs1800629(G-308A)-rs361525(G-238A) A-G haplotype, p = 2.384x10(-5). The present study found an association of the TNF/LTA gene region with type 1 diabetes. A careful assessment of TNF/LTA variants adjusted for linkage disequilibrium with HLA loci is needed to further clarify the role of these genes in type 1 diabetes susceptibility in the population of South Croatia.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079557</span>, <span class="grant">082370</span>, <span class="grant">088885</span></p>
<p class="publication"><span class="title">Human immunology</span>&nbsp;<span class="year">2009</span>;<span class="volume">70</span>;<span class="number">3</span>;<span class="pagerange">195-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19167443" rel="external">19167443</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2709221" rel="external">2709221</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.humimm.2008.12.010" rel="external">10.1016/j.humimm.2008.12.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Catweasel mice: a novel role for Six1 in sensory patch development and a model for branchio-oto-renal syndrome.</h4>
<p class="authors"><span class="author">Bosman EA</span>, <span class="author">Quint E</span>, <span class="author">Fuchs H</span>, <span class="author">Hrab&eacute; de Angelis M</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">Large-scale mouse mutagenesis initiatives have provided new mouse mutants that are useful models of human deafness and vestibular dysfunction. Catweasel is a novel N-ethyl-N-nitrosourea (ENU)-induced mutation. Heterozygous catweasel mutant mice exhibit mild headtossing associated with a posterior crista defect. We mapped the catweasel mutation to a critical region of 13 Mb on chromosome 12 containing the Six1, -4 and -6 genes. We identified a basepair substitution in exon 1 of the Six1 gene that changes a conserved glutamic acid (E) at position 121 to a glycine (G) in the Six1 homeodomain. Cwe/Cwe animals lack Preyer and righting reflexes, display severe headshaking and have severely truncated cochlea and semicircular canals. Cwe/Cwe animals had very few hair cells in the utricle, but their ampullae and cochlea were devoid of any hair cells. Bmp4, Jag1 and Sox2 expression were largely absent at early stages of sensory development and NeuroD expression was reduced in the developing vestibulo-acoustic ganglion. Lastly we show that Six1 genetically interacts with Jag1. We propose that the catweasel phenotype is due to a hypomorphic mutation in Six1 and that catweasel mice are a suitable model for branchio-oto-renal syndrome. In addition Six1 has a pivotal role in early sensory patch development and may act in the same genetic pathway as Jag1.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">328</span>;<span class="number">2</span>;<span class="pagerange">285-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19389353" rel="external">19389353</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682643" rel="external">2682643</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2009.01.030" rel="external">10.1016/j.ydbio.2009.01.030</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">IRS2 variants and syndromes of severe insulin resistance.</h4>
<p class="authors"><span class="author">Bottomley WE</span>, <span class="author">Soos MA</span>, <span class="author">Adams C</span>, <span class="author">Guran T</span>, <span class="author">Howlett TA</span>, <span class="author">Mackie A</span>, <span class="author">Miell J</span>, <span class="author">Monson JP</span>, <span class="author">Temple R</span>, <span class="author">Tenenbaum-Rakover Y</span>, <span class="author">Tymms J</span>, <span class="author">Savage DB</span>, <span class="author">Semple RK</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Barroso I</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">078986</span>, <span class="grant">078986/Z/06/Z</span>, <span class="grant">080952</span>, <span class="grant">080952/Z/06/Z</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2009</span>;<span class="volume">52</span>;<span class="number">6</span>;<span class="pagerange">1208-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19377890">19377890</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680062">2680062</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-009-1345-4">10.1007/s00125-009-1345-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of taurine cattle: a window to ruminant biology and evolution.</h4>
<p class="authors"><span class="author">Bovine Genome Sequencing and Analysis Consortium</span>, <span class="author">Elsik CG</span>, <span class="author">Tellam RL</span>, <span class="author">Worley KC</span>, <span class="author">Gibbs RA</span>, <span class="author">Muzny DM</span>, <span class="author">Weinstock GM</span>, <span class="author">Adelson DL</span>, <span class="author">Eichler EE</span>, <span class="author">Elnitski L</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Hamernik DL</span>, <span class="author">Kappes SM</span>, <span class="author">Lewin HA</span>, <span class="author">Lynn DJ</span>, <span class="author">Nicholas FW</span>, <span class="author">Reymond A</span>, <span class="author">Rijnkels M</span>, <span class="author">Skow LC</span>, <span class="author">Zdobnov EM</span>, <span class="author">Schook L</span>, <span class="author">Womack J</span>, <span class="author">Alioto T</span>, <span class="author">Antonarakis SE</span>, <span class="author">Astashyn A</span>, <span class="author">Chapple CE</span>, <span class="author">Chen HC</span>, <span class="author">Chrast J</span>, <span class="author">C&acirc;mara F</span>, <span class="author">Ermolaeva O</span>, <span class="author">Henrichsen CN</span>, <span class="author">Hlavina W</span>, <span class="author">Kapustin Y</span>, <span class="author">Kiryutin B</span>, <span class="author">Kitts P</span>, <span class="author">Kokocinski F</span>, <span class="author">Landrum M</span>, <span class="author">Maglott D</span>, <span class="author">Pruitt K</span>, <span class="author">Sapojnikov V</span>, <span class="author">Searle SM</span>, <span class="author">Solovyev V</span>, <span class="author">Souvorov A</span>, <span class="author">Ucla C</span>, <span class="author">Wyss C</span>, <span class="author">Anzola JM</span>, <span class="author">Gerlach D</span>, <span class="author">Elhaik E</span>, <span class="author">Graur D</span>, <span class="author">Reese JT</span>, <span class="author">Edgar RC</span>, <span class="author">McEwan JC</span>, <span class="author">Payne GM</span>, <span class="author">Raison JM</span>, <span class="author">Junier T</span>, <span class="author">Kriventseva EV</span>, <span class="author">Eyras E</span>, <span class="author">Plass M</span>, <span class="author">Donthu R</span>, <span class="author">Larkin DM</span>, <span class="author">Reecy J</span>, <span class="author">Yang MQ</span>, <span class="author">Chen L</span>, <span class="author">Cheng Z</span>, <span class="author">Chitko-McKown CG</span>, <span class="author">Liu GE</span>, <span class="author">Matukumalli LK</span>, <span class="author">Song J</span>, <span class="author">Zhu B</span>, <span class="author">Bradley DG</span>, <span class="author">Brinkman FS</span>, <span class="author">Lau LP</span>, <span class="author">Whiteside MD</span>, <span class="author">Walker A</span>, <span class="author">Wheeler TT</span>, <span class="author">Casey T</span>, <span class="author">German JB</span>, <span class="author">Lemay DG</span>, <span class="author">Maqbool NJ</span>, <span class="author">Molenaar AJ</span>, <span class="author">Seo S</span>, <span class="author">Stothard P</span>, <span class="author">Baldwin CL</span>, <span class="author">Baxter R</span>, <span class="author">Brinkmeyer-Langford CL</span>, <span class="author">Brown WC</span>, <span class="author">Childers CP</span>, <span class="author">Connelley T</span>, <span class="author">Ellis SA</span>, <span class="author">Fritz K</span>, <span class="author">Glass EJ</span>, <span class="author">Herzig CT</span>, <span class="author">Iivanainen A</span>, <span class="author">Lahmers KK</span>, <span class="author">Bennett AK</span>, <span class="author">Dickens CM</span>, <span class="author">Gilbert JG</span>, <span class="author">Hagen DE</span>, <span class="author">Salih H</span>, <span class="author">Aerts J</span>, <span class="author">Caetano AR</span>, <span class="author">Dalrymple B</span>, <span class="author">Garcia JF</span>, <span class="author">Gill CA</span>, <span class="author">Hiendleder SG</span>, <span class="author">Memili E</span>, <span class="author">Spurlock D</span>, <span class="author">Williams JL</span>, <span class="author">Alexander L</span>, <span class="author">Brownstein MJ</span>, <span class="author">Guan L</span>, <span class="author">Holt RA</span>, <span class="author">Jones SJ</span>, <span class="author">Marra MA</span>, <span class="author">Moore R</span>, <span class="author">Moore SS</span>, <span class="author">Roberts A</span>, <span class="author">Taniguchi M</span>, <span class="author">Waterman RC</span>, <span class="author">Chacko J</span>, <span class="author">Chandrabose MM</span>, <span class="author">Cree A</span>, <span class="author">Dao MD</span>, <span class="author">Dinh HH</span>, <span class="author">Gabisi RA</span>, <span class="author">Hines S</span>, <span class="author">Hume J</span>, <span class="author">Jhangiani SN</span>, <span class="author">Joshi V</span>, <span class="author">Kovar CL</span>, <span class="author">Lewis LR</span>, <span class="author">Liu YS</span>, <span class="author">Lopez J</span>, <span class="author">Morgan MB</span>, <span class="author">Nguyen NB</span>, <span class="author">Okwuonu GO</span>, <span class="author">Ruiz SJ</span>, <span class="author">Santibanez J</span>, <span class="author">Wright RA</span>, <span class="author">Buhay C</span>, <span class="author">Ding Y</span>, <span class="author">Dugan-Rocha S</span>, <span class="author">Herdandez J</span>, <span class="author">Holder M</span>, <span class="author">Sabo A</span>, <span class="author">Egan A</span>, <span class="author">Goodell J</span>, <span class="author">Wilczek-Boney K</span>, <span class="author">Fowler GR</span>, <span class="author">Hitchens ME</span>, <span class="author">Lozado RJ</span>, <span class="author">Moen C</span>, <span class="author">Steffen D</span>, <span class="author">Warren JT</span>, <span class="author">Zhang J</span>, <span class="author">Chiu R</span>, <span class="author">Schein JE</span>, <span class="author">Durbin KJ</span>, <span class="author">Havlak P</span>, <span class="author">Jiang H</span>, <span class="author">Liu Y</span>, <span class="author">Qin X</span>, <span class="author">Ren Y</span>, <span class="author">Shen Y</span>, <span class="author">Song H</span>, <span class="author">Bell SN</span>, <span class="author">Davis C</span>, <span class="author">Johnson AJ</span>, <span class="author">Lee S</span>, <span class="author">Nazareth LV</span>, <span class="author">Patel BM</span>, <span class="author">Pu LL</span>, <span class="author">Vattathil S</span>, <span class="author">Williams RL</span>, <span class="author">Curry S</span>, <span class="author">Hamilton C</span>, <span class="author">Sodergren E</span>, <span class="author">Wheeler DA</span>, <span class="author">Barris W</span>, <span class="author">Bennett GL</span>, <span class="author">Eggen A</span>, <span class="author">Green RD</span>, <span class="author">Harhay GP</span>, <span class="author">Hobbs M</span>, <span class="author">Jann O</span>, <span class="author">Keele JW</span>, <span class="author">Kent MP</span>, <span class="author">Lien S</span>, <span class="author">McKay SD</span>, <span class="author">McWilliam S</span>, <span class="author">Ratnakumar A</span>, <span class="author">Schnabel RD</span>, <span class="author">Smith T</span>, <span class="author">Snelling WM</span>, <span class="author">Sonstegard TS</span>, <span class="author">Stone RT</span>, <span class="author">Sugimoto Y</span>, <span class="author">Takasuga A</span>, <span class="author">Taylor JF</span>, <span class="author">Van Tassell CP</span>, <span class="author">Macneil MD</span>, <span class="author">Abatepaulo AR</span>, <span class="author">Abbey CA</span>, <span class="author">Ahola V</span>, <span class="author">Almeida IG</span>, <span class="author">Amadio AF</span>, <span class="author">Anatriello E</span>, <span class="author">Bahadue SM</span>, <span class="author">Biase FH</span>, <span class="author">Boldt CR</span>, <span class="author">Carroll JA</span>, <span class="author">Carvalho WA</span>, <span class="author">Cervelatti EP</span>, <span class="author">Chacko E</span>, <span class="author">Chapin JE</span>, <span class="author">Cheng Y</span>, <span class="author">Choi J</span>, <span class="author">Colley AJ</span>, <span class="author">de Campos TA</span>, <span class="author">De Donato M</span>, <span class="author">Santos IK</span>, <span class="author">de Oliveira CJ</span>, <span class="author">Deobald H</span>, <span class="author">Devinoy E</span>, <span class="author">Donohue KE</span>, <span class="author">Dovc P</span>, <span class="author">Eberlein A</span>, <span class="author">Fitzsimmons CJ</span>, <span class="author">Franzin AM</span>, <span class="author">Garcia GR</span>, <span class="author">Genini S</span>, <span class="author">Gladney CJ</span>, <span class="author">Grant JR</span>, <span class="author">Greaser ML</span>, <span class="author">Green JA</span>, <span class="author">Hadsell DL</span>, <span class="author">Hakimov HA</span>, <span class="author">Halgren R</span>, <span class="author">Harrow JL</span>, <span class="author">Hart EA</span>, <span class="author">Hastings N</span>, <span class="author">Hernandez M</span>, <span class="author">Hu ZL</span>, <span class="author">Ingham A</span>, <span class="author">Iso-Touru T</span>, <span class="author">Jamis C</span>, <span class="author">Jensen K</span>, <span class="author">Kapetis D</span>, <span class="author">Kerr T</span>, <span class="author">Khalil SS</span>, <span class="author">Khatib H</span>, <span class="author">Kolbehdari D</span>, <span class="author">Kumar CG</span>, <span class="author">Kumar D</span>, <span class="author">Leach R</span>, <span class="author">Lee JC</span>, <span class="author">Li C</span>, <span class="author">Logan KM</span>, <span class="author">Malinverni R</span>, <span class="author">Marques E</span>, <span class="author">Martin WF</span>, <span class="author">Martins NF</span>, <span class="author">Maruyama SR</span>, <span class="author">Mazza R</span>, <span class="author">McLean KL</span>, <span class="author">Medrano JF</span>, <span class="author">Moreno BT</span>, <span class="author">Mor&eacute; DD</span>, <span class="author">Muntean CT</span>, <span class="author">Nandakumar HP</span>, <span class="author">Nogueira MF</span>, <span class="author">Olsaker I</span>, <span class="author">Pant SD</span>, <span class="author">Panzitta F</span>, <span class="author">Pastor RC</span>, <span class="author">Poli MA</span>, <span class="author">Poslusny N</span>, <span class="author">Rachagani S</span>, <span class="author">Ranganathan S</span>, <span class="author">Razpet A</span>, <span class="author">Riggs PK</span>, <span class="author">Rincon G</span>, <span class="author">Rodriguez-Osorio N</span>, <span class="author">Rodriguez-Zas SL</span>, <span class="author">Romero NE</span>, <span class="author">Rosenwald A</span>, <span class="author">Sando L</span>, <span class="author">Schmutz SM</span>, <span class="author">Shen L</span>, <span class="author">Sherman L</span>, <span class="author">Southey BR</span>, <span class="author">Lutzow YS</span>, <span class="author">Sweedler JV</span>, <span class="author">Tammen I</span>, <span class="author">Telugu BP</span>, <span class="author">Urbanski JM</span>, <span class="author">Utsunomiya YT</span>, <span class="author">Verschoor CP</span>, <span class="author">Waardenberg AJ</span>, <span class="author">Wang Z</span>, <span class="author">Ward R</span>, <span class="author">Weikard R</span>, <span class="author">Welsh TH</span>, <span class="author">White SN</span>, <span class="author">Wilming LG</span>, <span class="author">Wunderlich KR</span>, <span class="author">Yang J</span> and <span class="author">Zhao FQ</span></p><p class="abstract">To understand the biology and evolution of ruminants, the cattle genome was sequenced to about sevenfold coverage. The cattle genome contains a minimum of 22,000 genes, with a core set of 14,345 orthologs shared among seven mammalian species of which 1217 are absent or undetected in noneutherian (marsupial or monotreme) genomes. Cattle-specific evolutionary breakpoint regions in chromosomes have a higher density of segmental duplications, enrichment of repetitive elements, and species-specific variations in genes associated with lactation and immune responsiveness. Genes involved in metabolism are generally highly conserved, although five metabolic genes are deleted or extensively diverged from their human orthologs. The cattle genome sequence thus provides a resource for understanding mammalian evolution and accelerating livestock genetic improvement for milk and meat production.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/B/13438</span>, <span class="grant">BBS/B/13446</span>; NHGRI NIH HHS: <span class="grant">U54 HG003273</span>, <span class="grant">U54 HG003273-04</span>, <span class="grant">U54 HG003273-04S1</span>, <span class="grant">U54 HG003273-05</span>, <span class="grant">U54 HG003273-05S1</span>, <span class="grant">U54 HG003273-05S2</span>, <span class="grant">U54 HG003273-06</span>, <span class="grant">U54 HG003273-06S1</span>, <span class="grant">U54 HG003273-06S2</span>, <span class="grant">U54 HG003273-07</span>, <span class="grant">U54 HG003273-08</span>; NIDA NIH HHS: <span class="grant">P30 DA018310</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">324</span>;<span class="number">5926</span>;<span class="pagerange">522-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19390049">19390049</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2943200">2943200</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1169588">10.1126/science.1169588</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds.</h4>
<p class="authors"><span class="author">Bovine HapMap Consortium</span>, <span class="author">Gibbs RA</span>, <span class="author">Taylor JF</span>, <span class="author">Van Tassell CP</span>, <span class="author">Barendse W</span>, <span class="author">Eversole KA</span>, <span class="author">Gill CA</span>, <span class="author">Green RD</span>, <span class="author">Hamernik DL</span>, <span class="author">Kappes SM</span>, <span class="author">Lien S</span>, <span class="author">Matukumalli LK</span>, <span class="author">McEwan JC</span>, <span class="author">Nazareth LV</span>, <span class="author">Schnabel RD</span>, <span class="author">Weinstock GM</span>, <span class="author">Wheeler DA</span>, <span class="author">Ajmone-Marsan P</span>, <span class="author">Boettcher PJ</span>, <span class="author">Caetano AR</span>, <span class="author">Garcia JF</span>, <span class="author">Hanotte O</span>, <span class="author">Mariani P</span>, <span class="author">Skow LC</span>, <span class="author">Sonstegard TS</span>, <span class="author">Williams JL</span>, <span class="author">Diallo B</span>, <span class="author">Hailemariam L</span>, <span class="author">Martinez ML</span>, <span class="author">Morris CA</span>, <span class="author">Silva LO</span>, <span class="author">Spelman RJ</span>, <span class="author">Mulatu W</span>, <span class="author">Zhao K</span>, <span class="author">Abbey CA</span>, <span class="author">Agaba M</span>, <span class="author">Araujo FR</span>, <span class="author">Bunch RJ</span>, <span class="author">Burton J</span>, <span class="author">Gorni C</span>, <span class="author">Olivier H</span>, <span class="author">Harrison BE</span>, <span class="author">Luff B</span>, <span class="author">Machado MA</span>, <span class="author">Mwakaya J</span>, <span class="author">Plastow G</span>, <span class="author">Sim W</span>, <span class="author">Smith T</span>, <span class="author">Thomas MB</span>, <span class="author">Valentini A</span>, <span class="author">Williams P</span>, <span class="author">Womack J</span>, <span class="author">Woolliams JA</span>, <span class="author">Liu Y</span>, <span class="author">Qin X</span>, <span class="author">Worley KC</span>, <span class="author">Gao C</span>, <span class="author">Jiang H</span>, <span class="author">Moore SS</span>, <span class="author">Ren Y</span>, <span class="author">Song XZ</span>, <span class="author">Bustamante CD</span>, <span class="author">Hernandez RD</span>, <span class="author">Muzny DM</span>, <span class="author">Patil S</span>, <span class="author">San Lucas A</span>, <span class="author">Fu Q</span>, <span class="author">Kent MP</span>, <span class="author">Vega R</span>, <span class="author">Matukumalli A</span>, <span class="author">McWilliam S</span>, <span class="author">Sclep G</span>, <span class="author">Bryc K</span>, <span class="author">Choi J</span>, <span class="author">Gao H</span>, <span class="author">Grefenstette JJ</span>, <span class="author">Murdoch B</span>, <span class="author">Stella A</span>, <span class="author">Villa-Angulo R</span>, <span class="author">Wright M</span>, <span class="author">Aerts J</span>, <span class="author">Jann O</span>, <span class="author">Negrini R</span>, <span class="author">Goddard ME</span>, <span class="author">Hayes BJ</span>, <span class="author">Bradley DG</span>, <span class="author">Barbosa da Silva M</span>, <span class="author">Lau LP</span>, <span class="author">Liu GE</span>, <span class="author">Lynn DJ</span>, <span class="author">Panzitta F</span> and <span class="author">Dodds KG</span></p><p class="abstract">The imprints of domestication and breed development on the genomes of livestock likely differ from those of companion animals. A deep draft sequence assembly of shotgun reads from a single Hereford female and comparative sequences sampled from six additional breeds were used to develop probes to interrogate 37,470 single-nucleotide polymorphisms (SNPs) in 497 cattle from 19 geographically and biologically diverse breeds. These data show that cattle have undergone a rapid recent decrease in effective population size from a very large ancestral population, possibly due to bottlenecks associated with domestication, selection, and breed formation. Domestication and artificial selection appear to have left detectable signatures of selection within the cattle genome, yet the current levels of diversity within breeds are at least as great as exists within humans.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM083606</span>, <span class="grant">R01 GM083606-02</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">324</span>;<span class="number">5926</span>;<span class="pagerange">528-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19390050">19390050</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2735092">2735092</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1167936">10.1126/science.1167936</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Trypanosomatid genomes contain several subfamilies of ingi-related retroposons.</h4>
<p class="authors"><span class="author">Bringaud F</span>, <span class="author">Berriman M</span> and <span class="author">Hertz-Fowler C</span></p>
<p class="affiliation">Centre de R&eacute;sonance Magn&eacute;tique des Syst&egrave;mes Biologiques, UMR-5536 CNRS, Universit&eacute; Victor Segalen Bordeaux 2, 146 rue L&eacute;o Saignat, 33076 Bordeaux, France. <a href="mailto:%62%72%69%6E%67%61%75%64%40%72%6D%73%62.%75-%62%6F%72%64%65%61%75%78%32.%66%72">&#98;&#114;&#105;&#110;&#103;&#97;&#117;&#100;&#64;&#114;&#109;&#115;&#98;&#46;&#117;&#45;&#98;&#111;&#114;&#100;&#101;&#97;&#117;&#120;&#50;&#46;&#102;&#114;</a></p><p class="abstract">Retroposons are ubiquitous transposable elements found in the genomes of most eukaryotes, including trypanosomatids. The African and American trypanosomes (Trypanosoma brucei and Trypanosoma cruzi) contain long autonomous retroposons of the ingi clade (Tbingi and L1Tc, respectively) and short nonautonomous truncated versions (TbRIME and NARTc, respectively), as well as degenerate ingi-related retroposons devoid of coding capacity (DIREs). In contrast, Leishmania major contains only remnants of extinct retroposons (LmDIREs) and of short nonautonomous heterogeneous elements (LmSIDERs). We extend this comparative and evolutionary analysis of retroposons to the genomes of two other African trypanosomes (Trypanosoma congolense and Trypanosoma vivax) and another Leishmania sp. (Leishmania braziliensis). Three new potentially functional retroposons of the ingi clade have been identified: Tvingi in T. vivax and Tcoingi and L1Tco in T. congolense. T. congolense is the first trypanosomatid containing two classes of potentially active retroposons of the ingi clade. We analyzed sequences located upstream of these new long autonomous ingi-related elements, which code for the recognition site of the retroposon-encoded endonuclease. The closely related Tcoingi and Tvingi elements show the same conserved pattern, indicating that the Tcoingi- and Tvingi-encoded endonucleases share site specificity. Similarly, the conserved pattern previously identified upstream of L1Tc has also been detected at the same relative position upstream of L1Tco elements. A phylogenetic analysis of all ingi-related retroposons identified so far, including DIREs, clearly shows that several distinct subfamilies have emerged and coexisted, though in the course of trypanosomatid evolution, only a few have been maintained as active elements in modern trypanosomatid (sub)species.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2009</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">1532-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19666780" rel="external">19666780</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2756862" rel="external">2756862</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.00183-09" rel="external">10.1128/EC.00183-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic variation in a global village: report of the 10th annual Human Genome Variation Meeting 2008.</h4>
<p class="authors"><span class="author">Brookes AJ</span>, <span class="author">Chanock SJ</span>, <span class="author">Hudson TJ</span>, <span class="author">Peltonen L</span>, <span class="author">Abecasis G</span>, <span class="author">Kwok PY</span> and <span class="author">Scherer SW</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, Leicester, United Kingdom.</p><p class="abstract">The Centre for Applied Genomics of the Hospital for Sick Children and the University of Toronto hosted the 10th Human Genome Variation (HGV) Meeting in Toronto, Canada, in October 2008, welcoming about 240 registrants from 34 countries. During the 3 days of plenary workshops, keynote address, and poster sessions, a strong cross-disciplinary trend was evident, integrating expertise from technology and computation, through biology and medicine, to ethics and law. Single nucleotide polymorphisms (SNPs) as well as the larger copy number variants (CNVs) are recognized by ever-improving array and next-generation sequencing technologies, and the data are being incorporated into studies that are increasingly genome-wide as well as global in scope. A greater challenge is to convert data to information, through databases, and to use the information for greater understanding of human variation. In the wake of publications of the first individual genome sequences, an inaugural public forum provided the opportunity to debate whether we are ready for personalized medicine through direct-to-consumer testing. The HGV meetings foster collaboration, and fruits of the interactions from 2008 are anticipated for the 11th annual meeting in September 2009.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2009</span>;<span class="volume">30</span>;<span class="number">7</span>;<span class="pagerange">1134-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19384970" rel="external">19384970</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.21008" rel="external">10.1002/humu.21008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Accurate and sensitive peptide identification with Mascot Percolator.</h4>
<p class="authors"><span class="author">Brosch M</span>, <span class="author">Yu L</span>, <span class="author">Hubbard T</span> and <span class="author">Choudhary J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kingdom.</p><p class="abstract">Sound scoring methods for sequence database search algorithms such as Mascot and Sequest are essential for sensitive and accurate peptide and protein identifications from proteomic tandem mass spectrometry data. In this paper, we present a software package that interfaces Mascot with Percolator, a well performing machine learning method for rescoring database search results, and demonstrate it to be amenable for both low and high accuracy mass spectrometry data, outperforming all available Mascot scoring schemes as well as providing reliable significance measures. Mascot Percolator can be readily used as a stand alone tool or integrated into existing data analysis pipelines.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Journal of proteome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">3176-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19338334">19338334</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2734080">2734080</a>; DOI: <a href="http://dx.doi.org/10.1021%2Fpr800982s">10.1021/pr800982s</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Observational study on variability between biobanks in the estimation of DNA concentration.</h4>
<p class="authors"><span class="author">Brown J</span>, <span class="author">Donev AN</span>, <span class="author">Aslanidis C</span>, <span class="author">Bracegirdle P</span>, <span class="author">Dixon KP</span>, <span class="author">Foedinger M</span>, <span class="author">Gwilliam R</span>, <span class="author">Hardy M</span>, <span class="author">Illig T</span>, <span class="author">Ke X</span>, <span class="author">Krinka D</span>, <span class="author">Lagerberg C</span>, <span class="author">Laiho P</span>, <span class="author">Lewis DH</span>, <span class="author">McArdle W</span>, <span class="author">Patton S</span>, <span class="author">Ring SM</span>, <span class="author">Schmitz G</span>, <span class="author">Stevens H</span>, <span class="author">Tybring G</span>, <span class="author">Wichmann HE</span>, <span class="author">Ollier WE</span> and <span class="author">Yuille MA</span></p>
<p class="affiliation">Centre for Integrated Genomic Medical Research, University of Manchester, Manchester, UK. <a href="mailto:%4A%61%79.%42%72%6F%77%6E%40%63%6D%6D%63.%6E%68%73.%75%6B">&#74;&#97;&#121;&#46;&#66;&#114;&#111;&#119;&#110;&#64;&#99;&#109;&#109;&#99;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: There is little confidence in the consistency of estimation of DNA concentrations when samples move between laboratories. Evidence on this consistency is largely anecdotal. Therefore there is a need first to measure this consistency among different laboratories and then identify and implement remedies. A pilot experiment to test logistics and provide initial data on consistency was therefore conceived.</p><p class="abstract"><strong>Methods</strong>: DNA aliquots at nominal concentrations between 10 and 300 ng/mul were dispensed into the wells of 96-well plates by one participant - the coordinating centre. Participants estimated the concentration in each well and returned estimates to the coordinating centre.</p><p class="abstract"><strong>Results</strong>: Considerable overall variability was observed among estimates. There were statistically significant differences between participants&#39; measurements and between fluorescence emission and absorption spectroscopy.</p><p class="abstract"><strong>Conclusion</strong>: Anecdotal evidence of variability in DNA concentration estimation has been substantiated. Reduction in variability between participants will require the identification of major sources of variation, specification of effective remedies and their implementation.</p>
<p class="publication"><span class="title">BMC research notes</span>&nbsp;<span class="year">2009</span>;<span class="volume">2</span>;<span class="pagerange">208</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19825187" rel="external">19825187</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2765451" rel="external">2765451</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1756-0500-2-208" rel="external">10.1186/1756-0500-2-208</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional diversity for REST (NRSF) is defined by in vivo binding affinity hierarchies at the DNA sequence level.</h4>
<p class="authors"><span class="author">Bruce AW</span>, <span class="author">L&oacute;pez-Contreras AJ</span>, <span class="author">Flicek P</span>, <span class="author">Down TA</span>, <span class="author">Dhami P</span>, <span class="author">Dillon SC</span>, <span class="author">Koch CM</span>, <span class="author">Langford CF</span>, <span class="author">Dunham I</span>, <span class="author">Andrews RM</span> and <span class="author">Vetrie D</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom. <a href="mailto:%61%77%62%34%31%40%63%61%6D.%61%63.%75%6B">&#97;&#119;&#98;&#52;&#49;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The molecular events that contribute to, and result from, the in vivo binding of transcription factors to their cognate DNA sequence motifs in mammalian genomes are poorly understood. We demonstrate that variations within the DNA sequence motifs that bind the transcriptional repressor REST (NRSF) encode in vivo DNA binding affinity hierarchies that contribute to regulatory function during lineage-specific and developmental programs in fundamental ways. First, canonical sequence motifs for REST facilitate strong REST binding and control functional classes of REST targets that are common to all cell types, whilst atypical motifs participate in weak interactions and control those targets, which are cell- or tissue-specific. Second, variations in REST binding relate directly to variations in expression and chromatin configurations of REST&#39;s target genes. Third, REST clearance from its binding sites is also associated with variations in the RE1 motif. Finally, and most surprisingly, weak REST binding sites reside in DNA sequences that show the highest levels of constraint through evolution, thus facilitating their roles in maintaining tissue-specific functions. These relationships have never been reported in mammalian systems for any transcription factor.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">6</span>;<span class="pagerange">994-1005</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19401398">19401398</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2694481">2694481</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.089086.108">10.1101/gr.089086.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide microarray-based comparative genomic hybridization analysis of lymphoplasmacytic lymphomas reveals heterogeneous aberrations.</h4>
<p class="authors"><span class="author">Buckley PG</span>, <span class="author">Walsh SH</span>, <span class="author">Laurell A</span>, <span class="author">Sundstr&ouml;m C</span>, <span class="author">Roos G</span>, <span class="author">Langford CF</span>, <span class="author">Dumanski JP</span> and <span class="author">Rosenquist R</span></p>
<p class="affiliation">Department of Cancer Genetics, Royal College of Surgeons in Ireland, Dublin, Ireland. <a href="mailto:%70%62%75%63%6B%6C%65%79%40%72%63%73%69.%69%65">&#112;&#98;&#117;&#99;&#107;&#108;&#101;&#121;&#64;&#114;&#99;&#115;&#105;&#46;&#105;&#101;</a></p><p class="abstract">Lymphoplasmacytic lymphoma (LPL) is not a sharply delineated lymphoma entity, either morphologically, phenotypically, or clinically. The diagnosis is often made by excluding other small cell lymphomas with plasmacytic differentiation, thus a genetic diagnostic marker would be of great benefit. Conventional cytogenetic techniques have previously demonstrated a deletion of 6q in a proportion of cases, varying from 7 to 55%. In this report, we apply array-based comparative genomic hybridization on 11 LPL samples. Genomic aberrations were detected in 9 of 11 cases, and included gains and losses. In general, the number of genetic aberrations was relatively low (two to three abnormalities per case). Recurrent aberrations detected were deletion of 6q (two cases), deletion of chromosome 17 (two cases), gain of 3q (two cases), and gain of chromosome 7 (two cases). This report not only confirms the reported loss of 6q in a proportion of cases but also highlights the genetic heterogeneity of LPL, in accordance with the known immunophenotypical, morphological, and clinical diversity of the disease.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Leukemia &amp; lymphoma</span>&nbsp;<span class="year">2009</span>;<span class="volume">50</span>;<span class="number">9</span>;<span class="pagerange">1528-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19672776">19672776</a>; DOI: <a href="http://dx.doi.org/10.1080%2F10428190903131763">10.1080/10428190903131763</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The T3SS effector EspT defines a new category of invasive enteropathogenic E. coli (EPEC) which form intracellular actin pedestals.</h4>
<p class="authors"><span class="author">Bulgin R</span>, <span class="author">Arbeloa A</span>, <span class="author">Goulding D</span>, <span class="author">Dougan G</span>, <span class="author">Crepin VF</span>, <span class="author">Raymond B</span> and <span class="author">Frankel G</span></p>
<p class="affiliation">Centre for Molecular Microbiology and Infection, Division of Cell and Molecular Biology, Imperial College London, London, United Kingdom.</p><p class="abstract">Enteropathogenic Escherichia coli (EPEC) strains are defined as extracellular pathogens which nucleate actin rich pedestal-like membrane extensions on intestinal enterocytes to which they intimately adhere. EPEC infection is mediated by type III secretion system effectors, which modulate host cell signaling. Recently we have shown that the WxxxE effector EspT activates Rac1 and Cdc42 leading to formation of membrane ruffles and lamellipodia. Here we report that EspT-induced membrane ruffles facilitate EPEC invasion into non-phagocytic cells in a process involving Rac1 and Wave2. Internalized EPEC resides within a vacuole and Tir is localized to the vacuolar membrane, resulting in actin polymerization and formation of intracellular pedestals. To the best of our knowledge this is the first time a pathogen has been shown to induce formation of actin comets across a vacuole membrane. Moreover, our data breaks the dogma of EPEC as an extracellular pathogen and defines a new category of invasive EPEC.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700823</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">12</span>;<span class="pagerange">e1000683</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20011125">20011125</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2782363">2782363</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1000683">10.1371/journal.ppat.1000683</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The evolution of protein domain families.</h4>
<p class="authors"><span class="author">Buljan M</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">Protein domains are the common currency of protein structure and function. Over 10,000 such protein families have now been collected in the Pfam database. Using these data along with animal gene phylogenies from TreeFam allowed us to investigate the gain and loss of protein domains. Most gains and losses of domains occur at protein termini. We show that the nature of changes is similar after speciation or duplication events. However, changes in domain architecture happen at a higher frequency after gene duplication. We suggest that the bias towards protein termini is largely because insertion and deletion of domains at most positions in a protein are likely to disrupt the structure of existing domains. We can also use Pfam to trace the evolution of specific families. For example, the immunoglobulin superfamily can be traced over 500 million years during its expansion into one of the largest families in the human genome. It can be shown that this protein family has its origins in basic animals such as the poriferan sponges where it is found in cell-surface-receptor proteins. We can trace how the structure and sequence of this family diverged during vertebrate evolution into constant and variable domains that are found in the antibodies of our immune system as well as in neural and muscle proteins.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2009</span>;<span class="volume">37</span>;<span class="number">Pt 4</span>;<span class="pagerange">751-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19614588" rel="external">19614588</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0370751" rel="external">10.1042/BST0370751</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of pathogenicity and sexual reproduction in eight Candida genomes.</h4>
<p class="authors"><span class="author">Butler G</span>, <span class="author">Rasmussen MD</span>, <span class="author">Lin MF</span>, <span class="author">Santos MA</span>, <span class="author">Sakthikumar S</span>, <span class="author">Munro CA</span>, <span class="author">Rheinbay E</span>, <span class="author">Grabherr M</span>, <span class="author">Forche A</span>, <span class="author">Reedy JL</span>, <span class="author">Agrafioti I</span>, <span class="author">Arnaud MB</span>, <span class="author">Bates S</span>, <span class="author">Brown AJ</span>, <span class="author">Brunke S</span>, <span class="author">Costanzo MC</span>, <span class="author">Fitzpatrick DA</span>, <span class="author">de Groot PW</span>, <span class="author">Harris D</span>, <span class="author">Hoyer LL</span>, <span class="author">Hube B</span>, <span class="author">Klis FM</span>, <span class="author">Kodira C</span>, <span class="author">Lennard N</span>, <span class="author">Logue ME</span>, <span class="author">Martin R</span>, <span class="author">Neiman AM</span>, <span class="author">Nikolaou E</span>, <span class="author">Quail MA</span>, <span class="author">Quinn J</span>, <span class="author">Santos MC</span>, <span class="author">Schmitzberger FF</span>, <span class="author">Sherlock G</span>, <span class="author">Shah P</span>, <span class="author">Silverstein KA</span>, <span class="author">Skrzypek MS</span>, <span class="author">Soll D</span>, <span class="author">Staggs R</span>, <span class="author">Stansfield I</span>, <span class="author">Stumpf MP</span>, <span class="author">Sudbery PE</span>, <span class="author">Srikantha T</span>, <span class="author">Zeng Q</span>, <span class="author">Berman J</span>, <span class="author">Berriman M</span>, <span class="author">Heitman J</span>, <span class="author">Gow NA</span>, <span class="author">Lorenz MC</span>, <span class="author">Birren BW</span>, <span class="author">Kellis M</span> and <span class="author">Cuomo CA</span></p>
<p class="affiliation">UCD School of Biomolecular and Biomedical Science, Conway Institute, University College Dublin, Belfield, Dublin 4, Ireland. <a href="mailto:%67%65%72%61%6C%64%69%6E%65.%62%75%74%6C%65%72%40%75%63%64.%69%65">&#103;&#101;&#114;&#97;&#108;&#100;&#105;&#110;&#101;&#46;&#98;&#117;&#116;&#108;&#101;&#114;&#64;&#117;&#99;&#100;&#46;&#105;&#101;</a></p><p class="abstract">Candida species are the most common cause of opportunistic fungal infection worldwide. Here we report the genome sequences of six Candida species and compare these and related pathogens and non-pathogens. There are significant expansions of cell wall, secreted and transporter gene families in pathogenic species, suggesting adaptations associated with virulence. Large genomic tracts are homozygous in three diploid species, possibly resulting from recent recombination events. Surprisingly, key components of the mating and meiosis pathways are missing from several species. These include major differences at the mating-type loci (MTL); Lodderomyces elongisporus lacks MTL, and components of the a1/2 cell identity determinant were lost in other species, raising questions about how mating and cell types are controlled. Analysis of the CUG leucine-to-serine genetic-code change reveals that 99% of ancestral CUG codons were erased and new ones arose elsewhere. Lastly, we revise the Candida albicans gene catalogue, identifying many new genes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0400284</span>; NHGRI NIH HHS: <span class="grant">R01 HG004037</span>, <span class="grant">R01 HG004037-02</span>, <span class="grant">U54 HG003067</span>, <span class="grant">U54 HG003067-06</span>; NIAID NIH HHS: <span class="grant">HHSN266200400001C</span>, <span class="grant">R01 AI050113</span>, <span class="grant">R01 AI075096</span>; NIDCR NIH HHS: <span class="grant">R01 DE015873</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2009</span>;<span class="volume">459</span>;<span class="number">7247</span>;<span class="pagerange">657-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19465905">19465905</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2834264">2834264</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08064">10.1038/nature08064</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrin-mediated axoglial interactions initiate myelination in the central nervous system.</h4>
<p class="authors"><span class="author">C&acirc;mara J</span>, <span class="author">Wang Z</span>, <span class="author">Nunes-Fonseca C</span>, <span class="author">Friedman HC</span>, <span class="author">Grove M</span>, <span class="author">Sherman DL</span>, <span class="author">Komiyama NH</span>, <span class="author">Grant SG</span>, <span class="author">Brophy PJ</span>, <span class="author">Peterson A</span> and <span class="author">ffrench-Constant C</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Cambridge CB2 1QP, England, UK.</p><p class="abstract">All but the smallest-diameter axons in the central nervous system are myelinated, but the signals that initiate myelination are unknown. Our prior work has shown that integrin signaling forms part of the cell-cell interactions that ensure only those oligodendrocytes contacting axons survive. Here, therefore, we have asked whether integrins regulate the interactions that lead to myelination. Using homologous recombination to insert a single-copy transgene into the hypoxanthine phosphoribosyl transferase (hprt) locus, we find that mice expressing a dominant-negative beta1 integrin in myelinating oligodendrocytes require a larger axon diameter to initiate timely myelination. Mice with a conditional deletion of focal adhesion kinase (a signaling molecule activated by integrins) exhibit a similar phenotype. Conversely, transgenic mice expressing dominant-negative beta3 integrin in oligodendrocytes display no myelination abnormalities. We conclude that beta1 integrin plays a key role in the axoglial interactions that sense axon size and initiate myelination, such that loss of integrin signaling leads to a delay in myelination of small-diameter axons.</p>
<p class="grants">Funded by: Multiple Sclerosis Society: <span class="grant">669</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of cell biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">185</span>;<span class="number">4</span>;<span class="pagerange">699-712</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19451276" rel="external">19451276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2711572" rel="external">2711572</a>; DOI: <a href="http://dx.doi.org/10.1083%2Fjcb.200807010" rel="external">10.1083/jcb.200807010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic and germline genetics at the JAK2 locus.</h4>
<p class="authors"><span class="author">Campbell PJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%70%63%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#99;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Myeloproliferative neoplasms are hematological malignancies frequently associated with somatically acquired mutation of the JAK2 gene. A new study shows that these mutations are preferentially found within a particular inherited JAK2 haplotype, implying the existence of a strong, but uncharacterized, interaction between somatic and germline genetics at this locus.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">4</span>;<span class="pagerange">385-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19338077">19338077</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reticulin accumulation in essential thrombocythemia: prognostic significance and relationship to therapy.</h4>
<p class="authors"><span class="author">Campbell PJ</span>, <span class="author">Bareford D</span>, <span class="author">Erber WN</span>, <span class="author">Wilkins BS</span>, <span class="author">Wright P</span>, <span class="author">Buck G</span>, <span class="author">Wheatley K</span>, <span class="author">Harrison CN</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, University of Cambridge, Cambridge, UK.</p><p class="abstract"><strong>Purpose</strong>: Essential thrombocythemia (ET) manifests substantial interpatient heterogeneity in rates of thrombosis, hemorrhage, and disease transformation. Bone marrow histology reflects underlying disease activity in ET but many morphological features show poor reproducibility.</p><p class="abstract">We evaluated the clinical significance of bone marrow reticulin, a measure previously shown to have relatively high interobserver reliability, in a large, prospectively-studied cohort of ET patients.</p><p class="abstract"><strong>Results</strong>: Reticulin grade positively correlated with white blood cell (P = .05) and platelet counts (P = .0001) at diagnosis. Elevated reticulin levels at presentation predicted higher rates of arterial thrombosis (hazard ratio [HR], 1.8; 95% CI, 1.1 to 2.9; P = .01), major hemorrhage (HR, 2.0; 95% CI, 1.0 to 3.9; P = .05), and myelofibrotic transformation (HR, 5.5; 95% CI, 1.7 to 18.4; P = .0007) independently of known risk factors. Higher reticulin levels at diagnosis were associated with greater subsequent falls in hemoglobin levels in patients treated with anagrelide (P &lt; .0001), but not in those receiving hydroxyurea (P = .9). Moreover, serial trephine specimens in patients randomly assigned to anagrelide showed significantly greater increases in reticulin grade compared with those allocated to hydroxyurea (P = .0003), and four patients who developed increased bone marrow reticulin on anagrelide showed regression of fibrosis when switched to hydroxyurea. These data suggest that patients receiving anagrelide therapy should undergo surveillance bone marrow biopsy every 2 to 3 years and that those who show substantially increasing reticulin levels are at risk of myelofibrotic transformation and may benefit from changing therapy before adverse clinical features develop.</p><p class="abstract"><strong>Conclusion</strong>: Our results demonstrate that bone marrow reticulin grade at diagnosis represents an independent prognostic marker in ET, reflecting activity and/or duration of disease, with implications for the monitoring of patients receiving anagrelide.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2009</span>;<span class="volume">27</span>;<span class="number">18</span>;<span class="pagerange">2991-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19364963" rel="external">19364963</a>; DOI: <a href="http://dx.doi.org/10.1200%2FJCO.2008.20.3174" rel="external">10.1200/JCO.2008.20.3174</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TLR9 polymorphisms in African populations: no association with severe malaria, but evidence of cis-variants acting on gene expression.</h4>
<p class="authors"><span class="author">Campino S</span>, <span class="author">Forton J</span>, <span class="author">Auburn S</span>, <span class="author">Fry A</span>, <span class="author">Diakite M</span>, <span class="author">Richardson A</span>, <span class="author">Hull J</span>, <span class="author">Jallow M</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Pinder M</span>, <span class="author">Molyneux ME</span>, <span class="author">Taylor TE</span>, <span class="author">Rockett K</span>, <span class="author">Clark TG</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%73%63%31%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#99;&#49;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: During malaria infection the Toll-like receptor 9 (TLR9) is activated through induction with plasmodium DNA or another malaria motif not yet identified. Although TLR9 activation by malaria parasites is well reported, the implication to the susceptibility to severe malaria is not clear. The aim of this study was to assess the contribution of genetic variation at TLR9 to severe malaria.</p><p class="abstract"><strong>Methods</strong>: This study explores the contribution of TLR9 genetic variants to severe malaria using two approaches. First, an association study of four common single nucleotide polymorphisms was performed on both family- and population-based studies from Malawian and Gambian populations (n&gt;6000 individual). Subsequently, it was assessed whether TLR9 expression is affected by cis-acting variants and if these variants could be mapped. For this work, an allele specific expression (ASE) assay on a panel of HapMap cell lines was carried out.</p><p class="abstract"><strong>Results</strong>: No convincing association was found with polymorphisms in TLR9 for malaria severity, in either Gambian or Malawian populations, using both case-control and family based study designs. Using an allele specific expression assay it was observed that TLR9 expression is affected by cis-acting variants, these results were replicated in a second experiment using biological replicates.</p><p class="abstract"><strong>Conclusion</strong>: By using the largest cohorts analysed to date, as well as a standardized phenotype definition and study design, no association of TLR9 genetic variants with severe malaria was found. This analysis considered all common variants in the region, but it is remains possible that there are rare variants with association signals. This report also shows that TLR9 expression is potentially modulated through cis-regulatory variants, which may lead to differential inflammatory responses to infection between individuals.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G19/9</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2009</span>;<span class="volume">8</span>;<span class="pagerange">44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19284650">19284650</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2660361">2660361</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-8-44">10.1186/1475-2875-8-44</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary breakpoints in the gibbon suggest association between cytosine methylation and karyotype evolution.</h4>
<p class="authors"><span class="author">Carbone L</span>, <span class="author">Harris RA</span>, <span class="author">Vessere GM</span>, <span class="author">Mootnick AR</span>, <span class="author">Humphray S</span>, <span class="author">Rogers J</span>, <span class="author">Kim SK</span>, <span class="author">Wall JD</span>, <span class="author">Martin D</span>, <span class="author">Jurka J</span>, <span class="author">Milosavljevic A</span> and <span class="author">de Jong PJ</span></p>
<p class="affiliation">Children&#39;s Hospital and Research Center Oakland, Oakland, California, United States of America. <a href="mailto:%6C%63%61%72%62%6F%6E%65%40%63%68%6F%72%69.%6F%72%67">&#108;&#99;&#97;&#114;&#98;&#111;&#110;&#101;&#64;&#99;&#104;&#111;&#114;&#105;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Gibbon species have accumulated an unusually high number of chromosomal changes since diverging from the common hominoid ancestor 15-18 million years ago. The cause of this increased rate of chromosomal rearrangements is not known, nor is it known if genome architecture has a role. To address this question, we analyzed sequences spanning 57 breaks of synteny between northern white-cheeked gibbons (Nomascus l. leucogenys) and humans. We find that the breakpoint regions are enriched in segmental duplications and repeats, with Alu elements being the most abundant. Alus located near the gibbon breakpoints (&lt;150 bp) have a higher CpG content than other Alus. Bisulphite allelic sequencing reveals that these gibbon Alus have a lower average density of methylated cytosine that their human orthologues. The finding of higher CpG content and lower average CpG methylation suggests that the gibbon Alu elements are epigenetically distinct from their human orthologues. The association between undermethylation and chromosomal rearrangement in gibbons suggests a correlation between epigenetic state and structural genome variation in evolution.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e1000538</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19557196" rel="external">19557196</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2695003" rel="external">2695003</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000538" rel="external">10.1371/journal.pgen.1000538</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome watch: What a scorcher!</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span></p><p class="abstract">This month&#39;s Genome Watch looks at the publication of four hyperthermophilic archaeal genomes, three of which belong to the Crenarchaeota phylum and one of which belongs to the newly defined Nanoarchaeota phylum.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">408-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19444245">19444245</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2154">10.1038/nrmicro2154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Induction of antibody responses to African horse sickness virus (AHSV) in ponies after vaccination with recombinant modified vaccinia Ankara (MVA).</h4>
<p class="authors"><span class="author">Chiam R</span>, <span class="author">Sharp E</span>, <span class="author">Maan S</span>, <span class="author">Rao S</span>, <span class="author">Mertens P</span>, <span class="author">Blacklaws B</span>, <span class="author">Davis-Poynter N</span>, <span class="author">Wood J</span> and <span class="author">Castillo-Olivares J</span></p>
<p class="affiliation">Animal Health Trust, Lanwades Park, Kentford, Newmarket, Suffolk, United Kingdom.</p><p class="abstract"><strong>Background</strong>: African horse sickness virus (AHSV) causes a non-contagious, infectious disease in equids, with mortality rates that can exceed 90% in susceptible horse populations. AHSV vaccines play a crucial role in the control of the disease; however, there are concerns over the use of polyvalent live attenuated vaccines particularly in areas where AHSV is not endemic. Therefore, it is important to consider alternative approaches for AHSV vaccine development. We have carried out a pilot study to investigate the ability of recombinant modified vaccinia Ankara (MVA) vaccines expressing VP2, VP7 or NS3 genes of AHSV to stimulate immune responses against AHSV antigens in the horse.</p><p class="abstract">VP2, VP7 and NS3 genes from AHSV-4/Madrid87 were cloned into the vaccinia transfer vector pSC11 and recombinant MVA viruses generated. Antigen expression or transcription of the AHSV genes from cells infected with the recombinant viruses was confirmed. Pairs of ponies were vaccinated with MVAVP2, MVAVP7 or MVANS3 and both MVA vector and AHSV antigen-specific antibody responses were analysed. Vaccination with MVAVP2 induced a strong AHSV neutralising antibody response (VN titre up to a value of 2). MVAVP7 also induced AHSV antigen-specific responses, detected by western blotting. NS3 specific antibody responses were not detected.</p><p class="abstract"><strong>Conclusions</strong>: This pilot study demonstrates the immunogenicity of recombinant MVA vectored AHSV vaccines, in particular MVAVP2, and indicates that further work to investigate whether these vaccines would confer protection from lethal AHSV challenge in the horse is justifiable.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/B/00654</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">6</span>;<span class="pagerange">e5997</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19543394">19543394</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2694985">2694985</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0005997">10.1371/journal.pone.0005997</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lineage-specific biology revealed by a finished genome assembly of the mouse.</h4>
<p class="authors"><span class="author">Church DM</span>, <span class="author">Goodstadt L</span>, <span class="author">Hillier LW</span>, <span class="author">Zody MC</span>, <span class="author">Goldstein S</span>, <span class="author">She X</span>, <span class="author">Bult CJ</span>, <span class="author">Agarwala R</span>, <span class="author">Cherry JL</span>, <span class="author">DiCuccio M</span>, <span class="author">Hlavina W</span>, <span class="author">Kapustin Y</span>, <span class="author">Meric P</span>, <span class="author">Maglott D</span>, <span class="author">Birtle Z</span>, <span class="author">Marques AC</span>, <span class="author">Graves T</span>, <span class="author">Zhou S</span>, <span class="author">Teague B</span>, <span class="author">Potamousis K</span>, <span class="author">Churas C</span>, <span class="author">Place M</span>, <span class="author">Herschleb J</span>, <span class="author">Runnheim R</span>, <span class="author">Forrest D</span>, <span class="author">Amos-Landgraf J</span>, <span class="author">Schwartz DC</span>, <span class="author">Cheng Z</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Eichler EE</span>, <span class="author">Ponting CP</span> and <span class="author">Mouse Genome Sequencing Consortium</span></p>
<p class="affiliation">National Center for Biotechnology Information, Bethesda, Maryland, United States of America. <a href="mailto:%63%68%75%72%63%68%40%6E%63%62%69.%6E%6C%6D.%6E%69%68.%67%6F%76">&#99;&#104;&#117;&#114;&#99;&#104;&#64;&#110;&#99;&#98;&#105;&#46;&#110;&#108;&#109;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">The mouse (Mus musculus) is the premier animal model for understanding human disease and development. Here we show that a comprehensive understanding of mouse biology is only possible with the availability of a finished, high-quality genome assembly. The finished clone-based assembly of the mouse strain C57BL/6J reported here has over 175,000 fewer gaps and over 139 Mb more of novel sequence, compared with the earlier MGSCv3 draft genome assembly. In a comprehensive analysis of this revised genome sequence, we are now able to define 20,210 protein-coding genes, over a thousand more than predicted in the human genome (19,042 genes). In addition, we identified 439 long, non-protein-coding RNAs with evidence for transcribed orthologs in human. We analyzed the complex and repetitive landscape of 267 Mb of sequence that was missing or misassembled in the previously published assembly, and we provide insights into the reasons for its resistance to sequencing and assembly by whole-genome shotgun approaches. Duplicated regions within newly assembled sequence tend to be of more recent ancestry than duplicates in the published draft, correcting our initial understanding of recent evolution on the mouse lineage. These duplicates appear to be largely composed of sequence regions containing transposable elements and duplicated protein-coding genes; of these, some may be fixed in the mouse population, but at least 40% of segmentally duplicated sequences are copy number variable even among laboratory mouse strains. Mouse lineage-specific regions contain 3,767 genes drawn mainly from rapidly-changing gene families associated with reproductive functions. The finished mouse genome assembly, therefore, greatly improves our understanding of rodent-specific biology and allows the delineation of ancestral biological functions that are shared with human from derived functions that are not.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U127561112</span>, <span class="grant">MC_U137761446</span>, <span class="grant">MC_U142684174</span>; NHGRI NIH HHS: <span class="grant">HG002385</span>, <span class="grant">R01 HG002385</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">e1000112</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19468303">19468303</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680341">2680341</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1000112">10.1371/journal.pbio.1000112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tumor necrosis factor and lymphotoxin-alpha polymorphisms and severe malaria in African populations.</h4>
<p class="authors"><span class="author">Clark TG</span>, <span class="author">Diakite M</span>, <span class="author">Auburn S</span>, <span class="author">Campino S</span>, <span class="author">Fry AE</span>, <span class="author">Green A</span>, <span class="author">Richardson A</span>, <span class="author">Small K</span>, <span class="author">Teo YY</span>, <span class="author">Wilson J</span>, <span class="author">Jallow M</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Pinder M</span>, <span class="author">Griffiths MJ</span>, <span class="author">Peshu N</span>, <span class="author">Williams TN</span>, <span class="author">Marsh K</span>, <span class="author">Molyneux ME</span>, <span class="author">Taylor TE</span>, <span class="author">Rockett KA</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Nuffield Department of Medicine, John Radcliffe Hospital, Oxford, United Kingdom. <a href="mailto:%74%67%63%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#116;&#103;&#99;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The tumor necrosis factor gene (TNF) and lymphotoxin-alpha gene (LTA) have long attracted attention as candidate genes for susceptibility traits for malaria, and several of their polymorphisms have been found to be associated with severe malaria (SM) phenotypes. In a large study involving &gt;10,000 individuals and encompassing 3 African populations, we found evidence to support the reported associations between the TNF -238 polymorphism and SM in The Gambia. However, no TNF/LTA polymorphisms were found to be associated with SM in cohorts in Kenya and Malawi. It has been suggested that the causal polymorphisms regulating the TNF and LTA responses may be located some distance from the genes. Therefore, more-detailed mapping of variants across TNF/LTA genes and their flanking regions in the Gambian and allied populations may need to be undertaken to find any causal polymorphisms.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">076934</span></p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2009</span>;<span class="volume">199</span>;<span class="number">4</span>;<span class="pagerange">569-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19281305">19281305</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2742199">2742199</a>; DOI: <a href="http://dx.doi.org/10.1086%2F596320">10.1086/596320</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neurotransmitters drive combinatorial multistate postsynaptic density networks.</h4>
<p class="authors"><span class="author">Coba MP</span>, <span class="author">Pocklington AJ</span>, <span class="author">Collins MO</span>, <span class="author">Kopanitsa MV</span>, <span class="author">Uren RT</span>, <span class="author">Swamy S</span>, <span class="author">Croning MD</span>, <span class="author">Choudhary JS</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition, Wellcome Trust Sanger Institute, Cambridgeshire, UK.</p><p class="abstract">The mammalian postsynaptic density (PSD) comprises a complex collection of approximately 1100 proteins. Despite extensive knowledge of individual proteins, the overall organization of the PSD is poorly understood. Here, we define maps of molecular circuitry within the PSD based on phosphorylation of postsynaptic proteins. Activation of a single neurotransmitter receptor, the N-methyl-D-aspartate receptor (NMDAR), changed the phosphorylation status of 127 proteins. Stimulation of ionotropic and metabotropic glutamate receptors and dopamine receptors activated overlapping networks with distinct combinatorial phosphorylation signatures. Using peptide array technology, we identified specific phosphorylation motifs and switching mechanisms responsible for the integration of neurotransmitter receptor pathways and their coordination of multiple substrates in these networks. These combinatorial networks confer high information-processing capacity and functional diversity on synapses, and their elucidation may provide new insights into disease mechanisms and new opportunities for drug discovery.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G90/93</span>; Wellcome Trust: <span class="grant">066717</span></p>
<p class="publication"><span class="title">Science signaling</span>&nbsp;<span class="year">2009</span>;<span class="volume">2</span>;<span class="number">68</span>;<span class="pagerange">ra19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19401593">19401593</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3280897">3280897</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscisignal.2000102">10.1126/scisignal.2000102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neuroproteomics</h4>
<p class="authors"><span class="author">Collins MO</span>, <span class="author">Grant SGN</span></p>
<p class="publication"><span class="title">Encyclopedia of Neuroscience</span> <span class="year">2009</span>;<span class="pagerange">971-81</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic analysis reveals extensive gene duplication within the bovine TRB locus.</h4>
<p class="authors"><span class="author">Connelley T</span>, <span class="author">Aerts J</span>, <span class="author">Law A</span> and <span class="author">Morrison WI</span></p>
<p class="affiliation">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush, Roslin, EH25 9RG, UK. <a href="mailto:%74%69%6D%6F%74%68%79.%63%6F%6E%6E%65%6C%6C%65%79%40%65%64.%61%63.%75%6B">&#116;&#105;&#109;&#111;&#116;&#104;&#121;&#46;&#99;&#111;&#110;&#110;&#101;&#108;&#108;&#101;&#121;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Diverse TR and IG repertoires are generated by V(D)J somatic recombination. Genomic studies have been pivotal in cataloguing the V, D, J and C genes present in the various TR/IG loci and describing how duplication events have expanded the number of these genes. Such studies have also provided insights into the evolution of these loci and the complex mechanisms that regulate TR/IG expression. In this study we analyze the sequence of the third bovine genome assembly to characterize the germline repertoire of bovine TRB genes and compare the organization, evolution and regulatory structure of the bovine TRB locus with that of humans and mice.</p><p class="abstract"><strong>Results</strong>: The TRB locus in the third bovine genome assembly is distributed over 5 scaffolds, extending to approximately 730 Kb. The available sequence contains 134 TRBV genes, assigned to 24 subgroups, and 3 clusters of DJC genes, each comprising a single TRBD gene, 5-7 TRBJ genes and a single TRBC gene. Seventy-nine of the TRBV genes are predicted to be functional. Comparison with the human and murine TRB loci shows that the gene order, as well as the sequences of non-coding elements that regulate TRB expression, are highly conserved in the bovine. Dot-plot analyses demonstrate that expansion of the genomic TRBV repertoire has occurred via a complex and extensive series of duplications, predominantly involving DNA blocks containing multiple genes. These duplication events have resulted in massive expansion of several TRBV subgroups, most notably TRBV6, 9 and 21 which contain 40, 35 and 16 members respectively. Similarly, duplication has lead to the generation of a third DJC cluster. Analyses of cDNA data confirms the diversity of the TRBV genes and, in addition, identifies a substantial number of TRBV genes, predominantly from the larger subgroups, which are still absent from the genome assembly. The observed gene duplication within the bovine TRB locus has created a repertoire of phylogenetically diverse functional TRBV genes, which is substantially larger than that described for humans and mice.</p><p class="abstract"><strong>Conclusion</strong>: The analyses completed in this study reveal that, although the gene content and organization of the bovine TRB locus are broadly similar to that of humans and mice, multiple duplication events have led to a marked expansion in the number of TRB genes. Similar expansions in other ruminant TR loci suggest strong evolutionary pressures in this lineage have selected for the development of enlarged sets of TR genes that can contribute to diverse TR repertoires.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">075820</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">192</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19393068" rel="external">19393068</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2685407" rel="external">2685407</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-10-192" rel="external">10.1186/1471-2164-10-192</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">First report of human infection with Salmonella enterica serovar Apapa resulting from exposure to a pet lizard.</h4>
<p class="authors"><span class="author">Cooke FJ</span>, <span class="author">De Pinna E</span>, <span class="author">Maguire C</span>, <span class="author">Guha S</span>, <span class="author">Pickard DJ</span>, <span class="author">Farrington M</span> and <span class="author">Threlfall EJ</span></p>
<p class="affiliation">Addenbrooke&#39;s Hospital, Cambridge, United Kingdom. <a href="mailto:%66%69%6F%6E%61.%63%6F%6F%6B%65%40%61%64%64%65%6E%62%72%6F%6F%6B%65%73.%6E%68%73.%75%6B">&#102;&#105;&#111;&#110;&#97;&#46;&#99;&#111;&#111;&#107;&#101;&#64;&#97;&#100;&#100;&#101;&#110;&#98;&#114;&#111;&#111;&#107;&#101;&#115;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract">We present the first documented human case of Salmonella enterica serovar Apapa infection, isolated concurrently from a hospital inpatient and a pet lizard. The isolates were identical by biochemical profiling and pulsed-field gel electrophoresis. This rare serotype is known to be associated with reptiles. The current practice for avoiding reptile-associated infections is reviewed.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">47</span>;<span class="number">8</span>;<span class="pagerange">2672-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19535527" rel="external">19535527</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2725640" rel="external">2725640</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.02475-08" rel="external">10.1128/JCM.02475-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large scale association analysis of novel genetic loci for coronary artery disease.</h4>
<p class="authors"><span class="author">Coronary Artery Disease Consortium</span>, <span class="author">Samani NJ</span>, <span class="author">Deloukas P</span>, <span class="author">Erdmann J</span>, <span class="author">Hengstenberg C</span>, <span class="author">Kuulasmaa K</span>, <span class="author">McGinnis R</span>, <span class="author">Schunkert H</span>, <span class="author">Soranzo N</span>, <span class="author">Thompson J</span>, <span class="author">Tiret L</span> and <span class="author">Ziegler A</span></p><p class="abstract"><strong>Background</strong>: Combined analysis of 2 genome-wide association studies in cases enriched for family history recently identified 7 loci (on 1p13.3, 1q41, 2q36.3, 6q25.1, 9p21, 10q11.21, and 15q22.33) that may affect risk of coronary artery disease (CAD). Apart from the 9p21 locus, the other loci await substantive replication. Furthermore, the effect of these loci on CAD risk in a broader range of individuals remains to be determined.</p><p class="abstract">We undertook association analysis of single nucleotide polymorphisms at each locus with CAD risk in 11,550 cases and 11,205 controls from 9 European studies. The 9p21.3 locus showed unequivocal association (rs1333049, combined odds ratio [OR]=1.20, 95% CI [1.16 to 1.25], probability value=2.81 x 10(-21)). We also confirmed association signals at 1p13.3 (rs599839, OR=1.13 [1.08 to 1.19], P=1.44 x 10(-7)), 1q41 (rs3008621, OR=1.10 [1.04 to 1.17], P=1.02 x 10(-3)), and 10q11.21 (rs501120, OR=1.11 [1.05 to 1.18], P=4.34 x 10(-4)). The associations with 6q25.1 (rs6922269, P=0.020) and 2q36.3 (rs2943634, P=0.032) were borderline and not statistically significant after correction for multiple testing. The 15q22.33 locus did not replicate. The 10q11.21 locus showed a possible sex interaction (P=0.015), with a significant effect in women (OR=1.29 [1.15 to 1.45], P=1.86 x 10(-5)) but not men (OR=1.03 [0.96 to 1.11], P=0.387). There were no other strong interactions of any of the loci with other traditional risk factors. The loci at 9p21, 1p13.3, 2q36.3, and 10q11.21 acted independently and cumulatively increased CAD risk by 15% (12% to 18%), per additional risk allele.</p><p class="abstract"><strong>Conclusions</strong>: The findings provide strong evidence for association between at least 4 genetic loci and CAD risk. Cumulatively, these novel loci have a significant impact on risk of CAD at least in European populations.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">CH/03/001/15569</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0701863</span>, <span class="grant">MC_U106179471</span>; Wellcome Trust: <span class="grant">077011</span>, <span class="grant">082371</span>, <span class="grant">091746</span></p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">29</span>;<span class="number">5</span>;<span class="pagerange">774-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19164808">19164808</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3315048">3315048</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.108.181388">10.1161/ATVBAHA.108.181388</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From small reads do mighty genomes grow.</h4>
<p class="authors"><span class="author">Croucher NJ</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This month&#39;s Genome Watch discusses the use of next-generation sequencing technologies to assemble draft genomes for two pseudomonad species.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">621</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19680245">19680245</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2211">10.1038/nrmicro2211</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Influenza-specific amino acid substitution model</h4>
<p class="authors"><span class="author">Cuong DC</span>, <span class="author">Vinh LS</span>, <span class="author">Quang LS</span></p>
<p class="publication"><span class="title">KSE 2009 - The 1st International Conference on Knowledge and Systems Engineering.</span> <span class="year">2009</span>;<span class="volume">5361735</span>;<span class="pagerange">19-25</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">X-box binding protein 1 contributes to induction of the Kaposi&#39;s sarcoma-associated herpesvirus lytic cycle under hypoxic conditions.</h4>
<p class="authors"><span class="author">Dalton-Griffin L</span>, <span class="author">Wilson SJ</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Department of Infection, UCL, London, United Kingdom.</p><p class="abstract">Kaposi&#39;s sarcoma-associated herpesvirus (KSHV), like other herpesviruses, has two stages to its life cycle: latency and lytic replication. KSHV is required for development of Kaposi&#39;s sarcoma, a tumor of endothelial origin, and is associated with the B-cell tumor primary effusion lymphoma (PEL) and the plasmablastic variant of multicentric Castleman&#39;s disease, all of which are characterized by predominantly latent KSHV infection. Recently, we and others have shown that the activated form of transcription factor X-box binding protein 1 (XBP-1) is a physiological trigger of KSHV lytic reactivation in PEL. Here, we show that XBP-1s transactivates the ORF50/RTA promoter though an ACGT core containing the XBP-1 response element, an element previously identified as a weakly active hypoxia response element (HRE). Hypoxia induces the KSHV lytic cycle, and active HREs that respond to hypoxia-inducible factor 1alpha are present in the ORF50/RTA promoter. Hypoxia also induces active XBP-1s, and here, we show that both transcription factors contribute to the induction of RTA expression, leading to the production of infectious KSHV under hypoxic conditions.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2009</span>;<span class="volume">83</span>;<span class="number">14</span>;<span class="pagerange">7202-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19403667">19403667</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2704782">2704782</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.00076-09">10.1128/JVI.00076-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A truncation mutation in TBC1D4 in a family with acanthosis nigricans and postprandial hyperinsulinemia.</h4>
<p class="authors"><span class="author">Dash S</span>, <span class="author">Sano H</span>, <span class="author">Rochford JJ</span>, <span class="author">Semple RK</span>, <span class="author">Yeo G</span>, <span class="author">Hyden CS</span>, <span class="author">Soos MA</span>, <span class="author">Clark J</span>, <span class="author">Rodin A</span>, <span class="author">Langenberg C</span>, <span class="author">Druet C</span>, <span class="author">Fawcett KA</span>, <span class="author">Tung YC</span>, <span class="author">Wareham NJ</span>, <span class="author">Barroso I</span>, <span class="author">Lienhard GE</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Savage DB</span></p>
<p class="affiliation">Departments of Medicine and Clinical Biochemistry, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom.</p><p class="abstract">Tre-2, BUB2, CDC16, 1 domain family member 4 (TBC1D4) (AS160) is a Rab-GTPase activating protein implicated in insulin-stimulated glucose transporter 4 (GLUT4) translocation in adipocytes and myotubes. To determine whether loss-of-function mutations in TBC1D4 might impair GLUT4 translocation and cause insulin resistance in humans, we screened the coding regions of this gene in 156 severely insulin-resistant patients. A female presenting at age 11 years with acanthosis nigricans and extreme postprandial hyperinsulinemia was heterozygous for a premature stop mutation (R363X) in TBC1D4. After demonstrating reduced expression of wild-type TBC1D4 protein and expression of the truncated protein in lymphocytes from the proband, we further characterized the biological effects of the truncated protein in 3T3L1 adipocytes. Prematurely truncated TBC1D4 protein tended to increase basal cell membrane GLUT4 levels (P = 0.053) and significantly reduced insulin-stimulated GLUT4 cell membrane translocation (P &lt; 0.05). When coexpressed with wild-type TBC1D4, the truncated protein dimerized with full-length TBC1D4, suggesting that the heterozygous truncated variant might interfere with its wild-type counterpart in a dominant negative fashion. Two overweight family members with the mutation also manifested normal fasting glucose and insulin levels but disproportionately elevated insulin levels following an oral glucose challenge. This family provides unique genetic evidence of TBC1D4 involvement in human insulin action.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council: <span class="grant">G0600414</span>; NCI NIH HHS: <span class="grant">P30 CA023108</span>; NIDDK NIH HHS: <span class="grant">DK25336</span>, <span class="grant">R01 DK025336</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2009</span>;<span class="volume">106</span>;<span class="number">23</span>;<span class="pagerange">9350-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19470471">19470471</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2695078">2695078</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0900909106">10.1073/pnas.0900909106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetics, gene expression and bioinformatics of the pituitary gland.</h4>
<p class="authors"><span class="author">Davis SW</span>, <span class="author">Potok MA</span>, <span class="author">Brinkmeier ML</span>, <span class="author">Carninci P</span>, <span class="author">Lyons RH</span>, <span class="author">MacDonald JW</span>, <span class="author">Fleming MT</span>, <span class="author">Mortensen AH</span>, <span class="author">Egashira N</span>, <span class="author">Ghosh D</span>, <span class="author">Steel KP</span>, <span class="author">Osamura RY</span>, <span class="author">Hayashizaki Y</span> and <span class="author">Camper SA</span></p>
<p class="affiliation">University of Michigan, Ann Arbor, MI 48109-0618, USA.</p><p class="abstract">Genetic cases of congenital pituitary hormone deficiency are common and many are caused by transcription factor defects. Mouse models with orthologous mutations are invaluable for uncovering the molecular mechanisms that lead to problems in organ development and typical patient characteristics. We are using mutant mice defective in the transcription factors PROP1 and POU1F1 for gene expression profiling to identify target genes for these critical transcription factors and candidates for cases of pituitary hormone deficiency of unknown aetiology. These studies reveal critical roles for Wnt signalling pathways, including the TCF/LEF transcription factors and interacting proteins of the groucho family, bone morphogenetic protein antagonists and targets of notch signalling. Current studies are investigating the roles of novel homeobox genes and pathways that regulate the transition from proliferation to differentiation, cell adhesion and cell migration. Pituitary adenomas are a common human health problem, yet most cases are sporadic, necessitating alternative approaches to traditional Mendelian genetic studies. Mouse models of adenoma formation offer the opportunity for gene expression profiling during progressive stages of hyperplasia, adenoma and tumorigenesis. This approach holds promise for the identification of relevant pathways and candidate genes as risk factors for adenoma formation, understanding mechanisms of progression, and identifying drug targets and clinically relevant biomarkers.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD R3730428</span>, <span class="grant">R01 HD030428-07</span>, <span class="grant">R01 HD034283-15</span>, <span class="grant">R01 HD34283</span>, <span class="grant">R37 HD030428-19</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Hormone research</span>&nbsp;<span class="year">2009</span>;<span class="volume">71 Suppl 2</span>;<span class="pagerange">101-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19407506" rel="external">19407506</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3140954" rel="external">3140954</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000192447" rel="external">10.1159/000192447</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A common variant associated with dyslexia reduces expression of the KIAA0319 gene.</h4>
<p class="authors"><span class="author">Dennis MY</span>, <span class="author">Paracchini S</span>, <span class="author">Scerri TS</span>, <span class="author">Prokunina-Olsson L</span>, <span class="author">Knight JC</span>, <span class="author">Wade-Martins R</span>, <span class="author">Coggill P</span>, <span class="author">Beck S</span>, <span class="author">Green ED</span> and <span class="author">Monaco AP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract">Numerous genetic association studies have implicated the KIAA0319 gene on human chromosome 6p22 in dyslexia susceptibility. The causative variant(s) remains unknown but may modulate gene expression, given that (1) a dyslexia-associated haplotype has been implicated in the reduced expression of KIAA0319, and (2) the strongest association has been found for the region spanning exon 1 of KIAA0319. Here, we test the hypothesis that variant(s) responsible for reduced KIAA0319 expression resides on the risk haplotype close to the gene&#39;s transcription start site. We identified seven single-nucleotide polymorphisms on the risk haplotype immediately upstream of KIAA0319 and determined that three of these are strongly associated with multiple reading-related traits. Using luciferase-expressing constructs containing the KIAA0319 upstream region, we characterized the minimal promoter and additional putative transcriptional regulator regions. This revealed that the minor allele of rs9461045, which shows the strongest association with dyslexia in our sample (max p-value = 0.0001), confers reduced luciferase expression in both neuronal and non-neuronal cell lines. Additionally, we found that the presence of this rs9461045 dyslexia-associated allele creates a nuclear protein-binding site, likely for the transcriptional silencer OCT-1. Knocking down OCT-1 expression in the neuronal cell line SHSY5Y using an siRNA restores KIAA0319 expression from the risk haplotype to nearly that seen from the non-risk haplotype. Our study thus pinpoints a common variant as altering the function of a dyslexia candidate gene and provides an illustrative example of the strategic approach needed to dissect the molecular basis of complex genetic traits.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">074318</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">e1000436</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19325871" rel="external">19325871</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2653637" rel="external">2653637</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000436" rel="external">10.1371/journal.pgen.1000436</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The evolution of protein functions and networks: a family-centric approach.</h4>
<p class="authors"><span class="author">Dessailly BH</span>, <span class="author">Reid AJ</span>, <span class="author">Yeats C</span>, <span class="author">Lees JG</span>, <span class="author">Cuff A</span> and <span class="author">Orengo CA</span></p>
<p class="affiliation">Department of Structural and Molecular Biology, University College London, London WC1E 6BT, UK.</p><p class="abstract">The study of superfamilies of protein domains using a combination of structure, sequence and function data provides insights into deep evolutionary history. In the present paper, analyses of functional diversity within such superfamilies as defined in the CATH-Gene3D resource are described. These analyses focus on structure-function relationships in very large and diverse superfamilies, and on the evolution of domain superfamily members in protein-protein complexes.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2009</span>;<span class="volume">37</span>;<span class="number">Pt 4</span>;<span class="pagerange">745-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19614587" rel="external">19614587</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0370745" rel="external">10.1042/BST0370745</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genetic association study in the Gambia using tagging polymorphisms in the major histocompatibility complex class III region implicates a HLA-B associated transcript 2 polymorphism in severe malaria susceptibility.</h4>
<p class="authors"><span class="author">Diakite M</span>, <span class="author">Clark TG</span>, <span class="author">Auburn S</span>, <span class="author">Campino S</span>, <span class="author">Fry AE</span>, <span class="author">Green A</span>, <span class="author">Morris AP</span>, <span class="author">Richardson A</span>, <span class="author">Jallow M</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Pinder M</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Rockett KA</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK.</p><p class="abstract">The tumour necrosis factor (TNF) gene and other genes flanking it in the major histocompatibility complex (MHC) class III region are potentially important mediators of both immunity and pathogenesis of malaria. We investigated the association of severe malaria with 11 haplotype tagging-polymorphisms for 11 MHC class III candidate genes, including TNF, lymphotoxin alpha (LTA), allograft inflammatory factor 1 (AIF1), and HLA-B associated transcript 2 (BAT2). An analysis of 2,162 case-controls demonstrated the first evidence of association between a BAT2 polymorphism (rs1046089) and severe malaria.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0200454(62635)</span>, <span class="grant">G0600230(77610)</span>, <span class="grant">G0600718(80133)</span>; Wellcome Trust: <span class="grant">077383</span>, <span class="grant">081682</span>, <span class="grant">082370</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">125</span>;<span class="number">1</span>;<span class="pagerange">105-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19039607" rel="external">19039607</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2992315" rel="external">2992315</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-008-0597-2" rel="external">10.1007/s00439-008-0597-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common regulatory variation impacts gene expression in a cell type-dependent manner.</h4>
<p class="authors"><span class="author">Dimas AS</span>, <span class="author">Deutsch S</span>, <span class="author">Stranger BE</span>, <span class="author">Montgomery SB</span>, <span class="author">Borel C</span>, <span class="author">Attar-Cohen H</span>, <span class="author">Ingle C</span>, <span class="author">Beazley C</span>, <span class="author">Gutierrez Arcelus M</span>, <span class="author">Sekowska M</span>, <span class="author">Gagnebin M</span>, <span class="author">Nisbett J</span>, <span class="author">Deloukas P</span>, <span class="author">Dermitzakis ET</span> and <span class="author">Antonarakis SE</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, CB10 1HH, Cambridge, UK.</p><p class="abstract">Studies correlating genetic variation to gene expression facilitate the interpretation of common human phenotypes and disease. As functional variants may be operating in a tissue-dependent manner, we performed gene expression profiling and association with genetic variants (single-nucleotide polymorphisms) on three cell types of 75 individuals. We detected cell type-specific genetic effects, with 69 to 80% of regulatory variants operating in a cell type-specific manner, and identified multiple expressive quantitative trait loci (eQTLs) per gene, unique or shared among cell types and positively correlated with the number of transcripts per gene. Cell type-specific eQTLs were found at larger distances from genes and at lower effect size, similar to known enhancers. These data suggest that the complete regulatory variant repertoire can only be uncovered in the context of cell-type specificity.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span>, <span class="grant">077046</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">325</span>;<span class="number">5945</span>;<span class="pagerange">1246-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19644074">19644074</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2867218">2867218</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1174148">10.1126/science.1174148</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ectopic recombination of a malaria var gene during mitosis associated with an altered var switch rate.</h4>
<p class="authors"><span class="author">Duffy MF</span>, <span class="author">Byrne TJ</span>, <span class="author">Carret C</span>, <span class="author">Ivens A</span> and <span class="author">Brown GV</span></p>
<p class="affiliation">Department of Medicine at RMH, University of Melbourne, Parkville 3050, Australia. <a href="mailto:%6D%64%75%66%66%79%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#109;&#100;&#117;&#102;&#102;&#121;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">The Plasmodium falciparum var multigene family encodes P. falciparum erythrocyte membrane protein 1, which is responsible for the pathogenic traits of antigenic variation and adhesion of infected erythrocytes to host receptors during malaria infection. Clonal antigenic variation of P. falciparum erythrocyte membrane protein 1 is controlled by the switching between exclusively transcribed var genes. The tremendous diversity of the var gene repertoire both within and between parasite strains is critical for the parasite&#39;s strategy of immune evasion. We show that ectopic recombination between var genes occurs during mitosis, providing P. falciparum with opportunities to diversify its var repertoire, even during the course of a single infection. We show that the regulation of the recombined var gene has been disrupted, resulting in its persistent activation although the regulation of most other var genes is unaffected. The var promoter and intron of the recombined var gene are not responsible for its atypically persistent activity, and we conclude that altered subtelomeric cis sequence is the most likely cause of the persistent activity of the recombined var gene.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">389</span>;<span class="number">3</span>;<span class="pagerange">453-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19389407">19389407</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3898907">3898907</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jmb.2009.04.032">10.1016/j.jmb.2009.04.032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic Loci associated with C-reactive protein levels and risk of coronary heart disease.</h4>
<p class="authors"><span class="author">Elliott P</span>, <span class="author">Chambers JC</span>, <span class="author">Zhang W</span>, <span class="author">Clarke R</span>, <span class="author">Hopewell JC</span>, <span class="author">Peden JF</span>, <span class="author">Erdmann J</span>, <span class="author">Braund P</span>, <span class="author">Engert JC</span>, <span class="author">Bennett D</span>, <span class="author">Coin L</span>, <span class="author">Ashby D</span>, <span class="author">Tzoulaki I</span>, <span class="author">Brown IJ</span>, <span class="author">Mt-Isa S</span>, <span class="author">McCarthy MI</span>, <span class="author">Peltonen L</span>, <span class="author">Freimer NB</span>, <span class="author">Farrall M</span>, <span class="author">Ruokonen A</span>, <span class="author">Hamsten A</span>, <span class="author">Lim N</span>, <span class="author">Froguel P</span>, <span class="author">Waterworth DM</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Jarvelin MR</span>, <span class="author">Mooser V</span>, <span class="author">Scott J</span>, <span class="author">Hall AS</span>, <span class="author">Schunkert H</span>, <span class="author">Anand SS</span>, <span class="author">Collins R</span>, <span class="author">Samani NJ</span>, <span class="author">Watkins H</span> and <span class="author">Kooner JS</span></p>
<p class="affiliation">Faculty of Medicine, Imperial College London, London, United Kingdom. <a href="mailto:%70.%65%6C%6C%69%6F%74%74%40%69%6D%70%65%72%69%61%6C.%61%63.%75%6B">&#112;&#46;&#101;&#108;&#108;&#105;&#111;&#116;&#116;&#64;&#105;&#109;&#112;&#101;&#114;&#105;&#97;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Context</strong>: Plasma levels of C-reactive protein (CRP) are independently associated with risk of coronary heart disease, but whether CRP is causally associated with coronary heart disease or merely a marker of underlying atherosclerosis is uncertain.</p><p class="abstract"><strong>Objective</strong>: To investigate association of genetic loci with CRP levels and risk of coronary heart disease.</p><p class="abstract">We first carried out a genome-wide association (n = 17,967) and replication study (n = 13,615) to identify genetic loci associated with plasma CRP concentrations. Data collection took place between 1989 and 2008 and genotyping between 2003 and 2008. We carried out a mendelian randomization study of the most closely associated single-nucleotide polymorphism (SNP) in the CRP locus and published data on other CRP variants involving a total of 28,112 cases and 100,823 controls, to investigate the association of CRP variants with coronary heart disease. We compared our finding with that predicted from meta-analysis of observational studies of CRP levels and risk of coronary heart disease. For the other loci associated with CRP levels, we selected the most closely associated SNP for testing against coronary heart disease among 14,365 cases and 32,069 controls.</p><p class="abstract">Risk of coronary heart disease.</p><p class="abstract"><strong>Results</strong>: Polymorphisms in 5 genetic loci were strongly associated with CRP levels (% difference per minor allele): SNP rs6700896 in LEPR (-14.8%; 95% confidence interval [CI], -17.6% to -12.0%; P = 6.2 x 10(-22)), rs4537545 in IL6R (-11.5%; 95% CI, -14.4% to -8.5%; P = 1.3 x 10(-12)), rs7553007 in the CRP locus (-20.7%; 95% CI, -23.4% to -17.9%; P = 1.3 x 10(-38)), rs1183910 in HNF1A (-13.8%; 95% CI, -16.6% to -10.9%; P = 1.9 x 10(-18)), and rs4420638 in APOE-CI-CII (-21.8%; 95% CI, -25.3% to -18.1%; P = 8.1 x 10(-26)). Association of SNP rs7553007 in the CRP locus with coronary heart disease gave an odds ratio (OR) of 0.98 (95% CI, 0.94 to 1.01) per 20% lower CRP level. Our mendelian randomization study of variants in the CRP locus showed no association with coronary heart disease: OR, 1.00; 95% CI, 0.97 to 1.02; per 20% lower CRP level, compared with OR, 0.94; 95% CI, 0.94 to 0.95; predicted from meta-analysis of the observational studies of CRP levels and coronary heart disease (z score, -3.45; P &lt; .001). SNPs rs6700896 in LEPR (OR, 1.06; 95% CI, 1.02 to 1.09; per minor allele), rs4537545 in IL6R (OR, 0.94; 95% CI, 0.91 to 0.97), and rs4420638 in the APOE-CI-CII cluster (OR, 1.16; 95% CI, 1.12 to 1.21) were all associated with risk of coronary heart disease.</p><p class="abstract"><strong>Conclusion</strong>: The lack of concordance between the effect on coronary heart disease risk of CRP genotypes and CRP levels argues against a causal association of CRP with coronary heart disease.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">LSHM33 CT-2007-037273</span>, <span class="grant">SP/04/002</span>; Cancer Research UK; Department of Health; Medical Research Council; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">JAMA : the journal of the American Medical Association</span>&nbsp;<span class="year">2009</span>;<span class="volume">302</span>;<span class="number">1</span>;<span class="pagerange">37-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19567438" rel="external">19567438</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2803020" rel="external">2803020</a>; DOI: <a href="http://dx.doi.org/10.1001%2Fjama.2009.954" rel="external">10.1001/jama.2009.954</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies variants at 9p21 and 22q13 associated with development of cutaneous nevi.</h4>
<p class="authors"><span class="author">Falchi M</span>, <span class="author">Bataille V</span>, <span class="author">Hayward NK</span>, <span class="author">Duffy DL</span>, <span class="author">Bishop JA</span>, <span class="author">Pastinen T</span>, <span class="author">Cervino A</span>, <span class="author">Zhao ZZ</span>, <span class="author">Deloukas P</span>, <span class="author">Soranzo N</span>, <span class="author">Elder DE</span>, <span class="author">Barrett JH</span>, <span class="author">Martin NG</span>, <span class="author">Bishop DT</span>, <span class="author">Montgomery GW</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Department of Twin Research &amp; Genetic Epidemiology, Kings College London, St. Thomas&#39; Hospital Campus, London, UK. <a href="mailto:%6D.%66%61%6C%63%68%69%40%69%6D%70%65%72%69%61%6C.%61%63.%75%6B">&#109;&#46;&#102;&#97;&#108;&#99;&#104;&#105;&#64;&#105;&#109;&#112;&#101;&#114;&#105;&#97;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A high melanocytic nevi count is the strongest known risk factor for cutaneous melanoma. We conducted a genome-wide association study for nevus count using 297,108 SNPs in 1,524 twins, with validation in an independent cohort of 4,107 individuals. We identified strongly associated variants in MTAP, a gene adjacent to the familial melanoma susceptibility locus CDKN2A on 9p21 (rs4636294, combined P = 3.4 x 10(-15)), as well as in PLA2G6 on 22q13.1 (rs2284063, combined P = 3.4 x 10(-8)). In addition, variants in these two loci showed association with melanoma risk in 3,131 melanoma cases from two independent studies, including rs10757257 at 9p21, combined P = 3.4 x 10(-8), OR = 1.23 (95% CI = 1.15-1.30) and rs132985 at 22q13.1, combined P = 2.6 x 10(-7), OR = 1.23 (95% CI = 1.15-1.30). This provides the first report of common variants associated to nevus number and demonstrates association of these variants with melanoma susceptibility.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10589</span>, <span class="grant">C588/A4994</span>; Department of Health; NCI NIH HHS: <span class="grant">CA88363</span>, <span class="grant">R01 CA083115</span>, <span class="grant">R01 CA083115-08</span>, <span class="grant">R01 CA83115</span>; Wellcome Trust: <span class="grant">077011</span>, <span class="grant">091746</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">8</span>;<span class="pagerange">915-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19578365">19578365</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3080738">3080738</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.410">10.1038/ng.410</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeted tandem affinity purification of PSD-95 recovers core postsynaptic complexes and schizophrenia susceptibility proteins.</h4>
<p class="authors"><span class="author">Fern&aacute;ndez E</span>, <span class="author">Collins MO</span>, <span class="author">Uren RT</span>, <span class="author">Kopanitsa MV</span>, <span class="author">Komiyama NH</span>, <span class="author">Croning MD</span>, <span class="author">Zografos L</span>, <span class="author">Armstrong JD</span>, <span class="author">Choudhary JS</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">The molecular complexity of mammalian proteomes demands new methods for mapping the organization of multiprotein complexes. Here, we combine mouse genetics and proteomics to characterize synapse protein complexes and interaction networks. New tandem affinity purification (TAP) tags were fused to the carboxyl terminus of PSD-95 using gene targeting in mice. Homozygous mice showed no detectable abnormalities in PSD-95 expression, subcellular localization or synaptic electrophysiological function. Analysis of multiprotein complexes purified under native conditions by mass spectrometry defined known and new interactors: 118 proteins comprising crucial functional components of synapses, including glutamate receptors, K+ channels, scaffolding and signaling proteins, were recovered. Network clustering of protein interactions generated five connected clusters, with two clusters containing all the major ionotropic glutamate receptors and one cluster with voltage-dependent K+ channels. Annotation of clusters with human disease associations revealed that multiple disorders map to the network, with a significant correlation of schizophrenia within the glutamate receptor clusters. This targeted TAP tagging strategy is generally applicable to mammalian proteomics and systems biology approaches to disease.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular systems biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="pagerange">269</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19455133">19455133</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2694677">2694677</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmsb.2009.27">10.1038/msb.2009.27</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Convergent extension movements and ciliary function are mediated by ofd1, a zebrafish orthologue of the human oral-facial-digital type 1 syndrome gene.</h4>
<p class="authors"><span class="author">Ferrante MI</span>, <span class="author">Romio L</span>, <span class="author">Castro S</span>, <span class="author">Collins JE</span>, <span class="author">Goulding DA</span>, <span class="author">Stemple DL</span>, <span class="author">Woolf AS</span> and <span class="author">Wilson SW</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK.</p><p class="abstract">In humans, OFD1 is mutated in oral-facial-digital type I syndrome leading to prenatal death in hemizygous males and dysmorphic faces and brain malformations, with polycystic kidneys presenting later in life in heterozygous females. To elucidate the function of Ofd1, we have studied its function during zebrafish embryonic development. In wild-type embryos, ofd1 mRNA is widely expressed and Ofd1-green fluorescent protein (GFP) fusion localizes to the centrosome/basal body. Disrupting Ofd1 using antisense morpholinos (MOs) led to bent body axes, hydrocephalus and oedema. Laterality was randomized in the brain, heart and viscera, likely a consequence of shorter cilia with disrupted axonemes and perturbed intravesicular fluid flow in Kupffer&#39;s vesicle. Embryos injected with ofd1 MOs also displayed convergent extension (CE) defects, which were enhanced by loss of Slb/Wnt11 or Tri/Vangl2, two proteins functioning in a non-canonical Wnt/Planar Cell Polarity (PCP) pathway. Pronephric glomerular midline fusion was compromised in vangl2 and ofd1 loss of function embryos and we suggest this anomaly may be a novel CE defect. Thus, Ofd1 is required for ciliary motility and function in zebrafish, supporting data showing that Ofd1 is essential for primary cilia function in mice. In addition, our data show that Ofd1 is important for CE during gastrulation, consistent with data linking primary cilia and non-canonical Wnt/PCP signalling.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust: <span class="grant">075311</span>, <span class="grant">WT077037/Z/05/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">2</span>;<span class="pagerange">289-303</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18971206" rel="external">18971206</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2638777" rel="external">2638777</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddn356" rel="external">10.1093/hmg/ddn356</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting Report: "Metagenomics, Metadata and Meta-analysis" (M3) Special Interest Group at ISMB 2009.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Friedberg I</span>, <span class="author">Sterk P</span>, <span class="author">Kottmann R</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Hirschman L</span>, <span class="author">Garrity GM</span>, <span class="author">Cochrane G</span>, <span class="author">Wooley J</span> and <span class="author">Gilbert J</span></p>
<p class="abstract">This report summarizes the proceedings of the "Metagenomics, Metadata and Meta-analysis" (M3) Special Interest Group (SIG) meeting held at the Intelligent Systems for Molecular Biology 2009 conference. The Genomic Standards Consortium (GSC) hosted this meeting to explore the bottlenecks and emerging solutions for obtaining biological insights through large-scale comparative analysis of metagenomic datasets. The M3 SIG included 16 talks, half of which were selected from submitted abstracts, a poster session and a panel discussion involving members of the GSC Board. This report summarizes this one-day SIG, attempts to identify shared themes and recapitulates community recommendations for the future of this field. The GSC will also host an M3 workshop at the Pacific Symposium on Biocomputing (PSB) in January 2010. Further information about the GSC and its range of activities can be found at http://gensc.org/.</p>
<p class="publication"><span class="title">Standards in genomic sciences</span>&nbsp;<span class="year">2009</span>;<span class="volume">1</span>;<span class="number">3</span>;<span class="pagerange">278-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21304668" rel="external">21304668</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3035241" rel="external">3035241</a>; DOI: <a href="http://dx.doi.org/10.4056%2Fsigs.641096" rel="external">10.4056/sigs.641096</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting Report from the Genomic Standards Consortium (GSC) Workshops 6 and 7.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Sterk P</span>, <span class="author">Kyrpides N</span>, <span class="author">Kottmann R</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Hirschman L</span>, <span class="author">Garrity GM</span>, <span class="author">Wooley J</span> and <span class="author">Gilna P</span></p>
<p class="abstract">This report summarizes the proceedings of the 6th and 7th workshops of the Genomic Standards Consortium (GSC), held back-to-back in 2008. GSC 6 focused on furthering the activities of GSC working groups, GSC 7 focused on outreach to the wider community. GSC 6 was held October 10-14, 2008 at the European Bioinformatics Institute, Cambridge, United Kingdom and included a two-day workshop focused on the refinement of the Genomic Contextual Data Markup Language (GCDML). GSC 7 was held as the opening day of the International Congress on Metagenomics 2008 in San Diego California. Major achievements of these combined meetings included an agreement from the International Nucleotide Sequence Database Consortium (INSDC) to create a "MIGS" keyword for capturing "Minimum Information about a Genome Sequence" compliant information within INSDC (DDBJ/EMBL /Genbank) records, launch of GCDML 1.0, MIGS compliance of the first set of "Genomic Encyclopedia of Bacteria and Archaea" project genomes, approval of a proposal to extend MIGS to 16S rRNA sequences within a "Minimum Information about an Environmental Sequence", finalization of plans for the GSC eJournal, "Standards in Genomic Sciences" (SIGS), and the formation of a GSC Board. Subsequently, the GSC has been awarded a Research Co-ordination Network (RCN4GSC) grant from the National Science Foundation, held the first SIGS workshop and launched the journal. The GSC will also be hosting outreach workshops at both ISMB 2009 and PSB 2010 focused on "Metagenomics, Metadata and MetaAnalysis" (M(3)). Further information about the GSC and its range of activities can be found at http://gensc.org, including videos of all the presentations at GSC 7.</p>
<p class="publication"><span class="title">Standards in genomic sciences</span>&nbsp;<span class="year">2009</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">68-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21304639" rel="external">21304639</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3035212" rel="external">3035212</a>; DOI: <a href="http://dx.doi.org/10.4056%2Fsigs.25165" rel="external">10.4056/sigs.25165</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources.</h4>
<p class="authors"><span class="author">Firth HV</span>, <span class="author">Richards SM</span>, <span class="author">Bevan AP</span>, <span class="author">Clayton S</span>, <span class="author">Corpas M</span>, <span class="author">Rajan D</span>, <span class="author">Van Vooren S</span>, <span class="author">Moreau Y</span>, <span class="author">Pettett RM</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Cambridge University Department of Medical Genetics, Addenbrooke&#39;s Hospital, Cambridge CB2 2QQ, UK. <a href="mailto:%68%76%66%32%31%40%63%61%6D.%61%63.%75%6B">&#104;&#118;&#102;&#50;&#49;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Many patients suffering from developmental disorders harbor submicroscopic deletions or duplications that, by affecting the copy number of dosage-sensitive genes or disrupting normal gene expression, lead to disease. However, many aberrations are novel or extremely rare, making clinical interpretation problematic and genotype-phenotype correlations uncertain. Identification of patients sharing a genomic rearrangement and having phenotypic features in common leads to greater certainty in the pathogenic nature of the rearrangement and enables new syndromes to be defined. To facilitate the analysis of these rare events, we have developed an interactive web-based database called DECIPHER (Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources) which incorporates a suite of tools designed to aid the interpretation of submicroscopic chromosomal imbalance, inversions, and translocations. DECIPHER catalogs common copy-number changes in normal populations and thus, by exclusion, enables changes that are novel and potentially pathogenic to be identified. DECIPHER enhances genetic counseling by retrieving relevant information from a variety of bioinformatics resources. Known and predicted genes within an aberration are listed in the DECIPHER patient report, and genes of recognized clinical importance are highlighted and prioritized. DECIPHER enables clinical scientists worldwide to maintain records of phenotype and chromosome rearrangement for their patients and, with informed consent, share this information with the wider clinical research community through display in the genome browser Ensembl. By sharing cases worldwide, clusters of rare cases having phenotype and structural rearrangement in common can be identified, leading to the delineation of new syndromes and furthering understanding of gene function.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">84</span>;<span class="number">4</span>;<span class="pagerange">524-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19344873">19344873</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2667985">2667985</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2009.03.010">10.1016/j.ajhg.2009.03.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic association study for RSV bronchiolitis in infancy at the 5q31 cytokine cluster.</h4>
<p class="authors"><span class="author">Forton JT</span>, <span class="author">Rowlands K</span>, <span class="author">Rockett K</span>, <span class="author">Hanchard N</span>, <span class="author">Herbert M</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Hull J</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, University of Oxford, UK. <a href="mailto:%6A%75%6C%69%61%6E.%66%6F%72%74%6F%6E%40%70%61%65%64%69%61%74%72%69%63%73.%6F%78.%61%63.%75%6B">&#106;&#117;&#108;&#105;&#97;&#110;&#46;&#102;&#111;&#114;&#116;&#111;&#110;&#64;&#112;&#97;&#101;&#100;&#105;&#97;&#116;&#114;&#105;&#99;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The pathophysiological basis of severe respiratory syncytial virus (RSV) bronchiolitis in infancy is poorly understood and has hindered vaccine development. Studies implicate the cell-mediated immune response in the pathogenesis of the disease. A recent twin study estimated a heritable contribution of 22% to RSV bronchiolitis. Genetic epidemiology provides a new approach to identifying important immune determinants of disease severity.</p><p class="abstract"><strong>Methods</strong>: A comprehensive high-density gene-region association study for severe RSV bronchiolitis in infancy at 5q31 across 11 genes including the Th2-cytokine cluster was performed. A haplotype tagging approach was used to analyse genetic variation at 113 single nucleotide polymorphisms (SNPs) in 780 independent cases and 1045 controls. The study had sufficient power to detect small effects, perform extensive haplotype analysis and analyse both a principal phenotype and a refined age-limited phenotype enriched for first-exposure RSV infection.</p><p class="abstract"><strong>Results</strong>: SNP associations were found at IL4 and a highly significant risk haplotype was identified across IL13 CNS-1 and IL4 (odds ratio 1.69, p&lt;0.0001), present in both case-control and family-based analyses. All associations were strongest for a phenotype limited to &lt;6 months of age, implicating this locus in primary RSV disease. The same risk haplotype has previously been shown to be associated with increased IL13 expression.</p><p class="abstract"><strong>Conclusions</strong>: A haplotype at IL13-1L4, which is associated with increased IL13 production, confers an increased risk of severe primary RSV bronchiolitis in early infancy. This study, together with previous studies implicating the same locus in atopic sensitisation, suggests that primary RSV bronchiolitis and atopy share a genetic contribution at the IL13-IL4 locus.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">071472</span>, <span class="grant">082370</span></p>
<p class="publication"><span class="title">Thorax</span>&nbsp;<span class="year">2009</span>;<span class="volume">64</span>;<span class="number">4</span>;<span class="pagerange">345-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19131452" rel="external">19131452</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3015100" rel="external">3015100</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fthx.2008.102111" rel="external">10.1136/thx.2008.102111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rfam: updates to the RNA families database.</h4>
<p class="authors"><span class="author">Gardner PP</span>, <span class="author">Daub J</span>, <span class="author">Tate JG</span>, <span class="author">Nawrocki EP</span>, <span class="author">Kolbe DL</span>, <span class="author">Lindgreen S</span>, <span class="author">Wilkinson AC</span>, <span class="author">Finn RD</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Eddy SR</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK. <a href="mailto:%70%67%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#103;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Rfam is a collection of RNA sequence families, represented by multiple sequence alignments and covariance models (CMs). The primary aim of Rfam is to annotate new members of known RNA families on nucleotide sequences, particularly complete genomes, using sensitive BLAST filters in combination with CMs. A minority of families with a very broad taxonomic range (e.g. tRNA and rRNA) provide the majority of the sequence annotations, whilst the majority of Rfam families (e.g. snoRNAs and miRNAs) have a limited taxonomic range and provide a limited number of annotations. Recent improvements to the website, methodologies and data used by Rfam are discussed. Rfam is freely available on the Web at http://rfam.sanger.ac.uk/and http://rfam.janelia.org/.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">077044</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2009</span>;<span class="volume">37</span>;<span class="number">Database issue</span>;<span class="pagerange">D136-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18953034">18953034</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2686503">2686503</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkn766">10.1093/nar/gkn766</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A home for RNA families at RNA Biology</h4>
<p class="authors"><span class="author">Gardner</span>, <span class="author">P. P</span>, <span class="author">Bateman</span>, <span class="author">A. G.</span></p>
<p class="publication"><span class="title">Rna Biology</span> <span class="year">2009</span>;<span class="volume">6</span>;<span class="pagerange">2-4</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reduced TFAP2A function causes variable optic fissure closure and retinal defects and sensitizes eye development to mutations in other morphogenetic regulators.</h4>
<p class="authors"><span class="author">Gestri G</span>, <span class="author">Osborne RJ</span>, <span class="author">Wyatt AW</span>, <span class="author">Gerrelli D</span>, <span class="author">Gribble S</span>, <span class="author">Stewart H</span>, <span class="author">Fryer A</span>, <span class="author">Bunyan DJ</span>, <span class="author">Prescott K</span>, <span class="author">Collin JR</span>, <span class="author">Fitzgerald T</span>, <span class="author">Robinson D</span>, <span class="author">Carter NP</span>, <span class="author">Wilson SW</span> and <span class="author">Ragge NK</span></p>
<p class="affiliation">Department of Cell and Developmental Biology, UCL, London, UK.</p><p class="abstract">Mutations in the transcription factor encoding TFAP2A gene underlie branchio-oculo-facial syndrome (BOFS), a rare dominant disorder characterized by distinctive craniofacial, ocular, ectodermal and renal anomalies. To elucidate the range of ocular phenotypes caused by mutations in TFAP2A, we took three approaches. First, we screened a cohort of 37 highly selected individuals with severe ocular anomalies plus variable defects associated with BOFS for mutations or deletions in TFAP2A. We identified one individual with a de novo TFAP2A four amino acid deletion, a second individual with two non-synonymous variations in an alternative splice isoform TFAP2A2, and a sibling-pair with a paternally inherited whole gene deletion with variable phenotypic expression. Second, we determined that TFAP2A is expressed in the lens, neural retina, nasal process, and epithelial lining of the oral cavity and palatal shelves of human and mouse embryos--sites consistent with the phenotype observed in patients with BOFS. Third, we used zebrafish to examine how partial abrogation of the fish ortholog of TFAP2A affects the penetrance and expressivity of ocular phenotypes due to mutations in genes encoding bmp4 or tcf7l1a. In both cases, we observed synthetic, enhanced ocular phenotypes including coloboma and anophthalmia when tfap2a is knocked down in embryos with bmp4 or tcf7l1a mutations. These results reveal that mutations in TFAP2A are associated with a wide range of eye phenotypes and that hypomorphic tfap2a mutations can increase the risk of developmental defects arising from mutations at other loci.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501487</span>, <span class="grant">G0700089</span>; Wellcome Trust: <span class="grant">074376</span>, <span class="grant">078047</span>, <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">126</span>;<span class="number">6</span>;<span class="pagerange">791-803</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19685247">19685247</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3083835">3083835</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-009-0730-x">10.1007/s00439-009-0730-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neonates harbour highly active gammadelta T cells with selective impairments in preterm infants.</h4>
<p class="authors"><span class="author">Gibbons DL</span>, <span class="author">Haque SF</span>, <span class="author">Silberzahn T</span>, <span class="author">Hamilton K</span>, <span class="author">Langford C</span>, <span class="author">Ellis P</span>, <span class="author">Carr R</span> and <span class="author">Hayday AC</span></p>
<p class="affiliation">Peter Gorer Department of Immunobiology, London, UK.</p><p class="abstract">Acknowledgement of the breadth of T-cell pleiotropy has provoked increasing interest in the degree to which functional responsiveness is elicited by environmental cues versus differentiation. This is particularly relevant for young animals requiring rapid responses to acute environmental exposure. In young mice, gammadelta T cells are disproportionately important for immuno-protection. To examine the situation in humans, we compared populations and clones of T cells from term and preterm babies, and adults. By comparison with alphabeta T cells, neonate-derived gammadelta cells show stronger, pleiotropic functional responsiveness, and lack signatory deficits in IFN-gamma production. Emphasising the acquisition of functional competence in utero, IFN-gamma was produced by gammadelta cells sampled from premature births, and, although one month&#39;s post-partum environmental exposure invariably increased their TNF-alpha production, it had no consistent effect on IFN-gamma or IL-2. In sum, gammadelta cells seem well positioned at birth to contribute to immuno-protection and immuno-regulation, possibly compensating for selective immaturity in the alphabeta compartment. With regard to the susceptibilities of preterm babies to viral infection, gammadelta cells from preterm neonates were commonly impaired in Toll-like receptor-3 and -7 expression and compared with cells from term babies failed to optimise cytokine production in response to coincident TCR and TLR agonists.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">R0161799</span>; Wellcome Trust: <span class="grant">071534</span></p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">2009</span>;<span class="volume">39</span>;<span class="number">7</span>;<span class="pagerange">1794-806</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19544311">19544311</a>; DOI: <a href="http://dx.doi.org/10.1002%2Feji.200939222">10.1002/eji.200939222</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The porin OmpD from nontyphoidal Salmonella is a key target for a protective B1b cell antibody response.</h4>
<p class="authors"><span class="author">Gil-Cruz C</span>, <span class="author">Bobat S</span>, <span class="author">Marshall JL</span>, <span class="author">Kingsley RA</span>, <span class="author">Ross EA</span>, <span class="author">Henderson IR</span>, <span class="author">Leyton DL</span>, <span class="author">Coughlan RE</span>, <span class="author">Khan M</span>, <span class="author">Jensen KT</span>, <span class="author">Buckley CD</span>, <span class="author">Dougan G</span>, <span class="author">MacLennan IC</span>, <span class="author">L&oacute;pez-Mac&iacute;as C</span> and <span class="author">Cunningham AF</span></p>
<p class="affiliation">School of Immunity and Infection and Medical Research Council Centre for Immune Regulation, University of Birmingham, Birmingham B15 2TT, United Kingdom.</p><p class="abstract">Invasive nontyphoidal Salmonella (NTS), including Salmonella typhimurium (STm), are major yet poorly-recognized killers of infants in sub-Saharan Africa. Death in these children is usually associated with bacteremia, commonly in the absence of gastrointestinal symptoms. Evidence from humans and animal studies suggest that severe infection and bacteremia occur when specific Ab is lacking. Understanding how Ab responses to Salmonella are regulated will help develop vaccines against these devastating infections. STm induces atypical Ab responses characterized by prominent, accelerated, extrafollicular T-independent (TI) Ab against a range of surface antigens. These responses develop without concomitant germinal centers, which only appear as infection resolves. Here, we show STm rapidly induces a population of TI B220(+)CD5(-) B1b cells during infection and TI Ab from B1b cells targets the outer membrane protein (Omp) porins OmpC, OmpD and OmpF but not flagellin. When porins are used as immunogens they can ablate bacteremia and provide equivalent protection against STm as killed bacterial vaccine and this is wholly B cell-dependent. Furthermore Ab from porin-immunized chimeras, that have B1b cells, is sufficient to impair infection. Infecting with porin-deficient bacteria identifies OmpD, a protein absent from Salmonella Typhi, as a key target of Ab in these infections. This work broadens the recognized repertoire of TI protein antigens and highlights the importance of Ab from different B cell subsets in controlling STm infection. OmpD is a strong candidate vaccine target and may, in part, explain the lack of cross-protection between Salmonella Typhi and STm infections.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2009</span>;<span class="volume">106</span>;<span class="number">24</span>;<span class="pagerange">9803-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19487686" rel="external">19487686</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2701014" rel="external">2701014</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0812431106" rel="external">10.1073/pnas.0812431106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A general basis for cognition in the evolution of synapse signaling complexes.</h4>
<p class="authors"><span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Cambridge, United Kingdom. <a href="mailto:%73%67%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Beneath the complexity of the human brain are molecular principles shaped by evolution explaining the origins of the behavioral repertoire. The role of the nervous system is to provide a repertoire of behaviors allowing the animal to respond and adapt to changing environments during the course of its life. Multiprotein complexes in the postsynaptic terminal of synapses control adaptive and cognitive processes in metazoan nervous systems. These multiprotein complexes are organized into molecular networks that detect and respond to patterns of neural activity. Combinations of proteins are used to build different complexes and pathways producing great diversity. These complexes evolved from an ancestral core set of proteins controlling adaptive behaviors in unicellular organisms known as the protosynapse. Later expansion in numbers and interactions resulted in more complex synapses in invertebrates and vertebrates. The resultant combinatorial complexity has contributed to the neuroanatomical, neurophysiological, and behavioral diversity in these species. Mutations in genes encoding the complexes result in many human diseases of the nervous system. This general mechanism of cognition provides a useful template for studying evolution of behavior in all animals.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">74</span>;<span class="pagerange">249-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19955255">19955255</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fsqb.2009.74.033">10.1101/sqb.2009.74.033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and epigenetic evidence for oxytocin receptor deficiency in autism.</h4>
<p class="authors"><span class="author">Gregory SG</span>, <span class="author">Connelly JJ</span>, <span class="author">Towers AJ</span>, <span class="author">Johnson J</span>, <span class="author">Biscocho D</span>, <span class="author">Markunas CA</span>, <span class="author">Lintas C</span>, <span class="author">Abramson RK</span>, <span class="author">Wright HH</span>, <span class="author">Ellis P</span>, <span class="author">Langford CF</span>, <span class="author">Worley G</span>, <span class="author">Delong GR</span>, <span class="author">Murphy SK</span>, <span class="author">Cuccaro ML</span>, <span class="author">Persico A</span> and <span class="author">Pericak-Vance MA</span></p>
<p class="affiliation">Duke Center for Human Genetics, DUMC, Durham, NC, USA. <a href="mailto:%73%69%6D%6F%6E.%67%72%65%67%6F%72%79%40%64%75%6B%65.%65%64%75">&#115;&#105;&#109;&#111;&#110;&#46;&#103;&#114;&#101;&#103;&#111;&#114;&#121;&#64;&#100;&#117;&#107;&#101;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Autism comprises a spectrum of behavioral and cognitive disturbances of childhood development and is known to be highly heritable. Although numerous approaches have been used to identify genes implicated in the development of autism, less than 10% of autism cases have been attributed to single gene disorders.</p><p class="abstract"><strong>Methods</strong>: We describe the use of high-resolution genome-wide tilepath microarrays and comparative genomic hybridization to identify copy number variants within 119 probands from multiplex autism families. We next carried out DNA methylation analysis by bisulfite sequencing in a proband and his family, expanding this analysis to methylation analysis of peripheral blood and temporal cortex DNA of autism cases and matched controls from independent datasets. We also assessed oxytocin receptor (OXTR) gene expression within the temporal cortex tissue by quantitative real-time polymerase chain reaction (PCR).</p><p class="abstract"><strong>Results</strong>: Our analysis revealed a genomic deletion containing the oxytocin receptor gene, OXTR (MIM accession no.: 167055), previously implicated in autism, was present in an autism proband and his mother who exhibits symptoms of obsessive-compulsive disorder. The proband&#39;s affected sibling did not harbor this deletion but instead may exhibit epigenetic misregulation of this gene through aberrant gene silencing by DNA methylation. Further DNA methylation analysis of the CpG island known to regulate OXTR expression identified several CpG dinucleotides that show independent statistically significant increases in the DNA methylation status in the peripheral blood cells and temporal cortex in independent datasets of individuals with autism as compared to control samples. Associated with the increase in methylation of these CpG dinucleotides is our finding that OXTR mRNA showed decreased expression in the temporal cortex tissue of autism cases matched for age and sex compared to controls.</p><p class="abstract"><strong>Conclusion</strong>: Together, these data provide further evidence for the role of OXTR and the oxytocin signaling pathway in the etiology of autism and, for the first time, implicate the epigenetic regulation of OXTR in the development of the disorder.See the related commentary by Gurrieri and Neri: http://www.biomedcentral.com/1741-7015/7/63.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">P01-NS026630</span>; PHS HHS: <span class="grant">R01-NIH080647</span></p>
<p class="publication"><span class="title">BMC medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="pagerange">62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19845972" rel="external">19845972</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2774338" rel="external">2774338</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1741-7015-7-62" rel="external">10.1186/1741-7015-7-62</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic utility of broadly defined bipolar schizoaffective disorder as a diagnostic concept.</h4>
<p class="authors"><span class="author">Hamshere ML</span>, <span class="author">Green EK</span>, <span class="author">Jones IR</span>, <span class="author">Jones L</span>, <span class="author">Moskvina V</span>, <span class="author">Kirov G</span>, <span class="author">Grozeva D</span>, <span class="author">Nikolov I</span>, <span class="author">Vukcevic D</span>, <span class="author">Caesar S</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Fraser C</span>, <span class="author">Russell E</span>, <span class="author">Breen G</span>, <span class="author">St Clair D</span>, <span class="author">Collier DA</span>, <span class="author">Young AH</span>, <span class="author">Ferrier IN</span>, <span class="author">Farmer A</span>, <span class="author">McGuffin P</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Holmans PA</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span> and <span class="author">Craddock N</span></p>
<p class="affiliation">Biostatistics and Bioinformatics Unit and Department of Psychological Medicine, School of Medicine, Cardiff University, Heath Park, Cardiff CF14 4XN, UK.</p><p class="abstract"><strong>Background</strong>: Psychiatric phenotypes are currently defined according to sets of descriptive criteria. Although many of these phenotypes are heritable, it would be useful to know whether any of the various diagnostic categories in current use identify cases that are particularly helpful for biological-genetic research.</p><p class="abstract"><strong>Aims</strong>: To use genome-wide genetic association data to explore the relative genetic utility of seven different descriptive operational diagnostic categories relevant to bipolar illness within a large UK case-control bipolar disorder sample.</p><p class="abstract"><strong>Method</strong>: We analysed our previously published Wellcome Trust Case Control Consortium (WTCCC) bipolar disorder genome-wide association data-set, comprising 1868 individuals with bipolar disorder and 2938 controls genotyped for 276 122 single nucleotide polymorphisms (SNPs) that met stringent criteria for genotype quality. For each SNP we performed a test of association (bipolar disorder group v. control group) and used the number of associated independent SNPs statistically significant at P&lt;0.00001 as a metric for the overall genetic signal in the sample. We next compared this metric with that obtained using each of seven diagnostic subsets of the group with bipolar disorder: Research Diagnostic Criteria (RDC): bipolar I disorder; manic disorder; bipolar II disorder; schizoaffective disorder, bipolar type; DSM-IV: bipolar I disorder; bipolar II disorder; schizoaffective disorder, bipolar type.</p><p class="abstract"><strong>Results</strong>: The RDC schizoaffective disorder, bipolar type (v. controls) stood out from the other diagnostic subsets as having a significant excess of independent association signals (P&lt;0.003) compared with that expected in samples of the same size selected randomly from the total bipolar disorder group data-set. The strongest association in this subset of participants with bipolar disorder was at rs4818065 (P = 2.42 x 10(-7)). Biological systems implicated included gamma amniobutyric acid (GABA)(A) receptors. Genes having at least one associated polymorphism at P&lt;10(-4) included B3GALTS, A2BP1, GABRB1, AUTS2, BSN, PTPRG, GIRK2 and CDH12.</p><p class="abstract"><strong>Conclusions</strong>: Our findings show that individuals with broadly defined bipolar schizoaffective features have either a particularly strong genetic contribution or that, as a group, are genetically more homogeneous than the other phenotypes tested. The results point to the importance of using diagnostic approaches that recognise this group of individuals. Our approach can be applied to similar data-sets for other psychiatric and non-psychiatric phenotypes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000647</span>, <span class="grant">G0000934</span>, <span class="grant">G0701003</span>; Wellcome Trust: <span class="grant">060620</span></p>
<p class="publication"><span class="title">The British journal of psychiatry : the journal of mental science</span>&nbsp;<span class="year">2009</span>;<span class="volume">195</span>;<span class="number">1</span>;<span class="pagerange">23-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19567891">19567891</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2802523">2802523</a>; DOI: <a href="http://dx.doi.org/10.1192%2Fbjp.bp.108.061424">10.1192/bjp.bp.108.061424</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cross-species chromosome painting corroborates microchromosome fusion during karyotype evolution of birds.</h4>
<p class="authors"><span class="author">Hansmann T</span>, <span class="author">Nanda I</span>, <span class="author">Volobouev V</span>, <span class="author">Yang F</span>, <span class="author">Schartl M</span>, <span class="author">Haaf T</span> and <span class="author">Schmid M</span></p>
<p class="affiliation">Department of a Human Genetics, University of W&uuml;rzburg, W&uuml;rzburg , Germany.</p><p class="abstract">The stone curlew, also known as thick-knee (Burhinus oedicnemus, BOE), represents a phylogenetically young species of the shorebirds (Charadriiformes) that exhibits one of the most atypical genome organizations known within the class of Aves, due to an extremely low diploid number (2n = 42) and only 6 pairs of microchromosomes in its complement. This distinct deviation from the &#39;typical&#39; avian karyotype is attributed to repeated fusions of ancestral microchromosomes. In order to compare different species with this atypical avian karyotype and to investigate the chromosome rearrangement patterns, chromosome-specific painting probes representing the whole genome of the stone curlew were used to delineate chromosome homology between BOE and 5 species belonging to 5 different avian orders: herring gull (Charadriiformes), cockatiel (Psittaciformes), rock pigeon (Columbiformes), great gray owl (Strigiformes) and Eurasian coot (Gruiformes). Paints derived from the 20 BOE autosomes delimited 28 to 33 evolutionarily conserved segments in the karyotypes of the 5 species, similar to the number recognized by BOE paints in such a basal lineage as the chicken (28 conserved segments). This suggests a high degree of conservation in genome organization in birds. BOE paints also revealed some species-specific rearrangements. In particular, chromosomes BOE1-4 and 14, as well as to a large extent BOE5 and 6, showed conserved synteny with macrochromosomes, whereas homologous regions for BOE7-13 are found to be largely distributed on microchromosomes in the species investigated. Interestingly, the 6 pairs of BOE microchromosomes 15-20 appear to have undergone very few rearrangements in the 5 lineages investigated. Although the arrangements of BOE homologous segments on some chromosomes can be explained by complex fusions and inversions, the occurrence of homologous regions at multiple sites may point to fission of ancestral chromosomes in the karyotypes of the species investigated. However, the present results demonstrate that the ancestral microchromosomes most likely experienced fusion in the stone curlew lineage forming the medium-sized BOE chromosomes, while they have been conserved as microchromosomes in the other neoavian lineages.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">126</span>;<span class="number">3</span>;<span class="pagerange">281-304</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20068299" rel="external">20068299</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000251965" rel="external">10.1159/000251965</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of MAMDC1 as a candidate susceptibility gene for systemic lupus erythematosus (SLE).</h4>
<p class="authors"><span class="author">Hellquist A</span>, <span class="author">Zucchelli M</span>, <span class="author">Lindgren CM</span>, <span class="author">Saarialho-Kere U</span>, <span class="author">J&auml;rvinen TM</span>, <span class="author">Koskenmies S</span>, <span class="author">Julkunen H</span>, <span class="author">Onkamo P</span>, <span class="author">Skoog T</span>, <span class="author">Panelius J</span>, <span class="author">R&auml;is&auml;nen-Sokolowski A</span>, <span class="author">Hasan T</span>, <span class="author">Widen E</span>, <span class="author">Gunnarson I</span>, <span class="author">Svenungsson E</span>, <span class="author">Padyukov L</span>, <span class="author">Assadi G</span>, <span class="author">Berglind L</span>, <span class="author">M&auml;kel&auml; VV</span>, <span class="author">Kivinen K</span>, <span class="author">Wong A</span>, <span class="author">Cunningham Graham DS</span>, <span class="author">Vyse TJ</span>, <span class="author">D&#39;Amato M</span> and <span class="author">Kere J</span></p>
<p class="affiliation">Department of Biosciences and Nutrition, Karolinska Institutet, Huddinge, Sweden.</p><p class="abstract"><strong>Background</strong>: Systemic lupus erythematosus (SLE) is a complex autoimmune disorder with multiple susceptibility genes. We have previously reported suggestive linkage to the chromosomal region 14q21-q23 in Finnish SLE families.</p><p class="abstract">Genetic fine mapping of this region in the same family material, together with a large collection of parent affected trios from UK and two independent case-control cohorts from Finland and Sweden, indicated that a novel uncharacterized gene, MAMDC1 (MAM domain containing glycosylphosphatidylinositol anchor 2, also known as MDGA2, MIM 611128), represents a putative susceptibility gene for SLE. In a combined analysis of the whole dataset, significant evidence of association was detected for the MAMDC1 intronic single nucleotide polymorphisms (SNP) rs961616 (P -value = 0.001, Odds Ratio (OR) = 1.292, 95% CI 1.103-1.513) and rs2297926 (P -value = 0.003, OR = 1.349, 95% CI 1.109-1.640). By Northern blot, real-time PCR (qRT-PCR) and immunohistochemical (IHC) analyses, we show that MAMDC1 is expressed in several tissues and cell types, and that the corresponding mRNA is up-regulated by the pro-inflammatory cytokines tumour necrosis factor alpha (TNF-alpha) and interferon gamma (IFN-gamma) in THP-1 monocytes. Based on its homology to known proteins with similar structure, MAMDC1 appears to be a novel member of the adhesion molecules of the immunoglobulin superfamily (IgCAM), which is involved in cell adhesion, migration, and recruitment to inflammatory sites. Remarkably, some IgCAMs have been shown to interact with ITGAM, the product of another SLE susceptibility gene recently discovered in two independent genome wide association (GWA) scans.</p><p class="abstract"><strong>Significance</strong>: Further studies focused on MAMDC1 and other molecules involved in these pathways might thus provide new insight into the pathogenesis of SLE.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">12</span>;<span class="pagerange">e8037</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19997561">19997561</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2785483">2785483</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0008037">10.1371/journal.pone.0008037</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of expressed sequence tags from the four main developmental stages of Trypanosoma congolense.</h4>
<p class="authors"><span class="author">Helm JR</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Aslett M</span>, <span class="author">Berriman M</span>, <span class="author">Sanders M</span>, <span class="author">Quail MA</span>, <span class="author">Soares MB</span>, <span class="author">Bonaldo MF</span>, <span class="author">Sakurai T</span>, <span class="author">Inoue N</span> and <span class="author">Donelson JE</span></p>
<p class="affiliation">Department of Biochemistry, Carver College of Medicine, University of Iowa, Iowa City, IA 52242, USA.</p><p class="abstract">Trypanosoma congolense is one of the most economically important pathogens of livestock in Africa. Culture-derived parasites of each of the three main insect stages of the T. congolense life cycle, i.e., the procyclic, epimastigote and metacyclic stages, and bloodstream stage parasites isolated from infected mice, were used to construct stage-specific cDNA libraries and expressed sequence tags (ESTs or cDNA clones) in each library were sequenced. Thirteen EST clusters encoding different variant surface glycoproteins (VSGs) were detected in the metacyclic library and 26 VSG EST clusters were found in the bloodstream library, 6 of which are shared by the metacyclic library. Rare VSG ESTs are present in the epimastigote library, and none were detected in the procyclic library. ESTs encoding enzymes that catalyze oxidative phosphorylation and amino acid metabolism are about twice as abundant in the procyclic and epimastigote stages as in the metacyclic and bloodstream stages. In contrast, ESTs encoding enzymes involved in glycolysis, the citric acid cycle and nucleotide metabolism are about the same in all four developmental stages. Cysteine proteases, kinases and phosphatases are the most abundant enzyme groups represented by the ESTs. All four libraries contain T. congolense-specific expressed sequences not present in the Trypanosoma brucei and Trypanosoma cruzi genomes. Normalized cDNA libraries were constructed from the metacyclic and bloodstream stages, and found to be further enriched for T. congolense-specific ESTs. Given that cultured T. congolense offers an experimental advantage over other African trypanosome species, these ESTs provide a basis for further investigation of the molecular properties of these four developmental stages, especially the epimastigote and metacyclic stages for which it is difficult to obtain large quantities of organisms. The T. congolense EST databases are available at: http://www.sanger.ac.uk/Projects/T_congolense/EST_index.shtml. The sequence data have been submitted to EMBL under the following accession numbers: FN263376-FN292969.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI059451-05</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2009</span>;<span class="volume">168</span>;<span class="number">1</span>;<span class="pagerange">34-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19559733" rel="external">19559733</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2741298" rel="external">2741298</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2009.06.004" rel="external">10.1016/j.molbiopara.2009.06.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid evolution of virulence and drug resistance in the emerging zoonotic pathogen Streptococcus suis.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Hauser H</span>, <span class="author">Sanders M</span>, <span class="author">Ngo TH</span>, <span class="author">Cherevach I</span>, <span class="author">Cronin A</span>, <span class="author">Goodhead I</span>, <span class="author">Mungall K</span>, <span class="author">Quail MA</span>, <span class="author">Price C</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Sharp S</span>, <span class="author">Croucher NJ</span>, <span class="author">Chieu TB</span>, <span class="author">Mai NT</span>, <span class="author">Diep TS</span>, <span class="author">Chinh NT</span>, <span class="author">Kehoe M</span>, <span class="author">Leigh JA</span>, <span class="author">Ward PN</span>, <span class="author">Dowson CG</span>, <span class="author">Whatmore AM</span>, <span class="author">Chanter N</span>, <span class="author">Iversen P</span>, <span class="author">Gottschalk M</span>, <span class="author">Slater JD</span>, <span class="author">Smith HE</span>, <span class="author">Spratt BG</span>, <span class="author">Xu J</span>, <span class="author">Ye C</span>, <span class="author">Bentley S</span>, <span class="author">Barrell BG</span>, <span class="author">Schultsz C</span>, <span class="author">Maskell DJ</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom. <a href="mailto:%6D%68%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#104;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Streptococcus suis is a zoonotic pathogen that infects pigs and can occasionally cause serious infections in humans. S. suis infections occur sporadically in human Europe and North America, but a recent major outbreak has been described in China with high levels of mortality. The mechanisms of S. suis pathogenesis in humans and pigs are poorly understood.</p><p class="abstract">The sequencing of whole genomes of S. suis isolates provides opportunities to investigate the genetic basis of infection. Here we describe whole genome sequences of three S. suis strains from the same lineage: one from European pigs, and two from human cases from China and Vietnam. Comparative genomic analysis was used to investigate the variability of these strains. S. suis is phylogenetically distinct from other Streptococcus species for which genome sequences are currently available. Accordingly, approximately 40% of the approximately 2 Mb genome is unique in comparison to other Streptococcus species. Finer genomic comparisons within the species showed a high level of sequence conservation; virtually all of the genome is common to the S. suis strains. The only exceptions are three approximately 90 kb regions, present in the two isolates from humans, composed of integrative conjugative elements and transposons. Carried in these regions are coding sequences associated with drug resistance. In addition, small-scale sequence variation has generated pseudogenes in putative virulence and colonization factors.</p><p class="abstract">The genomic inventories of genetically related S. suis strains, isolated from distinct hosts and diseases, exhibit high levels of conservation. However, the genomes provide evidence that horizontal gene transfer has contributed to the evolution of drug resistance.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/G019274/1</span>; Wellcome Trust: <span class="grant">089472</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">7</span>;<span class="pagerange">e6072</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19603075">19603075</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705793">2705793</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0006072">10.1371/journal.pone.0006072</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic evidence for the evolution of Streptococcus equi: host restriction, increased virulence, and genetic exchange with human pathogens.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Heather Z</span>, <span class="author">Paillot R</span>, <span class="author">Steward KF</span>, <span class="author">Webb K</span>, <span class="author">Ainslie F</span>, <span class="author">Jourdan T</span>, <span class="author">Bason NC</span>, <span class="author">Holroyd NE</span>, <span class="author">Mungall K</span>, <span class="author">Quail MA</span>, <span class="author">Sanders M</span>, <span class="author">Simmonds M</span>, <span class="author">Willey D</span>, <span class="author">Brooks K</span>, <span class="author">Aanensen DM</span>, <span class="author">Spratt BG</span>, <span class="author">Jolley KA</span>, <span class="author">Maiden MC</span>, <span class="author">Kehoe M</span>, <span class="author">Chanter N</span>, <span class="author">Bentley SD</span>, <span class="author">Robinson C</span>, <span class="author">Maskell DJ</span>, <span class="author">Parkhill J</span> and <span class="author">Waller AS</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The continued evolution of bacterial pathogens has major implications for both human and animal disease, but the exchange of genetic material between host-restricted pathogens is rarely considered. Streptococcus equi subspecies equi (S. equi) is a host-restricted pathogen of horses that has evolved from the zoonotic pathogen Streptococcus equi subspecies zooepidemicus (S. zooepidemicus). These pathogens share approximately 80% genome sequence identity with the important human pathogen Streptococcus pyogenes. We sequenced and compared the genomes of S. equi 4047 and S. zooepidemicus H70 and screened S. equi and S. zooepidemicus strains from around the world to uncover evidence of the genetic events that have shaped the evolution of the S. equi genome and led to its emergence as a host-restricted pathogen. Our analysis provides evidence of functional loss due to mutation and deletion, coupled with pathogenic specialization through the acquisition of bacteriophage encoding a phospholipase A(2) toxin, and four superantigens, and an integrative conjugative element carrying a novel iron acquisition system with similarity to the high pathogenicity island of Yersinia pestis. We also highlight that S. equi, S. zooepidemicus, and S. pyogenes share a common phage pool that enhances cross-species pathogen evolution. We conclude that the complex interplay of functional loss, pathogenic specialization, and genetic exchange between S. equi, S. zooepidemicus, and S. pyogenes continues to influence the evolution of these important streptococci.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">047072</span>, <span class="grant">087622</span>, <span class="grant">089472</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">e1000346</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19325880">19325880</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2654543">2654543</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1000346">10.1371/journal.ppat.1000346</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of Burkholderia cenocepacia J2315, an epidemic pathogen of cystic fibrosis patients.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Seth-Smith HM</span>, <span class="author">Crossman LC</span>, <span class="author">Sebaihia M</span>, <span class="author">Bentley SD</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Thomson NR</span>, <span class="author">Bason N</span>, <span class="author">Quail MA</span>, <span class="author">Sharp S</span>, <span class="author">Cherevach I</span>, <span class="author">Churcher C</span>, <span class="author">Goodhead I</span>, <span class="author">Hauser H</span>, <span class="author">Holroyd N</span>, <span class="author">Mungall K</span>, <span class="author">Scott P</span>, <span class="author">Walker D</span>, <span class="author">White B</span>, <span class="author">Rose H</span>, <span class="author">Iversen P</span>, <span class="author">Mil-Homens D</span>, <span class="author">Rocha EP</span>, <span class="author">Fialho AM</span>, <span class="author">Baldwin A</span>, <span class="author">Dowson C</span>, <span class="author">Barrell BG</span>, <span class="author">Govan JR</span>, <span class="author">Vandamme P</span>, <span class="author">Hart CA</span>, <span class="author">Mahenthiralingam E</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6D%68%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#104;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Bacterial infections of the lungs of cystic fibrosis (CF) patients cause major complications in the treatment of this common genetic disease. Burkholderia cenocepacia infection is particularly problematic since this organism has high levels of antibiotic resistance, making it difficult to eradicate; the resulting chronic infections are associated with severe declines in lung function and increased mortality rates. B. cenocepacia strain J2315 was isolated from a CF patient and is a member of the epidemic ET12 lineage that originated in Canada or the United Kingdom and spread to Europe. The 8.06-Mb genome of this highly transmissible pathogen comprises three circular chromosomes and a plasmid and encodes a broad array of functions typical of this metabolically versatile genus, as well as numerous virulence and drug resistance functions. Although B. cenocepacia strains can be isolated from soil and can be pathogenic to both plants and man, J2315 is representative of a lineage of B. cenocepacia rarely isolated from the environment and which spreads between CF patients. Comparative analysis revealed that ca. 21% of the genome is unique in comparison to other strains of B. cenocepacia, highlighting the genomic plasticity of this species. Pseudogenes in virulence determinants suggest that the pathogenic response of J2315 may have been recently selected to promote persistence in the CF lung. The J2315 genome contains evidence that its unique and highly adapted genetic content has played a significant role in its success as an epidemic CF pathogen.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">1</span>;<span class="pagerange">261-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18931103">18931103</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2612433">2612433</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01230-08">10.1128/JB.01230-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene ontology analysis of GWA study data sets provides insights into the biology of bipolar disorder.</h4>
<p class="authors"><span class="author">Holmans P</span>, <span class="author">Green EK</span>, <span class="author">Pahwa JS</span>, <span class="author">Ferreira MA</span>, <span class="author">Purcell SM</span>, <span class="author">Sklar P</span>, <span class="author">Wellcome Trust Case-Control Consortium</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span> and <span class="author">Craddock N</span></p>
<p class="affiliation">MRC Centre for Neuropsychiatric Genetics and Genomics, Department of Psychological Medicine and Neurology, School of Medicine, Heath Park, CF23 6BQ Cardiff, UK. <a href="mailto:%68%6F%6C%6D%61%6E%73%70%61%40%63%61%72%64%69%66%66.%61%63.%75%6B">&#104;&#111;&#108;&#109;&#97;&#110;&#115;&#112;&#97;&#64;&#99;&#97;&#114;&#100;&#105;&#102;&#102;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present a method for testing overrepresentation of biological pathways, indexed by gene-ontology terms, in lists of significant SNPs from genome-wide association studies. This method corrects for linkage disequilibrium between SNPs, variable gene size, and multiple testing of nonindependent pathways. The method was applied to the Wellcome Trust Case-Control Consortium Crohn disease (CD) data set. At a general level, the biological basis of CD is relatively well known for a complex genetic trait, and it thus acted as a test of the method. The method, known as ALIGATOR (Association LIst Go AnnoTatOR), successfully detected biological pathways implicated in CD. The method was also applied to a meta-analysis of bipolar disorder, and it implicated the modulation of transcription and cellular activity, including that which occurs via hormonal action, as an important player in pathogenesis.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">85</span>;<span class="number">1</span>;<span class="pagerange">13-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19539887">19539887</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2706963">2706963</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2009.05.011">10.1016/j.ajhg.2009.05.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detecting SNPs and estimating allele frequencies in clonal bacterial populations by sequencing pooled DNA.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Teo YY</span>, <span class="author">Li H</span>, <span class="author">Nair S</span>, <span class="author">Dougan G</span>, <span class="author">Wain J</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6B%68%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#107;&#104;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">SUMMARY: Here, we present a method for estimating the frequencies of SNP alleles present within pooled samples of DNA using high-throughput short-read sequencing. The method was tested on real data from six strains of the highly monomorphic pathogen Salmonella Paratyphi A, sequenced individually and in a pool. A variety of read mapping and quality-weighting procedures were tested to determine the optimal parameters, which afforded &gt; or =80% sensitivity of SNP detection and strong correlation with true SNP frequency at poolwide read depth of 40x, declining only slightly at read depths 20-40x. AVAILABILITY: The method was implemented in Perl and relies on the opensource software Maq for read mapping and SNP calling. The Perl script is freely available from ftp://ftp.sanger.ac.uk/pub/pathogens/pools/.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">16</span>;<span class="pagerange">2074-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19497932" rel="external">19497932</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2722999" rel="external">2722999</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp344" rel="external">10.1093/bioinformatics/btp344</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Extensive molecular differences between anterior- and posterior-half-sclerotomes underlie somite polarity and spinal nerve segmentation.</h4>
<p class="authors"><span class="author">Hughes DS</span>, <span class="author">Keynes RJ</span> and <span class="author">Tannahill D</span></p>
<p class="affiliation">Department of Physiology, Development and Neuroscience, University of Cambridge, Downing Street, Cambridge, CB3 2DY, UK. <a href="mailto:%64%73%74%68%75%67%68%65%73%40%67%6D%61%69%6C.%63%6F%6D">&#100;&#115;&#116;&#104;&#117;&#103;&#104;&#101;&#115;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: The polarization of somite-derived sclerotomes into anterior and posterior halves underlies vertebral morphogenesis and spinal nerve segmentation. To characterize the full extent of molecular differences that underlie this polarity, we have undertaken a systematic comparison of gene expression between the two sclerotome halves in the mouse embryo.</p><p class="abstract"><strong>Results</strong>: Several hundred genes are differentially-expressed between the two sclerotome halves, showing that a marked degree of molecular heterogeneity underpins the development of somite polarity.</p><p class="abstract"><strong>Conclusion</strong>: We have identified a set of genes that warrant further investigation as regulators of somite polarity and vertebral morphogenesis, as well as repellents of spinal axon growth. Moreover the results indicate that, unlike the posterior half-sclerotome, the central region of the anterior-half-sclerotome does not contribute bone and cartilage to the vertebral column, being associated instead with the development of the segmented spinal nerves.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">BMC developmental biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">9</span>;<span class="pagerange">30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19463158" rel="external">19463158</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2693541" rel="external">2693541</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-213X-9-30" rel="external">10.1186/1471-213X-9-30</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">InterPro: the integrative protein signature database.</h4>
<p class="authors"><span class="author">Hunter S</span>, <span class="author">Apweiler R</span>, <span class="author">Attwood TK</span>, <span class="author">Bairoch A</span>, <span class="author">Bateman A</span>, <span class="author">Binns D</span>, <span class="author">Bork P</span>, <span class="author">Das U</span>, <span class="author">Daugherty L</span>, <span class="author">Duquenne L</span>, <span class="author">Finn RD</span>, <span class="author">Gough J</span>, <span class="author">Haft D</span>, <span class="author">Hulo N</span>, <span class="author">Kahn D</span>, <span class="author">Kelly E</span>, <span class="author">Laugraud A</span>, <span class="author">Letunic I</span>, <span class="author">Lonsdale D</span>, <span class="author">Lopez R</span>, <span class="author">Madera M</span>, <span class="author">Maslen J</span>, <span class="author">McAnulla C</span>, <span class="author">McDowall J</span>, <span class="author">Mistry J</span>, <span class="author">Mitchell A</span>, <span class="author">Mulder N</span>, <span class="author">Natale D</span>, <span class="author">Orengo C</span>, <span class="author">Quinn AF</span>, <span class="author">Selengut JD</span>, <span class="author">Sigrist CJ</span>, <span class="author">Thimma M</span>, <span class="author">Thomas PD</span>, <span class="author">Valentin F</span>, <span class="author">Wilson D</span>, <span class="author">Wu CH</span> and <span class="author">Yeats C</span></p>
<p class="affiliation">EMBL Outstation European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%68%75%6E%74%65%72%40%65%62%69.%61%63.%75%6B">&#104;&#117;&#110;&#116;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The InterPro database (http://www.ebi.ac.uk/interpro/) integrates together predictive models or &#39;signatures&#39; representing protein domains, families and functional sites from multiple, diverse source databases: Gene3D, PANTHER, Pfam, PIRSF, PRINTS, ProDom, PROSITE, SMART, SUPERFAMILY and TIGRFAMs. Integration is performed manually and approximately half of the total approximately 58,000 signatures available in the source databases belong to an InterPro entry. Recently, we have started to also display the remaining un-integrated signatures via our web interface. Other developments include the provision of non-signature data, such as structural data, in new XML files on our FTP site, as well as the inclusion of matchless UniProtKB proteins in the existing match XML files. The web interface has been extended and now links out to the ADAN predicted protein-protein interaction database and the SPICE and Dasty viewers. The latest public release (v18.0) covers 79.8% of UniProtKB (v14.1) and consists of 16 549 entries. InterPro data may be accessed either via the web address above, via web services, by downloading files by anonymous FTP or by using the InterProScan search software (http://www.ebi.ac.uk/Tools/InterProScan/).</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F010435/1</span>, <span class="grant">BB/F010508/1</span>; NIGMS NIH HHS: <span class="grant">GM081084</span>; Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2009</span>;<span class="volume">37</span>;<span class="number">Database issue</span>;<span class="pagerange">D211-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18940856" rel="external">18940856</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2686546" rel="external">2686546</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkn785" rel="external">10.1093/nar/gkn785</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence and comparative genome analysis of enteropathogenic Escherichia coli O127:H6 strain E2348/69.</h4>
<p class="authors"><span class="author">Iguchi A</span>, <span class="author">Thomson NR</span>, <span class="author">Ogura Y</span>, <span class="author">Saunders D</span>, <span class="author">Ooka T</span>, <span class="author">Henderson IR</span>, <span class="author">Harris D</span>, <span class="author">Asadulghani M</span>, <span class="author">Kurokawa K</span>, <span class="author">Dean P</span>, <span class="author">Kenny B</span>, <span class="author">Quail MA</span>, <span class="author">Thurston S</span>, <span class="author">Dougan G</span>, <span class="author">Hayashi T</span>, <span class="author">Parkhill J</span> and <span class="author">Frankel G</span></p>
<p class="affiliation">Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, Miyazaki, Japan.</p><p class="abstract">Enteropathogenic Escherichia coli (EPEC) was the first pathovar of E. coli to be implicated in human disease; however, no EPEC strain has been fully sequenced until now. Strain E2348/69 (serotype O127:H6 belonging to E. coli phylogroup B2) has been used worldwide as a prototype strain to study EPEC biology, genetics, and virulence. Studies of E2348/69 led to the discovery of the locus of enterocyte effacement-encoded type III secretion system (T3SS) and its cognate effectors, which play a vital role in attaching and effacing lesion formation on gut epithelial cells. In this study, we determined the complete genomic sequence of E2348/69 and performed genomic comparisons with other important E. coli strains. We identified 424 E2348/69-specific genes, most of which are carried on mobile genetic elements, and a number of genetic traits specifically conserved in phylogroup B2 strains irrespective of their pathotypes, including the absence of the ETT2-related T3SS, which is present in E. coli strains belonging to all other phylogroups. The genome analysis revealed the entire gene repertoire related to E2348/69 virulence. Interestingly, E2348/69 contains only 21 intact T3SS effector genes, all of which are carried on prophages and integrative elements, compared to over 50 effector genes in enterohemorrhagic E. coli O157. As E2348/69 is the most-studied pathogenic E. coli strain, this study provides a genomic context for the vast amount of existing experimental data. The unexpected simplicity of the E2348/69 T3SS provides the first opportunity to fully dissect the entire virulence strategy of attaching and effacing pathogens in the genomic context.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700151</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">1</span>;<span class="pagerange">347-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18952797">18952797</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2612414">2612414</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01238-08">10.1128/JB.01238-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Weak preservation of local neutral substitution rates across mammalian genomes.</h4>
<p class="authors"><span class="author">Imamura H</span>, <span class="author">Karro JE</span> and <span class="author">Chuang JH</span></p>
<p class="affiliation">Boston College, Department of Biology, Chestnut Hill, MA 02467, USA. <a href="mailto:%68%69%6D%61%6D%75%72%61%40%69%74%67.%62%65">&#104;&#105;&#109;&#97;&#109;&#117;&#114;&#97;&#64;&#105;&#116;&#103;&#46;&#98;&#101;</a></p><p class="abstract"><strong>Background</strong>: The rate at which neutral (non-functional) bases undergo substitution is highly dependent on their location within a genome. However, it is not clear how fast these location-dependent rates change, or to what extent the substitution rate patterns are conserved between lineages. To address this question, which is critical not only for understanding the substitution process but also for evaluating phylogenetic footprinting algorithms, we examine ancestral repeats: a predominantly neutral dataset with a significantly higher genomic density than other datasets commonly used to study substitution rate variation. Using this repeat data, we measure the extent to which orthologous ancestral repeat sequences exhibit similar substitution patterns in separate mammalian lineages, allowing us to ascertain how well local substitution rates have been preserved across species.</p><p class="abstract"><strong>Results</strong>: We calculated substitution rates for each ancestral repeat in each of three independent mammalian lineages (primate - from human/macaque alignments, rodent - from mouse/rat alignments, and laurasiatheria - from dog/cow alignments). We then measured the correlation of local substitution rates among these lineages. Overall we found the correlations between lineages to be statistically significant, but too weak to have much predictive power (r2 &lt;5%). These correlations were found to be primarily driven by regional effects at the scale of several hundred kb or larger. A few repeat classes (e.g. 7SK, Charlie8, and MER121) also exhibited stronger conservation of rate patterns, likely due to the effect of repeat-specific purifying selection. These classes should be excluded when estimating local neutral substitution rates.</p><p class="abstract"><strong>Conclusion</strong>: Although local neutral substitution rates have some correlations among mammalian species, these correlations have little predictive power on the scale of individual repeats. This indicates that local substitution rates have changed significantly among the lineages we have studied, and are likely to have changed even more for more diverged lineages. The correlations that do persist are too weak to be responsible for many of the highly conserved elements found by phylogenetic footprinting algorithms, leading us to conclude that such elements must be conserved due to selective forces.</p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">9</span>;<span class="pagerange">89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19416516" rel="external">19416516</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2689173" rel="external">2689173</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-9-89" rel="external">10.1186/1471-2148-9-89</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at five new loci associated with early-onset inflammatory bowel disease.</h4>
<p class="authors"><span class="author">Imielinski M</span>, <span class="author">Baldassano RN</span>, <span class="author">Griffiths A</span>, <span class="author">Russell RK</span>, <span class="author">Annese V</span>, <span class="author">Dubinsky M</span>, <span class="author">Kugathasan S</span>, <span class="author">Bradfield JP</span>, <span class="author">Walters TD</span>, <span class="author">Sleiman P</span>, <span class="author">Kim CE</span>, <span class="author">Muise A</span>, <span class="author">Wang K</span>, <span class="author">Glessner JT</span>, <span class="author">Saeed S</span>, <span class="author">Zhang H</span>, <span class="author">Frackelton EC</span>, <span class="author">Hou C</span>, <span class="author">Flory JH</span>, <span class="author">Otieno G</span>, <span class="author">Chiavacci RM</span>, <span class="author">Grundmeier R</span>, <span class="author">Castro M</span>, <span class="author">Latiano A</span>, <span class="author">Dallapiccola B</span>, <span class="author">Stempak J</span>, <span class="author">Abrams DJ</span>, <span class="author">Taylor K</span>, <span class="author">McGovern D</span>, <span class="author">Western Regional Alliance for Pediatric IBD</span>, <span class="author">Silber G</span>, <span class="author">Wrobel I</span>, <span class="author">Quiros A</span>, <span class="author">International IBD Genetics Consortium</span>, <span class="author">Barrett JC</span>, <span class="author">Hansoul S</span>, <span class="author">Nicolae DL</span>, <span class="author">Cho JH</span>, <span class="author">Duerr RH</span>, <span class="author">Rioux JD</span>, <span class="author">Brant SR</span>, <span class="author">Silverberg MS</span>, <span class="author">Taylor KD</span>, <span class="author">Barmuda MM</span>, <span class="author">Bitton A</span>, <span class="author">Dassopoulos T</span>, <span class="author">Datta LW</span>, <span class="author">Green T</span>, <span class="author">Griffiths AM</span>, <span class="author">Kistner EO</span>, <span class="author">Murtha MT</span>, <span class="author">Regueiro MD</span>, <span class="author">Rotter JI</span>, <span class="author">Schumm LP</span>, <span class="author">Steinhart AH</span>, <span class="author">Targan SR</span>, <span class="author">Xavier RJ</span>, <span class="author">NIDDK IBD Genetics Consortium</span>, <span class="author">Libioulle C</span>, <span class="author">Sandor C</span>, <span class="author">Lathrop M</span>, <span class="author">Belaiche J</span>, <span class="author">Dewit O</span>, <span class="author">Gut I</span>, <span class="author">Heath S</span>, <span class="author">Laukens D</span>, <span class="author">Mni M</span>, <span class="author">Rutgeerts P</span>, <span class="author">Van Gossum A</span>, <span class="author">Zelenika D</span>, <span class="author">Franchimont D</span>, <span class="author">Hugot JP</span>, <span class="author">de Vos M</span>, <span class="author">Vermeire S</span>, <span class="author">Louis E</span>, <span class="author">Belgian-French IBD Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Cardon LR</span>, <span class="author">Anderson CA</span>, <span class="author">Drummond H</span>, <span class="author">Nimmo E</span>, <span class="author">Ahmad T</span>, <span class="author">Prescott NJ</span>, <span class="author">Onnie CM</span>, <span class="author">Fisher SA</span>, <span class="author">Marchini J</span>, <span class="author">Ghori J</span>, <span class="author">Bumpstead S</span>, <span class="author">Gwillam R</span>, <span class="author">Tremelling M</span>, <span class="author">Delukas P</span>, <span class="author">Mansfield J</span>, <span class="author">Jewell D</span>, <span class="author">Satsangi J</span>, <span class="author">Mathew CG</span>, <span class="author">Parkes M</span>, <span class="author">Georges M</span>, <span class="author">Daly MJ</span>, <span class="author">Heyman MB</span>, <span class="author">Ferry GD</span>, <span class="author">Kirschner B</span>, <span class="author">Lee J</span>, <span class="author">Essers J</span>, <span class="author">Grand R</span>, <span class="author">Stephens M</span>, <span class="author">Levine A</span>, <span class="author">Piccoli D</span>, <span class="author">Van Limbergen J</span>, <span class="author">Cucchiara S</span>, <span class="author">Monos DS</span>, <span class="author">Guthery SL</span>, <span class="author">Denson L</span>, <span class="author">Wilson DC</span>, <span class="author">Grant SF</span>, <span class="author">Daly M</span>, <span class="author">Silverberg MS</span>, <span class="author">Satsangi J</span> and <span class="author">Hakonarson H</span></p>
<p class="affiliation">Center for Applied Genomics, The Children&#39;s Hospital of Philadelphia, Philadelphia, Pennsylvania, USA.</p><p class="abstract">The inflammatory bowel diseases (IBD) Crohn&#39;s disease and ulcerative colitis are common causes of morbidity in children and young adults in the western world. Here we report the results of a genome-wide association study in early-onset IBD involving 3,426 affected individuals and 11,963 genetically matched controls recruited through international collaborations in Europe and North America, thereby extending the results from a previous study of 1,011 individuals with early-onset IBD. We have identified five new regions associated with early-onset IBD susceptibility, including 16p11 near the cytokine gene IL27 (rs8049439, P = 2.41 x 10(-9)), 22q12 (rs2412973, P = 1.55 x 10(-9)), 10q22 (rs1250550, P = 5.63 x 10(-9)), 2q37 (rs4676410, P = 3.64 x 10(-8)) and 19q13.11 (rs10500264, P = 4.26 x 10(-10)). Our scan also detected associations at 23 of 32 loci previously implicated in adult-onset Crohn&#39;s disease and at 8 of 17 loci implicated in adult-onset ulcerative colitis, highlighting the close pathogenetic relationship between early- and adult-onset IBD.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0600329</span>, <span class="grant">G0800675</span>, <span class="grant">G0800759</span>; NCRR NIH HHS: <span class="grant">C06-RR11234</span>, <span class="grant">M01 RR002172-26</span>, <span class="grant">M01-RR00064</span>; NIDDK NIH HHS: <span class="grant">DK062423</span>, <span class="grant">DK069513</span>, <span class="grant">K24 DK060617</span>, <span class="grant">K24 DK060617-07</span>, <span class="grant">P30 DK040561</span>, <span class="grant">P30 DK040561-14</span>, <span class="grant">P30 DK043351</span>, <span class="grant">T32 DK007477</span>, <span class="grant">U01 DK062420</span>, <span class="grant">U01 DK062420-08</span>; Wellcome Trust: <span class="grant">072789/Z/03/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">12</span>;<span class="pagerange">1335-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19915574">19915574</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3267927">3267927</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.489">10.1038/ng.489</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transposon-mediated genome manipulation in vertebrates.</h4>
<p class="authors"><span class="author">Ivics Z</span>, <span class="author">Li MA</span>, <span class="author">M&aacute;t&eacute;s L</span>, <span class="author">Boeke JD</span>, <span class="author">Nagy A</span>, <span class="author">Bradley A</span> and <span class="author">Izsv&aacute;k Z</span></p>
<p class="affiliation">Max Delbr&uuml;ck Center for Molecular Medicine, Berlin, Germany. <a href="mailto:%7A%69%76%69%63%73%40%6D%64%63-%62%65%72%6C%69%6E.%64%65">&#122;&#105;&#118;&#105;&#99;&#115;&#64;&#109;&#100;&#99;&#45;&#98;&#101;&#114;&#108;&#105;&#110;&#46;&#100;&#101;</a></p><p class="abstract">Transposable elements are DNA segments with the unique ability to move about in the genome. This inherent feature can be exploited to harness these elements as gene vectors for genome manipulation. Transposon-based genetic strategies have been established in vertebrate species over the last decade, and current progress in this field suggests that transposable elements will serve as indispensable tools. In particular, transposons can be applied as vectors for somatic and germline transgenesis, and as insertional mutagens in both loss-of-function and gain-of-function forward mutagenesis screens. In addition, transposons will gain importance in future cell-based clinical applications, including nonviral gene transfer into stem cells and the rapidly developing field of induced pluripotent stem cells. Here we provide an overview of transposon-based methods used in vertebrate model organisms with an emphasis on the mouse system and highlight the most important considerations concerning genetic applications of the transposon systems.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01 CA016519</span>, <span class="grant">P01 CA016519-340010</span>; NIGMS NIH HHS: <span class="grant">R01 GM036481</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2009</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">415-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19478801">19478801</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2867038">2867038</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1332">10.1038/nmeth.1332</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide and fine-resolution association analysis of malaria in West Africa.</h4>
<p class="authors"><span class="author">Jallow M</span>, <span class="author">Teo YY</span>, <span class="author">Small KS</span>, <span class="author">Rockett KA</span>, <span class="author">Deloukas P</span>, <span class="author">Clark TG</span>, <span class="author">Kivinen K</span>, <span class="author">Bojang KA</span>, <span class="author">Conway DJ</span>, <span class="author">Pinder M</span>, <span class="author">Sirugo G</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Usen S</span>, <span class="author">Auburn S</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Campino S</span>, <span class="author">Coffey A</span>, <span class="author">Dunham A</span>, <span class="author">Fry AE</span>, <span class="author">Green A</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt SE</span>, <span class="author">Inouye M</span>, <span class="author">Jeffreys AE</span>, <span class="author">Mendy A</span>, <span class="author">Palotie A</span>, <span class="author">Potter S</span>, <span class="author">Ragoussis J</span>, <span class="author">Rogers J</span>, <span class="author">Rowlands K</span>, <span class="author">Somaskantharajah E</span>, <span class="author">Whittaker P</span>, <span class="author">Widden C</span>, <span class="author">Donnelly P</span>, <span class="author">Howie B</span>, <span class="author">Marchini J</span>, <span class="author">Morris A</span>, <span class="author">SanJoaquin M</span>, <span class="author">Achidi EA</span>, <span class="author">Agbenyega T</span>, <span class="author">Allen A</span>, <span class="author">Amodu O</span>, <span class="author">Corran P</span>, <span class="author">Djimde A</span>, <span class="author">Dolo A</span>, <span class="author">Doumbo OK</span>, <span class="author">Drakeley C</span>, <span class="author">Dunstan S</span>, <span class="author">Evans J</span>, <span class="author">Farrar J</span>, <span class="author">Fernando D</span>, <span class="author">Hien TT</span>, <span class="author">Horstmann RD</span>, <span class="author">Ibrahim M</span>, <span class="author">Karunaweera N</span>, <span class="author">Kokwaro G</span>, <span class="author">Koram KA</span>, <span class="author">Lemnge M</span>, <span class="author">Makani J</span>, <span class="author">Marsh K</span>, <span class="author">Michon P</span>, <span class="author">Modiano D</span>, <span class="author">Molyneux ME</span>, <span class="author">Mueller I</span>, <span class="author">Parker M</span>, <span class="author">Peshu N</span>, <span class="author">Plowe CV</span>, <span class="author">Puijalon O</span>, <span class="author">Reeder J</span>, <span class="author">Reyburn H</span>, <span class="author">Riley EM</span>, <span class="author">Sakuntabhai A</span>, <span class="author">Singhasivanon P</span>, <span class="author">Sirima S</span>, <span class="author">Tall A</span>, <span class="author">Taylor TE</span>, <span class="author">Thera M</span>, <span class="author">Troye-Blomberg M</span>, <span class="author">Williams TN</span>, <span class="author">Wilson M</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Wellcome Trust Case Control Consortium</span> and <span class="author">Malaria Genomic Epidemiology Network</span></p>
<p class="affiliation">MRC Laboratories, Fajara, Banjul, Gambia.</p><p class="abstract">We report a genome-wide association (GWA) study of severe malaria in The Gambia. The initial GWA scan included 2,500 children genotyped on the Affymetrix 500K GeneChip, and a replication study included 3,400 children. We used this to examine the performance of GWA methods in Africa. We found considerable population stratification, and also that signals of association at known malaria resistance loci were greatly attenuated owing to weak linkage disequilibrium (LD). To investigate possible solutions to the problem of low LD, we focused on the HbS locus, sequencing this region of the genome in 62 Gambian individuals and then using these data to conduct multipoint imputation in the GWA samples. This increased the signal of association, from P = 4 &times; 10(-7) to P = 4 &times; 10(-14), with the peak of the signal located precisely at the HbS causal variant. Our findings provide proof of principle that fine-resolution multipoint imputation, based on population-specific sequencing data, can substantially boost authentic GWA signals and enable fine mapping of causal variants in African populations.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600230(77610)</span>, <span class="grant">G0600329</span>, <span class="grant">G0600718</span>, <span class="grant">G0800759</span>, <span class="grant">G19/9</span>, <span class="grant">G9828345</span>, <span class="grant">MC_U190081977</span>, <span class="grant">MC_U190081993</span>; NIAID NIH HHS: <span class="grant">U19 AI065683</span>, <span class="grant">U19 AI065683-04</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">064890</span>, <span class="grant">076113</span>, <span class="grant">076934</span>, <span class="grant">077011</span>, <span class="grant">077383</span>, <span class="grant">077383/Z/05/Z</span>, <span class="grant">081682</span>, <span class="grant">089062</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">657-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19465909">19465909</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2889040">2889040</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.388">10.1038/ng.388</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Repetitive sequence variation and dynamics in the ribosomal DNA array of Saccharomyces cerevisiae as revealed by whole-genome resequencing.</h4>
<p class="authors"><span class="author">James SA</span>, <span class="author">O&#39;Kelly MJ</span>, <span class="author">Carter DM</span>, <span class="author">Davey RP</span>, <span class="author">van Oudenaarden A</span> and <span class="author">Roberts IN</span></p>
<p class="affiliation">National Collection of Yeast Cultures, Institute of Food Research, Norwich Research Park, Colney, Norwich NR4 7UA, United Kingdom.</p><p class="abstract">Ribosomal DNA (rDNA) plays a key role in ribosome biogenesis, encoding genes for the structural RNA components of this important cellular organelle. These genes are vital for efficient functioning of the cellular protein synthesis machinery and as such are highly conserved and normally present in high copy numbers. In the baker&#39;s yeast Saccharomyces cerevisiae, there are more than 100 rDNA repeats located at a single locus on chromosome XII. Stability and sequence homogeneity of the rDNA array is essential for function, and this is achieved primarily by the mechanism of gene conversion. Detecting variation within these arrays is extremely problematic due to their large size and repetitive structure. In an attempt to address this, we have analyzed over 35 Mbp of rDNA sequence obtained from whole-genome shotgun sequencing (WGSS) of 34 strains of S. cerevisiae. Contrary to expectation, we find significant rDNA sequence variation exists within individual genomes. Many of the detected polymorphisms are not fully resolved. For this type of sequence variation, we introduce the term partial single nucleotide polymorphism, or pSNP. Comparative analysis of the complete data set reveals that different S. cerevisiae genomes possess different patterns of rDNA polymorphism, with much of the variation located within the rapidly evolving nontranscribed intergenic spacer (IGS) region. Furthermore, we find that strains known to have either structured or mosaic/hybrid genomes can be distinguished from one another based on rDNA pSNP number, indicating that pSNP dynamics may provide a reliable new measure of genome origin and stability.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">4</span>;<span class="pagerange">626-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19141593" rel="external">19141593</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2665781" rel="external">2665781</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.084517.108" rel="external">10.1101/gr.084517.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effects of calcium signaling on Plasmodium falciparum erythrocyte invasion and post-translational modification of gliding-associated protein 45 (PfGAP45).</h4>
<p class="authors"><span class="author">Jones ML</span>, <span class="author">Cottingham C</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">Department of Medicine, University of Alabama at Birmingham, Birmingham, AL 35294, USA.</p><p class="abstract">Plasmodium falciparum erythrocyte invasion is powered by an actin/myosin motor complex that is linked both to the tight junction and to the merozoite cytoskeleton through the Inner Membrane Complex (IMC). The IMC association of the myosin motor, PfMyoA, is maintained by its association with three proteins: PfMTIP, a myosin light chain, PfGAP45, an IMC peripheral membrane protein, and PfGAP50, an integral membrane protein of the IMC. This protein complex is referred to as the glideosome, and given its central role in erythrocyte invasion, this complex is likely the target of several specific regulatory effectors that ensure it is properly localized, assembled, and activated as the merozoite prepares to invade its target cell. However, little is known about how erythrocyte invasion as a whole is regulated, or about how or whether that regulation impacts the glideosome. Here we show that P. falciparum erythrocyte invasion is regulated by the release of intracellular calcium via the cyclic-ADP Ribose (cADPR) pathway, but that inhibition of cADPR-mediated calcium release does not affect PfGAP45 phosphorylation or glideosome association. By contrast, the serine/threonine kinase inhibitor, staurosporine, affects both PfGAP45 isoform distribution and the integrity of the glideosome complex. This data identifies specific regulatory elements involved in controlling P. falciparum erythrocyte invasion and reveals that the assembly status of the merozoite glideosome, which is central to erythrocyte invasion, is surprisingly dynamic.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">T32 AI055438</span>, <span class="grant">T32 AI055438-05</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2009</span>;<span class="volume">168</span>;<span class="number">1</span>;<span class="pagerange">55-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19576251">19576251</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2754074">2754074</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2009.06.007">10.1016/j.molbiopara.2009.06.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of a genetic isolate to identify rare disease variants: C7 on 5p associated with MS.</h4>
<p class="authors"><span class="author">Kallio SP</span>, <span class="author">Jakkula E</span>, <span class="author">Purcell S</span>, <span class="author">Suvela M</span>, <span class="author">Koivisto K</span>, <span class="author">Tienari PJ</span>, <span class="author">Elovaara I</span>, <span class="author">Pirttil&auml; T</span>, <span class="author">Reunanen M</span>, <span class="author">Bronnikov D</span>, <span class="author">Viander M</span>, <span class="author">Meri S</span>, <span class="author">Hillert J</span>, <span class="author">Lundmark F</span>, <span class="author">Harbo HF</span>, <span class="author">Lorentzen AR</span>, <span class="author">De Jager PL</span>, <span class="author">Daly MJ</span>, <span class="author">Hafler DA</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen L</span> and <span class="author">Saarela J</span></p>
<p class="affiliation">Finnish Institute for Molecular Medicine, Biomedicum, Helsinki, Finland.</p><p class="abstract">Large case-control genome-wide association studies primarily expose common variants contributing to disease pathogenesis with modest effects. Thus, alternative strategies are needed to tackle rare, possibly more penetrant alleles. One strategy is to use special populations with a founder effect and isolation, resulting in allelic enrichment. For multiple sclerosis such a unique setting is reported in Southern Ostrobothnia in Finland, where the prevalence and familial occurrence of multiple sclerosis (MS) are exceptionally high. Here, we have studied one of the best replicated MS loci, 5p, and monitored for haplotypes shared among 72 regional MS cases, the majority of which are genealogically distantly related. The haplotype analysis over the 45 Mb region, covering the linkage peak identified in Finnish MS families, revealed only modest association at IL7R (P = 0.04), recently implicated in MS, whereas most significant association was found with one haplotype covering the C7-FLJ40243 locus (P = 0.0001), 5.1 Mb centromeric of IL7R. The finding was validated in an independent sample from the isolate and resulted in an odds ratio of 2.73 (P = 0.000003) in the combined data set. The identified relatively rare risk haplotype contains C7 (complement component 7), an important player of the innate immune system. Suggestive association with alleles of the region was seen also in more heterogeneous populations. Interestingly, also the complement activity correlated with the identified risk haplotype. These results suggest that the MS predisposing locus on 5p is more complex than assumed and exemplify power of population isolates in the identification of rare disease alleles.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">U54 RR020278</span>; NIMH NIH HHS: <span class="grant">R01MH71425-01A1</span>; NINDS NIH HHS: <span class="grant">R01 NS 43559</span>; Wellcome Trust: <span class="grant">089061</span>, <span class="grant">089062</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">9</span>;<span class="pagerange">1670-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19221116" rel="external">19221116</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2667286" rel="external">2667286</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp073" rel="external">10.1093/hmg/ddp073</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Planning the human variome project: the Spain report.</h4>
<p class="authors"><span class="author">Kaput J</span>, <span class="author">Cotton RG</span>, <span class="author">Hardman L</span>, <span class="author">Watson M</span>, <span class="author">Al Aqeel AI</span>, <span class="author">Al-Aama JY</span>, <span class="author">Al-Mulla F</span>, <span class="author">Alonso S</span>, <span class="author">Aretz S</span>, <span class="author">Auerbach AD</span>, <span class="author">Bapat B</span>, <span class="author">Bernstein IT</span>, <span class="author">Bhak J</span>, <span class="author">Bleoo SL</span>, <span class="author">Bl&ouml;cker H</span>, <span class="author">Brenner SE</span>, <span class="author">Burn J</span>, <span class="author">Bustamante M</span>, <span class="author">Calzone R</span>, <span class="author">Cambon-Thomsen A</span>, <span class="author">Cargill M</span>, <span class="author">Carrera P</span>, <span class="author">Cavedon L</span>, <span class="author">Cho YS</span>, <span class="author">Chung YJ</span>, <span class="author">Claustres M</span>, <span class="author">Cutting G</span>, <span class="author">Dalgleish R</span>, <span class="author">den Dunnen JT</span>, <span class="author">D&iacute;az C</span>, <span class="author">Dobrowolski S</span>, <span class="author">dos Santos MR</span>, <span class="author">Ekong R</span>, <span class="author">Flanagan SB</span>, <span class="author">Flicek P</span>, <span class="author">Furukawa Y</span>, <span class="author">Genuardi M</span>, <span class="author">Ghang H</span>, <span class="author">Golubenko MV</span>, <span class="author">Greenblatt MS</span>, <span class="author">Hamosh A</span>, <span class="author">Hancock JM</span>, <span class="author">Hardison R</span>, <span class="author">Harrison TM</span>, <span class="author">Hoffmann R</span>, <span class="author">Horaitis R</span>, <span class="author">Howard HJ</span>, <span class="author">Barash CI</span>, <span class="author">Izagirre N</span>, <span class="author">Jung J</span>, <span class="author">Kojima T</span>, <span class="author">Laradi S</span>, <span class="author">Lee YS</span>, <span class="author">Lee JY</span>, <span class="author">Gil-da-Silva-Lopes VL</span>, <span class="author">Macrae FA</span>, <span class="author">Maglott D</span>, <span class="author">Marafie MJ</span>, <span class="author">Marsh SG</span>, <span class="author">Matsubara Y</span>, <span class="author">Messiaen LM</span>, <span class="author">M&ouml;slein G</span>, <span class="author">Netea MG</span>, <span class="author">Norton ML</span>, <span class="author">Oefner PJ</span>, <span class="author">Oetting WS</span>, <span class="author">O&#39;Leary JC</span>, <span class="author">de Ramirez AM</span>, <span class="author">Paalman MH</span>, <span class="author">Parboosingh J</span>, <span class="author">Patrinos GP</span>, <span class="author">Perozzi G</span>, <span class="author">Phillips IR</span>, <span class="author">Povey S</span>, <span class="author">Prasad S</span>, <span class="author">Qi M</span>, <span class="author">Quin DJ</span>, <span class="author">Ramesar RS</span>, <span class="author">Richards CS</span>, <span class="author">Savige J</span>, <span class="author">Scheible DG</span>, <span class="author">Scott RJ</span>, <span class="author">Seminara D</span>, <span class="author">Shephard EA</span>, <span class="author">Sijmons RH</span>, <span class="author">Smith TD</span>, <span class="author">Sobrido MJ</span>, <span class="author">Tanaka T</span>, <span class="author">Tavtigian SV</span>, <span class="author">Taylor GR</span>, <span class="author">Teague J</span>, <span class="author">T&ouml;pel T</span>, <span class="author">Ullman-Cullere M</span>, <span class="author">Utsunomiya J</span>, <span class="author">van Kranen HJ</span>, <span class="author">Vihinen M</span>, <span class="author">Webb E</span>, <span class="author">Weber TK</span>, <span class="author">Yeager M</span>, <span class="author">Yeom YI</span>, <span class="author">Yim SH</span>, <span class="author">Yoo HS</span> and <span class="author">Contributors to the Human Variome Project Planning Meeting</span></p>
<p class="affiliation">Division of Personalised Nutrition and Medicine, FDA/National Center for Toxicological Research, Jefferson, Arkansas 72079, USA. <a href="mailto:%4A%61%6D%65%73.%6B%61%70%75%74%40%66%64%61.%68%68%73.%67%6F%76">&#74;&#97;&#109;&#101;&#115;&#46;&#107;&#97;&#112;&#117;&#116;&#64;&#102;&#100;&#97;&#46;&#104;&#104;&#115;&#46;&#103;&#111;&#118;</a></p><p class="abstract">The remarkable progress in characterizing the human genome sequence, exemplified by the Human Genome Project and the HapMap Consortium, has led to the perception that knowledge and the tools (e.g., microarrays) are sufficient for many if not most biomedical research efforts. A large amount of data from diverse studies proves this perception inaccurate at best, and at worst, an impediment for further efforts to characterize the variation in the human genome. Because variation in genotype and environment are the fundamental basis to understand phenotypic variability and heritability at the population level, identifying the range of human genetic variation is crucial to the development of personalized nutrition and medicine. The Human Variome Project (HVP; http://www.humanvariomeproject.org/) was proposed initially to systematically collect mutations that cause human disease and create a cyber infrastructure to link locus specific databases (LSDB). We report here the discussions and recommendations from the 2008 HVP planning meeting held in San Feliu de Guixols, Spain, in May 2008.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2009</span>;<span class="volume">30</span>;<span class="number">4</span>;<span class="pagerange">496-510</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19306394" rel="external">19306394</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.20972" rel="external">10.1002/humu.20972</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The impact of newly identified loci on coronary heart disease, stroke and total mortality in the MORGAM prospective cohorts.</h4>
<p class="authors"><span class="author">Karvanen J</span>, <span class="author">Silander K</span>, <span class="author">Kee F</span>, <span class="author">Tiret L</span>, <span class="author">Salomaa V</span>, <span class="author">Kuulasmaa K</span>, <span class="author">Wiklund PG</span>, <span class="author">Virtamo J</span>, <span class="author">Saarela O</span>, <span class="author">Perret C</span>, <span class="author">Perola M</span>, <span class="author">Peltonen L</span>, <span class="author">Cambien F</span>, <span class="author">Erdmann J</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Evans A</span> and <span class="author">MORGAM Project</span></p>
<p class="affiliation">Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki, Finland. <a href="mailto:%6A%75%68%61.%6B%61%72%76%61%6E%65%6E%40%6B%74%6C.%66%69">&#106;&#117;&#104;&#97;&#46;&#107;&#97;&#114;&#118;&#97;&#110;&#101;&#110;&#64;&#107;&#116;&#108;&#46;&#102;&#105;</a></p><p class="abstract">Recently, genome wide association studies (GWAS) have identified a number of single nucleotide polymorphisms (SNPs) as being associated with coronary heart disease (CHD). We estimated the effect of these SNPs on incident CHD, stroke and total mortality in the prospective cohorts of the MORGAM Project. We studied cohorts from Finland, Sweden, France and Northern Ireland (total N=33,282, including 1,436 incident CHD events and 571 incident stroke events). The lead SNPs at seven loci identified thus far and additional SNPs (in total 42) were genotyped using a case-cohort design. We estimated the effect of the SNPs on disease history at baseline, disease events during follow-up and classic risk factors. Multiple testing was taken into account using false discovery rate (FDR) analysis. SNP rs1333049 on chromosome 9p21.3 was associated with both CHD and stroke (HR=1.20, 95% CI 1.08-1.34 for incident CHD events and 1.15, 0.99-1.34 for incident stroke). SNP rs11670734 (19q12) was associated with total mortality and stroke. SNP rs2146807 (10q11.21) showed some association with the fatality of acute coronary event. SNP rs2943634 (2q36.3) was associated with high density lipoprotein (HDL) cholesterol and SNPs rs599839, rs4970834 (1p13.3) and rs17228212 (15q22.23) were associated with non-HDL cholesterol. SNPs rs2943634 (2q36.3) and rs12525353 (6q25.1) were associated with blood pressure. These findings underline the need for replication studies in prospective settings and confirm the candidacy of several SNPs that may play a role in the etiology of cardiovascular disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">33</span>;<span class="number">3</span>;<span class="pagerange">237-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18979498" rel="external">18979498</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2696097" rel="external">2696097</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20374" rel="external">10.1002/gepi.20374</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of restless legs syndrome loci in three European populations.</h4>
<p class="authors"><span class="author">Kemlink D</span>, <span class="author">Polo O</span>, <span class="author">Frauscher B</span>, <span class="author">Gschliesser V</span>, <span class="author">H&ouml;gl B</span>, <span class="author">Poewe W</span>, <span class="author">Vodicka P</span>, <span class="author">Vavrova J</span>, <span class="author">Sonka K</span>, <span class="author">Nevsimalova S</span>, <span class="author">Schormair B</span>, <span class="author">Lichtner P</span>, <span class="author">Silander K</span>, <span class="author">Peltonen L</span>, <span class="author">Gieger C</span>, <span class="author">Wichmann HE</span>, <span class="author">Zimprich A</span>, <span class="author">Roeske D</span>, <span class="author">M&uuml;ller-Myhsok B</span>, <span class="author">Meitinger T</span> and <span class="author">Winkelmann J</span></p>
<p class="affiliation">Helmholtz Zentrum Munich, National Research Center of Environment and Health, Institute of Human Genetics, Munich, Germany.</p><p class="abstract"><strong>Background</strong>: Restless legs syndrome (RLS) is associated with common variants in three intronic and intergenic regions in MEIS1, BTBD9, and MAP2K5/LBXCOR1 on chromosomes 2p, 6p and 15q.</p><p class="abstract"><strong>Methods</strong>: Our study investigated these variants in 649 RLS patients and 1230 controls from the Czech Republic (290 cases and 450 controls), Austria (269 cases and 611 controls) and Finland (90 cases and 169 controls). Ten single nucleotide polymorphisms (SNPs) within the three genomic regions were selected according to the results of previous genome-wide scans. Samples were genotyped using Sequenom platforms.</p><p class="abstract"><strong>Results</strong>: We replicated associations for all loci in the combined samples set (rs2300478 in MEIS1, p = 1.26 x 10(-5), odds ratio (OR) = 1.47, rs3923809 in BTBD9, p = 4.11 x 10(-5), OR = 1.58 and rs6494696 in MAP2K5/LBXCOR1, p = 0.04764, OR = 1.27). Analysing only familial cases against all controls, all three loci were significantly associated. Using sporadic cases only, we could confirm the association only with BTBD9.</p><p class="abstract"><strong>Conclusion</strong>: Our study shows that variants in these three loci confer consistent disease risks in patients of European descent. Among the known loci, BTBD9 seems to be the most consistent in its effect on RLS across populations and is also most independent of familial clustering.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">46</span>;<span class="number">5</span>;<span class="pagerange">315-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19279021" rel="external">19279021</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2695663" rel="external">2695663</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2008.062992" rel="external">10.1136/jmg.2008.062992</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cyclin-dependent kinase inhibits reinitiation of a normal S-phase program during G2 in fission yeast.</h4>
<p class="authors"><span class="author">Kiang L</span>, <span class="author">Heichinger C</span>, <span class="author">Watt S</span>, <span class="author">B&auml;hler J</span> and <span class="author">Nurse P</span></p>
<p class="affiliation">Laboratory of Yeast Genetics and Cell Biology, The Rockefeller University, 1230 York Avenue, New York, NY 10065, USA. <a href="mailto:%6C%6B%69%61%6E%67%40%72%6F%63%6B%65%66%65%6C%6C%65%72.%65%64%75">&#108;&#107;&#105;&#97;&#110;&#103;&#64;&#114;&#111;&#99;&#107;&#101;&#102;&#101;&#108;&#108;&#101;&#114;&#46;&#101;&#100;&#117;</a></p><p class="abstract">To achieve faithful replication of the genome once in each cell cycle, reinitiation of S phase is prevented in G(2) and origins are restricted from refiring within S phase. We have investigated the block to rereplication during G(2) in fission yeast. The DNA synthesis that occurs when G(2)/M cyclin-dependent kinase (CDK) activity is depleted has been assumed to be repeated rounds of S phase without mitosis, but this has not been demonstrated to be the case. We show here that on G(2)/M CDK depletion in G(2), repeated S phases are induced, which are correlated with normal G(1)/S transcription and attainment of doublings in cell size. Mostly normal mitotic S-phase origins are utilized, although at different efficiencies, and replication is essentially equal across the genome. We conclude that CDK inhibits reinitiation of S phase during G(2), and if G(2)/M CDK is depleted, replication results from induction of a largely normal S-phase program with only small differences in origin usage and efficiency.</p>
<p class="grants">Funded by: Cancer Research UK; NIGMS NIH HHS: <span class="grant">GM 07739</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">29</span>;<span class="number">15</span>;<span class="pagerange">4025-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19487461" rel="external">19487461</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2715816" rel="external">2715816</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.00185-09" rel="external">10.1128/MCB.00185-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Linkage and linkage disequilibrium scan for autism loci in an extended pedigree from Finland.</h4>
<p class="authors"><span class="author">Kilpinen H</span>, <span class="author">Ylisaukko-oja T</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Ga&aacute;l E</span>, <span class="author">Turunen JA</span>, <span class="author">Kempas E</span>, <span class="author">von Wendt L</span>, <span class="author">Varilo T</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Molecular Medicine, Institute for Molecular Medicine, Finland.</p><p class="abstract">Population isolates, such as Finland, have proved beneficial in mapping rare causative genetic variants due to a limited number of founders resulting in reduced genetic heterogeneity and extensive linkage disequilibrium (LD). We have here used this special opportunity to identify rare alleles in autism by genealogically tracing 20 autism families into one extended pedigree with verified genealogical links reaching back to the 17th century. In this unique pedigree, we performed a dense microsatellite marker genome-wide scan of linkage and LD and followed initial findings with extensive fine-mapping. We identified a putative autism susceptibility locus at 19p13.3 and obtained further evidence for previously identified loci at 1q23 and 15q11-q13. Most promising candidate genes were TLE2 and TLE6 clustered at 19p13 and ATP1A2 at 1q23.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">15</span>;<span class="pagerange">2912-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19454485" rel="external">19454485</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2708134" rel="external">2708134</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp229" rel="external">10.1093/hmg/ddp229</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Typhoid Fever</h4>
<p class="authors"><span class="author">Kingsley,R.A.</span> and <span class="author">Dougan,G.;</span></p>
<p class="publication"><span class="title">Vaccines for Biodefense and Emerging and Neglected Diseases</span>&nbsp;<span class="year">2009</span>;<span class="number">Chapter 57</span>;<span class="pagerange">1147–1161</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-369408-9.00057-3">10.1016/B978-0-12-369408-9.00057-3</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123694089000573" href="http://www.sciencedirect.com/science/article/pii/B9780123694089000573">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123694089</span>000573</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Support for the involvement of large copy number variants in the pathogenesis of schizophrenia.</h4>
<p class="authors"><span class="author">Kirov G</span>, <span class="author">Grozeva D</span>, <span class="author">Norton N</span>, <span class="author">Ivanov D</span>, <span class="author">Mantripragada KK</span>, <span class="author">Holmans P</span>, <span class="author">International Schizophrenia Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Craddock N</span>, <span class="author">Owen MJ</span> and <span class="author">O&#39;Donovan MC</span></p>
<p class="affiliation">Department of Psychological Medicine, Cardiff University, Heath Park, Cardiff, UK.</p><p class="abstract">We investigated the involvement of rare (&lt;1%) copy number variants (CNVs) in 471 cases of schizophrenia and 2792 controls that had been genotyped using the Affymetrix GeneChip 500K Mapping Array. Large CNVs &gt;1 Mb were 2.26 times more common in cases (P = 0.00027), with the effect coming mostly from deletions (odds ratio, OR = 4.53, P = 0.00013) although duplications were also more common (OR = 1.71, P = 0.04). Two large deletions were found in two cases each, but in no controls: a deletion at 22q11.2 known to be a susceptibility factor for schizophrenia and a deletion on 17p12, at 14.0-15.4 Mb. The latter is known to cause hereditary neuropathy with liability to pressure palsies. The same deletion was found in 6 of 4618 (0.13%) cases and 6 of 36 092 (0.017%) controls in the re-analysed data of two recent large CNV studies of schizophrenia (OR = 7.82, P = 0.001), with the combined significance level for all three studies achieving P = 5 x 10(-5). One large duplication on 16p13.1, which has been previously implicated as a susceptibility factor for autism, was found in three cases and six controls (0.6% versus 0.2%, OR = 2.98, P = 0.13). We also provide the first support for a recently reported association between deletions at 15q11.2 and schizophrenia (P = 0.026). This study confirms the involvement of rare CNVs in the pathogenesis of schizophrenia and contributes to the growing list of specific CNVs that are implicated.</p>
<p class="grants">Funded by: Medical Research Council; NIMH NIH HHS: <span class="grant">2 P50 MH066392-05A1</span>; Wellcome Trust: <span class="grant">076113</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">8</span>;<span class="pagerange">1497-503</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19181681">19181681</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2664144">2664144</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp043">10.1093/hmg/ddp043</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of 28,141 individuals identifies common variants within five new loci that influence uric acid concentrations.</h4>
<p class="authors"><span class="author">Kolz M</span>, <span class="author">Johnson T</span>, <span class="author">Sanna S</span>, <span class="author">Teumer A</span>, <span class="author">Vitart V</span>, <span class="author">Perola M</span>, <span class="author">Mangino M</span>, <span class="author">Albrecht E</span>, <span class="author">Wallace C</span>, <span class="author">Farrall M</span>, <span class="author">Johansson A</span>, <span class="author">Nyholt DR</span>, <span class="author">Aulchenko Y</span>, <span class="author">Beckmann JS</span>, <span class="author">Bergmann S</span>, <span class="author">Bochud M</span>, <span class="author">Brown M</span>, <span class="author">Campbell H</span>, <span class="author">EUROSPAN Consortium</span>, <span class="author">Connell J</span>, <span class="author">Dominiczak A</span>, <span class="author">Homuth G</span>, <span class="author">Lamina C</span>, <span class="author">McCarthy MI</span>, <span class="author">ENGAGE Consortium</span>, <span class="author">Meitinger T</span>, <span class="author">Mooser V</span>, <span class="author">Munroe P</span>, <span class="author">Nauck M</span>, <span class="author">Peden J</span>, <span class="author">Prokisch H</span>, <span class="author">Salo P</span>, <span class="author">Salomaa V</span>, <span class="author">Samani NJ</span>, <span class="author">Schlessinger D</span>, <span class="author">Uda M</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Waeber G</span>, <span class="author">Waterworth D</span>, <span class="author">Wang-Sattler R</span>, <span class="author">Wright AF</span>, <span class="author">Adamski J</span>, <span class="author">Whitfield JB</span>, <span class="author">Gyllensten U</span>, <span class="author">Wilson JF</span>, <span class="author">Rudan I</span>, <span class="author">Pramstaller P</span>, <span class="author">Watkins H</span>, <span class="author">PROCARDIS Consortium</span>, <span class="author">Doering A</span>, <span class="author">Wichmann HE</span>, <span class="author">KORA Study</span>, <span class="author">Spector TD</span>, <span class="author">Peltonen L</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Nagaraja R</span>, <span class="author">Vollenweider P</span>, <span class="author">Caulfield M</span>, <span class="author">WTCCC</span>, <span class="author">Illig T</span> and <span class="author">Gieger C</span></p>
<p class="affiliation">Institute of Epidemiology, Helmholtz Zentrum M&uuml;nchen, National Research Center for Environment and Health, Neuherberg, Germany.</p><p class="abstract">Elevated serum uric acid levels cause gout and are a risk factor for cardiovascular disease and diabetes. To investigate the polygenetic basis of serum uric acid levels, we conducted a meta-analysis of genome-wide association scans from 14 studies totalling 28,141 participants of European descent, resulting in identification of 954 SNPs distributed across nine loci that exceeded the threshold of genome-wide significance, five of which are novel. Overall, the common variants associated with serum uric acid levels fall in the following nine regions: SLC2A9 (p = 5.2x10(-201)), ABCG2 (p = 3.1x10(-26)), SLC17A1 (p = 3.0x10(-14)), SLC22A11 (p = 6.7x10(-14)), SLC22A12 (p = 2.0x10(-9)), SLC16A9 (p = 1.1x10(-8)), GCKR (p = 1.4x10(-9)), LRRC16A (p = 8.5x10(-9)), and near PDZK1 (p = 2.7x10(-9)). Identified variants were analyzed for gender differences. We found that the minor allele for rs734553 in SLC2A9 has greater influence in lowering uric acid levels in women and the minor allele of rs2231142 in ABCG2 elevates uric acid levels more strongly in men compared to women. To further characterize the identified variants, we analyzed their association with a panel of metabolites. rs12356193 within SLC16A9 was associated with DL-carnitine (p = 4.0x10(-26)) and propionyl-L-carnitine (p = 5.0x10(-8)) concentrations, which in turn were associated with serum UA levels (p = 1.4x10(-57) and p = 8.1x10(-54), respectively), forming a triangle between SNP, metabolites, and UA levels. Taken together, these associations highlight additional pathways that are important in the regulation of serum uric acid levels and point toward novel potential targets for pharmacological intervention to prevent or treat hyperuricemia. In addition, these findings strongly support the hypothesis that transport proteins are key in regulating serum uric acid levels.</p>
<p class="grants">Funded by: Arthritis Research UK; British Heart Foundation: <span class="grant">FS/05/061/19501</span>, <span class="grant">PG02/128</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0400874</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_U127561128</span>; NIA NIH HHS: <span class="grant">N01-AG-1-2109</span>; NIAAA NIH HHS: <span class="grant">AA007535</span>; Wellcome Trust: <span class="grant">076113/B/04/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e1000504</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19503597">19503597</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2683940">2683940</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000504">10.1371/journal.pgen.1000504</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parental origin of sequence variants associated with complex diseases.</h4>
<p class="authors"><span class="author">Kong A</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Masson G</span>, <span class="author">Thorleifsson G</span>, <span class="author">Sulem P</span>, <span class="author">Besenbacher S</span>, <span class="author">Jonasdottir A</span>, <span class="author">Sigurdsson A</span>, <span class="author">Kristinsson KT</span>, <span class="author">Jonasdottir A</span>, <span class="author">Frigge ML</span>, <span class="author">Gylfason A</span>, <span class="author">Olason PI</span>, <span class="author">Gudjonsson SA</span>, <span class="author">Sverrisson S</span>, <span class="author">Stacey SN</span>, <span class="author">Sigurgeirsson B</span>, <span class="author">Benediktsdottir KR</span>, <span class="author">Sigurdsson H</span>, <span class="author">Jonsson T</span>, <span class="author">Benediktsson R</span>, <span class="author">Olafsson JH</span>, <span class="author">Johannsson OT</span>, <span class="author">Hreidarsson AB</span>, <span class="author">Sigurdsson G</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Ferguson-Smith AC</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Thorsteinsdottir U</span> and <span class="author">Stefansson K</span></p>
<p class="affiliation">deCODE genetics, Sturlugata 8, 101 Reykjav&iacute;k, Iceland. <a href="mailto:%6B%6F%6E%67%40%64%65%63%6F%64%65.%69%73">&#107;&#111;&#110;&#103;&#64;&#100;&#101;&#99;&#111;&#100;&#101;&#46;&#105;&#115;</a></p><p class="abstract">Effects of susceptibility variants may depend on from which parent they are inherited. Although many associations between sequence variants and human traits have been discovered through genome-wide associations, the impact of parental origin has largely been ignored. Here we show that for 38,167 Icelanders genotyped using single nucleotide polymorphism (SNP) chips, the parental origin of most alleles can be determined. For this we used a combination of genealogy and long-range phasing. We then focused on SNPs that associate with diseases and are within 500 kilobases of known imprinted genes. Seven independent SNP associations were examined. Five-one with breast cancer, one with basal-cell carcinoma and three with type 2 diabetes-have parental-origin-specific associations. These variants are located in two genomic regions, 11p15 and 7q32, each harbouring a cluster of imprinted genes. Furthermore, we observed a novel association between the SNP rs2334499 at 11p15 and type 2 diabetes. Here the allele that confers risk when paternally inherited is protective when maternally transmitted. We identified a differentially methylated CTCF-binding site at 11p15 and demonstrated correlation of rs2334499 with decreased methylation of that site.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G9723500</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179474</span>, <span class="grant">MC_U127592696</span>; NIAMS NIH HHS: <span class="grant">K08 AR055688</span>; NIDDK NIH HHS: <span class="grant">R01 DK029867</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2009</span>;<span class="volume">462</span>;<span class="number">7275</span>;<span class="pagerange">868-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20016592">20016592</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3746295">3746295</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08625">10.1038/nature08625</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication in genome-wide association studies.</h4>
<p class="authors"><span class="author">Kraft P</span>, <span class="author">Zeggini E</span> and <span class="author">Ioannidis JP</span></p>
<p class="affiliation">Departments of Epidemiology and Biostatistics, Harvard School of Public Health, Boston, MA, USA.</p><p class="abstract">Replication helps ensure that a genotype-phenotype association observed in a genome-wide association (GWA) study represents a credible association and is not a chance finding or an artifact due to uncontrolled biases. We discuss prerequisites for exact replication; issues of heterogeneity; advantages and disadvantages of different methods of data synthesis across multiple studies; frequentist vs. Bayesian inferences for replication; and challenges that arise from multi-team collaborations. While consistent replication can greatly improve the credibility of a genotype-phenotype association, it may not eliminate spurious associations due to biases shared by many studies. Conversely, lack of replication in well-powered follow-up studies usually invalidates the initially proposed association, although occasionally it may point to differences in linkage disequilibrium or effect modifiers across studies.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">UL1 RR025752-01</span></p>
<p class="publication"><span class="title">Statistical science : a review journal of the Institute of Mathematical Statistics</span>&nbsp;<span class="year">2009</span>;<span class="volume">24</span>;<span class="number">4</span>;<span class="pagerange">561-573</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20454541" rel="external">20454541</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2865141" rel="external">2865141</a>; DOI: <a href="http://dx.doi.org/10.1214%2F09-STS290" rel="external">10.1214/09-STS290</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Constitutional haploinsufficiency of tumor suppressor genes in mentally retarded patients with microdeletions in 17p13.1.</h4>
<p class="authors"><span class="author">Krepischi-Santos AC</span>, <span class="author">Rajan D</span>, <span class="author">Temple IK</span>, <span class="author">Shrubb V</span>, <span class="author">Crolla JA</span>, <span class="author">Huang S</span>, <span class="author">Beal S</span>, <span class="author">Otto PA</span>, <span class="author">Carter NP</span>, <span class="author">Vianna-Morgante AM</span> and <span class="author">Rosenberg C</span></p>
<p class="affiliation">Department of Genetics and Evolutionary Biology, Institute of Biosciences, University of S&atilde;o Paulo, S&atilde;o Paulo, Brazil.</p><p class="abstract">Chromosome microdeletions or duplications are detected in 10-20% of patients with mental impairment and normal karyotypes. A few cases have been reported of mental impairment with microdeletions comprising tumor suppressor genes. By array-CGH we detected 4 mentally impaired individuals carrying de novo microdeletions sharing an overlapping segment of approximately 180 kb in 17p13.1. This segment encompasses 18 genes, including 3 involved in cancer, namely KCTD11/REN, DLG4/PSD95, and GPS2. Furthermore, in 2 of the patients, the deletions also included TP53, the most frequently inactivated gene in human cancers. The 3 tumor suppressor genes KCTD11, DLG4, and GPS2, in addition to the GABARAP gene, have a known or suspected function in neuronal development and are candidates for causing mental impairment in our patients. Among our 4 patients with deletions in 17p13.1, 3 were part of a Brazilian cohort of 300 mentally retarded individuals, suggesting that this segment may be particularly prone to rearrangements and appears to be an important cause (approximately 1%) of mental retardation. Further, the constitutive deletion of tumor suppressor genes in these patients, particularly TP53, probably confers a significantly increased lifetime risk for cancer and warrants careful oncological surveillance of these patients. Constitutional chromosome deletions containing tumor suppressor genes in patients with mental impairment or congenital abnormalities may represent an important mechanism linking abnormal phenotypes with increased risks of cancer.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">125</span>;<span class="number">1</span>;<span class="pagerange">1-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19617690" rel="external">19617690</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2813804" rel="external">2813804</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000218743" rel="external">10.1159/000218743</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cross-species chromosome painting in Cetartiodactyla: reconstructing the karyotype evolution in key phylogenetic lineages.</h4>
<p class="authors"><span class="author">Kulemzina AI</span>, <span class="author">Trifonov VA</span>, <span class="author">Perelman PL</span>, <span class="author">Rubtsova NV</span>, <span class="author">Volobuev V</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Stanyon R</span>, <span class="author">Yang F</span> and <span class="author">Graphodatsky AS</span></p>
<p class="affiliation">Institute of Cytology and Genetics, Russian Academy of Sciences, Novosibirsk, 630090, Russia.</p><p class="abstract">Recent molecular and morphological studies place Artiodactyla and Cetacea into the order Cetartiodactyla. Within the Cetartiodactyla such families as Bovidae, Cervidae, and Suidae are well studied by comparative chromosome painting, but many taxa that are crucial for understanding cetartiodactyl phylogeny remain poorly studied. Here we present the genome-wide comparative maps of five cetartiodactyl species obtained by chromosome painting with human and dromedary paint probes from four taxa: Cetacea, Hippopotamidae, Giraffidae, and Moschidae. This is the first molecular cytogenetic report on pilot whale, hippopotamus, okapi, and Siberian musk deer. Our results, when integrated with previously published comparative chromosome maps allow us to reconstruct the evolutionary pathway and rates of chromosomal rearrangements in Cetartiodactyla. We hypothesize that the putative cetartiodactyl ancestral karyotype (CAK) contained 25-26 pairs of autosomes, 2n = 52-54, and that the association of human chromosomes 8/9 could be a cytogenetic signature that unites non-camelid cetartiodactyls. There are no unambiguous cytogenetic landmarks that unite Hippopotamidae and Cetacea. If we superimpose chromosome rearrangements on the supertree generated by Price and colleagues, several homoplasy events are needed to explain cetartiodactyl karyotype evolution. Our results apparently favour a model of non-random breakpoints in chromosome evolution. Cetariodactyl karyotype evolution is characterized by alternating periods of low and fast rates in various lineages. The highest rates are found in Suina (Suidae+Tayasuidae) lineage (1.76 rearrangements per million years (R/My)) and the lowest in Cetaceans (0.07 R/My). Our study demonstrates that the combined use of human and camel paints is highly informative for revealing evolutionary karyotypic rearrangements among cetartiodactyl species.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">419-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19350402" rel="external">19350402</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-009-9032-3" rel="external">10.1007/s10577-009-9032-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common genetic variation in the melatonin receptor 1B gene (MTNR1B) is associated with decreased early-phase insulin response.</h4>
<p class="authors"><span class="author">Langenberg C</span>, <span class="author">Pascoe L</span>, <span class="author">Mari A</span>, <span class="author">Tura A</span>, <span class="author">Laakso M</span>, <span class="author">Frayling TM</span>, <span class="author">Barroso I</span>, <span class="author">Loos RJ</span>, <span class="author">Wareham NJ</span>, <span class="author">Walker M</span> and <span class="author">RISC Consortium</span></p>
<p class="affiliation">MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge CB2 0QQ, UK. <a href="mailto:%63%6C%61%75%64%69%61.%6C%61%6E%67%65%6E%62%65%72%67%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#99;&#108;&#97;&#117;&#100;&#105;&#97;&#46;&#108;&#97;&#110;&#103;&#101;&#110;&#98;&#101;&#114;&#103;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We investigated whether variation in MTNR1B, which was recently identified as a common genetic determinant of fasting glucose levels in healthy, diabetes-free individuals, is associated with measures of beta cell function and whole-body insulin sensitivity.</p><p class="abstract"><strong>Methods</strong>: We studied 1,276 healthy individuals of European ancestry at 19 centres of the Relationship between Insulin Sensitivity and Cardiovascular disease (RISC) study. Whole-body insulin sensitivity was assessed by euglycaemic-hyperinsulinaemic clamp and indices of beta cell function were derived from a 75 g oral glucose tolerance test (including 30 min insulin response and glucose sensitivity). We studied rs10830963 in MTNR1B using additive genetic models, adjusting for age, sex and recruitment centre.</p><p class="abstract"><strong>Results</strong>: The minor (G) allele of rs10830963 in MTNR1B (frequency 0.30 in HapMap Centre d&#39;Etude du Polymorphisme [Utah residents with northern and western European ancestry] [CEU]; 0.29 in RISC participants) was associated with higher levels of fasting plasma glucose (standardised beta [95% CI] 0.17 [0.085, 0.25] per G allele, p = 5.8 x 10(-5)), consistent with recent observations. In addition, the G-allele was significantly associated with lower early insulin response (-0.19 [-0.28, -0.10], p = 1.7 x 10(-5)), as well as with decreased beta cell glucose sensitivity (-0.11 [-0.20, -0.027], p = 0.010). No associations were observed with clamp-assessed insulin sensitivity (p = 0.15) or different measures of body size (p &gt; 0.7 for all).</p><p class="abstract">Genetic variation in MTNR1B is associated with defective early insulin response and decreased beta cell glucose sensitivity, which may contribute to the higher glucose levels of non-diabetic individuals carrying the minor G allele of rs10830963 in MTNR1B.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council: <span class="grant">G0701863</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2009</span>;<span class="volume">52</span>;<span class="number">8</span>;<span class="pagerange">1537-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19455304">19455304</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2709880">2709880</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-009-1392-x">10.1007/s00125-009-1392-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Testing the water: marine metagenomics.</h4>
<p class="authors"><span class="author">Langridge G</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">8</span>;<span class="pagerange">552</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19609255">19609255</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2188">10.1038/nrmicro2188</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nontyphoidal Salmonella serovars cause different degrees of invasive disease globally.</h4>
<p class="authors"><span class="author">Langridge GC</span>, <span class="author">Nair S</span> and <span class="author">Wain J</span></p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2009</span>;<span class="volume">199</span>;<span class="number">4</span>;<span class="pagerange">602-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19281309" rel="external">19281309</a>; DOI: <a href="http://dx.doi.org/10.1086%2F596208" rel="external">10.1086/596208</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antibiotic treatment of clostridium difficile carrier mice triggers a supershedder state, spore-mediated transmission, and severe disease in immunocompromised hosts.</h4>
<p class="authors"><span class="author">Lawley TD</span>, <span class="author">Clare S</span>, <span class="author">Walker AW</span>, <span class="author">Goulding D</span>, <span class="author">Stabler RA</span>, <span class="author">Croucher N</span>, <span class="author">Mastroeni P</span>, <span class="author">Scott P</span>, <span class="author">Raisen C</span>, <span class="author">Mottram L</span>, <span class="author">Fairweather NF</span>, <span class="author">Wren BW</span>, <span class="author">Parkhill J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Microbial Pathogenesis Laboratory1 and Pathogen Genomics, Hinxton, United Kingdom. <a href="mailto:%74%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Clostridium difficile persists in hospitals by exploiting an infection cycle that is dependent on humans shedding highly resistant and infectious spores. Here we show that human virulent C. difficile can asymptomatically colonize the intestines of immunocompetent mice, establishing a carrier state that persists for many months. C. difficile carrier mice consistently shed low levels of spores but, surprisingly, do not transmit infection to cohabiting mice. However, antibiotic treatment of carriers triggers a highly contagious supershedder state, characterized by a dramatic reduction in the intestinal microbiota species diversity, C. difficile overgrowth, and excretion of high levels of spores. Stopping antibiotic treatment normally leads to recovery of the intestinal microbiota species diversity and suppresses C. difficile levels, although some mice persist in the supershedding state for extended periods. Spore-mediated transmission to immunocompetent mice treated with antibiotics results in self-limiting mucosal inflammation of the large intestine. In contrast, transmission to mice whose innate immune responses are compromised (Myd88(-/-)) leads to a severe intestinal disease that is often fatal. Thus, mice can be used to investigate distinct stages of the C. difficile infection cycle and can serve as a valuable surrogate for studying the spore-mediated transmission and interactions between C. difficile and the host and its microbiota, and the results obtained should guide infection control measures.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2009</span>;<span class="volume">77</span>;<span class="number">9</span>;<span class="pagerange">3661-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19564382">19564382</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2737984">2737984</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00558-09">10.1128/IAI.00558-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proteomic and genomic characterization of highly infectious Clostridium difficile 630 spores.</h4>
<p class="authors"><span class="author">Lawley TD</span>, <span class="author">Croucher NJ</span>, <span class="author">Yu L</span>, <span class="author">Clare S</span>, <span class="author">Sebaihia M</span>, <span class="author">Goulding D</span>, <span class="author">Pickard DJ</span>, <span class="author">Parkhill J</span>, <span class="author">Choudhary J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Microbial Pathogenesis Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%74%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Clostridium difficile, a major cause of antibiotic-associated diarrhea, produces highly resistant spores that contaminate hospital environments and facilitate efficient disease transmission. We purified C. difficile spores using a novel method and show that they exhibit significant resistance to harsh physical or chemical treatments and are also highly infectious, with &lt;7 environmental spores per cm(2) reproducibly establishing a persistent infection in exposed mice. Mass spectrometric analysis identified approximately 336 spore-associated polypeptides, with a significant proportion linked to translation, sporulation/germination, and protein stabilization/degradation. In addition, proteins from several distinct metabolic pathways associated with energy production were identified. Comparison of the C. difficile spore proteome to those of other clostridial species defined 88 proteins as the clostridial spore &quot;core&quot; and 29 proteins as C. difficile spore specific, including proteins that could contribute to spore-host interactions. Thus, our results provide the first molecular definition of C. difficile spores, opening up new opportunities for the development of diagnostic and therapeutic approaches.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">17</span>;<span class="pagerange">5377-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19542279">19542279</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2725610">2725610</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00597-09">10.1128/JB.00597-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GLIDERS--a web-based search engine for genome-wide linkage disequilibrium between HapMap SNPs.</h4>
<p class="authors"><span class="author">Lawrence R</span>, <span class="author">Day-Williams AG</span>, <span class="author">Mott R</span>, <span class="author">Broxholme J</span>, <span class="author">Cardon LR</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%7A.%72.%79%61%6E%67%40%65%78.%61%63.%75%6B">&#122;&#46;&#114;&#46;&#121;&#97;&#110;&#103;&#64;&#101;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: A number of tools for the examination of linkage disequilibrium (LD) patterns between nearby alleles exist, but none are available for quickly and easily investigating LD at longer ranges (&gt;500 kb). We have developed a web-based query tool (GLIDERS: Genome-wide LInkage DisEquilibrium Repository and Search engine) that enables the retrieval of pairwise associations with r2 &gt;or= 0.3 across the human genome for any SNP genotyped within HapMap phase 2 and 3, regardless of distance between the markers.</p><p class="abstract"><strong>Description</strong>: GLIDERS is an easy to use web tool that only requires the user to enter rs numbers of SNPs they want to retrieve genome-wide LD for (both nearby and long-range). The intuitive web interface handles both manual entry of SNP IDs as well as allowing users to upload files of SNP IDs. The user can limit the resulting inter SNP associations with easy to use menu options. These include MAF limit (5-45%), distance limits between SNPs (minimum and maximum), r2 (0.3 to 1), HapMap population sample (CEU, YRI and JPT+CHB combined) and HapMap build/release. All resulting genome-wide inter-SNP associations are displayed on a single output page, which has a link to a downloadable tab delimited text file.</p><p class="abstract"><strong>Conclusion</strong>: GLIDERS is a quick and easy way to retrieve genome-wide inter-SNP associations and to explore LD patterns for any number of SNPs of interest. GLIDERS can be useful in identifying SNPs with long-range LD. This can highlight mis-mapping or other potential association signal localisation problems.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079557</span>, <span class="grant">079557MA</span>, <span class="grant">088885/Z/09/Z</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">367</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19878600">19878600</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2777181">2777181</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-10-367">10.1186/1471-2105-10-367</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The brain structural disposition to social interaction.</h4>
<p class="authors"><span class="author">Lebreton M</span>, <span class="author">Barnes A</span>, <span class="author">Miettunen J</span>, <span class="author">Peltonen L</span>, <span class="author">Ridler K</span>, <span class="author">Veijola J</span>, <span class="author">Tanskanen P</span>, <span class="author">Suckling J</span>, <span class="author">Jarvelin MR</span>, <span class="author">Jones PB</span>, <span class="author">Isohanni M</span>, <span class="author">Bullmore ET</span> and <span class="author">Murray GK</span></p>
<p class="affiliation">Brain Mapping Unit, University of Cambridge, Cambridge, UK.</p><p class="abstract">Social reward dependence (RD) in humans is a stable pattern of attitudes and behaviour hypothesized to represent a favourable disposition towards social relationships and attachment as a personality dimension. It has been theorized that this long-term disposition to openness is linked to the capacity to process primary reward. Using brain structure measures from magnetic resonance imaging, and a measure of RD from Cloninger&#39;s temperament and character inventory, a self-reported questionnaire, in 41 male subjects sampled from a general population birth cohort, we investigated the neuro-anatomical basis of social RD. We found that higher social RD in men was significantly associated with increased gray matter density in the orbitofrontal cortex, basal ganglia and temporal lobes, regions that have been previously shown to be involved in processing of primary rewards. These findings provide evidence for a brain structural disposition to social interaction, and that sensitivity to social reward shares a common neural basis with systems for processing primary reward information.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">The European journal of neuroscience</span>&nbsp;<span class="year">2009</span>;<span class="volume">29</span>;<span class="number">11</span>;<span class="pagerange">2247-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19490022">19490022</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1460-9568.2009.06782.x">10.1111/j.1460-9568.2009.06782.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Statistical estimation of cell-cycle progression and lineage commitment in Plasmodium falciparum reveals a homogeneous pattern of transcription in ex vivo culture.</h4>
<p class="authors"><span class="author">Lemieux JE</span>, <span class="author">Gomez-Escobar N</span>, <span class="author">Feller A</span>, <span class="author">Carret C</span>, <span class="author">Amambua-Ngwa A</span>, <span class="author">Pinches R</span>, <span class="author">Day F</span>, <span class="author">Kyes SA</span>, <span class="author">Conway DJ</span>, <span class="author">Holmes CC</span> and <span class="author">Newbold CI</span></p>
<p class="affiliation">Weatherall Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford, OX3 9DS, United Kingdom.</p><p class="abstract">We have cultured Plasmodium falciparum directly from the blood of infected individuals to examine patterns of mature-stage gene expression in patient isolates. Analysis of the transcriptome of P. falciparum is complicated by the highly periodic nature of gene expression because small variations in the stage of parasite development between samples can lead to an apparent difference in gene expression values. To address this issue, we have developed statistical likelihood-based methods to estimate cell cycle progression and commitment to asexual or sexual development lineages in our samples based on microscopy and gene expression patterns. In cases subsequently matched for temporal development, we find that transcriptional patterns in ex vivo culture display little variation across patients with diverse clinical profiles and closely resemble transcriptional profiles that occur in vitro. These statistical methods, available to the research community, assist in the design and interpretation of P. falciparum expression profiling experiments where it is difficult to separate true differential expression from cell-cycle dependent expression. We reanalyze an existing dataset of in vivo patient expression profiles and conclude that previously observed discrete variation is consistent with the commitment of a varying proportion of the parasite population to the sexual development lineage.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2009</span>;<span class="volume">106</span>;<span class="number">18</span>;<span class="pagerange">7559-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19376968" rel="external">19376968</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2670243" rel="external">2670243</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0811829106" rel="external">10.1073/pnas.0811829106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An ENU-induced mutation of miR-96 associated with progressive hearing loss in mice.</h4>
<p class="authors"><span class="author">Lewis MA</span>, <span class="author">Quint E</span>, <span class="author">Glazier AM</span>, <span class="author">Fuchs H</span>, <span class="author">De Angelis MH</span>, <span class="author">Langford C</span>, <span class="author">van Dongen S</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">Piipari M</span>, <span class="author">Redshaw N</span>, <span class="author">Dalmay T</span>, <span class="author">Moreno-Pelayo MA</span>, <span class="author">Enright AJ</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">Progressive hearing loss is common in the human population, but little is known about the molecular basis. We report a new N-ethyl-N-nitrosurea (ENU)-induced mouse mutant, diminuendo, with a single base change in the seed region of Mirn96. Heterozygotes show progressive loss of hearing and hair cell anomalies, whereas homozygotes have no cochlear responses. Most microRNAs are believed to downregulate target genes by binding to specific sites on their mRNAs, so mutation of the seed should lead to target gene upregulation. Microarray analysis revealed 96 transcripts with significantly altered expression in homozygotes; notably, Slc26a5, Ocm, Gfi1, Ptprq and Pitpnm1 were downregulated. Hypergeometric P-value analysis showed that hundreds of genes were upregulated in mutants. Different genes, with target sites complementary to the mutant seed, were downregulated. This is the first microRNA found associated with deafness, and diminuendo represents a model for understanding and potentially moderating progressive hair cell degeneration in hearing loss more generally.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust: <span class="grant">077189</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">5</span>;<span class="pagerange">614-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19363478">19363478</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705913">2705913</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.369">10.1038/ng.369</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fast and accurate short read alignment with Burrows-Wheeler transform.</h4>
<p class="authors"><span class="author">Li H</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK.</p><p class="abstract"><strong>Motivation</strong>: The enormous amount of short reads generated by the new DNA sequencing technologies call for the development of fast and accurate read alignment programs. A first generation of hash table-based methods has been developed, including MAQ, which is accurate, feature rich and fast enough to align short reads from a single individual. However, MAQ does not support gapped alignment for single-end reads, which makes it unsuitable for alignment of longer reads where indels may occur frequently. The speed of MAQ is also a concern when the alignment is scaled up to the resequencing of hundreds of individuals.</p><p class="abstract"><strong>Results</strong>: We implemented Burrows-Wheeler Alignment tool (BWA), a new read alignment package that is based on backward search with Burrows-Wheeler Transform (BWT), to efficiently align short sequencing reads against a large reference sequence such as the human genome, allowing mismatches and gaps. BWA supports both base space reads, e.g. from Illumina sequencing machines, and color space reads from AB SOLiD machines. Evaluations on both simulated and real data suggest that BWA is approximately 10-20x faster than MAQ, while achieving similar accuracy. In addition, BWA outputs alignment in the new standard SAM (Sequence Alignment/Map) format. Variant calling and other downstream analyses after the alignment can be achieved with the open source SAMtools software package.</p><p class="abstract"><strong>Availability</strong>: http://maq.sourceforge.net.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077192/Z/05/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">14</span>;<span class="pagerange">1754-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19451168" rel="external">19451168</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705234" rel="external">2705234</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp324" rel="external">10.1093/bioinformatics/btp324</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Sequence Alignment/Map format and SAMtools.</h4>
<p class="authors"><span class="author">Li H</span>, <span class="author">Handsaker B</span>, <span class="author">Wysoker A</span>, <span class="author">Fennell T</span>, <span class="author">Ruan J</span>, <span class="author">Homer N</span>, <span class="author">Marth G</span>, <span class="author">Abecasis G</span>, <span class="author">Durbin R</span> and <span class="author">1000 Genome Project Data Processing Subgroup</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK, Broad Institute of MIT and Harvard, Cambridge, MA 02141, USA.</p><p class="abstract"><strong>Summary</strong>: The Sequence Alignment/Map (SAM) format is a generic alignment format for storing read alignments against reference sequences, supporting short and long reads (up to 128 Mbp) produced by different sequencing platforms. It is flexible in style, compact in size, efficient in random access and is the format in which alignments from the 1000 Genomes Project are released. SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments.</p><p class="abstract"><strong>Availability</strong>: http://samtools.sourceforge.net.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG004719</span>, <span class="grant">R01 HG004719-01</span>, <span class="grant">R01 HG004719-02</span>, <span class="grant">R01 HG004719-02S1</span>, <span class="grant">R01 HG004719-03</span>, <span class="grant">R01 HG004719-04</span>, <span class="grant">U54HG002750</span>; Wellcome Trust: <span class="grant">077192/Z/05/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">16</span>;<span class="pagerange">2078-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19505943">19505943</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2723002">2723002</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp352">10.1093/bioinformatics/btp352</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal mobilization and reintegration of Sleeping Beauty and PiggyBac transposons.</h4>
<p class="authors"><span class="author">Liang Q</span>, <span class="author">Kong J</span>, <span class="author">Stalker J</span> and <span class="author">Bradley A</span></p><p class="abstract">The Sleeping Beauty and PiggyBac DNA transposon systems have recently been developed as tools for insertional mutagenesis. We have compared the chromosomal mobilization efficiency and insertion site preference of the two transposons mobilized from the same donor site in mouse embryonic stem (ES) cells under conditions in which there were no selective constraints on the transposons&#39; insertion sites. Compared with Sleeping Beauty, PiggyBac exhibits higher transposition efficiencies, no evidence for local hopping and a significant bias toward reintegration in intragenic regions, which demonstrate its utility for insertional mutagenesis. Although Sleeping Beauty had no detectable genomic bias with respect to insertions in genes or intergenic regions, both Sleeping Beauty and PiggyBac transposons displayed preferential integration into actively transcribed loci.</p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2009</span>;<span class="volume">47</span>;<span class="number">6</span>;<span class="pagerange">404-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19391106">19391106</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvg.20508">10.1002/dvg.20508</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mining mammalian genomes for folding competent proteins using Tat-dependent genetic selection in Escherichia coli.</h4>
<p class="authors"><span class="author">Lim HK</span>, <span class="author">Mansell TJ</span>, <span class="author">Linderman SW</span>, <span class="author">Fisher AC</span>, <span class="author">Dyson MR</span> and <span class="author">DeLisa MP</span></p>
<p class="affiliation">School of Chemical and Biomolecular Engineering, Cornell University, Ithaca, New York 14853, USA.</p><p class="abstract">Recombinant expression of eukaryotic proteins in Escherichia coli is often limited by poor folding and solubility. To address this problem, we employed a recently developed genetic selection for protein folding and solubility based on the bacterial twin-arginine translocation (Tat) pathway to rapidly identify properly folded recombinant proteins or soluble protein domains of mammalian origin. The coding sequences for 29 different mammalian polypeptides were cloned as sandwich fusions between an N-terminal Tat export signal and a C-terminal selectable marker, namely beta-lactamase. Hence, expression of the selectable marker and survival on selective media was linked to Tat export of the target mammalian protein. Since the folding quality control feature of the Tat pathway prevents export of misfolded proteins, only correctly folded fusion proteins reached the periplasm and conferred cell survival. In general, the ability to confer growth was found to relate closely to the solubility profile and molecular weight of the protein, although other features such as number of contiguous hydrophobic amino acids and cysteine content may also be important. These results highlight the capacity of Tat selection to reveal the folding potential of mammalian proteins and protein domains without the need for structural or functional information about the target protein.</p>
<p class="publication"><span class="title">Protein science : a publication of the Protein Society</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">12</span>;<span class="pagerange">2537-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19830686" rel="external">19830686</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2821272" rel="external">2821272</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpro.262" rel="external">10.1002/pro.262</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA methylation-histone modification relationships across the desmin locus in human primary cells.</h4>
<p class="authors"><span class="author">Lindahl Allen M</span>, <span class="author">Koch CM</span>, <span class="author">Clelland GK</span>, <span class="author">Dunham I</span> and <span class="author">Antoniou M</span></p>
<p class="affiliation">Nuclear Biology Group, King&#39;s College London School of Medicine, Department of Medical and Molecular Genetics, 8th Floor Tower Wing, Guy&#39;s Hospital, London SE1 9RT, UK. <a href="mailto:%4D%61%72%69%61%6E%6E%65%5F%4C%69%6E%64%61%68%6C%41%6C%6C%65%6E%40%68%6D%73.%68%61%72%76%61%72%64.%65%64%75">&#77;&#97;&#114;&#105;&#97;&#110;&#110;&#101;&#95;&#76;&#105;&#110;&#100;&#97;&#104;&#108;&#65;&#108;&#108;&#101;&#110;&#64;&#104;&#109;&#115;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: We present here an extensive epigenetic analysis of a 500 kb region, which encompasses the human desmin gene (DES) and its 5&#39; locus control region (LCR), the only muscle-specific transcriptional regulatory element of this type described to date. These data complement and extend Encyclopaedia of DNA Elements (ENCODE) studies on region ENr133. We analysed histone modifications and underlying DNA methylation patterns in physiologically relevant DES expressing (myoblast/myotube) and non-expressing (peripheral blood mononuclear) primary human cells.</p><p class="abstract"><strong>Results</strong>: We found that in expressing myoblast/myotube but not peripheral blood mononuclear cell (PBMC) cultures, histone H4 acetylation displays a broadly distributed enrichment across a gene rich 200 kb region whereas H3 acetylation localizes at the transcriptional start site (TSS) of genes. We show that the DES LCR and TSS of DES are enriched with hyperacetylated domains of acetylated histone H3, with H3 lysine 4 di- and tri-methylation (H3K4me2 and me3) exhibiting a different distribution pattern across this locus. The CpG island that extends into the first intron of DES is methylation-free regardless of the gene&#39;s expression status and in non-expressing PBMCs is marked with histone H3 lysine 27 tri-methylation (H3K27me3).</p><p class="abstract"><strong>Conclusion</strong>: Overall, our results constitute the first study correlating patterns of histone modifications and underlying DNA methylation of a muscle-specific LCR and its associated downstream gene region whilst additionally placing this within a much broader genomic context. Our results clearly show that there are distinct patterns of histone H3 and H4 acetylation and H3 methylation at the DES LCR, promoter and intragenic region. In addition, the presence of H3K27me3 at the DES methylation-free CpG only in non-expressing PBMCs may serve to silence this gene in non-muscle tissues. Generally, our work demonstrates the importance of using multiple, physiologically relevant tissue types that represent different expressing/non-expressing states when investigating epigenetic marks and that underlying DNA methylation status should be correlated with histone modification patterns when studying chromatin structure.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G78/7909</span></p>
<p class="publication"><span class="title">BMC molecular biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19473514" rel="external">19473514</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2695444" rel="external">2695444</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2199-10-51" rel="external">10.1186/1471-2199-10-51</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association scan meta-analysis identifies three Loci influencing adiposity and fat distribution.</h4>
<p class="authors"><span class="author">Lindgren CM</span>, <span class="author">Heid IM</span>, <span class="author">Randall JC</span>, <span class="author">Lamina C</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Qi L</span>, <span class="author">Speliotes EK</span>, <span class="author">Thorleifsson G</span>, <span class="author">Willer CJ</span>, <span class="author">Herrera BM</span>, <span class="author">Jackson AU</span>, <span class="author">Lim N</span>, <span class="author">Scheet P</span>, <span class="author">Soranzo N</span>, <span class="author">Amin N</span>, <span class="author">Aulchenko YS</span>, <span class="author">Chambers JC</span>, <span class="author">Drong A</span>, <span class="author">Luan J</span>, <span class="author">Lyon HN</span>, <span class="author">Rivadeneira F</span>, <span class="author">Sanna S</span>, <span class="author">Timpson NJ</span>, <span class="author">Zillikens MC</span>, <span class="author">Zhao JH</span>, <span class="author">Almgren P</span>, <span class="author">Bandinelli S</span>, <span class="author">Bennett AJ</span>, <span class="author">Bergman RN</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Chanock SJ</span>, <span class="author">Cherkas L</span>, <span class="author">Chines P</span>, <span class="author">Coin L</span>, <span class="author">Cooper C</span>, <span class="author">Crawford G</span>, <span class="author">Doering A</span>, <span class="author">Dominiczak A</span>, <span class="author">Doney AS</span>, <span class="author">Ebrahim S</span>, <span class="author">Elliott P</span>, <span class="author">Erdos MR</span>, <span class="author">Estrada K</span>, <span class="author">Ferrucci L</span>, <span class="author">Fischer G</span>, <span class="author">Forouhi NG</span>, <span class="author">Gieger C</span>, <span class="author">Grallert H</span>, <span class="author">Groves CJ</span>, <span class="author">Grundy S</span>, <span class="author">Guiducci C</span>, <span class="author">Hadley D</span>, <span class="author">Hamsten A</span>, <span class="author">Havulinna AS</span>, <span class="author">Hofman A</span>, <span class="author">Holle R</span>, <span class="author">Holloway JW</span>, <span class="author">Illig T</span>, <span class="author">Isomaa B</span>, <span class="author">Jacobs LC</span>, <span class="author">Jameson K</span>, <span class="author">Jousilahti P</span>, <span class="author">Karpe F</span>, <span class="author">Kuusisto J</span>, <span class="author">Laitinen J</span>, <span class="author">Lathrop GM</span>, <span class="author">Lawlor DA</span>, <span class="author">Mangino M</span>, <span class="author">McArdle WL</span>, <span class="author">Meitinger T</span>, <span class="author">Morken MA</span>, <span class="author">Morris AP</span>, <span class="author">Munroe P</span>, <span class="author">Narisu N</span>, <span class="author">Nordstr&ouml;m A</span>, <span class="author">Nordstr&ouml;m P</span>, <span class="author">Oostra BA</span>, <span class="author">Palmer CN</span>, <span class="author">Payne F</span>, <span class="author">Peden JF</span>, <span class="author">Prokopenko I</span>, <span class="author">Renstr&ouml;m F</span>, <span class="author">Ruokonen A</span>, <span class="author">Salomaa V</span>, <span class="author">Sandhu MS</span>, <span class="author">Scott LJ</span>, <span class="author">Scuteri A</span>, <span class="author">Silander K</span>, <span class="author">Song K</span>, <span class="author">Yuan X</span>, <span class="author">Stringham HM</span>, <span class="author">Swift AJ</span>, <span class="author">Tuomi T</span>, <span class="author">Uda M</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Wallace C</span>, <span class="author">Walters GB</span>, <span class="author">Weedon MN</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Witteman JC</span>, <span class="author">Zhang C</span>, <span class="author">Zhang W</span>, <span class="author">Caulfield MJ</span>, <span class="author">Collins FS</span>, <span class="author">Davey Smith G</span>, <span class="author">Day IN</span>, <span class="author">Franks PW</span>, <span class="author">Hattersley AT</span>, <span class="author">Hu FB</span>, <span class="author">Jarvelin MR</span>, <span class="author">Kong A</span>, <span class="author">Kooner JS</span>, <span class="author">Laakso M</span>, <span class="author">Lakatta E</span>, <span class="author">Mooser V</span>, <span class="author">Morris AD</span>, <span class="author">Peltonen L</span>, <span class="author">Samani NJ</span>, <span class="author">Spector TD</span>, <span class="author">Strachan DP</span>, <span class="author">Tanaka T</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">Wareham NJ</span>, <span class="author">Hugh Watkins</span>, <span class="author">Procardis Consortia</span>, <span class="author">Waterworth DM</span>, <span class="author">Boehnke M</span>, <span class="author">Deloukas P</span>, <span class="author">Groop L</span>, <span class="author">Hunter DJ</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Schlessinger D</span>, <span class="author">Wichmann HE</span>, <span class="author">Frayling TM</span>, <span class="author">Abecasis GR</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Loos RJ</span>, <span class="author">Stefansson K</span>, <span class="author">Mohlke KL</span>, <span class="author">Barroso I</span>, <span class="author">McCarthy MI</span> and <span class="author">Giant Consortium</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, , Oxford, United Kingdom.</p><p class="abstract">To identify genetic loci influencing central obesity and fat distribution, we performed a meta-analysis of 16 genome-wide association studies (GWAS, N = 38,580) informative for adult waist circumference (WC) and waist-hip ratio (WHR). We selected 26 SNPs for follow-up, for which the evidence of association with measures of central adiposity (WC and/or WHR) was strong and disproportionate to that for overall adiposity or height. Follow-up studies in a maximum of 70,689 individuals identified two loci strongly associated with measures of central adiposity; these map near TFAP2B (WC, P = 1.9x10(-11)) and MSRA (WC, P = 8.9x10(-9)). A third locus, near LYPLAL1, was associated with WHR in women only (P = 2.6x10(-8)). The variants near TFAP2B appear to influence central adiposity through an effect on overall obesity/fat-mass, whereas LYPLAL1 displays a strong female-only association with fat distribution. By focusing on anthropometric measures of central obesity and fat distribution, we have identified three loci implicated in the regulation of human adiposity.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; British Heart Foundation; Medical Research Council: <span class="grant">0600705</span>, <span class="grant">G0000649</span>, <span class="grant">G0000934</span>, <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_UP_A620_1014</span>; NHLBI NIH HHS: <span class="grant">HL084729</span>, <span class="grant">HL087679</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK067288</span>, <span class="grant">DK07191</span>, <span class="grant">DK072193</span>, <span class="grant">DK075787</span>, <span class="grant">DK079466</span>, <span class="grant">DK080145</span>, <span class="grant">F32 DK079466</span>, <span class="grant">F32 DK079466-01</span>, <span class="grant">K23 DK080145</span>, <span class="grant">K23 DK080145-01</span>, <span class="grant">R01 DK029867</span>; PHS HHS: <span class="grant">G02651</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">068545/Z/02</span>, <span class="grant">081682</span>, <span class="grant">086596/Z/08/Z</span>, <span class="grant">GR069224</span>, <span class="grant">GR072960</span>, <span class="grant">GR076113</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e1000508</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19557161">19557161</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2695778">2695778</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000508">10.1371/journal.pgen.1000508</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HI: haplotype improver using paired-end short reads.</h4>
<p class="authors"><span class="author">Long Q</span>, <span class="author">MacArthur D</span>, <span class="author">Ning Z</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambs, UK. <a href="mailto:%71%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#113;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Summary</strong>: We present a program to improve haplotype reconstruction by incorporating information from paired-end reads, and demonstrate its utility on simulated data. We find that given a fixed coverage, longer reads (implying fewer of them) are preferable.</p><p class="abstract"><strong>Availability</strong>: The executable and user manual can be freely downloaded from ftp://ftp.sanger.ac.uk/pub/zn1/HI.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">18</span>;<span class="pagerange">2436-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19570807">19570807</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2735667">2735667</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp412">10.1093/bioinformatics/btp412</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adventitious changes in long-range gene expression caused by polymorphic structural variation and promoter competition.</h4>
<p class="authors"><span class="author">Lower KM</span>, <span class="author">Hughes JR</span>, <span class="author">De Gobbi M</span>, <span class="author">Henderson S</span>, <span class="author">Viprakasit V</span>, <span class="author">Fisher C</span>, <span class="author">Goriely A</span>, <span class="author">Ayyub H</span>, <span class="author">Sloane-Stanley J</span>, <span class="author">Vernimmen D</span>, <span class="author">Langford C</span>, <span class="author">Garrick D</span>, <span class="author">Gibbons RJ</span> and <span class="author">Higgs DR</span></p>
<p class="affiliation">Medical Research Council Molecular Haematology Unit, Weatherall Institute of Molecular Medicine, The John Radcliffe Hospital, Headington, Oxford, OX3 9DS, United Kingdom.</p><p class="abstract">It is well established that all of the cis-acting sequences required for fully regulated human alpha-globin expression are contained within a region of approximately 120 kb of conserved synteny. Here, we show that activation of this cluster in erythroid cells dramatically affects expression of apparently unrelated and noncontiguous genes in the 500 kb surrounding this domain, including a gene (NME4) located 300 kb from the alpha-globin cluster. Changes in NME4 expression are mediated by physical cis-interactions between this gene and the alpha-globin regulatory elements. Polymorphic structural variation within the globin cluster, altering the number of alpha-globin genes, affects the pattern of NME4 expression by altering the competition for the shared alpha-globin regulatory elements. These findings challenge the concept that the genome is organized into discrete, insulated regulatory domains. In addition, this work has important implications for our understanding of genome evolution, the interpretation of genome-wide expression, expression-quantitative trait loci, and copy number variant analyses.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2009</span>;<span class="volume">106</span>;<span class="number">51</span>;<span class="pagerange">21771-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19959666" rel="external">19959666</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2799829" rel="external">2799829</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0909331106" rel="external">10.1073/pnas.0909331106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Amplification and overexpression of Hsa-miR-30b, Hsa-miR-30d and KHDRBS3 at 8q24.22-q24.23 in medulloblastoma.</h4>
<p class="authors"><span class="author">Lu Y</span>, <span class="author">Ryan SL</span>, <span class="author">Elliott DJ</span>, <span class="author">Bignell GR</span>, <span class="author">Futreal PA</span>, <span class="author">Ellison DW</span>, <span class="author">Bailey S</span> and <span class="author">Clifford SC</span></p>
<p class="affiliation">Northern Institute for Cancer Research, Newcastle University, Newcastle upon Tyne, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Medulloblastoma is the most common malignant brain tumour of childhood. The identification of critical genes involved in its pathogenesis will be central to advances in our understanding of its molecular basis, and the development of improved therapeutic approaches.</p><p class="abstract">We performed a SNP-array based genome-wide copy number analysis in medulloblastoma cell lines, to identify regions of genomic amplification and homozygous deletion, which may harbour critical disease genes. A series of novel and established medulloblastoma defects were detected (MYC amplification (n = 4), 17q21.31 high-level gain (n = 1); 9p21.1-p21.3 (n = 1) and 6q23.1 (n = 1) homozygous deletion). Most notably, a novel recurrent region of genomic amplification at 8q24.22-q24.23 was identified (n = 2), and selected for further investigation. Additional analysis by interphase fluorescence in situ hybridisation (iFISH), PCR-based mapping and SNP-array revealed this novel amplification at 8q24.22-q24.23 is independent of MYC amplification at 8q24.21, and is unique to medulloblastoma in over 800 cancer cell lines assessed from different tumour types, suggesting it contains key genes specifically involved in medulloblastoma development. Detailed mapping identified a 3Mb common minimal region of amplification harbouring 3 coding genes (ZFAT1, LOC286094, KHDRBS3) and two genes encoding micro-RNAs (hsa-miR-30b, hsa-miR-30d). Of these, only expression of hsa-miR-30b, hsa-miR-30d and KHDRBS3 correlated with copy number status, and all three of these transcripts also displayed evidence of elevated expression in sub-sets of primary medulloblastomas, measured relative to the normal cerebellum.</p><p class="abstract">These data implicate hsa-miR-30b, hsa-miR-30d and KHDRBS3 as putative oncogenic target(s) of a novel recurrent medulloblastoma amplicon at 8q24.22-q24.23. Our findings suggest critical roles for these genes in medulloblastoma development, and further support the contribution of micro-RNA species to medulloblastoma pathogenesis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C8464/A5497</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">7</span>;<span class="pagerange">e6159</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19584924" rel="external">19584924</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2702821" rel="external">2702821</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0006159" rel="external">10.1371/journal.pone.0006159</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biology of Genomes: making sense of sequence.</h4>
<p class="authors"><span class="author">Macarthur DG</span></p>
<p class="affiliation">Human Evolution, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%6D%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#109;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">A report on the Biology of Genomes meeting held at Cold Spring Harbor Laboratory, NY, USA, 5-9 May 2009.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">1</span>;<span class="number">6</span>;<span class="pagerange">61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19566913">19566913</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2703870">2703870</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm61">10.1186/gm61</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study identifies a novel locus on chromosome 18q12.2 influencing white cell telomere length.</h4>
<p class="authors"><span class="author">Mangino M</span>, <span class="author">Richards JB</span>, <span class="author">Soranzo N</span>, <span class="author">Zhai G</span>, <span class="author">Aviv A</span>, <span class="author">Valdes AM</span>, <span class="author">Samani NJ</span>, <span class="author">Deloukas P</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Department of Twin Research &amp; Genetic Epidemiology, King&#39;s College London, London, UK.</p><p class="abstract"><strong>Background</strong>: Telomere length is a predictor for a number of common age related diseases and is a heritable trait.</p><p class="abstract">To identify new loci associated with mean leukocyte telomere length we conducted a genome wide association study of 314,075 single nucleotide polymorphisms (SNPs) and validated the results in a second cohort (n for both cohorts combined = 2790). We identified two novel associated variants (rs2162440, p = 2.6 x 10(-6); and rs7235755, p = 5.5 x 10(-6)) on chromosome 18q12.2 in the same region as the VPS34/PIKC3C gene, which has been directly implicated in the pathway controlling telomere length variation in yeast.</p><p class="abstract"><strong>Conclusion</strong>: These results provide new insights into the pathways regulating telomere homeostasis in humans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">46</span>;<span class="number">7</span>;<span class="pagerange">451-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19359265" rel="external">19359265</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2696823" rel="external">2696823</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2008.064956" rel="external">10.1136/jmg.2008.064956</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">LookSeq: a browser-based viewer for deep sequencing data.</h4>
<p class="authors"><span class="author">Manske HM</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6D%6D%36%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#109;&#54;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Sequencing a genome to great depth can be highly informative about heterogeneity within an individual or a population. Here we address the problem of how to visualize the multiple layers of information contained in deep sequencing data. We propose an interactive AJAX-based web viewer for browsing large data sets of aligned sequence reads. By enabling seamless browsing and fast zooming, the LookSeq program assists the user to assimilate information at different levels of resolution, from an overview of a genomic region to fine details such as heterogeneity within the sample. A specific problem, particularly if the sample is heterogeneous, is how to depict information about structural variation. LookSeq provides a simple graphical representation of paired sequence reads that is more revealing about potential insertions and deletions than are conventional methods.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">11</span>;<span class="pagerange">2125-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19679872">19679872</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2775587">2775587</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.093443.109">10.1101/gr.093443.109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SNP-o-matic.</h4>
<p class="authors"><span class="author">Manske HM</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%6D%6D%36%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#109;&#54;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: High throughput sequencing technologies generate large amounts of short reads. Mapping these to a reference sequence consumes large amounts of processing time and memory, and read mapping errors can lead to noisy or incorrect alignments. SNP-o-matic is a fast, memory-efficient and stringent read mapping tool offering a variety of analytical output functions, with an emphasis on genotyping.</p><p class="abstract"><strong>Availability</strong>: http://snpomatic.sourceforge.net.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">18</span>;<span class="pagerange">2434-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19574284">19574284</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2735664">2735664</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp403">10.1093/bioinformatics/btp403</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High prevalence of four long QT syndrome founder mutations in the Finnish population.</h4>
<p class="authors"><span class="author">Marjamaa A</span>, <span class="author">Salomaa V</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Porthan K</span>, <span class="author">Reunanen A</span>, <span class="author">Karanko H</span>, <span class="author">Jula A</span>, <span class="author">Lahermo P</span>, <span class="author">V&auml;&auml;n&auml;nen H</span>, <span class="author">Toivonen L</span>, <span class="author">Swan H</span>, <span class="author">Viitasalo M</span>, <span class="author">Nieminen MS</span>, <span class="author">Peltonen L</span>, <span class="author">Oikarinen L</span>, <span class="author">Palotie A</span> and <span class="author">Kontula K</span></p>
<p class="affiliation">Research Program in Molecular Medicine, Biomedicum Helsinki, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Aims</strong>: Long QT syndrome (LQTS) is an inherited arrhythmia disorder with an estimated prevalence of 0.01%-0.05%. In Finland, four founder mutations constitute up to 70% of the known genetic spectrum of LQTS. In the present survey, we sought to estimate the actual prevalence of the founder mutations and to determine their effect sizes in the general Finnish population.</p><p class="abstract">We genotyped 6334 subjects aged &gt; or =30 years from a population cohort (Health 2000 study) for the four Finnish founder mutations using Sequenom MALDI-TOF mass spectrometry. The electrocardiogram (ECG) parameters were measured from digital 12-lead ECGs, and QT intervals were adjusted for age, sex, and heart rate using linear regression. A total of 27 individuals carried one of the founder mutations resulting in their collective prevalence estimate of 0.4% (95% CI 0.3%-0.6%). The KCNQ1 G589D mutation (n=8) was associated with a 50 ms (SE 7.0) prolongation of the adjusted QT interval (P=9.0x10(-13)). The KCNH2 R176W variant (n=16) resulted in a 22 ms (SE 4.7) longer adjusted QT interval (P=2.1x10(-6)).</p><p class="abstract"><strong>Conclusion</strong>: In Finland 1 individual out of 250 carries a LQTS founder mutation, which is the highest documented prevalence of LQTS mutations that lead to a marked QT prolongation.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL 080025</span>; Wellcome Trust: <span class="grant">089061</span>, <span class="grant">089062</span></p>
<p class="publication"><span class="title">Annals of medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">3</span>;<span class="pagerange">234-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19160088" rel="external">19160088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2704397" rel="external">2704397</a>; DOI: <a href="http://dx.doi.org/10.1080%2F07853890802668530" rel="external">10.1080/07853890802668530</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Donor-recipient mismatch for common gene deletion polymorphisms in graft-versus-host disease.</h4>
<p class="authors"><span class="author">McCarroll SA</span>, <span class="author">Bradner JE</span>, <span class="author">Turpeinen H</span>, <span class="author">Volin L</span>, <span class="author">Martin PJ</span>, <span class="author">Chilewski SD</span>, <span class="author">Antin JH</span>, <span class="author">Lee SJ</span>, <span class="author">Ruutu T</span>, <span class="author">Storer B</span>, <span class="author">Warren EH</span>, <span class="author">Zhang B</span>, <span class="author">Zhao LP</span>, <span class="author">Ginsburg D</span>, <span class="author">Soiffer RJ</span>, <span class="author">Partanen J</span>, <span class="author">Hansen JA</span>, <span class="author">Ritz J</span>, <span class="author">Palotie A</span> and <span class="author">Altshuler D</span></p>
<p class="affiliation">Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, USA. <a href="mailto:%6D%63%63%61%72%72%6F%6C%6C%40%67%65%6E%65%74%69%63%73.%6D%65%64.%68%61%72%76%61%72%64.%65%64%75">&#109;&#99;&#99;&#97;&#114;&#114;&#111;&#108;&#108;&#64;&#103;&#101;&#110;&#101;&#116;&#105;&#99;&#115;&#46;&#109;&#101;&#100;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Transplantation and pregnancy, in which two diploid genomes reside in one body, can each lead to diseases in which immune cells from one individual target antigens encoded in the other&#39;s genome. One such disease, graft-versus-host disease (GVHD) after hematopoietic stem cell transplantation (HSCT, or bone marrow transplant), is common even after transplants between HLA-identical siblings, indicating that cryptic histocompatibility loci exist outside the HLA locus. The immune system of an individual whose genome is homozygous for a gene deletion could recognize epitopes encoded by that gene as alloantigens. Analyzing common gene deletions in three HSCT cohorts (1,345 HLA-identical sibling donor-recipient pairs), we found that risk of acute GVHD was greater (odds ratio (OR) = 2.5; 95% confidence interval (CI) 1.4-4.6) when donor and recipient were mismatched for homozygous deletion of UGT2B17, a gene expressed in GVHD-affected tissues and giving rise to multiple histocompatibility antigens. Human genome structural variation merits investigation as a potential mechanism in diseases of alloimmunity.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA18029</span>, <span class="grant">P01 CA018029</span>, <span class="grant">P01 CA018029-270048</span>, <span class="grant">P01 CA018029-349016</span>; NHLBI NIH HHS: <span class="grant">HL087690</span>, <span class="grant">P01 HL070149</span>, <span class="grant">P01 HL070149-05</span>, <span class="grant">R01 HL087690</span>, <span class="grant">R01 HL087690-03</span>; NIAID NIH HHS: <span class="grant">AI29530</span>, <span class="grant">AI33484</span>, <span class="grant">P01 AI029530</span>, <span class="grant">P01 AI029530-130007</span>, <span class="grant">P01 AI033484</span>, <span class="grant">P01 AI033484-13</span>; PHS HHS: <span class="grant">HA070149</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">12</span>;<span class="pagerange">1341-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19935662">19935662</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2804745">2804745</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.490">10.1038/ng.490</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microduplications of 16p11.2 are associated with schizophrenia.</h4>
<p class="authors"><span class="author">McCarthy SE</span>, <span class="author">Makarov V</span>, <span class="author">Kirov G</span>, <span class="author">Addington AM</span>, <span class="author">McClellan J</span>, <span class="author">Yoon S</span>, <span class="author">Perkins DO</span>, <span class="author">Dickel DE</span>, <span class="author">Kusenda M</span>, <span class="author">Krastoshevsky O</span>, <span class="author">Krause V</span>, <span class="author">Kumar RA</span>, <span class="author">Grozeva D</span>, <span class="author">Malhotra D</span>, <span class="author">Walsh T</span>, <span class="author">Zackai EH</span>, <span class="author">Kaplan P</span>, <span class="author">Ganesh J</span>, <span class="author">Krantz ID</span>, <span class="author">Spinner NB</span>, <span class="author">Roccanova P</span>, <span class="author">Bhandari A</span>, <span class="author">Pavon K</span>, <span class="author">Lakshmi B</span>, <span class="author">Leotta A</span>, <span class="author">Kendall J</span>, <span class="author">Lee YH</span>, <span class="author">Vacic V</span>, <span class="author">Gary S</span>, <span class="author">Iakoucheva LM</span>, <span class="author">Crow TJ</span>, <span class="author">Christian SL</span>, <span class="author">Lieberman JA</span>, <span class="author">Stroup TS</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Puura K</span>, <span class="author">Haldeman-Englert C</span>, <span class="author">Pearl J</span>, <span class="author">Goodell M</span>, <span class="author">Willour VL</span>, <span class="author">Derosse P</span>, <span class="author">Steele J</span>, <span class="author">Kassem L</span>, <span class="author">Wolff J</span>, <span class="author">Chitkara N</span>, <span class="author">McMahon FJ</span>, <span class="author">Malhotra AK</span>, <span class="author">Potash JB</span>, <span class="author">Schulze TG</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Cichon S</span>, <span class="author">Rietschel M</span>, <span class="author">Leibenluft E</span>, <span class="author">Kustanovich V</span>, <span class="author">Lajonchere CM</span>, <span class="author">Sutcliffe JS</span>, <span class="author">Skuse D</span>, <span class="author">Gill M</span>, <span class="author">Gallagher L</span>, <span class="author">Mendell NR</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Craddock N</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Shaikh TH</span>, <span class="author">Susser E</span>, <span class="author">Delisi LE</span>, <span class="author">Sullivan PF</span>, <span class="author">Deutsch CK</span>, <span class="author">Rapoport J</span>, <span class="author">Levy DL</span>, <span class="author">King MC</span> and <span class="author">Sebat J</span></p><p class="abstract">Recurrent microdeletions and microduplications of a 600-kb genomic region of chromosome 16p11.2 have been implicated in childhood-onset developmental disorders. We report the association of 16p11.2 microduplications with schizophrenia in two large cohorts. The microduplication was detected in 12/1,906 (0.63%) cases and 1/3,971 (0.03%) controls (P = 1.2 x 10(-5), OR = 25.8) from the initial cohort, and in 9/2,645 (0.34%) cases and 1/2,420 (0.04%) controls (P = 0.022, OR = 8.3) of the replication cohort. The 16p11.2 microduplication was associated with a 14.5-fold increased risk of schizophrenia (95% CI (3.3, 62)) in the combined sample. A meta-analysis of datasets for multiple psychiatric disorders showed a significant association of the microduplication with schizophrenia (P = 4.8 x 10(-7)), bipolar disorder (P = 0.017) and autism (P = 1.9 x 10(-7)). In contrast, the reciprocal microdeletion was associated only with autism and developmental disorders (P = 2.3 x 10(-13)). Head circumference was larger in patients with the microdeletion than in patients with the microduplication (P = 0.0007).</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800509</span>; NCRR NIH HHS: <span class="grant">RR000037</span>; NICHD NIH HHS: <span class="grant">HD04147</span>; NIDCR NIH HHS: <span class="grant">DE016442</span>; NIGMS NIH HHS: <span class="grant">GM081519</span>; NIMH NIH HHS: <span class="grant">1U24MH081810</span>, <span class="grant">K99 MH086756</span>, <span class="grant">K99 MH086756-01</span>, <span class="grant">K99 MH086756-02</span>, <span class="grant">MH061009</span>, <span class="grant">MH071523</span>, <span class="grant">MH074027</span>, <span class="grant">MH076431</span>, <span class="grant">MH077139</span>, <span class="grant">MH081810</span>, <span class="grant">MH083989</span>, <span class="grant">MH31340</span>, <span class="grant">MH44245</span>, <span class="grant">N01 MH90001</span>, <span class="grant">R00 MH086756</span>, <span class="grant">R00 MH086756-03</span>, <span class="grant">R01 MH091350</span>, <span class="grant">ZIA MH002581-19</span>; PHS HHS: <span class="grant">HF004222</span>; Wellcome Trust: <span class="grant">076113</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">11</span>;<span class="pagerange">1223-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19855392">19855392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2951180">2951180</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.474">10.1038/ng.474</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Geographical structure and differential natural selection among North European populations.</h4>
<p class="authors"><span class="author">McEvoy BP</span>, <span class="author">Montgomery GW</span>, <span class="author">McRae AF</span>, <span class="author">Ripatti S</span>, <span class="author">Perola M</span>, <span class="author">Spector TD</span>, <span class="author">Cherkas L</span>, <span class="author">Ahmadi KR</span>, <span class="author">Boomsma D</span>, <span class="author">Willemsen G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Pedersen NL</span>, <span class="author">Magnusson PK</span>, <span class="author">Kyvik KO</span>, <span class="author">Christensen K</span>, <span class="author">Kaprio J</span>, <span class="author">Heikkil&auml; K</span>, <span class="author">Palotie A</span>, <span class="author">Widen E</span>, <span class="author">Muilu J</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Liljedahl U</span>, <span class="author">Hardiman O</span>, <span class="author">Cronin S</span>, <span class="author">Peltonen L</span>, <span class="author">Martin NG</span> and <span class="author">Visscher PM</span></p>
<p class="affiliation">Queensland Institute of Medical Research, Brisbane, Queensland, Australia. <a href="mailto:%62%72%69%61%6E.%6D%63%65%76%6F%79%40%71%69%6D%72.%65%64%75.%61%75">&#98;&#114;&#105;&#97;&#110;&#46;&#109;&#99;&#101;&#118;&#111;&#121;&#64;&#113;&#105;&#109;&#114;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Population structure can provide novel insight into the human past, and recognizing and correcting for such stratification is a practical concern in gene mapping by many association methodologies. We investigate these patterns, primarily through principal component (PC) analysis of whole genome SNP polymorphism, in 2099 individuals from populations of Northern European origin (Ireland, United Kingdom, Netherlands, Denmark, Sweden, Finland, Australia, and HapMap European-American). The major trends (PC1 and PC2) demonstrate an ability to detect geographic substructure, even over a small area like the British Isles, and this information can then be applied to finely dissect the ancestry of the European-Australian and European-American samples. They simultaneously point to the importance of considering population stratification in what might be considered a small homogeneous region. There is evidence from F(ST)-based analysis of genic and nongenic SNPs that differential positive selection has operated across these populations despite their short divergence time and relatively similar geographic and environmental range. The pressure appears to have been focused on genes involved in immunity, perhaps reflecting response to infectious disease epidemic. Such an event may explain a striking selective sweep centered on the rs2508049-G allele, close to the HLA-G gene on chromosome 6. Evidence of the sweep extends over a 8-Mb/3.5-cM region. Overall, the results illustrate the power of dense genotype and sample data to explore regional population variation, the events that have crafted it, and their implications in both explaining disease prevalence and mapping these genes by association.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Department of Health; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">5</span>;<span class="pagerange">804-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19265028" rel="external">19265028</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2675969" rel="external">2675969</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.083394.108" rel="external">10.1101/gr.083394.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the seed region of human miR-96 are responsible for nonsyndromic progressive hearing loss.</h4>
<p class="authors"><span class="author">Menc&iacute;a A</span>, <span class="author">Modamio-H&oslash;ybj&oslash;r S</span>, <span class="author">Redshaw N</span>, <span class="author">Mor&iacute;n M</span>, <span class="author">Mayo-Merino F</span>, <span class="author">Olavarrieta L</span>, <span class="author">Aguirre LA</span>, <span class="author">del Castillo I</span>, <span class="author">Steel KP</span>, <span class="author">Dalmay T</span>, <span class="author">Moreno F</span> and <span class="author">Moreno-Pelayo MA</span></p>
<p class="affiliation">Unidad de Gen&eacute;tica Molecular, Hospital Ram&oacute;n y Cajal, Madrid, Spain.</p><p class="abstract">MicroRNAs (miRNAs) bind to complementary sites in their target mRNAs to mediate post-transcriptional repression, with the specificity of target recognition being crucially dependent on the miRNA seed region. Impaired miRNA target binding resulting from SNPs within mRNA target sites has been shown to lead to pathologies associated with dysregulated gene expression. However, no pathogenic mutations within the mature sequence of a miRNA have been reported so far. Here we show that point mutations in the seed region of miR-96, a miRNA expressed in hair cells of the inner ear, result in autosomal dominant, progressive hearing loss. This is the first study implicating a miRNA in a mendelian disorder. The identified mutations have a strong impact on miR-96 biogenesis and result in a significant reduction of mRNA targeting. We propose that these mutations alter the regulatory role of miR-96 in maintaining gene expression profiles in hair cells required for their normal function.</p>
<p class="grants">Funded by: Action on Hearing Loss: <span class="grant">G41</span>; Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">5</span>;<span class="pagerange">609-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19363479">19363479</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.355">10.1038/ng.355</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic structure of nomadic Bedouin from Kuwait.</h4>
<p class="authors"><span class="author">Mohammad T</span>, <span class="author">Xue Y</span>, <span class="author">Evison M</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Division of Genomic Medicine, University of Sheffield, Sheffield, UK.</p><p class="abstract">Bedouin are traditionally nomadic inhabitants of the Persian Gulf who claim descent from two male lineages: Adnani and Qahtani. We have investigated whether or not this tradition is reflected in the current genetic structure of a sample of 153 Bedouin males from six Kuwaiti tribes, including three tribes from each traditional lineage. Volunteers were genotyped using a panel of autosomal and Y-STRs, and Y-SNPs. The samples clustered with their geographical neighbours in both the autosomal and Y-chromosomal analyses, and showed strong evidence of genetic isolation and drift. Although there was no evidence of segregation into the two male lineages, other aspects of genetic structure were in accord with tradition.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Heredity</span>&nbsp;<span class="year">2009</span>;<span class="volume">103</span>;<span class="number">5</span>;<span class="pagerange">425-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19639002">19639002</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2869035">2869035</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fhdy.2009.72">10.1038/hdy.2009.72</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Current computational methods for prioritizing candidate regulatory polymorphisms.</h4>
<p class="authors"><span class="author">Montgomery S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Discovery of DNA sequence variants responsible for human phenotypic variation is key to advances in molecular diagnostics and medicines. Historically, variants that alter the protein-coding sequence of genes have been targeted when attempting to identify a trait&#39;s etiology; this is done because the rules governing these regions are generally well-understood and candidate variants can be easily selected. However, the effects of variants on gene regulation are increasingly regarded as being as important as protein-coding variation in uncovering the nature of phenotypic variation. I discuss resources and methodology that have recently been developed to computationally prioritize variants that may alter gene expression.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">569</span>;<span class="pagerange">89-114</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19623487" rel="external">19623487</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-59745-524-4_5" rel="external">10.1007/978-1-59745-524-4_5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovery of mating in the major African livestock pathogen Trypanosoma congolense.</h4>
<p class="authors"><span class="author">Morrison LJ</span>, <span class="author">Tweedie A</span>, <span class="author">Black A</span>, <span class="author">Pinchbeck GL</span>, <span class="author">Christley RM</span>, <span class="author">Schoenefeld A</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">MacLeod A</span>, <span class="author">Turner CM</span> and <span class="author">Tait A</span></p>
<p class="affiliation">Wellcome Centre for Molecular Parasitology, University of Glasgow, Glasgow Biomedical Research Centre, Glasgow, United Kingdom. <a href="mailto:%6C%6D%37%38%79%40%75%64%63%66.%67%6C%61.%61%63.%75%6B">&#108;&#109;&#55;&#56;&#121;&#64;&#117;&#100;&#99;&#102;&#46;&#103;&#108;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The protozoan parasite, Trypanosoma congolense, is one of the most economically important pathogens of livestock in Africa and, through its impact on cattle health and productivity, has a significant effect on human health and well being. Despite the importance of this parasite our knowledge of some of the fundamental biological processes is limited. For example, it is unknown whether mating takes place. In this paper we have taken a population genetics based approach to address this question. The availability of genome sequence of the parasite allowed us to identify polymorphic microsatellite markers, which were used to genotype T. congolense isolates from livestock in a discrete geographical area of The Gambia. The data showed a high level of diversity with a large number of distinct genotypes, but a deficit in heterozygotes. Further analysis identified cryptic genetic subdivision into four sub-populations. In one of these, parasite genotypic diversity could only be explained by the occurrence of frequent mating in T. congolense. These data are completely inconsistent with previous suggestions that the parasite expands asexually in the absence of mating. The discovery of mating in this species of trypanosome has significant consequences for the spread of critical traits, such as drug resistance, as well as for fundamental aspects of the biology and epidemiology of this neglected but economically important pathogen.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">074732</span>, <span class="grant">079703</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">5</span>;<span class="pagerange">e5564</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19440370" rel="external">19440370</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2679202" rel="external">2679202</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0005564" rel="external">10.1371/journal.pone.0005564</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inferring selection on amino acid preference in protein domains.</h4>
<p class="authors"><span class="author">Moses AM</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%61%6C%61%6E.%6D%6F%73%65%73%40%75%74%6F%72%6F%6E%74%6F.%63%61">&#97;&#108;&#97;&#110;&#46;&#109;&#111;&#115;&#101;&#115;&#64;&#117;&#116;&#111;&#114;&#111;&#110;&#116;&#111;&#46;&#99;&#97;</a></p><p class="abstract">Models that explicitly account for the effect of selection on new mutations have been proposed to account for &quot;codon bias&quot; or the excess of &quot;preferred&quot; codons that results from selection for translational efficiency and/or accuracy. In principle, such models can be applied to any mutation that results in a preferred allele, but in most cases, the fitness effect of a specific mutation cannot be predicted. Here we show that it is possible to assign preferred and unpreferred states to amino acid changing mutations that occur in protein domains. We propose that mutations that lead to more common amino acids (at a given position in a domain) can be considered &quot;preferred alleles&quot; just as are synonymous mutations leading to codons for more abundant tRNAs. We use genome-scale polymorphism data to show that alleles for preferred amino acids in protein domains occur at higher frequencies in the population, as has been shown for preferred codons. We show that this effect is quantitative, such that there is a correlation between the shift in frequency of preferred alleles and the predicted fitness effect. As expected, we also observe a reduction in the numbers of polymorphisms and substitutions at more important positions in domains, consistent with stronger selection at those positions. We examine the derived allele frequency distribution and polymorphism to divergence ratios of preferred and unpreferred differences and find evidence for both negative and positive selections acting to maintain protein domains in the human population. Finally, we analyze a model for selection on amino acid preferences in protein domains and find that it is consistent with the quantitative effects that we observe.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077192</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2009</span>;<span class="volume">26</span>;<span class="number">3</span>;<span class="pagerange">527-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19095755">19095755</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716081">2716081</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsn286">10.1093/molbev/msn286</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene-wide analyses of genome-wide association data sets: evidence for multiple common risk alleles for schizophrenia and bipolar disorder and for overlap in genetic risk.</h4>
<p class="authors"><span class="author">Moskvina V</span>, <span class="author">Craddock N</span>, <span class="author">Holmans P</span>, <span class="author">Nikolov I</span>, <span class="author">Pahwa JS</span>, <span class="author">Green E</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Owen MJ</span> and <span class="author">O&#39;Donovan MC</span></p>
<p class="affiliation">Department of Psychological Medicine, School of Medicine, Cardiff University, Cardiff, UK.</p><p class="abstract">Genome-wide association (GWAS) analyses have identified susceptibility loci for many diseases, but most risk for any complex disorder remains unattributed. There is therefore scope for complementary approaches to these data sets. Gene-wide approaches potentially offer additional insights. They might identify association to genes through multiple signals. Also, by providing support for genes rather than single nucleotide polymorphisms (SNPs), they offer an additional opportunity to compare the results across data sets. We have undertaken gene-wide analysis of two GWAS data sets: schizophrenia and bipolar disorder. We performed two forms of analysis, one based on the smallest P-value per gene, the other on a truncated product of P method. For each data set and at a range of statistical thresholds, we observed significantly more SNPs within genes (P(min) for excess&lt;0.001) showing evidence for association than expected whereas this was not true for extragenic SNPs (P(min) for excess&gt;0.1). At a range of thresholds of significance, we also observed substantially more associated genes than expected (P(min) for excess in schizophrenia=1.8 x 10(-8), in bipolar=2.4 x 10(-6)). Moreover, an excess of genes showed evidence for association across disorders. Among those genes surpassing thresholds highly enriched for true association, we observed evidence for association to genes reported in other GWAS data sets (CACNA1C) or to closely related family members of those genes including CSF2RB, CACNA1B and DGKI. Our analyses show that association signals are enriched in and around genes, large numbers of genes contribute to both disorders and gene-wide analyses offer useful complementary approaches to more standard methods.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0800509</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">079643</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2009</span>;<span class="volume">14</span>;<span class="number">3</span>;<span class="pagerange">252-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19065143">19065143</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3970088">3970088</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2008.133">10.1038/mp.2008.133</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel genes in cell cycle control and lipid metabolism with dynamically regulated binding sites for sterol regulatory element-binding protein 1 and RNA polymerase II in HepG2 cells detected by chromatin immunoprecipitation with microarray detection.</h4>
<p class="authors"><span class="author">Motallebipour M</span>, <span class="author">Enroth S</span>, <span class="author">Punga T</span>, <span class="author">Ameur A</span>, <span class="author">Koch C</span>, <span class="author">Dunham I</span>, <span class="author">Komorowski J</span>, <span class="author">Ericsson J</span> and <span class="author">Wadelius C</span></p>
<p class="affiliation">Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, Uppsala, Sweden.</p><p class="abstract">Sterol regulatory element-binding proteins 1 and 2 (SREBP-1 and SREBP-2) are important regulators of genes involved in cholesterol and fatty acid metabolism, but have also been implicated in the regulation of the cell cycle and have been associated with the pathogenesis of type 2 diabetes, atherosclerosis and obesity, among others. In this study, we aimed to characterize the binding sites of SREBP-1 and RNA polymerase II through chromatin immunoprecipitation and microarray analysis in 1% of the human genome, as defined by the Encyclopaedia of DNA Elements consortium, in a hepatocellular carcinoma cell line (HepG2). Our data identified novel binding sites for SREBP-1 in genes directly or indirectly involved in cholesterol metabolism, e.g. apolipoprotein C-III (APOC3). The most interesting biological findings were the binding sites for SREBP-1 in genes for host cell factor C1 (HCFC1), involved in cell cycle regulation, and for filamin A (FLNA). For RNA polymerase II, we found binding sites at classical promoters, but also in intergenic and intragenic regions. Furthermore, we found evidence of sterol-regulated binding of SREBP-1 and RNA polymerase II to HCFC1 and FLNA. From the results of this work, we infer that SREBP-1 may be involved in processes other than lipid metabolism.</p>
<p class="publication"><span class="title">The FEBS journal</span>&nbsp;<span class="year">2009</span>;<span class="volume">276</span>;<span class="number">7</span>;<span class="pagerange">1878-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19292868">19292868</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1742-4658.2009.06914.x">10.1111/j.1742-4658.2009.06914.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Abnormal behavior in a chromosome-engineered mouse model for human 15q11-13 duplication seen in autism.</h4>
<p class="authors"><span class="author">Nakatani J</span>, <span class="author">Tamada K</span>, <span class="author">Hatanaka F</span>, <span class="author">Ise S</span>, <span class="author">Ohta H</span>, <span class="author">Inoue K</span>, <span class="author">Tomonaga S</span>, <span class="author">Watanabe Y</span>, <span class="author">Chung YJ</span>, <span class="author">Banerjee R</span>, <span class="author">Iwamoto K</span>, <span class="author">Kato T</span>, <span class="author">Okazawa M</span>, <span class="author">Yamauchi K</span>, <span class="author">Tanda K</span>, <span class="author">Takao K</span>, <span class="author">Miyakawa T</span>, <span class="author">Bradley A</span> and <span class="author">Takumi T</span></p>
<p class="affiliation">Osaka Bioscience Institute, Suita, Osaka 565-0874, Japan.</p><p class="abstract">Substantial evidence suggests that chromosomal abnormalities contribute to the risk of autism. The duplication of human chromosome 15q11-13 is known to be the most frequent cytogenetic abnormality in autism. We have modeled this genetic change in mice by using chromosome engineering to generate a 6.3 Mb duplication of the conserved linkage group on mouse chromosome 7. Mice with a paternal duplication display poor social interaction, behavioral inflexibility, abnormal ultrasonic vocalizations, and correlates of anxiety. An increased MBII52 snoRNA within the duplicated region, affecting the serotonin 2c receptor (5-HT2cR), correlates with altered intracellular Ca(2+) responses elicited by a 5-HT2cR agonist in neurons of mice with a paternal duplication. This chromosome-engineered mouse model for autism seems to replicate various aspects of human autistic phenotypes and validates the relevance of the human chromosome abnormality. This model will facilitate forward genetics of developmental brain disorders and serve as an invaluable tool for therapeutic development.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2009</span>;<span class="volume">137</span>;<span class="number">7</span>;<span class="pagerange">1235-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19563756">19563756</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3710970">3710970</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2009.04.024">10.1016/j.cell.2009.04.024</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic structure of Europeans: a view from the North-East.</h4>
<p class="authors"><span class="author">Nelis M</span>, <span class="author">Esko T</span>, <span class="author">M&auml;gi R</span>, <span class="author">Zimprich F</span>, <span class="author">Zimprich A</span>, <span class="author">Toncheva D</span>, <span class="author">Karachanak S</span>, <span class="author">Pisk&aacute;ckov&aacute; T</span>, <span class="author">Balasc&aacute;k I</span>, <span class="author">Peltonen L</span>, <span class="author">Jakkula E</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Lathrop M</span>, <span class="author">Heath S</span>, <span class="author">Galan P</span>, <span class="author">Schreiber S</span>, <span class="author">Meitinger T</span>, <span class="author">Pfeufer A</span>, <span class="author">Wichmann HE</span>, <span class="author">Melegh B</span>, <span class="author">Polg&aacute;r N</span>, <span class="author">Toniolo D</span>, <span class="author">Gasparini P</span>, <span class="author">D&#39;Adamo P</span>, <span class="author">Klovins J</span>, <span class="author">Nikitina-Zake L</span>, <span class="author">Kucinskas V</span>, <span class="author">Kasnauskiene J</span>, <span class="author">Lubinski J</span>, <span class="author">Debniak T</span>, <span class="author">Limborska S</span>, <span class="author">Khrunin A</span>, <span class="author">Estivill X</span>, <span class="author">Rabionet R</span>, <span class="author">Marsal S</span>, <span class="author">Juli&agrave; A</span>, <span class="author">Antonarakis SE</span>, <span class="author">Deutsch S</span>, <span class="author">Borel C</span>, <span class="author">Attar H</span>, <span class="author">Gagnebin M</span>, <span class="author">Macek M</span>, <span class="author">Krawczak M</span>, <span class="author">Remm M</span> and <span class="author">Metspalu A</span></p>
<p class="affiliation">Institute of Molecular and Cell Biology, University of Tartu, Tartu, Estonia.</p><p class="abstract">Using principal component (PC) analysis, we studied the genetic constitution of 3,112 individuals from Europe as portrayed by more than 270,000 single nucleotide polymorphisms (SNPs) genotyped with the Illumina Infinium platform. In cohorts where the sample size was &gt;100, one hundred randomly chosen samples were used for analysis to minimize the sample size effect, resulting in a total of 1,564 samples. This analysis revealed that the genetic structure of the European population correlates closely with geography. The first two PCs highlight the genetic diversity corresponding to the northwest to southeast gradient and position the populations according to their approximate geographic origin. The resulting genetic map forms a triangular structure with a) Finland, b) the Baltic region, Poland and Western Russia, and c) Italy as its vertexes, and with d) Central- and Western Europe in its centre. Inter- and intra- population genetic differences were quantified by the inflation factor lambda (lambda) (ranging from 1.00 to 4.21), fixation index (F(st)) (ranging from 0.000 to 0.023), and by the number of markers exhibiting significant allele frequency differences in pair-wise population comparisons. The estimated lambda was used to assess the real diminishing impact to association statistics when two distinct populations are merged directly in an analysis. When the PC analysis was confined to the 1,019 Estonian individuals (0.1% of the Estonian population), a fine structure emerged that correlated with the geography of individual counties. With at least two cohorts available from several countries, genetic substructures were investigated in Czech, Finnish, German, Estonian and Italian populations. Together with previously published data, our results allow the creation of a comprehensive European genetic map that will greatly facilitate inter-population genetic studies including genome wide association studies (GWAS).</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">5</span>;<span class="pagerange">e5472</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19424496" rel="external">19424496</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2675054" rel="external">2675054</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0005472" rel="external">10.1371/journal.pone.0005472</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies eight loci associated with blood pressure.</h4>
<p class="authors"><span class="author">Newton-Cheh C</span>, <span class="author">Johnson T</span>, <span class="author">Gateva V</span>, <span class="author">Tobin MD</span>, <span class="author">Bochud M</span>, <span class="author">Coin L</span>, <span class="author">Najjar SS</span>, <span class="author">Zhao JH</span>, <span class="author">Heath SC</span>, <span class="author">Eyheramendy S</span>, <span class="author">Papadakis K</span>, <span class="author">Voight BF</span>, <span class="author">Scott LJ</span>, <span class="author">Zhang F</span>, <span class="author">Farrall M</span>, <span class="author">Tanaka T</span>, <span class="author">Wallace C</span>, <span class="author">Chambers JC</span>, <span class="author">Khaw KT</span>, <span class="author">Nilsson P</span>, <span class="author">van der Harst P</span>, <span class="author">Polidoro S</span>, <span class="author">Grobbee DE</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Bots ML</span>, <span class="author">Wain LV</span>, <span class="author">Elliott KS</span>, <span class="author">Teumer A</span>, <span class="author">Luan J</span>, <span class="author">Lucas G</span>, <span class="author">Kuusisto J</span>, <span class="author">Burton PR</span>, <span class="author">Hadley D</span>, <span class="author">McArdle WL</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Brown M</span>, <span class="author">Dominiczak A</span>, <span class="author">Newhouse SJ</span>, <span class="author">Samani NJ</span>, <span class="author">Webster J</span>, <span class="author">Zeggini E</span>, <span class="author">Beckmann JS</span>, <span class="author">Bergmann S</span>, <span class="author">Lim N</span>, <span class="author">Song K</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Waterworth DM</span>, <span class="author">Yuan X</span>, <span class="author">Groop L</span>, <span class="author">Orho-Melander M</span>, <span class="author">Allione A</span>, <span class="author">Di Gregorio A</span>, <span class="author">Guarrera S</span>, <span class="author">Panico S</span>, <span class="author">Ricceri F</span>, <span class="author">Romanazzi V</span>, <span class="author">Sacerdote C</span>, <span class="author">Vineis P</span>, <span class="author">Barroso I</span>, <span class="author">Sandhu MS</span>, <span class="author">Luben RN</span>, <span class="author">Crawford GJ</span>, <span class="author">Jousilahti P</span>, <span class="author">Perola M</span>, <span class="author">Boehnke M</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Collins FS</span>, <span class="author">Jackson AU</span>, <span class="author">Mohlke KL</span>, <span class="author">Stringham HM</span>, <span class="author">Valle TT</span>, <span class="author">Willer CJ</span>, <span class="author">Bergman RN</span>, <span class="author">Morken MA</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Gieger C</span>, <span class="author">Illig T</span>, <span class="author">Meitinger T</span>, <span class="author">Org E</span>, <span class="author">Pfeufer A</span>, <span class="author">Wichmann HE</span>, <span class="author">Kathiresan S</span>, <span class="author">Marrugat J</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Schwartz SM</span>, <span class="author">Siscovick DS</span>, <span class="author">Subirana I</span>, <span class="author">Freimer NB</span>, <span class="author">Hartikainen AL</span>, <span class="author">McCarthy MI</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Peltonen L</span>, <span class="author">Pouta A</span>, <span class="author">de Jong PE</span>, <span class="author">Snieder H</span>, <span class="author">van Gilst WH</span>, <span class="author">Clarke R</span>, <span class="author">Goel A</span>, <span class="author">Hamsten A</span>, <span class="author">Peden JF</span>, <span class="author">Seedorf U</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Tognoni G</span>, <span class="author">Lakatta EG</span>, <span class="author">Sanna S</span>, <span class="author">Scheet P</span>, <span class="author">Schlessinger D</span>, <span class="author">Scuteri A</span>, <span class="author">D&ouml;rr M</span>, <span class="author">Ernst F</span>, <span class="author">Felix SB</span>, <span class="author">Homuth G</span>, <span class="author">Lorbeer R</span>, <span class="author">Reffelmann T</span>, <span class="author">Rettig R</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Galan P</span>, <span class="author">Gut IG</span>, <span class="author">Hercberg S</span>, <span class="author">Lathrop GM</span>, <span class="author">Zelenika D</span>, <span class="author">Deloukas P</span>, <span class="author">Soranzo N</span>, <span class="author">Williams FM</span>, <span class="author">Zhai G</span>, <span class="author">Salomaa V</span>, <span class="author">Laakso M</span>, <span class="author">Elosua R</span>, <span class="author">Forouhi NG</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Uiterwaal CS</span>, <span class="author">van der Schouw YT</span>, <span class="author">Numans ME</span>, <span class="author">Matullo G</span>, <span class="author">Navis G</span>, <span class="author">Berglund G</span>, <span class="author">Bingham SA</span>, <span class="author">Kooner JS</span>, <span class="author">Connell JM</span>, <span class="author">Bandinelli S</span>, <span class="author">Ferrucci L</span>, <span class="author">Watkins H</span>, <span class="author">Spector TD</span>, <span class="author">Tuomilehto J</span>, <span class="author">Altshuler D</span>, <span class="author">Strachan DP</span>, <span class="author">Laan M</span>, <span class="author">Meneton P</span>, <span class="author">Wareham NJ</span>, <span class="author">Uda M</span>, <span class="author">Jarvelin MR</span>, <span class="author">Mooser V</span>, <span class="author">Melander O</span>, <span class="author">Loos RJ</span>, <span class="author">Elliott P</span>, <span class="author">Abecasis GR</span>, <span class="author">Caulfield M</span> and <span class="author">Munroe PB</span></p>
<p class="affiliation">Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA. <a href="mailto:%63%6E%65%77%74%6F%6E%63%68%65%68%40%63%68%67%72.%6D%67%68.%68%61%72%76%61%72%64.%65%64%75">&#99;&#110;&#101;&#119;&#116;&#111;&#110;&#99;&#104;&#101;&#104;&#64;&#99;&#104;&#103;&#114;&#46;&#109;&#103;&#104;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Elevated blood pressure is a common, heritable cause of cardiovascular disease worldwide. To date, identification of common genetic variants influencing blood pressure has proven challenging. We tested 2.5 million genotyped and imputed SNPs for association with systolic and diastolic blood pressure in 34,433 subjects of European ancestry from the Global BPgen consortium and followed up findings with direct genotyping (N &le; 71,225 European ancestry, N &le; 12,889 Indian Asian ancestry) and in silico comparison (CHARGE consortium, N = 29,136). We identified association between systolic or diastolic blood pressure and common variants in eight regions near the CYP17A1 (P = 7 &times; 10(-24)), CYP1A2 (P = 1 &times; 10(-23)), FGF5 (P = 1 &times; 10(-21)), SH2B3 (P = 3 &times; 10(-18)), MTHFR (P = 2 &times; 10(-13)), c10orf107 (P = 1 &times; 10(-9)), ZNF652 (P = 5 &times; 10(-9)) and PLCD3 (P = 1 &times; 10(-8)) genes. All variants associated with continuous blood pressure were associated with dichotomous hypertension. These associations between common variants and blood pressure and hypertension offer mechanistic insights into the regulation of blood pressure and may point to novel targets for interventions to prevent cardiovascular disease.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/05/061/19501</span>, <span class="grant">PG02/128</span>, <span class="grant">SP/04/002</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">85374</span>, <span class="grant">G0000934</span>, <span class="grant">G0400874</span>, <span class="grant">G0401527</span>, <span class="grant">G0501942</span>, <span class="grant">G0600329</span>, <span class="grant">G0701863</span>, <span class="grant">G0800759</span>, <span class="grant">G0801056</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U137686857</span>; NCRR NIH HHS: <span class="grant">U54RR020278</span>; NHGRI NIH HHS: <span class="grant">1Z01HG000024</span>; NHLBI NIH HHS: <span class="grant">K23 HL080025</span>, <span class="grant">K23 HL080025-04</span>, <span class="grant">K23HL083102</span>, <span class="grant">K23HL80025</span>, <span class="grant">R01 HL056931</span>, <span class="grant">R01 HL056931-02</span>, <span class="grant">R01 HL056931-03</span>, <span class="grant">R01 HL056931-04</span>, <span class="grant">R01HL056931</span>, <span class="grant">R01HL087676</span>, <span class="grant">R01HL087679</span>; NIA NIH HHS: <span class="grant">N01-AG-1-2109</span>, <span class="grant">N01AG-821336</span>, <span class="grant">N01AG-916413</span>; NICHD NIH HHS: <span class="grant">N01-HD-1-3107</span>; NIDA NIH HHS: <span class="grant">U54DA021519</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK072193</span>, <span class="grant">R01 DK029867</span>, <span class="grant">U01DK062418</span>; NIEHS NIH HHS: <span class="grant">P30ES007033</span>; NIMH NIH HHS: <span class="grant">RL1MH083268</span>; NIMHD NIH HHS: <span class="grant">263MD821336</span>, <span class="grant">263MD916413</span>; PHS HHS: <span class="grant">263-MA-410953</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">068545/Z/02</span>, <span class="grant">070191/Z/03/Z</span>, <span class="grant">076113</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">077011</span>, <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079557</span>, <span class="grant">079895</span>, <span class="grant">088885</span>, <span class="grant">089061</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">666-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19430483">19430483</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2891673">2891673</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.361">10.1038/ng.361</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal rearrangements underlying karyotype differences between Chinese pangolin (Manis pentadactyla) and Malayan pangolin (Manis javanica) revealed by chromosome painting.</h4>
<p class="authors"><span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Su W</span>, <span class="author">Wang Y</span> and <span class="author">Yang F</span></p>
<p class="affiliation">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, 650223, PR China. <a href="mailto:%77%68%6E%69%65%40%6D%61%69%6C.%6B%69%7A.%61%63.%63%6E">&#119;&#104;&#110;&#105;&#101;&#64;&#109;&#97;&#105;&#108;&#46;&#107;&#105;&#122;&#46;&#97;&#99;&#46;&#99;&#110;</a></p><p class="abstract">The Chinese pangolin (Manis pentadactyla), a representative species of the order Pholidota, has been enlisted in the mammalian whole-genome sequencing project mainly because of its phylogenetic importance. Previous studies showed that the diploid number of M. pentadactyla could vary from 2n = 36 to 42. To further characterize the genome organization of M. pentadactyla and to elucidate chromosomal mechanism underlying the karyotype diversity of Pholidota, we flow-sorted the chromosomes of 2n = 40 M. pentadactyla, and generated a set of chromosome-specific probes by DOP-PCR amplification of flow-sorted chromosomes. A comparative chromosome map between M. pentadactyla and the Malayan pangolin (Manis javanica, 2n = 38), as well as between human and M. pentadactyla, was established by chromosome painting for the first time. Our results demonstrate that seven Robertsonian rearrangements, together with considerable variations in the quantity of heterochromatin and in the number of nucleolar organizer regions (NORs) differentiate the karyotypes of 2n = 38 M. javanica and 2n = 40 M. pentadactyla. Moreover, we confirm that the M. javanica Y chromosome bears one NOR. Comparison of human homologous segment associations found in the genomes of M. javanica and M. pentadactyla revealed seven shared associations (HSA 1q/11, 2p/5, 2q/10q, 4p+q/20, 5/13, 6/19p and 8q/10p) that could constitute the potential Pholidota-specific signature rearrangements.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">321-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19283495" rel="external">19283495</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-009-9027-0" rel="external">10.1007/s10577-009-9027-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common genetic variation near the phospholamban gene is associated with cardiac repolarisation: meta-analysis of three genome-wide association studies.</h4>
<p class="authors"><span class="author">Nolte IM</span>, <span class="author">Wallace C</span>, <span class="author">Newhouse SJ</span>, <span class="author">Waggott D</span>, <span class="author">Fu J</span>, <span class="author">Soranzo N</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">Savelieva I</span>, <span class="author">Zheng D</span>, <span class="author">Dalageorgou C</span>, <span class="author">Farrall M</span>, <span class="author">Samani NJ</span>, <span class="author">Connell J</span>, <span class="author">Brown M</span>, <span class="author">Dominiczak A</span>, <span class="author">Lathrop M</span>, <span class="author">Zeggini E</span>, <span class="author">Wain LV</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">DCCT/EDIC Research Group</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Eijgelsheim M</span>, <span class="author">Rice K</span>, <span class="author">de Bakker PI</span>, <span class="author">QTGEN consortium</span>, <span class="author">Pfeufer A</span>, <span class="author">Sanna S</span>, <span class="author">Arking DE</span>, <span class="author">QTSCD consortium</span>, <span class="author">Asselbergs FW</span>, <span class="author">Spector TD</span>, <span class="author">Carter ND</span>, <span class="author">Jeffery S</span>, <span class="author">Tobin M</span>, <span class="author">Caulfield M</span>, <span class="author">Snieder H</span>, <span class="author">Paterson AD</span>, <span class="author">Munroe PB</span> and <span class="author">Jamshidi Y</span></p>
<p class="affiliation">Unit of Genetic Epidemiology and Bioinformatics, Department of Epidemiology, University Medical Center Groningen, University of Groningen, Groningen, the Netherlands.</p><p class="abstract">To identify loci affecting the electrocardiographic QT interval, a measure of cardiac repolarisation associated with risk of ventricular arrhythmias and sudden cardiac death, we conducted a meta-analysis of three genome-wide association studies (GWAS) including 3,558 subjects from the TwinsUK and BRIGHT cohorts in the UK and the DCCT/EDIC cohort from North America. Five loci were significantly associated with QT interval at P&lt;1x10(-6). To validate these findings we performed an in silico comparison with data from two QT consortia: QTSCD (n = 15,842) and QTGEN (n = 13,685). Analysis confirmed the association between common variants near NOS1AP (P = 1.4x10(-83)) and the phospholamban (PLN) gene (P = 1.9x10(-29)). The most associated SNP near NOS1AP (rs12143842) explains 0.82% variance; the SNP near PLN (rs11153730) explains 0.74% variance of QT interval duration. We found no evidence for interaction between these two SNPs (P = 0.99). PLN is a key regulator of cardiac diastolic function and is involved in regulating intracellular calcium cycling, it has only recently been identified as a susceptibility locus for QT interval. These data offer further mechanistic insights into genetic influence on the QT interval which may predispose to life threatening arrhythmias and sudden cardiac death.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">06/094</span>, <span class="grant">FS/05/061/19501</span>, <span class="grant">PG02/128</span>, <span class="grant">SP/02/001</span>; Department of Health; Medical Research Council: <span class="grant">G0400874</span>, <span class="grant">G0501942</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>; NCRR NIH HHS: <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">HL054512</span>, <span class="grant">HL86694</span>, <span class="grant">K23-HL-080025</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">U01 HL080295</span>; NIA NIH HHS: <span class="grant">N01-AG-1-2109</span>; NIDDK NIH HHS: <span class="grant">N01-DK-6-2204</span>, <span class="grant">R01-DK-077510</span>; PHS HHS: <span class="grant">263-MA-410953</span>, <span class="grant">HHSN268200625226C</span>; Wellcome Trust: <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">7</span>;<span class="pagerange">e6138</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19587794">19587794</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2704957">2704957</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0006138">10.1371/journal.pone.0006138</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Developmentally regulated impediments to skin reinnervation by injured peripheral sensory axon terminals.</h4>
<p class="authors"><span class="author">O&#39;Brien GS</span>, <span class="author">Martin SM</span>, <span class="author">S&ouml;llner C</span>, <span class="author">Wright GJ</span>, <span class="author">Becker CG</span>, <span class="author">Portera-Cailliau C</span> and <span class="author">Sagasti A</span></p>
<p class="affiliation">Department of Molecular Cell and Developmental Biology, University of California, Los Angeles, CA 90095, USA.</p><p class="abstract">The structural plasticity of neurites in the central nervous system (CNS) diminishes dramatically after initial development, but the peripheral nervous system (PNS) retains substantial plasticity into adulthood. Nevertheless, functional reinnervation by injured peripheral sensory neurons is often incomplete [1-6]. To investigate the developmental control of skin reinnervation, we imaged the regeneration of trigeminal sensory axon terminals in live zebrafish larvae following laser axotomy. When axons were injured during early stages of outgrowth, regenerating and uninjured axons grew into denervated skin and competed with one another for territory. At later stages, after the establishment of peripheral arbor territories, the ability of uninjured neighbors to sprout diminished severely, and although injured axons reinitiated growth, they were repelled by denervated skin. Regenerating axons were repelled specifically by their former territories, suggesting that local inhibitory factors persist in these regions. Antagonizing the function of several members of the Nogo receptor (NgR)/RhoA pathway improved the capacity of injured axons to grow into denervated skin. Thus, as in the CNS, impediments to reinnervation in the PNS arise after initial establishment of axon arbor structure.</p>
<p class="grants">Funded by: NIDCR NIH HHS: <span class="grant">R01 DE018496-01A2</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">24</span>;<span class="pagerange">2086-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19962310" rel="external">19962310</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2805760" rel="external">2805760</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2009.10.051" rel="external">10.1016/j.cub.2009.10.051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional genomics in zebrafish permits rapid characterization of novel platelet membrane proteins.</h4>
<p class="authors"><span class="author">O&#39;Connor MN</span>, <span class="author">Salles II</span>, <span class="author">Cvejic A</span>, <span class="author">Watkins NA</span>, <span class="author">Walker A</span>, <span class="author">Garner SF</span>, <span class="author">Jones CI</span>, <span class="author">Macaulay IC</span>, <span class="author">Steward M</span>, <span class="author">Zwaginga JJ</span>, <span class="author">Bray SL</span>, <span class="author">Dudbridge F</span>, <span class="author">de Bono B</span>, <span class="author">Goodall AH</span>, <span class="author">Deckmyn H</span>, <span class="author">Stemple DL</span>, <span class="author">Ouwehand WH</span> and <span class="author">Bloodomics Consortium</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">In this study, we demonstrate the suitability of the vertebrate Danio rerio (zebrafish) for functional screening of novel platelet genes in vivo by reverse genetics. Comparative transcript analysis of platelets and their precursor cell, the megakaryocyte, together with nucleated blood cell elements, endothelial cells, and erythroblasts, identified novel platelet membrane proteins with hitherto unknown roles in thrombus formation. We determined the phenotype induced by antisense morpholino oligonucleotide (MO)-based knockdown of 5 of these genes in a laser-induced arterial thrombosis model. To validate the model, the genes for platelet glycoprotein (GP) IIb and the coagulation protein factor VIII were targeted. MO-injected fish showed normal thrombus initiation but severely impaired thrombus growth, consistent with the mouse knockout phenotypes, and concomitant knockdown of both resulted in spontaneous bleeding. Knockdown of 4 of the 5 novel platelet proteins altered arterial thrombosis, as demonstrated by modified kinetics of thrombus initiation and/or development. We identified a putative role for BAMBI and LRRC32 in promotion and DCBLD2 and ESAM in inhibition of thrombus formation. We conclude that phenotypic analysis of MO-injected zebrafish is a fast and powerful method for initial screening of novel platelet proteins for function in thrombosis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077037/Z/05/Z</span>, <span class="grant">WT082597/Z/07/Z</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2009</span>;<span class="volume">113</span>;<span class="number">19</span>;<span class="pagerange">4754-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19109564">19109564</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680375">2680375</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2008-06-162693">10.1182/blood-2008-06-162693</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic mutation databases as tools for molecular epidemiology and molecular pathology of cancer: proposed guidelines for improving data collection, distribution, and integration.</h4>
<p class="authors"><span class="author">Olivier M</span>, <span class="author">Petitjean A</span>, <span class="author">Teague J</span>, <span class="author">Forbes S</span>, <span class="author">Dunnick JK</span>, <span class="author">den Dunnen JT</span>, <span class="author">Langer&oslash;d A</span>, <span class="author">Wilkinson JM</span>, <span class="author">Vihinen M</span>, <span class="author">Cotton RG</span>, <span class="author">Hainaut P</span>, <span class="author">IARC</span> and <span class="author">EC FP6</span></p>
<p class="affiliation">Group of Molecular Carcinogenesis and Biomarkers, International Agency for Research on Cancer, World Health Organization, Lyon, France. <a href="mailto:%6D%6F%6C%69%76%69%65%72%40%69%61%72%63.%66%72">&#109;&#111;&#108;&#105;&#118;&#105;&#101;&#114;&#64;&#105;&#97;&#114;&#99;&#46;&#102;&#114;</a></p><p class="abstract">There are currently less than 40 locus-specific databases (LSDBs) and one large general database that curate data on somatic mutations in human cancer genes. These databases have different scope and use different annotation standards and database systems, resulting in duplicated efforts in data curation, and making it difficult for users to find clear and consistent information. As data related to somatic mutations are generated at an increasing pace it is urgent to create a framework for improving the collecting of this information and making it more accessible to clinicians, scientists, and epidemiologists to facilitate research on biomarkers. Here we propose a data flow for improving the connectivity between existing databases and we provide practical guidelines for data reporting, database contents, and annotation standards. These proposals are based on common standards recommended by the Human Genome Variation Society (HGVS) with additions related to specific requirements of somatic mutations in cancer. Indeed, somatic mutations may be used in molecular pathology and clinical studies to characterize tumor types, help treatment choice, predict response to treatment and patient outcome, or in epidemiological studies as markers for tumor etiology or exposure assessment. Thus, specific annotations are required to cover these diverse research topics. This initiative is meant to promote collaboration and discussion on these issues and the development of adequate resources that would avoid the loss of extremely valuable information generated by years of basic and clinical research.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2009</span>;<span class="volume">30</span>;<span class="number">3</span>;<span class="pagerange">275-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19006239">19006239</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.20832">10.1002/humu.20832</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Findings from bipolar disorder genome-wide association studies replicate in a Finnish bipolar family-cohort.</h4>
<p class="authors"><span class="author">Ollila HM</span>, <span class="author">Soronen P</span>, <span class="author">Silander K</span>, <span class="author">Palo OM</span>, <span class="author">Kiesepp&auml; T</span>, <span class="author">Kaunisto MA</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Peltonen L</span>, <span class="author">Partonen T</span> and <span class="author">Paunio T</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2009</span>;<span class="volume">14</span>;<span class="number">4</span>;<span class="pagerange">351-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19308021" rel="external">19308021</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2694406" rel="external">2694406</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2008.122" rel="external">10.1038/mp.2008.122</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variation in LIN28B is associated with the timing of puberty.</h4>
<p class="authors"><span class="author">Ong KK</span>, <span class="author">Elks CE</span>, <span class="author">Li S</span>, <span class="author">Zhao JH</span>, <span class="author">Luan J</span>, <span class="author">Andersen LB</span>, <span class="author">Bingham SA</span>, <span class="author">Brage S</span>, <span class="author">Smith GD</span>, <span class="author">Ekelund U</span>, <span class="author">Gillson CJ</span>, <span class="author">Glaser B</span>, <span class="author">Golding J</span>, <span class="author">Hardy R</span>, <span class="author">Khaw KT</span>, <span class="author">Kuh D</span>, <span class="author">Luben R</span>, <span class="author">Marcus M</span>, <span class="author">McGeehin MA</span>, <span class="author">Ness AR</span>, <span class="author">Northstone K</span>, <span class="author">Ring SM</span>, <span class="author">Rubin C</span>, <span class="author">Sims MA</span>, <span class="author">Song K</span>, <span class="author">Strachan DP</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Waterworth DM</span>, <span class="author">Wong A</span>, <span class="author">Deloukas P</span>, <span class="author">Barroso I</span>, <span class="author">Mooser V</span>, <span class="author">Loos RJ</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Medical Research Council (MRC) Epidemiology Unit, Addenbrooke&#39;s Hospital, Cambridge, UK. <a href="mailto:%6B%65%6E.%6F%6E%67%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#107;&#101;&#110;&#46;&#111;&#110;&#103;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The timing of puberty is highly variable. We carried out a genome-wide association study for age at menarche in 4,714 women and report an association in LIN28B on chromosome 6 (rs314276, minor allele frequency (MAF) = 0.33, P = 1.5 &times; 10(-8)). In independent replication studies in 16,373 women, each major allele was associated with 0.12 years earlier menarche (95% CI = 0.08-0.16; P = 2.8 &times; 10(-10); combined P = 3.6 &times; 10(-16)). This allele was also associated with earlier breast development in girls (P = 0.001; N = 4,271); earlier voice breaking (P = 0.006, N = 1,026) and more advanced pubic hair development in boys (P = 0.01; N = 4,588); a faster tempo of height growth in girls (P = 0.00008; N = 4,271) and boys (P = 0.03; N = 4,588); and shorter adult height in women (P = 3.6 &times; 10(-7); N = 17,274) and men (P = 0.006; N = 9,840) in keeping with earlier growth cessation. These studies identify variation in LIN28B, a potent and specific regulator of microRNA processing, as the first genetic determinant regulating the timing of human pubertal growth and development.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">73437</span>, <span class="grant">G0000934</span>, <span class="grant">G0401527</span>, <span class="grant">G0401527(74922)</span>, <span class="grant">G0701863</span>, <span class="grant">G9815508</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106179473</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U123092721</span>, <span class="grant">U.1061.00.001 (79471)</span>, <span class="grant">U.1061.00.004(79472)</span>; Wellcome Trust: <span class="grant">068049</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076467/Z/05/Z</span>, <span class="grant">077011</span>, <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079996</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">729-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19448623">19448623</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3000552">3000552</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.382">10.1038/ng.382</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Combined effects of three independent SNPs greatly increase the risk estimate for RA at 6q23.</h4>
<p class="authors"><span class="author">Orozco G</span>, <span class="author">Hinks A</span>, <span class="author">Eyre S</span>, <span class="author">Ke X</span>, <span class="author">Gibbons LJ</span>, <span class="author">Bowes J</span>, <span class="author">Flynn E</span>, <span class="author">Martin P</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">YEAR consortium</span>, <span class="author">Wilson AG</span>, <span class="author">Bax DE</span>, <span class="author">Morgan AW</span>, <span class="author">Emery P</span>, <span class="author">Steer S</span>, <span class="author">Hocking L</span>, <span class="author">Reid DM</span>, <span class="author">Wordsworth P</span>, <span class="author">Harrison P</span>, <span class="author">Thomson W</span>, <span class="author">Barton A</span> and <span class="author">Worthington J</span></p>
<p class="affiliation">arc-Epidemiology Unit, Stopford Building, The University of Manchester, Manchester M13 9PT, UK. <a href="mailto:%67%69%73%65%6C%61.%6F%72%6F%7A%63%6F%40%6D%61%6E%63%68%65%73%74%65%72.%61%63.%75%6B">&#103;&#105;&#115;&#101;&#108;&#97;&#46;&#111;&#114;&#111;&#122;&#99;&#111;&#64;&#109;&#97;&#110;&#99;&#104;&#101;&#115;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The most consistent finding derived from the WTCCC GWAS for rheumatoid arthritis (RA) was association to a SNP at 6q23. We performed a fine-mapping of the region in order to search the 6q23 region for additional disease variants. 3962 RA patients and 3531 healthy controls were included in the study. We found 18 SNPs associated with RA. The SNP showing the strongest association was rs6920220 [P = 2.6 x 10(-6), OR (95% CI) 1.22 (1.13-1.33)]. The next most strongly associated SNP was rs13207033 [P = 0.0001, OR (95% CI) 0.86 (0.8-0.93)] which was perfectly correlated with rs10499194, a SNP previously associated with RA in a US/European series. Additionally, we found a number of new potential RA markers, including rs5029937, located in the intron 2 of TNFAIP3. Of the 18 associated SNPs, three polymorphisms, rs6920220, rs13207033 and rs5029937, remained significant after conditional logistic regression analysis. The combination of the carriage of both risk alleles of rs6920220 and rs5029937 together with the absence of the protective allele of rs13207033 was strongly associated with RA when compared with carriage of none [OR of 1.86 (95% CI) (1.51-2.29)]. This equates to an effect size of 1.50 (95% CI 1.21-1.85) compared with controls and is higher than that obtained for any SNP individually. This is the first study to show that the confirmed loci from the GWA studies, that confer only a modest effect size, could harbour a significantly greater effect once the effect of additional risk variants are accounted for.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>, <span class="grant">18475</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0600329</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">14</span>;<span class="pagerange">2693-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19417005">19417005</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2701332">2701332</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp193">10.1093/hmg/ddp193</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The discovery of genes implicated in myocardial infarction</h4>
<p class="authors"><span class="author">Ouwehand WH</span>, <span class="author">Bloodomics-Cardiogenics-Consortia </span></p>
<p class="publication"><span class="title">Journal of Thrombosis and Haemostasis; 22nd Congress of the International-Society-on-Thrombosis-and-Haemostasis.</span> <span class="year">2009</span>;<span class="volume">7</span>;<span class="pagerange">305-7</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sites of differential DNA methylation between placenta and peripheral blood: molecular markers for noninvasive prenatal diagnosis of aneuploidies.</h4>
<p class="authors"><span class="author">Papageorgiou EA</span>, <span class="author">Fiegler H</span>, <span class="author">Rakyan V</span>, <span class="author">Beck S</span>, <span class="author">Hulten M</span>, <span class="author">Lamnissou K</span>, <span class="author">Carter NP</span> and <span class="author">Patsalis PC</span></p>
<p class="affiliation">Cytogenetics and Genomics Department, The CyprusInstitute of Neurology and Genetics, Nicosia, Cyprus.</p><p class="abstract">The use of epigenetic differences between maternal whole blood and fetal (placental) DNA is one of the main areas of interest for the development of noninvasive prenatal diagnosis of aneuploidies. However, the lack of detailed chromosome-wide identification of differentially methylated sites has limited the application of this approach. In this study, we describe an analysis of chromosome-wide methylation status using methylation DNA immunoprecipitation coupled with high-resolution tiling oligonucleotide array analysis specific for chromosomes 21, 18, 13, X, and Y using female whole blood and placental DNA. We identified more than 2000 regions of differential methylation between female whole blood and placental DNA on each of the chromosomes tested. A subset of the differentially methylated regions identified was validated by real-time quantitative polymerase chain reaction. Additionally, correlation of these regions with CpG islands, genes, and promoter regions was investigated. Between 56 to 83% of the regions were located within nongenic regions whereas only 1 to 11% of the regions overlapped with CpG islands; of these, up to 65% were found in promoter regions. In summary, we identified a large number of previously unreported fetal epigenetic molecular markers that have the potential to be developed into targets for noninvasive prenatal diagnosis of trisomy 21 and other common aneuploidies. In addition, we demonstrated the effectiveness of the methylation DNA immunoprecipitation approach in the enrichment of hypermethylated fetal DNA.</p>
<p class="publication"><span class="title">The American journal of pathology</span>&nbsp;<span class="year">2009</span>;<span class="volume">174</span>;<span class="number">5</span>;<span class="pagerange">1609-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19349366" rel="external">19349366</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2671250" rel="external">2671250</a>; DOI: <a href="http://dx.doi.org/10.2353%2Fajpath.2009.081038" rel="external">10.2353/ajpath.2009.081038</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ethical data release in genome-wide association studies in developing countries.</h4>
<p class="authors"><span class="author">Parker M</span>, <span class="author">Bull SJ</span>, <span class="author">de Vries J</span>, <span class="author">Agbenyega T</span>, <span class="author">Doumbo OK</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Ethox Centre, University of Oxford, Oxford, United Kingdom. <a href="mailto:%6D%69%63%68%61%65%6C.%70%61%72%6B%65%72%40%65%74%68%6F%78.%6F%78.%61%63.%75%6B">&#109;&#105;&#99;&#104;&#97;&#101;&#108;&#46;&#112;&#97;&#114;&#107;&#101;&#114;&#64;&#101;&#116;&#104;&#111;&#120;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; PHS HHS: <span class="grant">566</span>; Wellcome Trust: <span class="grant">077383/Z/05/Z</span>, <span class="grant">087285/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e1000143</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19956792">19956792</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2771895">2771895</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pmed.1000143">10.1371/journal.pmed.1000143</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A strand-specific RNA-Seq analysis of the transcriptome of the typhoid bacillus Salmonella typhi.</h4>
<p class="authors"><span class="author">Perkins TT</span>, <span class="author">Kingsley RA</span>, <span class="author">Fookes MC</span>, <span class="author">Gardner PP</span>, <span class="author">James KD</span>, <span class="author">Yu L</span>, <span class="author">Assefa SA</span>, <span class="author">He M</span>, <span class="author">Croucher NJ</span>, <span class="author">Pickard DJ</span>, <span class="author">Maskell DJ</span>, <span class="author">Parkhill J</span>, <span class="author">Choudhary J</span>, <span class="author">Thomson NR</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">High-density, strand-specific cDNA sequencing (ssRNA-seq) was used to analyze the transcriptome of Salmonella enterica serovar Typhi (S. Typhi). By mapping sequence data to the entire S. Typhi genome, we analyzed the transcriptome in a strand-specific manner and further defined transcribed regions encoded within prophages, pseudogenes, previously un-annotated, and 3&#39;- or 5&#39;-untranslated regions (UTR). An additional 40 novel candidate non-coding RNAs were identified beyond those previously annotated. Proteomic analysis was combined with transcriptome data to confirm and refine the annotation of a number of hpothetical genes. ssRNA-seq was also combined with microarray and proteome analysis to further define the S. Typhi OmpR regulon and identify novel OmpR regulated transcripts. Thus, ssRNA-seq provides a novel and powerful approach to the characterization of the bacterial transcriptome.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">e1000569</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19609351">19609351</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2704369">2704369</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000569">10.1371/journal.pgen.1000569</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Circulating beta-carotene levels and type 2 diabetes-cause or effect?</h4>
<p class="authors"><span class="author">Perry JR</span>, <span class="author">Ferrucci L</span>, <span class="author">Bandinelli S</span>, <span class="author">Guralnik J</span>, <span class="author">Semba RD</span>, <span class="author">Rice N</span>, <span class="author">Melzer D</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Saxena R</span>, <span class="author">Scott LJ</span>, <span class="author">McCarthy MI</span>, <span class="author">Hattersley AT</span>, <span class="author">Zeggini E</span>, <span class="author">Weedon MN</span> and <span class="author">Frayling TM</span></p>
<p class="affiliation">Peninsula Medical School, Exeter, UK.</p><p class="abstract">Circulating beta-carotene levels are inversely associated with risk of type 2 diabetes, but the causal direction of this association is not certain. In this study we used a Mendelian randomisation approach to provide evidence for or against the causal role of the antioxidant vitamin beta-carotene in type 2 diabetes.</p><p class="abstract"><strong>Methods</strong>: We used a common polymorphism (rs6564851) near the BCMO1 gene, which is strongly associated with circulating beta-carotene levels (p = 2 x 10(-24)), with each G allele associated with a 0.27 standard deviation increase in levels. We used data from the InCHIANTI and Uppsala Longitudinal Study of Adult Men (ULSAM) studies to estimate the association between beta-carotene levels and type 2 diabetes. We next used a triangulation approach to estimate the expected effect of rs6564851 on type 2 diabetes risk and compared this with the observed effect using data from 4549 type 2 diabetes patients and 5579 controls from the Diabetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium.</p><p class="abstract"><strong>Results</strong>: A 0.27 standard deviation increase in beta-carotene levels was associated with an OR of 0.90 (95% CI 0.86-0.95) for type 2 diabetes in the InCHIANTI study. This association was similar to that of the ULSAM study (OR 0.90 [0.84-0.97]). In contrast, there was no association between rs6564851 and type 2 diabetes (OR 0.98 [0.93-1.04], p = 0.58); this effect size was also smaller than that expected, given the known associations between rs6564851 and beta-carotene levels, and the associations between beta-carotene levels and type 2 diabetes.</p><p class="abstract">Our findings in this Mendelian randomisation study are in keeping with randomised controlled trials suggesting that beta-carotene is not causally protective against type 2 diabetes.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">N01-AG-6-2101</span>, <span class="grant">N01-AG-6-2103</span>, <span class="grant">N01-AG-6-2106</span>, <span class="grant">R01 AG24233-01</span>, <span class="grant">Z01 AG000015-50</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2009</span>;<span class="volume">52</span>;<span class="number">10</span>;<span class="pagerange">2117-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19662379" rel="external">19662379</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2746424" rel="external">2746424</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-009-1475-8" rel="external">10.1007/s00125-009-1475-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association data identifies two loci influencing age at menarche.</h4>
<p class="authors"><span class="author">Perry JR</span>, <span class="author">Stolk L</span>, <span class="author">Franceschini N</span>, <span class="author">Lunetta KL</span>, <span class="author">Zhai G</span>, <span class="author">McArdle PF</span>, <span class="author">Smith AV</span>, <span class="author">Aspelund T</span>, <span class="author">Bandinelli S</span>, <span class="author">Boerwinkle E</span>, <span class="author">Cherkas L</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Estrada K</span>, <span class="author">Ferrucci L</span>, <span class="author">Folsom AR</span>, <span class="author">Garcia M</span>, <span class="author">Gudnason V</span>, <span class="author">Hofman A</span>, <span class="author">Karasik D</span>, <span class="author">Kiel DP</span>, <span class="author">Launer LJ</span>, <span class="author">van Meurs J</span>, <span class="author">Nalls MA</span>, <span class="author">Rivadeneira F</span>, <span class="author">Shuldiner AR</span>, <span class="author">Singleton A</span>, <span class="author">Soranzo N</span>, <span class="author">Tanaka T</span>, <span class="author">Visser JA</span>, <span class="author">Weedon MN</span>, <span class="author">Wilson SG</span>, <span class="author">Zhuang V</span>, <span class="author">Streeten EA</span>, <span class="author">Harris TB</span>, <span class="author">Murray A</span>, <span class="author">Spector TD</span>, <span class="author">Demerath EW</span>, <span class="author">Uitterlinden AG</span> and <span class="author">Murabito JM</span></p>
<p class="affiliation">Institute of Biomedical and Clinical Science, Peninsula Medical School, Exeter, UK.</p><p class="abstract">We conducted a meta-analysis of genome-wide association data to detect genes influencing age at menarche in 17,510 women. The strongest signal was at 9q31.2 (P = 1.7 &times; 10(-9)), where the nearest genes include TMEM38B, FKTN, FSD1L, TAL2 and ZNF462. The next best signal was near the LIN28B gene (rs7759938; P = 7.0 &times; 10(-9)), which also influences adult height. We provide the first evidence for common genetic variants influencing female sexual maturation.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">M01 RR 16500</span>, <span class="grant">M01 RR016500-02</span>; NHLBI NIH HHS: <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC055016</span>, <span class="grant">N01 HC055018</span>, <span class="grant">N01 HC055019</span>, <span class="grant">N01 HC055020</span>, <span class="grant">N01 HC055021</span>, <span class="grant">N01 HC055022</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N02 HL64278</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL72515</span>; NIA NIH HHS: <span class="grant">N.1-AG-1-1</span>, <span class="grant">N.1-AG-1-2111</span>, <span class="grant">N01 AG012100</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">N01-AG-5-0002</span>, <span class="grant">R01 AR/AG 41398</span>, <span class="grant">R21 AG032598</span>, <span class="grant">R21 AG032598-02</span>, <span class="grant">R21AG032598</span>, <span class="grant">U19 AG023122</span>, <span class="grant">U19 AG023122-05</span>; NIAMS NIH HHS: <span class="grant">R01 AR041398</span>, <span class="grant">R01 AR041398-15</span>; NIDDK NIH HHS: <span class="grant">P30 DK072488</span>, <span class="grant">P30 DK072488-02</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>, <span class="grant">263 MD821336</span>, <span class="grant">263 MD9164 13</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">648-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19448620">19448620</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2942986">2942986</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.386">10.1038/ng.386</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">OSBPL10, a novel candidate gene for high triglyceride trait in dyslipidemic Finnish subjects, regulates cellular lipid metabolism.</h4>
<p class="authors"><span class="author">Perttil&auml; J</span>, <span class="author">Merikanto K</span>, <span class="author">Naukkarinen J</span>, <span class="author">Surakka I</span>, <span class="author">Martin NW</span>, <span class="author">Tanhuanp&auml;&auml; K</span>, <span class="author">Grimard V</span>, <span class="author">Taskinen MR</span>, <span class="author">Thiele C</span>, <span class="author">Salomaa V</span>, <span class="author">Jula A</span>, <span class="author">Perola M</span>, <span class="author">Virtanen I</span>, <span class="author">Peltonen L</span> and <span class="author">Olkkonen VM</span></p>
<p class="affiliation">National Institute for Health and Welfare/Public Health Genomics Unit, Biomedicum, 00251 Helsinki, Finland.</p><p class="abstract">Analysis of variants in three genes encoding oxysterol-binding protein (OSBP) homologues (OSBPL2, OSBPL9, OSBPL10) in Finnish families with familial low high-density lipoprotein (HDL) levels (N = 426) or familial combined hyperlipidemia (N = 684) revealed suggestive linkage of OSBPL10 single-nucleotide polymorphisms (SNPs) with extreme end high triglyceride (TG; &gt;90th percentile) trait. Prompted by this initial finding, we carried out association analysis in a metabolic syndrome subcohort (Genmets) of Health2000 examination survey (N = 2,138), revealing association of multiple OSBPL10 SNPs with high serum TG levels (&gt;95th percentile). To investigate whether OSBPL10 could be the gene underlying the observed linkage and association, we carried out functional experiments in the human hepatoma cell line Huh7. Silencing of OSBPL10 increased the incorporation of [(3)H]acetate into cholesterol and both [(3)H]acetate and [(3)H]oleate into triglycerides and enhanced the accumulation of secreted apolipoprotein B100 in growth medium, suggesting that the encoded protein ORP10 suppresses hepatic lipogenesis and very-low-density lipoprotein production. ORP10 was shown to associate dynamically with microtubules, consistent with its involvement in intracellular transport or organelle positioning. The data introduces OSBPL10 as a gene whose variation may contribute to high triglyceride levels in dyslipidemic Finnish subjects and provides evidence for ORP10 as a regulator of cellular lipid metabolism.</p>
<p class="publication"><span class="title">Journal of molecular medicine (Berlin, Germany)</span>&nbsp;<span class="year">2009</span>;<span class="volume">87</span>;<span class="number">8</span>;<span class="pagerange">825-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19554302" rel="external">19554302</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2707950" rel="external">2707950</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00109-009-0490-z" rel="external">10.1007/s00109-009-0490-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Agouti C57BL/6N embryonic stem cells for mouse genetic resources.</h4>
<p class="authors"><span class="author">Pettitt SJ</span>, <span class="author">Liang Q</span>, <span class="author">Rairdan XY</span>, <span class="author">Moran JL</span>, <span class="author">Prosser HM</span>, <span class="author">Beier DR</span>, <span class="author">Lloyd KC</span>, <span class="author">Bradley A</span> and <span class="author">Skarnes WC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p><p class="abstract">We report the characterization of a highly germline competent C57BL/6N mouse embryonic stem cell line, JM8. To simplify breeding schemes, the dominant agouti coat color gene was restored in JM8 cells by targeted repair of the C57BL/6 nonagouti mutation. These cells provide a robust foundation for large-scale mouse knockout programs that aim to provide a public resource of targeted mutations in the C57BL/6 genetic background.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01-HG004080</span>; PHS HHS: <span class="grant">U01-42430</span>; Wellcome Trust: <span class="grant">077188</span>, <span class="grant">WT077187</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2009</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">493-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19525957">19525957</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3555078">3555078</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1342">10.1038/nmeth.1342</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation in Salmonella enterica serovar typhi IncHI1 plasmids during the global spread of resistant typhoid fever.</h4>
<p class="authors"><span class="author">Phan MD</span>, <span class="author">Kidgell C</span>, <span class="author">Nair S</span>, <span class="author">Holt KE</span>, <span class="author">Turner AK</span>, <span class="author">Hinds J</span>, <span class="author">Butcher P</span>, <span class="author">Cooke FJ</span>, <span class="author">Thomson NR</span>, <span class="author">Titball R</span>, <span class="author">Bhutta ZA</span>, <span class="author">Hasan R</span>, <span class="author">Dougan G</span> and <span class="author">Wain J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">A global collection of plasmids of the IncHI1 incompatibility group from Salmonella enterica serovar Typhi were analyzed by using a combination of DNA sequencing, DNA sequence analysis, PCR, and microarrays. The IncHI1 resistance plasmids of serovar Typhi display a backbone of conserved gene content and arrangement, within which are embedded preferred acquisition sites for horizontal DNA transfer events. The variable regions appear to be preferred acquisition sites for DNA, most likely through composite transposition, which is presumably driven by the acquisition of resistance genes. Plasmid multilocus sequence typing, a molecular typing method for IncHI1 plasmids, was developed using variation in six conserved loci to trace the spread of these plasmids and to elucidate their evolutionary relationships. The application of this method to a collection of 36 IncHI1 plasmids revealed a chronological clustering of plasmids despite their difference in geographical origins. Our findings suggest that the predominant plasmid types present after 1993 have not evolved directly from the earlier predominant plasmid type but have displaced them. We propose that antibiotic selection acts to maintain resistance genes on the plasmid, but there is also competition between plasmids encoding the same resistance phenotype.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">080039</span></p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2009</span>;<span class="volume">53</span>;<span class="number">2</span>;<span class="pagerange">716-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19015365" rel="external">19015365</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2630618" rel="external">2630618</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.00645-08" rel="external">10.1128/AAC.00645-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Preparation of bacteriophage lysates and pure DNA.</h4>
<p class="authors"><span class="author">Pickard DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Preparation of pure bacteriophage DNA used to rely on using CsCl gradients to give high purity or methods that yielded DNA that was either of low recovery or subject to significant genomic contamination. Recently though, new methods have come along that allow the purification of DNA from plate lysates that are not only capable of high yield but also, for all intents and purposes, free of genomic contamination (i.e. no visible genomic contamination on restriction analysis or when used for bacteriophage sequencing). This protocol that form the basis of this short section can be used to prepare bacteriophage DNA from one or two 9 cm L-agar plates. For these preps, the use of agarose in the top agar is recommended to avoid any restriction inhibitors that may be present in some agar preparations.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">502</span>;<span class="pagerange">3-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19082547">19082547</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-60327-565-1_1">10.1007/978-1-60327-565-1_1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of AKT1 with verbal learning, verbal memory, and regional cortical gray matter density in twins.</h4>
<p class="authors"><span class="author">Pietil&auml;inen OP</span>, <span class="author">Paunio T</span>, <span class="author">Loukola A</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Kiesepp&auml; T</span>, <span class="author">Thompson P</span>, <span class="author">Toga AW</span>, <span class="author">van Erp TG</span>, <span class="author">Silventoinen K</span>, <span class="author">Soronen P</span>, <span class="author">Hennah W</span>, <span class="author">Turunen JA</span>, <span class="author">Wedenoja J</span>, <span class="author">Palo OM</span>, <span class="author">Silander K</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Kaprio J</span>, <span class="author">Cannon TD</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">FIMM, Institute for Molecular Medicine Finland and National Public Health Institute, Biomedicum, Helsinki, Finland.</p><p class="abstract">AKT1, encoding the protein kinase B, has been associated with the genetic etiology of schizophrenia and bipolar disorder. However, minuscule data exist on the role of different alleles of AKT1 in measurable quantitative endophenotypes, such as cognitive abilities and neuroanatomical features, showing deviations in schizophrenia and bipolar disorder. We evaluated the contribution of AKT1 to quantitative cognitive traits and 3D high-resolution neuroanatomical images in a Finnish twin sample consisting of 298 twins: 61 pairs with schizophrenia (8 concordant), 31 pairs with bipolar disorder (5 concordant) and 65 control pairs matched for age, sex and demographics. An AKT1 allele defined by the SNP rs1130214 located in the UTR of the gene revealed association with cognitive traits related to verbal learning and memory (P = 0.0005 for a composite index). This association was further fortified by a higher degree of resemblance of verbal memory capacity in pairs sharing the rs1130214 genotype compared to pairs not sharing the genotype. Furthermore, the same allele was also associated with decreased gray matter density in medial and dorsolateral prefrontal cortex (P &lt; 0.05). Our findings support the role of AKT1 in the genetic background of cognitive and anatomical features, known to be affected by psychotic disorders. The established association of the same allelic variant of AKT1 with both cognitive and neuroanatomical aberrations could suggest that AKT1 exerts its effect on verbal learning and memory via neural networks involving prefrontal cortex.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">American journal of medical genetics. Part B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric Genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">150B</span>;<span class="number">5</span>;<span class="pagerange">683-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19051289" rel="external">19051289</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2708342" rel="external">2708342</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.b.30890" rel="external">10.1002/ajmg.b.30890</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mother&#39;s genome or maternally-inherited genes acting in the fetus influence gestational age in familial preterm birth.</h4>
<p class="authors"><span class="author">Plunkett J</span>, <span class="author">Feitosa MF</span>, <span class="author">Trusgnich M</span>, <span class="author">Wangler MF</span>, <span class="author">Palomar L</span>, <span class="author">Kistka ZA</span>, <span class="author">DeFranco EA</span>, <span class="author">Shen TT</span>, <span class="author">Stormo AE</span>, <span class="author">Puttonen H</span>, <span class="author">Hallman M</span>, <span class="author">Haataja R</span>, <span class="author">Luukkonen A</span>, <span class="author">Fellman V</span>, <span class="author">Peltonen L</span>, <span class="author">Palotie A</span>, <span class="author">Daw EW</span>, <span class="author">An P</span>, <span class="author">Teramo K</span>, <span class="author">Borecki I</span> and <span class="author">Muglia LJ</span></p>
<p class="affiliation">Department of Pediatrics, Washington University School of Medicine, St Louis, Miss, USA.</p><p class="abstract"><strong>Objective</strong>: While multiple lines of evidence suggest the importance of genetic contributors to risk of preterm birth, the nature of the genetic component has not been identified. We perform segregation analyses to identify the best fitting genetic model for gestational age, a quantitative proxy for preterm birth.</p><p class="abstract"><strong>Methods</strong>: Because either mother or infant can be considered the proband from a preterm delivery and there is evidence to suggest that genetic factors in either one or both may influence the trait, we performed segregation analysis for gestational age either attributed to the infant (infant&#39;s gestational age), or the mother (by averaging the gestational ages at which her children were delivered), using 96 multiplex preterm families.</p><p class="abstract"><strong>Results</strong>: These data lend further support to a genetic component contributing to birth timing since sporadic (i.e. no familial resemblance) and nontransmission (i.e. environmental factors alone contribute to gestational age) models are strongly rejected. Analyses of gestational age attributed to the infant support a model in which mother&#39;s genome and/or maternally-inherited genes acting in the fetus are largely responsible for birth timing, with a smaller contribution from the paternally-inherited alleles in the fetal genome.</p><p class="abstract"><strong>Conclusion</strong>: Our findings suggest that genetic influences on birth timing are important and likely complex.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">T32 GM081739-02</span>, <span class="grant">T32GM081739</span></p>
<p class="publication"><span class="title">Human heredity</span>&nbsp;<span class="year">2009</span>;<span class="volume">68</span>;<span class="number">3</span>;<span class="pagerange">209-19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19521103" rel="external">19521103</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2869074" rel="external">2869074</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000224641" rel="external">10.1159/000224641</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variants in MTNR1B influence fasting glucose levels.</h4>
<p class="authors"><span class="author">Prokopenko I</span>, <span class="author">Langenberg C</span>, <span class="author">Florez JC</span>, <span class="author">Saxena R</span>, <span class="author">Soranzo N</span>, <span class="author">Thorleifsson G</span>, <span class="author">Loos RJ</span>, <span class="author">Manning AK</span>, <span class="author">Jackson AU</span>, <span class="author">Aulchenko Y</span>, <span class="author">Potter SC</span>, <span class="author">Erdos MR</span>, <span class="author">Sanna S</span>, <span class="author">Hottenga JJ</span>, <span class="author">Wheeler E</span>, <span class="author">Kaakinen M</span>, <span class="author">Lyssenko V</span>, <span class="author">Chen WM</span>, <span class="author">Ahmadi K</span>, <span class="author">Beckmann JS</span>, <span class="author">Bergman RN</span>, <span class="author">Bochud M</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Buchanan TA</span>, <span class="author">Cao A</span>, <span class="author">Cervino A</span>, <span class="author">Coin L</span>, <span class="author">Collins FS</span>, <span class="author">Crisponi L</span>, <span class="author">de Geus EJ</span>, <span class="author">Dehghan A</span>, <span class="author">Deloukas P</span>, <span class="author">Doney AS</span>, <span class="author">Elliott P</span>, <span class="author">Freimer N</span>, <span class="author">Gateva V</span>, <span class="author">Herder C</span>, <span class="author">Hofman A</span>, <span class="author">Hughes TE</span>, <span class="author">Hunt S</span>, <span class="author">Illig T</span>, <span class="author">Inouye M</span>, <span class="author">Isomaa B</span>, <span class="author">Johnson T</span>, <span class="author">Kong A</span>, <span class="author">Krestyaninova M</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Lim N</span>, <span class="author">Lindblad U</span>, <span class="author">Lindgren CM</span>, <span class="author">McCann OT</span>, <span class="author">Mohlke KL</span>, <span class="author">Morris AD</span>, <span class="author">Naitza S</span>, <span class="author">Orr&ugrave; M</span>, <span class="author">Palmer CN</span>, <span class="author">Pouta A</span>, <span class="author">Randall J</span>, <span class="author">Rathmann W</span>, <span class="author">Saramies J</span>, <span class="author">Scheet P</span>, <span class="author">Scott LJ</span>, <span class="author">Scuteri A</span>, <span class="author">Sharp S</span>, <span class="author">Sijbrands E</span>, <span class="author">Smit JH</span>, <span class="author">Song K</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Stringham HM</span>, <span class="author">Tuomi T</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Voight BF</span>, <span class="author">Waterworth D</span>, <span class="author">Wichmann HE</span>, <span class="author">Willemsen G</span>, <span class="author">Witteman JC</span>, <span class="author">Yuan X</span>, <span class="author">Zhao JH</span>, <span class="author">Zeggini E</span>, <span class="author">Schlessinger D</span>, <span class="author">Sandhu M</span>, <span class="author">Boomsma DI</span>, <span class="author">Uda M</span>, <span class="author">Spector TD</span>, <span class="author">Penninx BW</span>, <span class="author">Altshuler D</span>, <span class="author">Vollenweider P</span>, <span class="author">Jarvelin MR</span>, <span class="author">Lakatta E</span>, <span class="author">Waeber G</span>, <span class="author">Fox CS</span>, <span class="author">Peltonen L</span>, <span class="author">Groop LC</span>, <span class="author">Mooser V</span>, <span class="author">Cupples LA</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Boehnke M</span>, <span class="author">Barroso I</span>, <span class="author">Van Duijn C</span>, <span class="author">Dupuis J</span>, <span class="author">Watanabe RM</span>, <span class="author">Stefansson K</span>, <span class="author">McCarthy MI</span>, <span class="author">Wareham NJ</span>, <span class="author">Meigs JB</span> and <span class="author">Abecasis GR</span></p>
<p class="affiliation">[1] Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Oxford OX3 7LJ, UK. [2] Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK. [3] These authors contributed equally to this work.</p><p class="abstract">To identify previously unknown genetic loci associated with fasting glucose concentrations, we examined the leading association signals in ten genome-wide association scans involving a total of 36,610 individuals of European descent. Variants in the gene encoding melatonin receptor 1B (MTNR1B) were consistently associated with fasting glucose across all ten studies. The strongest signal was observed at rs10830963, where each G allele (frequency 0.30 in HapMap CEU) was associated with an increase of 0.07 (95% CI = 0.06-0.08) mmol/l in fasting glucose levels (P = 3.2 x 10(-50)) and reduced beta-cell function as measured by homeostasis model assessment (HOMA-B, P = 1.1 x 10(-15)). The same allele was associated with an increased risk of type 2 diabetes (odds ratio = 1.09 (1.05-1.12), per G allele P = 3.3 x 10(-7)) in a meta-analysis of 13 case-control studies totaling 18,236 cases and 64,453 controls. Our analyses also confirm previous associations of fasting glucose with variants at the G6PC2 (rs560887, P = 1.1 x 10(-57)) and GCK (rs4607517, P = 1.0 x 10(-25)) loci.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000649</span>, <span class="grant">G016121</span>, <span class="grant">G0500539</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NCRR NIH HHS: <span class="grant">RR-163736</span>; NHGRI NIH HHS: <span class="grant">HG-02651</span>, <span class="grant">R01 HG002651</span>, <span class="grant">R01 HG002651-05</span>; NHLBI NIH HHS: <span class="grant">HC-25195</span>, <span class="grant">HL-084729</span>, <span class="grant">HL-087679</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL087679-02</span>, <span class="grant">U01 HL084729</span>, <span class="grant">U01 HL084729-03</span>; NIDA NIH HHS: <span class="grant">DA-021519</span>, <span class="grant">U54 DA021519</span>, <span class="grant">U54 DA021519-04</span>; NIDDK NIH HHS: <span class="grant">DK-062370</span>, <span class="grant">DK-065978</span>, <span class="grant">DK-072193</span>, <span class="grant">DK-078616</span>, <span class="grant">DK-080140</span>, <span class="grant">DK069922</span>, <span class="grant">K23 DK065978</span>, <span class="grant">K23 DK065978-05</span>, <span class="grant">K24 DK080140</span>, <span class="grant">K24 DK080140-01</span>, <span class="grant">K24 DK080140-02</span>, <span class="grant">R01 DK029867</span>, <span class="grant">R01 DK062370</span>, <span class="grant">R01 DK062370-05</span>, <span class="grant">R01 DK069922</span>, <span class="grant">R01 DK069922-02</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK072193-04</span>, <span class="grant">R01 DK078616</span>, <span class="grant">R01 DK078616-01A1</span>; NIMH NIH HHS: <span class="grant">MH059160</span>, <span class="grant">R01 MH059160</span>, <span class="grant">R01 MH059160-04</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">077016</span>, <span class="grant">079557</span>, <span class="grant">083948</span>, <span class="grant">089061</span>, <span class="grant">GR069224</span>, <span class="grant">GR072960</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">1</span>;<span class="pagerange">77-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19060907">19060907</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682768">2682768</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.290">10.1038/ng.290</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Linkage disequilibrium mapping of the replicated type 2 diabetes linkage signal on chromosome 1q.</h4>
<p class="authors"><span class="author">Prokopenko I</span>, <span class="author">Zeggini E</span>, <span class="author">Hanson RL</span>, <span class="author">Mitchell BD</span>, <span class="author">Rayner NW</span>, <span class="author">Akan P</span>, <span class="author">Baier L</span>, <span class="author">Das SK</span>, <span class="author">Elliott KS</span>, <span class="author">Fu M</span>, <span class="author">Frayling TM</span>, <span class="author">Groves CJ</span>, <span class="author">Gwilliam R</span>, <span class="author">Scott LJ</span>, <span class="author">Voight BF</span>, <span class="author">Hattersley AT</span>, <span class="author">Hu C</span>, <span class="author">Morris AD</span>, <span class="author">Ng M</span>, <span class="author">Palmer CN</span>, <span class="author">Tello-Ruiz M</span>, <span class="author">Vaxillaire M</span>, <span class="author">Wang CR</span>, <span class="author">Stein L</span>, <span class="author">Chan J</span>, <span class="author">Jia W</span>, <span class="author">Froguel P</span>, <span class="author">Elbein SC</span>, <span class="author">Deloukas P</span>, <span class="author">Bogardus C</span>, <span class="author">Shuldiner AR</span>, <span class="author">McCarthy MI</span> and <span class="author">International Type 2 Diabetes 1q Consortium</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract"><strong>Objective</strong>: Linkage of the chromosome 1q21-25 region to type 2 diabetes has been demonstrated in multiple ethnic groups. We performed common variant fine-mapping across a 23-Mb interval in a multiethnic sample to search for variants responsible for this linkage signal.</p><p class="abstract">In all, 5,290 single nucleotide polymorphisms (SNPs) were successfully genotyped in 3,179 type 2 diabetes case and control subjects from eight populations with evidence of 1q linkage. Samples were ascertained using strategies designed to enhance power to detect variants causal for 1q linkage. After imputation, we estimate approximately 80% coverage of common variation across the region (r (2) &gt; 0.8, Europeans). Association signals of interest were evaluated through in silico replication and de novo genotyping in approximately 8,500 case subjects and 12,400 control subjects.</p><p class="abstract"><strong>Results</strong>: Association mapping of the 23-Mb region identified two strong signals, both of which were restricted to the subset of European-descent samples. The first mapped to the NOS1AP (CAPON) gene region (lead SNP: rs7538490, odds ratio 1.38 [95% CI 1.21-1.57], P = 1.4 x 10(-6), in 999 case subjects and 1,190 control subjects); the second mapped within an extensive region of linkage disequilibrium that includes the ASH1L and PKLR genes (lead SNP: rs11264371, odds ratio 1.48 [1.18-1.76], P = 1.0 x 10(-5), under a dominant model). However, there was no evidence for association at either signal on replication, and, across all data (&gt;24,000 subjects), there was no indication that these variants were causally related to type 2 diabetes status.</p><p class="abstract"><strong>Conclusions</strong>: Detailed fine-mapping of the 23-Mb region of replicated linkage has failed to identify common variant signals contributing to the observed signal. Future studies should focus on identification of causal alleles of lower frequency and higher penetrance.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0601201</span>, <span class="grant">G0601201(79675)</span>; NCI NIH HHS: <span class="grant">K07-CA67960</span>; NCRR NIH HHS: <span class="grant">M01 RR14288</span>; NIA NIH HHS: <span class="grant">T32-AG00219</span>; NIDDK NIH HHS: <span class="grant">K24-DK02673</span>, <span class="grant">R01-DK073490</span>, <span class="grant">R01-DK39311</span>, <span class="grant">R01-DK54261</span>, <span class="grant">U01-DK58026</span>; Wellcome Trust: <span class="grant">072960</span>, <span class="grant">077011</span>, <span class="grant">079557</span>, <span class="grant">088885</span>, <span class="grant">GR072960</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2009</span>;<span class="volume">58</span>;<span class="number">7</span>;<span class="pagerange">1704-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19389826" rel="external">19389826</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2699860" rel="external">2699860</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb09-0081" rel="external">10.2337/db09-0081</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The consensus coding sequence (CCDS) project: Identifying a common protein-coding gene set for the human and mouse genomes.</h4>
<p class="authors"><span class="author">Pruitt KD</span>, <span class="author">Harrow J</span>, <span class="author">Harte RA</span>, <span class="author">Wallin C</span>, <span class="author">Diekhans M</span>, <span class="author">Maglott DR</span>, <span class="author">Searle S</span>, <span class="author">Farrell CM</span>, <span class="author">Loveland JE</span>, <span class="author">Ruef BJ</span>, <span class="author">Hart E</span>, <span class="author">Suner MM</span>, <span class="author">Landrum MJ</span>, <span class="author">Aken B</span>, <span class="author">Ayling S</span>, <span class="author">Baertsch R</span>, <span class="author">Fernandez-Banet J</span>, <span class="author">Cherry JL</span>, <span class="author">Curwen V</span>, <span class="author">Dicuccio M</span>, <span class="author">Kellis M</span>, <span class="author">Lee J</span>, <span class="author">Lin MF</span>, <span class="author">Schuster M</span>, <span class="author">Shkeda A</span>, <span class="author">Amid C</span>, <span class="author">Brown G</span>, <span class="author">Dukhanina O</span>, <span class="author">Frankish A</span>, <span class="author">Hart J</span>, <span class="author">Maidak BL</span>, <span class="author">Mudge J</span>, <span class="author">Murphy MR</span>, <span class="author">Murphy T</span>, <span class="author">Rajan J</span>, <span class="author">Rajput B</span>, <span class="author">Riddick LD</span>, <span class="author">Snow C</span>, <span class="author">Steward C</span>, <span class="author">Webb D</span>, <span class="author">Weber JA</span>, <span class="author">Wilming L</span>, <span class="author">Wu W</span>, <span class="author">Birney E</span>, <span class="author">Haussler D</span>, <span class="author">Hubbard T</span>, <span class="author">Ostell J</span>, <span class="author">Durbin R</span> and <span class="author">Lipman D</span></p>
<p class="affiliation">National Center for Biotechnology Information, National Library of Medicine, Bethesda, Maryland 20894, USA. <a href="mailto:%50%72%75%69%74%74%40%6E%63%62%69.%6E%6C%6D.%6E%69%68.%67%6F%76">&#80;&#114;&#117;&#105;&#116;&#116;&#64;&#110;&#99;&#98;&#105;&#46;&#110;&#108;&#109;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">Effective use of the human and mouse genomes requires reliable identification of genes and their products. Although multiple public resources provide annotation, different methods are used that can result in similar but not identical representation of genes, transcripts, and proteins. The collaborative consensus coding sequence (CCDS) project tracks identical protein annotations on the reference mouse and human genomes with a stable identifier (CCDS ID), and ensures that they are consistently represented on the NCBI, Ensembl, and UCSC Genome Browsers. Importantly, the project coordinates on manually reviewing inconsistent protein annotations between sites, as well as annotations for which new evidence suggests a revision is needed, to progressively converge on a complete protein-coding set for the human and mouse reference genomes, while maintaining a high standard of reliability and biological accuracy. To date, the project has identified 20,159 human and 17,707 mouse consensus coding regions from 17,052 human and 16,893 mouse genes. Three evaluation methods indicate that the entries in the CCDS set are highly likely to represent real proteins, more so than annotations from contributing groups not included in CCDS. The CCDS database thus centralizes the function of identifying well-supported, identically-annotated, protein-coding regions.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1U54HG004555-01</span>, <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span>, <span class="grant">WT062023</span>, <span class="grant">WT077198</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">7</span>;<span class="pagerange">1316-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19498102">19498102</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2704439">2704439</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.080531.108">10.1101/gr.080531.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improved protocols for the illumina genome analyzer sequencing system.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Swerdlow H</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">In this unit, we describe a set of improvements we have made to the standard Illumina Genome Analyzer protocols to make the sequencing process more reliable in a high-throughput environment, reduce amplification bias, narrow the distribution of insert sizes, and reliably obtain high yields of data.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Current protocols in human genetics / editorial board, Jonathan L. Haines ... [et al.]</span>&nbsp;<span class="year">2009</span>;<span class="volume">Chapter 18</span>;<span class="pagerange">Unit 18.2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19582764">19582764</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3849550">3849550</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471142905.hg1802s62">10.1002/0471142905.hg1802s62</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.</h4>
<p class="authors"><span class="author">Ramos AM</span>, <span class="author">Crooijmans RP</span>, <span class="author">Affara NA</span>, <span class="author">Amaral AJ</span>, <span class="author">Archibald AL</span>, <span class="author">Beever JE</span>, <span class="author">Bendixen C</span>, <span class="author">Churcher C</span>, <span class="author">Clark R</span>, <span class="author">Dehais P</span>, <span class="author">Hansen MS</span>, <span class="author">Hedegaard J</span>, <span class="author">Hu ZL</span>, <span class="author">Kerstens HH</span>, <span class="author">Law AS</span>, <span class="author">Megens HJ</span>, <span class="author">Milan D</span>, <span class="author">Nonneman DJ</span>, <span class="author">Rohrer GA</span>, <span class="author">Rothschild MF</span>, <span class="author">Smith TP</span>, <span class="author">Schnabel RD</span>, <span class="author">Van Tassell CP</span>, <span class="author">Taylor JF</span>, <span class="author">Wiedmann RT</span>, <span class="author">Schook LB</span> and <span class="author">Groenen MA</span></p>
<p class="affiliation">Wageningen University, Animal Breeding and Genomics Centre, Wageningen, The Netherlands.</p><p class="abstract"><strong>Background</strong>: The dissection of complex traits of economic importance to the pig industry requires the availability of a significant number of genetic markers, such as single nucleotide polymorphisms (SNPs). This study was conducted to discover several hundreds of thousands of porcine SNPs using next generation sequencing technologies and use these SNPs, as well as others from different public sources, to design a high-density SNP genotyping assay.</p><p class="abstract">A total of 19 reduced representation libraries derived from four swine breeds (Duroc, Landrace, Large White, Pietrain) and a Wild Boar population and three restriction enzymes (AluI, HaeIII and MspI) were sequenced using Illumina&#39;s Genome Analyzer (GA). The SNP discovery effort resulted in the de novo identification of over 372K SNPs. More than 549K SNPs were used to design the Illumina Porcine 60K+SNP iSelect Beadchip, now commercially available as the PorcineSNP60. A total of 64,232 SNPs were included on the Beadchip. Results from genotyping the 158 individuals used for sequencing showed a high overall SNP call rate (97.5%). Of the 62,621 loci that could be reliably scored, 58,994 were polymorphic yielding a SNP conversion success rate of 94%. The average minor allele frequency (MAF) for all scorable SNPs was 0.274.</p><p class="abstract">Overall, the results of this study indicate the utility of using next generation sequencing technologies to identify large numbers of reliable SNPs. In addition, the validation of the PorcineSNP60 Beadchip demonstrated that the assay is an excellent tool that will likely be used in a variety of future studies in pigs.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">8</span>;<span class="pagerange">e6524</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19654876" rel="external">19654876</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716536" rel="external">2716536</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0006524" rel="external">10.1371/journal.pone.0006524</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study of testicular germ cell tumor.</h4>
<p class="authors"><span class="author">Rapley EA</span>, <span class="author">Turnbull C</span>, <span class="author">Al Olama AA</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Linger R</span>, <span class="author">Huddart RA</span>, <span class="author">Renwick A</span>, <span class="author">Hughes D</span>, <span class="author">Hines S</span>, <span class="author">Seal S</span>, <span class="author">Morrison J</span>, <span class="author">Nsengimana J</span>, <span class="author">Deloukas P</span>, <span class="author">UK Testicular Cancer Collaboration</span>, <span class="author">Rahman N</span>, <span class="author">Bishop DT</span>, <span class="author">Easton DF</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK.</p><p class="abstract">We conducted a genome-wide association study for testicular germ cell tumor (TGCT), genotyping 307,666 SNPs in 730 cases and 1,435 controls from the UK and replicating associations in a further 571 cases and 1,806 controls. We found strong evidence for susceptibility loci on chromosome 5 (per allele OR = 1.37 (95% CI = 1.19-1.58), P = 3 x 10(-13)), chromosome 6 (OR = 1.50 (95% CI = 1.28-1.75), P = 10(-13)) and chromosome 12 (OR = 2.55 (95% CI = 2.05-3.19), P = 10(-31)). KITLG, encoding the ligand for the receptor tyrosine kinase KIT, which has previously been implicated in the pathogenesis of TGCT and the biology of germ cells, may explain the association on chromosome 12.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11022</span>, <span class="grant">A4994</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0700491</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">077012</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">7</span>;<span class="pagerange">807-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19483681">19483681</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871592">2871592</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.394">10.1038/ng.394</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A large and accurate collection of peptidase cleavages in the MEROPS database.</h4>
<p class="authors"><span class="author">Rawlings ND</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Peptidases are enzymes that hydrolyse peptide bonds in proteins and peptides. Peptidases are important in pathological conditions such as Alzheimer&#39;s disease, tumour and parasite invasion, and for processing viral polyproteins. The MEROPS database is an Internet resource containing information on peptidases, their substrates and inhibitors. The database now includes details of cleavage positions in substrates, both physiological and non-physiological, natural and synthetic. There are 39 118 cleavages in the collection; including 34 606 from a total of 10 513 different proteins and 2677 cleavages in synthetic substrates. The number of cleavages designated as &#39;physiological&#39; is 13 307. The data are derived from 6095 publications. At least one substrate cleavage is known for 45% of the 2415 different peptidases recognized in the MEROPS database. The website now has three new displays: two showing peptidase specificity as a logo and a frequency matrix, the third showing a dynamically generated alignment between each protein substrate and its most closely related homologues. Many of the proteins described in the literature as peptidase substrates have been studied only in vitro. On the assumption that a physiologically relevant cleavage site would be conserved between species, the conservation of every site in terms of peptidase preference has been examined and a number have been identified that are not conserved. There are a number of cogent reasons why a site might not be conserved. Each poorly conserved site has been examined and a reason postulated. Some sites are identified that are very poorly conserved where cleavage is more likely to be fortuitous than of physiological relevance. This data-set is freely available via the Internet and is a useful training set for algorithms to predict substrates for peptidases and cleavage positions within those substrates. The data may also be useful for the design of inhibitors and for engineering novel specificities into peptidases.Database URL:http://merops.sanger.ac.uk.</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2009</span>;<span class="volume">2009</span>;<span class="pagerange">bap015</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20157488" rel="external">20157488</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2790309" rel="external">2790309</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbap015" rel="external">10.1093/database/bap015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lessons learnt from large-scale exon re-sequencing of the X chromosome.</h4>
<p class="authors"><span class="author">Raymond FL</span>, <span class="author">Whibley A</span>, <span class="author">Stratton MR</span> and <span class="author">Gecz J</span></p>
<p class="affiliation">Cambridge Institute of Medical Research, University of Cambridge, Cambridge, UK. <a href="mailto:%66%6C%72%32%34%40%63%61%6D.%61%63.%75%6B">&#102;&#108;&#114;&#50;&#52;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A candidate gene approach to identifying novel causes of disease is concept-limiting and in the new era of high throughput sequencing there is now no need to restrict the experiment to a few interesting genes. We have recently completed a large-scale exon re-sequencing project using Sanger sequencing technology to analyse approximately 1 Mb of coding sequence of the X chromosome in probands from &gt;200 families with various forms of intellectual disability. We review the lessons learnt from this experience. Comparing large data sets will certainly reveal pathogenic mutations in genes that were not possible to identify previously. However, the task of distinguishing pathogenic mutations from rare sequence variants is not easy and is the most substantial challenge to the next decade. High-throughput technology has the attraction of being cheap, fast and comprehensive but for projects that require detailed coverage of a genomic region at an exhaustive level they may require a combination of large-scale with a small-scale follow-up of difficult regions to sequence. The number of rare truncating variants present in coding regions of the X chromosome that are not pathogenic was 1%. The importance of the quality of the starting material both clinically and molecularly and the number of sequence variants both rare and common that any one individual has across their coding sequence is discussed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">R1</span>;<span class="pagerange">R60-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19297402" rel="external">19297402</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2657946" rel="external">2657946</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp071" rel="external">10.1093/hmg/ddp071</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic hybridization: microarray design and data interpretation.</h4>
<p class="authors"><span class="author">Redon R</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust, Sanger Institute, Cambridge, UK.</p><p class="abstract">Microarray-based Comparative Genomic Hybridization (array-CGH) has been applied for a decade to screen for submicroscopic DNA gains and losses in tumor and constitutional DNA samples. This method has become increasingly flexible with the integration of new biological resources generated by genome sequencing projects. In this chapter, we describe alternative strategies for whole genome screening and high resolution breakpoint mapping of copy number changes by array-CGH, as well as tools available for accurate analysis of array-CGH experiments. Although most methods listed here have been designed for microarrays comprising large-insert clones, they can be adapted easily to other types of microarray platforms, such as those constructed from printed or synthesized oligonucleotides.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">529</span>;<span class="pagerange">37-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19381971" rel="external">19381971</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871310" rel="external">2871310</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-59745-538-1_3" rel="external">10.1007/978-1-59745-538-1_3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic hybridization: DNA labeling, hybridization and detection.</h4>
<p class="authors"><span class="author">Redon R</span>, <span class="author">Fitzgerald T</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust, Sanger Institute, Cambridge, UK.</p><p class="abstract">Array-CGH involves the comparison of a test to a reference genome using a microarray composed of target sequences with known chromosomal coordinates. The test and reference DNA samples are used as templates to generate two probe DNAs labeled with distinct fluorescent dyes. The two probe DNAs are co-hybridized on a microarray in the presence of Cot-1 DNA to suppress unspecific hybridization of repeat sequences. After slide washes and drying, microarray images are acquired on a laser scanner and fluorescent intensities from every target sequence spot on the array are extracted using dedicated computer programs. Intensity ratios are calculated and normalized to enable data interpretation. Although the protocols explained in this chapter correspond primarily to the use of large-insert clone microarrays in either manual or automated fashion, necessary adaptations for hybridization on microarrays comprising shorter target DNA sequences are also briefly described.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">529</span>;<span class="pagerange">267-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19381974" rel="external">19381974</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2867219" rel="external">2867219</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-59745-538-1_17" rel="external">10.1007/978-1-59745-538-1_17</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic hybridization: DNA preparation for microarray fabrication.</h4>
<p class="authors"><span class="author">Redon R</span>, <span class="author">Rigler D</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust, Sanger Institute, Cambridge, UK.</p><p class="abstract">The spatial resolution of microarray-based comparative genomic hybridization (array-CGH) is dependent on the length and density of target DNA sequences covering the chromosomal region of interest. Here we describe the methods developed at the Wellcome Trust Sanger Institute (Cambridge, UK) to construct microarrays comprising large-insert clones available through genome sequencing projects. These methods are applicable to Bacterial and Phage Artificial Chromosomes (BAC and PAC) as well as fosmid and cosmid clones. The protocols are scalable for the construction of microarrays composed of several hundreds up to several ten thousands clones.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">529</span>;<span class="pagerange">259-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19381975" rel="external">19381975</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2867217" rel="external">2867217</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-59745-538-1_16" rel="external">10.1007/978-1-59745-538-1_16</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Gene Ontology&#39;s Reference Genome Project: a unified framework for functional annotation across species.</h4>
<p class="authors"><span class="author">Reference Genome Group of the Gene Ontology Consortium</span></p><p class="abstract">The Gene Ontology (GO) is a collaborative effort that provides structured vocabularies for annotating the molecular function, biological role, and cellular location of gene products in a highly systematic way and in a species-neutral manner with the aim of unifying the representation of gene function across different organisms. Each contributing member of the GO Consortium independently associates GO terms to gene products from the organism(s) they are annotating. Here we introduce the Reference Genome project, which brings together those independent efforts into a unified framework based on the evolutionary relationships between genes in these different organisms. The Reference Genome project has two primary goals: to increase the depth and breadth of annotations for genes in each of the organisms in the project, and to create data sets and tools that enable other genome annotation efforts to infer GO annotations for homologous genes in their organisms. In addition, the project has several important incidental benefits, such as increasing annotation consistency across genome databases, and providing important improvements to the GO&#39;s logical structure and biological content.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">SP/07/007/23671</span>; Medical Research Council: <span class="grant">G0500293</span>; NHGRI NIH HHS: <span class="grant">HG00022</span>, <span class="grant">P41 HG000330</span>, <span class="grant">P41 HG001315</span>, <span class="grant">P41 HG002273</span>, <span class="grant">P41 HG002659-06</span>, <span class="grant">P41 HG02223</span>; NHLBI NIH HHS: <span class="grant">HL64541</span>; NICHD NIH HHS: <span class="grant">HD033745</span>; NIGMS NIH HHS: <span class="grant">GM64426</span>, <span class="grant">U24 GM07790</span></p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">e1000431</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19578431" rel="external">19578431</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2699109" rel="external">2699109</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.1000431" rel="external">10.1371/journal.pcbi.1000431</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Allelic variants in HTR3C show association with autism.</h4>
<p class="authors"><span class="author">Rehnstr&ouml;m K</span>, <span class="author">Ylisaukko-oja T</span>, <span class="author">Nummela I</span>, <span class="author">Ellonen P</span>, <span class="author">Kempas E</span>, <span class="author">Vanhala R</span>, <span class="author">von Wendt L</span>, <span class="author">J&auml;rvel&auml; I</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Molecular Medicine, National Public Health Institute and Institute for Molecular Medicine Finland, Helsinki, Finland. <a href="mailto:%6B%61%72%6F%6C%61.%72%65%68%6E%73%74%72%6F%6D%40%6B%74%6C.%66%69">&#107;&#97;&#114;&#111;&#108;&#97;&#46;&#114;&#101;&#104;&#110;&#115;&#116;&#114;&#111;&#109;&#64;&#107;&#116;&#108;&#46;&#102;&#105;</a></p><p class="abstract">Autism spectrum disorders (ASDs) are severe neurodevelopmental disorders with a strong genetic component. Only a few predisposing genes have been identified so far. We have previously performed a genome-wide linkage screen for ASDs in Finnish families where the most significant linkage peak was identified at 3q25-27. Here, 11 positional and functionally relevant candidate genes at 3q25-27 were tested for association with autistic disorder. Genotypes of 125 single nucleotide polymorphisms (SNPs) were determined in 97 families with at least one individual affected with autistic disorder. The most significant association was observed using two non-synonymous SNPs in HTR3C, rs6766410 and rs6807362, both resulting in P = 0.0012 in family-based association analysis. In addition, the haplotype C-C corresponding to amino acids N163-A405 was overtransmitted to affected individuals (P = 0.006). Sequencing revealed no other variants in the coding region or splice sites of HTR3C. Based on the association analysis results in a previously identified linkage region, we propose that HTR3C represents a novel candidate locus for ASDs and should be tested in other populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">American journal of medical genetics. Part B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric Genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">150B</span>;<span class="number">5</span>;<span class="pagerange">741-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19035560" rel="external">19035560</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2703778" rel="external">2703778</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.b.30882" rel="external">10.1002/ajmg.b.30882</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An essential role for the Plasmodium Nek-2 Nima-related protein kinase in the sexual development of malaria parasites.</h4>
<p class="authors"><span class="author">Reininger L</span>, <span class="author">Tewari R</span>, <span class="author">Fennell C</span>, <span class="author">Holland Z</span>, <span class="author">Goldring D</span>, <span class="author">Ranford-Cartwright L</span>, <span class="author">Billker O</span> and <span class="author">Doerig C</span></p>
<p class="affiliation">INSERM U609-Wellcome Centre for Molecular Parasitology, Biomedical Research Centre, Faculty of Biomedical and Life Sciences, University of Glasgow, 120 University Place, Glasgow G12 8TA, Scotland.</p><p class="abstract">The molecular control of cell division and development in malaria parasites is far from understood. We previously showed that a Plasmodium gametocyte-specific NIMA-related protein kinase, nek-4, is required for completion of meiosis in the ookinete, the motile form that develops from the zygote in the mosquito vector. Here, we show that another NIMA-related kinase, Pfnek-2, is also predominantly expressed in gametocytes, and that Pfnek-2 is an active enzyme displaying an in vitro substrate preference distinct from that of Pfnek-4. A functional nek-2 gene is required for transmission of both Plasmodium falciparum and the rodent malaria parasite Plasmodium berghei to the mosquito vector, which is explained by the observation that disruption of the nek-2 gene in P. berghei causes dysregulation of DNA replication during meiosis and blocks ookinete development. This has implications (i) in our understanding of sexual development of malaria parasites and (ii) in the context of control strategies aimed at interfering with malaria transmission.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2009</span>;<span class="volume">284</span>;<span class="number">31</span>;<span class="pagerange">20858-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19491095" rel="external">19491095</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2742851" rel="external">2742851</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M109.017988" rel="external">10.1074/jbc.M109.017988</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication and extension of genome-wide association study results for obesity in 4923 adults from northern Sweden.</h4>
<p class="authors"><span class="author">Renstr&ouml;m F</span>, <span class="author">Payne F</span>, <span class="author">Nordstr&ouml;m A</span>, <span class="author">Brito EC</span>, <span class="author">Rolandsson O</span>, <span class="author">Hallmans G</span>, <span class="author">Barroso I</span>, <span class="author">Nordstr&ouml;m P</span>, <span class="author">Franks PW</span> and <span class="author">GIANT Consortium</span></p>
<p class="affiliation">Department of Public Health and Clinical Medicine, Ume&aring; University Hospital, Ume&aring;, Sweden.</p><p class="abstract">Recent genome-wide association studies (GWAS) have identified multiple risk loci for common obesity (FTO, MC4R, TMEM18, GNPDA2, SH2B1, KCTD15, MTCH2, NEGR1 and PCSK1). Here we extend those studies by examining associations with adiposity and type 2 diabetes in Swedish adults. The nine single nucleotide polymorphisms (SNPs) were genotyped in 3885 non-diabetic and 1038 diabetic individuals with available measures of height, weight and body mass index (BMI). Adipose mass and distribution were objectively assessed using dual-energy X-ray absorptiometry in a sub-group of non-diabetics (n = 2206). In models with adipose mass traits, BMI or obesity as outcomes, the most strongly associated SNP was FTO rs1121980 (P &lt; 0.001). Five other SNPs (SH2B1 rs7498665, MTCH2 rs4752856, MC4R rs17782313, NEGR1 rs2815752 and GNPDA2 rs10938397) were significantly associated with obesity. To summarize the overall genetic burden, a weighted risk score comprising a subset of SNPs was constructed; those in the top quintile of the score were heavier (+2.6 kg) and had more total (+2.4 kg), gynoid (+191 g) and abdominal (+136 g) adipose tissue than those in the lowest quintile (all P &lt; 0.001). The genetic burden score significantly increased diabetes risk, with those in the highest quintile (n = 193/594 cases/controls) being at 1.55-fold (95% CI 1.21-1.99; P &lt; 0.0001) greater risk of type 2 diabetes than those in the lowest quintile (n = 130/655 cases/controls). In summary, we have statistically replicated six of the previously associated obese-risk loci and our results suggest that the weight-inducing effects of these variants are explained largely by increased adipose accumulation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">8</span>;<span class="pagerange">1489-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19164386">19164386</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2664142">2664142</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp041">10.1093/hmg/ddp041</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic analysis of ten Streptococcus pneumoniae temperate bacteriophages.</h4>
<p class="authors"><span class="author">Romero P</span>, <span class="author">Croucher NJ</span>, <span class="author">Hiller NL</span>, <span class="author">Hu FZ</span>, <span class="author">Ehrlich GD</span>, <span class="author">Bentley SD</span>, <span class="author">Garc&iacute;a E</span> and <span class="author">Mitchell TJ</span></p>
<p class="affiliation">Division of Infection and Immunity, Glasgow Biomedical Research Centre, University of Glasgow, Glasgow, United Kingdom.</p><p class="abstract">Streptococcus pneumoniae is an important human pathogen that often carries temperate bacteriophages. As part of a program to characterize the genetic makeup of prophages associated with clinical strains and to assess the potential roles that they play in the biology and pathogenesis in their host, we performed comparative genomic analysis of 10 temperate pneumococcal phages. All of the genomes are organized into five major gene clusters: lysogeny, replication, packaging, morphogenesis, and lysis clusters. All of the phage particles observed showed a Siphoviridae morphology. The only genes that are well conserved in all the genomes studied are those involved in the integration and the lysis of the host in addition to two genes, of unknown function, within the replication module. We observed that a high percentage of the open reading frames contained no similarities to any sequences catalogued in public databases; however, genes that were homologous to known phage virulence genes, including the pblB gene of Streptococcus mitis and the vapE gene of Dichelobacter nodosus, were also identified. Interestingly, bioinformatic tools showed the presence of a toxin-antitoxin system in the phage phiSpn_6, and this represents the first time that an addition system in a pneumophage has been identified. Collectively, the temperate pneumophages contain a diverse set of genes with various levels of similarity among them.</p>
<p class="grants">Funded by: NIDCD NIH HHS: <span class="grant">DC02148</span>, <span class="grant">DC04173</span>, <span class="grant">DC05659</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">15</span>;<span class="pagerange">4854-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19502408">19502408</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2715734">2715734</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01272-08">10.1128/JB.01272-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Partial lipodystrophy and insulin resistant diabetes in a patient with a homozygous nonsense mutation in CIDEC.</h4>
<p class="authors"><span class="author">Rubio-Cabezas O</span>, <span class="author">Puri V</span>, <span class="author">Murano I</span>, <span class="author">Saudek V</span>, <span class="author">Semple RK</span>, <span class="author">Dash S</span>, <span class="author">Hyden CS</span>, <span class="author">Bottomley W</span>, <span class="author">Vigouroux C</span>, <span class="author">Magr&eacute; J</span>, <span class="author">Raymond-Barker P</span>, <span class="author">Murgatroyd PR</span>, <span class="author">Chawla A</span>, <span class="author">Skepper JN</span>, <span class="author">Chatterjee VK</span>, <span class="author">Suliman S</span>, <span class="author">Patch AM</span>, <span class="author">Agarwal AK</span>, <span class="author">Garg A</span>, <span class="author">Barroso I</span>, <span class="author">Cinti S</span>, <span class="author">Czech MP</span>, <span class="author">Argente J</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Savage DB</span> and <span class="author">LD Screening Consortium</span></p>
<p class="affiliation">Department of Endocrinology, Hospital Infantil Universitario Ni&ntilde;o Jes&uacute;s, Madrid, Spain.</p><p class="abstract">Lipodystrophic syndromes are characterized by adipose tissue deficiency. Although rare, they are of considerable interest as they, like obesity, typically lead to ectopic lipid accumulation, dyslipidaemia and insulin resistant diabetes. In this paper we describe a female patient with partial lipodystrophy (affecting limb, femorogluteal and subcutaneous abdominal fat), white adipocytes with multiloculated lipid droplets and insulin-resistant diabetes, who was found to be homozygous for a premature truncation mutation in the lipid droplet protein cell death-inducing Dffa-like effector C (CIDEC) (E186X). The truncation disrupts the highly conserved CIDE-C domain and the mutant protein is mistargeted and fails to increase the lipid droplet size in transfected cells. In mice, Cidec deficiency also reduces fat mass and induces the formation of white adipocytes with multilocular lipid droplets, but in contrast to our patient, Cidec null mice are protected against diet-induced obesity and insulin resistance. In addition to describing a novel autosomal recessive form of familial partial lipodystrophy, these observations also suggest that CIDEC is required for unilocular lipid droplet formation and optimal energy storage in human fat.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600414</span>; NIDDK NIH HHS: <span class="grant">DK30898</span>, <span class="grant">DK32520</span>, <span class="grant">DK54387</span>, <span class="grant">DK60837</span>, <span class="grant">P30 DK032520</span>, <span class="grant">P30 DK032520-25</span>, <span class="grant">P30 DK032520-26</span>, <span class="grant">R01 DK054387</span>, <span class="grant">R37 DK030898</span>, <span class="grant">R37 DK030898-23</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span></p>
<p class="publication"><span class="title">EMBO molecular medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">1</span>;<span class="number">5</span>;<span class="pagerange">280-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20049731">20049731</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2891108">2891108</a>; DOI: <a href="http://dx.doi.org/10.1002%2Femmm.200900037">10.1002/emmm.200900037</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The versatility and adaptation of bacteria from the genus Stenotrophomonas.</h4>
<p class="authors"><span class="author">Ryan RP</span>, <span class="author">Monchy S</span>, <span class="author">Cardinale M</span>, <span class="author">Taghavi S</span>, <span class="author">Crossman L</span>, <span class="author">Avison MB</span>, <span class="author">Berg G</span>, <span class="author">van der Lelie D</span> and <span class="author">Dow JM</span></p>
<p class="affiliation">BIOMERIT Research Centre, Department of Microbiology, BioSciences Institute, University College Cork, Cork, Ireland. <a href="mailto:%72.%72%79%61%6E%40%75%63%63.%69%65">&#114;&#46;&#114;&#121;&#97;&#110;&#64;&#117;&#99;&#99;&#46;&#105;&#101;</a></p><p class="abstract">The genus Stenotrophomonas comprises at least eight species. These bacteria are found throughout the environment, particularly in close association with plants. Strains of the most predominant species, Stenotrophomonas maltophilia, have an extraordinary range of activities that include beneficial effects for plant growth and health, the breakdown of natural and man-made pollutants that are central to bioremediation and phytoremediation strategies and the production of biomolecules of economic value, as well as detrimental effects, such as multidrug resistance, in human pathogenic strains. Here, we discuss the versatility of the bacteria in the genus Stenotrophomonas and the insight that comparative genomic analysis of clinical and endophytic isolates of S. maltophilia has brought to our understanding of the adaptation of this genus to various niches.</p>
<p class="grants">Funded by: Austrian Science Fund FWF: <span class="grant">P 20542-B16</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">514-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19528958">19528958</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2163">10.1038/nrmicro2163</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Presence of interstereocilial links in waltzer mutants suggests Cdh23 is not essential for tip link formation.</h4>
<p class="authors"><span class="author">Rzadzinska AK</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kingdom.</p><p class="abstract">Cadherin23 has been proposed to form the upper part of the tip link, an interstereocilial link believed to control opening of transducer channels of sensory hair cells. However, we detect tip link-like links in mouse mutants with null alleles of Cdh23, suggesting the presence of other components that permit formation of a link between the tip of one stereocilium and the side of the adjacent taller stereocilium.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Neuroscience</span>&nbsp;<span class="year">2009</span>;<span class="volume">158</span>;<span class="number">2</span>;<span class="pagerange">365-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18996172">18996172</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2989438">2989438</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neuroscience.2008.10.012">10.1016/j.neuroscience.2008.10.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis of metabolic traits in a birth cohort from a founder population.</h4>
<p class="authors"><span class="author">Sabatti C</span>, <span class="author">Service SK</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">Ripatti S</span>, <span class="author">Brodsky J</span>, <span class="author">Jones CG</span>, <span class="author">Zaitlen NA</span>, <span class="author">Varilo T</span>, <span class="author">Kaakinen M</span>, <span class="author">Sovio U</span>, <span class="author">Ruokonen A</span>, <span class="author">Laitinen J</span>, <span class="author">Jakkula E</span>, <span class="author">Coin L</span>, <span class="author">Hoggart C</span>, <span class="author">Collins A</span>, <span class="author">Turunen H</span>, <span class="author">Gabriel S</span>, <span class="author">Elliot P</span>, <span class="author">McCarthy MI</span>, <span class="author">Daly MJ</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Freimer NB</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Human Genetics and Los Angeles, Los Angeles, California 90095, USA.</p><p class="abstract">Genome-wide association studies (GWAS) of longitudinal birth cohorts enable joint investigation of environmental and genetic influences on complex traits. We report GWAS results for nine quantitative metabolic traits (triglycerides, high-density lipoprotein, low-density lipoprotein, glucose, insulin, C-reactive protein, body mass index, and systolic and diastolic blood pressure) in the Northern Finland Birth Cohort 1966 (NFBC1966), drawn from the most genetically isolated Finnish regions. We replicate most previously reported associations for these traits and identify nine new associations, several of which highlight genes with metabolic functions: high-density lipoprotein with NR1H3 (LXRA), low-density lipoprotein with AR and FADS1-FADS2, glucose with MTNR1B, and insulin with PANK1. Two of these new associations emerged after adjustment of results for body mass index. Gene-environment interaction analyses suggested additional associations, which will require validation in larger samples. The currently identified loci, together with quantified environmental exposures, explain little of the trait variation in NFBC1966. The association observed between low-density lipoprotein and an infrequent variant in AR suggests the potential of such a cohort for identifying associations with both common, low-impact and rarer, high-impact quantitative trait loci.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">U54 RR020278</span>; NIGMS NIH HHS: <span class="grant">GM053275-14</span>; NIMH NIH HHS: <span class="grant">MH083268</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">1</span>;<span class="pagerange">35-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19060910">19060910</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687077">2687077</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.271">10.1038/ng.271</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Pakistan Risk of Myocardial Infarction Study: a resource for the study of genetic, lifestyle and other determinants of myocardial infarction in South Asia.</h4>
<p class="authors"><span class="author">Saleheen D</span>, <span class="author">Zaidi M</span>, <span class="author">Rasheed A</span>, <span class="author">Ahmad U</span>, <span class="author">Hakeem A</span>, <span class="author">Murtaza M</span>, <span class="author">Kayani W</span>, <span class="author">Faruqui A</span>, <span class="author">Kundi A</span>, <span class="author">Zaman KS</span>, <span class="author">Yaqoob Z</span>, <span class="author">Cheema LA</span>, <span class="author">Samad A</span>, <span class="author">Rasheed SZ</span>, <span class="author">Mallick NH</span>, <span class="author">Azhar M</span>, <span class="author">Jooma R</span>, <span class="author">Gardezi AR</span>, <span class="author">Memon N</span>, <span class="author">Ghaffar A</span>, <span class="author">Fazal-ur-Rehman</span>, <span class="author">Khan N</span>, <span class="author">Shah N</span>, <span class="author">Ali Shah A</span>, <span class="author">Samuel M</span>, <span class="author">Hanif F</span>, <span class="author">Yameen M</span>, <span class="author">Naz S</span>, <span class="author">Sultana A</span>, <span class="author">Nazir A</span>, <span class="author">Raza S</span>, <span class="author">Shazad M</span>, <span class="author">Nasim S</span>, <span class="author">Javed MA</span>, <span class="author">Ali SS</span>, <span class="author">Jafree M</span>, <span class="author">Nisar MI</span>, <span class="author">Daood MS</span>, <span class="author">Hussain A</span>, <span class="author">Sarwar N</span>, <span class="author">Kamal A</span>, <span class="author">Deloukas P</span>, <span class="author">Ishaq M</span>, <span class="author">Frossard P</span> and <span class="author">Danesh J</span></p>
<p class="affiliation">Center for Non-Communicable Diseases, Karachi, Pakistan. <a href="mailto:%64%73%34%33%36%40%6D%65%64%73%63%68%6C.%63%61%6D.%61%63.%75%6B">&#100;&#115;&#52;&#51;&#54;&#64;&#109;&#101;&#100;&#115;&#99;&#104;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The burden of coronary heart disease (CHD) is increasing at a greater rate in South Asia than in any other region globally, but there is little direct evidence about its determinants. The Pakistan Risk of Myocardial Infarction Study (PROMIS) is an epidemiological resource to enable reliable study of genetic, lifestyle and other determinants of CHD in South Asia. By March 2009, PROMIS had recruited over 5,000 cases of first-ever confirmed acute myocardial infarction (MI) and over 5,000 matched controls aged 30-80 years. For each participant, information has been recorded on demographic factors, lifestyle, medical and family history, anthropometry, and a 12-lead electrocardiogram. A range of biological samples has been collected and stored, including DNA, plasma, serum and whole blood. During its next stage, the study aims to expand recruitment to achieve a total of about 20,000 cases and about 20,000 controls, and, in subsets of participants, to enrich the resource by collection of monocytes, establishment of lymphoblastoid cell lines, and by resurveying participants. Measurements in progress include profiling of candidate biochemical factors, assay of 45,000 variants in 2,100 candidate genes, and a genomewide association scan of over 650,000 genetic markers. We have established a large epidemiological resource for CHD in South Asia. In parallel with its further expansion and enrichment, the PROMIS resource will be systematically harvested to help identify and evaluate genetic and other determinants of MI in South Asia. Findings from this study should advance scientific understanding and inform regionally appropriate disease prevention and control strategies.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council; Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">European journal of epidemiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">24</span>;<span class="number">6</span>;<span class="pagerange">329-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19404752" rel="external">19404752</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2697028" rel="external">2697028</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10654-009-9334-y" rel="external">10.1007/s10654-009-9334-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Schistosoma japonicum genome reveals features of host-parasite interplay.</h4>
<p class="authors"><span class="author">Schistosoma japonicum Genome Sequencing and Functional Analysis Consortium</span></p><p class="abstract">Schistosoma japonicum is a parasitic flatworm that causes human schistosomiasis, which is a significant cause of morbidity in China and the Philippines. Here we present a draft genomic sequence for the worm. The genome provides a global insight into the molecular architecture and host interaction of this complex metazoan pathogen, revealing that it can exploit host nutrients, neuroendocrine hormones and signalling pathways for growth, development and maturation. Having a complex nervous system and a well-developed sensory system, S. japonicum can accept stimulation of the corresponding ligands as a physiological response to different environments, such as fresh water or the tissues of its intermediate and mammalian hosts. Numerous proteases, including cercarial elastase, are implicated in mammalian skin penetration and haemoglobin degradation. The genomic information will serve as a valuable platform to facilitate development of new interventions for schistosomiasis control.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI39461</span>; Wellcome Trust: <span class="grant">085775</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2009</span>;<span class="volume">460</span>;<span class="number">7253</span>;<span class="pagerange">345-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19606140">19606140</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3747554">3747554</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08140">10.1038/nature08140</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression analysis of the Theileria parva subtelomere-encoded variable secreted protein gene family.</h4>
<p class="authors"><span class="author">Schmuckli-Maurer J</span>, <span class="author">Casanova C</span>, <span class="author">Schmied S</span>, <span class="author">Affentranger S</span>, <span class="author">Parvanova I</span>, <span class="author">Kang&#39;a S</span>, <span class="author">Nene V</span>, <span class="author">Katzer F</span>, <span class="author">McKeever D</span>, <span class="author">M&uuml;ller J</span>, <span class="author">Bishop R</span>, <span class="author">Pain A</span> and <span class="author">Dobbelaere DA</span></p>
<p class="affiliation">Molecular Pathobiology, Vetsuisse Faculty, University of Bern, Bern, Switzerland.</p><p class="abstract"><strong>Background</strong>: The intracellular protozoan parasite Theileria parva transforms bovine lymphocytes inducing uncontrolled proliferation. Proteins released from the parasite are assumed to contribute to phenotypic changes of the host cell and parasite persistence. With 85 members, genes encoding subtelomeric variable secreted proteins (SVSPs) form the largest gene family in T. parva. The majority of SVSPs contain predicted signal peptides, suggesting secretion into the host cell cytoplasm.</p><p class="abstract">We analysed SVSP expression in T. parva-transformed cell lines established in vitro by infection of T or B lymphocytes with cloned T. parva parasites. Microarray and quantitative real-time PCR analysis revealed mRNA expression for a wide range of SVSP genes. The pattern of mRNA expression was largely defined by the parasite genotype and not by host background or cell type, and found to be relatively stable in vitro over a period of two months. Interestingly, immunofluorescence analysis carried out on cell lines established from a cloned parasite showed that expression of a single SVSP encoded by TP03_0882 is limited to only a small percentage of parasites. Epitope-tagged TP03_0882 expressed in mammalian cells was found to translocate into the nucleus, a process that could be attributed to two different nuclear localisation signals.</p><p class="abstract"><strong>Conclusions</strong>: Our analysis reveals a complex pattern of Theileria SVSP mRNA expression, which depends on the parasite genotype. Whereas in cell lines established from a cloned parasite transcripts can be found corresponding to a wide range of SVSP genes, only a minority of parasites appear to express a particular SVSP protein. The fact that a number of SVSPs contain functional nuclear localisation signals suggests that proteins released from the parasite could contribute to phenotypic changes of the host cell. This initial characterisation will facilitate future studies on the regulation of SVSP gene expression and the potential biological role of these enigmatic proteins.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">064654</span>, <span class="grant">077431</span>, <span class="grant">GR075820MA</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">3</span>;<span class="pagerange">e4839</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19325907" rel="external">19325907</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2657828" rel="external">2657828</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0004839" rel="external">10.1371/journal.pone.0004839</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome flexibility in Neisseria meningitidis.</h4>
<p class="authors"><span class="author">Schoen C</span>, <span class="author">Tettelin H</span>, <span class="author">Parkhill J</span> and <span class="author">Frosch M</span></p>
<p class="affiliation">Institut f&uuml;r Hygiene und Mikrobiologie, der Universit&auml;t W&uuml;rzburg, Josef-Schneider-Strasse 2, Bau E1, W&uuml;rzburg 97877, Germany. <a href="mailto:%63%73%63%68%6F%65%6E%40%68%79%67%69%65%6E%65.%75%6E%69-%77%75%65%72%7A%62%75%72%67.%64%65">&#99;&#115;&#99;&#104;&#111;&#101;&#110;&#64;&#104;&#121;&#103;&#105;&#101;&#110;&#101;&#46;&#117;&#110;&#105;&#45;&#119;&#117;&#101;&#114;&#122;&#98;&#117;&#114;&#103;&#46;&#100;&#101;</a></p><p class="abstract">Neisseria meningitidis usually lives as a commensal bacterium in the upper airways of humans. However, occasionally some strains can also cause life-threatening diseases such as sepsis and bacterial meningitis. Comparative genomics demonstrates that only very subtle genetic differences between carriage and disease strains might be responsible for the observed virulence differences and that N. meningitidis is, evolutionarily, a very recent species. Comparative genome sequencing also revealed a panoply of genetic mechanisms underlying its enormous genomic flexibility which also might affect the virulence of particular strains. From these studies, N. meningitidis emerges as a paradigm for organisms that use genome variability as an adaptation to changing and thus challenging environments.</p>
<p class="publication"><span class="title">Vaccine</span>&nbsp;<span class="year">2009</span>;<span class="volume">27 Suppl 2</span>;<span class="pagerange">B103-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19477564">19477564</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3898611">3898611</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vaccine.2009.04.064">10.1016/j.vaccine.2009.04.064</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence and analysis of a plasmid-encoded mercury resistance operon from Mycobacterium marinum identifies MerH, a new mercuric ion transporter.</h4>
<p class="authors"><span class="author">Schu&eacute; M</span>, <span class="author">Dover LG</span>, <span class="author">Besra GS</span>, <span class="author">Parkhill J</span> and <span class="author">Brown NL</span></p>
<p class="affiliation">School of Biosciences, The University of Birmingham, Edgbaston, Birmingham, B15 2TT, United Kingdom. <a href="mailto:%6D%61%74%73%63%68%75%65%40%79%61%68%6F%6F.%66%72">&#109;&#97;&#116;&#115;&#99;&#104;&#117;&#101;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#102;&#114;</a></p><p class="abstract">In this study, we report the DNA sequence and biological analysis of a mycobacterial mercury resistance operon encoding a novel Hg(2+) transporter. MerH was found to transport mercuric ions in Escherichia coli via a pair of essential cysteine residues but only when coexpressed with the mercuric reductase.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">1</span>;<span class="pagerange">439-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18931130" rel="external">18931130</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2612448" rel="external">2612448</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01063-08" rel="external">10.1128/JB.01063-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The key role of genomics in modern vaccine and drug design for emerging infectious diseases.</h4>
<p class="authors"><span class="author">Seib KL</span>, <span class="author">Dougan G</span> and <span class="author">Rappuoli R</span></p>
<p class="affiliation">Novartis Vaccines and Diagnostics, Siena, Italy.</p><p class="abstract">It can be argued that the arrival of the &quot;genomics era&quot; has significantly shifted the paradigm of vaccine and therapeutics development from microbiological to sequence-based approaches. Genome sequences provide a previously unattainable route to investigate the mechanisms that underpin pathogenesis. Genomics, transcriptomics, metabolomics, structural genomics, proteomics, and immunomics are being exploited to perfect the identification of targets, to design new vaccines and drugs, and to predict their effects in patients. Furthermore, human genomics and related studies are providing insights into aspects of host biology that are important in infectious disease. This ever-growing body of genomic data and new genome-based approaches will play a critical role in the future to enable timely development of vaccines and therapeutics to control emerging infectious diseases.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">e1000612</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19855822" rel="external">19855822</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2752168" rel="external">2752168</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000612" rel="external">10.1371/journal.pgen.1000612</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome watch: breaking the ICE.</h4>
<p class="authors"><span class="author">Seth-Smith H</span> and <span class="author">Croucher NJ</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">328-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19369947">19369947</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2137">10.1038/nrmicro2137</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Co-evolution of genomes and plasmids within Chlamydia trachomatis and the emergence in Sweden of a new variant strain.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span>, <span class="author">Harris SR</span>, <span class="author">Persson K</span>, <span class="author">Marsh P</span>, <span class="author">Barron A</span>, <span class="author">Bignell A</span>, <span class="author">Bjartling C</span>, <span class="author">Clark L</span>, <span class="author">Cutcliffe LT</span>, <span class="author">Lambden PR</span>, <span class="author">Lennard N</span>, <span class="author">Lockey SJ</span>, <span class="author">Quail MA</span>, <span class="author">Salim O</span>, <span class="author">Skilton RJ</span>, <span class="author">Wang Y</span>, <span class="author">Holland MJ</span>, <span class="author">Parkhill J</span>, <span class="author">Thomson NR</span> and <span class="author">Clarke IN</span></p>
<p class="affiliation">Molecular Microbiology Group, University Medical School, Southampton General Hospital, Southampton, SO16 6YD, UK. <a href="mailto:%68%73%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#115;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Chlamydia trachomatis is the most common cause of sexually transmitted infections globally and the leading cause of preventable blindness in the developing world. There are two biovariants of C. trachomatis: &#39;trachoma&#39;, causing ocular and genital tract infections, and the invasive &#39;lymphogranuloma venereum&#39; strains. Recently, a new variant of the genital tract C. trachomatis emerged in Sweden. This variant escaped routine diagnostic tests because it carries a plasmid with a deletion. Failure to detect this strain has meant it has spread rapidly across the country provoking a worldwide alert. In addition to being a key diagnostic target, the plasmid has been linked to chlamydial virulence. Analysis of chlamydial plasmids and their cognate chromosomes was undertaken to provide insights into the evolutionary relationship between chromosome and plasmid. This is essential knowledge if the plasmid is to be continued to be relied on as a key diagnostic marker, and for an understanding of the evolution of Chlamydia trachomatis.</p><p class="abstract"><strong>Results</strong>: The genomes of two new C. trachomatis strains were sequenced, together with plasmids from six C. trachomatis isolates, including the new variant strain from Sweden. The plasmid from the new Swedish variant has a 377 bp deletion in the first predicted coding sequence, abolishing the site used for PCR detection, resulting in negative diagnosis. In addition, the variant plasmid has a 44 bp duplication downstream of the deletion. The region containing the second predicted coding sequence is the most highly conserved region of the plasmids investigated. Phylogenetic analysis of the plasmids and chromosomes are fully congruent. Moreover this analysis also shows that ocular and genital strains diverged from a common C. trachomatis progenitor.</p><p class="abstract"><strong>Conclusion</strong>: The evolutionary pathways of the chlamydial genome and plasmid imply that inheritance of the plasmid is tightly linked with its cognate chromosome. These data suggest that the plasmid is not a highly mobile genetic element and does not transfer readily between isolates. Comparative analysis of the plasmid sequences has revealed the most conserved regions that should be used to design future plasmid based nucleic acid amplification tests, to avoid diagnostic failures.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0601640</span>; Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">239</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19460133">19460133</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2693142">2693142</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-10-239">10.1186/1471-2164-10-239</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Worldwide patterns of haplotype diversity at 9p21.3, a locus associated with type 2 diabetes and coronary heart disease.</h4>
<p class="authors"><span class="author">Silander K</span>, <span class="author">Tang H</span>, <span class="author">Myles S</span>, <span class="author">Jakkula E</span>, <span class="author">Timpson NJ</span>, <span class="author">Cavalli-Sforza L</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Institute of Molecular Medicine FIMM, University of Helsinki, and Unit of Public Health Genomics, National Institute for Health and Welfare, Tukholmankatu 8, 00290 Helsinki, Finland. <a href="mailto:%6B%61%69%73%61.%73%69%6C%61%6E%64%65%72%40%74%68%6C.%66%69">&#107;&#97;&#105;&#115;&#97;&#46;&#115;&#105;&#108;&#97;&#110;&#100;&#101;&#114;&#64;&#116;&#104;&#108;&#46;&#102;&#105;</a>.</p>
<p class="abstract">A 100 kb region on 9p21.3 harbors two major disease susceptibility loci: one for type 2 diabetes (T2D) and one for coronary heart disease (CHD). The single nucleotide polymorphisms (SNPs) associated with these two diseases in Europeans reside on two adjacent haplotype blocks with independent effects on disease. To help delimit the regions that likely harbor the disease-causing variants in populations of non-European origin, we studied the haplotype diversity and allelic history of the 9p21.3 region using 938 unrelated individuals from 51 populations (Human Genome Diversity Panel). We used SNP data from Illumina's 650Y SNP arrays supplemented with five additional SNPs within the region of interest. Haplotype frequencies were analyzed with the EM algorithm implemented in PLINK. For the T2D locus, the TT risk haplotype of SNPs rs10811661 and rs10757283 was present at similar frequencies in all global populations, while a shared 6-SNP haplotype that carries the protective C allele of rs10811661 was found at a frequency of 2.9% in Africans and 41.3% in East Asians and was associated with low haplotype diversity. For the CHD locus, all populations shared a core risk haplotype spanning >17.5 kb, which shows dramatic increase in frequency between African (11.5%) and Middle Eastern (63.7%) populations. Interestingly, two SNPs (rs2891168 and rs10757278) tagging this CHD risk haplotype are most strongly associated with CHD disease status according to independent clinical fine-mapping studies. The large variation in linkage disequilibrium patterns identified between the populations demonstrates the importance of allelic background data when selecting SNPs for replication in global populations. Intriguingly, the protective allele for T2D and the risk allele for CHD show an increase in frequency in non-Africans compared to Africans, implying different population histories for these two adjacent disease loci.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2009</span>;<span class="volume">1</span>;<span class="number">5</span>;<span class="pagerange">51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19463184" rel="external">19463184</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2689443" rel="external">2689443</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm51" rel="external">10.1186/gm51</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and genetic analyses of diversity and plant interactions of Pseudomonas fluorescens.</h4>
<p class="authors"><span class="author">Silby MW</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Vernikos GS</span>, <span class="author">Giddens SR</span>, <span class="author">Jackson RW</span>, <span class="author">Preston GM</span>, <span class="author">Zhang XX</span>, <span class="author">Moon CD</span>, <span class="author">Gehrig SM</span>, <span class="author">Godfrey SA</span>, <span class="author">Knight CG</span>, <span class="author">Malone JG</span>, <span class="author">Robinson Z</span>, <span class="author">Spiers AJ</span>, <span class="author">Harris S</span>, <span class="author">Challis GL</span>, <span class="author">Yaxley AM</span>, <span class="author">Harris D</span>, <span class="author">Seeger K</span>, <span class="author">Murphy L</span>, <span class="author">Rutter S</span>, <span class="author">Squares R</span>, <span class="author">Quail MA</span>, <span class="author">Saunders E</span>, <span class="author">Mavromatis K</span>, <span class="author">Brettin TS</span>, <span class="author">Bentley SD</span>, <span class="author">Hothersall J</span>, <span class="author">Stephens E</span>, <span class="author">Thomas CM</span>, <span class="author">Parkhill J</span>, <span class="author">Levy SB</span>, <span class="author">Rainey PB</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Department of Molecular Biology and Microbiology, Tufts University School of Medicine, Centre for Adaptation Genetics and Drug Resistance, Boston, MA 02111, USA. <a href="mailto:%6D%61%72%6B.%73%69%6C%62%79%40%74%75%66%74%73.%65%64%75">&#109;&#97;&#114;&#107;&#46;&#115;&#105;&#108;&#98;&#121;&#64;&#116;&#117;&#102;&#116;&#115;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Pseudomonas fluorescens are common soil bacteria that can improve plant health through nutrient cycling, pathogen antagonism and induction of plant defenses. The genome sequences of strains SBW25 and Pf0-1 were determined and compared to each other and with P. fluorescens Pf-5. A functional genomic in vivo expression technology (IVET) screen provided insight into genes used by P. fluorescens in its natural environment and an improved understanding of the ecological significance of diversity within this species.</p><p class="abstract"><strong>Results</strong>: Comparisons of three P. fluorescens genomes (SBW25, Pf0-1, Pf-5) revealed considerable divergence: 61% of genes are shared, the majority located near the replication origin. Phylogenetic and average amino acid identity analyses showed a low overall relationship. A functional screen of SBW25 defined 125 plant-induced genes including a range of functions specific to the plant environment. Orthologues of 83 of these exist in Pf0-1 and Pf-5, with 73 shared by both strains. The P. fluorescens genomes carry numerous complex repetitive DNA sequences, some resembling Miniature Inverted-repeat Transposable Elements (MITEs). In SBW25, repeat density and distribution revealed &#39;repeat deserts&#39; lacking repeats, covering approximately 40% of the genome.</p><p class="abstract"><strong>Conclusions</strong>: P. fluorescens genomes are highly diverse. Strain-specific regions around the replication terminus suggest genome compartmentalization. The genomic heterogeneity among the three strains is reminiscent of a species complex rather than a single species. That 42% of plant-inducible genes were not shared by all strains reinforces this conclusion and shows that ecological success requires specialized and core functions. The diversity also indicates the significant size of genetic information within the Pseudomonas pan genome.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">104/P16729</span>, <span class="grant">P15257</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="number">5</span>;<span class="pagerange">R51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19432983">19432983</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2718517">2718517</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2009-10-5-r51">10.1186/gb-2009-10-5-r51</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide scans for human adult stature identifies novel Loci and associations with measures of skeletal frame size.</h4>
<p class="authors"><span class="author">Soranzo N</span>, <span class="author">Rivadeneira F</span>, <span class="author">Chinappen-Horsley U</span>, <span class="author">Malkina I</span>, <span class="author">Richards JB</span>, <span class="author">Hammond N</span>, <span class="author">Stolk L</span>, <span class="author">Nica A</span>, <span class="author">Inouye M</span>, <span class="author">Hofman A</span>, <span class="author">Stephens J</span>, <span class="author">Wheeler E</span>, <span class="author">Arp P</span>, <span class="author">Gwilliam R</span>, <span class="author">Jhamai PM</span>, <span class="author">Potter S</span>, <span class="author">Chaney A</span>, <span class="author">Ghori MJ</span>, <span class="author">Ravindrarajah R</span>, <span class="author">Ermakov S</span>, <span class="author">Estrada K</span>, <span class="author">Pols HA</span>, <span class="author">Williams FM</span>, <span class="author">McArdle WL</span>, <span class="author">van Meurs JB</span>, <span class="author">Loos RJ</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Ahmadi KR</span>, <span class="author">Hart DJ</span>, <span class="author">Ouwehand WH</span>, <span class="author">Wareham NJ</span>, <span class="author">Barroso I</span>, <span class="author">Sandhu MS</span>, <span class="author">Strachan DP</span>, <span class="author">Livshits G</span>, <span class="author">Spector TD</span>, <span class="author">Uitterlinden AG</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Human Genetics Department, Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Recent genome-wide (GW) scans have identified several independent loci affecting human stature, but their contribution through the different skeletal components of height is still poorly understood. We carried out a genome-wide scan in 12,611 participants, followed by replication in an additional 7,187 individuals, and identified 17 genomic regions with GW-significant association with height. Of these, two are entirely novel (rs11809207 in CATSPER4, combined P-value = 6.1x10(-8) and rs910316 in TMED10, P-value = 1.4x10(-7)) and two had previously been described with weak statistical support (rs10472828 in NPR3, P-value = 3x10(-7) and rs849141 in JAZF1, P-value = 3.2x10(-11)). One locus (rs1182188 at GNA12) identifies the first height eQTL. We also assessed the contribution of height loci to the upper- (trunk) and lower-body (hip axis and femur) skeletal components of height. We find evidence for several loci associated with trunk length (including rs6570507 in GPR126, P-value = 4x10(-5) and rs6817306 in LCORL, P-value = 4x10(-4)), hip axis length (including rs6830062 at LCORL, P-value = 4.8x10(-4) and rs4911494 at UQCC, P-value = 1.9x10(-4)), and femur length (including rs710841 at PRKG2, P-value = 2.4x10(-5) and rs10946808 at HIST1H1D, P-value = 6.4x10(-6)). Finally, we used conditional analyses to explore a possible differential contribution of the height loci to these different skeletal size measurements. In addition to validating four novel loci controlling adult stature, our study represents the first effort to assess the contribution of genetic loci to three skeletal components of height. Further statistical tests in larger numbers of individuals will be required to verify if the height loci affect height preferentially through these subcomponents of height.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0701863</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">e1000445</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19343178">19343178</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2661236">2661236</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000445">10.1371/journal.pgen.1000445</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MET and autism susceptibility: family and case-control studies.</h4>
<p class="authors"><span class="author">Sousa I</span>, <span class="author">Clark TG</span>, <span class="author">Toma C</span>, <span class="author">Kobayashi K</span>, <span class="author">Choma M</span>, <span class="author">Holt R</span>, <span class="author">Sykes NH</span>, <span class="author">Lamb JA</span>, <span class="author">Bailey AJ</span>, <span class="author">Battaglia A</span>, <span class="author">Maestrini E</span>, <span class="author">Monaco AP</span> and <span class="author">International Molecular Genetic Study of Autism Consortium (IMGSAC)</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract">Autism is a common, severe and highly heritable neurodevelopmental disorder. The International Molecular Genetic Study of Autism Consortium (IMGSAC) genome screen for linkage in affected sib-pair families identified a chromosome 7q susceptibility locus (AUTS1), that has subsequently shown evidence of increased sharing in several independent multiplex samples and in two meta-analyses. Taking into account the location of the MET gene under this linkage peak, and the fact that it has recently been reported to be associated with autism, the gene was further analyzed as a promising autism candidate. The gene encodes a transmembrane receptor tyrosine kinase of the hepatocyte growth factor/scatter factor (HGF/SF). MET is best known as an oncogene, but its signalling also participates in immune function, peripheral organ development and repair, and the development of the cerebral cortex and cerebellum (all of which have been observed earlier as being disregulated in individuals with autism). Here we present a family-based association analysis covering the entire MET locus. Significant results were obtained in both single locus and haplotype approaches with a single nucleotide polymorphism in intron 1 (rs38845, P&lt;0.004) and with one intronic haplotype (AAGTG, P&lt;0.009) in 325 multiplex IMGSAC families and 10 IMGSAC trios. Although these results failed to replicate in an independent sample of 82 Italian trios, the association itself was confirmed by a case-control analysis performed using the Italian cohort (P&lt;0.02). The previously reported positive association of rs1858830 failed to replicate in this study. Overall, our findings provide further evidence that MET may play a role in autism susceptibility.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">075491</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">749-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19002214" rel="external">19002214</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2685893" rel="external">2685893</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2008.215" rel="external">10.1038/ejhg.2008.215</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Is the thrifty genotype hypothesis supported by evidence based on confirmed type 2 diabetes- and obesity-susceptibility variants?</h4>
<p class="authors"><span class="author">Southam L</span>, <span class="author">Soranzo N</span>, <span class="author">Montgomery SB</span>, <span class="author">Frayling TM</span>, <span class="author">McCarthy MI</span>, <span class="author">Barroso I</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract">According to the thrifty genotype hypothesis, the high prevalence of type 2 diabetes and obesity is a consequence of genetic variants that have undergone positive selection during historical periods of erratic food supply. The recent expansion in the number of validated type 2 diabetes- and obesity-susceptibility loci, coupled with access to empirical data, enables us to look for evidence in support (or otherwise) of the thrifty genotype hypothesis using proven loci.</p><p class="abstract"><strong>Methods</strong>: We employed a range of tests to obtain complementary views of the evidence for selection: we determined whether the risk allele at associated &#39;index&#39; single-nucleotide polymorphisms is derived or ancestral, calculated the integrated haplotype score (iHS) and assessed the population differentiation statistic fixation index (F (ST)) for 17 type 2 diabetes and 13 obesity loci.</p><p class="abstract"><strong>Results</strong>: We found no evidence for significant differences for the derived/ancestral allele test. None of the studied loci showed strong evidence for selection based on the iHS score. We find a high F (ST) for rs7901695 at TCF7L2, the largest type 2 diabetes effect size found to date.</p><p class="abstract">Our results provide some evidence for selection at specific loci, but there are no consistent patterns of selection that provide conclusive confirmation of the thrifty genotype hypothesis. Discovery of more signals and more causal variants for type 2 diabetes and obesity is likely to allow more detailed examination of these issues.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0601261</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">079557</span>, <span class="grant">088885</span>, <span class="grant">WT077016/Z/05/Z</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2009</span>;<span class="volume">52</span>;<span class="number">9</span>;<span class="pagerange">1846-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19526209">19526209</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2723682">2723682</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-009-1419-3">10.1007/s00125-009-1419-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic determinants of height growth assessed longitudinally from infancy to adulthood in the northern Finland birth cohort 1966.</h4>
<p class="authors"><span class="author">Sovio U</span>, <span class="author">Bennett AJ</span>, <span class="author">Millwood IY</span>, <span class="author">Molitor J</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Timpson NJ</span>, <span class="author">Kaakinen M</span>, <span class="author">Laitinen J</span>, <span class="author">Haukka J</span>, <span class="author">Pillas D</span>, <span class="author">Tzoulaki I</span>, <span class="author">Molitor J</span>, <span class="author">Hoggart C</span>, <span class="author">Coin LJ</span>, <span class="author">Whittaker J</span>, <span class="author">Pouta A</span>, <span class="author">Hartikainen AL</span>, <span class="author">Freimer NB</span>, <span class="author">Widen E</span>, <span class="author">Peltonen L</span>, <span class="author">Elliott P</span>, <span class="author">McCarthy MI</span> and <span class="author">Jarvelin MR</span></p>
<p class="affiliation">Department of Epidemiology and Public Health, Imperial College London, London, United Kingdom.</p><p class="abstract">Recent genome-wide association (GWA) studies have identified dozens of common variants associated with adult height. However, it is unknown how these variants influence height growth during childhood. We derived peak height velocity in infancy (PHV1) and puberty (PHV2) and timing of pubertal height growth spurt from parametric growth curves fitted to longitudinal height growth data to test their association with known height variants. The study consisted of N = 3,538 singletons from the prospective Northern Finland Birth Cohort 1966 with genotype data and frequent height measurements (on average 20 measurements per person) from 0-20 years. Twenty-six of the 48 variants tested associated with adult height (p&lt;0.05, adjusted for sex and principal components) in this sample, all in the same direction as in previous GWA scans. Seven SNPs in or near the genes HHIP, DLEU7, UQCC, SF3B4/SV2A, LCORL, and HIST1H1D associated with PHV1 and five SNPs in or near SOCS2, SF3B4/SV2A, C17orf67, CABLES1, and DOT1L with PHV2 (p&lt;0.05). We formally tested variants for interaction with age (infancy versus puberty) and found biologically meaningful evidence for an age-dependent effect for the SNP in SOCS2 (p = 0.0030) and for the SNP in HHIP (p = 0.045). We did not have similar prior evidence for the association between height variants and timing of pubertal height growth spurt as we had for PHVs, and none of the associations were statistically significant after correction for multiple testing. The fact that in this sample, less than half of the variants associated with adult height had a measurable effect on PHV1 or PHV2 is likely to reflect limited power to detect these associations in this dataset. Our study is the first genetic association analysis on longitudinal height growth in a prospective cohort from birth to adulthood and gives grounding for future research on the genetic regulation of human height during different periods of growth.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600705</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">e1000409</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19266077" rel="external">19266077</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2646138" rel="external">2646138</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000409" rel="external">10.1371/journal.pgen.1000409</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and genic deletions of the FOX gene cluster on 16q24.1 and inactivating mutations of FOXF1 cause alveolar capillary dysplasia and other malformations.</h4>
<p class="authors"><span class="author">Stankiewicz P</span>, <span class="author">Sen P</span>, <span class="author">Bhatt SS</span>, <span class="author">Storer M</span>, <span class="author">Xia Z</span>, <span class="author">Bejjani BA</span>, <span class="author">Ou Z</span>, <span class="author">Wiszniewska J</span>, <span class="author">Driscoll DJ</span>, <span class="author">Maisenbacher MK</span>, <span class="author">Bolivar J</span>, <span class="author">Bauer M</span>, <span class="author">Zackai EH</span>, <span class="author">McDonald-McGinn D</span>, <span class="author">Nowaczyk MM</span>, <span class="author">Murray M</span>, <span class="author">Hustead V</span>, <span class="author">Mascotti K</span>, <span class="author">Schultz R</span>, <span class="author">Hallam L</span>, <span class="author">McRae D</span>, <span class="author">Nicholson AG</span>, <span class="author">Newbury R</span>, <span class="author">Durham-O&#39;Donnell J</span>, <span class="author">Knight G</span>, <span class="author">Kini U</span>, <span class="author">Shaikh TH</span>, <span class="author">Martin V</span>, <span class="author">Tyreman M</span>, <span class="author">Simonic I</span>, <span class="author">Willatt L</span>, <span class="author">Paterson J</span>, <span class="author">Mehta S</span>, <span class="author">Rajan D</span>, <span class="author">Fitzgerald T</span>, <span class="author">Gribble S</span>, <span class="author">Prigmore E</span>, <span class="author">Patel A</span>, <span class="author">Shaffer LG</span>, <span class="author">Carter NP</span>, <span class="author">Cheung SW</span>, <span class="author">Langston C</span> and <span class="author">Shaw-Smith C</span></p>
<p class="affiliation">Dept of Molecular &amp; Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA. <a href="mailto:%70%61%77%65%6C%73%40%62%63%6D.%65%64%75">&#112;&#97;&#119;&#101;&#108;&#115;&#64;&#98;&#99;&#109;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Alveolar capillary dysplasia with misalignment of pulmonary veins (ACD/MPV) is a rare, neonatally lethal developmental disorder of the lung with defining histologic abnormalities typically associated with multiple congenital anomalies (MCA). Using array CGH analysis, we have identified six overlapping microdeletions encompassing the FOX transcription factor gene cluster in chromosome 16q24.1q24.2 in patients with ACD/MPV and MCA. Subsequently, we have identified four different heterozygous mutations (frameshift, nonsense, and no-stop) in the candidate FOXF1 gene in unrelated patients with sporadic ACD/MPV and MCA. Custom-designed, high-resolution microarray analysis of additional ACD/MPV samples revealed one microdeletion harboring FOXF1 and two distinct microdeletions upstream of FOXF1, implicating a position effect. DNA sequence analysis revealed that in six of nine deletions, both breakpoints occurred in the portions of Alu elements showing eight to 43 base pairs of perfect microhomology, suggesting replication error Microhomology-Mediated Break-Induced Replication (MMBIR)/Fork Stalling and Template Switching (FoSTeS) as a mechanism of their formation. In contrast to the association of point mutations in FOXF1 with bowel malrotation, microdeletions of FOXF1 were associated with hypoplastic left heart syndrome and gastrointestinal atresias, probably due to haploinsufficiency for the neighboring FOXC2 and FOXL1 genes. These differences reveal the phenotypic consequences of gene alterations in cis.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">84</span>;<span class="number">6</span>;<span class="pagerange">780-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19500772">19500772</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2694971">2694971</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2009.05.005">10.1016/j.ajhg.2009.05.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants conferring risk of schizophrenia.</h4>
<p class="authors"><span class="author">Stefansson H</span>, <span class="author">Ophoff RA</span>, <span class="author">Steinberg S</span>, <span class="author">Andreassen OA</span>, <span class="author">Cichon S</span>, <span class="author">Rujescu D</span>, <span class="author">Werge T</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Mors O</span>, <span class="author">Mortensen PB</span>, <span class="author">Sigurdsson E</span>, <span class="author">Gustafsson O</span>, <span class="author">Nyegaard M</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Ingason A</span>, <span class="author">Hansen T</span>, <span class="author">Suvisaari J</span>, <span class="author">Lonnqvist J</span>, <span class="author">Paunio T</span>, <span class="author">B&oslash;rglum AD</span>, <span class="author">Hartmann A</span>, <span class="author">Fink-Jensen A</span>, <span class="author">Nordentoft M</span>, <span class="author">Hougaard D</span>, <span class="author">Norgaard-Pedersen B</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Olesen J</span>, <span class="author">Breuer R</span>, <span class="author">M&ouml;ller HJ</span>, <span class="author">Giegling I</span>, <span class="author">Rasmussen HB</span>, <span class="author">Timm S</span>, <span class="author">Mattheisen M</span>, <span class="author">Bitter I</span>, <span class="author">R&eacute;thelyi JM</span>, <span class="author">Magnusdottir BB</span>, <span class="author">Sigmundsson T</span>, <span class="author">Olason P</span>, <span class="author">Masson G</span>, <span class="author">Gulcher JR</span>, <span class="author">Haraldsson M</span>, <span class="author">Fossdal R</span>, <span class="author">Thorgeirsson TE</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Ruggeri M</span>, <span class="author">Tosato S</span>, <span class="author">Franke B</span>, <span class="author">Strengman E</span>, <span class="author">Kiemeney LA</span>, <span class="author">Genetic Risk and Outcome in Psychosis (GROUP)</span>, <span class="author">Melle I</span>, <span class="author">Djurovic S</span>, <span class="author">Abramova L</span>, <span class="author">Kaleda V</span>, <span class="author">Sanjuan J</span>, <span class="author">de Frutos R</span>, <span class="author">Bramon E</span>, <span class="author">Vassos E</span>, <span class="author">Fraser G</span>, <span class="author">Ettinger U</span>, <span class="author">Picchioni M</span>, <span class="author">Walker N</span>, <span class="author">Toulopoulou T</span>, <span class="author">Need AC</span>, <span class="author">Ge D</span>, <span class="author">Yoon JL</span>, <span class="author">Shianna KV</span>, <span class="author">Freimer NB</span>, <span class="author">Cantor RM</span>, <span class="author">Murray R</span>, <span class="author">Kong A</span>, <span class="author">Golimbet V</span>, <span class="author">Carracedo A</span>, <span class="author">Arango C</span>, <span class="author">Costas J</span>, <span class="author">J&ouml;nsson EG</span>, <span class="author">Terenius L</span>, <span class="author">Agartz I</span>, <span class="author">Petursson H</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Rietschel M</span>, <span class="author">Matthews PM</span>, <span class="author">Muglia P</span>, <span class="author">Peltonen L</span>, <span class="author">St Clair D</span>, <span class="author">Goldstein DB</span>, <span class="author">Stefansson K</span> and <span class="author">Collier DA</span></p>
<p class="affiliation">deCODE genetics, Sturlugata 8, IS-101 Reykjavik, Iceland.</p><p class="abstract">Schizophrenia is a complex disorder, caused by both genetic and environmental factors and their interactions. Research on pathogenesis has traditionally focused on neurotransmitter systems in the brain, particularly those involving dopamine. Schizophrenia has been considered a separate disease for over a century, but in the absence of clear biological markers, diagnosis has historically been based on signs and symptoms. A fundamental message emerging from genome-wide association studies of copy number variations (CNVs) associated with the disease is that its genetic basis does not necessarily conform to classical nosological disease boundaries. Certain CNVs confer not only high relative risk of schizophrenia but also of other psychiatric disorders. The structural variations associated with schizophrenia can involve several genes and the phenotypic syndromes, or the &#39;genomic disorders&#39;, have not yet been characterized. Single nucleotide polymorphism (SNP)-based genome-wide association studies with the potential to implicate individual genes in complex diseases may reveal underlying biological pathways. Here we combined SNP data from several large genome-wide scans and followed up the most significant association signals. We found significant association with several markers spanning the major histocompatibility complex (MHC) region on chromosome 6p21.3-22.1, a marker located upstream of the neurogranin gene (NRGN) on 11q24.2 and a marker in intron four of transcription factor 4 (TCF4) on 18q21.2. Our findings implicating the MHC region are consistent with an immune component to schizophrenia risk, whereas the association with NRGN and TCF4 points to perturbation of pathways involved in brain development, memory and cognition.</p>
<p class="grants">Funded by: Department of Health: <span class="grant">PDA/02/06/016</span>; NHLBI NIH HHS: <span class="grant">1R01HL087679-01</span>; NIMH NIH HHS: <span class="grant">R01 MH078075</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2009</span>;<span class="volume">460</span>;<span class="number">7256</span>;<span class="pagerange">744-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19571808">19571808</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3077530">3077530</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08186">10.1038/nature08186</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Arena syndrome is caused by a missense mutation in PLP1.</h4>
<p class="authors"><span class="author">Stevenson RE</span>, <span class="author">Tarpey P</span>, <span class="author">May MM</span>, <span class="author">Stratton MR</span> and <span class="author">Schwartz CE</span></p>
<p class="affiliation">JC Self Research Institute of Human Genetics, Greenwood Genetic Center, Greenwood, South Carolina 29646, USA. <a href="mailto:%72%65%73%40%67%67%63.%6F%72%67">&#114;&#101;&#115;&#64;&#103;&#103;&#99;&#46;&#111;&#114;&#103;</a></p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2009</span>;<span class="volume">149A</span>;<span class="number">5</span>;<span class="pagerange">1081</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19396823" rel="external">19396823</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.32795" rel="external">10.1002/ajmg.a.32795</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loci at chromosomes 13, 19 and 20 influence age at natural menopause.</h4>
<p class="authors"><span class="author">Stolk L</span>, <span class="author">Zhai G</span>, <span class="author">van Meurs JB</span>, <span class="author">Verbiest MM</span>, <span class="author">Visser JA</span>, <span class="author">Estrada K</span>, <span class="author">Rivadeneira F</span>, <span class="author">Williams FM</span>, <span class="author">Cherkas L</span>, <span class="author">Deloukas P</span>, <span class="author">Soranzo N</span>, <span class="author">de Keyzer JJ</span>, <span class="author">Pop VJ</span>, <span class="author">Lips P</span>, <span class="author">Lebrun CE</span>, <span class="author">van der Schouw YT</span>, <span class="author">Grobbee DE</span>, <span class="author">Witteman J</span>, <span class="author">Hofman A</span>, <span class="author">Pols HA</span>, <span class="author">Laven JS</span>, <span class="author">Spector TD</span> and <span class="author">Uitterlinden AG</span></p>
<p class="affiliation">Department of Internal Medicine, Erasmus MC, Rotterdam, The Netherlands.</p><p class="abstract">We conducted a genome-wide association study for age at natural menopause in 2,979 European women and identified six SNPs in three loci associated with age at natural menopause: chromosome 19q13.4 (rs1172822; -0.4 year per T allele (39%); P = 6.3 &times; 10(-11)), chromosome 20p12.3 (rs236114; +0.5 year per A allele (21%); P = 9.7 &times; 10(-11)) and chromosome 13q34 (rs7333181; +0.5 year per A allele (12%); P = 2.5 &times; 10(-8)). These common genetic variants regulate timing of ovarian aging, an important risk factor for breast cancer, osteoporosis and cardiovascular disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">6</span>;<span class="pagerange">645-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19448619">19448619</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3000545">3000545</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.387">10.1038/ng.387</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of single nucleotide polymorphisms based on the multilocus sequence typing database of Staphylococcus aureus using locked nucleic acid oligonucleotides.</h4>
<p class="authors"><span class="author">Stone M</span>, <span class="author">Bamford K</span> and <span class="author">Wain J</span></p>
<p class="publication"><span class="title">Journal of medical microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">58</span>;<span class="number">Pt 5</span>;<span class="pagerange">693-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19369537" rel="external">19369537</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fjmm.0.007534-0" rel="external">10.1099/jmm.0.007534-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The cancer genome.</h4>
<p class="authors"><span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%72%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#114;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">All cancers arise as a result of changes that have occurred in the DNA sequence of the genomes of cancer cells. Over the past quarter of a century much has been learnt about these mutations and the abnormal genes that operate in human cancers. We are now, however, moving into an era in which it will be possible to obtain the complete DNA sequence of large numbers of cancer genomes. These studies will provide us with a detailed and comprehensive perspective on how individual cancers have developed.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span>, <span class="grant">088340</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2009</span>;<span class="volume">458</span>;<span class="number">7239</span>;<span class="pagerange">719-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19360079">19360079</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2821689">2821689</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature07943">10.1038/nature07943</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deep short-read sequencing of chromosome 17 from the mouse strains A/J and CAST/Ei identifies significant germline variation and candidate genes that regulate liver triglyceride levels.</h4>
<p class="authors"><span class="author">Sudbery I</span>, <span class="author">Stalker J</span>, <span class="author">Simpson JT</span>, <span class="author">Keane T</span>, <span class="author">Rust AG</span>, <span class="author">Hurles ME</span>, <span class="author">Walter K</span>, <span class="author">Lynch D</span>, <span class="author">Teboul L</span>, <span class="author">Brown SD</span>, <span class="author">Li H</span>, <span class="author">Ning Z</span>, <span class="author">Nadeau JH</span>, <span class="author">Croniger CM</span>, <span class="author">Durbin R</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1HH, UK. <a href="mailto:%69%6D%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#109;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genome sequences are essential tools for comparative and mutational analyses. Here we present the short read sequence of mouse chromosome 17 from the Mus musculus domesticus derived strain A/J, and the Mus musculus castaneus derived strain CAST/Ei. We describe approaches for the accurate identification of nucleotide and structural variation in the genomes of vertebrate experimental organisms, and show how these techniques can be applied to help prioritize candidate genes within quantitative trait loci.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>; NIAAA NIH HHS: <span class="grant">P20 AA017837</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="number">10</span>;<span class="pagerange">R112</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19825173">19825173</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2784327">2784327</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2009-10-10-r112">10.1186/gb-2009-10-10-r112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human Molecular Genetics</h4>
<p class="authors"><span class="author">Sudbery</span>, <span class="author">P.</span>, <span class="author">Sudbery</span>, <span class="author">I.</span></p>
<p class="publication"><span class="title">3rd Edition;Pearson Educational;ISBN978-0132051576</span> <span class="year">2009</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Confirmation of multiple risk Loci and genetic impacts by a genome-wide association study of type 2 diabetes in the Japanese population.</h4>
<p class="authors"><span class="author">Takeuchi F</span>, <span class="author">Serizawa M</span>, <span class="author">Yamamoto K</span>, <span class="author">Fujisawa T</span>, <span class="author">Nakashima E</span>, <span class="author">Ohnaka K</span>, <span class="author">Ikegami H</span>, <span class="author">Sugiyama T</span>, <span class="author">Katsuya T</span>, <span class="author">Miyagishi M</span>, <span class="author">Nakashima N</span>, <span class="author">Nawata H</span>, <span class="author">Nakamura J</span>, <span class="author">Kono S</span>, <span class="author">Takayanagi R</span> and <span class="author">Kato N</span></p>
<p class="affiliation">Department of Medical Ecology and Informatics, Research Institute, International Medical Center of Japan, Tokyo, Japan.</p><p class="abstract"><strong>Objective</strong>: To identify novel type 2 diabetes gene variants and confirm previously identified ones, a three-staged genome-wide association study was performed in the Japanese population.</p><p class="abstract">In the stage 1 scan, we genotyped 519 case and 503 control subjects with 482,625 single nucleotide polymorphism (SNP) markers; in the stage 2 panel comprising 1,110 case subjects and 1,014 control subjects, we assessed 1,456 SNPs (P &lt; 0.0025, stage 1); additionally to direct genotyping, 964 healthy control subjects formed the in silico control panel. Along with genome-wide exploration, we aimed to replicate the disease association of 17 SNPs from 16 candidate loci previously identified in Europeans. The associated and/or replicated loci (23 SNPs; P &lt; 7 x 10(-5) for genome-wide exploration and P &lt; 0.05 for replication) were examined in the stage 3 panel comprising 4,000 case subjects and 12,569 population-based samples, from which 4,889 nondiabetic control subjects were preselected. The 12,569 subjects were used for overall risk assessment in the general population.</p><p class="abstract"><strong>Results</strong>: Four loci-1 novel with suggestive evidence (PEPD on 19q13, P = 1.4 x 10(-5)) and three previously reported-were identified; the association of CDKAL1, CDKN2A/CDKN2B, and KCNQ1 were confirmed (P &lt; 10(-19)). Moreover, significant associations were replicated in five other candidate loci: TCF7L2, IGF2BP2, SLC30A8, HHEX, and KCNJ11. There was substantial overlap of type 2 diabetes susceptibility genes between the two populations, whereas effect size and explained variance tended to be higher in the Japanese population.</p><p class="abstract"><strong>Conclusions</strong>: The strength of association was more prominent in the Japanese population than in Europeans for more than half of the confirmed type 2 diabetes loci.</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2009</span>;<span class="volume">58</span>;<span class="number">7</span>;<span class="pagerange">1690-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19401414" rel="external">19401414</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2699880" rel="external">2699880</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb08-1494" rel="external">10.2337/db08-1494</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation spectrum of Meckel syndrome genes: one group of syndromes or several distinct groups?</h4>
<p class="authors"><span class="author">Tallila J</span>, <span class="author">Salonen R</span>, <span class="author">Kohlschmidt N</span>, <span class="author">Peltonen L</span> and <span class="author">Kestil&auml; M</span></p>
<p class="affiliation">National Institute of Health and Welfare, Public Health Genomics Unit and FIMM, Institute for Molecular Medicine Finland, Helsinki 00290, Finland.</p><p class="abstract">Meckel syndrome (MKS) is a lethal malformation syndrome that belongs to the group of disorders that are associated with primary cilia dysfunction. Total of five genes are known to be involved in the molecular background of MKS. Here we have systematically analyzed all these genes in a total of 29 MKS families. Seven of the families were Finnish and the rest originated from elsewhere in Europe. We found 12 novel mutations in 13 families. Mutations in the MKS genes are also found in other syndromes and it seems reasonable to assume that there is a correlation between the syndromes and the mutations. To obtain some supportive information, we collected all the previously published mutations in the genes to see whether the different syndromes are dictated by the nature of the mutations. Based on this study, mutations play a role in the clinical phenotype, given that the same allelic combination of mutations has never been reported in two clinically distinct syndromes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2009</span>;<span class="volume">30</span>;<span class="number">8</span>;<span class="pagerange">E813-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19466712" rel="external">19466712</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2718326" rel="external">2718326</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.21057" rel="external">10.1002/humu.21057</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A systematic, large-scale resequencing screen of X-chromosome coding exons in mental retardation.</h4>
<p class="authors"><span class="author">Tarpey PS</span>, <span class="author">Smith R</span>, <span class="author">Pleasance E</span>, <span class="author">Whibley A</span>, <span class="author">Edkins S</span>, <span class="author">Hardy C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Latimer C</span>, <span class="author">Dicks E</span>, <span class="author">Menzies A</span>, <span class="author">Stephens P</span>, <span class="author">Blow M</span>, <span class="author">Greenman C</span>, <span class="author">Xue Y</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Thompson D</span>, <span class="author">Gray K</span>, <span class="author">Andrews J</span>, <span class="author">Barthorpe S</span>, <span class="author">Buck G</span>, <span class="author">Cole J</span>, <span class="author">Dunmore R</span>, <span class="author">Jones D</span>, <span class="author">Maddison M</span>, <span class="author">Mironenko T</span>, <span class="author">Turner R</span>, <span class="author">Turrell K</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Wray P</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Jenkinson A</span>, <span class="author">Jia M</span>, <span class="author">Richardson D</span>, <span class="author">Shepherd R</span>, <span class="author">Wooster R</span>, <span class="author">Tejada MI</span>, <span class="author">Martinez F</span>, <span class="author">Carvill G</span>, <span class="author">Goliath R</span>, <span class="author">de Brouwer AP</span>, <span class="author">van Bokhoven H</span>, <span class="author">Van Esch H</span>, <span class="author">Chelly J</span>, <span class="author">Raynaud M</span>, <span class="author">Ropers HH</span>, <span class="author">Abidi FE</span>, <span class="author">Srivastava AK</span>, <span class="author">Cox J</span>, <span class="author">Luo Y</span>, <span class="author">Mallya U</span>, <span class="author">Moon J</span>, <span class="author">Parnau J</span>, <span class="author">Mohammed S</span>, <span class="author">Tolmie JL</span>, <span class="author">Shoubridge C</span>, <span class="author">Corbett M</span>, <span class="author">Gardner A</span>, <span class="author">Haan E</span>, <span class="author">Rujirabanjerd S</span>, <span class="author">Shaw M</span>, <span class="author">Vandeleur L</span>, <span class="author">Fullston T</span>, <span class="author">Easton DF</span>, <span class="author">Boyle J</span>, <span class="author">Partington M</span>, <span class="author">Hackett A</span>, <span class="author">Field M</span>, <span class="author">Skinner C</span>, <span class="author">Stevenson RE</span>, <span class="author">Bobrow M</span>, <span class="author">Turner G</span>, <span class="author">Schwartz CE</span>, <span class="author">Gecz J</span>, <span class="author">Raymond FL</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Large-scale systematic resequencing has been proposed as the key future strategy for the discovery of rare, disease-causing sequence variants across the spectrum of human complex disease. We have sequenced the coding exons of the X chromosome in 208 families with X-linked mental retardation (XLMR), the largest direct screen for constitutional disease-causing mutations thus far reported. The screen has discovered nine genes implicated in XLMR, including SYP, ZNF711 and CASK reported here, confirming the power of this strategy. The study has, however, also highlighted issues confronting whole-genome sequencing screens, including the observation that loss of function of 1% or more of X-chromosome genes is compatible with apparently normal existence.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11022</span>; NICHD NIH HHS: <span class="grant">HD26202</span>; Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">5</span>;<span class="pagerange">535-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19377476">19377476</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2872007">2872007</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.367">10.1038/ng.367</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&#39;Putting our heads together&#39;: insights into genomic conservation between human and canine intracranial tumors.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Duke SE</span>, <span class="author">Wang HJ</span>, <span class="author">Breen TE</span>, <span class="author">Higgins RJ</span>, <span class="author">Linder KE</span>, <span class="author">Ellis P</span>, <span class="author">Langford CF</span>, <span class="author">Dickinson PJ</span>, <span class="author">Olby NJ</span> and <span class="author">Breen M</span></p>
<p class="affiliation">Department of Molecular Biomedical Sciences, College of Veterinary Medicine, North Carolina State University, Raleigh, NC 27606, USA.</p><p class="abstract">Numerous attributes render the domestic dog a highly pertinent model for cancer-associated gene discovery. We performed microarray-based comparative genomic hybridization analysis of 60 spontaneous canine intracranial tumors to examine the degree to which dog and human patients exhibit aberrations of ancestrally related chromosome regions, consistent with a shared pathogenesis. Canine gliomas and meningiomas both demonstrated chromosome copy number aberrations (CNAs) that share evolutionarily conserved synteny with those previously reported in their human counterpart. Interestingly, however, genomic imbalances orthologous to some of the hallmark aberrations of human intracranial tumors, including chromosome 22/NF2 deletions in meningiomas and chromosome 1p/19q deletions in oligodendrogliomas, were not major events in the dog. Furthermore, and perhaps most significantly, we identified highly recurrent CNAs in canine intracranial tumors for which the human orthologue has been reported previously at low frequency but which have not, thus far, been associated intimately with the pathogenesis of the tumor. The presence of orthologous CNAs in canine and human intracranial cancers is strongly suggestive of their biological significance in tumor development and/or progression. Moreover, the limited genetic heterogenity within purebred dog populations, coupled with the contrasting organization of the dog and human karyotypes, offers tremendous opportunities for refining evolutionarily conserved regions of tumor-associated genomic imbalance that may harbor novel candidate genes involved in their pathogenesis. A comparative approach to the study of canine and human intracranial tumors may therefore provide new insights into their genetic etiology, towards development of more sophisticated molecular subclassification and tailored therapies in both species.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">NS051190</span>, <span class="grant">R21 NS051190-01</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of neuro-oncology</span>&nbsp;<span class="year">2009</span>;<span class="volume">94</span>;<span class="number">3</span>;<span class="pagerange">333-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19333554" rel="external">19333554</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3225023" rel="external">3225023</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs11060-009-9877-5" rel="external">10.1007/s11060-009-9877-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microarray-based cytogenetic profiling reveals recurrent and subtype-associated genomic copy number aberrations in feline sarcomas.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Valli VE</span>, <span class="author">Ellis P</span>, <span class="author">Bell J</span>, <span class="author">Karlsson EK</span>, <span class="author">Cullen J</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Langford CF</span> and <span class="author">Breen M</span></p>
<p class="affiliation">Department of Molecular Biomedical Sciences, College of Veterinary Medicine, North Carolina State University, Raleigh, NC 27606, USA. <a href="mailto:%72%61%63%68%61%65%6C%5F%74%68%6F%6D%61%73%40%6E%63%73%75.%65%64%75">&#114;&#97;&#99;&#104;&#97;&#101;&#108;&#95;&#116;&#104;&#111;&#109;&#97;&#115;&#64;&#110;&#99;&#115;&#117;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Injection-site-associated sarcomas (ISAS), commonly arising at the site of routine vaccine administration, afflict as many as 22,000 domestic cats annually in the USA. These tumors are typically more aggressive and prone to recurrence than spontaneous sarcomas (non-ISAS), generally receiving a poorer long-term prognosis and warranting a more aggressive therapeutic approach. Although certain clinical and histological factors are highly suggestive of ISAS, timely diagnosis and optimal clinical management may be hindered by the absence of definitive markers that can distinguish between tumors with underlying injection-related etiology and their spontaneous counterpart. Specific nonrandom chromosome copy number aberrations (CNAs) have been associated with the clinical behavior of a vast spectrum of human tumors, providing an extensive resource of potential diagnostic and prognostic biomarkers. Although similar principles are now being applied with great success in other species, their relevance to feline molecular oncology has not yet been investigated in any detail. We report the construction of a genomic microarray platform for detection of recurrent CNAs in feline tumors through cytogenetic assignment of 210 large-insert DNA clones selected at intervals of approximately 15 Mb from the feline genome sequence assembly. Microarray-based profiling of 19 ISAS and 27 non-ISAS cases identified an extensive range of genomic imbalances that were highly recurrent throughout the combined panel of 46 sarcomas. Deletions of two specific regions were significantly associated with the non-ISAS phenotype. Further characterization of these regions may ultimately permit molecular distinction between ISAS and non-ISAS, as a tool for predicting tumor behavior and prognosis, as well as refining means for therapeutic intervention.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">8</span>;<span class="pagerange">987-1000</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19941159">19941159</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-009-9096-0">10.1007/s10577-009-9096-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Influence of genetic background on tumor karyotypes: evidence for breed-associated cytogenetic aberrations in canine appendicular osteosarcoma.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Wang HJ</span>, <span class="author">Tsai PC</span>, <span class="author">Langford CF</span>, <span class="author">Fosmire SP</span>, <span class="author">Jubala CM</span>, <span class="author">Getzy DM</span>, <span class="author">Cutter GR</span>, <span class="author">Modiano JF</span> and <span class="author">Breen M</span></p>
<p class="affiliation">Department of Molecular Biomedical Sciences, College of Veterinary Medicine, North Carolina State University, 4700 Hillsborough Street, Raleigh, NC 27606, USA.</p><p class="abstract">Recurrent chromosomal aberrations in solid tumors can reveal the genetic pathways involved in the evolution of a malignancy and in some cases predict biological behavior. However, the role of individual genetic backgrounds in shaping karyotypes of sporadic tumors is unknown. The genetic structure of purebred dog breeds, coupled with their susceptibility to spontaneous cancers, provides a robust model with which to address this question. We tested the hypothesis that there is an association between breed and the distribution of genomic copy number imbalances in naturally occurring canine tumors through assessment of a cohort of Golden Retrievers and Rottweilers diagnosed with spontaneous appendicular osteosarcoma. Our findings reveal significant correlations between breed and tumor karyotypes that are independent of gender, age at diagnosis, and histological classification. These data indicate for the first time that individual genetic backgrounds, as defined by breed in dogs, influence tumor karyotypes in a cancer with extensive genomic instability.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">365-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19337847">19337847</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-009-9028-z">10.1007/s10577-009-9028-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants in the region around Osterix are associated with bone mineral density and growth in childhood.</h4>
<p class="authors"><span class="author">Timpson NJ</span>, <span class="author">Tobias JH</span>, <span class="author">Richards JB</span>, <span class="author">Soranzo N</span>, <span class="author">Duncan EL</span>, <span class="author">Sims AM</span>, <span class="author">Whittaker P</span>, <span class="author">Kumanduri V</span>, <span class="author">Zhai G</span>, <span class="author">Glaser B</span>, <span class="author">Eisman J</span>, <span class="author">Jones G</span>, <span class="author">Nicholson G</span>, <span class="author">Prince R</span>, <span class="author">Seeman E</span>, <span class="author">Spector TD</span>, <span class="author">Brown MA</span>, <span class="author">Peltonen L</span>, <span class="author">Smith GD</span>, <span class="author">Deloukas P</span> and <span class="author">Evans DM</span></p>
<p class="affiliation">MRC Centre for Causal Analyses in Translational Epidemiology, Department of Social Medicine, University of Bristol, Bristol, UK.</p><p class="abstract">Peak bone mass achieved in adolescence is a determinant of bone mass in later life. In order to identify genetic variants affecting bone mineral density (BMD), we performed a genome-wide association study of BMD and related traits in 1518 children from the Avon Longitudinal Study of Parents and Children (ALSPAC). We compared results with a scan of 134 adults with high or low hip BMD. We identified associations with BMD in an area of chromosome 12 containing the Osterix (SP7) locus, a transcription factor responsible for regulating osteoblast differentiation (ALSPAC: P = 5.8 x 10(-4); Australia: P = 3.7 x 10(-4)). This region has previously shown evidence of association with adult hip and lumbar spine BMD in an Icelandic population, as well as nominal association in a UK population. A meta-analysis of these existing studies revealed strong association between SNPs in the Osterix region and adult lumbar spine BMD (P = 9.9 x 10(-11)). In light of these findings, we genotyped a further 3692 individuals from ALSPAC who had whole body BMD and confirmed the association in children as well (P = 5.4 x 10(-5)). Moreover, all SNPs were related to height in ALSPAC children, but not weight or body mass index, and when height was included as a covariate in the regression equation, the association with total body BMD was attenuated. We conclude that genetic variants in the region of Osterix are associated with BMD in children and adults probably through primary effects on growth.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800582</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">18</span>;<span class="number">8</span>;<span class="pagerange">1510-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19181680" rel="external">19181680</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2664147" rel="external">2664147</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp052" rel="external">10.1093/hmg/ddp052</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sir2 paralogues cooperate to regulate virulence genes and antigenic variation in Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Tonkin CJ</span>, <span class="author">Carret CK</span>, <span class="author">Duraisingh MT</span>, <span class="author">Voss TS</span>, <span class="author">Ralph SA</span>, <span class="author">Hommel M</span>, <span class="author">Duffy MF</span>, <span class="author">Silva LM</span>, <span class="author">Scherf A</span>, <span class="author">Ivens A</span>, <span class="author">Speed TP</span>, <span class="author">Beeson JG</span> and <span class="author">Cowman AF</span></p>
<p class="affiliation">The Walter and Eliza Hall Institute of Medical Research, Melbourne, Australia.</p><p class="abstract">Cytoadherance of Plasmodium falciparum-infected erythrocytes in the brain, organs and peripheral microvasculature is linked to morbidity and mortality associated with severe malaria. Parasite-derived P. falciparum Erythrocyte Membrane Protein 1 (PfEMP1) molecules displayed on the erythrocyte surface are responsible for cytoadherance and undergo antigenic variation in the course of an infection. Antigenic variation of PfEMP1 is achieved by in situ switching and mutually exclusive transcription of the var gene family, a process that is controlled by epigenetic mechanisms. Here we report characterisation of the P. falciparum silent information regulator&#39;s A and B (PfSir2A and PfSir2B) and their involvement in mutual exclusion and silencing of the var gene repertoire. Analysis of P. falciparum parasites lacking either PfSir2A or PfSir2B shows that these NAD(+)-dependent histone deacetylases are required for silencing of different var gene subsets classified by their conserved promoter type. We also demonstrate that in the absence of either of these molecules mutually exclusive expression of var genes breaks down. We show that var gene silencing originates within the promoter and PfSir2 paralogues are involved in cis spreading of silenced chromatin into adjacent regions. Furthermore, parasites lacking PfSir2A but not PfSir2B have considerably longer telomeric repeats, demonstrating a role for this molecule in telomeric end protection. This work highlights the pivotal but distinct role for both PfSir2 paralogues in epigenetic silencing of P. falciparum virulence genes and the control of pathogenicity of malaria infection.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">4</span>;<span class="pagerange">e84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19402747" rel="external">19402747</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672602" rel="external">2672602</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1000084" rel="external">10.1371/journal.pbio.1000084</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide haplotype association study identifies the SLC22A3-LPAL2-LPA gene cluster as a risk locus for coronary artery disease.</h4>
<p class="authors"><span class="author">Tr&eacute;gou&euml;t DA</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Erdmann J</span>, <span class="author">Munteanu A</span>, <span class="author">Braund PS</span>, <span class="author">Hall AS</span>, <span class="author">Grosshennig A</span>, <span class="author">Linsel-Nitschke P</span>, <span class="author">Perret C</span>, <span class="author">DeSuremain M</span>, <span class="author">Meitinger T</span>, <span class="author">Wright BJ</span>, <span class="author">Preuss M</span>, <span class="author">Balmforth AJ</span>, <span class="author">Ball SG</span>, <span class="author">Meisinger C</span>, <span class="author">Germain C</span>, <span class="author">Evans A</span>, <span class="author">Arveiler D</span>, <span class="author">Luc G</span>, <span class="author">Ruidavets JB</span>, <span class="author">Morrison C</span>, <span class="author">van der Harst P</span>, <span class="author">Schreiber S</span>, <span class="author">Neureuther K</span>, <span class="author">Sch&auml;fer A</span>, <span class="author">Bugert P</span>, <span class="author">El Mokhtari NE</span>, <span class="author">Schrezenmeir J</span>, <span class="author">Stark K</span>, <span class="author">Rubin D</span>, <span class="author">Wichmann HE</span>, <span class="author">Hengstenberg C</span>, <span class="author">Ouwehand W</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">Ziegler A</span>, <span class="author">Tiret L</span>, <span class="author">Thompson JR</span>, <span class="author">Cambien F</span>, <span class="author">Schunkert H</span> and <span class="author">Samani NJ</span></p>
<p class="affiliation">Institut National de la Sant&eacute; Et de la Recherche M&eacute;dicale (INSERM) Unit&eacute; Mixte de Recherche (UMR_S) 525, Universit&eacute; Pierre et Marie Curie (UPMC). Paris 06, Paris 75013, France. <a href="mailto:%64%61%76%69%64.%74%72%65%67%6F%75%65%74%40%75%70%6D%63.%66%72">&#100;&#97;&#118;&#105;&#100;&#46;&#116;&#114;&#101;&#103;&#111;&#117;&#101;&#116;&#64;&#117;&#112;&#109;&#99;&#46;&#102;&#114;</a></p><p class="abstract">We identify the SLC22A3-LPAL2-LPA gene cluster as a strong susceptibility locus for coronary artery disease (CAD) through a genome-wide haplotype association (GWHA) study. This locus was not identified from previous genome-wide association (GWA) studies focused on univariate analyses of SNPs. The proposed approach may have wide utility for analyzing GWA data for other complex traits.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">3</span>;<span class="pagerange">283-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19198611">19198611</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.314">10.1038/ng.314</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene-environmental interaction regarding alcohol-metabolizing enzymes in the Japanese general population.</h4>
<p class="authors"><span class="author">Tsuchihashi-Makaya M</span>, <span class="author">Serizawa M</span>, <span class="author">Yanai K</span>, <span class="author">Katsuya T</span>, <span class="author">Takeuchi F</span>, <span class="author">Fujioka A</span>, <span class="author">Yamori Y</span>, <span class="author">Ogihara T</span> and <span class="author">Kato N</span></p>
<p class="affiliation">Division of Genomic Epidemiology, Department of Clinical Research and Informatics, Research Institute, International Medical Center of Japan, Tokyo, Japan. <a href="mailto:%6D%69%6D%61%6B%61%79%61%40%72%69.%69%6D%63%6A.%67%6F.%6A%70">&#109;&#105;&#109;&#97;&#107;&#97;&#121;&#97;&#64;&#114;&#105;&#46;&#105;&#109;&#99;&#106;&#46;&#103;&#111;&#46;&#106;&#112;</a></p><p class="abstract">Epidemiological studies have shown that excessive alcohol consumption is a potent risk factor to develop hypertension. In addition, some polymorphisms of the alcohol metabolism genes have been reported to exert significant impacts on the risk of alcoholism. We investigate the relevance of genetic susceptibility to drinking behavior and its influence on the sensitivity to pressor effects of alcohol in the Japanese general population. We initially screened SNPs in four candidate genes by resequencing. From 35 SNPs thus identified, 10 tag SNPs were selected and used for large-scale association analysis in a total of 5724 subjects. Among the SNPs tested, significant association (P&lt;0.001) with drinking behavior was observed for ADH1B Arg47His (rs1229984) and ALDH2 Glu487Lys (rs671) polymorphisms. All subjects with Lys homozygote (AA genotype) of rs671 turned out to be nondrinkers and the combination of two SNP genotypes appeared to substantially influence people&#39;s drinking behavior in a synergistic manner. rs671 was significantly associated with blood pressure (P=0.0001-0.0491) in subgroups of drinkers. In the context of gene-environment interaction, our data clearly show the genetic impacts of two SNPs on drinking behavior and of one SNP on the sensitivity to the pressor effects of alcohol in the Japanese general population.</p>
<p class="publication"><span class="title">Hypertension research : official journal of the Japanese Society of Hypertension</span>&nbsp;<span class="year">2009</span>;<span class="volume">32</span>;<span class="number">3</span>;<span class="pagerange">207-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19262484" rel="external">19262484</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fhr.2009.3" rel="external">10.1038/hr.2009.3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput haplotype determination over long distances by haplotype fusion PCR and ligation haplotyping.</h4>
<p class="authors"><span class="author">Turner DJ</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%64%6A%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#106;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">When combined with haplotype fusion PCR (HF-PCR), ligation haplotyping is a robust, high-throughput method for empirical determination of haplotypes, which can be applied to assaying both sequence and structural variation over long distances. Unlike alternative approaches to haplotype determination, such as allele-specific PCR and long PCR, HF-PCR and ligation haplotyping do not suffer from mispriming or template-switching errors. In this method, HF-PCR is used to juxtapose DNA sequences from single-molecule templates, which contain single-nucleotide polymorphisms (SNPs) or paralogous sequence variants (PSVs) separated by several kilobases. HF-PCR uses an emulsion-based fusion PCR, which can be performed rapidly and in a 96-well format. Subsequently, a ligation-based assay is performed on the HF-PCR products to determine haplotypes. Products are resolved by capillary electrophoresis. Once optimized, the procedure can be performed quickly, taking a day and a half to generate phased haplotypes from genomic DNA.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077014</span>, <span class="grant">077014/Z/05/Z</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">12</span>;<span class="pagerange">1771-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20010928" rel="external">20010928</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871309" rel="external">2871309</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2009.184" rel="external">10.1038/nprot.2009.184</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation sequencing of vertebrate experimental organisms.</h4>
<p class="authors"><span class="author">Turner DJ</span>, <span class="author">Keane TM</span>, <span class="author">Sudbery I</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p><p class="abstract">Next-generation sequencing technologies are revolutionizing biology by allowing for genome-wide transcription factor binding-site profiling, transcriptome sequencing, and more recently, whole-genome resequencing. While it is currently not possible to generate complete de novo assemblies of higher-vertebrate genomes using next-generation sequencing, improvements in sequence read lengths and throughput, coupled with new assembly algorithms for large data sets, will soon make this a reality. These developments will in turn spawn a revolution in how genomic data are used to understand genetics and how model organisms are used for disease gene discovery. This review provides an overview of the current next-generation sequencing platforms and the newest computational tools for the analysis of next-generation sequencing data. We also describe how next-generation sequencing may be applied in the context of vertebrate model organism genetics.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2009</span>;<span class="volume">20</span>;<span class="number">6</span>;<span class="pagerange">327-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19452216">19452216</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2714443">2714443</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-009-9187-4">10.1007/s00335-009-9187-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic similarity of chromosome 6 between patients receiving hematopoietic stem cell transplantation and HLA matched sibling donors.</h4>
<p class="authors"><span class="author">Turpeinen H</span>, <span class="author">Volin L</span>, <span class="author">Nikkinen L</span>, <span class="author">Ojala P</span>, <span class="author">Palotie A</span>, <span class="author">Saarela J</span> and <span class="author">Partanen J</span></p>
<p class="affiliation">Finnish Red Cross Blood Service, Kivihaantie 7, 00310 Helsinki, Finland. <a href="mailto:%68%61%6E%6E%75.%74%75%72%70%65%69%6E%65%6E%40%76%65%72%69%70%61%6C%76%65%6C%75.%66%69">&#104;&#97;&#110;&#110;&#117;&#46;&#116;&#117;&#114;&#112;&#101;&#105;&#110;&#101;&#110;&#64;&#118;&#101;&#114;&#105;&#112;&#97;&#108;&#118;&#101;&#108;&#117;&#46;&#102;&#105;</a></p><p class="abstract"><strong>Background</strong>: Matching for HLA genes located on chromosome 6 is required in hematopoietic stem cell transplantation to reduce the incidence of graft-versus-host disease. However, a considerable proportion of patients still suffer from it, obviously due to genetic differences outside the HLA gene region.</p><p class="abstract">We studied the similarity of almost 4,000 single nucleotide polymorphisms on chromosome 6 between patients receiving hematopoietic stem cell transplantation and their HLA-matched sibling donors.</p><p class="abstract"><strong>Results</strong>: We observed that as a result of routine HLA matching the siblings in fact shared surprisingly long chromosomal fragments with similar single nucleotide polymorphism genotypes--from 11.65 Mb to 134.66 Mb. The number of genes mapped on these shared fragments varied from 402 to 1,302. Considering the whole chromosome 6, the HLA-matched siblings were apparently identical for 65.2-97.8% of the single nucleotide polymorphisms.</p><p class="abstract"><strong>Conclusions</strong>: Potentially, genes similar in some transplantation pairs while different in others might have a significant role in determining the outcome after hematopoietic stem cell transplantation.</p>
<p class="publication"><span class="title">Haematologica</span>&nbsp;<span class="year">2009</span>;<span class="volume">94</span>;<span class="number">4</span>;<span class="pagerange">528-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19278967" rel="external">19278967</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2663616" rel="external">2663616</a>; DOI: <a href="http://dx.doi.org/10.3324%2Fhaematol.2008.000919" rel="external">10.3324/haematol.2008.000919</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high-throughput splinkerette-PCR method for the isolation and sequencing of retroviral insertion sites.</h4>
<p class="authors"><span class="author">Uren AG</span>, <span class="author">Mikkers H</span>, <span class="author">Kool J</span>, <span class="author">van der Weyden L</span>, <span class="author">Lund AH</span>, <span class="author">Wilson CH</span>, <span class="author">Rance R</span>, <span class="author">Jonkers J</span>, <span class="author">van Lohuizen M</span>, <span class="author">Berns A</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Division of Molecular Genetics, Cancer Genomics Centre, Netherlands Cancer Institute, Plesmanlaan, Amsterdam, The Netherlands.</p><p class="abstract">Insertional mutagens such as viruses and transposons are a useful tool for performing forward genetic screens in mice to discover cancer genes. These screens are most effective when performed using hundreds of mice; however, until recently, the cost-effective isolation and sequencing of insertion sites has been a major limitation to performing screens on this scale. Here we present a method for the high-throughput isolation of insertion sites using a highly efficient splinkerette-PCR method coupled with capillary or 454 sequencing. This protocol includes a description of the procedure for DNA isolation, DNA digestion, linker or splinkerette ligation, primary and secondary PCR amplification, and sequencing. This method, which takes about 1 week to perform, has allowed us to isolate hundreds of thousands of insertion sites from mouse tumors and, unlike other methods, has been specifically optimized for the murine leukemia virus (MuLV), and can easily be performed in a 96-well plate format for the efficient multiplex isolation of insertion sites.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6542</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2009</span>;<span class="volume">4</span>;<span class="number">5</span>;<span class="pagerange">789-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19528954">19528954</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3627465">3627465</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2009.64">10.1038/nprot.2009.64</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Megaoesophagus in Rassf1a-null mice.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Happerfield L</span>, <span class="author">Arends MJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Megaoesophagus, or oesophageal achalasia, is a neuromuscular disorder characterized by an absence of peristalsis and flaccid dilatation of the oesophagus, resulting in the retention of ingesta in the dilated segment. The aetiology and pathogenesis of idiopathic (or primary) megaoesophagus are still poorly understood and very little is known about the genetic causes of megaoesophagus in humans. Attempts to develop animal models of this condition have been largely unsuccessful and although the ICRC/HiCri strain of mice spontaneously develop megaoesophagus, the underlying genetic cause remains unknown. In this report, we show that aged Rassf1a-null mice have an enhanced susceptibility to megaoesophagus compared with wild-type littermates (approximately 20%vs. approximately 2% incidence respectively; P = 0.01). Histological examination of the dilated oesophaguses shows a reduction in the numbers of nerve cells (both ganglia and nerve fibres) in the myenteric plexus of the dilated mid and lower oesophagus that was confirmed by S100 immunohistochemistry. There was also a chronic inflammatory infiltrate and subsequent fibrosis of the myenteric plexus and the muscle layers. These appearances closely mimic the gross and histopathological findings in human cases of megaoesophagus/achalasia, thus demonstrating that this is a representative mouse model of the disease. Thus, we have identified a genetic cause of the development of megaoesophagus/achalasia that could be screened for in patients, and may eventually facilitate the development of therapies that could prevent further progression of the disease once it is diagnosed at an early stage.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">International journal of experimental pathology</span>&nbsp;<span class="year">2009</span>;<span class="volume">90</span>;<span class="number">2</span>;<span class="pagerange">101-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19335548">19335548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2676702">2676702</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2613.2008.00635.x">10.1111/j.1365-2613.2008.00635.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic mutations of the histone H3K27 demethylase gene UTX in human cancer.</h4>
<p class="authors"><span class="author">van Haaften G</span>, <span class="author">Dalgliesh GL</span>, <span class="author">Davies H</span>, <span class="author">Chen L</span>, <span class="author">Bignell G</span>, <span class="author">Greenman C</span>, <span class="author">Edkins S</span>, <span class="author">Hardy C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Hinton J</span>, <span class="author">Latimer C</span>, <span class="author">Andrews J</span>, <span class="author">Barthorpe S</span>, <span class="author">Beare D</span>, <span class="author">Buck G</span>, <span class="author">Campbell PJ</span>, <span class="author">Cole J</span>, <span class="author">Forbes S</span>, <span class="author">Jia M</span>, <span class="author">Jones D</span>, <span class="author">Kok CY</span>, <span class="author">Leroy C</span>, <span class="author">Lin ML</span>, <span class="author">McBride DJ</span>, <span class="author">Maddison M</span>, <span class="author">Maquire S</span>, <span class="author">McLay K</span>, <span class="author">Menzies A</span>, <span class="author">Mironenko T</span>, <span class="author">Mulderrig L</span>, <span class="author">Mudie L</span>, <span class="author">Pleasance E</span>, <span class="author">Shepherd R</span>, <span class="author">Smith R</span>, <span class="author">Stebbings L</span>, <span class="author">Stephens P</span>, <span class="author">Tang G</span>, <span class="author">Tarpey PS</span>, <span class="author">Turner R</span>, <span class="author">Turrell K</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Wray P</span>, <span class="author">Collins VP</span>, <span class="author">Ichimura K</span>, <span class="author">Law S</span>, <span class="author">Wong J</span>, <span class="author">Yuen ST</span>, <span class="author">Leung SY</span>, <span class="author">Tonon G</span>, <span class="author">DePinho RA</span>, <span class="author">Tai YT</span>, <span class="author">Anderson KC</span>, <span class="author">Kahnoski RJ</span>, <span class="author">Massie A</span>, <span class="author">Khoo SK</span>, <span class="author">Teh BT</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">Somatically acquired epigenetic changes are present in many cancers. Epigenetic regulation is maintained via post-translational modifications of core histones. Here, we describe inactivating somatic mutations in the histone lysine demethylase gene UTX, pointing to histone H3 lysine methylation deregulation in multiple tumor types. UTX reintroduction into cancer cells with inactivating UTX mutations resulted in slowing of proliferation and marked transcriptional changes. These data identify UTX as a new human cancer gene.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span>, <span class="grant">088340</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">41</span>;<span class="number">5</span>;<span class="pagerange">521-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19330029">19330029</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873835">2873835</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.349">10.1038/ng.349</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improving global and regional resolution of male lineage differentiation by simple single-copy Y-chromosomal short tandem repeat polymorphisms.</h4>
<p class="authors"><span class="author">Vermeulen M</span>, <span class="author">Wollstein A</span>, <span class="author">van der Gaag K</span>, <span class="author">Lao O</span>, <span class="author">Xue Y</span>, <span class="author">Wang Q</span>, <span class="author">Roewer L</span>, <span class="author">Knoblauch H</span>, <span class="author">Tyler-Smith C</span>, <span class="author">de Knijff P</span> and <span class="author">Kayser M</span></p>
<p class="affiliation">Department of Forensic Molecular Biology, Erasmus University Medical Center Rotterdam, 3000 CA Rotterdam, The Netherlands.</p><p class="abstract">We analyzed 67 short tandem repeat polymorphisms from the non-recombining part of the Y-chromosome (Y-STRs), including 49 rarely studied simple single-copy (ss)Y-STRs and 18 widely used Y-STRs, in 590 males from 51 populations belonging to 8 worldwide regions (HGDP-CEPH panel). Although autosomal DNA profiling provided no evidence for close relationship, we found 18 Y-STR haplotypes (defined by 67 Y-STRs) that were shared by two to five men in 13 worldwide populations, revealing high and widespread levels of cryptic male relatedness. Maximal (95.9%) haplotype resolution was achieved with the best 25 out of 67 Y-STRs in the global dataset, and with the best 3-16 markers in regional datasets (89.6-100% resolution). From the 49 rarely studied ssY-STRs, the 25 most informative markers were sufficient to reach the highest possible male lineage differentiation in the global (92.2% resolution), and 3-15 markers in the regional datasets (85.4-100%). Considerably lower haplotype resolutions were obtained with the three commonly used Y-STR sets (Minimal Haplotype, PowerPlex Y, and AmpFlSTR Yfiler. Six ssY-STRs (DYS481, DYS533, DYS549, DYS570, DYS576 and DYS643) were most informative to supplement the existing Y-STR kits for increasing haplotype resolution, or - together with additional ssY-STRs - as a new set for maximizing male lineage differentiation. Mutation rates of the 49 ssY-STRs were estimated from 403 meiotic transfers in deep-rooted pedigrees, and ranged from approximately 4.8 x 10(-4) for 31 ssY-STRs with no mutations observed to 1.3 x 10(-2) and 1.5 x 10(-2) for DYS570 and DYS576, respectively, the latter representing the highest mutation rates reported for human Y-STRs so far. Our findings thus demonstrate that ssY-STRs are useful for maximizing global and regional resolution of male lineages, either as a new set, or when added to commonly used Y-STR sets, and support their application to forensic, genealogical and anthropological studies.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Forensic science international. Genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">3</span>;<span class="number">4</span>;<span class="pagerange">205-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19647704">19647704</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3312386">3312386</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.fsigen.2009.01.009">10.1016/j.fsigen.2009.01.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of smoking initiation and current smoking.</h4>
<p class="authors"><span class="author">Vink JM</span>, <span class="author">Smit AB</span>, <span class="author">de Geus EJ</span>, <span class="author">Sullivan P</span>, <span class="author">Willemsen G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Smit JH</span>, <span class="author">Hoogendijk WJ</span>, <span class="author">Zitman FG</span>, <span class="author">Peltonen L</span>, <span class="author">Kaprio J</span>, <span class="author">Pedersen NL</span>, <span class="author">Magnusson PK</span>, <span class="author">Spector TD</span>, <span class="author">Kyvik KO</span>, <span class="author">Morley KI</span>, <span class="author">Heath AC</span>, <span class="author">Martin NG</span>, <span class="author">Westendorp RG</span>, <span class="author">Slagboom PE</span>, <span class="author">Tiemeier H</span>, <span class="author">Hofman A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Aulchenko YS</span>, <span class="author">Amin N</span>, <span class="author">van Duijn C</span>, <span class="author">Penninx BW</span> and <span class="author">Boomsma DI</span></p>
<p class="affiliation">Department of Biological Psychology, Center for Neurogenomic and Cognitive Research, VU University Amsterdam, The Netherlands. <a href="mailto:%6A%6D.%76%69%6E%6B%40%70%73%79.%76%75.%6E%6C">&#106;&#109;&#46;&#118;&#105;&#110;&#107;&#64;&#112;&#115;&#121;&#46;&#118;&#117;&#46;&#110;&#108;</a></p><p class="abstract">For the identification of genes associated with smoking initiation and current smoking, genome-wide association analyses were carried out in 3497 subjects. Significant genes that replicated in three independent samples (n = 405, 5810, and 1648) were visualized into a biologically meaningful network showing cellular location and direct interaction of their proteins. Several interesting groups of proteins stood out, including glutamate receptors (e.g., GRIN2B, GRIN2A, GRIK2, GRM8), proteins involved in tyrosine kinase receptor signaling (e.g., NTRK2, GRB14), transporters (e.g., SLC1A2, SLC9A9) and cell-adhesion molecules (e.g., CDH23). We conclude that a network-based genome-wide association approach can identify genes influencing smoking behavior.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">MH074027</span>, <span class="grant">MH077139</span>, <span class="grant">MH081802</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2009</span>;<span class="volume">84</span>;<span class="number">3</span>;<span class="pagerange">367-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19268276" rel="external">19268276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2667987" rel="external">2667987</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2009.02.001" rel="external">10.1016/j.ajhg.2009.02.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Milk and two oligosaccharides.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2009</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">483</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19528955">19528955</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2169">10.1038/nrmicro2169</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single domain antibodies against the collagen signalling receptor glycoprotein VI are inhibitors of collagen induced thrombus formation.</h4>
<p class="authors"><span class="author">Walker A</span>, <span class="author">Pugh N</span>, <span class="author">Garner SF</span>, <span class="author">Stephens J</span>, <span class="author">Maddox B</span>, <span class="author">Ouwehand WH</span>, <span class="author">Farndale RW</span>, <span class="author">Steward M</span> and <span class="author">Bloodomics Consortium</span></p>
<p class="affiliation">Domantis Ltd., 315 Cambridge Science Park, Cambridge, UK. <a href="mailto:%41%64%61%6D.%57%61%6C%6B%65%72%40%44%6F%6D%61%6E%74%69%73.%63%6F%6D">&#65;&#100;&#97;&#109;&#46;&#87;&#97;&#108;&#107;&#101;&#114;&#64;&#68;&#111;&#109;&#97;&#110;&#116;&#105;&#115;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Human Domain Antibodies (dAbs) that bind to and inhibit the function of platelet glycoprotein VI (GPVI) have been isolated from phage display libraries and their efficacy demonstrated using in vitro models of platelet activation. Here we describe the properties of one such antibody, BLO8-1, which has been shown to specifically inhibit the binding of recombinant human GPVI to cross-linked collagen related peptide (CRP-XL) in vitro. BLO8-1 specifically binds to the platelet cell surface and prevents CRP-XL induced platelet aggregation in platelet-rich plasma, as well as inhibiting thrombus formation in whole blood under arterial shear conditions. Using a series of mutant GPVI molecules, BLO8-1 was shown to recognize an epitope within the collagen binding domain of GPVI, therefore the anti-thrombotic effect of this dAb is predicted to be due to direct blocking of the collagen-GPVI interaction. These data, together with the desirable properties of Domain Antibodies, show that dAbs could potentially be used to generate novel biopharmaceuticals with anti-thrombotic properties.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/003/27122</span>; Medical Research Council: <span class="grant">G0500707</span></p>
<p class="publication"><span class="title">Platelets</span>&nbsp;<span class="year">2009</span>;<span class="volume">20</span>;<span class="number">4</span>;<span class="pagerange">268-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19459133">19459133</a>; DOI: <a href="http://dx.doi.org/10.1080%2F09537100902893792">10.1080/09537100902893792</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CLIP: construction of cDNA libraries for high-throughput sequencing from RNAs cross-linked to proteins in vivo.</h4>
<p class="authors"><span class="author">Wang Z</span>, <span class="author">Tollervey J</span>, <span class="author">Briese M</span>, <span class="author">Turner D</span> and <span class="author">Ule J</span></p>
<p class="affiliation">MRC-Laboratory of Molecular Biology, Hills Road, Cambridge CB20QH, UK.</p><p class="abstract">UV cross-linking and immunoprecipitation assay (CLIP) can identify direct interaction sites between RNA-binding proteins and RNAs in vivo, and has been used to study several proteins in tissues and cell cultures. The main challenge of the method is to specifically amplify the low amount of isolated RNA. The current protocol is optimised for efficient RNA purification and ligation of barcoded RNA adapters. High-throughput sequencing of the multiplexed cDNA library allows for a comprehensive coverage of the target sequences.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105185858</span>; Wellcome Trust: <span class="grant">089701</span></p>
<p class="publication"><span class="title">Methods (San Diego, Calif.)</span>&nbsp;<span class="year">2009</span>;<span class="volume">48</span>;<span class="number">3</span>;<span class="pagerange">287-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19272451">19272451</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ymeth.2009.02.021">10.1016/j.ymeth.2009.02.021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A HaemAtlas: characterizing gene expression in differentiated human blood cells.</h4>
<p class="authors"><span class="author">Watkins NA</span>, <span class="author">Gusnanto A</span>, <span class="author">de Bono B</span>, <span class="author">De S</span>, <span class="author">Miranda-Saavedra D</span>, <span class="author">Hardie DL</span>, <span class="author">Angenent WG</span>, <span class="author">Attwood AP</span>, <span class="author">Ellis PD</span>, <span class="author">Erber W</span>, <span class="author">Foad NS</span>, <span class="author">Garner SF</span>, <span class="author">Isacke CM</span>, <span class="author">Jolley J</span>, <span class="author">Koch K</span>, <span class="author">Macaulay IC</span>, <span class="author">Morley SL</span>, <span class="author">Rendon A</span>, <span class="author">Rice KM</span>, <span class="author">Taylor N</span>, <span class="author">Thijssen-Timmer DC</span>, <span class="author">Tijssen MR</span>, <span class="author">van der Schoot CE</span>, <span class="author">Wernisch L</span>, <span class="author">Winzer T</span>, <span class="author">Dudbridge F</span>, <span class="author">Buckley CD</span>, <span class="author">Langford CF</span>, <span class="author">Teichmann S</span>, <span class="author">G&ouml;ttgens B</span>, <span class="author">Ouwehand WH</span> and <span class="author">Bloodomics Consortium</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, National Health Service Blood and Transplant, Cambridge, United Kingdom. <a href="mailto:%6E%61%77%32%33%40%63%61%6D.%61%63.%75%6B">&#110;&#97;&#119;&#50;&#51;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Hematopoiesis is a carefully controlled process that is regulated by complex networks of transcription factors that are, in part, controlled by signals resulting from ligand binding to cell-surface receptors. To further understand hematopoiesis, we have compared gene expression profiles of human erythroblasts, megakaryocytes, B cells, cytotoxic and helper T cells, natural killer cells, granulocytes, and monocytes using whole genome microarrays. A bioinformatics analysis of these data was performed focusing on transcription factors, immunoglobulin superfamily members, and lineage-specific transcripts. We observed that the numbers of lineage-specific genes varies by 2 orders of magnitude, ranging from 5 for cytotoxic T cells to 878 for granulocytes. In addition, we have identified novel coexpression patterns for key transcription factors involved in hematopoiesis (eg, GATA3-GFI1 and GATA2-KLF1). This study represents the most comprehensive analysis of gene expression in hematopoietic cells to date and has identified genes that play key roles in lineage commitment and cell function. The data, which are freely accessible, will be invaluable for future studies on hematopoiesis and the role of specific genes and will also aid the understanding of the recent genome-wide association studies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2009</span>;<span class="volume">113</span>;<span class="number">19</span>;<span class="pagerange">e1-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19228925">19228925</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680378">2680378</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2008-06-162958">10.1182/blood-2008-06-162958</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The global consequence of disruption of the AcrAB-TolC efflux pump in Salmonella enterica includes reduced expression of SPI-1 and other attributes required to infect the host.</h4>
<p class="authors"><span class="author">Webber MA</span>, <span class="author">Bailey AM</span>, <span class="author">Blair JM</span>, <span class="author">Morgan E</span>, <span class="author">Stevens MP</span>, <span class="author">Hinton JC</span>, <span class="author">Ivens A</span>, <span class="author">Wain J</span> and <span class="author">Piddock LJ</span></p>
<p class="affiliation">Antimicrobial Agents Research Group, Division of Immunity and Infection, University of Birmingham, Edgbaston, Birmingham, United Kingdom.</p><p class="abstract">The mechanisms by which RND pumps contribute to pathogenicity are currently not understood. Using the AcrAB-TolC system as a paradigm multidrug-resistant efflux pump and Salmonella enterica serovar Typhimurium as a model pathogen, we have demonstrated that AcrA, AcrB, and TolC are each required for efficient adhesion to and invasion of epithelial cells and macrophages by Salmonella in vitro. In addition, AcrB and TolC are necessary for Salmonella to colonize poultry. Mutants lacking acrA, acrB, or tolC showed differential expression of major operons and proteins involved in pathogenesis. These included chemotaxis and motility genes, including cheWY and flgLMK and 14 Salmonella pathogenicity island (SPI)-1-encoded type III secretion system genes, including sopE, and associated effector proteins. Reverse transcription-PCR confirmed these data for identical mutants in two other S. Typhimurium backgrounds. Western blotting showed reduced production of SipA, SipB, and SipC. The absence of AcrB or TolC also caused widespread repression of chemotaxis and motility genes in these mutants, and for acrB::aph, this was associated with decreased motility. For mutants lacking a functional acrA or acrB gene, the nap and nir operons were repressed, and both mutants grew poorly in anaerobic conditions. All phenotypes were restored to that of the wild type by trans-complementation with the wild-type allele of the respective inactivated gene. These data explain how mutants lacking a component of AcrAB-TolC are attenuated and that this phenotype is a result of decreased expression of numerous genes encoding proteins involved in pathogenicity. The link between antibiotic resistance and pathogenicity establishes the AcrAB-TolC system as fundamental to the biology of Salmonella.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2009</span>;<span class="volume">191</span>;<span class="number">13</span>;<span class="pagerange">4276-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19411325" rel="external">19411325</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2698494" rel="external">2698494</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00363-09" rel="external">10.1128/JB.00363-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Swift: primary data analysis for the Illumina Solexa sequencing platform.</h4>
<p class="authors"><span class="author">Whiteford N</span>, <span class="author">Skelly T</span>, <span class="author">Curtis C</span>, <span class="author">Ritchie ME</span>, <span class="author">L&ouml;hr A</span>, <span class="author">Zaranek AW</span>, <span class="author">Abnizova I</span> and <span class="author">Brown C</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6E%61%76%61.%77%68%69%74%65%66%6F%72%64%40%6E%61%6E%6F%70%6F%72%65%74%65%63%68.%63%6F%6D">&#110;&#97;&#118;&#97;&#46;&#119;&#104;&#105;&#116;&#101;&#102;&#111;&#114;&#100;&#64;&#110;&#97;&#110;&#111;&#112;&#111;&#114;&#101;&#116;&#101;&#99;&#104;&#46;&#99;&#111;&#109;</a></p><p class="abstract">MOTIVATION: Primary data analysis methods are of critical importance in second generation DNA sequencing. Improved methods have the potential to increase yield and reduce the error rates. Openly documented analysis tools enable the user to understand the primary data, this is important for the optimization and validity of their scientific work. RESULTS: In this article, we describe Swift, a new tool for performing primary data analysis on the Illumina Solexa Sequencing Platform. Swift is the first tool, outside of the vendors own software, which completes the full analysis process, from raw images through to base calls. As such it provides an alternative to, and independent validation of, the vendor supplied tool. Our results show that Swift is able to increase yield by 13.8%, at comparable error rate.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">17</span>;<span class="pagerange">2194-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19549630" rel="external">19549630</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2734321" rel="external">2734321</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp383" rel="external">10.1093/bioinformatics/btp383</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of the emerging human pathogen Photorhabdus asymbiotica with the insect pathogen Photorhabdus luminescens.</h4>
<p class="authors"><span class="author">Wilkinson P</span>, <span class="author">Waterfield NR</span>, <span class="author">Crossman L</span>, <span class="author">Corton C</span>, <span class="author">Sanchez-Contreras M</span>, <span class="author">Vlisidou I</span>, <span class="author">Barron A</span>, <span class="author">Bignell A</span>, <span class="author">Clark L</span>, <span class="author">Ormond D</span>, <span class="author">Mayho M</span>, <span class="author">Bason N</span>, <span class="author">Smith F</span>, <span class="author">Simmonds M</span>, <span class="author">Churcher C</span>, <span class="author">Harris D</span>, <span class="author">Thompson NR</span>, <span class="author">Quail M</span>, <span class="author">Parkhill J</span> and <span class="author">Ffrench-Constant RH</span></p>
<p class="affiliation">School of Biosciences, University of Exeter in Cornwall, Penryn TR10 9EZ, UK. <a href="mailto:%50.%41.%57%69%6C%6B%69%6E%73%6F%6E%40%65%78%65%74%65%72.%61%63.%75%6B">&#80;&#46;&#65;&#46;&#87;&#105;&#108;&#107;&#105;&#110;&#115;&#111;&#110;&#64;&#101;&#120;&#101;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The Gram-negative bacterium Photorhabdus asymbiotica (Pa) has been recovered from human infections in both North America and Australia. Recently, Pa has been shown to have a nematode vector that can also infect insects, like its sister species the insect pathogen P. luminescens (Pl). To understand the relationship between pathogenicity to insects and humans in Photorhabdus we have sequenced the complete genome of Pa strain ATCC43949 from North America. This strain (formerly referred to as Xenorhabdus luminescens strain 2) was isolated in 1977 from the blood of an 80 year old female patient with endocarditis, in Maryland, USA. Here we compare the complete genome of Pa ATCC43949 with that of the previously sequenced insect pathogen P. luminescens strain TT01 which was isolated from its entomopathogenic nematode vector collected from soil in Trinidad and Tobago.</p><p class="abstract"><strong>Results</strong>: We found that the human pathogen Pa had a smaller genome (5,064,808 bp) than that of the insect pathogen Pl (5,688,987 bp) but that each pathogen carries approximately one megabase of DNA that is unique to each strain. The reduced size of the Pa genome is associated with a smaller diversity in insecticidal genes such as those encoding the Toxin complexes (Tc&#39;s), Makes caterpillars floppy (Mcf) toxins and the Photorhabdus Virulence Cassettes (PVCs). The Pa genome, however, also shows the addition of a plasmid related to pMT1 from Yersinia pestis and several novel pathogenicity islands including a novel Type Three Secretion System (TTSS) encoding island. Together these data suggest that Pa may show virulence against man via the acquisition of the pMT1-like plasmid and specific effectors, such as SopB, that promote its persistence inside human macrophages. Interestingly the loss of insecticidal genes in Pa is not reflected by a loss of pathogenicity towards insects.</p><p class="abstract"><strong>Conclusion</strong>: Our results suggest that North American isolates of Pa have acquired virulence against man via the acquisition of a plasmid and specific virulence factors with similarity to those shown to play roles in pathogenicity against humans in other bacteria.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E021328/1</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2009</span>;<span class="volume">10</span>;<span class="pagerange">302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19583835">19583835</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2717986">2717986</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-10-302">10.1186/1471-2164-10-302</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Signal initiation in biological systems: the properties and detection of transient extracellular protein interactions.</h4>
<p class="authors"><span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%67%77%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#119;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Individual cells within biological systems frequently coordinate their functions through signals initiated by specific extracellular protein interactions involving receptors that bridge the cellular membrane. Due to their biochemical nature, these membrane-embedded receptor proteins are difficult to manipulate and their interactions are characterised by very weak binding strengths that cannot be detected using popular high throughput assays. This review will provide a general outline of the biochemical attributes of receptor proteins focussing in particular on the biophysical properties of their transient interactions. Methods that are able to detect these weak extracellular binding events and especially those that can be used for identifying novel interactions will be compared. Finally, I discuss the feasibility of constructing a complete and accurate extracellular protein interaction map, and the methods that are likely to be useful in achieving this goal.</p>
<p class="publication"><span class="title">Molecular bioSystems</span>&nbsp;<span class="year">2009</span>;<span class="volume">5</span>;<span class="number">12</span>;<span class="pagerange">1405-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19593473">19593473</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2898632">2898632</a>; DOI: <a href="http://dx.doi.org/10.1039%2FB903580J">10.1039/B903580J</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CARM1 is required in embryonic stem cells to maintain pluripotency and resist differentiation.</h4>
<p class="authors"><span class="author">Wu Q</span>, <span class="author">Bruce AW</span>, <span class="author">Jedrusik A</span>, <span class="author">Ellis PD</span>, <span class="author">Andrews RM</span>, <span class="author">Langford CF</span>, <span class="author">Glover DM</span> and <span class="author">Zernicka-Goetz M</span></p>
<p class="affiliation">Wellcome Trust and Cancer Research UK Gurdon Institute, Cambridge, United Kingdom.</p><p class="abstract">Histone H3 methylation at R17 and R26 recently emerged as a novel epigenetic mechanism regulating pluripotency in mouse embryos. Blastomeres of four-cell embryos with high H3 methylation at these sites show unrestricted potential, whereas those with lower levels cannot support development when aggregated in chimeras of like cells. Increasing histone H3 methylation, through expression of coactivator-associated-protein-arginine-methyltransferase 1 (CARM1) in embryos, elevates expression of key pluripotency genes and directs cells to the pluripotent inner cell mass. We demonstrate CARM1 is also required for the self-renewal and pluripotency of embryonic stem (ES) cells. In ES cells, CARM1 depletion downregulates pluripotency genes leading to their differentiation. CARM1 associates with Oct4/Pou5f1 and Sox2 promoters that display detectable levels of R17/26 histone H3 methylation. In CARM1 overexpressing ES cells, histone H3 arginine methylation is also at the Nanog promoter to which CARM1 now associates. Such cells express Nanog at elevated levels and delay their response to differentiation signals. Thus, like in four-cell embryo blastomeres, histone H3 arginine methylation by CARM1 in ES cells allows epigenetic modulation of pluripotency.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300723</span>, <span class="grant">G0800784</span>; Wellcome Trust: <span class="grant">064421</span></p>
<p class="publication"><span class="title">Stem cells (Dayton, Ohio)</span>&nbsp;<span class="year">2009</span>;<span class="volume">27</span>;<span class="number">11</span>;<span class="pagerange">2637-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19544422">19544422</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fstem.131">10.1002/stem.131</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human Y chromosome base-substitution mutation rate measured by direct sequencing in a deep-rooting pedigree.</h4>
<p class="authors"><span class="author">Xue Y</span>, <span class="author">Wang Q</span>, <span class="author">Long Q</span>, <span class="author">Ng BL</span>, <span class="author">Swerdlow H</span>, <span class="author">Burton J</span>, <span class="author">Skuce C</span>, <span class="author">Taylor R</span>, <span class="author">Abdellah Z</span>, <span class="author">Zhao Y</span>, <span class="author">Asan</span>, <span class="author">MacArthur DG</span>, <span class="author">Quail MA</span>, <span class="author">Carter NP</span>, <span class="author">Yang H</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambs CB10 1SA, UK. <a href="mailto:%79%6C%78%40%73%61%6E%67%65%72.%61%63.%75%6B">&#121;&#108;&#120;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Understanding the key process of human mutation is important for many aspects of medical genetics and human evolution. In the past, estimates of mutation rates have generally been inferred from phenotypic observations or comparisons of homologous sequences among closely related species. Here, we apply new sequencing technology to measure directly one mutation rate, that of base substitutions on the human Y chromosome. The Y chromosomes of two individuals separated by 13 generations were flow sorted and sequenced by Illumina (Solexa) paired-end sequencing to an average depth of 11x or 20x, respectively. Candidate mutations were further examined by capillary sequencing in cell-line and blood DNA from the donors and additional family members. Twelve mutations were confirmed in approximately 10.15 Mb; eight of these had occurred in vitro and four in vivo. The latter could be placed in different positions on the pedigree and led to a mutation-rate measurement of 3.0 x 10(-8) mutations/nucleotide/generation (95% CI: 8.9 x 10(-9)-7.0 x 10(-8)), consistent with estimates of 2.3 x 10(-8)-6.3 x 10(-8) mutations/nucleotide/generation for the same Y-chromosomal region from published human-chimpanzee comparisons depending on the generation and split times assumed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2009</span>;<span class="volume">19</span>;<span class="number">17</span>;<span class="pagerange">1453-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19716302">19716302</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2748900">2748900</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2009.07.032">10.1016/j.cub.2009.07.032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of Paint Probes by Flow-Sorted and Mocrodissected Chromosomes</h4>
<p class="authors"><span class="author">Yang</span>, <span class="author">F</span>, <span class="author">Trifonov</span>, <span class="author">V.A</span>, <span class="author">Ng</span>, <span class="author">B. L.</span>, <span class="author">Kosyakova</span>, <span class="author">N</span>, <span class="author">Carter</span>, <span class="author">N. P.</span></p>
<p class="publication"><span class="title">Fluorescence In Situ Hybridization (FISH) - Application Guide.</span> <span class="year">2009</span>;<span class="pagerange">35-52</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</h4>
<p class="authors"><span class="author">Ye K</span>, <span class="author">Schulz MH</span>, <span class="author">Long Q</span>, <span class="author">Apweiler R</span> and <span class="author">Ning Z</span></p>
<p class="affiliation">EMBL Outstation European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6B.%79%65%40%6C%75%6D%63.%6E%6C">&#107;&#46;&#121;&#101;&#64;&#108;&#117;&#109;&#99;&#46;&#110;&#108;</a></p><p class="abstract"><strong>Motivation</strong>: There is a strong demand in the genomic community to develop effective algorithms to reliably identify genomic variants. Indel detection using next-gen data is difficult and identification of long structural variations is extremely challenging.</p><p class="abstract"><strong>Results</strong>: We present Pindel, a pattern growth approach, to detect breakpoints of large deletions and medium-sized insertions from paired-end short reads. We use both simulated reads and real data to demonstrate the efficiency of the computer program and accuracy of the results.</p><p class="abstract"><strong>Availability</strong>: The binary code and a short user manual can be freely downloaded from http://www.ebi.ac.uk/ approximately kye/pindel/.</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%6B.%79%65%40%6C%75%6D%63.%6E%6C">&#107;&#46;&#121;&#101;&#64;&#108;&#117;&#109;&#99;&#46;&#110;&#108;</a>; <a href="mailto:%7A%6E%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#122;&#110;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2009</span>;<span class="volume">25</span>;<span class="number">21</span>;<span class="pagerange">2865-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19561018" rel="external">19561018</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2781750" rel="external">2781750</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp394" rel="external">10.1093/bioinformatics/btp394</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction of enteric bacterial pathogens with murine embryonic stem cells.</h4>
<p class="authors"><span class="author">Yu J</span>, <span class="author">Rossi R</span>, <span class="author">Hale C</span>, <span class="author">Goulding D</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6A%79%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#121;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Embryonic stem (ES) cells are susceptible to genetic manipulation and retain the potential to differentiate into diverse cell types, which are factors that make them potentially attractive cells for studying host-pathogen interactions. Murine ES cells were found to be susceptible to invasion by Salmonella enterica serovar Typhimurium and Shigella flexneri and to the formation of attaching and effacing lesions by enteropathogenic Escherichia coli. S. enterica serovar Typhimurium and S. flexneri cell entry was dependent on the Salmonella pathogenicity island 1 and Shigella mxi/spa type III secretion systems, respectively. Microscopy studies indicated that both S. enterica serovar Typhimurium and S. flexneri were located in intracellular niches in ES cells that were similar to the niches occupied in differentiated cells. ES cells were eventually killed following bacterial invasion, but no evidence of activation of classical caspase-associated apoptotic or innate immune pathways was found. To demonstrate the potential of mutant ES cells, we employed an ES cell line defective in cholesterol synthesis and found that the mutant cells were less susceptible to infection by Salmonella and Shigella than the parental ES cells. Thus, we highlighted the practical use of genetically modified ES cells for studying microbe-host interactions.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2009</span>;<span class="volume">77</span>;<span class="number">2</span>;<span class="pagerange">585-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19029302" rel="external">19029302</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2632033" rel="external">2632033</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.01003-08" rel="external">10.1128/IAI.01003-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of transgene-free induced pluripotent mouse stem cells by the piggyBac transposon.</h4>
<p class="authors"><span class="author">Yusa K</span>, <span class="author">Rad R</span>, <span class="author">Takeda J</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Induced pluripotent stem cells (iPSCs) have been generated from somatic cells by transgenic expression of Oct4 (Pou5f1), Sox2, Klf4 and Myc. A major difficulty in the application of this technology for regenerative medicine, however, is the delivery of reprogramming factors. Whereas retroviral transduction increases the risk of tumorigenicity, transient expression methods have considerably lower reprogramming efficiencies. Here we describe an efficient piggyBac transposon-based approach to generate integration-free iPSCs. Transposons carrying 2A peptide-linked reprogramming factors induced reprogramming of mouse embryonic fibroblasts with equivalent efficiencies to retroviral transduction. We removed transposons from these primary iPSCs by re-expressing transposase. Transgene-free iPSCs could be identified by negative selection. piggyBac excised without a footprint, leaving the iPSC genome without any genetic alteration. iPSCs fulfilled all criteria of pluripotency, such as pluripotency gene expression, teratoma formation and contribution to chimeras. piggyBac transposon-based reprogramming may be used to generate therapeutically applicable iPSCs.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077187</span>, <span class="grant">WT077187</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2009</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">363-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19337237">19337237</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2677165">2677165</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1323">10.1038/nmeth.1323</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/yl9liqf1"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/yl9liqf1" src="/qr/yl9liqf1.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:55:34 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
