<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2013 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2013.html 14982 2013-02-18 01:48:54Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2013</h1><p>Number of papers published in 2013: <strong>481</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bloomsbury report on mouse embryo phenotyping: recommendations from the IMPC workshop on embryonic lethal screening.</h4>
<p class="authors"><span class="author">Adams D</span>, <span class="author">Baldock R</span>, <span class="author">Bhattacharya S</span>, <span class="author">Copp AJ</span>, <span class="author">Dickinson M</span>, <span class="author">Greene ND</span>, <span class="author">Henkelman M</span>, <span class="author">Justice M</span>, <span class="author">Mohun T</span>, <span class="author">Murray SA</span>, <span class="author">Pauws E</span>, <span class="author">Raess M</span>, <span class="author">Rossant J</span>, <span class="author">Weaver T</span> and <span class="author">West D</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, CB10 1SA, UK.</p>
<p class="abstract">Identifying genes that are important for embryo development is a crucial first step towards understanding their many functions in driving the ordered growth, differentiation and organogenesis of embryos. It can also shed light on the origins of developmental disease and congenital abnormalities. Current international efforts to examine gene function in the mouse provide a unique opportunity to pinpoint genes that are involved in embryogenesis, owing to the emergence of embryonic lethal knockout mutants. Through internationally coordinated efforts, the International Knockout Mouse Consortium (IKMC) has generated a public resource of mouse knockout strains and, in April 2012, the International Mouse Phenotyping Consortium (IMPC), supported by the EU InfraCoMP programme, convened a workshop to discuss developing a phenotyping pipeline for the investigation of embryonic lethal knockout lines. This workshop brought together over 100 scientists, from 13 countries, who are working in the academic and commercial research sectors, including experts and opinion leaders in the fields of embryology, animal imaging, data capture, quality control and annotation, high-throughput mouse production, phenotyping, and reporter gene analysis. This article summarises the outcome of the workshop, including (1) the vital scientific importance of phenotyping embryonic lethal mouse strains for basic and translational research; (2) a common framework to harmonise international efforts within this context; (3) the types of phenotyping that are likely to be most appropriate for systematic use, with a focus on 3D embryo imaging; (4) the importance of centralising data in a standardised form to facilitate data mining; and (5) the development of online tools to allow open access to and dissemination of the phenotyping data.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">P30 HD024064</span></p>
<p class="publication"><span class="title">Disease models &amp;amp; mechanisms</span>&nbsp;<span class="year">2013</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">571-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23519032">23519032</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdmm.011833">10.1242/dmm.011833</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacteriotherapy for the treatment of intestinal dysbiosis caused by Clostridium difficile infection.</h4>
<p class="authors"><span class="author">Adamu BO</span> and <span class="author">Lawley TD</span></p>
<p class="affiliation">Bacterial Pathogenesis Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Faecal microbiota transplantation (FMT) has been used for more than five decades to treat a variety of intestinal diseases associated with pathological imbalances within the resident microbiota, termed dysbiosis. FMT has been particularly effective for treating patients with recurrent Clostridium difficile infection who are left with few clinical options other than continued antibiotic therapy. Our increasing knowledge of the structure and function of the human intestinal microbiota and C. difficile pathogenesis has led to the understanding that FMT promotes intestinal ecological restoration and highlights the microbiota as a viable therapeutic target. However, the use of undefined faecal samples creates a barrier for widespread clinical use because of safety and aesthetic issues. An emerging concept of bacteriotherapy, the therapeutic use of a defined mixture of harmless, health-associated bacteria, holds promise for the treatment of patients with severe C. difficile infection, and possibly represents a paradigm shift for the treatment of diseases linked to intestinal dysbiosis.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">93614</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Current opinion in microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">16</span>;<span class="number">5</span>;<span class="pagerange">596-601</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23866975">23866975</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3840269">3840269</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mib.2013.06.009">10.1016/j.mib.2013.06.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamic image-based modelling of kidney branching morphogenesis</h4>
<p class="authors"><span class="author">Adivarahan,S.;</span>, <span class="author">Menshykau,D.;</span>, <span class="author">MICHOS,O.;</span> and <span class="author">Iber,D.</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science</span>&nbsp;<span class="year">2013</span>;<span class="volume">8130 LNBI</span>;<span class="pagerange">106-19</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-40708-6_9">10.1007/978-3-642-40708-6_9</a>; URL: <a title="http://link.springer.com/chapter/10.1007%2F978-3-642-40708-6_9" href="http://link.springer.com/chapter/10.1007%2F978-3-642-40708-6_9">http://link.springer.com/chapter<span class="print-hide">...</span><span class="print-show">/10.1007%2F978-3-642-4</span>0708-6_9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing ancient calcified dental plaque shows changes in oral microbiota with dietary shifts of the Neolithic and Industrial revolutions.</h4>
<p class="authors"><span class="author">Adler CJ</span>, <span class="author">Dobney K</span>, <span class="author">Weyrich LS</span>, <span class="author">Kaidonis J</span>, <span class="author">Walker AW</span>, <span class="author">Haak W</span>, <span class="author">Bradshaw CJ</span>, <span class="author">Townsend G</span>, <span class="author">So&#x142;tysiak A</span>, <span class="author">Alt KW</span>, <span class="author">Parkhill J</span> and <span class="author">Cooper A</span></p>
<p class="affiliation">Australian Centre for Ancient DNA, School of Earth and Environmental Sciences, The University of Adelaide, Adelaide, South Australia, Australia.</p><p class="abstract">The importance of commensal microbes for human health is increasingly recognized, yet the impacts of evolutionary changes in human diet and culture on commensal microbiota remain almost unknown. Two of the greatest dietary shifts in human evolution involved the adoption of carbohydrate-rich Neolithic (farming) diets (beginning &sim;10,000 years before the present) and the more recent advent of industrially processed flour and sugar (in &sim;1850). Here, we show that calcified dental plaque (dental calculus) on ancient teeth preserves a detailed genetic record throughout this period. Data from 34 early European skeletons indicate that the transition from hunter-gatherer to farming shifted the oral microbial community to a disease-associated configuration. The composition of oral microbiota remained unexpectedly constant between Neolithic and medieval times, after which (the now ubiquitous) cariogenic bacteria became dominant, apparently during the Industrial Revolution. Modern oral microbiotic ecosystems are markedly less diverse than historic populations, which might be contributing to chronic oral (and other) disease in postindustrial lifestyles.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span>, <span class="grant">WT092799/Z/10/Z</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">4</span>;<span class="pagerange">450-5, 455e1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23416520">23416520</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3996550">3996550</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2536">10.1038/ng.2536</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Partial sleep restriction activates immune response-related gene expression pathways: experimental and epidemiological studies in humans.</h4>
<p class="authors"><span class="author">Aho V</span>, <span class="author">Ollila HM</span>, <span class="author">Rantanen V</span>, <span class="author">Kronholm E</span>, <span class="author">Surakka I</span>, <span class="author">van Leeuwen WM</span>, <span class="author">Lehto M</span>, <span class="author">Matikainen S</span>, <span class="author">Ripatti S</span>, <span class="author">H&auml;rm&auml; M</span>, <span class="author">Sallinen M</span>, <span class="author">Salomaa V</span>, <span class="author">Jauhiainen M</span>, <span class="author">Alenius H</span>, <span class="author">Paunio T</span> and <span class="author">Porkka-Heiskanen T</span></p>
<p class="affiliation">Department of Physiology, Institute of Biomedicine, University of Helsinki, Helsinki, Finland.</p>
<p class="abstract">Epidemiological studies have shown that short or insufficient sleep is associated with increased risk for metabolic diseases and mortality. To elucidate mechanisms behind this connection, we aimed to identify genes and pathways affected by experimentally induced, partial sleep restriction and to verify their connection to insufficient sleep at population level. The experimental design simulated sleep restriction during a working week: sleep of healthy men (N = 9) was restricted to 4 h/night for five nights. The control subjects (N = 4) spent 8 h/night in bed. Leukocyte RNA expression was analyzed at baseline, after sleep restriction, and after recovery using whole genome microarrays complemented with pathway and transcription factor analysis. Expression levels of the ten most up-regulated and ten most down-regulated transcripts were correlated with subjective assessment of insufficient sleep in a population cohort (N = 472). Experimental sleep restriction altered the expression of 117 genes. Eight of the 25 most up-regulated transcripts were related to immune function. Accordingly, fifteen of the 25 most up-regulated Gene Ontology pathways were also related to immune function, including those for B cell activation, interleukin 8 production, and NF-κB signaling (P<0.005). Of the ten most up-regulated genes, expression of STX16 correlated negatively with self-reported insufficient sleep in a population sample, while three other genes showed tendency for positive correlation. Of the ten most down-regulated genes, TBX21 and LGR6 correlated negatively and TGFBR3 positively with insufficient sleep. Partial sleep restriction affects the regulation of signaling pathways related to the immune system. Some of these changes appear to be long-lasting and may at least partly explain how prolonged sleep restriction can contribute to inflammation-associated pathological states, such as cardiometabolic diseases.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e77184</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24194869">24194869</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0077184">10.1371/journal.pone.0077184</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CCL3L1 copy number, HIV load, and immune reconstitution in sub-Saharan Africans.</h4>
<p class="authors"><span class="author">Aklillu E</span>, <span class="author">Odenthal-Hesse L</span>, <span class="author">Bowdrey J</span>, <span class="author">Habtewold A</span>, <span class="author">Ngaimisi E</span>, <span class="author">Yimer G</span>, <span class="author">Amogne W</span>, <span class="author">Mugusi S</span>, <span class="author">Minzi O</span>, <span class="author">Makonnen E</span>, <span class="author">Janabi M</span>, <span class="author">Mugusi F</span>, <span class="author">Aderaye G</span>, <span class="author">Hardwick R</span>, <span class="author">Fu B</span>, <span class="author">Viskaduraki M</span>, <span class="author">Yang F</span> and <span class="author">Hollox EJ</span></p>
<p class="abstract">Background: The role of copy number variation of the CCL3L1 gene, encoding MIP1alpha, in contributing to the host variation in susceptibility and response to HIV infection is controversial. Here we analyse a sub-Saharan African cohort from Tanzania and Ethiopia, two countries with a high prevalence of HIV-1 and a high co-morbidity of HIV with tuberculosis.

Methods: We use a form of quantitative PCR called the paralogue ratio test to determine CCL3L1 gene copy number in 1134 individuals and validate our copy number typing using array comparative genomic hybridisation and fiber-FISH.

Results: We find no significant association of CCL3L1 gene copy number with HIV load in antiretroviral-naive patients prior to initiation of combination highly active anti-retroviral therapy. However, we find a significant association of low CCL3L1 gene copy number with improved immune reconstitution following initiation of highly active anti-retroviral therapy (p = 0.012), replicating a previous study.

Conclusions: Our work supports a role for CCL3L1 copy number in immune constitution following antiretroviral therapy in HIV, and suggests that the MIP1alpha -CCR5 axis might be targeted to aid immune reconstitution.</p>
<p class="publication"><span class="title">BMC infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">536</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24219137">24219137</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3829100">3829100</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2334-13-536">10.1186/1471-2334-13-536</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New insights into the genetic basis of TAR (thrombocytopenia-absent radii) syndrome.</h4>
<p class="authors"><span class="author">Albers CA</span>, <span class="author">Newbury-Ecob R</span>, <span class="author">Ouwehand WH</span> and <span class="author">Ghevaert C</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, UK; NHS Blood and Transplant, Cambridge, UK; Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. Electronic address: <a href="mailto:%63.%61%6C%62%65%72%73%40%67%65%6E.%75%6D%63%6E.%6E%6C">&#99;&#46;&#97;&#108;&#98;&#101;&#114;&#115;&#64;&#103;&#101;&#110;&#46;&#117;&#109;&#99;&#110;&#46;&#110;&#108;</a>.</p>
<p class="abstract">Thrombocytopenia with absent radii (TAR) syndrome is a rare disorder combining specific skeletal abnormalities with a reduced platelet count. Rare proximal microdeletions of 1q21.1 are found in the majority of patients but are also found in unaffected parents. Recently it was shown that TAR syndrome is caused by the compound inheritance of a low-frequency noncoding SNP and a rare null allele in RBM8A, a gene encoding the exon-junction complex subunit member Y14 located in the deleted region. This finding provides new insight into the complex inheritance pattern and new clues to the molecular mechanisms underlying TAR syndrome. We discuss TAR syndrome in the context of abnormal phenotypes associated with proximal and distal 1q21.1 microdeletion and microduplications with incomplete penetrance and variable expressivity.</p>
<p class="publication"><span class="title">Current opinion in genetics &amp;amp; development</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23602329">23602329</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gde.2013.02.015">10.1016/j.gde.2013.02.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">AHT-ChIP-seq: a completely automated robotic protocol for high-throughput chromatin immunoprecipitation.</h4>
<p class="authors"><span class="author">Aldridge S</span>, <span class="author">Watt S</span>, <span class="author">Quail MA</span>, <span class="author">Rayner T</span>, <span class="author">Lukk M</span>, <span class="author">Bimson MF</span>, <span class="author">Gaffney D</span> and <span class="author">Odom DT</span></p><p class="abstract">ChIP-seq is an established manually-performed method for identifying DNA-protein interactions genome-wide. Here, we describe a protocol for automated high-throughput (AHT) ChIP-seq. To demonstrate the quality of data obtained using AHT-ChIP-seq, we applied it to five proteins in mouse livers using a single 96-well plate, demonstrating an extremely high degree of qualitative and quantitative reproducibility among biological and technical replicates. We estimated the optimum and minimum recommended cell numbers required to perform AHT-ChIP-seq by running an additional plate using HepG2 and MCF7 cells. With this protocol, commercially available robotics can perform four hundred experiments in five days.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">15603</span>, <span class="grant">A10185</span>; European Research Council: <span class="grant">202218</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">11</span>;<span class="pagerange">R124</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24200198">24200198</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4053851">4053851</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-11-r124">10.1186/gb-2013-14-11-r124</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Specificity and heterogeneity of terahertz radiation effect on gene expression in mouse mesenchymal stem cells.</h4>
<p class="authors"><span class="author">Alexandrov BS</span>, <span class="author">Phipps ML</span>, <span class="author">Alexandrov LB</span>, <span class="author">Booshehri LG</span>, <span class="author">Erat A</span>, <span class="author">Zabolotny J</span>, <span class="author">Mielke CH</span>, <span class="author">Chen HT</span>, <span class="author">Rodriguez G</span>, <span class="author">Rasmussen K&Oslash;</span>, <span class="author">Martinez JS</span>, <span class="author">Bishop AR</span> and <span class="author">Usheva A</span></p>
<p class="affiliation">Theoretical Division, Los Alamos National Laboratory , Los Alamos, NM 87545, USA ; Harvard Medical School, Beth Israel Deaconess Medical Center, Department of Medicine , Boston, MA 02215, USA.</p>
<p class="abstract">We report that terahertz (THz) irradiation of mouse mesenchymal stem cells (mMSCs) with a single-frequency (SF) 2.52â€…THz laser or pulsed broadband (centered at 10â€…THz) source results in irradiation specific heterogenic changes in gene expression. The THz effect depends on irradiation parameters such as the duration and type of THz source, and on the degree of stem cell differentiation. Our microarray survey and RT-PCR experiments demonstrate that prolonged broadband THz irradiation drives mMSCs toward differentiation, while 2-hour irradiation (regardless of THz sources) affects genes transcriptionally active in pluripotent stem cells. The strictly controlled experimental environment indicates minimal temperature changes and the absence of any discernable response to heat shock and cellular stress genes imply a non-thermal response. Computer simulations of the core promoters of two pluripotency markers reveal association between gene upregulation and propensity for DNA breathing. We propose that THz radiation has potential for non-contact control of cellular gene expression.</p>
<p class="publication"><span class="title">Scientific reports</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="pagerange">1184</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23378916">23378916</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3560359">3560359</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsrep01184">10.1038/srep01184</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Signatures of mutational processes in human cancer.</h4>
<p class="authors"><span class="author">Alexandrov LB</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Wedge DC</span>, <span class="author">Aparicio SA</span>, <span class="author">Behjati S</span>, <span class="author">Biankin AV</span>, <span class="author">Bignell GR</span>, <span class="author">Bolli N</span>, <span class="author">Borg A</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">Boyault S</span>, <span class="author">Burkhardt B</span>, <span class="author">Butler AP</span>, <span class="author">Caldas C</span>, <span class="author">Davies HR</span>, <span class="author">Desmedt C</span>, <span class="author">Eils R</span>, <span class="author">Eyfj&ouml;rd JE</span>, <span class="author">Foekens JA</span>, <span class="author">Greaves M</span>, <span class="author">Hosoda F</span>, <span class="author">Hutter B</span>, <span class="author">Ilicic T</span>, <span class="author">Imbeaud S</span>, <span class="author">Imielinski M</span>, <span class="author">Imielinsk M</span>, <span class="author">J&auml;ger N</span>, <span class="author">Jones DT</span>, <span class="author">Jones D</span>, <span class="author">Knappskog S</span>, <span class="author">Kool M</span>, <span class="author">Lakhani SR</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span>, <span class="author">Martin S</span>, <span class="author">Munshi NC</span>, <span class="author">Nakamura H</span>, <span class="author">Northcott PA</span>, <span class="author">Pajic M</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Paradiso A</span>, <span class="author">Pearson JV</span>, <span class="author">Puente XS</span>, <span class="author">Raine K</span>, <span class="author">Ramakrishna M</span>, <span class="author">Richardson AL</span>, <span class="author">Richter J</span>, <span class="author">Rosenstiel P</span>, <span class="author">Schlesner M</span>, <span class="author">Schumacher TN</span>, <span class="author">Span PN</span>, <span class="author">Teague JW</span>, <span class="author">Totoki Y</span>, <span class="author">Tutt AN</span>, <span class="author">Vald&eacute;s-Mas R</span>, <span class="author">van Buuren MM</span>, <span class="author">van &#39;t Veer L</span>, <span class="author">Vincent-Salomon A</span>, <span class="author">Waddell N</span>, <span class="author">Yates LR</span>, <span class="author">Australian Pancreatic Cancer Genome Initiative</span>, <span class="author">ICGC Breast Cancer Consortium</span>, <span class="author">ICGC MMML-Seq Consortium</span>, <span class="author">ICGC PedBrain</span>, <span class="author">Zucman-Rossi J</span>, <span class="author">Futreal PA</span>, <span class="author">McDermott U</span>, <span class="author">Lichter P</span>, <span class="author">Meyerson M</span>, <span class="author">Grimmond SM</span>, <span class="author">Siebert R</span>, <span class="author">Campo E</span>, <span class="author">Shibata T</span>, <span class="author">Pfister SM</span>, <span class="author">Campbell PJ</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">All cancers are caused by somatic mutations; however, understanding of the biological processes generating these mutations is limited. The catalogue of somatic mutations from a cancer genome bears the signatures of the mutational processes that have been operative. Here we analysed 4,938,362 mutations from 7,042 cancers and extracted more than 20 distinct mutational signatures. Some are present in many cancer types, notably a signature attributed to the APOBEC family of cytidine deaminases, whereas others are confined to a single cancer class. Certain signatures are associated with age of the patient at cancer diagnosis, known mutagenic exposures or defects in DNA maintenance, but many are of cryptic origin. In addition to these genome-wide mutational signatures, hypermutation localized to small genomic regions, &#39;kataegis&#39;, is found in many cancer types. The results reveal the diversity of mutational processes underlying the development of cancer, with potential implications for understanding of cancer aetiology, prevention and therapy.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">T32 CA009216</span>; Wellcome Trust: <span class="grant">093867</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">500</span>;<span class="number">7463</span>;<span class="pagerange">415-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23945592">23945592</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3776390">3776390</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12477">10.1038/nature12477</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deciphering signatures of mutational processes operative in human cancer.</h4>
<p class="authors"><span class="author">Alexandrov LB</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Wedge DC</span>, <span class="author">Campbell PJ</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">The genome of a cancer cell carries somatic mutations that are the cumulative consequences of the DNA damage and repair processes operative during the cellular lineage between the fertilized egg and the cancer cell. Remarkably, these mutational processes are poorly characterized. Global sequencing initiatives are yielding catalogs of somatic mutations from thousands of cancers, thus providing the unique opportunity to decipher the signatures of mutational processes operative in human cancer. However, until now there have been no theoretical models describing the signatures of mutational processes operative in cancer genomes and no systematic computational approaches are available to decipher these mutational signatures. Here, by modeling mutational processes as a blind source separation problem, we introduce a computational framework that effectively addresses these questions. Our approach provides a basis for characterizing mutational signatures from cancer-derived somatic mutational catalogs, paving the way to insights into the pathogenetic mechanism underlying all cancers.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span>, <span class="grant">093867</span>, <span class="grant">098051</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Cell reports</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">246-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23318258">23318258</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3588146">3588146</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.celrep.2012.12.008">10.1016/j.celrep.2012.12.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modeling the association of space, time, and host species with variation of the HA, NA, and NS genes of H5N1 highly pathogenic avian influenza viruses isolated from birds in Romania in 2005-2007.</h4>
<p class="authors"><span class="author">Alkhamis M</span>, <span class="author">Perez A</span>, <span class="author">Batey N</span>, <span class="author">Howard W</span>, <span class="author">Baillie G</span>, <span class="author">Watson S</span>, <span class="author">Franz S</span>, <span class="author">Focosi-Snyman R</span>, <span class="author">Onita I</span>, <span class="author">Cioranu R</span>, <span class="author">Turcitu M</span>, <span class="author">Kellam P</span>, <span class="author">Brown IH</span> and <span class="author">Breed AC</span></p>
<p class="affiliation">Center for Animal Disease Modeling and Surveillance (CADMS), School of Veterinary Medicine, One Shields Avenue, University of California, Davis, CA 95616, USA. <a href="mailto:%6D%61%61%6C%6B%61%6D%65%65%73%40%75%63%64%61%76%69%73.%65%64%75">&#109;&#97;&#97;&#108;&#107;&#97;&#109;&#101;&#101;&#115;&#64;&#117;&#99;&#100;&#97;&#118;&#105;&#115;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Molecular characterization studies of a diverse collection of avian influenza viruses (AIVs) have demonstrated that AIVs&#39; greatest genetic variability lies in the HA, NA, and NS genes. The objective here was to quantify the association between geographical locations, periods of time, and host species and pairwise nucleotide variation in the HA, NA, and NS genes of 70 isolates of H5N1 highly pathogenic avian influenza virus (HPAIV) collected from October 2005 to December 2007 from birds in Romania. A mixed-binomial Bayesian regression model was used to quantify the probability of nucleotide variation between isolates and its association with space, time, and host species. As expected for the three target genes, a higher probability of nucleotide differences (odds ratios [ORs] &gt; 1) was found between viruses sampled from places at greater geographical distances from each other, viruses sampled over greater periods of time, and viruses derived from different species. The modeling approach in the present study maybe useful in further understanding the molecular epidemiology of H5N1 HPAI virus in bird populations. The methodology presented here will be useful in predicting the most likely genetic distance for any of the three gene segments of viruses that have not yet been isolated or sequenced based on space, time, and host species during the course of an epidemic.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span>, <span class="grant">093724</span></p>
<p class="publication"><span class="title">Avian diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">57</span>;<span class="number">3</span>;<span class="pagerange">612-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24283126">24283126</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3980047">3980047</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The anatomy of successful computational biology software.</h4>
<p class="authors"><span class="author">Altschul S</span>, <span class="author">Demchak B</span>, <span class="author">Durbin R</span>, <span class="author">Gentleman R</span>, <span class="author">Krzywinski M</span>, <span class="author">Li H</span>, <span class="author">Nekrutenko A</span>, <span class="author">Robinson J</span>, <span class="author">Rasband W</span>, <span class="author">Taylor J</span> and <span class="author">Trapnell C</span></p>
<p class="affiliation">National Center for Biotechnology Information, Bethesda, Maryland.</p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2013</span>;<span class="volume">31</span>;<span class="number">10</span>;<span class="pagerange">894-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24104757">24104757</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.2721">10.1038/nbt.2721</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inappropriately low hepcidin levels in patients with myelodysplastic syndrome carrying a somatic mutation of SF3B1.</h4>
<p class="authors"><span class="author">Ambaglio I</span>, <span class="author">Malcovati L</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Laarakkers CM</span>, <span class="author">Della Porta MG</span>, <span class="author">Gall&igrave; A</span>, <span class="author">Da Vi&agrave; MC</span>, <span class="author">Bono E</span>, <span class="author">Ubezio M</span>, <span class="author">Travaglino E</span>, <span class="author">Albertini R</span>, <span class="author">Campbell PJ</span>, <span class="author">Swinkels DW</span> and <span class="author">Cazzola M</span></p>
<p class="affiliation"><a href="mailto:%6C%75%63%61.%6D%61%6C%63%6F%76%61%74%69%40%75%6E%69%70%76.%69%74">&#108;&#117;&#99;&#97;&#46;&#109;&#97;&#108;&#99;&#111;&#118;&#97;&#116;&#105;&#64;&#117;&#110;&#105;&#112;&#118;&#46;&#105;&#116;</a>.</p>
<p class="abstract">Somatic mutations of the RNA splicing machinery have been recently identified in myelodysplastic syndromes. In particular, a strong association has been found between SF3B1 mutation and refractory anemia with ring sider-oblasts, a condition characterized by ineffective erythropoiesis and parenchymal iron overload. We studied the relationship between SF3B1 mutation, erythroid activity and hepcidin levels in myelodysplastic syndrome patients. Erythroid activity was evaluated through the proportion of marrow erythroblasts, soluble transferrin receptor and serum growth differentiation factor 15. Significant relationships were found between SF3B1 mutation and marrow erythroblasts (P=0.001), soluble transferrin receptor (P=0.003) and serum growth differentiation factor 15 (P=0.033). Serum hepcidin varied considerably, and multivariable analysis showed that the hepcidin to ferritin ratio, a measure of adequacy of hepcidin levels relative to body iron stores, was inversely related to the SF3B1 mutation (P=0.013). These observations suggest that patients with SF3B1 mutation have inappropriately low hepcidin levels, which may explain their propensity to parenchymal iron loading.</p>
<p class="publication"><span class="title">Haematologica</span>&nbsp;<span class="year">2013</span>;<span class="volume">98</span>;<span class="number">3</span>;<span class="pagerange">420-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23300182">23300182</a>; DOI: <a href="http://dx.doi.org/10.3324%2Fhaematol.2012.077446">10.3324/haematol.2012.077446</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The African coelacanth genome provides insights into tetrapod evolution.</h4>
<p class="authors"><span class="author">Amemiya CT</span>, <span class="author">Alf&ouml;ldi J</span>, <span class="author">Lee AP</span>, <span class="author">Fan S</span>, <span class="author">Philippe H</span>, <span class="author">Maccallum I</span>, <span class="author">Braasch I</span>, <span class="author">Manousaki T</span>, <span class="author">Schneider I</span>, <span class="author">Rohner N</span>, <span class="author">Organ C</span>, <span class="author">Chalopin D</span>, <span class="author">Smith JJ</span>, <span class="author">Robinson M</span>, <span class="author">Dorrington RA</span>, <span class="author">Gerdol M</span>, <span class="author">Aken B</span>, <span class="author">Biscotti MA</span>, <span class="author">Barucca M</span>, <span class="author">Baurain D</span>, <span class="author">Berlin AM</span>, <span class="author">Blatch GL</span>, <span class="author">Buonocore F</span>, <span class="author">Burmester T</span>, <span class="author">Campbell MS</span>, <span class="author">Canapa A</span>, <span class="author">Cannon JP</span>, <span class="author">Christoffels A</span>, <span class="author">De Moro G</span>, <span class="author">Edkins AL</span>, <span class="author">Fan L</span>, <span class="author">Fausto AM</span>, <span class="author">Feiner N</span>, <span class="author">Forconi M</span>, <span class="author">Gamieldien J</span>, <span class="author">Gnerre S</span>, <span class="author">Gnirke A</span>, <span class="author">Goldstone JV</span>, <span class="author">Haerty W</span>, <span class="author">Hahn ME</span>, <span class="author">Hesse U</span>, <span class="author">Hoffmann S</span>, <span class="author">Johnson J</span>, <span class="author">Karchner SI</span>, <span class="author">Kuraku S</span>, <span class="author">Lara M</span>, <span class="author">Levin JZ</span>, <span class="author">Litman GW</span>, <span class="author">Mauceli E</span>, <span class="author">Miyake T</span>, <span class="author">Mueller MG</span>, <span class="author">Nelson DR</span>, <span class="author">Nitsche A</span>, <span class="author">Olmo E</span>, <span class="author">Ota T</span>, <span class="author">Pallavicini A</span>, <span class="author">Panji S</span>, <span class="author">Picone B</span>, <span class="author">Ponting CP</span>, <span class="author">Prohaska SJ</span>, <span class="author">Przybylski D</span>, <span class="author">Saha NR</span>, <span class="author">Ravi V</span>, <span class="author">Ribeiro FJ</span>, <span class="author">Sauka-Spengler T</span>, <span class="author">Scapigliati G</span>, <span class="author">Searle SM</span>, <span class="author">Sharpe T</span>, <span class="author">Simakov O</span>, <span class="author">Stadler PF</span>, <span class="author">Stegeman JJ</span>, <span class="author">Sumiyama K</span>, <span class="author">Tabbaa D</span>, <span class="author">Tafer H</span>, <span class="author">Turner-Maier J</span>, <span class="author">van Heusden P</span>, <span class="author">White S</span>, <span class="author">Williams L</span>, <span class="author">Yandell M</span>, <span class="author">Brinkmann H</span>, <span class="author">Volff JN</span>, <span class="author">Tabin CJ</span>, <span class="author">Shubin N</span>, <span class="author">Schartl M</span>, <span class="author">Jaffe DB</span>, <span class="author">Postlethwait JH</span>, <span class="author">Venkatesh B</span>, <span class="author">Di Palma F</span>, <span class="author">Lander ES</span>, <span class="author">Meyer A</span> and <span class="author">Lindblad-Toh K</span></p>
<p class="affiliation">Molecular Genetics Program, Benaroya Research Institute, Seattle, Washington 98101, USA. <a href="mailto:%63%61%6D%65%6D%69%79%61%40%62%65%6E%61%72%6F%79%61%72%65%73%65%61%72%63%68.%6F%72%67">&#99;&#97;&#109;&#101;&#109;&#105;&#121;&#97;&#64;&#98;&#101;&#110;&#97;&#114;&#111;&#121;&#97;&#114;&#101;&#115;&#101;&#97;&#114;&#99;&#104;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The discovery of a living coelacanth specimen in 1938 was remarkable, as this lineage of lobe-finned fish was thought to have become extinct 70 million years ago. The modern coelacanth looks remarkably similar to many of its ancient relatives, and its evolutionary proximity to our own fish ancestors provides a glimpse of the fish that first walked on land. Here we report the genome sequence of the African coelacanth, Latimeria chalumnae. Through a phylogenomic analysis, we conclude that the lungfish, and not the coelacanth, is the closest living relative of tetrapods. Coelacanth protein-coding genes are significantly more slowly evolving than those of tetrapods, unlike other genomic features. Analyses of changes in genes and regulatory elements during the vertebrate adaptation to land highlight genes involved in immunity, nitrogen excretion and the development of fins, tail, ear, eye, brain and olfaction. Functional assays of enhancers involved in the fin-to-limb transition and in the emergence of extra-embryonic tissues show the importance of the coelacanth genome as a blueprint for understanding tetrapod evolution.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U137761446</span>; NCRR NIH HHS: <span class="grant">R24 RR032670</span>; NHGRI NIH HHS: <span class="grant">R01 HG003474</span>, <span class="grant">U54 HG003067</span>; NICHD NIH HHS: <span class="grant">R37 HD032443</span>; NIEHS NIH HHS: <span class="grant">P42 ES007381</span>, <span class="grant">R01 ES006272</span>; NIH HHS: <span class="grant">R01 OD011116</span>, <span class="grant">R24 OD011199</span>; Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">496</span>;<span class="number">7445</span>;<span class="pagerange">311-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23598338">23598338</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3633110">3633110</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12027">10.1038/nature12027</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phosphoinositide 3-Kinase δ Gene Mutation Predisposes to Respiratory Infection and Airway Damage.</h4>
<p class="authors"><span class="author">Angulo I</span>, <span class="author">Vadas O</span>, <span class="author">Gar&ccedil;on F</span>, <span class="author">Banham-Hall E</span>, <span class="author">Plagnol V</span>, <span class="author">Leahy TR</span>, <span class="author">Baxendale H</span>, <span class="author">Coulter T</span>, <span class="author">Curtis J</span>, <span class="author">Wu C</span>, <span class="author">Blake-Palmer K</span>, <span class="author">Perisic O</span>, <span class="author">Smyth D</span>, <span class="author">Maes M</span>, <span class="author">Fiddler C</span>, <span class="author">Juss J</span>, <span class="author">Cilliers D</span>, <span class="author">Markelj G</span>, <span class="author">Chandra A</span>, <span class="author">Farmer G</span>, <span class="author">Kielkowska A</span>, <span class="author">Clark J</span>, <span class="author">Kracker S</span>, <span class="author">Debr&eacute; M</span>, <span class="author">Picard C</span>, <span class="author">Pellier I</span>, <span class="author">Jabado N</span>, <span class="author">Morris JA</span>, <span class="author">Barcenas-Morales G</span>, <span class="author">Fischer A</span>, <span class="author">Stephens L</span>, <span class="author">Hawkins P</span>, <span class="author">Barrett JC</span>, <span class="author">Abinun M</span>, <span class="author">Clatworthy M</span>, <span class="author">Durandy A</span>, <span class="author">Doffinger R</span>, <span class="author">Chilvers E</span>, <span class="author">Cant AJ</span>, <span class="author">Kumararatne D</span>, <span class="author">Okkenhaug K</span>, <span class="author">Williams RL</span>, <span class="author">Condliffe A</span> and <span class="author">Nejentsev S</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Cambridge, UK.</p>
<p class="abstract">Genetic mutations cause primary immunodeficiencies (PIDs), which predispose to infections. Here we describe Activated PI3K-δ Syndrome (APDS), a PID associated with a dominant gain-of-function mutation in which lysine replaced glutamic acid at residue 1021 (E1021K) in the p110δ protein, the catalytic subunit of phosphoinositide 3-kinase δ (PI3Kδ), encoded by the PIK3CD gene. We found E1021K in 17 patients from seven unrelated families, but not among 3346 healthy subjects. APDS was characterized by recurrent respiratory infections, progressive airway damage, lymphopenia, increased circulating transitional B cells, increased immunoglobulin M and reduced immunoglobulin G2 levels in serum and impaired vaccine responses. The E1021K mutation enhanced membrane association and kinase activity of p110δ. Patient-derived lymphocytes had increased levels of phosphatidylinositol 3,4,5-trisphosphate and phosphorylated AKT protein and were prone to activation-induced cell death. Selective p110δ inhibitors IC87114 and GS-1101 reduced the activity of the mutant enzyme in vitro, which suggested a therapeutic approach for patients with APDS.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24136356">24136356</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1243292">10.1126/science.1243292</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The COMBREX Project: Design, Methodology, and Initial Results.</h4>
<p class="authors"><span class="author">Anton BP</span>, <span class="author">Chang YC</span>, <span class="author">Brown P</span>, <span class="author">Choi HP</span>, <span class="author">Faller LL</span>, <span class="author">Guleria J</span>, <span class="author">Hu Z</span>, <span class="author">Klitgord N</span>, <span class="author">Levy-Moonshine A</span>, <span class="author">Maksad A</span>, <span class="author">Mazumdar V</span>, <span class="author">McGettrick M</span>, <span class="author">Osmani L</span>, <span class="author">Pokrzywa R</span>, <span class="author">Rachlin J</span>, <span class="author">Swaminathan R</span>, <span class="author">Allen B</span>, <span class="author">Housman G</span>, <span class="author">Monahan C</span>, <span class="author">Rochussen K</span>, <span class="author">Tao K</span>, <span class="author">Bhagwat AS</span>, <span class="author">Brenner SE</span>, <span class="author">Columbus L</span>, <span class="author">de Cr&eacute;cy-Lagard V</span>, <span class="author">Ferguson D</span>, <span class="author">Fomenkov A</span>, <span class="author">Gadda G</span>, <span class="author">Morgan RD</span>, <span class="author">Osterman AL</span>, <span class="author">Rodionov DA</span>, <span class="author">Rodionova IA</span>, <span class="author">Rudd KE</span>, <span class="author">S&ouml;ll D</span>, <span class="author">Spain J</span>, <span class="author">Xu SY</span>, <span class="author">Bateman A</span>, <span class="author">Blumenthal RM</span>, <span class="author">Bollinger JM</span>, <span class="author">Chang WS</span>, <span class="author">Ferrer M</span>, <span class="author">Friedberg I</span>, <span class="author">Galperin MY</span>, <span class="author">Gobeill J</span>, <span class="author">Haft D</span>, <span class="author">Hunt J</span>, <span class="author">Karp P</span>, <span class="author">Klimke W</span>, <span class="author">Krebs C</span>, <span class="author">Macelis D</span>, <span class="author">Madupu R</span>, <span class="author">Martin MJ</span>, <span class="author">Miller JH</span>, <span class="author">O&#39;Donovan C</span>, <span class="author">Palsson B</span>, <span class="author">Ruch P</span>, <span class="author">Setterdahl A</span>, <span class="author">Sutton G</span>, <span class="author">Tate J</span>, <span class="author">Yakunin A</span>, <span class="author">Tchigvintsev D</span>, <span class="author">Plata G</span>, <span class="author">Hu J</span>, <span class="author">Greiner R</span>, <span class="author">Horn D</span>, <span class="author">Sj&ouml;lander K</span>, <span class="author">Salzberg SL</span>, <span class="author">Vitkup D</span>, <span class="author">Letovsky S</span>, <span class="author">Segr&egrave; D</span>, <span class="author">Delisi C</span>, <span class="author">Roberts RJ</span>, <span class="author">Steffen M</span> and <span class="author">Kasif S</span></p>
<p class="affiliation">New England Biolabs, Ipswich, Massachusetts, United States of America.</p><p class="abstract">Experimental data exists for only a vanishingly small fraction of sequenced microbial genes. This community page discusses the progress made by the COMBREX project to address this important issue using both computational and experimental resources.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">8</span>;<span class="pagerange">e1001638</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24013487">24013487</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3754883">3754883</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1001638">10.1371/journal.pbio.1001638</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide meta-analysis identifies new susceptibility loci for migraine.</h4>
<p class="authors"><span class="author">Anttila V</span>, <span class="author">Winsvold BS</span>, <span class="author">Gormley P</span>, <span class="author">Kurth T</span>, <span class="author">Bettella F</span>, <span class="author">McMahon G</span>, <span class="author">Kallela M</span>, <span class="author">Malik R</span>, <span class="author">de Vries B</span>, <span class="author">Terwindt G</span>, <span class="author">Medland SE</span>, <span class="author">Todt U</span>, <span class="author">McArdle WL</span>, <span class="author">Quaye L</span>, <span class="author">Koiranen M</span>, <span class="author">Ikram MA</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Stam AH</span>, <span class="author">Ligthart L</span>, <span class="author">Wedenoja J</span>, <span class="author">Dunham I</span>, <span class="author">Neale BM</span>, <span class="author">Palta P</span>, <span class="author">Hamalainen E</span>, <span class="author">Sch&uuml;rks M</span>, <span class="author">Rose LM</span>, <span class="author">Buring JE</span>, <span class="author">Ridker PM</span>, <span class="author">Steinberg S</span>, <span class="author">Stefansson H</span>, <span class="author">Jakobsson F</span>, <span class="author">Lawlor DA</span>, <span class="author">Evans DM</span>, <span class="author">Ring SM</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Artto V</span>, <span class="author">Kaunisto MA</span>, <span class="author">Freilinger T</span>, <span class="author">Schoenen J</span>, <span class="author">Frants RR</span>, <span class="author">Pelzer N</span>, <span class="author">Weller CM</span>, <span class="author">Zielman R</span>, <span class="author">Heath AC</span>, <span class="author">Madden PA</span>, <span class="author">Montgomery GW</span>, <span class="author">Martin NG</span>, <span class="author">Borck G</span>, <span class="author">G&ouml;bel H</span>, <span class="author">Heinze A</span>, <span class="author">Heinze-Kuhn K</span>, <span class="author">Williams FM</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">van den Ende J</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">Amin N</span>, <span class="author">Hottenga JJ</span>, <span class="author">Vink JM</span>, <span class="author">Heikkil&auml; K</span>, <span class="author">Alexander M</span>, <span class="author">Muller-Myhsok B</span>, <span class="author">Schreiber S</span>, <span class="author">Meitinger T</span>, <span class="author">Wichmann HE</span>, <span class="author">Aromaa A</span>, <span class="author">Eriksson JG</span>, <span class="author">Traynor BJ</span>, <span class="author">Trabzuni D</span>, <span class="author">Rossin E</span>, <span class="author">Lage K</span>, <span class="author">Jacobs SB</span>, <span class="author">Gibbs JR</span>, <span class="author">Birney E</span>, <span class="author">Kaprio J</span>, <span class="author">Penninx BW</span>, <span class="author">Boomsma DI</span>, <span class="author">van Duijn C</span>, <span class="author">Raitakari O</span>, <span class="author">Jarvelin MR</span>, <span class="author">Zwart JA</span>, <span class="author">Cherkas L</span>, <span class="author">Strachan DP</span>, <span class="author">Kubisch C</span>, <span class="author">Ferrari MD</span>, <span class="author">van den Maagdenberg AM</span>, <span class="author">Dichgans M</span>, <span class="author">Wessman M</span>, <span class="author">Smith GD</span>, <span class="author">Stefansson K</span>, <span class="author">Daly MJ</span>, <span class="author">Nyholt DR</span>, <span class="author">Chasman DI</span>, <span class="author">Palotie A</span>, <span class="author">North American Brain Expression Consortium</span>, <span class="author">UK Brain Expression Consortium</span> and <span class="author">International Headache Genetics Consortium</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%61%6E%74%74%69%6C%61%40%61%74%67%75.%6D%67%68.%68%61%72%76%61%72%64.%65%64%75">&#97;&#110;&#116;&#116;&#105;&#108;&#97;&#64;&#97;&#116;&#103;&#117;&#46;&#109;&#103;&#104;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Migraine is the most common brain disorder, affecting approximately 14% of the adult population, but its molecular mechanisms are poorly understood. We report the results of a meta-analysis across 29 genome-wide association studies, including a total of 23,285 individuals with migraine (cases) and 95,425 population-matched controls. We identified 12 loci associated with migraine susceptibility (P&lt;5&times;10(-8)). Five loci are new: near AJAP1 at 1p36, near TSPAN2 at 1p13, within FHL5 at 6q16, within C7orf10 at 7p14 and near MMP16 at 8q21. Three of these loci were identified in disease subgroup analyses. Brain tissue expression quantitative trait locus analysis suggests potential functional candidate genes at four loci: APOA1BP, TBC1D7, FUT9, STAT6 and ATP5B.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0802462</span>, <span class="grant">G9815508</span>, <span class="grant">MC_UU_12013/4</span>; NIA NIH HHS: <span class="grant">Z01 AG000949-02</span>; NIAAA NIH HHS: <span class="grant">K05 AA017688</span>; NIGMS NIH HHS: <span class="grant">T32 GM007753</span>; Wellcome Trust: <span class="grant">089062</span>, <span class="grant">092731</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">8</span>;<span class="pagerange">912-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23793025">23793025</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4041123">4041123</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2676">10.1038/ng.2676</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of cytokine and toll-like receptor gene polymorphisms with severe malaria in three regions of cameroon.</h4>
<p class="authors"><span class="author">Apinjoh TO</span>, <span class="author">Anchang-Kimbi JK</span>, <span class="author">Njua-Yafi C</span>, <span class="author">Mugri RN</span>, <span class="author">Ngwai AN</span>, <span class="author">Rockett KA</span>, <span class="author">Mbunwe E</span>, <span class="author">Besingi RN</span>, <span class="author">Clark TG</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Achidi EA</span> and <span class="author">MalariaGEN Consortium</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology, University of Buea, Buea, Cameroon.</p>
<p class="abstract">P. falciparum malaria is one of the most widespread and deadliest infectious diseases in children under five years in endemic areas. The disease has been a strong force for evolutionary selection in the human genome, and uncovering the critical human genetic factors that confer resistance to the disease would provide clues to the molecular basis of protective immunity that would be invaluable for vaccine development. We investigated the effect of single nucleotide polymorphisms (SNPs) on malaria pathology in a case- control study of 1862 individuals from two major ethnic groups in three regions with intense perennial P. falciparum transmission in Cameroon. Twenty nine polymorphisms in cytokine and toll-like receptor (TLR) genes as well as the sickle cell trait (HbS) were assayed on the Sequenom iPLEX platform. Our results confirm the known protective effect of HbS against severe malaria and also reveal a protective effect of SNPs in interleukin-10 (IL10) cerebral malaria and hyperpyrexia. Furthermore, IL17RE rs708567 GA and hHbS rs334 AT individuals were associated with protection from uncomplicated malaria and anaemia respectively in this study. Meanwhile, individuals with the hHbS rs334 TT, IL10 rs3024500 AA, and IL17RD rs6780995 GA genotypes were more susceptible to severe malarial anaemia, cerebral malaria, and hyperpyrexia respectively. Taken together, our results suggest that polymorphisms in some immune response genes may have important implications for the susceptibility to severe malaria in Cameroonians. Moreover using uncomplicated malaria may allow us to identify novel pathways in the early development of the disease.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e81071</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24312262">24312262</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3842328">3842328</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0081071">10.1371/journal.pone.0081071</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide, whole mount in situ analysis of transcriptional regulators in zebrafish embryos.</h4>
<p class="authors"><span class="author">Armant O</span>, <span class="author">M&auml;rz M</span>, <span class="author">Schmidt R</span>, <span class="author">Ferg M</span>, <span class="author">Diotel N</span>, <span class="author">Ertzer R</span>, <span class="author">Bryne JC</span>, <span class="author">Yang L</span>, <span class="author">Baader I</span>, <span class="author">Reischl M</span>, <span class="author">Legradi J</span>, <span class="author">Mikut R</span>, <span class="author">Stemple D</span>, <span class="author">van IJcken W</span>, <span class="author">van der Sloot A</span>, <span class="author">Lenhard B</span>, <span class="author">Str&auml;hle U</span> and <span class="author">Rastegar S</span></p>
<p class="affiliation">Institute of Toxicology and Genetics, Karlsruhe Institute of Technology, Postfach 3640, 76021 Karlsruhe, Germany.</p><p class="abstract">Transcription is the primary step in the retrieval of genetic information. A substantial proportion of the protein repertoire of each organism consists of transcriptional regulators (TRs). It is believed that the differential expression and combinatorial action of these TRs is essential for vertebrate development and body homeostasis. We mined the zebrafish genome exhaustively for genes encoding TRs and determined their expression in the zebrafish embryo by sequencing to saturation and in situ hybridisation. At the evolutionary conserved phylotypic stage, 75% of the 3302 TR genes encoded in the genome are already expressed. The number of expressed TR genes increases only marginally in subsequent stages and is maintained during adulthood suggesting important roles of the TR genes in body homeostasis. Fewer than half of the TR genes (45%, n=1711 genes) are expressed in a tissue-restricted manner in the embryo. Transcripts of 207 genes were detected in a single tissue in the 24h embryo, potentially acting as regulators of specific processes. Other TR genes were expressed in multiple tissues. However, with the exception of certain territories in the nervous system, we did not find significant synexpression suggesting that most tissue-restricted TRs act in a freely combinatorial fashion. Our data indicate that elaboration of body pattern and function from the phylotypic stage onward relies mostly on redeployment of TRs and post-transcriptional processes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_UP_1102/1</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">380</span>;<span class="number">2</span>;<span class="pagerange">351-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23684812">23684812</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2013.05.006">10.1016/j.ydbio.2013.05.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The general population cohort in rural south-western Uganda: a platform for communicable and non-communicable disease studies.</h4>
<p class="authors"><span class="author">Asiki G</span>, <span class="author">Murphy G</span>, <span class="author">Nakiyingi-Miiro J</span>, <span class="author">Seeley J</span>, <span class="author">Nsubuga RN</span>, <span class="author">Karabarinde A</span>, <span class="author">Waswa L</span>, <span class="author">Biraro S</span>, <span class="author">Kasamba I</span>, <span class="author">Pomilla C</span>, <span class="author">Maher D</span>, <span class="author">Young EH</span>, <span class="author">Kamali A</span>, <span class="author">Sandhu MS</span> and <span class="author">On behalf of the GPC team</span></p>
<p class="affiliation">Medical Research Council/Uganda Virus Research Institute (MRC/UVRI), Uganda Research Unit on AIDS, Entebbe, Uganda, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK, Wellcome Trust Sanger Institute, Hinxton, UK, London School of Hygiene and Tropical Medicine, London, UK, School of International Development, University of East Anglia, Norwich, UK and Wellcome Trust, UK (formerly with MRC/UVRI Uganda Research Unit on AIDS, Entebbe, Uganda).</p>
<p class="abstract">The General Population Cohort (GPC) was set up in 1989 to examine trends in HIV prevalence and incidence, and their determinants in rural south-western Uganda. Recently, the research questions have included the epidemiology and genetics of communicable and non-communicable diseases (NCDs) to address the limited data on the burden and risk factors for NCDs in sub-Saharan Africa. The cohort comprises all residents (52% aged â‰¥13years, men and women in equal proportions) within one-half of a rural sub-county, residing in scattered houses, and largely farmers of three major ethnic groups. Data collected through annual surveys include; mapping for spatial analysis and participant location; census for individual socio-demographic and household socioeconomic status assessment; and a medical survey for health, lifestyle and biophysical and blood measurements to ascertain disease outcomes and risk factors for selected participants. This cohort offers a rich platform to investigate the interplay between communicable diseases and NCDs. There is robust infrastructure for data management, sample processing and storage, and diverse expertise in epidemiology, social and basic sciences. For any data access enquiries you may contact the director, MRC/UVRI, Uganda Research Unit on AIDS by email to <a href="mailto:%6D%72%63%40%6D%72%63%75%67%61%6E%64%61.%6F%72%67">&#109;&#114;&#99;&#64;&#109;&#114;&#99;&#117;&#103;&#97;&#110;&#100;&#97;&#46;&#111;&#114;&#103;</a> or the corresponding author.</p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23364209">23364209</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdys234">10.1093/ije/dys234</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hospital outbreak of Middle East respiratory syndrome coronavirus.</h4>
<p class="authors"><span class="author">Assiri A</span>, <span class="author">McGeer A</span>, <span class="author">Perl TM</span>, <span class="author">Price CS</span>, <span class="author">Al Rabeeah AA</span>, <span class="author">Cummings DA</span>, <span class="author">Alabdullatif ZN</span>, <span class="author">Assad M</span>, <span class="author">Almulhim A</span>, <span class="author">Makhdoom H</span>, <span class="author">Madani H</span>, <span class="author">Alhakeem R</span>, <span class="author">Al-Tawfiq JA</span>, <span class="author">Cotten M</span>, <span class="author">Watson SJ</span>, <span class="author">Kellam P</span>, <span class="author">Zumla AI</span>, <span class="author">Memish ZA</span> and <span class="author">KSA MERS-CoV Investigation Team</span></p>
<p class="affiliation">Global Center for Mass Gatherings Medicine, Ministry of Health, Riyadh, Saudi Arabia.</p><p class="abstract"><strong>Background</strong>: In September 2012, the World Health Organization reported the first cases of pneumonia caused by the novel Middle East respiratory syndrome coronavirus (MERS-CoV). We describe a cluster of health care-acquired MERS-CoV infections.</p><p class="abstract"><strong>Methods</strong>: Medical records were reviewed for clinical and demographic information and determination of potential contacts and exposures. Case patients and contacts were interviewed. The incubation period and serial interval (the time between the successive onset of symptoms in a chain of transmission) were estimated. Viral RNA was sequenced.</p><p class="abstract"><strong>Results</strong>: Between April 1 and May 23, 2013, a total of 23 cases of MERS-CoV infection were reported in the eastern province of Saudi Arabia. Symptoms included fever in 20 patients (87%), cough in 20 (87%), shortness of breath in 11 (48%), and gastrointestinal symptoms in 8 (35%); 20 patients (87%) presented with abnormal chest radiographs. As of June 12, a total of 15 patients (65%) had died, 6 (26%) had recovered, and 2 (9%) remained hospitalized. The median incubation period was 5.2 days (95% confidence interval [CI], 1.9 to 14.7), and the serial interval was 7.6 days (95% CI, 2.5 to 23.1). A total of 21 of the 23 cases were acquired by person-to-person transmission in hemodialysis units, intensive care units, or in-patient units in three different health care facilities. Sequencing data from four isolates revealed a single monophyletic clade. Among 217 household contacts and more than 200 health care worker contacts whom we identified, MERS-CoV infection developed in 5 family members (3 with laboratory-confirmed cases) and in 2 health care workers (both with laboratory-confirmed cases).</p><p class="abstract"><strong>Conclusions</strong>: Person-to-person transmission of MERS-CoV can occur in health care settings and may be associated with considerable morbidity. Surveillance and infection-control measures are critical to a global public health response.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U01 GM070708</span>, <span class="grant">U54 GM088491</span>; Wellcome Trust: <span class="grant">093724</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">369</span>;<span class="number">5</span>;<span class="pagerange">407-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23782161">23782161</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4029105">4029105</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1306742">10.1056/NEJMoa1306742</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome Sequencing Reveals Loci under Artificial Selection that Underlie Disease Phenotypes in the Laboratory Rat.</h4>
<p class="authors"><span class="author">Atanur SS</span>, <span class="author">Diaz AG</span>, <span class="author">Maratou K</span>, <span class="author">Sarkis A</span>, <span class="author">Rotival M</span>, <span class="author">Game L</span>, <span class="author">Tschannen MR</span>, <span class="author">Kaisaki PJ</span>, <span class="author">Otto GW</span>, <span class="author">John Ma MC</span>, <span class="author">Keane TM</span>, <span class="author">Hummel O</span>, <span class="author">Saar K</span>, <span class="author">Chen W</span>, <span class="author">Guryev V</span>, <span class="author">Gopalakrishnan K</span>, <span class="author">Garrett MR</span>, <span class="author">Joe B</span>, <span class="author">Citterio L</span>, <span class="author">Bianchi G</span>, <span class="author">McBride M</span>, <span class="author">Dominiczak A</span>, <span class="author">Adams DJ</span>, <span class="author">Serikawa T</span>, <span class="author">Flicek P</span>, <span class="author">Cuppen E</span>, <span class="author">Hubner N</span>, <span class="author">Petretto E</span>, <span class="author">Gauguier D</span>, <span class="author">Kwitek A</span>, <span class="author">Jacob H</span> and <span class="author">Aitman TJ</span></p>
<p class="affiliation">Physiological Genomic and Medicine Group, MRC Clinical Sciences Centre, Imperial College London, London W12 0NN, UK; National Heart and Lung Institute, Imperial College London, London W12 0NN, UK.</p>
<p class="abstract">Large numbers of inbred laboratory rat strains have been developed for a range of complex disease phenotypes. To gain insights into the evolutionary pressures underlying selection for these phenotypes, we sequenced the genomes of 27 rat strains, including 11 models of hypertension, diabetes, and insulin resistance, along with their respective control strains. Altogether, we identified more than 13 million single-nucleotide variants, indels, and structural variants across these rat strains. Analysis of strain-specific selective sweeps and gene clusters implicated genes and pathways involved in cation transport, angiotensin production, and regulators of oxidative stress in the development of cardiovascular disease phenotypes in rats. Many of the rat loci that we identified overlap with previously mapped loci for related traits in humans, indicating the presence of shared pathways underlying these phenotypes in rats and humans. These data represent a step change in resources available for evolutionary analysis of complex traits in disease models. PAPERCLIP:</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23890820">23890820</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.06.040">10.1016/j.cell.2013.06.040</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effective preparation of Plasmodium vivax field isolates for high-throughput whole genome sequencing.</h4>
<p class="authors"><span class="author">Auburn S</span>, <span class="author">Marfurt J</span>, <span class="author">Maslen G</span>, <span class="author">Campino S</span>, <span class="author">Ruano Rubio V</span>, <span class="author">Manske M</span>, <span class="author">Machunter B</span>, <span class="author">Kenangalem E</span>, <span class="author">Noviyanti R</span>, <span class="author">Trianty L</span>, <span class="author">Sebayang B</span>, <span class="author">Wirjanata G</span>, <span class="author">Sriprawat K</span>, <span class="author">Alcock D</span>, <span class="author">Macinnis B</span>, <span class="author">Miotto O</span>, <span class="author">Clark TG</span>, <span class="author">Russell B</span>, <span class="author">Anstey NM</span>, <span class="author">Nosten F</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Price RN</span></p>
<p class="affiliation">Global and Tropical Health Division, Menzies School of Health Research, Charles Darwin University, Darwin, Australia. <a href="mailto:%73%61%72%61%68.%61%75%62%75%72%6E%40%6D%65%6E%7A%69%65%73.%65%64%75.%61%75">&#115;&#97;&#114;&#97;&#104;&#46;&#97;&#117;&#98;&#117;&#114;&#110;&#64;&#109;&#101;&#110;&#122;&#105;&#101;&#115;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Whole genome sequencing (WGS) of Plasmodium vivax is problematic due to the reliance on clinical isolates which are generally low in parasitaemia and sample volume. Furthermore, clinical isolates contain a significant contaminating background of host DNA which confounds efforts to map short read sequence of the target P. vivax DNA. Here, we discuss a methodology to significantly improve the success of P. vivax WGS on natural (non-adapted) patient isolates. Using 37 patient isolates from Indonesia, Thailand, and travellers, we assessed the application of CF11-based white blood cell filtration alone and in combination with short term ex vivo schizont maturation. Although CF11 filtration reduced human DNA contamination in 8 Indonesian isolates tested, additional short-term culture increased the P. vivax DNA yield from a median of 0.15 to 6.2 ng &micro;l(-1) packed red blood cells (pRBCs) (p = 0.001) and reduced the human DNA percentage from a median of 33.9% to 6.22% (p = 0.008). Furthermore, post-CF11 and culture samples from Thailand gave a median P. vivax DNA yield of 2.34 ng &micro;l(-1) pRBCs, and 2.65% human DNA. In 22 P. vivax patient isolates prepared with the 2-step method, we demonstrate high depth (median 654X coverage) and breadth (&ge;89%) of coverage on the Illumina GAII and HiSeq platforms. In contrast to the A+T-rich P. falciparum genome, negligible bias was observed in coverage depth between coding and non-coding regions of the P. vivax genome. This uniform coverage will greatly facilitate the detection of SNPs and copy number variants across the genome, enabling unbiased exploration of the natural diversity in P. vivax populations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">089275</span>, <span class="grant">090532</span>, <span class="grant">091625</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">e53160</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23308154">23308154</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3537768">3537768</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0053160">10.1371/journal.pone.0053160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic triumph meets clinical reality.</h4>
<p class="authors"><span class="author">Ayub Q</span>, <span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p><p class="abstract">A report on the &#39;Genomic Disorders 2013: from 60 years of DNA to human genomes in the clinic&#39; meeting, held at Homerton College, Cambridge, UK, April 10-12, 2013.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">09851</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">307</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23714135">23714135</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4054008">4054008</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-5-307">10.1186/gb-2013-14-5-307</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FOXP2 targets show evidence of positive selection in European populations.</h4>
<p class="authors"><span class="author">Ayub Q</span>, <span class="author">Yngvadottir B</span>, <span class="author">Chen Y</span>, <span class="author">Xue Y</span>, <span class="author">Hu M</span>, <span class="author">Vernes SC</span>, <span class="author">Fisher SE</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%71%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#113;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Forkhead box P2 (FOXP2) is a highly conserved transcription factor that has been implicated in human speech and language disorders and plays important roles in the plasticity of the developing brain. The pattern of nucleotide polymorphisms in FOXP2 in modern populations suggests that it has been the target of positive (Darwinian) selection during recent human evolution. In our study, we searched for evidence of selection that might have followed FOXP2 adaptations in modern humans. We examined whether or not putative FOXP2 targets identified by chromatin-immunoprecipitation genomic screening show evidence of positive selection. We developed an algorithm that, for any given gene list, systematically generates matched lists of control genes from the Ensembl database, collates summary statistics for three frequency-spectrum-based neutrality tests from the low-coverage resequencing data of the 1000 Genomes Project, and determines whether these statistics are significantly different between the given gene targets and the set of controls. Overall, there was strong evidence of selection of FOXP2 targets in Europeans, but not in the Han Chinese, Japanese, or Yoruba populations. Significant outliers included several genes linked to cellular movement, reproduction, development, and immune cell trafficking, and 13 of these constituted a significant network associated with cardiac arteriopathy. Strong signals of selection were observed for CNTNAP2 and RBFOX1, key neurally expressed genes that have been consistently identified as direct FOXP2 targets in multiple studies and that have themselves been associated with neurodevelopmental disorders involving language dysfunction.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">92</span>;<span class="number">5</span>;<span class="pagerange">696-706</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23602712">23602712</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3644635">3644635</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.03.019">10.1016/j.ajhg.2013.03.019</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cooperativity of imprinted genes inactivated by acquired chromosome 20q deletions.</h4>
<p class="authors"><span class="author">Aziz A</span>, <span class="author">Baxter EJ</span>, <span class="author">Edwards C</span>, <span class="author">Cheong CY</span>, <span class="author">Ito M</span>, <span class="author">Bench A</span>, <span class="author">Kelley R</span>, <span class="author">Silber Y</span>, <span class="author">Beer PA</span>, <span class="author">Chng K</span>, <span class="author">Renfree MB</span>, <span class="author">McEwen K</span>, <span class="author">Gray D</span>, <span class="author">Nangalia J</span>, <span class="author">Mufti GJ</span>, <span class="author">Hellstrom-Lindberg E</span>, <span class="author">Kiladjian JJ</span>, <span class="author">McMullin MF</span>, <span class="author">Campbell PJ</span>, <span class="author">Ferguson-Smith AC</span> and <span class="author">Green AR</span></p>
<p class="abstract">Large regions of recurrent genomic loss are common in cancers; however, with a few well-characterized exceptions, how they contribute to tumor pathogenesis remains largely obscure. Here we identified primate-restricted imprinting of a gene cluster on chromosome 20 in the region commonly deleted in chronic myeloid malignancies. We showed that a single heterozygous 20q deletion consistently resulted in the complete loss of expression of the imprinted genes L3MBTL1 and SGK2, indicative of a pathogenetic role for loss of the active paternally inherited locus. Concomitant loss of both L3MBTL1 and SGK2 dysregulated erythropoiesis and megakaryopoiesis, 2 lineages commonly affected in chronic myeloid malignancies, with distinct consequences in each lineage. We demonstrated that L3MBTL1 and SGK2 collaborated in the transcriptional regulation of MYC by influencing different aspects of chromatin structure. L3MBTL1 is known to regulate nucleosomal compaction, and we here showed that SGK2 inactivated BRG1, a key ATP-dependent helicase within the SWI/SNF complex that regulates nucleosomal positioning. These results demonstrate a link between an imprinted gene cluster and malignancy, reveal a new pathogenetic mechanism associated with acquired regions of genomic loss, and underline the complex molecular and cellular consequences of "simple" cancer-associated chromosome deletions.</p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2013</span>;<span class="volume">123</span>;<span class="number">5</span>;<span class="pagerange">2169-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23543057">23543057</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3635733">3635733</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI66113">10.1172/JCI66113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y-chromosome and mtDNA genetics reveal significant contrasts in affinities of modern Middle Eastern populations with European and African populations.</h4>
<p class="authors"><span class="author">Badro DA</span>, <span class="author">Douaihy B</span>, <span class="author">Haber M</span>, <span class="author">Youhanna SC</span>, <span class="author">Salloum A</span>, <span class="author">Ghassibe-Sabbagh M</span>, <span class="author">Johnsrud B</span>, <span class="author">Khazen G</span>, <span class="author">Matisoo-Smith E</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Wells RS</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Platt DE</span>, <span class="author">Zalloua PA</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">The Lebanese American University, Chouran, Beirut, Lebanon.</p><p class="abstract">The Middle East was a funnel of human expansion out of Africa, a staging area for the Neolithic Agricultural Revolution, and the home to some of the earliest world empires. Post LGM expansions into the region and subsequent population movements created a striking genetic mosaic with distinct sex-based genetic differentiation. While prior studies have examined the mtDNA and Y-chromosome contrast in focal populations in the Middle East, none have undertaken a broad-spectrum survey including North and sub-Saharan Africa, Europe, and Middle Eastern populations. In this study 5,174 mtDNA and 4,658 Y-chromosome samples were investigated using PCA, MDS, mean-linkage clustering, AMOVA, and Fisher exact tests of F(ST)&#39;s, R(ST)&#39;s, and haplogroup frequencies. Geographic differentiation in affinities of Middle Eastern populations with Africa and Europe showed distinct contrasts between mtDNA and Y-chromosome data. Specifically, Lebanon&#39;s mtDNA shows a very strong association to Europe, while Yemen shows very strong affinity with Egypt and North and East Africa. Previous Y-chromosome results showed a Levantine coastal-inland contrast marked by J1 and J2, and a very strong North African component was evident throughout the Middle East. Neither of these patterns were observed in the mtDNA. While J2 has penetrated into Europe, the pattern of Y-chromosome diversity in Lebanon does not show the widespread affinities with Europe indicated by the mtDNA data. Lastly, while each population shows evidence of connections with expansions that now define the Middle East, Africa, and Europe, many of the populations in the Middle East show distinctive mtDNA and Y-haplogroup characteristics that indicate long standing settlement with relatively little impact from and movement into other populations.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">e54616</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382925">23382925</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3559847">3559847</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0054616">10.1371/journal.pone.0054616</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metagenomic study of the viruses of African straw-coloured fruit bats: detection of a chiropteran poxvirus and isolation of a novel adenovirus.</h4>
<p class="authors"><span class="author">Baker KS</span>, <span class="author">Leggett RM</span>, <span class="author">Bexfield NH</span>, <span class="author">Alston M</span>, <span class="author">Daly G</span>, <span class="author">Todd S</span>, <span class="author">Tachedjian M</span>, <span class="author">Holmes CE</span>, <span class="author">Crameri S</span>, <span class="author">Wang LF</span>, <span class="author">Heeney JL</span>, <span class="author">Suu-Ire R</span>, <span class="author">Kellam P</span>, <span class="author">Cunningham AA</span>, <span class="author">Wood JL</span>, <span class="author">Caccamo M</span> and <span class="author">Murcia PR</span></p>
<p class="affiliation">University of Cambridge, Department of Veterinary Medicine, Madingley Rd, Cambridge, Cambridgeshire, CB3 0ES, United Kingdom. <a href="mailto:%6B%66%32%38%31%40%63%61%6D.%61%63.%75%6B">&#107;&#102;&#50;&#56;&#49;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Viral emergence as a result of zoonotic transmission constitutes a continuous public health threat. Emerging viruses such as SARS coronavirus, hantaviruses and henipaviruses have wildlife reservoirs. Characterising the viruses of candidate reservoir species in geographical hot spots for viral emergence is a sensible approach to develop tools to predict, prevent, or contain emergence events. Here, we explore the viruses of Eidolon helvum, an Old World fruit bat species widely distributed in Africa that lives in close proximity to humans. We identified a great abundance and diversity of novel herpes and papillomaviruses, described the isolation of a novel adenovirus, and detected, for the first time, sequences of a chiropteran poxvirus closely related with Molluscum contagiosum. In sum, E. helvum display a wide variety of mammalian viruses, some of them genetically similar to known human pathogens, highlighting the possibility of zoonotic transmission.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801822</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Virology</span>&nbsp;<span class="year">2013</span>;<span class="volume">441</span>;<span class="number">2</span>;<span class="pagerange">95-106</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23562481">23562481</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3667569">3667569</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.virol.2013.03.014">10.1016/j.virol.2013.03.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fitness benefits in fluoroquinolone-resistant Salmonella Typhi in the absence of antimicrobial pressure.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Duy PT</span>, <span class="author">Nga TV</span>, <span class="author">Dung TT</span>, <span class="author">Phat VV</span>, <span class="author">Chau TT</span>, <span class="author">Turner AK</span>, <span class="author">Farrar J</span> and <span class="author">Boni MF</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Wellcome Trust Major Overseas Programme, Ho Chi Minh City, Vietnam.</p><p class="abstract">Fluoroquinolones (FQ) are the recommended antimicrobial treatment for typhoid, a severe systemic infection caused by the bacterium Salmonella enterica serovar Typhi. FQ-resistance mutations in S. Typhi have become common, hindering treatment and control efforts. Using in vitro competition experiments, we assayed the fitness of eleven isogenic S. Typhi strains with resistance mutations in the FQ target genes, gyrA and parC. In the absence of antimicrobial pressure, 6 out of 11 mutants carried a selective advantage over the antimicrobial-sensitive parent strain, indicating that FQ resistance in S. Typhi is not typically associated with fitness costs. Double-mutants exhibited higher than expected fitness as a result of synergistic epistasis, signifying that epistasis may be a critical factor in the evolution and molecular epidemiology of S. Typhi. Our findings have important implications for the management of drug-resistant S. Typhi, suggesting that FQ-resistant strains would be naturally maintained even if fluoroquinolone use were reduced. DOI: http://dx.doi.org/10.7554/eLife.01229.001.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">100087</span></p>
<p class="publication"><span class="title">eLife</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="pagerange">e01229</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24327559">24327559</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3857714">3857714</a>; DOI: <a href="http://dx.doi.org/10.7554%2FeLife.01229">10.7554/eLife.01229</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic screens in mice for genome integrity maintenance and cancer predisposition.</h4>
<p class="authors"><span class="author">Balmus G</span> and <span class="author">McIntyre RE</span></p>
<p class="affiliation">The Wellcome Trust/Cancer Research UK Gurdon Institute, University of Cambridge, Tennis Court Road, Cambridge CB2 1QN, UK; Experimental Cancer Genetics, The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Genome instability is a feature of nearly all cancers and can be exploited for therapy. In addition, a growing number of genome maintenance genes have been associated with developmental disorders. Efforts to understand the role of genome instability in these processes will be greatly facilitated by a more comprehensive understanding of their genetic network. We highlight recent genetic screens in model organisms that have assisted in the discovery of novel regulators of genome stability and focus on the contribution of mice as a model organism to understanding the role of genome instability during embryonic development, tumour formation and cancer therapy.</p>
<p class="publication"><span class="title">Current opinion in genetics &amp; development</span>&nbsp;<span class="year">2013</span>;<span class="volume">24C</span>;<span class="pagerange">1-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24657530">24657530</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gde.2013.10.010">10.1016/j.gde.2013.10.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Atypical mitogen-activated protein kinase phosphatase implicated in regulating transition from pre-S-Phase asexual intraerythrocytic development of Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Balu B</span>, <span class="author">Campbell C</span>, <span class="author">Sedillo J</span>, <span class="author">Maher S</span>, <span class="author">Singh N</span>, <span class="author">Thomas P</span>, <span class="author">Zhang M</span>, <span class="author">Pance A</span>, <span class="author">Otto TD</span>, <span class="author">Rayner JC</span> and <span class="author">Adams JH</span></p>
<p class="affiliation">Department of Global Health, College of Public Health, University of South Florida, Tampa, Florida, USA.</p><p class="abstract">Intraerythrocytic development of the human malaria parasite Plasmodium falciparum appears as a continuous flow through growth and proliferation. To develop a greater understanding of the critical regulatory events, we utilized piggyBac insertional mutagenesis to randomly disrupt genes. Screening a collection of piggyBac mutants for slow growth, we isolated the attenuated parasite C9, which carried a single insertion disrupting the open reading frame (ORF) of PF3D7_1305500. This gene encodes a protein structurally similar to a mitogen-activated protein kinase (MAPK) phosphatase, except for two notable characteristics that alter the signature motif of the dual-specificity phosphatase domain, suggesting that it may be a low-activity phosphatase or pseudophosphatase. C9 parasites demonstrated a significantly lower growth rate with delayed entry into the S/M phase of the cell cycle, which follows the stage of maximum PF3D7_1305500 expression in intact parasites. Genetic complementation with the full-length PF3D7_1305500 rescued the wild-type phenotype of C9, validating the importance of the putative protein phosphatase PF3D7_1305500 as a regulator of pre-S-phase cell cycle progression in P. falciparum.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">F31AI083053</span>, <span class="grant">R01 AI094973</span>, <span class="grant">R01AI033656</span>, <span class="grant">R01AI094973</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">9</span>;<span class="pagerange">1171-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23813392">23813392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3811562">3811562</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.00028-13">10.1128/EC.00028-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Imputation-based meta-analysis of severe malaria in three African populations.</h4>
<p class="authors"><span class="author">Band G</span>, <span class="author">Le QS</span>, <span class="author">Jostins L</span>, <span class="author">Pirinen M</span>, <span class="author">Kivinen K</span>, <span class="author">Jallow M</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Bojang K</span>, <span class="author">Pinder M</span>, <span class="author">Sirugo G</span>, <span class="author">Conway DJ</span>, <span class="author">Nyirongo V</span>, <span class="author">Kachala D</span>, <span class="author">Molyneux M</span>, <span class="author">Taylor T</span>, <span class="author">Ndila C</span>, <span class="author">Peshu N</span>, <span class="author">Marsh K</span>, <span class="author">Williams TN</span>, <span class="author">Alcock D</span>, <span class="author">Andrews R</span>, <span class="author">Edkins S</span>, <span class="author">Gray E</span>, <span class="author">Hubbart C</span>, <span class="author">Jeffreys A</span>, <span class="author">Rowlands K</span>, <span class="author">Schuldt K</span>, <span class="author">Clark TG</span>, <span class="author">Small KS</span>, <span class="author">Teo YY</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Rockett KA</span>, <span class="author">Barrett JC</span>, <span class="author">Spencer CC</span>, <span class="author">Malaria Genomic Epidemiology Network</span> and <span class="author">Malaria Genomic Epidemiological Network</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Oxford, United Kingdom.</p><p class="abstract">Combining data from genome-wide association studies (GWAS) conducted at different locations, using genotype imputation and fixed-effects meta-analysis, has been a powerful approach for dissecting complex disease genetics in populations of European ancestry. Here we investigate the feasibility of applying the same approach in Africa, where genetic diversity, both within and between populations, is far more extensive. We analyse genome-wide data from approximately 5,000 individuals with severe malaria and 7,000 population controls from three different locations in Africa. Our results show that the standard approach is well powered to detect known malaria susceptibility loci when sample sizes are large, and that modern methods for association analysis can control the potential confounding effects of population structure. We show that pattern of association around the haemoglobin S allele differs substantially across populations due to differences in haplotype structure. Motivated by these observations we consider new approaches to association analysis that might prove valuable for multicentre GWAS in Africa: we relax the assumptions of SNP-based fixed effect analysis; we apply Bayesian approaches to allow for heterogeneity in the effect of an allele on risk across studies; and we introduce a region-based test to allow for heterogeneity in the location of causal alleles.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">087285</span>, <span class="grant">090532</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">090770</span>, <span class="grant">090770/Z/09/Z</span>, <span class="grant">091758</span>, <span class="grant">091758/Z/10/Z</span>, <span class="grant">092654</span>, <span class="grant">096527</span>, <span class="grant">097364/Z/11/Z</span>, <span class="grant">WT077383/Z/05/Z</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">e1003509</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23717212">23717212</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3662650">3662650</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003509">10.1371/journal.pgen.1003509</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Approaches to querying bacterial genomes with transposon-insertion sequencing.</h4>
<p class="authors"><span class="author">Barquist L</span>, <span class="author">Boinett CJ</span> and <span class="author">Cain AK</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute; Hinxton, Cambridge, UK; EMBL-European Bioinformatics Institute; Hinxton, Cambridge, UK.</p>
<p class="abstract">In this review we discuss transposon-insertion sequencing, variously known in the literature as TraDIS, Tn-seq, INSeq and HITS. By monitoring a large library of single transposon-insertion mutants with high-throughput sequencing, these methods can rapidly identify genomic regions that contribute to organismal fitness under any condition assayable in the laboratory with exquisite resolution. We discuss the various protocols that have been developed and methods for analysis. We provide an overview of studies that have examined the reproducibility and accuracy of these methods, as well as studies showing the advantages offered by the high resolution and dynamic range of high-throughput sequencing over previous methods. We review a number of applications in the literature, from predicting genes essential for in vitro growth to directly assaying requirements for survival under infective conditions in vivo. We also highlight recent progress in assaying non-coding regions of the genome in addition to known coding sequences, including the combining of RNA-seq with high-throughput transposon mutagenesis.</p>
<p class="publication"><span class="title">RNA biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23635712">23635712</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comparison of dense transposon insertion libraries in the Salmonella serovars Typhi and Typhimurium.</h4>
<p class="authors"><span class="author">Barquist L</span>, <span class="author">Langridge GC</span>, <span class="author">Turner DJ</span>, <span class="author">Phan MD</span>, <span class="author">Turner AK</span>, <span class="author">Bateman A</span>, <span class="author">Parkhill J</span>, <span class="author">Wain J</span> and <span class="author">Gardner PP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6C%62%31%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#98;&#49;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Salmonella Typhi and Typhimurium diverged only &sim;50 000 years ago, yet have very different host ranges and pathogenicity. Despite the availability of multiple whole-genome sequences, the genetic differences that have driven these changes in phenotype are only beginning to be understood. In this study, we use transposon-directed insertion-site sequencing to probe differences in gene requirements for competitive growth in rich media between these two closely related serovars. We identify a conserved core of 281 genes that are required for growth in both serovars, 228 of which are essential in Escherichia coli. We are able to identify active prophage elements through the requirement for their repressors. We also find distinct differences in requirements for genes involved in cell surface structure biogenesis and iron utilization. Finally, we demonstrate that transposon-directed insertion-site sequencing is not only applicable to the protein-coding content of the cell but also has sufficient resolution to generate hypotheses regarding the functions of non-coding RNAs (ncRNAs) as well. We are able to assign probable functions to a number of cis-regulatory ncRNA elements, as well as to infer likely differences in trans-acting ncRNA regulatory networks.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600805</span>; Wellcome Trust: <span class="grant">WT076964</span>, <span class="grant">WT079643</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span>;<span class="volume">41</span>;<span class="number">8</span>;<span class="pagerange">4549-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23470992">23470992</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3632133">3632133</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkt148">10.1093/nar/gkt148</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neurolysin</h4>
<p class="authors"><span class="author">Barrett, A.J.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="pagerange">509-13</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00102-2%3B">10.1016/B978-0-12-382219-2.00102-2;</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192001022" href="http://www.sciencedirect.com/science/article/pii/B9780123822192001022">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>001022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cytosol Alanyl Aminopeptidase</h4>
<p class="authors"><span class="author">BARRETT,A.J.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="pagerange">431-4</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00086-7">10.1016/B978-0-12-382219-2.00086-7</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192000867" href="http://www.sciencedirect.com/science/article/pii/B9780123822192000867">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>000867</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metridin</h4>
<p class="authors"><span class="author">BARRETT,A.J.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">581</span>;<span class="pagerange">2624-5</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00581-0">10.1016/B978-0-12-382219-2.00581-0</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192005810" href="http://www.sciencedirect.com/science/article/pii/B9780123822192005810">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>005810</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Animal Legumain</h4>
<p class="authors"><span class="author">BARRETT,A.J.</span> and <span class="author">Chen,J.-M.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="number">518</span>;<span class="pagerange">2309-14</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00516-0">10.1016/B978-0-12-382219-2.00516-0</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192005160" href="http://www.sciencedirect.com/science/article/pii/B9780123822192005160">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>005160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: Unsequenced Serine Peptidases</h4>
<p class="authors"><span class="author">BARRETT,A.J</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">824</span>;<span class="pagerange">3737</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00824-3">10.1016/B978-0-12-382219-2.00824-3</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192008243" href="http://www.sciencedirect.com/science/article/pii/B9780123822192008243">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>008243</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Peptidyl-Dipeptidase B</h4>
<p class="authors"><span class="author">BARRETT,A.J</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="number">392</span>;<span class="pagerange">1710-11</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00392-6">10.1016/B978-0-12-382219-2.00392-6</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192003926" href="http://www.sciencedirect.com/science/article/pii/B9780123822192003926">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>003926</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identifying novel Plasmodium falciparum erythrocyte invasion receptors using systematic extracellular protein interaction screens.</h4>
<p class="authors"><span class="author">Bartholdson SJ</span>, <span class="author">Crosnier C</span>, <span class="author">Bustamante LY</span>, <span class="author">Rayner JC</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1HH; Malaria Programme, Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA.</p>
<p class="abstract">The invasion of host erythrocytes by the parasite Plasmodium falciparum initiates the blood stage of infection responsible for the symptoms of malaria. Invasion involves extracellular protein interactions between host erythrocyte receptors and ligands on the merozoite, the invasive form of the parasite. Despite significant research effort, many merozoite surface ligands have no known erythrocyte binding partner, most likely due to the intractable biochemical nature of membrane-tethered receptor proteins and their interactions. The few receptor-ligand pairs that have been described have largely relied on sourcing erythrocytes from patients with rare blood groups, a serendipitous approach that is unsatisfactory for systematically identifying novel receptors. We have recently developed a scalable assay called AVEXIS (for AVidity-based EXtracellular Interaction Screen), designed to circumvent the technical difficulties associated with the identification of extracellular protein interactions, and applied it to identify erythrocyte receptors for orphan Plasmodium falciparum merozoite ligands. Using this approach, we have recently identified Basigin (CD147) and Semaphorin-7A (CD108) as receptors for RH5 and MTRAP, respectively. In this essay, we review techniques used to identify Plasmodium receptors and discuss how they could be applied in the future to identify novel receptors both for Plasmodium parasites but also other pathogens.</p>
<p class="publication"><span class="title">Cellular microbiology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23617720">23617720</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fcmi.12151">10.1111/cmi.12151</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Network properties derived from deep sequencing of human B-cell receptor repertoires delineate B-cell populations.</h4>
<p class="authors"><span class="author">Bashford-Rogers RJ</span>, <span class="author">Palser AL</span>, <span class="author">Huntly BJ</span>, <span class="author">Rance R</span>, <span class="author">Vassiliou GS</span>, <span class="author">Follows GA</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom;</p><p class="abstract">The adaptive immune response selectively expands B- and T-cell clones following antigen recognition by B- and T-cell receptors (BCR and TCR), respectively. Next-generation sequencing is a powerful tool for dissecting the BCR and TCR populations at high resolution, but robust computational analyses are required to interpret such sequencing. Here, we develop a novel computational approach for BCR repertoire analysis using established next-generation sequencing methods coupled with network construction and population analysis. BCR sequences organize into networks based on sequence diversity, with differences in network connectivity clearly distinguishing between diverse repertoires of healthy individuals and clonally expanded repertoires from individuals with chronic lymphocytic leukemia (CLL) and other clonal blood disorders. Network population measures defined by the Gini Index and cluster sizes quantify the BCR clonality status and are robust to sampling and sequencing depths. BCR network analysis therefore allows the direct and quantifiable comparison of BCR repertoires between samples and intra-individual population changes between temporal or spatially separated samples and over the course of therapy.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079249</span>, <span class="grant">095663</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">11</span>;<span class="pagerange">1874-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23742949">23742949</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3814887">3814887</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.154815.113">10.1101/gr.154815.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ISCB Computational Biology Wikipedia Competition.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Kelso J</span>, <span class="author">Mietchen D</span>, <span class="author">Macintyre G</span>, <span class="author">Di Domenico T</span>, <span class="author">Abeel T</span>, <span class="author">Logan DW</span>, <span class="author">Radivojac P</span> and <span class="author">Rost B</span></p>
<p class="affiliation">European Molecular Biology Laboratory, European Bioinformatics Institute, Hinxton, United Kingdom.</p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">9</span>;<span class="pagerange">e1003242</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24068913">24068913</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3777890">3777890</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.1003242">10.1371/journal.pcbi.1003242</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Peripheral administration of prokineticin 2 potently reduces food intake and body weight in mice via the brainstem.</h4>
<p class="authors"><span class="author">Beale K</span>, <span class="author">Gardiner J</span>, <span class="author">Bewick G</span>, <span class="author">Hostomska K</span>, <span class="author">Patel N</span>, <span class="author">Hussain S</span>, <span class="author">Jayasena C</span>, <span class="author">Ebling F</span>, <span class="author">Jethwa P</span>, <span class="author">Prosser H</span>, <span class="author">Lattanzi R</span>, <span class="author">Negri L</span>, <span class="author">Ghatei M</span>, <span class="author">Bloom S</span> and <span class="author">Dhillo W</span></p>
<p class="affiliation">Section of Investigative Medicine, Imperial College London, London, UK.</p>
<p class="abstract">BACKGROUND AND PURPOSE: Prokineticin 2 (PK2) has recently been shown to acutely reduce food intake in rodents. We aimed to determine the CNS sites and receptors that mediate the anorectic effects of peripherally administered PK2 and its chronic effects on glucose and energy homeostasis. EXPERIMENTAL APPROACH: We investigated neuronal activation following i.p. administration of PK2 using c-Fos-like immunoreactivity (CFL-IR). The anorectic effect of PK2 was examined in mice with targeted deletion of either prokineticin receptor 1 (PKR1) or prokineticin receptor 2 (PKR2), and in wild-type mice following administration of the PKR1 antagonist, PC1. The effect of IP PK2 administration on glucose homeostasis was investigated. Finally, the effect of long-term administration of PK2 on glucose and energy homeostasis in diet-induced obese (DIO) mice was determined. KEY RESULTS: I.p. PK2 administration significantly increased CFL-IR in the dorsal motor vagal nucleus of the brainstem. The anorectic effect of PK2 was maintained in mice lacking the PKR2 but abolished in mice lacking PKR1 and in wild-type mice pre-treated with PC1. DIO mice treated chronically with PK2 had no changes in glucose levels but significantly reduced food intake and body weight compared to controls. CONCLUSIONS AND IMPLICATIONS: Together, our data suggest that the anorectic effects of peripherally administered PK2 are mediated via the brainstem and this effect requires PKR1 but not PKR2 signalling. Chronic administration of PK2 reduces food intake and body weight in a mouse model of human obesity, suggesting that PKR1-selective agonists have potential to be novel therapeutics for the treatment of obesity.</p>
<p class="publication"><span class="title">British journal of pharmacology</span>&nbsp;<span class="year">2013</span>;<span class="volume">168</span>;<span class="number">2</span>;<span class="pagerange">403-410</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22935107" rel="external">22935107</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1476-5381.2012.02191.x" rel="external">10.1111/j.1476-5381.2012.02191.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deep Resequencing of GWAS Loci Identifies Rare Variants in CARD9, IL23R and RNF186 That Are Associated with Ulcerative Colitis.</h4>
<p class="authors"><span class="author">Beaudoin M</span>, <span class="author">Goyette P</span>, <span class="author">Boucher G</span>, <span class="author">Lo KS</span>, <span class="author">Rivas MA</span>, <span class="author">Stevens C</span>, <span class="author">Alikashani A</span>, <span class="author">Ladouceur M</span>, <span class="author">Ellinghaus D</span>, <span class="author">T&ouml;rkvist L</span>, <span class="author">Goel G</span>, <span class="author">Lagac&eacute; C</span>, <span class="author">Annese V</span>, <span class="author">Bitton A</span>, <span class="author">Begun J</span>, <span class="author">Brant SR</span>, <span class="author">Bresso F</span>, <span class="author">Cho JH</span>, <span class="author">Duerr RH</span>, <span class="author">Halfvarson J</span>, <span class="author">McGovern DP</span>, <span class="author">Radford-Smith G</span>, <span class="author">Schreiber S</span>, <span class="author">Schumm PL</span>, <span class="author">Sharma Y</span>, <span class="author">Silverberg MS</span>, <span class="author">Weersma RK</span>, <span class="author">Quebec IBD Genetics Consortium</span>, <span class="author">NIDDK IBD Genetics Consortium</span>, <span class="author">International IBD Genetics Consortium</span>, <span class="author">D&#39;Amato M</span>, <span class="author">Vermeire S</span>, <span class="author">Franke A</span>, <span class="author">Lettre G</span>, <span class="author">Xavier RJ</span>, <span class="author">Daly MJ</span> and <span class="author">Rioux JD</span></p>
<p class="affiliation">Montreal Heart Institute, Research Center, Montreal, Quebec, Canada.</p>
<p class="abstract">Genome-wide association studies and follow-up meta-analyses in Crohn's disease (CD) and ulcerative colitis (UC) have recently identified 163 disease-associated loci that meet genome-wide significance for these two inflammatory bowel diseases (IBD). These discoveries have already had a tremendous impact on our understanding of the genetic architecture of these diseases and have directed functional studies that have revealed some of the biological functions that are important to IBD (e.g. autophagy). Nonetheless, these loci can only explain a small proportion of disease variance (∼14% in CD and 7.5% in UC), suggesting that not only are additional loci to be found but that the known loci may contain high effect rare risk variants that have gone undetected by GWAS. To test this, we have used a targeted sequencing approach in 200 UC cases and 150 healthy controls (HC), all of French Canadian descent, to study 55 genes in regions associated with UC. We performed follow-up genotyping of 42 rare non-synonymous variants in independent case-control cohorts (totaling 14,435 UC cases and 20,204 HC). Our results confirmed significant association to rare non-synonymous coding variants in both IL23R and CARD9, previously identified from sequencing of CD loci, as well as identified a novel association in RNF186. With the exception of CARD9 (OR = 0.39), the rare non-synonymous variants identified were of moderate effect (OR = 1.49 for RNF186 and OR = 0.79 for IL23R). RNF186 encodes a protein with a RING domain having predicted E3 ubiquitin-protein ligase activity and two transmembrane domains. Importantly, the disease-coding variant is located in the ubiquitin ligase domain. Finally, our results suggest that rare variants in genes identified by genome-wide association in UC are unlikely to contribute significantly to the overall variance for the disease. Rather, these are expected to help focus functional studies of the corresponding disease loci.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">9</span>;<span class="pagerange">e1003723</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24068945">24068945</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3772057">3772057</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003723">10.1371/journal.pgen.1003723</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinct H3F3A and H3F3B driver mutations define chondroblastoma and giant cell tumor of bone.</h4>
<p class="authors"><span class="author">Behjati S</span>, <span class="author">Tarpey PS</span>, <span class="author">Presneau N</span>, <span class="author">Scheipl S</span>, <span class="author">Pillay N</span>, <span class="author">Van Loo P</span>, <span class="author">Wedge DC</span>, <span class="author">Cooke SL</span>, <span class="author">Gundem G</span>, <span class="author">Davies H</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Martin S</span>, <span class="author">McLaren S</span>, <span class="author">Goody V</span>, <span class="author">Goodie V</span>, <span class="author">Robinson B</span>, <span class="author">Butler A</span>, <span class="author">Teague JW</span>, <span class="author">Halai D</span>, <span class="author">Khatri B</span>, <span class="author">Myklebost O</span>, <span class="author">Baumhoer D</span>, <span class="author">Jundt G</span>, <span class="author">Hamoudi R</span>, <span class="author">Tirabosco R</span>, <span class="author">Amary MF</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span> and <span class="author">Flanagan AM</span></p>
<p class="affiliation">1] Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. [2] Department of Paediatrics, University of Cambridge, Cambridge, UK. [3].</p><p class="abstract">It is recognized that some mutated cancer genes contribute to the development of many cancer types, whereas others are cancer type specific. For genes that are mutated in multiple cancer classes, mutations are usually similar in the different affected cancer types. Here, however, we report exquisite tumor type specificity for different histone H3.3 driver alterations. In 73 of 77 cases of chondroblastoma (95%), we found p.Lys36Met alterations predominantly encoded in H3F3B, which is one of two genes for histone H3.3. In contrast, in 92% (49/53) of giant cell tumors of bone, we found histone H3.3 alterations exclusively in H3F3A, leading to p.Gly34Trp or, in one case, p.Gly34Leu alterations. The mutations were restricted to the stromal cell population and were not detected in osteoclasts or their precursors. In the context of previously reported H3F3A mutations encoding p.Lys27Met and p.Gly34Arg or p.Gly34Val alterations in childhood brain tumors, a remarkable picture of tumor type specificity for histone H3.3 driver alterations emerges, indicating that histone H3.3 residues, mutations and genes have distinct functions.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">098051</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">12</span>;<span class="pagerange">1479-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24162739">24162739</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3839851">3839851</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2814">10.1038/ng.2814</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microbial genomes as cheat sheets.</h4>
<p class="authors"><span class="author">Bennett HM</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">5</span>;<span class="pagerange">302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23563106">23563106</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3014">10.1038/nrmicro3014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide meta-analysis identifies 11 new loci for anthropometric traits and provides insights into genetic architecture.</h4>
<p class="authors"><span class="author">Berndt SI</span>, <span class="author">Gustafsson S</span>, <span class="author">M&auml;gi R</span>, <span class="author">Ganna A</span>, <span class="author">Wheeler E</span>, <span class="author">Feitosa MF</span>, <span class="author">Justice AE</span>, <span class="author">Monda KL</span>, <span class="author">Croteau-Chonka DC</span>, <span class="author">Day FR</span>, <span class="author">Esko T</span>, <span class="author">Fall T</span>, <span class="author">Ferreira T</span>, <span class="author">Gentilini D</span>, <span class="author">Jackson AU</span>, <span class="author">Luan J</span>, <span class="author">Randall JC</span>, <span class="author">Vedantam S</span>, <span class="author">Willer CJ</span>, <span class="author">Winkler TW</span>, <span class="author">Wood AR</span>, <span class="author">Workalemahu T</span>, <span class="author">Hu YJ</span>, <span class="author">Lee SH</span>, <span class="author">Liang L</span>, <span class="author">Lin DY</span>, <span class="author">Min JL</span>, <span class="author">Neale BM</span>, <span class="author">Thorleifsson G</span>, <span class="author">Yang J</span>, <span class="author">Albrecht E</span>, <span class="author">Amin N</span>, <span class="author">Bragg-Gresham JL</span>, <span class="author">Cadby G</span>, <span class="author">den Heijer M</span>, <span class="author">Eklund N</span>, <span class="author">Fischer K</span>, <span class="author">Goel A</span>, <span class="author">Hottenga JJ</span>, <span class="author">Huffman JE</span>, <span class="author">Jarick I</span>, <span class="author">Johansson &Aring;</span>, <span class="author">Johnson T</span>, <span class="author">Kanoni S</span>, <span class="author">Kleber ME</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Kristiansson K</span>, <span class="author">Kutalik Z</span>, <span class="author">Lamina C</span>, <span class="author">Lecoeur C</span>, <span class="author">Li G</span>, <span class="author">Mangino M</span>, <span class="author">McArdle WL</span>, <span class="author">Medina-Gomez C</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Ngwa JS</span>, <span class="author">Nolte IM</span>, <span class="author">Paternoster L</span>, <span class="author">Pechlivanis S</span>, <span class="author">Perola M</span>, <span class="author">Peters MJ</span>, <span class="author">Preuss M</span>, <span class="author">Rose LM</span>, <span class="author">Shi J</span>, <span class="author">Shungin D</span>, <span class="author">Smith AV</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Surakka I</span>, <span class="author">Teumer A</span>, <span class="author">Trip MD</span>, <span class="author">Tyrer J</span>, <span class="author">Van Vliet-Ostaptchouk JV</span>, <span class="author">Vandenput L</span>, <span class="author">Waite LL</span>, <span class="author">Zhao JH</span>, <span class="author">Absher D</span>, <span class="author">Asselbergs FW</span>, <span class="author">Atalay M</span>, <span class="author">Attwood AP</span>, <span class="author">Balmforth AJ</span>, <span class="author">Basart H</span>, <span class="author">Beilby J</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Brambilla P</span>, <span class="author">Bruinenberg M</span>, <span class="author">Campbell H</span>, <span class="author">Chasman DI</span>, <span class="author">Chines PS</span>, <span class="author">Collins FS</span>, <span class="author">Connell JM</span>, <span class="author">Cookson WO</span>, <span class="author">de Faire U</span>, <span class="author">de Vegt F</span>, <span class="author">Dei M</span>, <span class="author">Dimitriou M</span>, <span class="author">Edkins S</span>, <span class="author">Estrada K</span>, <span class="author">Evans DM</span>, <span class="author">Farrall M</span>, <span class="author">Ferrario MM</span>, <span class="author">Ferri&egrave;res J</span>, <span class="author">Franke L</span>, <span class="author">Frau F</span>, <span class="author">Gejman PV</span>, <span class="author">Grallert H</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Gudnason V</span>, <span class="author">Hall AS</span>, <span class="author">Hall P</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hayward C</span>, <span class="author">Heard-Costa NL</span>, <span class="author">Heath AC</span>, <span class="author">Hebebrand J</span>, <span class="author">Homuth G</span>, <span class="author">Hu FB</span>, <span class="author">Hunt SE</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Iribarren C</span>, <span class="author">Jacobs KB</span>, <span class="author">Jansson JO</span>, <span class="author">Jula A</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kathiresan S</span>, <span class="author">Kee F</span>, <span class="author">Khaw KT</span>, <span class="author">Kivim&auml;ki M</span>, <span class="author">Koenig W</span>, <span class="author">Kraja AT</span>, <span class="author">Kumari M</span>, <span class="author">Kuulasmaa K</span>, <span class="author">Kuusisto J</span>, <span class="author">Laitinen JH</span>, <span class="author">Lakka TA</span>, <span class="author">Langenberg C</span>, <span class="author">Launer LJ</span>, <span class="author">Lind L</span>, <span class="author">Lindstr&ouml;m J</span>, <span class="author">Liu J</span>, <span class="author">Liuzzi A</span>, <span class="author">Lokki ML</span>, <span class="author">Lorentzon M</span>, <span class="author">Madden PA</span>, <span class="author">Magnusson PK</span>, <span class="author">Manunta P</span>, <span class="author">Marek D</span>, <span class="author">M&auml;rz W</span>, <span class="author">Mateo Leach I</span>, <span class="author">McKnight B</span>, <span class="author">Medland SE</span>, <span class="author">Mihailov E</span>, <span class="author">Milani L</span>, <span class="author">Montgomery GW</span>, <span class="author">Mooser V</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">Munroe PB</span>, <span class="author">Musk AW</span>, <span class="author">Narisu N</span>, <span class="author">Navis G</span>, <span class="author">Nicholson G</span>, <span class="author">Nohr EA</span>, <span class="author">Ong KK</span>, <span class="author">Oostra BA</span>, <span class="author">Palmer CN</span>, <span class="author">Palotie A</span>, <span class="author">Peden JF</span>, <span class="author">Pedersen N</span>, <span class="author">Peters A</span>, <span class="author">Polasek O</span>, <span class="author">Pouta A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Prokopenko I</span>, <span class="author">P&uuml;tter C</span>, <span class="author">Radhakrishnan A</span>, <span class="author">Raitakari O</span>, <span class="author">Rendon A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rudan I</span>, <span class="author">Saaristo TE</span>, <span class="author">Sambrook JG</span>, <span class="author">Sanders AR</span>, <span class="author">Sanna S</span>, <span class="author">Saramies J</span>, <span class="author">Schipf S</span>, <span class="author">Schreiber S</span>, <span class="author">Schunkert H</span>, <span class="author">Shin SY</span>, <span class="author">Signorini S</span>, <span class="author">Sinisalo J</span>, <span class="author">Skrobek B</span>, <span class="author">Soranzo N</span>, <span class="author">Stan&#x10D;&aacute;kov&aacute; A</span>, <span class="author">Stark K</span>, <span class="author">Stephens JC</span>, <span class="author">Stirrups K</span>, <span class="author">Stolk RP</span>, <span class="author">Stumvoll M</span>, <span class="author">Swift AJ</span>, <span class="author">Theodoraki EV</span>, <span class="author">Thorand B</span>, <span class="author">Tregouet DA</span>, <span class="author">Tremoli E</span>, <span class="author">Van der Klauw MM</span>, <span class="author">van Meurs JB</span>, <span class="author">Vermeulen SH</span>, <span class="author">Viikari J</span>, <span class="author">Virtamo J</span>, <span class="author">Vitart V</span>, <span class="author">Waeber G</span>, <span class="author">Wang Z</span>, <span class="author">Wid&eacute;n E</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Winkelmann BR</span>, <span class="author">Witteman JC</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Wong A</span>, <span class="author">Wright AF</span>, <span class="author">Zillikens MC</span>, <span class="author">Amouyel P</span>, <span class="author">Boehm BO</span>, <span class="author">Boerwinkle E</span>, <span class="author">Boomsma DI</span>, <span class="author">Caulfield MJ</span>, <span class="author">Chanock SJ</span>, <span class="author">Cupples LA</span>, <span class="author">Cusi D</span>, <span class="author">Dedoussis GV</span>, <span class="author">Erdmann J</span>, <span class="author">Eriksson JG</span>, <span class="author">Franks PW</span>, <span class="author">Froguel P</span>, <span class="author">Gieger C</span>, <span class="author">Gyllensten U</span>, <span class="author">Hamsten A</span>, <span class="author">Harris TB</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hicks AA</span>, <span class="author">Hingorani A</span>, <span class="author">Hinney A</span>, <span class="author">Hofman A</span>, <span class="author">Hovingh KG</span>, <span class="author">Hveem K</span>, <span class="author">Illig T</span>, <span class="author">Jarvelin MR</span>, <span class="author">J&ouml;ckel KH</span>, <span class="author">Keinanen-Kiukaanniemi SM</span>, <span class="author">Kiemeney LA</span>, <span class="author">Kuh D</span>, <span class="author">Laakso M</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Levinson DF</span>, <span class="author">Martin NG</span>, <span class="author">Metspalu A</span>, <span class="author">Morris AD</span>, <span class="author">Nieminen MS</span>, <span class="author">Nj&oslash;lstad I</span>, <span class="author">Ohlsson C</span>, <span class="author">Oldehinkel AJ</span>, <span class="author">Ouwehand WH</span>, <span class="author">Palmer LJ</span>, <span class="author">Penninx B</span>, <span class="author">Power C</span>, <span class="author">Province MA</span>, <span class="author">Psaty BM</span>, <span class="author">Qi L</span>, <span class="author">Rauramaa R</span>, <span class="author">Ridker PM</span>, <span class="author">Ripatti S</span>, <span class="author">Salomaa V</span>, <span class="author">Samani NJ</span>, <span class="author">Snieder H</span>, <span class="author">S&oslash;rensen TI</span>, <span class="author">Spector TD</span>, <span class="author">Stefansson K</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Uusitupa M</span>, <span class="author">van der Harst P</span>, <span class="author">Vollenweider P</span>, <span class="author">Wallaschofski H</span>, <span class="author">Wareham NJ</span>, <span class="author">Watkins H</span>, <span class="author">Wichmann HE</span>, <span class="author">Wilson JF</span>, <span class="author">Abecasis GR</span>, <span class="author">Assimes TL</span>, <span class="author">Barroso I</span>, <span class="author">Boehnke M</span>, <span class="author">Borecki IB</span>, <span class="author">Deloukas P</span>, <span class="author">Fox CS</span>, <span class="author">Frayling T</span>, <span class="author">Groop LC</span>, <span class="author">Haritunian T</span>, <span class="author">Heid IM</span>, <span class="author">Hunter D</span>, <span class="author">Kaplan RC</span>, <span class="author">Karpe F</span>, <span class="author">Moffatt MF</span>, <span class="author">Mohlke KL</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Pawitan Y</span>, <span class="author">Schadt EE</span>, <span class="author">Schlessinger D</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Strachan DP</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">van Duijn CM</span>, <span class="author">Visscher PM</span>, <span class="author">Di Blasio AM</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Lindgren CM</span>, <span class="author">Morris AP</span>, <span class="author">Meyre D</span>, <span class="author">Scherag A</span>, <span class="author">McCarthy MI</span>, <span class="author">Speliotes EK</span>, <span class="author">North KE</span>, <span class="author">Loos RJ</span> and <span class="author">Ingelsson E</span></p>
<p class="affiliation">US Department of Health and Human Services, Division of Cancer Epidemiology and Genetics, National Cancer Institute, US National Institutes of Health, Bethesda, Maryland, USA.</p><p class="abstract">Approaches exploiting trait distribution extremes may be used to identify loci associated with common traits, but it is unknown whether these loci are generalizable to the broader population. In a genome-wide search for loci associated with the upper versus the lower 5th percentiles of body mass index, height and waist-to-hip ratio, as well as clinical classes of obesity, including up to 263,407 individuals of European ancestry, we identified 4 new loci (IGFBP4, H6PD, RSRC1 and PPP2R2A) influencing height detected in the distribution tails and 7 new loci (HNF4G, RPTOR, GNAT2, MRPS33P4, ADCY9, HS6ST3 and ZZZ3) for clinical classes of obesity. Further, we find a large overlap in genetic structure and the distribution of variants between traits based on extremes and the general population and little etiological heterogeneity between obesity subgroups.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/11/63/29011</span>; Cancer Research UK: <span class="grant">14136</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0600237</span>, <span class="grant">G0601261</span>, <span class="grant">G1000143</span>, <span class="grant">G9521010</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U105260558</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>; NHLBI NIH HHS: <span class="grant">R01 HL105756</span>; NIAAA NIH HHS: <span class="grant">K05 AA017688</span>; NIDDK NIH HHS: <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK075787</span>; NIGMS NIH HHS: <span class="grant">T32 GM074905</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">097117</span>, <span class="grant">098017</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">5</span>;<span class="pagerange">501-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23563607">23563607</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3973018">3973018</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2606">10.1038/ng.2606</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The evolutionary dynamics of influenza A virus adaptation to mammalian hosts.</h4>
<p class="authors"><span class="author">Bhatt S</span>, <span class="author">Lam TT</span>, <span class="author">Lycett SJ</span>, <span class="author">Leigh Brown AJ</span>, <span class="author">Bowden TA</span>, <span class="author">Holmes EC</span>, <span class="author">Guan Y</span>, <span class="author">Wood JL</span>, <span class="author">Brown IH</span>, <span class="author">Kellam P</span>, <span class="author">Combating Swine Influenza Consortium</span> and <span class="author">Pybus OG</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Oxford, UK.</p><p class="abstract">Few questions on infectious disease are more important than understanding how and why avian influenza A viruses successfully emerge in mammalian populations, yet little is known about the rate and nature of the virus&#39; genetic adaptation in new hosts. Here, we measure, for the first time, the genomic rate of adaptive evolution of swine influenza viruses (SwIV) that originated in birds. By using a curated dataset of more than 24 000 human and swine influenza gene sequences, including 41 newly characterized genomes, we reconstructed the adaptive dynamics of three major SwIV lineages (Eurasian, EA; classical swine, CS; triple reassortant, TR). We found that, following the transfer of the EA lineage from birds to swine in the late 1970s, EA virus genes have undergone substantially faster adaptive evolution than those of the CS lineage, which had circulated among swine for decades. Further, the adaptation rates of the EA lineage antigenic haemagglutinin and neuraminidase genes were unexpectedly high and similar to those observed in human influenza A. We show that the successful establishment of avian influenza viruses in swine is associated with raised adaptive evolution across the entire genome for many years after zoonosis, reflecting the contribution of multiple mutations to the coordinated optimization of viral fitness in a new environment. This dynamics is replicated independently in the polymerase genes of the TR lineage, which established in swine following separate transmission from non-swine hosts.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/H014306/1</span>; Medical Research Council: <span class="grant">MC_G0902096</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2013</span>;<span class="volume">368</span>;<span class="number">1614</span>;<span class="pagerange">20120382</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382435">23382435</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3678336">3678336</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2012.0382">10.1098/rstb.2012.0382</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of intraocular pressure identifies the GLCCI1/ICA1 region as a glaucoma susceptibility locus.</h4>
<p class="authors"><span class="author">Blue Mountains Eye Study (BMES) and The Wellcome Trust Case Control Consortium 2 (WTCCC2)</span></p><p class="abstract">To discover quantitative trait loci for intraocular pressure, a major risk factor for glaucoma and the only modifiable one, we performed a genome-wide association study on a discovery cohort of 2175 individuals from Sydney, Australia. We found a novel association between intraocular pressure and a common variant at 7p21 near to GLCCI1 and ICA1. The findings in this region were confirmed through two UK replication cohorts totalling 4866 individuals (rs59072263, Pcombined = 1.10 &times; 10(-8)). A copy of the G allele at this SNP is associated with an increase in mean IOP of 0.45 mmHg (95%CI = 0.30-0.61 mmHg). These results lend support to the implication of vesicle trafficking and glucocorticoid inducibility pathways in the determination of intraocular pressure and in the pathogenesis of primary open-angle glaucoma.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">22</span>;<span class="pagerange">4653-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23836780">23836780</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddt293">10.1093/hmg/ddt293</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Uniparental markers in Italy reveal a sex-biased genetic structure and different historical strata.</h4>
<p class="authors"><span class="author">Boattini A</span>, <span class="author">Martinez-Cruz B</span>, <span class="author">Sarno S</span>, <span class="author">Harmant C</span>, <span class="author">Useli A</span>, <span class="author">Sanz P</span>, <span class="author">Yang-Yao D</span>, <span class="author">Manry J</span>, <span class="author">Ciani G</span>, <span class="author">Luiselli D</span>, <span class="author">Quintana-Murci L</span>, <span class="author">Comas D</span>, <span class="author">Pettener D</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Laboratorio di Antropologia Molecolare, Dipartimento di Scienze Biologiche, Geologiche e Ambientali, Universit&agrave; di Bologna, Bologna, Italy.</p><p class="abstract">Located in the center of the Mediterranean landscape and with an extensive coastal line, the territory of what is today Italy has played an important role in the history of human settlements and movements of Southern Europe and the Mediterranean Basin. Populated since Paleolithic times, the complexity of human movements during the Neolithic, the Metal Ages and the most recent history of the two last millennia (involving the overlapping of different cultural and demic strata) has shaped the pattern of the modern Italian genetic structure. With the aim of disentangling this pattern and understanding which processes more importantly shaped the distribution of diversity, we have analyzed the uniparentally-inherited markers in &sim;900 individuals from an extensive sampling across the Italian peninsula, Sardinia and Sicily. Spatial PCAs and DAPCs revealed a sex-biased pattern indicating different demographic histories for males and females. Besides the genetic outlier position of Sardinians, a North West-South East Y-chromosome structure is found in continental Italy. Such structure is in agreement with recent archeological syntheses indicating two independent and parallel processes of Neolithisation. In addition, date estimates pinpoint the importance of the cultural and demographic events during the late Neolithic and Metal Ages. On the other hand, mitochondrial diversity is distributed more homogeneously in agreement with older population events that might be related to the presence of an Italian Refugium during the last glacial period in Europe.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e65441</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23734255">23734255</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3666984">3666984</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0065441">10.1371/journal.pone.0065441</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Compression of FASTQ and SAM Format Sequencing Data.</h4>
<p class="authors"><span class="author">Bonfield JK</span> and <span class="author">Mahoney MV</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, United Kingdom.</p>
<p class="abstract">Storage and transmission of the data produced by modern DNA sequencing instruments has become a major concern, which prompted the Pistoia Alliance to pose the SequenceSqueeze contest for compression of FASTQ files. We present several compression entries from the competition, Fastqz and Samcomp/Fqzcomp, including the winning entry. These are compared against existing algorithms for both reference based compression (CRAM, Goby) and non-reference based compression (DSRC, BAM) and other recently published competition entries (Quip, SCALCE). The tools are shown to be the new Pareto frontier for FASTQ compression, offering state of the art ratios at affordable CPU costs. All programs are freely available on SourceForge. Fastqz: https://sourceforge.net/projects/fastqz/, fqzcomp: https://sourceforge.net/projects/fqzcomp/, and samcomp: https://sourceforge.net/projects/samcomp/.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e59190</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23533605">23533605</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0059190">10.1371/journal.pone.0059190</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Single Multilocus Sequence Typing (MLST) Scheme for Seven Pathogenic Leptospira Species.</h4>
<p class="authors"><span class="author">Boonsilp S</span>, <span class="author">Thaipadungpanit J</span>, <span class="author">Amornchai P</span>, <span class="author">Wuthiekanun V</span>, <span class="author">Bailey MS</span>, <span class="author">Holden MT</span>, <span class="author">Zhang C</span>, <span class="author">Jiang X</span>, <span class="author">Koizumi N</span>, <span class="author">Taylor K</span>, <span class="author">Galloway R</span>, <span class="author">Hoffmaster AR</span>, <span class="author">Craig S</span>, <span class="author">Smythe LD</span>, <span class="author">Hartskeerl RA</span>, <span class="author">Day NP</span>, <span class="author">Chantratita N</span>, <span class="author">Feil EJ</span>, <span class="author">Aanensen DM</span>, <span class="author">Spratt BG</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Mahidol-Oxford Tropical Medicine Research Unit, Faculty of Tropical Medicine, Mahidol University, Bangkok, Thailand ; Department of Microbiology and Immunology, Faculty of Tropical Medicine, Mahidol University, Bangkok, Thailand.</p>
<p class="abstract">Background: The available Leptospira multilocus sequence typing (MLST) scheme supported by a MLST website is limited to L. interrogans and L. kirschneri. Our aim was to broaden the utility of this scheme to incorporate a total of seven pathogenic species.

We modified the existing scheme by replacing one of the seven MLST loci (fadD was changed to caiB), as the former gene did not appear to be present in some pathogenic species. Comparison of the original and modified schemes using data for L. interrogans and L. kirschneri demonstrated that the discriminatory power of the two schemes was not significantly different. The modified scheme was used to further characterize 325 isolates (L. alexanderi [nâ€Š=â€Š5], L. borgpetersenii [nâ€Š=â€Š34], L. interrogans [nâ€Š=â€Š222], L. kirschneri [nâ€Š=â€Š29], L. noguchii [nâ€Š=â€Š9], L. santarosai [nâ€Š=â€Š10], and L. weilii [nâ€Š=â€Š16]). Phylogenetic analysis using concatenated sequences of the 7 loci demonstrated that each species corresponded to a discrete clade, and that no strains were misclassified at the species level. Comparison between genotype and serovar was possible for 254 isolates. Of the 31 sequence types (STs) represented by at least two isolates, 18 STs included isolates assigned to two or three different serovars. Conversely, 14 serovars were identified that contained between 2 to 10 different STs. New observations were made on the global phylogeography of Leptospira spp., and the utility of MLST in making associations between human disease and specific maintenance hosts was demonstrated.

Conclusion: The new MLST scheme, supported by an updated MLST website, allows the characterization and species assignment of isolates of the seven major pathogenic species associated with leptospirosis.</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">e1954</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23359622">23359622</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3554523">3554523</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001954">10.1371/journal.pntd.0001954</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Oxidative bisulfite sequencing of 5-methylcytosine and 5-hydroxymethylcytosine.</h4>
<p class="authors"><span class="author">Booth MJ</span>, <span class="author">Ost TW</span>, <span class="author">Beraldi D</span>, <span class="author">Bell NM</span>, <span class="author">Branco MR</span>, <span class="author">Reik W</span> and <span class="author">Balasubramanian S</span></p>
<p class="affiliation">Department of Chemistry, University of Cambridge, Cambridge, UK.</p>
<p class="abstract">To uncover the function of and interplay between the mammalian cytosine modifications 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), new techniques and advances in current technology are needed. To this end, we have developed oxidative bisulfite sequencing (oxBS-seq), which can quantitatively locate 5mC and 5hmC marks at single-base resolution in genomic DNA. In bisulfite sequencing (BS-seq), both 5mC and 5hmC are read as cytosines and thus cannot be discriminated; however, in oxBS-seq, specific oxidation of 5hmC to 5-formylcytosine (5fC) and conversion of the newly formed 5fC to uracil (under bisulfite conditions) means that 5hmC can be discriminated from 5mC. A positive readout of actual 5mC is gained from a single oxBS-seq run, and 5hmC levels are inferred by comparison with a BS-seq run. Here we describe an optimized second-generation protocol that can be completed in 2 d.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">1841-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24008380">24008380</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2013.115">10.1038/nprot.2013.115</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide screen identifies new candidate genes associated with artemisinin susceptibility in Plasmodium falciparum in Kenya.</h4>
<p class="authors"><span class="author">Borrmann S</span>, <span class="author">Straimer J</span>, <span class="author">Mwai L</span>, <span class="author">Abdi A</span>, <span class="author">Rippert A</span>, <span class="author">Okombo J</span>, <span class="author">Muriithi S</span>, <span class="author">Sasi P</span>, <span class="author">Kortok MM</span>, <span class="author">Lowe B</span>, <span class="author">Campino S</span>, <span class="author">Assefa S</span>, <span class="author">Auburn S</span>, <span class="author">Manske M</span>, <span class="author">Maslen G</span>, <span class="author">Peshu N</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Marsh K</span>, <span class="author">Nzila A</span> and <span class="author">Clark TG</span></p>
<p class="affiliation">1] KEMRI-Wellcome Trust Research Programme, Kilifi, Kenya [2] Institute of Microbiology, Magdeburg University School of Medicine, Germany.</p>
<p class="abstract">Early identification of causal genetic variants underlying antimalarial drug resistance could provide robust epidemiological tools for timely public health interventions. Using a novel natural genetics strategy for mapping novel candidate genes we analyzed >75,000 high quality single nucleotide polymorphisms selected from high-resolution whole-genome sequencing data in 27 isolates of Plasmodium falciparum. We identified genetic variants associated with susceptibility to dihydroartemisinin that implicate one region on chromosome 13, a candidate gene on chromosome 1 (PFA0220w, a UBP1 ortholog) and others (PFB0560w, PFB0630c, PFF0445w) with putative roles in protein homeostasis and stress response. There was a strong signal for positive selection on PFA0220w, but not the other candidate loci. Our results demonstrate the power of full-genome sequencing-based association studies for uncovering candidate genes that determine parasite sensitivity to artemisinins. Our study provides a unique reference for the interpretation of results from resistant infections.</p>
<p class="publication"><span class="title">Scientific reports</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="pagerange">3318</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24270944">24270944</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3839035">3839035</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsrep03318">10.1038/srep03318</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A variant in LDLR is associated with abdominal aortic aneurysm.</h4>
<p class="authors"><span class="author">Bradley DT</span>, <span class="author">Hughes AE</span>, <span class="author">Badger SA</span>, <span class="author">Jones GT</span>, <span class="author">Harrison SC</span>, <span class="author">Wright BJ</span>, <span class="author">Bumpstead S</span>, <span class="author">Baas AF</span>, <span class="author">Gr&eacute;tarsd&oacute;ttir S</span>, <span class="author">Burnand K</span>, <span class="author">Child AH</span>, <span class="author">Clough RE</span>, <span class="author">Cockerill G</span>, <span class="author">Hafez H</span>, <span class="author">Scott DJ</span>, <span class="author">Ari&euml;ns RA</span>, <span class="author">Johnson A</span>, <span class="author">Sohrabi S</span>, <span class="author">Smith A</span>, <span class="author">Thompson MM</span>, <span class="author">van Bockxmeer FM</span>, <span class="author">Waltham M</span>, <span class="author">Matth&iacute;asson SE</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Blankensteijn JD</span>, <span class="author">Teijink JA</span>, <span class="author">Wijmenga C</span>, <span class="author">de Graaf J</span>, <span class="author">Kiemeney LA</span>, <span class="author">Wild JB</span>, <span class="author">Edkins S</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt SE</span>, <span class="author">Potter S</span>, <span class="author">Lindholt JS</span>, <span class="author">Golledge J</span>, <span class="author">Norman PE</span>, <span class="author">van Rij A</span>, <span class="author">Powell JT</span>, <span class="author">Eriksson P</span>, <span class="author">Stef&aacute;nsson K</span>, <span class="author">Thompson JR</span>, <span class="author">Humphries SE</span>, <span class="author">Sayers RD</span>, <span class="author">Deloukas P</span>, <span class="author">Samani NJ</span> and <span class="author">Bown MJ</span></p><p class="abstract"><strong>Background</strong>: Abdominal aortic aneurysm (AAA) is a common cardiovascular disease among older people and demonstrates significant heritability. In contrast to similar complex diseases, relatively few genetic associations with AAA have been confirmed. We reanalyzed our genome-wide study and carried through to replication suggestive discovery associations at a lower level of significance.</p><p class="abstract">A genome-wide association study was conducted using 1830 cases from the United Kingdom, New Zealand, and Australia with infrarenal aorta diameter&ge;30 mm or ruptured AAA and 5435 unscreened controls from the 1958 Birth Cohort and National Blood Service cohort from the Wellcome Trust Case Control Consortium. Eight suggestive associations with P&lt;1&times;10(-4) were carried through to in silico replication in 1292 AAA cases and 30,503 controls. One single-nucleotide polymorphism associated with P&lt;0.05 after Bonferroni correction in the in silico study underwent further replication (706 AAA cases and 1063 controls from the United Kingdom, 507 AAA cases and 199 controls from Denmark, and 885 AAA cases and 1000 controls from New Zealand). Low-density lipoprotein receptor (LDLR) rs6511720 A was significantly associated overall and in 3 of 5 individual replication studies. The full study showed an association that reached genome-wide significance (odds ratio, 0.76; 95% confidence interval, 0.70-0.83; P=2.08&times;10(-10)).</p><p class="abstract"><strong>Conclusions</strong>: LDLR rs6511720 is associated with AAA. This finding is consistent with established effects of this variant on coronary artery disease. Shared causal pathways with other cardiovascular diseases may present novel opportunities for preventative and therapeutic strategies for AAA.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/11/16/28696</span>, <span class="grant">PG/10/001/28098</span>; Medical Research Council: <span class="grant">G1001799</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">084695</span>, <span class="grant">085475</span></p>
<p class="publication"><span class="title">Circulation. Cardiovascular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">498-504</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24046328">24046328</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCGENETICS.113.000165">10.1161/CIRCGENETICS.113.000165</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Platelet Genomics</h4>
<p class="authors"><span class="author">Bray,P.F.</span>, <span class="author">Jones,C.I.</span>, <span class="author">SORANZO,N.</span> and <span class="author">Ouwehand,W.H.</span></p>
<p class="publication"><span class="title">Platelets</span>&nbsp;<span class="year">2013</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-387837-3.00004-3">10.1016/B978-0-12-387837-3.00004-3</a>; URL: <a title="http://dx.doi.org/10.1016/B978-0-12-387837-3.00004-3" href="http://dx.doi.org/10.1016/B978-0-12-387837-3.00004-3">http://dx.doi.org/10.1016/B97<span class="print-hide">...</span><span class="print-show">8-0-12-38783</span>7-3.00004-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new method for high-resolution imaging of Ku foci to decipher mechanisms of DNA double-strand break repair.</h4>
<p class="authors"><span class="author">Britton S</span>, <span class="author">Coates J</span> and <span class="author">Jackson SP</span></p>
<p class="affiliation">The Wellcome Trust and Cancer Research UK Gurdon Institute, University of Cambridge, Cambridge, CB2 1QN, England, UK.</p><p class="abstract">DNA double-strand breaks (DSBs) are the most toxic of all genomic insults, and pathways dealing with their signaling and repair are crucial to prevent cancer and for immune system development. Despite intense investigations, our knowledge of these pathways has been technically limited by our inability to detect the main repair factors at DSBs in cells. In this paper, we present an original method that involves a combination of ribonuclease- and detergent-based preextraction with high-resolution microscopy. This method allows direct visualization of previously hidden repair complexes, including the main DSB sensor Ku, at virtually any type of DSB, including those induced by anticancer agents. We demonstrate its broad range of applications by coupling it to laser microirradiation, super-resolution microscopy, and single-molecule counting to investigate the spatial organization and composition of repair factories. Furthermore, we use our method to monitor DNA repair and identify mechanisms of repair pathway choice, and we show its utility in defining cellular sensitivities and resistance mechanisms to anticancer agents.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A11224</span>, <span class="grant">C6/A11224</span>, <span class="grant">C6946/A14492</span>; European Research Council: <span class="grant">268536</span>; Wellcome Trust: <span class="grant">092096</span>, <span class="grant">WT092096</span></p>
<p class="publication"><span class="title">The Journal of cell biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">202</span>;<span class="number">3</span>;<span class="pagerange">579-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23897892">23897892</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3734090">3734090</a>; DOI: <a href="http://dx.doi.org/10.1083%2Fjcb.201303073">10.1083/jcb.201303073</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neolithic mitochondrial haplogroup H genomes and the genetic origins of Europeans.</h4>
<p class="authors"><span class="author">Brotherton P</span>, <span class="author">Haak W</span>, <span class="author">Templeton J</span>, <span class="author">Brandt G</span>, <span class="author">Soubrier J</span>, <span class="author">Jane Adler C</span>, <span class="author">Richards SM</span>, <span class="author">Sarkissian CD</span>, <span class="author">Ganslmeier R</span>, <span class="author">Friederich S</span>, <span class="author">Dresely V</span>, <span class="author">van Oven M</span>, <span class="author">Kenyon R</span>, <span class="author">Van der Hoek MB</span>, <span class="author">Korlach J</span>, <span class="author">Luong K</span>, <span class="author">Ho SY</span>, <span class="author">Quintana-Murci L</span>, <span class="author">Behar DM</span>, <span class="author">Meller H</span>, <span class="author">Alt KW</span>, <span class="author">Cooper A</span>, <span class="author">Genographic Consortium</span>, <span class="author">Adhikarla S</span>, <span class="author">Ganesh Prasad AK</span>, <span class="author">Pitchappan R</span>, <span class="author">Varatharajan Santhakumari A</span>, <span class="author">Balanovska E</span>, <span class="author">Balanovsky O</span>, <span class="author">Bertranpetit J</span>, <span class="author">Comas D</span>, <span class="author">Mart&iacute;nez-Cruz B</span>, <span class="author">Mel&eacute; M</span>, <span class="author">Clarke AC</span>, <span class="author">Matisoo-Smith EA</span>, <span class="author">Dulik MC</span>, <span class="author">Gaieski JB</span>, <span class="author">Owings AC</span>, <span class="author">Schurr TG</span>, <span class="author">Vilar MG</span>, <span class="author">Hobbs A</span>, <span class="author">Soodyall H</span>, <span class="author">Javed A</span>, <span class="author">Parida L</span>, <span class="author">Platt DE</span>, <span class="author">Royyuru AK</span>, <span class="author">Jin L</span>, <span class="author">Li S</span>, <span class="author">Kaplan ME</span>, <span class="author">Merchant NC</span>, <span class="author">John Mitchell R</span>, <span class="author">Renfrew C</span>, <span class="author">Lacerda DR</span>, <span class="author">Santos FR</span>, <span class="author">Soria Hernanz DF</span>, <span class="author">Spencer Wells R</span>, <span class="author">Swamikrishnan P</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Paulo Vieira P</span> and <span class="author">Ziegle JS</span></p>
<p class="affiliation">The Australian Centre for Ancient DNA, School of Earth and Environmental Sciences, University of Adelaide, Adelaide, South Australia 5005, Australia. <a href="mailto:%70.%6D.%62%72%6F%74%68%65%72%74%6F%6E%40%68%75%64.%61%63.%75%6B">&#112;&#46;&#109;&#46;&#98;&#114;&#111;&#116;&#104;&#101;&#114;&#116;&#111;&#110;&#64;&#104;&#117;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Haplogroup H dominates present-day Western European mitochondrial DNA variability (&gt;40%), yet was less common (~19%) among Early Neolithic farmers (~5450 BC) and virtually absent in Mesolithic hunter-gatherers. Here we investigate this major component of the maternal population history of modern Europeans and sequence 39 complete haplogroup H mitochondrial genomes from ancient human remains. We then compare this &#39;real-time&#39; genetic data with cultural changes taking place between the Early Neolithic (~5450 BC) and Bronze Age (~2200 BC) in Central Europe. Our results reveal that the current diversity and distribution of haplogroup H were largely established by the Mid Neolithic (~4000 BC), but with substantial genetic contributions from subsequent pan-European cultures such as the Bell Beakers expanding out of Iberia in the Late Neolithic (~2800 BC). Dated haplogroup H genomes allow us to reconstruct the recent evolutionary history of haplogroup H and reveal a mutation rate 45% higher than current estimates for human mitochondria.</p>
<p class="publication"><span class="title">Nature communications</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="pagerange">1764</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23612305">23612305</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncomms2656">10.1038/ncomms2656</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Translating the human microbiome.</h4>
<p class="authors"><span class="author">Brown J</span>, <span class="author">de Vos WM</span>, <span class="author">DiStefano PS</span>, <span class="author">Dor&eacute; J</span>, <span class="author">Huttenhower C</span>, <span class="author">Knight R</span>, <span class="author">Lawley TD</span>, <span class="author">Raes J</span> and <span class="author">Turnbaugh P</span></p>
<p class="affiliation">GlaxoSmithKline, Collegeville, Pennsylvania, USA. <a href="mailto:%6A%61%6D%65%73.%72.%62%72%6F%77%6E%40%67%73%6B.%63%6F%6D">&#106;&#97;&#109;&#101;&#115;&#46;&#114;&#46;&#98;&#114;&#111;&#119;&#110;&#64;&#103;&#115;&#107;&#46;&#99;&#111;&#109;</a></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2013</span>;<span class="volume">31</span>;<span class="number">4</span>;<span class="pagerange">304-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23563424">23563424</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.2543">10.1038/nbt.2543</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Culture-free club.</h4>
<p class="authors"><span class="author">Bryant JM</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23748338">23748338</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3052">10.1038/nrmicro3052</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing to identify transmission of Mycobacterium abscessus between patients with cystic fibrosis: a retrospective cohort study.</h4>
<p class="authors"><span class="author">Bryant JM</span>, <span class="author">Grogono DM</span>, <span class="author">Greaves D</span>, <span class="author">Foweraker J</span>, <span class="author">Roddick I</span>, <span class="author">Inns T</span>, <span class="author">Reacher M</span>, <span class="author">Haworth CS</span>, <span class="author">Curran MD</span>, <span class="author">Harris SR</span>, <span class="author">Peacock SJ</span>, <span class="author">Parkhill J</span> and <span class="author">Floto RA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract"><strong>Background</strong>: Increasing numbers of individuals with cystic fibrosis are becoming infected with the multidrug-resistant non-tuberculous mycobacterium (NTM) Mycobacterium abscessus, which causes progressive lung damage and is extremely challenging to treat. How this organism is acquired is not currently known, but there is growing concern that person-to-person transmission could occur. We aimed to define the mechanisms of acquisition of M abscessus in individuals with cystic fibrosis.</p><p class="abstract"><strong>Method</strong>: Whole genome sequencing and antimicrobial susceptibility testing were done on 168 consecutive isolates of M abscessus from 31 patients attending an adult cystic fibrosis centre in the UK between 2007 and 2011. In parallel, we undertook detailed environmental testing for NTM and defined potential opportunities for transmission between patients both in and out of hospital using epidemiological data and social network analysis.</p><p class="abstract"><strong>Findings</strong>: Phylogenetic analysis revealed two clustered outbreaks of near-identical isolates of the M abscessus subspecies massiliense (from 11 patients), differing by less than ten base pairs. This variation represents less diversity than that seen within isolates from a single individual, strongly indicating between-patient transmission. All patients within these clusters had numerous opportunities for within-hospital transmission from other individuals, while comprehensive environmental sampling, initiated during the outbreak, failed to detect any potential point source of NTM infection. The clusters of M abscessus subspecies massiliense showed evidence of transmission of mutations acquired during infection of an individual to other patients. Thus, isolates with constitutive resistance to amikacin and clarithromycin were isolated from several individuals never previously exposed to long-term macrolides or aminoglycosides, further indicating cross-infection.</p><p class="abstract"><strong>Interpretation</strong>: Whole genome sequencing has revealed frequent transmission of multidrug resistant NTM between patients with cystic fibrosis despite conventional cross-infection measures. Although the exact transmission route is yet to be established, our epidemiological analysis suggests that it could be indirect.</p><p class="abstract"><strong>Funding</strong>: The Wellcome Trust, Papworth Hospital, NIHR Cambridge Biomedical Research Centre, UK Health Protection Agency, Medical Research Council, and the UKCRC Translational Infection Research Initiative.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">974668</span>, <span class="grant">G1000803</span>; Wellcome Trust: <span class="grant">084953</span>, <span class="grant">098051</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2013</span>;<span class="volume">381</span>;<span class="number">9877</span>;<span class="pagerange">1551-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23541540">23541540</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3664974">3664974</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2813%2960632-7">10.1016/S0140-6736(13)60632-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transmission of M abscessus in patients with cystic fibrosis - Authors' reply.</h4>
<p class="authors"><span class="author">Bryant JM</span>, <span class="author">Grogono DM</span>, <span class="author">Parkhill J</span> and <span class="author">Floto RA</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2013</span>;<span class="volume">382</span>;<span class="number">9891</span>;<span class="pagerange">504</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23931921">23931921</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2813%2961709-2">10.1016/S0140-6736(13)61709-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing to establish relapse or re-infection with Mycobacterium tuberculosis: a retrospective observational study.</h4>
<p class="authors"><span class="author">Bryant JM</span>, <span class="author">Harris SR</span>, <span class="author">Parkhill J</span>, <span class="author">Dawson R</span>, <span class="author">Diacon AH</span>, <span class="author">van Helden P</span>, <span class="author">Pym A</span>, <span class="author">Mahayiddin AA</span>, <span class="author">Chuchottaworn C</span>, <span class="author">Sanne IM</span>, <span class="author">Louw C</span>, <span class="author">Boeree MJ</span>, <span class="author">Hoelscher M</span>, <span class="author">McHugh TD</span>, <span class="author">Bateson AL</span>, <span class="author">Hunt RD</span>, <span class="author">Mwaigwisya S</span>, <span class="author">Wright L</span>, <span class="author">Gillespie SH</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p>
<p class="abstract">Background: Recurrence of tuberculosis after treatment makes management difficult and is a key factor for determining treatment efficacy. Two processes can cause recurrence: relapse of the primary infection or re-infection with an exogenous strain. Although re-infection can and does occur, its importance to tuberculosis epidemiology and its biological basis is still debated. We used whole-genome sequencing-which is more accurate than conventional typing used to date-to assess the frequency of recurrence and to gain insight into the biological basis of re-infection.

Methods: We assessed patients from the REMoxTB trial-a randomised controlled trial of tuberculosis treatment that enrolled previously untreated participants with Mycobacterium tuberculosis infection from Malaysia, South Africa, and Thailand. We did whole-genome sequencing and mycobacterial interspersed repetitive unit-variable number of tandem repeat (MIRU-VNTR) typing of pairs of isolates taken by sputum sampling: one from before treatment and another from either the end of failed treatment at 17 weeks or later or from a recurrent infection. We compared the number and location of SNPs between isolates collected at baseline and recurrence.

Findings: We assessed 47 pairs of isolates. Whole-genome sequencing identified 33 cases with little genetic distance (0-6 SNPs) between strains, deemed relapses, and three cases for which the genetic distance ranged from 1306 to 1419 SNPs, deemed re-infections. Six cases of relapse and six cases of mixed infection were classified differently by whole-genome sequencing and MIRU-VNTR. We detected five single positive isolates (positive culture followed by at least two negative cultures) without clinical evidence of disease.

Interpretation: Whole-genome sequencing enables the differentiation of relapse and re-infection cases with greater resolution than do genotyping methods used at present, such as MIRU-VNTR, and provides insights into the biology of recurrence. The additional clarity provided by whole-genome sequencing might have a role in defining endpoints for clinical trials.

Funding: Wellcome Trust, European Union, Medical Research Council, Global Alliance for TB Drug Development, European and Developing Country Clinical Trials Partnership.</p>
<p class="publication"><span class="title">The lancet. Respiratory medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">10</span>;<span class="pagerange">786-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24461758">24461758</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3861685">3861685</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS2213-2600%2813%2970231-5">10.1016/S2213-2600(13)70231-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inferring patient to patient transmission of Mycobacterium tuberculosis from whole genome sequencing data.</h4>
<p class="authors"><span class="author">Bryant JM</span>, <span class="author">Sch&uuml;rch AC</span>, <span class="author">van Deutekom H</span>, <span class="author">Harris SR</span>, <span class="author">de Beer JL</span>, <span class="author">de Jager V</span>, <span class="author">Kremer K</span>, <span class="author">van Hijum SA</span>, <span class="author">Siezen RJ</span>, <span class="author">Borgdorff M</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span> and <span class="author">van Soolingen D</span></p><p class="abstract"><strong>Background</strong>: Mycobacterium tuberculosis is characterised by limited genomic diversity, which makes the application of whole genome sequencing particularly attractive for clinical and epidemiological investigation. However, in order to confidently infer transmission events, an accurate knowledge of the rate of change in the genome over relevant timescales is required.</p><p class="abstract"><strong>Methods</strong>: We attempted to estimate a molecular clock by sequencing 199 isolates from epidemiologically linked tuberculosis cases, collected in the Netherlands spanning almost 16 years.</p><p class="abstract"><strong>Results</strong>: Multiple analyses support an average mutation rate of ~0.3 SNPs per genome per year. However, all analyses revealed a very high degree of variation around this mean, making the confirmation of links proposed by epidemiology, and inference of novel links, difficult. Despite this, in some cases, the phylogenetic context of other strains provided evidence supporting the confident exclusion of previously inferred epidemiological links.</p><p class="abstract"><strong>Conclusions</strong>: This in-depth analysis of the molecular clock revealed that it is slow and variable over short time scales, which limits its usefulness in transmission studies. However, the superior resolution of whole genome sequencing can provide the phylogenetic context to allow the confident exclusion of possible transmission events previously inferred via traditional DNA fingerprinting techniques and epidemiological cluster investigation. Despite the slow generation of variation even at the whole genome level we conclude that the investigation of tuberculosis transmission will benefit greatly from routine whole genome sequencing.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">974668</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">BMC infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="pagerange">110</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23446317">23446317</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3599118">3599118</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2334-13-110">10.1186/1471-2334-13-110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovery by the Epistasis Project of an epistatic interaction between the GSTM3 gene and the HHEX/IDE/KIF11 locus in the risk of Alzheimer&#39;s disease.</h4>
<p class="authors"><span class="author">Bullock JM</span>, <span class="author">Medway C</span>, <span class="author">Cortina-Borja M</span>, <span class="author">Turton JC</span>, <span class="author">Prince JA</span>, <span class="author">Ibrahim-Verbaas CA</span>, <span class="author">Schuur M</span>, <span class="author">Breteler MM</span>, <span class="author">van Duijn CM</span>, <span class="author">Kehoe PG</span>, <span class="author">Barber R</span>, <span class="author">Coto E</span>, <span class="author">Alvarez V</span>, <span class="author">Deloukas P</span>, <span class="author">Hammond N</span>, <span class="author">Combarros O</span>, <span class="author">Mateo I</span>, <span class="author">Warden DR</span>, <span class="author">Lehmann MG</span>, <span class="author">Belbin O</span>, <span class="author">Brown K</span>, <span class="author">Wilcock GK</span>, <span class="author">Heun R</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">Smith AD</span>, <span class="author">Lehmann DJ</span> and <span class="author">Morgan K</span></p>
<p class="affiliation">Human Genetics, School of Molecular Medical Sciences, Queen&#39;s Medical Centre, University of Nottingham, Nottingham, UK.</p><p class="abstract">Despite recent discoveries in the genetics of sporadic Alzheimer&#39;s disease, there remains substantial &quot;hidden heritability.&quot; It is thought that some of this missing heritability may be because of gene-gene, i.e., epistatic, interactions. We examined potential epistasis between 110 candidate polymorphisms in 1757 cases of Alzheimer&#39;s disease and 6294 control subjects of the Epistasis Project, divided between a discovery and a replication dataset. We found an epistatic interaction, between rs7483 in GSTM3 and rs1111875 in the HHEX/IDE/KIF11 gene cluster, with a closely similar, significant result in both datasets. The synergy factor (SF) in the combined dataset was 1.79, 95% confidence interval [CI], 1.35-2.36; p = 0.00004. Consistent interaction was also found in 7 out of the 8 additional subsets that we examined post hoc: i.e., it was shown in both North Europe and North Spain, in both men and women, in both those with and without the &epsilon;4 allele of apolipoprotein E, and in people older than 75 years (SF, 2.27; 95% CI, 1.60-3.20; p &lt; 0.00001), but not in those younger than 75 years (SF, 1.06; 95% CI, 0.59-1.91; p = 0.84). The association with Alzheimer&#39;s disease was purely epistatic with neither polymorphism showing an independent effect: odds ratio, 1.0; p &ge; 0.7. Indeed, each factor was associated with protection in the absence of the other factor, but with risk in its presence. In conclusion, this epistatic interaction showed a high degree of consistency when stratifying by sex, the &epsilon;4 allele of apolipoprotein E genotype, and geographic region.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0400546</span></p>
<p class="publication"><span class="title">Neurobiology of aging</span>&nbsp;<span class="year">2013</span>;<span class="volume">34</span>;<span class="number">4</span>;<span class="pagerange">1309.e1-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23036584">23036584</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neurobiolaging.2012.08.010">10.1016/j.neurobiolaging.2012.08.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Headbobber: a combined morphogenetic and cochleosaccular mouse model to study 10qter deletions in human deafness.</h4>
<p class="authors"><span class="author">Buniello A</span>, <span class="author">Hardisty-Hughes RE</span>, <span class="author">Pass JC</span>, <span class="author">Bober E</span>, <span class="author">Smith RJ</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom ; Wolfson Centre for Age-Related Diseases, King's College London, London, United Kingdom.</p>
<p class="abstract">The recessive mouse mutant headbobber () displays the characteristic behavioural traits associated with vestibular defects including headbobbing, circling and deafness. This mutation was caused by the insertion of a transgene into distal chromosome 7 affecting expression of native genes. We show that the inner ear of mutants lacks semicircular canals and cristae, and the saccule and utricle are fused together in a single utriculosaccular sac. Moreover, we detect severe abnormalities of the cochlear sensory hair cells, the stria vascularis looks severely disorganised, Reissner's membrane is collapsed and no endocochlear potential is detected. Myo7a and Kcnj10 expression analysis show a lack of the melanocyte-like intermediate cells in stria vascularis, which can explain the absence of endocochlear potential. We use Trp2 as a marker of melanoblasts migrating from the neural crest at E12.5 and show that they do not interdigitate into the developing strial epithelium, associated with abnormal persistence of the basal lamina in the cochlea. We perform array CGH, deep sequencing as well as an extensive expression analysis of candidate genes in the headbobber region of and littermate controls, and conclude that the headbobber phenotype is caused by: 1) effect of a 648 kb deletion on distal Chr7, resulting in the loss of three protein coding genes (, and ) with expression in the inner ear but unknown function; and 2) indirect, long range effect of the deletion on the expression of neighboring genes on Chr7, associated with downregulation of and homeobox transcription factors. Interestingly, deletions of the orthologous region in humans, affecting the same genes, have been reported in nineteen patients with common features including sensorineural hearing loss and vestibular problems. Therefore, we propose that headbobber is a useful model to gain insight into the mechanisms underlying deafness in human 10qter deletion syndrome.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">e56274</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23457544">23457544</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0056274">10.1371/journal.pone.0056274</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A full-length recombinant Plasmodium falciparum PfRH5 protein induces inhibitory antibodies that are effective across common PfRH5 genetic variants.</h4>
<p class="authors"><span class="author">Bustamante LY</span>, <span class="author">Bartholdson SJ</span>, <span class="author">Crosnier C</span>, <span class="author">Campos MG</span>, <span class="author">Wanaguru M</span>, <span class="author">Nguon C</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Wright GJ</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">Malaria Programme, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The lack of an effective licensed vaccine remains one of the most significant gaps in the portfolio of tools being developed to eliminate Plasmodium falciparum malaria. Vaccines targeting erythrocyte invasion - an essential step for both parasite development and malaria pathogenesis - have faced the particular challenge of genetic diversity. Immunity-driven balancing selection pressure on parasite invasion proteins often results in the presence of multiple, antigenically distinct, variants within a population, leading to variant-specific immune responses. Such variation makes it difficult to design a vaccine that covers the full range of diversity, and could potentially facilitate the evolution of vaccine-resistant parasite strains. In this study, we investigate the effect of genetic diversity on invasion inhibition by antibodies to a high priority P. falciparum invasion candidate antigen, P. falciparum Reticulocyte Binding Protein Homologue 5 (PfRH5). Previous work has shown that virally delivered PfRH5 can induce antibodies that protect against a wide range of genetic variants. Here, we show that a full-length recombinant PfRH5 protein expressed in mammalian cells is biochemically active, as judged by saturable binding to its receptor, basigin, and is able to induce antibodies that strongly inhibit P. falciparum growth and invasion. Whole genome sequencing of 290 clinical P. falciparum isolates from across the world identifies only five non-synonymous PfRH5 SNPs that are present at frequencies of 10% or more in at least one geographical region. Antibodies raised against the 3D7 variant of PfRH5 were able to inhibit nine different P. falciparum strains, which between them included all of the five most common PfRH5 SNPs in this dataset, with no evidence for strain-specific immunity. We conclude that protein-based PfRH5 vaccines are an urgent priority for human efficacy trials.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Vaccine</span>&nbsp;<span class="year">2013</span>;<span class="volume">31</span>;<span class="number">2</span>;<span class="pagerange">373-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23146673">23146673</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3538003">3538003</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vaccine.2012.10.106">10.1016/j.vaccine.2012.10.106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Missense mutations in Î²-1,3-N-acetylglucosaminyltransferase 1 (B3GNT1) cause Walker-Warburg syndrome.</h4>
<p class="authors"><span class="author">Buysse K</span>, <span class="author">Riemersma M</span>, <span class="author">Powell G</span>, <span class="author">van Reeuwijk J</span>, <span class="author">Chitayat D</span>, <span class="author">Roscioli T</span>, <span class="author">Kamsteeg EJ</span>, <span class="author">van den Elzen C</span>, <span class="author">van Beusekom E</span>, <span class="author">Blaser S</span>, <span class="author">Babul-Hirji R</span>, <span class="author">Halliday W</span>, <span class="author">Wright GJ</span>, <span class="author">Stemple DL</span>, <span class="author">Lin YY</span>, <span class="author">Lefeber DJ</span> and <span class="author">van Bokhoven H</span></p>
<p class="affiliation">The authors wish it to be known that, in their opinion, the first five authors should be regarded as joint First Authors.</p>
<p class="abstract">Several known or putative glycosyltransferases are required for the synthesis of laminin-binding glycans on alpha-dystroglycan (Î±DG), including POMT1, POMT2, POMGnT1, LARGE, Fukutin, FKRP, ISPD and GTDC2. Mutations in these glycosyltransferase genes result in defective Î±DG glycosylation and reduced ligand binding by Î±DG causing a clinically heterogeneous group of congenital muscular dystrophies, commonly referred to as dystroglycanopathies. The most severe clinical form, Walker-Warburg syndrome (WWS), is characterized by congenital muscular dystrophy and severe neurological and ophthalmological defects. Here, we report two homozygous missense mutations in the Î²-1,3-N-acetylglucosaminyltransferase 1 (B3GNT1) gene in a family affected with WWS. Functional studies confirmed the pathogenicity of the mutations. First, expression of wild-type but not mutant B3GNT1 in human prostate cancer (PC3) cells led to increased levels of Î±DG glycosylation. Second, morpholino knockdown of the zebrafish b3gnt1 orthologue caused characteristic muscular defects and reduced Î±DG glycosylation. These functional studies identify an important role of B3GNT1 in the synthesis of the uncharacterized laminin-binding glycan of Î±DG and implicate B3GNT1 as a novel causative gene for WWS.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">9</span>;<span class="pagerange">1746-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23359570">23359570</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3613162">3613162</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddt021">10.1093/hmg/ddt021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A CRISPR view of genome sequences.</h4>
<p class="authors"><span class="author">Cain AK</span> and <span class="author">Boinett CJ</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">This month's Genome Watch explores recent applications of the CRISPR immune system for bacterial phylogenetic analysis and genome editing.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23474684">23474684</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2997">10.1038/nrmicro2997</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale association analysis identifies new risk loci for coronary artery disease.</h4>
<p class="authors"><span class="author">CARDIoGRAMplusC4D Consortium</span>, <span class="author">Deloukas P</span>, <span class="author">Kanoni S</span>, <span class="author">Willenborg C</span>, <span class="author">Farrall M</span>, <span class="author">Assimes TL</span>, <span class="author">Thompson JR</span>, <span class="author">Ingelsson E</span>, <span class="author">Saleheen D</span>, <span class="author">Erdmann J</span>, <span class="author">Goldstein BA</span>, <span class="author">Stirrups K</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Cazier JB</span>, <span class="author">Johansson A</span>, <span class="author">Hall AS</span>, <span class="author">Lee JY</span>, <span class="author">Willer CJ</span>, <span class="author">Chambers JC</span>, <span class="author">Esko T</span>, <span class="author">Folkersen L</span>, <span class="author">Goel A</span>, <span class="author">Grundberg E</span>, <span class="author">Havulinna AS</span>, <span class="author">Ho WK</span>, <span class="author">Hopewell JC</span>, <span class="author">Eriksson N</span>, <span class="author">Kleber ME</span>, <span class="author">Kristiansson K</span>, <span class="author">Lundmark P</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Rafelt S</span>, <span class="author">Shungin D</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Thorleifsson G</span>, <span class="author">Tikkanen E</span>, <span class="author">Van Zuydam N</span>, <span class="author">Voight BF</span>, <span class="author">Waite LL</span>, <span class="author">Zhang W</span>, <span class="author">Ziegler A</span>, <span class="author">Absher D</span>, <span class="author">Altshuler D</span>, <span class="author">Balmforth AJ</span>, <span class="author">Barroso I</span>, <span class="author">Braund PS</span>, <span class="author">Burgdorf C</span>, <span class="author">Claudi-Boehm S</span>, <span class="author">Cox D</span>, <span class="author">Dimitriou M</span>, <span class="author">Do R</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">CARDIOGENICS Consortium</span>, <span class="author">Doney AS</span>, <span class="author">El Mokhtari N</span>, <span class="author">Eriksson P</span>, <span class="author">Fischer K</span>, <span class="author">Fontanillas P</span>, <span class="author">Franco-Cereceda A</span>, <span class="author">Gigante B</span>, <span class="author">Groop L</span>, <span class="author">Gustafsson S</span>, <span class="author">Hager J</span>, <span class="author">Hallmans G</span>, <span class="author">Han BG</span>, <span class="author">Hunt SE</span>, <span class="author">Kang HM</span>, <span class="author">Illig T</span>, <span class="author">Kessler T</span>, <span class="author">Knowles JW</span>, <span class="author">Kolovou G</span>, <span class="author">Kuusisto J</span>, <span class="author">Langenberg C</span>, <span class="author">Langford C</span>, <span class="author">Leander K</span>, <span class="author">Lokki ML</span>, <span class="author">Lundmark A</span>, <span class="author">McCarthy MI</span>, <span class="author">Meisinger C</span>, <span class="author">Melander O</span>, <span class="author">Mihailov E</span>, <span class="author">Maouche S</span>, <span class="author">Morris AD</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">MuTHER Consortium</span>, <span class="author">Nikus K</span>, <span class="author">Peden JF</span>, <span class="author">Rayner NW</span>, <span class="author">Rasheed A</span>, <span class="author">Rosinger S</span>, <span class="author">Rubin D</span>, <span class="author">Rumpf MP</span>, <span class="author">Sch&auml;fer A</span>, <span class="author">Sivananthan M</span>, <span class="author">Song C</span>, <span class="author">Stewart AF</span>, <span class="author">Tan ST</span>, <span class="author">Thorgeirsson G</span>, <span class="author">van der Schoot CE</span>, <span class="author">Wagner PJ</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Wells GA</span>, <span class="author">Wild PS</span>, <span class="author">Yang TP</span>, <span class="author">Amouyel P</span>, <span class="author">Arveiler D</span>, <span class="author">Basart H</span>, <span class="author">Boehnke M</span>, <span class="author">Boerwinkle E</span>, <span class="author">Brambilla P</span>, <span class="author">Cambien F</span>, <span class="author">Cupples AL</span>, <span class="author">de Faire U</span>, <span class="author">Dehghan A</span>, <span class="author">Diemert P</span>, <span class="author">Epstein SE</span>, <span class="author">Evans A</span>, <span class="author">Ferrario MM</span>, <span class="author">Ferri&egrave;res J</span>, <span class="author">Gauguier D</span>, <span class="author">Go AS</span>, <span class="author">Goodall AH</span>, <span class="author">Gudnason V</span>, <span class="author">Hazen SL</span>, <span class="author">Holm H</span>, <span class="author">Iribarren C</span>, <span class="author">Jang Y</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kee F</span>, <span class="author">Kim HS</span>, <span class="author">Klopp N</span>, <span class="author">Koenig W</span>, <span class="author">Kratzer W</span>, <span class="author">Kuulasmaa K</span>, <span class="author">Laakso M</span>, <span class="author">Laaksonen R</span>, <span class="author">Lee JY</span>, <span class="author">Lind L</span>, <span class="author">Ouwehand WH</span>, <span class="author">Parish S</span>, <span class="author">Park JE</span>, <span class="author">Pedersen NL</span>, <span class="author">Peters A</span>, <span class="author">Quertermous T</span>, <span class="author">Rader DJ</span>, <span class="author">Salomaa V</span>, <span class="author">Schadt E</span>, <span class="author">Shah SH</span>, <span class="author">Sinisalo J</span>, <span class="author">Stark K</span>, <span class="author">Stefansson K</span>, <span class="author">Tr&eacute;gou&euml;t DA</span>, <span class="author">Virtamo J</span>, <span class="author">Wallentin L</span>, <span class="author">Wareham N</span>, <span class="author">Zimmermann ME</span>, <span class="author">Nieminen MS</span>, <span class="author">Hengstenberg C</span>, <span class="author">Sandhu MS</span>, <span class="author">Pastinen T</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Hovingh GK</span>, <span class="author">Dedoussis G</span>, <span class="author">Franks PW</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Metspalu A</span>, <span class="author">Zalloua PA</span>, <span class="author">Siegbahn A</span>, <span class="author">Schreiber S</span>, <span class="author">Ripatti S</span>, <span class="author">Blankenberg SS</span>, <span class="author">Perola M</span>, <span class="author">Clarke R</span>, <span class="author">Boehm BO</span>, <span class="author">O&#39;Donnell C</span>, <span class="author">Reilly MP</span>, <span class="author">M&auml;rz W</span>, <span class="author">Collins R</span>, <span class="author">Kathiresan S</span>, <span class="author">Hamsten A</span>, <span class="author">Kooner JS</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Danesh J</span>, <span class="author">Palmer CN</span>, <span class="author">Roberts R</span>, <span class="author">Watkins H</span>, <span class="author">Schunkert H</span> and <span class="author">Samani NJ</span></p><p class="abstract">Coronary artery disease (CAD) is the commonest cause of death. Here, we report an association analysis in 63,746 CAD cases and 130,681 controls identifying 15 loci reaching genome-wide significance, taking the number of susceptibility loci for CAD to 46, and a further 104 independent variants (r(2) &lt; 0.2) strongly associated with CAD at a 5% false discovery rate (FDR). Together, these variants explain approximately 10.6% of CAD heritability. Of the 46 genome-wide significant lead SNPs, 12 show a significant association with a lipid trait, and 5 show a significant association with blood pressure, but none is significantly associated with diabetes. Network analysis with 233 candidate genes (loci at 10% FDR) generated 5 interaction networks comprising 85% of these putative genes involved in CAD. The four most significant pathways mapping to these networks are linked to lipid metabolism and inflammation, underscoring the causal role of these activities in the genetic etiology of CAD. Our study provides insights into the genetic basis of CAD and identifies key biological pathways.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/08/094/26019</span>, <span class="grant">RG/08/014/24067</span>; Medical Research Council: <span class="grant">G0801566</span>, <span class="grant">MC_U137686857</span>; NHLBI NIH HHS: <span class="grant">K24 HL107643</span>, <span class="grant">R00 HL094535</span>, <span class="grant">R01 HL111694</span>; NIDDK NIH HHS: <span class="grant">R01 DK062370</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">1</span>;<span class="pagerange">25-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23202125">23202125</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3679547">3679547</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2480">10.1038/ng.2480</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pitpnm1 is expressed in hair cells during development but is not required for hearing.</h4>
<p class="authors"><span class="author">Carlisle FA</span>, <span class="author">Pearson S</span>, <span class="author">Steel KP</span> and <span class="author">Lewis MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, Cambs, CB10 1SA, United Kingdom.</p>
<p class="abstract">Deafness is a genetically complex disorder with many contributing genes still unknown. Here we describe the expression of Pitpnm1 in the inner ear. It is expressed in the inner hair cells of the organ of Corti from late embryonic stages until adulthood, and transiently in the outer hair cells during early postnatal stages. Despite this specific expression, Pitpnm1 null mice showed no hearing defects, possibly due to redundancy with the paralogous genes Pitpnm2 and Pitpnm3.</p>
<p class="publication"><span class="title">Neuroscience</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23820044">23820044</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neuroscience.2013.06.045">10.1016/j.neuroscience.2013.06.045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adaptive Changes of the Insig1/SREBP1/SCD1 Set Point Help Adipose Tissue to Cope With Increased Storage Demands of Obesity.</h4>
<p class="authors"><span class="author">Carobbio S</span>, <span class="author">Hagen RM</span>, <span class="author">Lelliott CJ</span>, <span class="author">Slawik M</span>, <span class="author">Medina-Gomez G</span>, <span class="author">Tan CY</span>, <span class="author">Sicard A</span>, <span class="author">Atherton HJ</span>, <span class="author">Barbarroja N</span>, <span class="author">Bjursell M</span>, <span class="author">Bohlooly-Y M</span>, <span class="author">Virtue S</span>, <span class="author">Tuthill A</span>, <span class="author">Lefai E</span>, <span class="author">Laville M</span>, <span class="author">Wu T</span>, <span class="author">Considine RV</span>, <span class="author">Vidal H</span>, <span class="author">Langin D</span>, <span class="author">Oresic M</span>, <span class="author">Tinahones FJ</span>, <span class="author">Fernandez-Real JM</span>, <span class="author">Griffin JL</span>, <span class="author">Sethi JK</span>, <span class="author">L&oacute;pez M</span> and <span class="author">Vidal-Puig A</span></p>
<p class="affiliation">University of Cambridge, Metabolic Research Laboratories, Institute of Metabolic Science Addenbrooke's Treatment Centre, Addenbrooke's Hospital, Cambridge, U.K.</p>
<p class="abstract">The epidemic of obesity imposes unprecedented challenges on human adipose tissue (WAT) storage capacity that may benefit from adaptive mechanisms to maintain adipocyte functionality. Here, we demonstrate that changes in the regulatory feedback set point control of Insig1/SREBP1 represent an adaptive response that preserves WAT lipid homeostasis in obese and insulin-resistant states. In our experiments, we show that Insig1 mRNA expression decreases in WAT from mice with obesity-associated insulin resistance and from morbidly obese humans and in in vitro models of adipocyte insulin resistance. Insig1 downregulation is part of an adaptive response that promotes the maintenance of SREBP1 maturation and facilitates lipogenesis and availability of appropriate levels of fatty acid unsaturation, partially compensating the antilipogenic effect associated with insulin resistance. We describe for the first time the existence of this adaptive mechanism in WAT, which involves Insig1/SREBP1 and preserves the degree of lipid unsaturation under conditions of obesity-induced insulin resistance. These adaptive mechanisms contribute to maintain lipid desaturation through preferential SCD1 regulation and facilitate fat storage in WAT, despite on-going metabolic stress.</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2013</span>;<span class="volume">62</span>;<span class="number">11</span>;<span class="pagerange">3697-708</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23919961">23919961</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3806615">3806615</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb12-1748">10.2337/db12-1748</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adipogenesis: new insights into brown adipose tissue differentiation.</h4>
<p class="authors"><span class="author">Carobbio S</span>, <span class="author">Rosen B</span> and <span class="author">Vidal-Puig A</span></p>
<p class="affiliation">S Carobbio, Wellcome Trust Genome Campus, Welcome Trust Sanger Insitute, Cambridge, United Kingdom.</p>
<p class="abstract">Confirmation of the presence of functional brown adipose tissue (BAT) in humans has renewed the interest in investigating the potential therapeutic use of this tissue. The finding that its activity positively correlates with decreased BMI, fat content and augmented energy expenditure suggests that increasing BAT mass/activity or browning of WAT could be a strategy to prevent or treat obesity and its associated morbidities. The challenge now is to find a safe and efficient way to develop this idea. Whereas BAT has being widely studied in murine models both in vivo and in vitro, there is an urgent need for human cellular models to investigate BAT physiology and functionality from a molecular point of view. In our review, we focus on the latest insights surrounding BAT development and activation in rodents and humans. Then, we discuss how the availability of murine models has been essential to identify BAT progenitors and trace their lineage. Finally, we address how this information can be exploited to develop human cellular models for BAT differentiation/activation. In this context, human embryonic (hES) and induced plutipotent cells (hIPS)-based cellular models represent a resource of great potential value, as they can provide a virtually inexhaustible supply of starting material for functional genetic studies, -omics based analysis and validation of therapeutic approaches. Moreover, these cells can be easily genetically engineered, opening the possibility of generating patient-specific cellular models, allowing the investigation of the impact of different genetic backgrounds on BAT differentiation both in pathological or physiological states.</p>
<p class="publication"><span class="title">Journal of molecular endocrinology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24041933">24041933</a>; DOI: <a href="http://dx.doi.org/10.1530%2FJME-13-0158">10.1530/JME-13-0158</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in GDP-Mannose Pyrophosphorylase B Cause Congenital and Limb-Girdle Muscular Dystrophies Associated with Hypoglycosylation of α-Dystroglycan.</h4>
<p class="authors"><span class="author">Carss KJ</span>, <span class="author">Stevens E</span>, <span class="author">Foley AR</span>, <span class="author">Cirak S</span>, <span class="author">Riemersma M</span>, <span class="author">Torelli S</span>, <span class="author">Hoischen A</span>, <span class="author">Willer T</span>, <span class="author">van Scherpenzeel M</span>, <span class="author">Moore SA</span>, <span class="author">Messina S</span>, <span class="author">Bertini E</span>, <span class="author">B&ouml;nnemann CG</span>, <span class="author">Abdenur JE</span>, <span class="author">Grosmann CM</span>, <span class="author">Kesari A</span>, <span class="author">Punetha J</span>, <span class="author">Quinlivan R</span>, <span class="author">Waddell LB</span>, <span class="author">Young HK</span>, <span class="author">Wraige E</span>, <span class="author">Yau S</span>, <span class="author">Brodd L</span>, <span class="author">Feng L</span>, <span class="author">Sewry C</span>, <span class="author">Macarthur DG</span>, <span class="author">North KN</span>, <span class="author">Hoffman E</span>, <span class="author">Stemple DL</span>, <span class="author">Hurles ME</span>, <span class="author">van Bokhoven H</span>, <span class="author">Campbell KP</span>, <span class="author">Lefeber DJ</span>, <span class="author">UK10K Consortium</span>, <span class="author">Lin YY</span> and <span class="author">Muntoni F</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p>
<p class="abstract">Congenital muscular dystrophies with hypoglycosylation of α-dystroglycan (α-DG) are a heterogeneous group of disorders often associated with brain and eye defects in addition to muscular dystrophy. Causative variants in 14 genes thought to be involved in the glycosylation of α-DG have been identified thus far. Allelic mutations in these genes might also cause milder limb-girdle muscular dystrophy phenotypes. Using a combination of exome and Sanger sequencing in eight unrelated individuals, we present evidence that mutations in guanosine diphosphate mannose (GDP-mannose) pyrophosphorylase B (GMPPB) can result in muscular dystrophy variants with hypoglycosylated α-DG. GMPPB catalyzes the formation of GDP-mannose from GTP and mannose-1-phosphate. GDP-mannose is required for O-mannosylation of proteins, including α-DG, and it is the substrate of cytosolic mannosyltransferases. We found reduced α-DG glycosylation in the muscle biopsies of affected individuals and in available fibroblasts. Overexpression of wild-type GMPPB in fibroblasts from an affected individual partially restored glycosylation of α-DG. Whereas wild-type GMPPB localized to the cytoplasm, five of the identified missense mutations caused formation of aggregates in the cytoplasm or near membrane protrusions. Additionally, knockdown of the GMPPB ortholog in zebrafish caused structural muscle defects with decreased motility, eye abnormalities, and reduced glycosylation of α-DG. Together, these data indicate that GMPPB mutations are responsible for congenital and limb-girdle muscular dystrophies with hypoglycosylation of α-DG.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23768512">23768512</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.05.009">10.1016/j.ajhg.2013.05.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of Vitek 2 Antimicrobial Susceptibility Profile To Identify mecC in Methicillin-Resistant Staphylococcus aureus.</h4>
<p class="authors"><span class="author">Cartwright EJ</span>, <span class="author">Paterson GK</span>, <span class="author">Raven KE</span>, <span class="author">Harrison EM</span>, <span class="author">Gouliouris T</span>, <span class="author">Kearns A</span>, <span class="author">Pichon B</span>, <span class="author">Edwards G</span>, <span class="author">Skov RL</span>, <span class="author">Larsen AR</span>, <span class="author">Holmes MA</span>, <span class="author">Parkhill J</span>, <span class="author">Peacock SJ</span> and <span class="author">T&ouml;r&ouml;k ME</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Cambridge, United Kingdom.</p>
<p class="abstract">The emergence of mecC methicillin-resistant Staphylococcus aureus (MRSA) poses a diagnostic challenge for clinical microbiology laboratories. Using the Vitek 2 system, we tested a panel of 896 Staphylococcus aureus isolates and found that an oxacillin-sensitive/cefoxitin-resistant profile had a sensitivity of 88.7% and a specificity of 99.5% for the identification of mecC MRSA isolates. The presence of the mecC gene, determined by bacterial whole-genome sequencing, was used as the gold standard. This profile could provide a zero-cost screening method for identification of mecC-positive MRSA strains.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">8</span>;<span class="pagerange">2732-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23720794">23720794</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3719650">3719650</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.00847-13">10.1128/JCM.00847-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BamView: visualizing and interpretation of next-generation sequencing read alignments.</h4>
<p class="authors"><span class="author">Carver T</span>, <span class="author">Harris SR</span>, <span class="author">Otto TD</span>, <span class="author">Berriman M</span>, <span class="author">Parkhill J</span> and <span class="author">McQuillan JA</span></p>
<p class="affiliation">Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%72%74%65%6D%69%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#116;&#101;&#109;&#105;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">So-called next-generation sequencing (NGS) has provided the ability to sequence on a massive scale at low cost, enabling biologists to perform powerful experiments and gain insight into biological processes. BamView has been developed to visualize and analyse sequence reads from NGS platforms, which have been aligned to a reference sequence. It is a desktop application for browsing the aligned or mapped reads [Ruffalo, M, LaFramboise, T, Koyutürk, M. Comparative analysis of algorithms for next-generation sequencing read alignment. Bioinformatics 2011;27:2790-6] at different levels of magnification, from nucleotide level, where the base qualities can be seen, to genome or chromosome level where overall coverage is shown. To enable in-depth investigation of NGS data, various views are provided that can be configured to highlight interesting aspects of the data. Multiple read alignment files can be overlaid to compare results from different experiments, and filters can be applied to facilitate the interpretation of the aligned reads. As well as being a standalone application it can be used as an integrated part of the Artemis genome browser, BamView allows the user to study NGS data in the context of the sequence and annotation of the reference genome. Single nucleotide polymorphism (SNP) density and candidate SNP sites can be highlighted and investigated, and read-pair information can be used to discover large structural insertions and deletions. The application will also calculate simple analyses of the read mapping, including reporting the read counts and reads per kilobase per million mapped reads (RPKM) for genes selected by the user. Availability: BamView and Artemis are freely available software. These can be downloaded from their home pages: http://bamview.sourceforge.net/; http://www.sanger.ac.uk/resources/software/artemis/. Requirements: Java 1.6 or higher.</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">2</span>;<span class="pagerange">203-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22253280">22253280</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3603209">3603209</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbib%2Fbbr073">10.1093/bib/bbr073</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phosphoproteomics data classify hematological cancer cell lines according to tumor type and sensitivity to kinase inhibitors.</h4>
<p class="authors"><span class="author">Casado P</span>, <span class="author">Alcolea MP</span>, <span class="author">Iorio F</span>, <span class="author">Rodr&iacute;guez-Prados JC</span>, <span class="author">Vanhaesebroeck B</span>, <span class="author">Saez-Rodriguez J</span>, <span class="author">Joel S</span> and <span class="author">Cutillas PR</span></p>
<p class="affiliation">Analytical Signalling Group, Centre for Cell Signalling, Barts Cancer Institute, Queen Mary University of London, Charterhouse Square, London EC1B 6BQ, UK. <a href="mailto:%70%65%64%72%6F.%63%75%74%69%6C%6C%61%73%40%69%6D%70%65%72%69%61%6C.%61%63.%75%6B">&#112;&#101;&#100;&#114;&#111;&#46;&#99;&#117;&#116;&#105;&#108;&#108;&#97;&#115;&#64;&#105;&#109;&#112;&#101;&#114;&#105;&#97;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">BACKGROUND: Tumor classification based on their predicted responses to kinase inhibitors is a major goal for advancing targeted personalized therapies. Here, we used a phosphoproteomic approach to investigate biological heterogeneity across hematological cancer cell lines including acute myeloid leukemia, lymphoma, and multiple myeloma. RESULTS: Mass spectrometry was used to quantify 2,000 phosphorylation sites across three acute myeloid leukemia, three lymphoma, and three multiple myeloma cell lines in six biological replicates. The intensities of the phosphorylation sites grouped these cancer cell lines according to their tumor type. In addition, a phosphoproteomic analysis of seven acute myeloid leukemia cell lines revealed a battery of phosphorylation sites whose combined intensities correlated with the growth-inhibitory responses to three kinase inhibitors with remarkable correlation coefficients and fold changes (&gt; 100 between the most resistant and sensitive cells). Modeling based on regression analysis indicated that a subset of phosphorylation sites could be used to predict response to the tested drugs. Quantitative analysis of phosphorylation motifs indicated that resistant and sensitive cells differed in their patterns of kinase activities, but, interestingly, phosphorylations correlating with responses were not on members of the pathway being targeted; instead, these mainly were on parallel kinase pathways. CONCLUSION: This study reveals that the information on kinase activation encoded in phosphoproteomics data correlates remarkably well with the phenotypic responses of cancer cells to compounds that target kinase signaling and could be useful for the identification of novel markers of resistance or sensitivity to drugs that target the signaling network.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">4</span>;<span class="pagerange">R37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23628362">23628362</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-4-r37">10.1186/gb-2013-14-4-r37</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Persistence of HIV-1 Transmitted Drug Resistance Mutations.</h4>
<p class="authors"><span class="author">Castro H</span>, <span class="author">Pillay D</span>, <span class="author">Cane P</span>, <span class="author">Asboe D</span>, <span class="author">Cambiano V</span>, <span class="author">Phillips A</span>, <span class="author">Dunn DT</span> and <span class="author">UK Collaborative Group on HIV Drug Resistance</span></p>
<p class="affiliation">Medical Research Council Clinical Trials Unit, London, United Kingdom.</p>
<p class="abstract">There are few data on the persistence of individual human immunodeficiency virus type 1 (HIV-1) transmitted drug resistance (TDR) mutations in the absence of selective drug pressure. We studied 313 patients in whom TDR mutations were detected at their first resistance test and who had a subsequent test performed while ART-naive. The rate at which mutations became undetectable was estimated using exponential regression accounting for interval censoring. Most thymidine analogue mutations (TAMs) and T215 revertants (but not T215F/Y) were found to be highly stable, with NNRTI and PI mutations being relatively less persistent. Our estimates are important for informing HIV transmission models.</p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">208</span>;<span class="number">9</span>;<span class="pagerange">1459-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23904291">23904291</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3789571">3789571</a>; DOI: <a href="http://dx.doi.org/10.1093%2Finfdis%2Fjit345">10.1093/infdis/jit345</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comprehensive assignment of roles for salmonella typhimurium genes in intestinal colonization of food-producing animals.</h4>
<p class="authors"><span class="author">Chaudhuri RR</span>, <span class="author">Morgan E</span>, <span class="author">Peters SE</span>, <span class="author">Pleasance SJ</span>, <span class="author">Hudson DL</span>, <span class="author">Davies HM</span>, <span class="author">Wang J</span>, <span class="author">van Diemen PM</span>, <span class="author">Buckley AM</span>, <span class="author">Bowen AJ</span>, <span class="author">Pullinger GD</span>, <span class="author">Turner DJ</span>, <span class="author">Langridge GC</span>, <span class="author">Turner AK</span>, <span class="author">Parkhill J</span>, <span class="author">Charles IG</span>, <span class="author">Maskell DJ</span> and <span class="author">Stevens MP</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom.</p>
<p class="abstract">Chickens, pigs, and cattle are key reservoirs of Salmonella enterica, a foodborne pathogen of worldwide importance. Though a decade has elapsed since publication of the first Salmonella genome, thousands of genes remain of hypothetical or unknown function, and the basis of colonization of reservoir hosts is ill-defined. Moreover, previous surveys of the role of Salmonella genes in vivo have focused on systemic virulence in murine typhoid models, and the genetic basis of intestinal persistence and thus zoonotic transmission have received little study. We therefore screened pools of random insertion mutants of S. enterica serovar Typhimurium in chickens, pigs, and cattle by transposon-directed insertion-site sequencing (TraDIS). The identity and relative fitness in each host of 7,702 mutants was simultaneously assigned by massively parallel sequencing of transposon-flanking regions. Phenotypes were assigned to 2,715 different genes, providing a phenotype-genotype map of unprecedented resolution. The data are self-consistent in that multiple independent mutations in a given gene or pathway were observed to exert a similar fitness cost. Phenotypes were further validated by screening defined null mutants in chickens. Our data indicate that a core set of genes is required for infection of all three host species, and smaller sets of genes may mediate persistence in specific hosts. By assigning roles to thousands of Salmonella genes in key reservoir hosts, our data facilitate systems approaches to understand pathogenesis and the rational design of novel cross-protective vaccines and inhibitors. Moreover, by simultaneously assigning the genotype and phenotype of over 90% of mutants screened in complex pools, our data establish TraDIS as a powerful tool to apply rich functional annotation to microbial genomes with minimal animal use.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">4</span>;<span class="pagerange">e1003456</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23637626">23637626</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3630085">3630085</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003456">10.1371/journal.pgen.1003456</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mcph1-Deficient Mice Reveal a Role for MCPH1 in Otitis Media.</h4>
<p class="authors"><span class="author">Chen J</span>, <span class="author">Ingham N</span>, <span class="author">Clare S</span>, <span class="author">Raisen C</span>, <span class="author">Vancollie VE</span>, <span class="author">Ismail O</span>, <span class="author">McIntyre RE</span>, <span class="author">Tsang SH</span>, <span class="author">Mahajan VB</span>, <span class="author">Dougan G</span>, <span class="author">Adams DJ</span>, <span class="author">White JK</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom.</p>
<p class="abstract">Otitis media is a common reason for hearing loss, especially in children. Otitis media is a multifactorial disease and environmental factors, anatomic dysmorphology and genetic predisposition can all contribute to its pathogenesis. However, the reasons for the variable susceptibility to otitis media are elusive. MCPH1 mutations cause primary microcephaly in humans. So far, no hearing impairment has been reported either in the MCPH1 patients or mouse models with Mcph1 deficiency. In this study, Mcph1-deficient (Mcph1(tm1a) (/tm1a) ) mice were produced using embryonic stem cells with a targeted mutation by the Sanger Institute's Mouse Genetics Project. Auditory brainstem response measurements revealed that Mcph1(tm1a) (/tm1a) mice had mild to moderate hearing impairment with around 70% penetrance. We found otitis media with effusion in the hearing-impaired Mcph1(tm1a) (/tm1a) mice by anatomic and histological examinations. Expression of Mcph1 in the epithelial cells of middle ear cavities supported its involvement in the development of otitis media. Other defects of Mcph1(tm1a) (/tm1a) mice included small skull sizes, increased micronuclei in red blood cells, increased B cells and ocular abnormalities. These findings not only recapitulated the defects found in other Mcph1-deficient mice or MCPH1 patients, but also revealed an unexpected phenotype, otitis media with hearing impairment, which suggests Mcph1 is a new gene underlying genetic predisposition to otitis media.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e58156</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23516444">23516444</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3596415">3596415</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0058156">10.1371/journal.pone.0058156</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proteomic comparison of historic and recently emerged hypervirulent Clostridium difficile strains.</h4>
<p class="authors"><span class="author">Chen JW</span>, <span class="author">Scaria J</span>, <span class="author">Mao C</span>, <span class="author">Sobral B</span>, <span class="author">Zhang S</span>, <span class="author">Lawley T</span> and <span class="author">Chang YF</span></p>
<p class="affiliation">Department of Population Medicine and Diagnostic Sciences, Cornell University, Ithaca, New York 14853, United States.</p><p class="abstract">Clostridium difficile in recent years has undergone rapid evolution and has emerged as a serious human pathogen. Proteomic approaches can improve the understanding of the diversity of this important pathogen, especially in comparing the adaptive ability of different C. difficile strains. In this study, TMT labeling and nanoLC-MS/MS driven proteomics were used to investigate the responses of four C. difficile strains to nutrient shift and osmotic shock. We detected 126 and 67 differentially expressed proteins in at least one strain under nutrition shift and osmotic shock, respectively. During nutrient shift, several components of the phosphotransferase system (PTS) were found to be differentially expressed, which indicated that the carbon catabolite repression (CCR) was relieved to allow the expression of enzymes and transporters responsible for the utilization of alternate carbon sources. Some classical osmotic shock associated proteins, such as GroEL, RecA, CspG, and CspF, and other stress proteins such as PurG and SerA were detected during osmotic shock. Furthermore, the recently emerged strains were found to contain a more robust gene network in response to both stress conditions. This work represents the first comparative proteomic analysis of historic and recently emerged hypervirulent C. difficile strains, complementing the previously published proteomics studies utilizing only one reference strain.</p>
<p class="publication"><span class="title">Journal of proteome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">3</span>;<span class="pagerange">1151-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23298230">23298230</a>; DOI: <a href="http://dx.doi.org/10.1021%2Fpr3007528">10.1021/pr3007528</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hierarchical and spatially explicit clustering of DNA sequences with BAPS software.</h4>
<p class="authors"><span class="author">Cheng L</span>, <span class="author">Connor TR</span>, <span class="author">Sir&eacute;n J</span>, <span class="author">Aanensen DM</span> and <span class="author">Corander J</span></p>
<p class="affiliation">Department of Mathematics and statistics, University of Helsinki, 00014, Finland; Cardiff School of Biosciences, Cardiff University, Cardiff, CF10 3AX, UK; Department of Infectious Disease Epidemiology, Imperial College London, Norfolk Place, London, W2 1PG, UK; Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p>
<p class="abstract">Phylogeographical analyses have become commonplace for a myriad of organisms with the advent of cheap DNA sequencing technologies. Bayesian model-based clustering is a powerful tool for detecting important patterns in such data and can be used to decipher even quite subtle signals of systematic differences in molecular variation. Here we introduce two upgrades to the Bayesian Analysis of Population Structure (BAPS) software, which enable (1) spatially explicit modeling of variation in DNA sequences, and (2) hierarchical clustering of DNA sequence data to reveal nested genetic population structures. We provide a direct interface to map the results from spatial clustering with Google Maps using the portal http://www.spatialepidemiology.net/ and illustrate this approach using sequence data from Borrelia burgdorferii. The usefulness of hierarchical clustering is demonstrated through an analysis of the metapopulation structure within a bacterial population experiencing a high level of local horizontal gene transfer. The tools that are introduced are freely available at http://www.helsinki.fi/bsg/software/BAPS/.</p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23408797">23408797</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmst028">10.1093/molbev/mst028</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Your gut microbiota are what you eat.</h4>
<p class="authors"><span class="author">Chewapreecha C</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">1</span>;<span class="pagerange">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24336181">24336181</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3186">10.1038/nrmicro3186</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ISPD gene mutations are a common cause of congenital and limb-girdle muscular dystrophies.</h4>
<p class="authors"><span class="author">Cirak S</span>, <span class="author">Foley AR</span>, <span class="author">Herrmann R</span>, <span class="author">Willer T</span>, <span class="author">Yau S</span>, <span class="author">Stevens E</span>, <span class="author">Torelli S</span>, <span class="author">Brodd L</span>, <span class="author">Kamynina A</span>, <span class="author">Vondracek P</span>, <span class="author">Roper H</span>, <span class="author">Longman C</span>, <span class="author">Korinthenberg R</span>, <span class="author">Marrosu G</span>, <span class="author">N&uuml;rnberg P</span>, <span class="author">UK10K Consortium</span>, <span class="author">Michele DE</span>, <span class="author">Plagnol V</span>, <span class="author">Hurles M</span>, <span class="author">Moore SA</span>, <span class="author">Sewry CA</span>, <span class="author">Campbell KP</span>, <span class="author">Voit T</span> and <span class="author">Muntoni F</span></p>
<p class="affiliation">Dubowitz Neuromuscular Centre, UCL Institute of Child Health, University College London, 30 Guilford Street, London WC1N 1EH, UK. <a href="mailto:%66.%6D%75%6E%74%6F%6E%69%40%75%63%6C.%61%63.%75%6B">&#102;&#46;&#109;&#117;&#110;&#116;&#111;&#110;&#105;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Dystroglycanopathies are a clinically and genetically diverse group of recessively inherited conditions ranging from the most severe of the congenital muscular dystrophies, Walker-Warburg syndrome, to mild forms of adult-onset limb-girdle muscular dystrophy. Their hallmark is a reduction in the functional glycosylation of Î±-dystroglycan, which can be detected in muscle biopsies. An important part of this glycosylation is a unique O-mannosylation, essential for the interaction of Î±-dystroglycan with extracellular matrix proteins such as laminin-Î±2. Mutations in eight genes coding for proteins in the glycosylation pathway are responsible for âˆ¼50% of dystroglycanopathy cases. Despite multiple efforts using traditional positional cloning, the causative genes for unsolved dystroglycanopathy cases have escaped discovery for several years. In a recent collaborative study, we discovered that loss-of-function recessive mutations in a novel gene, called isoprenoid synthase domain containing (ISPD), are a relatively common cause of Walker-Warburg syndrome. In this article, we report the involvement of the ISPD gene in milder dystroglycanopathy phenotypes ranging from congenital muscular dystrophy to limb-girdle muscular dystrophy and identified allelic ISPD variants in nine cases belonging to seven families. In two ambulant cases, there was evidence of structural brain involvement, whereas in seven, the clinical manifestation was restricted to a dystrophic skeletal muscle phenotype. Although the function of ISPD in mammals is not yet known, mutations in this gene clearly lead to a reduction in the functional glycosylation of Î±-dystroglycan, which not only causes the severe Walker-Warburg syndrome but is also a common cause of the milder forms of dystroglycanopathy.</p>
<p class="publication"><span class="title">Brain : a journal of neurology</span>&nbsp;<span class="year">2013</span>;<span class="volume">136</span>;<span class="number">Pt 1</span>;<span class="pagerange">269-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23288328">23288328</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3562076">3562076</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbrain%2Faws312">10.1093/brain/aws312</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Elucidating emergence and transmission of multidrug-resistant tuberculosis in treatment experienced patients by whole genome sequencing.</h4>
<p class="authors"><span class="author">Clark TG</span>, <span class="author">Mallard K</span>, <span class="author">Coll F</span>, <span class="author">Preston M</span>, <span class="author">Assefa S</span>, <span class="author">Harris D</span>, <span class="author">Ogwang S</span>, <span class="author">Mumbowa F</span>, <span class="author">Kirenga B</span>, <span class="author">O&#39;Sullivan DM</span>, <span class="author">Okwera A</span>, <span class="author">Eisenach KD</span>, <span class="author">Joloba M</span>, <span class="author">Bentley SD</span>, <span class="author">Ellner JJ</span>, <span class="author">Parkhill J</span>, <span class="author">Jones-L&oacute;pez EC</span> and <span class="author">McNerney R</span></p>
<p class="affiliation">Faculty of Infectious and Tropical Diseases, London School of Hygiene & Tropical Medicine, London, United Kingdom ; Faculty of Epidemiology and Population Health, London School of Hygiene & Tropical Medicine, London, United Kingdom.</p>
<p class="abstract">Background: Understanding the emergence and spread of multidrug-resistant tuberculosis (MDR-TB) is crucial for its control. MDR-TB in previously treated patients is generally attributed to the selection of drug resistant mutants during inadequate therapy rather than transmission of a resistant strain. Traditional genotyping methods are not sufficient to distinguish strains in populations with a high burden of tuberculosis and it has previously been difficult to assess the degree of transmission in these settings. We have used whole genome analysis to investigate M. tuberculosis strains isolated from treatment experienced patients with MDR-TB in Uganda over a period of four years.

We used high throughput genome sequencing technology to investigate small polymorphisms and large deletions in 51 Mycobacterium tuberculosis samples from 41 treatment-experienced TB patients attending a TB referral and treatment clinic in Kampala. This was a convenience sample representing 69% of MDR-TB cases identified over the four year period. Low polymorphism was observed in longitudinal samples from individual patients (2-15 SNPs). Clusters of samples with less than 50 SNPs variation were examined. Three clusters comprising a total of 8 patients were found with almost identical genetic profiles, including mutations predictive for resistance to rifampicin and isoniazid, suggesting transmission of MDR-TB. Two patients with previous drug susceptible disease were found to have acquired MDR strains, one of which shared its genotype with an isolate from another patient in the cohort.

Conclusions: Whole genome sequence analysis identified MDR-TB strains that were shared by more than one patient. The transmission of multidrug-resistant disease in this cohort of retreatment patients emphasises the importance of early detection and need for infection control. Consideration should be given to rapid testing for drug resistance in patients undergoing treatment to monitor the emergence of resistance and permit early intervention to avoid onward transmission.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">e83012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24349420">24349420</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0083012">10.1371/journal.pone.0083012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome of Acanthamoeba castellanii highlights extensive lateral gene transfer and early evolution of tyrosine kinase signaling.</h4>
<p class="authors"><span class="author">Clarke M</span>, <span class="author">Lohan AJ</span>, <span class="author">Liu B</span>, <span class="author">Lagkouvardos I</span>, <span class="author">Roy S</span>, <span class="author">Zafar N</span>, <span class="author">Bertelli C</span>, <span class="author">Schilde C</span>, <span class="author">Kianianmomeni A</span>, <span class="author">B&uuml;rglin TR</span>, <span class="author">Frech C</span>, <span class="author">Turcotte B</span>, <span class="author">Kopec KO</span>, <span class="author">Synnott JM</span>, <span class="author">Choo C</span>, <span class="author">Paponov I</span>, <span class="author">Finkler A</span>, <span class="author">Heng Tan CS</span>, <span class="author">Hutchins AP</span>, <span class="author">Weinmeier T</span>, <span class="author">Rattei T</span>, <span class="author">Chu JS</span>, <span class="author">Gimenez G</span>, <span class="author">Irimia M</span>, <span class="author">Rigden DJ</span>, <span class="author">Fitzpatrick DA</span>, <span class="author">Lorenzo-Morales J</span>, <span class="author">Bateman A</span>, <span class="author">Chiu CH</span>, <span class="author">Tang P</span>, <span class="author">Hegemann P</span>, <span class="author">Fromm H</span>, <span class="author">Raoult D</span>, <span class="author">Greub G</span>, <span class="author">Miranda-Saavedra D</span>, <span class="author">Chen N</span>, <span class="author">Nash P</span>, <span class="author">Ginger ML</span>, <span class="author">Horn M</span>, <span class="author">Schaap P</span>, <span class="author">Caler L</span> and <span class="author">Loftus BJ</span></p>
<p class="affiliation">Conway Institute, University College Dublin, Belfield, Dublin 4, Ireland. <a href="mailto:%62%72%65%6E%64%61%6E.%6C%6F%66%74%75%73%40%75%63%64.%69%65">&#98;&#114;&#101;&#110;&#100;&#97;&#110;&#46;&#108;&#111;&#102;&#116;&#117;&#115;&#64;&#117;&#99;&#100;&#46;&#105;&#101;</a>.</p>
<p class="abstract">BACKGROUND: The Amoebozoa constitute one of the primary divisions of eukaryotes, encompassing taxa of both biomedical and evolutionary importance, yet its genomic diversity remains largely unsampled. Here we present an analysis of a whole genome assembly of Acanthamoeba castellanii (Ac) the first representative from a solitary free-living amoebozoan. RESULTS: Ac encodes 15,455 compact intron-rich genes, a significant number of which are predicted to have arisen through inter-kingdom lateral gene transfer (LGT). A majority of the LGT candidates have undergone a substantial degree of intronization and Ac appears to have incorporated them into established transcriptional programs. Ac manifests a complex signaling and cell communication repertoire, including a complete tyrosine kinase signaling toolkit and a comparable diversity of predicted extracellular receptors to that found in the facultatively multicellular dictyostelids. An important environmental host of a diverse range of bacteria and viruses, Ac utilizes a diverse repertoire of predicted pattern recognition receptors, many with predicted orthologous functions in the innate immune systems of higher organisms. CONCLUSIONS: Our analysis highlights the important role of LGT in the biology of Ac and in the diversification of microbial eukaryotes. The early evolution of a key signaling facility implicated in the evolution of metazoan multicellularity strongly argues for its emergence early in the Unikont lineage. Overall, the availability of an Ac genome should aid in deciphering the biology of the Amoebozoa and facilitate functional genomic studies in this important model organism and environmental host.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">2</span>;<span class="pagerange">R11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23375108">23375108</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-2-r11">10.1186/gb-2013-14-2-r11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of seven loci affecting mean telomere length and their association with disease.</h4>
<p class="authors"><span class="author">Codd V</span>, <span class="author">Nelson CP</span>, <span class="author">Albrecht E</span>, <span class="author">Mangino M</span>, <span class="author">Deelen J</span>, <span class="author">Buxton JL</span>, <span class="author">Hottenga JJ</span>, <span class="author">Fischer K</span>, <span class="author">Esko T</span>, <span class="author">Surakka I</span>, <span class="author">Broer L</span>, <span class="author">Nyholt DR</span>, <span class="author">Mateo Leach I</span>, <span class="author">Salo P</span>, <span class="author">H&auml;gg S</span>, <span class="author">Matthews MK</span>, <span class="author">Palmen J</span>, <span class="author">Norata GD</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Saleheen D</span>, <span class="author">Amin N</span>, <span class="author">Balmforth AJ</span>, <span class="author">Beekman M</span>, <span class="author">de Boer RA</span>, <span class="author">B&ouml;hringer S</span>, <span class="author">Braund PS</span>, <span class="author">Burton PR</span>, <span class="author">de Craen AJ</span>, <span class="author">Denniff M</span>, <span class="author">Dong Y</span>, <span class="author">Douroudis K</span>, <span class="author">Dubinina E</span>, <span class="author">Eriksson JG</span>, <span class="author">Garlaschelli K</span>, <span class="author">Guo D</span>, <span class="author">Hartikainen AL</span>, <span class="author">Henders AK</span>, <span class="author">Houwing-Duistermaat JJ</span>, <span class="author">Kananen L</span>, <span class="author">Karssen LC</span>, <span class="author">Kettunen J</span>, <span class="author">Klopp N</span>, <span class="author">Lagou V</span>, <span class="author">van Leeuwen EM</span>, <span class="author">Madden PA</span>, <span class="author">M&auml;gi R</span>, <span class="author">Magnusson PK</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">McCarthy MI</span>, <span class="author">Medland SE</span>, <span class="author">Mihailov E</span>, <span class="author">Montgomery GW</span>, <span class="author">Oostra BA</span>, <span class="author">Palotie A</span>, <span class="author">Peters A</span>, <span class="author">Pollard H</span>, <span class="author">Pouta A</span>, <span class="author">Prokopenko I</span>, <span class="author">Ripatti S</span>, <span class="author">Salomaa V</span>, <span class="author">Suchiman HE</span>, <span class="author">Valdes AM</span>, <span class="author">Verweij N</span>, <span class="author">Vi&ntilde;uela A</span>, <span class="author">Wang X</span>, <span class="author">Wichmann HE</span>, <span class="author">Widen E</span>, <span class="author">Willemsen G</span>, <span class="author">Wright MJ</span>, <span class="author">Xia K</span>, <span class="author">Xiao X</span>, <span class="author">van Veldhuisen DJ</span>, <span class="author">Catapano AL</span>, <span class="author">Tobin MD</span>, <span class="author">Hall AS</span>, <span class="author">Blakemore AI</span>, <span class="author">van Gilst WH</span>, <span class="author">Zhu H</span>, <span class="author">Consortium C</span>, <span class="author">Erdmann J</span>, <span class="author">Reilly MP</span>, <span class="author">Kathiresan S</span>, <span class="author">Schunkert H</span>, <span class="author">Talmud PJ</span>, <span class="author">Pedersen NL</span>, <span class="author">Perola M</span>, <span class="author">Ouwehand W</span>, <span class="author">Kaprio J</span>, <span class="author">Martin NG</span>, <span class="author">van Duijn CM</span>, <span class="author">Hovatta I</span>, <span class="author">Gieger C</span>, <span class="author">Metspalu A</span>, <span class="author">Boomsma DI</span>, <span class="author">Jarvelin MR</span>, <span class="author">Slagboom PE</span>, <span class="author">Thompson JR</span>, <span class="author">Spector TD</span>, <span class="author">van der Harst P</span> and <span class="author">Samani NJ</span></p>
<p class="affiliation">Department of Cardiovascular Sciences, University of Leicester, Leicester, UK.</p><p class="abstract">Interindividual variation in mean leukocyte telomere length (LTL) is associated with cancer and several age-associated diseases. We report here a genome-wide meta-analysis of 37,684 individuals with replication of selected variants in an additional 10,739 individuals. We identified seven loci, including five new loci, associated with mean LTL (P &lt; 5 &times; 10(-8)). Five of the loci contain candidate genes (TERC, TERT, NAF1, OBFC1 and RTEL1) that are known to be involved in telomere biology. Lead SNPs at two loci (TERC and TERT) associate with several cancers and other diseases, including idiopathic pulmonary fibrosis. Moreover, a genetic risk score analysis combining lead variants at all 7 loci in 22,233 coronary artery disease cases and 64,762 controls showed an association of the alleles associated with shorter LTL with increased risk of coronary artery disease (21% (95% confidence interval, 5-35%) per standard deviation in LTL, P = 0.014). Our findings support a causal role of telomere-length variation in some age-related diseases.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/08/014/24067</span>; Medical Research Council: <span class="grant">G0902313</span>; NIDA NIH HHS: <span class="grant">R56 DA012854</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">4</span>;<span class="pagerange">422-7, 427e1-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23535734">23535734</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4006270">4006270</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2528">10.1038/ng.2528</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Real-time genomic epidemiological evaluation of human campylobacter isolates by use of whole-genome multilocus sequence typing.</h4>
<p class="authors"><span class="author">Cody AJ</span>, <span class="author">McCarthy ND</span>, <span class="author">Jansen van Rensburg M</span>, <span class="author">Isinkaye T</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Dingle KE</span>, <span class="author">Bowler IC</span>, <span class="author">Jolley KA</span> and <span class="author">Maiden MC</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">Sequence-based typing is essential for understanding the epidemiology of Campylobacter infections, a major worldwide cause of bacterial gastroenteritis. We demonstrate the practical and rapid exploitation of whole-genome sequencing to provide routine definitive characterization of Campylobacter jejuni and Campylobacter coli for clinical and public health purposes. Short-read data from 384 Campylobacter clinical isolates collected over 4 months in Oxford, United Kingdom, were assembled de novo. Contigs were deposited at the pubMLST.org/campylobacter website and automatically annotated for 1,667 loci. Typing and phylogenetic information was extracted and comparative analyses were performed for various subsets of loci, up to the level of the whole genome, using the Genome Comparator and Neighbor-net algorithms. The assembled sequences (for 379 isolates) were diverse and resembled collections from previous studies of human campylobacteriosis. Small subsets of very closely related isolates originated mainly from repeated sampling from the same patients and, in one case, likely laboratory contamination. Much of the within-patient variation occurred in phase-variable genes. Clinically and epidemiologically informative data can be extracted from whole-genome sequence data in real time with straightforward, publicly available tools. These analyses are highly scalable, are transparent, do not require closely related genome reference sequences, and provide improved resolution (i) among Campylobacter clonal complexes and (ii) between very closely related isolates. Additionally, these analyses rapidly differentiated unrelated isolates, allowing the detection of single-strain clusters. The approach is widely applicable to analyses of human bacterial pathogens in real time in clinical laboratories, with little specialist training required.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">8</span>;<span class="pagerange">2526-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23698529">23698529</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3719633">3719633</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.00066-13">10.1128/JCM.00066-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genetic study of Wilson's disease in the United Kingdom.</h4>
<p class="authors"><span class="author">Coffey AJ</span>, <span class="author">Durkie M</span>, <span class="author">Hague S</span>, <span class="author">McLay K</span>, <span class="author">Emmerson J</span>, <span class="author">Lo C</span>, <span class="author">Klaffke S</span>, <span class="author">Joyce CJ</span>, <span class="author">Dhawan A</span>, <span class="author">Hadzic N</span>, <span class="author">Mieli-Vergani G</span>, <span class="author">Kirk R</span>, <span class="author">Elizabeth Allen K</span>, <span class="author">Nicholl D</span>, <span class="author">Wong S</span>, <span class="author">Griffiths W</span>, <span class="author">Smithson S</span>, <span class="author">Giffin N</span>, <span class="author">Taha A</span>, <span class="author">Connolly S</span>, <span class="author">Gillett GT</span>, <span class="author">Tanner S</span>, <span class="author">Bonham J</span>, <span class="author">Sharrack B</span>, <span class="author">Palotie A</span>, <span class="author">Rattray M</span>, <span class="author">Dalton A</span> and <span class="author">Bandmann O</span></p>
<p class="affiliation">1 Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK.</p>
<p class="abstract">Previous studies have failed to identify mutations in the Wilson's disease gene ATP7B in a significant number of clinically diagnosed cases. This has led to concerns about genetic heterogeneity for this condition but also suggested the presence of unusual mutational mechanisms. We now present our findings in 181 patients from the United Kingdom with clinically and biochemically confirmed Wilson's disease. A total of 116 different ATP7B mutations were detected, 32 of which are novel. The overall mutation detection frequency was 98%. The likelihood of mutations in genes other than ATP7B causing a Wilson's disease phenotype is therefore very low. We report the first cases with Wilson's disease due to segmental uniparental isodisomy as well as three patients with three ATP7B mutations and three families with Wilson's disease in two consecutive generations. We determined the genetic prevalence of Wilson's disease in the United Kingdom by sequencing the entire coding region and adjacent splice sites of ATP7B in 1000 control subjects. The frequency of all single nucleotide variants with in silico evidence of pathogenicity (Class 1 variant) was 0.056 or 0.040 if only those single nucleotide variants that had previously been reported as mutations in patients with Wilson's disease were included in the analysis (Class 2 variant). The frequency of heterozygote, putative or definite disease-associated ATP7B mutations was therefore considerably higher than the previously reported occurrence of 1:90 (or 0.011) for heterozygote ATP7B mutation carriers in the general population (P < 2.2 × 10(-16) for Class 1 variants or P < 5 × 10(-11) for Class 2 variants only). Subsequent exclusion of four Class 2 variants without additional in silico evidence of pathogenicity led to a further reduction of the mutation frequency to 0.024. Using this most conservative approach, the calculated frequency of individuals predicted to carry two mutant pathogenic ATP7B alleles is 1:7026 and thus still considerably higher than the typically reported prevalence of Wilson's disease of 1:30 000 (P = 0.00093). Our study provides strong evidence for monogenic inheritance of Wilson's disease. It also has major implications for ATP7B analysis in clinical practice, namely the need to consider unusual genetic mechanisms such as uniparental disomy or the possible presence of three ATP7B mutations. The marked discrepancy between the genetic prevalence and the number of clinically diagnosed cases of Wilson's disease may be due to both reduced penetrance of ATP7B mutations and failure to diagnose patients with this eminently treatable disorder.</p>
<p class="publication"><span class="title">Brain : a journal of neurology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23518715">23518715</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbrain%2Fawt035">10.1093/brain/awt035</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two Pfam protein families characterized by a crystal structure of protein lpg2210 from Legionella pneumophila.</h4>
<p class="authors"><span class="author">Coggill P</span>, <span class="author">Eberhardt RY</span>, <span class="author">Finn RD</span>, <span class="author">Chang Y</span>, <span class="author">Jaroszewski L</span>, <span class="author">Godzik A</span>, <span class="author">Das D</span>, <span class="author">Xu Q</span>, <span class="author">Axelrod HL</span>, <span class="author">Aravind L</span>, <span class="author">Murzin AG</span> and <span class="author">Bateman A</span></p>
<p class="abstract">Background: Every genome contains a large number of uncharacterized proteins that may encode entirely novel biological systems. Many of these uncharacterized proteins fall into related sequence families. By applying sequence and structural analysis we hope to provide insight into novel biology.

Results: We analyze a previously uncharacterized Pfam protein family called DUF4424 [Pfam:PF14415]. The recently solved three-dimensional structure of the protein lpg2210 from Legionella pneumophila provides the first structural information pertaining to this family. This protein additionally includes the first representative structure of another Pfam family called the YARHG domain [Pfam:PF13308]. The Pfam family DUF4424 adopts a 19-stranded beta-sandwich fold that shows similarity to the N-terminal domain of leukotriene A-4 hydrolase. The YARHG domain forms an all-helical domain at the C-terminus. Structure analysis allows us to recognize distant similarities between the DUF4424 domain and individual domains of M1 aminopeptidases and tricorn proteases, which form massive proteasome-like capsids in both archaea and bacteria.

Conclusions: Based on our analyses we hypothesize that the DUF4424 domain may have a role in forming large, multi-component enzyme complexes. We suggest that the YARGH domain may play a role in binding a moiety in proximity with peptidoglycan, such as a hydrophobic outer membrane lipid or lipopolysaccharide.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">265</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24004689">24004689</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-14-265">10.1186/1471-2105-14-265</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Toward knowledge support for analysis and interpretation of complex traits.</h4>
<p class="authors"><span class="author">Collier N</span>, <span class="author">Oellrich A</span> and <span class="author">Groza T</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%63%6F%6C%6C%69%65%72%40%65%62%69.%61%63.%75%6B">&#99;&#111;&#108;&#108;&#105;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">The systematic description of complex traits, from the organism to the cellular level, is important for hypothesis generation about underlying disease mechanisms. We discuss how intelligent algorithms might provide support, leading to faster throughput.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">9</span>;<span class="pagerange">214</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24079802">24079802</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-9-214">10.1186/gb-2013-14-9-214</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Learning to Recognize Phenotype Candidates in the Auto-Immune Literature Using SVM Re-Ranking.</h4>
<p class="authors"><span class="author">Collier N</span>, <span class="author">Tran MV</span>, <span class="author">Le HQ</span>, <span class="author">Ha QT</span>, <span class="author">Oellrich A</span> and <span class="author">Rebholz-Schuhmann D</span></p>
<p class="affiliation">European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Cambridge, United Kingdom ; National Institute of Informatics, Tokyo, Japan.</p>
<p class="abstract">The identification of phenotype descriptions in the scientific literature, case reports and patient records is a rewarding task for bio-medical text mining. Any progress will support knowledge discovery and linkage to other resources. However because of their wide variation a number of challenges still remain in terms of their identification and semantic normalisation before they can be fully exploited for research purposes. This paper presents novel techniques for identifying potential complex phenotype mentions by exploiting a hybrid model based on machine learning, rules and dictionary matching. A systematic study is made of how to combine sequence labels from these modules as well as the merits of various ontological resources. We evaluated our approach on a subset of Medline abstracts cited by the Online Mendelian Inheritance of Man database related to auto-immune diseases. Using partial matching the best micro-averaged F-score for phenotypes and five other entity classes was 79.9%. A best performance of 75.3% was achieved for phenotype candidates using all semantics resources. We observed the advantage of using SVM-based learn-to-rank for sequence label combination over maximum entropy and a priority list approach. The results indicate that the identification of simple entity types such as chemicals and genes are robustly supported by single semantic resources, whereas phenotypes require combinations. Altogether we conclude that our approach coped well with the compositional structure of phenotypes in the auto-immune domain.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e72965</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24155869">24155869</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3796529">3796529</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0072965">10.1371/journal.pone.0072965</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and proteomic dissection of the ubiquitous plant pathogen, Armillaria mellea: toward a new infection model system.</h4>
<p class="authors"><span class="author">Collins C</span>, <span class="author">Keane TM</span>, <span class="author">Turner DJ</span>, <span class="author">O&#39;Keeffe G</span>, <span class="author">Fitzpatrick DA</span> and <span class="author">Doyle S</span></p>
<p class="affiliation">Department of Biology, National University of Ireland Maynooth, Maynooth, Co Kildare, Ireland.</p><p class="abstract">Armillaria mellea is a major plant pathogen. Yet, no large-scale &quot;-omics&quot; data are available to enable new studies, and limited experimental models are available to investigate basidiomycete pathogenicity. Here we reveal that the A. mellea genome comprises 58.35 Mb, contains 14473 gene models, of average length 1575 bp (4.72 introns/gene). Tandem mass spectrometry identified 921 mycelial (n = 629 unique) and secreted (n = 183 unique) proteins. Almost 100 mycelial proteins were either species-specific or previously unidentified at the protein level. A number of proteins (n = 111) was detected in both mycelia and culture supernatant extracts. Signal sequence occurrence was 4-fold greater for secreted (50.2%) compared to mycelial (12%) proteins. Analyses revealed a rich reservoir of carbohydrate degrading enzymes, laccases, and lignin peroxidases in the A. mellea proteome, reminiscent of both basidiomycete and ascomycete glycodegradative arsenals. We discovered that A. mellea exhibits a specific killing effect against Candida albicans during coculture. Proteomic investigation of this interaction revealed the unique expression of defensive and potentially offensive A. mellea proteins (n = 30). Overall, our data reveal new insights into the origin of basidiomycete virulence and we present a new model system for further studies aimed at deciphering fungal pathogenic mechanisms.</p>
<p class="publication"><span class="title">Journal of proteome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">2552-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23656496">23656496</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3679558">3679558</a>; DOI: <a href="http://dx.doi.org/10.1021%2Fpr301131t">10.1021/pr301131t</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Small effective population size and genetic homogeneity in the Val Borbera isolate.</h4>
<p class="authors"><span class="author">Colonna V</span>, <span class="author">Pistis G</span>, <span class="author">Bomba L</span>, <span class="author">Mona S</span>, <span class="author">Matullo G</span>, <span class="author">Boano R</span>, <span class="author">Sala C</span>, <span class="author">Vigan&ograve; F</span>, <span class="author">Torroni A</span>, <span class="author">Achilli A</span>, <span class="author">Hooshiar Kashani B</span>, <span class="author">Malerba G</span>, <span class="author">Gambaro G</span>, <span class="author">Soranzo N</span> and <span class="author">Toniolo D</span></p>
<p class="affiliation">Institute of Genetics and Biophysics &#39;A. Buzzati-Traverso&#39;, National Research Council (CNR), Naples, Italy. <a href="mailto:%76%69%6E%63%65%6E%7A%61.%63%6F%6C%6F%6E%6E%61%40%69%67%62.%63%6E%72.%69%74">&#118;&#105;&#110;&#99;&#101;&#110;&#122;&#97;&#46;&#99;&#111;&#108;&#111;&#110;&#110;&#97;&#64;&#105;&#103;&#98;&#46;&#99;&#110;&#114;&#46;&#105;&#116;</a></p><p class="abstract">Population isolates are a valuable resource for medical genetics because of their reduced genetic, phenotypic and environmental heterogeneity. Further, extended linkage disequilibrium (LD) allows accurate haplotyping and imputation. In this study, we use nuclear and mitochondrial DNA data to determine to what extent the geographically isolated population of the Val Borbera valley also presents features of genetic isolation. We performed a comparative analysis of population structure and estimated effective population size exploiting LD data. We also evaluated haplotype sharing through the analysis of segments of autozygosity. Our findings reveal that the valley has features characteristic of a genetic isolate, including reduced genetic heterogeneity and reduced effective population size. We show that this population has been subject to prolonged genetic drift and thus we expect many variants that are rare in the general population to reach significant frequency values in the valley, making this population suitable for the identification of rare variants underlying complex traits.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2013</span>;<span class="volume">21</span>;<span class="number">1</span>;<span class="pagerange">89-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22713810">22713810</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3522197">3522197</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2012.113">10.1038/ejhg.2012.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Out-of-Africa migration and Neolithic coexpansion of Mycobacterium tuberculosis with modern humans.</h4>
<p class="authors"><span class="author">Comas I</span>, <span class="author">Coscolla M</span>, <span class="author">Luo T</span>, <span class="author">Borrell S</span>, <span class="author">Holt KE</span>, <span class="author">Kato-Maeda M</span>, <span class="author">Parkhill J</span>, <span class="author">Malla B</span>, <span class="author">Berg S</span>, <span class="author">Thwaites G</span>, <span class="author">Yeboah-Manu D</span>, <span class="author">Bothamley G</span>, <span class="author">Mei J</span>, <span class="author">Wei L</span>, <span class="author">Bentley S</span>, <span class="author">Harris SR</span>, <span class="author">Niemann S</span>, <span class="author">Diel R</span>, <span class="author">Aseffa A</span>, <span class="author">Gao Q</span>, <span class="author">Young D</span> and <span class="author">Gagneux S</span></p>
<p class="affiliation">1] Genomics and Health Unit, Centre for Public Health Research (CSISP-FISABIO), Valencia, Spain. [2] CIBER (Centros de Investigaci&oacute;n Biom&eacute;dica en Red) in Epidemiology and Public Health, Barcelona, Spain.</p><p class="abstract">Tuberculosis caused 20% of all human deaths in the Western world between the seventeenth and nineteenth centuries and remains a cause of high mortality in developing countries. In analogy to other crowd diseases, the origin of human tuberculosis has been associated with the Neolithic Demographic Transition, but recent studies point to a much earlier origin. We analyzed the whole genomes of 259 M. tuberculosis complex (MTBC) strains and used this data set to characterize global diversity and to reconstruct the evolutionary history of this pathogen. Coalescent analyses indicate that MTBC emerged about 70,000 years ago, accompanied migrations of anatomically modern humans out of Africa and expanded as a consequence of increases in human population density during the Neolithic period. This long coevolutionary history is consistent with MTBC displaying characteristics indicative of adaptation to both low and high host densities.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U117581288</span>, <span class="grant">MC_U117588500</span>, <span class="grant">U.1175.02.002.00015.01</span>, <span class="grant">U117581288</span>; NIAID NIH HHS: <span class="grant">AI090928 AND</span>, <span class="grant">R01 AI090928</span>; PHS HHS: <span class="grant">HHSN266200700022C</span>; Wellcome Trust: <span class="grant">089276</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">10</span>;<span class="pagerange">1176-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23995134">23995134</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3800747">3800747</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2744">10.1038/ng.2744</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenetic regulation of COL15A1 in smooth muscle cell replicative aging and atherosclerosis.</h4>
<p class="authors"><span class="author">Connelly JJ</span>, <span class="author">Cherepanova OA</span>, <span class="author">Doss JF</span>, <span class="author">Karaoli T</span>, <span class="author">Lillard TS</span>, <span class="author">Markunas CA</span>, <span class="author">Nelson S</span>, <span class="author">Wang T</span>, <span class="author">Ellis PD</span>, <span class="author">Langford CF</span>, <span class="author">Haynes C</span>, <span class="author">Seo DM</span>, <span class="author">Goldschmidt-Clermont PJ</span>, <span class="author">Shah SH</span>, <span class="author">Kraus WE</span>, <span class="author">Hauser ER</span> and <span class="author">Gregory SG</span></p>
<p class="affiliation">Department of Medicine and Division of Cardiovascular Medicine and.</p>
<p class="abstract">Smooth muscle cell (SMC) proliferation is a hallmark of vascular injury and disease. Global hypomethylation occurs during SMC proliferation in culture and in vivo during neointimal formation. Regardless of the programmed or stochastic nature of hypomethylation, identifying these changes is important in understanding vascular disease, as maintenance of a cells' epigenetic profile is essential for maintaining cellular phenotype. Global hypomethylation of proliferating aortic SMCs and concomitant decrease of DNMT1 expression were identified in culture during passage. An epigenome screen identified regions of the genome that were hypomethylated during proliferation and a region containing Collagen, type XV, alpha 1 (COL15A1) was selected by 'genomic convergence' for characterization. COL15A1 transcript and protein levels increased with passage-dependent decreases in DNA methylation and the transcript was sensitive to treatment with 5-Aza-2'-deoxycytidine, suggesting DNA methylation-mediated gene expression. Phenotypically, knockdown of COL15A1 increased SMC migration and decreased proliferation and Col15a1 expression was induced in an atherosclerotic lesion and localized to the atherosclerotic cap. A sequence variant in COL15A1 that is significantly associated with atherosclerosis (rs4142986, P = 0.017, OR = 1.434) was methylated and methylation of the risk allele correlated with decreased gene expression and increased atherosclerosis in human aorta. In summary, hypomethylation of COL15A1 occurs during SMC proliferation and the consequent increased gene expression may impact SMC phenotype and atherosclerosis formation. Hypomethylated genes, such as COL15A1, provide evidence for concomitant epigenetic regulation and genetic susceptibility, and define a class of causal targets that sit at the intersection of genetic and epigenetic predisposition in the etiology of complex disease.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">25</span>;<span class="pagerange">5107-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23912340">23912340</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3842173">3842173</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddt365">10.1093/hmg/ddt365</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detailed molecular characterisation of acute myeloid leukaemia with a normal karyotype using targeted DNA capture.</h4>
<p class="authors"><span class="author">Conte N</span>, <span class="author">Varela I</span>, <span class="author">Grove C</span>, <span class="author">Manes N</span>, <span class="author">Yusa K</span>, <span class="author">Moreno T</span>, <span class="author">Segonds-Pichon A</span>, <span class="author">Bench A</span>, <span class="author">Gudgin E</span>, <span class="author">Herman B</span>, <span class="author">Bolli N</span>, <span class="author">Ellis P</span>, <span class="author">Haddad D</span>, <span class="author">Costeas P</span>, <span class="author">Rad R</span>, <span class="author">Scott M</span>, <span class="author">Huntly B</span>, <span class="author">Bradley A</span> and <span class="author">Vassiliou GS</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Advances in sequencing technologies are giving unprecedented insights into the spectrum of somatic mutations underlying acute myeloid leukaemia with a normal karyotype (AML-NK). It is clear that the prognosis of individual patients is strongly influenced by the combination of mutations in their leukaemia and that many leukaemias are composed of multiple subclones, with differential susceptibilities to treatment. Here, we describe a method, employing targeted capture coupled with next-generation sequencing and tailored bioinformatic analysis, for the simultaneous study of 24 genes recurrently mutated in AML-NK. Mutational analysis was performed using open source software and an in-house script (Mutation Identification and Analysis Software), which identified dominant clone mutations with 100% specificity. In each of seven cases of AML-NK studied, we identified and verified mutations in 2-4 genes in the main leukaemic clone. Additionally, high sequencing depth enabled us to identify putative subclonal mutations and detect leukaemia-specific mutations in DNA from remission marrow. Finally, we used normalised read depths to detect copy number changes and identified and subsequently verified a tandem duplication of exons 2-9 of MLL and at least one deletion involving PTEN. This methodology reliably detects sequence and copy number mutations, and can thus greatly facilitate the classification, clinical research, diagnosis and management of AML-NK.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079249</span>, <span class="grant">095663</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">Leukemia</span>&nbsp;<span class="year">2013</span>;<span class="volume">27</span>;<span class="number">9</span>;<span class="pagerange">1820-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23702683">23702683</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3768109">3768109</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fleu.2013.117">10.1038/leu.2013.117</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Where genotype is not predictive of phenotype: towards an understanding of the molecular basis of reduced penetrance in human inherited disease.</h4>
<p class="authors"><span class="author">Cooper DN</span>, <span class="author">Krawczak M</span>, <span class="author">Polychronakos C</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Kehrer-Sawatzki H</span></p>
<p class="affiliation">Institute of Medical Genetics, School of Medicine, Cardiff University, Heath Park, Cardiff, CF14 4XN, UK, <a href="mailto:%63%6F%6F%70%65%72%44%4E%40%63%61%72%64%69%66%66.%61%63.%75%6B">&#99;&#111;&#111;&#112;&#101;&#114;&#68;&#78;&#64;&#99;&#97;&#114;&#100;&#105;&#102;&#102;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">Some individuals with a particular disease-causing mutation or genotype fail to express most if not all features of the disease in question, a phenomenon that is known as &#39;reduced (or incomplete) penetrance&#39;. Reduced penetrance is not uncommon; indeed, there are many known examples of &#39;disease-causing mutations&#39; that fail to cause disease in at least a proportion of the individuals who carry them. Reduced penetrance may therefore explain not only why genetic diseases are occasionally transmitted through unaffected parents, but also why healthy individuals can harbour quite large numbers of potentially disadvantageous variants in their genomes without suffering any obvious ill effects. Reduced penetrance can be a function of the specific mutation(s) involved or of allele dosage. It may also result from differential allelic expression, copy number variation or the modulating influence of additional genetic variants in cis or in trans. The penetrance of some pathogenic genotypes is known to be age- and/or sex-dependent. Variable penetrance may also reflect the action of unlinked modifier genes, epigenetic changes or environmental factors. At least in some cases, complete penetrance appears to require the presence of one or more genetic variants at other loci. In this review, we summarize the evidence for reduced penetrance being a widespread phenomenon in human genetics and explore some of the molecular mechanisms that may help to explain this enigmatic characteristic of human inherited disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">132</span>;<span class="number">10</span>;<span class="pagerange">1077-130</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23820649">23820649</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3778950">3778950</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-013-1331-2">10.1007/s00439-013-1331-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel Mycobacterium tuberculosis complex isolate from a wild chimpanzee.</h4>
<p class="authors"><span class="author">Coscolla M</span>, <span class="author">Lewin A</span>, <span class="author">Metzger S</span>, <span class="author">Maetz-Rennsing K</span>, <span class="author">Calvignac-Spencer S</span>, <span class="author">Nitsche A</span>, <span class="author">Dabrowski PW</span>, <span class="author">Radonic A</span>, <span class="author">Niemann S</span>, <span class="author">Parkhill J</span>, <span class="author">Couacy-Hymann E</span>, <span class="author">Feldman J</span>, <span class="author">Comas I</span>, <span class="author">Boesch C</span>, <span class="author">Gagneux S</span> and <span class="author">Leendertz FH</span></p>
<p class="affiliation">Swiss Tropical and Public Health Institute, Basel, Switzerland.</p><p class="abstract">Tuberculosis (TB) is caused by gram-positive bacteria known as the Mycobacterium tuberculosis complex (MTBC). MTBC include several human-associated lineages and several variants adapted to domestic and, more rarely, wild animal species. We report an M. tuberculosis strain isolated from a wild chimpanzee in C&ocirc;te d&#39;Ivoire that was shown by comparative genomic and phylogenomic analyses to belong to a new lineage of MTBC, closer to the human-associated lineage 6 (also known as M. africanum West Africa 2) than to the other classical animal-associated MTBC strains. These results show that the general view of the genetic diversity of MTBC is limited and support the possibility that other MTBC variants exist, particularly in wild mammals in Africa. Exploring this diversity is crucial to the understanding of the biology and evolutionary history of this widespread infectious disease.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI090928</span>; PHS HHS: <span class="grant">HHSN266200700022C</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">19</span>;<span class="number">6</span>;<span class="pagerange">969-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23735084">23735084</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3713819">3713819</a>; DOI: <a href="http://dx.doi.org/10.3201%2Feid1906.121012">10.3201/eid1906.121012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus.</h4>
<p class="authors"><span class="author">Cotten M</span>, <span class="author">Lam TT</span>, <span class="author">Watson SJ</span>, <span class="author">Palser AL</span>, <span class="author">Petrova V</span>, <span class="author">Grant P</span>, <span class="author">Pybus OG</span>, <span class="author">Rambaut A</span>, <span class="author">Guan Y</span>, <span class="author">Pillay D</span>, <span class="author">Kellam P</span> and <span class="author">Nastouli E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">A novel betacoronavirus associated with lethal respiratory and renal complications was recently identified in patients from several countries in the Middle East. We report the deep genome sequencing of the virus directly from a patient&#39;s sputum sample. Our high-throughput sequencing yielded a substantial depth of genome sequence assembly and showed the minority viral variants in the specimen. Detailed phylogenetic analysis of the virus genome (England/Qatar/2012) revealed its close relationship to European bat coronaviruses circulating among the bat species of the Vespertilionidae family. Molecular clock analysis showed that the 2 human infections of this betacoronavirus in June 2012 (EMC/2012) and September 2012 (England/Qatar/2012) share a common virus ancestor most likely considerably before early 2012, suggesting the human diversity is the result of multiple zoonotic events.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MR/K006584/1</span>; Wellcome Trust: <span class="grant">093724</span>, <span class="grant">095831</span></p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">19</span>;<span class="number">5</span>;<span class="pagerange">736-42B</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23693015">23693015</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3647518">3647518</a>; DOI: <a href="http://dx.doi.org/10.3201%2Feid1905.130057">10.3201/eid1905.130057</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association and longitudinal analyses reveal genetic loci linking pubertal height growth, pubertal timing and childhood adiposity.</h4>
<p class="authors"><span class="author">Cousminer DL</span>, <span class="author">Berry DJ</span>, <span class="author">Timpson NJ</span>, <span class="author">Ang W</span>, <span class="author">Thiering E</span>, <span class="author">Byrne EM</span>, <span class="author">Taal HR</span>, <span class="author">Huikari V</span>, <span class="author">Bradfield JP</span>, <span class="author">Kerkhof M</span>, <span class="author">Groen-Blokhuis MM</span>, <span class="author">Kreiner-M&oslash;ller E</span>, <span class="author">Marinelli M</span>, <span class="author">Holst C</span>, <span class="author">Leinonen JT</span>, <span class="author">Perry JR</span>, <span class="author">Surakka I</span>, <span class="author">Pietil&auml;inen O</span>, <span class="author">Kettunen J</span>, <span class="author">Anttila V</span>, <span class="author">Kaakinen M</span>, <span class="author">Sovio U</span>, <span class="author">Pouta A</span>, <span class="author">Das S</span>, <span class="author">Lagou V</span>, <span class="author">Power C</span>, <span class="author">Prokopenko I</span>, <span class="author">Evans DM</span>, <span class="author">Kemp JP</span>, <span class="author">St Pourcain B</span>, <span class="author">Ring S</span>, <span class="author">Palotie A</span>, <span class="author">Kajantie E</span>, <span class="author">Osmond C</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Viikari JS</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Warrington NM</span>, <span class="author">Lye SJ</span>, <span class="author">Palmer LJ</span>, <span class="author">Tiesler CM</span>, <span class="author">Flexeder C</span>, <span class="author">Montgomery GW</span>, <span class="author">Medland SE</span>, <span class="author">Hofman A</span>, <span class="author">Hakonarson H</span>, <span class="author">Guxens M</span>, <span class="author">Bartels M</span>, <span class="author">Salomaa V</span>, <span class="author">ReproGen Consortium</span>, <span class="author">Murabito JM</span>, <span class="author">Kaprio J</span>, <span class="author">S&oslash;rensen TI</span>, <span class="author">Ballester F</span>, <span class="author">Bisgaard H</span>, <span class="author">Boomsma DI</span>, <span class="author">Koppelman GH</span>, <span class="author">Grant SF</span>, <span class="author">Jaddoe VW</span>, <span class="author">Martin NG</span>, <span class="author">Heinrich J</span>, <span class="author">Pennell CE</span>, <span class="author">Raitakari OT</span>, <span class="author">Eriksson JG</span>, <span class="author">Smith GD</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">McCarthy MI</span>, <span class="author">Ripatti S</span>, <span class="author">Wid&eacute;n E</span> and <span class="author">Early Growth Genetics (EGG) Consortium</span></p>
<p class="affiliation">A full list of members is provided in the Supplementary Material.</p><p class="abstract">The pubertal height growth spurt is a distinctive feature of childhood growth reflecting both the central onset of puberty and local growth factors. Although little is known about the underlying genetics, growth variability during puberty correlates with adult risks for hormone-dependent cancer and adverse cardiometabolic health. The only gene so far associated with pubertal height growth, LIN28B, pleiotropically influences childhood growth, puberty and cancer progression, pointing to shared underlying mechanisms. To discover genetic loci influencing pubertal height and growth and to place them in context of overall growth and maturation, we performed genome-wide association meta-analyses in 18 737 European samples utilizing longitudinally collected height measurements. We found significant associations (P &lt; 1.67 &times; 10(-8)) at 10 loci, including LIN28B. Five loci associated with pubertal timing, all impacting multiple aspects of growth. In particular, a novel variant correlated with expression of MAPK3, and associated both with increased prepubertal growth and earlier menarche. Another variant near ADCY3-POMC associated with increased body mass index, reduced pubertal growth and earlier puberty. Whereas epidemiological correlations suggest that early puberty marks a pathway from rapid prepubertal growth to reduced final height and adult obesity, our study shows that individual loci associating with pubertal growth have variable longitudinal growth patterns that may differ from epidemiological observations. Overall, this study uncovers part of the complex genetic architecture linking pubertal height growth, the timing of puberty and childhood obesity and provides new information to pinpoint processes linking these traits.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">13</span>;<span class="pagerange">2735-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23449627">23449627</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3674797">3674797</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddt104">10.1093/hmg/ddt104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large scale variation in DNA copy number in chicken breeds.</h4>
<p class="authors"><span class="author">Crooijmans RP</span>, <span class="author">Fife MS</span>, <span class="author">Fitzgerald TW</span>, <span class="author">Strickland S</span>, <span class="author">Cheng HH</span>, <span class="author">Kaiser P</span>, <span class="author">Redon R</span> and <span class="author">Groenen MA</span></p>
<p class="affiliation">Animal Breeding and Genomics Centre, Wageningen University, P,O, box 338, Wageningen 6700 AH, The Netherlands. <a href="mailto:%72%69%63%68%61%72%64.%63%72%6F%6F%69%6A%6D%61%6E%73%40%77%75%72.%6E%6C">&#114;&#105;&#99;&#104;&#97;&#114;&#100;&#46;&#99;&#114;&#111;&#111;&#105;&#106;&#109;&#97;&#110;&#115;&#64;&#119;&#117;&#114;&#46;&#110;&#108;</a>.</p>
<p class="abstract">Background: Detecting genetic variation is a critical step in elucidating the molecular mechanisms underlying phenotypic diversity. Until recently, such detection has mostly focused on single nucleotide polymorphisms (SNPs) because of the ease in screening complete genomes. Another type of variant, copy number variation (CNV), is emerging as a significant contributor to phenotypic variation in many species. Here we describe a genome-wide CNV study using array comparative genomic hybridization (aCGH) in a wide variety of chicken breeds.

Results: We identified 3,154 CNVs, grouped into 1,556 CNV regions (CNVRs). Thirty percent of the CNVs were detected in at least 2 individuals. The average size of the CNVs detected was 46.3 kb with the largest CNV, located on GGAZ, being 4.3 Mb. Approximately 75% of the CNVs are copy number losses relatively to the Red Jungle Fowl reference genome. The genome coverage of CNVRs in this study is 60 Mb, which represents almost 5.4% of the chicken genome. In particular large gene families such as the keratin gene family and the MHC show extensive CNV.

Conclusions: A relative large group of the CNVs are line-specific, several of which were previously shown to be related to the causative mutation for a number of phenotypic variants. The chance that inter-specific CNVs fall into CNVRs detected in chicken is related to the evolutionary distance between the species. Our results provide a valuable resource for the study of genetic and phenotypic variation in this phenotypically diverse species.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">398</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23763846">23763846</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3751642">3751642</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-14-398">10.1186/1471-2164-14-398</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens.</h4>
<p class="authors"><span class="author">Crooks L</span>, <span class="author">Carlborg O</span>, <span class="author">Marklund S</span> and <span class="author">Johansson AM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute.</p>
<p class="abstract">We analyzed genotypes from ~10K SNPs in two families of an F2 intercross between Red Junglefowl and White Leghorn chickens. Possible null alleles were found by patterns of incompatible and missing genotypes. We estimated that 2.6% of SNPs had null alleles compared to 2.3% with genotyping errors and that 40% of SNPs where a parent and offspring were genotyped as different homozygotes had null alleles. Putative deletions were identified by null alleles at adjacent markers. We found two candidate deletions that were supported by fluorescence intensity data from a 60K SNP chip. One of the candidate deletions was from the Red Junglefowl and one was present in both the Red Junglefowl and White Leghorn. Both candidate deletions spanned protein-coding regions and were close to a previously detected QTL affecting body weight in this population. This study demonstrates that the ~50K SNP genotyping arrays now available for several agricultural species can be used to identify null alleles and deletions in data from large families. We suggest that our approach could be a useful complement to linkage analysis in experimental crosses.</p>
<p class="publication"><span class="title">G3 (Bethesda, Md.)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23708300">23708300</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fg3.113.006643">10.1534/g3.113.006643</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A library of functional recombinant cell-surface and secreted P. falciparum merozoite proteins.</h4>
<p class="authors"><span class="author">Crosnier C</span>, <span class="author">Wanaguru M</span>, <span class="author">McDade B</span>, <span class="author">Osier FH</span>, <span class="author">Marsh K</span>, <span class="author">Rayner JC</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling laboratory, Wellcome Trust Sanger Institute, Cambridge CB10 1HH, UK;</p><p class="abstract">Malaria, an infectious disease caused by parasites of the Plasmodium genus, is one of the world&#39;s major public health concerns causing up to a million deaths annually, mostly because of P. falciparum infections. All of the clinical symptoms are associated with the blood stage of the disease, an obligate part of the parasite life cycle, when a form of the parasite called the merozoite recognizes and invades host erythrocytes. During erythrocyte invasion, merozoites are directly exposed to the host humoral immune system making the blood stage of the parasite a conceptually attractive therapeutic target. Progress in the functional and molecular characterization of P. falciparum merozoite proteins, however, has been hampered by the technical challenges associated with expressing these proteins in a biochemically active recombinant form. This challenge is particularly acute for extracellular proteins, which are the likely targets of host antibody responses, because they contain structurally critical post-translational modifications that are not added by some recombinant expression systems. Here, we report the development of a method that uses a mammalian expression system to compile a protein resource containing the entire ectodomains of 42 P. falciparum merozoite secreted and cell surface proteins, many of which have not previously been characterized. Importantly, we are able to recapitulate known biochemical activities by showing that recombinant MSP1-MSP7 and P12-P41 directly interact, and that both recombinant EBA175 and EBA140 can bind human erythrocytes in a sialic acid-dependent manner. Finally, we use sera from malaria-exposed immune adults to profile the relative immunoreactivity of the proteins and show that the majority of the antigens contain conformational (heat-labile) epitopes. We envisage that this resource of recombinant proteins will make a valuable contribution toward a molecular understanding of the blood stage of P. falciparum infections and facilitate the comparative screening of antigens as blood-stage vaccine candidates.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MR/J002283/1</span>; Wellcome Trust: <span class="grant">092654</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">12</span>;<span class="pagerange">3976-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24043421">24043421</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3861738">3861738</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.O113.028357">10.1074/mcp.O113.028357</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic relationship between five psychiatric disorders estimated from genome-wide SNPs.</h4>
<p class="authors"><span class="author">Cross-Disorder Group of the Psychiatric Genomics Consortium</span>, <span class="author">Lee SH</span>, <span class="author">Ripke S</span>, <span class="author">Neale BM</span>, <span class="author">Faraone SV</span>, <span class="author">Purcell SM</span>, <span class="author">Perlis RH</span>, <span class="author">Mowry BJ</span>, <span class="author">Thapar A</span>, <span class="author">Goddard ME</span>, <span class="author">Witte JS</span>, <span class="author">Absher D</span>, <span class="author">Agartz I</span>, <span class="author">Akil H</span>, <span class="author">Amin F</span>, <span class="author">Andreassen OA</span>, <span class="author">Anjorin A</span>, <span class="author">Anney R</span>, <span class="author">Anttila V</span>, <span class="author">Arking DE</span>, <span class="author">Asherson P</span>, <span class="author">Azevedo MH</span>, <span class="author">Backlund L</span>, <span class="author">Badner JA</span>, <span class="author">Bailey AJ</span>, <span class="author">Banaschewski T</span>, <span class="author">Barchas JD</span>, <span class="author">Barnes MR</span>, <span class="author">Barrett TB</span>, <span class="author">Bass N</span>, <span class="author">Battaglia A</span>, <span class="author">Bauer M</span>, <span class="author">Bay&eacute;s M</span>, <span class="author">Bellivier F</span>, <span class="author">Bergen SE</span>, <span class="author">Berrettini W</span>, <span class="author">Betancur C</span>, <span class="author">Bettecken T</span>, <span class="author">Biederman J</span>, <span class="author">Binder EB</span>, <span class="author">Black DW</span>, <span class="author">Blackwood DH</span>, <span class="author">Bloss CS</span>, <span class="author">Boehnke M</span>, <span class="author">Boomsma DI</span>, <span class="author">Breen G</span>, <span class="author">Breuer R</span>, <span class="author">Bruggeman R</span>, <span class="author">Cormican P</span>, <span class="author">Buccola NG</span>, <span class="author">Buitelaar JK</span>, <span class="author">Bunney WE</span>, <span class="author">Buxbaum JD</span>, <span class="author">Byerley WF</span>, <span class="author">Byrne EM</span>, <span class="author">Caesar S</span>, <span class="author">Cahn W</span>, <span class="author">Cantor RM</span>, <span class="author">Casas M</span>, <span class="author">Chakravarti A</span>, <span class="author">Chambert K</span>, <span class="author">Choudhury K</span>, <span class="author">Cichon S</span>, <span class="author">Cloninger CR</span>, <span class="author">Collier DA</span>, <span class="author">Cook EH</span>, <span class="author">Coon H</span>, <span class="author">Cormand B</span>, <span class="author">Corvin A</span>, <span class="author">Coryell WH</span>, <span class="author">Craig DW</span>, <span class="author">Craig IW</span>, <span class="author">Crosbie J</span>, <span class="author">Cuccaro ML</span>, <span class="author">Curtis D</span>, <span class="author">Czamara D</span>, <span class="author">Datta S</span>, <span class="author">Dawson G</span>, <span class="author">Day R</span>, <span class="author">De Geus EJ</span>, <span class="author">Degenhardt F</span>, <span class="author">Djurovic S</span>, <span class="author">Donohoe GJ</span>, <span class="author">Doyle AE</span>, <span class="author">Duan J</span>, <span class="author">Dudbridge F</span>, <span class="author">Duketis E</span>, <span class="author">Ebstein RP</span>, <span class="author">Edenberg HJ</span>, <span class="author">Elia J</span>, <span class="author">Ennis S</span>, <span class="author">Etain B</span>, <span class="author">Fanous A</span>, <span class="author">Farmer AE</span>, <span class="author">Ferrier IN</span>, <span class="author">Flickinger M</span>, <span class="author">Fombonne E</span>, <span class="author">Foroud T</span>, <span class="author">Frank J</span>, <span class="author">Franke B</span>, <span class="author">Fraser C</span>, <span class="author">Freedman R</span>, <span class="author">Freimer NB</span>, <span class="author">Freitag CM</span>, <span class="author">Friedl M</span>, <span class="author">Fris&eacute;n L</span>, <span class="author">Gallagher L</span>, <span class="author">Gejman PV</span>, <span class="author">Georgieva L</span>, <span class="author">Gershon ES</span>, <span class="author">Geschwind DH</span>, <span class="author">Giegling I</span>, <span class="author">Gill M</span>, <span class="author">Gordon SD</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Green EK</span>, <span class="author">Greenwood TA</span>, <span class="author">Grice DE</span>, <span class="author">Gross M</span>, <span class="author">Grozeva D</span>, <span class="author">Guan W</span>, <span class="author">Gurling H</span>, <span class="author">De Haan L</span>, <span class="author">Haines JL</span>, <span class="author">Hakonarson H</span>, <span class="author">Hallmayer J</span>, <span class="author">Hamilton SP</span>, <span class="author">Hamshere ML</span>, <span class="author">Hansen TF</span>, <span class="author">Hartmann AM</span>, <span class="author">Hautzinger M</span>, <span class="author">Heath AC</span>, <span class="author">Henders AK</span>, <span class="author">Herms S</span>, <span class="author">Hickie IB</span>, <span class="author">Hipolito M</span>, <span class="author">Hoefels S</span>, <span class="author">Holmans PA</span>, <span class="author">Holsboer F</span>, <span class="author">Hoogendijk WJ</span>, <span class="author">Hottenga JJ</span>, <span class="author">Hultman CM</span>, <span class="author">Hus V</span>, <span class="author">Ingason A</span>, <span class="author">Ising M</span>, <span class="author">Jamain S</span>, <span class="author">Jones EG</span>, <span class="author">Jones I</span>, <span class="author">Jones L</span>, <span class="author">Tzeng JY</span>, <span class="author">K&auml;hler AK</span>, <span class="author">Kahn RS</span>, <span class="author">Kandaswamy R</span>, <span class="author">Keller MC</span>, <span class="author">Kennedy JL</span>, <span class="author">Kenny E</span>, <span class="author">Kent L</span>, <span class="author">Kim Y</span>, <span class="author">Kirov GK</span>, <span class="author">Klauck SM</span>, <span class="author">Klei L</span>, <span class="author">Knowles JA</span>, <span class="author">Kohli MA</span>, <span class="author">Koller DL</span>, <span class="author">Konte B</span>, <span class="author">Korszun A</span>, <span class="author">Krabbendam L</span>, <span class="author">Krasucki R</span>, <span class="author">Kuntsi J</span>, <span class="author">Kwan P</span>, <span class="author">Land&eacute;n M</span>, <span class="author">L&aring;ngstr&ouml;m N</span>, <span class="author">Lathrop M</span>, <span class="author">Lawrence J</span>, <span class="author">Lawson WB</span>, <span class="author">Leboyer M</span>, <span class="author">Ledbetter DH</span>, <span class="author">Lee PH</span>, <span class="author">Lencz T</span>, <span class="author">Lesch KP</span>, <span class="author">Levinson DF</span>, <span class="author">Lewis CM</span>, <span class="author">Li J</span>, <span class="author">Lichtenstein P</span>, <span class="author">Lieberman JA</span>, <span class="author">Lin DY</span>, <span class="author">Linszen DH</span>, <span class="author">Liu C</span>, <span class="author">Lohoff FW</span>, <span class="author">Loo SK</span>, <span class="author">Lord C</span>, <span class="author">Lowe JK</span>, <span class="author">Lucae S</span>, <span class="author">Macintyre DJ</span>, <span class="author">Madden PA</span>, <span class="author">Maestrini E</span>, <span class="author">Magnusson PK</span>, <span class="author">Mahon PB</span>, <span class="author">Maier W</span>, <span class="author">Malhotra AK</span>, <span class="author">Mane SM</span>, <span class="author">Martin CL</span>, <span class="author">Martin NG</span>, <span class="author">Mattheisen M</span>, <span class="author">Matthews K</span>, <span class="author">Mattingsdal M</span>, <span class="author">McCarroll SA</span>, <span class="author">McGhee KA</span>, <span class="author">McGough JJ</span>, <span class="author">McGrath PJ</span>, <span class="author">McGuffin P</span>, <span class="author">McInnis MG</span>, <span class="author">McIntosh A</span>, <span class="author">McKinney R</span>, <span class="author">McLean AW</span>, <span class="author">McMahon FJ</span>, <span class="author">McMahon WM</span>, <span class="author">McQuillin A</span>, <span class="author">Medeiros H</span>, <span class="author">Medland SE</span>, <span class="author">Meier S</span>, <span class="author">Melle I</span>, <span class="author">Meng F</span>, <span class="author">Meyer J</span>, <span class="author">Middeldorp CM</span>, <span class="author">Middleton L</span>, <span class="author">Milanova V</span>, <span class="author">Miranda A</span>, <span class="author">Monaco AP</span>, <span class="author">Montgomery GW</span>, <span class="author">Moran JL</span>, <span class="author">Moreno-De-Luca D</span>, <span class="author">Morken G</span>, <span class="author">Morris DW</span>, <span class="author">Morrow EM</span>, <span class="author">Moskvina V</span>, <span class="author">Muglia P</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">Muir WJ</span>, <span class="author">M&uuml;ller-Myhsok B</span>, <span class="author">Murtha M</span>, <span class="author">Myers RM</span>, <span class="author">Myin-Germeys I</span>, <span class="author">Neale MC</span>, <span class="author">Nelson SF</span>, <span class="author">Nievergelt CM</span>, <span class="author">Nikolov I</span>, <span class="author">Nimgaonkar V</span>, <span class="author">Nolen WA</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Nurnberger JI</span>, <span class="author">Nwulia EA</span>, <span class="author">Nyholt DR</span>, <span class="author">O&#39;Dushlaine C</span>, <span class="author">Oades RD</span>, <span class="author">Olincy A</span>, <span class="author">Oliveira G</span>, <span class="author">Olsen L</span>, <span class="author">Ophoff RA</span>, <span class="author">Osby U</span>, <span class="author">Owen MJ</span>, <span class="author">Palotie A</span>, <span class="author">Parr JR</span>, <span class="author">Paterson AD</span>, <span class="author">Pato CN</span>, <span class="author">Pato MT</span>, <span class="author">Penninx BW</span>, <span class="author">Pergadia ML</span>, <span class="author">Pericak-Vance MA</span>, <span class="author">Pickard BS</span>, <span class="author">Pimm J</span>, <span class="author">Piven J</span>, <span class="author">Posthuma D</span>, <span class="author">Potash JB</span>, <span class="author">Poustka F</span>, <span class="author">Propping P</span>, <span class="author">Puri V</span>, <span class="author">Quested DJ</span>, <span class="author">Quinn EM</span>, <span class="author">Ramos-Quiroga JA</span>, <span class="author">Rasmussen HB</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Reif A</span>, <span class="author">Ribas&eacute;s M</span>, <span class="author">Rice JP</span>, <span class="author">Rietschel M</span>, <span class="author">Roeder K</span>, <span class="author">Roeyers H</span>, <span class="author">Rossin L</span>, <span class="author">Rothenberger A</span>, <span class="author">Rouleau G</span>, <span class="author">Ruderfer D</span>, <span class="author">Rujescu D</span>, <span class="author">Sanders AR</span>, <span class="author">Sanders SJ</span>, <span class="author">Santangelo SL</span>, <span class="author">Sergeant JA</span>, <span class="author">Schachar R</span>, <span class="author">Schalling M</span>, <span class="author">Schatzberg AF</span>, <span class="author">Scheftner WA</span>, <span class="author">Schellenberg GD</span>, <span class="author">Scherer SW</span>, <span class="author">Schork NJ</span>, <span class="author">Schulze TG</span>, <span class="author">Schumacher J</span>, <span class="author">Schwarz M</span>, <span class="author">Scolnick E</span>, <span class="author">Scott LJ</span>, <span class="author">Shi J</span>, <span class="author">Shilling PD</span>, <span class="author">Shyn SI</span>, <span class="author">Silverman JM</span>, <span class="author">Slager SL</span>, <span class="author">Smalley SL</span>, <span class="author">Smit JH</span>, <span class="author">Smith EN</span>, <span class="author">Sonuga-Barke EJ</span>, <span class="author">St Clair D</span>, <span class="author">State M</span>, <span class="author">Steffens M</span>, <span class="author">Steinhausen HC</span>, <span class="author">Strauss JS</span>, <span class="author">Strohmaier J</span>, <span class="author">Stroup TS</span>, <span class="author">Sutcliffe JS</span>, <span class="author">Szatmari P</span>, <span class="author">Szelinger S</span>, <span class="author">Thirumalai S</span>, <span class="author">Thompson RC</span>, <span class="author">Todorov AA</span>, <span class="author">Tozzi F</span>, <span class="author">Treutlein J</span>, <span class="author">Uhr M</span>, <span class="author">van den Oord EJ</span>, <span class="author">Van Grootheest G</span>, <span class="author">Van Os J</span>, <span class="author">Vicente AM</span>, <span class="author">Vieland VJ</span>, <span class="author">Vincent JB</span>, <span class="author">Visscher PM</span>, <span class="author">Walsh CA</span>, <span class="author">Wassink TH</span>, <span class="author">Watson SJ</span>, <span class="author">Weissman MM</span>, <span class="author">Werge T</span>, <span class="author">Wienker TF</span>, <span class="author">Wijsman EM</span>, <span class="author">Willemsen G</span>, <span class="author">Williams N</span>, <span class="author">Willsey AJ</span>, <span class="author">Witt SH</span>, <span class="author">Xu W</span>, <span class="author">Young AH</span>, <span class="author">Yu TW</span>, <span class="author">Zammit S</span>, <span class="author">Zandi PP</span>, <span class="author">Zhang P</span>, <span class="author">Zitman FG</span>, <span class="author">Z&ouml;llner S</span>, <span class="author">International Inflammatory Bowel Disease Genetics Consortium (IIBDGC)</span>, <span class="author">Devlin B</span>, <span class="author">Kelsoe JR</span>, <span class="author">Sklar P</span>, <span class="author">Daly MJ</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Craddock N</span>, <span class="author">Sullivan PF</span>, <span class="author">Smoller JW</span>, <span class="author">Kendler KS</span> and <span class="author">Wray NR</span></p>
<p class="affiliation">The University of Queensland, Queensland Brain Institute, Brisbane, Queensland, Australia.</p><p class="abstract">Most psychiatric disorders are moderately to highly heritable. The degree to which genetic variation is unique to individual disorders or shared across disorders is unclear. To examine shared genetic etiology, we use genome-wide genotype data from the Psychiatric Genomics Consortium (PGC) for cases and controls in schizophrenia, bipolar disorder, major depressive disorder, autism spectrum disorders (ASD) and attention-deficit/hyperactivity disorder (ADHD). We apply univariate and bivariate methods for the estimation of genetic variation within and covariation between disorders. SNPs explained 17-29% of the variance in liability. The genetic correlation calculated using common SNPs was high between schizophrenia and bipolar disorder (0.68 &plusmn; 0.04 s.e.), moderate between schizophrenia and major depressive disorder (0.43 &plusmn; 0.06 s.e.), bipolar disorder and major depressive disorder (0.47 &plusmn; 0.06 s.e.), and ADHD and major depressive disorder (0.32 &plusmn; 0.07 s.e.), low between schizophrenia and ASD (0.16 &plusmn; 0.06 s.e.) and non-significant for other pairs of disorders as well as between psychiatric disorders and the negative control of Crohn&#39;s disease. This empirical evidence of shared genetic etiology for psychiatric disorders can inform nosology and encourages the investigation of common pathophysiologies for related disorders.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">9</span>;<span class="pagerange">984-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23933821">23933821</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2711">10.1038/ng.2711</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genomics of post-vaccine changes in pneumococcal epidemiology.</h4>
<p class="authors"><span class="author">Croucher NJ</span>, <span class="author">Finkelstein JA</span>, <span class="author">Pelton SI</span>, <span class="author">Mitchell PK</span>, <span class="author">Lee GM</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span>, <span class="author">Hanage WP</span> and <span class="author">Lipsitch M</span></p>
<p class="affiliation">Center for Communicable Disease Dynamics, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA.</p><p class="abstract">Whole-genome sequencing of 616 asymptomatically carried Streptococcus pneumoniae isolates was used to study the impact of the 7-valent pneumococcal conjugate vaccine. Comparison of closely related isolates showed the role of transformation in facilitating capsule switching to non-vaccine serotypes and the emergence of drug resistance. However, such recombination was found to occur at significantly different rates across the species, and the evolution of the population was primarily driven by changes in the frequency of distinct genotypes extant before the introduction of the vaccine. These alterations resulted in little overall effect on accessory genome composition at the population level, contrasting with the decrease in pneumococcal disease rates after the vaccine&#39;s introduction.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI066304</span>, <span class="grant">R01AI066304</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">656-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23644493">23644493</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3725542">3725542</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2625">10.1038/ng.2625</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacterial genomes in epidemiology--present and future.</h4>
<p class="authors"><span class="author">Croucher NJ</span>, <span class="author">Harris SR</span>, <span class="author">Grad YH</span> and <span class="author">Hanage WP</span></p>
<p class="affiliation">Department of Epidemiology, Center for Communicable Disease Dynamics, Harvard School of Public Health, Boston, MA, USA.</p>
<p class="abstract">Sequence data are well established in the reconstruction of the phylogenetic and demographic scenarios that have given rise to outbreaks of viral pathogens. The application of similar methods to bacteria has been hindered in the main by the lack of high-resolution nucleotide sequence data from quality samples. Developing and already available genomic methods have greatly increased the amount of data that can be used to characterize an isolate and its relationship to others. However, differences in sequencing platforms and data analysis mean that these enhanced data come with a cost in terms of portability: results from one laboratory may not be directly comparable with those from another. Moreover, genomic data for many bacteria bear the mark of a history including extensive recombination, which has the potential to greatly confound phylogenetic and coalescent analyses. Here, we discuss the exacting requirements of genomic epidemiology, and means by which the distorting signal of recombination can be minimized to permit the leverage of growing datasets of genomic data from bacterial pathogens.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">T32 AI007061</span>; NIGMS NIH HHS: <span class="grant">GM088558-01</span>, <span class="grant">U54 GM088558</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2013</span>;<span class="volume">368</span>;<span class="number">1614</span>;<span class="pagerange">20120202</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382424">23382424</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3678326">3678326</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2012.0202">10.1098/rstb.2012.0202</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dominant Role of Nucleotide Substitution in the Diversification of Serotype 3 Pneumococci over Decades and during a Single Infection.</h4>
<p class="authors"><span class="author">Croucher NJ</span>, <span class="author">Mitchell AM</span>, <span class="author">Gould KA</span>, <span class="author">Inverarity D</span>, <span class="author">Barquist L</span>, <span class="author">Feltwell T</span>, <span class="author">Fookes MC</span>, <span class="author">Harris SR</span>, <span class="author">Dordel J</span>, <span class="author">Salter SJ</span>, <span class="author">Browall S</span>, <span class="author">Zemlickova H</span>, <span class="author">Parkhill J</span>, <span class="author">Normark S</span>, <span class="author">Henriques-Normark B</span>, <span class="author">Hinds J</span>, <span class="author">Mitchell TJ</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Pathogen Genomics, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom ; Center for Communicable Disease Dynamics, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, United States of America.</p>
<p class="abstract">Streptococcus pneumoniae of serotype 3 possess a mucoid capsule and cause disease associated with high mortality rates relative to other pneumococci. Phylogenetic analysis of a complete reference genome and 81 draft sequences from clonal complex 180, the predominant serotype 3 clone in much of the world, found most sampled isolates belonged to a clade affected by few diversifying recombinations. However, other isolates indicate significant genetic variation has accumulated over the clonal complex's entire history. Two closely related genomes, one from the blood and another from the cerebrospinal fluid, were obtained from a patient with meningitis. The pair differed in their behaviour in a mouse model of disease and in their susceptibility to antimicrobials, with at least some of these changes attributable to a mutation that up-regulated the patAB efflux pump. This indicates clinically important phenotypic variation can accumulate rapidly through small alterations to the genotype.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">10</span>;<span class="pagerange">e1003868</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24130509">24130509</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003868">10.1371/journal.pgen.1003868</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.</h4>
<p class="authors"><span class="author">Cubillos FA</span>, <span class="author">Parts L</span>, <span class="author">Salinas F</span>, <span class="author">Bergstr&ouml;m A</span>, <span class="author">Scovacricchi E</span>, <span class="author">Zia A</span>, <span class="author">Illingworth CJ</span>, <span class="author">Mustonen V</span>, <span class="author">Ibstedt S</span>, <span class="author">Warringer J</span>, <span class="author">Louis EJ</span>, <span class="author">Durbin R</span> and <span class="author">Liti G</span></p>
<p class="affiliation">Centre for Genetics and Genomics, Queen's Medical Centre, University of Nottingham, Nottingham, NG7 2UH, United Kingdom.</p>
<p class="abstract">A large fraction of human complex trait heritability is due to a high number of variants with small marginal effects and their interactions with genotype and environment. Such alleles are more easily studied in model organisms, where environment, genetic makeup, and allele frequencies can be controlled. Here, we examine the effect of natural genetic variation on heritable traits in a very large pool of baker's yeast from a multiparent 12th generation intercross. We selected four representative founder strains to produce the Saccharomyces Genome Resequencing Project (SGRP)-4X mapping population and sequenced 192 segregants to generate an accurate genetic map. Using these individuals, we mapped 25 loci linked to growth traits under heat stress, arsenite, and paraquat, the majority of which were best explained by a diverging phenotype caused by a single allele in one condition. By sequencing pooled DNA from millions of segregants grown under heat stress, we further identified 34 and 39 regions selected in haploid and diploid pools, respectively, with most of the selection against a single allele. While the most parsimonious model for the majority of loci mapped using either approach was the effect of an allele private to one founder, we could validate examples of pleiotropic effects and complex allelic series at a locus. SGRP-4X is a deeply characterized resource that provides a framework for powerful and high-resolution genetic analysis of yeast phenotypes and serves as a test bed for testing avenues to attack human complex traits.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT077192/Z/05/Z</span></p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">195</span>;<span class="number">3</span>;<span class="pagerange">1141-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24037264">24037264</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3813843">3813843</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.113.155515">10.1534/genetics.113.155515</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SMIM1 underlies the Vel blood group and influences red blood cell traits.</h4>
<p class="authors"><span class="author">Cvejic A</span>, <span class="author">Haer-Wigman L</span>, <span class="author">Stephens JC</span>, <span class="author">Kostadima M</span>, <span class="author">Smethurst PA</span>, <span class="author">Frontini M</span>, <span class="author">van den Akker E</span>, <span class="author">Bertone P</span>, <span class="author">Bielczyk-Maczy&#x144;ska E</span>, <span class="author">Farrow S</span>, <span class="author">Fehrmann RS</span>, <span class="author">Gray A</span>, <span class="author">de Haas M</span>, <span class="author">Haver VG</span>, <span class="author">Jordan G</span>, <span class="author">Karjalainen J</span>, <span class="author">Kerstens HH</span>, <span class="author">Kiddle G</span>, <span class="author">Lloyd-Jones H</span>, <span class="author">Needs M</span>, <span class="author">Poole J</span>, <span class="author">Soussan AA</span>, <span class="author">Rendon A</span>, <span class="author">Rieneck K</span>, <span class="author">Sambrook JG</span>, <span class="author">Schepers H</span>, <span class="author">Sillj&eacute; HH</span>, <span class="author">Sipos B</span>, <span class="author">Swinkels D</span>, <span class="author">Tamuri AU</span>, <span class="author">Verweij N</span>, <span class="author">Watkins NA</span>, <span class="author">Westra HJ</span>, <span class="author">Stemple D</span>, <span class="author">Franke L</span>, <span class="author">Soranzo N</span>, <span class="author">Stunnenberg HG</span>, <span class="author">Goldman N</span>, <span class="author">van der Harst P</span>, <span class="author">van der Schoot CE</span>, <span class="author">Ouwehand WH</span> and <span class="author">Albers CA</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, Cambridge, UK. <a href="mailto:%61%73%38%38%39%40%63%61%6D.%61%63.%75%6B">&#97;&#115;&#56;&#56;&#57;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The blood group Vel was discovered 60 years ago, but the underlying gene is unknown. Individuals negative for the Vel antigen are rare and are required for the safe transfusion of patients with antibodies to Vel. To identify the responsible gene, we sequenced the exomes of five individuals negative for the Vel antigen and found that four were homozygous and one was heterozygous for a low-frequency 17-nucleotide frameshift deletion in the gene encoding the 78-amino-acid transmembrane protein SMIM1. A follow-up study showing that 59 of 64 Vel-negative individuals were homozygous for the same deletion and expression of the Vel antigen on SMIM1-transfected cells confirm SMIM1 as the gene underlying the Vel blood group. An expression quantitative trait locus (eQTL), the common SNP rs1175550 contributes to variable expression of the Vel antigen (P = 0.003) and influences the mean hemoglobin concentration of red blood cells (RBCs; P = 8.6 &times; 10(-15)). In vivo, zebrafish with smim1 knockdown showed a mild reduction in the number of RBCs, identifying SMIM1 as a new regulator of RBC formation. Our findings are of immediate relevance, as the homozygous presence of the deletion allows the unequivocal identification of Vel-negative blood donors.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>, <span class="grant">RG/09/12/28096</span>; Cancer Research UK: <span class="grant">14953</span>, <span class="grant">A14953</span>, <span class="grant">C45041/A14953</span>; Wellcome Trust: <span class="grant">082597</span>, <span class="grant">082597/Z/07/Z</span>, <span class="grant">084183</span>, <span class="grant">084183/Z/07/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">5</span>;<span class="pagerange">542-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23563608">23563608</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4179282">4179282</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2603">10.1038/ng.2603</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Horizontally acquired glycosyltransferase operons drive salmonellae lipopolysaccharide diversity.</h4>
<p class="authors"><span class="author">Davies MR</span>, <span class="author">Broadbent SE</span>, <span class="author">Harris SR</span>, <span class="author">Thomson NR</span> and <span class="author">van der Woude MW</span></p>
<p class="affiliation">Centre for Immunology and Infection, Hull York Medical School and the Department of Biology, University of York, York, United Kingdom.</p>
<p class="abstract">The immunodominant lipopolysaccharide is a key antigenic factor for Gram-negative pathogens such as salmonellae where it plays key roles in host adaptation, virulence, immune evasion, and persistence. Variation in the lipopolysaccharide is also the major differentiating factor that is used to classify Salmonella into over 2600 serovars as part of the Kaufmann-White scheme. While lipopolysaccharide diversity is generally associated with sequence variation in the lipopolysaccharide biosynthesis operon, extraneous genetic factors such as those encoded by the glucosyltransferase (gtr) operons provide further structural heterogeneity by adding additional sugars onto the O-antigen component of the lipopolysaccharide. Here we identify and examine the O-antigen modifying glucosyltransferase genes from the genomes of Salmonella enterica and Salmonella bongori serovars. We show that Salmonella generally carries between 1 and 4 gtr operons that we have classified into 10 families on the basis of gtrC sequence with apparent O-antigen modification detected for five of these families. The gtr operons localize to bacteriophage-associated genomic regions and exhibit a dynamic evolutionary history driven by recombination and gene shuffling events leading to new gene combinations. Furthermore, evidence of Dam- and OxyR-dependent phase variation of gtr gene expression was identified within eight gtr families. Thus, as O-antigen modification generates significant intra- and inter-strain phenotypic diversity, gtr-mediated modification is fundamental in assessing Salmonella strain variability. This will inform appropriate vaccine and diagnostic approaches, in addition to contributing to our understanding of host-pathogen interactions.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">6</span>;<span class="pagerange">e1003568</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23818865">23818865</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3688519">3688519</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003568">10.1371/journal.pgen.1003568</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural and functional annotation of the porcine immunome.</h4>
<p class="authors"><span class="author">Dawson HD</span>, <span class="author">Loveland JE</span>, <span class="author">Pascal G</span>, <span class="author">Gilbert JG</span>, <span class="author">Uenishi H</span>, <span class="author">Mann KM</span>, <span class="author">Sang Y</span>, <span class="author">Zhang J</span>, <span class="author">Carvalho-Silva D</span>, <span class="author">Hunt T</span>, <span class="author">Hardy M</span>, <span class="author">Hu Z</span>, <span class="author">Zhao SH</span>, <span class="author">Anselmo A</span>, <span class="author">Shinkai H</span>, <span class="author">Chen C</span>, <span class="author">Badaoui B</span>, <span class="author">Berman D</span>, <span class="author">Amid C</span>, <span class="author">Kay M</span>, <span class="author">Lloyd D</span>, <span class="author">Snow C</span>, <span class="author">Morozumi T</span>, <span class="author">Cheng RP</span>, <span class="author">Bystrom M</span>, <span class="author">Kapetanovic R</span>, <span class="author">Schwartz JC</span>, <span class="author">Kataria R</span>, <span class="author">Astley M</span>, <span class="author">Fritz E</span>, <span class="author">Steward C</span>, <span class="author">Thomas M</span>, <span class="author">Wilming L</span>, <span class="author">Toki D</span>, <span class="author">Archibald AL</span>, <span class="author">Bed&#39;Hom B</span>, <span class="author">Beraldi D</span>, <span class="author">Huang TH</span>, <span class="author">Ait-Ali T</span>, <span class="author">Blecha F</span>, <span class="author">Botti S</span>, <span class="author">Freeman TC</span>, <span class="author">Giuffra E</span>, <span class="author">Hume DA</span>, <span class="author">Lunney JK</span>, <span class="author">Murtaugh MP</span>, <span class="author">Reecy JM</span>, <span class="author">Harrow JL</span>, <span class="author">Rogel-Gaillard C</span> and <span class="author">Tuggle CK</span></p>
<p class="affiliation">USDA-ARS, Beltsville Human Nutrition Research Center, Diet, Genomics, Immunology Laboratory, Beltsville, MD 20705, USA.</p><p class="abstract"><strong>Background</strong>: The domestic pig is known as an excellent model for human immunology and the two species share many pathogens. Susceptibility to infectious disease is one of the major constraints on swine performance, yet the structure and function of genes comprising the pig immunome are not well-characterized. The completion of the pig genome provides the opportunity to annotate the pig immunome, and compare and contrast pig and human immune systems.</p><p class="abstract"><strong>Results</strong>: The Immune Response Annotation Group (IRAG) used computational curation and manual annotation of the swine genome assembly 10.2 (Sscrofa10.2) to refine the currently available automated annotation of 1,369 immunity-related genes through sequence-based comparison to genes in other species. Within these genes, we annotated 3,472 transcripts. Annotation provided evidence for gene expansions in several immune response families, and identified artiodactyl-specific expansions in the cathelicidin and type 1 Interferon families. We found gene duplications for 18 genes, including 13 immune response genes and five non-immune response genes discovered in the annotation process. Manual annotation provided evidence for many new alternative splice variants and 8 gene duplications. Over 1,100 transcripts without porcine sequence evidence were detected using cross-species annotation. We used a functional approach to discover and accurately annotate porcine immune response genes. A co-expression clustering analysis of transcriptomic data from selected experimental infections or immune stimulations of blood, macrophages or lymph nodes identified a large cluster of genes that exhibited a correlated positive response upon infection across multiple pathogens or immune stimuli. Interestingly, this gene cluster (cluster 4) is enriched for known general human immune response genes, yet contains many un-annotated porcine genes. A phylogenetic analysis of the encoded proteins of cluster 4 genes showed that 15% exhibited an accelerated evolution as compared to 4.1% across the entire genome.</p><p class="abstract"><strong>Conclusions</strong>: This extensive annotation dramatically extends the genome-based knowledge of the molecular genetics and structure of a major portion of the porcine immunome. Our complementary functional approach using co-expression during immune response has provided new putative immune response annotation for over 500 porcine genes. Our phylogenetic analysis of this core immunome cluster confirms rapid evolutionary change in this set of genes, and that, as in other species, such genes are important components of the pig&#39;s adaptation to pathogen challenge over evolutionary time. These comprehensive and integrated analyses increase the value of the porcine genome sequence and provide important tools for global analyses and data-mining of the porcine immune response.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E010520/1</span>, <span class="grant">BB/E010520/2</span>, <span class="grant">BB/G004013/1</span>, <span class="grant">BB/I025328/1</span>, <span class="grant">EC FP6</span>; NCRR NIH HHS: <span class="grant">P20-RR017686</span>; NIAID NIH HHS: <span class="grant">T32 AI083196</span>, <span class="grant">T32 AI83196</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">332</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23676093">23676093</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3658956">3658956</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-14-332">10.1186/1471-2164-14-332</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prelamin A causes progeria through cell-extrinsic mechanisms and prevents cancer invasion.</h4>
<p class="authors"><span class="author">de la Rosa J</span>, <span class="author">Freije JM</span>, <span class="author">Cabanillas R</span>, <span class="author">Osorio FG</span>, <span class="author">Fraga MF</span>, <span class="author">Fern&aacute;ndez-Garc&iacute;a MS</span>, <span class="author">Rad R</span>, <span class="author">Fanjul V</span>, <span class="author">Ugalde AP</span>, <span class="author">Liang Q</span>, <span class="author">Prosser HM</span>, <span class="author">Bradley A</span>, <span class="author">Cadi&ntilde;anos J</span> and <span class="author">L&oacute;pez-Ot&iacute;n C</span></p>
<p class="affiliation">Instituto de Medicina Oncol&oacute;gica y Molecular de Asturias IMOMA, 33193 Oviedo, Spain.</p><p class="abstract">Defining the relationship between ageing and cancer is a crucial but challenging task. Mice deficient in Zmpste24, a metalloproteinase mutated in human progeria and involved in nuclear prelamin A maturation, recapitulate multiple features of ageing. However, their short lifespan and serious cell-intrinsic and cell-extrinsic alterations restrict the application and interpretation of carcinogenesis protocols. Here we present Zmpste24 mosaic mice that lack these limitations. Zmpste24 mosaic mice develop normally and keep similar proportions of Zmpste24-deficient (prelamin A-accumulating) and Zmpste24-proficient (mature lamin A-containing) cells throughout life, revealing that cell-extrinsic mechanisms are preeminent for progeria development. Moreover, prelamin A accumulation does not impair tumour initiation and growth, but it decreases the incidence of infiltrating oral carcinomas. Accordingly, silencing of ZMPSTE24 reduces human cancer cell invasiveness. Our results support the potential of cell-based and systemic therapies for progeria and highlight ZMPSTE24 as a new anticancer target.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Nature communications</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="pagerange">2268</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23917225">23917225</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3758871">3758871</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncomms3268">10.1038/ncomms3268</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutational genomics for cancer pathway discovery</h4>
<p class="authors"><span class="author">De Ridder,J.</span>, <span class="author">Kool,J</span>, <span class="author">Uren,A.G.</span>, <span class="author">Bot,J.</span>, <span class="author">De Jong,J.</span>, <span class="author">RUST,A.G.</span>, <span class="author">Berns,A.</span>, <span class="author">Van Lohuizen,M.</span>, <span class="author">ADAMS,D.J.</span>, <span class="author">Wessels,L.</span> and <span class="author">Reinders,M.;</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science </span>&nbsp;<span class="year">2013</span>;<span class="volume">7986</span>;<span class="pagerange">35-46</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-39159-0_4">10.1007/978-3-642-39159-0_4</a>; URL: <a title="http://link.springer.com/chapter/10.1007/978-3-642-39159-0_4" href="http://link.springer.com/chapter/10.1007/978-3-642-39159-0_4">http://link.springer.com/chapter<span class="print-hide">...</span><span class="print-show">/10.1007/978-3-642-3</span>9159-0_4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of heart rate-associated loci and their effects on cardiac conduction and rhythm disorders.</h4>
<p class="authors"><span class="author">den Hoed M</span>, <span class="author">Eijgelsheim M</span>, <span class="author">Esko T</span>, <span class="author">Brundel BJ</span>, <span class="author">Peal DS</span>, <span class="author">Evans DM</span>, <span class="author">Nolte IM</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">Holm H</span>, <span class="author">Handsaker RE</span>, <span class="author">Westra HJ</span>, <span class="author">Johnson T</span>, <span class="author">Isaacs A</span>, <span class="author">Yang J</span>, <span class="author">Lundby A</span>, <span class="author">Zhao JH</span>, <span class="author">Kim YJ</span>, <span class="author">Go MJ</span>, <span class="author">Almgren P</span>, <span class="author">Bochud M</span>, <span class="author">Boucher G</span>, <span class="author">Cornelis MC</span>, <span class="author">Gudbjartsson D</span>, <span class="author">Hadley D</span>, <span class="author">van der Harst P</span>, <span class="author">Hayward C</span>, <span class="author">den Heijer M</span>, <span class="author">Igl W</span>, <span class="author">Jackson AU</span>, <span class="author">Kutalik Z</span>, <span class="author">Luan J</span>, <span class="author">Kemp JP</span>, <span class="author">Kristiansson K</span>, <span class="author">Ladenvall C</span>, <span class="author">Lorentzon M</span>, <span class="author">Montasser ME</span>, <span class="author">Njajou OT</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Padmanabhan S</span>, <span class="author">St Pourcain B</span>, <span class="author">Rankinen T</span>, <span class="author">Salo P</span>, <span class="author">Tanaka T</span>, <span class="author">Timpson NJ</span>, <span class="author">Vitart V</span>, <span class="author">Waite L</span>, <span class="author">Wheeler W</span>, <span class="author">Zhang W</span>, <span class="author">Draisma HH</span>, <span class="author">Feitosa MF</span>, <span class="author">Kerr KF</span>, <span class="author">Lind PA</span>, <span class="author">Mihailov E</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Song C</span>, <span class="author">Weedon MN</span>, <span class="author">Xie W</span>, <span class="author">Yengo L</span>, <span class="author">Absher D</span>, <span class="author">Albert CM</span>, <span class="author">Alonso A</span>, <span class="author">Arking DE</span>, <span class="author">de Bakker PI</span>, <span class="author">Balkau B</span>, <span class="author">Barlassina C</span>, <span class="author">Benaglio P</span>, <span class="author">Bis JC</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Brage S</span>, <span class="author">Chanock SJ</span>, <span class="author">Chines PS</span>, <span class="author">Chung M</span>, <span class="author">Darbar D</span>, <span class="author">Dina C</span>, <span class="author">D&ouml;rr M</span>, <span class="author">Elliott P</span>, <span class="author">Felix SB</span>, <span class="author">Fischer K</span>, <span class="author">Fuchsberger C</span>, <span class="author">de Geus EJ</span>, <span class="author">Goyette P</span>, <span class="author">Gudnason V</span>, <span class="author">Harris TB</span>, <span class="author">Hartikainen AL</span>, <span class="author">Havulinna AS</span>, <span class="author">Heckbert SR</span>, <span class="author">Hicks AA</span>, <span class="author">Hofman A</span>, <span class="author">Holewijn S</span>, <span class="author">Hoogstra-Berends F</span>, <span class="author">Hottenga JJ</span>, <span class="author">Jensen MK</span>, <span class="author">Johansson A</span>, <span class="author">Junttila J</span>, <span class="author">K&auml;&auml;b S</span>, <span class="author">Kanon B</span>, <span class="author">Ketkar S</span>, <span class="author">Khaw KT</span>, <span class="author">Knowles JW</span>, <span class="author">Kooner AS</span>, <span class="author">Kors JA</span>, <span class="author">Kumari M</span>, <span class="author">Milani L</span>, <span class="author">Laiho P</span>, <span class="author">Lakatta EG</span>, <span class="author">Langenberg C</span>, <span class="author">Leusink M</span>, <span class="author">Liu Y</span>, <span class="author">Luben RN</span>, <span class="author">Lunetta KL</span>, <span class="author">Lynch SN</span>, <span class="author">Markus MR</span>, <span class="author">Marques-Vidal P</span>, <span class="author">Mateo Leach I</span>, <span class="author">McArdle WL</span>, <span class="author">McCarroll SA</span>, <span class="author">Medland SE</span>, <span class="author">Miller KA</span>, <span class="author">Montgomery GW</span>, <span class="author">Morrison AC</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Navarro P</span>, <span class="author">Nelis M</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Ong KK</span>, <span class="author">Newman AB</span>, <span class="author">Peters A</span>, <span class="author">Polasek O</span>, <span class="author">Pouta A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Psaty BM</span>, <span class="author">Rao DC</span>, <span class="author">Ring SM</span>, <span class="author">Rossin EJ</span>, <span class="author">Rudan D</span>, <span class="author">Sanna S</span>, <span class="author">Scott RA</span>, <span class="author">Sehmi JS</span>, <span class="author">Sharp S</span>, <span class="author">Shin JT</span>, <span class="author">Singleton AB</span>, <span class="author">Smith AV</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Stewart C</span>, <span class="author">Stringham HM</span>, <span class="author">Tarasov KV</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Vandenput L</span>, <span class="author">Hwang SJ</span>, <span class="author">Whitfield JB</span>, <span class="author">Wijmenga C</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Wilson JF</span>, <span class="author">Witteman JC</span>, <span class="author">Wong A</span>, <span class="author">Wong Q</span>, <span class="author">Jamshidi Y</span>, <span class="author">Zitting P</span>, <span class="author">Boer JM</span>, <span class="author">Boomsma DI</span>, <span class="author">Borecki IB</span>, <span class="author">van Duijn CM</span>, <span class="author">Ekelund U</span>, <span class="author">Forouhi NG</span>, <span class="author">Froguel P</span>, <span class="author">Hingorani A</span>, <span class="author">Ingelsson E</span>, <span class="author">Kivimaki M</span>, <span class="author">Kronmal RA</span>, <span class="author">Kuh D</span>, <span class="author">Lind L</span>, <span class="author">Martin NG</span>, <span class="author">Oostra BA</span>, <span class="author">Pedersen NL</span>, <span class="author">Quertermous T</span>, <span class="author">Rotter JI</span>, <span class="author">van der Schouw YT</span>, <span class="author">Verschuren WM</span>, <span class="author">Walker M</span>, <span class="author">Albanes D</span>, <span class="author">Arnar DO</span>, <span class="author">Assimes TL</span>, <span class="author">Bandinelli S</span>, <span class="author">Boehnke M</span>, <span class="author">de Boer RA</span>, <span class="author">Bouchard C</span>, <span class="author">Caulfield WL</span>, <span class="author">Chambers JC</span>, <span class="author">Curhan G</span>, <span class="author">Cusi D</span>, <span class="author">Eriksson J</span>, <span class="author">Ferrucci L</span>, <span class="author">van Gilst WH</span>, <span class="author">Glorioso N</span>, <span class="author">de Graaf J</span>, <span class="author">Groop L</span>, <span class="author">Gyllensten U</span>, <span class="author">Hsueh WC</span>, <span class="author">Hu FB</span>, <span class="author">Huikuri HV</span>, <span class="author">Hunter DJ</span>, <span class="author">Iribarren C</span>, <span class="author">Isomaa B</span>, <span class="author">Jarvelin MR</span>, <span class="author">Jula A</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kiemeney LA</span>, <span class="author">van der Klauw MM</span>, <span class="author">Kooner JS</span>, <span class="author">Kraft P</span>, <span class="author">Iacoviello L</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Lokki ML</span>, <span class="author">Mitchell BD</span>, <span class="author">Navis G</span>, <span class="author">Nieminen MS</span>, <span class="author">Ohlsson C</span>, <span class="author">Poulter NR</span>, <span class="author">Qi L</span>, <span class="author">Raitakari OT</span>, <span class="author">Rimm EB</span>, <span class="author">Rioux JD</span>, <span class="author">Rizzi F</span>, <span class="author">Rudan I</span>, <span class="author">Salomaa V</span>, <span class="author">Sever PS</span>, <span class="author">Shields DC</span>, <span class="author">Shuldiner AR</span>, <span class="author">Sinisalo J</span>, <span class="author">Stanton AV</span>, <span class="author">Stolk RP</span>, <span class="author">Strachan DP</span>, <span class="author">Tardif JC</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Tuomilehto J</span>, <span class="author">van Veldhuisen DJ</span>, <span class="author">Virtamo J</span>, <span class="author">Viikari J</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Widen E</span>, <span class="author">Cho YS</span>, <span class="author">Olsen JV</span>, <span class="author">Visscher PM</span>, <span class="author">Willer C</span>, <span class="author">Franke L</span>, <span class="author">Global BPgen Consortium</span>, <span class="author">CARDIoGRAM Consortium</span>, <span class="author">Erdmann J</span>, <span class="author">Thompson JR</span>, <span class="author">PR GWAS Consortium</span>, <span class="author">Pfeufer A</span>, <span class="author">QRS GWAS Consortium</span>, <span class="author">Sotoodehnia N</span>, <span class="author">QT-IGC Consortium</span>, <span class="author">Newton-Cheh C</span>, <span class="author">CHARGE-AF Consortium</span>, <span class="author">Ellinor PT</span>, <span class="author">Stricker BH</span>, <span class="author">Metspalu A</span>, <span class="author">Perola M</span>, <span class="author">Beckmann JS</span>, <span class="author">Smith GD</span>, <span class="author">Stefansson K</span>, <span class="author">Wareham NJ</span>, <span class="author">Munroe PB</span>, <span class="author">Sibon OC</span>, <span class="author">Milan DJ</span>, <span class="author">Snieder H</span>, <span class="author">Samani NJ</span> and <span class="author">Loos RJ</span></p>
<p class="affiliation">Medical Research Council MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge, UK.</p><p class="abstract">Elevated resting heart rate is associated with greater risk of cardiovascular disease and mortality. In a 2-stage meta-analysis of genome-wide association studies in up to 181,171 individuals, we identified 14 new loci associated with heart rate and confirmed associations with all 7 previously established loci. Experimental downregulation of gene expression in Drosophila melanogaster and Danio rerio identified 20 genes at 11 loci that are relevant for heart rate regulation and highlight a role for genes involved in signal transmission, embryonic cardiac development and the pathophysiology of dilated cardiomyopathy, congenital heart failure and/or sudden cardiac death. In addition, genetic susceptibility to increased heart rate is associated with altered cardiac conduction and reduced risk of sick sinus syndrome, and both heart rate-increasing and heart rate-decreasing variants associate with risk of atrial fibrillation. Our findings provide fresh insights into the mechanisms regulating heart rate and identify new therapeutic targets.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/11/63/29011</span>, <span class="grant">PG/12/38/29615</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0600705</span>, <span class="grant">G0801056</span>, <span class="grant">G1000143</span>, <span class="grant">G9815508</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106179473</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UU_12013/3</span>, <span class="grant">MC_UU_12013/4</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NHLBI NIH HHS: <span class="grant">K24 HL105780</span>, <span class="grant">R00 HL094535</span>, <span class="grant">R01 HL090620</span>, <span class="grant">R01 HL092217</span>, <span class="grant">R01 HL105756</span>, <span class="grant">R01 HL111314</span>, <span class="grant">U19 HL065962</span>; NIDDK NIH HHS: <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK072488</span>; NIGMS NIH HHS: <span class="grant">T32 GM007753</span>; Wellcome Trust: <span class="grant">092731</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">621-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23583979">23583979</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3696959">3696959</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2610">10.1038/ng.2610</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of HIV and ART with cardiometabolic traits in sub-Saharan Africa: a systematic review and meta-analysis.</h4>
<p class="authors"><span class="author">Dillon DG</span>, <span class="author">Gurdasani D</span>, <span class="author">Riha J</span>, <span class="author">Ekoru K</span>, <span class="author">Asiki G</span>, <span class="author">Mayanja BN</span>, <span class="author">Levitt NS</span>, <span class="author">Crowther NJ</span>, <span class="author">Nyirenda M</span>, <span class="author">Njelekela M</span>, <span class="author">Ramaiya K</span>, <span class="author">Nyan O</span>, <span class="author">Adewole OO</span>, <span class="author">Anastos K</span>, <span class="author">Azzoni L</span>, <span class="author">Boom WH</span>, <span class="author">Compostella C</span>, <span class="author">Dave JA</span>, <span class="author">Dawood H</span>, <span class="author">Erikstrup C</span>, <span class="author">Fourie CM</span>, <span class="author">Friis H</span>, <span class="author">Kruger A</span>, <span class="author">Idoko JA</span>, <span class="author">Longenecker CT</span>, <span class="author">Mbondi S</span>, <span class="author">Mukaya JE</span>, <span class="author">Mutimura E</span>, <span class="author">Ndhlovu CE</span>, <span class="author">Praygod G</span>, <span class="author">Pefura Yone EW</span>, <span class="author">Pujades-Rodriguez M</span>, <span class="author">Range N</span>, <span class="author">Sani MU</span>, <span class="author">Schutte AE</span>, <span class="author">Sliwa K</span>, <span class="author">Tien PC</span>, <span class="author">Vorster EH</span>, <span class="author">Walsh C</span>, <span class="author">Zinyama R</span>, <span class="author">Mashili F</span>, <span class="author">Sobngwi E</span>, <span class="author">Adebamowo C</span>, <span class="author">Kamali A</span>, <span class="author">Seeley J</span>, <span class="author">Young EH</span>, <span class="author">Smeeth L</span>, <span class="author">Motala AA</span>, <span class="author">Kaleebu P</span>, <span class="author">Sandhu MS</span> and <span class="author">African Partnership for Chronic Disease Research (APCDR)</span></p>
<p class="affiliation">Department of Public Health and Primary Care, Institute of Public Health, University of Cambridge, Cambridge, UK, Genetic Epidemiology Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK, MRC/UVRI Uganda Research Unit on AIDS, Entebbe, Uganda, Division of Diabetic Medicine and Endocrinology, Department of Medicine, University of Cape Town, Cape Town, South Africa; Chronic Diseases Initiative in Africa, Department of Chemical Pathology, National Health Laboratory Service, University of the Witwatersrand Medical School, Johannesburg, South Africa, Malawi-Liverpool-Wellcome Trust Clinical Research Programme, Blantyre, Malawi, Department of Physiology, Muhimbili University of Health and Allied Sciences, Dar es Salaam, Tanzania, Department of Medicine, Muhimbili University of Health and Allied Sciences, Dar es Salaam, Tanzania, Royal Victoria Teaching Hospital, School of Medicine, University of The Gambia, Banjul, The Gambia, Department of Medicine, Obafemi Awolowo University, Ile Ife, Nigeria, Women's Equity in Access to Care &Treatment, Kigali, Rwanda, HIV-1 Immunopathogenesis Laboratory, Wistar Institute, Philadelphia, PA, Tuberculosis Research Unit, Department of Medicine, Case Western Reserve University, Cleveland, OH, Department of Medical and Surgical Sciences, University of Padua, Padua, Italy, Division of Diabetic Medicine and Endocrinology, Department of Medicine, University of Cape Town, Cape Town, South Africa, Infectious Diseases Unit, Department of Medicine, Grey's Hospital, Pietermaritzburg, South Africa, Department of Clinical Immunology, Aarhus University Hospital, Aarhus, Denmark, HART (Hypertension in Africa Research Team), North-West University, Potchefstroom, South Africa, Department of Nutrition, Exercise and Sports, Faculty of Science, University of Copenhagen, Copenhagen, Denmark, Africa Unit for Transdisciplinary Health Research (AUTHeR), North-West University, Potchefstroom, South Africa, Department of Medicine, Jos University Teachin</p>
<p class="abstract">Background: Sub-Saharan Africa (SSA) has the highest burden of HIV in the world and a rising prevalence of cardiometabolic disease; however, the interrelationship between HIV, antiretroviral therapy (ART) and cardiometabolic traits is not well described in SSA populations.

Methods: We conducted a systematic review and meta-analysis through MEDLINE and EMBASE (up to January 2012), as well as direct author contact. Eligible studies provided summary or individual-level data on one or more of the following traits in HIV+ and HIV-, or ART+ and ART- subgroups in SSA: body mass index (BMI), systolic blood pressure (SBP), diastolic blood pressure (DBP), high-density lipoprotein (HDL), low-density lipoprotein (LDL), triglycerides (TGs) and fasting blood glucose (FBG) or glycated hemoglobin (HbA1c). Information was synthesized under a random-effects model and the primary outcomes were the standardized mean differences (SMD) of the specified traits between subgroups of participants.

Results: Data were obtained from 49 published and 3 unpublished studies which reported on 29 755 individuals. HIV infection was associated with higher TGs [SMD, 0.26; 95% confidence interval (CI), 0.08 to 0.44] and lower HDL (SMD, -0.59; 95% CI, -0.86 to -0.31), BMI (SMD, -0.32; 95% CI, -0.45 to -0.18), SBP (SMD, -0.40; 95% CI, -0.55 to -0.25) and DBP (SMD, -0.34; 95% CI, -0.51 to -0.17). Among HIV+ individuals, ART use was associated with higher LDL (SMD, 0.43; 95% CI, 0.14 to 0.72) and HDL (SMD, 0.39; 95% CI, 0.11 to 0.66), and lower HbA1c (SMD, -0.34; 95% CI, -0.62 to -0.06). Fully adjusted estimates from analyses of individual participant data were consistent with meta-analysis of summary estimates for most traits.

Conclusions: Broadly consistent with results from populations of European descent, these results suggest differences in cardiometabolic traits between HIV-infected and uninfected individuals in SSA, which might be modified by ART use. In a region with the highest burden of HIV, it will be important to clarify these findings to reliably assess the need for monitoring and managing cardiometabolic risk in HIV-infected populations in SSA.</p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">42</span>;<span class="number">6</span>;<span class="pagerange">1754-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24415610">24415610</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdyt198">10.1093/ije/dyt198</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multi-allelic Phenotyping - A systematic approach for the simultaneous analysis of multiple induced mutations.</h4>
<p class="authors"><span class="author">Dooley CM</span>, <span class="author">Scahill C</span>, <span class="author">F&eacute;nyes F</span>, <span class="author">W Kettleborough RN</span>, <span class="author">Stemple DL</span> and <span class="author">Busch-Nentwich EM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p>
<p class="abstract">The zebrafish mutation project (ZMP) aims to generate a loss of function allele for every protein-coding gene, but importantly to also characterise the phenotypes of these alleles during the first five days of development. Such a large-scale screen requires a systematic approach both to identifying phenotypes, and also to linking those phenotypes to specific mutations. This phenotyping pipeline simultaneously assesses the consequences of multiple alleles in a two-step process. First, mutations that do not produce a visible phenotype during the first five days of development are identified, while a second round of phenotyping focuses on detailed analysis of those alleles that are suspected to cause a phenotype. Allele-specific PCR single nucleotide polymorphism (SNP) assays are used to genotype F2 parents and individual F3 fry for mutations known to be present in the F1 founder. With this method specific phenotypes can be linked to induced mutations. In addition a method is described for cryopreserving sperm samples of mutagenised males and their subsequent use for in vitro fertilisation to generate F2 families for phenotyping. Ultimately this approach will lead to the functional annotation of the zebrafish genome, which will deepen our understanding of gene function in development and disease.</p>
<p class="publication"><span class="title">Methods (San Diego, Calif.)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23624102">23624102</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ymeth.2013.04.013">10.1016/j.ymeth.2013.04.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Back to the future!</h4>
<p class="authors"><span class="author">Dordel J</span> and <span class="author">Reuter S</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">9</span>;<span class="pagerange">600</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23949601">23949601</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3099">10.1038/nrmicro3099</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Histone deacetylase 1 and 2 are essential for normal T-cell development and genomic stability in mice.</h4>
<p class="authors"><span class="author">Dovey OM</span>, <span class="author">Foster CT</span>, <span class="author">Conte N</span>, <span class="author">Edwards SA</span>, <span class="author">Edwards JM</span>, <span class="author">Singh R</span>, <span class="author">Vassiliou G</span>, <span class="author">Bradley A</span> and <span class="author">Cowley SM</span></p>
<p class="affiliation">Department of Biochemistry, University of Leicester, Leicester, UK.</p><p class="abstract">Histone deacetylase 1 and 2 (HDAC1/2) regulate chromatin structure as the catalytic core of the Sin3A, NuRD and CoREST co-repressor complexes. To better understand the key pathways regulated by HDAC1/2 in the adaptive immune system and inform their exploitation as drug targets, we have generated mice with a T-cell specific deletion. Loss of either HDAC1 or HDAC2 alone has little effect, while dual inactivation results in a 5-fold reduction in thymocyte cellularity, accompanied by developmental arrest at the double-negative to double-positive transition. Transcriptome analysis revealed 892 misregulated genes in Hdac1/2 knock-out thymocytes, including down-regulation of LAT, Themis and Itk, key components of the T-cell receptor (TCR) signaling pathway. Down-regulation of these genes suggests a model in which HDAC1/2 deficiency results in defective propagation of TCR signaling, thus blocking development. Furthermore, mice with reduced HDAC1/2 activity (Hdac1 deleted and a single Hdac2 allele) develop a lethal pathology by 3-months of age, caused by neoplastic transformation of immature T cells in the thymus. Tumor cells become aneuploid, express increased levels of c-Myc and show elevated levels of the DNA damage marker, &gamma;H2AX. These data demonstrate a crucial role for HDAC1/2 in T-cell development and the maintenance of genomic stability.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600135</span>, <span class="grant">MR/J009202/1</span>; Wellcome Trust: <span class="grant">079643</span>, <span class="grant">095663</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2013</span>;<span class="volume">121</span>;<span class="number">8</span>;<span class="pagerange">1335-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23287868">23287868</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836254">3836254</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2012-07-441949">10.1182/blood-2012-07-441949</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The presence of methylation quantitative trait Loci indicates a direct genetic influence on the level of DNA methylation in adipose tissue.</h4>
<p class="authors"><span class="author">Drong AW</span>, <span class="author">Nicholson G</span>, <span class="author">Hedman AK</span>, <span class="author">Meduri E</span>, <span class="author">Grundberg E</span>, <span class="author">Small KS</span>, <span class="author">Shin SY</span>, <span class="author">Bell JT</span>, <span class="author">Karpe F</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">McCarthy MI</span>, <span class="author">Deloukas P</span>, <span class="author">Rantalainen M</span>, <span class="author">Lindgren CM</span> and <span class="author">MolPAGE Consortia</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">Genetic variants that associate with DNA methylation at CpG sites (methylation quantitative trait loci, meQTLs) offer a potential biological mechanism of action for disease associated SNPs. We investigated whether meQTLs exist in abdominal subcutaneous adipose tissue (SAT) and if CpG methylation associates with metabolic syndrome (MetSyn) phenotypes. We profiled 27,718 genomic regions in abdominal SAT samples of 38 unrelated individuals using differential methylation hybridization (DMH) together with genotypes at 5,227,243 SNPs and expression of 17,209 mRNA transcripts. Validation and replication of significant meQTLs was pursued in an independent cohort of 181 female twins. We find that, at 5% false discovery rate, methylation levels of 149 DMH regions associate with at least one SNP in a ±500 kilobase cis-region in our primary study. We sought to validate 19 of these in the replication study and find that five of these significantly associate with the corresponding meQTL SNPs from the primary study. We find that none of the 149 meQTL top SNPs is a significant expression quantitative trait locus in our expression data, but we observed association between expression levels of two mRNA transcripts and cis-methylation status. Our results indicate that DNA CpG methylation in abdominal SAT is partly under genetic control. This study provides a starting point for future investigations of DNA methylation in adipose tissue.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">e55923</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23431366">23431366</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3576415">3576415</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0055923">10.1371/journal.pone.0055923</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and Functional Annotation of Avian Pathogenic Escherichia coli Serogroup O78 Strains Reveal the Evolution of E. coli Lineages Pathogenic for Poultry via Distinct Mechanisms.</h4>
<p class="authors"><span class="author">Dziva F</span>, <span class="author">Hauser H</span>, <span class="author">Connor TR</span>, <span class="author">van Diemen PM</span>, <span class="author">Prescott G</span>, <span class="author">Langridge GC</span>, <span class="author">Eckert S</span>, <span class="author">Chaudhuri RR</span>, <span class="author">Ewers C</span>, <span class="author">Mellata M</span>, <span class="author">Mukhopadhyay S</span>, <span class="author">Curtiss R</span>, <span class="author">Dougan G</span>, <span class="author">Wieler LH</span>, <span class="author">Thomson NR</span>, <span class="author">Pickard DJ</span> and <span class="author">Stevens MP</span></p>
<p class="affiliation">Enteric Bacterial Pathogens Laboratory, Institute for Animal Health, Compton, Berkshire, United Kingdom.</p>
<p class="abstract">Avian pathogenic Escherichia coli (APEC) causes respiratory and systemic disease in poultry. Sequencing of a multilocus sequence type 95 (ST95) serogroup O1 strain previously indicated that APEC resembles E. coli causing extraintestinal human diseases. We sequenced the genomes of two strains of another dominant APEC lineage (ST23 serogroup O78 strains Ï‡7122 and IMT2125) and compared them to each other and to the reannotated APEC O1 sequence. For comparison, we also sequenced a human enterotoxigenic E. coli (ETEC) strain of the same ST23 serogroup O78 lineage. Phylogenetic analysis indicated that the APEC O78 strains were more closely related to human ST23 ETEC than to APEC O1, indicating that separation of pathotypes on the basis of their extraintestinal or diarrheagenic nature is not supported by their phylogeny. The accessory genome of APEC ST23 strains exhibited limited conservation of APEC O1 genomic islands and a distinct repertoire of virulence-associated loci. In light of this diversity, we surveyed the phenotype of 2,185 signature-tagged transposon mutants of Ï‡7122 following intra-air sac inoculation of turkeys. This procedure identified novel APEC ST23 genes that play strain- and tissue-specific roles during infection. For example, genes mediating group 4 capsule synthesis were required for the virulence of Ï‡7122 and were conserved in IMT2125 but absent from APEC O1. Our data reveal the genetic diversity of E. coli strains adapted to cause the same avian disease and indicate that the core genome of the ST23 lineage serves as a chassis for the evolution of E. coli strains adapted to cause avian or human disease via acquisition of distinct virulence genes.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2013</span>;<span class="volume">81</span>;<span class="number">3</span>;<span class="pagerange">838-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23275093">23275093</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3584874">3584874</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00585-12">10.1128/IAI.00585-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The SHOCT Domain: A Widespread Domain Under-Represented in Model Organisms.</h4>
<p class="authors"><span class="author">Eberhardt RY</span>, <span class="author">Bartholdson SJ</span>, <span class="author">Punta M</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom ; Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p>
<p class="abstract">We have identified a new protein domain, which we have named the SHOCT domain (ort -erminal domain). This domain is widespread in bacteria with over a thousand examples. But we found it is missing from the most commonly studied model organisms, despite being present in closely related species. It's predominantly C-terminal location, co-occurrence with numerous other domains and short size is reminiscent of the Gram-positive anchor motif, however it is present in a much wider range of species. We suggest several hypotheses about the function of SHOCT, including oligomerisation and nucleic acid binding. Our initial experiments do not support its role as an oligomerisation domain.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">e57848</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23451277">23451277</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3581485">3581485</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0057848">10.1371/journal.pone.0057848</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Filling out the structural map of the NTF2-like superfamily.</h4>
<p class="authors"><span class="author">Eberhardt RY</span>, <span class="author">Chang Y</span>, <span class="author">Bateman A</span>, <span class="author">Murzin AG</span>, <span class="author">Axelrod HL</span>, <span class="author">Hwang WC</span> and <span class="author">Aravind L</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%72%65%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#101;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: The NTF2-like superfamily is a versatile group of protein domains sharing a common fold. The sequences of these domains are very diverse and they share no common sequence motif. These domains serve a range of different functions within the proteins in which they are found, including both catalytic and non-catalytic versions. Clues to the function of protein domains belonging to such a diverse superfamily can be gleaned from analysis of the proteins and organisms in which they are found.

Results: Here we describe three protein domains of unknown function found mainly in bacteria: DUF3828, DUF3887 and DUF4878. Structures of representatives of each of these domains: BT_3511 from Bacteroides thetaiotaomicron (strain VPI-5482) [PDB:3KZT], Cj0202c from Campylobacter jejuni subsp. jejuni serotype O:2 (strain NCTC 11168) [PDB:3K7C], rumgna_01855) and RUMGNA_01855 from Ruminococcus gnavus (strain ATCC 29149) [PDB:4HYZ] have been solved by X-ray crystallography. All three domains are similar in structure and all belong to the NTF2-like superfamily. Although the function of these domains remains unknown at present, our analysis enables us to present a hypothesis concerning their role.

Conclusions: Our analysis of these three protein domains suggests a potential non-catalytic ligand-binding role. This may regulate the activities of domains with which they are combined in the same polypeptide or via operonic linkages, such as signaling domains (e.g. serine/threonine protein kinase), peptidoglycan-processing hydrolases (e.g. NlpC/P60 peptidases) or nucleic acid binding domains (e.g. Zn-ribbons).</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">327</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24246060">24246060</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-14-327">10.1186/1471-2105-14-327</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The GenoChip: a new tool for genetic anthropology.</h4>
<p class="authors"><span class="author">Elhaik E</span>, <span class="author">Greenspan E</span>, <span class="author">Staats S</span>, <span class="author">Krahn T</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Xue Y</span>, <span class="author">Tofanelli S</span>, <span class="author">Francalacci P</span>, <span class="author">Cucca F</span>, <span class="author">Pagani L</span>, <span class="author">Jin L</span>, <span class="author">Li H</span>, <span class="author">Schurr TG</span>, <span class="author">Greenspan B</span>, <span class="author">Spencer Wells R</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Department of Mental Health, Johns Hopkins University Bloomberg School of Public Health, USA.</p><p class="abstract">The Genographic Project is an international effort aimed at charting human migratory history. The project is nonprofit and nonmedical, and, through its Legacy Fund, supports locally led efforts to preserve indigenous and traditional cultures. Although the first phase of the project was focused on uniparentally inherited markers on the Y-chromosome and mitochondrial DNA (mtDNA), the current phase focuses on markers from across the entire genome to obtain a more complete understanding of human genetic variation. Although many commercial arrays exist for genome-wide single-nucleotide polymorphism (SNP) genotyping, they were designed for medical genetic studies and contain medically related markers that are inappropriate for global population genetic studies. GenoChip, the Genographic Project&#39;s new genotyping array, was designed to resolve these issues and enable higher resolution research into outstanding questions in genetic anthropology. The GenoChip includes ancestry informative markers obtained for over 450 human populations, an ancient human (Saqqaq), and two archaic hominins (Neanderthal and Denisovan) and was designed to identify all known Y-chromosome and mtDNA haplogroups. The chip was carefully vetted to avoid inclusion of medically relevant markers. To demonstrate its capabilities, we compared the FST distributions of GenoChip SNPs to those of two commercial arrays. Although all arrays yielded similarly shaped (inverse J) FST distributions, the GenoChip autosomal and X-chromosomal distributions had the highest mean FST, attesting to its ability to discern subpopulations. The chip performances are illustrated in a principal component analysis for 14 worldwide populations. In summary, the GenoChip is a dedicated genotyping platform for genetic anthropology. With an unprecedented number of approximately 12,000 Y-chromosomal and approximately 3,300 mtDNA SNPs and over 130,000 autosomal and X-chromosomal SNPs without any known health, medical, or phenotypic relevance, the GenoChip is a useful tool for genetic anthropology and population genetics.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">T32 MH014592</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome biology and evolution</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">1021-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23666864">23666864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3673633">3673633</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fgbe%2Fevt066">10.1093/gbe/evt066</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The 5q31 region in two African populations as a facet of natural selection by infectious diseases.</h4>
<p class="authors"><span class="author">Elhassan AA</span>, <span class="author">Hussein AA</span>, <span class="author">Mohamed HS</span>, <span class="author">Rockett K</span>, <span class="author">Kwiatkowski D</span>, <span class="author">Elhassan AM</span> and <span class="author">Ibrahim ME</span></p>
<p class="affiliation">Unit of Disease and Diversity, Department of Molecular Biology, Institute of Endemic Diseases, University of Khartoum, Khartoum, Sudan. <a href="mailto:%61%62%6A%69%6C%30%34%40%79%61%68%6F%6F.%63%6F%6D">&#97;&#98;&#106;&#105;&#108;&#48;&#52;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Cases of extreme natural selection could lead either to rapid fixation or extinction of alleles depending on the population structure and size. It may also manifest in excess of heterozygosity and the locus concerned will be displaying such drastic features of allele change. We suspect the 5q31 in chromosome 5 to mirror situation of such extreme natural selection particularly that the region encompasses genes of type 2 cytokine known to associate with a number of infectious and non-infectious diseases. We typed two sets of single nucleotide polymorphisms (SNPS) in two populations: an initial limited set of only 4 SNP within the genes of IL-4, IL-13, IL-5 and IL-9 in 108 unrelated individuals and a replicating set of 14 SN P in 924 individuals from the same populations with disregard to relatedness. The results suggest the 5q31 area to be under intense selective pressure as indicated by marked heterozygosity independent of Linkage Disequilibrium (LD); difference in heterozygosity, allele, and haplotype frequencies between generations and departure from Hardy-Weinberg expectations (DHWE). The study area is endemic for several infectious diseases including malaria and visceral leishmaniasis (VL). Malaria caused by Plasmodiumfalciparum, however, occurs mostly with mild clinical symptoms in all ages, which makes it unlikely to account for these indices. The strong selection signals seems to emanate from recent outbreaks of VL which affected both populations to varying extent.</p>
<p class="publication"><span class="title">Genetika</span>&nbsp;<span class="year">2013</span>;<span class="volume">49</span>;<span class="number">2</span>;<span class="pagerange">279-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23668094">23668094</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluation of the genetic overlap between osteoarthritis with body mass index and height using genome-wide association scan data.</h4>
<p class="authors"><span class="author">Elliott KS</span>, <span class="author">Chapman K</span>, <span class="author">Day-Williams A</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Southam L</span>, <span class="author">Lindgren CM</span>, <span class="author">GIANT consortium</span>, <span class="author">Arden N</span>, <span class="author">Aslam N</span>, <span class="author">Birrell F</span>, <span class="author">Carluke I</span>, <span class="author">Carr A</span>, <span class="author">Deloukas P</span>, <span class="author">Doherty M</span>, <span class="author">Loughlin J</span>, <span class="author">McCaskie A</span>, <span class="author">Ollier WE</span>, <span class="author">Rai A</span>, <span class="author">Ralston S</span>, <span class="author">Reed MR</span>, <span class="author">Spector TD</span>, <span class="author">Valdes AM</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson M</span>, <span class="author">arcOGEN consortium</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p>
<p class="abstract">Objectives: Obesity as measured by body mass index (BMI) is one of the major risk factors for osteoarthritis. In addition, genetic overlap has been reported between osteoarthritis and normal adult height variation. We investigated whether this relationship is due to a shared genetic aetiology on a genome-wide scale.

Methods: We compared genetic association summary statistics (effect size, p value) for BMI and height from the GIANT consortium genome-wide association study (GWAS) with genetic association summary statistics from the arcOGEN consortium osteoarthritis GWAS. Significance was evaluated by permutation. Replication of osteoarthritis association of the highlighted signals was investigated in an independent dataset. Phenotypic information of height and BMI was accounted for in a separate analysis using osteoarthritis-free controls.

Results: We found significant overlap between osteoarthritis and height (p=3.3×10(-5) for signals with p≤0.05) when the GIANT and arcOGEN GWAS were compared. For signals with p≤0.001 we found 17 shared signals between osteoarthritis and height and four between osteoarthritis and BMI. However, only one of the height or BMI signals that had shown evidence of association with osteoarthritis in the arcOGEN GWAS was also associated with osteoarthritis in the independent dataset: rs12149832, within the FTO gene (combined p=2.3×10(-5)). As expected, this signal was attenuated when we adjusted for BMI.

Conclusions: We found a significant excess of shared signals between both osteoarthritis and height and osteoarthritis and BMI, suggestive of a common genetic aetiology. However, only one signal showed association with osteoarthritis when followed up in a new dataset.</p>
<p class="publication"><span class="title">Annals of the rheumatic diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">72</span>;<span class="number">6</span>;<span class="pagerange">935-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22956599">22956599</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3664369">3664369</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fannrheumdis-2012-202081">10.1136/annrheumdis-2012-202081</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic evaluation of spliced alignment programs for RNA-seq data.</h4>
<p class="authors"><span class="author">Engstr&ouml;m PG</span>, <span class="author">Steijger T</span>, <span class="author">Sipos B</span>, <span class="author">Grant GR</span>, <span class="author">Kahles A</span>, <span class="author">R&auml;tsch G</span>, <span class="author">Goldman N</span>, <span class="author">Hubbard TJ</span>, <span class="author">Harrow J</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Bertone P</span> and <span class="author">RGASP Consortium</span></p>
<p class="affiliation">1] European Molecular Biology Laboratory, European Bioinformatics Institute, Cambridge, UK. [2].</p><p class="abstract">High-throughput RNA sequencing is an increasingly accessible method for studying gene structure and activity on a genome-wide scale. A critical step in RNA-seq data analysis is the alignment of partial transcript reads to a reference genome sequence. To assess the performance of current mapping software, we invited developers of RNA-seq aligners to process four large human and mouse RNA-seq data sets. In total, we compared 26 mapping protocols based on 11 programs and pipelines and found major performance differences between methods on numerous benchmarks, including alignment yield, basewise accuracy, mismatch and gap placement, exon junction discovery and suitability of alignments for transcript reconstruction. We observed concordant results on real and simulated RNA-seq data, confirming the relevance of the metrics employed. Future developments in RNA-seq alignment methods would benefit from improved placement of multimapped reads, balanced utilization of existing gene annotation and a reduced false discovery rate for splice junctions.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG006272</span>, <span class="grant">U41 HG007234</span>, <span class="grant">U54 HG004555</span>, <span class="grant">U54 HG004557</span>, <span class="grant">U54HG004555</span>, <span class="grant">U54HG004557</span>; Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT09805</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">12</span>;<span class="pagerange">1185-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24185836">24185836</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4018468">4018468</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.2722">10.1038/nmeth.2722</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A meta-analysis of genome-wide association studies identifies novel variants associated with osteoarthritis of the hip.</h4>
<p class="authors"><span class="author">Evangelou E</span>, <span class="author">Kerkhof HJ</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Ntzani EE</span>, <span class="author">Bos SD</span>, <span class="author">Esko T</span>, <span class="author">Evans DS</span>, <span class="author">Metrustry S</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Ramos YF</span>, <span class="author">Thorleifsson G</span>, <span class="author">Tsilidis KK</span>, <span class="author">arcOGEN Consortium</span>, <span class="author">Arden N</span>, <span class="author">Aslam N</span>, <span class="author">Bellamy N</span>, <span class="author">Birrell F</span>, <span class="author">Blanco FJ</span>, <span class="author">Carr A</span>, <span class="author">Chapman K</span>, <span class="author">Day-Williams AG</span>, <span class="author">Deloukas P</span>, <span class="author">Doherty M</span>, <span class="author">Engstr&ouml;m G</span>, <span class="author">Helgadottir HT</span>, <span class="author">Hofman A</span>, <span class="author">Ingvarsson T</span>, <span class="author">Jonsson H</span>, <span class="author">Keis A</span>, <span class="author">Keurentjes JC</span>, <span class="author">Kloppenburg M</span>, <span class="author">Lind PA</span>, <span class="author">McCaskie A</span>, <span class="author">Martin NG</span>, <span class="author">Milani L</span>, <span class="author">Montgomery GW</span>, <span class="author">Nelissen RG</span>, <span class="author">Nevitt MC</span>, <span class="author">Nilsson PM</span>, <span class="author">Ollier WE</span>, <span class="author">Parimi N</span>, <span class="author">Rai A</span>, <span class="author">Ralston SH</span>, <span class="author">Reed MR</span>, <span class="author">Riancho JA</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rodriguez-Fontenla C</span>, <span class="author">Southam L</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Tsezou A</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson JM</span>, <span class="author">Gonzalez A</span>, <span class="author">Lane NE</span>, <span class="author">Lohmander LS</span>, <span class="author">Loughlin J</span>, <span class="author">Metspalu A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Jonsdottir I</span>, <span class="author">Stefansson K</span>, <span class="author">Slagboom PE</span>, <span class="author">Zeggini E</span>, <span class="author">Meulenbelt I</span>, <span class="author">Ioannidis JP</span>, <span class="author">Spector TD</span>, <span class="author">van Meurs JB</span> and <span class="author">Valdes AM</span></p>
<p class="affiliation">Department of Hygiene and Epidemiology, University of Ioannina Medical School, Ioannina, Greece.</p><p class="abstract"><strong>Objectives</strong>: Osteoarthritis (OA) is the most common form of arthritis with a clear genetic component. To identify novel loci associated with hip OA we performed a meta-analysis of genome-wide association studies (GWAS) on European subjects.</p><p class="abstract"><strong>Methods</strong>: We performed a two-stage meta-analysis on more than 78 000 participants. In stage 1, we synthesised data from eight GWAS whereas data from 10 centres were used for &#39;in silico&#39; or &#39;de novo&#39; replication. Besides the main analysis, a stratified by sex analysis was performed to detect possible sex-specific signals. Meta-analysis was performed using inverse-variance fixed effects models. A random effects approach was also used.</p><p class="abstract"><strong>Results</strong>: We accumulated 11 277 cases of radiographic and symptomatic hip OA. We prioritised eight single nucleotide polymorphism (SNPs) for follow-up in the discovery stage (4349 OA cases); five from the combined analysis, two male specific and one female specific. One locus, at 20q13, represented by rs6094710 (minor allele frequency (MAF) 4%) near the NCOA3 (nuclear receptor coactivator 3) gene, reached genome-wide significance level with p=7.9&times;10(-9) and OR=1.28 (95% CI 1.18 to 1.39) in the combined analysis of discovery (p=5.6&times;10(-8)) and follow-up studies (p=7.3&times;10(-4)). We showed that this gene is expressed in articular cartilage and its expression was significantly reduced in OA-affected cartilage. Moreover, two loci remained suggestive associated; rs5009270 at 7q31 (MAF 30%, p=9.9&times;10(-7), OR=1.10) and rs3757837 at 7p13 (MAF 6%, p=2.2&times;10(-6), OR=1.27 in male specific analysis).</p><p class="abstract"><strong>Conclusions</strong>: Novel genetic loci for hip OA were found in this meta-analysis of GWAS.</p>
<p class="publication"><span class="title">Annals of the rheumatic diseases</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23989986">23989986</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fannrheumdis-2012-203114">10.1136/annrheumdis-2012-203114</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DOT1L rs12982744 polymorphism is associated with osteoarthritis of the hip with genome-wide statistical significance in males.</h4>
<p class="authors"><span class="author">Evangelou E</span>, <span class="author">Valdes AM</span>, <span class="author">Castano-Betancourt MC</span>, <span class="author">Doherty M</span>, <span class="author">Doherty S</span>, <span class="author">Esko T</span>, <span class="author">Ingvarsson T</span>, <span class="author">Ioannidis JP</span>, <span class="author">Kloppenburg M</span>, <span class="author">Metspalu A</span>, <span class="author">Ntzani EE</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Slagboom PE</span>, <span class="author">Southam L</span>, <span class="author">Spector TD</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Stefanson K</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Wheeler M</span>, <span class="author">Zeggini E</span>, <span class="author">Meulenbelt I</span>, <span class="author">van Meurs JB</span> and <span class="author">arcOGEN consortium, the TREAT-OA consortium</span></p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">18030</span>, <span class="grant">19542</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Annals of the rheumatic diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">72</span>;<span class="number">7</span>;<span class="pagerange">1264-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23505243">23505243</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3686326">3686326</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fannrheumdis-2012-203182">10.1136/annrheumdis-2012-203182</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Defining the range of pathogens susceptible to ifitm3 restriction using a knockout mouse model.</h4>
<p class="authors"><span class="author">Everitt AR</span>, <span class="author">Clare S</span>, <span class="author">McDonald JU</span>, <span class="author">Kane L</span>, <span class="author">Harcourt K</span>, <span class="author">Ahras M</span>, <span class="author">Lall A</span>, <span class="author">Hale C</span>, <span class="author">Rodgers A</span>, <span class="author">Young DB</span>, <span class="author">Haque A</span>, <span class="author">Billker O</span>, <span class="author">Tregoning JS</span>, <span class="author">Dougan G</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p>
<p class="abstract">The interferon-inducible transmembrane (IFITM) family of proteins has been shown to restrict a broad range of viruses in vitro and in vivo by halting progress through the late endosomal pathway. Further, single nucleotide polymorphisms (SNPs) in its sequence have been linked with risk of developing severe influenza virus infections in humans. The number of viruses restricted by this host protein has continued to grow since it was first demonstrated as playing an antiviral role; all of which enter cells via the endosomal pathway. We therefore sought to test the limits of antimicrobial restriction by Ifitm3 using a knockout mouse model. We showed that Ifitm3 does not impact on the restriction or pathogenesis of bacterial (Salmonella typhimurium, Citrobacter rodentium, Mycobacterium tuberculosis) or protozoan (Plasmodium berghei) pathogens, despite in vitro evidence. However, Ifitm3 is capable of restricting respiratory syncytial virus (RSV) in vivo either through directly restricting RSV cell infection, or by exerting a previously uncharacterised function controlling disease pathogenesis. This represents the first demonstration of a virus that enters directly through the plasma membrane, without the need for the endosomal pathway, being restricted by the IFITM family; therefore further defining the role of these antiviral proteins.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e80723</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24278312">24278312</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836756">3836756</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0080723">10.1371/journal.pone.0080723</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Role of Adiposity in Cardiometabolic Traits: A Mendelian Randomization Analysis.</h4>
<p class="authors"><span class="author">Fall T</span>, <span class="author">H&auml;gg S</span>, <span class="author">M&auml;gi R</span>, <span class="author">Ploner A</span>, <span class="author">Fischer K</span>, <span class="author">Horikoshi M</span>, <span class="author">Sarin AP</span>, <span class="author">Thorleifsson G</span>, <span class="author">Ladenvall C</span>, <span class="author">Kals M</span>, <span class="author">Kuningas M</span>, <span class="author">Draisma HH</span>, <span class="author">Ried JS</span>, <span class="author">van Zuydam NR</span>, <span class="author">Huikari V</span>, <span class="author">Mangino M</span>, <span class="author">Sonestedt E</span>, <span class="author">Benyamin B</span>, <span class="author">Nelson CP</span>, <span class="author">Rivera NV</span>, <span class="author">Kristiansson K</span>, <span class="author">Shen HY</span>, <span class="author">Havulinna AS</span>, <span class="author">Dehghan A</span>, <span class="author">Donnelly LA</span>, <span class="author">Kaakinen M</span>, <span class="author">Nuotio ML</span>, <span class="author">Robertson N</span>, <span class="author">de Bruijn RF</span>, <span class="author">Ikram MA</span>, <span class="author">Amin N</span>, <span class="author">Balmforth AJ</span>, <span class="author">Braund PS</span>, <span class="author">Doney AS</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Elliott P</span>, <span class="author">Esko T</span>, <span class="author">Franco OH</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Hartikainen AL</span>, <span class="author">Heikkil&auml; K</span>, <span class="author">Herzig KH</span>, <span class="author">Holm H</span>, <span class="author">Hottenga JJ</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Illig T</span>, <span class="author">Isaacs A</span>, <span class="author">Isomaa B</span>, <span class="author">Karssen LC</span>, <span class="author">Kettunen J</span>, <span class="author">Koenig W</span>, <span class="author">Kuulasmaa K</span>, <span class="author">Laatikainen T</span>, <span class="author">Laitinen J</span>, <span class="author">Lindgren C</span>, <span class="author">Lyssenko V</span>, <span class="author">L&auml;&auml;r&auml; E</span>, <span class="author">Rayner NW</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">Pouta A</span>, <span class="author">Rathmann W</span>, <span class="author">Rivadeneira F</span>, <span class="author">Ruokonen A</span>, <span class="author">Savolainen MJ</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Small KS</span>, <span class="author">Smit JH</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Taanila A</span>, <span class="author">Tobin MD</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Willems SM</span>, <span class="author">Willemsen G</span>, <span class="author">Witteman J</span>, <span class="author">Perola M</span>, <span class="author">Evans A</span>, <span class="author">Ferri&egrave;res J</span>, <span class="author">Virtamo J</span>, <span class="author">Kee F</span>, <span class="author">Tregouet DA</span>, <span class="author">Arveiler D</span>, <span class="author">Amouyel P</span>, <span class="author">Ferrario MM</span>, <span class="author">Brambilla P</span>, <span class="author">Hall AS</span>, <span class="author">Heath AC</span>, <span class="author">Madden PA</span>, <span class="author">Martin NG</span>, <span class="author">Montgomery GW</span>, <span class="author">Whitfield JB</span>, <span class="author">Jula A</span>, <span class="author">Knekt P</span>, <span class="author">Oostra B</span>, <span class="author">van Duijn CM</span>, <span class="author">Penninx BW</span>, <span class="author">Davey Smith G</span>, <span class="author">Kaprio J</span>, <span class="author">Samani NJ</span>, <span class="author">Gieger C</span>, <span class="author">Peters A</span>, <span class="author">Wichmann HE</span>, <span class="author">Boomsma DI</span>, <span class="author">de Geus EJ</span>, <span class="author">Tuomi T</span>, <span class="author">Power C</span>, <span class="author">Hammond CJ</span>, <span class="author">Spector TD</span>, <span class="author">Lind L</span>, <span class="author">Orho-Melander M</span>, <span class="author">Palmer CN</span>, <span class="author">Morris AD</span>, <span class="author">Groop L</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Salomaa V</span>, <span class="author">Vartiainen E</span>, <span class="author">Hofman A</span>, <span class="author">Ripatti S</span>, <span class="author">Metspalu A</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">Pedersen NL</span>, <span class="author">McCarthy MI</span>, <span class="author">Ingelsson E</span>, <span class="author">Prokopenko I</span> and <span class="author">for the European Network for Genetic and Genomic Epidemiology (ENGAGE) consortium</span></p>
<p class="affiliation">Molecular Epidemiology and Science for Life Laboratory, Department of Medical Sciences, Uppsala University, Uppsala, Sweden ; Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden.</p>
<p class="abstract">Background: The association between adiposity and cardiometabolic traits is well known from epidemiological studies. Whilst the causal relationship is clear for some of these traits, for others it is not. We aimed to determine whether adiposity is causally related to various cardiometabolic traits using the Mendelian randomization approach.

We used the adiposity-associated variant rs9939609 at the FTO locus as an instrumental variable (IV) for body mass index (BMI) in a Mendelian randomization design. Thirty-six population-based studies of individuals of European descent contributed to the analyses. Age- and sex-adjusted regression models were fitted to test for association between (i) rs9939609 and BMI (n = 198,502), (ii) rs9939609 and 24 traits, and (iii) BMI and 24 traits. The causal effect of BMI on the outcome measures was quantified by IV estimators. The estimators were compared to the BMI-trait associations derived from the same individuals. In the IV analysis, we demonstrated novel evidence for a causal relationship between adiposity and incident heart failure (hazard ratio, 1.19 per BMI-unit increase; 95% CI, 1.03-1.39) and replicated earlier reports of a causal association with type 2 diabetes, metabolic syndrome, dyslipidemia, and hypertension (odds ratio for IV estimator, 1.1-1.4; all p<0.05). For quantitative traits, our results provide novel evidence for a causal effect of adiposity on the liver enzymes alanine aminotransferase and gamma-glutamyl transferase and confirm previous reports of a causal effect of adiposity on systolic and diastolic blood pressure, fasting insulin, 2-h post-load glucose from the oral glucose tolerance test, C-reactive protein, triglycerides, and high-density lipoprotein cholesterol levels (all p<0.05). The estimated causal effects were in agreement with traditional observational measures in all instances except for type 2 diabetes, where the causal estimate was larger than the observational estimate (p = 0.001).

Conclusions: We provide novel evidence for a causal relationship between adiposity and heart failure as well as between adiposity and increased liver enzymes. Please see later in the article for the Editors' Summary.</p>
<p class="publication"><span class="title">PLoS medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">6</span>;<span class="pagerange">e1001474</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23824655">23824655</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pmed.1001474">10.1371/journal.pmed.1001474</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ImmunoChip Study Implicates Antigen Presentation to T Cells in Narcolepsy.</h4>
<p class="authors"><span class="author">Faraco J</span>, <span class="author">Lin L</span>, <span class="author">Kornum BR</span>, <span class="author">Kenny EE</span>, <span class="author">Trynka G</span>, <span class="author">Einen M</span>, <span class="author">Rico TJ</span>, <span class="author">Lichtner P</span>, <span class="author">Dauvilliers Y</span>, <span class="author">Arnulf I</span>, <span class="author">Lecendreux M</span>, <span class="author">Javidi S</span>, <span class="author">Geisler P</span>, <span class="author">Mayer G</span>, <span class="author">Pizza F</span>, <span class="author">Poli F</span>, <span class="author">Plazzi G</span>, <span class="author">Overeem S</span>, <span class="author">Lammers GJ</span>, <span class="author">Kemlink D</span>, <span class="author">Sonka K</span>, <span class="author">Nevsimalova S</span>, <span class="author">Rouleau G</span>, <span class="author">Desautels A</span>, <span class="author">Montplaisir J</span>, <span class="author">Frauscher B</span>, <span class="author">Ehrmann L</span>, <span class="author">H&ouml;gl B</span>, <span class="author">Jennum P</span>, <span class="author">Bourgin P</span>, <span class="author">Peraita-Adrados R</span>, <span class="author">Iranzo A</span>, <span class="author">Bassetti C</span>, <span class="author">Chen WM</span>, <span class="author">Concannon P</span>, <span class="author">Thompson SD</span>, <span class="author">Damotte V</span>, <span class="author">Fontaine B</span>, <span class="author">Breban M</span>, <span class="author">Gieger C</span>, <span class="author">Klopp N</span>, <span class="author">Deloukas P</span>, <span class="author">Wijmenga C</span>, <span class="author">Hallmayer J</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Rich SS</span>, <span class="author">Winkelmann J</span> and <span class="author">Mignot E</span></p>
<p class="affiliation">Center for Sleep Sciences and Medicine, Stanford University, Palo Alto, California, United States of America.</p>
<p class="abstract">Recent advances in the identification of susceptibility genes and environmental exposures provide broad support for a post-infectious autoimmune basis for narcolepsy/hypocretin (orexin) deficiency. We genotyped loci associated with other autoimmune and inflammatory diseases in 1,886 individuals with hypocretin-deficient narcolepsy and 10,421 controls, all of European ancestry, using a custom genotyping array (ImmunoChip). Three loci located outside the Human Leukocyte Antigen (HLA) region on chromosome 6 were significantly associated with disease risk. In addition to a strong signal in the T cell receptor alpha (TRA@), variants in two additional narcolepsy loci, Cathepsin H () and Tumor necrosis factor (ligand) superfamily member 4 (, also called ), attained genome-wide significance. These findings underline the importance of antigen presentation by HLA Class II to T cells in the pathophysiology of this autoimmune disease.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">e1003270</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23459209">23459209</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3573113">3573113</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003270">10.1371/journal.pgen.1003270</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A method for selectively enriching microbial DNA from contaminating vertebrate host DNA.</h4>
<p class="authors"><span class="author">Feehery GR</span>, <span class="author">Yigit E</span>, <span class="author">Oyola SO</span>, <span class="author">Langhorst BW</span>, <span class="author">Schmidt VT</span>, <span class="author">Stewart FJ</span>, <span class="author">Dimalanta ET</span>, <span class="author">Amaral-Zettler LA</span>, <span class="author">Davis T</span>, <span class="author">Quail MA</span> and <span class="author">Pradhan S</span></p>
<p class="affiliation">New England Biolabs Inc., Ipswich, Massachusetts, United States of America.</p><p class="abstract">DNA samples derived from vertebrate skin, bodily cavities and body fluids contain both host and microbial DNA; the latter often present as a minor component. Consequently, DNA sequencing of a microbiome sample frequently yields reads originating from the microbe(s) of interest, but with a vast excess of host genome-derived reads. In this study, we used a methyl-CpG binding domain (MBD) to separate methylated host DNA from microbial DNA based on differences in CpG methylation density. MBD fused to the Fc region of a human antibody (MBD-Fc) binds strongly to protein A paramagnetic beads, forming an effective one-step enrichment complex that was used to remove human or fish host DNA from bacterial and protistan DNA for subsequent sequencing and analysis. We report enrichment of DNA samples from human saliva, human blood, a mock malaria-infected blood sample and a black molly fish. When reads were mapped to reference genomes, sequence reads aligning to host genomes decreased 50-fold, while bacterial and Plasmodium DNA sequences reads increased 8-11.5-fold. The Shannon-Wiener diversity index was calculated for 149 bacterial species in saliva before and after enrichment. Unenriched saliva had an index of 4.72, while the enriched sample had an index of 4.80. The similarity of these indices demonstrates that bacterial species diversity and relative phylotype abundance remain conserved in enriched samples. Enrichment using the MBD-Fc method holds promise for targeted microbiome sequence analysis across a broad range of sample types.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079355/Z/06/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e76096</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24204593">24204593</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3810253">3810253</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0076096">10.1371/journal.pone.0076096</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reprogramming by cell fusion: boosted by tets.</h4>
<p class="authors"><span class="author">Ficz G</span> and <span class="author">Reik W</span></p>
<p class="affiliation">Epigenetics Programme, The Babraham Institute, Cambridge CB22 3AT, UK.</p>
<p class="abstract">Pluripotent cells, when fused with somatic cells, have the dominant ability to reprogram the somatic genome. Work by Piccolo et al. (2013) shows that the Tet1 and Tet2 hydroxylases are important for DNA methylation reprogramming of pluripotency genes and parental imprints.</p>
<p class="publication"><span class="title">Molecular cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">49</span>;<span class="number">6</span>;<span class="pagerange">1017-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23541036">23541036</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molcel.2013.03.014">10.1016/j.molcel.2013.03.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FGF signaling inhibition in ESCs drives rapid genome-wide demethylation to the epigenetic ground state of pluripotency.</h4>
<p class="authors"><span class="author">Ficz G</span>, <span class="author">Hore TA</span>, <span class="author">Santos F</span>, <span class="author">Lee HJ</span>, <span class="author">Dean W</span>, <span class="author">Arand J</span>, <span class="author">Krueger F</span>, <span class="author">Oxley D</span>, <span class="author">Paul YL</span>, <span class="author">Walter J</span>, <span class="author">Cook SJ</span>, <span class="author">Andrews S</span>, <span class="author">Branco MR</span> and <span class="author">Reik W</span></p>
<p class="affiliation">Epigenetics Programme, The Babraham Institute, Cambridge, CB22 3AT, UK. <a href="mailto:%67%61%62%72%69%65%6C%6C%61.%66%69%63%7A%40%62%61%62%72%61%68%61%6D.%61%63.%75%6B">&#103;&#97;&#98;&#114;&#105;&#101;&#108;&#108;&#97;&#46;&#102;&#105;&#99;&#122;&#64;&#98;&#97;&#98;&#114;&#97;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genome-wide erasure of DNA methylation takes place in primordial germ cells (PGCs) and early embryos and is linked with pluripotency. Inhibition of Erk1/2 and Gsk3&beta; signaling in mouse embryonic stem cells (ESCs) by small-molecule inhibitors (called 2i) has recently been shown to induce hypomethylation. We show by whole-genome bisulphite sequencing that 2i induces rapid and genome-wide demethylation on a scale and pattern similar to that in migratory PGCs and early embryos. Major satellites, intracisternal A particles (IAPs), and&nbsp;imprinted genes remain relatively resistant to&nbsp;erasure. Demethylation involves oxidation of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), impaired maintenance of 5mC and 5hmC, and repression of&nbsp;the de novo methyltransferases (Dnmt3a and Dnmt3b) and Dnmt3L. We identify a Prdm14- and Nanog-binding cis-acting regulatory region in Dnmt3b that is highly responsive to signaling. These insights provide a framework for understanding how signaling pathways regulate reprogramming to an epigenetic ground state of pluripotency.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/E/B/000C0417</span>; Medical Research Council: <span class="grant">G0801156</span>, <span class="grant">G0801727</span>; Wellcome Trust: <span class="grant">095645</span></p>
<p class="publication"><span class="title">Cell stem cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="number">3</span>;<span class="pagerange">351-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23850245">23850245</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3765959">3765959</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.stem.2013.06.004">10.1016/j.stem.2013.06.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global analysis of the sporulation pathway of Clostridium difficile.</h4>
<p class="authors"><span class="author">Fimlaid KA</span>, <span class="author">Bond JP</span>, <span class="author">Schutz KC</span>, <span class="author">Putnam EE</span>, <span class="author">Leung JM</span>, <span class="author">Lawley TD</span> and <span class="author">Shen A</span></p>
<p class="affiliation">Department of Microbiology and Molecular Genetics, University of Vermont, Burlington, Vermont, USA.</p><p class="abstract">The Gram-positive, spore-forming pathogen Clostridium difficile is the leading definable cause of healthcare-associated diarrhea worldwide. C. difficile infections are difficult to treat because of their frequent recurrence, which can cause life-threatening complications such as pseudomembranous colitis. The spores of C. difficile are responsible for these high rates of recurrence, since they are the major transmissive form of the organism and resistant to antibiotics and many disinfectants. Despite the importance of spores to the pathogenesis of C. difficile, little is known about their composition or formation. Based on studies in Bacillus subtilis and other Clostridium spp., the sigma factors &sigma;(F), &sigma;(E), &sigma;(G), and &sigma;(K) are predicted to control the transcription of genes required for sporulation, although their specific functions vary depending on the organism. In order to determine the roles of &sigma;(F), &sigma;(E), &sigma;(G), and &sigma;(K) in regulating C. difficile sporulation, we generated loss-of-function mutations in genes encoding these sporulation sigma factors and performed RNA-Sequencing to identify specific sigma factor-dependent genes. This analysis identified 224 genes whose expression was collectively activated by sporulation sigma factors: 183 were &sigma;(F)-dependent, 169 were &sigma;(E)-dependent, 34 were &sigma;(G)-dependent, and 31 were &sigma;(K)-dependent. In contrast with B. subtilis, C. difficile &sigma;(E) was dispensable for &sigma;(G) activation, &sigma;(G) was dispensable for &sigma;(K) activation, and &sigma;(F) was required for post-translationally activating &sigma;(G). Collectively, these results provide the first genome-wide transcriptional analysis of genes induced by specific sporulation sigma factors in the Clostridia and highlight that diverse mechanisms regulate sporulation sigma factor activity in the Firmicutes.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P20RR021905</span>; NIGMS NIH HHS: <span class="grant">P30 GM103498</span>, <span class="grant">R00GM092934</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">8</span>;<span class="pagerange">e1003660</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23950727">23950727</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3738446">3738446</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003660">10.1371/journal.pgen.1003660</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EMu: probabilistic inference of mutational processes and their localization in the cancer genome.</h4>
<p class="authors"><span class="author">Fischer A</span>, <span class="author">Illingworth CJ</span>, <span class="author">Campbell PJ</span> and <span class="author">Mustonen V</span></p><p class="abstract">The spectrum of mutations discovered in cancer genomes can be explained by the activity of a few elementary mutational processes. We present a novel probabilistic method, EMu, to infer the mutational signatures of these processes from a collection of sequenced tumors. EMu naturally incorporates the tumor-specific opportunity for different mutation types according to sequence composition. Applying EMu to breast cancer data, we derive detailed maps of the activity of each process, both genome-wide and within specific local regions of the genome. Our work provides new opportunities to study the mutational processes underlying cancer development. EMu is available at http://www.sanger.ac.uk/resources/software/emu/.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">4</span>;<span class="pagerange">R39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23628380">23628380</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3663107">3663107</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-4-r39">10.1186/gb-2013-14-4-r39</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2013.</h4>
<p class="authors"><span class="author">Flicek P</span>, <span class="author">Ahmed I</span>, <span class="author">Amode MR</span>, <span class="author">Barrell D</span>, <span class="author">Beal K</span>, <span class="author">Brent S</span>, <span class="author">Carvalho-Silva D</span>, <span class="author">Clapham P</span>, <span class="author">Coates G</span>, <span class="author">Fairley S</span>, <span class="author">Fitzgerald S</span>, <span class="author">Gil L</span>, <span class="author">Garc&iacute;a-Gir&oacute;n C</span>, <span class="author">Gordon L</span>, <span class="author">Hourlier T</span>, <span class="author">Hunt S</span>, <span class="author">Juettemann T</span>, <span class="author">K&auml;h&auml;ri AK</span>, <span class="author">Keenan S</span>, <span class="author">Komorowska M</span>, <span class="author">Kulesha E</span>, <span class="author">Longden I</span>, <span class="author">Maurel T</span>, <span class="author">McLaren WM</span>, <span class="author">Muffato M</span>, <span class="author">Nag R</span>, <span class="author">Overduin B</span>, <span class="author">Pignatelli M</span>, <span class="author">Pritchard B</span>, <span class="author">Pritchard E</span>, <span class="author">Riat HS</span>, <span class="author">Ritchie GR</span>, <span class="author">Ruffier M</span>, <span class="author">Schuster M</span>, <span class="author">Sheppard D</span>, <span class="author">Sobral D</span>, <span class="author">Taylor K</span>, <span class="author">Thormann A</span>, <span class="author">Trevanion S</span>, <span class="author">White S</span>, <span class="author">Wilder SP</span>, <span class="author">Aken BL</span>, <span class="author">Birney E</span>, <span class="author">Cunningham F</span>, <span class="author">Dunham I</span>, <span class="author">Harrow J</span>, <span class="author">Herrero J</span>, <span class="author">Hubbard TJ</span>, <span class="author">Johnson N</span>, <span class="author">Kinsella R</span>, <span class="author">Parker A</span>, <span class="author">Spudich G</span>, <span class="author">Yates A</span>, <span class="author">Zadissa A</span> and <span class="author">Searle SM</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton Cambridge CB10 1SD, UK. <a href="mailto:%66%6C%69%63%65%6B%40%65%62%69.%61%63.%75%6B">&#102;&#108;&#105;&#99;&#101;&#107;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl project (http://www.ensembl.org) provides genome information for sequenced chordate genomes with a particular focus on human, mouse, zebrafish and rat. Our resources include evidenced-based gene sets for all supported species; large-scale whole genome multiple species alignments across vertebrates and clade-specific alignments for eutherian mammals, primates, birds and fish; variation data resources for 17 species and regulation annotations based on ENCODE and other data sets. Ensembl data are accessible through the genome browser at http://www.ensembl.org and through other tools and programmatic interfaces.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/I025328/1</span>, <span class="grant">BB/I025506/1</span>; NHGRI NIH HHS: <span class="grant">U01HG004695</span>, <span class="grant">U41HG006104</span>, <span class="grant">U54HG004563</span>; Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT062023</span>, <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span>;<span class="volume">41</span>;<span class="number">Database issue</span>;<span class="pagerange">D48-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23203987">23203987</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3531136">3531136</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgks1236">10.1093/nar/gks1236</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Spindle checkpoint deficiency is tolerated by murine epidermal cells but not hair follicle stem cells.</h4>
<p class="authors"><span class="author">Foijer F</span>, <span class="author">Ditommaso T</span>, <span class="author">Donati G</span>, <span class="author">Hautaviita K</span>, <span class="author">Xie SZ</span>, <span class="author">Heath E</span>, <span class="author">Smyth I</span>, <span class="author">Watt FM</span>, <span class="author">Sorger PK</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Mouse Genomics, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, United Kingdom.</p>
<p class="abstract">The spindle assembly checkpoint (SAC) ensures correct chromosome segregation during mitosis by preventing aneuploidy, an event that is detrimental to the fitness and survival of normal cells but oncogenic in tumor cells. Deletion of SAC genes is incompatible with early mouse development, and RNAi-mediated depletion of SAC components in cultured cells results in rapid death. Here we describe the use of a conditional KO of mouse Mad2, an essential component of the SAC signaling cascade, as a means to selectively induce chromosome instability and aneuploidy in the epidermis of the skin. We observe that SAC inactivation is tolerated by interfollicular epidermal cells but results in depletion of hair follicle bulge stem cells. Eventually, a histologically normal epidermis develops within âˆ¼1 mo after birth, albeit without any hair. Mad2-deficient cells in this epidermis exhibited abnormal transcription of metabolic genes, consistent with aneuploid cell state. Hair follicle bulge stem cells were completely absent, despite the continued presence of rudimentary hair follicles. These data demonstrate that different cell lineages within a single tissue respond differently to chromosome instability: some proliferating cell lineages can survive, but stem cells are highly sensitive.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382243">23382243</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1217388110">10.1073/pnas.1217388110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome Sequence of Klebsiella pneumoniae Ecl8, a Reference Strain for Targeted Genetic Manipulation.</h4>
<p class="authors"><span class="author">Fookes M</span>, <span class="author">Yu J</span>, <span class="author">De Majumdar S</span>, <span class="author">Thomson N</span> and <span class="author">Schneiders T</span></p>
<p class="affiliation">Wellcome Trust, Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">We report the genome sequence of Klebsiella pneumoniae subsp. pneumoniae Ecl8, a spontaneous streptomycin-resistant mutant of strain ECL4, derived from NCIB 418. K. pneumoniae Ecl8 has been shown to be genetically tractable for targeted gene deletion strategies and so provides a platform for in-depth analyses of this species.</p>
<p class="publication"><span class="title">Genome announcements</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23405357">23405357</a>; DOI: <a href="http://dx.doi.org/10.1128%2FgenomeA.00027-12">10.1128/genomeA.00027-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global Analysis of Apicomplexan Protein S-Acyl Transferases Reveals an Enzyme Essential for Invasion.</h4>
<p class="authors"><span class="author">Fr&eacute;nal K</span>, <span class="author">Tay CL</span>, <span class="author">Mueller C</span>, <span class="author">Bushell ES</span>, <span class="author">Jia Y</span>, <span class="author">Graindorge A</span>, <span class="author">Billker O</span>, <span class="author">Rayner JC</span> and <span class="author">Soldati-Favre D</span></p>
<p class="affiliation">Department of Microbiology and Molecular Medicine, CMU, University of Geneva, Rue Michel-Servet 1, CH-1211, Geneva 4, Switzerland.</p>
<p class="abstract">The advent of techniques to study palmitoylation on a whole proteome scale has revealed that it is an important reversible modification that plays a role in regulating multiple biological processes. Palmitoylation can control the affinity of a protein for lipid membranes, which allows it to impact protein trafficking, stability, folding, signalling and interactions. The publication of the palmitome of the schizont stage of Plasmodium falciparum implicated a role for palmitoylation in host cell invasion, protein export and organelle biogenesis. However, nothing is known so far about the repertoire of protein S-acyl transferases (PATs) that catalyse this modification in Apicomplexa. We undertook a comprehensive analysis of the repertoire of Asp-His-His-Cys cysteine-rich domain (DHHC-CRD) PAT family in Toxoplasma gondii and Plasmodium berghei by assessing their localization and essentiality. Unlike functional redundancies reported in other eukaryotes, some apicomplexan-specific DHHCs are essential for parasite growth, and several are targeted to organelles unique to this phylum. Of particular interest is DHHC7, which localizes to rhoptry organelles in all parasites tested, including the major human pathogen P. falciparum. TgDHHC7 interferes with the localization of the rhoptry palmitoylated protein TgARO and affects the apical positioning of the rhoptry organelles. This PAT has a major impact on T. gondii host cell invasion, but not on the parasite's ability to egress.</p>
<p class="publication"><span class="title">Traffic (Copenhagen, Denmark)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23638681">23638681</a>; DOI: <a href="http://dx.doi.org/10.1111%2Ftra.12081">10.1111/tra.12081</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clonal Expansion Analysis of Transposon Insertions by High-Throughput Sequencing Identifies Candidate Cancer Genes in a PiggyBac Mutagenesis Screen.</h4>
<p class="authors"><span class="author">Friedel RH</span>, <span class="author">Friedel CC</span>, <span class="author">Bonfert T</span>, <span class="author">Shi R</span>, <span class="author">Rad R</span> and <span class="author">Soriano P</span></p>
<p class="affiliation">Department of Neuroscience, Department of Developmental and Regenerative Biology, Department of Neurosurgery, Icahn School of Medicine at Mount, Sinai, New York, New York, United States of America.</p>
<p class="abstract">Somatic transposon mutagenesis in mice is an efficient strategy to investigate the genetic mechanisms of tumorigenesis. The identification of tumor driving transposon insertions traditionally requires the generation of large tumor cohorts to obtain information about common insertion sites. Tumor driving insertions are also characterized by their clonal expansion in tumor tissue, a phenomenon that is facilitated by the slow and evolving transformation process of transposon mutagenesis. We describe here an improved approach for the detection of tumor driving insertions that assesses the clonal expansion of insertions by quantifying the relative proportion of sequence reads obtained in individual tumors. To this end, we have developed a protocol for insertion site sequencing that utilizes acoustic shearing of tumor DNA and Illumina sequencing. We analyzed various solid tumors generated by PiggyBac mutagenesis and for each tumor >10(6) reads corresponding to >10(4) insertion sites were obtained. In each tumor, 9 to 25 insertions stood out by their enriched sequence read frequencies when compared to frequencies obtained from tail DNA controls. These enriched insertions are potential clonally expanded tumor driving insertions, and thus identify candidate cancer genes. The candidate cancer genes of our study comprised many established cancer genes, but also novel candidate genes such as Mastermind-like1 (Mamld1) and Diacylglycerolkinase delta (Dgkd). We show that clonal expansion analysis by high-throughput sequencing is a robust approach for the identification of candidate cancer genes in insertional mutagenesis screens on the level of individual tumors.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">e72338</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23940809">23940809</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0072338">10.1371/journal.pone.0072338</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A CpG Mutational Hotspot in a ONECUT Binding Site Accounts for the Prevalent Variant of Hemophilia B Leyden.</h4>
<p class="authors"><span class="author">Funnell AP</span>, <span class="author">Wilson MD</span>, <span class="author">Ballester B</span>, <span class="author">Mak KS</span>, <span class="author">Burdach J</span>, <span class="author">Magan N</span>, <span class="author">Pearson RC</span>, <span class="author">Lemaigre FP</span>, <span class="author">Stowell KM</span>, <span class="author">Odom DT</span>, <span class="author">Flicek P</span> and <span class="author">Crossley M</span></p>
<p class="affiliation">School of Biotechnology and Biomolecular Sciences, University of New South Wales, Kensington NSW 2052, Australia.</p>
<p class="abstract">Hemophilia B, or the "royal disease," arises from mutations in coagulation factor IX (F9). Mutations within the F9 promoter are associated with a remarkable hemophilia B subtype, termed hemophilia B Leyden, in which symptoms ameliorate after puberty. Mutations at the -5/-6 site (nucleotides -5 and -6 relative to the transcription start site, designated +1) account for the majority of Leyden cases and have been postulated to disrupt the binding of a transcriptional activator, the identity of which has remained elusive for more than 20 years. Here, we show that ONECUT transcription factors (ONECUT1 and ONECUT2) bind to the -5/-6 site. The various hemophilia B Leyden mutations that have been reported in this site inhibit ONECUT binding to varying degrees, which correlate well with their associated clinical severities. In addition, expression of F9 is crucially dependent on ONECUT factors in vivo, and as such, mice deficient in ONECUT1, ONECUT2, or both exhibit depleted levels of F9. Taken together, our findings establish ONECUT transcription factors as the missing hemophilia B Leyden regulators that operate through the -5/-6 site.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">92</span>;<span class="number">3</span>;<span class="pagerange">460-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23472758">23472758</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3591849">3591849</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.02.003">10.1016/j.ajhg.2013.02.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global properties and functional complexity of human gene regulatory variation.</h4>
<p class="authors"><span class="author">Gaffney DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, United Kingdom. <a href="mailto:%64%67%31%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#103;&#49;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Identification and functional interpretation of gene regulatory variants is a major focus of modern genomics. The application of genetic mapping to molecular and cellular traits has enabled the detection of regulatory variation on genome-wide scales and revealed an enormous diversity of regulatory architecture in humans and other species. In this review I summarise the insights gained and questions raised by a decade of genetic mapping of gene expression variation. I discuss recent extensions of this approach using alternative molecular phenotypes that have revealed some of the biological mechanisms that drive gene expression variation between individuals. Finally, I highlight outstanding problems and future directions for development.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">e1003501</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23737752">23737752</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3667745">3667745</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003501">10.1371/journal.pgen.1003501</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An elephantine viral problem.</h4>
<p class="authors"><span class="author">Gall A</span> and <span class="author">Palser A</span></p><p class="abstract">This month&#39;s Genome Watch highlights how deep sequencing was used to generate the first full genomes of herpesviruses associated with a fatal disease in elephants.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">8</span>;<span class="pagerange">512</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23832239">23832239</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3075">10.1038/nrmicro3075</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Restriction of V3 region sequence divergence in the HIV-1 envelope gene during antiretroviral treatment in a cohort of recent seroconverters.</h4>
<p class="authors"><span class="author">Gall A</span>, <span class="author">Kaye S</span>, <span class="author">Hu&eacute; S</span>, <span class="author">Bonsall D</span>, <span class="author">Rance R</span>, <span class="author">Baillie GJ</span>, <span class="author">Fidler SJ</span>, <span class="author">Weber JN</span>, <span class="author">McClure MO</span>, <span class="author">Kellam P</span> and <span class="author">SPARTAC Trial Investigators</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Background</strong>: Dynamic changes in Human Immunodeficiency Virus 1 (HIV-1) sequence diversity and divergence are associated with immune control during primary infection and progression to AIDS. Consensus sequencing or single genome amplification sequencing of the HIV-1 envelope (env) gene, in particular the variable (V) regions, is used as a marker for HIV-1 genome diversity, but population diversity is only minimally, or semi-quantitatively sampled using these methods.</p><p class="abstract"><strong>Results</strong>: Here we use second generation deep sequencing to determine inter-and intra-patient sequence heterogeneity and to quantify minor variants in a cohort of individuals either receiving or not receiving antiretroviral treatment following seroconversion; the SPARTAC trial. We show, through a cross-sectional study of sequence diversity of the env V3 in 30 antiretroviral-naive patients during primary infection that considerable population structure diversity exists, with some individuals exhibiting highly constrained plasma virus diversity. Diversity was independent of clinical markers (viral load, time from seroconversion, CD4 cell count) of infection. Serial sampling over 60 weeks of non-treated individuals that define three initially different diversity profiles showed that complex patterns of continuing HIV-1 sequence diversification and divergence could be readily detected. Evidence for minor sequence turnover, emergence of new variants and re-emergence of archived variants could be inferred from this analysis. Analysis of viral divergence over the same time period in patients who received short (12 weeks, ART12) or long course antiretroviral therapy (48 weeks, ART48) and a non-treated control group revealed that ART48 successfully suppressed viral divergence while ART12 did not have a significant effect.</p><p class="abstract"><strong>Conclusions</strong>: Deep sequencing is a sensitive and reliable method for investigating the diversity of the env V3 as an important component of HIV-1 genome diversity. Detailed insights into the complex early intra-patient dynamics of env V3 diversity and divergence were explored in antiretroviral-na&iuml;ve recent seroconverters. Long course antiretroviral therapy, initiated soon after seroconversion and administered for 48 weeks, restricts HIV-1 divergence significantly. The effect of ART12 and ART48 on clinical markers of HIV infection and progression is currently investigated in the SPARTAC trial.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI046995</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Retrovirology</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="pagerange">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23331949">23331949</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605130">3605130</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1742-4690-10-8">10.1186/1742-4690-10-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reprogramming to pluripotency using designer TALE transcription factors targeting enhancers.</h4>
<p class="authors"><span class="author">Gao X</span>, <span class="author">Yang J</span>, <span class="author">Tsang JC</span>, <span class="author">Ooi J</span>, <span class="author">Wu D</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">The modular DNA recognition code of the transcription-activator-like effectors (TALEs) from plant pathogenic bacterial genus Xanthomonas provides a powerful genetic tool to create designer transcription factors (dTFs) targeting specific DNA sequences for manipulating gene expression. Previous studies have suggested critical roles of enhancers in gene regulation and reprogramming. Here, we report dTF activator targeting the distal enhancer of the Pou5f1 (Oct4) locus induces epigenetic changes, reactivates its expression, and substitutes exogenous OCT4 in reprogramming mouse embryonic fibroblast cells (MEFs) to induced pluripotent stem cells (iPSCs). Similarly, dTF activator targeting a Nanog enhancer activates Nanog expression and reprograms epiblast stem cells (EpiSCs) to iPSCs. Conversely, dTF repressors targeting the same genetic elements inhibit expression of these loci, and effectively block reprogramming. This study indicates that dTFs targeting specific enhancers can be used to study other biological processes such as transdifferentiation or directed differentiation of stem cells.</p>
<p class="publication"><span class="title">Stem cell reports</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">2</span>;<span class="pagerange">183-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24052952">24052952</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3757749">3757749</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.stemcr.2013.06.002">10.1016/j.stemcr.2013.06.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide haplotype analysis of cis expression quantitative trait Loci in monocytes.</h4>
<p class="authors"><span class="author">Garnier S</span>, <span class="author">Truong V</span>, <span class="author">Brocheton J</span>, <span class="author">Zeller T</span>, <span class="author">Rovital M</span>, <span class="author">Wild PS</span>, <span class="author">Ziegler A</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">Munzel T</span>, <span class="author">Tiret L</span>, <span class="author">Blankenberg S</span>, <span class="author">Deloukas P</span>, <span class="author">Erdmann J</span>, <span class="author">Hengstenberg C</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Ouwehand WH</span>, <span class="author">Goodall AH</span>, <span class="author">Cambien F</span> and <span class="author">Tr&eacute;gou&euml;t DA</span></p>
<p class="affiliation">INSERM, UMR_S 937, Pierre and Marie Curie University (UPMC, Paris 6), Paris, France ; ICAN Institute for Cardiometabolism and Nutrition, Pierre and Marie Curie University (UPMC, Paris 6), Paris, France.</p>
<p class="abstract">In order to assess whether gene expression variability could be influenced by several SNPs acting in cis, either through additive or more complex haplotype effects, a systematic genome-wide search for cis haplotype expression quantitative trait loci (eQTL) was conducted in a sample of 758 individuals, part of the Cardiogenics Transcriptomic Study, for which genome-wide monocyte expression and GWAS data were available. 19,805 RNA probes were assessed for cis haplotypic regulation through investigation of âˆ¼2,1Ã—10(9) haplotypic combinations. 2,650 probes demonstrated haplotypic p-valuesâ€‰>10(4)-fold smaller than the best single SNP p-value. Replication of significant haplotype effects were tested for 412 probes for which SNPs (or proxies) that defined the detected haplotypes were available in the Gutenberg Health Study composed of 1,374 individuals. At the Bonferroni correction level of 1.2Ã—10(-4) (âˆ¼0.05/412), 193 haplotypic signals replicated. 1000G imputation was then conducted, and 105 haplotypic signals still remained more informative than imputed SNPs. In-depth analysis of these 105 cis eQTL revealed that at 76 loci genetic associations were compatible with additive effects of several SNPs, while for the 29 remaining regions data could be compatible with a more complex haplotypic pattern. As 24 of the 105 cis eQTL have previously been reported to be disease-associated loci, this work highlights the need for conducting haplotype-based and 1000G imputed cis eQTL analysis before commencing functional studies at disease-associated loci.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">e1003240</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382694">23382694</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3561129">3561129</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003240">10.1371/journal.pgen.1003240</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sleeping Beauty mutagenesis in a mouse medulloblastoma model defines networks that discriminate between human molecular subgroups.</h4>
<p class="authors"><span class="author">Genovesi LA</span>, <span class="author">Ng CG</span>, <span class="author">Davis MJ</span>, <span class="author">Remke M</span>, <span class="author">Taylor MD</span>, <span class="author">Adams DJ</span>, <span class="author">Rust AG</span>, <span class="author">Ward JM</span>, <span class="author">Ban KH</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span> and <span class="author">Wainwright BJ</span></p>
<p class="affiliation">Institute for Molecular Bioscience, The University of Queensland, St. Lucia, QLD 4072, Australia.</p>
<p class="abstract">The Sleeping Beauty (SB) transposon mutagenesis screen is a powerful tool to facilitate the discovery of cancer genes that drive tumorigenesis in mouse models. In this study, we sought to identify genes that functionally cooperate with sonic hedgehog signaling to initiate medulloblastoma (MB), a tumor of the cerebellum. By combining SB mutagenesis with Patched1 heterozygous mice (Ptch1(lacZ/+)), we observed an increased frequency of MB and decreased tumor-free survival compared with Ptch1(lacZ/+) controls. From an analysis of 85 tumors, we identified 77 common insertion sites that map to 56 genes potentially driving increased tumorigenesis. The common insertion site genes identified in the mutagenesis screen were mapped to human orthologs, which were used to select probes and corresponding expression data from an independent set of previously described human MB samples, and surprisingly were capable of accurately clustering known molecular subgroups of MB, thereby defining common regulatory networks underlying all forms of MB irrespective of subgroup. We performed a network analysis to discover the likely mechanisms of action of subnetworks and used an in vivo model to confirm a role for a highly ranked candidate gene, Nfia, in promoting MB formation. Our analysis implicates candidate cancer genes in the deregulation of apoptosis and translational elongation, and reveals a strong signature of transcriptional regulation that will have broad impact on expression programs in MB. These networks provide functional insights into the complex biology of human MB and identify potential avenues for intervention common to all clinical subgroups.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">46</span>;<span class="pagerange">E4325-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24167280">24167280</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1318639110">10.1073/pnas.1318639110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression changes with age in skin, adipose tissue, blood and brain.</h4>
<p class="authors"><span class="author">Glass D</span>, <span class="author">Vi&ntilde;uela A</span>, <span class="author">Davies MN</span>, <span class="author">Ramasamy A</span>, <span class="author">Parts L</span>, <span class="author">Knowles D</span>, <span class="author">Brown AA</span>, <span class="author">Hedman AK</span>, <span class="author">Small KS</span>, <span class="author">Buil A</span>, <span class="author">Grundberg E</span>, <span class="author">Nica AC</span>, <span class="author">Meglio P</span>, <span class="author">Nestle FO</span>, <span class="author">Ryten M</span>, <span class="author">the UK Brain Expression consortium</span>, <span class="author">the MuTHER consortium</span>, <span class="author">Durbin R</span>, <span class="author">McCarthy MI</span>, <span class="author">Deloukas P</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Weale ME</span>, <span class="author">Bataille V</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King's College London, St Thomas' Campus, Westminster Bridge Road, London SE1 7EH, UK. <a href="mailto:%62%61%74%61%69%6C%6C%65%40%64%6F%63%74%6F%72%73.%6F%72%67.%75%6B">&#98;&#97;&#116;&#97;&#105;&#108;&#108;&#101;&#64;&#100;&#111;&#99;&#116;&#111;&#114;&#115;&#46;&#111;&#114;&#103;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: Previous studies have demonstrated that gene expression levels change with age. These changes are hypothesized to influence the aging rate of an individual. We analyzed gene expression changes with age in abdominal skin, subcutaneous adipose tissue and lymphoblastoid cell lines in 856 female twins in the age range of 39-85 years. Additionally, we investigated genotypic variants involved in genotype-by-age interactions to understand how the genomic regulation of gene expression alters with age.

Results: Using a linear mixed model, differential expression with age was identified in 1,672 genes in skin and 188 genes in adipose tissue. Only two genes expressed in lymphoblastoid cell lines showed significant changes with age. Genes significantly regulated by age were compared with expression profiles in 10 brain regions from 100 postmortem brains aged 16 to 83 years. We identified only one age-related gene common to the three tissues. There were 12 genes that showed differential expression with age in both skin and brain tissue and three common to adipose and brain tissues.

Conclusions: Skin showed the most age-related gene expression changes of all the tissues investigated, with many of the genes being previously implicated in fatty acid metabolism, mitochondrial activity, cancer and splicing. A significant proportion of age-related changes in gene expression appear to be tissue-specific with only a few genes sharing an age effect in expression across tissues. More research is needed to improve our understanding of the genetic influences on aging and the relationship with age-related diseases.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">7</span>;<span class="pagerange">R75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23889843">23889843</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-7-r75">10.1186/gb-2013-14-7-r75</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovery and refinement of loci associated with lipid levels.</h4>
<p class="authors"><span class="author">Global Lipids Genetics Consortium</span>, <span class="author">Willer CJ</span>, <span class="author">Schmidt EM</span>, <span class="author">Sengupta S</span>, <span class="author">Peloso GM</span>, <span class="author">Gustafsson S</span>, <span class="author">Kanoni S</span>, <span class="author">Ganna A</span>, <span class="author">Chen J</span>, <span class="author">Buchkovich ML</span>, <span class="author">Mora S</span>, <span class="author">Beckmann JS</span>, <span class="author">Bragg-Gresham JL</span>, <span class="author">Chang HY</span>, <span class="author">Demirkan A</span>, <span class="author">Den Hertog HM</span>, <span class="author">Do R</span>, <span class="author">Donnelly LA</span>, <span class="author">Ehret GB</span>, <span class="author">Esko T</span>, <span class="author">Feitosa MF</span>, <span class="author">Ferreira T</span>, <span class="author">Fischer K</span>, <span class="author">Fontanillas P</span>, <span class="author">Fraser RM</span>, <span class="author">Freitag DF</span>, <span class="author">Gurdasani D</span>, <span class="author">Heikkil&auml; K</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Isaacs A</span>, <span class="author">Jackson AU</span>, <span class="author">Johansson A</span>, <span class="author">Johnson T</span>, <span class="author">Kaakinen M</span>, <span class="author">Kettunen J</span>, <span class="author">Kleber ME</span>, <span class="author">Li X</span>, <span class="author">Luan J</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Magnusson PK</span>, <span class="author">Mangino M</span>, <span class="author">Mihailov E</span>, <span class="author">Montasser ME</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Nolte IM</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Palmer CD</span>, <span class="author">Perola M</span>, <span class="author">Petersen AK</span>, <span class="author">Sanna S</span>, <span class="author">Saxena R</span>, <span class="author">Service SK</span>, <span class="author">Shah S</span>, <span class="author">Shungin D</span>, <span class="author">Sidore C</span>, <span class="author">Song C</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Surakka I</span>, <span class="author">Tanaka T</span>, <span class="author">Teslovich TM</span>, <span class="author">Thorleifsson G</span>, <span class="author">Van den Herik EG</span>, <span class="author">Voight BF</span>, <span class="author">Volcik KA</span>, <span class="author">Waite LL</span>, <span class="author">Wong A</span>, <span class="author">Wu Y</span>, <span class="author">Zhang W</span>, <span class="author">Absher D</span>, <span class="author">Asiki G</span>, <span class="author">Barroso I</span>, <span class="author">Been LF</span>, <span class="author">Bolton JL</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Brambilla P</span>, <span class="author">Burnett MS</span>, <span class="author">Cesana G</span>, <span class="author">Dimitriou M</span>, <span class="author">Doney AS</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Elliott P</span>, <span class="author">Epstein SE</span>, <span class="author">Eyjolfsson GI</span>, <span class="author">Gigante B</span>, <span class="author">Goodarzi MO</span>, <span class="author">Grallert H</span>, <span class="author">Gravito ML</span>, <span class="author">Groves CJ</span>, <span class="author">Hallmans G</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hayward C</span>, <span class="author">Hernandez D</span>, <span class="author">Hicks AA</span>, <span class="author">Holm H</span>, <span class="author">Hung YJ</span>, <span class="author">Illig T</span>, <span class="author">Jones MR</span>, <span class="author">Kaleebu P</span>, <span class="author">Kastelein JJ</span>, <span class="author">Khaw KT</span>, <span class="author">Kim E</span>, <span class="author">Klopp N</span>, <span class="author">Komulainen P</span>, <span class="author">Kumari M</span>, <span class="author">Langenberg C</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Lin SY</span>, <span class="author">Lindstr&ouml;m J</span>, <span class="author">Loos RJ</span>, <span class="author">Mach F</span>, <span class="author">McArdle WL</span>, <span class="author">Meisinger C</span>, <span class="author">Mitchell BD</span>, <span class="author">M&uuml;ller G</span>, <span class="author">Nagaraja R</span>, <span class="author">Narisu N</span>, <span class="author">Nieminen TV</span>, <span class="author">Nsubuga RN</span>, <span class="author">Olafsson I</span>, <span class="author">Ong KK</span>, <span class="author">Palotie A</span>, <span class="author">Papamarkou T</span>, <span class="author">Pomilla C</span>, <span class="author">Pouta A</span>, <span class="author">Rader DJ</span>, <span class="author">Reilly MP</span>, <span class="author">Ridker PM</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rudan I</span>, <span class="author">Ruokonen A</span>, <span class="author">Samani N</span>, <span class="author">Scharnagl H</span>, <span class="author">Seeley J</span>, <span class="author">Silander K</span>, <span class="author">Stanc&aacute;kov&aacute; A</span>, <span class="author">Stirrups K</span>, <span class="author">Swift AJ</span>, <span class="author">Tiret L</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Pelt LJ</span>, <span class="author">Vedantam S</span>, <span class="author">Wainwright N</span>, <span class="author">Wijmenga C</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Wilsgaard T</span>, <span class="author">Wilson JF</span>, <span class="author">Young EH</span>, <span class="author">Zhao JH</span>, <span class="author">Adair LS</span>, <span class="author">Arveiler D</span>, <span class="author">Assimes TL</span>, <span class="author">Bandinelli S</span>, <span class="author">Bennett F</span>, <span class="author">Bochud M</span>, <span class="author">Boehm BO</span>, <span class="author">Boomsma DI</span>, <span class="author">Borecki IB</span>, <span class="author">Bornstein SR</span>, <span class="author">Bovet P</span>, <span class="author">Burnier M</span>, <span class="author">Campbell H</span>, <span class="author">Chakravarti A</span>, <span class="author">Chambers JC</span>, <span class="author">Chen YD</span>, <span class="author">Collins FS</span>, <span class="author">Cooper RS</span>, <span class="author">Danesh J</span>, <span class="author">Dedoussis G</span>, <span class="author">de Faire U</span>, <span class="author">Feranil AB</span>, <span class="author">Ferri&egrave;res J</span>, <span class="author">Ferrucci L</span>, <span class="author">Freimer NB</span>, <span class="author">Gieger C</span>, <span class="author">Groop LC</span>, <span class="author">Gudnason V</span>, <span class="author">Gyllensten U</span>, <span class="author">Hamsten A</span>, <span class="author">Harris TB</span>, <span class="author">Hingorani A</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Hofman A</span>, <span class="author">Hovingh GK</span>, <span class="author">Hsiung CA</span>, <span class="author">Humphries SE</span>, <span class="author">Hunt SC</span>, <span class="author">Hveem K</span>, <span class="author">Iribarren C</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Jula A</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kaprio J</span>, <span class="author">Kes&auml;niemi A</span>, <span class="author">Kivimaki M</span>, <span class="author">Kooner JS</span>, <span class="author">Koudstaal PJ</span>, <span class="author">Krauss RM</span>, <span class="author">Kuh D</span>, <span class="author">Kuusisto J</span>, <span class="author">Kyvik KO</span>, <span class="author">Laakso M</span>, <span class="author">Lakka TA</span>, <span class="author">Lind L</span>, <span class="author">Lindgren CM</span>, <span class="author">Martin NG</span>, <span class="author">M&auml;rz W</span>, <span class="author">McCarthy MI</span>, <span class="author">McKenzie CA</span>, <span class="author">Meneton P</span>, <span class="author">Metspalu A</span>, <span class="author">Moilanen L</span>, <span class="author">Morris AD</span>, <span class="author">Munroe PB</span>, <span class="author">Nj&oslash;lstad I</span>, <span class="author">Pedersen NL</span>, <span class="author">Power C</span>, <span class="author">Pramstaller PP</span>, <span class="author">Price JF</span>, <span class="author">Psaty BM</span>, <span class="author">Quertermous T</span>, <span class="author">Rauramaa R</span>, <span class="author">Saleheen D</span>, <span class="author">Salomaa V</span>, <span class="author">Sanghera DK</span>, <span class="author">Saramies J</span>, <span class="author">Schwarz PE</span>, <span class="author">Sheu WH</span>, <span class="author">Shuldiner AR</span>, <span class="author">Siegbahn A</span>, <span class="author">Spector TD</span>, <span class="author">Stefansson K</span>, <span class="author">Strachan DP</span>, <span class="author">Tayo BO</span>, <span class="author">Tremoli E</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uusitupa M</span>, <span class="author">van Duijn CM</span>, <span class="author">Vollenweider P</span>, <span class="author">Wallentin L</span>, <span class="author">Wareham NJ</span>, <span class="author">Whitfield JB</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Ordovas JM</span>, <span class="author">Boerwinkle E</span>, <span class="author">Palmer CN</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Chasman DI</span>, <span class="author">Rotter JI</span>, <span class="author">Franks PW</span>, <span class="author">Ripatti S</span>, <span class="author">Cupples LA</span>, <span class="author">Sandhu MS</span>, <span class="author">Rich SS</span>, <span class="author">Boehnke M</span>, <span class="author">Deloukas P</span>, <span class="author">Kathiresan S</span>, <span class="author">Mohlke KL</span>, <span class="author">Ingelsson E</span> and <span class="author">Abecasis GR</span></p>
<p class="affiliation">1] Department of Internal Medicine, Division of Cardiovascular Medicine, University of Michigan, Ann Arbor, Michigan, USA. [2] Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, Michigan, USA. [3] Department of Human Genetics, University of Michigan, Ann Arbor, Michigan, USA. [4] Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA. [5] [6].</p><p class="abstract">Levels of low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol, triglycerides and total cholesterol are heritable, modifiable risk factors for coronary artery disease. To identify new loci and refine known loci influencing these lipids, we examined 188,577 individuals using genome-wide and custom genotyping arrays. We identify and annotate 157 loci associated with lipid levels at P &lt; 5 &times; 10(-8), including 62 loci not previously associated with lipid levels in humans. Using dense genotyping in individuals of European, East Asian, South Asian and African ancestry, we narrow association signals in 12 loci. We find that loci associated with blood lipid levels are often associated with cardiovascular and metabolic traits, including coronary artery disease, type 2 diabetes, blood pressure, waist-hip ratio and body mass index. Our results demonstrate the value of using genetic data from individuals of diverse ancestry and provide insights into the biological mechanisms regulating blood lipids to guide future genetic, biological and therapeutic research.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/08/094/26019</span>, <span class="grant">RG/08/008/25291</span>, <span class="grant">RG/08/014/24067</span>; Chief Scientist Office: <span class="grant">CZB/4/672</span>, <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0801566</span>, <span class="grant">G0901213</span>, <span class="grant">G1000143</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U950080926</span>, <span class="grant">MR/K013351/1</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NHLBI NIH HHS: <span class="grant">R00 HL094535</span>, <span class="grant">R01 HL105756</span>, <span class="grant">R01 HL109946</span>, <span class="grant">U01 HL069757</span>; NIDDK NIH HHS: <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK072193</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">11</span>;<span class="pagerange">1274-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24097068">24097068</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3838666">3838666</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2797">10.1038/ng.2797</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clonal analyses reveal associations of JAK2V617F homozygosity with hematologic features, age and gender in polycythemia vera and essential thrombocythemia.</h4>
<p class="authors"><span class="author">Godfrey AL</span>, <span class="author">Chen E</span>, <span class="author">Pagano F</span>, <span class="author">Silber Y</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="affiliation"><a href="mailto:%61%72%67%31%30%30%30%40%63%61%6D.%61%63.%75%6B">&#97;&#114;&#103;&#49;&#48;&#48;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Subclones homozygous for JAK2V617F are more common and larger in patients with polycythemia vera compared to essential thrombocythemia, but their role in determining phenotype remains unclear. We genotyped 4564 erythroid colonies from 59 patients with polycythemia vera or essential thrombocythemia to investigate whether the proportion of JAK2V617F -homozygous precursors, compared to heterozygous precursors, is associated with clinical or demographic features. In polycythemia vera, a higher proportion of homozygous-mutant precursors was associated with more extreme blood counts at diagnosis, consistent with a causal role for homozygosity in polycythemia vera pathogenesis. Larger numbers of homozygous-mutant colonies were associated with older age, and with male gender in polycythemia vera but female gender in essential thrombocythemia. These results suggest that age promotes development or expansion of homozygous-mutant clones and that gender modulates the phenotypic consequences of JAK2V617F homozygosity, thus providing a potential explanation for the long-standing observations of a preponderance of men with polycythemia vera but of women with essential thrombocythemia.</p>
<p class="publication"><span class="title">Haematologica</span>&nbsp;<span class="year">2013</span>;<span class="volume">98</span>;<span class="number">5</span>;<span class="pagerange">718-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23633544">23633544</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3640115">3640115</a>; DOI: <a href="http://dx.doi.org/10.3324%2Fhaematol.2012.079129">10.3324/haematol.2012.079129</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptome analysis of human tissues and cell lines reveals one dominant transcript per gene.</h4>
<p class="authors"><span class="author">Gonz&agrave;lez-Porta M</span>, <span class="author">Frankish A</span>, <span class="author">Rung J</span>, <span class="author">Harrow J</span> and <span class="author">Brazma A</span></p>
<p class="affiliation">European Molecular Biology Laboratory - European Bioinformatics Institute, EMBL-EBI, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, United Kingdom. <a href="mailto:%62%72%61%7A%6D%61%40%65%62%69.%61%63.%75%6B">&#98;&#114;&#97;&#122;&#109;&#97;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: RNA sequencing has opened new avenues for the study of transcriptome composition. Significant evidence has accumulated showing that the human transcriptome contains in excess of a hundred thousand different transcripts. However, it is still not clear to what extent this diversity prevails when considering the relative abundances of different transcripts from the same gene.

Results: Here we show that, in a given condition, most protein coding genes have one major transcript expressed at significantly higher level than others, that in human tissues the major transcripts contribute almost 85 percent to the total mRNA from protein coding loci, and that often the same major transcript is expressed in many tissues. We detect a high degree of overlap between the set of major transcripts and a recently published set of alternatively spliced transcripts that are predicted to be translated utilizing proteomic data. Thus, we hypothesize that although some minor transcripts may play a functional role, the major ones are likely to be the main contributors to the proteome. However, we still detect a non-negligible fraction of protein coding genes for which the major transcript does not code a protein.

Conclusions: Overall, our findings suggest that the transcriptome from protein coding loci is dominated by one transcript per gene and that not all the transcripts that contribute to transcriptome diversity are equally likely to contribute to protein diversity. This observation can help to prioritize candidate targets in proteomics research and to predict the functional impact of the detected changes in variation studies.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">7</span>;<span class="pagerange">R70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23815980">23815980</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-7-r70">10.1186/gb-2013-14-7-r70</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Computational approaches to identify functional genetic variants in cancer genomes.</h4>
<p class="authors"><span class="author">Gonzalez-Perez A</span>, <span class="author">Mustonen V</span>, <span class="author">Reva B</span>, <span class="author">Ritchie GR</span>, <span class="author">Creixell P</span>, <span class="author">Karchin R</span>, <span class="author">Vazquez M</span>, <span class="author">Fink JL</span>, <span class="author">Kassahn KS</span>, <span class="author">Pearson JV</span>, <span class="author">Bader GD</span>, <span class="author">Boutros PC</span>, <span class="author">Muthuswamy L</span>, <span class="author">Ouellette BF</span>, <span class="author">Reimand J</span>, <span class="author">Linding R</span>, <span class="author">Shibata T</span>, <span class="author">Valencia A</span>, <span class="author">Butler A</span>, <span class="author">Dronov S</span>, <span class="author">Flicek P</span>, <span class="author">Shannon NB</span>, <span class="author">Carter H</span>, <span class="author">Ding L</span>, <span class="author">Sander C</span>, <span class="author">Stuart JM</span>, <span class="author">Stein LD</span>, <span class="author">Lopez-Bigas N</span> and <span class="author">International Cancer Genome Consortium Mutation Pathways and Consequences Subgroup of the Bioinformatics Analyses Working Group</span></p>
<p class="affiliation">Research Unit on Biomedical Informatics, University Pompeu Fabra, Barcelona, Spain.</p><p class="abstract">The International Cancer Genome Consortium (ICGC) aims to catalog genomic abnormalities in tumors from 50 different cancer types. Genome sequencing reveals hundreds to thousands of somatic mutations in each tumor but only a minority of these drive tumor progression. We present the result of discussions within the ICGC on how to address the challenge of identifying mutations that contribute to oncogenesis, tumor maintenance or response to therapy, and recommend computational techniques to annotate somatic variants and predict their impact on cancer phenotype.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01 CA180778</span>; NHGRI NIH HHS: <span class="grant">U01 HG006517</span>, <span class="grant">U54 HG003079</span>; Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">8</span>;<span class="pagerange">723-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23900255">23900255</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3919555">3919555</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.2562">10.1038/nmeth.2562</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in C10orf11, Encoding a Melanocyte-Differentiation Gene, Cause Autosomal-Recessive Albinism.</h4>
<p class="authors"><span class="author">Gr&oslash;nskov K</span>, <span class="author">Dooley CM</span>, <span class="author">Ostergaard E</span>, <span class="author">Kelsh RN</span>, <span class="author">Hansen L</span>, <span class="author">Levesque MP</span>, <span class="author">Vilhelmsen K</span>, <span class="author">M&oslash;llg&aring;rd K</span>, <span class="author">Stemple DL</span> and <span class="author">Rosenberg T</span></p>
<p class="affiliation">Applied Human Molecular Genetics, Kennedy Center, Copenhagen University Hospital, Rigshospitalet, DK-2100 Copenhagen, Denmark; Department of Cellular and Molecular Medicine, University of Copenhagen, DK-2200 Copenhagen, Denmark. Electronic address: <a href="mailto:%6B%61%72%65%6E.%67%72%6F%65%6E%73%6B%6F%76%40%72%65%67%69%6F%6E%68.%64%6B">&#107;&#97;&#114;&#101;&#110;&#46;&#103;&#114;&#111;&#101;&#110;&#115;&#107;&#111;&#118;&#64;&#114;&#101;&#103;&#105;&#111;&#110;&#104;&#46;&#100;&#107;</a>.</p>
<p class="abstract">Autosomal-recessive albinism is a hypopigmentation disorder with a broad phenotypic range. A substantial fraction of individuals with albinism remain genetically unresolved, and it has been hypothesized that more genes are to be identified. By using homozygosity mapping of an inbred Faroese family, we identified a 3.5 Mb homozygous region (10q22.2-q22.3) on chromosome 10. The region contains five protein-coding genes, and sequencing of one of these, C10orf11, revealed a nonsense mutation that segregated with the disease and showed a recessive inheritance pattern. Investigation of additional albinism-affected individuals from the Faroe Islands revealed that five out of eight unrelated affected persons had the nonsense mutation in C10orf11. Screening of a cohort of autosomal-recessive-albinism-affected individuals residing in Denmark showed a homozygous 1 bp duplication in C10orf11 in an individual originating from Lithuania. Immunohistochemistry showed localization of C10orf11 in melanoblasts and melanocytes in human fetal tissue, but no localization was seen in retinal pigment epithelial cells. Knockdown of the zebrafish (Danio rerio) homolog with the use of morpholinos resulted in substantially decreased pigmentation and a reduction of the apparent number of pigmented melanocytes. The morphant phenotype was rescued by wild-type C10orf11, but not by mutant C10orf11. In conclusion, we have identified a melanocyte-differentiation gene, C10orf11, which when mutated causes autosomal-recessive albinism in humans.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23395477">23395477</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.01.006">10.1016/j.ajhg.2013.01.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Examination of the relationship between variation at 17q21 and childhood wheeze phenotypes.</h4>
<p class="authors"><span class="author">Granell R</span>, <span class="author">Henderson AJ</span>, <span class="author">Timpson N</span>, <span class="author">St Pourcain B</span>, <span class="author">Kemp JP</span>, <span class="author">Ring SM</span>, <span class="author">Ho K</span>, <span class="author">Montgomery SB</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Evans DM</span> and <span class="author">Sterne JA</span></p>
<p class="affiliation">School of Social and Community Medicine, University of Bristol, Bristol, United Kingdom. <a href="mailto:%72%61%71%75%65%6C.%67%72%61%6E%65%6C%6C%40%62%72%69%73%74%6F%6C.%61%63.%75%6B">&#114;&#97;&#113;&#117;&#101;&#108;&#46;&#103;&#114;&#97;&#110;&#101;&#108;&#108;&#64;&#98;&#114;&#105;&#115;&#116;&#111;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Genome-wide association studies have identified associations of genetic variants at 17q21 near ORMDL3 with childhood asthma.</p><p class="abstract"><strong>Objectives</strong>: We sought to determine whether associations in this region are specific to particular asthma phenotypes and specific to ORMDL3.</p><p class="abstract"><strong>Methods</strong>: We examined associations between 244 independent single nucleotide polymorphisms (SNPs) plus 13 previously identified asthma-related SNPs in the region between 34 and 36&nbsp;Mb on chromosome 17 and early wheezing phenotypes, doctor-diagnosed asthma and atopy at 7&frac12; years, and bronchial hyperresponsiveness and lung function at 8&frac12; years in 7045 children from the Avon Longitudinal Study of Parents and Children birth cohort study. With this, cis expression quantitative trait loci signals for the same SNPs were assessed in 875 samples across genes in the same region.</p><p class="abstract"><strong>Results</strong>: The strongest evidence for phenotypic association was&nbsp;seen for persistent wheezing (rs8076131 near ORMDL3: relative risk ratio [RRR], 1.60 [95% CI, 1.40-1.84], P&nbsp;= 1.4 &times; 10(-11); rs2305480 near GSDML: RRR, 1.60 [95% CI, 1.39-1.83], P&nbsp;= 1.5 &times; 10(-11); and rs9303277 near IKZF3: RRR, 1.57 [95%&nbsp;CI, 1.37-1.79], P&nbsp;= 4.4 &times; 10(-11)). Similar but less precisely estimated effects were seen for intermediate-onset wheeze, but there was little evidence of associations with other wheezing phenotypes. There was some evidence of associations with bronchial hyperresponsiveness. SNPs across the whole region show strong evidence of association with differential levels of&nbsp;expression at GSDML, IKZF3, and MED24, as well as ORMDL3.</p><p class="abstract"><strong>Conclusions</strong>: Associations of SNPs in the 17q21 locus are specific to asthma and specific wheezing phenotypes and are not&nbsp;explained by associations with intermediate phenotypes, such as atopy or lung function.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0401540</span>, <span class="grant">G9815508</span>; Wellcome Trust: <span class="grant">092731</span>, <span class="grant">WT083431MA</span></p>
<p class="publication"><span class="title">The Journal of allergy and clinical immunology</span>&nbsp;<span class="year">2013</span>;<span class="volume">131</span>;<span class="number">3</span>;<span class="pagerange">685-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23154084">23154084</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jaci.2012.09.021">10.1016/j.jaci.2012.09.021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of bipolar disorder susceptibility alleles and identification of two novel genome-wide significant associations in a new bipolar disorder case-control sample.</h4>
<p class="authors"><span class="author">Green EK</span>, <span class="author">Hamshere M</span>, <span class="author">Forty L</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Fraser C</span>, <span class="author">Russell E</span>, <span class="author">Grozeva D</span>, <span class="author">Kirov G</span>, <span class="author">Holmans P</span>, <span class="author">Moran JL</span>, <span class="author">Purcell S</span>, <span class="author">Sklar P</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Jones L</span>, <span class="author">WTCCC</span>, <span class="author">Jones IR</span> and <span class="author">Craddock N</span></p>
<p class="affiliation">Department of Psychological Medicine, MRC Centre for Neuropsychiatric Genetics and Genomics, Cardiff University, Heath Park, Cardiff, UK.</p><p class="abstract">We have conducted a genotyping study using a custom Illumina Infinium HD genotyping array, the ImmunoChip, in a new UK sample of 1218 bipolar disorder (BD) cases and 2913 controls that have not been used in any studies previously reported independently or in meta-analyses. The ImmunoChip was designed before the publication of the Psychiatric Genome-Wide Association Study Consortium Bipolar Disorder Working Group (PGC-BD) meta-analysis data. As such 3106 single-nucleotide polymorphisms (SNPs) with a P-value &lt;1 &times; 10(-3) from the BD meta-analysis by Ferreira et al. were genotyped. We report support for two of the three most strongly associated chromosomal regions in the Ferreira study, CACNA1C (rs1006737, P=4.09 &times; 10(-4)) and 15q14 (rs2172835, P=0.043) but not ANK3 (rs10994336, P=0.912). We have combined our ImmunoChip data (569 quasi-independent SNPs from the 3016 SNPs genotyped) with the recently published PGC-BD meta-analysis data, using either the PGC-BD combined discovery and replication data where available or just the discovery data where the SNP was not typed in a replication sample in PGC-BD. Our data provide support for two regions, at ODZ4 and CACNA1C, with prior evidence for genome-wide significant (GWS) association in PGC-BD meta-analysis. In addition, the combined analysis shows two novel GWS associations. First, rs7296288 (P=8.97 &times; 10(-9), odds ratio (OR)=0.9), an intergenic polymorphism on chromosome 12 located between RHEBL1 and DHH. Second, rs3818253 (P=3.88 &times; 10(-8), OR=1.16), an intronic SNP on chromosome 20q11.2 in the gene TRPC4AP, which lies in a high linkage disequilibrium region along with the genes GSS and MYH7B.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/0</span>, <span class="grant">076113/C/04/Z</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2013</span>;<span class="volume">18</span>;<span class="number">12</span>;<span class="pagerange">1302-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23070075">23070075</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2012.142">10.1038/mp.2012.142</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Massively parallel sequencing reveals the complex structure of an irradiated human chromosome on a mouse background in the tc1 model of down syndrome.</h4>
<p class="authors"><span class="author">Gribble SM</span>, <span class="author">Wiseman FK</span>, <span class="author">Clayton S</span>, <span class="author">Prigmore E</span>, <span class="author">Langley E</span>, <span class="author">Yang F</span>, <span class="author">Maguire S</span>, <span class="author">Fu B</span>, <span class="author">Rajan D</span>, <span class="author">Sheppard O</span>, <span class="author">Scott C</span>, <span class="author">Hauser H</span>, <span class="author">Stephens PJ</span>, <span class="author">Stebbings LA</span>, <span class="author">Ng BL</span>, <span class="author">Fitzgerald T</span>, <span class="author">Quail MA</span>, <span class="author">Banerjee R</span>, <span class="author">Rothkamm K</span>, <span class="author">Tybulewicz VL</span>, <span class="author">Fisher EM</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">Down syndrome (DS) is caused by trisomy of chromosome 21 (Hsa21) and presents a complex phenotype that arises from abnormal dosage of genes on this chromosome. However, the individual dosage-sensitive genes underlying each phenotype remain largely unknown. To help dissect genotype - phenotype correlations in this complex syndrome, the first fully transchromosomic mouse model, the Tc1 mouse, which carries a copy of human chromosome 21 was produced in 2005. The Tc1 strain is trisomic for the majority of genes that cause phenotypes associated with DS, and this freely available mouse strain has become used widely to study DS, the effects of gene dosage abnormalities, and the effect on the basic biology of cells when a mouse carries a freely segregating human chromosome. Tc1 mice were created by a process that included irradiation microcell-mediated chromosome transfer of Hsa21 into recipient mouse embryonic stem cells. Here, the combination of next generation sequencing, array-CGH and fluorescence in situ hybridization technologies has enabled us to identify unsuspected rearrangements of Hsa21 in this mouse model; revealing one deletion, six duplications and more than 25 de novo structural rearrangements. Our study is not only essential for informing functional studies of the Tc1 mouse but also (1) presents for the first time a detailed sequence analysis of the effects of gamma radiation on an entire human chromosome, which gives some mechanistic insight into the effects of radiation damage on DNA, and (2) overcomes specific technical difficulties of assaying a human chromosome on a mouse background where highly conserved sequences may confound the analysis. Sequence data generated in this study is deposited in the ENA database, Study Accession number: ERP000439.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e60482</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23596509">23596509</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3626651">3626651</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0060482">10.1371/journal.pone.0060482</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reduced burden of very large and rare CNVs in bipolar affective disorder.</h4>
<p class="authors"><span class="author">Grozeva D</span>, <span class="author">Kirov G</span>, <span class="author">Conrad DF</span>, <span class="author">Barnes CP</span>, <span class="author">Hurles M</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span> and <span class="author">Craddock N</span></p>
<p class="affiliation">MRC Centre for Neuropsychiatric Genetics and Genomics, Cardiff.</p><p class="abstract"><strong>Objectives</strong>: Large, rare chromosomal copy number variants (CNVs) have been shown to increase the risk for schizophrenia and other neuropsychiatric disorders including autism, attention-deficit hyperactivity disorder, learning difficulties, and epilepsy. Their role in bipolar disorder (BD) is less clear. There are no reports of an increase in large, rare CNVs in BD in general, but some have reported an increase in early-onset cases. We previously found that the rate of such CNVs in individuals with BD was not increased, even in early-onset cases. Our aim here was to examine the rate of large rare CNVs in BD in comparison with a new large independent reference sample from the same country.</p><p class="abstract"><strong>Methods</strong>: We studied the CNVs in a case-control sample consisting of 1,650 BD cases (reported previously) and 10,259 reference individuals without a known psychiatric disorder who took part in the original Wellcome Trust Case Control Consortium (WTCCC) study. The 10,259 reference individuals were affected with six non-psychiatric disorders (coronary artery disease, types 1 and 2 diabetes, hypertension, Crohn&#39;s disease, and rheumatoid arthritis). Affymetrix 500K array genotyping data were used to call the CNVs.</p><p class="abstract"><strong>Results</strong>: The rate of CNVs &gt; 100 kb was not statistically different between cases and controls. The rate of very large (defined as &gt; 1 Mb) and rare (&lt; 1%) CNVs was significantly lower in patients with BD compared with the reference group. CNV loci associated with schizophrenia were not enriched in BD and, in fact, cases of BD had the lowest number of such CNVs compared with any of the WTCCC cohorts; this finding held even for the early-onset BD cases.</p><p class="abstract"><strong>Conclusions</strong>: Schizophrenia and BD differ with respect to CNV burden and association with specific CNVs. Our findings support the hypothesis that BD is etiologically distinct from schizophrenia with respect to large, rare CNVs and the accompanying associated neurodevelopmental abnormalities.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076113</span>, <span class="grant">085475</span></p>
<p class="publication"><span class="title">Bipolar disorders</span>&nbsp;<span class="year">2013</span>;<span class="volume">15</span>;<span class="number">8</span>;<span class="pagerange">893-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24127788">24127788</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fbdi.12125">10.1111/bdi.12125</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global analysis of DNA methylation variation in adipose tissue from twins reveals links to disease-associated variants in distal regulatory elements.</h4>
<p class="authors"><span class="author">Grundberg E</span>, <span class="author">Meduri E</span>, <span class="author">Sandling JK</span>, <span class="author">Hedman AK</span>, <span class="author">Keildson S</span>, <span class="author">Buil A</span>, <span class="author">Busche S</span>, <span class="author">Yuan W</span>, <span class="author">Nisbet J</span>, <span class="author">Sekowska M</span>, <span class="author">Wilk A</span>, <span class="author">Barrett A</span>, <span class="author">Small KS</span>, <span class="author">Ge B</span>, <span class="author">Caron M</span>, <span class="author">Shin SY</span>, <span class="author">Multiple Tissue Human Expression Resource Consortium</span>, <span class="author">Lathrop M</span>, <span class="author">Dermitzakis ET</span>, <span class="author">McCarthy MI</span>, <span class="author">Spector TD</span>, <span class="author">Bell JT</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, CB101SA Hinxton, UK; Department of Twin Research and Genetic Epidemiology, King&#39;s College London, SE17EH London, UK. Electronic address: <a href="mailto:%65%6C%69%6E.%67%72%75%6E%64%62%65%72%67%40%6D%63%67%69%6C%6C.%63%61">&#101;&#108;&#105;&#110;&#46;&#103;&#114;&#117;&#110;&#100;&#98;&#101;&#114;&#103;&#64;&#109;&#99;&#103;&#105;&#108;&#108;&#46;&#99;&#97;</a>.</p><p class="abstract">Epigenetic modifications such as DNA methylation play a key role in gene regulation and disease susceptibility. However, little is known about the genome-wide frequency, localization, and function of methylation variation and how it is regulated by genetic and environmental factors. We utilized the Multiple Tissue Human Expression Resource (MuTHER) and generated Illumina 450K adipose methylome data from 648 twins. We found that individual CpGs had low variance and that variability was suppressed in promoters. We noted that DNA methylation variation was highly heritable (h(2)median = 0.34) and that shared environmental effects correlated with metabolic phenotype-associated CpGs. Analysis of methylation quantitative-trait loci (metQTL) revealed that 28% of CpGs were associated with nearby SNPs, and when overlapping them with adipose expression quantitative-trait loci (eQTL) from the same individuals, we found that 6% of the loci played a role in regulating both gene expression and DNA methylation. These associations were bidirectional, but there were pronounced negative associations for promoter CpGs. Integration of metQTL with adipose reference epigenomes and disease associations revealed significant enrichment of metQTL overlapping metabolic-trait or disease loci in enhancers (the strongest effects were for high-density lipoprotein cholesterol and body mass index [BMI]). We followed up with the BMI SNP rs713586, a cg01884057 metQTL that overlaps an enhancer upstream of ADCY3, and used bisulphite sequencing to refine this region. Our results showed widespread population invariability yet sequence dependence on adipose DNA methylation but that incorporating maps of regulatory elements aid in linking CpG variation to gene regulation and disease risk in a tissue-dependent manner.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">EP1-120608</span>; Medical Research Council: <span class="grant">G9824984</span>, <span class="grant">MC_UU_12012/1</span>, <span class="grant">MC_UU_12012/5/B</span>; Wellcome Trust: <span class="grant">081917/Z/07/Z</span>, <span class="grant">083270/Z/07/Z</span>, <span class="grant">090532</span>, <span class="grant">098051</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">93</span>;<span class="number">5</span>;<span class="pagerange">876-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24183450">24183450</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3824131">3824131</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.10.004">10.1016/j.ajhg.2013.10.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene-centric meta-analyses of 108 912 individuals confirm known body mass index loci and reveal three novel signals.</h4>
<p class="authors"><span class="author">Guo Y</span>, <span class="author">Lanktree MB</span>, <span class="author">Taylor KC</span>, <span class="author">Hakonarson H</span>, <span class="author">Lange LA</span>, <span class="author">Keating BJ</span> and <span class="author">The IBC 50K SNP array BMI Consortium</span></p>
<p class="affiliation">List of authors is given in the Full Author List Section of Appendix.</p>
<p class="abstract">Recent genetic association studies have made progress in uncovering components of the genetic architecture of the body mass index (BMI). We used the ITMAT-Broad-Candidate Gene Association Resource (CARe) (IBC) array comprising up to 49 320 single nucleotide polymorphisms (SNPs) across âˆ¼2100 metabolic and cardiovascular-related loci to genotype up to 108 912 individuals of European ancestry (EA), African-Americans, Hispanics and East Asians, from 46 studies, to provide additional insight into SNPs underpinning BMI. We used a five-phase study design: Phase I focused on meta-analysis of EA studies providing individual level genotype data; Phase II performed a replication of cohorts providing summary level EA data; Phase III meta-analyzed results from the first two phases; associated SNPs from Phase III were used for replication in Phase IV; finally in Phase V, a multi-ethnic meta-analysis of all samples from four ethnicities was performed. At an array-wide significance (P < 2.40E-06), we identify novel BMI associations in loci translocase of outer mitochondrial membrane 40 homolog (yeast) - apolipoprotein E - apolipoprotein C-I (TOMM40-APOE-APOC1) (rs2075650, P = 2.95E-10), sterol regulatory element binding transcription factor 2 (SREBF2, rs5996074, P = 9.43E-07) and neurotrophic tyrosine kinase, receptor, type 2 [NTRK2, a brain-derived neurotrophic factor (BDNF) receptor gene, rs1211166, P = 1.04E-06] in the Phase IV meta-analysis. Of 10 loci with previous evidence for BMI association represented on the IBC array, eight were replicated, with the remaining two showing nominal significance. Conditional analyses revealed two independent BMI-associated signals in BDNF and melanocortin 4 receptor (MC4R) regions. Of the 11 array-wide significant SNPs, three are associated with gene expression levels in both primary B-cells and monocytes; with rs4788099 in SH2B adaptor protein 1 (SH2B1) notably being associated with the expression of multiple genes in cis. These multi-ethnic meta-analyses expand our knowledge of BMI genetics.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">R37 AG011099</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">1</span>;<span class="pagerange">184-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23001569" rel="external">23001569</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3522401" rel="external">3522401</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fdds396" rel="external">10.1093/hmg/dds396</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide diversity in the levant reveals recent structuring by culture.</h4>
<p class="authors"><span class="author">Haber M</span>, <span class="author">Gauguier D</span>, <span class="author">Youhanna S</span>, <span class="author">Patterson N</span>, <span class="author">Moorjani P</span>, <span class="author">Botigu&eacute; LR</span>, <span class="author">Platt DE</span>, <span class="author">Matisoo-Smith E</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Wells RS</span>, <span class="author">Bertranpetit J</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Comas D</span> and <span class="author">Zalloua PA</span></p>
<p class="affiliation">Institut de Biologia Evolutiva (CSIC-UPF), Departament de Ci&egrave;ncies de la Salut i de la Vida, Universitat Pompeu Fabra, Barcelona, Spain.</p><p class="abstract">The Levant is a region in the Near East with an impressive record of continuous human existence and major cultural developments since the Paleolithic period. Genetic and archeological studies present solid evidence placing the Middle East and the Arabian Peninsula as the first stepping-stone outside Africa. There is, however, little understanding of demographic changes in the Middle East, particularly the Levant, after the first Out-of-Africa expansion and how the Levantine peoples relate genetically to each other and to their neighbors. In this study we analyze more than 500,000 genome-wide SNPs in 1,341 new samples from the Levant and compare them to samples from 48 populations worldwide. Our results show recent genetic stratifications in the Levant are driven by the religious affiliations of the populations within the region. Cultural changes within the last two millennia appear to have facilitated/maintained admixture between culturally similar populations from the Levant, Arabian Peninsula, and Africa. The same cultural changes seem to have resulted in genetic isolation of other groups by limiting admixture with culturally different neighboring populations. Consequently, Levant populations today fall into two main groups: one sharing more genetic characteristics with modern-day Europeans and Central Asians, and the other with closer genetic affinities to other Middle Easterners and Africans. Finally, we identify a putative Levantine ancestral component that diverged from other Middle Easterners &sim;23,700-15,500 years ago during the last glacial period, and diverged from Europeans &sim;15,900-9,100 years ago between the last glacial warming and the start of the Neolithic.</p>
<p class="grants">Funded by: PEPFAR: <span class="grant">098051</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">e1003316</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23468648">23468648</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3585000">3585000</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003316">10.1371/journal.pgen.1003316</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Role of Salt Bridges, Charge Density, and Subunit Flexibility in Determining Disassembly Routes of Protein Complexes.</h4>
<p class="authors"><span class="author">Hall Z</span>, <span class="author">Hern&aacute;ndez H</span>, <span class="author">Marsh JA</span>, <span class="author">Teichmann SA</span> and <span class="author">Robinson CV</span></p>
<p class="affiliation">Physical and Theoretical Chemistry Laboratory, Department of Chemistry, University of Oxford, Oxford OX1 3QZ, UK.</p>
<p class="abstract">Mass spectrometry can be used to characterize multiprotein complexes, defining their subunit stoichiometry and composition following solution disruption and collision-induced dissociation (CID). While CID of protein complexes in the gas phase typically results in the dissociation of unfolded subunits, a second atypical route is possible wherein compact subunits or subcomplexes are ejected without unfolding. Because tertiary structure and subunit interactions may be retained, this is the preferred route for structural investigations. How can we influence which pathway is adopted? By studying properties of a series of homomeric and heteromeric protein complexes and varying their overall charge in solution, we found that low subunit flexibility, higher charge densities, fewer salt bridges, and smaller interfaces are likely to be involved in promoting dissociation routes without unfolding. Manipulating the charge on a protein complex therefore enables us to direct dissociation through structurally informative pathways that mimic those followed in solution.</p>
<p class="publication"><span class="title">Structure (London, England : 1993)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23850452">23850452</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.str.2013.06.004">10.1016/j.str.2013.06.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Contributions of protein-coding and regulatory change to adaptive molecular evolution in murid rodents.</h4>
<p class="authors"><span class="author">Halligan DL</span>, <span class="author">Kousathanas A</span>, <span class="author">Ness RW</span>, <span class="author">Harr B</span>, <span class="author">E&ouml;ry L</span>, <span class="author">Keane TM</span>, <span class="author">Adams DJ</span> and <span class="author">Keightley PD</span></p>
<p class="affiliation">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, United Kingdom.</p>
<p class="abstract">The contribution of regulatory versus protein change to adaptive evolution has long been controversial. In principle, the rate and strength of adaptation within functional genetic elements can be quantified on the basis of an excess of nucleotide substitutions between species compared to the neutral expectation or from effects of recent substitutions on nucleotide diversity at linked sites. Here, we infer the nature of selective forces acting in proteins, their UTRs and conserved noncoding elements (CNEs) using genome-wide patterns of diversity in wild house mice and divergence to related species. By applying an extension of the McDonald-Kreitman test, we infer that adaptive substitutions are widespread in protein-coding genes, UTRs and CNEs, and we estimate that there are at least four times as many adaptive substitutions in CNEs and UTRs as in proteins. We observe pronounced reductions in mean diversity around nonsynonymous sites (whether or not they have experienced a recent substitution). This can be explained by selection on multiple, linked CNEs and exons. We also observe substantial dips in mean diversity (after controlling for divergence) around protein-coding exons and CNEs, which can also be explained by the combined effects of many linked exons and CNEs. A model of background selection (BGS) can adequately explain the reduction in mean diversity observed around CNEs. However, BGS fails to explain the wide reductions in mean diversity surrounding exons (encompassing ~100 Kb, on average), implying that there is a substantial role for adaptation within exons or closely linked sites. The wide dips in diversity around exons, which are hard to explain by BGS, suggest that the fitness effects of adaptive amino acid substitutions could be substantially larger than substitutions in CNEs. We conclude that although there appear to be many more adaptive noncoding changes, substitutions in proteins may dominate phenotypic evolution.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">e1003995</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24339797">24339797</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3854965">3854965</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003995">10.1371/journal.pgen.1003995</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fine mapping of type 1 diabetes regions Idd9.1 and Idd9.2 reveals genetic complexity.</h4>
<p class="authors"><span class="author">Hamilton-Williams EE</span>, <span class="author">Rainbow DB</span>, <span class="author">Cheung J</span>, <span class="author">Christensen M</span>, <span class="author">Lyons PA</span>, <span class="author">Peterson LB</span>, <span class="author">Steward CA</span>, <span class="author">Sherman LA</span> and <span class="author">Wicker LS</span></p>
<p class="affiliation">Department of Immunology and Microbial Sciences, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, CA, 92037, USA.</p><p class="abstract">Nonobese diabetic (NOD) mice congenic for C57BL/10 (B10)-derived genes in the Idd9 region of chromosome 4 are highly protected from type 1 diabetes (T1D). Idd9 has been divided into three protective subregions (Idd9.1, 9.2, and 9.3), each of which partially prevents disease. In this study we have fine-mapped the Idd9.1 and Idd9.2 regions, revealing further genetic complexity with at least two additional subregions contributing to protection from T1D. Using the NOD sequence from bacterial artificial chromosome clones of the Idd9.1 and Idd9.2 regions as well as whole-genome sequence data recently made available, sequence polymorphisms within the regions highlight a high degree of polymorphism between the NOD and B10 strains in the Idd9 regions. Among numerous candidate genes are several with immunological importance. The Idd9.1 region has been separated into Idd9.1 and Idd9.4, with Lck remaining a candidate gene within Idd9.1. One of the Idd9.2 regions contains the candidate genes Masp2 (encoding mannan-binding lectin serine peptidase 2) and Mtor (encoding mammalian target of rapamycin). From mRNA expression analyses, we have also identified several other differentially expressed candidate genes within the Idd9.1 and Idd9.2 regions. These findings highlight that multiple, relatively small genetic effects combine and interact to produce significant changes in immune tolerance and diabetes onset.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 070351</span>, <span class="grant">AI 15416</span>, <span class="grant">U19AI050864-07</span>; Wellcome Trust: <span class="grant">091157</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2013</span>;<span class="volume">24</span>;<span class="number">9-10</span>;<span class="pagerange">358-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23934554">23934554</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3824839">3824839</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-013-9466-y">10.1007/s00335-013-9466-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in B4GALNT1 (GM2 synthase) underlie a new disorder of ganglioside biosynthesis.</h4>
<p class="authors"><span class="author">Harlalka GV</span>, <span class="author">Lehman A</span>, <span class="author">Chioza B</span>, <span class="author">Baple EL</span>, <span class="author">Maroofian R</span>, <span class="author">Cross H</span>, <span class="author">Sreekantan-Nair A</span>, <span class="author">Priestman DA</span>, <span class="author">Al-Turki S</span>, <span class="author">McEntagart ME</span>, <span class="author">Proukakis C</span>, <span class="author">Royle L</span>, <span class="author">Kozak RP</span>, <span class="author">Bastaki L</span>, <span class="author">Patton M</span>, <span class="author">Wagner K</span>, <span class="author">Coblentz R</span>, <span class="author">Price J</span>, <span class="author">Mezei M</span>, <span class="author">Schlade-Bartusiak K</span>, <span class="author">Platt FM</span>, <span class="author">Hurles ME</span> and <span class="author">Crosby AH</span></p>
<p class="affiliation">1 Institute of Biomedical and Clinical Science, University of Exeter Medical School, St. Luke's Campus, Heavitree Road, EX1 2LU, Exeter, Devon, UK.</p>
<p class="abstract">Glycosphingolipids are ubiquitous constituents of eukaryotic plasma membranes, and their sialylated derivatives, gangliosides, are the major class of glycoconjugates expressed by neurons. Deficiencies in their catabolic pathways give rise to a large and well-studied group of inherited disorders, the lysosomal storage diseases. Although many glycosphingolipid catabolic defects have been defined, only one proven inherited disease arising from a defect in ganglioside biosynthesis is known. This disease, because of defects in the first step of ganglioside biosynthesis (GM3 synthase), results in a severe epileptic disorder found at high frequency amongst the Old Order Amish. Here we investigated an unusual neurodegenerative phenotype, most commonly classified as a complex form of hereditary spastic paraplegia, present in families from Kuwait, Italy and the Old Order Amish. Our genetic studies identified mutations in B4GALNT1 (GM2 synthase), encoding the enzyme that catalyzes the second step in complex ganglioside biosynthesis, as the cause of this neurodegenerative phenotype. Biochemical profiling of glycosphingolipid biosynthesis confirmed a lack of GM2 in affected subjects in association with a predictable increase in levels of its precursor, GM3, a finding that will greatly facilitate diagnosis of this condition. With the description of two neurological human diseases involving defects in two sequentially acting enzymes in ganglioside biosynthesis, there is the real possibility that a previously unidentified family of ganglioside deficiency diseases exist. The study of patients and animal models of these disorders will pave the way for a greater understanding of the role gangliosides play in neuronal structure and function and provide insights into the development of effective treatment therapies.</p>
<p class="publication"><span class="title">Brain : a journal of neurology</span>&nbsp;<span class="year">2013</span>;<span class="volume">136</span>;<span class="number">Pt 12</span>;<span class="pagerange">3618-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24103911">24103911</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3859217">3859217</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbrain%2Fawt270">10.1093/brain/awt270</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing for analysis of an outbreak of meticillin-resistant Staphylococcus aureus: a descriptive study.</h4>
<p class="authors"><span class="author">Harris SR</span>, <span class="author">Cartwright EJ</span>, <span class="author">T&ouml;r&ouml;k ME</span>, <span class="author">Holden MT</span>, <span class="author">Brown NM</span>, <span class="author">Ogilvy-Stuart AL</span>, <span class="author">Ellington MJ</span>, <span class="author">Quail MA</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: The emergence of meticillin-resistant Staphylococcus aureus (MRSA) that can persist in the community and replace existing hospital-adapted lineages of MRSA means that it is necessary to understand transmission dynamics in terms of hospitals and the community as one entity. We assessed the use of whole-genome sequencing to enhance detection of MRSA transmission between these settings.</p><p class="abstract"><strong>Methods</strong>: We studied a putative MRSA outbreak on a special care baby unit (SCBU) at a National Health Service Foundation Trust in Cambridge, UK. We used whole-genome sequencing to validate and expand findings from an infection-control team who assessed the outbreak through conventional analysis of epidemiological data and antibiogram profiles. We sequenced isolates from all colonised patients in the SCBU, and sequenced MRSA isolates from patients in the hospital or community with the same antibiotic susceptibility profile as the outbreak strain.</p><p class="abstract"><strong>Findings</strong>: The hospital infection-control team identified 12 infants colonised with MRSA in a 6 month period in 2011, who were suspected of being linked, but a persistent outbreak could not be confirmed with conventional methods. With whole-genome sequencing, we identified 26 related cases of MRSA carriage, and showed transmission occurred within the SCBU, between mothers on a postnatal ward, and in the community. The outbreak MRSA type was a new sequence type (ST) 2371, which is closely related to ST22, but contains genes encoding Panton-Valentine leucocidin. Whole-genome sequencing data were used to propose and confirm that MRSA carriage by a staff member had allowed the outbreak to persist during periods without known infection on the SCBU and after a deep clean.</p><p class="abstract"><strong>Interpretation</strong>: Whole-genome sequencing holds great promise for rapid, accurate, and comprehensive identification of bacterial transmission pathways in hospital and community settings, with concomitant reductions in infections, morbidity, and costs.</p><p class="abstract"><strong>Funding</strong>: UK Clinical Research Collaboration Translational Infection Research Initiative, Wellcome Trust, Health Protection Agency, and the National Institute for Health Research Cambridge Biomedical Research Centre.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Chief Scientist Office; Department of Health; Medical Research Council: <span class="grant">G1000803</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">The Lancet. Infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="number">2</span>;<span class="pagerange">130-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23158674">23158674</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3556525">3556525</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS1473-3099%2812%2970268-2">10.1016/S1473-3099(12)70268-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Read and assembly metrics inconsequential for clinical utility of whole-genome sequencing in mapping outbreaks.</h4>
<p class="authors"><span class="author">Harris SR</span>, <span class="author">T&ouml;r&ouml;k ME</span>, <span class="author">Cartwright EJ</span>, <span class="author">Quail MA</span>, <span class="author">Peacock SJ</span> and <span class="author">Parkhill J</span></p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1000803</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2013</span>;<span class="volume">31</span>;<span class="number">7</span>;<span class="pagerange">592-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23839141">23839141</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.2616">10.1038/nbt.2616</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diagnostic pathway for the investigation of thrombocytosis.</h4>
<p class="authors"><span class="author">Harrison CN</span>, <span class="author">Butt N</span>, <span class="author">Campbell P</span>, <span class="author">Conneally E</span>, <span class="author">Drummond M</span>, <span class="author">Green AR</span>, <span class="author">Murrin R</span>, <span class="author">Radia DH</span>, <span class="author">Reilly JT</span> and <span class="author">McMullin MF</span></p>
<p class="affiliation">Department of Haematology, Guy's and St Thomas, Hospitals' NHS Foundation Trust, London, UK.</p>
<p class="publication"><span class="title">British journal of haematology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23480550">23480550</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fbjh.12283">10.1111/bjh.12283</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome sequencing identifies zoonotic transmission of MRSA isolates with the novel mecA homologue mecC.</h4>
<p class="authors"><span class="author">Harrison EM</span>, <span class="author">Paterson GK</span>, <span class="author">Holden MT</span>, <span class="author">Larsen J</span>, <span class="author">Stegger M</span>, <span class="author">Larsen AR</span>, <span class="author">Petersen A</span>, <span class="author">Skov RL</span>, <span class="author">Christensen JM</span>, <span class="author">Bak Zeuthen A</span>, <span class="author">Heltberg O</span>, <span class="author">Harris SR</span>, <span class="author">Zadoks RN</span>, <span class="author">Parkhill J</span>, <span class="author">Peacock SJ</span> and <span class="author">Holmes MA</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, UK.</p>
<p class="abstract">Several methicillin-resistant Staphylococcus aureus (MRSA) lineages that carry a novel mecA homologue (mecC) have recently been described in livestock and humans. In Denmark, two independent human cases of mecC-MRSA infection have been linked to a livestock reservoir. We investigated the molecular epidemiology of the associated MRSA isolates using whole genome sequencing (WGS). Single nucleotide polymorphisms (SNP) were defined and compared to a reference genome to place the isolates into a phylogenetic context. Phylogenetic analysis revealed two distinct farm-specific clusters comprising isolates from the human case and their own livestock, whereas human and animal isolates from the same farm only differed by a small number of SNPs, which supports the likelihood of zoonotic transmission. Further analyses identified a number of genes and mutations that may be associated with host interaction and virulence. This study demonstrates that mecC-MRSA ST130 isolates are capable of transmission between animals and humans, and underscores the potential of WGS in epidemiological investigations and source tracking of bacterial infections. â†’See accompanying article http://dx.doi.org/10.1002/emmm.201302622.</p>
<p class="publication"><span class="title">EMBO molecular medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">509-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23526809">23526809</a>; DOI: <a href="http://dx.doi.org/10.1002%2Femmm.201202413">10.1002/emmm.201202413</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Staphylococcus xylosus Isolate with a New mecC Allotype.</h4>
<p class="authors"><span class="author">Harrison EM</span>, <span class="author">Paterson GK</span>, <span class="author">Holden MT</span>, <span class="author">Morgan FJ</span>, <span class="author">Larsen AR</span>, <span class="author">Petersen A</span>, <span class="author">Leroy S</span>, <span class="author">De Vliegher S</span>, <span class="author">Perreten V</span>, <span class="author">Fox LK</span>, <span class="author">Lam TJ</span>, <span class="author">Sampimon OC</span>, <span class="author">Zadoks RN</span>, <span class="author">Peacock SJ</span>, <span class="author">Parkhill J</span> and <span class="author">Holmes MA</span></p>
<p class="affiliation">University of Cambridge, Department of Veterinary Medicine, Cambridge, United Kingdom.</p>
<p class="abstract">Recently, a novel variant of mecA known as mecC (mecA(LGA251)) was identified in Staphylococcus aureus isolates from both humans and animals. In this study, we identified a Staphylococcus xylosus isolate that harbors a new allotype of the mecC gene, mecC1. Whole-genome sequencing revealed that mecC1 forms part of a class E mec complex (mecI-mecR1-mecC1-blaZ) located at the orfX locus as part of a likely staphylococcal cassette chromosome mec element (SCCmec) remnant, which also contains a number of other genes present on the type XI SCCmec.</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2013</span>;<span class="volume">57</span>;<span class="number">3</span>;<span class="pagerange">1524-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23274660">23274660</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.01882-12">10.1128/AAC.01882-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Description and Nomenclature of Neisseria meningitidis Capsule Locus.</h4>
<p class="authors"><span class="author">Harrison OB</span>, <span class="author">Claus H</span>, <span class="author">Jiang Y</span>, <span class="author">Bennett JS</span>, <span class="author">Bratcher HB</span>, <span class="author">Jolley KA</span>, <span class="author">Corton C</span>, <span class="author">Care R</span>, <span class="author">Poolman JT</span>, <span class="author">Zollinger WD</span>, <span class="author">Frasch CE</span>, <span class="author">Stephens DS</span>, <span class="author">Feavers I</span>, <span class="author">Frosch M</span>, <span class="author">Parkhill J</span>, <span class="author">Vogel U</span>, <span class="author">Quail MA</span>, <span class="author">Bentley SD</span> and <span class="author">Maiden MC</span></p>
<p class="abstract">Pathogenic Neisseria meningitidis isolates contain a polysaccharide capsule that is the main virulence determinant for this bacterium. Thirteen capsular polysaccharides have been described, and nuclear magnetic resonance spectroscopy has enabled determination of the structure of capsular polysaccharides responsible for serogroup specificity. Molecular mechanisms involved in N. meningitidis capsule biosynthesis have also been identified, and genes involved in this process and in cell surface translocation are clustered at a single chromosomal locus termed cps. The use of multiple names for some of the genes involved in capsule synthesis, combined with the need for rapid diagnosis of serogroups commonly associated with invasive meningococcal disease, prompted a requirement for a consistent approach to the nomenclature of capsule genes. In this report, a comprehensive description of all N. meningitidis serogroups is provided, along with a proposed nomenclature, which was presented at the 2012 XVIIIth International Pathogenic Neisseria Conference.</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">19</span>;<span class="number">4</span>;<span class="pagerange">566-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23628376">23628376</a>; DOI: <a href="http://dx.doi.org/10.3201%2Feid1904.111799">10.3201/eid1904.111799</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">VS-5584, a novel and highly selective PI3K/mTOR kinase inhibitor for the treatment of cancer.</h4>
<p class="authors"><span class="author">Hart S</span>, <span class="author">Novotny-Diermayr V</span>, <span class="author">Goh KC</span>, <span class="author">Williams M</span>, <span class="author">Tan YC</span>, <span class="author">Ong LC</span>, <span class="author">Cheong A</span>, <span class="author">Ng BK</span>, <span class="author">Amalini C</span>, <span class="author">Madan B</span>, <span class="author">Nagaraj H</span>, <span class="author">Jayaraman R</span>, <span class="author">Pasha KM</span>, <span class="author">Ethirajulu K</span>, <span class="author">Chng WJ</span>, <span class="author">Mustafa N</span>, <span class="author">Goh BC</span>, <span class="author">Benes C</span>, <span class="author">McDermott U</span>, <span class="author">Garnett M</span>, <span class="author">Dymock B</span> and <span class="author">Wood JM</span></p>
<p class="affiliation">S*BIO Pte Ltd., Singapore 117528, Singapore. <a href="mailto:%73%74%65%66%61%6E.%73%62%69%6F%40%67%6D%61%69%6C.%63%6F%6D">&#115;&#116;&#101;&#102;&#97;&#110;&#46;&#115;&#98;&#105;&#111;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Dysregulation of the PI3K/mTOR pathway, either through amplifications, deletions, or as a direct result of mutations, has been closely linked to the development and progression of a wide range of cancers. Moreover, this pathway activation is a poor prognostic marker for many tumor types and confers resistance to various cancer therapies. Here, we describe VS-5584, a novel, low-molecular weight compound with equivalent potent activity against mTOR (IC(50) = 37 nmol/L) and all class I phosphoinositide 3-kinase (PI3K) isoforms IC(50): PI3K&alpha; = 16 nmol/L; PI3K&beta; = 68 nmol/L; PI3K&gamma; = 25 nmol/L; PI3K&delta; = 42 nmol/L, without relevant activity on 400 lipid and protein kinases. VS-5584 shows robust modulation of cellular PI3K/mTOR pathways, inhibiting phosphorylation of substrates downstream of PI3K and mTORC1/2. A large human cancer cell line panel screen (436 lines) revealed broad antiproliferative sensitivity and that cells harboring mutations in PI3KCA are generally more sensitive toward VS-5584 treatment. VS-5584 exhibits favorable pharmacokinetic properties after oral dosing in mice and is well tolerated. VS-5584 induces long-lasting and dose-dependent inhibition of PI3K/mTOR signaling in tumor tissue, leading to tumor growth inhibition in various rapalog-sensitive and -resistant human xenograft models. Furthermore, VS-5584 is synergistic with an EGF receptor inhibitor in a gastric tumor model. The unique selectivity profile and favorable pharmacologic and pharmaceutical properties of VS-5584 and its efficacy in a wide range of human tumor models supports further investigations of VS-5584 in clinical trials.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093868</span></p>
<p class="publication"><span class="title">Molecular cancer therapeutics</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">151-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23270925">23270925</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3588144">3588144</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1535-7163.MCT-12-0466">10.1158/1535-7163.MCT-12-0466</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of the zebrafish maternal and paternal transcriptomes.</h4>
<p class="authors"><span class="author">Harvey SA</span>, <span class="author">Sealy I</span>, <span class="author">Kettleborough R</span>, <span class="author">Fenyes F</span>, <span class="author">White R</span>, <span class="author">Stemple D</span> and <span class="author">Smith JC</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">Transcription is an essential component of basic cellular and developmental processes. However, early embryonic development occurs in the absence of transcription and instead relies upon maternal mRNAs and proteins deposited in the egg during oocyte maturation. Although the early zebrafish embryo is competent to transcribe exogenous DNA, factors present in the embryo maintain genomic DNA in a state that is incompatible with transcription. The cell cycles of the early embryo titrate out these factors, leading to zygotic transcription initiation, presumably in response to a change in genomic DNA chromatin structure to a state that supports transcription. To understand the molecular mechanisms controlling this maternal to zygotic transition, it is important to distinguish between the maternal and zygotic transcriptomes during this period. Here we use exome sequencing and RNA-seq to achieve such discrimination and in doing so have identified the first zygotic genes to be expressed in the embryo. Our work revealed different profiles of maternal mRNA post-transcriptional regulation prior to zygotic transcription initiation. Finally, we demonstrate that maternal mRNAs are required for different modes of zygotic transcription initiation, which is not simply dependent on the titration of factors that maintain genomic DNA in a transcriptionally incompetent state.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2013</span>;<span class="volume">140</span>;<span class="number">13</span>;<span class="pagerange">2703-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23720042">23720042</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.095091">10.1242/dev.095091</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A blood pressure genetic risk score is a significant predictor of incident cardiovascular events in 32 669 individuals.</h4>
<p class="authors"><span class="author">Havulinna AS</span>, <span class="author">Kettunen J</span>, <span class="author">Ukkola O</span>, <span class="author">Osmond C</span>, <span class="author">Eriksson JG</span>, <span class="author">Kes&auml;niemi YA</span>, <span class="author">Jula A</span>, <span class="author">Peltonen L</span>, <span class="author">Kontula K</span>, <span class="author">Salomaa V</span> and <span class="author">Newton-Cheh C</span></p>
<p class="affiliation">Center for Human Genetic Research, Cardiovascular Research Center, Massachusetts General Hospital, 185 Cambridge St, CPZN 5.242, Boston, MA 02114.</p>
<p class="abstract">Recent genome-wide association studies have identified genetic variants associated with blood pressure (BP). We investigated whether genetic risk scores (GRSs) constructed of these variants would predict incident cardiovascular disease (CVD) events. We genotyped 32 common single nucleotide polymorphisms in several Finnish cohorts, with up to 32 669 individuals after exclusion of prevalent CVD cases. The median follow-up was 9.8 years, during which 2295 incident CVD events occurred. We created GRSs separately for systolic BP and diastolic BP by multiplying the risk allele count of each single nucleotide polymorphism by the effect size estimated in published genome-wide association studies. We performed Cox regression analyses with and without adjustment for clinical factors, including BP at baseline in each cohort. The results were combined by inverse variance-weighted fixed-effects meta-analysis. The GRSs were strongly associated with systolic BP and diastolic BP, and baseline hypertension (all P<10(-62)). Hazard ratios comparing the highest quintiles of systolic BP and diastolic BP GRSs with the lowest quintiles after adjustment for age, age squared, and sex were 1.25 (1.07-1.46; P=0.006) and 1.23 (1.05-1.43; P=0.01), respectively, for incident coronary heart disease; 1.24 (1.01-1.53; P=0.04) and 1.35 (1.09-1.66; P=0.005), respectively, for incident stroke; and 1.23 (1.08-1.40; P=2×10(-6)) and 1.26 (1.11-1.44; P=5×10(-4)), respectively, for composite CVD. In conclusion, BP findings from genome-wide association studies are strongly replicated. GRSs comprising bona fide BP-single nucleotide polymorphisms predicted CVD risk, consistent with a lifelong effect on BP of these variants collectively.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01 HL098283</span></p>
<p class="publication"><span class="title">Hypertension</span>&nbsp;<span class="year">2013</span>;<span class="volume">61</span>;<span class="number">5</span>;<span class="pagerange">987-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23509078">23509078</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3648219">3648219</a>; DOI: <a href="http://dx.doi.org/10.1161%2FHYPERTENSIONAHA.111.00649">10.1161/HYPERTENSIONAHA.111.00649</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mcl-1 and FBW7 control a dominant survival pathway underlying HDAC and Bcl-2 inhibitor synergy in squamous cell carcinoma.</h4>
<p class="authors"><span class="author">He L</span>, <span class="author">Torres-Lockhart K</span>, <span class="author">Forster N</span>, <span class="author">Ramakrishnan S</span>, <span class="author">Greninger P</span>, <span class="author">Garnett MJ</span>, <span class="author">McDermott U</span>, <span class="author">Rothenberg SM</span>, <span class="author">Benes CH</span> and <span class="author">Ellisen LW</span></p>
<p class="affiliation">Massachusetts General Hospital Cancer Center and Harvard Medical School, Boston, Massachusetts 02114, USA.</p><p class="abstract">Effective targeted therapeutics for squamous cell carcinoma (SCC) are lacking. Here, we uncover Mcl-1 as a dominant and tissue-specific survival factor in SCC, providing a roadmap for a new therapeutic approach. Treatment with the histone deacetylase (HDAC) inhibitor vorinostat regulates Bcl-2 family member expression to disable the Mcl-1 axis and thereby induce apoptosis in SCC cells. Although Mcl-1 dominance renders SCC cells resistant to the BH3-mimetic ABT-737, vorinostat primes them for sensitivity to ABT-737 by shuttling Bim from Mcl-1 to Bcl-2/Bcl-xl, resulting in dramatic synergy for this combination and sustained tumor regression in vivo. Moreover, somatic FBW7 mutation in SCC is associated with stabilized Mcl-1 and high Bim levels, resulting in a poor response to standard chemotherapy but a robust response to HDAC inhibitors and enhanced synergy with the combination vorinostat/ABT-737. Collectively, our findings provide a biochemical rationale and predictive markers for the application of this therapeutic combination in SCC.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">BC093523</span>; NIDCR NIH HHS: <span class="grant">K08 DE020139</span>, <span class="grant">NIH KO8 DE-020139</span>, <span class="grant">R01 DE015945</span>; Wellcome Trust: <span class="grant">086357</span>, <span class="grant">093868</span></p>
<p class="publication"><span class="title">Cancer discovery</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">324-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23274910">23274910</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3595349">3595349</a>; DOI: <a href="http://dx.doi.org/10.1158%2F2159-8290.CD-12-0417">10.1158/2159-8290.CD-12-0417</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Emergence and global spread of epidemic healthcare-associated Clostridium difficile.</h4>
<p class="authors"><span class="author">He M</span>, <span class="author">Miyajima F</span>, <span class="author">Roberts P</span>, <span class="author">Ellison L</span>, <span class="author">Pickard DJ</span>, <span class="author">Martin MJ</span>, <span class="author">Connor TR</span>, <span class="author">Harris SR</span>, <span class="author">Fairley D</span>, <span class="author">Bamford KB</span>, <span class="author">D&#39;Arc S</span>, <span class="author">Brazier J</span>, <span class="author">Brown D</span>, <span class="author">Coia JE</span>, <span class="author">Douce G</span>, <span class="author">Gerding D</span>, <span class="author">Kim HJ</span>, <span class="author">Koh TH</span>, <span class="author">Kato H</span>, <span class="author">Senoh M</span>, <span class="author">Louie T</span>, <span class="author">Michell S</span>, <span class="author">Butt E</span>, <span class="author">Peacock SJ</span>, <span class="author">Brown NM</span>, <span class="author">Riley T</span>, <span class="author">Songer G</span>, <span class="author">Wilcox M</span>, <span class="author">Pirmohamed M</span>, <span class="author">Kuijper E</span>, <span class="author">Hawkey P</span>, <span class="author">Wren BW</span>, <span class="author">Dougan G</span>, <span class="author">Parkhill J</span> and <span class="author">Lawley TD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Epidemic C. difficile (027/BI/NAP1) has rapidly emerged in the past decade as the leading cause of antibiotic-associated diarrhea worldwide. However, the key events in evolutionary history leading to its emergence and the subsequent patterns of global spread remain unknown. Here, we define the global population structure of C. difficile 027/BI/NAP1 using whole-genome sequencing and phylogenetic analysis. We show that two distinct epidemic lineages, FQR1 and FQR2, not one as previously thought, emerged in North America within a relatively short period after acquiring the same fluoroquinolone resistance-conferring mutation and a highly related conjugative transposon. The two epidemic lineages showed distinct patterns of global spread, and the FQR2 lineage spread more widely, leading to healthcare-associated outbreaks in the UK, continental Europe and Australia. Our analysis identifies key genetic changes linked to the rapid transcontinental dissemination of epidemic C. difficile 027/BI/NAP1 and highlights the routes by which it spreads through the global healthcare system.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">93614</span>, <span class="grant">G0901743</span>, <span class="grant">G1000214</span>, <span class="grant">MR/K000551/1</span>; Wellcome Trust: <span class="grant">086418</span>, <span class="grant">093869</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">1</span>;<span class="pagerange">109-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23222960">23222960</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605770">3605770</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2478">10.1038/ng.2478</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study of depressive symptoms.</h4>
<p class="authors"><span class="author">Hek K</span>, <span class="author">Demirkan A</span>, <span class="author">Lahti J</span>, <span class="author">Terracciano A</span>, <span class="author">Teumer A</span>, <span class="author">Cornelis MC</span>, <span class="author">Amin N</span>, <span class="author">Bakshis E</span>, <span class="author">Baumert J</span>, <span class="author">Ding J</span>, <span class="author">Liu Y</span>, <span class="author">Marciante K</span>, <span class="author">Meirelles O</span>, <span class="author">Nalls MA</span>, <span class="author">Sun YV</span>, <span class="author">Vogelzangs N</span>, <span class="author">Yu L</span>, <span class="author">Bandinelli S</span>, <span class="author">Benjamin EJ</span>, <span class="author">Bennett DA</span>, <span class="author">Boomsma D</span>, <span class="author">Cannas A</span>, <span class="author">Coker LH</span>, <span class="author">de Geus E</span>, <span class="author">De Jager PL</span>, <span class="author">Diez-Roux AV</span>, <span class="author">Purcell S</span>, <span class="author">Hu FB</span>, <span class="author">Rimm EB</span>, <span class="author">Hunter DJ</span>, <span class="author">Jensen MK</span>, <span class="author">Curhan G</span>, <span class="author">Rice K</span>, <span class="author">Penman AD</span>, <span class="author">Rotter JI</span>, <span class="author">Sotoodehnia N</span>, <span class="author">Emeny R</span>, <span class="author">Eriksson JG</span>, <span class="author">Evans DA</span>, <span class="author">Ferrucci L</span>, <span class="author">Fornage M</span>, <span class="author">Gudnason V</span>, <span class="author">Hofman A</span>, <span class="author">Illig T</span>, <span class="author">Kardia S</span>, <span class="author">Kelly-Hayes M</span>, <span class="author">Koenen K</span>, <span class="author">Kraft P</span>, <span class="author">Kuningas M</span>, <span class="author">Massaro JM</span>, <span class="author">Melzer D</span>, <span class="author">Mulas A</span>, <span class="author">Mulder CL</span>, <span class="author">Murray A</span>, <span class="author">Oostra BA</span>, <span class="author">Palotie A</span>, <span class="author">Penninx B</span>, <span class="author">Petersmann A</span>, <span class="author">Pilling LC</span>, <span class="author">Psaty B</span>, <span class="author">Rawal R</span>, <span class="author">Reiman EM</span>, <span class="author">Schulz A</span>, <span class="author">Shulman JM</span>, <span class="author">Singleton AB</span>, <span class="author">Smith AV</span>, <span class="author">Sutin AR</span>, <span class="author">Uitterlinden AG</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Widen E</span>, <span class="author">Yaffe K</span>, <span class="author">Zonderman AB</span>, <span class="author">Cucca F</span>, <span class="author">Harris T</span>, <span class="author">Ladwig KH</span>, <span class="author">Llewellyn DJ</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Tanaka T</span>, <span class="author">van Duijn CM</span>, <span class="author">Grabe HJ</span>, <span class="author">Launer LJ</span>, <span class="author">Lunetta KL</span>, <span class="author">Mosley TH</span>, <span class="author">Newman AB</span>, <span class="author">Tiemeier H</span> and <span class="author">Murabito J</span></p>
<p class="affiliation">Research Centre O3, Department of Psychiatry, Erasmus MC, Rotterdam, The Netherlands; Department of Epidemiology, Erasmus MC, Rotterdam, The Netherlands.</p><p class="abstract"><strong>Background</strong>: Depression is a heritable trait that exists on a continuum of varying severity and duration. Yet, the search for genetic variants associated with depression has had few successes. We exploit the entire continuum of depression to find common variants for depressive symptoms.</p><p class="abstract"><strong>Methods</strong>: In this genome-wide association study, we combined the results of 17 population-based studies assessing depressive symptoms with the Center for Epidemiological Studies Depression Scale. Replication of the independent top hits (p&lt;1&times;10(-5)) was performed in five studies assessing depressive symptoms with other instruments. In addition, we performed a combined meta-analysis of all 22 discovery and replication studies.</p><p class="abstract"><strong>Results</strong>: The discovery sample comprised 34,549 individuals (mean age of 66.5) and no loci reached genome-wide significance (lowest p = 1.05&times;10(-7)). Seven independent single nucleotide polymorphisms were considered for replication. In the replication set (n = 16,709), we found suggestive association of one single nucleotide polymorphism with depressive symptoms (rs161645, 5q21, p = 9.19&times;10(-3)). This 5q21 region reached genome-wide significance (p = 4.78&times;10(-8)) in the overall meta-analysis combining discovery and replication studies (n = 51,258).</p><p class="abstract"><strong>Conclusions</strong>: The results suggest that only a large sample comprising more than 50,000 subjects may be sufficiently powered to detect genes for depressive symptoms.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">5UO1CA098233</span>, <span class="grant">CA49449</span>, <span class="grant">CA50385</span>, <span class="grant">CA65725</span>, <span class="grant">CA67262</span>, <span class="grant">CA87969</span>; NCRR NIH HHS: <span class="grant">UL1-RR-024156</span>, <span class="grant">UL1RR025005</span>, <span class="grant">UL1RR033176</span>; NHGRI NIH HHS: <span class="grant">U01-HG004402</span>; NHLBI NIH HHS: <span class="grant">HL075366</span>, <span class="grant">HL080295</span>, <span class="grant">HL087652</span>, <span class="grant">HL105756</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-65226</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-85239</span>, <span class="grant">N01-HC-95159</span>, <span class="grant">N01-HC-95169</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL101161</span>, <span class="grant">R01-HL087641</span>, <span class="grant">R01-HL093029</span>, <span class="grant">R01-HL70825</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">AG-023629</span>, <span class="grant">AG-027058</span>, <span class="grant">AG-15928</span>, <span class="grant">AG-20098</span>, <span class="grant">AG]916413</span>, <span class="grant">K08 AG034290</span>, <span class="grant">K08AG34290</span>, <span class="grant">N01-AG-1-2109</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>, <span class="grant">N01]AG]821336</span>, <span class="grant">P30AG10161</span>, <span class="grant">R01 AG015819</span>, <span class="grant">R01-AG29451</span>, <span class="grant">R01AG15819</span>, <span class="grant">R01AG17917</span>, <span class="grant">R01AG30146</span>, <span class="grant">ZIA AG000183-22</span>, <span class="grant">ZIA AG000183-23</span>, <span class="grant">ZIA AG000196-03</span>, <span class="grant">ZIA AG000196-04</span>, <span class="grant">ZIA AG000197-03</span>, <span class="grant">ZIA AG000197-04</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>; NIMH NIH HHS: <span class="grant">R01 MH086498</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164 13</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>; Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2013</span>;<span class="volume">73</span>;<span class="number">7</span>;<span class="pagerange">667-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23290196">23290196</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3845085">3845085</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2012.09.033">10.1016/j.biopsych.2012.09.033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Aberrant 3&#39; oligoadenylation of spliceosomal U6 small nuclear RNA in poikiloderma with neutropenia.</h4>
<p class="authors"><span class="author">Hilcenko C</span>, <span class="author">Simpson PJ</span>, <span class="author">Finch AJ</span>, <span class="author">Bowler FR</span>, <span class="author">Churcher MJ</span>, <span class="author">Jin L</span>, <span class="author">Packman LC</span>, <span class="author">Shlien A</span>, <span class="author">Campbell P</span>, <span class="author">Kirwan M</span>, <span class="author">Dokal I</span> and <span class="author">Warren AJ</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Cambridge, UK.</p><p class="abstract">The recessive disorder poikiloderma with neutropenia (PN) is caused by mutations in the C16orf57 gene that encodes the highly conserved USB1 protein. Here, we present the 1.1 &Aring; resolution crystal structure of human USB1, defining it as a member of the LigT-like superfamily of 2H phosphoesterases. We show that human USB1 is a distributive 3&#39;-5&#39; exoribonuclease that posttranscriptionally removes uridine and adenosine nucleosides from the 3&#39; end of spliceosomal U6 small nuclear RNA (snRNA), directly catalyzing terminal 2&#39;, 3&#39; cyclic phosphate formation. USB1 measures the appropriate length of the U6 oligo(U) tail by reading the position of a key adenine nucleotide (A102) and pausing 5 uridine residues downstream.We show that the 3&#39; ends of U6 snRNA in PN patient lymphoblasts are elongated and unexpectedly carry nontemplated 3&#39; oligo(A) tails that are characteristic of nuclear RNA surveillance targets. Thus, our study reveals a novel quality control pathway in which posttranscriptional 3&#39;-end processing by USB1 protects U6 snRNA from targeting and destruction by the nuclear exosome. Our data implicate aberrant oligoadenylation of U6 snRNA in the pathogenesis of the leukemia predisposition disorder PN.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800784</span>, <span class="grant">MR/L003368/1</span>, <span class="grant">U105161083</span>; Wellcome Trust: <span class="grant">079249</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2013</span>;<span class="volume">121</span>;<span class="number">6</span>;<span class="pagerange">1028-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23190533">23190533</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2012-10-461491">10.1182/blood-2012-10-461491</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dense genotyping of immune-related disease regions identifies 14 new susceptibility loci for juvenile idiopathic arthritis.</h4>
<p class="authors"><span class="author">Hinks A</span>, <span class="author">Cobb J</span>, <span class="author">Marion MC</span>, <span class="author">Prahalad S</span>, <span class="author">Sudman M</span>, <span class="author">Bowes J</span>, <span class="author">Martin P</span>, <span class="author">Comeau ME</span>, <span class="author">Sajuthi S</span>, <span class="author">Andrews R</span>, <span class="author">Brown M</span>, <span class="author">Chen WM</span>, <span class="author">Concannon P</span>, <span class="author">Deloukas P</span>, <span class="author">Edkins S</span>, <span class="author">Eyre S</span>, <span class="author">Gaffney PM</span>, <span class="author">Guthery SL</span>, <span class="author">Guthridge JM</span>, <span class="author">Hunt SE</span>, <span class="author">James JA</span>, <span class="author">Keddache M</span>, <span class="author">Moser KL</span>, <span class="author">Nigrovic PA</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Onslow ML</span>, <span class="author">Ros&eacute; CD</span>, <span class="author">Rich SS</span>, <span class="author">Steel KJ</span>, <span class="author">Wakeland EK</span>, <span class="author">Wallace CA</span>, <span class="author">Wedderburn LR</span>, <span class="author">Woo P</span>, <span class="author">Boston Children&#39;s JIA Registry</span>, <span class="author">British Society of Paediatric and Adolescent Rheumatology (BSPAR) Study Group</span>, <span class="author">Childhood Arthritis Prospective Study (CAPS)</span>, <span class="author">Childhood Arthritis Response to Medication Study (CHARMS)</span>, <span class="author">German Society for Pediatric Rheumatology (GKJR)</span>, <span class="author">JIA Gene Expression Study</span>, <span class="author">NIAMS JIA Genetic Registry</span>, <span class="author">TREAT Study</span>, <span class="author">United Kingdom Juvenile Idiopathic Arthritis Genetics Consortium (UKJIAGC)</span>, <span class="author">Bohnsack JF</span>, <span class="author">Haas JP</span>, <span class="author">Glass DN</span>, <span class="author">Langefeld CD</span>, <span class="author">Thomson W</span> and <span class="author">Thompson SD</span></p>
<p class="affiliation">1] Arthritis Research UK Epidemiology Unit, Manchester Academic Health Science Centre, The University of Manchester, Manchester, UK. [2] National Institute for Health Research Manchester Musculoskeletal Biomedical Research Unit, Central Manchester University Hospitals National Health Service Foundation Trust, Manchester Academic Health Science Centre, Manchester, UK. [3].</p><p class="abstract">We used the Immunochip array to analyze 2,816 individuals with juvenile idiopathic arthritis (JIA), comprising the most common subtypes (oligoarticular and rheumatoid factor-negative polyarticular JIA), and 13,056 controls. We confirmed association of 3 known JIA risk loci (the human leukocyte antigen (HLA) region, PTPN22 and PTPN2) and identified 14 loci reaching genome-wide significance (P &lt; 5 &times; 10(-8)) for the first time. Eleven additional new regions showed suggestive evidence of association with JIA (P &lt; 1 &times; 10(-6)). Dense mapping of loci along with bioinformatics analysis refined the associations to one gene in each of eight regions, highlighting crucial pathways, including the interleukin (IL)-2 pathway, in JIA disease pathogenesis. The entire Immunochip content, the HLA region and the top 27 loci (P &lt; 1 &times; 10(-6)) explain an estimated 18, 13 and 6% of the risk of JIA, respectively. In summary, this is the largest collection of JIA cases investigated so far and provides new insight into the genetic basis of this childhood autoimmune disease.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">U01 DK062418</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">664-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23603761">23603761</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3673707">3673707</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2614">10.1038/ng.2614</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">JAK2V617F leads to intrinsic changes in platelet formation and reactivity in a knock-in mouse model of essential thrombocythemia.</h4>
<p class="authors"><span class="author">Hobbs CM</span>, <span class="author">Manning H</span>, <span class="author">Bennett C</span>, <span class="author">Vasquez L</span>, <span class="author">Severin S</span>, <span class="author">Brain L</span>, <span class="author">Mazharian A</span>, <span class="author">Guerrero JA</span>, <span class="author">Li J</span>, <span class="author">Soranzo N</span>, <span class="author">Green AR</span>, <span class="author">Watson SP</span> and <span class="author">Ghevaert C</span></p>
<p class="affiliation">National Health System Blood and Transplant, Cambridge, United Kingdom;</p>
<p class="abstract">The principal morbidity and mortality in patients with essential thrombocythemia (ET) and polycythemia rubra vera (PV) stems from thrombotic events. Most patients with ET/PV harbor a JAK2V617F mutation, but its role in the thrombotic diathesis remains obscure. Platelet function studies in patients are difficult to interpret because of interindividual heterogeneity, reflecting variations in the proportion of platelets derived from the malignant clone, differences in the presence of additional mutations, and the effects of medical treatments. To circumvent these issues, we have studied a JAK2V617F knock-in mouse model of ET in which all megakaryocytes and platelets express JAK2V617F at a physiological level, equivalent to that present in human ET patients. We show that, in addition to increased differentiation, JAK2V617F-positive megakaryocytes display greater migratory ability and proplatelet formation. We demonstrate in a range of assays that platelet reactivity to agonists is enhanced, with a concomitant increase in platelet aggregation in vitro and a reduced duration of bleeding in vivo. These data suggest that JAK2V617F leads to intrinsic changes in both megakaryocyte and platelet biology beyond an increase in cell number. In support of this hypothesis, we identify multiple differentially expressed genes in JAK2V617F megakaryocytes that may underlie the observed biological differences.</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2013</span>;<span class="volume">122</span>;<span class="number">23</span>;<span class="pagerange">3787-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24085768">24085768</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3843237">3843237</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2013-06-501452">10.1182/blood-2013-06-501452</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genomic portrait of the emergence, evolution, and global spread of a methicillin-resistant Staphylococcus aureus pandemic.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Hsu LY</span>, <span class="author">Kurt K</span>, <span class="author">Weinert LA</span>, <span class="author">Mather AE</span>, <span class="author">Harris SR</span>, <span class="author">Strommenger B</span>, <span class="author">Layer F</span>, <span class="author">Witte W</span>, <span class="author">de Lencastre H</span>, <span class="author">Skov R</span>, <span class="author">Westh H</span>, <span class="author">Zemlickov&aacute; H</span>, <span class="author">Coombs G</span>, <span class="author">Kearns AM</span>, <span class="author">Hill RL</span>, <span class="author">Edgeworth J</span>, <span class="author">Gould I</span>, <span class="author">Gant V</span>, <span class="author">Cooke J</span>, <span class="author">Edwards GF</span>, <span class="author">McAdam PR</span>, <span class="author">Templeton KE</span>, <span class="author">McCann A</span>, <span class="author">Zhou Z</span>, <span class="author">Castillo-Ram&iacute;rez S</span>, <span class="author">Feil EJ</span>, <span class="author">Hudson LO</span>, <span class="author">Enright MC</span>, <span class="author">Balloux F</span>, <span class="author">Aanensen DM</span>, <span class="author">Spratt BG</span>, <span class="author">Fitzgerald JR</span>, <span class="author">Parkhill J</span>, <span class="author">Achtman M</span>, <span class="author">Bentley SD</span> and <span class="author">N&uuml;bel U</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB19 1SA, United Kingdom.</p><p class="abstract">The widespread use of antibiotics in association with high-density clinical care has driven the emergence of drug-resistant bacteria that are adapted to thrive in hospitalized patients. Of particular concern are globally disseminated methicillin-resistant Staphylococcus aureus (MRSA) clones that cause outbreaks and epidemics associated with health care. The most rapidly spreading and tenacious health-care-associated clone in Europe currently is EMRSA-15, which was first detected in the UK in the early 1990s and subsequently spread throughout Europe and beyond. Using phylogenomic methods to analyze the genome sequences for 193 S. aureus isolates, we were able to show that the current pandemic population of EMRSA-15 descends from a health-care-associated MRSA epidemic that spread throughout England in the 1980s, which had itself previously emerged from a primarily community-associated methicillin-sensitive population. The emergence of fluoroquinolone resistance in this EMRSA-15 subclone in the English Midlands during the mid-1980s appears to have played a key role in triggering pandemic spread, and occurred shortly after the first clinical trials of this drug. Genome-based coalescence analysis estimated that the population of this subclone over the last 20 yr has grown four times faster than its progenitor. Using comparative genomic analysis we identified the molecular genetic basis of 99.8% of the antimicrobial resistance phenotypes of the isolates, highlighting the potential of pathogen genome sequencing as a diagnostic tool. We document the genetic changes associated with adaptation to the hospital environment and with increasing drug resistance over time, and how MRSA evolution likely has been influenced by country-specific drug use regimens.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Chief Scientist Office: <span class="grant">CZB/4/717</span>; Medical Research Council: <span class="grant">G0800777</span>, <span class="grant">MR/K001744/1</span>; PHS HHS: <span class="grant">2 RO1I457838-12</span>; Wellcome Trust: <span class="grant">089472</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">4</span>;<span class="pagerange">653-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23299977">23299977</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3613582">3613582</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.147710.112">10.1101/gr.147710.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tracking the establishment of local endemic populations of an emergent enteric pathogen.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Thieu Nga TV</span>, <span class="author">Thanh DP</span>, <span class="author">Vinh H</span>, <span class="author">Kim DW</span>, <span class="author">Vu Tra MP</span>, <span class="author">Campbell JI</span>, <span class="author">Hoang NV</span>, <span class="author">Vinh NT</span>, <span class="author">Minh PV</span>, <span class="author">Thuy CT</span>, <span class="author">Nga TT</span>, <span class="author">Thompson C</span>, <span class="author">Dung TT</span>, <span class="author">Nhu NT</span>, <span class="author">Vinh PV</span>, <span class="author">Tuyet PT</span>, <span class="author">Phuc HL</span>, <span class="author">Lien NT</span>, <span class="author">Phu BD</span>, <span class="author">Ai NT</span>, <span class="author">Tien NM</span>, <span class="author">Dong N</span>, <span class="author">Parry CM</span>, <span class="author">Hien TT</span>, <span class="author">Farrar JJ</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span>, <span class="author">Thomson NR</span> and <span class="author">Baker S</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology, Bio21 Molecular Science and Biotechnology Institute, University of Melbourne, Parkville, VIC 3010, Australia.</p><p class="abstract">Shigella sonnei is a human-adapted pathogen that is emerging globally as the dominant agent of bacterial dysentery. To investigate local establishment, we sequenced the genomes of 263 Vietnamese S. sonnei isolated over 15 y. Our data show that S. sonnei was introduced into Vietnam in the 1980s and has undergone localized clonal expansion, punctuated by genomic fixation events through periodic selective sweeps. We uncover geographical spread, spatially restricted frontier populations, and convergent evolution through local gene pool sampling. This work provides a unique, high-resolution insight into the microevolution of a pioneering human pathogen during its establishment in a new host population.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093724</span>, <span class="grant">098051</span>, <span class="grant">100087</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">43</span>;<span class="pagerange">17522-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24082120">24082120</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3808646">3808646</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1308632110">10.1073/pnas.1308632110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Arginine Catabolic Mobile Element in Methicillin-Resistant Staphylococcus aureus (MRSA) Clonal Group ST239-MRSA-III Isolates in Singapore: Implications for PCR-Based Screening Tests.</h4>
<p class="authors"><span class="author">Hon PY</span>, <span class="author">Chan KS</span>, <span class="author">Holden MT</span>, <span class="author">Harris SR</span>, <span class="author">Tan TY</span>, <span class="author">Zu YB</span>, <span class="author">Krishnan P</span>, <span class="author">Oon LL</span>, <span class="author">Koh TH</span> and <span class="author">Hsu LY</span></p>
<p class="affiliation">Department of Medicine, National University Health System, Singapore, Singapore.</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2013</span>;<span class="volume">57</span>;<span class="number">3</span>;<span class="pagerange">1563-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23318798">23318798</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.02518-12">10.1128/AAC.02518-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WikiGWA: an open platform for collecting and using genome-wide association results.</h4>
<p class="authors"><span class="author">Huang J</span>, <span class="author">Liu EY</span>, <span class="author">Welch R</span>, <span class="author">Willer C</span>, <span class="author">Hindorff LA</span> and <span class="author">Li Y</span></p>
<p class="affiliation">Department of Human Genetics, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">The number of discovered genetic variants from genome-wide association (GWA) studies (GWAS) has been growing rapidly. Centralized efforts such as the National Human Genome Research Institute&#39;s GWAS catalog provide regular updates and a convenient interface for quick lookup. However, the catalog entries are manually curated and rely on data from published articles. Other tools such as SNPedia (http://www.snpedia.com) collect published results regarding functional consequences of genetic variations. Here, we propose an approach that allows individual investigators to share their GWA results through an open platform. Unlike GWAS catalog or SNPedia, wikiGWA collects first-hand GWAS results and in a much larger scale. Investigators are not only able to post a much larger amount of results, but also post results from unpublished studies, which could alleviate publication bias and facilitate identification of weak signals. Our interface allows for flexible and fast queries, and the query results are formatted to work seamlessly with the LocusZoom program for visualization and annotation. We here describe wikiGWA, made publically available at http://www.wikiGWA.org.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG006292</span>, <span class="grant">R01 HG006703</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2013</span>;<span class="volume">21</span>;<span class="number">4</span>;<span class="pagerange">471-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22929026">22929026</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3598322">3598322</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2012.187">10.1038/ejhg.2012.187</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The duck genome and transcriptome provide insight into an avian influenza virus reservoir species.</h4>
<p class="authors"><span class="author">Huang Y</span>, <span class="author">Li Y</span>, <span class="author">Burt DW</span>, <span class="author">Chen H</span>, <span class="author">Zhang Y</span>, <span class="author">Qian W</span>, <span class="author">Kim H</span>, <span class="author">Gan S</span>, <span class="author">Zhao Y</span>, <span class="author">Li J</span>, <span class="author">Yi K</span>, <span class="author">Feng H</span>, <span class="author">Zhu P</span>, <span class="author">Li B</span>, <span class="author">Liu Q</span>, <span class="author">Fairley S</span>, <span class="author">Magor KE</span>, <span class="author">Du Z</span>, <span class="author">Hu X</span>, <span class="author">Goodman L</span>, <span class="author">Tafer H</span>, <span class="author">Vignal A</span>, <span class="author">Lee T</span>, <span class="author">Kim KW</span>, <span class="author">Sheng Z</span>, <span class="author">An Y</span>, <span class="author">Searle S</span>, <span class="author">Herrero J</span>, <span class="author">Groenen MA</span>, <span class="author">Crooijmans RP</span>, <span class="author">Faraut T</span>, <span class="author">Cai Q</span>, <span class="author">Webster RG</span>, <span class="author">Aldridge JR</span>, <span class="author">Warren WC</span>, <span class="author">Bartschat S</span>, <span class="author">Kehr S</span>, <span class="author">Marz M</span>, <span class="author">Stadler PF</span>, <span class="author">Smith J</span>, <span class="author">Kraus RH</span>, <span class="author">Zhao Y</span>, <span class="author">Ren L</span>, <span class="author">Fei J</span>, <span class="author">Morisson M</span>, <span class="author">Kaiser P</span>, <span class="author">Griffin DK</span>, <span class="author">Rao M</span>, <span class="author">Pitel F</span>, <span class="author">Wang J</span> and <span class="author">Li N</span></p>
<p class="affiliation">State Key Laboratory for Agrobiotechnology, China Agricultural University, Beijing, China.</p><p class="abstract">The duck (Anas platyrhynchos) is one of the principal natural hosts of influenza A viruses. We present the duck genome sequence and perform deep transcriptome analyses to investigate immune-related genes. Our data indicate that the duck possesses a contractive immune gene repertoire, as in chicken and zebra finch, and this repertoire has been shaped through lineage-specific duplications. We identify genes that are responsive to influenza A viruses using the lung transcriptomes of control ducks and ones that were infected with either a highly pathogenic (A/duck/Hubei/49/05) or a weakly pathogenic (A/goose/Hubei/65/05) H5N1 virus. Further, we show how the duck&#39;s defense mechanisms against influenza infection have been optimized through the diversification of its &beta;-defensin and butyrophilin-like repertoires. These analyses, in combination with the genomic and transcriptomic data, provide a resource for characterizing the interaction between host and influenza viruses.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">7</span>;<span class="pagerange">776-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23749191">23749191</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2657">10.1038/ng.2657</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Olfaction and olfactory-mediated behaviour in psychiatric disease models.</h4>
<p class="authors"><span class="author">Huckins LM</span>, <span class="author">Logan DW</span> and <span class="author">S&aacute;nchez-Andrade G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">Rats and mice are the most widely used species for modelling psychiatric disease. Assessment of these rodent models typically involves the analysis of aberrant behaviour with behavioural interactions often being manipulated to generate the model. Rodents rely heavily on their excellent sense of smell and almost all their social interactions have a strong olfactory component. Therefore, experimental paradigms that exploit these olfactory-mediated behaviours are among the most robust available and are highly prevalent in psychiatric disease research. These include tests of aggression and maternal instinct, foraging, olfactory memory and habituation and the establishment of social hierarchies. An appreciation of the way that rodents regulate these behaviours in an ethological context can assist experimenters to generate better data from their models and to avoid common pitfalls. We describe some of the more commonly used behavioural paradigms from a rodent olfactory perspective and discuss their application in existing models of psychiatric disease. We introduce the four olfactory subsystems that integrate to mediate the behavioural responses and the types of sensory cue that promote them and discuss their control and practical implementation to improve experimental outcomes. In addition, because smell is critical for normal behaviour in rodents and yet olfactory dysfunction is often associated with neuropsychiatric disease, we introduce some tests for olfactory function that can be applied to rodent models of psychiatric disorders as part of behavioural analysis.</p>
<p class="publication"><span class="title">Cell and tissue research</span>&nbsp;<span class="year">2013</span>;<span class="volume">354</span>;<span class="number">1</span>;<span class="pagerange">69-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23604803">23604803</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00441-013-1617-7">10.1007/s00441-013-1617-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel Loci Associated with Increased Risk of Sudden Cardiac Death in the Context of Coronary Artery Disease.</h4>
<p class="authors"><span class="author">Huertas-Vazquez A</span>, <span class="author">Nelson CP</span>, <span class="author">Guo X</span>, <span class="author">Reinier K</span>, <span class="author">Uy-Evanado A</span>, <span class="author">Teodorescu C</span>, <span class="author">Ayala J</span>, <span class="author">Jerger K</span>, <span class="author">Chugh H</span>, <span class="author">Wtccc</span>, <span class="author">Braund PS</span>, <span class="author">Deloukas P</span>, <span class="author">Hall AS</span>, <span class="author">Balmforth AJ</span>, <span class="author">Jones M</span>, <span class="author">Taylor KD</span>, <span class="author">Pulit SL</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Gunson K</span>, <span class="author">Jui J</span>, <span class="author">Rotter JI</span>, <span class="author">Albert CM</span>, <span class="author">Samani NJ</span> and <span class="author">Chugh SS</span></p>
<p class="affiliation">The Heart Institute, Cedars-Sinai Medical Center, Los Angeles, California, United States of America.</p>
<p class="abstract">Background: Recent genome-wide association studies (GWAS) have identified novel loci associated with sudden cardiac death (SCD). Despite this progress, identified DNA variants account for a relatively small portion of overall SCD risk, suggesting that additional loci contributing to SCD susceptibility await discovery. The objective of this study was to identify novel DNA variation associated with SCD in the context of coronary artery disease (CAD).

Using the MetaboChip custom array we conducted a case-control association analysis of 119,117 SNPs in 948 SCD cases (with underlying CAD) from the Oregon Sudden Unexpected Death Study (Oregon-SUDS) and 3,050 controls with CAD from the Wellcome Trust Case-Control Consortium (WTCCC). Two newly identified loci were significantly associated with increased risk of SCD after correction for multiple comparisons at: rs6730157 in the RAB3GAP1 gene on chromosome 2 (Pâ€Š=â€Š4.93Ã—10(-12), ORâ€Š=â€Š1.60) and rs2077316 in the ZNF365 gene on chromosome 10 (Pâ€Š=â€Š3.64Ã—10(-8), ORâ€Š=â€Š2.41).

Conclusions: Our findings suggest that RAB3GAP1 and ZNF365 are relevant candidate genes for SCD and will contribute to the mechanistic understanding of SCD susceptibility.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e59905</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23593153">23593153</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0059905">10.1371/journal.pone.0059905</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Negligible impact of rare autoimmune-locus coding-region variants on missing heritability.</h4>
<p class="authors"><span class="author">Hunt KA</span>, <span class="author">Mistry V</span>, <span class="author">Bockett NA</span>, <span class="author">Ahmad T</span>, <span class="author">Ban M</span>, <span class="author">Barker JN</span>, <span class="author">Barrett JC</span>, <span class="author">Blackburn H</span>, <span class="author">Brand O</span>, <span class="author">Burren O</span>, <span class="author">Capon F</span>, <span class="author">Compston A</span>, <span class="author">Gough SC</span>, <span class="author">Jostins L</span>, <span class="author">Kong Y</span>, <span class="author">Lee JC</span>, <span class="author">Lek M</span>, <span class="author">Macarthur DG</span>, <span class="author">Mansfield JC</span>, <span class="author">Mathew CG</span>, <span class="author">Mein CA</span>, <span class="author">Mirza M</span>, <span class="author">Nutland S</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Papouli E</span>, <span class="author">Parkes M</span>, <span class="author">Rich SS</span>, <span class="author">Sawcer S</span>, <span class="author">Satsangi J</span>, <span class="author">Simmonds MJ</span>, <span class="author">Trembath RC</span>, <span class="author">Walker NM</span>, <span class="author">Wozniak E</span>, <span class="author">Todd JA</span>, <span class="author">Simpson MA</span>, <span class="author">Plagnol V</span> and <span class="author">van Heel DA</span></p>
<p class="affiliation">Blizard Institute, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London E1 2AT, UK.</p>
<p class="abstract">Genome-wide association studies (GWAS) have identified common variants of modest-effect size at hundreds of loci for common autoimmune diseases; however, a substantial fraction of heritability remains unexplained, to which rare variants may contribute. To discover rare variants and test them for association with a phenotype, most studies re-sequence a small initial sample size and then genotype the discovered variants in a larger sample set. This approach fails to analyse a large fraction of the rare variants present in the entire sample set. Here we perform simultaneous amplicon-sequencing-based variant discovery and genotyping for coding exons of 25 GWAS risk genes in 41,911 UK residents of white European origin, comprising 24,892 subjects with six autoimmune disease phenotypes and 17,019 controls, and show that rare coding-region variants at known loci have a negligible role in common autoimmune disease susceptibility. These results do not support the rare-variant synthetic genome-wide-association hypothesis (in which unobserved rare causal variants lead to association detected at common tag variants). Many known autoimmune disease risk loci contain multiple, independently associated, common and low-frequency variants, and so genes at these loci are a priori stronger candidates for harbouring rare coding-region variants than other genes. Our data indicate that the missing heritability for common autoimmune diseases may not be attributable to the rare coding-region variant portion of the allelic spectrum, but perhaps, as others have proposed, may be a result of many common-variant loci of weak effect.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23698362">23698362</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12170">10.1038/nature12170</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">REAPR: a universal tool for genome assembly evaluation.</h4>
<p class="authors"><span class="author">Hunt M</span>, <span class="author">Kikuchi T</span>, <span class="author">Sanders M</span>, <span class="author">Newbold C</span>, <span class="author">Berriman M</span> and <span class="author">Otto TD</span></p><p class="abstract">Methods to reliably assess the accuracy of genome sequence data are lacking. Currently completeness is only described qualitatively and mis-assemblies are overlooked. Here we present REAPR, a tool that precisely identifies errors in genome assemblies without the need for a reference sequence. We have validated REAPR on complete genomes or de novo assemblies from bacteria, malaria and Caenorhabditis elegans, and demonstrate that 86% and 82% of the human and mouse reference genomes are error-free, respectively. When applied to an ongoing genome project, REAPR provides corrected assembly statistics allowing the quantitative comparison of multiple assemblies. REAPR is available at http://www.sanger.ac.uk/resources/software/reapr/.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">082130/Z/07/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">R47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23710727">23710727</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3798757">3798757</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-5-r47">10.1186/gb-2013-14-5-r47</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">LUD, a new protein domain associated with lactate utilization.</h4>
<p class="authors"><span class="author">Hwang WC</span>, <span class="author">Bakolitsa C</span>, <span class="author">Punta M</span>, <span class="author">Coggill PC</span>, <span class="author">Bateman A</span>, <span class="author">Axelrod HL</span>, <span class="author">Rawlings ND</span>, <span class="author">Sedova M</span>, <span class="author">Peterson SN</span>, <span class="author">Eberhardt RY</span>, <span class="author">Aravind L</span>, <span class="author">Pascual J</span> and <span class="author">Godzik A</span></p>
<p class="affiliation">Joint Center for Structural Genomics, La Jolla, CA 92037, USA. <a href="mailto:%77%63%68%77%61%6E%67%40%73%61%6E%66%6F%72%64%62%75%72%6E%68%61%6D.%6F%72%67">&#119;&#99;&#104;&#119;&#97;&#110;&#103;&#64;&#115;&#97;&#110;&#102;&#111;&#114;&#100;&#98;&#117;&#114;&#110;&#104;&#97;&#109;&#46;&#111;&#114;&#103;</a>.</p>
<p class="abstract">Background: A novel highly conserved protein domain, DUF162 [Pfam: PF02589], can be mapped to two proteins: LutB and LutC. Both proteins are encoded by a highly conserved LutABC operon, which has been implicated in lactate utilization in bacteria. Based on our analysis of its sequence, structure, and recent experimental evidence reported by other groups, we hereby redefine DUF162 as the LUD domain family.

Results: JCSG solved the first crystal structure [PDB:2G40] from the LUD domain family: LutC protein, encoded by ORF DR_1909, of Deinococcus radiodurans. LutC shares features with domains in the functionally diverse ISOCOT superfamily. We have observed that the LUD domain has an increased abundance in the human gut microbiome.

Conclusions: We propose a model for the substrate and cofactor binding and regulation in LUD domain. The significance of LUD-containing proteins in the human gut microbiome, and the implication of lactate metabolism in the radiation-resistance of Deinococcus radiodurans are discussed.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">341</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24274019">24274019</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-14-341">10.1186/1471-2105-14-341</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Astroglial IFITM3 mediates neuronal impairments following neonatal immune challenge in mice.</h4>
<p class="authors"><span class="author">Ibi D</span>, <span class="author">Nagai T</span>, <span class="author">Nakajima A</span>, <span class="author">Mizoguchi H</span>, <span class="author">Kawase T</span>, <span class="author">Tsuboi D</span>, <span class="author">Kano SI</span>, <span class="author">Sato Y</span>, <span class="author">Hayakawa M</span>, <span class="author">Lange UC</span>, <span class="author">Adams DJ</span>, <span class="author">Surani MA</span>, <span class="author">Satoh T</span>, <span class="author">Sawa A</span>, <span class="author">Kaibuchi K</span>, <span class="author">Nabeshima T</span> and <span class="author">Yamada K</span></p>
<p class="affiliation">Department of Neuropsychopharmacology and Hospital Pharmacy, Nagoya University Graduate School of Medicine, Nagoya, Japan; Department of Chemical Pharmacology, Graduate School of Pharmaceutical Sciences, Meijo University, Nagoya, Japan.</p>
<p class="abstract">Interferon-induced transmembrane protein 3 (IFITM3) Ä±plays a crucial role in the antiviral responses of Type I interferons (IFNs). The role of IFITM3 in the central nervous system (CNS) is, however, largely unknown, despite the fact that its expression is increased in the brains of patients with neurologic and neuropsychiatric diseases. Here, we show the role of IFITM3 in long-lasting neuronal impairments in mice following polyriboinosinic-polyribocytidylic acid (polyI:C, a synthetic double-stranded RNA)-induced immune challenge during the early stages of development. We found that the induction of IFITM3 expression in the brain of mice treated with polyI:C was observed only in astrocytes. Cultured astrocytes were activated by polyI:C treatment, leading to an increase in the mRNA levels of inflammatory cytokines as well as Ifitm3. When cultured neurons were treated with the conditioned medium of polyI:C-treated astrocytes (polyI:C-ACM), neurite development was impaired. These polyI:C-ACM-induced neurodevelopmental abnormalities were alleviated by ifitm3(-) (/) (-) astrocyte-conditioned medium. Furthermore, decreases of MAP2 expression, spine density, and dendrite complexity in the frontal cortex as well as memory impairment were evident in polyI:C-treated wild-type mice, but such neuronal impairments were not observed in ifitm3(-) (/) (-) mice. We also found that IFITM3 proteins were localized to the early endosomes of astrocytes following polyI:C treatment and reduced endocytic activity. These findings suggest that the induction of IFITM3 expression in astrocytes by the activation of the innate immune system during the early stages of development has non-cell autonomous effects that affect subsequent neurodevelopment, leading to neuropathological impairments and brain dysfunction, by impairing endocytosis in astrocytes. GLIA 2013.</p>
<p class="publication"><span class="title">Glia</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382131">23382131</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fglia.22461">10.1002/glia.22461</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of high-throughput technologies in clinical cancer genomics.</h4>
<p class="authors"><span class="author">Idris SF</span>, <span class="author">Ahmad SS</span>, <span class="author">Scott MA</span>, <span class="author">Vassiliou GS</span> and <span class="author">Hadfield J</span></p>
<p class="affiliation">Department of Hematology/Oncology, Cambridge University NHS Hospitals Foundation Trust, Cambridge, CB2&nbsp;0QQ, UK.</p><p class="abstract">Cancer is a genetic disease driven by both heritable and somatic alterations in DNA, which underpin not only oncogenesis but also progression and eventual metastasis. The major impetus for elucidating the nature and function of somatic mutations in cancer genomes is the potential for the development of effective targeted anticancer therapies. Over the last decade, high-throughput technologies have allowed us unprecedented access to a host of cancer genomes, leading to an influx of new information about their pathobiology. The challenge now is to integrate such emerging information into clinical practice to achieve tangible benefits for cancer patients. This review examines the roles array-based comparative genomic hybridization and next-generation sequencing are playing in furthering our understanding of both hematological and solid-organ tumors. Furthermore, the authors discuss the current challenges in translating the role of these technologies from bench to bedside.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095663</span></p>
<p class="publication"><span class="title">Expert review of molecular diagnostics</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="number">2</span>;<span class="pagerange">167-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23477557">23477557</a>; DOI: <a href="http://dx.doi.org/10.1586%2Ferm.13.1">10.1586/erm.13.1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inferring genome-wide recombination landscapes from advanced intercross lines: application to yeast crosses.</h4>
<p class="authors"><span class="author">Illingworth CJ</span>, <span class="author">Parts L</span>, <span class="author">Bergstr&ouml;m A</span>, <span class="author">Liti G</span> and <span class="author">Mustonen V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">Accurate estimates of recombination rates are of great importance for understanding evolution. In an experimental genetic cross, recombination breaks apart and rejoins genetic material, such that the genomes of the resulting isolates are comprised of distinct blocks of differing parental origin. We here describe a method exploiting this fact to infer genome-wide recombination profiles from sequenced isolates from an advanced intercross line (AIL). We verified the accuracy of the method against simulated data. Next, we sequenced 192 isolates from a twelve-generation cross between West African and North American yeast Saccharomyces cerevisiae strains and inferred the underlying recombination landscape at a fine genomic resolution (mean segregating site distance 0.22 kb). Comparison was made with landscapes inferred for a similar cross between four yeast strains, and with a previous single-generation, intra-strain cross (Mancera et al., Nature 2008). Moderate congruence was identified between landscapes (correlation 0.58-0.77 at 5 kb resolution), albeit with variance between mean genome-wide recombination rates. The multiple generations of mating undergone in the AILs gave more precise inference of recombination rates than could be achieved from a single-generation cross, in particular in identifying recombination cold-spots. The recombination landscapes we describe have particular utility; both AILs are part of a resource to study complex yeast traits (see e.g. Parts et al., Genome Res 2011). Our results will enable future applications of this resource to take better account of local linkage structure heterogeneities. Our method has general applicability to other crossing experiments, including a variety of experimental designs.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e62266</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23658715">23658715</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3642125">3642125</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0062266">10.1371/journal.pone.0062266</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of immune-related loci identifies 48 new susceptibility variants for multiple sclerosis.</h4>
<p class="authors"><span class="author">International Multiple Sclerosis Genetics Consortium (IMSGC)</span>, <span class="author">Beecham AH</span>, <span class="author">Patsopoulos NA</span>, <span class="author">Xifara DK</span>, <span class="author">Davis MF</span>, <span class="author">Kemppinen A</span>, <span class="author">Cotsapas C</span>, <span class="author">Shah TS</span>, <span class="author">Spencer C</span>, <span class="author">Booth D</span>, <span class="author">Goris A</span>, <span class="author">Oturai A</span>, <span class="author">Saarela J</span>, <span class="author">Fontaine B</span>, <span class="author">Hemmer B</span>, <span class="author">Martin C</span>, <span class="author">Zipp F</span>, <span class="author">D&#39;Alfonso S</span>, <span class="author">Martinelli-Boneschi F</span>, <span class="author">Taylor B</span>, <span class="author">Harbo HF</span>, <span class="author">Kockum I</span>, <span class="author">Hillert J</span>, <span class="author">Olsson T</span>, <span class="author">Ban M</span>, <span class="author">Oksenberg JR</span>, <span class="author">Hintzen R</span>, <span class="author">Barcellos LF</span>, <span class="author">Wellcome Trust Case Control Consortium 2 (WTCCC2)</span>, <span class="author">International IBD Genetics Consortium (IIBDGC)</span>, <span class="author">Agliardi C</span>, <span class="author">Alfredsson L</span>, <span class="author">Alizadeh M</span>, <span class="author">Anderson C</span>, <span class="author">Andrews R</span>, <span class="author">S&oslash;ndergaard HB</span>, <span class="author">Baker A</span>, <span class="author">Band G</span>, <span class="author">Baranzini SE</span>, <span class="author">Barizzone N</span>, <span class="author">Barrett J</span>, <span class="author">Bellenguez C</span>, <span class="author">Bergamaschi L</span>, <span class="author">Bernardinelli L</span>, <span class="author">Berthele A</span>, <span class="author">Biberacher V</span>, <span class="author">Binder TM</span>, <span class="author">Blackburn H</span>, <span class="author">Bomfim IL</span>, <span class="author">Brambilla P</span>, <span class="author">Broadley S</span>, <span class="author">Brochet B</span>, <span class="author">Brundin L</span>, <span class="author">Buck D</span>, <span class="author">Butzkueven H</span>, <span class="author">Caillier SJ</span>, <span class="author">Camu W</span>, <span class="author">Carpentier W</span>, <span class="author">Cavalla P</span>, <span class="author">Celius EG</span>, <span class="author">Coman I</span>, <span class="author">Comi G</span>, <span class="author">Corrado L</span>, <span class="author">Cosemans L</span>, <span class="author">Cournu-Rebeix I</span>, <span class="author">Cree BA</span>, <span class="author">Cusi D</span>, <span class="author">Damotte V</span>, <span class="author">Defer G</span>, <span class="author">Delgado SR</span>, <span class="author">Deloukas P</span>, <span class="author">di Sapio A</span>, <span class="author">Dilthey AT</span>, <span class="author">Donnelly P</span>, <span class="author">Dubois B</span>, <span class="author">Duddy M</span>, <span class="author">Edkins S</span>, <span class="author">Elovaara I</span>, <span class="author">Esposito F</span>, <span class="author">Evangelou N</span>, <span class="author">Fiddes B</span>, <span class="author">Field J</span>, <span class="author">Franke A</span>, <span class="author">Freeman C</span>, <span class="author">Frohlich IY</span>, <span class="author">Galimberti D</span>, <span class="author">Gieger C</span>, <span class="author">Gourraud PA</span>, <span class="author">Graetz C</span>, <span class="author">Graham A</span>, <span class="author">Grummel V</span>, <span class="author">Guaschino C</span>, <span class="author">Hadjixenofontos A</span>, <span class="author">Hakonarson H</span>, <span class="author">Halfpenny C</span>, <span class="author">Hall G</span>, <span class="author">Hall P</span>, <span class="author">Hamsten A</span>, <span class="author">Harley J</span>, <span class="author">Harrower T</span>, <span class="author">Hawkins C</span>, <span class="author">Hellenthal G</span>, <span class="author">Hillier C</span>, <span class="author">Hobart J</span>, <span class="author">Hoshi M</span>, <span class="author">Hunt SE</span>, <span class="author">Jagodic M</span>, <span class="author">Jel&#x10D;i&#x107; I</span>, <span class="author">Jochim A</span>, <span class="author">Kendall B</span>, <span class="author">Kermode A</span>, <span class="author">Kilpatrick T</span>, <span class="author">Koivisto K</span>, <span class="author">Konidari I</span>, <span class="author">Korn T</span>, <span class="author">Kronsbein H</span>, <span class="author">Langford C</span>, <span class="author">Larsson M</span>, <span class="author">Lathrop M</span>, <span class="author">Lebrun-Frenay C</span>, <span class="author">Lechner-Scott J</span>, <span class="author">Lee MH</span>, <span class="author">Leone MA</span>, <span class="author">Lepp&auml; V</span>, <span class="author">Liberatore G</span>, <span class="author">Lie BA</span>, <span class="author">Lill CM</span>, <span class="author">Lind&eacute;n M</span>, <span class="author">Link J</span>, <span class="author">Luessi F</span>, <span class="author">Lycke J</span>, <span class="author">Macciardi F</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">Manrique CP</span>, <span class="author">Martin R</span>, <span class="author">Martinelli V</span>, <span class="author">Mason D</span>, <span class="author">Mazibrada G</span>, <span class="author">McCabe C</span>, <span class="author">Mero IL</span>, <span class="author">Mescheriakova J</span>, <span class="author">Moutsianas L</span>, <span class="author">Myhr KM</span>, <span class="author">Nagels G</span>, <span class="author">Nicholas R</span>, <span class="author">Nilsson P</span>, <span class="author">Piehl F</span>, <span class="author">Pirinen M</span>, <span class="author">Price SE</span>, <span class="author">Quach H</span>, <span class="author">Reunanen M</span>, <span class="author">Robberecht W</span>, <span class="author">Robertson NP</span>, <span class="author">Rodegher M</span>, <span class="author">Rog D</span>, <span class="author">Salvetti M</span>, <span class="author">Schnetz-Boutaud NC</span>, <span class="author">Sellebjerg F</span>, <span class="author">Selter RC</span>, <span class="author">Schaefer C</span>, <span class="author">Shaunak S</span>, <span class="author">Shen L</span>, <span class="author">Shields S</span>, <span class="author">Siffrin V</span>, <span class="author">Slee M</span>, <span class="author">Sorensen PS</span>, <span class="author">Sorosina M</span>, <span class="author">Sospedra M</span>, <span class="author">Spurkland A</span>, <span class="author">Strange A</span>, <span class="author">Sundqvist E</span>, <span class="author">Thijs V</span>, <span class="author">Thorpe J</span>, <span class="author">Ticca A</span>, <span class="author">Tienari P</span>, <span class="author">van Duijn C</span>, <span class="author">Visser EM</span>, <span class="author">Vucic S</span>, <span class="author">Westerlind H</span>, <span class="author">Wiley JS</span>, <span class="author">Wilkins A</span>, <span class="author">Wilson JF</span>, <span class="author">Winkelmann J</span>, <span class="author">Zajicek J</span>, <span class="author">Zindler E</span>, <span class="author">Haines JL</span>, <span class="author">Pericak-Vance MA</span>, <span class="author">Ivinson AJ</span>, <span class="author">Stewart G</span>, <span class="author">Hafler D</span>, <span class="author">Hauser SL</span>, <span class="author">Compston A</span>, <span class="author">McVean G</span>, <span class="author">De Jager P</span>, <span class="author">Sawcer SJ</span> and <span class="author">McCauley JL</span></p>
<p class="affiliation">1] John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, Florida, USA. [2].</p><p class="abstract">Using the ImmunoChip custom genotyping array, we analyzed 14,498 subjects with multiple sclerosis and 24,091 healthy controls for 161,311 autosomal variants and identified 135 potentially associated regions (P &lt; 1.0 &times; 10(-4)). In a replication phase, we combined these data with previous genome-wide association study (GWAS) data from an independent 14,802 subjects with multiple sclerosis and 26,703 healthy controls. In these 80,094 individuals of European ancestry, we identified 48 new susceptibility variants (P &lt; 5.0 &times; 10(-8)), 3 of which we found after conditioning on previously identified variants. Thus, there are now 110 established multiple sclerosis risk variants at 103 discrete loci outside of the major histocompatibility complex. With high-resolution Bayesian fine mapping, we identified five regions where one variant accounted for more than 50% of the posterior probability of association. This study enhances the catalog of multiple sclerosis risk variants and illustrates the value of fine mapping in the resolution of GWAS signals.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0700061</span>; Multiple Sclerosis Society: <span class="grant">862</span>, <span class="grant">894</span>, <span class="grant">898</span>, <span class="grant">955</span>; NCI NIH HHS: <span class="grant">R01 CA104021</span>; NCRR NIH HHS: <span class="grant">UL1 RR024975</span>; NIAID NIH HHS: <span class="grant">R01 AI076544</span>; NIEHS NIH HHS: <span class="grant">R01 ES017080</span>; NIGMS NIH HHS: <span class="grant">RC2 GM093080</span>; NINDS NIH HHS: <span class="grant">R01 NS026799</span>, <span class="grant">R01 NS032830</span>, <span class="grant">R01 NS049477</span>, <span class="grant">R01 NS049510</span>, <span class="grant">R01 NS067305</span>, <span class="grant">RC2 NS070340</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">085475DONNELLY</span>, <span class="grant">085475PELTONEN</span>, <span class="grant">090532</span>, <span class="grant">095552</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">11</span>;<span class="pagerange">1353-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24076602">24076602</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3832895">3832895</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2770">10.1038/ng.2770</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Network based elucidation of drug response: from modulators to targets.</h4>
<p class="authors"><span class="author">Iorio F</span>, <span class="author">Saez-Rodriguez J</span> and <span class="author">di Bernardo D</span></p>
<p class="affiliation">Telethon Institute of Genetics and Medicine, Naples, Italy. <a href="mailto:%64%69%62%65%72%6E%61%72%64%6F%40%74%69%67%65%6D.%69%74">&#100;&#105;&#98;&#101;&#114;&#110;&#97;&#114;&#100;&#111;&#64;&#116;&#105;&#103;&#101;&#109;&#46;&#105;&#116;</a>.</p>
<p class="abstract">: Network-based drug discovery aims at harnessing the power of networks to investigate the mechanism of action of existing drugs, or new molecules, in order to identify innovative therapeutic treatments. In this review, we describe some of the most recent advances in the field of network pharmacology, starting with approaches relying on computational models of transcriptional networks, then moving to protein and signaling network models and concluding with "drug networks". These networks are derived from different sources of experimental data, or literature-based analysis, and provide a complementary view of drug mode of action. Molecular and drug networks are powerful integrated computational and experimental approaches that will likely speed up and improve the drug discovery process, once fully integrated into the academic and industrial drug discovery pipeline.</p>
<p class="publication"><span class="title">BMC systems biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">7</span>;<span class="pagerange">139</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24330611">24330611</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3878740">3878740</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1752-0509-7-139">10.1186/1752-0509-7-139</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Cell-surface Phylome for African Trypanosomes.</h4>
<p class="authors"><span class="author">Jackson AP</span>, <span class="author">Allison HC</span>, <span class="author">Barry JD</span>, <span class="author">Field MC</span>, <span class="author">Hertz-Fowler C</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Pathogen Genomics Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, England, United Kingdom ; Department of Infection Biology, Institute of Infection and Global Health, University of Liverpool, Liverpool, England, United Kingdom.</p>
<p class="abstract">The cell surface of Trypanosoma brucei, like many protistan blood parasites, is crucial for mediating host-parasite interactions and is instrumental to the initiation, maintenance and severity of infection. Previous comparisons with the related trypanosomatid parasites T. cruzi and Leishmania major suggest that the cell-surface proteome of T. brucei is largely taxon-specific. Here we compare genes predicted to encode cell surface proteins of T. brucei with those from two related African trypanosomes, T. congolense and T. vivax. We created a cell surface phylome (CSP) by estimating phylogenies for 79 gene families with putative surface functions to understand the more recent evolution of African trypanosome surface architecture. Our findings demonstrate that the transferrin receptor genes essential for bloodstream survival in T. brucei are conserved in T. congolense but absent from T. vivax and include an expanded gene family of insect stage-specific surface glycoproteins that includes many currently uncharacterized genes. We also identify species-specific features and innovations and confirm that these include most expression site-associated genes (ESAGs) in T. brucei, which are absent from T. congolense and T. vivax. The CSP presents the first global picture of the origins and dynamics of cell surface architecture in African trypanosomes, representing the principal differences in genomic repertoire between African trypanosome species and provides a basis from which to explore the developmental and pathological differences in surface architectures. All data can be accessed at: http://www.genedb.org/Page/trypanosoma_surface_phylome.</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">e2121</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23556014">23556014</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605285">3605285</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0002121">10.1371/journal.pntd.0002121</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">iAnn: An Event Sharing Platform for the Life Sciences.</h4>
<p class="authors"><span class="author">Jimenez RC</span>, <span class="author">Albar JP</span>, <span class="author">Bhak J</span>, <span class="author">Blatter MC</span>, <span class="author">Blicher T</span>, <span class="author">Brazas MD</span>, <span class="author">Brooksbank C</span>, <span class="author">Budd A</span>, <span class="author">De Las Rivas J</span>, <span class="author">Dreyer J</span>, <span class="author">van Driel MA</span>, <span class="author">Dunn MJ</span>, <span class="author">Fernandes PL</span>, <span class="author">van Gelder CW</span>, <span class="author">Hermjakob H</span>, <span class="author">Ioannidis V</span>, <span class="author">Judge DP</span>, <span class="author">Kahlem P</span>, <span class="author">Korpelainen E</span>, <span class="author">Kraus HJ</span>, <span class="author">Loveland J</span>, <span class="author">Mayer C</span>, <span class="author">McDowall J</span>, <span class="author">Moran F</span>, <span class="author">Mulder N</span>, <span class="author">Nyronen T</span>, <span class="author">Rother K</span>, <span class="author">Salazar GA</span>, <span class="author">Schneider MV</span>, <span class="author">Schneider R</span>, <span class="author">Via A</span>, <span class="author">Villaveces JM</span>, <span class="author">Yu P</span>, <span class="author">Attwood TK</span> and <span class="author">Corpas M</span></p>
<p class="affiliation">European Bioinformatics Institute, Hinxton, UK, National Center for Biotecnology-CSIC, Madrid, Spain, Theragen BioInstitute, South Korea, SIB Swiss Institute of Bioinformatics, Gen&egrave;ve, Switzerland, NNF Center for Protein Research, Copenhagen, Denmark, Ontario Institute for Cancer Research, Toronto, Canada, European Molecular Biology Laboratory, Heidelberg, Germany, Cancer Research Center (IBMCC-CSIC), Salamanca, Spain, Netherlands Bioinformatics Centre, Nijmegen, Netherlands, University College Dublin, Dublin, Ireland, Instituto Gulbenkian de Ci&ecirc;ncia, Oeiras, Portugal, SIB Swiss Institute of Bioinformatics, Lausanne, Switzerland, University of Cambridge, Cambridge, UK, CSC-Scientific Computing Ltd., Espoo, Finland, WILEY-VCH Verlag, Weinheim, Germany, Wellcome Trust Sanger Institute, Hinxton, UK, Universidad Complutense, Madrid, Spain, Dept. Clinical Laboratory Sciences, IIDMM, University of Cape Town, South Africa, Academis Training, Berlin, Germany, The Genome Analysis Centre, Norwich, UK, Luxembourg Center for Systems Biomedicine, University of Luxembourg, Luxemburg, Sapienza University, Rome, Italy, Max Planck Institute for Biology of Ageing, Cologne, Germany, Itico, Fulbourn, Cambridge, UK, The University of Manchester, Manchester, UK.</p><p class="abstract">SUMMARY: We present iAnn, an open source community-driven platform for dissemination of life science events, such as courses, conferences and workshops. iAnn allows automatic visualisation and integration of customised event reports. A central repository lies at the core of the platform: curators add submitted events, and these are subsequently accessed via Web services. Thus, once an iAnn widget is incorporated into a website, it permanently shows timely, relevant information as if it were native to the remote site. At the same time, announcements submitted to the repository are automatically disseminated to all portals that query the system. To facilitate the visualisation of announcements, iAnn provides powerful filtering options and views, integrated in Google Maps and Google Calendar. All iAnn widgets are freely available. AVAILABILITY: http://iann.pro/iannviewer CONTACT: <a href="mailto:%6D%61%6E%75%65%6C.%63%6F%72%70%61%73%40%74%67%61%63.%61%63.%75%6B">&#109;&#97;&#110;&#117;&#101;&#108;&#46;&#99;&#111;&#114;&#112;&#97;&#115;&#64;&#116;&#103;&#97;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23742982">23742982</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtt306">10.1093/bioinformatics/btt306</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The CD225 domain of IFITM3 is required for both IFITM protein association and inhibition of influenza A virus and dengue virus replication.</h4>
<p class="authors"><span class="author">John SP</span>, <span class="author">Chin CR</span>, <span class="author">Perreira JM</span>, <span class="author">Feeley EM</span>, <span class="author">Aker AM</span>, <span class="author">Savidis G</span>, <span class="author">Smith SE</span>, <span class="author">Elia AE</span>, <span class="author">Everitt AR</span>, <span class="author">Vora M</span>, <span class="author">Pertel T</span>, <span class="author">Elledge SJ</span>, <span class="author">Kellam P</span> and <span class="author">Brass AL</span></p>
<p class="affiliation">Department of Microbiology and Physiological Systems, University of Massachusetts Medical School, Worcester, Massachusetts, USA.</p><p class="abstract">The interferon-induced transmembrane protein 3 (IFITM3) gene is an interferon-stimulated gene that inhibits the replication of multiple pathogenic viruses in vitro and in vivo. IFITM3 is a member of a large protein superfamily, whose members share a functionally undefined area of high amino acid conservation, the CD225 domain. We performed mutational analyses of IFITM3 and identified multiple residues within the CD225 domain, consisting of the first intramembrane domain (intramembrane domain 1 [IM1]) and a conserved intracellular loop (CIL), that are required for restriction of both influenza A virus (IAV) and dengue virus (DENV) infection in vitro. Two phenylalanines within IM1 (F75 and F78) also mediate a physical association between IFITM proteins, and the loss of this interaction decreases IFITM3-mediated restriction. By extension, similar IM1-mediated associations may contribute to the functions of additional members of the CD225 domain family. IFITM3&#39;s distal N-terminal domain is also needed for full antiviral activity, including a tyrosine (Y20), whose alteration results in mislocalization of a portion of IFITM3 to the cell periphery and surface. Comparative analyses demonstrate that similar molecular determinants are needed for IFITM3&#39;s restriction of both IAV and DENV. However, a portion of the CIL including Y99 and R87 is preferentially needed for inhibition of the orthomyxovirus. Several IFITM3 proteins engineered with rare single-nucleotide polymorphisms demonstrated reduced expression or mislocalization, and these events were associated with enhanced viral replication in vitro, suggesting that possessing such alleles may impact an individual&#39;s risk for viral infection. On the basis of this and other data, we propose a model for IFITM3-mediated restriction.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NIAID NIH HHS: <span class="grant">1R01AI091786</span>, <span class="grant">R01 AI091786</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2013</span>;<span class="volume">87</span>;<span class="number">14</span>;<span class="pagerange">7837-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23658454">23658454</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3700195">3700195</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.00481-13">10.1128/JVI.00481-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Presynaptic maturation in auditory hair cells requires a critical period of sensory-independent spiking activity.</h4>
<p class="authors"><span class="author">Johnson SL</span>, <span class="author">Kuhn S</span>, <span class="author">Franz C</span>, <span class="author">Ingham N</span>, <span class="author">Furness DN</span>, <span class="author">Knipper M</span>, <span class="author">Steel KP</span>, <span class="author">Adelman JP</span>, <span class="author">Holley MC</span> and <span class="author">Marcotti W</span></p>
<p class="affiliation">Department of Biomedical Science, University of Sheffield, Sheffield S10 2TN, United Kingdom.</p>
<p class="abstract">The development of neural circuits relies on spontaneous electrical activity that occurs during immature stages of development. In the developing mammalian auditory system, spontaneous calcium action potentials are generated by inner hair cells (IHCs), which form the primary sensory synapse. It remains unknown whether this electrical activity is required for the functional maturation of the auditory system. We found that sensory-independent electrical activity controls synaptic maturation in IHCs. We used a mouse model in which the potassium channel SK2 is normally overexpressed, but can be modulated in vivo using doxycycline. SK2 overexpression affected the frequency and duration of spontaneous action potentials, which prevented the development of the Ca(2+)-sensitivity of vesicle fusion at IHC ribbon synapses, without affecting their morphology or general cell development. By manipulating the in vivo expression of SK2 channels, we identified the "critical period" during which spiking activity influences IHC synaptic maturation. Here we provide direct evidence that IHC development depends upon a specific temporal pattern of calcium spikes before sound-driven neuronal activity.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">21</span>;<span class="pagerange">8720-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23650376">23650376</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1219578110">10.1073/pnas.1219578110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Open science and community norms: Data retention and publication moratoria policies in genomics projects</h4>
<p class="authors"><span class="author">Joly,Y.;</span>, <span class="author">Dove,E.S.;</span>, <span class="author">KENNEDY,K.L.;</span>, <span class="author">Bobrow,M.;</span>, <span class="author">Ouellette,B.F.F.;</span>, <span class="author">DYKE,S.O.M.;</span>, <span class="author">Kato,K.;</span> and <span class="author">Knoppers,B.M.</span></p>
<p class="publication"><span class="title">Medical Law International</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">92-120</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1177%2F0968533212458431">10.1177/0968533212458431</a>; URL: <a title="http://mli.sagepub.com/content/12/2/92" href="http://mli.sagepub.com/content/12/2/92">http://mli.sagepub.com/content/12/2/92</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A sequence variant associated with sortilin-1 (SORT1) on 1p13.3 is independently associated with abdominal aortic aneurysm.</h4>
<p class="authors"><span class="author">Jones GT</span>, <span class="author">Bown MJ</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Romaine SP</span>, <span class="author">Helgadottir A</span>, <span class="author">Yu G</span>, <span class="author">Tromp G</span>, <span class="author">Norman PE</span>, <span class="author">Jin C</span>, <span class="author">Baas AF</span>, <span class="author">Blankensteijn JD</span>, <span class="author">Kullo IJ</span>, <span class="author">Phillips LV</span>, <span class="author">Williams MJ</span>, <span class="author">Topless R</span>, <span class="author">Merriman TR</span>, <span class="author">Vasudevan TM</span>, <span class="author">Lewis DR</span>, <span class="author">Blair RD</span>, <span class="author">Hill AA</span>, <span class="author">Sayers RD</span>, <span class="author">Powell JT</span>, <span class="author">Deloukas P</span>, <span class="author">Thorleifsson G</span>, <span class="author">Matthiasson SE</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Golledge J</span>, <span class="author">Ari&euml;ns RA</span>, <span class="author">Johnson A</span>, <span class="author">Sohrabi S</span>, <span class="author">Scott DJ</span>, <span class="author">Carey DJ</span>, <span class="author">Erdman R</span>, <span class="author">Elmore JR</span>, <span class="author">Kuivaniemi H</span>, <span class="author">Samani NJ</span>, <span class="author">Stefansson K</span> and <span class="author">van Rij AM</span></p><p class="abstract">Abdominal aortic aneurysm (AAA) is a common human disease with a high estimated heritability (0.7); however, only a small number of associated genetic loci have been reported to date. In contrast, over 100 loci have now been reproducibly associated with either blood lipid profile and/or coronary artery disease (CAD) (both risk factors for AAA) in large-scale meta-analyses. This study employed a staged design to investigate whether the loci for these two phenotypes are also associated with AAA. Validated CAD and dyslipidaemia loci underwent screening using the Otago AAA genome-wide association data set. Putative associations underwent staged secondary validation in 10 additional cohorts. A novel association between the SORT1 (1p13.3) locus and AAA was identified. The rs599839 G allele, which has been previously associated with both dyslipidaemia and CAD, reached genome-wide significance in 11 combined independent cohorts (meta-analysis with 7048 AAA cases and 75 976 controls: G allele OR 0.81, 95% CI 0.76-0.85, P = 7.2 &times; 10(-14)). Modelling for confounding interactions of concurrent dyslipidaemia, heart disease and other risk factors suggested that this marker is an independent predictor of AAA susceptibility. In conclusion, a genetic marker associated with cardiovascular risk factors, and in particular concurrent vascular disease, appeared to independently contribute to susceptibility for AAA. Given the potential genetic overlap between risk factor and disease phenotypes, the use of well-characterized case-control cohorts allowing for modelling of cardiovascular disease risk confounders will be an important component in the future discovery of genetic markers for conditions such as AAA.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01 HL064310</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">14</span>;<span class="pagerange">2941-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23535823">23535823</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3690970">3690970</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddt141">10.1093/hmg/ddt141</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using genetic prediction from known complex disease Loci to guide the design of next-generation sequencing experiments.</h4>
<p class="authors"><span class="author">Jostins L</span>, <span class="author">Levine AP</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Medical Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom ; Wellcome Trust Centre for Human Genetics, Univeristy of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">A central focus of complex disease genetics after genome-wide association studies (GWAS) is to identify low frequency and rare risk variants, which may account for an important fraction of disease heritability unexplained by GWAS. A profusion of studies using next-generation sequencing are seeking such risk alleles. We describe how already-known complex trait loci (largely from GWAS) can be used to guide the design of these new studies by selecting cases, controls, or families who are most likely to harbor undiscovered risk alleles. We show that genetic risk prediction can select unrelated cases from large cohorts who are enriched for unknown risk factors, or multiply-affected families that are more likely to harbor high-penetrance risk alleles. We derive the frequency of an undiscovered risk allele in selected cases and controls, and show how this relates to the variance explained by the risk score, the disease prevalence and the population frequency of the risk allele. We also describe a new method for informing the design of sequencing studies using genetic risk prediction in large partially-genotyped families using an extension of the Inside-Outside algorithm for inference on trees. We explore several study design scenarios using both simulated and real data, and show that in many cases genetic risk prediction can provide significant increases in power to detect low-frequency and rare risk alleles. The same approach can also be used to aid discovery of non-genetic risk factors, suggesting possible future utility of genetic risk prediction in conventional epidemiology. Software implementing the methods in this paper is available in the R package Mangrove.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e76328</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24204614">24204614</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3799779">3799779</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0076328">10.1371/journal.pone.0076328</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Near in place linear time minimum redundancy coding</h4>
<p class="authors"><span class="author">K&auml;rkk&auml;inen,J.;</span> and <span class="author">TISCHLER,G.;</span></p>
<p class="publication"><span class="title">Data Compression Conference Proceedings</span>&nbsp;<span class="year">2013</span>;<span class="pagerange">411-20</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1109%2FDCC.2013.49">10.1109/DCC.2013.49</a>; URL: <a title="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6543077" href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6543077">http://ieeexplore.ieee.org/xpl/ar<span class="print-hide">...</span><span class="print-show">ticleDetails.jsp?arnumber=</span>6543077</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction and accessibility of a cross-species phenotype ontology along with gene annotations for biomedical research.</h4>
<p class="authors"><span class="author">K&ouml;hler S</span>, <span class="author">Doelken SC</span>, <span class="author">Ruef BJ</span>, <span class="author">Bauer S</span>, <span class="author">Washington N</span>, <span class="author">Westerfield M</span>, <span class="author">Gkoutos G</span>, <span class="author">Schofield P</span>, <span class="author">Smedley D</span>, <span class="author">Lewis SE</span>, <span class="author">Robinson PN</span> and <span class="author">Mungall CJ</span></p>
<p class="affiliation">Institute for Medical and Human Genetics, Chairit&eacute;-Universitatsmedizin Berlin, Berlin, 13353, Germany ; Berlin-Brandenberg Center for Regenerative Therapies (BCRT), Charit&eacute;-Universitatsmedizin Berlin, Berlin, 13352, Germany.</p><p class="abstract">Phenotype analyses, e.g. investigating metabolic processes, tissue formation, or organism behavior, are an important element of most biological and medical research activities. Biomedical researchers are making increased use of ontological standards and methods to capture the results of such analyses, with one focus being the comparison and analysis of phenotype information between species. We have generated a cross-species phenotype ontology for human, mouse and zebra fish that contains zebrafish phenotypes. We also provide up-to-date annotation data connecting human genes to phenotype classes from the generated ontology. We have included the data generation pipeline into our continuous integration system ensuring stable and up-to-date releases. This article describes the data generation process and is intended to help interested researchers access both the phenotype annotation data and the associated cross-species phenotype ontology. The resource described here can be used in sophisticated semantic similarity and gene set enrichment analyses for phenotype data across species. The stable releases of this resource can be obtained from http://purl.obolibrary.org/obo/hp/uberpheno/.</p>
<p class="publication"><span class="title">F1000Research</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24358873">24358873</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3799545">3799545</a>; DOI: <a href="http://dx.doi.org/10.12688%2Ff1000research.2-30.v1">10.12688/f1000research.2-30.v1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing for rapid susceptibility testing of M. tuberculosis.</h4>
<p class="authors"><span class="author">K&ouml;ser CU</span>, <span class="author">Bryant JM</span>, <span class="author">Becq J</span>, <span class="author">T&ouml;r&ouml;k ME</span>, <span class="author">Ellington MJ</span>, <span class="author">Marti-Renom MA</span>, <span class="author">Carmichael AJ</span>, <span class="author">Parkhill J</span>, <span class="author">Smith GP</span> and <span class="author">Peacock SJ</span></p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Chief Scientist Office: <span class="grant">G1000803</span>; Department of Health; Medical Research Council: <span class="grant">G1000803</span>; Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">369</span>;<span class="number">3</span>;<span class="pagerange">290-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23863072">23863072</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836233">3836233</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMc1215305">10.1056/NEJMc1215305</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Consequences of whiB7 (Rv3197A) Mutations in Beijing Genotype Isolates of the Mycobacterium tuberculosis Complex.</h4>
<p class="authors"><span class="author">K&ouml;ser CU</span>, <span class="author">Bryant JM</span>, <span class="author">Parkhill J</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Public Health England, Cambridge, United Kingdom.</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2013</span>;<span class="volume">57</span>;<span class="number">7</span>;<span class="pagerange">3461</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23761426">23761426</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.00626-13">10.1128/AAC.00626-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analyses identify 18 new loci associated with serum urate concentrations.</h4>
<p class="authors"><span class="author">K&ouml;ttgen A</span>, <span class="author">Albrecht E</span>, <span class="author">Teumer A</span>, <span class="author">Vitart V</span>, <span class="author">Krumsiek J</span>, <span class="author">Hundertmark C</span>, <span class="author">Pistis G</span>, <span class="author">Ruggiero D</span>, <span class="author">O&#39;Seaghdha CM</span>, <span class="author">Haller T</span>, <span class="author">Yang Q</span>, <span class="author">Tanaka T</span>, <span class="author">Johnson AD</span>, <span class="author">Kutalik Z</span>, <span class="author">Smith AV</span>, <span class="author">Shi J</span>, <span class="author">Struchalin M</span>, <span class="author">Middelberg RP</span>, <span class="author">Brown MJ</span>, <span class="author">Gaffo AL</span>, <span class="author">Pirastu N</span>, <span class="author">Li G</span>, <span class="author">Hayward C</span>, <span class="author">Zemunik T</span>, <span class="author">Huffman J</span>, <span class="author">Yengo L</span>, <span class="author">Zhao JH</span>, <span class="author">Demirkan A</span>, <span class="author">Feitosa MF</span>, <span class="author">Liu X</span>, <span class="author">Malerba G</span>, <span class="author">Lopez LM</span>, <span class="author">van der Harst P</span>, <span class="author">Li X</span>, <span class="author">Kleber ME</span>, <span class="author">Hicks AA</span>, <span class="author">Nolte IM</span>, <span class="author">Johansson A</span>, <span class="author">Murgia F</span>, <span class="author">Wild SH</span>, <span class="author">Bakker SJ</span>, <span class="author">Peden JF</span>, <span class="author">Dehghan A</span>, <span class="author">Steri M</span>, <span class="author">Tenesa A</span>, <span class="author">Lagou V</span>, <span class="author">Salo P</span>, <span class="author">Mangino M</span>, <span class="author">Rose LM</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Woodward OM</span>, <span class="author">Okada Y</span>, <span class="author">Tin A</span>, <span class="author">M&uuml;ller C</span>, <span class="author">Oldmeadow C</span>, <span class="author">Putku M</span>, <span class="author">Czamara D</span>, <span class="author">Kraft P</span>, <span class="author">Frogheri L</span>, <span class="author">Thun GA</span>, <span class="author">Grotevendt A</span>, <span class="author">Gislason GK</span>, <span class="author">Harris TB</span>, <span class="author">Launer LJ</span>, <span class="author">McArdle P</span>, <span class="author">Shuldiner AR</span>, <span class="author">Boerwinkle E</span>, <span class="author">Coresh J</span>, <span class="author">Schmidt H</span>, <span class="author">Schallert M</span>, <span class="author">Martin NG</span>, <span class="author">Montgomery GW</span>, <span class="author">Kubo M</span>, <span class="author">Nakamura Y</span>, <span class="author">Tanaka T</span>, <span class="author">Munroe PB</span>, <span class="author">Samani NJ</span>, <span class="author">Jacobs DR</span>, <span class="author">Liu K</span>, <span class="author">D&#39;Adamo P</span>, <span class="author">Ulivi S</span>, <span class="author">Rotter JI</span>, <span class="author">Psaty BM</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Campbell S</span>, <span class="author">Devuyst O</span>, <span class="author">Navarro P</span>, <span class="author">Kolcic I</span>, <span class="author">Hastie N</span>, <span class="author">Balkau B</span>, <span class="author">Froguel P</span>, <span class="author">Esko T</span>, <span class="author">Salumets A</span>, <span class="author">Khaw KT</span>, <span class="author">Langenberg C</span>, <span class="author">Wareham NJ</span>, <span class="author">Isaacs A</span>, <span class="author">Kraja A</span>, <span class="author">Zhang Q</span>, <span class="author">Wild PS</span>, <span class="author">Scott RJ</span>, <span class="author">Holliday EG</span>, <span class="author">Org E</span>, <span class="author">Viigimaa M</span>, <span class="author">Bandinelli S</span>, <span class="author">Metter JE</span>, <span class="author">Lupo A</span>, <span class="author">Trabetti E</span>, <span class="author">Sorice R</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Lattka E</span>, <span class="author">Strauch K</span>, <span class="author">Theis F</span>, <span class="author">Waldenberger M</span>, <span class="author">Wichmann HE</span>, <span class="author">Davies G</span>, <span class="author">Gow AJ</span>, <span class="author">Bruinenberg M</span>, <span class="author">LifeLines Cohort Study</span>, <span class="author">Stolk RP</span>, <span class="author">Kooner JS</span>, <span class="author">Zhang W</span>, <span class="author">Winkelmann BR</span>, <span class="author">Boehm BO</span>, <span class="author">Lucae S</span>, <span class="author">Penninx BW</span>, <span class="author">Smit JH</span>, <span class="author">Curhan G</span>, <span class="author">Mudgal P</span>, <span class="author">Plenge RM</span>, <span class="author">Portas L</span>, <span class="author">Persico I</span>, <span class="author">Kirin M</span>, <span class="author">Wilson JF</span>, <span class="author">Mateo Leach I</span>, <span class="author">van Gilst WH</span>, <span class="author">Goel A</span>, <span class="author">Ongen H</span>, <span class="author">Hofman A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Imboden M</span>, <span class="author">von Eckardstein A</span>, <span class="author">Cucca F</span>, <span class="author">Nagaraja R</span>, <span class="author">Piras MG</span>, <span class="author">Nauck M</span>, <span class="author">Schurmann C</span>, <span class="author">Budde K</span>, <span class="author">Ernst F</span>, <span class="author">Farrington SM</span>, <span class="author">Theodoratou E</span>, <span class="author">Prokopenko I</span>, <span class="author">Stumvoll M</span>, <span class="author">Jula A</span>, <span class="author">Perola M</span>, <span class="author">Salomaa V</span>, <span class="author">Shin SY</span>, <span class="author">Spector TD</span>, <span class="author">Sala C</span>, <span class="author">Ridker PM</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Viikari J</span>, <span class="author">Hengstenberg C</span>, <span class="author">Nelson CP</span>, <span class="author">CARDIoGRAM Consortium</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">ICBP Consortium</span>, <span class="author">MAGIC Consortium</span>, <span class="author">Meschia JF</span>, <span class="author">Nalls MA</span>, <span class="author">Sharma P</span>, <span class="author">Singleton AB</span>, <span class="author">Kamatani N</span>, <span class="author">Zeller T</span>, <span class="author">Burnier M</span>, <span class="author">Attia J</span>, <span class="author">Laan M</span>, <span class="author">Klopp N</span>, <span class="author">Hillege HL</span>, <span class="author">Kloiber S</span>, <span class="author">Choi H</span>, <span class="author">Pirastu M</span>, <span class="author">Tore S</span>, <span class="author">Probst-Hensch NM</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Gudnason V</span>, <span class="author">Parsa A</span>, <span class="author">Schmidt R</span>, <span class="author">Whitfield JB</span>, <span class="author">Fornage M</span>, <span class="author">Gasparini P</span>, <span class="author">Siscovick DS</span>, <span class="author">Pola&scaron;ek O</span>, <span class="author">Campbell H</span>, <span class="author">Rudan I</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Metspalu A</span>, <span class="author">Loos RJ</span>, <span class="author">van Duijn CM</span>, <span class="author">Borecki IB</span>, <span class="author">Ferrucci L</span>, <span class="author">Gambaro G</span>, <span class="author">Deary IJ</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Chambers JC</span>, <span class="author">M&auml;rz W</span>, <span class="author">Pramstaller PP</span>, <span class="author">Snieder H</span>, <span class="author">Gyllensten U</span>, <span class="author">Wright AF</span>, <span class="author">Navis G</span>, <span class="author">Watkins H</span>, <span class="author">Witteman JC</span>, <span class="author">Sanna S</span>, <span class="author">Schipf S</span>, <span class="author">Dunlop MG</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Ripatti S</span>, <span class="author">Soranzo N</span>, <span class="author">Toniolo D</span>, <span class="author">Chasman DI</span>, <span class="author">Raitakari O</span>, <span class="author">Kao WH</span>, <span class="author">Ciullo M</span>, <span class="author">Fox CS</span>, <span class="author">Caulfield M</span>, <span class="author">Bochud M</span> and <span class="author">Gieger C</span></p>
<p class="affiliation">Renal Division, Freiburg University Hospital, Freiburg, Germany. <a href="mailto:%61%6E%6E%61.%6B%6F%65%74%74%67%65%6E%40%75%6E%69%6B%6C%69%6E%69%6B-%66%72%65%69%62%75%72%67.%64%65">&#97;&#110;&#110;&#97;&#46;&#107;&#111;&#101;&#116;&#116;&#103;&#101;&#110;&#64;&#117;&#110;&#105;&#107;&#108;&#105;&#110;&#105;&#107;&#45;&#102;&#114;&#101;&#105;&#98;&#117;&#114;&#103;&#46;&#100;&#101;</a></p><p class="abstract">Elevated serum urate concentrations can cause gout, a prevalent and painful inflammatory arthritis. By combining data from &gt;140,000 individuals of European ancestry within the Global Urate Genetics Consortium (GUGC), we identified and replicated 28 genome-wide significant loci in association with serum urate concentrations (18 new regions in or near TRIM46, INHBB, SFMBT1, TMEM171, VEGFA, BAZ1B, PRKAG2, STC1, HNF4G, A1CF, ATXN2, UBE2Q2, IGF1R, NFAT5, MAF, HLF, ACVR1B-ACVRL1 and B3GNT4). Associations for many of the loci were of similar magnitude in individuals of non-European ancestry. We further characterized these loci for associations with gout, transcript expression and the fractional excretion of urate. Network analyses implicate the inhibins-activins signaling pathways and glucose metabolism in systemic urate control. New candidate genes for serum urate concentration highlight the importance of metabolic control of urate production and excretion, which may have implications for the treatment and prevention of gout.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">12076</span>, <span class="grant">14136</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0600237</span>, <span class="grant">G0700704</span>, <span class="grant">G1000143</span>, <span class="grant">G9521010</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127527198</span>, <span class="grant">MC_U127592696</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NCI NIH HHS: <span class="grant">P01 CA087969</span>, <span class="grant">R01 CA047988</span>; NCRR NIH HHS: <span class="grant">K12 RR023250</span>, <span class="grant">M01 RR016500</span>, <span class="grant">UL1 RR025005</span>; NHGRI NIH HHS: <span class="grant">U01 HG004402</span>, <span class="grant">U01 HG004424</span>, <span class="grant">U01 HG004446</span>, <span class="grant">U01 HG004729</span>; NHLBI NIH HHS: <span class="grant">HHSN268201100005C</span>, <span class="grant">HHSN268201100006C</span>, <span class="grant">HHSN268201100007C</span>, <span class="grant">HHSN268201100008C</span>, <span class="grant">HHSN268201100009C</span>, <span class="grant">HHSN268201100010C</span>, <span class="grant">HHSN268201100011C</span>, <span class="grant">HHSN268201100012C</span>, <span class="grant">HHSN268201200036C</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC045134</span>, <span class="grant">N01 HC095170</span>, <span class="grant">N01 HC095171</span>, <span class="grant">N01 HC095172</span>, <span class="grant">N01HC05187</span>, <span class="grant">N01HC45204</span>, <span class="grant">N01HC45205</span>, <span class="grant">N01HC48047</span>, <span class="grant">N01HC48048</span>, <span class="grant">N01HC48049</span>, <span class="grant">N01HC48050</span>, <span class="grant">N01HC55222</span>, <span class="grant">N01HC75150</span>, <span class="grant">N01HC85079</span>, <span class="grant">N01HC85086</span>, <span class="grant">N01HC95095</span>, <span class="grant">N02HL64278</span>, <span class="grant">R01 HL043851</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL080295</span>, <span class="grant">R01 HL084099</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL105756</span>, <span class="grant">T32 HL007024</span>, <span class="grant">U01 HL069757</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL084756</span>; NIA NIH HHS: <span class="grant">N01AG12100</span>, <span class="grant">N01AG12109</span>, <span class="grant">R01 AG015928</span>, <span class="grant">R01 AG018728</span>, <span class="grant">R01 AG020098</span>, <span class="grant">R01 AG023629</span>, <span class="grant">R01 AG027058</span>, <span class="grant">Z01 AG000954-06</span>; NIAAA NIH HHS: <span class="grant">K05 AA017688</span>, <span class="grant">P50 AA011998</span>, <span class="grant">R01 AA007535</span>, <span class="grant">R01 AA013320</span>, <span class="grant">R01 AA013321</span>, <span class="grant">R01 AA013326</span>, <span class="grant">R01 AA014041</span>; NIAMS NIH HHS: <span class="grant">P60 AR047785</span>, <span class="grant">R01 AR056291</span>, <span class="grant">R21 AR056042</span>; NIDA NIH HHS: <span class="grant">R01 DA012854</span>; NIDDK NIH HHS: <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P60 DK079637</span>; NIGMS NIH HHS: <span class="grant">U01 GM074518</span>; NIMH NIH HHS: <span class="grant">R01 MH066206</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">2</span>;<span class="pagerange">145-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23263486">23263486</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3663712">3663712</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2500">10.1038/ng.2500</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">KAT5 tyrosine phosphorylation couples chromatin sensing to ATM signalling.</h4>
<p class="authors"><span class="author">Kaidi A</span> and <span class="author">Jackson SP</span></p>
<p class="affiliation">The Gurdon Institute and Department of Biochemistry, University of Cambridge, Tennis Court Road, Cambridge CB2 1QN, UK.</p><p class="abstract">The detection of DNA lesions within chromatin represents a critical step in cellular responses to DNA damage. However, the regulatory mechanisms that couple chromatin sensing to DNA-damage signalling in mammalian cells are not well understood. Here we show that tyrosine phosphorylation of the protein acetyltransferase KAT5 (also known as TIP60) increases after DNA damage in a manner that promotes KAT5 binding to the histone mark H3K9me3. This triggers KAT5-mediated acetylation of the ATM kinase, promoting DNA-damage-checkpoint activation and cell survival. We also establish that chromatin alterations can themselves enhance KAT5 tyrosine phosphorylation and ATM-dependent signalling, and identify the proto-oncogene c-Abl as a mediator of this modification. These findings define KAT5 tyrosine phosphorylation as a key event in the sensing of genomic and chromatin perturbations, and highlight a key role for c-Abl in such processes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A11224</span>, <span class="grant">C6/A11224</span>; European Research Council: <span class="grant">268536</span>; Wellcome Trust: <span class="grant">092096</span>, <span class="grant">WT092096</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">498</span>;<span class="number">7452</span>;<span class="pagerange">70-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23708966">23708966</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3859897">3859897</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12201">10.1038/nature12201</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Unusual features in organisation of capsular polysaccharide-related genes of C. jejuni strain X.</h4>
<p class="authors"><span class="author">Karlyshev AV</span>, <span class="author">Quail MA</span>, <span class="author">Parkhill J</span> and <span class="author">Wren BW</span></p>
<p class="affiliation">School of Life Sciences, Kingston University, Faculty of Science, Engineering and Computing, Penrhyn Road, Kingston-upon Thames, KT1 2EE, UK. <a href="mailto:%61.%6B%61%72%6C%79%73%68%65%76%40%6B%69%6E%67%73%74%6F%6E.%61%63.%75%6B">&#97;&#46;&#107;&#97;&#114;&#108;&#121;&#115;&#104;&#101;&#118;&#64;&#107;&#105;&#110;&#103;&#115;&#116;&#111;&#110;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">PCR probing of the genome of Campylobacter jejuni strain X using conserved capsular polysaccharide (CPS)-related genes allowed elucidation of a complete sequence of the respective gene cluster (cps). This is the largest known Campylobacter cps cluster (38 kb excluding flanking kps regions), which includes a number of genes not detected in other Campylobacter strains. Sequence analysis suggests genetic rearrangements both within and outside the cps gene cluster, a mechanism which may be responsible for mosaic organisation of sugar transferase-related genes leading to structural variability of the capsular polysaccharide (CPS).</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">EGA16167</span></p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">2013</span>;<span class="volume">522</span>;<span class="number">1</span>;<span class="pagerange">37-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23562723">23562723</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gene.2013.03.087">10.1016/j.gene.2013.03.087</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human melioidosis, Malawi, 2011.</h4>
<p class="authors"><span class="author">Katangwe T</span>, <span class="author">Purcell J</span>, <span class="author">Bar-Zeev N</span>, <span class="author">Denis B</span>, <span class="author">Montgomery J</span>, <span class="author">Alaerts M</span>, <span class="author">Heyderman RS</span>, <span class="author">Dance DA</span>, <span class="author">Kennedy N</span>, <span class="author">Feasey N</span> and <span class="author">Moxon CA</span></p>
<p class="abstract">A case of human melioidosis caused by a novel sequence type of Burkholderia pseudomallei occurred in a child in Malawi, southern Africa. A literature review showed that human cases reported from the continent have been increasing.</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">19</span>;<span class="number">6</span>;<span class="pagerange">981-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23735189">23735189</a>; DOI: <a href="http://dx.doi.org/10.3201%2Feid1906.120717">10.3201/eid1906.120717</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Activation of the B cell antigen receptor triggers reactivation of latent Kaposi&#39;s sarcoma-associated herpesvirus in B cells.</h4>
<p class="authors"><span class="author">Kati S</span>, <span class="author">Tsao EH</span>, <span class="author">G&uuml;nther T</span>, <span class="author">Weidner-Glunde M</span>, <span class="author">Roth&auml;mel T</span>, <span class="author">Grundhoff A</span>, <span class="author">Kellam P</span> and <span class="author">Schulz TF</span></p>
<p class="affiliation">Institute of Virology, Hanover Medical School, Hanover, Germany.</p><p class="abstract">Kaposi&#39;s sarcoma-associated herpesvirus (KSHV) is an oncogenic herpesvirus and the cause of Kaposi&#39;s sarcoma, primary effusion lymphoma (PEL) and multicentric Castleman&#39;s disease. Latently infected B cells are the main reservoir of this virus in vivo, but the nature of the stimuli that lead to its reactivation in B cells is only partially understood. We established stable BJAB cell lines harboring latent KSHV by cell-free infection with recombinant virus carrying a puromycin resistance marker. Our latently infected B cell lines, termed BrK.219, can be reactivated by triggering the B cell receptor (BCR) with antibodies to surface IgM, a stimulus imitating antigen recognition. Using this B cell model system we studied the mechanisms that mediate the reactivation of KSHV in B cells following the stimulation of the BCR and could identify phosphatidylinositol 3-kinase (PI3K) and X-box binding protein 1 (XBP-1) as proteins that play an important role in the BCR-mediated reactivation of latent KSHV.</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2013</span>;<span class="volume">87</span>;<span class="number">14</span>;<span class="pagerange">8004-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23678173">23678173</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3700181">3700181</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.00506-13">10.1128/JVI.00506-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RetroSeq: transposable element discovery from next-generation sequencing data.</h4>
<p class="authors"><span class="author">Keane TM</span>, <span class="author">Wong K</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%74%6B%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#107;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Unlabelled</strong>: A significant proportion of eukaryote genomes consist of transposable element (TE)-derived sequence. These elements are known to have the capacity to modulate gene function and genome evolution. We have developed RetroSeq for detecting non-reference TE insertions from Illumina paired-end whole-genome sequencing data. We evaluate RetroSeq on a human trio from the 1000 Genomes Project, showing that it produces highly accurate TE calls.</p><p class="abstract"><strong>Availabilty</strong>: RetroSeq is open-source and available from https://github.com/tk2/RetroSeq.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0800024</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2013</span>;<span class="volume">29</span>;<span class="number">3</span>;<span class="pagerange">389-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23233656">23233656</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3562067">3562067</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts697">10.1093/bioinformatics/bts697</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Different patterns of Epstein-Barr virus latency in endemic Burkitt lymphoma (BL) lead to distinct variants within the BL-associated gene expression signature.</h4>
<p class="authors"><span class="author">Kelly GL</span>, <span class="author">Stylianou J</span>, <span class="author">Rasaiyaah J</span>, <span class="author">Wei W</span>, <span class="author">Thomas W</span>, <span class="author">Croom-Carter D</span>, <span class="author">Kohler C</span>, <span class="author">Spang R</span>, <span class="author">Woodman C</span>, <span class="author">Kellam P</span>, <span class="author">Rickinson AB</span> and <span class="author">Bell AI</span></p>
<p class="affiliation">School of Cancer Sciences, College of Medical and Dental Sciences, University of Birmingham, Edgbaston, Birmingham, United Kingdom.</p><p class="abstract">Epstein-Barr virus (EBV) is present in all cases of endemic Burkitt lymphoma (BL) but in few European/North American sporadic BLs. Gene expression arrays of sporadic tumors have defined a consensus BL profile within which tumors are classifiable as &quot;molecular BL&quot; (mBL). Where endemic BLs fall relative to this profile remains unclear, since they not only carry EBV but also display one of two different forms of virus latency. Here, we use early-passage BL cell lines from different tumors, and BL subclones from a single tumor, to compare EBV-negative cells with EBV-positive cells displaying either classical latency I EBV infection (where EBNA1 is the only EBV antigen expressed from the wild-type EBV genome) or Wp-restricted latency (where an EBNA2 gene-deleted virus genome broadens antigen expression to include the EBNA3A, -3B, and -3C proteins and BHRF1). Expression arrays show that both types of endemic BL fall within the mBL classification. However, while EBV-negative and latency I BLs show overlapping profiles, Wp-restricted BLs form a distinct subgroup, characterized by a detectable downregulation of the germinal center (GC)-associated marker Bcl6 and upregulation of genes marking early plasmacytoid differentiation, notably IRF4 and BLIMP1. Importantly, these same changes can be induced in EBV-negative or latency I BL cells by infection with an EBNA2-knockout virus. Thus, we infer that the distinct gene profile of Wp-restricted BLs does not reflect differences in the identity of the tumor progenitor cell per se but differences imposed on a common progenitor by broadened EBV gene expression.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">15032</span>, <span class="grant">C910/A8829</span></p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2013</span>;<span class="volume">87</span>;<span class="number">5</span>;<span class="pagerange">2882-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23269792">23269792</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3571367">3571367</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.03003-12">10.1128/JVI.03003-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A systematic genome-wide analysis of zebrafish protein-coding gene function.</h4>
<p class="authors"><span class="author">Kettleborough RN</span>, <span class="author">Busch-Nentwich EM</span>, <span class="author">Harvey SA</span>, <span class="author">Dooley CM</span>, <span class="author">de Bruijn E</span>, <span class="author">van Eeden F</span>, <span class="author">Sealy I</span>, <span class="author">White RJ</span>, <span class="author">Herd C</span>, <span class="author">Nijman IJ</span>, <span class="author">F&eacute;nyes F</span>, <span class="author">Mehroke S</span>, <span class="author">Scahill C</span>, <span class="author">Gibbons R</span>, <span class="author">Wali N</span>, <span class="author">Carruthers S</span>, <span class="author">Hall A</span>, <span class="author">Yen J</span>, <span class="author">Cuppen E</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Since the publication of the human reference genome, the identities of specific genes associated with human diseases are being discovered at a rapid rate. A central problem is that the biological activity of these genes is often unclear. Detailed investigations in model vertebrate organisms, typically mice, have been essential for understanding the activities of many orthologues of these disease-associated genes. Although gene-targeting approaches and phenotype analysis have led to a detailed understanding of nearly 6,000 protein-coding genes, this number falls considerably short of the more than 22,000 mouse protein-coding genes. Similarly, in zebrafish genetics, one-by-one gene studies using positional cloning, insertional mutagenesis, antisense morpholino oligonucleotides, targeted re-sequencing, and zinc finger and TAL endonucleases have made substantial contributions to our understanding of the biological activity of vertebrate genes, but again the number of genes studied falls well short of the more than 26,000 zebrafish protein-coding genes. Importantly, for both mice and zebrafish, none of these strategies are particularly suited to the rapid generation of knockouts in thousands of genes and the assessment of their biological activity. Here we describe an active project that aims to identify and phenotype the disruptive mutations in every zebrafish protein-coding gene, using a well-annotated zebrafish reference genome sequence, high-throughput sequencing and efficient chemical mutagenesis. So far we have identified potentially disruptive mutations in more than 38% of all known zebrafish protein-coding genes. We have developed a multi-allelic phenotyping scheme to efficiently assess the effects of each allele during embryogenesis and have analysed the phenotypic consequences of over 1,000 alleles. All mutant alleles and data are available to the community and our phenotyping scheme is adaptable to phenotypic analysis beyond embryogenesis.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0777791</span>; NHGRI NIH HHS: <span class="grant">5R01HG00481</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">496</span>;<span class="number">7446</span>;<span class="pagerange">494-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23594742">23594742</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3743023">3743023</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11992">10.1038/nature11992</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide analysis of populations from European Russia reveals a new pole of genetic diversity in northern europe.</h4>
<p class="authors"><span class="author">Khrunin AV</span>, <span class="author">Khokhrin DV</span>, <span class="author">Filippova IN</span>, <span class="author">Esko T</span>, <span class="author">Nelis M</span>, <span class="author">Bebyakova NA</span>, <span class="author">Bolotova NL</span>, <span class="author">Klovins J</span>, <span class="author">Nikitina-Zake L</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Ripatti S</span>, <span class="author">Schreiber S</span>, <span class="author">Franke A</span>, <span class="author">Macek M</span>, <span class="author">Kruli&scaron;ov&aacute; V</span>, <span class="author">Lubinski J</span>, <span class="author">Metspalu A</span> and <span class="author">Limborska SA</span></p>
<p class="affiliation">Department of Molecular Bases of Human Genetics, Institute of Molecular Genetics, Russian Academy of Sciences, Moscow, Russia.</p>
<p class="abstract">Several studies examined the fine-scale structure of human genetic variation in Europe. However, the European sets analyzed represent mainly northern, western, central, and southern Europe. Here, we report an analysis of approximately 166,000 single nucleotide polymorphisms in populations from eastern (northeastern) Europe: four Russian populations from European Russia, and three populations from the northernmost Finno-Ugric ethnicities (Veps and two contrast groups of Komi people). These were compared with several reference European samples, including Finns, Estonians, Latvians, Poles, Czechs, Germans, and Italians. The results obtained demonstrated genetic heterogeneity of populations living in the region studied. Russians from the central part of European Russia (Tver, Murom, and Kursk) exhibited similarities with populations from central-eastern Europe, and were distant from Russian sample from the northern Russia (Mezen district, Archangelsk region). Komi samples, especially Izhemski Komi, were significantly different from all other populations studied. These can be considered as a second pole of genetic diversity in northern Europe (in addition to the pole, occupied by Finns), as they had a distinct ancestry component. Russians from Mezen and the Finnic-speaking Veps were positioned between the two poles, but differed from each other in the proportions of Komi and Finnic ancestries. In general, our data provides a more complete genetic map of Europe accounting for the diversity in its most eastern (northeastern) populations.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e58552</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23505534">23505534</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3591355">3591355</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0058552">10.1371/journal.pone.0058552</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrative annotation of variants from 1092 humans: application to cancer genomics.</h4>
<p class="authors"><span class="author">Khurana E</span>, <span class="author">Fu Y</span>, <span class="author">Colonna V</span>, <span class="author">Mu XJ</span>, <span class="author">Kang HM</span>, <span class="author">Lappalainen T</span>, <span class="author">Sboner A</span>, <span class="author">Lochovsky L</span>, <span class="author">Chen J</span>, <span class="author">Harmanci A</span>, <span class="author">Das J</span>, <span class="author">Abyzov A</span>, <span class="author">Balasubramanian S</span>, <span class="author">Beal K</span>, <span class="author">Chakravarty D</span>, <span class="author">Challis D</span>, <span class="author">Chen Y</span>, <span class="author">Clarke D</span>, <span class="author">Clarke L</span>, <span class="author">Cunningham F</span>, <span class="author">Evani US</span>, <span class="author">Flicek P</span>, <span class="author">Fragoza R</span>, <span class="author">Garrison E</span>, <span class="author">Gibbs R</span>, <span class="author">G&uuml;m&uuml;s ZH</span>, <span class="author">Herrero J</span>, <span class="author">Kitabayashi N</span>, <span class="author">Kong Y</span>, <span class="author">Lage K</span>, <span class="author">Liluashvili V</span>, <span class="author">Lipkin SM</span>, <span class="author">MacArthur DG</span>, <span class="author">Marth G</span>, <span class="author">Muzny D</span>, <span class="author">Pers TH</span>, <span class="author">Ritchie GR</span>, <span class="author">Rosenfeld JA</span>, <span class="author">Sisu C</span>, <span class="author">Wei X</span>, <span class="author">Wilson M</span>, <span class="author">Xue Y</span>, <span class="author">Yu F</span>, <span class="author">1000 Genomes Project Consortium</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Yu H</span>, <span class="author">Rubin MA</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Gerstein M</span></p>
<p class="affiliation">Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT 06520, USA.</p><p class="abstract">Interpreting variants, especially noncoding ones, in the increasing number of personal genomes is challenging. We used patterns of polymorphisms in functionally annotated regions in 1092 humans to identify deleterious variants; then we experimentally validated candidates. We analyzed both coding and noncoding regions, with the former corroborating the latter. We found regions particularly sensitive to mutations (&quot;ultrasensitive&quot;) and variants that are disruptive because of mechanistic effects on transcription-factor binding (that is, &quot;motif-breakers&quot;). We also found variants in regions with higher network centrality tend to be deleterious. Insertions and deletions followed a similar pattern to single-nucleotide variants, with some notable exceptions (e.g., certain deletions and enhancers). On the basis of these patterns, we developed a computational tool (FunSeq), whose application to ~90 cancer genomes reveals nearly a hundred candidate noncoding drivers.</p>
<p class="grants">Funded by: NCATS NIH HHS: <span class="grant">UL1 TR000457</span>; NCI NIH HHS: <span class="grant">CA167824</span>, <span class="grant">R01 CA166661</span>, <span class="grant">R01CA152057</span>, <span class="grant">U01 CA111275</span>; NCRR NIH HHS: <span class="grant">G12 RR003050</span>; NHGRI NIH HHS: <span class="grant">HG005718</span>, <span class="grant">HG007000</span>, <span class="grant">R01 HG002898</span>, <span class="grant">R01HG4719</span>, <span class="grant">U01 HG005718</span>, <span class="grant">U01HG6513</span>, <span class="grant">U41 HG007000</span>, <span class="grant">U54 HG003079</span>; NIGMS NIH HHS: <span class="grant">GM104424</span>; NIMHD NIH HHS: <span class="grant">G12 MD007579</span>; Wellcome Trust: <span class="grant">085532</span>, <span class="grant">090532</span>, <span class="grant">095908</span>, <span class="grant">098051</span>, <span class="grant">WT085532</span>, <span class="grant">WT095908</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">342</span>;<span class="number">6154</span>;<span class="pagerange">1235587</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24092746">24092746</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3947637">3947637</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1235587">10.1126/science.1235587</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome and Transcriptome Adaptation Accompanying Emergence of the Definitive Type 2 Host-Restricted Salmonella enterica Serovar Typhimurium Pathovar.</h4>
<p class="authors"><span class="author">Kingsley RA</span>, <span class="author">Kay S</span>, <span class="author">Connor T</span>, <span class="author">Barquist L</span>, <span class="author">Sait L</span>, <span class="author">Holt KE</span>, <span class="author">Sivaraman K</span>, <span class="author">Wileman T</span>, <span class="author">Goulding D</span>, <span class="author">Clare S</span>, <span class="author">Hale C</span>, <span class="author">Seshasayee A</span>, <span class="author">Harris S</span>, <span class="author">Thomson NR</span>, <span class="author">Gardner P</span>, <span class="author">Rabsch W</span>, <span class="author">Wigley P</span>, <span class="author">Humphrey T</span>, <span class="author">Parkhill J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, the Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">ABSTRACT Salmonella enterica serovar Typhimurium definitive type 2 (DT2) is host restricted to Columba livia (rock or feral pigeon) but is also closely related to S.&nbsp;Typhimurium isolates that circulate in livestock and cause a zoonosis characterized by gastroenteritis in humans. DT2 isolates formed a distinct phylogenetic cluster within S.&nbsp;Typhimurium based on whole-genome-sequence polymorphisms. Comparative genome analysis of DT2 94-213 and S.&nbsp;Typhimurium SL1344, DT104, and D23580 identified few differences in gene content with the exception of variations within prophages. However, DT2 94-213 harbored 22 pseudogenes that were intact in other closely related S.&nbsp;Typhimurium strains. We report a novel in silico approach to identify single amino acid substitutions in proteins that have a high probability of a functional impact. One polymorphism identified using this method, a single-residue deletion in the Tar protein, abrogated chemotaxis to aspartate in vitro. DT2 94-213 also exhibited an altered transcriptional profile in response to culture at 42&deg;C compared to that of SL1344. Such differentially regulated genes included a number involved in flagellum biosynthesis and motility. IMPORTANCE Whereas Salmonella enterica serovar Typhimurium can infect a wide range of animal species, some variants within this serovar exhibit a more limited host range and altered disease potential. Phylogenetic analysis based on whole-genome sequences can identify lineages associated with specific virulence traits, including host adaptation. This study represents one of the first to link pathogen-specific genetic signatures, including coding capacity, genome degradation, and transcriptional responses to host adaptation within a Salmonella serovar. We performed comparative genome analysis of reference and pigeon-adapted definitive type 2 (DT2) S.&nbsp;Typhimurium isolates alongside phenotypic and transcriptome analyses, to identify genetic signatures linked to host adaptation within the DT2 lineage.</p>
<p class="publication"><span class="title">mBio</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="number">5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23982073">23982073</a>; DOI: <a href="http://dx.doi.org/10.1128%2FmBio.00565-13">10.1128/mBio.00565-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of Tumor Heterogeneity and Cancer Gene Networks Using Deep Sequencing of MMTV-Induced Mouse Mammary Tumors.</h4>
<p class="authors"><span class="author">Klijn C</span>, <span class="author">Koudijs MJ</span>, <span class="author">Kool J</span>, <span class="author">Ten Hoeve J</span>, <span class="author">Boer M</span>, <span class="author">de Moes J</span>, <span class="author">Akhtar W</span>, <span class="author">van Miltenburg M</span>, <span class="author">Vendel-Zwaagstra A</span>, <span class="author">Reinders MJ</span>, <span class="author">Adams DJ</span>, <span class="author">van Lohuizen M</span>, <span class="author">Hilkens J</span>, <span class="author">Wessels LF</span> and <span class="author">Jonkers J</span></p>
<p class="affiliation">Division of Molecular Pathology, The Netherlands Cancer Institute, Amsterdam, The Netherlands ; Division of Molecular Carcinogenesis, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</p>
<p class="abstract">Cancer develops through a multistep process in which normal cells progress to malignant tumors via the evolution of their genomes as a result of the acquisition of mutations in cancer driver genes. The number, identity and mode of action of cancer driver genes, and how they contribute to tumor evolution is largely unknown. This study deployed the Mouse Mammary Tumor Virus (MMTV) as an insertional mutagen to find both the driver genes and the networks in which they function. Using deep insertion site sequencing we identified around 31000 retroviral integration sites in 604 MMTV-induced mammary tumors from mice with mammary gland-specific deletion of Trp53, Pten heterozygous knockout mice, or wildtype strains. We identified 18 known common integration sites (CISs) and 12 previously unknown CISs marking new candidate cancer genes. Members of the Wnt, Fgf, Fgfr, Rspo and Pdgfr gene families were commonly mutated in a mutually exclusive fashion. The sequence data we generated yielded also information on the clonality of insertions in individual tumors, allowing us to develop a data-driven model of MMTV-induced tumor development. Insertional mutations near Wnt and Fgf genes mark the earliest "initiating" events in MMTV induced tumorigenesis, whereas Fgfr genes are targeted later during tumor progression. Our data shows that insertional mutagenesis can be used to discover the mutational networks, the timing of mutations, and the genes that initiate and drive tumor evolution.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e62113</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23690930">23690930</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0062113">10.1371/journal.pone.0062113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Current application and future perspectives of molecular typing methods to study Clostridium difficile infections.</h4>
<p class="authors"><span class="author">Knetsch CW</span>, <span class="author">Lawley TD</span>, <span class="author">Hensgens MP</span>, <span class="author">Corver J</span>, <span class="author">Wilcox MW</span> and <span class="author">Kuijper EJ</span></p>
<p class="affiliation">Section Experimental Microbiology, Department of Medical Microbiology, Center of Infectious Diseases, Leiden University Medical Center, Leiden, Netherlands.</p><p class="abstract">Molecular typing is an essential tool to monitor Clostridium difficile infections and outbreaks within healthcare facilities. Molecular typing also plays a key role in defining the regional and global changes in circulating C. difficile types. The patterns of C. difficile types circulating within Europe (and globally) remain poorly understood, although international efforts are under way to understand the spatial and temporal patterns of C. difficile types. A complete picture is essential to properly investigate type-specific risk factors for C. difficile infections (CDI) and track long-range transmission. Currently, conventional agarose gel-based polymerase chain reaction (PCR) ribotyping is the most common typing method used in Europe to type C. difficile. Although this method has proved to be useful to study epidemiology on local, national and European level, efforts are made to replace it with capillary electrophoresis PCR ribotyping to increase pattern recognition, reproducibility and interpretation. However, this method lacks sufficient discriminatory power to study outbreaks and therefore multilocus variable-number tandem repeat analysis (MLVA) has been developed to study transmission between humans, animals and food. Sequence-based methods are increasingly being used for C. difficile fingerprinting/typing because of their ability to discriminate between highly related strains, the ease of data interpretation and transferability of data. The first studies using whole-genome single nucleotide polymorphism typing of healthcare-associated C. difficile within a clinically relevant timeframe are very promising and, although limited to select facilities because of complex data interpretation and high costs, these approaches will likely become commonly used over the coming years.</p>
<p class="publication"><span class="title">Euro surveillance : bulletin Europ&eacute;en sur les maladies transmissibles = European communicable disease bulletin</span>&nbsp;<span class="year">2013</span>;<span class="volume">18</span>;<span class="number">4</span>;<span class="pagerange">20381</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23369393">23369393</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tracking chromosome evolution in southern African gerbils using flow-sorted chromosome paints.</h4>
<p class="authors"><span class="author">Knight LI</span>, <span class="author">Ng BL</span>, <span class="author">Cheng W</span>, <span class="author">Fu B</span>, <span class="author">Yang F</span> and <span class="author">Rambau RV</span></p>
<p class="affiliation">Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Stellenbosch, South Africa.</p><p class="abstract">Desmodillus and Gerbilliscus (formerly Tatera) comprise a monophyletic group of gerbils (subfamily Gerbillinae) which last shared an ancestor approximately 8 million years ago; diploid chromosome number variation among the species ranges from 2n = 36 to 2n = 50. In an attempt to shed more light on chromosome evolution and speciation in these rodents, we compared the karyotypes of 7 species, representing 3 genera, based on homology data revealed by chromosome painting with probes derived from flow-sorted chromosomes of the hairy footed gerbil, Gerbillurus paeba (2n = 36). The fluorescent in situ hybridization data revealed remarkable genome conservation: these species share a high proportion of conserved chromosomes, and differences are due to 10 Robertsonian (Rb) rearrangements (3 autapomorphies, 3 synapomorphies and 4 hemiplasies/homoplasies). Our data suggest that chromosome evolution in Desmodillus occurred at a rate of ~1.25 rearrangements per million years (Myr), and that the rate among Gerbilliscus over a time period spanning 8 Myr is also ~1.25 rearrangements/Myr. The recently diverged Gerbillurus (G. tytonis and G. paeba) share an identical karyotype, while Gerbilliscus kempi, G. afra and G. leucogaster differ by 6 Rb rearrangements (a rate of ~1 rearrangement/Myr). Thus, our data suggests a very slow rate of chromosomal evolution in Southern African gerbils.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">139</span>;<span class="number">4</span>;<span class="pagerange">267-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23652816">23652816</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3721133">3721133</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000350696">10.1159/000350696</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A P3G generic access agreement for population genomic studies.</h4>
<p class="authors"><span class="author">Knoppers BM</span>, <span class="author">Chisholm RL</span>, <span class="author">Kaye J</span>, <span class="author">Cox D</span>, <span class="author">Thorogood A</span>, <span class="author">Burton P</span>, <span class="author">Brookes AJ</span>, <span class="author">Fortier I</span>, <span class="author">Goodwin P</span>, <span class="author">Harris JR</span>, <span class="author">Hveem K</span>, <span class="author">Kent A</span>, <span class="author">Little J</span>, <span class="author">Riegman PH</span>, <span class="author">Ripatti S</span> and <span class="author">Stolk RP</span></p>
<p class="affiliation">Centre of Genomics and Policy, McGill University, Montreal, Quebec, Canada.</p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2013</span>;<span class="volume">31</span>;<span class="number">5</span>;<span class="pagerange">384-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23657386">23657386</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.2567">10.1038/nbt.2567</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Host responses to melioidosis and tuberculosis are both dominated by interferon-mediated signaling.</h4>
<p class="authors"><span class="author">Koh GC</span>, <span class="author">Schreiber MF</span>, <span class="author">Bautista R</span>, <span class="author">Maude RR</span>, <span class="author">Dunachie S</span>, <span class="author">Limmathurotsakul D</span>, <span class="author">Day NP</span>, <span class="author">Dougan G</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom ; Department of Medicine, University of Cambridge, Cambridge, United Kingdom ; Mahidol-Oxford Tropical Medicine Research Unit, Mahidol University, Bangkok, Thailand ; Department of Infection and Tropical Diseases, Birmingham Heartlands Hospital, Birmingham, United Kingdom.</p>
<p class="abstract">Melioidosis (Burkholderia pseudomallei infection) is a common cause of community-acquired sepsis in Northeast Thailand and northern Australia. B. pseudomallei is a soil saprophyte endemic to Southeast Asia and northern Australia. The clinical presentation of melioidosis may mimic tuberculosis (both cause chronic suppurative lesions unresponsive to conventional antibiotics and both commonly affect the lungs). The two diseases have overlapping risk profiles (e.g., diabetes, corticosteroid use), and both B. pseudomallei and Mycobacterium tuberculosis are intracellular pathogens. There are however important differences: the majority of melioidosis cases are acute, not chronic, and present with severe sepsis and a mortality rate that approaches 50% despite appropriate antimicrobial therapy. By contrast, tuberculosis is characteristically a chronic illness with mortality <2% with appropriate antimicrobial chemotherapy. We examined the gene expression profiles of total peripheral leukocytes in two cohorts of patients, one with acute melioidosis (30 patients and 30 controls) and another with tuberculosis (20 patients and 24 controls). Interferon-mediated responses dominate the host response to both infections, and both type 1 and type 2 interferon responses are important. An 86-gene signature previously thought to be specific for tuberculosis is also found in melioidosis. We conclude that the host responses to melioidosis and to tuberculosis are similar: both are dominated by interferon-signalling pathways and this similarity means gene expression signatures from whole blood do not distinguish between these two diseases.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">e54961</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23383015">23383015</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0054961">10.1371/journal.pone.0054961</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromatin Accessibility Data Sets Show Bias Due to Sequence Specificity of the DNase I Enzyme.</h4>
<p class="authors"><span class="author">Koohy H</span>, <span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">Background: DNase I is an enzyme which cuts duplex DNA at a rate that depends strongly upon its chromatin environment. In combination with high-throughput sequencing (HTS) technology, it can be used to infer genome-wide landscapes of open chromatin regions. Using this technology, systematic identification of hundreds of thousands of DNase I hypersensitive sites (DHS) per cell type has been possible, and this in turn has helped to precisely delineate genomic regulatory compartments. However, to date there has been relatively little investigation into possible biases affecting this data.

Results: We report a significant degree of sequence preference spanning sites cut by DNase I in a number of published data sets. The two major protocols in current use each show a different pattern, but for a given protocol the pattern of sequence specificity seems to be quite consistent. The patterns are substantially different from biases seen in other types of HTS data sets, and in some cases the most constrained position lies outside the sequenced fragment, implying that this constraint must relate to the digestion process rather than events occurring during library preparation or sequencing.

Conclusions: DNase I is a sequence-specific enzyme, with a specificity that may depend on experimental conditions. This sequence specificity is not taken into account by existing pipelines for identifying open chromatin regions. Care must be taken when interpreting DNase I results, especially when looking at the precise locations of the reads. Future studies may be able to improve the sensitivity and precision of chromatin state measurement by compensating for sequence bias.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">e69853</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23922824">23922824</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0069853">10.1371/journal.pone.0069853</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Criteria for inference of chromothripsis in cancer genomes.</h4>
<p class="authors"><span class="author">Korbel JO</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany. Electronic address: <a href="mailto:%6A%61%6E.%6B%6F%72%62%65%6C%40%65%6D%62%6C.%64%65">&#106;&#97;&#110;&#46;&#107;&#111;&#114;&#98;&#101;&#108;&#64;&#101;&#109;&#98;&#108;&#46;&#100;&#101;</a>.</p>
<p class="abstract">Chromothripsis scars the genome when localized chromosome shattering and repair occurs in a one-off catastrophe. Outcomes of this process are detectable as massive DNA rearrangements affecting one or a few chromosomes. Although recent findings suggest a crucial role of chromothripsis in cancer development, the reproducible inference of this process remains challenging, requiring that cataclysmic one-off rearrangements be distinguished from localized lesions that occur progressively. We describe conceptual criteria for the inference of chromothripsis, based on ruling out the alternative hypothesis that stepwise rearrangements occurred. Robust means of inference may facilitate in-depth studies on the impact of, and the mechanisms underlying, chromothripsis.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">152</span>;<span class="number">6</span>;<span class="pagerange">1226-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23498933">23498933</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.02.023">10.1016/j.cell.2013.02.023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Piliation of Invasive Streptococcus pneumoniae Isolates in the Era before Pneumococcal Conjugate Vaccine Introduction in Malawi.</h4>
<p class="authors"><span class="author">Kulohoma BW</span>, <span class="author">Gray K</span>, <span class="author">Kamng&#39;ona A</span>, <span class="author">Cornick J</span>, <span class="author">Bentley SD</span>, <span class="author">Heyderman RS</span> and <span class="author">Everett DB</span></p>
<p class="affiliation">The Malawi-Liverpool-Wellcome Trust Clinical Research Programme, Blantyre, Malawi.</p>
<p class="abstract">The pneumococcal pilus has been shown to be an important determinant of adhesion and virulence in mouse models of colonization, pneumonia, and bacteremia. A pilus is capable of inducing protective immunity, supporting its inclusion in next-generation pneumococcal protein vaccine formulations. Whether this vaccine target is common among pneumococci in sub-Saharan Africa is uncertain. To define the prevalence and genetic diversity of type I and II pili among invasive pneumococci in Malawi prior to the introduction of the 13-valent pneumococcal conjugate vaccine (PCV13) into routine childhood immunization, we examined 188 Streptococcus pneumoniae isolates collected between 2002 and 2008 (17% serotype 1). In this region of high disease burden, we found a low frequency of invasive piliated pneumococci (14%) and pilus gene sequence diversity similar to that seen previously in multiple global pneumococcal lineages. All common serotypes with pilus were covered by PCV13 and so we predict that pilus prevalence will be reduced in the Malawian pneumococcal population after PCV13 introduction.</p>
<p class="publication"><span class="title">Clinical and vaccine immunology : CVI</span>&nbsp;<span class="year">2013</span>;<span class="volume">20</span>;<span class="number">11</span>;<span class="pagerange">1729-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24027261">24027261</a>; DOI: <a href="http://dx.doi.org/10.1128%2FCVI.00403-13">10.1128/CVI.00403-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome and transcriptome of Haemonchus contortus, a key model parasite for drug and vaccine discovery.</h4>
<p class="authors"><span class="author">Laing R</span>, <span class="author">Kikuchi T</span>, <span class="author">Martinelli A</span>, <span class="author">Tsai IJ</span>, <span class="author">Beech RN</span>, <span class="author">Redman E</span>, <span class="author">Holroyd N</span>, <span class="author">Bartley DJ</span>, <span class="author">Beasley H</span>, <span class="author">Britton C</span>, <span class="author">Curran D</span>, <span class="author">Devaney E</span>, <span class="author">Gilabert A</span>, <span class="author">Hunt M</span>, <span class="author">Jackson F</span>, <span class="author">Johnston SL</span>, <span class="author">Kryukov I</span>, <span class="author">Li K</span>, <span class="author">Morrison AA</span>, <span class="author">Reid AJ</span>, <span class="author">Sargison N</span>, <span class="author">Saunders GI</span>, <span class="author">Wasmuth JD</span>, <span class="author">Wolstenholme A</span>, <span class="author">Berriman M</span>, <span class="author">Gilleard JS</span> and <span class="author">Cotton JA</span></p><p class="abstract"><strong>Background</strong>: The small ruminant parasite Haemonchus contortus is the most widely used parasitic nematode in drug discovery, vaccine development and anthelmintic resistance research. Its remarkable propensity to develop resistance threatens the viability of the sheep industry in many regions of the world and provides a cautionary example of the effect of mass drug administration to control parasitic nematodes. Its phylogenetic position makes it particularly well placed for comparison with the free-living nematode Caenorhabditis elegans and the most economically important parasites of livestock and humans.</p><p class="abstract"><strong>Results</strong>: Here we report the detailed analysis of a draft genome assembly and extensive transcriptomic dataset for H. contortus. This represents the first genome to be published for a strongylid nematode and the most extensive transcriptomic dataset for any parasitic nematode reported to date. We show a general pattern of conservation of genome structure and gene content between H. contortus and C. elegans, but also a dramatic expansion of important parasite gene families. We identify genes involved in parasite-specific pathways such as blood feeding, neurological function, and drug metabolism. In particular, we describe complete gene repertoires for known drug target families, providing the most comprehensive understanding yet of the action of several important anthelmintics. Also, we identify a set of genes enriched in the parasitic stages of the lifecycle and the parasite gut that provide a rich source of vaccine and drug target candidates.</p><p class="abstract"><strong>Conclusions</strong>: The H. contortus genome and transcriptome provide an essential platform for postgenomic research in this and other important strongylid parasites.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E018130/1</span>; Canadian Institutes of Health Research: <span class="grant">230927</span>; Wellcome Trust: <span class="grant">067811</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">8</span>;<span class="pagerange">R88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23985316">23985316</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4054779">4054779</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-8-r88">10.1186/gb-2013-14-8-r88</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Etoposide Induces Nuclear Re-Localisation of AID.</h4>
<p class="authors"><span class="author">Lambert LJ</span>, <span class="author">Walker S</span>, <span class="author">Feltham J</span>, <span class="author">Lee HJ</span>, <span class="author">Reik W</span> and <span class="author">Houseley J</span></p>
<p class="affiliation">Epigenetics Programme, The Babraham Institute, Cambridge, United Kingdom.</p>
<p class="abstract">During B cell activation, the DNA lesions that initiate somatic hypermutation and class switch recombination are introduced by activation-induced cytidine deaminase (AID). AID is a highly mutagenic protein that is maintained in the cytoplasm at steady state, however AID is shuttled across the nuclear membrane and the protein transiently present in the nucleus appears sufficient for targeted alteration of immunoglobulin loci. AID has been implicated in epigenetic reprogramming in primordial germ cells and cell fusions and in induced pluripotent stem cells (iPS cells), however AID expression in non-B cells is very low. We hypothesised that epigenetic reprogramming would require a pathway that instigates prolonged nuclear residence of AID. Here we show that AID is completely re-localised to the nucleus during drug withdrawal following etoposide treatment, in the period in which double strand breaks (DSBs) are repaired. Re-localisation occurs 2-6 hours after etoposide treatment, and AID remains in the nucleus for 10 or more hours, during which time cells remain live and motile. Re-localisation is cell-cycle dependent and is only observed in G2. Analysis of DSB dynamics shows that AID is re-localised in response to etoposide treatment, however re-localisation occurs substantially after DSB formation and the levels of re-localisation do not correlate with γH2AX levels. We conclude that DSB formation initiates a slow-acting pathway which allows stable long-term nuclear localisation of AID, and that such a pathway may enable AID-induced DNA demethylation during epigenetic reprogramming.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">e82110</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24324754">24324754</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3852760">3852760</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0082110">10.1371/journal.pone.0082110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cerebral organoids model human brain development and microcephaly.</h4>
<p class="authors"><span class="author">Lancaster MA</span>, <span class="author">Renner M</span>, <span class="author">Martin CA</span>, <span class="author">Wenzel D</span>, <span class="author">Bicknell LS</span>, <span class="author">Hurles ME</span>, <span class="author">Homfray T</span>, <span class="author">Penninger JM</span>, <span class="author">Jackson AP</span> and <span class="author">Knoblich JA</span></p>
<p class="affiliation">Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna 1030, Austria.</p>
<p class="abstract">The complexity of the human brain has made it difficult to study many brain disorders in model organisms, highlighting the need for an in vitro model of human brain development. Here we have developed a human pluripotent stem cell-derived three-dimensional organoid culture system, termed cerebral organoids, that develop various discrete, although interdependent, brain regions. These include a cerebral cortex containing progenitor populations that organize and produce mature cortical neuron subtypes. Furthermore, cerebral organoids are shown to recapitulate features of human cortical development, namely characteristic progenitor zone organization with abundant outer radial glial stem cells. Finally, we use RNA interference and patient-specific induced pluripotent stem cells to model microcephaly, a disorder that has been difficult to recapitulate in mice. We demonstrate premature neuronal differentiation in patient organoids, a defect that could help to explain the disease phenotype. Together, these data show that three-dimensional organoids can recapitulate development and disease even in this most complex human tissue.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23995685">23995685</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12517">10.1038/nature12517</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression of recombinant ITGA2 and CD109 for the detection of human platelet antigen (HPA)-5 and -15 alloantibodies.</h4>
<p class="authors"><span class="author">Lane-Serff H</span>, <span class="author">Sun Y</span>, <span class="author">Metcalfe P</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton Cambridge, UK.</p>
<p class="publication"><span class="title">British journal of haematology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23406260">23406260</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fbjh.12252">10.1111/bjh.12252</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intestinal colonization resistance.</h4>
<p class="authors"><span class="author">Lawley TD</span> and <span class="author">Walker AW</span></p>
<p class="affiliation">Bacterial Pathogenesis Laboratory, Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%74%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Dense, complex microbial communities, collectively termed the microbiota, occupy a diverse array of niches along the length of the mammalian intestinal tract. During health and in the absence of antibiotic exposure the microbiota can effectively inhibit colonization and overgrowth by invading microbes such as pathogens. This phenomenon is called &#39;colonization resistance&#39; and is associated with a stable and diverse microbiota in tandem with a controlled lack of inflammation, and involves specific interactions between the mucosal immune system and the microbiota. Here we overview the microbial ecology of the healthy mammalian intestinal tract and highlight the microbe-microbe and microbe-host interactions that promote colonization resistance. Emerging themes highlight immunological (T helper type 17/regulatory T-cell balance), microbiota (diverse and abundant) and metabolic (short-chain fatty acid) signatures of intestinal health and colonization resistance. Intestinal pathogens use specific virulence factors or exploit antibiotic use to subvert colonization resistance for their own benefit by triggering inflammation to disrupt the harmony of the intestinal ecosystem. A holistic view that incorporates immunological and microbiological facets of the intestinal ecosystem should facilitate the development of immunomodulatory and microbe-modulatory therapies that promote intestinal homeostasis and colonization resistance.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">93614</span>; Wellcome Trust: <span class="grant">076964</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Immunology</span>&nbsp;<span class="year">2013</span>;<span class="volume">138</span>;<span class="number">1</span>;<span class="pagerange">1-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23240815">23240815</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3533696">3533696</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2567.2012.03616.x">10.1111/j.1365-2567.2012.03616.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Murine models to study Clostridium difficile infection and transmission.</h4>
<p class="authors"><span class="author">Lawley TD</span> and <span class="author">Young VB</span></p>
<p class="affiliation">Bacterial Pathogenesis Laboratory, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Clostridium difficile is the leading cause of antibiotic-associated diarrhea in healthcare facilities worldwide. C. difficile infections are difficult to treat because of the high rate of disease recurrence after antibiotic therapy, leaving few treatment options for patients. C. difficile is also difficult to contain within a healthcare setting due to a highly-transmissible, resistant spore form that challenges standard infection control measures. The recent development of murine infection models to study the interactions between C. difficile, the host and the microbiota are providing novel insight into the mechanisms of pathogenesis and transmission that should guide the development of therapies and intervention measures.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0901743</span>; NIAID NIH HHS: <span class="grant">AI090871</span>, <span class="grant">U19 AI090871</span>; Wellcome Trust: <span class="grant">076964</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Anaerobe</span>&nbsp;<span class="year">2013</span>;<span class="volume">24</span>;<span class="pagerange">94-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24076318">24076318</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4089867">4089867</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.anaerobe.2013.09.008">10.1016/j.anaerobe.2013.09.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Richness of human gut microbiome correlates with metabolic markers.</h4>
<p class="authors"><span class="author">Le Chatelier E</span>, <span class="author">Nielsen T</span>, <span class="author">Qin J</span>, <span class="author">Prifti E</span>, <span class="author">Hildebrand F</span>, <span class="author">Falony G</span>, <span class="author">Almeida M</span>, <span class="author">Arumugam M</span>, <span class="author">Batto JM</span>, <span class="author">Kennedy S</span>, <span class="author">Leonard P</span>, <span class="author">Li J</span>, <span class="author">Burgdorf K</span>, <span class="author">Grarup N</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Brandslund I</span>, <span class="author">Nielsen HB</span>, <span class="author">Juncker AS</span>, <span class="author">Bertalan M</span>, <span class="author">Levenez F</span>, <span class="author">Pons N</span>, <span class="author">Rasmussen S</span>, <span class="author">Sunagawa S</span>, <span class="author">Tap J</span>, <span class="author">Tims S</span>, <span class="author">Zoetendal EG</span>, <span class="author">Brunak S</span>, <span class="author">Cl&eacute;ment K</span>, <span class="author">Dor&eacute; J</span>, <span class="author">Kleerebezem M</span>, <span class="author">Kristiansen K</span>, <span class="author">Renault P</span>, <span class="author">Sicheritz-Ponten T</span>, <span class="author">de Vos WM</span>, <span class="author">Zucker JD</span>, <span class="author">Raes J</span>, <span class="author">Hansen T</span>, <span class="author">MetaHIT consortium</span>, <span class="author">Bork P</span>, <span class="author">Wang J</span>, <span class="author">Ehrlich SD</span> and <span class="author">Pedersen O</span></p>
<p class="affiliation">INRA, Institut National de la Recherche Agronomique, US1367 Metagenopolis, 78350 Jouy en Josas, France.</p><p class="abstract">We are facing a global metabolic health crisis provoked by an obesity epidemic. Here we report the human gut microbial composition in a population sample of 123 non-obese and 169 obese Danish individuals. We find two groups of individuals that differ by the number of gut microbial genes and thus gut bacterial richness. They contain known and previously unknown bacterial species at different proportions; individuals with a low bacterial richness (23% of the population) are characterized by more marked overall adiposity, insulin resistance and dyslipidaemia and a more pronounced inflammatory phenotype when compared with high bacterial richness individuals. The obese individuals among the lower bacterial richness group also gain more weight over time. Only a few bacterial species are sufficient to distinguish between individuals with high and low bacterial richness, and even between lean and obese participants. Our classifications based on variation in the gut microbiome identify subsets of individuals in the general white adult population who may be at increased risk of progressing to adiposity-associated co-morbidities.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">500</span>;<span class="number">7464</span>;<span class="pagerange">541-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23985870">23985870</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12506">10.1038/nature12506</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human SNP Links Differential Outcomes in Inflammatory and Infectious Disease to a FOXO3-Regulated Pathway.</h4>
<p class="authors"><span class="author">Lee JC</span>, <span class="author">Esp&eacute;li M</span>, <span class="author">Anderson CA</span>, <span class="author">Linterman MA</span>, <span class="author">Pocock JM</span>, <span class="author">Williams NJ</span>, <span class="author">Roberts R</span>, <span class="author">Viatte S</span>, <span class="author">Fu B</span>, <span class="author">Peshu N</span>, <span class="author">Hien TT</span>, <span class="author">Phu NH</span>, <span class="author">Wesley E</span>, <span class="author">Edwards C</span>, <span class="author">Ahmad T</span>, <span class="author">Mansfield JC</span>, <span class="author">Gearry R</span>, <span class="author">Dunstan S</span>, <span class="author">Williams TN</span>, <span class="author">Barton A</span>, <span class="author">Vinuesa CG</span>, <span class="author">UK IBD Genetics Consortium</span>, <span class="author">Parkes M</span>, <span class="author">Lyons PA</span> and <span class="author">Smith KG</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, University of Cambridge, Cambridge Biomedical Campus, Cambridge CB2 0XY, UK; Department of Medicine, University of Cambridge School of Clinical Medicine, Addenbrooke's Hospital, Cambridge CB2 0QQ, UK.</p>
<p class="abstract">The clinical course and eventual outcome, or prognosis, of complex diseases varies enormously between affected individuals. This variability critically determines the impact a disease has on a patient's life but is very poorly understood. Here, we exploit existing genome-wide association study data to gain insight into the role of genetics in prognosis. We identify a noncoding polymorphism in FOXO3A (rs12212067: T > G) at which the minor (G) allele, despite not being associated with disease susceptibility, is associated with a milder course of Crohn's disease and rheumatoid arthritis and with increased risk of severe malaria. Minor allele carriage is shown to limit inflammatory responses in monocytes via a FOXO3-driven pathway, which through TGFβ1 reduces production of proinflammatory cytokines, including TNFα, and increases production of anti-inflammatory cytokines, including IL-10. Thus, we uncover a shared genetic contribution to prognosis in distinct diseases that operates via a FOXO3-driven pathway modulating inflammatory responses.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24035192">24035192</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.08.034">10.1016/j.cell.2013.08.034</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants in the HLA-DRB1-HLA-DQA1 HLA class II region are associated with susceptibility to visceral leishmaniasis.</h4>
<p class="authors"><span class="author">LeishGEN Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Fakiola M</span>, <span class="author">Strange A</span>, <span class="author">Cordell HJ</span>, <span class="author">Miller EN</span>, <span class="author">Pirinen M</span>, <span class="author">Su Z</span>, <span class="author">Mishra A</span>, <span class="author">Mehrotra S</span>, <span class="author">Monteiro GR</span>, <span class="author">Band G</span>, <span class="author">Bellenguez C</span>, <span class="author">Dronov S</span>, <span class="author">Edkins S</span>, <span class="author">Freeman C</span>, <span class="author">Giannoulatou E</span>, <span class="author">Gray E</span>, <span class="author">Hunt SE</span>, <span class="author">Lacerda HG</span>, <span class="author">Langford C</span>, <span class="author">Pearson R</span>, <span class="author">Pontes NN</span>, <span class="author">Rai M</span>, <span class="author">Singh SP</span>, <span class="author">Smith L</span>, <span class="author">Sousa O</span>, <span class="author">Vukcevic D</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Casas JP</span>, <span class="author">Corvin A</span>, <span class="author">Duncanson A</span>, <span class="author">Jankowski J</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">Palmer CN</span>, <span class="author">Plomin R</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Trembath RC</span>, <span class="author">Viswanathan AC</span>, <span class="author">Wood NW</span>, <span class="author">Wilson ME</span>, <span class="author">Deloukas P</span>, <span class="author">Peltonen L</span>, <span class="author">Christiansen F</span>, <span class="author">Witt C</span>, <span class="author">Jeronimo SM</span>, <span class="author">Sundar S</span>, <span class="author">Spencer CC</span>, <span class="author">Blackwell JM</span> and <span class="author">Donnelly P</span></p>
<p class="affiliation">1] Cambridge Institute for Medical Research, University of Cambridge School of Clinical Medicine, Addenbrooke's Hospital, Cambridge, UK. [2].</p>
<p class="abstract">To identify susceptibility loci for visceral leishmaniasis, we undertook genome-wide association studies in two populations: 989 cases and 1,089 controls from India and 357 cases in 308 Brazilian families (1,970 individuals). The HLA-DRB1-HLA-DQA1 locus was the only region to show strong evidence of association in both populations. Replication at this region was undertaken in a second Indian population comprising 941 cases and 990 controls, and combined analysis across the three cohorts for rs9271858 at this locus showed P(combined) = 2.76 × 10(-17) and odds ratio (OR) = 1.41, 95% confidence interval (CI) = 1.30-1.52. A conditional analysis provided evidence for multiple associations within the HLA-DRB1-HLA-DQA1 region, and a model in which risk differed between three groups of haplotypes better explained the signal and was significant in the Indian discovery and replication cohorts. In conclusion, the HLA-DRB1-HLA-DQA1 HLA class II region contributes to visceral leishmaniasis susceptibility in India and Brazil, suggesting shared genetic risk factors for visceral leishmaniasis that cross the epidemiological divides of geography and parasite species.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">2</span>;<span class="pagerange">208-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23291585">23291585</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2518">10.1038/ng.2518</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide profiling of chromosome interactions in Plasmodium falciparum characterizes nuclear architecture and reconfigurations associated with antigenic variation.</h4>
<p class="authors"><span class="author">Lemieux JE</span>, <span class="author">Kyes SA</span>, <span class="author">Otto TD</span>, <span class="author">Feller AI</span>, <span class="author">Eastman RT</span>, <span class="author">Pinches RA</span>, <span class="author">Berriman M</span>, <span class="author">Su XZ</span> and <span class="author">Newbold CI</span></p>
<p class="affiliation">Weatherall Institute of Molecular Medicine, Headington, Oxford, OX3 9DS, UK; National Institute of Allergy and Infectious Disease, NIH, Rockville, MD, 20892, USA.</p><p class="abstract">Spatial relationships within the eukaryotic nucleus are essential for proper nuclear function. In Plasmodium falciparum, the repositioning of chromosomes has been implicated in the regulation of the expression of genes responsible for antigenic variation, and the formation of a single, peri-nuclear nucleolus results in the clustering of rDNA. Nevertheless, the precise spatial relationships between chromosomes remain poorly understood, because, until recently, techniques with sufficient resolution have been lacking. Here we have used chromosome conformation capture and second-generation sequencing to study changes in chromosome folding and spatial positioning that occur during switches in var gene expression. We have generated maps of chromosomal spatial affinities within the P.&thinsp;falciparum nucleus at 25&thinsp;Kb resolution, revealing a structured nucleolus, an absence of chromosome territories, and confirming previously identified clustering of heterochromatin foci. We show that switches in var gene expression do not appear to involve interaction with a distant enhancer, but do result in local changes at the active locus. These maps reveal the folding properties of malaria chromosomes, validate known physical associations, and characterize the global landscape of spatial interactions. Collectively, our data provide critical information for a better understanding of gene expression regulation and antigenic variation in malaria parasites.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">082130</span>, <span class="grant">082130/Z/07/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">90</span>;<span class="number">3</span>;<span class="pagerange">519-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23980881">23980881</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3894959">3894959</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fmmi.12381">10.1111/mmi.12381</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The NGS WikiBook: a dynamic collaborative online training effort with long-term sustainability.</h4>
<p class="authors"><span class="author">Li JW</span>, <span class="author">Bolser D</span>, <span class="author">Manske M</span>, <span class="author">Giorgi FM</span>, <span class="author">Vyahhi N</span>, <span class="author">Usadel B</span>, <span class="author">Clavijo BJ</span>, <span class="author">Chan TF</span>, <span class="author">Wong N</span>, <span class="author">Zerbino D</span> and <span class="author">Schneider MV</span></p>
<p class="affiliation">School of Life Sciences, The Chinese University of Hong Kong, Shatin, New Territories, Hong Kong SAR. Tel.: +852-39431302; <a href="mailto:%6D%61%72%63%6F%77%61%6E%67%65%72%40%67%6D%61%69%6C.%63%6F%6D">&#109;&#97;&#114;&#99;&#111;&#119;&#97;&#110;&#103;&#101;&#114;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a>.</p><p class="abstract">Next-generation sequencing (NGS) is increasingly being adopted as the backbone of biomedical research. With the commercialization of various affordable desktop sequencers, NGS will be reached by increasing numbers of cellular and molecular biologists, necessitating community consensus on bioinformatics protocols to tackle the exponential increase in quantity of sequence data. The current resources for NGS informatics are extremely fragmented. Finding a centralized synthesis is difficult. A multitude of tools exist for NGS data analysis; however, none of these satisfies all possible uses and needs. This gap in functionality could be filled by integrating different methods in customized pipelines, an approach helped by the open-source nature of many NGS programmes. Drawing from community spirit and with the use of the Wikipedia framework, we have initiated a collaborative NGS resource: The NGS WikiBook. We have collected a sufficient amount of text to incentivize a broader community to contribute to it. Users can search, browse, edit and create new content, so as to facilitate self-learning and feedback to the community. The overall structure and style for this dynamic material is designed for the bench biologists and non-bioinformaticians. The flexibility of online material allows the readers to ignore details in a first read, yet have immediate access to the information they need. Each chapter comes with practical exercises so readers may familiarize themselves with each step. The NGS WikiBook aims to create a collective laboratory book and protocol that explains the key concepts and describes best practices in this fast-evolving field.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">548-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23793381">23793381</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3771235">3771235</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbib%2Fbbt045">10.1093/bib/bbt045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The piggybac transposon displays local and distant reintegration preferences and can cause mutations at non-canonical integration sites.</h4>
<p class="authors"><span class="author">Li MA</span>, <span class="author">Pettitt SJ</span>, <span class="author">Eckert S</span>, <span class="author">Ning Z</span>, <span class="author">Rice S</span>, <span class="author">Cadi&ntilde;anos J</span>, <span class="author">Yusa K</span>, <span class="author">Conte N</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, CB10 1SA.</p>
<p class="abstract">The DNA transposon piggyBac is widely used as a tool in mammalian experimental systems for transgenesis, mutagenesis and genome engineering. We have characterised genome-wide insertion site preferences of piggyBac by sequencing a large set of integration sites arising from transposition from two separate genomic loci and a plasmid donor in mouse embryonic stem cells. We found that piggyBac preferentially integrates locally to the excision site when mobilised from a chromosomal location, and identified other non-local regions of the genome with elevated insertion frequencies. PiggyBac insertions were associated with expressed genes and markers of open chromatin structure, and were excluded from heterochromatin. At the nucleotide level, piggyBac prefers to insert into TA-rich regions within a broader GC-rich context. We also found that piggyBac can insert into sites other than its known TTAA insertion site at low frequency (2%). Such insertions introduce mismatches that are repaired with signatures of host cell mismatch repair pathways. Transposons could be mobilised from plasmids with the observed mismatches, indicating that piggyBac could generate point mutations in the genome.</p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23358416">23358416</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.00670-12">10.1128/MCB.00670-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Non dominant-negative KCNJ2 gene mutations leading to Andersen-Tawil syndrome with an isolated cardiac phenotype.</h4>
<p class="authors"><span class="author">Limberg MM</span>, <span class="author">Zumhagen S</span>, <span class="author">Netter MF</span>, <span class="author">Coffey AJ</span>, <span class="author">Grace A</span>, <span class="author">Rogers J</span>, <span class="author">B&ouml;ckelmann D</span>, <span class="author">Rinn&eacute; S</span>, <span class="author">Stallmeyer B</span>, <span class="author">Decher N</span> and <span class="author">Schulze-Bahr E</span></p>
<p class="affiliation">Institut f&uuml;r Physiologie und Pathophysiologie, Vegetative Physiologie, Philipps-University Marburg, Deutschhausstra&szlig;e 1-2, 35037 Marburg, Germany.</p><p class="abstract">Andersen-Tawil syndrome (ATS) is characterized by dysmorphic features, periodic paralyses and abnormal ventricular repolarization. After genotyping a large set of patients with congenital long-QT syndrome, we identified two novel, heterozygous KCNJ2 mutations (p.N318S, p.W322C) located in the C-terminus of the Kir2.1 subunit. These mutations have a different localization than classical ATS mutations which are mostly located at a potential interaction face with the slide helix or at the interface between the C-termini. Mutation carriers were without the key features of ATS, causing an isolated cardiac phenotype. While the N318S mutants regularly reached the plasma membrane, W322C mutants primarily resided in late endosomes. Co-expression of N318S or W322C with wild-type Kir2.1 reduced current amplitudes only by 20-25&nbsp;%. This mild loss-of-function for the heteromeric channels resulted from defective channel trafficking (W322C) or gating (N318S). Strikingly, and in contrast to the majority of ATS mutations, neither mutant caused a dominant-negative suppression of wild-type Kir2.1, Kir2.2 and Kir2.3 currents. Thus, a mild reduction of native Kir2.x currents by non dominant-negative mutants may cause ATS with an isolated cardiac phenotype.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Basic research in cardiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">108</span>;<span class="number">3</span>;<span class="pagerange">353</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23644778">23644778</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00395-013-0353-1">10.1007/s00395-013-0353-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Amphotericin B increases influenza A virus infection by preventing IFITM3-mediated restriction.</h4>
<p class="authors"><span class="author">Lin TY</span>, <span class="author">Chin CR</span>, <span class="author">Everitt AR</span>, <span class="author">Clare S</span>, <span class="author">Perreira JM</span>, <span class="author">Savidis G</span>, <span class="author">Aker AM</span>, <span class="author">John SP</span>, <span class="author">Sarlah D</span>, <span class="author">Carreira EM</span>, <span class="author">Elledge SJ</span>, <span class="author">Kellam P</span> and <span class="author">Brass AL</span></p>
<p class="affiliation">Department of Microbiology and Physiological Systems, University of Massachusetts Medical School, Worcester, MA 01655, USA.</p><p class="abstract">The IFITMs inhibit influenza A virus (IAV) replication in vitro and in vivo. Here, we establish that the antimycotic heptaen, amphotericin B (AmphoB), prevents IFITM3-mediated restriction of IAV, thereby increasing viral replication. Consistent with its neutralization of IFITM3, a clinical preparation of AmphoB, AmBisome, reduces the majority of interferon&#39;s protective effect against IAV in vitro. Mechanistic studies reveal that IFITM1 decreases host-membrane fluidity, suggesting both a possible mechanism for IFITM-mediated restriction and its negation by AmphoB. Notably, we reveal that mice treated with AmBisome succumbed to a normally mild IAV infection, similar to animals deficient in Ifitm3. Therefore, patients receiving antifungal therapy with clinical preparations of AmphoB may be functionally immunocompromised and thus more vulnerable to influenza, as well as other IFITM3-restricted viral infections.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">1R01AI091786</span>, <span class="grant">R01 AI091786</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cell reports</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">895-908</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24268777">24268777</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3898084">3898084</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.celrep.2013.10.033">10.1016/j.celrep.2013.10.033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A combination of improved differential and global RNA-seq reveals pervasive transcription initiation and events in all stages of the life-cycle of functional RNAs in Propionibacterium acnes, a major contributor to wide-spread human disease.</h4>
<p class="authors"><span class="author">Lin YF</span>, <span class="author">A DR</span>, <span class="author">Guan S</span>, <span class="author">Mamanova L</span> and <span class="author">McDowall KJ</span></p>
<p class="abstract">Background: Sequencing of the genome of Propionibacterium acnes produced a catalogue of genes many of which enable this organism to colonise skin and survive exposure to the elements. Despite this platform, there was little understanding of the gene regulation that gives rise to an organism that has a major impact on human health and wellbeing and causes infections beyond the skin. To address this situation, we have undertaken a genome--wide study of gene regulation using a combination of improved differential and global RNA-sequencing and an analytical approach that takes into account the inherent noise within the data.

Results: We have produced nucleotide-resolution transcriptome maps that identify and differentiate sites of transcription initiation from sites of stable RNA processing and mRNA cleavage. Moreover, analysis of these maps provides strong evidence for 'pervasive' transcription and shows that contrary to initial indications it is not biased towards the production of antisense RNAs. In addition, the maps reveal an extensive array of riboswitches, leaderless mRNAs and small non-protein-coding RNAs alongside vegetative promoters and post-transcriptional events, which includes unusual tRNA processing. The identification of such features will inform models of complex gene regulation, as illustrated here for ribonucleotide reductases and a potential quorum-sensing, two-component system.

Conclusions: The approach described here, which is transferable to any bacterial species, has produced a step increase in whole-cell knowledge of gene regulation in P. acnes. Continued expansion of our maps to include transcription associated with different growth conditions and genetic backgrounds will provide a new platform from which to computationally model the gene expression that determines the physiology of P. acnes and its role in human disease.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">620</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24034785">24034785</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-14-620">10.1186/1471-2164-14-620</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The future role of genetic screening to detect newborns at risk of childhood-onset hearing loss.</h4>
<p class="authors"><span class="author">Linden Phillips L</span>, <span class="author">Bitner-Glindzicz M</span>, <span class="author">Lench N</span>, <span class="author">Steel KP</span>, <span class="author">Langford C</span>, <span class="author">Dawson SJ</span>, <span class="author">Davis A</span>, <span class="author">Simpson S</span> and <span class="author">Packer C</span></p>
<p class="affiliation">* National Institute for Health Research (NIHR) Horizon Scanning Centre, School of Health and Population Sciences, University of Birmingham , Birmingham , UK.</p>
<p class="abstract">Abstract Objective: To explore the future potential of genetic screening to detect newborns at risk of childhood-onset hearing loss. Design: An expert led discussion of current and future developments in genetic technology and the knowledge base of genetic hearing loss to determine the viability of genetic screening and the implications for screening policy. Results and Discussion: Despite increasing pressure to adopt genetic technologies, a major barrier for genetic screening in hearing loss is the uncertain clinical significance of the identified mutations and their interactions. Only when a reliable estimate of the future risk of hearing loss can be made at a reasonable cost, will genetic screening become viable. Given the speed of technological advancement this may be within the next 10 years. Decision-makers should start to consider how genetic screening could augment current screening programmes as well as the associated data processing and storage requirements. Conclusion: In the interim, we suggest that decision makers consider the benefits of (1) genetically testing all newborns and children with hearing loss, to determine aetiology and to increase knowledge of the genetic causes of hearing loss, and (2) consider screening pregnant women for the m.1555A> G mutation to reduce the risk of aminoglycoside antibiotic-associated hearing loss.</p>
<p class="publication"><span class="title">International journal of audiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">52</span>;<span class="number">2</span>;<span class="pagerange">124-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23131088" rel="external">23131088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3545543" rel="external">3545543</a>; DOI: <a href="http://dx.doi.org/10.3109%2F14992027.2012.733424" rel="external">10.3109/14992027.2012.733424</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Survey of culture, goldengate assay, universal biosensor assay, and 16S rRNA Gene sequencing as alternative methods of bacterial pathogen detection.</h4>
<p class="authors"><span class="author">Lindsay B</span>, <span class="author">Pop M</span>, <span class="author">Antonio M</span>, <span class="author">Walker AW</span>, <span class="author">Mai V</span>, <span class="author">Ahmed D</span>, <span class="author">Oundo J</span>, <span class="author">Tamboura B</span>, <span class="author">Panchalingam S</span>, <span class="author">Levine MM</span>, <span class="author">Kotloff K</span>, <span class="author">Li S</span>, <span class="author">Magder LS</span>, <span class="author">Paulson JN</span>, <span class="author">Liu B</span>, <span class="author">Ikumapayi U</span>, <span class="author">Ebruke C</span>, <span class="author">Dione M</span>, <span class="author">Adeyemi M</span>, <span class="author">Rance R</span>, <span class="author">Stares MD</span>, <span class="author">Ukhanova M</span>, <span class="author">Barnes B</span>, <span class="author">Lewis I</span>, <span class="author">Ahmed F</span>, <span class="author">Alam MT</span>, <span class="author">Amin R</span>, <span class="author">Siddiqui S</span>, <span class="author">Ochieng JB</span>, <span class="author">Ouma E</span>, <span class="author">Juma J</span>, <span class="author">Mailu E</span>, <span class="author">Omore R</span>, <span class="author">O&#39;Reilly CE</span>, <span class="author">Hannis J</span>, <span class="author">Manalili S</span>, <span class="author">Deleon J</span>, <span class="author">Yasuda I</span>, <span class="author">Blyn L</span>, <span class="author">Ranken R</span>, <span class="author">Li F</span>, <span class="author">Housley R</span>, <span class="author">Ecker DJ</span>, <span class="author">Hossain MA</span>, <span class="author">Breiman RF</span>, <span class="author">Morris JG</span>, <span class="author">McDaniel TK</span>, <span class="author">Parkhill J</span>, <span class="author">Saha D</span>, <span class="author">Sampath R</span>, <span class="author">Stine OC</span> and <span class="author">Nataro JP</span></p>
<p class="affiliation">University of Maryland, School of Medicine, Baltimore, Maryland, USA.</p><p class="abstract">Cultivation-based assays combined with PCR or enzyme-linked immunosorbent assay (ELISA)-based methods for finding virulence factors are standard methods for detecting bacterial pathogens in stools; however, with emerging molecular technologies, new methods have become available. The aim of this study was to compare four distinct detection technologies for the identification of pathogens in stools from children under 5 years of age in The Gambia, Mali, Kenya, and Bangladesh. The children were identified, using currently accepted clinical protocols, as either controls or cases with moderate to severe diarrhea. A total of 3,610 stool samples were tested by established clinical culture techniques: 3,179 DNA samples by the Universal Biosensor assay (Ibis Biosciences, Inc.), 1,466 DNA samples by the GoldenGate assay (Illumina), and 1,006 DNA samples by sequencing of 16S rRNA genes. Each method detected different proportions of samples testing positive for each of seven enteric pathogens, enteroaggregative Escherichia coli (EAEC), enterotoxigenic E. coli (ETEC), enteropathogenic E. coli (EPEC), Shigella spp., Campylobacter jejuni, Salmonella enterica, and Aeromonas spp. The comparisons among detection methods included the frequency of positive stool samples and kappa values for making pairwise comparisons. Overall, the standard culture methods detected Shigella spp., EPEC, ETEC, and EAEC in smaller proportions of the samples than either of the methods based on detection of the virulence genes from DNA in whole stools. The GoldenGate method revealed the greatest agreement with the other methods. The agreement among methods was higher in cases than in controls. The new molecular technologies have a high potential for highly sensitive identification of bacterial diarrheal pathogens.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">10</span>;<span class="pagerange">3263-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23884998">23884998</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3811648">3811648</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.01342-13">10.1128/JCM.01342-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dense genotyping of immune-related disease regions identifies nine new risk loci for primary sclerosing cholangitis.</h4>
<p class="authors"><span class="author">Liu JZ</span>, <span class="author">Hov JR</span>, <span class="author">Folseraas T</span>, <span class="author">Ellinghaus E</span>, <span class="author">Rushbrook SM</span>, <span class="author">Doncheva NT</span>, <span class="author">Andreassen OA</span>, <span class="author">Weersma RK</span>, <span class="author">Weism&uuml;ller TJ</span>, <span class="author">Eksteen B</span>, <span class="author">Invernizzi P</span>, <span class="author">Hirschfield GM</span>, <span class="author">Gotthardt DN</span>, <span class="author">Pares A</span>, <span class="author">Ellinghaus D</span>, <span class="author">Shah T</span>, <span class="author">Juran BD</span>, <span class="author">Milkiewicz P</span>, <span class="author">Rust C</span>, <span class="author">Schramm C</span>, <span class="author">M&uuml;ller T</span>, <span class="author">Srivastava B</span>, <span class="author">Dalekos G</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Herms S</span>, <span class="author">Winkelmann J</span>, <span class="author">Mitrovic M</span>, <span class="author">Braun F</span>, <span class="author">Ponsioen CY</span>, <span class="author">Croucher PJ</span>, <span class="author">Sterneck M</span>, <span class="author">Teufel A</span>, <span class="author">Mason AL</span>, <span class="author">Saarela J</span>, <span class="author">Leppa V</span>, <span class="author">Dorfman R</span>, <span class="author">Alvaro D</span>, <span class="author">Floreani A</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Rich SS</span>, <span class="author">Thompson WK</span>, <span class="author">Schork AJ</span>, <span class="author">N&aelig;ss S</span>, <span class="author">Thomsen I</span>, <span class="author">Mayr G</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Hveem K</span>, <span class="author">Cleynen I</span>, <span class="author">Gutierrez-Achury J</span>, <span class="author">Rica&ntilde;o-Ponce I</span>, <span class="author">van Heel D</span>, <span class="author">Bj&ouml;rnsson E</span>, <span class="author">Sandford RN</span>, <span class="author">Durie PR</span>, <span class="author">Melum E</span>, <span class="author">Vatn MH</span>, <span class="author">Silverberg MS</span>, <span class="author">Duerr RH</span>, <span class="author">Padyukov L</span>, <span class="author">Brand S</span>, <span class="author">Sans M</span>, <span class="author">Annese V</span>, <span class="author">Achkar JP</span>, <span class="author">Boberg KM</span>, <span class="author">Marschall HU</span>, <span class="author">Chazouill&egrave;res O</span>, <span class="author">Bowlus CL</span>, <span class="author">Wijmenga C</span>, <span class="author">Schrumpf E</span>, <span class="author">Vermeire S</span>, <span class="author">Albrecht M</span>, <span class="author">UK-PSCSC Consortium</span>, <span class="author">International IBD Genetics Consortium</span>, <span class="author">Rioux JD</span>, <span class="author">Alexander G</span>, <span class="author">Bergquist A</span>, <span class="author">Cho J</span>, <span class="author">Schreiber S</span>, <span class="author">Manns MP</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Dale AM</span>, <span class="author">Chapman RW</span>, <span class="author">Lazaridis KN</span>, <span class="author">International PSC Study Group</span>, <span class="author">Franke A</span>, <span class="author">Anderson CA</span> and <span class="author">Karlsen TH</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Primary sclerosing cholangitis (PSC) is a severe liver disease of unknown etiology leading to fibrotic destruction of the bile ducts and ultimately to the need for liver transplantation. We compared 3,789 PSC cases of European ancestry to 25,079 population controls across 130,422 SNPs genotyped using the Immunochip. We identified 12 genome-wide significant associations outside the human leukocyte antigen (HLA) complex, 9 of which were new, increasing the number of known PSC risk loci to 16. Despite comorbidity with inflammatory bowel disease (IBD) in 72% of the cases, 6 of the 12 loci showed significantly stronger association with PSC than with IBD, suggesting overlapping yet distinct genetic architectures for these two diseases. We incorporated association statistics from 7 diseases clinically occurring with PSC in the analysis and found suggestive evidence for 33 additional pleiotropic PSC risk loci. Together with network analyses, these findings add to the genetic risk map of PSC and expand on the relationship between PSC and other immune-mediated diseases.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0601816</span>; NCATS NIH HHS: <span class="grant">UL1 TR000005</span>; NIDDK NIH HHS: <span class="grant">U01 DK062432</span>; Wellcome Trust: <span class="grant">091745</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">670-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23603763">23603763</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3667736">3667736</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2616">10.1038/ng.2616</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fine Mapping of the Pond Snail Left-Right Asymmetry (Chirality) Locus Using RAD-Seq and Fibre-FISH.</h4>
<p class="authors"><span class="author">Liu MM</span>, <span class="author">Davey JW</span>, <span class="author">Banerjee R</span>, <span class="author">Han J</span>, <span class="author">Yang F</span>, <span class="author">Aboobaker A</span>, <span class="author">Blaxter ML</span> and <span class="author">Davison A</span></p>
<p class="affiliation">School of Biology, University of Nottingham, University Park, Nottingham, United Kingdom ; Department of Plant Sciences, University of Cambridge, Cambridge, United Kingdom.</p>
<p class="abstract">The left-right asymmetry of snails, including the direction of shell coiling, is determined by the delayed effect of a maternal gene on the chiral twist that takes place during early embryonic cell divisions. Yet, despite being a well-established classical problem, the identity of the gene and the means by which left-right asymmetry is established in snails remain unknown. We here demonstrate the power of new genomic approaches for identification of the chirality gene, "D". First, heterozygous (Dd) pond snails Lymnaea stagnalis were self-fertilised or backcrossed, and the genotype of more than six thousand offspring inferred, either dextral (DD/Dd) or sinistral (dd). Then, twenty of the offspring were used for Restriction-site-Associated DNA Sequencing (RAD-Seq) to identify anonymous molecular markers that are linked to the chirality locus. A local genetic map was constructed by genotyping three flanking markers in over three thousand snails. The three markers lie either side of the chirality locus, with one very tightly linked (<0.1 cM). Finally, bacterial artificial chromosomes (BACs) were isolated that contained the three loci. Fluorescent in situ hybridization (FISH) of pachytene cells showed that the three BACs tightly cluster on the same bivalent chromosome. Fibre-FISH identified a region of greater that ∼0.4 Mb between two BAC clone markers that must contain D. This work therefore establishes the resources for molecular identification of the chirality gene and the variation that underpins sinistral and dextral coiling. More generally, the results also show that combining genomic technologies, such as RAD-Seq and high resolution FISH, is a robust approach for mapping key loci in non-model systems.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">e71067</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23951082">23951082</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3741322">3741322</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0071067">10.1371/journal.pone.0071067</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detecting and Characterizing Genomic Signatures of Positive Selection in Global Populations.</h4>
<p class="authors"><span class="author">Liu X</span>, <span class="author">Ong RT</span>, <span class="author">Pillai EN</span>, <span class="author">Elzein AM</span>, <span class="author">Small KS</span>, <span class="author">Clark TG</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Teo YY</span></p>
<p class="affiliation">NUS Graduate School for Integrative Science and Engineering, National University of Singapore, Singapore 117456, Singapore; Saw Swee Hock School of Public Health, National University of Singapore, Singapore 117597, Singapore.</p>
<p class="abstract">Natural selection is a significant force that shapes the architecture of the human genome and introduces diversity across global populations. The question of whether advantageous mutations have arisen in the human genome as a result of single or multiple mutation events remains unanswered except for the fact that there exist a handful of genes such as those that confer lactase persistence, affect skin pigmentation, or cause sickle cell anemia. We have developed a long-range-haplotype method for identifying genomic signatures of positive selection to complement existing methods, such as the integrated haplotype score (iHS) or cross-population extended haplotype homozygosity (XP-EHH), for locating signals across the entire allele frequency spectrum. Our method also locates the founder haplotypes that carry the advantageous variants and infers their corresponding population frequencies. This presents an opportunity to systematically interrogate the whole human genome whether a selection signal shared across different populations is the consequence of a single mutation process followed subsequently by gene flow between populations or of convergent evolution due to the occurrence of multiple independent mutation events either at the same variant or within the same gene. The application of our method to data from 14 populations across the world revealed that positive-selection events tend to cluster in populations of the same ancestry. Comparing the founder haplotypes for events that are present across different populations revealed that convergent evolution is a rare occurrence and that the majority of shared signals stem from the same evolutionary event.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23731540">23731540</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.04.021">10.1016/j.ajhg.2013.04.021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenetic conservation at gene regulatory elements revealed by non-methylated DNA profiling in seven vertebrates.</h4>
<p class="authors"><span class="author">Long HK</span>, <span class="author">Sims D</span>, <span class="author">Heger A</span>, <span class="author">Blackledge NP</span>, <span class="author">Kutter C</span>, <span class="author">Wright ML</span>, <span class="author">Gr&uuml;tzner F</span>, <span class="author">Odom DT</span>, <span class="author">Patient R</span>, <span class="author">Ponting CP</span> and <span class="author">Klose RJ</span></p>
<p class="affiliation">Department of Biochemistry , University of Oxford , Oxford , United Kingdom ; Weatherall Institute of Molecular Medicine, University of Oxford , Oxford , United Kingdom.</p>
<p class="abstract">Two-thirds of gene promoters in mammals are associated with regions of non-methylated DNA, called CpG islands (CGIs), which counteract the repressive effects of DNA methylation on chromatin. In cold-blooded vertebrates, computational CGI predictions often reside away from gene promoters, suggesting a major divergence in gene promoter architecture across vertebrates. By experimentally identifying non-methylated DNA in the genomes of seven diverse vertebrates, we instead reveal that non-methylated islands (NMIs) of DNA are a central feature of vertebrate gene promoters. Furthermore, NMIs are present at orthologous genes across vast evolutionary distances, revealing a surprising level of conservation in this epigenetic feature. By profiling NMIs in different tissues and developmental stages we uncover a unifying set of features that are central to the function of NMIs in vertebrates. Together these findings demonstrate an ancient logic for NMI usage at gene promoters and reveal an unprecedented level of epigenetic conservation across vertebrate evolution. DOI:http://dx.doi.org/10.7554/eLife.00348.001.</p>
<p class="publication"><span class="title">eLife</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="pagerange">e00348</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23467541">23467541</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3583005">3583005</a>; DOI: <a href="http://dx.doi.org/10.7554%2FeLife.00348">10.7554/eLife.00348</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human Spermatogenic Failure Purges Deleterious Mutation Load from the Autosomes and Both Sex Chromosomes, including the Gene DMRT1.</h4>
<p class="authors"><span class="author">Lopes AM</span>, <span class="author">Aston KI</span>, <span class="author">Thompson E</span>, <span class="author">Carvalho F</span>, <span class="author">Gon&ccedil;alves J</span>, <span class="author">Huang N</span>, <span class="author">Matthiesen R</span>, <span class="author">Noordam MJ</span>, <span class="author">Quintela I</span>, <span class="author">Ramu A</span>, <span class="author">Seabra C</span>, <span class="author">Wilfert AB</span>, <span class="author">Dai J</span>, <span class="author">Downie JM</span>, <span class="author">Fernandes S</span>, <span class="author">Guo X</span>, <span class="author">Sha J</span>, <span class="author">Amorim A</span>, <span class="author">Barros A</span>, <span class="author">Carracedo A</span>, <span class="author">Hu Z</span>, <span class="author">Hurles ME</span>, <span class="author">Moskovtsev S</span>, <span class="author">Ober C</span>, <span class="author">Paduch DA</span>, <span class="author">Schiffman JD</span>, <span class="author">Schlegel PN</span>, <span class="author">Sousa M</span>, <span class="author">Carrell DT</span> and <span class="author">Conrad DF</span></p>
<p class="affiliation">Institute of Molecular Pathology and Immunology of the University of Porto (IPATIMUP), Porto, Portugal.</p>
<p class="abstract">Gonadal failure, along with early pregnancy loss and perinatal death, may be an important filter that limits the propagation of harmful mutations in the human population. We hypothesized that men with spermatogenic impairment, a disease with unknown genetic architecture and a common cause of male infertility, are enriched for rare deleterious mutations compared to men with normal spermatogenesis. After assaying genomewide SNPs and CNVs in 323 Caucasian men with idiopathic spermatogenic impairment and more than 1,100 controls, we estimate that each rare autosomal deletion detected in our study multiplicatively changes a man's risk of disease by 10% (OR 1.10 [1.04-1.16], p<2Ã—10(-3)), rare X-linked CNVs by 29%, (OR 1.29 [1.11-1.50], p<1Ã—10(-3)), and rare Y-linked duplications by 88% (OR 1.88 [1.13-3.13], p<0.03). By contrasting the properties of our case-specific CNVs with those of CNV callsets from cases of autism, schizophrenia, bipolar disorder, and intellectual disability, we propose that the CNV burden in spermatogenic impairment is distinct from the burden of large, dominant mutations described for neurodevelopmental disorders. We identified two patients with deletions of DMRT1, a gene on chromosome 9p24.3 orthologous to the putative sex determination locus of the avian ZW chromosome system. In an independent sample of Han Chinese men, we identified 3 more DMRT1 deletions in 979 cases of idiopathic azoospermia and none in 1,734 controls, and found none in an additional 4,519 controls from public databases. The combined results indicate that DMRT1 loss-of-function mutations are a risk factor and potential genetic cause of human spermatogenic failure (frequency of 0.38% in 1306 cases and 0% in 7,754 controls, pâ€Š=â€Š6.2Ã—10(-5)). Our study identifies other recurrent CNVs as potential causes of idiopathic azoospermia and generates hypotheses for directing future studies on the genetic basis of male infertility and IVF outcomes.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">3</span>;<span class="pagerange">e1003349</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23555275">23555275</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605256">3605256</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003349">10.1371/journal.pgen.1003349</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study on detailed profiles of smoking behavior and nicotine dependence in a twin sample.</h4>
<p class="authors"><span class="author">Loukola A</span>, <span class="author">Wedenoja J</span>, <span class="author">Keskitalo-Vuokko K</span>, <span class="author">Broms U</span>, <span class="author">Korhonen T</span>, <span class="author">Ripatti S</span>, <span class="author">Sarin AP</span>, <span class="author">Pitk&auml;niemi J</span>, <span class="author">He L</span>, <span class="author">H&auml;pp&ouml;l&auml; A</span>, <span class="author">Heikkil&auml; K</span>, <span class="author">Chou YL</span>, <span class="author">Pergadia ML</span>, <span class="author">Heath AC</span>, <span class="author">Montgomery GW</span>, <span class="author">Martin NG</span>, <span class="author">Madden PA</span> and <span class="author">Kaprio J</span></p>
<p class="affiliation">Department of Public Health, Hjelt Institute, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Smoking is a major risk factor for several somatic diseases and is also emerging as a causal factor for neuropsychiatric disorders. Genome-wide association (GWA) and candidate gene studies for smoking behavior and nicotine dependence (ND) have disclosed too few predisposing variants to account for the high estimated heritability. Previous large-scale GWA studies have had very limited phenotypic definitions of relevance to smoking-related behavior, which has likely impeded the discovery of genetic effects. We performed GWA analyses on 1114 adult twins ascertained for ever smoking from the population-based Finnish Twin Cohort study. The availability of 17 smoking-related phenotypes allowed us to comprehensively portray the dimensions of smoking behavior, clustered into the domains of smoking initiation, amount smoked and ND. Our results highlight a locus on 16p12.3, with several single-nucleotide polymorphisms (SNPs) in the vicinity of CLEC19A showing association (P&lt;1 &times; 10(-6)) with smoking quantity. Interestingly, CLEC19A is located close to a previously reported attention-deficit hyperactivity disorder (ADHD) linkage locus and an evident link between ADHD and smoking has been established. Intriguing preliminary association (P&lt;1 &times; 10(-5)) was detected between DSM-IV (Diagnostic and Statistical Manual of Mental Disorders, 4th edition) ND diagnosis and several SNPs in ERBB4, coding for a Neuregulin receptor, on 2q33. The association between ERBB4 and DSM-IV ND diagnosis was replicated in an independent Australian sample. Recently, a significant increase in ErbB4 and Neuregulin 3 (Nrg3) expression was revealed following chronic nicotine exposure and withdrawal in mice and an association between NRG3 SNPs and smoking cessation success was detected in a clinical trial. ERBB4 has previously been associated with schizophrenia; further, it is located within an established schizophrenia linkage locus and within a linkage locus for a smoker phenotype identified in this sample. In conclusion, we disclose novel tentative evidence for the involvement of ERBB4 in ND, suggesting the involvement of the Neuregulin/ErbB signalling pathway in addictions and providing a plausible link between the high co-morbidity of schizophrenia and ND.Molecular Psychiatry advance online publication, 11 June 2013; doi:10.1038/mp.2013.72.</p>
<p class="grants">Funded by: NIAAA NIH HHS: <span class="grant">K05 AA017688</span>, <span class="grant">P60 AA011998</span>, <span class="grant">R01 AA013320</span>; NIDA NIH HHS: <span class="grant">R01 DA012854</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23752247">23752247</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2013.72">10.1038/mp.2013.72</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of a Tn5 transposon library in Haemophilus parasuis and analysis by transposon-directed insertion-site sequencing (TraDIS).</h4>
<p class="authors"><span class="author">Luan SL</span>, <span class="author">Chaudhuri RR</span>, <span class="author">Peters SE</span>, <span class="author">Mayho M</span>, <span class="author">Weinert LA</span>, <span class="author">Crowther SA</span>, <span class="author">Wang J</span>, <span class="author">Langford PR</span>, <span class="author">Rycroft A</span>, <span class="author">Wren BW</span>, <span class="author">Tucker AW</span>, <span class="author">Maskell DJ</span> and <span class="author">BRaDP1T Consortium</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Madingley Road, Cambridge CB3 0ES, UK. <a href="mailto:%73%6C%34%37%30%40%63%61%6D.%61%63.%75%6B">&#115;&#108;&#52;&#55;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Haemophilus parasuis is an important respiratory tract pathogen of swine and the etiological agent of Gl&auml;sser&#39;s disease. The molecular pathogenesis of H. parasuis is not well studied, mainly due to the lack of efficient tools for genetic manipulation of this bacterium. In this study we describe a Tn5-based random mutagenesis method for use in H. parasuis. A novel chloramphenicol-resistant Tn5 transposome was electroporated into the virulent H. parasuis serovar 5 strain 29755. High transposition efficiency of Tn5, up to 10(4) transformants/&mu;g of transposon DNA, was obtained by modification of the Tn5 DNA in the H. parasuis strain HS071 and establishment of optimal electrotransformation conditions, and a library of approximately 10,500 mutants was constructed. Analysis of the library using transposon-directed insertion-site sequencing (TraDIS) revealed that the insertion of Tn5 was evenly distributed throughout the genome. 10,001 individual mutants were identified, with 1561 genes being disrupted (69.4% of the genome). This newly-developed, efficient mutagenesis approach will be a powerful tool for genetic manipulation of H. parasuis in order to study its physiology and pathogenesis.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/G003203/1</span>, <span class="grant">BB/G018553/1</span>, <span class="grant">BB/G019177/1</span>, <span class="grant">BB/G019274/1</span>, <span class="grant">BB/G020744/1</span></p>
<p class="publication"><span class="title">Veterinary microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">166</span>;<span class="number">3-4</span>;<span class="pagerange">558-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23928120">23928120</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vetmic.2013.07.008">10.1016/j.vetmic.2013.07.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Distribution and 'In Vivo' Phase Variation Status of Haemoglobin Receptors in Invasive Meningococcal Serogroup B Disease: Genotypic and Phenotypic Analysis.</h4>
<p class="authors"><span class="author">Lucidarme J</span>, <span class="author">Findlow J</span>, <span class="author">Chan H</span>, <span class="author">Feavers IM</span>, <span class="author">Gray SJ</span>, <span class="author">Kaczmarski EB</span>, <span class="author">Parkhill J</span>, <span class="author">Bai X</span>, <span class="author">Borrow R</span> and <span class="author">Bayliss CD</span></p>
<p class="affiliation">Public Health England, Manchester, United Kingdom.</p>
<p class="abstract">Two haemoglobin-binding proteins, HmbR and HpuAB, contribute to iron acquisition by Neisseria meningitidis. These receptors are subject to high frequency, reversible switches in gene expression - phase variation (PV) - due to mutations in homopolymeric (poly-G) repeats present in the open reading frame. The distribution and PV state of these receptors was assessed for a representative collection of isolates from invasive meningococcal disease patients of England, Wales and Northern Ireland. Most of the major clonal complexes had only the HmbR receptor whilst the recently expanding ST-275-centred cluster of the ST-269 clonal complex had both receptors. At least one of the receptors was in an 'ON' configuration in 76.3% of the isolates, a finding that was largely consistent with phenotypic analyses. As PV status may change during isolation and culture of meningococci, a PCR-based protocol was utilised to confirm the expression status of the receptors within contemporaneously acquired clinical specimens (blood/cerebrospinal fluid) from the respective patients. The expression state was confirmed for all isolate/specimen pairs with <15 tract repeats indicating that the PV status of these receptors is stable during isolation. This study therefore establishes a protocol for determining in vivo PV status to aid in determining the contributions of phase variable genes to invasive meningococcal disease. Furthermore, the results of the study support a putative but non-essential role of the meningococcal haemoglobin receptors as virulence factors whilst further highlighting their vaccine candidacy.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">9</span>;<span class="pagerange">e76932</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24098814">24098814</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3786947">3786947</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0076932">10.1371/journal.pone.0076932</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RocA Truncation Underpins Hyper-Encapsulation, Carriage Longevity and Transmissibility of Serotype M18 Group A Streptococci.</h4>
<p class="authors"><span class="author">Lynskey NN</span>, <span class="author">Goulding D</span>, <span class="author">Gierula M</span>, <span class="author">Turner CE</span>, <span class="author">Dougan G</span>, <span class="author">Edwards RJ</span> and <span class="author">Sriskandan S</span></p>
<p class="affiliation">Faculty of Medicine, Imperial College London, Hammersmith Hospital, London, United Kingdom.</p>
<p class="abstract">Group A streptococcal isolates of serotype M18 are historically associated with epidemic waves of pharyngitis and the non-suppurative immune sequela rheumatic fever. The serotype is defined by a unique, highly encapsulated phenotype, yet the molecular basis for this unusual colony morphology is unknown. Here we identify a truncation in the regulatory protein RocA, unique to and conserved within our serotype M18 GAS collection, and demonstrate that it underlies the characteristic M18 capsule phenotype. Reciprocal allelic exchange mutagenesis of rocA between M18 GAS and M89 GAS demonstrated that truncation of RocA was both necessary and sufficient for hyper-encapsulation via up-regulation of both precursors required for hyaluronic acid synthesis. Although RocA was shown to positively enhance covR transcription, quantitative proteomics revealed RocA to be a metabolic regulator with activity beyond the CovR/S regulon. M18 GAS demonstrated a uniquely protuberant chain formation following culture on agar that was dependent on excess capsule and the RocA mutation. Correction of the M18 rocA mutation reduced GAS survival in human blood, and in vivo naso-pharyngeal carriage longevity in a murine model, with an associated drop in bacterial airborne transmission during infection. In summary, a naturally occurring truncation in a regulator explains the encapsulation phenotype, carriage longevity and transmissibility of M18 GAS, highlighting the close interrelation of metabolism, capsule and virulence.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">e1003842</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24367267">24367267</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3868526">3868526</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003842">10.1371/journal.ppat.1003842</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation sequencing of disseminated tumor cells.</h4>
<p class="authors"><span class="author">M&oslash;ller EK</span>, <span class="author">Kumar P</span>, <span class="author">Voet T</span>, <span class="author">Peterson A</span>, <span class="author">Van Loo P</span>, <span class="author">Mathiesen RR</span>, <span class="author">Fjelldal R</span>, <span class="author">Grundstad J</span>, <span class="author">Borgen E</span>, <span class="author">Baumbusch LO</span>, <span class="author">Naume B</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">White KP</span>, <span class="author">Nord S</span> and <span class="author">Kristensen VN</span></p>
<p class="affiliation">Department of Genetics, Institute for Cancer Research, Oslo University Hospital Radiumhospitalet , Oslo , Norway ; K.G. Jebsen Center for Breast Cancer Research, Institute for Clinical Medicine, Faculty of Medicine, University of Oslo , Oslo , Norway.</p>
<p class="abstract">Disseminated tumor cells (DTCs) detected in the bone marrow have been shown as an independent prognostic factor for women with breast cancer. However, the mechanisms behind the tumor cell dissemination are still unclear and more detailed knowledge is needed to fully understand why some cells remain dormant and others metastasize. Sequencing of single cells has opened for the possibility to dissect the genetic content of subclones of a primary tumor, as well as DTCs. Previous studies of genetic changes in DTCs have employed single-cell array comparative genomic hybridization which provides information about larger aberrations. To date, next-generation sequencing provides the possibility to discover new, smaller, and copy neutral genetic changes. In this study, we performed whole-genome amplification and subsequently next-generation sequencing to analyze DTCs from two breast cancer patients. We compared copy-number profiles of the DTCs and the corresponding primary tumor generated from sequencing and SNP-comparative genomic hybridization (CGH) data, respectively. While one tumor revealed mostly whole-arm gains and losses, the other had more complex alterations, as well as subclonal amplification and deletions. Whole-arm gains or losses in the primary tumor were in general also observed in the corresponding DTC. Both primary tumors showed amplification of chromosome 1q and deletion of parts of chromosome 16q, which was recaptured in the corresponding DTCs. Interestingly, clear differences were also observed, indicating that the DTC underwent further evolution at the copy-number level. This study provides a proof-of-principle for sequencing of DTCs and correlation with primary copy-number profiles. The analyses allow insight into tumor cell dissemination and show ongoing copy-number evolution in DTCs compared to the primary tumors.</p>
<p class="publication"><span class="title">Frontiers in oncology</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="pagerange">320</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24427740">24427740</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3876274">3876274</a>; DOI: <a href="http://dx.doi.org/10.3389%2Ffonc.2013.00320">10.3389/fonc.2013.00320</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study in a Chinese population identifies a susceptibility locus for type 2 diabetes at 7q32 near PAX4.</h4>
<p class="authors"><span class="author">Ma RC</span>, <span class="author">Hu C</span>, <span class="author">Tam CH</span>, <span class="author">Zhang R</span>, <span class="author">Kwan P</span>, <span class="author">Leung TF</span>, <span class="author">Thomas GN</span>, <span class="author">Go MJ</span>, <span class="author">Hara K</span>, <span class="author">Sim X</span>, <span class="author">Ho JS</span>, <span class="author">Wang C</span>, <span class="author">Li H</span>, <span class="author">Lu L</span>, <span class="author">Wang Y</span>, <span class="author">Li JW</span>, <span class="author">Wang Y</span>, <span class="author">Lam VK</span>, <span class="author">Wang J</span>, <span class="author">Yu W</span>, <span class="author">Kim YJ</span>, <span class="author">Ng DP</span>, <span class="author">Fujita H</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Day-Williams AG</span>, <span class="author">Lee HM</span>, <span class="author">Ng AC</span>, <span class="author">Fang YJ</span>, <span class="author">Kong AP</span>, <span class="author">Jiang F</span>, <span class="author">Ma X</span>, <span class="author">Hou X</span>, <span class="author">Tang S</span>, <span class="author">Lu J</span>, <span class="author">Yamauchi T</span>, <span class="author">Tsui SK</span>, <span class="author">Woo J</span>, <span class="author">Leung PC</span>, <span class="author">Zhang X</span>, <span class="author">Tang NL</span>, <span class="author">Sy HY</span>, <span class="author">Liu J</span>, <span class="author">Wong TY</span>, <span class="author">Lee JY</span>, <span class="author">Maeda S</span>, <span class="author">Xu G</span>, <span class="author">Cherny SS</span>, <span class="author">Chan TF</span>, <span class="author">Ng MC</span>, <span class="author">Xiang K</span>, <span class="author">Morris AP</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Keildson S</span>, <span class="author">The MuTHER Consortium</span>, <span class="author">Hu R</span>, <span class="author">Ji L</span>, <span class="author">Lin X</span>, <span class="author">Cho YS</span>, <span class="author">Kadowaki T</span>, <span class="author">Tai ES</span>, <span class="author">Zeggini E</span>, <span class="author">McCarthy MI</span>, <span class="author">Hon KL</span>, <span class="author">Baum L</span>, <span class="author">Tomlinson B</span>, <span class="author">So WY</span>, <span class="author">Bao Y</span>, <span class="author">Chan JC</span> and <span class="author">Jia W</span></p>
<p class="affiliation">Department of Medicine and Therapeutics, Chinese University of Hong Kong, Prince of Wales Hospital, Shatin, New Territories, Hong Kong, SAR, People's Republic of China, <a href="mailto:%72%63%77%6D%61%40%63%75%68%6B.%65%64%75.%68%6B">&#114;&#99;&#119;&#109;&#97;&#64;&#99;&#117;&#104;&#107;&#46;&#101;&#100;&#117;&#46;&#104;&#107;</a>.</p>
<p class="abstract">AIMS/HYPOTHESIS: Most genetic variants identified for type 2 diabetes have been discovered in European populations. We performed genome-wide association studies (GWAS) in a Chinese population with the aim of identifying novel variants for type 2 diabetes in Asians. METHODS: We performed a meta-analysis of three GWAS comprising 684 patients with type 2 diabetes and 955 controls of Southern Han Chinese descent. We followed up the top signals in two independent Southern Han Chinese cohorts (totalling 10,383 cases and 6,974 controls), and performed in silico replication in multiple populations. RESULTS: We identified CDKN2A/B and four novel type 2 diabetes association signals with pâ€‰<â€‰1â€‰Ã—â€‰10(-5) from the meta-analysis. Thirteen variants within these four loci were followed up in two independent Chinese cohorts, and rs10229583 at 7q32 was found to be associated with type 2 diabetes in a combined analysis of 11,067 cases and 7,929 controls (p metaâ€‰=â€‰2.6â€‰Ã—â€‰10(-8); OR [95% CI] 1.18 [1.11, 1.25]). In silico replication revealed consistent associations across multiethnic groups, including five East Asian populations (p metaâ€‰=â€‰2.3â€‰Ã—â€‰10(-10)) and a population of European descent (pâ€‰=â€‰8.6â€‰Ã—â€‰10(-3)). The rs10229583 risk variant was associated with elevated fasting plasma glucose, impaired beta cell function in controls, and an earlier age at diagnosis for the cases. The novel variant lies within an islet-selective cluster of open regulatory elements. There was significant heterogeneity of effect between Han Chinese and individuals of European descent, Malaysians and Indians. CONCLUSIONS/INTERPRETATION: Our study identifies rs10229583 near PAX4 as a novel locus for type 2 diabetes in Chinese and other populations and provides new insights into the pathogenesis of type 2 diabetes.</p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23532257">23532257</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-013-2874-4">10.1007/s00125-013-2874-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human candidate polymorphisms in sympatric ethnic groups differing in malaria susceptibility in mali.</h4>
<p class="authors"><span class="author">Maiga B</span>, <span class="author">Dolo A</span>, <span class="author">Tour&eacute; O</span>, <span class="author">Dara V</span>, <span class="author">Tapily A</span>, <span class="author">Campino S</span>, <span class="author">Sepulveda N</span>, <span class="author">Risley P</span>, <span class="author">Silva N</span>, <span class="author">Corran P</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski D</span>, <span class="author">MalariaGEN Consortium</span>, <span class="author">Clark TG</span>, <span class="author">Troye-Blomberg M</span> and <span class="author">Doumbo OK</span></p>
<p class="affiliation">Malaria Research and Training Center / Department of Epidemiology of Parasitic Diseases / Faculty of Medicine, Pharmacy and Odonto - Stomatology, BP 1805, Bamako, USTTB, Mali ; Department of Molecular Biosciences, The Wenner-Gren Institute, Stockholm University, Stockholm, Sweden.</p>
<p class="abstract">Malaria still remains a major public health problem in Mali, although disease susceptibility varies between ethnic groups, particularly between the Fulani and Dogon. These two sympatric groups share similar socio-cultural factors and malaria transmission rates, but Fulani individuals tend to show significantly higher spleen enlargement scores, lower parasite prevalence, and seem less affected by the disease than their Dogon neighbours. We have used genetic polymorphisms from malaria-associated genes to investigate associations with various malaria metrics between the Fulanai and Dogon groups. Two cross sectional surveys (transmission season 2006, dry season 2007) were performed. Healthy volunteers from the both ethnic groups (n=939) were recruited in a rural setting. In each survey, clinical (spleen enlargement, axillary temperature, weight) and parasitological data (malaria parasite densities and species) were collected, as well as blood samples. One hundred and sixty six SNPs were genotyped and 5 immunoassays (AMA1, CSP, MSP1, MSP2, total IgE) were performed on the DNA and serum samples respectively. The data confirm the reduced malaria susceptibility in the Fulani, with a higher level of the protective O-blood group, and increased circulating antibody levels to several malaria antigens (p<10(-15)). We identified SNP allele frequency differences between the 2 ethnic groups in CD36, IL4, RTN3 and ADCY9. Moreover, polymorphisms in FCER1A, RAD50, TNF, SLC22A4, and IL13 genes were correlated with antibody production (p-value<0.003). Further work is required to understand the mechanisms underpinning these genetic factors.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e75675</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24098393">24098393</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3788813">3788813</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0075675">10.1371/journal.pone.0075675</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A follow-up linkage study of Finnish pre-eclampsia families identifies a new fetal susceptibility locus on chromosome 18.</h4>
<p class="authors"><span class="author">Majander KK</span>, <span class="author">Villa PM</span>, <span class="author">Kivinen K</span>, <span class="author">Kere J</span> and <span class="author">Laivuori H</span></p>
<p class="affiliation">1] Department of Medical Genetics, Haartman Institute, University of Helsinki, Helsinki, Finland [2] Research Programs Unit, Women's Health, University of Helsinki, Helsinki, Finland.</p>
<p class="abstract">Pre-eclampsia is a common vascular disorder of pregnancy. It originates in the placenta and targets the maternal endothelium. According to epidemiological research, >50% of the liability to this disorder can be accounted for by genetic factors. Both maternal and fetal genes contribute to the risk, but especially the fetal genetic risk profile is still poorly understood. We have previously detected linkage signals in multiplex Finnish families on chromosomes 2p25, 4q32, and 9p13 using maternal phenotypes. We performed a linkage analysis using updated maternal phenotypes and an unprecedented linkage analysis using fetal phenotypes. Markers genotyped were available from 237 individuals in 15 Finnish families, including 72 affected mothers and 49 affected fetuses. The MERLIN software was used for sample and marker quality control and linkage analysis. The results were compared against the original ones obtained by using the GENEHUNTER 2.1 software. The previous identification of the maternal susceptibility locus to a genetic location at 21.70â€‰cM near marker D2S168 on chromosome 2 was confirmed by using both maternal and fetal phenotypes (maternal non-parametric linkage (NPL) score 3.79, P=0.00008, LOD (logarithm (base 10) of odds)=2.20 and fetal NPL score 2.95, P=0.002, LOD=1.71). As a novel finding, we present a suggestive linkage to chromosome 18 at 86.80â€‰cM near marker D18S64 (NPL score 2.51, P=0.006, LOD=1.20) using the fetal phenotype. We propose that chromosome 18 may harbor a new fetal susceptibility locus for pre-eclampsia.European Journal of Human Genetics advance online publication, 6 February 2013; doi:10.1038/ejhg.2013.6.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23386034">23386034</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2013.6">10.1038/ejhg.2013.6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The agr locus regulates virulence and colonization genes in Clostridium difficile 027.</h4>
<p class="authors"><span class="author">Martin MJ</span>, <span class="author">Clare S</span>, <span class="author">Goulding D</span>, <span class="author">Faulds-Pain A</span>, <span class="author">Barquist L</span>, <span class="author">Browne HP</span>, <span class="author">Pettit L</span>, <span class="author">Dougan G</span>, <span class="author">Lawley TD</span> and <span class="author">Wren BW</span></p>
<p class="affiliation">Department of Pathogen Molecular Biology, London School of Hygiene and Tropical Medicine, University of London, London, United Kingdom.</p><p class="abstract">The transcriptional regulator AgrA, a member of the LytTR family of proteins, plays a key role in controlling gene expression in some Gram-positive pathogens, including Staphylococcus aureus and Enterococcus faecalis. AgrA is encoded by the agrACDB global regulatory locus, and orthologues are found within the genome of most Clostridium difficile isolates, including the epidemic lineage 027/BI/NAP1. Comparative RNA sequencing of the wild type and otherwise isogenic agrA null mutant derivatives of C. difficile R20291 revealed a network of approximately 75 differentially regulated transcripts at late exponential growth phase, including many genes associated with flagellar assembly and function, such as the major structural subunit, FliC. Other differentially regulated genes include several involved in bis-(3&#39;-5&#39;)-cyclic dimeric GMP (c-di-GMP) synthesis and toxin A expression. C. difficile 027 R20291 agrA mutant derivatives were poorly flagellated and exhibited reduced levels of colonization and relapses in the murine infection model. Thus, the agr locus likely plays a contributory role in the fitness and virulence potential of C. difficile strains in the 027/BI/NAP1 lineage.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">93614</span>, <span class="grant">G1000214</span>; Wellcome Trust: <span class="grant">086418</span>, <span class="grant">086418/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2013</span>;<span class="volume">195</span>;<span class="number">16</span>;<span class="pagerange">3672-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23772065">23772065</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3754575">3754575</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00473-13">10.1128/JB.00473-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Playing fast and loose with mutation.</h4>
<p class="authors"><span class="author">Mather AE</span> and <span class="author">Harris SR</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">12</span>;<span class="pagerange">822</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24232567">24232567</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3164">10.1038/nrmicro3164</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinguishable epidemics of multidrug-resistant Salmonella Typhimurium DT104 in different hosts.</h4>
<p class="authors"><span class="author">Mather AE</span>, <span class="author">Reid SW</span>, <span class="author">Maskell DJ</span>, <span class="author">Parkhill J</span>, <span class="author">Fookes MC</span>, <span class="author">Harris SR</span>, <span class="author">Brown DJ</span>, <span class="author">Coia JE</span>, <span class="author">Mulvey MR</span>, <span class="author">Gilmour MW</span>, <span class="author">Petrovska L</span>, <span class="author">de Pinna E</span>, <span class="author">Kuroda M</span>, <span class="author">Akiba M</span>, <span class="author">Izumiya H</span>, <span class="author">Connor TR</span>, <span class="author">Suchard MA</span>, <span class="author">Lemey P</span>, <span class="author">Mellor DJ</span>, <span class="author">Haydon DT</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">The global epidemic of multidrug-resistant Salmonella Typhimurium DT104 provides an important example, both in terms of the agent and its resistance, of a widely disseminated zoonotic pathogen. Here, with an unprecedented national collection of isolates collected contemporaneously from humans and animals and including a sample of internationally derived isolates, we have used whole-genome sequencing to dissect the phylogenetic associations of the bacterium and its antimicrobial resistance genes through the course of an epidemic. Contrary to current tenets supporting a single homogeneous epidemic, we demonstrate that the bacterium and its resistance genes were largely maintained within animal and human populations separately and that there was limited transmission, in either direction. We also show considerable variation in the resistance profiles, in contrast to the largely stable bacterial core genome, which emphasizes the critical importance of integrated genotypic data sets in understanding the ecology of bacterial zoonoses and antimicrobial resistance.</p>
<p class="grants">Funded by: European Research Council: <span class="grant">260864</span>; NHGRI NIH HHS: <span class="grant">HG006139</span>, <span class="grant">R01 HG006139</span>; NIAID NIH HHS: <span class="grant">AI107034</span>, <span class="grant">R01 AI107034</span>; NIGMS NIH HHS: <span class="grant">R01 GM086887</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">341</span>;<span class="number">6153</span>;<span class="pagerange">1514-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24030491">24030491</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4012302">4012302</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1240578">10.1126/science.1240578</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mosaic copy number variation in human neurons.</h4>
<p class="authors"><span class="author">McConnell MJ</span>, <span class="author">Lindberg MR</span>, <span class="author">Brennand KJ</span>, <span class="author">Piper JC</span>, <span class="author">Voet T</span>, <span class="author">Cowing-Zitron C</span>, <span class="author">Shumilina S</span>, <span class="author">Lasken RS</span>, <span class="author">Vermeesch JR</span>, <span class="author">Hall IM</span> and <span class="author">Gage FH</span></p>
<p class="affiliation">Laboratory of Genetics, Salk Institute for Biological Studies, La Jolla, CA 92037, USA.</p>
<p class="abstract">We used single-cell genomic approaches to map DNA copy number variation (CNV) in neurons obtained from human induced pluripotent stem cell (hiPSC) lines and postmortem human brains. We identified aneuploid neurons, as well as numerous subchromosomal CNVs in euploid neurons. Neurotypic hiPSC-derived neurons had larger CNVs than fibroblasts, and several large deletions were found in hiPSC-derived neurons but not in matched neural progenitor cells. Single-cell sequencing of endogenous human frontal cortex neurons revealed that 13 to 41% of neurons have at least one megabase-scale de novo CNV, that deletions are twice as common as duplications, and that a subset of neurons have highly aberrant genomes marked by multiple alterations. Our results show that mosaic CNV is abundant in human neurons.</p>
<p class="grants">Funded by: NCCDPHP CDC HHS: <span class="grant">DP20D006493-01</span>; NICHD NIH HHS: <span class="grant">N01-HD-9-011</span>; NIMH NIH HHS: <span class="grant">R01 MH095741</span>; PHS HHS: <span class="grant">HHSN2752009000011C</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">342</span>;<span class="number">6158</span>;<span class="pagerange">632-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24179226">24179226</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1243472">10.1126/science.1243472</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Retrospective analysis of whole genome sequencing compared to prospective typing data in further informing the epidemiological investigation of an outbreak of Shigella sonnei in the UK.</h4>
<p class="authors"><span class="author">McDonnell J</span>, <span class="author">Dallman T</span>, <span class="author">Atkin S</span>, <span class="author">Turbitt DA</span>, <span class="author">Connor TR</span>, <span class="author">Grant KA</span>, <span class="author">Thomson NR</span> and <span class="author">Jenkins C</span></p>
<p class="affiliation">North East and North Central London Health Protection Unit, Health Protection Agency, London, UK.</p><p class="abstract">The aim of this study was to retrospectively assess the value of whole genome sequencing (WGS) compared to conventional typing methods in the investigation and control of an outbreak of Shigella sonnei in the Orthodox Jewish (OJ) community in the UK. The genome sequence analysis showed that the strains implicated in the outbreak formed three phylogenetically distinct clusters. One cluster represented cases associated with recent exposure to a single strain, whereas the other two clusters represented related but distinct strains of S. sonnei circulating in the OJ community across the UK. The WGS data challenged the conclusions drawn during the initial outbreak investigation and allowed cases of dysentery to be implicated or ruled out of the outbreak that were previously misclassified. This study showed that the resolution achieved using WGS would have clearly defined the outbreak, thus facilitating the promotion of infection control measures within local schools and the dissemination of a stronger public health message to the community.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Epidemiology and infection</span>&nbsp;<span class="year">2013</span>;<span class="volume">141</span>;<span class="number">12</span>;<span class="pagerange">2568-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23425681">23425681</a>; DOI: <a href="http://dx.doi.org/10.1017%2FS0950268813000137">10.1017/S0950268813000137</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association Study of Common Genetic Variants and HIV-1 Acquisition in 6,300 Infected Cases and 7,200 Controls.</h4>
<p class="authors"><span class="author">McLaren PJ</span>, <span class="author">Coulonges C</span>, <span class="author">Ripke S</span>, <span class="author">van den Berg L</span>, <span class="author">Buchbinder S</span>, <span class="author">Carrington M</span>, <span class="author">Cossarizza A</span>, <span class="author">Dalmau J</span>, <span class="author">Deeks SG</span>, <span class="author">Delaneau O</span>, <span class="author">De Luca A</span>, <span class="author">Goedert JJ</span>, <span class="author">Haas D</span>, <span class="author">Herbeck JT</span>, <span class="author">Kathiresan S</span>, <span class="author">Kirk GD</span>, <span class="author">Lambotte O</span>, <span class="author">Luo M</span>, <span class="author">Mallal S</span>, <span class="author">van Manen D</span>, <span class="author">Martinez-Picado J</span>, <span class="author">Meyer L</span>, <span class="author">Miro JM</span>, <span class="author">Mullins JI</span>, <span class="author">Obel N</span>, <span class="author">O&#39;Brien SJ</span>, <span class="author">Pereyra F</span>, <span class="author">Plummer FA</span>, <span class="author">Poli G</span>, <span class="author">Qi Y</span>, <span class="author">Rucart P</span>, <span class="author">Sandhu MS</span>, <span class="author">Shea PR</span>, <span class="author">Schuitemaker H</span>, <span class="author">Theodorou I</span>, <span class="author">Vannberg F</span>, <span class="author">Veldink J</span>, <span class="author">Walker BD</span>, <span class="author">Weintrob A</span>, <span class="author">Winkler CA</span>, <span class="author">Wolinsky S</span>, <span class="author">Telenti A</span>, <span class="author">Goldstein DB</span>, <span class="author">de Bakker PI</span>, <span class="author">Zagury JF</span> and <span class="author">Fellay J</span></p>
<p class="affiliation">School of Life Sciences, &Eacute;cole Polytechnique F&eacute;d&eacute;rale de Lausanne, Lausanne, Switzerland ; Institute of Microbiology, University Hospital Center and University of Lausanne, Lausanne, Switzerland ; Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts, United States of America.</p><p class="abstract">Multiple genome-wide association studies (GWAS) have been performed in HIV-1 infected individuals, identifying common genetic influences on viral control and disease course. Similarly, common genetic correlates of acquisition of HIV-1 after exposure have been interrogated using GWAS, although in generally small samples. Under the auspices of the International Collaboration for the Genomics of HIV, we have combined the genome-wide single nucleotide polymorphism (SNP) data collected by 25 cohorts, studies, or institutions on HIV-1 infected individuals and compared them to carefully matched population-level data sets (a list of all collaborators appears in Note S1 in Text S1). After imputation using the 1,000 Genomes Project reference panel, we tested approximately 8 million common DNA variants (SNPs and indels) for association with HIV-1 acquisition in 6,334 infected patients and 7,247 population samples of European ancestry. Initial association testing identified the SNP rs4418214, the C allele of which is known to tag the HLA-B*57:01 and B*27:05 alleles, as genome-wide significant (p&#x200A;=&#x200A;3.6&times;10(-11)). However, restricting analysis to individuals with a known date of seroconversion suggested that this association was due to the frailty bias in studies of lethal diseases. Further analyses including testing recessive genetic models, testing for bulk effects of non-genome-wide significant variants, stratifying by sexual or parenteral transmission risk and testing previously reported associations showed no evidence for genetic influence on HIV-1 acquisition (with the exception of CCR5&Delta;32 homozygosity). Thus, these data suggest that genetic influences on HIV acquisition are either rare or have smaller effects than can be detected by this sample size.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">7</span>;<span class="pagerange">e1003515</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23935489">23935489</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3723635">3723635</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003515">10.1371/journal.ppat.1003515</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Evolutionary Path to Extraintestinal Pathogenic, Drug-Resistant Escherichia coli Is Marked by Drastic Reduction in Detectable Recombination within the Core Genome.</h4>
<p class="authors"><span class="author">McNally A</span>, <span class="author">Cheng L</span>, <span class="author">Harris SR</span> and <span class="author">Corander J</span></p>
<p class="affiliation">Pathogen Research Group, Nottingham Trent University, United Kingdom.</p>
<p class="abstract">Escherichia coli is a highly diverse group of pathogens ranging from commensal of the intestinal tract, through to intestinal pathogen, and extraintestinal pathogen. Here, we present data on the population diversity of E. coli, using Bayesian analysis to identify 13 distinct clusters within the population from multilocus sequence typing data, which map onto a whole-genome-derived phylogeny based on 62 genome sequences. Bayesian analysis of recombination within the core genome identified reduction in detectable core genome recombination as one moves from the commensals, through the intestinal pathogens down to the multidrug-resistant extraintestinal pathogenic clone E. coli ST131. Our data show that the emergence of a multidrug-resistant, extraintestinal pathogenic lineage of E. coli is marked by substantial reduction in detectable core genome recombination, resulting in a lineage which is phylogenetically distinct and sexually isolated in terms of core genome recombination.</p>
<p class="publication"><span class="title">Genome biology and evolution</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">699-710</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23493634">23493634</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3641635">3641635</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fgbe%2Fevt038">10.1093/gbe/evt038</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Selecting antagonistic antibodies that control differentiation through inducible expression in embryonic stem cells.</h4>
<p class="authors"><span class="author">Melidoni AN</span>, <span class="author">Dyson MR</span>, <span class="author">Wormald S</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, Cambridge CB2 1QW, United Kingdom.</p>
<p class="abstract">Antibodies that modulate receptor function have great untapped potential in the control of stem cell differentiation. In contrast to many natural ligands, antibodies are stable, exquisitely specific, and are unaffected by the regulatory mechanisms that act on natural ligands. Here we describe an innovative system for identifying such antibodies by introducing and expressing antibody gene populations in ES cells. Following induced antibody expression and secretion, changes in differentiation outcomes of individual antibody-expressing ES clones are monitored using lineage-specific gene expression to identify clones that encode and express signal-modifying antibodies. This in-cell expression and reporting system was exemplified by generating blocking antibodies to FGF4 and its receptor FGFR1β, identified through delayed onset of ES cell differentiation. Functionality of the selected antibodies was confirmed by addition of exogenous antibodies to three different ES reporter cell lines, where retained expression of pluripotency markers Oct4, Nanog, and Rex1 was observed. This work demonstrates the potential for discovery and utility of functional antibodies in stem cell differentiation. This work is also unique in constituting an example of ES cells carrying an inducible antibody that causes a functional protein "knock-down" and allows temporal control of stable signaling components at the protein level.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">44</span>;<span class="pagerange">17802-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24082130">24082130</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3816485">3816485</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1312062110">10.1073/pnas.1312062110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Machine learning prediction of cancer cell sensitivity to drugs based on genomic and chemical properties.</h4>
<p class="authors"><span class="author">Menden MP</span>, <span class="author">Iorio F</span>, <span class="author">Garnett M</span>, <span class="author">McDermott U</span>, <span class="author">Benes CH</span>, <span class="author">Ballester PJ</span> and <span class="author">Saez-Rodriguez J</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus-Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Predicting the response of a specific cancer to a therapy is a major goal in modern oncology that should ultimately lead to a personalised treatment. High-throughput screenings of potentially active compounds against a panel of genomically heterogeneous cancer cell lines have unveiled multiple relationships between genomic alterations and drug responses. Various computational approaches have been proposed to predict sensitivity based on genomic features, while others have used the chemical properties of the drugs to ascertain their effect. In an effort to integrate these complementary approaches, we developed machine learning models to predict the response of cancer cell lines to drug treatment, quantified through IC&#x2085;&#x2080; values, based on both the genomic features of the cell lines and the chemical properties of the considered drugs. Models predicted IC&#x2085;&#x2080; values in a 8-fold cross-validation and an independent blind test with coefficient of determination R&sup2; of 0.72 and 0.64 respectively. Furthermore, models were able to predict with comparable accuracy (R&sup2; of 0.61) IC50s of cell lines from a tissue not used in the training stage. Our in silico models can be used to optimise the experimental design of drug-cell screenings by estimating a large proportion of missing IC&#x2085;&#x2080; values rather than experimentally measuring them. The implications of our results go beyond virtual drug screening design: potentially thousands of drugs could be probed in silico to systematically test their potential efficacy as anti-tumour agents based on their structure, thus providing a computational framework to identify new drug repositioning opportunities as well as ultimately be useful for personalized medicine by linking the genomic traits of patients to drug sensitivity.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0902106</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e61318</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23646105">23646105</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3640019">3640019</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0061318">10.1371/journal.pone.0061318</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biomarkers for type 2 diabetes and impaired fasting glucose using a nontargeted metabolomics approach.</h4>
<p class="authors"><span class="author">Menni C</span>, <span class="author">Fauman E</span>, <span class="author">Erte I</span>, <span class="author">Perry JR</span>, <span class="author">Kastenm&uuml;ller G</span>, <span class="author">Shin SY</span>, <span class="author">Petersen AK</span>, <span class="author">Hyde C</span>, <span class="author">Psatha M</span>, <span class="author">Ward KJ</span>, <span class="author">Yuan W</span>, <span class="author">Milburn M</span>, <span class="author">Palmer CN</span>, <span class="author">Frayling TM</span>, <span class="author">Trimmer J</span>, <span class="author">Bell JT</span>, <span class="author">Gieger C</span>, <span class="author">Mohney RP</span>, <span class="author">Brosnan MJ</span>, <span class="author">Suhre K</span>, <span class="author">Soranzo N</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King&#39;s College London, London, U.K.</p><p class="abstract">Using a nontargeted metabolomics approach of 447 fasting plasma metabolites, we searched for novel molecular markers that arise before and after hyperglycemia in a large population-based cohort of 2,204 females (115 type 2 diabetic [T2D] case subjects, 192 individuals with impaired fasting glucose [IFG], and 1,897 control subjects) from TwinsUK. Forty-two metabolites from three major fuel sources (carbohydrates, lipids, and proteins) were found to significantly correlate with T2D after adjusting for multiple testing; of these, 22 were previously reported as associated with T2D or insulin resistance. Fourteen metabolites were found to be associated with IFG. Among the metabolites identified, the branched-chain keto-acid metabolite 3-methyl-2-oxovalerate was the strongest predictive biomarker for IFG after glucose (odds ratio [OR] 1.65 [95% CI 1.39-1.95], P = 8.46 &times; 10(-9)) and was moderately heritable (h(2) = 0.20). The association was replicated in an independent population (n = 720, OR 1.68 [ 1.34-2.11], P = 6.52 &times; 10(-6)) and validated in 189 twins with urine metabolomics taken at the same time as plasma (OR 1.87 [1.27-2.75], P = 1 &times; 10(-3)). Results confirm an important role for catabolism of branched-chain amino acids in T2D and IFG. In conclusion, this T2D-IFG biomarker study has surveyed the broadest panel of nontargeted metabolites to date, revealing both novel and known associated metabolites and providing potential novel targets for clinical prediction and a deeper understanding of causal mechanisms.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">092447/Z/10/Z</span>, <span class="grant">WT091310</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2013</span>;<span class="volume">62</span>;<span class="number">12</span>;<span class="pagerange">4270-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23884885">23884885</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3837024">3837024</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb13-0570">10.2337/db13-0570</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metabolomic markers reveal novel pathways of ageing and early development in human populations.</h4>
<p class="authors"><span class="author">Menni C</span>, <span class="author">Kastenm&uuml;ller G</span>, <span class="author">Petersen AK</span>, <span class="author">Bell JT</span>, <span class="author">Psatha M</span>, <span class="author">Tsai PC</span>, <span class="author">Gieger C</span>, <span class="author">Schulz H</span>, <span class="author">Erte I</span>, <span class="author">John S</span>, <span class="author">Brosnan MJ</span>, <span class="author">Wilson SG</span>, <span class="author">Tsaprouni L</span>, <span class="author">Lim EM</span>, <span class="author">Stuckey B</span>, <span class="author">Deloukas P</span>, <span class="author">Mohney R</span>, <span class="author">Suhre K</span>, <span class="author">Spector TD</span> and <span class="author">Valdes AM</span></p>
<p class="affiliation">Department of Twin Research &amp; Genetic Epidemiology, King&#39;s College London, London, UK, Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum M&uuml;nchen, Neuherberg, Germany, Institute of Genetic Epidemiology, Helmholtz Zentrum M&uuml;nchen, Neuherberg, Germany, Institute of Epidemiology I, Helmholtz Zentrum M&uuml;nchen, Neuherberg, Germany, Pfizer Research Laboratories, Groton, CT, USA, Worldwide R&amp;D, Pfizer Inc., Cambridge, MA, USA, School of Medicine and Pharmacology, University of Western Australia, Crawley, WA, Australia, Department of Endocrinology and Diabetes, Sir Charles Gairdner Hospital, Nedlands, WA, Australia, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK, Metabolon Inc., 617 Davis Drive, Durham, NC 27713, USA; Department of Physiology and Biophysics, Weill Cornell Medical College in Qatar, Education City, Qatar Foundation, Doha, State of Qatar and Academic Rheumatology, University of Nottingham, Nottingham City Hospital, Nottingham, UK.</p><p class="abstract"><strong>Background</strong>: Human ageing is a complex, multifactorial process and early developmental factors affect health outcomes in old age.</p><p class="abstract"><strong>Methods</strong>: Metabolomic profiling on fasting blood was carried out in 6055 individuals from the UK. Stepwise regression was performed to identify a panel of independent metabolites which could be used as a surrogate for age. We also investigated the association with birthweight overall and within identical discordant twins and with genome-wide methylation levels.</p><p class="abstract"><strong>Results</strong>: We identified a panel of 22 metabolites which combined are strongly correlated with age (R(2) = 59%) and with age-related clinical traits independently of age. One particular metabolite, C-glycosyl tryptophan (C-glyTrp), correlated strongly with age (beta = 0.03, SE = 0.001, P = 7.0 &times; 10(-157)) and lung function (FEV1 beta = -0.04, SE = 0.008, P = 1.8 &times; 10(-8) adjusted for age and confounders) and was replicated in an independent population (n = 887). C-glyTrp was also associated with bone mineral density (beta = -0.01, SE = 0.002, P = 1.9 &times; 10(-6)) and birthweight (beta = -0.06, SE = 0.01, P = 2.5 &times; 10(-9)). The difference in C-glyTrp levels explained 9.4% of the variance in the difference in birthweight between monozygotic twins. An epigenome-wide association study in 172 individuals identified three CpG-sites, associated with levels of C-glyTrp (P &lt; 2 &times; 10(-6)). We replicated one CpG site in the promoter of the WDR85 gene in an independent sample of 350 individuals (beta = -0.20, SE = 0.04, P = 2.9 &times; 10(-8)). WDR85 is a regulator of translation elongation factor 2, essential for protein synthesis in eukaryotes.</p><p class="abstract"><strong>Conclusions</strong>: Our data illustrate how metabolomic profiling linked with epigenetic studies can identify some key molecular mechanisms potentially determined in early development that produce long-term physiological changes influencing human health and ageing.</p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23838602">23838602</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdyt094">10.1093/ije/dyt094</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quantifying single nucleotide variant detection sensitivity in exome sequencing.</h4>
<p class="authors"><span class="author">Meynert AM</span>, <span class="author">Bicknell LS</span>, <span class="author">Hurles ME</span>, <span class="author">Jackson AP</span> and <span class="author">Taylor MS</span></p>
<p class="affiliation">MRC Human Genetics Unit, MRC Institute for Genetics and Molecular Medicine, University of Edinburgh, Western General Hospital, Crewe Road, Edinburgh, UK. <a href="mailto:%61%6C%69%73%6F%6E.%6D%65%79%6E%65%72%74%40%69%67%6D%6D.%65%64.%61%63.%75%6B">&#97;&#108;&#105;&#115;&#111;&#110;&#46;&#109;&#101;&#121;&#110;&#101;&#114;&#116;&#64;&#105;&#103;&#109;&#109;&#46;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: The targeted capture and sequencing of genomic regions has rapidly demonstrated its utility in genetic studies. Inherent in this technology is considerable heterogeneity of target coverage and this is expected to systematically impact our sensitivity to detect genuine polymorphisms. To fully interpret the polymorphisms identified in a genetic study it is often essential to both detect polymorphisms and to understand where and with what probability real polymorphisms may have been missed.

Results: Using down-sampling of 30 deeply sequenced exomes and a set of gold-standard single nucleotide variant (SNV) genotype calls for each sample, we developed an empirical model relating the read depth at a polymorphic site to the probability of calling the correct genotype at that site. We find that measured sensitivity in SNV detection is substantially worse than that predicted from the naive expectation of sampling from a binomial. This calibrated model allows us to produce single nucleotide resolution SNV sensitivity estimates which can be merged to give summary sensitivity measures for any arbitrary partition of the target sequences (nucleotide, exon, gene, pathway, exome). These metrics are directly comparable between platforms and can be combined between samples to give "power estimates" for an entire study. We estimate a local read depth of 13X is required to detect the alleles and genotype of a heterozygous SNV 95% of the time, but only 3X for a homozygous SNV. At a mean on-target read depth of 20X, commonly used for rare disease exome sequencing studies, we predict 5-15% of heterozygous and 1-4% of homozygous SNVs in the targeted regions will be missed.

Conclusions: Non-reference alleles in the heterozygote state have a high chance of being missed when commonly applied read coverage thresholds are used despite the widely held assumption that there is good polymorphism detection at these coverage levels. Such alleles are likely to be of functional importance in population based studies of rare diseases, somatic mutations in cancer and explaining the "missing heritability" of quantitative traits.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">195</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23773188">23773188</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3695811">3695811</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-14-195">10.1186/1471-2105-14-195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Empirical research on the ethics of genomic research.</h4>
<p class="authors"><span class="author">Middleton A</span>, <span class="author">Parker M</span>, <span class="author">Wright CF</span>, <span class="author">Bragin E</span>, <span class="author">Hurles ME</span> and <span class="author">DDD Study</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%61%6D%33%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#109;&#51;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2013</span>;<span class="volume">161A</span>;<span class="number">8</span>;<span class="pagerange">2099-101</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23813698">23813698</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3884757">3884757</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.36067">10.1002/ajmg.a.36067</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple populations of artemisinin-resistant Plasmodium falciparum in Cambodia.</h4>
<p class="authors"><span class="author">Miotto O</span>, <span class="author">Almagro-Garcia J</span>, <span class="author">Manske M</span>, <span class="author">Macinnis B</span>, <span class="author">Campino S</span>, <span class="author">Rockett KA</span>, <span class="author">Amaratunga C</span>, <span class="author">Lim P</span>, <span class="author">Suon S</span>, <span class="author">Sreng S</span>, <span class="author">Anderson JM</span>, <span class="author">Duong S</span>, <span class="author">Nguon C</span>, <span class="author">Chuor CM</span>, <span class="author">Saunders D</span>, <span class="author">Se Y</span>, <span class="author">Lon C</span>, <span class="author">Fukuda MM</span>, <span class="author">Amenga-Etego L</span>, <span class="author">Hodgson AV</span>, <span class="author">Asoala V</span>, <span class="author">Imwong M</span>, <span class="author">Takala-Harrison S</span>, <span class="author">Nosten F</span>, <span class="author">Su XZ</span>, <span class="author">Ringwald P</span>, <span class="author">Ariey F</span>, <span class="author">Dolecek C</span>, <span class="author">Hien TT</span>, <span class="author">Boni MF</span>, <span class="author">Thai CQ</span>, <span class="author">Amambua-Ngwa A</span>, <span class="author">Conway DJ</span>, <span class="author">Djimd&eacute; AA</span>, <span class="author">Doumbo OK</span>, <span class="author">Zongo I</span>, <span class="author">Ouedraogo JB</span>, <span class="author">Alcock D</span>, <span class="author">Drury E</span>, <span class="author">Auburn S</span>, <span class="author">Koch O</span>, <span class="author">Sanders M</span>, <span class="author">Hubbart C</span>, <span class="author">Maslen G</span>, <span class="author">Ruano-Rubio V</span>, <span class="author">Jyothi D</span>, <span class="author">Miles A</span>, <span class="author">O&#39;Brien J</span>, <span class="author">Gamble C</span>, <span class="author">Oyola SO</span>, <span class="author">Rayner JC</span>, <span class="author">Newbold CI</span>, <span class="author">Berriman M</span>, <span class="author">Spencer CC</span>, <span class="author">McVean G</span>, <span class="author">Day NP</span>, <span class="author">White NJ</span>, <span class="author">Bethell D</span>, <span class="author">Dondorp AM</span>, <span class="author">Plowe CV</span>, <span class="author">Fairhurst RM</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Medical Research Council MRC Centre for Genomics and Global Health, University of Oxford, Oxford, UK.</p><p class="abstract">We describe an analysis of genome variation in 825 P. falciparum samples from Asia and Africa that identifies an unusual pattern of parasite population structure at the epicenter of artemisinin resistance in western Cambodia. Within this relatively small geographic area, we have discovered several distinct but apparently sympatric parasite subpopulations with extremely high levels of genetic differentiation. Of particular interest are three subpopulations, all associated with clinical resistance to artemisinin, which have skewed allele frequency spectra and high levels of haplotype homozygosity, indicative of founder effects and recent population expansion. We provide a catalog of SNPs that show high levels of differentiation in the artemisinin-resistant subpopulations, including codon variants in transporter proteins and DNA mismatch repair proteins. These data provide a population-level genetic framework for investigating the biological origins of artemisinin resistance and for defining molecular markers to assist in its elimination.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute: <span class="grant">55005502</span>; Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>, <span class="grant">MC_U190081987</span>; Wellcome Trust: <span class="grant">082370</span>, <span class="grant">089275</span>, <span class="grant">089276</span>, <span class="grant">090532</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">090770</span>, <span class="grant">090770/Z/09/Z</span>, <span class="grant">093956</span>, <span class="grant">098051</span>, <span class="grant">G0600718</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">648-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23624527">23624527</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3807790">3807790</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2624">10.1038/ng.2624</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The challenge of increasing Pfam coverage of the human proteome.</h4>
<p class="authors"><span class="author">Mistry J</span>, <span class="author">Coggill P</span>, <span class="author">Eberhardt RY</span>, <span class="author">Deiana A</span>, <span class="author">Giansanti A</span>, <span class="author">Finn RD</span>, <span class="author">Bateman A</span> and <span class="author">Punta M</span></p>
<p class="affiliation">EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK.</p>
<p class="abstract">It is a worthy goal to completely characterize all human proteins in terms of their domains. Here, using the Pfam database, we asked how far we have progressed in this endeavour. Ninety per cent of proteins in the human proteome matched at least one of 5494 manually curated Pfam-A families. In contrast, human residue coverage by Pfam-A families was <45%, with 9418 automatically generated Pfam-B families adding a further 10%. Even after excluding predicted signal peptide regions and short regions (<50 consecutive residues) unlikely to harbour new families, for âˆ¼38% of the human protein residues, there was no information in Pfam about conservation and evolutionary relationship with other protein regions. This uncovered portion of the human proteome was found to be distributed over almost 25 000 distinct protein regions. Comparison with proteins in the UniProtKB database suggested that the human regions that exhibited similarity to thousands of other sequences were often either divergent elements or N- or C-terminal extensions of existing families. Thirty-four per cent of regions, on the other hand, matched fewer than 100 sequences in UniProtKB. Most of these did not appear to share any relationship with existing Pfam-A families, suggesting that thousands of new families would need to be generated to cover them. Also, these latter regions were particularly rich in amino acid compositional bias such as the one associated with intrinsic disorder. This could represent a significant obstacle toward their inclusion into new Pfam families. Based on these observations, a major focus for increasing Pfam coverage of the human proteome will be to improve the definition of existing families. New families will also be built, prioritizing those that have been experimentally functionally characterized. Database URL: http://pfam.sanger.ac.uk/</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2013</span>;<span class="volume">2013</span>;<span class="pagerange">bat023</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23603847">23603847</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3630804">3630804</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbat023">10.1093/database/bat023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions.</h4>
<p class="authors"><span class="author">Mistry J</span>, <span class="author">Finn RD</span>, <span class="author">Eddy SR</span>, <span class="author">Bateman A</span> and <span class="author">Punta M</span></p>
<p class="affiliation">EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK and HHMI Janelia Farm Research Campus, 19700 Helix Drive, Ashburn, VA 20147, USA.</p>
<p class="abstract">Detection of protein homology via sequence similarity has important applications in biology, from protein structure and function prediction to reconstruction of phylogenies. Although current methods for aligning protein sequences are powerful, challenges remain, including problems with homologous overextension of alignments and with regions under convergent evolution. Here, we test the ability of the profile hidden Markov model method HMMER3 to correctly assign homologous sequences to >13 000 manually curated families from the Pfam database. We identify problem families using protein regions that match two or more Pfam families not currently annotated as related in Pfam. We find that HMMER3 E-value estimates seem to be less accurate for families that feature periodic patterns of compositional bias, such as the ones typically observed in coiled-coils. These results support the continued use of manually curated inclusion thresholds in the Pfam database, especially on the subset of families that have been identified as problematic in experiments such as these. They also highlight the need for developing new methods that can correct for this particular type of compositional bias.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23598997">23598997</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkt263">10.1093/nar/gkt263</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nuclear Wave1 is required for reprogramming transcription in oocytes and for normal development.</h4>
<p class="authors"><span class="author">Miyamoto K</span>, <span class="author">Teperek M</span>, <span class="author">Yusa K</span>, <span class="author">Allen GE</span>, <span class="author">Bradshaw CR</span> and <span class="author">Gurdon JB</span></p>
<p class="affiliation">Wellcome Trust/Cancer Research UK Gurdon Institute, The Henry Wellcome Building of Cancer and Developmental Biology, Cambridge, UK. <a href="mailto:%6B.%6D%69%79%61%6D%6F%74%6F%40%67%75%72%64%6F%6E.%63%61%6D.%61%63.%75%6B">&#107;&#46;&#109;&#105;&#121;&#97;&#109;&#111;&#116;&#111;&#64;&#103;&#117;&#114;&#100;&#111;&#110;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="abstract">Eggs and oocytes have a remarkable ability to induce transcription of sperm after normal fertilization and in somatic nuclei after somatic cell nuclear transfer. This ability of eggs and oocytes is essential for normal development. Nuclear actin and actin-binding proteins have been shown to contribute to transcription, although their mode of action is elusive. Here, we find that Xenopus Wave1, previously characterized as a protein involved in actin cytoskeleton organization, is present in the oocyte nucleus and is required for efficient transcriptional reprogramming. Moreover, Wave1 knockdown in embryos results in abnormal development and defective hox gene activation. Nuclear Wave1 binds by its WHD domain to active transcription components, and this binding contributes to the action of RNA polymerase II. We identify Wave1 as a maternal reprogramming factor that also has a necessary role in gene activation in development.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1001690/1</span>; Wellcome Trust: <span class="grant">101050/Z/13/Z</span>, <span class="grant">WT077187</span>, <span class="grant">WT089613</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">341</span>;<span class="number">6149</span>;<span class="pagerange">1002-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23990560">23990560</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1240376">10.1126/science.1240376</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deciphering the Mechanisms of Developmental Disorders (DMDD): a new programme for phenotyping embryonic lethal mice.</h4>
<p class="authors"><span class="author">Mohun T</span>, <span class="author">Adams DJ</span>, <span class="author">Baldock R</span>, <span class="author">Bhattacharya S</span>, <span class="author">Copp AJ</span>, <span class="author">Hemberger M</span>, <span class="author">Houart C</span>, <span class="author">Hurles ME</span>, <span class="author">Robertson E</span>, <span class="author">Smith JC</span>, <span class="author">Weaver T</span> and <span class="author">Weninger W</span></p>
<p class="affiliation">MRC National Institute for Medical Research, London, NW7 1AA, UK.</p>
<p class="abstract">International efforts to test gene function in the mouse by the systematic knockout of each gene are creating many lines in which embryonic development is compromised. These homozygous lethal mutants represent a potential treasure trove for the biomedical community. Developmental biologists could exploit them in their studies of tissue differentiation and organogenesis; for clinical researchers they offer a powerful resource for investigating the origins of developmental diseases that affect newborns. Here, we outline a new programme of research in the UK aiming to kick-start research with embryonic lethal mouse lines. The 'Deciphering the Mechanisms of Developmental Disorders' (DMDD) programme has the ambitious goal of identifying all embryonic lethal knockout lines made in the UK over the next 5 years, and will use a combination of comprehensive imaging and transcriptomics to identify abnormalities in embryo structure and development. All data will be made freely available, enabling individual researchers to identify lines relevant to their research. The DMDD programme will coordinate its work with similar international efforts through the umbrella of the International Mouse Phenotyping Consortium [see accompanying Special Article (Adams et al., 2013)] and, together, these programmes will provide a novel database for embryonic development, linking gene identity with molecular profiles and morphology phenotypes.</p>
<p class="publication"><span class="title">Disease models &amp;amp; mechanisms</span>&nbsp;<span class="year">2013</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">562-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23519034">23519034</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdmm.011957">10.1242/dmm.011957</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MiR-210 is induced by Oct-2, regulates B cells, and inhibits autoantibody production.</h4>
<p class="authors"><span class="author">Mok Y</span>, <span class="author">Schwierzeck V</span>, <span class="author">Thomas DC</span>, <span class="author">Vigorito E</span>, <span class="author">Rayner TF</span>, <span class="author">Jarvis LB</span>, <span class="author">Prosser HM</span>, <span class="author">Bradley A</span>, <span class="author">Withers DR</span>, <span class="author">M&aring;rtensson IL</span>, <span class="author">Corcoran LM</span>, <span class="author">Blenkiron C</span>, <span class="author">Miska EA</span>, <span class="author">Lyons PA</span> and <span class="author">Smith KG</span></p>
<p class="affiliation">Department of Medicine, Cambridge Institute for Medical Research, University of Cambridge School of Clinical Medicine, Addenbrooke&#39;s Hospital, Cambridge CB2 0XY, United Kingdom.</p><p class="abstract">MicroRNAs (MiRs) are small, noncoding RNAs that regulate gene expression posttranscriptionally. In this study, we show that MiR-210 is induced by Oct-2, a key transcriptional mediator of B cell activation. Germline deletion of MiR-210 results in the development of autoantibodies from 5 mo of age. Overexpression of MiR-210 in vivo resulted in cell autonomous expansion of the B1 lineage and impaired fitness of B2 cells. Mice overexpressing MiR-210 exhibited impaired class-switched Ab responses, a finding confirmed in wild-type B cells transfected with a MiR-210 mimic. In vitro studies demonstrated defects in cellular proliferation and cell cycle entry, which were consistent with the transcriptomic analysis demonstrating downregulation of genes involved in cellular proliferation and B cell activation. These findings indicate that Oct-2 induction of MiR-210 provides a novel inhibitory mechanism for the control of B cells and autoantibody production.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Department of Health: <span class="grant">1693</span>; Medical Research Council; Wellcome Trust: <span class="grant">06753AIA</span>, <span class="grant">083650</span>, <span class="grant">100140</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2013</span>;<span class="volume">191</span>;<span class="number">6</span>;<span class="pagerange">3037-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23960236">23960236</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4162006">4162006</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1301289">10.4049/jimmunol.1301289</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The origin, evolution, and functional impact of short insertion-deletion variants identified in 179 human genomes.</h4>
<p class="authors"><span class="author">Montgomery SB</span>, <span class="author">Goode DL</span>, <span class="author">Kvikstad E</span>, <span class="author">Albers CA</span>, <span class="author">Zhang ZD</span>, <span class="author">Mu XJ</span>, <span class="author">Ananda G</span>, <span class="author">Howie B</span>, <span class="author">Karczewski KJ</span>, <span class="author">Smith KS</span>, <span class="author">Anaya V</span>, <span class="author">Richardson R</span>, <span class="author">Davis J</span>, <span class="author">1000 Genomes Project Consortium</span>, <span class="author">Macarthur DG</span>, <span class="author">Sidow A</span>, <span class="author">Duret L</span>, <span class="author">Gerstein M</span>, <span class="author">Makova KD</span>, <span class="author">Marchini J</span>, <span class="author">McVean G</span> and <span class="author">Lunter G</span></p>
<p class="affiliation">Department of Genetic Medicine and Development, University of Geneva Medical School, Geneva, 1211, Switzerland;</p>
<p class="abstract">Short insertions and deletions (indels) are the second most abundant form of human genetic variation, but our understanding of their origins and functional effects lags behind that of other types of variants. Using population-scale sequencing, we have identified a high-quality set of 1.6 million indels from 179 individuals representing three diverse human populations. We show that rates of indel mutagenesis are highly heterogeneous, with 43%-48% of indels occurring in 4.03% of the genome, whereas in the remaining 96% their prevalence is 16 times lower than SNPs. Polymerase slippage can explain upwards of three-fourths of all indels, with the remainder being mostly simple deletions in complex sequence. However, insertions do occur and are significantly associated with pseudo-palindromic sequence features compatible with the fork stalling and template switching (FoSTeS) mechanism more commonly associated with large structural variations. We introduce a quantitative model of polymerase slippage, which enables us to identify indel-hypermutagenic protein-coding genes, some of which are associated with recurrent mutations leading to disease. Accounting for mutational rate heterogeneity due to sequence context, we find that indels across functional sequence are generally subject to stronger purifying selection than SNPs. We find that indel length modulates selection strength, and that indels affecting multiple functionally constrained nucleotides undergo stronger purifying selection. We further find that indels are enriched in associations with gene expression and find evidence for a contribution of nonsense-mediated decay. Finally, we show that indels can be integrated in existing genome-wide association studies (GWAS); although we do not find direct evidence that potentially causal protein-coding indels are enriched with associations to known disease-associated SNPs, our findings suggest that the causal variant underlying some of these associations may be indels.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">5</span>;<span class="pagerange">749-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23478400">23478400</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3638132">3638132</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.148718.112">10.1101/gr.148718.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Implementing a successful data-management framework: the UK10K managed access model.</h4>
<p class="authors"><span class="author">Muddyman D</span>, <span class="author">Smee C</span>, <span class="author">Griffin H</span> and <span class="author">Kaye J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">This paper outlines the history behind open access principles and describes the development of a managed access data-sharing process for the UK10K Project, currently Britain&#39;s largest genomic sequencing consortium (2010 to 2013). Funded by the Wellcome Trust, the purpose of UK10K was two-fold: to investigate how low-frequency and rare genetic variants contribute to human disease, and to provide an enduring data resource for future research into human genetics. In this paper, we discuss the challenge of reconciling data-sharing principles with the practicalities of delivering a sequencing project of UK10K&#39;s scope and magnitude. We describe the development of a sustainable, easy-to-use managed access system that allowed rapid access to UK10K data, while protecting the interests of participants and data generators alike. Specifically, we focus in depth on the three key issues that emerge in the data pipeline: study recruitment, data release and data access.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">092731</span></p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">100</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24229443">24229443</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3978569">3978569</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm504">10.1186/gm504</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional transcriptomics in the post-ENCODE era.</h4>
<p class="authors"><span class="author">Mudge JM</span>, <span class="author">Frankish A</span> and <span class="author">Harrow J</span></p>
<p class="affiliation">Department of Informatics, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, United Kingdom.</p>
<p class="abstract">The last decade has seen tremendous effort committed to the annotation of the human genome sequence, most notably perhaps in the form of the ENCODE project. One of the major findings of ENCODE, and other genome analysis projects, is that the human transcriptome is far larger and more complex than previously thought. This complexity manifests, for example, as alternative splicing within protein-coding genes, as well as in the discovery of thousands of long noncoding RNAs. It is also possible that significant numbers of human transcripts have not yet been described by annotation projects, while existing transcript models are frequently incomplete. The question as to what proportion of this complexity is truly functional remains open, however, and this ambiguity presents a serious challenge to genome scientists. In this article, we will discuss the current state of human transcriptome annotation, drawing on our experience gained in generating the GENCODE gene annotation set. We highlight the gaps in our knowledge of transcript functionality that remain, and consider the potential computational and experimental strategies that can be used to help close them. We propose that an understanding of the true overlap between transcriptional complexity and functionality will not be gained in the short term. However, significant steps toward obtaining this knowledge can now be taken by using an integrated strategy, combining all of the experimental resources at our disposal.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24172201">24172201</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.161315.113">10.1101/gr.161315.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Independent specialization of the human and mouse X chromosomes for the male germ line.</h4>
<p class="authors"><span class="author">Mueller JL</span>, <span class="author">Skaletsky H</span>, <span class="author">Brown LG</span>, <span class="author">Zaghlul S</span>, <span class="author">Rock S</span>, <span class="author">Graves T</span>, <span class="author">Auger K</span>, <span class="author">Warren WC</span>, <span class="author">Wilson RK</span> and <span class="author">Page DC</span></p>
<p class="affiliation">Whitehead Institute, Cambridge, Massachusetts, USA.</p><p class="abstract">We compared the human and mouse X chromosomes to systematically test Ohno&#39;s law, which states that the gene content of X chromosomes is conserved across placental mammals. First, we improved the accuracy of the human X-chromosome reference sequence through single-haplotype sequencing of ampliconic regions. The new sequence closed gaps in the reference sequence, corrected previously misassembled regions and identified new palindromic amplicons. Our subsequent analysis led us to conclude that the evolution of human and mouse X chromosomes was bimodal. In accord with Ohno&#39;s law, 94-95% of X-linked single-copy genes are shared by humans and mice; most are expressed in both sexes. Notably, most X-ampliconic genes are exceptions to Ohno&#39;s law: only 31% of human and 22% of mouse X-ampliconic genes had orthologs in the other species. X-ampliconic genes are expressed predominantly in testicular germ cells, and many were independently acquired since divergence from the common ancestor of humans and mice, specializing portions of their X chromosomes for sperm production.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23872635">23872635</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2705">10.1038/ng.2705</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A powerful molecular synergy between mutant Nucleophosmin and Flt3-ITD drives acute myeloid leukemia in mice.</h4>
<p class="authors"><span class="author">Mupo A</span>, <span class="author">Celani L</span>, <span class="author">Dovey O</span>, <span class="author">Cooper JL</span>, <span class="author">Grove C</span>, <span class="author">Rad R</span>, <span class="author">Sportoletti P</span>, <span class="author">Falini B</span>, <span class="author">Bradley A</span> and <span class="author">Vassiliou GS</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095663</span></p>
<p class="publication"><span class="title">Leukemia</span>&nbsp;<span class="year">2013</span>;<span class="volume">27</span>;<span class="number">9</span>;<span class="pagerange">1917-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23478666">23478666</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3768110">3768110</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fleu.2013.77">10.1038/leu.2013.77</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of equine influenza virus in vaccinated horses.</h4>
<p class="authors"><span class="author">Murcia PR</span>, <span class="author">Baillie GJ</span>, <span class="author">Stack JC</span>, <span class="author">Jervis C</span>, <span class="author">Elton D</span>, <span class="author">Mumford JA</span>, <span class="author">Daly J</span>, <span class="author">Kellam P</span>, <span class="author">Grenfell BT</span>, <span class="author">Holmes EC</span> and <span class="author">Wood JL</span></p>
<p class="affiliation">Medical Research Council-University of Glasgow Centre for Virus Research, Institute of Infection, Inflammation and Immunity, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, United Kingdom.</p><p class="abstract">Influenza A viruses are characterized by their ability to evade host immunity, even in vaccinated individuals. To determine how prior immunity shapes viral diversity in vivo, we studied the intra- and interhost evolution of equine influenza virus in vaccinated horses. Although the level and structure of genetic diversity were similar to those in na&iuml;ve horses, intrahost bottlenecks may be more stringent in vaccinated animals, and mutations shared among horses often fall close to putative antigenic sites.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801822</span>; NIGMS NIH HHS: <span class="grant">R01 GM080533</span>, <span class="grant">R01 GM080533-06</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2013</span>;<span class="volume">87</span>;<span class="number">8</span>;<span class="pagerange">4768-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23388708">23388708</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3624384">3624384</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.03379-12">10.1128/JVI.03379-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sociodemographic distribution of non-communicable disease risk factors in rural Uganda: a cross-sectional study.</h4>
<p class="authors"><span class="author">Murphy GA</span>, <span class="author">Asiki G</span>, <span class="author">Ekoru K</span>, <span class="author">Nsubuga RN</span>, <span class="author">Nakiyingi-Miiro J</span>, <span class="author">Young EH</span>, <span class="author">Seeley J</span>, <span class="author">Sandhu MS</span> and <span class="author">Kamali A</span></p>
<p class="affiliation">Department of Public Health &amp; Primary Care, University of Cambridge, Cambridge, UK, Wellcome Trust Sanger Institute, Hinxton, UK, Medical Research Council/Uganda Virus Research Institute (MRC/UVRI), Uganda Research Unit on AIDS, Entebbe, Uganda, London School of Hygiene and Tropical Medicine, London, UK and School of International Development, University of East Anglia, Norwich, UK.</p><p class="abstract"><strong>Background</strong>: Non-communicable diseases (NCDs) are rapidly becoming leading causes of morbidity and mortality in low- and middle-income countries, including those in sub-Saharan Africa. In contrast to high-income countries, the sociodemographic distribution, including socioeconomic inequalities, of NCDs and their risk factors is unclear in sub-Saharan Africa, particularly among rural populations.</p><p class="abstract"><strong>Methods</strong>: We undertook a cross-sectional population-based survey of 7809 residents aged 13 years or older in the General Population Cohort in south-western rural Uganda. Information on behavioural, physiological and biochemical risk factors was obtained using standardized methods as recommended by the WHO STEPwise Approach to Surveillance. Socioeconomic status (SES) was determined by principal component analysis including household features, ownership, and occupation and education of the head of household.</p><p class="abstract"><strong>Results</strong>: SES was found to be associated with NCD risk factors in this rural population. Smoking, alcohol consumption (men only) and low high-density lipoprotein (HDL) cholesterol were more common among those of lower SES. For example, the prevalence of smoking decreased 4-fold from the lowest to the highest SES groups, from 22.0% to 5.7% for men and 2.2% to 0.4% for women, respectively. In contrast, overweight, raised blood pressure, raised HbA1c (women only) and raised cholesterol were more common among those of higher SES. For example, the prevalence of overweight increased 5-fold from 2.1% to 10.1% for men, and 2-fold from 12.0% to 23.4% for women, from the lowest to highest SES groups respectively. However, neither low physical activity nor fruit, vegetable or staples consumption was associated with SES. Furthermore, associations between NCD risk factors and SES were modified by age and sex.</p><p class="abstract"><strong>Conclusions</strong>: Within this rural population, NCD risk factors are common and vary both inversely and positively across the SES gradient. A better understanding of the determinants of the sociodemographic distribution of NCDs and their risk factors in rural sub-Saharan African populations will help identify populations at most risk of developing NCDs and help plan interventions to reduce their burden.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801566</span>, <span class="grant">G0901213</span>, <span class="grant">G0901213-92157</span>, <span class="grant">MC_U950080926</span></p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">42</span>;<span class="number">6</span>;<span class="pagerange">1740-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24191304">24191304</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4234905">4234905</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdyt184">10.1093/ije/dyt184</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cardiometabolic risk in a rural ugandan population.</h4>
<p class="authors"><span class="author">Murphy GA</span>, <span class="author">Asiki G</span>, <span class="author">Young EH</span>, <span class="author">Seeley J</span>, <span class="author">Nsubuga RN</span>, <span class="author">Sandhu MS</span> and <span class="author">Kamali A</span></p>
<p class="affiliation">Corresponding author: Georgina A.V. Murphy, <a href="mailto:%67%6D%37%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#109;&#55;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Diabetes care</span>&nbsp;<span class="year">2013</span>;<span class="volume">36</span>;<span class="number">9</span>;<span class="pagerange">e143</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23970722">23970722</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdc13-0739">10.2337/dc13-0739</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic CALR mutations in myeloproliferative neoplasms with nonmutated JAK2.</h4>
<p class="authors"><span class="author">Nangalia J</span>, <span class="author">Massie CE</span>, <span class="author">Baxter EJ</span>, <span class="author">Nice FL</span>, <span class="author">Gundem G</span>, <span class="author">Wedge DC</span>, <span class="author">Avezov E</span>, <span class="author">Li J</span>, <span class="author">Kollmann K</span>, <span class="author">Kent DG</span>, <span class="author">Aziz A</span>, <span class="author">Godfrey AL</span>, <span class="author">Hinton J</span>, <span class="author">Martincorena I</span>, <span class="author">Van Loo P</span>, <span class="author">Jones AV</span>, <span class="author">Guglielmelli P</span>, <span class="author">Tarpey P</span>, <span class="author">Harding HP</span>, <span class="author">Fitzpatrick JD</span>, <span class="author">Goudie CT</span>, <span class="author">Ortmann CA</span>, <span class="author">Loughran SJ</span>, <span class="author">Raine K</span>, <span class="author">Jones DR</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">O&#39;Meara S</span>, <span class="author">McLaren S</span>, <span class="author">Bianchi M</span>, <span class="author">Silber Y</span>, <span class="author">Dimitropoulou D</span>, <span class="author">Bloxham D</span>, <span class="author">Mudie L</span>, <span class="author">Maddison M</span>, <span class="author">Robinson B</span>, <span class="author">Keohane C</span>, <span class="author">Maclean C</span>, <span class="author">Hill K</span>, <span class="author">Orchard K</span>, <span class="author">Tauro S</span>, <span class="author">Du MQ</span>, <span class="author">Greaves M</span>, <span class="author">Bowen D</span>, <span class="author">Huntly BJ</span>, <span class="author">Harrison CN</span>, <span class="author">Cross NC</span>, <span class="author">Ron D</span>, <span class="author">Vannucchi AM</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="affiliation">The authors&#39; full names, degrees, and affiliations are listed in the Appendix.</p><p class="abstract"><strong>Background</strong>: Somatic mutations in the Janus kinase 2 gene (JAK2) occur in many myeloproliferative neoplasms, but the molecular pathogenesis of myeloproliferative neoplasms with nonmutated JAK2 is obscure, and the diagnosis of these neoplasms remains a challenge.</p><p class="abstract"><strong>Methods</strong>: We performed exome sequencing of samples obtained from 151 patients with myeloproliferative neoplasms. The mutation status of the gene encoding calreticulin (CALR) was assessed in an additional 1345 hematologic cancers, 1517 other cancers, and 550 controls. We established phylogenetic trees using hematopoietic colonies. We assessed calreticulin subcellular localization using immunofluorescence and flow cytometry.</p><p class="abstract"><strong>Results</strong>: Exome sequencing identified 1498 mutations in 151 patients, with medians of 6.5, 6.5, and 13.0 mutations per patient in samples of polycythemia vera, essential thrombocythemia, and myelofibrosis, respectively. Somatic CALR mutations were found in 70 to 84% of samples of myeloproliferative neoplasms with nonmutated JAK2, in 8% of myelodysplasia samples, in occasional samples of other myeloid cancers, and in none of the other cancers. A total of 148 CALR mutations were identified with 19 distinct variants. Mutations were located in exon 9 and generated a +1 base-pair frameshift, which would result in a mutant protein with a novel C-terminal. Mutant calreticulin was observed in the endoplasmic reticulum without increased cell-surface or Golgi accumulation. Patients with myeloproliferative neoplasms carrying CALR mutations presented with higher platelet counts and lower hemoglobin levels than patients with mutated JAK2. Mutation of CALR was detected in hematopoietic stem and progenitor cells. Clonal analyses showed CALR mutations in the earliest phylogenetic node, a finding consistent with its role as an initiating mutation in some patients.</p><p class="abstract"><strong>Conclusions</strong>: Somatic mutations in the endoplasmic reticulum chaperone CALR were found in a majority of patients with myeloproliferative neoplasms with nonmutated JAK2. (Funded by the Kay Kendall Leukaemia Fund and others.).</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Cancer Research UK; Wellcome Trust: <span class="grant">079249</span>, <span class="grant">084812</span>, <span class="grant">093867</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">369</span>;<span class="number">25</span>;<span class="pagerange">2391-405</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24325359">24325359</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3966280">3966280</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1312542">10.1056/NEJMoa1312542</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Special focus: bioinformatics.</h4>
<p class="authors"><span class="author">Nawrocki EP</span> and <span class="author">Burge SW</span></p>
<p class="affiliation">Eddy Lab; HHMI Janelia Farm Research Campus; Ashburn, VA USA.</p>
<p class="publication"><span class="title">RNA biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">7</span>;<span class="pagerange">1160</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23948768">23948768</a>; DOI: <a href="http://dx.doi.org/10.4161%2Frna.25606">10.4161/rna.25606</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis investigating associations between healthy diet and fasting glucose and insulin levels and modification by Loci associated with glucose homeostasis in data from 15 cohorts.</h4>
<p class="authors"><span class="author">Nettleton JA</span>, <span class="author">Hivert MF</span>, <span class="author">Lemaitre RN</span>, <span class="author">McKeown NM</span>, <span class="author">Mozaffarian D</span>, <span class="author">Tanaka T</span>, <span class="author">Wojczynski MK</span>, <span class="author">Hruby A</span>, <span class="author">Djouss&eacute; L</span>, <span class="author">Ngwa JS</span>, <span class="author">Follis JL</span>, <span class="author">Dimitriou M</span>, <span class="author">Ganna A</span>, <span class="author">Houston DK</span>, <span class="author">Kanoni S</span>, <span class="author">Mikkil&auml; V</span>, <span class="author">Manichaikul A</span>, <span class="author">Ntalla I</span>, <span class="author">Renstr&ouml;m F</span>, <span class="author">Sonestedt E</span>, <span class="author">van Rooij FJ</span>, <span class="author">Bandinelli S</span>, <span class="author">de Koning L</span>, <span class="author">Ericson U</span>, <span class="author">Hassanali N</span>, <span class="author">Kiefte-de Jong JC</span>, <span class="author">Lohman KK</span>, <span class="author">Raitakari O</span>, <span class="author">Papoutsakis C</span>, <span class="author">Sjogren P</span>, <span class="author">Stirrups K</span>, <span class="author">Ax E</span>, <span class="author">Deloukas P</span>, <span class="author">Groves CJ</span>, <span class="author">Jacques PF</span>, <span class="author">Johansson I</span>, <span class="author">Liu Y</span>, <span class="author">McCarthy MI</span>, <span class="author">North K</span>, <span class="author">Viikari J</span>, <span class="author">Zillikens MC</span>, <span class="author">Dupuis J</span>, <span class="author">Hofman A</span>, <span class="author">Kolovou G</span>, <span class="author">Mukamal K</span>, <span class="author">Prokopenko I</span>, <span class="author">Rolandsson O</span>, <span class="author">Sepp&auml;l&auml; I</span>, <span class="author">Cupples LA</span>, <span class="author">Hu FB</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Borecki IB</span>, <span class="author">Ferrucci L</span>, <span class="author">Jacobs DR</span>, <span class="author">Kritchevsky SB</span>, <span class="author">Orho-Melander M</span>, <span class="author">Pankow JS</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Witteman JC</span>, <span class="author">Ingelsson E</span>, <span class="author">Siscovick DS</span>, <span class="author">Dedoussis G</span>, <span class="author">Meigs JB</span> and <span class="author">Franks PW</span></p>
<p class="abstract">Whether loci that influence fasting glucose (FG) and fasting insulin (FI) levels, as identified by genome-wide association studies, modify associations of diet with FG or FI is unknown. We utilized data from 15 US and European cohort studies comprising 51,289 persons without diabetes to test whether genotype and diet interact to influence FG or FI concentration. We constructed a diet score using study-specific quartile rankings for intakes of whole grains, fish, fruits, vegetables, and nuts/seeds (favorable) and red/processed meats, sweets, sugared beverages, and fried potatoes (unfavorable). We used linear regression within studies, followed by inverse-variance-weighted meta-analysis, to quantify 1) associations of diet score with FG and FI levels and 2) interactions of diet score with 16 FG-associated loci and 2 FI-associated loci. Diet score (per unit increase) was inversely associated with FG (Î² = -0.004 mmol/L, 95% confidence interval: -0.005, -0.003) and FI (Î² = -0.008 ln-pmol/L, 95% confidence interval: -0.009, -0.007) levels after adjustment for demographic factors, lifestyle, and body mass index. Genotype variation at the studied loci did not modify these associations. Healthier diets were associated with lower FG and FI concentrations regardless of genotype at previously replicated FG- and FI-associated loci. Studies focusing on genomic regions that do not yield highly statistically significant associations from main-effect genome-wide association studies may be more fruitful in identifying diet-gene interactions.</p>
<p class="publication"><span class="title">American journal of epidemiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">177</span>;<span class="number">2</span>;<span class="pagerange">103-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23255780" rel="external">23255780</a>; DOI: <a href="http://dx.doi.org/10.1093%2Faje%2Fkws297" rel="external">10.1093/aje/kws297</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The relative timing of mutations in a breast cancer genome.</h4>
<p class="authors"><span class="author">Newman S</span>, <span class="author">Howarth KD</span>, <span class="author">Greenman CD</span>, <span class="author">Bignell GR</span>, <span class="author">Tavar&eacute; S</span> and <span class="author">Edwards PA</span></p>
<p class="affiliation">Hutchison/MRC Research Centre and Department of Pathology, University of Cambridge, Cambridge, United Kingdom.</p>
<p class="abstract">Many tumors have highly rearranged genomes, but a major unknown is the relative importance and timing of genome rearrangements compared to sequence-level mutation. Chromosome instability might arise early, be a late event contributing little to cancer development, or happen as a single catastrophic event. Another unknown is which of the point mutations and rearrangements are selected. To address these questions we show, using the breast cancer cell line HCC1187 as a model, that we can reconstruct the likely history of a breast cancer genome. We assembled probably the most complete map to date of a cancer genome, by combining molecular cytogenetic analysis with sequence data. In particular, we assigned most sequence-level mutations to individual chromosomes by sequencing of flow sorted chromosomes. The parent of origin of each chromosome was assigned from SNP arrays. We were then able to classify most of the mutations as earlier or later according to whether they occurred before or after a landmark event in the evolution of the genome, endoreduplication (duplication of its entire genome). Genome rearrangements and sequence-level mutations were fairly evenly divided earlier and later, suggesting that genetic instability was relatively constant throughout the life of this tumor, and chromosome instability was not a late event. Mutations that caused chromosome instability would be in the earlier set. Strikingly, the great majority of inactivating mutations and in-frame gene fusions happened earlier. The non-random timing of some of the mutations may be evidence that they were selected.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">e64991</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23762276">23762276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3677865">3677865</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0064991">10.1371/journal.pone.0064991</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics in Chlamydomonas and Plasmodium identifies an ancient nuclear envelope protein family essential for sexual reproduction in protists, fungi, plants, and vertebrates.</h4>
<p class="authors"><span class="author">Ning J</span>, <span class="author">Otto TD</span>, <span class="author">Pfander C</span>, <span class="author">Schwach F</span>, <span class="author">Brochet M</span>, <span class="author">Bushell E</span>, <span class="author">Goulding D</span>, <span class="author">Sanders M</span>, <span class="author">Lefebvre PA</span>, <span class="author">Pei J</span>, <span class="author">Grishin NV</span>, <span class="author">Vanderlaan G</span>, <span class="author">Billker O</span> and <span class="author">Snell WJ</span></p>
<p class="affiliation">Department of Cell Biology, University of Texas Southwestern Medical School, Dallas, Texas 75390, USA.</p><p class="abstract">Fertilization is a crucial yet poorly characterized event in eukaryotes. Our previous discovery that the broadly conserved protein HAP2 (GCS1) functioned in gamete membrane fusion in the unicellular green alga Chlamydomonas and the malaria pathogen Plasmodium led us to exploit the rare biological phenomenon of isogamy in Chlamydomonas in a comparative transcriptomics strategy to uncover additional conserved sexual reproduction genes. All previously identified Chlamydomonas fertilization-essential genes fell into related clusters based on their expression patterns. Out of several conserved genes in a minus gamete cluster, we focused on Cre06.g280600, an ortholog of the fertilization-related Arabidopsis GEX1. Gene disruption, cell biological, and immunolocalization studies show that CrGEX1 functions in nuclear fusion in Chlamydomonas. Moreover, CrGEX1 and its Plasmodium ortholog, PBANKA_113980, are essential for production of viable meiotic progeny in both organisms and thus for mosquito transmission of malaria. Remarkably, we discovered that the genes are members of a large, previously unrecognized family whose first-characterized member, KAR5, is essential for nuclear fusion during yeast sexual reproduction. Our comparative transcriptomics approach provides a new resource for studying sexual development and demonstrates that exploiting the data can lead to the discovery of novel biology that is conserved across distant taxa.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; NCRR NIH HHS: <span class="grant">C06 RR 30414</span>; NIGMS NIH HHS: <span class="grant">GM25661</span>, <span class="grant">GM56778</span>, <span class="grant">R01 GM025661</span>, <span class="grant">R01 GM056778</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2013</span>;<span class="volume">27</span>;<span class="number">10</span>;<span class="pagerange">1198-215</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23699412">23699412</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672651">3672651</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.212746.112">10.1101/gad.212746.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Binding of nucleoid-associated protein fis to DNA is regulated by DNA breathing dynamics.</h4>
<p class="authors"><span class="author">Nowak-Lovato K</span>, <span class="author">Alexandrov LB</span>, <span class="author">Banisadr A</span>, <span class="author">Bauer AL</span>, <span class="author">Bishop AR</span>, <span class="author">Usheva A</span>, <span class="author">Mu F</span>, <span class="author">Hong-Geller E</span>, <span class="author">Rasmussen K&Oslash;</span>, <span class="author">Hlavacek WS</span> and <span class="author">Alexandrov BS</span></p>
<p class="affiliation">Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America.</p>
<p class="abstract">Physicochemical properties of DNA, such as shape, affect protein-DNA recognition. However, the properties of DNA that are most relevant for predicting the binding sites of particular transcription factors (TFs) or classes of TFs have yet to be fully understood. Here, using a model that accurately captures the melting behavior and breathing dynamics (spontaneous local openings of the double helix) of double-stranded DNA, we simulated the dynamics of known binding sites of the TF and nucleoid-associated protein Fis in Escherichia coli. Our study involves simulations of breathing dynamics, analysis of large published in vitro and genomic datasets, and targeted experimental tests of our predictions. Our simulation results and available in vitro binding data indicate a strong correlation between DNA breathing dynamics and Fis binding. Indeed, we can define an average DNA breathing profile that is characteristic of Fis binding sites. This profile is significantly enriched among the identified in vivo E. coli Fis binding sites. To test our understanding of how Fis binding is influenced by DNA breathing dynamics, we designed base-pair substitutions, mismatch, and methylation modifications of DNA regions that are known to interact (or not interact) with Fis. The goal in each case was to make the local DNA breathing dynamics either closer to or farther from the breathing profile characteristic of a strong Fis binding site. For the modified DNA segments, we found that Fis-DNA binding, as assessed by gel-shift assay, changed in accordance with our expectations. We conclude that Fis binding is associated with DNA breathing dynamics, which in turn may be regulated by various nucleotide modifications.</p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">e1002881</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23341768">23341768</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3547798">3547798</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.1002881">10.1371/journal.pcbi.1002881</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chlamydia trachomatis clinical isolates identified as tetracycline resistant do not exhibit resistance in vitro: whole-genome sequencing reveals a mutation in porB but no evidence for tetracycline resistance genes.</h4>
<p class="authors"><span class="author">O&#39;Neill CE</span>, <span class="author">Seth-Smith HM</span>, <span class="author">Van Der Pol B</span>, <span class="author">Harris SR</span>, <span class="author">Thomson NR</span>, <span class="author">Cutcliffe LT</span> and <span class="author">Clarke IN</span></p>
<p class="affiliation">Faculty of Medicine, CES Academic Unit, University of Southampton, Southampton General Hospital, Tremona Road, Southampton, UK. c.o&#39;<a href="mailto:%6E%65%69%6C%6C%40%73%6F%74%6F%6E.%61%63.%75%6B">&#110;&#101;&#105;&#108;&#108;&#64;&#115;&#111;&#116;&#111;&#110;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chlamydia trachomatis is the most common bacterial sexually transmitted infection worldwide and the leading cause of preventable blindness in developing countries. Tetracycline is commonly the drug of choice for treating C. trachomatis infections, but cases of antibiotic resistance in clinical isolates have previously been reported. Here, we used antibiotic resistance assays and whole-genome sequencing to interrogate the hypothesis that two clinical isolates (IU824 and IU888) have acquired mechanisms of antibiotic resistance. Immunofluorescence staining was used to identify C. trachomatis inclusions in cell cultures grown in the presence of tetracycline; however, only antibiotic-free control cultures yielded the strong fluorescence associated with the presence of chlamydial inclusions. Infectivity was lost upon passage of harvested cultures grown in the presence of tetracycline into antibiotic-free medium, so we conclude that these isolates were phenotypically sensitive to tetracycline. Comparisons of the genome and plasmid sequences for the two isolates with tetracycline-sensitive strains did not identify regions of low sequence identity that could accommodate horizontally acquired resistance genes, and the tetracycline binding region of the 16S rRNA gene was identical to that of the sensitive control strains. The porB gene of strain IU824, however, was found to contain a premature stop codon not previously identified, which is noteworthy but unlikely to be related to tetracycline resistance. In conclusion, we found no evidence of tetracycline resistance in the two strains investigated, and it seems most likely that the small, aberrant inclusions previously identified resulted from the high chlamydial load used in the original antibiotic resistance assays.</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2013</span>;<span class="volume">159</span>;<span class="number">Pt 4</span>;<span class="pagerange">748-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23378575">23378575</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.065391-0">10.1099/mic.0.065391-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies identifies six new Loci for serum calcium concentrations.</h4>
<p class="authors"><span class="author">O&#39;Seaghdha CM</span>, <span class="author">Wu H</span>, <span class="author">Yang Q</span>, <span class="author">Kapur K</span>, <span class="author">Guessous I</span>, <span class="author">Zuber AM</span>, <span class="author">K&ouml;ttgen A</span>, <span class="author">Stoudmann C</span>, <span class="author">Teumer A</span>, <span class="author">Kutalik Z</span>, <span class="author">Mangino M</span>, <span class="author">Dehghan A</span>, <span class="author">Zhang W</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Li G</span>, <span class="author">Tanaka T</span>, <span class="author">Portas L</span>, <span class="author">Lopez LM</span>, <span class="author">Hayward C</span>, <span class="author">Lohman K</span>, <span class="author">Matsuda K</span>, <span class="author">Padmanabhan S</span>, <span class="author">Firsov D</span>, <span class="author">Sorice R</span>, <span class="author">Ulivi S</span>, <span class="author">Brockhaus AC</span>, <span class="author">Kleber ME</span>, <span class="author">Mahajan A</span>, <span class="author">Ernst FD</span>, <span class="author">Gudnason V</span>, <span class="author">Launer LJ</span>, <span class="author">Mace A</span>, <span class="author">Boerwinckle E</span>, <span class="author">Arking DE</span>, <span class="author">Tanikawa C</span>, <span class="author">Nakamura Y</span>, <span class="author">Brown MJ</span>, <span class="author">Gaspoz JM</span>, <span class="author">Theler JM</span>, <span class="author">Siscovick DS</span>, <span class="author">Psaty BM</span>, <span class="author">Bergmann S</span>, <span class="author">Vollenweider P</span>, <span class="author">Vitart V</span>, <span class="author">Wright AF</span>, <span class="author">Zemunik T</span>, <span class="author">Boban M</span>, <span class="author">Kolcic I</span>, <span class="author">Navarro P</span>, <span class="author">Brown EM</span>, <span class="author">Estrada K</span>, <span class="author">Ding J</span>, <span class="author">Harris TB</span>, <span class="author">Bandinelli S</span>, <span class="author">Hernandez D</span>, <span class="author">Singleton AB</span>, <span class="author">Girotto G</span>, <span class="author">Ruggiero D</span>, <span class="author">d&#39;Adamo AP</span>, <span class="author">Robino A</span>, <span class="author">Meitinger T</span>, <span class="author">Meisinger C</span>, <span class="author">Davies G</span>, <span class="author">Starr JM</span>, <span class="author">Chambers JC</span>, <span class="author">Boehm BO</span>, <span class="author">Winkelmann BR</span>, <span class="author">Huang J</span>, <span class="author">Murgia F</span>, <span class="author">Wild SH</span>, <span class="author">Campbell H</span>, <span class="author">Morris AP</span>, <span class="author">Franco OH</span>, <span class="author">Hofman A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Rivadeneira F</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Hannemann A</span>, <span class="author">Biffar R</span>, <span class="author">Hoffmann W</span>, <span class="author">Shin SY</span>, <span class="author">Lescuyer P</span>, <span class="author">Henry H</span>, <span class="author">Schurmann C</span>, <span class="author">SUNLIGHT consortium</span>, <span class="author">GEFOS consortium</span>, <span class="author">Munroe PB</span>, <span class="author">Gasparini P</span>, <span class="author">Pirastu N</span>, <span class="author">Ciullo M</span>, <span class="author">Gieger C</span>, <span class="author">M&auml;rz W</span>, <span class="author">Lind L</span>, <span class="author">Spector TD</span>, <span class="author">Smith AV</span>, <span class="author">Rudan I</span>, <span class="author">Wilson JF</span>, <span class="author">Polasek O</span>, <span class="author">Deary IJ</span>, <span class="author">Pirastu M</span>, <span class="author">Ferrucci L</span>, <span class="author">Liu Y</span>, <span class="author">Kestenbaum B</span>, <span class="author">Kooner JS</span>, <span class="author">Witteman JC</span>, <span class="author">Nauck M</span>, <span class="author">Kao WH</span>, <span class="author">Wallaschofski H</span>, <span class="author">Bonny O</span>, <span class="author">Fox CS</span> and <span class="author">Bochud M</span></p>
<p class="affiliation">National Heart, Lung, and Blood Institute's Framingham Heart Study and Center for Population Studies, Framingham, Massachusetts, United States of America ; Renal Division, Massachusetts General Hospital, Boston, Massachusetts, United States of America.</p>
<p class="abstract">Calcium is vital to the normal functioning of multiple organ systems and its serum concentration is tightly regulated. Apart from CASR, the genes associated with serum calcium are largely unknown. We conducted a genome-wide association meta-analysis of 39,400 individuals from 17 population-based cohorts and investigated the 14 most strongly associated loci in ≤21,679 additional individuals. Seven loci (six new regions) in association with serum calcium were identified and replicated. Rs1570669 near CYP24A1 (P = 9.1E-12), rs10491003 upstream of GATA3 (P = 4.8E-09) and rs7481584 in CARS (P = 1.2E-10) implicate regions involved in Mendelian calcemic disorders: Rs1550532 in DGKD (P = 8.2E-11), also associated with bone density, and rs7336933 near DGKH/KIAA0564 (P = 9.1E-10) are near genes that encode distinct isoforms of diacylglycerol kinase. Rs780094 is in GCKR. We characterized the expression of these genes in gut, kidney, and bone, and demonstrate modulation of gene expression in bone in response to dietary calcium in mice. Our results shed new light on the genetics of calcium homeostasis.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">9</span>;<span class="pagerange">e1003796</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24068962">24068962</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3778004">3778004</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003796">10.1371/journal.pgen.1003796</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in BICD2 Cause Dominant Congenital Spinal Muscular Atrophy and Hereditary Spastic Paraplegia.</h4>
<p class="authors"><span class="author">Oates EC</span>, <span class="author">Rossor AM</span>, <span class="author">Hafezparast M</span>, <span class="author">Gonzalez M</span>, <span class="author">Speziani F</span>, <span class="author">Macarthur DG</span>, <span class="author">Lek M</span>, <span class="author">Cottenie E</span>, <span class="author">Scoto M</span>, <span class="author">Foley AR</span>, <span class="author">Hurles M</span>, <span class="author">Houlden H</span>, <span class="author">Greensmith L</span>, <span class="author">Auer-Grumbach M</span>, <span class="author">Pieber TR</span>, <span class="author">Strom TM</span>, <span class="author">Schule R</span>, <span class="author">Herrmann DN</span>, <span class="author">Sowden JE</span>, <span class="author">Acsadi G</span>, <span class="author">Menezes MP</span>, <span class="author">Clarke NF</span>, <span class="author">Z&uuml;chner S</span>, <span class="author">UK10K</span>, <span class="author">Muntoni F</span>, <span class="author">North KN</span> and <span class="author">Reilly MM</span></p>
<p class="affiliation">Institute for Neuroscience and Muscle Research, Children's Hospital at Westmead, Westmead, Sydney, NSW 2145, Australia; Discipline of Paediatrics and Child Health, Faculty of Medicine, The University of Sydney, Sydney, NSW 2006, Australia.</p>
<p class="abstract">Dominant congenital spinal muscular atrophy (DCSMA) is a disorder of developing anterior horn cells and shows lower-limb predominance and clinical overlap with hereditary spastic paraplegia (HSP), a lower-limb-predominant disorder of corticospinal motor neurons. We have identified four mutations in bicaudal D homolog 2 (Drosophila) (BICD2) in six kindreds affected by DCSMA, DCSMA with upper motor neuron features, or HSP. BICD2 encodes BICD2, a key adaptor protein that interacts with the dynein-dynactin motor complex, which facilitates trafficking of cellular cargos that are critical to motor neuron development and maintenance. We demonstrate that mutations resulting in amino acid substitutions in two binding regions of BICD2 increase its binding affinity for the cytoplasmic dynein-dynactin complex, which might result in the perturbation of BICD2-dynein-dynactin-mediated trafficking, and impair neurite outgrowth. These findings provide insight into the mechanism underlying both the static and the slowly progressive clinical features and the motor neuron pathology that characterize BICD2-associated diseases, and underscore the importance of the dynein-dynactin transport pathway in the development and survival of both lower and upper motor neurons.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23664120">23664120</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.04.018">10.1016/j.ajhg.2013.04.018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Getting ready for the human phenome project: the 2012 forum of the human variome project.</h4>
<p class="authors"><span class="author">Oetting WS</span>, <span class="author">Robinson PN</span>, <span class="author">Greenblatt MS</span>, <span class="author">Cotton RG</span>, <span class="author">Beck T</span>, <span class="author">Carey JC</span>, <span class="author">Doelken SC</span>, <span class="author">Girdea M</span>, <span class="author">Groza T</span>, <span class="author">Hamilton CM</span>, <span class="author">Hamosh A</span>, <span class="author">Kerner B</span>, <span class="author">Macarthur JA</span>, <span class="author">Maglott DR</span>, <span class="author">Mons B</span>, <span class="author">Rehm HL</span>, <span class="author">Schofield PN</span>, <span class="author">Searle BA</span>, <span class="author">Smedley D</span>, <span class="author">Smith CL</span>, <span class="author">Bernstein IT</span>, <span class="author">Zankl A</span> and <span class="author">Zhao EY</span></p>
<p class="affiliation">Department of Experimental and Clinical Pharmacology, College of Pharmacy, University of Minnesota, Minneapolis, Minnesota.</p>
<p class="abstract">A forum of the Human Variome Project (HVP) was held as a satellite to the 2012 Annual Meeting of the American Society of Human Genetics in San Francisco, California. The theme of this meeting was "Getting Ready for the Human Phenome Project." Understanding the genetic contribution to both rare single-gene "Mendelian" disorders and more complex common diseases will require integration of research efforts among many fields and better defined phenotypes. The HVP is dedicated to bringing together researchers and research populations throughout the world to provide the resources to investigate the impact of genetic variation on disease. To this end, there needs to be a greater sharing of phenotype and genotype data. For this to occur, many databases that currently exist will need to become interoperable to allow for the combining of cohorts with similar phenotypes to increase statistical power for studies attempting to identify novel disease genes or causative genetic variants. Improved systems and tools that enhance the collection of phenotype data from clinicians are urgently needed. This meeting begins the HVP's effort toward this important goal.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2013</span>;<span class="volume">34</span>;<span class="number">4</span>;<span class="pagerange">661-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23401191">23401191</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.22293">10.1002/humu.22293</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cereal killers.</h4>
<p class="authors"><span class="author">Otto TD</span> and <span class="author">Reid AJ</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">11</span>;<span class="pagerange">744</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24129509">24129509</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3144">10.1038/nrmicro3144</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Autologous antibody capture to enrich immunogenic viruses for viral discovery.</h4>
<p class="authors"><span class="author">Oude Munnink BB</span>, <span class="author">Jazaeri Farsani SM</span>, <span class="author">Deijs M</span>, <span class="author">Jonkers J</span>, <span class="author">Verhoeven JT</span>, <span class="author">Ieven M</span>, <span class="author">Goossens H</span>, <span class="author">de Jong MD</span>, <span class="author">Berkhout B</span>, <span class="author">Loens K</span>, <span class="author">Kellam P</span>, <span class="author">Bakker M</span>, <span class="author">Canuti M</span>, <span class="author">Cotten M</span> and <span class="author">van der Hoek L</span></p>
<p class="affiliation">Laboratory of Experimental Virology, Department of Medical Microbiology, Center for Infection and Immunity Amsterdam (CINIMA), Academic Medical Center of the University of Amsterdam, Amsterdam, The Netherlands.</p>
<p class="abstract">Discovery of new viruses has been boosted by novel deep sequencing technologies. Currently, many viruses can be identified by sequencing without knowledge of the pathogenicity of the virus. However, attributing the presence of a virus in patient material to a disease in the patient can be a challenge. One approach to meet this challenge is identification of viral sequences based on enrichment by autologous patient antibody capture. This method facilitates identification of viruses that have provoked an immune response within the patient and may increase the sensitivity of the current virus discovery techniques. To demonstrate the utility of this method, virus discovery deep sequencing (VIDISCA-454) was performed on clinical samples from 19 patients: 13 with a known respiratory viral infection and 6 with a known gastrointestinal viral infection. Patient sera was collected from one to several months after the acute infection phase. Input and antibody capture material was sequenced and enrichment was assessed. In 18 of the 19 patients, viral reads from immunogenic viruses were enriched by antibody capture (ranging between 1.5x to 343x in respiratory material, and 1.4x to 53x in stool). Enriched reads were also determined in an identity independent manner by using a novel algorithm Xcompare. In 16 of the 19 patients, 21% to 100% of the enriched reads were derived from infecting viruses. In conclusion, the technique provides a novel approach to specifically identify immunogenic viral sequences among the bulk of sequences which are usually encountered during virus discovery metagenomics.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e78454</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24223808">24223808</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3817278">3817278</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0078454">10.1371/journal.pone.0078454</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficient depletion of host DNA contamination in malaria clinical sequencing.</h4>
<p class="authors"><span class="author">Oyola SO</span>, <span class="author">Gu Y</span>, <span class="author">Manske M</span>, <span class="author">Otto TD</span>, <span class="author">O&#39;Brien J</span>, <span class="author">Alcock D</span>, <span class="author">Macinnis B</span>, <span class="author">Berriman M</span>, <span class="author">Newbold CI</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Swerdlow HP</span> and <span class="author">Quail MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%53%61%6D%75%65%6C.%6F%79%6F%6C%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#83;&#97;&#109;&#117;&#101;&#108;&#46;&#111;&#121;&#111;&#108;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The cost of whole-genome sequencing (WGS) is decreasing rapidly as next-generation sequencing technology continues to advance, and the prospect of making WGS available for public health applications is becoming a reality. So far, a number of studies have demonstrated the use of WGS as an epidemiological tool for typing and controlling outbreaks of microbial pathogens. Success of these applications is hugely dependent on efficient generation of clean genetic material that is free from host DNA contamination for rapid preparation of sequencing libraries. The presence of large amounts of host DNA severely affects the efficiency of characterizing pathogens using WGS and is therefore a serious impediment to clinical and epidemiological sequencing for health care and public health applications. We have developed a simple enzymatic treatment method that takes advantage of the methylation of human DNA to selectively deplete host contamination from clinical samples prior to sequencing. Using malaria clinical samples with over 80% human host DNA contamination, we show that the enzymatic treatment enriches Plasmodium falciparum DNA up to &sim;9-fold and generates high-quality, nonbiased sequence reads covering &gt;98% of 86,158 catalogued typeable single-nucleotide polymorphism loci.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">079355/Z/06/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">3</span>;<span class="pagerange">745-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23224084">23224084</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3592063">3592063</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.02507-12">10.1128/JCM.02507-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diversity among human non-typhoidal salmonellae isolates from Zimbabwe.</h4>
<p class="authors"><span class="author">Paglietti B</span>, <span class="author">Falchi G</span>, <span class="author">Mason P</span>, <span class="author">Chitsatso O</span>, <span class="author">Nair S</span>, <span class="author">Gwanzura L</span>, <span class="author">Uzzau S</span>, <span class="author">Cappuccinelli P</span>, <span class="author">Wain J</span> and <span class="author">Rubino S</span></p>
<p class="affiliation">Microbiologica, Dipartimento di Scienze Biomediche, Universita di Sassari, Sassari, Sardegna, Italy.</p><p class="abstract"><strong>Background</strong>: Non-typhoidal Salmonella infections are an important public health problem in sub-Saharan Africa, especially among children and HIV-seropositive patients in whom they may cause invasive disease.</p><p class="abstract"><strong>Methods</strong>: In order to better understand the epidemiology of Salmonella infections in southern Africa we typed, using serotyping, phage typing and multilocus sequence typing, 167 non-typhoidal Salmonella strains isolated from human clinical specimens during 1995-2000.</p><p class="abstract"><strong>Results</strong>: The most common serovars were Salmonella Typhimurium DT56/ST313, Salmonella Enteritidis PT4 and Salmonella Isangi ST216. Isolates of Salmonella Isangi showed a multidrug-resistant phenotype that was resistant to extended-spectrum cephalosporins. Twelve new sequence types and six new serotypes of Salmonella were identified.</p><p class="abstract"><strong>Conclusions</strong>: Given the diversity detected in the study it seems likely that many new variants of S. enterica are extant in Zimbabwe and by implication across sub-Saharan Africa. We have demonstrated the presence in Zimbabwe of a multidrug-resistant strain of the serovar Salmonella Isangi and demonstrated the diversity of Salmonella circulating in one sub-Saharan African country. Further studies on the characteristics of Salmonella Isangi isolates from Zimbabwe, including plasmid typing and genotyping, are essential if effective control of the spread of this potential pathogen in sub-Saharan Africa is to be achieved.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600805</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Transactions of the Royal Society of Tropical Medicine and Hygiene</span>&nbsp;<span class="year">2013</span>;<span class="volume">107</span>;<span class="number">8</span>;<span class="pagerange">487-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23843558">23843558</a>; DOI: <a href="http://dx.doi.org/10.1093%2Ftrstmh%2Ftrt046">10.1093/trstmh/trt046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Probing the brain of comorbidity.</h4>
<p class="authors"><span class="author">Palotie A</span>, <span class="author">Kallela M</span> and <span class="author">Anttila V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">Inherited sleep disorders might provide insights for migraine pathophysiology (Brennan et al., this issue).</p>
<p class="publication"><span class="title">Science translational medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">183</span>;<span class="pagerange">183fs15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23636091">23636091</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscitranslmed.3006229">10.1126/scitranslmed.3006229</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ectopic Expression of Activated Notch or SOX2 Reveals Similar and Unique Roles in the Development of the Sensory Cell Progenitors in the Mammalian Inner Ear.</h4>
<p class="authors"><span class="author">Pan W</span>, <span class="author">Jin Y</span>, <span class="author">Chen J</span>, <span class="author">Rottier RJ</span>, <span class="author">Steel KP</span> and <span class="author">Kiernan AE</span></p>
<p class="affiliation">Department of Ophthalmology and Department of Biomedical Genetics, University of Rochester, Rochester, New York 14642, Department of Pediatric Surgery, Erasmus MC-Sophia Children's Hospital, 3000 CA Rotterdam, the Netherlands, and Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p>
<p class="abstract">Hearing impairment or vestibular dysfunction in humans often results from a permanent loss of critical cell types in the sensory regions of the inner ear, including hair cells, supporting cells, or cochleovestibular neurons. These important cell types arise from a common sensory or neurosensory progenitor, although little is known about how these progenitors are specified. Studies have shown that Notch signaling and the transcription factor Sox2 are required for the development of these lineages. Previously we and others demonstrated that ectopic activation of Notch can direct nonsensory cells to adopt a sensory fate, indicating a role for Notch in early specification events. Here, we explore the relationship between Notch and SOX2 by ectopically activating these factors in nonsensory regions of the mouse cochlea, and demonstrate that, similar to Notch, SOX2 can specify sensory progenitors, consistent with a role downstream of Notch signaling. However, we also show that Notch has a unique role in promoting the proliferation of the sensory progenitors. We further demonstrate that Notch can only induce ectopic sensory regions within a certain time window of development, and that the ectopic hair cells display specialized stereocilia bundles similar to endogenous hair cells. These results demonstrate that Notch and SOX2 can both drive the sensory program in nonsensory cells, indicating these factors may be useful in cell replacement strategies in the inner ear.</p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2013</span>;<span class="volume">33</span>;<span class="number">41</span>;<span class="pagerange">16146-16157</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24107947">24107947</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.3150-12.2013">10.1523/JNEUROSCI.3150-12.2013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tailoring the models of transcription.</h4>
<p class="authors"><span class="author">Pance A</span></p>
<p class="affiliation">The Welcome Trust Sanger Institute, Genome Campus Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%70%39%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#112;&#57;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">Molecular biology is a rapidly evolving field that has led to the development of increasingly sophisticated technologies to improve our capacity to study cellular processes in much finer detail. Transcription is the first step in protein expression and the major point of regulation of the components that determine the characteristics, fate and functions of cells. The study of transcriptional regulation has been greatly facilitated by the development of reporter genes and transcription factor expression vectors, which have become versatile tools for manipulating promoters, as well as transcription factors in order to examine their function. The understanding of promoter complexity and transcription factor structure offers an insight into the mechanisms of transcriptional control and their impact on cell behaviour. This review focuses on some of the many applications of molecular cut-and-paste tools for the manipulation of promoters and transcription factors leading to the understanding of crucial aspects of transcriptional regulation.</p>
<p class="publication"><span class="title">International journal of molecular sciences</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">4</span>;<span class="pagerange">7583-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23567272">23567272</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3645704">3645704</a>; DOI: <a href="http://dx.doi.org/10.3390%2Fijms14047583">10.3390/ijms14047583</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Advances in osteoarthritis genetics.</h4>
<p class="authors"><span class="author">Panoutsopoulou K</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Department of Human Genetics, Wellcome Trust Sanger Institute, Cambridgeshire, UK.</p>
<p class="abstract">Osteoarthritis (OA), the most common form of arthritis, is a highly debilitating disease of the joints and can lead to severe pain and disability. There is no cure for OA. Current treatments often fail to alleviate its symptoms leading to an increased demand for joint replacement surgery. Previous epidemiological and genetic research has established that OA is a multifactorial disease with both environmental and genetic components. Over the past 6 years, a candidate gene study and several genome-wide association scans (GWAS) in populations of Asian and European descent have collectively established 15 loci associated with knee or hip OA that have been replicated with genome-wide significance, shedding some light on the aetiogenesis of the disease. All OA associated variants to date are common in frequency and appear to confer moderate to small effect sizes. Some of the associated variants are found within or near genes with clear roles in OA pathogenesis, whereas others point to unsuspected, less characterised pathways. These studies have also provided further evidence in support of the existence of ethnic, sex, and joint specific effects in OA and have highlighted the importance of expanded and more homogeneous phenotype definitions in genetic studies of OA.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23868913">23868913</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmedgenet-2013-101754">10.1136/jmedgenet-2013-101754</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In search of low-frequency and rare variants affecting complex traits.</h4>
<p class="authors"><span class="author">Panoutsopoulou K</span>, <span class="author">Tachmazidou I</span> and <span class="author">Zeggini E</span></p><p class="abstract">The allelic architecture of complex traits is likely to be underpinned by a combination of multiple common frequency and rare variants. Targeted genotyping arrays and next-generation sequencing technologies at the whole-genome sequencing (WGS) and whole-exome scales (WES) are increasingly employed to access sequence variation across the full minor allele frequency (MAF) spectrum. Different study design strategies that make use of diverse technologies, imputation and sample selection approaches are an active target of development and evaluation efforts. Initial insights into the contribution of rare variants in common diseases and medically relevant quantitative traits point to low-frequency and rare alleles acting either independently or in aggregate and in several cases alongside common variants. Studies conducted in population isolates have been successful in detecting rare variant associations with complex phenotypes. Statistical methodologies that enable the joint analysis of rare variants across regions of the genome continue to evolve with current efforts focusing on incorporating information such as functional annotation, and on the meta-analysis of these burden tests. In addition, population stratification, defining genome-wide statistical significance thresholds and the design of appropriate replication experiments constitute important considerations for the powerful analysis and interpretation of rare variant association studies. Progress in addressing these emerging challenges and the accrual of sufficiently large data sets are poised to help the field of complex trait genetics enter a promising era of discovery.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">19542</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">R1</span>;<span class="pagerange">R16-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23922232">23922232</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3782074">3782074</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddt376">10.1093/hmg/ddt376</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study for osteoarthritis</h4>
<p class="authors"><span class="author">PANOUTSOPOULOU,K.;</span>, <span class="author">SOUTHAM,L.;</span> and <span class="author">ZEGGINI,E.;</span></p>
<p class="publication"><span class="title">The Lancet</span>&nbsp;<span class="year">2013</span>;<span class="volume">381</span>;<span class="number">9864</span>;<span class="pagerange">373</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2813%2960168-3">10.1016/S0140-6736(13)60168-3</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/S0140673613601683" href="http://www.sciencedirect.com/science/article/pii/S0140673613601683">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/S0140673613</span>601683</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clinical and biological implications of driver mutations in myelodysplastic syndromes.</h4>
<p class="authors"><span class="author">Papaemmanuil E</span>, <span class="author">Gerstung M</span>, <span class="author">Malcovati L</span>, <span class="author">Tauro S</span>, <span class="author">Gundem G</span>, <span class="author">Van Loo P</span>, <span class="author">Yoon CJ</span>, <span class="author">Ellis P</span>, <span class="author">Wedge DC</span>, <span class="author">Pellagatti A</span>, <span class="author">Shlien A</span>, <span class="author">Groves MJ</span>, <span class="author">Forbes SA</span>, <span class="author">Raine K</span>, <span class="author">Hinton J</span>, <span class="author">Mudie LJ</span>, <span class="author">McLaren S</span>, <span class="author">Hardy C</span>, <span class="author">Latimer C</span>, <span class="author">Della Porta MG</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Ambaglio I</span>, <span class="author">Galli A</span>, <span class="author">Butler AP</span>, <span class="author">Waldin G</span>, <span class="author">Teague JW</span>, <span class="author">Quek L</span>, <span class="author">Sternberg A</span>, <span class="author">Gambacorti-Passerini C</span>, <span class="author">Cross NC</span>, <span class="author">Green AR</span>, <span class="author">Boultwood J</span>, <span class="author">Vyas P</span>, <span class="author">Hellstrom-Lindberg E</span>, <span class="author">Bowen D</span>, <span class="author">Cazzola M</span>, <span class="author">Stratton MR</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, United Kingdom;</p>
<p class="abstract">Myelodysplastic syndromes (MDS) are a heterogeneous group of chronic hematological malignancies characterized by dysplasia, ineffective hematopoiesis and a variable risk of progression to acute myeloid leukemia. Sequencing of MDS genomes has identified mutations in genes implicated in RNA splicing, DNA modification, chromatin regulation and cell signaling. We sequenced 111 genes across 738 patients with MDS or closely related neoplasms (including CMML and MDS-MPN) to explore the role of acquired mutations in MDS biology and clinical phenotype. 78% patients had one or more oncogenic mutations. We identify complex patterns of pairwise association between genes, indicative of epistatic interactions involving components of the spliceosome machinery and epigenetic modifiers. Coupled with inferences on subclonal mutations, these data suggest a hypothesis of genetic 'predestination', in which early driver mutations, typically affecting genes involved in RNA splicing, dictate future trajectories of disease evolution with distinct clinical phenotypes. Driver mutations had equivalent prognostic significance whether clonal or subclonal, and leukemia-free survival deteriorated steadily as numbers of driver mutations increased. Thus, analysis of oncogenic mutations in large, well-characterized cohorts of patients illustrates the interconnections between the cancer genome and disease biology, with considerable potential for clinical application.</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24030381">24030381</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2013-08-518886">10.1182/blood-2013-08-518886</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A member of the Plasmodium falciparum PHIST family binds to the erythrocyte cytoskeleton component band 4.1.</h4>
<p class="authors"><span class="author">Parish LA</span>, <span class="author">Mai DW</span>, <span class="author">Jones ML</span>, <span class="author">Kitson EL</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">William C Gorgas Center for Geographic Medicine, Division of Infectious Diseases, Department of Medicine, University of Alabama at Birmingham, 845 19th St, South, Birmingham, AL, 35294-2170, USA. <a href="mailto:%6A%75%6C%69%61%6E.%72%61%79%6E%65%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#117;&#108;&#105;&#97;&#110;&#46;&#114;&#97;&#121;&#110;&#101;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: Plasmodium falciparum parasites export more than 400 proteins into the cytosol of their host erythrocytes. These exported proteins catalyse the formation of knobs on the erythrocyte plasma membrane and an overall increase in erythrocyte rigidity, presumably by modulating the endogenous erythrocyte cytoskeleton. In uninfected erythrocytes, Band 4.1 (4.1R) plays a key role in regulating erythrocyte shape by interacting with multiple proteins through the three lobes of its cloverleaf-shaped N-terminal domain. In P. falciparum-infected erythrocytes, the C-lobe of 4.1R interacts with the P. falciparum protein mature parasite-infected erythrocyte surface antigen (MESA), but it is not currently known whether other P. falciparum proteins bind to other lobes of the 4.1R N-terminal domain.

Methods: In order to identify novel 4.1R interacting proteins, a yeast two-hybrid screen was performed with a fragment of 4.1R containing both the N- and α-lobes. Positive interactions were confirmed and investigated using site-directed mutagenesis, and antibodies were raised against the interacting partner to characterise it's expression and distribution in P. falciparum infected erythrocytes.

Results: Yeast two-hybrid screening identified a positive interaction between the 4.1R N- and α-lobes and PF3D7_0402000. PF3D7_0402000 is a member of a large family of exported proteins that share a domain of unknown function, the PHIST domain. Domain mapping and site-directed mutagenesis established that it is the PHIST domain of PF3D7_0402000 that interacts with 4.1R. Native PF3D7_0402000 is localized at the parasitophorous vacuole membrane (PVM), and colocalizes with a subpopulation of 4.1R.

Discussion: The function of the majority of P. falciparum exported proteins, including most members of the PHIST family, is unknown, and in only a handful of cases has a direct interaction between P. falciparum-exported proteins and components of the erythrocyte cytoskeleton been established. The interaction between 4.1R and PF3D7_0402000, and localization of PF3D7_0402000 with a sub-population of 4.1R at the PVM could indicate a role in modulating PVM structure. Further investigation into the mechanisms for 4.1R recruitment is needed.

Conclusion: PF3D7_0402000 was identified as a new binding partner for the major erythrocyte cytoskeletal protein, 4.1R. This interaction is consistent with a growing body of literature that suggests the PHIST family members function by interacting directly with erythrocyte proteins.</p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="pagerange">160</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23663475">23663475</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3658886">3658886</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-12-160">10.1186/1475-2875-12-160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">What has high-throughput sequencing ever done for us?</h4>
<p class="authors"><span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">This month&#39;s Genome Watch looks back over the past 10 years and highlights how the incredible advances in sequencing technologies have transformed research into microbial genomes.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">664-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23979431">23979431</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3112">10.1038/nrmicro3112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proteomic and Genetic Analyses Demonstrate that Plasmodium berghei Blood Stages Export a Large and Diverse Repertoire of Proteins.</h4>
<p class="authors"><span class="author">Pasini EM</span>, <span class="author">Braks JA</span>, <span class="author">Fonager J</span>, <span class="author">Klop O</span>, <span class="author">Aime E</span>, <span class="author">Spaccapelo R</span>, <span class="author">Otto TD</span>, <span class="author">Berriman M</span>, <span class="author">Hiss JA</span>, <span class="author">Thomas AW</span>, <span class="author">Mann M</span>, <span class="author">Janse CJ</span>, <span class="author">Kocken CH</span> and <span class="author">Franke-Fayard B</span></p>
<p class="affiliation">â€¡Biomedical Primate Research Centre, 2288 GJ Rijswijk, The Netherlands;</p>
<p class="abstract">Malaria parasites actively remodel the infected red blood cell (irbc) by exporting proteins into the host cell cytoplasm. The human parasite Plasmodium falciparum exports particularly large numbers of proteins, including proteins that establish a vesicular network allowing the trafficking of proteins onto the surface of irbcs that are responsible for tissue sequestration. Like P. falciparum, the rodent parasite P. berghei ANKA sequesters via irbc interactions with the host receptor CD36. We have applied proteomic, genomic, and reverse-genetic approaches to identify P. berghei proteins potentially involved in the transport of proteins to the irbc surface. A comparative proteomics analysis of P. berghei non-sequestering and sequestering parasites was used to determine changes in the irbc membrane associated with sequestration. Subsequent tagging experiments identified 13 proteins (Plasmodium export element (PEXEL)-positive as well as PEXEL-negative) that are exported into the irbc cytoplasm and have distinct localization patterns: a dispersed and/or patchy distribution, a punctate vesicle-like pattern in the cytoplasm, or a distinct location at the irbc membrane. Members of the PEXEL-negative BIR and PEXEL-positive Pb-fam-3 show a dispersed localization in the irbc cytoplasm, but not at the irbc surface. Two of the identified exported proteins are transported to the irbc membrane and were named erythrocyte membrane associated proteins. EMAP1 is a member of the PEXEL-negative Pb-fam-1 family, and EMAP2 is a PEXEL-positive protein encoded by a single copy gene; neither protein plays a direct role in sequestration. Our observations clearly indicate that P. berghei traffics a diverse range of proteins to different cellular locations via mechanisms that are analogous to those employed by P. falciparum. This information can be exploited to generate transgenic humanized rodent P. berghei parasites expressing chimeric P. berghei/P. falciparum proteins on the surface of rodent irbc, thereby opening new avenues for in vivo screening adjunct therapies that block sequestration.</p>
<p class="publication"><span class="title">Molecular &amp;amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2013</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">426-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23197789">23197789</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3567864">3567864</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M112.021238">10.1074/mcp.M112.021238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Incidence and Characterisation of Methicillin-Resistant Staphylococcus aureus (MRSA) from Nasal Colonisation in Participants Attending a Cattle Veterinary Conference in the UK.</h4>
<p class="authors"><span class="author">Paterson GK</span>, <span class="author">Harrison EM</span>, <span class="author">Craven EF</span>, <span class="author">Petersen A</span>, <span class="author">Larsen AR</span>, <span class="author">Ellington MJ</span>, <span class="author">T&ouml;r&ouml;k ME</span>, <span class="author">Peacock SJ</span>, <span class="author">Parkhill J</span>, <span class="author">Zadoks RN</span> and <span class="author">Holmes MA</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Madingley Road, Cambridge, United Kingdom.</p>
<p class="abstract">We sought to determine the prevalence of nasal colonisation with methicillin-resistant Staphylococcus aureus among cattle veterinarians in the UK. There was particular interest in examining the frequency of colonisation with MRSA harbouring mecC, as strains with this mecA homologue were originally identified in bovine milk and may represent a zoonotic risk to those in contact with dairy livestock. Three hundred and seven delegates at the British Cattle Veterinarian Association (BCVA) Congress 2011 in Southport, UK were screening for nasal colonisation with MRSA. Isolates were characterised by whole genome sequencing and antimicrobial susceptibility testing. Eight out of three hundred and seven delegates (2.6%) were positive for nasal colonisation with MRSA. All strains were positive for mecA and none possessed mecC. The time since a delegate's last visit to a farm was significantly shorter in the MRSA-positive group than in MRSA-negative counterparts. BCVA delegates have an increased risk of MRSA colonisation compared to the general population but their frequency of colonisation is lower than that reported from other types of veterinarian conference, and from that seen in human healthcare workers. The results indicate that recent visitation to a farm is a risk factor for MRSA colonisation and that mecC-MRSA are rare among BCVA delegates (<1% based on sample size). Contact with livestock, including dairy cattle, may still be a risk factor for human colonisation with mecC-MRSA but occurs at a rate below the lower limit of detection available in this study.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">e68463</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23869220">23869220</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3711812">3711812</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0068463">10.1371/journal.pone.0068463</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A sequence-based variation map of zebrafish.</h4>
<p class="authors"><span class="author">Patowary A</span>, <span class="author">Purkanti R</span>, <span class="author">Singh M</span>, <span class="author">Chauhan R</span>, <span class="author">Singh AR</span>, <span class="author">Swarnkar M</span>, <span class="author">Singh N</span>, <span class="author">Pandey V</span>, <span class="author">Torroja C</span>, <span class="author">Clark MD</span>, <span class="author">Kocher JP</span>, <span class="author">Clark KJ</span>, <span class="author">Stemple DL</span>, <span class="author">Klee EW</span>, <span class="author">Ekker SC</span>, <span class="author">Scaria V</span> and <span class="author">Sivasubbu S</span></p>
<p class="affiliation">1 CSIR- Institute of Genomics and Integrative Biology (CSIR-IGIB), Delhi, India .</p>
<p class="abstract">Abstract Zebrafish (Danio rerio) is a popular vertebrate model organism largely deployed using outbred laboratory animals. The nonisogenic nature of the zebrafish as a model system offers the opportunity to understand natural variations and their effect in modulating phenotype. In an effort to better characterize the range of natural variation in this model system and to complement the zebrafish reference genome project, the whole genome sequence of a wild zebrafish at 39-fold genome coverage was determined. Comparative analysis with the zebrafish reference genome revealed approximately 5.2 million single nucleotide variations and over 1.6 million insertion-deletion variations. This dataset thus represents a new catalog of genetic variations in the zebrafish genome. Further analysis revealed selective enrichment for variations in genes involved in immune function and response to the environment, suggesting genome-level adaptations to environmental niches. We also show that human disease gene orthologs in the sequenced wild zebrafish genome show a lower ratio of nonsynonymous to synonymous single nucleotide variations.</p>
<p class="publication"><span class="title">Zebrafish</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">1</span>;<span class="pagerange">15-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23590399">23590399</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3629779">3629779</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fzeb.2012.0848">10.1089/zeb.2012.0848</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The cell-cycle state of stem cells determines cell fate propensity.</h4>
<p class="authors"><span class="author">Pauklin S</span> and <span class="author">Vallier L</span></p>
<p class="affiliation">Wellcome Trust-Medical Research Council Cambridge Stem Cell Institute, Anne McLaren Laboratory for Regenerative Medicine and Department of Surgery, University of Cambridge, Cambridge CB2 0SZ, UK. Electronic address: <a href="mailto:%73%70%35%37%39%40%63%61%6D.%61%63.%75%6B">&#115;&#112;&#53;&#55;&#57;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">Self-renewal and differentiation of stem cells are fundamentally associated with cell-cycle progression to enable tissue specification, organ homeostasis, and potentially tumorigenesis. However, technical challenges have impaired the study of the molecular interactions coordinating cell fate choice and cell-cycle progression. Here, we bypass these limitations by using the FUCCI reporter system in human pluripotent stem cells and show that their capacity of differentiation varies during the progression of their cell cycle. These mechanisms are governed by the cell-cycle regulators cyclin D1-3 that control differentiation signals such as the TGF-&beta;-Smad2/3 pathway. Conversely, cell-cycle manipulation using a small molecule directs differentiation of hPSCs and provides an approach to generate cell types with a clinical interest. Our results demonstrate that cell fate decisions are tightly associated with the cell-cycle machinery and reveal insights in the mechanisms synchronizing differentiation and proliferation in developing tissues.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701448</span>; Wellcome Trust: <span class="grant">079249</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">155</span>;<span class="number">1</span>;<span class="pagerange">135-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24074866">24074866</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3898746">3898746</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.08.031">10.1016/j.cell.2013.08.031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maps of open chromatin highlight cell type-restricted patterns of regulatory sequence variation at hematological trait loci.</h4>
<p class="authors"><span class="author">Paul DS</span>, <span class="author">Albers CA</span>, <span class="author">Rendon A</span>, <span class="author">Voss K</span>, <span class="author">Stephens J</span>, <span class="author">HaemGen Consortium</span>, <span class="author">van der Harst P</span>, <span class="author">Chambers JC</span>, <span class="author">Soranzo N</span>, <span class="author">Ouwehand WH</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%64.%70%61%75%6C%40%75%63%6C.%61%63.%75%6B">&#100;&#46;&#112;&#97;&#117;&#108;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Nearly three-quarters of the 143 genetic signals associated with platelet and erythrocyte phenotypes identified by meta-analyses of genome-wide association (GWA) studies are located at non-protein-coding regions. Here, we assessed the role of candidate regulatory variants associated with cell type-restricted, closely related hematological quantitative traits in biologically relevant hematopoietic cell types. We used formaldehyde-assisted isolation of regulatory elements followed by next-generation sequencing (FAIRE-seq) to map regions of open chromatin in three primary human blood cells of the myeloid lineage. In the precursors of platelets and erythrocytes, as well as in monocytes, we found that open chromatin signatures reflect the corresponding hematopoietic lineages of the studied cell types and associate with the cell type-specific gene expression patterns. Dependent on their signal strength, open chromatin regions showed correlation with promoter and enhancer histone marks, distance to the transcription start site, and ontology classes of nearby genes. Cell type-restricted regions of open chromatin were enriched in sequence variants associated with hematological indices. The majority (63.6%) of such candidate functional variants at platelet quantitative trait loci (QTLs) coincided with binding sites of five transcription factors key in regulating megakaryopoiesis. We experimentally tested 13 candidate regulatory variants at 10 platelet QTLs and found that 10 (76.9%) affected protein binding, suggesting that this is a frequent mechanism by which regulatory variants influence quantitative trait levels. Our findings demonstrate that combining large-scale GWA data with open chromatin profiles of relevant cell types can be a powerful means of dissecting the genetic architecture of closely related quantitative traits.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/12/28096</span>; Wellcome Trust: <span class="grant">097117</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">7</span>;<span class="pagerange">1130-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23570689">23570689</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3698506">3698506</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.155127.113">10.1101/gr.155127.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meander: visually exploring the structural variome using space-filling curves.</h4>
<p class="authors"><span class="author">Pavlopoulos GA</span>, <span class="author">Kumar P</span>, <span class="author">Sifrim A</span>, <span class="author">Sakai R</span>, <span class="author">Lin ML</span>, <span class="author">Voet T</span>, <span class="author">Moreau Y</span> and <span class="author">Aerts J</span></p>
<p class="affiliation">Department of Electrical Engineering (ESAT/SCD), University of Leuven, Kasteelpark Arenberg 10, Box 2446, 3001 Leuven, Belgium, iMinds Future Health Department, University of Leuven, Kasteelpark Arenberg 10, Box 2446, 3001 Leuven, Belgium, Division of Basic Sciences, University of Crete, Medical School, Heraklion, 71110 Crete, Greece, Laboratory of Reproductive Genomics, Department of Human Genetics, University of Leuven, Herestraat 49, 3000 Leuven, Belgium and Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton - Cambridge, CB10 1SA, UK.</p>
<p class="abstract">The introduction of next generation sequencing methods in genome studies has made it possible to shift research from a gene-centric approach to a genome wide view. Although methods and tools to detect single nucleotide polymorphisms are becoming more mature, methods to identify and visualize structural variation (SV) are still in their infancy. Most genome browsers can only compare a given sequence to a reference genome; therefore, direct comparison of multiple individuals still remains a challenge. Therefore, the implementation of efficient approaches to explore and visualize SVs and directly compare two or more individuals is desirable. In this article, we present a visualization approach that uses space-filling Hilbert curves to explore SVs based on both read-depth and pair-end information. An interactive open-source Java application, called Meander, implements the proposed methodology, and its functionality is demonstrated using two cases. With Meander, users can explore variations at different levels of resolution and simultaneously compare up to four different individuals against a common reference. The application was developed using Java version 1.6 and Processing.org and can be run on any platform. It can be found at http://homes.esat.kuleuven.be/âˆ¼bioiuser/meander.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23605045">23605045</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkt254">10.1093/nar/gkt254</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">KSR2 mutations are associated with obesity, insulin resistance, and impaired cellular fuel oxidation.</h4>
<p class="authors"><span class="author">Pearce LR</span>, <span class="author">Atanassova N</span>, <span class="author">Banton MC</span>, <span class="author">Bottomley B</span>, <span class="author">van der Klaauw AA</span>, <span class="author">Revelli JP</span>, <span class="author">Hendricks A</span>, <span class="author">Keogh JM</span>, <span class="author">Henning E</span>, <span class="author">Doree D</span>, <span class="author">Jeter-Jones S</span>, <span class="author">Garg S</span>, <span class="author">Bochukova EG</span>, <span class="author">Bounds R</span>, <span class="author">Ashford S</span>, <span class="author">Gayton E</span>, <span class="author">Hindmarsh PC</span>, <span class="author">Shield JP</span>, <span class="author">Crowne E</span>, <span class="author">Barford D</span>, <span class="author">Wareham NJ</span>, <span class="author">UK10K consortium</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Murphy MP</span>, <span class="author">Powell DR</span>, <span class="author">Barroso I</span> and <span class="author">Farooqi IS</span></p><p class="abstract">Kinase suppressor of Ras 2 (KSR2) is an intracellular scaffolding protein involved in multiple signaling pathways. Targeted deletion of Ksr2 leads to obesity in mice, suggesting a role in energy homeostasis. We explored the role of KSR2 in humans by sequencing 2,101 individuals with severe early-onset obesity and 1,536 controls. We identified multiple rare variants in KSR2 that disrupt signaling through the Raf-MEKERK pathway and impair cellular fatty acid oxidation and glucose oxidation in transfected cells; effects that can be ameliorated by the commonly prescribed antidiabetic drug, metformin. Mutation carriers exhibit hyperphagia in childhood, low heart rate, reduced basal metabolic rate and severe insulin resistance. These data establish KSR2 as an important regulator of energy intake, energy expenditure, and substrate utilization in humans. Modulation of KSR2-mediated effects may represent a novel therapeutic strategy for obesity and type 2 diabetes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">14109</span>; Department of Health: <span class="grant">NF-SI-0512-10135</span>; Medical Research Council: <span class="grant">G0502115</span>, <span class="grant">G0900554</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_UU_12012/5/B</span>, <span class="grant">MC_UU_12015/1</span>; Wellcome Trust: <span class="grant">077016/Z/05/Z</span>, <span class="grant">096106/Z/11/Z</span>, <span class="grant">098497</span>, <span class="grant">098497/Z/12/Z</span>, <span class="grant">WT091310</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">155</span>;<span class="number">4</span>;<span class="pagerange">765-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24209692">24209692</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3898740">3898740</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.09.058">10.1016/j.cell.2013.09.058</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Continent-wide panmixia of an African fruit bat facilitates transmission of potentially zoonotic viruses.</h4>
<p class="authors"><span class="author">Peel AJ</span>, <span class="author">Sargan DR</span>, <span class="author">Baker KS</span>, <span class="author">Hayman DT</span>, <span class="author">Barr JA</span>, <span class="author">Crameri G</span>, <span class="author">Suu-Ire R</span>, <span class="author">Broder CC</span>, <span class="author">Lembo T</span>, <span class="author">Wang LF</span>, <span class="author">Fooks AR</span>, <span class="author">Rossiter SJ</span>, <span class="author">Wood JL</span> and <span class="author">Cunningham AA</span></p>
<p class="affiliation">1] Department of Veterinary Medicine, University of Cambridge, Cambridge CB3 0ES, UK [2] Institute of Zoology, Zoological Society of London, Regent's Park, London NW1 4RY, UK.</p>
<p class="abstract">The straw-coloured fruit bat, Eidolon helvum, is Africa's most widely distributed and commonly hunted fruit bat, often living in close proximity to human populations. This species has been identified as a reservoir of potentially zoonotic viruses, but uncertainties remain regarding viral transmission dynamics and mechanisms of persistence. Here we combine genetic and serological analyses of populations across Africa, to determine the extent of epidemiological connectivity among E. helvum populations. Multiple markers reveal panmixia across the continental range, at a greater geographical scale than previously recorded for any other mammal, whereas populations on remote islands were genetically distinct. Multiple serological assays reveal antibodies to henipaviruses and Lagos bat virus in all locations, including small isolated island populations, indicating that factors other than population size and connectivity may be responsible for viral persistence. Our findings have potentially important public health implications, and highlight a need to avoid disturbances that may precipitate viral spillover.</p>
<p class="publication"><span class="title">Nature communications</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="pagerange">2770</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24253424">24253424</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836177">3836177</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncomms3770">10.1038/ncomms3770</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variants associated with warfarin dose in African-American individuals: a genome-wide association study.</h4>
<p class="authors"><span class="author">Perera MA</span>, <span class="author">Cavallari LH</span>, <span class="author">Limdi NA</span>, <span class="author">Gamazon ER</span>, <span class="author">Konkashbaev A</span>, <span class="author">Daneshjou R</span>, <span class="author">Pluzhnikov A</span>, <span class="author">Crawford DC</span>, <span class="author">Wang J</span>, <span class="author">Liu N</span>, <span class="author">Tatonetti N</span>, <span class="author">Bourgeois S</span>, <span class="author">Takahashi H</span>, <span class="author">Bradford Y</span>, <span class="author">Burkley BM</span>, <span class="author">Desnick RJ</span>, <span class="author">Halperin JL</span>, <span class="author">Khalifa SI</span>, <span class="author">Langaee TY</span>, <span class="author">Lubitz SA</span>, <span class="author">Nutescu EA</span>, <span class="author">Oetjens M</span>, <span class="author">Shahin MH</span>, <span class="author">Patel SR</span>, <span class="author">Sagreiya H</span>, <span class="author">Tector M</span>, <span class="author">Weck KE</span>, <span class="author">Rieder MJ</span>, <span class="author">Scott SA</span>, <span class="author">Wu AH</span>, <span class="author">Burmester JK</span>, <span class="author">Wadelius M</span>, <span class="author">Deloukas P</span>, <span class="author">Wagner MJ</span>, <span class="author">Mushiroda T</span>, <span class="author">Kubo M</span>, <span class="author">Roden DM</span>, <span class="author">Cox NJ</span>, <span class="author">Altman RB</span>, <span class="author">Klein TE</span>, <span class="author">Nakamura Y</span> and <span class="author">Johnson JA</span></p>
<p class="affiliation">Section of Genetic Medicine, Department of Medicine, University of Chicago, IL, USA.</p>
<p class="abstract">BACKGROUND: VKORC1 and CYP2C9 are important contributors to warfarin dose variability, but explain less variability for individuals of African descent than for those of European or Asian descent. We aimed to identify additional variants contributing to warfarin dose requirements in African Americans. METHODS: We did a genome-wide association study of discovery and replication cohorts. Samples from African-American adults (aged ≥18 years) who were taking a stable maintenance dose of warfarin were obtained at International Warfarin Pharmacogenetics Consortium (IWPC) sites and the University of Alabama at Birmingham (Birmingham, AL, USA). Patients enrolled at IWPC sites but who were not used for discovery made up the independent replication cohort. All participants were genotyped. We did a stepwise conditional analysis, conditioning first for VKORC1 -1639G→A, followed by the composite genotype of CYP2C9*2 and CYP2C9*3. We prespecified a genome-wide significance threshold of p<5×10(-8) in the discovery cohort and p<0·0038 in the replication cohort. FINDINGS: The discovery cohort contained 533 participants and the replication cohort 432 participants. After the prespecified conditioning in the discovery cohort, we identified an association between a novel single nucleotide polymorphism in the CYP2C cluster on chromosome 10 (rs12777823) and warfarin dose requirement that reached genome-wide significance (p=1·51×10(-8)). This association was confirmed in the replication cohort (p=5·04×10(-5)); analysis of the two cohorts together produced a p value of 4·5×10(-12). Individuals heterozygous for the rs12777823 A allele need a dose reduction of 6·92 mg/week and those homozygous 9·34 mg/week. Regression analysis showed that the inclusion of rs12777823 significantly improves warfarin dose variability explained by the IWPC dosing algorithm (21% relative improvement). INTERPRETATION: A novel CYP2C single nucleotide polymorphism exerts a clinically relevant effect on warfarin dose in African Americans, independent of CYP2C9*2 and CYP2C9*3. Incorporation of this variant into pharmacogenetic dosing algorithms could improve warfarin dose prediction in this population. FUNDING: National Institutes of Health, American Heart Association, Howard Hughes Medical Institute, Wisconsin Network for Health Research, and the Wellcome Trust.</p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23755828">23755828</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2813%2960681-9">10.1016/S0140-6736(13)60681-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Computational proteomics pitfalls and challenges: HavanaBioinfo 2012 Workshop report.</h4>
<p class="authors"><span class="author">Perez-Riverol Y</span>, <span class="author">Hermjakob H</span>, <span class="author">Kohlbacher O</span>, <span class="author">Martens L</span>, <span class="author">Creasy D</span>, <span class="author">Cox J</span>, <span class="author">Leprevost F</span>, <span class="author">Shan BP</span>, <span class="author">P&eacute;rez-Nueno VI</span>, <span class="author">Blazejczyk M</span>, <span class="author">Punta M</span>, <span class="author">Vierlinger K</span>, <span class="author">Valiente PA</span>, <span class="author">Leon K</span>, <span class="author">Chinea G</span>, <span class="author">Guirola O</span>, <span class="author">Bringas R</span>, <span class="author">Cabrera G</span>, <span class="author">Guillen G</span>, <span class="author">Padron G</span>, <span class="author">Gonzalez LJ</span> and <span class="author">Besada V</span></p>
<p class="affiliation">Center for Genetic Engineering and Biotechnology, Ave 31 e/158 y 190, Cubanacán, Playa, Havana, Cuba; European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. Electronic address: <a href="mailto:%79%61%73%73%65%74.%70%65%72%65%7A%40%62%69%6F%63%6F%6D%70.%63%69%67%62.%65%64%75.%63%75">&#121;&#97;&#115;&#115;&#101;&#116;&#46;&#112;&#101;&#114;&#101;&#122;&#64;&#98;&#105;&#111;&#99;&#111;&#109;&#112;&#46;&#99;&#105;&#103;&#98;&#46;&#101;&#100;&#117;&#46;&#99;&#117;</a>.</p>
<p class="abstract">The workshop "Bioinformatics for Biotechnology Applications (HavanaBioinfo 2012)", held December 8-11, 2012 in Havana, aimed at exploring new bioinformatics tools and approaches for large-scale proteomics, genomics and chemoinformatics. Major conclusions of the workshop include the following: (i) development of new applications and bioinformatics tools for proteomic repository analysis is crucial; current proteomic repositories contain enough data (spectra/identifications) that can be used to increase the annotations in protein databases and to generate new tools for protein identification; (ii) spectral libraries, de novo sequencing and database search tools should be combined to increase the number of protein identifications; (iii) protein probabilities and FDR are not yet sufficiently mature; (iv) computational proteomics software needs to become more intuitive; and at the same time appropriate education and training should be provided to help in the efficient exchange of knowledge between mass spectrometrists and experimental biologists and bioinformaticians in order to increase their bioinformatics background, especially statistics knowledge.</p>
<p class="publication"><span class="title">Journal of proteomics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23376229">23376229</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jprot.2013.01.019">10.1016/j.jprot.2013.01.019</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Automatic event detection within thrombus formation based on integer programming</h4>
<p class="authors"><span class="author">Peter,L.</span>, <span class="author">Pauly,O.</span>, <span class="author">Jansen,S.B.G.</span>, <span class="author">Smethurst,P.A.</span>, <span class="author">Ouwehand,W.H.</span> and <span class="author">Navab,N.</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science </span>&nbsp;<span class="year">2013</span>;<span class="volume">7766</span>;<span class="pagerange">215-24</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-36620-8_21">10.1007/978-3-642-36620-8_21</a>; URL: <a title="http://link.springer.com/chapter/10.1007%2F978-3-642-36620-8_21" href="http://link.springer.com/chapter/10.1007%2F978-3-642-36620-8_21">http://link.springer.com/chapter<span class="print-hide">...</span><span class="print-show">/10.1007%2F978-3-642-36</span>620-8_21</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recombination-mediated genetic engineering of Plasmodium berghei DNA.</h4>
<p class="authors"><span class="author">Pfander C</span>, <span class="author">Anar B</span>, <span class="author">Brochet M</span>, <span class="author">Rayner JC</span> and <span class="author">Billker O</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">DNA of Plasmodium berghei is difficult to manipulate in Escherichia coli by conventional restriction and ligation methods due to its high content of adenine and thymine (AT) nucleotides. This limits our ability to clone large genes and to generate complex vectors for modifying the parasite genome. We here describe a protocol for using lambda Red recombinase to modify inserts of a P. berghei genomic DNA library constructed in a linear, low-copy, phage-derived vector. The method uses primer extensions of 50 bp, which provide sufficient homology for an antibiotic resistance marker to recombine efficiently with a P. berghei genomic DNA insert in E. coli. In a subsequent in vitro Gateway reaction the bacterial marker is replaced with a cassette for selection in P. berghei. The insert is then released and used for transfection. The basic techniques we describe here can be adapted to generate highly efficient vectors for gene deletion, tagging, targeted mutagenesis, or genetic complementation with larger genomic regions.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span></p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">923</span>;<span class="pagerange">127-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22990774">22990774</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-62703-026-7_8">10.1007/978-1-62703-026-7_8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of Salmonella enterica Serovar Typhi Genotypes by Use of Rapid Multiplex Ligation-Dependent Probe Amplification.</h4>
<p class="authors"><span class="author">Pham Thanh D</span>, <span class="author">Tran Vu Thieu N</span>, <span class="author">Tran Thuy C</span>, <span class="author">Lod&eacute;n M</span>, <span class="author">Tuin K</span>, <span class="author">Campbell JI</span>, <span class="author">Van Minh Hoang N</span>, <span class="author">Voong Vinh P</span>, <span class="author">Farrar JJ</span>, <span class="author">Holt KE</span>, <span class="author">Dougan G</span> and <span class="author">Baker S</span></p>
<p class="affiliation">The Hospital for Tropical Diseases, Wellcome Trust Major Overseas Programme, Oxford University Clinical Research Unit, Ho Chi Minh City, Vietnam.</p>
<p class="abstract">Salmonella enterica serovar Typhi, the causative agent of typhoid fever, is highly clonal and genetically conserved, making isolate subtyping difficult. We describe a standardized multiplex ligation-dependent probe amplification (MLPA) genotyping scheme targeting 11 key phylogenetic markers of the S. Typhi genome. The MLPA method demonstrated 90% concordance with single nucleotide polymorphism (SNP) typing, the gold standard for S. Typhi genotyping, and had the ability to identify isolates of the H58 haplotype, which is associated with resistance to multiple antimicrobials. Additionally, the assay permitted the detection of fluoroquinolone resistance-associated mutations in the DNA gyrase-encoding gene gyrA and the topoisomerase gene parC with a sensitivity of 100%. The MLPA methodology is simple and reliable, providing phylogenetically and phenotypically relevant genotyping information. This MLPA scheme offers a more-sensitive and interpretable alternative to the nonphylogenetic subgrouping methodologies that are currently used in reference and research laboratories in areas where typhoid is endemic.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">9</span>;<span class="pagerange">2950-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23824765">23824765</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.01010-13">10.1128/JCM.01010-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide mutagenesis screen identifies multiple genes contributing to Vi capsular expression in Salmonella Typhi.</h4>
<p class="authors"><span class="author">Pickard D</span>, <span class="author">Kingsley RA</span>, <span class="author">Hale C</span>, <span class="author">Turner K</span>, <span class="author">Sivaraman K</span>, <span class="author">Wetter M</span>, <span class="author">Langridge G</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p>
<p class="abstract">A transposon-based, genome-wide mutagenesis screen exploiting the killing activity of a lytic ViII bacteriophage was used to identify Salmonella Typhi genes that contribute to Vi polysaccharide capsule expression. Genes enriched in the screen included those within the viaB locus (tviABCDE, vexABCDE) as well as oxyR, barA/sirA and yrfF, which have not previously been associated with Vi expression. The role of these genes in Vi expression was confirmed by constructing defined null mutant derivatives of S. Typhi and these were negative for Vi expression as determined by agglutination assays with Vi-specific sera or susceptibility to Vi-targeting bacteriophage. Transcriptome analysis confirmed a reduction in expression from the viaB locus in these S. Typhi mutant derivatives and defined regulatory networks associated with Vi expression.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23316043" rel="external">23316043</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01632-12" rel="external">10.1128/JB.01632-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptional Regulation of Culex pipiens Mosquitoes by Wolbachia Influences Cytoplasmic Incompatibility.</h4>
<p class="authors"><span class="author">Pinto SB</span>, <span class="author">Stainton K</span>, <span class="author">Harris S</span>, <span class="author">Kambris Z</span>, <span class="author">Sutton ER</span>, <span class="author">Bonsall MB</span>, <span class="author">Parkhill J</span> and <span class="author">Sinkins SP</span></p>
<p class="affiliation">Peter Medawar Building for Pathogen Research and Nuffield Department of Medicine (NDM), University of Oxford, Oxford, United Kingdom ; Department of Zoology, University of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">Cytoplasmic incompatibility (CI) induced by the endosymbiont Wolbachia pipientis causes complex patterns of crossing sterility between populations of the Culex pipiens group of mosquitoes. The molecular basis of the phenotype is yet to be defined. In order to investigate what host changes may underlie CI at the molecular level, we examined the transcription of a homolog of the Drosophila melanogaster gene grauzone that encodes a zinc finger protein and acts as a regulator of female meiosis, in which mutations can cause sterility. Upregulation was observed in Wolbachia-infected C. pipiens group individuals relative to Wolbachia-cured lines and the level of upregulation differed between lines that were reproductively incompatible. Knockdown analysis of this gene using RNAi showed an effect on hatch rates in a Wolbachia infected Culex molestus line. Furthermore, in later stages of development an effect on developmental progression in CI embryos occurs in bidirectionally incompatible crosses. The genome of a wPip Wolbachia strain variant from Culex molestus was sequenced and compared with the genome of a wPip variant with which it was incompatible. Three genes in inserted or deleted regions were newly identified in the C. molestus wPip genome, one of which is a transcriptional regulator labelled wtrM. When this gene was transfected into adult Culex mosquitoes, upregulation of the grauzone homolog was observed. These data suggest that Wolbachia-mediated regulation of host gene expression is a component of the mechanism of cytoplasmic incompatibility.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">10</span>;<span class="pagerange">e1003647</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24204251">24204251</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3814344">3814344</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003647">10.1371/journal.ppat.1003647</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome Wide Association Analysis of a Founder Population Identified TAF3 as a Gene for MCHC in Humans.</h4>
<p class="authors"><span class="author">Pistis G</span>, <span class="author">Okonkwo SU</span>, <span class="author">Traglia M</span>, <span class="author">Sala C</span>, <span class="author">Shin SY</span>, <span class="author">Masciullo C</span>, <span class="author">Buetti I</span>, <span class="author">Massacane R</span>, <span class="author">Mangino M</span>, <span class="author">Thein SL</span>, <span class="author">Spector TD</span>, <span class="author">Ganesh S</span>, <span class="author">Pirastu N</span>, <span class="author">Gasparini P</span>, <span class="author">Soranzo N</span>, <span class="author">Camaschella C</span>, <span class="author">Hart D</span>, <span class="author">Green MR</span> and <span class="author">Toniolo D</span></p>
<p class="affiliation">Division of Genetics and Cell Biology, San Raffaele Research Institute and Vita Salute University, Milano, Italy.</p>
<p class="abstract">The red blood cell related traits are highly heritable but their genetics are poorly defined. Only 5-10% of the total observed variance is explained by the genetic loci found to date, suggesting that additional loci should be searched using approaches alternative to large meta analysis. GWAS (Genome Wide Association Study) for red blood cell traits in a founder population cohort from Northern Italy identified a new locus for mean corpuscular hemoglobin concentration (MCHC) in the TAF3 gene. The association was replicated in two cohorts (rs1887582, P = 4.25E-09). TAF3 encodes a transcription cofactor that participates in core promoter recognition complex, and is involved in zebrafish and mouse erythropoiesis. We show here that TAF3 is required for transcription of the SPTA1 gene, encoding alpha spectrin, one of the proteins that link the plasma membrane to the actin cytoskeleton. Mutations in SPTA1 are responsible for hereditary spherocytosis, a monogenic disorder of MCHC, as well as for the normal MCHC level. Based on our results, we propose that TAF3 is required for normal erythropoiesis in human and that it might have a role in controlling the ratio between hemoglobin (Hb) and cell volume and in the dynamics of RBC maturation in healthy individuals. Finally, TAF3 represents a potential candidate or a modifier gene for disorders of red cell membrane.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">e69206</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23935956">23935956</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3729833">3729833</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0069206">10.1371/journal.pone.0069206</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NDUFA4 Mutations Underlie Dysfunction of a Cytochrome c Oxidase Subunit Linked to Human Neurological Disease.</h4>
<p class="authors"><span class="author">Pitceathly RD</span>, <span class="author">Rahman S</span>, <span class="author">Wedatilake Y</span>, <span class="author">Polke JM</span>, <span class="author">Cirak S</span>, <span class="author">Foley AR</span>, <span class="author">Sailer A</span>, <span class="author">Hurles ME</span>, <span class="author">Stalker J</span>, <span class="author">Hargreaves I</span>, <span class="author">Woodward CE</span>, <span class="author">Sweeney MG</span>, <span class="author">Muntoni F</span>, <span class="author">Houlden H</span>, <span class="author">UK10K Consortium</span>, <span class="author">Taanman JW</span> and <span class="author">Hanna MG</span></p>
<p class="affiliation">MRC Centre for Neuromuscular Diseases, UCL Institute of Neurology and National Hospital for Neurology and Neurosurgery, Queen Square, London WC1N 3BG, UK.</p><p class="abstract">The molecular basis of cytochrome c oxidase (COX, complex IV) deficiency remains genetically undetermined in many cases. Homozygosity mapping and whole-exome sequencing were performed in a consanguineous pedigree with isolated COX deficiency linked to&nbsp;a Leigh syndrome neurological phenotype. Unexpectedly, affected individuals harbored homozygous splice donor site mutations in NDUFA4, a gene previously assigned to encode a mitochondrial respiratory chain complex I (NADH:ubiquinone oxidoreductase) subunit. Western blot analysis of denaturing gels and immunocytochemistry revealed undetectable steady-state NDUFA4 protein levels, indicating that the mutation causes a loss-of-function effect in the homozygous state. Analysis of one- and two-dimensional blue-native polyacrylamide gels confirmed an interaction between NDUFA4 and the COX enzyme complex in control muscle, whereas the COX enzyme complex without NDUFA4 was detectable with no abnormal subassemblies in patient muscle. These observations support recent work in cell lines suggesting that NDUFA4 is an additional COX subunit and demonstrate that NDUFA4 mutations cause human disease. Our findings support reassignment of the NDUFA4 protein to complex IV and suggest that patients with unexplained COX deficiency should be screened for NDUFA4 mutations.</p>
<p class="publication"><span class="title">Cell reports</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23746447">23746447</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.celrep.2013.05.005">10.1016/j.celrep.2013.05.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">COX10 mutations resulting in complex multisystem mitochondrial disease that remains stable into adulthood.</h4>
<p class="authors"><span class="author">Pitceathly RD</span>, <span class="author">Taanman JW</span>, <span class="author">Rahman S</span>, <span class="author">Meunier B</span>, <span class="author">Sadowski M</span>, <span class="author">Cirak S</span>, <span class="author">Hargreaves I</span>, <span class="author">Land JM</span>, <span class="author">Nanji T</span>, <span class="author">Polke JM</span>, <span class="author">Woodward CE</span>, <span class="author">Sweeney MG</span>, <span class="author">Solanki S</span>, <span class="author">Foley AR</span>, <span class="author">Hurles ME</span>, <span class="author">Stalker J</span>, <span class="author">Blake J</span>, <span class="author">Holton JL</span>, <span class="author">Phadke R</span>, <span class="author">Muntoni F</span>, <span class="author">Reilly MM</span>, <span class="author">Hanna MG</span> and <span class="author">UK10K Consortium</span></p>
<p class="affiliation">Medical Research Council Centre for Neuromuscular Diseases, University College London Institute of Neurology and National Hospital for Neurology and Neurosurgery, London, England.</p><p class="abstract"><strong>Importance</strong>: Isolated cytochrome-c oxidase (COX) deficiency is one of the most frequent respiratory chain defects seen in human mitochondrial disease. Typically, patients present with severe neonatal multisystem disease and have an early fatal outcome. We describe an adult patient with isolated COX deficiency associated with a relatively mild clinical phenotype comprising myopathy; demyelinating neuropathy; premature ovarian failure; short stature; hearing loss; pigmentary maculopathy; and renal tubular dysfunction.</p><p class="abstract"><strong>Observations</strong>: Whole-exome sequencing detected 1 known pathogenic and 1 novel COX10 mutation: c.1007A&gt;T; p.Asp336Val, previously associated with fatal infantile COX deficiency, and c.1015C&gt;T; p.Arg339Trp. Muscle COX holoenzyme and subassemblies were undetectable on immunoblots of blue-native gels, whereas denaturing gels and immunocytochemistry showed reduced core subunit MTCO1. Heme absorption spectra revealed low heme aa3 compatible with heme A:farnesyltransferase deficiency due to COX10 dysfunction. Both mutations demonstrated respiratory deficiency in yeast, confirming pathogenicity. A COX10 protein model was used to predict the structural consequences of the novel Arg339Trp and all previously reported substitutions.</p><p class="abstract">These findings establish that COX10 mutations cause adult mitochondrial disease. Nuclear modifiers, epigenetic phenomenon, and/or environmental factors may influence the disease phenotype caused by reduced COX activity and contribute to the variable clinical severity related to COX10 dysfunction.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0601943</span>, <span class="grant">G0800674</span>, <span class="grant">MR/K000608/1</span>, <span class="grant">U117581331</span>; NINDS NIH HHS: <span class="grant">1U54NS065712-01</span>; Wellcome Trust: <span class="grant">WT091310</span></p>
<p class="publication"><span class="title">JAMA neurology</span>&nbsp;<span class="year">2013</span>;<span class="volume">70</span>;<span class="number">12</span>;<span class="pagerange">1556-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24100867">24100867</a>; DOI: <a href="http://dx.doi.org/10.1001%2Fjamaneurol.2013.3242">10.1001/jamaneurol.2013.3242</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-fat feeding rapidly induces obesity and lipid derangements in C57BL/6N mice.</h4>
<p class="authors"><span class="author">Podrini C</span>, <span class="author">Cambridge EL</span>, <span class="author">Lelliott CJ</span>, <span class="author">Carragher DM</span>, <span class="author">Estabel J</span>, <span class="author">Gerdin AK</span>, <span class="author">Karp NA</span>, <span class="author">Scudamore CL</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Ramirez-Solis R</span> and <span class="author">White JK</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p>
<p class="abstract">C57BL/6N (B6N) is becoming the standard background for genetic manipulation of the mouse genome. The B6N, whose genome is very closely related to the reference C57BL/6J genome, is versatile in a wide range of phenotyping and experimental settings and large repositories of B6N ES cells have been developed. Here, we present a series of studies showing the baseline characteristics of B6N fed a high-fat diet (HFD) for up to 12 weeks. We show that HFD-fed B6N mice show increased weight gain, fat mass, and hypercholesterolemia compared to control diet-fed mice. In addition, HFD-fed B6N mice display a rapid onset of lipid accumulation in the liver with both macro- and microvacuolation, which became more severe with increasing duration of HFD. Our results suggest that the B6N mouse strain is a versatile background for studying diet-induced metabolic syndrome and may also represent a model for early nonalcoholic fatty liver disease.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23712496">23712496</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-013-9456-0">10.1007/s00335-013-9456-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide mutational signatures of aristolochic acid and its application as a screening tool.</h4>
<p class="authors"><span class="author">Poon SL</span>, <span class="author">Pang ST</span>, <span class="author">McPherson JR</span>, <span class="author">Yu W</span>, <span class="author">Huang KK</span>, <span class="author">Guan P</span>, <span class="author">Weng WH</span>, <span class="author">Siew EY</span>, <span class="author">Liu Y</span>, <span class="author">Heng HL</span>, <span class="author">Chong SC</span>, <span class="author">Gan A</span>, <span class="author">Tay ST</span>, <span class="author">Lim WK</span>, <span class="author">Cutcutache I</span>, <span class="author">Huang D</span>, <span class="author">Ler LD</span>, <span class="author">Nairism&auml;gi ML</span>, <span class="author">Lee MH</span>, <span class="author">Chang YH</span>, <span class="author">Yu KJ</span>, <span class="author">Chan-On W</span>, <span class="author">Li BK</span>, <span class="author">Yuan YF</span>, <span class="author">Qian CN</span>, <span class="author">Ng KF</span>, <span class="author">Wu CF</span>, <span class="author">Hsu CL</span>, <span class="author">Bunte RM</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Sung WK</span>, <span class="author">Chuang CK</span>, <span class="author">Ong CK</span>, <span class="author">Rozen SG</span>, <span class="author">Tan P</span> and <span class="author">Teh BT</span></p>
<p class="affiliation">NCCS-VARI Translational Research Laboratory, Division of Medical Sciences, National Cancer Centre Singapore, 11 Hospital Drive, Singapore 169610, Singapore.</p><p class="abstract">Aristolochic acid (AA), a natural product of Aristolochia plants found in herbal remedies and health supplements, is a group 1 carcinogen that can cause nephrotoxicity and upper urinary tract urothelial cell carcinoma (UTUC). Whole-genome and exome analysis of nine AA-associated UTUCs revealed a strikingly high somatic mutation rate (150 mutations/Mb), exceeding smoking-associated lung cancer (8 mutations/Mb) and ultraviolet radiation-associated melanoma (111 mutations/Mb). The AA-UTUC mutational signature was characterized by A:T to T:A transversions at the sequence motif A[C|T]AGG, located primarily on nontranscribed strands. AA-induced mutations were also significantly enriched at splice sites, suggesting a role for splice-site mutations in UTUC pathogenesis. RNA sequencing of AA-UTUC confirmed a general up-regulation of nonsense-mediated decay machinery components and aberrant splicing events associated with splice-site mutations. We observed a high frequency of somatic mutations in chromatin modifiers, particularly KDM6A, in AA-UTUC, demonstrated the sufficiency of AA to induce renal dysplasia in mice, and reproduced the AA mutational signature in experimentally treated human renal tubular cells. Finally, exploring other malignancies that were not known to be associated with AA, we screened 93 hepatocellular carcinoma genomes/exomes and identified AA-like mutational signatures in 11. Our study highlights an unusual genome-wide AA mutational signature and the potential use of mutation signatures as &quot;molecular fingerprints&quot; for interrogating high-throughput cancer genome data to infer previous carcinogen exposures.</p>
<p class="publication"><span class="title">Science translational medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">197</span>;<span class="pagerange">197ra101</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23926199">23926199</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscitranslmed.3006086">10.1126/scitranslmed.3006086</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A meta-analysis of thyroid-related traits reveals novel Loci and gender-specific differences in the regulation of thyroid function.</h4>
<p class="authors"><span class="author">Porcu E</span>, <span class="author">Medici M</span>, <span class="author">Pistis G</span>, <span class="author">Volpato CB</span>, <span class="author">Wilson SG</span>, <span class="author">Cappola AR</span>, <span class="author">Bos SD</span>, <span class="author">Deelen J</span>, <span class="author">den Heijer M</span>, <span class="author">Freathy RM</span>, <span class="author">Lahti J</span>, <span class="author">Liu C</span>, <span class="author">Lopez LM</span>, <span class="author">Nolte IM</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Tanaka T</span>, <span class="author">Trompet S</span>, <span class="author">Arnold A</span>, <span class="author">Bandinelli S</span>, <span class="author">Beekman M</span>, <span class="author">B&ouml;hringer S</span>, <span class="author">Brown SJ</span>, <span class="author">Buckley BM</span>, <span class="author">Camaschella C</span>, <span class="author">de Craen AJ</span>, <span class="author">Davies G</span>, <span class="author">de Visser MC</span>, <span class="author">Ford I</span>, <span class="author">Forsen T</span>, <span class="author">Frayling TM</span>, <span class="author">Fugazzola L</span>, <span class="author">G&ouml;gele M</span>, <span class="author">Hattersley AT</span>, <span class="author">Hermus AR</span>, <span class="author">Hofman A</span>, <span class="author">Houwing-Duistermaat JJ</span>, <span class="author">Jensen RA</span>, <span class="author">Kajantie E</span>, <span class="author">Kloppenburg M</span>, <span class="author">Lim EM</span>, <span class="author">Masciullo C</span>, <span class="author">Mariotti S</span>, <span class="author">Minelli C</span>, <span class="author">Mitchell BD</span>, <span class="author">Nagaraja R</span>, <span class="author">Netea-Maier RT</span>, <span class="author">Palotie A</span>, <span class="author">Persani L</span>, <span class="author">Piras MG</span>, <span class="author">Psaty BM</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Richards JB</span>, <span class="author">Rivadeneira F</span>, <span class="author">Sala C</span>, <span class="author">Sabra MM</span>, <span class="author">Sattar N</span>, <span class="author">Shields BM</span>, <span class="author">Soranzo N</span>, <span class="author">Starr JM</span>, <span class="author">Stott DJ</span>, <span class="author">Sweep FC</span>, <span class="author">Usala G</span>, <span class="author">van der Klauw MM</span>, <span class="author">van Heemst D</span>, <span class="author">van Mullem A</span>, <span class="author">H Vermeulen S</span>, <span class="author">Visser WE</span>, <span class="author">Walsh JP</span>, <span class="author">Westendorp RG</span>, <span class="author">Widen E</span>, <span class="author">Zhai G</span>, <span class="author">Cucca F</span>, <span class="author">Deary IJ</span>, <span class="author">Eriksson JG</span>, <span class="author">Ferrucci L</span>, <span class="author">Fox CS</span>, <span class="author">Jukema JW</span>, <span class="author">Kiemeney LA</span>, <span class="author">Pramstaller PP</span>, <span class="author">Schlessinger D</span>, <span class="author">Shuldiner AR</span>, <span class="author">Slagboom EP</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Vaidya B</span>, <span class="author">Visser TJ</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Meulenbelt I</span>, <span class="author">Rotter JI</span>, <span class="author">Spector TD</span>, <span class="author">Hicks AA</span>, <span class="author">Toniolo D</span>, <span class="author">Sanna S</span>, <span class="author">Peeters RP</span> and <span class="author">Naitza S</span></p>
<p class="affiliation">Istituto di Ricerca Genetica e Biomedica (IRGB), Consiglio Nazionale delle Ricerche, c/o Cittadella Universitaria di Monserrato, Monserrato, Cagliari, Italy ; Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy.</p>
<p class="abstract">Thyroid hormone is essential for normal metabolism and development, and overt abnormalities in thyroid function lead to common endocrine disorders affecting approximately 10% of individuals over their life span. In addition, even mild alterations in thyroid function are associated with weight changes, atrial fibrillation, osteoporosis, and psychiatric disorders. To identify novel variants underlying thyroid function, we performed a large meta-analysis of genome-wide association studies for serum levels of the highly heritable thyroid function markers TSH and FT4, in up to 26,420 and 17,520 euthyroid subjects, respectively. Here we report 26 independent associations, including several novel loci for TSH (PDE10A, VEGFA, IGFBP5, NFIA, SOX9, PRDM11, FGF7, INSR, ABO, MIR1179, NRG1, MBIP, ITPK1, SASH1, GLIS3) and FT4 (LHX3, FOXE1, AADAT, NETO1/FBXO15, LPCAT2/CAPNS2). Notably, only limited overlap was detected between TSH and FT4 associated signals, in spite of the feedback regulation of their circulating levels by the hypothalamic-pituitary-thyroid axis. Five of the reported loci (PDE8B, PDE10A, MAF/LOC440389, NETO1/FBXO15, and LPCAT2/CAPNS2) show strong gender-specific differences, which offer clues for the known sexual dimorphism in thyroid function and related pathologies. Importantly, the TSH-associated loci contribute not only to variation within the normal range, but also to TSH values outside the reference range, suggesting that they may be involved in thyroid dysfunction. Overall, our findings explain, respectively, 5.64% and 2.30% of total TSH and FT4 trait variance, and they improve the current knowledge of the regulation of hypothalamic-pituitary-thyroid axis function and the consequences of genetic variation for hypo- or hyperthyroidism.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">e1003266</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23408906">23408906</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3567175">3567175</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003266">10.1371/journal.pgen.1003266</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single-cell mutational profiling and clonal phylogeny in cancer.</h4>
<p class="authors"><span class="author">Potter NE</span>, <span class="author">Ermini L</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Cazzaniga G</span>, <span class="author">Vijayaraghavan G</span>, <span class="author">Titley I</span>, <span class="author">Ford A</span>, <span class="author">Campbell P</span>, <span class="author">Kearney L</span> and <span class="author">Greaves M</span></p>
<p class="affiliation">The Institute of Cancer Research, London, SM2 5NG, United Kingdom;</p>
<p class="abstract">The development of cancer is a dynamic evolutionary process in which intraclonal, genetic diversity provides a substrate for clonal selection and a source of therapeutic escape. The complexity and topography of intraclonal genetic architectures have major implications for biopsy-based prognosis and for targeted therapy. High-depth, next-generation sequencing (NGS) efficiently captures the mutational load of individual tumors or biopsies. But, being a snapshot portrait of total DNA, it disguises the fundamental features of subclonal variegation of genetic lesions and of clonal phylogeny. Single-cell genetic profiling provides a potential resolution to this problem, but methods developed to date all have limitations. We present a novel solution to this challenge using leukemic cells with known mutational spectra as a tractable model. DNA from flow-sorted single cells is screened using multiplex targeted Q-PCR within a microfluidic platform allowing unbiased single-cell selection, high-throughput, and comprehensive analysis for all main varieties of genetic abnormalities: chimeric gene fusions, copy number alterations, and single-nucleotide variants. We show, in this proof-of-principle study, that the method has a low error rate and can provide detailed subclonal genetic architectures and phylogenies.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24056532">24056532</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.159913.113">10.1101/gr.159913.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of Small Exonic CNV from Whole-Exome Sequence Data and Application to Autism Spectrum Disorder.</h4>
<p class="authors"><span class="author">Poultney CS</span>, <span class="author">Goldberg AP</span>, <span class="author">Drapeau E</span>, <span class="author">Kou Y</span>, <span class="author">Harony-Nicolas H</span>, <span class="author">Kajiwara Y</span>, <span class="author">De Rubeis S</span>, <span class="author">Durand S</span>, <span class="author">Stevens C</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Palotie A</span>, <span class="author">Daly MJ</span>, <span class="author">Ma&#39;ayan A</span>, <span class="author">Fromer M</span> and <span class="author">Buxbaum JD</span></p>
<p class="affiliation">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA; Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA.</p>
<p class="abstract">Copy number variation (CNV) is an important determinant of human diversity and plays important roles in susceptibility to disease. Most studies of CNV carried out to date have made use of chromosome microarray and have had a lower size limit for detection of about 30 kilobases (kb). With the emergence of whole-exome sequencing studies, we asked whether such data could be used to reliably call rare exonic CNV in the size range of 1-30 kilobases (kb), making use of the eXome Hidden Markov Model (XHMM) program. By using both transmission information and validation by molecular methods, we confirmed that small CNV encompassing as few as three exons can be reliably called from whole-exome data. We applied this approach to an autism case-control sample (n = 811, mean per-target read depth = 161) and observed a significant increase in the burden of rare (MAF ≤1%) 1-30 kb CNV, 1-30 kb deletions, and 1-10 kb deletions in ASD. CNV in the 1-30 kb range frequently hit just a single gene, and we were therefore able to carry out enrichment and pathway analyses, where we observed enrichment for disruption of genes in cytoskeletal and autophagy pathways in ASD. In summary, our results showed that XHMM provided an effective means to assess small exonic CNV from whole-exome data, indicated that rare 1-30 kb exonic deletions could contribute to risk in up to 7% of individuals with ASD, and implicated a candidate pathway in developmental delay syndromes.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">93</span>;<span class="number">4</span>;<span class="pagerange">607-19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24094742">24094742</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3791269">3791269</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.09.001">10.1016/j.ajhg.2013.09.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative study of transcriptome profiles of mechanical- and skin-transformed Schistosoma mansoni schistosomula.</h4>
<p class="authors"><span class="author">Protasio AV</span>, <span class="author">Dunne DW</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">Schistosome infection begins with the penetration of cercariae through healthy unbroken host skin. This process leads to the transformation of the free-living larvae into obligate parasites called schistosomula. This irreversible transformation, which occurs in as little as two hours, involves casting the cercaria tail and complete remodelling of the surface membrane. At this stage, parasites are vulnerable to host immune attack and oxidative stress. Consequently, the mechanisms by which the parasite recognises and swiftly adapts to the human host are still the subject of many studies, especially in the context of development of intervention strategies against schistosomiasis infection. Because obtaining enough material from in vivo infections is not always feasible for such studies, the transformation process is often mimicked in the laboratory by application of shear pressure to a cercarial sample resulting in mechanically transformed (MT) schistosomula. These parasites share remarkable morphological and biochemical similarity to the naturally transformed counterparts and have been considered a good proxy for parasites undergoing natural infection. Relying on this equivalency, MT schistosomula have been used almost exclusively in high-throughput studies of gene expression, identification of drug targets and identification of effective drugs against schistosomes. However, the transcriptional equivalency between skin-transformed (ST) and MT schistosomula has never been proven. In our approach to compare these two types of schistosomula preparations and to explore differences in gene expression triggered by the presence of a skin barrier, we performed RNA-seq transcriptome profiling of ST and MT schistosomula at 24 hours post transformation. We report that these two very distinct schistosomula preparations differ only in the expression of 38 genes (out of &sim;11,000), providing convincing evidence to resolve the skin vs. mechanical long-lasting controversy.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 083931/Z/07/Z</span>, <span class="grant">WT 098051</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">e2091</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23516644">23516644</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3597483">3597483</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0002091">10.1371/journal.pntd.0002091</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeting MYCN in neuroblastoma by BET bromodomain inhibition.</h4>
<p class="authors"><span class="author">Puissant A</span>, <span class="author">Frumm SM</span>, <span class="author">Alexe G</span>, <span class="author">Bassil CF</span>, <span class="author">Qi J</span>, <span class="author">Chanthery YH</span>, <span class="author">Nekritz EA</span>, <span class="author">Zeid R</span>, <span class="author">Gustafson WC</span>, <span class="author">Greninger P</span>, <span class="author">Garnett MJ</span>, <span class="author">McDermott U</span>, <span class="author">Benes CH</span>, <span class="author">Kung AL</span>, <span class="author">Weiss WA</span>, <span class="author">Bradner JE</span> and <span class="author">Stegmaier K</span></p>
<p class="affiliation">Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, USA.</p><p class="abstract">Bromodomain inhibition comprises a promising therapeutic strategy in cancer, particularly for hematologic malignancies. To date, however, genomic biomarkers to direct clinical translation have been lacking. We conducted a cell-based screen of genetically defined cancer cell lines using a prototypical inhibitor of BET bromodomains. Integration of genetic features with chemosensitivity data revealed a robust correlation between MYCN amplification and sensitivity to bromodomain inhibition. We characterized the mechanistic and translational significance of this finding in neuroblastoma, a childhood cancer with frequent amplification of MYCN. Genome-wide expression analysis showed downregulation of the MYCN transcriptional program accompanied by suppression of MYCN transcription. Functionally, bromodomain-mediated inhibition of MYCN impaired growth and induced apoptosis in neuroblastoma. BRD4 knockdown phenocopied these effects, establishing BET bromodomains as transcriptional regulators of MYCN. BET inhibition conferred a significant survival advantage in 3 in vivo neuroblastoma models, providing a compelling rationale for developing BET bromodomain inhibitors in patients with neuroblastoma.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01 CA081403</span>, <span class="grant">P01CA081403</span>, <span class="grant">R01 CA102321</span>, <span class="grant">R01CA102321</span>, <span class="grant">T32 CA128583</span>, <span class="grant">T32CA151022</span>; NINDS NIH HHS: <span class="grant">K08 NS079485</span>, <span class="grant">K08NS079485</span>; Wellcome Trust: <span class="grant">086357</span>, <span class="grant">093868</span></p>
<p class="publication"><span class="title">Cancer discovery</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">308-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23430699">23430699</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672953">3672953</a>; DOI: <a href="http://dx.doi.org/10.1158%2F2159-8290.CD-12-0418">10.1158/2159-8290.CD-12-0418</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SpoIVA and SipL are Clostridium difficile spore morphogenetic proteins.</h4>
<p class="authors"><span class="author">Putnam EE</span>, <span class="author">Nock AM</span>, <span class="author">Lawley TD</span> and <span class="author">Shen A</span></p>
<p class="affiliation">Department of Microbiology and Molecular Genetics, University of Vermont, Burlington, Vermont, USA.</p><p class="abstract">Clostridium difficile is a major nosocomial pathogen whose infections are difficult to treat because of their frequent recurrence. The spores of C. difficile are responsible for these clinical features, as they resist common disinfectants and antibiotic treatment. Although spores are the major transmissive form of C. difficile, little is known about their composition or morphogenesis. Spore morphogenesis has been well characterized for Bacillus sp., but Bacillus sp. spore coat proteins are poorly conserved in Clostridium sp. Of the known spore morphogenetic proteins in Bacillus subtilis, SpoIVA is one of the mostly highly conserved in the Bacilli and the Clostridia. Using genetic analyses, we demonstrate that SpoIVA is required for proper spore morphogenesis in C. difficile. In particular, a spoIVA mutant exhibits defects in spore coat localization but not cortex formation. Our study also identifies SipL, a previously uncharacterized protein found in proteomic studies of C. difficile spores, as another critical spore morphogenetic protein, since a sipL mutant phenocopies a spoIVA mutant. Biochemical analyses and mutational analyses indicate that SpoIVA and SipL directly interact. This interaction depends on the Walker A ATP binding motif of SpoIVA and the LysM domain of SipL. Collectively, these results provide the first insights into spore morphogenesis in C. difficile.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P20RR021905</span>; NIGMS NIH HHS: <span class="grant">R00 GM092934</span>, <span class="grant">R00GM092934</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2013</span>;<span class="volume">195</span>;<span class="number">6</span>;<span class="pagerange">1214-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23292781">23292781</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3592010">3592010</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.02181-12">10.1128/JB.02181-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genetic progression model of Braf(V600E)-induced intestinal tumorigenesis reveals targets for therapeutic intervention.</h4>
<p class="authors"><span class="author">Rad R</span>, <span class="author">Cadi&ntilde;anos J</span>, <span class="author">Rad L</span>, <span class="author">Varela I</span>, <span class="author">Strong A</span>, <span class="author">Kriegl L</span>, <span class="author">Constantino-Casas F</span>, <span class="author">Eser S</span>, <span class="author">Hieber M</span>, <span class="author">Seidler B</span>, <span class="author">Price S</span>, <span class="author">Fraga MF</span>, <span class="author">Calvanese V</span>, <span class="author">Hoffman G</span>, <span class="author">Ponstingl H</span>, <span class="author">Schneider G</span>, <span class="author">Yusa K</span>, <span class="author">Grove C</span>, <span class="author">Schmid RM</span>, <span class="author">Wang W</span>, <span class="author">Vassiliou G</span>, <span class="author">Kirchner T</span>, <span class="author">McDermott U</span>, <span class="author">Liu P</span>, <span class="author">Saur D</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Medicine II, Klinikum Rechts der Isar, Technische Universit&auml;t M&uuml;nchen, 81675, M&uuml;nchen, Germany. <a href="mailto:%72%6F%6C%61%6E%64.%72%61%64%40%6C%72%7A.%74%75%6D.%64%65">&#114;&#111;&#108;&#97;&#110;&#100;&#46;&#114;&#97;&#100;&#64;&#108;&#114;&#122;&#46;&#116;&#117;&#109;&#46;&#100;&#101;</a></p><p class="abstract">We show that BRAF(V600E) initiates an alternative pathway to colorectal cancer (CRC), which progresses through a hyperplasia/adenoma/carcinoma sequence. This pathway underlies significant subsets of CRCs with distinctive pathomorphologic/genetic/epidemiologic/clinical characteristics. Genetic and functional analyses in mice revealed a series of stage-specific molecular alterations driving different phases of tumor evolution and uncovered mechanisms underlying this stage specificity. We further demonstrate dose-dependent effects of oncogenic signaling, with physiologic Braf(V600E) expression being sufficient for hyperplasia induction, but later stage intensified Mapk-signaling driving both tumor progression and activation of intrinsic tumor suppression. Such phenomena explain, for example, the inability of p53 to restrain tumor initiation as well as its importance in invasiveness control, and the late stage specificity of its somatic mutation. Finally, systematic drug screening revealed sensitivity of this CRC subtype to targeted therapeutics, including Mek or combinatorial PI3K/Braf inhibition.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095663</span></p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">24</span>;<span class="number">1</span>;<span class="pagerange">15-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23845441">23845441</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3706745">3706745</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ccr.2013.05.014">10.1016/j.ccr.2013.05.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dnmt2-dependent methylomes lack defined DNA methylation patterns.</h4>
<p class="authors"><span class="author">Raddatz G</span>, <span class="author">Guzzardo PM</span>, <span class="author">Olova N</span>, <span class="author">Fantappi&eacute; MR</span>, <span class="author">Rampp M</span>, <span class="author">Schaefer M</span>, <span class="author">Reik W</span>, <span class="author">Hannon GJ</span> and <span class="author">Lyko F</span></p>
<p class="affiliation">Division of Epigenetics, DKFZ-ZMBH Alliance, German Cancer Research Center, 69120 Heidelberg, Germany.</p>
<p class="abstract">Several organisms have retained methyltransferase 2 (Dnmt2) as their only candidate DNA methyltransferase gene. However, information about Dnmt2-dependent methylation patterns has been limited to a few isolated loci and the results have been discussed controversially. In addition, recent studies have shown that Dnmt2 functions as a tRNA methyltransferase, which raised the possibility that Dnmt2-only genomes might be unmethylated. We have now used whole-genome bisulfite sequencing to analyze the methylomes of Dnmt2-only organisms at single-base resolution. Our results show that the genomes of Schistosoma mansoni and Drosophila melanogaster lack detectable DNA methylation patterns. Residual unconverted cytosine residues shared many attributes with bisulfite deamination artifacts and were observed at comparable levels in Dnmt2-deficient flies. Furthermore, genetically modified Dnmt2-only mouse embryonic stem cells lost the DNA methylation patterns found in wild-type cells. Our results thus uncover fundamental differences among animal methylomes and suggest that DNA methylation is dispensable for a considerable number of eukaryotic organisms.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">21</span>;<span class="pagerange">8627-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23641003">23641003</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1306723110">10.1073/pnas.1306723110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare variants in single-minded 1 (SIM1) are associated with severe obesity.</h4>
<p class="authors"><span class="author">Ramachandrappa S</span>, <span class="author">Raimondo A</span>, <span class="author">Cali AM</span>, <span class="author">Keogh JM</span>, <span class="author">Henning E</span>, <span class="author">Saeed S</span>, <span class="author">Thompson A</span>, <span class="author">Garg S</span>, <span class="author">Bochukova EG</span>, <span class="author">Brage S</span>, <span class="author">Trowse V</span>, <span class="author">Wheeler E</span>, <span class="author">Sullivan AE</span>, <span class="author">Dattani M</span>, <span class="author">Clayton PE</span>, <span class="author">Datta V</span>, <span class="author">Datta V</span>, <span class="author">Bruning JB</span>, <span class="author">Wareham NJ</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Peet DJ</span>, <span class="author">Barroso I</span>, <span class="author">Whitelaw ML</span> and <span class="author">Farooqi IS</span></p>
<p class="affiliation">University of Cambridge Metabolic Research Laboratories and NIHR Cambridge Biomedical Research Centre, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom.</p><p class="abstract">Single-minded 1 (SIM1) is a basic helix-loop-helix transcription factor involved in the development and function of the paraventricular nucleus of the hypothalamus. Obesity has been reported in Sim1 haploinsufficient mice and in a patient with a balanced translocation disrupting SIM1. We sequenced the coding region of SIM1 in 2,100 patients with severe, early onset obesity and in 1,680 controls. Thirteen different heterozygous variants in SIM1 were identified in 28 unrelated severely obese patients. Nine of the 13 variants significantly reduced the ability of SIM1 to activate a SIM1-responsive reporter gene when studied in stably transfected cells coexpressing the heterodimeric partners of SIM1 (ARNT or ARNT2). SIM1 variants with reduced activity cosegregated with obesity in extended family studies with variable penetrance. We studied the phenotype of patients carrying variants that exhibited reduced activity in vitro. Variant carriers exhibited increased ad libitum food intake at a test meal, normal basal metabolic rate, and evidence of autonomic dysfunction. Eleven of the 13 probands had evidence of a neurobehavioral phenotype. The phenotypic similarities between patients with SIM1 deficiency and melanocortin 4 receptor (MC4R) deficiency suggest that some of the effects of SIM1 deficiency on energy homeostasis are mediated by altered melanocortin signaling.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900554</span>, <span class="grant">G9824984</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179473</span>, <span class="grant">MC_U106188470</span>; NHLBI NIH HHS: <span class="grant">HL-102923</span>, <span class="grant">HL-102924</span>, <span class="grant">HL-102925</span>, <span class="grant">HL-102926</span>, <span class="grant">HL-103010</span>; Wellcome Trust: <span class="grant">077016/Z/05/Z</span>, <span class="grant">082390/Z/07/Z</span>, <span class="grant">098497</span></p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2013</span>;<span class="volume">123</span>;<span class="number">7</span>;<span class="pagerange">3042-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23778139">23778139</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3696558">3696558</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI68016">10.1172/JCI68016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DeNovoGear: de novo indel and point mutation discovery and phasing.</h4>
<p class="authors"><span class="author">Ramu A</span>, <span class="author">Noordam MJ</span>, <span class="author">Schwartz RS</span>, <span class="author">Wuster A</span>, <span class="author">Hurles ME</span>, <span class="author">Cartwright RA</span> and <span class="author">Conrad DF</span></p>
<p class="affiliation">1] Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, USA. [2].</p>
<p class="abstract">We present DeNovoGear software for analyzing de novo mutations from familial and somatic tissue sequencing data. DeNovoGear uses likelihood-based error modeling to reduce the false positive rate of mutation discovery in exome analysis and fragment information to identify the parental origin of germ-line mutations. We used DeNovoGear on human whole-genome sequencing data to produce a set of predicted de novo insertion and/or deletion (indel) mutations with a 95% validation rate.</p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23975140">23975140</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.2611">10.1038/nmeth.2611</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sex-stratified genome-wide association studies including 270,000 individuals show sexual dimorphism in genetic loci for anthropometric traits.</h4>
<p class="authors"><span class="author">Randall JC</span>, <span class="author">Winkler TW</span>, <span class="author">Kutalik Z</span>, <span class="author">Berndt SI</span>, <span class="author">Jackson AU</span>, <span class="author">Monda KL</span>, <span class="author">Kilpel&auml;inen TO</span>, <span class="author">Esko T</span>, <span class="author">M&auml;gi R</span>, <span class="author">Li S</span>, <span class="author">Workalemahu T</span>, <span class="author">Feitosa MF</span>, <span class="author">Croteau-Chonka DC</span>, <span class="author">Day FR</span>, <span class="author">Fall T</span>, <span class="author">Ferreira T</span>, <span class="author">Gustafsson S</span>, <span class="author">Locke AE</span>, <span class="author">Mathieson I</span>, <span class="author">Scherag A</span>, <span class="author">Vedantam S</span>, <span class="author">Wood AR</span>, <span class="author">Liang L</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Thorleifsson G</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Dimas AS</span>, <span class="author">Karpe F</span>, <span class="author">Min JL</span>, <span class="author">Nicholson G</span>, <span class="author">Clegg DJ</span>, <span class="author">Person T</span>, <span class="author">Krohn JP</span>, <span class="author">Bauer S</span>, <span class="author">Buechler C</span>, <span class="author">Eisinger K</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Bonnefond A</span>, <span class="author">Froguel P</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Hottenga JJ</span>, <span class="author">Prokopenko I</span>, <span class="author">Waite LL</span>, <span class="author">Harris TB</span>, <span class="author">Smith AV</span>, <span class="author">Shuldiner AR</span>, <span class="author">McArdle WL</span>, <span class="author">Caulfield MJ</span>, <span class="author">Munroe PB</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Chen YD</span>, <span class="author">Li G</span>, <span class="author">Beckmann JS</span>, <span class="author">Johnson T</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Teder-Laving M</span>, <span class="author">Khaw KT</span>, <span class="author">Wareham NJ</span>, <span class="author">Zhao JH</span>, <span class="author">Amin N</span>, <span class="author">Oostra BA</span>, <span class="author">Kraja AT</span>, <span class="author">Province MA</span>, <span class="author">Cupples LA</span>, <span class="author">Heard-Costa NL</span>, <span class="author">Kaprio J</span>, <span class="author">Ripatti S</span>, <span class="author">Surakka I</span>, <span class="author">Collins FS</span>, <span class="author">Saramies J</span>, <span class="author">Tuomilehto J</span>, <span class="author">Jula A</span>, <span class="author">Salomaa V</span>, <span class="author">Erdmann J</span>, <span class="author">Hengstenberg C</span>, <span class="author">Loley C</span>, <span class="author">Schunkert H</span>, <span class="author">Lamina C</span>, <span class="author">Wichmann HE</span>, <span class="author">Albrecht E</span>, <span class="author">Gieger C</span>, <span class="author">Hicks AA</span>, <span class="author">Johansson A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Kathiresan S</span>, <span class="author">Speliotes EK</span>, <span class="author">Penninx B</span>, <span class="author">Hartikainen AL</span>, <span class="author">Jarvelin MR</span>, <span class="author">Gyllensten U</span>, <span class="author">Boomsma DI</span>, <span class="author">Campbell H</span>, <span class="author">Wilson JF</span>, <span class="author">Chanock SJ</span>, <span class="author">Farrall M</span>, <span class="author">Goel A</span>, <span class="author">Medina-Gomez C</span>, <span class="author">Rivadeneira F</span>, <span class="author">Estrada K</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">Zillikens MC</span>, <span class="author">den Heijer M</span>, <span class="author">Kiemeney LA</span>, <span class="author">Maschio A</span>, <span class="author">Hall P</span>, <span class="author">Tyrer J</span>, <span class="author">Teumer A</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Kovacs P</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Mangino M</span>, <span class="author">Spector TD</span>, <span class="author">Hayward C</span>, <span class="author">Rudan I</span>, <span class="author">Hall AS</span>, <span class="author">Samani NJ</span>, <span class="author">Attwood AP</span>, <span class="author">Sambrook JG</span>, <span class="author">Hung J</span>, <span class="author">Palmer LJ</span>, <span class="author">Lokki ML</span>, <span class="author">Sinisalo J</span>, <span class="author">Boucher G</span>, <span class="author">Huikuri H</span>, <span class="author">Lorentzon M</span>, <span class="author">Ohlsson C</span>, <span class="author">Eklund N</span>, <span class="author">Eriksson JG</span>, <span class="author">Barlassina C</span>, <span class="author">Rivolta C</span>, <span class="author">Nolte IM</span>, <span class="author">Snieder H</span>, <span class="author">Van der Klauw MM</span>, <span class="author">Van Vliet-Ostaptchouk JV</span>, <span class="author">Gejman PV</span>, <span class="author">Shi J</span>, <span class="author">Jacobs KB</span>, <span class="author">Wang Z</span>, <span class="author">Bakker SJ</span>, <span class="author">Mateo Leach I</span>, <span class="author">Navis G</span>, <span class="author">van der Harst P</span>, <span class="author">Martin NG</span>, <span class="author">Medland SE</span>, <span class="author">Montgomery GW</span>, <span class="author">Yang J</span>, <span class="author">Chasman DI</span>, <span class="author">Ridker PM</span>, <span class="author">Rose LM</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Raitakari O</span>, <span class="author">Absher D</span>, <span class="author">Iribarren C</span>, <span class="author">Basart H</span>, <span class="author">Hovingh KG</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Power C</span>, <span class="author">Anderson D</span>, <span class="author">Beilby JP</span>, <span class="author">Hui J</span>, <span class="author">Jolley J</span>, <span class="author">Sager H</span>, <span class="author">Bornstein SR</span>, <span class="author">Schwarz PE</span>, <span class="author">Kristiansson K</span>, <span class="author">Perola M</span>, <span class="author">Lindstr&ouml;m J</span>, <span class="author">Swift AJ</span>, <span class="author">Uusitupa M</span>, <span class="author">Atalay M</span>, <span class="author">Lakka TA</span>, <span class="author">Rauramaa R</span>, <span class="author">Bolton JL</span>, <span class="author">Fowkes G</span>, <span class="author">Fraser RM</span>, <span class="author">Price JF</span>, <span class="author">Fischer K</span>, <span class="author">Krjut&aring; Kov K</span>, <span class="author">Metspalu A</span>, <span class="author">Mihailov E</span>, <span class="author">Langenberg C</span>, <span class="author">Luan J</span>, <span class="author">Ong KK</span>, <span class="author">Chines PS</span>, <span class="author">Keinanen-Kiukaanniemi SM</span>, <span class="author">Saaristo TE</span>, <span class="author">Edkins S</span>, <span class="author">Franks PW</span>, <span class="author">Hallmans G</span>, <span class="author">Shungin D</span>, <span class="author">Morris AD</span>, <span class="author">Palmer CN</span>, <span class="author">Erbel R</span>, <span class="author">Moebus S</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Pechlivanis S</span>, <span class="author">Hveem K</span>, <span class="author">Narisu N</span>, <span class="author">Hamsten A</span>, <span class="author">Humphries SE</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Tremoli E</span>, <span class="author">Grallert H</span>, <span class="author">Thorand B</span>, <span class="author">Illig T</span>, <span class="author">Koenig W</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Peters A</span>, <span class="author">Boehm BO</span>, <span class="author">Kleber ME</span>, <span class="author">M&auml;rz W</span>, <span class="author">Winkelmann BR</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Arveiler D</span>, <span class="author">Cesana G</span>, <span class="author">Kuulasmaa K</span>, <span class="author">Virtamo J</span>, <span class="author">Yarnell JW</span>, <span class="author">Kuh D</span>, <span class="author">Wong A</span>, <span class="author">Lind L</span>, <span class="author">de Faire U</span>, <span class="author">Gigante B</span>, <span class="author">Magnusson PK</span>, <span class="author">Pedersen NL</span>, <span class="author">Dedoussis G</span>, <span class="author">Dimitriou M</span>, <span class="author">Kolovou G</span>, <span class="author">Kanoni S</span>, <span class="author">Stirrups K</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Nj&oslash;lstad I</span>, <span class="author">Wilsgaard T</span>, <span class="author">Ganna A</span>, <span class="author">Rehnberg E</span>, <span class="author">Hingorani A</span>, <span class="author">Kivimaki M</span>, <span class="author">Kumari M</span>, <span class="author">Assimes TL</span>, <span class="author">Barroso I</span>, <span class="author">Boehnke M</span>, <span class="author">Borecki IB</span>, <span class="author">Deloukas P</span>, <span class="author">Fox CS</span>, <span class="author">Frayling T</span>, <span class="author">Groop LC</span>, <span class="author">Haritunians T</span>, <span class="author">Hunter D</span>, <span class="author">Ingelsson E</span>, <span class="author">Kaplan R</span>, <span class="author">Mohlke KL</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Schlessinger D</span>, <span class="author">Strachan DP</span>, <span class="author">Stefansson K</span>, <span class="author">van Duijn CM</span>, <span class="author">Abecasis GR</span>, <span class="author">McCarthy MI</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Qi L</span>, <span class="author">Loos RJ</span>, <span class="author">Lindgren CM</span>, <span class="author">North KE</span> and <span class="author">Heid IM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Given the anthropometric differences between men and women and previous evidence of sex-difference in genetic effects, we conducted a genome-wide search for sexually dimorphic associations with height, weight, body mass index, waist circumference, hip circumference, and waist-to-hip-ratio (133,723 individuals) and took forward 348 SNPs into follow-up (additional 137,052 individuals) in a total of 94 studies. Seven loci displayed significant sex-difference (FDR&lt;5%), including four previously established (near GRB14/COBLL1, LYPLAL1/SLC30A10, VEGFA, ADAMTS9) and three novel anthropometric trait loci (near MAP3K1, HSD17B4, PPARG), all of which were genome-wide significant in women (P&lt;5&times;10(-8)), but not in men. Sex-differences were apparent only for waist phenotypes, not for height, weight, BMI, or hip circumference. Moreover, we found no evidence for genetic effects with opposite directions in men versus women. The PPARG locus is of specific interest due to its role in diabetes genetics and therapy. Our results demonstrate the value of sex-specific GWAS to unravel the sexually dimorphic genetic underpinning of complex traits.</p>
<p class="grants">Funded by: AHRQ HHS: <span class="grant">R01HS006516</span>; British Heart Foundation: <span class="grant">PG/07/133/24260</span>; CSR NIH HHS: <span class="grant">RG2008/014</span>, <span class="grant">RG2008/08</span>; Cancer Research UK: <span class="grant">14136</span>, <span class="grant">C490/A10124</span>, <span class="grant">C490/A8339</span>; Chief Scientist Office: <span class="grant">CZB/4/672</span>, <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">85374</span>, <span class="grant">G0000649</span>, <span class="grant">G0000934</span>, <span class="grant">G0500539</span>, <span class="grant">G0600237</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G1000143</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MR/K006584/1</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NCI NIH HHS: <span class="grant">P01CA87969</span>, <span class="grant">R01CA047988</span>, <span class="grant">R01CA049449</span>, <span class="grant">R01CA050385</span>, <span class="grant">R01CA065725</span>, <span class="grant">R01CA067262</span>, <span class="grant">U01CA098233</span>; NCRR NIH HHS: <span class="grant">M01RR16500</span>, <span class="grant">U54RR020278</span>, <span class="grant">UL1RR025005</span>, <span class="grant">UL1RR033176</span>; NHGRI NIH HHS: <span class="grant">N01HG65403</span>, <span class="grant">R01HG002651</span>, <span class="grant">U01HG004402</span>, <span class="grant">Z01HG000024</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>, <span class="grant">N01HC25195</span>, <span class="grant">N01HC35129</span>, <span class="grant">N01HC85079</span>, <span class="grant">N01HC85080</span>, <span class="grant">N01HC85081</span>, <span class="grant">N01HC85082</span>, <span class="grant">N01HC85083</span>, <span class="grant">N01HC85084</span>, <span class="grant">N01HC85085</span>, <span class="grant">N01HC85086</span>, <span class="grant">N01HC85239</span>, <span class="grant">N02HL64278</span>, <span class="grant">R01HL036310</span>, <span class="grant">R01HL043851</span>, <span class="grant">R01HL059367</span>, <span class="grant">R01HL071981</span>, <span class="grant">R01HL075366</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL087647</span>, <span class="grant">R01HL087652</span>, <span class="grant">R01HL087676</span>, <span class="grant">R01HL087679</span>, <span class="grant">R01HL087700</span>, <span class="grant">R01HL088215</span>, <span class="grant">R01HL105756</span>, <span class="grant">U01HL069757</span>, <span class="grant">U01HL072515</span>, <span class="grant">U01HL080295</span>, <span class="grant">U01HL084729</span>, <span class="grant">U01HL084756</span>; NIA NIH HHS: <span class="grant">N01AG12100</span>, <span class="grant">N01AG12109</span>, <span class="grant">R01AG004563</span>, <span class="grant">R01AG008724</span>, <span class="grant">R01AG008861</span>, <span class="grant">R01AG010175</span>, <span class="grant">R01AG013196</span>, <span class="grant">R01AG015928</span>, <span class="grant">R01AG020098</span>, <span class="grant">R01AG023629</span>, <span class="grant">R01AG027058</span>, <span class="grant">R01AG028555</span>; NIAAA NIH HHS: <span class="grant">R01AA007535</span>, <span class="grant">R01AA013320</span>, <span class="grant">R01AA013321</span>, <span class="grant">R01AA013326</span>, <span class="grant">R01AA014041</span>; NICHD NIH HHS: <span class="grant">R01HD042157</span>; NIDA NIH HHS: <span class="grant">R01DA12854</span>; NIDDK NIH HHS: <span class="grant">K23DK080145</span>, <span class="grant">P01 DK088761</span>, <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30DK063491</span>, <span class="grant">P30DK072488</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK075787</span>, <span class="grant">R01 DK091718</span>, <span class="grant">R01DK062370</span>, <span class="grant">R01DK072193</span>, <span class="grant">R01DK073490</span>, <span class="grant">R01DK075681</span>, <span class="grant">R01DK075787</span>, <span class="grant">R01DK089256</span>, <span class="grant">U01DK062418</span>; NIGMS NIH HHS: <span class="grant">T32 GM007092</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706</span>, <span class="grant">R01MH059565</span>, <span class="grant">R01MH059566</span>, <span class="grant">R01MH059571</span>, <span class="grant">R01MH059586</span>, <span class="grant">R01MH059587</span>, <span class="grant">R01MH059588</span>, <span class="grant">R01MH060870</span>, <span class="grant">R01MH060879</span>, <span class="grant">R01MH061675</span>, <span class="grant">R01MH067257</span>, <span class="grant">R01MH079469</span>, <span class="grant">R01MH079470</span>, <span class="grant">R01MH081800</span>, <span class="grant">RL1MH083268</span>, <span class="grant">U24MH068457</span>; NLM NIH HHS: <span class="grant">R01LM010098</span>; OAPP OPHS HHS: <span class="grant">PG/02/128</span>; PHS HHS: <span class="grant">268200625226C</span>, <span class="grant">268201100005C</span>, <span class="grant">268201100006C</span>, <span class="grant">268201100007C</span>, <span class="grant">268201100008C</span>, <span class="grant">268201100009C</span>, <span class="grant">268201100010C</span>, <span class="grant">268201100011C</span>, <span class="grant">268201100012C</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">069224</span>, <span class="grant">072960</span>, <span class="grant">075491</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">076113/K/04/Z</span>, <span class="grant">079557</span>, <span class="grant">079895</span>, <span class="grant">081682</span>, <span class="grant">083270</span>, <span class="grant">084183/Z/07/Z</span>, <span class="grant">085301</span>, <span class="grant">086596/Z/08/Z</span>, <span class="grant">089061/Z/09/Z</span>, <span class="grant">089062/Z/09/Z</span>, <span class="grant">090532</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">6</span>;<span class="pagerange">e1003500</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23754948">23754948</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3674993">3674993</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003500">10.1371/journal.pgen.1003500</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer gene discovery: exploiting insertional mutagenesis.</h4>
<p class="authors"><span class="author">Ranzani M</span>, <span class="author">Annunziato S</span>, <span class="author">Adams DJ</span> and <span class="author">Montini E</span></p>
<p class="affiliation">San Raffaele-Telethon Institute for Gene Therapy, via Olgettina 58, 20132, Milan, Italy. <a href="mailto:%72%61%6E%7A%61%6E%69.%6D%61%72%63%6F%40%68%73%72.%69%74">&#114;&#97;&#110;&#122;&#97;&#110;&#105;&#46;&#109;&#97;&#114;&#99;&#111;&#64;&#104;&#115;&#114;&#46;&#105;&#116;</a>.</p><p class="abstract">Insertional mutagenesis has been used as a functional forward genetics screen for the identification of novel genes involved in the pathogenesis of human cancers. Different insertional mutagens have been successfully used to reveal new cancer genes. For example, retroviruses are integrating viruses with the capacity to induce the deregulation of genes in the neighborhood of the insertion site. Retroviruses have been used for more than 30 years to identify cancer genes in the hematopoietic system and mammary gland. Similarly, another tool that has revolutionized cancer gene discovery is the cut-and-paste transposons. These DNA elements have been engineered to contain strong promoters and stop cassettes that may function to perturb gene expression upon integration proximal to genes. In addition, complex mouse models characterized by tissue-restricted activity of transposons have been developed to identify oncogenes and tumor suppressor genes that control the development of a wide range of solid tumor types, extending beyond those tissues accessible using retrovirus-based approaches. Most recently, lentiviral vectors have appeared on the scene for use in cancer gene screens. Lentiviral vectors are replication-defective integrating vectors that have the advantage of being able to infect nondividing cells, in a wide range of cell types and tissues. In this review, we describe the various insertional mutagens focusing on their advantages/limitations, and we discuss the new and promising tools that will improve the insertional mutagenesis screens of the future. Mol Cancer Res; 11(10); 1141-58. &copy;2013 AACR.</p>
<p class="publication"><span class="title">Molecular cancer research : MCR</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">1141-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23928056">23928056</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1541-7786.MCR-13-0244">10.1158/1541-7786.MCR-13-0244</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cake: a bioinformatics pipeline for the integrated analysis of somatic variants in cancer genomes.</h4>
<p class="authors"><span class="author">Rashid M</span>, <span class="author">Robles-Espinoza CD</span>, <span class="author">Rust AG</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.</p><p class="abstract">We have developed Cake, a bioinformatics software pipeline that integrates four publicly available somatic variant-calling algorithms to identify single nucleotide variants with higher sensitivity and accuracy than any one algorithm alone. Cake can be run on a high-performance computer cluster or used as a stand-alone application. Availabilty: Cake is open-source and is available from http://cakesomatic.sourceforge.net/</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2013</span>;<span class="volume">29</span>;<span class="number">17</span>;<span class="pagerange">2208-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23803469">23803469</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3740632">3740632</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtt371">10.1093/bioinformatics/btt371</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.</h4>
<p class="authors"><span class="author">Rat Genome Sequencing and Mapping Consortium</span>, <span class="author">Baud A</span>, <span class="author">Hermsen R</span>, <span class="author">Guryev V</span>, <span class="author">Stridh P</span>, <span class="author">Graham D</span>, <span class="author">McBride MW</span>, <span class="author">Foroud T</span>, <span class="author">Calderari S</span>, <span class="author">Diez M</span>, <span class="author">Ockinger J</span>, <span class="author">Beyeen AD</span>, <span class="author">Gillett A</span>, <span class="author">Abdelmagid N</span>, <span class="author">Guerreiro-Cacais AO</span>, <span class="author">Jagodic M</span>, <span class="author">Tuncel J</span>, <span class="author">Norin U</span>, <span class="author">Beattie E</span>, <span class="author">Huynh N</span>, <span class="author">Miller WH</span>, <span class="author">Koller DL</span>, <span class="author">Alam I</span>, <span class="author">Falak S</span>, <span class="author">Osborne-Pellegrin M</span>, <span class="author">Martinez-Membrives E</span>, <span class="author">Canete T</span>, <span class="author">Blazquez G</span>, <span class="author">Vicens-Costa E</span>, <span class="author">Mont-Cardona C</span>, <span class="author">Diaz-Moran S</span>, <span class="author">Tobena A</span>, <span class="author">Hummel O</span>, <span class="author">Zelenika D</span>, <span class="author">Saar K</span>, <span class="author">Patone G</span>, <span class="author">Bauerfeind A</span>, <span class="author">Bihoreau MT</span>, <span class="author">Heinig M</span>, <span class="author">Lee YA</span>, <span class="author">Rintisch C</span>, <span class="author">Schulz H</span>, <span class="author">Wheeler DA</span>, <span class="author">Worley KC</span>, <span class="author">Muzny DM</span>, <span class="author">Gibbs RA</span>, <span class="author">Lathrop M</span>, <span class="author">Lansu N</span>, <span class="author">Toonen P</span>, <span class="author">Ruzius FP</span>, <span class="author">de Bruijn E</span>, <span class="author">Hauser H</span>, <span class="author">Adams DJ</span>, <span class="author">Keane T</span>, <span class="author">Atanur SS</span>, <span class="author">Aitman TJ</span>, <span class="author">Flicek P</span>, <span class="author">Malinauskas T</span>, <span class="author">Jones EY</span>, <span class="author">Ekman D</span>, <span class="author">Lopez-Aumatell R</span>, <span class="author">Dominiczak AF</span>, <span class="author">Johannesson M</span>, <span class="author">Holmdahl R</span>, <span class="author">Olsson T</span>, <span class="author">Gauguier D</span>, <span class="author">Hubner N</span>, <span class="author">Fernandez-Teruel A</span>, <span class="author">Cuppen E</span>, <span class="author">Mott R</span> and <span class="author">Flint J</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Oxford, UK.</p>
<p class="abstract">Genetic mapping on fully sequenced individuals is transforming understanding of the relationship between molecular variation and variation in complex traits. Here we report a combined sequence and genetic mapping analysis in outbred rats that maps 355 quantitative trait loci for 122 phenotypes. We identify 35 causal genes involved in 31 phenotypes, implicating new genes in models of anxiety, heart disease and multiple sclerosis. The relationship between sequence and genetic variation is unexpectedly complex: at approximately 40% of quantitative trait loci, a single sequence variant cannot account for the phenotypic effect. Using comparable sequence and mapping data from mice, we show that the extent and spatial pattern of variation in inbred rats differ substantially from those of inbred mice and that the genetic variants in orthologous genes rarely contribute to the same phenotype in both species.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">7</span>;<span class="pagerange">767-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23708188">23708188</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2644">10.1038/ng.2644</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of the thermopsin peptidase family (a5).</h4>
<p class="authors"><span class="author">Rawlings ND</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom ; European Molecular Biology Laboratory, European Bioinformatics Institute, Hinxton, Cambridgeshire, United Kingdom.</p>
<p class="abstract">Thermopsin is a peptidase from Sulfolobus acidocaldarius that is active at low pH and high temperature. From reversible inhibition with pepstatin, thermopsin is thought to be an aspartic peptidase. It is a member of the only family of peptidases to be restricted entirely to the archaea, namely peptidase family A5. Evolution within this family has been mapped, using a taxonomic tree based on the known classification of archaea. Homologues are found only in archaeans that are both hyperthermophiles and acidophiles, and this implies lateral transfer of genes between archaea, because species with homologues are not necessarily closely related. Despite the remarkable stability and activity in extreme conditions, no tertiary structure has been solved for any member of the family, and the catalytic mechanism is unknown. Putative catalytic residues have been predicted here by examination of aligned sequences.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e78998</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24312173">24312173</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3842238">3842238</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0078998">10.1371/journal.pone.0078998</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and prioritization of novel uncharacterized peptidases for biochemical characterization.</h4>
<p class="authors"><span class="author">Rawlings ND</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">Genome sequencing projects are generating enormous amounts of biological data that require analysis, which in turn identifies genes and proteins that require characterization. Enzymes that act on proteins are especially difficult to characterize because of the time required to distinguish one from another. This is particularly true of peptidases, the enzymes that activate, inactivate and degrade proteins. This article aims to identify clusters of sequences each of which represents the species variants of a single putative peptidase that is widely distributed and is thus merits biochemical characterization. The MEROPS database maintains large collections of sequences, references, substrate cleavage positions and inhibitor interactions of peptidases and their homologues. MEROPS also maintains a hierarchical classification of peptidase homologues, in which sequences are clustered as species variants of a single peptidase; homologous sequences are assembled into a family; and families are clustered into a clan. For each family, an alignment and a phylogenetic tree are generated. By assigning an identifier to a peptidase that has been biochemically characterized from a particular species (called a holotype), the identifier can be automatically extended to sequences from other species that cluster with the holotype. This permits transference of annotation from the holotype to other members of the cluster. By extending this concept to all peptidase homologues (including those of unknown function that have not been characterized) from model organisms representing all the major divisions of cellular life, clusters of sequences representing putative peptidases can also be identified. The 42 most widely distributed of these putative peptidases have been identified and discussed here and are prioritized as ideal candidates for biochemical characterization. Database URL: http://merops.sanger.ac.uk.</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2013</span>;<span class="volume">2013</span>;<span class="pagerange">bat022</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23584835">23584835</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3625958">3625958</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbat022">10.1093/database/bat022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ADAM15 Peptidase</h4>
<p class="authors"><span class="author">Rawlings, N.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">253</span>;<span class="pagerange">1122-5</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00253-2">10.1016/B978-0-12-382219-2.00253-2</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192002532" href="http://www.sciencedirect.com/science/article/pii/B9780123822192002532">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>002532</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: Metallopeptidases and Their Clans</h4>
<p class="authors"><span class="author">Rawlings, N.</span> and <span class="author">Barrett, A.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">77</span>;<span class="pagerange">325-70</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00077-6">10.1016/B978-0-12-382219-2.00077-6</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192000776" href="http://www.sciencedirect.com/science/article/pii/B9780123822192000776">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>000776</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: The Clans and Families of Cysteine Peptidases</h4>
<p class="authors"><span class="author">Rawlings, N.</span> and <span class="author">Barrett, A.</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="number">404</span>;<span class="pagerange">1743-73</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00404-X">10.1016/B978-0-12-382219-2.00404-X</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B978012382219200404X" href="http://www.sciencedirect.com/science/article/pii/B978012382219200404X">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>00404X</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: Clan PB Containing N-terminal Nucleophile Peptidases</h4>
<p class="authors"><span class="author">Rawlings, N.</span> and <span class="author">Barrett, A</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">808</span>;<span class="pagerange">3648-53</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00808-5">10.1016/B978-0-12-382219-2.00808-5</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192008085" href="http://www.sciencedirect.com/science/article/pii/B9780123822192008085">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>008085</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: Aspartic and Glutamic Peptidases and Their Clans</h4>
<p class="authors"><span class="author">Rawlings, N;</span> and <span class="author">Barrett, A</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">3-19</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00001-6">10.1016/B978-0-12-382219-2.00001-6</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192000016" href="http://www.sciencedirect.com/science/article/pii/B9780123822192000016">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>000016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antarease</h4>
<p class="authors"><span class="author">Rawlings, N</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">246</span>;<span class="pagerange">1079-81</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00246-5">10.1016/B978-0-12-382219-2.00246-5</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192002465" href="http://www.sciencedirect.com/science/article/pii/B9780123822192002465">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>002465</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacteriophage T4 Prohead Endopeptidase</h4>
<p class="authors"><span class="author">Rawlings, N</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">789</span>;<span class="pagerange">3560-2</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00789-4">10.1016/B978-0-12-382219-2.00789-4</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192007894" href="http://www.sciencedirect.com/science/article/pii/B9780123822192007894">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>007894</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: Peptidases of Unknown Catalytic Type</h4>
<p class="authors"><span class="author">Rawlings, N</span> and <span class="author">Barrett, A</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">827</span>;<span class="pagerange">3747-9</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00827-9">10.1016/B978-0-12-382219-2.00827-9</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192008279" href="http://www.sciencedirect.com/science/article/pii/B9780123822192008279">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>008279</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Introduction: Serine Peptidases and Their Clans</h4>
<p class="authors"><span class="author">Rawlings, N</span> and <span class="author">Barrett, A</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">559</span>;<span class="pagerange">2491-523</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00559-7">10.1016/B978-0-12-382219-2.00559-7</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192005597" href="http://www.sciencedirect.com/science/article/pii/B9780123822192005597">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>005597</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Handbook of Proteolytic Enzymes</h4>
<p class="authors"><span class="author">Rawlings,N.D.</span> and <span class="author">Salvesen,G.;</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span></p>
<p class="links">URL: <a title="http://www.sciencedirect.com/science/book/9780123822192" href="http://www.sciencedirect.com/science/book/9780123822192">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/book/9780123</span>822192</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genes involved in host-parasite interactions can be revealed by their correlated expression.</h4>
<p class="authors"><span class="author">Reid AJ</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Parasite genomics group, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%61%72%31%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#49;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Molecular interactions between a parasite and its host are key to the ability of the parasite to enter the host and persist. Our understanding of the genes and proteins involved in these interactions is limited. To better understand these processes it would be advantageous to have a range of methods to predict pairs of genes involved in such interactions. Correlated gene expression profiles can be used to identify molecular interactions within a species. Here we have extended the concept to different species, showing that genes with correlated expression are more likely to encode proteins, which directly or indirectly participate in host-parasite interaction. We go on to examine our predictions of molecular interactions between the malaria parasite and both its mammalian host and insect vector. Our approach could be applied to study any interaction between species, for example, between a host and its parasites or pathogens, but also symbiotic and commensal pairings.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span>;<span class="volume">41</span>;<span class="number">3</span>;<span class="pagerange">1508-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23275547">23275547</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3561955">3561955</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgks1340">10.1093/nar/gks1340</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Secretory meningiomas are defined by combined KLF4 K409Q and TRAF7 mutations.</h4>
<p class="authors"><span class="author">Reuss DE</span>, <span class="author">Piro RM</span>, <span class="author">Jones DT</span>, <span class="author">Simon M</span>, <span class="author">Ketter R</span>, <span class="author">Kool M</span>, <span class="author">Becker A</span>, <span class="author">Sahm F</span>, <span class="author">Pusch S</span>, <span class="author">Meyer J</span>, <span class="author">Hagenlocher C</span>, <span class="author">Schweizer L</span>, <span class="author">Capper D</span>, <span class="author">Kickingereder P</span>, <span class="author">Mucha J</span>, <span class="author">Koelsche C</span>, <span class="author">J&auml;ger N</span>, <span class="author">Santarius T</span>, <span class="author">Tarpey PS</span>, <span class="author">Stephens PJ</span>, <span class="author">Andrew Futreal P</span>, <span class="author">Wellenreuther R</span>, <span class="author">Kraus J</span>, <span class="author">Lenartz D</span>, <span class="author">Herold-Mende C</span>, <span class="author">Hartmann C</span>, <span class="author">Mawrin C</span>, <span class="author">Giese N</span>, <span class="author">Eils R</span>, <span class="author">Collins VP</span>, <span class="author">K&ouml;nig R</span>, <span class="author">Wiestler OD</span>, <span class="author">Pfister SM</span> and <span class="author">von Deimling A</span></p>
<p class="affiliation">Department of Neuropathology, Institute of Pathology, Ruprecht-Karls-University Heidelberg, Im Neuenheimer Feld 224, 69120, Heidelberg, Germany.</p><p class="abstract">Meningiomas are among the most frequent intracranial tumors. The secretory variant of meningioma is characterized by glandular differentiation, formation of intracellular lumina and pseudopsammoma bodies, expression of a distinct pattern of cytokeratins and clinically by pronounced perifocal brain edema. Here we describe whole-exome sequencing analysis of DNA from 16 secretory meningiomas and corresponding constitutional tissues. All secretory meningiomas invariably harbored a mutation in both KLF4 and TRAF7. Validation in an independent cohort of 14 secretory meningiomas by Sanger sequencing or derived cleaved amplified polymorphic sequence (dCAPS) assay detected the same pattern, with KLF4 mutations observed in a total of 30/30 and TRAF7 mutations in 29/30 of these tumors. All KLF4 mutations were identical, affected codon 409 and resulted in a lysine to glutamine exchange (K409Q). KLF4 mutations were not found in 89 non-secretory meningiomas, 267 other intracranial tumors including gliomas, glioneuronal tumors, pituitary adenomas and metastases, 59 peripheral nerve sheath tumors and 52 pancreatic tumors. TRAF7 mutations were restricted to the WD40 domains. While KLF4 mutations were exclusively seen in secretory meningiomas, TRAF7 mutations were also observed in 7/89 (8 %) of non-secretory meningiomas. KLF4 and TRAF7 mutations were mutually exclusive with NF2 mutations. In conclusion, our findings suggest an essential contribution of combined KLF4 K409Q and TRAF7 mutations in the genesis of secretory meningioma and demonstrate a role for TRAF7 alterations in other non-NF2 meningiomas.</p>
<p class="publication"><span class="title">Acta neuropathologica</span>&nbsp;<span class="year">2013</span>;<span class="volume">125</span>;<span class="number">3</span>;<span class="pagerange">351-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23404370">23404370</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00401-013-1093-x">10.1007/s00401-013-1093-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid bacterial whole-genome sequencing to enhance diagnostic and public health microbiology.</h4>
<p class="authors"><span class="author">Reuter S</span>, <span class="author">Ellington MJ</span>, <span class="author">Cartwright EJ</span>, <span class="author">K&ouml;ser CU</span>, <span class="author">T&ouml;r&ouml;k ME</span>, <span class="author">Gouliouris T</span>, <span class="author">Harris SR</span>, <span class="author">Brown NM</span>, <span class="author">Holden MT</span>, <span class="author">Quail M</span>, <span class="author">Parkhill J</span>, <span class="author">Smith GP</span>, <span class="author">Bentley SD</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, England.</p><p class="abstract"><strong>Importance</strong>: The latest generation of benchtop DNA sequencing platforms can provide an accurate whole-genome sequence (WGS) for a broad range of bacteria in less than a day. These could be used to more effectively contain the spread of multidrug-resistant pathogens.</p><p class="abstract"><strong>Objective</strong>: To compare WGS with standard clinical microbiology practice for the investigation of nosocomial outbreaks caused by multidrug-resistant bacteria, the identification of genetic determinants of antimicrobial resistance, and typing of other clinically important pathogens.</p><p class="abstract">A laboratory-based study of hospital inpatients with a range of bacterial infections at Cambridge University Hospitals NHS Foundation Trust, a secondary and tertiary referral center in England, comparing WGS with standard diagnostic microbiology using stored bacterial isolates and clinical information.</p><p class="abstract">Specimens were taken and processed as part of routine clinical care, and cultured isolates stored and referred for additional reference laboratory testing as necessary. Isolates underwent DNA extraction and library preparation prior to sequencing on the Illumina MiSeq platform. Bioinformatic analyses were performed by persons blinded to the clinical, epidemiologic, and antimicrobial susceptibility data.</p><p class="abstract"><strong>Results</strong>: We investigated 2 putative nosocomial outbreaks, one caused by vancomycin-resistant Enterococcus faecium and the other by carbapenem-resistant Enterobacter cloacae; WGS accurately discriminated between outbreak and nonoutbreak isolates and was superior to conventional typing methods. We compared WGS with standard methods for the identification of the mechanism of carbapenem resistance in a range of gram-negative bacteria (Acinetobacter baumannii, E cloacae, Escherichia coli, and Klebsiella pneumoniae). This demonstrated concordance between phenotypic and genotypic results, and the ability to determine whether resistance was attributable to the presence of carbapenemases or other resistance mechanisms. Whole-genome sequencing was used to recapitulate reference laboratory typing of clinical isolates of Neisseria meningitidis and to provide extended phylogenetic analyses of these.</p><p class="abstract">The speed, accuracy, and depth of information provided by WGS platforms to confirm or refute outbreaks in hospitals and the community, and to accurately define transmission of multidrug-resistant and other organisms, represents an important advance.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1000803</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">JAMA internal medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">173</span>;<span class="number">15</span>;<span class="pagerange">1397-404</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23857503">23857503</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4001082">4001082</a>; DOI: <a href="http://dx.doi.org/10.1001%2Fjamainternmed.2013.7734">10.1001/jamainternmed.2013.7734</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A pilot study of rapid whole-genome sequencing for the investigation of a Legionella outbreak.</h4>
<p class="authors"><span class="author">Reuter S</span>, <span class="author">Harrison TG</span>, <span class="author">K&ouml;ser CU</span>, <span class="author">Ellington MJ</span>, <span class="author">Smith GP</span>, <span class="author">Parkhill J</span>, <span class="author">Peacock SJ</span>, <span class="author">Bentley SD</span> and <span class="author">T&ouml;r&ouml;k ME</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, UK.</p>
<p class="abstract">Objectives: Epidemiological investigations of Legionnaires' disease outbreaks rely on the rapid identification and typing of clinical and environmental Legionella isolates in order to identify and control the source of infection. Rapid bacterial whole-genome sequencing (WGS) is an emerging technology that has the potential to rapidly discriminate outbreak from non-outbreak isolates in a clinically relevant time frame.

Methods: We performed a pilot study to determine the feasibility of using bacterial WGS to differentiate outbreak from non-outbreak isolates collected during an outbreak of Legionnaires' disease. Seven Legionella isolates (three clinical and four environmental) were obtained from the reference laboratory and sequenced using the Illumina MiSeq platform at Addenbrooke's Hospital, Cambridge. Bioinformatic analysis was performed blinded to the epidemiological data at the Wellcome Trust Sanger Institute.

Results: We were able to distinguish outbreak from non-outbreak isolates using bacterial WGS, and to confirm the probable environmental source. Our analysis also highlighted constraints, which were the small number of Legionella pneumophila isolates available for sequencing, and the limited number of published genomes for comparison.

Conclusions: We have demonstrated the feasibility of using rapid WGS to investigate an outbreak of Legionnaires' disease. Future work includes building larger genomic databases of L pneumophila from both clinical and environmental sources, developing automated data interpretation software, and conducting a cost-benefit analysis of WGS versus current typing methods.</p>
<p class="publication"><span class="title">BMJ open</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="number">1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23306006" rel="external">23306006</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fbmjopen-2012-002175" rel="external">10.1136/bmjopen-2012-002175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using Varying Negative Examples to Improve Computational Predictions of Transcription Factor Binding Sites</h4>
<p class="authors"><span class="author">Rezwan,F.;</span>, <span class="author">Sun,Y.;</span>, <span class="author">Davey,N.;</span>, <span class="author">Adams,R.;</span>, <span class="author">RUST,A.G.;</span> and <span class="author">Robinson,M.</span></p>
<p class="publication"><span class="title">Communications in Computer and Information Science</span>&nbsp;<span class="year">2013</span>;<span class="volume">311</span>;<span class="pagerange">234-43</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-32909-8_24">10.1007/978-3-642-32909-8_24</a>; URL: <a title="http://link.springer.com/chapter/10.1007%2F978-3-642-32909-8_24" href="http://link.springer.com/chapter/10.1007%2F978-3-642-32909-8_24">http://link.springer.com/chapter<span class="print-hide">...</span><span class="print-show">/10.1007%2F978-3-642-32</span>909-8_24</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two new cytotypes reinforce that Micronycteris hirsuta Peters, 1869 does not represent a monotypic taxon.</h4>
<p class="authors"><span class="author">Ribas TF</span>, <span class="author">Rodrigues LR</span>, <span class="author">Nagamachi CY</span>, <span class="author">Gomes AJ</span>, <span class="author">Benathar TC</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Yang F</span>, <span class="author">Ferguson-Smith MA</span> and <span class="author">Pieczarka JC</span></p>
<p class="affiliation">Laborat&oacute;rio de Citogen&eacute;tica, Instituto de Ci&ecirc;ncias Biol&oacute;gicas, Universidade Federal do Par&aacute;, Campus do Guam&aacute;, Av, Perimetral, sn, Guam&aacute;, Bel&eacute;m, Par&aacute;, 66075-900, Brazil. <a href="mailto:%6A%75%6C%69%6F%70%69%65%63%7A%61%72%6B%61%40%67%6D%61%69%6C.%63%6F%6D">&#106;&#117;&#108;&#105;&#111;&#112;&#105;&#101;&#99;&#122;&#97;&#114;&#107;&#97;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a>.</p><p class="abstract"><strong>Background</strong>: The genus Micronycteris is a diverse group of phyllostomid bats currently comprising 11 species, with diploid number (2n) ranging from 26 to 40 chromosomes. The karyotypic relationships within Micronycteris and between Micronycteris and other phyllostomids remain poorly understood. The karyotype of Micronycteris hirsuta is of particular interest: three different diploid numbers were reported for this species in South and Central Americas with 2n = 26, 28 and 30 chromosomes. Although current evidence suggests some geographic differentiation among populations of M. hirsuta based on chromosomal, morphological, and nuclear and mitochondrial DNA markers, the recognition of new species or subspecies has been avoided due to the need for additional data, mainly chromosomal data.</p><p class="abstract"><strong>Results</strong>: We describe two new cytotypes for Micronycteris hirsuta (MHI) (2n = 26 and 25, NF = 32), whose differences in diploid number are interpreted as the products of Robertsonian rearrangements. C-banding revealed a small amount of constitutive heterochromatin at the centromere and the NOR was located in the interstitial portion of the short arm of a second pair, confirmed by FISH. Telomeric probes hybridized to the centromeric regions and weakly to telomeric regions of most chromosomes. The G-banding analysis and chromosome painting with whole chromosome probes from Carollia brevicauda (CBR) and Phyllostomus hastatus (PHA) enabled the establishment of genome-wide homologies between MHI, CBR and PHA.</p><p class="abstract"><strong>Conclusions</strong>: The karyotypes of Brazilian specimens of Micronycteris hirsuta described here are new to Micronycteris and reinforce that M. hirsuta does not represent a monotypic taxon. Our results corroborate the hypothesis of karyotypic megaevolution within Micronycteris, and strong evidence for this is that the entire chromosome complement of M. hirsuta was shown to be derivative with respect to species compared in this study.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">119</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24359225">24359225</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3878135">3878135</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2156-14-119">10.1186/1471-2156-14-119</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GWAS of 126,559 Individuals Identifies Genetic Variants Associated with Educational Attainment.</h4>
<p class="authors"><span class="author">Rietveld CA</span>, <span class="author">Medland SE</span>, <span class="author">Derringer J</span>, <span class="author">Yang J</span>, <span class="author">Esko T</span>, <span class="author">Martin NW</span>, <span class="author">Westra HJ</span>, <span class="author">Shakhbazov K</span>, <span class="author">Abdellaoui A</span>, <span class="author">Agrawal A</span>, <span class="author">Albrecht E</span>, <span class="author">Alizadeh BZ</span>, <span class="author">Amin N</span>, <span class="author">Barnard J</span>, <span class="author">Baumeister SE</span>, <span class="author">Benke KS</span>, <span class="author">Bielak LF</span>, <span class="author">Boatman JA</span>, <span class="author">Boyle PA</span>, <span class="author">Davies G</span>, <span class="author">de Leeuw C</span>, <span class="author">Eklund N</span>, <span class="author">Evans DS</span>, <span class="author">Ferhmann R</span>, <span class="author">Fischer K</span>, <span class="author">Gieger C</span>, <span class="author">Gjessing HK</span>, <span class="author">H&auml;gg S</span>, <span class="author">Harris JR</span>, <span class="author">Hayward C</span>, <span class="author">Holzapfel C</span>, <span class="author">Ibrahim-Verbaas CA</span>, <span class="author">Ingelsson E</span>, <span class="author">Jacobsson B</span>, <span class="author">Joshi PK</span>, <span class="author">Jugessur A</span>, <span class="author">Kaakinen M</span>, <span class="author">Kanoni S</span>, <span class="author">Karjalainen J</span>, <span class="author">Kolcic I</span>, <span class="author">Kristiansson K</span>, <span class="author">Kutalik Z</span>, <span class="author">Lahti J</span>, <span class="author">Lee SH</span>, <span class="author">Lin P</span>, <span class="author">Lind PA</span>, <span class="author">Liu Y</span>, <span class="author">Lohman K</span>, <span class="author">Loitfelder M</span>, <span class="author">McMahon G</span>, <span class="author">Vidal PM</span>, <span class="author">Meirelles O</span>, <span class="author">Milani L</span>, <span class="author">Myhre R</span>, <span class="author">Nuotio ML</span>, <span class="author">Oldmeadow CJ</span>, <span class="author">Petrovic KE</span>, <span class="author">Peyrot WJ</span>, <span class="author">Polasek O</span>, <span class="author">Quaye L</span>, <span class="author">Reinmaa E</span>, <span class="author">Rice JP</span>, <span class="author">Rizzi TS</span>, <span class="author">Schmidt H</span>, <span class="author">Schmidt R</span>, <span class="author">Smith AV</span>, <span class="author">Smith JA</span>, <span class="author">Tanaka T</span>, <span class="author">Terracciano A</span>, <span class="author">van der Loos MJ</span>, <span class="author">Vitart V</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wellmann J</span>, <span class="author">Yu L</span>, <span class="author">Zhao W</span>, <span class="author">Allik J</span>, <span class="author">Attia JR</span>, <span class="author">Bandinelli S</span>, <span class="author">Bastardot F</span>, <span class="author">Beauchamp J</span>, <span class="author">Bennett DA</span>, <span class="author">Berger K</span>, <span class="author">Bierut LJ</span>, <span class="author">Boomsma DI</span>, <span class="author">B&uuml;ltmann U</span>, <span class="author">Campbell H</span>, <span class="author">Chabris CF</span>, <span class="author">Cherkas L</span>, <span class="author">Chung MK</span>, <span class="author">Cucca F</span>, <span class="author">de Andrade M</span>, <span class="author">De Jager PL</span>, <span class="author">De Neve JE</span>, <span class="author">Deary IJ</span>, <span class="author">Dedoussis GV</span>, <span class="author">Deloukas P</span>, <span class="author">Dimitriou M</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Elderson MF</span>, <span class="author">Eriksson JG</span>, <span class="author">Evans DM</span>, <span class="author">Faul JD</span>, <span class="author">Ferrucci L</span>, <span class="author">Garcia ME</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Gudnason V</span>, <span class="author">Hall P</span>, <span class="author">Harris JM</span>, <span class="author">Harris TB</span>, <span class="author">Hastie ND</span>, <span class="author">Heath AC</span>, <span class="author">Hernandez DG</span>, <span class="author">Hoffmann W</span>, <span class="author">Hofman A</span>, <span class="author">Holle R</span>, <span class="author">Holliday EG</span>, <span class="author">Hottenga JJ</span>, <span class="author">Iacono WG</span>, <span class="author">Illig T</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kaprio J</span>, <span class="author">Kirkpatrick RM</span>, <span class="author">Kowgier M</span>, <span class="author">Latvala A</span>, <span class="author">Launer LJ</span>, <span class="author">Lawlor DA</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Li J</span>, <span class="author">Lichtenstein P</span>, <span class="author">Lichtner P</span>, <span class="author">Liewald DC</span>, <span class="author">Madden PA</span>, <span class="author">Magnusson PK</span>, <span class="author">M&auml;kinen TE</span>, <span class="author">Masala M</span>, <span class="author">McGue M</span>, <span class="author">Metspalu A</span>, <span class="author">Mielck A</span>, <span class="author">Miller MB</span>, <span class="author">Montgomery GW</span>, <span class="author">Mukherjee S</span>, <span class="author">Nyholt DR</span>, <span class="author">Oostra BA</span>, <span class="author">Palmer LJ</span>, <span class="author">Palotie A</span>, <span class="author">Penninx B</span>, <span class="author">Perola M</span>, <span class="author">Peyser PA</span>, <span class="author">Preisig M</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Raitakari OT</span>, <span class="author">Realo A</span>, <span class="author">Ring SM</span>, <span class="author">Ripatti S</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rudan I</span>, <span class="author">Rustichini A</span>, <span class="author">Salomaa V</span>, <span class="author">Sarin AP</span>, <span class="author">Schlessinger D</span>, <span class="author">Scott RJ</span>, <span class="author">Snieder H</span>, <span class="author">St Pourcain B</span>, <span class="author">Starr JM</span>, <span class="author">Sul JH</span>, <span class="author">Surakka I</span>, <span class="author">Svento R</span>, <span class="author">Teumer A</span>, <span class="author">The LifeLines Cohort Study</span>, <span class="author">Tiemeier H</span>, <span class="author">van Rooij FJ</span>, <span class="author">Van Wagoner DR</span>, <span class="author">Vartiainen E</span>, <span class="author">Viikari J</span>, <span class="author">Vollenweider P</span>, <span class="author">Vonk JM</span>, <span class="author">Waeber G</span>, <span class="author">Weir DR</span>, <span class="author">Wichmann HE</span>, <span class="author">Widen E</span>, <span class="author">Willemsen G</span>, <span class="author">Wilson JF</span>, <span class="author">Wright AF</span>, <span class="author">Conley D</span>, <span class="author">Davey-Smith G</span>, <span class="author">Franke L</span>, <span class="author">Groenen PJ</span>, <span class="author">Hofman A</span>, <span class="author">Johannesson M</span>, <span class="author">Kardia SL</span>, <span class="author">Krueger RF</span>, <span class="author">Laibson D</span>, <span class="author">Martin NG</span>, <span class="author">Meyer MN</span>, <span class="author">Posthuma D</span>, <span class="author">Thurik AR</span>, <span class="author">Timpson NJ</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">Visscher PM</span>, <span class="author">Benjamin DJ</span>, <span class="author">Cesarini D</span> and <span class="author">Koellinger PD</span></p>
<p class="affiliation">Department of Applied Economics, Erasmus School of Economics, Erasmus University Rotterdam, 3000 DR Rotterdam, The Netherlands.</p>
<p class="abstract">A genome-wide association study of educational attainment was conducted in a discovery sample of 101,069 individuals and a replication sample of 25,490. Three independent SNPs are genome-wide significant (rs9320913, rs11584700, rs4851266), and all three replicate. Estimated effects sizes are small (R(2) â‰ˆ 0.02%), approximately 1 month of schooling per allele. A linear polygenic score from all measured SNPs accounts for â‰ˆ 2% of the variance in both educational attainment and cognitive function. Genes in the region of the loci have previously been associated with health, cognitive, and central nervous system phenotypes, and bioinformatics analyses suggest the involvement of the anterior caudate nucleus. These findings provide promising candidate SNPs for follow-up work, and our effect size estimates can anchor power analyses in social-science genetics.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23722424">23722424</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1235488">10.1126/science.1235488</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The first structure in a family of peptidase inhibitors reveals an unusual Ig-like fold.</h4>
<p class="authors"><span class="author">Rigden DJ</span>, <span class="author">Xu Q</span>, <span class="author">Chang Y</span>, <span class="author">Eberhardt RY</span>, <span class="author">Finn RD</span> and <span class="author">Rawlings ND</span></p>
<p class="affiliation">Institute of Integrative Biology, University of Liverpool, Liverpool, L69 7ZB, UK.</p><p class="abstract">We report the crystal structure solution of the Intracellular Protease Inhibitor (IPI) protein from Bacillus subtilis, which has been reported to be an inhibitor of the intracellular subtilisin Isp1 from the same organism. The structure of IPI is a variant of the all-beta, immunoglobulin (Ig) fold. It is possible that IPI is important for protein-protein interactions, of which inhibition of Isp1 is one. The intracellular nature of ISP is questioned, because an alternative ATG codon in the ipi gene would produce a protein with an N-terminal extension containing a signal peptide. It is possible that alternative initiation exists, producing either an intracellular inhibitor or a secreted form that may be associated with the cell surface.&nbsp; Homologues of the IPI protein from other species are multi-domain proteins, containing signal peptides and domains also associated with the bacterial cell-surface. The cysteine peptidase inhibitors chagasin and amoebiasin also have Ig-like folds, but their topology differs significantly from that of IPI, and they share no recent common ancestor. A model of IPI docked to Isp1 shows similarities to other subtilisin:inhibitor complexes, particularly where the inhibitor interacts with the peptidase active site.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54 GM094586</span></p>
<p class="publication"><span class="title">F1000Research</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="pagerange">154</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24555072">24555072</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3901451">3901451</a>; DOI: <a href="http://dx.doi.org/10.12688%2Ff1000research.2-154.v2">10.12688/f1000research.2-154.v2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis identifies 13 new risk loci for schizophrenia.</h4>
<p class="authors"><span class="author">Ripke S</span>, <span class="author">O&#39;Dushlaine C</span>, <span class="author">Chambert K</span>, <span class="author">Moran JL</span>, <span class="author">K&auml;hler AK</span>, <span class="author">Akterin S</span>, <span class="author">Bergen SE</span>, <span class="author">Collins AL</span>, <span class="author">Crowley JJ</span>, <span class="author">Fromer M</span>, <span class="author">Kim Y</span>, <span class="author">Lee SH</span>, <span class="author">Magnusson PK</span>, <span class="author">Sanchez N</span>, <span class="author">Stahl EA</span>, <span class="author">Williams S</span>, <span class="author">Wray NR</span>, <span class="author">Xia K</span>, <span class="author">Bettella F</span>, <span class="author">Borglum AD</span>, <span class="author">Bulik-Sullivan BK</span>, <span class="author">Cormican P</span>, <span class="author">Craddock N</span>, <span class="author">de Leeuw C</span>, <span class="author">Durmishi N</span>, <span class="author">Gill M</span>, <span class="author">Golimbet V</span>, <span class="author">Hamshere ML</span>, <span class="author">Holmans P</span>, <span class="author">Hougaard DM</span>, <span class="author">Kendler KS</span>, <span class="author">Lin K</span>, <span class="author">Morris DW</span>, <span class="author">Mors O</span>, <span class="author">Mortensen PB</span>, <span class="author">Neale BM</span>, <span class="author">O&#39;Neill FA</span>, <span class="author">Owen MJ</span>, <span class="author">Milovancevic MP</span>, <span class="author">Posthuma D</span>, <span class="author">Powell J</span>, <span class="author">Richards AL</span>, <span class="author">Riley BP</span>, <span class="author">Ruderfer D</span>, <span class="author">Rujescu D</span>, <span class="author">Sigurdsson E</span>, <span class="author">Silagadze T</span>, <span class="author">Smit AB</span>, <span class="author">Stefansson H</span>, <span class="author">Steinberg S</span>, <span class="author">Suvisaari J</span>, <span class="author">Tosato S</span>, <span class="author">Verhage M</span>, <span class="author">Walters JT</span>, <span class="author">Multicenter Genetic Studies of Schizophrenia Consortium</span>, <span class="author">Levinson DF</span>, <span class="author">Gejman PV</span>, <span class="author">Kendler KS</span>, <span class="author">Laurent C</span>, <span class="author">Mowry BJ</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Owen MJ</span>, <span class="author">Pulver AE</span>, <span class="author">Riley BP</span>, <span class="author">Schwab SG</span>, <span class="author">Wildenauer DB</span>, <span class="author">Dudbridge F</span>, <span class="author">Holmans P</span>, <span class="author">Shi J</span>, <span class="author">Albus M</span>, <span class="author">Alexander M</span>, <span class="author">Campion D</span>, <span class="author">Cohen D</span>, <span class="author">Dikeos D</span>, <span class="author">Duan J</span>, <span class="author">Eichhammer P</span>, <span class="author">Godard S</span>, <span class="author">Hansen M</span>, <span class="author">Lerer FB</span>, <span class="author">Liang KY</span>, <span class="author">Maier W</span>, <span class="author">Mallet J</span>, <span class="author">Nertney DA</span>, <span class="author">Nestadt G</span>, <span class="author">Norton N</span>, <span class="author">O&#39;Neill FA</span>, <span class="author">Papadimitriou GN</span>, <span class="author">Ribble R</span>, <span class="author">Sanders AR</span>, <span class="author">Silverman JM</span>, <span class="author">Walsh D</span>, <span class="author">Williams NM</span>, <span class="author">Wormley B</span>, <span class="author">Psychosis Endophenotypes International Consortium</span>, <span class="author">Arranz MJ</span>, <span class="author">Bakker S</span>, <span class="author">Bender S</span>, <span class="author">Bramon E</span>, <span class="author">Collier D</span>, <span class="author">Crespo-Facorro B</span>, <span class="author">Hall J</span>, <span class="author">Iyegbe C</span>, <span class="author">Jablensky A</span>, <span class="author">Kahn RS</span>, <span class="author">Kalaydjieva L</span>, <span class="author">Lawrie S</span>, <span class="author">Lewis CM</span>, <span class="author">Lin K</span>, <span class="author">Linszen DH</span>, <span class="author">Mata I</span>, <span class="author">McIntosh A</span>, <span class="author">Murray RM</span>, <span class="author">Ophoff RA</span>, <span class="author">Powell J</span>, <span class="author">Rujescu D</span>, <span class="author">Van Os J</span>, <span class="author">Walshe M</span>, <span class="author">Weisbrod M</span>, <span class="author">Wiersma D</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Donnelly P</span>, <span class="author">Barroso I</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Casas JP</span>, <span class="author">Corvin AP</span>, <span class="author">Deloukas P</span>, <span class="author">Duncanson A</span>, <span class="author">Jankowski J</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">Palmer CN</span>, <span class="author">Plomin R</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Trembath RC</span>, <span class="author">Viswanathan AC</span>, <span class="author">Wood NW</span>, <span class="author">Spencer CC</span>, <span class="author">Band G</span>, <span class="author">Bellenguez C</span>, <span class="author">Freeman C</span>, <span class="author">Hellenthal G</span>, <span class="author">Giannoulatou E</span>, <span class="author">Pirinen M</span>, <span class="author">Pearson RD</span>, <span class="author">Strange A</span>, <span class="author">Su Z</span>, <span class="author">Vukcevic D</span>, <span class="author">Donnelly P</span>, <span class="author">Langford C</span>, <span class="author">Hunt SE</span>, <span class="author">Edkins S</span>, <span class="author">Gwilliam R</span>, <span class="author">Blackburn H</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Dronov S</span>, <span class="author">Gillman M</span>, <span class="author">Gray E</span>, <span class="author">Hammond N</span>, <span class="author">Jayakumar A</span>, <span class="author">McCann OT</span>, <span class="author">Liddle J</span>, <span class="author">Potter SC</span>, <span class="author">Ravindrarajah R</span>, <span class="author">Ricketts M</span>, <span class="author">Tashakkori-Ghanbaria A</span>, <span class="author">Waller MJ</span>, <span class="author">Weston P</span>, <span class="author">Widaa S</span>, <span class="author">Whittaker P</span>, <span class="author">Barroso I</span>, <span class="author">Deloukas P</span>, <span class="author">Mathew CG</span>, <span class="author">Blackwell JM</span>, <span class="author">Brown MA</span>, <span class="author">Corvin AP</span>, <span class="author">McCarthy MI</span>, <span class="author">Spencer CC</span>, <span class="author">Bramon E</span>, <span class="author">Corvin AP</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Stefansson K</span>, <span class="author">Scolnick E</span>, <span class="author">Purcell S</span>, <span class="author">McCarroll SA</span>, <span class="author">Sklar P</span>, <span class="author">Hultman CM</span> and <span class="author">Sullivan PF</span></p>
<p class="affiliation">1] Analytical and Translational Genetics Unit, Massachusetts General Hospital, Boston, Massachusetts, USA. [2] Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, Massachusetts, USA. [3].</p><p class="abstract">Schizophrenia is an idiopathic mental disorder with a heritable component and a substantial public health impact. We conducted a multi-stage genome-wide association study (GWAS) for schizophrenia beginning with a Swedish national sample (5,001 cases and 6,243 controls) followed by meta-analysis with previous schizophrenia GWAS (8,832 cases and 12,067 controls) and finally by replication of SNPs in 168 genomic regions in independent samples (7,413 cases, 19,762 controls and 581 parent-offspring trios). We identified 22 loci associated at genome-wide significance; 13 of these are new, and 1 was previously implicated in bipolar disorder. Examination of candidate genes at these loci suggests the involvement of neuronal calcium signaling. We estimate that 8,300 independent, mostly common SNPs (95% credible interval of 6,300-10,200 SNPs) contribute to risk for schizophrenia and that these collectively account for at least 32% of the variance in liability. Common genetic variation has an important role in the etiology of schizophrenia, and larger studies will allow more detailed understanding of this disorder.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600429</span>, <span class="grant">G0601635</span>, <span class="grant">G0800509</span>, <span class="grant">G1000718</span>, <span class="grant">G1100583</span>; NIMH NIH HHS: <span class="grant">K01 MH094406</span>, <span class="grant">R01 MH077139</span>, <span class="grant">R01 MH083094</span>, <span class="grant">R01 MH095034</span>, <span class="grant">U01 MH094421</span>; Wellcome Trust: <span class="grant">085475/B/08/Z</span>, <span class="grant">085475/Z/08/Z</span>, <span class="grant">090532</span>, <span class="grant">095552</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">10</span>;<span class="pagerange">1150-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23974872">23974872</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3827979">3827979</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2742">10.1038/ng.2742</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GATA1-mutant clones are frequent and often unsuspected in babies with Down syndrome: identification of a population at risk of leukemia.</h4>
<p class="authors"><span class="author">Roberts I</span>, <span class="author">Alford K</span>, <span class="author">Hall G</span>, <span class="author">Juban G</span>, <span class="author">Richmond H</span>, <span class="author">Norton A</span>, <span class="author">Vallance G</span>, <span class="author">Perkins K</span>, <span class="author">Marchi E</span>, <span class="author">McGowan S</span>, <span class="author">Roy A</span>, <span class="author">Cowan G</span>, <span class="author">Anthony M</span>, <span class="author">Gupta A</span>, <span class="author">Ho J</span>, <span class="author">Uthaya S</span>, <span class="author">Curley A</span>, <span class="author">Rasiah SV</span>, <span class="author">Watts T</span>, <span class="author">Nicholl R</span>, <span class="author">Bedford-Russell A</span>, <span class="author">Blumberg R</span>, <span class="author">Thomas A</span>, <span class="author">Gibson B</span>, <span class="author">Halsey C</span>, <span class="author">Lee PW</span>, <span class="author">Godambe S</span>, <span class="author">Sweeney C</span>, <span class="author">Bhatnagar N</span>, <span class="author">Goriely A</span>, <span class="author">Campbell P</span>, <span class="author">Vyas P</span> and <span class="author">Oxford-Imperial Down Syndrome Cohort Study Group</span></p>
<p class="affiliation">Centre for Haematology, Hammersmith Campus, Imperial College London, London, United Kingdom;</p>
<p class="abstract">Transient abnormal myelopoiesis (TAM), a preleukemic disorder unique to neonates with Down syndrome (DS), may transform to childhood acute myeloid leukemia (ML-DS). Acquired GATA1 mutations are present in both TAM and ML-DS. Current definitions of TAM specify neither the percentage of blasts nor the role of GATA1 mutation analysis. To define TAM, we prospectively analyzed clinical findings, blood counts and smears, and GATA1 mutation status in 200 DS neonates. All DS neonates had multiple blood count and smear abnormalities. Surprisingly, 195 of 200 (97.5%) had circulating blasts. GATA1 mutations were detected by Sanger sequencing/denaturing high performance liquid chromatography (Ss/DHPLC) in 17 of 200 (8.5%), all with blasts >10%. Furthermore low-abundance GATA1 mutant clones were detected by targeted next-generation resequencing (NGS) in 18 of 88 (20.4%; sensitivity ∼0.3%) DS neonates without Ss/DHPLC-detectable GATA1 mutations. No clinical or hematologic features distinguished these 18 neonates. We suggest the term "silent TAM" for neonates with DS with GATA1 mutations detectable only by NGS. To identify all babies at risk of ML-DS, we suggest GATA1 mutation and blood count and smear analyses should be performed in DS neonates. Ss/DPHLC can be used for initial screening, but where GATA1 mutations are undetectable by Ss/DHPLC, NGS-based methods can identify neonates with small GATA1 mutant clones.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">091182</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2013</span>;<span class="volume">122</span>;<span class="number">24</span>;<span class="pagerange">3908-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24021668">24021668</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2013-07-515148">10.1182/blood-2013-07-515148</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&mu;-Opioid receptor gene (OPRM1) polymorphism A118G: lack of association in Finnish populations with alcohol dependence or alcohol consumption.</h4>
<p class="authors"><span class="author">Rouvinen-Lagerstr&ouml;m N</span>, <span class="author">Lahti J</span>, <span class="author">Alho H</span>, <span class="author">Kovanen L</span>, <span class="author">Aalto M</span>, <span class="author">Partonen T</span>, <span class="author">Silander K</span>, <span class="author">Sinclair D</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Eriksson JG</span>, <span class="author">Palotie A</span>, <span class="author">Koskinen S</span> and <span class="author">Saarikoski ST</span></p>
<p class="affiliation">Ministry of Social Affairs and Health, Department of Occupational Safety and Health, PO Box 33, FI-00023 Government, Finland.</p><p class="abstract"><strong>Aims</strong>: The molecular epidemiological studies on the association of the opioid receptor &micro;-1 (OPRM1) polymorphism A118G (Asn40Asp, rs1799971) and alcohol use disorders have given conflicting results. The aim of this study was to test the possible association of A118G polymorphism and alcohol use disorders and alcohol consumption in three large cohort-based study samples.</p><p class="abstract"><strong>Methods</strong>: The association between the OPRM1 A118G (Asn40Asp, rs1799971) polymorphism and alcohol use disorders and alcohol consumption was analyzed using three different population-based samples: (a) a Finnish cohort study, Health 2000, with 503 participants having a DSM-IV diagnosis for alcohol dependence and/or alcohol abuse and 506 age- and sex-matched controls; (b) a Finnish cohort study, FINRISK (n = 2360) and (c) the Helsinki Birth Cohort Study (n = 1384). The latter two populations lacked diagnosis-based phenotypes, but included detailed information on alcohol consumption.</p><p class="abstract"><strong>Results</strong>: We found no statistically significant differences in genotypic or allelic distribution between controls and subjects with alcohol dependence or abuse diagnoses. Likewise no significant effects were observed between the A118G genotype and alcohol consumption.</p><p class="abstract"><strong>Conclusion</strong>: These results suggest that A118G (Asn40Asp) polymorphism may not have a major effect on the development of alcohol use disorders at least in the Finnish population.</p>
<p class="publication"><span class="title">Alcohol and alcoholism (Oxford, Oxfordshire)</span>&nbsp;<span class="year">2013</span>;<span class="volume">48</span>;<span class="number">5</span>;<span class="pagerange">519-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23729673">23729673</a>; DOI: <a href="http://dx.doi.org/10.1093%2Falcalc%2Fagt050">10.1093/alcalc/agt050</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of nine new susceptibility loci for testicular cancer, including variants near DAZL and PRDM14.</h4>
<p class="authors"><span class="author">Ruark E</span>, <span class="author">Seal S</span>, <span class="author">McDonald H</span>, <span class="author">Zhang F</span>, <span class="author">Elliot A</span>, <span class="author">Lau K</span>, <span class="author">Perdeaux E</span>, <span class="author">Rapley E</span>, <span class="author">Eeles R</span>, <span class="author">Peto J</span>, <span class="author">Kote-Jarai Z</span>, <span class="author">Muir K</span>, <span class="author">Nsengimana J</span>, <span class="author">Shipley J</span>, <span class="author">UK Testicular Cancer Collaboration (UKTCC)</span>, <span class="author">Bishop DT</span>, <span class="author">Stratton MR</span>, <span class="author">Easton DF</span>, <span class="author">Huddart RA</span>, <span class="author">Rahman N</span> and <span class="author">Turnbull C</span></p>
<p class="affiliation">Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, UK.</p><p class="abstract">Testicular germ cell tumor (TGCT) is the most common cancer in young men and is notable for its high familial risks. So far, six loci associated with TGCT have been reported. From genome-wide association study (GWAS) analysis of 307,291 SNPs in 986 TGCT cases and 4,946 controls, we selected for follow-up 694 SNPs, which we genotyped in a further 1,064 TGCT cases and 10,082 controls from the UK. We identified SNPs at nine new loci (1q22, 1q24.1, 3p24.3, 4q24, 5q31.1, 8q13.3, 16q12.1, 17q22 and 21q22.3) showing association with TGCT (P &lt; 5 &times; 10(-8)), which together account for an additional 4-6% of the familial risk of TGCT. The loci include genes plausibly related to TGCT development. PRDM14, at 8q13.3, is essential for early germ cell specification, and DAZL, at 3p24.3, is required for the regulation of germ cell development. Furthermore, PITX1, at 5q31.1, regulates TERT expression and is the third TGCT-associated locus implicated in telomerase regulation.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">686-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23666240">23666240</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3680037">3680037</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2635">10.1038/ng.2635</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mosaic PPM1D mutations are associated with predisposition to breast and ovarian cancer.</h4>
<p class="authors"><span class="author">Ruark E</span>, <span class="author">Snape K</span>, <span class="author">Humburg P</span>, <span class="author">Loveday C</span>, <span class="author">Bajrami I</span>, <span class="author">Brough R</span>, <span class="author">Rodrigues DN</span>, <span class="author">Renwick A</span>, <span class="author">Seal S</span>, <span class="author">Ramsay E</span>, <span class="author">Duarte Sdel V</span>, <span class="author">Rivas MA</span>, <span class="author">Warren-Perry M</span>, <span class="author">Zachariou A</span>, <span class="author">Campion-Flora A</span>, <span class="author">Hanks S</span>, <span class="author">Murray A</span>, <span class="author">Ansari Pour N</span>, <span class="author">Douglas J</span>, <span class="author">Gregory L</span>, <span class="author">Rimmer A</span>, <span class="author">Walker NM</span>, <span class="author">Yang TP</span>, <span class="author">Adlard JW</span>, <span class="author">Barwell J</span>, <span class="author">Berg J</span>, <span class="author">Brady AF</span>, <span class="author">Brewer C</span>, <span class="author">Brice G</span>, <span class="author">Chapman C</span>, <span class="author">Cook J</span>, <span class="author">Davidson R</span>, <span class="author">Donaldson A</span>, <span class="author">Douglas F</span>, <span class="author">Eccles D</span>, <span class="author">Evans DG</span>, <span class="author">Greenhalgh L</span>, <span class="author">Henderson A</span>, <span class="author">Izatt L</span>, <span class="author">Kumar A</span>, <span class="author">Lalloo F</span>, <span class="author">Miedzybrodzka Z</span>, <span class="author">Morrison PJ</span>, <span class="author">Paterson J</span>, <span class="author">Porteous M</span>, <span class="author">Rogers MT</span>, <span class="author">Shanley S</span>, <span class="author">Walker L</span>, <span class="author">Gore M</span>, <span class="author">Houlston R</span>, <span class="author">Brown MA</span>, <span class="author">Caufield MJ</span>, <span class="author">Deloukas P</span>, <span class="author">McCarthy MI</span>, <span class="author">Todd JA</span>, <span class="author">Breast and Ovarian Cancer Susceptibility Collaboration</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Turnbull C</span>, <span class="author">Reis-Filho JS</span>, <span class="author">Ashworth A</span>, <span class="author">Antoniou AC</span>, <span class="author">Lord CJ</span>, <span class="author">Donnelly P</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Division of Genetics &amp; Epidemiology, The Institute of Cancer Research, Sutton SM2 5NG, UK.</p><p class="abstract">Improved sequencing technologies offer unprecedented opportunities for investigating the role of rare genetic variation in common disease. However, there are considerable challenges with respect to study design, data analysis and replication. Using pooled next-generation sequencing of 507 genes implicated in the repair of DNA in 1,150 samples, an analytical strategy focused on protein-truncating variants (PTVs) and a large-scale sequencing case-control replication experiment in 13,642 individuals, here we show that rare PTVs in the p53-inducible protein phosphatase PPM1D are associated with predisposition to breast cancer and ovarian cancer. PPM1D PTV mutations were present in 25 out of 7,781 cases versus 1 out of 5,861 controls (P = 1.12&thinsp;&times;&thinsp;10(-5)), including 18 mutations in 6,912 individuals with breast cancer (P = 2.42&thinsp;&times;&thinsp;10(-4)) and 12 mutations in 1,121 individuals with ovarian cancer (P = 3.10&thinsp;&times;&thinsp;10(-9)). Notably, all of the identified PPM1D PTVs were mosaic in lymphocyte DNA and clustered within a 370-base-pair region in the final exon of the gene, carboxy-terminal to the phosphatase catalytic domain. Functional studies demonstrate that the mutations result in enhanced suppression of p53 in response to ionizing radiation exposure, suggesting that the mutant alleles encode hyperactive PPM1D isoforms. Thus, although the mutations cause premature protein truncation, they do not result in the simple loss-of-function effect typically associated with this class of variant, but instead probably have a gain-of-function effect. Our results have implications for the detection and management of breast and ovarian cancer risk. More generally, these data provide new insights into the role of rare and of mosaic genetic variants in common conditions, and the use of sequencing in their identification.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C12292/A11174</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0900747 91070</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">091157</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">493</span>;<span class="number">7432</span>;<span class="pagerange">406-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23242139">23242139</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11725">10.1038/nature11725</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of GluN2A/B cytoplasmic domains diversified vertebrate synaptic plasticity and behavior.</h4>
<p class="authors"><span class="author">Ryan TJ</span>, <span class="author">Kopanitsa MV</span>, <span class="author">Indersmitten T</span>, <span class="author">Nithianantharajah J</span>, <span class="author">Afinowi NO</span>, <span class="author">Pettit C</span>, <span class="author">Stanford LE</span>, <span class="author">Sprengel R</span>, <span class="author">Saksida LM</span>, <span class="author">Bussey TJ</span>, <span class="author">O&#39;Dell TJ</span>, <span class="author">Grant SG</span> and <span class="author">Komiyama NH</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">Two genome duplications early in the vertebrate lineage expanded gene families, including GluN2 subunits of the NMDA receptor. Diversification between the four mammalian GluN2 proteins occurred primarily at their intracellular C-terminal domains (CTDs). To identify shared ancestral functions and diversified subunit-specific functions, we exchanged the exons encoding the GluN2A (also known as Grin2a) and GluN2B (also known as Grin2b) CTDs in two knock-in mice and analyzed the mice&#39;s biochemistry, synaptic physiology, and multiple learned and innate behaviors. The eight behaviors were genetically separated into four groups, including one group comprising three types of learning linked to conserved GluN2A/B regions. In contrast, the remaining five behaviors exhibited subunit-specific regulation. GluN2A/B CTD diversification conferred differential binding to cytoplasmic MAGUK proteins and differential forms of long-term potentiation. These data indicate that vertebrate behavior and synaptic signaling acquired increased complexity from the duplication and diversification of ancestral GluN2 genes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0802238</span>; NIMH NIH HHS: <span class="grant">R01 MH060919</span>; Wellcome Trust: <span class="grant">089703</span></p>
<p class="publication"><span class="title">Nature neuroscience</span>&nbsp;<span class="year">2013</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">25-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23201971">23201971</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3979286">3979286</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnn.3277">10.1038/nn.3277</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular Characterization of Mutant Mouse Strains Generated from the EUCOMM/KOMP-CSD ES Cell Resource.</h4>
<p class="authors"><span class="author">Ryder E</span>, <span class="author">Gleeson D</span>, <span class="author">Sethi D</span>, <span class="author">Vyas S</span>, <span class="author">Miklejewska E</span>, <span class="author">Dalvi P</span>, <span class="author">Habib B</span>, <span class="author">Cook R</span>, <span class="author">Hardy M</span>, <span class="author">Jhaveri K</span>, <span class="author">Bottomley J</span>, <span class="author">Wardle-Jones H</span>, <span class="author">Bussell JN</span>, <span class="author">Houghton R</span>, <span class="author">Salisbury J</span>, <span class="author">Skarnes WC</span>, <span class="author">Sanger Mouse Genetics Project</span> and <span class="author">Ramirez-Solis R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, UK, <a href="mailto:%65%72%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#114;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">The Sanger Mouse Genetics Project generates knockout mice strains using the EUCOMM/KOMP-CSD embryonic stem (ES) cell collection and characterizes the consequences of the mutations using a high-throughput primary phenotyping screen. Upon achieving germline transmission, new strains are subject to a panel of quality control (QC) PCR- and qPCR-based assays to confirm the correct targeting, cassette structure, and the presence of the 3&#39; LoxP site (required for the potential conditionality of the allele). We report that over 86&nbsp;% of the 731 strains studied showed the correct targeting and cassette structure, of which 97&nbsp;% retained the 3&#39; LoxP site. We discuss the characteristics of the lines that failed QC and postulate that the majority of these may be due to mixed ES cell populations which were not detectable with the original screening techniques employed when creating the ES cell resource.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23912999">23912999</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-013-9467-x">10.1007/s00335-013-9467-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic analysis of a novel spontaneous albino C57BL/6N mouse strain.</h4>
<p class="authors"><span class="author">Ryder E</span>, <span class="author">Wong K</span>, <span class="author">Gleeson D</span>, <span class="author">Keane TM</span>, <span class="author">Sethi D</span>, <span class="author">Vyas S</span>, <span class="author">Wardle-Jones H</span>, <span class="author">Bussell JN</span>, <span class="author">Houghton R</span>, <span class="author">Salisbury J</span>, <span class="author">Harvey N</span>, <span class="author">Adams DJ</span>, <span class="author">The Sanger Mouse Genetics Project</span> and <span class="author">Ramirez-Solis R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, United Kingdom.</p>
<p class="abstract">We report an albino C57BL/6N mouse strain carrying a spontaneous mutation in the Tyrosinase gene (C57BL/6N-Tyr(cWTSI) ). Deep whole genome sequencing of founder mice revealed very little divergence from C57BL/6NJ and C57BL/6N (Taconic). This co-isogenic strain will be of great utility for the International Mouse Phenotyping Consortium (IMPC), which uses the EUCOMM/KOMP targeted C57BL/6N ES cell resource, and other investigators wishing to work on a defined C57BL/6N background. © 2013 Wiley Periodicals, Inc.</p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23620107">23620107</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvg.22398">10.1002/dvg.22398</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization and comparative analysis of the complete Haemonchus contortus &beta;-tubulin gene family and implications for benzimidazole resistance in strongylid nematodes.</h4>
<p class="authors"><span class="author">Saunders GI</span>, <span class="author">Wasmuth JD</span>, <span class="author">Beech R</span>, <span class="author">Laing R</span>, <span class="author">Hunt M</span>, <span class="author">Naghra H</span>, <span class="author">Cotton JA</span>, <span class="author">Berriman M</span>, <span class="author">Britton C</span> and <span class="author">Gilleard JS</span></p>
<p class="affiliation">Institute of Infection, Immunity and Inflammation, College of Medical, Veterinary and Life Sciences, University of Glasgow, 464 Bearsden Road, Glasgow, Scotland G61 1QH, UK.</p><p class="abstract">Parasitic nematode &beta;-tubulin genes are of particular interest because they are the targets of benzimidazole drugs. However, in spite of this, the full &beta;-tubulin gene family has not been characterized for any parasitic nematode to date. Haemonchus contortus is the parasite species for which we understand benzimidazole resistance the best and its close phylogenetic relationship with Caenorhabditis elegans potentially allows inferences of gene function by comparative analysis. Consequently, we have characterized the full &beta;-tubulin gene family in H. contortus. Further to the previously identified Hco-tbb-iso-1 and Hco-tbb-iso-2 genes, we have characterized two additional family members designated Hco-tbb-iso-3 and Hco-tbb-iso-4. We show that Hco-tbb-iso-1 is not a one-to-one orthologue with Cel-ben-1, the only &beta;-tubulin gene in C. elegans that is a benzimidazole drug target. Instead, both Hco-tbb-iso-1 and Hco-tbb-iso-2 have a complex evolutionary relationship with three C. elegans &beta;-tubulin genes: Cel-ben-1, Cel-tbb-1 and Cel-tbb-2. Furthermore, we show that both Hco-tbb-iso-1 and Hco-tbb-iso-2 are highly expressed in adult worms; in contrast, Hco-tbb-iso-3 and Hco-tbb-iso-4 are expressed only at very low levels and are orthologous to the Cel-mec-7 and Cel-tbb-4 genes, respectively, suggesting that they have specialized functional roles. Indeed, we have found that the expression pattern of Hco-tbb-iso-3 in H. contortus is identical to that of Cel-mec-7 in C. elegans, being expressed in just six &quot;touch receptor&quot; mechano-sensory neurons. These results suggest that further investigation is warranted into the potential involvement of strongylid isotype-2 &beta;-tubulin genes in mechanisms of benzimidazole resistance.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">230937</span>; Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2013</span>;<span class="volume">43</span>;<span class="number">6</span>;<span class="pagerange">465-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23416426">23416426</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijpara.2012.12.011">10.1016/j.ijpara.2012.12.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies a novel locus contributing to type 2 diabetes susceptibility in sikhs of punjabi origin from India.</h4>
<p class="authors"><span class="author">Saxena R</span>, <span class="author">Saleheen D</span>, <span class="author">Been LF</span>, <span class="author">Garavito ML</span>, <span class="author">Braun T</span>, <span class="author">Bjonnes A</span>, <span class="author">Young R</span>, <span class="author">Ho WK</span>, <span class="author">Rasheed A</span>, <span class="author">Frossard P</span>, <span class="author">Sim X</span>, <span class="author">Hassanali N</span>, <span class="author">Radha V</span>, <span class="author">Chidambaram M</span>, <span class="author">Liju S</span>, <span class="author">Rees SD</span>, <span class="author">Ng DP</span>, <span class="author">Wong TY</span>, <span class="author">Yamauchi T</span>, <span class="author">Hara K</span>, <span class="author">Tanaka Y</span>, <span class="author">Hirose H</span>, <span class="author">McCarthy MI</span>, <span class="author">Morris AP</span>, <span class="author">DIAGRAM</span>, <span class="author">MuTHER</span>, <span class="author">AGEN</span>, <span class="author">Basit A</span>, <span class="author">Barnett AH</span>, <span class="author">Katulanda P</span>, <span class="author">Matthews D</span>, <span class="author">Mohan V</span>, <span class="author">Wander GS</span>, <span class="author">Singh JR</span>, <span class="author">Mehra NK</span>, <span class="author">Ralhan S</span>, <span class="author">Kamboh MI</span>, <span class="author">Mulvihill JJ</span>, <span class="author">Maegawa H</span>, <span class="author">Tobe K</span>, <span class="author">Maeda S</span>, <span class="author">Cho YS</span>, <span class="author">Tai ES</span>, <span class="author">Kelly MA</span>, <span class="author">Chambers JC</span>, <span class="author">Kooner JS</span>, <span class="author">Kadowaki T</span>, <span class="author">Deloukas P</span>, <span class="author">Rader DJ</span>, <span class="author">Danesh J</span> and <span class="author">Sanghera DK</span></p>
<p class="affiliation">Center for Human Genetic Research and Department of Anesthesia, Critical Care and Pain Medicine, Massachusetts General Hospital, Harvard Medical School, Boston, Massachusetts.</p>
<p class="abstract">We performed a genome-wide association study (GWAS) and a multistage meta-analysis of type 2 diabetes (T2D) in Punjabi Sikhs from India. Our discovery GWAS in 1,616 individuals (842 case subjects) was followed by in silico replication of the top 513 independent single nucleotide polymorphisms (SNPs) (P < 10(-3)) in Punjabi Sikhs (n = 2,819; 801 case subjects). We further replicated 66 SNPs (P < 10(-4)) through genotyping in a Punjabi Sikh sample (n = 2,894; 1,711 case subjects). On combined meta-analysis in Sikh populations (n = 7,329; 3,354 case subjects), we identified a novel locus in association with T2D at 13q12 represented by a directly genotyped intronic SNP (rs9552911, P = 1.82 × 10(-8)) in the SGCG gene. Next, we undertook in silico replication (stage 2b) of the top 513 signals (P < 10(-3)) in 29,157 non-Sikh South Asians (10,971 case subjects) and de novo genotyping of up to 31 top signals (P < 10(-4)) in 10,817 South Asians (5,157 case subjects) (stage 3b). In combined South Asian meta-analysis, we observed six suggestive associations (P < 10(-5) to < 10(-7)), including SNPs at HMG1L1/CTCFL, PLXNA4, SCAP, and chr5p11. Further evaluation of 31 top SNPs in 33,707 East Asians (16,746 case subjects) (stage 3c) and 47,117 Europeans (8,130 case subjects) (stage 3d), and joint meta-analysis of 128,127 individuals (44,358 case subjects) from 27 multiethnic studies, did not reveal any additional loci nor was there any evidence of replication for the new variant. Our findings provide new evidence on the presence of a population-specific signal in relation to T2D, which may provide additional insights into T2D pathogenesis.</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2013</span>;<span class="volume">62</span>;<span class="number">5</span>;<span class="pagerange">1746-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23300278">23300278</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3636649">3636649</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb12-1077">10.2337/db12-1077</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide survey of genetic variation in gorillas using reduced representation sequencing.</h4>
<p class="authors"><span class="author">Scally A</span>, <span class="author">Yngvadottir B</span>, <span class="author">Xue Y</span>, <span class="author">Ayub Q</span>, <span class="author">Durbin R</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">All non-human great apes are endangered in the wild, and it is therefore important to gain an understanding of their demography and genetic diversity. Whole genome assembly projects have provided an invaluable foundation for understanding genetics in all four genera, but to date genetic studies of multiple individuals within great ape species have largely been confined to mitochondrial DNA and a small number of other loci. Here, we present a genome-wide survey of genetic variation in gorillas using a reduced representation sequencing approach, focusing on the two lowland subspecies. We identify 3,006,670 polymorphic sites in 14 individuals: 12 western lowland gorillas (Gorilla gorilla gorilla) and 2 eastern lowland gorillas (Gorilla beringei graueri). We find that the two species are genetically distinct, based on levels of heterozygosity and patterns of allele sharing. Focusing on the western lowland population, we observe evidence for population substructure, and a deficit of rare genetic variants suggesting a recent episode of population contraction. In western lowland gorillas, there is an elevation of variation towards telomeres and centromeres on the chromosomal scale. On a finer scale, we find substantial variation in genetic diversity, including a marked reduction close to the major histocompatibility locus, perhaps indicative of recent strong selection there. These findings suggest that despite their maintaining an overall level of genetic diversity equal to or greater than that of humans, population decline, perhaps associated with disease, has been a significant factor in recent and long-term pressures on wild gorilla populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">e65066</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23750230">23750230</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672199">3672199</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0065066">10.1371/journal.pone.0065066</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">On Your MARK, Get SET(D2), Go! H3K36me3 Primes DNA Mismatch Repair.</h4>
<p class="authors"><span class="author">Schmidt CK</span> and <span class="author">Jackson SP</span></p>
<p class="affiliation">The Gurdon Institute and the Department of Biochemistry, University of Cambridge, Cambridge CB2 1QN, UK.</p>
<p class="abstract">Trimethylation of histone H3 on Lys36 (H3K36me3) by SETD2 is linked to actively transcribed regions. Li et al. identify a novel role for H3K36me3 that facilitates DNA mismatch repair (MMR) in cells by targeting the MMR machinery to chromatin during the cell cycle, thereby explaining certain cases of MMR-defective cancers.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">153</span>;<span class="number">3</span>;<span class="pagerange">513-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23622237">23622237</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.04.018">10.1016/j.cell.2013.04.018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing identifies DYNC2H1 mutations as a common cause of asphyxiating thoracic dystrophy (Jeune syndrome) without major polydactyly, renal or retinal involvement.</h4>
<p class="authors"><span class="author">Schmidts M</span>, <span class="author">Arts HH</span>, <span class="author">Bongers EM</span>, <span class="author">Yap Z</span>, <span class="author">Oud MM</span>, <span class="author">Antony D</span>, <span class="author">Duijkers L</span>, <span class="author">Emes RD</span>, <span class="author">Stalker J</span>, <span class="author">Yntema JB</span>, <span class="author">Plagnol V</span>, <span class="author">Hoischen A</span>, <span class="author">Gilissen C</span>, <span class="author">Forsythe E</span>, <span class="author">Lausch E</span>, <span class="author">Veltman JA</span>, <span class="author">Roeleveld N</span>, <span class="author">Superti-Furga A</span>, <span class="author">Kutkowska-Kazmierczak A</span>, <span class="author">Kamsteeg EJ</span>, <span class="author">El&ccedil;ioglu N</span>, <span class="author">van Maarle MC</span>, <span class="author">Graul-Neumann LM</span>, <span class="author">Devriendt K</span>, <span class="author">Smithson SF</span>, <span class="author">Wellesley D</span>, <span class="author">Verbeek NE</span>, <span class="author">Hennekam RC</span>, <span class="author">Kayserili H</span>, <span class="author">Scambler PJ</span>, <span class="author">Beales PL</span>, <span class="author">UK10K</span>, <span class="author">Knoers NV</span>, <span class="author">Roepman R</span> and <span class="author">Mitchison HM</span></p>
<p class="affiliation">1Molecular Medicine Unit, Birth Defects Research Centre, University College London (UCL) Institute of Child Health, London, UK.</p>
<p class="abstract">BACKGROUND: Jeune asphyxiating thoracic dystrophy (JATD) is a rare, often lethal, recessively inherited chondrodysplasia characterised by shortened ribs and long bones, sometimes accompanied by polydactyly, and renal, liver and retinal disease. Mutations in intraflagellar transport (IFT) genes cause JATD, including the IFT dynein-2 motor subunit gene DYNC2H1. Genetic heterogeneity and the large DYNC2H1 gene size have hindered JATD genetic diagnosis. AIMS AND METHODS: To determine the contribution to JATD we screened DYNC2H1 in 71 JATD patients JATD patients combining SNP mapping, Sanger sequencing and exome sequencing. RESULTS AND CONCLUSIONS: We detected 34 DYNC2H1 mutations in 29/71 (41%) patients from 19/57 families (33%), showing it as a major cause of JATD especially in Northern European patients. This included 13 early protein termination mutations (nonsense/frameshift, deletion, splice site) but no patients carried these in combination, suggesting the human phenotype is at least partly hypomorphic. In addition, 21 missense mutations were distributed across DYNC2H1 and these showed some clustering to functional domains, especially the ATP motor domain. DYNC2H1 patients largely lacked significant extra-skeletal involvement, demonstrating an important genotype-phenotype correlation in JATD. Significant variability exists in the course and severity of the thoracic phenotype, both between affected siblings with identical DYNC2H1 alleles and among individuals with different alleles, which suggests the DYNC2H1 phenotype might be subject to modifier alleles, non-genetic or epigenetic factors. Assessment of fibroblasts from patients showed accumulation of anterograde IFT proteins in the ciliary tips, confirming defects similar to patients with other retrograde IFT machinery mutations, which may be of undervalued potential for diagnostic purposes.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23456818">23456818</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmedgenet-2012-101284">10.1136/jmedgenet-2012-101284</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Combined NGS approaches identify mutations in the intraflagellar transport gene IFT140 in skeletal ciliopathies with early progressive kidney Disease.</h4>
<p class="authors"><span class="author">Schmidts M</span>, <span class="author">Frank V</span>, <span class="author">Eisenberger T</span>, <span class="author">Al Turki S</span>, <span class="author">Bizet AA</span>, <span class="author">Antony D</span>, <span class="author">Rix S</span>, <span class="author">Decker C</span>, <span class="author">Bachmann N</span>, <span class="author">Bald M</span>, <span class="author">Vinke T</span>, <span class="author">Toenshoff B</span>, <span class="author">Di Donato N</span>, <span class="author">Neuhann T</span>, <span class="author">Hartley JL</span>, <span class="author">Maher ER</span>, <span class="author">Bogdanovi&#x107; R</span>, <span class="author">Peco-Anti&#x107; A</span>, <span class="author">Mache C</span>, <span class="author">Hurles ME</span>, <span class="author">Joksi&#x107; I</span>, <span class="author">Gu&#x107;-&Scaron;&#x107;eki&#x107; M</span>, <span class="author">Dobricic J</span>, <span class="author">Brankovic-Magic M</span>, <span class="author">Bolz HJ</span>, <span class="author">Pazour GJ</span>, <span class="author">Beales PL</span>, <span class="author">Scambler PJ</span>, <span class="author">Saunier S</span>, <span class="author">Mitchison HM</span> and <span class="author">Bergmann C</span></p>
<p class="affiliation">Molecular Medicine Unit, University College London Institute of Child Health, London, UK.</p><p class="abstract">Ciliopathies are genetically heterogeneous disorders characterized by variable expressivity and overlaps between different disease entities. This is exemplified by the short rib-polydactyly syndromes, Jeune, Sensenbrenner, and Mainzer-Saldino chondrodysplasia syndromes. These three syndromes are frequently caused by mutations in intraflagellar transport (IFT) genes affecting the primary cilia, which play a crucial role in skeletal and chondral development. Here, we identified mutations in IFT140, an IFT complex A gene, in five Jeune asphyxiating thoracic dystrophy (JATD) and two Mainzer-Saldino syndrome (MSS) families, by screening a cohort of 66 JATD/MSS patients using whole exome sequencing and targeted resequencing of a customized ciliopathy gene panel. We also found an enrichment of rare IFT140 alleles in JATD compared with nonciliopathy diseases, implying putative modifier effects for certain alleles. IFT140 patients presented with mild chest narrowing, but all had end-stage renal failure under 13 years of age and retinal dystrophy when examined for ocular dysfunction. This is consistent with the severe cystic phenotype of Ift140 conditional knockout mice, and the higher level of Ift140 expression in kidney and retina compared with the skeleton at E15.5 in the mouse. IFT140 is therefore a major cause of cono-renal syndromes (JATD and MSS). The present study strengthens the rationale for IFT140 screening in skeletal ciliopathy spectrum patients that have kidney disease and/or retinal dystrophy.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM060992</span>, <span class="grant">R01 GM060992</span>; Wellcome Trust: <span class="grant">091310</span>, <span class="grant">098051</span>, <span class="grant">UK10K</span>, <span class="grant">WT091310</span></p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2013</span>;<span class="volume">34</span>;<span class="number">5</span>;<span class="pagerange">714-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23418020">23418020</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4226634">4226634</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.22294">10.1002/humu.22294</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the Gene Encoding IFT Dynein Complex Component WDR34 Cause Jeune Asphyxiating Thoracic Dystrophy.</h4>
<p class="authors"><span class="author">Schmidts M</span>, <span class="author">Vodopiutz J</span>, <span class="author">Christou-Savina S</span>, <span class="author">Cort&eacute;s CR</span>, <span class="author">McInerney-Leo AM</span>, <span class="author">Emes RD</span>, <span class="author">Arts HH</span>, <span class="author">T&uuml;ys&uuml;z B</span>, <span class="author">D&#39;Silva J</span>, <span class="author">Leo PJ</span>, <span class="author">Giles TC</span>, <span class="author">Oud MM</span>, <span class="author">Harris JA</span>, <span class="author">Koopmans M</span>, <span class="author">Marshall M</span>, <span class="author">El&ccedil;ioglu N</span>, <span class="author">Kuechler A</span>, <span class="author">Bockenhauer D</span>, <span class="author">Moore AT</span>, <span class="author">Wilson LC</span>, <span class="author">Janecke AR</span>, <span class="author">Hurles ME</span>, <span class="author">Emmet W</span>, <span class="author">Gardiner B</span>, <span class="author">Streubel B</span>, <span class="author">Dopita B</span>, <span class="author">Zankl A</span>, <span class="author">Kayserili H</span>, <span class="author">Scambler PJ</span>, <span class="author">Brown MA</span>, <span class="author">Beales PL</span>, <span class="author">Wicking C</span>, <span class="author">UK10K</span>, <span class="author">Duncan EL</span> and <span class="author">Mitchison HM</span></p>
<p class="affiliation">Molecular Medicine Unit and Birth Defect Research Centre, Institute of Child Health, University College London (UCL), London WC1N 1EH, UK. Electronic address: <a href="mailto:%6D%69%72%69%61%6D.%73%63%68%6D%69%64%74%73.%31%30%40%75%63%6C.%61%63.%75%6B">&#109;&#105;&#114;&#105;&#97;&#109;&#46;&#115;&#99;&#104;&#109;&#105;&#100;&#116;&#115;&#46;&#49;&#48;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">Bidirectional (anterograde and retrograde) motor-based intraflagellar transport (IFT) governs cargo transport and delivery processes that are essential for primary cilia growth and maintenance and for hedgehog signaling functions. The IFT dynein-2 motor complex that regulates ciliary retrograde protein transport contains a heavy chain dynein ATPase/motor subunit, DYNC2H1, along with other less well functionally defined subunits. Deficiency of IFT proteins, including DYNC2H1, underlies a spectrum of skeletal ciliopathies. Here, by using exome sequencing and a targeted next-generation sequencing panel, we identified a total of 11 mutations in WDR34 in 9 families with the clinical diagnosis of Jeune syndrome (asphyxiating thoracic dystrophy). WDR34 encodes a WD40 repeat-containing protein orthologous to Chlamydomonas FAP133, a dynein intermediate chain associated with the retrograde intraflagellar transport motor. Three-dimensional protein modeling suggests that the identified mutations all affect residues critical for WDR34 protein-protein interactions. We find that WDR34 concentrates around the centrioles and basal bodies in mammalian cells, also showing axonemal staining. WDR34 coimmunoprecipitates with the dynein-1 light chain DYNLL1 in&nbsp;vitro, and mining of proteomics data suggests that WDR34 could represent a previously unrecognized link between the cytoplasmic dynein-1 and IFT dynein-2 motors. Together, these data show that WDR34 is critical for ciliary functions essential to normal development and survival, most probably as a previously unrecognized component of the mammalian dynein-IFT machinery.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">93</span>;<span class="number">5</span>;<span class="pagerange">932-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24183451">24183451</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3824113">3824113</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.10.003">10.1016/j.ajhg.2013.10.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Co-binding by YY1 identifies the transcriptionally active, highly conserved set of CTCF-bound regions in primate genomes.</h4>
<p class="authors"><span class="author">Schwalie PC</span>, <span class="author">Ward MC</span>, <span class="author">Cain CE</span>, <span class="author">Faure AJ</span>, <span class="author">Gilad Y</span>, <span class="author">Odom DT</span> and <span class="author">Flicek P</span></p>
<p class="affiliation">European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%64%75%6E%63%61%6E.%6F%64%6F%6D%40%63%72%75%6B.%63%61%6D.%61%63.%75%6B">&#100;&#117;&#110;&#99;&#97;&#110;&#46;&#111;&#100;&#111;&#109;&#64;&#99;&#114;&#117;&#107;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract"><strong>Background</strong>: The genomic binding of CTCF is highly conserved across mammals, but the mechanisms that underlie its stability are poorly understood. One transcription factor known to functionally interact with CTCF in the context of X-chromosome inactivation is the ubiquitously expressed YY1. Because combinatorial transcription factor binding can contribute to the evolutionary stabilization of regulatory regions, we tested whether YY1 and CTCF co-binding could in part account for conservation of CTCF binding.</p><p class="abstract"><strong>Results</strong>: Combined analysis of CTCF and YY1 binding in lymphoblastoid cell lines from seven primates, as well as in mouse and human livers, reveals extensive genome-wide co-localization specifically at evolutionarily stable CTCF-bound regions. CTCF-YY1 co-bound regions resemble regions bound by YY1 alone, as they enrich for active histone marks, RNA polymerase II and transcription factor binding. Although these highly conserved, transcriptionally active CTCF-YY1 co-bound regions are often promoter-proximal, gene-distal regions show similar molecular features.</p><p class="abstract"><strong>Conclusions</strong>: Our results reveal that these two ubiquitously expressed, multi-functional zinc-finger proteins collaborate in functionally active regions to stabilize one another&#39;s genome-wide binding across primate evolution.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM077959</span>; NIH HHS: <span class="grant">P51 OD011132</span>; Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">12</span>;<span class="pagerange">R148</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24380390">24380390</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-12-r148">10.1186/gb-2013-14-12-r148</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mechanisms controlling the temporal degradation of Nek2A and Kif18A by the APC/C-Cdc20 complex.</h4>
<p class="authors"><span class="author">Sedgwick GG</span>, <span class="author">Hayward DG</span>, <span class="author">Di Fiore B</span>, <span class="author">Pardo M</span>, <span class="author">Yu L</span>, <span class="author">Pines J</span> and <span class="author">Nilsson J</span></p>
<p class="affiliation">The Novo Nordisk Foundation Center for Protein Research, University of Copenhagen, Copenhagen, Denmark.</p><p class="abstract">The Anaphase Promoting Complex/Cyclosome (APC/C) in complex with its co-activator Cdc20 is responsible for targeting proteins for ubiquitin-mediated degradation during mitosis. The activity of APC/C-Cdc20 is inhibited during prometaphase by the Spindle Assembly Checkpoint (SAC) yet certain substrates escape this inhibition. Nek2A degradation during prometaphase depends on direct binding of Nek2A to the APC/C via a C-terminal MR dipeptide but whether this motif alone is sufficient is not clear. Here, we identify Kif18A as a novel APC/C-Cdc20 substrate and show that Kif18A degradation depends on a C-terminal LR motif. However in contrast to Nek2A, Kif18A is not degraded until anaphase showing that additional mechanisms contribute to Nek2A degradation. We find that dimerization via the leucine zipper, in combination with the MR motif, is required for stable Nek2A binding to and ubiquitination by the APC/C. Nek2A and the mitotic checkpoint complex (MCC) have an overlap in APC/C subunit requirements for binding and we propose that Nek2A binds with high affinity to apo-APC/C and is degraded by the pool of Cdc20 that avoids inhibition by the SAC.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13678</span>; Wellcome Trust: <span class="grant">079643/Z/06/Z</span>, <span class="grant">092096</span></p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2013</span>;<span class="volume">32</span>;<span class="number">2</span>;<span class="pagerange">303-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23288039">23288039</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3553385">3553385</a>; DOI: <a href="http://dx.doi.org/10.1038%2Femboj.2012.335">10.1038/emboj.2012.335</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conceptual links between DNA methylation reprogramming in the early embryo and primordial germ cells.</h4>
<p class="authors"><span class="author">Seisenberger S</span>, <span class="author">Peat JR</span> and <span class="author">Reik W</span></p>
<p class="affiliation">Epigenetics Programme, The Babraham Institute, Cambridge CB22 3AT, UK. Electronic address: <a href="mailto:%73%74%65%66%61%6E%69%65.%73%65%69%73%65%6E%62%65%72%67%65%72%40%62%61%62%72%61%68%61%6D.%61%63.%75%6B">&#115;&#116;&#101;&#102;&#97;&#110;&#105;&#101;&#46;&#115;&#101;&#105;&#115;&#101;&#110;&#98;&#101;&#114;&#103;&#101;&#114;&#64;&#98;&#97;&#98;&#114;&#97;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">DNA methylation is a carrier of important regulatory information that undergoes global reprogramming in the mammalian germ line, including pre-implantation embryos and primordial germ cells (PGCs). A flurry of recent studies have employed technical advances to generate global profiles of methylation and hydroxymethylation in these cells, unravelling the dynamics of methylation erasure at single locus resolution. Active demethylation in the zygote, involving extensive oxidation, is followed by passive loss over early cell divisions. Certain gamete-contributed methylation marks appear to have evolved non-canonical mechanisms for targeted maintenance of methylation in the face of these processes. These protected sequences include the imprinting control regions (ICRs) required for parental imprinting but also a surprising number of other regions. Such targeted maintenance mechanisms may also operate at certain sequences during early PGC migration when global passive demethylation occurs. In later gonadal PGCs, imprints must be reset and this may be achieved through the targeting of active mechanisms including oxidation. Thus, emerging evidence paints a complex picture whereby active and passive demethylation pathways operate synergistically and in parallel to ensure robust erasure in the early embryo and PGCs.</p>
<p class="publication"><span class="title">Current opinion in cell biology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23510682">23510682</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ceb.2013.02.013">10.1016/j.ceb.2013.02.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reprogramming DNA methylation in the mammalian life cycle: building and breaking epigenetic barriers.</h4>
<p class="authors"><span class="author">Seisenberger S</span>, <span class="author">Peat JR</span>, <span class="author">Hore TA</span>, <span class="author">Santos F</span>, <span class="author">Dean W</span> and <span class="author">Reik W</span></p>
<p class="affiliation">Epigenetics Programme, The Babraham Institute, , Cambridge CB22 3AT, UK.</p>
<p class="abstract">In mammalian development, epigenetic modifications, including DNA methylation patterns, play a crucial role in defining cell fate but also represent epigenetic barriers that restrict developmental potential. At two points in the life cycle, DNA methylation marks are reprogrammed on a global scale, concomitant with restoration of developmental potency. DNA methylation patterns are subsequently re-established with the commitment towards a distinct cell fate. This reprogramming of DNA methylation takes place firstly on fertilization in the zygote, and secondly in primordial germ cells (PGCs), which are the direct progenitors of sperm or oocyte. In each reprogramming window, a unique set of mechanisms regulates DNA methylation erasure and re-establishment. Recent advances have uncovered roles for the TET3 hydroxylase and passive demethylation, together with base excision repair (BER) and the elongator complex, in methylation erasure from the zygote. Deamination by AID, BER and passive demethylation have been implicated in reprogramming in PGCs, but the process in its entirety is still poorly understood. In this review, we discuss the dynamics of DNA methylation reprogramming in PGCs and the zygote, the mechanisms involved and the biological significance of these events. Advances in our understanding of such natural epigenetic reprogramming are beginning to aid enhancement of experimental reprogramming in which the role of potential mechanisms can be investigated in vitro. Conversely, insights into in vitro reprogramming techniques may aid our understanding of epigenetic reprogramming in the germline and supply important clues in reprogramming for therapies in regenerative medicine.</p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2013</span>;<span class="volume">368</span>;<span class="number">1609</span>;<span class="pagerange">20110330</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23166394" rel="external">23166394</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2011.0330" rel="external">10.1098/rstb.2011.0330</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cohesin-based chromatin interactions enable regulated gene expression within preexisting architectural compartments.</h4>
<p class="authors"><span class="author">Seitan VC</span>, <span class="author">Faure AJ</span>, <span class="author">Zhan Y</span>, <span class="author">McCord RP</span>, <span class="author">Lajoie BR</span>, <span class="author">Ing-Simmons E</span>, <span class="author">Lenhard B</span>, <span class="author">Giorgetti L</span>, <span class="author">Heard E</span>, <span class="author">Fisher AG</span>, <span class="author">Flicek P</span>, <span class="author">Dekker J</span> and <span class="author">Merkenschlager M</span></p>
<p class="affiliation">Lymphocyte Development Group, MRC Clinical Sciences Centre, Imperial College London, London W12 0NN, United Kingdom;</p>
<p class="abstract">Chromosome conformation capture approaches have shown that interphase chromatin is partitioned into spatially segregated Mb-sized compartments and sub-Mb-sized topological domains. This compartmentalization is thought to facilitate the matching of genes and regulatory elements, but its precise function and mechanistic basis remain unknown. Cohesin controls chromosome topology to enable DNA repair and chromosome segregation in cycling cells. In addition, cohesin associates with active enhancers and promoters and with CTCF to form long-range interactions important for gene regulation. Although these findings suggest an important role for cohesin in genome organization, this role has not been assessed on a global scale. Unexpectedly, we find that architectural compartments are maintained in noncycling mouse thymocytes after genetic depletion of cohesin in vivo. Cohesin was, however, required for specific long-range interactions within compartments where cohesin-regulated genes reside. Cohesin depletion diminished interactions between cohesin-bound sites, whereas alternative interactions between chromatin features associated with transcriptional activation and repression became more prominent, with corresponding changes in gene expression. Our findings indicate that cohesin-mediated long-range interactions facilitate discrete gene expression states within preexisting chromosomal compartments.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">12</span>;<span class="pagerange">2066-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24002784">24002784</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3847776">3847776</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.161620.113">10.1101/gr.161620.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Playing the 'next-generation game'.</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Advances in single-molecule DNA sequencing are enabling research into the fine resolution of DNA structure, and rapid, direct sequencing of pathogen genomes.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">2</span>;<span class="pagerange">74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23321531" rel="external">23321531</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2956" rel="external">10.1038/nrmicro2956</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generating whole bacterial genome sequences of low-abundance species from complex samples with IMS-MDA.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span>, <span class="author">Harris SR</span>, <span class="author">Scott P</span>, <span class="author">Parmar S</span>, <span class="author">Marsh P</span>, <span class="author">Unemo M</span>, <span class="author">Clarke IN</span>, <span class="author">Parkhill J</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">1] Pathogen Genomics, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridgeshire, UK. [2].</p>
<p class="abstract">The study of bacterial populations using whole-genome sequencing is of considerable scientific and clinical interest. However, obtaining bacterial genomic information is not always trivial: the target bacteria may be difficult to culture or uncultured, and they may be found within samples containing complex mixtures of other contaminating microbes and/or host cells, from which it is very difficult to derive robust sequencing data. Here we describe our procedure to generate sufficient DNA for whole-genome sequencing from clinical samples and without the need for culture, as successfully used on the difficult-to-culture, obligate intracellular pathogen Chlamydia trachomatis. Our protocol combines immunomagnetic separation (IMS) for targeted bacterial enrichment with multiple displacement amplification (MDA) for whole-genome amplification (WGA), which is followed by high-throughput sequencing. Compared with other techniques that might be used to generate such data, IMS-MDA is an inexpensive, low-technology and highly transferable process that provides amplified genomic DNA for sequencing from target bacteria in under 5 h, with little hands-on time.</p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">2404-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24202554">24202554</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2013.147">10.1038/nprot.2013.147</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequences of Chlamydia trachomatis directly from clinical samples without culture.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span>, <span class="author">Harris SR</span>, <span class="author">Skilton RJ</span>, <span class="author">Radebe FM</span>, <span class="author">Golparian D</span>, <span class="author">Shipitsyna E</span>, <span class="author">Duy PT</span>, <span class="author">Scott P</span>, <span class="author">Cutcliffe LT</span>, <span class="author">O&#39;Neill C</span>, <span class="author">Parmar S</span>, <span class="author">Pitt R</span>, <span class="author">Baker S</span>, <span class="author">Ison CA</span>, <span class="author">Marsh P</span>, <span class="author">Jalal H</span>, <span class="author">Lewis DA</span>, <span class="author">Unemo M</span>, <span class="author">Clarke IN</span>, <span class="author">Parkhill J</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Pathogen Genomics, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom. <a href="mailto:%68%73%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#115;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The use of whole-genome sequencing as a tool for the study of infectious bacteria is of growing clinical interest. Chlamydia trachomatis is responsible for sexually transmitted infections and the blinding disease trachoma, which affect hundreds of millions of people worldwide. Recombination is widespread within the genome of C. trachomatis, thus whole-genome sequencing is necessary to understand the evolution, diversity, and epidemiology of this pathogen. Culture of C. trachomatis has, until now, been a prerequisite to obtain DNA for whole-genome sequencing; however, as C. trachomatis is an obligate intracellular pathogen, this procedure is technically demanding and time consuming. Discarded clinical samples represent a large resource for sequencing the genomes of pathogens, yet clinical swabs frequently contain very low levels of C. trachomatis DNA and large amounts of contaminating microbial and human DNA. To determine whether it is possible to obtain whole-genome sequences from bacteria without the need for culture, we have devised an approach that combines immunomagnetic separation (IMS) for targeted bacterial enrichment with multiple displacement amplification (MDA) for whole-genome amplification. Using IMS-MDA in conjunction with high-throughput multiplexed Illumina sequencing, we have produced the first whole bacterial genome sequences direct from clinical samples. We also show that this method can be used to generate genome data from nonviable archived samples. This method will prove a useful tool in answering questions relating to the biology of many difficult-to-culture or fastidious bacteria of clinical concern.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089276</span>, <span class="grant">098051</span>, <span class="grant">100087</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">5</span>;<span class="pagerange">855-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23525359">23525359</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3638141">3638141</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.150037.112">10.1101/gr.150037.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome Sequence of Chlamydia psittaci Strain 01DC12 Originating from Swine.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span>, <span class="author">Sait M</span>, <span class="author">Sachse K</span>, <span class="author">Gaede W</span>, <span class="author">Longbottom D</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Pathogen Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p>
<p class="abstract">Chlamydia psittaci is the etiological agent of psittacosis and is a zoonotic pathogen infecting birds and a variety of mammalian hosts. Here we report the genome sequence of the porcine strain 01DC12 which is representative of a novel clade of C. psittaci belonging to ompA genotype E.</p>
<p class="publication"><span class="title">Genome announcements</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23405306">23405306</a>; DOI: <a href="http://dx.doi.org/10.1128%2FgenomeA.00078-12">10.1128/genomeA.00078-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution. Great apes and zoonoses.</h4>
<p class="authors"><span class="author">Sharp PM</span>, <span class="author">Rayner JC</span> and <span class="author">Hahn BH</span></p>
<p class="affiliation">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3JT, UK.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">340</span>;<span class="number">6130</span>;<span class="pagerange">284-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23599472">23599472</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1236958">10.1126/science.1236958</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficient Knockin Mouse Generation by ssDNA Oligonucleotides and Zinc-Finger Nuclease Assisted Homologous Recombination in Zygotes.</h4>
<p class="authors"><span class="author">Shen B</span>, <span class="author">Zhang X</span>, <span class="author">Du Y</span>, <span class="author">Wang J</span>, <span class="author">Gong J</span>, <span class="author">Zhang X</span>, <span class="author">Tate PH</span>, <span class="author">Li H</span>, <span class="author">Huang X</span> and <span class="author">Zhang W</span></p>
<p class="affiliation">MOE Key Laboratory of Model Animal for Disease Study, Model Animal Research Center of Nanjing University, National Resource Center for Mutant Mice, Nanjing, China.</p>
<p class="abstract">The generation of specific mutant animal models is critical for functional analysis of human genes. The conventional gene targeting approach in embryonic stem cells (ESCs) by homologous recombination is however laborious, slow, expensive, and limited to species with functional ESCs. It is therefore a long-sought goal to develop an efficient and simple alternative gene targeting strategy. Here we demonstrate that, by combining an efficient ZFN pair and ssODN, a restriction site and a loxP site were successfully introduced into a specific genomic locus. A targeting efficiency up to 22.22% was achieved by coinciding the insertion site and the ZFN cleavage site isogenic and keeping the length of the homology arms equal and isogenic to the endogenous target locus. Furthermore, we determine that ZFN and ssODN-assisted HR is ssODN homology arm length dependent. We further show that mutant alleles generated by ZFN and ssODN-assisted HR can be transmitted through the germline successfully. This study establishes an efficient gene targeting strategy by ZFN and ssODN-assisted HR in mouse zygotes, and provides a potential avenue for genome engineering in animal species without functional ES cell lines.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e77696</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24167580">24167580</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3805579">3805579</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0077696">10.1371/journal.pone.0077696</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Historical Zoonoses and Other Changes in Host Tropism of Staphylococcus aureus, Identified by Phylogenetic Analysis of a Population Dataset.</h4>
<p class="authors"><span class="author">Shepheard MA</span>, <span class="author">Fleming VM</span>, <span class="author">Connor TR</span>, <span class="author">Corander J</span>, <span class="author">Feil EJ</span>, <span class="author">Fraser C</span> and <span class="author">Hanage WP</span></p>
<p class="affiliation">MRC Centre for Outbreak Analysis and Modelling, Department of Infectious Disease Epidemiology, School of Public Health, Imperial College London, London, United Kingdom.</p>
<p class="abstract">BACKGROUND: Staphylococcus aureus exhibits tropisms to many distinct animal hosts. While spillover events can occur wherever there is an interface between host species, changes in host tropism only occur with the establishment of sustained transmission in the new host species, leading to clonal expansion. Although the genomic variation underpinning adaptation in S. aureus genotypes infecting bovids and poultry has been well characterized the frequency of switches from one host to another remains obscure. We sought to identify sustained switches in host tropism in the S. aureus population, both anthroponotic and zoonotic, and their distribution over the species phylogeny. METHODOLOGIESRESULTS: We have used a sample of 3042 isolates, representing 696 distinct MLST genotypes, from a well-established database (www.mlst.net). Using an empirical parsimony approach (AdaptML) we have investigated the distribution of switches in host association between both human and non-human (henceforth referred to as animal) hosts. We reconstructed a credible description of past events in the form of a phylogenetic tree; the nodes and leaves of which are statistically associated with either human or animal habitats, estimated from extant host-association and the degree of sequence divergence between genotypes. We identified 15 likely historical switching events; 13 anthroponoses and two zoonoses. Importantly, we identified two human-associated clade candidates (CC25 and CC59) that have arisen from animal-associated ancestors; this demonstrates that a human-specific lineage can emerge from an animal host. We also highlight novel rabbit-associated genotypes arising from a human ancestor. CONCLUSIONS: S. aureus is an organism with the capacity to switch into and adapt to novel hosts, even after long periods of isolation in a single host species. Based on this evidence, animal-adapted S. aureus lineages exhibiting resistance to antibiotics must be considered a major threat to public health, as they can adapt to the human population.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e62369</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23667472">23667472</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0062369">10.1371/journal.pone.0062369</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Progressive genome-wide introgression in agricultural Campylobacter coli.</h4>
<p class="authors"><span class="author">Sheppard SK</span>, <span class="author">Didelot X</span>, <span class="author">Jolley KA</span>, <span class="author">Darling AE</span>, <span class="author">Pascoe B</span>, <span class="author">Meric G</span>, <span class="author">Kelly DJ</span>, <span class="author">Cody A</span>, <span class="author">Colles FM</span>, <span class="author">Strachan NJ</span>, <span class="author">Ogden ID</span>, <span class="author">Forbes K</span>, <span class="author">French NP</span>, <span class="author">Carter P</span>, <span class="author">Miller WG</span>, <span class="author">McCarthy ND</span>, <span class="author">Owen R</span>, <span class="author">Litrup E</span>, <span class="author">Egholm M</span>, <span class="author">Affourtit JP</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Maiden MC</span> and <span class="author">Falush D</span></p>
<p class="affiliation">Department of Zoology, The Tinbergen Building, University of Oxford, South Parks Road, Oxford, OX1 3PS, UK. <a href="mailto:%73.%6B.%73%68%65%70%70%61%72%64%40%73%77%61%6E%73%65%61.%61%63.%75%6B">&#115;&#46;&#107;&#46;&#115;&#104;&#101;&#112;&#112;&#97;&#114;&#100;&#64;&#115;&#119;&#97;&#110;&#115;&#101;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Hybridization between distantly related organisms can facilitate rapid adaptation to novel environments, but is potentially constrained by epistatic fitness interactions among cell components. The zoonotic pathogens Campylobacter coli and C. jejuni differ from each other by around 15% at the nucleotide level, corresponding to an average of nearly 40 amino acids per protein-coding gene. Using whole genome sequencing, we show that a single C. coli lineage, which has successfully colonized an agricultural niche, has been progressively accumulating C. jejuni DNA. Members of this lineage belong to two groups, the ST-828 and ST-1150 clonal complexes. The ST-1150 complex is less frequently isolated and has undergone a substantially greater amount of introgression leading to replacement of up to 23% of the C. coli core genome as well as import of novel DNA. By contrast, the more commonly isolated ST-828 complex bacteria have 10-11% introgressed DNA, and C. jejuni and nonagricultural C. coli lineages each have &lt;2%. Thus, the C. coli that colonize agriculture, and consequently cause most human disease, have hybrid origin, but this cross-species exchange has so far not had a substantial impact on the gene pools of either C. jejuni or nonagricultural C. coli. These findings also indicate remarkable interchangeability of basic cellular machinery after a prolonged period of independent evolution.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Molecular ecology</span>&nbsp;<span class="year">2013</span>;<span class="volume">22</span>;<span class="number">4</span>;<span class="pagerange">1051-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23279096">23279096</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fmec.12162">10.1111/mec.12162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies vitamin B5 biosynthesis as a host specificity factor in Campylobacter.</h4>
<p class="authors"><span class="author">Sheppard SK</span>, <span class="author">Didelot X</span>, <span class="author">Meric G</span>, <span class="author">Torralbo A</span>, <span class="author">Jolley KA</span>, <span class="author">Kelly DJ</span>, <span class="author">Bentley SD</span>, <span class="author">Maiden MC</span>, <span class="author">Parkhill J</span> and <span class="author">Falush D</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Oxford OX1 3PS, United Kingdom.</p><p class="abstract">Genome-wide association studies have the potential to identify causal genetic factors underlying important phenotypes but have rarely been performed in bacteria. We present an association mapping method that takes into account the clonal population structure of bacteria and is applicable to both core and accessory genome variation. Campylobacter is a common cause of human gastroenteritis as a consequence of its proliferation in multiple farm animal species and its transmission via contaminated meat and poultry. We applied our association mapping method to identify the factors responsible for adaptation to cattle and chickens among 192 Campylobacter isolates from these and other host sources. Phylogenetic analysis implied frequent host switching but also showed that some lineages were strongly associated with particular hosts. A seven-gene region with a host association signal was found. Genes in this region were almost universally present in cattle but were frequently absent in isolates from chickens and wild birds. Three of the seven genes encoded vitamin B5 biosynthesis. We found that isolates from cattle were better able to grow in vitamin B5-depleted media and propose that this difference may be an adaptation to host diet.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">29</span>;<span class="pagerange">11923-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23818615">23818615</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3718156">3718156</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1305559110">10.1073/pnas.1305559110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comparative phenotypic and genomic analysis of C57BL/6J and C57BL/6N mouse strains.</h4>
<p class="authors"><span class="author">Simon MM</span>, <span class="author">Greenaway S</span>, <span class="author">White JK</span>, <span class="author">Fuchs H</span>, <span class="author">Gailus-Durner V</span>, <span class="author">Wells S</span>, <span class="author">Sorg T</span>, <span class="author">Wong K</span>, <span class="author">Bedu E</span>, <span class="author">Cartwright EJ</span>, <span class="author">Dacquin R</span>, <span class="author">Djebali S</span>, <span class="author">Estabel J</span>, <span class="author">Graw J</span>, <span class="author">Ingham NJ</span>, <span class="author">Jackson IJ</span>, <span class="author">Lengeling A</span>, <span class="author">Mandillo S</span>, <span class="author">Marvel J</span>, <span class="author">Meziane H</span>, <span class="author">Preitner F</span>, <span class="author">Puk O</span>, <span class="author">Roux M</span>, <span class="author">Adams DJ</span>, <span class="author">Atkins S</span>, <span class="author">Ayadi A</span>, <span class="author">Becker L</span>, <span class="author">Blake A</span>, <span class="author">Brooker D</span>, <span class="author">Cater H</span>, <span class="author">Champy MF</span>, <span class="author">Combe R</span>, <span class="author">Danecek P</span>, <span class="author">di Fenza A</span>, <span class="author">Gates H</span>, <span class="author">Gerdin AK</span>, <span class="author">Golini E</span>, <span class="author">Hancock JM</span>, <span class="author">Hans W</span>, <span class="author">H&ouml;lter SM</span>, <span class="author">Hough T</span>, <span class="author">Jurdic P</span>, <span class="author">Keane TM</span>, <span class="author">Morgan H</span>, <span class="author">M&uuml;ller W</span>, <span class="author">Neff F</span>, <span class="author">Nicholson G</span>, <span class="author">Pasche B</span>, <span class="author">Roberson LA</span>, <span class="author">Rozman J</span>, <span class="author">Sanderson M</span>, <span class="author">Santos L</span>, <span class="author">Selloum M</span>, <span class="author">Shannon C</span>, <span class="author">Southwell A</span>, <span class="author">Tocchini-Valentini GP</span>, <span class="author">Vancollie VE</span>, <span class="author">Westerberg H</span>, <span class="author">Wurst W</span>, <span class="author">Zi M</span>, <span class="author">Yalcin B</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Steel KP</span>, <span class="author">Mallon AM</span>, <span class="author">de Angelis MH</span>, <span class="author">Herault Y</span> and <span class="author">Brown SD</span></p>
<p class="affiliation">Medical Research Council Harwell (Mammalian Genetics Unit and Mary Lyon Centre), Harwell Science Campus, OX11 0RD, UK. <a href="mailto:%73.%62%72%6F%77%6E%40%68%61%72.%6D%72%63.%61%63.%75%6B">&#115;&#46;&#98;&#114;&#111;&#119;&#110;&#64;&#104;&#97;&#114;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: The mouse inbred line C57BL/6J is widely used in mouse genetics and its genome has been incorporated into many genetic reference populations. More recently large initiatives such as the International Knockout Mouse Consortium (IKMC) are using the C57BL/6N mouse strain to generate null alleles for all mouse genes. Hence both strains are now widely used in mouse genetics studies. Here we perform a comprehensive genomic and phenotypic analysis of the two strains to identify differences that may influence their underlying genetic mechanisms.

Results: We undertake genome sequence comparisons of C57BL/6J and C57BL/6N to identify SNPs, indels and structural variants, with a focus on identifying all coding variants. We annotate 34 SNPs and 2 indels that distinguish C57BL/6J and C57BL/6N coding sequences, as well as 15 structural variants that overlap a gene. In parallel we assess the comparative phenotypes of the two inbred lines utilizing the EMPReSSslim phenotyping pipeline, a broad based assessment encompassing diverse biological systems. We perform additional secondary phenotyping assessments to explore other phenotype domains and to elaborate phenotype differences identified in the primary assessment. We uncover significant phenotypic differences between the two lines, replicated across multiple centers, in a number of physiological, biochemical and behavioral systems.

Conclusions: Comparison of C57BL/6J and C57BL/6N demonstrates a range of phenotypic differences that have the potential to impact upon penetrance and expressivity of mutational effects in these strains. Moreover, the sequence variants we identify provide a set of candidate genes for the phenotypic differences observed between the two strains.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">7</span>;<span class="pagerange">R82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23902802">23902802</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-7-r82">10.1186/gb-2013-14-7-r82</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Stress-induced lipocalin-2 controls dendritic spine formation and neuronal activity in the amygdala.</h4>
<p class="authors"><span class="author">Skrzypiec AE</span>, <span class="author">Shah RS</span>, <span class="author">Schiavon E</span>, <span class="author">Baker E</span>, <span class="author">Skene N</span>, <span class="author">Pawlak R</span> and <span class="author">Mucha M</span></p>
<p class="affiliation">University of Exeter Medical School, Exeter, United Kingdom.</p>
<p class="abstract">Behavioural adaptation to psychological stress is dependent on neuronal plasticity and dysfunction at this cellular level may underlie the pathogenesis of affective disorders such as depression and post-traumatic stress disorder. Taking advantage of genome-wide microarray assay, we performed detailed studies of stress-affected transcripts in the amygdala - an area which forms part of the innate fear circuit in mammals. Having previously demonstrated the role of lipocalin-2 (Lcn-2) in promoting stress-induced changes in dendritic spine morphology/function and neuronal excitability in the mouse hippocampus, we show here that the Lcn-2 gene is one of the most highly upregulated transcripts detected by microarray analysis in the amygdala after acute restraint-induced psychological stress. This is associated with increased Lcn-2 protein synthesis, which is found on immunohistochemistry to be predominantly localised to neurons. Stress-naïve Lcn-2(-/-) mice show a higher spine density in the basolateral amygdala and a 2-fold higher rate of neuronal firing rate compared to wild-type mice. Unlike their wild-type counterparts, Lcn-2(-/-) mice did not show an increase in dendritic spine density in response to stress but did show a distinct pattern of spine morphology. Thus, amygdala-specific neuronal responses to Lcn-2 may represent a mechanism for behavioural adaptation to psychological stress.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e61046</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23593384">23593384</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3621903">3621903</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0061046">10.1371/journal.pone.0061046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PhenoDigm: analyzing curated annotations to associate animal models with human diseases.</h4>
<p class="authors"><span class="author">Smedley D</span>, <span class="author">Oellrich A</span>, <span class="author">K&ouml;hler S</span>, <span class="author">Ruef B</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Westerfield M</span>, <span class="author">Robinson P</span>, <span class="author">Lewis S</span> and <span class="author">Mungall C</span></p>
<p class="affiliation">Mouse Informatics Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%64%73%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The ultimate goal of studying model organisms is to translate what is learned into useful knowledge about normal human biology and disease to facilitate treatment and early screening for diseases. Recent advances in genomic technologies allow for rapid generation of models with a range of targeted genotypes as well as their characterization by high-throughput phenotyping. As an abundance of phenotype data become available, only systematic analysis will facilitate valid conclusions to be drawn from these data and transferred to human diseases. Owing to the volume of data, automated methods are preferable, allowing for a reliable analysis of the data and providing evidence about possible gene-disease associations. Here, we propose Phenotype comparisons for DIsease Genes and Models (PhenoDigm), as an automated method to provide evidence about gene-disease associations by analysing phenotype information. PhenoDigm integrates data from a variety of model organisms and, at the same time, uses several intermediate scoring methods to identify only strongly data-supported gene candidates for human genetic diseases. We show results of an automated evaluation as well as selected manually assessed examples that support the validity of PhenoDigm. Furthermore, we provide guidance on how to browse the data with PhenoDigm&#39;s web interface and illustrate its usefulness in supporting research. Database URL: http://www.sanger.ac.uk/resources/databases/phenodigm</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01HG004838-02</span>; NIH HHS: <span class="grant">R24 OD011883</span>, <span class="grant">R24OD011883</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2013</span>;<span class="volume">2013</span>;<span class="pagerange">bat025</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23660285">23660285</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3649640">3649640</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbat025">10.1093/database/bat025</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.</h4>
<p class="authors"><span class="author">Smith JJ</span>, <span class="author">Kuraku S</span>, <span class="author">Holt C</span>, <span class="author">Sauka-Spengler T</span>, <span class="author">Jiang N</span>, <span class="author">Campbell MS</span>, <span class="author">Yandell MD</span>, <span class="author">Manousaki T</span>, <span class="author">Meyer A</span>, <span class="author">Bloom OE</span>, <span class="author">Morgan JR</span>, <span class="author">Buxbaum JD</span>, <span class="author">Sachidanandam R</span>, <span class="author">Sims C</span>, <span class="author">Garruss AS</span>, <span class="author">Cook M</span>, <span class="author">Krumlauf R</span>, <span class="author">Wiedemann LM</span>, <span class="author">Sower SA</span>, <span class="author">Decatur WA</span>, <span class="author">Hall JA</span>, <span class="author">Amemiya CT</span>, <span class="author">Saha NR</span>, <span class="author">Buckley KM</span>, <span class="author">Rast JP</span>, <span class="author">Das S</span>, <span class="author">Hirano M</span>, <span class="author">McCurley N</span>, <span class="author">Guo P</span>, <span class="author">Rohner N</span>, <span class="author">Tabin CJ</span>, <span class="author">Piccinelli P</span>, <span class="author">Elgar G</span>, <span class="author">Ruffier M</span>, <span class="author">Aken BL</span>, <span class="author">Searle SM</span>, <span class="author">Muffato M</span>, <span class="author">Pignatelli M</span>, <span class="author">Herrero J</span>, <span class="author">Jones M</span>, <span class="author">Brown CT</span>, <span class="author">Chung-Davidson YW</span>, <span class="author">Nanlohy KG</span>, <span class="author">Libants SV</span>, <span class="author">Yeh CY</span>, <span class="author">McCauley DW</span>, <span class="author">Langeland JA</span>, <span class="author">Pancer Z</span>, <span class="author">Fritzsch B</span>, <span class="author">de Jong PJ</span>, <span class="author">Zhu B</span>, <span class="author">Fulton LL</span>, <span class="author">Theising B</span>, <span class="author">Flicek P</span>, <span class="author">Bronner ME</span>, <span class="author">Warren WC</span>, <span class="author">Clifton SW</span>, <span class="author">Wilson RK</span> and <span class="author">Li W</span></p>
<p class="affiliation">1] Department of Biology, University of Kentucky, Lexington, Kentucky, USA. [2] Benaroya Research Institute at Virginia Mason, Seattle, Washington, USA.</p>
<p class="abstract">Lampreys are representatives of an ancient vertebrate lineage that diverged from our own âˆ¼500 million years ago. By virtue of this deeply shared ancestry, the sea lamprey (P. marinus) genome is uniquely poised to provide insight into the ancestry of vertebrate genomes and the underlying principles of vertebrate biology. Here, we present the first lamprey whole-genome sequence and assembly. We note challenges faced owing to its high content of repetitive elements and GC bases, as well as the absence of broad-scale sequence information from closely related species. Analyses of the assembly indicate that two whole-genome duplications likely occurred before the divergence of ancestral lamprey and gnathostome lineages. Moreover, the results help define key evolutionary events within vertebrate lineages, including the origin of myelin-associated proteins and the development of appendages. The lamprey genome provides an important resource for reconstructing vertebrate origins and the evolutionary events that have shaped the genomes of extant organisms.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23435085">23435085</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2568">10.1038/ng.2568</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sherlock Genomes - viral investigator.</h4>
<p class="authors"><span class="author">Smith SE</span> and <span class="author">Wash RS</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">This month&#39;s Genome Watch highlights how deep sequencing technologies have vastly reduced the time and prior knowledge needed to generate viral genomes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">975284</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">3</span>;<span class="pagerange">150</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23411861">23411861</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2979">10.1038/nrmicro2979</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chicken interferon-inducible transmembrane protein 3 restricts influenza viruses and lyssaviruses in vitro.</h4>
<p class="authors"><span class="author">Smith SE</span>, <span class="author">Gibson MS</span>, <span class="author">Wash RS</span>, <span class="author">Ferrara F</span>, <span class="author">Wright E</span>, <span class="author">Temperton N</span>, <span class="author">Kellam P</span> and <span class="author">Fife M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">Interferon-inducible transmembrane protein 3 (IFITM3) is an effector protein of the innate immune system. It confers potent, cell-intrinsic resistance to infection by diverse enveloped viruses both in vitro and in vivo, including influenza viruses, West Nile virus, and dengue virus. IFITM3 prevents cytosolic entry of these viruses by blocking complete virus envelope fusion with cell endosome membranes. Although the IFITM locus, which includes IFITM1, -2, -3, and -5, is present in mammalian species, this locus has not been unambiguously identified or functionally characterized in avian species. Here, we show that the IFITM locus exists in chickens and is syntenic with the IFITM locus in mammals. The chicken IFITM3 protein restricts cell infection by influenza A viruses and lyssaviruses to a similar level as its human orthologue. Furthermore, we show that chicken IFITM3 is functional in chicken cells and that knockdown of constitutive expression in chicken fibroblasts results in enhanced infection by influenza A virus. Chicken IFITM2 and -3 are constitutively expressed in all tissues examined, whereas IFITM1 is only expressed in the bursa of Fabricius, gastrointestinal tract, cecal tonsil, and trachea. Despite being highly divergent at the amino acid level, IFITM3 proteins of birds and mammals can restrict replication of viruses that are able to infect different host species, suggesting IFITM proteins may provide a crucial barrier for zoonotic infections.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/J004448/1</span>, <span class="grant">BB/L00397X/1</span>; Medical Research Council: <span class="grant">975284</span>, <span class="grant">G0600369</span>, <span class="grant">G1000413</span>; PHS HHS: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2013</span>;<span class="volume">87</span>;<span class="number">23</span>;<span class="pagerange">12957-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24067955">24067955</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3838109">3838109</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.01443-13">10.1128/JVI.01443-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variants from lipid-related pathways and risk for incident myocardial infarction.</h4>
<p class="authors"><span class="author">Song C</span>, <span class="author">Pedersen NL</span>, <span class="author">Reynolds CA</span>, <span class="author">Sabater-Lleal M</span>, <span class="author">Kanoni S</span>, <span class="author">Willenborg C</span>, <span class="author">CARDIoGRAMplusC4D Consortium</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Watkins H</span>, <span class="author">Hamsten A</span>, <span class="author">Prince JA</span> and <span class="author">Ingelsson E</span></p>
<p class="affiliation">Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden.</p><p class="abstract"><strong>Background</strong>: Circulating lipids levels, as well as several familial lipid metabolism disorders, are strongly associated with initiation and progression of atherosclerosis and incidence of myocardial infarction (MI).</p><p class="abstract"><strong>Objectives</strong>: We hypothesized that genetic variants associated with circulating lipid levels would also be associated with MI incidence, and have tested this in three independent samples.</p><p class="abstract">Using age- and sex-adjusted additive genetic models, we analyzed 554 single nucleotide polymorphisms (SNPs) in 41 candidate gene regions proposed to be involved in lipid-related pathways potentially predisposing to incidence of MI in 2,602 participants of the Swedish Twin Register (STR; 57% women). All associations with nominal P&lt;0.01 were further investigated in the Uppsala Longitudinal Study of Adult Men (ULSAM; N = 1,142).</p><p class="abstract"><strong>Results</strong>: In the present study, we report associations of lipid-related SNPs with incident MI in two community-based longitudinal studies with in silico replication in a meta-analysis of genome-wide association studies. Overall, there were 9 SNPs in STR with nominal P-value &lt;0.01 that were successfully genotyped in ULSAM. rs4149313 located in ABCA1 was associated with MI incidence in both longitudinal study samples with nominal significance (hazard ratio, 1.36 and 1.40; P-value, 0.004 and 0.015 in STR and ULSAM, respectively). In silico replication supported the association of rs4149313 with coronary artery disease in an independent meta-analysis including 173,975 individuals of European descent from the CARDIoGRAMplusC4D consortium (odds ratio, 1.03; P-value, 0.048).</p><p class="abstract"><strong>Conclusions</strong>: rs4149313 is one of the few amino acid changing variants in ABCA1 known to associate with reduced cholesterol efflux. Our results are suggestive of a weak association between this variant and the development of atherosclerosis and MI.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">AG 04563</span>, <span class="grant">AG028555</span>, <span class="grant">AG08724</span>, <span class="grant">AG08861</span>, <span class="grant">AG10175</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e60454</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23555974">23555974</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3612051">3612051</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0060454">10.1371/journal.pone.0060454</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal evolution among leaf-nosed nectarivorous bats--evidence from cross-species chromosome painting (Phyllostomidae, Chiroptera).</h4>
<p class="authors"><span class="author">Sotero-Caio CG</span>, <span class="author">Volleth M</span>, <span class="author">Gollahon LS</span>, <span class="author">Fu B</span>, <span class="author">Cheng W</span>, <span class="author">Ng BL</span>, <span class="author">Yang F</span> and <span class="author">Baker RJ</span></p>
<p class="affiliation">Department of Biological Sciences, Texas Tech University, Lubbock, TX 79409, USA. <a href="mailto:%63%69%62%65%6C%65.%63%61%69%6F%40%74%74%75.%65%64%75">&#99;&#105;&#98;&#101;&#108;&#101;&#46;&#99;&#97;&#105;&#111;&#64;&#116;&#116;&#117;&#46;&#101;&#100;&#117;</a>.</p>
<p class="abstract">Background: New World leaf-nosed bats, Phyllostomidae, represent a lineage of Chiroptera marked by unprecedented morphological/ecological diversity and extensive intergeneric chromosomal reorganization. There are still disagreements regarding their systematic relationships due to morphological convergence among some groups. Their history of karyotypic evolution also remains to be documented.

Results: To better understand the evolutionary relationships within Phyllostomidae, we developed chromosome paints from the bat species Macrotus californicus. We tested the potential of these paints as phylogenetic tools by looking for chromosomal signatures in two lineages of nectarivorous phyllostomids whose independent origins have been statistically supported by molecular phylogenies. By examining the chromosomal homologies defined by chromosome painting among two representatives of the subfamily Glossophaginae (Glossophaga soricina and Anoura cultrata) and one species from the subfamily Lonchophyllinae (Lonchophylla concava), we found chromosomal correspondence in regions not previously detected by other comparative cytogenetic techniques. We proposed the corresponding human chromosomal segments for chromosomes of the investigated species and found two syntenic associations shared by G. soricina and A. cultrata.

Conclusion: Comparative painting with whole chromosome-specific paints of M. californicus demonstrates an extensive chromosomal reorganization within the two lineages of nectarivorous phyllostomids, with a large number of chromosomes shared between M. californicus and G. soricina. We show that the evolution of nectar-feeding bats occurs mainly by reshuffling of chiropteran Evolutionarily Conserved Units (ECUs). Robertsonian fusions/fissions and inversions seem to be important modifiers of phyllostomid karyotypes, and autapomorphic character states are common within species. Macrotus californicus chromosome paints will be a valuable tool for documenting the pattern of karyotypic evolution within Phyllostomidae radiation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="pagerange">276</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24369737">24369737</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3880000">3880000</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-13-276">10.1186/1471-2148-13-276</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ear</h4>
<p class="authors"><span class="author">SPIDEN,S.L.;</span> and <span class="author">STEEL,K.P.;</span></p>
<p class="publication"><span class="title">Embryos, Genes and Birth Defects</span>&nbsp;<span class="year">2013</span>;<span class="number">10</span>;<span class="pagerange">231-62</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1002%2F9780470090121.ch10">10.1002/9780470090121.ch10</a>; URL: <a title="http://onlinelibrary.wiley.com/doi/10.1002/9780470090121.ch10/summary;jsessionid=77DED0C0B7226E872C3D32FCAED386DE.f02t02" href="http://onlinelibrary.wiley.com/doi/10.1002/9780470090121.ch10/summary;jsessionid=77DED0C0B7226E872C3D32FCAED386DE.f02t02">http://onlinelibrary.wiley.com/doi/<span class="print-hide">...</span><span class="print-show">10.1002/9780470090121.ch10/summary;jsessionid=77DED0C0B7226E872C3D32FCAED386DE.f</span>02t02</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cooperativity and rapid evolution of cobound transcription factors in closely related mammals.</h4>
<p class="authors"><span class="author">Stefflova K</span>, <span class="author">Thybert D</span>, <span class="author">Wilson MD</span>, <span class="author">Streeter I</span>, <span class="author">Aleksic J</span>, <span class="author">Karagianni P</span>, <span class="author">Brazma A</span>, <span class="author">Adams DJ</span>, <span class="author">Talianidis I</span>, <span class="author">Marioni JC</span>, <span class="author">Flicek P</span> and <span class="author">Odom DT</span></p>
<p class="affiliation">Cancer Research UK Cambridge Institute, Li Ka Shing Centre, University of Cambridge, Cambridge CB2 0RE, UK.</p>
<p class="abstract">To mechanistically characterize the microevolutionary processes active in altering transcription factor (TF) binding among closely related mammals, we compared the genome-wide binding of three tissue-specific TFs that control liver gene expression in six rodents. Despite an overall fast turnover of TF binding locations between species, we identified thousands of TF regions of highly constrained TF binding intensity. Although individual mutations in bound sequence motifs can influence TF binding, most binding differences occur in the absence of nearby sequence variations. Instead, combinatorial binding was found to be significant for genetic and evolutionary stability; cobound TFs tend to disappear in concert and were sensitive to genetic knockout of partner TFs. The large, qualitative differences in genomic regions bound between closely related mammals, when contrasted with the smaller, quantitative TF binding differences among Drosophila species, illustrate how genome structure and population genetics together shape regulatory evolution.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">154</span>;<span class="number">3</span>;<span class="pagerange">530-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23911320">23911320</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3732390">3732390</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.07.007">10.1016/j.cell.2013.07.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessment of transcript reconstruction methods for RNA-seq.</h4>
<p class="authors"><span class="author">Steijger T</span>, <span class="author">Abril JF</span>, <span class="author">Engstr&ouml;m PG</span>, <span class="author">Kokocinski F</span>, <span class="author">Hubbard TJ</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Harrow J</span>, <span class="author">Bertone P</span> and <span class="author">RGASP Consortium</span></p>
<p class="affiliation">European Molecular Biology Laboratory, European Bioinformatics Institute, Cambridge, UK.</p><p class="abstract">We evaluated 25 protocol variants of 14 independent computational methods for exon identification, transcript reconstruction and expression-level quantification from RNA-seq data. Our results show that most algorithms are able to identify discrete transcript components with high success rates but that assembly of complete isoform structures poses a major challenge even when all constituent elements are identified. Expression-level estimates also varied widely across methods, even when based on similar transcript models. Consequently, the complexity of higher eukaryotic genomes imposes severe limitations on transcript recall and splice product discrimination that are likely to remain limiting factors for the analysis of current-generation RNA-seq data.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U41 HG007234</span>, <span class="grant">U54 HG004555</span>, <span class="grant">U54HG004555</span>, <span class="grant">U54HG004557</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span>, <span class="grant">095908</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2013</span>;<span class="volume">10</span>;<span class="number">12</span>;<span class="pagerange">1177-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24185837">24185837</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3851240">3851240</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.2714">10.1038/nmeth.2714</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">So, you want to sequence a genome...</h4>
<p class="authors"><span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%64%73%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">7</span>;<span class="pagerange">128</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23906089">23906089</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-7-128">10.1186/gb-2013-14-7-128</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole exome sequencing of adenoid cystic carcinoma.</h4>
<p class="authors"><span class="author">Stephens PJ</span>, <span class="author">Davies HR</span>, <span class="author">Mitani Y</span>, <span class="author">Van Loo P</span>, <span class="author">Shlien A</span>, <span class="author">Tarpey PS</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Cheverton A</span>, <span class="author">Bignell GR</span>, <span class="author">Butler AP</span>, <span class="author">Gamble J</span>, <span class="author">Gamble S</span>, <span class="author">Hardy C</span>, <span class="author">Hinton J</span>, <span class="author">Jia M</span>, <span class="author">Jayakumar A</span>, <span class="author">Jones D</span>, <span class="author">Latimer C</span>, <span class="author">McLaren S</span>, <span class="author">McBride DJ</span>, <span class="author">Menzies A</span>, <span class="author">Mudie L</span>, <span class="author">Maddison M</span>, <span class="author">Raine K</span>, <span class="author">Nik-Zainal S</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Teague JW</span>, <span class="author">Varela I</span>, <span class="author">Wedge DC</span>, <span class="author">Whitmore I</span>, <span class="author">Lippman SM</span>, <span class="author">McDermott U</span>, <span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span>, <span class="author">El-Naggar AK</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom.</p><p class="abstract">Adenoid cystic carcinoma (ACC) is a rare malignancy that can occur in multiple organ sites and is primarily found in the salivary gland. While the identification of recurrent fusions of the MYB-NFIB genes have begun to shed light on the molecular underpinnings, little else is known about the molecular genetics of this frequently fatal cancer. We have undertaken exome sequencing in a series of 24 ACC to further delineate the genetics of the disease. We identified multiple mutated genes that, combined, implicate chromatin deregulation in half of cases. Further, mutations were identified in known cancer genes, including PIK3CA, ATM, CDKN2A, SF3B1, SUFU, TSC1, and CYLD. Mutations in NOTCH1/2 were identified in 3 cases, and we identify the negative NOTCH signaling regulator, SPEN, as a new cancer gene in ACC with mutations in 5 cases. Finally, the identification of 3 likely activating mutations in the tyrosine kinase receptor FGFR2, analogous to those reported in ovarian and endometrial carcinoma, point to potential therapeutic avenues for a subset of cases.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P50 CA097007</span>; NIDCR NIH HHS: <span class="grant">U01DE019765</span>; WETP NIH HHS: <span class="grant">WT088340MA</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">093867</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2013</span>;<span class="volume">123</span>;<span class="number">7</span>;<span class="pagerange">2965-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23778141">23778141</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3999050">3999050</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI67201">10.1172/JCI67201</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The intermediate filament protein, vimentin, is a regulator of NOD2 activity.</h4>
<p class="authors"><span class="author">Stevens C</span>, <span class="author">Henderson P</span>, <span class="author">Nimmo ER</span>, <span class="author">Soares DC</span>, <span class="author">Dogan B</span>, <span class="author">Simpson KW</span>, <span class="author">Barrett JC</span>, <span class="author">International Inflammatory Bowel Disease Genetics Consortium</span>, <span class="author">Wilson DC</span> and <span class="author">Satsangi J</span></p>
<p class="affiliation">Centre for Molecular Medicine, University of Edinburgh, Western General Hospital, Crewe Road, Edinburgh EH4 2XU, UK. <a href="mailto:%63%72%61%69%67.%73%74%65%76%65%6E%73%40%65%64.%61%63.%75%6B">&#99;&#114;&#97;&#105;&#103;&#46;&#115;&#116;&#101;&#118;&#101;&#110;&#115;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Objective</strong>: Mutations in the nucleotide-binding oligomerisation domain-containing protein 2 (NOD2) gene remain the strongest genetic determinants for Crohn&#39;s disease (CD). Having previously identified vimentin as a novel NOD2-interacting protein, the authors aimed to investigate the regulatory effects of vimentin on NOD2 function and the association of variants in Vim with CD susceptibility.</p><p class="abstract"><strong>Design</strong>: Coimmunoprecipitation, fluorescent microscopy and fractionation were used to confirm the interaction between NOD2 and vimentin. HEK293 cells stably expressing wild-type NOD2 or a NOD2 frameshift variant (L1007fs) and SW480 colonic epithelial cells were used alongside the vimentin inhibitor, withaferin A (WFA), to assess effects on NOD2 function using the nuclear factor-kappaB (NF-&kappa;B) reporter gene, green fluorescent protein-LC3-based autophagy, and bacterial gentamicin protection assays. International genome-wide association meta-analysis data were used to test for associations of single-nucleotide polymorphisms in Vim with CD susceptibility.</p><p class="abstract"><strong>Results</strong>: The leucine-rich repeat domain of NOD2 contained the elements required for vimentin binding; CD-associated polymorphisms disrupted this interaction. NOD2 and vimentin colocalised at the cell plasma membrane, and cytosolic mislocalisation of the L1007fs and R702W variants correlated with an inability to interact with vimentin. Use of WFA demonstrated that vimentin was required for NOD2-dependent NF-&kappa;B activation and muramyl dipeptide-induced autophagy induction, and that NOD2 and vimentin regulated the invasion and survival properties of a CD-associated adherent-invasive Escherichia coli strain. Genetic analysis revealed an association signal across the haplotype block containing Vim.</p><p class="abstract"><strong>Conclusion</strong>: Vimentin is an important regulator of NOD2 function and a potential novel therapeutic target in the treatment of CD. In addition, Vim is a candidate susceptibility gene for CD, supporting the functional data.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">CZB/4/585</span>, <span class="grant">ETM/137</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0600329</span>, <span class="grant">G0800675</span>, <span class="grant">G0800759</span></p>
<p class="publication"><span class="title">Gut</span>&nbsp;<span class="year">2013</span>;<span class="volume">62</span>;<span class="number">5</span>;<span class="pagerange">695-707</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22684479">22684479</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4225453">4225453</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fgutjnl-2011-301775">10.1136/gutjnl-2011-301775</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in B3GALNT2 cause congenital muscular dystrophy and hypoglycosylation of &alpha;-dystroglycan.</h4>
<p class="authors"><span class="author">Stevens E</span>, <span class="author">Carss KJ</span>, <span class="author">Cirak S</span>, <span class="author">Foley AR</span>, <span class="author">Torelli S</span>, <span class="author">Willer T</span>, <span class="author">Tambunan DE</span>, <span class="author">Yau S</span>, <span class="author">Brodd L</span>, <span class="author">Sewry CA</span>, <span class="author">Feng L</span>, <span class="author">Haliloglu G</span>, <span class="author">Orhan D</span>, <span class="author">Dobyns WB</span>, <span class="author">Enns GM</span>, <span class="author">Manning M</span>, <span class="author">Krause A</span>, <span class="author">Salih MA</span>, <span class="author">Walsh CA</span>, <span class="author">Hurles M</span>, <span class="author">Campbell KP</span>, <span class="author">Manzini MC</span>, <span class="author">UK10K Consortium</span>, <span class="author">Stemple D</span>, <span class="author">Lin YY</span> and <span class="author">Muntoni F</span></p>
<p class="affiliation">Dubowitz Neuromuscular Centre, UCL Institute of Child Health, London, UK.</p><p class="abstract">Mutations in several known or putative glycosyltransferases cause glycosylation defects in &alpha;-dystroglycan (&alpha;-DG), an integral component of the dystrophin glycoprotein complex. The hypoglycosylation reduces the ability of &alpha;-DG to bind laminin and other extracellular matrix ligands and is responsible for the pathogenesis of an inherited subset of muscular dystrophies known as the dystroglycanopathies. By exome and Sanger sequencing we identified two individuals affected by a dystroglycanopathy with mutations in &beta;-1,3-N-acetylgalactosaminyltransferase 2 (B3GALNT2). B3GALNT2 transfers N-acetyl galactosamine (GalNAc) in a &beta;-1,3 linkage to N-acetyl glucosamine (GlcNAc). A subsequent study of a separate cohort of individuals identified recessive mutations in four additional cases that were all affected by dystroglycanopathy with structural brain involvement. We show that functional dystroglycan glycosylation was reduced in the fibroblasts and muscle (when available) of these individuals via flow cytometry, immunoblotting, and immunocytochemistry. B3GALNT2 localized to the endoplasmic reticulum, and this localization was perturbed by some of the missense mutations identified. Moreover, knockdown of b3galnt2 in zebrafish recapitulated the human congenital muscular dystrophy phenotype with reduced motility, brain abnormalities, and disordered muscle fibers with evidence of damage to both the myosepta and the sarcolemma. Functional dystroglycan glycosylation was also reduced in the b3galnt2 knockdown zebrafish embryos. Together these results demonstrate a role for B3GALNT2 in the glycosylation of &alpha;-DG and show that B3GALNT2 mutations can cause dystroglycanopathy with muscle and brain involvement.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Medical Research Council: <span class="grant">MR/K000608/1</span>; NICHD NIH HHS: <span class="grant">K99HD067379</span>, <span class="grant">P30HD19655</span>; NIMH NIH HHS: <span class="grant">RC2MH089952</span>; NINDS NIH HHS: <span class="grant">1U54NS053672</span>, <span class="grant">U54 NS053672</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">92</span>;<span class="number">3</span>;<span class="pagerange">354-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23453667">23453667</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3591840">3591840</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.01.016">10.1016/j.ajhg.2013.01.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The non-obese diabetic mouse sequence, annotation and variation resource: an aid for investigating type 1 diabetes.</h4>
<p class="authors"><span class="author">Steward CA</span>, <span class="author">Gonzalez JM</span>, <span class="author">Trevanion S</span>, <span class="author">Sheppard D</span>, <span class="author">Kerry G</span>, <span class="author">Gilbert JG</span>, <span class="author">Wicker LS</span>, <span class="author">Rogers J</span> and <span class="author">Harrow JL</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%63%61%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Model organisms are becoming increasingly important for the study of complex diseases such as type 1 diabetes (T1D). The non-obese diabetic (NOD) mouse is an experimental model for T1D having been bred to develop the disease spontaneously in a process that is similar to humans. Genetic analysis of the NOD mouse has identified around 50 disease loci, which have the nomenclature Idd for insulin-dependent diabetes, distributed across at least 11 different chromosomes. In total, 21 Idd regions across 6 chromosomes, that are major contributors to T1D susceptibility or resistance, were selected for finished sequencing and annotation at the Wellcome Trust Sanger Institute. Here we describe the generation of 40.4 mega base-pairs of finished sequence from 289 bacterial artificial chromosomes for the NOD mouse. Manual annotation has identified 738 genes in the diabetes sensitive NOD mouse and 765 genes in homologous regions of the diabetes resistant C57BL/6J reference mouse across 19 candidate Idd regions. This has allowed us to call variation consequences between homologous exonic sequences for all annotated regions in the two mouse strains. We demonstrate the importance of this resource further by illustrating the technical difficulties that regions of inter-strain structural variation between the NOD mouse and the C57BL/6J reference mouse can cause for current next generation sequencing and assembly techniques. Furthermore, we have established that the variation rate in the Idd regions is 2.3 times higher than the mean found for the whole genome assembly for the NOD/ShiLtJ genome, which we suggest reflects the fact that positive selection for functional variation in immune genes is beneficial in regard to host defence. In summary, we provide an important resource, which aids the analysis of potential causative genes involved in T1D susceptibility. Database URLs: http://www.sanger.ac.uk/resources/mouse/nod/; http://vega-previous.sanger.ac.uk/info/data/mouse_regions.html#Idd</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 15416</span>; Wellcome Trust: <span class="grant">091157</span>, <span class="grant">096388</span>, <span class="grant">100140</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2013</span>;<span class="volume">2013</span>;<span class="pagerange">bat032</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23729657">23729657</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3668384">3668384</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbat032">10.1093/database/bat032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deletion of TOP3&beta;, a component of FMRP-containing mRNPs, contributes to neurodevelopmental disorders.</h4>
<p class="authors"><span class="author">Stoll G</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Linder B</span>, <span class="author">Suvisaari J</span>, <span class="author">Brosi C</span>, <span class="author">Hennah W</span>, <span class="author">Lepp&auml; V</span>, <span class="author">Torniainen M</span>, <span class="author">Ripatti S</span>, <span class="author">Ala-Mello S</span>, <span class="author">Pl&ouml;ttner O</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Varilo T</span>, <span class="author">Tallila J</span>, <span class="author">Kristiansson K</span>, <span class="author">Isohanni M</span>, <span class="author">Kaprio J</span>, <span class="author">Eriksson JG</span>, <span class="author">Raitakari OT</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Jarvelin MR</span>, <span class="author">Salomaa V</span>, <span class="author">Hurles M</span>, <span class="author">Stefansson H</span>, <span class="author">Peltonen L</span>, <span class="author">Sullivan PF</span>, <span class="author">Paunio T</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Daly MJ</span>, <span class="author">Fischer U</span>, <span class="author">Freimer NB</span> and <span class="author">Palotie A</span></p>
<p class="affiliation">Department of Biochemistry, University of W&uuml;rzburg, W&uuml;rzburg, Germany.</p><p class="abstract">Implicating particular genes in the generation of complex brain and behavior phenotypes requires multiple lines of evidence. The rarity of most high-impact genetic variants typically precludes the possibility of accruing statistical evidence that they are associated with a given trait. We found that the enrichment of a rare chromosome 22q11.22 deletion in a recently expanded Northern Finnish sub-isolate enabled the detection of association between TOP3B and both schizophrenia and cognitive impairment. Biochemical analysis of TOP3&beta; revealed that this topoisomerase was a component of cytosolic messenger ribonucleoproteins (mRNPs) and was catalytically active on RNA. The recruitment of TOP3&beta; to mRNPs was independent of RNA cis-elements and was coupled to the co-recruitment of FMRP, the disease gene product in fragile X mental retardation syndrome. Our results indicate a previously unknown role for TOP3&beta; in mRNA metabolism and suggest that it is involved in neurodevelopmental disorders.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">RL1MH083268</span>; NINDS NIH HHS: <span class="grant">P30NS062691</span>; Wellcome Trust: <span class="grant">079643</span>, <span class="grant">WT089062</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nature neuroscience</span>&nbsp;<span class="year">2013</span>;<span class="volume">16</span>;<span class="number">9</span>;<span class="pagerange">1228-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23912948">23912948</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3986889">3986889</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnn.3484">10.1038/nn.3484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Harnessing the genome: development of a hierarchical typing scheme for meticillin-resistant Staphylococcus aureus.</h4>
<p class="authors"><span class="author">Stone MJ</span>, <span class="author">Wain J</span>, <span class="author">Ivens A</span>, <span class="author">Feltwell T</span>, <span class="author">Kearns AM</span> and <span class="author">Bamford KB</span></p>
<p class="affiliation">Department of Microbiology, Imperial College Healthcare NHS Trust, London, UK.</p><p class="abstract">A major barrier to using genome sequencing in medical microbiology is the ability to interpret the data. New schemes that provide information about the importance of sequence variation in both clinical and public health settings are required. Meticillin-resistant Staphylococcus aureus (MRSA) is an important nosocomial pathogen that is being observed with increasing frequency in community settings. Better tools are needed to improve our understanding of its transmissibility and micro-epidemiology in order to develop effective interventions. Using DNA microarray technology we identified a set of 20 binary targets whose presence or absence could be determined by PCR, producing a PCR binary typing scheme (PCR-BT). This was combined with multi-locus sequence type-based, sequence nucleotide polymorphism typing to form a hierarchical typing scheme. When applied to a set of epidemiologically unrelated isolates, a high degree of concordance was observed with PFGE (98.8 %). The scheme was able to detect the presence or absence of an outbreak strain in eight out of nine outbreak investigations, demonstrating epidemiological concordance. PCR-BT was better than PFGE at distinguishing between outbreak strains, particularly where epidemic MRSA-15 was involved. The method developed here is a rapid, digital typing scheme for S. aureus for use in both micro- and macro-epidemiological investigations that has the advantage of being suitable for use in routine diagnostic laboratories. The targets are defined and therefore the types can be defined by any platform capable of detecting the sequences used, including whole genome sequencing.</p>
<p class="publication"><span class="title">Journal of medical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">62</span>;<span class="number">Pt 1</span>;<span class="pagerange">36-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23002072">23002072</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fjmm.0.049957-0">10.1099/jmm.0.049957-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Journeys into the genome of cancer cells.</h4>
<p class="authors"><span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%6D%72%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#114;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">EMBO molecular medicine</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23339072" rel="external">23339072</a>; DOI: <a href="http://dx.doi.org/10.1002%2Femmm.201202388" rel="external">10.1002/emmm.201202388</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and physiological variability within Group II (non-proteolytic) Clostridium botulinum.</h4>
<p class="authors"><span class="author">Stringer SC</span>, <span class="author">Carter AT</span>, <span class="author">Webb MD</span>, <span class="author">Wachnicka E</span>, <span class="author">Crossman LC</span>, <span class="author">Sebaihia M</span> and <span class="author">Peck MW</span></p>
<p class="affiliation">Institute of Food Research (IFR), Norwich Research Park, Colney, Norwich NR4 7UA, UK. <a href="mailto:%73%61%6E%64%72%61.%73%74%72%69%6E%67%65%72%40%69%66%72.%61%63.%75%6B">&#115;&#97;&#110;&#100;&#114;&#97;&#46;&#115;&#116;&#114;&#105;&#110;&#103;&#101;&#114;&#64;&#105;&#102;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: Clostridium botulinum is a group of four physiologically and phylogenetically distinct bacteria that produce botulinum neurotoxin. While studies have characterised variability between strains of Group I (proteolytic) C. botulinum, the genetic and physiological variability and relationships between strains within Group II (non-proteolytic) C. botulinum are not well understood. In this study the genome of Group II strain C. botulinum Eklund 17B (NRP) was sequenced and used to construct a whole genome DNA microarray. This was used in a comparative genomic indexing study to compare the relatedness of 43 strains of Group II C. botulinum (14 type B, 24 type E and 5 type F). These results were compared with characteristics determined from physiological tests.

Results: Whole genome indexing showed that strains of Group II C. botulinum isolated from a wide variety of environments over more than 75 years clustered together indicating the genetic background of Group II C. botulinum is stable. Further analysis showed that strains forming type B or type F toxin are closely related with only toxin cluster genes targets being unique to either type. Strains producing type E toxin formed a separate subset. Carbohydrate fermentation tests supported the observation that type B and F strains form a separate subset to type E strains. All the type F strains and most of type B strains produced acid from amylopectin, amylose and glycogen whereas type E strains did not. However, these two subsets did not differ strongly in minimum growth temperature or maximum NaCl concentration for growth. No relationship was found between tellurite resistance and toxin type despite all the tested type B and type F strains carrying tehB, while the sequence was absent or diverged in all type E strains.

Conclusions: Although Group II C. botulinum form a tight genetic group, genomic and physiological analysis indicates there are two distinct subsets within this group. All type B strains and type F strains are in one subset and all type E strains in the other.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">333</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23679073">23679073</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672017">3672017</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-14-333">10.1186/1471-2164-14-333</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The malarial serine protease SUB1 plays an essential role in parasite liver stage development.</h4>
<p class="authors"><span class="author">Suarez C</span>, <span class="author">Volkmann K</span>, <span class="author">Gomes AR</span>, <span class="author">Billker O</span> and <span class="author">Blackman MJ</span></p>
<p class="affiliation">Division of Parasitology, Medical Research Council National Institute for Medical Research, Mill Hill, London, United Kingdom.</p><p class="abstract">Transmission of the malaria parasite to its vertebrate host involves an obligatory exoerythrocytic stage in which extensive asexual replication of the parasite takes place in infected hepatocytes. The resulting liver schizont undergoes segmentation to produce thousands of daughter merozoites. These are released to initiate the blood stage life cycle, which causes all the pathology associated with the disease. Whilst elements of liver stage merozoite biology are similar to those in the much better-studied blood stage merozoites, little is known of the molecular players involved in liver stage merozoite production. To facilitate the study of liver stage biology we developed a strategy for the rapid production of complex conditional alleles by recombinase mediated engineering in Escherichia coli, which we used in combination with existing Plasmodium berghei deleter lines expressing Flp recombinase to study subtilisin-like protease 1 (SUB1), a conserved Plasmodium serine protease previously implicated in blood stage merozoite maturation and egress. We demonstrate that SUB1 is not required for the early stages of intrahepatic growth, but is essential for complete development of the liver stage schizont and for production of hepatic merozoites. Our results indicate that inhibitors of SUB1 could be used in prophylactic approaches to control or block the clinically silent pre-erythrocytic stage of the malaria parasite life cycle.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>, <span class="grant">U117532063</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">e1003811</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24348254">24348254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3861531">3861531</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003811">10.1371/journal.ppat.1003811</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">De Novo Loss-of-Function Mutations in CHD2 Cause a Fever-Sensitive Myoclonic Epileptic Encephalopathy Sharing Features with Dravet Syndrome.</h4>
<p class="authors"><span class="author">Suls A</span>, <span class="author">Jaehn JA</span>, <span class="author">Kecsk&eacute;s A</span>, <span class="author">Weber Y</span>, <span class="author">Weckhuysen S</span>, <span class="author">Craiu DC</span>, <span class="author">Siekierska A</span>, <span class="author">Dj&eacute;mi&eacute; T</span>, <span class="author">Afrikanova T</span>, <span class="author">Gormley P</span>, <span class="author">von Spiczak S</span>, <span class="author">Kluger G</span>, <span class="author">Iliescu CM</span>, <span class="author">Talvik T</span>, <span class="author">Talvik I</span>, <span class="author">Meral C</span>, <span class="author">Caglayan HS</span>, <span class="author">Giraldez BG</span>, <span class="author">Serratosa J</span>, <span class="author">Lemke JR</span>, <span class="author">Hoffman-Zacharska D</span>, <span class="author">Szczepanik E</span>, <span class="author">Barisic N</span>, <span class="author">Komarek V</span>, <span class="author">Hjalgrim H</span>, <span class="author">M&oslash;ller RS</span>, <span class="author">Linnankivi T</span>, <span class="author">Dimova P</span>, <span class="author">Striano P</span>, <span class="author">Zara F</span>, <span class="author">Marini C</span>, <span class="author">Guerrini R</span>, <span class="author">Depienne C</span>, <span class="author">Baulac S</span>, <span class="author">Kuhlenb&auml;umer G</span>, <span class="author">Crawford AD</span>, <span class="author">Lehesjoki AE</span>, <span class="author">de Witte PA</span>, <span class="author">Palotie A</span>, <span class="author">Lerche H</span>, <span class="author">Esguerra CV</span>, <span class="author">De Jonghe P</span>, <span class="author">Helbig I</span> and <span class="author">EuroEPINOMICS RES Consortium</span></p>
<p class="affiliation">Neurogenetics group, Department of Molecular Genetics, VIB, 2610 Antwerp, Belgium; Laboratory of Neurogenetics, Institute Born-Bunge, University of Antwerp, 2610 Antwerp, Belgium.</p><p class="abstract">Dravet syndrome is a severe epilepsy syndrome characterized by infantile onset of therapy-resistant, fever-sensitive seizures followed by cognitive decline. Mutations in SCN1A explain about 75% of cases with Dravet syndrome; 90% of these mutations arise de novo. We studied a cohort of nine Dravet-syndrome-affected individuals without an SCN1A mutation (these included some atypical cases with onset at up to 2 years of age) by using whole-exome sequencing in proband-parent trios. In two individuals, we identified a de novo loss-of-function mutation in CHD2 (encoding chromodomain helicase DNA binding protein 2). A third CHD2 mutation was identified in an epileptic proband of a second (stage 2) cohort. All three individuals with a CHD2 mutation had intellectual disability and fever-sensitive generalized seizures, as well as prominent myoclonic seizures starting in the second year of life or later. To explore the functional relevance of CHD2 haploinsufficiency in an in&nbsp;vivo model system, we knocked down chd2 in zebrafish by using targeted morpholino antisense oligomers. chd2-knockdown larvae exhibited altered locomotor activity, and the epileptic nature of this seizure-like behavior was confirmed by field-potential recordings that revealed epileptiform discharges similar to seizures in affected persons. Both altered locomotor activity and epileptiform discharges were absent in appropriate control larvae. Our study provides evidence that de novo loss-of-function mutations in CHD2 are a cause of epileptic encephalopathy with generalized seizures.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">93</span>;<span class="number">5</span>;<span class="pagerange">967-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24207121">24207121</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3824114">3824114</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.09.017">10.1016/j.ajhg.2013.09.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detecting low-affinity extracellular protein interactions using protein microarrays.</h4>
<p class="authors"><span class="author">Sun Y</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">Low-affinity extracellular protein interactions are critical for cellular recognition processes, but are not generally detected by methods that can be applied in a high-throughput manner. This unit describes a protein microarray platform that significantly improves the throughput of assays capable of detecting transient extracellular protein interactions. These methodological improvements now permit screening for novel extracellular receptor-ligand interactions on a genome-wide scale. Curr. Protoc. Protein Sci. 72:27.5.1-27.5.15. © 2013 by John Wiley & Sons, Inc.</p>
<p class="publication"><span class="title">Current protocols in protein science / editorial board, John E. Coligan ... [et al.]</span>&nbsp;<span class="year">2013</span>;<span class="volume">Chapter 27</span>;<span class="pagerange">Unit27.5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23546623">23546623</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471140864.ps2705s72">10.1002/0471140864.ps2705s72</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plasmodium falciparum-like parasites infecting wild apes in southern Cameroon do not represent a recurrent source of human malaria.</h4>
<p class="authors"><span class="author">Sundararaman SA</span>, <span class="author">Liu W</span>, <span class="author">Keele BF</span>, <span class="author">Learn GH</span>, <span class="author">Bittinger K</span>, <span class="author">Mouacha F</span>, <span class="author">Ahuka-Mundeke S</span>, <span class="author">Manske M</span>, <span class="author">Sherrill-Mix S</span>, <span class="author">Li Y</span>, <span class="author">Malenke JA</span>, <span class="author">Delaporte E</span>, <span class="author">Laurent C</span>, <span class="author">Mpoudi Ngole E</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Shaw GM</span>, <span class="author">Rayner JC</span>, <span class="author">Peeters M</span>, <span class="author">Sharp PM</span>, <span class="author">Bushman FD</span> and <span class="author">Hahn BH</span></p>
<p class="affiliation">Departments of Medicine and Microbiology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104.</p>
<p class="abstract">Wild-living chimpanzees and gorillas harbor a multitude of Plasmodium species, including six of the subgenus Laverania, one of which served as the progenitor of Plasmodium falciparum. Despite the magnitude of this reservoir, it is unknown whether apes represent a source of human infections. Here, we used Plasmodium species-specific PCR, single-genome amplification, and 454 sequencing to screen humans from remote areas of southern Cameroon for ape Laverania infections. Among 1,402 blood samples, we found 1,000 to be Plasmodium mitochondrial DNA (mtDNA) positive, all of which contained human parasites as determined by sequencing and/or restriction enzyme digestion. To exclude low-abundance infections, we subjected 514 of these samples to 454 sequencing, targeting a region of the mtDNA genome that distinguishes ape from human Laverania species. Using algorithms specifically developed to differentiate rare Plasmodium variants from 454-sequencing error, we identified single and mixed-species infections with P. falciparum, Plasmodium malariae, and/or Plasmodium ovale. However, none of the human samples contained ape Laverania parasites, including the gorilla precursor of P. falciparum. To characterize further the diversity of P. falciparum in Cameroon, we used single-genome amplification to amplify 3.4-kb mtDNA fragments from 229 infected humans. Phylogenetic analysis identified 62 new variants, all of which clustered with extant P. falciparum, providing further evidence that P. falciparum emerged following a single gorilla-to-human transmission. Thus, unlike Plasmodium knowlesi-infected macaques in southeast Asia, African apes harboring Laverania parasites do not seem to serve as a recurrent source of human malaria, a finding of import to ongoing control and eradication measures.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">17</span>;<span class="pagerange">7020-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23569255">23569255</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3637760">3637760</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1305201110">10.1073/pnas.1305201110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid whole-genome sequencing for investigation of a suspected tuberculosis outbreak.</h4>
<p class="authors"><span class="author">T&ouml;r&ouml;k ME</span>, <span class="author">Reuter S</span>, <span class="author">Bryant J</span>, <span class="author">K&ouml;ser CU</span>, <span class="author">Stinchcombe SV</span>, <span class="author">Nazareth B</span>, <span class="author">Ellington MJ</span>, <span class="author">Bentley SD</span>, <span class="author">Smith GP</span>, <span class="author">Parkhill J</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Cambridge, United Kingdom.</p>
<p class="abstract">Two Southeast Asian students attending the same school in the United Kingdom presented with pulmonary tuberculosis. An epidemiological investigation failed to link the two cases, and drug resistance profiles of the Mycobacterium tuberculosis isolates were discrepant. Whole-genome sequencing of the isolates found them to be genetically identical, suggesting a missed transmission event.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">2</span>;<span class="pagerange">611-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23175259">23175259</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3553910">3553910</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.02279-12">10.1128/JCM.02279-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Spindle Assembly Checkpoint of Oocytes Depends on a Kinetochore Structure Determined by Cohesin in Meiosis I.</h4>
<p class="authors"><span class="author">Tachibana-Konwalski K</span>, <span class="author">Godwin J</span>, <span class="author">Borsos M</span>, <span class="author">Rattani A</span>, <span class="author">Adams DJ</span> and <span class="author">Nasmyth K</span></p>
<p class="affiliation">Institute of Molecular Biotechnology of the Austrian Academy of Sciences (IMBA), Dr. Bohr-Gasse 3, 1030 Vienna, Austria; Department of Biochemistry, University of Oxford, South&nbsp;Parks Road, Oxford OX1 3QU, UK. Electronic address: <a href="mailto:%6B%69%6B%75%65.%74%61%63%68%69%62%61%6E%61%40%69%6D%62%61.%6F%65%61%77.%61%63.%61%74">&#107;&#105;&#107;&#117;&#101;&#46;&#116;&#97;&#99;&#104;&#105;&#98;&#97;&#110;&#97;&#64;&#105;&#109;&#98;&#97;&#46;&#111;&#101;&#97;&#119;&#46;&#97;&#99;&#46;&#97;&#116;</a>.</p><p class="abstract">Since the dissolution of sister chromatid cohesion by separase and cyclin B destruction is irreversible, it is essential to delay both until all chromosomes have bioriented on the mitotic spindle. Kinetochores that are not correctly attached to the spindle generate the mitotic checkpoint complex (MCC), which inhibits the anaphase-promoting complex/cyclosome (APC/C) and blocks anaphase onset. This process is known as the spindle assembly checkpoint (SAC) [1]. The SAC is especially important in meiosis I, where bivalents consisting of homologous chromosomes held together by chiasmata biorient. Since the first meiotic division is unaffected by rare achiasmatic chromosomes or misaligned bivalents [2-7], it is thought that several tensionless kinetochores are required to produce sufficient MCC for APC/C inhibition. Consistent with this, univalents lacking chiasmata elicit a SAC-mediated arrest in Mlh1(-/-) oocytes. In contrast, chromatids generated by TEV protease-induced cohesin cleavage in Rec8(TEV/TEV) oocytes merely delay APC/C activation. Since the arrest of Mlh1(-/-)Rec8(TEV/TEV) oocytes is alleviated by TEV protease, even when targeted to kinetochores, we conclude that their SAC depends on cohesin as well as dedicated kinetochore proteins. This has important implications for aging oocytes [8, 9], where cohesin deterioration will induce sister kinetochore biorientation and compromise MCC production, leading to chromosome missegregation and aneuploid fetuses.</p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24291092">24291092</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2013.10.052">10.1016/j.cub.2013.10.052</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A rare functional cardioprotective APOC3 variant has risen in frequency in distinct population isolates.</h4>
<p class="authors"><span class="author">Tachmazidou I</span>, <span class="author">Dedoussis G</span>, <span class="author">Southam L</span>, <span class="author">Farmaki AE</span>, <span class="author">Ritchie GR</span>, <span class="author">Xifara DK</span>, <span class="author">Matchan A</span>, <span class="author">Hatzikotoulas K</span>, <span class="author">Rayner NW</span>, <span class="author">Chen Y</span>, <span class="author">Pollin TI</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Yerges-Armstrong LM</span>, <span class="author">Kiagiadaki C</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Schwartzentruber J</span>, <span class="author">Moutsianas L</span>, <span class="author">UK10K consortium</span>, <span class="author">Tsafantakis E</span>, <span class="author">Tyler-Smith C</span>, <span class="author">McVean G</span>, <span class="author">Xue Y</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Isolated populations can empower the identification of rare variation associated with complex traits through next generation association studies, but the generalizability of such findings remains unknown. Here we genotype 1,267 individuals from a Greek population isolate on the Illumina HumanExome Beadchip, in search of functional coding variants associated with lipids traits. We find genome-wide significant evidence for association between R19X, a functional variant in APOC3, with increased high-density lipoprotein and decreased triglycerides levels. Approximately 3.8% of individuals are heterozygous for this cardioprotective variant, which was previously thought to be private to the Amish founder population. R19X is rare (&lt;0.05% frequency) in outbred European populations. The increased frequency of R19X enables discovery of this lipid traits signal at genome-wide significance in a small sample size. This work exemplifies the value of isolated populations in successfully detecting transferable rare variant associations of high medical relevance.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">K01 HL116770</span>, <span class="grant">R01 HL104193</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL105198</span>; NIDDK NIH HHS: <span class="grant">P30 DK072488</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">098051</span>, <span class="grant">WT091310</span></p>
<p class="publication"><span class="title">Nature communications</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="pagerange">2872</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24343240">24343240</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3905724">3905724</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncomms3872">10.1038/ncomms3872</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The ancestor of extant Japanese fancy mice contributed to the mosaic genomes of classical inbred strains.</h4>
<p class="authors"><span class="author">Takada T</span>, <span class="author">Ebata T</span>, <span class="author">Noguchi H</span>, <span class="author">Keane TM</span>, <span class="author">Adams DJ</span>, <span class="author">Narita T</span>, <span class="author">Shin-I T</span>, <span class="author">Fujisawa H</span>, <span class="author">Toyoda A</span>, <span class="author">Abe K</span>, <span class="author">Obata Y</span>, <span class="author">Sakaki Y</span>, <span class="author">Moriwaki K</span>, <span class="author">Fujiyama A</span>, <span class="author">Kohara Y</span> and <span class="author">Shiroishi T</span></p>
<p class="affiliation">Mammalian Genetics Laboratory, National Institute of Genetics, Mishima, Shizuoka 411-8540, Japan;</p>
<p class="abstract">Commonly used classical inbred mouse strains have mosaic genomes with sequences from different subspecific origins. Their genomes are derived predominantly from the Western European subspecies Mus musculus domesticus, with the remaining sequences derived mostly from the Japanese subspecies Mus musculus molossinus. However, it remains unknown how this intersubspecific genome introgression occurred during the establishment of classical inbred strains. In this study, we resequenced the genomes of two M. m. molossinus-derived inbred strains, MSM/Ms and JF1/Ms. MSM/Ms originated from Japanese wild mice, and the ancestry of JF1/Ms was originally found in Europe and then transferred to Japan. We compared the characteristics of these sequences to those of the C57BL/6J reference sequence and the recent data sets from the resequencing of 17 inbred strains in the Mouse Genome Project (MGP), and the results unequivocally show that genome introgression from M. m. molossinus into M. m. domesticus provided the primary framework for the mosaic genomes of classical inbred strains. Furthermore, the genomes of C57BL/6J and other classical inbred strains have long consecutive segments with extremely high similarity (>99.998%) to the JF1/Ms strain. In the early 20th century, Japanese waltzing mice with a morphological phenotype resembling that of JF1/Ms mice were often crossed with European fancy mice for early studies of "Mendelism," which suggests that the ancestor of the extant JF1/Ms strain provided the origin of the M. m. molossinus genome in classical inbred strains and largely contributed to its intersubspecific genome diversity.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">8</span>;<span class="pagerange">1329-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23604024">23604024</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3730106">3730106</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.156497.113">10.1101/gr.156497.113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide meta-analysis of observational studies shows common genetic variants associated with macronutrient intake.</h4>
<p class="authors"><span class="author">Tanaka T</span>, <span class="author">Ngwa JS</span>, <span class="author">van Rooij FJ</span>, <span class="author">Zillikens MC</span>, <span class="author">Wojczynski MK</span>, <span class="author">Frazier-Wood AC</span>, <span class="author">Houston DK</span>, <span class="author">Kanoni S</span>, <span class="author">Lemaitre RN</span>, <span class="author">Luan J</span>, <span class="author">Mikkil&auml; V</span>, <span class="author">Renstrom F</span>, <span class="author">Sonestedt E</span>, <span class="author">Zhao JH</span>, <span class="author">Chu AY</span>, <span class="author">Qi L</span>, <span class="author">Chasman DI</span>, <span class="author">de Oliveira Otto MC</span>, <span class="author">Dhurandhar EJ</span>, <span class="author">Feitosa MF</span>, <span class="author">Johansson I</span>, <span class="author">Khaw KT</span>, <span class="author">Lohman KK</span>, <span class="author">Manichaikul A</span>, <span class="author">McKeown NM</span>, <span class="author">Mozaffarian D</span>, <span class="author">Singleton A</span>, <span class="author">Stirrups K</span>, <span class="author">Viikari J</span>, <span class="author">Ye Z</span>, <span class="author">Bandinelli S</span>, <span class="author">Barroso I</span>, <span class="author">Deloukas P</span>, <span class="author">Forouhi NG</span>, <span class="author">Hofman A</span>, <span class="author">Liu Y</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">North KE</span>, <span class="author">Dimitriou M</span>, <span class="author">Hallmans G</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Langenberg C</span>, <span class="author">Ordovas JM</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hu FB</span>, <span class="author">Kalafati IP</span>, <span class="author">Raitakari O</span>, <span class="author">Franco OH</span>, <span class="author">Johnson A</span>, <span class="author">Emilsson V</span>, <span class="author">Schrack JA</span>, <span class="author">Semba RD</span>, <span class="author">Siscovick DS</span>, <span class="author">Arnett DK</span>, <span class="author">Borecki IB</span>, <span class="author">Franks PW</span>, <span class="author">Kritchevsky SB</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Loos RJ</span>, <span class="author">Orho-Melander M</span>, <span class="author">Rotter JI</span>, <span class="author">Wareham NJ</span>, <span class="author">Witteman JC</span>, <span class="author">Ferrucci L</span>, <span class="author">Dedoussis G</span>, <span class="author">Cupples LA</span> and <span class="author">Nettleton JA</span></p>
<p class="affiliation">Translational Gerontology Branch, National Institute on Aging, Baltimore, MD 21225, USA. <a href="mailto:%74%61%6E%61%6B%61%74%6F%40%6D%61%69%6C.%6E%69%68.%67%6F%76">&#116;&#97;&#110;&#97;&#107;&#97;&#116;&#111;&#64;&#109;&#97;&#105;&#108;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract"><strong>Background</strong>: Macronutrient intake varies substantially between individuals, and there is evidence that this variation is partly accounted for by genetic variants.</p><p class="abstract"><strong>Objective</strong>: The objective of the study was to identify common genetic variants that are associated with macronutrient intake.</p><p class="abstract"><strong>Design</strong>: We performed 2-stage genome-wide association (GWA) meta-analysis of macronutrient intake in populations of European descent. Macronutrients were assessed by using food-frequency questionnaires and analyzed as percentages of total energy consumption from total fat, protein, and carbohydrate. From the discovery GWA (n = 38,360), 35 independent loci associated with macronutrient intake at P &lt; 5 &times; 10(-6) were identified and taken forward to replication in 3 additional cohorts (n = 33,533) from the DietGen Consortium. For one locus, fat mass obesity-associated protein (FTO), cohorts with Illumina MetaboChip genotype data (n = 7724) provided additional replication data.</p><p class="abstract"><strong>Results</strong>: A variant in the chromosome 19 locus (rs838145) was associated with higher carbohydrate (&beta; &plusmn; SE: 0.25 &plusmn; 0.04%; P = 1.68 &times; 10(-8)) and lower fat (&beta; &plusmn; SE: -0.21 &plusmn; 0.04%; P = 1.57 &times; 10(-9)) consumption. A candidate gene in this region, fibroblast growth factor 21 (FGF21), encodes a fibroblast growth factor involved in glucose and lipid metabolism. The variants in this locus were associated with circulating FGF21 protein concentrations (P &lt; 0.05) but not mRNA concentrations in blood or brain. The body mass index (BMI)-increasing allele of the FTO variant (rs1421085) was associated with higher protein intake (&beta; &plusmn; SE: 0.10 &plusmn; 0.02%; P = 9.96 &times; 10(-10)), independent of BMI (after adjustment for BMI, &beta; &plusmn; SE: 0.08 &plusmn; 0.02%; P = 3.15 &times; 10(-7)).</p><p class="abstract"><strong>Conclusion</strong>: Our results indicate that variants in genes involved in nutrient metabolism and obesity are associated with macronutrient consumption in humans. Trials related to this study were registered at clinicaltrials.gov as NCT00005131 (Atherosclerosis Risk in Communities), NCT00005133 (Cardiovascular Health Study), NCT00005136 (Family Heart Study), NCT00005121 (Framingham Heart Study), NCT00083369 (Genetic and Environmental Determinants of Triglycerides), NCT01331512 (InCHIANTI Study), and NCT00005487 (Multi-Ethnic Study of Atherosclerosis).</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Cancer Research UK: <span class="grant">14136</span>; Medical Research Council: <span class="grant">G1000143</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_UP_A100_1003</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NHLBI NIH HHS: <span class="grant">R01 HL105756</span>, <span class="grant">T32 HL007575</span>; NIDDK NIH HHS: <span class="grant">P30 DK063491</span>, <span class="grant">R01 DK091718</span>; NIGMS NIH HHS: <span class="grant">T32 GM074905</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The American journal of clinical nutrition</span>&nbsp;<span class="year">2013</span>;<span class="volume">97</span>;<span class="number">6</span>;<span class="pagerange">1395-402</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23636237">23636237</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3652928">3652928</a>; DOI: <a href="http://dx.doi.org/10.3945%2Fajcn.112.052183">10.3945/ajcn.112.052183</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Frequent mutation of the major cartilage collagen gene COL2A1 in chondrosarcoma.</h4>
<p class="authors"><span class="author">Tarpey PS</span>, <span class="author">Behjati S</span>, <span class="author">Cooke SL</span>, <span class="author">Van Loo P</span>, <span class="author">Wedge DC</span>, <span class="author">Pillay N</span>, <span class="author">Marshall J</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Davies H</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Beare D</span>, <span class="author">Butler A</span>, <span class="author">Gamble J</span>, <span class="author">Hardy C</span>, <span class="author">Hinton J</span>, <span class="author">Jia MM</span>, <span class="author">Jayakumar A</span>, <span class="author">Jones D</span>, <span class="author">Latimer C</span>, <span class="author">Maddison M</span>, <span class="author">Martin S</span>, <span class="author">McLaren S</span>, <span class="author">Menzies A</span>, <span class="author">Mudie L</span>, <span class="author">Raine K</span>, <span class="author">Teague JW</span>, <span class="author">Tubio JM</span>, <span class="author">Halai D</span>, <span class="author">Tirabosco R</span>, <span class="author">Amary F</span>, <span class="author">Campbell PJ</span>, <span class="author">Stratton MR</span>, <span class="author">Flanagan AM</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Chondrosarcoma is a heterogeneous collection of malignant bone tumors and is the second most common primary malignancy of bone after osteosarcoma. Recent work has identified frequent, recurrent mutations in IDH1 or IDH2 in nearly half of central chondrosarcomas. However, there has been little systematic genomic analysis of this tumor type, and, thus, the contribution of other genes is unclear. Here we report comprehensive genomic analyses of 49 individuals with chondrosarcoma (cases). We identified hypermutability of the major cartilage collagen gene COL2A1, with insertions, deletions and rearrangements identified in 37% of cases. The patterns of mutation were consistent with selection for variants likely to impair normal collagen biosynthesis. In addition, we identified mutations in IDH1 or IDH2 (59%), TP53 (20%), the RB1 pathway (33%) and Hedgehog signaling (18%).</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">093867</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">8</span>;<span class="pagerange">923-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23770606">23770606</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3743157">3743157</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2668">10.1038/ng.2668</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA deaminases induce break-associated mutation showers with implication of APOBEC3B and 3A in breast cancer kataegis.</h4>
<p class="authors"><span class="author">Taylor BJ</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Wu YL</span>, <span class="author">Stebbings LA</span>, <span class="author">Raine K</span>, <span class="author">Campbell PJ</span>, <span class="author">Rada C</span>, <span class="author">Stratton MR</span> and <span class="author">Neuberger MS</span></p>
<p class="affiliation">Protein and Nucleic Acid Chemistry Division , Medical Research Council Laboratory of Molecular Biology , Cambridge , United Kingdom.</p>
<p class="abstract">Breast cancer genomes have revealed a novel form of mutation showers (kataegis) in which multiple same-strand substitutions at C:G pairs spaced one to several hundred nucleotides apart are clustered over kilobase-sized regions, often associated with sites of DNA rearrangement. We show kataegis can result from AID/APOBEC-catalysed cytidine deamination in the vicinity of DNA breaks, likely through action on single-stranded DNA exposed during resection. Cancer-like kataegis can be recapitulated by expression of AID/APOBEC family deaminases in yeast where it largely depends on uracil excision, which generates an abasic site for strand breakage. Localized kataegis can also be nucleated by an I-SceI-induced break. Genome-wide patterns of APOBEC3-catalyzed deamination in yeast reveal APOBEC3B and 3A as the deaminases whose mutational signatures are most similar to those of breast cancer kataegic mutations. Together with expression and functional assays, the results implicate APOBEC3B/A in breast cancer hypermutation and give insight into the mechanism of kataegis. DOI:http://dx.doi.org/10.7554/eLife.00534.001.</p>
<p class="publication"><span class="title">eLife</span>&nbsp;<span class="year">2013</span>;<span class="volume">2</span>;<span class="pagerange">e00534</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23599896">23599896</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3628087">3628087</a>; DOI: <a href="http://dx.doi.org/10.7554%2FeLife.00534">10.7554/eLife.00534</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A common biological basis of obesity and nicotine addiction.</h4>
<p class="authors"><span class="author">Thorgeirsson TE</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Sulem P</span>, <span class="author">Besenbacher S</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Thorleifsson G</span>, <span class="author">Walters GB</span>, <span class="author">TAG Consortium</span>, <span class="author">Oxford-GSK Consortium</span>, <span class="author">ENGAGE consortium</span>, <span class="author">Furberg H</span>, <span class="author">Sullivan PF</span>, <span class="author">Marchini J</span>, <span class="author">McCarthy MI</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Thorsteinsdottir U</span> and <span class="author">Stefansson K</span></p>
<p class="affiliation">Decode genetics/AMGEN, Sturlugata 8, Reykjavik, Iceland.</p>
<p class="abstract">Smoking influences body weight such that smokers weigh less than non-smokers and smoking cessation often leads to weight increase. The relationship between body weight and smoking is partly explained by the effect of nicotine on appetite and metabolism. However, the brain reward system is involved in the control of the intake of both food and tobacco. We evaluated the effect of single-nucleotide polymorphisms (SNPs) affecting body mass index (BMI) on smoking behavior, and tested the 32 SNPs identified in a meta-analysis for association with two smoking phenotypes, smoking initiation (SI) and the number of cigarettes smoked per day (CPD) in an Icelandic sample (N=34 216 smokers). Combined according to their effect on BMI, the SNPs correlate with both SI (r=0.019, P=0.00054) and CPD (r=0.032, P=8.0 × 10(-7)). These findings replicate in a second large data set (N=127 274, thereof 76 242 smokers) for both SI (P=1.2 × 10(-5)) and CPD (P=9.3 × 10(-5)). Notably, the variant most strongly associated with BMI (rs1558902-A in FTO) did not associate with smoking behavior. The association with smoking behavior is not due to the effect of the SNPs on BMI. Our results strongly point to a common biological basis of the regulation of our appetite for tobacco and food, and thus the vulnerability to nicotine addiction and obesity.</p>
<p class="publication"><span class="title">Translational psychiatry</span>&nbsp;<span class="year">2013</span>;<span class="volume">3</span>;<span class="pagerange">e308</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24084939">24084939</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3818010">3818010</a>; DOI: <a href="http://dx.doi.org/10.1038%2Ftp.2013.81">10.1038/tp.2013.81</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic risk prediction and a 2-stage risk screening strategy for coronary heart disease.</h4>
<p class="authors"><span class="author">Tikkanen E</span>, <span class="author">Havulinna AS</span>, <span class="author">Palotie A</span>, <span class="author">Salomaa V</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Objective</strong>: Genome-wide association studies have identified several genetic variants associated with coronary heart disease (CHD). The aim of this study was to evaluate the genetic risk discrimination and reclassification and apply the results for a 2-stage population risk screening strategy for CHD.</p><p class="abstract">We genotyped 28 genetic variants in 24 124 participants in 4 Finnish population-based, prospective cohorts (recruitment years 1992-2002). We constructed a multilocus genetic risk score and evaluated its association with incident cardiovascular disease events. During the median follow-up time of 12 years (interquartile range 8.75-15.25 years), we observed 1093 CHD, 1552 cardiovascular disease, and 731 acute coronary syndrome events. Adding genetic information to conventional risk factors and family history improved risk discrimination of CHD (C-index 0.856 versus 0.851; P=0.0002) and other end points (cardiovascular disease: C-index 0.840 versus 0.837, P=0.0004; acute coronary syndrome: C-index 0.859 versus 0.855, P=0.001). In a standard population of 100 000 individuals, additional genetic screening of subjects at intermediate risk for CHD would reclassify 2144 subjects (12%) into high-risk category. Statin allocation for these subjects is estimated to prevent 135 CHD cases over 14 years. Similar results were obtained by external validation, where the effects were estimated from a training data set and applied for a test data set.</p><p class="abstract"><strong>Conclusions</strong>: Genetic risk score improves risk prediction of CHD and helps to identify individuals at high risk for the first CHD event. Genetic screening for individuals at intermediate cardiovascular risk could help to prevent future cases through better targeting of statins.</p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">33</span>;<span class="number">9</span>;<span class="pagerange">2261-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23599444">23599444</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4210840">4210840</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.112.301120">10.1161/ATVBAHA.112.301120</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Haploid Genetic Screens Identify an Essential Role for PLP2 in the Downregulation of Novel Plasma Membrane Targets by Viral E3 Ubiquitin Ligases.</h4>
<p class="authors"><span class="author">Timms RT</span>, <span class="author">Duncan LM</span>, <span class="author">Tchasovnikarova IA</span>, <span class="author">Antrobus R</span>, <span class="author">Smith DL</span>, <span class="author">Dougan G</span>, <span class="author">Weekes MP</span> and <span class="author">Lehner PJ</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Addenbrooke's Hospital, Cambridge, United Kingdom.</p>
<p class="abstract">The Kaposi's sarcoma-associated herpesvirus gene products K3 and K5 are viral ubiquitin E3 ligases which downregulate MHC-I and additional cell surface immunoreceptors. To identify novel cellular genes required for K5 function we performed a forward genetic screen in near-haploid human KBM7 cells. The screen identified proteolipid protein 2 (PLP2), a MARVEL domain protein of unknown function, as essential for K5 activity. Genetic loss of PLP2 traps the viral ligase in the endoplasmic reticulum, where it is unable to ubiquitinate and degrade its substrates. Subsequent analysis of the plasma membrane proteome of K5-expressing KBM7 cells in the presence and absence of PLP2 revealed a wide range of novel K5 targets, all of which required PLP2 for their K5-mediated downregulation. This work ascribes a critical function to PLP2 for viral ligase activity and underlines the power of non-lethal haploid genetic screens in human cells to identify the genes involved in pathogen manipulation of the host immune system.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">11</span>;<span class="pagerange">e1003772</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24278019">24278019</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836740">3836740</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003772">10.1371/journal.ppat.1003772</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Community-associated MRSA from the Indian subcontinent.</h4>
<p class="authors"><span class="author">Tong SY</span> and <span class="author">Kearns AM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK; Menzies School of Health Research, Darwin, NT, Australia. Electronic address: <a href="mailto:%73%74%65%76%65%6E.%74%6F%6E%67%40%6D%65%6E%7A%69%65%73.%65%64%75.%61%75">&#115;&#116;&#101;&#118;&#101;&#110;&#46;&#116;&#111;&#110;&#103;&#64;&#109;&#101;&#110;&#122;&#105;&#101;&#115;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a>.</p>
<p class="publication"><span class="title">The Lancet infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="number">9</span>;<span class="pagerange">734-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23969208">23969208</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS1473-3099%2813%2970231-7">10.1016/S1473-3099(13)70231-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Extensive Diversity of Streptococcus pyogenes in a Remote Human Population Reflects Global-Scale Transmission Rather than Localised Diversification.</h4>
<p class="authors"><span class="author">Towers RJ</span>, <span class="author">Carapetis JR</span>, <span class="author">Currie BJ</span>, <span class="author">Davies MR</span>, <span class="author">Walker MJ</span>, <span class="author">Dougan G</span> and <span class="author">Giffard PM</span></p>
<p class="affiliation">Menzies School of Health Research, Division of Global and Tropical Health, Casuarina, Northern Territory, Australia.</p>
<p class="abstract">The Indigenous population of the Northern Territory of Australia (NT) suffers from a very high burden of Streptococcus pyogenes disease, including cardiac and renal sequelae. The aim of this study was to determine if S. pyogenes isolated from this population represent NT endemic strains, or conversely reflect strains with global distribution. emm sequence typing data were used to select 460 S. pyogenes isolates representing NT S. pyogenes diversity from 1987-2008. These isolates were genotyped using either multilocus sequence typing (MLST) or a high resolution melting-based MLST surrogate (Minim typing). These data were combined with MLST data from other studies on NT S. pyogenes to yield a set of 731 MLST or Minim typed isolates for analysis. goeBURST analysis of MLST allelic profiles and neighbour-joining trees of the MLST allele sequences revealed that a large proportion of the known global S. pyogenes MLST-defined diversity has now been found in the NT. Specifically, fully sequence typed NT isolates encompass 19% of known S. pyogenes STs and 43% of known S. pyogenes MLST alleles. These analyses provided no evidence for major NT-endemic strains, with many STs and MLST alleles shared between the NT and the rest of the world. The relationship between the number of known Minim types, and the probability that a Minim type identified in a calendar year would be novel was determined. This revealed that Minim types typically persist in the NT for >1 year, and indicate that the majority of NT Minim types have been identified. This study revealed that many diverse S. pyogenes strains exhibit global scale mobility that extends to isolated populations. The burden of S. pyogenes disease in the NT is unlikely to be due to the nature of NT S. pyogenes strains, but is rather a function of social and living conditions.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">9</span>;<span class="pagerange">e73851</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24066079">24066079</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0073851">10.1371/journal.pone.0073851</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genomes of four tapeworm species reveal adaptations to parasitism.</h4>
<p class="authors"><span class="author">Tsai IJ</span>, <span class="author">Zarowiecki M</span>, <span class="author">Holroyd N</span>, <span class="author">Garciarrubio A</span>, <span class="author">Sanchez-Flores A</span>, <span class="author">Brooks KL</span>, <span class="author">Tracey A</span>, <span class="author">Bobes RJ</span>, <span class="author">Fragoso G</span>, <span class="author">Sciutto E</span>, <span class="author">Aslett M</span>, <span class="author">Beasley H</span>, <span class="author">Bennett HM</span>, <span class="author">Cai J</span>, <span class="author">Camicia F</span>, <span class="author">Clark R</span>, <span class="author">Cucher M</span>, <span class="author">De Silva N</span>, <span class="author">Day TA</span>, <span class="author">Deplazes P</span>, <span class="author">Estrada K</span>, <span class="author">Fern&aacute;ndez C</span>, <span class="author">Holland PW</span>, <span class="author">Hou J</span>, <span class="author">Hu S</span>, <span class="author">Huckvale T</span>, <span class="author">Hung SS</span>, <span class="author">Kamenetzky L</span>, <span class="author">Keane JA</span>, <span class="author">Kiss F</span>, <span class="author">Koziol U</span>, <span class="author">Lambert O</span>, <span class="author">Liu K</span>, <span class="author">Luo X</span>, <span class="author">Luo Y</span>, <span class="author">Macchiaroli N</span>, <span class="author">Nichol S</span>, <span class="author">Paps J</span>, <span class="author">Parkinson J</span>, <span class="author">Pouchkina-Stantcheva N</span>, <span class="author">Riddiford N</span>, <span class="author">Rosenzvit M</span>, <span class="author">Salinas G</span>, <span class="author">Wasmuth JD</span>, <span class="author">Zamanian M</span>, <span class="author">Zheng Y</span>, <span class="author">Taenia solium Genome Consortium</span>, <span class="author">Cai X</span>, <span class="author">Sober&oacute;n X</span>, <span class="author">Olson PD</span>, <span class="author">Laclette JP</span>, <span class="author">Brehm K</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Tapeworms (Cestoda) cause neglected diseases that can be fatal and are difficult to treat, owing to inefficient drugs. Here we present an analysis of tapeworm genome sequences using the human-infective species Echinococcus multilocularis, E. granulosus, Taenia solium and the laboratory model Hymenolepis microstoma as examples. The 115- to 141-megabase genomes offer insights into the evolution of parasitism. Synteny is maintained with distantly related blood flukes but we find extreme losses of genes and pathways that are ubiquitous in other animals, including 34 homeobox families and several determinants of stem cell fate. Tapeworms have specialized detoxification pathways, metabolism that is finely tuned to rely on nutrients scavenged from their hosts, and species-specific expansions of non-canonical heat shock proteins and families of known antigens. We identify new potential drug targets, including some on which existing pharmaceuticals may act. The genomes provide a rich resource to underpin the development of urgently needed treatments and control.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBG0038151</span>; Canadian Institutes of Health Research: <span class="grant">MOP#84556</span>; FIC NIH HHS: <span class="grant">TW008588</span>; Wellcome Trust: <span class="grant">085775</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2013</span>;<span class="volume">496</span>;<span class="number">7443</span>;<span class="pagerange">57-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23485966">23485966</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3964345">3964345</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature12031">10.1038/nature12031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular Analysis of an Outbreak of Lethal Postpartum Sepsis Caused by Streptococcus pyogenes.</h4>
<p class="authors"><span class="author">Turner CE</span>, <span class="author">Dryden M</span>, <span class="author">Holden MT</span>, <span class="author">Davies FJ</span>, <span class="author">Lawrenson RA</span>, <span class="author">Farzaneh L</span>, <span class="author">Bentley SD</span>, <span class="author">Efstratiou A</span> and <span class="author">Sriskandan S</span></p>
<p class="affiliation">Infectious Diseases &amp; Immunity, Imperial College London, Hammersmith Hospital, London, United Kingdom.</p><p class="abstract">Sepsis is now the leading direct cause of maternal death in the United Kingdom, and Streptococcus pyogenes is the leading pathogen. We combined conventional and genomic analyses to define the duration and scale of a lethal outbreak. Two postpartum deaths caused by S. pyogenes occurred within 24 h; one was characterized by bacteremia and shock and the other by hemorrhagic pneumonia. The women gave birth within minutes of each other in the same maternity unit 2 days earlier. Seven additional infections in health care and household contacts were subsequently detected and treated. All cluster-associated S. pyogenes isolates were genotype emm1 and were initially indistinguishable from other United Kingdom emm1 isolates. Sequencing of the virulence gene sic revealed that all outbreak isolates had the same unique sic type. Genome sequencing confirmed that the cluster was caused by a unique S. pyogenes clone. Transmission between patients occurred on a single day and was associated with casual contact only. A single isolate from one patient demonstrated a sequence change in sic consistent with longer infection duration. Transmission to health care workers was traced to single clinical contacts with index cases. The last case was detected 18 days after the first case. Following enhanced surveillance, the outbreak isolate was not detected again. Mutations in bacterial regulatory genes played no detectable role in this outbreak, illustrating the intrinsic ability of emm1 S. pyogenes to spread while retaining virulence. This fast-moving outbreak highlights the potential of S. pyogenes to cause a range of diseases in the puerperium with rapid transmission, underlining the importance of immediate recognition and response by clinical infection and occupational health teams.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">7</span>;<span class="pagerange">2089-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23616448">23616448</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3697669">3697669</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.00679-13">10.1128/JCM.00679-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Public Health Value of Next-Generation DNA Sequencing of Enterohemorrhagic Escherichia coli Isolates from an Outbreak.</h4>
<p class="authors"><span class="author">Underwood AP</span>, <span class="author">Dallman T</span>, <span class="author">Thomson NR</span>, <span class="author">Williams M</span>, <span class="author">Harker K</span>, <span class="author">Perry N</span>, <span class="author">Adak B</span>, <span class="author">Willshaw G</span>, <span class="author">Cheasty T</span>, <span class="author">Green J</span>, <span class="author">Dougan G</span>, <span class="author">Parkhill J</span> and <span class="author">Wain J</span></p>
<p class="affiliation">Health Protection Agency, London, United Kingdom.</p>
<p class="abstract">In 2009, an outbreak of enterohemorrhagic Escherichia coli (EHEC) on an open farm infected 93 persons, and approximately 22% of these individuals developed hemolytic-uremic syndrome (HUS). Genome sequencing was used to investigate outbreak-derived animal and human EHEC isolates. Phylogeny based on the whole-genome sequence was used to place outbreak isolates in the context of the overall E. coli species and the O157:H7 sequence type 11 (ST11) subgroup. Four informative single nucleotide polymorphisms (SNPs) were identified and used to design an assay to type 122 other outbreak isolates. The SNP phylogeny demonstrated that the outbreak strain was from a lineage distinct from previously reported O157:H7 ST11 EHEC and was not a member of the hypervirulent clade 8. The strain harbored determinants for two Stx2 verotoxins and other putative virulence factors. When linked to the epidemiological information, the sequence data indicate that gross contamination of a single outbreak strain occurred across the farm prior to the first clinical report of HUS. The most likely explanation for these results is that a single successful strain of EHEC spread from a single introduction through the farm by clonal expansion and that contamination of the environment (including the possible colonization of several animals) led ultimately to human cases.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">51</span>;<span class="number">1</span>;<span class="pagerange">232-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23135946" rel="external">23135946</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.01696-12" rel="external">10.1128/JCM.01696-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Histone methyltransferase MLL3 contributes to genome-scale circadian transcription.</h4>
<p class="authors"><span class="author">Valekunja UK</span>, <span class="author">Edgar RS</span>, <span class="author">Oklejewicz M</span>, <span class="author">van der Horst GT</span>, <span class="author">O&#39;Neill JS</span>, <span class="author">Tamanini F</span>, <span class="author">Turner DJ</span> and <span class="author">Reddy AB</span></p>
<p class="affiliation">Department of Clinical Neurosciences, University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, National Institute for Health Research Cambridge Biomedical Research Centre, Addenbrooke's Hospital, University of Cambridge, Cambridge CB2 0QQ, United Kingdom.</p>
<p class="abstract">Daily cyclical expression of thousands of genes in tissues such as the liver is orchestrated by the molecular circadian clock, the disruption of which is implicated in metabolic disorders and cancer. Although we understand much about the circadian transcription factors that can switch gene expression on and off, it is still unclear how global changes in rhythmic transcription are controlled at the genomic level. Here, we demonstrate circadian modification of an activating histone mark at a significant proportion of gene loci that undergo daily transcription, implicating widespread epigenetic modification as a key node regulated by the clockwork. Furthermore, we identify the histone-remodelling enzyme mixed lineage leukemia (MLL)3 as a clock-controlled factor that is able to directly and indirectly modulate over a hundred epigenetically targeted circadian "output" genes in the liver. Importantly, catalytic inactivation of the histone methyltransferase activity of MLL3 also severely compromises the oscillation of "core" clock gene promoters, including Bmal1, mCry1, mPer2, and Rev-erbÎ±, suggesting that rhythmic histone methylation is vital for robust transcriptional oscillator function. This highlights a pathway by which the clockwork exerts genome-wide control over transcription, which is critical for sustaining temporal programming of tissue physiology.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">4</span>;<span class="pagerange">1554-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23297224">23297224</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1214168110">10.1073/pnas.1214168110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic and molecular basis of drug resistance and species-specific drug action in schistosome parasites.</h4>
<p class="authors"><span class="author">Valentim CL</span>, <span class="author">Cioli D</span>, <span class="author">Chevalier FD</span>, <span class="author">Cao X</span>, <span class="author">Taylor AB</span>, <span class="author">Holloway SP</span>, <span class="author">Pica-Mattoccia L</span>, <span class="author">Guidi A</span>, <span class="author">Basso A</span>, <span class="author">Tsai IJ</span>, <span class="author">Berriman M</span>, <span class="author">Carvalho-Queiroz C</span>, <span class="author">Almeida M</span>, <span class="author">Aguilar H</span>, <span class="author">Frantz DE</span>, <span class="author">Hart PJ</span>, <span class="author">LoVerde PT</span> and <span class="author">Anderson TJ</span></p>
<p class="affiliation">Departments of Biochemistry and Pathology, University of Texas Health Science Center, San Antonio, TX 78229, USA.</p><p class="abstract">Oxamniquine resistance evolved in the human blood fluke (Schistosoma mansoni) in Brazil in the 1970s. We crossed parental parasites differing ~500-fold in drug response, determined drug sensitivity and marker segregation in clonally derived second-generation progeny, and identified a single quantitative trait locus (logarithm of odds = 31) on chromosome 6. A sulfotransferase was identified as the causative gene by using RNA interference knockdown and biochemical complementation assays, and we subsequently demonstrated independent origins of loss-of-function mutations in field-derived and laboratory-selected resistant parasites. These results demonstrate the utility of linkage mapping in a human helminth parasite, while crystallographic analyses of protein-drug interactions illuminate the mode of drug action and provide a framework for rational design of oxamniquine derivatives that kill both S. mansoni and S. haematobium, the two species responsible for &gt;99% of schistosomiasis cases worldwide.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">C06 RR013556</span>; NIAID NIH HHS: <span class="grant">5R21-AI072704</span>, <span class="grant">5R21-AI096277</span>, <span class="grant">R01 AI097576</span>, <span class="grant">R01-AI097576</span>, <span class="grant">R21 AI072704</span>, <span class="grant">R21 AI096277</span>; PHS HHS: <span class="grant">HHSN272201000005I</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">342</span>;<span class="number">6164</span>;<span class="pagerange">1385-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24263136">24263136</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1243106">10.1126/science.1243106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The miRNA Profile of Human Pancreatic Islets and Beta-Cells and Relationship to Type 2 Diabetes Pathogenesis.</h4>
<p class="authors"><span class="author">van de Bunt M</span>, <span class="author">Gaulton KJ</span>, <span class="author">Parts L</span>, <span class="author">Moran I</span>, <span class="author">Johnson PR</span>, <span class="author">Lindgren CM</span>, <span class="author">Ferrer J</span>, <span class="author">Gloyn AL</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">Oxford Centre for Diabetes, Endocrinology & Metabolism, University of Oxford, Oxford, United Kingdom ; Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">Recent advances in the understanding of the genetics of type 2 diabetes (T2D) susceptibility have focused attention on the regulation of transcriptional activity within the pancreatic beta-cell. MicroRNAs (miRNAs) represent an important component of regulatory control, and have proven roles in the development of human disease and control of glucose homeostasis. We set out to establish the miRNA profile of human pancreatic islets and of enriched beta-cell populations, and to explore their potential involvement in T2D susceptibility. We used Illumina small RNA sequencing to profile the miRNA fraction in three preparations each of primary human islets and of enriched beta-cells generated by fluorescence-activated cell sorting. In total, 366 miRNAs were found to be expressed (i.e. >100 cumulative reads) in islets and 346 in beta-cells; of the total of 384 unique miRNAs, 328 were shared. A comparison of the islet-cell miRNA profile with those of 15 other human tissues identified 40 miRNAs predominantly expressed (i.e. >50% of all reads seen across the tissues) in islets. Several highly-expressed islet miRNAs, such as miR-375, have established roles in the regulation of islet function, but others (e.g. miR-27b-3p, miR-192-5p) have not previously been described in the context of islet biology. As a first step towards exploring the role of islet-expressed miRNAs and their predicted mRNA targets in T2D pathogenesis, we looked at published T2D association signals across these sites. We found evidence that predicted mRNA targets of islet-expressed miRNAs were globally enriched for signals of T2D association (p-values <0.01, q-values <0.1). At six loci with genome-wide evidence for T2D association (AP3S2, KCNK16, NOTCH2, SCL30A8, VPS26A, and WFS1) predicted mRNA target sites for islet-expressed miRNAs overlapped potentially causal variants. In conclusion, we have described the miRNA profile of human islets and beta-cells and provide evidence linking islet miRNAs to T2D pathogenesis.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">e55272</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23372846">23372846</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3555946">3555946</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0055272">10.1371/journal.pone.0055272</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Preimplantation genetic diagnosis guided by single-cell genomics.</h4>
<p class="authors"><span class="author">Van der Aa N</span>, <span class="author">Esteki MZ</span>, <span class="author">Vermeesch JR</span> and <span class="author">Voet T</span></p>
<p class="affiliation">Laboratory of Reproductive Genomics, Department of Human Genetics, KU Leuven, Leuven 3000, Belgium. <a href="mailto:%54%68%69%65%72%72%79.%56%6F%65%74%40%6D%65%64.%6B%75%6C%65%75%76%65%6E.%62%65">&#84;&#104;&#105;&#101;&#114;&#114;&#121;&#46;&#86;&#111;&#101;&#116;&#64;&#109;&#101;&#100;&#46;&#107;&#117;&#108;&#101;&#117;&#118;&#101;&#110;&#46;&#98;&#101;</a>.</p>
<p class="abstract">Preimplantation genetic diagnosis (PGD) aims to help couples with heritable genetic disorders to avoid the birth of diseased offspring or the recurrence of loss of conception. Following in vitro fertilization, one or a few cells are biopsied from each human preimplantation embryo for genetic testing, allowing diagnosis and selection of healthy embryos for uterine transfer. Although classical methods, including single-cell PCR and fluorescent in situ hybridization, enable PGD for many genetic disorders, they have limitations. They often require family-specific designs and can be labor intensive, resulting in long waiting lists. Furthermore, certain types of genetic anomalies are not easy to diagnose using these classical approaches, and healthy offspring carrying the parental mutant allele(s) can result. Recently, state-of-the-art methods for single-cell genomics have flourished, which may overcome the limitations associated with classical PGD, and these underpin the development of generic assays for PGD that enable selection of embryos not only for the familial genetic disorder in question, but also for various other genetic aberrations and traits at once. Here, we discuss the latest single-cell genomics methodologies based on DNA microarrays, single-nucleotide polymorphism arrays or next-generation sequence analysis. We focus on their strengths, their validation status, their weaknesses and the challenges for implementing them in PGD.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2013</span>;<span class="volume">5</span>;<span class="number">8</span>;<span class="pagerange">71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23998893">23998893</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm475">10.1186/gm475</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Molecular Genetic Architecture of Self-Employment.</h4>
<p class="authors"><span class="author">van der Loos MJ</span>, <span class="author">Rietveld CA</span>, <span class="author">Eklund N</span>, <span class="author">Koellinger PD</span>, <span class="author">Rivadeneira F</span>, <span class="author">Abecasis GR</span>, <span class="author">Ankra-Badu GA</span>, <span class="author">Baumeister SE</span>, <span class="author">Benjamin DJ</span>, <span class="author">Biffar R</span>, <span class="author">Blankenberg S</span>, <span class="author">Boomsma DI</span>, <span class="author">Cesarini D</span>, <span class="author">Cucca F</span>, <span class="author">de Geus EJ</span>, <span class="author">Dedoussis G</span>, <span class="author">Deloukas P</span>, <span class="author">Dimitriou M</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Eriksson J</span>, <span class="author">Gieger C</span>, <span class="author">Gudnason V</span>, <span class="author">H&ouml;hne B</span>, <span class="author">Holle R</span>, <span class="author">Hottenga JJ</span>, <span class="author">Isaacs A</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Johannesson M</span>, <span class="author">Kaakinen M</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kanoni S</span>, <span class="author">Laaksonen MA</span>, <span class="author">Lahti J</span>, <span class="author">Launer LJ</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Loitfelder M</span>, <span class="author">Magnusson PK</span>, <span class="author">Naitza S</span>, <span class="author">Oostra BA</span>, <span class="author">Perola M</span>, <span class="author">Petrovic K</span>, <span class="author">Quaye L</span>, <span class="author">Raitakari O</span>, <span class="author">Ripatti S</span>, <span class="author">Scheet P</span>, <span class="author">Schlessinger D</span>, <span class="author">Schmidt CO</span>, <span class="author">Schmidt H</span>, <span class="author">Schmidt R</span>, <span class="author">Senft A</span>, <span class="author">Smith AV</span>, <span class="author">Spector TD</span>, <span class="author">Surakka I</span>, <span class="author">Svento R</span>, <span class="author">Terracciano A</span>, <span class="author">Tikkanen E</span>, <span class="author">van Duijn CM</span>, <span class="author">Viikari J</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wichmann HE</span>, <span class="author">Wild PS</span>, <span class="author">Willems SM</span>, <span class="author">Willemsen G</span>, <span class="author">van Rooij FJ</span>, <span class="author">Groenen PJ</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span> and <span class="author">Thurik AR</span></p>
<p class="affiliation">Department of Applied Economics, Erasmus School of Economics, Erasmus University Rotterdam, Rotterdam, The Netherlands ; Department of Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands.</p>
<p class="abstract">Economic variables such as income, education, and occupation are known to affect mortality and morbidity, such as cardiovascular disease, and have also been shown to be partly heritable. However, very little is known about which genes influence economic variables, although these genes may have both a direct and an indirect effect on health. We report results from the first large-scale collaboration that studies the molecular genetic architecture of an economic variable-entrepreneurship-that was operationalized using self-employment, a widely-available proxy. Our results suggest that common SNPs when considered jointly explain about half of the narrow-sense heritability of self-employment estimated in twin data (Ïƒg (2)/ÏƒP (2)â€Š=â€Š25%, h (2)â€Š=â€Š55%). However, a meta-analysis of genome-wide association studies across sixteen studies comprising 50,627 participants did not identify genome-wide significant SNPs. 58 SNPs with p<10(-5) were tested in a replication sample (nâ€Š=â€Š3,271), but none replicated. Furthermore, a gene-based test shows that none of the genes that were previously suggested in the literature to influence entrepreneurship reveal significant associations. Finally, SNP-based genetic scores that use results from the meta-analysis capture less than 0.2% of the variance in self-employment in an independent sample (pâ‰¥0.039). Our results are consistent with a highly polygenic molecular genetic architecture of self-employment, with many genetic variants of small effect. Although self-employment is a multi-faceted, heavily environmentally influenced, and biologically distal trait, our results are similar to those for other genetically complex and biologically more proximate outcomes, such as height, intelligence, personality, and several diseases.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e60542</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23593239">23593239</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0060542">10.1371/journal.pone.0060542</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer of mice and men: Old twists and new tails.</h4>
<p class="authors"><span class="author">van der Weyden L</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p>
<p class="abstract">In this review we set out to celebrate the contribution that mouse models of human cancer have made to our understanding of the fundamental mechanisms driving tumourigenesis. We take the opportunity to look forward to how the mouse will be used to model cancer and the tools and technologies that will be applied, and indulge in looking back at the key advances the mouse has made possible.</p>
<p class="publication"><span class="title">The Journal of pathology</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23436574">23436574</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpath.4184">10.1002/path.4184</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Jdp2 downregulates Trp53 transcription to promote leukaemogenesis in the context of Trp53 heterozygosity.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Rust AG</span>, <span class="author">McIntyre RE</span>, <span class="author">Robles-Espinoza CD</span>, <span class="author">del Castillo Velasco-Herrera M</span>, <span class="author">Strogantsev R</span>, <span class="author">Ferguson-Smith AC</span>, <span class="author">McCarthy S</span>, <span class="author">Keane TM</span>, <span class="author">Arends MJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">We performed a genetic screen in mice to identify candidate genes that are associated with leukaemogenesis in the context of Trp53 heterozygosity. To do this we generated Trp53 heterozygous mice carrying the T2/Onc transposon and SB11 transposase alleles to allow transposon-mediated insertional mutagenesis to occur. From the resulting leukaemias/lymphomas that developed in these mice, we identified nine loci that are potentially associated with tumour formation in the context of Trp53 heterozygosity, including AB041803 and the Jun dimerization protein 2 (Jdp2). We show that Jdp2 transcriptionally regulates the Trp53 promoter, via an atypical AP-1 site, and that Jdp2 expression negatively regulates Trp53 expression levels. This study is the first to identify a genetic mechanism for tumour formation in the context of Trp53 heterozygosity.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Wellcome Trust: <span class="grant">095606</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2013</span>;<span class="volume">32</span>;<span class="number">3</span>;<span class="pagerange">397-402</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22370638">22370638</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3550594">3550594</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fonc.2012.56">10.1038/onc.2012.56</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tumor exome analysis reveals neoantigen-specific T-cell reactivity in an ipilimumab-responsive melanoma.</h4>
<p class="authors"><span class="author">van Rooij N</span>, <span class="author">van Buuren MM</span>, <span class="author">Philips D</span>, <span class="author">Velds A</span>, <span class="author">Toebes M</span>, <span class="author">Heemskerk B</span>, <span class="author">van Dijk LJ</span>, <span class="author">Behjati S</span>, <span class="author">Hilkmann H</span>, <span class="author">El Atmioui D</span>, <span class="author">Nieuwland M</span>, <span class="author">Stratton MR</span>, <span class="author">Kerkhoven RM</span>, <span class="author">Kesmir C</span>, <span class="author">Haanen JB</span>, <span class="author">Kvistborg P</span> and <span class="author">Schumacher TN</span></p>
<p class="affiliation">Netherlands Cancer Institute, Amsterdam, the Netherlands.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2013</span>;<span class="volume">31</span>;<span class="number">32</span>;<span class="pagerange">e439-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24043743">24043743</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836220">3836220</a>; DOI: <a href="http://dx.doi.org/10.1200%2FJCO.2012.47.7521">10.1200/JCO.2012.47.7521</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insect Gut Carboxypeptidase 3</h4>
<p class="authors"><span class="author">Vendrell,J.;</span>, <span class="author">Avil&eacute;s,F.X.;</span> and <span class="author">BAY&Eacute;S,A.;</span></p>
<p class="publication"><span class="title">Handbook of Proteolytic Enzymes</span>&nbsp;<span class="year">2013</span>;<span class="volume">1</span>;<span class="number">3</span>;<span class="pagerange">1370-5</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-382219-2.00307-0">10.1016/B978-0-12-382219-2.00307-0</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780123822192003070" href="http://www.sciencedirect.com/science/article/pii/B9780123822192003070">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780123822192</span>003070</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Best practices in bioinformatics training for life scientists.</h4>
<p class="authors"><span class="author">Via A</span>, <span class="author">Blicher T</span>, <span class="author">Bongcam-Rudloff E</span>, <span class="author">Brazas MD</span>, <span class="author">Brooksbank C</span>, <span class="author">Budd A</span>, <span class="author">De Las Rivas J</span>, <span class="author">Dreyer J</span>, <span class="author">Fernandes PL</span>, <span class="author">van Gelder C</span>, <span class="author">Jacob J</span>, <span class="author">Jimenez RC</span>, <span class="author">Loveland J</span>, <span class="author">Moran F</span>, <span class="author">Mulder N</span>, <span class="author">Nyr&ouml;nen T</span>, <span class="author">Rother K</span>, <span class="author">Schneider MV</span> and <span class="author">Attwood TK</span></p>
<p class="affiliation">Department of Physics, Sapienza University, P.le Aldo Moro 5, 00185 Rome, Italy. <a href="mailto:%61%6C%6C%65%67%72%61.%76%69%61%40%75%6E%69%72%6F%6D%61%31.%69%74">&#97;&#108;&#108;&#101;&#103;&#114;&#97;&#46;&#118;&#105;&#97;&#64;&#117;&#110;&#105;&#114;&#111;&#109;&#97;&#49;&#46;&#105;&#116;</a>.</p>
<p class="abstract">The mountains of data thrusting from the new landscape of modern high-throughput biology are irrevocably changing biomedical research and creating a near-insatiable demand for training in data management and manipulation and data mining and analysis. Among life scientists, from clinicians to environmental researchers, a common theme is the need not just to use, and gain familiarity with, bioinformatics tools and resources but also to understand their underlying fundamental theoretical and practical concepts. Providing bioinformatics training to empower life scientists to handle and analyse their data efficiently, and progress their research, is a challenge across the globe. Delivering good training goes beyond traditional lectures and resource-centric demos, using interactivity, problem-solving exercises and cooperative learning to substantially enhance training quality and learning outcomes. In this context, this article discusses various pragmatic criteria for identifying training needs and learning objectives, for selecting suitable trainees and trainers, for developing and maintaining training skills and evaluating training quality. Adherence to these criteria may help not only to guide course organizers and trainers on the path towards bioinformatics training excellence but, importantly, also to improve the training experience for life scientists.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">528-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23803301">23803301</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3771230">3771230</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbib%2Fbbt043">10.1093/bib/bbt043</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beyond the Sympathetic Tone: The New Brown Fat Activators.</h4>
<p class="authors"><span class="author">Villarroya F</span> and <span class="author">Vidal-Puig A</span></p>
<p class="affiliation">Departament de Bioquimica i Biologia Molecular, Institute of Biomedicine (IBUB), University of Barcelona, and CIBER Fisiopatologia de la Obesidad y Nutrición, Av Diagonal 643, 08028 Barcelona, Catalonia, Spain. Electronic address: <a href="mailto:%66%76%69%6C%6C%61%72%72%6F%79%61%40%75%62.%65%64%75">&#102;&#118;&#105;&#108;&#108;&#97;&#114;&#114;&#111;&#121;&#97;&#64;&#117;&#98;&#46;&#101;&#100;&#117;</a>.</p>
<p class="abstract">If we could avoid the side effects associated with global sympathetic activation, activating brown adipose tissue to increase thermogenesis would be a safe way to lose weight. The discovery of adrenergic-independent brown fat activators opens the prospect of developing this alternative way to efficiently and safely induce negative energy balance.</p>
<p class="publication"><span class="title">Cell metabolism</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23583169">23583169</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cmet.2013.02.020">10.1016/j.cmet.2013.02.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression of Cellulosome Components and Type IV Pili within the Extracellular Proteome of Ruminococcus flavefaciens 007.</h4>
<p class="authors"><span class="author">Vodovnik M</span>, <span class="author">Duncan SH</span>, <span class="author">Reid MD</span>, <span class="author">Cantlay L</span>, <span class="author">Turner K</span>, <span class="author">Parkhill J</span>, <span class="author">Lamed R</span>, <span class="author">Yeoman CJ</span>, <span class="author">Miller ME</span>, <span class="author">White BA</span>, <span class="author">Bayer EA</span>, <span class="author">Marin&scaron;ek-Logar R</span> and <span class="author">Flint HJ</span></p>
<p class="affiliation">Chair for Microbiology and Microbial Biotechnology, Biotechnical Faculty, University of Ljubljana, Ljubljana, Slovenia ; Rowett Institute of Nutrition and Health, University of Aberdeen, Aberdeen, United Kingdom.</p>
<p class="abstract">Background: Ruminococcus flavefaciens is an important fibre-degrading bacterium found in the mammalian gut. Cellulolytic strains from the bovine rumen have been shown to produce complex cellulosome structures that are associated with the cell surface. R. flavefaciens 007 is a highly cellulolytic strain whose ability to degrade dewaxed cotton, but not Avicel cellulose, was lost following initial isolation in the variant 007S. The ability was recovered after serial subculture to give the cotton-degrading strain 007C. This has allowed us to investigate the factors required for degradation of this particularly recalcitrant form of cellulose. METHODOLOGYPRINCIPAL FINDINGS: The major proteins associated with the bacterial cell surface and with the culture supernatant were analyzed for R. flavefaciens 007S and 007C grown with cellobiose, xylan or Avicel cellulose as energy sources. Identification of the proteins was enabled by a draft genome sequence obtained for 007C. Among supernatant proteins a cellulosomal GH48 hydrolase, a rubrerthyrin-like protein and a protein with type IV pili N-terminal domain were the most strongly up-regulated in 007C cultures grown on Avicel compared with cellobiose. Strain 007S also showed substrate-related changes, but supernatant expression of the Pil protein and rubrerythrin in particular were markedly lower in 007S than in 007C during growth on Avicel. CONCLUSIONSSIGNIFICANCE: This study provides new information on the extracellular proteome of R. flavefaciens and its regulation in response to different growth substrates. Furthermore it suggests that the cotton cellulose non-degrading strain (007S) has altered regulation of multiple proteins that may be required for breakdown of cotton cellulose. One of these, the type IV pilus was previously shown to play a role in adhesion to cellulose in R. albus, and a related pilin protein was identified here for the first time as a major extracellular protein in R. flavefaciens.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">e65333</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23750253">23750253</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672088">3672088</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0065333">10.1371/journal.pone.0065333</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single-cell paired-end genome sequencing reveals structural variation per cell cycle.</h4>
<p class="authors"><span class="author">Voet T</span>, <span class="author">Kumar P</span>, <span class="author">Van Loo P</span>, <span class="author">Cooke SL</span>, <span class="author">Marshall J</span>, <span class="author">Lin ML</span>, <span class="author">Zamani Esteki M</span>, <span class="author">Van der Aa N</span>, <span class="author">Mateiu L</span>, <span class="author">McBride DJ</span>, <span class="author">Bignell GR</span>, <span class="author">McLaren S</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Raine K</span>, <span class="author">Stebbings LA</span>, <span class="author">Quail MA</span>, <span class="author">D&#39;Hooghe T</span>, <span class="author">Moreau Y</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Vermeesch JR</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Department of Human Genetics, KU Leuven, Leuven, 3000, Belgium. <a href="mailto:%54%68%69%65%72%72%79.%56%6F%65%74%40%6D%65%64.%6B%75%6C%65%75%76%65%6E.%62%65">&#84;&#104;&#105;&#101;&#114;&#114;&#121;&#46;&#86;&#111;&#101;&#116;&#64;&#109;&#101;&#100;&#46;&#107;&#117;&#108;&#101;&#117;&#118;&#101;&#110;&#46;&#98;&#101;</a></p><p class="abstract">The nature and pace of genome mutation is largely unknown. Because standard methods sequence DNA from populations of cells, the genetic composition of individual cells is lost, de novo mutations in cells are concealed within the bulk signal and per cell cycle mutation rates and mechanisms remain elusive. Although single-cell genome analyses could resolve these problems, such analyses are error-prone because of whole-genome amplification (WGA) artefacts and are limited in the types of DNA mutation that can be discerned. We developed methods for paired-end sequence analysis of single-cell WGA products that enable (i) detecting multiple classes of DNA mutation, (ii) distinguishing DNA copy number changes from allelic WGA-amplification artefacts by the discovery of matching aberrantly mapping read pairs among the surfeit of paired-end WGA and mapping artefacts and (iii) delineating the break points and architecture of structural variants. By applying the methods, we capture DNA copy number changes acquired over one cell cycle in breast cancer cells and in blastomeres derived from a human zygote after in vitro fertilization. Furthermore, we were able to discover and fine-map a heritable inter-chromosomal rearrangement t(1;16)(p36;p12) by sequencing a single blastomere. The methods will expedite applications in basic genome research and provide a stepping stone to novel approaches for clinical genetic diagnosis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span>;<span class="volume">41</span>;<span class="number">12</span>;<span class="pagerange">6119-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23630320">23630320</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3695511">3695511</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkt345">10.1093/nar/gkt345</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">UBE2QL1 is Disrupted by a Constitutional Translocation Associated with Renal Tumor Predisposition and is a Novel Candidate Renal Tumor Suppressor Gene.</h4>
<p class="authors"><span class="author">Wake NC</span>, <span class="author">Ricketts CJ</span>, <span class="author">Morris MR</span>, <span class="author">Prigmore E</span>, <span class="author">Gribble SM</span>, <span class="author">Skytte AB</span>, <span class="author">Brown M</span>, <span class="author">Clarke N</span>, <span class="author">Banks RE</span>, <span class="author">Hodgson S</span>, <span class="author">Turnell AS</span>, <span class="author">Maher ER</span> and <span class="author">Woodward ER</span></p>
<p class="affiliation">Centre for Rare Diseases and Personalised Medicine, School of Clinical and Experimental Medicine, College of Medical and Dental Sciences, University of Birmingham, Birmingham, UK.</p>
<p class="abstract">Investigation of rare familial forms of renal cell carcinoma (RCC) has led to the identification of genes such as VHL and MET that are also implicated in the pathogenesis of sporadic RCC. In order to identify a novel candidate renal tumor suppressor gene, we characterized the breakpoints of a constitutional balanced translocation, t(5;19)(p15.3;q12), associated with familial RCC and found that a previously uncharacterized gene UBE2QL1 was disrupted by the chromosome 5 breakpoint. UBE2QL1 mRNA expression was downregulated in 78.6% of sporadic RCC and, although no intragenic mutations were detected, gene deletions and promoter region hypermethylation were detected in 17.3% and 20.3%, respectively, of sporadic RCC. Reexpression of UBE2QL1 in a deficient RCC cell line suppressed anchorage-independent growth. UBE2QL1 shows homology to the E2 class of ubiquitin conjugating enzymes and we found that (1) UBE2QL1 possesses an active-site cysteine (C88) that is monoubiquitinated in vivo, and (2) UBE2QL1 interacts with FBXW7 (an F box protein providing substrate recognition to the SCF E3 ubiquitin ligase) and facilitates the degradation of the known FBXW7 targets, CCNE1 and mTOR. These findings suggest UBE2QL1 as a novel candidate renal tumor suppressor gene.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24000165">24000165</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.22433">10.1002/humu.22433</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deep-sea striving.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2013</span>;<span class="volume">11</span>;<span class="number">6</span>;<span class="pagerange">364</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23669892">23669892</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro3037">10.1038/nrmicro3037</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Therapeutic modulation of intestinal dysbiosis.</h4>
<p class="authors"><span class="author">Walker AW</span> and <span class="author">Lawley TD</span></p>
<p class="affiliation">Pathogen Genomics Group, Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%61%77%36%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#119;&#54;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The human gastrointestinal tract is home to an extremely numerous and diverse collection of microbes, collectively termed the &quot;intestinal microbiota&quot;. This microbiota is considered to play a number of key roles in the maintenance of host health, including aiding digestion of otherwise indigestible dietary compounds, synthesis of vitamins and other beneficial metabolites, immune system regulation and enhanced resistance against colonisation by pathogenic microorganisms. Conversely, the intestinal microbiota is also a potent source of antigens and potentially harmful compounds. In health, humans can therefore be considered to exist in a state of natural balance with their microbial inhabitants. A shift in the balance of microbiota composition such that it may become deleterious to host health is termed &quot;dysbiosis&quot;. Dysbiosis of the gut microbiota has been implicated in numerous disorders, ranging from intestinal maladies such as inflammatory bowel diseases and colorectal cancer to disorders with more systemic effects such as diabetes, metabolic syndrome and atopy. Given the far reaching influence of the intestinal microbiota on human health a clear future goal must be to develop reliable means to alter the composition of the microbiota and restore a healthy balance of microbial species. While it is clear that much fundamental research remains to be done, potentially important therapeutic options include narrow spectrum antibiotics, novel probiotics, dietary interventions and more radical techniques such as faecal transplantation, all of which aim to suppress clinical dysbiosis, restore intestinal microbiota diversity and improve host health.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0901743</span>; Wellcome Trust: <span class="grant">076964</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Pharmacological research : the official journal of the Italian Pharmacological Society</span>&nbsp;<span class="year">2013</span>;<span class="volume">69</span>;<span class="number">1</span>;<span class="pagerange">75-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23017673">23017673</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.phrs.2012.09.008">10.1016/j.phrs.2012.09.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microbiology. Fighting obesity with bacteria.</h4>
<p class="authors"><span class="author">Walker AW</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Pathogen Genomics Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">341</span>;<span class="number">6150</span>;<span class="pagerange">1069-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24009379">24009379</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1243787">10.1126/science.1243787</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing to delineate Mycobacterium tuberculosis outbreaks: a retrospective observational study.</h4>
<p class="authors"><span class="author">Walker TM</span>, <span class="author">Ip CL</span>, <span class="author">Harrell RH</span>, <span class="author">Evans JT</span>, <span class="author">Kapatai G</span>, <span class="author">Dedicoat MJ</span>, <span class="author">Eyre DW</span>, <span class="author">Wilson DJ</span>, <span class="author">Hawkey PM</span>, <span class="author">Crook DW</span>, <span class="author">Parkhill J</span>, <span class="author">Harris D</span>, <span class="author">Walker AS</span>, <span class="author">Bowden R</span>, <span class="author">Monk P</span>, <span class="author">Smith EG</span> and <span class="author">Peto TE</span></p>
<p class="affiliation">Nuffield Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK. <a href="mailto:%74%69%6D%6F%74%68%79.%77%61%6C%6B%65%72%40%6E%64%6D.%6F%78.%61%63.%75%6B">&#116;&#105;&#109;&#111;&#116;&#104;&#121;&#46;&#119;&#97;&#108;&#107;&#101;&#114;&#64;&#110;&#100;&#109;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Tuberculosis incidence in the UK has risen in the past decade. Disease control depends on epidemiological data, which can be difficult to obtain. Whole-genome sequencing can detect microevolution within Mycobacterium tuberculosis strains. We aimed to estimate the genetic diversity of related M tuberculosis strains in the UK Midlands and to investigate how this measurement might be used to investigate community outbreaks.</p><p class="abstract"><strong>Methods</strong>: In a retrospective observational study, we used Illumina technology to sequence M tuberculosis genomes from an archive of frozen cultures. We characterised isolates into four groups: cross-sectional, longitudinal, household, and community. We measured pairwise nucleotide differences within hosts and between hosts in household outbreaks and estimated the rate of change in DNA sequences. We used the findings to interpret network diagrams constructed from 11 community clusters derived from mycobacterial interspersed repetitive-unit-variable-number tandem-repeat data.</p><p class="abstract"><strong>Findings</strong>: We sequenced 390 separate isolates from 254 patients, including representatives from all five major lineages of M tuberculosis. The estimated rate of change in DNA sequences was 0.5 single nucleotide polymorphisms (SNPs) per genome per year (95% CI 0.3-0.7) in longitudinal isolates from 30 individuals and 25 families. Divergence is rarely higher than five SNPs in 3 years. 109 (96%) of 114 paired isolates from individuals and households differed by five or fewer SNPs. More than five SNPs separated isolates from none of 69 epidemiologically linked patients, two (15%) of 13 possibly linked patients, and 13 (17%) of 75 epidemiologically unlinked patients (three-way comparison exact p&lt;0.0001). Genetic trees and clinical and epidemiological data suggest that super-spreaders were present in two community clusters.</p><p class="abstract"><strong>Interpretation</strong>: Whole-genome sequencing can delineate outbreaks of tuberculosis and allows inference about direction of transmission between cases. The technique could identify super-spreaders and predict the existence of undiagnosed cases, potentially leading to early treatment of infectious patients and their contacts.</p><p class="abstract"><strong>Funding</strong>: Medical Research Council, Wellcome Trust, National Institute for Health Research, and the Health Protection Agency.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Department of Health: <span class="grant">DRF-2010-03-40</span>, <span class="grant">G0800778</span>; Medical Research Council: <span class="grant">G0800778</span>, <span class="grant">MR/J011398/1</span>; Wellcome Trust: <span class="grant">087646/Z/08/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">The Lancet. Infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">13</span>;<span class="number">2</span>;<span class="pagerange">137-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23158499">23158499</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3556524">3556524</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS1473-3099%2812%2970277-3">10.1016/S1473-3099(12)70277-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biochemical analysis of the Plasmodium falciparum erythrocyte-binding antigen-175 (EBA175)-glycophorin-A interaction: implications for vaccine design.</h4>
<p class="authors"><span class="author">Wanaguru M</span>, <span class="author">Crosnier C</span>, <span class="author">Johnson S</span>, <span class="author">Rayner JC</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">From the Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH.</p><p class="abstract">PfEBA175 has an important role in the invasion of human erythrocytes by Plasmodium falciparum and is therefore considered a high priority blood-stage malaria vaccine candidate. PfEBA175 mediates adhesion to erythrocytes through binding of the Duffy-binding-like (DBL) domains in its extracellular domain to Neu5Ac&alpha;2-3Gal displayed on the O-linked glycans of glycophorin-A (GYPA). Because of the difficulties in expressing active full-length (FL) P. falciparum proteins in a recombinant form, previous analyses of the PfEBA175-GYPA interaction have largely focused on the DBL domains alone, and therefore they have not been performed in the context of the native protein sequence. Here, we express the entire ectodomain of PfEBA175 (PfEBA175 FL) in soluble form, allowing us to compare the biochemical and immunological properties with a fragment containing only the tandem DBL domains (&quot;region II,&quot; PfEBA175 RII). Recombinant PfEBA175 FL bound human erythrocytes in a trypsin and neuraminidase-sensitive manner and recognized Neu5Ac&alpha;2-3Gal-containing glycans, confirming its biochemical activity. A quantitative binding analysis showed that PfEBA175 FL interacted with native GYPA with a KD &sim;0.26 &mu;M and is capable of self-association. By comparison, the RII fragment alone bound GYPA with a lower affinity demonstrating that regions outside of the DBL domains are important for interactions with GYPA; antibodies directed to these other regions also contributed to the inhibition of parasite invasion. These data demonstrate the importance of PfEBA175 regions other than the DBL domains in the interaction with GYPA and merit their inclusion in an EBA175-based vaccine.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900888</span>, <span class="grant">MR/J002283/1</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2013</span>;<span class="volume">288</span>;<span class="number">45</span>;<span class="pagerange">32106-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24043627">24043627</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3820850">3820850</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M113.484840">10.1074/jbc.M113.484840</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RH5-Basigin interaction plays a major role in the host tropism of Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Wanaguru M</span>, <span class="author">Liu W</span>, <span class="author">Hahn BH</span>, <span class="author">Rayner JC</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory and Malaria Programme, Wellcome Trust Sanger Institute, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Plasmodium falciparum, the cause of almost all human malaria mortality, is a member of the Laverania subgenus which infects African great apes. Interestingly, Laverania parasites exhibit strict host specificity in their natural environment: P. reichenowi, P. billcollinsi, and P. gaboni infect only chimpanzees; P. praefalciparum, P. blacklocki, and P. adleri are restricted to gorillas, and P. falciparum is pandemic in humans. The molecular mechanism(s) responsible for these host restrictions are not understood, although the interaction between the parasite blood-stage invasion ligand EBA175 and the host erythrocyte receptor Glycophorin-A (GYPA) has been implicated previously. We reexamined the role of the EBA175-GYPA interaction in host tropism using recombinant proteins and biophysical assays and found that EBA175 orthologs from the chimpanzee-restricted parasites P. reichenowi and P. billcollinsi both bound to human GYPA with affinities similar to that of P. falciparum, suggesting that the EBA175-GYPA interaction is unlikely to be the sole determinant of Laverania host specificity. We next investigated the contribution of the recently discovered Reticulocyte-binding protein Homolog 5 (RH5)-Basigin (BSG) interaction in host-species selectivity and found that P. falciparum RH5 bound chimpanzee BSG with a significantly lower affinity than human BSG and did not bind gorilla BSG, mirroring the known host tropism of P. falciparum. Using site-directed mutagenesis, we identified residues in BSG that are responsible for the species specificity of PfRH5 binding. Consistent with the essential role of the PfRH5-BSG interaction in erythrocyte invasion, we conclude that species-specific differences in the BSG receptor provide a molecular explanation for the restriction of P. falciparum to its human host.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MR/J002283/1</span>; NIAID NIH HHS: <span class="grant">P30 AI045008</span>, <span class="grant">R01 AI050529</span>, <span class="grant">R01 AI58715</span>, <span class="grant">R37 AI050529</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2013</span>;<span class="volume">110</span>;<span class="number">51</span>;<span class="pagerange">20735-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24297912">24297912</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3870751">3870751</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1320771110">10.1073/pnas.1320771110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic basis of Y-linked hearing impairment.</h4>
<p class="authors"><span class="author">Wang Q</span>, <span class="author">Xue Y</span>, <span class="author">Zhang Y</span>, <span class="author">Long Q</span>, <span class="author">Asan</span>, <span class="author">Yang F</span>, <span class="author">Turner DJ</span>, <span class="author">Fitzgerald T</span>, <span class="author">Ng BL</span>, <span class="author">Zhao Y</span>, <span class="author">Chen Y</span>, <span class="author">Liu Q</span>, <span class="author">Yang W</span>, <span class="author">Han D</span>, <span class="author">Quail MA</span>, <span class="author">Swerdlow H</span>, <span class="author">Burton J</span>, <span class="author">Fahey C</span>, <span class="author">Ning Z</span>, <span class="author">Hurles ME</span>, <span class="author">Carter NP</span>, <span class="author">Yang H</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Department of Otolaryngology, Head and Neck Surgery, Chinese PLA Institute of Otolaryngology, Chinese PLA General Hospital, Beijing, China.</p><p class="abstract">A single Mendelian trait has been mapped to the human Y chromosome: Y-linked hearing impairment. The molecular basis of this disorder is unknown. Here, we report the detailed characterization of the DFNY1 Y chromosome and its comparison with a closely related Y chromosome from an unaffected branch of the family. The DFNY1 chromosome carries a complex rearrangement, including duplication of several noncontiguous segments of the Y chromosome and insertion of &sim;160 kb of DNA from chromosome 1, in the pericentric region of Yp. This segment of chromosome 1 is derived entirely from within a known hearing impairment locus, DFNA49. We suggest that a third copy of one or more genes from the shared segment of chromosome 1 might be responsible for the hearing-loss phenotype.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">92</span>;<span class="number">2</span>;<span class="pagerange">301-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23352258">23352258</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3567277">3567277</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.12.015">10.1016/j.ajhg.2012.12.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The draft genomes of soft-shell turtle and green sea turtle yield insights into the development and evolution of the turtle-specific body plan.</h4>
<p class="authors"><span class="author">Wang Z</span>, <span class="author">Pascual-Anaya J</span>, <span class="author">Zadissa A</span>, <span class="author">Li W</span>, <span class="author">Niimura Y</span>, <span class="author">Huang Z</span>, <span class="author">Li C</span>, <span class="author">White S</span>, <span class="author">Xiong Z</span>, <span class="author">Fang D</span>, <span class="author">Wang B</span>, <span class="author">Ming Y</span>, <span class="author">Chen Y</span>, <span class="author">Zheng Y</span>, <span class="author">Kuraku S</span>, <span class="author">Pignatelli M</span>, <span class="author">Herrero J</span>, <span class="author">Beal K</span>, <span class="author">Nozawa M</span>, <span class="author">Li Q</span>, <span class="author">Wang J</span>, <span class="author">Zhang H</span>, <span class="author">Yu L</span>, <span class="author">Shigenobu S</span>, <span class="author">Wang J</span>, <span class="author">Liu J</span>, <span class="author">Flicek P</span>, <span class="author">Searle S</span>, <span class="author">Wang J</span>, <span class="author">Kuratani S</span>, <span class="author">Yin Y</span>, <span class="author">Aken B</span>, <span class="author">Zhang G</span> and <span class="author">Irie N</span></p>
<p class="affiliation">BGI-Shenzhen, Shenzhen, China.</p><p class="abstract">The unique anatomical features of turtles have raised unanswered questions about the origin of their unique body plan. We generated and analyzed draft genomes of the soft-shell turtle (Pelodiscus sinensis) and the green sea turtle (Chelonia mydas); our results indicated the close relationship of the turtles to the bird-crocodilian lineage, from which they split &sim;267.9-248.3 million years ago (Upper Permian to Triassic). We also found extensive expansion of olfactory receptor genes in these turtles. Embryonic gene expression analysis identified an hourglass-like divergence of turtle and chicken embryogenesis, with maximal conservation around the vertebrate phylotypic period, rather than at later stages that show the amniote-common pattern. Wnt5a expression was found in the growth zone of the dorsal shell, supporting the possible co-option of limb-associated Wnt signaling in the acquisition of this turtle-specific novelty. Our results suggest that turtle evolution was accompanied by an unexpectedly conservative vertebrate phylotypic period, followed by turtle-specific repatterning of development to yield the novel structure of the shell.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT095908</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">701-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23624526">23624526</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2615">10.1038/ng.2615</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Viral population analysis and minority-variant detection using short read next-generation sequencing.</h4>
<p class="authors"><span class="author">Watson SJ</span>, <span class="author">Welkers MR</span>, <span class="author">Depledge DP</span>, <span class="author">Coulter E</span>, <span class="author">Breuer JM</span>, <span class="author">de Jong MD</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">RNA viruses within infected individuals exist as a population of evolutionary-related variants. Owing to evolutionary change affecting the constitution of this population, the frequency and/or occurrence of individual viral variants can show marked or subtle fluctuations. Since the development of massively parallel sequencing platforms, such viral populations can now be investigated to unprecedented resolution. A critical problem with such analyses is the presence of sequencing-related errors that obscure the identification of true biological variants present at low frequency. Here, we report the development and assessment of the Quality Assessment of Short Read (QUASR) Pipeline (http://sourceforge.net/projects/quasr) specific for virus genome short read analysis that minimizes sequencing errors from multiple deep-sequencing platforms, and enables post-mapping analysis of the minority variants within the viral population. QUASR significantly reduces the error-related noise in deep-sequencing datasets, resulting in increased mapping accuracy and reduction of erroneous mutations. Using QUASR, we have determined influenza virus genome dynamics in sequential samples from an in vitro evolution of 2009 pandemic H1N1 (A/H1N1/09) influenza from samples sequenced on both the Roche 454 GSFLX and Illumina GAIIx platforms. Importantly, concordance between the 454 and Illumina sequencing allowed unambiguous minority-variant detection and accurate determination of virus population turnover in vitro.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700814</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2013</span>;<span class="volume">368</span>;<span class="number">1614</span>;<span class="pagerange">20120205</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382427">23382427</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3678329">3678329</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2012.0205">10.1098/rstb.2012.0205</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clinically relevant mutant DNA gyrase alters supercoiling, changes the transcriptome, and confers multidrug resistance.</h4>
<p class="authors"><span class="author">Webber MA</span>, <span class="author">Ricci V</span>, <span class="author">Whitehead R</span>, <span class="author">Patel M</span>, <span class="author">Fookes M</span>, <span class="author">Ivens A</span> and <span class="author">Piddock LJ</span></p>
<p class="affiliation">Antimicrobial Agents Research Group, School of Immunity and Infection, Institute of Microbiology and Infection, The University of Birmingham, Edgbaston, Birmingham, United Kingdom.</p>
<p class="abstract">Unlabelled: Bacterial DNA is maintained in a supercoiled state controlled by the action of topoisomerases. Alterations in supercoiling affect fundamental cellular processes, including transcription. Here, we show that substitution at position 87 of GyrA of Salmonella influences sensitivity to antibiotics, including nonquinolone drugs, alters global supercoiling, and results in an altered transcriptome with increased expression of stress response pathways. Decreased susceptibility to multiple antibiotics seen with a GyrA Asp87Gly mutant was not a result of increased efflux activity or reduced reactive-oxygen production. These data show that a frequently observed and clinically relevant substitution within GyrA results in altered expression of numerous genes, including those important in bacterial survival of stress, suggesting that GyrA mutants may have a selective advantage under specific conditions. Our findings help contextualize the high rate of quinolone resistance in pathogenic strains of bacteria and may partly explain why such mutant strains are evolutionarily successful.

Importance: Fluoroquinolones are a powerful group of antibiotics that target bacterial enzymes involved in helping bacteria maintain the conformation of their chromosome. Mutations in the target enzymes allow bacteria to become resistant to these antibiotics, and fluoroquinolone resistance is common. We show here that these mutations also provide protection against a broad range of other antimicrobials by triggering a defensive stress response in the cell. This work suggests that fluoroquinolone resistance mutations may be beneficial under a range of conditions.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/G012016/1</span>; Medical Research Council: <span class="grant">G0501415</span>, <span class="grant">G0807977</span></p>
<p class="publication"><span class="title">mBio</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="number">4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23882012">23882012</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3735185">3735185</a>; DOI: <a href="http://dx.doi.org/10.1128%2FmBio.00273-13">10.1128/mBio.00273-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A calibrated human Y-chromosomal phylogeny based on resequencing.</h4>
<p class="authors"><span class="author">Wei W</span>, <span class="author">Ayub Q</span>, <span class="author">Chen Y</span>, <span class="author">McCarthy S</span>, <span class="author">Hou Y</span>, <span class="author">Carbone I</span>, <span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">We have identified variants present in high-coverage complete sequences of 36 diverse human Y chromosomes from Africa, Europe, South Asia, East Asia, and the Americas, representing eight major haplogroups. After restricting our analysis to 8.97 Mb of the unique male-specific Y sequence, we identified 6662 high-confidence variants, including single-nucleotide polymorphisms (SNPs), multi-nucleotide polymorphisms (MNPs), and indels. We constructed phylogenetic trees using these variants, or subsets of them, and recapitulated the known structure of the tree. Assuming a male mutation rate of 1 &times; 10(-9) per base pair per year, the time depth of the tree (haplogroups A3-R) was ~101,000-115,000 yr, and the lineages found outside Africa dated to 57,000-74,000 yr, both as expected. In addition, we dated a striking Paleolithic male lineage expansion to 41,000-52,000 yr ago and the node representing the major European Y lineage, R1b, to 4000-13,000 yr ago, supporting a Neolithic origin for these modern European Y chromosomes. In all, we provide a nearly 10-fold increase in the number of Y markers with phylogenetic information, and novel historical insights derived from placing them on a calibrated phylogenetic tree.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">2</span>;<span class="pagerange">388-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23038768">23038768</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3561879">3561879</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.143198.112">10.1101/gr.143198.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comparison of Y-chromosomal lineage dating using either resequencing or Y-SNP plus Y-STR genotyping.</h4>
<p class="authors"><span class="author">Wei W</span>, <span class="author">Ayub Q</span>, <span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">We have compared phylogenies and time estimates for Y-chromosomal lineages based on resequencing &sim;9 Mb of DNA and applying the program GENETREE to similar analyses based on the more standard approach of genotyping 26 Y-SNPs plus 21 Y-STRs and applying the programs NETWORK and BATWING. We find that deep phylogenetic structure is not adequately reconstructed after Y-SNP plus Y-STR genotyping, and that times estimated using observed Y-STR mutation rates are several-fold too recent. In contrast, an evolutionary mutation rate gives times that are more similar to the resequencing data. In principle, systematic comparisons of this kind can in future studies be used to identify the combinations of Y-SNP and Y-STR markers, and time estimation methodologies, that correspond best to resequencing data.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Forensic science international. Genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">568-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23768990">23768990</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3820021">3820021</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.fsigen.2013.03.014">10.1016/j.fsigen.2013.03.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic identification of trans eQTLs as putative drivers of known disease associations.</h4>
<p class="authors"><span class="author">Westra HJ</span>, <span class="author">Peters MJ</span>, <span class="author">Esko T</span>, <span class="author">Yaghootkar H</span>, <span class="author">Schurmann C</span>, <span class="author">Kettunen J</span>, <span class="author">Christiansen MW</span>, <span class="author">Fairfax BP</span>, <span class="author">Schramm K</span>, <span class="author">Powell JE</span>, <span class="author">Zhernakova A</span>, <span class="author">Zhernakova DV</span>, <span class="author">Veldink JH</span>, <span class="author">Van den Berg LH</span>, <span class="author">Karjalainen J</span>, <span class="author">Withoff S</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">Rivadeneira F</span>, <span class="author">&#39;t Hoen PA</span>, <span class="author">Reinmaa E</span>, <span class="author">Fischer K</span>, <span class="author">Nelis M</span>, <span class="author">Milani L</span>, <span class="author">Melzer D</span>, <span class="author">Ferrucci L</span>, <span class="author">Singleton AB</span>, <span class="author">Hernandez DG</span>, <span class="author">Nalls MA</span>, <span class="author">Homuth G</span>, <span class="author">Nauck M</span>, <span class="author">Radke D</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Perola M</span>, <span class="author">Salomaa V</span>, <span class="author">Brody J</span>, <span class="author">Suchy-Dicey A</span>, <span class="author">Gharib SA</span>, <span class="author">Enquobahrie DA</span>, <span class="author">Lumley T</span>, <span class="author">Montgomery GW</span>, <span class="author">Makino S</span>, <span class="author">Prokisch H</span>, <span class="author">Herder C</span>, <span class="author">Roden M</span>, <span class="author">Grallert H</span>, <span class="author">Meitinger T</span>, <span class="author">Strauch K</span>, <span class="author">Li Y</span>, <span class="author">Jansen RC</span>, <span class="author">Visscher PM</span>, <span class="author">Knight JC</span>, <span class="author">Psaty BM</span>, <span class="author">Ripatti S</span>, <span class="author">Teumer A</span>, <span class="author">Frayling TM</span>, <span class="author">Metspalu A</span>, <span class="author">van Meurs JB</span> and <span class="author">Franke L</span></p>
<p class="affiliation">1] Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands. [2].</p>
<p class="abstract">Identifying the downstream effects of disease-associated SNPs is challenging. To help overcome this problem, we performed expression quantitative trait locus (eQTL) meta-analysis in non-transformed peripheral blood samples from 5,311 individuals with replication in 2,775 individuals. We identified and replicated trans eQTLs for 233 SNPs (reflecting 103 independent loci) that were previously associated with complex traits at genome-wide significance. Some of these SNPs affect multiple genes in trans that are known to be altered in individuals with disease: rs4917014, previously associated with systemic lupus erythematosus (SLE), altered gene expression of C1QB and five type I interferon response genes, both hallmarks of SLE. DeepSAGE RNA sequencing showed that rs4917014 strongly alters the 3' UTR levels of IKZF1 in cis, and chromatin immunoprecipitation and sequencing analysis of the trans-regulated genes implicated IKZF1 as the causal gene. Variants associated with cholesterol metabolism and type 1 diabetes showed similar phenomena, indicating that large-scale eQTL mapping provides insight into the downstream effects of many trait-associated variants.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24013639">24013639</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2756">10.1038/ng.2756</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide SNP and CNV analysis identifies common and low-frequency variants associated with severe early-onset obesity.</h4>
<p class="authors"><span class="author">Wheeler E</span>, <span class="author">Huang N</span>, <span class="author">Bochukova EG</span>, <span class="author">Keogh JM</span>, <span class="author">Lindsay S</span>, <span class="author">Garg S</span>, <span class="author">Henning E</span>, <span class="author">Blackburn H</span>, <span class="author">Loos RJ</span>, <span class="author">Wareham NJ</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Hurles ME</span>, <span class="author">Barroso I</span> and <span class="author">Farooqi IS</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">Common and rare variants associated with body mass index (BMI) and obesity account for &lt;5% of the variance in BMI. We performed SNP and copy number variation (CNV) association analyses in 1,509 children with obesity at the extreme tail (&gt;3 s.d. from the mean) of the BMI distribution and 5,380 controls. Evaluation of 29 SNPs (P &lt; 1 &times; 10(-5)) in an additional 971 severely obese children and 1,990 controls identified 4 new loci associated with severe obesity (LEPR, PRKCH, PACS1 and RMST). A previously reported 43-kb deletion at the NEGR1 locus was significantly associated with severe obesity (P = 6.6 &times; 10(-7)). However, this signal was entirely driven by a flanking 8-kb deletion; absence of this deletion increased risk for obesity (P = 6.1 &times; 10(-11)). We found a significant burden of rare, single CNVs in severely obese cases (P &lt; 0.0001). Integrative gene network pathway analysis of rare deletions indicated enrichment of genes affecting G protein-coupled receptors (GPCRs) involved in the neuronal regulation of energy homeostasis.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0900554</span>, <span class="grant">G9824984</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NIDA NIH HHS: <span class="grant">R25 DA027995</span>; Wellcome Trust: <span class="grant">084713</span>, <span class="grant">098497</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">45</span>;<span class="number">5</span>;<span class="pagerange">513-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23563609">23563609</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4106235">4106235</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2607">10.1038/ng.2607</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide generation and systematic phenotyping of knockout mice reveals new roles for many genes.</h4>
<p class="authors"><span class="author">White JK</span>, <span class="author">Gerdin AK</span>, <span class="author">Karp NA</span>, <span class="author">Ryder E</span>, <span class="author">Buljan M</span>, <span class="author">Bussell JN</span>, <span class="author">Salisbury J</span>, <span class="author">Clare S</span>, <span class="author">Ingham NJ</span>, <span class="author">Podrini C</span>, <span class="author">Houghton R</span>, <span class="author">Estabel J</span>, <span class="author">Bottomley JR</span>, <span class="author">Melvin DG</span>, <span class="author">Sunter D</span>, <span class="author">Adams NC</span>, <span class="author">Sanger Institute Mouse Genetics Project</span>, <span class="author">Tannahill D</span>, <span class="author">Logan DW</span>, <span class="author">Macarthur DG</span>, <span class="author">Flint J</span>, <span class="author">Mahajan VB</span>, <span class="author">Tsang SH</span>, <span class="author">Smyth I</span>, <span class="author">Watt FM</span>, <span class="author">Skarnes WC</span>, <span class="author">Dougan G</span>, <span class="author">Adams DJ</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Bradley A</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Mutations in whole organisms are powerful ways of interrogating gene function in a realistic context. We describe a program, the Sanger Institute Mouse Genetics Project, that provides a step toward the aim of knocking out all genes and screening each line for a broad range of traits. We found that hitherto unpublished genes were as likely to reveal phenotypes as known genes, suggesting that novel genes represent a rich resource for investigating the molecular basis of disease. We found many unexpected phenotypes detected only because we screened for them, emphasizing the value of screening all mutants for a wide range of traits. Haploinsufficiency and pleiotropy were both surprisingly common. Forty-two percent of genes were essential for viability, and these were less likely to have a paralog and more likely to contribute to a protein complex than other genes. Phenotypic data and more than 900 mutants are openly available for further analysis. PAPERCLIP:</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">12401</span>, <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>, <span class="grant">RG45277 PCAG/116</span>; NEI NIH HHS: <span class="grant">5K08EY020530-02</span>, <span class="grant">EY08213</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">096540</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2013</span>;<span class="volume">154</span>;<span class="number">2</span>;<span class="pagerange">452-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23870131">23870131</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3717207">3717207</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2013.06.022">10.1016/j.cell.2013.06.022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNA-seq reveals the RNA binding proteins, Hfq and RsmA, play various roles in virulence, antibiotic production and genomic flux in Serratia sp. ATCC 39006.</h4>
<p class="authors"><span class="author">Wilf NM</span>, <span class="author">Reid AJ</span>, <span class="author">Ramsay JP</span>, <span class="author">Williamson NR</span>, <span class="author">Croucher NJ</span>, <span class="author">Gatto L</span>, <span class="author">Hester SS</span>, <span class="author">Goulding D</span>, <span class="author">Barquist L</span>, <span class="author">Lilley KS</span>, <span class="author">Kingsley RA</span>, <span class="author">Dougan G</span> and <span class="author">Salmond GP</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, Tennis Court Road, Cambridge, CB2 1QW, UK. <a href="mailto:%67%70%63%73%32%40%63%61%6D.%61%63.%75%6B">&#103;&#112;&#99;&#115;&#50;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Background: Serratia sp. ATCC 39006 (S39006) is a Gram-negative enterobacterium that is virulent in plant and animal models. It produces a red-pigmented trypyrrole secondary metabolite, prodigiosin (Pig), and a carbapenem antibiotic (Car), as well as the exoenzymes, pectate lyase and cellulase. Secondary metabolite production in this strain is controlled by a complex regulatory network involving quorum sensing (QS). Hfq and RsmA (two RNA binding proteins and major post-transcriptional regulators of gene expression) play opposing roles in the regulation of several key phenotypes within S39006. Prodigiosin and carbapenem production was abolished, and virulence attenuated, in an S39006 ∆hfq mutant, while the converse was observed in an S39006 rsmA transposon insertion mutant.

Results: In order to define the complete regulon of Hfq and RsmA, deep sequencing of cDNA libraries (RNA-seq) was used to analyse the whole transcriptome of S39006 ∆hfq and rsmA::Tn mutants. Moreover, we investigated global changes in the proteome using an LC-MS/MS approach. Analysis of differential gene expression showed that Hfq and RsmA directly or indirectly regulate (at the level of RNA) 4% and 19% of the genome, respectively, with some correlation between RNA and protein expression. Pathways affected include those involved in antibiotic regulation, virulence, flagella synthesis, and surfactant production. Although Hfq and RsmA are reported to activate flagellum production in E. coli and an adherent-invasive E. coli hfq mutant was shown to have no flagella by electron microscopy, we found that flagellar production was increased in the S39006 rsmA and hfq mutants. Additionally, deletion of rsmA resulted in greater genomic flux with increased activity of two mobile genetic elements. This was confirmed by qPCR and analysis of rsmA culture supernatant revealed the presence of prophage DNA and phage particles. Finally, expression of a hypothetical protein containing DUF364 increased prodigiosin production and was controlled by a putative 5' cis-acting regulatory RNA element.

Conclusion: Using a combination of transcriptomics and proteomics this study provides a systems-level understanding of Hfq and RsmA regulation and identifies similarities and differences in the regulons of two major regulators. Additionally our study indicates that RsmA regulates both core and variable genome regions and contributes to genome stability.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="pagerange">822</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24267595">24267595</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-14-822">10.1186/1471-2164-14-822</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ischemic stroke is associated with the ABO locus: The EuroCLOT study.</h4>
<p class="authors"><span class="author">Williams FM</span>, <span class="author">Carter AM</span>, <span class="author">Hysi PG</span>, <span class="author">Surdulescu G</span>, <span class="author">Hodgkiss D</span>, <span class="author">Soranzo N</span>, <span class="author">Traylor M</span>, <span class="author">Bevan S</span>, <span class="author">Dichgans M</span>, <span class="author">Rothwell PM</span>, <span class="author">Sudlow C</span>, <span class="author">Farrall M</span>, <span class="author">Silander K</span>, <span class="author">Kaunisto M</span>, <span class="author">Wagner P</span>, <span class="author">Saarela O</span>, <span class="author">Kuulasmaa K</span>, <span class="author">Virtamo J</span>, <span class="author">Salomaa V</span>, <span class="author">Amouyel P</span>, <span class="author">Arveiler D</span>, <span class="author">Ferrieres J</span>, <span class="author">Wiklund PG</span>, <span class="author">Ikram MA</span>, <span class="author">Hofman A</span>, <span class="author">Boncoraglio GB</span>, <span class="author">Parati EA</span>, <span class="author">Helgadottir A</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Thorleifsson G</span>, <span class="author">Stefansson K</span>, <span class="author">Seshadri S</span>, <span class="author">Destefano A</span>, <span class="author">Gschwendtner A</span>, <span class="author">Psaty B</span>, <span class="author">Longstreth W</span>, <span class="author">Mitchell BD</span>, <span class="author">Cheng YC</span>, <span class="author">Clarke R</span>, <span class="author">Ferrario M</span>, <span class="author">Bis JC</span>, <span class="author">Levi C</span>, <span class="author">Attia J</span>, <span class="author">Holliday EG</span>, <span class="author">Scott RJ</span>, <span class="author">Fornage M</span>, <span class="author">Sharma P</span>, <span class="author">Furie KL</span>, <span class="author">Rosand J</span>, <span class="author">Nalls M</span>, <span class="author">Meschia J</span>, <span class="author">Mosely TH</span>, <span class="author">Evans A</span>, <span class="author">Palotie A</span>, <span class="author">Markus HS</span>, <span class="author">Grant PJ</span>, <span class="author">Spector TD</span> and <span class="author">EuroCLOT Investigators the Wellcome Trust Case Control Consortium 2 MOnica Risk, Genetics, Archiving and Monograph MetaStroke and the International Stroke Genetics Consortium</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King's College London, London, United Kingdom. <a href="mailto:%66%72%61%6E%63%65%73.%77%69%6C%6C%69%61%6D%73%40%6B%63%6C.%61%63.%75%6B">&#102;&#114;&#97;&#110;&#99;&#101;&#115;&#46;&#119;&#105;&#108;&#108;&#105;&#97;&#109;&#115;&#64;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Objective: End-stage coagulation and the structure/function of fibrin are implicated in the pathogenesis of ischemic stroke. We explored whether genetic variants associated with end-stage coagulation in healthy REFVIDunteers account for the genetic predisposition to ischemic stroke and examined their influence on stroke subtype. Methods: Common genetic variants identified through genome-wide association studies of coagulation factors and fibrin structure/function in healthy twins (n = 2,100, Stage 1) were examined in ischemic stroke (n = 4,200 cases) using 2 independent samples of European ancestry (Stage 2). A third clinical collection having stroke subtyping (total 8,900 cases, 55,000 controls) was used for replication (Stage 3). Results: Stage 1 identified 524 single nucleotide polymorphisms (SNPs) from 23 linkage disequilibrium blocks having significant association (p < 5 × 10(-8) ) with 1 or more coagulation/fibrin phenotypes. The most striking associations included SNP rs5985 with factor XIII activity (p = 2.6 × 10(-186) ), rs10665 with FVII (p = 2.4 × 10(-47) ), and rs505922 in the ABO gene with both von Willebrand factor (p = 4.7 × 10(-57) ) and factor VIII (p = 1.2 × 10(-36) ). In Stage 2, the 23 independent SNPs were examined in stroke cases/noncases using MOnica Risk, Genetics, Archiving and Monograph (MORGAM) and Wellcome Trust Case Control Consortium 2 collections. SNP rs505922 was nominally associated with ischemic stroke (odds ratio = 0.94, 95% confidence interval = 0.88-0.99, p = 0.023). Independent replication in Meta-Stroke confirmed the rs505922 association with stroke, beta (standard error, SE) = 0.066 (0.02), p = 0.001, a finding specific to large-vessel and cardioembolic stroke (p = 0.001 and p = < 0.001, respectively) but not seen with small-vessel stroke (p = 0.811). Interpretation: ABO gene variants are associated with large-vessel and cardioembolic stroke but not small-vessel disease. This work sheds light on the different pathogenic mechanisms underpinning stroke subtype. Ann Neurol 2013.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">UL1 RR033176</span>; NHGRI NIH HHS: <span class="grant">U01 HG004402</span>, <span class="grant">U01 HG004446</span>; NHLBI NIH HHS: <span class="grant">N01 HC035129</span>, <span class="grant">R01 HL075366</span>, <span class="grant">R01 HL080295</span>, <span class="grant">R01 HL085251</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL093029</span>, <span class="grant">R01 HL105756</span>; NIA NIH HHS: <span class="grant">R01 AG008122</span>, <span class="grant">R01 AG015928</span>, <span class="grant">R01 AG016495</span>, <span class="grant">R01 AG020098</span>, <span class="grant">R01 AG023629</span>, <span class="grant">R01 AG027058</span>, <span class="grant">R01 AG031287</span>, <span class="grant">R01 AG033193</span>; NIDDK NIH HHS: <span class="grant">P30 DK063491</span>; NINDS NIH HHS: <span class="grant">R01 NS017950</span>, <span class="grant">R01 NS045012</span>, <span class="grant">U01 NS069208</span></p>
<p class="publication"><span class="title">Annals of neurology</span>&nbsp;<span class="year">2013</span>;<span class="volume">73</span>;<span class="number">1</span>;<span class="pagerange">16-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23381943">23381943</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3582024">3582024</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fana.23838">10.1002/ana.23838</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and comparative analysis of the gorilla MHC genomic sequence.</h4>
<p class="authors"><span class="author">Wilming LG</span>, <span class="author">Hart EA</span>, <span class="author">Coggill PC</span>, <span class="author">Horton R</span>, <span class="author">Gilbert JG</span>, <span class="author">Clee C</span>, <span class="author">Jones M</span>, <span class="author">Lloyd C</span>, <span class="author">Palmer S</span>, <span class="author">Sims S</span>, <span class="author">Whitehead S</span>, <span class="author">Wiley D</span>, <span class="author">Beck S</span> and <span class="author">Harrow JL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1HH, UK.</p><p class="abstract">Major histocompatibility complex (MHC) genes play a critical role in vertebrate immune response and because the MHC is linked to a significant number of auto-immune and other diseases it is of great medical interest. Here we describe the clone-based sequencing and subsequent annotation of the MHC region of the gorilla genome. Because the MHC is subject to extensive variation, both structural and sequence-wise, it is not readily amenable to study in whole genome shotgun sequence such as the recently published gorilla genome. The variation of the MHC also makes it of evolutionary interest and therefore we analyse the sequence in the context of human and chimpanzee. In our comparisons with human and re-annotated chimpanzee MHC sequence we find that gorilla has a trimodular RCCX cluster, versus the reference human bimodular cluster, and additional copies of Class I (pseudo)genes between Gogo-K and Gogo-A (the orthologues of HLA-K and -A). We also find that Gogo-H (and Patr-H) is coding versus the HLA-H pseudogene and, conversely, there is a Gogo-DQB2 pseudogene versus the HLA-DQB2 coding gene. Our analysis, which is freely available through the VEGA genome browser, provides the research community with a comprehensive dataset for comparative and evolutionary research of the MHC.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2013</span>;<span class="volume">2013</span>;<span class="pagerange">bat011</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23589541">23589541</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3626023">3626023</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbat011">10.1093/database/bat011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TM4SF20 Ancestral Deletion and Susceptibility to a Pediatric Disorder of Early Language Delay and Cerebral White Matter Hyperintensities.</h4>
<p class="authors"><span class="author">Wiszniewski W</span>, <span class="author">Hunter JV</span>, <span class="author">Hanchard NA</span>, <span class="author">Willer JR</span>, <span class="author">Shaw C</span>, <span class="author">Tian Q</span>, <span class="author">Illner A</span>, <span class="author">Wang X</span>, <span class="author">Cheung SW</span>, <span class="author">Patel A</span>, <span class="author">Campbell IM</span>, <span class="author">Hixson P</span>, <span class="author">Ester AR</span>, <span class="author">Azamian MS</span>, <span class="author">Potocki L</span>, <span class="author">Zapata G</span>, <span class="author">Hernandez PP</span>, <span class="author">Ramocki MB</span>, <span class="author">Santos-Cortez RL</span>, <span class="author">Wang G</span>, <span class="author">York MK</span>, <span class="author">Justice MJ</span>, <span class="author">Chu ZD</span>, <span class="author">Bader PI</span>, <span class="author">Omo-Griffith L</span>, <span class="author">Madduri NS</span>, <span class="author">Scharer G</span>, <span class="author">Crawford HP</span>, <span class="author">Yanatatsaneejit P</span>, <span class="author">Eifert A</span>, <span class="author">Kerr J</span>, <span class="author">Bacino CA</span>, <span class="author">Franklin AI</span>, <span class="author">Goin-Kochel RP</span>, <span class="author">Simpson G</span>, <span class="author">Immken L</span>, <span class="author">Haque ME</span>, <span class="author">Stosic M</span>, <span class="author">Williams MD</span>, <span class="author">Morgan TM</span>, <span class="author">Pruthi S</span>, <span class="author">Omary R</span>, <span class="author">Boyadjiev SA</span>, <span class="author">Win KK</span>, <span class="author">Thida A</span>, <span class="author">Hurles M</span>, <span class="author">Hibberd ML</span>, <span class="author">Khor CC</span>, <span class="author">Van Vinh Chau N</span>, <span class="author">Gallagher TE</span>, <span class="author">Mutirangura A</span>, <span class="author">Stankiewicz P</span>, <span class="author">Beaudet AL</span>, <span class="author">Maletic-Savatic M</span>, <span class="author">Rosenfeld JA</span>, <span class="author">Shaffer LG</span>, <span class="author">Davis EE</span>, <span class="author">Belmont JW</span>, <span class="author">Dunstan S</span>, <span class="author">Simmons CP</span>, <span class="author">Bonnen PE</span>, <span class="author">Leal SM</span>, <span class="author">Katsanis N</span>, <span class="author">Lupski JR</span> and <span class="author">Lalani SR</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA.</p>
<p class="abstract">White matter hyperintensities (WMHs) of the brain are important markers of aging and small-vessel disease. WMHs are rare in healthy children and, when observed, often occur with comorbid neuroinflammatory or vasculitic processes. Here, we describe a complex 4 kb deletion in 2q36.3 that segregates with early childhood communication disorders and WMH in 15 unrelated families predominantly from Southeast Asia. The premature brain aging phenotype with punctate and multifocal WMHs was observed in ∼70% of young carrier parents who underwent brain MRI. The complex deletion removes the penultimate exon 3 of TM4SF20, a gene encoding a transmembrane protein of unknown function. Minigene analysis showed that the resultant net loss of an exon introduces a premature stop codon, which, in turn, leads to the generation of a stable protein that fails to target to the plasma membrane and accumulates in the cytoplasm. Finally, we report this deletion to be enriched in individuals of Vietnamese Kinh descent, with an allele frequency of about 1%, embedded in an ancestral haplotype. Our data point to a constellation of early language delay and WMH phenotypes, driven by a likely toxic mechanism of TM4SF20 truncation, and highlight the importance of understanding and managing population-specific low-frequency pathogenic alleles.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">K23 NS078056</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23810381">23810381</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2013.05.027">10.1016/j.ajhg.2013.05.027</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Go retro and get a GRIP.</h4>
<p class="authors"><span class="author">Wong K</span>, <span class="author">Adams DJ</span> and <span class="author">Keane TM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK. <a href="mailto:%64%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Gene retrocopy insertions are a source of new genes and new gene functions, and can now be identified using paired-end whole genome sequencing data.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2013</span>;<span class="volume">14</span>;<span class="number">3</span>;<span class="pagerange">108</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23514103">23514103</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2013-14-3-108">10.1186/gb-2013-14-3-108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the yehUT Two-Component Regulatory System of Salmonella enterica Serovar Typhi and Typhimurium.</h4>
<p class="authors"><span class="author">Wong VK</span>, <span class="author">Pickard DJ</span>, <span class="author">Barquist L</span>, <span class="author">Sivaraman K</span>, <span class="author">Page AJ</span>, <span class="author">Hart PJ</span>, <span class="author">Arends MJ</span>, <span class="author">Holt KE</span>, <span class="author">Kane L</span>, <span class="author">Mottram LF</span>, <span class="author">Ellison L</span>, <span class="author">Bautista R</span>, <span class="author">McGee CJ</span>, <span class="author">Kay SJ</span>, <span class="author">Wileman TM</span>, <span class="author">Kenney LJ</span>, <span class="author">Maclennan CA</span>, <span class="author">Kingsley RA</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom ; University of Cambridge Department of Pathology, Addenbrooke's Hospital, Cambridge, United Kingdom.</p>
<p class="abstract">Proteins exhibiting hyper-variable sequences within a bacterial pathogen may be associated with host adaptation. Several lineages of the monophyletic pathogen Salmonella enterica serovar Typhi (S. Typhi) have accumulated non-synonymous mutations in the putative two-component regulatory system yehUT. Consequently we evaluated the function of yehUT in S. Typhi BRD948 and S. Typhimurium ST4/74. Transcriptome analysis identified the cstA gene, encoding a carbon starvation protein as the predominantly yehUT regulated gene in both these serovars. Deletion of yehUT had no detectable effect on the ability of these mutant Salmonella to invade cultured epithelial cells (S. Typhi and S. Typhimurium) or induce colitis in a murine model (S. Typhimurium only). Growth, metabolic and antimicrobial susceptibility tests identified no obvious influences of yehUT on these phenotypes.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">e84567</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24386394">24386394</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3875573">3875573</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0084567">10.1371/journal.pone.0084567</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Duplication and retention biases of essential and non-essential genes revealed by systematic knockdown analyses.</h4>
<p class="authors"><span class="author">Woods S</span>, <span class="author">Coghlan A</span>, <span class="author">Rivers D</span>, <span class="author">Warnecke T</span>, <span class="author">Jeffries SJ</span>, <span class="author">Kwon T</span>, <span class="author">Rogers A</span>, <span class="author">Hurst LD</span> and <span class="author">Ahringer J</span></p>
<p class="affiliation">The Gurdon Institute and Department of Genetics, University of Cambridge, Cambridge, United Kingdom.</p>
<p class="abstract">When a duplicate gene has no apparent loss-of-function phenotype, it is commonly considered that the phenotype has been masked as a result of functional redundancy with the remaining paralog. This is supported by indirect evidence showing that multi-copy genes show loss-of-function phenotypes less often than single-copy genes and by direct tests of phenotype masking using select gene sets. Here we take a systematic genome-wide RNA interference approach to assess phenotype masking in paralog pairs in the Caenorhabditis elegans genome. Remarkably, in contrast to expectations, we find that phenotype masking makes only a minor contribution to the low knockdown phenotype rate for duplicate genes. Instead, we find that non-essential genes are highly over-represented among duplicates, leading to a low observed loss-of-function phenotype rate. We further find that duplicate pairs derived from essential and non-essential genes have contrasting evolutionary dynamics: whereas non-essential genes are both more often successfully duplicated (fixed) and lost, essential genes are less often duplicated but upon successful duplication are maintained over longer periods. We expect the fundamental evolutionary duplication dynamics presented here to be broadly applicable.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2013</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">e1003330</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23675306">23675306</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3649981">3649981</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003330">10.1371/journal.pgen.1003330</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Policy challenges of clinical genome sequencing.</h4>
<p class="authors"><span class="author">Wright CF</span>, <span class="author">Middleton A</span>, <span class="author">Burton H</span>, <span class="author">Cunningham F</span>, <span class="author">Humphries SE</span>, <span class="author">Hurst J</span>, <span class="author">Birney E</span> and <span class="author">Firth HV</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">BMJ (Clinical research ed.)</span>&nbsp;<span class="year">2013</span>;<span class="volume">347</span>;<span class="pagerange">f6845</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24270507">24270507</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bcl11a Controls Flt3 Expression in Early Hematopoietic Progenitors and Is Required for pDC Development In Vivo.</h4>
<p class="authors"><span class="author">Wu X</span>, <span class="author">Satpathy AT</span>, <span class="author">Kc W</span>, <span class="author">Liu P</span>, <span class="author">Murphy TL</span> and <span class="author">Murphy KM</span></p>
<p class="affiliation">Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri, United States of America.</p>
<p class="abstract">Bcl11a is a transcription factor known to regulate lymphoid and erythroid development. Recent bioinformatic analysis of global gene expression patterns has suggested a role for Bcl11a in the development of dendritic cell (DC) lineages. We tested this hypothesis by analyzing the development of DC and other lineages in Bcl11a (-/-) mice. We found that Bcl11a was required for expression of IL-7 receptor (IL-7R) and Flt3 in early hematopoietic progenitor cells. In addition, we found severely decreased numbers of plasmacytoid dendritic cells (pDCs) in Bcl11a (-/-) fetal livers and in the bone marrow of Bcl11a (-/-) fetal liver chimeras. Moreover, Bcl11a (-/-) cells showed severely impaired in vitro development of Flt3L-derived pDCs and classical DCs (cDCs). In contrast, we found normal in vitro development of DCs from Bcl11a (-/-) fetal liver cells treated with GM-CSF. These results suggest that the persistent cDC development observed in Bcl11a (-/-) fetal liver chimeras reflects derivation from a Bcl11a- and Flt3-independent pathway in vivo.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e64800</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23741395">23741395</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3669380">3669380</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0064800">10.1371/journal.pone.0064800</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pneumococcal capsular switching: a historical perspective.</h4>
<p class="authors"><span class="author">Wyres KL</span>, <span class="author">Lambertsen LM</span>, <span class="author">Croucher NJ</span>, <span class="author">McGee L</span>, <span class="author">von Gottberg A</span>, <span class="author">Li&ntilde;ares J</span>, <span class="author">Jacobs MR</span>, <span class="author">Kristinsson KG</span>, <span class="author">Beall BW</span>, <span class="author">Klugman KP</span>, <span class="author">Parkhill J</span>, <span class="author">Hakenbeck R</span>, <span class="author">Bentley SD</span> and <span class="author">Brueggemann AB</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, and.</p>
<p class="abstract">Background.â€ƒChanges in serotype prevalence among pneumococcal populations result from both serotype replacement and serotype (capsular) switching. Temporal changes in serotype distributions are well documented, but the contribution of capsular switching to such changes is unknown. Furthermore, it is unclear to what extent vaccine-induced selective pressures drive capsular switching. Methods.â€ƒSerotype and multilocus sequence typing data for 426 pneumococci dated from 1937 through 2007 were analyzed. Whole-genome sequence data for a subset of isolates were used to investigate capsular switching events. Results.â€ƒWe identified 36 independent capsular switch events, 18 of which were explored in detail with whole-genome sequence data. Recombination fragment lengths were estimated for 11 events and ranged from approximately 19.0 kb to â‰¥58.2 kb. Two events took place no later than 1960, and the imported DNA included the capsular locus and the nearby penicillin-binding protein genes pbp2x and pbp1a. Conclusions.â€ƒCapsular switching has been a regular occurrence among pneumococcal populations throughout the past 7 decades. Recombination of large DNA fragments (>30 kb), sometimes including the capsular locus and penicillin-binding protein genes, predated both vaccine introduction and widespread antibiotic use. This type of recombination has likely been an intrinsic feature throughout the history of pneumococcal evolution.</p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2013</span>;<span class="volume">207</span>;<span class="number">3</span>;<span class="pagerange">439-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23175765" rel="external">23175765</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3537446" rel="external">3537446</a>; DOI: <a href="http://dx.doi.org/10.1093%2Finfdis%2Fjis703" rel="external">10.1093/infdis/jis703</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">INMEX--a web-based tool for integrative meta-analysis of expression data.</h4>
<p class="authors"><span class="author">Xia J</span>, <span class="author">Fjell CD</span>, <span class="author">Mayer ML</span>, <span class="author">Pena OM</span>, <span class="author">Wishart DS</span> and <span class="author">Hancock RE</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of British Columbia, Vancouver, V6T 1Z3, Canada, Centre for Microbial Diseases and Immunity Research, University of British Columbia, Vancouver, V6T 1Z4, Canada, James Hogg Research Centre, University of British Columbia, Vancouver, V6Z 1Y6, Canada, Department of Computing Science, University of Alberta, T6G 2E8, Canada, Department of Biological Sciences, University of Alberta, T6G 2E8, Canada and Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK.</p>
<p class="abstract">The widespread applications of various 'omics' technologies in biomedical research together with the emergence of public data repositories have resulted in a plethora of data sets for almost any given physiological state or disease condition. Properly combining or integrating these data sets with similar basic hypotheses can help reduce study bias, increase statistical power and improve overall biological understanding. However, the difficulties in data management and the complexities of analytical approaches have significantly limited data integration to enable meta-analysis. Here, we introduce integrative meta-analysis of expression data (INMEX), a user-friendly web-based tool designed to support meta-analysis of multiple gene-expression data sets, as well as to enable integration of data sets from gene expression and metabolomics experiments. INMEX contains three functional modules. The data preparation module supports flexible data processing, annotation and visualization of individual data sets. The statistical analysis module allows researchers to combine multiple data sets based on P-values, effect sizes, rank orders and other features. The significant genes can be examined in functional analysis module for enriched Gene Ontology terms or Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways, or expression profile visualization. INMEX has built-in support for common gene/metabolite identifiers (IDs), as well as 45 popular microarray platforms for human, mouse and rat. Complex operations are performed through a user-friendly web interface in a step-by-step manner. INMEX is freely available at http://www.inmex.ca.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23766290">23766290</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkt338">10.1093/nar/gkt338</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">INVEX - a web-based tool for integrative visualization of expression data.</h4>
<p class="authors"><span class="author">Xia J</span>, <span class="author">Lyle NH</span>, <span class="author">Mayer ML</span>, <span class="author">Pena OM</span> and <span class="author">Hancock RE</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of British Columbia, Vancouver, Canada V6T 1Z3; Centre for Microbial Diseases and Immunity Research, University of British Columbia, Vancouver, Canada V6T 1Z4; Wellcome Trust Sanger Institute, Hinxton, United Kingdom CB10 1SA.</p>
<p class="abstract">Summary: Gene expression or metabolomics data generated from clinical settings is often associated with multiple metadata (i.e. diagnosis, genotype, gender, etc.). It is of great interest to analyze and to visualize the data in these contexts. Here, we introduce INVEX - a novel web-based tool that integrates the server-side capabilities for data analysis with the browse-based technology for data visualization. INVEX has two key features: 1) flexible differential expression analysis for a wide variety of experimental designs; and 2) interactive visualization within the context of metadata and biological annotations. INVEX has built-in support for gene / metabolite annotation, and a fully functional heatmap builder.

Availability: Freely available at http://www.invex.ca.

Contact: <a href="mailto:%62%6F%62%40%68%61%6E%63%6F%63%6B%6C%61%62.%75%62%63.%63%61">&#98;&#111;&#98;&#64;&#104;&#97;&#110;&#99;&#111;&#99;&#107;&#108;&#97;&#98;&#46;&#117;&#98;&#99;&#46;&#99;&#97;</a>.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2013</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24078684">24078684</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtt562">10.1093/bioinformatics/btt562</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Serotonin: The Central Link between Bone Mass and Energy Metabolism</h4>
<p class="authors"><span class="author">Yadav, V.K.</span></p>
<p class="affiliation">YADAV,V.K.</p>
<p class="publication"><span class="title">Translational Endocrinology of Bone</span>&nbsp;<span class="year">2013</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-415784-2.00005-1">10.1016/B978-0-12-415784-2.00005-1</a>; URL: <a title="http://dx.doi.org/10.1016/B978-0-12-415784-2.00005-1" href="http://dx.doi.org/10.1016/B978-0-12-415784-2.00005-1">http://dx.doi.org/10.1016/B97<span class="print-hide">...</span><span class="print-show">8-0-12-41578</span>4-2.00005-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Serotonin: The Central Link between Bone Mass and Energy Metabolism</h4>
<p class="authors"><span class="author">Yadav,V.K.;</span></p>
<p class="publication"><span class="title">Translational Endocrinology of Bone</span>&nbsp;<span class="year">2013</span>;<span class="number">Chapter 5</span>;<span class="pagerange">51-62</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1016%2FB978-0-12-415784-2.00005-1">10.1016/B978-0-12-415784-2.00005-1</a>; URL: <a title="http://www.sciencedirect.com/science/article/pii/B9780124157842000051" href="http://www.sciencedirect.com/science/article/pii/B9780124157842000051">http://www.sciencedirect.com/scien<span class="print-hide">...</span><span class="print-show">ce/article/pii/B9780124157842</span>000051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhancement of microhomology-mediated genomic rearrangements by transient loss of mouse Bloom syndrome helicase.</h4>
<p class="authors"><span class="author">Yamanishi A</span>, <span class="author">Yusa K</span>, <span class="author">Horie K</span>, <span class="author">Tokunaga M</span>, <span class="author">Kusano K</span>, <span class="author">Kokubu C</span> and <span class="author">Takeda J</span></p>
<p class="affiliation">Department of Social and Environmental Medicine, Graduate School of Medicine, Osaka University, Suita, Osaka 565-0871, Japan;</p>
<p class="abstract">Bloom syndrome, an autosomal recessive disorder of the BLM gene, confers predisposition to a broad spectrum of early-onset cancers in multiple tissue types. Loss of genomic integrity is a primary hallmark of such human malignancies, but many studies using disease-affected specimens are limited in that they are retrospective and devoid of an appropriate experimental control. To overcome this, we devised an experimental system to recapitulate the early molecular events in genetically engineered mouse embryonic stem cells, in which cells undergoing loss of heterozygosity (LOH) can be enriched after inducible down-regulation of Blm expression, with or without site-directed DNA double-strand break (DSB) induction. Transient loss of BLM increased the rate of LOH, whose breakpoints were distributed along the chromosome. Combined with site-directed DSB induction, loss of BLM synergistically increased the rate of LOH and concentrated the breakpoints around the targeted chromosomal region. We characterized the LOH events using specifically tailored genomic tools, such as high-resolution array comparative genomic hybridization and high-density single nucleotide polymorphism genotyping, revealing that the combination of BLM suppression and DSB induction enhanced genomic rearrangements, including deletions and insertions, whose breakpoints were clustered in genomic inverted repeats and associated with junctional microhomologies. Our experimental approach successfully uncovered the detailed molecular mechanisms of as-yet-uncharacterized loss of heterozygosities and reveals the significant contribution of microhomology-mediated genomic rearrangements, which could be widely applicable to the early steps of cancer formation in general.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2013</span>;<span class="volume">23</span>;<span class="number">9</span>;<span class="pagerange">1462-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23908384">23908384</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3759722">3759722</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.152744.112">10.1101/gr.152744.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced Production of Recombinant Secretory Proteins in Pichia pastoris by Optimizing Kex2 P1' site.</h4>
<p class="authors"><span class="author">Yang S</span>, <span class="author">Kuang Y</span>, <span class="author">Li H</span>, <span class="author">Liu Y</span>, <span class="author">Hui X</span>, <span class="author">Li P</span>, <span class="author">Jiang Z</span>, <span class="author">Zhou Y</span>, <span class="author">Wang Y</span>, <span class="author">Xu A</span>, <span class="author">Li S</span>, <span class="author">Liu P</span> and <span class="author">Wu D</span></p>
<p class="affiliation">The Key Laboratory of Regenerative Biology and The Guangdong Provincial Key Laboratory of Stem Cell and Regenerative Medicine, Guangzhou Institute of Biomedicine and Health, Chinese Academy of Sciences, Guangzhou, China.</p>
<p class="abstract">Pichiapastoris is one of the most widely used expression systems for the production of recombinant secretory proteins. Its universal application is, however, somewhat hampered by its unpredictable yields for different heterologous proteins, which is now believed to be caused in part by their varied efficiencies to traffic through the host secretion machinery. The yeast endoprotease Kex2 removes the signal peptides from pre-proteins and releases the mature form of secreted proteins, thus, plays a pivotal role in the yeast secretory pathways. In this study, we found that the yields of many recombinant proteins were greatly influenced by Kex2 P1' site residues and the optimized P1's amino acid residue could largely determine the final amount of secretory proteins synthesized and secreted. A further improvement of secretory yield was achieved by genomic integration of additional Kex2 copies, which again highlighted the importance of Kex2 cleavage to the production of recombinant secretory proteins in Pichia yeast.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">9</span>;<span class="pagerange">e75347</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24069404">24069404</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3777899">3777899</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0075347">10.1371/journal.pone.0075347</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.</h4>
<p class="authors"><span class="author">Yang W</span>, <span class="author">Soares J</span>, <span class="author">Greninger P</span>, <span class="author">Edelman EJ</span>, <span class="author">Lightfoot H</span>, <span class="author">Forbes S</span>, <span class="author">Bindal N</span>, <span class="author">Beare D</span>, <span class="author">Smith JA</span>, <span class="author">Thompson IR</span>, <span class="author">Ramaswamy S</span>, <span class="author">Futreal PA</span>, <span class="author">Haber DA</span>, <span class="author">Stratton MR</span>, <span class="author">Benes C</span>, <span class="author">McDermott U</span> and <span class="author">Garnett MJ</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Alterations in cancer genomes strongly influence clinical responses to treatment and in many instances are potent biomarkers for response to drugs. The Genomics of Drug Sensitivity in Cancer (GDSC) database (www.cancerRxgene.org) is the largest public resource for information on drug sensitivity in cancer cells and molecular markers of drug response. Data are freely available without restriction. GDSC currently contains drug sensitivity data for almost 75 000 experiments, describing response to 138 anticancer drugs across almost 700 cancer cell lines. To identify molecular markers of drug response, cell line drug sensitivity data are integrated with large genomic datasets obtained from the Catalogue of Somatic Mutations in Cancer database, including information on somatic mutations in cancer genes, gene amplification and deletion, tissue type and transcriptional data. Analysis of GDSC data is through a web portal focused on identifying molecular biomarkers of drug sensitivity based on queries of specific anticancer drugs or cancer genes. Graphical representations of the data are used throughout with links to related resources and all datasets are fully downloadable. GDSC provides a unique resource incorporating large drug sensitivity and genomic datasets to facilitate the discovery of new therapeutic biomarkers for cancer therapies.</p>
<p class="grants">Funded by: Cancer Research UK; NIGMS NIH HHS: <span class="grant">T32 GM071340</span>; Wellcome Trust: <span class="grant">086357</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2013</span>;<span class="volume">41</span>;<span class="number">Database issue</span>;<span class="pagerange">D955-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23180760">23180760</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3531057">3531057</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgks1111">10.1093/nar/gks1111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Aging as accelerated accumulation of somatic variants: whole-genome sequencing of centenarian and middle-aged monozygotic twin pairs.</h4>
<p class="authors"><span class="author">Ye K</span>, <span class="author">Beekman M</span>, <span class="author">Lameijer EW</span>, <span class="author">Zhang Y</span>, <span class="author">Moed MH</span>, <span class="author">van den Akker EB</span>, <span class="author">Deelen J</span>, <span class="author">Houwing-Duistermaat JJ</span>, <span class="author">Kremer D</span>, <span class="author">Anvar SY</span>, <span class="author">Laros JF</span>, <span class="author">Jones D</span>, <span class="author">Raine K</span>, <span class="author">Blackburne B</span>, <span class="author">Potluri S</span>, <span class="author">Long Q</span>, <span class="author">Guryev V</span>, <span class="author">van der Breggen R</span>, <span class="author">Westendorp RG</span>, <span class="author">&#39;t Hoen PA</span>, <span class="author">den Dunnen J</span>, <span class="author">van Ommen GJ</span>, <span class="author">Willemsen G</span>, <span class="author">Pitts SJ</span>, <span class="author">Cox DR</span>, <span class="author">Ning Z</span>, <span class="author">Boomsma DI</span> and <span class="author">Slagboom PE</span></p>
<p class="affiliation">Molecular Epidemiology, Medical Statistics and Bioinformatics, Leiden University Medical Center, Leiden, the Netherlands.</p><p class="abstract">It has been postulated that aging is the consequence of an accelerated accumulation of somatic DNA mutations and that subsequent errors in the primary structure of proteins ultimately reach levels sufficient to affect organismal functions. The technical limitations of detecting somatic changes and the lack of insight about the minimum level of erroneous proteins to cause an error catastrophe hampered any firm conclusions on these theories. In this study, we sequenced the whole genome of DNA in whole blood of two pairs of monozygotic (MZ) twins, 40 and 100 years old, by two independent next-generation sequencing (NGS) platforms (Illumina and Complete Genomics). Potentially discordant single-base substitutions supported by both platforms were validated extensively by Sanger, Roche 454, and Ion Torrent sequencing. We demonstrate that the genomes of the two twin pairs are germ-line identical between co-twins, and that the genomes of the 100-year-old MZ twins are discerned by eight confirmed somatic single-base substitutions, five of which are within introns. Putative somatic variation between the 40-year-old twins was not confirmed in the validation phase. We conclude from this systematic effort that by using two independent NGS platforms, somatic single nucleotide substitutions can be detected, and that a century of life did not result in a large number of detectable somatic mutations in blood. The low number of somatic variants observed by using two NGS platforms might provide a framework for detecting disease-related somatic variants in phenotypically discordant MZ twins.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">P30 AG038072</span></p>
<p class="publication"><span class="title">Twin research and human genetics : the official journal of the International Society for Twin Studies</span>&nbsp;<span class="year">2013</span>;<span class="volume">16</span>;<span class="number">6</span>;<span class="pagerange">1026-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24182360">24182360</a>; DOI: <a href="http://dx.doi.org/10.1017%2Fthg.2013.73">10.1017/thg.2013.73</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Zebrafish models of cancer: progress and future challenges.</h4>
<p class="authors"><span class="author">Yen J</span>, <span class="author">White RM</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The need for scalable strategies to probe the biological consequences of candidate cancer genes has never been more pressing. The zebrafish, with its capacity for high-throughput transgenesis, in vivo imaging and chemical/genetic screening, has ideal features for undertaking this task. Unique biological insights from zebrafish have already led to the identification of novel oncogenic drivers and small molecules being used to treat the human cancer. This review summarizes the recent main findings and describes pertinent areas where the zebrafish can greatly contribute to our understanding of cancer biology and treatment.</p>
<p class="publication"><span class="title">Current opinion in genetics &amp; development</span>&nbsp;<span class="year">2013</span>;<span class="volume">24C</span>;<span class="pagerange">38-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24657535">24657535</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gde.2013.11.003">10.1016/j.gde.2013.11.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Seamless genome editing in human pluripotent stem cells using custom endonuclease-based gene targeting and the piggyBac transposon.</h4>
<p class="authors"><span class="author">Yusa K</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p>
<p class="abstract">I report here a detailed protocol for seamless genome editing using the piggyBac transposon in human pluripotent stem cells (hPSCs). Recent advances in custom endonucleases have enabled us to routinely perform genome editing in hPSCs. Conventional approaches use the Cre/loxP system that leaves behind residual sequences in the targeted genome. I used the piggyBac transposon to seamlessly remove a drug selection cassette and demonstrated safe genetic correction of a mutation causing α-1 antitrypsin deficiency in patient-derived hPSCs. An alternative approach to using the piggyBac transposon to correct mutations involves using single-stranded oligonucleotides, which is a faster process to complete. However, this experimental procedure is rather complicated and it may be hard to achieve homozygous modifications. In contrast, using the piggyBac transposon with drug selection-based enrichment of genetic modifications, as described here, is simple and can yield multiple correctly targeted clones, including homozygotes. Although two rounds of genetic manipulation are required to achieve homozygote modifications, the entire process takes ∼3 months to complete.</p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2013</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">2061-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24071911">24071911</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2013.126">10.1038/nprot.2013.126</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interferon-induced transmembrane protein-3 genetic variant rs12252-C is associated with severe influenza in Chinese individuals.</h4>
<p class="authors"><span class="author">Zhang YH</span>, <span class="author">Zhao Y</span>, <span class="author">Li N</span>, <span class="author">Peng YC</span>, <span class="author">Giannoulatou E</span>, <span class="author">Jin RH</span>, <span class="author">Yan HP</span>, <span class="author">Wu H</span>, <span class="author">Liu JH</span>, <span class="author">Liu N</span>, <span class="author">Wang DY</span>, <span class="author">Shu YL</span>, <span class="author">Ho LP</span>, <span class="author">Kellam P</span>, <span class="author">McMichael A</span> and <span class="author">Dong T</span></p>
<p class="affiliation">Beijing You&#39;an Hospital, Capital Medical University, Beijing PO 100069, China.</p><p class="abstract">The SNP rs12252-C allele alters the function of interferon-induced transmembrane protein-3 increasing the disease severity of influenza virus infection in Caucasians, but the allele is rare. However, rs12252-C is much more common in Han Chinese. Here we report that the CC genotype is found in 69% of Chinese patients with severe pandemic influenza A H1N1/09 virus infection compared with 25% in those with mild infection. Specifically, the CC genotype was estimated to confer a sixfold greater risk for severe infection than the CT and TT genotypes. More importantly, because the risk genotype occurs with such a high frequency, its effect translates to a large population-attributable risk of 54.3% for severe infection in the Chinese population studied compared with 5.4% in Northern Europeans. Interferon-induced transmembrane protein-3 genetic variants could, therefore, have a strong effect of the epidemiology of influenza in China and in people of Chinese descent.</p>
<p class="grants">Funded by: Department of Health: <span class="grant">DHCS/04/G121/68</span>; Medical Research Council: <span class="grant">G0600371</span>, <span class="grant">G0600520</span>, <span class="grant">G1001046</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature communications</span>&nbsp;<span class="year">2013</span>;<span class="volume">4</span>;<span class="pagerange">1418</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23361009">23361009</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3562464">3562464</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncomms2433">10.1038/ncomms2433</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide methylation analyses of primary human leukocyte subsets identifies functionally important cell-type-specific hypomethylated regions.</h4>
<p class="authors"><span class="author">Zilbauer M</span>, <span class="author">Rayner TF</span>, <span class="author">Clark C</span>, <span class="author">Coffey AJ</span>, <span class="author">Joyce CJ</span>, <span class="author">Palta P</span>, <span class="author">Palotie A</span>, <span class="author">Lyons PA</span> and <span class="author">Smith KG</span></p>
<p class="affiliation">Department of Medicine and.</p>
<p class="abstract">DNA methylation is an important mechanism by which gene transcription and hence cellular function are regulated. Here, we provide detailed functional genome-wide methylome maps of 5 primary peripheral blood leukocyte subsets including T cells, B cells, monocytes/macrophages, and neutrophils obtained from healthy individuals. A comparison of these methylomes revealed highly specific cell-lineage and cell-subset methylation profiles. DNA hypomethylation is known to be permissive for gene expression and we identified cell-subset-specific hypomethylated regions (HMRs) that strongly correlate with gene transcription levels suggesting these HMRs may regulate corresponding cell functions. Single-nucleotide polymorphisms associated with immune-mediated disease in genome-wide association studies preferentially localized to these cell-specific regulatory HMRs, offering insight into pathogenesis by highlighting cell subsets in which specific epigenetic changes may drive disease. Our data provide a valuable reference tool for researchers aiming to investigate the role of DNA methylation in regulating primary leukocyte function in health and immune-mediated disease.</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2013</span>;<span class="volume">122</span>;<span class="number">25</span>;<span class="pagerange">e52-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24159175">24159175</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3862273">3862273</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2013-05-503201">10.1182/blood-2013-05-503201</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/3hi3vrxe"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/3hi3vrxe" src="/qr/3hi3vrxe.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Mon, 18 Feb 2013 01:48:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
