<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2003 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2003.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2003</h1><p>Number of papers published in 2003: <strong>113</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human TREM gene cluster at 6p21.1 encodes both activating and inhibitory single IgV domain receptors and includes NKp44.</h4>
<p class="authors"><span class="author">Allcock RJ</span>, <span class="author">Barrow AD</span>, <span class="author">Forbes S</span>, <span class="author">Beck S</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Wellcome Trust/MRC building, Addenbrookes Hospital, Cambridge, GB.</p><p class="abstract">We have characterized a cluster of single immunoglobulin variable (IgV) domain receptors centromeric of the major histocompatibility complex (MHC) on human chromosome 6. In addition to triggering receptor expressed on myeloid cells (TREM)-1 and TREM2, the cluster contains NKp44, a triggering receptor whose expression is limited to NK cells. We identified three new related genes and two gene fragments within a cluster of approximately 200 kb. Two of the three new genes lack charged residues in their transmembrane domain tails. Further, one of the genes contains two potential immunotyrosine Inhibitory motifs in its cytoplasmic tail, suggesting that it delivers inhibitory signals. The human and mouse TREM clusters appear to have diverged such that there are unique sequences in each species. Finally, each gene in the TREM cluster was expressed in a different range of cell types.</p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">2003</span>;<span class="volume">33</span>;<span class="number">2</span>;<span class="pagerange">567-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12645956" rel="external">12645956</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fimmu.200310033" rel="external">10.1002/immu.200310033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Co-duplication of olfactory receptor and MHC class I genes in the mouse major histocompatibility complex.</h4>
<p class="authors"><span class="author">Amadou C</span>, <span class="author">Younger RM</span>, <span class="author">Sims S</span>, <span class="author">Matthews LH</span>, <span class="author">Rogers J</span>, <span class="author">Kumanovics A</span>, <span class="author">Ziegler A</span>, <span class="author">Beck S</span> and <span class="author">Lindahl KF</span></p>
<p class="affiliation">Howard Hughes Medical Institute and Center for Immunology, University of Texas Southwestern Medical Center, Dallas, 75390-9050, USA.</p><p class="abstract">We report the 897 kb sequence of a cluster of olfactory receptor (OR) genes located at the distal end of the major histocompatibility complex (MHC) class I region on mouse chromosome 17 of strain 129/SvJ (H2bc). With additional information from the mouse genome draft sequence, we identified 59 OR loci (approximately 20% pseudogenes) in contrast to only 25 OR loci (approximately 50% pseudogenes) in the corresponding centromeric OR cluster that is part of the &#39;extended MHC class I region&#39; on human chromosome 6. Comparative analysis leads to three major observations: (i) most of the OR subfamilies have evolved independently in the two species, expanding more in the mouse, and resulting in co-orthologs--subfamilies of highly similar paralogs that keep orthologous relationships with their human counterparts; (ii) three of the mouse OR subfamilies have no orthologs in humans; and (iii) MHC class I loci are interspersed in the OR cluster in mouse but not in human, and were subjected to co-duplication with OR genes. Screening of our sequence against the available sequences of other strains/haplotypes revealed that most of the OR loci are polymorphic and that the number of OR loci may vary among strains/haplotypes. Our findings that MHC-linked OR loci share duplication with MHC class I loci, have duplicated extensively and are polymorphic revives questions about potential reciprocal influences acting on the dynamics and evolution of the H2 region and the H2-linked OR loci.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">12</span>;<span class="number">22</span>;<span class="pagerange">3025-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14506126" rel="external">14506126</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddg317" rel="external">10.1093/hmg/ddg317</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene annotation: prediction and testing.</h4>
<p class="authors"><span class="author">Ashurst JL</span> and <span class="author">Collins JE</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6A%6C%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#108;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Fifty years after the publication of DNA structure, the whole human genome sequence will be officially finished. This achievement marks the beginning of the task to catalogue every human gene and identify each of their function expression patterns. Currently, researchers estimate that there are about 30,000 human genes and approximately 70% of these can be automatically predicted using a combination of ab initio and similarity-based programs. However, to experimentally investigate every gene&#39;s function, the research community requires a high-quality annotation of alternative splicing, pseudogenes, and promoter regions that can only be provided by manual intervention. Manual curation of the human genome will be a long-term project as experimental data are continually produced to confirm or refine the predictions, and new features such as noncoding RNAs and enhancers have not been fully identified. Such a highly curated human gene-set made publicly available will be a great asset for the experimental community and for future comparative genome projects.</p>
<p class="publication"><span class="title">Annual review of genomics and human genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="pagerange">69-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14527297">14527297</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev.genom.4.070802.110300">10.1146/annurev.genom.4.070802.110300</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Managing peptidases in the genomic era.</h4>
<p class="authors"><span class="author">Barrett AJ</span>, <span class="author">Tolle DP</span> and <span class="author">Rawlings ND</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxon, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The enzymes that hydrolyse peptide bonds, called peptidases or proteases, are very important to mankind and are also very numerous. The many scientists working on these enzymes are rapidly acquiring new data, and they need good methods to store it and retrieve it. The storage and retrieval require effective systems of classification and nomenclature, and it is the design and implementation of these that we mean by &#39;managing&#39; peptidases. Ten years ago Rawlings and Barrett proposed the first comprehensive system for the classification of peptidases, which included a set of simple names for the families. In the present article we describe how the system has developed since then. The peptidase classification has now been adopted for use by many other databases, and provides the structure around which the MEROPS protease database (http://merops.sanger.ac.uk) is built.</p>
<p class="publication"><span class="title">Biological chemistry</span>&nbsp;<span class="year">2003</span>;<span class="volume">384</span>;<span class="number">6</span>;<span class="pagerange">873-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12887054" rel="external">12887054</a>; DOI: <a href="http://dx.doi.org/10.1515%2FBC.2003.098" rel="external">10.1515/BC.2003.098</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate gene association study in type 2 diabetes indicates a role for genes involved in beta-cell function as well as insulin action.</h4>
<p class="authors"><span class="author">Barroso I</span>, <span class="author">Luan J</span>, <span class="author">Middelberg RP</span>, <span class="author">Harding AH</span>, <span class="author">Franks PW</span>, <span class="author">Jakes RW</span>, <span class="author">Clayton D</span>, <span class="author">Schafer AJ</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Incyte, Palo Alto, California, USA. <a href="mailto:%69%62%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#98;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Type 2 diabetes is an increasingly common, serious metabolic disorder with a substantial inherited component. It is characterised by defects in both insulin secretion and action. Progress in identification of specific genetic variants predisposing to the disease has been limited. To complement ongoing positional cloning efforts, we have undertaken a large-scale candidate gene association study. We examined 152 SNPs in 71 candidate genes for association with diabetes status and related phenotypes in 2,134 Caucasians in a case-control study and an independent quantitative trait (QT) cohort in the United Kingdom. Polymorphisms in five of 15 genes (33%) encoding molecules known to primarily influence pancreatic beta-cell function-ABCC8 (sulphonylurea receptor), KCNJ11 (KIR6.2), SLC2A2 (GLUT2), HNF4A (HNF4alpha), and INS (insulin)-significantly altered disease risk, and in three genes, the risk allele, haplotype, or both had a biologically consistent effect on a relevant physiological trait in the QT study. We examined 35 genes predicted to have their major influence on insulin action, and three (9%)-INSR, PIK3R1, and SOS1-showed significant associations with diabetes. These results confirm the genetic complexity of Type 2 diabetes and provide evidence that common variants in genes influencing pancreatic beta-cell function may make a significant contribution to the inherited component of this disease. This study additionally demonstrates that the systematic examination of panels of biological candidate genes in large, well-characterised populations can be an effective complement to positional cloning approaches. The absence of large single-gene effects and the detection of multiple small effects accentuate the need for the study of larger populations in order to reliably identify the size of effect we now expect for complex diseases.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">E20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14551916">14551916</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC212698">212698</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0000020">10.1371/journal.pbio.0000020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The TROVE module: a common element in Telomerase, Ro and Vault ribonucleoproteins.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Kickhoefer V</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Ribonucleoproteins carry out a variety of important tasks in the cell. In this study we show that a number of these contain a novel module, that we speculate mediates RNA-binding.</p><p class="abstract"><strong>Results</strong>: The TROVE module--Telomerase, Ro and Vault module--is found in TEP1 and Ro60 the protein components of three ribonucleoprotein particles. This novel module, consisting of one or more domains, may be involved in binding the RNA components of the three RNPs, which are telomerase RNA, Y RNA and vault RNA. A second conserved region in these proteins is shown to be a member of the vWA domain family. The vWA domain in TEP1 is closely related to the previously recognised vWA domain in VPARP a second component of the vault particle. This vWA domain may mediate interactions between these vault components or bind as yet unidentified components of the RNPs.</p><p class="abstract"><strong>Conclusions</strong>: This work suggests that a number of ribonucleoprotein components use a common RNA-binding module. The TROVE module is also found in bacterial ribonucleoproteins suggesting an ancient origin for these ribonucleoproteins.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="pagerange">49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14563212" rel="external">14563212</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC270063" rel="external">270063</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-4-49" rel="external">10.1186/1471-2105-4-49</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The CHAP domain: a large family of amidases including GSP amidase and peptidoglycan hydrolases.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Rawlings ND</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK CB10 1SA. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Cleavage of peptidoglycan plays an important role in bacterial cell division, cell growth and cell lysis. Here, we reveal that several known peptidoglycan amidases fall into a family, which includes many proteins of previously unknown function. The family includes two different peptidoglycan cleavage activities: L-muramoyl-L-alanine amidase and D-alanyl-glycyl endopeptidase activity. The family includes the amidase portion of the bifunctional glutathionylspermidine synthase/amidase enzyme from bacteria and pathogenic trypanosomes. The glutathionylspermidine synthase is thought to be a key component of the alternative pathway in trypanosomes for protection from oxygen-radical damage and has been proposed as a potential drug target. The CHAP (cysteine, histidine-dependent amidohydrolases/peptidases) domain is often found in association with other domains that cleave peptidoglycan. The large number of multifunctional hydrolases suggests that they might act in a cooperative manner to cleave specialized substrates.</p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2003</span>;<span class="volume">28</span>;<span class="number">5</span>;<span class="pagerange">234-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12765834" rel="external">12765834</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0968-0004(03)00061-6" rel="external">10.1016/S0968-0004(03)00061-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Revisiting the mouse mitochondrial DNA sequence.</h4>
<p class="authors"><span class="author">Bayona-Bafaluy MP</span>, <span class="author">Ac&iacute;n-P&eacute;rez R</span>, <span class="author">Mullikin JC</span>, <span class="author">Park JS</span>, <span class="author">Moreno-Loshuertos R</span>, <span class="author">Hu P</span>, <span class="author">P&eacute;rez-Martos A</span>, <span class="author">Fern&aacute;ndez-Silva P</span>, <span class="author">Bai Y</span> and <span class="author">Enr&iacute;quez JA</span></p>
<p class="affiliation">Departamento de Bioqu&iacute;mica y Biolog&iacute;a Molecular y Celular, Universidad de Zaragoza, Miguel Servet 177, Zaragoza 50013, Spain.</p><p class="abstract">The existence of reliable mtDNA reference sequences for each species is of great relevance in a variety of fields, from phylogenetic and population genetics studies to pathogenetic determination of mtDNA variants in humans or in animal models of mtDNA-linked diseases. We present compelling evidence for the existence of sequencing errors on the current mouse mtDNA reference sequence. This includes the deletion of a full codon in two genes, the substitution of one amino acid on five occasions and also the involvement of tRNA and rRNA genes. The conclusions are supported by: (i) the re-sequencing of the original cell line used by Bibb and Clayton, the LA9 cell line, (ii) the sequencing of a second L-derivative clone (L929), and (iii) the comparison with 12 other mtDNA sequences from live mice, 10 of them maternally related with the mouse from which the L cells were generated. Two of the latest sequences are reported for the first time in this study (Balb/cJ and C57BL/6J). In addition, we found that both the LA9 and L929 mtDNAs also contain private clone polymorphic variants that, at least in the case of L929, promote functional impairment of the oxidative phosphorylation system. Consequently, the mtDNA of the strain used for the mouse genome project (C57BL/6J) is proposed as the new standard for the mouse mtDNA sequence.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2003</span>;<span class="volume">31</span>;<span class="number">18</span>;<span class="pagerange">5349-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12954771" rel="external">12954771</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC203322" rel="external">203322</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA sequence variation of Homo sapiens.</h4>
<p class="authors"><span class="author">Bentley DR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">68</span>;<span class="pagerange">55-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15338603" rel="external">15338603</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and analysis of the genome of the Whipple&#39;s disease bacterium Tropheryma whipplei.</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Maiwald M</span>, <span class="author">Murphy LD</span>, <span class="author">Pallen MJ</span>, <span class="author">Yeats CA</span>, <span class="author">Dover LG</span>, <span class="author">Norbertczak HT</span>, <span class="author">Besra GS</span>, <span class="author">Quail MA</span>, <span class="author">Harris DE</span>, <span class="author">von Herbay A</span>, <span class="author">Goble A</span>, <span class="author">Rutter S</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Barrell BG</span>, <span class="author">Parkhill J</span> and <span class="author">Relman DA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Whipple&#39;s disease is a rare multisystem chronic infection, involving the intestinal tract as well as various other organs. The causative agent, Tropheryma whipplei, is a Gram-positive bacterium about which little is known. Our aim was to investigate the biology of this organism by generating and analysing the complete DNA sequence of its genome.</p><p class="abstract"><strong>Methods</strong>: We isolated and propagated T whipplei strain TW08/27 from the cerebrospinal fluid of a patient diagnosed with Whipple&#39;s disease. We generated the complete sequence of the genome by the whole genome shotgun method, and analysed it with a combination of automatic and manual bioinformatic techniques.</p><p class="abstract"><strong>Findings</strong>: Sequencing revealed a condensed 925938 bp genome with a lack of key biosynthetic pathways and a reduced capacity for energy metabolism. A family of large surface proteins was identified, some associated with large amounts of non-coding repetitive DNA, and an unexpected degree of sequence variation.</p><p class="abstract"><strong>Interpretation</strong>: The genome reduction and lack of metabolic capabilities point to a host-restricted lifestyle for the organism. The sequence variation indicates both known and novel mechanisms for the elaboration and variation of surface structures, and suggests that immune evasion and host interaction play an important part in the lifestyle of this persistent bacterial pathogen.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK56339</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2003</span>;<span class="volume">361</span>;<span class="number">9358</span>;<span class="pagerange">637-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12606174">12606174</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2803%2912597-4">10.1016/S0140-6736(03)12597-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The devil is in the detail.</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Thomson NR</span>, <span class="author">Sebaihia M</span>, <span class="author">Crossman LC</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%73%64%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#100;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">6</span>;<span class="pagerange">256-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12823941" rel="external">12823941</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl: a genome infrastructure.</h4>
<p class="authors"><span class="author">Birney E</span> and <span class="author">Ensembl Team</span></p>
<p class="affiliation">EBI, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, United Kingdom.</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">68</span>;<span class="pagerange">213-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15338620" rel="external">15338620</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The intestinal protozoan parasite Entamoeba histolytica contains 20 cysteine protease genes, of which only a small subset is expressed during in vitro cultivation.</h4>
<p class="authors"><span class="author">Bruchhaus I</span>, <span class="author">Loftus BJ</span>, <span class="author">Hall N</span> and <span class="author">Tannich E</span></p>
<p class="affiliation">Bernhard Nocht Institute for Tropical Medicine, 20359 Hamburg, Germany. <a href="mailto:%62%72%75%63%68%68%61%75%73%40%62%6E%69.%75%6E%69-%68%61%6D%62%75%72%67.%64%65">&#98;&#114;&#117;&#99;&#104;&#104;&#97;&#117;&#115;&#64;&#98;&#110;&#105;&#46;&#117;&#110;&#105;&#45;&#104;&#97;&#109;&#98;&#117;&#114;&#103;&#46;&#100;&#101;</a></p><p class="abstract">Cysteine proteases are known to be important pathogenicity factors of the protozoan parasite Entamoeba histolytica. So far, a total of eight genes coding for cysteine proteases have been identified in E. histolytica, two of which are absent in the closely related nonpathogenic species E. dispar. However, present knowledge is restricted to enzymes expressed during in vitro cultivation of the parasite, which might represent only a subset of the entire repertoire. Taking advantage of the current E. histolytica genome-sequencing efforts, we analyzed databases containing more than 99% of all ameba gene sequences for the presence of cysteine protease genes. A total of 20 full-length genes was identified (including all eight genes previously reported), which show 10 to 86% sequence identity. The various genes obviously originated from two separate ancestors since they form two distinct clades. Despite cathepsin B-like substrate specificities, all of the ameba polypeptides are structurally related to cathepsin L-like enzymes. None of the previously described enzymes but 7 of the 12 newly identified proteins are unique compared to cathepsins of higher eukaryotes in that they are predicted to have transmembrane or glycosylphosphatidylinositol anchor attachment domains. Southern blot analysis revealed that orthologous sequences for all of the newly identified proteases are present in E. dispar. Interestingly, the majority of the various cysteine protease genes are not expressed in E. histolytica or E. dispar trophozoites during in vitro cultivation. Therefore, it is likely that at least some of these enzymes are required for infection of the human host and/or for completion of the parasite life cycle.</p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2003</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">501-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12796295" rel="external">12796295</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC161451" rel="external">161451</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel consensus DNA-binding sequence for BRCA1 protein complexes.</h4>
<p class="authors"><span class="author">Cable PL</span>, <span class="author">Wilson CA</span>, <span class="author">Calzone FJ</span>, <span class="author">Rauscher FJ</span>, <span class="author">Scully R</span>, <span class="author">Livingston DM</span>, <span class="author">Li L</span>, <span class="author">Blackwell CB</span>, <span class="author">Futreal PA</span> and <span class="author">Afshari CA</span></p>
<p class="affiliation">Laboratory of Molecular Carcinogenesis, National Institute of Environmental Health Sciences, Research Triangle Park, North Carolina, USA.</p><p class="abstract">Increasing evidence continues to emerge supporting the early hypothesis that BRCA1 might be involved in transcriptional processes. BRCA1 physically associates with more than 15 different proteins involved in transcription and is paradoxically involved in both transcriptional activation and repression. However, the underlying mechanism by which BRCA1 affects the gene expression of various genes remains speculative. In this study, we provide evidence that BRCA1 protein complexes interact with specific DNA sequences. We provide data showing that the upstream stimulatory factor 2 (USF2) physically associates with BRCA1 and is a component of this DNA-binding complex. Interestingly, these DNA-binding complexes are downregulated in breast cancer cell lines containing wild-type BRCA1, providing a critical link between modulations of BRCA1 function in sporadic breast cancers that do not involve germline BRCA1 mutations. The functional specificity of BRCA1 tumor suppression for breast and ovarian tissues is supported by our experiments, which demonstrate that BRCA1 DNA-binding complexes are modulated by serum and estrogen. Finally, functional analysis indicates that missense mutations in BRCA1 that lead to subsequent cancer susceptibility may result in improper gene activation. In summary, these findings establish a role for endogenous BRCA1 protein complexes in transcription via a defined DNA-binding sequence and indicate that one function of BRCA1 is to co-regulate the expression of genes involved in various cellular processes.</p>
<p class="publication"><span class="title">Molecular carcinogenesis</span>&nbsp;<span class="year">2003</span>;<span class="volume">38</span>;<span class="number">2</span>;<span class="pagerange">85-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14502648" rel="external">14502648</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fmc.10148" rel="external">10.1002/mc.10148</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Gene Ontology Annotation (GOA) project: implementation of GO in SWISS-PROT, TrEMBL, and InterPro.</h4>
<p class="authors"><span class="author">Camon E</span>, <span class="author">Magrane M</span>, <span class="author">Barrell D</span>, <span class="author">Binns D</span>, <span class="author">Fleischmann W</span>, <span class="author">Kersey P</span>, <span class="author">Mulder N</span>, <span class="author">Oinn T</span>, <span class="author">Maslen J</span>, <span class="author">Cox A</span> and <span class="author">Apweiler R</span></p>
<p class="affiliation">EMBL Outstation-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK.</p><p class="abstract">Gene Ontology Annotation (GOA) is a project run by the European Bioinformatics Institute (EBI) that aims to provide assignments of terms from the Gene Ontology (GO) resource to gene products in a number of its databases (http://www.ebi.ac.uk/GOA). In the first stage of this project, GO assignments have been applied to a data set representing the complete human proteome by a combination of electronic mappings and manual curation. This vocabulary has also been applied to the nonredundant proteome sets for all other completely sequenced organisms as well as to proteins from a wide range of organisms where the proteome is not yet complete.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1R01HGO2273-01</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">662-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12654719" rel="external">12654719</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC430163" rel="external">430163</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.461403" rel="external">10.1101/gr.461403</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A matter of fitness.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Crossman L</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">3</span>;<span class="pagerange">111-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12648939" rel="external">12648939</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complete genome sequence and analysis of Corynebacterium diphtheriae NCTC13129.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Efstratiou A</span>, <span class="author">Dover LG</span>, <span class="author">Holden MT</span>, <span class="author">Pallen M</span>, <span class="author">Bentley SD</span>, <span class="author">Besra GS</span>, <span class="author">Churcher C</span>, <span class="author">James KD</span>, <span class="author">De Zoysa A</span>, <span class="author">Chillingworth T</span>, <span class="author">Cronin A</span>, <span class="author">Dowd L</span>, <span class="author">Feltwell T</span>, <span class="author">Hamlin N</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rutherford KM</span>, <span class="author">Thomson NR</span>, <span class="author">Unwin L</span>, <span class="author">Whitehead S</span>, <span class="author">Barrell BG</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Corynebacterium diphtheriae is a Gram-positive, non-spore forming, non-motile, pleomorphic rod belonging to the genus Corynebacterium and the actinomycete group of organisms. The organism produces a potent bacteriophage-encoded protein exotoxin, diphtheria toxin (DT), which causes the symptoms of diphtheria. This potentially fatal infectious disease is controlled in many developed countries by an effective immunisation programme. However, the disease has made a dramatic return in recent years, in particular within the Eastern European region. The largest, and still on-going, outbreak since the advent of mass immunisation started within Russia and the newly independent states of the former Soviet Union in the 1990s. We have sequenced the genome of a UK clinical isolate (biotype gravis strain NCTC13129), representative of the clone responsible for this outbreak. The genome consists of a single circular chromosome of 2 488 635 bp, with no plasmids. It provides evidence that recent acquisition of pathogenicity factors goes beyond the toxin itself, and includes iron-uptake systems, adhesins and fimbrial proteins. This is in contrast to Corynebacterium&#39;s nearest sequenced pathogenic relative, Mycobacterium tuberculosis, where there is little evidence of recent horizontal DNA acquisition. The genome itself shows an unusually extreme large-scale compositional bias, being noticeably higher in G+C near the origin than at the terminus.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2003</span>;<span class="volume">31</span>;<span class="number">22</span>;<span class="pagerange">6516-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14602910">14602910</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC275568">275568</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BRAF and KRAS mutations in colorectal hyperplastic polyps and serrated adenomas.</h4>
<p class="authors"><span class="author">Chan TL</span>, <span class="author">Zhao W</span>, <span class="author">Leung SY</span>, <span class="author">Yuen ST</span> and <span class="author">Cancer Genome Project</span></p>
<p class="affiliation">Department of Pathology, The University of Hong Kong, Queen Mary Hospital, Hong Kong.</p><p class="abstract">Colorectal cancer is believed to progress through an adenoma-carcinoma sequence. However, recent evidence increasingly supports the existence of an alternative route for colorectal carcinogenesis through serrated polyps, a group that encompasses a morphological spectrum, including hyperplastic polyp (HP), admixed hyperplastic polyp/adenoma (HP/AD), and serrated adenoma (SA; the latter two manifest epithelial dysplasia). We have studied a large series of serrated polyps for BRAF and KRAS mutations. BRAF mutations were detected in 18 of 50 (36%) HPs, 2 of 10 (20%) HP/ADs, and 9 of 9 (100%) SAs. Twenty-six of 29 mutations caused amino acid substitutions at valine 599, the known hotspot. KRAS mutations were detected in 9 of 50 (18%) HPs, 6 of 10 (60%) HP/ADs, and 0 of 9 (0%) SAs. BRAF and KRAS mutations are mutually exclusive (P = 0.001). The associations of BRAF mutations with SAs (P &lt; 0.001) and KRAS mutations with HP/ADs (P = 0.005) are statistically significant. A majority (90%) of the serrated polyps showing dysplasia had mutations in either BRAF or KRAS, significantly different from those without dysplasia (54%; P = 0.014). Our data highlight the important role of activation of the RAS-RAF-mitogen-activated protein/extracellular signal-regulated kinase kinase-extracellular signal-regulated kinase-mitogen-activated protein kinase pathway in the initiation and progression of serrated neoplasms. Acquisition of a BRAF mutation appears to be associated with the progression of HP to SA, whereas progression to HP/AD is predominantly associated with acquisition of a KRAS mutation. The high incidence of BRAF mutations in HPs and SAs is consistent with the notion that the group of colorectal cancers carrying BRAF mutations may harbor most that have progressed through the HP-SA-carcinoma pathway.</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2003</span>;<span class="volume">63</span>;<span class="number">16</span>;<span class="pagerange">4878-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12941809" rel="external">12941809</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative and functional analyses of LYL1 loci establish marsupial sequences as a model for phylogenetic footprinting.</h4>
<p class="authors"><span class="author">Chapman MA</span>, <span class="author">Charchar FJ</span>, <span class="author">Kinston S</span>, <span class="author">Bird CP</span>, <span class="author">Grafham D</span>, <span class="author">Rogers J</span>, <span class="author">Gr&uuml;tzner F</span>, <span class="author">Graves JA</span>, <span class="author">Green AR</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, Cambridge University, Hills Road, Cambridge CB2 2XY, UK.</p><p class="abstract">Comparative genomic sequence analysis is a powerful technique for identifying regulatory regions in genomic DNA. However, its utility largely depends on the evolutionary distances between the species involved. Here we describe the screening of a genomic BAC library from the stripe-faced dunnart, Sminthopsis macroura, formerly known as the narrow-footed marsupial mouse. We isolated a clone containing the LYL1 locus, completely sequenced the 60.6-kb insert, and compared it with orthologous human and mouse sequences. Noncoding homology was substantially reduced in the human/dunnart analysis compared with human/mouse, yet we could readily identify all promoters and exons. Human/mouse/dunnart alignments of the LYL1 candidate promoter allowed us to identify putative transcription factor binding sites, revealing a pattern highly reminiscent of critical regulatory regions of the LYL1 paralogue, SCL. This newly identified LYL1 promoter showed strong activity in myeloid progenitor cells and was bound in vivo by Fli1, Elf1, and Gata2-transcription factors all previously shown to bind to the SCL stem cell enhancer. This study represents the first large-scale comparative analysis involving marsupial genomic sequence and demonstrates that such comparisons provide a powerful approach to characterizing mammalian regulatory elements.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2003</span>;<span class="volume">81</span>;<span class="number">3</span>;<span class="pagerange">249-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12659809" rel="external">12659809</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global transcriptional responses of fission yeast to environmental stress.</h4>
<p class="authors"><span class="author">Chen D</span>, <span class="author">Toone WM</span>, <span class="author">Mata J</span>, <span class="author">Lyne R</span>, <span class="author">Burns G</span>, <span class="author">Kivinen K</span>, <span class="author">Brazma A</span>, <span class="author">Jones N</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">We explored transcriptional responses of the fission yeast Schizosaccharomyces pombe to various environmental stresses. DNA microarrays were used to characterize changes in expression profiles of all known and predicted genes in response to five stress conditions: oxidative stress caused by hydrogen peroxide, heavy metal stress caused by cadmium, heat shock caused by temperature increase to 39 degrees C, osmotic stress caused by sorbitol, and DNA damage caused by the alkylating agent methylmethane sulfonate. We define a core environmental stress response (CESR) common to all, or most, stresses. There was a substantial overlap between CESR genes of fission yeast and the genes of budding yeast that are stereotypically regulated during stress. CESR genes were controlled primarily by the stress-activated mitogen-activated protein kinase Sty1p and the transcription factor Atf1p. S. pombe also activated gene expression programs more specialized for a given stress or a subset of stresses. In general, these &quot;stress-specific&quot; responses were less dependent on the Sty1p mitogen-activated protein kinase pathway and may involve specific regulatory factors. Promoter motifs associated with some of the groups of coregulated genes were identified. We compare and contrast global regulation of stress genes in fission and budding yeasts and discuss evolutionary implications.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular biology of the cell</span>&nbsp;<span class="year">2003</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">214-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12529438" rel="external">12529438</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC140239" rel="external">140239</a>; DOI: <a href="http://dx.doi.org/10.1091%2Fmbc.E02-08-0499" rel="external">10.1091/mbc.E02-08-0499</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2002: accommodating comparative genomics.</h4>
<p class="authors"><span class="author">Clamp M</span>, <span class="author">Andrews D</span>, <span class="author">Barker D</span>, <span class="author">Bevan P</span>, <span class="author">Cameron G</span>, <span class="author">Chen Y</span>, <span class="author">Clark L</span>, <span class="author">Cox T</span>, <span class="author">Cuff J</span>, <span class="author">Curwen V</span>, <span class="author">Down T</span>, <span class="author">Durbin R</span>, <span class="author">Eyras E</span>, <span class="author">Gilbert J</span>, <span class="author">Hammond M</span>, <span class="author">Hubbard T</span>, <span class="author">Kasprzyk A</span>, <span class="author">Keefe D</span>, <span class="author">Lehvaslaiho H</span>, <span class="author">Iyer V</span>, <span class="author">Melsopp C</span>, <span class="author">Mongin E</span>, <span class="author">Pettett R</span>, <span class="author">Potter S</span>, <span class="author">Rust A</span>, <span class="author">Schmidt E</span>, <span class="author">Searle S</span>, <span class="author">Slater G</span>, <span class="author">Smith J</span>, <span class="author">Spooner W</span>, <span class="author">Stabenau A</span>, <span class="author">Stalker J</span>, <span class="author">Stupka E</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Vastrik I</span> and <span class="author">Birney E</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The Ensembl (http://www.ensembl.org/) database project provides a bioinformatics framework to organise biology around the sequences of large genomes. It is a comprehensive source of stable automatic annotation of human, mouse and other genome sequences, available as either an interactive web site or as flat files. Ensembl also integrates manually annotated gene structures from external sources where available. As well as being one of the leading sources of genome annotation, Ensembl is an open source software engineering project to develop a portable system able to handle very large genomes and associated requirements. These range from sequence analysis to data storage and visualisation and installations exist around the world in both companies and at academic sites. With both human and mouse genome sequences available and more vertebrate sequences to follow, many of the recent developments in Ensembl have focusing on developing automatic comparative genome analysis and visualisation.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2003</span>;<span class="volume">31</span>;<span class="number">1</span>;<span class="pagerange">38-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12519943">12519943</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC165530">165530</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide screening for complete genetic loss in prostate cancer by comparative hybridization onto cDNA microarrays.</h4>
<p class="authors"><span class="author">Clark J</span>, <span class="author">Edwards S</span>, <span class="author">Feber A</span>, <span class="author">Flohr P</span>, <span class="author">John M</span>, <span class="author">Giddings I</span>, <span class="author">Crossland S</span>, <span class="author">Stratton MR</span>, <span class="author">Wooster R</span>, <span class="author">Campbell C</span> and <span class="author">Cooper CS</span></p>
<p class="affiliation">Molecular Carcinogenesis Section, Male Urological Cancer Research Center, Institute of Cancer Research, Sutton, Surrey, UK. <a href="mailto:%63%73%63%40%69%63%72.%61%63.%75%6B">&#99;&#115;&#99;&#64;&#105;&#99;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We demonstrate that comparative genomic hybridization (CGH) onto cDNA microarrays may be used to carry out genome-wide screens for regions of genetic loss, including homozygous (complete) deletions that may represent the possible location of tumour suppressor genes in human cancer. Screening of the prostate cancer cell lines LNCaP, PC3 and DU145 allowed the mapping of specific regions where genome copy number appeared altered and led to the identification of two novel regions of complete loss at 17q21.31 (500 kb spanning STAT3) and at 10q23.1 (50-350 kb spanning SFTPA2) in the PC3 cell line.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2003</span>;<span class="volume">22</span>;<span class="number">8</span>;<span class="pagerange">1247-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12606952" rel="external">12606952</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1206247" rel="external">10.1038/sj.onc.1206247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced protein domain discovery by using language modeling techniques from speech recognition.</h4>
<p class="authors"><span class="author">Coin L</span>, <span class="author">Bateman A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Most modern speech recognition uses probabilistic models to interpret a sequence of sounds. Hidden Markov models, in particular, are used to recognize words. The same techniques have been adapted to find domains in protein sequences of amino acids. To increase word accuracy in speech recognition, language models are used to capture the information that certain word combinations are more likely than others, thus improving detection based on context. However, to date, these context techniques have not been applied to protein domain discovery. Here we show that the application of statistical language modeling methods can significantly enhance domain recognition in protein sequences. As an example, we discover an unannotated Tf_Otx Pfam domain on the cone rod homeobox protein, which suggests a possible mechanism for how the V242M mutation on this protein causes cone-rod dystrophy.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2003</span>;<span class="volume">100</span>;<span class="number">8</span>;<span class="pagerange">4516-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12668763" rel="external">12668763</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC404693" rel="external">404693</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0737502100" rel="external">10.1073/pnas.0737502100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reevaluating human gene annotation: a second-generation analysis of chromosome 22.</h4>
<p class="authors"><span class="author">Collins JE</span>, <span class="author">Goward ME</span>, <span class="author">Cole CG</span>, <span class="author">Smink LJ</span>, <span class="author">Huckle EJ</span>, <span class="author">Knowles S</span>, <span class="author">Bye JM</span>, <span class="author">Beare DM</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">We report a second-generation gene annotation of human chromosome 22. Using expressed sequence databases, comparative sequence analysis, and experimental verification, we have extended genes, fused previously fragmented structures, and identified new genes. The total length in exons of annotation was increased by 74% over our previously published annotation and includes 546 protein-coding genes and 234 pseudogenes. Thirty-two potential protein-coding annotations are partial copies of other genes, and may represent duplications on an evolutionary path to change or loss of function. We also identified 31 non-protein-coding transcripts, including 16 possible antisense RNAs. By extrapolation, we estimate the human genome contains 29,000-36,000 protein-coding genes, 21,300 pseudogenes, and 1500 antisense RNAs. We suggest that our revised annotation criteria provide a paradigm for future annotation of the human genome.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">27-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12529303">12529303</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC430954">430954</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.695703">10.1101/gr.695703</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Boudicca, a retrovirus-like long terminal repeat retrotransposon from the genome of the human blood fluke Schistosoma mansoni.</h4>
<p class="authors"><span class="author">Copeland CS</span>, <span class="author">Brindley PJ</span>, <span class="author">Heyers O</span>, <span class="author">Michael SF</span>, <span class="author">Johnston DA</span>, <span class="author">Williams DL</span>, <span class="author">Ivens AC</span> and <span class="author">Kalinna BH</span></p>
<p class="affiliation">Department of Tropical Medicine, School of Public Health and Tropical Medicine, Tulane University Health Sciences Center, New Orleans, Louisiana 70112, USA.</p><p class="abstract">The genome of Schistosoma mansoni contains a proviral form of a retrovirus-like long terminal repeat (LTR) retrotransposon, designated BOUDICCA: Sequence and structural characterization of the new mobile genetic element, which was found in bacterial artificial chromosomes prepared from S. mansoni genomic DNA, revealed the presence of three putative open reading frames (ORFs) bounded by direct LTRs of 328 bp in length. ORF1 encoded a retrovirus-like major homology region and a Cys/His box motif, also present in Gag polyproteins of related retrotransposons and retroviruses. ORF2 encoded enzymatic domains and motifs characteristic of a retrovirus-like polyprotein, including aspartic protease, reverse transcriptase, RNase H, and integrase, in that order, a domain order similar to that of the gypsy/Ty3 retrotransposons. An additional ORF at the 3&#39; end of the retrotransposon may encode an envelope protein. Phylogenetic comparison based on the reverse transcriptase domain of ORF2 confirmed that Boudicca was a gypsy-like retrotransposon and showed that it was most closely related to CsRn1 from the Oriental liver fluke Clonorchis sinensis and to kabuki from Bombyx mori. Bioinformatics approaches together with Southern hybridization analysis of genomic DNA of S. mansoni and the screening of a bacterial artificial chromosome library representing approximately 8-fold coverage of the S. mansoni genome revealed that numerous copies of Boudicca were interspersed throughout the schistosome genome. By reverse transcription-PCR, mRNA transcripts were detected in the sporocyst, cercaria, and adult developmental stages of S. mansoni, indicating that Boudicca is actively transcribed in this trematode.</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2003</span>;<span class="volume">77</span>;<span class="number">11</span>;<span class="pagerange">6153-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12743272" rel="external">12743272</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC154989" rel="external">154989</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The not-so-humble worm.</h4>
<p class="authors"><span class="author">Crombie C</span>, <span class="author">Junio A</span> and <span class="author">Fraser A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14709170" rel="external">14709170</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC395730" rel="external">395730</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2003-5-1-301" rel="external">10.1186/gb-2003-5-1-301</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pathogenomics.</h4>
<p class="authors"><span class="author">Crossman L</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Bentley S</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The genomes described this month reflect the overall historical bias of microbial genomics towards pathogenic bacteria. Although the balance is now being redressed to some extent, especially through the study of extremophiles, it is still the case that the opportunities provided by genomic studies are primarily taken up by those who study bacterial pathogenicity. This part of the field is, however, being broadened by including the study of pathogens of animals, insects and plants alongside those that afflict humans.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">1</span>;<span class="number">3</span>;<span class="pagerange">176-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15035020">15035020</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro778">10.1038/nrmicro778</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A DNA damage checkpoint response in telomere-initiated senescence.</h4>
<p class="authors"><span class="author">d&#39;Adda di Fagagna F</span>, <span class="author">Reaper PM</span>, <span class="author">Clay-Farrace L</span>, <span class="author">Fiegler H</span>, <span class="author">Carr P</span>, <span class="author">Von Zglinicki T</span>, <span class="author">Saretzki G</span>, <span class="author">Carter NP</span> and <span class="author">Jackson SP</span></p>
<p class="affiliation">[1] The Wellcome Trust/Cancer Research UK Institute of Cancer and Developmental Biology, University of Cambridge, Cambridge CB2 1QR, UK [2] Present address: IFOM-FIRC Institute of Molecular Oncology, via Adamello 16, 20139 Milan, Italy.</p><p class="abstract">Most human somatic cells can undergo only a limited number of population doublings in vitro. This exhaustion of proliferative potential, called senescence, can be triggered when telomeres--the ends of linear chromosomes-cannot fulfil their normal protective functions. Here we show that senescent human fibroblasts display molecular markers characteristic of cells bearing DNA double-strand breaks. These markers include nuclear foci of phosphorylated histone H2AX and their co-localization with DNA repair and DNA damage checkpoint factors such as 53BP1, MDC1 and NBS1. We also show that senescent cells contain activated forms of the DNA damage checkpoint kinases CHK1 and CHK2. Furthermore, by chromatin immunoprecipitation and whole-genome scanning approaches, we show that the chromosome ends of senescent cells directly contribute to the DNA damage response, and that uncapped telomeres directly associate with many, but not all, DNA damage response proteins. Finally, we show that inactivation of DNA damage checkpoint kinases in senescent cells can restore cell-cycle progression into S phase. Thus, we propose that telomere-initiated senescence reflects a DNA damage checkpoint response that is activated with a direct contribution from dysfunctional telomeres.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">426</span>;<span class="number">6963</span>;<span class="pagerange">194-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14608368">14608368</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02118">10.1038/nature02118</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ddbRNA: detection of conserved secondary structures in multiple alignments.</h4>
<p class="authors"><span class="author">di Bernardo D</span>, <span class="author">Down T</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">Telethon Institute of Genetics and Medicine, Via P Castellino 111, 80133 Naples, Italy. <a href="mailto:%64%69%62%65%72%6E%61%72%64%6F%40%74%69%67%65%6D.%69%74">&#100;&#105;&#98;&#101;&#114;&#110;&#97;&#114;&#100;&#111;&#64;&#116;&#105;&#103;&#101;&#109;&#46;&#105;&#116;</a></p><p class="abstract"><strong>Motivation</strong>: Structured non-coding RNAs (ncRNAs) have a very important functional role in the cell. No distinctive general features common to all ncRNA have yet been discovered. This makes it difficult to design computational tools able to detect novel ncRNAs in the genomic sequence.</p><p class="abstract"><strong>Results</strong>: We devised an algorithm able to detect conserved secondary structures in both pairwise and multiple DNA sequence alignments with computational time proportional to the square of the sequence length. We implemented the algorithm for the case of pairwise and three-way alignments and tested it on ncRNAs obtained from public databases. On the test sets, the pairwise algorithm has a specificity greater than 97% with a sensitivity varying from 22.26% for Blast alignments to 56.35% for structural alignments. The three-way algorithm behaves similarly. Our algorithm is able to efficiently detect a conserved secondary structure in multiple alignments.</p>
<p class="grants">Funded by: Telethon: <span class="grant">TGM03P17</span>, <span class="grant">TGM06S01</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2003</span>;<span class="volume">19</span>;<span class="number">13</span>;<span class="pagerange">1606-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12967955">12967955</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recruitment of heterogeneous nuclear ribonucleoprotein A1 in vivo to the LMP/TAP region of the major histocompatibility complex.</h4>
<p class="authors"><span class="author">Donev R</span>, <span class="author">Horton R</span>, <span class="author">Beck S</span>, <span class="author">Doneva T</span>, <span class="author">Vatcheva R</span>, <span class="author">Bowen WR</span> and <span class="author">Sheer D</span></p>
<p class="affiliation">Human Cytogenetics Laboratory, Cancer Research, UK London Research Institute, Lincoln&#39;s Inn Fields Laboratories, 44 Lincoln&#39;s Inn Fields, London WC2A 3PX, United Kingdom.</p><p class="abstract">Sequences containing the matrix recognition signature were identified adjacent to the LMP/TAP gene cluster in the human and mouse major histocompatibility complex class II region. These sequences were shown to function as nuclear matrix attachment regions (MARs). Three of the five human MARs and the single mouse MAR recruit heterogeneous nuclear ribonucleoprotein A1 (hnRNP-A1) in vivo during transcriptional up-regulation of the major histocompatibility complex class II genes. The timing of this recruitment correlates with a rise in mature TAP1 mRNA. Two of the human MARs bind hnRNP-A1 in vitro directly within a 35-bp sequence that shows over 90% similarity to certain Alu repeat sequences. This study shows that MARs recruit and bind hnRNP-A1 upon transcriptional up-regulation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A3585</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2003</span>;<span class="volume">278</span>;<span class="number">7</span>;<span class="pagerange">5214-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12435746" rel="external">12435746</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M206621200" rel="external">10.1074/jbc.M206621200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Highly parallel SNP genotyping.</h4>
<p class="authors"><span class="author">Fan JB</span>, <span class="author">Oliphant A</span>, <span class="author">Shen R</span>, <span class="author">Kermani BG</span>, <span class="author">Garcia F</span>, <span class="author">Gunderson KL</span>, <span class="author">Hansen M</span>, <span class="author">Steemers F</span>, <span class="author">Butler SL</span>, <span class="author">Deloukas P</span>, <span class="author">Galver L</span>, <span class="author">Hunt S</span>, <span class="author">McBride C</span>, <span class="author">Bibikova M</span>, <span class="author">Rubano T</span>, <span class="author">Chen J</span>, <span class="author">Wickham E</span>, <span class="author">Doucet D</span>, <span class="author">Chang W</span>, <span class="author">Campbell D</span>, <span class="author">Zhang B</span>, <span class="author">Kruglyak S</span>, <span class="author">Bentley D</span>, <span class="author">Haas J</span>, <span class="author">Rigault P</span>, <span class="author">Zhou L</span>, <span class="author">Stuelpnagel J</span> and <span class="author">Chee MS</span></p>
<p class="affiliation">llumina, Inc., San Diego, California 92121, USA.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R43 CA-81952</span>; NHGRI NIH HHS: <span class="grant">HG-002753</span>, <span class="grant">R44 HG-02003</span></p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">68</span>;<span class="pagerange">69-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15338605" rel="external">15338605</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA microarrays for comparative genomic hybridization based on DOP-PCR amplification of BAC and PAC clones.</h4>
<p class="authors"><span class="author">Fiegler H</span>, <span class="author">Carr P</span>, <span class="author">Douglas EJ</span>, <span class="author">Burford DC</span>, <span class="author">Hunt S</span>, <span class="author">Scott CE</span>, <span class="author">Smith J</span>, <span class="author">Vetrie D</span>, <span class="author">Gorman P</span>, <span class="author">Tomlinson IP</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute/Cancer Research UK Genomic Microarray Group, Hinxton, Cambridge, CB10 1SA, United Kingdom.</p><p class="abstract">We have designed DOP-PCR primers specifically for the amplification of large insert clones for use in the construction of DNA microarrays. A bioinformatic approach was used to construct primers that were efficient in the general amplification of human DNA but were poor at amplifying E. coli DNA, a common contaminant of DNA preparations from large insert clones. We chose the three most selective primers for use in printing DNA microarrays. DNA combined from the amplification of large insert clones by use of these three primers and spotted onto glass slides showed more than a sixfold increase in the human to E. coli hybridization ratio when compared to the standard DOP-PCR primer, 6MW. The microarrays reproducibly delineated previously characterized gains and deletions in a cancer cell line and identified a small gain not detected by use of conventional CGH. We also describe a method for the bulk testing of the hybridization characteristics of chromosome-specific clones spotted on microarrays by use of DNA amplified from flow-sorted chromosomes. Finally, we describe a set of clones selected from the publicly available Golden Path of the human genome at 1-Mb intervals and a view in the Ensembl genome browser from which data required for the use of these clones in array CGH and other experiments can be downloaded across the Internet.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2003</span>;<span class="volume">36</span>;<span class="number">4</span>;<span class="pagerange">361-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12619160">12619160</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.10155">10.1002/gcc.10155</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Array painting: a method for the rapid analysis of aberrant chromosomes using DNA microarrays.</h4>
<p class="authors"><span class="author">Fiegler H</span>, <span class="author">Gribble SM</span>, <span class="author">Burford DC</span>, <span class="author">Carr P</span>, <span class="author">Prigmore E</span>, <span class="author">Porter KM</span>, <span class="author">Clegg S</span>, <span class="author">Crolla JA</span>, <span class="author">Dennis NR</span>, <span class="author">Jacobs P</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Objective</strong>: The authors describe a method, termed array painting, which allows the rapid, high resolution analysis of the content and breakpoints of aberrant chromosomes.</p><p class="abstract"><strong>Methods</strong>: Array painting is similar in concept to reverse chromosome painting and involves the hybridisation of probes generated by PCR of small numbers of flow sorted chromosomes on large insert genomic clone DNA microarrays.</p><p class="abstract"><strong>Results</strong>: and Conclusions: By analysing patients with cytogenetically balanced chromosome rearrangements, the authors show the effectiveness of array painting as a method to map breakpoints prior to cloning and sequencing chromosome rearrangements.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">40</span>;<span class="number">9</span>;<span class="pagerange">664-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12960211" rel="external">12960211</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735585" rel="external">1735585</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identifying protein domains with the Pfam database.</h4>
<p class="authors"><span class="author">Finn R</span>, <span class="author">Griffiths-Jones S</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs, United Kingdom.</p><p class="abstract">Pfam is a database of such protein domain families, with each family represented by multiple sequence alignments and profile hidden Markov models (HMMs). In addition, each family has associated annotation, literature references and links to other databases. The entries in Pfam are available via the worldwide web and in flatfile format. This unit contains detailed information on how to access and utilise the information present in the Pfam database, namely the families, multiple alignments and annotation. Details on running Pfam, both remotely and locally are presented.</p>
<p class="publication"><span class="title">Current protocols in bioinformatics / editoral board, Andreas D. Baxevanis ... [et al.]</span>&nbsp;<span class="year">2003</span>;<span class="volume">Chapter 2</span>;<span class="pagerange">Unit 2.5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18428696" rel="external">18428696</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471250953.bi0205s01" rel="external">10.1002/0471250953.bi0205s01</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Worms in L.A.</h4>
<p class="authors"><span class="author">Fraser A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%67%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">35</span>;<span class="number">1</span>;<span class="pagerange">3-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12947400" rel="external">12947400</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0903-3" rel="external">10.1038/ng0903-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complete genome sequence of Mycobacterium bovis.</h4>
<p class="authors"><span class="author">Garnier T</span>, <span class="author">Eiglmeier K</span>, <span class="author">Camus JC</span>, <span class="author">Medina N</span>, <span class="author">Mansoor H</span>, <span class="author">Pryor M</span>, <span class="author">Duthoy S</span>, <span class="author">Grondin S</span>, <span class="author">Lacroix C</span>, <span class="author">Monsempe C</span>, <span class="author">Simon S</span>, <span class="author">Harris B</span>, <span class="author">Atkin R</span>, <span class="author">Doggett J</span>, <span class="author">Mayes R</span>, <span class="author">Keating L</span>, <span class="author">Wheeler PR</span>, <span class="author">Parkhill J</span>, <span class="author">Barrell BG</span>, <span class="author">Cole ST</span>, <span class="author">Gordon SV</span> and <span class="author">Hewinson RG</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne and PT4 Annotation, G&eacute;nopole, Institut Pasteur, 28 Rue du Docteur Roux, 75724 Paris Cedex 15, France.</p><p class="abstract">Mycobacterium bovis is the causative agent of tuberculosis in a range of animal species and man, with worldwide annual losses to agriculture of $3 billion. The human burden of tuberculosis caused by the bovine tubercle bacillus is still largely unknown. M. bovis was also the progenitor for the M. bovis bacillus Calmette-Gu&eacute;rin vaccine strain, the most widely used human vaccine. Here we describe the 4,345,492-bp genome sequence of M. bovis AF2122/97 and its comparison with the genomes of Mycobacterium tuberculosis and Mycobacterium leprae. Strikingly, the genome sequence of M. bovis is &gt;99.95% identical to that of M. tuberculosis, but deletion of genetic information has led to a reduced genome size. Comparison with M. leprae reveals a number of common gene losses, suggesting the removal of functional redundancy. Cell wall components and secreted proteins show the greatest variation, indicating their potential role in host-bacillus interactions or immune evasion. Furthermore, there are no genes unique to M. bovis, implying that differential gene expression may be the key to the host tropisms of human and bovine bacilli. The genome sequence therefore offers major insight on the evolution, host preference, and pathobiology of M. bovis.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2003</span>;<span class="volume">100</span>;<span class="number">13</span>;<span class="pagerange">7877-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12788972">12788972</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC164681">164681</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1130426100">10.1073/pnas.1130426100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Synapse signalling complexes and networks: machines underlying cognition.</h4>
<p class="authors"><span class="author">Grant SG</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB1D 1SA, UK. <a href="mailto:%73%67%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">All thoughts and actions are encoded in patterns of neuronal electrical activity. Circuits of nerve cells connected by synapses are dedicated to processing information in these patterns. Information is not only transmitted across the synapse but also monitored by postsynaptic molecular machines. These machines are macromolecular complexes of approximately 100 proteins organised into a network of protein interactions. The network can be mathematically described as a scale-free network. Components of the complexes are necessary for decoding the neural code and converting electrical information into biochemical changes. The network properties of these complexes may explain many of the features of neuronal plasticity and cognitive function in rodents. Importantly, these multiprotein complexes and their network properties shed new light on the basis of human cognitive diseases including schizophrenia, autism, Huntington&#39;s disease and mental retardation. Supplementary material for this article can be found on the BioEssays website http://www.interscience.wiley.com/jpages/0265-9247/suppmat/index.html.</p>
<p class="publication"><span class="title">BioEssays : news and reviews in molecular, cellular and developmental biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">25</span>;<span class="number">12</span>;<span class="pagerange">1229-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14635258" rel="external">14635258</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fbies.10381" rel="external">10.1002/bies.10381</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systems biology in neuroscience: bridging genes to cognition.</h4>
<p class="authors"><span class="author">Grant SG</span></p>
<p class="affiliation">Division of Neuroscience, 1 George Square, Edinburgh EH8 9JZ, UK. <a href="mailto:%73%65%74%68.%67%72%61%6E%74%40%65%64.%61%63.%75%6B">&#115;&#101;&#116;&#104;&#46;&#103;&#114;&#97;&#110;&#116;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Systems biology is a new branch of biology aimed at understanding biological complexity. Genomic and proteomic methods integrated with cellular and organismal analyses allow modelling of physiological processes. Progress in understanding synapse composition and new experimental and bioinformatics methods indicate the synapse is an excellent starting point for global systems biology of the brain. A neuroscience systems biology programme, organized as a consortium, is proposed.</p>
<p class="publication"><span class="title">Current opinion in neurobiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">5</span>;<span class="pagerange">577-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14630221" rel="external">14630221</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular cytogenetics of polycythaemia vera: lack of occult rearrangements detectable by 20q LSP screening, CGH, and M-FISH.</h4>
<p class="authors"><span class="author">Gribble SM</span>, <span class="author">Reid AG</span>, <span class="author">Bench AJ</span>, <span class="author">Huntly BJ</span>, <span class="author">Grace C</span>, <span class="author">Green AR</span> and <span class="author">Nacheva EP</span></p>
<p class="publication"><span class="title">Leukemia : official journal of the Leukemia Society of America, Leukemia Research Fund, U.K</span>&nbsp;<span class="year">2003</span>;<span class="volume">17</span>;<span class="number">7</span>;<span class="pagerange">1419-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12835736" rel="external">12835736</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.leu.2402980" rel="external">10.1038/sj.leu.2402980</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rfam: an RNA family database.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span>, <span class="author">Bateman A</span>, <span class="author">Marshall M</span>, <span class="author">Khanna A</span> and <span class="author">Eddy SR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Rfam is a collection of multiple sequence alignments and covariance models representing non-coding RNA families. Rfam is available on the web in the UK at http://www.sanger.ac.uk/Software/Rfam/ and in the US at http://rfam.wustl.edu/. These websites allow the user to search a query sequence against a library of covariance models, and view multiple sequence alignments and family annotation. The database can also be downloaded in flatfile form and searched locally using the INFERNAL package (http://infernal.wustl.edu/). The first release of Rfam (1.0) contains 25 families, which annotate over 50 000 non-coding RNA genes in the taxonomic divisions of the EMBL nucleotide database.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2003</span>;<span class="volume">31</span>;<span class="number">1</span>;<span class="pagerange">439-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12520045">12520045</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC165453">165453</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic sequence and transcriptional profile of the boundary between pericentromeric satellites and genes on human chromosome arm 10p.</h4>
<p class="authors"><span class="author">Guy J</span>, <span class="author">Hearn T</span>, <span class="author">Crosier M</span>, <span class="author">Mudge J</span>, <span class="author">Viggiano L</span>, <span class="author">Koczan D</span>, <span class="author">Thiesen HJ</span>, <span class="author">Bailey JA</span>, <span class="author">Horvath JE</span>, <span class="author">Eichler EE</span>, <span class="author">Earthrowl ME</span>, <span class="author">Deloukas P</span>, <span class="author">French L</span>, <span class="author">Rogers J</span>, <span class="author">Bentley D</span> and <span class="author">Jackson MS</span></p>
<p class="affiliation">The Institute of Human Genetics, The International Centre for Life, University of Newcastle upon Tyne, Newcastle upon Tyne NE1 3BZ, UK.</p><p class="abstract">Contiguous finished sequence from highly duplicated pericentromeric regions of human chromosomes is needed if we are to understand the role of pericentromeric instability in disease, and in gene and karyotype evolution. Here, we have constructed a BAC contig spanning the transition from pericentromeric satellites to genes on the short arm of human chromosome 10, and used this to generate 1.4 Mb of finished genomic sequence. Combining RT-PCR, in silico gene prediction, and paralogy analysis, we can identify two domains within the sequence. The proximal 600 kb consists of satellite-rich pericentromerically duplicated DNA which is transcript poor, containing only three unspliced transcripts. In contrast, the distal 850 kb contains four known genes (ZNF248, ZNF25, ZNF33A, and ZNF37A) and up to 32 additional transcripts of unknown function. This distal region also contains seven out of the eight intrachromosomal duplications within the sequence, including the p arm copy of the approximately 250-kb duplication which gave rise to ZNF33A and ZNF33B. By sequencing orthologs of the duplicated ZNF33 genes we have established that ZNF33A has diverged significantly at residues critical for DNA binding but ZNF33B has not, indicating that ZNF33B has remained constrained by selection for ancestral gene function. These results provide further evidence of gene formation within intrachromosomal duplications, but indicate that recent interchromosomal duplications at this centromere have involved transcriptionally inert, satellite rich DNA, which is likely to be heterochromatic. This suggests that any novel gene structures formed by these interchromosomal events would require relocation to a more open chromatin environment to be expressed.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">2</span>;<span class="pagerange">159-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12566394" rel="external">12566394</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC420363" rel="external">420363</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.644503" rel="external">10.1101/gr.644503</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the gene encoding capillary morphogenesis protein 2 cause juvenile hyaline fibromatosis and infantile systemic hyalinosis.</h4>
<p class="authors"><span class="author">Hanks S</span>, <span class="author">Adams S</span>, <span class="author">Douglas J</span>, <span class="author">Arbour L</span>, <span class="author">Atherton DJ</span>, <span class="author">Balci S</span>, <span class="author">Bode H</span>, <span class="author">Campbell ME</span>, <span class="author">Feingold M</span>, <span class="author">Keser G</span>, <span class="author">Kleijer W</span>, <span class="author">Mancini G</span>, <span class="author">McGrath JA</span>, <span class="author">Muntoni F</span>, <span class="author">Nanda A</span>, <span class="author">Teare MD</span>, <span class="author">Warman M</span>, <span class="author">Pope FM</span>, <span class="author">Superti-Furga A</span>, <span class="author">Futreal PA</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, United Kingdom.</p><p class="abstract">Juvenile hyaline fibromatosis (JHF) and infantile systemic hyalinosis (ISH) are autosomal recessive conditions characterized by multiple subcutaneous skin nodules, gingival hypertrophy, joint contractures, and hyaline deposition. We previously mapped the gene for JHF to chromosome 4q21. We now report the identification of 15 different mutations in the gene encoding capillary morphogenesis protein 2 (CMG2) in 17 families with JHF or ISH. CMG2 is a transmembrane protein that is induced during capillary morphogenesis and that binds laminin and collagen IV via a von Willebrand factor type A (vWA) domain. Of interest, CMG2 also functions as a cellular receptor for anthrax toxin. Preliminary genotype-phenotype analyses suggest that abrogation of binding by the vWA domain results in severe disease typical of ISH, whereas in-frame mutations affecting a novel, highly conserved cytoplasmic domain result in a milder phenotype. These data (1) demonstrate that JHF and ISH are allelic conditions and (2) implicate perturbation of basement-membrane matrix assembly as the cause of the characteristic perivascular hyaline deposition seen in these conditions.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">73</span>;<span class="number">4</span>;<span class="pagerange">791-800</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14508707" rel="external">14508707</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1180602" rel="external">1180602</a>; DOI: <a href="http://dx.doi.org/10.1086%2F378418" rel="external">10.1086/378418</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Refined mapping of the HMSNR critical gene region--construction of a high-density integrated genetic and physical map.</h4>
<p class="authors"><span class="author">Hantke J</span>, <span class="author">Rogers T</span>, <span class="author">French L</span>, <span class="author">Tournev I</span>, <span class="author">Guergueltcheva V</span>, <span class="author">Urtizberea JA</span>, <span class="author">Colomer J</span>, <span class="author">Corches A</span>, <span class="author">Lupu C</span>, <span class="author">Merlini L</span>, <span class="author">Thomas PK</span> and <span class="author">Kalaydjieva L</span></p>
<p class="affiliation">Western Australian Institute for Medical Research and Centre for Medical Research, University of Western Australia, Perth, Australia.</p><p class="abstract">Hereditary motor and sensory neuropathy russe, a form of autosomal recessive Charcot-Marie-Tooth disease, is a rare disorder found in several Roma families from Europe. The gene has been mapped to a 1Mb region on 10q22. Detailed analysis led to the exclusion of 22 candidate genes and the assembly of a high-density genetic map comprising 141 polymorphic markers. Extensive genotyping in an extended sample of affected families resulted in a 10-fold reduction of the critical hereditary motor and sensory neuropathy russe gene region, which is now contained within a single completely sequenced BAC clone. The fact that no sequence variant has been detected in the known genes in the critical region indicates that the hereditary motor and sensory neuropathy russe mutation affects a novel gene that remains to be identified.</p>
<p class="publication"><span class="title">Neuromuscular disorders : NMD</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">9</span>;<span class="pagerange">729-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14561496" rel="external">14561496</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: a cross-species database for comparative genomics.</h4>
<p class="authors"><span class="author">Harris TW</span>, <span class="author">Lee R</span>, <span class="author">Schwarz E</span>, <span class="author">Bradnam K</span>, <span class="author">Lawson D</span>, <span class="author">Chen W</span>, <span class="author">Blasier D</span>, <span class="author">Kenny E</span>, <span class="author">Cunningham F</span>, <span class="author">Kishore R</span>, <span class="author">Chan J</span>, <span class="author">Muller HM</span>, <span class="author">Petcherski A</span>, <span class="author">Thorisson G</span>, <span class="author">Day A</span>, <span class="author">Bieri T</span>, <span class="author">Rogers A</span>, <span class="author">Chen CK</span>, <span class="author">Spieth J</span>, <span class="author">Sternberg P</span>, <span class="author">Durbin R</span> and <span class="author">Stein LD</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY 11724, USA.</p><p class="abstract">WormBase (http://www.wormbase.org/) is a web-accessible central data repository for information about Caenorhabditis elegans and related nematodes. The past two years have seen a significant expansion in the biological scope of WormBase, including the integration of large-scale, genome-wide data sets, the inclusion of genome sequence and gene predictions from related species and active literature curation. This expansion of data has also driven the development and refinement of user interfaces and operability, including a new Genome Browser, new searches and facilities for data access and the inclusion of extensive documentation. These advances have expanded WormBase beyond the obvious target audience of C. elegans researchers, to include researchers wishing to explore problems in functional and comparative genomics within the context of a powerful genetic system.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2003</span>;<span class="volume">31</span>;<span class="number">1</span>;<span class="pagerange">133-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12519966" rel="external">12519966</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC165500" rel="external">165500</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BACE1 (beta-secretase) transgenic and knockout mice: identification of neurochemical deficits and behavioral changes.</h4>
<p class="authors"><span class="author">Harrison SM</span>, <span class="author">Harper AJ</span>, <span class="author">Hawkins J</span>, <span class="author">Duddy G</span>, <span class="author">Grau E</span>, <span class="author">Pugh PL</span>, <span class="author">Winter PH</span>, <span class="author">Shilliam CS</span>, <span class="author">Hughes ZA</span>, <span class="author">Dawson LA</span>, <span class="author">Gonzalez MI</span>, <span class="author">Upton N</span>, <span class="author">Pangalos MN</span> and <span class="author">Dingwall C</span></p>
<p class="affiliation">Department of Comparative Genomics, GlaxoSmithKline, New Frontiers Science Park (North), Third Avenue, Harlow, Essex CM19 5AW, UK.</p><p class="abstract">BACE1 is a key enzyme in the generation of Abeta, the major component of senile plaques in the brains of Alzheimer&#39;s disease patients. We have generated transgenic mice expressing human BACE1 with the Cam Kinase II promoter driving neuronal-specific expression. The transgene contains the full-length coding sequence of human BACE1 preceding an internal ribosome entry site element followed by a LacZ reporter gene. These animals exhibit a bold, exploratory behavior and show elevated 5-hydroxytryptamine turnover. We have also generated a knockout mouse in which LacZ replaces the first exon of murine BACE1. Interestingly these animals show a contrasting behavior, being timid and less exploratory. Despite these clear differences both mouse lines are viable and fertile with no changes in morbidity. These results suggest an unexpected role for BACE1 in neurotransmission, perhaps through changes in amyloid precursor protein processing and Abeta levels.</p>
<p class="publication"><span class="title">Molecular and cellular neurosciences</span>&nbsp;<span class="year">2003</span>;<span class="volume">24</span>;<span class="number">3</span>;<span class="pagerange">646-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14664815" rel="external">14664815</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Streptomyces coelicolor A3(2) plasmid SCP2*: deductions from the complete sequence.</h4>
<p class="authors"><span class="author">Haug I</span>, <span class="author">Weissenborn A</span>, <span class="author">Brolle D</span>, <span class="author">Bentley S</span>, <span class="author">Kieser T</span> and <span class="author">Altenbuchner J</span></p>
<p class="affiliation">Institut f&uuml;r Industrielle Genetik, Universit&auml;t Stuttgart, Allmandring 31, 70569 Stuttgart, Germany.</p><p class="abstract">Plasmid SCP2* is a 31 kb, circular, low-copy-number plasmid originally identified in Streptomyces coelicolor A3(2) as a fertility factor. The plasmid was completely sequenced. The analysis of the 31 317 bp sequence revealed 34 ORFs encoding putative proteins from 31 to 710 aa long, most of them lacking similarity to known proteins. Three functional regions had been identified previously: the replication region, the transfer and spreading region, and the stability region. Three genes were identified in the stability region which contribute to the stability of SCP2 as shown by plasmid stability testing. The first gene, mrpA, encodes a new member of the lambda integrase family of site-specific recombinases. The two genes downstream of mrpA were called parA and parB. The gene product, ParA, shows similarity to a family of ATPases involved in plasmid partition. An increase of plasmid stability could be seen only when both genes were present. By deletion analysis, the replication region could be narrowed down to a 1.6 kb region, consisting of a 650 bp non-coding region and two genes, repI and repII, encoding proteins of 161 and 131 aa. Only RepI exhibits similarities to DNA binding elements and contains a putative helix-turn-helix motif. The traA gene that is essential for DNA transfer and pock formation was identified previously. Upstream of traA, 10 ORFs were found in the same orientation as traA which might be involved in conjugation and DNA spreading, together with one gene in the opposite orientation with similarities to transcriptional regulators of DNA transfer. Two transposable elements were found on SCP2*. IS1648 belongs to the IS3 family of insertion sequences. The second element, Tn5417, shows the highest similarity to the Tn4811 element located in the terminal inverted repeats of the Streptomyces lividans chromosome.</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2003</span>;<span class="volume">149</span>;<span class="number">Pt 2</span>;<span class="pagerange">505-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12624212" rel="external">12624212</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The magnificent seven.</h4>
<p class="authors"><span class="author">Holden M</span>, <span class="author">Bentley S</span>, <span class="author">Sebaihia M</span>, <span class="author">Thomson N</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">1</span>;<span class="pagerange">12-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12526849" rel="external">12526849</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Accessing HLA sequencing data through the 6ace database.</h4>
<p class="authors"><span class="author">Horton R</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Sanger Centre, Hinxton, United Kingdom.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2003</span>;<span class="volume">210</span>;<span class="pagerange">23-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12412449" rel="external">12412449</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EMSY links the BRCA2 pathway to sporadic breast and ovarian cancer.</h4>
<p class="authors"><span class="author">Hughes-Davies L</span>, <span class="author">Huntsman D</span>, <span class="author">Ruas M</span>, <span class="author">Fuks F</span>, <span class="author">Bye J</span>, <span class="author">Chin SF</span>, <span class="author">Milner J</span>, <span class="author">Brown LA</span>, <span class="author">Hsu F</span>, <span class="author">Gilks B</span>, <span class="author">Nielsen T</span>, <span class="author">Schulzer M</span>, <span class="author">Chia S</span>, <span class="author">Ragaz J</span>, <span class="author">Cahn A</span>, <span class="author">Linger L</span>, <span class="author">Ozdag H</span>, <span class="author">Cattaneo E</span>, <span class="author">Jordanova ES</span>, <span class="author">Schuuring E</span>, <span class="author">Yu DS</span>, <span class="author">Venkitaraman A</span>, <span class="author">Ponder B</span>, <span class="author">Doherty A</span>, <span class="author">Aparicio S</span>, <span class="author">Bentley D</span>, <span class="author">Theillet C</span>, <span class="author">Ponting CP</span>, <span class="author">Caldas C</span> and <span class="author">Kouzarides T</span></p>
<p class="affiliation">Cancer Research UK/Wellcome Trust Institute and Department of Pathology, Tennis Court Road, Cambridge CB2 1QR, United Kingdom. <a href="mailto:%6C%68%32%30%35%40%63%75%73.%63%61%6D.%61%63.%75%6B">&#108;&#104;&#50;&#48;&#53;&#64;&#99;&#117;&#115;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The BRCA2 gene is mutated in familial breast and ovarian cancer, and its product is implicated in DNA repair and transcriptional regulation. Here we identify a protein, EMSY, which binds BRCA2 within a region (exon 3) deleted in cancer. EMSY is capable of silencing the activation potential of BRCA2 exon 3, associates with chromatin regulators HP1beta and BS69, and localizes to sites of repair following DNA damage. EMSY maps to chromosome 11q13.5, a region known to be involved in breast and ovarian cancer. We show that the EMSY gene is amplified almost exclusively in sporadic breast cancer (13%) and higher-grade ovarian cancer (17%). In addition, EMSY amplification is associated with worse survival, particularly in node-negative breast cancer, suggesting that it may be of prognostic value. The remarkable clinical overlap between sporadic EMSY amplification and familial BRCA2 deletion implicates a BRCA2 pathway in sporadic breast and ovarian cancer.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2003</span>;<span class="volume">115</span>;<span class="number">5</span>;<span class="pagerange">523-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14651845" rel="external">14651845</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The International HapMap Project.</h4>
<p class="authors"><span class="author">International HapMap Consortium</span></p><p class="abstract">The goal of the International HapMap Project is to determine the common patterns of DNA sequence variation in the human genome and to make this information freely available in the public domain. An international consortium is developing a map of these patterns across the genome by determining the genotypes of one million or more sequence variants, their frequencies and the degree of association between them, in DNA samples from populations with ancestry from parts of Africa, Asia and Europe. The HapMap will allow the discovery of sequence variants that affect common disease, will facilitate development of diagnostic tools, and will enhance our ability to choose targets for therapeutic intervention.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">426</span>;<span class="number">6968</span>;<span class="pagerange">789-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14685227">14685227</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02168">10.1038/nature02168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Kaposi&#39;s sarcoma-associated herpesvirus-infected primary effusion lymphoma has a plasma cell gene expression profile.</h4>
<p class="authors"><span class="author">Jenner RG</span>, <span class="author">Maillard K</span>, <span class="author">Cattini N</span>, <span class="author">Weiss RA</span>, <span class="author">Boshoff C</span>, <span class="author">Wooster R</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wohl Virion Centre, Department of Immunology and Molecular Pathology, Windeyer Institute, University College London, London W1T 4JF, United Kingdom.</p><p class="abstract">Kaposi&#39;s sarcoma-associated herpesvirus is associated with three human tumors: Kaposi&#39;s sarcoma, and the B cell lymphomas, plasmablastic lymphoma associated with multicentric Castleman&#39;s disease, and primary effusion lymphoma (PEL). Epstein-Barr virus, the closest human relative of Kaposi&#39;s sarcoma-associated herpesvirus, mimics host B cell signaling pathways to direct B cell development toward a memory B cell phenotype. Epstein-Barr virus-associated B cell tumors are presumed to arise as a consequence of this virus-mediated B cell activation. The stage of B cell development represented by PEL, how this stage relates to tumor pathology, and how this information may be used to treat the disease are largely unknown. In this study we used gene expression profiling to order a range of B cell tumors by stage of development. PEL gene expression closely resembles that of malignant plasma cells, including the low expression of mature B cell genes. The unfolded protein response is partially activated in PEL, but is fully activated in plasma cell tumors, linking endoplasmic reticulum stress to plasma cell development through XBP-1. PEL cells can be defined by the overexpression of genes involved in inflammation, cell adhesion, and invasion, which may be responsible for their presentation in body cavities. Similar to malignant plasma cells, all PEL samples tested express the vitamin D receptor and are sensitive to the vitamin D analogue drug EB 1089 (Seocalcitol).</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2003</span>;<span class="volume">100</span>;<span class="number">18</span>;<span class="pagerange">10399-404</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12925741">12925741</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC193573">193573</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1630810100">10.1073/pnas.1630810100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic functional analysis of the Caenorhabditis elegans genome using RNAi.</h4>
<p class="authors"><span class="author">Kamath RS</span>, <span class="author">Fraser AG</span>, <span class="author">Dong Y</span>, <span class="author">Poulin G</span>, <span class="author">Durbin R</span>, <span class="author">Gotta M</span>, <span class="author">Kanapin A</span>, <span class="author">Le Bot N</span>, <span class="author">Moreno S</span>, <span class="author">Sohrmann M</span>, <span class="author">Welchman DP</span>, <span class="author">Zipperlen P</span> and <span class="author">Ahringer J</span></p>
<p class="affiliation">Wellcome Trust/Cancer Research UK Institute and Department of Genetics, University of Cambridge, Tennis Court Road, Cambridge CB2 1QR, UK.</p><p class="abstract">A principal challenge currently facing biologists is how to connect the complete DNA sequence of an organism to its development and behaviour. Large-scale targeted-deletions have been successful in defining gene functions in the single-celled yeast Saccharomyces cerevisiae, but comparable analyses have yet to be performed in an animal. Here we describe the use of RNA interference to inhibit the function of approximately 86% of the 19,427 predicted genes of C. elegans. We identified mutant phenotypes for 1,722 genes, about two-thirds of which were not previously associated with a phenotype. We find that genes of similar functions are clustered in distinct, multi-megabase regions of individual chromosomes; genes in these regions tend to share transcriptional profiles. Our resulting data set and reusable RNAi library of 16,757 bacterial clones will facilitate systematic analyses of the connections among gene sequence, chromosomal location and gene function in C. elegans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">054523</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">421</span>;<span class="number">6920</span>;<span class="pagerange">231-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12529635">12529635</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01278">10.1038/nature01278</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CASP5 target classification.</h4>
<p class="authors"><span class="author">Kinch LN</span>, <span class="author">Qi Y</span>, <span class="author">Hubbard TJ</span> and <span class="author">Grishin NV</span></p>
<p class="affiliation">Howard Hughes Medical Institute, University of Texas Southwestern Medical Center, Dallas 75390-9050, USA. <a href="mailto:%6C%6B%69%6E%63%68%40%63%68%6F%70.%73%77%6D%65%64.%65%64%75">&#108;&#107;&#105;&#110;&#99;&#104;&#64;&#99;&#104;&#111;&#112;&#46;&#115;&#119;&#109;&#101;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">This report summarizes the Critical Assessment of Protein Structure Prediction (CASP5) target proteins, which included 67 experimental models submitted from various structural genomics efforts and independent research groups. Throughout this special issue, CASP5 targets are referred to with the identification numbers T0129-T0195. Several of these targets were excluded from the assessment for various reasons: T0164 and T0166 were cancelled by the organizers; T0131, T0144, T0158, T0163, T0171, T0175, and T0180 were not available in time; T0145 was &quot;natively unfolded&quot;; the T0139 structure became available before the target expired; and T0194 was solved for a different sequence than the one submitted. Table I outlines the sequence and structural information available for CASP5 proteins in the context of existing folds and evolutionary relationships. This information provided the basis for a domain-based classification of the target structures into three assessment categories: comparative modeling (CM), fold recognition (FR), and new fold (NF). The FR category was further subdivided into homologues [FR(H)] and analogs [FR(A)] based on evolutionary considerations, and the overlap between assessment categories was classified as CM/FR(H) and FR(A)/NF. CASP5 domains are illustrated in Figure 1. Examples of nontrivial links between CASP5 target domains and existing structures that support our classifications are provided.</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2003</span>;<span class="volume">53 Suppl 6</span>;<span class="pagerange">340-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14579323">14579323</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656935">2656935</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fprot.10555">10.1002/prot.10555</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Serial BLAST searching.</h4>
<p class="authors"><span class="author">Korf I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%69%6B%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#107;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">MOTIVATION: The translating BLAST algorithms are powerful tools for finding protein-coding genes because they identify amino acid similarities in nucleotide sequences. Unfortunately, these kinds of searches are computationally intensive and often represent bottlenecks in sequence analysis pipelines. Tuning parameters for speed can make the searches much faster, but one risks losing low-scoring alignments. However, high scoring alignments are relatively resistant to such changes in parameters, and this fact makes it possible to use a serial strategy where a fast, insensitive search is used to pre-screen a database for similar sequences, and a slow, sensitive search is used to produce the sequence alignments. RESULTS: Serial BLAST searches improve both the speed and sensitivity.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2003</span>;<span class="volume">19</span>;<span class="number">12</span>;<span class="pagerange">1492-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12912829" rel="external">12912829</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The multifaceted C. elegans major sperm protein: an ephrin signaling antagonist in oocyte maturation.</h4>
<p class="authors"><span class="author">Kuwabara PE</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK CB10 1SA.</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2003</span>;<span class="volume">17</span>;<span class="number">2</span>;<span class="pagerange">155-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12533505" rel="external">12533505</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.1061103" rel="external">10.1101/gad.1061103</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular classification of synovial sarcomas, leiomyosarcomas and malignant fibrous histiocytomas by gene expression profiling.</h4>
<p class="authors"><span class="author">Lee YF</span>, <span class="author">John M</span>, <span class="author">Edwards S</span>, <span class="author">Clark J</span>, <span class="author">Flohr P</span>, <span class="author">Maillard K</span>, <span class="author">Edema M</span>, <span class="author">Baker L</span>, <span class="author">Mangham DC</span>, <span class="author">Grimer R</span>, <span class="author">Wooster R</span>, <span class="author">Thomas JM</span>, <span class="author">Fisher C</span>, <span class="author">Judson I</span> and <span class="author">Cooper CS</span></p>
<p class="affiliation">The Male Urological Cancer Research Centre, Institute of Cancer Research, Sutton, Surrey, UK. <a href="mailto:%79%66%6C%65%65%40%69%63%72.%61%63.%75%6B">&#121;&#102;&#108;&#101;&#101;&#64;&#105;&#99;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In this study, we have used genome-wide expression profiling to categorise synovial sarcomas, leiomyosarcomas and malignant fibrous histiocytomas (MFHs). Following hierarchical clustering analysis of the expression data, the best match between tumour clusters and conventional diagnosis was observed for synovial sarcomas. Eight of nine synovial sarcomas examined formed a cluster that was characterised by higher expression of a set of 48 genes. In contrast, sarcomas conventionally classified as leiomyosarcomas and MFHs did not match the clusters defined by hierarchical clustering analysis. One major cluster contained a mixture of both leiomyosarcomas and MFHs and was defined by the lower expression of a set of 202 genes. A cluster containing a subgroup of MFHs was also detected. These results may have implications for the classification of soft tissue sarcomas, and are consistent with the view that sarcomas conventionally defined as MFHs do not represent a separate diagnostic category.</p>
<p class="publication"><span class="title">British journal of cancer</span>&nbsp;<span class="year">2003</span>;<span class="volume">88</span>;<span class="number">4</span>;<span class="pagerange">510-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12592363" rel="external">12592363</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2377178" rel="external">2377178</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.bjc.6600766" rel="external">10.1038/sj.bjc.6600766</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adult midgut expressed sequence tags from the tsetse fly Glossina morsitans morsitans and expression analysis of putative immune response genes.</h4>
<p class="authors"><span class="author">Lehane MJ</span>, <span class="author">Aksoy S</span>, <span class="author">Gibson W</span>, <span class="author">Kerhornou A</span>, <span class="author">Berriman M</span>, <span class="author">Hamilton J</span>, <span class="author">Soares MB</span>, <span class="author">Bonaldo MF</span>, <span class="author">Lehane S</span> and <span class="author">Hall N</span></p>
<p class="affiliation">School of Biological Sciences, University of Wales, Bangor, LL57 2UW, UK. <a href="mailto:%6D.%6A.%6C%65%68%61%6E%65%40%62%61%6E%67%6F%72.%61%63.%75%6B">&#109;&#46;&#106;&#46;&#108;&#101;&#104;&#97;&#110;&#101;&#64;&#98;&#97;&#110;&#103;&#111;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Tsetse flies transmit African trypanosomiasis leading to half a million cases annually. Trypanosomiasis in animals (nagana) remains a massive brake on African agricultural development. While trypanosome biology is widely studied, knowledge of tsetse flies is very limited, particularly at the molecular level. This is a serious impediment to investigations of tsetse-trypanosome interactions. We have undertaken an expressed sequence tag (EST) project on the adult tsetse midgut, the major organ system for establishment and early development of trypanosomes.</p><p class="abstract"><strong>Results</strong>: A total of 21,427 ESTs were produced from the midgut of adult Glossina morsitans morsitans and grouped into 8,876 clusters or singletons potentially representing unique genes. Putative functions were ascribed to 4,035 of these by homology. Of these, a remarkable 3,884 had their most significant matches in the Drosophila protein database. We selected 68 genes with putative immune-related functions, macroarrayed them and determined their expression profiles following bacterial or trypanosome challenge. In both infections many genes are downregulated, suggesting a malaise response in the midgut. Trypanosome and bacterial challenge result in upregulation of different genes, suggesting that different recognition pathways are involved in the two responses. The most notable block of genes upregulated in response to trypanosome challenge are a series of Toll and Imd genes and a series of genes involved in oxidative stress responses.</p><p class="abstract"><strong>Conclusions</strong>: The project increases the number of known Glossina genes by two orders of magnitude. Identification of putative immunity genes and their preliminary characterization provides a resource for the experimental dissection of tsetse-trypanosome interactions.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="number">10</span>;<span class="pagerange">R63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14519198">14519198</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC328452">328452</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2003-4-10-r63">10.1186/gb-2003-4-10-r63</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cdh23 mutations in the mouse are associated with retinal dysfunction but not retinal degeneration.</h4>
<p class="authors"><span class="author">Libby RT</span>, <span class="author">Kitamoto J</span>, <span class="author">Holme RH</span>, <span class="author">Williams DS</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">MRC Institute of Hearing Research, University Park, Nottingham NG7 2RD, UK. <a href="mailto:%72%74%6C%40%6A%61%78.%6F%72%67">&#114;&#116;&#108;&#64;&#106;&#97;&#120;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Mutations in the cadherin 23 gene (CDH23) cause Usher syndrome type 1D in humans, a disease that results in retinitis pigmentosa and deafness. Cdh23 is also mutated in the waltzer mouse. In order to determine if the retina of the waltzer mouse undergoes retinal degeneration and to gain insight into the function of cadherin 23 in the retina, we have characterized the anatomy and physiology of retinas of waltzer mouse mutants. Three mutant alleles of Cdh23 were examined by histology and electroretinography (ERG). ERGs of the three Cdh23 mutant groups revealed two of them to have abnormal retinal function. One allele had a- and b-waves that were only approximately 80% of Cdh23 heterozygotes. Another allele had a significantly faster implicit time for both the a- and b-waves of the ERG. No anatomical abnormality was detected in any of the Cdh23 mutants by light microscopy. Because the mutant Cdh23 phenotype was found to be similar to the previously reported retinal phenotype of Myo7a mutant mice, the orthologue of another Usher syndrome (type 1B) gene, we generated mice that carried mutations in both genes to test for genetic interaction in the retina. No functional interaction between cadherin 23 and myosin VIIa was detected by either microscopy or ERG.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY07042</span>, <span class="grant">EY12598</span>, <span class="grant">R01 EY007042-18</span></p>
<p class="publication"><span class="title">Experimental eye research</span>&nbsp;<span class="year">2003</span>;<span class="volume">77</span>;<span class="number">6</span>;<span class="pagerange">731-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14609561" rel="external">14609561</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dispersal of NK homeobox gene clusters in amphioxus and humans.</h4>
<p class="authors"><span class="author">Luke GN</span>, <span class="author">Castro LF</span>, <span class="author">McLay K</span>, <span class="author">Bird C</span>, <span class="author">Coulson A</span> and <span class="author">Holland PW</span></p>
<p class="affiliation">School of Animal and Microbial Sciences, University of Reading, Whiteknights, Reading RG6 6AJ, United Kingdom.</p><p class="abstract">The Drosophila melanogaster genome has six physically clustered NK-related homeobox genes in just 180 kb. Here we show that the NK homeobox gene cluster was an ancient feature of bilaterian animal genomes, but has been secondarily split in chordate ancestry. The NK homeobox gene clusters of amphioxus and vertebrates are each split and dispersed at two equivalent intergenic positions. From the ancestral NK gene cluster, only the Tlx-Lbx and NK3-NK4 linkages have been retained in chordates. This evolutionary pattern is in marked contrast to the Hox and ParaHox gene clusters, which are compact in amphioxus and vertebrates, but have been disrupted in Drosophila.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2003</span>;<span class="volume">100</span>;<span class="number">9</span>;<span class="pagerange">5292-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12704239" rel="external">12704239</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC154338" rel="external">154338</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0836141100" rel="external">10.1073/pnas.0836141100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome microarrays of fission yeast: characteristics, accuracy, reproducibility, and processing of array data.</h4>
<p class="authors"><span class="author">Lyne R</span>, <span class="author">Burns G</span>, <span class="author">Mata J</span>, <span class="author">Penkett CJ</span>, <span class="author">Rustici G</span>, <span class="author">Chen D</span>, <span class="author">Langford C</span>, <span class="author">Vetrie D</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%72%61%63%68%65%6C%40%66%6C%79%6D%69%6E%65.%6F%72%67">&#114;&#97;&#99;&#104;&#101;&#108;&#64;&#102;&#108;&#121;&#109;&#105;&#110;&#101;&#46;&#111;&#114;&#103;</a></p><p class="abstract"><strong>Background</strong>: The genome of the fission yeast Schizosaccharomyces pombe has recently been sequenced, setting the stage for the post-genomic era of this increasingly popular model organism. We have built fission yeast microarrays, optimised protocols to improve array performance, and carried out experiments to assess various characteristics of microarrays.</p><p class="abstract"><strong>Results</strong>: We designed PCR primers to amplify specific probes (180-500 bp) for all known and predicted fission yeast genes, which are printed in duplicate onto separate regions of glass slides together with control elements (approximately 13,000 spots/slide). Fluorescence signal intensities depended on the size and intragenic position of the array elements, whereas the signal ratios were largely independent of element properties. Only the coding strand is covalently linked to the slides, and our array elements can discriminate transcriptional direction. The microarrays can distinguish sequences with up to 70% identity, above which cross-hybridisation contributes to the signal intensity. We tested the accuracy of signal ratios and measured the reproducibility of array data caused by biological and technical factors. Because the technical variability is lower, it is best to use samples prepared from independent biological experiments to obtain repeated measurements with swapping of fluorochromes to prevent dye bias. We also developed a script that discards unreliable data and performs a normalization to correct spatial artefacts.</p><p class="abstract"><strong>Conclusions</strong>: This paper provides data for several microarray properties that are rarely measured. The results define critical parameters for microarray design and experiments and provide a framework to optimise and interpret array data. Our arrays give reproducible and accurate expression ratios with high sensitivity. The scripts for primer design and initial data processing as well as primer sequences and detailed protocols are available from our website.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="number">1</span>;<span class="pagerange">27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12854975" rel="external">12854975</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC179895" rel="external">179895</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-4-27" rel="external">10.1186/1471-2164-4-27</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Correlations between gene expression and gene conservation in fission yeast.</h4>
<p class="authors"><span class="author">Mata J</span> and <span class="author">Bahler J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Genes can be expressed at a wide range of levels, and they show different degrees of cross-species conservation. We compared gene expression levels to gene conservation by integrating microarray data from fission yeast (Schizosaccharomyces pombe) with lists of &quot;core&quot; genes (present in worm and budding and fission yeasts), &quot;yeast-specific&quot; genes (present in budding and fission yeasts, but not in worm), and &quot;pombe-specific&quot; genes (present in fission yeast only). Whereas a disproportionate number of core genes are highly expressed in vegetatively growing cells, many pombe-specific genes are expressed at lower levels. This bias is less pronounced in cells undergoing sexual development, when many pombe-specific genes become highly expressed. This implies that organism-specific proteins are more likely to function during specialized processes such as cellular differentiation. Accordingly, pombe-specific genes were overrepresented among genes induced during sexual development; they were particularly enriched in a group of genes induced during meiotic prophase, when homologous chromosomes pair and recombine. This raises the possibility that organism-specific genes with functions in meiotic prophase favor speciation by preventing fruitful meiosis between closely related organisms. Finally, the set of genes induced late during sexual differentiation, at the time of spore formation, was enriched in yeast-specific genes, indicating that these genes play specialized roles in ascospore development.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">12</span>;<span class="pagerange">2686-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14613978" rel="external">14613978</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC403810" rel="external">403810</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1420903" rel="external">10.1101/gr.1420903</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pilot survey of expressed sequence tags (ESTs) from the asexual blood stages of Plasmodium vivax in human patients.</h4>
<p class="authors"><span class="author">Merino EF</span>, <span class="author">Fernandez-Becerra C</span>, <span class="author">Madeira AM</span>, <span class="author">Machado AL</span>, <span class="author">Durham A</span>, <span class="author">Gruber A</span>, <span class="author">Hall N</span> and <span class="author">del Portillo HA</span></p>
<p class="affiliation">Departamento de Parasitologia, ICB, Universidade de S&atilde;o Paulo, S&atilde;o Paulo, Brazil. <a href="mailto:%65%66%6D%65%72%69%6E%6F%40%79%61%68%6F%6F.%63%6F%6D">&#101;&#102;&#109;&#101;&#114;&#105;&#110;&#111;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: Plasmodium vivax is the most widely distributed human malaria, responsible for 70-80 million clinical cases each year and large socio-economical burdens for countries such as Brazil where it is the most prevalent species. Unfortunately, due to the impossibility of growing this parasite in continuous in vitro culture, research on P. vivax remains largely neglected.</p><p class="abstract"><strong>Methods</strong>: A pilot survey of expressed sequence tags (ESTs) from the asexual blood stages of P. vivax was performed. To do so, 1,184 clones from a cDNA library constructed with parasites obtained from 10 different human patients in the Brazilian Amazon were sequenced. Sequences were automatedly processed to remove contaminants and low quality reads. A total of 806 sequences with an average length of 586 bp met such criteria and their clustering revealed 666 distinct events. The consensus sequence of each cluster and the unique sequences of the singlets were used in similarity searches against different databases that included P. vivax, Plasmodium falciparum, Plasmodium yoelii, Plasmodium knowlesi, Apicomplexa and the GenBank non-redundant database. An E-value of &lt;10(-30) was used to define a significant database match. ESTs were manually assigned a gene ontology (GO) terminology</p><p class="abstract"><strong>Results</strong>: A total of 769 ESTs could be assigned a putative identity based upon sequence similarity to known proteins in GenBank. Moreover, 292 ESTs were annotated and a GO terminology was assigned to 164 of them.</p><p class="abstract"><strong>Conclusion</strong>: These are the first ESTs reported for P. vivax and, as such, they represent a valuable resource to assist in the annotation of the P. vivax genome currently being sequenced. Moreover, since the GC-content of the P. vivax genome is strikingly different from that of P. falciparum, these ESTs will help in the validation of gene predictions for P. vivax and to create a gene index of this malaria parasite.</p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2003</span>;<span class="volume">2</span>;<span class="pagerange">21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12914668" rel="external">12914668</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC183858" rel="external">183858</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-2-21" rel="external">10.1186/1475-2875-2-21</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The phusion assembler.</h4>
<p class="authors"><span class="author">Mullikin JC</span> and <span class="author">Ning Z</span></p>
<p class="affiliation">Informatics Department, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6A%63%6D%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#99;&#109;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Phusion assembler has assembled the mouse genome from the whole-genome shotgun (WGS) dataset collected by the Mouse Genome Sequencing Consortium, at ~7.5x sequence coverage, producing a high-quality draft assembly 2.6 gigabases in size, of which 90% of these bases are in 479 scaffolds. For the mouse genome, which is a large and repeat-rich genome, the input dataset was designed to include a high proportion of paired end sequences of various size selected inserts, from 2-200 kbp lengths, into various host vector templates. Phusion uses sequence data, called reads, and information about reads that share common templates, called read pairs, to drive the assembly of this large genome to highly accurate results. The preassembly stage, which clusters the reads into sensible groups, is a key element of the entire assembler, because it permits a simple approach to parallelization of the assembly stage, as each cluster can be treated independent of the others. In addition to the application of Phusion to the mouse genome, we will also present results from the WGS assembly of Caenorhabditis briggsae sequenced to about 11x coverage. The C. briggsae assembly was accessioned through EMBL, http://www.ebi.ac.uk/services/index.html, using the series CAAC01000001-CAAC01000578, however, the Phusion mouse assembly described here was not accessioned. The mouse data was generated by the Mouse Genome Sequencing Consortium. The C. briggsae sequence was generated at The Wellcome Trust Sanger Institute and the Genome Sequencing Center, Washington University School of Medicine.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">81-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12529309">12529309</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC430959">430959</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.731003">10.1101/gr.731003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DNA sequence and analysis of human chromosome 6.</h4>
<p class="authors"><span class="author">Mungall AJ</span>, <span class="author">Palmer SA</span>, <span class="author">Sims SK</span>, <span class="author">Edwards CA</span>, <span class="author">Ashurst JL</span>, <span class="author">Wilming L</span>, <span class="author">Jones MC</span>, <span class="author">Horton R</span>, <span class="author">Hunt SE</span>, <span class="author">Scott CE</span>, <span class="author">Gilbert JG</span>, <span class="author">Clamp ME</span>, <span class="author">Bethel G</span>, <span class="author">Milne S</span>, <span class="author">Ainscough R</span>, <span class="author">Almeida JP</span>, <span class="author">Ambrose KD</span>, <span class="author">Andrews TD</span>, <span class="author">Ashwell RI</span>, <span class="author">Babbage AK</span>, <span class="author">Bagguley CL</span>, <span class="author">Bailey J</span>, <span class="author">Banerjee R</span>, <span class="author">Barker DJ</span>, <span class="author">Barlow KF</span>, <span class="author">Bates K</span>, <span class="author">Beare DM</span>, <span class="author">Beasley H</span>, <span class="author">Beasley O</span>, <span class="author">Bird CP</span>, <span class="author">Blakey S</span>, <span class="author">Bray-Allen S</span>, <span class="author">Brook J</span>, <span class="author">Brown AJ</span>, <span class="author">Brown JY</span>, <span class="author">Burford DC</span>, <span class="author">Burrill W</span>, <span class="author">Burton J</span>, <span class="author">Carder C</span>, <span class="author">Carter NP</span>, <span class="author">Chapman JC</span>, <span class="author">Clark SY</span>, <span class="author">Clark G</span>, <span class="author">Clee CM</span>, <span class="author">Clegg S</span>, <span class="author">Cobley V</span>, <span class="author">Collier RE</span>, <span class="author">Collins JE</span>, <span class="author">Colman LK</span>, <span class="author">Corby NR</span>, <span class="author">Coville GJ</span>, <span class="author">Culley KM</span>, <span class="author">Dhami P</span>, <span class="author">Davies J</span>, <span class="author">Dunn M</span>, <span class="author">Earthrowl ME</span>, <span class="author">Ellington AE</span>, <span class="author">Evans KA</span>, <span class="author">Faulkner L</span>, <span class="author">Francis MD</span>, <span class="author">Frankish A</span>, <span class="author">Frankland J</span>, <span class="author">French L</span>, <span class="author">Garner P</span>, <span class="author">Garnett J</span>, <span class="author">Ghori MJ</span>, <span class="author">Gilby LM</span>, <span class="author">Gillson CJ</span>, <span class="author">Glithero RJ</span>, <span class="author">Grafham DV</span>, <span class="author">Grant M</span>, <span class="author">Gribble S</span>, <span class="author">Griffiths C</span>, <span class="author">Griffiths M</span>, <span class="author">Hall R</span>, <span class="author">Halls KS</span>, <span class="author">Hammond S</span>, <span class="author">Harley JL</span>, <span class="author">Hart EA</span>, <span class="author">Heath PD</span>, <span class="author">Heathcott R</span>, <span class="author">Holmes SJ</span>, <span class="author">Howden PJ</span>, <span class="author">Howe KL</span>, <span class="author">Howell GR</span>, <span class="author">Huckle E</span>, <span class="author">Humphray SJ</span>, <span class="author">Humphries MD</span>, <span class="author">Hunt AR</span>, <span class="author">Johnson CM</span>, <span class="author">Joy AA</span>, <span class="author">Kay M</span>, <span class="author">Keenan SJ</span>, <span class="author">Kimberley AM</span>, <span class="author">King A</span>, <span class="author">Laird GK</span>, <span class="author">Langford C</span>, <span class="author">Lawlor S</span>, <span class="author">Leongamornlert DA</span>, <span class="author">Leversha M</span>, <span class="author">Lloyd CR</span>, <span class="author">Lloyd DM</span>, <span class="author">Loveland JE</span>, <span class="author">Lovell J</span>, <span class="author">Martin S</span>, <span class="author">Mashreghi-Mohammadi M</span>, <span class="author">Maslen GL</span>, <span class="author">Matthews L</span>, <span class="author">McCann OT</span>, <span class="author">McLaren SJ</span>, <span class="author">McLay K</span>, <span class="author">McMurray A</span>, <span class="author">Moore MJ</span>, <span class="author">Mullikin JC</span>, <span class="author">Niblett D</span>, <span class="author">Nickerson T</span>, <span class="author">Novik KL</span>, <span class="author">Oliver K</span>, <span class="author">Overton-Larty EK</span>, <span class="author">Parker A</span>, <span class="author">Patel R</span>, <span class="author">Pearce AV</span>, <span class="author">Peck AI</span>, <span class="author">Phillimore B</span>, <span class="author">Phillips S</span>, <span class="author">Plumb RW</span>, <span class="author">Porter KM</span>, <span class="author">Ramsey Y</span>, <span class="author">Ranby SA</span>, <span class="author">Rice CM</span>, <span class="author">Ross MT</span>, <span class="author">Searle SM</span>, <span class="author">Sehra HK</span>, <span class="author">Sheridan E</span>, <span class="author">Skuce CD</span>, <span class="author">Smith S</span>, <span class="author">Smith M</span>, <span class="author">Spraggon L</span>, <span class="author">Squares SL</span>, <span class="author">Steward CA</span>, <span class="author">Sycamore N</span>, <span class="author">Tamlyn-Hall G</span>, <span class="author">Tester J</span>, <span class="author">Theaker AJ</span>, <span class="author">Thomas DW</span>, <span class="author">Thorpe A</span>, <span class="author">Tracey A</span>, <span class="author">Tromans A</span>, <span class="author">Tubby B</span>, <span class="author">Wall M</span>, <span class="author">Wallis JM</span>, <span class="author">West AP</span>, <span class="author">White SS</span>, <span class="author">Whitehead SL</span>, <span class="author">Whittaker H</span>, <span class="author">Wild A</span>, <span class="author">Willey DJ</span>, <span class="author">Wilmer TE</span>, <span class="author">Wood JM</span>, <span class="author">Wray PW</span>, <span class="author">Wyatt JC</span>, <span class="author">Young L</span>, <span class="author">Younger RM</span>, <span class="author">Bentley DR</span>, <span class="author">Coulson A</span>, <span class="author">Durbin R</span>, <span class="author">Hubbard T</span>, <span class="author">Sulston JE</span>, <span class="author">Dunham I</span>, <span class="author">Rogers J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%6A%6D%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#106;&#109;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chromosome 6 is a metacentric chromosome that constitutes about 6% of the human genome. The finished sequence comprises 166,880,988 base pairs, representing the largest chromosome sequenced so far. The entire sequence has been subjected to high-quality manual annotation, resulting in the evidence-supported identification of 1,557 genes and 633 pseudogenes. Here we report that at least 96% of the protein-coding genes have been identified, as assessed by multi-species comparative sequence analysis, and provide evidence for the presence of further, otherwise unsupported exons/genes. Among these are genes directly implicated in cancer, schizophrenia, autoimmunity and many other diseases. Chromosome 6 harbours the largest transfer RNA gene cluster in the genome; we show that this cluster co-localizes with a region of high transcriptional activity. Within the essential immune loci of the major histocompatibility complex, we find HLA-B to be the most polymorphic gene on chromosome 6 and in the human genome.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">425</span>;<span class="number">6960</span>;<span class="pagerange">805-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14574404">14574404</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02055">10.1038/nature02055</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A table-driven, full-sensitivity similarity search algorithm.</h4>
<p class="authors"><span class="author">Myers G</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Department of Computer Science, University of California, Berkeley, Berkeley, CA 94720-1776, USA. <a href="mailto:%67%65%6E%65%40%63%73.%62%65%72%6B%65%6C%65%79.%65%64%75">&#103;&#101;&#110;&#101;&#64;&#99;&#115;&#46;&#98;&#101;&#114;&#107;&#101;&#108;&#101;&#121;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Searching a database for a local alignment to a query under a typical scoring scheme, such as PAM120 or BLOSUM62 with affine gap costs, is a computation that has resisted algorithmic improvement due to its basis in dynamic programming and the weak nature of the signals being searched for. In a query preprocessing step, a set of tables can be built that permit one to (a) eliminate a large fraction of the dynamic programming matrix from consideration and (b) to compute several steps of the remainder with a single table lookup. While this result is not an asymptotic improvement over the original Smith-Waterman algorithm, its complexity is characterized in terms of some sparse features of the matrix and it yields the fastest software implementation to date for such searches.</p>
<p class="publication"><span class="title">Journal of computational biology : a journal of computational molecular cell biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">10</span>;<span class="number">2</span>;<span class="pagerange">103-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12804086" rel="external">12804086</a>; DOI: <a href="http://dx.doi.org/10.1089%2F106652703321825919" rel="external">10.1089/106652703321825919</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of putative noncoding RNAs among the RIKEN mouse full-length cDNA collection.</h4>
<p class="authors"><span class="author">Numata K</span>, <span class="author">Kanai A</span>, <span class="author">Saito R</span>, <span class="author">Kondo S</span>, <span class="author">Adachi J</span>, <span class="author">Wilming LG</span>, <span class="author">Hume DA</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Tomita M</span>, <span class="author">RIKEN GER Group</span> and <span class="author">GSL Members</span></p>
<p class="affiliation">Graduate School of Media and Governance, Bioinformatics Program, Keio University, Fujisawa, Kanagawa 252-8520, Japan.</p><p class="abstract">With the sequencing and annotation of genomes and transcriptomes of several eukaryotes, the importance of noncoding RNA (ncRNA)-RNA molecules that are not translated to protein products-has become more evident. A subclass of ncRNA transcripts are encoded by highly regulated, multi-exon, transcriptional units, are processed like typical protein-coding mRNAs and are increasingly implicated in regulation of many cellular functions in eukaryotes. This study describes the identification of candidate functional ncRNAs from among the RIKEN mouse full-length cDNA collection, which contains 60,770 sequences, by using a systematic computational filtering approach. We initially searched for previously reported ncRNAs and found nine murine ncRNAs and homologs of several previously described nonmouse ncRNAs. Through our computational approach to filter artifact-free clones that lack protein coding potential, we extracted 4280 transcripts as the largest-candidate set. Many clones in the set had EST hits, potential CpG islands surrounding the transcription start sites, and homologies with the human genome. This implies that many candidates are indeed transcribed in a regulated manner. Our results demonstrate that ncRNAs are a major functional subclass of processed transcripts in mammals.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">13</span>;<span class="number">6B</span>;<span class="pagerange">1301-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12819127">12819127</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC403720">403720</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1011603">10.1101/gr.1011603</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Y chromosomal influence on prostate cancer risk: the multi-ethnic cohort study.</h4>
<p class="authors"><span class="author">Paracchini S</span>, <span class="author">Pearce CL</span>, <span class="author">Kolonel LN</span>, <span class="author">Altshuler D</span>, <span class="author">Henderson BE</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Department of Biochemistry, University of Oxford, South Parks Road, Oxford OX1 3QU, UK.</p><p class="abstract"><strong>Background</strong>: A Y chromosomal role in prostate cancer has previously been suggested by both cytogenetic findings and patterns of Y chromosomal gene expression. We took advantage of the well established and stable phylogeny of the non-recombining segment of the Y chromosome to investigate the association between Y chromosomal DNA variation and prostate cancer risk.</p><p class="abstract"><strong>Methods</strong>: We examined the distribution of 116 Y lineages in 930 prostate cancer cases and 1208 controls from four ethnic groups from a cohort study in Hawaii and California.</p><p class="abstract"><strong>Results</strong>: One lineage, found only among the Japanese group in our study, was associated with a statistically significant predisposition to prostate cancer (odds ratio (OR) = 1.63; 95% confidence interval (CI) 1.07 to 2.47), and, in particular, to high severity disease in younger individuals (OR = 3.89; 95% CI 1.34 to 11.31).</p><p class="abstract"><strong>Conclusions</strong>: This finding suggests that a Y chromosomal factor contributes significantly to the development of prostate cancer in Japanese men.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01 CA54281</span>, <span class="grant">R01 CA63464</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">40</span>;<span class="number">11</span>;<span class="pagerange">815-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14627670" rel="external">14627670</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735314" rel="external">1735314</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics: Relative pathogenic values.</h4>
<p class="authors"><span class="author">Parkhill J</span> and <span class="author">Berry C</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">423</span>;<span class="number">6935</span>;<span class="pagerange">23-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12721608">12721608</a>; DOI: <a href="http://dx.doi.org/10.1038%2F423023a">10.1038/423023a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary strategies of human pathogens.</h4>
<p class="authors"><span class="author">Parkhill J</span> and <span class="author">Thomson N</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kingdom.</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">68</span>;<span class="pagerange">151-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15338613" rel="external">15338613</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative analysis of the genome sequences of Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica.</h4>
<p class="authors"><span class="author">Parkhill J</span>, <span class="author">Sebaihia M</span>, <span class="author">Preston A</span>, <span class="author">Murphy LD</span>, <span class="author">Thomson N</span>, <span class="author">Harris DE</span>, <span class="author">Holden MT</span>, <span class="author">Churcher CM</span>, <span class="author">Bentley SD</span>, <span class="author">Mungall KL</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Temple L</span>, <span class="author">James K</span>, <span class="author">Harris B</span>, <span class="author">Quail MA</span>, <span class="author">Achtman M</span>, <span class="author">Atkin R</span>, <span class="author">Baker S</span>, <span class="author">Basham D</span>, <span class="author">Bason N</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth T</span>, <span class="author">Collins M</span>, <span class="author">Cronin A</span>, <span class="author">Davis P</span>, <span class="author">Doggett J</span>, <span class="author">Feltwell T</span>, <span class="author">Goble A</span>, <span class="author">Hamlin N</span>, <span class="author">Hauser H</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">Leather S</span>, <span class="author">Moule S</span>, <span class="author">Norberczak H</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Ormond D</span>, <span class="author">Price C</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rutter S</span>, <span class="author">Sanders M</span>, <span class="author">Saunders D</span>, <span class="author">Seeger K</span>, <span class="author">Sharp S</span>, <span class="author">Simmonds M</span>, <span class="author">Skelton J</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Stevens K</span>, <span class="author">Unwin L</span>, <span class="author">Whitehead S</span>, <span class="author">Barrell BG</span> and <span class="author">Maskell DJ</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%70%61%72%6B%68%69%6C%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#114;&#107;&#104;&#105;&#108;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica are closely related Gram-negative beta-proteobacteria that colonize the respiratory tracts of mammals. B. pertussis is a strict human pathogen of recent evolutionary origin and is the primary etiologic agent of whooping cough. B. parapertussis can also cause whooping cough, and B. bronchiseptica causes chronic respiratory infections in a wide range of animals. We sequenced the genomes of B. bronchiseptica RB50 (5,338,400 bp; 5,007 predicted genes), B. parapertussis 12822 (4,773,551 bp; 4,404 genes) and B. pertussis Tohama I (4,086,186 bp; 3,816 genes). Our analysis indicates that B. parapertussis and B. pertussis are independent derivatives of B. bronchiseptica-like ancestors. During the evolution of these two host-restricted species there was large-scale gene loss and inactivation; host adaptation seems to be a consequence of loss, not gain, of function, and differences in virulence may be related to loss of regulatory or control functions.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">35</span>;<span class="number">1</span>;<span class="pagerange">32-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12910271">12910271</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1227">10.1038/ng1227</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">400000 nematode ESTs on the Net.</h4>
<p class="authors"><span class="author">Parkinson J</span>, <span class="author">Mitreva M</span>, <span class="author">Hall N</span>, <span class="author">Blaxter M</span> and <span class="author">McCarter JP</span></p>
<p class="affiliation">Institute of Cell, Animal and Population Biology, Ashworth Laboratories, King&#39;s Buildings, West Mains Rd, University of Edinburgh, EH9 3JT, Edinburgh, UK. <a href="mailto:%6A%6F%68%6E.%70%61%72%6B%69%6E%73%6F%6E%40%65%64.%61%63.%75%6B">&#106;&#111;&#104;&#110;&#46;&#112;&#97;&#114;&#107;&#105;&#110;&#115;&#111;&#110;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a> &lt;<a href="mailto:%6A%6F%68%6E.%70%61%72%6B%69%6E%73%6F%6E%40%65%64.%61%63.%75%6B&%67%74">&#106;&#111;&#104;&#110;&#46;&#112;&#97;&#114;&#107;&#105;&#110;&#115;&#111;&#110;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;&amp;&#103;&#116;</a>;</p><p class="abstract">The parasitic nematode expressed sequence tag (EST) project, a collaboration between University of Edinburgh and the Wellcome Trust Sanger Institute in the UK and the Genome Sequencing Center, St Louis, MO, USA, is currently generating sequence information from &gt;30 different species of nematode. Over 400000 nematode ESTs are now available and at least another 130000 are planned. Here, an update is provided on the status of the project and describes the database tools being developed to disseminate these data.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 46593</span></p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2003</span>;<span class="volume">19</span>;<span class="number">7</span>;<span class="pagerange">283-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12855373" rel="external">12855373</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple inverted DNA repeats of Bacteroides fragilis that control polysaccharide antigenic variation are similar to the hin region inverted repeats of Salmonella typhimurium.</h4>
<p class="authors"><span class="author">Patrick S</span>, <span class="author">Parkhill J</span>, <span class="author">McCoy LJ</span>, <span class="author">Lennard N</span>, <span class="author">Larkin MJ</span>, <span class="author">Collins M</span>, <span class="author">Sczaniecka M</span> and <span class="author">Blakely G</span></p>
<p class="affiliation">Microbiology and Immunobiology, The Queen&#39;s University of Belfast, Grosvenor Road, Belfast BT12 6BN, UK. <a href="mailto:%73.%70%61%74%72%69%63%6B%40%71%75%62.%61%63.%75%6B">&#115;&#46;&#112;&#97;&#116;&#114;&#105;&#99;&#107;&#64;&#113;&#117;&#98;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The important opportunistic pathogen Bacteroides fragilis is a strictly anaerobic Gram-negative bacterium and a member of the normal resident human gastrointestinal microbiota. Our earlier studies indicated that there is considerable within-strain variation in polysaccharide expression, as detected by mAb labelling. Analysis of the genome sequence has revealed multiple invertible DNA regions, designated fragilis invertible (fin) regions, seven of which are upstream of polysaccharide biosynthesis loci and are approximately 226 bp in size. Using orientation-specific PCR primers and sequence analysis with populations enriched for one antigenic type, two of these invertible regions were assigned to heteropolymeric polysaccharides with different sizes of repeating units, as determined by PAGE pattern. The implication of these findings is that inversion of the fin regions switches biosynthesis of these polysaccharides off and on. The invertible regions are bound by inverted repeats of 30 or 32 bp with striking similarity to the Salmonella typhimurium H flagellar antigen inversion cross-over (hix) recombination sites of the invertible hin region. It has been demonstrated that a plasmid-encoded Hin invertase homologue (FinB), present in B. fragilis NCTC 9343, binds specifically to the invertible regions and the recombination sites have been designated as fragilis inversion cross-over (fix) sites.</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2003</span>;<span class="volume">149</span>;<span class="number">Pt 4</span>;<span class="pagerange">915-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12686634" rel="external">12686634</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of a structurally distinct CD101 molecule encoded in the 950-kb Idd10 region of NOD mice.</h4>
<p class="authors"><span class="author">Penha-Gon&ccedil;alves C</span>, <span class="author">Moule C</span>, <span class="author">Smink LJ</span>, <span class="author">Howson J</span>, <span class="author">Gregory S</span>, <span class="author">Rogers J</span>, <span class="author">Lyons PA</span>, <span class="author">Suttie JJ</span>, <span class="author">Lord CJ</span>, <span class="author">Peterson LB</span>, <span class="author">Todd JA</span> and <span class="author">Wicker LS</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust (JDRF/WT) Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge CB2 2XY, U.K.</p><p class="abstract">Genes affecting autoimmune type 1 diabetes susceptibility in the nonobese diabetic (NOD) mouse (Idd loci) have been mapped using a congenic strain breeding strategy. In the present study, we used a combination of BAC clone contig construction, polymorphism analysis of DNA from congenic strains, and sequence mining of the human orthologous region to generate an integrated map of the Idd10 region on mouse chromosome 3. We found seven genes and one pseudogene in the 950-kb Idd10 region. Although all seven genes in the interval are Idd10 candidates, we suggest the gene encoding the EWI immunoglobulin subfamily member EWI-101 (Cd101) as the most likely Idd10 candidate because of the previously reported immune-associated properties of the human CD101 molecule. Additional support for the candidacy of Cd101 is the presence of 17 exonic single-nucleotide polymorphisms that differ between the NOD and B6 sequences, 10 causing amino acid substitutions in the predicted CD101 protein. Four of these 10 substitutions are nonconservative, 2 of which could potentially alter N-linked glycosylation. Considering our results together with those previous reports that antibodies recognizing human CD101 modulate human T-cell and dendritic cell function, there is now justification to test whether the alteration of CD101 function affects autoimmune islet destruction.</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2003</span>;<span class="volume">52</span>;<span class="number">6</span>;<span class="pagerange">1551-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12765969">12765969</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Composition, acquisition, and distribution of the Vi exopolysaccharide-encoding Salmonella enterica pathogenicity island SPI-7.</h4>
<p class="authors"><span class="author">Pickard D</span>, <span class="author">Wain J</span>, <span class="author">Baker S</span>, <span class="author">Line A</span>, <span class="author">Chohan S</span>, <span class="author">Fookes M</span>, <span class="author">Barron A</span>, <span class="author">Gaora PO</span>, <span class="author">Chabalgoity JA</span>, <span class="author">Thanky N</span>, <span class="author">Scholes C</span>, <span class="author">Thomson N</span>, <span class="author">Quail M</span>, <span class="author">Parkhill J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Centre for Molecular Microbiology and Infection, Department of Biological Sciences, Imperial College of Science, Technology and Medicine, Armstrong Road, London SW7 2AZ, UK. <a href="mailto:%64.%70%69%63%6B%61%72%64%40%69%6D%70%65%72%69%61%6C.%61%63.%75%6B">&#100;&#46;&#112;&#105;&#99;&#107;&#97;&#114;&#100;&#64;&#105;&#109;&#112;&#101;&#114;&#105;&#97;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Vi capsular polysaccharide production is encoded by the viaB locus, which has a limited distribution in Salmonella enterica serovars. In S. enterica serovar Typhi, viaB is encoded on a 134-kb pathogenicity island known as SPI-7 that is located between partially duplicated tRNA(pheU) sites. Functional and bioinformatic analysis suggests that SPI-7 has a mosaic structure and may have evolved as a consequence of several independent insertion events. Analysis of viaB-associated DNA in Vi-positive S. enterica serovar Paratyphi C and S. enterica serovar Dublin isolates revealed the presence of similar SPI-7 islands. In S. enterica serovars Paratyphi C and Dublin, the SopE bacteriophage and a 15-kb fragment adjacent to the intact tRNA(pheU) site were absent. In S. enterica serovar Paratyphi C only, a region encoding a type IV pilus involved in the adherence of S. enterica serovar Typhi to host cells was missing. The remainder of the SPI-7 islands investigated exhibited over 99% DNA sequence identity in the three serovars. Of 30 other Salmonella serovars examined, 24 contained no insertions at the equivalent tRNA(pheU) site, 2 had a 3.7-kb insertion, and 4 showed sequence variation at the tRNA(pheU)-phoN junction, which was not analyzed further. Sequence analysis of the SPI-7 region from S. enterica serovar Typhi strain CT18 revealed significant synteny with clusters of genes from a variety of saprophytic bacteria and phytobacteria, including Pseudomonas aeruginosa and Xanthomonas axonopodis pv. citri. This analysis suggested that SPI-7 may be a mobile element, such as a conjugative transposon or an integrated plasmid remnant.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2003</span>;<span class="volume">185</span>;<span class="number">17</span>;<span class="pagerange">5055-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12923078" rel="external">12923078</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC180996" rel="external">180996</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cobalamin synthesis in Yersinia enterocolitica 8081. Functional aspects of a putative metabolic island.</h4>
<p class="authors"><span class="author">Prentice MB</span>, <span class="author">Cuccui J</span>, <span class="author">Thomson N</span>, <span class="author">Parkhill J</span>, <span class="author">Deery E</span> and <span class="author">Warren MJ</span></p>
<p class="affiliation">Bart&#39;s and the London Medical School, London EC1A 7BE, UK.</p>
<p class="publication"><span class="title">Advances in experimental medicine and biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">529</span>;<span class="pagerange">43-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12756726" rel="external">12756726</a>; DOI: <a href="http://dx.doi.org/10.1007%2F0-306-48416-1_8" rel="external">10.1007/0-306-48416-1_8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transgenics at breaking-point.</h4>
<p class="authors"><span class="author">Prosser H</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">In this issue of Cancer Cell, Forster et al. (2003) have generated mice that recapitulate both the mechanism (sporadic somatic translocation) and the consequences (expression of two translocation fusion genes) leading to an accurate leukemia model.</p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2003</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">411-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12781356" rel="external">12781356</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Manipulation of the mouse genome: a multiple impact resource for drug discovery and development.</h4>
<p class="authors"><span class="author">Prosser H</span> and <span class="author">Rastan S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA UK.</p><p class="abstract">Few would deny that the pharmaceutical industry&#39;s investment in genomics throughout the 1990s has yet to deliver in terms of drugs on the market. The reasons are complex and beyond the scope of this review. The unique ability to manipulate the mouse genome, however, has already had a positive impact on all stages of the drug discovery process and, increasingly, on the drug development process too. We give an overview of some recent applications of so-called &#39;transgenic&#39; mouse technology in pharmaceutical research and development. We show how genetic manipulation in the mouse can be employed at multiple points in the drug discovery and development process, providing new solutions to old problems.</p>
<p class="publication"><span class="title">Trends in biotechnology</span>&nbsp;<span class="year">2003</span>;<span class="volume">21</span>;<span class="number">5</span>;<span class="pagerange">224-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12727384" rel="external">12727384</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0167-7799(03)00087-8" rel="external">10.1016/S0167-7799(03)00087-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">OXBench: a benchmark for evaluation of protein multiple sequence alignment accuracy.</h4>
<p class="authors"><span class="author">Raghava GP</span>, <span class="author">Searle SM</span>, <span class="author">Audley PC</span>, <span class="author">Barber JD</span> and <span class="author">Barton GJ</span></p>
<p class="affiliation">School of Life Sciences, University of Dundee, Dow St, Dundee, DD1 5EH, Scotland, UK. <a href="mailto:%72%61%67%68%61%76%61%40%69%6D%74%65%63%68.%72%65%73.%69%6E">&#114;&#97;&#103;&#104;&#97;&#118;&#97;&#64;&#105;&#109;&#116;&#101;&#99;&#104;&#46;&#114;&#101;&#115;&#46;&#105;&#110;</a></p><p class="abstract"><strong>Background</strong>: The alignment of two or more protein sequences provides a powerful guide in the prediction of the protein structure and in identifying key functional residues, however, the utility of any prediction is completely dependent on the accuracy of the alignment. In this paper we describe a suite of reference alignments derived from the comparison of protein three-dimensional structures together with evaluation measures and software that allow automatically generated alignments to be benchmarked. We test the OXBench benchmark suite on alignments generated by the AMPS multiple alignment method, then apply the suite to compare eight different multiple alignment algorithms. The benchmark shows the current state-of-the art for alignment accuracy and provides a baseline against which new alignment algorithms may be judged.</p><p class="abstract"><strong>Results</strong>: The simple hierarchical multiple alignment algorithm, AMPS, performed as well as or better than more modern methods such as CLUSTALW once the PAM250 pair-score matrix was replaced by a BLOSUM series matrix. AMPS gave an accuracy in Structurally Conserved Regions (SCRs) of 89.9% over a set of 672 alignments. The T-COFFEE method on a data set of families with &lt;8 sequences gave 91.4% accuracy, significantly better than CLUSTALW (88.9%) and all other methods considered here. The complete suite is available from http://www.compbio.dundee.ac.uk.</p><p class="abstract"><strong>Conclusions</strong>: The OXBench suite of reference alignments, evaluation software and results database provide a convenient method to assess progress in sequence alignment techniques. Evaluation measures that were dependent on comparison to a reference alignment were found to give good discrimination between methods. The STAMP Sc Score which is independent of a reference alignment also gave good discrimination. Application of OXBench in this paper shows that with the exception of T-COFFEE, the majority of the improvement in alignment accuracy seen since 1985 stems from improved pair-score matrices rather than algorithmic refinements. The maximum theoretical alignment accuracy obtained by pooling results over all methods was 94.5% with 52.5% accuracy for alignments in the 0-10 percentage identity range. This suggests that further improvements in accuracy will be possible in the future.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="pagerange">47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14552658" rel="external">14552658</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC280650" rel="external">280650</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-4-47" rel="external">10.1186/1471-2105-4-47</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ehlers-Danlos syndrome with severe early-onset periodontal disease (EDS-VIII) is a distinct, heterogeneous disorder with one predisposition gene at chromosome 12p13.</h4>
<p class="authors"><span class="author">Rahman N</span>, <span class="author">Dunstan M</span>, <span class="author">Teare MD</span>, <span class="author">Hanks S</span>, <span class="author">Douglas J</span>, <span class="author">Coleman K</span>, <span class="author">Bottomly WE</span>, <span class="author">Campbell ME</span>, <span class="author">Berglund B</span>, <span class="author">Nordenskj&ouml;ld M</span>, <span class="author">Forssell B</span>, <span class="author">Burrows N</span>, <span class="author">Lunt P</span>, <span class="author">Young I</span>, <span class="author">Williams N</span>, <span class="author">Bignell GR</span>, <span class="author">Futreal PA</span> and <span class="author">Pope FM</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, Brooks-Lawley Building, 15 Cotswold Road, Sutton, Surrey SM2 5NG, United Kingdom. <a href="mailto:%6E%61%7A%6E%65%65%6E%40%69%63%72.%61%63.%75%6B">&#110;&#97;&#122;&#110;&#101;&#101;&#110;&#64;&#105;&#99;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ehlers-Danlos VIII (EDS-VIII) is an autosomal dominant disorder characterized by severe early-onset periodontal disease in conjunction with the features of Ehlers-Danlos syndrome (EDS). We performed a genomewide linkage search in a large Swedish pedigree with EDS-VIII and established linkage to a 7-cM interval on chromosome 12p13, generating a maximum multipoint LOD score of 5.17. Analysis of four further pedigrees with EDS-VIII revealed two consistent with linkage to 12p13 and two in which linkage could be excluded, indicating that EDS-VIII is a genetically heterogeneous disorder. Chromosome 12p13 has not previously been implicated in either EDS or periodontal disease and contains no known collagen genes or collagen-processing enzymes. Mutational screening of the microfibril-associated glycoprotein-2 gene, a strong candidate within the minimal interval, did not reveal any likely pathogenic mutations.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">73</span>;<span class="number">1</span>;<span class="pagerange">198-204</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12776252" rel="external">12776252</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1180581" rel="external">1180581</a>; DOI: <a href="http://dx.doi.org/10.1086%2F376416" rel="external">10.1086/376416</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Determination of Escherichia coli RNA polymerase structure by single particle cryoelectron microscopy.</h4>
<p class="authors"><span class="author">Ray P</span>, <span class="author">Klaholz BP</span>, <span class="author">Finn RD</span>, <span class="author">Orlova EV</span>, <span class="author">Burrows PC</span>, <span class="author">Gowen B</span>, <span class="author">Buck M</span> and <span class="author">van Heel M</span></p>
<p class="affiliation">Department of Biological Sciences, Wolfson Laboratories, Imperial College of London, Rm. 313, London SW7 2AY, United Kingdom.</p>
<p class="publication"><span class="title">Methods in enzymology</span>&nbsp;<span class="year">2003</span>;<span class="volume">370</span>;<span class="pagerange">24-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14712631" rel="external">14712631</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The finished genome sequence of Homo sapiens.</h4>
<p class="authors"><span class="author">Rogers J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">68</span>;<span class="pagerange">1-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15338597" rel="external">15338597</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variants in CHEK2 other than 1100delC do not make a major contribution to breast cancer susceptibility.</h4>
<p class="authors"><span class="author">Schutte M</span>, <span class="author">Seal S</span>, <span class="author">Barfoot R</span>, <span class="author">Meijers-Heijboer H</span>, <span class="author">Wasielewski M</span>, <span class="author">Evans DG</span>, <span class="author">Eccles D</span>, <span class="author">Meijers C</span>, <span class="author">Lohman F</span>, <span class="author">Klijn J</span>, <span class="author">van den Ouweland A</span>, <span class="author">Futreal PA</span>, <span class="author">Nathanson KL</span>, <span class="author">Weber BL</span>, <span class="author">Easton DF</span>, <span class="author">Stratton MR</span>, <span class="author">Rahman N</span> and <span class="author">Breast Cancer Linkage Consortium</span></p>
<p class="affiliation">Department of Medical Oncology, Erasmus Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">We recently reported that a sequence variant in the cell-cycle-checkpoint kinase CHEK2 (CHEK2 1100delC) is a low-penetrance breast cancer-susceptibility allele in noncarriers of BRCA1 or BRCA2 mutations. To investigate whether other CHEK2 variants confer susceptibility to breast cancer, we screened the full CHEK2 coding sequence in BRCA1/2-negative breast cancer cases from 89 pedigrees with three or more cases of breast cancer. We identified one novel germline variant, R117G, in two separate families. To evaluate the possible association of R117G and two germline variants reported elsewhere, R145W and I157T with breast cancer, we screened 737 BRCA1/2-negative familial breast cancer cases from 605 families, 459 BRCA1/2-positive cases from 335 families, and 723 controls from the United Kingdom, the Netherlands, and North America. All three variants were rare in all groups, and none occurred at significantly elevated frequency in familial breast cancer cases compared with controls. These results indicate that 1100delC may be the only CHEK2 allele that makes an appreciable contribution to breast cancer susceptibility.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">72</span>;<span class="number">4</span>;<span class="pagerange">1023-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12610780" rel="external">12610780</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1180332" rel="external">1180332</a>; DOI: <a href="http://dx.doi.org/10.1086%2F373965" rel="external">10.1086/373965</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluation of Fanconi Anemia genes in familial breast cancer predisposition.</h4>
<p class="authors"><span class="author">Seal S</span>, <span class="author">Barfoot R</span>, <span class="author">Jayatilake H</span>, <span class="author">Smith P</span>, <span class="author">Renwick A</span>, <span class="author">Bascombe L</span>, <span class="author">McGuffog L</span>, <span class="author">Evans DG</span>, <span class="author">Eccles D</span>, <span class="author">Easton DF</span>, <span class="author">Stratton MR</span>, <span class="author">Rahman N</span> and <span class="author">Breast Cancer Susceptibility Collaboration</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, United Kingdom.</p><p class="abstract">Fanconi Anemia (FA) is an autosomal recessive syndrome characterized by congenital abnormalities, progressive bone marrow failure, and susceptibility to cancer. FA has eight known complementation groups and is caused by mutations in at least seven genes. Biallelic BRCA2 mutations were shown recently to cause FA-D1. Monoallelic (heterozygous) BRCA2 mutations confer a high risk of breast cancer and are a major cause of familial breast cancer. To investigate whether heterozygous variants in other FA genes are high penetrance breast cancer susceptibility alleles, we screened germ-line DNA from 88 BRCA1/2-negative families, each with at least three cases of breast cancer, for mutations in FANCA, FANCC, FANCD2, FANCE, FANCF, and FANCG. Sixty-nine sequence variants were identified of which 25 were exonic. None of the exonic variants resulted in translational frameshifts or nonsense codons and 14 were polymorphisms documented previously. Of the remaining 11 exonic variants, 2 resulted in synonymous changes, and 7 were present in controls. Only 2 conservative missense variants, 1 in FANCA and 1 in FANCE, were each found in a single family and were not present in 300 controls. The results indicate that FA gene mutations, other than in BRCA2, are unlikely to be a frequent cause of highly penetrant breast cancer predisposition.</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2003</span>;<span class="volume">63</span>;<span class="number">24</span>;<span class="pagerange">8596-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14695169" rel="external">14695169</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A bad combination.</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Bentley S</span>, <span class="author">Crossman L</span>, <span class="author">Thomson N</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%73%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#115;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">7</span>;<span class="pagerange">297-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12875811" rel="external">12875811</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The good, the bad and the ugly?</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Bentley SD</span>, <span class="author">Holden MT</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">5</span>;<span class="pagerange">204-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12781519" rel="external">12781519</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction and integration of radiation-hybrid and cytogenetic maps of dog Chromosome X.</h4>
<p class="authors"><span class="author">Spriggs HF</span>, <span class="author">Holmes NG</span>, <span class="author">Breen MG</span>, <span class="author">Deloukas PG</span>, <span class="author">Langford CF</span>, <span class="author">Ross MT</span>, <span class="author">Carter NP</span>, <span class="author">Davis ME</span>, <span class="author">Knights CE</span>, <span class="author">Smith AE</span>, <span class="author">Farr CJ</span>, <span class="author">McCarthy LC</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Animal Health Trust, Lanwades Park, Kentford, Newmarket, Suffolk, CB8 7UU, UK. <a href="mailto:%68%66%73%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#104;&#102;&#115;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chromosome (chr) X is under-represented in current maps of the genome of the domestic dog ( Canis familiaris). To address this problem, we have constructed a small-insert, genomic DNA library in pBluescript from flow-sorted canine Chr X DNA. Fluorescence in situ hybridization (FISH) studies confirmed that the library was highly enriched for Chr X. Clones containing microsatellites were identified and sequenced. Database searches detected significant sequence identity between four X-derived clones and genes previously characterized in other species. Thirty-seven markers derived from these clones were mapped on Chr X by FISH, and of these, 28 were mapped by using the female-derived T72 whole-genome radiation hybrid (RH) panel (Research Genetics). Four X-linked canine genes from publicly available data were also mapped. Eight RH linkage groups with LOD &gt;4.0 were identified, and FISH data were used to locate the groups on the chromosome; four groups could be unambiguously orientated by FISH data. In each case, the FISH and RH data were mutually consistent. The data suggest strongly conserved synteny between canine and human X Chrs. The pseudoautosomal region has been further characterized, and the putative or actual locations of nine genes of clinical relevance have been suggested.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2003</span>;<span class="volume">14</span>;<span class="number">3</span>;<span class="pagerange">214-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12647244" rel="external">12647244</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-002-3051-0" rel="external">10.1007/s00335-002-3051-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics.</h4>
<p class="authors"><span class="author">Stein LD</span>, <span class="author">Bao Z</span>, <span class="author">Blasiar D</span>, <span class="author">Blumenthal T</span>, <span class="author">Brent MR</span>, <span class="author">Chen N</span>, <span class="author">Chinwalla A</span>, <span class="author">Clarke L</span>, <span class="author">Clee C</span>, <span class="author">Coghlan A</span>, <span class="author">Coulson A</span>, <span class="author">D&#39;Eustachio P</span>, <span class="author">Fitch DH</span>, <span class="author">Fulton LA</span>, <span class="author">Fulton RE</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Harris TW</span>, <span class="author">Hillier LW</span>, <span class="author">Kamath R</span>, <span class="author">Kuwabara PE</span>, <span class="author">Mardis ER</span>, <span class="author">Marra MA</span>, <span class="author">Miner TL</span>, <span class="author">Minx P</span>, <span class="author">Mullikin JC</span>, <span class="author">Plumb RW</span>, <span class="author">Rogers J</span>, <span class="author">Schein JE</span>, <span class="author">Sohrmann M</span>, <span class="author">Spieth J</span>, <span class="author">Stajich JE</span>, <span class="author">Wei C</span>, <span class="author">Willey D</span>, <span class="author">Wilson RK</span>, <span class="author">Durbin R</span> and <span class="author">Waterston RH</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, USA.. <a href="mailto:%6C%73%74%65%69%6E%40%63%73%68%6C.%6F%72%67">&#108;&#115;&#116;&#101;&#105;&#110;&#64;&#99;&#115;&#104;&#108;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The soil nematodes Caenorhabditis briggsae and Caenorhabditis elegans diverged from a common ancestor roughly 100 million years ago and yet are almost indistinguishable by eye. They have the same chromosome number and genome sizes, and they occupy the same ecological niche. To explore the basis for this striking conservation of structure and function, we have sequenced the C. briggsae genome to a high-quality draft stage and compared it to the finished C. elegans sequence. We predict approximately 19,500 protein-coding genes in the C. briggsae genome, roughly the same as in C. elegans. Of these, 12,200 have clear C. elegans orthologs, a further 6,500 have one or more clearly detectable C. elegans homologs, and approximately 800 C. briggsae genes have no detectable matches in C. elegans. Almost all of the noncoding RNAs (ncRNAs) known are shared between the two species. The two genomes exhibit extensive colinearity, and the rate of divergence appears to be higher in the chromosomal arms than in the centers. Operons, a distinctive feature of C. elegans, are highly conserved in C. briggsae, with the arrangement of genes being preserved in 96% of cases. The difference in size between the C. briggsae (estimated at approximately 104 Mbp) and C. elegans (100.3 Mbp) genomes is almost entirely due to repetitive sequence, which accounts for 22.4% of the C. briggsae genome in contrast to 16.5% of the C. elegans genome. Few, if any, repeat families are shared, suggesting that most were acquired after the two species diverged or are undergoing rapid evolution. Coclustering the C. elegans and C. briggsae proteins reveals 2,169 protein families of two or more members. Most of these are shared between the two species, but some appear to be expanding or contracting, and there seem to be as many as several hundred novel C. briggsae gene families. The C. briggsae draft sequence will greatly improve the annotation of the C. elegans genome. Based on similarity to C. briggsae, we found strong evidence for 1,300 new C. elegans genes. In addition, comparisons of the two genomes will help to understand the evolutionary forces that mold nematode genomes.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5P01 HG00956</span>, <span class="grant">5U01 HG02042</span>, <span class="grant">P41 HG02223</span>; NIGMS NIH HHS: <span class="grant">R01 GM42432</span>, <span class="grant">T32 GM07754-22</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">1</span>;<span class="number">2</span>;<span class="pagerange">E45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14624247">14624247</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC261899">261899</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0000045">10.1371/journal.pbio.0000045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of candidate tumor-suppressor genes in 6q27 by combined deletion mapping and electronic expression profiling in lymphoid neoplasms.</h4>
<p class="authors"><span class="author">Steinemann D</span>, <span class="author">Gesk S</span>, <span class="author">Zhang Y</span>, <span class="author">Harder L</span>, <span class="author">Pilarsky C</span>, <span class="author">Hinzmann B</span>, <span class="author">Martin-Subero JI</span>, <span class="author">Calasanz MJ</span>, <span class="author">Mungall A</span>, <span class="author">Rosenthal A</span>, <span class="author">Siebert R</span> and <span class="author">Schlegelberger B</span></p>
<p class="affiliation">Institute of Cell and Molecular Pathology, Hannover Medical School, Hannover, Germany. <a href="mailto:%73%74%65%69%6E%65%6D%61%6E%6E.%64%6F%72%69%73%40%6D%68%5F%68%61%6E%6E%6F%76%65%72.%64%65">&#115;&#116;&#101;&#105;&#110;&#101;&#109;&#97;&#110;&#110;&#46;&#100;&#111;&#114;&#105;&#115;&#64;&#109;&#104;&#95;&#104;&#97;&#110;&#110;&#111;&#118;&#101;&#114;&#46;&#100;&#101;</a></p><p class="abstract">Deletions in the long arm of chromosome 6 (6q) are among the most frequent chromosome aberrations in lymphoid neoplasms. Recently, the region of minimal deletion (RMD1) in 6q27 was narrowed down to 5-9 Mb. In the present study, we aimed to define the distal border of the commonly lost region in 6q27 more precisely and to identify and investigate tumor-suppressor genes (TSGs) from this region. Twenty-nine cases, in which our previous fluorescence in situ hybridization (FISH) screening that used a set of 36 YAC probes revealed loss in 6q25-27, were further investigated by means of FISH. In all cases, deletions of 6q27 extended from yeast artificial chromosome (YAC) 977e10 spanning the proximal border of RMD1 to the most telomeric YAC 933f7 within the recently established YAC-contig of this region. An interstitial homozygous deletion, flanked by the telomeric probe TelVysion6q and YAC 971g12, was detected, which substantially narrows down the RMD1. To identify candidate TSGs down-regulated in malignant lymphomas from this region of homozygous loss, we performed electronic profiling of expressed sequences mapped to this region. This analysis suggested the gene PDCD2 originally thought to be involved in programmed cell death to be probably down-regulated in malignant B-cell lymphomas compared to normal B lymphocytes. Nevertheless, mutation analyses failed to identify mutations in the coding region of PDCD2 in nine lymphomas with FISH-proved 6q27 deletions. Furthermore, epigenetic studies in these nine and an additional 48 lymphomas did not show altered methylation of the PDCD2 locus in these tumors. Possibly haploinsufficiency is effectual in accelerating tumor progression.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2003</span>;<span class="volume">37</span>;<span class="number">4</span>;<span class="pagerange">421-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12800155" rel="external">12800155</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.10231" rel="external">10.1002/gcc.10231</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insights into the effects on metal binding of the systematic substitution of five key glutamate ligands in the ferritin of Escherichia coli.</h4>
<p class="authors"><span class="author">Stillman TJ</span>, <span class="author">Connolly PP</span>, <span class="author">Latimer CL</span>, <span class="author">Morland AF</span>, <span class="author">Quail MA</span>, <span class="author">Andrews SC</span>, <span class="author">Treffry A</span>, <span class="author">Guest JR</span>, <span class="author">Artymiuk PJ</span> and <span class="author">Harrison PM</span></p>
<p class="affiliation">Krebs Institute, Department of Molecular Biology and Biotechnology, University of Sheffield, Sheffield S10 2TN, United Kingdom.</p><p class="abstract">Ferritins are nearly ubiquitous iron storage proteins playing a fundamental role in iron metabolism. They are composed of 24 subunits forming a spherical protein shell encompassing a central iron storage cavity. The iron storage mechanism involves the initial binding and subsequent O2-dependent oxidation of two Fe2+ ions located at sites A and B within the highly conserved dinuclear &quot;ferroxidase center&quot; in individual subunits. Unlike animal ferritins and the heme-containing bacterioferritins, the Escherichia coli ferritin possesses an additional iron-binding site (site C) located on the inner surface of the protein shell close to the ferroxidase center. We report the structures of five E. coli ferritin variants and their Fe3+ and Zn2+ (a redox-stable alternative for Fe2+) derivatives. Single carboxyl ligand replacements in sites A, B, and C gave unique effects on metal binding, which explain the observed changes in Fe2+ oxidation rates. Binding of Fe2+ at both A and B sites is clearly essential for rapid Fe2+ oxidation, and the linking of FeB2+ to FeC2+ enables the oxidation of three Fe2+ ions. The transient binding of Fe2+ at one of three newly observed Zn2+ sites may allow the oxidation of four Fe2+ by one dioxygen molecule.</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2003</span>;<span class="volume">278</span>;<span class="number">28</span>;<span class="pagerange">26275-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12730190" rel="external">12730190</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M207354200" rel="external">10.1074/jbc.M207354200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence-based cancer genomics: progress, lessons and opportunities.</h4>
<p class="authors"><span class="author">Strausberg RL</span>, <span class="author">Simpson AJ</span> and <span class="author">Wooster R</span></p>
<p class="affiliation">National Cancer Institute, 31 Center Drive, Room 10A07, Bethesda, Maryland 20892, USA. <a href="mailto:%52%4C%53%40%6E%69%68.%67%6F%76">&#82;&#76;&#83;&#64;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">Technologies that provide a genome-wide view offer an unprecedented opportunity to scrutinize the molecular biology of the cancer cell. The information that is derived from these technologies is well suited to the development of public databases of alterations in the cancer genome and its expression. Here, we describe the synergistic efforts of research programmes in Brazil, the United Kingdom and the United States towards building integrated databases that are widely accessible to the research community, to enable basic and applied applications in cancer research.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="number">6</span>;<span class="pagerange">409-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12776211">12776211</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1085">10.1038/nrg1085</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Domain architectures of sigma54-dependent transcriptional activators.</h4>
<p class="authors"><span class="author">Studholme DJ</span> and <span class="author">Dixon R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%64%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2003</span>;<span class="volume">185</span>;<span class="number">6</span>;<span class="pagerange">1757-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12618438" rel="external">12618438</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC150144" rel="external">150144</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A DNA element recognised by the molybdenum-responsive transcription factor ModE is conserved in Proteobacteria, green sulphur bacteria and Archaea.</h4>
<p class="authors"><span class="author">Studholme DJ</span> and <span class="author">Pau RN</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%64%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The transition metal molybdenum is essential for life. Escherichia coli imports this metal into the cell in the form of molybdate ions, which are taken up via an ABC transport system. In E. coli and other Proteobacteria molybdenum metabolism and homeostasis are regulated by the molybdate-responsive transcription factor ModE.</p><p class="abstract"><strong>Results</strong>: Orthologues of ModE are widespread amongst diverse prokaryotes, but not ubiquitous. We identified probable ModE-binding sites upstream of genes implicated in molybdenum metabolism in green sulphur bacteria and methanogenic Archaea as well as in Proteobacteria. We also present evidence of horizontal transfer of nitrogen fixation genes between green sulphur bacteria and methanogenic Archaea.</p><p class="abstract"><strong>Conclusions</strong>: Whereas most of the archaeal helix-turn-helix-containing transcription factors belong to families that are Archaea-specific, ModE is unusual in that it is found in both Archaea and Bacteria. Moreover, its cognate upstream DNA recognition sequence is also conserved between Archaea and Bacteria, despite the fundamental differences in their core transcription machinery. ModE is the third example of a transcriptional regulator with a binding signal that is conserved in Bacteria and Archaea.</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">3</span>;<span class="pagerange">24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14641908" rel="external">14641908</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC317290" rel="external">317290</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-3-24" rel="external">10.1186/1471-2180-3-24</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comparison of Pfam and MEROPS: two databases, one comprehensive, and one specialised.</h4>
<p class="authors"><span class="author">Studholme DJ</span>, <span class="author">Rawlings ND</span>, <span class="author">Barrett AJ</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%64%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: We wished to compare two databases based on sequence similarity: one that aims to be comprehensive in its coverage of known sequences, and one that specialises in a relatively small subset of known sequences. One of the motivations behind this study was quality control. Pfam is a comprehensive collection of alignments and hidden Markov models representing families of proteins and domains. MEROPS is a catalogue and classification of enzymes with proteolytic activity (peptidases or proteases). These secondary databases are used by researchers worldwide, yet their contents are not peer reviewed. Therefore, we hoped that a systematic comparison of the contents of Pfam and MEROPS would highlight missing members and false-positives leading to improvements in quality of both databases. An additional reason for carrying out this study was to explore the extent of consensus in the definition of a protein family.</p><p class="abstract"><strong>Results</strong>: About half (89 out of 174) of the peptidase families in MEROPS overlapped single Pfam families. A further 32 MEROPS families overlapped multiple Pfam families. Where possible, new Pfam families were built to represent most of the MEROPS families that did not overlap Pfam. When comparing the numbers of sequences found in the overlap between a MEROPS family and its corresponding Pfam family, in most cases the overlap was substantial (52 pairs of MEROPS and Pfam families had an intersection size of greater than 75% of the union) but there were some differences in the sets of sequences included in the MEROPS families versus the overlapping Pfam families.</p><p class="abstract"><strong>Conclusions</strong>: A number of the discrepancies between MEROPS families and their corresponding Pfam families arose from differences in the aims and philosophies of the two databases. Examination of some of the discrepancies highlighted additional members of families, which have subsequently been added in both Pfam and MEROPS. This has led to improvements in the quality of both databases. Overall there was a great deal of consensus between the databases in definitions of a protein family.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2003</span>;<span class="volume">4</span>;<span class="pagerange">17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12740029" rel="external">12740029</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC156619" rel="external">156619</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-4-17" rel="external">10.1186/1471-2105-4-17</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beyond release: the equitable use of genomic information.</h4>
<p class="authors"><span class="author">Sulston J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1RQ, Cambridge, UK. <a href="mailto:%6A%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a> &lt;<a href="mailto:%6A%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B&%67%74">&#106;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;&amp;&#103;&#116;</a>;</p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2003</span>;<span class="volume">362</span>;<span class="number">9381</span>;<span class="pagerange">400-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12907016" rel="external">12907016</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(03)14030-5" rel="external">10.1016/S0140-6736(03)14030-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A canine cancer-gene microarray for CGH analysis of tumors.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Fiegler H</span>, <span class="author">Ostrander EA</span>, <span class="author">Galibert F</span>, <span class="author">Carter NP</span> and <span class="author">Breen M</span></p>
<p class="affiliation">Oncology Research Group, Centre for Preventive Medicine, Animal Health Trust, Lanwades Park, Kentford, Newmarket, Suffolk, UK.</p><p class="abstract">As with many human cancers, canine tumors demonstrate recurrent chromosome aberrations. A detailed knowledge of such aberrations may facilitate diagnosis, prognosis and the selection of appropriate therapy. Following recent advances made in human genomics, we are developing a DNA microarray for the domestic dog, to be used in the detection and characterization of copy number changes in canine tumors. As a proof of principle, we have developed a small-scale microarray comprising 87 canine BAC clones. The array is composed of 26 clones selected from a panel of 24 canine cancer genes, representing 18 chromosomes, and an additional set of clones representing dog chromosomes 11, 13, 14 and 31. These chromosomes were shown previously to be commonly aberrant in canine multicentric malignant lymphoma. Clones representing the sex chromosomes were also included. We outline the principles of canine microarray development, and present data obtained from microarray analysis of three canine lymphoma cases previously characterized using conventional cytogenetic techniques.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2003</span>;<span class="volume">102</span>;<span class="number">1-4</span>;<span class="pagerange">254-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14970712" rel="external">14970712</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000075758" rel="external">10.1159/000075758</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fitting the niche by genomic adaptation.</h4>
<p class="authors"><span class="author">Thomson N</span>, <span class="author">Bentley S</span>, <span class="author">Holden M</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Studying microbial genomics has shown that the genomes of bacteria are extremely dynamic in evolutionary terms. Many research groups have linked the adaptation of an organism to a niche to large changes in genome size and content. A number of recent papers have underlined the degree to which the genomes of different organisms are a reflection of the opportunities and constraints imposed by their chosen niche.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">1</span>;<span class="number">2</span>;<span class="pagerange">92-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15035037">15035037</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro756">10.1038/nrmicro756</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The value of comparison.</h4>
<p class="authors"><span class="author">Thomson N</span>, <span class="author">Sebaihia M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Bentley S</span>, <span class="author">Crossman L</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">With the number of published microbial genomes now in excess of 100, any new genome that is sequenced is likely to have a close relative available for comparison. Indeed, it is increasingly difficult to perform any genomic analysis that is not comparative. This should, however, not be seen as a drawback; it is often the case that a large amount of information can be drawn from these comparisons, especially between closely related organisms. Several genome sequences published recently indicate the value of comparisons at the genomic level.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">11-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15040174">15040174</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro734">10.1038/nrmicro734</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The building blocks of pathogenicity.</h4>
<p class="authors"><span class="author">Thomson NR</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">2</span>;<span class="pagerange">66-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12598126" rel="external">12598126</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">All walks of life.</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Crossman L</span>, <span class="author">Sebaihia M</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">11</span>;<span class="number">4</span>;<span class="pagerange">159-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12706992" rel="external">12706992</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of the T-cell regulatory gene CTLA4 with susceptibility to autoimmune disease.</h4>
<p class="authors"><span class="author">Ueda H</span>, <span class="author">Howson JM</span>, <span class="author">Esposito L</span>, <span class="author">Heward J</span>, <span class="author">Snook H</span>, <span class="author">Chamberlain G</span>, <span class="author">Rainbow DB</span>, <span class="author">Hunter KM</span>, <span class="author">Smith AN</span>, <span class="author">Di Genova G</span>, <span class="author">Herr MH</span>, <span class="author">Dahlman I</span>, <span class="author">Payne F</span>, <span class="author">Smyth D</span>, <span class="author">Lowe C</span>, <span class="author">Twells RC</span>, <span class="author">Howlett S</span>, <span class="author">Healy B</span>, <span class="author">Nutland S</span>, <span class="author">Rance HE</span>, <span class="author">Everett V</span>, <span class="author">Smink LJ</span>, <span class="author">Lam AC</span>, <span class="author">Cordell HJ</span>, <span class="author">Walker NM</span>, <span class="author">Bordin C</span>, <span class="author">Hulme J</span>, <span class="author">Motzo C</span>, <span class="author">Cucca F</span>, <span class="author">Hess JF</span>, <span class="author">Metzker ML</span>, <span class="author">Rogers J</span>, <span class="author">Gregory S</span>, <span class="author">Allahabadia A</span>, <span class="author">Nithiyananthan R</span>, <span class="author">Tuomilehto-Wolf E</span>, <span class="author">Tuomilehto J</span>, <span class="author">Bingley P</span>, <span class="author">Gillespie KM</span>, <span class="author">Undlien DE</span>, <span class="author">R&oslash;nningen KS</span>, <span class="author">Guja C</span>, <span class="author">Ionescu-T&icirc;rgovi&#x15F;te C</span>, <span class="author">Savage DA</span>, <span class="author">Maxwell AP</span>, <span class="author">Carson DJ</span>, <span class="author">Patterson CC</span>, <span class="author">Franklyn JA</span>, <span class="author">Clayton DG</span>, <span class="author">Peterson LB</span>, <span class="author">Wicker LS</span>, <span class="author">Todd JA</span> and <span class="author">Gough SC</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Cambridge, CB2 2XY, UK.</p><p class="abstract">Genes and mechanisms involved in common complex diseases, such as the autoimmune disorders that affect approximately 5% of the population, remain obscure. Here we identify polymorphisms of the cytotoxic T lymphocyte antigen 4 gene (CTLA4)--which encodes a vital negative regulatory molecule of the immune system--as candidates for primary determinants of risk of the common autoimmune disorders Graves&#39; disease, autoimmune hypothyroidism and type 1 diabetes. In humans, disease susceptibility was mapped to a non-coding 6.1 kb 3&#39; region of CTLA4, the common allelic variation of which was correlated with lower messenger RNA levels of the soluble alternative splice form of CTLA4. In the mouse model of type 1 diabetes, susceptibility was also associated with variation in CTLA-4 gene splicing with reduced production of a splice form encoding a molecule lacking the CD80/CD86 ligand-binding domain. Genetic mapping of variants conferring a small disease risk can identify pathways in complex disorders, as exemplified by our discovery of inherited, quantitative alterations of CTLA4 contributing to autoimmune tissue destruction.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">423</span>;<span class="number">6939</span>;<span class="pagerange">506-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12724780">12724780</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature01621">10.1038/nature01621</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">QUAD system offers fair shares to all authors.</h4>
<p class="authors"><span class="author">Verhagen JV</span>, <span class="author">Wallace KJ</span>, <span class="author">Collins SC</span> and <span class="author">Scott TR</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2003</span>;<span class="volume">426</span>;<span class="number">6967</span>;<span class="pagerange">602</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14668833">14668833</a>; DOI: <a href="http://dx.doi.org/10.1038%2F426602a">10.1038/426602a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deoxycorticosterone upregulates PDS (Slc26a4) in mouse kidney: role of pendrin in mineralocorticoid-induced hypertension.</h4>
<p class="authors"><span class="author">Verlander JW</span>, <span class="author">Hassell KA</span>, <span class="author">Royaux IE</span>, <span class="author">Glapion DM</span>, <span class="author">Wang ME</span>, <span class="author">Everett LA</span>, <span class="author">Green ED</span> and <span class="author">Wall SM</span></p>
<p class="affiliation">Department of Medicine, University of Florida College of Medicine, Gainesville, USA.</p><p class="abstract">Pendrin is an anion exchanger expressed along the apical plasma membrane and apical cytoplasmic vesicles of type B and of non-A, non-B intercalated cells of the distal convoluted tubule, connecting tubule, and cortical collecting duct. Thus, Pds (Slc26a4) is a candidate gene for the putative apical anion-exchange process of the type B intercalated cell. Because apical anion exchange-mediated transport is upregulated with deoxycorticosterone pivalate (DOCP), we tested whether Pds mRNA and protein expression in mouse kidney were upregulated after administration of this aldosterone analogue by using quantitative real-time polymerase chain reaction as well as light and electron microscopic immunolocalization. In kidneys from DOCP-treated mice, Pds mRNA increased 60%, whereas pendrin protein expression in the apical plasma membrane increased 2-fold in non-A, non-B intercalated cells and increased 6-fold in type B cells. Because pendrin transports HCO3- and Cl-, we tested whether DOCP treatment unmasks abnormalities in acid-base or NaCl balance in Pds (-/-) mice. In the absence of DOCP, arterial pH, systolic blood pressure, and body weight were similar in Pds (+/+) and Pds (-/-) mice. After DOCP treatment, weight gain and hypertension were observed in Pds (+/+) but not in Pds (-/-) mice. Moreover, after DOCP administration, metabolic alkalosis was more severe in Pds (-/-) than Pds (+/+) mice. We conclude that pendrin is upregulated with aldosterone analogues and is critical in the pathogenesis of mineralocorticoid-induced hypertension and metabolic alkalosis.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK 52935</span></p>
<p class="publication"><span class="title">Hypertension</span>&nbsp;<span class="year">2003</span>;<span class="volume">42</span>;<span class="number">3</span>;<span class="pagerange">356-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12925556" rel="external">12925556</a>; DOI: <a href="http://dx.doi.org/10.1161%2F01.HYP.0000088321.67254.B7" rel="external">10.1161/01.HYP.0000088321.67254.B7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complex transcription and splicing of odorant receptor genes.</h4>
<p class="authors"><span class="author">Volz A</span>, <span class="author">Ehlers A</span>, <span class="author">Younger R</span>, <span class="author">Forbes S</span>, <span class="author">Trowsdale J</span>, <span class="author">Schnorr D</span>, <span class="author">Beck S</span> and <span class="author">Ziegler A</span></p>
<p class="affiliation">Institut f&uuml;r Immungenetik, Universit&auml;tsklinikum Charit&eacute;, Humboldt-Universit&auml;t zu Berlin, Spandauer Damm 130, Germany.</p><p class="abstract">Human major histocompatibility (human leucocyte antigen (HLA)) complex-linked odorant receptor (OR) genes are among the best characterized OR genes in the human genome. In addition to their functions as odorant receptors in olfactory epithelium, they have been suggested to play a role in the fertilization process. Here, we report the first in-depth analysis of their expression and regulation within testicular tissue. Sixteen HLA-linked OR and three non-HLA-linked OR were analyzed. One OR gene (hs6M1-16, in positive transcriptional orientation) exhibited six different transcriptional start sites combined with extensive alternative splicing within the 5&#39;-untranslated region, the coding exon, and the 3&#39;-untranslated region. Long distance splicing, exon sharing, and premature polyadenylation were features of another three OR loci (hs6M1-18, -21, and -27, all upstream of hs6M1-16, but in negative transcriptional orientation). Determination of the transcriptional start sites of these OR genes identified a region of 81 bp with potential bi-directional transcriptional activity. The results demonstrate that HLA-linked OR genes are subject to unusually complex transcriptional regulatory mechanisms.</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2003</span>;<span class="volume">278</span>;<span class="number">22</span>;<span class="pagerange">19691-701</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12637542" rel="external">12637542</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M212424200" rel="external">10.1074/jbc.M212424200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modeling del(17)(p11.2p11.2) and dup(17)(p11.2p11.2) contiguous gene syndromes by chromosome engineering in mice: phenotypic consequences of gene dosage imbalance.</h4>
<p class="authors"><span class="author">Walz K</span>, <span class="author">Caratini-Rivera S</span>, <span class="author">Bi W</span>, <span class="author">Fonseca P</span>, <span class="author">Mansouri DL</span>, <span class="author">Lynch J</span>, <span class="author">Vogel H</span>, <span class="author">Noebels JL</span>, <span class="author">Bradley A</span> and <span class="author">Lupski JR</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA.</p><p class="abstract">Contiguous gene syndromes (CGS) are a group of disorders associated with chromosomal rearrangements of which the phenotype is thought to result from altered copy numbers of physically linked dosage-sensitive genes. Smith-Magenis syndrome (SMS) is a CGS associated with a deletion within band p11.2 of chromosome 17. Recently, patients harboring the predicted reciprocal duplication product [dup(17)(p11.2p11.2)] have been described as having a relatively mild phenotype. By chromosomal engineering, we created rearranged chromosomes carrying the deletion [Df(11)17] or duplication [Dp(11)17] of the syntenic region on mouse chromosome 11 that spans the genomic interval commonly deleted in SMS patients. Df(11)17/+ mice exhibit craniofacial abnormalities, seizures, marked obesity, and male-specific reduced fertility. Dp(11)17/+ animals are underweight and do not have seizures, craniofacial abnormalities, or reduced fertility. Examination of Df(11)17/Dp(11)17 animals suggests that most of the observed phenotypes result from gene dosage effects. Our murine models represent a powerful tool to analyze the consequences of gene dosage imbalance in this genomic interval and to investigate the molecular genetic bases of both SMS and dup(17)(p11.2p11.2).</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01 CA 75719</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">23</span>;<span class="number">10</span>;<span class="pagerange">3646-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12724422" rel="external">12724422</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC154242" rel="external">154242</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene discovery in the Entamoeba invadens genome.</h4>
<p class="authors"><span class="author">Wang Z</span>, <span class="author">Samuelson J</span>, <span class="author">Clark CG</span>, <span class="author">Eichinger D</span>, <span class="author">Paul J</span>, <span class="author">Van Dellen K</span>, <span class="author">Hall N</span>, <span class="author">Anderson I</span> and <span class="author">Loftus B</span></p>
<p class="affiliation">Center for Bio/Molecular Science Naval Research Laboratory, Washington, DC 20375, USA.</p><p class="abstract">Entamoeba invadens, a parasite of reptiles, is a model for the study of encystation by the human enteric pathogen Entamoeba histolytica, because E. invadens form cysts in axenic culture. With approximately 0.5-fold sequence coverage of the genome, we were able to get insights into E. invadens gene and genome features. Overall, the E. invadens genome displays many of the features that are emerging from ongoing genome sequencing efforts in E. histolytica. At the nucleotide level the E. invadens genome has on average 60% sequence identity with that of E. histolytica. The presence of introns in E. invadens was predicted with similar consensus (GTTTGT em leader A/TAG) sequences to those identified in E. histolytica and Entamoeba dispar. Sequences highly repeated in the genome of E. histolytica (rRNAs, tRNAs, CXXC-rich proteins, and Leu-rich repeat proteins) were found to be highly repeated in the E. invadens genome. Numerous proteins homologous to those implicated in amoebic virulence, (Gal/GalNAc lectins, amoebapores, and cysteine proteinases) and drug resistance (p-glycoproteins) were identified. Homologs of proteins involved in cell cycle, vesicular trafficking and signal transduction were identified, which may be involved in en/excystation and cell growth of E. invadens. Finally, multiple copies of a number of E. invadens genes coding for predicted enzymes involved in core metabolism and the targets of anti-amoebic drugs were identified.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI46516</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2003</span>;<span class="volume">129</span>;<span class="number">1</span>;<span class="pagerange">23-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12798503" rel="external">12798503</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">More on the sequencing of the human genome.</h4>
<p class="authors"><span class="author">Waterston RH</span>, <span class="author">Lander ES</span> and <span class="author">Sulston JE</span></p>
<p class="affiliation">Department of Genome Sciences, University of Washington, Box 357730, Seattle, WA 98195, USA. <a href="mailto:%77%61%74%65%72%73%74%6F%6E%40%67%73.%77%61%73%68%69%6E%67%74%6F%6E.%65%64%75">&#119;&#97;&#116;&#101;&#114;&#115;&#116;&#111;&#110;&#64;&#103;&#115;&#46;&#119;&#97;&#115;&#104;&#105;&#110;&#103;&#116;&#111;&#110;&#46;&#101;&#100;&#117;</a></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2003</span>;<span class="volume">100</span>;<span class="number">6</span>;<span class="pagerange">3022-4; author reply 3025-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12631699" rel="external">12631699</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC152236" rel="external">152236</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0634129100" rel="external">10.1073/pnas.0634129100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Breast and ovarian cancer.</h4>
<p class="authors"><span class="author">Wooster R</span> and <span class="author">Weber BL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2003</span>;<span class="volume">348</span>;<span class="number">23</span>;<span class="pagerange">2339-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12788999" rel="external">12788999</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMra012284" rel="external">10.1056/NEJMra012284</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Leishmania major chromosome 3 contains two long convergent polycistronic gene clusters separated by a tRNA gene.</h4>
<p class="authors"><span class="author">Worthey EA</span>, <span class="author">Martinez-Calvillo S</span>, <span class="author">Schnaufer A</span>, <span class="author">Aggarwal G</span>, <span class="author">Cawthra J</span>, <span class="author">Fazelinia G</span>, <span class="author">Fong C</span>, <span class="author">Fu G</span>, <span class="author">Hassebrock M</span>, <span class="author">Hixson G</span>, <span class="author">Ivens AC</span>, <span class="author">Kiser P</span>, <span class="author">Marsolini F</span>, <span class="author">Rickel E</span>, <span class="author">Rickell E</span>, <span class="author">Salavati R</span>, <span class="author">Sisk E</span>, <span class="author">Sunkin SM</span>, <span class="author">Stuart KD</span> and <span class="author">Myler PJ</span></p>
<p class="affiliation">Seattle Biomedical Research Institute, 4 Nickerson Street, Seattle, WA 98109-1651, USA.</p><p class="abstract">Leishmania parasites (order Kinetoplastida, family Trypanosomatidae) cause a spectrum of human diseases ranging from asymptomatic to lethal. The approximately 33.6 Mb genome is distributed among 36 chromosome pairs that range in size from approximately 0.3 to 2.8 Mb. The complete nucleotide sequence of Leishmania major Friedlin chromosome 1 revealed 79 protein-coding genes organized into two divergent polycistronic gene clusters with the mRNAs transcribed towards the telomeres. We report here the complete nucleotide sequence of chromosome 3 (384 518 bp) and an analysis revealing 95 putative protein-coding ORFs. The ORFs are primarily organized into two large convergent polycistronic gene clusters (i.e. transcribed from the telomeres). In addition, a single gene at the left end is transcribed divergently towards the telomere, and a tRNA gene separates the two convergent gene clusters. Numerous genes have been identified, including those for metabolic enzymes, kinases, transporters, ribosomal proteins, spliceosome components, helicases, an RNA-binding protein and a DNA primase subunit.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI053667-01</span>, <span class="grant">R01 AI053667-02</span>, <span class="grant">R01 AI40599</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2003</span>;<span class="volume">31</span>;<span class="number">14</span>;<span class="pagerange">4201-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12853638" rel="external">12853638</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC167632" rel="external">167632</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">COP9 signalosome subunit 3 is essential for maintenance of cell proliferation in the mouse embryonic epiblast.</h4>
<p class="authors"><span class="author">Yan J</span>, <span class="author">Walz K</span>, <span class="author">Nakamura H</span>, <span class="author">Carattini-Rivera S</span>, <span class="author">Zhao Q</span>, <span class="author">Vogel H</span>, <span class="author">Wei N</span>, <span class="author">Justice MJ</span>, <span class="author">Bradley A</span> and <span class="author">Lupski JR</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Texas Children&#39;s Hospital, Houston, Texas 77030, USA.</p><p class="abstract">Csn3 (Cops3) maps to the mouse chromosome 11 region syntenic to the common deletion interval for the Smith-Magenis syndrome, a contiguous gene deletion syndrome. It encodes the third subunit of an eight-subunit protein complex, the COP9 signalosome (CSN), which controls a wide variety of molecules of different functions. Mutants of this complex caused lethality at early development of both plants and Drosophila melanogaster. CSN function in vivo in mammals is unknown. We disrupted the murine Csn3 gene in three independent ways with insertional vectors, including constructing a approximately 3-Mb inversion chromosome. The heterozygous mice appeared normal, although the protein level was reduced. Csn3(-/-) embryos arrested after 5.5 days postcoitum (dpc) and resorbed by 8.5 dpc. Mutant embryos form an abnormal egg cylinder which does not gastrulate. They have reduced numbers of epiblast cells, mainly due to increased cell death. In the Csn3(-/-) mice, subunit 8 of the COP9 complex was not detected by immunohistochemical techniques, suggesting that the absence of Csn3 may disrupt the entire COP9 complex. Therefore, Csn3 is important for maintaining the integrity of the COP9 signalosome and is crucial to maintain the survival of epiblast cells and thus the development of the postimplantation embryo in mice.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01CA75719</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2003</span>;<span class="volume">23</span>;<span class="number">19</span>;<span class="pagerange">6798-808</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12972600" rel="external">12972600</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC193933" rel="external">193933</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The BON domain: a putative membrane-binding domain.</h4>
<p class="authors"><span class="author">Yeats C</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK CB10 1SA. <a href="mailto:%63%61%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2003</span>;<span class="volume">28</span>;<span class="number">7</span>;<span class="pagerange">352-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12878000" rel="external">12878000</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0968-0004(03)00115-4" rel="external">10.1016/S0968-0004(03)00115-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.</h4>
<p class="authors"><span class="author">Yeats C</span>, <span class="author">Bentley S</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK. <a href="mailto:%63%61%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Streptomyces coelicolor has long been considered a remarkable bacterium with a complex life-cycle, ubiquitous environmental distribution, linear chromosomes and plasmids, and a huge range of pharmaceutically useful secondary metabolites. Completion of the genome sequence demonstrated that this diversity carried through to the genetic level, with over 7000 genes identified. We sought to expand our understanding of this organism at the molecular level through identification and annotation of novel protein domains. Protein domains are the evolutionary conserved units from which proteins are formed.</p><p class="abstract"><strong>Results</strong>: Two automated methods were employed to rapidly generate an optimised set of targets, which were subsequently analysed manually. A final set of 37 domains or structural repeats, represented 204 times in the genome, was developed. Using these families enabled us to correlate items of information from many different resources. Several immediately enhance our understanding both of S. coelicolor and also general bacterial molecular mechanisms, including cell wall biosynthesis regulation and streptomycete telomere maintenance.</p><p class="abstract"><strong>Discussion</strong>: Delineation of protein domain families enables detailed analysis of protein function, as well as identification of likely regions or residues of particular interest. Hence this kind of prior approach can increase the rate of discovery in the laboratory. Furthermore we demonstrate that using this type of in silico method it is possible to fairly rapidly generate new biological information from previously uncorrelated data.</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2003</span>;<span class="volume">3</span>;<span class="pagerange">3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12625841" rel="external">12625841</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC151604" rel="external">151604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of the A2 gene locus in Leishmania donovani and Leishmania major and its control over cutaneous infection.</h4>
<p class="authors"><span class="author">Zhang WW</span>, <span class="author">Mendez S</span>, <span class="author">Ghosh A</span>, <span class="author">Myler P</span>, <span class="author">Ivens A</span>, <span class="author">Clos J</span>, <span class="author">Sacks DL</span> and <span class="author">Matlashewski G</span></p>
<p class="affiliation">Department of Microbiology and Immunology, McGill University, Montreal, Quebec H3A 2B4, Canada.</p><p class="abstract">In Old World Leishmania infections, Leishmania donovani is responsible for fatal visceral leishmaniasis, and L. major is responsible for non-fatal cutaneous leishmaniasis in humans. The genetic differences between these species which govern the pathology or site of infection are not known. We have therefore carried out detailed analysis of the A2 loci in L. major and L. donovani because A2 is expressed in L. donovani but not L. major, and A2 is required for survival in visceral organs by L. donovani. We demonstrate that although L. major contains A2 gene regulatory sequences, the multiple repeats that exist in L. donovani A2 protein coding regions are absent in L. major, and the remaining corresponding A2 sequences appear to represent non-expressed pseudogenes. It was possible to restore amastigote-specific A2 expression to L. major, confirming that A2 regulatory sequences remain functional in L. major. Although L. major is a cutaneous parasite in rodents and humans, restoring A2 expression to L. major inhibited its ability to establish a cutaneous infection in susceptible BALB/c or resistant C57BL6 mice, a phenotype typical of L. donovani. There was no detectable cellular immune response against L. major after cutaneous infection with A2-expressing L. major, suggesting that the lack of growth was not attributable to acquired host resistance but to an A2-mediated suppression of parasite survival in skin macrophages. These observations argue that the lack of A2 expression in L. major contributed to its divergence from L. donovani with respect to the pathology of infection.</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2003</span>;<span class="volume">278</span>;<span class="number">37</span>;<span class="pagerange">35508-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12829719" rel="external">12829719</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M305030200" rel="external">10.1074/jbc.M305030200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Positional cloning of a quantitative trait locus on chromosome 13q14 that influences immunoglobulin E levels and asthma.</h4>
<p class="authors"><span class="author">Zhang Y</span>, <span class="author">Leaves NI</span>, <span class="author">Anderson GG</span>, <span class="author">Ponting CP</span>, <span class="author">Broxholme J</span>, <span class="author">Holt R</span>, <span class="author">Edser P</span>, <span class="author">Bhattacharyya S</span>, <span class="author">Dunham A</span>, <span class="author">Adcock IM</span>, <span class="author">Pulleyn L</span>, <span class="author">Barnes PJ</span>, <span class="author">Harper JI</span>, <span class="author">Abecasis G</span>, <span class="author">Cardon L</span>, <span class="author">White M</span>, <span class="author">Burton J</span>, <span class="author">Matthews L</span>, <span class="author">Mott R</span>, <span class="author">Ross M</span>, <span class="author">Cox R</span>, <span class="author">Moffatt MF</span> and <span class="author">Cookson WO</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK.</p><p class="abstract">Atopic or immunoglobulin E (IgE)-mediated diseases include the common disorders of asthma, atopic dermatitis and allergic rhinitis. Chromosome 13q14 shows consistent linkage to atopy and the total serum IgE concentration. We previously identified association between total serum IgE levels and a novel 13q14 microsatellite (USAT24G1; ref. 7) and have now localized the underlying quantitative-trait locus (QTL) in a comprehensive single-nucleotide polymorphism (SNP) map. We found replicated association to IgE levels that was attributed to several alleles in a single gene, PHF11. We also found association with these variants to severe clinical asthma. The gene product (PHF11) contains two PHD zinc fingers and probably regulates transcription. Distinctive splice variants were expressed in immune tissues and cells.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2003</span>;<span class="volume">34</span>;<span class="number">2</span>;<span class="pagerange">181-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/12754510">12754510</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1166">10.1038/ng1166</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/bqgysojw"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/bqgysojw" src="/qr/bqgysojw.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
