<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2012 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2012.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2012</h1><p>Number of papers published in 2012: <strong>482</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated map of genetic variation from 1,092 human genomes.</h4>
<p class="authors"><span class="author">1000 Genomes Project Consortium</span>, <span class="author">Abecasis GR</span>, <span class="author">Auton A</span>, <span class="author">Brooks LD</span>, <span class="author">DePristo MA</span>, <span class="author">Durbin RM</span>, <span class="author">Handsaker RE</span>, <span class="author">Kang HM</span>, <span class="author">Marth GT</span> and <span class="author">McVean GA</span></p><p class="abstract">By characterizing the geographic and functional spectrum of human genetic variation, the 1000 Genomes Project aims to build a resource to help to understand the genetic contribution to disease. Here we describe the genomes of 1,092 individuals from 14 populations, constructed using a combination of low-coverage whole-genome and exome sequencing. By developing methods to integrate information across several algorithms and diverse data sources, we provide a validated haplotype map of 38&thinsp;million single nucleotide polymorphisms, 1.4&thinsp;million short insertions and deletions, and more than 14,000 larger deletions. We show that individuals from different populations carry different profiles of rare and common variants, and that low-frequency variants show substantial geographic differentiation, which is further increased by the action of purifying selection. We show that evolutionary conservation and coding consequence are key determinants of the strength of purifying selection, that rare-variant load varies substantially across biological pathways, and that each individual contains hundreds of rare non-coding variants at conserved sites, such as motif-disrupting changes in transcription-factor-binding sites. This resource, which captures up to 98% of accessible single nucleotide polymorphisms at a frequency of 1% in related populations, enables analysis of common and low-frequency variants in individuals from diverse, including admixed, populations.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/I021213/1</span>, <span class="grant">BB/I02593X/1</span>; British Heart Foundation: <span class="grant">RG/09/012/28096</span>, <span class="grant">RG/09/12/28096</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0701805</span>, <span class="grant">G0801823</span>, <span class="grant">G0900747</span>, <span class="grant">G0900747(91070)</span>; NCI NIH HHS: <span class="grant">R01 CA166661</span>, <span class="grant">R01CA166661</span>; NCRR NIH HHS: <span class="grant">G12 RR003050</span>, <span class="grant">UL1RR024131</span>; NHGRI NIH HHS: <span class="grant">P01 HG004120</span>, <span class="grant">P01HG4120</span>, <span class="grant">P41HG2371</span>, <span class="grant">P41HG4221</span>, <span class="grant">R01 HG002898</span>, <span class="grant">R01 HG004960</span>, <span class="grant">R01 HG007022</span>, <span class="grant">R01HG2898</span>, <span class="grant">R01HG3698</span>, <span class="grant">R01HG4719</span>, <span class="grant">R01HG4960</span>, <span class="grant">R01HG5701</span>, <span class="grant">RC2HG5552</span>, <span class="grant">RC2HG5581</span>, <span class="grant">U01 HG005728</span>, <span class="grant">U01 HG006513</span>, <span class="grant">U01HG5208</span>, <span class="grant">U01HG5209</span>, <span class="grant">U01HG5211</span>, <span class="grant">U01HG5214</span>, <span class="grant">U01HG5715</span>, <span class="grant">U01HG5725</span>, <span class="grant">U01HG5728</span>, <span class="grant">U01HG6513</span>, <span class="grant">U01HG6569</span>, <span class="grant">U41HG4568</span>, <span class="grant">U54 HG003273</span>, <span class="grant">U54HG3067</span>, <span class="grant">U54HG3079</span>, <span class="grant">U54HG3273</span>; NHLBI NIH HHS: <span class="grant">HL078885</span>, <span class="grant">R01HL95045</span>, <span class="grant">RC2HL102925</span>, <span class="grant">T32HL94284</span>; NIA NIH HHS: <span class="grant">P30 AG038072</span>; NIAID NIH HHS: <span class="grant">AI077439</span>, <span class="grant">AI2009061</span>; NIEHS NIH HHS: <span class="grant">ES015794</span>; NIGMS NIH HHS: <span class="grant">R01GM59290</span>, <span class="grant">T32GM7748</span>, <span class="grant">T32GM8283</span>; NIH HHS: <span class="grant">DP2OD6514</span>; NIMH NIH HHS: <span class="grant">R01MH84698</span>; NIMHD NIH HHS: <span class="grant">G12 MD007579</span>; NLM NIH HHS: <span class="grant">T15 LM007056</span>, <span class="grant">T15LM7033</span>; PHS HHS: <span class="grant">HHSN268201100040C</span>; Wellcome Trust: <span class="grant">085532</span>, <span class="grant">086084</span>, <span class="grant">090532</span>, <span class="grant">095908</span>, <span class="grant">WT085475/Z/08/Z</span>, <span class="grant">WT085532AIA</span>, <span class="grant">WT086084/Z/08/Z</span>, <span class="grant">WT089250/Z/09/Z</span>, <span class="grant">WT090532/Z/09/Z</span>, <span class="grant">WT095552/Z/11/Z</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">491</span>;<span class="number">7422</span>;<span class="pagerange">56-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23128226">23128226</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3498066">3498066</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11632">10.1038/nature11632</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of context-dependent errors for illumina sequencing.</h4>
<p class="authors"><span class="author">Abnizova I</span>, <span class="author">Leonard S</span>, <span class="author">Skelly T</span>, <span class="author">Brown A</span>, <span class="author">Jackson D</span>, <span class="author">Gourtovaia M</span>, <span class="author">Qi G</span>, <span class="author">Te Boekhorst R</span>, <span class="author">Faruque N</span>, <span class="author">Lewis K</span> and <span class="author">Cox T</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%69%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The new generation of short-read sequencing technologies requires reliable measures of data quality. Such measures are especially important for variant calling. However, in the particular case of SNP calling, a great number of false-positive SNPs may be obtained. One needs to distinguish putative SNPs from sequencing or other errors. We found that not only the probability of sequencing errors (i.e. the quality value) is important to distinguish an FP-SNP but also the conditional probability of &quot;correcting&quot; this error (the &quot;second best call&quot; probability, conditional on that of the first call). Surprisingly, around 80% of mismatches can be &quot;corrected&quot; with this second call. Another way to reduce the rate of FP-SNPs is to retrieve DNA motifs that seem to be prone to sequencing errors, and to attach a corresponding conditional quality value to these motifs. We have developed several measures to distinguish between sequence errors and candidate SNPs, based on a base call&#39;s nucleotide context and its mismatch type. In addition, we suggested a simple method to correct the majority of mismatches, based on conditional probability of their &quot;second&quot; best intensity call. We attach a corresponding second call confidence (quality value) of being corrected to each mismatch.</p>
<p class="publication"><span class="title">Journal of bioinformatics and computational biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">2</span>;<span class="pagerange">1241005</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22809341">22809341</a>; DOI: <a href="http://dx.doi.org/10.1142%2FS0219720012410053">10.1142/S0219720012410053</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multilocus sequence typing as a replacement for serotyping in Salmonella enterica.</h4>
<p class="authors"><span class="author">Achtman M</span>, <span class="author">Wain J</span>, <span class="author">Weill FX</span>, <span class="author">Nair S</span>, <span class="author">Zhou Z</span>, <span class="author">Sangal V</span>, <span class="author">Krauland MG</span>, <span class="author">Hale JL</span>, <span class="author">Harbottle H</span>, <span class="author">Uesbeck A</span>, <span class="author">Dougan G</span>, <span class="author">Harrison LH</span>, <span class="author">Brisse S</span> and <span class="author">S. Enterica MLST Study Group</span></p>
<p class="affiliation">Environmental Research Institute and Department of Microbiology, University College Cork, Cork, Ireland. <a href="mailto:%6D.%61%63%68%74%6D%61%6E%40%75%63%63.%69%65">&#109;&#46;&#97;&#99;&#104;&#116;&#109;&#97;&#110;&#64;&#117;&#99;&#99;&#46;&#105;&#101;</a></p>
<p class="abstract">Salmonella enterica subspecies enterica is traditionally subdivided into serovars by serological and nutritional characteristics. We used Multilocus Sequence Typing (MLST) to assign 4,257 isolates from 554 serovars to 1092 sequence types (STs). The majority of the isolates and many STs were grouped into 138 genetically closely related clusters called eBurstGroups (eBGs). Many eBGs correspond to a serovar, for example most Typhimurium are in eBG1 and most Enteritidis are in eBG4, but many eBGs contained more than one serovar. Furthermore, most serovars were polyphyletic and are distributed across multiple unrelated eBGs. Thus, serovar designations confounded genetically unrelated isolates and failed to recognize natural evolutionary groupings. An inability of serotyping to correctly group isolates was most apparent for Paratyphi B and its variant Java. Most Paratyphi B were included within a sub-cluster of STs belonging to eBG5, which also encompasses a separate sub-cluster of Java STs. However, diphasic Java variants were also found in two other eBGs and monophasic Java variants were in four other eBGs or STs, one of which is in subspecies salamae and a second of which includes isolates assigned to Enteritidis, Dublin and monophasic Paratyphi B. Similarly, Choleraesuis was found in eBG6 and is closely related to Paratyphi C, which is in eBG20. However, Choleraesuis var. Decatur consists of isolates from seven other, unrelated eBGs or STs. The serological assignment of these Decatur isolates to Choleraesuis likely reflects lateral gene transfer of flagellar genes between unrelated bacteria plus purifying selection. By confounding multiple evolutionary groups, serotyping can be misleading about the disease potential of S. enterica. Unlike serotyping, MLST recognizes evolutionary groupings and we recommend that Salmonella classification by serotyping should be replaced by MLST or its equivalents.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">e1002776</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22737074" rel="external">22737074</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3380943" rel="external">3380943</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002776" rel="external">10.1371/journal.ppat.1002776</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BLUEPRINT to decode the epigenetic signature written in blood.</h4>
<p class="authors"><span class="author">Adams D</span>, <span class="author">Altucci L</span>, <span class="author">Antonarakis SE</span>, <span class="author">Ballesteros J</span>, <span class="author">Beck S</span>, <span class="author">Bird A</span>, <span class="author">Bock C</span>, <span class="author">Boehm B</span>, <span class="author">Campo E</span>, <span class="author">Caricasole A</span>, <span class="author">Dahl F</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Enver T</span>, <span class="author">Esteller M</span>, <span class="author">Estivill X</span>, <span class="author">Ferguson-Smith A</span>, <span class="author">Fitzgibbon J</span>, <span class="author">Flicek P</span>, <span class="author">Giehl C</span>, <span class="author">Graf T</span>, <span class="author">Grosveld F</span>, <span class="author">Guigo R</span>, <span class="author">Gut I</span>, <span class="author">Helin K</span>, <span class="author">Jarvius J</span>, <span class="author">K&uuml;ppers R</span>, <span class="author">Lehrach H</span>, <span class="author">Lengauer T</span>, <span class="author">Lernmark &Aring;</span>, <span class="author">Leslie D</span>, <span class="author">Loeffler M</span>, <span class="author">Macintyre E</span>, <span class="author">Mai A</span>, <span class="author">Martens JH</span>, <span class="author">Minucci S</span>, <span class="author">Ouwehand WH</span>, <span class="author">Pelicci PG</span>, <span class="author">Pendeville H</span>, <span class="author">Porse B</span>, <span class="author">Rakyan V</span>, <span class="author">Reik W</span>, <span class="author">Schrappe M</span>, <span class="author">Sch&uuml;beler D</span>, <span class="author">Seifert M</span>, <span class="author">Siebert R</span>, <span class="author">Simmons D</span>, <span class="author">Soranzo N</span>, <span class="author">Spicuglia S</span>, <span class="author">Stratton M</span>, <span class="author">Stunnenberg HG</span>, <span class="author">Tanay A</span>, <span class="author">Torrents D</span>, <span class="author">Valencia A</span>, <span class="author">Vellenga E</span>, <span class="author">Vingron M</span>, <span class="author">Walter J</span> and <span class="author">Willcocks S</span></p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0801156</span>, <span class="grant">MR/J001597/1</span>; Wellcome Trust: <span class="grant">079249</span>, <span class="grant">095606</span>, <span class="grant">095645</span>, <span class="grant">095908</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2012</span>;<span class="volume">30</span>;<span class="number">3</span>;<span class="pagerange">224-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22398613">22398613</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.2153">10.1038/nbt.2153</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A suggested new bacteriophage genus: &quot;Viunalikevirus&quot;.</h4>
<p class="authors"><span class="author">Adriaenssens EM</span>, <span class="author">Ackermann HW</span>, <span class="author">Anany H</span>, <span class="author">Blasdel B</span>, <span class="author">Connerton IF</span>, <span class="author">Goulding D</span>, <span class="author">Griffiths MW</span>, <span class="author">Hooton SP</span>, <span class="author">Kutter EM</span>, <span class="author">Kropinski AM</span>, <span class="author">Lee JH</span>, <span class="author">Maes M</span>, <span class="author">Pickard D</span>, <span class="author">Ryu S</span>, <span class="author">Sepehrizadeh Z</span>, <span class="author">Shahrbabak SS</span>, <span class="author">Toribio AL</span> and <span class="author">Lavigne R</span></p>
<p class="affiliation">Laboratory of Gene Technology, Katholieke Universiteit Leuven, Kasteelpark Arenberg 21, Heverlee, Belgium.</p><p class="abstract">We suggest a bacteriophage genus, &quot;Viunalikevirus&quot;, as a new genus within the family Myoviridae. To date, this genus includes seven sequenced members: Salmonella phages ViI, SFP10 and &Phi;SH19; Escherichia phages CBA120 and PhaxI; Shigella phage phiSboM-AG3; and Dickeya phage LIMEstone1. Their shared myovirus morphology, with comparable head sizes and tail dimensions, and genome organization are considered distinguishing features. They appear to have conserved regulatory sequences, a horizontally acquired tRNA set and the probable substitution of an alternate base for thymine in the DNA. A close examination of the tail spike region in the DNA revealed four distinct tail spike proteins, an arrangement which might lead to the umbrella-like structures of the tails visible on electron micrographs. These properties set the suggested genus apart from the recently ratified subfamily Tevenvirinae, although a significant evolutionary relationship can be observed.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">2R15GM63637-3A1</span></p>
<p class="publication"><span class="title">Archives of virology</span>&nbsp;<span class="year">2012</span>;<span class="volume">157</span>;<span class="number">10</span>;<span class="pagerange">2035-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22707043">22707043</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00705-012-1360-5">10.1007/s00705-012-1360-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Work-related exhaustion and telomere length: a population-based study.</h4>
<p class="authors"><span class="author">Ahola K</span>, <span class="author">Sir&eacute;n I</span>, <span class="author">Kivim&auml;ki M</span>, <span class="author">Ripatti S</span>, <span class="author">Aromaa A</span>, <span class="author">L&ouml;nnqvist J</span> and <span class="author">Hovatta I</span></p>
<p class="affiliation">Work Organizations, Finnish Institute of Occupational Health, Helsinki, Finland.</p>
<p class="abstract">Background: Psychological stress is suggested to accelerate the rate of biological aging. We investigated whether work-related exhaustion, an indicator of prolonged work stress, is associated with accelerated biological aging, as indicated by shorter leukocyte telomeres, that is, the DNA-protein complexes that cap chromosomal ends in cells.

Methods: We used data from a representative sample of the Finnish working-age population, the Health 2000 Study. Our sample consisted of 2911 men and women aged 30-64. Work-related exhaustion was assessed using the Maslach Burnout Inventory - General Survey. We determined relative leukocyte telomere length using a quantitative real-time polymerase chain reaction (PCR) -based method.

Results: After adjustment for age and sex, individuals with severe exhaustion had leukocyte telomeres on average 0.043 relative units shorter (standard error of the mean 0.016) than those with no exhaustion (pâ€Š=â€Š0.009). The association between exhaustion and relative telomere length remained significant after additional adjustment for marital and socioeconomic status, smoking, body mass index, and morbidities (adjusted difference 0.044 relative units, standard error of the mean 0.017, pâ€Š=â€Š0.008).

Conclusions: These data suggest that work-related exhaustion is related to the acceleration of the rate of biological aging. This hypothesis awaits confirmation in a prospective study measuring changes in relative telomere length over time.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e40186</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22808115" rel="external">22808115</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3394788" rel="external">3394788</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0040186" rel="external">10.1371/journal.pone.0040186</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phenotypic and Genomic Analysis of Hypervirulent Human-associated Bordetella bronchiseptica.</h4>
<p class="authors"><span class="author">Ahuja U</span>, <span class="author">Liu M</span>, <span class="author">Tomida S</span>, <span class="author">Park J</span>, <span class="author">Souda P</span>, <span class="author">Whitelegge J</span>, <span class="author">Li H</span>, <span class="author">Harvill ET</span>, <span class="author">Parkhill J</span> and <span class="author">Miller JF</span></p>
<p class="affiliation">Department of Microbiology, Immunology and Molecular Genetics, University of California, BSRB 254, 615 Charles E, Young Drive East, Los Angeles, CA, 90095-1747, USA. <a href="mailto:%6A%66%6D%69%6C%6C%65%72%40%75%63%6C%61.%65%64%75">&#106;&#102;&#109;&#105;&#108;&#108;&#101;&#114;&#64;&#117;&#99;&#108;&#97;&#46;&#101;&#100;&#117;</a>.</p>
<p class="abstract">Unlabelled: ABSTRACT:

Background: B. bronchiseptica infections are usually associated with wild or domesticated animals, but infrequently with humans. A recent phylogenetic analysis distinguished two distinct B. bronchiseptica subpopulations, designated complexes I and IV. Complex IV isolates appear to have a bias for infecting humans; however, little is known regarding their epidemiology, virulence properties, or comparative genomics.

Results: Here we report a characterization of the virulence of human-associated complex IV B. bronchiseptica strains. In in vitro cytotoxicity assays, complex IV strains showed increased cytotoxicity in comparison to a panel of complex I strains. Some complex IV isolates were remarkably cytotoxic, resulting in LDH release levels in A549 cells that were 10- to 20-fold greater than complex I strains. In vivo, a subset of complex IV strains was found to be hypervirulent, with an increased ability to cause lethal pulmonary infections in mice. Hypercytotoxicity in vitro and hypervirulence in vivo were both dependent on the activity of the bsc T3SS and the BteA effector. To clarify differences between lineages, representative complex IV isolates were sequenced and their genomes were compared to complex I isolates. Although our analysis showed there were no genomic sequences that can be considered unique to complex IV strains, there were several loci that were predominantly found in complex IV isolates.

Conclusion: Our observations reveal a T3SS-dependent hypervirulence phenotype in human-associated complex IV isolates, highlighting the need for further studies on the epidemiology and evolutionary dynamics of this B. bronchiseptica lineage.</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="pagerange">167</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22863321" rel="external">22863321</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3462115" rel="external">3462115</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-12-167" rel="external">10.1186/1471-2180-12-167</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Compound inheritance of a low-frequency regulatory SNP and a rare null mutation in exon-junction complex subunit RBM8A causes TAR syndrome.</h4>
<p class="authors"><span class="author">Albers CA</span>, <span class="author">Paul DS</span>, <span class="author">Schulze H</span>, <span class="author">Freson K</span>, <span class="author">Stephens JC</span>, <span class="author">Smethurst PA</span>, <span class="author">Jolley JD</span>, <span class="author">Cvejic A</span>, <span class="author">Kostadima M</span>, <span class="author">Bertone P</span>, <span class="author">Breuning MH</span>, <span class="author">Debili N</span>, <span class="author">Deloukas P</span>, <span class="author">Favier R</span>, <span class="author">Fiedler J</span>, <span class="author">Hobbs CM</span>, <span class="author">Huang N</span>, <span class="author">Hurles ME</span>, <span class="author">Kiddle G</span>, <span class="author">Krapels I</span>, <span class="author">Nurden P</span>, <span class="author">Ruivenkamp CA</span>, <span class="author">Sambrook JG</span>, <span class="author">Smith K</span>, <span class="author">Stemple DL</span>, <span class="author">Strauss G</span>, <span class="author">Thys C</span>, <span class="author">van Geet C</span>, <span class="author">Newbury-Ecob R</span>, <span class="author">Ouwehand WH</span> and <span class="author">Ghevaert C</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, Cambridge, UK. <a href="mailto:%63%61%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The exon-junction complex (EJC) performs essential RNA processing tasks. Here, we describe the first human disorder, thrombocytopenia with absent radii (TAR), caused by deficiency in one of the four EJC subunits. Compound inheritance of a rare null allele and one of two low-frequency SNPs in the regulatory regions of RBM8A, encoding the Y14 subunit of EJC, causes TAR. We found that this inheritance mechanism explained 53 of 55 cases (P &lt; 5 &times; 10(-228)) of the rare congenital malformation syndrome. Of the 53 cases with this inheritance pattern, 51 carried a submicroscopic deletion of 1q21.1 that has previously been associated with TAR, and two carried a truncation or frameshift null mutation in RBM8A. We show that the two regulatory SNPs result in diminished RBM8A transcription in vitro and that Y14 expression is reduced in platelets from individuals with TAR. Our data implicate Y14 insufficiency and, presumably, an EJC defect as the cause of TAR syndrome.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/09/039</span>, <span class="grant">FS/09/039/27788</span>, <span class="grant">RG/09/012/28096</span>, <span class="grant">RG/09/12/28096</span>; Wellcome Trust: <span class="grant">082597</span>, <span class="grant">084183</span>, <span class="grant">WT-082597/Z/07/Z</span>, <span class="grant">WT-084183/2/07/2</span>, <span class="grant">WT091310</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">4</span>;<span class="pagerange">435-9, S1-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22366785">22366785</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428915">3428915</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1083">10.1038/ng.1083</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Powerful Identification of Cis-regulatory SNPs in Human Primary Monocytes Using Allele-Specific Gene Expression.</h4>
<p class="authors"><span class="author">Alml&ouml;f JC</span>, <span class="author">Lundmark P</span>, <span class="author">Lundmark A</span>, <span class="author">Ge B</span>, <span class="author">Maouche S</span>, <span class="author">G&ouml;ring HH</span>, <span class="author">Liljedahl U</span>, <span class="author">Enstr&ouml;m C</span>, <span class="author">Brocheton J</span>, <span class="author">Proust C</span>, <span class="author">Godefroy T</span>, <span class="author">Sambrook JG</span>, <span class="author">Jolley J</span>, <span class="author">Crisp-Hihn A</span>, <span class="author">Foad N</span>, <span class="author">Lloyd-Jones H</span>, <span class="author">Stephens J</span>, <span class="author">Gwilliam R</span>, <span class="author">Rice CM</span>, <span class="author">Hengstenberg C</span>, <span class="author">Samani NJ</span>, <span class="author">Erdmann J</span>, <span class="author">Schunkert H</span>, <span class="author">Pastinen T</span>, <span class="author">Deloukas P</span>, <span class="author">Goodall AH</span>, <span class="author">Ouwehand WH</span>, <span class="author">Cambien F</span> and <span class="author">Syv&auml;nen AC</span></p>
<p class="affiliation">Department of Medical Sciences, Molecular Medicine and Science for Life Laboratory, Uppsala University, Uppsala, Sweden.</p>
<p class="abstract">A large number of genome-wide association studies have been performed during the past five years to identify associations between SNPs and human complex diseases and traits. The assignment of a functional role for the identified disease-associated SNP is not straight-forward. Genome-wide expression quantitative trait locus (eQTL) analysis is frequently used as the initial step to define a function while allele-specific gene expression (ASE) analysis has not yet gained a wide-spread use in disease mapping studies. We compared the power to identify cis-acting regulatory SNPs (cis-rSNPs) by genome-wide allele-specific gene expression (ASE) analysis with that of traditional expression quantitative trait locus (eQTL) mapping. Our study included 395 healthy blood donors for whom global gene expression profiles in circulating monocytes were determined by Illumina BeadArrays. ASE was assessed in a subset of these monocytes from 188 donors by quantitative genotyping of mRNA using a genome-wide panel of SNP markers. The performance of the two methods for detecting cis-rSNPs was evaluated by comparing associations between SNP genotypes and gene expression levels in sample sets of varying size. We found that up to 8-fold more samples are required for eQTL mapping to reach the same statistical power as that obtained by ASE analysis for the same rSNPs. The performance of ASE is insensitive to SNPs with low minor allele frequencies and detects a larger number of significantly associated rSNPs using the same sample size as eQTL mapping. An unequivocal conclusion from our comparison is that ASE analysis is more sensitive for detecting cis-rSNPs than standard eQTL mapping. Our study shows the potential of ASE mapping in tissue samples and primary cells which are difficult to obtain in large numbers.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e52260</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23300628" rel="external">23300628</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0052260" rel="external">10.1371/journal.pone.0052260</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput decoding of antitrypanosomal drug efficacy and resistance.</h4>
<p class="authors"><span class="author">Alsford S</span>, <span class="author">Eckert S</span>, <span class="author">Baker N</span>, <span class="author">Glover L</span>, <span class="author">Sanchez-Flores A</span>, <span class="author">Leung KF</span>, <span class="author">Turner DJ</span>, <span class="author">Field MC</span>, <span class="author">Berriman M</span> and <span class="author">Horn D</span></p>
<p class="affiliation">London School of Hygiene and Tropical Medicine, Keppel Street, London, WC1E 7HT, UK.</p><p class="abstract">The concept of disease-specific chemotherapy was developed a century ago. Dyes and arsenical compounds that displayed selectivity against trypanosomes were central to this work, and the drugs that emerged remain in use for treating human African trypanosomiasis (HAT). The importance of understanding the mechanisms underlying selective drug action and resistance for the development of improved HAT therapies has been recognized, but these mechanisms have remained largely unknown. Here we use all five current HAT drugs for genome-scale RNA interference target sequencing (RIT-seq) screens in Trypanosoma brucei, revealing the transporters, organelles, enzymes and metabolic pathways that function to facilitate antitrypanosomal drug action. RIT-seq profiling identifies both known drug importers and the only known pro-drug activator, and links more than fifty additional genes to drug action. A bloodstream stage-specific invariant surface glycoprotein (ISG75) family mediates suramin uptake, and the AP1 adaptin complex, lysosomal proteases and major lysosomal transmembrane protein, as well as spermidine and N-acetylglucosamine biosynthesis, all contribute to suramin action. Further screens link ubiquinone availability to nitro-drug action, plasma membrane P-type H(+)-ATPases to pentamidine action, and trypanothione and several putative kinases to melarsoprol action. We also demonstrate a major role for aquaglyceroporins in pentamidine and melarsoprol cross-resistance. These advances in our understanding of mechanisms of antitrypanosomal drug efficacy and resistance will aid the rational design of new therapies and help to combat drug resistance, and provide unprecedented molecular insight into the mode of action of antitrypanosomal drugs.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085775</span>, <span class="grant">085775/Z/08/Z</span>, <span class="grant">090007</span>, <span class="grant">090007/Z/09/Z</span>, <span class="grant">093010</span>, <span class="grant">093010/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">482</span>;<span class="number">7384</span>;<span class="pagerange">232-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22278056">22278056</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3303116">3303116</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10771">10.1038/nature10771</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genomic scan for candidate signatures of balancing selection to guide antigen characterization in malaria parasites.</h4>
<p class="authors"><span class="author">Amambua-Ngwa A</span>, <span class="author">Tetteh KK</span>, <span class="author">Manske M</span>, <span class="author">Gomez-Escobar N</span>, <span class="author">Stewart LB</span>, <span class="author">Deerhake ME</span>, <span class="author">Cheeseman IH</span>, <span class="author">Newbold CI</span>, <span class="author">Holder AA</span>, <span class="author">Knuepfer E</span>, <span class="author">Janha O</span>, <span class="author">Jallow M</span>, <span class="author">Campino S</span>, <span class="author">Macinnis B</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Conway DJ</span></p>
<p class="affiliation">Medical Research Council Unit, Fajara, Banjul, The Gambia.</p>
<p class="abstract">Acquired immunity in vertebrates maintains polymorphisms in endemic pathogens, leading to identifiable signatures of balancing selection. To comprehensively survey for genes under such selection in the human malaria parasite Plasmodium falciparum, we generated paired-end short-read sequences of parasites in clinical isolates from an endemic Gambian population, which were mapped to the 3D7 strain reference genome to yield high-quality genome-wide coding sequence data for 65 isolates. A minority of genes did not map reliably, including the hypervariable var, rifin, and stevor families, but 5,056 genes (90.9% of all in the genome) had >70% sequence coverage with minimum read depth of 5 for at least 50 isolates, of which 2,853 genes contained 3 or more single nucleotide polymorphisms (SNPs) for analysis of polymorphic site frequency spectra. Against an overall background of negatively skewed frequencies, as expected from historical population expansion combined with purifying selection, the outlying minority of genes with signatures indicating exceptionally intermediate frequencies were identified. Comparing genes with different stage-specificity, such signatures were most common in those with peak expression at the merozoite stage that invades erythrocytes. Members of clag, PfMC-2TM, surfin, and msp3-like gene families were highly represented, the strongest signature being in the msp3-like gene PF10_0355. Analysis of msp3-like transcripts in 45 clinical and 11 laboratory adapted isolates grown to merozoite-containing schizont stages revealed surprisingly low expression of PF10_0355. In diverse clonal parasite lines the protein product was expressed in a minority of mature schizonts (<1% in most lines and ∼10% in clone HB3), and eight sub-clones of HB3 cultured separately had an intermediate spectrum of positive frequencies (0.9 to 7.5%), indicating phase variable expression of this polymorphic antigen. This and other identified targets of balancing selection are now prioritized for functional study.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">U117532067</span>; Wellcome Trust: <span class="grant">074695/Z/04/B</span>, <span class="grant">090770/Z/09/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e1002992</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23133397">23133397</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3486833">3486833</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002992">10.1371/journal.pgen.1002992</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis of coffee drinking suggests association with CYP1A1/CYP1A2 and NRCAM.</h4>
<p class="authors"><span class="author">Amin N</span>, <span class="author">Byrne E</span>, <span class="author">Johnson J</span>, <span class="author">Chenevix-Trench G</span>, <span class="author">Walter S</span>, <span class="author">Nolte IM</span>, <span class="author">kConFab Investigators</span>, <span class="author">Vink JM</span>, <span class="author">Rawal R</span>, <span class="author">Mangino M</span>, <span class="author">Teumer A</span>, <span class="author">Keers JC</span>, <span class="author">Verwoert G</span>, <span class="author">Baumeister S</span>, <span class="author">Biffar R</span>, <span class="author">Petersmann A</span>, <span class="author">Dahmen N</span>, <span class="author">Doering A</span>, <span class="author">Isaacs A</span>, <span class="author">Broer L</span>, <span class="author">Wray NR</span>, <span class="author">Montgomery GW</span>, <span class="author">Levy D</span>, <span class="author">Psaty BM</span>, <span class="author">Gudnason V</span>, <span class="author">Chakravarti A</span>, <span class="author">Sulem P</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Kiemeney LA</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">van Rooij FJ</span>, <span class="author">Aulchenko YS</span>, <span class="author">Hottenga JJ</span>, <span class="author">Rivadeneira FR</span>, <span class="author">Hofman A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hammond CJ</span>, <span class="author">Shin SY</span>, <span class="author">Ikram A</span>, <span class="author">Witteman JC</span>, <span class="author">Janssens AC</span>, <span class="author">Snieder H</span>, <span class="author">Tiemeier H</span>, <span class="author">Wolfenbuttel BH</span>, <span class="author">Oostra BA</span>, <span class="author">Heath AC</span>, <span class="author">Wichmann E</span>, <span class="author">Spector TD</span>, <span class="author">Grabe HJ</span>, <span class="author">Boomsma DI</span>, <span class="author">Martin NG</span> and <span class="author">van Duijn CM</span></p>
<p class="affiliation">Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands.</p>
<p class="abstract">Coffee consumption is a model for addictive behavior. We performed a meta-analysis of genome-wide association studies (GWASs) on coffee intake from 8 Caucasian cohorts (N=18â€‰176) and sought replication of our top findings in a further 7929 individuals. We also performed a gene expression analysis treating different cell lines with caffeine. Genome-wide significant association was observed for two single-nucleotide polymorphisms (SNPs) in the 15q24 region. The two SNPs rs2470893 and rs2472297 (P-values=1.6 Ã— 10(-11) and 2.7 Ã— 10(-11)), which were also in strong linkage disequilibrium (r(2)=0.7) with each other, lie in the 23-kb long commonly shared 5' flanking region between CYP1A1 and CYP1A2 genes. CYP1A1 was found to be downregulated in lymphoblastoid cell lines treated with caffeine. CYP1A1 is known to metabolize polycyclic aromatic hydrocarbons, which are important constituents of coffee, whereas CYP1A2 is involved in the primary metabolism of caffeine. Significant evidence of association was also detected at rs382140 (P-value=3.9 Ã— 10(-09)) near NRCAM-a gene implicated in vulnerability to addiction, and at another independent hit rs6495122 (P-value=7.1 Ã— 10(-09))-an SNP associated with blood pressure-in the 15q24 region near the gene ULK3, in the meta-analysis of discovery and replication cohorts. Our results from GWASs and expression analysis also strongly implicate CAB39L in coffee drinking. Pathway analysis of differentially expressed genes revealed significantly enriched ubiquitin proteasome (P-value=2.2 Ã— 10(-05)) and Parkinson's disease pathways (P-value=3.6 Ã— 10(-05)).</p>
<p class="grants">Funded by: NIAAA NIH HHS: <span class="grant">K05 AA017688</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2012</span>;<span class="volume">17</span>;<span class="number">11</span>;<span class="pagerange">1116-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21876539" rel="external">21876539</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3482684" rel="external">3482684</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2011.101" rel="external">10.1038/mp.2011.101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of high-depth sequence data for studying viral diversity: a comparison of next generation sequencing platforms using Segminator II.</h4>
<p class="authors"><span class="author">Archer J</span>, <span class="author">Baillie G</span>, <span class="author">Watson SJ</span>, <span class="author">Kellam P</span>, <span class="author">Rambaut A</span> and <span class="author">Robertson DL</span></p>
<p class="affiliation">Computational and Evolutionary Biology, Faculty of Life Sciences, University of Manchester, Manchester, UK. <a href="mailto:%6A%6F%68%6E.%61%72%63%68%65%72%40%6D%61%6E%63%68%65%73%74%65%72.%61%63.%75%6B">&#106;&#111;&#104;&#110;&#46;&#97;&#114;&#99;&#104;&#101;&#114;&#64;&#109;&#97;&#110;&#99;&#104;&#101;&#115;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Next generation sequencing provides detailed insight into the variation present within viral populations, introducing the possibility of treatment strategies that are both reactive and predictive. Current software tools, however, need to be scaled up to accommodate for high-depth viral data sets, which are often temporally or spatially linked. In addition, due to the development of novel sequencing platforms and chemistries, each with implicit strengths and weaknesses, it will be helpful for researchers to be able to routinely compare and combine data sets from different platforms/chemistries. In particular, error associated with a specific sequencing process must be quantified so that true biological variation may be identified.</p><p class="abstract"><strong>Results</strong>: Segminator II was developed to allow for the efficient comparison of data sets derived from different sources. We demonstrate its usage by comparing large data sets from 12 influenza H1N1 samples sequenced on both the 454 Life Sciences and Illumina platforms, permitting quantification of platform error. For mismatches median error rates at 0.10 and 0.12%, respectively, suggested that both platforms performed similarly. For insertions and deletions median error rates within the 454 data (at 0.3 and 0.2%, respectively) were significantly higher than those within the Illumina data (0.004 and 0.006%, respectively). In agreement with previous observations these higher rates were strongly associated with homopolymeric stretches on the 454 platform. Outside of such regions both platforms had similar indel error profiles. Additionally, we apply our software to the identification of low frequency variants.</p><p class="abstract"><strong>Conclusion</strong>: We have demonstrated, using Segminator II, that it is possible to distinguish platform specific error from biological variation using data derived from two different platforms. We have used this approach to quantify the amount of error present within the 454 and Illumina platforms in relation to genomic location as well as location on the read. Given that next generation data is increasingly important in the analysis of drug-resistance and vaccine trials, this software will be useful to the pathogen research community. A zip file containing the source code and jar file is freely available for download from http://www.bioinf.manchester.ac.uk/segminator/.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/H012419/1</span>; Wellcome Trust: <span class="grant">095831</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22443413">22443413</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3359224">3359224</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-13-47">10.1186/1471-2105-13-47</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study.</h4>
<p class="authors"><span class="author">arcOGEN Consortium</span>, <span class="author">arcOGEN Collaborators</span>, <span class="author">Zeggini E</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Southam L</span>, <span class="author">Rayner NW</span>, <span class="author">Day-Williams AG</span>, <span class="author">Lopes MC</span>, <span class="author">Boraska V</span>, <span class="author">Esko T</span>, <span class="author">Evangelou E</span>, <span class="author">Hoffman A</span>, <span class="author">Houwing-Duistermaat JJ</span>, <span class="author">Ingvarsson T</span>, <span class="author">Jonsdottir I</span>, <span class="author">Jonnson H</span>, <span class="author">Kerkhof HJ</span>, <span class="author">Kloppenburg M</span>, <span class="author">Bos SD</span>, <span class="author">Mangino M</span>, <span class="author">Metrustry S</span>, <span class="author">Slagboom PE</span>, <span class="author">Thorleifsson G</span>, <span class="author">Raine EV</span>, <span class="author">Ratnayake M</span>, <span class="author">Ricketts M</span>, <span class="author">Beazley C</span>, <span class="author">Blackburn H</span>, <span class="author">Bumpstead S</span>, <span class="author">Elliott KS</span>, <span class="author">Hunt SE</span>, <span class="author">Potter SC</span>, <span class="author">Shin SY</span>, <span class="author">Yadav VK</span>, <span class="author">Zhai G</span>, <span class="author">Sherburn K</span>, <span class="author">Dixon K</span>, <span class="author">Arden E</span>, <span class="author">Aslam N</span>, <span class="author">Battley PK</span>, <span class="author">Carluke I</span>, <span class="author">Doherty S</span>, <span class="author">Gordon A</span>, <span class="author">Joseph J</span>, <span class="author">Keen R</span>, <span class="author">Koller NC</span>, <span class="author">Mitchell S</span>, <span class="author">O&#39;Neill F</span>, <span class="author">Paling E</span>, <span class="author">Reed MR</span>, <span class="author">Rivadeneira F</span>, <span class="author">Swift D</span>, <span class="author">Walker K</span>, <span class="author">Watkins B</span>, <span class="author">Wheeler M</span>, <span class="author">Birrell F</span>, <span class="author">Ioannidis JP</span>, <span class="author">Meulenbelt I</span>, <span class="author">Metspalu A</span>, <span class="author">Rai A</span>, <span class="author">Salter D</span>, <span class="author">Stefansson K</span>, <span class="author">Stykarsdottir U</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Meurs JB</span>, <span class="author">Chapman K</span>, <span class="author">Deloukas P</span>, <span class="author">Ollier WE</span>, <span class="author">Wallis GA</span>, <span class="author">Arden N</span>, <span class="author">Carr A</span>, <span class="author">Doherty M</span>, <span class="author">McCaskie A</span>, <span class="author">Willkinson JM</span>, <span class="author">Ralston SH</span>, <span class="author">Valdes AM</span>, <span class="author">Spector TD</span> and <span class="author">Loughlin J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Morgan Building, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1HH, UK. <a href="mailto:%65%6C%65%66%74%68%65%72%69%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#108;&#101;&#102;&#116;&#104;&#101;&#114;&#105;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Osteoarthritis is the most common form of arthritis worldwide and is a major cause of pain and disability in elderly people. The health economic burden of osteoarthritis is increasing commensurate with obesity prevalence and longevity. Osteoarthritis has a strong genetic component but the success of previous genetic studies has been restricted due to insufficient sample sizes and phenotype heterogeneity.</p><p class="abstract"><strong>Methods</strong>: We undertook a large genome-wide association study (GWAS) in 7410 unrelated and retrospectively and prospectively selected patients with severe osteoarthritis in the arcOGEN study, 80% of whom had undergone total joint replacement, and 11,009 unrelated controls from the UK. We replicated the most promising signals in an independent set of up to 7473 cases and 42,938 controls, from studies in Iceland, Estonia, the Netherlands, and the UK. All patients and controls were of European descent.</p><p class="abstract"><strong>Findings</strong>: We identified five genome-wide significant loci (binomial test p&le;5&middot;0&times;10(-8)) for association with osteoarthritis and three loci just below this threshold. The strongest association was on chromosome 3 with rs6976 (odds ratio 1&middot;12 [95% CI 1&middot;08-1&middot;16]; p=7&middot;24&times;10(-11)), which is in perfect linkage disequilibrium with rs11177. This SNP encodes a missense polymorphism within the nucleostemin-encoding gene GNL3. Levels of nucleostemin were raised in chondrocytes from patients with osteoarthritis in functional studies. Other significant loci were on chromosome 9 close to ASTN2, chromosome 6 between FILIP1 and SENP6, chromosome 12 close to KLHDC5 and PTHLH, and in another region of chromosome 12 close to CHST11. One of the signals close to genome-wide significance was within the FTO gene, which is involved in regulation of bodyweight-a strong risk factor for osteoarthritis. All risk variants were common in frequency and exerted small effects.</p><p class="abstract"><strong>Interpretation</strong>: Our findings provide insight into the genetics of arthritis and identify new pathways that might be amenable to future therapeutic intervention.</p><p class="abstract"><strong>Funding</strong>: arcOGEN was funded by a special purpose grant from Arthritis Research UK.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">18030</span>; Medical Research Council: <span class="grant">G0100594</span>, <span class="grant">G0901461</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2012</span>;<span class="volume">380</span>;<span class="number">9844</span>;<span class="pagerange">815-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22763110">22763110</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3443899">3443899</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2812%2960681-3">10.1016/S0140-6736(12)60681-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population differentiation of southern Indian male lineages correlates with agricultural expansions predating the caste system.</h4>
<p class="authors"><span class="author">Arunkumar G</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Kavitha VJ</span>, <span class="author">Arun VS</span>, <span class="author">Syama A</span>, <span class="author">Ashokan KS</span>, <span class="author">Gandhirajan KT</span>, <span class="author">Vijayakumar K</span>, <span class="author">Narayanan M</span>, <span class="author">Jayalakshmi M</span>, <span class="author">Ziegle JS</span>, <span class="author">Royyuru AK</span>, <span class="author">Parida L</span>, <span class="author">Wells RS</span>, <span class="author">Renfrew C</span>, <span class="author">Schurr TG</span>, <span class="author">Smith CT</span>, <span class="author">Platt DE</span>, <span class="author">Pitchappan R</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">The Genographic Laboratory, School of Biological Sciences, Madurai Kamaraj University, Madurai, Tamil Nadu, India.</p><p class="abstract">Previous studies that pooled Indian populations from a wide variety of geographical locations, have obtained contradictory conclusions about the processes of the establishment of the Varna caste system and its genetic impact on the origins and demographic histories of Indian populations. To further investigate these questions we took advantage that both Y chromosome and caste designation are paternally inherited, and genotyped 1,680 Y chromosomes representing 12 tribal and 19 non-tribal (caste) endogamous populations from the predominantly Dravidian-speaking Tamil Nadu state in the southernmost part of India. Tribes and castes were both characterized by an overwhelming proportion of putatively Indian autochthonous Y-chromosomal haplogroups (H-M69, F-M89, R1a1-M17, L1-M27, R2-M124, and C5-M356; 81% combined) with a shared genetic heritage dating back to the late Pleistocene (10-30 Kya), suggesting that more recent Holocene migrations from western Eurasia contributed &lt;20% of the male lineages. We found strong evidence for genetic structure, associated primarily with the current mode of subsistence. Coalescence analysis suggested that the social stratification was established 4-6 Kya and there was little admixture during the last 3 Kya, implying a minimal genetic impact of the Varna (caste) system from the historically-documented Brahmin migrations into the area. In contrast, the overall Y-chromosomal patterns, the time depth of population diversifications and the period of differentiation were best explained by the emergence of agricultural technology in South Asia. These results highlight the utility of detailed local genetic studies within India, without prior assumptions about the importance of Varna rank status for population grouping, to obtain new insights into the relative influences of past demographic events for the population structure of the whole of modern India.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">11</span>;<span class="pagerange">e50269</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23209694">23209694</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3508930">3508930</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0050269">10.1371/journal.pone.0050269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evaluation of different meta-analysis approaches in the presence of allelic heterogeneity.</h4>
<p class="authors"><span class="author">Asimit J</span>, <span class="author">Day-Williams A</span>, <span class="author">Zgaga L</span>, <span class="author">Rudan I</span>, <span class="author">Boraska V</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Department of Human Genetics, Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">Meta-analysis has proven a useful tool in genetic association studies. Allelic heterogeneity can arise from ethnic background differences across populations being meta-analyzed (for example, in search of common frequency variants through genome-wide association studies), and through the presence of multiple low frequency and rare associated variants in the same functional unit of interest (for example, within a gene or a regulatory region). The latter challenge will be increasingly relevant in whole-genome and whole-exome sequencing studies investigating association with complex traits. Here, we evaluate the performance of different approaches to meta-analysis in the presence of allelic heterogeneity. We simulate allelic heterogeneity scenarios in three populations and examine the performance of current approaches to the analysis of these data. We show that current approaches can detect only a small fraction of common frequency causal variants. We also find that for low-frequency variants with large effects (odds ratios 2-3), single-point tests have high power, but also high false-positive rates. P-value based meta-analysis of summary results from allele-matching locus-wide tests outperforms collapsing approaches. We conclude that current strategies for the combination of genetic association data in the presence of allelic heterogeneity are insufficiently powered.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2012</span>;<span class="volume">20</span>;<span class="number">6</span>;<span class="pagerange">709-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22293689">22293689</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3355266">3355266</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2011.274">10.1038/ejhg.2011.274</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Imputation of rare variants in next-generation association studies.</h4>
<p class="authors"><span class="author">Asimit JL</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%6A%61%31%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#97;&#49;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The role of rare variants has become a focus in the search for association with complex traits. Imputation is a powerful and cost-efficient tool to access variants that have not been directly typed, but there are several challenges when imputing rare variants, most notably reference panel selection. Extensions to rare variant association tests to incorporate genotype uncertainty from imputation are discussed, as well as the use of imputed low-frequency and rare variants in the study of population isolates.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Human heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">74</span>;<span class="number">3-4</span>;<span class="pagerange">196-204</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23594497">23594497</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3954458">3954458</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000345602">10.1159/000345602</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ARIEL and AMELIA: testing for an accumulation of rare variants using next-generation sequencing data.</h4>
<p class="authors"><span class="author">Asimit JL</span>, <span class="author">Day-Williams AG</span>, <span class="author">Morris AP</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract"><strong>Objectives</strong>: There is increasing evidence that rare variants play a role in some complex traits, but their analysis is not straightforward. Locus-based tests become necessary due to low power in rare variant single-point association analyses. In addition, variant quality scores are available for sequencing data, but are rarely taken into account. Here, we propose two locus-based methods that incorporate variant quality scores: a regression-based collapsing approach and an allele-matching method.</p><p class="abstract"><strong>Methods</strong>: Using simulated sequencing data we compare 4 locus-based tests of trait association under different scenarios of data quality. We test two collapsing-based approaches and two allele-matching-based approaches, taking into account variant quality scores and ignoring variant quality scores. We implement the collapsing and allele-matching approaches accounting for variant quality in the freely available ARIEL and AMELIA software.</p><p class="abstract"><strong>Results</strong>: The incorporation of variant quality scores in locus-based association tests has power advantages over weighting each variant equally. The allele-matching methods are robust to the presence of both protective and risk variants in a locus, while collapsing methods exhibit a dramatic loss of power in this scenario.</p><p class="abstract"><strong>Conclusions</strong>: The incorporation of variant quality scores should be a standard protocol when performing locus-based association analysis on sequencing data. The ARIEL and AMELIA software implement collapsing and allele-matching locus association analysis methods, respectively, that allow the incorporation of variant quality scores.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088885</span>, <span class="grant">090532</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Human heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">73</span>;<span class="number">2</span>;<span class="pagerange">84-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22441326">22441326</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3477640">3477640</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000336982">10.1159/000336982</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-Scale Gene-Centric Meta-analysis across 32 Studies Identifies Multiple Lipid Loci.</h4>
<p class="authors"><span class="author">Asselbergs FW</span>, <span class="author">Guo Y</span>, <span class="author">van Iperen EP</span>, <span class="author">Sivapalaratnam S</span>, <span class="author">Tragante V</span>, <span class="author">Lanktree MB</span>, <span class="author">Lange LA</span>, <span class="author">Almoguera B</span>, <span class="author">Appelman YE</span>, <span class="author">Barnard J</span>, <span class="author">Baumert J</span>, <span class="author">Beitelshees AL</span>, <span class="author">Bhangale TR</span>, <span class="author">Chen YD</span>, <span class="author">Gaunt TR</span>, <span class="author">Gong Y</span>, <span class="author">Hopewell JC</span>, <span class="author">Johnson T</span>, <span class="author">Kleber ME</span>, <span class="author">Langaee TY</span>, <span class="author">Li M</span>, <span class="author">Li YR</span>, <span class="author">Liu K</span>, <span class="author">McDonough CW</span>, <span class="author">Meijs MF</span>, <span class="author">Middelberg RP</span>, <span class="author">Musunuru K</span>, <span class="author">Nelson CP</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Padmanabhan S</span>, <span class="author">Pankow JS</span>, <span class="author">Pankratz N</span>, <span class="author">Rafelt S</span>, <span class="author">Rajagopalan R</span>, <span class="author">Romaine SP</span>, <span class="author">Schork NJ</span>, <span class="author">Shaffer J</span>, <span class="author">Shen H</span>, <span class="author">Smith EN</span>, <span class="author">Tischfield SE</span>, <span class="author">van der Most PJ</span>, <span class="author">van Vliet-Ostaptchouk JV</span>, <span class="author">Verweij N</span>, <span class="author">Volcik KA</span>, <span class="author">Zhang L</span>, <span class="author">Bailey KR</span>, <span class="author">Bailey KM</span>, <span class="author">Bauer F</span>, <span class="author">Boer JM</span>, <span class="author">Braund PS</span>, <span class="author">Burt A</span>, <span class="author">Burton PR</span>, <span class="author">Buxbaum SG</span>, <span class="author">Chen W</span>, <span class="author">Cooper-Dehoff RM</span>, <span class="author">Cupples LA</span>, <span class="author">Dejong JS</span>, <span class="author">Delles C</span>, <span class="author">Duggan D</span>, <span class="author">Fornage M</span>, <span class="author">Furlong CE</span>, <span class="author">Glazer N</span>, <span class="author">Gums JG</span>, <span class="author">Hastie C</span>, <span class="author">Holmes MV</span>, <span class="author">Illig T</span>, <span class="author">Kirkland SA</span>, <span class="author">Kivimaki M</span>, <span class="author">Klein R</span>, <span class="author">Klein BE</span>, <span class="author">Kooperberg C</span>, <span class="author">Kottke-Marchant K</span>, <span class="author">Kumari M</span>, <span class="author">Lacroix AZ</span>, <span class="author">Mallela L</span>, <span class="author">Murugesan G</span>, <span class="author">Ordovas J</span>, <span class="author">Ouwehand WH</span>, <span class="author">Post WS</span>, <span class="author">Saxena R</span>, <span class="author">Scharnagl H</span>, <span class="author">Schreiner PJ</span>, <span class="author">Shah T</span>, <span class="author">Shields DC</span>, <span class="author">Shimbo D</span>, <span class="author">Srinivasan SR</span>, <span class="author">Stolk RP</span>, <span class="author">Swerdlow DI</span>, <span class="author">Taylor HA</span>, <span class="author">Topol EJ</span>, <span class="author">Toskala E</span>, <span class="author">van Pelt JL</span>, <span class="author">van Setten J</span>, <span class="author">Yusuf S</span>, <span class="author">Whittaker JC</span>, <span class="author">Zwinderman AH</span>, <span class="author">LifeLines Cohort Study</span>, <span class="author">Anand SS</span>, <span class="author">Balmforth AJ</span>, <span class="author">Berenson GS</span>, <span class="author">Bezzina CR</span>, <span class="author">Boehm BO</span>, <span class="author">Boerwinkle E</span>, <span class="author">Casas JP</span>, <span class="author">Caulfield MJ</span>, <span class="author">Clarke R</span>, <span class="author">Connell JM</span>, <span class="author">Cruickshanks KJ</span>, <span class="author">Davidson KW</span>, <span class="author">Day IN</span>, <span class="author">de Bakker PI</span>, <span class="author">Doevendans PA</span>, <span class="author">Dominiczak AF</span>, <span class="author">Hall AS</span>, <span class="author">Hartman CA</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hillege HL</span>, <span class="author">Hofker MH</span>, <span class="author">Humphries SE</span>, <span class="author">Jarvik GP</span>, <span class="author">Johnson JA</span>, <span class="author">Kaess BM</span>, <span class="author">Kathiresan S</span>, <span class="author">Koenig W</span>, <span class="author">Lawlor DA</span>, <span class="author">M&auml;rz W</span>, <span class="author">Melander O</span>, <span class="author">Mitchell BD</span>, <span class="author">Montgomery GW</span>, <span class="author">Munroe PB</span>, <span class="author">Murray SS</span>, <span class="author">Newhouse SJ</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Poulter N</span>, <span class="author">Psaty B</span>, <span class="author">Redline S</span>, <span class="author">Rich SS</span>, <span class="author">Rotter JI</span>, <span class="author">Schunkert H</span>, <span class="author">Sever P</span>, <span class="author">Shuldiner AR</span>, <span class="author">Silverstein RL</span>, <span class="author">Stanton A</span>, <span class="author">Thorand B</span>, <span class="author">Trip MD</span>, <span class="author">Tsai MY</span>, <span class="author">van der Harst P</span>, <span class="author">van der Schoot E</span>, <span class="author">van der Schouw YT</span>, <span class="author">Verschuren WM</span>, <span class="author">Watkins H</span>, <span class="author">Wilde AA</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Whitfield JB</span>, <span class="author">Hovingh GK</span>, <span class="author">Ballantyne CM</span>, <span class="author">Wijmenga C</span>, <span class="author">Reilly MP</span>, <span class="author">Martin NG</span>, <span class="author">Wilson JG</span>, <span class="author">Rader DJ</span>, <span class="author">Samani NJ</span>, <span class="author">Reiner AP</span>, <span class="author">Hegele RA</span>, <span class="author">Kastelein JJ</span>, <span class="author">Hingorani AD</span>, <span class="author">Talmud PJ</span>, <span class="author">Hakonarson H</span>, <span class="author">Elbers CC</span>, <span class="author">Keating BJ</span> and <span class="author">Drenos F</span></p>
<p class="affiliation">Department of Cardiology, Division of Heart and Lungs, University Medical Center Utrecht, 3508 GA Utrecht, The Netherlands; Julius Center for Health Sciences and Primary Care, University Medical Center Utrecht, 3508 GA Utrecht, The Netherlands; Department of Medical Genetics, Biomedical Genetics, University Medical Center Utrecht, 3584 CX Utrecht, The Netherlands.</p>
<p class="abstract">Genome-wide association studies (GWASs) have identified many SNPs underlying variations in plasma-lipid levels. We explore whether additional loci associated with plasma-lipid phenotypes, such as high-density lipoprotein cholesterol (HDL-C), low-density lipoprotein cholesterol (LDL-C), total cholesterol (TC), and triglycerides (TGs), can be identified by a dense gene-centric approach. Our meta-analysis of 32 studies in 66,240 individuals of European ancestry was based on the custom âˆ¼50,000 SNP genotyping array (the ITMAT-Broad-CARe array) covering âˆ¼2,000 candidate genes. SNP-lipid associations were replicated either in a cohort comprising an additional 24,736 samples or within the Global Lipid Genetic Consortium. We identified four, six, ten, and four unreported SNPs in established lipid genes for HDL-C, LDL-C, TC, and TGs, respectively. We also identified several lipid-related SNPs in previously unreported genes: DGAT2, HCAR2, GPIHBP1, PPARG, and FTO for HDL-C; SOCS3, APOH, SPTY2D1, BRCA2, and VLDLR for LDL-C; SOCS3, UGT1A1, BRCA2, UBE3B, FCGR2A, CHUK, and INSIG2 for TC; and SERPINF2, C4B, GCK, GATA4, INSR, and LPAL2 for TGs. The proportion of explained phenotypic variance in the subset of studies providing individual-level data was 9.9% for HDL-C, 9.5% for LDL-C, 10.3% for TC, and 8.0% for TGs. This large meta-analysis of lipid phenotypes with the use of a dense gene-centric approach identified multiple SNPs not previously described in established lipid genes and several previously unknown loci. The explained phenotypic variance from this approach was comparable to that from a meta-analysis of GWAS data, suggesting that a focused genotyping approach can further increase the understanding of heritability of plasma lipids.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23063622" rel="external">23063622</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.08.032" rel="external">10.1016/j.ajhg.2012.08.032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of within-host Plasmodium falciparum diversity using next-generation sequence data.</h4>
<p class="authors"><span class="author">Auburn S</span>, <span class="author">Campino S</span>, <span class="author">Miotto O</span>, <span class="author">Djimde AA</span>, <span class="author">Zongo I</span>, <span class="author">Manske M</span>, <span class="author">Maslen G</span>, <span class="author">Mangano V</span>, <span class="author">Alcock D</span>, <span class="author">MacInnis B</span>, <span class="author">Rockett KA</span>, <span class="author">Clark TG</span>, <span class="author">Doumbo OK</span>, <span class="author">Ou&eacute;draogo JB</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom. <a href="mailto:%73%61%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#97;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Our understanding of the composition of multi-clonal malarial infections and the epidemiological factors which shape their diversity remain poorly understood. Traditionally within-host diversity has been defined in terms of the multiplicity of infection (MOI) derived by PCR-based genotyping. Massively parallel, single molecule sequencing technologies now enable individual read counts to be derived on genome-wide datasets facilitating the development of new statistical approaches to describe within-host diversity. In this class of measures the F(WS) metric characterizes within-host diversity and its relationship to population level diversity. Utilizing P. falciparum field isolates from patients in West Africa we here explore the relationship between the traditional MOI and F(WS) approaches. F(WS) statistics were derived from read count data at 86,158 SNPs in 64 samples sequenced on the Illumina GA platform. MOI estimates were derived by PCR at the msp-1 and -2 loci. Significant correlations were observed between the two measures, particularly with the msp-1 locus (P&#x200A;=&#x200A;5.92&times;10(-5)). The F(WS) metric should be more robust than the PCR-based approach owing to reduced sensitivity to potential locus-specific artifacts. Furthermore the F(WS) metric captures information on a range of parameters which influence out-crossing risk including the number of clones (MOI), their relative proportions and genetic divergence. This approach should provide novel insights into the factors which correlate with, and shape within-host diversity.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute: <span class="grant">55005502</span>; Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">089275</span>, <span class="grant">090532</span>, <span class="grant">090770</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e32891</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22393456">22393456</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3290604">3290604</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0032891">10.1371/journal.pone.0032891</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mouse large-scale phenotyping initiatives: overview of the European Mouse Disease Clinic (EUMODIC) and of the Wellcome Trust Sanger Institute Mouse Genetics Project.</h4>
<p class="authors"><span class="author">Ayadi A</span>, <span class="author">Birling MC</span>, <span class="author">Bottomley J</span>, <span class="author">Bussell J</span>, <span class="author">Fuchs H</span>, <span class="author">Fray M</span>, <span class="author">Gailus-Durner V</span>, <span class="author">Greenaway S</span>, <span class="author">Houghton R</span>, <span class="author">Karp N</span>, <span class="author">Leblanc S</span>, <span class="author">Lengger C</span>, <span class="author">Maier H</span>, <span class="author">Mallon AM</span>, <span class="author">Marschall S</span>, <span class="author">Melvin D</span>, <span class="author">Morgan H</span>, <span class="author">Pavlovic G</span>, <span class="author">Ryder E</span>, <span class="author">Skarnes WC</span>, <span class="author">Selloum M</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Sorg T</span>, <span class="author">Teboul L</span>, <span class="author">Vasseur L</span>, <span class="author">Walling A</span>, <span class="author">Weaver T</span>, <span class="author">Wells S</span>, <span class="author">White JK</span>, <span class="author">Bradley A</span>, <span class="author">Adams DJ</span>, <span class="author">Steel KP</span>, <span class="author">Hrab&#x11B; de Angelis M</span>, <span class="author">Brown SD</span> and <span class="author">Herault Y</span></p>
<p class="affiliation">Institut Clinique de la Souris, PHENOMIN, IGBMC/ICS-MCI, CNRS, INSERM, Universit&eacute; de Strasbourg, UMR7104, UMR964, 1 rue Laurent Fries, 67404 Illkirch, France.</p><p class="abstract">Two large-scale phenotyping efforts, the European Mouse Disease Clinic (EUMODIC) and the Wellcome Trust Sanger Institute Mouse Genetics Project (SANGER-MGP), started during the late 2000s with the aim to deliver a comprehensive assessment of phenotypes or to screen for robust indicators of diseases in mouse mutants. They both took advantage of available mouse mutant lines but predominantly of the embryonic stem (ES) cells resources derived from the European Conditional Mouse Mutagenesis programme (EUCOMM) and the Knockout Mouse Project (KOMP) to produce and study 799 mouse models that were systematically analysed with a comprehensive set of physiological and behavioural paradigms. They captured more than 400 variables and an additional panel of metadata describing the conditions of the tests. All the data are now available through EuroPhenome database (www.europhenome.org) and the WTSI mouse portal (http://www.sanger.ac.uk/mouseportal/), and the corresponding mouse lines are available through the European Mouse Mutant Archive (EMMA), the International Knockout Mouse Consortium (IKMC), or the Knockout Mouse Project (KOMP) Repository. Overall conclusions from both studies converged, with at least one phenotype scored in at least 80% of the mutant lines. In addition, 57% of the lines were viable, 13% subviable, 30% embryonic lethal, and 7% displayed fertility impairments. These efforts provide an important underpinning for a future global programme that will undertake the complete functional annotation of the mammalian genome in the mouse model.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>, <span class="grant">MC_U142684171</span>, <span class="grant">MC_U142684172</span>, <span class="grant">MC_U142684175</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span>;<span class="volume">23</span>;<span class="number">9-10</span>;<span class="pagerange">600-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22961258">22961258</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3463797">3463797</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9418-y">10.1007/s00335-012-9418-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ubiquitous Hepatocystis infections, but no evidence of Plasmodium falciparum-like malaria parasites in wild greater spot-nosed monkeys (Cercopithecus nictitans).</h4>
<p class="authors"><span class="author">Ayouba A</span>, <span class="author">Mouacha F</span>, <span class="author">Learn GH</span>, <span class="author">Mpoudi-Ngole E</span>, <span class="author">Rayner JC</span>, <span class="author">Sharp PM</span>, <span class="author">Hahn BH</span>, <span class="author">Delaporte E</span> and <span class="author">Peeters M</span></p>
<p class="affiliation">Institut de Recherche pour le D&eacute;veloppement, University of Montpellier, 34394 Montpellier, France.</p><p class="abstract">Western gorillas (Gorilla gorilla) have been identified as the natural reservoir of the parasites that were the immediate precursor of Plasmodium falciparum infecting humans. Recently, a P. falciparum-like sequence was reported in a sample from a captive greater spot-nosed monkey (Cercopithecus nictitans), and was taken to indicate that this species may also be a natural reservoir for P. falciparum-related parasites. To test this hypothesis we screened blood samples from 292 wild C. nictitans monkeys that had been hunted for bushmeat in Cameroon. We detected Hepatocystis spp. in 49% of the samples, as well as one sequence from a clade of Plasmodium spp. previously found in birds, lizards and bats. However, none of the 292 wild C. nictitans harbored P. falciparum-like parasites.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI91595</span>, <span class="grant">R01 AI50529</span>; Wellcome Trust: <span class="grant">090851</span></p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2012</span>;<span class="volume">42</span>;<span class="number">8</span>;<span class="pagerange">709-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22691606">22691606</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijpara.2012.05.004">10.1016/j.ijpara.2012.05.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare and Low Frequency Variant Stratification in the UK Population: Description and Impact on Association Tests.</h4>
<p class="authors"><span class="author">Babron MC</span>, <span class="author">de Tayrac M</span>, <span class="author">Rutledge DN</span>, <span class="author">Zeggini E</span> and <span class="author">G&eacute;nin E</span></p>
<p class="affiliation">Inserm UMRS-946, Genetic variability and human diseases, Paris, France ; Institut Universitaire d'Hématologie, Univ Paris Diderot, Paris, France.</p>
<p class="abstract">Although variations in allele frequencies at common SNPs have been extensively studied in different populations, little is known about the stratification of rare variants and its impact on association tests. In this paper, we used Affymetrix 500K genotype data from the WTCCC to investigate if variants in three different frequency categories (below 1%, between 1 and 5%, above 5%) show different stratification patterns in the UK population. We found that these patterns are indeed different. The top principal component extracted from the rare variant category shows poor correlations with any principal component or combination of principal components from the low frequency or common variant categories. These results could suggest that a suitable solution to avoid false positive association due to population stratification would involve adjusting for the respective PCs when testing for variants in different allele frequency categories. However, we found this was not the case both on type 2 diabetes data and on simulated data. Indeed, adjusting rare variant association tests on PCs derived from rare variants does no better to correct for population stratification than adjusting on PCs derived from more common variants. Mixed models perform slightly better for low frequency variants than PC based adjustments but less well for the rarest variants. These results call for the need of new methodological developments specifically devoted to address rare variant stratification issues in association tests.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">e46519</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23071581" rel="external">23071581</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0046519" rel="external">10.1371/journal.pone.0046519</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A dominantly acting murine allele of Mcm4 causes chromosomal abnormalities and promotes tumorigenesis.</h4>
<p class="authors"><span class="author">Bagley BN</span>, <span class="author">Keane TM</span>, <span class="author">Maklakova VI</span>, <span class="author">Marshall JG</span>, <span class="author">Lester RA</span>, <span class="author">Cancel MM</span>, <span class="author">Paulsen AR</span>, <span class="author">Bendzick LE</span>, <span class="author">Been RA</span>, <span class="author">Kogan SC</span>, <span class="author">Cormier RT</span>, <span class="author">Kendziorski C</span>, <span class="author">Adams DJ</span> and <span class="author">Collier LS</span></p>
<p class="affiliation">School of Pharmacy and UW Carbone Cancer Center, University of Wisconsin Madison, Madison, WI, USA.</p><p class="abstract">Here we report the isolation of a murine model for heritable T cell lymphoblastic leukemia/lymphoma (T-ALL) called Spontaneous dominant leukemia (Sdl). Sdl heterozygous mice develop disease with a short latency and high penetrance, while mice homozygous for the mutation die early during embryonic development. Sdl mice exhibit an increase in the frequency of micronucleated reticulocytes, and T-ALLs from Sdl mice harbor small amplifications and deletions, including activating deletions at the Notch1 locus. Using exome sequencing it was determined that Sdl mice harbor a spontaneously acquired mutation in Mcm4 (Mcm4(D573H)). MCM4 is part of the heterohexameric complex of MCM2-7 that is important for licensing of DNA origins prior to S phase and also serves as the core of the replicative helicase that unwinds DNA at replication forks. Previous studies in murine models have discovered that genetic reductions of MCM complex levels promote tumor formation by causing genomic instability. However, Sdl mice possess normal levels of Mcms, and there is no evidence for loss-of-heterozygosity at the Mcm4 locus in Sdl leukemias. Studies in Saccharomyces cerevisiae indicate that the Sdl mutation produces a biologically inactive helicase. Together, these data support a model in which chromosomal abnormalities in Sdl mice result from the ability of MCM4(D573H) to incorporate into MCM complexes and render them inactive. Our studies indicate that dominantly acting alleles of MCMs can be compatible with viability but have dramatic oncogenic consequences by causing chromosomal abnormalities.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>; NCI NIH HHS: <span class="grant">K01CA122183</span>, <span class="grant">P30 CA014520</span>, <span class="grant">P30CA014520</span>, <span class="grant">R03CA137751</span>; NIGMS NIH HHS: <span class="grant">GM102756</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e1003034</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23133403">23133403</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3486839">3486839</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003034">10.1371/journal.pgen.1003034</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary dynamics of local pandemic H1N1/2009 influenza virus lineages revealed by whole-genome analysis.</h4>
<p class="authors"><span class="author">Baillie GJ</span>, <span class="author">Galiano M</span>, <span class="author">Agapow PM</span>, <span class="author">Myers R</span>, <span class="author">Chiam R</span>, <span class="author">Gall A</span>, <span class="author">Palser AL</span>, <span class="author">Watson SJ</span>, <span class="author">Hedge J</span>, <span class="author">Underwood A</span>, <span class="author">Platt S</span>, <span class="author">McLean E</span>, <span class="author">Pebody RG</span>, <span class="author">Rambaut A</span>, <span class="author">Green J</span>, <span class="author">Daniels R</span>, <span class="author">Pybus OG</span>, <span class="author">Kellam P</span> and <span class="author">Zambon M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, United Kingdom.</p><p class="abstract">Virus gene sequencing and phylogenetics can be used to study the epidemiological dynamics of rapidly evolving viruses. With complete genome data, it becomes possible to identify and trace individual transmission chains of viruses such as influenza virus during the course of an epidemic. Here we sequenced 153 pandemic influenza H1N1/09 virus genomes from United Kingdom isolates from the first (127 isolates) and second (26 isolates) waves of the 2009 pandemic and used their sequences, dates of isolation, and geographical locations to infer the genetic epidemiology of the epidemic in the United Kingdom. We demonstrate that the epidemic in the United Kingdom was composed of many cocirculating lineages, among which at least 13 were exclusively or predominantly United Kingdom clusters. The estimated divergence times of two of the clusters predate the detection of pandemic H1N1/09 virus in the United Kingdom, suggesting that the pandemic H1N1/09 virus was already circulating in the United Kingdom before the first clinical case. Crucially, three clusters contain isolates from the second wave of infections in the United Kingdom, two of which represent chains of transmission that appear to have persisted within the United Kingdom between the first and second waves. This demonstrates that whole-genome analysis can track in fine detail the behavior of individual influenza virus lineages during the course of a single epidemic or pandemic.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U117512723</span>; Wellcome Trust: <span class="grant">095831</span></p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2012</span>;<span class="volume">86</span>;<span class="number">1</span>;<span class="pagerange">11-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22013031">22013031</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3255882">3255882</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.05347-11">10.1128/JVI.05347-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Frequency and patterns of protease gene resistance mutations in HIV-infected patients treated with lopinavir/ritonavir as their first protease inhibitor.</h4>
<p class="authors"><span class="author">Barber TJ</span>, <span class="author">Harrison L</span>, <span class="author">Asboe D</span>, <span class="author">Williams I</span>, <span class="author">Kirk S</span>, <span class="author">Gilson R</span>, <span class="author">Bansi L</span>, <span class="author">Pillay D</span>, <span class="author">Dunn D</span> and <span class="author">UK HIV Drug Resistance Database and UK Collaborative HIV Cohort (UK CHIC) Study Steering Committees</span></p>
<p class="affiliation">Medical Research Council Clinical Trials Unit, St Stephen&#39;s Centre, Chelsea and Westminster Hospital, 125 Kingsway, London, UK. <a href="mailto:%74.%62%61%72%62%65%72%40%6E%68%73.%6E%65%74">&#116;&#46;&#98;&#97;&#114;&#98;&#101;&#114;&#64;&#110;&#104;&#115;&#46;&#110;&#101;&#116;</a></p><p class="abstract"><strong>Background</strong>: Selection of protease mutations on antiretroviral therapy (ART) including a ritonavir-boosted protease inhibitor (PI) has been reported infrequently. Scarce data exist from long-term cohorts on resistance incidence or mutational patterns emerging to different PIs.</p><p class="abstract"><strong>Methods</strong>: We studied UK patients receiving lopinavir/ritonavir as their first PI, either while naive to ART or having previously received non-PI-based ART. Virological failure was defined as viral load &ge; 400 copies/mL after previous suppression &lt;400 copies/mL, or failure to achieve &lt;400 copies/mL during the first 6 months. pol sequences whilst failing lopinavir or within 30 days after stopping were analysed. Major and minor mutations (IAS-USA 2008-after exclusion of polymorphisms) were considered. Predicted susceptibility was determined using the Stanford HIVdb algorithm.</p><p class="abstract"><strong>Results</strong>: Three thousand and fifty-six patients were followed for a median (IQR) of 14 (6-30) months, of whom 811 (27%) experienced virological failure. Of these, resistance test results were available on 291 (36%). One or more protease mutations were detected in 32 (11%) patients; the most frequent were I54V (n = 12), M46I (n = 11), V82A (n = 7) and L76V (n = 3). No association with viral subtype was evident. Many patients retained virus predicted to be susceptible to lopinavir (14, 44%), tipranavir (26, 81%) and darunavir (27, 84%).</p><p class="abstract"><strong>Conclusions</strong>: This study reflects the experience of patients in routine care. Selection of protease gene mutations by lopinavir/ritonavir occurred at a much higher rate than in clinical trials. The mutations observed showed only partial overlap with those previously identified by structural chemistry models, serial cell culture passage and genotype-phenotype analyses. There remained a low degree of predicted cross-resistance to other widely used PIs.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G00001999</span>, <span class="grant">G0600337</span>, <span class="grant">G0900274</span></p>
<p class="publication"><span class="title">The Journal of antimicrobial chemotherapy</span>&nbsp;<span class="year">2012</span>;<span class="volume">67</span>;<span class="number">4</span>;<span class="pagerange">995-1000</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22258921">22258921</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fjac%2Fdkr569">10.1093/jac/dkr569</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From HLA association to function.</h4>
<p class="authors"><span class="author">Barrett JC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%62%61%72%72%65%74%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#97;&#114;&#114;&#101;&#116;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A new study refines the association signals for rheumatoid arthritis susceptibility in the major histocompatibility complex (MHC) region to five amino-acid positions encoded in three HLA genes, all within peptide-binding grooves. By adapting statistical methods from genome-wide association studies (GWAS) and using imputation from a large reference panel, they demonstrate the potential for this approach to identify functional variants in associated regions.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">235-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22366857">22366857</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2207">10.1038/ng.2207</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Semaphorin-7A is an erythrocyte receptor for P. falciparum merozoite-specific TRAP homolog, MTRAP.</h4>
<p class="authors"><span class="author">Bartholdson SJ</span>, <span class="author">Bustamante LY</span>, <span class="author">Crosnier C</span>, <span class="author">Johnson S</span>, <span class="author">Lea S</span>, <span class="author">Rayner JC</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom ; Malaria Programme, Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The motility and invasion of Plasmodium parasites is believed to require a cytoplasmic actin-myosin motor associated with a cell surface ligand belonging to the TRAP (thrombospondin-related anonymous protein) family. Current models of invasion usually invoke the existence of specific receptors for the TRAP-family ligands on the surface of the host cell; however, the identities of these receptors remain largely unknown. Here, we identify the GPI-linked protein Semaphorin-7A (CD108) as an erythrocyte receptor for the P. falciparum merozoite-specific TRAP homolog (MTRAP) by using a systematic screening approach designed to detect extracellular protein interactions. The specificity of the interaction was demonstrated by showing that binding was saturable and by quantifying the equilibrium and kinetic biophysical binding parameters using surface plasmon resonance. We found that two MTRAP monomers interact via their tandem TSR domains with the Sema domains of a Semaphorin-7A homodimer. Known naturally-occurring polymorphisms in Semaphorin-7A did not quantitatively affect MTRAP binding nor did the presence of glycans on the receptor. Attempts to block the interaction during in vitro erythrocyte invasion assays using recombinant proteins and antibodies showed no significant inhibitory effect, suggesting the inaccessibility of the complex to proteinaceous blocking agents. These findings now provide important experimental evidence to support the model that parasite TRAP-family ligands interact with specific host receptors during cellular invasion.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MR/J002283/1</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e1003031</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23166499">23166499</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3499583">3499583</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003031">10.1371/journal.ppat.1003031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deficiency for the ubiquitin ligase UBE3B in a blepharophimosis-ptosis-intellectual-disability syndrome.</h4>
<p class="authors"><span class="author">Basel-Vanagaite L</span>, <span class="author">Dallapiccola B</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Segref A</span>, <span class="author">Thiele H</span>, <span class="author">Edwards A</span>, <span class="author">Arends MJ</span>, <span class="author">Mir&oacute; X</span>, <span class="author">White JK</span>, <span class="author">D&eacute;sir J</span>, <span class="author">Abramowicz M</span>, <span class="author">Dentici ML</span>, <span class="author">Lepri F</span>, <span class="author">Hofmann K</span>, <span class="author">Har-Zahav A</span>, <span class="author">Ryder E</span>, <span class="author">Karp NA</span>, <span class="author">Estabel J</span>, <span class="author">Gerdin AK</span>, <span class="author">Podrini C</span>, <span class="author">Ingham NJ</span>, <span class="author">Altm&uuml;ller J</span>, <span class="author">N&uuml;rnberg G</span>, <span class="author">Frommolt P</span>, <span class="author">Abdelhak S</span>, <span class="author">Pasmanik-Chor M</span>, <span class="author">Konen O</span>, <span class="author">Kelley RI</span>, <span class="author">Shohat M</span>, <span class="author">N&uuml;rnberg P</span>, <span class="author">Flint J</span>, <span class="author">Steel KP</span>, <span class="author">Hoppe T</span>, <span class="author">Kubisch C</span>, <span class="author">Adams DJ</span> and <span class="author">Borck G</span></p>
<p class="affiliation">Raphael Recanati Genetics Institute, Rabin Medical Center, Beilinson Campus, Petah Tikva 49100, Israel. <a href="mailto:%62%61%73%65%6C%40%70%6F%73%74.%74%61%75.%61%63.%69%6C">&#98;&#97;&#115;&#101;&#108;&#64;&#112;&#111;&#115;&#116;&#46;&#116;&#97;&#117;&#46;&#97;&#99;&#46;&#105;&#108;</a></p><p class="abstract">Ubiquitination plays a crucial role in neurodevelopment as exemplified by Angelman syndrome, which is caused by genetic alterations of the ubiquitin ligase-encoding UBE3A gene. Although the function of UBE3A has been widely studied, little is known about its paralog UBE3B. By using exome and capillary sequencing, we here identify biallelic UBE3B mutations in four patients from three unrelated families presenting an autosomal-recessive blepharophimosis-ptosis-intellectual-disability syndrome characterized by developmental delay, growth retardation with a small head circumference, facial dysmorphisms, and low cholesterol levels. UBE3B encodes an uncharacterized E3 ubiquitin ligase. The identified UBE3B variants include one frameshift and two splice-site mutations as well as a missense substitution affecting the highly conserved HECT domain. Disruption of mouse Ube3b leads to reduced viability and recapitulates key aspects of the human disorder, such as reduced weight and brain size and a downregulation of cholesterol synthesis. We establish that the probable Caenorhabditis elegans ortholog of UBE3B, oxi-1, functions in the ubiquitin/proteasome system in vivo and is especially required under oxidative stress conditions. Our data reveal the pleiotropic effects of UBE3B deficiency and reinforce the physiological importance of ubiquitination in neuronal development and function in mammals.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">91</span>;<span class="number">6</span>;<span class="pagerange">998-1010</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23200864">23200864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3516591">3516591</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.10.011">10.1016/j.ajhg.2012.10.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid-throughput skeletal phenotyping of 100 knockout mice identifies 9 new genes that determine bone strength.</h4>
<p class="authors"><span class="author">Bassett JH</span>, <span class="author">Gogakos A</span>, <span class="author">White JK</span>, <span class="author">Evans H</span>, <span class="author">Jacques RM</span>, <span class="author">van der Spek AH</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Ryder E</span>, <span class="author">Sunter D</span>, <span class="author">Boyde A</span>, <span class="author">Campbell MJ</span>, <span class="author">Croucher PI</span> and <span class="author">Williams GR</span></p>
<p class="affiliation">Molecular Endocrinology Group, Department of Medicine, Imperial College London, London, United Kingdom.</p><p class="abstract">Osteoporosis is a common polygenic disease and global healthcare priority but its genetic basis remains largely unknown. We report a high-throughput multi-parameter phenotype screen to identify functionally significant skeletal phenotypes in mice generated by the Wellcome Trust Sanger Institute Mouse Genetics Project and discover novel genes that may be involved in the pathogenesis of osteoporosis. The integrated use of primary phenotype data with quantitative x-ray microradiography, micro-computed tomography, statistical approaches and biomechanical testing in 100 unselected knockout mouse strains identified nine new genetic determinants of bone mass and strength. These nine new genes include five whose deletion results in low bone mass and four whose deletion results in high bone mass. None of the nine genes have been implicated previously in skeletal disorders and detailed analysis of the biomechanical consequences of their deletion revealed a novel functional classification of bone structure and strength. The organ-specific and disease-focused strategy described in this study can be applied to any biological system or tractable polygenic disease, thus providing a general basis to define gene function in a system-specific manner. Application of the approach to diseases affecting other physiological systems will help to realize the full potential of the International Mouse Phenotyping Consortium.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">h UK 18292</span>; Medical Research Council: <span class="grant">G0800261</span>; Wellcome Trust: <span class="grant">094134</span>, <span class="grant">77157/Z/05/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">e1002858</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22876197">22876197</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3410859">3410859</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002858">10.1371/journal.pgen.1002858</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenome-wide scans identify differentially methylated regions for age and age-related phenotypes in a healthy ageing population.</h4>
<p class="authors"><span class="author">Bell JT</span>, <span class="author">Tsai PC</span>, <span class="author">Yang TP</span>, <span class="author">Pidsley R</span>, <span class="author">Nisbet J</span>, <span class="author">Glass D</span>, <span class="author">Mangino M</span>, <span class="author">Zhai G</span>, <span class="author">Zhang F</span>, <span class="author">Valdes A</span>, <span class="author">Shin SY</span>, <span class="author">Dempster EL</span>, <span class="author">Murray RM</span>, <span class="author">Grundberg E</span>, <span class="author">Hedman AK</span>, <span class="author">Nica A</span>, <span class="author">Small KS</span>, <span class="author">MuTHER Consortium</span>, <span class="author">Dermitzakis ET</span>, <span class="author">McCarthy MI</span>, <span class="author">Mill J</span>, <span class="author">Spector TD</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%6A%6F%72%64%61%6E%61.%62%65%6C%6C%40%6B%63%6C.%61%63.%75%6B">&#106;&#111;&#114;&#100;&#97;&#110;&#97;&#46;&#98;&#101;&#108;&#108;&#64;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Age-related changes in DNA methylation have been implicated in cellular senescence and longevity, yet the causes and functional consequences of these variants remain unclear. To elucidate the role of age-related epigenetic changes in healthy ageing and potential longevity, we tested for association between whole-blood DNA methylation patterns in 172 female twins aged 32 to 80 with age and age-related phenotypes. Twin-based DNA methylation levels at 26,690 CpG-sites showed evidence for mean genome-wide heritability of 18%, which was supported by the identification of 1,537 CpG-sites with methylation QTLs in cis at FDR 5%. We performed genome-wide analyses to discover differentially methylated regions (DMRs) for sixteen age-related phenotypes (ap-DMRs) and chronological age (a-DMRs). Epigenome-wide association scans (EWAS) identified age-related phenotype DMRs (ap-DMRs) associated with LDL (STAT5A), lung function (WT1), and maternal longevity (ARL4A, TBX20). In contrast, EWAS for chronological age identified hundreds of predominantly hyper-methylated age DMRs (490 a-DMRs at FDR 5%), of which only one (TBX20) was also associated with an age-related phenotype. Therefore, the majority of age-related changes in DNA methylation are not associated with phenotypic measures of healthy ageing in later life. We replicated a large proportion of a-DMRs in a sample of 44 younger adult MZ twins aged 20 to 61, suggesting that a-DMRs may initiate at an earlier age. We next explored potential genetic and environmental mechanisms underlying a-DMRs and ap-DMRs. Genome-wide overlap across cis-meQTLs, genotype-phenotype associations, and EWAS ap-DMRs identified CpG-sites that had cis-meQTLs with evidence for genotype-phenotype association, where the CpG-site was also an ap-DMR for the same phenotype. Monozygotic twin methylation difference analyses identified one potential environmentally-mediated ap-DMR associated with total cholesterol and LDL (CSMD1). Our results suggest that in a small set of genes DNA methylation may be a candidate mechanism of mediating not only environmental, but also genetic effects on age-related phenotypes.</p>
<p class="grants">Funded by: European Research Council: <span class="grant">250157</span>; Medical Research Council: <span class="grant">G0900339</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e1002629</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22532803">22532803</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3330116">3330116</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002629">10.1371/journal.pgen.1002629</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A robust clustering algorithm for identifying problematic samples in genome-wide association studies.</h4>
<p class="authors"><span class="author">Bellenguez C</span>, <span class="author">Strange A</span>, <span class="author">Freeman C</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Donnelly P</span> and <span class="author">Spencer CC</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK.</p><p class="abstract"><strong>Summary</strong>: High-throughput genotyping arrays provide an efficient way to survey single nucleotide polymorphisms (SNPs) across the genome in large numbers of individuals. Downstream analysis of the data, for example in genome-wide association studies (GWAS), often involves statistical models of genotype frequencies across individuals. The complexities of the sample collection process and the potential for errors in the experimental assay can lead to biases and artefacts in an individual&#39;s inferred genotypes. Rather than attempting to model these complications, it has become a standard practice to remove individuals whose genome-wide data differ from the sample at large. Here we describe a simple, but robust, statistical algorithm to identify samples with atypical summaries of genome-wide variation. Its use as a semi-automated quality control tool is demonstrated using several summary statistics, selected to identify different potential problems, and it is applied to two different genotyping platforms and sample collections.</p><p class="abstract"><strong>Availability</strong>: The algorithm is written in R and is freely available at www.well.ox.ac.uk/chris-spencer</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%63%68%72%69%73.%73%70%65%6E%63%65%72%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#99;&#104;&#114;&#105;&#115;&#46;&#115;&#112;&#101;&#110;&#99;&#101;&#114;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Supplementary data are available at Bioinformatics online.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">075491/Z/04/B</span>, <span class="grant">084575/Z/08/Z</span>, <span class="grant">090532</span>, <span class="grant">090532/Z/09/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">1</span>;<span class="pagerange">134-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22057162">22057162</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3244763">3244763</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr599">10.1093/bioinformatics/btr599</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genomic approach to bacterial taxonomy: an examination and proposed reclassification of species within the genus Neisseria.</h4>
<p class="authors"><span class="author">Bennett JS</span>, <span class="author">Jolley KA</span>, <span class="author">Earle SG</span>, <span class="author">Corton C</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span> and <span class="author">Maiden MC</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Oxford, UK. <a href="mailto:%6A%75%6C%69%61.%62%65%6E%6E%65%74%74%40%7A%6F%6F.%6F%78.%61%63.%75%6B">&#106;&#117;&#108;&#105;&#97;&#46;&#98;&#101;&#110;&#110;&#101;&#116;&#116;&#64;&#122;&#111;&#111;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In common with other bacterial taxa, members of the genus Neisseria are classified using a range of phenotypic and biochemical approaches, which are not entirely satisfactory in assigning isolates to species groups. Recently, there has been increasing interest in using nucleotide sequences for bacterial typing and taxonomy, but to date, no broadly accepted alternative to conventional methods is available. Here, the taxonomic relationships of 55 representative members of the genus Neisseria have been analysed using whole-genome sequence data. As genetic material belonging to the accessory genome is widely shared among different taxa but not present in all isolates, this analysis indexed nucleotide sequence variation within sets of genes, specifically protein-coding genes that were present and directly comparable in all isolates. Variation in these genes identified seven species groups, which were robust to the choice of genes and phylogenetic clustering methods used. The groupings were largely, but not completely, congruent with current species designations, with some minor changes in nomenclature and the reassignment of a few isolates necessary. In particular, these data showed that isolates classified as Neisseria polysaccharea are polyphyletic and probably include more than one taxonomically distinct organism. The seven groups could be reliably and rapidly generated with sequence variation within the 53 ribosomal protein subunit (rps) genes, further demonstrating that ribosomal multilocus sequence typing (rMLST) is a practicable and powerful means of characterizing bacteria at all levels, from domain to strain.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087622</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">158</span>;<span class="number">Pt 6</span>;<span class="pagerange">1570-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22422752">22422752</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3541776">3541776</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.056077-0">10.1099/mic.0.056077-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of Mycobacterium africanum West African 2 reveals a lineage-specific locus and genome erosion common to the M. tuberculosis complex.</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Comas I</span>, <span class="author">Bryant JM</span>, <span class="author">Walker D</span>, <span class="author">Smith NH</span>, <span class="author">Harris SR</span>, <span class="author">Thurston S</span>, <span class="author">Gagneux S</span>, <span class="author">Wood J</span>, <span class="author">Antonio M</span>, <span class="author">Quail MA</span>, <span class="author">Gehre F</span>, <span class="author">Adegbola RA</span>, <span class="author">Parkhill J</span> and <span class="author">de Jong BC</span></p>
<p class="affiliation">Wellcome Trust Genome Campus, Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract"><strong>Background</strong>: M. africanum West African 2 constitutes an ancient lineage of the M. tuberculosis complex that commonly causes human tuberculosis in West Africa and has an attenuated phenotype relative to M. tuberculosis.</p><p class="abstract">In search of candidate genes underlying these differences, the genome of M. africanum West African 2 was sequenced using classical capillary sequencing techniques. Our findings reveal a unique sequence, RD900, that was independently lost during the evolution of two important lineages within the complex: the &quot;modern&quot; M. tuberculosis group and the lineage leading to M. bovis. Closely related to M. bovis and other animal strains within the M. tuberculosis complex, M. africanum West African 2 shares an abundance of pseudogenes with M. bovis but also with M. africanum West African clade 1. Comparison with other strains of the M. tuberculosis complex revealed pseudogenes events in all the known lineages pointing toward ongoing genome erosion likely due to increased genetic drift and relaxed selection linked to serial transmission-bottlenecks and an intracellular lifestyle.</p><p class="abstract">The genomic differences identified between M. africanum West African 2 and the other strains of the Mycobacterium tuberculosis complex may explain its attenuated phenotype, and pave the way for targeted experiments to elucidate the phenotypic characteristic of M. africanum. Moreover, availability of the whole genome data allows for verification of conservation of targets used for the next generation of diagnostics and vaccines, in order to ensure similar efficacy in West Africa.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U190071468</span>, <span class="grant">MC_U190074190</span>, <span class="grant">MC_U190081982</span>, <span class="grant">MC_U190081991</span>, <span class="grant">MC_U190085850</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">e1552</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22389744">22389744</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3289620">3289620</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001552">10.1371/journal.pntd.0001552</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An insertional mutagenesis screen identifies genes that cooperate with Mll-AF9 in a murine leukemogenesis model.</h4>
<p class="authors"><span class="author">Bergerson RJ</span>, <span class="author">Collier LS</span>, <span class="author">Sarver AL</span>, <span class="author">Been RA</span>, <span class="author">Lugthart S</span>, <span class="author">Diers MD</span>, <span class="author">Zuber J</span>, <span class="author">Rappaport AR</span>, <span class="author">Nixon MJ</span>, <span class="author">Silverstein KA</span>, <span class="author">Fan D</span>, <span class="author">Lamblin AF</span>, <span class="author">Wolff L</span>, <span class="author">Kersey JH</span>, <span class="author">Delwel R</span>, <span class="author">Lowe SW</span>, <span class="author">O&#39;Sullivan MG</span>, <span class="author">Kogan SC</span>, <span class="author">Adams DJ</span> and <span class="author">Largaespada DA</span></p>
<p class="affiliation">Department of Genetics, Cell Biology and Development, Masonic Cancer Center, University of Minnesota Twin Cities, Minneapolis, MN 55455, USA.</p><p class="abstract">Patients with a t(9;11) translocation (MLL-AF9) develop acute myeloid leukemia (AML), and while in mice the expression of this fusion oncogene also results in the development of myeloid leukemia, it is with long latency. To identify mutations that cooperate with Mll-AF9, we infected neonatal wild-type (WT) or Mll-AF9 mice with a murine leukemia virus (MuLV). MuLV-infected Mll-AF9 mice succumbed to disease significantly faster than controls presenting predominantly with myeloid leukemia while infected WT animals developed predominantly lymphoid leukemia. We identified 88 candidate cancer genes near common sites of proviral insertion. Analysis of transcript levels revealed significantly elevated expression of Mn1, and a trend toward increased expression of Bcl11a and Fosb in Mll-AF9 murine leukemia samples with proviral insertions proximal to these genes. Accordingly, FOSB and BCL11A were also overexpressed in human AML harboring MLL gene translocations. FOSB was revealed to be essential for growth in mouse and human myeloid leukemia cells using shRNA lentiviral vectors in vitro. Importantly, MN1 cooperated with Mll-AF9 in leukemogenesis in an in vivo BM viral transduction and transplantation assay. Together, our data identified genes that define transcription factor networks and important genetic pathways acting during progression of leukemia induced by MLL fusion oncogenes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Howard Hughes Medical Institute; NCI NIH HHS: <span class="grant">CA009138</span>, <span class="grant">F32 CA106192</span>, <span class="grant">K01 CA122183</span>, <span class="grant">U01 CA84221</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2012</span>;<span class="volume">119</span>;<span class="number">19</span>;<span class="pagerange">4512-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22427200">22427200</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3362364">3362364</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2010-04-281428">10.1182/blood-2010-04-281428</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic Comparison of the Closely Related Salmonella enterica Serovars Enteritidis and Dublin.</h4>
<p class="authors"><span class="author">Betancor L</span>, <span class="author">Yim L</span>, <span class="author">Mart&iacute;nez A</span>, <span class="author">Fookes M</span>, <span class="author">Sasias S</span>, <span class="author">Schelotto F</span>, <span class="author">Thomson N</span>, <span class="author">Maskell D</span> and <span class="author">Chabalgoity JA</span></p>
<p class="affiliation">Departamento de Desarrollo Biotecnol&oacute;gico, Instituto de Higiene, Facultad de Medicina, Universidad de la Rep&uacute;blica, Av. A. Navarro 3051, CP 11600, Montevideo, Uruguay.</p><p class="abstract">The Enteritidis and Dublin serovars of Salmonella enterica are closely related, yet they differ significantly in pathogenicity and epidemiology. S. Enteritidis is a broad host range serovar that commonly causes gastroenteritis and infrequently causes invasive disease in humans. S. Dublin mainly colonizes cattle but upon infecting humans often results in invasive disease.To gain a broader view of the extent of these differences we conducted microarray-based comparative genomics between several field isolates from each serovar. Genome degradation has been correlated with host adaptation in Salmonella, thus we also compared at whole genome scale the available genomic sequences of them to evaluate pseudogene composition within each serovar.Microarray analysis revealed 3771 CDS shared by both serovars while 33 were only present in Enteritidis and 87 were exclusive to Dublin. Pseudogene evaluation showed 177 inactive CDS in S. Dublin which correspond to active genes in S. Enteritidis, nine of which are also inactive in the host adapted S. Gallinarum and S. Choleraesuis serovars. Sequencing of these 9 CDS in several S. Dublin clinical isolates revealed that they are pseudogenes in all of them, indicating that this feature is not peculiar to the sequenced strain. Among these CDS, shdA (Peyer&acute;s patch colonization factor) and mglA (galactoside transport ATP binding protein), appear also to be inactive in the human adapted S. Typhi and S. Paratyphi A, suggesting that functionality of these genes may be relevant for the capacity of certain Salmonella serovars to infect a broad range of hosts.</p>
<p class="publication"><span class="title">The open microbiology journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="pagerange">5-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22371816" rel="external">22371816</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3282883" rel="external">3282883</a>; DOI: <a href="http://dx.doi.org/10.2174%2F1874285801206010005" rel="external">10.2174/1874285801206010005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pancreatic cancer genomes reveal aberrations in axon guidance pathway genes.</h4>
<p class="authors"><span class="author">Biankin AV</span>, <span class="author">Waddell N</span>, <span class="author">Kassahn KS</span>, <span class="author">Gingras MC</span>, <span class="author">Muthuswamy LB</span>, <span class="author">Johns AL</span>, <span class="author">Miller DK</span>, <span class="author">Wilson PJ</span>, <span class="author">Patch AM</span>, <span class="author">Wu J</span>, <span class="author">Chang DK</span>, <span class="author">Cowley MJ</span>, <span class="author">Gardiner BB</span>, <span class="author">Song S</span>, <span class="author">Harliwong I</span>, <span class="author">Idrisoglu S</span>, <span class="author">Nourse C</span>, <span class="author">Nourbakhsh E</span>, <span class="author">Manning S</span>, <span class="author">Wani S</span>, <span class="author">Gongora M</span>, <span class="author">Pajic M</span>, <span class="author">Scarlett CJ</span>, <span class="author">Gill AJ</span>, <span class="author">Pinho AV</span>, <span class="author">Rooman I</span>, <span class="author">Anderson M</span>, <span class="author">Holmes O</span>, <span class="author">Leonard C</span>, <span class="author">Taylor D</span>, <span class="author">Wood S</span>, <span class="author">Xu Q</span>, <span class="author">Nones K</span>, <span class="author">Fink JL</span>, <span class="author">Christ A</span>, <span class="author">Bruxner T</span>, <span class="author">Cloonan N</span>, <span class="author">Kolle G</span>, <span class="author">Newell F</span>, <span class="author">Pinese M</span>, <span class="author">Mead RS</span>, <span class="author">Humphris JL</span>, <span class="author">Kaplan W</span>, <span class="author">Jones MD</span>, <span class="author">Colvin EK</span>, <span class="author">Nagrial AM</span>, <span class="author">Humphrey ES</span>, <span class="author">Chou A</span>, <span class="author">Chin VT</span>, <span class="author">Chantrill LA</span>, <span class="author">Mawson A</span>, <span class="author">Samra JS</span>, <span class="author">Kench JG</span>, <span class="author">Lovell JA</span>, <span class="author">Daly RJ</span>, <span class="author">Merrett ND</span>, <span class="author">Toon C</span>, <span class="author">Epari K</span>, <span class="author">Nguyen NQ</span>, <span class="author">Barbour A</span>, <span class="author">Zeps N</span>, <span class="author">Australian Pancreatic Cancer Genome Initiative</span>, <span class="author">Kakkar N</span>, <span class="author">Zhao F</span>, <span class="author">Wu YQ</span>, <span class="author">Wang M</span>, <span class="author">Muzny DM</span>, <span class="author">Fisher WE</span>, <span class="author">Brunicardi FC</span>, <span class="author">Hodges SE</span>, <span class="author">Reid JG</span>, <span class="author">Drummond J</span>, <span class="author">Chang K</span>, <span class="author">Han Y</span>, <span class="author">Lewis LR</span>, <span class="author">Dinh H</span>, <span class="author">Buhay CJ</span>, <span class="author">Beck T</span>, <span class="author">Timms L</span>, <span class="author">Sam M</span>, <span class="author">Begley K</span>, <span class="author">Brown A</span>, <span class="author">Pai D</span>, <span class="author">Panchal A</span>, <span class="author">Buchner N</span>, <span class="author">De Borja R</span>, <span class="author">Denroche RE</span>, <span class="author">Yung CK</span>, <span class="author">Serra S</span>, <span class="author">Onetto N</span>, <span class="author">Mukhopadhyay D</span>, <span class="author">Tsao MS</span>, <span class="author">Shaw PA</span>, <span class="author">Petersen GM</span>, <span class="author">Gallinger S</span>, <span class="author">Hruban RH</span>, <span class="author">Maitra A</span>, <span class="author">Iacobuzio-Donahue CA</span>, <span class="author">Schulick RD</span>, <span class="author">Wolfgang CL</span>, <span class="author">Morgan RA</span>, <span class="author">Lawlor RT</span>, <span class="author">Capelli P</span>, <span class="author">Corbo V</span>, <span class="author">Scardoni M</span>, <span class="author">Tortora G</span>, <span class="author">Tempero MA</span>, <span class="author">Mann KM</span>, <span class="author">Jenkins NA</span>, <span class="author">Perez-Mancera PA</span>, <span class="author">Adams DJ</span>, <span class="author">Largaespada DA</span>, <span class="author">Wessels LF</span>, <span class="author">Rust AG</span>, <span class="author">Stein LD</span>, <span class="author">Tuveson DA</span>, <span class="author">Copeland NG</span>, <span class="author">Musgrove EA</span>, <span class="author">Scarpa A</span>, <span class="author">Eshleman JR</span>, <span class="author">Hudson TJ</span>, <span class="author">Sutherland RL</span>, <span class="author">Wheeler DA</span>, <span class="author">Pearson JV</span>, <span class="author">McPherson JD</span>, <span class="author">Gibbs RA</span> and <span class="author">Grimmond SM</span></p>
<p class="affiliation">The Kinghorn Cancer Centre, 370 Victoria Street, Darlinghurst, Sydney, New South Wales 2010, Australia.</p><p class="abstract">Pancreatic cancer is a highly lethal malignancy with few effective therapies. We performed exome sequencing and copy number analysis to define genomic aberrations in a prospectively accrued clinical cohort (n = 142) of early (stage I and II) sporadic pancreatic ductal adenocarcinoma. Detailed analysis of 99 informative tumours identified substantial heterogeneity with 2,016 non-silent mutations and 1,628 copy-number variations. We define 16 significantly mutated genes, reaffirming known mutations (KRAS, TP53, CDKN2A, SMAD4, MLL3, TGFBR2, ARID1A and SF3B1), and uncover novel mutated genes including additional genes involved in chromatin modification (EPC1 and ARID2), DNA damage repair (ATM) and other mechanisms (ZIM2, MAP2K4, NALCN, SLC16A4 and MAGEA6). Integrative analysis with in vitro functional data and animal models provided supportive evidence for potential roles for these genetic aberrations in carcinogenesis. Pathway-based analysis of recurrently mutated genes recapitulated clustering in core signalling pathways in pancreatic ductal adenocarcinoma, and identified new mutated genes in each pathway. We also identified frequent and diverse somatic aberrations in genes described traditionally as embryonic regulators of axon guidance, particularly SLIT/ROBO signalling, which was also evident in murine Sleeping Beauty transposon-mediated somatic mutagenesis models of pancreatic cancer, providing further supportive evidence for the potential involvement of axon guidance genes in pancreatic carcinogenesis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; NCI NIH HHS: <span class="grant">2P50CA101955</span>, <span class="grant">P01CA134292</span>, <span class="grant">P50 CA062924</span>, <span class="grant">P50 CA101955</span>, <span class="grant">P50 CA102701</span>, <span class="grant">P50CA062924</span>, <span class="grant">R01 CA97075</span>; NHGRI NIH HHS: <span class="grant">U54 HG003273</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">491</span>;<span class="number">7424</span>;<span class="pagerange">399-405</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23103869">23103869</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3530898">3530898</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11547">10.1038/nature11547</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Impact of common variation in bone-related genes on type 2 diabetes and related traits.</h4>
<p class="authors"><span class="author">Billings LK</span>, <span class="author">Hsu YH</span>, <span class="author">Ackerman RJ</span>, <span class="author">Dupuis J</span>, <span class="author">Voight BF</span>, <span class="author">Rasmussen-Torvik LJ</span>, <span class="author">Hercberg S</span>, <span class="author">Lathrop M</span>, <span class="author">Barnes D</span>, <span class="author">Langenberg C</span>, <span class="author">Hui J</span>, <span class="author">Fu M</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Lecoeur C</span>, <span class="author">An P</span>, <span class="author">Magnusson PK</span>, <span class="author">Surakka I</span>, <span class="author">Ripatti S</span>, <span class="author">Christiansen L</span>, <span class="author">Dalg&aring;rd C</span>, <span class="author">Folkersen L</span>, <span class="author">Grundberg E</span>, <span class="author">MAGIC Investigators</span>, <span class="author">DIAGRAM + Consortium</span>, <span class="author">MuTHER Consortium</span>, <span class="author">ASCOT Investigators</span>, <span class="author">GEFOS Consortium</span>, <span class="author">Eriksson P</span>, <span class="author">Kaprio J</span>, <span class="author">Ohm Kyvik K</span>, <span class="author">Pedersen NL</span>, <span class="author">Borecki IB</span>, <span class="author">Province MA</span>, <span class="author">Balkau B</span>, <span class="author">Froguel P</span>, <span class="author">Shuldiner AR</span>, <span class="author">Palmer LJ</span>, <span class="author">Wareham N</span>, <span class="author">Meneton P</span>, <span class="author">Johnson T</span>, <span class="author">Pankow JS</span>, <span class="author">Karasik D</span>, <span class="author">Meigs JB</span>, <span class="author">Kiel DP</span> and <span class="author">Florez JC</span></p>
<p class="affiliation">Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA.</p><p class="abstract">Exploring genetic pleiotropy can provide clues to a mechanism underlying the observed epidemiological association between type 2 diabetes and heightened fracture risk. We examined genetic variants associated with bone mineral density (BMD) for association with type 2 diabetes and glycemic traits in large well-phenotyped and -genotyped consortia. We undertook follow-up analysis in &sim;19,000 individuals and assessed gene expression. We queried single nucleotide polymorphisms (SNPs) associated with BMD at levels of genome-wide significance, variants in linkage disequilibrium (r(2) &gt; 0.5), and BMD candidate genes. SNP rs6867040, at the ITGA1 locus, was associated with a 0.0166 mmol/L (0.004) increase in fasting glucose per C allele in the combined analysis. Genetic variants in the ITGA1 locus were associated with its expression in the liver but not in adipose tissue. ITGA1 variants appeared among the top loci associated with type 2 diabetes, fasting insulin, &beta;-cell function by homeostasis model assessment, and 2-h post-oral glucose tolerance test glucose and insulin levels. ITGA1 has demonstrated genetic pleiotropy in prior studies, and its suggested role in liver fibrosis, insulin secretion, and bone healing lends credence to its contribution to both osteoporosis and type 2 diabetes. These findings further underscore the link between skeletal and glucose metabolism and highlight a locus to direct future investigations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900339</span>, <span class="grant">MC_U106179471</span>; NCATS NIH HHS: <span class="grant">UL1 TR000150</span>; NCRR NIH HHS: <span class="grant">1-S10-RR-163736-01A1</span>, <span class="grant">M01-RR-16500</span>, <span class="grant">UL1-RR-025005</span>; NHGRI NIH HHS: <span class="grant">U01-HG-004402</span>; NHLBI NIH HHS: <span class="grant">5-R01-HL-08770003</span>, <span class="grant">5-R01-HL-08821502</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01-HL-086694</span>, <span class="grant">R01-HL-087641</span>, <span class="grant">R01-HL-59367</span>, <span class="grant">U01-HL-72515</span>; NIA NIH HHS: <span class="grant">R01-AG-18728</span>, <span class="grant">R01-AR/AG-41398</span>; NIAMS NIH HHS: <span class="grant">R21-AR-056405</span>; NIDDK NIH HHS: <span class="grant">1-L30-DK-089944-01</span>, <span class="grant">5-R01-DK-06833603</span>, <span class="grant">5-R01-DK-07568102</span>, <span class="grant">K24 DK080140</span>, <span class="grant">K24-DK-080140</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30-DK-072488</span>, <span class="grant">P60-DK-079637</span>, <span class="grant">R01-DK-04261</span>, <span class="grant">R01-DK-078616</span>, <span class="grant">T32-DK-007028-35</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268201100005C</span>, <span class="grant">HHSN268201100006C</span>, <span class="grant">HHSN268201100007C</span>, <span class="grant">HHSN268201100008C</span>, <span class="grant">HHSN268201100009C</span>, <span class="grant">HHSN268201100010C</span>, <span class="grant">HHSN268201100011C</span>, <span class="grant">HHSN268201100012C</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2012</span>;<span class="volume">61</span>;<span class="number">8</span>;<span class="pagerange">2176-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22698912">22698912</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3402303">3402303</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb11-1515">10.2337/db11-1515</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare MTNR1B variants impairing melatonin receptor 1B function contribute to type 2 diabetes.</h4>
<p class="authors"><span class="author">Bonnefond A</span>, <span class="author">Cl&eacute;ment N</span>, <span class="author">Fawcett K</span>, <span class="author">Yengo L</span>, <span class="author">Vaillant E</span>, <span class="author">Guillaume JL</span>, <span class="author">Dechaume A</span>, <span class="author">Payne F</span>, <span class="author">Roussel R</span>, <span class="author">Czernichow S</span>, <span class="author">Hercberg S</span>, <span class="author">Hadjadj S</span>, <span class="author">Balkau B</span>, <span class="author">Marre M</span>, <span class="author">Lantieri O</span>, <span class="author">Langenberg C</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Meta-Analysis of Glucose and Insulin-Related Traits Consortium (MAGIC)</span>, <span class="author">Charpentier G</span>, <span class="author">Vaxillaire M</span>, <span class="author">Rocheleau G</span>, <span class="author">Wareham NJ</span>, <span class="author">Sladek R</span>, <span class="author">McCarthy MI</span>, <span class="author">Dina C</span>, <span class="author">Barroso I</span>, <span class="author">Jockers R</span> and <span class="author">Froguel P</span></p>
<p class="affiliation">Centre National de la Recherche Scientifique Unit&eacute; Mixte de Recherche, Lille Pasteur Institute, France.</p><p class="abstract">Genome-wide association studies have revealed that common noncoding variants in MTNR1B (encoding melatonin receptor 1B, also known as MT(2)) increase type 2 diabetes (T2D) risk(1,2). Although the strongest association signal was highly significant (P &lt; 1 &times; 10(-20)), its contribution to T2D risk was modest (odds ratio (OR) of &sim;1.10-1.15)(1-3). We performed large-scale exon resequencing in 7,632 Europeans, including 2,186 individuals with T2D, and identified 40 nonsynonymous variants, including 36 very rare variants (minor allele frequency (MAF) &lt;0.1%), associated with T2D (OR = 3.31, 95% confidence interval (CI) = 1.78-6.18; P = 1.64 &times; 10(-4)). A four-tiered functional investigation of all 40 mutants revealed that 14 were non-functional and rare (MAF &lt; 1%), and 4 were very rare with complete loss of melatonin binding and signaling capabilities. Among the very rare variants, the partial- or total-loss-of-function variants but not the neutral ones contributed to T2D (OR = 5.67, CI = 2.17-14.82; P = 4.09 &times; 10(-4)). Genotyping the four complete loss-of-function variants in 11,854 additional individuals revealed their association with T2D risk (8,153 individuals with T2D and 10,100 controls; OR = 3.88, CI = 1.49-10.07; P = 5.37 &times; 10(-3)). This study establishes a firm functional link between MTNR1B and T2D risk.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">297-301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22286214">22286214</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3773908">3773908</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1053">10.1038/ng.1053</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis of eating disorder-related symptoms, behaviors, and personality traits.</h4>
<p class="authors"><span class="author">Boraska V</span>, <span class="author">Davis OS</span>, <span class="author">Cherkas LF</span>, <span class="author">Helder SG</span>, <span class="author">Harris J</span>, <span class="author">Krug I</span>, <span class="author">Pei-Chi Liao T</span>, <span class="author">Treasure J</span>, <span class="author">Ntalla I</span>, <span class="author">Karhunen L</span>, <span class="author">Keski-Rahkonen A</span>, <span class="author">Christakopoulou D</span>, <span class="author">Raevuori A</span>, <span class="author">Shin SY</span>, <span class="author">Dedoussis GV</span>, <span class="author">Kaprio J</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Collier DA</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK; Department of Medical Biology, University of Split School of Medicine, Split, Croatia. <a href="mailto:%76%62%6F%72%61%73%6B%61%40%6D%65%66%73%74.%68%72">&#118;&#98;&#111;&#114;&#97;&#115;&#107;&#97;&#64;&#109;&#101;&#102;&#115;&#116;&#46;&#104;&#114;</a>, <a href="mailto:%76%62%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#118;&#98;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Eating disorders (EDs) are common, complex psychiatric disorders thought to be caused by both genetic and environmental factors. They share many symptoms, behaviors, and personality traits, which may have overlapping heritability. The aim of the present study is to perform a genome-wide association scan (GWAS) of six ED phenotypes comprising three symptom traits from the Eating Disorders Inventory 2 [Drive for Thinness (DT), Body Dissatisfaction (BD), and Bulimia], Weight Fluctuation symptom, Breakfast Skipping behavior and Childhood Obsessive-Compulsive Personality Disorder trait (CHIRP). Investigated traits were derived from standardized self-report questionnaires completed by the TwinsUK population-based cohort. We tested 283,744 directly typed SNPs across six phenotypes of interest in the TwinsUK discovery dataset and followed-up signals from various strata using a two-stage replication strategy in two independent cohorts of European ancestry. We meta-analyzed a total of 2,698 individuals for DT, 2,680 for BD, 2,789 (821 cases/1,968 controls) for Bulimia, 1,360 (633 cases/727 controls) for Childhood Obsessive-Compulsive Personality Disorder trait, 2,773 (761 cases/2,012 controls) for Breakfast Skipping, and 2,967 (798 cases/2,169 controls) for Weight Fluctuation symptom. In this GWAS analysis of six ED-related phenotypes, we detected association of eight genetic variants with Pâ€‰<â€‰10(-5) . Genetic variants that showed suggestive evidence of association were previously associated with several psychiatric disorders and ED-related phenotypes. Our study indicates that larger-scale collaborative studies will be needed to achieve the necessary power to detect loci underlying ED-related traits. Â© 2012 Wiley Periodicals, Inc.</p>
<p class="publication"><span class="title">American journal of medical genetics. Part B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric Genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">159B</span>;<span class="number">7</span>;<span class="pagerange">803-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22911880" rel="external">22911880</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.b.32087" rel="external">10.1002/ajmg.b.32087</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study to identify common variants associated with brachial circumference: a meta-analysis of 14 cohorts.</h4>
<p class="authors"><span class="author">Boraska V</span>, <span class="author">Day-Williams A</span>, <span class="author">Franklin CS</span>, <span class="author">Elliott KS</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Tachmazidou I</span>, <span class="author">Albrecht E</span>, <span class="author">Bandinelli S</span>, <span class="author">Beilin LJ</span>, <span class="author">Bochud M</span>, <span class="author">Cadby G</span>, <span class="author">Ernst F</span>, <span class="author">Evans DM</span>, <span class="author">Hayward C</span>, <span class="author">Hicks AA</span>, <span class="author">Huffman J</span>, <span class="author">Huth C</span>, <span class="author">James AL</span>, <span class="author">Klopp N</span>, <span class="author">Kolcic I</span>, <span class="author">Kutalik Z</span>, <span class="author">Lawlor DA</span>, <span class="author">Musk AW</span>, <span class="author">Pehlic M</span>, <span class="author">Pennell CE</span>, <span class="author">Perry JR</span>, <span class="author">Peters A</span>, <span class="author">Polasek O</span>, <span class="author">St Pourcain B</span>, <span class="author">Ring SM</span>, <span class="author">Salvi E</span>, <span class="author">Schipf S</span>, <span class="author">Staessen JA</span>, <span class="author">Teumer A</span>, <span class="author">Timpson N</span>, <span class="author">Vitart V</span>, <span class="author">Warrington NM</span>, <span class="author">Yaghootkar H</span>, <span class="author">Zemunik T</span>, <span class="author">Zgaga L</span>, <span class="author">An P</span>, <span class="author">Anttila V</span>, <span class="author">Borecki IB</span>, <span class="author">Holmen J</span>, <span class="author">Ntalla I</span>, <span class="author">Palotie A</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Wedenoja J</span>, <span class="author">Winsvold BS</span>, <span class="author">Dedoussis GV</span>, <span class="author">Kaprio J</span>, <span class="author">Province MA</span>, <span class="author">Zwart JA</span>, <span class="author">Burnier M</span>, <span class="author">Campbell H</span>, <span class="author">Cusi D</span>, <span class="author">Smith GD</span>, <span class="author">Frayling TM</span>, <span class="author">Gieger C</span>, <span class="author">Palmer LJ</span>, <span class="author">Pramstaller PP</span>, <span class="author">Rudan I</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wichmann HE</span>, <span class="author">Wright AF</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom. <a href="mailto:%76%62%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#118;&#98;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Brachial circumference (BC), also known as upper arm or mid arm circumference, can be used as an indicator of muscle mass and fat tissue, which are distributed differently in men and women. Analysis of anthropometric measures of peripheral fat distribution such as BC could help in understanding the complex pathophysiology behind overweight and obesity. The purpose of this study is to identify genetic variants associated with BC through a large-scale genome-wide association scan (GWAS) meta-analysis. We used fixed-effects meta-analysis to synthesise summary results across 14 GWAS discovery and 4 replication cohorts comprising overall 22,376 individuals (12,031 women and 10,345 men) of European ancestry. Individual analyses were carried out for men, women, and combined across sexes using linear regression and an additive genetic model: adjusted for age and adjusted for age and BMI. We prioritised signals for follow-up in two-stages. We did not detect any signals reaching genome-wide significance. The FTO rs9939609 SNP showed nominal evidence for association (p&lt;0.05) in the age-adjusted strata for men and across both sexes. In this first GWAS meta-analysis for BC to date, we have not identified any genome-wide significant signals and do not observe robust association of previously established obesity loci with BC. Large-scale collaborations will be necessary to achieve higher power to detect loci underlying BC.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">MOP-82893</span>; Medical Research Council: <span class="grant">G0800582</span>, <span class="grant">G9815508</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U127561128</span>; NHLBI NIH HHS: <span class="grant">R01-HL-087700</span>, <span class="grant">R01-HL-088215</span>; NIA NIH HHS: <span class="grant">N01-AG-5-0002</span>, <span class="grant">N1-AG-1-1</span>, <span class="grant">N1-AG-1-2111</span>; NIDDK NIH HHS: <span class="grant">R01-DK-075681</span>, <span class="grant">R01-DK-8925601</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>; Wellcome Trust: <span class="grant">092731</span>, <span class="grant">098051</span>, <span class="grant">WT089062</span>, <span class="grant">WT092731</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">e31369</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22479309">22479309</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3315559">3315559</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0031369">10.1371/journal.pone.0031369</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide meta-analysis of common variant differences between men and women.</h4>
<p class="authors"><span class="author">Boraska V</span>, <span class="author">Jeron&#x10D;i&#x107; A</span>, <span class="author">Colonna V</span>, <span class="author">Southam L</span>, <span class="author">Nyholt DR</span>, <span class="author">Rayner NW</span>, <span class="author">Perry JR</span>, <span class="author">Toniolo D</span>, <span class="author">Albrecht E</span>, <span class="author">Ang W</span>, <span class="author">Bandinelli S</span>, <span class="author">Barbalic M</span>, <span class="author">Barroso I</span>, <span class="author">Beckmann JS</span>, <span class="author">Biffar R</span>, <span class="author">Boomsma D</span>, <span class="author">Campbell H</span>, <span class="author">Corre T</span>, <span class="author">Erdmann J</span>, <span class="author">Esko T</span>, <span class="author">Fischer K</span>, <span class="author">Franceschini N</span>, <span class="author">Frayling TM</span>, <span class="author">Girotto G</span>, <span class="author">Gonzalez JR</span>, <span class="author">Harris TB</span>, <span class="author">Heath AC</span>, <span class="author">Heid IM</span>, <span class="author">Hoffmann W</span>, <span class="author">Hofman A</span>, <span class="author">Horikoshi M</span>, <span class="author">Zhao JH</span>, <span class="author">Jackson AU</span>, <span class="author">Hottenga JJ</span>, <span class="author">Jula A</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Khaw KT</span>, <span class="author">Kiemeney LA</span>, <span class="author">Klopp N</span>, <span class="author">Kutalik Z</span>, <span class="author">Lagou V</span>, <span class="author">Launer LJ</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Lemire M</span>, <span class="author">Lokki ML</span>, <span class="author">Loley C</span>, <span class="author">Luan J</span>, <span class="author">Mangino M</span>, <span class="author">Mateo Leach I</span>, <span class="author">Medland SE</span>, <span class="author">Mihailov E</span>, <span class="author">Montgomery GW</span>, <span class="author">Navis G</span>, <span class="author">Newnham J</span>, <span class="author">Nieminen MS</span>, <span class="author">Palotie A</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Peters A</span>, <span class="author">Pirastu N</span>, <span class="author">Polasek O</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Ripatti S</span>, <span class="author">Ritchie GR</span>, <span class="author">Rivadeneira F</span>, <span class="author">Robino A</span>, <span class="author">Samani NJ</span>, <span class="author">Shin SY</span>, <span class="author">Sinisalo J</span>, <span class="author">Smit JH</span>, <span class="author">Soranzo N</span>, <span class="author">Stolk L</span>, <span class="author">Swinkels DW</span>, <span class="author">Tanaka T</span>, <span class="author">Teumer A</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Traglia M</span>, <span class="author">Tuomilehto J</span>, <span class="author">Valsesia A</span>, <span class="author">van Gilst WH</span>, <span class="author">van Meurs JB</span>, <span class="author">Smith AV</span>, <span class="author">Viikari J</span>, <span class="author">Vink JM</span>, <span class="author">Waeber G</span>, <span class="author">Warrington NM</span>, <span class="author">Widen E</span>, <span class="author">Willemsen G</span>, <span class="author">Wright AF</span>, <span class="author">Zanke BW</span>, <span class="author">Zgaga L</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Boehnke M</span>, <span class="author">d&#39;Adamo AP</span>, <span class="author">de Geus E</span>, <span class="author">Demerath EW</span>, <span class="author">den Heijer M</span>, <span class="author">Eriksson JG</span>, <span class="author">Ferrucci L</span>, <span class="author">Gieger C</span>, <span class="author">Gudnason V</span>, <span class="author">Hayward C</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hudson TJ</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Kogevinas M</span>, <span class="author">Loos RJ</span>, <span class="author">Martin NG</span>, <span class="author">Metspalu A</span>, <span class="author">Pennell CE</span>, <span class="author">Penninx BW</span>, <span class="author">Perola M</span>, <span class="author">Raitakari O</span>, <span class="author">Salomaa V</span>, <span class="author">Schreiber S</span>, <span class="author">Schunkert H</span>, <span class="author">Spector TD</span>, <span class="author">Stumvoll M</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Ulivi S</span>, <span class="author">van der Harst P</span>, <span class="author">Vollenweider P</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wareham NJ</span>, <span class="author">Wichmann HE</span>, <span class="author">Wilson JF</span>, <span class="author">Rudan I</span>, <span class="author">Xue Y</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%76%62%6F%72%61%73%6B%61%40%6D%65%66%73%74.%68%72">&#118;&#98;&#111;&#114;&#97;&#115;&#107;&#97;&#64;&#109;&#101;&#102;&#115;&#116;&#46;&#104;&#114;</a></p><p class="abstract">The male-to-female sex ratio at birth is constant across world populations with an average of 1.06 (106 male to 100 female live births) for populations of European descent. The sex ratio is considered to be affected by numerous biological and environmental factors and to have a heritable component. The aim of this study was to investigate the presence of common allele modest effects at autosomal and chromosome X variants that could explain the observed sex ratio at birth. We conducted a large-scale genome-wide association scan (GWAS) meta-analysis across 51 studies, comprising overall 114 863 individuals (61 094 women and 53 769 men) of European ancestry and 2 623 828 common (minor allele frequency &gt;0.05) single-nucleotide polymorphisms (SNPs). Allele frequencies were compared between men and women for directly-typed and imputed variants within each study. Forward-time simulations for unlinked, neutral, autosomal, common loci were performed under the demographic model for European populations with a fixed sex ratio and a random mating scheme to assess the probability of detecting significant allele frequency differences. We do not detect any genome-wide significant (P &lt; 5 &times; 10(-8)) common SNP differences between men and women in this well-powered meta-analysis. The simulated data provided results entirely consistent with these findings. This large-scale investigation across ~115 000 individuals shows no detectable contribution from common genetic variants to the observed skew in the sex ratio. The absence of sex-specific differences is useful in guiding genetic association study design, for example when using mixed controls for sex-biased traits.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">MOP-82893</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G1000143</span>, <span class="grant">G1001799</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U127561128</span>; NCRR NIH HHS: <span class="grant">RR018787</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">HL65234</span>, <span class="grant">HL67466</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>; NIA NIH HHS: <span class="grant">N.1-AG-1-1</span>, <span class="grant">N.1-AG-1-2111</span>, <span class="grant">N01-AG-1-2100</span>, <span class="grant">N01-AG-5-0002</span>; NIAAA NIH HHS: <span class="grant">AA07535</span>, <span class="grant">AA10248</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">AA14041</span>, <span class="grant">K05 AA017688</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>; NIMH NIH HHS: <span class="grant">MH081802</span>, <span class="grant">MH66206</span>, <span class="grant">R01 MH059160</span>, <span class="grant">U24 MH068457-06</span>; NLM NIH HHS: <span class="grant">LM010098</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268201100005C</span>, <span class="grant">HHSN268201100006C</span>, <span class="grant">HHSN268201100007C</span>, <span class="grant">HHSN268201100008C</span>, <span class="grant">HHSN268201100009C</span>, <span class="grant">HHSN268201100010C</span>, <span class="grant">HHSN268201100011C</span>, <span class="grant">HHSN268201100012C</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">089062/Z/09/Z</span>, <span class="grant">092447/Z/10/Z</span>, <span class="grant">095831</span>, <span class="grant">098051</span>, <span class="grant">89061/Z/09/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">21</span>;<span class="pagerange">4805-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22843499">22843499</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3471397">3471397</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fdds304">10.1093/hmg/dds304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Omi, a recessive mutation on chromosome 10, is a novel allele of Ostm1.</h4>
<p class="authors"><span class="author">Bosman EA</span>, <span class="author">Estabel J</span>, <span class="author">Ismail O</span>, <span class="author">Podrini C</span>, <span class="author">White JK</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK, <a href="mailto:%65%61%62%6F%73%6D%61%6E%40%79%61%68%6F%6F.%63%6F%6D">&#101;&#97;&#98;&#111;&#115;&#109;&#97;&#110;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#99;&#111;&#109;</a>.</p>
<p class="abstract">Large-scale N-ethyl-N-nitrosourea (ENU) mutagenesis has provided many rodent models for human disease. Here we describe the initial characterization and mapping of a recessive mutation that leads to degeneration of the incisors, failure of molars to erupt, a grey coat colour, and mild osteopetrosis. We mapped the omi mutation to chromosome 10 between D10Mit214 and D10Mit194. The Ostm1 gene is a likely candidate gene in this region and the grey-lethal allele, Ostm1 ( gl ), and omi mutations fail to complement each other. We show that om/om mice have reduced levels of Ostm1 protein. To date we have not been able to identify the causative mutation. We propose that omi is a novel hypomorphic mutation affecting Ostm1 expression, potentially in a regulatory element.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23160729" rel="external">23160729</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9438-7" rel="external">10.1007/s00335-012-9438-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association meta-analysis identifies new childhood obesity loci.</h4>
<p class="authors"><span class="author">Bradfield JP</span>, <span class="author">Taal HR</span>, <span class="author">Timpson NJ</span>, <span class="author">Scherag A</span>, <span class="author">Lecoeur C</span>, <span class="author">Warrington NM</span>, <span class="author">Hypponen E</span>, <span class="author">Holst C</span>, <span class="author">Valcarcel B</span>, <span class="author">Thiering E</span>, <span class="author">Salem RM</span>, <span class="author">Schumacher FR</span>, <span class="author">Cousminer DL</span>, <span class="author">Sleiman PM</span>, <span class="author">Zhao J</span>, <span class="author">Berkowitz RI</span>, <span class="author">Vimaleswaran KS</span>, <span class="author">Jarick I</span>, <span class="author">Pennell CE</span>, <span class="author">Evans DM</span>, <span class="author">St Pourcain B</span>, <span class="author">Berry DJ</span>, <span class="author">Mook-Kanamori DO</span>, <span class="author">Hofman A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">van der Valk RJ</span>, <span class="author">de Jongste JC</span>, <span class="author">Postma DS</span>, <span class="author">Boomsma DI</span>, <span class="author">Gauderman WJ</span>, <span class="author">Hassanein MT</span>, <span class="author">Lindgren CM</span>, <span class="author">M&auml;gi R</span>, <span class="author">Boreham CA</span>, <span class="author">Neville CE</span>, <span class="author">Moreno LA</span>, <span class="author">Elliott P</span>, <span class="author">Pouta A</span>, <span class="author">Hartikainen AL</span>, <span class="author">Li M</span>, <span class="author">Raitakari O</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Eriksson JG</span>, <span class="author">Palotie A</span>, <span class="author">Dallongeville J</span>, <span class="author">Das S</span>, <span class="author">Deloukas P</span>, <span class="author">McMahon G</span>, <span class="author">Ring SM</span>, <span class="author">Kemp JP</span>, <span class="author">Buxton JL</span>, <span class="author">Blakemore AI</span>, <span class="author">Bustamante M</span>, <span class="author">Guxens M</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Gillman MW</span>, <span class="author">Kreiner-M&oslash;ller E</span>, <span class="author">Bisgaard H</span>, <span class="author">Gilliland FD</span>, <span class="author">Heinrich J</span>, <span class="author">Wheeler E</span>, <span class="author">Barroso I</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Meirhaeghe A</span>, <span class="author">S&oslash;rensen TI</span>, <span class="author">Power C</span>, <span class="author">Palmer LJ</span>, <span class="author">Hinney A</span>, <span class="author">Widen E</span>, <span class="author">Farooqi IS</span>, <span class="author">McCarthy MI</span>, <span class="author">Froguel P</span>, <span class="author">Meyre D</span>, <span class="author">Hebebrand J</span>, <span class="author">Jarvelin MR</span>, <span class="author">Jaddoe VW</span>, <span class="author">Smith GD</span>, <span class="author">Hakonarson H</span>, <span class="author">Grant SF</span> and <span class="author">Early Growth Genetics Consortium</span></p>
<p class="affiliation">Center for Applied Genomics, Abramson Research Center, The Children&rsquo;s Hospital of Philadelphia, Philadelphia, Pennsylvania, USA.</p><p class="abstract">Multiple genetic variants have been associated with adult obesity and a few with severe obesity in childhood; however, less progress has been made in establishing genetic influences on common early-onset obesity. We performed a North American, Australian and European collaborative meta-analysis of 14 studies consisting of 5,530 cases (&ge;95th percentile of body mass index (BMI)) and 8,318 controls (&lt;50th percentile of BMI) of European ancestry. Taking forward the eight newly discovered signals yielding association with P &lt; 5 &times; 10(-6) in nine independent data sets (2,818 cases and 4,083 controls), we observed two loci that yielded genome-wide significant combined P values near OLFM4 at 13q14 (rs9568856; P = 1.82 &times; 10(-9); odds ratio (OR) = 1.22) and within HOXB5 at 17q21 (rs9299; P = 3.54 &times; 10(-9); OR = 1.14). Both loci continued to show association when two extreme childhood obesity cohorts were included (2,214 cases and 2,674 controls). These two loci also yielded directionally consistent associations in a previous meta-analysis of adult BMI(1).</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/09/023</span>, <span class="grant">PG/09/023/26806</span>, <span class="grant">PG/1996183/9569</span>; Canadian Institutes of Health Research: <span class="grant">MOP-82893</span>; Medical Research Council: <span class="grant">74882</span>, <span class="grant">G0000934</span>, <span class="grant">G0100103</span>, <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">G0601653</span>, <span class="grant">G0800582</span>, <span class="grant">G0801056</span>, <span class="grant">G0900554</span>, <span class="grant">MC_UP_A620_1014</span>; NHLBI NIH HHS: <span class="grant">1RC2HL101543</span>, <span class="grant">1RC2HL101651</span>, <span class="grant">5R01HL061768</span>, <span class="grant">5R01HL076647</span>, <span class="grant">5R01HL087679-02</span>, <span class="grant">5R01HL087680</span>; NICHD NIH HHS: <span class="grant">R01 HD056465</span>, <span class="grant">R01 HD056465-01A1</span>, <span class="grant">R01 HD056465-02</span>, <span class="grant">R01 HD056465-03</span>, <span class="grant">R01 HD056465-04</span>, <span class="grant">R01 HD056465-05</span>, <span class="grant">R24 HD050924</span>; NIDDK NIH HHS: <span class="grant">R01 DK075787</span>, <span class="grant">U01 DK062418</span>; NIEHS NIH HHS: <span class="grant">5P01ES009581</span>, <span class="grant">5P01ES011627</span>, <span class="grant">5P30ES007048</span>, <span class="grant">5R01ES014447</span>, <span class="grant">5R01ES014708</span>, <span class="grant">5R01ES016535</span>, <span class="grant">5R03ES014046</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span>; ORD VA: <span class="grant">RD831861-01</span>; PHS HHS: <span class="grant">R826708-01</span>; Wellcome Trust: <span class="grant">052515/2/97/2</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076467</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">083948</span>, <span class="grant">086596/Z/08/Z</span>, <span class="grant">090532</span>, <span class="grant">092731</span>, <span class="grant">GR069224</span>, <span class="grant">WT088431MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">526-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22484627">22484627</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3370100">3370100</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2247">10.1038/ng.2247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The mammalian gene function resource: the international knockout mouse consortium.</h4>
<p class="authors"><span class="author">Bradley A</span>, <span class="author">Anastassiadis K</span>, <span class="author">Ayadi A</span>, <span class="author">Battey JF</span>, <span class="author">Bell C</span>, <span class="author">Birling MC</span>, <span class="author">Bottomley J</span>, <span class="author">Brown SD</span>, <span class="author">B&uuml;rger A</span>, <span class="author">Bult CJ</span>, <span class="author">Bushell W</span>, <span class="author">Collins FS</span>, <span class="author">Desaintes C</span>, <span class="author">Doe B</span>, <span class="author">Economides A</span>, <span class="author">Eppig JT</span>, <span class="author">Finnell RH</span>, <span class="author">Fletcher C</span>, <span class="author">Fray M</span>, <span class="author">Frendewey D</span>, <span class="author">Friedel RH</span>, <span class="author">Grosveld FG</span>, <span class="author">Hansen J</span>, <span class="author">H&eacute;rault Y</span>, <span class="author">Hicks G</span>, <span class="author">H&ouml;rlein A</span>, <span class="author">Houghton R</span>, <span class="author">Hrab&eacute; de Angelis M</span>, <span class="author">Huylebroeck D</span>, <span class="author">Iyer V</span>, <span class="author">de Jong PJ</span>, <span class="author">Kadin JA</span>, <span class="author">Kaloff C</span>, <span class="author">Kennedy K</span>, <span class="author">Koutsourakis M</span>, <span class="author">Kent Lloyd KC</span>, <span class="author">Marschall S</span>, <span class="author">Mason J</span>, <span class="author">McKerlie C</span>, <span class="author">McLeod MP</span>, <span class="author">von Melchner H</span>, <span class="author">Moore M</span>, <span class="author">Mujica AO</span>, <span class="author">Nagy A</span>, <span class="author">Nefedov M</span>, <span class="author">Nutter LM</span>, <span class="author">Pavlovic G</span>, <span class="author">Peterson JL</span>, <span class="author">Pollock J</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Rancourt DE</span>, <span class="author">Raspa M</span>, <span class="author">Remacle JE</span>, <span class="author">Ringwald M</span>, <span class="author">Rosen B</span>, <span class="author">Rosenthal N</span>, <span class="author">Rossant J</span>, <span class="author">Ruiz Noppinger P</span>, <span class="author">Ryder E</span>, <span class="author">Schick JZ</span>, <span class="author">Schn&uuml;tgen F</span>, <span class="author">Schofield P</span>, <span class="author">Seisenberger C</span>, <span class="author">Selloum M</span>, <span class="author">Simpson EM</span>, <span class="author">Skarnes WC</span>, <span class="author">Smedley D</span>, <span class="author">Stanford WL</span>, <span class="author">Francis Stewart A</span>, <span class="author">Stone K</span>, <span class="author">Swan K</span>, <span class="author">Tadepally H</span>, <span class="author">Teboul L</span>, <span class="author">Tocchini-Valentini GP</span>, <span class="author">Valenzuela D</span>, <span class="author">West AP</span>, <span class="author">Yamamura KI</span>, <span class="author">Yoshinaga Y</span> and <span class="author">Wurst W</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1HH, UK, <a href="mailto:%61%62%72%61%64%6C%65%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#98;&#114;&#97;&#100;&#108;&#101;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">In 2007, the International Knockout Mouse Consortium (IKMC) made the ambitious promise to generate mutations in virtually every protein-coding gene of the mouse genome in a concerted worldwide action. Now, 5 years later, the IKMC members have developed high-throughput gene trapping and, in particular, gene-targeting pipelines and generated more than 17,400 mutant murine embryonic stem (ES) cell clones and more than 1,700 mutant mouse strains, most of them conditional. A common IKMC web portal ( www.knockoutmouse.org ) has been established, allowing easy access to this unparalleled biological resource. The IKMC materials considerably enhance functional gene annotation of the mammalian genome and will have a major impact on future biomedical research.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22968824" rel="external">22968824</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9422-2" rel="external">10.1007/s00335-012-9422-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Image-based characterization of thrombus formation in time-lapse DIC microscopy.</h4>
<p class="authors"><span class="author">Brieu N</span>, <span class="author">Navab N</span>, <span class="author">Serbanovic-Canic J</span>, <span class="author">Ouwehand WH</span>, <span class="author">Stemple DL</span>, <span class="author">Cvejic A</span> and <span class="author">Groher M</span></p>
<p class="affiliation">Computer Aided Medical Procedures, Technische Universit&auml;t M&uuml;nchen (TUM), Garching bei M&uuml;nchen 85748, Germany. <a href="mailto:%62%72%69%65%75%40%69%6E.%74%75%6D.%64%65">&#98;&#114;&#105;&#101;&#117;&#64;&#105;&#110;&#46;&#116;&#117;&#109;&#46;&#100;&#101;</a></p><p class="abstract">The characterization of thrombus formation in time-lapse DIC microscopy is of increased interest for identifying genes which account for atherothrombosis and coronary artery diseases (CADs). In particular, we are interested in large-scale studies on zebrafish, which result in large amount of data, and require automatic processing. In this work, we present an image-based solution for the automatized extraction of parameters quantifying the temporal development of thrombotic plugs. Our system is based on the joint segmentation of thrombotic and aortic regions over time. This task is made difficult by the low contrast and the high dynamic conditions observed in vivo DIC microscopic scenes. Our key idea is to perform this segmentation by distinguishing the different motion patterns in image time series rather than by solving standard image segmentation tasks in each image frame. Thus, we are able to compensate for the poor imaging conditions. We model motion patterns by energies based on the idea of dynamic textures, and regularize the model by two prior energies on the shape of the aortic region and on the topological relationship between the thrombus and the aorta. We demonstrate the performance of our segmentation algorithm by qualitative and quantitative experiments on synthetic examples as well as on real in vivo microscopic sequences.</p>
<p class="publication"><span class="title">Medical image analysis</span>&nbsp;<span class="year">2012</span>;<span class="volume">16</span>;<span class="number">4</span>;<span class="pagerange">915-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22482997">22482997</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3740235">3740235</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.media.2012.02.002">10.1016/j.media.2012.02.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of Detailed Phenotype Profiles Reveals CHRNA5-CHRNA3-CHRNB4 Gene Cluster Association With Several Nicotine Dependence Traits.</h4>
<p class="authors"><span class="author">Broms U</span>, <span class="author">Wedenoja J</span>, <span class="author">Largeau MR</span>, <span class="author">Korhonen T</span>, <span class="author">Pitk&auml;niemi J</span>, <span class="author">Keskitalo-Vuokko K</span>, <span class="author">H&auml;pp&ouml;l&auml; A</span>, <span class="author">Heikkil&auml; KH</span>, <span class="author">Heikkil&auml; K</span>, <span class="author">Ripatti S</span>, <span class="author">Sarin AP</span>, <span class="author">Salminen O</span>, <span class="author">Paunio T</span>, <span class="author">Pergadia ML</span>, <span class="author">Madden PA</span>, <span class="author">Kaprio J</span> and <span class="author">Loukola A</span></p>
<p class="affiliation">Department of Public Health, Hjelt Institute, University of Helsinki, P.O.Box 41, FI-00014, Helsinki, Finland. <a href="mailto:%61%6E%75.%6C%6F%75%6B%6F%6C%61%40%68%65%6C%73%69%6E%6B%69.%66%69">&#97;&#110;&#117;&#46;&#108;&#111;&#117;&#107;&#111;&#108;&#97;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a>.</p><p class="abstract"><strong>Introduction</strong>: The role of the nicotinic acetylcholine receptor gene cluster on chromosome 15q24-25 in the etiology of nicotine dependence (ND) is still being defined. In this study, we included all 15 tagging single nucleotide polymorphisms (SNPs) within the CHRNA5-CHRNA3-CHRNB4 cluster and tested associations with 30 smoking-related phenotypes.</p><p class="abstract"><strong>Methods</strong>: The study sample was ascertained from the Finnish Twin Cohort study. Twin pairs born 1938-1957 and concordant for a history of cigarette smoking were recruited along with their family members (mainly siblings), as part of the Nicotine Addiction Genetics consortium. The study sample consisted of 1,428 individuals (59% males) from 735 families, with mean age 55.6 years.</p><p class="abstract"><strong>Results</strong>: We detected multiple novel associations for ND. DSM-IV ND symptoms associated significantly with the proxy SNP Locus 1 (rs2036527, p = .000009) and Locus 2 (rs578776, p = .0001) and tolerance factor of the Nicotine Dependence Syndrome Scale (NDSS) showed suggestive association to rs11636753 (p = .0059), rs11634351 (p = .0069), and rs1948 (p = .0071) in CHRNB4. Furthermore, we report significant association with DSM-IV ND diagnosis (rs2036527, p = .0003) for the first time in a Caucasian population. Several SNPs indicated suggestive association for traits related to ages at smoking initiation. Also, rs11636753 in CHRNB4 showed suggestive association with regular drinking (p = .0029) and the comorbidity of depression and ND (p = .0034). Conclusions: We demonstrate novel associations of DSM-IV ND symptoms and the NDSS tolerance subscale. Our results confirm and extend association findings for other ND measures. We show pleiotropic effects of this gene cluster on multiple measures of ND and also regular drinking and the comorbidity of ND and depression.</p>
<p class="grants">Funded by: NIDA NIH HHS: <span class="grant">R01 DA012854-10</span></p>
<p class="publication"><span class="title">Nicotine &amp; tobacco research : official journal of the Society for Research on Nicotine and Tobacco</span>&nbsp;<span class="year">2012</span>;<span class="volume">14</span>;<span class="number">6</span>;<span class="pagerange">720-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22241830" rel="external">22241830</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3356294" rel="external">3356294</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fntr%2Fntr283" rel="external">10.1093/ntr/ntr283</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Developing insights into the mechanisms of evolution of bacterial pathogens from whole-genome sequences.</h4>
<p class="authors"><span class="author">Bryant J</span>, <span class="author">Chewapreecha C</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Pathogen Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p>
<p class="abstract">Evolution of bacterial pathogen populations has been detected in a variety of ways including phenotypic tests, such as metabolic activity, reaction to antisera and drug resistance and genotypic tests that measure variation in chromosome structure, repetitive loci and individual gene sequences. While informative, these methods only capture a small subset of the total variation and, therefore, have limited resolution. Advances in sequencing technologies have made it feasible to capture whole-genome sequence variation for each sample under study, providing the potential to detect all changes at all positions in the genome from single nucleotide changes to large-scale insertions and deletions. In this review, we focus on recent work that has applied this powerful new approach and summarize some of the advances that this has brought in our understanding of the details of how bacterial pathogens evolve.</p>
<p class="publication"><span class="title">Future microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">11</span>;<span class="pagerange">1283-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23075447" rel="external">23075447</a>; DOI: <a href="http://dx.doi.org/10.2217%2Ffmb.12.108" rel="external">10.2217/fmb.12.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovering genes involved in alcohol dependence and other alcohol responses: role of animal models.</h4>
<p class="authors"><span class="author">Buck KJ</span>, <span class="author">Milner LC</span>, <span class="author">Denmark DL</span>, <span class="author">Grant SG</span> and <span class="author">Kozell LB</span></p>
<p class="affiliation">Oregon Health & Science University, Portland, Oregon and the Department of Veterans Affairs Medical Center, Portland, Oregon.</p>
<p class="abstract">The genetic determinants of alcoholism still are largely unknown, hindering effective treatment and prevention. Systematic approaches to gene discovery are critical if novel genes and mechanisms involved in alcohol dependence are to be identified. Although no animal model can duplicate all aspects of alcoholism in humans, robust animal models for specific alcohol-related traits, including physiological alcohol dependence and associated withdrawal, have been invaluable resources. Using a variety of genetic animal models, the identification of regions of chromosomal DNA that contain a gene or genes which affect a complex phenotype (i.e., quantitative trait loci [QTLs]) has allowed unbiased searches for candidate genes. Several QTLs with large effects on alcohol withdrawal severity in mice have been detected, and fine mapping of these QTLs has placed them in small intervals on mouse chromosomes 1 and 4 (which correspond to certain regions on human chromosomes 1 and 9). Subsequent work led to the identification of underlying quantitative trait genes (QTGs) (e.g., Mpdz) and high-quality QTG candidates (e.g., Kcnj9 and genes involved in mitochondrial respiration and oxidative stress) and their plausible mechanisms of action. Human association studies provide supporting evidence that these QTLs and QTGs may be directly relevant to alcohol risk factors in clinical populations.</p>
<p class="publication"><span class="title">Alcohol research : current reviews</span>&nbsp;<span class="year">2012</span>;<span class="volume">34</span>;<span class="number">3</span>;<span class="pagerange">367-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23134054" rel="external">23134054</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tissue-Specific Splicing of Disordered Segments that Embed Binding Motifs Rewires Protein Interaction Networks.</h4>
<p class="authors"><span class="author">Buljan M</span>, <span class="author">Chalancon G</span>, <span class="author">Eustermann S</span>, <span class="author">Wagner GP</span>, <span class="author">Fuxreiter M</span>, <span class="author">Bateman A</span> and <span class="author">Babu MM</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge CB2 0QH, UK; Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">Alternative inclusion of exons increases the functional diversity of proteins. Among alternatively spliced exons, tissue-specific exons play a critical role in maintaining tissue identity. This raises the question of how tissue-specific protein-coding exons influence protein function. Here we investigate the structural, functional, interaction, and evolutionary properties of constitutive, tissue-specific, and other alternative exons in human. We find that tissue-specific protein segments often contain disordered regions, are enriched in posttranslational modification sites, and frequently embed conserved binding motifs. Furthermore, genes containing tissue-specific exons tend to occupy central positions in interaction networks and display distinct interaction partners in the respective tissues, and are enriched in signaling, development, and disease genes. Based on these findings, we propose that tissue-specific inclusion of disordered segments that contain binding motifs rewires interaction networks and signaling pathways. In this way, tissue-specific splicing may contribute to functional versatility of proteins and increases the diversity of interaction networks across tissues.</p>
<p class="publication"><span class="title">Molecular cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">46</span>;<span class="number">6</span>;<span class="pagerange">871-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22749400" rel="external">22749400</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molcel.2012.05.039" rel="external">10.1016/j.molcel.2012.05.039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biocurators and biocuration: surveying the 21st century challenges.</h4>
<p class="authors"><span class="author">Burge S</span>, <span class="author">Attwood TK</span>, <span class="author">Bateman A</span>, <span class="author">Berardini TZ</span>, <span class="author">Cherry M</span>, <span class="author">O&#39;Donovan C</span>, <span class="author">Xenarios L</span> and <span class="author">Gaudet P</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p><p class="abstract">Curated databases are an integral part of the tool set that researchers use on a daily basis for their work. For most users, however, how databases are maintained, and by whom, is rather obscure. The International Society for Biocuration (ISB) represents biocurators, software engineers, developers and researchers with an interest in biocuration. Its goals include fostering communication between biocurators, promoting and describing their work, and highlighting the added value of biocuration to the world. The ISB recently conducted a survey of biocurators to better understand their educational and scientific backgrounds, their motivations for choosing a curatorial job and their career goals. The results are reported here. From the responses received, it is evident that biocuration is performed by highly trained scientists and perceived to be a stimulating career, offering both intellectual challenges and the satisfaction of performing work essential to the modern scientific community. It is also apparent that the ISB has at least a dual role to play to facilitate biocurators&#39; work: (i) to promote biocuration as a career within the greater scientific community; (ii) to aid the development of resources for biomedical research through promotion of nomenclature and data-sharing standards that will allow interconnection of biological databases and better exploit the pivotal contributions that biocurators are making. DATABASE URL: http://biocurator.org.</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2012</span>;<span class="volume">2012</span>;<span class="pagerange">bar059</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22434828" rel="external">22434828</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308150" rel="external">3308150</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbar059" rel="external">10.1093/database/bar059</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rfam 11.0: 10 years of RNA families.</h4>
<p class="authors"><span class="author">Burge SW</span>, <span class="author">Daub J</span>, <span class="author">Eberhardt R</span>, <span class="author">Tate J</span>, <span class="author">Barquist L</span>, <span class="author">Nawrocki EP</span>, <span class="author">Eddy SR</span>, <span class="author">Gardner PP</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK, Janelia Farm Research Campus, Howard Hughes Medical Institute, Ashburn, VA 20147, USA and School of Biological Sciences, University of Canterbury, Private Bag 4800, Christchurch, 8140 New Zealand.</p>
<p class="abstract">The Rfam database (available via the website at http://rfam.sanger.ac.uk and through our mirror at http://rfam.janelia.org) is a collection of non-coding RNA families, primarily RNAs with a conserved RNA secondary structure, including both RNA genes and mRNA cis-regulatory elements. Each family is represented by a multiple sequence alignment, predicted secondary structure and covariance model. Here we discuss updates to the database in the latest release, Rfam 11.0, including the introduction of genome-based alignments for large families, the introduction of the Rfam Biomart as well as other user interface improvements. Rfam is available under the Creative Commons Zero license.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23125362" rel="external">23125362</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgks1005" rel="external">10.1093/nar/gks1005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New translational assays for preclinical modelling of cognition in schizophrenia: the touchscreen testing method for mice and rats.</h4>
<p class="authors"><span class="author">Bussey TJ</span>, <span class="author">Holmes A</span>, <span class="author">Lyon L</span>, <span class="author">Mar AC</span>, <span class="author">McAllister KA</span>, <span class="author">Nithianantharajah J</span>, <span class="author">Oomen CA</span> and <span class="author">Saksida LM</span></p>
<p class="affiliation">Department of Experimental Psychology, University of Cambridge, Downing Street, Cambridge CB2 3EB, UK. <a href="mailto:%74%6A%62%31%30%30%30%40%63%61%6D.%61%63.%75%6B">&#116;&#106;&#98;&#49;&#48;&#48;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We describe a touchscreen method that satisfies a proposed &#39;wish-list&#39; of desirables for a cognitive testing method for assessing rodent models of schizophrenia. A number of tests relevant to schizophrenia research are described which are currently being developed and validated using this method. These tests can be used to study reward learning, memory, perceptual discrimination, object-place associative learning, attention, impulsivity, compulsivity, extinction, simple Pavlovian conditioning, and other constructs. The tests can be deployed using a &#39;flexible battery&#39; approach to establish a cognitive profile for a particular mouse or rat model. We have found these tests to be capable of detecting not just impairments in function, but enhancements as well, which is essential for testing putative cognitive therapies. New tests are being continuously developed, many of which may prove particularly valuable for schizophrenia research.</p>
<p class="publication"><span class="title">Neuropharmacology</span>&nbsp;<span class="year">2012</span>;<span class="volume">62</span>;<span class="number">3</span>;<span class="pagerange">1191-203</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21530550" rel="external">21530550</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3168710" rel="external">3168710</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neuropharm.2011.04.011" rel="external">10.1016/j.neuropharm.2011.04.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The PRoteomics IDEntification (PRIDE) Converter 2 framework: an improved suite of tools to facilitate data submission to the PRIDE database and the ProteomeXchange consortium.</h4>
<p class="authors"><span class="author">C&ocirc;t&eacute; RG</span>, <span class="author">Griss J</span>, <span class="author">Dianes JA</span>, <span class="author">Wang R</span>, <span class="author">Wright JC</span>, <span class="author">van den Toorn HW</span>, <span class="author">van Breukelen B</span>, <span class="author">Heck AJ</span>, <span class="author">Hulstaert N</span>, <span class="author">Martens L</span>, <span class="author">Reisinger F</span>, <span class="author">Csordas A</span>, <span class="author">Ovelleiro D</span>, <span class="author">Perez-Rivevol Y</span>, <span class="author">Barsnes H</span>, <span class="author">Hermjakob H</span> and <span class="author">Vizca&iacute;no JA</span></p>
<p class="affiliation">Proteomics Services Team, EMBL Outstation, European Bioinformatics Institute (EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">The original PRIDE Converter tool greatly simplified the process of submitting mass spectrometry (MS)-based proteomics data to the PRIDE database. However, after much user feedback, it was noted that the tool had some limitations and could not handle several user requirements that were now becoming commonplace. This prompted us to design and implement a whole new suite of tools that would build on the successes of the original PRIDE Converter and allow users to generate submission-ready, well-annotated PRIDE XML files. The PRIDE Converter 2 tool suite allows users to convert search result files into PRIDE XML (the format needed for performing submissions to the PRIDE database), generate mzTab skeleton files that can be used as a basis to submit quantitative and gel-based MS data, and post-process PRIDE XML files by filtering out contaminants and empty spectra, or by merging several PRIDE XML files together. All the tools have both a graphical user interface that provides a dialog-based, user-friendly way to convert and prepare files for submission, as well as a command-line interface that can be used to integrate the tools into existing or novel pipelines, for batch processing and power users. The PRIDE Converter 2 tool suite will thus become a cornerstone in the submission process to PRIDE and, by extension, to the ProteomeXchange consortium of MS-proteomics data repositories.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/I024204/1</span>; Wellcome Trust: <span class="grant">WT085949MA</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="number">12</span>;<span class="pagerange">1682-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22949509">22949509</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3518121">3518121</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.O112.021543">10.1074/mcp.O112.021543</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Telomeres and cancer: from crisis to stability to crisis to stability.</h4>
<p class="authors"><span class="author">Campbell PJ</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">Telomere attrition unleashes genomic instability, promoting cancer development. Once established, however, the malignant clone often re-establishes genomic stability through overexpression of telomerase. In two papers, one in this issue of Cell and one in the subsequent issue, DePinho and colleagues explore the consequences of telomerase re-expression and its validity as a therapeutic target in mouse models of cancer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093867</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">148</span>;<span class="number">4</span>;<span class="pagerange">633-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22341437" rel="external">22341437</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322332" rel="external">3322332</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2012.01.043" rel="external">10.1016/j.cell.2012.01.043</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Correlation of blood counts with vascular complications in essential thrombocythemia: analysis of the prospective PT1 cohort.</h4>
<p class="authors"><span class="author">Campbell PJ</span>, <span class="author">Maclean C</span>, <span class="author">Beer PA</span>, <span class="author">Buck G</span>, <span class="author">Wheatley K</span>, <span class="author">Kiladjian JJ</span>, <span class="author">Forsyth C</span>, <span class="author">Harrison CN</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, United Kingdom;</p><p class="abstract">Essential thrombocythemia, a myeloproliferative neoplasm, is associated with increased platelet count and risk of thrombosis or hemorrhage. Cytoreductive therapy aims to normalize platelet counts, despite minimal association between platelet count and complication rates. Evidence is increasing for correlation between white cell count and thrombosis but prospective data are lacking. We studied the relationship between vascular complications and 21,887 longitudinal blood counts in a prospective, multicenter cohort of 776 ET patients. After correction for confounding variables, no association was seen between blood counts at diagnosis and future complications. However, platelet count outside normal range during follow-up was associated with immediate risk of major hemorrhage (p=0.0005) but not thrombosis (p=0.7). Elevated white cell count during follow-up correlated with thrombosis (p=0.05) and major hemorrhage (p=0.01). These data imply that the aim of cytoreduction in essential thrombocythemia should be to keep the platelet count, and arguably the white cell count, within the normal range. This study is registered at http://isrctn.org as #72251782.</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22709688" rel="external">22709688</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2012-04-424911" rel="external">10.1182/blood-2012-04-424911</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Specific expression of Kcna10, Pxn and Odf2 in the organ of Corti.</h4>
<p class="authors"><span class="author">Carlisle FA</span>, <span class="author">Steel KP</span> and <span class="author">Lewis MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The development of the organ of Corti and the highly specialized cells required for hearing involves a multitude of genes, many of which remain unknown. Here we describe the expression pattern of three genes not previously studied in the inner ear in mice at a range of ages both embryonic and early postnatal. Kcna10, a tetrameric Shaker-like potassium channel, is expressed strongly in the hair cells themselves. Odf2, as its centriolar isoform Cenexin, marks the dendrites extending to and contacting hair cells, and Pxn, a focal adhesion scaffold protein, is most strongly expressed in pillar cells during the ages studied. The roles of these genes are yet to be elucidated, but their specific expression patterns imply potential functional significance in the inner ear.</p>
<p class="publication"><span class="title">Gene expression patterns : GEP</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="number">5-6</span>;<span class="pagerange">172-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22446089" rel="external">22446089</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3368262" rel="external">3368262</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gep.2012.03.001" rel="external">10.1016/j.gep.2012.03.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association study of nonsynonymous single nucleotide polymorphisms in schizophrenia.</h4>
<p class="authors"><span class="author">Carrera N</span>, <span class="author">Arrojo M</span>, <span class="author">Sanju&aacute;n J</span>, <span class="author">Ramos-R&iacute;os R</span>, <span class="author">Paz E</span>, <span class="author">Su&aacute;rez-Rama JJ</span>, <span class="author">P&aacute;ramo M</span>, <span class="author">Agra S</span>, <span class="author">Brenlla J</span>, <span class="author">Mart&iacute;nez S</span>, <span class="author">Rivero O</span>, <span class="author">Collier DA</span>, <span class="author">Palotie A</span>, <span class="author">Cichon S</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Rietschel M</span>, <span class="author">Rujescu D</span>, <span class="author">Stefansson H</span>, <span class="author">Steinberg S</span>, <span class="author">Sigurdsson E</span>, <span class="author">St Clair D</span>, <span class="author">Tosato S</span>, <span class="author">Werge T</span>, <span class="author">Stefansson K</span>, <span class="author">Gonz&aacute;lez JC</span>, <span class="author">Valero J</span>, <span class="author">Guti&eacute;rrez-Zotes A</span>, <span class="author">Labad A</span>, <span class="author">Martorell L</span>, <span class="author">Vilella E</span>, <span class="author">Carracedo &Aacute;</span> and <span class="author">Costas J</span></p>
<p class="affiliation">Fundaci&oacute;n P&uacute;blica Galega de Medicina Xen&oacute;mica-SERGAS, Hospital Cl&iacute;nico Universitario, Santiago de Compostela, Spain.</p><p class="abstract"><strong>Background</strong>: Genome-wide association studies using several hundred thousand anonymous markers present limited statistical power. Alternatively, association studies restricted to common nonsynonymous single nucleotide polymorphisms (nsSNPs) have the advantage of strongly reducing the multiple testing problem, while increasing the probability of testing functional single nucleotide polymorphisms (SNPs).</p><p class="abstract"><strong>Methods</strong>: We performed a case-control association study of common nsSNPs in Galician (northwest Spain) samples using the Affymetrix GeneChip Human 20k cSNP Kit, followed by a replication study of the more promising results. After quality control procedures, the discovery sample consisted of 5100 nsSNPs at minor allele frequency &gt;5% analyzed in 476 schizophrenia patients and 447 control subjects. The replication sample consisted of 4069 cases and 15,128 control subjects of European origin. We also performed multilocus analysis, using aggregated scores of nsSNPs at liberal significance thresholds and cross-validation procedures.</p><p class="abstract"><strong>Results</strong>: The 5 independent nsSNPs with false discovery rate q &le; .25, as well as 13 additional nsSNPs at p &lt; .01 and located in functional candidate genes, were genotyped in the replication samples. One SNP, rs13107325, located at the metal ions transporter gene SLC39A8, reached significance in the combined sample after Bonferroni correction (trend test, p = 2.7 &times; 10(-6), allelic odds ratio = 1.32). This SNP presents minor allele frequency of 5% to 10% in many European populations but is rare outside Europe. We also confirmed the polygenic component of susceptibility.</p><p class="abstract"><strong>Conclusions</strong>: Taking into account that another metal ions transporter gene, SLC39A3, is associated to bipolar disorder, our findings reveal a role for brain metal homeostasis in psychosis.</p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2012</span>;<span class="volume">71</span>;<span class="number">2</span>;<span class="pagerange">169-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22078303">22078303</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2011.09.032">10.1016/j.biopsych.2011.09.032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Artemis: an integrated platform for visualization and analysis of high-throughput sequence-based experimental data.</h4>
<p class="authors"><span class="author">Carver T</span>, <span class="author">Harris SR</span>, <span class="author">Berriman M</span>, <span class="author">Parkhill J</span> and <span class="author">McQuillan JA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%72%74%65%6D%69%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#116;&#101;&#109;&#105;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: High-throughput sequencing (HTS) technologies have made low-cost sequencing of large numbers of samples commonplace. An explosion in the type, not just number, of sequencing experiments has also taken place including genome re-sequencing, population-scale variation detection, whole transcriptome sequencing and genome-wide analysis of protein-bound nucleic acids.</p><p class="abstract"><strong>Results</strong>: We present Artemis as a tool for integrated visualization and computational analysis of different types of HTS datasets in the context of a reference genome and its corresponding annotation.</p><p class="abstract"><strong>Availability</strong>: Artemis is freely available (under a GPL licence) for download (for MacOSX, UNIX and Windows) at the Wellcome Trust Sanger Institute websites: http://www.sanger.ac.uk/resources/software/artemis/.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 076964</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">4</span>;<span class="pagerange">464-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22199388">22199388</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3278759">3278759</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr703">10.1093/bioinformatics/btr703</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microevolution of extensively drug-resistant tuberculosis in Russia.</h4>
<p class="authors"><span class="author">Casali N</span>, <span class="author">Nikolayevskyy V</span>, <span class="author">Balabanova Y</span>, <span class="author">Ignatyeva O</span>, <span class="author">Kontsevaya I</span>, <span class="author">Harris SR</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Nejentsev S</span>, <span class="author">Hoffner SE</span>, <span class="author">Horstmann RD</span>, <span class="author">Brown T</span> and <span class="author">Drobniewski F</span></p>
<p class="affiliation">National Mycobacterium Reference Laboratory, Blizard Institute, Queen Mary, University of London, London E1 2AT, United Kingdom.</p><p class="abstract">Extensively drug-resistant (XDR) tuberculosis (TB), which is resistant to both first- and second-line antibiotics, is an escalating problem, particularly in the Russian Federation. Molecular fingerprinting of 2348 Mycobacterium tuberculosis isolates collected in Samara Oblast, Russia, revealed that 72% belonged to the Beijing lineage, a genotype associated with enhanced acquisition of drug resistance and increased virulence. Whole-genome sequencing of 34 Samaran isolates, plus 25 isolates representing global M. tuberculosis complex diversity, revealed that Beijing isolates originating in Eastern Europe formed a monophyletic group. Homoplasic polymorphisms within this clade were almost invariably associated with antibiotic resistance, indicating that the evolution of this population is primarily driven by drug therapy. Resistance genotypes showed a strong correlation with drug susceptibility phenotypes. A novel homoplasic mutation in rpoC, found only in isolates carrying a common rpoB rifampicin-resistance mutation, may play a role in fitness compensation. Most multidrug-resistant (MDR) isolates also had mutations in the promoter of a virulence gene, eis, which increase its expression and confer kanamycin resistance. Kanamycin therapy may thus select for mutants with increased virulence, helping preserve bacterial fitness and promoting transmission of drug-resistant TB strains. The East European clade was dominated by two MDR clusters, each disseminated across Samara. Polymorphisms conferring fluoroquinolone resistance were independently acquired multiple times within each cluster, indicating that XDR TB is currently not widely transmitted.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">4</span>;<span class="pagerange">735-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22294518" rel="external">22294518</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3317155" rel="external">3317155</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.128678.111" rel="external">10.1101/gr.128678.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phylogeographic variation in recombination rates within a global clone of methicillin-resistant Staphylococcus aureus.</h4>
<p class="authors"><span class="author">Castillo-Ram&iacute;rez S</span>, <span class="author">Corander J</span>, <span class="author">Marttinen P</span>, <span class="author">Aldeljawi M</span>, <span class="author">Hanage WP</span>, <span class="author">Westh H</span>, <span class="author">Boye K</span>, <span class="author">Gulay Z</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Holden MT</span> and <span class="author">Feil EJ</span></p>
<p class="affiliation">Department of Biology and Biochemistry, University of Bath, Claverton Down Bath, Bath and North East Somerset BA2 7AY, UK. <a href="mailto:%45.%46%65%69%6C%40%62%61%74%68.%61%63.%75%6B">&#69;&#46;&#70;&#101;&#105;&#108;&#64;&#98;&#97;&#116;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">BACKGROUND: Next-generation sequencing (NGS) is a powerful tool for understanding both patterns of descent over time and space (phylogeography) and the molecular processes underpinning genome divergence in pathogenic bacteria. Here, we describe a synthesis between these perspectives by employing a recently developed Bayesian approach, BRATNextGen, for detecting recombination on an expanded NGS dataset of the globally disseminated methicillin-resistant Staphylococcus aureus (MRSA) clone ST239. RESULTS: The data confirm strong geographical clustering at continental, national and city scales and demonstrate that the rate of recombination varies significantly between phylogeographic sub-groups representing independent introductions from Europe. These differences are most striking when mobile non-core genes are included, but remain apparent even when only considering the stable core genome. The monophyletic ST239 sub-group corresponding to isolates from South America shows heightened recombination, the sub-group predominantly from Asia shows an intermediate level, and a very low level of recombination is noted in a third sub-group representing a large collection from Turkey. CONCLUSIONS: We show that the rapid global dissemination of a single pathogenic bacterial clone results in local variation in measured recombination rates. Possible explanatory variables include the size and time since emergence of each defined sub-population (as determined by the sampling frame), variation in transmission dynamics due to host movement, and changes in the bacterial genome affecting the propensity for recombination.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">12</span>;<span class="pagerange">R126</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23270620">23270620</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-12-r126">10.1186/gb-2012-13-12-r126</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of the GGCX (CAA)16/17 repeat polymorphism with higher warfarin dose requirements in African Americans.</h4>
<p class="authors"><span class="author">Cavallari LH</span>, <span class="author">Perera M</span>, <span class="author">Wadelius M</span>, <span class="author">Deloukas P</span>, <span class="author">Taube G</span>, <span class="author">Patel SR</span>, <span class="author">Aquino-Michaels K</span>, <span class="author">Viana MA</span>, <span class="author">Shapiro NL</span> and <span class="author">Nutescu EA</span></p>
<p class="affiliation">Department of Pharmacy Practice, University of Illinois, Chicago, IL 60612-7230, USA. <a href="mailto:%68%75%6D%6D%61%40%75%69%63.%65%64%75">&#104;&#117;&#109;&#109;&#97;&#64;&#117;&#105;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Objective</strong>: Little is known about genetic contributors to higher than usual warfarin dose requirements, particularly for African Americans. This study tested the hypothesis that the &gamma;-glutamyl carboxylase (GGCX) genotype contributes to warfarin dose requirements greater than 7.5 mg/day in an African American population.</p><p class="abstract"><strong>Methods</strong>: A total of 338 African Americans on a stable dose of warfarin were enrolled. The GGCX rs10654848 (CAA)n, rs12714145 (G&gt;A), and rs699664 (p.R325Q); VKORC1 c.-1639G&gt;A and rs61162043; and CYP2C9*2, *3, *5, *8, *11, and rs7089580 genotypes were tested for their association with dose requirements greater than 7.5 mg/day alone and in the context of other variables known to influence dose variability.</p><p class="abstract"><strong>Results</strong>: The GGCX rs10654848 (CAA)16 or 17 repeat occurred at a frequency of 2.6% in African Americans and was overrepresented among patients requiring greater than 7.5 mg/day versus those who required lower doses (12 vs. 3%, P=0.003; odds ratio 4.0, 95% confidence interval, 1.5-10.5). The GGCX rs10654848 genotype remained associated with high dose requirements on regression analysis including age, body size, and VKORC1 genotype. On linear regression, the GGCX rs10654848 genotype explained 2% of the overall variability in warfarin dose in African Americans. An examination of the GGCX rs10654848 genotype in warfarin-treated Caucasians revealed a (CAA)16 repeat frequency of only 0.27% (P=0.008 compared with African Americans).</p><p class="abstract"><strong>Conclusion</strong>: These data support the GGCX rs10654848 genotype as a predictor of higher than usual warfarin doses in African Americans, who have a 10-fold higher frequency of the (CAA)16/17 repeat compared with Caucasians.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">K23 HL089808-01A2</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Pharmacogenetics and genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">2</span>;<span class="pagerange">152-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22158446" rel="external">22158446</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3261355" rel="external">3261355</a>; DOI: <a href="http://dx.doi.org/10.1097%2FFPC.0b013e32834f288f" rel="external">10.1097/FPC.0b013e32834f288f</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A common X-linked inborn error of carnitine biosynthesis may be a risk factor for nondysmorphic autism.</h4>
<p class="authors"><span class="author">Celestino-Soper PB</span>, <span class="author">Violante S</span>, <span class="author">Crawford EL</span>, <span class="author">Luo R</span>, <span class="author">Lionel AC</span>, <span class="author">Delaby E</span>, <span class="author">Cai G</span>, <span class="author">Sadikovic B</span>, <span class="author">Lee K</span>, <span class="author">Lo C</span>, <span class="author">Gao K</span>, <span class="author">Person RE</span>, <span class="author">Moss TJ</span>, <span class="author">German JR</span>, <span class="author">Huang N</span>, <span class="author">Shinawi M</span>, <span class="author">Treadwell-Deering D</span>, <span class="author">Szatmari P</span>, <span class="author">Roberts W</span>, <span class="author">Fernandez B</span>, <span class="author">Schroer RJ</span>, <span class="author">Stevenson RE</span>, <span class="author">Buxbaum JD</span>, <span class="author">Betancur C</span>, <span class="author">Scherer SW</span>, <span class="author">Sanders SJ</span>, <span class="author">Geschwind DH</span>, <span class="author">Sutcliffe JS</span>, <span class="author">Hurles ME</span>, <span class="author">Wanders RJ</span>, <span class="author">Shaw CA</span>, <span class="author">Leal SM</span>, <span class="author">Cook EH</span>, <span class="author">Goin-Kochel RP</span>, <span class="author">Vaz FM</span> and <span class="author">Beaudet AL</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA.</p>
<p class="abstract">We recently reported a deletion of exon 2 of the trimethyllysine hydroxylase epsilon (TMLHE) gene in a proband with autism. TMLHE maps to the X chromosome and encodes the first enzyme in carnitine biosynthesis, 6-N-trimethyllysine dioxygenase. Deletion of exon 2 of TMLHE causes enzyme deficiency, resulting in increased substrate concentration (6-N-trimethyllysine) and decreased product levels (3-hydroxy-6-N-trimethyllysine and Î³-butyrobetaine) in plasma and urine. TMLHE deficiency is common in control males (24 in 8,787 or 1 in 366) and was not significantly increased in frequency in probands from simplex autism families (9 in 2,904 or 1 in 323). However, it was 2.82-fold more frequent in probands from male-male multiplex autism families compared with controls (7 in 909 or 1 in 130; P = 0.023). Additionally, six of seven autistic male siblings of probands in male-male multiplex families had the deletion, suggesting that TMLHE deficiency is a risk factor for autism (metaanalysis Z-score = 2.90 and P = 0.0037), although with low penetrance (2-4%). These data suggest that dysregulation of carnitine metabolism may be important in nondysmorphic autism; that abnormalities of carnitine intake, loss, transport, or synthesis may be important in a larger fraction of nondysmorphic autism cases; and that the carnitine pathway may provide a novel target for therapy or prevention of autism.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD-37283</span>, <span class="grant">P30HD-0240640</span>; NIMH NIH HHS: <span class="grant">1U24MH081810</span>, <span class="grant">R01 MH061009</span>; NINDS NIH HHS: <span class="grant">R01 NS049261</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077014/Z/05/Z</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">21</span>;<span class="pagerange">7974-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22566635" rel="external">22566635</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3361440" rel="external">3361440</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1120210109" rel="external">10.1073/pnas.1120210109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Finding a needle in a haystack. Microbial metatranscriptomes.</h4>
<p class="authors"><span class="author">Chappell L</span></p><p class="abstract">This month&#39;s Genome Watch highlights some of the technical challenges that need to be overcome to gain further insight into microbial metatranscriptomes.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">7</span>;<span class="pagerange">446</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22699963">22699963</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2821">10.1038/nrmicro2821</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inheritance of coronary artery disease in men: an analysis of the role of the Y chromosome.</h4>
<p class="authors"><span class="author">Charchar FJ</span>, <span class="author">Bloomer LD</span>, <span class="author">Barnes TA</span>, <span class="author">Cowley MJ</span>, <span class="author">Nelson CP</span>, <span class="author">Wang Y</span>, <span class="author">Denniff M</span>, <span class="author">Debiec R</span>, <span class="author">Christofidou P</span>, <span class="author">Nankervis S</span>, <span class="author">Dominiczak AF</span>, <span class="author">Bani-Mustafa A</span>, <span class="author">Balmforth AJ</span>, <span class="author">Hall AS</span>, <span class="author">Erdmann J</span>, <span class="author">Cambien F</span>, <span class="author">Deloukas P</span>, <span class="author">Hengstenberg C</span>, <span class="author">Packard C</span>, <span class="author">Schunkert H</span>, <span class="author">Ouwehand WH</span>, <span class="author">Ford I</span>, <span class="author">Goodall AH</span>, <span class="author">Jobling MA</span>, <span class="author">Samani NJ</span> and <span class="author">Tomaszewski M</span></p>
<p class="affiliation">School of Health Sciences, University of Ballarat, Ballarat, VIC, Australia.</p><p class="abstract"><strong>Background</strong>: A sexual dimorphism exists in the incidence and prevalence of coronary artery disease--men are more commonly affected than are age-matched women. We explored the role of the Y chromosome in coronary artery disease in the context of this sexual inequity.</p><p class="abstract"><strong>Methods</strong>: We genotyped 11 markers of the male-specific region of the Y chromosome in 3233 biologically unrelated British men from three cohorts: the British Heart Foundation Family Heart Study (BHF-FHS), West of Scotland Coronary Prevention Study (WOSCOPS), and Cardiogenics Study. On the basis of this information, each Y chromosome was tracked back into one of 13 ancient lineages defined as haplogroups. We then examined associations between common Y chromosome haplogroups and the risk of coronary artery disease in cross-sectional BHF-FHS and prospective WOSCOPS. Finally, we undertook functional analysis of Y chromosome effects on monocyte and macrophage transcriptome in British men from the Cardiogenics Study.</p><p class="abstract"><strong>Findings</strong>: Of nine haplogroups identified, two (R1b1b2 and I) accounted for roughly 90% of the Y chromosome variants among British men. Carriers of haplogroup I had about a 50% higher age-adjusted risk of coronary artery disease than did men with other Y chromosome lineages in BHF-FHS (odds ratio 1&middot;75, 95% CI 1&middot;20-2&middot;54, p=0&middot;004), WOSCOPS (1&middot;45, 1&middot;08-1&middot;95, p=0&middot;012), and joint analysis of both populations (1&middot;56, 1&middot;24-1&middot;97, p=0&middot;0002). The association between haplogroup I and increased risk of coronary artery disease was independent of traditional cardiovascular and socioeconomic risk factors. Analysis of macrophage transcriptome in the Cardiogenics Study revealed that 19 molecular pathways showing strong differential expression between men with haplogroup I and other lineages of the Y chromosome were interconnected by common genes related to inflammation and immunity, and that some of them have a strong relevance to atherosclerosis.</p><p class="abstract"><strong>Interpretation</strong>: The human Y chromosome is associated with risk of coronary artery disease in men of European ancestry, possibly through interactions of immunity and inflammation.</p><p class="abstract"><strong>Funding</strong>: British Heart Foundation; UK National Institute for Health Research; LEW Carty Charitable Fund; National Health and Medical Research Council of Australia; European Union 6th Framework Programme; Wellcome Trust.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/06/097/21331</span>; Wellcome Trust: <span class="grant">087576</span>, <span class="grant">WT-0841383/2/07/2</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2012</span>;<span class="volume">379</span>;<span class="number">9819</span>;<span class="pagerange">915-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22325189" rel="external">22325189</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3314981" rel="external">3314981</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(11)61453-0" rel="external">10.1016/S0140-6736(11)61453-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using family-based imputation in genome-wide association studies with large complex pedigrees: the framingham heart study.</h4>
<p class="authors"><span class="author">Chen MH</span>, <span class="author">Huang J</span>, <span class="author">Chen WM</span>, <span class="author">Larson MG</span>, <span class="author">Fox CS</span>, <span class="author">Vasan RS</span>, <span class="author">Seshadri S</span>, <span class="author">O&#39;Donnell CJ</span> and <span class="author">Yang Q</span></p>
<p class="affiliation">Department of Neurology, Boston University School of Medicine, Boston, Massachusetts, United States of America ; Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts, United States of America ; The National Heart, Lung, Blood Institute's Framingham Heart Study, Framingham, Massachusetts, United States of America.</p>
<p class="abstract">Imputation has been widely used in genome-wide association studies (GWAS) to infer genotypes of un-genotyped variants based on the linkage disequilibrium in external reference panels such as the HapMap and 1000 Genomes. However, imputation has only rarely been performed based on family relationships to infer genotypes of un-genotyped individuals. Using 8998 Framingham Heart Study (FHS) participants genotyped with Affymetrix 550K SNPs, we imputed genotypes of same set of SNPs for additional 3121 participants, most of whom were never genotyped due to lack of DNA sample. Prior to imputation, 122 pedigrees were too large to be handled by the imputation software Merlin. Therefore, we developed a novel pedigree splitting algorithm that can maximize the number of genotyped relatives for imputing each un-genotyped individual, while keeping new sub-pedigrees under a pre-specified size. In GWAS of four phenotypes available in FHS (Alzheimer disease, circulating levels of fibrinogen, high-density lipoprotein cholesterol, and uric acid), we compared results using genotyped individuals only with results using both genotyped and imputed individuals. We studied the impact of applying different imputation quality filtering thresholds on the association results and did not found a universal threshold that always resulted in a more significant p-value for previously identified loci. However most of these loci had a lower p-value when we only included imputed genotypes with with â‰¥60% SNP- and â‰¥50% person-specific imputation certainty. In summary, we developed a novel algorithm for splitting large pedigrees for imputation and found a plausible imputation quality filtering threshold based on FHS. Further examination may be required to generalize this threshold to other studies.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e51589</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23284720" rel="external">23284720</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3524237" rel="external">3524237</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0051589" rel="external">10.1371/journal.pone.0051589</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bayesian estimation of bacterial community composition from 454 sequencing data.</h4>
<p class="authors"><span class="author">Cheng L</span>, <span class="author">Walker AW</span> and <span class="author">Corander J</span></p>
<p class="affiliation">Department of Mathematics and Statistics, P.O.Box 68 (Gustaf H&auml;llstr&ouml;min katu 2b), University of Helsinki, 00014 Helsinki, Finland and Pathogen Genomics Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">Estimating bacterial community composition from a mixed sample in different applied contexts is an important task for many microbiologists. The bacterial community composition is commonly estimated by clustering polymerase chain reaction amplified 16S rRNA gene sequences. Current taxonomy-independent clustering methods for analyzing these sequences, such as UCLUST, ESPRIT-Tree and CROP, have two limitations: (i) expert knowledge is needed, i.e. a difference cutoff between species needs to be specified; (ii) closely related species cannot be separated. The first limitation imposes a burden on the user, since considerable effort is needed to select appropriate parameters, whereas the second limitation leads to an inaccurate description of the underlying bacterial community composition. We propose a probabilistic model-based method to estimate bacterial community composition which tackles these limitations. Our method requires very little expert knowledge, where only the possible maximum number of clusters needs to be specified. Also our method demonstrates its ability to separate closely related species in two experiments, in spite of sequencing errors and individual variations.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22406836" rel="external">22406836</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgks227" rel="external">10.1093/nar/gks227</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Copy number variation of the APC gene is associated with regulation of bone mineral density.</h4>
<p class="authors"><span class="author">Chew S</span>, <span class="author">Dastani Z</span>, <span class="author">Brown SJ</span>, <span class="author">Lewis JR</span>, <span class="author">Dudbridge F</span>, <span class="author">Soranzo N</span>, <span class="author">Surdulescu GL</span>, <span class="author">Richards JB</span>, <span class="author">Spector TD</span> and <span class="author">Wilson SG</span></p>
<p class="affiliation">Department of Endocrinology and Diabetes, Sir Charles Gairdner Hospital, Nedlands 6009, Australia. <a href="mailto:%53%68%65%6C%62%79.%43%68%65%77%40%68%65%61%6C%74%68.%77%61.%67%6F%76.%61%75">&#83;&#104;&#101;&#108;&#98;&#121;&#46;&#67;&#104;&#101;&#119;&#64;&#104;&#101;&#97;&#108;&#116;&#104;&#46;&#119;&#97;&#46;&#103;&#111;&#118;&#46;&#97;&#117;</a></p><p class="abstract"><strong>Introduction</strong>: Genetic studies of osteoporosis have commonly examined SNPs in candidate genes or whole genome analyses, but insertions and deletions of DNA, collectively called copy number variations (CNVs), also comprise a large amount of the genetic variability between individuals. Previously, SNPs in the APC gene have been strongly associated with femoral neck and lumbar spine volumetric bone mineral density in older men. In addition, familial adenomatous polyposis patients carrying heterozygous mutations in the APC gene have been shown to have significantly higher mean bone mineral density than age- and sex-matched controls suggesting the importance of this gene in regulating bone mineral density. We examined CNV within the APC gene region to test for association with bone mineral density.</p><p class="abstract"><strong>Methods</strong>: DNA was extracted from venous blood, genotyped using the Human Hap610 arrays and CNV determined from the fluorescence intensity data in 2070 Caucasian men and women aged 47.0 &plusmn; 13.0 (mean &plusmn; SD) years, to assess the effects of the CNV on bone mineral density at the forearm, spine and total hip sites.</p><p class="abstract"><strong>Results</strong>: Data for covariate adjusted bone mineral density from subjects grouped by APC CNV genotype showed significant difference (P=0.02-0.002). Subjects with a single copy loss of APC had a 7.95%, 13.10% and 13.36% increase in bone mineral density at the forearm, spine and total hip sites respectively, compared to subjects with two copies of the APC gene.</p><p class="abstract"><strong>Conclusions</strong>: These data support previous findings of APC regulating bone mineral density and demonstrate that a novel CNV of the APC gene is significantly associated with bone mineral density in Caucasian men and women.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Wellcome Trust</p>
<p class="publication"><span class="title">Bone</span>&nbsp;<span class="year">2012</span>;<span class="volume">51</span>;<span class="number">5</span>;<span class="pagerange">939-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22884971">22884971</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3918860">3918860</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.bone.2012.07.022">10.1016/j.bone.2012.07.022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome watch: Natural transformers.</h4>
<p class="authors"><span class="author">Chewapreecha C</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">9</span>;<span class="pagerange">598</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22890145" rel="external">22890145</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2865" rel="external">10.1038/nrmicro2865</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conversion from mouse embryonic to extra-embryonic endoderm stem cells reveals distinct differentiation capacities of pluripotent stem cell states.</h4>
<p class="authors"><span class="author">Cho LT</span>, <span class="author">Wamaitha SE</span>, <span class="author">Tsai IJ</span>, <span class="author">Artus J</span>, <span class="author">Sherwood RI</span>, <span class="author">Pedersen RA</span>, <span class="author">Hadjantonakis AK</span> and <span class="author">Niakan KK</span></p>
<p class="affiliation">The Anne McLaren Laboratory for Regenerative Medicine, Stem Cell Institute, University of Cambridge, Cambridge CB2 0SZ, UK.</p>
<p class="abstract">The inner cell mass of the mouse pre-implantation blastocyst comprises epiblast progenitor and primitive endoderm cells of which cognate embryonic (mESCs) or extra-embryonic (XEN) stem cell lines can be derived. Importantly, each stem cell type retains the defining properties and lineage restriction of their in vivo tissue of origin. Recently, we demonstrated that XEN-like cells arise within mESC cultures. This raises the possibility that mESCs can generate self-renewing XEN cells without the requirement for gene manipulation. We have developed a novel approach to convert mESCs to XEN cells (cXEN) using growth factors. We confirm that the downregulation of the pluripotency transcription factor Nanog and the expression of primitive endoderm-associated genes Gata6, Gata4, Sox17 and Pdgfra are necessary for cXEN cell derivation. This approach highlights an important function for Fgf4 in cXEN cell derivation. Paracrine FGF signalling compensates for the loss of endogenous Fgf4, which is necessary to exit mESC self-renewal, but not for XEN cell maintenance. Our cXEN protocol also reveals that distinct pluripotent stem cells respond uniquely to differentiation promoting signals. cXEN cells can be derived from mESCs cultured with Erk and Gsk3 inhibitors (2i), and LIF, similar to conventional mESCs. However, we find that epiblast stem cells (EpiSCs) derived from the post-implantation embryo are refractory to cXEN cell establishment, consistent with the hypothesis that EpiSCs represent a pluripotent state distinct from mESCs. In all, these findings suggest that the potential of mESCs includes the capacity to give rise to both extra-embryonic and embryonic lineages.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">139</span>;<span class="number">16</span>;<span class="pagerange">2866-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22791892" rel="external">22791892</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.078519" rel="external">10.1242/dev.078519</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced susceptibility to Citrobacter rodentium infection in miR-155-deficient mice.</h4>
<p class="authors"><span class="author">Clare S</span>, <span class="author">John V</span>, <span class="author">Walker AW</span>, <span class="author">Hill JL</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">Hale C</span>, <span class="author">Goulding D</span>, <span class="author">Lawley TD</span>, <span class="author">Mastroeni P</span>, <span class="author">Frankel G</span>, <span class="author">Enright AJ</span>, <span class="author">Vigorito E</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p>
<p class="abstract">MicroRNAs (miRNAs) are small non-coding molecules that control gene expression post-transcriptionally, with miR-155 one of the first to be implicated in immune regulation. Here we show that miR-155-deficient mice are less able to eradicate a mucosal Citrobacter rodentium infection compared with wild-type C57BL/6 mice. miR-155-deficient mice exhibited prolonged colonisation associated with a higher C. rodentium burden in gastrointestinal tissue and spread into systemic tissues. Germinal centre formation and humoral immune responses against C. rodentium were severely impaired in infected miR-155 deficient mice. A similarly susceptible phenotype was observed in Î¼MT mice reconstituted with miR-155-deficient B cells indicating that miR-155 is required intrinsically for mediating protection against this predominantly luminal bacterial pathogen.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23264052" rel="external">23264052</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00969-12" rel="external">10.1128/IAI.00969-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&#39;Sifting the significance from the data&#39; - the impact of high-throughput genomic technologies on human genetics and health care.</h4>
<p class="authors"><span class="author">Clarke AJ</span>, <span class="author">Cooper DN</span>, <span class="author">Krawczak M</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Wallace HM</span>, <span class="author">Wilkie AO</span>, <span class="author">Raymond FL</span>, <span class="author">Chadwick R</span>, <span class="author">Craddock N</span>, <span class="author">John R</span>, <span class="author">Gallacher J</span> and <span class="author">Chiano M</span></p>
<p class="affiliation">Institute of Medical Genetics, School of Medicine, Cardiff University, Cardiff, Wales CF14 4XN, UK. <a href="mailto:%63%6C%61%72%6B%65%61%6A%40%63%61%72%64%69%66%66.%61%63.%75%6B">&#99;&#108;&#97;&#114;&#107;&#101;&#97;&#106;&#64;&#99;&#97;&#114;&#100;&#105;&#102;&#102;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This report is of a round-table discussion held in Cardiff in September 2009 for Cesagen, a research centre within the Genomics Network of the UK&#39;s Economic and Social Research Council. The meeting was arranged to explore ideas as to the likely future course of human genomics. The achievements of genomics research were reviewed, and the likely constraints on the pace of future progress were explored. New knowledge is transforming biology and our understanding of evolution and human disease. The difficulties we face now concern the interpretation rather than the generation of new sequence data. Our understanding of gene-environment interaction is held back by our current primitive tools for measuring environmental factors, and in addition, there may be fundamental constraints on what can be known about these complex interactions.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="pagerange">11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23244462">23244462</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3500243">3500243</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1479-7364-6-11">10.1186/1479-7364-6-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cfp1 integrates both CpG content and gene activity for accurate H3K4me3 deposition in embryonic stem cells.</h4>
<p class="authors"><span class="author">Clouaire T</span>, <span class="author">Webb S</span>, <span class="author">Skene P</span>, <span class="author">Illingworth R</span>, <span class="author">Kerr A</span>, <span class="author">Andrews R</span>, <span class="author">Lee JH</span>, <span class="author">Skalnik D</span> and <span class="author">Bird A</span></p>
<p class="affiliation">Wellcome Trust Centre for Cell Biology, University of Edinburgh, Edinburgh EH9 3JR, United Kingdom;</p>
<p class="abstract">Trimethylation of histone H3 Lys 4 (H3K4me3) is a mark of active and poised promoters. The Set1 complex is responsible for most somatic H3K4me3 and contains the conserved subunit CxxC finger protein 1 (Cfp1), which binds to unmethylated CpGs and links H3K4me3 with CpG islands (CGIs). Here we report that Cfp1 plays unanticipated roles in organizing genome-wide H3K4me3 in embryonic stem cells. Cfp1 deficiency caused two contrasting phenotypes: drastic loss of H3K4me3 at expressed CGI-associated genes, with minimal consequences for transcription, and creation of "ectopic" H3K4me3 peaks at numerous regulatory regions. DNA binding by Cfp1 was dispensable for targeting H3K4me3 to active genes but was required to prevent ectopic H3K4me3 peaks. The presence of ectopic peaks at enhancers often coincided with increased expression of nearby genes. This suggests that CpG targeting prevents "leakage" of H3K4me3 to inappropriate chromatin compartments. Our results demonstrate that Cfp1 is a specificity factor that integrates multiple signals, including promoter CpG content and gene activity, to regulate genome-wide patterns of H3K4me3.</p>
<p class="publication"><span class="title">Genes &amp;amp; development</span>&nbsp;<span class="year">2012</span>;<span class="volume">26</span>;<span class="number">15</span>;<span class="pagerange">1714-28</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22855832" rel="external">22855832</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3418589" rel="external">3418589</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.194209.112" rel="external">10.1101/gad.194209.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TNiK is required for postsynaptic and nuclear signaling pathways and cognitive function.</h4>
<p class="authors"><span class="author">Coba MP</span>, <span class="author">Komiyama NH</span>, <span class="author">Nithianantharajah J</span>, <span class="author">Kopanitsa MV</span>, <span class="author">Indersmitten T</span>, <span class="author">Skene NG</span>, <span class="author">Tuck EJ</span>, <span class="author">Fricker DG</span>, <span class="author">Elsegood KA</span>, <span class="author">Stanford LE</span>, <span class="author">Afinowi NO</span>, <span class="author">Saksida LM</span>, <span class="author">Bussey TJ</span>, <span class="author">O&#39;Dell TJ</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom.</p><p class="abstract">Traf2 and NcK interacting kinase (TNiK) contains serine-threonine kinase and scaffold domains and has been implicated in cell proliferation and glutamate receptor regulation in vitro. Here we report its role in vivo using mice carrying a knock-out mutation. TNiK binds protein complexes in the synapse linking it to the NMDA receptor (NMDAR) via AKAP9. NMDAR and metabotropic receptors bidirectionally regulate TNiK phosphorylation and TNiK is required for AMPA expression and synaptic function. TNiK also organizes nuclear complexes and in the absence of TNiK, there was a marked elevation in GSK3&beta; and phosphorylation levels of its cognate phosphorylation sites on NeuroD1 with alterations in Wnt pathway signaling. We observed impairments in dentate gyrus neurogenesis in TNiK knock-out mice and cognitive testing using the touchscreen apparatus revealed impairments in pattern separation on a test of spatial discrimination. Object-location paired associate learning, which is dependent on glutamatergic signaling, was also impaired. Additionally, TNiK knock-out mice displayed hyperlocomotor behavior that could be rapidly reversed by GSK3&beta; inhibitors, indicating the potential for pharmacological rescue of a behavioral phenotype. These data establish TNiK as a critical regulator of cognitive functions and suggest it may play a regulatory role in diseases impacting on its interacting proteins and complexes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0802238</span>; NIMH NIH HHS: <span class="grant">MH609197</span>, <span class="grant">R01 MH060919</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2012</span>;<span class="volume">32</span>;<span class="number">40</span>;<span class="pagerange">13987-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23035106">23035106</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3978779">3978779</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.2433-12.2012">10.1523/JNEUROSCI.2433-12.2012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The YARHG Domain: An Extracellular Domain in Search of a Function.</h4>
<p class="authors"><span class="author">Coggill P</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">We have identified a new bacterial protein domain that we hypothesise binds to peptidoglycan. This domain is called the YARHG domain after the most highly conserved sequence-segment. The domain is found in the extracellular space and is likely to be composed of four alpha-helices. The domain is found associated with protein kinase domains, suggesting it is associated with signalling in some bacteria. The domain is also found associated with three different families of peptidases. The large number of different domains that are found associated with YARHG suggests that it is a useful functional module that nature has recombined multiple times.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">e35575</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22615736" rel="external">22615736</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3355149" rel="external">3355149</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0035575" rel="external">10.1371/journal.pone.0035575</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SpolPred: rapid and accurate prediction of Mycobacterium tuberculosis spoligotypes from short genomic sequences.</h4>
<p class="authors"><span class="author">Coll F</span>, <span class="author">Mallard K</span>, <span class="author">Preston MD</span>, <span class="author">Bentley S</span>, <span class="author">Parkhill J</span>, <span class="author">McNerney R</span>, <span class="author">Martin N</span> and <span class="author">Clark TG</span></p>
<p class="affiliation">Faculty of Infectious and Tropical Diseases, London School of Hygiene and Tropical Medicine, London, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton and Department of Computer Science and Information Systems, Pathogen Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Birkbeck College, London, UK.</p>
<p class="abstract">Summary: Spoligotyping is a well-established genotyping technique based on the presence of unique DNA sequences in Mycobacterium tuberculosis (Mtb), the causal agent of tuberculosis disease (TB). Although advances in sequencing technologies are leading to whole-genome bacterial characterization, tens of thousands of isolates have been spoligotyped, giving a global view of Mtb strain diversity. To bridge the gap, we have developed SpolPred, a software to predict the spoligotype from raw sequence reads. Our approach is compared with experimentally and de novo assembly determined strain types in a set of 44 Mtb isolates. In silico and experimental results are identical for almost all isolates (39/44). However, SpolPred detected five experimentally false spoligotypes and was more accurate and faster than the assembling strategy. Application of SpolPred to an additional seven isolates with no laboratory data led to types that clustered with identical experimental types in a phylogenetic analysis using single-nucleotide polymorphisms. Our results demonstrate the usefulness of the tool and its role in revealing experimental limitations. Availability and implementation: SpolPred is written in C and is available from www.pathogenseq.org/spolpred. CONTACT: <a href="mailto:%66%72%61%6E%63%65%73%63.%63%6F%6C%6C%40%6C%73%68%74%6D.%61%63.%75%6B">&#102;&#114;&#97;&#110;&#99;&#101;&#115;&#99;&#46;&#99;&#111;&#108;&#108;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a> SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics Online.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">22</span>;<span class="pagerange">2991-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23014632" rel="external">23014632</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3496340" rel="external">3496340</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts544" rel="external">10.1093/bioinformatics/bts544</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fibrinogen-binding and platelet-aggregation activities of a Lactobacillus salivarius septicaemia isolate are mediated by a novel fibrinogen-binding protein.</h4>
<p class="authors"><span class="author">Collins J</span>, <span class="author">van Pijkeren JP</span>, <span class="author">Svensson L</span>, <span class="author">Claesson MJ</span>, <span class="author">Sturme M</span>, <span class="author">Li Y</span>, <span class="author">Cooney JC</span>, <span class="author">van Sinderen D</span>, <span class="author">Walker AW</span>, <span class="author">Parkhill J</span>, <span class="author">Shannon O</span> and <span class="author">O&#39;Toole PW</span></p>
<p class="affiliation">Department of Microbiology and Alimentary Pharmabiotic Centre, University College Cork, Cork, Ireland.</p><p class="abstract">The marketplace for probiotic foods is burgeoning, measured in billions of euro per annum. It is imperative, however, that all bacterial strains are fully assessed for human safety. The ability to bind fibrinogen is considered a potential pathogenicity trait that can lead to platelet aggregation, serious medical complications, and in some instances, death. Here we examined strains from species frequently used as probiotics for their ability to bind human fibrinogen. Only one strain (CCUG 47825), a Lactobacillus salivarius isolate from a case of septicaemia, was found to strongly adhere to fibrinogen. Furthermore, this strain was found to aggregate human platelets at a level comparable to the human pathogen Staphylococcus aureus. By sequencing the genome of CCUG 47825, we were able to identify candidate genes responsible for fibrinogen binding. Complementing the genetic analysis with traditional molecular microbiological techniques enabled the identification of the novel fibrinogen receptor, CCUG_2371. Although only strain CCUG 47825 bound fibrinogen under laboratory conditions, homologues of the novel fibrinogen binding gene CCUG_2371 are widespread among L. salivarius strains, maintaining their potential to bind fibrinogen if expressed. We highlight the fact that without a full genetic analysis of strains for human consumption, potential pathogenicity traits may go undetected.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">85</span>;<span class="number">5</span>;<span class="pagerange">862-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22724453">22724453</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2012.08148.x">10.1111/j.1365-2958.2012.08148.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Incorporating RNA-seq data into the zebrafish Ensembl genebuild.</h4>
<p class="authors"><span class="author">Collins JE</span>, <span class="author">White S</span>, <span class="author">Searle SM</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom.</p>
<p class="abstract">Ensembl gene annotation provides a comprehensive catalog of transcripts aligned to the reference sequence. It relies on publicly available species-specific and orthologous transcripts plus their inferred protein sequence. The accuracy of gene models is improved by increasing the species-specific component that can be cost-effectively achieved using RNA-seq. Two zebrafish gene annotations are presented in Ensembl version 62 built on the Zv9 reference sequence. Firstly, RNA-seq data from five tissues and seven developmental stages were assembled into 25,748 gene models. A 3'-end capture and sequencing protocol was developed to predict the 3' ends of transcripts, and 46.1% of the original models were subsequently refined. Secondly, a standard Ensembl genebuild, incorporating carefully filtered elements from the RNA-seq-only build, followed by a merge with the manually curated VEGA database, produced a comprehensive annotation of 26,152 genes represented by 51,569 transcripts. The RNA-seq-only and the Ensembl/VEGA genebuilds contribute contrasting elements to the final genebuild. The RNA-seq genebuild was used to adjust intron/exon boundaries of orthologous defined models, confirm their expression, and improve 3' untranslated regions. Importantly, the inferred protein alignments within the Ensembl genebuild conferred proof of model contiguity for the RNA-seq models. The zebrafish gene annotation has been enhanced by the incorporation of RNA-seq data and the pipeline will be used for other organisms. Organisms with little species-specific cDNA data will generally benefit the most.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">10</span>;<span class="pagerange">2067-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22798491" rel="external">22798491</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3460200" rel="external">3460200</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.137901.112" rel="external">10.1101/gr.137901.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Early environment and neurobehavioral development predict adult temperament clusters.</h4>
<p class="authors"><span class="author">Congdon E</span>, <span class="author">Service S</span>, <span class="author">Wessman J</span>, <span class="author">Sepp&auml;nen JK</span>, <span class="author">Sch&ouml;nauer S</span>, <span class="author">Miettunen J</span>, <span class="author">Turunen H</span>, <span class="author">Koiranen M</span>, <span class="author">Joukamaa M</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Peltonen L</span>, <span class="author">Veijola J</span>, <span class="author">Mannila H</span>, <span class="author">Paunio T</span> and <span class="author">Freimer NB</span></p>
<p class="affiliation">University of California Los Angeles Center for Neurobehavioral Genetics, Semel Institute for Neuroscience and Human Behavior, University of California Los Angeles, Los Angeles, California, United States of America.</p>
<p class="abstract">Background: Investigation of the environmental influences on human behavioral phenotypes is important for our understanding of the causation of psychiatric disorders. However, there are complexities associated with the assessment of environmental influences on behavior.

We conducted a series of analyses using a prospective, longitudinal study of a nationally representative birth cohort from Finland (the Northern Finland 1966 Birth Cohort). Participants included a total of 3,761 male and female cohort members who were living in Finland at the age of 16 years and who had complete temperament scores. Our initial analyses (Wessman et al., in press) provide evidence in support of four stable and robust temperament clusters. Using these temperament clusters, as well as independent temperament dimensions for comparison, we conducted a data-driven analysis to assess the influence of a broad set of life course measures, assessed pre-natally, in infancy, and during adolescence, on adult temperament.

Results: Measures of early environment, neurobehavioral development, and adolescent behavior significantly predict adult temperament, classified by both cluster membership and temperament dimensions. Specifically, our results suggest that a relatively consistent set of life course measures are associated with adult temperament profiles, including maternal education, characteristics of the family's location and residence, adolescent academic performance, and adolescent smoking.

Conclusions: Our finding that a consistent set of life course measures predict temperament clusters indicate that these clusters represent distinct developmental temperament trajectories and that information about a subset of life course measures has implications for adult health outcomes.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e38065</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22815688" rel="external">22815688</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3399831" rel="external">3399831</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0038065" rel="external">10.1371/journal.pone.0038065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Circulating DNA and next-generation sequencing.</h4>
<p class="authors"><span class="author">Cooke S</span> and <span class="author">Campbell P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">Personalising cancer medicine depends upon the implementation of personalised diagnostics and therapeutics. Detailed genomic screening is likely to play a central role in this. As the range of drugs and other therapies for cancer continues to increase, there is an increasingly urgent need for sensitive and specific measures of disease burden to guide treatment regimens. The ability to quantify disease burden with high accuracy and sensitivity in patients with cancer would open many potential routes to personalising therapeutic choices. For example, the intensity of therapy could be guided by the amount of disease at diagnosis; monitoring the response of patients to drugs could allow extension of the period of treatment in responders or early changeover of therapy in nonresponders; and early prediction of recurrence could allow salvage therapy to be instituted before complications of relapse develop. The detection of tumour-specific rearrangements in DNA free in the serum or plasma may provide a substantial advance in the accuracy of monitoring disease burden in patients with solid tumours.</p>
<p class="publication"><span class="title">Recent results in cancer research. Fortschritte der Krebsforschung. Progr&egrave;s dans les recherches sur le cancer</span>&nbsp;<span class="year">2012</span>;<span class="volume">195</span>;<span class="pagerange">143-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22527501" rel="external">22527501</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-28160-0_12" rel="external">10.1007/978-3-642-28160-0_12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The rocky road to personalized medicine: Computational and statistical challenges</h4>
<p class="authors"><span class="author">Corander J</span>, <span class="author">Aittokallio T</span>, <span class="author">RIPATTI S</span>, <span class="author">Kaski S</span></p>
<p class="publication"><span class="title">Personalized Medicine.</span> <span class="year">2012</span>;<span class="volume">9</span>;<span class="pagerange">109-14</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Streptococcus pneumoniae: the evolution of antimicrobial resistance to beta-lactams, fluoroquinolones and macrolides.</h4>
<p class="authors"><span class="author">Cornick JE</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Malawi-Liverpool-Wellcome Clinical Research Programme, PO Box 30096, Chichiri, Blantyre 3, Malawi; Institute of Infection and Global Health, The University of Liverpool, Liverpool, UK.</p><p class="abstract">Multi drug resistant Streptococcus pneumoniae constitute a major public health concern worldwide. In this review we discuss how the transformable nature of the pneumococcus, in parallel with antimicrobial induced stress, contributes to the evolution of antimicrobial resistance; and how the introduction of the pneumococcal conjugate vaccine has affected the situation.</p>
<p class="publication"><span class="title">Microbes and infection / Institut Pasteur</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22342898" rel="external">22342898</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.micinf.2012.01.012" rel="external">10.1016/j.micinf.2012.01.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interpretation of genomic copy number variants using DECIPHER.</h4>
<p class="authors"><span class="author">Corpas M</span>, <span class="author">Bragin E</span>, <span class="author">Clayton S</span>, <span class="author">Bevan P</span> and <span class="author">Firth HV</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Many patients suffering from developmental disorders have submicroscopic deletions or duplications affecting the copy number of dosage-sensitive genes or disrupting normal gene expression. Many of these changes are novel or extremely rare, making clinical interpretation problematic and genotype/phenotype correlations difficult. Identification of patients sharing a genomic rearrangement and having phenotypes in common increases certainty in the diagnosis and allows characterization of new syndromes. The DECIPHER database is an online repository of genotype and phenotype data whose chief objective is to facilitate the association of genomic variation with phenotype to enable the clinical interpretation of copy number variation (CNV). This unit shows how DECIPHER can be used to (1) search for consented patients sharing a defined chromosomal location, (2) navigate regions of interest using in-house visualization tools and the Ensembl genome browser, (3) analyze affected genes and prioritize them according to their likelihood of haploinsufficiency, (4) upload patient aberrations and phenotypes, and (5) create printouts at different levels of detail. By following this protocol, clinicians and researchers alike will be able to learn how to characterize their patients&#39; chromosomal imbalances using DECIPHER.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Current protocols in human genetics / editorial board, Jonathan L. Haines ... [et al.]</span>&nbsp;<span class="year">2012</span>;<span class="volume">Chapter 8</span>;<span class="pagerange">Unit 8.14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22241657">22241657</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471142905.hg0814s72">10.1002/0471142905.hg0814s72</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Direct sequencing of small genomes on the Pacific Biosciences RS without library preparation.</h4>
<p class="authors"><span class="author">Coupland P</span>, <span class="author">Chandra T</span>, <span class="author">Quail M</span>, <span class="author">Reik W</span> and <span class="author">Swerdlow H</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%70%63%31%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#99;&#49;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have developed a sequencing method on the Pacific Biosciences RS sequencer (the PacBio) for small DNA molecules that avoids the need for a standard library preparation. To date this approach has been applied toward sequencing single-stranded and double-stranded viral genomes, bacterial plasmids, plasmid vector models for DNA-modification analysis, and linear DNA fragments covering an entire bacterial genome. Using direct sequencing it is possible to generate sequence data from as little as 1 ng of DNA, offering a significant advantage over current protocols which typically require 400-500 ng of sheared DNA for the library preparation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801156</span>; Wellcome Trust: <span class="grant">095645</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">BioTechniques</span>&nbsp;<span class="year">2012</span>;<span class="volume">53</span>;<span class="number">6</span>;<span class="pagerange">365-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23227987">23227987</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3808810">3808810</a>; DOI: <a href="http://dx.doi.org/10.2144%2F000113962">10.2144/000113962</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association meta-analysis of circulating sex hormone-binding globulin reveals multiple Loci implicated in sex steroid hormone regulation.</h4>
<p class="authors"><span class="author">Coviello AD</span>, <span class="author">Haring R</span>, <span class="author">Wellons M</span>, <span class="author">Vaidya D</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Keildson S</span>, <span class="author">Lunetta KL</span>, <span class="author">He C</span>, <span class="author">Fornage M</span>, <span class="author">Lagou V</span>, <span class="author">Mangino M</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Chen B</span>, <span class="author">Eriksson J</span>, <span class="author">Garcia M</span>, <span class="author">Liu YM</span>, <span class="author">Koster A</span>, <span class="author">Lohman K</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Petersen AK</span>, <span class="author">Prescott J</span>, <span class="author">Stolk L</span>, <span class="author">Vandenput L</span>, <span class="author">Wood AR</span>, <span class="author">Zhuang WV</span>, <span class="author">Ruokonen A</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">Bandinelli S</span>, <span class="author">Biffar R</span>, <span class="author">Brabant G</span>, <span class="author">Cox DG</span>, <span class="author">Chen Y</span>, <span class="author">Cummings S</span>, <span class="author">Ferrucci L</span>, <span class="author">Gunter MJ</span>, <span class="author">Hankinson SE</span>, <span class="author">Martikainen H</span>, <span class="author">Hofman A</span>, <span class="author">Homuth G</span>, <span class="author">Illig T</span>, <span class="author">Jansson JO</span>, <span class="author">Johnson AD</span>, <span class="author">Karasik D</span>, <span class="author">Karlsson M</span>, <span class="author">Kettunen J</span>, <span class="author">Kiel DP</span>, <span class="author">Kraft P</span>, <span class="author">Liu J</span>, <span class="author">Ljunggren &Ouml;</span>, <span class="author">Lorentzon M</span>, <span class="author">Maggio M</span>, <span class="author">Markus MR</span>, <span class="author">Mellstr&ouml;m D</span>, <span class="author">Miljkovic I</span>, <span class="author">Mirel D</span>, <span class="author">Nelson S</span>, <span class="author">Morin Papunen L</span>, <span class="author">Peeters PH</span>, <span class="author">Prokopenko I</span>, <span class="author">Raffel L</span>, <span class="author">Reincke M</span>, <span class="author">Reiner AP</span>, <span class="author">Rexrode K</span>, <span class="author">Rivadeneira F</span>, <span class="author">Schwartz SM</span>, <span class="author">Siscovick D</span>, <span class="author">Soranzo N</span>, <span class="author">St&ouml;ckl D</span>, <span class="author">Tworoger S</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Gils CH</span>, <span class="author">Vasan RS</span>, <span class="author">Wichmann HE</span>, <span class="author">Zhai G</span>, <span class="author">Bhasin S</span>, <span class="author">Bidlingmaier M</span>, <span class="author">Chanock SJ</span>, <span class="author">De Vivo I</span>, <span class="author">Harris TB</span>, <span class="author">Hunter DJ</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Liu S</span>, <span class="author">Ouyang P</span>, <span class="author">Spector TD</span>, <span class="author">van der Schouw YT</span>, <span class="author">Viikari J</span>, <span class="author">Wallaschofski H</span>, <span class="author">McCarthy MI</span>, <span class="author">Frayling TM</span>, <span class="author">Murray A</span>, <span class="author">Franks S</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">de Jong FH</span>, <span class="author">Raitakari O</span>, <span class="author">Teumer A</span>, <span class="author">Ohlsson C</span>, <span class="author">Murabito JM</span> and <span class="author">Perry JR</span></p>
<p class="affiliation">Section of Preventive Medicine and Epidemiology, Boston University School of Medicine, Boston, Massachusetts, United States of America.</p>
<p class="abstract">Sex hormone-binding globulin (SHBG) is a glycoprotein responsible for the transport and biologic availability of sex steroid hormones, primarily testosterone and estradiol. SHBG has been associated with chronic diseases including type 2 diabetes (T2D) and with hormone-sensitive cancers such as breast and prostate cancer. We performed a genome-wide association study (GWAS) meta-analysis of 21,791 individuals from 10 epidemiologic studies and validated these findings in 7,046 individuals in an additional six studies. We identified twelve genomic regions (SNPs) associated with circulating SHBG concentrations. Loci near the identified SNPs included SHBG (rs12150660, 17p13.1, p = 1.8 × 10(-106)), PRMT6 (rs17496332, 1p13.3, p = 1.4 × 10(-11)), GCKR (rs780093, 2p23.3, p = 2.2 × 10(-16)), ZBTB10 (rs440837, 8q21.13, p = 3.4 × 10(-09)), JMJD1C (rs7910927, 10q21.3, p = 6.1 × 10(-35)), SLCO1B1 (rs4149056, 12p12.1, p = 1.9 × 10(-08)), NR2F2 (rs8023580, 15q26.2, p = 8.3 × 10(-12)), ZNF652 (rs2411984, 17q21.32, p = 3.5 × 10(-14)), TDGF3 (rs1573036, Xq22.3, p = 4.1 × 10(-14)), LHCGR (rs10454142, 2p16.3, p = 1.3 × 10(-07)), BAIAP2L1 (rs3779195, 7q21.3, p = 2.7 × 10(-08)), and UGT2B15 (rs293428, 4q13.2, p = 5.5 × 10(-06)). These genes encompass multiple biologic pathways, including hepatic function, lipid metabolism, carbohydrate metabolism and T2D, androgen and estrogen receptor function, epigenetic effects, and the biology of sex steroid hormone-responsive cancers including breast and prostate cancer. We found evidence of sex-differentiated genetic influences on SHBG. In a sex-specific GWAS, the loci 4q13.2-UGT2B15 was significant in men only (men p = 2.5 × 10(-08), women p = 0.66, heterogeneity p = 0.003). Additionally, three loci showed strong sex-differentiated effects: 17p13.1-SHBG and Xq22.3-TDGF3 were stronger in men, whereas 8q21.12-ZBTB10 was stronger in women. Conditional analyses identified additional signals at the SHBG gene that together almost double the proportion of variance explained at the locus. Using an independent study of 1,129 individuals, all SNPs identified in the overall or sex-differentiated or conditional analyses explained ~15.6% and ~8.4% of the genetic variation of SHBG concentrations in men and women, respectively. The evidence for sex-differentiated effects and allelic heterogeneity highlight the importance of considering these features when estimating complex trait variance.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">QLG1-CT-2000-01643</span>; NCI NIH HHS: <span class="grant">CA128034</span>, <span class="grant">CA40356</span>, <span class="grant">CA49449</span>, <span class="grant">CA87969</span>, <span class="grant">U01-CA98233</span>; NCRR NIH HHS: <span class="grant">RR-024156</span>, <span class="grant">U54 RR020278</span>; NHGRI NIH HHS: <span class="grant">U01 HG005152</span>, <span class="grant">U01-HG-004424</span>, <span class="grant">U01-HG-004446</span>, <span class="grant">U01-HG-004729</span>; NHLBI NIH HHS: <span class="grant">5-K23-HL087114</span>, <span class="grant">5R01HL087679-02</span>, <span class="grant">HL074338</span>, <span class="grant">HL074406</span>, <span class="grant">HL087679</span>, <span class="grant">N01 HC-95159</span>, <span class="grant">N01-HC-05187</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-45134</span>, <span class="grant">N01-HC-45204</span>, <span class="grant">N01-HC-45205</span>, <span class="grant">N01-HC-48047</span>, <span class="grant">N01-HC-48048</span>, <span class="grant">N01-HC-48049</span>, <span class="grant">N01-HC-48050</span>, <span class="grant">N01-HC-95095</span>, <span class="grant">N01-HC-95160</span>, <span class="grant">N01-HC-95161</span>, <span class="grant">N01-HC-95162</span>, <span class="grant">N01-HC-95163</span>, <span class="grant">N01-HC-95164</span>, <span class="grant">N01-HC-95165</span>, <span class="grant">N01-HC-95166</span>, <span class="grant">N01-HC-95167</span>, <span class="grant">N01-HC-95168</span>, <span class="grant">N01-HC-95169</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01-HL-084099</span>, <span class="grant">R01-HL065611</span>, <span class="grant">R01HL094755</span>; NIA NIH HHS: <span class="grant">1-AG-1-2111</span>, <span class="grant">1R01AG032098-01A1</span>, <span class="grant">N.1-AG-1-1</span>, <span class="grant">N01-AG-5-0002</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>, <span class="grant">R01AG31206</span>, <span class="grant">R21AG032598</span>; NIAMS NIH HHS: <span class="grant">AR 41398</span>; NIGMS NIH HHS: <span class="grant">GM053275-14</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706</span>, <span class="grant">MH083268</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>; PHS HHS: <span class="grant">24152</span>, <span class="grant">32100-2</span>, <span class="grant">32105-6</span>, <span class="grant">32108-9</span>, <span class="grant">32111-13</span>, <span class="grant">32115</span>, <span class="grant">32118-32119</span>, <span class="grant">32122</span>, <span class="grant">41071</span>, <span class="grant">42107-26</span>, <span class="grant">42129-32</span>, <span class="grant">44221</span>, <span class="grant">HHSN268200782096C</span>, <span class="grant">NFBC1966</span>; WHI NIH HHS: <span class="grant">N01WH22110</span>; Wellcome Trust: <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">e1002805</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22829776" rel="external">22829776</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3400553" rel="external">3400553</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002805" rel="external">10.1371/journal.pgen.1002805</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">LRP1B deletion in high-grade serous ovarian cancers is associated with acquired chemotherapy resistance to liposomal doxorubicin.</h4>
<p class="authors"><span class="author">Cowin PA</span>, <span class="author">George J</span>, <span class="author">Fereday S</span>, <span class="author">Loehrer E</span>, <span class="author">Van Loo P</span>, <span class="author">Cullinane C</span>, <span class="author">Etemadmoghadam D</span>, <span class="author">Ftouni S</span>, <span class="author">Galletta L</span>, <span class="author">Anglesio MS</span>, <span class="author">Hendley J</span>, <span class="author">Bowes L</span>, <span class="author">Sheppard KE</span>, <span class="author">Christie EL</span>, <span class="author">Pearson RB</span>, <span class="author">Harnett PR</span>, <span class="author">Heinzelmann-Schwarz V</span>, <span class="author">Friedlander M</span>, <span class="author">McNally O</span>, <span class="author">Quinn M</span>, <span class="author">Campbell P</span>, <span class="author">deFazio A</span>, <span class="author">Bowtell DD</span> and <span class="author">Australian Ovarian Cancer Study</span></p>
<p class="affiliation">Cancer Genomics and Genetics Program, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.</p><p class="abstract">High-grade serous cancer (HGSC), the most common subtype of ovarian cancer, often becomes resistant to chemotherapy, leading to poor patient outcomes. Intratumoral heterogeneity occurs in nearly all solid cancers, including ovarian cancer, contributing to the development of resistance mechanisms. In this study, we examined the spatial and temporal genomic variation in HGSC using high-resolution single-nucleotide polymorphism arrays. Multiple metastatic lesions from individual patients were analyzed along with 22 paired pretreatment and posttreatment samples. We documented regions of differential DNA copy number between multiple tumor biopsies that correlated with altered expression of genes involved in cell polarity and adhesion. In the paired primary and relapse cohort, we observed a greater degree of genomic change in tumors from patients that were initially sensitive to chemotherapy and had longer progression-free interval compared with tumors from patients that were resistant to primary chemotherapy. Notably, deletion or downregulation of the lipid transporter LRP1B emerged as a significant correlate of acquired resistance in our analysis. Functional studies showed that reducing LRP1B expression was sufficient to reduce the sensitivity of HGSC cell lines to liposomal doxorubicin, but not to doxorubicin, whereas LRP1B overexpression was sufficient to increase sensitivity to liposomal doxorubicin. Together, our findings underscore the large degree of variation in DNA copy number in spatially and temporally separated tumors in HGSC patients, and they define LRP1B as a potential contributor to the emergence of chemotherapy resistance in these patients.</p>
<p class="grants">Funded by: Worldwide Cancer Research: <span class="grant">04-0416</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2012</span>;<span class="volume">72</span>;<span class="number">16</span>;<span class="pagerange">4060-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22896685">22896685</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-12-0203">10.1158/0008-5472.CAN-12-0203</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A small predatory core genome in the divergent marine Bacteriovorax marinus SJ and the terrestrial Bdellovibrio bacteriovorus.</h4>
<p class="authors"><span class="author">Crossman LC</span>, <span class="author">Chen H</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Brooks K</span>, <span class="author">Quail MA</span>, <span class="author">Pineiro SA</span>, <span class="author">Hobley L</span>, <span class="author">Sockett RE</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Williams HN</span> and <span class="author">Stine OC</span></p>
<p class="affiliation">1] Department of Bioinformatics, The Genome Analysis Centre, Norwich Research Park, Norwich, UK [2] School of Biological Sciences, University of East Anglia, Norwich, UK.</p>
<p class="abstract">Bacteriovorax marinus SJ is a predatory delta-proteobacterium isolated from a marine environment. The genome sequence of this strain provides an interesting contrast to that of the terrestrial predatory bacterium Bdellovibrio bacteriovorus HD100. Based on their predatory lifestyle, Bacteriovorax were originally designated as members of the genus Bdellovibrio but subsequently were re-assigned to a new genus and family based on genetic and phenotypic differences. B. marinus attaches to Gram-negative bacteria, penetrates through the cell wall to form a bdelloplast, in which it replicates, as shown using microscopy. Bacteriovorax is distinct, as it shares only 30% of its gene products with its closest sequenced relatives. Remarkably, 34% of predicted genes over 500â€‰nt in length were completely unique with no significant matches in the databases. As expected, Bacteriovorax shares several characteristic loci with the other delta-proteobacteria. A geneset shared between Bacteriovorax and Bdellovibrio that is not conserved among other delta-proteobacteria such as Myxobacteria (which destroy prey bacteria externally via lysis), or the non-predatory Desulfo-bacteria and Geobacter species was identified. These 291 gene orthologues common to both Bacteriovorax and Bdellovibrio may be the key indicators of host-interaction predatory-specific processes required for prey entry. The locus from Bdellovibrio bacteriovorus is implicated in the switch from predatory to prey/host-independent growth. Although the locus is conserved in B. marinus, the sequence has only limited similarity. The results of this study advance understanding of both the similarities and differences between Bdellovibrio and Bacteriovorax and confirm the distant relationship between the two and their separation into different families.The ISME Journal advance online publication, 6 September 2012; doi:10.1038/ismej.2012.90.</p>
<p class="publication"><span class="title">The ISME journal</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955231" rel="external">22955231</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fismej.2012.90" rel="external">10.1038/ismej.2012.90</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high-resolution view of genome-wide pneumococcal transformation.</h4>
<p class="authors"><span class="author">Croucher NJ</span>, <span class="author">Harris SR</span>, <span class="author">Barquist L</span>, <span class="author">Parkhill J</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Pathogen Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">Transformation is an important mechanism of microbial evolution through which bacteria have been observed to rapidly adapt in response to clinical interventions; examples include facilitating vaccine evasion and the development of penicillin resistance in the major respiratory pathogen Streptococcus pneumoniae. To characterise the process in detail, the genomes of 124 S. pneumoniae isolates produced through in vitro transformation were sequenced and recombination events detected. Those recombinations importing the selected marker were independent of unselected events elsewhere in the genome, the positions of which were not significantly affected by local sequence similarity between donor and recipient or mismatch repair processes. However, both types of recombinations were sometimes mosaic, with multiple non-contiguous segments originating from the same molecule of donor DNA. The lengths of the unselected events were exponentially distributed with a mean of 2.3 kb, implying that recombinations are stochastically resolved with a fixed per base probability of 4.4×10(-4) bp(-1). This distribution of recombination sizes, coupled with an observed under representation of large insertions within transferred sequence, suggests transformation has the potential to reduce the size of bacterial genomes, and is unlikely to act as an efficient mechanism for the uptake of accessory genomic loci.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">e1002745</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22719250" rel="external">22719250</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002745" rel="external">10.1371/journal.ppat.1002745</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Investigation of Host Candidate Malaria-Associated Risk/Protective SNPs in a Brazilian Amazonian Population.</h4>
<p class="authors"><span class="author">da Silva Santos S</span>, <span class="author">Clark TG</span>, <span class="author">Campino S</span>, <span class="author">Suarez-Mutis MC</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Fernandes O</span></p>
<p class="affiliation">Laborat&oacute;rio Interdisciplinar de Pesquisas M&eacute;dicas, IOC, Fiocruz, Rio de Janeiro, Brazil.</p><p class="abstract">The Brazilian Amazon is a hypo-endemic malaria region with nearly 300,000 cases each year. A variety of genetic polymorphisms, particularly in erythrocyte receptors and immune response related genes, have been described to be associated with susceptibility and resistance to malaria. In order to identify polymorphisms that might be associated with malaria clinical outcomes in a Brazilian Amazonian population, sixty-four human single nucleotide polymorphisms in 37 genes were analyzed using a Sequenom massARRAY iPLEX platform. A total of 648 individuals from two malaria endemic areas were studied, including 535 malaria cases (113 individuals with clinical mild malaria, 122 individuals with asymptomatic infection and 300 individuals with history of previous mild malaria) and 113 health controls with no history of malaria. The data revealed significant associations (p&lt;0.003) between one SNP in the IL10 gene (rs1800896) and one SNP in the TLR4 gene (rs4986790) with reduced risk for clinical malaria, one SNP in the IRF1 gene (rs2706384) with increased risk for clinical malaria, one SNP in the LTA gene (rs909253) with protection from clinical malaria and one SNP in the TNF gene (RS1800750) associated with susceptibility to clinical malaria. Also, a new association was found between a SNP in the CTL4 gene (rs2242665), located at the major histocompatibility complex III region, and reduced risk for clinical malaria. This study represents the first association study from an Amazonian population involving a large number of host genetic polymorphisms with susceptibility or resistance to Plasmodium infection and malaria outcomes. Further studies should include a larger number of individuals, refined parameters and a fine-scale map obtained through DNA sequencing to increase the knowledge of the Amazonian population genetic diversity.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">e36692</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22615793" rel="external">22615793</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3353949" rel="external">3353949</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0036692" rel="external">10.1371/journal.pone.0036692</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High levels of RNA-editing site conservation amongst 15 laboratory mouse strains.</h4>
<p class="authors"><span class="author">Danecek P</span>, <span class="author">Nell&aring;ker C</span>, <span class="author">McIntyre RE</span>, <span class="author">Buendia-Buendia JE</span>, <span class="author">Bumpstead S</span>, <span class="author">Ponting CP</span>, <span class="author">Flint J</span>, <span class="author">Durbin R</span>, <span class="author">Keane TM</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1HH, UK.</p><p class="abstract"><strong>Background</strong>: Adenosine-to-inosine (A-to-I) editing is a site-selective post-transcriptional alteration of double-stranded RNA by ADAR deaminases that is crucial for homeostasis and development. Recently the Mouse Genomes Project generated genome sequences for 17 laboratory mouse strains and rich catalogues of variants. We also generated RNA-seq data from whole brain RNA from 15 of the sequenced strains.</p><p class="abstract"><strong>Results</strong>: Here we present a computational approach that takes an initial set of transcriptome/genome mismatch sites and filters these calls taking into account systematic biases in alignment, single nucleotide variant calling, and sequencing depth to identify RNA editing sites with high accuracy. We applied this approach to our panel of mouse strain transcriptomes identifying 7,389 editing sites with an estimated false-discovery rate of between 2.9 and 10.5%. The overwhelming majority of these edits were of the A-to-I type, with less than 2.4% not of this class, and only three of these edits could not be explained as alignment artifacts. We validated 24 novel RNA editing sites in coding sequence, including two non-synonymous edits in the Cacna1d gene that fell into the IQ domain portion of the Cav1.2 voltage-gated calcium channel, indicating a potential role for editing in the generation of transcript diversity.</p><p class="abstract"><strong>Conclusions</strong>: We show that despite over two million years of evolutionary divergence, the sites edited and the level of editing at each site is remarkably consistent across the 15 strains. In the Cds2 gene we find evidence for RNA editing acting to preserve the ancestral transcript sequence despite genomic sequence divergence.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>, <span class="grant">MC_U137761446</span>; Wellcome Trust: <span class="grant">079912</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22524474">22524474</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3446300">3446300</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-4-r26">10.1186/gb-2012-13-4-r26</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel loci for adiponectin levels and their influence on type 2 diabetes and metabolic traits: a multi-ethnic meta-analysis of 45,891 individuals.</h4>
<p class="authors"><span class="author">Dastani Z</span>, <span class="author">Hivert MF</span>, <span class="author">Timpson N</span>, <span class="author">Perry JR</span>, <span class="author">Yuan X</span>, <span class="author">Scott RA</span>, <span class="author">Henneman P</span>, <span class="author">Heid IM</span>, <span class="author">Kizer JR</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Fuchsberger C</span>, <span class="author">Tanaka T</span>, <span class="author">Morris AP</span>, <span class="author">Small K</span>, <span class="author">Isaacs A</span>, <span class="author">Beekman M</span>, <span class="author">Coassin S</span>, <span class="author">Lohman K</span>, <span class="author">Qi L</span>, <span class="author">Kanoni S</span>, <span class="author">Pankow JS</span>, <span class="author">Uh HW</span>, <span class="author">Wu Y</span>, <span class="author">Bidulescu A</span>, <span class="author">Rasmussen-Torvik LJ</span>, <span class="author">Greenwood CM</span>, <span class="author">Ladouceur M</span>, <span class="author">Grimsby J</span>, <span class="author">Manning AK</span>, <span class="author">Liu CT</span>, <span class="author">Kooner J</span>, <span class="author">Mooser VE</span>, <span class="author">Vollenweider P</span>, <span class="author">Kapur KA</span>, <span class="author">Chambers J</span>, <span class="author">Wareham NJ</span>, <span class="author">Langenberg C</span>, <span class="author">Frants R</span>, <span class="author">Willems-Vandijk K</span>, <span class="author">Oostra BA</span>, <span class="author">Willems SM</span>, <span class="author">Lamina C</span>, <span class="author">Winkler TW</span>, <span class="author">Psaty BM</span>, <span class="author">Tracy RP</span>, <span class="author">Brody J</span>, <span class="author">Chen I</span>, <span class="author">Viikari J</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Pramstaller PP</span>, <span class="author">Evans DM</span>, <span class="author">St Pourcain B</span>, <span class="author">Sattar N</span>, <span class="author">Wood AR</span>, <span class="author">Bandinelli S</span>, <span class="author">Carlson OD</span>, <span class="author">Egan JM</span>, <span class="author">B&ouml;hringer S</span>, <span class="author">van Heemst D</span>, <span class="author">Kedenko L</span>, <span class="author">Kristiansson K</span>, <span class="author">Nuotio ML</span>, <span class="author">Loo BM</span>, <span class="author">Harris T</span>, <span class="author">Garcia M</span>, <span class="author">Kanaya A</span>, <span class="author">Haun M</span>, <span class="author">Klopp N</span>, <span class="author">Wichmann HE</span>, <span class="author">Deloukas P</span>, <span class="author">Katsareli E</span>, <span class="author">Couper DJ</span>, <span class="author">Duncan BB</span>, <span class="author">Kloppenburg M</span>, <span class="author">Adair LS</span>, <span class="author">Borja JB</span>, <span class="author">DIAGRAM+ Consortium</span>, <span class="author">MAGIC Consortium</span>, <span class="author">GLGC Investigators</span>, <span class="author">MuTHER Consortium</span>, <span class="author">Wilson JG</span>, <span class="author">Musani S</span>, <span class="author">Guo X</span>, <span class="author">Johnson T</span>, <span class="author">Semple R</span>, <span class="author">Teslovich TM</span>, <span class="author">Allison MA</span>, <span class="author">Redline S</span>, <span class="author">Buxbaum SG</span>, <span class="author">Mohlke KL</span>, <span class="author">Meulenbelt I</span>, <span class="author">Ballantyne CM</span>, <span class="author">Dedoussis GV</span>, <span class="author">Hu FB</span>, <span class="author">Liu Y</span>, <span class="author">Paulweber B</span>, <span class="author">Spector TD</span>, <span class="author">Slagboom PE</span>, <span class="author">Ferrucci L</span>, <span class="author">Jula A</span>, <span class="author">Perola M</span>, <span class="author">Raitakari O</span>, <span class="author">Florez JC</span>, <span class="author">Salomaa V</span>, <span class="author">Eriksson JG</span>, <span class="author">Frayling TM</span>, <span class="author">Hicks AA</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Smith GD</span>, <span class="author">Siscovick DS</span>, <span class="author">Kronenberg F</span>, <span class="author">van Duijn C</span>, <span class="author">Loos RJ</span>, <span class="author">Waterworth DM</span>, <span class="author">Meigs JB</span>, <span class="author">Dupuis J</span>, <span class="author">Richards JB</span>, <span class="author">Voight BF</span>, <span class="author">Scott LJ</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Dina C</span>, <span class="author">Welch RP</span>, <span class="author">Zeggini E</span>, <span class="author">Huth C</span>, <span class="author">Aulchenko YS</span>, <span class="author">Thorleifsson G</span>, <span class="author">McCulloch LJ</span>, <span class="author">Ferreira T</span>, <span class="author">Grallert H</span>, <span class="author">Amin N</span>, <span class="author">Wu G</span>, <span class="author">Willer CJ</span>, <span class="author">Raychaudhuri S</span>, <span class="author">McCarroll SA</span>, <span class="author">Hofmann OM</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">van Hoek M</span>, <span class="author">Navarro P</span>, <span class="author">Ardlie K</span>, <span class="author">Balkau B</span>, <span class="author">Benediktsson R</span>, <span class="author">Bennett AJ</span>, <span class="author">Blagieva R</span>, <span class="author">Boerwinkle E</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bostr&ouml;m KB</span>, <span class="author">Bravenboer B</span>, <span class="author">Bumpstead S</span>, <span class="author">Burtt NP</span>, <span class="author">Charpentier G</span>, <span class="author">Chines PS</span>, <span class="author">Cornelis M</span>, <span class="author">Crawford G</span>, <span class="author">Doney AS</span>, <span class="author">Elliott KS</span>, <span class="author">Elliott AL</span>, <span class="author">Erdos MR</span>, <span class="author">Fox CS</span>, <span class="author">Franklin CS</span>, <span class="author">Ganser M</span>, <span class="author">Gieger C</span>, <span class="author">Grarup N</span>, <span class="author">Green T</span>, <span class="author">Griffin S</span>, <span class="author">Groves CJ</span>, <span class="author">Guiducci C</span>, <span class="author">Hadjadj S</span>, <span class="author">Hassanali N</span>, <span class="author">Herder C</span>, <span class="author">Isomaa B</span>, <span class="author">Jackson AU</span>, <span class="author">Johnson PR</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Kao WH</span>, <span class="author">Kong A</span>, <span class="author">Kraft P</span>, <span class="author">Kuusisto J</span>, <span class="author">Lauritzen T</span>, <span class="author">Li M</span>, <span class="author">Lieverse A</span>, <span class="author">Lindgren CM</span>, <span class="author">Lyssenko V</span>, <span class="author">Marre M</span>, <span class="author">Meitinger T</span>, <span class="author">Midthjell K</span>, <span class="author">Morken MA</span>, <span class="author">Narisu N</span>, <span class="author">Nilsson P</span>, <span class="author">Owen KR</span>, <span class="author">Payne F</span>, <span class="author">Petersen AK</span>, <span class="author">Platou C</span>, <span class="author">Proen&ccedil;a C</span>, <span class="author">Prokopenko I</span>, <span class="author">Rathmann W</span>, <span class="author">Rayner NW</span>, <span class="author">Robertson NR</span>, <span class="author">Rocheleau G</span>, <span class="author">Roden M</span>, <span class="author">Sampson MJ</span>, <span class="author">Saxena R</span>, <span class="author">Shields BM</span>, <span class="author">Shrader P</span>, <span class="author">Sigurdsson G</span>, <span class="author">Spars&oslash; T</span>, <span class="author">Strassburger K</span>, <span class="author">Stringham HM</span>, <span class="author">Sun Q</span>, <span class="author">Swift AJ</span>, <span class="author">Thorand B</span>, <span class="author">Tichet J</span>, <span class="author">Tuomi T</span>, <span class="author">van Dam RM</span>, <span class="author">van Haeften TW</span>, <span class="author">van Herpt T</span>, <span class="author">van Vliet-Ostaptchouk JV</span>, <span class="author">Walters GB</span>, <span class="author">Weedon MN</span>, <span class="author">Wijmenga C</span>, <span class="author">Witteman J</span>, <span class="author">Bergman RN</span>, <span class="author">Cauchi S</span>, <span class="author">Collins FS</span>, <span class="author">Gloyn AL</span>, <span class="author">Gyllensten U</span>, <span class="author">Hansen T</span>, <span class="author">Hide WA</span>, <span class="author">Hitman GA</span>, <span class="author">Hofman A</span>, <span class="author">Hunter DJ</span>, <span class="author">Hveem K</span>, <span class="author">Laakso M</span>, <span class="author">Morris AD</span>, <span class="author">Palmer CN</span>, <span class="author">Rudan I</span>, <span class="author">Sijbrands E</span>, <span class="author">Stein LD</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uitterlinden A</span>, <span class="author">Walker M</span>, <span class="author">Watanabe RM</span>, <span class="author">Abecasis GR</span>, <span class="author">Boehm BO</span>, <span class="author">Campbell H</span>, <span class="author">Daly MJ</span>, <span class="author">Hattersley AT</span>, <span class="author">Pedersen O</span>, <span class="author">Barroso I</span>, <span class="author">Groop L</span>, <span class="author">Sladek R</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Wilson JF</span>, <span class="author">Illig T</span>, <span class="author">Froguel P</span>, <span class="author">van Duijn CM</span>, <span class="author">Stefansson K</span>, <span class="author">Altshuler D</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span>, <span class="author">Soranzo N</span>, <span class="author">Wheeler E</span>, <span class="author">Glazer NL</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">M&auml;gi R</span>, <span class="author">Randall J</span>, <span class="author">Elliott P</span>, <span class="author">Rybin D</span>, <span class="author">Dehghan A</span>, <span class="author">Hottenga JJ</span>, <span class="author">Song K</span>, <span class="author">Goel A</span>, <span class="author">Lajunen T</span>, <span class="author">Doney A</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Kumari M</span>, <span class="author">Timpson NJ</span>, <span class="author">Zabena C</span>, <span class="author">Ingelsson E</span>, <span class="author">An P</span>, <span class="author">O&#39;Connell J</span>, <span class="author">Luan J</span>, <span class="author">Elliott A</span>, <span class="author">McCarroll SA</span>, <span class="author">Roccasecca RM</span>, <span class="author">Pattou F</span>, <span class="author">Sethupathy P</span>, <span class="author">Ariyurek Y</span>, <span class="author">Barter P</span>, <span class="author">Beilby JP</span>, <span class="author">Ben-Shlomo Y</span>, <span class="author">Bergmann S</span>, <span class="author">Bochud M</span>, <span class="author">Bonnefond A</span>, <span class="author">Borch-Johnsen K</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Brunner E</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Chen YD</span>, <span class="author">Chines P</span>, <span class="author">Clarke R</span>, <span class="author">Coin LJ</span>, <span class="author">Cooper MN</span>, <span class="author">Crisponi L</span>, <span class="author">Day IN</span>, <span class="author">de Geus EJ</span>, <span class="author">Delplanque J</span>, <span class="author">Fedson AC</span>, <span class="author">Fischer-Rosinsky A</span>, <span class="author">Forouhi NG</span>, <span class="author">Franzosi MG</span>, <span class="author">Galan P</span>, <span class="author">Goodarzi MO</span>, <span class="author">Graessler J</span>, <span class="author">Grundy S</span>, <span class="author">Gwilliam R</span>, <span class="author">Hallmans G</span>, <span class="author">Hammond N</span>, <span class="author">Han X</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hayward C</span>, <span class="author">Heath SC</span>, <span class="author">Hercberg S</span>, <span class="author">Hillman DR</span>, <span class="author">Hingorani AD</span>, <span class="author">Hui J</span>, <span class="author">Hung J</span>, <span class="author">Kaakinen M</span>, <span class="author">Kaprio J</span>, <span class="author">Kesaniemi YA</span>, <span class="author">Kivimaki M</span>, <span class="author">Knight B</span>, <span class="author">Koskinen S</span>, <span class="author">Kovacs P</span>, <span class="author">Kyvik KO</span>, <span class="author">Lathrop GM</span>, <span class="author">Lawlor DA</span>, <span class="author">Le Bacquer O</span>, <span class="author">Lecoeur C</span>, <span class="author">Li Y</span>, <span class="author">Mahley R</span>, <span class="author">Mangino M</span>, <span class="author">Mart&iacute;nez-Larrad MT</span>, <span class="author">McAteer JB</span>, <span class="author">McPherson R</span>, <span class="author">Meisinger C</span>, <span class="author">Melzer D</span>, <span class="author">Meyre D</span>, <span class="author">Mitchell BD</span>, <span class="author">Mukherjee S</span>, <span class="author">Naitza S</span>, <span class="author">Neville MJ</span>, <span class="author">Orr&ugrave; M</span>, <span class="author">Pakyz R</span>, <span class="author">Paolisso G</span>, <span class="author">Pattaro C</span>, <span class="author">Pearson D</span>, <span class="author">Peden JF</span>, <span class="author">Pedersen NL</span>, <span class="author">Pfeiffer AF</span>, <span class="author">Pichler I</span>, <span class="author">Polasek O</span>, <span class="author">Posthuma D</span>, <span class="author">Potter SC</span>, <span class="author">Pouta A</span>, <span class="author">Province MA</span>, <span class="author">Rayner NW</span>, <span class="author">Rice K</span>, <span class="author">Ripatti S</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rolandsson O</span>, <span class="author">Sandbaek A</span>, <span class="author">Sandhu M</span>, <span class="author">Sanna S</span>, <span class="author">Sayer AA</span>, <span class="author">Scheet P</span>, <span class="author">Seedorf U</span>, <span class="author">Sharp SJ</span>, <span class="author">Shields B</span>, <span class="author">Sigur&eth;sson G</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Silveira A</span>, <span class="author">Simpson L</span>, <span class="author">Singleton A</span>, <span class="author">Smith NL</span>, <span class="author">Sovio U</span>, <span class="author">Swift A</span>, <span class="author">Syddall H</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Dijk KW</span>, <span class="author">Varma D</span>, <span class="author">Visvikis-Siest S</span>, <span class="author">Vitart V</span>, <span class="author">Vogelzangs N</span>, <span class="author">Waeber G</span>, <span class="author">Wagner PJ</span>, <span class="author">Walley A</span>, <span class="author">Ward KL</span>, <span class="author">Watkins H</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Witteman JC</span>, <span class="author">Yarnell JW</span>, <span class="author">Zelenika D</span>, <span class="author">Zethelius B</span>, <span class="author">Zhai G</span>, <span class="author">Zhao JH</span>, <span class="author">Zillikens MC</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">GIANT Consortium</span>, <span class="author">Global B Pgen Consortium</span>, <span class="author">Borecki IB</span>, <span class="author">Meneton P</span>, <span class="author">Magnusson PK</span>, <span class="author">Nathan DM</span>, <span class="author">Williams GH</span>, <span class="author">Silander K</span>, <span class="author">Bornstein SR</span>, <span class="author">Schwarz P</span>, <span class="author">Spranger J</span>, <span class="author">Karpe F</span>, <span class="author">Shuldiner AR</span>, <span class="author">Cooper C</span>, <span class="author">Serrano-R&iacute;os M</span>, <span class="author">Lind L</span>, <span class="author">Palmer LJ</span>, <span class="author">Hu FB</span>, <span class="author">Franks PW</span>, <span class="author">Ebrahim S</span>, <span class="author">Marmot M</span>, <span class="author">Kao WH</span>, <span class="author">Pramstaller PP</span>, <span class="author">Wright AF</span>, <span class="author">Stumvoll M</span>, <span class="author">Hamsten A</span>, <span class="author">Procardis Consortium</span>, <span class="author">Buchanan TA</span>, <span class="author">Valle TT</span>, <span class="author">Rotter JI</span>, <span class="author">Penninx BW</span>, <span class="author">Boomsma DI</span>, <span class="author">Cao A</span>, <span class="author">Scuteri A</span>, <span class="author">Schlessinger D</span>, <span class="author">Uda M</span>, <span class="author">Ruokonen A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Peltonen L</span>, <span class="author">Mooser V</span>, <span class="author">Sladek R</span>, <span class="author">MAGIC investigators</span>, <span class="author">GLGC Consortium</span>, <span class="author">Musunuru K</span>, <span class="author">Smith AV</span>, <span class="author">Edmondson AC</span>, <span class="author">Stylianou IM</span>, <span class="author">Koseki M</span>, <span class="author">Pirruccello JP</span>, <span class="author">Chasman DI</span>, <span class="author">Johansen CT</span>, <span class="author">Fouchier SW</span>, <span class="author">Peloso GM</span>, <span class="author">Barbalic M</span>, <span class="author">Ricketts SL</span>, <span class="author">Bis JC</span>, <span class="author">Feitosa MF</span>, <span class="author">Orho-Melander M</span>, <span class="author">Melander O</span>, <span class="author">Li X</span>, <span class="author">Li M</span>, <span class="author">Cho YS</span>, <span class="author">Go MJ</span>, <span class="author">Kim YJ</span>, <span class="author">Lee JY</span>, <span class="author">Park T</span>, <span class="author">Kim K</span>, <span class="author">Sim X</span>, <span class="author">Ong RT</span>, <span class="author">Croteau-Chonka DC</span>, <span class="author">Lange LA</span>, <span class="author">Smith JD</span>, <span class="author">Ziegler A</span>, <span class="author">Zhang W</span>, <span class="author">Zee RY</span>, <span class="author">Whitfield JB</span>, <span class="author">Thompson JR</span>, <span class="author">Surakka I</span>, <span class="author">Spector TD</span>, <span class="author">Smit JH</span>, <span class="author">Sinisalo J</span>, <span class="author">Scott J</span>, <span class="author">Saharinen J</span>, <span class="author">Sabatti C</span>, <span class="author">Rose LM</span>, <span class="author">Roberts R</span>, <span class="author">Rieder M</span>, <span class="author">Parker AN</span>, <span class="author">Pare G</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Nieminen MS</span>, <span class="author">Nickerson DA</span>, <span class="author">Montgomery GW</span>, <span class="author">McArdle W</span>, <span class="author">Masson D</span>, <span class="author">Martin NG</span>, <span class="author">Marroni F</span>, <span class="author">Lucas G</span>, <span class="author">Luben R</span>, <span class="author">Lokki ML</span>, <span class="author">Lettre G</span>, <span class="author">Launer LJ</span>, <span class="author">Lakatta EG</span>, <span class="author">Laaksonen R</span>, <span class="author">Kyvik KO</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Khaw KT</span>, <span class="author">Kaplan LM</span>, <span class="author">Johansson &Aring;</span>, <span class="author">Janssens AC</span>, <span class="author">Igl W</span>, <span class="author">Hovingh GK</span>, <span class="author">Hengstenberg C</span>, <span class="author">Havulinna AS</span>, <span class="author">Hastie ND</span>, <span class="author">Harris TB</span>, <span class="author">Haritunians T</span>, <span class="author">Hall AS</span>, <span class="author">Groop LC</span>, <span class="author">Gonzalez E</span>, <span class="author">Freimer NB</span>, <span class="author">Erdmann J</span>, <span class="author">Ejebe KG</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Dominiczak AF</span>, <span class="author">Demissie S</span>, <span class="author">Deloukas P</span>, <span class="author">de Faire U</span>, <span class="author">Crawford G</span>, <span class="author">Chen YD</span>, <span class="author">Caulfield MJ</span>, <span class="author">Boekholdt SM</span>, <span class="author">Assimes TL</span>, <span class="author">Quertermous T</span>, <span class="author">Seielstad M</span>, <span class="author">Wong TY</span>, <span class="author">Tai ES</span>, <span class="author">Feranil AB</span>, <span class="author">Kuzawa CW</span>, <span class="author">Taylor HA</span>, <span class="author">Gabriel SB</span>, <span class="author">Holm H</span>, <span class="author">Gudnason V</span>, <span class="author">Krauss RM</span>, <span class="author">Ordovas JM</span>, <span class="author">Munroe PB</span>, <span class="author">Kooner JS</span>, <span class="author">Tall AR</span>, <span class="author">Hegele RA</span>, <span class="author">Kastelein JJ</span>, <span class="author">Schadt EE</span>, <span class="author">Strachan DP</span>, <span class="author">Reilly MP</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Cupples LA</span>, <span class="author">Sandhu MS</span>, <span class="author">Ridker PM</span>, <span class="author">Rader DJ</span> and <span class="author">Kathiresan S</span></p>
<p class="affiliation">Department of Epidemiology, Biostatistics, and Occupational Health, Jewish General Hospital, Lady Davis Institute, McGill University, Montreal, Canada.</p><p class="abstract">Circulating levels of adiponectin, a hormone produced predominantly by adipocytes, are highly heritable and are inversely associated with type 2 diabetes mellitus (T2D) and other metabolic traits. We conducted a meta-analysis of genome-wide association studies in 39,883 individuals of European ancestry to identify genes associated with metabolic disease. We identified 8 novel loci associated with adiponectin levels and confirmed 2 previously reported loci (P&#x200A;=&#x200A;4.5&times;10(-8)-1.2&times;10(-43)). Using a novel method to combine data across ethnicities (N&#x200A;=&#x200A;4,232 African Americans, N&#x200A;=&#x200A;1,776 Asians, and N&#x200A;=&#x200A;29,347 Europeans), we identified two additional novel loci. Expression analyses of 436 human adipocyte samples revealed that mRNA levels of 18 genes at candidate regions were associated with adiponectin concentrations after accounting for multiple testing (p&lt;3&times;10(-4)). We next developed a multi-SNP genotypic risk score to test the association of adiponectin decreasing risk alleles on metabolic traits and diseases using consortia-level meta-analytic data. This risk score was associated with increased risk of T2D (p&#x200A;=&#x200A;4.3&times;10(-3), n&#x200A;=&#x200A;22,044), increased triglycerides (p&#x200A;=&#x200A;2.6&times;10(-14), n&#x200A;=&#x200A;93,440), increased waist-to-hip ratio (p&#x200A;=&#x200A;1.8&times;10(-5), n&#x200A;=&#x200A;77,167), increased glucose two hours post oral glucose tolerance testing (p&#x200A;=&#x200A;4.4&times;10(-3), n&#x200A;=&#x200A;15,234), increased fasting insulin (p&#x200A;=&#x200A;0.015, n&#x200A;=&#x200A;48,238), but with lower in HDL-cholesterol concentrations (p&#x200A;=&#x200A;4.5&times;10(-13), n&#x200A;=&#x200A;96,748) and decreased BMI (p&#x200A;=&#x200A;1.4&times;10(-4), n&#x200A;=&#x200A;121,335). These findings identify novel genetic determinants of adiponectin levels, which, taken together, influence risk of T2D and markers of insulin resistance.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">PG/08/094/26019</span>, <span class="grant">PG/09/002/26056</span>, <span class="grant">RG/07/008/23674</span>; Canadian Institutes of Health Research; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Department of Health: <span class="grant">DHCS/07/07/008</span>; FIC NIH HHS: <span class="grant">TW05596</span>; Medical Research Council: <span class="grant">G0100222</span>, <span class="grant">G0401527</span>, <span class="grant">G0600705</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0800582</span>, <span class="grant">G0801056</span>, <span class="grant">G0900339</span>, <span class="grant">G0901213</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_U137686857</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>, <span class="grant">RR-024156</span>, <span class="grant">RR20649</span>, <span class="grant">UL1 RR025008</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">HL085144</span>, <span class="grant">HL094555</span>, <span class="grant">HL105756</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01 HC-95159</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-65226</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-95160</span>, <span class="grant">N01-HC-95161</span>, <span class="grant">N01-HC-95162</span>, <span class="grant">N01-HC-95163</span>, <span class="grant">N01-HC-95164</span>, <span class="grant">N01-HC-95165</span>, <span class="grant">N01-HC-95166</span>, <span class="grant">N01-HC-95167</span>, <span class="grant">N01-HC-95168</span>, <span class="grant">N01-HC-95169</span>, <span class="grant">N01-HC-95170</span>, <span class="grant">N01-HC-95171</span>, <span class="grant">N01-HC-95172</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01-HL085251</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">RC2 HL102419</span>, <span class="grant">U01 HL080295</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>; NICHD NIH HHS: <span class="grant">R24 HD050924</span>; NIDDK NIH HHS: <span class="grant">1 R01 DK075787-01A1</span>, <span class="grant">DK063491</span>, <span class="grant">DK078150</span>, <span class="grant">DK56350</span>, <span class="grant">K24 DK080140</span>, <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK078150</span>, <span class="grant">R01DK056918</span>; NIEHS NIH HHS: <span class="grant">ES10126</span>, <span class="grant">P30 ES010126</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>; The Dunhill Medical Trust: <span class="grant">R69/0208</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">081682</span>, <span class="grant">090532</span>, <span class="grant">092731</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e1002607</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22479202">22479202</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3315470">3315470</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002607">10.1371/journal.pgen.1002607</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-Scale Identification of MicroRNA Targets in Murine Dgcr8-Deficient Embryonic Stem Cell Lines.</h4>
<p class="authors"><span class="author">Davis MP</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">van Dongen S</span>, <span class="author">Lu D</span>, <span class="author">Tate PH</span>, <span class="author">Bartonicek N</span>, <span class="author">Kutter C</span>, <span class="author">Liu P</span>, <span class="author">Skarnes WC</span>, <span class="author">Enright AJ</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p>
<p class="abstract">Small RNAs such as microRNAs play important roles in embryonic stem cell maintenance and differentiation. A broad range of microRNAs is expressed in embryonic stem cells while only a fraction of their targets have been identified. We have performed large-scale identification of embryonic stem cell microRNA targets using a murine embryonic stem cell line deficient in the expression of Dgcr8. These cells are heavily depleted for microRNAs, allowing us to reintroduce specific microRNA duplexes and identify refined target sets. We used deep sequencing of small RNAs, mRNA expression profiling and bioinformatics analysis of microRNA seed matches in 3' UTRs to identify target transcripts. Consequently, we have identified a network of microRNAs that converge on the regulation of several important cellular pathways. Additionally, our experiments have revealed a novel candidate for Dgcr8-independent microRNA genesis and highlighted the challenges currently facing miRNA annotation.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">8</span>;<span class="pagerange">e41762</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22912678" rel="external">22912678</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0041762" rel="external">10.1371/journal.pone.0041762</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Long-range DNA looping and gene expression analyses identify DEXI as an autoimmune disease candidate gene.</h4>
<p class="authors"><span class="author">Davison LJ</span>, <span class="author">Wallace C</span>, <span class="author">Cooper JD</span>, <span class="author">Cope NF</span>, <span class="author">Wilson NK</span>, <span class="author">Smyth DJ</span>, <span class="author">Howson JM</span>, <span class="author">Saleh N</span>, <span class="author">Al-Jeffery A</span>, <span class="author">Angus KL</span>, <span class="author">Stevens HE</span>, <span class="author">Nutland S</span>, <span class="author">Duley S</span>, <span class="author">Coulson RM</span>, <span class="author">Walker NM</span>, <span class="author">Burren OS</span>, <span class="author">Rice CM</span>, <span class="author">Cambien F</span>, <span class="author">Zeller T</span>, <span class="author">Munzel T</span>, <span class="author">Lackner K</span>, <span class="author">Blankenberg S</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">Fraser P</span>, <span class="author">Gottgens B</span>, <span class="author">Todd JA</span>, <span class="author">Attwood T</span>, <span class="author">Belz S</span>, <span class="author">Braund P</span>, <span class="author">Cambien F</span>, <span class="author">Cooper J</span>, <span class="author">Crisp-Hihn A</span>, <span class="author">Diemert P</span>, <span class="author">Deloukas P</span>, <span class="author">Foad N</span>, <span class="author">Erdmann J</span>, <span class="author">Goodall AH</span>, <span class="author">Gracey J</span>, <span class="author">Gray E</span>, <span class="author">Gwilliams R</span>, <span class="author">Heimerl S</span>, <span class="author">Hengstenberg C</span>, <span class="author">Jolley J</span>, <span class="author">Krishnan U</span>, <span class="author">Lloyd-Jones H</span>, <span class="author">Lugauer I</span>, <span class="author">Lundmark P</span>, <span class="author">Maouche S</span>, <span class="author">Moore JS</span>, <span class="author">Muir D</span>, <span class="author">Murray E</span>, <span class="author">Nelson CP</span>, <span class="author">Neudert J</span>, <span class="author">Niblett D</span>, <span class="author">O&#39;Leary K</span>, <span class="author">Ouwehand WH</span>, <span class="author">Pollard H</span>, <span class="author">Rankin A</span>, <span class="author">Rice CM</span>, <span class="author">Sager H</span>, <span class="author">Samani NJ</span>, <span class="author">Sambrook J</span>, <span class="author">Schmitz G</span>, <span class="author">Scholz M</span>, <span class="author">Schroeder L</span>, <span class="author">Schunkert H</span>, <span class="author">Syvannen AC</span>, <span class="author">Tennstedt S</span> and <span class="author">Wallace C</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, NIHR Biomedical Research Centre, University of Cambridge, Cambridge, UK. <a href="mailto:%6C%75%63%79.%64%61%76%69%73%6F%6E%40%63%69%6D%72.%63%61%6D.%61%63.%75%6B">&#108;&#117;&#99;&#121;&#46;&#100;&#97;&#118;&#105;&#115;&#111;&#110;&#64;&#99;&#105;&#109;&#114;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The chromosome 16p13 region has been associated with several autoimmune diseases, including type 1 diabetes (T1D) and multiple sclerosis (MS). CLEC16A has been reported as the most likely candidate gene in the region, since it contains the most disease-associated single-nucleotide polymorphisms (SNPs), as well as an imunoreceptor tyrosine-based activation motif. However, here we report that intron 19 of CLEC16A, containing the most autoimmune disease-associated SNPs, appears to behave as a regulatory sequence, affecting the expression of a neighbouring gene, DEXI. The CLEC16A alleles that are protective from T1D and MS are associated with increased expression of DEXI, and no other genes in the region, in two independent monocyte gene expression data sets. Critically, using chromosome conformation capture (3C), we identified physical proximity between the DEXI promoter region and intron 19 of CLEC16A, separated by a loop of &gt;150 kb. In reciprocal experiments, a 20 kb fragment of intron 19 of CLEC16A, containing SNPs associated with T1D and MS, as well as with DEXI expression, interacted with the promotor region of DEXI but not with candidate DNA fragments containing other potential causal genes in the region, including CLEC16A. Intron 19 of CLEC16A is highly enriched for transcription-factor-binding events and markers associated with enhancer activity. Taken together, these data indicate that although the causal variants in the 16p13 region lie within CLEC16A, DEXI is an unappreciated autoimmune disease candidate gene, and illustrate the power of the 3C approach in progressing from genome-wide association studies results to candidate causal genes.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">076113</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">079895</span>, <span class="grant">082549/Z/07/Z</span>, <span class="grant">089989/Z/09/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">2</span>;<span class="pagerange">322-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21989056" rel="external">21989056</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3276289" rel="external">3276289</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr468" rel="external">10.1093/hmg/ddr468</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diagnostic interpretation of array data using public databases and internet sources.</h4>
<p class="authors"><span class="author">de Leeuw N</span>, <span class="author">Dijkhuizen T</span>, <span class="author">Hehir-Kwa JY</span>, <span class="author">Carter NP</span>, <span class="author">Feuk L</span>, <span class="author">Firth HV</span>, <span class="author">Kuhn RM</span>, <span class="author">Ledbetter DH</span>, <span class="author">Martin CL</span>, <span class="author">van Ravenswaaij-Arts CM</span>, <span class="author">Scherer SW</span>, <span class="author">Shams S</span>, <span class="author">Van Vooren S</span>, <span class="author">Sijmons R</span>, <span class="author">Swertz M</span> and <span class="author">Hastings R</span></p>
<p class="affiliation">Department of Human Genetics, Radboud University Nijmegen Medical Centre, Nijmegen, the Netherlands. <a href="mailto:%4E.%64%65%4C%65%65%75%77%40%61%6E%74%72%67.%75%6D%63%6E.%6E%6C">&#78;&#46;&#100;&#101;&#76;&#101;&#101;&#117;&#119;&#64;&#97;&#110;&#116;&#114;&#103;&#46;&#117;&#109;&#99;&#110;&#46;&#110;&#108;</a>.</p><p class="abstract">The range of commercially available array platforms and analysis software packages is expanding and their utility is improving, making reliable detection of copy number variation (CNV) relatively straightforward. Reliable interpretation of CNV data, however, is often difficult and requires expertise. With our knowledge of the human genome growing rapidly, applications for array testing continuously broadening, and the resolution of CNV detection increasing, this leads to great complexity in interpreting what can be daunting data. Correct CNV interpretation and optimal use of the genotype information provided by SNP probes on an array depends largely on knowledge present in various resources. In addition to the availability of host laboratories&#39; own datasets and national registries, there are several public databases and Internet resources with genotype and phenotype information that can be used for array data interpretation. With so many resources now available, it is important to know which are fit-for-purpose in a diagnostic setting. We summarise the characteristics of the most commonly used Internet databases and resources, and propose a general data interpretation strategy that can be used for comparative hybridisation, comparative intensity and genotype-based array data.</p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22334422">22334422</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.22049">10.1002/humu.22049</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies for personality.</h4>
<p class="authors"><span class="author">de Moor MH</span>, <span class="author">Costa PT</span>, <span class="author">Terracciano A</span>, <span class="author">Krueger RF</span>, <span class="author">de Geus EJ</span>, <span class="author">Toshiko T</span>, <span class="author">Penninx BW</span>, <span class="author">Esko T</span>, <span class="author">Madden PA</span>, <span class="author">Derringer J</span>, <span class="author">Amin N</span>, <span class="author">Willemsen G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Distel MA</span>, <span class="author">Uda M</span>, <span class="author">Sanna S</span>, <span class="author">Spinhoven P</span>, <span class="author">Hartman CA</span>, <span class="author">Sullivan P</span>, <span class="author">Realo A</span>, <span class="author">Allik J</span>, <span class="author">Heath AC</span>, <span class="author">Pergadia ML</span>, <span class="author">Agrawal A</span>, <span class="author">Lin P</span>, <span class="author">Grucza R</span>, <span class="author">Nutile T</span>, <span class="author">Ciullo M</span>, <span class="author">Rujescu D</span>, <span class="author">Giegling I</span>, <span class="author">Konte B</span>, <span class="author">Widen E</span>, <span class="author">Cousminer DL</span>, <span class="author">Eriksson JG</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen L</span>, <span class="author">Luciano M</span>, <span class="author">Tenesa A</span>, <span class="author">Davies G</span>, <span class="author">Lopez LM</span>, <span class="author">Hansell NK</span>, <span class="author">Medland SE</span>, <span class="author">Ferrucci L</span>, <span class="author">Schlessinger D</span>, <span class="author">Montgomery GW</span>, <span class="author">Wright MJ</span>, <span class="author">Aulchenko YS</span>, <span class="author">Janssens AC</span>, <span class="author">Oostra BA</span>, <span class="author">Metspalu A</span>, <span class="author">Abecasis GR</span>, <span class="author">Deary IJ</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Bierut LJ</span>, <span class="author">Martin NG</span>, <span class="author">van Duijn CM</span> and <span class="author">Boomsma DI</span></p>
<p class="affiliation">Department of Biological Psychology, VU University Amsterdam, Amsterdam, The Netherlands. <a href="mailto:%6D%68%6D.%64%65.%6D%6F%6F%72%40%70%73%79.%76%75.%6E%6C">&#109;&#104;&#109;&#46;&#100;&#101;&#46;&#109;&#111;&#111;&#114;&#64;&#112;&#115;&#121;&#46;&#118;&#117;&#46;&#110;&#108;</a></p><p class="abstract">Personality can be thought of as a set of characteristics that influence people&#39;s thoughts, feelings and behavior across a variety of settings. Variation in personality is predictive of many outcomes in life, including mental health. Here we report on a meta-analysis of genome-wide association (GWA) data for personality in 10 discovery samples (17,375 adults) and five in silico replication samples (3294 adults). All participants were of European ancestry. Personality scores for Neuroticism, Extraversion, Openness to Experience, Agreeableness and Conscientiousness were based on the NEO Five-Factor Inventory. Genotype data of &asymp; 2.4M single-nucleotide polymorphisms (SNPs; directly typed and imputed using HapMap data) were available. In the discovery samples, classical association analyses were performed under an additive model followed by meta-analysis using the weighted inverse variance method. Results showed genome-wide significance for Openness to Experience near the RASA1 gene on 5q14.3 (rs1477268 and rs2032794, P=2.8 &times; 10(-8) and 3.1 &times; 10(-8)) and for Conscientiousness in the brain-expressed KATNAL2 gene on 18q21.1 (rs2576037, P=4.9 &times; 10(-8)). We further conducted a gene-based test that confirmed the association of KATNAL2 to Conscientiousness. In silico replication did not, however, show significant associations of the top SNPs with Openness and Conscientiousness, although the direction of effect of the KATNAL2 SNP on Conscientiousness was consistent in all replication samples. Larger scale GWA studies and alternative approaches are required for confirmation of KATNAL2 as a novel gene affecting Conscientiousness.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; NCI NIH HHS: <span class="grant">CA89392</span>, <span class="grant">P01 CA089392</span>, <span class="grant">P01 CA089392-08</span>; NHGRI NIH HHS: <span class="grant">U01 HG004422</span>, <span class="grant">U01 HG004422-02</span>, <span class="grant">U01 HG004446</span>, <span class="grant">U01HG004438</span>; NIA NIH HHS: <span class="grant">N01-AG-1-2109</span>, <span class="grant">Z99 AG999999</span>, <span class="grant">ZIA AG000180-25</span>, <span class="grant">ZIA AG000180-26</span>, <span class="grant">ZIA AG000196-03</span>, <span class="grant">ZIA AG000196-04</span>, <span class="grant">ZIA AG000197-03</span>, <span class="grant">ZIA AG000197-04</span>; NIAAA NIH HHS: <span class="grant">AA07580</span>, <span class="grant">AA07728</span>, <span class="grant">AA11998</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">K05 AA017688-04</span>, <span class="grant">U10 AA008401</span>, <span class="grant">U10AA008401</span>; NIDA NIH HHS: <span class="grant">DA019951</span>, <span class="grant">DA12854</span>, <span class="grant">R01 DA012854-10</span>, <span class="grant">R01 DA013423</span>, <span class="grant">R01 DA013423-05</span>, <span class="grant">R01 DA019963-01A2</span>, <span class="grant">R01 DA019963-02</span>, <span class="grant">R01 DA019963-03</span>; NIMH NIH HHS: <span class="grant">MH081802</span>, <span class="grant">R01 MH059160</span>; Wellcome Trust: <span class="grant">089062/Z/09/Z</span>, <span class="grant">89061/Z/09/Z</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2012</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">337-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21173776">21173776</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2010.128">10.1038/mp.2010.128</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Investigating the potential for ethnic group harm in collaborative genomics research in Africa: Is ethnic stigmatisation likely?</h4>
<p class="authors"><span class="author">de Vries J</span>, <span class="author">Jallow M</span>, <span class="author">Williams TN</span>, <span class="author">Kwiatkowski D</span>, <span class="author">Parker M</span> and <span class="author">Fitzpatrick R</span></p>
<p class="affiliation">The Ethox Centre, Department of Public Health, University of Oxford, UK; Division of Human Genetics, Faculty of Health Sciences, University of Cape Town, South Africa.</p>
<p class="abstract">A common assumption in genomics research is that the use of ethnic categories has the potential to lead to ethnic stigmatisation - particularly when the research is done on minority populations. Yet few empirical studies have sought to investigate the relation between genomics and stigma, and fewer still with a focus on Africa. In this paper, we investigate the potential for genomics research to lead to harms to ethnic groups. We carried out 49 semi-structured, open-ended interviews with stakeholders in a current medical genomics research project in Africa, MalariaGEN. Interviews were conducted with MalariaGEN researchers, fieldworkers, members of three ethics committees who reviewed MalariaGEN project proposals, and with members of the two funding bodies providing support to the MalariaGEN project. Interviews were conducted in Kenya, The Gambia and the UK between June 2008 and October 2009. They covered a range of aspects relating to the use of ethnicity in the genomics project, including views on adverse effects of the inclusion of ethnicity in such research. Drawing on the empirical data, we argue that the risk of harm to ethnic groups is likely to be more acute in specific types of genomics research. We develop a typology of research questions and projects that carry a greater risk of harm to the populations included in genomics research. We conclude that the potential of generating harm to ethnic groups in genomics research is present if research includes populations that are already stigmatised or discriminated against, or where the research investigates questions with particular normative implications. We identify a clear need for genomics researchers to take account of the social context of the work they are proposing to do, including understanding the local realities and relations between ethnic groups, and whether diseases are already stigmatised.</p>
<p class="publication"><span class="title">Social science &amp;amp; medicine (1982)</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22749442" rel="external">22749442</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.socscimed.2012.05.020" rel="external">10.1016/j.socscimed.2012.05.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic restructuring in the Tasmanian devil facial tumour: chromosome painting and gene mapping provide clues to evolution of a transmissible tumour.</h4>
<p class="authors"><span class="author">Deakin JE</span>, <span class="author">Bender HS</span>, <span class="author">Pearse AM</span>, <span class="author">Rens W</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Cheng Y</span>, <span class="author">Morris K</span>, <span class="author">Taylor R</span>, <span class="author">Stuart A</span>, <span class="author">Belov K</span>, <span class="author">Amemiya CT</span>, <span class="author">Murchison EP</span>, <span class="author">Papenfuss AT</span> and <span class="author">Graves JA</span></p>
<p class="affiliation">Research School of Biology, The Australian National University, Canberra, Australia. <a href="mailto:%6A%61%6E%69%6E%65.%64%65%61%6B%69%6E%40%61%6E%75.%65%64%75.%61%75">&#106;&#97;&#110;&#105;&#110;&#101;&#46;&#100;&#101;&#97;&#107;&#105;&#110;&#64;&#97;&#110;&#117;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Devil facial tumour disease (DFTD) is a fatal, transmissible malignancy that threatens the world&#39;s largest marsupial carnivore, the Tasmanian devil, with extinction. First recognised in 1996, DFTD has had a catastrophic effect on wild devil numbers, and intense research efforts to understand and contain the disease have since demonstrated that the tumour is a clonal cell line transmitted by allograft. We used chromosome painting and gene mapping to deconstruct the DFTD karyotype and determine the chromosome and gene rearrangements involved in carcinogenesis. Chromosome painting on three different DFTD tumour strains determined the origins of marker chromosomes and provided a general overview of the rearrangement in DFTD karyotypes. Mapping of 105 BAC clones by fluorescence in situ hybridisation provided a finer level of resolution of genome rearrangements in DFTD strains. Our findings demonstrate that only limited regions of the genome, mainly chromosomes 1 and X, are rearranged in DFTD. Regions rearranged in DFTD are also highly rearranged between different marsupials. Differences between strains are limited, reflecting the unusually stable nature of DFTD. Finally, our detailed maps of both the devil and tumour karyotypes provide a physical framework for future genomic investigations into DFTD.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">e1002483</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22359511" rel="external">22359511</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3280961" rel="external">3280961</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002483" rel="external">10.1371/journal.pgen.1002483</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Clostridium difficile spo0A gene is a persistence and transmission factor.</h4>
<p class="authors"><span class="author">Deakin LJ</span>, <span class="author">Clare S</span>, <span class="author">Fagan RP</span>, <span class="author">Dawson LF</span>, <span class="author">Pickard DJ</span>, <span class="author">West MR</span>, <span class="author">Wren BW</span>, <span class="author">Fairweather NF</span>, <span class="author">Dougan G</span> and <span class="author">Lawley TD</span></p>
<p class="affiliation">Microbial Pathogenesis Laboratory, Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">Clostridium difficile is a major cause of chronic antibiotic-associated diarrhea and a significant health care-associated pathogen that forms highly resistant and infectious spores. Spo0A is a highly conserved transcriptional regulator that plays a key role in initiating sporulation in Bacillus and Clostridium species. Here, we use a murine model to study the role of the C. difficile spo0A gene during infection and transmission. We demonstrate that C. difficile spo0A mutant derivatives can cause intestinal disease but are unable to persist within and effectively transmit between mice. Thus, the C. difficile Spo0A protein plays a key role in persistent infection, including recurrence and host-to-host transmission in mice.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800170</span>, <span class="grant">G0901743</span>; Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT086418MA</span></p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2012</span>;<span class="volume">80</span>;<span class="number">8</span>;<span class="pagerange">2704-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22615253">22615253</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3434595">3434595</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00147-12">10.1128/IAI.00147-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular mechanisms of drug resistance in natural Leishmania populations vary with genetic background.</h4>
<p class="authors"><span class="author">Decuypere S</span>, <span class="author">Vanaerschot M</span>, <span class="author">Brunker K</span>, <span class="author">Imamura H</span>, <span class="author">M&uuml;ller S</span>, <span class="author">Khanal B</span>, <span class="author">Rijal S</span>, <span class="author">Dujardin JC</span> and <span class="author">Coombs GH</span></p>
<p class="affiliation">Department of Biomedical Sciences, Institute of Tropical Medicine, Antwerp, Belgium.</p><p class="abstract">The evolution of drug-resistance in pathogens is a major global health threat. Elucidating the molecular basis of pathogen drug-resistance has been the focus of many studies but rarely is it known whether a drug-resistance mechanism identified is universal for the studied pathogen; it has seldom been clarified whether drug-resistance mechanisms vary with the pathogen&#39;s genotype. Nevertheless this is of critical importance in gaining an understanding of the complexity of this global threat and in underpinning epidemiological surveillance of pathogen drug resistance in the field. This study aimed to assess the molecular and phenotypic heterogeneity that emerges in natural parasite populations under drug treatment pressure. We studied lines of the protozoan parasite Leishmania (L.) donovani with differential susceptibility to antimonial drugs; the lines being derived from clinical isolates belonging to two distinct genetic populations that circulate in the leishmaniasis endemic region of Nepal. Parasite pathways known to be affected by antimonial drugs were characterised on five experimental levels in the lines of the two populations. Characterisation of DNA sequence, gene expression, protein expression and thiol levels revealed a number of molecular features that mark antimonial-resistant parasites in only one of the two populations studied. A final series of in vitro stress phenotyping experiments confirmed this heterogeneity amongst drug-resistant parasites from the two populations. These data provide evidence that the molecular changes associated with antimonial-resistance in natural Leishmania populations depend on the genetic background of the Leishmania population, which has resulted in a divergent set of resistance markers in the Leishmania populations. This heterogeneity of parasite adaptations provides severe challenges for the control of drug resistance in the field and the design of molecular surveillance tools for widespread applicability.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085349</span>, <span class="grant">WT061173MA-SM</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">e1514</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22389733" rel="external">22389733</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3289598" rel="external">3289598</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001514" rel="external">10.1371/journal.pntd.0001514</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Angiopoietin-2 Is a Direct Transcriptional Target of TAL1, LYL1 and LMO2 in Endothelial Cells.</h4>
<p class="authors"><span class="author">Deleuze V</span>, <span class="author">El-Hajj R</span>, <span class="author">Chalhoub E</span>, <span class="author">Dohet C</span>, <span class="author">Pinet V</span>, <span class="author">Couttet P</span> and <span class="author">Mathieu D</span></p>
<p class="affiliation">Institut de Génétique Moléculaire de Montpellier UMR 5535 CNRS, Montpellier, France.</p>
<p class="abstract">The two related basic helix-loop-helix, TAL1 and LYL1, and their cofactor LIM-only-2 protein (LMO2) are present in blood and endothelial cells. While their crucial role in early hematopoiesis is well established, their function in endothelial cells and especially in angiogenesis is less understood. Here, we identified ANGIOPOIETIN-2 (ANG-2), which encodes a major regulator of angiogenesis, as a direct transcriptional target of TAL1, LYL1 and LMO2. Knockdown of any of the three transcription factors in human blood and lymphatic endothelial cells caused ANG-2 mRNA and protein down-regulation. Transient transfections showed that the full activity of the ANG-2 promoter required the integrity of a highly conserved Ebox-GATA composite element. Accordingly, chromatin immunoprecipitation assays demonstrated that TAL1, LYL1, LMO2 and GATA2 occupied this region of ANG-2 promoter in human endothelial cells. Furthermore, we showed that LMO2 played a central role in assembling TAL1-E47, LYL1-LYL1 or/and LYL1-TAL1 dimers with GATA2. The resulting complexes were able to activate endogenous ANG-2 expression in endothelial cells as well as in non-endothelial cells. Finally, we showed that ANG-2 gene activation during angiogenesis concurred with the up-regulation of TAL1 and LMO2. Altogether, we identified ANG-2 as a bona fide target gene of LMO2-complexes with TAL1 and/or LYL1, highlighting a new function of the three hematopoietic factors in the endothelial lineage.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e40484</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22792348" rel="external">22792348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3391236" rel="external">3391236</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0040484" rel="external">10.1371/journal.pone.0040484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies novel loci associated with circulating phospho- and sphingolipid concentrations.</h4>
<p class="authors"><span class="author">Demirkan A</span>, <span class="author">van Duijn CM</span>, <span class="author">Ugocsai P</span>, <span class="author">Isaacs A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Liebisch G</span>, <span class="author">Wilson JF</span>, <span class="author">Johansson &Aring;</span>, <span class="author">Rudan I</span>, <span class="author">Aulchenko YS</span>, <span class="author">Kirichenko AV</span>, <span class="author">Janssens AC</span>, <span class="author">Jansen RC</span>, <span class="author">Gnewuch C</span>, <span class="author">Domingues FS</span>, <span class="author">Pattaro C</span>, <span class="author">Wild SH</span>, <span class="author">Jonasson I</span>, <span class="author">Polasek O</span>, <span class="author">Zorkoltseva IV</span>, <span class="author">Hofman A</span>, <span class="author">Karssen LC</span>, <span class="author">Struchalin M</span>, <span class="author">Floyd J</span>, <span class="author">Igl W</span>, <span class="author">Biloglav Z</span>, <span class="author">Broer L</span>, <span class="author">Pfeufer A</span>, <span class="author">Pichler I</span>, <span class="author">Campbell S</span>, <span class="author">Zaboli G</span>, <span class="author">Kolcic I</span>, <span class="author">Rivadeneira F</span>, <span class="author">Huffman J</span>, <span class="author">Hastie ND</span>, <span class="author">Uitterlinden A</span>, <span class="author">Franke L</span>, <span class="author">Franklin CS</span>, <span class="author">Vitart V</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Nelson CP</span>, <span class="author">Preuss M</span>, <span class="author">CARDIoGRAM Consortium</span>, <span class="author">Bis JC</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Franceschini N</span>, <span class="author">CHARGE Consortium</span>, <span class="author">Witteman JC</span>, <span class="author">Axenovich T</span>, <span class="author">Oostra BA</span>, <span class="author">Meitinger T</span>, <span class="author">Hicks AA</span>, <span class="author">Hayward C</span>, <span class="author">Wright AF</span>, <span class="author">Gyllensten U</span>, <span class="author">Campbell H</span>, <span class="author">Schmitz G</span> and <span class="author">EUROSPAN consortium</span></p>
<p class="affiliation">Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">Phospho- and sphingolipids are crucial cellular and intracellular compounds. These lipids are required for active transport, a number of enzymatic processes, membrane formation, and cell signalling. Disruption of their metabolism leads to several diseases, with diverse neurological, psychiatric, and metabolic consequences. A large number of phospholipid and sphingolipid species can be detected and measured in human plasma. We conducted a meta-analysis of five European family-based genome-wide association studies (N&#x200A;=&#x200A;4034) on plasma levels of 24 sphingomyelins (SPM), 9 ceramides (CER), 57 phosphatidylcholines (PC), 20 lysophosphatidylcholines (LPC), 27 phosphatidylethanolamines (PE), and 16 PE-based plasmalogens (PLPE), as well as their proportions in each major class. This effort yielded 25 genome-wide significant loci for phospholipids (smallest P-value&#x200A;=&#x200A;9.88&times;10(-204)) and 10 loci for sphingolipids (smallest P-value&#x200A;=&#x200A;3.10&times;10(-57)). After a correction for multiple comparisons (P-value&lt;2.2&times;10(-9)), we observed four novel loci significantly associated with phospholipids (PAQR9, AGPAT1, PKD2L1, PDXDC1) and two with sphingolipids (PLD2 and APOE) explaining up to 3.1% of the variance. Further analysis of the top findings with respect to within class molar proportions uncovered three additional loci for phospholipids (PNLIPRP2, PCDH20, and ABDH3) suggesting their involvement in either fatty acid elongation/saturation processes or fatty acid specific turnover mechanisms. Among those, 14 loci (KCNH7, AGPAT1, PNLIPRP2, SYT9, FADS1-2-3, DLG2, APOA1, ELOVL2, CDK17, LIPC, PDXDC1, PLD2, LASS4, and APOE) mapped into the glycerophospholipid and 12 loci (ILKAP, ITGA9, AGPAT1, FADS1-2-3, APOA1, PCDH20, LIPC, PDXDC1, SGPP1, APOE, LASS4, and PLD2) to the sphingolipid pathways. In large meta-analyses, associations between FADS1-2-3 and carotid intima media thickness, AGPAT1 and type 2 diabetes, and APOA1 and coronary artery disease were observed. In conclusion, our study identified nine novel phospho- and sphingolipid loci, substantially increasing our knowledge of the genetic basis for these traits.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U127561128</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NHLBI NIH HHS: <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL105756</span>; NIDDK NIH HHS: <span class="grant">P30 DK063491</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">e1002490</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22359512">22359512</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3280968">3280968</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002490">10.1371/journal.pgen.1002490</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression.</h4>
<p class="authors"><span class="author">Derrien T</span>, <span class="author">Johnson R</span>, <span class="author">Bussotti G</span>, <span class="author">Tanzer A</span>, <span class="author">Djebali S</span>, <span class="author">Tilgner H</span>, <span class="author">Guernec G</span>, <span class="author">Martin D</span>, <span class="author">Merkel A</span>, <span class="author">Knowles DG</span>, <span class="author">Lagarde J</span>, <span class="author">Veeravalli L</span>, <span class="author">Ruan X</span>, <span class="author">Ruan Y</span>, <span class="author">Lassmann T</span>, <span class="author">Carninci P</span>, <span class="author">Brown JB</span>, <span class="author">Lipovich L</span>, <span class="author">Gonzalez JM</span>, <span class="author">Thomas M</span>, <span class="author">Davis CA</span>, <span class="author">Shiekhattar R</span>, <span class="author">Gingeras TR</span>, <span class="author">Hubbard TJ</span>, <span class="author">Notredame C</span>, <span class="author">Harrow J</span> and <span class="author">Guig&oacute; R</span></p>
<p class="affiliation">Bioinformatics and Genomics, Centre for Genomic Regulation and UPF, 08003 Barcelona, Catalonia, Spain.</p><p class="abstract">The human genome contains many thousands of long noncoding RNAs (lncRNAs). While several studies have demonstrated compelling biological and disease roles for individual examples, analytical and experimental approaches to investigate these genes have been hampered by the lack of comprehensive lncRNA annotation. Here, we present and analyze the most complete human lncRNA annotation to date, produced by the GENCODE consortium within the framework of the ENCODE project and comprising 9277 manually annotated genes producing 14,880 transcripts. Our analyses indicate that lncRNAs are generated through pathways similar to that of protein-coding genes, with similar histone-modification profiles, splicing signals, and exon/intron lengths. In contrast to protein-coding genes, however, lncRNAs display a striking bias toward two-exon transcripts, they are predominantly localized in the chromatin and nucleus, and a fraction appear to be preferentially processed into small RNAs. They are under stronger selective pressure than neutrally evolving sequences-particularly in their promoter regions, which display levels of selection comparable to protein-coding genes. Importantly, about one-third seem to have arisen within the primate lineage. Comprehensive analysis of their expression in multiple human organs and brain regions shows that lncRNAs are generally lower expressed than protein-coding genes, and display more tissue-specific expression patterns, with a large fraction of tissue-specific lncRNAs expressed in the brain. Expression correlation analysis indicates that lncRNAs show particularly striking positive correlation with the expression of antisense coding genes. This GENCODE annotation represents a valuable resource for future studies of lncRNAs.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1U54HG004555-01</span>, <span class="grant">1U54HG004557-01</span>, <span class="grant">K99 HG006698</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">9</span>;<span class="pagerange">1775-89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955988">22955988</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3431493">3431493</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.132159.111">10.1101/gr.132159.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation sequencing in breast cancer: first take home messages.</h4>
<p class="authors"><span class="author">Desmedt C</span>, <span class="author">Voet T</span>, <span class="author">Sotiriou C</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">aBreast Cancer Translational Laboratory, Université Libre de Bruxelles, Jules Bordet Institute, Brussels bDepartment of Human Genetics, University of Leuven, Leuven, Belgium cWellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK.</p>
<p class="abstract">PURPOSE OF REVIEW: We are currently on the threshold of a revolution in breast cancer research, thanks to the emergence of novel technologies based on next-generation sequencing (NGS). In this review, we will describe the different sequencing technologies and platforms, and summarize the main findings from the latest sequencing articles in breast cancer. RECENT FINDINGS: Firstly, the sequencing of a few hundreds of breast tumors has revealed new cancer genes. Although these were not frequently mutated, mutated genes from different patients could be grouped into the deregulation of similar pathways. Secondly, NGS allowed further exploration of intratumor heterogeneity and revealed that although subclonal mutations were present in all tumors, there was always a dominant clone, which comprised at least 50% of the tumor cells. Finally, tumor-specific DNA rearrangements could be detected in the patient's plasma, suggesting that NGS could be used to personalize the monitoring of the disease. SUMMARY: The application of NGS to breast cancer has been associated with tremendous advances and promises for increasing the understanding of the disease. However, there still remain many unanswered questions, such as the role of structural changes of tumor genomes in cancer progression and treatment response/resistance.</p>
<p class="publication"><span class="title">Current opinion in oncology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23014189" rel="external">23014189</a>; DOI: <a href="http://dx.doi.org/10.1097%2FCCO.0b013e328359554e" rel="external">10.1097/CCO.0b013e328359554e</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic polymorphisms associated with anti-malarial antibody levels in a low and unstable malaria transmission area in southern Sri Lanka.</h4>
<p class="authors"><span class="author">Dewasurendra RL</span>, <span class="author">Suriyaphol P</span>, <span class="author">Fernando SD</span>, <span class="author">Carter R</span>, <span class="author">Rockett K</span>, <span class="author">Corran P</span>, <span class="author">Kwiatkowski D</span>, <span class="author">Karunaweera ND</span> and <span class="author">7 MalariaGEN Consortium</span></p>
<p class="affiliation">Department of Parasitology, Faculty of Medicine, University of Colombo, Colombo, Sri Lanka.</p>
<p class="abstract">Background: The incidence of malaria in Sri Lanka has significantly declined in recent years. Similar trends were seen in Kataragama, a known malaria endemic location within the southern province of the country, over the past five years. This is a descriptive study of anti-malarial antibody levels and selected host genetic mutations in residents of Kataragama, under low malaria transmission conditions.

Methods: Sera were collected from 1,011 individuals residing in Kataragama and anti-malarial antibodies and total IgE levels were measured by a standardized ELISA technique. Host DNA was extracted and used for genotyping of selected SNPs in known genes associated with malaria. The antibody levels were analysed in relation to the past history of malaria (during past 10â€‰years), age, sex, the location of residence within Kataragama and selected host genetic markers.

Results: A significant increase in antibodies against Plasmodium falciparum antigens AMA1, MSP2, NANP and Plasmodium vivax antigen MSP1 in individuals with past history of malaria were observed when compared to those who did not. A marked increase of anti-MSP1(Pf) and anti-AMA1(Pv) was also evident in individuals between 45-59â€‰years (when compared to other age groups). Allele frequencies for two SNPs in genes that code for IL-13 and TRIM-5 were found to be significantly different between those who have experienced one or more malaria attacks within past 10â€‰years and those who did not. When antibody levels were classified into a low-high binary trait, significant associations were found with four SNPs for anti-AMA1(Pf); two SNPs for anti-MSP1(Pf); eight SNPs for anti-NANP(Pf); three SNPs for anti-AMA1(Pv); seven SNPs for anti-MSP1(Pv); and nine SNPs for total IgE. Eleven of these SNPs with significant associations with anti-malarial antibody levels were found to be non-synonymous.

Conclusions: Evidence is suggestive of an age-acquired immunity in this study population in spite of low malaria transmission levels. Several SNPs were in linkage disequilibrium and had a significant association with elevated antibody levels, suggesting that these host genetic mutations might have an individual or collective effect on inducing or/and maintaining high anti-malarial antibody levels.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>; Wellcome Trust: <span class="grant">075491/Z04</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">090770/Z/09/Z</span>, <span class="grant">WT077383/Z/05/Z</span></p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="pagerange">281</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22905743">22905743</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3459805">3459805</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-11-281">10.1186/1475-2875-11-281</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Landscape of transcription in human cells.</h4>
<p class="authors"><span class="author">Djebali S</span>, <span class="author">Davis CA</span>, <span class="author">Merkel A</span>, <span class="author">Dobin A</span>, <span class="author">Lassmann T</span>, <span class="author">Mortazavi A</span>, <span class="author">Tanzer A</span>, <span class="author">Lagarde J</span>, <span class="author">Lin W</span>, <span class="author">Schlesinger F</span>, <span class="author">Xue C</span>, <span class="author">Marinov GK</span>, <span class="author">Khatun J</span>, <span class="author">Williams BA</span>, <span class="author">Zaleski C</span>, <span class="author">Rozowsky J</span>, <span class="author">R&ouml;der M</span>, <span class="author">Kokocinski F</span>, <span class="author">Abdelhamid RF</span>, <span class="author">Alioto T</span>, <span class="author">Antoshechkin I</span>, <span class="author">Baer MT</span>, <span class="author">Bar NS</span>, <span class="author">Batut P</span>, <span class="author">Bell K</span>, <span class="author">Bell I</span>, <span class="author">Chakrabortty S</span>, <span class="author">Chen X</span>, <span class="author">Chrast J</span>, <span class="author">Curado J</span>, <span class="author">Derrien T</span>, <span class="author">Drenkow J</span>, <span class="author">Dumais E</span>, <span class="author">Dumais J</span>, <span class="author">Duttagupta R</span>, <span class="author">Falconnet E</span>, <span class="author">Fastuca M</span>, <span class="author">Fejes-Toth K</span>, <span class="author">Ferreira P</span>, <span class="author">Foissac S</span>, <span class="author">Fullwood MJ</span>, <span class="author">Gao H</span>, <span class="author">Gonzalez D</span>, <span class="author">Gordon A</span>, <span class="author">Gunawardena H</span>, <span class="author">Howald C</span>, <span class="author">Jha S</span>, <span class="author">Johnson R</span>, <span class="author">Kapranov P</span>, <span class="author">King B</span>, <span class="author">Kingswood C</span>, <span class="author">Luo OJ</span>, <span class="author">Park E</span>, <span class="author">Persaud K</span>, <span class="author">Preall JB</span>, <span class="author">Ribeca P</span>, <span class="author">Risk B</span>, <span class="author">Robyr D</span>, <span class="author">Sammeth M</span>, <span class="author">Schaffer L</span>, <span class="author">See LH</span>, <span class="author">Shahab A</span>, <span class="author">Skancke J</span>, <span class="author">Suzuki AM</span>, <span class="author">Takahashi H</span>, <span class="author">Tilgner H</span>, <span class="author">Trout D</span>, <span class="author">Walters N</span>, <span class="author">Wang H</span>, <span class="author">Wrobel J</span>, <span class="author">Yu Y</span>, <span class="author">Ruan X</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Harrow J</span>, <span class="author">Gerstein M</span>, <span class="author">Hubbard T</span>, <span class="author">Reymond A</span>, <span class="author">Antonarakis SE</span>, <span class="author">Hannon G</span>, <span class="author">Giddings MC</span>, <span class="author">Ruan Y</span>, <span class="author">Wold B</span>, <span class="author">Carninci P</span>, <span class="author">Guig&oacute; R</span> and <span class="author">Gingeras TR</span></p>
<p class="affiliation">Centre for Genomic Regulation and UPF, Doctor Aiguader 88, Barcelona 08003, Catalonia, Spain.</p><p class="abstract">Eukaryotic cells make many types of primary and processed RNAs that are found either in specific subcellular compartments or throughout the cells. A complete catalogue of these RNAs is not yet available and their characteristic subcellular localizations are also poorly understood. Because RNA represents the direct output of the genetic information encoded by genomes and a significant proportion of a cell&#39;s regulatory capabilities are focused on its synthesis, processing, transport, modification and translation, the generation of such a catalogue is crucial for understanding genome function. Here we report evidence that three-quarters of the human genome is capable of being transcribed, as well as observations about the range and levels of expression, localization, processing fates, regulatory regions and modifications of almost all currently annotated and thousands of previously unannotated RNAs. These observations, taken together, prompt a redefinition of the concept of a gene.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P30 CA045508</span>; NHGRI NIH HHS: <span class="grant">1RC2HG005591</span>, <span class="grant">R01 HG003700</span>, <span class="grant">R01HG003700</span>, <span class="grant">RC2 HG005591</span>, <span class="grant">U01 HG003147</span>, <span class="grant">U54 HG004555</span>, <span class="grant">U54 HG004557</span>, <span class="grant">U54 HG004558</span>, <span class="grant">U54 HG004576</span>, <span class="grant">U54 HG007004</span>, <span class="grant">U54HG004555</span>, <span class="grant">U54HG004557</span>, <span class="grant">U54HG004558</span>, <span class="grant">U54HG004576</span>; NIGMS NIH HHS: <span class="grant">R37 GM062534</span>; Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">489</span>;<span class="number">7414</span>;<span class="pagerange">101-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955620">22955620</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3684276">3684276</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11233">10.1038/nature11233</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evidence for transcript networks composed of chimeric RNAs in human cells.</h4>
<p class="authors"><span class="author">Djebali S</span>, <span class="author">Lagarde J</span>, <span class="author">Kapranov P</span>, <span class="author">Lacroix V</span>, <span class="author">Borel C</span>, <span class="author">Mudge JM</span>, <span class="author">Howald C</span>, <span class="author">Foissac S</span>, <span class="author">Ucla C</span>, <span class="author">Chrast J</span>, <span class="author">Ribeca P</span>, <span class="author">Martin D</span>, <span class="author">Murray RR</span>, <span class="author">Yang X</span>, <span class="author">Ghamsari L</span>, <span class="author">Lin C</span>, <span class="author">Bell I</span>, <span class="author">Dumais E</span>, <span class="author">Drenkow J</span>, <span class="author">Tress ML</span>, <span class="author">Gelp&iacute; JL</span>, <span class="author">Orozco M</span>, <span class="author">Valencia A</span>, <span class="author">van Berkum NL</span>, <span class="author">Lajoie BR</span>, <span class="author">Vidal M</span>, <span class="author">Stamatoyannopoulos J</span>, <span class="author">Batut P</span>, <span class="author">Dobin A</span>, <span class="author">Harrow J</span>, <span class="author">Hubbard T</span>, <span class="author">Dekker J</span>, <span class="author">Frankish A</span>, <span class="author">Salehi-Ashtiani K</span>, <span class="author">Reymond A</span>, <span class="author">Antonarakis SE</span>, <span class="author">Guig&oacute; R</span> and <span class="author">Gingeras TR</span></p>
<p class="affiliation">Bioinformatics and Genomics, Centre for Genomic Regulation and Universitat Pompeu Fabra, Barcelona, Catalonia, Spain.</p><p class="abstract">The classic organization of a gene structure has followed the Jacob and Monod bacterial gene model proposed more than 50 years ago. Since then, empirical determinations of the complexity of the transcriptomes found in yeast to human has blurred the definition and physical boundaries of genes. Using multiple analysis approaches we have characterized individual gene boundaries mapping on human chromosomes 21 and 22. Analyses of the locations of the 5&#39; and 3&#39; transcriptional termini of 492 protein coding genes revealed that for 85% of these genes the boundaries extend beyond the current annotated termini, most often connecting with exons of transcripts from other well annotated genes. The biological and evolutionary importance of these chimeric transcripts is underscored by (1) the non-random interconnections of genes involved, (2) the greater phylogenetic depth of the genes involved in many chimeric interactions, (3) the coordination of the expression of connected genes and (4) the close in vivo and three dimensional proximity of the genomic regions being transcribed and contributing to parts of the chimeric RNAs. The non-random nature of the connection of the genes involved suggest that chimeric transcripts should not be studied in isolation, but together, as an RNA network.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG003143</span>, <span class="grant">R01 HG003143</span>, <span class="grant">U01HG003147</span>, <span class="grant">U01HG003150</span>, <span class="grant">U54 HG004592</span>, <span class="grant">U54HG004557</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">e28213</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22238572">22238572</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3251577">3251577</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0028213">10.1371/journal.pone.0028213</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human SH2B1 mutations are associated with maladaptive behaviors and obesity.</h4>
<p class="authors"><span class="author">Doche ME</span>, <span class="author">Bochukova EG</span>, <span class="author">Su HW</span>, <span class="author">Pearce LR</span>, <span class="author">Keogh JM</span>, <span class="author">Henning E</span>, <span class="author">Cline JM</span>, <span class="author">Saeed S</span>, <span class="author">Dale A</span>, <span class="author">Cheetham T</span>, <span class="author">Barroso I</span>, <span class="author">Argetsinger LS</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Rui L</span>, <span class="author">Carter-Su C</span> and <span class="author">Farooqi IS</span></p>
<p class="affiliation">Department of Molecular and Integrative Physiology, University of Michigan Medical School, Ann Arbor, Michigan 48109-5622, USA.</p><p class="abstract">Src homology 2 B adapter protein 1 (SH2B1) modulates signaling by a variety of ligands that bind to receptor tyrosine kinases or JAK-associated cytokine receptors, including leptin, insulin, growth hormone (GH), and nerve growth factor (NGF). Targeted deletion of Sh2b1 in mice results in increased food intake, obesity, and insulin resistance, with an intermediate phenotype seen in heterozygous null mice on a high-fat diet. We identified SH2B1 loss-of-function mutations in a large cohort of patients with severe early-onset obesity. Mutation carriers exhibited hyperphagia, childhood-onset obesity, disproportionate insulin resistance, and reduced final height as adults. Unexpectedly, mutation carriers exhibited a spectrum of behavioral abnormalities that were not reported in controls, including social isolation and aggression. We conclude that SH2B1 plays a critical role in the control of human food intake and body weight and is implicated in maladaptive human behavior.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900554</span>, <span class="grant">G9824984</span>; NCI NIH HHS: <span class="grant">P30-CA46592</span>; NIDDK NIH HHS: <span class="grant">P60 DK020572</span>, <span class="grant">P60-DK20572</span>, <span class="grant">R01 DK054222</span>, <span class="grant">R01 DK065122</span>, <span class="grant">R01 DK073601</span>, <span class="grant">R01-DK065122</span>, <span class="grant">R01-DK073601</span>, <span class="grant">R01-DK54222</span>; NIGMS NIH HHS: <span class="grant">T32 GM008322</span>, <span class="grant">T32-GM008322</span>; Wellcome Trust: <span class="grant">077016/Z/05/Z</span>, <span class="grant">082390/Z/07/Z</span>, <span class="grant">098497</span></p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2012</span>;<span class="volume">122</span>;<span class="number">12</span>;<span class="pagerange">4732-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23160192">23160192</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3533535">3533535</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI62696">10.1172/JCI62696</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hyperactive piggyBac gene transfer in human cells and in vivo.</h4>
<p class="authors"><span class="author">Doherty JE</span>, <span class="author">Huye LE</span>, <span class="author">Yusa K</span>, <span class="author">Zhou L</span>, <span class="author">Craig NL</span> and <span class="author">Wilson MH</span></p>
<p class="affiliation">Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030, USA.</p><p class="abstract">We characterized a recently developed hyperactive piggyBac (pB) transposase enzyme [containing seven mutations (7pB)] for gene transfer in human cells in vitro and to somatic cells in mice in vivo. Despite a protein level expression similar to that of native pB, 7pB significantly increased the gene transfer efficiency of a neomycin resistance cassette transposon in both HEK293 and HeLa cultured human cells. Native pB and SB100X, the most active transposase of the Sleeping Beauty transposon system, exhibited similar transposition efficiency in cultured human cell lines. When delivered to primary human T cells ex vivo, 7pB increased gene delivery two- to threefold compared with piggyBac and SB100X. The activity of hyperactive 7pB transposase was not affected by the addition of a 24-kDa N-terminal tag, whereas SB100X manifested a 50% reduction in transposition. Hyperactive 7pB was compared with native pB and SB100X in vivo in mice using hydrodynamic tail-vein injection of a limiting dose of transposase DNA combined with luciferase reporter transposons. We followed transgene expression for up to 6 months and observed approximately 10-fold greater long-term gene expression in mice injected with a codon-optimized version of 7pB compared with mice injected with native pB or SB100X. We conclude that hyperactive piggyBac elements can increase gene transfer in human cells and in vivo and should enable improved gene delivery using the piggyBac transposon system in a variety of cell and gene-therapy applications.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">T32DK064717</span>; NIGMS NIH HHS: <span class="grant">T32GM007330</span></p>
<p class="publication"><span class="title">Human gene therapy</span>&nbsp;<span class="year">2012</span>;<span class="volume">23</span>;<span class="number">3</span>;<span class="pagerange">311-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21992617" rel="external">21992617</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3300075" rel="external">3300075</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fhum.2011.138" rel="external">10.1089/hum.2011.138</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The microbiological and clinical characteristics of invasive salmonella in gallbladders from cholecystectomy patients in kathmandu, Nepal.</h4>
<p class="authors"><span class="author">Dongol S</span>, <span class="author">Thompson CN</span>, <span class="author">Clare S</span>, <span class="author">Nga TV</span>, <span class="author">Duy PT</span>, <span class="author">Karkey A</span>, <span class="author">Arjyal A</span>, <span class="author">Koirala S</span>, <span class="author">Khatri NS</span>, <span class="author">Maskey P</span>, <span class="author">Poudel S</span>, <span class="author">Jaiswal VK</span>, <span class="author">Vaidya S</span>, <span class="author">Dougan G</span>, <span class="author">Farrar JJ</span>, <span class="author">Dolecek C</span>, <span class="author">Basnyat B</span> and <span class="author">Baker S</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Patan Academy of Health Sciences, Kathmandu, Nepal.</p>
<p class="abstract">Gallbladder carriage of invasive Salmonella is considered fundamental in sustaining typhoid fever transmission. Bile and tissue was obtained from 1,377 individuals undergoing cholecystectomy in Kathmandu to investigate the prevalence, characteristics and relevance of invasive Salmonella in the gallbladder in an endemic area. Twenty percent of bile samples contained a Gram-negative organism, with Salmonella Typhi and Salmonella Paratyphi A isolated from 24 and 22 individuals, respectively. Gallbladders that contained Salmonella were more likely to show evidence of acute inflammation with extensive neutrophil infiltrate than those without Salmonella, corresponding with higher neutrophil and lower lymphocyte counts in the blood of Salmonella positive individuals. Antimicrobial resistance in the invasive Salmonella isolates was limited, indicating that gallbladder colonization is unlikely to be driven by antimicrobial resistance. The overall role of invasive Salmonella carriage in the gallbladder is not understood; here we show that 3.5% of individuals undergoing cholecystectomy in this setting have a high concentration of antimicrobial sensitive, invasive Salmonella in their bile. We predict that such individuals will become increasingly important if current transmission mechanisms are disturbed; prospectively identifying these individuals is, therefore, paramount for rapid local and regional elimination.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">e47342</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23077595" rel="external">23077595</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3471863" rel="external">3471863</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0047342" rel="external">10.1371/journal.pone.0047342</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The battle of the SNPs.</h4>
<p class="authors"><span class="author">Downing T</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">1</span>;<span class="pagerange">6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22138960">22138960</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2716">10.1038/nrmicro2716</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide SNP and microsatellite variation illuminate population-level epidemiology in the Leishmania donovani species complex.</h4>
<p class="authors"><span class="author">Downing T</span>, <span class="author">Stark O</span>, <span class="author">Vanaerschot M</span>, <span class="author">Imamura H</span>, <span class="author">Sanders M</span>, <span class="author">Decuypere S</span>, <span class="author">de Doncker S</span>, <span class="author">Maes I</span>, <span class="author">Rijal S</span>, <span class="author">Sundar S</span>, <span class="author">Dujardin JC</span>, <span class="author">Berriman M</span> and <span class="author">Sch&ouml;nian G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%54%69%6D.%44%6F%77%6E%69%6E%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#84;&#105;&#109;&#46;&#68;&#111;&#119;&#110;&#105;&#110;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The species of the Leishmania donovani species complex cause visceral leishmaniasis, a debilitating infectious disease transmitted by sandflies. Understanding molecular changes associated with population structure in these parasites can help unravel their epidemiology and spread in humans. In this study, we used a panel of standard microsatellite loci and genome-wide SNPs to investigate population-level diversity in L. donovani strains recently isolated from a small geographic area spanning India, Bihar and Nepal, and compared their variation to that found in diverse strains of the L. donovani complex isolates from Europe, Africa and Asia. Microsatellites and SNPs could clearly resolve the phylogenetic relationships of the strains between continents, and microsatellite phylogenies indicated that certain older Indian strains were closely related to African strains. In the context of the anti-malaria spraying campaigns in the 1960s, this was consistent with a pattern of episodic population size contractions and clonal expansions in these parasites that was supported by population history simulations. In sharp contrast to the low resolution provided by microsatellites, SNPs retained a much more fine-scale resolution of population-level variability to the extent that they identified four different lineages from the same region one of which was more closely related to African and European strains than to Indian or Nepalese ones. Joining results of in vitro testing the antimonial drug sensitivity with the phylogenetic signals from the SNP data highlighted protein-level mutations revealing a distinct drug-resistant group of Nepalese and Indian L. donovani. This study demonstrates the power of genomic data for exploring parasite population structure. Furthermore, markers defining different genetic groups have been discovered that could potentially be applied to investigate drug resistance in clinical Leishmania strains.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Infection, genetics and evolution : journal of molecular epidemiology and evolutionary genetics in infectious diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="number">1</span>;<span class="pagerange">149-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22119748">22119748</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3315668">3315668</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.meegid.2011.11.005">10.1016/j.meegid.2011.11.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fluorescence-based phenotypic selection allows forward genetic screens in haploid human cells.</h4>
<p class="authors"><span class="author">Duncan LM</span>, <span class="author">Timms RT</span>, <span class="author">Zavodszky E</span>, <span class="author">Cano F</span>, <span class="author">Dougan G</span>, <span class="author">Randow F</span> and <span class="author">Lehner PJ</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Addenbrooke's Hospital, Cambridge, United Kingdom.</p>
<p class="abstract">The isolation of haploid cell lines has recently allowed the power of forward genetic screens to be applied to mammalian cells. The interest in applying this powerful genetic approach to a mammalian system is only tempered by the limited utility of these screens, if confined to lethal phenotypes. Here we expand the scope of these approaches beyond live/dead screens and show that selection for a cell surface phenotype via fluorescence-activated cell sorting can identify the key molecules in an intracellular pathway, in this case MHC class I antigen presentation. Non-lethal haploid genetic screens are widely applicable to identify genes involved in essentially any cellular pathway.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e39651</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22745803" rel="external">22745803</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3382162" rel="external">3382162</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0039651" rel="external">10.1371/journal.pone.0039651</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation in human genes encoding adhesion and proinflammatory molecules are associated with severe malaria in the Vietnamese.</h4>
<p class="authors"><span class="author">Dunstan SJ</span>, <span class="author">Rockett KA</span>, <span class="author">Quyen NT</span>, <span class="author">Teo YY</span>, <span class="author">Thai CQ</span>, <span class="author">Hang NT</span>, <span class="author">Jeffreys A</span>, <span class="author">Clark TG</span>, <span class="author">Small KS</span>, <span class="author">Simmons CP</span>, <span class="author">Day N</span>, <span class="author">O&#39;Riordan SE</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Farrar J</span>, <span class="author">Phu NH</span>, <span class="author">Hien TT</span> and <span class="author">MalariaGEN Consortium</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Ho Chi Minh City, Vietnam. <a href="mailto:%73%64%75%6E%73%74%61%6E%40%6F%75%63%72%75.%6F%72%67">&#115;&#100;&#117;&#110;&#115;&#116;&#97;&#110;&#64;&#111;&#117;&#99;&#114;&#117;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The genetic basis for susceptibility to malaria has been studied widely in African populations but less is known of the contribution of specific genetic variants in Asian populations. We genotyped 67 single-nucleotide polymorphisms (SNPs) in 1030 severe malaria cases and 2840 controls from Vietnam. After data quality control, genotyping data of 956 cases and 2350 controls were analysed for 65 SNPs (3 gender confirmation, 62 positioned in/near 42 malarial candidate genes). A total of 14 SNPs were monomorphic and 2 (rs8078340 and rs33950507) were not in Hardy-Weinberg equilibrium in controls (P&lt;0.01). In all, 7/46 SNPs in 6 genes (ICAM1, IL1A, IL17RC, IL13, LTA and TNF) were associated with severe malaria, with 3/7 SNPs in the TNF/LTA region. Genotype-phenotype correlations between SNPs and clinical parameters revealed that genotypes of rs708567 (IL17RC) correlate with parasitemia (P=0.028, r(2)=0.0086), with GG homozygotes having the lowest parasite burden. Additionally, rs708567 GG homozygotes had a decreased risk of severe malaria (P=0.007, OR=0.78 (95% CI; 0.65-0.93)) and death (P=0.028, OR=0.58 (95% CI; 0.37-0.93)) than those with AA and AG genotypes. In summary, variants in six genes encoding adhesion and proinflammatory molecules are associated with severe malaria in the Vietnamese. Further replicative studies in independent populations will be necessary to confirm these findings.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">089276/Z/09/Z</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">098051</span>, <span class="grant">WT077383/Z/05/Z</span></p>
<p class="publication"><span class="title">Genes and immunity</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">6</span>;<span class="pagerange">503-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22673309">22673309</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fgene.2012.25">10.1038/gene.2012.25</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">AntiFam: a tool to help identify spurious ORFs in protein annotation.</h4>
<p class="authors"><span class="author">Eberhardt RY</span>, <span class="author">Haft DH</span>, <span class="author">Punta M</span>, <span class="author">Martin M</span>, <span class="author">O&#39;Donovan C</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA. UK. <a href="mailto:%72%65%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#101;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">As the deluge of genomic DNA sequence grows the fraction of protein sequences that have been manually curated falls. In turn, as the number of laboratories with the ability to sequence genomes in a high-throughput manner grows, the informatics capability of those labs to accurately identify and annotate all genes within a genome may often be lacking. These issues have led to fears about transitive annotation errors making sequence databases less reliable. During the lifetime of the Pfam protein families database a number of protein families have been built, which were later identified as composed solely of spurious open reading frames (ORFs) either on the opposite strand or in a different, overlapping reading frame with respect to the true protein-coding or non-coding RNA gene. These families were deleted and are no longer available in Pfam. However, we realized that these may perform a useful function to identify new spurious ORFs. We have collected these families together in AntiFam along with additional custom-made families of spurious ORFs. This resource currently contains 23 families that identified 1310 spurious proteins in UniProtKB and a further 4119 spurious proteins in a collection of metagenomic sequences. UniProt has adopted AntiFam as a part of the UniProtKB quality control process and will investigate these spurious proteins for exclusion.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG004881).</span>; Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2012</span>;<span class="volume">2012</span>;<span class="pagerange">bas003</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22434837" rel="external">22434837</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308159" rel="external">3308159</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbas003" rel="external">10.1093/database/bas003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics: ENCODE explained.</h4>
<p class="authors"><span class="author">Ecker JR</span>, <span class="author">Bickmore WA</span>, <span class="author">Barroso I</span>, <span class="author">Pritchard JK</span>, <span class="author">Gilad Y</span> and <span class="author">Segal E</span></p>
<p class="affiliation">Howard Hughes Medical Institute and the Salk Institute for Biological Studies, La Jolla, California 92037, USA. <a href="mailto:%65%63%6B%65%72%40%73%61%6C%6B.%65%64%75">&#101;&#99;&#107;&#101;&#114;&#64;&#115;&#97;&#108;&#107;&#46;&#101;&#100;&#117;</a></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">489</span>;<span class="number">7414</span>;<span class="pagerange">52-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955614">22955614</a>; DOI: <a href="http://dx.doi.org/10.1038%2F489052a">10.1038/489052a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated encyclopedia of DNA elements in the human genome.</h4>
<p class="authors"><span class="author">ENCODE Project Consortium</span></p><p class="abstract">The human genome encodes the blueprint of life, but the function of the vast majority of its nearly three billion bases is unknown. The Encyclopedia of DNA Elements (ENCODE) project has systematically mapped regions of transcription, transcription factor association, chromatin structure and histone modification. These data enabled us to assign biochemical functions for 80% of the genome, in particular outside of the well-studied protein-coding regions. Many discovered candidate regulatory elements are physically associated with one another and with expressed genes, providing new insights into the mechanisms of gene regulation. The newly identified elements also show a statistical correspondence to sequence variants linked to human disease, and can thereby guide interpretation of this variation. Overall, the project provides new insights into the organization and regulation of our genes and genome, and is an expansive resource of functional annotations for biomedical research.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P30 CA016086</span>, <span class="grant">P30 CA045508</span>; NHGRI NIH HHS: <span class="grant">K99 HG006698</span>, <span class="grant">R01 HG003143</span>, <span class="grant">R01 HG003988</span>, <span class="grant">R01 HG005085</span>, <span class="grant">R01HG003143</span>, <span class="grant">R01HG003541</span>, <span class="grant">R01HG003700</span>, <span class="grant">R01HG003988</span>, <span class="grant">R01HG004456-03</span>, <span class="grant">RC2 HG005573</span>, <span class="grant">RC2HG005591</span>, <span class="grant">RC2HG005679</span>, <span class="grant">U01 HG004695</span>, <span class="grant">U01HG004561</span>, <span class="grant">U01HG004571</span>, <span class="grant">U01HG004695</span>, <span class="grant">U41HG004568</span>, <span class="grant">U54 HG006997</span>, <span class="grant">U54 HG006998</span>, <span class="grant">U54HG004555</span>, <span class="grant">U54HG004557</span>, <span class="grant">U54HG004558</span>, <span class="grant">U54HG004563</span>, <span class="grant">U54HG004570</span>, <span class="grant">U54HG004576</span>, <span class="grant">U54HG004592</span>; NIDDK NIH HHS: <span class="grant">R01 DK054369</span>, <span class="grant">R01 DK065806</span>, <span class="grant">R37 DK044746</span>; NIGMS NIH HHS: <span class="grant">T32 GM007205</span>, <span class="grant">T32 GM081739</span>; PHS HHS: <span class="grant">ZIAHG200323</span>, <span class="grant">ZIAHG200341</span>; Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">489</span>;<span class="number">7414</span>;<span class="pagerange">57-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955616">22955616</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3439153">3439153</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11247">10.1038/nature11247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic characterization of northeastern Italian population isolates in the context of broader European genetic diversity.</h4>
<p class="authors"><span class="author">Esko T</span>, <span class="author">Mezzavilla M</span>, <span class="author">Nelis M</span>, <span class="author">Borel C</span>, <span class="author">Debniak T</span>, <span class="author">Jakkula E</span>, <span class="author">Julia A</span>, <span class="author">Karachanak S</span>, <span class="author">Khrunin A</span>, <span class="author">Kisfali P</span>, <span class="author">Krulisova V</span>, <span class="author">Au&scaron;rel&eacute; Ku&#x10D;inskien&eacute; Z</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Traglia M</span>, <span class="author">Nikitina-Zake L</span>, <span class="author">Zimprich F</span>, <span class="author">Antonarakis SE</span>, <span class="author">Estivill X</span>, <span class="author">Glava&#x10D; D</span>, <span class="author">Gut I</span>, <span class="author">Klovins J</span>, <span class="author">Krawczak M</span>, <span class="author">Ku&#x10D;inskas V</span>, <span class="author">Lathrop M</span>, <span class="author">Macek M</span>, <span class="author">Marsal S</span>, <span class="author">Meitinger T</span>, <span class="author">Melegh B</span>, <span class="author">Limborska S</span>, <span class="author">Lubinski J</span>, <span class="author">Paolotie A</span>, <span class="author">Schreiber S</span>, <span class="author">Toncheva D</span>, <span class="author">Toniolo D</span>, <span class="author">Wichmann HE</span>, <span class="author">Zimprich A</span>, <span class="author">Metspalu M</span>, <span class="author">Gasparini P</span>, <span class="author">Metspalu A</span> and <span class="author">D&#39;Adamo P</span></p>
<p class="affiliation">1] Estonian Genome Center, University of Tartu, Tartu, Estonia [2] Department of Biotechnology, Institute of Molecular and Cell Biology, University of Tartu, Tartu, Estonia [3] Estonian Biocentre, Tartu, Estonia.</p>
<p class="abstract">Population genetic studies on European populations have highlighted Italy as one of genetically most diverse regions. This is possibly due to the country's complex demographic history and large variability in terrain throughout the territory. This is the reason why Italy is enriched for population isolates, Sardinia being the best-known example. As the population isolates have a great potential in disease-causing genetic variants identification, we aimed to genetically characterize a region from northeastern Italy, which is known for isolated communities. Total of 1310 samples, collected from six geographically isolated villages, were genotyped at >145â€‰000 single-nucleotide polymorphism positions. Newly genotyped data were analyzed jointly with the available genome-wide data sets of individuals of European descent, including several population isolates. Despite the linguistic differences and geographical isolation the village populations still show the greatest genetic similarity to other Italian samples. The genetic isolation and small effective population size of the village populations is manifested by higher levels of genomic homozygosity and elevated linkage disequilibrium. These estimates become even more striking when the detected substructure is taken into account. The observed level of genetic isolation in Friuli-Venezia Giulia region is more extreme according to several measures of isolation compared with Sardinians, French Basques and northern Finns, thus proving the status of an isolate.European Journal of Human Genetics advance online publication, 19 December 2012; doi:10.1038/ejhg.2012.229.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23249956" rel="external">23249956</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2012.229" rel="external">10.1038/ejhg.2012.229</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of methods for competitive tests of pathway analysis.</h4>
<p class="authors"><span class="author">Evangelou M</span>, <span class="author">Rendon A</span>, <span class="author">Ouwehand WH</span>, <span class="author">Wernisch L</span> and <span class="author">Dudbridge F</span></p>
<p class="affiliation">Medical Research Council Biostatistics Unit, Institute of Public Health, Cambridge, United Kingdom.</p>
<p class="abstract">It has been suggested that pathway analysis can complement single-SNP analysis in exploring genomewide association data. Pathway analysis incorporates the available biological knowledge of genes and SNPs and is expected to improve the chances of revealing the underlying genetic architecture of complex traits. Methods for pathway analysis can be classified as competitive (enrichment) or self-contained (association) according to the hypothesis tested. Although association tests are statistically more powerful than enrichment tests they can be difficult to calibrate because biases in analysis accumulate across multiple SNPs or genes. Furthermore, enrichment tests can be more scientifically relevant than association tests, as they detect pathways with relatively more evidence for association than the remaining genes. Here we show how some well known association tests can be simply adapted to test for enrichment, and compare their performance to some established enrichment tests. We propose versions of the Adaptive Rank Truncated Product (ARTP), Tail Strength Measure and Fisher's combination of p-values for testing the enrichment null hypothesis. We compare the behaviour of these proposed methods with the established Hypergeometric Test and Gene-Set Enrichment Analysis (GSEA). The results of the simulation study show that the modified version of the ARTP method has generally the best performance across the situations considered. The methods were also applied for finding enriched pathways for body mass index (BMI) and platelet function phenotypes. The pathway analysis of BMI identified the Vasoactive Intestinal Peptide pathway as significantly associated with BMI. This pathway has been previously reported as associated with BMI and the risk of obesity. The ARTP method was the method that identified the largest number of enriched pathways across all tested pathway databases and phenotypes. The simulation and data application results are in agreement with previous work on association tests and suggests that the ARTP should be preferred for both enrichment and association testing.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e41018</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22859961" rel="external">22859961</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3409204" rel="external">3409204</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0041018" rel="external">10.1371/journal.pone.0041018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">IFITM3 restricts the morbidity and mortality associated with influenza.</h4>
<p class="authors"><span class="author">Everitt AR</span>, <span class="author">Clare S</span>, <span class="author">Pertel T</span>, <span class="author">John SP</span>, <span class="author">Wash RS</span>, <span class="author">Smith SE</span>, <span class="author">Chin CR</span>, <span class="author">Feeley EM</span>, <span class="author">Sims JS</span>, <span class="author">Adams DJ</span>, <span class="author">Wise HM</span>, <span class="author">Kane L</span>, <span class="author">Goulding D</span>, <span class="author">Digard P</span>, <span class="author">Anttila V</span>, <span class="author">Baillie JK</span>, <span class="author">Walsh TS</span>, <span class="author">Hume DA</span>, <span class="author">Palotie A</span>, <span class="author">Xue Y</span>, <span class="author">Colonna V</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Dunning J</span>, <span class="author">Gordon SB</span>, <span class="author">GenISIS Investigators</span>, <span class="author">MOSAIC Investigators</span>, <span class="author">Smyth RL</span>, <span class="author">Openshaw PJ</span>, <span class="author">Dougan G</span>, <span class="author">Brass AL</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">The 2009 H1N1 influenza pandemic showed the speed with which a novel respiratory virus can spread and the ability of a generally mild infection to induce severe morbidity and mortality in a subset of the population. Recent in vitro studies show that the interferon-inducible transmembrane (IFITM) protein family members potently restrict the replication of multiple pathogenic viruses. Both the magnitude and breadth of the IFITM proteins&#39; in vitro effects suggest that they are critical for intrinsic resistance to such viruses, including influenza viruses. Using a knockout mouse model, we now test this hypothesis directly and find that IFITM3 is essential for defending the host against influenza A virus in vivo. Mice lacking Ifitm3 display fulminant viral pneumonia when challenged with a normally low-pathogenicity influenza virus, mirroring the destruction inflicted by the highly pathogenic 1918 &#39;Spanish&#39; influenza. Similar increased viral replication is seen in vitro, with protection rescued by the re-introduction of Ifitm3. To test the role of IFITM3 in human influenza virus infection, we assessed the IFITM3 alleles of individuals hospitalized with seasonal or pandemic influenza H1N1/09 viruses. We find that a statistically significant number of hospitalized subjects show enrichment for a minor IFITM3 allele (SNP rs12252-C) that alters a splice acceptor site, and functional assays show the minor CC genotype IFITM3 has reduced influenza virus restriction in vitro. Together these data reveal that the action of a single intrinsic immune effector, IFITM3, profoundly alters the course of influenza virus infection in mouse and humans.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Chief Scientist Office; Medical Research Council: <span class="grant">G0600511</span>, <span class="grant">G0800767</span>, <span class="grant">G0800777</span>, <span class="grant">G0802752</span>, <span class="grant">G0901697</span>, <span class="grant">MC_G1001212</span>, <span class="grant">MC_U122785833</span>; NIAID NIH HHS: <span class="grant">R01 AI091786</span>, <span class="grant">R01AI091786</span>; NIDDK NIH HHS: <span class="grant">P30 DK043351</span>; Wellcome Trust: <span class="grant">090382</span>, <span class="grant">090382/Z/09/Z</span>, <span class="grant">090385/Z/09/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">484</span>;<span class="number">7395</span>;<span class="pagerange">519-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22446628">22446628</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3648786">3648786</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10921">10.1038/nature10921</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-density genetic mapping identifies new susceptibility loci for rheumatoid arthritis.</h4>
<p class="authors"><span class="author">Eyre S</span>, <span class="author">Bowes J</span>, <span class="author">Diogo D</span>, <span class="author">Lee A</span>, <span class="author">Barton A</span>, <span class="author">Martin P</span>, <span class="author">Zhernakova A</span>, <span class="author">Stahl E</span>, <span class="author">Viatte S</span>, <span class="author">McAllister K</span>, <span class="author">Amos CI</span>, <span class="author">Padyukov L</span>, <span class="author">Toes RE</span>, <span class="author">Huizinga TW</span>, <span class="author">Wijmenga C</span>, <span class="author">Trynka G</span>, <span class="author">Franke L</span>, <span class="author">Westra HJ</span>, <span class="author">Alfredsson L</span>, <span class="author">Hu X</span>, <span class="author">Sandor C</span>, <span class="author">de Bakker PI</span>, <span class="author">Davila S</span>, <span class="author">Khor CC</span>, <span class="author">Heng KK</span>, <span class="author">Andrews R</span>, <span class="author">Edkins S</span>, <span class="author">Hunt SE</span>, <span class="author">Langford C</span>, <span class="author">Symmons D</span>, <span class="author">Biologics in Rheumatoid Arthritis Genetics and Genomics Study Syndicate</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Concannon P</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Rich SS</span>, <span class="author">Deloukas P</span>, <span class="author">Gonzalez-Gay MA</span>, <span class="author">Rodriguez-Rodriguez L</span>, <span class="author">&Auml;rlsetig L</span>, <span class="author">Martin J</span>, <span class="author">Rantap&auml;&auml;-Dahlqvist S</span>, <span class="author">Plenge RM</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Klareskog L</span>, <span class="author">Gregersen PK</span> and <span class="author">Worthington J</span></p>
<p class="affiliation">Arthritis Research UK Epidemiology Unit, Centre for Musculoskeletal Research, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK.</p><p class="abstract">Using the Immunochip custom SNP array, which was designed for dense genotyping of 186 loci identified through genome-wide association studies (GWAS), we analyzed 11,475 individuals with rheumatoid arthritis (cases) of European ancestry and 15,870 controls for 129,464 markers. We combined these data in a meta-analysis with GWAS data from additional independent cases (n = 2,363) and controls (n = 17,872). We identified 14 new susceptibility loci, 9 of which were associated with rheumatoid arthritis overall and five of which were specifically associated with disease that was positive for anticitrullinated peptide antibodies, bringing the number of confirmed rheumatoid arthritis risk loci in individuals of European ancestry to 46. We refined the peak of association to a single gene for 19 loci, identified secondary independent effects at 6 loci and identified association to low-frequency variants at 4 loci. Bioinformatic analyses generated strong hypotheses for the causal SNP at seven loci. This study illustrates the advantages of dense SNP mapping analysis to inform subsequent functional investigations.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>, <span class="grant">18475</span>; Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/137</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0600329</span>, <span class="grant">G0800759</span>, <span class="grant">MC_U147585819</span>, <span class="grant">MC_UP_A620_1014</span>; NIAID NIH HHS: <span class="grant">R01 AI068759</span>; NIAMS NIH HHS: <span class="grant">1R01AR062886-01</span>, <span class="grant">K08AR055688</span>, <span class="grant">N01-AR-2-2263</span>, <span class="grant">N01-AR1-2256</span>, <span class="grant">R01 AR056768</span>, <span class="grant">R01 AR062886</span>, <span class="grant">R01-AR-4-4422</span>, <span class="grant">R01-AR056768</span>, <span class="grant">R01-AR057108</span>, <span class="grant">R01-AR059648</span>, <span class="grant">RC2AR059092-01</span>; NIGMS NIH HHS: <span class="grant">U01-GM092691</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">090532</span>, <span class="grant">091157</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">12</span>;<span class="pagerange">1336-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23143596">23143596</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605761">3605761</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2462">10.1038/ng.2462</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative proteomics reveals a significant bias toward alternative protein isoforms with conserved structure and function.</h4>
<p class="authors"><span class="author">Ezkurdia I</span>, <span class="author">del Pozo A</span>, <span class="author">Frankish A</span>, <span class="author">Rodriguez JM</span>, <span class="author">Harrow J</span>, <span class="author">Ashman K</span>, <span class="author">Valencia A</span> and <span class="author">Tress ML</span></p>
<p class="affiliation">Structural Biology and Biocomputing Programme, Spanish National Cancer Research Centre, Madrid, Spain.</p><p class="abstract">Advances in high-throughput mass spectrometry are making proteomics an increasingly important tool in genome annotation projects. Peptides detected in mass spectrometry experiments can be used to validate gene models and verify the translation of putative coding sequences (CDSs). Here, we have identified peptides that cover 35% of the genes annotated by the GENCODE consortium for the human genome as part of a comprehensive analysis of experimental spectra from two large publicly available mass spectrometry databases. We detected the translation to protein of &quot;novel&quot; and &quot;putative&quot; protein-coding transcripts as well as transcripts annotated as pseudogenes and nonsense-mediated decay targets. We provide a detailed overview of the population of alternatively spliced protein isoforms that are detectable by peptide identification methods. We found that 150 genes expressed multiple alternative protein isoforms. This constitutes the largest set of reliably confirmed alternatively spliced proteins yet discovered. Three groups of genes were highly overrepresented. We detected alternative isoforms for 10 of the 25 possible heterogeneous nuclear ribonucleoproteins, proteins with a key role in the splicing process. Alternative isoforms generated from interchangeable homologous exons and from short indels were also significantly enriched, both in human experiments and in parallel analyses of mouse and Drosophila proteomics experiments. Our results show that a surprisingly high proportion (almost 25%) of the detected alternative isoforms are only subtly different from their constitutive counterparts. Many of the alternative splicing events that give rise to these alternative isoforms are conserved in mouse. It was striking that very few of these conserved splicing events broke Pfam functional domains or would damage globular protein structures. This evidence of a strong bias toward subtle differences in CDS and likely conserved cellular function and structure is remarkable and strongly suggests that the translation of alternative transcripts may be subject to selective constraints.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG0004555</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2012</span>;<span class="volume">29</span>;<span class="number">9</span>;<span class="pagerange">2265-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22446687">22446687</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3424414">3424414</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmss100">10.1093/molbev/mss100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles.</h4>
<p class="authors"><span class="author">Fairfax BP</span>, <span class="author">Makino S</span>, <span class="author">Radhakrishnan J</span>, <span class="author">Plant K</span>, <span class="author">Leslie S</span>, <span class="author">Dilthey A</span>, <span class="author">Ellis P</span>, <span class="author">Langford C</span>, <span class="author">Vannberg FO</span> and <span class="author">Knight JC</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%62%66%61%69%72%66%61%78%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#98;&#102;&#97;&#105;&#114;&#102;&#97;&#120;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Trans-acting genetic variants have a substantial, albeit poorly characterized, role in the heritable determination of gene expression. Using paired purified primary monocytes and B cells, we identify new predominantly cell type-specific cis and trans expression quantitative trait loci (eQTLs), including multi-locus trans associations to LYZ and KLF4 in monocytes and B cells, respectively. Additionally, we observe a B cell-specific trans association of rs11171739 at 12q13.2, a known autoimmune disease locus, with IP6K2 (P = 5.8 &times; 10(-15)), PRIC285 (P = 3.0 &times; 10(-10)) and an upstream region of CDKN1A (P = 2 &times; 10(-52)), suggesting roles for cell cycle regulation and peroxisome proliferator-activated receptor &gamma; (PPAR&gamma;) signaling in autoimmune pathogenesis. We also find that specific human leukocyte antigen (HLA) alleles form trans associations with the expression of AOAH and ARHGAP24 in monocytes but not in B cells. In summary, we show that mapping gene expression in defined primary cell populations identifies new cell type-specific trans-regulated networks and provides insights into the genetic basis of disease susceptibility.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">074318</span>, <span class="grant">075491/Z/04</span>, <span class="grant">088891</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">502-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22446964" rel="external">22446964</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2205" rel="external">10.1038/ng.2205</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of anti-Notch antibodies and their application in blocking Notch signalling in neural stem cells.</h4>
<p class="authors"><span class="author">Falk R</span>, <span class="author">Falk A</span>, <span class="author">Dyson MR</span>, <span class="author">Melidoni A</span>, <span class="author">Parthiban K</span>, <span class="author">Young J</span>, <span class="author">Roake W</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">University of Cambridge, Department of Biochemistry, Tennis Court Road, CB2 1QW Cambridge, UK.</p>
<p class="abstract">Notch signalling occurs via direct cell-cell interactions and plays an important role in linking the fates of neighbouring cells. There are four different mammalian Notch receptors that can be activated by five cell surface ligands. The ability to inhibit specific Notch receptors would help identify the roles of individual family members and potentially provide a means to study and control cell differentiation. Anti-Notch antibodies in the form of single chain Fvs were generated from an antibody phage display library by selection on either the ligand binding domain or the negative regulatory region (NRR) of Notch1 and Notch2. Six antibodies targeting the NRR of Notch1 and four antibodies recognising the NRR of Notch2 were found to prevent receptor activation in cell-based luciferase reporter assays. These antibodies were potent, highly specific inhibitors of individual Notch receptors and interfered with endogenous signalling in stem cell systems of both human and mouse origin. Antibody-mediated inhibition of Notch efficiently down-regulated transcription of the immediate Notch target gene hairy and enhancer of split 5 (Hes5) in both mouse and human neural stem cells and revealed a redundant regulation of Hes5 in these cells as complete down-regulation was seen only after simultaneous blocking of Notch1 and Notch2. In addition, these antibodies promoted differentiation of neural stem cells towards a neuronal fate. In contrast to the widely used small molecule Î³-secretase inhibitors, which block all 4 Notch receptors (and a multitude of other signalling pathways), antibodies allow blockade of individual Notch family members in a highly specific way. Specific inhibition will allow examination of the effect of individual Notch receptors in complex differentiation schemes regulated by the co-ordinated action of multiple signalling pathways.</p>
<p class="publication"><span class="title">Methods (San Diego, Calif.)</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22842086" rel="external">22842086</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ymeth.2012.07.008" rel="external">10.1016/j.ymeth.2012.07.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Automatic categorization of diverse experimental information in the bioscience literature.</h4>
<p class="authors"><span class="author">Fang R</span>, <span class="author">Schindelman G</span>, <span class="author">Van Auken K</span>, <span class="author">Fernandes J</span>, <span class="author">Chen W</span>, <span class="author">Wang X</span>, <span class="author">Davis P</span>, <span class="author">Tuli MA</span>, <span class="author">Marygold SJ</span>, <span class="author">Millburn G</span>, <span class="author">Matthews B</span>, <span class="author">Zhang H</span>, <span class="author">Brown N</span>, <span class="author">Gelbart WM</span> and <span class="author">Sternberg PW</span></p>
<p class="affiliation">Howard Hughes Medical Institute and Biology Division, California Institute of Technology, Pasadena, CA 91125, USA.</p><p class="abstract"><strong>Background</strong>: Curation of information from bioscience literature into biological knowledge databases is a crucial way of capturing experimental information in a computable form. During the biocuration process, a critical first step is to identify from all published literature the papers that contain results for a specific data type the curator is interested in annotating. This step normally requires curators to manually examine many papers to ascertain which few contain information of interest and thus, is usually time consuming. We developed an automatic method for identifying papers containing these curation data types among a large pool of published scientific papers based on the machine learning method Support Vector Machine (SVM). This classification system is completely automatic and can be readily applied to diverse experimental data types. It has been in use in production for automatic categorization of 10 different experimental datatypes in the biocuration process at WormBase for the past two years and it is in the process of being adopted in the biocuration process at FlyBase and the Saccharomyces Genome Database (SGD). We anticipate that this method can be readily adopted by various databases in the biocuration community and thereby greatly reducing time spent on an otherwise laborious and demanding task. We also developed a simple, readily automated procedure to utilize training papers of similar data types from different bodies of literature such as C. elegans and D. melanogaster to identify papers with any of these data types for a single database. This approach has great significance because for some data types, especially those of low occurrence, a single corpus often does not have enough training papers to achieve satisfactory performance.</p><p class="abstract"><strong>Results</strong>: We successfully tested the method on ten data types from WormBase, fifteen data types from FlyBase and three data types from Mouse Genomics Informatics (MGI). It is being used in the curation work flow at WormBase for automatic association of newly published papers with ten data types including RNAi, antibody, phenotype, gene regulation, mutant allele sequence, gene expression, gene product interaction, overexpression phenotype, gene interaction, and gene structure correction.</p><p class="abstract"><strong>Conclusions</strong>: Our methods are applicable to a variety of data types with training set containing several hundreds to a few thousand documents. It is completely automatic and, thus can be readily incorporated to different workflow at different literature-based databases. We believe that the work presented here can contribute greatly to the tremendous task of automating the important yet labor-intensive biocuration effort.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">P41 HG000739</span>, <span class="grant">P41 HG002223</span>, <span class="grant">P41 HG002223-10S1</span>, <span class="grant">R01 HG004090</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22280404" rel="external">22280404</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3305665" rel="external">3305665</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-13-16" rel="external">10.1186/1471-2105-13-16</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bifidobacterial surface-exopolysaccharide facilitates commensal-host interaction through immune modulation and pathogen protection.</h4>
<p class="authors"><span class="author">Fanning S</span>, <span class="author">Hall LJ</span>, <span class="author">Cronin M</span>, <span class="author">Zomer A</span>, <span class="author">MacSharry J</span>, <span class="author">Goulding D</span>, <span class="author">Motherway MO</span>, <span class="author">Shanahan F</span>, <span class="author">Nally K</span>, <span class="author">Dougan G</span> and <span class="author">van Sinderen D</span></p>
<p class="affiliation">Alimentary Pharmabiotic Centre, University College Cork, Cork, Ireland.</p><p class="abstract">Bifidobacteria comprise a significant proportion of the human gut microbiota. Several bifidobacterial strains are currently used as therapeutic interventions, claiming various health benefits by acting as probiotics. However, the precise mechanisms by which they maintain habitation within their host and consequently provide these benefits are not fully understood. Here we show that Bifidobacterium breve UCC2003 produces a cell surface-associated exopolysaccharide (EPS), the biosynthesis of which is directed by either half of a bidirectional gene cluster, thus leading to production of one of two possible EPSs. Alternate transcription of the two opposing halves of this cluster appears to be the result of promoter reorientation. Surface EPS provided stress tolerance and promoted in vivo persistence, but not initial colonization. Marked differences were observed in host immune response: strains producing surface EPS (EPS(+)) failed to elicit a strong immune response compared with EPS-deficient variants. Specifically, EPS production was shown to be linked to the evasion of adaptive B-cell responses. Furthermore, presence of EPS(+) B. breve reduced colonization levels of the gut pathogen Citrobacter rodentium. Our data thus assigns a pivotal and beneficial role for EPS in modulating various aspects of bifidobacterial-host interaction, including the ability of commensal bacteria to remain immunologically silent and in turn provide pathogen protection. This finding enforces the probiotic concept and provides mechanistic insights into health-promoting benefits for both animal and human hosts.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">6</span>;<span class="pagerange">2108-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22308390" rel="external">22308390</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3277520" rel="external">3277520</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1115621109" rel="external">10.1073/pnas.1115621109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cohesin regulates tissue-specific expression by stabilising highly occupied cis-regulatory modules.</h4>
<p class="authors"><span class="author">Faure AJ</span>, <span class="author">Schmidt D</span>, <span class="author">Watt S</span>, <span class="author">Schwalie PC</span>, <span class="author">Wilson MD</span>, <span class="author">Xu H</span>, <span class="author">Ramsay RG</span>, <span class="author">Odom DT</span> and <span class="author">Flicek P</span></p>
<p class="affiliation">European Bioinformatics Institute;</p>
<p class="abstract">The cohesin protein complex contributes to transcriptional regulation in a CTCF-independent manner by colocalising with master regulators at tissue-specific loci. The regulation of transcription involves the concerted action of multiple transcription factors (TFs) and cohesin's role in this context of combinatorial TF binding remains unexplored. To investigate cohesin-non-CTCF (CNC) binding events in vivo we mapped cohesin and CTCF, as well as a collection of tissue-specific and ubiquitous transcriptional regulators, using ChIP-seq in primary mouse liver. We observe a positive correlation between the number of distinct TFs bound and the presence of CNC sites. In contrast to regions of the genome where cohesin and CTCF colocalise, CNC sites coincide with the binding of master regulators and enhancer-markers and are significantly associated with liver-specific expressed genes. We also show that cohesin presence partially explains the commonly observed discrepancy between TF motif score and ChIP signal. Evidence from these statistical analyses in wild type cells, and comparisons to maps of TF binding in Rad21-cohesin haploinsufficient mouse liver, suggests that cohesin helps to stabilise large protein-DNA complexes. Finally, we observe that the presence of mirrored CTCF binding events at promoters and their nearby cohesin-bound enhancers is associated with elevated expression levels.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22780989" rel="external">22780989</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.136507.111" rel="external">10.1101/gr.136507.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Invasive non-typhoidal salmonella disease: an emerging and neglected tropical disease in Africa.</h4>
<p class="authors"><span class="author">Feasey NA</span>, <span class="author">Dougan G</span>, <span class="author">Kingsley RA</span>, <span class="author">Heyderman RS</span> and <span class="author">Gordon MA</span></p>
<p class="affiliation">Malawi-Liverpool-Wellcome Trust Clinical Research Programme, Blantyre, Malawi; Department of Gastroenterology, Institute of Translational Medicine, University of Liverpool, Liverpool, UK.</p><p class="abstract">Invasive strains of non-typhoidal salmonellae have emerged as a prominent cause of bloodstream infection in African adults and children, with an associated case fatality of 20-25%. The clinical presentation of invasive non-typhoidal salmonella disease in Africa is diverse: fever, hepatosplenomegaly, and respiratory symptoms are common, and features of enterocolitis are often absent. The most important risk factors are HIV infection in adults, and malaria, HIV, and malnutrition in children. A distinct genotype of Salmonella enterica var Typhimurium, ST313, has emerged as a new pathogenic clade in sub-Saharan Africa, and might have adapted to cause invasive disease in human beings. Multidrug-resistant ST313 has caused epidemics in several African countries, and has driven the use of expensive antimicrobial drugs in the poorest health services in the world. Studies of systemic cellular and humoral immune responses in adults infected with HIV have revealed key host immune defects contributing to invasive non-typhoidal salmonella disease. This emerging pathogen might therefore have adapted to occupy an ecological and immunological niche provided by HIV, malaria, and malnutrition in Africa. A good understanding of the epidemiology of this neglected disease will open new avenues for development and implementation of vaccine and public health strategies to prevent infections and interrupt transmission.</p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22587967" rel="external">22587967</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(11)61752-2" rel="external">10.1016/S0140-6736(11)61752-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Making your database available through Wikipedia: the pros and cons.</h4>
<p class="authors"><span class="author">Finn RD</span>, <span class="author">Gardner PP</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">HHMI Janelia Farm Research Campus, 19700 Helix Drive, Ashburn, VA, USA. <a href="mailto:%66%69%6E%6E%72%40%6A%61%6E%65%6C%69%61.%68%68%6D%69.%6F%72%67">&#102;&#105;&#110;&#110;&#114;&#64;&#106;&#97;&#110;&#101;&#108;&#105;&#97;&#46;&#104;&#104;&#109;&#105;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Wikipedia, the online encyclopedia, is the most famous wiki in use today. It contains over 3.7 million pages of content; with many pages written on scientific subject matters that include peer-reviewed citations, yet are written in an accessible manner and generally reflect the consensus opinion of the community. In this, the 19th Annual Database Issue of Nucleic Acids Research, there are 11 articles that describe the use of a wiki in relation to a biological database. In this commentary, we discuss how biological databases can be integrated with Wikipedia, thereby utilising the pre-existing infrastructure, tools and above all, large community of authors (or Wikipedians). The limitations to the content that can be included in Wikipedia are highlighted, with examples drawn from articles found in this issue and other wiki-based resources, indicating why other wiki solutions are necessary. We discuss the merits of using open wikis, like Wikipedia, versus other models, with particular reference to potential vandalism. Finally, we raise the question about the future role of dedicated database biocurators in context of the thousands of crowdsourced, community annotations that are now being stored in wikis.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D9-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22144683" rel="external">22144683</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245093" rel="external">3245093</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr1195" rel="external">10.1093/nar/gkr1195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Progressive cross-reactivity in IgE responses: an explanation for the slow development of human immunity to schistosomiasis?</h4>
<p class="authors"><span class="author">Fitzsimmons CM</span>, <span class="author">Jones FM</span>, <span class="author">Pinot de Moira A</span>, <span class="author">Protasio AV</span>, <span class="author">Khalife J</span>, <span class="author">Dickinson HA</span>, <span class="author">Tukahebwa EM</span> and <span class="author">Dunne DW</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Cambridge, United Kindgdom. <a href="mailto:%63%6D%66%31%30%30%30%40%63%61%6D.%61%63.%75%6B">&#99;&#109;&#102;&#49;&#48;&#48;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">People in regions of Schistosoma mansoni endemicity slowly acquire immunity, but why this takes years to develop is still not clear. It has been associated with increases in parasite-specific IgE, induced, some investigators propose, to antigens exposed during the death of adult worms. These antigens include members of the tegumental-allergen-like protein family (TAL1 to TAL13). Previously, in a group of S. mansoni-infected Ugandan males, we showed that IgE responses to three TALs expressed in worms (TAL1, -3, and -5) became more prevalent with age. Now, in a subcohort we examined associations of these responses with resistance to reinfection and use the data to propose a mechanism for the slow development of immunity. IgE was measured 9 weeks posttreatment and at reinfection at 2 years (n = 144). An anti-TAL5 IgE (herein referred to as TAL5 IgE) response was associated with reduced reinfection even after adjusting for age using regression analysis (geometric mean odds ratio, 0.24; P = 0.016). TAL5 IgE responders were a subset of TAL3 IgE responders, themselves a subset of TAL1 responders. TAL3 IgE and TAL5 IgE were highly cross-reactive, with TAL3 the immunizing antigen and TAL5 the cross-reactive antigen. Transcriptional and translational studies show that TAL3 is most abundant in adult worms and that TAL5 is most abundant in infectious larvae. We propose that in chronic schistosomiasis, older individuals have repeatedly experienced IgE antigens exposed when adult worms die (e.g., TAL3) and that this leads to increasing cross-reactivity with antigens of invading larvae (e.g., TAL5). Progressive accumulation of worm/larvae cross-reactivity could explain the age-dependent immunity observed in areas of endemicity.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">083931/&sim;/07/Z</span></p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2012</span>;<span class="volume">80</span>;<span class="number">12</span>;<span class="pagerange">4264-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23006852">23006852</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3497412">3497412</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00641-12">10.1128/IAI.00641-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2012.</h4>
<p class="authors"><span class="author">Flicek P</span>, <span class="author">Amode MR</span>, <span class="author">Barrell D</span>, <span class="author">Beal K</span>, <span class="author">Brent S</span>, <span class="author">Carvalho-Silva D</span>, <span class="author">Clapham P</span>, <span class="author">Coates G</span>, <span class="author">Fairley S</span>, <span class="author">Fitzgerald S</span>, <span class="author">Gil L</span>, <span class="author">Gordon L</span>, <span class="author">Hendrix M</span>, <span class="author">Hourlier T</span>, <span class="author">Johnson N</span>, <span class="author">K&auml;h&auml;ri AK</span>, <span class="author">Keefe D</span>, <span class="author">Keenan S</span>, <span class="author">Kinsella R</span>, <span class="author">Komorowska M</span>, <span class="author">Koscielny G</span>, <span class="author">Kulesha E</span>, <span class="author">Larsson P</span>, <span class="author">Longden I</span>, <span class="author">McLaren W</span>, <span class="author">Muffato M</span>, <span class="author">Overduin B</span>, <span class="author">Pignatelli M</span>, <span class="author">Pritchard B</span>, <span class="author">Riat HS</span>, <span class="author">Ritchie GR</span>, <span class="author">Ruffier M</span>, <span class="author">Schuster M</span>, <span class="author">Sobral D</span>, <span class="author">Tang YA</span>, <span class="author">Taylor K</span>, <span class="author">Trevanion S</span>, <span class="author">Vandrovcova J</span>, <span class="author">White S</span>, <span class="author">Wilson M</span>, <span class="author">Wilder SP</span>, <span class="author">Aken BL</span>, <span class="author">Birney E</span>, <span class="author">Cunningham F</span>, <span class="author">Dunham I</span>, <span class="author">Durbin R</span>, <span class="author">Fern&aacute;ndez-Suarez XM</span>, <span class="author">Harrow J</span>, <span class="author">Herrero J</span>, <span class="author">Hubbard TJ</span>, <span class="author">Parker A</span>, <span class="author">Proctor G</span>, <span class="author">Spudich G</span>, <span class="author">Vogel J</span>, <span class="author">Yates A</span>, <span class="author">Zadissa A</span> and <span class="author">Searle SM</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton Cambridge CB10 1SD, UK. <a href="mailto:%66%6C%69%63%65%6B%40%65%62%69.%61%63.%75%6B">&#102;&#108;&#105;&#99;&#101;&#107;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl project (http://www.ensembl.org) provides genome resources for chordate genomes with a particular focus on human genome data as well as data for key model organisms such as mouse, rat and zebrafish. Five additional species were added in the last year including gibbon (Nomascus leucogenys) and Tasmanian devil (Sarcophilus harrisii) bringing the total number of supported species to 61 as of Ensembl release 64 (September 2011). Of these, 55 species appear on the main Ensembl website and six species are provided on the Ensembl preview site (Pre!Ensembl; http://pre.ensembl.org) with preliminary support. The past year has also seen improvements across the project.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01HG004695</span>, <span class="grant">U41HG006104</span>, <span class="grant">U54HG004563</span>; Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT062023</span>, <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D84-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22086963">22086963</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245178">3245178</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr991">10.1093/nar/gkr991</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Resisting resistance.</h4>
<p class="authors"><span class="author">Foth BJ</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">This month's Genome Watch describes how knowledge of the malaria parasite genome can be used to better understand and mitigate the emergence of drug resistance.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">8</span>;<span class="pagerange">524</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22796882" rel="external">22796882</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2847" rel="external">10.1038/nrmicro2847</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The importance of identifying alternative splicing in vertebrate genome annotation.</h4>
<p class="authors"><span class="author">Frankish A</span>, <span class="author">Mudge JM</span>, <span class="author">Thomas M</span> and <span class="author">Harrow J</span></p>
<p class="affiliation">Human and Vertebrate Analysis and Annotation Team, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%66%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#102;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">While alternative splicing (AS) can potentially expand the functional repertoire of vertebrate genomes, relatively few AS transcripts have been experimentally characterized. We describe our detailed manual annotation of vertebrate genomes, which is generating a publicly available geneset rich in AS. In order to achieve this we have adopted a highly sensitive approach to annotating gene models supported by correctly mapped, canonically spliced transcriptional evidence combined with a highly cautious approach to adding unsupported extensions to models and making decisions on their functional potential. We use information about the predicted functional potential and structural properties of every AS transcript annotated at a protein-coding or non-coding locus to place them into one of eleven subclasses. We describe the incorporation of new sequencing and proteomics technologies into our annotation pipelines, which are used to identify and validate AS. Combining all data sources has led to the production of a rich geneset containing an average of 6.3&thinsp;AS transcripts for every human multi-exon protein-coding gene. The datasets produced have proved very useful in providing context to studies investigating the functional potential of genes and the effect of variation may have on gene structure and function. DATABASE URL: http://www.ensembl.org/index.html, http://vega.sanger.ac.uk/index.html.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5U54HG004555-04S1</span>; Wellcome Trust: <span class="grant">WT077198</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2012</span>;<span class="volume">2012</span>;<span class="pagerange">bas014</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22434846">22434846</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308168">3308168</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbas014">10.1093/database/bas014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis identifies susceptibility loci for migraine without aura.</h4>
<p class="authors"><span class="author">Freilinger T</span>, <span class="author">Anttila V</span>, <span class="author">de Vries B</span>, <span class="author">Malik R</span>, <span class="author">Kallela M</span>, <span class="author">Terwindt GM</span>, <span class="author">Pozo-Rosich P</span>, <span class="author">Winsvold B</span>, <span class="author">Nyholt DR</span>, <span class="author">van Oosterhout WP</span>, <span class="author">Artto V</span>, <span class="author">Todt U</span>, <span class="author">H&auml;m&auml;l&auml;inen E</span>, <span class="author">Fern&aacute;ndez-Morales J</span>, <span class="author">Louter MA</span>, <span class="author">Kaunisto MA</span>, <span class="author">Schoenen J</span>, <span class="author">Raitakari O</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Vila-Pueyo M</span>, <span class="author">G&ouml;bel H</span>, <span class="author">Wichmann E</span>, <span class="author">Sintas C</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Heinze A</span>, <span class="author">Tronvik E</span>, <span class="author">van Duijn CM</span>, <span class="author">Kaprio J</span>, <span class="author">Cormand B</span>, <span class="author">Wessman M</span>, <span class="author">Frants RR</span>, <span class="author">Meitinger T</span>, <span class="author">M&uuml;ller-Myhsok B</span>, <span class="author">Zwart JA</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Macaya A</span>, <span class="author">Ferrari MD</span>, <span class="author">Kubisch C</span>, <span class="author">Palotie A</span>, <span class="author">Dichgans M</span>, <span class="author">van den Maagdenberg AM</span> and <span class="author">International Headache Genetics Consortium</span></p>
<p class="affiliation">Institute for Stroke and Dementia Research, Klinikum der Universit&auml;t M&uuml;nchen, Munich, Germany.</p><p class="abstract">Migraine without aura is the most common form of migraine, characterized by recurrent disabling headache and associated autonomic symptoms. To identify common genetic variants associated with this migraine type, we analyzed genome-wide association data of 2,326 clinic-based German and Dutch individuals with migraine without aura and 4,580 population-matched controls. We selected SNPs from 12 loci with 2 or more SNPs associated with P values of &lt;1 &times; 10(-5) for replication testing in 2,508 individuals with migraine without aura and 2,652 controls. SNPs at two of these loci showed convincing replication: at 1q22 (in MEF2D; replication P = 4.9 &times; 10(-4); combined P = 7.06 &times; 10(-11)) and at 3p24 (near TGFBR2; replication P = 1.0 &times; 10(-4); combined P = 1.17 &times; 10(-9)). In addition, SNPs at the PHACTR1 and ASTN2 loci showed suggestive evidence of replication (P = 0.01; combined P = 3.20 &times; 10(-8) and P = 0.02; combined P = 3.86 &times; 10(-8), respectively). We also replicated associations at two previously reported migraine loci in or near TRPM8 and LRP1. This study identifies the first susceptibility loci for migraine without aura, thereby expanding our knowledge of this debilitating neurological disorder.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">7</span>;<span class="pagerange">777-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22683712">22683712</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3773912">3773912</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2307">10.1038/ng.2307</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A familial case with interstitial 2q36 deletion: variable phenotypic expression in full and mosaic state.</h4>
<p class="authors"><span class="author">Freitas &Eacute;L</span>, <span class="author">Gribble SM</span>, <span class="author">Simioni M</span>, <span class="author">Vieira TP</span>, <span class="author">Prigmore E</span>, <span class="author">Krepischi AC</span>, <span class="author">Rosenberg C</span>, <span class="author">Pearson PL</span>, <span class="author">Melo DG</span> and <span class="author">Gil-da-Silva-Lopes VL</span></p>
<p class="affiliation">Department of Medical Genetics, Faculty of Medical Sciences, University of Campinas, Rua Tess&aacute;lia Vieira de Camargo, 126 CEP 13083-887 Campinas, S&atilde;o Paulo, Brazil.</p><p class="abstract">Submicroscopic chromosomal anomalies play an important role in the etiology of craniofacial malformations, including midline facial defects with hypertelorism (MFDH). MFDH is a common feature combination in several conditions, of which Frontonasal Dysplasia is the most frequently encountered manifestation; in most cases the etiology remains unknown. We identified a parent to child transmission of a 6.2 Mb interstitial deletion of chromosome region 2q36.1q36.3 by array-CGH and confirmed by FISH and microsatellite analysis. The patient and her mother both presented an MFDH phenotype although the phenotype in the mother was much milder than her daughter. Inspection of haplotype segregation within the family of 2q36.1 region suggests that the deletion arose on a chromosome derived from the maternal grandfather. Evidences based on FISH, microsatellite and array-CGH analysis point to a high frequency mosaicism for presence of a deleted region 2q36 occurring in blood of the mother. The frequency of mosaicism in other tissues could not be determined. We here suggest that the milder phenotype observed in the proband&#39;s mother can be explained by the mosaic state of the deletion. This most likely arose by an early embryonic deletion in the maternal embryo resulting in both gonadal and somatic mosaicism of two cell lines, with and without the deleted chromosome. The occurrence of gonadal mosaicism increases the recurrence risk significantly and is often either underestimated or not even taken into account in genetic counseling where new mutation is suspected.</p>
<p class="publication"><span class="title">European journal of medical genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">55</span>;<span class="number">11</span>;<span class="pagerange">660-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22820457">22820457</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ejmg.2012.06.015">10.1016/j.ejmg.2012.06.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intracranial aneurysm risk locus 5q23.2 is associated with elevated systolic blood pressure.</h4>
<p class="authors"><span class="author">Ga&aacute;l EI</span>, <span class="author">Salo P</span>, <span class="author">Kristiansson K</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Kettunen J</span>, <span class="author">Sarin AP</span>, <span class="author">Niemel&auml; M</span>, <span class="author">Jula A</span>, <span class="author">Raitakari OT</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Eriksson JG</span>, <span class="author">Widen E</span>, <span class="author">G&uuml;nel M</span>, <span class="author">Kurki M</span>, <span class="author">von und Zu Fraunberg M</span>, <span class="author">J&auml;&auml;skel&auml;inen JE</span>, <span class="author">Hernesniemi J</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Pouta A</span>, <span class="author">International Consortium for Blood Pressure Genome-Wide Association Studies</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Salomaa V</span>, <span class="author">Palotie A</span> and <span class="author">Perola M</span></p>
<p class="affiliation">Public Health Genomics Unit, Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland. <a href="mailto:%65%6D%69%6C%69%61.%67%61%61%6C%40%68%65%6C%73%69%6E%6B%69.%66%69">&#101;&#109;&#105;&#108;&#105;&#97;&#46;&#103;&#97;&#97;&#108;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a></p><p class="abstract">Although genome-wide association studies (GWAS) have identified hundreds of complex trait loci, the pathomechanisms of most remain elusive. Studying the genetics of risk factors predisposing to disease is an attractive approach to identify targets for functional studies. Intracranial aneurysms (IA) are rupture-prone pouches at cerebral artery branching sites. IA is a complex disease for which GWAS have identified five loci with strong association and a further 14 loci with suggestive association. To decipher potential underlying disease mechanisms, we tested whether there are IA loci that convey their effect through elevating blood pressure (BP), a strong risk factor of IA. We performed a meta-analysis of four population-based Finnish cohorts (n(FIN)&#x200A; = &#x200A;11 266) not selected for IA, to assess the association of previously identified IA candidate loci (n &#x200A;=&#x200A; 19) with BP. We defined systolic BP (SBP), diastolic BP, mean arterial pressure, and pulse pressure as quantitative outcome variables. The most significant result was further tested for association in the ICBP-GWAS cohort of 200 000 individuals. We found that the suggestive IA locus at 5q23.2 in PRDM6 was significantly associated with SBP in individuals of European descent (p(FIN) &#x200A;=&#x200A; 3.01E-05, p(ICBP-GWAS) &#x200A;= &#x200A;0.0007, p(ALL)&#x200A; = &#x200A;8.13E-07). The risk allele of IA was associated with higher SBP. PRDM6 encodes a protein predominantly expressed in vascular smooth muscle cells. Our study connects a complex disease (IA) locus with a common risk factor for the disease (SBP). We hypothesize that common variants in PRDM6 can contribute to altered vascular wall structure, hence increasing SBP and predisposing to IA. True positive associations often fail to reach genome-wide significance in GWAS. Our findings show that analysis of traditional risk factors as intermediate phenotypes is an effective tool for deciphering hidden heritability. Further, we demonstrate that common disease loci identified in a population isolate may bear wider significance.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600705</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e1002563</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22438818" rel="external">22438818</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3305343" rel="external">3305343</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002563" rel="external">10.1371/journal.pgen.1002563</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Controls of nucleosome positioning in the human genome.</h4>
<p class="authors"><span class="author">Gaffney DJ</span>, <span class="author">McVicker G</span>, <span class="author">Pai AA</span>, <span class="author">Fondufe-Mittendorf YN</span>, <span class="author">Lewellen N</span>, <span class="author">Michelini K</span>, <span class="author">Widom J</span>, <span class="author">Gilad Y</span> and <span class="author">Pritchard JK</span></p>
<p class="affiliation">Department of Human Genetics, University of Chicago, Chicago, Illinois, United States of America. <a href="mailto:%64%67%31%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#103;&#49;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Nucleosomes are important for gene regulation because their arrangement on the genome can control which proteins bind to DNA. Currently, few human nucleosomes are thought to be consistently positioned across cells; however, this has been difficult to assess due to the limited resolution of existing data. We performed paired-end sequencing of micrococcal nuclease-digested chromatin (MNase-seq) from seven lymphoblastoid cell lines and mapped over 3.6 billion MNase-seq fragments to the human genome to create the highest-resolution map of nucleosome occupancy to date in a human cell type. In contrast to previous results, we find that most nucleosomes have more consistent positioning than expected by chance and a substantial fraction (8.7%) of nucleosomes have moderate to strong positioning. In aggregate, nucleosome sequences have 10 bp periodic patterns in dinucleotide frequency and DNase I sensitivity; and, across cells, nucleosomes frequently have translational offsets that are multiples of 10 bp. We estimate that almost half of the genome contains regularly spaced arrays of nucleosomes, which are enriched in active chromatin domains. Single nucleotide polymorphisms that reduce DNase I sensitivity can disrupt the phasing of nucleosome arrays, which indicates that they often result from positioning against a barrier formed by other proteins. However, nucleosome arrays can also be created by DNA sequence alone. The most striking example is an array of over 400 nucleosomes on chromosome 12 that is created by tandem repetition of sequences with strong positioning properties. In summary, a large fraction of nucleosomes are consistently positioned--in some regions because they adopt favored sequence positions, and in other regions because they are forced into specific arrangements by chromatin remodeling or DNA binding proteins.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">HG006123</span>; NIMH NIH HHS: <span class="grant">MH090951</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e1003036</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23166509">23166509</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3499251">3499251</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003036">10.1371/journal.pgen.1003036</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dissecting the regulatory architecture of gene expression QTLs.</h4>
<p class="authors"><span class="author">Gaffney DJ</span>, <span class="author">Veyrieras JB</span>, <span class="author">Degner JF</span>, <span class="author">Pique-Regi R</span>, <span class="author">Pai AA</span>, <span class="author">Crawford GE</span>, <span class="author">Stephens M</span>, <span class="author">Gilad Y</span> and <span class="author">Pritchard JK</span></p>
<p class="affiliation">Department of Human Genetics, University of Chicago, 920 E58th Street, Chicago, IL 60637, USA. <a href="mailto:%64%67%31%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#103;&#49;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Expression quantitative trait loci (eQTLs) are likely to play an important role in the genetics of complex traits; however, their functional basis remains poorly understood. Using the HapMap lymphoblastoid cell lines, we combine 1000 Genomes genotypes and an extensive catalogue of human functional elements to investigate the biological mechanisms that eQTLs perturb.</p><p class="abstract"><strong>Results</strong>: We use a Bayesian hierarchical model to estimate the enrichment of eQTLs in a wide variety of regulatory annotations. We find that approximately 40% of eQTLs occur in open chromatin, and that they are particularly enriched in transcription factor binding sites, suggesting that many directly impact protein-DNA interactions. Analysis of core promoter regions shows that eQTLs also frequently disrupt some known core promoter motifs but, surprisingly, are not enriched in other well-known motifs such as the TATA box. We also show that information from regulatory annotations alone, when weighted by the hierarchical model, can provide a meaningful ranking of the SNPs that are most likely to drive gene expression variation.</p><p class="abstract"><strong>Conclusions</strong>: Our study demonstrates how regulatory annotation and the association signal derived from eQTL-mapping can be combined into a single framework. We used this approach to further our understanding of the biology that drives human gene expression variation, and of the putatively causal SNPs that underlie it.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">R01 HG006123</span>; NHLBI NIH HHS: <span class="grant">R01 HL092206</span>; NIGMS NIH HHS: <span class="grant">GM077959</span>; NIMH NIH HHS: <span class="grant">MH084703</span>, <span class="grant">MH090951</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">R7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22293038">22293038</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3334587">3334587</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-1-r7">10.1186/gb-2012-13-1-r7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Universal amplification, next-generation sequencing, and assembly of HIV-1 genomes.</h4>
<p class="authors"><span class="author">Gall A</span>, <span class="author">Ferns B</span>, <span class="author">Morris C</span>, <span class="author">Watson S</span>, <span class="author">Cotten M</span>, <span class="author">Robinson M</span>, <span class="author">Berry N</span>, <span class="author">Pillay D</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Whole HIV-1 genome sequences are pivotal for large-scale studies of inter- and intrahost evolution, including the acquisition of drug resistance mutations. The ability to rapidly and cost-effectively generate large numbers of HIV-1 genome sequences from different populations and geographical locations and determine the effect of minority genetic variants is, however, a limiting factor. Next-generation sequencing promises to bridge this gap but is hindered by the lack of methods for the enrichment of virus genomes across the phylogenetic breadth of HIV-1 and methods for the robust assembly of the virus genomes from short-read data. Here we report a method for the amplification, next-generation sequencing, and unbiased de novo assembly of HIV-1 genomes of groups M, N, and O, as well as recombinants, that does not require prior knowledge of the sequence or subtype. A sensitivity of at least 3,000 copies/ml was determined by using plasma virus samples of known copy numbers. We applied our novel method to compare the genome diversities of HIV-1 groups, subtypes, and genes. The highest level of diversity was found in the env, nef, vpr, tat, and rev genes and parts of the gag gene. Furthermore, we used our method to investigate mutations associated with HIV-1 drug resistance in clinical samples at the level of the complete genome. Drug resistance mutations were detected as both major variant and minor species. In conclusion, we demonstrate the feasibility of our method for large-scale HIV-1 genome sequencing. This will enable the phylogenetic and phylodynamic resolution of the ongoing pandemic and efficient monitoring of complex HIV-1 drug resistance genotypes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">S0753</span></p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">50</span>;<span class="number">12</span>;<span class="pagerange">3838-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22993180">22993180</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3502977">3502977</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.01516-12">10.1128/JCM.01516-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exploiting genetic complexity in cancer to improve therapeutic strategies.</h4>
<p class="authors"><span class="author">Garnett MJ</span> and <span class="author">McDermott U</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%6D%6A%31%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#106;&#49;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Advances in genome sequencing technologies are enabling researchers to make rapid progress in defining the entire repertoire of causal genetic changes in cancer. The response of patients with cancer to therapy is often highly variable and there is an increasing number of examples where mutations in cancer genomes have been shown to have a profound effect on the clinical effectiveness of drugs. An urgent challenge for the research and clinical communities is how to translate these genomic data sets into new and improved therapeutic strategies for the treatment of patients. The use of large-scale cell line-based drug screens to identify genomic &#39;biomarkers&#39; of drug response for the stratification of patients has the potential to transform how patients with cancer are treated.</p>
<p class="publication"><span class="title">Drug discovery today</span>&nbsp;<span class="year">2012</span>;<span class="volume">17</span>;<span class="number">5-6</span>;<span class="pagerange">188-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22342219" rel="external">22342219</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.drudis.2012.01.025" rel="external">10.1016/j.drudis.2012.01.025</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic identification of genomic markers of drug sensitivity in cancer cells.</h4>
<p class="authors"><span class="author">Garnett MJ</span>, <span class="author">Edelman EJ</span>, <span class="author">Heidorn SJ</span>, <span class="author">Greenman CD</span>, <span class="author">Dastur A</span>, <span class="author">Lau KW</span>, <span class="author">Greninger P</span>, <span class="author">Thompson IR</span>, <span class="author">Luo X</span>, <span class="author">Soares J</span>, <span class="author">Liu Q</span>, <span class="author">Iorio F</span>, <span class="author">Surdez D</span>, <span class="author">Chen L</span>, <span class="author">Milano RJ</span>, <span class="author">Bignell GR</span>, <span class="author">Tam AT</span>, <span class="author">Davies H</span>, <span class="author">Stevenson JA</span>, <span class="author">Barthorpe S</span>, <span class="author">Lutz SR</span>, <span class="author">Kogera F</span>, <span class="author">Lawrence K</span>, <span class="author">McLaren-Douglas A</span>, <span class="author">Mitropoulos X</span>, <span class="author">Mironenko T</span>, <span class="author">Thi H</span>, <span class="author">Richardson L</span>, <span class="author">Zhou W</span>, <span class="author">Jewitt F</span>, <span class="author">Zhang T</span>, <span class="author">O&#39;Brien P</span>, <span class="author">Boisvert JL</span>, <span class="author">Price S</span>, <span class="author">Hur W</span>, <span class="author">Yang W</span>, <span class="author">Deng X</span>, <span class="author">Butler A</span>, <span class="author">Choi HG</span>, <span class="author">Chang JW</span>, <span class="author">Baselga J</span>, <span class="author">Stamenkovic I</span>, <span class="author">Engelman JA</span>, <span class="author">Sharma SV</span>, <span class="author">Delattre O</span>, <span class="author">Saez-Rodriguez J</span>, <span class="author">Gray NS</span>, <span class="author">Settleman J</span>, <span class="author">Futreal PA</span>, <span class="author">Haber DA</span>, <span class="author">Stratton MR</span>, <span class="author">Ramaswamy S</span>, <span class="author">McDermott U</span> and <span class="author">Benes CH</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Clinical responses to anticancer therapies are often restricted to a subset of patients. In some cases, mutated cancer genes are potent biomarkers for responses to targeted agents. Here, to uncover new biomarkers of sensitivity and resistance to cancer therapeutics, we screened a panel of several hundred cancer cell lines--which represent much of the tissue-type and genetic diversity of human cancers--with 130 drugs under clinical and preclinical investigation. In aggregate, we found that mutated cancer genes were associated with cellular response to most currently available cancer drugs. Classic oncogene addiction paradigms were modified by additional tissue-specific or expression biomarkers, and some frequently mutated genes were associated with sensitivity to a broad range of therapeutic agents. Unexpected relationships were revealed, including the marked sensitivity of Ewing&#39;s sarcoma cells harbouring the EWS (also known as EWSR1)-FLI1 gene translocation to poly(ADP-ribose) polymerase (PARP) inhibitors. By linking drug activity to the functional complexity of cancer genomes, systematic pharmacogenomic profiling in cancer cell lines provides a powerful biomarker discovery platform to guide rational cancer therapeutic strategies.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">1U54HG006097-01</span>; NIGMS NIH HHS: <span class="grant">P41GM079575-02</span>; Wellcome Trust: <span class="grant">086357</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">483</span>;<span class="number">7391</span>;<span class="pagerange">570-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22460902">22460902</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3349233">3349233</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11005">10.1038/nature11005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recent advances in biocuration: meeting report from the fifth International Biocuration Conference.</h4>
<p class="authors"><span class="author">Gaudet P</span>, <span class="author">Arighi C</span>, <span class="author">Bastian F</span>, <span class="author">Bateman A</span>, <span class="author">Blake JA</span>, <span class="author">Cherry MJ</span>, <span class="author">D&#39;Eustachio P</span>, <span class="author">Finn R</span>, <span class="author">Giglio M</span>, <span class="author">Hirschman L</span>, <span class="author">Kania R</span>, <span class="author">Klimke W</span>, <span class="author">Martin MJ</span>, <span class="author">Karsch-Mizrachi I</span>, <span class="author">Munoz-Torres M</span>, <span class="author">Natale D</span>, <span class="author">O&#39;Donovan C</span>, <span class="author">Ouellette F</span>, <span class="author">Pruitt KD</span>, <span class="author">Robinson-Rechavi M</span>, <span class="author">Sansone SA</span>, <span class="author">Schofield P</span>, <span class="author">Sutton G</span>, <span class="author">Van Auken K</span>, <span class="author">Vasudevan S</span>, <span class="author">Wu C</span>, <span class="author">Young J</span> and <span class="author">Mazumder R</span></p>
<p class="affiliation">International Society for Biocuration and CALIPHO Group, Swiss Institute of Bioinformatics, 1 Rue Michel Servet, Geneva, Switzerland. <a href="mailto:%70%61%73%63%61%6C%65.%67%61%75%64%65%74%40%69%73%62-%73%69%62.%63%68">&#112;&#97;&#115;&#99;&#97;&#108;&#101;&#46;&#103;&#97;&#117;&#100;&#101;&#116;&#64;&#105;&#115;&#98;&#45;&#115;&#105;&#98;&#46;&#99;&#104;</a></p>
<p class="abstract">The 5th International Biocuration Conference brought together over 300 scientists to exchange on their work, as well as discuss issues relevant to the International Society for Biocuration's (ISB) mission. Recurring themes this year included the creation and promotion of gold standards, the need for more ontologies, and more formal interactions with journals. The conference is an essential part of the ISB's goal to support exchanges among members of the biocuration community. Next year's conference will be held in Cambridge, UK, from 7 to 10 April 2013. In the meanwhile, the ISB website provides information about the society's activities (http://biocurator.org), as well as related events of interest.</p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2012</span>;<span class="volume">2012</span>;<span class="pagerange">bas036</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23110974">23110974</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3483532">3483532</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbas036">10.1093/database/bas036</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic analysis of Xenopus tropicalis.</h4>
<p class="authors"><span class="author">Geach TJ</span>, <span class="author">Stemple DL</span> and <span class="author">Zimmerman LB</span></p>
<p class="affiliation">National Institute for Medical Research, London, England, UK.</p><p class="abstract">The pipid frog Xenopus tropicalis has emerged as a powerful new model system for combining genetic and genomic analysis of tetrapod development with robust embryological, molecular, and biochemical assays. Its early development closely resembles that of its well-understood relative X. laevis, from which techniques and reagents can be readily transferred. In contrast to the tetraploid X. laevis, X. tropicalis has a compact diploid genome with strong synteny to those of amniotes. Recently, advances in high-throughput sequencing together with solution-hybridization whole-exome enrichment technology offer powerful strategies for cloning novel mutations as well as reverse genetic identification of sequence lesions in specific genes of interest. Further advantages include the wide range of functional and molecular assays available, the large number of embryos/meioses produced, and the ease of haploid genetics and gynogenesis. The addition of these genetic tools to X. tropicalis provides a uniquely flexible platform for analysis of gene function in vertebrate development.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U117560482</span>, <span class="grant">U117560482</span>; Wellcome Trust: <span class="grant">WT 077047/Z/05/Z</span></p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2012</span>;<span class="volume">917</span>;<span class="pagerange">69-110</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22956083">22956083</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-61779-992-1_5">10.1007/978-1-61779-992-1_5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Experimental and husbandry procedures as potential modifiers of the results of phenotyping tests.</h4>
<p class="authors"><span class="author">Gerdin AK</span>, <span class="author">Igosheva N</span>, <span class="author">Roberson LA</span>, <span class="author">Ismail O</span>, <span class="author">Karp N</span>, <span class="author">Sanderson M</span>, <span class="author">Cambridge E</span>, <span class="author">Shannon C</span>, <span class="author">Sunter D</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Bussell J</span> and <span class="author">White JK</span></p><p class="abstract">To maximize the sensitivity of detecting affects of genetic variants in mice, variables have been minimized through the use of inbred mouse lines, by eliminating infectious organisms and controlling environmental variables. However, the impact of standard animal husbandry and experimental procedures on the validity of experimental data is under appreciated. In this study we monitored the impact of these procedures by using parameters that reflect stress and physiological responses to it. Short-term measures included telemetered heart rate and systolic arterial pressure, core body temperature and blood glucose, while longer-term parameters were assessed such as body weight. Male and female C57BL6/NTac mice were subjected to a range of stressors with different perceived severities ranging from repeated blood glucose and core temperature measurement procedures, intra-peritoneal injection and overnight fasting to cage transport and cage changing. Our studies reveal that common husbandry and experimental procedures significantly influence mouse physiology and behaviour. Systolic arterial pressure, heart rate, locomotor activity, core temperature and blood glucose were elevated in response to a range of experimental procedures. Differences between sexes were evident, female mice displayed more sustained cardiovascular responses and locomotor activity than male mice. These results have important implications for the design and implementation of multiple component experiments where the lasting effects of stress from previous tests may modify the outcomes of subsequent ones.</p>
<p class="publication"><span class="title">Physiology &amp; behavior</span>&nbsp;<span class="year">2012</span>;<span class="volume">106</span>;<span class="number">5</span>;<span class="pagerange">602-611</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22713295" rel="external">22713295</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.physbeh.2012.03.026" rel="external">10.1016/j.physbeh.2012.03.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intratumor heterogeneity and branched evolution revealed by multiregion sequencing.</h4>
<p class="authors"><span class="author">Gerlinger M</span>, <span class="author">Rowan AJ</span>, <span class="author">Horswell S</span>, <span class="author">Larkin J</span>, <span class="author">Endesfelder D</span>, <span class="author">Gronroos E</span>, <span class="author">Martinez P</span>, <span class="author">Matthews N</span>, <span class="author">Stewart A</span>, <span class="author">Tarpey P</span>, <span class="author">Varela I</span>, <span class="author">Phillimore B</span>, <span class="author">Begum S</span>, <span class="author">McDonald NQ</span>, <span class="author">Butler A</span>, <span class="author">Jones D</span>, <span class="author">Raine K</span>, <span class="author">Latimer C</span>, <span class="author">Santos CR</span>, <span class="author">Nohadani M</span>, <span class="author">Eklund AC</span>, <span class="author">Spencer-Dene B</span>, <span class="author">Clark G</span>, <span class="author">Pickering L</span>, <span class="author">Stamp G</span>, <span class="author">Gore M</span>, <span class="author">Szallasi Z</span>, <span class="author">Downward J</span>, <span class="author">Futreal PA</span> and <span class="author">Swanton C</span></p>
<p class="affiliation">Cancer Research UK London Research Institute, London, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Intratumor heterogeneity may foster tumor evolution and adaptation and hinder personalized-medicine strategies that depend on results from single tumor-biopsy samples.</p><p class="abstract"><strong>Methods</strong>: To examine intratumor heterogeneity, we performed exome sequencing, chromosome aberration analysis, and ploidy profiling on multiple spatially separated samples obtained from primary renal carcinomas and associated metastatic sites. We characterized the consequences of intratumor heterogeneity using immunohistochemical analysis, mutation functional analysis, and profiling of messenger RNA expression.</p><p class="abstract"><strong>Results</strong>: Phylogenetic reconstruction revealed branched evolutionary tumor growth, with 63 to 69% of all somatic mutations not detectable across every tumor region. Intratumor heterogeneity was observed for a mutation within an autoinhibitory domain of the mammalian target of rapamycin (mTOR) kinase, correlating with S6 and 4EBP phosphorylation in vivo and constitutive activation of mTOR kinase activity in vitro. Mutational intratumor heterogeneity was seen for multiple tumor-suppressor genes converging on loss of function; SETD2, PTEN, and KDM5C underwent multiple distinct and spatially separated inactivating mutations within a single tumor, suggesting convergent phenotypic evolution. Gene-expression signatures of good and poor prognosis were detected in different regions of the same tumor. Allelic composition and ploidy profiling analysis revealed extensive intratumor heterogeneity, with 26 of 30 tumor samples from four tumors harboring divergent allelic-imbalance profiles and with ploidy heterogeneity in two of four tumors.</p><p class="abstract"><strong>Conclusions</strong>: Intratumor heterogeneity can lead to underestimation of the tumor genomics landscape portrayed from single tumor-biopsy samples and may present major challenges to personalized-medicine and biomarker development. Intratumor heterogeneity, associated with heterogeneous protein function, may foster tumor adaptation and therapeutic failure through Darwinian selection. (Funded by the Medical Research Council and others.).</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0701935</span>, <span class="grant">G0902275</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2012</span>;<span class="volume">366</span>;<span class="number">10</span>;<span class="pagerange">883-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22397650">22397650</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1113205">10.1056/NEJMoa1113205</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of variation at A&beta;PP, PSEN1, PSEN2, and MAPT in late onset Alzheimer&#39;s disease.</h4>
<p class="authors"><span class="author">Gerrish A</span>, <span class="author">Russo G</span>, <span class="author">Richards A</span>, <span class="author">Moskvina V</span>, <span class="author">Ivanov D</span>, <span class="author">Harold D</span>, <span class="author">Sims R</span>, <span class="author">Abraham R</span>, <span class="author">Hollingworth P</span>, <span class="author">Chapman J</span>, <span class="author">Hamshere M</span>, <span class="author">Pahwa JS</span>, <span class="author">Dowzell K</span>, <span class="author">Williams A</span>, <span class="author">Jones N</span>, <span class="author">Thomas C</span>, <span class="author">Stretton A</span>, <span class="author">Morgan AR</span>, <span class="author">Lovestone S</span>, <span class="author">Powell J</span>, <span class="author">Proitsi P</span>, <span class="author">Lupton MK</span>, <span class="author">Brayne C</span>, <span class="author">Rubinsztein DC</span>, <span class="author">Gill M</span>, <span class="author">Lawlor B</span>, <span class="author">Lynch A</span>, <span class="author">Morgan K</span>, <span class="author">Brown KS</span>, <span class="author">Passmore PA</span>, <span class="author">Craig D</span>, <span class="author">McGuinness B</span>, <span class="author">Todd S</span>, <span class="author">Johnston JA</span>, <span class="author">Holmes C</span>, <span class="author">Mann D</span>, <span class="author">Smith AD</span>, <span class="author">Love S</span>, <span class="author">Kehoe PG</span>, <span class="author">Hardy J</span>, <span class="author">Mead S</span>, <span class="author">Fox N</span>, <span class="author">Rossor M</span>, <span class="author">Collinge J</span>, <span class="author">Maier W</span>, <span class="author">Jessen F</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">Heun R</span>, <span class="author">Sch&uuml;rmann B</span>, <span class="author">van den Bussche H</span>, <span class="author">Heuser I</span>, <span class="author">Kornhuber J</span>, <span class="author">Wiltfang J</span>, <span class="author">Dichgans M</span>, <span class="author">Fr&ouml;lich L</span>, <span class="author">Hampel H</span>, <span class="author">H&uuml;ll M</span>, <span class="author">Rujescu D</span>, <span class="author">Goate AM</span>, <span class="author">Kauwe JS</span>, <span class="author">Cruchaga C</span>, <span class="author">Nowotny P</span>, <span class="author">Morris JC</span>, <span class="author">Mayo K</span>, <span class="author">Livingston G</span>, <span class="author">Bass NJ</span>, <span class="author">Gurling H</span>, <span class="author">McQuillin A</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">Davies G</span>, <span class="author">Harris SE</span>, <span class="author">Starr JM</span>, <span class="author">Deary IJ</span>, <span class="author">Al-Chalabi A</span>, <span class="author">Shaw CE</span>, <span class="author">Tsolaki M</span>, <span class="author">Singleton AB</span>, <span class="author">Guerreiro R</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Moebus S</span>, <span class="author">J&ouml;ckel KH</span>, <span class="author">Klopp N</span>, <span class="author">Wichmann HE</span>, <span class="author">Carrasquillo MM</span>, <span class="author">Pankratz VS</span>, <span class="author">Younkin SG</span>, <span class="author">Jones L</span>, <span class="author">Holmans PA</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Owen MJ</span> and <span class="author">Williams J</span></p>
<p class="affiliation">MRC Centre for Neuropsychiatric Genetics and Genomics, Department of Psychological Medicine and Neurology, School of Medicine, Neuroscience and Mental Health Research Institute, Cardiff University, Cardiff, UK.</p><p class="abstract">Rare mutations in A&beta;PP, PSEN1, and PSEN2 cause uncommon early onset forms of Alzheimer&#39;s disease (AD), and common variants in MAPT are associated with risk of other neurodegenerative disorders. We sought to establish whether common genetic variation in these genes confer risk to the common form of AD which occurs later in life (&gt;65 years). We therefore tested single-nucleotide polymorphisms at these loci for association with late-onset AD (LOAD) in a large case-control sample consisting of 3,940 cases and 13,373 controls. Single-marker analysis did not identify any variants that reached genome-wide significance, a result which is supported by other recent genome-wide association studies. However, we did observe a significant association at the MAPT locus using a gene-wide approach (p = 0.009). We also observed suggestive association between AD and the marker rs9468, which defines the H1 haplotype, an extended haplotype that spans the MAPT gene and has previously been implicated in other neurodegenerative disorders including Parkinson&#39;s disease, progressive supranuclear palsy, and corticobasal degeneration. In summary common variants at A&beta;PP, PSEN1, and PSEN2 and MAPT are unlikely to make strong contributions to susceptibility for LOAD. However, the gene-wide effect observed at MAPT indicates a possible contribution to disease risk which requires further study.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G0700704/84698</span>; Chief Scientist Office; Medical Research Council: <span class="grant">G0601846</span>, <span class="grant">G0701075</span>, <span class="grant">G0900688</span>, <span class="grant">MC_U123160651</span>, <span class="grant">MC_U123160657</span>; NIA NIH HHS: <span class="grant">P01 AG003991</span>, <span class="grant">P50 AG005681</span>; Wellcome Trust: <span class="grant">095317</span></p>
<p class="publication"><span class="title">Journal of Alzheimer&#39;s disease : JAD</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">2</span>;<span class="pagerange">377-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22027014">22027014</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4118466">4118466</a>; DOI: <a href="http://dx.doi.org/10.3233%2FJAD-2011-110824">10.3233/JAD-2011-110824</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Haplotype analyses of haemoglobin C and haemoglobin s and the dynamics of the evolutionary response to malaria in kassena-nankana district of ghana.</h4>
<p class="authors"><span class="author">Ghansah A</span>, <span class="author">Rockett KA</span>, <span class="author">Clark TG</span>, <span class="author">Wilson MD</span>, <span class="author">Koram KA</span>, <span class="author">Oduro AR</span>, <span class="author">Amenga-Etego L</span>, <span class="author">Anyorigiya T</span>, <span class="author">Hodgson A</span>, <span class="author">Milligan P</span>, <span class="author">Rogers WO</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Noguchi Memorial Institute for Medical Research, University of Ghana, Accra, Ghana.</p><p class="abstract"><strong>Background</strong>: Haemoglobin S (HbS) and C (HbC) are variants of the HBB gene which both protect against malaria. It is not clear, however, how these two alleles have evolved in the West African countries where they co-exist at high frequencies. Here we use haplotypic signatures of selection to investigate the evolutionary history of the malaria-protective alleles HbS and HbC in the Kassena-Nankana District (KND) of Ghana.</p><p class="abstract">The haplotypic structure of HbS and HbC alleles was investigated, by genotyping 56 SNPs around the HBB locus. We found that, in the KND population, both alleles reside on extended haplotypes (approximately 1.5 Mb for HbS and 650 Kb for HbC) that are significantly less diverse than those of the ancestral HbA allele. The extended haplotypes span a recombination hotspot that is known to exist in this region of the genome</p><p class="abstract"><strong>Significance</strong>: Our findings show strong support for recent positive selection of both the HbS and HbC alleles and provide insights into how these two alleles have both evolved in the population of northern Ghana.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">4</span>;<span class="pagerange">e34565</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22506028" rel="external">22506028</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3323552" rel="external">3323552</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0034565" rel="external">10.1371/journal.pone.0034565</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">JAK2V617F homozygosity arises commonly and recurrently in PV and ET, but PV is characterized by expansion of a dominant homozygous subclone.</h4>
<p class="authors"><span class="author">Godfrey AL</span>, <span class="author">Chen E</span>, <span class="author">Pagano F</span>, <span class="author">Ortmann CA</span>, <span class="author">Silber Y</span>, <span class="author">Bellosillo B</span>, <span class="author">Guglielmelli P</span>, <span class="author">Harrison CN</span>, <span class="author">Reilly JT</span>, <span class="author">Stegelmann F</span>, <span class="author">Bijou F</span>, <span class="author">Lippert E</span>, <span class="author">McMullin MF</span>, <span class="author">Boiron JM</span>, <span class="author">D&ouml;hner K</span>, <span class="author">Vannucchi AM</span>, <span class="author">Besses C</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Cambridge Institute for Medical Research and Department of Haematology, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Subclones homozygous for JAK2V617F are more common in polycythemia vera (PV) than essential thrombocythemia (ET), but their prevalence and significance remain unclear. The JAK2 mutation status of 6495 BFU-E, grown in low erythropoietin conditions, was determined in 77 patients with PV or ET. Homozygous-mutant colonies were common in patients with JAK2V617F-positive PV and were surprisingly prevalent in JAK2V617F-positive ET and JAK2 exon 12-mutated PV. Using microsatellite PCR to map loss-of-heterozygosity breakpoints within individual colonies, we demonstrate that recurrent acquisition of JAK2V617F homozygosity occurs frequently in both PV and ET. PV was distinguished from ET by expansion of a dominant homozygous subclone, the selective advantage of which is likely to reflect additional genetic or epigenetic lesions. Our results suggest a model in which development of a dominant JAK2V617F-homzygous subclone drives erythrocytosis in many PV patients, with alternative mechanisms operating in those with small or undetectable homozygous-mutant clones.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2012</span>;<span class="volume">120</span>;<span class="number">13</span>;<span class="pagerange">2704-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22898600">22898600</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672970">3672970</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2012-05-431791">10.1182/blood-2012-05-431791</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Extensive compensatory cis-trans regulation in the evolution of mouse gene expression.</h4>
<p class="authors"><span class="author">Goncalves A</span>, <span class="author">Leigh-Brown S</span>, <span class="author">Thybert D</span>, <span class="author">Stefflova K</span>, <span class="author">Turro E</span>, <span class="author">Flicek P</span>, <span class="author">Brazma A</span>, <span class="author">Odom DT</span> and <span class="author">Marioni JC</span></p>
<p class="affiliation">European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, United Kingdom.</p><p class="abstract">Gene expression levels are thought to diverge primarily via regulatory mutations in trans within species, and in cis between species. To test this hypothesis in mammals we used RNA-sequencing to measure gene expression divergence between C57BL/6J and CAST/EiJ mouse strains and allele-specific expression in their F1 progeny. We identified 535 genes with parent-of-origin specific expression patterns, although few of these showed full allelic silencing. This suggests that the number of imprinted genes in a typical mouse somatic tissue is relatively small. In the set of nonimprinted genes, 32% showed evidence of divergent expression between the two strains. Of these, 2% could be attributed purely to variants acting in trans, while 43% were attributable only to variants acting in cis. The genes with expression divergence driven by changes in trans showed significantly higher sequence constraint than genes where the divergence was explained by variants acting in cis. The remaining genes with divergent patterns of expression (55%) were regulated by a combination of variants acting in cis and variants acting in trans. Intriguingly, the changes in expression induced by the cis and trans variants were in opposite directions more frequently than expected by chance, implying that compensatory regulation to stabilize gene expression levels is widespread. We propose that expression levels of genes regulated by this mechanism are fine-tuned by cis variants that arise following regulatory changes in trans, suggesting that many cis variants are not the primary targets of natural selection.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">15603</span>, <span class="grant">A15603</span>; European Research Council: <span class="grant">202218</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">12</span>;<span class="pagerange">2376-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22919075">22919075</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3514667">3514667</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.142281.112">10.1101/gr.142281.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Estimation of rearrangement phylogeny for cancer genomes.</h4>
<p class="authors"><span class="author">Greenman CD</span>, <span class="author">Pleasance ED</span>, <span class="author">Newman S</span>, <span class="author">Yang F</span>, <span class="author">Fu B</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Jones D</span>, <span class="author">Lau KW</span>, <span class="author">Carter N</span>, <span class="author">Edwards PA</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%43.%47%72%65%65%6E%6D%61%6E%40%75%65%61.%61%63.%75%6B">&#67;&#46;&#71;&#114;&#101;&#101;&#110;&#109;&#97;&#110;&#64;&#117;&#101;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Cancer genomes are complex, carrying thousands of somatic mutations including base substitutions, insertions and deletions, rearrangements, and copy number changes that have been acquired over decades. Recently, technologies have been introduced that allow generation of high-resolution, comprehensive catalogs of somatic alterations in cancer genomes. However, analyses of these data sets generally do not indicate the order in which mutations have occurred, or the resulting karyotype. Here, we introduce a mathematical framework that begins to address this problem. By using samples with accurate data sets, we can reconstruct relatively complex temporal sequences of rearrangements and provide an assembly of genomic segments into digital karyotypes. For cancer genes mutated in rearranged regions, this information can provide a chronological examination of the selective events that have taken place.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">2</span>;<span class="pagerange">346-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21994251" rel="external">21994251</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3266042" rel="external">3266042</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.118414.110" rel="external">10.1101/gr.118414.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comprehensive Exploration of the Effects of miRNA SNPs on Monocyte Gene Expression.</h4>
<p class="authors"><span class="author">Greliche N</span>, <span class="author">Zeller T</span>, <span class="author">Wild PS</span>, <span class="author">Rotival M</span>, <span class="author">Schillert A</span>, <span class="author">Ziegler A</span>, <span class="author">Deloukas P</span>, <span class="author">Erdmann J</span>, <span class="author">Hengstenberg C</span>, <span class="author">Ouwehand WH</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Munzel T</span>, <span class="author">Lackner KJ</span>, <span class="author">Cambien F</span>, <span class="author">Goodall AH</span>, <span class="author">Tiret L</span>, <span class="author">Blankenberg S</span>, <span class="author">Tr&eacute;gou&euml;t DA</span> and <span class="author">Cardiogenics Consortium</span></p>
<p class="affiliation">INSERM UMR_S 937, Pierre and Marie Curie University (UPMC, Paris 6), Paris, France ; Université Paris-Sud, Paris, France.</p>
<p class="abstract">We aimed to assess whether pri-miRNA SNPs (miSNPs) could influence monocyte gene expression, either through marginal association or by interacting with polymorphisms located in 3'UTR regions (3utrSNPs). We then conducted a genome-wide search for marginal miSNPs effects and pairwise miSNPs × 3utrSNPs interactions in a sample of 1,467 individuals for which genome-wide monocyte expression and genotype data were available. Statistical associations that survived multiple testing correction were tested for replication in an independent sample of 758 individuals with both monocyte gene expression and genotype data. In both studies, the hsa-mir-1279 rs1463335 was found to modulate in cis the expression of LYZ and in trans the expression of CNTN6, CTRC, COPZ2, KRT9, LRRFIP1, NOD1, PCDHA6, ST5 and TRAF3IP2 genes, supporting the role of hsa-mir-1279 as a regulator of several genes in monocytes. In addition, we identified two robust miSNPs × 3utrSNPs interactions, one involving HLA-DPB1 rs1042448 and hsa-mir-219-1 rs107822, the second the H1F0 rs1894644 and hsa-mir-659 rs5750504, modulating the expression of the associated genes.As some of the aforementioned genes have previously been reported to reside at disease-associated loci, our findings provide novel arguments supporting the hypothesis that the genetic variability of miRNAs could also contribute to the susceptibility to human diseases.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e45863</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23029284" rel="external">23029284</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3448685" rel="external">3448685</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0045863" rel="external">10.1371/journal.pone.0045863</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of cytoplasmic nucleophosmin expression by imaging flow cytometry.</h4>
<p class="authors"><span class="author">Grimwade L</span>, <span class="author">Gudgin E</span>, <span class="author">Bloxham D</span>, <span class="author">Bottley G</span>, <span class="author">Vassiliou G</span>, <span class="author">Huntly B</span>, <span class="author">Scott MA</span> and <span class="author">Erber WN</span></p>
<p class="affiliation">Haemato-Oncology Diagnostics Service, Department of Haematology, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom. <a href="mailto:%6C%69%7A%7A.%67%72%69%6D%77%61%64%65%40%61%64%64%65%6E%62%72%6F%6F%6B%65%73.%6E%68%73.%75%6B">&#108;&#105;&#122;&#122;&#46;&#103;&#114;&#105;&#109;&#119;&#97;&#100;&#101;&#64;&#97;&#100;&#100;&#101;&#110;&#98;&#114;&#111;&#111;&#107;&#101;&#115;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract">Mutations within the nucleophosmin NPM1 gene occur in approximately one-third of cases of acute myeloid leukemia (AML). These mutations result in cytoplasmic accumulation of the mutant NPM protein. NPM1 mutations are currently detected by molecular methods. Using samples from 37 AML patients, we investigated whether imaging flow cytometry could be a viable alternative to this current technique. Bone marrow/peripheral blood cells were stained with anti-NPM antibody and DRAQ5 nuclear stain, and data were acquired on an ImageStream imaging flow cytometer (Amnis Corp., Seattle, USA). Using the similarity feature for data analysis, we demonstrated that this technique could successfully identify cases of AML with a NPM1 mutation based on cytoplasmic NPM protein staining (at similarity threshold of 1.1 sensitivity 88% and specificity 90%). Combining data of mean fluorescence intensity and % dissimilar staining in a 0-2 scoring system further improved the sensitivity (100%). Imaging flow cytometry has the potential to be included as part of a standard flow cytometry antibody panel to identify potential NPM1 mutations as part of diagnosis and minimal residual disease monitoring. Imaging flow cytometry is an exciting technology that has many possible applications in the diagnosis of hematological malignancies, including the potential to integrate modalities.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095663</span></p>
<p class="publication"><span class="title">Cytometry. Part A : the journal of the International Society for Analytical Cytology</span>&nbsp;<span class="year">2012</span>;<span class="volume">81</span>;<span class="number">10</span>;<span class="pagerange">896-900</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22968966">22968966</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fcyto.a.22116">10.1002/cyto.a.22116</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analyses of pig genomes provide insight into porcine demography and evolution.</h4>
<p class="authors"><span class="author">Groenen MA</span>, <span class="author">Archibald AL</span>, <span class="author">Uenishi H</span>, <span class="author">Tuggle CK</span>, <span class="author">Takeuchi Y</span>, <span class="author">Rothschild MF</span>, <span class="author">Rogel-Gaillard C</span>, <span class="author">Park C</span>, <span class="author">Milan D</span>, <span class="author">Megens HJ</span>, <span class="author">Li S</span>, <span class="author">Larkin DM</span>, <span class="author">Kim H</span>, <span class="author">Frantz LA</span>, <span class="author">Caccamo M</span>, <span class="author">Ahn H</span>, <span class="author">Aken BL</span>, <span class="author">Anselmo A</span>, <span class="author">Anthon C</span>, <span class="author">Auvil L</span>, <span class="author">Badaoui B</span>, <span class="author">Beattie CW</span>, <span class="author">Bendixen C</span>, <span class="author">Berman D</span>, <span class="author">Blecha F</span>, <span class="author">Blomberg J</span>, <span class="author">Bolund L</span>, <span class="author">Bosse M</span>, <span class="author">Botti S</span>, <span class="author">Bujie Z</span>, <span class="author">Bystrom M</span>, <span class="author">Capitanu B</span>, <span class="author">Carvalho-Silva D</span>, <span class="author">Chardon P</span>, <span class="author">Chen C</span>, <span class="author">Cheng R</span>, <span class="author">Choi SH</span>, <span class="author">Chow W</span>, <span class="author">Clark RC</span>, <span class="author">Clee C</span>, <span class="author">Crooijmans RP</span>, <span class="author">Dawson HD</span>, <span class="author">Dehais P</span>, <span class="author">De Sapio F</span>, <span class="author">Dibbits B</span>, <span class="author">Drou N</span>, <span class="author">Du ZQ</span>, <span class="author">Eversole K</span>, <span class="author">Fadista J</span>, <span class="author">Fairley S</span>, <span class="author">Faraut T</span>, <span class="author">Faulkner GJ</span>, <span class="author">Fowler KE</span>, <span class="author">Fredholm M</span>, <span class="author">Fritz E</span>, <span class="author">Gilbert JG</span>, <span class="author">Giuffra E</span>, <span class="author">Gorodkin J</span>, <span class="author">Griffin DK</span>, <span class="author">Harrow JL</span>, <span class="author">Hayward A</span>, <span class="author">Howe K</span>, <span class="author">Hu ZL</span>, <span class="author">Humphray SJ</span>, <span class="author">Hunt T</span>, <span class="author">Hornsh&oslash;j H</span>, <span class="author">Jeon JT</span>, <span class="author">Jern P</span>, <span class="author">Jones M</span>, <span class="author">Jurka J</span>, <span class="author">Kanamori H</span>, <span class="author">Kapetanovic R</span>, <span class="author">Kim J</span>, <span class="author">Kim JH</span>, <span class="author">Kim KW</span>, <span class="author">Kim TH</span>, <span class="author">Larson G</span>, <span class="author">Lee K</span>, <span class="author">Lee KT</span>, <span class="author">Leggett R</span>, <span class="author">Lewin HA</span>, <span class="author">Li Y</span>, <span class="author">Liu W</span>, <span class="author">Loveland JE</span>, <span class="author">Lu Y</span>, <span class="author">Lunney JK</span>, <span class="author">Ma J</span>, <span class="author">Madsen O</span>, <span class="author">Mann K</span>, <span class="author">Matthews L</span>, <span class="author">McLaren S</span>, <span class="author">Morozumi T</span>, <span class="author">Murtaugh MP</span>, <span class="author">Narayan J</span>, <span class="author">Nguyen DT</span>, <span class="author">Ni P</span>, <span class="author">Oh SJ</span>, <span class="author">Onteru S</span>, <span class="author">Panitz F</span>, <span class="author">Park EW</span>, <span class="author">Park HS</span>, <span class="author">Pascal G</span>, <span class="author">Paudel Y</span>, <span class="author">Perez-Enciso M</span>, <span class="author">Ramirez-Gonzalez R</span>, <span class="author">Reecy JM</span>, <span class="author">Rodriguez-Zas S</span>, <span class="author">Rohrer GA</span>, <span class="author">Rund L</span>, <span class="author">Sang Y</span>, <span class="author">Schachtschneider K</span>, <span class="author">Schraiber JG</span>, <span class="author">Schwartz J</span>, <span class="author">Scobie L</span>, <span class="author">Scott C</span>, <span class="author">Searle S</span>, <span class="author">Servin B</span>, <span class="author">Southey BR</span>, <span class="author">Sperber G</span>, <span class="author">Stadler P</span>, <span class="author">Sweedler JV</span>, <span class="author">Tafer H</span>, <span class="author">Thomsen B</span>, <span class="author">Wali R</span>, <span class="author">Wang J</span>, <span class="author">Wang J</span>, <span class="author">White S</span>, <span class="author">Xu X</span>, <span class="author">Yerle M</span>, <span class="author">Zhang G</span>, <span class="author">Zhang J</span>, <span class="author">Zhang J</span>, <span class="author">Zhao S</span>, <span class="author">Rogers J</span>, <span class="author">Churcher C</span> and <span class="author">Schook LB</span></p>
<p class="affiliation">Animal Breeding and Genomics Centre, Wageningen University, De Elst 1, 6708 WD, Wageningen, The Netherlands. <a href="mailto:%6D%61%72%74%69%65%6E.%67%72%6F%65%6E%65%6E%40%77%75%72.%6E%6C">&#109;&#97;&#114;&#116;&#105;&#101;&#110;&#46;&#103;&#114;&#111;&#101;&#110;&#101;&#110;&#64;&#119;&#117;&#114;&#46;&#110;&#108;</a></p><p class="abstract">For 10,000&thinsp;years pigs and humans have shared a close and complex relationship. From domestication to modern breeding practices, humans have shaped the genomes of domestic pigs. Here we present the assembly and analysis of the genome sequence of a female domestic Duroc pig (Sus scrofa) and a comparison with the genomes of wild and domestic pigs from Europe and Asia. Wild pigs emerged in South East Asia and subsequently spread across Eurasia. Our results reveal a deep phylogenetic split between European and Asian wild boars &sim;1 million years ago, and a selective sweep analysis indicates selection on genes involved in RNA processing and regulation. Genes associated with immune response and olfaction exhibit fast evolution. Pigs have the largest repertoire of functional olfactory receptor genes, reflecting the importance of smell in this scavenging animal. The pig genome sequence provides an important resource for further improvements of this important livestock species, and our identification of many putative disease-causing variants extends the potential of the pig as a biomedical model.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E010520/1</span>, <span class="grant">BB/E010520/2</span>, <span class="grant">BB/E011640/1</span>, <span class="grant">BB/G004013/1</span>, <span class="grant">BB/H005935/1</span>, <span class="grant">BB/I025328/1</span>, <span class="grant">BB/I025360/1</span>; European Research Council: <span class="grant">249894</span>; Medical Research Council: <span class="grant">G0900950</span>; NCRR NIH HHS: <span class="grant">P20-RR017686</span>, <span class="grant">R13 RR020283A</span>, <span class="grant">R13 RR032267A</span>; NHGRI NIH HHS: <span class="grant">R21 HG006464</span>; NIAID NIH HHS: <span class="grant">T32 AI083196</span>; NIDA NIH HHS: <span class="grant">P30 DA018310</span>, <span class="grant">R21 DA027548</span>; NLM NIH HHS: <span class="grant">5 P41 LM006252</span>, <span class="grant">5 P41LM006252</span>; Wellcome Trust: <span class="grant">095908</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">491</span>;<span class="number">7424</span>;<span class="pagerange">393-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23151582">23151582</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3566564">3566564</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11622">10.1038/nature11622</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping cis- and trans-regulatory effects across multiple tissues in twins.</h4>
<p class="authors"><span class="author">Grundberg E</span>, <span class="author">Small KS</span>, <span class="author">Hedman &Aring;K</span>, <span class="author">Nica AC</span>, <span class="author">Buil A</span>, <span class="author">Keildson S</span>, <span class="author">Bell JT</span>, <span class="author">Yang TP</span>, <span class="author">Meduri E</span>, <span class="author">Barrett A</span>, <span class="author">Nisbett J</span>, <span class="author">Sekowska M</span>, <span class="author">Wilk A</span>, <span class="author">Shin SY</span>, <span class="author">Glass D</span>, <span class="author">Travers M</span>, <span class="author">Min JL</span>, <span class="author">Ring S</span>, <span class="author">Ho K</span>, <span class="author">Thorleifsson G</span>, <span class="author">Kong A</span>, <span class="author">Thorsteindottir U</span>, <span class="author">Ainali C</span>, <span class="author">Dimas AS</span>, <span class="author">Hassanali N</span>, <span class="author">Ingle C</span>, <span class="author">Knowles D</span>, <span class="author">Krestyaninova M</span>, <span class="author">Lowe CE</span>, <span class="author">Di Meglio P</span>, <span class="author">Montgomery SB</span>, <span class="author">Parts L</span>, <span class="author">Potter S</span>, <span class="author">Surdulescu G</span>, <span class="author">Tsaprouni L</span>, <span class="author">Tsoka S</span>, <span class="author">Bataille V</span>, <span class="author">Durbin R</span>, <span class="author">Nestle FO</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Soranzo N</span>, <span class="author">Lindgren CM</span>, <span class="author">Zondervan KT</span>, <span class="author">Ahmadi KR</span>, <span class="author">Schadt EE</span>, <span class="author">Stefansson K</span>, <span class="author">Smith GD</span>, <span class="author">McCarthy MI</span>, <span class="author">Deloukas P</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Spector TD</span> and <span class="author">Multiple Tissue Human Expression Resource (MuTHER) Consortium</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p><p class="abstract">Sequence-based variation in gene expression is a key driver of disease risk. Common variants regulating expression in cis have been mapped in many expression quantitative trait locus (eQTL) studies, typically in single tissues from unrelated individuals. Here, we present a comprehensive analysis of gene expression across multiple tissues conducted in a large set of mono- and dizygotic twins that allows systematic dissection of genetic (cis and trans) and non-genetic effects on gene expression. Using identity-by-descent estimates, we show that at least 40% of the total heritable cis effect on expression cannot be accounted for by common cis variants, a finding that reveals the contribution of low-frequency and rare regulatory variants with respect to both transcriptional regulation and complex trait susceptibility. We show that a substantial proportion of gene expression heritability is trans to the structural gene, and we identify several replicating trans variants that act predominantly in a tissue-restricted manner and may regulate the transcription of many genes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900339</span>, <span class="grant">G9815508</span>; Wellcome Trust: <span class="grant">081917/Z/07/Z</span>, <span class="grant">085235</span>, <span class="grant">090532</span>, <span class="grant">092731</span>, <span class="grant">095515</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">10</span>;<span class="pagerange">1084-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22941192">22941192</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3784328">3784328</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2394">10.1038/ng.2394</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chado controller: advanced annotation management with a community annotation system.</h4>
<p class="authors"><span class="author">Guignon V</span>, <span class="author">Droc G</span>, <span class="author">Alaux M</span>, <span class="author">Baurens FC</span>, <span class="author">Garsmeur O</span>, <span class="author">Poiron C</span>, <span class="author">Carver T</span>, <span class="author">Rouard M</span> and <span class="author">Bocs S</span></p>
<p class="affiliation">CIRAD, UMR AGAP, F-34398 Montpellier, France. <a href="mailto:%76%61%6C%65%6E%74%69%6E.%67%75%69%67%6E%6F%6E%40%63%69%72%61%64.%66%72">&#118;&#97;&#108;&#101;&#110;&#116;&#105;&#110;&#46;&#103;&#117;&#105;&#103;&#110;&#111;&#110;&#64;&#99;&#105;&#114;&#97;&#100;&#46;&#102;&#114;</a></p><p class="abstract"><strong>Summary</strong>: We developed a controller that is compliant with the Chado database schema, GBrowse and genome annotation-editing tools such as Artemis and Apollo. It enables the management of public and private data, monitors manual annotation (with controlled vocabularies, structural and functional annotation controls) and stores versions of annotation for all modified features. The Chado controller uses PostgreSQL and Perl.</p><p class="abstract"><strong>Availability</strong>: The Chado Controller package is available for download at http://www.gnpannot.org/content/chado-controller and runs on any Unix-like operating system, and documentation is available at http://www.gnpannot.org/content/chado-controller-doc The system can be tested using the GNPAnnot Sandbox at http://www.gnpannot.org/content/gnpannot-sandbox-form</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%76%61%6C%65%6E%74%69%6E.%67%75%69%67%6E%6F%6E%40%63%69%72%61%64.%66%72">&#118;&#97;&#108;&#101;&#110;&#116;&#105;&#110;&#46;&#103;&#117;&#105;&#103;&#110;&#111;&#110;&#64;&#99;&#105;&#114;&#97;&#100;&#46;&#102;&#114;</a>; <a href="mailto:%73%74%65%70%68%61%6E%69%65.%73%69%64%69%62%65-%62%6F%63%73%40%63%69%72%61%64.%66%72">&#115;&#116;&#101;&#112;&#104;&#97;&#110;&#105;&#101;&#46;&#115;&#105;&#100;&#105;&#98;&#101;&#45;&#98;&#111;&#99;&#115;&#64;&#99;&#105;&#114;&#97;&#100;&#46;&#102;&#114;</a></p><p class="abstract">Supplementary data are available at Bioinformatics online.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">7</span>;<span class="pagerange">1054-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22285827" rel="external">22285827</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3315714" rel="external">3315714</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts046" rel="external">10.1093/bioinformatics/bts046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lipoprotein(a) and risk of coronary, cerebrovascular, and peripheral artery disease: the EPIC-Norfolk prospective population study.</h4>
<p class="authors"><span class="author">Gurdasani D</span>, <span class="author">Sjouke B</span>, <span class="author">Tsimikas S</span>, <span class="author">Hovingh GK</span>, <span class="author">Luben RN</span>, <span class="author">Wainwright NW</span>, <span class="author">Pomilla C</span>, <span class="author">Wareham NJ</span>, <span class="author">Khaw KT</span>, <span class="author">Boekholdt SM</span> and <span class="author">Sandhu MS</span></p>
<p class="affiliation">Department of Public Health and Primary Care, Institute of Public Health, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract"><strong>Objective</strong>: Although the association between circulating levels of lipoprotein(a) [Lp(a)] and risk of coronary artery disease (CAD) and stroke is well established, its role in risk of peripheral arterial disease (PAD) remains unclear. Here, we examine the association between Lp(a) levels and PAD in a large prospective cohort. To contextualize these findings, we also examined the association between Lp(a) levels and risk of stroke and CAD and studied the role of low-density lipoprotein as an effect modifier of Lp(a)-associated cardiovascular risk.</p><p class="abstract">Lp(a) levels were measured in apparently healthy participants in the European Prospective Investigation of Cancer (EPIC)-Norfolk cohort. Cox regression was used to quantify the association between Lp(a) levels and risk of PAD, stroke, and CAD outcomes. During 212 981 person-years at risk, a total of 2365 CAD, 284 ischemic stroke, and 596 PAD events occurred in 18 720 participants. Lp(a) was associated with PAD and CAD outcomes but not with ischemic stroke (hazard ratio per 2.7-fold increase in Lp(a) of 1.37, 95% CI 1.25-1.50, 1.13, 95% CI 1.04-1.22 and 0.91, 95% CI 0.79-1.03, respectively). Low-density lipoprotein cholesterol levels did not modify these associations.</p><p class="abstract"><strong>Conclusions</strong>: Lp(a) levels were associated with future PAD and CAD events. The association between Lp(a) and cardiovascular disease was not modified by low-density lipoprotein cholesterol levels.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801566</span></p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">32</span>;<span class="number">12</span>;<span class="pagerange">3058-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23065826">23065826</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.112.255521">10.1161/ATVBAHA.112.255521</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeted deletion of microRNA-22 promotes stress-induced cardiac dilation and contractile dysfunction.</h4>
<p class="authors"><span class="author">Gurha P</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">Wang T</span>, <span class="author">Ramirez MO</span>, <span class="author">Drumond AL</span>, <span class="author">van Dongen S</span>, <span class="author">Chen Y</span>, <span class="author">Bartonicek N</span>, <span class="author">Enright AJ</span>, <span class="author">Lee B</span>, <span class="author">Kelm RJ</span>, <span class="author">Reddy AK</span>, <span class="author">Taffet GE</span>, <span class="author">Bradley A</span>, <span class="author">Wehrens XH</span>, <span class="author">Entman ML</span> and <span class="author">Rodriguez A</span></p>
<p class="affiliation">Baylor College of Medicine, Department of Molecular and Human Genetics, One Baylor Plaza, Houston, TX, 77030, USA.</p>
<p class="abstract">Background: Delineating the role of microRNAs (miRNAs) in the posttranscriptional gene regulation offers new insights into how the heart adapts to pathological stress. We developed a knockout of miR-22 in mice and investigated its function in the heart.

Here, we show that miR-22-deficient mice are impaired in inotropic and lusitropic response to acute stress by dobutamine. Furthermore, the absence of miR-22 sensitized mice to cardiac decompensation and left ventricular dilation after long-term stimulation by pressure overload. Calcium transient analysis revealed reduced sarcoplasmic reticulum Ca(2+) load in association with repressed sarcoplasmic reticulum Ca(2+) ATPase activity in mutant myocytes. Genetic ablation of miR-22 also led to a decrease in cardiac expression levels for Serca2a and muscle-restricted genes encoding proteins in the vicinity of the cardiac Z disk/titin cytoskeleton. These phenotypes were attributed in part to inappropriate repression of serum response factor activity in stressed hearts. Global analysis revealed increased expression of the transcriptional/translational repressor purine-rich element binding protein B, a highly conserved miR-22 target implicated in the negative control of muscle expression.

Conclusion: These data indicate that miR-22 functions as an integrator of Ca(2+) homeostasis and myofibrillar protein content during stress in the heart and shed light on the mechanisms that enhance propensity toward heart failure.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL089598</span>, <span class="grant">HL089792</span>, <span class="grant">HL091947</span>, <span class="grant">K25 HL73041</span>, <span class="grant">R01 HL22512</span></p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2012</span>;<span class="volume">125</span>;<span class="number">22</span>;<span class="pagerange">2751-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22570371" rel="external">22570371</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.111.044354" rel="external">10.1161/CIRCULATIONAHA.111.044354</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Afghanistan&#39;s ethnic groups share a Y-chromosomal heritage structured by historical events.</h4>
<p class="authors"><span class="author">Haber M</span>, <span class="author">Platt DE</span>, <span class="author">Ashrafian Bonab M</span>, <span class="author">Youhanna SC</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Mart&iacute;nez-Cruz B</span>, <span class="author">Douaihy B</span>, <span class="author">Ghassibe-Sabbagh M</span>, <span class="author">Rafatpanah H</span>, <span class="author">Ghanbari M</span>, <span class="author">Whale J</span>, <span class="author">Balanovsky O</span>, <span class="author">Wells RS</span>, <span class="author">Comas D</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Zalloua PA</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">The Lebanese American University, Chouran, Beirut, Lebanon.</p><p class="abstract">Afghanistan has held a strategic position throughout history. It has been inhabited since the Paleolithic and later became a crossroad for expanding civilizations and empires. Afghanistan&#39;s location, history, and diverse ethnic groups present a unique opportunity to explore how nations and ethnic groups emerged, and how major cultural evolutions and technological developments in human history have influenced modern population structures. In this study we have analyzed, for the first time, the four major ethnic groups in present-day Afghanistan: Hazara, Pashtun, Tajik, and Uzbek, using 52 binary markers and 19 short tandem repeats on the non-recombinant segment of the Y-chromosome. A total of 204 Afghan samples were investigated along with more than 8,500 samples from surrounding populations important to Afghanistan&#39;s history through migrations and conquests, including Iranians, Greeks, Indians, Middle Easterners, East Europeans, and East Asians. Our results suggest that all current Afghans largely share a heritage derived from a common unstructured ancestral population that could have emerged during the Neolithic revolution and the formation of the first farming communities. Our results also indicate that inter-Afghan differentiation started during the Bronze Age, probably driven by the formation of the first civilizations in the region. Later migrations and invasions into the region have been assimilated differentially among the ethnic groups, increasing inter-population genetic differences, and giving the Afghans a unique genetic diversity in Central Asia.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">e34288</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22470552">22470552</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3314501">3314501</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0034288">10.1371/journal.pone.0034288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Toward a roadmap in global biobanking for health.</h4>
<p class="authors"><span class="author">Harris JR</span>, <span class="author">Burton P</span>, <span class="author">Knoppers BM</span>, <span class="author">Lindpaintner K</span>, <span class="author">Bledsoe M</span>, <span class="author">Brookes AJ</span>, <span class="author">Budin-Lj&oslash;sne I</span>, <span class="author">Chisholm R</span>, <span class="author">Cox D</span>, <span class="author">Desch&ecirc;nes M</span>, <span class="author">Fortier I</span>, <span class="author">Hainaut P</span>, <span class="author">Hewitt R</span>, <span class="author">Kaye J</span>, <span class="author">Litton JE</span>, <span class="author">Metspalu A</span>, <span class="author">Ollier B</span>, <span class="author">Palmer LJ</span>, <span class="author">Palotie A</span>, <span class="author">Pasterk M</span>, <span class="author">Perola M</span>, <span class="author">Riegman PH</span>, <span class="author">van Ommen GJ</span>, <span class="author">Yuille M</span> and <span class="author">Zatloukal K</span></p>
<p class="affiliation">Department of Genes and Environment, Division of Epidemiology, The Norwegian Institute of Public Health, Oslo, Norway.</p>
<p class="abstract">Biobanks can have a pivotal role in elucidating disease etiology, translation, and advancing public health. However, meeting these challenges hinges on a critical shift in the way science is conducted and requires biobank harmonization. There is growing recognition that a common strategy is imperative to develop biobanking globally and effectively. To help guide this strategy, we articulate key principles, goals, and priorities underpinning a roadmap for global biobanking to accelerate health science, patient care, and public health. The need to manage and share very large amounts of data has driven innovations on many fronts. Although technological solutions are allowing biobanks to reach new levels of integration, increasingly powerful data-collection tools, analytical techniques, and the results they generate raise new ethical and legal issues and challenges, necessitating a reconsideration of previous policies, practices, and ethical norms. These manifold advances and the investments that support them are also fueling opportunities for biobanks to ultimately become integral parts of health-care systems in many countries. International harmonization to increase interoperability and sustainability are two strategic priorities for biobanking. Tackling these issues requires an environment favorably inclined toward scientific funding and equipped to address socio-ethical challenges. Cooperation and collaboration must extend beyond systems to enable the exchange of data and samples to strategic alliances between many organizations, including governmental bodies, funding agencies, public and private science enterprises, and other stakeholders, including patients. A common vision is required and we articulate the essential basis of such a vision herein.European Journal of Human Genetics advance online publication, 20 June 2012; doi:10.1038/ejhg.2012.96.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22713808" rel="external">22713808</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2012.96" rel="external">10.1038/ejhg.2012.96</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome analysis of diverse Chlamydia trachomatis strains identifies phylogenetic relationships masked by current clinical typing.</h4>
<p class="authors"><span class="author">Harris SR</span>, <span class="author">Clarke IN</span>, <span class="author">Seth-Smith HM</span>, <span class="author">Solomon AW</span>, <span class="author">Cutcliffe LT</span>, <span class="author">Marsh P</span>, <span class="author">Skilton RJ</span>, <span class="author">Holland MJ</span>, <span class="author">Mabey D</span>, <span class="author">Peeling RW</span>, <span class="author">Lewis DA</span>, <span class="author">Spratt BG</span>, <span class="author">Unemo M</span>, <span class="author">Persson K</span>, <span class="author">Bjartling C</span>, <span class="author">Brunham R</span>, <span class="author">de Vries HJ</span>, <span class="author">Morr&eacute; SA</span>, <span class="author">Speksnijder A</span>, <span class="author">B&eacute;b&eacute;ar CM</span>, <span class="author">Clerc M</span>, <span class="author">de Barbeyrac B</span>, <span class="author">Parkhill J</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Pathogen Genomics, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%73%68%31%36%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#104;&#49;&#54;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chlamydia trachomatis is responsible for both trachoma and sexually transmitted infections, causing substantial morbidity and economic cost globally. Despite this, our knowledge of its population and evolutionary genetics is limited. Here we present a detailed phylogeny based on whole-genome sequencing of representative strains of C. trachomatis from both trachoma and lymphogranuloma venereum (LGV) biovars from temporally and geographically diverse sources. Our analysis shows that predicting phylogenetic structure using ompA, which is traditionally used to classify Chlamydia, is misleading because extensive recombination in this region masks any true relationships present. We show that in many instances, ompA is a chimera that can be exchanged in part or as a whole both within and between biovars. We also provide evidence for exchange of, and recombination within, the cryptic plasmid, which is another key diagnostic target. We used our phylogenetic framework to show how genetic exchange has manifested itself in ocular, urogenital and LGV C. trachomatis strains, including the epidemic LGV serotype L2b.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">080348</span>, <span class="grant">089472</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">4</span>;<span class="pagerange">413-9, S1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22406642">22406642</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3378690">3378690</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2214">10.1038/ng.2214</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GENCODE: the reference human genome annotation for The ENCODE Project.</h4>
<p class="authors"><span class="author">Harrow J</span>, <span class="author">Frankish A</span>, <span class="author">Gonzalez JM</span>, <span class="author">Tapanari E</span>, <span class="author">Diekhans M</span>, <span class="author">Kokocinski F</span>, <span class="author">Aken BL</span>, <span class="author">Barrell D</span>, <span class="author">Zadissa A</span>, <span class="author">Searle S</span>, <span class="author">Barnes I</span>, <span class="author">Bignell A</span>, <span class="author">Boychenko V</span>, <span class="author">Hunt T</span>, <span class="author">Kay M</span>, <span class="author">Mukherjee G</span>, <span class="author">Rajan J</span>, <span class="author">Despacio-Reyes G</span>, <span class="author">Saunders G</span>, <span class="author">Steward C</span>, <span class="author">Harte R</span>, <span class="author">Lin M</span>, <span class="author">Howald C</span>, <span class="author">Tanzer A</span>, <span class="author">Derrien T</span>, <span class="author">Chrast J</span>, <span class="author">Walters N</span>, <span class="author">Balasubramanian S</span>, <span class="author">Pei B</span>, <span class="author">Tress M</span>, <span class="author">Rodriguez JM</span>, <span class="author">Ezkurdia I</span>, <span class="author">van Baren J</span>, <span class="author">Brent M</span>, <span class="author">Haussler D</span>, <span class="author">Kellis M</span>, <span class="author">Valencia A</span>, <span class="author">Reymond A</span>, <span class="author">Gerstein M</span>, <span class="author">Guig&oacute; R</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6A%6C%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#108;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The GENCODE Consortium aims to identify all gene features in the human genome using a combination of computational analysis, manual annotation, and experimental validation. Since the first public release of this annotation data set, few new protein-coding loci have been added, yet the number of alternative splicing transcripts annotated has steadily increased. The GENCODE 7 release contains 20,687 protein-coding and 9640 long noncoding RNA loci and has 33,977 coding transcripts not represented in UCSC genes and RefSeq. It also has the most comprehensive annotation of long noncoding RNA (lncRNA) loci publicly available with the predominant transcript form consisting of two exons. We have examined the completeness of the transcript annotation and found that 35% of transcriptional start sites are supported by CAGE clusters and 62% of protein-coding genes have annotated polyA sites. Over one-third of GENCODE protein-coding genes are supported by peptide hits derived from mass spectrometry spectra submitted to Peptide Atlas. New models derived from the Illumina Body Map 2.0 RNA-seq data identify 3689 new loci not currently in GENCODE, of which 3127 consist of two exon models indicating that they are possibly unannotated long noncoding loci. GENCODE 7 is publicly available from gencodegenes.org and via the Ensembl and UCSC Genome Browsers.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5U54HG004555</span>; Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">9</span>;<span class="pagerange">1760-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955987">22955987</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3431492">3431492</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.135350.111">10.1101/gr.135350.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tracking and coordinating an international curation effort for the CCDS Project.</h4>
<p class="authors"><span class="author">Harte RA</span>, <span class="author">Farrell CM</span>, <span class="author">Loveland JE</span>, <span class="author">Suner MM</span>, <span class="author">Wilming L</span>, <span class="author">Aken B</span>, <span class="author">Barrell D</span>, <span class="author">Frankish A</span>, <span class="author">Wallin C</span>, <span class="author">Searle S</span>, <span class="author">Diekhans M</span>, <span class="author">Harrow J</span> and <span class="author">Pruitt KD</span></p>
<p class="affiliation">Center for Biomolecular Science and Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA.</p><p class="abstract">The Consensus Coding Sequence (CCDS) collaboration involves curators at multiple centers with a goal of producing a conservative set of high quality, protein-coding region annotations for the human and mouse reference genome assemblies. The CCDS data set reflects a &#39;gold standard&#39; definition of best supported protein annotations, and corresponding genes, which pass a standard series of quality assurance checks and are supported by manual curation. This data set supports use of genome annotation information by human and mouse researchers for effective experimental design, analysis and interpretation. The CCDS project consists of analysis of automated whole-genome annotation builds to identify identical CDS annotations, quality assurance testing and manual curation support. Identical CDS annotations are tracked with a CCDS identifier (ID) and any future change to the annotated CDS structure must be agreed upon by the collaborating members. CCDS curation guidelines were developed to address some aspects of curation in order to improve initial annotation consistency and to reduce time spent in discussing proposed annotation updates. Here, we present the current status of the CCDS database and details on our procedures to track and coordinate our efforts. We also present the relevant background and reasoning behind the curation standards that we have developed for CCDS database treatment of transcripts that are nonsense-mediated decay (NMD) candidates, for transcripts containing upstream open reading frames, for identifying the most likely translation start codons and for the annotation of readthrough transcripts. Examples are provided to illustrate the application of these guidelines. DATABASE URL: http://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5U54 HG004555</span>, <span class="grant">5U54HG00455-04</span>; Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT062023</span>, <span class="grant">WT077198</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2012</span>;<span class="volume">2012</span>;<span class="pagerange">bas008</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22434842">22434842</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308164">3308164</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbas008">10.1093/database/bas008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The future of technologies for personalised medicine.</h4>
<p class="authors"><span class="author">Harvey A</span>, <span class="author">Brand A</span>, <span class="author">Holgate ST</span>, <span class="author">Kristiansen LV</span>, <span class="author">Lehrach H</span>, <span class="author">Palotie A</span> and <span class="author">Prainsack B</span></p>
<p class="affiliation">Finchley Court, Norwich, UK.</p>
<p class="abstract">Personalised medicine promises prediction, prevention and treatment of illness that is targeted to individualsâ€™ needs. New technologies for detailed biological profiling of individuals at the molecular level have been crucial in initiating the move to personalised medicine; further novel technologies will be necessary if the vision is to become a reality. We will need to develop new technologies to collect and analyse data in a way that is not just linear but integrated (understanding system level functioning) and dynamic (understanding system in flux). Key factors in the development of technologies for personalised medicine are standardisation, integration and harmonisation. For example, the tools and processes for data collection and analysis must be standardised across research sites. Research activity at different sites must be integrated to maximise synergies, and scientific research must be integrated with healthcare to ensure effective translation. There must also be harmonisation between scientific practices in different research sites, between science and healthcare and between science, healthcare and wider society, including the ethical and regulatory frameworks, the prevailing political and cultural ethos and the expectations of patients/citizens.</p>
<p class="publication"><span class="title">New biotechnology</span>&nbsp;<span class="year">2012</span>;<span class="volume">29</span>;<span class="number">6</span>;<span class="pagerange">625-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23091837" rel="external">23091837</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic analyses of the Taylorellae.</h4>
<p class="authors"><span class="author">Hauser H</span>, <span class="author">Richter DC</span>, <span class="author">van Tonder A</span>, <span class="author">Clark L</span> and <span class="author">Preston A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK.</p><p class="abstract">Contagious equine metritis (CEM) is an important venereal disease of horses that is of concern to the thoroughbred industry. Taylorella equigenitalis is a causative agent of CEM but very little is known about it or its close relative Taylorella asinigenitalis. To reveal novel information about Taylorella biology, comparative genomic analyses were undertaken. Whole genome sequencing was performed for the T. equigenitalis type strain, NCTC11184. Draft genome sequences were produced for a second T. equigenitalis strain and for a strain of T. asinigenitalis. These genome sequences were analysed and compared to each other and the recently released genome sequence of T. equigenitalis MCE9. These analyses revealed that T. equigenitalis strains appear to be very similar to each other with relatively little strain-specific DNA content. A number of genes were identified that encode putative toxins and adhesins that are possibly involved in infection. Analysis of T. asinigenitalis revealed that it has a very similar gene repertoire to that of T. equigenitalis but shares surprisingly little DNA sequence identity with it. The generation of genome sequence information greatly increases knowledge of these poorly characterised bacteria and greatly facilitates study of them.</p>
<p class="publication"><span class="title">Veterinary microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">159</span>;<span class="number">1-2</span>;<span class="pagerange">195-203</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22541164">22541164</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vetmic.2012.03.041">10.1016/j.vetmic.2012.03.041</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of the Exomes of Common Carp (Cyprinus carpio) and Zebrafish (Danio rerio).</h4>
<p class="authors"><span class="author">Henkel CV</span>, <span class="author">Dirks RP</span>, <span class="author">Jansen HJ</span>, <span class="author">Forlenza M</span>, <span class="author">Wiegertjes GF</span>, <span class="author">Howe K</span>, <span class="author">van den Thillart GE</span> and <span class="author">Spaink HP</span></p>
<p class="affiliation">1 ZF-Screens B.V. , Leiden, The Netherlands .</p>
<p class="abstract">Abstract Research on common carp, Cyprinus carpio, is beneficial for zebrafish research because of resources available owing to its large body size, such as the availability of sufficient organ material for transcriptomics, proteomics, and metabolomics. Here we describe the shot gun sequencing of a clonal double-haploid common carp line. The assembly consists of 511891 scaffolds with an N50 of 17â€‰kb, predicting a total genome size of 1.4-1.5â€‰Gb. A detailed analysis of the ten largest scaffolds indicates that the carp genome has a considerably lower repeat coverage than zebrafish, whilst the average intron size is significantly smaller, making it comparable to the fugu genome. The quality of the scaffolding was confirmed by comparisons with RNA deep sequencing data sets and a manual analysis for synteny with the zebrafish, especially the Hox gene clusters. In the ten largest scaffolds analyzed, the synteny of genes is almost complete. Comparisons of predicted exons of common carp with those of the zebrafish revealed only few genes specific for either zebrafish or carp, most of these being of unknown function. This supports the hypothesis of an additional genome duplication event in the carp evolutionary history, which-due to a higher degree of compactness-did not result in a genome larger than that of zebrafish.</p>
<p class="publication"><span class="title">Zebrafish</span>&nbsp;<span class="year">2012</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">59-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22715948" rel="external">22715948</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3371262" rel="external">3371262</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fzeb.2012.0773" rel="external">10.1089/zeb.2012.0773</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation sequencing and large genome assemblies.</h4>
<p class="authors"><span class="author">Henson J</span>, <span class="author">Tischler G</span> and <span class="author">Ning Z</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">The next-generation sequencing (NGS) revolution has drastically reduced time and cost requirements for sequencing of large genomes, and also qualitatively changed the problem of assembly. This article reviews the state of the art in de novo genome assembly, paying particular attention to mammalian-sized genomes. The strengths and weaknesses of the main sequencing platforms are highlighted, leading to a discussion of assembly and the new challenges associated with NGS data. Current approaches to assembly are outlined and the various software packages available are introduced and compared. The question of whether quality assemblies can be produced using short-read NGS data alone, or whether it must be combined with more expensive sequencing techniques, is considered. Prospects for future assemblers and tests of assembly performance are also discussed.</p>
<p class="publication"><span class="title">Pharmacogenomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">8</span>;<span class="pagerange">901-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22676195" rel="external">22676195</a>; DOI: <a href="http://dx.doi.org/10.2217%2Fpgs.12.72" rel="external">10.2217/pgs.12.72</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interactions between PPAR-&alpha; and inflammation-related cytokine genes on the development of Alzheimer&#39;s disease, observed by the Epistasis Project.</h4>
<p class="authors"><span class="author">Heun R</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">Ibrahim-Verbaas CA</span>, <span class="author">Combarros O</span>, <span class="author">Aulchenko YS</span>, <span class="author">Breteler M</span>, <span class="author">Schuur M</span>, <span class="author">van Duijn CM</span>, <span class="author">Hammond N</span>, <span class="author">Belbin O</span>, <span class="author">Cortina-Borja M</span>, <span class="author">Wilcock GK</span>, <span class="author">Brown K</span>, <span class="author">Barber R</span>, <span class="author">Kehoe PG</span>, <span class="author">Coto E</span>, <span class="author">Alvarez V</span>, <span class="author">Lehmann MG</span>, <span class="author">Deloukas P</span>, <span class="author">Mateo I</span>, <span class="author">Morgan K</span>, <span class="author">Warden DR</span>, <span class="author">Smith AD</span> and <span class="author">Lehmann DJ</span></p>
<p class="affiliation">Department of Psychiatry, University of Bonn, Bonn, Germany; 2Department of Psychiatry, Royal Derby Hospital,Uttoxeter Road, Derby DE22 3WQ, UK.</p><p class="abstract"><strong>Objective</strong>: Neuroinflammation contributes to the pathogenesis of sporadic Alzheimer&#39;s disease (AD). Variations in genes relevant to inflammation may be candidate genes for AD risk. Whole-genome association studies have identified relevant new and known genes. Their combined effects do not explain 100% of the risk, genetic interactions may contribute. We investigated whether genes involved in inflammation, i.e. PPAR-&alpha;, interleukins (IL) IL- 1&alpha;, IL-1&beta;, IL-6, and IL-10 may interact to increase AD risk.</p><p class="abstract"><strong>Methods</strong>: The Epistasis Project identifies interactions that affect the risk of AD. Genotyping of single nucleotide polymorphisms (SNPs) in PPARA, IL1A, IL1B, IL6 and IL10 was performed. Possible associations were analyzed by fitting logistic regression models with AD as outcome, controlling for centre, age, sex and presence of apolipoprotein &epsilon;4 allele (APOE&epsilon;4). Adjusted synergy factors were derived from interaction terms (p&lt;0.05 two-sided).</p><p class="abstract"><strong>Results</strong>: We observed four significant interactions between different SNPs in PPARA and in interleukins IL1A, IL1B, IL10 that may affect AD risk. There were no significant interactions between PPARA and IL6.</p><p class="abstract"><strong>Conclusions</strong>: In addition to an association of the PPARA L162V polymorphism with the AD risk, we observed four significant interactions between SNPs in PPARA and SNPs in IL1A, IL1B and IL10 affecting AD risk. We prove that gene-gene interactions explain part of the heritability of AD and are to be considered when assessing the genetic risk. Necessary replications will require between 1450 and 2950 of both cases and controls, depending on the prevalence of the SNP, to have 80% power to detect the observed synergy factors.</p>
<p class="publication"><span class="title">International journal of molecular epidemiology and genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">39-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22493750" rel="external">22493750</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3316448" rel="external">3316448</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A review of the role of stem cells in the development and treatment of glioma.</h4>
<p class="authors"><span class="author">Heywood RM</span>, <span class="author">Marcus HJ</span>, <span class="author">Ryan DJ</span>, <span class="author">Piccirillo SG</span>, <span class="author">Al-Mayhani TM</span> and <span class="author">Watts C</span></p>
<p class="affiliation">Centre for Brain Repair, Department Clinical Neurosciences, Cambridge University, E.D. Adrian Building, Forvie Site Hills Road, Cambridge, CB2 0PY, UK.</p><p class="abstract">The neurosurgical management of patients with intrinsic glial cancers is one of the most rapidly evolving areas of practice. This has been fuelled by advances in surgical technique not only in cytoreduction but also in drug delivery. Further innovation will depend on a deeper understanding of the biology of the disease and an appreciation of the limitations of current knowledge. Here we review the controversial topic of cancer stem cells applied to glioma to provide neurosurgeons with a working overview. It is now recognised that the adult human brain contains regionally specified cell populations capable of self-renewal that may contribute to tumour growth and maintenance following accumulated mutational change. Tumour cells adapted to maintain growth demonstrate some stem-like characteristics and as such constitute a legitimate therapeutic target. Cellular reprogramming technologies raise the potential of developing stem cells as novel surgical tools to target disease and possibly ameliorate some of the consequences of treatment. Achieving these goals remains a significant challenge to neurosurgical oncologists, not least in challenging how we think about treating brain cancer. This review will briefly examine our understanding of adult stem cells within the brain, the evidence that they contribute to the development of brain tumours as tumour-initiating cells, and the potential implications for therapy. It will also look at the role stem cells may play in the future management of glioma.</p>
<p class="publication"><span class="title">Acta neurochirurgica</span>&nbsp;<span class="year">2012</span>;<span class="volume">154</span>;<span class="number">6</span>;<span class="pagerange">951-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22527576" rel="external">22527576</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00701-012-1338-9" rel="external">10.1007/s00701-012-1338-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome mapping and genomics of caenorhabditis elegans</h4>
<p class="authors"><span class="author">Hodgkin,J.</span>, <span class="author">PAULINI,M.</span> and <span class="author">TULI,M.A.</span></p>
<p class="publication"><span class="title">Genome Mapping and Genomics in Animals</span>&nbsp;<span class="year">2012</span>;<span class="volume">4</span>;<span class="pagerange">17-30</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-31316-5-2">10.1007/978-3-642-31316-5-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovery of new treatments in the context of delivering personalized medicine</h4>
<p class="authors"><span class="author">Holgate ST</span>, <span class="author">PALOTIE A</span></p>
<p class="publication"><span class="title">Personalized Medicine.</span> <span class="year">2012</span>;<span class="volume">9</span>;<span class="pagerange">101-4</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Producing parasitic helminth reference and draft genomes at the Wellcome Trust Sanger Institute.</h4>
<p class="authors"><span class="author">Holroyd N</span> and <span class="author">Sanchez-Flores A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6E%65%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#101;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Wellcome Trust Sanger Institute (WTSI) is producing de novo reference quality genomes for parasitic helminth species from platyhelminth tapeworms (cestodes), flukes (trematodes) and roundworms (nematodes) primarily using second-generation (Illumina and 454) sequencing technologies. The reference genomes will be followed with draft coverage from a number of related strains or species. Comparing species- or strain-specific differences will help to unravel the genomic basis for differences in the organism&#39;s biology and ultimately contribute towards identifying potential novel targets for vaccine therapies. Second-generation sequencing technologies are revolutionizing parasite genomics. This article reviews the impact that sequencing technologies has had on genomics and how it has shaped the parasitic helminth genome sequencing initiative at WTSI.</p>
<p class="publication"><span class="title">Parasite immunology</span>&nbsp;<span class="year">2012</span>;<span class="volume">34</span>;<span class="number">2-3</span>;<span class="pagerange">100-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21707658" rel="external">21707658</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-3024.2011.01311.x" rel="external">10.1111/j.1365-3024.2011.01311.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shigella sonnei genome sequencing and phylogenetic analysis indicate recent global dissemination from Europe.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Baker S</span>, <span class="author">Weill FX</span>, <span class="author">Holmes EC</span>, <span class="author">Kitchen A</span>, <span class="author">Yu J</span>, <span class="author">Sangal V</span>, <span class="author">Brown DJ</span>, <span class="author">Coia JE</span>, <span class="author">Kim DW</span>, <span class="author">Choi SY</span>, <span class="author">Kim SH</span>, <span class="author">da Silveira WD</span>, <span class="author">Pickard DJ</span>, <span class="author">Farrar JJ</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of Melbourne, Victoria, Australia. <a href="mailto:%6B%68%6F%6C%74%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#107;&#104;&#111;&#108;&#116;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Shigella are human-adapted Escherichia coli that have gained the ability to invade the human gut mucosa and cause dysentery(1,2), spreading efficiently via low-dose fecal-oral transmission(3,4). Historically, S. sonnei has been predominantly responsible for dysentery in developed countries but is now emerging as a problem in the developing world, seeming to replace the more diverse Shigella flexneri in areas undergoing economic development and improvements in water quality(4-6). Classical approaches have shown that S. sonnei is genetically conserved and clonal(7). We report here whole-genome sequencing of 132 globally distributed isolates. Our phylogenetic analysis shows that the current S. sonnei population descends from a common ancestor that existed less than 500 years ago and that diversified into several distinct lineages with unique characteristics. Our analysis suggests that the majority of this diversification occurred in Europe and was followed by more recent establishment of local pathogen populations on other continents, predominantly due to the pandemic spread of a single, rapidly evolving, multidrug-resistant lineage.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800173</span>, <span class="grant">G0800173(86345)</span>; Wellcome Trust: <span class="grant">0689</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">9</span>;<span class="pagerange">1056-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22863732">22863732</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3442231">3442231</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2369">10.1038/ng.2369</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution genotyping of the endemic Salmonella Typhi population during a Vi (typhoid) vaccination trial in Kolkata.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Dutta S</span>, <span class="author">Manna B</span>, <span class="author">Bhattacharya SK</span>, <span class="author">Bhaduri B</span>, <span class="author">Pickard DJ</span>, <span class="author">Ochiai RL</span>, <span class="author">Ali M</span>, <span class="author">Clemens JD</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of Melbourne, Melbourne, Australia. <a href="mailto:%6B%68%6F%6C%74%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#107;&#104;&#111;&#108;&#116;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract"><strong>Background</strong>: Typhoid fever, caused by Salmonella enterica serovar Typhi (S. Typhi), is a major health problem especially in developing countries. Vaccines against typhoid are commonly used by travelers but less so by residents of endemic areas.</p><p class="abstract"><strong>Methodology</strong>: We used single nucleotide polymorphism (SNP) typing to investigate the population structure of 372 S. Typhi isolated during a typhoid disease burden study and Vi vaccine trial in Kolkata, India. Approximately sixty thousand people were enrolled for fever surveillance for 19 months prior to, and 24 months following, Vi vaccination of one third of the study population (May 2003-December 2006, vaccinations given December 2004).</p><p class="abstract">A diverse S. Typhi population was detected, including 21 haplotypes. The most common were of the H58 haplogroup (69%), which included all multidrug resistant isolates (defined as resistance to chloramphenicol, ampicillin and co-trimoxazole). Quinolone resistance was particularly high among H58-G isolates (97% Nalidixic acid resistant, 30% with reduced susceptibility to ciprofloxacin). Multiple typhoid fever episodes were detected in 22 households, however household clustering was not associated with specific S. Typhi haplotypes.</p><p class="abstract"><strong>Conclusions</strong>: Typhoid fever in Kolkata is caused by a diverse population of S. Typhi, however H58 haplotypes dominate and are associated with multidrug and quinolone resistance. Vi vaccination did not obviously impact on the haplotype population structure of the S. Typhi circulating during the study period.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">e1490</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22303491" rel="external">22303491</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3269425" rel="external">3269425</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001490" rel="external">10.1371/journal.pntd.0001490</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Combining RT-PCR-seq and RNA-seq to catalog all genic elements encoded in the human genome.</h4>
<p class="authors"><span class="author">Howald C</span>, <span class="author">Tanzer A</span>, <span class="author">Chrast J</span>, <span class="author">Kokocinski F</span>, <span class="author">Derrien T</span>, <span class="author">Walters N</span>, <span class="author">Gonzalez JM</span>, <span class="author">Frankish A</span>, <span class="author">Aken BL</span>, <span class="author">Hourlier T</span>, <span class="author">Vogel JH</span>, <span class="author">White S</span>, <span class="author">Searle S</span>, <span class="author">Harrow J</span>, <span class="author">Hubbard TJ</span>, <span class="author">Guig&oacute; R</span> and <span class="author">Reymond A</span></p>
<p class="affiliation">Center for Integrative Genomics, University of Lausanne, 1015 Lausanne, Switzerland.</p><p class="abstract">Within the ENCODE Consortium, GENCODE aimed to accurately annotate all protein-coding genes, pseudogenes, and noncoding transcribed loci in the human genome through manual curation and computational methods. Annotated transcript structures were assessed, and less well-supported loci were systematically, experimentally validated. Predicted exon-exon junctions were evaluated by RT-PCR amplification followed by highly multiplexed sequencing readout, a method we called RT-PCR-seq. Seventy-nine percent of all assessed junctions are confirmed by this evaluation procedure, demonstrating the high quality of the GENCODE gene set. RT-PCR-seq was also efficient to screen gene models predicted using the Human Body Map (HBM) RNA-seq data. We validated 73% of these predictions, thus confirming 1168 novel genes, mostly noncoding, which will further complement the GENCODE annotation. Our novel experimental validation pipeline is extremely sensitive, far more than unbiased transcriptome profiling through RNA sequencing, which is becoming the norm. For example, exon-exon junctions unique to GENCODE annotated transcripts are five times more likely to be corroborated with our targeted approach than with extensive large human transcriptome profiling. Data sets such as the HBM and ENCODE RNA-seq data fail sampling of low-expressed transcripts. Our RT-PCR-seq targeted approach also has the advantage of identifying novel exons of known genes, as we discovered unannotated exons in ~11% of assessed introns. We thus estimate that at least 18% of known loci have yet-unannotated exons. Our work demonstrates that the cataloging of all of the genic elements encoded in the human genome will necessitate a coordinated effort between unbiased and targeted approaches, like RNA-seq and RT-PCR-seq.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>, <span class="grant">U54 HG004557</span>; Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT077198/Z/05/Z</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">9</span>;<span class="pagerange">1698-710</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22955982">22955982</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3431487">3431487</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.134478.111">10.1101/gr.134478.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: Annotating many nematode genomes.</h4>
<p class="authors"><span class="author">Howe K</span>, <span class="author">Davis P</span>, <span class="author">Paulini M</span>, <span class="author">Tuli MA</span>, <span class="author">Williams G</span>, <span class="author">Yook K</span>, <span class="author">Durbin R</span>, <span class="author">Kersey P</span> and <span class="author">Sternberg PW</span></p>
<p class="affiliation">European Bioinformatics Institute; Wellcome Trust Genome Campus; Hinxton, Cambridge UK.</p>
<p class="abstract">WormBase (www.wormbase.org) has been serving the scientific community for over 11 years as the central repository for genomic and genetic information for the soil nematode Caenorhabditis elegans. The resource has evolved from its beginnings as a database housing the genomic sequence and genetic and physical maps of a single species, and now represents the breadth and diversity of nematode research, currently serving genome sequence and annotation for around 20 nematodes. In this article, we focus on WormBase's role of genome sequence annotation, describing how we annotate and integrate data from a growing collection of nematode species and strains. We also review our approaches to sequence curation, and discuss the impact on annotation quality of large functional genomics projects such as modENCODE.</p>
<p class="publication"><span class="title">Worm</span>&nbsp;<span class="year">2012</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">15-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24058818">24058818</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3670165">3670165</a>; DOI: <a href="http://dx.doi.org/10.4161%2Fworm.19574">10.4161/worm.19574</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exploration of signals of positive selection derived from genotype-based human genome scans using re-sequencing data.</h4>
<p class="authors"><span class="author">Hu M</span>, <span class="author">Ayub Q</span>, <span class="author">Guerra-Assun&ccedil;&atilde;o JA</span>, <span class="author">Long Q</span>, <span class="author">Ning Z</span>, <span class="author">Huang N</span>, <span class="author">Romero IG</span>, <span class="author">Mamanova L</span>, <span class="author">Akan P</span>, <span class="author">Liu X</span>, <span class="author">Coffey AJ</span>, <span class="author">Turner DJ</span>, <span class="author">Swerdlow H</span>, <span class="author">Burton J</span>, <span class="author">Quail MA</span>, <span class="author">Conrad DF</span>, <span class="author">Enright AJ</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Xue Y</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p><p class="abstract">We have investigated whether regions of the genome showing signs of positive selection in scans based on haplotype structure also show evidence of positive selection when sequence-based tests are applied, whether the target of selection can be localized more precisely, and whether such extra evidence can lead to increased biological insights. We used two tools: simulations under neutrality or selection, and experimental investigation of two regions identified by the HapMap2 project as putatively selected in human populations. Simulations suggested that neutral and selected regions should be readily distinguished and that it should be possible to localize the selected variant to within 40 kb at least half of the time. Re-sequencing of two ~300 kb regions (chr4:158Mb and chr10:22Mb) lacking known targets of selection in HapMap CHB individuals provided strong evidence for positive selection within each and suggested the micro-RNA gene hsa-miR-548c as the best candidate target in one region, and changes in regulation of the sperm protein gene SPAG6 in the other.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">131</span>;<span class="number">5</span>;<span class="pagerange">665-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22057783">22057783</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3325425">3325425</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-011-1111-9">10.1007/s00439-011-1111-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data.</h4>
<p class="authors"><span class="author">Huang J</span>, <span class="author">Ellinghaus D</span>, <span class="author">Franke A</span>, <span class="author">Howie B</span> and <span class="author">Li Y</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">We hypothesize that imputation based on data from the 1000 Genomes Project can identify novel association signals on a genome-wide scale due to the dense marker map and the large number of haplotypes. To test the hypothesis, the Wellcome Trust Case Control Consortium (WTCCC) Phase I genotype data were imputed using 1000 genomes as reference (20100804 EUR), and seven case/control association studies were performed using imputed dosages. We observed two &#39;missed&#39; disease-associated variants that were undetectable by the original WTCCC analysis, but were reported by later studies after the 2007 WTCCC publication. One is within the IL2RA gene for association with type 1 diabetes and the other in proximity with the CDKN2B gene for association with type 2 diabetes. We also identified two refined associations. One is SNP rs11209026 in exon 9 of IL23R for association with Crohn&#39;s disease, which is predicted to be probably damaging by PolyPhen2. The other refined variant is in the CUX2 gene region for association with type 1 diabetes, where the newly identified top SNP rs1265564 has an association P-value of 1.68 &times; 10(-16). The new lead SNP for the two refined loci provides a more plausible explanation for the disease association. We demonstrated that 1000 Genomes-based imputation could indeed identify both novel (in our case, &#39;missed&#39; because they were detected and replicated by studies after 2007) and refined signals. We anticipate the findings derived from this study to provide timely information when individual groups and consortia are beginning to engage in 1000 genomes-based imputation.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01 CA082659-11S1</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2012</span>;<span class="volume">20</span>;<span class="number">7</span>;<span class="pagerange">801-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22293688" rel="external">22293688</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2012.3" rel="external">10.1038/ejhg.2012.3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study for circulating levels of PAI-1 provides novel insights into its regulation.</h4>
<p class="authors"><span class="author">Huang J</span>, <span class="author">Sabater-Lleal M</span>, <span class="author">Asselbergs FW</span>, <span class="author">Tregouet D</span>, <span class="author">Shin SY</span>, <span class="author">Ding J</span>, <span class="author">Baumert J</span>, <span class="author">Oudot-Mellakh T</span>, <span class="author">Folkersen L</span>, <span class="author">Johnson AD</span>, <span class="author">Smith NL</span>, <span class="author">Williams SM</span>, <span class="author">Ikram MA</span>, <span class="author">Kleber ME</span>, <span class="author">Becker DM</span>, <span class="author">Truong V</span>, <span class="author">Mychaleckyj JC</span>, <span class="author">Tang W</span>, <span class="author">Yang Q</span>, <span class="author">Sennblad B</span>, <span class="author">Moore JH</span>, <span class="author">Williams FM</span>, <span class="author">Dehghan A</span>, <span class="author">Silbernagel G</span>, <span class="author">Schrijvers EM</span>, <span class="author">Smith S</span>, <span class="author">Karakas M</span>, <span class="author">Tofler GH</span>, <span class="author">Silveira A</span>, <span class="author">Navis GJ</span>, <span class="author">Lohman K</span>, <span class="author">Chen MH</span>, <span class="author">Peters A</span>, <span class="author">Goel A</span>, <span class="author">Hopewell JC</span>, <span class="author">Chambers JC</span>, <span class="author">Saleheen D</span>, <span class="author">Lundmark P</span>, <span class="author">Psaty BM</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Boehm BO</span>, <span class="author">Carter AM</span>, <span class="author">Meisinger C</span>, <span class="author">Peden JF</span>, <span class="author">Bis JC</span>, <span class="author">McKnight B</span>, <span class="author">&Ouml;hrvik J</span>, <span class="author">Taylor K</span>, <span class="author">Franzosi MG</span>, <span class="author">Seedorf U</span>, <span class="author">Collins R</span>, <span class="author">Franco-Cereceda A</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Goodall AH</span>, <span class="author">Yanek LR</span>, <span class="author">Cushman M</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Folsom AR</span>, <span class="author">Basu S</span>, <span class="author">Matijevic N</span>, <span class="author">van Gilst WH</span>, <span class="author">Kooner JS</span>, <span class="author">Hofman A</span>, <span class="author">Danesh J</span>, <span class="author">Clarke R</span>, <span class="author">Meigs JB</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">Kathiresan S</span>, <span class="author">Reilly MP</span>, <span class="author">CARDIoGRAM Consortium</span>, <span class="author">Klopp N</span>, <span class="author">Harris TB</span>, <span class="author">Winkelmann BR</span>, <span class="author">Grant PJ</span>, <span class="author">Hillege HL</span>, <span class="author">Watkins H</span>, <span class="author">C4D Consortium</span>, <span class="author">Spector TD</span>, <span class="author">Becker LC</span>, <span class="author">Tracy RP</span>, <span class="author">M&auml;rz W</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Eriksson P</span>, <span class="author">Cambien F</span>, <span class="author">CARDIOGENICS Consortium</span>, <span class="author">Morange PE</span>, <span class="author">Koenig W</span>, <span class="author">Soranzo N</span>, <span class="author">van der Harst P</span>, <span class="author">Liu Y</span>, <span class="author">O&#39;Donnell CJ</span> and <span class="author">Hamsten A</span></p>
<p class="affiliation">National Heart, Lung, and Blood Institute (NHBLI) Framingham Heart Study, Framingham, MA 01702, USA.</p><p class="abstract">We conducted a genome-wide association study to identify novel associations between genetic variants and circulating plasminogen activator inhibitor-1 (PAI-1) concentration, and examined functional implications of variants and genes that were discovered. A discovery meta-analysis was performed in 19 599 subjects, followed by replication analysis of genome-wide significant (P &lt; 5 &times; 10(-8)) single nucleotide polymorphisms (SNPs) in 10 796 independent samples. We further examined associations with type 2 diabetes and coronary artery disease, assessed the functional significance of the SNPs for gene expression in human tissues, and conducted RNA-silencing experiments for one novel association. We confirmed the association of the 4G/5G proxy SNP rs2227631 in the promoter region of SERPINE1 (7q22.1) and discovered genome-wide significant associations at 3 additional loci: chromosome 7q22.1 close to SERPINE1 (rs6976053, discovery P = 3.4 &times; 10(-10)); chromosome 11p15.2 within ARNTL (rs6486122, discovery P = 3.0 &times; 10(-8)); and chromosome 3p25.2 within PPARG (rs11128603, discovery P = 2.9 &times; 10(-8)). Replication was achieved for the 7q22.1 and 11p15.2 loci. There was nominal association with type 2 diabetes and coronary artery disease at ARNTL (P &lt; .05). Functional studies identified MUC3 as a candidate gene for the second association signal on 7q22.1. In summary, SNPs in SERPINE1 and ARNTL and an SNP associated with the expression of MUC3 were robustly associated with circulating levels of PAI-1.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; British Heart Foundation: <span class="grant">RG/08/014/24067</span>; Department of Health; Medical Research Council: <span class="grant">G0700931</span>; NCRR NIH HHS: <span class="grant">M01 RR00052</span>, <span class="grant">RR-024156</span>, <span class="grant">RR018787</span>, <span class="grant">UL1-RR-025005</span>; NHGRI NIH HHS: <span class="grant">U01-HG-004402</span>; NHLBI NIH HHS: <span class="grant">1U01 HL072518</span>, <span class="grant">HL080295</span>, <span class="grant">HL087652</span>, <span class="grant">HL105756</span>, <span class="grant">HL65234</span>, <span class="grant">HL67466</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01 HC085086</span>, <span class="grant">N01 HC095166</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-85239</span>, <span class="grant">N01-HC-95159</span>, <span class="grant">N01-HC-95160</span>, <span class="grant">N01-HC-95161</span>, <span class="grant">N01-HC-95162</span>, <span class="grant">N01-HC-95163</span>, <span class="grant">N01-HC-95164</span>, <span class="grant">N01-HC-95165</span>, <span class="grant">N01-HC-95166</span>, <span class="grant">N01-HC-95167</span>, <span class="grant">N01-HC-95168</span>, <span class="grant">N01-HC-95169</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">P01 HL074940</span>, <span class="grant">R01 HL092196</span>, <span class="grant">R01 HL095603</span>, <span class="grant">R01 HL095796</span>, <span class="grant">R01 HL59684</span>, <span class="grant">R01-HL-086694</span>, <span class="grant">R01-HL-087641</span>, <span class="grant">R01-HL-59367</span>, <span class="grant">R01-HL59367</span>, <span class="grant">R01HL071051</span>, <span class="grant">R01HL071205</span>, <span class="grant">R01HL071250</span>, <span class="grant">R01HL071251</span>, <span class="grant">R01HL071252</span>, <span class="grant">R01HL071258</span>, <span class="grant">R01HL071259</span>, <span class="grant">U01 HL072518</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">AG-023629</span>, <span class="grant">AG-027058</span>, <span class="grant">AG-15928</span>, <span class="grant">AG-20098</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">K24 DK080140</span>; NIGMS NIH HHS: <span class="grant">P20 GM103534</span>, <span class="grant">T32 GM080178</span>; NLM NIH HHS: <span class="grant">LM010098</span>, <span class="grant">R01 LM010098</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>, <span class="grant">HHSN268201200036C</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2012</span>;<span class="volume">120</span>;<span class="number">24</span>;<span class="pagerange">4873-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22990020">22990020</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3520624">3520624</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2012-06-436188">10.1182/blood-2012-06-436188</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MED12 controls the response to multiple cancer drugs through regulation of TGF-&beta; receptor signaling.</h4>
<p class="authors"><span class="author">Huang S</span>, <span class="author">H&ouml;lzel M</span>, <span class="author">Knijnenburg T</span>, <span class="author">Schlicker A</span>, <span class="author">Roepman P</span>, <span class="author">McDermott U</span>, <span class="author">Garnett M</span>, <span class="author">Grernrum W</span>, <span class="author">Sun C</span>, <span class="author">Prahallad A</span>, <span class="author">Groenendijk FH</span>, <span class="author">Mittempergher L</span>, <span class="author">Nijkamp W</span>, <span class="author">Neefjes J</span>, <span class="author">Salazar R</span>, <span class="author">Ten Dijke P</span>, <span class="author">Uramoto H</span>, <span class="author">Tanaka F</span>, <span class="author">Beijersbergen RL</span>, <span class="author">Wessels LF</span> and <span class="author">Bernards R</span></p>
<p class="affiliation">Division of Molecular Carcinogenesis, Cancer Genomics Center and Cancer Systems Biology Center, The Netherlands Cancer Institute, Plesmanlaan 121, 1066 CX Amsterdam, The Netherlands.</p><p class="abstract">Inhibitors of the ALK and EGF receptor tyrosine kinases provoke dramatic but short-lived responses in lung cancers harboring EML4-ALK translocations or activating mutations of EGFR, respectively. We used a large-scale RNAi screen to identify MED12, a component of the transcriptional MEDIATOR complex that is mutated in cancers, as a determinant of response to ALK and EGFR inhibitors. MED12 is in part cytoplasmic where it negatively regulates TGF-&beta;R2 through physical interaction. MED12 suppression therefore results in activation of TGF-&beta;R signaling, which is both necessary and sufficient for drug resistance. TGF-&beta; signaling causes MEK/ERK activation, and consequently MED12 suppression also confers resistance to MEK and BRAF inhibitors in other cancers. MED12 loss induces an EMT-like phenotype, which is associated with chemotherapy resistance in colon cancer patients and to gefitinib in lung cancer. Inhibition of TGF-&beta;R signaling restores drug responsiveness in MED12(KD) cells, suggesting a strategy to treat drug-resistant tumors that have lost MED12.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093868</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">151</span>;<span class="number">5</span>;<span class="pagerange">937-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23178117">23178117</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3672971">3672971</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2012.10.035">10.1016/j.cell.2012.10.035</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Isolation of homozygous mutant mouse embryonic stem cells using a dual selection system.</h4>
<p class="authors"><span class="author">Huang Y</span>, <span class="author">Pettitt SJ</span>, <span class="author">Guo G</span>, <span class="author">Liu G</span>, <span class="author">Li MA</span>, <span class="author">Yang F</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%68%75%61%6E%67%79%75%65%40%70%75%6D%63.%65%64%75.%63%6E">&#104;&#117;&#97;&#110;&#103;&#121;&#117;&#101;&#64;&#112;&#117;&#109;&#99;&#46;&#101;&#100;&#117;&#46;&#99;&#110;</a></p><p class="abstract">Obtaining random homozygous mutants in mammalian cells for forward genetic studies has always been problematic due to the diploid genome. With one mutation per cell, only one allele of an autosomal gene can be disrupted, and the resulting heterozygous mutant is unlikely to display a phenotype. In cells with a genetic background deficient for the Bloom&#39;s syndrome helicase, such heterozygous mutants segregate homozygous daughter cells at a low frequency due to an elevated rate of crossover following mitotic recombination between homologous chromosomes. We constructed DNA vectors that are selectable based on their copy number and used these to isolate these rare homozygous mutant cells independent of their phenotype. We use the piggyBac transposon to limit the initial mutagenesis to one copy per cell, and select for cells that have increased the transposon copy number to two or more. This yields homozygous mutants with two allelic mutations, but also cells that have duplicated the mutant chromosome and become aneuploid during culture. On average, 26% of the copy number gain events occur by the mitotic recombination pathway. We obtained homozygous cells from 40% of the heterozygous mutants tested. This method can provide homozygous mammalian loss-of-function mutants for forward genetic applications.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">79643</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">3</span>;<span class="pagerange">e21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22127858" rel="external">22127858</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3273828" rel="external">3273828</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr908" rel="external">10.1093/nar/gkr908</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">No evidence of an association between mitochondrial DNA variants and osteoarthritis in 7393 cases and 5122 controls.</h4>
<p class="authors"><span class="author">Hudson G</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Wilson I</span>, <span class="author">Southam L</span>, <span class="author">Rayner NW</span>, <span class="author">Arden N</span>, <span class="author">Birrell F</span>, <span class="author">Carluke I</span>, <span class="author">Carr A</span>, <span class="author">Chapman K</span>, <span class="author">Deloukas P</span>, <span class="author">Doherty M</span>, <span class="author">McCaskie A</span>, <span class="author">Ollier WE</span>, <span class="author">Ralston SH</span>, <span class="author">Reed MR</span>, <span class="author">Spector TD</span>, <span class="author">Valdes AM</span>, <span class="author">Wallis GA</span>, <span class="author">Wilkinson JM</span>, <span class="author">Zeggini E</span>, <span class="author">Samuels DC</span>, <span class="author">Loughlin J</span>, <span class="author">Chinnery PF</span> and <span class="author">arcOGEN Consortium</span></p>
<p class="affiliation">1Institute of Genetic Medicine, Newcastle University, Newcastle upon Tyne, UK.</p>
<p class="abstract">OBJECTIVES: Osteoarthritis (OA) has a complex aetiology with a strong genetic component. Genome-wide association studies implicate several nuclear genes in the aetiology, but a major component of the heritability has yet to be defined at the molecular level. Initial studies implicate maternally inherited variants of mitochondrial DNA (mtDNA) in subgroups of patients with OA based on gender and specific joint involvement, but these findings have not been replicated. METHODS: The authors studied 138 maternally inherited mtDNA variants genotyped in a two cohort genetic association study across a total of 7393 OA cases from the arcOGEN consortium and 5122 controls genotyped in the Wellcome Trust Case Control consortium 2 study. RESULTS: Following data quality control we examined 48 mtDNA variants that were common in cohort 1 and cohort 2, and found no association with OA. None of the phenotypic subgroups previously associated with mtDNA haplogroups were associated in this study. CONCLUSIONS: We were not able to replicate previously published findings in the largest mtDNA association study to date. The evidence linking OA to mtDNA is not compelling at present.</p>
<p class="publication"><span class="title">Annals of the rheumatic diseases</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22984172" rel="external">22984172</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fannrheumdis-2012-201932" rel="external">10.1136/annrheumdis-2012-201932</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fitness of Escherichia coli strains carrying expressed and partially silent IncN and IncP1 plasmids.</h4>
<p class="authors"><span class="author">Humphrey B</span>, <span class="author">Thomson NR</span>, <span class="author">Thomas CM</span>, <span class="author">Brooks K</span>, <span class="author">Sanders M</span>, <span class="author">Delsol AA</span>, <span class="author">Roe JM</span>, <span class="author">Bennett PM</span> and <span class="author">Enne VI</span></p>
<p class="affiliation">Bristol Centre for Antimicrobial Research, Department of Cellular and Molecular Medicine, University of Bristol, Medical Sciences Building, University Walk, Bristol, BS8 1TD, UK.</p>
<p class="abstract">Background: Understanding the survival of resistance plasmids in the absence of selective pressure for the antibiotic resistance genes they carry is important for assessing the value of interventions to combat resistant bacteria. Here, several poorly explored questions regarding the fitness impact of IncP1 and IncN broad host range plasmids on their bacterial hosts are examined; namely, whether related plasmids have similar fitness impacts, whether this varies according to host genetic background, and what effect antimicrobial resistance gene silencing has on fitness.

Results: For the IncP1 group pairwise in vitro growth competition demonstrated that the fitness cost of plasmid RP1 depends on the host strain. For the IncN group, plasmids R46 and N3 whose sequence is presented for the first time conferred remarkably different fitness costs despite sharing closely related backbone structures, implicating the accessory genes in fitness. Silencing of antimicrobial resistance genes was found to be beneficial for host fitness with RP1 but not for IncN plasmid pVE46.

Conclusions: These findings suggest that the fitness impact of a given plasmid on its host cannot be inferred from results obtained with other host-plasmid combinations, even if these are closely related.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span>, <span class="grant">089222</span></p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="pagerange">53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22475035" rel="external">22475035</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3347995" rel="external">3347995</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-12-53" rel="external">10.1186/1471-2180-12-53</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare and functional SIAE variants are not associated with autoimmune disease risk in up to 66,924 individuals of European ancestry.</h4>
<p class="authors"><span class="author">Hunt KA</span>, <span class="author">Smyth DJ</span>, <span class="author">Balschun T</span>, <span class="author">Ban M</span>, <span class="author">Mistry V</span>, <span class="author">Ahmad T</span>, <span class="author">Anand V</span>, <span class="author">Barrett JC</span>, <span class="author">Bhaw-Rosun L</span>, <span class="author">Bockett NA</span>, <span class="author">Brand OJ</span>, <span class="author">Brouwer E</span>, <span class="author">Concannon P</span>, <span class="author">Cooper JD</span>, <span class="author">Dias KR</span>, <span class="author">van Diemen CC</span>, <span class="author">Dubois PC</span>, <span class="author">Edkins S</span>, <span class="author">F&ouml;lster-Holst R</span>, <span class="author">Fransen K</span>, <span class="author">Glass DN</span>, <span class="author">Heap GA</span>, <span class="author">Hofmann S</span>, <span class="author">Huizinga TW</span>, <span class="author">Hunt S</span>, <span class="author">Langford C</span>, <span class="author">Lee J</span>, <span class="author">Mansfield J</span>, <span class="author">Marrosu MG</span>, <span class="author">Mathew CG</span>, <span class="author">Mein CA</span>, <span class="author">M&uuml;ller-Quernheim J</span>, <span class="author">Nutland S</span>, <span class="author">Onengut-Gumuscu S</span>, <span class="author">Ouwehand W</span>, <span class="author">Pearce K</span>, <span class="author">Prescott NJ</span>, <span class="author">Posthumus MD</span>, <span class="author">Potter S</span>, <span class="author">Rosati G</span>, <span class="author">Sambrook J</span>, <span class="author">Satsangi J</span>, <span class="author">Schreiber S</span>, <span class="author">Shtir C</span>, <span class="author">Simmonds MJ</span>, <span class="author">Sudman M</span>, <span class="author">Thompson SD</span>, <span class="author">Toes R</span>, <span class="author">Trynka G</span>, <span class="author">Vyse TJ</span>, <span class="author">Walker NM</span>, <span class="author">Weidinger S</span>, <span class="author">Zhernakova A</span>, <span class="author">Zoledziewska M</span>, <span class="author">Type 1 Diabetes Genetics Consortium</span>, <span class="author">UK Inflammatory Bowel Disease (IBD) Genetics Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Weersma RK</span>, <span class="author">Gough SC</span>, <span class="author">Sawcer S</span>, <span class="author">Wijmenga C</span>, <span class="author">Parkes M</span>, <span class="author">Cucca F</span>, <span class="author">Franke A</span>, <span class="author">Deloukas P</span>, <span class="author">Rich SS</span>, <span class="author">Todd JA</span> and <span class="author">van Heel DA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">1</span>;<span class="pagerange">3-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22200769" rel="external">22200769</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3287292" rel="external">3287292</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1037" rel="external">10.1038/ng.1037</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">InterPro in 2011: new developments in the family and domain prediction database.</h4>
<p class="authors"><span class="author">Hunter S</span>, <span class="author">Jones P</span>, <span class="author">Mitchell A</span>, <span class="author">Apweiler R</span>, <span class="author">Attwood TK</span>, <span class="author">Bateman A</span>, <span class="author">Bernard T</span>, <span class="author">Binns D</span>, <span class="author">Bork P</span>, <span class="author">Burge S</span>, <span class="author">de Castro E</span>, <span class="author">Coggill P</span>, <span class="author">Corbett M</span>, <span class="author">Das U</span>, <span class="author">Daugherty L</span>, <span class="author">Duquenne L</span>, <span class="author">Finn RD</span>, <span class="author">Fraser M</span>, <span class="author">Gough J</span>, <span class="author">Haft D</span>, <span class="author">Hulo N</span>, <span class="author">Kahn D</span>, <span class="author">Kelly E</span>, <span class="author">Letunic I</span>, <span class="author">Lonsdale D</span>, <span class="author">Lopez R</span>, <span class="author">Madera M</span>, <span class="author">Maslen J</span>, <span class="author">McAnulla C</span>, <span class="author">McDowall J</span>, <span class="author">McMenamin C</span>, <span class="author">Mi H</span>, <span class="author">Mutowo-Muellenet P</span>, <span class="author">Mulder N</span>, <span class="author">Natale D</span>, <span class="author">Orengo C</span>, <span class="author">Pesseat S</span>, <span class="author">Punta M</span>, <span class="author">Quinn AF</span>, <span class="author">Rivoire C</span>, <span class="author">Sangrador-Vegas A</span>, <span class="author">Selengut JD</span>, <span class="author">Sigrist CJ</span>, <span class="author">Scheremetjew M</span>, <span class="author">Tate J</span>, <span class="author">Thimmajanarthanan M</span>, <span class="author">Thomas PD</span>, <span class="author">Wu CH</span>, <span class="author">Yeats C</span> and <span class="author">Yong SY</span></p>
<p class="affiliation">EMBL Outstation European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SD Cambridge, UK. <a href="mailto:%68%75%6E%74%65%72%40%65%62%69.%61%63.%75%6B">&#104;&#117;&#110;&#116;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">InterPro (http://www.ebi.ac.uk/interpro/) is a database that integrates diverse information about protein families, domains and functional sites, and makes it freely available to the public via Web-based interfaces and services. Central to the database are diagnostic models, known as signatures, against which protein sequences can be searched to determine their potential function. InterPro has utility in the large-scale analysis of whole genomes and meta-genomes, as well as in characterizing individual protein sequences. Herein we give an overview of new developments in the database and its associated software since 2009, including updates to database content, curation processes and Web and programmatic interfaces.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F010508/1</span>; NIGMS NIH HHS: <span class="grant">GM081084</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D306-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22096229" rel="external">22096229</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245097" rel="external">3245097</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr948" rel="external">10.1093/nar/gkr948</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at 6q22 and 17q21 are associated with intracranial volume.</h4>
<p class="authors"><span class="author">Ikram MA</span>, <span class="author">Fornage M</span>, <span class="author">Smith AV</span>, <span class="author">Seshadri S</span>, <span class="author">Schmidt R</span>, <span class="author">Debette S</span>, <span class="author">Vrooman HA</span>, <span class="author">Sigurdsson S</span>, <span class="author">Ropele S</span>, <span class="author">Taal HR</span>, <span class="author">Mook-Kanamori DO</span>, <span class="author">Coker LH</span>, <span class="author">Longstreth WT</span>, <span class="author">Niessen WJ</span>, <span class="author">DeStefano AL</span>, <span class="author">Beiser A</span>, <span class="author">Zijdenbos AP</span>, <span class="author">Struchalin M</span>, <span class="author">Jack CR</span>, <span class="author">Rivadeneira F</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Knopman DS</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pennell CE</span>, <span class="author">Thiering E</span>, <span class="author">Steegers EA</span>, <span class="author">Hakonarson H</span>, <span class="author">Heinrich J</span>, <span class="author">Palmer LJ</span>, <span class="author">Jarvelin MR</span>, <span class="author">McCarthy MI</span>, <span class="author">Grant SF</span>, <span class="author">St Pourcain B</span>, <span class="author">Timpson NJ</span>, <span class="author">Smith GD</span>, <span class="author">Sovio U</span>, <span class="author">Early Growth Genetics Consortium</span>, <span class="author">Nalls MA</span>, <span class="author">Au R</span>, <span class="author">Hofman A</span>, <span class="author">Gudnason H</span>, <span class="author">van der Lugt A</span>, <span class="author">Harris TB</span>, <span class="author">Meeks WM</span>, <span class="author">Vernooij MW</span>, <span class="author">van Buchem MA</span>, <span class="author">Catellier D</span>, <span class="author">Jaddoe VW</span>, <span class="author">Gudnason V</span>, <span class="author">Windham BG</span>, <span class="author">Wolf PA</span>, <span class="author">van Duijn CM</span>, <span class="author">Mosley TH</span>, <span class="author">Schmidt H</span>, <span class="author">Launer LJ</span>, <span class="author">Breteler MM</span>, <span class="author">DeCarli C</span> and <span class="author">Cohorts for Heart and Aging Research in Genomic Epidemiology Consortium</span></p>
<p class="affiliation">Department of Epidemiology, Erasmus Medical Center University Medical Center, Rotterdam, The Netherlands. <a href="mailto:%6D.%61.%69%6B%72%61%6D%40%65%72%61%73%6D%75%73%6D%63.%6E%6C">&#109;&#46;&#97;&#46;&#105;&#107;&#114;&#97;&#109;&#64;&#101;&#114;&#97;&#115;&#109;&#117;&#115;&#109;&#99;&#46;&#110;&#108;</a></p><p class="abstract">During aging, intracranial volume remains unchanged and represents maximally attained brain size, while various interacting biological phenomena lead to brain volume loss. Consequently, intracranial volume and brain volume in late life reflect different genetic influences. Our genome-wide association study (GWAS) in 8,175 community-dwelling elderly persons did not reveal any associations at genome-wide significance (P &lt; 5 &times; 10(-8)) for brain volume. In contrast, intracranial volume was significantly associated with two loci: rs4273712 (P = 3.4 &times; 10(-11)), a known height-associated locus on chromosome 6q22, and rs9915547 (P = 1.5 &times; 10(-12)), localized to the inversion on chromosome 17q21. We replicated the associations of these loci with intracranial volume in a separate sample of 1,752 elderly persons (P = 1.1 &times; 10(-3) for 6q22 and 1.2 &times; 10(-3) for 17q21). Furthermore, we also found suggestive associations of the 17q21 locus with head circumference in 10,768 children (mean age of 14.5 months). Our data identify two loci associated with head size, with the inversion at 17q21 also likely to be involved in attaining maximal brain size.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">MOP 82893</span>; Medical Research Council: <span class="grant">74882</span>, <span class="grant">G0500539</span>; NCRR NIH HHS: <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01-HG004402</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>, <span class="grant">HL093029</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01-HL087641</span>, <span class="grant">R01-HL093029</span>; NIA NIH HHS: <span class="grant">AG031287</span>, <span class="grant">AG033040</span>, <span class="grant">AG033193</span>, <span class="grant">AG08122</span>, <span class="grant">AG16495</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">P30AG013846</span>; NICHD NIH HHS: <span class="grant">1R01HD056465-01A1</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; NINDS NIH HHS: <span class="grant">NS17950</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>; Wellcome Trust: <span class="grant">076467</span>, <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">539-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22504418" rel="external">22504418</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2245" rel="external">10.1038/ng.2245</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A method to infer positive selection from marker dynamics in an asexual population.</h4>
<p class="authors"><span class="author">Illingworth CJ</span> and <span class="author">Mustonen V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Motivation</strong>: The observation of positive selection acting on a mutant indicates that the corresponding mutation has some form of functional relevance. Determining the fitness effects of mutations thus has relevance to many interesting biological questions. One means of identifying beneficial mutations in an asexual population is to observe changes in the frequency of marked subsets of the population. We here describe a method to estimate the establishment times and fitnesses of beneficial mutations from neutral marker frequency data.</p><p class="abstract"><strong>Results</strong>: The method accurately reproduces complex marker frequency trajectories. In simulations for which positive selection is close to 5% per generation, we obtain correlations upwards of 0.91 between correct and inferred haplotype establishment times. Where mutation selection coefficients are exponentially distributed, the inferred distribution of haplotype fitnesses is close to being correct. Applied to data from a bacterial evolution experiment, our method reproduces an observed correlation between evolvability and initial fitness defect.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">6</span>;<span class="pagerange">831-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22223745">22223745</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3307107">3307107</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr722">10.1093/bioinformatics/btr722</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Components of selection in the evolution of the influenza virus: linkage effects beat inherent selection.</h4>
<p class="authors"><span class="author">Illingworth CJ</span> and <span class="author">Mustonen V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%63%69%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#105;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The influenza virus is an important human pathogen, with a rapid rate of evolution in the human population. The rate of homologous recombination within genes of influenza is essentially zero. As such, where two alleles within the same gene are in linkage disequilibrium, interference between alleles will occur, whereby selection acting upon one allele has an influence upon the frequency of the other. We here measured the relative importance of selection and interference effects upon the evolution of influenza. We considered time-resolved allele frequency data from the global evolutionary history of the haemagglutinin gene of human influenza A/H3N2, conducting an in-depth analysis of sequences collected since 1996. Using a model that accounts for selection-caused interference between alleles in linkage disequilibrium, we estimated the inherent selective benefit of individual polymorphisms in the viral population. These inherent selection coefficients were in turn used to calculate the total selective effect of interference acting upon each polymorphism, considering the effect of the initial background upon which a mutation arose, and the subsequent effect of interference from other alleles that were under selection. Viewing events in retrospect, we estimated the influence of each of these components in determining whether a mutant allele eventually fixed or died in the global viral population. Our inherent selection coefficients, when combined across different regions of the protein, were consistent with previous measurements of dN/dS for the same system. Alleles going on to fix in the global population tended to be under more positive selection, to arise on more beneficial backgrounds, and to avoid strong negative interference from other alleles under selection. However, on average, the fate of a polymorphism was determined more by the combined influence of interference effects than by its inherent selection coefficient.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">e1003091</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23300444">23300444</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3531508">3531508</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1003091">10.1371/journal.ppat.1003091</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quantifying selection acting on a complex trait using allele frequency time series data.</h4>
<p class="authors"><span class="author">Illingworth CJ</span>, <span class="author">Parts L</span>, <span class="author">Schiffels S</span>, <span class="author">Liti G</span> and <span class="author">Mustonen V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">When selection is acting on a large genetically diverse population, beneficial alleles increase in frequency. This fact can be used to map quantitative trait loci by sequencing the pooled DNA from the population at consecutive time points and observing allele frequency changes. Here, we present a population genetic method to analyze time series data of allele frequencies from such an experiment. Beginning with a range of proposed evolutionary scenarios, the method measures the consistency of each with the observed frequency changes. Evolutionary theory is utilized to formulate equations of motion for the allele frequencies, following which likelihoods for having observed the sequencing data under each scenario are derived. Comparison of these likelihoods gives an insight into the prevailing dynamics of the system under study. We illustrate the method by quantifying selective effects from an experiment, in which two phenotypically different yeast strains were first crossed and then propagated under heat stress (Parts L, Cubillos FA, Warringer J, et al. [14 co-authors]. 2011. Revealing the genetic structure of a trait by sequencing a population under selection. Genome Res). From these data, we discover that about 6% of polymorphic sites evolve nonneutrally under heat stress conditions, either because of their linkage to beneficial (driver) alleles or because they are drivers themselves. We further identify 44 genomic regions containing one or more candidate driver alleles, quantify their apparent selective advantage, obtain estimates of recombination rates within the regions, and show that the dynamics of the drivers display a strong signature of selection going beyond additive models. Our approach is applicable to study adaptation in a range of systems under different evolutionary pressures.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT077192/Z/05/Z</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2012</span>;<span class="volume">29</span>;<span class="number">4</span>;<span class="pagerange">1187-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22114362">22114362</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3731369">3731369</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsr289">10.1093/molbev/msr289</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel Loci for metabolic networks and multi-tissue expression studies reveal genes for atherosclerosis.</h4>
<p class="authors"><span class="author">Inouye M</span>, <span class="author">Ripatti S</span>, <span class="author">Kettunen J</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Oksala N</span>, <span class="author">Laurila PP</span>, <span class="author">Kangas AJ</span>, <span class="author">Soininen P</span>, <span class="author">Savolainen MJ</span>, <span class="author">Viikari J</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Perola M</span>, <span class="author">Salomaa V</span>, <span class="author">Raitakari O</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Taskinen MR</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Ala-Korpela M</span>, <span class="author">Palotie A</span> and <span class="author">de Bakker PI</span></p>
<p class="affiliation">Medical Systems Biology, Departments of Pathology and of Microbiology and Immunology, The University of Melbourne, Parkville, Victoria, Australia.</p><p class="abstract">Association testing of multiple correlated phenotypes offers better power than univariate analysis of single traits. We analyzed 6,600 individuals from two population-based cohorts with both genome-wide SNP data and serum metabolomic profiles. From the observed correlation structure of 130 metabolites measured by nuclear magnetic resonance, we identified 11 metabolic networks and performed a multivariate genome-wide association analysis. We identified 34 genomic loci at genome-wide significance, of which 7 are novel. In comparison to univariate tests, multivariate association analysis identified nearly twice as many significant associations in total. Multi-tissue gene expression studies identified variants in our top loci, SERPINA1 and AQP9, as eQTLs and showed that SERPINA1 and AQP9 expression in human blood was associated with metabolites from their corresponding metabolic networks. Finally, liver expression of AQP9 was associated with atherosclerotic lesion area in mice, and in human arterial tissue both SERPINA1 and AQP9 were shown to be upregulated (6.3-fold and 4.6-fold, respectively) in atherosclerotic plaques. Our study illustrates the power of multi-phenotype GWAS and highlights candidate genes for atherosclerosis.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">e1002907</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22916037" rel="external">22916037</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3420921" rel="external">3420921</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002907" rel="external">10.1371/journal.pgen.1002907</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies a variant in HDAC9 associated with large vessel ischemic stroke.</h4>
<p class="authors"><span class="author">International Stroke Genetics Consortium (ISGC)</span>, <span class="author">Wellcome Trust Case Control Consortium 2 (WTCCC2)</span>, <span class="author">Bellenguez C</span>, <span class="author">Bevan S</span>, <span class="author">Gschwendtner A</span>, <span class="author">Spencer CC</span>, <span class="author">Burgess AI</span>, <span class="author">Pirinen M</span>, <span class="author">Jackson CA</span>, <span class="author">Traylor M</span>, <span class="author">Strange A</span>, <span class="author">Su Z</span>, <span class="author">Band G</span>, <span class="author">Syme PD</span>, <span class="author">Malik R</span>, <span class="author">Pera J</span>, <span class="author">Norrving B</span>, <span class="author">Lemmens R</span>, <span class="author">Freeman C</span>, <span class="author">Schanz R</span>, <span class="author">James T</span>, <span class="author">Poole D</span>, <span class="author">Murphy L</span>, <span class="author">Segal H</span>, <span class="author">Cortellini L</span>, <span class="author">Cheng YC</span>, <span class="author">Woo D</span>, <span class="author">Nalls MA</span>, <span class="author">M&uuml;ller-Myhsok B</span>, <span class="author">Meisinger C</span>, <span class="author">Seedorf U</span>, <span class="author">Ross-Adams H</span>, <span class="author">Boonen S</span>, <span class="author">Wloch-Kopec D</span>, <span class="author">Valant V</span>, <span class="author">Slark J</span>, <span class="author">Furie K</span>, <span class="author">Delavaran H</span>, <span class="author">Langford C</span>, <span class="author">Deloukas P</span>, <span class="author">Edkins S</span>, <span class="author">Hunt S</span>, <span class="author">Gray E</span>, <span class="author">Dronov S</span>, <span class="author">Peltonen L</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">Boncoraglio GB</span>, <span class="author">Parati EA</span>, <span class="author">Attia J</span>, <span class="author">Holliday E</span>, <span class="author">Levi C</span>, <span class="author">Franzosi MG</span>, <span class="author">Goel A</span>, <span class="author">Helgadottir A</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Casas JP</span>, <span class="author">Corvin A</span>, <span class="author">Duncanson A</span>, <span class="author">Jankowski J</span>, <span class="author">Mathew CG</span>, <span class="author">Palmer CN</span>, <span class="author">Plomin R</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Trembath RC</span>, <span class="author">Viswanathan AC</span>, <span class="author">Wood NW</span>, <span class="author">Worrall BB</span>, <span class="author">Kittner SJ</span>, <span class="author">Mitchell BD</span>, <span class="author">Kissela B</span>, <span class="author">Meschia JF</span>, <span class="author">Thijs V</span>, <span class="author">Lindgren A</span>, <span class="author">Macleod MJ</span>, <span class="author">Slowik A</span>, <span class="author">Walters M</span>, <span class="author">Rosand J</span>, <span class="author">Sharma P</span>, <span class="author">Farrall M</span>, <span class="author">Sudlow CL</span>, <span class="author">Rothwell PM</span>, <span class="author">Dichgans M</span>, <span class="author">Donnelly P</span> and <span class="author">Markus HS</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, UK.</p><p class="abstract">Genetic factors have been implicated in stroke risk, but few replicated associations have been reported. We conducted a genome-wide association study (GWAS) for ischemic stroke and its subtypes in 3,548 affected individuals and 5,972 controls, all of European ancestry. Replication of potential signals was performed in 5,859 affected individuals and 6,281 controls. We replicated previous associations for cardioembolic stroke near PITX2 and ZFHX3 and for large vessel stroke at a 9p21 locus. We identified a new association for large vessel stroke within HDAC9 (encoding histone deacetylase 9) on chromosome 7p21.1 (including further replication in an additional 735 affected individuals and 28,583 controls) (rs11984041; combined P = 1.87 &times; 10(-11); odds ratio (OR) = 1.42, 95% confidence interval (CI) = 1.28-1.57). All four loci exhibited evidence for heterogeneity of effect across the stroke subtypes, with some and possibly all affecting risk for only one subtype. This suggests distinct genetic architectures for different stroke subtypes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; NCATS NIH HHS: <span class="grant">UL1 TR000077</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">084724</span>, <span class="grant">085475</span>, <span class="grant">085475/B/08/Z</span>, <span class="grant">085475/Z/08/Z</span>, <span class="grant">WT084724MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">328-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22306652">22306652</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3303115">3303115</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1081">10.1038/ng.1081</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptional data: a new gateway to drug repositioning?</h4>
<p class="authors"><span class="author">Iorio F</span>, <span class="author">Rittman T</span>, <span class="author">Ge H</span>, <span class="author">Menden M</span> and <span class="author">Saez-Rodriguez J</span></p>
<p class="affiliation">EMBL - European Bioinformatics Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SD, UK; Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SD, UK.</p>
<p class="abstract">Recent advances in computational biology suggest that any perturbation to the transcriptional programme of the cell can be summarised by a proper 'signature': a set of genes combined with a pattern of expression. Therefore, it should be possible to generate proxies of clinicopathological phenotypes and drug effects through signatures acquired via DNA microarray technology. Gene expression signatures have recently been assembled and compared through genome-wide metrics, unveiling unexpected drug-disease and drug-drug 'connections' by matching corresponding signatures. Consequently, novel applications for existing drugs have been predicted and experimentally validated. Here, we describe related methods, case studies and resources while discussing challenges and benefits of exploiting existing repositories of microarray data that could serve as a search space for systematic drug repositioning.</p>
<p class="publication"><span class="title">Drug discovery today</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22897878" rel="external">22897878</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.drudis.2012.07.014" rel="external">10.1016/j.drudis.2012.07.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study implicates HLA-C*01:02 as a risk factor at the major histocompatibility complex locus in schizophrenia.</h4>
<p class="authors"><span class="author">Irish Schizophrenia Genomics Consortium and the Wellcome Trust Case Control Consortium 2</span></p>
<p class="abstract">Background: We performed a genome-wide association study (GWAS) to identify common risk variants for schizophrenia.

Methods: The discovery scan included 1606 patients and 1794 controls from Ireland, using 6,212,339 directly genotyped or imputed single nucleotide polymorphisms (SNPs). A subset of this sample (270 cases and 860 controls) was subsequently included in the Psychiatric GWAS Consortium-schizophrenia GWAS meta-analysis.

Results: One hundred eight SNPs were taken forward for replication in an independent sample of 13,195 cases and 31,021 control subjects. The most significant associations in discovery, corrected for genomic inflation, were (rs204999, p combined = 1.34 × 10(-9) and in combined samples (rs2523722 p combined = 2.88 × 10(-16)) mapped to the major histocompatibility complex (MHC) region. We imputed classical human leukocyte antigen (HLA) alleles at the locus; the most significant finding was with HLA-C*01:02. This association was distinct from the top SNP signal. The HLA alleles DRB1*03:01 and B*08:01 were protective, replicating a previous study.

Conclusions: This study provides further support for involvement of MHC class I molecules in schizophrenia. We found evidence of association with previously reported risk alleles at the TCF4, VRK2, and ZNF804A loci.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute: <span class="grant">072894/Z/03/Z</span>, <span class="grant">075491/Z/04/B</span>; Medical Research Council: <span class="grant">G0000934</span>; NIMH NIH HHS: <span class="grant">MH 41953</span>, <span class="grant">MH083094</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">085475/B/08/Z</span>, <span class="grant">085475/Z/08/Z</span>, <span class="grant">090532/Z/09/Z</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2012</span>;<span class="volume">72</span>;<span class="number">8</span>;<span class="pagerange">620-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22883433" rel="external">22883433</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2012.05.035" rel="external">10.1016/j.biopsych.2012.05.035</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Guidelines for Reporting Novel mecA Gene Homologues.</h4>
<p class="authors"><span class="author">Ito T</span>, <span class="author">Hiramatsu K</span>, <span class="author">Tomasz A</span>, <span class="author">de Lencastre H</span>, <span class="author">Perreten V</span>, <span class="author">Holden MT</span>, <span class="author">Coleman DC</span>, <span class="author">Goering R</span>, <span class="author">Giffard PM</span>, <span class="author">Skov RL</span>, <span class="author">Zhang K</span>, <span class="author">Westh H</span>, <span class="author">O&#39;Brien F</span>, <span class="author">Tenover FC</span>, <span class="author">Oliveira DC</span>, <span class="author">Boyle-Vavra S</span>, <span class="author">Laurent F</span>, <span class="author">Kearns AM</span>, <span class="author">Kreiswirth B</span>, <span class="author">Ko KS</span>, <span class="author">Grundmann H</span>, <span class="author">Sollid JE</span>, <span class="author">John JF</span>, <span class="author">Daum R</span>, <span class="author">Soderquist B</span>, <span class="author">Buist G</span> and <span class="author">on behalf of the International Working Group on the Classification of Staphylococcal Cassette Chromosome Elements (IWG-SCC)</span></p>
<p class="affiliation">Department of Bacteriology, Juntendo University, Tokyo, Japan.</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2012</span>;<span class="volume">56</span>;<span class="number">10</span>;<span class="pagerange">4997-4999</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22869575" rel="external">22869575</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3457410" rel="external">3457410</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.01199-12" rel="external">10.1128/AAC.01199-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antigenic diversity is generated by distinct evolutionary mechanisms in African trypanosome species.</h4>
<p class="authors"><span class="author">Jackson AP</span>, <span class="author">Berry A</span>, <span class="author">Aslett M</span>, <span class="author">Allison HC</span>, <span class="author">Burton P</span>, <span class="author">Vavrova-Anderson J</span>, <span class="author">Brown R</span>, <span class="author">Browne H</span>, <span class="author">Corton N</span>, <span class="author">Hauser H</span>, <span class="author">Gamble J</span>, <span class="author">Gilderthorp R</span>, <span class="author">Marcello L</span>, <span class="author">McQuillan J</span>, <span class="author">Otto TD</span>, <span class="author">Quail MA</span>, <span class="author">Sanders MJ</span>, <span class="author">van Tonder A</span>, <span class="author">Ginger ML</span>, <span class="author">Field MC</span>, <span class="author">Barry JD</span>, <span class="author">Hertz-Fowler C</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%61%6E%64%72%65%77.%6A%61%63%6B%73%6F%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#110;&#100;&#114;&#101;&#119;&#46;&#106;&#97;&#99;&#107;&#115;&#111;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Antigenic variation enables pathogens to avoid the host immune response by continual switching of surface proteins. The protozoan blood parasite Trypanosoma brucei causes human African trypanosomiasis (&quot;sleeping sickness&quot;) across sub-Saharan Africa and is a model system for antigenic variation, surviving by periodically replacing a monolayer of variant surface glycoproteins (VSG) that covers its cell surface. We compared the genome of Trypanosoma brucei with two closely related parasites Trypanosoma congolense and Trypanosoma vivax, to reveal how the variant antigen repertoire has evolved and how it might affect contemporary antigenic diversity. We reconstruct VSG diversification showing that Trypanosoma congolense uses variant antigens derived from multiple ancestral VSG lineages, whereas in Trypanosoma brucei VSG have recent origins, and ancestral gene lineages have been repeatedly co-opted to novel functions. These historical differences are reflected in fundamental differences between species in the scale and mechanism of recombination. Using phylogenetic incompatibility as a metric for genetic exchange, we show that the frequency of recombination is comparable between Trypanosoma congolense and Trypanosoma brucei but is much lower in Trypanosoma vivax. Furthermore, in showing that the C-terminal domain of Trypanosoma brucei VSG plays a crucial role in facilitating exchange, we reveal substantial species differences in the mechanism of VSG diversification. Our results demonstrate how past VSG evolution indirectly determines the ability of contemporary parasites to generate novel variant antigens through recombination and suggest that the current model for antigenic variation in Trypanosoma brucei is only one means by which these parasites maintain chronic infections.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085349</span>, <span class="grant">085349/Z/08/Z</span>, <span class="grant">WT 055558/Z/98/A</span>, <span class="grant">WT 055558/Z/98/C</span>, <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">9</span>;<span class="pagerange">3416-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22331916">22331916</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3295286">3295286</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1117313109">10.1073/pnas.1117313109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing and genetic testing for MODY.</h4>
<p class="authors"><span class="author">Johansson S</span>, <span class="author">Irgens H</span>, <span class="author">Chudasama KK</span>, <span class="author">Molnes J</span>, <span class="author">Aerts J</span>, <span class="author">Roque FS</span>, <span class="author">Jonassen I</span>, <span class="author">Levy S</span>, <span class="author">Lima K</span>, <span class="author">Knappskog PM</span>, <span class="author">Bell GI</span>, <span class="author">Molven A</span> and <span class="author">Nj&oslash;lstad PR</span></p>
<p class="affiliation">Department of Clinical Medicine, University of Bergen, Bergen, Norway.</p>
<p class="abstract">Context: Genetic testing for monogenic diabetes is important for patient care. Given the extensive genetic and clinical heterogeneity of diabetes, exome sequencing might provide additional diagnostic potential when standard Sanger sequencing-based diagnostics is inconclusive.

Objective: The aim of the study was to examine the performance of exome sequencing for a molecular diagnosis of MODY in patients who have undergone conventional diagnostic sequencing of candidate genes with negative results.

We performed exome enrichment followed by high-throughput sequencing in nine patients with suspected MODY. They were Sanger sequencing-negative for mutations in the HNF1A, HNF4A, GCK, HNF1B and INS genes. We excluded common, non-coding and synonymous gene variants, and performed in-depth analysis on filtered sequence variants in a pre-defined set of 111 genes implicated in glucose metabolism.

Results: On average, we obtained 45 X median coverage of the entire targeted exome and found 199 rare coding variants per individual. We identified 0-4 rare non-synonymous and nonsense variants per individual in our a priori list of 111 candidate genes. Three of the variants were considered pathogenic (in ABCC8, HNF4A and PPARG, respectively), thus exome sequencing led to a genetic diagnosis in at least three of the nine patients. Approximately 91% of known heterozygous SNPs in the target exomes were detected, but we also found low coverage in some key diabetes genes using our current exome sequencing approach. Novel variants in the genes ARAP1, GLIS3, MADD, NOTCH2 and WFS1 need further investigation to reveal their possible role in diabetes.

Conclusion: Our results demonstrate that exome sequencing can improve molecular diagnostics of MODY when used as a complement to Sanger sequencing. However, improvements will be needed, especially concerning coverage, before the full potential of exome sequencing can be realized.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">e38050</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22662265" rel="external">22662265</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3360646" rel="external">3360646</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0038050" rel="external">10.1371/journal.pone.0038050</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bcl11a is required for neuronal morphogenesis and sensory circuit formation in dorsal spinal cord development.</h4>
<p class="authors"><span class="author">John A</span>, <span class="author">Brylka H</span>, <span class="author">Wiegreffe C</span>, <span class="author">Simon R</span>, <span class="author">Liu P</span>, <span class="author">J&uuml;ttner R</span>, <span class="author">Crenshaw EB</span>, <span class="author">Luyten FP</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Birchmeier C</span> and <span class="author">Britsch S</span></p>
<p class="affiliation">Institute of Molecular and Cellular Anatomy, Ulm University, 89081 Ulm, Germany.</p><p class="abstract">Dorsal spinal cord neurons receive and integrate somatosensory information provided by neurons located in dorsal root ganglia. Here we demonstrate that dorsal spinal neurons require the Kr&uuml;ppel-C(2)H(2) zinc-finger transcription factor Bcl11a for terminal differentiation and morphogenesis. The disrupted differentiation of dorsal spinal neurons observed in Bcl11a mutant mice interferes with their correct innervation by cutaneous sensory neurons. To understand the mechanism underlying the innervation deficit, we characterized changes in gene expression in the dorsal horn of Bcl11a mutants and identified dysregulated expression of the gene encoding secreted frizzled-related protein 3 (sFRP3, or Frzb). Frzb mutant mice show a deficit in the innervation of the spinal cord, suggesting that the dysregulated expression of Frzb can account in part for the phenotype of Bcl11a mutants. Thus, our genetic analysis of Bcl11a reveals essential functions of this transcription factor in neuronal morphogenesis and sensory wiring of the dorsal spinal cord and identifies Frzb, a component of the Wnt pathway, as a downstream acting molecule involved in this process.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">139</span>;<span class="number">10</span>;<span class="pagerange">1831-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22491945">22491945</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.072850">10.1242/dev.072850</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Resolution of a meningococcal disease outbreak from whole-genome sequence data with rapid web-based analysis methods.</h4>
<p class="authors"><span class="author">Jolley KA</span>, <span class="author">Hill DM</span>, <span class="author">Bratcher HB</span>, <span class="author">Harrison OB</span>, <span class="author">Feavers IM</span>, <span class="author">Parkhill J</span> and <span class="author">Maiden MC</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">The increase in the capacity and reduction in cost of whole-genome sequencing methods present the imminent prospect of such data being used routinely in real time for investigations of bacterial disease outbreaks. For this to be realized, however, it is necessary that generic, portable, and robust analysis frameworks be available, which can be readily interpreted and used in real time by microbiologists, clinicians, and public health epidemiologists. We have achieved this with a set of analysis tools integrated into the PubMLST.org website, which can in principle be used for the analysis of any pathogen. The approach is demonstrated with genomic data from isolates obtained during a well-characterized meningococcal disease outbreak at the University of Southampton, United Kingdom, that occurred in 1997. Whole-genome sequence data were collected, de novo assembled, and deposited into the PubMLST Neisseria BIGSdb database, which automatically annotated the sequences. This enabled the immediate and backwards-compatible classification of the isolates with a number of schemes, including the following: conventional, extended, and ribosomal multilocus sequence typing (MLST, eMLST, and rMLST); antigen gene sequence typing (AGST); analysis based on genes conferring antibiotic susceptibility. The isolates were also compared to a reference isolate belonging to the same clonal complex (ST-11) at 1,975 loci. Visualization of the data with the NeighborNet algorithm, implemented in SplitsTree 4 within the PubMLST website, permitted complete resolution of the outbreak and related isolates, demonstrating that multiple closely related but distinct strains were simultaneously present in asymptomatic carriage and disease, with two causing disease and one responsible for the outbreak itself.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">50</span>;<span class="number">9</span>;<span class="pagerange">3046-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22785191" rel="external">22785191</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3421817" rel="external">3421817</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.01312-12" rel="external">10.1128/JCM.01312-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genomic basis of adaptive evolution in threespine sticklebacks.</h4>
<p class="authors"><span class="author">Jones FC</span>, <span class="author">Grabherr MG</span>, <span class="author">Chan YF</span>, <span class="author">Russell P</span>, <span class="author">Mauceli E</span>, <span class="author">Johnson J</span>, <span class="author">Swofford R</span>, <span class="author">Pirun M</span>, <span class="author">Zody MC</span>, <span class="author">White S</span>, <span class="author">Birney E</span>, <span class="author">Searle S</span>, <span class="author">Schmutz J</span>, <span class="author">Grimwood J</span>, <span class="author">Dickson MC</span>, <span class="author">Myers RM</span>, <span class="author">Miller CT</span>, <span class="author">Summers BR</span>, <span class="author">Knecht AK</span>, <span class="author">Brady SD</span>, <span class="author">Zhang H</span>, <span class="author">Pollen AA</span>, <span class="author">Howes T</span>, <span class="author">Amemiya C</span>, <span class="author">Broad Institute Genome Sequencing Platform &amp;amp; Whole Genome Assembly Team</span>, <span class="author">Baldwin J</span>, <span class="author">Bloom T</span>, <span class="author">Jaffe DB</span>, <span class="author">Nicol R</span>, <span class="author">Wilkinson J</span>, <span class="author">Lander ES</span>, <span class="author">Di Palma F</span>, <span class="author">Lindblad-Toh K</span> and <span class="author">Kingsley DM</span></p>
<p class="affiliation">Department of Developmental Biology, Beckman Center B300, Stanford University School of Medicine, Stanford California 94305, USA.</p><p class="abstract">Marine stickleback fish have colonized and adapted to thousands of streams and lakes formed since the last ice age, providing an exceptional opportunity to characterize genomic mechanisms underlying repeated ecological adaptation in nature. Here we develop a high-quality reference genome assembly for threespine sticklebacks. By sequencing the genomes of twenty additional individuals from a global set of marine and freshwater populations, we identify a genome-wide set of loci that are consistently associated with marine-freshwater divergence. Our results indicate that reuse of globally shared standing genetic variation, including chromosomal inversions, has an important role in repeated evolution of distinct marine and freshwater sticklebacks, and in the maintenance of divergent ecotypes during early stages of reproductive isolation. Both coding and regulatory changes occur in the set of loci underlying marine-freshwater evolution, but regulatory changes appear to predominate in this well known example of repeated adaptive evolution in nature.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">P50-HG002568</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">484</span>;<span class="number">7392</span>;<span class="pagerange">55-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22481358" rel="external">22481358</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3322419" rel="external">3322419</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10944" rel="external">10.1038/nature10944</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of protein palmitoylation reveals a pervasive role in Plasmodium development and pathogenesis.</h4>
<p class="authors"><span class="author">Jones ML</span>, <span class="author">Collins MO</span>, <span class="author">Goulding D</span>, <span class="author">Choudhary JS</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">Malaria Programme, The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Asexual stage Plasmodium falciparum replicates and undergoes a tightly regulated developmental process in human erythrocytes. One mechanism involved in the regulation of this process is posttranslational modification (PTM) of parasite proteins. Palmitoylation is a PTM in which cysteine residues undergo a reversible lipid modification, which can regulate target proteins in diverse ways. Using complementary palmitoyl protein purification approaches and quantitative mass spectrometry, we examined protein palmitoylation in asexual-stage P. falciparum parasites and identified over 400 palmitoylated proteins, including those involved in cytoadherence, drug resistance, signaling, development, and invasion. Consistent with the prevalence of palmitoylated proteins, palmitoylation is essential for P. falciparum asexual development and influences erythrocyte invasion by directly regulating the stability of components of the actin-myosin invasion motor. Furthermore, P. falciparum uses palmitoylation in diverse ways, stably modifying some proteins while dynamically palmitoylating others. Palmitoylation therefore plays a central role in regulating P. falciparum blood stage development.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643/Z/06/Z</span>, <span class="grant">089084</span></p>
<p class="publication"><span class="title">Cell host &amp; microbe</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">246-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22901544">22901544</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3501726">3501726</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.chom.2012.06.005">10.1016/j.chom.2012.06.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Getting stuck in: protein palmitoylation in Plasmodium.</h4>
<p class="authors"><span class="author">Jones ML</span>, <span class="author">Tay CL</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">Malaria Programme, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">Palmitoylation is the reversible post-translational addition of a lipid moiety to cysteine residues on targeted proteins. The recent use of proteomic-scale techniques to study protein palmitoylation in multiple organisms has radically changed our understanding of the diversity of proteins and signaling pathways that are affected by palmitoylation. These experiments have made clear that, similarly to phosphorylation, palmitoylation is a regulatory tool that has an impact upon a wide range of essential eukaryotic processes. A recent proteome-level analysis of protein palmitoylation in Plasmodium has revealed the importance of palmitoylation in parasite biology and has raised new and exciting questions about several Plasmodium-specific and virulence-associated processes.</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23009804" rel="external">23009804</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2012.08.009" rel="external">10.1016/j.pt.2012.08.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dispatches from the functional phase of genome biology.</h4>
<p class="authors"><span class="author">Jostins L</span></p>
<p class="affiliation">Statistical and Computational Genetics, Wellcome Trust Sanger Institute, Cambridge CB10 1HH, UK. <a href="mailto:%6C%6A%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#106;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">ABSTRACT: A report on the 25th annual meeting on The Biology of Genomes, Cold Spring Harbor, USA, 8-12 May 2012.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">6</span>;<span class="pagerange">316</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22738333" rel="external">22738333</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3446312" rel="external">3446312</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-6-316" rel="external">10.1186/gb-2012-13-6-316</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Misuse of hierarchical linear models overstates the significance of a reported association between OXTR and prosociality.</h4>
<p class="authors"><span class="author">Jostins L</span>, <span class="author">Pickrell JK</span>, <span class="author">MacArthur DG</span> and <span class="author">Barrett JC</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">18</span>;<span class="pagerange">E1048</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22499788">22499788</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3344973">3344973</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1202539109">10.1073/pnas.1202539109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Host-microbe interactions have shaped the genetic architecture of inflammatory bowel disease.</h4>
<p class="authors"><span class="author">Jostins L</span>, <span class="author">Ripke S</span>, <span class="author">Weersma RK</span>, <span class="author">Duerr RH</span>, <span class="author">McGovern DP</span>, <span class="author">Hui KY</span>, <span class="author">Lee JC</span>, <span class="author">Schumm LP</span>, <span class="author">Sharma Y</span>, <span class="author">Anderson CA</span>, <span class="author">Essers J</span>, <span class="author">Mitrovic M</span>, <span class="author">Ning K</span>, <span class="author">Cleynen I</span>, <span class="author">Theatre E</span>, <span class="author">Spain SL</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Goyette P</span>, <span class="author">Wei Z</span>, <span class="author">Abraham C</span>, <span class="author">Achkar JP</span>, <span class="author">Ahmad T</span>, <span class="author">Amininejad L</span>, <span class="author">Ananthakrishnan AN</span>, <span class="author">Andersen V</span>, <span class="author">Andrews JM</span>, <span class="author">Baidoo L</span>, <span class="author">Balschun T</span>, <span class="author">Bampton PA</span>, <span class="author">Bitton A</span>, <span class="author">Boucher G</span>, <span class="author">Brand S</span>, <span class="author">B&uuml;ning C</span>, <span class="author">Cohain A</span>, <span class="author">Cichon S</span>, <span class="author">D&#39;Amato M</span>, <span class="author">De Jong D</span>, <span class="author">Devaney KL</span>, <span class="author">Dubinsky M</span>, <span class="author">Edwards C</span>, <span class="author">Ellinghaus D</span>, <span class="author">Ferguson LR</span>, <span class="author">Franchimont D</span>, <span class="author">Fransen K</span>, <span class="author">Gearry R</span>, <span class="author">Georges M</span>, <span class="author">Gieger C</span>, <span class="author">Glas J</span>, <span class="author">Haritunians T</span>, <span class="author">Hart A</span>, <span class="author">Hawkey C</span>, <span class="author">Hedl M</span>, <span class="author">Hu X</span>, <span class="author">Karlsen TH</span>, <span class="author">Kupcinskas L</span>, <span class="author">Kugathasan S</span>, <span class="author">Latiano A</span>, <span class="author">Laukens D</span>, <span class="author">Lawrance IC</span>, <span class="author">Lees CW</span>, <span class="author">Louis E</span>, <span class="author">Mahy G</span>, <span class="author">Mansfield J</span>, <span class="author">Morgan AR</span>, <span class="author">Mowat C</span>, <span class="author">Newman W</span>, <span class="author">Palmieri O</span>, <span class="author">Ponsioen CY</span>, <span class="author">Potocnik U</span>, <span class="author">Prescott NJ</span>, <span class="author">Regueiro M</span>, <span class="author">Rotter JI</span>, <span class="author">Russell RK</span>, <span class="author">Sanderson JD</span>, <span class="author">Sans M</span>, <span class="author">Satsangi J</span>, <span class="author">Schreiber S</span>, <span class="author">Simms LA</span>, <span class="author">Sventoraityte J</span>, <span class="author">Targan SR</span>, <span class="author">Taylor KD</span>, <span class="author">Tremelling M</span>, <span class="author">Verspaget HW</span>, <span class="author">De Vos M</span>, <span class="author">Wijmenga C</span>, <span class="author">Wilson DC</span>, <span class="author">Winkelmann J</span>, <span class="author">Xavier RJ</span>, <span class="author">Zeissig S</span>, <span class="author">Zhang B</span>, <span class="author">Zhang CK</span>, <span class="author">Zhao H</span>, <span class="author">International IBD Genetics Consortium (IIBDGC)</span>, <span class="author">Silverberg MS</span>, <span class="author">Annese V</span>, <span class="author">Hakonarson H</span>, <span class="author">Brant SR</span>, <span class="author">Radford-Smith G</span>, <span class="author">Mathew CG</span>, <span class="author">Rioux JD</span>, <span class="author">Schadt EE</span>, <span class="author">Daly MJ</span>, <span class="author">Franke A</span>, <span class="author">Parkes M</span>, <span class="author">Vermeire S</span>, <span class="author">Barrett JC</span> and <span class="author">Cho JH</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Crohn&#39;s disease and ulcerative colitis, the two common forms of inflammatory bowel disease (IBD), affect over 2.5 million people of European ancestry, with rising prevalence in other populations. Genome-wide association studies and subsequent meta-analyses of these two diseases as separate phenotypes have implicated previously unsuspected mechanisms, such as autophagy, in their pathogenesis and showed that some IBD loci are shared with other inflammatory diseases. Here we expand on the knowledge of relevant pathways by undertaking a meta-analysis of Crohn&#39;s disease and ulcerative colitis genome-wide association scans, followed by extensive validation of significant findings, with a combined total of more than 75,000 cases and controls. We identify 71 new associations, for a total of 163 IBD loci, that meet genome-wide significance thresholds. Most loci contribute to both phenotypes, and both directional (consistently favouring one allele over the course of human history) and balancing (favouring the retention of both alleles within populations) selection effects are evident. Many IBD loci are also implicated in other immune-mediated disorders, most notably with ankylosing spondylitis and psoriasis. We also observe considerable overlap between susceptibility loci for IBD and mycobacterial infection. Gene co-expression network analysis emphasizes this relationship, with pathways shared between host responses to mycobacteria and those predisposing to IBD.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">G0000934</span>; Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/137</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0600329</span>, <span class="grant">G0800675</span>, <span class="grant">G0800759</span>, <span class="grant">G1002033</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>, <span class="grant">UL1 TR000124-01</span>; NCI NIH HHS: <span class="grant">CA141743</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>; NIAID NIH HHS: <span class="grant">AI062773</span>; NIDDK NIH HHS: <span class="grant">DK043351</span>, <span class="grant">DK062413</span>, <span class="grant">DK062420</span>, <span class="grant">DK062422</span>, <span class="grant">DK062423</span>, <span class="grant">DK062429</span>, <span class="grant">DK062429-S1</span>, <span class="grant">DK062431</span>, <span class="grant">DK062432</span>, <span class="grant">DK063491</span>, <span class="grant">DK076984</span>, <span class="grant">DK084554</span>, <span class="grant">DK83756</span>, <span class="grant">K23 DK097142</span>, <span class="grant">P01DK046763</span>, <span class="grant">P30 DK043351</span>, <span class="grant">R01 DK055731</span>, <span class="grant">U01 DK062418</span>, <span class="grant">U01 DK062422</span>, <span class="grant">U01 DK062429</span>, <span class="grant">U01 DK062431</span>, <span class="grant">U01 DK062432</span>; NIGMS NIH HHS: <span class="grant">T32 GM007205</span>, <span class="grant">T32GM07205</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">083948/Z/07/Z</span>, <span class="grant">085475/B/08/Z</span>, <span class="grant">085475/Z/08/Z</span>, <span class="grant">089120</span>, <span class="grant">090532</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">491</span>;<span class="number">7422</span>;<span class="pagerange">119-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23128233">23128233</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3491803">3491803</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11582">10.1038/nature11582</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reactome - a curated knowledgebase of biological pathways: megakaryocytes and platelets.</h4>
<p class="authors"><span class="author">Jupe S</span>, <span class="author">Akkerman JW</span>, <span class="author">Soranzo N</span> and <span class="author">Ouwehand WH</span></p>
<p class="affiliation">European Bioinformatics Institute, Hinxton, Cambridge, United Kingdom Department of Clinical Chemistry and Haematology, University Medical Centre Utrecht, Utrecht, the Netherlands The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom Department of Haematology, University of Cambridge and National Health Service Blood and Transplant, Cambridge, United Kingdom.</p>
<p class="abstract">The platelet field is undergoing a radical transformation from reductionist simplification to large scale integration. Following the era of simplification whereby biological processes were dissected at the molecular and atomic level, new technologies have now generated an overwhelming flow of information that can only be comprehended in an integrated approach. High throughput analyses of transcription and translation in megakaryocytes and platelets, individual analyses of membranes and secretory granules, the clustering of pathways for platelet activation and inhibition in signalosomes all add to a complexity that requires platforms for knowledge accumulation.</p>
<p class="publication"><span class="title">Journal of thrombosis and haemostasis : JTH</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22985186" rel="external">22985186</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1538-7836.2012.04930.x" rel="external">10.1111/j.1538-7836.2012.04930.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction of insulin and PPAR-&alpha; genes in Alzheimer&#39;s disease: the Epistasis Project.</h4>
<p class="authors"><span class="author">K&ouml;lsch H</span>, <span class="author">Lehmann DJ</span>, <span class="author">Ibrahim-Verbaas CA</span>, <span class="author">Combarros O</span>, <span class="author">van Duijn CM</span>, <span class="author">Hammond N</span>, <span class="author">Belbin O</span>, <span class="author">Cortina-Borja M</span>, <span class="author">Lehmann MG</span>, <span class="author">Aulchenko YS</span>, <span class="author">Schuur M</span>, <span class="author">Breteler M</span>, <span class="author">Wilcock GK</span>, <span class="author">Brown K</span>, <span class="author">Kehoe PG</span>, <span class="author">Barber R</span>, <span class="author">Coto E</span>, <span class="author">Alvarez V</span>, <span class="author">Deloukas P</span>, <span class="author">Mateo I</span>, <span class="author">Maier W</span>, <span class="author">Morgan K</span>, <span class="author">Warden DR</span>, <span class="author">Smith AD</span> and <span class="author">Heun R</span></p>
<p class="affiliation">Department of Psychiatry, University of Bonn, Bonn, Germany. <a href="mailto:%68%65%69%6B%65.%6B%6F%65%6C%73%63%68%40%75%6E%69-%62%6F%6E%6E.%64%65">&#104;&#101;&#105;&#107;&#101;&#46;&#107;&#111;&#101;&#108;&#115;&#99;&#104;&#64;&#117;&#110;&#105;&#45;&#98;&#111;&#110;&#110;&#46;&#100;&#101;</a></p><p class="abstract">Altered glucose metabolism has been described in Alzheimer&#39;s disease (AD). We re-investigated the interaction of the insulin (INS) and the peroxisome proliferator-activated receptor alpha (PPARA) genes in AD risk in the Epistasis Project, including 1,757 AD cases and 6,294 controls. Allele frequencies of both SNPs (PPARA L162V, INS intron 0 A/T) differed between Northern Europeans and Northern Spanish. The PPARA 162LL genotype increased AD risk in Northern Europeans (p&nbsp;=&nbsp;0.04), but not in Northern Spanish (p&nbsp;=&nbsp;0.2). There was no association of the INS intron 0 TT genotype with AD. We observed an interaction on AD risk between PPARA 162LL and INS intron 0 TT genotypes in Northern Europeans (Synergy factor 2.5, p&nbsp;=&nbsp;0.016), but not in Northern Spanish. We suggest that dysregulation of glucose metabolism contributes to the development of AD and might be due in part to genetic variations in INS and PPARA and their interaction especially in Northern Europeans.</p>
<p class="publication"><span class="title">Journal of neural transmission (Vienna, Austria : 1996)</span>&nbsp;<span class="year">2012</span>;<span class="volume">119</span>;<span class="number">4</span>;<span class="pagerange">473-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22065208">22065208</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00702-011-0732-4">10.1007/s00702-011-0732-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Routine use of microbial whole genome sequencing in diagnostic and public health microbiology.</h4>
<p class="authors"><span class="author">K&ouml;ser CU</span>, <span class="author">Ellington MJ</span>, <span class="author">Cartwright EJ</span>, <span class="author">Gillespie SH</span>, <span class="author">Brown NM</span>, <span class="author">Farrington M</span>, <span class="author">Holden MT</span>, <span class="author">Dougan G</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Addenbrooke's Hospital, Cambridge, United Kingdom.</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">e1002824</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22876174" rel="external">22876174</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3410874" rel="external">3410874</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002824" rel="external">10.1371/journal.ppat.1002824</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The B10 Idd9.3 locus mediates accumulation of functionally superior CD137(+) regulatory T cells in the nonobese diabetic type 1 diabetes model.</h4>
<p class="authors"><span class="author">Kachapati K</span>, <span class="author">Adams DE</span>, <span class="author">Wu Y</span>, <span class="author">Steward CA</span>, <span class="author">Rainbow DB</span>, <span class="author">Wicker LS</span>, <span class="author">Mittler RS</span> and <span class="author">Ridgway WM</span></p>
<p class="affiliation">Division of Immunology, Allergy and Rheumatology, University of Cincinnati College of Medicine, Cincinnati, OH 45267, USA.</p><p class="abstract">CD137 is a T cell costimulatory molecule encoded by the prime candidate gene (designated Tnfrsf9) in NOD.B10 Idd9.3 congenic mice protected from type 1 diabetes (T1D). NOD T cells show decreased CD137-mediated T cell signaling compared with NOD.B10 Idd9.3 T cells, but it has been unclear how this decreased CD137 T cell signaling could mediate susceptibility to T1D. We and others have shown that a subset of regulatory T cells (Tregs) constitutively expresses CD137 (whereas effector T cells do not, and only express CD137 briefly after activation). In this study, we show that the B10 Idd9.3 region intrinsically contributes to accumulation of CD137(+) Tregs with age. NOD.B10 Idd9.3 mice showed significantly increased percentages and numbers of CD137(+) peripheral Tregs compared with NOD mice. Moreover, Tregs expressing the B10 Idd9.3 region preferentially accumulated in mixed bone marrow chimeric mice reconstituted with allotypically marked NOD and NOD.B10 Idd9.3 bone marrow. We demonstrate a possible significance of increased numbers of CD137(+) Tregs by showing functional superiority of FACS-purified CD137(+) Tregs in vitro compared with CD137(-) Tregs in T cell-suppression assays. Increased functional suppression was also associated with increased production of the alternatively spliced CD137 isoform, soluble CD137, which has been shown to suppress T cell proliferation. We show for the first time, to our knowledge, that CD137(+) Tregs are the primary cellular source of soluble CD137. NOD.B10 Idd9.3 mice showed significantly increased serum soluble CD137 compared with NOD mice with age, consistent with their increased numbers of CD137(+) Tregs with age. These studies demonstrate the importance of CD137(+) Tregs in T1D and offer a new hypothesis for how the NOD Idd9.3 region could act to increase T1D susceptibility.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">U19 AI056374</span>, <span class="grant">U19AI56374</span>; Wellcome Trust: <span class="grant">079895</span>, <span class="grant">091157</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2012</span>;<span class="volume">189</span>;<span class="number">10</span>;<span class="pagerange">5001-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23066155">23066155</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3505683">3505683</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1101013">10.4049/jimmunol.1101013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Potential Novel Spontaneous Preterm Birth Gene, AR, Identified by Linkage and Association Analysis of X Chromosomal Markers.</h4>
<p class="authors"><span class="author">Karjalainen MK</span>, <span class="author">Huusko JM</span>, <span class="author">Ulvila J</span>, <span class="author">Sotkasiira J</span>, <span class="author">Luukkonen A</span>, <span class="author">Teramo K</span>, <span class="author">Plunkett J</span>, <span class="author">Anttila V</span>, <span class="author">Palotie A</span>, <span class="author">Haataja R</span>, <span class="author">Muglia LJ</span> and <span class="author">Hallman M</span></p>
<p class="affiliation">Department of Pediatrics, Institute of Clinical Medicine, University of Oulu, Oulu, Finland.</p>
<p class="abstract">Preterm birth is the major cause of neonatal mortality and morbidity. In many cases, it has severe life-long consequences for the health and neurological development of the newborn child. More than 50% of all preterm births are spontaneous, and currently there is no effective prevention. Several studies suggest that genetic factors play a role in spontaneous preterm birth (SPTB). However, its genetic background is insufficiently characterized. The aim of the present study was to perform a linkage analysis of X chromosomal markers in SPTB in large northern Finnish families with recurrent SPTBs. We found a significant linkage signal (HLOD â€Š=â€Š3.72) on chromosome locus Xq13.1 when the studied phenotype was being born preterm. There were no significant linkage signals when the studied phenotype was giving preterm deliveries. Two functional candidate genes, those encoding the androgen receptor (AR) and the interleukin-2 receptor gamma subunit (IL2RG), located near this locus were analyzed as candidates for SPTB in subsequent case-control association analyses. Nine single-nucleotide polymorphisms (SNPs) within these genes and an AR exon-1 CAG repeat, which was previously demonstrated to be functionally significant, were analyzed in mothers with preterm delivery (nâ€Š=â€Š272) and their offspring (nâ€Š=â€Š269), and in mothers with exclusively term deliveries (nâ€Š=â€Š201) and their offspring (nâ€Š=â€Š199), all originating from northern Finland. A replication study population consisting of individuals born preterm (nâ€Š=â€Š111) and term (nâ€Š=â€Š197) from southern Finland was also analyzed. Long AR CAG repeats (â‰¥26) were overrepresented and short repeats (â‰¤19) underrepresented in individuals born preterm compared to those born at term. Thus, our linkage and association results emphasize the role of the fetal genome in genetic predisposition to SPTB and implicate AR as a potential novel fetal susceptibility gene for SPTB.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e51378</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23227263" rel="external">23227263</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3515491" rel="external">3515491</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0051378" rel="external">10.1371/journal.pone.0051378</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Robust and sensitive analysis of mouse knockout phenotypes.</h4>
<p class="authors"><span class="author">Karp NA</span>, <span class="author">Melvin D</span>, <span class="author">Sanger Mouse Genetics Project</span> and <span class="author">Mott RF</span></p>
<p class="affiliation">Mouse Informatics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">A significant challenge of in-vivo studies is the identification of phenotypes with a method that is robust and reliable. The challenge arises from practical issues that lead to experimental designs which are not ideal. Breeding issues, particularly in the presence of fertility or fecundity problems, frequently lead to data being collected in multiple batches. This problem is acute in high throughput phenotyping programs. In addition, in a high throughput environment operational issues lead to controls not being measured on the same day as knockouts. We highlight how application of traditional methods, such as a Student's t-Test or a 2-way ANOVA, in these situations give flawed results and should not be used. We explore the use of mixed models using worked examples from Sanger Mouse Genome Project focusing on Dual-Energy X-Ray Absorptiometry data for the analysis of mouse knockout data and compare to a reference range approach. We show that mixed model analysis is more sensitive and less prone to artefacts allowing the discovery of subtle quantitative phenotypes essential for correlating a gene's function to human disease. We demonstrate how a mixed model approach has the additional advantage of being able to include covariates, such as body weight, to separate effect of genotype from these covariates. This is a particular issue in knockout studies, where body weight is a common phenotype and will enhance the precision of assigning phenotypes and the subsequent selection of lines for secondary phenotyping. The use of mixed models with in-vivo studies has value not only in improving the quality and sensitivity of the data analysis but also ethically as a method suitable for small batches which reduces the breeding burden of a colony. This will reduce the use of animals, increase throughput, and decrease cost whilst improving the quality and depth of knowledge gained.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e52410</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23300663" rel="external">23300663</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0052410" rel="external">10.1371/journal.pone.0052410</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The fallacy of ratio correction to address confounding factors.</h4>
<p class="authors"><span class="author">Karp NA</span>, <span class="author">Segonds-Pichon A</span>, <span class="author">Gerdin AK</span>, <span class="author">Ram&iacute;rez-Solis R</span> and <span class="author">White JK</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Scientists aspire to measure cause and effect. Unfortunately confounding variables, ones that are associated with both the probable cause and the outcome, can lead to an association that is true but potentially misleading. For example, altered body weight is often observed in a gene knockout; however, many other variables, such as lean mass, will also change as the body weight changes. This leaves the researcher asking whether the change in that variable is expected for that change in weight. Ratio correction, which is often referred to as normalization, is a method used commonly to remove the effect of a confounding variable. Although ratio correction is used widely in biological research, it is not the method recommended in the statistical literature to address confounding factors; instead regression methods such as the analysis of covariance (ANCOVA) are proposed. This method examines the difference in means after adjusting for the confounding relationship. Using real data, this manuscript demonstrates how the ratio correction approach is flawed and can result in erroneous calls of significance leading to inappropriate biological conclusions. This arises as some of the underlying assumptions are not met. The manuscript goes on to demonstrate that researchers should use ANCOVA, and discusses how graphical tools can be used readily to judge the robustness of this method. This study is therefore a clear example of why assumption testing is an important component of a study and thus why it is included in the Animal Research: Reporting of In Vivo Experiment (ARRIVE) guidelines.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077157/Z/05/Z</span></p>
<p class="publication"><span class="title">Laboratory animals</span>&nbsp;<span class="year">2012</span>;<span class="volume">46</span>;<span class="number">3</span>;<span class="pagerange">245-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22829707">22829707</a>; DOI: <a href="http://dx.doi.org/10.1258%2Fla.2012.012003">10.1258/la.2012.012003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using genome-wide complex trait analysis to quantify 'missing heritability' in Parkinson's disease.</h4>
<p class="authors"><span class="author">Keller MF</span>, <span class="author">Saad M</span>, <span class="author">Bras J</span>, <span class="author">Bettella F</span>, <span class="author">Nicolaou N</span>, <span class="author">Sim&oacute;n-S&aacute;nchez J</span>, <span class="author">Mittag F</span>, <span class="author">B&uuml;chel F</span>, <span class="author">Sharma M</span>, <span class="author">Gibbs JR</span>, <span class="author">Schulte C</span>, <span class="author">Moskvina V</span>, <span class="author">Durr A</span>, <span class="author">Holmans P</span>, <span class="author">Kilarski LL</span>, <span class="author">Guerreiro R</span>, <span class="author">Hernandez DG</span>, <span class="author">Brice A</span>, <span class="author">Ylikotila P</span>, <span class="author">Stef&aacute;nsson H</span>, <span class="author">Majamaa K</span>, <span class="author">Morris HR</span>, <span class="author">Williams N</span>, <span class="author">Gasser T</span>, <span class="author">Heutink P</span>, <span class="author">Wood NW</span>, <span class="author">Hardy J</span>, <span class="author">Martinez M</span>, <span class="author">Singleton AB</span>, <span class="author">Nalls MA</span> and <span class="author">for the International Parkinson&#39;s Disease Genomics Consortium (IPDGC) and The Wellcome Trust Case Control Consortium 2 (WTCCC2)</span></p>
<p class="affiliation">A full list of The International Parkinson's Disease Genomics Consortium (IPDGC) and The Wellcome Trust Case Control Consortium 2 (WTCCC2) members and affiliations appears at the end of this manuscript.</p>
<p class="abstract">Genome-wide association studies (GWASs) have been successful at identifying single-nucleotide polymorphisms (SNPs) highly associated with common traits; however, a great deal of the heritable variation associated with common traits remains unaccounted for within the genome. Genome-wide complex trait analysis (GCTA) is a statistical method that applies a linear mixed model to estimate phenotypic variance of complex traits explained by genome-wide SNPs, including those not associated with the trait in a GWAS. We applied GCTA to 8 cohorts containing 7096 case and 19 455 control individuals of European ancestry in order to examine the missing heritability present in Parkinson's disease (PD). We meta-analyzed our initial results to produce robust heritability estimates for PD types across cohorts. Our results identify 27% (95% CI 17-38, P = 8.08E - 08) phenotypic variance associated with all types of PD, 15% (95% CI -0.2 to 33, P = 0.09) phenotypic variance associated with early-onset PD and 31% (95% CI 17-44, P = 1.34E - 05) phenotypic variance associated with late-onset PD. This is a substantial increase from the genetic variance identified by top GWAS hits alone (between 3 and 5%) and indicates there are substantially more risk loci to be identified. Our results suggest that although GWASs are a useful tool in identifying the most common variants associated with complex disease, a great deal of common variants of small effect remain to be discovered.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">22</span>;<span class="pagerange">4996-5009</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22892372" rel="external">22892372</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fdds335" rel="external">10.1093/hmg/dds335</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reply to &quot;Human genetic studies on osteoarthritis from clinicians&#39; viewpoints&quot;.</h4>
<p class="authors"><span class="author">Kerkhof HJ</span>, <span class="author">Evangelou E</span>, <span class="author">Meulenbelt I</span>, <span class="author">van Meurs JB</span>, <span class="author">Zeggini E</span> and <span class="author">Valdes AM</span></p>
<p class="publication"><span class="title">Osteoarthritis and cartilage / OARS, Osteoarthritis Research Society</span>&nbsp;<span class="year">2012</span>;<span class="volume">20</span>;<span class="number">3</span>;<span class="pagerange">250-1; author reply 252</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22233813" rel="external">22233813</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.joca.2011.09.008" rel="external">10.1016/j.joca.2011.09.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Avidity-based extracellular interaction screening (AVEXIS) for the scalable detection of low-affinity extracellular receptor-ligand interactions.</h4>
<p class="authors"><span class="author">Kerr JS</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute.</p><p class="abstract">Extracellular protein:protein interactions between secreted or membrane-tethered proteins are critical for both initiating intercellular communication and ensuring cohesion within multicellular organisms. Proteins predicted to form extracellular interactions are encoded by approximately a quarter of human genes, but despite their importance and abundance, the majority of these proteins have no documented binding partner. Primarily, this is due to their biochemical intractability: membrane-embedded proteins are difficult to solubilise in their native conformation and contain structurally-important posttranslational modifications. Also, the interaction affinities between receptor proteins are often characterised by extremely low interaction strengths (half-lives &lt; 1 second) precluding their detection with many commonly-used high throughput methods. Here, we describe an assay, AVEXIS (AVidity-based EXtracellular Interaction Screen) that overcomes these technical challenges enabling the detection of very weak protein interactions (t(1/2) &le; 0.1 sec) with a low false positive rate. The assay is usually implemented in a high throughput format to enable the systematic screening of many thousands of interactions in a convenient microtitre plate format (Fig. 1). It relies on the production of soluble recombinant protein libraries that contain the ectodomain fragments of cell surface receptors or secreted proteins within which to screen for interactions; therefore, this approach is suitable for type I, type II, GPI-linked cell surface receptors and secreted proteins but not for multipass membrane proteins such as ion channels or transporters. The recombinant protein libraries are produced using a convenient and high-level mammalian expression system, to ensure that important posttranslational modifications such as glycosylation and disulphide bonds are added. Expressed recombinant proteins are secreted into the medium and produced in two forms: a biotinylated bait which can be captured on a streptavidin-coated solid phase suitable for screening, and a pentamerised enzyme-tagged (&beta;-lactamase) prey. The bait and prey proteins are presented to each other in a binary fashion to detect direct interactions between them, similar to a conventional ELISA (Fig. 1). The pentamerisation of the proteins in the prey is achieved through a peptide sequence from the cartilage oligomeric matrix protein (COMP) and increases the local concentration of the ectodomains thereby providing significant avidity gains to enable even very transient interactions to be detected. By normalising the activities of both the bait and prey to predetermined levels prior to screening, we have shown that interactions having monomeric half-lives of 0.1 sec can be detected with low false positive rates.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077108</span></p>
<p class="publication"><span class="title">Journal of visualized experiments : JoVE</span>&nbsp;<span class="year">2012</span>;<span class="number">61</span>;<span class="pagerange">e3881</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22414956" rel="external">22414956</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3460577" rel="external">3460577</a>; DOI: <a href="http://dx.doi.org/10.3791%2F3881" rel="external">10.3791/3881</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies multiple loci influencing human serum metabolite levels.</h4>
<p class="authors"><span class="author">Kettunen J</span>, <span class="author">Tukiainen T</span>, <span class="author">Sarin AP</span>, <span class="author">Ortega-Alonso A</span>, <span class="author">Tikkanen E</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Kangas AJ</span>, <span class="author">Soininen P</span>, <span class="author">W&uuml;rtz P</span>, <span class="author">Silander K</span>, <span class="author">Dick DM</span>, <span class="author">Rose RJ</span>, <span class="author">Savolainen MJ</span>, <span class="author">Viikari J</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Inouye M</span>, <span class="author">McCarthy MI</span>, <span class="author">Jula A</span>, <span class="author">Eriksson J</span>, <span class="author">Raitakari OT</span>, <span class="author">Salomaa V</span>, <span class="author">Kaprio J</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Peltonen L</span>, <span class="author">Perola M</span>, <span class="author">Freimer NB</span>, <span class="author">Ala-Korpela M</span>, <span class="author">Palotie A</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, University of Helsinki, Finland.</p><p class="abstract">Nuclear magnetic resonance assays allow for measurement of a wide range of metabolic phenotypes. We report here the results of a GWAS on 8,330 Finnish individuals genotyped and imputed at 7.7 million SNPs for a range of 216 serum metabolic phenotypes assessed by NMR of serum samples. We identified significant associations (P &lt; 2.31 &times; 10(-10)) at 31 loci, including 11 for which there have not been previous reports of associations to a metabolic trait or disorder. Analyses of Finnish twin pairs suggested that the metabolic measures reported here show higher heritability than comparable conventional metabolic phenotypes. In accordance with our expectations, SNPs at the 31 loci associated with individual metabolites account for a greater proportion of the genetic component of trait variance (up to 40%) than is typically observed for conventional serum metabolic phenotypes. The identification of such associations may provide substantial insight into cardiometabolic disorders.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600705</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679</span>; NIAAA NIH HHS: <span class="grant">AA-08315</span>, <span class="grant">AA-09203</span>, <span class="grant">AA-12502</span>, <span class="grant">AA-15416</span>, <span class="grant">R01 AA012502</span>, <span class="grant">R01 AA015416</span>, <span class="grant">R37 AA012502</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268</span>; Wellcome Trust: <span class="grant">089062/Z/09/Z</span>, <span class="grant">090532</span>, <span class="grant">098051</span>, <span class="grant">89061/Z/09/Z</span>, <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">269-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22286219">22286219</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605033">3605033</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1073">10.1038/ng.1073</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">How next-generation sequencing is transforming complex disease genetics.</h4>
<p class="authors"><span class="author">Kilpinen H</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Department of Genetic Medicine and Development, University of Geneva Medical School, Geneva 1211, Switzerland; Swiss Institute of Bioinformatics, Geneva 1211, Switzerland.</p>
<p class="abstract">Progress in understanding the genetics of human disease is closely tied to technological developments in DNA sequencing. Recently, next-generation technology has transformed the scale of sequencing; compared to the methods used in the Human Genome Project, modern sequencers are 50000-fold faster. Complex disease genetics presents an immediate opportunity to use this technology to move from approaches using only partial information (linkage and genome-wide association studies, GWAS) to complete analysis of the relationship between genomic variation and phenotype. We first describe sequence-based improvements to existing study designs, followed by prioritization of both samples and genomic regions to be sequenced, and then address the ultimate goal of analyzing thousands of whole-genome sequences. Finally, we discuss how the same technology will also fundamentally change the way we understand the biological mechanisms underlying disease associations discovered through sequencing.</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23103023" rel="external">23103023</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2012.10.001" rel="external">10.1016/j.tig.2012.10.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Properties of a Novel PBP2A Protein Homolog from Staphylococcus aureus Strain LGA251 and Its Contribution to the Î²-Lactam-resistant Phenotype.</h4>
<p class="authors"><span class="author">Kim C</span>, <span class="author">Milheiri&ccedil;o C</span>, <span class="author">Gardete S</span>, <span class="author">Holmes MA</span>, <span class="author">Holden MT</span>, <span class="author">de Lencastre H</span> and <span class="author">Tomasz A</span></p>
<p class="affiliation">From the Laboratory of Microbiology and Infectious Diseases, The Rockefeller University, New York, New York 10065.</p>
<p class="abstract">Methicillin-resistant Staphylococcus aureus (MRSA) strains show strain-to-strain variation in resistance level, in genetic background, and also in the structure of the chromosomal cassette (SCCmec) that carries the resistance gene mecA. In contrast, strain-to-strain variation in the sequence of the mecA determinant was found to be much more limited among MRSA isolates examined so far. The first exception to this came with the recent identification of MRSA strain LGA251, which carries a new homolog of this gene together with regulatory elements mecI/mecR that also have novel, highly divergent structures. After cloning and purification in Escherichia coli, PBP2A(LGA), the protein product of the new mecA homolog, showed aberrant mobility in SDS-PAGE, structural instability and loss of activity at 37 Â°C, and a higher relative affinity for oxacillin as compared with cefoxitin. The mecA homolog free of its regulatory elements was cloned into a plasmid and introduced into the background of the Î²-lactam-susceptible S. aureus strain COL-S. In this background, the mecA homolog expressed a high-level resistance to cefoxitin (MIC = 400 Î¼g/ml) and a somewhat lower resistance to oxacillin (minimal inhibitory concentration = 200 Î¼g/ml). Similar to PBP2A, the protein homolog PBP2A(LGA) was able to replace the essential function of the S. aureus PBP2 for growth. In contrast to PBP2A, PBP2A(LGA) did not depend on the transglycosylase activity of the native PBP2 for expression of high level resistance to oxacillin, suggesting that the PBP2A homolog may preferentially cooperate with a monofunctional transglycosylase as the alternative source of transglycosylase activity.</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2012</span>;<span class="volume">287</span>;<span class="number">44</span>;<span class="pagerange">36854-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22977239" rel="external">22977239</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3481288" rel="external">3481288</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M112.395962" rel="external">10.1074/jbc.M112.395962</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">De novo CNV analysis implicates specific abnormalities of postsynaptic signalling complexes in the pathogenesis of schizophrenia.</h4>
<p class="authors"><span class="author">Kirov G</span>, <span class="author">Pocklington AJ</span>, <span class="author">Holmans P</span>, <span class="author">Ivanov D</span>, <span class="author">Ikeda M</span>, <span class="author">Ruderfer D</span>, <span class="author">Moran J</span>, <span class="author">Chambert K</span>, <span class="author">Toncheva D</span>, <span class="author">Georgieva L</span>, <span class="author">Grozeva D</span>, <span class="author">Fjodorova M</span>, <span class="author">Wollerton R</span>, <span class="author">Rees E</span>, <span class="author">Nikolov I</span>, <span class="author">van de Lagemaat LN</span>, <span class="author">Bay&eacute;s A</span>, <span class="author">Fernandez E</span>, <span class="author">Olason PI</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Komiyama NH</span>, <span class="author">Collins MO</span>, <span class="author">Choudhary J</span>, <span class="author">Stefansson K</span>, <span class="author">Stefansson H</span>, <span class="author">Grant SG</span>, <span class="author">Purcell S</span>, <span class="author">Sklar P</span>, <span class="author">O&#39;Donovan MC</span> and <span class="author">Owen MJ</span></p>
<p class="affiliation">Department of Psychological Medicine and Neurology, MRC Centre for Neuropsychiatric Genetics and Genomics, School of Medicine, Neuroscience and Mental Health Research Institute, Cardiff University, Cardiff, UK. <a href="mailto:%6B%69%72%6F%76%40%63%61%72%64%69%66%66.%61%63.%75%6B">&#107;&#105;&#114;&#111;&#118;&#64;&#99;&#97;&#114;&#100;&#105;&#102;&#102;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A small number of rare, recurrent genomic copy number variants (CNVs) are known to substantially increase susceptibility to schizophrenia. As a consequence of the low fecundity in people with schizophrenia and other neurodevelopmental phenotypes to which these CNVs contribute, CNVs with large effects on risk are likely to be rapidly removed from the population by natural selection. Accordingly, such CNVs must frequently occur as recurrent de novo mutations. In a sample of 662 schizophrenia proband-parent trios, we found that rare de novo CNV mutations were significantly more frequent in cases (5.1% all cases, 5.5% family history negative) compared with 2.2% among 2623 controls, confirming the involvement of de novo CNVs in the pathogenesis of schizophrenia. Eight de novo CNVs occurred at four known schizophrenia loci (3q29, 15q11.2, 15q13.3 and 16p11.2). De novo CNVs of known pathogenic significance in other genomic disorders were also observed, including deletion at the TAR (thrombocytopenia absent radius) region on 1q21.1 and duplication at the WBS (Williams-Beuren syndrome) region at 7q11.23. Multiple de novos spanned genes encoding members of the DLG (discs large) family of membrane-associated guanylate kinases (MAGUKs) that are components of the postsynaptic density (PSD). Two de novos also affected EHMT1, a histone methyl transferase known to directly regulate DLG family members. Using a systems biology approach and merging novel CNV and proteomics data sets, systematic analysis of synaptic protein complexes showed that, compared with control CNVs, case de novos were significantly enriched for the PSD proteome (P=1.72 &times; 10&#x207B;&#x2076;. This was largely explained by enrichment for members of the N-methyl-D-aspartate receptor (NMDAR) (P=4.24 &times; 10&#x207B;&#x2076;) and neuronal activity-regulated cytoskeleton-associated protein (ARC) (P=3.78 &times; 10&#x207B;&#x2078;) postsynaptic signalling complexes. In an analysis of 18&thinsp;492 subjects (7907 cases and 10&thinsp;585 controls), case CNVs were enriched for members of the NMDAR complex (P=0.0015) but not ARC (P=0.14). Our data indicate that defects in NMDAR postsynaptic signalling and, possibly, ARC complexes, which are known to be important in synaptic plasticity and cognition, play a significant role in the pathogenesis of schizophrenia.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800509</span>; NIMH NIH HHS: <span class="grant">MH066392-05A1</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2012</span>;<span class="volume">17</span>;<span class="number">2</span>;<span class="pagerange">142-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22083728">22083728</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3603134">3603134</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2011.154">10.1038/mp.2011.154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression profiles in white blood cells of volunteers exposed to a 50 Hz electromagnetic field.</h4>
<p class="authors"><span class="author">Kirschenlohr H</span>, <span class="author">Ellis P</span>, <span class="author">Hesketh R</span> and <span class="author">Metcalfe J</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, Sanger Building, Cambridge, CB2 1GA, United Kingdom.</p><p class="abstract">Consistent and independently replicated laboratory evidence to support a causative relationship between environmental exposure to extremely low-frequency electromagnetic fields (EMFs) at power line frequencies and the associated increase in risk of childhood leukemia has not been obtained. In particular, although gene expression responses have been reported in a wide variety of cells, none has emerged as robust, widely replicated effects. DNA microarrays facilitate comprehensive searches for changes in gene expression without a requirement to select candidate responsive genes. To determine if gene expression changes occur in white blood cells of volunteers exposed to an ELF-EMF, each of 17 pairs of male volunteers age 20-30 was subjected either to a 50 Hz EMF exposure of 62.0 &plusmn; 7.1 &mu;T for 2 h or to a sham exposure (0.21 &plusmn; 0.05 &mu;T) at the same time (11:00 a.m. to 13:00 p.m.). The alternative regime for each volunteer was repeated on the following day and the two-day sequence was repeated 6 days later, with the exception that a null exposure (0.085 &plusmn; 0.01 &mu;T) replaced the sham exposure. Five blood samples (10 ml) were collected at 2 h intervals from 9:00 to 17:00 with five additional samples during the exposure and sham or null exposure periods on each study day. RNA samples were pooled for the same time on each study day for the group of 17 volunteers that were subjected to the ELF-EMF exposure/sham or null exposure sequence and were analyzed on Illumina microarrays. Time courses for 16 mammalian genes previously reported to be responsive to ELF-EMF exposure, including immediate early genes, stress response, cell proliferation and apoptotic genes were examined in detail. No genes or gene sets showed consistent response profiles to repeated ELF-EMF exposures. A stress response was detected as a transient increase in plasma cortisol at the onset of either exposure or sham exposure on the first study day. The cortisol response diminished progressively on subsequent exposures or sham exposures, and was attributable to mild stress associated with the experimental protocol.</p>
<p class="publication"><span class="title">Radiation research</span>&nbsp;<span class="year">2012</span>;<span class="volume">178</span>;<span class="number">3</span>;<span class="pagerange">138-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22856684">22856684</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural genomics plucks high-hanging membrane proteins.</h4>
<p class="authors"><span class="author">Kloppmann E</span>, <span class="author">Punta M</span> and <span class="author">Rost B</span></p>
<p class="affiliation">Department of Bioinformatics and Computational Biology, Technical University Munich, Boltzmannstr. 3, 85748 Garching/Munich, Germany; New York Consortium on Membrane Protein Structure, New York Structural Biology Center, 89 Convent Avenue, New York, NY 10027, USA.</p><p class="abstract">Recent years have seen the establishment of structural genomics centers that explicitly target integral membrane proteins. Here, we review the advances in targeting these extremely high-hanging fruits of structural biology in high-throughput mode. We observe that the experimental determination of high-resolution structures of integral membrane proteins is increasingly successful both in terms of getting structures and of covering important protein families, for example, from Pfam. Structural genomics has begun to contribute significantly toward this progress. An important component of this contribution is the set up of robotic pipelines that generate a wealth of experimental data for membrane proteins. We argue that prediction methods for the identification of membrane regions and for the comparison of membrane proteins largely suffice to meet the challenges of target selection for structural genomics of membrane proteins. In contrast, we need better methods to prioritize the most promising members in a family of closely related proteins and to annotate protein function from sequence and structure in absence of homology.</p>
<p class="publication"><span class="title">Current opinion in structural biology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22622032" rel="external">22622032</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.sbi.2012.05.002" rel="external">10.1016/j.sbi.2012.05.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The impact of diabetes on the pathogenesis of sepsis.</h4>
<p class="authors"><span class="author">Koh GC</span>, <span class="author">Peacock SJ</span>, <span class="author">van der Poll T</span> and <span class="author">Wiersinga WJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%67%61%76%69%6E.%6B%6F%68%40%67%6D%61%69%6C.%63%6F%6D">&#103;&#97;&#118;&#105;&#110;&#46;&#107;&#111;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Diabetes is associated with an increased susceptibility to infection and sepsis. Conflicting data exist on whether the mortality of patients with sepsis is influenced by the presence of diabetes, fuelling the ongoing debate on the benefit of tight glucose regulation in patients with sepsis. The main reason for which diabetes predisposes to infection appears to be abnormalities of the host response, particularly in neutrophil chemotaxis, adhesion and intracellular killing, defects that have been attributed to the effect of hyperglycaemia. There is also evidence for defects in humoral immunity, and this may play a larger role than previously recognised. We review the literature on the immune response in diabetes and its potential contribution to the pathogenesis of sepsis. In addition, the effect of diabetes treatment on the immune response is discussed, with specific reference to insulin, metformin, sulphonylureas and thiazolidinediones.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">086532/Z/08/Z</span></p>
<p class="publication"><span class="title">European journal of clinical microbiology &amp; infectious diseases : official publication of the European Society of Clinical Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">31</span>;<span class="number">4</span>;<span class="pagerange">379-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21805196" rel="external">21805196</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3303037" rel="external">3303037</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10096-011-1337-4" rel="external">10.1007/s10096-011-1337-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of vancomycin-resistant Staphylococcus aureus strains and their positions within the clade most commonly associated with Methicillin-resistant S. aureus hospital-acquired infection in the United States.</h4>
<p class="authors"><span class="author">Kos VN</span>, <span class="author">Desjardins CA</span>, <span class="author">Griggs A</span>, <span class="author">Cerqueira G</span>, <span class="author">Van Tonder A</span>, <span class="author">Holden MT</span>, <span class="author">Godfrey P</span>, <span class="author">Palmer KL</span>, <span class="author">Bodi K</span>, <span class="author">Mongodin EF</span>, <span class="author">Wortman J</span>, <span class="author">Feldgarden M</span>, <span class="author">Lawley T</span>, <span class="author">Gill SR</span>, <span class="author">Haas BJ</span>, <span class="author">Birren B</span> and <span class="author">Gilmore MS</span></p>
<p class="affiliation">Department of Ophthalmology, Harvard Microbial Sciences Initiative, Harvard Medical School, Massachusetts Eye and Ear Infirmary, Boston, Massachusetts, USA.</p>
<p class="abstract">Methicillin-resistant Staphylococcus aureus (MRSA) strains are leading causes of hospital-acquired infections in the United States, and clonal cluster 5 (CC5) is the predominant lineage responsible for these infections. Since 2002, there have been 12 cases of vancomycin-resistant S. aureus (VRSA) infection in the United States-all CC5 strains. To understand this genetic background and what distinguishes it from other lineages, we generated and analyzed high-quality draft genome sequences for all available VRSA strains. Sequence comparisons show unambiguously that each strain independently acquired Tn1546 and that all VRSA strains last shared a common ancestor over 50 years ago, well before the occurrence of vancomycin resistance in this species. In contrast to existing hypotheses on what predisposes this lineage to acquire Tn1546, the barrier posed by restriction systems appears to be intact in most VRSA strains. However, VRSA (and other CC5) strains were found to possess a constellation of traits that appears to be optimized for proliferation in precisely the types of polymicrobic infection where transfer could occur. They lack a bacteriocin operon that would be predicted to limit the occurrence of non-CC5 strains in mixed infection and harbor a cluster of unique superantigens and lipoproteins to confound host immunity. A frameshift in dprA, which in other microbes influences uptake of foreign DNA, may also make this lineage conducive to foreign DNA acquisition. IMPORTANCE: Invasive methicillin-resistant Staphylococcus aureus (MRSA) infection now ranks among the leading causes of death in the United States. Vancomycin is a key last-line bactericidal drug for treating these infections. However, since 2002, vancomycin resistance has entered this species. Of the now 12 cases of vancomycin-resistant S. aureus (VRSA), each was believed to represent a new acquisition of the vancomycin-resistant transposon Tn1546 from enterococcal donors. All acquisitions of Tn1546 so far have occurred in MRSA strains of the clonal cluster 5 genetic background, the most common hospital lineage causing hospital-acquired MRSA infection. To understand the nature of these strains, we determined and examined the nucleotide sequences of the genomes of all available VRSA. Genome comparison identified candidate features that position strains of this lineage well for acquiring resistance to antibiotics in mixed infection.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY017381</span>, <span class="grant">R01 EY017381</span>; NIAID NIH HHS: <span class="grant">AI083214</span>, <span class="grant">P01 AI083214</span>, <span class="grant">R01 AI072360</span>; PHS HHS: <span class="grant">HHSN272200700055C</span>, <span class="grant">HHSN27220090018C</span></p>
<p class="publication"><span class="title">mBio</span>&nbsp;<span class="year">2012</span>;<span class="volume">3</span>;<span class="number">3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22617140" rel="external">22617140</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3372964" rel="external">3372964</a>; DOI: <a href="http://dx.doi.org/10.1128%2FmBio.00112-12" rel="external">10.1128/mBio.00112-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transposon mutagenesis identifies genes that transform neural stem cells into glioma-initiating cells.</h4>
<p class="authors"><span class="author">Koso H</span>, <span class="author">Takeda H</span>, <span class="author">Yew CC</span>, <span class="author">Ward JM</span>, <span class="author">Nariai N</span>, <span class="author">Ueno K</span>, <span class="author">Nagasaki M</span>, <span class="author">Watanabe S</span>, <span class="author">Rust AG</span>, <span class="author">Adams DJ</span>, <span class="author">Copeland NG</span> and <span class="author">Jenkins NA</span></p>
<p class="affiliation">Division of Genetics and Genomics, Institute of Molecular and Cell Biology, Agency for Science, Technology and Research, Singapore 138673.</p><p class="abstract">Neural stem cells (NSCs) are considered to be the cell of origin of glioblastoma multiforme (GBM). However, the genetic alterations that transform NSCs into glioma-initiating cells remain elusive. Using a unique transposon mutagenesis strategy that mutagenizes NSCs in culture, followed by additional rounds of mutagenesis to generate tumors in vivo, we have identified genes and signaling pathways that can transform NSCs into glioma-initiating cells. Mobilization of Sleeping Beauty transposons in NSCs induced the immortalization of astroglial-like cells, which were then able to generate tumors with characteristics of the mesenchymal subtype of GBM on transplantation, consistent with a potential astroglial origin for mesenchymal GBM. Sequence analysis of transposon insertion sites from tumors and immortalized cells identified more than 200 frequently mutated genes, including human GBM-associated genes, such as Met and Nf1, and made it possible to discriminate between genes that function during astroglial immortalization vs. later stages of tumor development. We also functionally validated five GBM candidate genes using a previously undescribed high-throughput method. Finally, we show that even clonally related tumors derived from the same immortalized line have acquired distinct combinations of genetic alterations during tumor development, suggesting that tumor formation in this model system involves competition among genetically variant cells, which is similar to the Darwinian evolutionary processes now thought to generate many human cancers. This mutagenesis strategy is faster and simpler than conventional transposon screens and can potentially be applied to any tissue stem/progenitor cells that can be grown and differentiated in vitro.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">44</span>;<span class="pagerange">E2998-3007</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23045694">23045694</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3497753">3497753</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1215899109">10.1073/pnas.1215899109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel mutations consolidate KCTD7 as a progressive myoclonus epilepsy gene.</h4>
<p class="authors"><span class="author">Kousi M</span>, <span class="author">Anttila V</span>, <span class="author">Schulz A</span>, <span class="author">Calafato S</span>, <span class="author">Jakkula E</span>, <span class="author">Riesch E</span>, <span class="author">Myllykangas L</span>, <span class="author">Kalimo H</span>, <span class="author">Top&ccedil;u M</span>, <span class="author">G&ouml;kben S</span>, <span class="author">Alehan F</span>, <span class="author">Lemke JR</span>, <span class="author">Alber M</span>, <span class="author">Palotie A</span>, <span class="author">Kopra O</span> and <span class="author">Lehesjoki AE</span></p>
<p class="affiliation">Folkh&auml;lsan Institute of Genetics, Biomedicum Helsinki, PO Box 63, Haartmaninkatu 8, University of Helsinki, FIN-00014 Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: The progressive myoclonus epilepsies (PMEs) comprise a group of clinically and genetically heterogeneous disorders characterised by myoclonus, epilepsy, and neurological deterioration. This study aimed to identify the underlying gene(s) in childhood onset PME patients with unknown molecular genetic background.</p><p class="abstract"><strong>Methods</strong>: Homozygosity mapping was applied on genome-wide single nucleotide polymorphism data of 18 Turkish patients. The potassium channel tetramerisation domain-containing 7 (KCTD7) gene, previously associated with PME in a single inbred family, was screened for mutations. The spatiotemporal expression of KCTD7 was assessed in cellular cultures and mouse brain tissue.</p><p class="abstract"><strong>Results</strong>: Overlapping homozygosity in 8/18 patients defined a 1.5 Mb segment on 7q11.21 as the major candidate locus. Screening of the positional candidate gene KCTD7 revealed homozygous missense mutations in two of the eight cases. Screening of KCTD7 in a further 132 PME patients revealed four additional mutations (two missense, one in-frame deletion, and one frameshift-causing) in five families. Eight patients presented with myoclonus and epilepsy and one with ataxia, the mean age of onset being 19 months. Within 2 years after onset, progressive loss of mental and motor skills ensued leading to severe dementia and motor handicap. KCTD7 showed cytosolic localisation and predominant neuronal expression, with widespread expression throughout the brain. None of three polypeptides carrying patient missense mutations affected the subcellular distribution of KCTD7.</p><p class="abstract"><strong>Discussion</strong>: These data confirm the causality of KCTD7 defects in PME, and imply that KCTD7 mutation screening should be considered in PME patients with onset around 2 years of age followed by rapid mental and motor deterioration.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">49</span>;<span class="number">6</span>;<span class="pagerange">391-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22693283">22693283</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmedgenet-2012-100859">10.1136/jmedgenet-2012-100859</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The transcriptional landscape and small RNAs of Salmonella enterica serovar Typhimurium.</h4>
<p class="authors"><span class="author">Kr&ouml;ger C</span>, <span class="author">Dillon SC</span>, <span class="author">Cameron AD</span>, <span class="author">Papenfort K</span>, <span class="author">Sivasankaran SK</span>, <span class="author">Hokamp K</span>, <span class="author">Chao Y</span>, <span class="author">Sittka A</span>, <span class="author">H&eacute;brard M</span>, <span class="author">H&auml;ndler K</span>, <span class="author">Colgan A</span>, <span class="author">Leekitcharoenphon P</span>, <span class="author">Langridge GC</span>, <span class="author">Lohan AJ</span>, <span class="author">Loftus B</span>, <span class="author">Lucchini S</span>, <span class="author">Ussery DW</span>, <span class="author">Dorman CJ</span>, <span class="author">Thomson NR</span>, <span class="author">Vogel J</span> and <span class="author">Hinton JC</span></p>
<p class="affiliation">Department of Microbiology, School of Genetics and Microbiology, Moyne Institute of Preventive Medicine, Trinity College Dublin, Dublin 2, Ireland.</p><p class="abstract">More than 50 y of research have provided great insight into the physiology, metabolism, and molecular biology of Salmonella enterica serovar Typhimurium (S. Typhimurium), but important gaps in our knowledge remain. It is clear that a precise choreography of gene expression is required for Salmonella infection, but basic genetic information such as the global locations of transcription start sites (TSSs) has been lacking. We combined three RNA-sequencing techniques and two sequencing platforms to generate a robust picture of transcription in S. Typhimurium. Differential RNA sequencing identified 1,873 TSSs on the chromosome of S. Typhimurium SL1344 and 13% of these TSSs initiated antisense transcripts. Unique findings include the TSSs of the virulence regulators phoP, slyA, and invF. Chromatin immunoprecipitation revealed that RNA polymerase was bound to 70% of the TSSs, and two-thirds of these TSSs were associated with &sigma;(70) (including phoP, slyA, and invF) from which we identified the -10 and -35 motifs of &sigma;(70)-dependent S. Typhimurium gene promoters. Overall, we corrected the location of important genes and discovered 18 times more promoters than identified previously. S. Typhimurium expresses 140 small regulatory RNAs (sRNAs) at early stationary phase, including 60 newly identified sRNAs. Almost half of the experimentally verified sRNAs were found to be unique to the Salmonella genus, and &lt;20% were found throughout the Enterobacteriaceae. This description of the transcriptional map of SL1344 advances our understanding of S. Typhimurium, arguably the most important bacterial infection model.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">20</span>;<span class="pagerange">E1277-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22538806">22538806</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3356629">3356629</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1201061109">10.1073/pnas.1201061109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrated molecular profiles of invasive breast tumors and ductal carcinoma in situ (DCIS) reveal differential vascular and interleukin signaling.</h4>
<p class="authors"><span class="author">Kristensen VN</span>, <span class="author">Vaske CJ</span>, <span class="author">Ursini-Siegel J</span>, <span class="author">Van Loo P</span>, <span class="author">Nordgard SH</span>, <span class="author">Sachidanandam R</span>, <span class="author">S&oslash;rlie T</span>, <span class="author">W&auml;rnberg F</span>, <span class="author">Haakensen VD</span>, <span class="author">Helland &Aring;</span>, <span class="author">Naume B</span>, <span class="author">Perou CM</span>, <span class="author">Haussler D</span>, <span class="author">Troyanskaya OG</span> and <span class="author">B&oslash;rresen-Dale AL</span></p>
<p class="affiliation">Department of Genetics, Institute for Cancer Research, and Cancer Clinic, Oslo University Hospital Radiumhospitalet, 0310 Oslo, Norway.</p><p class="abstract">We use an integrated approach to understand breast cancer heterogeneity by modeling mRNA, copy number alterations, microRNAs, and methylation in a pathway context utilizing the pathway recognition algorithm using data integration on genomic models (PARADIGM). We demonstrate that combining mRNA expression and DNA copy number classified the patients in groups that provide the best predictive value with respect to prognosis and identified key molecular and stromal signatures. A chronic inflammatory signature, which promotes the development and/or progression of various epithelial tumors, is uniformly present in all breast cancers. We further demonstrate that within the adaptive immune lineage, the strongest predictor of good outcome is the acquisition of a gene signature that favors a high T-helper 1 (Th1)/cytotoxic T-lymphocyte response at the expense of Th2-driven humoral immunity. Patients who have breast cancer with a basal HER2-negative molecular profile (PDGM2) are characterized by high expression of protumorigenic Th2/humoral-related genes (24-38%) and a low Th1/Th2 ratio. The luminal molecular subtypes are again differentiated by low or high FOXM1 and ERBB4 signaling. We show that the interleukin signaling profiles observed in invasive cancers are absent or weakly expressed in healthy tissue but already prominent in ductal carcinoma in situ, together with ECM and cell-cell adhesion regulating pathways. The most prominent difference between low and high mammographic density in healthy breast tissue by PARADIGM was that of STAT4 signaling. In conclusion, by means of a pathway-based modeling methodology (PARADIGM) integrating different layers of molecular data from whole-tumor samples, we demonstrate that we can stratify immune signatures that predict patient survival.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCI NIH HHS: <span class="grant">P50-CA58223</span>, <span class="grant">R01 CA138255</span>, <span class="grant">R01-CA138255</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">8</span>;<span class="pagerange">2802-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21908711" rel="external">21908711</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3286992" rel="external">3286992</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1108781108" rel="external">10.1073/pnas.1108781108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Auditory function in the Tc1 mouse model of down syndrome suggests a limited region of human chromosome 21 involved in otitis media.</h4>
<p class="authors"><span class="author">Kuhn S</span>, <span class="author">Ingham N</span>, <span class="author">Pearson S</span>, <span class="author">Gribble SM</span>, <span class="author">Clayton S</span>, <span class="author">Steel KP</span> and <span class="author">Marcotti W</span></p>
<p class="affiliation">Department of Biomedical Science, University of Sheffield, Sheffield, United Kingdom.</p>
<p class="abstract">Down syndrome is one of the most common congenital disorders leading to a wide range of health problems in humans, including frequent otitis media. The Tc1 mouse carries a significant part of human chromosome 21 (Hsa21) in addition to the full set of mouse chromosomes and shares many phenotypes observed in humans affected by Down syndrome with trisomy of chromosome 21. However, it is unknown whether Tc1 mice exhibit a hearing phenotype and might thus represent a good model for understanding the hearing loss that is common in Down syndrome. In this study we carried out a structural and functional assessment of hearing in Tc1 mice. Auditory brainstem response (ABR) measurements in Tc1 mice showed normal thresholds compared to littermate controls and ABR waveform latencies and amplitudes were equivalent to controls. The gross anatomy of the middle and inner ears was also similar between Tc1 and control mice. The physiological properties of cochlear sensory receptors (inner and outer hair cells: IHCs and OHCs) were investigated using single-cell patch clamp recordings from the acutely dissected cochleae. Adult Tc1 IHCs exhibited normal resting membrane potentials and expressed all K(+) currents characteristic of control hair cells. However, the size of the large conductance (BK) Ca(2+) activated K(+) current (I(K,f)), which enables rapid voltage responses essential for accurate sound encoding, was increased in Tc1 IHCs. All physiological properties investigated in OHCs were indistinguishable between the two genotypes. The normal functional hearing and the gross structural anatomy of the middle and inner ears in the Tc1 mouse contrast to that observed in the Ts65Dn model of Down syndrome which shows otitis media. Genes that are trisomic in Ts65Dn but disomic in Tc1 may predispose to otitis media when an additional copy is active.</p>
<p class="grants">Funded by: Action on Hearing Loss: <span class="grant">G41</span>; Wellcome Trust: <span class="grant">077189</span>, <span class="grant">088719</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e31433</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22348087" rel="external">22348087</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3279367" rel="external">3279367</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0031433" rel="external">10.1371/journal.pone.0031433</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid turnover of long noncoding RNAs and the evolution of gene expression.</h4>
<p class="authors"><span class="author">Kutter C</span>, <span class="author">Watt S</span>, <span class="author">Stefflova K</span>, <span class="author">Wilson MD</span>, <span class="author">Goncalves A</span>, <span class="author">Ponting CP</span>, <span class="author">Odom DT</span> and <span class="author">Marques AC</span></p>
<p class="affiliation">Cancer Research UK, Cambridge Research Institute, Li Ka Shing Centre, Cambridge, United Kingdom.</p><p class="abstract">A large proportion of functional sequence within mammalian genomes falls outside protein-coding exons and can be transcribed into long RNAs. However, the roles in mammalian biology of long noncoding RNA (lncRNA) are not well understood. Few lncRNAs have experimentally determined roles, with some of these being lineage-specific. Determining the extent by which transcription of lncRNA loci is retained or lost across multiple evolutionary lineages is essential if we are to understand their contribution to mammalian biology and to lineage-specific traits. Here, we experimentally investigated the conservation of lncRNA expression among closely related rodent species, allowing the evolution of DNA sequence to be uncoupled from evolution of transcript expression. We generated total RNA (RNAseq) and H3K4me3-bound (ChIPseq) DNA data, and combined both to construct catalogues of transcripts expressed in the adult liver of Mus musculus domesticus (C57BL/6J), Mus musculus castaneus, and Rattus norvegicus. We estimated the rate of transcriptional turnover of lncRNAs and investigated the effects of their lineage-specific birth or death. LncRNA transcription showed considerably greater gain and loss during rodent evolution, compared with protein-coding genes. Nucleotide substitution rates were found to mirror the in vivo transcriptional conservation of intergenic lncRNAs between rodents: only the sequences of noncoding loci with conserved transcription were constrained. Finally, we found that lineage-specific intergenic lncRNAs appear to be associated with modestly elevated expression of genomically neighbouring protein-coding genes. Our findings show that nearly half of intergenic lncRNA loci have been gained or lost since the last common ancestor of mouse and rat, and they predict that such rapid transcriptional turnover contributes to the evolution of tissue- and lineage-specific gene expression.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">15603</span>, <span class="grant">A15603</span>; European Research Council: <span class="grant">202218</span>; Medical Research Council: <span class="grant">MC_U137761446</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">e1002841</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22844254">22844254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3406015">3406015</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002841">10.1371/journal.pgen.1002841</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The population pharmacokinetics of R- and S-warfarin: effect of genetic and clinical factors.</h4>
<p class="authors"><span class="author">Lane S</span>, <span class="author">Al-Zubiedi S</span>, <span class="author">Hatch E</span>, <span class="author">Matthews I</span>, <span class="author">Jorgensen AL</span>, <span class="author">Deloukas P</span>, <span class="author">Daly AK</span>, <span class="author">Park BK</span>, <span class="author">Aarons L</span>, <span class="author">Ogungbenro K</span>, <span class="author">Kamali F</span>, <span class="author">Hughes D</span> and <span class="author">Pirmohamed M</span></p>
<p class="affiliation">Department of Biostatstics, Brownlow Street, University of Liverpool, Liverpool L69 3GS, UK. <a href="mailto:%73%6C%61%6E%65%40%6C%69%76%65%72%70%6F%6F%6C.%61%63.%75%6B">&#115;&#108;&#97;&#110;&#101;&#64;&#108;&#105;&#118;&#101;&#114;&#112;&#111;&#111;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Warfarin is a drug with a narrow therapeutic index and large interindividual variability in daily dosing requirements. Patients commencing warfarin treatment are at risk of bleeding due to excessive anticoagulation caused by overdosing. The interindividual variability in dose requirements is influenced by a number of factors, including polymorphisms in genes mediating warfarin pharmacology, co-medication, age, sex, body size and diet.</p><p class="abstract"><strong>Aims</strong>: To develop population pharmacokinetic models of both R- and S-warfarin using clinical and genetic factors and to identify the covariates which influence the interindividual variability in the pharmacokinetic parameters of clearance and volume of distribution in patients on long-term warfarin therapy.</p><p class="abstract"><strong>Methods</strong>: Patients commencing warfarin therapy were followed up for 26 weeks. Plasma warfarin enantiomer concentrations were determined in 306 patients for S-warfarin and in 309 patients for R-warfarin at 1, 8 and 26 weeks. Patients were also genotyped for CYP2C9 variants (CYP2C9*1,*2 and *3), two single-nucleotide polymorphisms (SNPs) in CYP1A2, one SNP in CYP3A4 and six SNPs in CYP2C19. A base pharmacokinetic model was developed using NONMEM software to determine the warfarin clearance and volume of distribution. The model was extended to include covariates that influenced the between-subject variability.</p><p class="abstract"><strong>Results</strong>: Bodyweight, age, sex and CYP2C9 genotype significantly influenced S-warfarin clearance. The S-warfarin clearance was estimated to be 0.144&emsp;l&emsp;h&#x207B;&sup1; (95% confidence interval 0.131, 0.157) in a 70&emsp;kg woman aged 69.8 years with the wild-type CYP2C9 genotype, and the volume of distribution was 16.6&emsp;l (95% confidence interval 13.5, 19.7). Bodyweight and age, along with the SNPs rs3814637 (in CYP2C19) and rs2242480 (in CYP3A4), significantly influenced R-warfarin clearance. The R-warfarin clearance was estimated to be 0.125&emsp;l&emsp;h&#x207B;&sup1; (95% confidence interval 0.115, 0.135) in a 70&emsp;kg individual aged 69.8 years with the wild-type CYP2C19 and CYP3A4 genotypes, and the volume of distribution was 10.9&emsp;l (95% confidence interval 8.63, 13.2).</p><p class="abstract"><strong>Conclusions</strong>: Our analysis, based on exposure rather than dose, provides quantitative estimates of the clinical and genetic factors impacting on the clearance of both the S- and R-enantiomers of warfarin, which can be used in developing improved dosing algorithms.</p>
<p class="grants">Funded by: Department of Health; Wellcome Trust</p>
<p class="publication"><span class="title">British journal of clinical pharmacology</span>&nbsp;<span class="year">2012</span>;<span class="volume">73</span>;<span class="number">1</span>;<span class="pagerange">66-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21692828" rel="external">21692828</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3248257" rel="external">3248257</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2125.2011.04051.x" rel="external">10.1111/j.1365-2125.2011.04051.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeted restoration of the intestinal microbiota with a simple, defined bacteriotherapy resolves relapsing Clostridium difficile disease in mice.</h4>
<p class="authors"><span class="author">Lawley TD</span>, <span class="author">Clare S</span>, <span class="author">Walker AW</span>, <span class="author">Stares MD</span>, <span class="author">Connor TR</span>, <span class="author">Raisen C</span>, <span class="author">Goulding D</span>, <span class="author">Rad R</span>, <span class="author">Schreiber F</span>, <span class="author">Brandt C</span>, <span class="author">Deakin LJ</span>, <span class="author">Pickard DJ</span>, <span class="author">Duncan SH</span>, <span class="author">Flint HJ</span>, <span class="author">Clark TG</span>, <span class="author">Parkhill J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom. <a href="mailto:%74%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Relapsing C. difficile disease in humans is linked to a pathological imbalance within the intestinal microbiota, termed dysbiosis, which remains poorly understood. We show that mice infected with epidemic C. difficile (genotype 027/BI) develop highly contagious, chronic intestinal disease and persistent dysbiosis characterized by a distinct, simplified microbiota containing opportunistic pathogens and altered metabolite production. Chronic C. difficile 027/BI infection was refractory to vancomycin treatment leading to relapsing disease. In contrast, treatment of C. difficile 027/BI infected mice with feces from healthy mice rapidly restored a diverse, healthy microbiota and resolved C. difficile disease and contagiousness. We used this model to identify a simple mixture of six phylogenetically diverse intestinal bacteria, including novel species, which can re-establish a health-associated microbiota and clear C. difficile 027/BI infection from mice. Thus, targeting a dysbiotic microbiota with a defined mixture of phylogenetically diverse bacteria can trigger major shifts in the microbial community structure that displaces C. difficile and, as a result, resolves disease and contagiousness. Further, we demonstrate a rational approach to harness the therapeutic potential of health-associated microbial communities to treat C. difficile disease and potentially other forms of intestinal dysbiosis.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">93614</span>, <span class="grant">G0901743</span>; Wellcome Trust: <span class="grant">076964</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">e1002995</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23133377">23133377</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3486913">3486913</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002995">10.1371/journal.ppat.1002995</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization and gene expression analysis of the cir multi-gene family of Plasmodium chabaudi chabaudi (AS).</h4>
<p class="authors"><span class="author">Lawton J</span>, <span class="author">Brugat T</span>, <span class="author">Yan YX</span>, <span class="author">Reid AJ</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Otto TD</span>, <span class="author">Pain A</span>, <span class="author">Jackson A</span>, <span class="author">Berriman M</span>, <span class="author">Cunningham D</span>, <span class="author">Preiser P</span> and <span class="author">Langhorne J</span></p>
<p class="affiliation">Division of Parasitology, MRC National Institute for Medical Research, London, UK.</p><p class="abstract"><strong>Background</strong>: The pir genes comprise the largest multi-gene family in Plasmodium, with members found in P. vivax, P. knowlesi and the rodent malaria species. Despite comprising up to 5% of the genome, little is known about the functions of the proteins encoded by pir genes. P. chabaudi causes chronic infection in mice, which may be due to antigenic variation. In this model, pir genes are called cirs and may be involved in this mechanism, allowing evasion of host immune responses. In order to fully understand the role(s) of CIR proteins during P. chabaudi infection, a detailed characterization of the cir gene family was required.</p><p class="abstract"><strong>Results</strong>: The cir repertoire was annotated and a detailed bioinformatic characterization of the encoded CIR proteins was performed. Two major sub-families were identified, which have been named A and B. Members of each sub-family displayed different amino acid motifs, and were thus predicted to have undergone functional divergence. In addition, the expression of the entire cir repertoire was analyzed via RNA sequencing and microarray. Up to 40% of the cir gene repertoire was expressed in the parasite population during infection, and dominant cir transcripts could be identified. In addition, some differences were observed in the pattern of expression between the cir subgroups at the peak of P. chabaudi infection. Finally, specific cir genes were expressed at different time points during asexual blood stages.</p><p class="abstract"><strong>Conclusions</strong>: In conclusion, the large number of cir genes and their expression throughout the intraerythrocytic cycle of development indicates that CIR proteins are likely to be important for parasite survival. In particular, the detection of dominant cir transcripts at the peak of P. chabaudi infection supports the idea that CIR proteins are expressed, and could perform important functions in the biology of this parasite. Further application of the methodologies described here may allow the elucidation of CIR sub-family A and B protein functions, including their contribution to antigenic variation and immune evasion.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_EX_G0901345</span>, <span class="grant">U117584248</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">125</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22458863">22458863</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3384456">3384456</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-125">10.1186/1471-2164-13-125</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comprehensive sequence analysis of nine Usher syndrome genes in the UK National Collaborative Usher Study.</h4>
<p class="authors"><span class="author">Le Quesne Stabej P</span>, <span class="author">Saihan Z</span>, <span class="author">Rangesh N</span>, <span class="author">Steele-Stallard HB</span>, <span class="author">Ambrose J</span>, <span class="author">Coffey A</span>, <span class="author">Emmerson J</span>, <span class="author">Haralambous E</span>, <span class="author">Hughes Y</span>, <span class="author">Steel KP</span>, <span class="author">Luxon LM</span>, <span class="author">Webster AR</span> and <span class="author">Bitner-Glindzicz M</span></p>
<p class="affiliation">Clinical and Molecular Genetics, Institute of Child Health, UCL, London, UK.</p><p class="abstract"><strong>Background</strong>: Usher syndrome (USH) is an autosomal recessive disorder comprising retinitis pigmentosa, hearing loss and, in some cases, vestibular dysfunction. It is clinically and genetically heterogeneous with three distinctive clinical types (I-III) and nine Usher genes identified. This study is a comprehensive clinical and genetic analysis of 172 Usher patients and evaluates the contribution of digenic inheritance.</p><p class="abstract"><strong>Methods</strong>: The genes MYO7A, USH1C, CDH23, PCDH15, USH1G, USH2A, GPR98, WHRN, CLRN1 and the candidate gene SLC4A7 were sequenced in 172 UK Usher patients, regardless of clinical type.</p><p class="abstract"><strong>Results</strong>: No subject had definite mutations (nonsense, frameshift or consensus splice site mutations) in two different USH genes. Novel missense variants were classified UV1-4 (unclassified variant): UV4 is &#39;probably pathogenic&#39;, based on control frequency &lt;0.23%, identification in trans to a pathogenic/probably pathogenic mutation and segregation with USH in only one family; and UV3 (&#39;likely pathogenic&#39;) as above, but no information on phase. Overall 79% of identified pathogenic/UV4/UV3 variants were truncating and 21% were missense changes. MYO7A accounted for 53.2%, and USH1C for 14.9% of USH1 families (USH1C:c.496+1G&gt;A being the most common USH1 mutation in the cohort). USH2A was responsible for 79.3% of USH2 families and GPR98 for only 6.6%. No mutations were found in USH1G, WHRN or SLC4A7.</p><p class="abstract"><strong>Conclusions</strong>: One or two pathogenic/likely pathogenic variants were identified in 86% of cases. No convincing cases of digenic inheritance were found. It is concluded that digenic inheritance does not make a significant contribution to Usher syndrome; the observation of multiple variants in different genes is likely to reflect polymorphic variation, rather than digenic effects.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">49</span>;<span class="number">1</span>;<span class="pagerange">27-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22135276">22135276</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3678402">3678402</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmedgenet-2011-100468">10.1136/jmedgenet-2011-100468</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterising chromosome rearrangements: recent technical advances in molecular cytogenetics.</h4>
<p class="authors"><span class="author">Le Scouarnec S</span> and <span class="author">Gribble SM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%73%6C%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#108;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genomic rearrangements can result in losses, amplifications, translocations and inversions of DNA fragments thereby modifying genome architecture, and potentially having clinical consequences. Many genomic disorders caused by structural variation have initially been uncovered by early cytogenetic methods. The last decade has seen significant progression in molecular cytogenetic techniques, allowing rapid and precise detection of structural rearrangements on a whole-genome scale. The high resolution attainable with these recently developed techniques has also uncovered the role of structural variants in normal genetic variation alongside single-nucleotide polymorphisms (SNPs). We describe how array-based comparative genomic hybridisation, SNP arrays, array painting and next-generation sequencing analytical methods (read depth, read pair and split read) allow the extensive characterisation of chromosome rearrangements in human genomes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">108</span>;<span class="number">1</span>;<span class="pagerange">75-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22086080" rel="external">22086080</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3238113" rel="external">3238113</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fhdy.2011.100" rel="external">10.1038/hdy.2011.100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genotype-based test in mapping cis-regulatory variants from allele-specific expression data.</h4>
<p class="authors"><span class="author">Lefebvre JF</span>, <span class="author">Vello E</span>, <span class="author">Ge B</span>, <span class="author">Montgomery SB</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Pastinen T</span> and <span class="author">Labuda D</span></p>
<p class="affiliation">Centre de Recherche du CHU Sainte-Justine, Université de Montréal, Montréal, Québec, Canada.</p>
<p class="abstract">Identifying and understanding the impact of gene regulatory variation is of considerable importance in evolutionary and medical genetics; such variants are thought to be responsible for human-specific adaptation [1] and to have an important role in genetic disease. Regulatory variation in cis is readily detected in individuals showing uneven expression of a transcript from its two allelic copies, an observation referred to as allelic imbalance (AI). Identifying individuals exhibiting AI allows mapping of regulatory DNA regions and the potential to identify the underlying causal genetic variant(s). However, existing mapping methods require knowledge of the haplotypes, which make them sensitive to phasing errors. In this study, we introduce a genotype-based mapping test that does not require haplotype-phase inference to locate regulatory regions. The test relies on partitioning genotypes of individuals exhibiting AI and those not expressing AI in a 2×3 contingency table. The performance of this test to detect linkage disequilibrium (LD) between a potential regulatory site and a SNP located in this region was examined by analyzing the simulated and the empirical AI datasets. In simulation experiments, the genotype-based test outperforms the haplotype-based tests with the increasing distance separating the regulatory region from its regulated transcript. The genotype-based test performed equally well with the experimental AI datasets, either from genome-wide cDNA hybridization arrays or from RNA sequencing. By avoiding the need of haplotype inference, the genotype-based test will suit AI analyses in population samples of unknown haplotype structure and will additionally facilitate the identification of cis-regulatory variants that are located far away from the regulated transcript.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e38667</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22685595" rel="external">22685595</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3369843" rel="external">3369843</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0038667" rel="external">10.1371/journal.pone.0038667</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transferrin and HFE genes interact in Alzheimer&#39;s disease risk: the Epistasis Project.</h4>
<p class="authors"><span class="author">Lehmann DJ</span>, <span class="author">Schuur M</span>, <span class="author">Warden DR</span>, <span class="author">Hammond N</span>, <span class="author">Belbin O</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">Lehmann MG</span>, <span class="author">Wilcock GK</span>, <span class="author">Brown K</span>, <span class="author">Kehoe PG</span>, <span class="author">Morris CM</span>, <span class="author">Barker R</span>, <span class="author">Coto E</span>, <span class="author">Alvarez V</span>, <span class="author">Deloukas P</span>, <span class="author">Mateo I</span>, <span class="author">Gwilliam R</span>, <span class="author">Combarros O</span>, <span class="author">Arias-V&aacute;squez A</span>, <span class="author">Aulchenko YS</span>, <span class="author">Ikram MA</span>, <span class="author">Breteler MM</span>, <span class="author">van Duijn CM</span>, <span class="author">Oulhaj A</span>, <span class="author">Heun R</span>, <span class="author">Cortina-Borja M</span>, <span class="author">Morgan K</span>, <span class="author">Robson K</span> and <span class="author">Smith AD</span></p>
<p class="affiliation">Oxford Project to Investigate Memory and Ageing, University Department of Physiology, Anatomy and Genetics, Oxford, UK. <a href="mailto:%64%6F%6E%61%6C%64.%6C%65%68%6D%61%6E%6E%40%70%68%61%72%6D.%6F%78.%61%63.%75%6B">&#100;&#111;&#110;&#97;&#108;&#100;&#46;&#108;&#101;&#104;&#109;&#97;&#110;&#110;&#64;&#112;&#104;&#97;&#114;&#109;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Iron overload may contribute to the risk of Alzheimer&#39;s disease (AD). In the Epistasis Project, with 1757 cases of AD and 6295 controls, we studied 4 variants in 2 genes of iron metabolism: hemochromatosis (HFE) C282Y and H63D, and transferrin (TF) C2 and -2G/A. We replicated the reported interaction between HFE 282Y and TF C2 in the risk of AD: synergy factor, 1.75 (95% confidence interval, 1.1-2.8, p = 0.02) in Northern Europeans. The synergy factor was 3.1 (1.4-6.9; 0.007) in subjects with the APOE&epsilon;4 allele. We found another interaction, between HFE 63HH and TF -2AA, markedly modified by age. Both interactions were found mainly or only in Northern Europeans. The interaction between HFE 282Y and TF C2 has now been replicated twice, in altogether 2313 cases of AD and 7065 controls, and has also been associated with increased iron load. We therefore suggest that iron overload may be a causative factor in the development of AD. Treatment for iron overload might thus be protective in some cases.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0400074</span>, <span class="grant">G0400546</span>, <span class="grant">G0502157</span>, <span class="grant">G0900652</span>, <span class="grant">G1100540</span></p>
<p class="publication"><span class="title">Neurobiology of aging</span>&nbsp;<span class="year">2012</span>;<span class="volume">33</span>;<span class="number">1</span>;<span class="pagerange">202.e1-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20817350">20817350</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neurobiolaging.2010.07.018">10.1016/j.neurobiolaging.2010.07.018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A cornucopia of candidates for deafness.</h4>
<p class="authors"><span class="author">Lewis MA</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="abstract">Many genes involved in deafness are yet to be discovered. Here, Senthilan et al. focus on the Drosophila Johnston's organ to uncover a wide variety of genes, including several unexpected candidates as well as those already known to underlie deafness in mice and humans.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">150</span>;<span class="number">5</span>;<span class="pagerange">879-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22939617" rel="external">22939617</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2012.08.007" rel="external">10.1016/j.cell.2012.08.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using mouse models to study function of transcriptional factors in T cell development.</h4>
<p class="authors"><span class="author">Li P</span>, <span class="author">Xiao Y</span>, <span class="author">Liu Z</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Key Laboratory of Regenerative Biology, Guangzchou Institutes of Biomedicine and Health, Chinese Academy of Sciences, Guangzhou, China ; Guangdong Provincial Key Laboratory of Stem Cell and Regenerative Medicine, Guangzhou, China.</p><p class="abstract">Laboratory mice have widely been used as tools for basic biological research and models for studying human diseases. With the advances of genetic engineering and conditional knockout (CKO) mice, we now understand hematopoiesis is a dynamic stepwise process starting from hematopoietic stem cells (HSCs) which are responsible for replenishing all blood cells. Transcriptional factors play important role in hematopoiesis. In this review we compile several studies on using genetic modified mice and humanized mice to study function of transcriptional factors in lymphopoiesis, including T lymphocyte and Natural killer (NK) cell development. Finally, we focused on the key transcriptional factor Bcl11b and its function in regulating T cell specification and commitment.</p>
<p class="publication"><span class="title">Cell regeneration (London, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/25408871">25408871</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4230505">4230505</a>; DOI: <a href="http://dx.doi.org/10.1186%2F2045-9769-1-8">10.1186/2045-9769-1-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Six novel susceptibility Loci for early-onset androgenetic alopecia and their unexpected association with common diseases.</h4>
<p class="authors"><span class="author">Li R</span>, <span class="author">Brockschmidt FF</span>, <span class="author">Kiefer AK</span>, <span class="author">Stefansson H</span>, <span class="author">Nyholt DR</span>, <span class="author">Song K</span>, <span class="author">Vermeulen SH</span>, <span class="author">Kanoni S</span>, <span class="author">Glass D</span>, <span class="author">Medland SE</span>, <span class="author">Dimitriou M</span>, <span class="author">Waterworth D</span>, <span class="author">Tung JY</span>, <span class="author">Geller F</span>, <span class="author">Heilmann S</span>, <span class="author">Hillmer AM</span>, <span class="author">Bataille V</span>, <span class="author">Eigelshoven S</span>, <span class="author">Hanneken S</span>, <span class="author">Moebus S</span>, <span class="author">Herold C</span>, <span class="author">den Heijer M</span>, <span class="author">Montgomery GW</span>, <span class="author">Deloukas P</span>, <span class="author">Eriksson N</span>, <span class="author">Heath AC</span>, <span class="author">Becker T</span>, <span class="author">Sulem P</span>, <span class="author">Mangino M</span>, <span class="author">Vollenweider P</span>, <span class="author">Spector TD</span>, <span class="author">Dedoussis G</span>, <span class="author">Martin NG</span>, <span class="author">Kiemeney LA</span>, <span class="author">Mooser V</span>, <span class="author">Stefansson K</span>, <span class="author">Hinds DA</span>, <span class="author">N&ouml;then MM</span> and <span class="author">Richards JB</span></p>
<p class="affiliation">Departments of Medicine, Human Genetics, Epidemiology, and Biostatistics, Lady Davis Institute, Jewish General Hospital, McGill University, Montreal, Quebec, Canada.</p>
<p class="abstract">Androgenetic alopecia (AGA) is a highly heritable condition and the most common form of hair loss in humans. Susceptibility loci have been described on the X chromosome and chromosome 20, but these loci explain a minority of its heritable variance. We conducted a large-scale meta-analysis of seven genome-wide association studies for early-onset AGA in 12,806 individuals of European ancestry. While replicating the two AGA loci on the X chromosome and chromosome 20, six novel susceptibility loci reached genome-wide significance (pâ€Š=â€Š2.62Ã—10(-9)-1.01Ã—10(-12)). Unexpectedly, we identified a risk allele at 17q21.31 that was recently associated with Parkinson's disease (PD) at a genome-wide significant level. We then tested the association between early-onset AGA and the risk of PD in a cross-sectional analysis of 568 PD cases and 7,664 controls. Early-onset AGA cases had significantly increased odds of subsequent PD (ORâ€Š=â€Š1.28, 95% confidence interval: 1.06-1.55, pâ€Š=â€Š8.9Ã—10(-3)). Further, the AGA susceptibility alleles at the 17q21.31 locus are on the H1 haplotype, which is under negative selection in Europeans and has been linked to decreased fertility. Combining the risk alleles of six novel and two established susceptibility loci, we created a genotype risk score and tested its association with AGA in an additional sample. Individuals in the highest risk quartile of a genotype score had an approximately six-fold increased risk of early-onset AGA [odds ratio (OR)â€Š=â€Š5.78, pâ€Š=â€Š1.4Ã—10(-88)]. Our results highlight unexpected associations between early-onset AGA, Parkinson's disease, and decreased fertility, providing important insights into the pathophysiology of these conditions.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e1002746</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22693459" rel="external">22693459</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3364959" rel="external">3364959</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002746" rel="external">10.1371/journal.pgen.1002746</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Design and implementation of ProteinWorldDB</h4>
<p class="authors"><span class="author">Lifschitz,S.</span>, <span class="author">Viana,C.J.M.</span>, <span class="author">Trist&atilde;o,C.</span>, <span class="author">Catanho,M.</span>, <span class="author">Degrave,W.M.</span>, <span class="author">De Miranda,A.B.</span>, <span class="author">Bezerra,M.</span> and <span class="author">OTTO,T.D.</span></p>
<p class="affiliation">Affiliation: PUC-Rio, Departamento de Informática, Rio de Janeiro, RJ, Brazil; Affiliation: Laboratório de Genômica Funcional e Bioinformática, Instituto Oswaldo Cruz, Rio de Janeiro, RJ, Brazil; Affiliation: Laboratório de Biologia Computacional e Sistemas, Instituto Oswaldo Cruz, Rio de Janeiro, RJ, Brazil; Affiliation: Wellcome Trust Sanger Institute, Hinxton, United Kingdom; Correspondence Address: Lifschitz, S.; PUC-Rio, Departamento de Informática, Rio de Janeiro, RJ, Brazil</p>
<p class="abstract">This work involves the comparison of protein information in a genomic scale. The main goal is to improve the quality and interpretation of biological data, besides our understanding of biological systems and their interactions. Stringent comparisons were obtained after the application of the Smith-Waterman algorithm in a pair wise manner to all predicted proteins encoded in both completely sequenced and unfinished genomes available in the public database RefSeq. Comparisons were run through a computational grid and the complete result reaches a volume of over 900 GB. Consequently, the database system design is a critical step in order to store and manage the information from comparisons' results. This paper describes database conceptual design issues for the creation of a database that represents a data set of protein sequence cross-comparisons. We show that our conceptual schema and its relational mapping enables users to extract relevant information, from simple to complex queries integrating distinct data sources</p>
<p class="publication"><span class="title">Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics) </span>&nbsp;<span class="year">2012</span>;<span class="volume">7409</span>;<span class="pagerange">144-55</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-31927-3_13" rel="external">10.1007/978-3-642-31927-3_13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The 2-methylcitrate cycle is implicated in the detoxification of propionate in Toxoplasma gondii.</h4>
<p class="authors"><span class="author">Limenitakis J</span>, <span class="author">Oppenheim RD</span>, <span class="author">Creek DJ</span>, <span class="author">Foth BJ</span>, <span class="author">Barrett MP</span> and <span class="author">Soldati-Favre D</span></p>
<p class="affiliation">Department of Microbiology and Molecular Medicine, Faculty of Medicine, University of Geneva, CMU 1 Rue Michel Servet, 1211, Geneva, Switzerland.</p>
<p class="abstract">Toxoplasma gondii belongs to the coccidian subgroup of the Apicomplexa phylum. The Coccidia are obligate intracellular pathogens that establish infection in their mammalian host via the enteric route. These parasites lack a mitochondrial pyruvate dehydrogenase complex but have preserved the degradation of branched-chain amino acids (BCAA) as a possible pathway to generate acetyl-CoA. Importantly, degradation of leucine, isoleucine and valine could lead to concomitant accumulation of propionyl-CoA, a toxic metabolite that inhibits cell growth. Like fungi and bacteria, the Coccidia possess the complete set of enzymes necessary to metabolize and detoxify propionate by oxidation to pyruvate via the 2-methylcitrate cycle (2-MCC). Phylogenetic analysis provides evidence that the 2-MCC was acquired via horizontal gene transfer. In T.â€‰gondii tachyzoites, this pathway is split between the cytosol and the mitochondrion. Although the rate-limiting enzyme 2-methylisocitrate lyase is dispensable for parasite survival, its substrates accumulate in parasites deficient in the enzyme and its absence confers increased sensitivity to propionic acid. BCAA is also dispensable in tachyzoites, leaving unresolved the source of mitochondrial acetyl-CoA.</p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23279335" rel="external">23279335</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fmmi.12139" rel="external">10.1111/mmi.12139</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Muscle diseases in the zebrafish.</h4>
<p class="authors"><span class="author">Lin YY</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1HH, United Kingdom. <a href="mailto:%59%59%4C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#89;&#89;&#76;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="abstract">Animal models in biomedical research are important for understanding the pathological mechanisms of human diseases at a molecular and cellular level. Several aspects of mammalian animals, however, may limit their use in modelling neuromuscular disorders. Many attributes of zebrafish (Danio rerio) are complementary to mammalian experimental systems, establishing the zebrafish as a powerful model organism in disease biology. This review focuses on a number of key studies using the zebrafish to model hereditary muscle diseases with additional emphasis on recent advances in zebrafish functional genomics and drug discovery. Increasing research in zebrafish disease models, combined with knowledge from mammalian models, will bring novel insights into the disease pathogenesis of neuromuscular disorders, as well as facilitate the development of effective therapeutic strategies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Neuromuscular disorders : NMD</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">8</span>;<span class="pagerange">673-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22647769" rel="external">22647769</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.nmd.2012.04.007" rel="external">10.1016/j.nmd.2012.04.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mosaic overgrowth with fibroadipose hyperplasia is caused by somatic activating mutations in PIK3CA.</h4>
<p class="authors"><span class="author">Lindhurst MJ</span>, <span class="author">Parker VE</span>, <span class="author">Payne F</span>, <span class="author">Sapp JC</span>, <span class="author">Rudge S</span>, <span class="author">Harris J</span>, <span class="author">Witkowski AM</span>, <span class="author">Zhang Q</span>, <span class="author">Groeneveld MP</span>, <span class="author">Scott CE</span>, <span class="author">Daly A</span>, <span class="author">Huson SM</span>, <span class="author">Tosi LL</span>, <span class="author">Cunningham ML</span>, <span class="author">Darling TN</span>, <span class="author">Geer J</span>, <span class="author">Gucev Z</span>, <span class="author">Sutton VR</span>, <span class="author">Tziotzios C</span>, <span class="author">Dixon AK</span>, <span class="author">Helliwell T</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Savage DB</span>, <span class="author">Wakelam MJ</span>, <span class="author">Barroso I</span>, <span class="author">Biesecker LG</span> and <span class="author">Semple RK</span></p>
<p class="affiliation">The National Human Genome Research Institute, US National Institutes of Health, Bethesda, Maryland, USA.</p><p class="abstract">The phosphatidylinositol 3-kinase (PI3K)-AKT signaling pathway is critical for cellular growth and metabolism. Correspondingly, loss of function of PTEN, a negative regulator of PI3K, or activating mutations in AKT1, AKT2 or AKT3 have been found in distinct disorders featuring overgrowth or hypoglycemia. We performed exome sequencing of DNA from unaffected and affected cells from an individual with an unclassified syndrome of congenital progressive segmental overgrowth of fibrous and adipose tissue and bone and identified the cancer-associated mutation encoding p.His1047Leu in PIK3CA, the gene that encodes the p110&alpha; catalytic subunit of PI3K, only in affected cells. Sequencing of PIK3CA in ten additional individuals with overlapping syndromes identified either the p.His1047Leu alteration or a second cancer-associated alteration, p.His1047Arg, in nine cases. Affected dermal fibroblasts showed enhanced basal and epidermal growth factor (EGF)-stimulated phosphatidylinositol 3,4,5-trisphosphate (PIP(3)) generation and concomitant activation of downstream signaling relative to their unaffected counterparts. Our findings characterize a distinct overgrowth syndrome, biochemically demonstrate activation of PI3K signaling and thereby identify a rational therapeutic target.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/E/B/0000C227</span>; Medical Research Council: <span class="grant">G0600717B</span>; NCATS NIH HHS: <span class="grant">UL1 TR000423</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">078986</span>, <span class="grant">078986/Z/06/Z</span>, <span class="grant">080952</span>, <span class="grant">091551</span>, <span class="grant">091551/Z/10/Z</span>, <span class="grant">095515</span>, <span class="grant">097721</span>, <span class="grant">097721/Z/11/Z</span>, <span class="grant">098051/Z/05/Z</span>, <span class="grant">80952/Z/06/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">8</span>;<span class="pagerange">928-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22729222">22729222</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3461408">3461408</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2332">10.1038/ng.2332</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metal binding in proteins: Machine learning complements X-ray absorption spectroscopy</h4>
<p class="authors"><span class="author">Lippi,M.</span>, <span class="author">Passerini,A.</span>, <span class="author">PUNTA,M.</span> and <span class="author">Frasconi,P.</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science </span>&nbsp;<span class="year">2012</span>;<span class="volume">7524</span>;<span class="pagerange">854-857</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-3-642-33486-3_63">10.1007/978-3-642-33486-3_63</a>; URL: <a title="http://link.springer.com/chapter/10.1007%2F978-3-642-33486-3_63" href="http://link.springer.com/chapter/10.1007%2F978-3-642-33486-3_63">http://link.springer.com/chapter<span class="print-hide">...</span><span class="print-show">/10.1007%2F978-3-642-33</span>486-3_63</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High prevalence of posterior polymorphous corneal dystrophy in the czech republic; linkage disequilibrium mapping and dating an ancestral mutation.</h4>
<p class="authors"><span class="author">Liskova P</span>, <span class="author">Gwilliam R</span>, <span class="author">Filipec M</span>, <span class="author">Jirsova K</span>, <span class="author">Reinstein Merjava S</span>, <span class="author">Deloukas P</span>, <span class="author">Webb TR</span>, <span class="author">Bhattacharya SS</span>, <span class="author">Ebenezer ND</span>, <span class="author">Morris AG</span> and <span class="author">Hardcastle AJ</span></p>
<p class="affiliation">Laboratory of the Biology and Pathology of the Eye, Institute of Inherited Metabolic Diseases, First Faculty of Medicine, Charles University in Prague and General University Hospital in Prague, Prague, Czech Republic ; Department of Ophthalmology, First Faculty of Medicine, Charles University in Prague and General University Hospital in Prague, Prague, Czech Republic ; UCL Institute of Ophthalmology, London, United Kingdom.</p>
<p class="abstract">Posterior polymorphous corneal dystrophy (PPCD) is a rare autosomal dominant genetically heterogeneous disorder. Nineteen Czech PPCD pedigrees with 113 affected family members were identified, and 17 of these kindreds were genotyped for markers on chromosome 20p12.1- 20q12. Comparison of haplotypes in 81 affected members, 20 unaffected first degree relatives and 13 spouses, as well as 55 unrelated controls, supported the hypothesis of a shared ancestor in 12 families originating from one geographic location. In 38 affected individuals from nine of these pedigrees, a common haplotype was observed between D20S48 and D20S107 spanning approximately 23 Mb, demonstrating segregation of disease with the PPCD1 locus. This haplotype was not detected in 110 ethnically matched control chromosomes. Within the common founder haplotype, a core mini-haplotype was detected for D20S605, D20S182 and M189K2 in all 67 affected members from families 1-12, however alleles representing the core mini-haplotype were also detected in population matched controls. The most likely location of the responsible gene within the disease interval, and estimated mutational age, were inferred by linkage disequilibrium mapping (DMLE+2.3). The appearance of a disease-causing mutation was dated between 64-133 generations. The inferred ancestral locus carrying a PPCD1 disease-causing variant within the disease interval spans 60 Kb on 20p11.23, which contains a single known protein coding gene, ZNF133. However, direct sequence analysis of coding and untranslated exons did not reveal a potential pathogenic mutation. Microdeletion or duplication was also excluded by comparative genomic hybridization using a dense chromosome 20 specific array. Geographical origin, haplotype and statistical analysis suggest that in 14 unrelated families an as yet undiscovered mutation on 20p11.23 was inherited from a common ancestor. Prevalence of PPCD in the Czech Republic appears to be the highest worldwide and our data suggests that at least one other novel locus for PPCD also exists.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e45495</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23049806" rel="external">23049806</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0045495" rel="external">10.1371/journal.pone.0045495</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dense fine-mapping study identifies new susceptibility loci for primary biliary cirrhosis.</h4>
<p class="authors"><span class="author">Liu JZ</span>, <span class="author">Almarri MA</span>, <span class="author">Gaffney DJ</span>, <span class="author">Mells GF</span>, <span class="author">Jostins L</span>, <span class="author">Cordell HJ</span>, <span class="author">Ducker SJ</span>, <span class="author">Day DB</span>, <span class="author">Heneghan MA</span>, <span class="author">Neuberger JM</span>, <span class="author">Donaldson PT</span>, <span class="author">Bathgate AJ</span>, <span class="author">Burroughs A</span>, <span class="author">Davies MH</span>, <span class="author">Jones DE</span>, <span class="author">Alexander GJ</span>, <span class="author">Barrett JC</span>, <span class="author">Sandford RN</span>, <span class="author">Anderson CA</span>, <span class="author">UK Primary Biliary Cirrhosis (PBC) Consortium</span> and <span class="author">Wellcome Trust Case Control Consortium 3</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We genotyped 2,861 cases of primary biliary cirrhosis (PBC) from the UK PBC Consortium and 8,514 UK population controls across 196,524 variants within 186 known autoimmune risk loci. We identified 3 loci newly associated with PBC (at P&lt;5&times;10(-8)), increasing the number of known susceptibility loci to 25. The most associated variant at 19p12 is a low-frequency nonsynonymous SNP in TYK2, further implicating JAK-STAT and cytokine signaling in disease pathogenesis. An additional five loci contained nonsynonymous variants in high linkage disequilibrium (LD; r2&gt;0.8) with the most associated variant at the locus. We found multiple independent common, low-frequency and rare variant association signals at five loci. Of the 26 independent non-human leukocyte antigen (HLA) signals tagged on the Immunochip, 15 have SNPs in B-lymphoblastoid open chromatin regions in high LD (r2&gt;0.8) with the most associated variant. This study shows how data from dense fine-mapping arrays coupled with functional genomic data can be used to identify candidate causal variants for functional follow-up.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0500020</span>, <span class="grant">G0800460</span>, <span class="grant">G0802068</span>; NIDDK NIH HHS: <span class="grant">U01-DK-062418</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">085925/Z/08/Z</span>, <span class="grant">090532</span>, <span class="grant">098051</span>, <span class="grant">WT090355/A/09/Z</span>, <span class="grant">WT090355/B/09/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">10</span>;<span class="pagerange">1137-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22961000">22961000</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3459817">3459817</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2395">10.1038/ng.2395</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression of chemosensory proteins in the tsetse fly Glossina morsitans morsitans is related to female host-seeking behaviour.</h4>
<p class="authors"><span class="author">Liu R</span>, <span class="author">He X</span>, <span class="author">Lehane S</span>, <span class="author">Lehane M</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Berriman M</span>, <span class="author">Field LM</span> and <span class="author">Zhou JJ</span></p>
<p class="affiliation">Department of Biological Chemistry, Rothamsted Research, Harpenden, UK.</p><p class="abstract">Chemosensory proteins (CSPs) are a class of soluble proteins present in high concentrations in the sensilla of insect antennae. It has been proposed that they play an important role in insect olfaction by mediating interactions between odorants and odorant receptors. Here we report, for the first time, the presence of five CSP genes in the tsetse fly Glossina morsitans morsitans, a major vector transmitting nagana in livestock. Real-time quantitative reverse transcription PCR showed that three of the CSPs are expressed in antennae. One of them, GmmCSP2, is transcribed at a very high level and could be involved in olfaction. We also determined expression in the antennae of both males and females at different life stages and with different blood feeding regimes. The transcription of GmmCSP2 was lower in male antennae than in females, with a sharp increase in 10-week-old flies, 48 h after a bloodmeal. Thus there is a clear relationship between CSP gene transcription and host searching behaviour. Genome annotation and phylogenetic analyses comparing G. morsitans morsitans CSPs with those of other Diptera showed rapid evolution after speciation of mosquitoes.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust: <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Insect molecular biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">1</span>;<span class="pagerange">41-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22074189">22074189</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3664020">3664020</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2583.2011.01114.x">10.1111/j.1365-2583.2011.01114.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Learned recognition of maternal signature odors mediates the first suckling episode in mice.</h4>
<p class="authors"><span class="author">Logan DW</span>, <span class="author">Brunet LJ</span>, <span class="author">Webb WR</span>, <span class="author">Cutforth T</span>, <span class="author">Ngai J</span> and <span class="author">Stowers L</span></p>
<p class="affiliation">Department of Cell Biology, The Scripps Research Institute, La Jolla, CA 92037, USA.</p><p class="abstract"><strong>Background</strong>: Soon after birth, all mammals must initiate milk suckling to survive. In rodents, this innate behavior is critically dependent on uncharacterized maternally derived chemosensory ligands. Recently, the first pheromone sufficient to initiate suckling was isolated from the rabbit. Identification of the olfactory cues that trigger first suckling in the mouse would provide the means to determine the neural mechanisms that generate innate behavior.</p><p class="abstract"><strong>Results</strong>: Here we use behavioral analysis, metabolomics, and calcium imaging of primary sensory neurons and find no evidence of ligands with intrinsic bioactivity, such as pheromones, acting to promote first suckling in the mouse. Instead, we find that the initiation of suckling is dependent on variable blends of maternal &quot;signature odors&quot; that are learned and recognized prior to first suckling.</p><p class="abstract"><strong>Conclusions</strong>: As observed with pheromone-mediated behavior, the response to signature odors releases innate behavior. However, this mechanism tolerates variability in both the signaling ligands and sensory neurons, which may maximize the probability that this first essential behavior is successfully initiated. These results suggest that mammalian species have evolved multiple strategies to ensure the onset of this critical behavior.</p>
<p class="grants">Funded by: NIDCD NIH HHS: <span class="grant">R01 DC006885</span>, <span class="grant">R01 DC009413</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">21</span>;<span class="pagerange">1998-2007</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23041191">23041191</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3494771">3494771</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2012.08.041">10.1016/j.cub.2012.08.041</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GeneDB--an annotation database for pathogens.</h4>
<p class="authors"><span class="author">Logan-Klumpler FJ</span>, <span class="author">De Silva N</span>, <span class="author">Boehme U</span>, <span class="author">Rogers MB</span>, <span class="author">Velarde G</span>, <span class="author">McQuillan JA</span>, <span class="author">Carver T</span>, <span class="author">Aslett M</span>, <span class="author">Olsen C</span>, <span class="author">Subramanian S</span>, <span class="author">Phan I</span>, <span class="author">Farris C</span>, <span class="author">Mitra S</span>, <span class="author">Ramasamy G</span>, <span class="author">Wang H</span>, <span class="author">Tivey A</span>, <span class="author">Jackson A</span>, <span class="author">Houston R</span>, <span class="author">Parkhill J</span>, <span class="author">Holden M</span>, <span class="author">Harb OS</span>, <span class="author">Brunk BP</span>, <span class="author">Myler PJ</span>, <span class="author">Roos D</span>, <span class="author">Carrington M</span>, <span class="author">Smith DF</span>, <span class="author">Hertz-Fowler C</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%66%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">GeneDB (http://www.genedb.org) is a genome database for prokaryotic and eukaryotic pathogens and closely related organisms. The resource provides a portal to genome sequence and annotation data, which is primarily generated by the Pathogen Genomics group at the Wellcome Trust Sanger Institute. It combines data from completed and ongoing genome projects with curated annotation, which is readily accessible from a web based resource. The development of the database in recent years has focused on providing database-driven annotation tools and pipelines, as well as catering for increasingly frequent assembly updates. The website has been significantly redesigned to take advantage of current web technologies, and improve usability. The current release stores 41 data sets, of which 17 are manually curated and maintained by biologists, who review and incorporate data from the scientific literature, as well as other sources. GeneDB is primarily a production and annotation database for the genomes of predominantly pathogenic organisms.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 043565</span>, <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D98-108</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22116062" rel="external">22116062</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245030" rel="external">3245030</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr1032" rel="external">10.1093/nar/gkr1032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A combined functional annotation score for non-synonymous variants.</h4>
<p class="authors"><span class="author">Lopes MC</span>, <span class="author">Joyce C</span>, <span class="author">Ritchie GR</span>, <span class="author">John SL</span>, <span class="author">Cunningham F</span>, <span class="author">Asimit J</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Hinxton, UK. <a href="mailto:%6D%6C%31%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#108;&#49;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Aims</strong>: Next-generation sequencing has opened the possibility of large-scale sequence-based disease association studies. A major challenge in interpreting whole-exome data is predicting which of the discovered variants are deleterious or neutral. To address this question in silico, we have developed a score called Combined Annotation scoRing toOL (CAROL), which combines information from 2 bioinformatics tools: PolyPhen-2 and SIFT, in order to improve the prediction of the effect of non-synonymous coding variants.</p><p class="abstract"><strong>Methods</strong>: We used a weighted Z method that combines the probabilistic scores of PolyPhen-2 and SIFT. We defined 2 dataset pairs to train and test CAROL using information from the dbSNP: &#39;HGMD-PUBLIC&#39; and 1000 Genomes Project databases. The training pair comprises a total of 980 positive control (disease-causing) and 4,845 negative control (non-disease-causing) variants. The test pair consists of 1,959 positive and 9,691 negative controls.</p><p class="abstract"><strong>Results</strong>: CAROL has higher predictive power and accuracy for the effect of non-synonymous variants than each individual annotation tool (PolyPhen-2 and SIFT) and benefits from higher coverage.</p><p class="abstract"><strong>Conclusion</strong>: The combination of annotation tools can help improve automated prediction of whole-genome/exome non-synonymous variant functional consequences.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">095908</span>, <span class="grant">098051</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Human heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">73</span>;<span class="number">1</span>;<span class="pagerange">47-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22261837">22261837</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3390741">3390741</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000334984">10.1159/000334984</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Do genome-wide association scans have potential for translation?</h4>
<p class="authors"><span class="author">Lopes MC</span>, <span class="author">Zeggini E</span> and <span class="author">Panoutsopoulou K</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p>
<p class="abstract">The success of genome-wide association studies (GWAS) in identifying replicating associations has greatly contributed to understanding of the genetic aetiology of complex diseases. This review discusses and provides examples of the potential of GWAS findings to be translated into clinical practice, i.e., diagnosis, prediction, prognosis, novel treatments and response to treatment of common diseases. The biological insights afforded by newly-identified robust associations represent the largest, albeit indirect, translational contribution of GWAS.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">18030</span>; Wellcome Trust: <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Clinical chemistry and laboratory medicine : CCLM / FESCC</span>&nbsp;<span class="year">2012</span>;<span class="volume">50</span>;<span class="number">2</span>;<span class="pagerange">255-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22022988" rel="external">22022988</a>; DOI: <a href="http://dx.doi.org/10.1515%2FCCLM.2011.748" rel="external">10.1515/CCLM.2011.748</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mice deficient in the H+-ATPase a4 subunit have severe hearing impairment associated with enlarged endolymphatic compartments within the inner ear.</h4>
<p class="authors"><span class="author">Lorente-C&aacute;novas B</span>, <span class="author">Ingham N</span>, <span class="author">Norgett EE</span>, <span class="author">Golder ZJ</span>, <span class="author">Karet Frankl FE</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK;</p>
<p class="abstract">Mutations in the ATP6V0A4 gene lead to autosomal recessive distal renal tubular acidosis in patients who often show sensorineural hearing impairment. A first Atp6v0a4 knockout mouse model that recapitulates the loss of H(+)-ATPase function seen in humans has been generated and recently published (Norgett et al., 2012). Here we present the first detailed analysis of the structure and function of the auditory system in Atp6v0a4(-/-) knockout mice. Measurements of the auditory brainstem response (ABR) showed significantly elevated thresholds in homozygous mutant mice, which indicate severe hearing impairment. Heterozygote thresholds were normal. Analysis of paint-filled inner ears and sections from E16.5 embryos revealed a marked expansion of cochlear and endolymphatic ducts in Atp6v0a4(-/-) mice. A regulatory link between Atp6v0a4, Foxi1 and Pds has been reported and we found that the endolymphatic sac of Atp6v0a4(-/-) mice expresses both Foxi1 and Pds, which suggests a downstream position of Atp6v0a4. These mutants also showed a lack of endocochlear potential, suggesting a functional defect of the stria vascularis on the lateral wall of the cochlear duct. However, the main K(+) channels involved in the generation of endocochlear potential, Kcnj10 and Kcnq1, are strongly expressed in Atp6v0a4(-/-) mice. Our results lead to a better understanding of the role of this proton pump in hearing function.</p>
<p class="publication"><span class="title">Disease models &amp;amp; mechanisms</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23065636" rel="external">23065636</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdmm.010645" rel="external">10.1242/dmm.010645</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Community gene annotation in practice.</h4>
<p class="authors"><span class="author">Loveland JE</span>, <span class="author">Gilbert JG</span>, <span class="author">Griffiths E</span> and <span class="author">Harrow JL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%6A%65%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#101;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Manual annotation of genomic data is extremely valuable to produce an accurate reference gene set but is expensive compared with automatic methods and so has been limited to model organisms. Annotation tools that have been developed at the Wellcome Trust Sanger Institute (WTSI, http://www.sanger.ac.uk/.) are being used to fill that gap, as they can be used remotely and so open up viable community annotation collaborations. We introduce the &#39;Blessed&#39; annotator and &#39;Gatekeeper&#39; approach to Community Annotation using the Otterlace/ZMap genome annotation tool. We also describe the strategies adopted for annotation consistency, quality control and viewing of the annotation. DATABASE URL: http://vega.sanger.ac.uk/index.html.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F02195X/1</span>; Wellcome Trust: <span class="grant">WT077198</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2012</span>;<span class="volume">2012</span>;<span class="pagerange">bas009</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22434843">22434843</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308165">3308165</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbas009">10.1093/database/bas009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MiR-25 Regulates Wwp2 and Fbxw7 and Promotes Reprogramming of Mouse Fibroblast Cells to iPSCs.</h4>
<p class="authors"><span class="author">Lu D</span>, <span class="author">Davis MP</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">Wang W</span>, <span class="author">Campos LS</span>, <span class="author">Siede J</span>, <span class="author">Vigorito E</span>, <span class="author">Skarnes WC</span>, <span class="author">Dunham I</span>, <span class="author">Enright AJ</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">Background: miRNAs are a class of small non-coding RNAs that regulate gene expression and have critical functions in various biological processes. Hundreds of miRNAs have been identified in mammalian genomes but only a small number of them have been functionally characterized. Recent studies also demonstrate that some miRNAs have important roles in reprogramming somatic cells to induced pluripotent stem cells (iPSCs).

Methods: We screened 52 miRNAs cloned in a piggybac (PB) vector for their roles in reprogramming of mouse embryonic fibroblast cells to iPSCs. To identify targets of miRNAs, we made Dgcr8-deficient embryonic stem (ES) cells and introduced miRNA mimics to these cells, which lack miRNA biogenesis. The direct target genes of miRNA were identified through global gene expression analysis and target validation.

We found that over-expressing miR-25 or introducing miR-25 mimics enhanced production of iPSCs. We identified a number of miR-25 candidate gene targets. Of particular interest were two ubiquitin ligases, Wwp2 and Fbxw7, which have been proposed to regulate Oct4, c-Myc and Klf5, respectively. Our findings thus highlight the complex interplay between miRNAs and transcription factors involved in reprogramming, stem cell self-renewal and maintenance of pluripotency.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">8</span>;<span class="pagerange">e40938</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22912667" rel="external">22912667</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0040938" rel="external">10.1371/journal.pone.0040938</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide Transcriptome Profiling Reveals the Functional Impact of Rare De Novo and Recurrent CNVs in Autism Spectrum Disorders.</h4>
<p class="authors"><span class="author">Luo R</span>, <span class="author">Sanders SJ</span>, <span class="author">Tian Y</span>, <span class="author">Voineagu I</span>, <span class="author">Huang N</span>, <span class="author">Chu SH</span>, <span class="author">Klei L</span>, <span class="author">Cai C</span>, <span class="author">Ou J</span>, <span class="author">Lowe JK</span>, <span class="author">Hurles ME</span>, <span class="author">Devlin B</span>, <span class="author">State MW</span> and <span class="author">Geschwind DH</span></p>
<p class="affiliation">Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA; Center for Autism Research and Treatment, Semel Institute, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA.</p>
<p class="abstract">Copy-number variants (CNVs) are a major contributor to the pathophysiology of autism spectrum disorders (ASDs), but the functional impact of CNVs remains largely unexplored. Because brain tissue is not available from most samples, we interrogated gene expression in lymphoblasts from 244 families with discordant siblings in the Simons Simplex Collection in order to identify potentially pathogenic variation. Our results reveal that the overall frequency of significantly misexpressed genes (which we refer to here as outliers) identified in probands and unaffected siblings does not differ. However, in probands, but not their unaffected siblings, the group of outlier genes is significantly enriched in neural-related pathways, including neuropeptide signaling, synaptogenesis, and cell adhesion. We demonstrate that outlier genes cluster within the most pathogenic CNVs (rare de novo CNVs) and can be used for the prioritization of rare CNVs of potentially unknown significance. Several nonrecurrent CNVs with significant gene-expression alterations are identified (these include deletions in chromosomal regions 3q27, 3p13, and 3p26 and duplications at 2p15), suggesting that these are potential candidate ASD loci. In addition, we identify distinct expression changes in 16p11.2 microdeletions, 16p11.2 microduplications, and 7q11.23 duplications, and we show that specific genes within the 16p CNV interval correlate with differences in head circumference, an ASD-relevant phenotype. This study provides evidence that pathogenic structural variants have a functional impact via transcriptome alterations in ASDs at a genome-wide level and demonstrates the utility of integrating gene expression with mutation data for the prioritization of genes disrupted by potentially pathogenic mutations.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22726847" rel="external">22726847</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.05.011" rel="external">10.1016/j.ajhg.2012.05.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A balanced translocation truncates Neurotrimin in a family with intracranial and thoracic aortic aneurysm.</h4>
<p class="authors"><span class="author">Luukkonen TM</span>, <span class="author">P&ouml;yh&ouml;nen M</span>, <span class="author">Palotie A</span>, <span class="author">Ellonen P</span>, <span class="author">Lagstr&ouml;m S</span>, <span class="author">Lee JH</span>, <span class="author">Terwilliger JD</span>, <span class="author">Salonen R</span> and <span class="author">Varilo T</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland FIMM, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: Balanced chromosomal rearrangements occasionally have strong phenotypic effects, which may be useful in understanding pathobiology. However, conventional strategies for characterising breakpoints are laborious and inaccurate. We present here a proband with a thoracic aortic aneurysm (TAA) and a balanced translocation t(10;11) (q23.2;q24.2). Our purpose was to sequence the chromosomal breaks in this family to reveal a novel candidate gene for aneurysm.</p><p class="abstract">Intracranial aneurysm (IA) and TAAs appear to run in the family in an autosomal dominant manner: After exploring the family history, we observed that the proband&#39;s two siblings both died from cerebral haemorrhage, and the proband&#39;s parent and parent&#39;s sibling died from aortic rupture. After application of a genome-wide paired-end DNA sequencing method for breakpoint mapping, we demonstrate that this translocation breaks intron 1 of a splicing isoform of Neurotrimin at 11q25 in a previously implicated candidate region for IAs and AAs (OMIM 612161).</p><p class="abstract"><strong>Conclusions</strong>: Our results demonstrate the feasibility of genome-wide paired-end sequencing for the characterisation of balanced rearrangements and identification of candidate genes in patients with potentially disease-associated chromosome rearrangements. The family samples were gathered as a part of our recently launched National Registry of Reciprocal Balanced Translocations and Inversions in Finland (n=2575), and we believe that such a registry will be a powerful resource for the localisation of chromosomal aberrations, which can bring insight into the aetiology of related phenotypes.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">MH084995</span>, <span class="grant">R01 MH084995</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">49</span>;<span class="number">10</span>;<span class="pagerange">621-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23054244">23054244</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4039200">4039200</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmedgenet-2012-100977">10.1136/jmedgenet-2012-100977</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Estimating reassortment rates in co-circulating Eurasian swine influenza viruses.</h4>
<p class="authors"><span class="author">Lycett SJ</span>, <span class="author">Baillie G</span>, <span class="author">Coulter E</span>, <span class="author">Bhatt S</span>, <span class="author">Kellam P</span>, <span class="author">McCauley JW</span>, <span class="author">Wood JL</span>, <span class="author">Brown IH</span>, <span class="author">Pybus OG</span>, <span class="author">Leigh Brown AJ</span> and <span class="author">Combating Swine Influenza Initiative-COSI Consortium</span></p>
<p class="affiliation">Institute of Evolutionary Biology, University of Edinburgh, Kings Buildings, West Mains Road, Edinburgh EH9 3JT, UK. <a href="mailto:%73%61%6D%61%6E%74%68%61.%6C%79%63%65%74%74%40%65%64.%61%63.%75%6B">&#115;&#97;&#109;&#97;&#110;&#116;&#104;&#97;&#46;&#108;&#121;&#99;&#101;&#116;&#116;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Swine have often been considered as a mixing vessel for different influenza strains. In order to assess their role in more detail, we undertook a retrospective sequencing study to detect and characterize the reassortants present in European swine and to estimate the rate of reassortment between H1N1, H1N2 and H3N2 subtypes with Eurasian (avian-like) internal protein-coding segments. We analysed 69 newly obtained whole genome sequences of subtypes H1N1-H3N2 from swine influenza viruses sampled between 1982 and 2008, using Illumina and 454 platforms. Analyses of these genomes, together with previously published genomes, revealed a large monophyletic clade of Eurasian swine-lineage polymerase segments containing H1N1, H1N2 and H3N2 subtypes. We subsequently examined reassortments between the haemagglutinin and neuraminidase segments and estimated the reassortment rates between lineages using a recently developed evolutionary analysis method. High rates of reassortment between H1N2 and H1N1 Eurasian swine lineages were detected in European strains, with an average of one reassortment every 2-3 years. This rapid reassortment results from co-circulating lineages in swine, and in consequence we should expect further reassortments between currently circulating swine strains and the recent swine-origin H1N1v pandemic strain.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/H014306/1</span>; Medical Research Council: <span class="grant">MC_G0902096</span>, <span class="grant">MC_U117512723</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of general virology</span>&nbsp;<span class="year">2012</span>;<span class="volume">93</span>;<span class="number">Pt 11</span>;<span class="pagerange">2326-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22971819">22971819</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3542128">3542128</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fvir.0.044503-0">10.1099/vir.0.044503-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetically distinct subsets within ANCA-associated vasculitis.</h4>
<p class="authors"><span class="author">Lyons PA</span>, <span class="author">Rayner TF</span>, <span class="author">Trivedi S</span>, <span class="author">Holle JU</span>, <span class="author">Watts RA</span>, <span class="author">Jayne DR</span>, <span class="author">Baslund B</span>, <span class="author">Brenchley P</span>, <span class="author">Bruchfeld A</span>, <span class="author">Chaudhry AN</span>, <span class="author">Cohen Tervaert JW</span>, <span class="author">Deloukas P</span>, <span class="author">Feighery C</span>, <span class="author">Gross WL</span>, <span class="author">Guillevin L</span>, <span class="author">Gunnarsson I</span>, <span class="author">Harper L</span>, <span class="author">Hru&scaron;kov&aacute; Z</span>, <span class="author">Little MA</span>, <span class="author">Martorana D</span>, <span class="author">Neumann T</span>, <span class="author">Ohlsson S</span>, <span class="author">Padmanabhan S</span>, <span class="author">Pusey CD</span>, <span class="author">Salama AD</span>, <span class="author">Sanders JS</span>, <span class="author">Savage CO</span>, <span class="author">Segelmark M</span>, <span class="author">Stegeman CA</span>, <span class="author">Tesa&#x159; V</span>, <span class="author">Vaglio A</span>, <span class="author">Wieczorek S</span>, <span class="author">Wilde B</span>, <span class="author">Zwerina J</span>, <span class="author">Rees AJ</span>, <span class="author">Clayton DG</span> and <span class="author">Smith KG</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, and Department of Medicine, University of Cambridge School of Clinical Medicine, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom. <a href="mailto:%6B%67%63%73%32%40%63%61%6D.%61%63.%75%6B">&#107;&#103;&#99;&#115;&#50;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Antineutrophil cytoplasmic antibody (ANCA)-associated vasculitis is a severe condition encompassing two major syndromes: granulomatosis with polyangiitis (formerly known as Wegener&#39;s granulomatosis) and microscopic polyangiitis. Its cause is unknown, and there is debate about whether it is a single disease entity and what role ANCA plays in its pathogenesis. We investigated its genetic basis.</p><p class="abstract"><strong>Methods</strong>: A genomewide association study was performed in a discovery cohort of 1233 U.K. patients with ANCA-associated vasculitis and 5884 controls and was replicated in 1454 Northern European case patients and 1666 controls. Quality control, population stratification, and statistical analyses were performed according to standard criteria.</p><p class="abstract"><strong>Results</strong>: We found both major-histocompatibility-complex (MHC) and non-MHC associations with ANCA-associated vasculitis and also that granulomatosis with polyangiitis and microscopic polyangiitis were genetically distinct. The strongest genetic associations were with the antigenic specificity of ANCA, not with the clinical syndrome. Anti-proteinase 3 ANCA was associated with HLA-DP and the genes encoding &alpha;(1)-antitrypsin (SERPINA1) and proteinase 3 (PRTN3) (P=6.2&times;10(-89), P=5.6&times;10(-12,) and P=2.6&times;10(-7), respectively). Anti-myeloperoxidase ANCA was associated with HLA-DQ (P=2.1&times;10(-8)).</p><p class="abstract"><strong>Conclusions</strong>: This study confirms that the pathogenesis of ANCA-associated vasculitis has a genetic component, shows genetic distinctions between granulomatosis with polyangiitis and microscopic polyangiitis that are associated with ANCA specificity, and suggests that the response against the autoantigen proteinase 3 is a central pathogenic feature of proteinase 3 ANCA-associated vasculitis. These data provide preliminary support for the concept that proteinase 3 ANCA-associated vasculitis and myeloperoxidase ANCA-associated vasculitis are distinct autoimmune syndromes. (Funded by the British Heart Foundation and others.).</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">SP/09/001/27117</span>; Medical Research Council; Wellcome Trust: <span class="grant">083650/Z/07/Z</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2012</span>;<span class="volume">367</span>;<span class="number">3</span>;<span class="pagerange">214-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22808956">22808956</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa1108735">10.1056/NEJMoa1108735</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis of imputed rare variants: application to seven common complex diseases.</h4>
<p class="authors"><span class="author">M&auml;gi R</span>, <span class="author">Asimit JL</span>, <span class="author">Day-Williams AG</span>, <span class="author">Zeggini E</span> and <span class="author">Morris AP</span></p>
<p class="affiliation">Estonian Genome Centre, University of Tartu, Tartu, Estonia.</p><p class="abstract">Genome-wide association studies have been successful in identifying loci contributing effects to a range of complex human traits. The majority of reproducible associations within these loci are with common variants, each of modest effect, which together explain only a small proportion of heritability. It has been suggested that much of the unexplained genetic component of complex traits can thus be attributed to rare variation. However, genome-wide association study genotyping chips have been designed primarily to capture common variation, and thus are underpowered to detect the effects of rare variants. Nevertheless, we demonstrate here, by simulation, that imputation from an existing scaffold of genome-wide genotype data up to high-density reference panels has the potential to identify rare variant associations with complex traits, without the need for costly re-sequencing experiments. By application of this approach to genome-wide association studies of seven common complex diseases, imputed up to publicly available reference panels, we identify genome-wide significant evidence of rare variant association in PRDM10 with coronary artery disease and multiple genes in the major histocompatibility complex (MHC) with type 1 diabetes. The results of our analyses highlight that genome-wide association studies have the potential to offer an exciting opportunity for gene discovery through association with rare variants, conceivably leading to substantial advancements in our understanding of the genetic architecture underlying complex human traits.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">090532</span>, <span class="grant">098017</span>, <span class="grant">WT076113</span>, <span class="grant">WT081682</span>, <span class="grant">WT090532</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">36</span>;<span class="number">8</span>;<span class="pagerange">785-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22951892">22951892</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3569874">3569874</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.21675">10.1002/gepi.21675</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A systematic survey of loss-of-function variants in human protein-coding genes.</h4>
<p class="authors"><span class="author">MacArthur DG</span>, <span class="author">Balasubramanian S</span>, <span class="author">Frankish A</span>, <span class="author">Huang N</span>, <span class="author">Morris J</span>, <span class="author">Walter K</span>, <span class="author">Jostins L</span>, <span class="author">Habegger L</span>, <span class="author">Pickrell JK</span>, <span class="author">Montgomery SB</span>, <span class="author">Albers CA</span>, <span class="author">Zhang ZD</span>, <span class="author">Conrad DF</span>, <span class="author">Lunter G</span>, <span class="author">Zheng H</span>, <span class="author">Ayub Q</span>, <span class="author">DePristo MA</span>, <span class="author">Banks E</span>, <span class="author">Hu M</span>, <span class="author">Handsaker RE</span>, <span class="author">Rosenfeld JA</span>, <span class="author">Fromer M</span>, <span class="author">Jin M</span>, <span class="author">Mu XJ</span>, <span class="author">Khurana E</span>, <span class="author">Ye K</span>, <span class="author">Kay M</span>, <span class="author">Saunders GI</span>, <span class="author">Suner MM</span>, <span class="author">Hunt T</span>, <span class="author">Barnes IH</span>, <span class="author">Amid C</span>, <span class="author">Carvalho-Silva DR</span>, <span class="author">Bignell AH</span>, <span class="author">Snow C</span>, <span class="author">Yngvadottir B</span>, <span class="author">Bumpstead S</span>, <span class="author">Cooper DN</span>, <span class="author">Xue Y</span>, <span class="author">Romero IG</span>, <span class="author">1000 Genomes Project Consortium</span>, <span class="author">Wang J</span>, <span class="author">Li Y</span>, <span class="author">Gibbs RA</span>, <span class="author">McCarroll SA</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Pritchard JK</span>, <span class="author">Barrett JC</span>, <span class="author">Harrow J</span>, <span class="author">Hurles ME</span>, <span class="author">Gerstein MB</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%6D%61%63%61%72%74%68%75%72%40%61%74%67%75.%6D%67%68.%68%61%72%76%61%72%64.%65%64%75">&#109;&#97;&#99;&#97;&#114;&#116;&#104;&#117;&#114;&#64;&#97;&#116;&#103;&#117;&#46;&#109;&#103;&#104;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Genome-sequencing studies indicate that all humans carry many genetic variants predicted to cause loss of function (LoF) of protein-coding genes, suggesting unexpected redundancy in the human genome. Here we apply stringent filters to 2951 putative LoF variants obtained from 185 human genomes to determine their true prevalence and properties. We estimate that human genomes typically contain ~100 genuine LoF variants with ~20 genes completely inactivated. We identify rare and likely deleterious LoF alleles, including 26 known and 21 predicted severe disease-causing variants, as well as common LoF variants in nonessential genes. We describe functional and evolutionary differences between LoF-tolerant and recessive disease genes and a method for using these differences to prioritize candidate genes found in clinical sequencing studies.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; NHGRI NIH HHS: <span class="grant">U54 HG003273</span>; Wellcome Trust: <span class="grant">085532</span>, <span class="grant">090532</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2012</span>;<span class="volume">335</span>;<span class="number">6070</span>;<span class="pagerange">823-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22344438">22344438</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3299548">3299548</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1215040">10.1126/science.1215040</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A New Isolation with Migration Model along Complete Genomes Infers Very Different Divergence Processes among Closely Related Great Ape Species.</h4>
<p class="authors"><span class="author">Mailund T</span>, <span class="author">Halager AE</span>, <span class="author">Westergaard M</span>, <span class="author">Dutheil JY</span>, <span class="author">Munch K</span>, <span class="author">Andersen LN</span>, <span class="author">Lunter G</span>, <span class="author">Pr&uuml;fer K</span>, <span class="author">Scally A</span>, <span class="author">Hobolth A</span> and <span class="author">Schierup MH</span></p>
<p class="affiliation">Bioinformatics Research Center, Aarhus University, Aarhus, Denmark.</p>
<p class="abstract">We present a hidden Markov model (HMM) for inferring gradual isolation between two populations during speciation, modelled as a time interval with restricted gene flow. The HMM describes the history of adjacent nucleotides in two genomic sequences, such that the nucleotides can be separated by recombination, can migrate between populations, or can coalesce at variable time points, all dependent on the parameters of the model, which are the effective population sizes, splitting times, recombination rate, and migration rate. We show by extensive simulations that the HMM can accurately infer all parameters except the recombination rate, which is biased downwards. Inference is robust to variation in the mutation rate and the recombination rate over the sequence and also robust to unknown phase of genomes unless they are very closely related. We provide a test for whether divergence is gradual or instantaneous, and we apply the model to three key divergence processes in great apes: (a) the bonobo and common chimpanzee, (b) the eastern and western gorilla, and (c) the Sumatran and Bornean orang-utan. We find that the bonobo and chimpanzee appear to have undergone a clear split, whereas the divergence processes of the gorilla and orang-utan species occurred over several hundred thousands years with gene flow stopping quite recently. We also apply the model to the Homo/Pan speciation event and find that the most likely scenario involves an extended period of gene flow during speciation.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">e1003125</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23284294" rel="external">23284294</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3527290" rel="external">3527290</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003125" rel="external">10.1371/journal.pgen.1003125</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Accessing data from the International Mouse Phenotyping Consortium: state of the art and future plans.</h4>
<p class="authors"><span class="author">Mallon AM</span>, <span class="author">Iyer V</span>, <span class="author">Melvin D</span>, <span class="author">Morgan H</span>, <span class="author">Parkinson H</span>, <span class="author">Brown SD</span>, <span class="author">Flicek P</span> and <span class="author">Skarnes WC</span></p>
<p class="affiliation">Mammalian Genetics Unit, Medical Research Council Harwell, Harwell, Oxfordshire OX11 0RD, UK. <a href="mailto:%61.%6D%61%6C%6C%6F%6E%40%68%61%72.%6D%72%63.%61%63.%75%6B">&#97;&#46;&#109;&#97;&#108;&#108;&#111;&#110;&#64;&#104;&#97;&#114;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The International Mouse Phenotyping Consortium (IMPC) (http://www.mousephenotype.org) will reveal the pleiotropic functions of every gene in the mouse genome and uncover the wider role of genetic loci within diverse biological systems. Comprehensive informatics solutions are vital to ensuring that this vast array of data is captured in a standardised manner and made accessible to the scientific community for interrogation and analysis. Here we review the existing EuroPhenome and WTSI phenotype informatics systems and the IKMC portal, and present plans for extending these systems and lessons learned to the development of a robust IMPC informatics infrastructure.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U142684172</span>, <span class="grant">MC_U142684175</span>; NHGRI NIH HHS: <span class="grant">U54 HG006370</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span>;<span class="volume">23</span>;<span class="number">9-10</span>;<span class="pagerange">641-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22991088">22991088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4106044">4106044</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9428-9">10.1007/s00335-012-9428-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Neuronal Transcriptome Response Involving Stress Pathways is Buffered by Neuronal microRNAs.</h4>
<p class="authors"><span class="author">Manakov SA</span>, <span class="author">Morton A</span>, <span class="author">Enright AJ</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute Cambridge, UK ; RNA Genomics Lab, European Molecular Biology Laboratory-European Bioinformatics Institute Cambridge, UK.</p>
<p class="abstract">A single microRNA (miRNA) can inhibit a large number of mRNA transcripts. This widespread regulatory function has been experimentally demonstrated for a number of miRNAs. However, even when a multitude of targets is confirmed, function of a miRNA is frequently interpreted through a prism of a handful arbitrarily selected "interesting" targets. In this work we first show that hundreds of transcripts with target sites for two miRNAs expressed endogenously in neurons, miR-124 and miR-434-3p, are coordinately upregulated in a variety of neuronal stresses. This creates a landscape where these two miRNAs can exert their widespread inhibitory potential on stress-induced transcripts. Next, we experimentally demonstrate that overexpression of these two miRNAs indeed significantly inhibits expression of hundreds of stress-induced transcripts, thus confirming that these transcripts are enriched in true targets of examined miRNAs. A number of miRNAs were previously shown to have important roles in the regulation of stress responses, and our results suggest that these roles should be understood in light of a wide spread activation of miRNA targets during stresses. Importantly, a popular cationic lipid transfection reagent triggers such induction of miRNA targets. Therefore, when a transfection paradigm is employed to study miRNA function, the results of such studies should be interpreted with consideration for the inadvertent induction of miRNA targets.</p>
<p class="publication"><span class="title">Frontiers in neuroscience</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="pagerange">156</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23115547" rel="external">23115547</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3483788" rel="external">3483788</a>; DOI: <a href="http://dx.doi.org/10.3389%2Ffnins.2012.00156" rel="external">10.3389/fnins.2012.00156</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate human genetic polymorphisms and severe malaria in a Tanzanian population.</h4>
<p class="authors"><span class="author">Manjurano A</span>, <span class="author">Clark TG</span>, <span class="author">Nadjm B</span>, <span class="author">Mtove G</span>, <span class="author">Wangai H</span>, <span class="author">Sepulveda N</span>, <span class="author">Campino SG</span>, <span class="author">Maxwell C</span>, <span class="author">Olomi R</span>, <span class="author">Rockett KR</span>, <span class="author">Jeffreys A</span>, <span class="author">MalariaGen Consortium</span>, <span class="author">Riley EM</span>, <span class="author">Reyburn H</span> and <span class="author">Drakeley C</span></p>
<p class="affiliation">Joint Malaria Programme, Kilimanjaro Christian Medical Centre, Moshi, Tanzania.</p>
<p class="abstract">Human genetic background strongly influences susceptibility to malaria infection and progression to severe disease and death. Classical genetic studies identified haemoglobinopathies and erythrocyte-associated polymorphisms, as protective against severe disease. High throughput genotyping by mass spectrometry allows multiple single nucleotide polymorphisms (SNPs) to be examined simultaneously. We compared the prevalence of 65 human SNP's, previously associated with altered risk of malaria, between Tanzanian children with and without severe malaria. Five hundred children, aged 1-10 years, with severe malaria were recruited from those admitted to hospital in Muheza, Tanzania and compared with matched controls. Genotyping was performed by Sequenom MassArray, and conventional PCR was used to detect deletions in the alpha-thalassaemia gene. SNPs in two X-linked genes were associated with altered risk of severe malaria in females but not in males: heterozygosity for one or other of two SNPs in the G6PD gene was associated with protection from all forms of severe disease whilst two SNPs in the gene encoding CD40L were associated with respiratory distress. A SNP in the adenyl cyclase 9 (ADCY9) gene was associated with protection from acidosis whilst a polymorphism in the IL-1Î± gene (IL1A) was associated with an increased risk of acidosis. SNPs in the genes encoding IL-13 and reticulon-3 (RTN3) were associated with increased risk of cerebral malaria. This study confirms previously known genetic associations with protection from severe malaria (HbS, G6PD). It identifies two X-linked genes associated with altered risk of severe malaria in females, identifies mutations in ADCY9, IL1A and CD40L as being associated with altered risk of severe respiratory distress and acidosis, both of which are characterised by high serum lactate levels, and also identifies novel genetic associations with severe malaria (TRIM5) and cerebral malaria(IL-13 and RTN3). Further studies are required to test the generality of these associations and to understand their functional consequences.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">087285</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">090770/Z/09/Z</span>, <span class="grant">096527</span>, <span class="grant">77383/Z/05/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">e47463</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23144702">23144702</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3483265">3483265</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0047463">10.1371/journal.pone.0047463</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sleeping Beauty mutagenesis reveals cooperating mutations and pathways in pancreatic adenocarcinoma.</h4>
<p class="authors"><span class="author">Mann KM</span>, <span class="author">Ward JM</span>, <span class="author">Yew CC</span>, <span class="author">Kovochich A</span>, <span class="author">Dawson DW</span>, <span class="author">Black MA</span>, <span class="author">Brett BT</span>, <span class="author">Sheetz TE</span>, <span class="author">Dupuy AJ</span>, <span class="author">Australian Pancreatic Cancer Genome Initiative</span>, <span class="author">Chang DK</span>, <span class="author">Biankin AV</span>, <span class="author">Waddell N</span>, <span class="author">Kassahn KS</span>, <span class="author">Grimmond SM</span>, <span class="author">Rust AG</span>, <span class="author">Adams DJ</span>, <span class="author">Jenkins NA</span> and <span class="author">Copeland NG</span></p>
<p class="affiliation">Division of Genetics and Genomics, Institute of Molecular and Cell Biology, Singapore 138673.</p><p class="abstract">Pancreatic cancer is one of the most deadly cancers affecting the Western world. Because the disease is highly metastatic and difficult to diagnosis until late stages, the 5-y survival rate is around 5%. The identification of molecular cancer drivers is critical for furthering our understanding of the disease and development of improved diagnostic tools and therapeutics. We have conducted a mutagenic screen using Sleeping Beauty (SB) in mice to identify new candidate cancer genes in pancreatic cancer. By combining SB with an oncogenic Kras allele, we observed highly metastatic pancreatic adenocarcinomas. Using two independent statistical methods to identify loci commonly mutated by SB in these tumors, we identified 681 loci that comprise 543 candidate cancer genes (CCGs); 75 of these CCGs, including Mll3 and Ptk2, have known mutations in human pancreatic cancer. We identified point mutations in human pancreatic patient samples for another 11 CCGs, including Acvr2a and Map2k4. Importantly, 10% of the CCGs are involved in chromatin remodeling, including Arid4b, Kdm6a, and Nsd3, and all SB tumors have at least one mutated gene involved in this process; 20 CCGs, including Ctnnd1, Fbxo11, and Vgll4, are also significantly associated with poor patient survival. SB mutagenesis provides a rich resource of mutations in potential cancer drivers for cross-comparative analyses with ongoing sequencing efforts in human pancreatic adenocarcinoma.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">16</span>;<span class="pagerange">5934-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22421440">22421440</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3341075">3341075</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1202490109">10.1073/pnas.1202490109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adaptive mechanisms in pathogens: universal aneuploidy in Leishmania.</h4>
<p class="authors"><span class="author">Mannaert A</span>, <span class="author">Downing T</span>, <span class="author">Imamura H</span> and <span class="author">Dujardin JC</span></p>
<p class="affiliation">Unit of Molecular Parasitology, Department of Biomedical Sciences, Institute of Tropical Medicine, Antwerp, Belgium.</p>
<p class="abstract">Genomic stability and maintenance of the correct chromosome number are assumed to be essential for normal development in eukaryotes. Aneuploidy is usually associated with severe abnormalities and decrease of cell fitness, but some organisms appear to rely on aneuploidy for rapid adaptation to changing environments. This phenomenon is mostly described in pathogenic fungi and cancer cells. However, recent genome studies highlight the importance of Leishmania as a new model for studies on aneuploidy. Several reports revealed extensive variation in chromosome copy number, indicating that aneuploidy is a constitutive feature of this protozoan parasite genus. Aneuploidy appears to be beneficial in organisms that are primarily asexual, unicellular, and that undergo sporadic epidemic expansions, including common pathogens as well as cancer.</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22789456" rel="external">22789456</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2012.06.003" rel="external">10.1016/j.pt.2012.06.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide approach accounting for body mass index identifies genetic variants influencing fasting glycemic traits and insulin resistance.</h4>
<p class="authors"><span class="author">Manning AK</span>, <span class="author">Hivert MF</span>, <span class="author">Scott RA</span>, <span class="author">Grimsby JL</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Chen H</span>, <span class="author">Rybin D</span>, <span class="author">Liu CT</span>, <span class="author">Bielak LF</span>, <span class="author">Prokopenko I</span>, <span class="author">Amin N</span>, <span class="author">Barnes D</span>, <span class="author">Cadby G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Ingelsson E</span>, <span class="author">Jackson AU</span>, <span class="author">Johnson T</span>, <span class="author">Kanoni S</span>, <span class="author">Ladenvall C</span>, <span class="author">Lagou V</span>, <span class="author">Lahti J</span>, <span class="author">Lecoeur C</span>, <span class="author">Liu Y</span>, <span class="author">Martinez-Larrad MT</span>, <span class="author">Montasser ME</span>, <span class="author">Navarro P</span>, <span class="author">Perry JR</span>, <span class="author">Rasmussen-Torvik LJ</span>, <span class="author">Salo P</span>, <span class="author">Sattar N</span>, <span class="author">Shungin D</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Tanaka T</span>, <span class="author">van Duijn CM</span>, <span class="author">An P</span>, <span class="author">de Andrade M</span>, <span class="author">Andrews JS</span>, <span class="author">Aspelund T</span>, <span class="author">Atalay M</span>, <span class="author">Aulchenko Y</span>, <span class="author">Balkau B</span>, <span class="author">Bandinelli S</span>, <span class="author">Beckmann JS</span>, <span class="author">Beilby JP</span>, <span class="author">Bellis C</span>, <span class="author">Bergman RN</span>, <span class="author">Blangero J</span>, <span class="author">Boban M</span>, <span class="author">Boehnke M</span>, <span class="author">Boerwinkle E</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Boomsma DI</span>, <span class="author">Borecki IB</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Bouchard C</span>, <span class="author">Brunner E</span>, <span class="author">Budimir D</span>, <span class="author">Campbell H</span>, <span class="author">Carlson O</span>, <span class="author">Chines PS</span>, <span class="author">Clarke R</span>, <span class="author">Collins FS</span>, <span class="author">Corbat&oacute;n-Anchuelo A</span>, <span class="author">Couper D</span>, <span class="author">de Faire U</span>, <span class="author">Dedoussis GV</span>, <span class="author">Deloukas P</span>, <span class="author">Dimitriou M</span>, <span class="author">Egan JM</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Erdos MR</span>, <span class="author">Eriksson JG</span>, <span class="author">Eury E</span>, <span class="author">Ferrucci L</span>, <span class="author">Ford I</span>, <span class="author">Forouhi NG</span>, <span class="author">Fox CS</span>, <span class="author">Franzosi MG</span>, <span class="author">Franks PW</span>, <span class="author">Frayling TM</span>, <span class="author">Froguel P</span>, <span class="author">Galan P</span>, <span class="author">de Geus E</span>, <span class="author">Gigante B</span>, <span class="author">Glazer NL</span>, <span class="author">Goel A</span>, <span class="author">Groop L</span>, <span class="author">Gudnason V</span>, <span class="author">Hallmans G</span>, <span class="author">Hamsten A</span>, <span class="author">Hansson O</span>, <span class="author">Harris TB</span>, <span class="author">Hayward C</span>, <span class="author">Heath S</span>, <span class="author">Hercberg S</span>, <span class="author">Hicks AA</span>, <span class="author">Hingorani A</span>, <span class="author">Hofman A</span>, <span class="author">Hui J</span>, <span class="author">Hung J</span>, <span class="author">Jarvelin MR</span>, <span class="author">Jhun MA</span>, <span class="author">Johnson PC</span>, <span class="author">Jukema JW</span>, <span class="author">Jula A</span>, <span class="author">Kao WH</span>, <span class="author">Kaprio J</span>, <span class="author">Kardia SL</span>, <span class="author">Keinanen-Kiukaanniemi S</span>, <span class="author">Kivimaki M</span>, <span class="author">Kolcic I</span>, <span class="author">Kovacs P</span>, <span class="author">Kumari M</span>, <span class="author">Kuusisto J</span>, <span class="author">Kyvik KO</span>, <span class="author">Laakso M</span>, <span class="author">Lakka T</span>, <span class="author">Lannfelt L</span>, <span class="author">Lathrop GM</span>, <span class="author">Launer LJ</span>, <span class="author">Leander K</span>, <span class="author">Li G</span>, <span class="author">Lind L</span>, <span class="author">Lindstrom J</span>, <span class="author">Lobbens S</span>, <span class="author">Loos RJ</span>, <span class="author">Luan J</span>, <span class="author">Lyssenko V</span>, <span class="author">M&auml;gi R</span>, <span class="author">Magnusson PK</span>, <span class="author">Marmot M</span>, <span class="author">Meneton P</span>, <span class="author">Mohlke KL</span>, <span class="author">Mooser V</span>, <span class="author">Morken MA</span>, <span class="author">Miljkovic I</span>, <span class="author">Narisu N</span>, <span class="author">O&#39;Connell J</span>, <span class="author">Ong KK</span>, <span class="author">Oostra BA</span>, <span class="author">Palmer LJ</span>, <span class="author">Palotie A</span>, <span class="author">Pankow JS</span>, <span class="author">Peden JF</span>, <span class="author">Pedersen NL</span>, <span class="author">Pehlic M</span>, <span class="author">Peltonen L</span>, <span class="author">Penninx B</span>, <span class="author">Pericic M</span>, <span class="author">Perola M</span>, <span class="author">Perusse L</span>, <span class="author">Peyser PA</span>, <span class="author">Polasek O</span>, <span class="author">Pramstaller PP</span>, <span class="author">Province MA</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Rauramaa R</span>, <span class="author">Rehnberg E</span>, <span class="author">Rice K</span>, <span class="author">Rotter JI</span>, <span class="author">Rudan I</span>, <span class="author">Ruokonen A</span>, <span class="author">Saaristo T</span>, <span class="author">Sabater-Lleal M</span>, <span class="author">Salomaa V</span>, <span class="author">Savage DB</span>, <span class="author">Saxena R</span>, <span class="author">Schwarz P</span>, <span class="author">Seedorf U</span>, <span class="author">Sennblad B</span>, <span class="author">Serrano-Rios M</span>, <span class="author">Shuldiner AR</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Siscovick DS</span>, <span class="author">Smit JH</span>, <span class="author">Small KS</span>, <span class="author">Smith NL</span>, <span class="author">Smith AV</span>, <span class="author">Stan&#x10D;&aacute;kov&aacute; A</span>, <span class="author">Stirrups K</span>, <span class="author">Stumvoll M</span>, <span class="author">Sun YV</span>, <span class="author">Swift AJ</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Tuomilehto J</span>, <span class="author">Trompet S</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Uusitupa M</span>, <span class="author">Vikstr&ouml;m M</span>, <span class="author">Vitart V</span>, <span class="author">Vohl MC</span>, <span class="author">Voight BF</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Waterworth DM</span>, <span class="author">Watkins H</span>, <span class="author">Wheeler E</span>, <span class="author">Widen E</span>, <span class="author">Wild SH</span>, <span class="author">Willems SM</span>, <span class="author">Willemsen G</span>, <span class="author">Wilson JF</span>, <span class="author">Witteman JC</span>, <span class="author">Wright AF</span>, <span class="author">Yaghootkar H</span>, <span class="author">Zelenika D</span>, <span class="author">Zemunik T</span>, <span class="author">Zgaga L</span>, <span class="author">DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium</span>, <span class="author">Multiple Tissue Human Expression Resource (MUTHER) Consortium</span>, <span class="author">Wareham NJ</span>, <span class="author">McCarthy MI</span>, <span class="author">Barroso I</span>, <span class="author">Watanabe RM</span>, <span class="author">Florez JC</span>, <span class="author">Dupuis J</span>, <span class="author">Meigs JB</span> and <span class="author">Langenberg C</span></p>
<p class="affiliation">Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts, USA.</p><p class="abstract">Recent genome-wide association studies have described many loci implicated in type 2 diabetes (T2D) pathophysiology and &beta;-cell dysfunction but have contributed little to the understanding of the genetic basis of insulin resistance. We hypothesized that genes implicated in insulin resistance pathways might be uncovered by accounting for differences in body mass index (BMI) and potential interactions between BMI and genetic variants. We applied a joint meta-analysis approach to test associations with fasting insulin and glucose on a genome-wide scale. We present six previously unknown loci associated with fasting insulin at P &lt; 5 &times; 10(-8) in combined discovery and follow-up analyses of 52 studies comprising up to 96,496 non-diabetic individuals. Risk variants were associated with higher triglyceride and lower high-density lipoprotein (HDL) cholesterol levels, suggesting a role for these loci in insulin resistance pathways. The discovery of these loci will aid further characterization of the role of insulin resistance in T2D pathophysiology.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/07/008/23674</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0100222</span>, <span class="grant">G0701863</span>, <span class="grant">G0900339</span>, <span class="grant">G0902037</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_U137686857</span>, <span class="grant">MC_UP_A100_1003</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NCRR NIH HHS: <span class="grant">S10 RR029392</span>; NHLBI NIH HHS: <span class="grant">R01 HL105756</span>; NIDDK NIH HHS: <span class="grant">K24 DK080140</span>, <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK078616</span>; NIMH NIH HHS: <span class="grant">R37 MH059490</span>; Wellcome Trust: <span class="grant">090532</span>, <span class="grant">091551</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">6</span>;<span class="pagerange">659-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22581228">22581228</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3613127">3613127</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2274">10.1038/ng.2274</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of Plasmodium falciparum diversity in natural infections by deep sequencing.</h4>
<p class="authors"><span class="author">Manske M</span>, <span class="author">Miotto O</span>, <span class="author">Campino S</span>, <span class="author">Auburn S</span>, <span class="author">Almagro-Garcia J</span>, <span class="author">Maslen G</span>, <span class="author">O&#39;Brien J</span>, <span class="author">Djimde A</span>, <span class="author">Doumbo O</span>, <span class="author">Zongo I</span>, <span class="author">Ouedraogo JB</span>, <span class="author">Michon P</span>, <span class="author">Mueller I</span>, <span class="author">Siba P</span>, <span class="author">Nzila A</span>, <span class="author">Borrmann S</span>, <span class="author">Kiara SM</span>, <span class="author">Marsh K</span>, <span class="author">Jiang H</span>, <span class="author">Su XZ</span>, <span class="author">Amaratunga C</span>, <span class="author">Fairhurst R</span>, <span class="author">Socheat D</span>, <span class="author">Nosten F</span>, <span class="author">Imwong M</span>, <span class="author">White NJ</span>, <span class="author">Sanders M</span>, <span class="author">Anastasi E</span>, <span class="author">Alcock D</span>, <span class="author">Drury E</span>, <span class="author">Oyola S</span>, <span class="author">Quail MA</span>, <span class="author">Turner DJ</span>, <span class="author">Ruano-Rubio V</span>, <span class="author">Jyothi D</span>, <span class="author">Amenga-Etego L</span>, <span class="author">Hubbart C</span>, <span class="author">Jeffreys A</span>, <span class="author">Rowlands K</span>, <span class="author">Sutherland C</span>, <span class="author">Roper C</span>, <span class="author">Mangano V</span>, <span class="author">Modiano D</span>, <span class="author">Tan JC</span>, <span class="author">Ferdig MT</span>, <span class="author">Amambua-Ngwa A</span>, <span class="author">Conway DJ</span>, <span class="author">Takala-Harrison S</span>, <span class="author">Plowe CV</span>, <span class="author">Rayner JC</span>, <span class="author">Rockett KA</span>, <span class="author">Clark TG</span>, <span class="author">Newbold CI</span>, <span class="author">Berriman M</span>, <span class="author">MacInnis B</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Malaria elimination strategies require surveillance of the parasite population for genetic changes that demand a public health response, such as new forms of drug resistance. Here we describe methods for the large-scale analysis of genetic variation in Plasmodium falciparum by deep sequencing of parasite DNA obtained from the blood of patients with malaria, either directly or after short-term culture. Analysis of 86,158 exonic single nucleotide polymorphisms that passed genotyping quality control in 227 samples from Africa, Asia and Oceania provides genome-wide estimates of allele frequency distribution, population structure and linkage disequilibrium. By comparing the genetic diversity of individual infections with that of the local parasite population, we derive a metric of within-host diversity that is related to the level of inbreeding in the population. An open-access web application has been established for the exploration of regional differences in allele frequency and of highly differentiated loci in the P.&thinsp;falciparum genome.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute: <span class="grant">55005502</span>; Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">082370</span>, <span class="grant">089275</span>, <span class="grant">090532</span>, <span class="grant">090532/Z/09/Z</span>, <span class="grant">090770</span>, <span class="grant">090770/Z/09/Z</span>, <span class="grant">092654</span>, <span class="grant">093956</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">487</span>;<span class="number">7407</span>;<span class="pagerange">375-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22722859">22722859</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3738909">3738909</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11174">10.1038/nature11174</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of Brachyspira pilosicoli strains: genome rearrangements, reductions and correlation of genetic compliment with phenotypic diversity.</h4>
<p class="authors"><span class="author">Mappley LJ</span>, <span class="author">Black ML</span>, <span class="author">AbuOun M</span>, <span class="author">Darby AC</span>, <span class="author">Woodward MJ</span>, <span class="author">Parkhill J</span>, <span class="author">Turner AK</span>, <span class="author">Bellgard MI</span>, <span class="author">La T</span>, <span class="author">Phillips ND</span>, <span class="author">La Ragione RM</span> and <span class="author">Hampson DJ</span></p>
<p class="affiliation">Department of Bacteriology, Animal Health and Veterinary Laboratories Agency, Reading University, Addlestone, Surrey, UK. <a href="mailto:%6C.%6D%61%70%70%6C%65%79%40%72%65%61%64%69%6E%67.%61%63.%75%6B">&#108;&#46;&#109;&#97;&#112;&#112;&#108;&#101;&#121;&#64;&#114;&#101;&#97;&#100;&#105;&#110;&#103;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="abstract">Background: The anaerobic spirochaete Brachyspira pilosicoli causes enteric disease in avian, porcine and human hosts, amongst others. To date, the only available genome sequence of B. pilosicoli is that of strain 95/1000, a porcine isolate. In the first intra-species genome comparison within the Brachyspira genus, we report the whole genome sequence of B. pilosicoli B2904, an avian isolate, the incomplete genome sequence of B. pilosicoli WesB, a human isolate, and the comparisons with B. pilosicoli 95/1000. We also draw on incomplete genome sequences from three other Brachyspira species. Finally we report the first application of the high-throughput Biolog phenotype screening tool on the B. pilosicoli strains for detailed comparisons between genotype and phenotype.

Results: Feature and sequence genome comparisons revealed a high degree of similarity between the three B. pilosicoli strains, although the genomes of B2904 and WesB were larger than that of 95/1000 (~2,765, 2.890 and 2.596 Mb, respectively). Genome rearrangements were observed which correlated largely with the positions of mobile genetic elements. Through comparison of the B2904 and WesB genomes with the 95/1000 genome, features that we propose are non-essential due to their absence from 95/1000 include a peptidase, glycine reductase complex components and transposases. Novel bacteriophages were detected in the newly-sequenced genomes, which appeared to have involvement in intra- and inter-species horizontal gene transfer. Phenotypic differences predicted from genome analysis, such as the lack of genes for glucuronate catabolism in 95/1000, were confirmed by phenotyping.

Conclusions: The availability of multiple B. pilosicoli genome sequences has allowed us to demonstrate the substantial genomic variation that exists between these strains, and provides an insight into genetic events that are shaping the species. In addition, phenotype screening allowed determination of how genotypic differences translated to phenotype. Further application of such comparisons will improve understanding of the metabolic capabilities of Brachyspira species.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">454</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22947175">22947175</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3532143">3532143</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-454">10.1186/1471-2164-13-454</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The capsular polysaccharide Vi from Salmonella typhi is a B1b antigen.</h4>
<p class="authors"><span class="author">Marshall JL</span>, <span class="author">Flores-Langarica A</span>, <span class="author">Kingsley RA</span>, <span class="author">Hitchcock JR</span>, <span class="author">Ross EA</span>, <span class="author">L&oacute;pez-Mac&iacute;as C</span>, <span class="author">Lakey J</span>, <span class="author">Martin LB</span>, <span class="author">Toellner KM</span>, <span class="author">MacLennan CA</span>, <span class="author">MacLennan IC</span>, <span class="author">Henderson IR</span>, <span class="author">Dougan G</span> and <span class="author">Cunningham AF</span></p>
<p class="affiliation">Medical Research Council Centre for Immune Regulation, School of Immunity and Infection, Institute for Biomedical Research, University of Birmingham, Edgbaston, Birmingham B15 2TT, United Kingdom.</p>
<p class="abstract">Vaccination with purified capsular polysaccharide Vi Ag from Salmonella typhi can protect against typhoid fever, although the mechanism for its efficacy is not clearly established. In this study, we have characterized the B cell response to this vaccine in wild-type and T cell-deficient mice. We show that immunization with typhoid Vi polysaccharide vaccine rapidly induces proliferation in B1b peritoneal cells, but not in B1a cells or marginal zone B cells. This induction of B1b proliferation is concomitant with the detection of splenic Vi-specific Ab-secreting cells and protective Ab in Rag1-deficient B1b cell chimeras generated by adoptive transfer-induced specific Ab after Vi immunization. Furthermore, Ab derived from peritoneal B cells is sufficient to confer protection against Salmonella that express Vi Ag. Expression of Vi by Salmonella during infection did not inhibit the development of early Ab responses to non-Vi Ags. Despite this, the protection conferred by immunization of mice with porin proteins from Salmonella, which induce Ab-mediated protection, was reduced postinfection with Vi-expressing Salmonella, although protection was not totally abrogated. This work therefore suggests that, in mice, B1b cells contribute to the protection induced by Vi Ag, and targeting non-Vi Ags as subunit vaccines may offer an attractive strategy to augment current Vi-based vaccine strategies.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2012</span>;<span class="volume">189</span>;<span class="number">12</span>;<span class="pagerange">5527-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23162127">23162127</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3605773">3605773</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1103166">10.4049/jimmunol.1103166</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Subtype of GluN2 C-terminal Domain Determines the Response to Excitotoxic Insults.</h4>
<p class="authors"><span class="author">Martel MA</span>, <span class="author">Ryan TJ</span>, <span class="author">Bell KF</span>, <span class="author">Fowler JH</span>, <span class="author">McMahon A</span>, <span class="author">Al-Mubarak B</span>, <span class="author">Komiyama NH</span>, <span class="author">Horsburgh K</span>, <span class="author">Kind PC</span>, <span class="author">Grant SG</span>, <span class="author">Wyllie DJ</span> and <span class="author">Hardingham GE</span></p>
<p class="affiliation">Centre for Integrative Physiology, University of Edinburgh School of Biomedical Sciences, Hugh Robson Building, George Square, Edinburgh&nbsp;EH8 9XD, UK.</p><p class="abstract">It is currently unclear whether the GluN2 subtype influences NMDA receptor (NMDAR) excitotoxicity. We report that the toxicity of NMDAR-mediated Ca(2+) influx is differentially controlled by the cytoplasmic C-terminal domains of GluN2B (CTD(2B)) and GluN2A (CTD(2A)). Studying the effects of acute expression of GluN2A/2B-based chimeric subunits with reciprocal exchanges of their CTDs revealed that CTD(2B) enhances NMDAR toxicity, compared to CTD(2A). Furthermore, the vulnerability of forebrain neurons in&nbsp;vitro and in&nbsp;vivo to NMDAR-dependent Ca(2+) influx is lowered by replacing the CTD of GluN2B with that of GluN2A by targeted exon exchange in a mouse knockin model. Mechanistically, CTD(2B) exhibits stronger physical/functional coupling to the PSD-95-nNOS pathway, which suppresses protective CREB activation. Dependence of NMDAR excitotoxicity on the GluN2 CTD subtype can be overcome by inducing high levels of NMDAR activity. Thus, the identity (2A versus 2B) of the GluN2 CTD controls the toxicity dose-response to episodes of NMDAR activity.</p>
<p class="publication"><span class="title">Neuron</span>&nbsp;<span class="year">2012</span>;<span class="volume">74</span>;<span class="number">3</span>;<span class="pagerange">543-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22578505" rel="external">22578505</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neuron.2012.03.021" rel="external">10.1016/j.neuron.2012.03.021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of recombination events in bacterial genomes from large population samples.</h4>
<p class="authors"><span class="author">Marttinen P</span>, <span class="author">Hanage WP</span>, <span class="author">Croucher NJ</span>, <span class="author">Connor TR</span>, <span class="author">Harris SR</span>, <span class="author">Bentley SD</span> and <span class="author">Corander J</span></p>
<p class="affiliation">Department of Biomedical Engineering and Computational Science, Aalto University, PO Box 12200, FI-00076 AALTO, Finland. <a href="mailto:%70%65%6B%6B%61.%6D%61%72%74%74%69%6E%65%6E%40%61%61%6C%74%6F.%66%69">&#112;&#101;&#107;&#107;&#97;&#46;&#109;&#97;&#114;&#116;&#116;&#105;&#110;&#101;&#110;&#64;&#97;&#97;&#108;&#116;&#111;&#46;&#102;&#105;</a></p><p class="abstract">Analysis of important human pathogen populations is currently under transition toward whole-genome sequencing of growing numbers of samples collected on a global scale. Since recombination in bacteria is often an important factor shaping their evolution by enabling resistance elements and virulence traits to rapidly transfer from one evolutionary lineage to another, it is highly beneficial to have access to tools that can detect recombination events. Multiple advanced statistical methods exist for such purposes; however, they are typically limited either to only a few samples or to data from relatively short regions of a total genome. By harnessing the power of recent advances in Bayesian modeling techniques, we introduce here a method for detecting homologous recombination events from whole-genome sequence data for bacterial population samples on a large scale. Our statistical approach can efficiently handle hundreds of whole genome sequenced population samples and identify separate origins of the recombinant sequence, offering an enhanced insight into the diversification of bacterial clones at the level of the whole genome. A data set of 241 whole genome sequences from an important pandemic lineage of Streptococcus pneumoniae is used together with multiple simulated data sets to demonstrate the potential of our approach.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54GM088558</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">1</span>;<span class="pagerange">e6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22064866" rel="external">22064866</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245952" rel="external">3245952</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr928" rel="external">10.1093/nar/gkr928</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrating cytogenetics and genomics in comparative evolutionary studies of cichlid fish.</h4>
<p class="authors"><span class="author">Mazzuchelli J</span>, <span class="author">Kocher TD</span>, <span class="author">Yang F</span> and <span class="author">Martins C</span></p>
<p class="affiliation">Department of Morphology, Bioscience Institute, UNESP - São Paulo State University, 18618-970, Botucatu, SP, Brazil. <a href="mailto:%63%6D%61%72%74%69%6E%73%40%69%62%62.%75%6E%65%73%70.%62%72">&#99;&#109;&#97;&#114;&#116;&#105;&#110;&#115;&#64;&#105;&#98;&#98;&#46;&#117;&#110;&#101;&#115;&#112;&#46;&#98;&#114;</a>.</p>
<p class="abstract">Unlabelled: ABSTRACT:

Background: The availability of a large number of recently sequenced vertebrate genomes opens new avenues to integrate cytogenetics and genomics in comparative and evolutionary studies. Cytogenetic mapping can offer alternative means to identify conserved synteny shared by distinct genomes and also to define genome regions that are still not fine characterized even after wide-ranging nucleotide sequence efforts. An efficient way to perform comparative cytogenetic mapping is based on BAC clones mapping by fluorescence in situ hybridization. In this report, to address the knowledge gap on the genome evolution in cichlid fishes, BAC clones of an Oreochromis niloticus library covering the linkage groups (LG) 1, 3, 5, and 7 were mapped onto the chromosomes of 9 African cichlid species. The cytogenetic mapping data were also integrated with BAC-end sequences information of O. niloticus and comparatively analyzed against the genome of other fish species and vertebrates.

Results: The location of BACs from LG1, 3, 5, and 7 revealed a strong chromosomal conservation among the analyzed cichlid species genomes, which evidenced a synteny of the markers of each LG. Comparative in silico analysis also identified large genomic blocks that were conserved in distantly related fish groups and also in other vertebrates.

Conclusions: Although it has been suggested that fishes contain plastic genomes with high rates of chromosomal rearrangements and probably low rates of synteny conservation, our results evidence that large syntenic chromosome segments have been maintained conserved during evolution, at least for the considered markers. Additionally, our current cytogenetic mapping efforts integrated with genomic approaches conduct to a new perspective to address important questions involving chromosome evolution in fishes.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">463</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22958299" rel="external">22958299</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3463429" rel="external">3463429</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-463" rel="external">10.1186/1471-2164-13-463</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular tracing of the emergence, adaptation, and transmission of hospital-associated methicillin-resistant Staphylococcus aureus.</h4>
<p class="authors"><span class="author">McAdam PR</span>, <span class="author">Templeton KE</span>, <span class="author">Edwards GF</span>, <span class="author">Holden MT</span>, <span class="author">Feil EJ</span>, <span class="author">Aanensen DM</span>, <span class="author">Bargawi HJ</span>, <span class="author">Spratt BG</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Enright MC</span>, <span class="author">Holmes A</span>, <span class="author">Girvan EK</span>, <span class="author">Godfrey PA</span>, <span class="author">Feldgarden M</span>, <span class="author">Kearns AM</span>, <span class="author">Rambaut A</span>, <span class="author">Robinson DA</span> and <span class="author">Fitzgerald JR</span></p>
<p class="affiliation">The Roslin Institute and Edinburgh Infectious Diseases, Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH259RG, United Kingdom.</p>
<p class="abstract">Hospital-associated infections caused by methicillin-resistant Staphylococcus aureus (MRSA) are a global health burden dominated by a small number of bacterial clones. The pandemic EMRSA-16 clone (ST36-II) has been widespread in UK hospitals for 20 y, but its evolutionary origin and the molecular basis for its hospital association are unclear. We carried out a Bayesian phylogenetic reconstruction on the basis of the genome sequences of 87 S. aureus isolates including 60 EMRSA-16 and 27 additional clonal complex 30 (CC30) isolates, collected from patients in three continents over a 53-y period. The three major pandemic clones to originate from the CC30 lineage, including phage type 80/81, Southwest Pacific, and EMRSA-16, shared a most recent common ancestor that existed over 100 y ago, whereas the hospital-associated EMRSA-16 clone is estimated to have emerged about 35 y ago. Our CC30 genome-wide analysis revealed striking molecular correlates of hospital- or community-associated pandemics represented by mobile genetic elements and nonsynonymous mutations affecting antibiotic resistance and virulence. Importantly, phylogeographic analysis indicates that EMRSA-16 spread within the United Kingdom by transmission from hospitals in large population centers in London and Glasgow to regional health-care settings, implicating patient referrals as an important cause of nationwide transmission. Taken together, the high-resolution phylogenomic approach used resulted in a unique understanding of the emergence and transmission of a major MRSA clone and provided molecular correlates of its hospital adaptation. Similar approaches for hospital-associated clones of other bacterial pathogens may inform appropriate measures for controlling their intra- and interhospital spread.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Chief Scientist Office; Medical Research Council; NIGMS NIH HHS: <span class="grant">GM080602</span>, <span class="grant">R01 GM080602</span>; PHS HHS: <span class="grant">HHSN272200900018C</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">23</span>;<span class="pagerange">9107-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22586109" rel="external">22586109</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3384211" rel="external">3384211</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1202869109" rel="external">10.1073/pnas.1202869109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tandem duplication of chromosomal segments is common in ovarian and breast cancer genomes.</h4>
<p class="authors"><span class="author">McBride DJ</span>, <span class="author">Etemadmoghadam D</span>, <span class="author">Cooke SL</span>, <span class="author">Alsop K</span>, <span class="author">George J</span>, <span class="author">Butler A</span>, <span class="author">Cho J</span>, <span class="author">Galappaththige D</span>, <span class="author">Greenman C</span>, <span class="author">Howarth KD</span>, <span class="author">Lau KW</span>, <span class="author">Ng CK</span>, <span class="author">Raine K</span>, <span class="author">Teague J</span>, <span class="author">Wedge DC</span>, <span class="author">Cancer Study Group AO</span>, <span class="author">Caubit X</span>, <span class="author">Stratton MR</span>, <span class="author">Brenton JD</span>, <span class="author">Campbell PJ</span>, <span class="author">Futreal PA</span> and <span class="author">Bowtell DD</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%64%61%76%69%64.%62%6F%77%74%65%6C%6C%40%70%65%74%65%72%6D%61%63.%6F%72%67">&#100;&#97;&#118;&#105;&#100;&#46;&#98;&#111;&#119;&#116;&#101;&#108;&#108;&#64;&#112;&#101;&#116;&#101;&#114;&#109;&#97;&#99;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The application of paired-end next generation sequencing approaches has made it possible to systematically characterize rearrangements of the cancer genome to base-pair level. Utilizing this approach, we report the first detailed analysis of ovarian cancer rearrangements, comparing high-grade serous and clear cell cancers, and these histotypes with other solid cancers. Somatic rearrangements were systematically characterized in eight high-grade serous and five clear cell ovarian cancer genomes and we report here the identification of &gt; 600 somatic rearrangements. Recurrent rearrangements of the transcriptional regulator gene, TSHZ3, were found in three of eight serous cases. Comparison to breast, pancreatic and prostate cancer genomes revealed that a subset of ovarian cancers share a marked tandem duplication phenotype with triple-negative breast cancers. The tandem duplication phenotype was not linked to BRCA1/2 mutation, suggesting that other common mechanisms or carcinogenic exposures are operative. High-grade serous cancers arising in women with germline BRCA1 or BRCA2 mutation showed a high frequency of small chromosomal deletions. These findings indicate that BRCA1/2 germline mutation may contribute to widespread structural change and that other undefined mechanism(s), which are potentially shared with triple-negative breast cancer, promote tandem chromosomal duplications that sculpt the ovarian cancer genome.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">The Journal of pathology</span>&nbsp;<span class="year">2012</span>;<span class="volume">227</span>;<span class="number">4</span>;<span class="pagerange">446-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22514011" rel="external">22514011</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428857" rel="external">3428857</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpath.4042" rel="external">10.1002/path.4042</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Disruption of mouse Cenpj, a regulator of centriole biogenesis, phenocopies Seckel syndrome.</h4>
<p class="authors"><span class="author">McIntyre RE</span>, <span class="author">Lakshminarasimhan Chavali P</span>, <span class="author">Ismail O</span>, <span class="author">Carragher DM</span>, <span class="author">Sanchez-Andrade G</span>, <span class="author">Forment JV</span>, <span class="author">Fu B</span>, <span class="author">Del Castillo Velasco-Herrera M</span>, <span class="author">Edwards A</span>, <span class="author">van der Weyden L</span>, <span class="author">Yang F</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Estabel J</span>, <span class="author">Gallagher FA</span>, <span class="author">Logan DW</span>, <span class="author">Arends MJ</span>, <span class="author">Tsang SH</span>, <span class="author">Mahajan VB</span>, <span class="author">Scudamore CL</span>, <span class="author">White JK</span>, <span class="author">Jackson SP</span>, <span class="author">Gergely F</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">Disruption of the centromere protein J gene, CENPJ (CPAP, MCPH6, SCKL4), which is a highly conserved and ubiquitiously expressed centrosomal protein, has been associated with primary microcephaly and the microcephalic primordial dwarfism disorder Seckel syndrome. The mechanism by which disruption of CENPJ causes the proportionate, primordial growth failure that is characteristic of Seckel syndrome is unknown. By generating a hypomorphic allele of Cenpj, we have developed a mouse (Cenpj(tm/tm)) that recapitulates many of the clinical features of Seckel syndrome, including intrauterine dwarfism, microcephaly with memory impairment, ossification defects, and ocular and skeletal abnormalities, thus providing clear confirmation that specific mutations of CENPJ can cause Seckel syndrome. Immunohistochemistry revealed increased levels of DNA damage and apoptosis throughout Cenpj(tm/tm) embryos and adult mice showed an elevated frequency of micronucleus induction, suggesting that Cenpj-deficiency results in genomic instability. Notably, however, genomic instability was not the result of defective ATR-dependent DNA damage signaling, as is the case for the majority of genes associated with Seckel syndrome. Instead, Cenpj(tm/tm) embryonic fibroblasts exhibited irregular centriole and centrosome numbers and mono- and multipolar spindles, and many were near-tetraploid with numerical and structural chromosomal abnormalities when compared to passage-matched wild-type cells. Increased cell death due to mitotic failure during embryonic development is likely to contribute to the proportionate dwarfism that is associated with CENPJ-Seckel syndrome.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">11224</span>, <span class="grant">12401</span>, <span class="grant">13031</span>, <span class="grant">A11224</span>; European Research Council: <span class="grant">268536</span>; Medical Research Council: <span class="grant">G0901338</span>; NEI NIH HHS: <span class="grant">K08 EY020530</span>, <span class="grant">NIH 1K08EY020530-01A1</span>; Wellcome Trust: <span class="grant">092096</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e1003022</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23166506">23166506</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3499256">3499256</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003022">10.1371/journal.pgen.1003022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer gene discovery in the mouse.</h4>
<p class="authors"><span class="author">McIntyre RE</span>, <span class="author">van der Weyden L</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, The Wellcome Trust Sanger Institute, Hinxton, Cambs CB10 1HH, UK.</p><p class="abstract">Developments in high-throughput genome analysis and in computational tools have made it possible to rapidly profile entire cancer genomes with basepair resolution. In parallel with these advances, mouse models of cancer have evolved into powerful tools for cancer gene discovery. Here we discuss some of the approaches that may be used for cancer gene identification in the mouse and discuss how a cross-species &#39;oncogenomics&#39; approach to cancer gene discovery represents a powerful strategy for finding genes that drive tumorigenesis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in genetics &amp; development</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">1</span>;<span class="pagerange">14-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22265936">22265936</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gde.2011.12.003">10.1016/j.gde.2011.12.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SynGAP isoforms exert opposing effects on synaptic strength.</h4>
<p class="authors"><span class="author">McMahon AC</span>, <span class="author">Barnett MW</span>, <span class="author">O&#39;Leary TS</span>, <span class="author">Stoney PN</span>, <span class="author">Collins MO</span>, <span class="author">Papadia S</span>, <span class="author">Choudhary JS</span>, <span class="author">Komiyama NH</span>, <span class="author">Grant SG</span>, <span class="author">Hardingham GE</span>, <span class="author">Wyllie DJ</span> and <span class="author">Kind PC</span></p>
<p class="affiliation">Centre for Integrative Physiology, University of Edinburgh, Edinburgh EH8 9XD, UK.</p><p class="abstract">Alternative promoter usage and alternative splicing enable diversification of the transcriptome. Here we demonstrate that the function of Synaptic GTPase-Activating Protein (SynGAP), a key synaptic protein, is determined by the combination of its amino-terminal sequence with its carboxy-terminal sequence. 5&#39; rapid amplification of cDNA ends and primer extension show that different N-terminal protein sequences arise through alternative promoter usage that are regulated by synaptic activity and postnatal age. Heterogeneity in C-terminal protein sequence arises through alternative splicing. Overexpression of SynGAP &alpha;1 versus &alpha;2 C-termini-containing proteins in hippocampal neurons has opposing effects on synaptic strength, decreasing and increasing miniature excitatory synaptic currents amplitude/frequency, respectively. The magnitude of this C-terminal-dependent effect is modulated by the N-terminal peptide sequence. This is the first demonstration that activity-dependent alternative promoter usage can change the function of a synaptic protein at excitatory synapses. Furthermore, the direction and degree of synaptic modulation exerted by different protein isoforms from a single gene locus is dependent on the combination of differential promoter usage and alternative splicing.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300466</span>, <span class="grant">G0601584</span>, <span class="grant">G0700967</span>, <span class="grant">G0902044</span>, <span class="grant">G0902044(94018)</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature communications</span>&nbsp;<span class="year">2012</span>;<span class="volume">3</span>;<span class="pagerange">900</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22692543">22692543</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3621422">3621422</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncomms1900">10.1038/ncomms1900</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New insights into the bacterial fitness-associated mechanisms revealed by the characterization of large plasmids of an avian pathogenic E. coli.</h4>
<p class="authors"><span class="author">Mellata M</span>, <span class="author">Maddux JT</span>, <span class="author">Nam T</span>, <span class="author">Thomson N</span>, <span class="author">Hauser H</span>, <span class="author">Stevens MP</span>, <span class="author">Mukhopadhyay S</span>, <span class="author">Sarker S</span>, <span class="author">Crabb&eacute; A</span>, <span class="author">Nickerson CA</span>, <span class="author">Santander J</span> and <span class="author">Curtiss R</span></p>
<p class="affiliation">The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America. <a href="mailto:%6D%65%6C%68%61.%6D%65%6C%6C%61%74%61%40%61%73%75.%65%64%75">&#109;&#101;&#108;&#104;&#97;&#46;&#109;&#101;&#108;&#108;&#97;&#116;&#97;&#64;&#97;&#115;&#117;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Extra-intestinal pathogenic E. coli (ExPEC), including avian pathogenic E. coli (APEC), pose a considerable threat to both human and animal health, with illness causing substantial economic loss. APEC strain &chi;7122 (O78&#x2236;K80&#x2236;H9), containing three large plasmids [pChi7122-1 (IncFIB/FIIA-FIC), pChi7122-2 (IncFII), and pChi7122-3 (IncI(2))]; and a small plasmid pChi7122-4 (ColE2-like), has been used for many years as a model strain to study the molecular mechanisms of ExPEC pathogenicity and zoonotic potential. We previously sequenced and characterized the plasmid pChi7122-1 and determined its importance in systemic APEC infection; however the roles of the other pChi7122 plasmids were still ambiguous. Herein we present the sequence of the remaining pChi7122 plasmids, confirming that pChi7122-2 and pChi7122-3 encode an ABC iron transport system (eitABCD) and a putative type IV fimbriae respectively, whereas pChi7122-4 is a cryptic plasmid. New features were also identified, including a gene cluster on pChi7122-2 that is not present in other E. coli strains but is found in Salmonella serovars and is predicted to encode the sugars catabolic pathways. In vitro evaluation of the APEC &chi;7122 derivative strains with the three large plasmids, either individually or in combinations, provided new insights into the role of plasmids in biofilm formation, bile and acid tolerance, and the interaction of E. coli strains with 3-D cultures of intestinal epithelial cells. In this study, we show that the nature and combinations of plasmids, as well as the background of the host strains, have an effect on these phenomena. Our data reveal new insights into the role of extra-chromosomal sequences in fitness and diversity of ExPEC in their phenotypes.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R21 AI090416</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">e29481</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22238616" rel="external">22238616</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3251573" rel="external">3251573</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0029481" rel="external">10.1371/journal.pone.0029481</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Communication about DTC testing: commentary on a &#39;family experience of personal genomics&#39;.</h4>
<p class="authors"><span class="author">Middleton A</span></p><p class="abstract">This paper provides a commentary on &#39;Family Experience of Personal Genomics&#39; (Corpas 2012). An overview is offered on the communication literature available to help support individuals and families to communicate about genetic information. Despite there being a wealth of evidence, built on years of genetic counseling practice, this does not appear to have been translated clearly to the Direct to Consumer (DTC) testing market. In many countries it is possible to order a DTC genetic test without the involvement of any health professional; there has been heated debate about whether this is appropriate or not. Much of the focus surrounding this has been on whether it is necessary to have a health professional available to offer their clinical knowledge and help with interpreting the DTC genetic test data. What has been missed from this debate is the importance of enabling customers of DTC testing services access to the abundance of information about how to communicate their genetic risks to others, including immediate family. Family communication about health and indeed genetics can be fraught with difficulty. Genetic health professionals, specifically genetic counselors, have particular expertise in family communication about genetics. Such information could be incredibly useful to kinships as they grapple with knowing how to communicate their genomic information with relatives.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Journal of genetic counseling</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">392-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22223062">22223062</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10897-011-9472-8">10.1007/s10897-011-9472-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of the Sotos syndrome deletion in mice.</h4>
<p class="authors"><span class="author">Migdalska AM</span>, <span class="author">van der Weyden L</span>, <span class="author">Ismail O</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Rust AG</span>, <span class="author">Rashid M</span>, <span class="author">White JK</span>, <span class="author">S&aacute;nchez-Andrade G</span>, <span class="author">Lupski JR</span>, <span class="author">Logan DW</span>, <span class="author">Arends MJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1HH, UK.</p><p class="abstract">Haploinsufficiency of the human 5q35 region spanning the NSD1 gene results in a rare genomic disorder known as Sotos syndrome (Sotos), with patients displaying a variety of clinical features, including pre- and postnatal overgrowth, intellectual disability, and urinary/renal abnormalities. We used chromosome engineering to generate a segmental monosomy, i.e., mice carrying a heterozygous 1.5-Mb deletion of 36 genes on mouse chromosome 13 (4732471D19Rik-B4galt7), syntenic with 5q35.2-q35.3 in humans (Df(13)Ms2Dja ( +/- ) mice). Surprisingly Df(13)Ms2Dja ( +/- ) mice were significantly smaller for their gestational age and also showed decreased postnatal growth, in contrast to Sotos patients. Df(13)Ms2Dja ( +/- ) mice did, however, display deficits in long-term memory retention and dilation of the pelvicalyceal system, which in part may model the learning difficulties and renal abnormalities observed in Sotos patients. Thus, haploinsufficiency of genes within the mouse 4732471D19Rik-B4galt7 deletion interval play important roles in growth, memory retention, and the development of the renal pelvicalyceal system.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span>;<span class="volume">23</span>;<span class="number">11-12</span>;<span class="pagerange">749-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22926222">22926222</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3510424">3510424</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9416-0">10.1007/s00335-012-9416-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modeling partial monosomy for human chromosome 21q11.2-q21.1 reveals haploinsufficient genes influencing behavior and fat deposition.</h4>
<p class="authors"><span class="author">Migdalska AM</span>, <span class="author">van der Weyden L</span>, <span class="author">Ismail O</span>, <span class="author">White JK</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">S&aacute;nchez-Andrade G</span>, <span class="author">Logan DW</span>, <span class="author">Arends MJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Haploinsufficiency of part of human chromosome 21 results in a rare condition known as Monosomy 21. This disease displays a variety of clinical phenotypes, including intellectual disability, craniofacial dysmorphology, skeletal and cardiac abnormalities, and respiratory complications. To search for dosage-sensitive genes involved in this disorder, we used chromosome engineering to generate a mouse model carrying a deletion of the Lipi-Usp25 interval, syntenic with 21q11.2-q21.1 in humans. Haploinsufficiency for the 6 genes in this interval resulted in no gross morphological defects and behavioral analysis performed using an open field test, a test of anxiety, and tests for social interaction were normal in monosomic mice. Monosomic mice did, however, display impaired memory retention compared to control animals. Moreover, when fed a high-fat diet (HFD) monosomic mice exhibited a significant increase in fat mass/fat percentage estimate compared with controls, severe fatty changes in their livers, and thickened subcutaneous fat. Thus, genes within the Lipi-Usp25 interval may participate in memory retention and in the regulation of fat deposition.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">e29681</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22276124">22276124</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3262805">3262805</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0029681">10.1371/journal.pone.0029681</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FiRePat - Finding Regulatory Patterns between sRNAs and Genes</h4>
<p class="authors"><span class="author">Mohorianu,I.</span>, <span class="author">Lopez-Gomollon,S.</span>, <span class="author">SCHWACH,F.</span>, <span class="author">Dalmay,T.</span> and <span class="author">Moulton,V.</span></p>
<p class="publication"><span class="title">Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery</span>&nbsp;<span class="year">2012</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">273-84</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1002%2Fwidm.1053" rel="external">10.1002/widm.1053</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale association analysis provides insights into the genetic architecture and pathophysiology of type 2 diabetes.</h4>
<p class="authors"><span class="author">Morris AP</span>, <span class="author">Voight BF</span>, <span class="author">Teslovich TM</span>, <span class="author">Ferreira T</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Khan H</span>, <span class="author">Grallert H</span>, <span class="author">Mahajan A</span>, <span class="author">Prokopenko I</span>, <span class="author">Kang HM</span>, <span class="author">Dina C</span>, <span class="author">Esko T</span>, <span class="author">Fraser RM</span>, <span class="author">Kanoni S</span>, <span class="author">Kumar A</span>, <span class="author">Lagou V</span>, <span class="author">Langenberg C</span>, <span class="author">Luan J</span>, <span class="author">Lindgren CM</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Pechlivanis S</span>, <span class="author">Rayner NW</span>, <span class="author">Scott LJ</span>, <span class="author">Wiltshire S</span>, <span class="author">Yengo L</span>, <span class="author">Kinnunen L</span>, <span class="author">Rossin EJ</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Johnson AD</span>, <span class="author">Dimas AS</span>, <span class="author">Loos RJ</span>, <span class="author">Vedantam S</span>, <span class="author">Chen H</span>, <span class="author">Florez JC</span>, <span class="author">Fox C</span>, <span class="author">Liu CT</span>, <span class="author">Rybin D</span>, <span class="author">Couper DJ</span>, <span class="author">Kao WH</span>, <span class="author">Li M</span>, <span class="author">Cornelis MC</span>, <span class="author">Kraft P</span>, <span class="author">Sun Q</span>, <span class="author">van Dam RM</span>, <span class="author">Stringham HM</span>, <span class="author">Chines PS</span>, <span class="author">Fischer K</span>, <span class="author">Fontanillas P</span>, <span class="author">Holmen OL</span>, <span class="author">Hunt SE</span>, <span class="author">Jackson AU</span>, <span class="author">Kong A</span>, <span class="author">Lawrence R</span>, <span class="author">Meyer J</span>, <span class="author">Perry JR</span>, <span class="author">Platou CG</span>, <span class="author">Potter S</span>, <span class="author">Rehnberg E</span>, <span class="author">Robertson N</span>, <span class="author">Sivapalaratnam S</span>, <span class="author">Stan&#x10D;&aacute;kov&aacute; A</span>, <span class="author">Stirrups K</span>, <span class="author">Thorleifsson G</span>, <span class="author">Tikkanen E</span>, <span class="author">Wood AR</span>, <span class="author">Almgren P</span>, <span class="author">Atalay M</span>, <span class="author">Benediktsson R</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Burtt N</span>, <span class="author">Carey J</span>, <span class="author">Charpentier G</span>, <span class="author">Crenshaw AT</span>, <span class="author">Doney AS</span>, <span class="author">Dorkhan M</span>, <span class="author">Edkins S</span>, <span class="author">Emilsson V</span>, <span class="author">Eury E</span>, <span class="author">Forsen T</span>, <span class="author">Gertow K</span>, <span class="author">Gigante B</span>, <span class="author">Grant GB</span>, <span class="author">Groves CJ</span>, <span class="author">Guiducci C</span>, <span class="author">Herder C</span>, <span class="author">Hreidarsson AB</span>, <span class="author">Hui J</span>, <span class="author">James A</span>, <span class="author">Jonsson A</span>, <span class="author">Rathmann W</span>, <span class="author">Klopp N</span>, <span class="author">Kravic J</span>, <span class="author">Krjut&scaron;kov K</span>, <span class="author">Langford C</span>, <span class="author">Leander K</span>, <span class="author">Lindholm E</span>, <span class="author">Lobbens S</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">Mirza G</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">Musk B</span>, <span class="author">Parkin M</span>, <span class="author">Rallidis L</span>, <span class="author">Saramies J</span>, <span class="author">Sennblad B</span>, <span class="author">Shah S</span>, <span class="author">Sigur&eth;sson G</span>, <span class="author">Silveira A</span>, <span class="author">Steinbach G</span>, <span class="author">Thorand B</span>, <span class="author">Trakalo J</span>, <span class="author">Veglia F</span>, <span class="author">Wennauer R</span>, <span class="author">Winckler W</span>, <span class="author">Zabaneh D</span>, <span class="author">Campbell H</span>, <span class="author">van Duijn C</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">Sijbrands E</span>, <span class="author">Abecasis GR</span>, <span class="author">Owen KR</span>, <span class="author">Zeggini E</span>, <span class="author">Trip MD</span>, <span class="author">Forouhi NG</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Eriksson JG</span>, <span class="author">Peltonen L</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Balkau B</span>, <span class="author">Palmer CN</span>, <span class="author">Lyssenko V</span>, <span class="author">Tuomi T</span>, <span class="author">Isomaa B</span>, <span class="author">Hunter DJ</span>, <span class="author">Qi L</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Meta-Analyses of Glucose and Insulin-related traits Consortium (MAGIC) Investigators</span>, <span class="author">Genetic Investigation of ANthropometric Traits (GIANT) Consortium</span>, <span class="author">Asian Genetic Epidemiology Network&ndash;Type 2 Diabetes (AGEN-T2D) Consortium</span>, <span class="author">South Asian Type 2 Diabetes (SAT2D) Consortium</span>, <span class="author">Shuldiner AR</span>, <span class="author">Roden M</span>, <span class="author">Barroso I</span>, <span class="author">Wilsgaard T</span>, <span class="author">Beilby J</span>, <span class="author">Hovingh K</span>, <span class="author">Price JF</span>, <span class="author">Wilson JF</span>, <span class="author">Rauramaa R</span>, <span class="author">Lakka TA</span>, <span class="author">Lind L</span>, <span class="author">Dedoussis G</span>, <span class="author">Nj&oslash;lstad I</span>, <span class="author">Pedersen NL</span>, <span class="author">Khaw KT</span>, <span class="author">Wareham NJ</span>, <span class="author">Keinanen-Kiukaanniemi SM</span>, <span class="author">Saaristo TE</span>, <span class="author">Korpi-Hy&ouml;v&auml;lti E</span>, <span class="author">Saltevo J</span>, <span class="author">Laakso M</span>, <span class="author">Kuusisto J</span>, <span class="author">Metspalu A</span>, <span class="author">Collins FS</span>, <span class="author">Mohlke KL</span>, <span class="author">Bergman RN</span>, <span class="author">Tuomilehto J</span>, <span class="author">Boehm BO</span>, <span class="author">Gieger C</span>, <span class="author">Hveem K</span>, <span class="author">Cauchi S</span>, <span class="author">Froguel P</span>, <span class="author">Baldassarre D</span>, <span class="author">Tremoli E</span>, <span class="author">Humphries SE</span>, <span class="author">Saleheen D</span>, <span class="author">Danesh J</span>, <span class="author">Ingelsson E</span>, <span class="author">Ripatti S</span>, <span class="author">Salomaa V</span>, <span class="author">Erbel R</span>, <span class="author">J&ouml;ckel KH</span>, <span class="author">Moebus S</span>, <span class="author">Peters A</span>, <span class="author">Illig T</span>, <span class="author">de Faire U</span>, <span class="author">Hamsten A</span>, <span class="author">Morris AD</span>, <span class="author">Donnelly PJ</span>, <span class="author">Frayling TM</span>, <span class="author">Hattersley AT</span>, <span class="author">Boerwinkle E</span>, <span class="author">Melander O</span>, <span class="author">Kathiresan S</span>, <span class="author">Nilsson PM</span>, <span class="author">Deloukas P</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Groop LC</span>, <span class="author">Stefansson K</span>, <span class="author">Hu F</span>, <span class="author">Pankow JS</span>, <span class="author">Dupuis J</span>, <span class="author">Meigs JB</span>, <span class="author">Altshuler D</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span> and <span class="author">DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, UK. <a href="mailto:%61%6D%6F%72%72%69%73%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#97;&#109;&#111;&#114;&#114;&#105;&#115;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">To extend understanding of the genetic architecture and molecular basis of type 2 diabetes (T2D), we conducted a meta-analysis of genetic variants on the Metabochip, including 34,840 cases and 114,981 controls, overwhelmingly of European descent. We identified ten previously unreported T2D susceptibility loci, including two showing sex-differentiated association. Genome-wide analyses of these data are consistent with a long tail of additional common variant loci explaining much of the variation in susceptibility to T2D. Exploration of the enlarged set of susceptibility loci implicates several processes, including CREBBP-related transcription, adipocytokine signaling and cell cycle regulation, in diabetes pathogenesis.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/07/008/23674</span>, <span class="grant">RG/98002</span>, <span class="grant">RG2008/08</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/672</span>, <span class="grant">CZB/4/710</span>; Department of Health: <span class="grant">DHCS/07/07/008</span>; Medical Research Council: <span class="grant">G0000649</span>, <span class="grant">G0100222</span>, <span class="grant">G0401527</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_UP_A100_1003</span>; NCI NIH HHS: <span class="grant">CA055075</span>; NCRR NIH HHS: <span class="grant">UL1 RR029887</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">1Z01HG000024</span>, <span class="grant">N01HG65403</span>, <span class="grant">U01HG004399</span>, <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">N01HC25195</span>, <span class="grant">N02HL64278</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>; NIA NIH HHS: <span class="grant">AG028555</span>, <span class="grant">AG04563</span>, <span class="grant">AG08724</span>, <span class="grant">AG08861</span>, <span class="grant">AG10175</span>; NIDDK NIH HHS: <span class="grant">DK058845</span>, <span class="grant">DK062370</span>, <span class="grant">DK072193</span>, <span class="grant">DK073490</span>, <span class="grant">DK078616</span>, <span class="grant">DK080140</span>, <span class="grant">K24 DK080140</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK073490</span>, <span class="grant">U01 DK085545</span>; NIGMS NIH HHS: <span class="grant">T32 GM007753</span>; NINDS NIH HHS: <span class="grant">1R21NS064908</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268201100005C</span>, <span class="grant">HHSN268201100006C</span>, <span class="grant">HHSN268201100007C</span>, <span class="grant">HHSN268201100008C</span>, <span class="grant">HHSN268201100009C</span>, <span class="grant">HHSN268201100010C</span>, <span class="grant">HHSN268201100011C</span>, <span class="grant">HHSN268201100012C</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">081682</span>, <span class="grant">090367</span>, <span class="grant">090532</span>, <span class="grant">098017</span>, <span class="grant">098395</span>, <span class="grant">GR072960</span>, <span class="grant">GR076113</span>, <span class="grant">GR077016</span>, <span class="grant">GR081682</span>, <span class="grant">GR083270</span>, <span class="grant">GR083948</span>, <span class="grant">GR084711</span>, <span class="grant">GR086596</span>, <span class="grant">GR090532</span>, <span class="grant">GR098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">9</span>;<span class="pagerange">981-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22885922">22885922</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3442244">3442244</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2383">10.1038/ng.2383</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Olorin: combining gene flow with exome sequencing in large family studies of complex disease.</h4>
<p class="authors"><span class="author">Morris JA</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1HH, UK. <a href="mailto:%6F%6C%6F%72%69%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#111;&#108;&#111;&#114;&#105;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: The existence of families with many individuals affected by the same complex disease has long suggested the possibility of rare alleles of high penetrance. In contrast to Mendelian diseases, however, linkage studies have identified very few reproducibly linked loci in diseases such as diabetes and autism. Genome-wide association studies have had greater success with such diseases, but these results explain neither the extreme disease load nor the within-family linkage peaks, of some large pedigrees. Combining linkage information with exome or genome sequencing from large complex disease pedigrees might finally identify family-specific, high-penetrance mutations.</p><p class="abstract"><strong>Results</strong>: Olorin is a tool, which integrates gene flow within families with next generation sequencing data to enable the analysis of complex disease pedigrees. Users can interactively filter and prioritize variants based on haplotype sharing across selected individuals and other measures of importance, including predicted functional consequence and population frequency.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">24</span>;<span class="pagerange">3320-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23052039">23052039</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3519455">3519455</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts609">10.1093/bioinformatics/bts609</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of multipotent lung and airway progenitors from mouse ESCs and patient-specific cystic fibrosis iPSCs.</h4>
<p class="authors"><span class="author">Mou H</span>, <span class="author">Zhao R</span>, <span class="author">Sherwood R</span>, <span class="author">Ahfeldt T</span>, <span class="author">Lapey A</span>, <span class="author">Wain J</span>, <span class="author">Sicilian L</span>, <span class="author">Izvolsky K</span>, <span class="author">Musunuru K</span>, <span class="author">Cowan C</span> and <span class="author">Rajagopal J</span></p>
<p class="affiliation">Center for Regenerative Medicine, Massachusetts General Hospital, Boston, 02114, USA.</p><p class="abstract">Deriving lung progenitors from patient-specific pluripotent cells is a key step in producing differentiated lung epithelium for disease modeling and transplantation. By mimicking the signaling events that occur during mouse lung development, we generated murine lung progenitors in a series of discrete steps. Definitive endoderm derived from mouse embryonic stem cells (ESCs) was converted into foregut endoderm, then into replicating Nkx2.1+ lung endoderm, and finally into multipotent embryonic lung progenitor and airway progenitor cells. We demonstrated that precisely-timed BMP, FGF, and WNT signaling are required for NKX2.1 induction. Mouse ESC-derived Nkx2.1+ progenitor cells formed respiratory epithelium (tracheospheres) when transplanted subcutaneously into mice. We then adapted this strategy to produce disease-specific lung progenitor cells from human Cystic Fibrosis induced pluripotent stem cells (iPSCs), creating a platform for dissecting human lung disease. These disease-specific human lung progenitors formed respiratory epithelium when subcutaneously engrafted into immunodeficient mice.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">P30 HL101287</span>, <span class="grant">R21 HL108055</span>, <span class="grant">R21 HL109786</span></p>
<p class="publication"><span class="title">Cell stem cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">4</span>;<span class="pagerange">385-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22482504">22482504</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3474327">3474327</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.stem.2012.01.018">10.1016/j.stem.2012.01.018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genotypic homogeneity of multidrug resistant s. Typhimurium infecting distinct adult and childhood susceptibility groups in Blantyre, Malawi.</h4>
<p class="authors"><span class="author">Msefula CL</span>, <span class="author">Kingsley RA</span>, <span class="author">Gordon MA</span>, <span class="author">Molyneux E</span>, <span class="author">Molyneux ME</span>, <span class="author">Maclennan CA</span>, <span class="author">Dougan G</span> and <span class="author">Heyderman RS</span></p>
<p class="affiliation">Department of Microbiology, College of Medicine, University of Malawi, Blantyre, Malawi.</p>
<p class="abstract">Nontyphoidal Salmonella (NTS) serovars are a common cause of bacteraemia in young children and HIV-infected adults in Malawi and elsewhere in sub-Saharan Africa. These patient populations provide diverse host-immune environments that have the potential to drive bacterial adaptation and evolution. We therefore investigated the diversity of 27 multidrug resistant (MDR) Salmonella Typhimurium strains isolated over 6 years (2002-2008) from HIV-infected adults and children and HIV-uninfected children. Sequence reads from whole-genome sequencing of these isolates using the Illumina GA platform were mapped to the genome of the laboratory strain S. Typhimurium SL1344 excluding homoplastic regions that contained prophage and insertion elements. A phylogenetic tree generated from single nucleotide polymorphisms showed that all 27 strains clustered with the prototypical MDR strain D23580. There was no clustering of strains based on host HIV status or age, suggesting that these susceptible populations acquire S. Typhimurium from common sources or that isolates are transmitted freely between these populations. However, 7/14 of the most recent isolates (2006/2008) formed a distinct clade that branched off 22 SNPs away from the cluster containing earlier isolates. These data suggest that the MDR bacterial population is not static, but is undergoing microevolution which might result in further epidemiology change.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e42085</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22848711" rel="external">22848711</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3407126" rel="external">3407126</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0042085" rel="external">10.1371/journal.pone.0042085</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic Pathology of SLE-Associated Copy-Number Variation at the FCGR2C/FCGR3B/FCGR2B Locus.</h4>
<p class="authors"><span class="author">Mueller M</span>, <span class="author">Barros P</span>, <span class="author">Witherden AS</span>, <span class="author">Roberts AL</span>, <span class="author">Zhang Z</span>, <span class="author">Schaschl H</span>, <span class="author">Yu CY</span>, <span class="author">Hurles ME</span>, <span class="author">Schaffner C</span>, <span class="author">Floto RA</span>, <span class="author">Game L</span>, <span class="author">Steinberg KM</span>, <span class="author">Wilson RK</span>, <span class="author">Graves TA</span>, <span class="author">Eichler EE</span>, <span class="author">Cook HT</span>, <span class="author">Vyse TJ</span> and <span class="author">Aitman TJ</span></p>
<p class="affiliation">Physiological Genomics and Medicine Group, MRC Clinical Sciences Centre, Faculty of Medicine, Imperial College London, London W12 0NN, UK.</p>
<p class="abstract">Reduced FCGR3B copy number is associated with increased risk of systemic lupus erythematosus (SLE). The five FCGR2/FCGR3 genes are arranged across two highly paralogous genomic segments on chromosome 1q23. Previous studies have suggested mechanisms for structural rearrangements at the FCGR2/FCGR3 locus and have proposed mechanisms whereby altered FCGR3B copy number predisposes to autoimmunity, but the high degree of sequence similarity between paralogous segments has prevented precise definition of the molecular events and their functional consequences. To pursue the genomic pathology associated with FCGR3B copy-number variation, we integrated sequencing data from fosmid and bacterial artificial chromosome clones and sequence-captured DNA from FCGR3B-deleted genomes to establish a detailed map of allelic and paralogous sequence variation across the FCGR2/FCGR3 locus. This analysis identified two highly paralogous 24.5 kb blocks within the FCGR2C/FCGR3B/FCGR2B locus that are devoid of nonpolymorphic paralogous sequence variations and that define the limits of the genomic regions in which nonallelic homologous recombination leads to FCGR2C/FCGR3B copy-number variation. Further, the data showed evidence of swapping of haplotype blocks between these highly paralogous blocks that most likely arose from sequential ancestral recombination events across the region. Functionally, we found by flow cytometry, immunoblotting and cDNA sequencing that individuals with FCGR3B-deleted alleles show ectopic presence of FcÎ³RIIb on natural killer (NK) cells. We conclude that FCGR3B deletion juxtaposes the 5'-regulatory sequences of FCGR2C with the coding sequence of FCGR2B, creating a chimeric gene that results in an ectopic accumulation of FcÎ³RIIb on NK cells and provides an explanation for SLE risk associated with reduced FCGR3B gene copy number.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23261299" rel="external">23261299</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.11.013" rel="external">10.1016/j.ajhg.2012.11.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Behavior and target site selection of conjugative transposon Tn916 in two different strains of toxigenic Clostridium difficile.</h4>
<p class="authors"><span class="author">Mullany P</span>, <span class="author">Williams R</span>, <span class="author">Langridge GC</span>, <span class="author">Turner DJ</span>, <span class="author">Whalan R</span>, <span class="author">Clayton C</span>, <span class="author">Lawley T</span>, <span class="author">Hussain H</span>, <span class="author">McCurrie K</span>, <span class="author">Morden N</span>, <span class="author">Allan E</span> and <span class="author">Roberts AP</span></p>
<p class="affiliation">Department of Microbial Diseases, UCL Eastman Dental Institute, University College London, London, UK. <a href="mailto:%70.%6D%75%6C%6C%61%6E%79%40%75%63%6C.%61%63.%75%6B">&#112;&#46;&#109;&#117;&#108;&#108;&#97;&#110;&#121;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The insertion sites of the conjugative transposon Tn916 in the anaerobic pathogen Clostridium difficile were determined using Illumina Solexa high-throughput DNA sequencing of Tn916 insertion libraries in two different clinical isolates: 630&Delta;E, an erythromycin-sensitive derivative of 630 (ribotype 012), and the ribotype 027 isolate R20291, which was responsible for a severe outbreak of C. difficile disease. A consensus 15-bp Tn916 insertion sequence was identified which was similar in both strains, although an extended consensus sequence was observed in R20291. A search of the C. difficile 630 genome showed that the Tn916 insertion motif was present 100,987 times, with approximately 63,000 of these motifs located in genes and 35,000 in intergenic regions. To test the usefulness of Tn916 as a mutagen, a functional screen allowed the isolation of a mutant. This mutant contained Tn916 inserted into a gene involved in flagellar biosynthesis.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0601176</span></p>
<p class="publication"><span class="title">Applied and environmental microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">78</span>;<span class="number">7</span>;<span class="pagerange">2147-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22267673">22267673</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3302608">3302608</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAEM.06193-11">10.1128/AEM.06193-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequencing and analysis of the Tasmanian devil and its transmissible cancer.</h4>
<p class="authors"><span class="author">Murchison EP</span>, <span class="author">Schulz-Trieglaff OB</span>, <span class="author">Ning Z</span>, <span class="author">Alexandrov LB</span>, <span class="author">Bauer MJ</span>, <span class="author">Fu B</span>, <span class="author">Hims M</span>, <span class="author">Ding Z</span>, <span class="author">Ivakhno S</span>, <span class="author">Stewart C</span>, <span class="author">Ng BL</span>, <span class="author">Wong W</span>, <span class="author">Aken B</span>, <span class="author">White S</span>, <span class="author">Alsop A</span>, <span class="author">Becq J</span>, <span class="author">Bignell GR</span>, <span class="author">Cheetham RK</span>, <span class="author">Cheng W</span>, <span class="author">Connor TR</span>, <span class="author">Cox AJ</span>, <span class="author">Feng ZP</span>, <span class="author">Gu Y</span>, <span class="author">Grocock RJ</span>, <span class="author">Harris SR</span>, <span class="author">Khrebtukova I</span>, <span class="author">Kingsbury Z</span>, <span class="author">Kowarsky M</span>, <span class="author">Kreiss A</span>, <span class="author">Luo S</span>, <span class="author">Marshall J</span>, <span class="author">McBride DJ</span>, <span class="author">Murray L</span>, <span class="author">Pearse AM</span>, <span class="author">Raine K</span>, <span class="author">Rasolonjatovo I</span>, <span class="author">Shaw R</span>, <span class="author">Tedder P</span>, <span class="author">Tregidgo C</span>, <span class="author">Vilella AJ</span>, <span class="author">Wedge DC</span>, <span class="author">Woods GM</span>, <span class="author">Gormley N</span>, <span class="author">Humphray S</span>, <span class="author">Schroth G</span>, <span class="author">Smith G</span>, <span class="author">Hall K</span>, <span class="author">Searle SM</span>, <span class="author">Carter NP</span>, <span class="author">Papenfuss AT</span>, <span class="author">Futreal PA</span>, <span class="author">Campbell PJ</span>, <span class="author">Yang F</span>, <span class="author">Bentley DR</span>, <span class="author">Evers DJ</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK. <a href="mailto:%65%6C%69%7A%61%62%65%74%68.%6D%75%72%63%68%69%73%6F%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#108;&#105;&#122;&#97;&#98;&#101;&#116;&#104;&#46;&#109;&#117;&#114;&#99;&#104;&#105;&#115;&#111;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Tasmanian devil (Sarcophilus harrisii), the largest marsupial carnivore, is endangered due to a transmissible facial cancer spread by direct transfer of living cancer cells through biting. Here we describe the sequencing, assembly, and annotation of the Tasmanian devil genome and whole-genome sequences for two geographically distant subclones of the cancer. Genomic analysis suggests that the cancer first arose from a female Tasmanian devil and that the clone has subsequently genetically diverged during its spread across Tasmania. The devil cancer genome contains more than 17,000 somatic base substitution mutations and bears the imprint of a distinct mutational process. Genotyping of somatic mutations in 104 geographically and temporally distributed Tasmanian devil tumors reveals the pattern of evolution and spread of this parasitic clonal lineage, with evidence of a selective sweep in one geographical area and persistence of parallel lineages in other populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">095908</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">148</span>;<span class="number">4</span>;<span class="pagerange">780-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22341448">22341448</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3281993">3281993</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2011.11.065">10.1016/j.cell.2011.11.065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of an Eurasian avian-like influenza virus in na&iuml;ve and vaccinated pigs.</h4>
<p class="authors"><span class="author">Murcia PR</span>, <span class="author">Hughes J</span>, <span class="author">Battista P</span>, <span class="author">Lloyd L</span>, <span class="author">Baillie GJ</span>, <span class="author">Ramirez-Gonzalez RH</span>, <span class="author">Ormond D</span>, <span class="author">Oliver K</span>, <span class="author">Elton D</span>, <span class="author">Mumford JA</span>, <span class="author">Caccamo M</span>, <span class="author">Kellam P</span>, <span class="author">Grenfell BT</span>, <span class="author">Holmes EC</span> and <span class="author">Wood JL</span></p>
<p class="affiliation">Cambridge Infectious Diseases Consortium, Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Influenza viruses are characterized by an ability to cross species boundaries and evade host immunity, sometimes with devastating consequences. The 2009 pandemic of H1N1 influenza A virus highlights the importance of pigs in influenza emergence, particularly as intermediate hosts by which avian viruses adapt to mammals before emerging in humans. Although segment reassortment has commonly been associated with influenza emergence, an expanded host-range is also likely to be associated with the accumulation of specific beneficial point mutations. To better understand the mechanisms that shape the genetic diversity of avian-like viruses in pigs, we studied the evolutionary dynamics of an Eurasian Avian-like swine influenza virus (EA-SIV) in na&iuml;ve and vaccinated pigs linked by natural transmission. We analyzed multiple clones of the hemagglutinin 1 (HA1) gene derived from consecutive daily viral populations. Strikingly, we observed both transient and fixed changes in the consensus sequence along the transmission chain. Hence, the mutational spectrum of intra-host EA-SIV populations is highly dynamic and allele fixation can occur with extreme rapidity. In addition, mutations that could potentially alter host-range and antigenicity were transmitted between animals and mixed infections were commonplace, even in vaccinated pigs. Finally, we repeatedly detected distinct stop codons in virus samples from co-housed pigs, suggesting that they persisted within hosts and were transmitted among them. This implies that mutations that reduce viral fitness in one host, but which could lead to fitness benefits in a novel host, can circulate at low frequencies.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801822</span>; NICHD NIH HHS: <span class="grant">R24 HD047879</span>; NIGMS NIH HHS: <span class="grant">R01 GM080533-05</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e1002730</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22693449">22693449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3364949">3364949</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002730">10.1371/journal.ppat.1002730</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An atypical facial appearance and growth pattern in a child with Cornelia de Lange Syndrome: an intragenic deletion predicting loss of the N-terminal region of NIPBL.</h4>
<p class="authors"><span class="author">Murray JE</span>, <span class="author">Walayat M</span>, <span class="author">Gillett P</span>, <span class="author">Sharkey FH</span>, <span class="author">Rajan D</span>, <span class="author">Carter NP</span> and <span class="author">FitzPatrick DR</span></p>
<p class="affiliation">South-east Scotland Clinical Genetics Services Western General Hospital, Edinburgh, UK. <a href="mailto:%4A%65%6E%6E%69%65.%6D%75%72%72%61%79%40%6C%75%68%74.%73%63%6F%74.%6E%68%73.%75%6B">&#74;&#101;&#110;&#110;&#105;&#101;&#46;&#109;&#117;&#114;&#114;&#97;&#121;&#64;&#108;&#117;&#104;&#116;&#46;&#115;&#99;&#111;&#116;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract">Cornelia de Lange Syndrome (CdLS) is a multisystem disorder with a live birth prevalence of approximately one per 15 000. Clinical diagnosis is based on a characteristic facies &ndash; low frontal hair line, short nose, triangular nasal tip, crescent shaped mouth, upturned nose, and arched eyebrows &ndash; characteristic limb defects and a distinctive pattern of growth and development. Approximately half of all classical cases of CdLS have heterozygous loss of-function mutations in the gene encoding NIPBL, a component of the cohesion-loading apparatus (Dorsett and Krantz, 2009). Herein we describe a patient with a rare intragenic deletion of NIPBL who has typical microcephaly and developmental problems but atypical growth pattern and facial features.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_PC_U127561093</span>, <span class="grant">MC_U127561093</span>; Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Clinical dysmorphology</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">1</span>;<span class="pagerange">22-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21934607">21934607</a>; DOI: <a href="http://dx.doi.org/10.1097%2FMCD.0b013e32834c4afc">10.1097/MCD.0b013e32834c4afc</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beyond knockouts: cre resources for conditional mutagenesis.</h4>
<p class="authors"><span class="author">Murray SA</span>, <span class="author">Eppig JT</span>, <span class="author">Smedley D</span>, <span class="author">Simpson EM</span> and <span class="author">Rosenthal N</span></p>
<p class="affiliation">The Jackson Laboratory, 600 Main Street, Bar Harbor, ME, 04609, USA, <a href="mailto:%73%74%65%76%65.%6D%75%72%72%61%79%40%6A%61%78.%6F%72%67">&#115;&#116;&#101;&#118;&#101;&#46;&#109;&#117;&#114;&#114;&#97;&#121;&#64;&#106;&#97;&#120;&#46;&#111;&#114;&#103;</a>.</p>
<p class="abstract">With the effort of the International Phenotyping Consortium to produce thousands of strains with conditional potential gathering steam, there is growing recognition that it must be supported by a rich toolbox of cre driver strains. The approaches to build cre strains have evolved in both sophistication and reliability, replacing first-generation strains with tools that can target individual cell populations with incredible precision and specificity. The modest set of cre drivers generated by individual labs over the past 15+ years is now growing rapidly, thanks to a number of large-scale projects to produce new cre strains for the community. The power of this growing resource, however, depends upon the proper deep characterization of strain function, as even the best designed strain can display a variety of undesirable features that must be considered in experimental design. This must be coupled with the parallel development of informatics tools to provide functional data to the user and facilitated access to the strains through public repositories. We discuss the current progress on all of these fronts and the challenges that remain to ensure the scientific community can capitalize on the tremendous number of mouse resources at their disposal.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22926223" rel="external">22926223</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9430-2" rel="external">10.1007/s00335-012-9430-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacterial frequent flyers.</h4>
<p class="authors"><span class="author">Mutreja A</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">This month's Genome Watch describes how sequencing technology is providing insight into the geographical relationships and global travel of bacterial pathogens.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">11</span>;<span class="pagerange">734</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23070555" rel="external">23070555</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2899" rel="external">10.1038/nrmicro2899</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome wide adaptations of Plasmodium falciparum in response to lumefantrine selective drug pressure.</h4>
<p class="authors"><span class="author">Mwai L</span>, <span class="author">Diriye A</span>, <span class="author">Masseno V</span>, <span class="author">Muriithi S</span>, <span class="author">Feltwell T</span>, <span class="author">Musyoki J</span>, <span class="author">Lemieux J</span>, <span class="author">Feller A</span>, <span class="author">Mair GR</span>, <span class="author">Marsh K</span>, <span class="author">Newbold C</span>, <span class="author">Nzila A</span> and <span class="author">Carret CK</span></p>
<p class="affiliation">Kenya Medical Research Institute, Welcome Trust Research Programme, Kilifi, Kenya.</p><p class="abstract">The combination therapy of the Artemisinin-derivative Artemether (ART) with Lumefantrine (LM) (Coartem&reg;) is an important malaria treatment regimen in many endemic countries. Resistance to Artemisinin has already been reported, and it is feared that LM resistance (LMR) could also evolve quickly. Therefore molecular markers which can be used to track Coartem&reg; efficacy are urgently needed. Often, stable resistance arises from initial, unstable phenotypes that can be identified in vitro. Here we have used the Plasmodium falciparum multidrug resistant reference strain V1S to induce LMR in vitro by culturing the parasite under continuous drug pressure for 16 months. The initial IC(50) (inhibitory concentration that kills 50% of the parasite population) was 24 nM. The resulting resistant strain V1S(LM), obtained after culture for an estimated 166 cycles under LM pressure, grew steadily in 378 nM of LM, corresponding to 15 times the IC(50) of the parental strain. However, after two weeks of culturing V1S(LM) in drug-free medium, the IC(50) returned to that of the initial, parental strain V1S. This transient drug tolerance was associated with major changes in gene expression profiles: using the PFSANGER Affymetrix custom array, we identified 184 differentially expressed genes in V1S(LM). Among those are 18 known and putative transporters including the multidrug resistance gene 1 (pfmdr1), the multidrug resistance associated protein and the V-type H+ pumping pyrophosphatase 2 (pfvp2) as well as genes associated with fatty acid metabolism. In addition we detected a clear selective advantage provided by two genomic loci in parasites grown under LM drug pressure, suggesting that all, or some of those genes contribute to development of LM tolerance--they may prove useful as molecular markers to monitor P. falciparum LM susceptibility.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e31623</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22384044" rel="external">22384044</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3288012" rel="external">3288012</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0031623" rel="external">10.1371/journal.pone.0031623</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A GWAS sequence variant for platelet volume marks an alternative DNM3 promoter in megakaryocytes near a MEIS1 binding site.</h4>
<p class="authors"><span class="author">N&uuml;rnberg ST</span>, <span class="author">Rendon A</span>, <span class="author">Smethurst PA</span>, <span class="author">Paul DS</span>, <span class="author">Voss K</span>, <span class="author">Thon JN</span>, <span class="author">Lloyd-Jones H</span>, <span class="author">Sambrook JG</span>, <span class="author">Tijssen MR</span>, <span class="author">HaemGen Consortium</span>, <span class="author">Italiano JE</span>, <span class="author">Deloukas P</span>, <span class="author">Gottgens B</span>, <span class="author">Soranzo N</span> and <span class="author">Ouwehand WH</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge and National Health Service Blood and Transplant, Cambridge, United Kingdom.</p><p class="abstract">We recently identified 68 genomic loci where common sequence variants are associated with platelet count and volume. Platelets are formed in the bone marrow by megakaryocytes, which are derived from hematopoietic stem cells by a process mainly controlled by transcription factors. The homeobox transcription factor MEIS1 is uniquely transcribed in megakaryocytes and not in the other lineage-committed blood cells. By ChIP-seq, we show that 5 of the 68 loci pinpoint a MEIS1 binding event within a group of 252 MK-overexpressed genes. In one such locus in DNM3, regulating platelet volume, the MEIS1 binding site falls within a region acting as an alternative promoter that is solely used in megakaryocytes, where allelic variation dictates different levels of a shorter transcript. The importance of dynamin activity to the latter stages of thrombopoiesis was confirmed by the observation that the inhibitor Dynasore reduced murine proplatelet for-mation in vitro.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/12/28096</span>; Cancer Research UK: <span class="grant">14136</span>; Medical Research Council: <span class="grant">G0800784</span>, <span class="grant">G1000143</span>; NHLBI NIH HHS: <span class="grant">HL68130</span>; Wellcome Trust: <span class="grant">WT-084183/2/07/2</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2012</span>;<span class="volume">120</span>;<span class="number">24</span>;<span class="pagerange">4859-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22972982">22972982</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3520622">3520622</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2012-01-401893">10.1182/blood-2012-01-401893</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic islands of divergence in hybridizing Heliconius butterflies identified by large-scale targeted sequencing.</h4>
<p class="authors"><span class="author">Nadeau NJ</span>, <span class="author">Whibley A</span>, <span class="author">Jones RT</span>, <span class="author">Davey JW</span>, <span class="author">Dasmahapatra KK</span>, <span class="author">Baxter SW</span>, <span class="author">Quail MA</span>, <span class="author">Joron M</span>, <span class="author">ffrench-Constant RH</span>, <span class="author">Blaxter ML</span>, <span class="author">Mallet J</span> and <span class="author">Jiggins CD</span></p>
<p class="affiliation">Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, UK. <a href="mailto:%6E%6A%6E%32%37%40%63%61%6D.%61%63.%75%6B">&#110;&#106;&#110;&#50;&#55;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Heliconius butterflies represent a recent radiation of species, in which wing pattern divergence has been implicated in speciation. Several loci that control wing pattern phenotypes have been mapped and two were identified through sequencing. These same gene regions play a role in adaptation across the whole Heliconius radiation. Previous studies of population genetic patterns at these regions have sequenced small amplicons. Here, we use targeted next-generation sequence capture to survey patterns of divergence across these entire regions in divergent geographical races and species of Heliconius. This technique was successful both within and between species for obtaining high coverage of almost all coding regions and sufficient coverage of non-coding regions to perform population genetic analyses. We find major peaks of elevated population differentiation between races across hybrid zones, which indicate regions under strong divergent selection. These &#39;islands&#39; of divergence appear to be more extensive between closely related species, but there is less clear evidence for such islands between more distantly related species at two further points along the &#39;speciation continuum&#39;. We also sequence fosmid clones across these regions in different Heliconius melpomene races. We find no major structural rearrangements but many relatively large (greater than 1 kb) insertion/deletion events (including gain/loss of transposable elements) that are variable between races.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council</p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2012</span>;<span class="volume">367</span>;<span class="number">1587</span>;<span class="pagerange">343-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22201164" rel="external">22201164</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3233711" rel="external">3233711</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2011.0198" rel="external">10.1098/rstb.2011.0198</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Zebrafish breeding in the laboratory environment.</h4>
<p class="authors"><span class="author">Nasiadka A</span> and <span class="author">Clark MD</span></p>
<p class="affiliation">Zebrafish International Resource Center, University of Oregon, Eugene, OR 97403, USA. <a href="mailto:%61%6E%64%72%7A%65%6A%40%7A%65%62%72%61%66%69%73%68.%6F%72%67">&#97;&#110;&#100;&#114;&#122;&#101;&#106;&#64;&#122;&#101;&#98;&#114;&#97;&#102;&#105;&#115;&#104;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The zebrafish, Danio rerio, has become a major model organism used in biomedical studies. The widespread use of Danio rerio in research laboratories requires a comprehensive understanding of the husbandry of this species to ensure efficient propagation and maintenance of healthy and genetically diverse colonies. Breeding is a key element in zebrafish husbandry. It is a complex process influenced by a number of factors. Mate choice and mating behavior depend, for example, on olfactory cues, visual stimuli, and social interactions. Spawning is affected by the age and size of fish, interval at which fish are used for egg production, light cycle, diet, and fish health status. A number of breeding strategies, based on either single-pair matings or group crosses, are commonly employed in the laboratory to propagate lines and to identify carriers of specific mutations and/or transgenes. Propagation of zebrafish lines, in particular wild-type-derived strains, is closely monitored to ensure that genetic diversity and vigor are maintained. A robust zebrafish line typically carries a large number of polymorphic variations, which may interfere with reproducibility of experiments. To get a better insight into these variations, a wild-type hybrid Sanger AB T&uuml;bingen line has been generated from sequenced homozygous founders.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P40 RR012546</span>; Wellcome Trust</p>
<p class="publication"><span class="title">ILAR journal / National Research Council, Institute of Laboratory Animal Resources</span>&nbsp;<span class="year">2012</span>;<span class="volume">53</span>;<span class="number">2</span>;<span class="pagerange">161-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23382347">23382347</a>; DOI: <a href="http://dx.doi.org/10.1093%2Filar.53.2.161">10.1093/ilar.53.2.161</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genomic landscape shaped by selection on transposable elements across 18 mouse strains.</h4>
<p class="authors"><span class="author">Nell&aring;ker C</span>, <span class="author">Keane TM</span>, <span class="author">Yalcin B</span>, <span class="author">Wong K</span>, <span class="author">Agam A</span>, <span class="author">Belgard TG</span>, <span class="author">Flint J</span>, <span class="author">Adams DJ</span>, <span class="author">Frankel WN</span> and <span class="author">Ponting CP</span></p>
<p class="affiliation">MRC Functional Genomics Unit, Department of Physiology, Anatomy and Genetics, University of Oxford, Oxford, UK. <a href="mailto:%63%68%72%69%73%74%6F%66%66%65%72.%6E%65%6C%6C%61%6B%65%72%40%64%70%61%67.%6F%78.%61%63.%75%6B">&#99;&#104;&#114;&#105;&#115;&#116;&#111;&#102;&#102;&#101;&#114;&#46;&#110;&#101;&#108;&#108;&#97;&#107;&#101;&#114;&#64;&#100;&#112;&#97;&#103;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Transposable element (TE)-derived sequence dominates the landscape of mammalian genomes and can modulate gene function by dysregulating transcription and translation. Our current knowledge of TEs in laboratory mouse strains is limited primarily to those present in the C57BL/6J reference genome, with most mouse TEs being drawn from three distinct classes, namely short interspersed nuclear elements (SINEs), long interspersed nuclear elements (LINEs) and the endogenous retrovirus (ERV) superfamily. Despite their high prevalence, the different genomic and gene properties controlling whether TEs are preferentially purged from, or are retained by, genetic drift or positive selection in mammalian genomes remain poorly defined.</p><p class="abstract"><strong>Results</strong>: Using whole genome sequencing data from 13 classical laboratory and 4 wild-derived mouse inbred strains, we developed a comprehensive catalogue of 103,798 polymorphic TE variants. We employ this extensive data set to characterize TE variants across the Mus lineage, and to infer neutral and selective processes that have acted over 2 million years. Our results indicate that the majority of TE variants are introduced though the male germline and that only a minority of TE variants exert detectable changes in gene expression. However, among genes with differential expression across the strains there are twice as many TE variants identified as being putative causal variants as expected.</p><p class="abstract"><strong>Conclusions</strong>: Most TE variants that cause gene expression changes appear to be purged rapidly by purifying selection. Our findings demonstrate that past TE insertions have often been highly deleterious, and help to prioritize TE variants according to their likely contribution to gene expression or phenotype variation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0800024</span>, <span class="grant">MC_U137761446</span>; NINDS NIH HHS: <span class="grant">R01 NS031348</span>; Wellcome Trust: <span class="grant">079912</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">6</span>;<span class="pagerange">R45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22703977">22703977</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3446317">3446317</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-6-r45">10.1186/gb-2012-13-6-r45</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated functional genomics approach identifies the regulatory network directed by brachyury (T) in chordoma.</h4>
<p class="authors"><span class="author">Nelson AC</span>, <span class="author">Pillay N</span>, <span class="author">Henderson S</span>, <span class="author">Presneau N</span>, <span class="author">Tirabosco R</span>, <span class="author">Halai D</span>, <span class="author">Berisha F</span>, <span class="author">Flicek P</span>, <span class="author">Stemple DL</span>, <span class="author">Stern CD</span>, <span class="author">Wardle FC</span> and <span class="author">Flanagan AM</span></p>
<p class="affiliation">Randall Division of Cell and Molecular Biophysics, New Hunt&#39;s House, King&#39;s College London, Guy&#39;s Campus, London, SE1 1UL, UK.</p><p class="abstract">Chordoma is a rare malignant tumour of bone, the molecular marker of which is the expression of the transcription factor, brachyury. Having recently demonstrated that silencing brachyury induces growth arrest in a chordoma cell line, we now seek to identify its downstream target genes. Here we use an integrated functional genomics approach involving shRNA-mediated brachyury knockdown, gene expression microarray, ChIP-seq experiments, and bioinformatics analysis to achieve this goal. We confirm that the T-box binding motif of human brachyury is identical to that found in mouse, Xenopus, and zebrafish development, and that brachyury acts primarily as an activator of transcription. Using human chordoma samples for validation purposes, we show that brachyury binds 99 direct targets and indirectly influences the expression of 64 other genes, thereby acting as a master regulator of an elaborate oncogenic transcriptional network encompassing diverse signalling pathways including components of the cell cycle, and extracellular matrix components. Given the wide repertoire of its active binding and the relative specific localization of brachyury to the tumour cells, we propose that an RNA interference-based gene therapy approach is a plausible therapeutic avenue worthy of investigation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700095</span>, <span class="grant">G0700213</span></p>
<p class="publication"><span class="title">The Journal of pathology</span>&nbsp;<span class="year">2012</span>;<span class="volume">228</span>;<span class="number">3</span>;<span class="pagerange">274-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22847733">22847733</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpath.4082">10.1002/path.4082</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal rearrangements and karyotype evolution in carnivores revealed by chromosome painting.</h4>
<p class="authors"><span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Su W</span>, <span class="author">Wang D</span>, <span class="author">Tanomtong A</span>, <span class="author">Perelman PL</span>, <span class="author">Graphodatsky AS</span> and <span class="author">Yang F</span></p>
<p class="affiliation">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, the Chinese Academy of Sciences, Kunming, Yunnan, PR <a href="mailto:%43%68%69%6E%61.%77%68%6E%69%65%40%6D%61%69%6C.%6B%69%7A.%61%63.%63%6E">&#67;&#104;&#105;&#110;&#97;&#46;&#119;&#104;&#110;&#105;&#101;&#64;&#109;&#97;&#105;&#108;&#46;&#107;&#105;&#122;&#46;&#97;&#99;&#46;&#99;&#110;</a></p><p class="abstract">Chromosomal evolution in carnivores has been revisited extensively using cross-species chromosome painting. Painting probes derived from flow-sorted chromosomes of the domestic dog, which has one of the most rearranged karyotypes in mammals and the highest dipoid number (2n=78) in carnivores, are a powerful tool in detecting both evolutionary intra- and inter-chromosomal rearrangements. However, only a few comparative maps have been established between dog and other non-Canidae species. Here, we extended cross-species painting with dog probes to seven more species representing six carnivore families: Eurasian lynx (Lynx lynx), the stone marten (Martes foina), the small Indian civet (Viverricula indica), the Asian palm civet (Paradoxurus hermaphrodites), Javan mongoose (Hepestes javanicas), the raccoon (Procyon lotor) and the giant panda (Ailuropoda melanoleuca). The numbers and positions of intra-chromosomal rearrangements were found to differ among these carnivore species. A comparative map between human and stone marten, and a map among the Yangtze finless porpoise (Neophocaena phocaenoides asiaeorientalis), stone marten and human were also established to facilitate outgroup comparison and to integrate comparative maps between stone marten and other carnivores with such maps between human and other species. These comparative maps give further insight into genome evolution and karyotype phylogenetic relationships among carnivores, and will facilitate the transfer of gene mapping data from human, domestic dog and cat to other species.</p>
<p class="publication"><span class="title">Heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">108</span>;<span class="number">1</span>;<span class="pagerange">17-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22086079" rel="external">22086079</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3238119" rel="external">3238119</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fhdy.2011.107" rel="external">10.1038/hdy.2011.107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of sphingosine-1-phosphate transporter Spns2 in immune system function.</h4>
<p class="authors"><span class="author">Nijnik A</span>, <span class="author">Clare S</span>, <span class="author">Hale C</span>, <span class="author">Chen J</span>, <span class="author">Raisen C</span>, <span class="author">Mottram L</span>, <span class="author">Lucas M</span>, <span class="author">Estabel J</span>, <span class="author">Ryder E</span>, <span class="author">Adissu H</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Adams NC</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">White JK</span>, <span class="author">Steel KP</span>, <span class="author">Dougan G</span> and <span class="author">Hancock RE</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%61%6E%61%73%74%61%73%69%79%61.%6E%79%7A%68%6E%79%6B%40%6D%63%67%69%6C%6C.%63%61">&#97;&#110;&#97;&#115;&#116;&#97;&#115;&#105;&#121;&#97;&#46;&#110;&#121;&#122;&#104;&#110;&#121;&#107;&#64;&#109;&#99;&#103;&#105;&#108;&#108;&#46;&#99;&#97;</a></p><p class="abstract">Sphingosine-1-phosphate (S1P) is lipid messenger involved in the regulation of embryonic development, immune system functions, and many other physiological processes. However, the mechanisms of S1P transport across cellular membranes remain poorly understood, with several ATP-binding cassette family members and the spinster 2 (Spns2) member of the major facilitator superfamily known to mediate S1P transport in cell culture. Spns2 was also shown to control S1P activities in zebrafish in vivo and to play a critical role in zebrafish cardiovascular development. However, the in vivo roles of Spns2 in mammals and its involvement in the different S1P-dependent physiological processes have not been investigated. In this study, we characterized Spns2-null mouse line carrying the Spns2(tm1a(KOMP)Wtsi) allele (Spns2(tm1a)). The Spns2(tm1a/tm1a) animals were viable, indicating a divergence in Spns2 function from its zebrafish ortholog. However, the immunological phenotype of the Spns2(tm1a/tm1a) mice closely mimicked the phenotypes of partial S1P deficiency and impaired S1P-dependent lymphocyte trafficking, with a depletion of lymphocytes in circulation, an increase in mature single-positive T cells in the thymus, and a selective reduction in mature B cells in the spleen and bone marrow. Spns2 activity in the nonhematopoietic cells was critical for normal lymphocyte development and localization. Overall, Spns2(tm1a/tm1a) resulted in impaired humoral immune responses to immunization. This study thus demonstrated a physiological role for Spns2 in mammalian immune system functions but not in cardiovascular development. Other components of the S1P signaling network are investigated as drug targets for immunosuppressive therapy, but the selective action of Spns2 may present an advantage in this regard.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Cancer Research UK: <span class="grant">13031</span>; Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2012</span>;<span class="volume">189</span>;<span class="number">1</span>;<span class="pagerange">102-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22664872">22664872</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3381845">3381845</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1200282">10.4049/jimmunol.1200282</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The critical role of histone H2A-deubiquitinase Mysm1 in hematopoiesis and lymphocyte differentiation.</h4>
<p class="authors"><span class="author">Nijnik A</span>, <span class="author">Clare S</span>, <span class="author">Hale C</span>, <span class="author">Raisen C</span>, <span class="author">McIntyre RE</span>, <span class="author">Yusa K</span>, <span class="author">Everitt AR</span>, <span class="author">Mottram L</span>, <span class="author">Podrini C</span>, <span class="author">Lucas M</span>, <span class="author">Estabel J</span>, <span class="author">Goulding D</span>, <span class="author">Sanger Institute Microarray Facility</span>, <span class="author">Sanger Mouse Genetics Project</span>, <span class="author">Adams N</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">White JK</span>, <span class="author">Adams DJ</span>, <span class="author">Hancock RE</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Genome Campus, The Wellcome Trust Sanger Institute, Cambridge, United Kingdom. <a href="mailto:%61%6E%61%73%74%61%73%69%79%61.%6E%79%7A%68%6E%79%6B%40%6D%63%67%69%6C%6C.%63%61">&#97;&#110;&#97;&#115;&#116;&#97;&#115;&#105;&#121;&#97;&#46;&#110;&#121;&#122;&#104;&#110;&#121;&#107;&#64;&#109;&#99;&#103;&#105;&#108;&#108;&#46;&#99;&#97;</a></p><p class="abstract">Stem cell differentiation and lineage specification depend on coordinated programs of gene expression, but our knowledge of the chromatin-modifying factors regulating these events remains incomplete. Ubiquitination of histone H2A (H2A-K119u) is a common chromatin modification associated with gene silencing, and controlled by the ubiquitin-ligase polycomb repressor complex 1 (PRC1) and H2A-deubiquitinating enzymes (H2A-DUBs). The roles of H2A-DUBs in mammalian development, stem cells, and hematopoiesis have not been addressed. Here we characterized an H2A-DUB targeted mouse line Mysm1(tm1a/tm1a) and demonstrated defects in BM hematopoiesis, resulting in lymphopenia, anemia, and thrombocytosis. Development of lymphocytes was impaired from the earliest stages of their differentiation, and there was also a depletion of erythroid cells and a defect in erythroid progenitor function. These phenotypes resulted from a cell-intrinsic requirement for Mysm1 in the BM. Importantly, Mysm1(tm1a/tm1a) HSCs were functionally impaired, and this was associated with elevated levels of reactive oxygen species, &gamma;H2AX DNA damage marker, and p53 protein in the hematopoietic progenitors. Overall, these data establish a role for Mysm1 in the maintenance of BM stem cell function, in the control of oxidative stress and genetic stability in hematopoietic progenitors, and in the development of lymphoid and erythroid lineages.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Wellcome Trust</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2012</span>;<span class="volume">119</span>;<span class="number">6</span>;<span class="pagerange">1370-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22184403">22184403</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2011-05-352666">10.1182/blood-2011-05-352666</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutational processes molding the genomes of 21 breast cancers.</h4>
<p class="authors"><span class="author">Nik-Zainal S</span>, <span class="author">Alexandrov LB</span>, <span class="author">Wedge DC</span>, <span class="author">Van Loo P</span>, <span class="author">Greenman CD</span>, <span class="author">Raine K</span>, <span class="author">Jones D</span>, <span class="author">Hinton J</span>, <span class="author">Marshall J</span>, <span class="author">Stebbings LA</span>, <span class="author">Menzies A</span>, <span class="author">Martin S</span>, <span class="author">Leung K</span>, <span class="author">Chen L</span>, <span class="author">Leroy C</span>, <span class="author">Ramakrishna M</span>, <span class="author">Rance R</span>, <span class="author">Lau KW</span>, <span class="author">Mudie LJ</span>, <span class="author">Varela I</span>, <span class="author">McBride DJ</span>, <span class="author">Bignell GR</span>, <span class="author">Cooke SL</span>, <span class="author">Shlien A</span>, <span class="author">Gamble J</span>, <span class="author">Whitmore I</span>, <span class="author">Maddison M</span>, <span class="author">Tarpey PS</span>, <span class="author">Davies HR</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Stephens PJ</span>, <span class="author">McLaren S</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">J&ouml;nsson G</span>, <span class="author">Garber JE</span>, <span class="author">Silver D</span>, <span class="author">Miron P</span>, <span class="author">Fatima A</span>, <span class="author">Boyault S</span>, <span class="author">Langer&oslash;d A</span>, <span class="author">Tutt A</span>, <span class="author">Martens JW</span>, <span class="author">Aparicio SA</span>, <span class="author">Borg &Aring;</span>, <span class="author">Salomon AV</span>, <span class="author">Thomas G</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">Richardson AL</span>, <span class="author">Neuberger MS</span>, <span class="author">Futreal PA</span>, <span class="author">Campbell PJ</span>, <span class="author">Stratton MR</span> and <span class="author">Breast Cancer Working Group of the International Cancer Genome Consortium</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">All cancers carry somatic mutations. The patterns of mutation in cancer genomes reflect the DNA damage and repair processes to which cancer cells and their precursors have been exposed. To explore these mechanisms further, we generated catalogs of somatic mutation from 21 breast cancers and applied mathematical methods to extract mutational signatures of the underlying processes. Multiple distinct single- and double-nucleotide substitution signatures were discernible. Cancers with BRCA1 or BRCA2 mutations exhibited a characteristic combination of substitution mutation signatures and a distinctive profile of deletions. Complex relationships between somatic mutation prevalence and transcription were detected. A remarkable phenomenon of localized hypermutation, termed &quot;kataegis,&quot; was observed. Regions of kataegis differed between cancers but usually colocalized with somatic rearrangements. Base substitutions in these regions were almost exclusively of cytosine at TpC dinucleotides. The mechanisms underlying most of these mutational signatures are unknown. However, a role for the APOBEC family of cytidine deaminases is proposed.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">MC_U105178806</span>; NCI NIH HHS: <span class="grant">CA089393</span>; Wellcome Trust: <span class="grant">088340</span>, <span class="grant">098051</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">149</span>;<span class="number">5</span>;<span class="pagerange">979-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22608084">22608084</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3414841">3414841</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2012.04.024">10.1016/j.cell.2012.04.024</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The life history of 21 breast cancers.</h4>
<p class="authors"><span class="author">Nik-Zainal S</span>, <span class="author">Van Loo P</span>, <span class="author">Wedge DC</span>, <span class="author">Alexandrov LB</span>, <span class="author">Greenman CD</span>, <span class="author">Lau KW</span>, <span class="author">Raine K</span>, <span class="author">Jones D</span>, <span class="author">Marshall J</span>, <span class="author">Ramakrishna M</span>, <span class="author">Shlien A</span>, <span class="author">Cooke SL</span>, <span class="author">Hinton J</span>, <span class="author">Menzies A</span>, <span class="author">Stebbings LA</span>, <span class="author">Leroy C</span>, <span class="author">Jia M</span>, <span class="author">Rance R</span>, <span class="author">Mudie LJ</span>, <span class="author">Gamble SJ</span>, <span class="author">Stephens PJ</span>, <span class="author">McLaren S</span>, <span class="author">Tarpey PS</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Davies HR</span>, <span class="author">Varela I</span>, <span class="author">McBride DJ</span>, <span class="author">Bignell GR</span>, <span class="author">Leung K</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">Martin S</span>, <span class="author">J&ouml;nsson G</span>, <span class="author">Mariani O</span>, <span class="author">Boyault S</span>, <span class="author">Miron P</span>, <span class="author">Fatima A</span>, <span class="author">Langer&oslash;d A</span>, <span class="author">Aparicio SA</span>, <span class="author">Tutt A</span>, <span class="author">Sieuwerts AM</span>, <span class="author">Borg &Aring;</span>, <span class="author">Thomas G</span>, <span class="author">Salomon AV</span>, <span class="author">Richardson AL</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Campbell PJ</span> and <span class="author">Breast Cancer Working Group of the International Cancer Genome Consortium</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Cancer evolves dynamically as clonal expansions supersede one another driven by shifting selective pressures, mutational processes, and disrupted cancer genes. These processes mark the genome, such that a cancer&#39;s life history is encrypted in the somatic mutations present. We developed algorithms to decipher this narrative and applied them to 21 breast cancers. Mutational processes evolve across a cancer&#39;s lifespan, with many emerging late but contributing extensive genetic variation. Subclonal diversification is prominent, and most mutations are found in just a fraction of tumor cells. Every tumor has a dominant subclonal lineage, representing more than 50% of tumor cells. Minimal expansion of these subclones occurs until many hundreds to thousands of mutations have accumulated, implying the existence of long-lived, quiescent cell lineages capable of substantial proliferation upon acquisition of enabling genomic changes. Expansion of the dominant subclone to an appreciable mass may therefore represent the final rate-limiting step in a breast cancer&#39;s development, triggering diagnosis.</p>
<p class="grants">Funded by: Department of Health; NCI NIH HHS: <span class="grant">CA089393</span>; Wellcome Trust: <span class="grant">088340</span>, <span class="grant">093867</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">149</span>;<span class="number">5</span>;<span class="pagerange">994-1007</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22608083">22608083</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428864">3428864</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2012.04.023">10.1016/j.cell.2012.04.023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The transporter ABCB7 is a mediator of the phenotype of acquired refractory anemia with ring sideroblasts.</h4>
<p class="authors"><span class="author">Nikpour M</span>, <span class="author">Scharenberg C</span>, <span class="author">Liu A</span>, <span class="author">Conte S</span>, <span class="author">Karimi M</span>, <span class="author">Mortera-Blanco T</span>, <span class="author">Giai V</span>, <span class="author">Fernandez-Mercado M</span>, <span class="author">Papaemmanuil E</span>, <span class="author">H&ouml;gstrand K</span>, <span class="author">Jansson M</span>, <span class="author">Vedin I</span>, <span class="author">Stephen Wainscoat J</span>, <span class="author">Campbell P</span>, <span class="author">Cazzola M</span>, <span class="author">Boultwood J</span>, <span class="author">Grandien A</span> and <span class="author">Hellstr&ouml;m-Lindberg E</span></p>
<p class="affiliation">Department of Medicine, Center for Hematology and Regenerative Medicine, Karolinska Institutet, Karolinska University Hospital Huddinge, Stockholm, Sweden.</p>
<p class="abstract">Refractory anemia with ring sideroblasts (RARS) is characterized by mitochondrial ferritin (FTMT) accumulation and markedly suppressed expression of the iron transporter ABCB7. To test the hypothesis that ABCB7 is a key mediator of ineffective erythropoiesis of RARS, we modulated its expression in hematopoietic cells. ABCB7 up and downregulation did not influence growth and survival of K562 cells. In normal bone marrow, ABCB7 downregulation reduced erythroid differentiation, growth and colony formation, and resulted in a gene expression pattern similar to that observed in intermediate RARS erythroblasts, and in the accumulation of FTMT. Importantly, forced ABCB7 expression restored erythroid colony growth and decreased FTMT expression level in RARS CD34+ marrow cells. Mutations in the SF3B1 gene, a core component of the RNA splicing machinery, were recently identified in a high proportion of patients with RARS and 11 of the 13 RARS patients in this study carried this mutation. Interestingly, ABCB7 exon usage differed between normal bone marrow and RARS, as well as within the RARS cohort. In addition, SF3B1 silencing resulted in downregulation of ABCB7 in K562 cells undergoing erythroid differentiation. Our findings support that ABCB7 is implicated in the phenotype of acquired RARS and suggest a relation between SF3B1 mutations and ABCB7 downregulation.Leukemia advance online publication, 20 November 2012; doi:10.1038/leu.2012.298.</p>
<p class="publication"><span class="title">Leukemia : official journal of the Leukemia Society of America, Leukemia Research Fund, U.K</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23070040" rel="external">23070040</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fleu.2012.298" rel="external">10.1038/leu.2012.298</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expansion of CORE-SINEs in the genome of the Tasmanian devil.</h4>
<p class="authors"><span class="author">Nilsson MA</span>, <span class="author">Janke A</span>, <span class="author">Murchison EP</span>, <span class="author">Ning Z</span> and <span class="author">Hallstr&ouml;m BM</span></p>
<p class="affiliation">LOEWE - Biodiversity and Climate Research Center, BiK-F, Senckenberganlage 25, Frankfurt am Main, D-60325, Germany. <a href="mailto:%6D%61%72%69%61.%6E%69%6C%73%73%6F%6E-%6A%61%6E%6B%65%40%73%65%6E%63%6B%65%6E%62%65%72%67.%64%65">&#109;&#97;&#114;&#105;&#97;&#46;&#110;&#105;&#108;&#115;&#115;&#111;&#110;&#45;&#106;&#97;&#110;&#107;&#101;&#64;&#115;&#101;&#110;&#99;&#107;&#101;&#110;&#98;&#101;&#114;&#103;&#46;&#100;&#101;</a>.</p>
<p class="abstract">Unlabelled: ABSTRACT:

Background: The genome of the carnivorous marsupial, the Tasmanian devil (Sarcophilus harrisii, Order: Dasyuromorphia), was sequenced in the hopes of finding a cure for or gaining a better understanding of the contagious devil facial tumor disease that is threatening the species' survival. To better understand the Tasmanian devil genome, we screened it for transposable elements and investigated the dynamics of short interspersed element (SINE) retroposons.

Results: The temporal history of Tasmanian devil SINEs, elucidated using a transposition in transposition analysis, indicates that WSINE1, a CORE-SINE present in around 200,000 copies, is the most recently active element. Moreover, we discovered a new subtype of WSINE1 (WSINE1b) that comprises at least 90% of all Tasmanian devil WSINE1s. The frequencies of WSINE1 subtypes differ in the genomes of two of the other Australian marsupial orders. A co-segregation analysis indicated that at least 66 subfamilies of WSINE1 evolved during the evolution of Dasyuromorphia. Using a substitution rate derived from WSINE1 insertions, the ages of the subfamilies were estimated and correlated with a newly established phylogeny of Dasyuromorphia. Phylogenetic analyses and divergence time estimates of mitochondrial genome data indicate a rapid radiation of the Tasmanian devil and the closest relative the quolls (Dasyurus) around 14 million years ago.

Conclusions: The radiation and abundance of CORE-SINEs in marsupial genomes indicates that they may be a major player in the evolution of marsupials. It is evident that the early phases of evolution of the carnivorous marsupial order Dasyuromorphia was characterized by a burst of SINE activity. A correlation between a speciation event and a major burst of retroposon activity is for the first time shown in a marsupial genome.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">172</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22559330" rel="external">22559330</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3403934" rel="external">3403934</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-172" rel="external">10.1186/1471-2164-13-172</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Atp6v0a4 knockout mouse is a model of distal renal tubular acidosis with hearing loss, with additional extrarenal phenotype.</h4>
<p class="authors"><span class="author">Norgett EE</span>, <span class="author">Golder ZJ</span>, <span class="author">Lorente-C&aacute;novas B</span>, <span class="author">Ingham N</span>, <span class="author">Steel KP</span> and <span class="author">Karet Frankl FE</span></p>
<p class="affiliation">Department of Medical Genetics, University of Cambridge, Cambridge CB2 0XY, United Kingdom.</p>
<p class="abstract">Autosomal recessive distal renal tubular acidosis (dRTA) is a severe disorder of acid-base homeostasis, often accompanied by sensorineural deafness. We and others have previously shown that mutations in the tissue-restricted a4 and B1 subunits of the H(+)-ATPase underlie this syndrome. Here, we describe an Atp6v0a4 knockout mouse, which lacks the a4 subunit. Using Î²-galactosidase as a reporter for the null gene, developmental a4 expression was detected in developing bone, nose, eye, and skin, in addition to that expected in kidney and inner ear. By the time of weaning, Atp6v0a4(-/-) mice demonstrated severe metabolic acidosis, hypokalemia, and early nephrocalcinosis. Null mice were hypocitraturic, but hypercalciuria was absent. They were severely hearing-impaired, as shown by elevated auditory brainstem response thresholds and absent endocochlear potential. They died rapidly unless alkalinized. If they survived weaning with alkali supplementation, treatment could later be withdrawn, but -/- animals remained acidotic with alkaline urine. They also had an impaired sense of smell. Heterozygous animals were biochemically normal until acid-challenged, when they became more acidotic than +/+ animals. This mouse model recapitulates the loss of H(+)-ATPase function seen in human disease and can provide additional insights into dRTA and the physiology of the a4 subunit.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">34</span>;<span class="pagerange">13775-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22872862" rel="external">22872862</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3427075" rel="external">3427075</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1204257109" rel="external">10.1073/pnas.1204257109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association meta-analysis identifies new endometriosis risk loci.</h4>
<p class="authors"><span class="author">Nyholt DR</span>, <span class="author">Low SK</span>, <span class="author">Anderson CA</span>, <span class="author">Painter JN</span>, <span class="author">Uno S</span>, <span class="author">Morris AP</span>, <span class="author">Macgregor S</span>, <span class="author">Gordon SD</span>, <span class="author">Henders AK</span>, <span class="author">Martin NG</span>, <span class="author">Attia J</span>, <span class="author">Holliday EG</span>, <span class="author">McEvoy M</span>, <span class="author">Scott RJ</span>, <span class="author">Kennedy SH</span>, <span class="author">Treloar SA</span>, <span class="author">Missmer SA</span>, <span class="author">Adachi S</span>, <span class="author">Tanaka K</span>, <span class="author">Nakamura Y</span>, <span class="author">Zondervan KT</span>, <span class="author">Zembutsu H</span> and <span class="author">Montgomery GW</span></p>
<p class="affiliation">1] Queensland Institute of Medical Research, Brisbane, Queensland, Australia. [2].</p>
<p class="abstract">We conducted a genome-wide association meta-analysis of 4,604 endometriosis cases and 9,393 controls of Japanese and European ancestry. We show that rs12700667 on chromosome 7p15.2, previously found to associate with disease in Europeans, replicates in Japanese (P = 3.6 × 10(-3)), and we confirm association of rs7521902 at 1p36.12 near WNT4. In addition, we establish an association of rs13394619 in GREB1 at 2p25.1 with endometriosis and identify a newly associated locus at 12q22 near VEZT (rs10859871). Excluding cases of European ancestry of minimal or unknown severity, we identified additional previously unknown loci at 2p14 (rs4141819), 6p22.3 (rs7739264) and 9p21.3 (rs1537377). All seven SNP effects were replicated in an independent cohort and associated at P <5 × 10(-8) in a combined analysis. Finally, we found a significant overlap in polygenic risk for endometriosis between the genome-wide association cohorts of European and Japanese descent (P = 8.8 × 10(-11)), indicating that many weakly associated SNPs represent true endometriosis risk loci and that risk prediction and future targeted disease therapy may be transferred across these populations.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23104006" rel="external">23104006</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2445" rel="external">10.1038/ng.2445</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sex-specific influence of DRD2 on ADHD-type temperament in a large population-based birth cohort.</h4>
<p class="authors"><span class="author">Nyman ES</span>, <span class="author">Loukola A</span>, <span class="author">Varilo T</span>, <span class="author">Taanila A</span>, <span class="author">Hurtig T</span>, <span class="author">Moilanen I</span>, <span class="author">Loo S</span>, <span class="author">McGough JJ</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Smalley SL</span>, <span class="author">Nelson SF</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Public Health Genomics Unit, Institute for Molecular Medicine Finland FIMM, University of Helsinki and National Institute for Health and Welfare, Helsinki, Finland. <a href="mailto:%65%6D%6D%61.%6E%79%6D%61%6E%40%74%68%6C.%66%69">&#101;&#109;&#109;&#97;&#46;&#110;&#121;&#109;&#97;&#110;&#64;&#116;&#104;&#108;&#46;&#102;&#105;</a></p><p class="abstract">Attention-deficit/hyperactivity disorder (ADHD) is a childhood-onset neurodevelopmental disorder with a significant public-health impact. Previously, we described a candidate gene study in a population-based birth cohort that demonstrated an association with ADHD-affected males and the dopamine receptor D2 (DRD2). The current study evaluates potential associations of dopamine receptor genes and Cloninger temperament traits within this same sample. Participants with stringent lifetime ADHD diagnoses were ascertained systematically from the genetically isolated Northern Finland 1986 Birth Cohort (n=9432), resulting in 178 cases and 157 controls. Markers in all known dopamine receptor genes were genotyped. We report an association of DRD2 with low Persistence in females (rs1079727 P=0.02, rs1124491 P=0.02, rs1800497 P=0.03). The associated DRD2 minor allelic haplotype (CAA, P=0.03) is the same haplotype we previously associated with ADHD in males in this birth cohort. The current study further supports previous results on the role of DRD2 in individuals with ADHD. Investigations suggest that DRD2 may have an impact on both males and females, but the particular outcome appears sex-specific, manifesting as ADHD in males and low Persistence in females. Furthermore, these findings suggest that the putative role of low Persistence as an endophenotype for ADHD deserves further investigation.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">MH01966</span>, <span class="grant">MH063706</span></p>
<p class="publication"><span class="title">Psychiatric genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">4</span>;<span class="pagerange">197-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22531292">22531292</a>; DOI: <a href="http://dx.doi.org/10.1097%2FYPG.0b013e32834c0cc8">10.1097/YPG.0b013e32834c0cc8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The rhoptry proteome of Eimeria tenella sporozoites.</h4>
<p class="authors"><span class="author">Oakes RD</span>, <span class="author">Kurian D</span>, <span class="author">Bromley E</span>, <span class="author">Ward C</span>, <span class="author">Lal K</span>, <span class="author">Blake DP</span>, <span class="author">Reid AJ</span>, <span class="author">Pain A</span>, <span class="author">Sinden RE</span>, <span class="author">Wastling JM</span> and <span class="author">Tomley FM</span></p>
<p class="affiliation">Institute for Animal Health, Compton, Newbury, Berkshire, UK.</p><p class="abstract">Proteins derived from the rhoptry secretory organelles are crucial for the invasion and survival of apicomplexan parasites within host cells. The rhoptries are club-shaped organelles that contain two distinct subpopulations of proteins that localise to separate compartments of the organelle. Proteins from the neck region (rhoptry neck proteins, RON) are secreted early in invasion and a subset of these is critical for the formation and function of the moving junction between parasite and host membranes. Proteins from the bulb compartment (rhoptry protein, ROP) are released later, into the nascent parasitophorous vacuole where they have a role in modifying the vacuolar environment, and into the host cell where they act as key determinants of virulence through their ability to interact with host cell signalling pathways, causing an array of downstream effects. In this paper we present the results of an extensive proteomics analysis of the rhoptry organelles from the coccidian parasite, Eimeria tenella, which is a highly pathogenic parasite of the domestic chicken causing severe caecal coccidiosis. Several different classes of rhoptry protein have been identified. First are the RON proteins that have varying degrees of similarity to proteins of Toxoplasma gondii and Neospora caninum. For some RON families, E. tenella expresses more than one gene product and many of the individual RON proteins are differentially expressed between the sporozoite and merozoite developmental stages. The E. tenella sporozoite rhoptry expresses only a limited repertoire of proteins with homology to known ROP proteins from other coccidia, including just two secreted ROP kinases, both of which appear to be equipped for catalytic activity. Finally, a large number of hitherto undescribed proteins that map to the sporozoite rhoptry are identified, many of which have orthologous proteins encoded within the genomes of T. gondii and N. caninum.</p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23262303">23262303</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijpara.2012.10.024">10.1016/j.ijpara.2012.10.024</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transmission of malaria to mosquitoes blocked by bumped kinase inhibitors.</h4>
<p class="authors"><span class="author">Ojo KK</span>, <span class="author">Pfander C</span>, <span class="author">Mueller NR</span>, <span class="author">Burstroem C</span>, <span class="author">Larson ET</span>, <span class="author">Bryan CM</span>, <span class="author">Fox AM</span>, <span class="author">Reid MC</span>, <span class="author">Johnson SM</span>, <span class="author">Murphy RC</span>, <span class="author">Kennedy M</span>, <span class="author">Mann H</span>, <span class="author">Leibly DJ</span>, <span class="author">Hewitt SN</span>, <span class="author">Verlinde CL</span>, <span class="author">Kappe S</span>, <span class="author">Merritt EA</span>, <span class="author">Maly DJ</span>, <span class="author">Billker O</span> and <span class="author">Van Voorhis WC</span></p>
<p class="affiliation">Division of Allergy and Infectious Diseases, Department of Medicine, University of Washington, Seattle, Washington 98195-6423, USA.</p><p class="abstract">Effective control and eradication of malaria will require new tools to prevent transmission. Current antimalarial therapies targeting the asexual stage of Plasmodium do not prevent transmission of circulating gametocytes from infected humans to mosquitoes. Here, we describe a new class of transmission-blocking compounds, bumped kinase inhibitors (BKIs), which inhibit microgametocyte exflagellation. Oocyst formation and sporozoite production, necessary for transmission to mammals, were inhibited in mosquitoes fed on either BKI-1-treated human blood or mice treated with BKI-1. BKIs are hypothesized to act via inhibition of Plasmodium calcium-dependent protein kinase 4 and predicted to have little activity against mammalian kinases. Our data show that BKIs do not inhibit proliferation of mammalian cell lines and are well tolerated in mice. Used in combination with drugs active against asexual stages of Plasmodium, BKIs could prove an important tool for malaria control and eradication.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; NIAID NIH HHS: <span class="grant">R01 AI089441</span>, <span class="grant">R01AI080625</span>, <span class="grant">R01AI089441</span>; NIGMS NIH HHS: <span class="grant">R01 GM086858</span>, <span class="grant">R01GM086858</span>; Wellcome Trust: <span class="grant">WT089085/Z/09/Z</span></p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2012</span>;<span class="volume">122</span>;<span class="number">6</span>;<span class="pagerange">2301-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22565309">22565309</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3366411">3366411</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI61822">10.1172/JCI61822</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intracontinental spread of human invasive Salmonella Typhimurium pathovariants in sub-Saharan Africa.</h4>
<p class="authors"><span class="author">Okoro CK</span>, <span class="author">Kingsley RA</span>, <span class="author">Connor TR</span>, <span class="author">Harris SR</span>, <span class="author">Parry CM</span>, <span class="author">Al-Mashhadani MN</span>, <span class="author">Kariuki S</span>, <span class="author">Msefula CL</span>, <span class="author">Gordon MA</span>, <span class="author">de Pinna E</span>, <span class="author">Wain J</span>, <span class="author">Heyderman RS</span>, <span class="author">Obaro S</span>, <span class="author">Alonso PL</span>, <span class="author">Mandomando I</span>, <span class="author">MacLennan CA</span>, <span class="author">Tapia MD</span>, <span class="author">Levine MM</span>, <span class="author">Tennant SM</span>, <span class="author">Parkhill J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">A highly invasive form of non-typhoidal Salmonella (iNTS) disease has recently been documented in many countries in sub-Saharan Africa. The most common Salmonella enterica serovar causing this disease is Typhimurium (Salmonella Typhimurium). We applied whole-genome sequence-based phylogenetic methods to define the population structure of sub-Saharan African invasive Salmonella Typhimurium isolates and compared these to global Salmonella Typhimurium populations. Notably, the vast majority of sub-Saharan invasive Salmonella Typhimurium isolates fell within two closely related, highly clustered phylogenetic lineages that we estimate emerged independently &sim;52 and &sim;35 years ago in close temporal association with the current HIV pandemic. Clonal replacement of isolates from lineage I by those from lineage II was potentially influenced by the use of chloramphenicol for the treatment of iNTS disease. Our analysis suggests that iNTS disease is in part an epidemic in sub-Saharan Africa caused by highly related Salmonella Typhimurium lineages that may have occupied new niches associated with a compromised human population and antibiotic treatment.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">11</span>;<span class="pagerange">1215-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23023330">23023330</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3491877">3491877</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2423">10.1038/ng.2423</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution single nucleotide polymorphism analysis distinguishes recrudescence and reinfection in recurrent invasive nontyphoidal Salmonella typhimurium disease.</h4>
<p class="authors"><span class="author">Okoro CK</span>, <span class="author">Kingsley RA</span>, <span class="author">Quail MA</span>, <span class="author">Kankwatira AM</span>, <span class="author">Feasey NA</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">Gordon MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Bloodstream infection with invasive nontyphoidal Salmonella (iNTS) is common and severe among human immunodeficiency virus (HIV)-infected adults throughout sub-Saharan Africa. The epidemiology of iNTS is poorly understood. Survivors frequently experience multiply recurrent iNTS disease, despite appropriate antimicrobial therapy, but recrudescence and reinfection have previously been difficult to distinguish.</p><p class="abstract"><strong>Methods</strong>: We used high-resolution single nucleotide polymorphism (SNP) typing and whole-genome phylogenetics to investigate 47 iNTS isolates from 14 patients with multiple recurrences following an index presentation with iNTS disease in Blantyre, Malawi. We isolated nontyphoidal salmonellae organisms from blood (n = 35), bone marrow (n = 8), stool (n = 2), urine (n = 1), and throat (n = 1) samples; these isolates comprised serotypes Typhimurium (n = 43) and Enteritidis (n = 4).</p><p class="abstract"><strong>Results</strong>: Recrudescence with identical or highly phylogenetically related isolates accounted for 78% of recurrences, and reinfection with phylogenetically distinct isolates accounted for 22% of recurrences. Both recrudescence and reinfection could occur in the same individual, and reinfection could either precede or follow recrudescence. The number of days to recurrence (23-486 d) was not different for recrudescence or reinfection. The number of days to recrudescence was unrelated to the number of SNPs accumulated by recrudescent organisms, suggesting that there was little genetic change during persistence in the host, despite exposure to multiple courses of antibiotics. Of Salmonella Typhimurium isolates, 42 of 43 were pathovar ST313.</p><p class="abstract"><strong>Conclusions</strong>: High-resolution whole-genome phylogenetics successfully discriminated recrudescent iNTS from reinfection, despite a high level of clonality within and among individuals, giving insights into pathogenesis and management. These methods also have adequate resolution to investigate the epidemiology and transmission of this important African pathogen.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">54</span>;<span class="number">7</span>;<span class="pagerange">955-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22318974" rel="external">22318974</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3297646" rel="external">3297646</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fcid%2Fcir1032" rel="external">10.1093/cid/cir1032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recessive HYDIN mutations cause primary ciliary dyskinesia without randomization of left-right body asymmetry.</h4>
<p class="authors"><span class="author">Olbrich H</span>, <span class="author">Schmidts M</span>, <span class="author">Werner C</span>, <span class="author">Onoufriadis A</span>, <span class="author">Loges NT</span>, <span class="author">Raidt J</span>, <span class="author">Banki NF</span>, <span class="author">Shoemark A</span>, <span class="author">Burgoyne T</span>, <span class="author">Al Turki S</span>, <span class="author">Hurles ME</span>, <span class="author">UK10K Consortium</span>, <span class="author">K&ouml;hler G</span>, <span class="author">Schroeder J</span>, <span class="author">N&uuml;rnberg G</span>, <span class="author">N&uuml;rnberg P</span>, <span class="author">Chung EM</span>, <span class="author">Reinhardt R</span>, <span class="author">Marthin JK</span>, <span class="author">Nielsen KG</span>, <span class="author">Mitchison HM</span> and <span class="author">Omran H</span></p>
<p class="affiliation">Department of General Pediatrics, University Children&#39;s Hospital Muenster, 48149 Muenster, Germany.</p><p class="abstract">Primary ciliary dyskinesia (PCD) is a genetically heterogeneous recessive disorder characterized by defective cilia and flagella motility. Chronic destructive-airway disease is caused by abnormal respiratory-tract mucociliary clearance. Abnormal propulsion of sperm flagella contributes to male infertility. Genetic defects in most individuals affected by PCD cause randomization of left-right body asymmetry; approximately half show situs inversus or situs ambiguous. Almost 70 years after the hy3 mouse possessing Hydin mutations was described as a recessive hydrocephalus model, we report HYDIN mutations in PCD-affected persons without hydrocephalus. By homozygosity mapping, we identified a PCD-associated locus, chromosomal region 16q21-q23, which contains HYDIN. However, a nearly identical 360 kb paralogous segment (HYDIN2) in chromosomal region 1q21.1 complicated mutational analysis. In three affected German siblings linked to HYDIN, we identified homozygous c.3985G&gt;T mutations that affect an evolutionary conserved splice acceptor site and that subsequently cause aberrantly spliced transcripts predicting premature protein termination in respiratory cells. Parallel whole-exome sequencing identified a homozygous nonsense HYDIN mutation, c.922A&gt;T (p.Lys307(&lowast;)), in six individuals from three Faroe Island PCD-affected families that all carried an 8.8 Mb shared haplotype across HYDIN, indicating an ancestral founder mutation in this isolated population. We demonstrate by electron microscopy tomography that, consistent with the effects of loss-of-function mutations, HYDIN mutant respiratory cilia lack the C2b projection of the central pair (CP) apparatus; similar findings were reported in Hydin-deficient Chlamydomonas and mice. High-speed videomicroscopy demonstrated markedly reduced beating amplitudes of respiratory cilia and stiff sperm flagella. Like the hy3 mouse model, all nine PCD-affected persons had normal body composition because nodal cilia function is apparently not dependent on the function of the CP apparatus.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">090532</span>, <span class="grant">091310</span>, <span class="grant">091551</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">91</span>;<span class="number">4</span>;<span class="pagerange">672-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23022101">23022101</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3484652">3484652</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.08.016">10.1016/j.ajhg.2012.08.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Stat6-regulated KRAB domain zinc finger protein Zfp157 regulates the balance of lineages in mammary glands and compensates for loss of Gata-3.</h4>
<p class="authors"><span class="author">Oliver CH</span>, <span class="author">Khaled WT</span>, <span class="author">Frend H</span>, <span class="author">Nichols J</span> and <span class="author">Watson CJ</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Cambridge CB2 1QP, United Kingdom.</p>
<p class="abstract">Lineage commitment studies in mammary glands have focused on identifying cell populations that display stem or progenitor properties. However, the mechanisms that control cell fate have been incompletely explored. Herein we show that zinc finger protein 157 (Zfp157) is required to establish the balance between luminal alveolar pStat5- and Gata-3-expressing cells in the murine mammary gland. Using mice in which the zfp157 gene was disrupted, we found that alveologenesis was accelerated concomitantly with a dramatic skewing of the proportion of pStat5-expressing cells relative to Gata-3âº cells. This suppression of the Gata-3âº lineage was associated with increased expression of the inhibitor of helix-loop-helix protein Id2. Surprisingly, Gata-3 becomes dispensable in the absence of Zfp157, as mice deficient for both Zfp157 and Gata-3 lactate normally, although the glands display a mild epithelial dysplasia. These data suggest that the luminal alveolar compartment of the mammary gland is comprised of a number of distinct cell populations that, although interdependant, exhibit considerable cell fate plasticity.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/H006206/1</span></p>
<p class="publication"><span class="title">Genes &amp;amp; development</span>&nbsp;<span class="year">2012</span>;<span class="volume">26</span>;<span class="number">10</span>;<span class="pagerange">1086-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22588720" rel="external">22588720</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3360563" rel="external">3360563</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.184051.111" rel="external">10.1101/gad.184051.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cestode genomics - progress and prospects for advancing basic and applied aspects of flatworm biology.</h4>
<p class="authors"><span class="author">Olson PD</span>, <span class="author">Zarowiecki M</span>, <span class="author">Kiss F</span> and <span class="author">Brehm K</span></p>
<p class="affiliation">Department of Zoology, The Natural History Museum, London, UK.</p><p class="abstract">Characterization of the first tapeworm genome, Echinococcus multilocularis, is now nearly complete, and genome assemblies of E.&nbsp;granulosus, Taenia solium and Hymenolepis microstoma are in advanced draft versions. These initiatives herald the beginning of a genomic era in cestodology and underpin a diverse set of research agendas targeting both basic and applied aspects of tapeworm biology. We discuss the progress in the genomics of these species, provide insights into the presence and composition of immunologically relevant gene families, including the antigen B- and EG95/45W families, and discuss chemogenomic approaches toward the development of novel chemotherapeutics against cestode diseases. In addition, we discuss the evolution of tapeworm parasites and introduce the research programmes linked to genome initiatives that are aimed at understanding signalling systems involved in basic host-parasite interactions and morphogenesis.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBG0038151</span></p>
<p class="publication"><span class="title">Parasite immunology</span>&nbsp;<span class="year">2012</span>;<span class="volume">34</span>;<span class="number">2-3</span>;<span class="pagerange">130-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21793855">21793855</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-3024.2011.01319.x">10.1111/j.1365-3024.2011.01319.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing of liver fluke-associated cholangiocarcinoma.</h4>
<p class="authors"><span class="author">Ong CK</span>, <span class="author">Subimerb C</span>, <span class="author">Pairojkul C</span>, <span class="author">Wongkham S</span>, <span class="author">Cutcutache I</span>, <span class="author">Yu W</span>, <span class="author">McPherson JR</span>, <span class="author">Allen GE</span>, <span class="author">Ng CC</span>, <span class="author">Wong BH</span>, <span class="author">Myint SS</span>, <span class="author">Rajasegaran V</span>, <span class="author">Heng HL</span>, <span class="author">Gan A</span>, <span class="author">Zang ZJ</span>, <span class="author">Wu Y</span>, <span class="author">Wu J</span>, <span class="author">Lee MH</span>, <span class="author">Huang D</span>, <span class="author">Ong P</span>, <span class="author">Chan-on W</span>, <span class="author">Cao Y</span>, <span class="author">Qian CN</span>, <span class="author">Lim KH</span>, <span class="author">Ooi A</span>, <span class="author">Dykema K</span>, <span class="author">Furge K</span>, <span class="author">Kukongviriyapan V</span>, <span class="author">Sripa B</span>, <span class="author">Wongkham C</span>, <span class="author">Yongvanit P</span>, <span class="author">Futreal PA</span>, <span class="author">Bhudhisawasdi V</span>, <span class="author">Rozen S</span>, <span class="author">Tan P</span> and <span class="author">Teh BT</span></p>
<p class="affiliation">National Cancer Centre Singapore-Van Andel Research Institute Translational Research Laboratory, Division of Medical Sciences, Singapore.</p><p class="abstract">Opisthorchis viverrini-related cholangiocarcinoma (CCA), a fatal bile duct cancer, is a major public health concern in areas endemic for this parasite. We report here whole-exome sequencing of eight O. viverrini-related tumors and matched normal tissue. We identified and validated 206 somatic mutations in 187 genes using Sanger sequencing and selected 15 genes for mutation prevalence screening in an additional 46 individuals with CCA (cases). In addition to the known cancer-related genes TP53 (mutated in 44.4% of cases), KRAS (16.7%) and SMAD4 (16.7%), we identified somatic mutations in 10 newly implicated genes in 14.8-3.7% of cases. These included inactivating mutations in MLL3 (in 14.8% of cases), ROBO2 (9.3%), RNF43 (9.3%) and PEG3 (5.6%), and activating mutations in the GNAS oncogene (9.3%). These genes have functions that can be broadly grouped into three biological classes: (i) deactivation of histone modifiers, (ii) activation of G protein signaling and (iii) loss of genome stability. This study provides insight into the mutational landscape contributing to O. viverrini-related CCA.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">6</span>;<span class="pagerange">690-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22561520">22561520</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2273">10.1038/ng.2273</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Delineating nuclear reprogramming.</h4>
<p class="authors"><span class="author">Ooi J</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK.</p><p class="abstract">Nuclear reprogramming is described as a molecular switch, triggered by the conversion of one cell type to another. Several key experiments in the past century have provided insight into the field of nuclear reprogramming. Previously deemed impossible, this research area is now brimming with new findings and developments. In this review, we aim to give a historical perspective on how the notion of nuclear reprogramming was established, describing main experiments that were performed, including (1) somatic cell nuclear transfer, (2) exposure to cell extracts and cell fusion, and (3) transcription factor induced lineage switch. Ultimately, we focus on (4) transcription factor induced pluripotency, as initiated by a landmark discovery in 2006, where the process of converting somatic cells to a pluripotent state was narrowed down to four transcription factors. The conception that somatic cells possess the capacity to revert to an immature status brings about huge clinical implications including personalized therapy, drug screening and disease modeling. Although this technology has potential to revolutionize the medical field, it is still impeded by technical and biological obstacles. This review describes the effervescent changes in this field, addresses bottlenecks hindering its advancement and in conclusion, applies the latest findings to overcome these issues.</p>
<p class="publication"><span class="title">Protein &amp; cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">329-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22467264" rel="external">22467264</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs13238-012-2920-x" rel="external">10.1007/s13238-012-2920-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pluripotency and its layers of complexity.</h4>
<p class="authors"><span class="author">Ooi J</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1SA UK ; Technology and Research, Agency for Science, 1 Fusionopolis Way, #20-10, Connexis North Tower, Kragujevac, 138632 Singapore.</p><p class="abstract">Pluripotency is depicted by a self-renewing state that can competently differentiate to form the three germ layers. Different stages of early murine development can be captured on a petri dish, delineating a spectrum of pluripotent states, ranging from embryonic stem cells, embryonic germ cells to epiblast stem cells. Anomalous cell populations displaying signs of pluripotency have also been uncovered, from the isolation of embryonic carcinoma cells to the derivation of induced pluripotent stem cells. Gaining insight into the molecular circuitry within these cell types enlightens us about the significance and contribution of each stage, hence deepening our understanding of vertebrate development. In this review, we aim to describe experimental milestones that led to the understanding of embryonic development and the conception of pluripotency. We also discuss attempts at exploring the realm of pluripotency with the identification of pluripotent stem cells within mouse teratocarcinomas and embryos, and the generation of pluripotent cells through nuclear reprogramming. In conclusion, we illustrate pluripotent cells derived from other organisms, including human derivatives, and describe current paradigms in the comprehension of human pluripotency.</p>
<p class="publication"><span class="title">Cell regeneration (London, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">1</span>;<span class="number">1</span>;<span class="pagerange">7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/25408870">25408870</a>; DOI: <a href="http://dx.doi.org/10.1186%2F2045-9769-1-7">10.1186/2045-9769-1-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sex-specific differences in effect size estimates at established complex trait loci.</h4>
<p class="authors"><span class="author">Orozco G</span>, <span class="author">Ioannidis JP</span>, <span class="author">Morris A</span>, <span class="author">Zeggini E</span> and <span class="author">DIAGRAM consortium</span></p>
<p class="affiliation">Arthritis Research UK Epidemiology Unit, Manchester Academic Health Science Centre, The University of Manchester, UK, Stanford University Prevention Research Center, Stanford University School of Medicine, Stanford, CA 94305, USA, Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina 45110, Greece, Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK and Wellcome Trust Sanger Institute, Hinxton, UK.</p>
<p class="abstract">BACKGROUND: Genetic differences between men and women may contribute to sex differences in prevalence and progression of many common complex diseases.Using the WTCCC GWAS, we analysed whether there are sex-specific differences in effect size estimates at 142 established loci for seven complex diseases: rheumatoid arthritis, type 1 diabetes (T1D), Crohn's disease, type 2 diabetes (T2D), hypertension, coronary artery disease and bipolar disorder. METHODS: For each Single nucleotide polymorphism (SNP), we calculated the per-allele odds ratio for each sex and the relative odds ratios (RORs; the effect size is higher in men with ROR greater than one). RORs were then meta-analysed across loci within each disease and across diseases. RESULTS: For each disease, summary RORs were not different from one, but there was between-SNP heterogeneity in the RORs for T1D and T2D. Four loci in T1D, three in Crohn's disease and three in T2D showed differences in the genetic effect between men and women (Pâ€‰<â€‰0.05). We probed these differences in additional independent replication samples for T1D and T2D. The differences remained for the T1D loci CTSH, 17q21 and 20p13 and the T2D locus BCL11A, when WTCCC data and replication data were meta-analysed. Only CTSH showed different genetic effect between men and women in the replication data alone. CONCLUSION: Our results exclude the presence of large and frequent differences in the effect size estimates between men and women for the established loci in the seven common diseases explored. Documenting small differences in genetic effects between men and women requires large studies and systematic evaluation.</p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22825589" rel="external">22825589</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdys104" rel="external">10.1093/ije/dys104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Xirp proteins mark injured skeletal muscle in zebrafish.</h4>
<p class="authors"><span class="author">Otten C</span>, <span class="author">van der Ven PF</span>, <span class="author">Lewrenz I</span>, <span class="author">Paul S</span>, <span class="author">Steinhagen A</span>, <span class="author">Busch-Nentwich E</span>, <span class="author">Eichhorst J</span>, <span class="author">Wiesner B</span>, <span class="author">Stemple D</span>, <span class="author">Str&auml;hle U</span>, <span class="author">F&uuml;rst DO</span> and <span class="author">Abdelilah-Seyfried S</span></p>
<p class="affiliation">Max Delbr&uuml;ck Center (MDC) for Molecular Medicine, Berlin, Germany.</p><p class="abstract">Myocellular regeneration in vertebrates involves the proliferation of activated progenitor or dedifferentiated myogenic cells that have the potential to replenish lost tissue. In comparison little is known about cellular repair mechanisms within myocellular tissue in response to small injuries caused by biomechanical or cellular stress. Using a microarray analysis for genes upregulated upon myocellular injury, we identified zebrafish Xin-actin-binding repeat-containing protein1 (Xirp1) as a marker for wounded skeletal muscle cells. By combining laser-induced micro-injury with proliferation analyses, we found that Xirp1 and Xirp2a localize to nascent myofibrils within wounded skeletal muscle cells and that the repair of injuries does not involve cell proliferation or Pax7(+) cells. Through the use of Xirp1 and Xirp2a as markers, myocellular injury can now be detected, even though functional studies indicate that these proteins are not essential in this process. Previous work in chicken has implicated Xirps in cardiac looping morphogenesis. However, we found that zebrafish cardiac morphogenesis is normal in the absence of Xirp expression, and animals deficient for cardiac Xirp expression are adult viable. Although the functional involvement of Xirps in developmental and repair processes currently remains enigmatic, our findings demonstrate that skeletal muscle harbours a rapid, cell-proliferation-independent response to injury which has now become accessible to detailed molecular and cellular characterizations.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e31041</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22355335" rel="external">22355335</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3280289" rel="external">3280289</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0031041" rel="external">10.1371/journal.pone.0031041</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes.</h4>
<p class="authors"><span class="author">Oyola SO</span>, <span class="author">Otto TD</span>, <span class="author">Gu Y</span>, <span class="author">Maslen G</span>, <span class="author">Manske M</span>, <span class="author">Campino S</span>, <span class="author">Turner DJ</span>, <span class="author">Macinnis B</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Swerdlow HP</span> and <span class="author">Quail MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%73%6F%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#111;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Massively parallel sequencing technology is revolutionizing approaches to genomic and genetic research. Since its advent, the scale and efficiency of Next-Generation Sequencing (NGS) has rapidly improved. In spite of this success, sequencing genomes or genomic regions with extremely biased base composition is still a great challenge to the currently available NGS platforms. The genomes of some important pathogenic organisms like Plasmodium falciparum (high AT content) and Mycobacterium tuberculosis (high GC content) display extremes of base composition. The standard library preparation procedures that employ PCR amplification have been shown to cause uneven read coverage particularly across AT and GC rich regions, leading to problems in genome assembly and variation analyses. Alternative library-preparation approaches that omit PCR amplification require large quantities of starting material and hence are not suitable for small amounts of DNA/RNA such as those from clinical isolates. We have developed and optimized library-preparation procedures suitable for low quantity starting material and tolerant to extremely high AT content sequences.</p><p class="abstract"><strong>Results</strong>: We have used our optimized conditions in parallel with standard methods to prepare Illumina sequencing libraries from a non-clinical and a clinical isolate (containing ~53% host contamination). By analyzing and comparing the quality of sequence data generated, we show that our optimized conditions that involve a PCR additive (TMAC), produces amplified libraries with improved coverage of extremely AT-rich regions and reduced bias toward GC neutral templates.</p><p class="abstract"><strong>Conclusion</strong>: We have developed a robust and optimized Next-Generation Sequencing library amplification method suitable for extremely AT-rich genomes. The new amplification conditions significantly reduce bias and retain the complexity of either extremes of base composition. This development will greatly benefit sequencing clinical samples that often require amplification due to low mass of DNA starting material.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">079355/Z/06/Z</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22214261">22214261</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3312816">3312816</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-1">10.1186/1471-2164-13-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mir-133a regulates VKORC1, a key protein in vitamin K cycle.</h4>
<p class="authors"><span class="author">P&eacute;rez-Andreu V</span>, <span class="author">Teruel R</span>, <span class="author">Corral J</span>, <span class="author">Rold&aacute;n V</span>, <span class="author">Garc&iacute;a-Barber&aacute; N</span>, <span class="author">Salloum-Asfar S</span>, <span class="author">G&oacute;mez-Lech&oacute;n MJ</span>, <span class="author">Bourgeois S</span>, <span class="author">Deloukas P</span>, <span class="author">Wadelius M</span>, <span class="author">Vicente V</span>, <span class="author">Gonz&aacute;lez-Conejero R</span> and <span class="author">Mart&iacute;nez C</span></p>
<p class="affiliation">Centro Regional de Hemodonación. University of Murcia. Murcia, Spain.</p>
<p class="abstract">Regulation of key proteins by microRNAs (miRNAs) is an emergent field in biomedicine. Vitamin K 2,3-epoxide reductase complex subunit 1 (VKORC1) is a relevant molecule for cardiovascular diseases, since it is the target of oral anticoagulant drugs and plays a role in soft tissue calcification. The objective of this study was to determine the influence of miRNAs on the expression of VKORC1. Potential miRNAs targeting VKORC1 mRNA were searched using on-line algorithms. Validation studies were carried out in HepG2 cells using miRNA precursors; direct miRNA interaction was investigated with reporter assays.In silico studies identified two putative conserved binding sites for miR-133a and miR-137 on VKORC1 mRNA. Ex vivo studies showed that only miR-133a was expressed in liver; transfection of miRNA precursors of miR-133a in HepG2 cells reduced VKORC1 mRNA expression in a dose-dependent manner as assessed by qRT-PCR as well as protein expression. Reporter assays in HEK293T cells showed that miR-133a interacts with the 3'UTR of VKORC1. Additionally, miR-133a levels correlated inversely with VKORC1 mRNA levels in 23 liver samples from healthy subjects.In conclusion, miR-133a appears to have a direct regulatory effect on expression of VKORC1 in humans; this regulation may have potential importance for anticoagulant therapy or aortic calcification.</p>
<p class="publication"><span class="title">Molecular medicine (Cambridge, Mass.)</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23154637">23154637</a>; DOI: <a href="http://dx.doi.org/10.2119%2Fmolmed.2012.00062">10.2119/molmed.2012.00062</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The deubiquitinase USP9X suppresses pancreatic ductal adenocarcinoma.</h4>
<p class="authors"><span class="author">P&eacute;rez-Mancera PA</span>, <span class="author">Rust AG</span>, <span class="author">van der Weyden L</span>, <span class="author">Kristiansen G</span>, <span class="author">Li A</span>, <span class="author">Sarver AL</span>, <span class="author">Silverstein KA</span>, <span class="author">Gr&uuml;tzmann R</span>, <span class="author">Aust D</span>, <span class="author">R&uuml;mmele P</span>, <span class="author">Kn&ouml;sel T</span>, <span class="author">Herd C</span>, <span class="author">Stemple DL</span>, <span class="author">Kettleborough R</span>, <span class="author">Brosnan JA</span>, <span class="author">Li A</span>, <span class="author">Morgan R</span>, <span class="author">Knight S</span>, <span class="author">Yu J</span>, <span class="author">Stegeman S</span>, <span class="author">Collier LS</span>, <span class="author">ten Hoeve JJ</span>, <span class="author">de Ridder J</span>, <span class="author">Klein AP</span>, <span class="author">Goggins M</span>, <span class="author">Hruban RH</span>, <span class="author">Chang DK</span>, <span class="author">Biankin AV</span>, <span class="author">Grimmond SM</span>, <span class="author">Australian Pancreatic Cancer Genome Initiative</span>, <span class="author">Wessels LF</span>, <span class="author">Wood SA</span>, <span class="author">Iacobuzio-Donahue CA</span>, <span class="author">Pilarsky C</span>, <span class="author">Largaespada DA</span>, <span class="author">Adams DJ</span> and <span class="author">Tuveson DA</span></p>
<p class="affiliation">Li Ka Shing Centre, Cambridge Research Institute, Cancer Research UK, Cambridge CB2 0RE, UK.</p><p class="abstract">Pancreatic ductal adenocarcinoma (PDA) remains a lethal malignancy despite much progress concerning its molecular characterization. PDA tumours harbour four signature somatic mutations in addition to numerous lower frequency genetic events of uncertain significance. Here we use Sleeping Beauty (SB) transposon-mediated insertional mutagenesis in a mouse model of pancreatic ductal preneoplasia to identify genes that cooperate with oncogenic Kras(G12D) to accelerate tumorigenesis and promote progression. Our screen revealed new candidate genes for PDA and confirmed the importance of many genes and pathways previously implicated in human PDA. The most commonly mutated gene was the X-linked deubiquitinase Usp9x, which was inactivated in over 50% of the tumours. Although previous work had attributed a pro-survival role to USP9X in human neoplasia, we found instead that loss of Usp9x enhances transformation and protects pancreatic cancer cells from anoikis. Clinically, low USP9X protein and messenger RNA expression in PDA correlates with poor survival after surgery, and USP9X levels are inversely associated with metastatic burden in advanced disease. Furthermore, chromatin modulation with trichostatin A or 5-aza-2&#39;-deoxycytidine elevates USP9X expression in human PDA cell lines, indicating a clinical approach for certain patients. The conditional deletion of Usp9x cooperated with Kras(G12D) to accelerate pancreatic tumorigenesis in mice, validating their genetic interaction. We propose that USP9X is a major tumour suppressor gene with prognostic and therapeutic relevance in PDA.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; NCI NIH HHS: <span class="grant">2P50CA101955</span>, <span class="grant">CA106610</span>, <span class="grant">CA122183</span>, <span class="grant">CA128920</span>, <span class="grant">CA62924</span>, <span class="grant">K01 CA122183</span>, <span class="grant">K01 CA122183-05</span>, <span class="grant">P50 CA062924</span>, <span class="grant">P50 CA101955</span>, <span class="grant">P50CA62924</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">486</span>;<span class="number">7402</span>;<span class="pagerange">266-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22699621">22699621</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3376394">3376394</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11114">10.1038/nature11114</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DvD: An R/Cytoscape pipeline for drug repurposing using public repositories of gene expression data.</h4>
<p class="authors"><span class="author">Pacini C</span>, <span class="author">Iorio F</span>, <span class="author">Gon&ccedil;alves E</span>, <span class="author">Iskar M</span>, <span class="author">Klabunde T</span>, <span class="author">Bork P</span> and <span class="author">Saez-Rodriguez J</span></p>
<p class="affiliation">European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Cambridge, UK CB10 1SD Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK, CB10 1HH EMBL, Meyerhofstrasse 1, 69117 Heidelberg, Germany Sanofi-Aventis Deutschland GmbH, Frankurt am Main, Germany.</p>
<p class="abstract">SUMMARY: Drug versus Disease (DvD) provides a pipeline, available through R or Cytoscape, for the comparison of drug and disease gene expression profiles from public microarray repositories. Negatively correlated profiles can be used to generate hypotheses of drug-repurposing whilst positively correlated profiles may be used to infer side-effects of drugs. DvD allows users to compare drug and disease signatures with dynamic access to databases Array Express, GEO and data from the Connectivity Map.Availability and Implementation: R package (submitted to Bioconductor) under GPL 3 and Cytoscape plug-in freely available for download at www.ebi.ac.uk/saezrodriguez/DVD/ CONTACT: <a href="mailto:%73%61%65%7A%72%6F%64%72%69%67%75%65%7A%40%65%62%69.%61%63.%75%6B">&#115;&#97;&#101;&#122;&#114;&#111;&#100;&#114;&#105;&#103;&#117;&#101;&#122;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a> SUPPLEMENTARY INFORMATION: Accompanies this paper.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23129297" rel="external">23129297</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts656" rel="external">10.1093/bioinformatics/bts656</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High altitude adaptation in Daghestani populations from the Caucasus.</h4>
<p class="authors"><span class="author">Pagani L</span>, <span class="author">Ayub Q</span>, <span class="author">MacArthur DG</span>, <span class="author">Xue Y</span>, <span class="author">Baillie JK</span>, <span class="author">Chen Y</span>, <span class="author">Kozarewa I</span>, <span class="author">Turner DJ</span>, <span class="author">Tofanelli S</span>, <span class="author">Bulayeva K</span>, <span class="author">Kidd K</span>, <span class="author">Paoli G</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%6C%70%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#112;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have surveyed 15 high-altitude adaptation candidate genes for signals of positive selection in North Caucasian highlanders using targeted re-sequencing. A total of 49 unrelated Daghestani from three ethnic groups (Avars, Kubachians, and Laks) living in ancient villages located at around 2,000 m above sea level were chosen as the study population. Caucasian (Adygei living at sea level, N = 20) and CEU (CEPH Utah residents with ancestry from northern and western Europe; N = 20) were used as controls. Candidate genes were compared with 20 putatively neutral control regions resequenced in the same individuals. The regions of interest were amplified by long-PCR, pooled according to individual, indexed by adding an eight-nucleotide tag, and sequenced using the Illumina GAII platform. 1,066 SNPs were called using false discovery and false negative thresholds of ~6%. The neutral regions provided an empirical null distribution to compare with the candidate genes for signals of selection. Two genes stood out. In Laks, a non-synonymous variant within HIF1A already known to be associated with improvement in oxygen metabolism was rediscovered, and in Kubachians a cluster of 13 SNPs located in a conserved intronic region within EGLN1 showing high population differentiation was found. These variants illustrate both the common pathways of adaptation to high altitude in different populations and features specific to the Daghestani populations, showing how even a mildly hypoxic environment can lead to genetic adaptation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">131</span>;<span class="number">3</span>;<span class="pagerange">423-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21904933">21904933</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3312735">3312735</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-011-1084-8">10.1007/s00439-011-1084-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ethiopian genetic diversity reveals linguistic stratification and complex influences on the Ethiopian gene pool.</h4>
<p class="authors"><span class="author">Pagani L</span>, <span class="author">Kivisild T</span>, <span class="author">Tarekegn A</span>, <span class="author">Ekong R</span>, <span class="author">Plaster C</span>, <span class="author">Gallego Romero I</span>, <span class="author">Ayub Q</span>, <span class="author">Mehdi SQ</span>, <span class="author">Thomas MG</span>, <span class="author">Luiselli D</span>, <span class="author">Bekele E</span>, <span class="author">Bradman N</span>, <span class="author">Balding DJ</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Division of Biological Anthropology, University of Cambridge, UK. <a href="mailto:%6C%70%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#112;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Humans and their ancestors have traversed the Ethiopian landscape for millions of years, and present-day Ethiopians show great cultural, linguistic, and historical diversity, which makes them essential for understanding African variability and human origins. We genotyped 235 individuals from ten Ethiopian and two neighboring (South Sudanese and Somali) populations on an Illumina Omni 1M chip. Genotypes were compared with published data from several African and non-African populations. Principal-component and STRUCTURE-like analyses confirmed substantial genetic diversity both within and between populations, and revealed a match between genetic data and linguistic affiliation. Using comparisons with African and non-African reference samples in 40-SNP genomic windows, we identified &quot;African&quot; and &quot;non-African&quot; haplotypic components for each Ethiopian individual. The non-African component, which includes the SLC24A5 allele associated with light skin pigmentation in Europeans, may represent gene flow into Africa, which we estimate to have occurred ~3 thousand years ago (kya). The non-African component was found to be more similar to populations inhabiting the Levant rather than the Arabian Peninsula, but the principal route for the expansion out of Africa ~60 kya remains unresolved. Linkage-disequilibrium decay with genomic distance was less rapid in both the whole genome and the African component than in southern African samples, suggesting a less ancient history for Ethiopian populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">91</span>;<span class="number">1</span>;<span class="pagerange">83-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22726845">22726845</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3397267">3397267</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.05.015">10.1016/j.ajhg.2012.05.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association search for type 2 diabetes genes in African Americans.</h4>
<p class="authors"><span class="author">Palmer ND</span>, <span class="author">McDonough CW</span>, <span class="author">Hicks PJ</span>, <span class="author">Roh BH</span>, <span class="author">Wing MR</span>, <span class="author">An SS</span>, <span class="author">Hester JM</span>, <span class="author">Cooke JN</span>, <span class="author">Bostrom MA</span>, <span class="author">Rudock ME</span>, <span class="author">Talbert ME</span>, <span class="author">Lewis JP</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Ferrara A</span>, <span class="author">Lu L</span>, <span class="author">Ziegler JT</span>, <span class="author">Sale MM</span>, <span class="author">Divers J</span>, <span class="author">Shriner D</span>, <span class="author">Adeyemo A</span>, <span class="author">Rotimi CN</span>, <span class="author">Ng MC</span>, <span class="author">Langefeld CD</span>, <span class="author">Freedman BI</span>, <span class="author">Bowden DW</span>, <span class="author">Voight BF</span>, <span class="author">Scott LJ</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Morris AP</span>, <span class="author">Dina C</span>, <span class="author">Welch RP</span>, <span class="author">Zeggini E</span>, <span class="author">Huth C</span>, <span class="author">Aulchenko YS</span>, <span class="author">Thorleifsson G</span>, <span class="author">McCulloch LJ</span>, <span class="author">Ferreira T</span>, <span class="author">Grallert H</span>, <span class="author">Amin N</span>, <span class="author">Wu G</span>, <span class="author">Willer CJ</span>, <span class="author">Raychaudhuri S</span>, <span class="author">McCarroll SA</span>, <span class="author">Langenberg C</span>, <span class="author">Hofmann OM</span>, <span class="author">Dupuis J</span>, <span class="author">Qi L</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">van Hoek M</span>, <span class="author">Navarro P</span>, <span class="author">Ardlie K</span>, <span class="author">Balkau B</span>, <span class="author">Benediktsson R</span>, <span class="author">Bennett AJ</span>, <span class="author">Blagieva R</span>, <span class="author">Boerwinkle E</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bostr&ouml;m KB</span>, <span class="author">Bravenboer B</span>, <span class="author">Bumpstead S</span>, <span class="author">Burtt NP</span>, <span class="author">Charpentier G</span>, <span class="author">Chines PS</span>, <span class="author">Cornelis M</span>, <span class="author">Couper DJ</span>, <span class="author">Crawford G</span>, <span class="author">Doney AS</span>, <span class="author">Elliott KS</span>, <span class="author">Elliott AL</span>, <span class="author">Erdos MR</span>, <span class="author">Fox CS</span>, <span class="author">Franklin CS</span>, <span class="author">Ganser M</span>, <span class="author">Gieger C</span>, <span class="author">Grarup N</span>, <span class="author">Green T</span>, <span class="author">Griffin S</span>, <span class="author">Groves CJ</span>, <span class="author">Guiducci C</span>, <span class="author">Hadjadj S</span>, <span class="author">Hassanali N</span>, <span class="author">Herder C</span>, <span class="author">Isomaa B</span>, <span class="author">Jackson AU</span>, <span class="author">Johnson PR</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Kao WH</span>, <span class="author">Klopp N</span>, <span class="author">Kong A</span>, <span class="author">Kraft P</span>, <span class="author">Kuusisto J</span>, <span class="author">Lauritzen T</span>, <span class="author">Li M</span>, <span class="author">Lieverse A</span>, <span class="author">Lindgren CM</span>, <span class="author">Lyssenko V</span>, <span class="author">Marre M</span>, <span class="author">Meitinger T</span>, <span class="author">Midthjell K</span>, <span class="author">Morken MA</span>, <span class="author">Narisu N</span>, <span class="author">Nilsson P</span>, <span class="author">Owen KR</span>, <span class="author">Payne F</span>, <span class="author">Perry JR</span>, <span class="author">Petersen AK</span>, <span class="author">Platou C</span>, <span class="author">Proen&ccedil;a C</span>, <span class="author">Prokopenko I</span>, <span class="author">Rathmann W</span>, <span class="author">Rayner NW</span>, <span class="author">Robertson NR</span>, <span class="author">Rocheleau G</span>, <span class="author">Roden M</span>, <span class="author">Sampson MJ</span>, <span class="author">Saxena R</span>, <span class="author">Shields BM</span>, <span class="author">Shrader P</span>, <span class="author">Sigurdsson G</span>, <span class="author">Spars&oslash; T</span>, <span class="author">Strassburger K</span>, <span class="author">Stringham HM</span>, <span class="author">Sun Q</span>, <span class="author">Swift AJ</span>, <span class="author">Thorand B</span>, <span class="author">Tichet J</span>, <span class="author">Tuomi T</span>, <span class="author">van Dam RM</span>, <span class="author">van Haeften TW</span>, <span class="author">van Herpt T</span>, <span class="author">van Vliet-Ostaptchouk JV</span>, <span class="author">Walters GB</span>, <span class="author">Weedon MN</span>, <span class="author">Wijmenga C</span>, <span class="author">Witteman J</span>, <span class="author">Bergman RN</span>, <span class="author">Cauchi S</span>, <span class="author">Collins FS</span>, <span class="author">Gloyn AL</span>, <span class="author">Gyllensten U</span>, <span class="author">Hansen T</span>, <span class="author">Hide WA</span>, <span class="author">Hitman GA</span>, <span class="author">Hofman A</span>, <span class="author">Hunter DJ</span>, <span class="author">Hveem K</span>, <span class="author">Laakso M</span>, <span class="author">Mohlke KL</span>, <span class="author">Morris AD</span>, <span class="author">Palmer CN</span>, <span class="author">Pramstaller PP</span>, <span class="author">Rudan I</span>, <span class="author">Sijbrands E</span>, <span class="author">Stein LD</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uitterlinden A</span>, <span class="author">Walker M</span>, <span class="author">Wareham NJ</span>, <span class="author">Watanabe RM</span>, <span class="author">Abecasis GR</span>, <span class="author">Boehm BO</span>, <span class="author">Campbell H</span>, <span class="author">Daly MJ</span>, <span class="author">Hattersley AT</span>, <span class="author">Hu FB</span>, <span class="author">Meigs JB</span>, <span class="author">Pankow JS</span>, <span class="author">Pedersen O</span>, <span class="author">Wichmann HE</span>, <span class="author">Barroso I</span>, <span class="author">Florez JC</span>, <span class="author">Frayling TM</span>, <span class="author">Groop L</span>, <span class="author">Sladek R</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Wilson JF</span>, <span class="author">Illig T</span>, <span class="author">Froguel P</span>, <span class="author">van Duijn CM</span>, <span class="author">Stefansson K</span>, <span class="author">Altshuler D</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span>, <span class="author">Soranzo N</span>, <span class="author">Wheeler E</span>, <span class="author">Glazer NL</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">M&auml;gi R</span>, <span class="author">Randall J</span>, <span class="author">Johnson T</span>, <span class="author">Elliott P</span>, <span class="author">Rybin D</span>, <span class="author">Henneman P</span>, <span class="author">Dehghan A</span>, <span class="author">Hottenga JJ</span>, <span class="author">Song K</span>, <span class="author">Goel A</span>, <span class="author">Egan JM</span>, <span class="author">Lajunen T</span>, <span class="author">Doney A</span>, <span class="author">Kanoni S</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Kumari M</span>, <span class="author">Timpson NJ</span>, <span class="author">Zabena C</span>, <span class="author">Ingelsson E</span>, <span class="author">An P</span>, <span class="author">O&#39;Connell J</span>, <span class="author">Luan J</span>, <span class="author">Elliott A</span>, <span class="author">McCarroll SA</span>, <span class="author">Roccasecca RM</span>, <span class="author">Pattou F</span>, <span class="author">Sethupathy P</span>, <span class="author">Ariyurek Y</span>, <span class="author">Barter P</span>, <span class="author">Beilby JP</span>, <span class="author">Ben-Shlomo Y</span>, <span class="author">Bergmann S</span>, <span class="author">Bochud M</span>, <span class="author">Bonnefond A</span>, <span class="author">Borch-Johnsen K</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Brunner E</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Chen YD</span>, <span class="author">Chines P</span>, <span class="author">Clarke R</span>, <span class="author">Coin LJ</span>, <span class="author">Cooper MN</span>, <span class="author">Crisponi L</span>, <span class="author">Day IN</span>, <span class="author">de Geus EJ</span>, <span class="author">Delplanque J</span>, <span class="author">Fedson AC</span>, <span class="author">Fischer-Rosinsky A</span>, <span class="author">Forouhi NG</span>, <span class="author">Frants R</span>, <span class="author">Franzosi MG</span>, <span class="author">Galan P</span>, <span class="author">Goodarzi MO</span>, <span class="author">Graessler J</span>, <span class="author">Grundy S</span>, <span class="author">Gwilliam R</span>, <span class="author">Hallmans G</span>, <span class="author">Hammond N</span>, <span class="author">Han X</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hayward C</span>, <span class="author">Heath SC</span>, <span class="author">Hercberg S</span>, <span class="author">Hicks AA</span>, <span class="author">Hillman DR</span>, <span class="author">Hingorani AD</span>, <span class="author">Hui J</span>, <span class="author">Hung J</span>, <span class="author">Jula A</span>, <span class="author">Kaakinen M</span>, <span class="author">Kaprio J</span>, <span class="author">Kesaniemi YA</span>, <span class="author">Kivimaki M</span>, <span class="author">Knight B</span>, <span class="author">Koskinen S</span>, <span class="author">Kovacs P</span>, <span class="author">Kyvik KO</span>, <span class="author">Lathrop GM</span>, <span class="author">Lawlor DA</span>, <span class="author">Le Bacquer O</span>, <span class="author">Lecoeur C</span>, <span class="author">Li Y</span>, <span class="author">Mahley R</span>, <span class="author">Mangino M</span>, <span class="author">Manning AK</span>, <span class="author">Mart&iacute;nez-Larrad MT</span>, <span class="author">McAteer JB</span>, <span class="author">McPherson R</span>, <span class="author">Meisinger C</span>, <span class="author">Melzer D</span>, <span class="author">Meyre D</span>, <span class="author">Mitchell BD</span>, <span class="author">Mukherjee S</span>, <span class="author">Naitza S</span>, <span class="author">Neville MJ</span>, <span class="author">Oostra BA</span>, <span class="author">Orr&ugrave; M</span>, <span class="author">Pakyz R</span>, <span class="author">Paolisso G</span>, <span class="author">Pattaro C</span>, <span class="author">Pearson D</span>, <span class="author">Peden JF</span>, <span class="author">Pedersen NL</span>, <span class="author">Perola M</span>, <span class="author">Pfeiffer AF</span>, <span class="author">Pichler I</span>, <span class="author">Polasek O</span>, <span class="author">Posthuma D</span>, <span class="author">Potter SC</span>, <span class="author">Pouta A</span>, <span class="author">Province MA</span>, <span class="author">Psaty BM</span>, <span class="author">Rayner NW</span>, <span class="author">Rice K</span>, <span class="author">Ripatti S</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rolandsson O</span>, <span class="author">Sandbaek A</span>, <span class="author">Sandhu M</span>, <span class="author">Sanna S</span>, <span class="author">Sayer AA</span>, <span class="author">Scheet P</span>, <span class="author">Seedorf U</span>, <span class="author">Sharp SJ</span>, <span class="author">Shields B</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Silveira A</span>, <span class="author">Simpson L</span>, <span class="author">Singleton A</span>, <span class="author">Smith NL</span>, <span class="author">Sovio U</span>, <span class="author">Swift A</span>, <span class="author">Syddall H</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Tanaka T</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Dijk KW</span>, <span class="author">Varma D</span>, <span class="author">Visvikis-Siest S</span>, <span class="author">Vitart V</span>, <span class="author">Vogelzangs N</span>, <span class="author">Waeber G</span>, <span class="author">Wagner PJ</span>, <span class="author">Walley A</span>, <span class="author">Ward KL</span>, <span class="author">Watkins H</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Witteman JC</span>, <span class="author">Yarnell JW</span>, <span class="author">Zelenika D</span>, <span class="author">Zethelius B</span>, <span class="author">Zhai G</span>, <span class="author">Zhao JH</span>, <span class="author">Zillikens MC</span>, <span class="author">Borecki IB</span>, <span class="author">Loos RJ</span>, <span class="author">Meneton P</span>, <span class="author">Magnusson PK</span>, <span class="author">Nathan DM</span>, <span class="author">Williams GH</span>, <span class="author">Silander K</span>, <span class="author">Salomaa V</span>, <span class="author">Smith GD</span>, <span class="author">Bornstein SR</span>, <span class="author">Schwarz P</span>, <span class="author">Spranger J</span>, <span class="author">Karpe F</span>, <span class="author">Shuldiner AR</span>, <span class="author">Cooper C</span>, <span class="author">Dedoussis GV</span>, <span class="author">Serrano-R&iacute;os M</span>, <span class="author">Lind L</span>, <span class="author">Palmer LJ</span>, <span class="author">Franks PW</span>, <span class="author">Ebrahim S</span>, <span class="author">Marmot M</span>, <span class="author">Kao WH</span>, <span class="author">Pramstaller PP</span>, <span class="author">Wright AF</span>, <span class="author">Stumvoll M</span>, <span class="author">Hamsten A</span>, <span class="author">Buchanan TA</span>, <span class="author">Valle TT</span>, <span class="author">Rotter JI</span>, <span class="author">Siscovick DS</span>, <span class="author">Penninx BW</span>, <span class="author">Boomsma DI</span>, <span class="author">Deloukas P</span>, <span class="author">Spector TD</span>, <span class="author">Ferrucci L</span>, <span class="author">Cao A</span>, <span class="author">Scuteri A</span>, <span class="author">Schlessinger D</span>, <span class="author">Uda M</span>, <span class="author">Ruokonen A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Waterworth DM</span>, <span class="author">Vollenweider P</span>, <span class="author">Peltonen L</span>, <span class="author">Mooser V</span> and <span class="author">Sladek R</span></p>
<p class="affiliation">Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, North Carolina, United States of America. <a href="mailto:%6E%61%6C%6C%72%65%64%40%77%66%75%62%6D%63.%65%64%75">&#110;&#97;&#108;&#108;&#114;&#101;&#100;&#64;&#119;&#102;&#117;&#98;&#109;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract">African Americans are disproportionately affected by type 2 diabetes (T2DM) yet few studies have examined T2DM using genome-wide association approaches in this ethnicity. The aim of this study was to identify genes associated with T2DM in the African American population. We performed a Genome Wide Association Study (GWAS) using the Affymetrix 6.0 array in 965 African-American cases with T2DM and end-stage renal disease (T2DM-ESRD) and 1029 population-based controls. The most significant SNPs (n&#x200A;=&#x200A;550 independent loci) were genotyped in a replication cohort and 122 SNPs (n&#x200A;=&#x200A;98 independent loci) were further tested through genotyping three additional validation cohorts followed by meta-analysis in all five cohorts totaling 3,132 cases and 3,317 controls. Twelve SNPs had evidence of association in the GWAS (P&lt;0.0071), were directionally consistent in the Replication cohort and were associated with T2DM in subjects without nephropathy (P&lt;0.05). Meta-analysis in all cases and controls revealed a single SNP reaching genome-wide significance (P&lt;2.5&times;10(-8)). SNP rs7560163 (P&#x200A;=&#x200A;7.0&times;10(-9), OR (95% CI)&#x200A;=&#x200A;0.75 (0.67-0.84)) is located intergenically between RND3 and RBM43. Four additional loci (rs7542900, rs4659485, rs2722769 and rs7107217) were associated with T2DM (P&lt;0.05) and reached more nominal levels of significance (P&lt;2.5&times;10(-5)) in the overall analysis and may represent novel loci that contribute to T2DM. We have identified novel T2DM-susceptibility variants in the African-American population. Notably, T2DM risk was associated with the major allele and implies an interesting genetic architecture in this population. These results suggest that multiple loci underlie T2DM susceptibility in the African-American population and that these loci are distinct from those identified in other ethnic populations.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/07/008/23674</span>; Department of Health: <span class="grant">DHCS/07/07/008</span>; Medical Research Council: <span class="grant">G0001164</span>, <span class="grant">G0801056</span>, <span class="grant">G0902037</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179474</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1015</span>; NCRR NIH HHS: <span class="grant">M01 RR07122</span>; NHLBI NIH HHS: <span class="grant">K99 HL098459</span>, <span class="grant">R01 HL56266</span>; NIDDK NIH HHS: <span class="grant">K24 DK080140</span>, <span class="grant">K99 DK081350</span>, <span class="grant">P30 DK063491</span>, <span class="grant">R00 DK081350</span>, <span class="grant">R01 DK053591</span>, <span class="grant">R01 DK066358</span>, <span class="grant">R01 DK070941</span>; PHS HHS: <span class="grant">HHSC268200782096C</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">e29202</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22238593">22238593</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3251563">3251563</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0029202">10.1371/journal.pone.0029202</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From Genetic Discovery to Future Personalized Health Research.</h4>
<p class="authors"><span class="author">Palotie A</span>, <span class="author">Widen E</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Cambridge, UK; Institute of Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki Finland; the Broad Institute of MIT and Harvard, Cambridge MA. Electronic address: <a href="mailto:%61%61%72%6E%6F.%70%61%6C%6F%74%69%65%40%68%65%6C%73%69%6E%6B%69.%66%69">&#97;&#97;&#114;&#110;&#111;&#46;&#112;&#97;&#108;&#111;&#116;&#105;&#101;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a>.</p>
<p class="abstract">During the past ten years the field of human disease genetics has made major leaps, including the completion of the Human Genome Project, the HapMap Project, the development of the genome-wide association (GWA) studies to identify common disease-predisposing variants, and the introduction of large- scale whole-genome and whole-exome sequencing studies. The introduction of new technologies has enabled researchers to utilize novel study designs to tackle previously unexplored research questions in human genomics. These new types of studies typically need large sample sizes to overcome the multiple testing challenges caused by the huge number of interrogated genetic variants. As a consequence, large consortia-studies are at present the default in disease genetics research. The systematic planning of the GWA-studies was a key element in the success of the approach. Similar planning and rigor in statistical inferences will likely be beneficial also to future sequencing studies. Already today, the next-generation exome sequencing has lead to the identification of a number of genes underlying Mendelian diseases. In spite of the clear benefits, the method has proven more challenging than anticipated. In the case of complex diseases, next-generation sequencing aims to identify disease-associated low-frequency alleles. However, their robust detection will require very large study samples, even larger than in the case of the GWA-studies. This has stimulated study designs that capitalize on enriching sets of low-frequency alleles, e.g studies focusing on population isolates such as Finland or Iceland. One example is the collaborative SISu Project (Sequencing Initiative Suomi) that aims to provide near complete genome variation information from Finnish study samples and pave the way for large, nationwide genome health initiative studies.</p>
<p class="publication"><span class="title">New biotechnology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23165095" rel="external">23165095</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.nbt.2012.11.013" rel="external">10.1016/j.nbt.2012.11.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assignment of protein interactions from affinity purification/mass spectrometry data.</h4>
<p class="authors"><span class="author">Pardo M</span> and <span class="author">Choudhary JS</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute , Wellcome Trust Genome Campus, Hinxton, CB10 1SA Cambridgeshire, United Kingdom. <a href="mailto:%6D%70%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#112;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The combination of affinity purification with mass spectrometry analysis has become the method of choice for protein complex characterization. With the improved performance of mass spectrometry technology, the sensitivity of the analyses is increasing, probing deeper into molecular interactions and yielding longer lists of proteins. These identify not only core complex subunits but also the more inaccessible proteins that interact weakly or transiently. Alongside them, contaminant proteins, which are often abundant proteins in the cell, tend to be recovered in affinity experiments because they bind nonspecifically and with low affinity to matrix, tag, and/or antibody. The challenge now lies in discriminating nonspecific binders from true interactors, particularly at the low level and in a larger scale. This review aims to summarize the variety of methods that have been used to distinguish contaminants from specific interactions in the past few years, ranging from manual elimination using heuristic rules to more sophisticated probabilistic scoring approaches. We aim to give awareness on the processing that takes place before an interaction list is reported and on the different types of list curation approaches suited to the different experiments.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643/Z/06/Z</span></p>
<p class="publication"><span class="title">Journal of proteome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="number">3</span>;<span class="pagerange">1462-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22283744">22283744</a>; DOI: <a href="http://dx.doi.org/10.1021%2Fpr2011632">10.1021/pr2011632</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of the classical Bordetella subspecies: the evolution and exchange of virulence-associated diversity amongst closely related pathogens.</h4>
<p class="authors"><span class="author">Park J</span>, <span class="author">Zhang Y</span>, <span class="author">Buboltz AM</span>, <span class="author">Zhang X</span>, <span class="author">Schuster SC</span>, <span class="author">Ahuja U</span>, <span class="author">Liu M</span>, <span class="author">Miller JF</span>, <span class="author">Sebaihia M</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span> and <span class="author">Harvill ET</span></p>
<p class="affiliation">Department of Veterinary and Biomedical Sciences, The Pennsylvania State University, University Park, USA.</p>
<p class="abstract">Background: The classical Bordetella subspecies are phylogenetically closely related, yet differ in some of the most interesting and important characteristics of pathogens, such as host range, virulence and persistence. The compelling picture from previous comparisons of the three sequenced genomes was of genome degradation, with substantial loss of genome content (up to 24%) associated with adaptation to humans.

Results: For a more comprehensive picture of lineage evolution, we employed comparative genomic and phylogenomic analyses using seven additional diverse, newly sequenced Bordetella isolates. Genome-wide single nucleotide polymorphism (SNP) analysis supports a reevaluation of the phylogenetic relationships between the classical Bordetella subspecies, and suggests a closer link between ovine and human B. parapertussis lineages than has been previously proposed. Comparative analyses of genome content revealed that only 50% of the pan-genome is conserved in all strains, reflecting substantial diversity of genome content in these closely related pathogens that may relate to their different host ranges, virulence and persistence characteristics. Strikingly, these analyses suggest possible horizontal gene transfer (HGT) events in multiple loci encoding virulence factors, including O-antigen and pertussis toxin (Ptx). Segments of the pertussis toxin locus (ptx) and its secretion system locus (ptl) appear to have been acquired by the classical Bordetella subspecies and are divergent in different lineages, suggesting functional divergence in the classical Bordetellae.

Conclusions: Together, these observations, especially in key virulence factors, reveal that multiple mechanisms, such as point mutations, gain or loss of genes, as well as HGTs, contribute to the substantial phenotypic diversity of these versatile subspecies in various hosts.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">5R01GM083113</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">545</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23051057">23051057</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3533505">3533505</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-545">10.1186/1471-2164-13-545</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Transcriptomic Analysis of Echinococcus granulosus Larval Stages: Implications for Parasite Biology and Host Adaptation.</h4>
<p class="authors"><span class="author">Parkinson J</span>, <span class="author">Wasmuth JD</span>, <span class="author">Salinas G</span>, <span class="author">Bizarro CV</span>, <span class="author">Sanford C</span>, <span class="author">Berriman M</span>, <span class="author">Ferreira HB</span>, <span class="author">Zaha A</span>, <span class="author">Blaxter ML</span>, <span class="author">Maizels RM</span> and <span class="author">Fern&aacute;ndez C</span></p>
<p class="affiliation">Program in Molecular Structure and Function, Hospital for Sick Children, University of Toronto, Toronto, Canada.</p>
<p class="abstract">Background: The cestode Echinococcus granulosus - the agent of cystic echinococcosis, a zoonosis affecting humans and domestic animals worldwide - is an excellent model for the study of host-parasite cross-talk that interfaces with two mammalian hosts. To develop the molecular analysis of these interactions, we carried out an EST survey of E. granulosus larval stages. We report the salient features of this study with a focus on genes reflecting physiological adaptations of different parasite stages.

We generated âˆ¼10,000 ESTs from two sets of full-length enriched libraries (derived from oligo-capped and trans-spliced cDNAs) prepared with three parasite materials: hydatid cyst wall, larval worms (protoscoleces), and pepsin/H(+)-activated protoscoleces. The ESTs were clustered into 2700 distinct gene products. In the context of the biology of E. granulosus, our analyses reveal: (i) a diverse group of abundant long non-protein coding transcripts showing homology to a middle repetitive element (EgBRep) that could either be active molecular species or represent precursors of small RNAs (like piRNAs); (ii) an up-regulation of fermentative pathways in the tissue of the cyst wall; (iii) highly expressed thiol- and selenol-dependent antioxidant enzyme targets of thioredoxin glutathione reductase, the functional hub of redox metabolism in parasitic flatworms; (iv) candidate apomucins for the external layer of the tissue-dwelling hydatid cyst, a mucin-rich structure that is critical for survival in the intermediate host; (v) a set of tetraspanins, a protein family that appears to have expanded in the cestode lineage; and (vi) a set of platyhelminth-specific gene products that may offer targets for novel pan-platyhelminth drug development.

This survey has greatly increased the quality and the quantity of the molecular information on E. granulosus and constitutes a valuable resource for gene prediction on the parasite genome and for further genomic and proteomic analyses focused on cestodes and platyhelminths.</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e1897</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23209850" rel="external">23209850</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3510090" rel="external">3510090</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001897" rel="external">10.1371/journal.pntd.0001897</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of celiac disease genes with inflammatory bowel disease in Finnish and Swedish patients.</h4>
<p class="authors"><span class="author">Parmar AS</span>, <span class="author">Lappalainen M</span>, <span class="author">Paavola-Sakki P</span>, <span class="author">Halme L</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Turunen U</span>, <span class="author">Kontula K</span>, <span class="author">Aromaa A</span>, <span class="author">Salomaa V</span>, <span class="author">Peltonen L</span>, <span class="author">Halfvarson J</span>, <span class="author">T&ouml;rkvist L</span>, <span class="author">D&#39;Amato M</span>, <span class="author">Saavalainen P</span> and <span class="author">Einarsdottir E</span></p>
<p class="affiliation">1] Research Program for Molecular Medicine, University of Helsinki, Helsinki, Finland [2] Department of Medical Genetics, Haartman Institute, Biomedicum Helsinki, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Some genetic loci may affect susceptibility to multiple immune system-related diseases. In the current study, we investigated whether the known susceptibility loci for celiac disease (CelD) also associate with Crohn&#39;s disease (CD) and/or ulcerative colitis (UC), the two main forms of inflammatory bowel disease (IBD), in Finnish patients. A total of 45 genetic markers were genotyped in a Finnish data set comprising 699 IBD patients and 2482 controls. Single-marker association with IBD and its subphenotypes was tested. A meta-analysis with a Swedish UC data set was also performed. A total of 12 single-nucleotide polymorphisms associated with CD and/or UC (P&lt;0.05). In the subphenotype analysis, rs6974491-ELMO1 (P=0.0002, odds ratio (OR): 2.20) and rs2298428-UBE2L3 (P=5.44 &times; 10(-5), OR: 2.59) associated with pediatric UC and CD, respectively. In the meta-analysis, rs4819388-ICOSLG (P=0.00042, OR: 0.79) associated with UC. In the subphenotype meta-analysis, rs1738074-TAGAP (P=7.40 &times; 10(-5), OR: 0.61), rs6974491-ELMO1 (P=0.00052, OR: 1.73) and rs4819388-ICOSLG (P=0.00019, OR: 0.75) associated with familial UC, pediatric UC and sporadic UC, respectively. Multiple CelD risk loci also confer susceptibility for CD and/or UC in the Finnish and Swedish populations. Certain genetic risk variants may furthermore predispose an individual for developing a particular disease phenotype.Genes and Immunity advance online publication, 17 May 2012; doi:10.1038/gene.2012.21.</p>
<p class="publication"><span class="title">Genes and immunity</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22592522" rel="external">22592522</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fgene.2012.21" rel="external">10.1038/gene.2012.21</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Extent, causes, and consequences of small RNA expression variation in human adipose tissue.</h4>
<p class="authors"><span class="author">Parts L</span>, <span class="author">Hedman AK</span>, <span class="author">Keildson S</span>, <span class="author">Knights AJ</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">van de Bunt M</span>, <span class="author">Guerra-Assun&ccedil;&atilde;o JA</span>, <span class="author">Bartonicek N</span>, <span class="author">van Dongen S</span>, <span class="author">M&auml;gi R</span>, <span class="author">Nisbet J</span>, <span class="author">Barrett A</span>, <span class="author">Rantalainen M</span>, <span class="author">Nica AC</span>, <span class="author">Quail MA</span>, <span class="author">Small KS</span>, <span class="author">Glass D</span>, <span class="author">Enright AJ</span>, <span class="author">Winn J</span>, <span class="author">MuTHER Consortium</span>, <span class="author">Deloukas P</span>, <span class="author">Dermitzakis ET</span>, <span class="author">McCarthy MI</span>, <span class="author">Spector TD</span>, <span class="author">Durbin R</span> and <span class="author">Lindgren CM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">Small RNAs are functional molecules that modulate mRNA transcripts and have been implicated in the aetiology of several common diseases. However, little is known about the extent of their variability within the human population. Here, we characterise the extent, causes, and effects of naturally occurring variation in expression and sequence of small RNAs from adipose tissue in relation to genotype, gene expression, and metabolic traits in the MuTHER reference cohort. We profiled the expression of 15 to 30 base pair RNA molecules in subcutaneous adipose tissue from 131 individuals using high-throughput sequencing, and quantified levels of 591 microRNAs and small nucleolar RNAs. We identified three genetic variants and three RNA editing events. Highly expressed small RNAs are more conserved within mammals than average, as are those with highly variable expression. We identified 14 genetic loci significantly associated with nearby small RNA expression levels, seven of which also regulate an mRNA transcript level in the same region. In addition, these loci are enriched for variants significant in genome-wide association studies for body mass index. Contrary to expectation, we found no evidence for negative correlation between expression level of a microRNA and its target mRNAs. Trunk fat mass, body mass index, and fasting insulin were associated with more than twenty small RNA expression levels each, while fasting glucose had no significant associations. This study highlights the similar genetic complexity and shared genetic control of small RNA and mRNA transcripts, and gives a quantitative picture of small RNA expression variation in the human population.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">e1002704</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22589741" rel="external">22589741</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3349731" rel="external">3349731</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002704" rel="external">10.1371/journal.pgen.1002704</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association studies identifies three new risk loci for atopic dermatitis.</h4>
<p class="authors"><span class="author">Paternoster L</span>, <span class="author">Standl M</span>, <span class="author">Chen CM</span>, <span class="author">Ramasamy A</span>, <span class="author">B&oslash;nnelykke K</span>, <span class="author">Duijts L</span>, <span class="author">Ferreira MA</span>, <span class="author">Alves AC</span>, <span class="author">Thyssen JP</span>, <span class="author">Albrecht E</span>, <span class="author">Baurecht H</span>, <span class="author">Feenstra B</span>, <span class="author">Sleiman PM</span>, <span class="author">Hysi P</span>, <span class="author">Warrington NM</span>, <span class="author">Curjuric I</span>, <span class="author">Myhre R</span>, <span class="author">Curtin JA</span>, <span class="author">Groen-Blokhuis MM</span>, <span class="author">Kerkhof M</span>, <span class="author">S&auml;&auml;f A</span>, <span class="author">Franke A</span>, <span class="author">Ellinghaus D</span>, <span class="author">F&ouml;lster-Holst R</span>, <span class="author">Dermitzakis E</span>, <span class="author">Montgomery SB</span>, <span class="author">Prokisch H</span>, <span class="author">Heim K</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">Pekkanen J</span>, <span class="author">Blakemore AI</span>, <span class="author">Buxton JL</span>, <span class="author">Kaakinen M</span>, <span class="author">Duffy DL</span>, <span class="author">Madden PA</span>, <span class="author">Heath AC</span>, <span class="author">Montgomery GW</span>, <span class="author">Thompson PJ</span>, <span class="author">Matheson MC</span>, <span class="author">Le Sou&euml;f P</span>, <span class="author">Australian Asthma Genetics Consortium (AAGC)</span>, <span class="author">St Pourcain B</span>, <span class="author">Smith GD</span>, <span class="author">Henderson J</span>, <span class="author">Kemp JP</span>, <span class="author">Timpson NJ</span>, <span class="author">Deloukas P</span>, <span class="author">Ring SM</span>, <span class="author">Wichmann HE</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Novak N</span>, <span class="author">Klopp N</span>, <span class="author">Rodr&iacute;guez E</span>, <span class="author">McArdle W</span>, <span class="author">Linneberg A</span>, <span class="author">Menn&eacute; T</span>, <span class="author">Nohr EA</span>, <span class="author">Hofman A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">Rivadeneira F</span>, <span class="author">de Jongste JC</span>, <span class="author">van der Valk RJ</span>, <span class="author">Wjst M</span>, <span class="author">Jogi R</span>, <span class="author">Geller F</span>, <span class="author">Boyd HA</span>, <span class="author">Murray JC</span>, <span class="author">Kim C</span>, <span class="author">Mentch F</span>, <span class="author">March M</span>, <span class="author">Mangino M</span>, <span class="author">Spector TD</span>, <span class="author">Bataille V</span>, <span class="author">Pennell CE</span>, <span class="author">Holt PG</span>, <span class="author">Sly P</span>, <span class="author">Tiesler CM</span>, <span class="author">Thiering E</span>, <span class="author">Illig T</span>, <span class="author">Imboden M</span>, <span class="author">Nystad W</span>, <span class="author">Simpson A</span>, <span class="author">Hottenga JJ</span>, <span class="author">Postma D</span>, <span class="author">Koppelman GH</span>, <span class="author">Smit HA</span>, <span class="author">S&ouml;derh&auml;ll C</span>, <span class="author">Chawes B</span>, <span class="author">Kreiner-M&oslash;ller E</span>, <span class="author">Bisgaard H</span>, <span class="author">Mel&eacute;n E</span>, <span class="author">Boomsma DI</span>, <span class="author">Custovic A</span>, <span class="author">Jacobsson B</span>, <span class="author">Probst-Hensch NM</span>, <span class="author">Palmer LJ</span>, <span class="author">Glass D</span>, <span class="author">Hakonarson H</span>, <span class="author">Melbye M</span>, <span class="author">Jarvis DL</span>, <span class="author">Jaddoe VW</span>, <span class="author">Gieger C</span>, <span class="author">Genetics of Overweight Young Adults (GOYA) Consortium</span>, <span class="author">Strachan DP</span>, <span class="author">Martin NG</span>, <span class="author">Jarvelin MR</span>, <span class="author">Heinrich J</span>, <span class="author">Evans DM</span>, <span class="author">Weidinger S</span> and <span class="author">EArly Genetics &amp;amp; Lifecourse Epidemiology (EAGLE) Consortium</span></p>
<p class="affiliation">Medical Research Council Centre for Causal Analyses in Translational Epidemiology, University of Bristol, Bristol, UK.</p><p class="abstract">Atopic dermatitis (AD) is a commonly occurring chronic skin disease with high heritability. Apart from filaggrin (FLG), the genes influencing atopic dermatitis are largely unknown. We conducted a genome-wide association meta-analysis of 5,606 affected individuals and 20,565 controls from 16 population-based cohorts and then examined the ten most strongly associated new susceptibility loci in an additional 5,419 affected individuals and 19,833 controls from 14 studies. Three SNPs reached genome-wide significance in the discovery and replication cohorts combined, including rs479844 upstream of OVOL1 (odds ratio (OR) = 0.88, P = 1.1 &times; 10(-13)) and rs2164983 near ACTL9 (OR = 1.16, P = 7.1 &times; 10(-9)), both of which are near genes that have been implicated in epidermal proliferation and differentiation, as well as rs2897442 in KIF3A within the cytokine cluster at 5q31.1 (OR = 1.11, P = 3.8 &times; 10(-8)). We also replicated association with the FLG locus and with two recently identified association signals at 11q13.5 (rs7927894; P = 0.008) and 20q13.33 (rs6010620; P = 0.002). Our results underline the importance of both epidermal barrier function and immune dysregulation in atopic dermatitis pathogenesis.</p>
<p class="grants">Funded by: NIDA NIH HHS: <span class="grant">R01 DA012854-10</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">2</span>;<span class="pagerange">187-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22197932" rel="external">22197932</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3272375" rel="external">3272375</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1017" rel="external">10.1038/ng.1017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The newly described mecA homologue, mecALGA251, is present in methicillin-resistant Staphylococcus aureus isolates from a diverse range of host species.</h4>
<p class="authors"><span class="author">Paterson GK</span>, <span class="author">Larsen AR</span>, <span class="author">Robb A</span>, <span class="author">Edwards GE</span>, <span class="author">Pennycott TW</span>, <span class="author">Foster G</span>, <span class="author">Mot D</span>, <span class="author">Hermans K</span>, <span class="author">Baert K</span>, <span class="author">Peacock SJ</span>, <span class="author">Parkhill J</span>, <span class="author">Zadoks RN</span> and <span class="author">Holmes MA</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Madingley Road, Cambridge CB3 0ES, UK.</p>
<p class="abstract">Objectives: A previously unidentified mecA homologue, mecA(LGA251), has recently been described in methicillin-resistant Staphylococcus aureus (MRSA) from humans and dairy cattle. The origin and epidemiology of this novel homologue are unclear. The objective of this study was to provide basic descriptive information of MRSA isolates harbouring mecA(LGA251) from a range of host animal species.

Methods: A number of S. aureus isolates from historical animal isolate collections were chosen for investigation based on their similarity to known mecA(LGA251) MRSA isolates. The presence of mecA(LGA251) was determined using a multiplex PCR and antimicrobial susceptibility testing performed by disc diffusion.

Results: MRSA harbouring mecA(LGA251) were found in isolates from a domestic dog, brown rats, a rabbit, a common seal, sheep and a chaffinch. All of the isolates were phenotypically MRSA, although this depended on which test was used; some isolates would be considered susceptible with certain assays. All isolates were susceptible to linezolid, rifampicin, kanamycin, norfloxacin, erythromycin, clindamycin, fusidic acid, tetracycline, trimethoprim/sulfamethoxazole and mupirocin. Five multilocus sequence types were represented (2273, 130, 425, 1764 and 1245) and six spa types (t208, t6293, t742, t6594, t7914 and t843).

Conclusions: The discovery of MRSA isolates possessing mecA(LGA251) from a diverse range of host species, including different taxonomic classes, has important implications for the diagnosis of MRSA in these species and our understanding of the epidemiology of this novel mecA homologue.</p>
<p class="publication"><span class="title">The Journal of antimicrobial chemotherapy</span>&nbsp;<span class="year">2012</span>;<span class="volume">67</span>;<span class="number">12</span>;<span class="pagerange">2809-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22941897" rel="external">22941897</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3494845" rel="external">3494845</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fjac%2Fdks329" rel="external">10.1093/jac/dks329</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">First detection of livestock-associated meticillin-resistant Staphylococcus aureus CC398 in bulk tank milk in the United Kingdom, January to July 2012.</h4>
<p class="authors"><span class="author">Paterson GK</span>, <span class="author">Larsen J</span>, <span class="author">Harrison EM</span>, <span class="author">Larsen AR</span>, <span class="author">Morgan FJ</span>, <span class="author">Peacock SJ</span>, <span class="author">Parkhill J</span>, <span class="author">Zadoks RN</span> and <span class="author">Holmes MA</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Livestock-associated meticillin-resistant Staphylococcus aureus belonging to clonal complex 398 (LA-MRSA CC398) is an important cause of zoonotic infections in several countries, but there is only a single published report of this lineage from the United Kingdom (UK). Here, we describe the isolation of LA-MRSA CC398 from bulk tank milk from five geographically dispersed farms in the UK. Our findings suggest that LA-MRSA CC398 is established in livestock in the UK. Awareness of the potential occupational risks and surveillance in other food-producing animal species should be promoted.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1001787/1</span></p>
<p class="publication"><span class="title">Euro surveillance : bulletin Europ&eacute;en sur les maladies transmissibles = European communicable disease bulletin</span>&nbsp;<span class="year">2012</span>;<span class="volume">17</span>;<span class="number">50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23241232">23241232</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The GENCODE pseudogene resource.</h4>
<p class="authors"><span class="author">Pei B</span>, <span class="author">Sisu C</span>, <span class="author">Frankish A</span>, <span class="author">Howald C</span>, <span class="author">Habegger L</span>, <span class="author">Mu XJ</span>, <span class="author">Harte R</span>, <span class="author">Balasubramanian S</span>, <span class="author">Tanzer A</span>, <span class="author">Diekhans M</span>, <span class="author">Reymond A</span>, <span class="author">Hubbard TJ</span>, <span class="author">Harrow J</span> and <span class="author">Gerstein MB</span></p>
<p class="affiliation">Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT 06520, USA.</p><p class="abstract"><strong>Background</strong>: Pseudogenes have long been considered as nonfunctional genomic sequences. However, recent evidence suggests that many of them might have some form of biological activity, and the possibility of functionality has increased interest in their accurate annotation and integration with functional genomics data.</p><p class="abstract"><strong>Results</strong>: As part of the GENCODE annotation of the human genome, we present the first genome-wide pseudogene assignment for protein-coding genes, based on both large-scale manual annotation and in silico pipelines. A key aspect of this coupled approach is that it allows us to identify pseudogenes in an unbiased fashion as well as untangle complex events through manual evaluation. We integrate the pseudogene annotations with the extensive ENCODE functional genomics information. In particular, we determine the expression level, transcription-factor and RNA polymerase II binding, and chromatin marks associated with each pseudogene. Based on their distribution, we develop simple statistical models for each type of activity, which we validate with large-scale RT-PCR-Seq experiments. Finally, we compare our pseudogenes with conservation and variation data from primate alignments and the 1000 Genomes project, producing lists of pseudogenes potentially under selection.</p><p class="abstract"><strong>Conclusions</strong>: At one extreme, some pseudogenes possess conventional characteristics of functionality; these may represent genes that have recently died. On the other hand, we find interesting patterns of partial activity, which may suggest that dead genes are being resurrected as functioning non-coding RNAs. The activity data of each pseudogene are stored in an associated resource, psiDR, which will be useful for the initial identification of potentially functional pseudogenes.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">77198/Z/05/Z</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">9</span>;<span class="pagerange">R51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22951037">22951037</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3491395">3491395</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-9-r51">10.1186/gb-2012-13-9-r51</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Impact of restricted marital practices on genetic variation in an endogamous Gujarati group.</h4>
<p class="authors"><span class="author">Pemberton TJ</span>, <span class="author">Li FY</span>, <span class="author">Hanson EK</span>, <span class="author">Mehta NU</span>, <span class="author">Choi S</span>, <span class="author">Ballantyne J</span>, <span class="author">Belmont JW</span>, <span class="author">Rosenberg NA</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Patel PI</span></p>
<p class="affiliation">Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA 90033, USA. <a href="mailto:%74%72%65%76%6F%72%70%40%73%74%61%6E%66%6F%72%64.%65%64%75">&#116;&#114;&#101;&#118;&#111;&#114;&#112;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Recent studies have examined the influence on patterns of human genetic variation of a variety of cultural practices. In India, centuries-old marriage customs have introduced extensive social structuring into the contemporary population, potentially with significant consequences for genetic variation. Social stratification in India is evident as social classes that are defined by endogamous groups known as castes. Within a caste, there exist endogamous groups known as gols (marriage circles), each of which comprises a small number of exogamous gotra (lineages). Thus, while consanguinity is strictly avoided and some randomness in mate selection occurs within the gol, gene flow is limited with groups outside the gol. Gujarati Patels practice this form of &quot;exogamic endogamy.&quot; We have analyzed genetic variation in one such group of Gujarati Patels, the Chha Gaam Patels (CGP), who comprise individuals from six villages. Population structure analysis of 1,200 autosomal loci offers support for the existence of distinctive multilocus genotypes in the CGP with respect to both non-Gujaratis and other Gujaratis, and indicates that CGP individuals are genetically very similar. Analysis of Y-chromosomal and mitochondrial haplotypes provides support for both patrilocal and patrilineal practices within the gol, and a low-level of female gene flow into the gol. Our study illustrates how the practice of gol endogamy has introduced fine-scale genetic structure into the population of India, and contributes more generally to an understanding of the way in which marriage practices affect patterns of genetic variation.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA62528-01</span>; NCRR NIH HHS: <span class="grant">RR10600-01</span>, <span class="grant">RR14514-01</span>; NICHD NIH HHS: <span class="grant">P30 HD024064</span>; NIGMS NIH HHS: <span class="grant">GM081441</span>, <span class="grant">R01 GM081441</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of physical anthropology</span>&nbsp;<span class="year">2012</span>;<span class="volume">149</span>;<span class="number">1</span>;<span class="pagerange">92-103</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22729696">22729696</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3436606">3436606</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajpa.22101">10.1002/ajpa.22101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bioimage informatics: a new category in Bioinformatics.</h4>
<p class="authors"><span class="author">Peng H</span>, <span class="author">Bateman A</span>, <span class="author">Valencia A</span> and <span class="author">Wren JD</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">8</span>;<span class="pagerange">1057</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22399678" rel="external">22399678</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3324521" rel="external">3324521</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts111" rel="external">10.1093/bioinformatics/bts111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ChIP-seq and transcriptome analysis of the OmpR regulon of Salmonella enterica serovars Typhi and Typhimurium reveals accessory genes implicated in host colonisation.</h4>
<p class="authors"><span class="author">Perkins TT</span>, <span class="author">Davies MR</span>, <span class="author">Klemm EJ</span>, <span class="author">Rowley G</span>, <span class="author">Wileman T</span>, <span class="author">James K</span>, <span class="author">Keane T</span>, <span class="author">Maskell D</span>, <span class="author">Hinton JC</span>, <span class="author">Dougan G</span> and <span class="author">Kingsley RA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kingdom.</p>
<p class="abstract">OmpR is a multifunctional DNA binding regulator with orthologues in many enteric bacteria, that exhibits classical regulator activity as well as nucleoid-associated-protein-like characteristics. In the enteric pathogen Salmonella enterica, using chromatin immunoprecipitation of OmpR::FLAG and nucleotide sequencing, 43 putative OmpR-binding sites were identified in S. enterica serovar Typhi, 22 of which were associated with OmpR-regulated genes. Mutation of a sequence motif (TGTWACAW) that was associated with the putative OmpR-binding sites abrogated binding of OmpR::6xhis to the tviA upstream region. A core set of 31 orthologous genes were found to exhibit OmpR-dependent expression in both S. Typhi and S. Typhimurium. S. Typhimurium-encoded orthologues of two divergently transcribed OmpR regulated operons (SL1068-71 and SL1066-67) had a putative OmpR binding site in the inter-operon region in S. Typhi, and were characterized using in vitro and in vivo assays. These operons are widely distributed within Salmonella enterica but absent from the closely related Escherichia coli. SL1066 and SL1067 were required for growth on N-acetyl muramic acid as a sole carbon source. SL1068-71 exhibited sequence similarity to sialic acid uptake systems and contributed to colonization of the ileum and cecum in the streptomycin pretreated mouse model of colitis.</p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23190111" rel="external">23190111</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fmmi.12111" rel="external">10.1111/mmi.12111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary genetics of the human Rh blood group system.</h4>
<p class="authors"><span class="author">Perry GH</span>, <span class="author">Xue Y</span>, <span class="author">Smith RS</span>, <span class="author">Meyer WK</span>, <span class="author">Cal&#x131;&#x15F;kan M</span>, <span class="author">Yanez-Cuna O</span>, <span class="author">Lee AS</span>, <span class="author">Guti&eacute;rrez-Arcelus M</span>, <span class="author">Ober C</span>, <span class="author">Hollox EJ</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Lee C</span></p>
<p class="affiliation">Department of Anthropology, Pennsylvania State University, University Park, PA 16801, USA.</p><p class="abstract">The evolutionary history of variation in the human Rh blood group system, determined by variants in the RHD and RHCE genes, has long been an unresolved puzzle in human genetics. Prior to medical treatments and interventions developed in the last century, the D-positive (RhD positive) children of D-negative (RhD negative) women were at risk for hemolytic disease of the newborn, if the mother produced anti-D antibodies following sensitization to the blood of a previous D-positive child. Given the deleterious fitness consequences of this disease, the appreciable frequencies in European populations of the responsible RHD gene deletion variant (for example, 0.43 in our study) seem surprising. In this study, we used new molecular and genomic data generated from four HapMap population samples to test the idea that positive selection for an as-of-yet unknown fitness benefit of the RHD deletion may have offset the otherwise negative fitness effects of hemolytic disease of the newborn. We found no evidence that positive natural selection affected the frequency of the RHD deletion. Thus, the initial rise to intermediate frequency of the RHD deletion in European populations may simply be explained by genetic drift/founder effect, or by an older or more complex sweep that we are insufficiently powered to detect. However, our simulations recapitulate previous findings that selection on the RHD deletion is frequency dependent and weak or absent near 0.5. Therefore, once such a frequency was achieved, it could have been maintained by a relatively small amount of genetic drift. We unexpectedly observed evidence for positive selection on the C allele of RHCE in non-African populations (on chromosomes with intact copies of the RHD gene) in the form of an unusually high F( ST ) value and the high frequency of a single haplotype carrying the C allele. RhCE function is not well understood, but the C/c antigenic variant is clinically relevant and can result in hemolytic disease of the newborn, albeit much less commonly and severely than that related to the D-negative blood type. Therefore, the potential fitness benefits of the RHCE C allele are currently unknown but merit further exploration.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801123</span>, <span class="grant">GO801123</span>; NHGRI NIH HHS: <span class="grant">P41-HG004221</span>; NICHD NIH HHS: <span class="grant">R01-HD21244</span>; Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">131</span>;<span class="number">7</span>;<span class="pagerange">1205-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22367406">22367406</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3378649">3378649</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-012-1147-5">10.1007/s00439-012-1147-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clinically significant copy number alterations and complex rearrangements of MYB and NFIB in head and neck adenoid cystic carcinoma.</h4>
<p class="authors"><span class="author">Persson M</span>, <span class="author">Andr&eacute;n Y</span>, <span class="author">Moskaluk CA</span>, <span class="author">Frierson HF</span>, <span class="author">Cooke SL</span>, <span class="author">Futreal PA</span>, <span class="author">Kling T</span>, <span class="author">Nelander S</span>, <span class="author">Nordkvist A</span>, <span class="author">Persson F</span> and <span class="author">Stenman G</span></p>
<p class="affiliation">Sahlgrenska Cancer Center, Department of Pathology, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden.</p><p class="abstract">Adenoid cystic carcinoma (ACC) of the head and neck is a malignant tumor with poor long-term prognosis. Besides the recently identified MYB-NFIB fusion oncogene generated by a t(6;9) translocation, little is known about other genetic alterations in ACC. Using high-resolution, array-based comparative genomic hybridization, and massively paired-end sequencing, we explored genomic alterations in 40 frozen ACCs. Eighty-six percent of the tumors expressed MYB-NFIB fusion transcripts and 97% overexpressed MYB mRNA, indicating that MYB activation is a hallmark of ACC. Thirty-five recurrent copy number alterations (CNAs) were detected, including losses involving 12q, 6q, 9p, 11q, 14q, 1p, and 5q and gains involving 1q, 9p, and 22q. Grade III tumors had on average a significantly higher number of CNAs/tumor compared to Grade I and II tumors (P = 0.007). Losses of 1p, 6q, and 15q were associated with high-grade tumors, whereas losses of 14q were exclusively seen in Grade I tumors. The t(6;9) rearrangements were associated with a complex pattern of breakpoints, deletions, insertions, inversions, and for 9p also gains. Analyses of fusion-negative ACCs using high-resolution arrays and massively paired-end sequencing revealed that MYB may also be deregulated by other mechanisms in addition to gene fusion. Our studies also identified several down-regulated candidate tumor suppressor genes (CTNNBIP1, CASP9, PRDM2, and SFN) in 1p36.33-p35.3 that may be of clinical significance in high-grade tumors. Further, studies of these and other potential target genes may lead to the identification of novel driver genes in ACC.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2012</span>;<span class="volume">51</span>;<span class="number">8</span>;<span class="pagerange">805-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22505352">22505352</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.21965">10.1002/gcc.21965</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Automatic segmentation and tracking of thrombus formation within in vitro microscopic video sequences</h4>
<p class="authors"><span class="author">Peter,L.</span>, <span class="author">Brieu,N</span>, <span class="author">Jansen,S.</span>, <span class="author">Smethurst,P.A.</span>, <span class="author">OUWEHAND,W.H.</span> and <span class="author">Navab,N.</span></p>
<p class="affiliation">Computer Aided Medical Procedures, Technische Universitaet Muenchen, Munich, Germany; Department of Haematology, University of Cambridge, Cambridge, United Kingdom; National Health Service Blood and Transplant, Cambridge, United Kingdom; Wellcome Trust Sanger Institute, Hinxton, United Kingdom</p>
<p class="publication"><span class="title">Proceedings - International Symposium on Biomedical Imaging</span>&nbsp;<span class="year">2012</span>;<span class="pagerange">1635-7</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1109%2FISBI.2012.6235890" rel="external">10.1109/ISBI.2012.6235890</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recurrent SETBP1 mutations in atypical chronic myeloid leukemia.</h4>
<p class="authors"><span class="author">Piazza R</span>, <span class="author">Valletta S</span>, <span class="author">Winkelmann N</span>, <span class="author">Redaelli S</span>, <span class="author">Spinelli R</span>, <span class="author">Pirola A</span>, <span class="author">Antolini L</span>, <span class="author">Mologni L</span>, <span class="author">Donadoni C</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Schnittger S</span>, <span class="author">Kim DW</span>, <span class="author">Boultwood J</span>, <span class="author">Rossi F</span>, <span class="author">Gaipa G</span>, <span class="author">De Martini GP</span>, <span class="author">di Celle PF</span>, <span class="author">Jang HG</span>, <span class="author">Fantin V</span>, <span class="author">Bignell GR</span>, <span class="author">Magistroni V</span>, <span class="author">Haferlach T</span>, <span class="author">Pogliani EM</span>, <span class="author">Campbell PJ</span>, <span class="author">Chase AJ</span>, <span class="author">Tapper WJ</span>, <span class="author">Cross NC</span> and <span class="author">Gambacorti-Passerini C</span></p>
<p class="affiliation">1] Department of Health Sciences, University of Milano-Bicocca, Monza, Italy. [2].</p>
<p class="abstract">Atypical chronic myeloid leukemia (aCML) shares clinical and laboratory features with CML, but it lacks the BCR-ABL1 fusion. We performed exome sequencing of eight aCMLs and identified somatic alterations of SETBP1 (encoding a p.Gly870Ser alteration) in two cases. Targeted resequencing of 70 aCMLs, 574 diverse hematological malignancies and 344 cancer cell lines identified SETBP1 mutations in 24 cases, including 17 of 70 aCMLs (24.3%; 95% confidence interval (CI) = 16-35%). Most mutations (92%) were located between codons 858 and 871 and were identical to changes seen in individuals with Schinzel-Giedion syndrome. Individuals with mutations had higher white blood cell counts (P = 0.008) and worse prognosis (P = 0.01). The p.Gly870Ser alteration abrogated a site for ubiquitination, and cells exogenously expressing this mutant exhibited higher amounts of SETBP1 and SET protein, lower PP2A activity and higher proliferation rates relative to those expressing the wild-type protein. In summary, mutated SETBP1 represents a newly discovered oncogene present in aCML and closely related diseases.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23222956" rel="external">23222956</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2495" rel="external">10.1038/ng.2495</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A common single-nucleotide variant in T is strongly associated with chordoma.</h4>
<p class="authors"><span class="author">Pillay N</span>, <span class="author">Plagnol V</span>, <span class="author">Tarpey PS</span>, <span class="author">Lobo SB</span>, <span class="author">Presneau N</span>, <span class="author">Szuhai K</span>, <span class="author">Halai D</span>, <span class="author">Berisha F</span>, <span class="author">Cannon SR</span>, <span class="author">Mead S</span>, <span class="author">Kasperaviciute D</span>, <span class="author">Palmen J</span>, <span class="author">Talmud PJ</span>, <span class="author">Kindblom LG</span>, <span class="author">Amary MF</span>, <span class="author">Tirabosco R</span> and <span class="author">Flanagan AM</span></p>
<p class="affiliation">Cancer Institute, University College London, UK.</p><p class="abstract">Chordoma is a rare malignant bone tumor that expresses the transcription factor T. We conducted an association study of 40 individuals with chordoma and 358 ancestry-matched controls, with replication in an independent cohort. Whole-exome and Sanger sequencing of T exons showed strong association of the common nonsynonymous SNP rs2305089 with chordoma risk (allelic odds ratio (OR) = 6.1, 95% confidence interval (CI) = 3.1-12.1; P = 4.4 &times; 10(-9)), a finding that is exceptional in cancers with a non-Mendelian mode of inheritance.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U123160651</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">11</span>;<span class="pagerange">1185-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23064415">23064415</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2419">10.1038/ng.2419</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A tetracycline-repressible transactivator system to study essential genes in malaria parasites.</h4>
<p class="authors"><span class="author">Pino P</span>, <span class="author">Sebastian S</span>, <span class="author">Kim EA</span>, <span class="author">Bush E</span>, <span class="author">Brochet M</span>, <span class="author">Volkmann K</span>, <span class="author">Kozlowski E</span>, <span class="author">Llin&aacute;s M</span>, <span class="author">Billker O</span> and <span class="author">Soldati-Favre D</span></p>
<p class="affiliation">Department of Microbiology and Molecular Medicine, CMU, University of Geneva, 1 Rue Michel-Servet, 1211 Geneva 4, Switzerland.</p><p class="abstract">A major obstacle in analyzing gene function in apicomplexan parasites is the absence of a practical regulatable expression system. Here, we identified functional transcriptional activation domains within Apicomplexan AP2 (ApiAP2) family transcription factors. These ApiAP2 transactivation domains were validated in blood-, liver-, and mosquito-stage parasites and used to create a robust conditional expression system for stage-specific, tetracycline-dependent gene regulation in Toxoplasma gondii, Plasmodium berghei, and Plasmodium falciparum. To demonstrate the utility of this system, we created conditional knockdowns of two essential P.&nbsp;berghei genes: profilin (PRF), a protein implicated in parasite invasion, and N-myristoyltransferase (NMT), which catalyzes protein acylation. Tetracycline-induced repression of PRF and NMT expression resulted in a dramatic reduction in parasite viability. This efficient regulatable system will allow for the functional characterization of essential proteins that are found in these important parasites.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0501670</span>; NIAID NIH HHS: <span class="grant">R01 AI076276</span>; NIGMS NIH HHS: <span class="grant">P50 GM071508</span>, <span class="grant">P50GM071508</span>; Wellcome Trust: <span class="grant">WT077502MA</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Cell host &amp; microbe</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">824-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23245327">23245327</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3712325">3712325</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.chom.2012.10.016">10.1016/j.chom.2012.10.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shared loci for migraine and epilepsy on chromosomes 14q12-q23 and 12q24.2-q24.3.</h4>
<p class="authors"><span class="author">Polvi A</span>, <span class="author">Siren A</span>, <span class="author">Kallela M</span>, <span class="author">Rantala H</span>, <span class="author">Artto V</span>, <span class="author">Sobel EM</span>, <span class="author">Palotie A</span>, <span class="author">Lehesjoki AE</span> and <span class="author">Wessman M</span></p>
<p class="affiliation">Folkh&auml;lsan Institute of Genetics, Folkh&auml;lsan Research Center, Helsinki, Finland. <a href="mailto:%61%6E%6E%65.%70%6F%6C%76%69%40%68%65%6C%73%69%6E%6B%69.%66%69">&#97;&#110;&#110;&#101;&#46;&#112;&#111;&#108;&#118;&#105;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a></p><p class="abstract"><strong>Objectives</strong>: To describe clinical characteristics and to identify susceptibility loci for epilepsy and migraine in a Finnish family with a complex phenotype.</p><p class="abstract"><strong>Methods</strong>: Participating family members were interviewed and medical files were reviewed. The seizure classification was made according to International League Against Epilepsy criteria. Migraine diagnosis was made using the validated Finnish Migraine Specific Questionnaire for Family Studies and criteria according to the current International Classification of Headache Disorders-II. DNA samples were obtained from 56 family members and nonparametric genome-wide linkage analyses were performed using 382 polymorphic microsatellite markers. The most promising loci were fine-mapped with additional microsatellite markers.</p><p class="abstract"><strong>Results</strong>: Clinical data were obtained from 60 family members of whom 12 (20%) had idiopathic epileptic seizures. Eight of those 12 (67%) also had migraine. Altogether 33 of the 60 family members (55%) had migraine. Significant evidence of linkage was found between a locus on 14q12-q23 and migraine (p = 0.0001). Suggestive evidence of linkage in this region was also found for epilepsy with generalized tonic-clonic seizures (p = 0.0034). In addition, significant evidence of linkage was found at a locus on 12q24.2-q24.3 (p &lt; 0.001) for migraine alone and for the combined phenotype of migraine and epilepsy.</p><p class="abstract"><strong>Conclusions</strong>: Our data suggest the occurrence of common susceptibility loci for epilepsy and migraine on chromosomes 14q12-q23 and 12q24.2-q24.3, implicating a shared genetic etiology for these 2 diseases.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG006139</span>; NIGMS NIH HHS: <span class="grant">R01 GM053275</span></p>
<p class="publication"><span class="title">Neurology</span>&nbsp;<span class="year">2012</span>;<span class="volume">78</span>;<span class="number">3</span>;<span class="pagerange">202-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22218271">22218271</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3653199">3653199</a>; DOI: <a href="http://dx.doi.org/10.1212%2FWNL.0b013e31823fcd87">10.1212/WNL.0b013e31823fcd87</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Do muscle founder cells exist in vertebrates?</h4>
<p class="authors"><span class="author">Powell GT</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1HH, UK.</p>
<p class="abstract">Skeletal muscle is formed by the iterative fusion of precursor cells (myocytes) into long multinuclear fibres. Extensive studies of fusion in Drosophila embryos have lead to a paradigm in which myoblasts are divided into two distinct subtypes - founder and fusion-competent myoblasts (FCMs) - that can fuse to each other, but not among themselves. Only founder cells can direct the formation of muscle fibres, while FCMs act as a cellular substrate. Recent studies in zebrafish and mice have demonstrated conservation of the molecules originally identified in Drosophila, but an important question remains: is vertebrate fusion regulated by specifying myocyte subtypes? Stated simply: do vertebrate founder cells exist? In light of recent findings, we argue that a different regulatory mechanism has evolved in vertebrates.</p>
<p class="publication"><span class="title">Trends in cell biology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22710008" rel="external">22710008</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tcb.2012.05.003" rel="external">10.1016/j.tcb.2012.05.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic organisation, embryonic expression and biochemical interactions of the zebrafish junctional adhesion molecule family of receptors.</h4>
<p class="authors"><span class="author">Powell GT</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p>
<p class="abstract">The mammalian JAM family is composed of three cell surface receptors. Interactions between the proteins have well-characterised roles in inflammation and tight junction formation, but little is known about their function in early development. Recently, we identified a role for jamb and jamc in zebrafish myocyte fusion. Genome duplication in the teleost lineage raised the possibility that additional JAM family paralogues may also function in muscle development. To address this, we searched the zebrafish genome to identify potential paralogues and confirmed their homology, bringing the total number of zebrafish jam family members to six. We then compared the physical binding properties of each paralogue by surface plasmon resonance and determined the gene expression patterns of all zebrafish jam genes at different stages of development. Our results suggest a significant sub-functionalisation of JAM-B and JAM-C orthologues with respect to binding strength (but not specificity) and gene expression. The paralogous genes, jamb2 and jamc2, were not detected in the somites or myotome of wild-type embryos. We conclude that it is unlikely that the paralogues have a function in primary myogenesis.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e40810</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22815827" rel="external">22815827</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3399880" rel="external">3399880</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0040810" rel="external">10.1371/journal.pone.0040810</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The transcription factor T-bet regulates intestinal inflammation mediated by interleukin-7 receptor+ innate lymphoid cells.</h4>
<p class="authors"><span class="author">Powell N</span>, <span class="author">Walker AW</span>, <span class="author">Stolarczyk E</span>, <span class="author">Canavan JB</span>, <span class="author">G&ouml;kmen MR</span>, <span class="author">Marks E</span>, <span class="author">Jackson I</span>, <span class="author">Hashim A</span>, <span class="author">Curtis MA</span>, <span class="author">Jenner RG</span>, <span class="author">Howard JK</span>, <span class="author">Parkhill J</span>, <span class="author">MacDonald TT</span> and <span class="author">Lord GM</span></p>
<p class="affiliation">Department of Experimental Immunobiology, Division of Transplantation Immunology and Mucosal Biology, King&#39;s College London, UK.</p><p class="abstract">Mice lacking the transcription factor T-bet in the innate immune system develop microbiota-dependent colitis. Here, we show that interleukin-17A (IL-17A)-producing IL-7R&alpha;(+) innate lymphoid cells (ILCs) were potent promoters of disease in Tbx21(-/-)Rag2(-/-) ulcerative colitis (TRUC) mice. TNF-&alpha; produced by CD103(-)CD11b(+) dendritic cells synergized with IL-23 to drive IL-17A production by ILCs, demonstrating a previously unrecognized layer of cellular crosstalk between dendritic cells and ILCs. We have identified Helicobacter typhlonius as a key disease trigger driving excess TNF-&alpha; production and promoting colitis in TRUC mice. Crucially, T-bet also suppressed the expression of IL-7R, a key molecule involved in controlling intestinal ILC homeostasis. The importance of IL-7R signaling in TRUC disease was highlighted by the dramatic reduction in intestinal ILCs and attenuated colitis following IL-7R blockade. Taken together, these data demonstrate the mechanism by which T-bet regulates the complex interplay between mucosal dendritic cells, ILCs, and the intestinal microbiota.</p>
<p class="grants">Funded by: Department of Health: <span class="grant">DRF-2009-02-22</span>; Medical Research Council: <span class="grant">G0600081</span>, <span class="grant">G0802068</span>, <span class="grant">MR/J006742/1</span>, <span class="grant">MR/J011118/1</span>, <span class="grant">MR/K002996/1</span>; Wellcome Trust: <span class="grant">WT076964</span>, <span class="grant">WT088747MA</span></p>
<p class="publication"><span class="title">Immunity</span>&nbsp;<span class="year">2012</span>;<span class="volume">37</span>;<span class="number">4</span>;<span class="pagerange">674-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23063332">23063332</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3540260">3540260</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.immuni.2012.09.008">10.1016/j.immuni.2012.09.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Investigations into genome diversity of Haemophilus influenzae using whole genome sequencing of clinical isolates and laboratory transformants.</h4>
<p class="authors"><span class="author">Power PM</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Moxon ER</span> and <span class="author">Hood DW</span></p>
<p class="affiliation">University of Oxford Department of Paediatrics, Medical Sciences Division, John Radcliffe Hospital, Headington, Oxford OX3 9DU, UK. <a href="mailto:%72%69%63%68%61%72%64.%6D%6F%78%6F%6E%40%70%61%65%64%69%61%74%72%69%63%73.%6F%78.%61%63.%75%6B">&#114;&#105;&#99;&#104;&#97;&#114;&#100;&#46;&#109;&#111;&#120;&#111;&#110;&#64;&#112;&#97;&#101;&#100;&#105;&#97;&#116;&#114;&#105;&#99;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Unlabelled: ABSTRACT:

Background: Haemophilus influenzae is an important human commensal pathogen associated with significant levels of disease. High-throughput DNA sequencing was used to investigate differences in genome content within this species.

Results: Genomic DNA sequence was obtained from 85 strains of H. influenzae and from other related species, selected based on geographical site of isolation, disease association and documented genotypic and phenotypic differences. When compared by Mauve alignment these indicated groupings of H. influenzae that were consistent with previously published analyses; capsule expressing strains fell into two distinct groups and those of serotype b (Hib) were found in two closely positioned lineages. For 18 Hib strains representing both lineages we found many discrete regions (up to 40% of the total genome) displaying sequence variation when compared to a common reference strain. Evidence that this naturally occurring pattern of inter-strain variation in H. influenzae can be mediated by transformation was obtained through sequencing DNA obtained from a pool of 200 independent transformants of a recipient (strain Rd) using donor DNA from a heterologous Hib strain (Eagan).

Conclusion: Much of the inter-strain variation in genome sequence in H. influenzae is likely the result of inter-strain exchanges of DNA, most plausibly through transformation.</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="pagerange">273</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23176117">23176117</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3539920">3539920</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-12-273">10.1186/1471-2180-12-273</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">VarB: A Variation Browsing and analysis tool for variants derived from next-generation sequencing data.</h4>
<p class="authors"><span class="author">Preston MD</span>, <span class="author">Manske M</span>, <span class="author">Horner N</span>, <span class="author">Assefa S</span>, <span class="author">Campino S</span>, <span class="author">Auburn S</span>, <span class="author">Zongo I</span>, <span class="author">Oudraogo JB</span>, <span class="author">Nosten F</span>, <span class="author">Anderson T</span> and <span class="author">Clark TG</span></p>
<p class="affiliation">London School of Hygiene and Tropical Medicine, London, UK; Wellcome Trust Sanger Institute (WTSI), Hinxton, UK; Faculty of Health, Cranfield University, Cranfield, Bedfordshire, UK; Institut de Recherche en Sciences de la Sant, Bobo-Dioulasso, Burkina Faso Shoklo Malaria Research Unit, Mae Sot, Thailand; Texas Biomedical Research Institute, San Antonio, Texas, USA.</p>
<p class="abstract">SUMMARY: There is an immediate need for tools to both analyse and visualize in real-time single nucleotide polymorphisms, insertions and deletions, and other structural variants from new sequence file formats. We have developed VarB software that can be used to visualise Variant Call Format (VCF) files in real-time, as well as identify regions under balancing selection and informative markers to differentiate user-defined groups (e.g. populations). We demonstrate its utility using sequence data from fifty Plasmodium falciparum isolates comprising two different continents, and confirm known signals from genomic regions that contain important antigenic and anti-malarial drug-resistance genes.Availability and implementation: The C++-based software VarB and user manual are available from www.pathogenseq.org/varb. CONTACT: <a href="mailto:%6D%61%72%6B.%70%72%65%73%74%6F%6E%40%6C%73%68%74%6D.%61%63.%75%6B">&#109;&#97;&#114;&#107;&#46;&#112;&#114;&#101;&#115;&#116;&#111;&#110;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a> or <a href="mailto:%74%61%61%6E%65.%63%6C%61%72%6B%40%6C%73%68%74%6D.%61%63.%75%6B">&#116;&#97;&#97;&#110;&#101;&#46;&#99;&#108;&#97;&#114;&#107;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22976080" rel="external">22976080</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts557" rel="external">10.1093/bioinformatics/bts557</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A systematically improved high quality genome and transcriptome of the human blood fluke Schistosoma mansoni.</h4>
<p class="authors"><span class="author">Protasio AV</span>, <span class="author">Tsai IJ</span>, <span class="author">Babbage A</span>, <span class="author">Nichol S</span>, <span class="author">Hunt M</span>, <span class="author">Aslett MA</span>, <span class="author">De Silva N</span>, <span class="author">Velarde GS</span>, <span class="author">Anderson TJ</span>, <span class="author">Clark RC</span>, <span class="author">Davidson C</span>, <span class="author">Dillon GP</span>, <span class="author">Holroyd NE</span>, <span class="author">LoVerde PT</span>, <span class="author">Lloyd C</span>, <span class="author">McQuillan J</span>, <span class="author">Oliveira G</span>, <span class="author">Otto TD</span>, <span class="author">Parker-Manuel SJ</span>, <span class="author">Quail MA</span>, <span class="author">Wilson RA</span>, <span class="author">Zerlotini A</span>, <span class="author">Dunne DW</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">Schistosomiasis is one of the most prevalent parasitic diseases, affecting millions of people in developing countries. Amongst the human-infective species, Schistosoma mansoni is also the most commonly used in the laboratory and here we present the systematic improvement of its draft genome. We used Sanger capillary and deep-coverage Illumina sequencing from clonal worms to upgrade the highly fragmented draft 380 Mb genome to one with only 885 scaffolds and more than 81% of the bases organised into chromosomes. We have also used transcriptome sequencing (RNA-seq) from four time points in the parasite&#39;s life cycle to refine gene predictions and profile their expression. More than 45% of predicted genes have been extensively modified and the total number has been reduced from 11,807 to 10,852. Using the new version of the genome, we identified trans-splicing events occurring in at least 11% of genes and identified clear cases where it is used to resolve polycistronic transcripts. We have produced a high-resolution map of temporal changes in expression for 9,535 genes, covering an unprecedented dynamic range for this organism. All of these data have been consolidated into a searchable format within the GeneDB (www.genedb.org) and SchistoDB (www.schistodb.net) databases. With further transcriptional profiling and genome sequencing increasingly accessible, the upgraded genome will form a fundamental dataset to underpin further advances in schistosome research.</p>
<p class="grants">Funded by: FIC NIH HHS: <span class="grant">TW007012</span>; PHS HHS: <span class="grant">HHSN272201000009I</span>; Wellcome Trust: <span class="grant">085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">e1455</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22253936">22253936</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3254664">3254664</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001455">10.1371/journal.pntd.0001455</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Pfam protein families database.</h4>
<p class="authors"><span class="author">Punta M</span>, <span class="author">Coggill PC</span>, <span class="author">Eberhardt RY</span>, <span class="author">Mistry J</span>, <span class="author">Tate J</span>, <span class="author">Boursnell C</span>, <span class="author">Pang N</span>, <span class="author">Forslund K</span>, <span class="author">Ceric G</span>, <span class="author">Clements J</span>, <span class="author">Heger A</span>, <span class="author">Holm L</span>, <span class="author">Sonnhammer EL</span>, <span class="author">Eddy SR</span>, <span class="author">Bateman A</span> and <span class="author">Finn RD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%6D%70%31%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#112;&#49;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Pfam is a widely used database of protein families, currently containing more than 13,000 manually curated protein families as of release 26.0. Pfam is available via servers in the UK (http://pfam.sanger.ac.uk/), the USA (http://pfam.janelia.org/) and Sweden (http://pfam.sbc.su.se/). Here, we report on changes that have occurred since our 2010 NAR paper (release 24.0). Over the last 2 years, we have generated 1840 new families and increased coverage of the UniProt Knowledgebase (UniProtKB) to nearly 80%. Notably, we have taken the step of opening up the annotation of our families to the Wikipedia community, by linking Pfam families to relevant Wikipedia pages and encouraging the Pfam and Wikipedia communities to improve and expand those pages. We continue to improve the Pfam website and add new visualizations, such as the &#39;sunburst&#39; representation of taxonomic distribution of families. In this work we additionally address two topics that will be of particular interest to the Pfam community. First, we explain the definition and use of family-specific, manually curated gathering thresholds. Second, we discuss some of the features of domains of unknown function (also known as DUFs), which constitute a rapidly growing class of families within Pfam.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F010435/1</span>; Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D290-301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22127870">22127870</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245129">3245129</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr1065">10.1093/nar/gkr1065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluation and optimisation of preparative semi-automated electrophoresis systems for Illumina library preparation.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Gu Y</span>, <span class="author">Swerdlow H</span> and <span class="author">Mayho M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6D%71%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#113;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Size selection can be a critical step in preparation of next-generation sequencing libraries. Traditional methods employing gel electrophoresis lack reproducibility, are labour intensive, do not scale well and employ hazardous interchelating dyes. In a high-throughput setting, solid-phase reversible immobilisation beads are commonly used for size-selection, but result in quite a broad fragment size range. We have evaluated and optimised the use of two semi-automated preparative DNA electrophoresis systems, the Caliper Labchip XT and the Sage Science Pippin Prep, for size selection of Illumina sequencing libraries.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Electrophoresis</span>&nbsp;<span class="year">2012</span>;<span class="volume">33</span>;<span class="number">23</span>;<span class="pagerange">3521-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23147856">23147856</a>; DOI: <a href="http://dx.doi.org/10.1002%2Felps.201200128">10.1002/elps.201200128</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Optimal enzymes for amplifying sequencing libraries.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Otto TD</span>, <span class="author">Gu Y</span>, <span class="author">Harris SR</span>, <span class="author">Skelly TF</span>, <span class="author">McQuillan JA</span>, <span class="author">Swerdlow HP</span> and <span class="author">Oyola SO</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2012</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">10-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22205512">22205512</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1814">10.1038/nmeth.1814</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Smith M</span>, <span class="author">Coupland P</span>, <span class="author">Otto TD</span>, <span class="author">Harris SR</span>, <span class="author">Connor TR</span>, <span class="author">Bertoni A</span>, <span class="author">Swerdlow HP</span> and <span class="author">Gu Y</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%6D%71%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#113;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Next generation sequencing (NGS) technology has revolutionized genomic and genetic research. The pace of change in this area is rapid with three major new sequencing platforms having been released in 2011: Ion Torrent&#39;s PGM, Pacific Biosciences&#39; RS and the Illumina MiSeq. Here we compare the results obtained with those platforms to the performance of the Illumina HiSeq, the current market leader. In order to compare these platforms, and get sufficient coverage depth to allow meaningful analysis, we have sequenced a set of 4 microbial genomes with mean GC content ranging from 19.3 to 67.7%. Together, these represent a comprehensive range of genome content. Here we report our analysis of that sequence data in terms of coverage distribution, bias, GC distribution, variant detection and accuracy.</p><p class="abstract"><strong>Results</strong>: Sequence generated by Ion Torrent, MiSeq and Pacific Biosciences technologies displays near perfect coverage behaviour on GC-rich, neutral and moderately AT-rich genomes, but a profound bias was observed upon sequencing the extremely AT-rich genome of Plasmodium falciparum on the PGM, resulting in no coverage for approximately 30% of the genome. We analysed the ability to call variants from each platform and found that we could call slightly more variants from Ion Torrent data compared to MiSeq data, but at the expense of a higher false positive rate. Variant calling from Pacific Biosciences data was possible but higher coverage depth was required. Context specific errors were observed in both PGM and MiSeq data, but not in that from the Pacific Biosciences platform.</p><p class="abstract"><strong>Conclusions</strong>: All three fast turnaround sequencers evaluated here were able to generate usable sequence. However there are key differences between the quality of that data and the applications it will support.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">341</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22827831">22827831</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3431227">3431227</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-341">10.1186/1471-2164-13-341</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale mouse knockouts and phenotypes.</h4>
<p class="authors"><span class="author">Ram&iacute;rez-Solis R</span>, <span class="author">Ryder E</span>, <span class="author">Houghton R</span>, <span class="author">White JK</span> and <span class="author">Bottomley J</span></p>
<p class="affiliation">Mouse Pipelines, Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%72%72%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#114;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Standardized phenotypic analysis of mutant forms of every gene in the mouse genome will provide fundamental insights into mammalian gene function and advance human and animal health. The availability of the human and mouse genome sequences, the development of embryonic stem cell mutagenesis technology, the standardization of phenotypic analysis pipelines, and the paradigm-shifting industrialization of these processes have made this a realistic and achievable goal. The size of this enterprise will require global coordination to ensure economies of scale in both the generation and primary phenotypic analysis of the mutant strains, and to minimize unnecessary duplication of effort. To provide more depth to the functional annotation of the genome, effective mechanisms will also need to be developed to disseminate the information and resources produced to the wider community. Better models of disease, potential new drug targets with novel mechanisms of action, and completely unsuspected genotype-phenotype relationships covering broad aspects of biology will become apparent. To reach these goals, solutions to challenges in mouse production and distribution, as well as development of novel, ever more powerful phenotypic analysis modalities will be necessary. It is a challenging and exciting time to work in mouse genetics. WIREs Syst Biol Med 2012 doi: 10.1002/wsbm.1183 For further resources related to this article, please visit the WIREs website.</p>
<p class="publication"><span class="title">Wiley interdisciplinary reviews. Systems biology and medicine</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22899600" rel="external">22899600</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fwsbm.1183" rel="external">10.1002/wsbm.1183</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-Wide Association Studies of Asthma in Population-Based Cohorts Confirm Known and Suggested Loci and Identify an Additional Association near HLA.</h4>
<p class="authors"><span class="author">Ramasamy A</span>, <span class="author">Kuokkanen M</span>, <span class="author">Vedantam S</span>, <span class="author">Gajdos ZK</span>, <span class="author">Couto Alves A</span>, <span class="author">Lyon HN</span>, <span class="author">Ferreira MA</span>, <span class="author">Strachan DP</span>, <span class="author">Zhao JH</span>, <span class="author">Abramson MJ</span>, <span class="author">Brown MA</span>, <span class="author">Coin L</span>, <span class="author">Dharmage SC</span>, <span class="author">Duffy DL</span>, <span class="author">Haahtela T</span>, <span class="author">Heath AC</span>, <span class="author">Janson C</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Khaw KT</span>, <span class="author">Laitinen J</span>, <span class="author">Le Souef P</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Australian Asthma Genetics Consortium collaborators</span>, <span class="author">Madden PA</span>, <span class="author">Marks GB</span>, <span class="author">Martin NG</span>, <span class="author">Matheson MC</span>, <span class="author">Palmer CD</span>, <span class="author">Palotie A</span>, <span class="author">Pouta A</span>, <span class="author">Robertson CF</span>, <span class="author">Viikari J</span>, <span class="author">Widen E</span>, <span class="author">Wjst M</span>, <span class="author">Jarvis DL</span>, <span class="author">Montgomery GW</span>, <span class="author">Thompson PJ</span>, <span class="author">Wareham N</span>, <span class="author">Eriksson J</span>, <span class="author">Jousilahti P</span>, <span class="author">Laitinen T</span>, <span class="author">Pekkanen J</span>, <span class="author">Raitakari OT</span>, <span class="author">O&#39;Connor GT</span>, <span class="author">Salomaa V</span>, <span class="author">Jarvelin MR</span> and <span class="author">Hirschhorn JN</span></p>
<p class="affiliation">Respiratory Epidemiology and Public Health, Imperial College London, London, United Kingdom ; Department of Epidemiology and Biostatistics, Imperial College London, London, United Kingdom ; Department of Medical and Molecular Genetics, King's College London, London, United Kingdom.</p>
<p class="abstract">Rationale: Asthma has substantial morbidity and mortality and a strong genetic component, but identification of genetic risk factors is limited by availability of suitable studies.

Objectives: To test if population-based cohorts with self-reported physician-diagnosed asthma and genome-wide association (GWA) data could be used to validate known associations with asthma and identify novel associations.

Methods: The APCAT (Analysis in Population-based Cohorts of Asthma Traits) consortium consists of 1,716 individuals with asthma and 16,888 healthy controls from six European-descent population-based cohorts. We examined associations in APCAT of thirteen variants previously reported as genome-wide significant (P<5x10(-8)) and three variants reported as suggestive (P<5Ã—10(-7)). We also searched for novel associations in APCAT (Stage 1) and followed-up the most promising variants in 4,035 asthmatics and 11,251 healthy controls (Stage 2). Finally, we conducted the first genome-wide screen for interactions with smoking or hay fever.

We observed association in the same direction for all thirteen previously reported variants and nominally replicated ten of them. One variant that was previously suggestive, rs11071559 in RORA, now reaches genome-wide significance when combined with our data (Pâ€Š=â€Š2.4Ã—10(-9)). We also identified two genome-wide significant associations: rs13408661 near IL1RL1/IL18R1 (P(Stage1+Stage2)â€Š=â€Š1.1x10(-9)), which is correlated with a variant recently shown to be associated with asthma (rs3771180), and rs9268516 in the HLA region (P(Stage1+Stage2)â€Š=â€Š1.1x10(-8)), which appears to be independent of previously reported associations in this locus. Finally, we found no strong evidence for gene-environment interactions with smoking or hay fever status.

Conclusions: Population-based cohorts with simple asthma phenotypes represent a valuable and largely untapped resource for genetic studies of asthma.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e44008</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23028483" rel="external">23028483</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3461045" rel="external">3461045</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0044008" rel="external">10.1371/journal.pone.0044008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MEROPS: the database of proteolytic enzymes, their substrates and inhibitors.</h4>
<p class="authors"><span class="author">Rawlings ND</span>, <span class="author">Barrett AJ</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Peptidases, their substrates and inhibitors are of great relevance to biology, medicine and biotechnology. The MEROPS database (http://merops.sanger.ac.uk) aims to fulfil the need for an integrated source of information about these. The database has hierarchical classifications in which homologous sets of peptidases and protein inhibitors are grouped into protein species, which are grouped into families, which are in turn grouped into clans. The database has been expanded to include proteolytic enzymes other than peptidases. Special identifiers for peptidases from a variety of model organisms have been established so that orthologues can be detected in other species. A table of predicted active-site residue and metal ligand positions and the residue ranges of the peptidase domains in orthologues has been added to each peptidase summary. New displays of tertiary structures, which can be rotated or have the surfaces displayed, have been added to the structure pages. New indexes for gene names and peptidase substrates have been made available. Among the enhancements to existing features are the inclusion of small-molecule inhibitors in the tables of peptidase-inhibitor interactions, a table of known cleavage sites for each protein substrate, and tables showing the substrate-binding preferences of peptidases derived from combinatorial peptide substrate libraries.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D343-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22086950" rel="external">22086950</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245014" rel="external">3245014</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr987" rel="external">10.1093/nar/gkr987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Changes in HDL cholesterol and cardiovascular outcomes after lipid modification therapy.</h4>
<p class="authors"><span class="author">Ray K</span>, <span class="author">Wainwright NW</span>, <span class="author">Visser L</span>, <span class="author">Witteman J</span>, <span class="author">Breteler M</span>, <span class="author">Ambegaonkar B</span>, <span class="author">Hofman A</span>, <span class="author">Stricker B</span>, <span class="author">Wareham N</span>, <span class="author">Khaw KT</span> and <span class="author">Sandhu M</span></p>
<p class="affiliation">Division of Clinical Sciences, St George&#39;s University of London, Cranmer Terrace, London, UK. <a href="mailto:%6B%72%61%79%40%73%67%75%6C.%61%63.%75%6B">&#107;&#114;&#97;&#121;&#64;&#115;&#103;&#117;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Lipid modification therapy (LMT) produces cardiovascular benefits principally through reductions in low density lipoprotein cholesterol. While recent evidence, using data from 454 participants in the Framingham Offspring Study, has suggested that increases in high density lipoprotein cholesterol (HDL-C) are also associated with a reduction in cardiovascular outcomes, independently of changes in low density lipoprotein cholesterol, replication of this finding is important. The authors therefore present further results using data from the EPIC-Norfolk (UK) and Rotterdam (The Netherlands) prospective cohort studies.</p><p class="abstract"><strong>Methods</strong>: A total of 1148 participants, 446 from the EPIC-Norfolk and 702 from the Rotterdam study, were assessed for lipids before and after starting LMT. Subsequent risk of cardiovascular events, ascertained through linkage with mortality records and hospital databases, was investigated using Cox proportional hazards regression. Random effects meta-analysis was used to combine results across studies.</p><p class="abstract"><strong>Results</strong>: Based on combined data from the EPIC-Norfolk and Rotterdam studies there was some evidence that change in HDL-C resulting from LMT was associated with reduced cardiovascular risk (HR per pooled SD (=0.34 mmol/l) increase=0.74, 95% CI 0.56 to 0.99, adjusted for age, sex and baseline HDL-C). However, this association was attenuated and was not (statistically) significant with further adjustments for non-HDL-C and for cigarette smoking history, prevalent diabetes, systolic blood pressure, body mass index, use of antihypertensive medication, previous myocardial infarction, prevalent angina and previous stroke (0.92, 0.701.20).</p><p class="abstract"><strong>Conclusions</strong>: Following adjustment for conventional non-lipid risk factors of cardiovascular disease, this study provides no evidence to support a significant benefit from increasing HDL-C independent of the effect of lowering non-HDL-C.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council</p>
<p class="publication"><span class="title">Heart (British Cardiac Society)</span>&nbsp;<span class="year">2012</span>;<span class="volume">98</span>;<span class="number">10</span>;<span class="pagerange">780-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22447463">22447463</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fheartjnl-2011-301405">10.1136/heartjnl-2011-301405</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adapting to domesticity.</h4>
<p class="authors"><span class="author">Reid AJ</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">3</span>;<span class="pagerange">163</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22337165" rel="external">22337165</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2752" rel="external">10.1038/nrmicro2752</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of the apicomplexan parasites Toxoplasma gondii and Neospora caninum: Coccidia differing in host range and transmission strategy.</h4>
<p class="authors"><span class="author">Reid AJ</span>, <span class="author">Vermont SJ</span>, <span class="author">Cotton JA</span>, <span class="author">Harris D</span>, <span class="author">Hill-Cawthorne GA</span>, <span class="author">K&ouml;nen-Waisman S</span>, <span class="author">Latham SM</span>, <span class="author">Mourier T</span>, <span class="author">Norton R</span>, <span class="author">Quail MA</span>, <span class="author">Sanders M</span>, <span class="author">Shanmugam D</span>, <span class="author">Sohal A</span>, <span class="author">Wasmuth JD</span>, <span class="author">Brunk B</span>, <span class="author">Grigg ME</span>, <span class="author">Howard JC</span>, <span class="author">Parkinson J</span>, <span class="author">Roos DS</span>, <span class="author">Trees AJ</span>, <span class="author">Berriman M</span>, <span class="author">Pain A</span> and <span class="author">Wastling JM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgshire, United Kingdom.</p><p class="abstract">Toxoplasma gondii is a zoonotic protozoan parasite which infects nearly one third of the human population and is found in an extraordinary range of vertebrate hosts. Its epidemiology depends heavily on horizontal transmission, especially between rodents and its definitive host, the cat. Neospora caninum is a recently discovered close relative of Toxoplasma, whose definitive host is the dog. Both species are tissue-dwelling Coccidia and members of the phylum Apicomplexa; they share many common features, but Neospora neither infects humans nor shares the same wide host range as Toxoplasma, rather it shows a striking preference for highly efficient vertical transmission in cattle. These species therefore provide a remarkable opportunity to investigate mechanisms of host restriction, transmission strategies, virulence and zoonotic potential. We sequenced the genome of N. caninum and transcriptomes of the invasive stage of both species, undertaking an extensive comparative genomics and transcriptomics analysis. We estimate that these organisms diverged from their common ancestor around 28 million years ago and find that both genomes and gene expression are remarkably conserved. However, in N. caninum we identified an unexpected expansion of surface antigen gene families and the divergence of secreted virulence factors, including rhoptry kinases. Specifically we show that the rhoptry kinase ROP18 is pseudogenised in N. caninum and that, as a possible consequence, Neospora is unable to phosphorylate host immunity-related GTPases, as Toxoplasma does. This defense strategy is thought to be key to virulence in Toxoplasma. We conclude that the ecological niches occupied by these species are influenced by a relatively small number of gene products which operate at the host-parasite interface and that the dominance of vertical transmission in N. caninum may be associated with the evolution of reduced virulence in this species.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/B/08493</span>; Canadian Institutes of Health Research; Wellcome Trust: <span class="grant">085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">e1002567</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22457617">22457617</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3310773">3310773</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002567">10.1371/journal.ppat.1002567</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Marked endotheliotropism of highly pathogenic avian influenza virus H5N1 following intestinal inoculation in cats.</h4>
<p class="authors"><span class="author">Reperant LA</span>, <span class="author">van de Bildt MW</span>, <span class="author">van Amerongen G</span>, <span class="author">Leijten LM</span>, <span class="author">Watson S</span>, <span class="author">Palser A</span>, <span class="author">Kellam P</span>, <span class="author">Eissens AC</span>, <span class="author">Frijlink HW</span>, <span class="author">Osterhaus AD</span> and <span class="author">Kuiken T</span></p>
<p class="affiliation">Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, USA.</p><p class="abstract">Highly pathogenic avian influenza virus (HPAIV) H5N1 can infect mammals via the intestine; this is unusual since influenza viruses typically infect mammals via the respiratory tract. The dissemination of HPAIV H5N1 following intestinal entry and associated pathogenesis are largely unknown. To assess the route of spread of HPAIV H5N1 to other organs and to determine its associated pathogenesis, we inoculated infected chicken liver homogenate directly into the intestine of cats by use of enteric-coated capsules. Intestinal inoculation of HPAIV H5N1 resulted in fatal systemic disease. The spread of HPAIV H5N1 from the lumen of the intestine to other organs took place via the blood and lymphatic vascular systems but not via neuronal transmission. Remarkably, the systemic spread of the virus via the vascular system was associated with massive infection of endothelial and lymphendothelial cells, resulting in widespread hemorrhages. This is unique for influenza in mammals and resembles the pathogenesis of HPAIV infection in terrestrial poultry. It contrasts with the pathogenesis of systemic disease from the same virus following entry via the respiratory tract, where lesions are characterized mainly by necrosis and inflammation and are associated with the presence of influenza virus antigen in parenchymal, not endothelial cells. The marked endotheliotropism of the virus following intestinal inoculation indicates that the pathogenesis of systemic influenza virus infection in mammals may differ according to the portal of entry.</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2012</span>;<span class="volume">86</span>;<span class="number">2</span>;<span class="pagerange">1158-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22090101">22090101</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3255817">3255817</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.06375-11">10.1128/JVI.06375-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary Dynamics of the Yersinia enterocolitica Complex.</h4>
<p class="authors"><span class="author">Reuter S</span>, <span class="author">Thomson NR</span> and <span class="author">McNally A</span></p>
<p class="affiliation">Pathogen Sequencing Group, Wellcome Trust Sanger Institute, Cambridge, UK.</p>
<p class="publication"><span class="title">Advances in experimental medicine and biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">954</span>;<span class="pagerange">15-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22782741" rel="external">22782741</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-4614-3561-7_2" rel="external">10.1007/978-1-4614-3561-7_2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improving transcription factor binding site predictions by using randomised negative examples</h4>
<p class="authors"><span class="author">Rezwan F</span>, <span class="author">Sun Y</span>, <span class="author">Davey N</span>, <span class="author">Adams R</span>, <span class="author">RUST AG</span>, <span class="author">Robinson M</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science</span> <span class="year">2012</span>;<span class="volume">7223</span>;<span class="pagerange">225-37</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The TCA cycle is not required for selection or survival of multidrug-resistant Salmonella.</h4>
<p class="authors"><span class="author">Ricci V</span>, <span class="author">Loman N</span>, <span class="author">Pallen M</span>, <span class="author">Ivens A</span>, <span class="author">Fookes M</span>, <span class="author">Langridge GC</span>, <span class="author">Wain J</span> and <span class="author">Piddock LJ</span></p>
<p class="affiliation">Antimicrobial Agents Research Group, School of Immunity and Infection, University of Birmingham, Edgbaston, Birmingham, UK.</p><p class="abstract"><strong>Objectives</strong>: The initial aim of this study was to use a systems biology approach to analyse a ciprofloxacin-selected multidrug-resistant (MDR) Salmonella enterica serotype Typhimurium, L664.</p><p class="abstract"><strong>Methods</strong>: The whole genome sequence and transcriptome of L664 were analysed. Site-directed mutagenesis to recreate each mutation was carried out, followed by phenotypic characterization and mutation frequency analysis. As a mutation in the TCA cycle was detected we tested the controversial hypothesis regarding the bacterial response to bactericidal antibiotics, put forward by Kohanski et al. (Cell 2007; 130: 797-810 and Mol Cell 2010; 37: 311-20), that exposure of bacteria to agents such as ciprofloxacin produces reactive oxygen species (ROS), which transiently increase the mutation rate giving rise to MDR bacteria.</p><p class="abstract"><strong>Results</strong>: L664 contained a mutation in ramR that conferred MDR. A mutation in tctA affected the TCA cycle and conferred the inability to grow on minimal agar. The virulence of L664 was not attenuated. Ciprofloxacin exposure produced ROS in L664 and SL1344 (tctA::aph), but it was reduced and occurred later. There were no significant differences in the rates of killing or mutations per generation to antibiotic resistance between the strains.</p><p class="abstract"><strong>Conclusions</strong>: Whilst we confirm production of ROS in response to ciprofloxacin, we have no data to support the hypothesis that this leads to selection of MDR strains. Our results indicate that the mutations in tctA and glgA were random as they did not pre-exist in the parental strain, and that the mutation in tctA did not provide a survival advantage or disadvantage in the presence of antibiotic.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501415</span>, <span class="grant">G0801977</span>, <span class="grant">GO501415</span></p>
<p class="publication"><span class="title">The Journal of antimicrobial chemotherapy</span>&nbsp;<span class="year">2012</span>;<span class="volume">67</span>;<span class="number">3</span>;<span class="pagerange">589-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22186876">22186876</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4125622">4125622</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fjac%2Fdkr515">10.1093/jac/dkr515</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Germline transgenesis and insertional mutagenesis in Schistosoma mansoni mediated by murine leukemia virus.</h4>
<p class="authors"><span class="author">Rinaldi G</span>, <span class="author">Eckert SE</span>, <span class="author">Tsai IJ</span>, <span class="author">Suttiprapa S</span>, <span class="author">Kines KJ</span>, <span class="author">Tort JF</span>, <span class="author">Mann VH</span>, <span class="author">Turner DJ</span>, <span class="author">Berriman M</span> and <span class="author">Brindley PJ</span></p>
<p class="affiliation">Department of Microbiology, Immunology &amp; Tropical Medicine, School of Medicine &amp; Health Sciences, The George Washington University, Washington, DC, United States of America.</p><p class="abstract">Functional studies will facilitate characterization of role and essentiality of newly available genome sequences of the human schistosomes, Schistosoma mansoni, S. japonicum and S. haematobium. To develop transgenesis as a functional approach for these pathogens, we previously demonstrated that pseudotyped murine leukemia virus (MLV) can transduce schistosomes leading to chromosomal integration of reporter transgenes and short hairpin RNA cassettes. Here we investigated vertical transmission of transgenes through the developmental cycle of S. mansoni after introducing transgenes into eggs. Although MLV infection of schistosome eggs from mouse livers was efficient in terms of snail infectivity, &gt;10-fold higher transgene copy numbers were detected in cercariae derived from in vitro laid eggs (IVLE). After infecting snails with miracidia from eggs transduced by MLV, sequencing of genomic DNA from cercariae released from the snails also revealed the presence of transgenes, demonstrating that transgenes had been transmitted through the asexual developmental cycle, and thereby confirming germline transgenesis. High-throughput sequencing of genomic DNA from schistosome populations exposed to MLV mapped widespread and random insertion of transgenes throughout the genome, along each of the autosomes and sex chromosomes, validating the utility of this approach for insertional mutagenesis. In addition, the germline-transmitted transgene encoding neomycin phosphotransferase rescued cultured schistosomules from toxicity of the antibiotic G418, and PCR analysis of eggs resulting from sexual reproduction of the transgenic worms in mice confirmed that retroviral transgenes were transmitted to the next (F1) generation. These findings provide the first description of wide-scale, random insertional mutagenesis of chromosomes and of germline transmission of a transgene in schistosomes. Transgenic lines of schistosomes expressing antibiotic resistance could advance functional genomics for these significant human pathogens. DATABASE ACCESSION: Sequence data from this study have been submitted to the European Nucleotide Archive (http://www.ebi.ac.uk/embl) under accession number ERP000379.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01AI072773</span>; PHS HHS: <span class="grant">HHSN272201000005I</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">e1002820</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22911241">22911241</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3406096">3406096</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002820">10.1371/journal.ppat.1002820</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular cytogenetics: karyotype evolution, phylogenomics and future prospects.</h4>
<p class="authors"><span class="author">Robinson TJ</span> and <span class="author">Yang F</span></p>
<p class="publication"><span class="title">Heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">108</span>;<span class="number">1</span>;<span class="pagerange">1-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22167088" rel="external">22167088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3238121" rel="external">3238121</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fhdy.2011.117" rel="external">10.1038/hdy.2011.117</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in ISPD cause Walker-Warburg syndrome and defective glycosylation of &alpha;-dystroglycan.</h4>
<p class="authors"><span class="author">Roscioli T</span>, <span class="author">Kamsteeg EJ</span>, <span class="author">Buysse K</span>, <span class="author">Maystadt I</span>, <span class="author">van Reeuwijk J</span>, <span class="author">van den Elzen C</span>, <span class="author">van Beusekom E</span>, <span class="author">Riemersma M</span>, <span class="author">Pfundt R</span>, <span class="author">Vissers LE</span>, <span class="author">Schraders M</span>, <span class="author">Altunoglu U</span>, <span class="author">Buckley MF</span>, <span class="author">Brunner HG</span>, <span class="author">Grisart B</span>, <span class="author">Zhou H</span>, <span class="author">Veltman JA</span>, <span class="author">Gilissen C</span>, <span class="author">Mancini GM</span>, <span class="author">Delr&eacute;e P</span>, <span class="author">Willemsen MA</span>, <span class="author">Ramad&#x17E;a DP</span>, <span class="author">Chitayat D</span>, <span class="author">Bennett C</span>, <span class="author">Sheridan E</span>, <span class="author">Peeters EA</span>, <span class="author">Tan-Sindhunata GM</span>, <span class="author">de Die-Smulders CE</span>, <span class="author">Devriendt K</span>, <span class="author">Kayserili H</span>, <span class="author">El-Hashash OA</span>, <span class="author">Stemple DL</span>, <span class="author">Lefeber DJ</span>, <span class="author">Lin YY</span> and <span class="author">van Bokhoven H</span></p>
<p class="affiliation">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands.</p><p class="abstract">Walker-Warburg syndrome (WWS) is an autosomal recessive multisystem disorder characterized by complex eye and brain abnormalities with congenital muscular dystrophy (CMD) and aberrant a-dystroglycan glycosylation. Here we report mutations in the ISPD gene (encoding isoprenoid synthase domain containing) as the second most common cause of WWS. Bacterial IspD is a nucleotidyl transferase belonging to a large glycosyltransferase family, but the role of the orthologous protein in chordates is obscure to date, as this phylum does not have the corresponding non-mevalonate isoprenoid biosynthesis pathway. Knockdown of ispd in zebrafish recapitulates the human WWS phenotype with hydrocephalus, reduced eye size, muscle degeneration and hypoglycosylated a-dystroglycan. These results implicate ISPD in a-dystroglycan glycosylation in maintaining sarcolemma integrity in vertebrates.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077037</span>, <span class="grant">077037/Z/05/Z</span>, <span class="grant">077047/Z/05/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">581-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22522421">22522421</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3378661">3378661</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2253">10.1038/ng.2253</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction of salmonella typhimurium with dendritic cells derived from pluripotent embryonic stem cells.</h4>
<p class="authors"><span class="author">Rossi R</span>, <span class="author">Hale C</span>, <span class="author">Goulding D</span>, <span class="author">Andrews R</span>, <span class="author">Abdellah Z</span>, <span class="author">Fairchild PJ</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, United Kingdom.</p>
<p class="abstract">Using an in vitro differentiation protocol we isolated cells with the properties of dendritic cells (DCs) from immunologically refractive pluripotent murine embryonic stem cells (ESCs). These ES-derived dendritic cells (ESDCs) expressed cytokines and were able to present antigen to a T cell line. Infection of ESDCs with Salmonella Typhimurium stimulated the expression of immune cell markers and thousands of murine genes, many associated with the immune response. Consequently, this system provides a novel in vitro model, amenable to genetic modification, for monitoring host/pathogen interactions.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e52232</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23284947" rel="external">23284947</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3532158" rel="external">3532158</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0052232" rel="external">10.1371/journal.pone.0052232</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The industrial evolution.</h4>
<p class="authors"><span class="author">Salter SJ</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">This month's Genome Watch highlights new insights into the use of bacteria in industrial processes such as electricity generation and bioremediation.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23147705" rel="external">23147705</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2915" rel="external">10.1038/nrmicro2915</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation at the capsule locus, cps, of mistyped and non-typable Streptococcus pneumoniae isolates.</h4>
<p class="authors"><span class="author">Salter SJ</span>, <span class="author">Hinds J</span>, <span class="author">Gould KA</span>, <span class="author">Lambertsen L</span>, <span class="author">Hanage WP</span>, <span class="author">Antonio M</span>, <span class="author">Turner P</span>, <span class="author">Hermans PW</span>, <span class="author">Bootsma HJ</span>, <span class="author">O&#39;Brien KL</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%73%62%31%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#98;&#49;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The capsule polysaccharide locus (cps) is the site of the capsule biosynthesis gene cluster in encapsulated Streptococcus pneumoniae. A set of pneumococcal samples and non-pneumococcal streptococci from Denmark, the Gambia, the Netherlands, Thailand, the UK and the USA were sequenced at the cps locus to elucidate serologically mistyped or non-typable isolates. We identified a novel serotype 33B/33C mosaic capsule cluster and previously unseen serotype 22F capsule genes, disrupted and deleted cps clusters, the presence of aliB and nspA genes that are unrelated to capsule production, and similar genes in the non-pneumococcal samples. These data provide greater understanding of diversity at a locus which is crucial to the antigenic diversity of the pathogen and current vaccine strategies.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54 GM088558</span>, <span class="grant">U54GM088558</span>; Wellcome Trust: <span class="grant">076964</span>, <span class="grant">083735</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">158</span>;<span class="number">Pt 6</span>;<span class="pagerange">1560-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22403189">22403189</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3541774">3541774</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.056580-0">10.1099/mic.0.056580-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing parasite populations.</h4>
<p class="authors"><span class="author">Sanchez-Flores A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">2</span>;<span class="pagerange">85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22245933" rel="external">22245933</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2738" rel="external">10.1038/nrmicro2738</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global phylogeny of Shigella sonnei from limited single nucleotide polymorphisms and development of a rapid and cost-effective SNP-typing scheme for strain identification by high resolution melting.</h4>
<p class="authors"><span class="author">Sangal V</span>, <span class="author">Holt KE</span>, <span class="author">Yuan J</span>, <span class="author">Brown DJ</span>, <span class="author">Filliol-Toutain I</span>, <span class="author">Weill FX</span>, <span class="author">Kim DW</span>, <span class="author">da Silveira WD</span>, <span class="author">Pickard D</span>, <span class="author">Thomson NR</span>, <span class="author">Parkhill J</span> and <span class="author">Yu J</span></p>
<p class="affiliation">Strathclyde Institute of Pharmacy and Biomedical Sciences, University of Strathclyde, Glasgow, UK (V. Sangal, J. Yuan, J. Yu); Department of Microbiology and Immunology, University of Melbourne, Australia (K.E. Holt); Scottish Salmonella, Shigella & Clostridium difficile Reference Laboratory, Stobhill Hospital, Glasgow, UK (D.J. Brown); French National Reference Centre for Escherichia coli and Shigella, Institut Pasteur, Paris, France (I. Filliol, F.-X. Weill); Hanyang University, 55 Hanyangdaehak- ro, Sangnok - gu, Ansan Kyeonggi- do, 426-791, Korea (D-W. Kim); Biology Institute, University of Campinas, Brazil (W.D. da Silveira); Wellcome Trust Sanger Institute, Cambridge, UK (D. Pickard, N. Thomson, J. Parkhill).</p>
<p class="abstract">The current Shigella sonnei pandemic involves geographically associated, multidrug resistant clones. This study has demonstrated that S. sonnei phylogeny can be accurately defined with limited SNPs. By typing 6 informative SNPs using high-resolution melting (HRM) assay major S. sonnei lineages/sublineages can be identified as defined from whole-genome variation.</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23115259" rel="external">23115259</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.02238-12" rel="external">10.1128/JCM.02238-12</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Revising the human mutation rate: implications for understanding human evolution.</h4>
<p class="authors"><span class="author">Scally A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK.</p>
<p class="abstract">It is now possible to make direct measurements of the mutation rate in modern humans using next-generation sequencing. These measurements reveal a value that is approximately half of that previously derived from fossil calibration, and this has implications for our understanding of demographic events in human evolution and other aspects of population genetics. Here, we discuss the implications of a lower-than-expected mutation rate in relation to the timescale of human evolution.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22965354" rel="external">22965354</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg3295" rel="external">10.1038/nrg3295</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insights into hominid evolution from the gorilla genome sequence.</h4>
<p class="authors"><span class="author">Scally A</span>, <span class="author">Dutheil JY</span>, <span class="author">Hillier LW</span>, <span class="author">Jordan GE</span>, <span class="author">Goodhead I</span>, <span class="author">Herrero J</span>, <span class="author">Hobolth A</span>, <span class="author">Lappalainen T</span>, <span class="author">Mailund T</span>, <span class="author">Marques-Bonet T</span>, <span class="author">McCarthy S</span>, <span class="author">Montgomery SH</span>, <span class="author">Schwalie PC</span>, <span class="author">Tang YA</span>, <span class="author">Ward MC</span>, <span class="author">Xue Y</span>, <span class="author">Yngvadottir B</span>, <span class="author">Alkan C</span>, <span class="author">Andersen LN</span>, <span class="author">Ayub Q</span>, <span class="author">Ball EV</span>, <span class="author">Beal K</span>, <span class="author">Bradley BJ</span>, <span class="author">Chen Y</span>, <span class="author">Clee CM</span>, <span class="author">Fitzgerald S</span>, <span class="author">Graves TA</span>, <span class="author">Gu Y</span>, <span class="author">Heath P</span>, <span class="author">Heger A</span>, <span class="author">Karakoc E</span>, <span class="author">Kolb-Kokocinski A</span>, <span class="author">Laird GK</span>, <span class="author">Lunter G</span>, <span class="author">Meader S</span>, <span class="author">Mort M</span>, <span class="author">Mullikin JC</span>, <span class="author">Munch K</span>, <span class="author">O&#39;Connor TD</span>, <span class="author">Phillips AD</span>, <span class="author">Prado-Martinez J</span>, <span class="author">Rogers AS</span>, <span class="author">Sajjadian S</span>, <span class="author">Schmidt D</span>, <span class="author">Shaw K</span>, <span class="author">Simpson JT</span>, <span class="author">Stenson PD</span>, <span class="author">Turner DJ</span>, <span class="author">Vigilant L</span>, <span class="author">Vilella AJ</span>, <span class="author">Whitener W</span>, <span class="author">Zhu B</span>, <span class="author">Cooper DN</span>, <span class="author">de Jong P</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Eichler EE</span>, <span class="author">Flicek P</span>, <span class="author">Goldman N</span>, <span class="author">Mundy NI</span>, <span class="author">Ning Z</span>, <span class="author">Odom DT</span>, <span class="author">Ponting CP</span>, <span class="author">Quail MA</span>, <span class="author">Ryder OA</span>, <span class="author">Searle SM</span>, <span class="author">Warren WC</span>, <span class="author">Wilson RK</span>, <span class="author">Schierup MH</span>, <span class="author">Rogers J</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">Gorillas are humans&#39; closest living relatives after chimpanzees, and are of comparable importance for the study of human origins and evolution. Here we present the assembly and analysis of a genome sequence for the western lowland gorilla, and compare the whole genomes of all extant great ape genera. We propose a synthesis of genetic and fossil evidence consistent with placing the human-chimpanzee and human-chimpanzee-gorilla speciation events at approximately 6 and 10 million years ago. In 30% of the genome, gorilla is closer to human or chimpanzee than the latter are to each other; this is rarer around coding genes, indicating pervasive selection throughout great ape evolution, and has functional consequences in gene expression. A comparison of protein coding genes reveals approximately 500 genes showing accelerated evolution on each of the gorilla, human and chimpanzee lineages, and evidence for parallel acceleration, particularly of genes involved in hearing. We also compare the western and eastern gorilla species, estimating an average sequence divergence time 1.75 million years ago, but with evidence for more recent genetic exchange and a population bottleneck in the eastern species. The use of the genome sequence in these and future analyses will promote a deeper understanding of great ape biology and evolution.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Cancer Research UK: <span class="grant">15603</span>, <span class="grant">A15603</span>; European Research Council: <span class="grant">202218</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0501331</span>, <span class="grant">G0701805</span>; NHGRI NIH HHS: <span class="grant">HG002385</span>, <span class="grant">U54 HG003079</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">075491/Z/04</span>, <span class="grant">077009</span>, <span class="grant">077192</span>, <span class="grant">077198</span>, <span class="grant">089066</span>, <span class="grant">090532</span>, <span class="grant">095908</span>, <span class="grant">WT062023</span>, <span class="grant">WT077009</span>, <span class="grant">WT077192</span>, <span class="grant">WT077198</span>, <span class="grant">WT089066</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">483</span>;<span class="number">7388</span>;<span class="pagerange">169-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22398555">22398555</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3303130">3303130</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10842">10.1038/nature10842</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Waves of retrotransposon expansion remodel genome organization and CTCF binding in multiple mammalian lineages.</h4>
<p class="authors"><span class="author">Schmidt D</span>, <span class="author">Schwalie PC</span>, <span class="author">Wilson MD</span>, <span class="author">Ballester B</span>, <span class="author">Gon&ccedil;alves A</span>, <span class="author">Kutter C</span>, <span class="author">Brown GD</span>, <span class="author">Marshall A</span>, <span class="author">Flicek P</span> and <span class="author">Odom DT</span></p>
<p class="affiliation">Cancer Research UK, Cambridge Research Institute, Li Ka Shing Centre, Robinson Way, Cambridge CB2 0RE, UK.</p><p class="abstract">CTCF-binding locations represent regulatory sequences that are highly constrained over the course of evolution. To gain insight into how these DNA elements are conserved and spread through the genome, we defined the full spectrum of CTCF-binding sites, including a 33/34-mer motif, and identified over five thousand highly conserved, robust, and tissue-independent CTCF-binding locations by comparing ChIP-seq data from six mammals. Our data indicate that activation of retroelements has produced species-specific expansions of CTCF binding in rodents, dogs, and opossum, which often functionally serve as chromatin and transcriptional insulators. We discovered fossilized repeat elements flanking deeply conserved CTCF-binding regions, indicating that similar retrotransposon expansions occurred hundreds of millions of years ago. Repeat-driven dispersal of CTCF binding is a fundamental, ancient, and still highly active mechanism of genome evolution in mammalian lineages.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">15603</span>, <span class="grant">A15603</span>; European Research Council: <span class="grant">202218</span>; Wellcome Trust: <span class="grant">095908</span>, <span class="grant">WT062023</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">148</span>;<span class="number">1-2</span>;<span class="pagerange">335-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22244452">22244452</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3368268">3368268</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2011.11.058">10.1016/j.cell.2011.11.058</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bioinformatics Training Network (BTN): a community resource for bioinformatics trainers.</h4>
<p class="authors"><span class="author">Schneider MV</span>, <span class="author">Walter P</span>, <span class="author">Blatter MC</span>, <span class="author">Watson J</span>, <span class="author">Brazas MD</span>, <span class="author">Rother K</span>, <span class="author">Budd A</span>, <span class="author">Via A</span>, <span class="author">van Gelder CW</span>, <span class="author">Jacob J</span>, <span class="author">Fernandes P</span>, <span class="author">Nyr&ouml;nen TH</span>, <span class="author">De Las Rivas J</span>, <span class="author">Blicher T</span>, <span class="author">Jimenez RC</span>, <span class="author">Loveland J</span>, <span class="author">McDowall J</span>, <span class="author">Jones P</span>, <span class="author">Vaughan BW</span>, <span class="author">Lopez R</span>, <span class="author">Attwood TK</span> and <span class="author">Brooksbank C</span></p>
<p class="affiliation">EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%76%69%63%6B%79%40%65%62%69.%61%63.%75%6B">&#118;&#105;&#99;&#107;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Funding bodies are increasingly recognizing the need to provide graduates and researchers with access to short intensive courses in a variety of disciplines, in order both to improve the general skills base and to provide solid foundations on which researchers may build their careers. In response to the development of &#39;high-throughput biology&#39;, the need for training in the field of bioinformatics, in particular, is seeing a resurgence: it has been defined as a key priority by many Institutions and research programmes and is now an important component of many grant proposals. Nevertheless, when it comes to planning and preparing to meet such training needs, tension arises between the reward structures that predominate in the scientific community which compel individuals to publish or perish, and the time that must be devoted to the design, delivery and maintenance of high-quality training materials. Conversely, there is much relevant teaching material and training expertise available worldwide that, were it properly organized, could be exploited by anyone who needs to provide training or needs to set up a new course. To do this, however, the materials would have to be centralized in a database and clearly tagged in relation to target audiences, learning objectives, etc. Ideally, they would also be peer reviewed, and easily and efficiently accessible for downloading. Here, we present the Bioinformatics Training Network (BTN), a new enterprise that has been initiated to address these needs and review it, respectively, to similar initiatives and collections.</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">3</span>;<span class="pagerange">383-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22110242">22110242</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3357490">3357490</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbib%2Fbbr064">10.1093/bib/bbr064</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">No interactions between previously associated 2-hour glucose gene variants and physical activity or BMI on 2-hour glucose levels.</h4>
<p class="authors"><span class="author">Scott RA</span>, <span class="author">Chu AY</span>, <span class="author">Grarup N</span>, <span class="author">Manning AK</span>, <span class="author">Hivert MF</span>, <span class="author">Shungin D</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Yesupriya A</span>, <span class="author">Barnes D</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Glazer NL</span>, <span class="author">Jackson AU</span>, <span class="author">Kutalik Z</span>, <span class="author">Lagou V</span>, <span class="author">Marek D</span>, <span class="author">Rasmussen-Torvik LJ</span>, <span class="author">Stringham HM</span>, <span class="author">Tanaka T</span>, <span class="author">Aadahl M</span>, <span class="author">Arking DE</span>, <span class="author">Bergmann S</span>, <span class="author">Boerwinkle E</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bornstein SR</span>, <span class="author">Brunner E</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Brage S</span>, <span class="author">Carlson OD</span>, <span class="author">Chen H</span>, <span class="author">Chen YD</span>, <span class="author">Chines PS</span>, <span class="author">Collins FS</span>, <span class="author">Couper DJ</span>, <span class="author">Dennison EM</span>, <span class="author">Dowling NF</span>, <span class="author">Egan JS</span>, <span class="author">Ekelund U</span>, <span class="author">Erdos MR</span>, <span class="author">Forouhi NG</span>, <span class="author">Fox CS</span>, <span class="author">Goodarzi MO</span>, <span class="author">Gr&auml;ssler J</span>, <span class="author">Gustafsson S</span>, <span class="author">Hallmans G</span>, <span class="author">Hansen T</span>, <span class="author">Hingorani A</span>, <span class="author">Holloway JW</span>, <span class="author">Hu FB</span>, <span class="author">Isomaa B</span>, <span class="author">Jameson KA</span>, <span class="author">Johansson I</span>, <span class="author">Jonsson A</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Kivimaki M</span>, <span class="author">Kovacs P</span>, <span class="author">Kumari M</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Lecoeur C</span>, <span class="author">L&eacute;vy-Marchal C</span>, <span class="author">Li G</span>, <span class="author">Loos RJ</span>, <span class="author">Lyssenko V</span>, <span class="author">Marmot M</span>, <span class="author">Marques-Vidal P</span>, <span class="author">Morken MA</span>, <span class="author">M&uuml;ller G</span>, <span class="author">North KE</span>, <span class="author">Pankow JS</span>, <span class="author">Payne F</span>, <span class="author">Prokopenko I</span>, <span class="author">Psaty BM</span>, <span class="author">Renstr&ouml;m F</span>, <span class="author">Rice K</span>, <span class="author">Rotter JI</span>, <span class="author">Rybin D</span>, <span class="author">Sandholt CH</span>, <span class="author">Sayer AA</span>, <span class="author">Shrader P</span>, <span class="author">Schwarz PE</span>, <span class="author">Siscovick DS</span>, <span class="author">Stanc&aacute;kov&aacute; A</span>, <span class="author">Stumvoll M</span>, <span class="author">Teslovich TM</span>, <span class="author">Waeber G</span>, <span class="author">Williams GH</span>, <span class="author">Witte DR</span>, <span class="author">Wood AR</span>, <span class="author">Xie W</span>, <span class="author">Boehnke M</span>, <span class="author">Cooper C</span>, <span class="author">Ferrucci L</span>, <span class="author">Froguel P</span>, <span class="author">Groop L</span>, <span class="author">Kao WH</span>, <span class="author">Vollenweider P</span>, <span class="author">Walker M</span>, <span class="author">Watanabe RM</span>, <span class="author">Pedersen O</span>, <span class="author">Meigs JB</span>, <span class="author">Ingelsson E</span>, <span class="author">Barroso I</span>, <span class="author">Florez JC</span>, <span class="author">Franks PW</span>, <span class="author">Dupuis J</span>, <span class="author">Wareham NJ</span> and <span class="author">Langenberg C</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Institute of Metabolic Science, Addenbrooke&rsquo;s Hospital, Cambridge, UK. <a href="mailto:%72%6F%62%65%72%74.%73%63%6F%74%74%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#114;&#111;&#98;&#101;&#114;&#116;&#46;&#115;&#99;&#111;&#116;&#116;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Gene-lifestyle interactions have been suggested to contribute to the development of type 2 diabetes. Glucose levels 2 h after a standard 75-g glucose challenge are used to diagnose diabetes and are associated with both genetic and lifestyle factors. However, whether these factors interact to determine 2-h glucose levels is unknown. We meta-analyzed single nucleotide polymorphism (SNP) &times; BMI and SNP &times; physical activity (PA) interaction regression models for five SNPs previously associated with 2-h glucose levels from up to 22 studies comprising 54,884 individuals without diabetes. PA levels were dichotomized, with individuals below the first quintile classified as inactive (20%) and the remainder as active (80%). BMI was considered a continuous trait. Inactive individuals had higher 2-h glucose levels than active individuals (&beta; = 0.22 mmol/L [95% CI 0.13-0.31], P = 1.63 &times; 10(-6)). All SNPs were associated with 2-h glucose (&beta; = 0.06-0.12 mmol/allele, P &le; 1.53 &times; 10(-7)), but no significant interactions were found with PA (P &gt; 0.18) or BMI (P &ge; 0.04). In this large study of gene-lifestyle interaction, we observed no interactions between genetic and lifestyle factors, both of which were associated with 2-h glucose. It is perhaps unlikely that top loci from genome-wide association studies will exhibit strong subgroup-specific effects, and may not, therefore, make the best candidates for the study of interactions.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/07/008/23674</span>; Medical Research Council: <span class="grant">G0100222</span>, <span class="grant">G0701863</span>, <span class="grant">G0902037</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179473</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NCATS NIH HHS: <span class="grant">UL1 TR000124</span>; NCRR NIH HHS: <span class="grant">UL1 RR024148</span>, <span class="grant">UL1 RR025741</span>; NHLBI NIH HHS: <span class="grant">T32 HL007575</span>; NIDDK NIH HHS: <span class="grant">K24 DK080140</span>, <span class="grant">P30 DK063491</span>, <span class="grant">R01 DK072041</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2012</span>;<span class="volume">61</span>;<span class="number">5</span>;<span class="pagerange">1291-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22415877">22415877</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3331745">3331745</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb11-0973">10.2337/db11-0973</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale association analyses identify new loci influencing glycemic traits and provide insight into the underlying biological pathways.</h4>
<p class="authors"><span class="author">Scott RA</span>, <span class="author">Lagou V</span>, <span class="author">Welch RP</span>, <span class="author">Wheeler E</span>, <span class="author">Montasser ME</span>, <span class="author">Luan J</span>, <span class="author">M&auml;gi R</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Rehnberg E</span>, <span class="author">Gustafsson S</span>, <span class="author">Kanoni S</span>, <span class="author">Rasmussen-Torvik LJ</span>, <span class="author">Yengo L</span>, <span class="author">Lecoeur C</span>, <span class="author">Shungin D</span>, <span class="author">Sanna S</span>, <span class="author">Sidore C</span>, <span class="author">Johnson PC</span>, <span class="author">Jukema JW</span>, <span class="author">Johnson T</span>, <span class="author">Mahajan A</span>, <span class="author">Verweij N</span>, <span class="author">Thorleifsson G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Shah S</span>, <span class="author">Smith AV</span>, <span class="author">Sennblad B</span>, <span class="author">Gieger C</span>, <span class="author">Salo P</span>, <span class="author">Perola M</span>, <span class="author">Timpson NJ</span>, <span class="author">Evans DM</span>, <span class="author">Pourcain BS</span>, <span class="author">Wu Y</span>, <span class="author">Andrews JS</span>, <span class="author">Hui J</span>, <span class="author">Bielak LF</span>, <span class="author">Zhao W</span>, <span class="author">Horikoshi M</span>, <span class="author">Navarro P</span>, <span class="author">Isaacs A</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Stirrups K</span>, <span class="author">Vitart V</span>, <span class="author">Hayward C</span>, <span class="author">Esko T</span>, <span class="author">Mihailov E</span>, <span class="author">Fraser RM</span>, <span class="author">Fall T</span>, <span class="author">Voight BF</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Chen H</span>, <span class="author">Lindgren CM</span>, <span class="author">Morris AP</span>, <span class="author">Rayner NW</span>, <span class="author">Robertson N</span>, <span class="author">Rybin D</span>, <span class="author">Liu CT</span>, <span class="author">Beckmann JS</span>, <span class="author">Willems SM</span>, <span class="author">Chines PS</span>, <span class="author">Jackson AU</span>, <span class="author">Kang HM</span>, <span class="author">Stringham HM</span>, <span class="author">Song K</span>, <span class="author">Tanaka T</span>, <span class="author">Peden JF</span>, <span class="author">Goel A</span>, <span class="author">Hicks AA</span>, <span class="author">An P</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">Franco-Cereceda A</span>, <span class="author">Folkersen L</span>, <span class="author">Marullo L</span>, <span class="author">Jansen H</span>, <span class="author">Oldehinkel AJ</span>, <span class="author">Bruinenberg M</span>, <span class="author">Pankow JS</span>, <span class="author">North KE</span>, <span class="author">Forouhi NG</span>, <span class="author">Loos RJ</span>, <span class="author">Edkins S</span>, <span class="author">Varga TV</span>, <span class="author">Hallmans G</span>, <span class="author">Oksa H</span>, <span class="author">Antonella M</span>, <span class="author">Nagaraja R</span>, <span class="author">Trompet S</span>, <span class="author">Ford I</span>, <span class="author">Bakker SJ</span>, <span class="author">Kong A</span>, <span class="author">Kumari M</span>, <span class="author">Gigante B</span>, <span class="author">Herder C</span>, <span class="author">Munroe PB</span>, <span class="author">Caulfield M</span>, <span class="author">Antti J</span>, <span class="author">Mangino M</span>, <span class="author">Small K</span>, <span class="author">Miljkovic I</span>, <span class="author">Liu Y</span>, <span class="author">Atalay M</span>, <span class="author">Kiess W</span>, <span class="author">James AL</span>, <span class="author">Rivadeneira F</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Palmer CN</span>, <span class="author">Doney AS</span>, <span class="author">Willemsen G</span>, <span class="author">Smit JH</span>, <span class="author">Campbell S</span>, <span class="author">Polasek O</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Hercberg S</span>, <span class="author">Dimitriou M</span>, <span class="author">Bolton JL</span>, <span class="author">Fowkes GR</span>, <span class="author">Kovacs P</span>, <span class="author">Lindstr&ouml;m J</span>, <span class="author">Zemunik T</span>, <span class="author">Bandinelli S</span>, <span class="author">Wild SH</span>, <span class="author">Basart HV</span>, <span class="author">Rathmann W</span>, <span class="author">Grallert H</span>, <span class="author">DIAbetes Genetics Replication and Meta-analysis (DIAGRAM) Consortium</span>, <span class="author">Maerz W</span>, <span class="author">Kleber ME</span>, <span class="author">Boehm BO</span>, <span class="author">Peters A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Province MA</span>, <span class="author">Borecki IB</span>, <span class="author">Hastie ND</span>, <span class="author">Rudan I</span>, <span class="author">Campbell H</span>, <span class="author">Watkins H</span>, <span class="author">Farrall M</span>, <span class="author">Stumvoll M</span>, <span class="author">Ferrucci L</span>, <span class="author">Waterworth DM</span>, <span class="author">Bergman RN</span>, <span class="author">Collins FS</span>, <span class="author">Tuomilehto J</span>, <span class="author">Watanabe RM</span>, <span class="author">de Geus EJ</span>, <span class="author">Penninx BW</span>, <span class="author">Hofman A</span>, <span class="author">Oostra BA</span>, <span class="author">Psaty BM</span>, <span class="author">Vollenweider P</span>, <span class="author">Wilson JF</span>, <span class="author">Wright AF</span>, <span class="author">Hovingh GK</span>, <span class="author">Metspalu A</span>, <span class="author">Uusitupa M</span>, <span class="author">Magnusson PK</span>, <span class="author">Kyvik KO</span>, <span class="author">Kaprio J</span>, <span class="author">Price JF</span>, <span class="author">Dedoussis GV</span>, <span class="author">Deloukas P</span>, <span class="author">Meneton P</span>, <span class="author">Lind L</span>, <span class="author">Boehnke M</span>, <span class="author">Shuldiner AR</span>, <span class="author">van Duijn CM</span>, <span class="author">Morris AD</span>, <span class="author">Toenjes A</span>, <span class="author">Peyser PA</span>, <span class="author">Beilby JP</span>, <span class="author">K&ouml;rner A</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Bornstein SR</span>, <span class="author">Schwarz PE</span>, <span class="author">Lakka TA</span>, <span class="author">Rauramaa R</span>, <span class="author">Adair LS</span>, <span class="author">Smith GD</span>, <span class="author">Spector TD</span>, <span class="author">Illig T</span>, <span class="author">de Faire U</span>, <span class="author">Hamsten A</span>, <span class="author">Gudnason V</span>, <span class="author">Kivimaki M</span>, <span class="author">Hingorani A</span>, <span class="author">Keinanen-Kiukaanniemi SM</span>, <span class="author">Saaristo TE</span>, <span class="author">Boomsma DI</span>, <span class="author">Stefansson K</span>, <span class="author">van der Harst P</span>, <span class="author">Dupuis J</span>, <span class="author">Pedersen NL</span>, <span class="author">Sattar N</span>, <span class="author">Harris TB</span>, <span class="author">Cucca F</span>, <span class="author">Ripatti S</span>, <span class="author">Salomaa V</span>, <span class="author">Mohlke KL</span>, <span class="author">Balkau B</span>, <span class="author">Froguel P</span>, <span class="author">Pouta A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Wareham NJ</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">McCarthy MI</span>, <span class="author">Franks PW</span>, <span class="author">Meigs JB</span>, <span class="author">Teslovich TM</span>, <span class="author">Florez JC</span>, <span class="author">Langenberg C</span>, <span class="author">Ingelsson E</span>, <span class="author">Prokopenko I</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge, UK.</p><p class="abstract">Through genome-wide association meta-analyses of up to 133,010 individuals of European ancestry without diabetes, including individuals newly genotyped using the Metabochip, we have increased the number of confirmed loci influencing glycemic traits to 53, of which 33 also increase type 2 diabetes risk (q &lt; 0.05). Loci influencing fasting insulin concentration showed association with lipid levels and fat distribution, suggesting impact on insulin resistance. Gene-based analyses identified further biologically plausible loci, suggesting that additional loci beyond those reaching genome-wide significance are likely to represent real associations. This conclusion is supported by an excess of directionally consistent and nominally significant signals between discovery and follow-up studies. Functional analysis of these newly discovered loci will further improve our understanding of glycemic control.</p>
<p class="grants">Funded by: AHRQ HHS: <span class="grant">HS06516</span>; Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; British Heart Foundation: <span class="grant">PG/07/133/24260</span>, <span class="grant">RG/07/008/23674</span>; Chief Scientist Office: <span class="grant">CZB/4/672</span>, <span class="grant">CZB/4/710</span>; Department of Health; FIC NIH HHS: <span class="grant">TW05596</span>; Medical Research Council: <span class="grant">74882</span>, <span class="grant">85374</span>, <span class="grant">G0100222</span>, <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">G0701863</span>, <span class="grant">G0800582</span>, <span class="grant">G0902037</span>, <span class="grant">G1000143</span>, <span class="grant">G1002084</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">G9521010</span>, <span class="grant">MC_PC_U127561128</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">U.1061.00.001 (79471)</span>; NCATS NIH HHS: <span class="grant">UL1 TR000130</span>; NCRR NIH HHS: <span class="grant">M01 RR 16500</span>, <span class="grant">M01-RR00425</span>, <span class="grant">RR20649</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">1Z01-HG000024</span>, <span class="grant">N01-HG-65403</span>, <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>, <span class="grant">HL075366</span>, <span class="grant">HL080295</span>, <span class="grant">HL085144</span>, <span class="grant">HL087652</span>, <span class="grant">HL087660</span>, <span class="grant">HL100245</span>, <span class="grant">HL105756</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01-HC-85239</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01-HL-087700</span>, <span class="grant">R01-HL-088215</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL84756</span>; NIA NIH HHS: <span class="grant">1R01AG032098-01A1</span>, <span class="grant">AG-023629</span>, <span class="grant">AG-027058</span>, <span class="grant">AG-15928</span>, <span class="grant">AG-20098</span>, <span class="grant">AG028555</span>, <span class="grant">AG04563</span>, <span class="grant">AG08724</span>, <span class="grant">AG08861</span>, <span class="grant">AG10175</span>, <span class="grant">AG13196</span>, <span class="grant">N01-AG-1-2100</span>, <span class="grant">N01-AG-1-2109</span>, <span class="grant">N01AG62101</span>, <span class="grant">N01AG62103</span>, <span class="grant">N01AG62106</span>, <span class="grant">R01 AG18728</span>, <span class="grant">T32 AG000219</span>; NIAMS NIH HHS: <span class="grant">K08AR055688</span>; NICHD NIH HHS: <span class="grant">R24 HD050924</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">DK078150</span>, <span class="grant">DK56350</span>, <span class="grant">K24 DK080140</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30 DK72488</span>, <span class="grant">P60DK79637</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK078150</span>, <span class="grant">R01 DK078616</span>, <span class="grant">R01 DK54261</span>, <span class="grant">R01-DK-075681</span>, <span class="grant">R01-DK-8925601</span>, <span class="grant">R01-DK062370</span>, <span class="grant">R01-DK072193</span>; NIEHS NIH HHS: <span class="grant">ES10126</span>, <span class="grant">P30 ES010126</span>; NIGMS NIH HHS: <span class="grant">U01 GM074518</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span>, <span class="grant">U24 MH068457-06</span>; NLM NIH HHS: <span class="grant">LM010098</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>, <span class="grant">R01D0042157-01A</span>; Wellcome Trust: <span class="grant">075491/Z/04</span>, <span class="grant">076467</span>, <span class="grant">081682</span>, <span class="grant">083948</span>, <span class="grant">090532</span>, <span class="grant">092731</span>, <span class="grant">098017</span>, <span class="grant">098051</span>, <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">9</span>;<span class="pagerange">991-1005</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22885924">22885924</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3433394">3433394</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2385">10.1038/ng.2385</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Plasmodium calcium-dependent protein kinase controls zygote development and transmission by translationally activating repressed mRNAs.</h4>
<p class="authors"><span class="author">Sebastian S</span>, <span class="author">Brochet M</span>, <span class="author">Collins MO</span>, <span class="author">Schwach F</span>, <span class="author">Jones ML</span>, <span class="author">Goulding D</span>, <span class="author">Rayner JC</span>, <span class="author">Choudhary JS</span> and <span class="author">Billker O</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Calcium-dependent protein kinases (CDPKs) play key regulatory roles in the life cycle of the malaria parasite, but in many cases their precise molecular functions are unknown. Using the rodent malaria parasite Plasmodium berghei, we show that CDPK1, which is known to be essential in the asexual blood stage of the parasite, is expressed in all life stages and is indispensable during the sexual mosquito life-cycle stages. Knockdown of CDPK1 in sexual stages resulted in developmentally arrested parasites and prevented mosquito transmission, and these effects were independent of the previously proposed function for CDPK1 in regulating parasite motility. In-depth translational and transcriptional profiling of arrested parasites revealed that CDPK1 translationally activates mRNA species in the developing zygote that in macrogametes remain repressed via their 3&#39; and 5&#39;UTRs. These findings indicate that CDPK1 is a multifunctional protein that translationally regulates mRNAs to ensure timely and stage-specific protein expression.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; Wellcome Trust: <span class="grant">079643/Z/06/Z</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">Cell host &amp; microbe</span>&nbsp;<span class="year">2012</span>;<span class="volume">12</span>;<span class="number">1</span>;<span class="pagerange">9-19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22817984">22817984</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3414820">3414820</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.chom.2012.05.014">10.1016/j.chom.2012.05.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The dynamics of genome-wide DNA methylation reprogramming in mouse primordial germ cells.</h4>
<p class="authors"><span class="author">Seisenberger S</span>, <span class="author">Andrews S</span>, <span class="author">Krueger F</span>, <span class="author">Arand J</span>, <span class="author">Walter J</span>, <span class="author">Santos F</span>, <span class="author">Popp C</span>, <span class="author">Thienpont B</span>, <span class="author">Dean W</span> and <span class="author">Reik W</span></p>
<p class="affiliation">Epigenetics Programme, The Babraham Institute, Cambridge CB22 3AT, UK.</p><p class="abstract">Genome-wide DNA methylation reprogramming occurs in mouse primordial germ cells (PGCs) and preimplantation embryos, but the precise dynamics and biological outcomes are largely unknown. We have carried out whole-genome bisulfite sequencing (BS-Seq) and RNA-Seq across key stages from E6.5 epiblast to E16.5 PGCs. Global loss of methylation takes place during PGC expansion and migration&nbsp;with evidence for passive demethylation, but sequences that carry long-term epigenetic memory (imprints, CpG islands on the X chromosome, germline-specific genes) only become demethylated upon entry of PGCs into the gonads. The transcriptional profile of PGCs is tightly controlled despite global hypomethylation, with transient expression of the pluripotency network, suggesting that reprogramming and pluripotency are inextricably linked. Our results provide a framework for the understanding of the epigenetic ground state of pluripotency in the germline.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council: <span class="grant">G0700098</span>, <span class="grant">G0801156</span>; Wellcome Trust: <span class="grant">095645</span></p>
<p class="publication"><span class="title">Molecular cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">48</span>;<span class="number">6</span>;<span class="pagerange">849-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23219530">23219530</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3533687">3533687</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molcel.2012.11.001">10.1016/j.molcel.2012.11.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer develops, progresses and responds to therapies through restricted perturbation of the protein-protein interaction network.</h4>
<p class="authors"><span class="author">Serra-Musach J</span>, <span class="author">Aguilar H</span>, <span class="author">Iorio F</span>, <span class="author">Comellas F</span>, <span class="author">Berenguer A</span>, <span class="author">Brunet J</span>, <span class="author">Saez-Rodriguez J</span> and <span class="author">Pujana MA</span></p>
<p class="affiliation">Translational Research Laboratory, Breast Cancer Unit, Catalan Institute of Oncology (ICO), Bellvitge Institute for Biomedical Research (IDIBELL), Gran via 199, L&#39;Hospitalet del Llobregat, Barcelona 08908, Catalonia, Spain.</p><p class="abstract">The products of genes mutated or differentially expressed in cancer tend to occupy central positions within the network of protein-protein interactions, or the interactome network. Integration of different types of gene and protein relationships has considerably increased the understanding of the mechanisms of carcinogenesis, while also enhancing the applicability of expression signatures. In this scenario, however, it remains unknown how cancer develops, progresses and responds to therapies in a potentially controlled manner at the systems level. Here, by applying the concepts of load transfer and cascading failures in power grids, we examine the impact and transmission of cancer-related gene expression changes in the interactome network. Relative to random perturbations, this study reveals topological robustness associated with all cancer conditions. In addition, experimental perturbation of a central cancer node, which consists of over-expression of the &alpha;-synuclein (SNCA) protein in MCF7 breast cancer cells, also reveals robustness. Conversely, a search for proteins with an opposite topological impact identifies the autophagy pathway. Mechanistically, the existence of smaller shortest paths among cancer-related proteins appears to be a topological feature that partially contributes to the restricted perturbation of the network. Together, the results of this study suggest that cancer develops, progresses and responds to therapies following controlled, restricted perturbation of the interactome network.</p>
<p class="publication"><span class="title">Integrative biology : quantitative biosciences from nano to macro</span>&nbsp;<span class="year">2012</span>;<span class="volume">4</span>;<span class="number">9</span>;<span class="pagerange">1038-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22806580">22806580</a>; DOI: <a href="http://dx.doi.org/10.1039%2Fc2ib20052j">10.1039/c2ib20052j</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide meta-analysis of association studies of Cloninger's Temperament Scales.</h4>
<p class="authors"><span class="author">Service SK</span>, <span class="author">Verweij KJ</span>, <span class="author">Lahti J</span>, <span class="author">Congdon E</span>, <span class="author">Ekelund J</span>, <span class="author">Hintsanen M</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Widen E</span>, <span class="author">Taanila A</span>, <span class="author">Veijola J</span>, <span class="author">Heath AC</span>, <span class="author">Madden PA</span>, <span class="author">Montgomery GW</span>, <span class="author">Sabatti C</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Palotie A</span>, <span class="author">Raitakari O</span>, <span class="author">Viikari J</span>, <span class="author">Martin NG</span>, <span class="author">Eriksson JG</span>, <span class="author">Keltikangas-J&auml;rvinen L</span>, <span class="author">Wray NR</span> and <span class="author">Freimer NB</span></p>
<p class="affiliation">Center for Neurobehavioral Genetics, University of California, Los Angeles, CA, USA.</p>
<p class="abstract">Temperament has a strongly heritable component, yet multiple independent genome-wide studies have failed to identify significant genetic associations. We have assembled the largest sample to date of persons with genome-wide genotype data, who have been assessed with Cloninger's Temperament and Character Inventory. Sum scores for novelty seeking, harm avoidance, reward dependence and persistence have been measured in over 11â€‰000 persons collected in four different cohorts. Our study had >80% power to identify genome-wide significant loci (P<1.25 Ã— 10(-8), with correction for testing four scales) accounting for î‹¶0.4% of the phenotypic variance in temperament scales. Using meta-analysis techniques, gene-based tests and pathway analysis we have tested over 1.2 million single-nucleotide polymorphisms (SNPs) for association to each of the four temperament dimensions. We did not discover any SNPs, genes, or pathways to be significantly related to the four temperament dimensions, after correcting for multiple testing. Less than 1% of the variability in any temperament dimension appears to be accounted for by a risk score derived from the SNPs showing strongest association to the temperament dimensions. Elucidation of genetic loci significantly influencing temperament and personality will require potentially very large samples, and/or a more refined phenotype. Item response theory methodology may be a way to incorporate data from cohorts assessed with multiple personality instruments, and might be a method by which a large sample of a more refined phenotype could be acquired.</p>
<p class="publication"><span class="title">Translational psychiatry</span>&nbsp;<span class="year">2012</span>;<span class="volume">2</span>;<span class="pagerange">e116</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22832960" rel="external">22832960</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3365256" rel="external">3365256</a>; DOI: <a href="http://dx.doi.org/10.1038%2Ftp.2012.37" rel="external">10.1038/tp.2012.37</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Finding NECA: zebrafish screen identifies key signalling pathway in &beta;-cell regeneration.</h4>
<p class="authors"><span class="author">Seth A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%73%32%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#115;&#50;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Disease models &amp; mechanisms</span>&nbsp;<span class="year">2012</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">709-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23115198">23115198</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3484851">3484851</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdmm.010876">10.1242/dmm.010876</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beyond the palaeomicrobiology.</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p><p class="abstract">This month&#39;s Genome Watch highlights the power of palaeomicrobiology in extracting detailed information about the genomes of ancient microorganisms.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">4</span>;<span class="pagerange">240</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22406951" rel="external">22406951</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2768" rel="external">10.1038/nrmicro2768</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome sequencing of bacterial sexually transmitted infections: implications for clinicians.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Pathogen Genomics, The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p>
<p class="abstract">PURPOSE OF REVIEW: Increasingly, genomics is being used to answer detailed clinical questions. Although genome analysis of bacterial sexually transmitted infections (STIs) lags far behind that of many other bacterial pathogens, genomics can reveal previously inaccessible aspects of pathogen biology. RECENT FINDINGS: Comparative genomic studies on the most common bacterial STI, chlamydia, have revolutionized our understanding of this intracellular bacterium, demonstrating that it undergoes extensive recombination and that the traditional typing schemes can be misleading. Genome projects can also help us to understand the recently observed phenomenon of 'diagnostic escape' seen in both Chlamydia trachomatis and Neisseria gonorrhoeae. SUMMARY: The routine use of genomics in clinical settings is becoming a reality. For STIs, a primary requirement is an understanding of the diversity of circulating strains and how they change over time. This can help to inform future studies and allow us to address real clinical issues such as outbreak identification, global spread of successful clones and antimicrobial resistance monitoring.</p>
<p class="publication"><span class="title">Current opinion in infectious diseases</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23221768" rel="external">23221768</a>; DOI: <a href="http://dx.doi.org/10.1097%2FQCO.0b013e32835c2159" rel="external">10.1097/QCO.0b013e32835c2159</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structure, diversity, and mobility of the Salmonella pathogenicity island 7 family of integrative and conjugative elements within Enterobacteriaceae.</h4>
<p class="authors"><span class="author">Seth-Smith HM</span>, <span class="author">Fookes MC</span>, <span class="author">Okoro CK</span>, <span class="author">Baker S</span>, <span class="author">Harris SR</span>, <span class="author">Scott P</span>, <span class="author">Pickard D</span>, <span class="author">Quail MA</span>, <span class="author">Churcher C</span>, <span class="author">Sanders M</span>, <span class="author">Harmse J</span>, <span class="author">Dougan G</span>, <span class="author">Parkhill J</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Pathogen Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK.</p><p class="abstract">Integrative and conjugative elements (ICEs) are self-mobile genetic elements found in the genomes of some bacteria. These elements may confer a fitness advantage upon their host bacteria through the cargo genes that they carry. Salmonella pathogenicity island 7 (SPI-7), found within some pathogenic strains of Salmonella enterica, possesses features indicative of an ICE and carries genes implicated in virulence. We aimed to identify and fully analyze ICEs related to SPI-7 within the genus Salmonella and other Enterobacteriaceae. We report the sequence of two novel SPI-7-like elements, found within strains of Salmonella bongori, which share 97% nucleotide identity over conserved regions with SPI-7 and with each other. Although SPI-7 within Salmonella enterica serovar Typhi appears to be fixed within the chromosome, we present evidence that these novel elements are capable of excision and self-mobility. Phylogenetic analyses show that these Salmonella mobile elements share an ancestor which existed approximately 3.6 to 15.8 million years ago. Additionally, we identified more distantly related ICEs, with distinct cargo regions, within other strains of Salmonella as well as within Citrobacter, Erwinia, Escherichia, Photorhabdus, and Yersinia species. In total, we report on a collection of 17 SPI-7 related ICEs within enterobacterial species, of which six are novel. Using comparative and mutational studies, we have defined a core of 27 genes essential for conjugation. We present a growing family of SPI-7-related ICEs whose mobility, abundance, and cargo variability indicate that these elements may have had a large impact on the evolution of the Enterobacteriaceae.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2012</span>;<span class="volume">194</span>;<span class="number">6</span>;<span class="pagerange">1494-504</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22247511" rel="external">22247511</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3294861" rel="external">3294861</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.06403-11" rel="external">10.1128/JB.06403-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">optiCall: a robust genotype-calling algorithm for rare, low-frequency and common variants.</h4>
<p class="authors"><span class="author">Shah TS</span>, <span class="author">Liu JZ</span>, <span class="author">Floyd JA</span>, <span class="author">Morris JA</span>, <span class="author">Wirth N</span>, <span class="author">Barrett JC</span> and <span class="author">Anderson CA</span></p>
<p class="affiliation">Human Genetics, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract"><strong>Motivation</strong>: Existing microarray genotype-calling algorithms adopt either SNP-by-SNP (SNP-wise) or sample-by-sample (sample-wise) approaches to calling. We have developed a novel genotype-calling algorithm for the Illumina platform, optiCall, that uses both SNP-wise and sample-wise calling to more accurately ascertain genotypes at rare, low-frequency and common variants.</p><p class="abstract"><strong>Results</strong>: Using data from 4537 individuals from the 1958 British Birth Cohort genotyped on the Immunochip, we estimate the proportion of SNPs lost to downstream analysis due to false quality control failures, and rare variants misclassified as monomorphic, is only 1.38% with optiCall, in comparison to 3.87, 7.85 and 4.09% for Illuminus, GenoSNP and GenCall, respectively. We show that optiCall accurately captures rare variants and can correctly account for SNPs where probe intensity clouds are shifted from their expected positions.</p><p class="abstract">optiCall is implemented in C++ for use on UNIX operating systems and is available for download at http://www.sanger.ac.uk/resources/software/opticall/.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1001799</span>; Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">12</span>;<span class="pagerange">1598-603</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22500001">22500001</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3371828">3371828</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbts180">10.1093/bioinformatics/bts180</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A dual function of Bcl11b/Ctip2 in hippocampal neurogenesis.</h4>
<p class="authors"><span class="author">Simon R</span>, <span class="author">Brylka H</span>, <span class="author">Schwegler H</span>, <span class="author">Venkataramanappa S</span>, <span class="author">Andratschke J</span>, <span class="author">Wiegreffe C</span>, <span class="author">Liu P</span>, <span class="author">Fuchs E</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Birchmeier C</span> and <span class="author">Britsch S</span></p>
<p class="affiliation">Institute of Molecular and Cellular Anatomy, Ulm University, Ulm, Germany.</p><p class="abstract">The development of the dentate gyrus is characterized by distinct phases establishing a durable stem-cell pool required for postnatal and adult neurogenesis. Here, we report that Bcl11b/Ctip2, a zinc finger transcription factor expressed in postmitotic neurons, plays a critical role during postnatal development of the dentate gyrus. Forebrain-specific ablation of Bcl11b uncovers dual phase-specific functions of Bcl11b demonstrated by feedback control of the progenitor cell compartment as well as regulation of granule cell differentiation, leading to impaired spatial learning and memory in mutants. Surprisingly, we identified Desmoplakin as a direct transcriptional target of Bcl11b. Similarly to Bcl11b, postnatal neurogenesis and granule cell differentiation are impaired in Desmoplakin mutants. Re-expression of Desmoplakin in Bcl11b mutants rescues impaired neurogenesis, suggesting Desmoplakin to be an essential downstream effector of Bcl11b in hippocampal development. Together, our data define an important novel regulatory pathway in hippocampal development, by linking transcriptional functions of Bcl11b to Desmoplakin, a molecule known to act on cell adhesion.</p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22588081" rel="external">22588081</a>; DOI: <a href="http://dx.doi.org/10.1038%2Femboj.2012.142" rel="external">10.1038/emboj.2012.142</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficient de novo assembly of large genomes using compressed data structures.</h4>
<p class="authors"><span class="author">Simpson JT</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">De novo genome sequence assembly is important both to generate new sequence assemblies for previously uncharacterized genomes and to identify the genome sequence of individuals in a reference-unbiased way. We present memory efficient data structures and algorithms for assembly using the FM-index derived from the compressed Burrows-Wheeler transform, and a new assembler based on these called SGA (String Graph Assembler). We describe algorithms to error-correct, assemble, and scaffold large sets of sequence data. SGA uses the overlap-based string graph model of assembly, unlike most de novo assemblers that rely on de Bruijn graphs, and is simply parallelizable. We demonstrate the error correction and assembly performance of SGA on 1.2 billion sequence reads from a human genome, which we are able to assemble using 54 GB of memory. The resulting contigs are highly accurate and contiguous, while covering 95% of the reference genome (excluding contigs &lt;200 bp in length). Because of the low memory requirements and parallelization without requiring inter-process communication, SGA provides the first practical assembler to our knowledge for a mammalian-sized genome on a low-end computing cluster.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077192</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2012</span>;<span class="volume">22</span>;<span class="number">3</span>;<span class="pagerange">549-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22156294" rel="external">22156294</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3290790" rel="external">3290790</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.126953.111" rel="external">10.1101/gr.126953.111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a novel Xenopus tropicalis cell line as a model for in vitro studies.</h4>
<p class="authors"><span class="author">Sinzelle L</span>, <span class="author">Thuret R</span>, <span class="author">Hwang HY</span>, <span class="author">Herszberg B</span>, <span class="author">Paillard E</span>, <span class="author">Bronchain OJ</span>, <span class="author">Stemple DL</span>, <span class="author">Dhorne-Pollet S</span> and <span class="author">Pollet N</span></p>
<p class="affiliation">Institute of Systems and Synthetic Biology, Genopole, CNRS, Universit&eacute; d&#39;Evry Val d&#39;Essonne, Genavenir 3-Genopole campus 3, 1 rue Pierre Fontaine, Evry, France.</p><p class="abstract">Cell lines are useful tools to facilitate in vitro studies of many biological and molecular processes. We describe a new permanent fibroblast-type cell line obtained from disaggregated Xenopus tropicalis limb bud. The cell line population doubling time was ~24 h. Its karyotype was genetically stable with a chromosome number of 2n = 21 and a chromosome 10 trisomy. These cells could be readily transfected and expressed transgenes faithfully. We obtained stable transformants using transposon-based gene transfer technology. These cells responded to thyroid hormone and thus can provide a complementary research tool to study thyroid hormone signaling events. In conclusion, this cell line baptized &quot;Speedy&quot; should prove useful to couple in vitro and in vivo biological studies in the X. tropicalis frog model.</p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2012</span>;<span class="volume">50</span>;<span class="number">3</span>;<span class="pagerange">316-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22083648" rel="external">22083648</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3503257" rel="external">3503257</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvg.20822" rel="external">10.1002/dvg.20822</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Monocyte gene expression signature of patients with early onset coronary artery disease.</h4>
<p class="authors"><span class="author">Sivapalaratnam S</span>, <span class="author">Basart H</span>, <span class="author">Watkins NA</span>, <span class="author">Maiwald S</span>, <span class="author">Rendon A</span>, <span class="author">Krishnan U</span>, <span class="author">Sondermeijer BM</span>, <span class="author">Creemers EE</span>, <span class="author">Pinto-Sietsma SJ</span>, <span class="author">Hovingh K</span>, <span class="author">Ouwehand WH</span>, <span class="author">Kastelein JJ</span>, <span class="author">Goodall AH</span> and <span class="author">Trip MD</span></p>
<p class="affiliation">Department of Vascular Medicine, Academic Medical Center, Amsterdam, The Netherlands. <a href="mailto:%73.%73%69%76%61%70%61%6C%61%72%61%74%6E%61%6D%40%61%6D%63.%75%76%61.%6E%6C">&#115;&#46;&#115;&#105;&#118;&#97;&#112;&#97;&#108;&#97;&#114;&#97;&#116;&#110;&#97;&#109;&#64;&#97;&#109;&#99;&#46;&#117;&#118;&#97;&#46;&#110;&#108;</a></p><p class="abstract">The burden of cardiovascular disease (CVD) cannot be fully addressed by therapy targeting known pathophysiological pathways. Even with stringent control of all risk factors CVD events are only diminished by half. A number of additional pathways probably play a role in the development of CVD and might serve as novel therapeutic targets. Genome wide expression studies represent a powerful tool to identify such novel pathways. We compared the expression profiles in monocytes from twenty two young male patients with premature familial CAD with those from controls matched for age, sex and smoking status, without a family history of CVD. Since all patients were on statins and aspirin treatment, potentially affecting the expression of genes in monocytes, twelve controls were subsequently treated with simvastatin and aspirin for 6 and 2 weeks, respectively. By whole genome expression arrays six genes were identified to have differential expression in the monocytes of patients versus controls; ABCA1, ABCG1 and RGS1 were downregulated in patients, whereas ADRB2, FOLR3 and GSTM1 were upregulated. Differential expression of all genes, apart from GSTM1, was confirmed by qPCR. Aspirin and statins altered gene expression of ABCG1 and ADBR2. All finding were validated in a second group of twenty four patients and controls. Differential expression of ABCA1, RSG1 and ADBR2 was replicated. In conclusion, we identified these 3 genes to be expressed differently in CAD cases which might play a role in the pathogenesis of atherosclerotic vascular disease.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">e32166</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22363809" rel="external">22363809</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3283726" rel="external">3283726</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0032166" rel="external">10.1371/journal.pone.0032166</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of Shiga toxin encoding bacteriophages.</h4>
<p class="authors"><span class="author">Smith DL</span>, <span class="author">Rooks DJ</span>, <span class="author">Fogg PC</span>, <span class="author">Darby AC</span>, <span class="author">Thomson NR</span>, <span class="author">McCarthy AJ</span> and <span class="author">Allison HE</span></p>
<p class="affiliation">Microbiology Research Group, Institute of Integrative Biology, University of Liverpool, Crown Street, Liverpool, L69 7ZB, UK. <a href="mailto:%68%61%6C%6C%69%73%6F%6E%40%6C%69%76.%61%63.%75%6B">&#104;&#97;&#108;&#108;&#105;&#115;&#111;&#110;&#64;&#108;&#105;&#118;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Unlabelled: ABSTRACT:

Background: Stx bacteriophages are responsible for driving the dissemination of Stx toxin genes (stx) across their bacterial host range. Lysogens carrying Stx phages can cause severe, life-threatening disease and Stx toxin is an integral virulence factor. The Stx-bacteriophage vB_EcoP-24B, commonly referred to as Ð¤24B, is capable of multiply infecting a single bacterial host cell at a high frequency, with secondary infection increasing the rate at which subsequent bacteriophage infections can occur. This is biologically unusual, therefore determining the genomic content and context of Ð¤24B compared to other lambdoid Stx phages is important to understanding the factors controlling this phenomenon and determining whether they occur in other Stx phages.

Results: The genome of the Stx2 encoding phage, Ð¤24B was sequenced and annotated. The genomic organisation and general features are similar to other sequenced Stx bacteriophages induced from Enterohaemorrhagic Escherichia coli (EHEC), however Ð¤24B possesses significant regions of heterogeneity, with implications for phage biology and behaviour. The Ð¤24B genome was compared to other sequenced Stx phages and the archetypal lambdoid phage, lambda, using the Circos genome comparison tool and a PCR-based multi-loci comparison system.

Conclusions: The data support the hypothesis that Stx phages are mosaic, and recombination events between the host, phages and their remnants within the same infected bacterial cell will continue to drive the evolution of Stx phage variants and the subsequent dissemination of shigatoxigenic potential.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">311</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22799768" rel="external">22799768</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3430580" rel="external">3430580</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-311" rel="external">10.1186/1471-2164-13-311</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An In-Solution Hybridisation Method for the Isolation of Pathogen DNA from Human DNA-rich Clinical Samples for Analysis by NGS.</h4>
<p class="authors"><span class="author">Smith M</span>, <span class="author">Campino S</span>, <span class="author">Gu Y</span>, <span class="author">Clark TG</span>, <span class="author">Otto TD</span>, <span class="author">Maslen G</span>, <span class="author">Manske M</span>, <span class="author">Imwong M</span>, <span class="author">Dondorp AM</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Quail MA</span> and <span class="author">Swerdlow H</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK.</p><p class="abstract">Studies on DNA from pathogenic organisms, within clinical samples, are often complicated by the presence of large amounts of host, e.g., human DNA. Isolation of pathogen DNA from these samples would improve the efficiency of next-generation sequencing (NGS) and pathogen identification. Here we describe a solution-based hybridisation method for isolation of pathogen DNA from a mixed population. This straightforward and inexpensive technique uses probes made from whole-genome DNA and off-the-shelf reagents. In this study, Escherichia coli DNA was successfully enriched from a mixture of E.coli and human DNA. After enrichment, genome coverage following NGS was significantly higher and the evenness of coverage and GC content were unaffected. This technique was also applied to samples containing a mixture of human and Plasmodium falciparum DNA. The P.falciparum genome is particularly difficult to sequence due to its high AT content (80.6%) and repetitive nature. Post enrichment, a bias in the recovered DNA was observed, with a poorer representation of the AT-rich non-coding regions. This uneven coverage was also observed in pre-enrichment samples, but to a lesser degree. Despite the coverage bias in enriched samples, SNP (single-nucleotide polymorphism) calling in coding regions was unaffected and the majority of samples had over 90% of their coding region covered at 5&times; depth. This technique shows significant promise as an effective method to enrich pathogen DNA from samples with heavy human contamination, particularly when applied to GC-neutral genomes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">082370</span></p>
<p class="publication"><span class="title">The open genomics journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/24273626">24273626</a>; DOI: <a href="http://dx.doi.org/10.2174%2F1875693X01205010018">10.2174/1875693X01205010018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An in-solution hybridisation method for the isolation of pathogen DNA from human DNA-rich clinical samples for analysis by NGS</h4>
<p class="authors"><span class="author">SMITH,M.</span>, <span class="author">CAMPINO,S.</span>, <span class="author">GU,Y.</span>, <span class="author">Clark,T.G.</span>, <span class="author">OTTO,T.D.</span>, <span class="author">MASLEN,G.</span>, <span class="author">MANSKE,M.</span>, <span class="author">Imwong,M.</span>, <span class="author">Dondorp,A.M.</span>, <span class="author">KWIATKOWSKI,D.P.</span>, <span class="author">QUAIL,M.A.</span> and <span class="author">SWERDLOW,H.</span></p>
<p class="publication"><span class="title">Open Genomics Journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">18-29</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.2174%2F1875693X01205010018" rel="external">10.2174/1875693X01205010018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Predisposition gene identification in common cancers by exome sequencing: insights from familial breast cancer.</h4>
<p class="authors"><span class="author">Snape K</span>, <span class="author">Ruark E</span>, <span class="author">Tarpey P</span>, <span class="author">Renwick A</span>, <span class="author">Turnbull C</span>, <span class="author">Seal S</span>, <span class="author">Murray A</span>, <span class="author">Hanks S</span>, <span class="author">Douglas J</span>, <span class="author">Stratton MR</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Division of Genetics and Epidemiology, Institute of Cancer Research, 15 Cotswold Road, Sutton, Surrey, SM2 5NG, UK.</p><p class="abstract">The genetic component of breast cancer predisposition remains largely unexplained. Candidate gene case-control resequencing has identified predisposition genes characterised by rare, protein truncating mutations that confer moderate risks of disease. In theory, exome sequencing should yield additional genes of this class. Here, we explore the feasibility and design considerations of this approach. We performed exome sequencing in 50 individuals with familial breast cancer, applying frequency and protein function filters to identify variants most likely to be pathogenic. We identified 867,378 variants that passed the call quality filters of which 1,296 variants passed the frequency and protein truncation filters. The median number of validated, rare, protein truncating variants was 10 in individuals with, and without, mutations in known genes. The functional candidacy of mutated genes was similar in both groups. Without prior knowledge, the known genes would not have been recognisable as breast cancer predisposition genes. Everyone carries multiple rare mutations that are plausibly related to disease. Exome sequencing in common conditions will therefore require intelligent sample and variant prioritisation strategies in large case-control studies to deliver robust genetic evidence of disease association.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C8620/A8372</span>, <span class="grant">C8620/A8857</span></p>
<p class="publication"><span class="title">Breast cancer research and treatment</span>&nbsp;<span class="year">2012</span>;<span class="volume">134</span>;<span class="number">1</span>;<span class="pagerange">429-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22527104">22527104</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10549-012-2057-x">10.1007/s10549-012-2057-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phenotype-specific effect of chromosome 1q21.1 rearrangements and GJA5 duplications in 2436 congenital heart disease patients and 6760 controls.</h4>
<p class="authors"><span class="author">Soemedi R</span>, <span class="author">Topf A</span>, <span class="author">Wilson IJ</span>, <span class="author">Darlay R</span>, <span class="author">Rahman T</span>, <span class="author">Glen E</span>, <span class="author">Hall D</span>, <span class="author">Huang N</span>, <span class="author">Bentham J</span>, <span class="author">Bhattacharya S</span>, <span class="author">Cosgrove C</span>, <span class="author">Brook JD</span>, <span class="author">Granados-Riveron J</span>, <span class="author">Setchfield K</span>, <span class="author">Bu&#39;lock F</span>, <span class="author">Thornborough C</span>, <span class="author">Devriendt K</span>, <span class="author">Breckpot J</span>, <span class="author">Hofbeck M</span>, <span class="author">Lathrop M</span>, <span class="author">Rauch A</span>, <span class="author">Blue GM</span>, <span class="author">Winlaw DS</span>, <span class="author">Hurles M</span>, <span class="author">Santibanez-Koref M</span>, <span class="author">Cordell HJ</span>, <span class="author">Goodship JA</span> and <span class="author">Keavney BD</span></p>
<p class="affiliation">Institute of Genetic Medicine, Newcastle University, Newcastle, UK.</p><p class="abstract">Recurrent rearrangements of chromosome 1q21.1 that occur via non-allelic homologous recombination have been associated with variable phenotypes exhibiting incomplete penetrance, including congenital heart disease (CHD). However, the gene or genes within the ~1 Mb critical region responsible for each of the associated phenotypes remains unknown. We examined the 1q21.1 locus in 948 patients with tetralogy of Fallot (TOF), 1488 patients with other forms of CHD and 6760 ethnically matched controls using single nucleotide polymorphism genotyping arrays (Illumina 660W and Affymetrix 6.0) and multiplex ligation-dependent probe amplification. We found that duplication of 1q21.1 was more common in cases of TOF than in controls [odds ratio (OR) 30.9, 95% confidence interval (CI) 8.9-107.6); P = 2.2 &times; 10(-7)], but deletion was not. In contrast, deletion of 1q21.1 was more common in cases of non-TOF CHD than in controls [OR 5.5 (95% CI 1.4-22.0); P = 0.04] while duplication was not. We also detected rare (n = 3) 100-200 kb duplications within the critical region of 1q21.1 in cases of TOF. These small duplications encompassed a single gene in common, GJA5, and were enriched in cases of TOF in comparison to controls [OR = 10.7 (95% CI 1.8-64.3), P = 0.01]. These findings show that duplication and deletion at chromosome 1q21.1 exhibit a degree of phenotypic specificity in CHD, and implicate GJA5 as the gene responsible for the CHD phenotypes observed with copy number imbalances at this locus.</p>
<p class="grants">Funded by: British Heart Foundation</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">7</span>;<span class="pagerange">1513-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22199024" rel="external">22199024</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3298277" rel="external">3298277</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr589" rel="external">10.1093/hmg/ddr589</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bayesian inference analyses of the polygenic architecture of rheumatoid arthritis.</h4>
<p class="authors"><span class="author">Stahl EA</span>, <span class="author">Wegmann D</span>, <span class="author">Trynka G</span>, <span class="author">Gutierrez-Achury J</span>, <span class="author">Do R</span>, <span class="author">Voight BF</span>, <span class="author">Kraft P</span>, <span class="author">Chen R</span>, <span class="author">Kallberg HJ</span>, <span class="author">Kurreeman FA</span>, <span class="author">Diabetes Genetics Replication and Meta-analysis Consortium</span>, <span class="author">Myocardial Infarction Genetics Consortium</span>, <span class="author">Kathiresan S</span>, <span class="author">Wijmenga C</span>, <span class="author">Gregersen PK</span>, <span class="author">Alfredsson L</span>, <span class="author">Siminovitch KA</span>, <span class="author">Worthington J</span>, <span class="author">de Bakker PI</span>, <span class="author">Raychaudhuri S</span> and <span class="author">Plenge RM</span></p>
<p class="affiliation">Division of Rheumatology Immunology and Allergy, Brigham and Women&#39;s Hospital, Boston, Massachusetts, USA. <a href="mailto:%65%73%74%61%68%6C%40%72%69%63%73.%62%77%68.%68%61%72%76%61%72%64.%65%64%75">&#101;&#115;&#116;&#97;&#104;&#108;&#64;&#114;&#105;&#99;&#115;&#46;&#98;&#119;&#104;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The genetic architectures of common, complex diseases are largely uncharacterized. We modeled the genetic architecture underlying genome-wide association study (GWAS) data for rheumatoid arthritis and developed a new method using polygenic risk-score analyses to infer the total liability-scale variance explained by associated GWAS SNPs. Using this method, we estimated that, together, thousands of SNPs from rheumatoid arthritis GWAS explain an additional 20% of disease risk (excluding known associated loci). We further tested this method on datasets for three additional diseases and obtained comparable estimates for celiac disease (43% excluding the major histocompatibility complex), myocardial infarction and coronary artery disease (48%) and type 2 diabetes (49%). Our results are consistent with simulated genetic models in which hundreds of associated loci harbor common causal variants and a smaller number of loci harbor multiple rare causal variants. These analyses suggest that GWAS will continue to be highly productive for the discovery of additional susceptibility loci for common diseases.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">IIN-84042</span>, <span class="grant">MOP79321</span>; Medical Research Council: <span class="grant">MC_U106179471</span>; NIAMS NIH HHS: <span class="grant">K08AR055688-01A1</span>, <span class="grant">N01-AR-2-2263</span>, <span class="grant">R01 AR056768</span>, <span class="grant">R01-AR056768</span>, <span class="grant">R01-AR057108</span>, <span class="grant">R01-AR059648</span>, <span class="grant">R01-AR44422</span>; NIGMS NIH HHS: <span class="grant">U01 GM092691</span>, <span class="grant">U01-GM092691</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">483-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22446960">22446960</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3454878">3454878</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2232">10.1038/ng.2232</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gut inflammation can boost horizontal gene transfer between pathogenic and commensal Enterobacteriaceae.</h4>
<p class="authors"><span class="author">Stecher B</span>, <span class="author">Denzler R</span>, <span class="author">Maier L</span>, <span class="author">Bernet F</span>, <span class="author">Sanders MJ</span>, <span class="author">Pickard DJ</span>, <span class="author">Barthel M</span>, <span class="author">Westendorf AM</span>, <span class="author">Krogfelt KA</span>, <span class="author">Walker AW</span>, <span class="author">Ackermann M</span>, <span class="author">Dobrindt U</span>, <span class="author">Thomson NR</span> and <span class="author">Hardt WD</span></p>
<p class="affiliation">Institute of Microbiology, ETH Z&uuml;rich, 8093 Z&uuml;rich, Switzerland.</p><p class="abstract">The mammalian gut harbors a dense microbial community interacting in multiple ways, including horizontal gene transfer (HGT). Pangenome analyses established particularly high levels of genetic flux between Gram-negative Enterobacteriaceae. However, the mechanisms fostering intraenterobacterial HGT are incompletely understood. Using a mouse colitis model, we found that Salmonella-inflicted enteropathy elicits parallel blooms of the pathogen and of resident commensal Escherichia coli. These blooms boosted conjugative HGT of the colicin-plasmid p2 from Salmonella enterica serovar Typhimurium to E. coli. Transconjugation efficiencies of ~100% in vivo were attributable to high intrinsic p2-transfer rates. Plasmid-encoded fitness benefits contributed little. Under normal conditions, HGT was blocked by the commensal microbiota inhibiting contact-dependent conjugation between Enterobacteriaceae. Our data show that pathogen-driven inflammatory responses in the gut can generate transient enterobacterial blooms in which conjugative transfer occurs at unprecedented rates. These blooms may favor reassortment of plasmid-encoded genes between pathogens and commensals fostering the spread of fitness-, virulence-, and antibiotic-resistance determinants.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2012</span>;<span class="volume">109</span>;<span class="number">4</span>;<span class="pagerange">1269-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22232693" rel="external">22232693</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3268327" rel="external">3268327</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1113246109" rel="external">10.1073/pnas.1113246109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutanlallemand (mtl) and Belly Spot and Deafness (bsd) Are Two New Mutations of Lmx1a Causing Severe Cochlear and Vestibular Defects.</h4>
<p class="authors"><span class="author">Steffes G</span>, <span class="author">Lorente-C&aacute;novas B</span>, <span class="author">Pearson S</span>, <span class="author">Brooker RH</span>, <span class="author">Spiden S</span>, <span class="author">Kiernan AE</span>, <span class="author">Gu&eacute;net JL</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p>
<p class="abstract">Mutanlallemand (mtl) and Belly Spot and Deafness (bsd) are two new spontaneous alleles of the Lmx1a gene in mice. Homozygous mutants show head tossing and circling behaviour, indicative of vestibular defects, and they have short tails and white belly patches of variable size. The analysis of auditory brainstem responses (ABR) showed that mtl and bsd homozygotes are deaf, whereas heterozygous and wildtype littermates have normal hearing. Paint-filled inner ears at E16.5 revealed that mtl and bsd homozygotes lack endolymphatic ducts and semicircular canals and have short cochlear ducts. These new alleles show similarities with dreher (Lmx1a) mutants. Complementation tests between mtl and dreher and between mtl and bsd suggest that mtl and bsd are new mutant alleles of the Lmx1a gene. To determine the Lmx1a mutation in mtl and bsd mutant mice we performed PCR followed by sequencing of genomic DNA and cDNA. The mtl mutation is a single point mutation in the 3' splice site of exon 4 leading to an exon extension and the activation of a cryptic splice site 44 base pairs downstream, whereas the bsd mutation is a genomic deletion that includes exon 3. Both mutations lead to a truncated LMX1A protein affecting the homeodomain (mtl) or LIM2-domain (bsd), which is critical for LMX1A protein function. Moreover, the levels of Lmx1a transcript in mtl and bsd mutants are significantly down-regulated. Hmx2/3 and Pax2 expression are also down-regulated in mtl and bsd mutants, suggesting a role of Lmx1a upstream of these transcription factors in early inner ear morphogenesis. We have found that these mutants develop sensory patches although they are misshapen. The characterization of these two new Lmx1a alleles highlights the critical role of this gene in the development of the cochlea and vestibular system.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">11</span>;<span class="pagerange">e51065</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23226461" rel="external">23226461</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3511360" rel="external">3511360</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0051065" rel="external">10.1371/journal.pone.0051065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using probabilistic estimation of expression residuals (PEER) to obtain increased power and interpretability of gene expression analyses.</h4>
<p class="authors"><span class="author">Stegle O</span>, <span class="author">Parts L</span>, <span class="author">Piipari M</span>, <span class="author">Winn J</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Max Planck Institute for Intelligent Systems, T&uuml;bingen, Germany. <a href="mailto:%6F%6C%69%76%65%72.%73%74%65%67%6C%65%40%74%75%65%62%69%6E%67%65%6E.%6D%70%67.%64%65">&#111;&#108;&#105;&#118;&#101;&#114;&#46;&#115;&#116;&#101;&#103;&#108;&#101;&#64;&#116;&#117;&#101;&#98;&#105;&#110;&#103;&#101;&#110;&#46;&#109;&#112;&#103;&#46;&#100;&#101;</a></p><p class="abstract">We present PEER (probabilistic estimation of expression residuals), a software package implementing statistical models that improve the sensitivity and interpretability of genetic associations in population-scale expression data. This approach builds on factor analysis methods that infer broad variance components in the measurements. PEER takes as input transcript profiles and covariates from a set of individuals, and then outputs hidden factors that explain much of the expression variability. Optionally, these factors can be interpreted as pathway or transcription factor activations by providing prior information about which genes are involved in the pathway or targeted by the factor. The inferred factors are used in genetic association analyses. First, they are treated as additional covariates, and are included in the model to increase detection power for mapping expression traits. Second, they are analyzed as phenotypes themselves to understand the causes of global expression variability. PEER extends previous related surrogate variable models and can be implemented within hours on a desktop computer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077192/Z/05/Z</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">500-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22343431" rel="external">22343431</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2011.457" rel="external">10.1038/nprot.2011.457</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The landscape of cancer genes and mutational processes in breast cancer.</h4>
<p class="authors"><span class="author">Stephens PJ</span>, <span class="author">Tarpey PS</span>, <span class="author">Davies H</span>, <span class="author">Van Loo P</span>, <span class="author">Greenman C</span>, <span class="author">Wedge DC</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Martin S</span>, <span class="author">Varela I</span>, <span class="author">Bignell GR</span>, <span class="author">Yates LR</span>, <span class="author">Papaemmanuil E</span>, <span class="author">Beare D</span>, <span class="author">Butler A</span>, <span class="author">Cheverton A</span>, <span class="author">Gamble J</span>, <span class="author">Hinton J</span>, <span class="author">Jia M</span>, <span class="author">Jayakumar A</span>, <span class="author">Jones D</span>, <span class="author">Latimer C</span>, <span class="author">Lau KW</span>, <span class="author">McLaren S</span>, <span class="author">McBride DJ</span>, <span class="author">Menzies A</span>, <span class="author">Mudie L</span>, <span class="author">Raine K</span>, <span class="author">Rad R</span>, <span class="author">Chapman MS</span>, <span class="author">Teague J</span>, <span class="author">Easton D</span>, <span class="author">Langer&oslash;d A</span>, <span class="author">Oslo Breast Cancer Consortium (OSBREAC)</span>, <span class="author">Lee MT</span>, <span class="author">Shen CY</span>, <span class="author">Tee BT</span>, <span class="author">Huimin BW</span>, <span class="author">Broeks A</span>, <span class="author">Vargas AC</span>, <span class="author">Turashvili G</span>, <span class="author">Martens J</span>, <span class="author">Fatima A</span>, <span class="author">Miron P</span>, <span class="author">Chin SF</span>, <span class="author">Thomas G</span>, <span class="author">Boyault S</span>, <span class="author">Mariani O</span>, <span class="author">Lakhani SR</span>, <span class="author">van de Vijver M</span>, <span class="author">van &#39;t Veer L</span>, <span class="author">Foekens J</span>, <span class="author">Desmedt C</span>, <span class="author">Sotiriou C</span>, <span class="author">Tutt A</span>, <span class="author">Caldas C</span>, <span class="author">Reis-Filho JS</span>, <span class="author">Aparicio SA</span>, <span class="author">Salomon AV</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">Richardson AL</span>, <span class="author">Campbell PJ</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">All cancers carry somatic mutations in their genomes. A subset, known as driver mutations, confer clonal selective advantage on cancer cells and are causally implicated in oncogenesis, and the remainder are passenger mutations. The driver mutations and mutational processes operative in breast cancer have not yet been comprehensively explored. Here we examine the genomes of 100 tumours for somatic copy number changes and mutations in the coding exons of protein-coding genes. The number of somatic mutations varied markedly between individual tumours. We found strong correlations between mutation number, age at which cancer was diagnosed and cancer histological grade, and observed multiple mutational signatures, including one present in about ten per cent of tumours characterized by numerous mutations of cytosine at TpC dinucleotides. Driver mutations were identified in several new cancer genes including AKT2, ARID1B, CASP8, CDKN1B, MAP3K1, MAP3K13, NCOR1, SMARCD1 and TBX3. Among the 100 tumours, we found driver mutations in at least 40 cancer genes and 73 different combinations of mutated cancer genes. The results highlight the substantial genetic diversity underlying this common disease.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11022</span>; Chief Scientist Office; Department of Health; NCI NIH HHS: <span class="grant">CA089393</span>, <span class="grant">P30 CA016672</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">486</span>;<span class="number">7403</span>;<span class="pagerange">400-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22722201">22722201</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428862">3428862</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11017">10.1038/nature11017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analyses identify 13 loci associated with age at menopause and highlight DNA repair and immune pathways.</h4>
<p class="authors"><span class="author">Stolk L</span>, <span class="author">Perry JR</span>, <span class="author">Chasman DI</span>, <span class="author">He C</span>, <span class="author">Mangino M</span>, <span class="author">Sulem P</span>, <span class="author">Barbalic M</span>, <span class="author">Broer L</span>, <span class="author">Byrne EM</span>, <span class="author">Ernst F</span>, <span class="author">Esko T</span>, <span class="author">Franceschini N</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Hottenga JJ</span>, <span class="author">Kraft P</span>, <span class="author">McArdle PF</span>, <span class="author">Porcu E</span>, <span class="author">Shin SY</span>, <span class="author">Smith AV</span>, <span class="author">van Wingerden S</span>, <span class="author">Zhai G</span>, <span class="author">Zhuang WV</span>, <span class="author">Albrecht E</span>, <span class="author">Alizadeh BZ</span>, <span class="author">Aspelund T</span>, <span class="author">Bandinelli S</span>, <span class="author">Lauc LB</span>, <span class="author">Beckmann JS</span>, <span class="author">Boban M</span>, <span class="author">Boerwinkle E</span>, <span class="author">Broekmans FJ</span>, <span class="author">Burri A</span>, <span class="author">Campbell H</span>, <span class="author">Chanock SJ</span>, <span class="author">Chen C</span>, <span class="author">Cornelis MC</span>, <span class="author">Corre T</span>, <span class="author">Coviello AD</span>, <span class="author">d&#39;Adamo P</span>, <span class="author">Davies G</span>, <span class="author">de Faire U</span>, <span class="author">de Geus EJ</span>, <span class="author">Deary IJ</span>, <span class="author">Dedoussis GV</span>, <span class="author">Deloukas P</span>, <span class="author">Ebrahim S</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Emilsson V</span>, <span class="author">Eriksson JG</span>, <span class="author">Fauser BC</span>, <span class="author">Ferreli L</span>, <span class="author">Ferrucci L</span>, <span class="author">Fischer K</span>, <span class="author">Folsom AR</span>, <span class="author">Garcia ME</span>, <span class="author">Gasparini P</span>, <span class="author">Gieger C</span>, <span class="author">Glazer N</span>, <span class="author">Grobbee DE</span>, <span class="author">Hall P</span>, <span class="author">Haller T</span>, <span class="author">Hankinson SE</span>, <span class="author">Hass M</span>, <span class="author">Hayward C</span>, <span class="author">Heath AC</span>, <span class="author">Hofman A</span>, <span class="author">Ingelsson E</span>, <span class="author">Janssens AC</span>, <span class="author">Johnson AD</span>, <span class="author">Karasik D</span>, <span class="author">Kardia SL</span>, <span class="author">Keyzer J</span>, <span class="author">Kiel DP</span>, <span class="author">Kolcic I</span>, <span class="author">Kutalik Z</span>, <span class="author">Lahti J</span>, <span class="author">Lai S</span>, <span class="author">Laisk T</span>, <span class="author">Laven JS</span>, <span class="author">Lawlor DA</span>, <span class="author">Liu J</span>, <span class="author">Lopez LM</span>, <span class="author">Louwers YV</span>, <span class="author">Magnusson PK</span>, <span class="author">Marongiu M</span>, <span class="author">Martin NG</span>, <span class="author">Klaric IM</span>, <span class="author">Masciullo C</span>, <span class="author">McKnight B</span>, <span class="author">Medland SE</span>, <span class="author">Melzer D</span>, <span class="author">Mooser V</span>, <span class="author">Navarro P</span>, <span class="author">Newman AB</span>, <span class="author">Nyholt DR</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Palotie A</span>, <span class="author">Par&eacute; G</span>, <span class="author">Parker AN</span>, <span class="author">Pedersen NL</span>, <span class="author">Peeters PH</span>, <span class="author">Pistis G</span>, <span class="author">Plump AS</span>, <span class="author">Polasek O</span>, <span class="author">Pop VJ</span>, <span class="author">Psaty BM</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Rehnberg E</span>, <span class="author">Rotter JI</span>, <span class="author">Rudan I</span>, <span class="author">Sala C</span>, <span class="author">Salumets A</span>, <span class="author">Scuteri A</span>, <span class="author">Singleton A</span>, <span class="author">Smith JA</span>, <span class="author">Snieder H</span>, <span class="author">Soranzo N</span>, <span class="author">Stacey SN</span>, <span class="author">Starr JM</span>, <span class="author">Stathopoulou MG</span>, <span class="author">Stirrups K</span>, <span class="author">Stolk RP</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Sun YV</span>, <span class="author">Tenesa A</span>, <span class="author">Thorand B</span>, <span class="author">Toniolo D</span>, <span class="author">Tryggvadottir L</span>, <span class="author">Tsui K</span>, <span class="author">Ulivi S</span>, <span class="author">van Dam RM</span>, <span class="author">van der Schouw YT</span>, <span class="author">van Gils CH</span>, <span class="author">van Nierop P</span>, <span class="author">Vink JM</span>, <span class="author">Visscher PM</span>, <span class="author">Voorhuis M</span>, <span class="author">Waeber G</span>, <span class="author">Wallaschofski H</span>, <span class="author">Wichmann HE</span>, <span class="author">Widen E</span>, <span class="author">Wijnands-van Gent CJ</span>, <span class="author">Willemsen G</span>, <span class="author">Wilson JF</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Wright AF</span>, <span class="author">Yerges-Armstrong LM</span>, <span class="author">Zemunik T</span>, <span class="author">Zgaga L</span>, <span class="author">Zillikens MC</span>, <span class="author">Zygmunt M</span>, <span class="author">Study TL</span>, <span class="author">Arnold AM</span>, <span class="author">Boomsma DI</span>, <span class="author">Buring JE</span>, <span class="author">Crisponi L</span>, <span class="author">Demerath EW</span>, <span class="author">Gudnason V</span>, <span class="author">Harris TB</span>, <span class="author">Hu FB</span>, <span class="author">Hunter DJ</span>, <span class="author">Launer LJ</span>, <span class="author">Metspalu A</span>, <span class="author">Montgomery GW</span>, <span class="author">Oostra BA</span>, <span class="author">Ridker PM</span>, <span class="author">Sanna S</span>, <span class="author">Schlessinger D</span>, <span class="author">Spector TD</span>, <span class="author">Stefansson K</span>, <span class="author">Streeten EA</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Uda M</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Murray A</span>, <span class="author">Murabito JM</span>, <span class="author">Visser JA</span> and <span class="author">Lunetta KL</span></p>
<p class="affiliation">Department of Internal Medicine, Erasmus MC, Rotterdam, The Netherlands.</p><p class="abstract">To newly identify loci for age at natural menopause, we carried out a meta-analysis of 22 genome-wide association studies (GWAS) in 38,968 women of European descent, with replication in up to 14,435 women. In addition to four known loci, we identified 13 loci newly associated with age at natural menopause (at P &lt; 5 &times; 10(-8)). Candidate genes located at these newly associated loci include genes implicated in DNA repair (EXO1, HELQ, UIMC1, FAM175A, FANCI, TLK1, POLG and PRIM1) and immune function (IL11, NLRP11 and PRRC2A (also known as BAT2)). Gene-set enrichment pathway analyses using the full GWAS data set identified exoDNase, NF-&kappa;B signaling and mitochondrial dysfunction as biological processes related to timing of menopause.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">260-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22267201" rel="external">22267201</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3288642" rel="external">3288642</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.1051" rel="external">10.1038/ng.1051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNAseq Analysis of the Parasitic Nematode Strongyloides stercoralis Reveals Divergent Regulation of Canonical Dauer Pathways.</h4>
<p class="authors"><span class="author">Stoltzfus JD</span>, <span class="author">Minot S</span>, <span class="author">Berriman M</span>, <span class="author">Nolan TJ</span> and <span class="author">Lok JB</span></p>
<p class="affiliation">Department of Pathobiology, University of Pennsylvania School of Veterinary Medicine, Philadelphia, Pennsylvania, United States of America.</p>
<p class="abstract">The infectious form of many parasitic nematodes, which afflict over one billion people globally, is a developmentally arrested third-stage larva (L3i). The parasitic nematode Strongyloides stercoralis differs from other nematode species that infect humans, in that its life cycle includes both parasitic and free-living forms, which can be leveraged to investigate the mechanisms of L3i arrest and activation. The free-living nematode Caenorhabditis elegans has a similar developmentally arrested larval form, the dauer, whose formation is controlled by four pathways: cyclic GMP (cGMP) signaling, insulin/IGF-1-like signaling (IIS), transforming growth factor Î² (TGFÎ²) signaling, and biosynthesis of dafachronic acid (DA) ligands that regulate a nuclear hormone receptor. We hypothesized that homologous pathways are present in S. stercoralis, have similar developmental regulation, and are involved in L3i arrest and activation. To test this, we undertook a deep-sequencing study of the polyadenylated transcriptome, generating over 2.3 billion paired-end reads from seven developmental stages. We constructed developmental expression profiles for S. stercoralis homologs of C. elegans dauer genes identified by BLAST searches of the S. stercoralis genome as well as de novo assembled transcripts. Intriguingly, genes encoding cGMP pathway components were coordinately up-regulated in L3i. In comparison to C. elegans, S. stercoralis has a paucity of genes encoding IIS ligands, several of which have abundance profiles suggesting involvement in L3i development. We also identified seven S. stercoralis genes encoding homologs of the single C. elegans dauer regulatory TGFÎ² ligand, three of which are only expressed in L3i. Putative DA biosynthetic genes did not appear to be coordinately regulated in L3i development. Our data suggest that while dauer pathway genes are present in S. stercoralis and may play a role in L3i development, there are significant differences between the two species. Understanding the mechanisms governing L3i development may lead to novel treatment and control strategies.</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">e1854</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23145190" rel="external">23145190</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3493385" rel="external">3493385</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0001854" rel="external">10.1371/journal.pntd.0001854</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Patterns of cis regulatory variation in diverse human populations.</h4>
<p class="authors"><span class="author">Stranger BE</span>, <span class="author">Montgomery SB</span>, <span class="author">Dimas AS</span>, <span class="author">Parts L</span>, <span class="author">Stegle O</span>, <span class="author">Ingle CE</span>, <span class="author">Sekowska M</span>, <span class="author">Smith GD</span>, <span class="author">Evans D</span>, <span class="author">Gutierrez-Arcelus M</span>, <span class="author">Price A</span>, <span class="author">Raj T</span>, <span class="author">Nisbett J</span>, <span class="author">Nica AC</span>, <span class="author">Beazley C</span>, <span class="author">Durbin R</span>, <span class="author">Deloukas P</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK.</p><p class="abstract">The genetic basis of gene expression variation has long been studied with the aim to understand the landscape of regulatory variants, but also more recently to assist in the interpretation and elucidation of disease signals. To date, many studies have looked in specific tissues and population-based samples, but there has been limited assessment of the degree of inter-population variability in regulatory variation. We analyzed genome-wide gene expression in lymphoblastoid cell lines from a total of 726 individuals from 8 global populations from the HapMap3 project and correlated gene expression levels with HapMap3 SNPs located in cis to the genes. We describe the influence of ancestry on gene expression levels within and between these diverse human populations and uncover a non-negligible impact on global patterns of gene expression. We further dissect the specific functional pathways differentiated between populations. We also identify 5,691 expression quantitative trait loci (eQTLs) after controlling for both non-genetic factors and population admixture and observe that half of the cis-eQTLs are replicated in one or more of the populations. We highlight patterns of eQTL-sharing between populations, which are partially determined by population genetic relatedness, and discover significant sharing of eQTL effects between Asians, European-admixed, and African subpopulations. Specifically, we observe that both the effect size and the direction of effect for eQTLs are highly conserved across populations. We observe an increasing proximity of eQTLs toward the transcription start site as sharing of eQTLs among populations increases, highlighting that variants close to TSS have stronger effects and therefore are more likely to be detected across a wider panel of populations. Together these results offer a unique picture and resource of the degree of differentiation among human populations in functional regulatory variation and provide an estimate for the transferability of complex trait variants across populations.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">e1002639</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22532805">22532805</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3330104">3330104</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1002639">10.1371/journal.pgen.1002639</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A retinoblastoma orthologue is a major regulator of s-phase, mitotic, and developmental gene expression in dictyostelium.</h4>
<p class="authors"><span class="author">Strasser K</span>, <span class="author">Bloomfield G</span>, <span class="author">Macwilliams A</span>, <span class="author">Ceccarelli A</span>, <span class="author">Macwilliams H</span> and <span class="author">Tsang A</span></p>
<p class="affiliation">Biology Department and Centre for Structural and Functional Genomics, Concordia University, Montreal, Canada.</p>
<p class="abstract">Background: The retinoblastoma tumour suppressor, Rb, has two major functions. First, it represses genes whose products are required for S-phase entry and progression thus stabilizing cells in G1. Second, Rb interacts with factors that induce cell-cycle exit and terminal differentiation. Dictyostelium lacks a G1 phase in its cell cycle but it has a retinoblastoma orthologue, rblA.

Using microarray analysis and mRNA-Seq transcriptional profiling, we show that RblA strongly represses genes whose products are involved in S phase and mitosis. Both S-phase and mitotic genes are upregulated at a single point in late G2 and again in mid-development, near the time when cell cycling is reactivated. RblA also activates a set of genes unique to slime moulds that function in terminal differentiation.

Conclusions: Like its mammalian counterpart Dictyostelium, RblA plays a dual role, regulating cell-cycle progression and transcriptional events leading to terminal differentiation. In the absence of a G1 phase, however, RblA functions in late G2 controlling the expression of both S-phase and mitotic genes.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e39914</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22768168" rel="external">22768168</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3386910" rel="external">3386910</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0039914" rel="external">10.1371/journal.pone.0039914</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lineage-specific virulence determinants of Haemophilus influenzae biogroup aegyptius.</h4>
<p class="authors"><span class="author">Strouts FR</span>, <span class="author">Power P</span>, <span class="author">Croucher NJ</span>, <span class="author">Corton N</span>, <span class="author">van Tonder A</span>, <span class="author">Quail MA</span>, <span class="author">Langford PR</span>, <span class="author">Hudson MJ</span>, <span class="author">Parkhill J</span>, <span class="author">Kroll JS</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Imperial College London, Medicine, St Mary&rsquo;s Hospital campus, Norfolk Place, London W2 1PG, UK.</p><p class="abstract">An emergent clone of Haemophilus influenzae biogroup aegyptius (Hae) is responsible for outbreaks of Brazilian purpuric fever (BPF). First recorded in Brazil in 1984, the so-called BPF clone of Hae caused a fulminant disease that started with conjunctivitis but developed into septicemic shock; mortality rates were as high as 70%. To identify virulence determinants, we conducted a pan-genomic analysis. Sequencing of the genomes of the BPF clone strain F3031 and a noninvasive conjunctivitis strain, F3047, and comparison of these sequences with 5 other complete H. influenzae genomes showed that &gt;77% of the F3031 genome is shared among all H. influenzae strains. Delineation of the Hae accessory genome enabled characterization of 163 predicted protein-coding genes; identified differences in established autotransporter adhesins; and revealed a suite of novel adhesins unique to Hae, including novel trimeric autotransporter adhesins and 4 new fimbrial operons. These novel adhesins might play a critical role in host-pathogen interactions.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">18</span>;<span class="number">3</span>;<span class="pagerange">449-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22377449">22377449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3309571">3309571</a>; DOI: <a href="http://dx.doi.org/10.3201%2Feid1803.110728">10.3201/eid1803.110728</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss-of-function mutations in IGSF1 cause an X-linked syndrome of central hypothyroidism and testicular enlargement.</h4>
<p class="authors"><span class="author">Sun Y</span>, <span class="author">Bak B</span>, <span class="author">Schoenmakers N</span>, <span class="author">van Trotsenburg AS</span>, <span class="author">Oostdijk W</span>, <span class="author">Voshol P</span>, <span class="author">Cambridge E</span>, <span class="author">White JK</span>, <span class="author">le Tissier P</span>, <span class="author">Gharavy SN</span>, <span class="author">Martinez-Barbera JP</span>, <span class="author">Stokvis-Brantsma WH</span>, <span class="author">Vulsma T</span>, <span class="author">Kempers MJ</span>, <span class="author">Persani L</span>, <span class="author">Campi I</span>, <span class="author">Bonomi M</span>, <span class="author">Beck-Peccoz P</span>, <span class="author">Zhu H</span>, <span class="author">Davis TM</span>, <span class="author">Hokken-Koelega AC</span>, <span class="author">Del Blanco DG</span>, <span class="author">Rangasami JJ</span>, <span class="author">Ruivenkamp CA</span>, <span class="author">Laros JF</span>, <span class="author">Kriek M</span>, <span class="author">Kant SG</span>, <span class="author">Bosch CA</span>, <span class="author">Biermasz NR</span>, <span class="author">Appelman-Dijkstra NM</span>, <span class="author">Corssmit EP</span>, <span class="author">Hovens GC</span>, <span class="author">Pereira AM</span>, <span class="author">Dunnen JT</span>, <span class="author">Wade MG</span>, <span class="author">Breuning MH</span>, <span class="author">Hennekam RC</span>, <span class="author">Chatterjee K</span>, <span class="author">Dattani MT</span>, <span class="author">Wit JM</span> and <span class="author">Bernard DJ</span></p>
<p class="affiliation">1] Center for Human and Clinical Genetics, Leiden University Medical Center, Leiden, The Netherlands. [2].</p>
<p class="abstract">Congenital central hypothyroidism occurs either in isolation or in conjunction with other pituitary hormone deficits. Using exome and candidate gene sequencing, we identified 8 distinct mutations and 2 deletions in IGSF1 in males from 11 unrelated families with central hypothyroidism, testicular enlargement and variably low prolactin concentrations. IGSF1 is a membrane glycoprotein that is highly expressed in the anterior pituitary gland, and the identified mutations impair its trafficking to the cell surface in heterologous cells. Igsf1-deficient male mice show diminished pituitary and serum thyroid-stimulating hormone (TSH) concentrations, reduced pituitary thyrotropin-releasing hormone (TRH) receptor expression, decreased triiodothyronine concentrations and increased body mass. Collectively, our observations delineate a new X-linked disorder in which loss-of-function mutations in IGSF1 cause central hypothyroidism, likely secondary to an associated impairment in pituitary TRH signaling.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23143598" rel="external">23143598</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2453" rel="external">10.1038/ng.2453</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A benchmarked protein microarray-based platform for the identification of novel low-affinity extracellular protein interactions.</h4>
<p class="authors"><span class="author">Sun Y</span>, <span class="author">Gallagher-Jones M</span>, <span class="author">Barker C</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Low-affinity extracellular protein interactions are critical for cellular recognition processes, but existing methods to detect them are limited in scale, making genome-wide interaction screens technically challenging. To address this, we report here the miniaturization of the AVEXIS (avidity-based extracellular interaction screen) assay by using protein microarray technology. To achieve this, we have developed protein tags and sample preparation methods that enable the parallel purification of hundreds of recombinant proteins expressed in mammalian cells. We benchmarked the protein microarray-based assay against a set of known quantified receptor-ligand pairs and show that it is sensitive enough to detect even very weak interactions that are typical of this class of interactions. The increase in scale enables interaction screening against a dilution series of immobilized proteins on the microarray enabling the observation of saturation binding behaviors to show interaction specificity and also the estimation of interaction affinities directly from the primary screen. These methodological improvements now permit screening for novel extracellular receptor-ligand interactions on a genome-wide scale.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077108</span></p>
<p class="publication"><span class="title">Analytical biochemistry</span>&nbsp;<span class="year">2012</span>;<span class="volume">424</span>;<span class="number">1</span>;<span class="pagerange">45-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22342946">22342946</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3325482">3325482</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ab.2012.01.034">10.1016/j.ab.2012.01.034</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study of monozygotic twin-pairs suggests a locus related to variability of serum high-density lipoprotein cholesterol.</h4>
<p class="authors"><span class="author">Surakka I</span>, <span class="author">Whitfield JB</span>, <span class="author">Perola M</span>, <span class="author">Visscher PM</span>, <span class="author">Montgomery GW</span>, <span class="author">Falchi M</span>, <span class="author">Willemsen G</span>, <span class="author">de Geus EJ</span>, <span class="author">Magnusson PK</span>, <span class="author">Christensen K</span>, <span class="author">S&oslash;rensen TI</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Rantanen T</span>, <span class="author">Silander K</span>, <span class="author">Wid&eacute;n E</span>, <span class="author">Muilu J</span>, <span class="author">Rahman I</span>, <span class="author">Liljedahl U</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Palotie A</span>, <span class="author">Kaprio J</span>, <span class="author">Kyvik KO</span>, <span class="author">Pedersen NL</span>, <span class="author">Boomsma DI</span>, <span class="author">Spector T</span>, <span class="author">Martin NG</span>, <span class="author">Ripatti S</span>, <span class="author">Peltonen L</span> and <span class="author">GenomEUtwin Project</span></p>
<p class="affiliation">Institute for Molecular Medicine, University of Helsinki, Finland.</p><p class="abstract">Genome-wide association analysis on monozygotic twin-pairs offers a route to discovery of gene environment interactions through testing for variability loci associated with sensitivity to individual environment/lifestyle. We present a genome-wide scan of loci associated with intra-pair differences in serum lipid and apolipoprotein levels. We report data for 1,720 monozygotic female twin-pairs from GenomEUtwin project with 2.5 million SNPs, imputed or genotyped, and measured serum lipid fractions for both twins. We found one locus associated with intra-pair differences in high-density lipoprotein cholesterol, rs2483058 in an intron of SRGAP2, where twins carrying the C allele are more sensitive to environmental factors(P=3.98 x 10-8). We followed up the association in further genotyped monozygotic twins (N= 1,261),which showed a moderate association for the variant (P= 0.200, same direction of an effect). In addition,we report a new association on the level of apolipoprotein A-ll (P= 4.03 x 1 o-8).</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">AG028555</span>, <span class="grant">AG04563</span>, <span class="grant">AG08724</span>, <span class="grant">AG08861</span>, <span class="grant">AG10175</span>; NIAAA NIH HHS: <span class="grant">AA07535</span>, <span class="grant">AA10248</span>, <span class="grant">AA11998</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">AA14041</span>, <span class="grant">AA17688</span>; NIDA NIH HHS: <span class="grant">DA12854</span>; NIMH NIH HHS: <span class="grant">MH66206</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Twin research and human genetics : the official journal of the International Society for Twin Studies</span>&nbsp;<span class="year">2012</span>;<span class="volume">15</span>;<span class="number">6</span>;<span class="pagerange">691-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23031429">23031429</a>; DOI: <a href="http://dx.doi.org/10.1017%2Fthg.2012.63">10.1017/thg.2012.63</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A post-assembly genome-improvement toolkit (PAGIT) to obtain annotated genomes from contigs.</h4>
<p class="authors"><span class="author">Swain MT</span>, <span class="author">Tsai IJ</span>, <span class="author">Assefa SA</span>, <span class="author">Newbold C</span>, <span class="author">Berriman M</span> and <span class="author">Otto TD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p><p class="abstract">Genome projects now produce draft assemblies within weeks owing to advanced high-throughput sequencing technologies. For milestone projects such as Escherichia coli or Homo sapiens, teams of scientists were employed to manually curate and finish these genomes to a high standard. Nowadays, this is not feasible for most projects, and the quality of genomes is generally of a much lower standard. This protocol describes software (PAGIT) that is used to improve the quality of draft genomes. It offers flexible functionality to close gaps in scaffolds, correct base errors in the consensus sequence and exploit reference genomes (if available) in order to improve scaffolding and generating annotations. The protocol is most accessible for bacterial and small eukaryotic genomes (up to 300 Mb), such as pathogenic bacteria, malaria and parasitic worms. Applying PAGIT to an E. coli assembly takes &sim;24 h: it doubles the average contig size and annotates over 4,300 gene models.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">1260-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22678431">22678431</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3648784">3648784</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2012.068">10.1038/nprot.2012.068</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DECIPHER: web-based, community resource for clinical interpretation of rare variants in developmental disorders.</h4>
<p class="authors"><span class="author">Swaminathan GJ</span>, <span class="author">Bragin E</span>, <span class="author">Chatzimichali EA</span>, <span class="author">Corpas M</span>, <span class="author">Bevan AP</span>, <span class="author">Wright CF</span>, <span class="author">Carter NP</span>, <span class="author">Hurles ME</span> and <span class="author">Firth HV</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">Patients with developmental disorders often harbour sub-microscopic deletions or duplications that lead to a disruption of normal gene expression or perturbation in the copy number of dosage-sensitive genes. Clinical interpretation for such patients in isolation is hindered by the rarity and novelty of such disorders. The DECIPHER project (https://decipher.sanger.ac.uk) was established in 2004 as an accessible online repository of genomic and associated phenotypic data with the primary goal of aiding the clinical interpretation of rare copy-number variants (CNVs). DECIPHER integrates information from a variety of bioinformatics resources and uses visualization tools to identify potential disease genes within a CNV. A two-tier access system permits clinicians and clinical scientists to maintain confidential linked anonymous records of phenotypes and CNVs for their patients that, with informed consent, can subsequently be shared with the wider clinical genetics and research communities. Advances in next-generation sequencing technologies are making it practical and affordable to sequence the whole exome/genome of patients who display features suggestive of a genetic disorder. This approach enables the identification of smaller intragenic mutations including single-nucleotide variants that are not accessible even with high-resolution genomic array analysis. This article briefly summarizes the current status and achievements of the DECIPHER project and looks ahead to the opportunities and challenges of jointly analysing structural and sequence variation in the human genome.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">R1</span>;<span class="pagerange">R37-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22962312">22962312</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3459644">3459644</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fdds362">10.1093/hmg/dds362</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid whole-genome sequencing of bacterial pathogens in the clinical microbiology laboratory--pipe dream or reality?</h4>
<p class="authors"><span class="author">T&ouml;r&ouml;k ME</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Box 157, Addenbrooke&#39;s Hospital, Hills Road, Cambridge CB2 0QQ, UK. <a href="mailto:%65%73%74%65%65.%74%6F%72%6F%6B%40%61%64%64%65%6E%62%72%6F%6F%6B%65%73.%6E%68%73.%75%6B">&#101;&#115;&#116;&#101;&#101;&#46;&#116;&#111;&#114;&#111;&#107;&#64;&#97;&#100;&#100;&#101;&#110;&#98;&#114;&#111;&#111;&#107;&#101;&#115;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract">The ability to perform rapid, high-throughput whole-genome sequencing using bench-top platforms represents a step-change in capabilities for diagnostic and public health microbiology laboratories. As the cost of sequencing continues to decline, the challenge will be to define when and where to apply this technology. This article reviews its potential applications in the clinical microbiology laboratory and discusses the current barriers to implementation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1000803</span></p>
<p class="publication"><span class="title">The Journal of antimicrobial chemotherapy</span>&nbsp;<span class="year">2012</span>;<span class="volume">67</span>;<span class="number">10</span>;<span class="pagerange">2307-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22729921">22729921</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fjac%2Fdks247">10.1093/jac/dks247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at 12q15 and 12q24 are associated with infant head circumference.</h4>
<p class="authors"><span class="author">Taal HR</span>, <span class="author">St Pourcain B</span>, <span class="author">Thiering E</span>, <span class="author">Das S</span>, <span class="author">Mook-Kanamori DO</span>, <span class="author">Warrington NM</span>, <span class="author">Kaakinen M</span>, <span class="author">Kreiner-M&oslash;ller E</span>, <span class="author">Bradfield JP</span>, <span class="author">Freathy RM</span>, <span class="author">Geller F</span>, <span class="author">Guxens M</span>, <span class="author">Cousminer DL</span>, <span class="author">Kerkhof M</span>, <span class="author">Timpson NJ</span>, <span class="author">Ikram MA</span>, <span class="author">Beilin LJ</span>, <span class="author">B&oslash;nnelykke K</span>, <span class="author">Buxton JL</span>, <span class="author">Charoen P</span>, <span class="author">Chawes BL</span>, <span class="author">Eriksson J</span>, <span class="author">Evans DM</span>, <span class="author">Hofman A</span>, <span class="author">Kemp JP</span>, <span class="author">Kim CE</span>, <span class="author">Klopp N</span>, <span class="author">Lahti J</span>, <span class="author">Lye SJ</span>, <span class="author">McMahon G</span>, <span class="author">Mentch FD</span>, <span class="author">M&uuml;ller-Nurasyid M</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Prokopenko I</span>, <span class="author">Rivadeneira F</span>, <span class="author">Steegers EA</span>, <span class="author">Sunyer J</span>, <span class="author">Tiesler C</span>, <span class="author">Yaghootkar H</span>, <span class="author">Cohorts for Heart and Aging Research in Genetic Epidemiology Consortium</span>, <span class="author">Breteler MM</span>, <span class="author">Decarli C</span>, <span class="author">Breteler MM</span>, <span class="author">Debette S</span>, <span class="author">Fornage M</span>, <span class="author">Gudnason V</span>, <span class="author">Launer LJ</span>, <span class="author">van der Lugt A</span>, <span class="author">Mosley TH</span>, <span class="author">Seshadri S</span>, <span class="author">Smith AV</span>, <span class="author">Vernooij MW</span>, <span class="author">Early Genetics &amp;amp; Lifecourse Epidemiology Consortium</span>, <span class="author">Blakemore AI</span>, <span class="author">Chiavacci RM</span>, <span class="author">Feenstra B</span>, <span class="author">Fernandez-Banet J</span>, <span class="author">Grant SF</span>, <span class="author">Hartikainen AL</span>, <span class="author">van der Heijden AJ</span>, <span class="author">I&ntilde;iguez C</span>, <span class="author">Lathrop M</span>, <span class="author">McArdle WL</span>, <span class="author">M&oslash;lgaard A</span>, <span class="author">Newnham JP</span>, <span class="author">Palmer LJ</span>, <span class="author">Palotie A</span>, <span class="author">Pouta A</span>, <span class="author">Ring SM</span>, <span class="author">Sovio U</span>, <span class="author">Standl M</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Wichmann HE</span>, <span class="author">Vissing NH</span>, <span class="author">DeCarli C</span>, <span class="author">van Duijn CM</span>, <span class="author">McCarthy MI</span>, <span class="author">Koppelman GH</span>, <span class="author">Estivill X</span>, <span class="author">Hattersley AT</span>, <span class="author">Melbye M</span>, <span class="author">Bisgaard H</span>, <span class="author">Pennell CE</span>, <span class="author">Widen E</span>, <span class="author">Hakonarson H</span>, <span class="author">Smith GD</span>, <span class="author">Heinrich J</span>, <span class="author">Jarvelin MR</span>, <span class="author">Jaddoe VW</span> and <span class="author">Early Growth Genetics Consortium</span></p>
<p class="affiliation">Department of Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">To identify genetic variants associated with head circumference in infancy, we performed a meta-analysis of seven genome-wide association studies (GWAS) (N = 10,768 individuals of European ancestry enrolled in pregnancy and/or birth cohorts) and followed up three lead signals in six replication studies (combined N = 19,089). rs7980687 on chromosome 12q24 (P = 8.1 &times; 10(-9)) and rs1042725 on chromosome 12q15 (P = 2.8 &times; 10(-10)) were robustly associated with head circumference in infancy. Although these loci have previously been associated with adult height, their effects on infant head circumference were largely independent of height (P = 3.8 &times; 10(-7) for rs7980687 and P = 1.3 &times; 10(-7) for rs1042725 after adjustment for infant height). A third signal, rs11655470 on chromosome 17q21, showed suggestive evidence of association with head circumference (P = 3.9 &times; 10(-6)). SNPs correlated to the 17q21 signal have shown genome-wide association with adult intracranial volume, Parkinson&#39;s disease and other neurodegenerative diseases, indicating that a common genetic variant in this region might link early brain growth with neurological disease in later life.</p>
<p class="grants">Funded by: British Heart Foundation; Canadian Institutes of Health Research: <span class="grant">MOP 82893</span>; Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600331</span>, <span class="grant">G0600705</span>, <span class="grant">G0800582</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>, <span class="grant">R01 HL087679</span>; NIA NIH HHS: <span class="grant">P30 AG010129</span>; NICHD NIH HHS: <span class="grant">1R01HD056465-01A1</span>, <span class="grant">R01 HD056465</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">085541/Z/08/Z</span>, <span class="grant">GR069224</span>, <span class="grant">WT083431MA</span>, <span class="grant">WT088431MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">532-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22504419">22504419</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2238">10.1038/ng.2238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare variant association testing for next-generation sequencing data via hierarchical clustering.</h4>
<p class="authors"><span class="author">Tachmazidou I</span>, <span class="author">Morris A</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%69%6F%61%6E%6E%61.%74%61%63%68%6D%61%7A%69%64%6F%75%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#111;&#97;&#110;&#110;&#97;&#46;&#116;&#97;&#99;&#104;&#109;&#97;&#122;&#105;&#100;&#111;&#117;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Objectives</strong>: It is thought that a proportion of the genetic susceptibility to complex diseases is due to low-frequency and rare variants. Next-generation sequencing in large populations facilitates the detection of rare variant associations to disease risk. In order to achieve adequate power to detect association at low-frequency and rare variants, locus-specific statistical methods are being developed that combine information across variants within a functional unit and test for association with this enriched signal through so-called burden tests.</p><p class="abstract"><strong>Methods</strong>: We propose a hierarchical clustering approach and a similarity kernel-based association test for continuous phenotypes. This method clusters individuals into groups, within which samples are assumed to be genetically similar, and subsequently tests the group effects among the different clusters.</p><p class="abstract"><strong>Results</strong>: The power of this approach is comparable to that of collapsing methods when causal variants have the same direction of effect, but its power is significantly higher compared to burden tests when both protective and risk variants are present in the region of interest. Overall, we observe that the Sequence Kernel Association Test (SKAT) is the most powerful approach under the allelic architectures considered.</p><p class="abstract"><strong>Conclusions</strong>: In our overall comparison, we find the analytical framework within which SKAT operates to yield higher power and to control type I error appropriately.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span>, <span class="grant">WT090532</span>, <span class="grant">WT098017</span></p>
<p class="publication"><span class="title">Human heredity</span>&nbsp;<span class="year">2012</span>;<span class="volume">74</span>;<span class="number">3-4</span>;<span class="pagerange">165-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23594494">23594494</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3668801">3668801</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000346022">10.1159/000346022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biochemical and functional analysis of two Plasmodium falciparum blood-stage 6-cys proteins: P12 and P41.</h4>
<p class="authors"><span class="author">Taechalertpaisarn T</span>, <span class="author">Crosnier C</span>, <span class="author">Bartholdson SJ</span>, <span class="author">Hodder AN</span>, <span class="author">Thompson J</span>, <span class="author">Bustamante LY</span>, <span class="author">Wilson DW</span>, <span class="author">Sanders PR</span>, <span class="author">Wright GJ</span>, <span class="author">Rayner JC</span>, <span class="author">Cowman AF</span>, <span class="author">Gilson PR</span> and <span class="author">Crabb BS</span></p>
<p class="affiliation">Burnet Institute, Melbourne, Victoria, Australia.</p><p class="abstract">The genomes of Plasmodium parasites that cause malaria in humans, other primates, birds, and rodents all encode multiple 6-cys proteins. Distinct 6-cys protein family members reside on the surface at each extracellular life cycle stage and those on the surface of liver infective and sexual stages have been shown to play important roles in hepatocyte growth and fertilization respectively. However, 6-cys proteins associated with the blood-stage forms of the parasite have no known function. Here we investigate the biochemical nature and function of two blood-stage 6-cys proteins in Plasmodium falciparum, the most pathogenic species to afflict humans. We show that native P12 and P41 form a stable heterodimer on the infective merozoite surface and are secreted following invasion, but could find no evidence that this complex mediates erythrocyte-receptor binding. That P12 and P41 do not appear to have a major role as adhesins to erythrocyte receptors was supported by the observation that antisera to these proteins did not substantially inhibit erythrocyte invasion. To investigate other functional roles for these proteins their genes were successfully disrupted in P. falciparum, however P12 and P41 knockout parasites grew at normal rates in vitro and displayed no other obvious phenotypic changes. It now appears likely that these blood-stage 6-cys proteins operate as a pair and play redundant roles either in erythrocyte invasion or in host-immune interactions.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e41937</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22848665">22848665</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3407074">3407074</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0041937">10.1371/journal.pone.0041937</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An X-linked channelopathy with cardiomegaly due to a CLIC2 mutation enhancing ryanodine receptor channel activity.</h4>
<p class="authors"><span class="author">Takano K</span>, <span class="author">Liu D</span>, <span class="author">Tarpey P</span>, <span class="author">Gallant E</span>, <span class="author">Lam A</span>, <span class="author">Witham S</span>, <span class="author">Alexov E</span>, <span class="author">Chaubey A</span>, <span class="author">Stevenson RE</span>, <span class="author">Schwartz CE</span>, <span class="author">Board PG</span> and <span class="author">Dulhunty AF</span></p>
<p class="affiliation">K.T. and D.L. contributed equally to the paper.</p>
<p class="abstract">Chloride intracellular channel 2 (CLIC2) protein is a member of the glutathione transferase class of proteins. Its' only known function is the regulation of ryanodine receptor (RyR) intracellular Ca(2+) release channels. These RyR proteins play a major role in the regulation of Ca(2+) signaling in many cells. Utilizing exome capture and deep sequencing of genes on the X-chromosome, we have identified a mutation in CLIC2 (c.303C>G, p.H101Q) which is associated with X-linked intellectual disability (ID), atrial fibrillation, cardiomegaly, congestive heart failure (CHF), some somatic features and seizures. Functional studies of the H101Q variant indicated that it stimulated rather than inhibited the action of RyR channels, with channels remaining open for longer times and potentially amplifying Ca(2+) signals dependent on RyR channel activity. The overly active RyRs in cardiac and skeletal muscle cells and neuronal cells would result in abnormal cardiac function and trigger post-synaptic pathways and neurotransmitter release. The presence of both cardiomegaly and CHF in the two affected males and atrial fibrillation in one are consistent with abnormal RyR2 channel function. Since the dysfunction of RyR2 channels in the brain via 'leaky mutations' can result in mild developmental delay and seizures, our data also suggest a vital role for the CLIC2 protein in maintaining normal cognitive function via its interaction with RyRs in the brain. Therefore, our patients appear to suffer from a new channelopathy comprised of ID, seizures and cardiac problems because of enhanced Ca(2+) release through RyRs in neuronal cells and cardiac muscle cells.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM093937</span>; NLM NIH HHS: <span class="grant">R03 LM009748</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">20</span>;<span class="pagerange">4497-507</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22814392" rel="external">22814392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3459470" rel="external">3459470</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fdds292" rel="external">10.1093/hmg/dds292</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at the MHC locus and at chromosome 16q24.1 predispose to Barrett's esophagus.</h4>
<p class="authors"><span class="author">The Esophageal Adenocarcinoma Genetics Consortium</span>, <span class="author">The Wellcome Trust Case Control Consortium 2</span>, <span class="author">Su Z</span>, <span class="author">Gay LJ</span>, <span class="author">Strange A</span>, <span class="author">Palles C</span>, <span class="author">Band G</span>, <span class="author">Whiteman DC</span>, <span class="author">Lescai F</span>, <span class="author">Langford C</span>, <span class="author">Nanji M</span>, <span class="author">Edkins S</span>, <span class="author">van der Winkel A</span>, <span class="author">Levine D</span>, <span class="author">Sasieni P</span>, <span class="author">Bellenguez C</span>, <span class="author">Howarth K</span>, <span class="author">Freeman C</span>, <span class="author">Trudgill N</span>, <span class="author">Tucker AT</span>, <span class="author">Pirinen M</span>, <span class="author">Peppelenbosch MP</span>, <span class="author">van der Laan LJ</span>, <span class="author">Kuipers EJ</span>, <span class="author">Drenth JP</span>, <span class="author">Peters WH</span>, <span class="author">Reynolds JV</span>, <span class="author">Kelleher DP</span>, <span class="author">McManus R</span>, <span class="author">Grabsch H</span>, <span class="author">Prenen H</span>, <span class="author">Bisschops R</span>, <span class="author">Krishnadath K</span>, <span class="author">Siersema PD</span>, <span class="author">van Baal JW</span>, <span class="author">Middleton M</span>, <span class="author">Petty R</span>, <span class="author">Gillies R</span>, <span class="author">Burch N</span>, <span class="author">Bhandari P</span>, <span class="author">Paterson S</span>, <span class="author">Edwards C</span>, <span class="author">Penman I</span>, <span class="author">Vaidya K</span>, <span class="author">Ang Y</span>, <span class="author">Murray I</span>, <span class="author">Patel P</span>, <span class="author">Ye W</span>, <span class="author">Mullins P</span>, <span class="author">Wu AH</span>, <span class="author">Bird NC</span>, <span class="author">Dallal H</span>, <span class="author">Shaheen NJ</span>, <span class="author">Murray LJ</span>, <span class="author">Koss K</span>, <span class="author">Bernstein L</span>, <span class="author">Romero Y</span>, <span class="author">Hardie LJ</span>, <span class="author">Zhang R</span>, <span class="author">Winter H</span>, <span class="author">Corley DA</span>, <span class="author">Panter S</span>, <span class="author">Risch HA</span>, <span class="author">Reid BJ</span>, <span class="author">Sargeant I</span>, <span class="author">Gammon MD</span>, <span class="author">Smart H</span>, <span class="author">Dhar A</span>, <span class="author">McMurtry H</span>, <span class="author">Ali H</span>, <span class="author">Liu G</span>, <span class="author">Casson AG</span>, <span class="author">Chow WH</span>, <span class="author">Rutter M</span>, <span class="author">Tawil A</span>, <span class="author">Morris D</span>, <span class="author">Nwokolo C</span>, <span class="author">Isaacs P</span>, <span class="author">Rodgers C</span>, <span class="author">Ragunath K</span>, <span class="author">Macdonald C</span>, <span class="author">Haigh C</span>, <span class="author">Monk D</span>, <span class="author">Davies G</span>, <span class="author">Wajed S</span>, <span class="author">Johnston D</span>, <span class="author">Gibbons M</span>, <span class="author">Cullen S</span>, <span class="author">Church N</span>, <span class="author">Langley R</span>, <span class="author">Griffin M</span>, <span class="author">Alderson D</span>, <span class="author">Deloukas P</span>, <span class="author">Hunt SE</span>, <span class="author">Gray E</span>, <span class="author">Dronov S</span>, <span class="author">Potter SC</span>, <span class="author">Tashakkori-Ghanbaria A</span>, <span class="author">Anderson M</span>, <span class="author">Brooks C</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Casas JP</span>, <span class="author">Corvin A</span>, <span class="author">Duncanson A</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">Palmer CN</span>, <span class="author">Plomin R</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Trembath RC</span>, <span class="author">Viswanathan AC</span>, <span class="author">Wood N</span>, <span class="author">Trynka G</span>, <span class="author">Wijmenga C</span>, <span class="author">Cazier JB</span>, <span class="author">Atherfold P</span>, <span class="author">Nicholson AM</span>, <span class="author">Gellatly NL</span>, <span class="author">Glancy D</span>, <span class="author">Cooper SC</span>, <span class="author">Cunningham D</span>, <span class="author">Lind T</span>, <span class="author">Hapeshi J</span>, <span class="author">Ferry D</span>, <span class="author">Rathbone B</span>, <span class="author">Brown J</span>, <span class="author">Love S</span>, <span class="author">Attwood S</span>, <span class="author">Macgregor S</span>, <span class="author">Watson P</span>, <span class="author">Sanders S</span>, <span class="author">Ek W</span>, <span class="author">Harrison RF</span>, <span class="author">Moayyedi P</span>, <span class="author">de Caestecker J</span>, <span class="author">Barr H</span>, <span class="author">Stupka E</span>, <span class="author">Vaughan TL</span>, <span class="author">Peltonen L</span>, <span class="author">Spencer CC</span>, <span class="author">Tomlinson I</span>, <span class="author">Donnelly P</span> and <span class="author">Jankowski JA</span></p>
<p class="affiliation">1] Wellcome Trust Centre for Human Genetics, Oxford, UK. [2].</p>
<p class="abstract">Barrett's esophagus is an increasingly common disease that is strongly associated with reflux of stomach acid and usually a hiatus hernia, and it strongly predisposes to esophageal adenocarcinoma (EAC), a tumor with a very poor prognosis. We report the first genome-wide association study on Barrett's esophagus, comprising 1,852 UK cases and 5,172 UK controls in the discovery stage and 5,986 cases and 12,825 controls in the replication stage. Variants at two loci were associated with disease risk: chromosome 6p21, rs9257809 (P(combined) = 4.09 × 10(-9); odds ratio (OR) = 1.21, 95% confidence interval (CI) =1.13-1.28), within the major histocompatibility complex locus, and chromosome 16q24, rs9936833 (P(combined) = 2.74 × 10(-10); OR = 1.14, 95% CI = 1.10-1.19), for which the closest protein-coding gene is FOXF1, which is implicated in esophageal development and structure. We found evidence that many common variants of small effect contribute to genetic susceptibility to Barrett's esophagus and that SNP alleles predisposing to obesity also increase risk for Barrett's esophagus.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22961001" rel="external">22961001</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2408" rel="external">10.1038/ng.2408</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Seeking consent to genetic and genomic research in a rural Ghanaian setting: A qualitative study of the MalariaGEN experience.</h4>
<p class="authors"><span class="author">Tindana P</span>, <span class="author">Bull S</span>, <span class="author">Amenga-Etego L</span>, <span class="author">de Vries J</span>, <span class="author">Aborigo R</span>, <span class="author">Koram K</span>, <span class="author">Kwiatkowski D</span> and <span class="author">Parker M</span></p>
<p class="affiliation">The Ethox Centre, Department of Public Health, University of Oxford, Old Road Campus, Headington, Oxford, OX3 7LF, United Kingdom. <a href="mailto:%73%75%73%61%6E.%62%75%6C%6C%40%65%74%68%6F%78.%6F%78.%61%63.%75%6B">&#115;&#117;&#115;&#97;&#110;&#46;&#98;&#117;&#108;&#108;&#64;&#101;&#116;&#104;&#111;&#120;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">Unlabelled: ABSTRACT:

Background: Seeking consent for genetic and genomic research can be challenging, particularly in populations with low literacy levels, and in emergency situations. All of these factors were relevant to the MalariaGEN study of genetic factors influencing immune responses to malaria in northern rural Ghana. This study sought to identify issues arising in practice during the enrolment of paediatric cases with severe malaria and matched healthy controls into the MalariaGEN study.

Methods: The study used a rapid assessment incorporating multiple qualitative methods including in depth interviews, focus group discussions and observations of consent processes. Differences between verbal information provided during community engagement processes, and consent processes during the enrolment of cases and controls were identified, as well as the factors influencing the tailoring of such information.

Results: MalariaGEN participants and field staff seeking consent were generally satisfied with their understanding of the project and were familiar with aspects of the study relating to malaria. Some genetic aspects of the study were also well understood. Participants and staff seeking consent were less aware of the methodologies employed during genomic research and their implications, such as the breadth of data generated and the potential for future secondary research.Moreover, trust in and previous experience with the Navrongo Health Research Centre which was conducting the research influenced beliefs about the benefits of participating in the MalariaGEN study and subsequent decision-making about research participation.

Conclusions: It is important to recognise that some aspects of complex genomic research may be of less interest to and less well understood by research participants and that such gaps in understanding may not be entirely addressed by best practice in the design and conduct of consent processes. In such circumstances consideration needs to be given to additional protections for participants that may need to be implemented in such research, and how best to provide such protections.Capacity building for research ethics committees with limited familiarity with genetic and genomic research, and appropriate engagement with communities to elicit opinions of the ethical issues arising and acceptability of downstream uses of genome wide association data are likely to be important.</p>
<p class="publication"><span class="title">BMC medical ethics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22747883" rel="external">22747883</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3441464" rel="external">3441464</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1472-6939-13-15" rel="external">10.1186/1472-6939-13-15</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The tomato genome sequence provides insights into fleshy fruit evolution.</h4>
<p class="authors"><span class="author">Tomato Genome Consortium</span></p><p class="abstract">Tomato (Solanum lycopersicum) is a major crop plant and a model system for fruit development. Solanum is one of the largest angiosperm genera and includes annual and perennial plants from diverse habitats. Here we present a high-quality genome sequence of domesticated tomato, a draft sequence of its closest wild relative, Solanum pimpinellifolium, and compare them to each other and to the potato genome (Solanum tuberosum). The two tomato genomes show only 0.6% nucleotide divergence and signs of recent admixture, but show more than 8% divergence from potato, with nine large and several smaller inversions. In contrast to Arabidopsis, but similar to soybean, tomato and potato small RNAs map predominantly to gene-rich chromosomal regions, including gene promoters. The Solanum lineage has experienced two consecutive genome triplications: one that is ancient and shared with rosids, and a more recent one. These triplications set the stage for the neofunctionalization of genes controlling fruit characteristics, such as colour and fleshiness.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C509731/1</span>, <span class="grant">BB/G006199/1</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">485</span>;<span class="number">7400</span>;<span class="pagerange">635-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22660326">22660326</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3378239">3378239</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11119">10.1038/nature11119</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic polish for shoe-leather epidemiology.</h4>
<p class="authors"><span class="author">Tong SY</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">This month's Genome Watch highlights the advantages of using whole-genome sequencing for infectious-disease surveillance and infection control.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23202526" rel="external">23202526</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2935" rel="external">10.1038/nrmicro2935</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate polymorphisms and severe malaria in a malian population.</h4>
<p class="authors"><span class="author">Toure O</span>, <span class="author">Konate S</span>, <span class="author">Sissoko S</span>, <span class="author">Niangaly A</span>, <span class="author">Barry A</span>, <span class="author">Sall AH</span>, <span class="author">Diarra E</span>, <span class="author">Poudiougou B</span>, <span class="author">Sepulveda N</span>, <span class="author">Campino S</span>, <span class="author">Rockett KA</span>, <span class="author">Clark TG</span>, <span class="author">Thera MA</span>, <span class="author">Doumbo O</span> and <span class="author">in collaboration with The MalariaGEN Consortium</span></p>
<p class="affiliation">Malaria Research and Training Centre, Faculty of Medicine, Pharmacy and Dentistry, University of Bamako, Bamako, Mali.</p>
<p class="abstract">Malaria is a major health burden in sub-Saharan African countries, including Mali. The disease is complex, with multiple genetic determinants influencing the observed variation in response to infection, progression, and severity. We assess the influence of sixty-four candidate loci, including the sickle cell polymorphism (HbS), on severe malaria in a case-control study consisting of over 900 individuals from Bamako, Mali. We confirm the known protective effects of the blood group O and the HbS AS genotype on life-threatening malaria. In addition, our analysis revealed a marginal susceptibility effect for the CD40 ligand (CD40L)+220C allele. The lack of statistical evidence for other candidates may demonstrate the need for large-scale genome-wide association studies in malaria to discover new polymorphisms. It also demonstrates the need for establishing the region-specific repertoire of functional variation in important genes, including the glucose-6-phosphatase deficiency gene, before embarking on focused genotyping.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e43987</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22957039" rel="external">22957039</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3434208" rel="external">3434208</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0043987" rel="external">10.1371/journal.pone.0043987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The first green revolution.</h4>
<p class="authors"><span class="author">Tsai IJ</span></p><p class="abstract">This month&#39;s Genome Watch describes how analysis of a basal member of the Plantae can inform us about the evolution of photosynthetic eukaryotes.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">5</span>;<span class="pagerange">314</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22504859" rel="external">22504859</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2781" rel="external">10.1038/nrmicro2781</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular characterization of the 2011 Hong Kong scarlet fever outbreak.</h4>
<p class="authors"><span class="author">Tse H</span>, <span class="author">Bao JY</span>, <span class="author">Davies MR</span>, <span class="author">Maamary P</span>, <span class="author">Tsoi HW</span>, <span class="author">Tong AH</span>, <span class="author">Ho TC</span>, <span class="author">Lin CH</span>, <span class="author">Gillen CM</span>, <span class="author">Barnett TC</span>, <span class="author">Chen JH</span>, <span class="author">Lee M</span>, <span class="author">Yam WC</span>, <span class="author">Wong CK</span>, <span class="author">Ong CL</span>, <span class="author">Chan YW</span>, <span class="author">Wu CW</span>, <span class="author">Ng T</span>, <span class="author">Lim WW</span>, <span class="author">Tsang TH</span>, <span class="author">Tse CW</span>, <span class="author">Dougan G</span>, <span class="author">Walker MJ</span>, <span class="author">Lok S</span> and <span class="author">Yuen KY</span></p>
<p class="affiliation">Department of Microbiology, The University of Hong Kong, Hong Kong Special Administrative Region, China. <a href="mailto:%68%74%73%65%40%68%6B%75%63%63.%68%6B%75.%68%6B">&#104;&#116;&#115;&#101;&#64;&#104;&#107;&#117;&#99;&#99;&#46;&#104;&#107;&#117;&#46;&#104;&#107;</a></p><p class="abstract">A scarlet fever outbreak occurred in Hong Kong in 2011. The majority of cases resulted in the isolation of Streptococcus pyogenes emm12 with multiple antibiotic resistances. Phylogenetic analysis of 22 emm12 scarlet fever outbreak isolates, 7 temporally and geographically matched emm12 non-scarlet fever isolates, and 18 emm12 strains isolated during 2005-2010 indicated the outbreak was multiclonal. Genome sequencing of 2 nonclonal scarlet fever isolates (HKU16 and HKU30), coupled with diagnostic polymerase chain reaction assays, identified 2 mobile genetic elements distributed across the major lineages: a 64.9-kb integrative and conjugative element encoding tetracycline and macrolide resistance and a 46.4-kb prophage encoding superantigens SSA and SpeC and the DNase Spd1. Phenotypic comparison of HKU16 and HKU30 with the S. pyogenes M1T1 strain 5448 revealed that HKU16 displays increased adherence to HEp-2 human epithelial cells, whereas HKU16, HKU30, and 5448 exhibit equivalent resistance to neutrophils and virulence in a humanized plasminogen murine model. However, in contrast to M1T1, the virulence of HKU16 and HKU30 was not associated with covRS mutation. The multiclonal nature of the emm12 scarlet fever isolates suggests that factors such as mobile genetic elements, environmental factors, and host immune status may have contributed to the 2011 scarlet fever outbreak.</p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">206</span>;<span class="number">3</span>;<span class="pagerange">341-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22615319">22615319</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4125623">4125623</a>; DOI: <a href="http://dx.doi.org/10.1093%2Finfdis%2Fjis362">10.1093/infdis/jis362</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of 15 new psoriasis susceptibility loci highlights the role of innate immunity.</h4>
<p class="authors"><span class="author">Tsoi LC</span>, <span class="author">Spain SL</span>, <span class="author">Knight J</span>, <span class="author">Ellinghaus E</span>, <span class="author">Stuart PE</span>, <span class="author">Capon F</span>, <span class="author">Ding J</span>, <span class="author">Li Y</span>, <span class="author">Tejasvi T</span>, <span class="author">Gudjonsson JE</span>, <span class="author">Kang HM</span>, <span class="author">Allen MH</span>, <span class="author">McManus R</span>, <span class="author">Novelli G</span>, <span class="author">Samuelsson L</span>, <span class="author">Schalkwijk J</span>, <span class="author">St&aring;hle M</span>, <span class="author">Burden AD</span>, <span class="author">Smith CH</span>, <span class="author">Cork MJ</span>, <span class="author">Estivill X</span>, <span class="author">Bowcock AM</span>, <span class="author">Krueger GG</span>, <span class="author">Weger W</span>, <span class="author">Worthington J</span>, <span class="author">Tazi-Ahnini R</span>, <span class="author">Nestle FO</span>, <span class="author">Hayday A</span>, <span class="author">Hoffmann P</span>, <span class="author">Winkelmann J</span>, <span class="author">Wijmenga C</span>, <span class="author">Langford C</span>, <span class="author">Edkins S</span>, <span class="author">Andrews R</span>, <span class="author">Blackburn H</span>, <span class="author">Strange A</span>, <span class="author">Band G</span>, <span class="author">Pearson RD</span>, <span class="author">Vukcevic D</span>, <span class="author">Spencer CC</span>, <span class="author">Deloukas P</span>, <span class="author">Mrowietz U</span>, <span class="author">Schreiber S</span>, <span class="author">Weidinger S</span>, <span class="author">Koks S</span>, <span class="author">Kingo K</span>, <span class="author">Esko T</span>, <span class="author">Metspalu A</span>, <span class="author">Lim HW</span>, <span class="author">Voorhees JJ</span>, <span class="author">Weichenthal M</span>, <span class="author">Wichmann HE</span>, <span class="author">Chandran V</span>, <span class="author">Rosen CF</span>, <span class="author">Rahman P</span>, <span class="author">Gladman DD</span>, <span class="author">Griffiths CE</span>, <span class="author">Reis A</span>, <span class="author">Kere J</span>, <span class="author">Collaborative Association Study of Psoriasis (CASP)</span>, <span class="author">Duffin KC</span>, <span class="author">Helms C</span>, <span class="author">Goldgar D</span>, <span class="author">Li Y</span>, <span class="author">Paschall J</span>, <span class="author">Malloy MJ</span>, <span class="author">Pullinger CR</span>, <span class="author">Kane JP</span>, <span class="author">Gardner J</span>, <span class="author">Perlmutter A</span>, <span class="author">Miner A</span>, <span class="author">Feng BJ</span>, <span class="author">Hiremagalore R</span>, <span class="author">Ike RW</span>, <span class="author">Christophers E</span>, <span class="author">Henseler T</span>, <span class="author">Ruether A</span>, <span class="author">Schrodi SJ</span>, <span class="author">Prahalad S</span>, <span class="author">Guthery SL</span>, <span class="author">Fischer J</span>, <span class="author">Liao W</span>, <span class="author">Kwok P</span>, <span class="author">Menter A</span>, <span class="author">Lathrop GM</span>, <span class="author">Wise C</span>, <span class="author">Begovich AB</span>, <span class="author">Genetic Analysis of Psoriasis Consortium</span>, <span class="author">Onoufriadis A</span>, <span class="author">Weale ME</span>, <span class="author">Hofer A</span>, <span class="author">Salmhofer W</span>, <span class="author">Wolf P</span>, <span class="author">Kainu K</span>, <span class="author">Saarialho-Kere U</span>, <span class="author">Suomela S</span>, <span class="author">Badorf P</span>, <span class="author">H&uuml;ffmeier U</span>, <span class="author">Kurrat W</span>, <span class="author">K&uuml;ster W</span>, <span class="author">Lascorz J</span>, <span class="author">M&ouml;ssner R</span>, <span class="author">Sch&uuml;rmeier-Horst F</span>, <span class="author">St&auml;nder M</span>, <span class="author">Traupe H</span>, <span class="author">Bergboer JG</span>, <span class="author">Heijer MD</span>, <span class="author">van de Kerkhof PC</span>, <span class="author">Zeeuwen PL</span>, <span class="author">Barnes L</span>, <span class="author">Campbell LE</span>, <span class="author">Cusack C</span>, <span class="author">Coleman C</span>, <span class="author">Conroy J</span>, <span class="author">Ennis S</span>, <span class="author">Fitzgerald O</span>, <span class="author">Gallagher P</span>, <span class="author">Irvine AD</span>, <span class="author">Kirby B</span>, <span class="author">Markham T</span>, <span class="author">McLean WH</span>, <span class="author">McPartlin J</span>, <span class="author">Rogers SF</span>, <span class="author">Ryan AW</span>, <span class="author">Zawirska A</span>, <span class="author">Giardina E</span>, <span class="author">Lepre T</span>, <span class="author">Perricone C</span>, <span class="author">Mart&iacute;n-Ezquerra G</span>, <span class="author">Pujol RM</span>, <span class="author">Riveira-Munoz E</span>, <span class="author">Inerot A</span>, <span class="author">Naluai AT</span>, <span class="author">Mallbris L</span>, <span class="author">Wolk K</span>, <span class="author">Leman J</span>, <span class="author">Barton A</span>, <span class="author">Warren RB</span>, <span class="author">Young HS</span>, <span class="author">Ricano-Ponce I</span>, <span class="author">Trynka G</span>, <span class="author">Psoriasis Association Genetics Extension</span>, <span class="author">Pellett FJ</span>, <span class="author">Henschel A</span>, <span class="author">Aurand M</span>, <span class="author">Bebo B</span>, <span class="author">Gieger C</span>, <span class="author">Illig T</span>, <span class="author">Moebus S</span>, <span class="author">J&ouml;ckel KH</span>, <span class="author">Erbel R</span>, <span class="author">Wellcome Trust Case Control Consortium 2</span>, <span class="author">Donnelly P</span>, <span class="author">Peltonen L</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Brown MA</span>, <span class="author">Casas JP</span>, <span class="author">Corvin A</span>, <span class="author">Craddock N</span>, <span class="author">Duncanson A</span>, <span class="author">Jankowski J</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">McCarthy MI</span>, <span class="author">Palmer CN</span>, <span class="author">Plomin R</span>, <span class="author">Rautanen A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Samani N</span>, <span class="author">Viswanathan AC</span>, <span class="author">Wood NW</span>, <span class="author">Bellenguez C</span>, <span class="author">Freeman C</span>, <span class="author">Hellenthal G</span>, <span class="author">Giannoulatou E</span>, <span class="author">Pirinen M</span>, <span class="author">Su Z</span>, <span class="author">Hunt SE</span>, <span class="author">Gwilliam R</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Dronov S</span>, <span class="author">Gillman M</span>, <span class="author">Gray E</span>, <span class="author">Hammond N</span>, <span class="author">Jayakumar A</span>, <span class="author">McCann OT</span>, <span class="author">Liddle J</span>, <span class="author">Perez ML</span>, <span class="author">Potter SC</span>, <span class="author">Ravindrarajah R</span>, <span class="author">Ricketts M</span>, <span class="author">Waller M</span>, <span class="author">Weston P</span>, <span class="author">Widaa S</span>, <span class="author">Whittaker P</span>, <span class="author">Nair RP</span>, <span class="author">Franke A</span>, <span class="author">Barker JN</span>, <span class="author">Abecasis GR</span>, <span class="author">Elder JT</span> and <span class="author">Trembath RC</span></p>
<p class="affiliation">1] Department of Biostatistics, Center for Statistical Genetics, University of Michigan Ann Arbor, Michigan, USA. [2].</p>
<p class="abstract">To gain further insight into the genetic architecture of psoriasis, we conducted a meta-analysis of 3 genome-wide association studies (GWAS) and 2 independent data sets genotyped on the Immunochip, including 10,588 cases and 22,806 controls. We identified 15 new susceptibility loci, increasing to 36 the number associated with psoriasis in European individuals. We also identified, using conditional analyses, five independent signals within previously known loci. The newly identified loci shared with other autoimmune diseases include candidate genes with roles in regulating T-cell function (such as RUNX3, TAGAP and STAT3). Notably, they included candidate genes whose products are involved in innate host defense, including interferon-mediated antiviral responses (DDX58), macrophage activation (ZC3H12C) and nuclear factor (NF)-ÎºB signaling (CARD14 and CARM1). These results portend a better understanding of shared and distinctive genetic determinants of immune-mediated inflammatory disorders and emphasize the importance of the skin in innate and acquired host defense.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23143594" rel="external">23143594</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2467" rel="external">10.1038/ng.2467</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detailed metabolic and genetic characterization reveals new associations for 30 known lipid loci.</h4>
<p class="authors"><span class="author">Tukiainen T</span>, <span class="author">Kettunen J</span>, <span class="author">Kangas AJ</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Soininen P</span>, <span class="author">Sarin AP</span>, <span class="author">Tikkanen E</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Savolainen MJ</span>, <span class="author">Kaski K</span>, <span class="author">Pouta A</span>, <span class="author">Jula A</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Viikari J</span>, <span class="author">Taskinen MR</span>, <span class="author">Jauhiainen M</span>, <span class="author">Eriksson JG</span>, <span class="author">Raitakari O</span>, <span class="author">Salomaa V</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Perola M</span>, <span class="author">Palotie A</span>, <span class="author">Ala-Korpela M</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland FIMM, Helsinki University Hospital, FI-00014 University of Helsinki, Helsinki, Finland.</p><p class="abstract">Almost 100 genetic loci are known to affect serum cholesterol and triglyceride levels. For many of these loci, the biological function and causal variants remain unknown. We performed an association analysis of the reported 95 lipid loci against 216 metabolite measures, including 95 measurements on lipids and lipoprotein subclasses, obtained via serum nuclear magnetic resonance metabolomics and four enzymatic lipid traits in 8330 individuals from Finland. The genetic variation in the loci was investigated using a dense set of 440 807 directly genotyped and imputed variants around the previously identified lead single nucleotide polymorphisms (SNPs). For 30 of the 95 loci, we identified new metabolic or genetic associations (P &lt; 5 &times; 10(-8)). In the majority of the loci, the strongest association was to a more specific metabolite measure than the enzymatic lipids. In four loci, the smallest high-density lipoprotein measures showed effects opposite to the larger ones, and 14 loci had associations beyond the individual lipoprotein measures. In 27 loci, we identified SNPs with a stronger association than the previously reported markers and 12 loci harboured multiple, statistically independent variants. Our data show considerable diversity in association patterns between the loci originally identified through associations with enzymatic lipid measures and reveal association profiles of far greater detail than from routine clinical lipid measures. Additionally, a dense marker set and a homogeneous population allow for detailed characterization of the genetic association signals to a resolution exceeding that achieved so far. Further understanding of the rich variability in genetic effects on metabolites provides insights into the biological processes modifying lipid levels.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">6</span>;<span class="pagerange">1444-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22156771">22156771</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr581">10.1093/hmg/ddr581</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Climate change at northern latitudes: rising atmospheric humidity decreases transpiration, N-uptake and growth rate of hybrid aspen.</h4>
<p class="authors"><span class="author">Tullus A</span>, <span class="author">Kupper P</span>, <span class="author">Sellin A</span>, <span class="author">Parts L</span>, <span class="author">S&otilde;ber J</span>, <span class="author">Tullus T</span>, <span class="author">L&otilde;hmus K</span>, <span class="author">S&otilde;ber A</span> and <span class="author">Tullus H</span></p>
<p class="affiliation">Department of Silviculture, Institute of Forestry and Rural Engineering, Estonian University of Life Sciences, Tartu, Estonia.</p>
<p class="abstract">At northern latitudes a rise in atmospheric humidity and precipitation is predicted as a consequence of global climate change. We studied several growth and functional traits of hybrid aspen (Populus tremula L.×P. tremuloides Michx.) in response to elevated atmospheric humidity (on average 7% over the ambient level) in a free air experimental facility during three growing seasons (2008-2010) in Estonia, which represents northern temperate climate (boreo-nemoral zone). Data were collected from three humidified (H) and three control (C) plots, and analysed using nested linear models. Elevated air humidity significantly reduced height, stem diameter and stem volume increments and transpiration of the trees whereas these effects remained highly significant also after considering the side effects from soil-related confounders within the 2.7 ha study area. Tree leaves were smaller, lighter and had lower leaf mass per area (LMA) in H plots. The magnitude and significance of the humidity treatment effect - inhibition of above-ground growth rate - was more pronounced in larger trees. The lower growth rate in the humidified plots can be partly explained by a decrease in transpiration-driven mass flow of NO(3) (-) in soil, resulting in a significant reduction in the measured uptake of N to foliage in the H plots. The results suggest that the potential growth improvement of fast-growing trees like aspens, due to increasing temperature and atmospheric CO(2) concentration, might be smaller than expected at high latitudes if a rise in atmospheric humidity simultaneously takes place.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">8</span>;<span class="pagerange">e42648</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22880067" rel="external">22880067</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3412825" rel="external">3412825</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0042648" rel="external">10.1371/journal.pone.0042648</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene-gene interactions in breast cancer susceptibility.</h4>
<p class="authors"><span class="author">Turnbull C</span>, <span class="author">Seal S</span>, <span class="author">Renwick A</span>, <span class="author">Warren-Perry M</span>, <span class="author">Hughes D</span>, <span class="author">Elliott A</span>, <span class="author">Pernet D</span>, <span class="author">Peock S</span>, <span class="author">Adlard JW</span>, <span class="author">Barwell J</span>, <span class="author">Berg J</span>, <span class="author">Brady AF</span>, <span class="author">Brewer C</span>, <span class="author">Brice G</span>, <span class="author">Chapman C</span>, <span class="author">Cook J</span>, <span class="author">Davidson R</span>, <span class="author">Donaldson A</span>, <span class="author">Douglas F</span>, <span class="author">Greenhalgh L</span>, <span class="author">Henderson A</span>, <span class="author">Izatt L</span>, <span class="author">Kumar A</span>, <span class="author">Lalloo F</span>, <span class="author">Miedzybrodzka Z</span>, <span class="author">Morrison PJ</span>, <span class="author">Paterson J</span>, <span class="author">Porteous M</span>, <span class="author">Rogers MT</span>, <span class="author">Shanley S</span>, <span class="author">Walker L</span>, <span class="author">Breast Cancer Susceptibility Collaboration (UK), EMBRACE</span>, <span class="author">Ahmed M</span>, <span class="author">Eccles D</span>, <span class="author">Evans DG</span>, <span class="author">Donnelly P</span>, <span class="author">Easton DF</span>, <span class="author">Stratton MR</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Division of Genetics and Epidemiology, The Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK. <a href="mailto:%63%6C%61%72%65.%74%75%72%6E%62%75%6C%6C%40%69%63%72.%61%63.%75%6B">&#99;&#108;&#97;&#114;&#101;&#46;&#116;&#117;&#114;&#110;&#98;&#117;&#108;&#108;&#64;&#105;&#99;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">There have been few definitive examples of gene-gene interactions in humans. Through mutational analyses in 7325 individuals, we report four interactions (defined as departures from a multiplicative model) between mutations in the breast cancer susceptibility genes ATM and CHEK2 with BRCA1 and BRCA2 (case-only interaction between ATM and BRCA1/BRCA2 combined, P = 5.9 &times; 10(-4); ATM and BRCA1, P= 0.01; ATM and BRCA2, P= 0.02; CHEK2 and BRCA1/BRCA2 combined, P = 2.1 &times; 10(-4); CHEK2 and BRCA1, P= 0.01; CHEK2 and BRCA2, P= 0.01). The interactions are such that the resultant risk of breast cancer is lower than the multiplicative product of the constituent risks, and plausibly reflect the functional relationships of the encoded proteins in DNA repair. These findings have important implications for models of disease predisposition and clinical translation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11990</span>, <span class="grant">C1287/A10118</span>, <span class="grant">C1287/A8874</span>, <span class="grant">C8620/A8372</span>, <span class="grant">C8620/A8857</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0700491</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">4</span>;<span class="pagerange">958-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22072393">22072393</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4125627">4125627</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddr525">10.1093/hmg/ddr525</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Superantigenic Activity of emm3 Streptococcus pyogenes Is Abrogated by a Conserved, Naturally Occurring smeZ Mutation.</h4>
<p class="authors"><span class="author">Turner CE</span>, <span class="author">Sommerlad M</span>, <span class="author">McGregor K</span>, <span class="author">Davies FJ</span>, <span class="author">Pichon B</span>, <span class="author">Chong DL</span>, <span class="author">Farzaneh L</span>, <span class="author">Holden MT</span>, <span class="author">Spratt BG</span>, <span class="author">Efstratiou A</span> and <span class="author">Sriskandan S</span></p>
<p class="affiliation">Department of Infectious Diseases & Immunity, Imperial College London, London, United Kingdom.</p>
<p class="abstract">Streptococcus pyogenes M/emm3 strains have been epidemiologically linked with enhanced infection severity and risk of streptococcal toxic shock syndrome (STSS), a syndrome triggered by superantigenic stimulation of T cells. Comparison of S. pyogenes strains causing STSS demonstrated that emm3 strains were surprisingly less mitogenic than other emm-types (emm1, emm12, emm18, emm28, emm87, emm89) both in vitro and in vivo, indicating poor superantigenic activity. We identified a 13 bp deletion in the superantigen smeZ gene of all emm3 strains tested. The deletion led to a premature stop codon in smeZ, and was not present in other major emm-types tested. Expression of a functional non-M3-smeZ gene successfully enhanced mitogenic activity in emm3 S. pyogenes and also restored mitogenic activity to emm1 and emm89 S. pyogenes strains where the smeZ gene had been disrupted. In contrast, the M3-smeZ gene with the 13 bp deletion could not enhance or restore mitogenicity in any of these S. pyogenes strains, confirming that M3-smeZ is non-functional regardless of strain background. The mutation in M3-smeZ reduced the potential for M3 S. pyogenes to induce cytokines in human tonsil, but not during invasive infection of superantigen-sensitive mice. Notwithstanding epidemiological associations with STSS and disease severity, emm3 strains have inherently poor superantigenicity that is explained by a conserved mutation in smeZ.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">e46376</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23049698" rel="external">23049698</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0046376" rel="external">10.1371/journal.pone.0046376</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A British approach to sampling.</h4>
<p class="authors"><span class="author">Tyler-Smith C</span> and <span class="author">Xue Y</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2012</span>;<span class="volume">20</span>;<span class="number">2</span>;<span class="pagerange">129-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21829226">21829226</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3260911">3260911</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2011.153">10.1038/ejhg.2011.153</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sibling rivalry among paralogs promotes evolution of the human brain.</h4>
<p class="authors"><span class="author">Tyler-Smith C</span> and <span class="author">Xue Y</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%63%74%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#116;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Geneticists have long sought to identify the genetic changes that made us human, but pinpointing the functionally relevant changes has been challenging. Two papers in this issue suggest that partial duplication of SRGAP2, producing an incomplete protein that antagonizes the original, contributed to human brain evolution.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2012</span>;<span class="volume">149</span>;<span class="number">4</span>;<span class="pagerange">737-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22579279">22579279</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3477634">3477634</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2012.04.020">10.1016/j.cell.2012.04.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of diversity by somatic mutation in the Camelus dromedarius T-cell receptor gamma variable domains.</h4>
<p class="authors"><span class="author">Vaccarelli G</span>, <span class="author">Antonacci R</span>, <span class="author">Tasco G</span>, <span class="author">Yang F</span>, <span class="author">Giordano L</span>, <span class="author">El Ashmaoui HM</span>, <span class="author">Hassanane MS</span>, <span class="author">Massari S</span>, <span class="author">Casadio R</span> and <span class="author">Ciccarese S</span></p>
<p class="affiliation">Department of Biology, University of Bari, Bari, Italy.</p><p class="abstract">In jawed vertebrates the V-(D)-J rearrangement is the main mechanism generating limitless variations of antigen-specific receptors, immunoglobulins (IGs), and T-cell receptors (TCRs) from few genes. Once the initial diversity is established in primary lymphoid organs, further diversification occurs in IGs by somatic hypermutation, a mechanism from which rearranged TCR genes were thought to be excluded. Here, we report the locus organization and expression of the T-cell receptor gamma (TCRG) genes in the Arabian camel (Camelus dromedarius). Expression data provide evidence that dromedary utilizes only two TCRG V-J genomic arrangements and, as expected, CDR3 contributes the major variability in the V domain. The data also suggest that diversity might be generated by mutation in the productively rearranged TCRGV genes. As for IG genes, the mutational target is biased toward G and C bases and (A/G/T)G(C/T)(A/T) motif (or DGYW). The replacement and synonymous substitutions (R/S) ratios in TCRGV regions are higher for CDR than for framework region, thus suggesting selection toward amino acid changes in CDR. Using the counterpart human TCR &gamma;&delta; receptor as a template, structural models computed adopting a comparative procedure show that nonconservative mutations contribute to diversity in CDR2 and at the &gamma;&delta; V domain interface.</p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">2012</span>;<span class="volume">42</span>;<span class="number">12</span>;<span class="pagerange">3416-28</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22961631">22961631</a>; DOI: <a href="http://dx.doi.org/10.1002%2Feji.201142176">10.1002/eji.201142176</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Seventy-five genetic loci influencing the human red blood cell.</h4>
<p class="authors"><span class="author">van der Harst P</span>, <span class="author">Zhang W</span>, <span class="author">Mateo Leach I</span>, <span class="author">Rendon A</span>, <span class="author">Verweij N</span>, <span class="author">Sehmi J</span>, <span class="author">Paul DS</span>, <span class="author">Elling U</span>, <span class="author">Allayee H</span>, <span class="author">Li X</span>, <span class="author">Radhakrishnan A</span>, <span class="author">Tan ST</span>, <span class="author">Voss K</span>, <span class="author">Weichenberger CX</span>, <span class="author">Albers CA</span>, <span class="author">Al-Hussani A</span>, <span class="author">Asselbergs FW</span>, <span class="author">Ciullo M</span>, <span class="author">Danjou F</span>, <span class="author">Dina C</span>, <span class="author">Esko T</span>, <span class="author">Evans DM</span>, <span class="author">Franke L</span>, <span class="author">G&ouml;gele M</span>, <span class="author">Hartiala J</span>, <span class="author">Hersch M</span>, <span class="author">Holm H</span>, <span class="author">Hottenga JJ</span>, <span class="author">Kanoni S</span>, <span class="author">Kleber ME</span>, <span class="author">Lagou V</span>, <span class="author">Langenberg C</span>, <span class="author">Lopez LM</span>, <span class="author">Lyytik&auml;inen LP</span>, <span class="author">Melander O</span>, <span class="author">Murgia F</span>, <span class="author">Nolte IM</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Padmanabhan S</span>, <span class="author">Parsa A</span>, <span class="author">Pirastu N</span>, <span class="author">Porcu E</span>, <span class="author">Portas L</span>, <span class="author">Prokopenko I</span>, <span class="author">Ried JS</span>, <span class="author">Shin SY</span>, <span class="author">Tang CS</span>, <span class="author">Teumer A</span>, <span class="author">Traglia M</span>, <span class="author">Ulivi S</span>, <span class="author">Westra HJ</span>, <span class="author">Yang J</span>, <span class="author">Zhao JH</span>, <span class="author">Anni F</span>, <span class="author">Abdellaoui A</span>, <span class="author">Attwood A</span>, <span class="author">Balkau B</span>, <span class="author">Bandinelli S</span>, <span class="author">Bastardot F</span>, <span class="author">Benyamin B</span>, <span class="author">Boehm BO</span>, <span class="author">Cookson WO</span>, <span class="author">Das D</span>, <span class="author">de Bakker PI</span>, <span class="author">de Boer RA</span>, <span class="author">de Geus EJ</span>, <span class="author">de Moor MH</span>, <span class="author">Dimitriou M</span>, <span class="author">Domingues FS</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Engstr&ouml;m G</span>, <span class="author">Eyjolfsson GI</span>, <span class="author">Ferrucci L</span>, <span class="author">Fischer K</span>, <span class="author">Galanello R</span>, <span class="author">Garner SF</span>, <span class="author">Genser B</span>, <span class="author">Gibson QD</span>, <span class="author">Girotto G</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Harris SE</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hastie CE</span>, <span class="author">Hedblad B</span>, <span class="author">Illig T</span>, <span class="author">Jolley J</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kema IP</span>, <span class="author">Kemp JP</span>, <span class="author">Liang L</span>, <span class="author">Lloyd-Jones H</span>, <span class="author">Loos RJ</span>, <span class="author">Meacham S</span>, <span class="author">Medland SE</span>, <span class="author">Meisinger C</span>, <span class="author">Memari Y</span>, <span class="author">Mihailov E</span>, <span class="author">Miller K</span>, <span class="author">Moffatt MF</span>, <span class="author">Nauck M</span>, <span class="author">Novatchkova M</span>, <span class="author">Nutile T</span>, <span class="author">Olafsson I</span>, <span class="author">Onundarson PT</span>, <span class="author">Parracciani D</span>, <span class="author">Penninx BW</span>, <span class="author">Perseu L</span>, <span class="author">Piga A</span>, <span class="author">Pistis G</span>, <span class="author">Pouta A</span>, <span class="author">Puc U</span>, <span class="author">Raitakari O</span>, <span class="author">Ring SM</span>, <span class="author">Robino A</span>, <span class="author">Ruggiero D</span>, <span class="author">Ruokonen A</span>, <span class="author">Saint-Pierre A</span>, <span class="author">Sala C</span>, <span class="author">Salumets A</span>, <span class="author">Sambrook J</span>, <span class="author">Schepers H</span>, <span class="author">Schmidt CO</span>, <span class="author">Sillj&eacute; HH</span>, <span class="author">Sladek R</span>, <span class="author">Smit JH</span>, <span class="author">Starr JM</span>, <span class="author">Stephens J</span>, <span class="author">Sulem P</span>, <span class="author">Tanaka T</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Tragante V</span>, <span class="author">van Gilst WH</span>, <span class="author">van Pelt LJ</span>, <span class="author">van Veldhuisen DJ</span>, <span class="author">V&ouml;lker U</span>, <span class="author">Whitfield JB</span>, <span class="author">Willemsen G</span>, <span class="author">Winkelmann BR</span>, <span class="author">Wirnsberger G</span>, <span class="author">Algra A</span>, <span class="author">Cucca F</span>, <span class="author">d&#39;Adamo AP</span>, <span class="author">Danesh J</span>, <span class="author">Deary IJ</span>, <span class="author">Dominiczak AF</span>, <span class="author">Elliott P</span>, <span class="author">Fortina P</span>, <span class="author">Froguel P</span>, <span class="author">Gasparini P</span>, <span class="author">Greinacher A</span>, <span class="author">Hazen SL</span>, <span class="author">Jarvelin MR</span>, <span class="author">Khaw KT</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Maerz W</span>, <span class="author">Martin NG</span>, <span class="author">Metspalu A</span>, <span class="author">Mitchell BD</span>, <span class="author">Montgomery GW</span>, <span class="author">Moore C</span>, <span class="author">Navis G</span>, <span class="author">Pirastu M</span>, <span class="author">Pramstaller PP</span>, <span class="author">Ramirez-Solis R</span>, <span class="author">Schadt E</span>, <span class="author">Scott J</span>, <span class="author">Shuldiner AR</span>, <span class="author">Smith GD</span>, <span class="author">Smith JG</span>, <span class="author">Snieder H</span>, <span class="author">Sorice R</span>, <span class="author">Spector TD</span>, <span class="author">Stefansson K</span>, <span class="author">Stumvoll M</span>, <span class="author">Tang WH</span>, <span class="author">Toniolo D</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Visscher PM</span>, <span class="author">Vollenweider P</span>, <span class="author">Wareham NJ</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Boomsma DI</span>, <span class="author">Beckmann JS</span>, <span class="author">Dedoussis GV</span>, <span class="author">Deloukas P</span>, <span class="author">Ferreira MA</span>, <span class="author">Sanna S</span>, <span class="author">Uda M</span>, <span class="author">Hicks AA</span>, <span class="author">Penninger JM</span>, <span class="author">Gieger C</span>, <span class="author">Kooner JS</span>, <span class="author">Ouwehand WH</span>, <span class="author">Soranzo N</span> and <span class="author">Chambers JC</span></p>
<p class="affiliation">Department of Cardiology, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, The Netherlands. <a href="mailto:%70.%76%61%6E.%64%65%72.%68%61%72%73%74%40%75%6D%63%67.%6E%6C">&#112;&#46;&#118;&#97;&#110;&#46;&#100;&#101;&#114;&#46;&#104;&#97;&#114;&#115;&#116;&#64;&#117;&#109;&#99;&#103;&#46;&#110;&#108;</a></p><p class="abstract">Anaemia is a chief determinant of global ill health, contributing to cognitive impairment, growth retardation and impaired physical capacity. To understand further the genetic factors influencing red blood cells, we carried out a genome-wide association study of haemoglobin concentration and related parameters in up to 135,367 individuals. Here we identify 75 independent genetic loci associated with one or more red blood cell phenotypes at P&thinsp;&lt;&thinsp;10(-8), which together explain 4-9% of the phenotypic variance per trait. Using expression quantitative trait loci and bioinformatic strategies, we identify 121 candidate genes enriched in functions relevant to red blood cell biology. The candidate genes are expressed preferentially in red blood cell precursors, and 43 have haematopoietic phenotypes in Mus musculus or Drosophila melanogaster. Through open-chromatin and coding-variant analyses we identify potential causal genetic variants at 41 loci. Our findings provide extensive new insights into genetic mechanisms and biological pathways controlling red blood cell formation and function.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Cancer Research UK: <span class="grant">14136</span>; Chief Scientist Office: <span class="grant">CZB/4/505</span>, <span class="grant">ETM/55</span>; Medical Research Council: <span class="grant">G0600705</span>, <span class="grant">G0700704</span>, <span class="grant">G0801056</span>, <span class="grant">G1000143</span>, <span class="grant">G9815508</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NCATS NIH HHS: <span class="grant">UL1 TR000439</span>; NCI NIH HHS: <span class="grant">R01 CA165001</span>; NCRR NIH HHS: <span class="grant">K12 RR023250</span>, <span class="grant">U54 RR020278</span>, <span class="grant">UL1 RR025005</span>; NHGRI NIH HHS: <span class="grant">T32 HG002536</span>, <span class="grant">U01 HG004402</span>; NHLBI NIH HHS: <span class="grant">HHSN268201100005C</span>, <span class="grant">HHSN268201100006C</span>, <span class="grant">HHSN268201100007C</span>, <span class="grant">HHSN268201100008C</span>, <span class="grant">HHSN268201100009C</span>, <span class="grant">HHSN268201100010C</span>, <span class="grant">HHSN268201100011C</span>, <span class="grant">HHSN268201100012C</span>, <span class="grant">P01 HL076491</span>, <span class="grant">P01 HL098055</span>, <span class="grant">P20 HL113452</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087679</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL103866</span>, <span class="grant">R01 HL103931</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL084756</span>; NIA NIH HHS: <span class="grant">N01AG12109</span>, <span class="grant">R01 AG018728</span>; NICHD NIH HHS: <span class="grant">R01 HD042157</span>; NIDA NIH HHS: <span class="grant">HHSN271201100005C</span>; NIDDK NIH HHS: <span class="grant">P30 DK072488</span>; NIGMS NIH HHS: <span class="grant">R01 GM053275</span>, <span class="grant">U01 GM074518</span>; NIMH NIH HHS: <span class="grant">R01 MH081802</span>, <span class="grant">RL1 MH083268</span>, <span class="grant">U24 MH068457</span>; NLM NIH HHS: <span class="grant">R01 LM010098</span>; Wellcome Trust: <span class="grant">092731</span>, <span class="grant">097117</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">492</span>;<span class="number">7429</span>;<span class="pagerange">369-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23222517">23222517</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3623669">3623669</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature11677">10.1038/nature11677</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Increased tumorigenesis associated with loss of the tumor suppressor gene Cadm1.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Arends MJ</span>, <span class="author">Rust AG</span>, <span class="author">Poulogiannis G</span>, <span class="author">McIntyre RE</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, The Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1HH, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: CADM1 encodes an immunoglobulin superfamily (IGSF) cell adhesion molecule. Inactivation of CADM1, either by promoter hypermethylation or loss of heterozygosity, has been reported in a wide variety of tumor types, thus it has been postulated as a tumor suppressor gene.</p><p class="abstract"><strong>Findings</strong>: We show for the first time that Cadm1 homozygous null mice die significantly faster than wildtype controls due to the spontaneous development of tumors at an earlier age and an increased tumor incidence of predominantly lymphomas, but also some solid tumors. Tumorigenesis was accelerated after irradiation of Cadm1 mice, with the reduced latency in tumor formation suggesting there are genes that collaborate with loss of Cadm1 in tumorigenesis. To identify these co-operating genetic events, we performed a Sleeping Beauty transposon-mediated insertional mutagenesis screen in Cadm1 mice, and identified several common insertion sites (CIS) found specifically on a Cadm1-null background (and not wildtype background).</p><p class="abstract"><strong>Conclusion</strong>: We confirm that Cadm1 is indeed a bona fide tumor suppressor gene and provide new insights into genetic partners that co-operate in tumorigenesis when Cadm1-expression is lost.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span></p>
<p class="publication"><span class="title">Molecular cancer</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="pagerange">29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22553910">22553910</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3489691">3489691</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1476-4598-11-29">10.1186/1476-4598-11-29</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss of RASSF1A synergizes with deregulated RUNX2 signaling in tumorigenesis.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Papaspyropoulos A</span>, <span class="author">Poulogiannis G</span>, <span class="author">Rust AG</span>, <span class="author">Rashid M</span>, <span class="author">Adams DJ</span>, <span class="author">Arends MJ</span> and <span class="author">O&#39;Neill E</span></p>
<p class="affiliation">Experimental Cancer Genetics, The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The tumor suppressor gene RASSF1A is inactivated through point mutation or promoter hypermethylation in many human cancers. In this study, we conducted a Sleeping Beauty transposon-mediated insertional mutagenesis screen in Rassf1a-null mice to identify candidate genes that collaborate with loss of Rassf1a in tumorigenesis. We identified 10 genes, including the transcription factor Runx2, a transcriptional partner of Yes-associated protein (YAP1) that displays tumor suppressive activity through competing with the oncogenic TEA domain family of transcription factors (TEAD) for YAP1 association. While loss of RASSF1A promoted the formation of oncogenic YAP1-TEAD complexes, the combined loss of both RASSF1A and RUNX2 further increased YAP1-TEAD levels, showing that loss of RASSF1A, together with RUNX2, is consistent with the multistep model of tumorigenesis. Clinically, RUNX2 expression was frequently downregulated in various cancers, and reduced RUNX2 expression was associated with poor survival in patients with diffuse large B-cell or atypical Burkitt/Burkitt-like lymphomas. Interestingly, decreased expression levels of RASSF1 and RUNX2 were observed in both precursor T-cell acute lymphoblastic leukemia and colorectal cancer, further supporting the hypothesis that dual regulation of YAP1-TEAD promotes oncogenic activity. Together, our findings provide evidence that loss of RASSF1A expression switches YAP1 from a tumor suppressor to an oncogene through regulating its association with transcription factors, thereby suggesting a novel mechanism for RASSF1A-mediated tumor suppression.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>, <span class="grant">A6997</span>, <span class="grant">A9318</span>, <span class="grant">AI2932</span>, <span class="grant">C20510/A6997</span>; Medical Research Council: <span class="grant">985361</span>; Wellcome Trust: <span class="grant">082356</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2012</span>;<span class="volume">72</span>;<span class="number">15</span>;<span class="pagerange">3817-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22710434">22710434</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4067151">4067151</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-11-3343">10.1158/0008-5472.CAN-11-3343</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ABSOLUTE cancer genomics.</h4>
<p class="authors"><span class="author">Van Loo P</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, UK, the Center for the Biology of Disease, VIB, Leuven, Belgium, and the Department of Human Genetics, KU Leuven, Leuven, Belgium.</p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2012</span>;<span class="volume">30</span>;<span class="number">7</span>;<span class="pagerange">620-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22781683" rel="external">22781683</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.2293" rel="external">10.1038/nbt.2293</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic markers for SSG resistance in Leishmania donovani and SSG treatment failure in visceral leishmaniasis patients of the Indian subcontinent.</h4>
<p class="authors"><span class="author">Vanaerschot M</span>, <span class="author">Decuypere S</span>, <span class="author">Downing T</span>, <span class="author">Imamura H</span>, <span class="author">Stark O</span>, <span class="author">De Doncker S</span>, <span class="author">Roy S</span>, <span class="author">Ostyn B</span>, <span class="author">Maes L</span>, <span class="author">Khanal B</span>, <span class="author">Boelaert M</span>, <span class="author">Sch&ouml;nian G</span>, <span class="author">Berriman M</span>, <span class="author">Chappuis F</span>, <span class="author">Dujardin JC</span>, <span class="author">Sundar S</span> and <span class="author">Rijal S</span></p>
<p class="affiliation">Department of Biomedical Sciences, Institute of Tropical Medicine Antwerp, Belgium.</p><p class="abstract">The current standard to assess pentavalent antimonial (SSG) susceptibility of Leishmania is a laborious in vitro assay of which the result has little clinical value because SSG-resistant parasites are also found in SSG-cured patients. Candidate genetic markers for clinically relevant SSG-resistant parasites identified by full genome sequencing were here validated on a larger set of clinical strains. We show that 3 genomic locations suffice to specifically detect the SSG-resistant parasites found only in patients experiencing SSG treatment failure. This finding allows the development of rapid assays to monitor the emergence and spread of clinically relevant SSG-resistant Leishmania parasites.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076355</span>, <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2012</span>;<span class="volume">206</span>;<span class="number">5</span>;<span class="pagerange">752-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22753945">22753945</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4125624">4125624</a>; DOI: <a href="http://dx.doi.org/10.1093%2Finfdis%2Fjis424">10.1093/infdis/jis424</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using CF11 cellulose columns to inexpensively and effectively remove human DNA from Plasmodium falciparum-infected whole blood samples.</h4>
<p class="authors"><span class="author">Venkatesan M</span>, <span class="author">Amaratunga C</span>, <span class="author">Campino S</span>, <span class="author">Auburn S</span>, <span class="author">Koch O</span>, <span class="author">Lim P</span>, <span class="author">Uk S</span>, <span class="author">Socheat D</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Fairhurst RM</span> and <span class="author">Plowe CV</span></p>
<p class="affiliation">Howard Hughes Medical Institute, University of Maryland School of Medicine, Baltimore, MD, USA.</p><p class="abstract"><strong>Background</strong>: Genome and transcriptome studies of Plasmodium nucleic acids obtained from parasitized whole blood are greatly improved by depletion of human DNA or enrichment of parasite DNA prior to next-generation sequencing and microarray hybridization. The most effective method currently used is a two-step procedure to deplete leukocytes: centrifugation using density gradient media followed by filtration through expensive, commercially available columns. This method is not easily implemented in field studies that collect hundreds of samples and simultaneously process samples for multiple laboratory analyses. Inexpensive syringes, hand-packed with CF11 cellulose powder, were recently shown to improve ex vivo cultivation of Plasmodium vivax obtained from parasitized whole blood. This study was undertaken to determine whether CF11 columns could be adapted to isolate Plasmodium falciparum DNA from parasitized whole blood and achieve current quantity and purity requirements for Illumina sequencing.</p><p class="abstract"><strong>Methods</strong>: The CF11 procedure was compared with the current two-step standard of leukocyte depletion using parasitized red blood cells cultured in vitro and parasitized blood obtained ex vivo from Cambodian patients with malaria. Procedural variations in centrifugation and column size were tested, along with a range of blood volumes and parasite densities.</p><p class="abstract"><strong>Results</strong>: CF11 filtration reliably produces 500 nanograms of DNA with less than 50% human DNA contamination, which is comparable to that obtained by the two-step method and falls within the current quality control requirements for Illumina sequencing. In addition, a centrifuge-free version of the CF11 filtration method to isolate P. falciparum DNA at remote and minimally equipped field sites in malaria-endemic areas was validated.</p><p class="abstract"><strong>Conclusions</strong>: CF11 filtration is a cost-effective, scalable, one-step approach to remove human DNA from P. falciparum-infected whole blood samples.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">089275</span>, <span class="grant">090532</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="pagerange">41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22321373">22321373</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3295709">3295709</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-11-41">10.1186/1475-2875-11-41</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MAINTENANCE OF GENETIC VARIATION IN HUMAN PERSONALITY: TESTING EVOLUTIONARY MODELS BY ESTIMATING HERITABILITY DUE TO COMMON CAUSAL VARIANTS AND INVESTIGATING THE EFFECT OF DISTANT INBREEDING.</h4>
<p class="authors"><span class="author">Verweij KJ</span>, <span class="author">Yang J</span>, <span class="author">Lahti J</span>, <span class="author">Veijola J</span>, <span class="author">Hintsanen M</span>, <span class="author">Pulkki-R&aring;back L</span>, <span class="author">Heinonen K</span>, <span class="author">Pouta A</span>, <span class="author">Pesonen AK</span>, <span class="author">Widen E</span>, <span class="author">Taanila A</span>, <span class="author">Isohanni M</span>, <span class="author">Miettunen J</span>, <span class="author">Palotie A</span>, <span class="author">Penke L</span>, <span class="author">Service SK</span>, <span class="author">Heath AC</span>, <span class="author">Montgomery GW</span>, <span class="author">Raitakari O</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Viikari J</span>, <span class="author">R&auml;ikk&ouml;nen K</span>, <span class="author">Eriksson JG</span>, <span class="author">Keltikangas-J&auml;rvinen L</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Martin NG</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Visscher PM</span>, <span class="author">Keller MC</span> and <span class="author">Zietsch BP</span></p>
<p class="affiliation">Genetic Epidemiology, Molecular Epidemiology, and Queensland Statistical Genetics Laboratories, Queensland Institute of Medical Research, Herston 4006, Brisbane, Queensland, Australia School of Psychology, University of Queensland, St Lucia 4067, Brisbane, Queensland, Australia These authors contributed equally Institute of Behavioural Sciences, University of Helsinki, Helsinki 00014, Finland Institute of Clinical Medicine, University of Oulu, Oulu 90014, Finland Institute of Behavioural Sciences / Psychology, University of Helsinki, Helsinki 00014, Finland National Institute for Health and Welfare, Oulu 90101, Finland Institute for Molecular Medicine Finland, University of Helsinki, Helsinki 00014, Finland Institute of Health Sciences, University of Oulu, Oulu 90014, Finland Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10, 1SA, United Kingdom Department of Medical Genetics, University of Helsinki and University Central Hospital, Helsinki 00014, Finland Department of Psychology, and Centre for Cognitive Ageing and Cognitive Epidemiology, University of Edinburgh, Edinburgh EH8 9JZ, United Kingdom University of California Los Angeles, Semel Institute of Neuroscience and Human Behavior, Los Angeles, California 90095 Department of Psychiatry, Washington University School of Medicine, St Louis 63130, Missouri Department of Clinical Physiology, Turku University Hospital and Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Turku 20521, Finland Department of Clinical Physiology, Tampere University Hospital and University of Tampere School of Medicine, Tampere 33014, Finland Department of Medicine, Turku University Hospital and University of Turku, Turku 20521, Finland Department of General Practice and Primary health Care, University of Helsinki, Helsinki 00014, Finland Helsinki University Central Hospital, Unit of General Practice, Helsinki 00029, Finland Folkhälsan Research Centre, Helsinki 00014, Finland Vasa Central Hospital, Vasa 65130, Finland Department of Clinical Chemistry, Tampere University Hospital and University of Tampere School of Medicine, Tampere 33014, Finland Department of Epidemiology and Biostatistics, School of Public Health, MRC-HPA Centre for Environment and Health, Faculty of Medicine, Imperial College, London W2 1PG, United Kingdom Biocenter Oulu, University of Oulu, Oulu 90014, Finland Department of Lifecourse and Services, National Institute for Health and Welfare, Oulu 90101, Finland Department of Psychology and Neuroscience, University of Colorado, Boulder 80309, Colorado Institute for Behavioral Genetics, University of Colorado, Boulder 80303, Colorado E-mail: <a href="mailto:%42%72%65%6E%64%61%6E.%5A%69%65%74%73%63%68%40%71%69%6D%72.%65%64%75.%61%75">&#66;&#114;&#101;&#110;&#100;&#97;&#110;&#46;&#90;&#105;&#101;&#116;&#115;&#99;&#104;&#64;&#113;&#105;&#109;&#114;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a>.</p>
<p class="abstract">Personality traits are basic dimensions of behavioral variation, and twin, family, and adoption studies show that around 30% of the between-individual variation is due to genetic variation. There is rapidly growing interest in understanding the evolutionary basis of this genetic variation. Several evolutionary mechanisms could explain how genetic variation is maintained in traits, and each of these makes predictions in terms of the relative contribution of rare and common genetic variants to personality variation, the magnitude of nonadditive genetic influences, and whether personality is affected by inbreeding. Using genome-wide single nucleotide polymorphism (SNP) data from > 8000 individuals, we estimated that little variation in the Cloninger personality dimensions (7.2% on average) is due to the combined effect of common, additive genetic variants across the genome, suggesting that most heritable variation in personality is due to rare variant effects and/or a combination of dominance and epistasis. Furthermore, higher levels of inbreeding were associated with less socially desirable personality trait levels in three of the four personality dimensions. These findings are consistent with genetic variation in personality traits having been maintained by mutation-selection balance.</p>
<p class="publication"><span class="title">Evolution; international journal of organic evolution</span>&nbsp;<span class="year">2012</span>;<span class="volume">66</span>;<span class="number">10</span>;<span class="pagerange">3238-3251</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23025612" rel="external">23025612</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1558-5646.2012.01679.x" rel="external">10.1111/j.1558-5646.2012.01679.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plasma HDL cholesterol and risk of myocardial infarction: a mendelian randomisation study.</h4>
<p class="authors"><span class="author">Voight BF</span>, <span class="author">Peloso GM</span>, <span class="author">Orho-Melander M</span>, <span class="author">Frikke-Schmidt R</span>, <span class="author">Barbalic M</span>, <span class="author">Jensen MK</span>, <span class="author">Hindy G</span>, <span class="author">H&oacute;lm H</span>, <span class="author">Ding EL</span>, <span class="author">Johnson T</span>, <span class="author">Schunkert H</span>, <span class="author">Samani NJ</span>, <span class="author">Clarke R</span>, <span class="author">Hopewell JC</span>, <span class="author">Thompson JF</span>, <span class="author">Li M</span>, <span class="author">Thorleifsson G</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Musunuru K</span>, <span class="author">Pirruccello JP</span>, <span class="author">Saleheen D</span>, <span class="author">Chen L</span>, <span class="author">Stewart A</span>, <span class="author">Schillert A</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Thorgeirsson G</span>, <span class="author">Anand S</span>, <span class="author">Engert JC</span>, <span class="author">Morgan T</span>, <span class="author">Spertus J</span>, <span class="author">Stoll M</span>, <span class="author">Berger K</span>, <span class="author">Martinelli N</span>, <span class="author">Girelli D</span>, <span class="author">McKeown PP</span>, <span class="author">Patterson CC</span>, <span class="author">Epstein SE</span>, <span class="author">Devaney J</span>, <span class="author">Burnett MS</span>, <span class="author">Mooser V</span>, <span class="author">Ripatti S</span>, <span class="author">Surakka I</span>, <span class="author">Nieminen MS</span>, <span class="author">Sinisalo J</span>, <span class="author">Lokki ML</span>, <span class="author">Perola M</span>, <span class="author">Havulinna A</span>, <span class="author">de Faire U</span>, <span class="author">Gigante B</span>, <span class="author">Ingelsson E</span>, <span class="author">Zeller T</span>, <span class="author">Wild P</span>, <span class="author">de Bakker PI</span>, <span class="author">Klungel OH</span>, <span class="author">Maitland-van der Zee AH</span>, <span class="author">Peters BJ</span>, <span class="author">de Boer A</span>, <span class="author">Grobbee DE</span>, <span class="author">Kamphuisen PW</span>, <span class="author">Deneer VH</span>, <span class="author">Elbers CC</span>, <span class="author">Onland-Moret NC</span>, <span class="author">Hofker MH</span>, <span class="author">Wijmenga C</span>, <span class="author">Verschuren WM</span>, <span class="author">Boer JM</span>, <span class="author">van der Schouw YT</span>, <span class="author">Rasheed A</span>, <span class="author">Frossard P</span>, <span class="author">Demissie S</span>, <span class="author">Willer C</span>, <span class="author">Do R</span>, <span class="author">Ordovas JM</span>, <span class="author">Abecasis GR</span>, <span class="author">Boehnke M</span>, <span class="author">Mohlke KL</span>, <span class="author">Daly MJ</span>, <span class="author">Guiducci C</span>, <span class="author">Burtt NP</span>, <span class="author">Surti A</span>, <span class="author">Gonzalez E</span>, <span class="author">Purcell S</span>, <span class="author">Gabriel S</span>, <span class="author">Marrugat J</span>, <span class="author">Peden J</span>, <span class="author">Erdmann J</span>, <span class="author">Diemert P</span>, <span class="author">Willenborg C</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Fischer M</span>, <span class="author">Hengstenberg C</span>, <span class="author">Ziegler A</span>, <span class="author">Buysschaert I</span>, <span class="author">Lambrechts D</span>, <span class="author">Van de Werf F</span>, <span class="author">Fox KA</span>, <span class="author">El Mokhtari NE</span>, <span class="author">Rubin D</span>, <span class="author">Schrezenmeir J</span>, <span class="author">Schreiber S</span>, <span class="author">Sch&auml;fer A</span>, <span class="author">Danesh J</span>, <span class="author">Blankenberg S</span>, <span class="author">Roberts R</span>, <span class="author">McPherson R</span>, <span class="author">Watkins H</span>, <span class="author">Hall AS</span>, <span class="author">Overvad K</span>, <span class="author">Rimm E</span>, <span class="author">Boerwinkle E</span>, <span class="author">Tybjaerg-Hansen A</span>, <span class="author">Cupples LA</span>, <span class="author">Reilly MP</span>, <span class="author">Melander O</span>, <span class="author">Mannucci PM</span>, <span class="author">Ardissino D</span>, <span class="author">Siscovick D</span>, <span class="author">Elosua R</span>, <span class="author">Stefansson K</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Salomaa V</span>, <span class="author">Rader DJ</span>, <span class="author">Peltonen L</span>, <span class="author">Schwartz SM</span>, <span class="author">Altshuler D</span> and <span class="author">Kathiresan S</span></p>
<p class="affiliation">Department of Pharmacology, University of Pennsylvania, Philadelphia, PA, USA.</p><p class="abstract"><strong>Background</strong>: High plasma HDL cholesterol is associated with reduced risk of myocardial infarction, but whether this association is causal is unclear. Exploiting the fact that genotypes are randomly assigned at meiosis, are independent of non-genetic confounding, and are unmodified by disease processes, mendelian randomisation can be used to test the hypothesis that the association of a plasma biomarker with disease is causal.</p><p class="abstract"><strong>Methods</strong>: We performed two mendelian randomisation analyses. First, we used as an instrument a single nucleotide polymorphism (SNP) in the endothelial lipase gene (LIPG Asn396Ser) and tested this SNP in 20 studies (20,913 myocardial infarction cases, 95,407 controls). Second, we used as an instrument a genetic score consisting of 14 common SNPs that exclusively associate with HDL cholesterol and tested this score in up to 12,482 cases of myocardial infarction and 41,331 controls. As a positive control, we also tested a genetic score of 13 common SNPs exclusively associated with LDL cholesterol.</p><p class="abstract"><strong>Findings</strong>: Carriers of the LIPG 396Ser allele (2&middot;6% frequency) had higher HDL cholesterol (0&middot;14 mmol/L higher, p=8&times;10(-13)) but similar levels of other lipid and non-lipid risk factors for myocardial infarction compared with non-carriers. This difference in HDL cholesterol is expected to decrease risk of myocardial infarction by 13% (odds ratio [OR] 0&middot;87, 95% CI 0&middot;84-0&middot;91). However, we noted that the 396Ser allele was not associated with risk of myocardial infarction (OR 0&middot;99, 95% CI 0&middot;88-1&middot;11, p=0&middot;85). From observational epidemiology, an increase of 1 SD in HDL cholesterol was associated with reduced risk of myocardial infarction (OR 0&middot;62, 95% CI 0&middot;58-0&middot;66). However, a 1 SD increase in HDL cholesterol due to genetic score was not associated with risk of myocardial infarction (OR 0&middot;93, 95% CI 0&middot;68-1&middot;26, p=0&middot;63). For LDL cholesterol, the estimate from observational epidemiology (a 1 SD increase in LDL cholesterol associated with OR 1&middot;54, 95% CI 1&middot;45-1&middot;63) was concordant with that from genetic score (OR 2&middot;13, 95% CI 1&middot;69-2&middot;69, p=2&times;10(-10)).</p><p class="abstract"><strong>Interpretation</strong>: Some genetic mechanisms that raise plasma HDL cholesterol do not seem to lower risk of myocardial infarction. These data challenge the concept that raising of plasma HDL cholesterol will uniformly translate into reductions in risk of myocardial infarction.</p><p class="abstract"><strong>Funding</strong>: US National Institutes of Health, The Wellcome Trust, European Union, British Heart Foundation, and the German Federal Ministry of Education and Research.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/08/014/24067</span>; Medical Research Council: <span class="grant">MC_U137686857</span>; NCRR NIH HHS: <span class="grant">UL1 RR024148</span>; NHLBI NIH HHS: <span class="grant">N01 HC025195</span>, <span class="grant">R00 HL094535</span>; NIDDK NIH HHS: <span class="grant">R01 DK072193</span>; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2012</span>;<span class="volume">380</span>;<span class="number">9841</span>;<span class="pagerange">572-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22607825">22607825</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3419820">3419820</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2812%2960312-2">10.1016/S0140-6736(12)60312-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The alveolin IMC1h is required for normal ookinete and sporozoite motility behaviour and host colonisation in Plasmodium berghei.</h4>
<p class="authors"><span class="author">Volkmann K</span>, <span class="author">Pfander C</span>, <span class="author">Burstroem C</span>, <span class="author">Ahras M</span>, <span class="author">Goulding D</span>, <span class="author">Rayner JC</span>, <span class="author">Frischknecht F</span>, <span class="author">Billker O</span> and <span class="author">Brochet M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">Alveolins, or inner membrane complex (IMC) proteins, are components of the subpellicular network that forms a structural part of the pellicle of malaria parasites. In Plasmodium berghei, deletions of three alveolins, IMC1a, b, and h, each resulted in reduced mechanical strength and gliding velocity of ookinetes or sporozoites. Using time lapse imaging, we show here that deletion of IMC1h (PBANKA_143660) also has an impact on the directionality and motility behaviour of both ookinetes and sporozoites. Despite their marked motility defects, sporozoites lacking IMC1h were able to invade mosquito salivary glands, allowing us to investigate the role of IMC1h in colonisation of the mammalian host. We show that IMC1h is essential for sporozoites to progress through the dermis in vivo but does not play a significant role in hepatoma cell transmigration and invasion in vitro. Colocalisation of IMC1h with the residual IMC in liver stages was detected up to 30 hours after infection and parasites lacking IMC1h showed developmental defects in vitro and a delayed onset of blood stage infection in vivo. Together, these results suggest that IMC1h is involved in maintaining the cellular architecture which supports normal motility behaviour, access of the sporozoites to the blood stream, and further colonisation of the mammalian host.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>; Wellcome Trust: <span class="grant">89085/Z/09/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e41409</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22844474">22844474</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3402405">3402405</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0041409">10.1371/journal.pone.0041409</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Unveiling malaria's "cloak of invisibility": The role of PfSET10 in the genetic control of P. falciparum virulence.</h4>
<p class="authors"><span class="author">Volz JC</span> and <span class="author">Cowman AF</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute; Cambridge, UK.</p>
<p class="publication"><span class="title">Virulence</span>&nbsp;<span class="year">2012</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">449-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23076248" rel="external">23076248</a>; DOI: <a href="http://dx.doi.org/10.4161%2Fviru.20994" rel="external">10.4161/viru.20994</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metabolic signatures of insulin resistance in 7,098 young adults.</h4>
<p class="authors"><span class="author">W&uuml;rtz P</span>, <span class="author">M&auml;kinen VP</span>, <span class="author">Soininen P</span>, <span class="author">Kangas AJ</span>, <span class="author">Tukiainen T</span>, <span class="author">Kettunen J</span>, <span class="author">Savolainen MJ</span>, <span class="author">Tammelin T</span>, <span class="author">Viikari JS</span>, <span class="author">R&ouml;nnemaa T</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Ripatti S</span>, <span class="author">Raitakari OT</span>, <span class="author">J&auml;rvelin MR</span> and <span class="author">Ala-Korpela M</span></p>
<p class="affiliation">Corresponding authors: Peter W&uuml;rtz, <a href="mailto:%70%65%74%65%72.%77%79%72%74%7A%40%63%6F%6D%70%75%74%61%74%69%6F%6E%61%6C%6D%65%64%69%63%69%6E%65.%66%69">&#112;&#101;&#116;&#101;&#114;&#46;&#119;&#121;&#114;&#116;&#122;&#64;&#99;&#111;&#109;&#112;&#117;&#116;&#97;&#116;&#105;&#111;&#110;&#97;&#108;&#109;&#101;&#100;&#105;&#99;&#105;&#110;&#101;&#46;&#102;&#105;</a>.</p><p class="abstract">Metabolite associations with insulin resistance were studied in 7,098 young Finns (age 31 &plusmn; 3 years; 52% women) to elucidate underlying metabolic pathways. Insulin resistance was assessed by the homeostasis model (HOMA-IR) and circulating metabolites quantified by high-throughput nuclear magnetic resonance spectroscopy in two population-based cohorts. Associations were analyzed using regression models adjusted for age, waist, and standard lipids. Branched-chain and aromatic amino acids, gluconeogenesis intermediates, ketone bodies, and fatty acid composition and saturation were associated with HOMA-IR (P &lt; 0.0005 for 20 metabolite measures). Leu, Ile, Val, and Tyr displayed sex- and obesity-dependent interactions, with associations being significant for women only if they were abdominally obese. Origins of fasting metabolite levels were studied with dietary and physical activity data. Here, protein energy intake was associated with Val, Phe, Tyr, and Gln but not insulin resistance index. We further tested if 12 genetic variants regulating the metabolites also contributed to insulin resistance. The genetic determinants of metabolite levels were not associated with HOMA-IR, with the exception of a variant in GCKR associated with 12 metabolites, including amino acids (P &lt; 0.0005). Nonetheless, metabolic signatures extending beyond obesity and lipid abnormalities reflected the degree of insulin resistance evidenced in young, normoglycemic adults with sex-specific fingerprints.</p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2012</span>;<span class="volume">61</span>;<span class="number">6</span>;<span class="pagerange">1372-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22511205" rel="external">22511205</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3357275" rel="external">3357275</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb11-1355" rel="external">10.2337/db11-1355</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Welcome to the plasmidome.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">This month&#39;s Genome Watch reviews a recent paper that used a metagenomics approach to characterize the plasmid content of bovine rumen samples.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span>;<span class="volume">10</span>;<span class="number">6</span>;<span class="pagerange">379</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22580363" rel="external">22580363</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2804" rel="external">10.1038/nrmicro2804</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Statistical colocalization of monocyte gene expression and genetic risk variants for type 1 diabetes.</h4>
<p class="authors"><span class="author">Wallace C</span>, <span class="author">Rotival M</span>, <span class="author">Cooper JD</span>, <span class="author">Rice CM</span>, <span class="author">Yang JH</span>, <span class="author">McNeill M</span>, <span class="author">Smyth DJ</span>, <span class="author">Niblett D</span>, <span class="author">Cambien F</span>, <span class="author">The Cardiogenics Consortium</span>, <span class="author">Tiret L</span>, <span class="author">Todd JA</span>, <span class="author">Clayton DG</span> and <span class="author">Blankenberg S</span></p>
<p class="affiliation">These authors contributed equally.</p><p class="abstract">One mechanism by which disease-associated DNA variation can alter disease risk is altering gene expression. However, linkage disequilibrium (LD) between variants, mostly single-nucleotide polymorphisms (SNPs), means it is not sufficient to show that a particular variant associates with both disease and expression, as there could be two distinct causal variants in LD. Here, we describe a formal statistical test of colocalization and apply it to type 1 diabetes (T1D)-associated regions identified mostly through genome-wide association studies and expression quantitative trait loci (eQTLs) discovered in a recently determined large monocyte expression data set from the Gutenberg Health Study (1370 individuals), with confirmation sought in an additional data set from the Cardiogenics Transcriptome Study (558 individuals). We excluded 39 out of 60 overlapping eQTLs in 49 T1D regions from possible colocalization and identified 21 coincident eQTLs, representing 21 genes in 14 distinct T1D regions. Our results reflect the importance of monocyte (and their derivatives, macrophage and dendritic cell) gene expression in human T1D and support the candidacy of several genes as causal factors in autoimmune pancreatic beta-cell destruction, including AFF3, CD226, CLECL1, DEXI, FKRP, PRKD2, RNLS, SMARCE1 and SUOX, in addition to the recently described GPR183 (EBI2) gene.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2815-2824</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22403184" rel="external">22403184</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3363338" rel="external">3363338</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fdds098" rel="external">10.1093/hmg/dds098</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Silencing of Germline-Expressed Genes by DNA Elimination in Somatic Cells.</h4>
<p class="authors"><span class="author">Wang J</span>, <span class="author">Mitreva M</span>, <span class="author">Berriman M</span>, <span class="author">Thorne A</span>, <span class="author">Magrini V</span>, <span class="author">Koutsovoulos G</span>, <span class="author">Kumar S</span>, <span class="author">Blaxter ML</span> and <span class="author">Davis RE</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045, USA.</p>
<p class="abstract">Chromatin diminution is the programmed elimination of specific DNA sequences during development. ItÂ occurs in diverse species, but the function(s) of diminution and the specificity of sequence loss remain largely unknown. Diminution in the nematode Ascaris suum occurs during early embryonic cleavages and leads to the loss of germline genome sequences and the formation of a distinct genome in somatic cells. We found that âˆ¼43 Mb (âˆ¼13%) of genome sequence is eliminated in A.Â suum somatic cells, including âˆ¼12.7 Mb of unique sequence. The eliminated sequences and location of the DNA breaks are the same in all somatic lineages from a single individual and between different individuals. At least 685 genes are eliminated. These genes areÂ preferentially expressed in the germline and during early embryogenesis. We propose that diminution is a mechanism of germline gene regulation that specifically removes a large number of genes involved in gametogenesis and early embryogenesis.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI049558</span>, <span class="grant">R21 AI078087</span>, <span class="grant">R21 AI098421</span></p>
<p class="publication"><span class="title">Developmental cell</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23123092" rel="external">23123092</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.devcel.2012.09.020" rel="external">10.1016/j.devcel.2012.09.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Permissive and restricted virus infection of murine embryonic stem cells.</h4>
<p class="authors"><span class="author">Wash R</span>, <span class="author">Calabressi S</span>, <span class="author">Franz S</span>, <span class="author">Griffiths SJ</span>, <span class="author">Goulding D</span>, <span class="author">Tan EP</span>, <span class="author">Wise H</span>, <span class="author">Digard P</span>, <span class="author">Haas J</span>, <span class="author">Efstathiou S</span> and <span class="author">Kellam P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">Recent RNA interference (RNAi) studies have identified many host proteins that modulate virus infection, but small interfering RNA &#39;off-target&#39; effects and the use of transformed cell lines limit their conclusiveness. As murine embryonic stem (mES) cells can be genetically modified and resources exist where many and eventually all known mouse genes are insertionally inactivated, it was reasoned that mES cells would provide a useful alternative to RNAi screens. Beyond allowing investigation of host-pathogen interactions in vitro, mES cells have the potential to differentiate into other primary cell types, as well as being used to generate knockout mice for in vivo studies. However, mES cells are poorly characterized for virus infection. To investigate whether ES cells can be used to explore host-virus interactions, this study characterized the responses of mES cells following infection by herpes simplex virus type 1 (HSV-1) and influenza A virus. HSV-1 replicated lytically in mES cells, although mES cells were less permissive than most other cell types tested. Influenza virus was able to enter mES cells and express some viral proteins, but the replication cycle was incomplete and no infectious virus was produced. Knockdown of the host protein AHCYL1 in mES cells reduced HSV-1 replication, showing the potential for using mES cells to study host-virus interactions. Transcriptional profiling, however, indicated the lack of an efficient innate immune response in these cells. mES cells may thus be useful to identify host proteins that play a role in virus replication, but they are not suitable to determine factors that are involved in innate host defence.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G9800943</span>, <span class="grant">MR/J002232/1</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of general virology</span>&nbsp;<span class="year">2012</span>;<span class="volume">93</span>;<span class="number">Pt 10</span>;<span class="pagerange">2118-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22815272">22815272</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3541792">3541792</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fvir.0.043406-0">10.1099/vir.0.043406-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bayesian refinement of association signals for 14 loci in 3 common diseases.</h4>
<p class="authors"><span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Maller JB</span>, <span class="author">McVean G</span>, <span class="author">Byrnes J</span>, <span class="author">Vukcevic D</span>, <span class="author">Palin K</span>, <span class="author">Su Z</span>, <span class="author">Howson JM</span>, <span class="author">Auton A</span>, <span class="author">Myers S</span>, <span class="author">Morris A</span>, <span class="author">Pirinen M</span>, <span class="author">Brown MA</span>, <span class="author">Burton PR</span>, <span class="author">Caulfield MJ</span>, <span class="author">Compston A</span>, <span class="author">Farrall M</span>, <span class="author">Hall AS</span>, <span class="author">Hattersley AT</span>, <span class="author">Hill AV</span>, <span class="author">Mathew CG</span>, <span class="author">Pembrey M</span>, <span class="author">Satsangi J</span>, <span class="author">Stratton MR</span>, <span class="author">Worthington J</span>, <span class="author">Craddock N</span>, <span class="author">Hurles M</span>, <span class="author">Ouwehand W</span>, <span class="author">Parkes M</span>, <span class="author">Rahman N</span>, <span class="author">Duncanson A</span>, <span class="author">Todd JA</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Samani NJ</span>, <span class="author">Gough SC</span>, <span class="author">McCarthy MI</span>, <span class="author">Deloukas P</span> and <span class="author">Donnelly P</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract">To further investigate susceptibility loci identified by genome-wide association studies, we genotyped 5,500 SNPs across 14 associated regions in 8,000 samples from a control group and 3 diseases: type 2 diabetes (T2D), coronary artery disease (CAD) and Graves&#39; disease. We defined, using Bayes theorem, credible sets of SNPs that were 95% likely, based on posterior probability, to contain the causal disease-associated SNPs. In 3 of the 14 regions, TCF7L2 (T2D), CTLA4 (Graves&#39; disease) and CDKN2A-CDKN2B (T2D), much of the posterior probability rested on a single SNP, and, in 4 other regions (CDKN2A-CDKN2B (CAD) and CDKAL1, FTO and HHEX (T2D)), the 95% sets were small, thereby excluding most SNPs as potentially causal. Very few SNPs in our credible sets had annotated functions, illustrating the limitations in understanding the mechanisms underlying susceptibility to common diseases. Our results also show the value of more detailed mapping to target sequences for functional studies.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">091157</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">12</span>;<span class="pagerange">1294-301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23104008">23104008</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3791416">3791416</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2435">10.1038/ng.2435</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Temperament clusters in a normal population: implications for health and disease.</h4>
<p class="authors"><span class="author">Wessman J</span>, <span class="author">Sch&ouml;nauer S</span>, <span class="author">Miettunen J</span>, <span class="author">Turunen H</span>, <span class="author">Parviainen P</span>, <span class="author">Sepp&auml;nen JK</span>, <span class="author">Congdon E</span>, <span class="author">Service S</span>, <span class="author">Koiranen M</span>, <span class="author">Ekelund J</span>, <span class="author">Laitinen J</span>, <span class="author">Taanila A</span>, <span class="author">Tammelin T</span>, <span class="author">Hintsanen M</span>, <span class="author">Pulkki-R&aring;back L</span>, <span class="author">Keltikangas-J&auml;rvinen L</span>, <span class="author">Viikari J</span>, <span class="author">Raitakari OT</span>, <span class="author">Joukamaa M</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Freimer N</span>, <span class="author">Peltonen L</span>, <span class="author">Veijola J</span>, <span class="author">Mannila H</span> and <span class="author">Paunio T</span></p>
<p class="affiliation">Helsinki Institute for Information Technology and Department of Computer Science, University of Helsinki, Helsinki, Finland.</p>
<p class="abstract">Background: The object of this study was to identify temperament patterns in the Finnish population, and to determine the relationship between these profiles and life habits, socioeconomic status, and health.

A cluster analysis of the Temperament and Character Inventory subscales was performed on 3,761 individuals from the Northern Finland Birth Cohort 1966 and replicated on 2,097 individuals from the Cardiovascular Risk in Young Finns study. Clusters were formed using the k-means method and their relationship with 115 variables from the areas of life habits, socioeconomic status and health was examined.

Results: Four clusters were identified for both genders. Individuals from Cluster I are characterized by high persistence, low extravagance and disorderliness. They have healthy life habits, and lowest scores in most of the measures for psychiatric disorders. Cluster II individuals are characterized by low harm avoidance and high novelty seeking. They report the best physical capacity and highest level of income, but also high rate of divorce, smoking, and alcohol consumption. Individuals from Cluster III are not characterized by any extreme characteristic. Individuals from Cluster IV are characterized by high levels of harm avoidance, low levels of exploratory excitability and attachment, and score the lowest in most measures of health and well-being.

Conclusions: This study shows that the temperament subscales do not distribute randomly but have an endogenous structure, and that these patterns have strong associations to health, life events, and well-being.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">e33088</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22815673" rel="external">22815673</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3399883" rel="external">3399883</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0033088" rel="external">10.1371/journal.pone.0033088</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GFAP-Cre-Mediated Transgenic Activation of Bmi1 Results in Pituitary Tumors.</h4>
<p class="authors"><span class="author">Westerman BA</span>, <span class="author">Blom M</span>, <span class="author">Tanger E</span>, <span class="author">van der Valk M</span>, <span class="author">Song JY</span>, <span class="author">van Santen M</span>, <span class="author">Gadiot J</span>, <span class="author">Cornelissen-Steijger P</span>, <span class="author">Zevenhoven J</span>, <span class="author">Prosser HM</span>, <span class="author">Uren A</span>, <span class="author">Aronica E</span> and <span class="author">van Lohuizen M</span></p>
<p class="affiliation">Division of Molecular Genetics, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</p><p class="abstract">Bmi1 is a member of the polycomb repressive complex 1 and plays different roles during embryonic development, depending on the developmental context. Bmi1 over expression is observed in many types of cancer, including tumors of astroglial and neural origin. Although genetic depletion of Bmi1 has been described to result in tumor inhibitory effects partly through INK4A/Arf mediated senescence and apoptosis and also through INK4A/Arf independent effects, it has not been proven that Bmi1 can be causally involved in the formation of these tumors. To see whether this is the case, we developed two conditional Bmi1 transgenic models that were crossed with GFAP-Cre mice to activate transgenic expression in neural and glial lineages. We show here that these mice generate intermediate and anterior lobe pituitary tumors that are positive for ACTH and beta-endorphin. Combined transgenic expression of Bmi1 together with conditional loss of Rb resulted in pituitary tumors but was insufficient to induce medulloblastoma therefore indicating that the oncogenic function of Bmi1 depends on regulation of p16(INK4A)/Rb rather than on regulation of p19(ARF)/p53. Human pituitary adenomas show Bmi1 overexpression in over 50% of the cases, which indicates that Bmi1 could be causally involved in formation of these tumors similarly as in our mouse model.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">e35943</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22574128" rel="external">22574128</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3344841" rel="external">3344841</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0035943" rel="external">10.1371/journal.pone.0035943</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular Characterization of the viaB Locus Encoding the Biosynthetic Machinery for Vi Capsule Formation in Salmonella Typhi.</h4>
<p class="authors"><span class="author">Wetter M</span>, <span class="author">Goulding D</span>, <span class="author">Pickard D</span>, <span class="author">Kowarik M</span>, <span class="author">Waechter CJ</span>, <span class="author">Dougan G</span> and <span class="author">Wacker M</span></p>
<p class="affiliation">GlycoVaxyn AG, Schlieren, Switzerland.</p>
<p class="abstract">The Vi capsular polysaccharide (CPS) of Salmonella enterica serovar Typhi, the cause of human typhoid, is important for infectivity and virulence. The Vi biosynthetic machinery is encoded within the viaB locus composed of 10 genes involved in regulation of expression (tviA), polymer synthesis (tviB-tviE), and cell surface localization of the CPS (vexA-vexE). We cloned the viaB locus from S. Typhi and transposon insertion mutants of individual viaB genes were characterized in Escherichia coli DH5Î±. Phenotype analysis of viaB mutants revealed that tviB, tviC, tviD and tviE are involved in Vi polymer synthesis. Furthermore, expression of tviB-tviE in E. coli DH5Î± directed the synthesis of cytoplasmic Vi antigen. Mutants of the ABC transporter genes vexBC and the polysaccharide copolymerase gene vexD accumulated the Vi polymer within the cytoplasm and productivity in these mutants was greatly reduced. In contrast, de novo synthesis of Vi polymer in the export deficient vexA mutant was comparable to wild-type cells, with drastic effects on cell stability. VexE mutant cells exported the Vi, but the CPS was not retained at the cell surface. The secreted polymer of a vexE mutant had different physical characteristics compared to the wild-type Vi.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">9</span>;<span class="pagerange">e45609</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23029132" rel="external">23029132</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3448643" rel="external">3448643</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0045609" rel="external">10.1371/journal.pone.0045609</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EGFR Gene Variants Are Associated with Specific Somatic Aberrations in Glioma.</h4>
<p class="authors"><span class="author">Wibom C</span>, <span class="author">Ghasimi S</span>, <span class="author">Van Loo P</span>, <span class="author">Br&auml;nnstr&ouml;m T</span>, <span class="author">Trygg J</span>, <span class="author">Lau C</span>, <span class="author">Henriksson R</span>, <span class="author">Bergenheim T</span>, <span class="author">Andersson U</span>, <span class="author">Ryd&eacute;n P</span> and <span class="author">Melin B</span></p>
<p class="affiliation">Department of Radiation Sciences, Oncology, Computational Life Science Cluster (CLiC), Umeå University, Umeå, Sweden.</p>
<p class="abstract">A number of gene variants have been associated with an increased risk of developing glioma. We hypothesized that the reported risk variants may be associated with tumor genomic instability. To explore potential correlations between germline risk variants and somatic genetic events, we analyzed matched tumor and blood samples from 95 glioma patients by means of SNP genotyping. The generated genotype data was used to calculate genome-wide allele-specific copy number profiles of the tumor samples. We compared the copy number profiles across samples and found two EGFR gene variants (rs17172430 and rs11979158) that were associated with homozygous deletion at the CDKN2A/B locus. One of the EGFR variants (rs17172430) was also associated with loss of heterozygosity at the EGFR locus. Our findings were confirmed in a separate dataset consisting of matched blood and tumor samples from 300 glioblastoma patients, compiled from publically available TCGA data. These results imply there is a functional effect of germline EGFR variants on tumor progression.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">e47929</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23236348" rel="external">23236348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3517607" rel="external">3517607</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0047929" rel="external">10.1371/journal.pone.0047929</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pubertal timing and growth influences cardiometabolic risk factors in adult males and females.</h4>
<p class="authors"><span class="author">Wid&eacute;n E</span>, <span class="author">Silventoinen K</span>, <span class="author">Sovio U</span>, <span class="author">Ripatti S</span>, <span class="author">Cousminer DL</span>, <span class="author">Hartikainen AL</span>, <span class="author">Laitinen J</span>, <span class="author">Pouta A</span>, <span class="author">Kaprio J</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Peltonen L</span> and <span class="author">Palotie A</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, University of Helsinki, Helsinki, Finland. <a href="mailto:%65%6C%69%73%61%62%65%74%68.%77%69%64%65%6E%40%68%65%6C%73%69%6E%6B%69.%66%69">&#101;&#108;&#105;&#115;&#97;&#98;&#101;&#116;&#104;&#46;&#119;&#105;&#100;&#101;&#110;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a></p><p class="abstract"><strong>Objective</strong>: Early pubertal onset in females is associated with increased risk for adult obesity and cardiovascular disease, but whether this relationship is independent of preceding childhood growth events is unclear. Furthermore, the association between male puberty and adult disease remains unknown. To clarify the link between puberty and adult health, we evaluated the relationship between pubertal timing and risk factors for type 2 diabetes and cardiovascular disease in both males and females from a large, prospective, and randomly ascertained birth cohort from Northern Finland.</p><p class="abstract">Pubertal timing was estimated based on pubertal height growth in 5,058 subjects (2,417 males and 2,641 females), and the relationship between puberty and body weight, glucose and lipid homeostasis, and blood pressure at age 31 years was evaluated with linear regression modeling.</p><p class="abstract"><strong>Results</strong>: Earlier pubertal timing associated with higher adult BMI, fasting insulin, diastolic blood pressure, and decreased HDL cholesterol in both sexes (P &lt; 0.002) and with higher total serum cholesterol, LDL cholesterol, and triglycerides in males. The association with BMI and diastolic blood pressure remained statistically significant in both sexes, as did the association with insulin levels and HDL cholesterol concentrations in males after adjusting for covariates reflecting both fetal and childhood growth including childhood BMI.</p><p class="abstract"><strong>Conclusions</strong>: We demonstrate independent association between earlier pubertal timing and adult metabolic syndrome-related derangements both in males and females. The connection emphasizes that the mechanisms advancing puberty may also contribute to adult metabolic disorders.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>; Wellcome Trust: <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">Diabetes care</span>&nbsp;<span class="year">2012</span>;<span class="volume">35</span>;<span class="number">4</span>;<span class="pagerange">850-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22338106" rel="external">22338106</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3308310" rel="external">3308310</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdc11-1365" rel="external">10.2337/dc11-1365</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhancing Blockade of Plasmodium falciparum Erythrocyte Invasion: Assessing Combinations of Antibodies against PfRH5 and Other Merozoite Antigens.</h4>
<p class="authors"><span class="author">Williams AR</span>, <span class="author">Douglas AD</span>, <span class="author">Miura K</span>, <span class="author">Illingworth JJ</span>, <span class="author">Choudhary P</span>, <span class="author">Murungi LM</span>, <span class="author">Furze JM</span>, <span class="author">Diouf A</span>, <span class="author">Miotto O</span>, <span class="author">Crosnier C</span>, <span class="author">Wright GJ</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Fairhurst RM</span>, <span class="author">Long CA</span> and <span class="author">Draper SJ</span></p>
<p class="affiliation">The Jenner Institute, University of Oxford, Oxford, United Kingdom.</p>
<p class="abstract">No vaccine has yet proven effective against the blood-stages of Plasmodium falciparum, which cause the symptoms and severe manifestations of malaria. We recently found that PfRH5, a P. falciparum-specific protein expressed in merozoites, is efficiently targeted by broadly-neutralizing, vaccine-induced antibodies. Here we show that antibodies against PfRH5 efficiently inhibit the in vitro growth of short-term-adapted parasite isolates from Cambodia, and that the EC(50) values of antigen-specific antibodies against PfRH5 are lower than those against PfAMA1. Since antibody responses elicited by multiple antigens are speculated to improve the efficacy of blood-stage vaccines, we conducted detailed assessments of parasite growth inhibition by antibodies against PfRH5 in combination with antibodies against seven other merozoite antigens. We found that antibodies against PfRH5 act synergistically with antibodies against certain other merozoite antigens, most notably with antibodies against other erythrocyte-binding antigens such as PfRH4, to inhibit the growth of a homologous P. falciparum clone. A combination of antibodies against PfRH4 and basigin, the erythrocyte receptor for PfRH5, also potently inhibited parasite growth. This methodology provides the first quantitative evidence that polyclonal vaccine-induced antibodies can act synergistically against P. falciparum antigens and should help to guide the rational development of future multi-antigen vaccines.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G1000527(95710)</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">e1002991</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23144611" rel="external">23144611</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3493472" rel="external">3493472</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1002991" rel="external">10.1371/journal.ppat.1002991</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genes contributing to pain sensitivity in the normal population: an exome sequencing study.</h4>
<p class="authors"><span class="author">Williams FM</span>, <span class="author">Scollen S</span>, <span class="author">Cao D</span>, <span class="author">Memari Y</span>, <span class="author">Hyde CL</span>, <span class="author">Zhang B</span>, <span class="author">Sidders B</span>, <span class="author">Ziemek D</span>, <span class="author">Shi Y</span>, <span class="author">Harris J</span>, <span class="author">Harrow I</span>, <span class="author">Dougherty B</span>, <span class="author">Malarstig A</span>, <span class="author">McEwen R</span>, <span class="author">Stephens JC</span>, <span class="author">Patel K</span>, <span class="author">Menni C</span>, <span class="author">Shin SY</span>, <span class="author">Hodgkiss D</span>, <span class="author">Surdulescu G</span>, <span class="author">He W</span>, <span class="author">Jin X</span>, <span class="author">McMahon SB</span>, <span class="author">Soranzo N</span>, <span class="author">John S</span>, <span class="author">Wang J</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King's College London, London, United Kingdom.</p>
<p class="abstract">Sensitivity to pain varies considerably between individuals and is known to be heritable. Increased sensitivity to experimental pain is a risk factor for developing chronic pain, a common and debilitating but poorly understood symptom. To understand mechanisms underlying pain sensitivity and to search for rare gene variants (MAF<5%) influencing pain sensitivity, we explored the genetic variation in individuals' responses to experimental pain. Quantitative sensory testing to heat pain was performed in 2,500 volunteers from TwinsUK (TUK): exome sequencing to a depth of 70Ã— was carried out on DNA from singletons at the high and low ends of the heat pain sensitivity distribution in two separate subsamples. Thus in TUK1, 101 pain-sensitive and 102 pain-insensitive were examined, while in TUK2 there were 114 and 96 individuals respectively. A combination of methods was used to test the association between rare variants and pain sensitivity, and the function of the genes identified was explored using network analysis. Using causal reasoning analysis on the genes with different patterns of SNVs by pain sensitivity status, we observed a significant enrichment of variants in genes of the angiotensin pathway (Bonferroni corrected pâ€Š=â€Š3.8Ã—10(-4)). This pathway is already implicated in animal models and human studies of pain, supporting the notion that it may provide fruitful new targets in pain management. The approach of sequencing extreme exome variation in normal individuals has provided important insights into gene networks mediating pain sensitivity in humans and will be applicable to other common complex traits.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">e1003095</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23284290" rel="external">23284290</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3527205" rel="external">3527205</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1003095" rel="external">10.1371/journal.pgen.1003095</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nuclear receptor binding protein 1 regulates intestinal progenitor cell homeostasis and tumour formation.</h4>
<p class="authors"><span class="author">Wilson CH</span>, <span class="author">Crombie C</span>, <span class="author">van der Weyden L</span>, <span class="author">Poulogiannis G</span>, <span class="author">Rust AG</span>, <span class="author">Pardo M</span>, <span class="author">Gracia T</span>, <span class="author">Yu L</span>, <span class="author">Choudhary J</span>, <span class="author">Poulin GB</span>, <span class="author">McIntyre RE</span>, <span class="author">Winton DJ</span>, <span class="author">March HN</span>, <span class="author">Arends MJ</span>, <span class="author">Fraser AG</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK.</p><p class="abstract">Genetic screens in simple model organisms have identified many of the key components of the conserved signal transduction pathways that are oncogenic when misregulated. Here, we identify H37N21.1 as a gene that regulates vulval induction in let-60(n1046gf), a strain with a gain-of-function mutation in the Caenorhabditis elegans Ras orthologue, and show that somatic deletion of Nrbp1, the mouse orthologue of this gene, results in an intestinal progenitor cell phenotype that leads to profound changes in the proliferation and differentiation of all intestinal cell lineages. We show that Nrbp1 interacts with key components of the ubiquitination machinery and that loss of Nrbp1 in the intestine results in the accumulation of Sall4, a key mediator of stem cell fate, and of Tsc22d2. We also reveal that somatic loss of Nrbp1 results in tumourigenesis, with haematological and intestinal tumours predominating, and that nuclear receptor binding protein 1 (NRBP1) is downregulated in a range of human tumours, where low expression correlates with a poor prognosis. Thus NRBP1 is a conserved regulator of cell fate, that plays an important role in tumour suppression.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>, <span class="grant">17044</span>; Medical Research Council: <span class="grant">G0600127</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">31</span>;<span class="number">11</span>;<span class="pagerange">2486-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22510880">22510880</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3365428">3365428</a>; DOI: <a href="http://dx.doi.org/10.1038%2Femboj.2012.91">10.1038/emboj.2012.91</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diversity in parasitic nematode genomes: the microRNAs of Brugia pahangi and Haemonchus contortus are largely novel.</h4>
<p class="authors"><span class="author">Winter AD</span>, <span class="author">Weir W</span>, <span class="author">Hunt M</span>, <span class="author">Berriman M</span>, <span class="author">Gilleard JS</span>, <span class="author">Devaney E</span> and <span class="author">Britton C</span></p>
<p class="affiliation">Institute of Infection, Immunity and Inflammation, College of Medical, Veterinary and Life Sciences; University of Glasgow, Garscube Estate, Bearsden Road, Glasgow, G61 1QH, UK. <a href="mailto:%41%6C%61%6E.%57%69%6E%74%65%72%40%67%6C%61%73%67%6F%77.%61%63.%75%6B">&#65;&#108;&#97;&#110;&#46;&#87;&#105;&#110;&#116;&#101;&#114;&#64;&#103;&#108;&#97;&#115;&#103;&#111;&#119;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: MicroRNAs (miRNAs) play key roles in regulating post-transcriptional gene expression and are essential for development in the free-living nematode Caenorhabditis elegans and in higher organisms. Whether microRNAs are involved in regulating developmental programs of parasitic nematodes is currently unknown. Here we describe the the miRNA repertoire of two important parasitic nematodes as an essential first step in addressing this question.</p><p class="abstract"><strong>Results</strong>: The small RNAs from larval and adult stages of two parasitic species, Brugia pahangi and Haemonchus contortus, were identified using deep-sequencing and bioinformatic approaches. Comparative analysis to known miRNA sequences reveals that the majority of these miRNAs are novel. Some novel miRNAs are abundantly expressed and display developmental regulation, suggesting important functional roles. Despite the lack of conservation in the miRNA repertoire, genomic positioning of certain miRNAs within or close to specific coding genes is remarkably conserved across diverse species, indicating selection for these associations. Endogenous small-interfering RNAs and Piwi-interacting (pi)RNAs, which regulate gene and transposon expression, were also identified. piRNAs are expressed in adult stage H. contortus, supporting a conserved role in germline maintenance in some parasitic nematodes.</p><p class="abstract"><strong>Conclusions</strong>: This in-depth comparative analysis of nematode miRNAs reveals the high level of divergence across species and identifies novel sequences potentially involved in development. Expression of novel miRNAs may reflect adaptations to different environments and lifestyles. Our findings provide a detailed foundation for further study of the evolution and function of miRNAs within nematodes and for identifying potential targets for intervention.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085775/Z/08/Z</span>, <span class="grant">086823/Z/08/Z</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22216965" rel="external">22216965</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3282659" rel="external">3282659</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-4" rel="external">10.1186/1471-2164-13-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and characterization of the FVB/NJ mouse genome.</h4>
<p class="authors"><span class="author">Wong K</span>, <span class="author">Bumpstead S</span>, <span class="author">Van Der Weyden L</span>, <span class="author">Reinholdt LG</span>, <span class="author">Wilming LG</span>, <span class="author">Adams DJ</span> and <span class="author">Keane TM</span></p><p class="abstract"><strong>Background</strong>: The FVB/NJ mouse strain has its origins in a colony of outbred Swiss mice established in 1935 at the National Institutes of Health. Mice derived from this source were selectively bred for sensitivity to histamine diphosphate and the B strain of Friend leukemia virus. This led to the establishment of the FVB/N inbred strain, which was subsequently imported to the Jackson Laboratory and designated FVB/NJ. The FVB/NJ mouse has several distinct characteristics, such as large pronuclear morphology, vigorous reproductive performance, and consistently large litters that make it highly desirable for transgenic strain production and general purpose use.</p><p class="abstract"><strong>Results</strong>: Using next-generation sequencing technology, we have sequenced the genome of FVB/NJ to approximately 50-fold coverage, and have generated a comprehensive catalog of single nucleotide polymorphisms, small insertion/deletion polymorphisms, and structural variants, relative to the reference C57BL/6J genome. We have examined a previously identified quantitative trait locus for atherosclerosis susceptibility on chromosome 10 and identify several previously unknown candidate causal variants.</p><p class="abstract"><strong>Conclusion</strong>: The sequencing of the FVB/NJ genome and generation of this catalog has increased the number of known variant sites in FVB/NJ by a factor of four, and will help accelerate the identification of the precise molecular variants that are responsible for phenotypes observed in this widely used strain.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>; NIH HHS: <span class="grant">P40 OD010972</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">8</span>;<span class="pagerange">R72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22916792">22916792</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3491372">3491372</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-8-r72">10.1186/gb-2012-13-8-r72</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PomBase: a comprehensive online resource for fission yeast.</h4>
<p class="authors"><span class="author">Wood V</span>, <span class="author">Harris MA</span>, <span class="author">McDowall MD</span>, <span class="author">Rutherford K</span>, <span class="author">Vaughan BW</span>, <span class="author">Staines DM</span>, <span class="author">Aslett M</span>, <span class="author">Lock A</span>, <span class="author">B&auml;hler J</span>, <span class="author">Kersey PJ</span> and <span class="author">Oliver SG</span></p>
<p class="affiliation">Cambridge Systems Biology Centre, Department of Biochemistry, University of Cambridge, Sanger Building, 80 Tennis Court Road, Cambridge CB2 1GA, UK. <a href="mailto:%76%77%32%35%33%40%63%61%6D.%61%63.%75%6B">&#118;&#119;&#50;&#53;&#51;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">PomBase (www.pombase.org) is a new model organism database established to provide access to comprehensive, accurate, and up-to-date molecular data and biological information for the fission yeast Schizosaccharomyces pombe to effectively support both exploratory and hypothesis-driven research. PomBase encompasses annotation of genomic sequence and features, comprehensive manual literature curation and genome-wide data sets, and supports sophisticated user-defined queries. The implementation of PomBase integrates a Chado relational database that houses manually curated data with Ensembl software that supports sequence-based annotation and web access. PomBase will provide user-friendly tools to promote curation by experts within the fission yeast community. This will make a key contribution to shaping its content and ensuring its comprehensiveness and long-term relevance.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT090548MA</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D695-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22039153" rel="external">22039153</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245111" rel="external">3245111</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr853" rel="external">10.1093/nar/gkr853</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Questions over high frequency of mutant PfATP6 haplotypes in traveller isolates.</h4>
<p class="authors"><span class="author">Woodrow CJ</span>, <span class="author">Gardner KB</span> and <span class="author">Bustamante LY</span></p>
<p class="affiliation">Mahidol-Oxford University Tropical Medicine Research Unit (MORU), Bangkok, Thailand. <a href="mailto:%63%68%61%72%6C%69%65%40%74%72%6F%70%6D%65%64%72%65%73.%61%63">&#99;&#104;&#97;&#114;&#108;&#105;&#101;&#64;&#116;&#114;&#111;&#112;&#109;&#101;&#100;&#114;&#101;&#115;&#46;&#97;&#99;</a>.</p>
<p class="abstract">ABSTRACT: A recent paper in Malaria Journal suggests that a high proportion of Plasmodium falciparum isolates found in travellers returning from a range of African countries carry the PfATP6 A623E S769N haplotype, and that this genotype is associated with artemether resistance. Such a finding would represent a substantial departure from the extensive literature reporting these individual mutations to be very rare, with the double mutation never documented. The number of isolates screened to obtain these double mutants is unstated, but highly relevant, not least because selection of isolates could have introduced significant confounders, such as timing of in vitro testing. An additional concern relates to the location of sequencing primers used to assess these positions. In the absence of clear information on these fundamental questions it would be appropriate to treat the findings with caution.</p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="pagerange">186</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22681876" rel="external">22681876</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3483201" rel="external">3483201</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-11-186" rel="external">10.1186/1475-2875-11-186</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced peptide identification by electron transfer dissociation using an improved Mascot Percolator.</h4>
<p class="authors"><span class="author">Wright JC</span>, <span class="author">Collins MO</span>, <span class="author">Yu L</span>, <span class="author">K&auml;ll L</span>, <span class="author">Brosch M</span> and <span class="author">Choudhary JS</span></p>
<p class="affiliation">Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute, Hinxton, Cambridge.</p><p class="abstract">Peptide identification using tandem mass spectrometry is a core technology in proteomics. Latest generations of mass spectrometry instruments enable the use of electron transfer dissociation (ETD) to complement collision induced dissociation (CID) for peptide fragmentation. However, a critical limitation to the use of ETD has been optimal database search software. Percolator is a post-search algorithm, which uses semi-supervised machine learning to improve the rate of peptide spectrum identifications (PSMs) together with providing reliable significance measures. We have previously interfaced the Mascot search engine with Percolator and demonstrated sensitivity and specificity benefits with CID data. Here, we report recent developments in the Mascot Percolator V2.0 software including an improved feature calculator and support for a wider range of ion series. The updated software is applied to the analysis of several CID and ETD fragmented peptide data sets. This version of Mascot Percolator increases the number of CID PSMs by up to 80% and ETD PSMs by up to 60% at a 0.01 q-value (1% false discovery rate) threshold over a standard Mascot search, notably recovering PSMs from high charge state precursor ions. The greatly increased number of PSMs and peptide coverage afforded by Mascot Percolator has enabled a fuller assessment of CID/ETD complementarity to be performed. Using a data set of CID and ETcaD spectral pairs, we find that at a 1% false discovery rate, the overlap in peptide identifications by CID and ETD is 83%, which is significantly higher than that obtained using either stand-alone Mascot (69%) or OMSSA (39%). We conclude that Mascot Percolator is a highly sensitive and accurate post-search algorithm for peptide identification and allows direct comparison of peptide identifications using multiple alternative fragmentation techniques.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643/Z/06/Z</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2012</span>;<span class="volume">11</span>;<span class="number">8</span>;<span class="pagerange">478-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22493177">22493177</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3412976">3412976</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.O111.014522">10.1074/mcp.O111.014522</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Direct-to-Consumer Genetic Testing; Molecular Genetics and Personalized Medicine</h4>
<p class="authors"><span class="author">WRIGHT,C. F.</span> and <span class="author">MACARTHUR,D. G.</span></p>
<p class="publication"><span class="title">Molecular and Translational Medicine</span>&nbsp;<span class="year">2012</span>;<span class="pagerange">215-36</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1007%2F978-1-61779-530-5_10" rel="external">10.1007/978-1-61779-530-5_10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic variation in the vomeronasal receptor gene repertoires of inbred mice.</h4>
<p class="authors"><span class="author">Wynn EH</span>, <span class="author">S&aacute;nchez-Andrade G</span>, <span class="author">Carss KJ</span> and <span class="author">Logan DW</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Vomeronasal receptors (VRs), expressed in sensory neurons of the vomeronasal organ, are thought to bind pheromones and mediate innate behaviours. The mouse reference genome has over 360 functional VRs arranged in highly homologous clusters, but the vast majority are of unknown function. Differences in these receptors within and between closely related species of mice are likely to underpin a range of behavioural responses. To investigate these differences, we interrogated the VR gene repertoire from 17 inbred strains of mice using massively parallel sequencing.</p><p class="abstract"><strong>Results</strong>: Approximately half of the 6222 VR genes that we investigated could be successfully resolved, and those that were unambiguously mapped resulted in an extremely accurate dataset. Collectively VRs have over twice the coding sequence variation of the genome average; but we identify striking non-random distribution of these variants within and between genes, clusters, clades and functional classes of VRs. We show that functional VR gene repertoires differ considerably between different Mus subspecies and species, suggesting these receptors may play a role in mediating behavioural adaptations. Finally, we provide evidence that widely-used, highly inbred laboratory-derived strains have a greatly reduced, but not entirely redundant capacity for differential pheromone-mediated behaviours.</p><p class="abstract"><strong>Conclusions</strong>: Together our results suggest that the unusually variable VR repertoires of mice have a significant role in encoding differences in olfactory-mediated responses and behaviours. Our dataset has expanded over nine fold the known number of mouse VR alleles, and will enable mechanistic analyses into the genetics of innate behavioural differences in mice.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="pagerange">415</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22908939">22908939</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3460788">3460788</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-13-415">10.1186/1471-2164-13-415</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The multidrug-resistant PMEN1 pneumococcus is a paradigm for genetic success.</h4>
<p class="authors"><span class="author">Wyres KL</span>, <span class="author">Lambertsen LM</span>, <span class="author">Croucher NJ</span>, <span class="author">McGee L</span>, <span class="author">von Gottberg A</span>, <span class="author">Li&ntilde;ares J</span>, <span class="author">Jacobs MR</span>, <span class="author">Kristinsson KG</span>, <span class="author">Beall BW</span>, <span class="author">Klugman KP</span>, <span class="author">Parkhill J</span>, <span class="author">Hakenbeck R</span>, <span class="author">Bentley SD</span> and <span class="author">Brueggemann AB</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, South Parks Road, Oxford, OX1 3PS, UK. <a href="mailto:%61%6E%67%65%6C%61.%62%72%75%65%67%67%65%6D%61%6E%6E%40%7A%6F%6F.%6F%78.%61%63.%75%6B">&#97;&#110;&#103;&#101;&#108;&#97;&#46;&#98;&#114;&#117;&#101;&#103;&#103;&#101;&#109;&#97;&#110;&#110;&#64;&#122;&#111;&#111;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">ABSTRACT: BACKGROUND: Streptococcus pneumoniae, also called the pneumococcus, is a major bacterial pathogen. Since its introduction in the 1940s, penicillin has been the primary treatment for pneumococcal diseases. Penicillin resistance rapidly increased among pneumococci over the past 30 years, and one particular multidrug-resistant clone, PMEN1, became highly prevalent globally. We studied a collection of 426 pneumococci isolated between 1937 and 2007 to better understand the evolution of penicillin resistance within this species. RESULTS: We discovered that one of the earliest known penicillin-nonsusceptible pneumococci, recovered in 1967 from Australia, was the likely ancestor of PMEN1, since approximately 95% of coding sequences identified within its genome were highly similar to those of PMEN1. The regions of the PMEN1 genome that differed from the ancestor contained genes associated with antibiotic resistance, transmission and virulence. We also revealed that PMEN1 was uniquely promiscuous with its DNA, donating penicillin-resistance genes and sometimes many other genes associated with antibiotic resistance, virulence and cell adherence to many genotypically diverse pneumococci. In particular, we describe two strains in which up to 10% of the PMEN1 genome was acquired in multiple fragments, some as long as 32 kb, distributed around the recipient genomes. This type of directional genetic promiscuity from a single clone to numerous unrelated clones has, to our knowledge, never before been described. CONCLUSIONS: These findings suggest that PMEN1 is a paradigm of genetic success both through its epidemiology and promiscuity. These findings also challenge the existing views about horizontal gene transfer among pneumococci.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">11</span>;<span class="pagerange">R103</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23158461">23158461</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-11-r103">10.1186/gb-2012-13-11-r103</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deleterious- and disease-allele prevalence in healthy individuals: insights from current predictions, mutation databases, and population-scale resequencing.</h4>
<p class="authors"><span class="author">Xue Y</span>, <span class="author">Chen Y</span>, <span class="author">Ayub Q</span>, <span class="author">Huang N</span>, <span class="author">Ball EV</span>, <span class="author">Mort M</span>, <span class="author">Phillips AD</span>, <span class="author">Shaw K</span>, <span class="author">Stenson PD</span>, <span class="author">Cooper DN</span>, <span class="author">Tyler-Smith C</span> and <span class="author">1000 Genomes Project Consortium</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">We have assessed the numbers of potentially deleterious variants in the genomes of apparently healthy humans by using (1) low-coverage whole-genome sequence data from 179 individuals in the 1000 Genomes Pilot Project and (2) current predictions and databases of deleterious variants. Each individual carried 281-515 missense substitutions, 40-85 of which were homozygous, predicted to be highly damaging. They also carried 40-110 variants classified by the Human Gene Mutation Database (HGMD) as disease-causing mutations (DMs), 3-24 variants in the homozygous state, and many polymorphisms putatively associated with disease. Whereas many of these DMs are likely to represent disease-allele-annotation errors, between 0 and 8 DMs (0-1 homozygous) per individual are predicted to be highly damaging, and some of them provide information of medical relevance. These analyses emphasize the need for improved annotation of disease alleles both in mutation databases and in the primary literature; some HGMD mutation data have been recategorized on the basis of the present findings, an iterative process that is both necessary and ongoing. Our estimates of deleterious-allele numbers are likely to be subject to both overcounting and undercounting. However, our current best mean estimates of ~400 damaging variants and ~2 bona fide disease mutations per individual are likely to increase rather than decrease as sequencing studies ascertain rare variants more effectively and as additional disease alleles are discovered.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085532</span>, <span class="grant">WT098051</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">91</span>;<span class="number">6</span>;<span class="pagerange">1022-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23217326">23217326</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3516590">3516590</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2012.10.015">10.1016/j.ajhg.2012.10.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation sequencing of experimental mouse strains.</h4>
<p class="authors"><span class="author">Yalcin B</span>, <span class="author">Adams DJ</span>, <span class="author">Flint J</span> and <span class="author">Keane TM</span></p>
<p class="affiliation">Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland. <a href="mailto:%42%69%6E%6E%61%7A.%59%61%6C%63%69%6E%40%75%6E%69%6C.%63%68">&#66;&#105;&#110;&#110;&#97;&#122;&#46;&#89;&#97;&#108;&#99;&#105;&#110;&#64;&#117;&#110;&#105;&#108;&#46;&#99;&#104;</a></p><p class="abstract">Since the turn of the century the complete genome sequence of just one mouse strain, C57BL/6J, has been available. Knowing the sequence of this strain has enabled large-scale forward genetic screens to be performed, the creation of an almost complete set of embryonic stem (ES) cell lines with targeted alleles for protein-coding genes, and the generation of a rich catalog of mouse genomic variation. However, many experiments that use other common laboratory mouse strains have been hindered by a lack of whole-genome sequence data for these strains. The last 5 years has witnessed a revolution in DNA sequencing technologies. Recently, these technologies have been used to expand the repertoire of fully sequenced mouse genomes. In this article we review the main findings of these studies and discuss how the sequence of mouse genomes is helping pave the way from sequence to phenotype. Finally, we discuss the prospects for using de novo assembly techniques to obtain high-quality assembled genome sequences of these laboratory mouse strains, and what advances in sequencing technologies may be required to achieve this goal.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council; Wellcome Trust: <span class="grant">090532</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2012</span>;<span class="volume">23</span>;<span class="number">9-10</span>;<span class="pagerange">490-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22772437">22772437</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3463794">3463794</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-012-9402-6">10.1007/s00335-012-9402-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The fine-scale architecture of structural variants in 17 mouse genomes.</h4>
<p class="authors"><span class="author">Yalcin B</span>, <span class="author">Wong K</span>, <span class="author">Bhomra A</span>, <span class="author">Goodson M</span>, <span class="author">Keane TM</span>, <span class="author">Adams DJ</span> and <span class="author">Flint J</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, Oxford, UK. <a href="mailto:%42%69%6E%6E%61%7A.%59%61%6C%63%69%6E%40%75%6E%69%6C.%63%68">&#66;&#105;&#110;&#110;&#97;&#122;&#46;&#89;&#97;&#108;&#99;&#105;&#110;&#64;&#117;&#110;&#105;&#108;&#46;&#99;&#104;</a></p><p class="abstract"><strong>Background</strong>: Accurate catalogs of structural variants (SVs) in mammalian genomes are necessary to elucidate the potential mechanisms that drive SV formation and to assess their functional impact. Next generation sequencing methods for SV detection are an advance on array-based methods, but are almost exclusively limited to four basic types: deletions, insertions, inversions and copy number gains.</p><p class="abstract"><strong>Results</strong>: By visual inspection of 100 Mbp of genome to which next generation sequence data from 17 inbred mouse strains had been aligned, we identify and interpret 21 paired-end mapping patterns, which we validate by PCR. These paired-end mapping patterns reveal a greater diversity and complexity in SVs than previously recognized. In addition, Sanger-based sequence analysis of 4,176 breakpoints at 261 SV sites reveal additional complexity at approximately a quarter of structural variants analyzed. We find micro-deletions and micro-insertions at SV breakpoints, ranging from 1 to 107 bp, and SNPs that extend breakpoint micro-homology and may catalyze SV formation.</p><p class="abstract"><strong>Conclusions</strong>: An integrative approach using experimental analyses to train computational SV calling is essential for the accurate resolution of the architecture of SVs. We find considerable complexity in SV formation; about a quarter of SVs in the mouse are composed of a complex mixture of deletion, insertion, inversion and copy number gain. Computational methods can be adapted to identify most paired-end mapping patterns.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">13031</span>; Medical Research Council; Wellcome Trust: <span class="grant">079912</span>, <span class="grant">090532</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2012</span>;<span class="volume">13</span>;<span class="number">3</span>;<span class="pagerange">R18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22439878">22439878</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3439969">3439969</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2012-13-3-r18">10.1186/gb-2012-13-3-r18</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intratumor heterogeneity: seeing the wood for the trees.</h4>
<p class="authors"><span class="author">Yap TA</span>, <span class="author">Gerlinger M</span>, <span class="author">Futreal PA</span>, <span class="author">Pusztai L</span> and <span class="author">Swanton C</span></p>
<p class="affiliation">Department of Medicine, Royal Marsden NHS Foundation Trust, Sutton, Surrey, UK.</p>
<p class="abstract">Most advanced solid tumors remain incurable, with resistance to chemotherapeutics and targeted therapies a common cause of poor clinical outcome. Intratumor heterogeneity may contribute to this failure by initiating phenotypic diversity enabling drug resistance to emerge and by introducing tumor sampling bias. Envisaging tumor growth as a Darwinian tree with the trunk representing ubiquitous mutations and the branches representing heterogeneous mutations may help in drug discovery and the development of predictive biomarkers of drug response.</p>
<p class="publication"><span class="title">Science translational medicine</span>&nbsp;<span class="year">2012</span>;<span class="volume">4</span>;<span class="number">127</span>;<span class="pagerange">127ps10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22461637" rel="external">22461637</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscitranslmed.3003854" rel="external">10.1126/scitranslmed.3003854</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of the cancer genome.</h4>
<p class="authors"><span class="author">Yates LR</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p>
<p class="abstract">The advent of massively parallel sequencing technologies has allowed the characterization of cancer genomes at an unprecedented resolution. Investigation of the mutational landscape of tumours is providing new insights into cancer genome evolution, laying bare the interplay of somatic mutation, adaptation of clones to their environment and natural selection. These studies have demonstrated the extent of the heterogeneity of cancer genomes, have allowed inferences to be made about the forces that act on nascent cancer clones as they evolve and have shown insight into the mutational processes that generate genetic variation. Here we review our emerging understanding of the dynamic evolution of the cancer genome and of the implications for basic cancer biology and the development of antitumour therapy.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23044827" rel="external">23044827</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg3317" rel="external">10.1038/nrg3317</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Harnessing genomics to identify environmental determinants of heritable disease.</h4>
<p class="authors"><span class="author">Yauk CL</span>, <span class="author">Lucas Argueso J</span>, <span class="author">Auerbach SS</span>, <span class="author">Awadalla P</span>, <span class="author">Davis SR</span>, <span class="author">Demarini DM</span>, <span class="author">Douglas GR</span>, <span class="author">Dubrova YE</span>, <span class="author">Elespuru RK</span>, <span class="author">Glover TW</span>, <span class="author">Hales BF</span>, <span class="author">Hurles ME</span>, <span class="author">Klein CB</span>, <span class="author">Lupski JR</span>, <span class="author">Manchester DK</span>, <span class="author">Marchetti F</span>, <span class="author">Montpetit A</span>, <span class="author">Mulvihill JJ</span>, <span class="author">Robaire B</span>, <span class="author">Robbins WA</span>, <span class="author">Rouleau GA</span>, <span class="author">Shaughnessy DT</span>, <span class="author">Somers CM</span>, <span class="author">Taylor JG</span>, <span class="author">Trasler J</span>, <span class="author">Waters MD</span>, <span class="author">Wilson TE</span>, <span class="author">Witt KL</span> and <span class="author">Bishop JB</span></p>
<p class="affiliation">Health Canada, Canada. Electronic address: <a href="mailto:%63%61%72%6F%6C%65.%79%61%75%6B%40%68%63-%73%63.%67%63.%63%61">&#99;&#97;&#114;&#111;&#108;&#101;&#46;&#121;&#97;&#117;&#107;&#64;&#104;&#99;&#45;&#115;&#99;&#46;&#103;&#99;&#46;&#99;&#97;</a>.</p>
<p class="abstract">Next-generation sequencing technologies can now be used to directly measure heritable de novo DNA sequence mutations in humans. However, these techniques have not been used to examine environmental factors that induce such mutations and their associated diseases. To address this issue, a working group on environmentally induced germline mutation analysis (ENIGMA) met in October 2011 to propose the necessary foundational studies, which include sequencing of parent-offspring trios from highly exposed human populations, and controlled dose-response experiments in animals. These studies will establish background levels of variability in germline mutation rates and identify environmental agents that influence these rates and heritable disease. Guidance for the types of exposures to examine come from rodent studies that have identified agents such as cancer chemotherapeutic drugs, ionizing radiation, cigarette smoke, and air pollution as germ-cell mutagens. Research is urgently needed to establish the health consequences of parental exposures on subsequent generations.</p>
<p class="publication"><span class="title">Mutation research</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22935230" rel="external">22935230</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mrrev.2012.08.002" rel="external">10.1016/j.mrrev.2012.08.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-Wide Screening for Genetic Alterations in Esophageal Cancer by aCGH Identifies 11q13 Amplification Oncogenes Associated with Nodal Metastasis.</h4>
<p class="authors"><span class="author">Ying J</span>, <span class="author">Shan L</span>, <span class="author">Li J</span>, <span class="author">Zhong L</span>, <span class="author">Xue L</span>, <span class="author">Zhao H</span>, <span class="author">Li L</span>, <span class="author">Langford C</span>, <span class="author">Guo L</span>, <span class="author">Qiu T</span>, <span class="author">Lu N</span> and <span class="author">Tao Q</span></p>
<p class="affiliation">Department of Pathology, Cancer Hospital, Peking Union Medical College & Chinese Academy of Medical Sciences, Beijing, China.</p>
<p class="abstract">Background: Esophageal squamous cell carcinoma (ESCC) is highly prevalent in China and other Asian countries, as a major cause of cancer-related mortality. ESCC displays complex chromosomal abnormalities, including multiple structural and numerical aberrations. Chromosomal abnormalities, such as recurrent amplifications and homozygous deletions, directly contribute to tumorigenesis through altering the expression of key oncogenes and tumor suppressor genes.

To understand the role of genetic alterations in ESCC pathogenesis and identify critical amplification/deletion targets, we performed genome-wide 1-Mb array comparative genomic hybridization (aCGH) analysis for 10 commonly used ESCC cell lines. Recurrent chromosomal gains were frequently detected on 3q26-27, 5p15-14, 8p12, 8p22-24, 11q13, 13q21-31, 18p11 and 20q11-13, with frequent losses also found on 8p23-22, 11q22, 14q32 and 18q11-23. Gain of 11q13.3-13.4 was the most frequent alteration in ESCC. Within this region, CCND1 oncogene was identified with high level of amplification and overexpression in ESCC, while FGF19 and SHANK2 was also remarkably over-expressed. Moreover, a high concordance (91.5%) of gene amplification and protein overexpression of CCND1 was observed in primary ESCC tumors. CCND1 amplification/overexpression was also significantly correlated with the lymph node metastasis of ESCC.

Conclusion: These findings suggest that genomic gain of 11q13 is the major mechanism contributing to the amplification. Novel oncogenes identified within the 11q13 amplicon including FGF19 and SHANK2 may play important roles in ESCC tumorigenesis.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2012</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">e39797</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22761904" rel="external">22761904</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3382571" rel="external">3382571</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0039797" rel="external">10.1371/journal.pone.0039797</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase 2012: more genomes, more data, new website.</h4>
<p class="authors"><span class="author">Yook K</span>, <span class="author">Harris TW</span>, <span class="author">Bieri T</span>, <span class="author">Cabunoc A</span>, <span class="author">Chan J</span>, <span class="author">Chen WJ</span>, <span class="author">Davis P</span>, <span class="author">de la Cruz N</span>, <span class="author">Duong A</span>, <span class="author">Fang R</span>, <span class="author">Ganesan U</span>, <span class="author">Grove C</span>, <span class="author">Howe K</span>, <span class="author">Kadam S</span>, <span class="author">Kishore R</span>, <span class="author">Lee R</span>, <span class="author">Li Y</span>, <span class="author">Muller HM</span>, <span class="author">Nakamura C</span>, <span class="author">Nash B</span>, <span class="author">Ozersky P</span>, <span class="author">Paulini M</span>, <span class="author">Raciti D</span>, <span class="author">Rangarajan A</span>, <span class="author">Schindelman G</span>, <span class="author">Shi X</span>, <span class="author">Schwarz EM</span>, <span class="author">Ann Tuli M</span>, <span class="author">Van Auken K</span>, <span class="author">Wang D</span>, <span class="author">Wang X</span>, <span class="author">Williams G</span>, <span class="author">Hodgkin J</span>, <span class="author">Berriman M</span>, <span class="author">Durbin R</span>, <span class="author">Kersey P</span>, <span class="author">Spieth J</span>, <span class="author">Stein L</span> and <span class="author">Sternberg PW</span></p>
<p class="affiliation">Division of Biology 156-29, California Institute of Technology, Pasadena, CA 91125, USA. <a href="mailto:%6B%79%6F%6F%6B%40%77%6F%72%6D%62%61%73%65.%6F%72%67">&#107;&#121;&#111;&#111;&#107;&#64;&#119;&#111;&#114;&#109;&#98;&#97;&#115;&#101;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Since its release in 2000, WormBase (http://www.wormbase.org) has grown from a small resource focusing on a single species and serving a dedicated research community, to one now spanning 15 species essential to the broader biomedical and agricultural research fields. To enhance the rate of curation, we have automated the identification of key data in the scientific literature and use similar methodology for data extraction. To ease access to the data, we are collaborating with journals to link entities in research publications to their report pages at WormBase. To facilitate discovery, we have added new views of the data, integrated large-scale datasets and expanded descriptions of models for human disease. Finally, we have introduced a dramatic overhaul of the WormBase website for public beta testing. Designed to balance complexity and usability, the new site is species-agnostic, highly customizable, and interactive. Casual users and developers alike will be able to leverage the public RESTful application programming interface (API) to generate custom data mining solutions and extensions to the site. We report on the growth of our database and on our work in keeping pace with the growing demand for data, efforts to anticipate the requirements of users and new collaborations with the larger science community.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G070119</span>, <span class="grant">G0701197</span>; NHGRI NIH HHS: <span class="grant">P41 HG02223</span>, <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span>;<span class="volume">40</span>;<span class="number">Database issue</span>;<span class="pagerange">D735-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22067452">22067452</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3245152">3245152</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkr954">10.1093/nar/gkr954</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic determinants of lipid homeostasis.</h4>
<p class="authors"><span class="author">Young EH</span>, <span class="author">Papamarkou T</span>, <span class="author">Wainwright NW</span> and <span class="author">Sandhu MS</span></p>
<p class="affiliation">Genetic Epidemiology Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%6C%69%7A.%79%6F%75%6E%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#105;&#122;&#46;&#121;&#111;&#117;&#110;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Circulating levels of blood lipids are heritable risk factors for atherosclerosis and heart disease, and are the target of therapeutic intervention. Studies of monogenic disorders and - more recently - genome-wide association studies have identified several important genetic determinants of blood lipid levels. These have the potential to provide new drug targets to alter blood lipid levels and may improve prediction of cardiovascular disease. Better functional validation of lipid loci is required to clarify the biological role of proteins encoded by specific genomic regions and understand how they influence lipid metabolism and confer disease risk.</p>
<p class="publication"><span class="title">Best practice &amp; research. Clinical endocrinology &amp; metabolism</span>&nbsp;<span class="year">2012</span>;<span class="volume">26</span>;<span class="number">2</span>;<span class="pagerange">203-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22498249" rel="external">22498249</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.beem.2011.11.004" rel="external">10.1016/j.beem.2011.11.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bcl11a is essential for lymphoid development and negatively regulates p53.</h4>
<p class="authors"><span class="author">Yu Y</span>, <span class="author">Wang J</span>, <span class="author">Khaled W</span>, <span class="author">Burke S</span>, <span class="author">Li P</span>, <span class="author">Chen X</span>, <span class="author">Yang W</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Zhang S</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of Ministry of Education, Huazhong Agricultural University, Wuhan 430070, China.</p><p class="abstract">Transcription factors play important roles in lymphopoiesis. We have previously demonstrated that Bcl11a is essential for normal lymphocyte development in the mouse embryo. We report here that, in the adult mouse, Bcl11a is expressed in most hematopoietic cells and is highly enriched in B cells, early T cell progenitors, common lymphoid progenitors (CLPs), and hematopoietic stem cells (HSCs). In the adult mouse, Bcl11a deletion causes apoptosis in early B cells and CLPs and completely abolishes the lymphoid development potential of HSCs to B, T, and NK cells. Myeloid development, in contrast, is not obviously affected by the loss of Bcl11a. Bcl11a regulates expression of Bcl2, Bcl2-xL, and Mdm2, which inhibits p53 activities. Overexpression of Bcl2 and Mdm2, or p53 deficiency, rescues both lethality and proliferative defects in Bcl11a-deficient early B cells and enables the mutant CLPs to differentiate to lymphocytes. Bcl11a is therefore essential for lymphopoiesis and negatively regulates p53 activities. Deletion of Bcl11a may represent a new approach for generating a mouse model that completely lacks an adaptive immune system.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">098051</span></p>
<p class="publication"><span class="title">The Journal of experimental medicine</span>&nbsp;<span class="year">2012</span>;<span class="volume">209</span>;<span class="number">13</span>;<span class="pagerange">2467-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23230003">23230003</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3526365">3526365</a>; DOI: <a href="http://dx.doi.org/10.1084%2Fjem.20121846">10.1084/jem.20121846</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human Serum Metabolic Profiles Are Age Dependent.</h4>
<p class="authors"><span class="author">Yu Z</span>, <span class="author">Zhai G</span>, <span class="author">Singmann P</span>, <span class="author">He Y</span>, <span class="author">Xu T</span>, <span class="author">Prehn C</span>, <span class="author">R&ouml;misch-Margl W</span>, <span class="author">Lattka E</span>, <span class="author">Gieger C</span>, <span class="author">Soranzo N</span>, <span class="author">Heinrich J</span>, <span class="author">Standl M</span>, <span class="author">Thiering E</span>, <span class="author">Mittelstra&szlig; K</span>, <span class="author">Wichmann HE</span>, <span class="author">Peters A</span>, <span class="author">Suhre K</span>, <span class="author">Li Y</span>, <span class="author">Adamski J</span>, <span class="author">Spector TD</span>, <span class="author">Illig T</span> and <span class="author">Wang-Sattler R</span></p>
<p class="affiliation">Research Unit of Molecular Epidemiology, Helmholtz ZentrumMünchen, 85764 Neuherberg, Germany. Department of Twin Research and Genetic Epidemiology, King's College London, London, UK. Discipline of Genetics, Faculty of Medicine, Memorial University of Newfoundland, St John's, NL Canada. Shanghai Center for Bioinformation Technology, 200235Shanghai, China. Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, 200031Shanghai,China. Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, 85764 Neuherberg, Germany. Institute ofBioinformaticsand Systems Biology, Helmholtz Zentrum München, 85764 Neuherberg, Germany. Institute ofGeneticEpidemiology, Helmholtz Zentrum München, 85764 Neuherberg, Germany. Wellcome Trust Sanger Institute Genome Campus, Hinxton, UK. Institute ofEpidemiology I, Helmholtz Zentrum München, 85764 Neuherberg, Germany. Institute of Medical Informatics, Biometry and Epidemiology, Chair of Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany. Klinikum Grosshadern, Munich, Germany. Institute ofEpidemiology II, Helmholtz Zentrum München, 85764 Neuherberg, Germany. Harvard School of Public Health Adjunct Associate Professor of Environmental Epidemiology Department of Environmental Health, Boston, USA. Faculty of Biology, Ludwig-Maximilians-Universität, 82152 Planegg-Martinsried, Germany. Department of Physiology and Biophysics,Weill Cornell Medical College in Qatar, 24144 Education City-Qatar Foundation, Doha, Qatar. Institute of Experimental Genetics, Life and Food Science Center Weihenstephan, TechnischeUniversitätMünchen, 85354 Freising-Weihenstephan, Germany. Hannover Unified Biobank, Hannover Medical School, 30625 Hannover, Germany.</p>
<p class="abstract">Understanding the complexity of aging is of utmost importance. This can now be addressed by the novel and powerful approach of metabolomics. However, to date, only a few metabolic studies based on large samples are available. Here, we provide novel and specific information on age-related metabolite concentration changes in human homeostasis. We report results from two population-based studies: the KORA F4 study from Germany as a discovery cohort, with 1038 female and 1124 male participants (32-81 years), and the TwinsUK study as replication, with 724 females. Targeted metabolomics of fasting serum samples quantified 131 metabolites by FIA-MS/MS. Among these, 71/34 metabolites were significantly associated with age in females/males (BMI adjusted). We further identified a set of 13 independent metabolites in women (with P values ranging from 4.6 Ã— 10(-04) to 7.8 Ã— 10(-42) , Î±(corr) = 0.004). Eleven of these 13 metabolites were replicated in the TwinsUK study, including seven metabolite concentrations that increased with age (C0, C10:1, C12:1, C18:1, SM C16:1, SM C18:1, and PC aa C28:1), while histidine decreased. These results indicate that metabolic profiles are age dependent and might reflect different aging processes, such as incomplete mitochondrial fatty acid oxidation. The use of metabolomics will increase our understanding of aging networks, and may lead to discoveries that help enhance healthy aging. Â© 2012 The Authors Aging Cell Â© 2012 Blackwell Publishing Ltd/Anatomical Society of Great Britain and Ireland.</p>
<p class="publication"><span class="title">Aging cell</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22834969" rel="external">22834969</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1474-9726.2012.00865.x" rel="external">10.1111/j.1474-9726.2012.00865.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Validity of the Family-Based Association Test for Copy Number Variant Data in the Case of Non-Linear Intensity-Genotype Relationship.</h4>
<p class="authors"><span class="author">Zanda M</span>, <span class="author">Onengut S</span>, <span class="author">Walker N</span>, <span class="author">Todd JA</span>, <span class="author">Clayton DG</span>, <span class="author">Rich SS</span>, <span class="author">Hurles ME</span> and <span class="author">Plagnol V</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22972682" rel="external">22972682</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.21674" rel="external">10.1002/gepi.21674</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exome sequencing of gastric adenocarcinoma identifies recurrent somatic mutations in cell adhesion and chromatin remodeling genes.</h4>
<p class="authors"><span class="author">Zang ZJ</span>, <span class="author">Cutcutache I</span>, <span class="author">Poon SL</span>, <span class="author">Zhang SL</span>, <span class="author">McPherson JR</span>, <span class="author">Tao J</span>, <span class="author">Rajasegaran V</span>, <span class="author">Heng HL</span>, <span class="author">Deng N</span>, <span class="author">Gan A</span>, <span class="author">Lim KH</span>, <span class="author">Ong CK</span>, <span class="author">Huang D</span>, <span class="author">Chin SY</span>, <span class="author">Tan IB</span>, <span class="author">Ng CC</span>, <span class="author">Yu W</span>, <span class="author">Wu Y</span>, <span class="author">Lee M</span>, <span class="author">Wu J</span>, <span class="author">Poh D</span>, <span class="author">Wan WK</span>, <span class="author">Rha SY</span>, <span class="author">So J</span>, <span class="author">Salto-Tellez M</span>, <span class="author">Yeoh KG</span>, <span class="author">Wong WK</span>, <span class="author">Zhu YJ</span>, <span class="author">Futreal PA</span>, <span class="author">Pang B</span>, <span class="author">Ruan Y</span>, <span class="author">Hillmer AM</span>, <span class="author">Bertrand D</span>, <span class="author">Nagarajan N</span>, <span class="author">Rozen S</span>, <span class="author">Teh BT</span> and <span class="author">Tan P</span></p>
<p class="affiliation">Cellular and Molecular Research, National Cancer Centre, Singapore.</p><p class="abstract">Gastric cancer is a major cause of global cancer mortality. We surveyed the spectrum of somatic alterations in gastric cancer by sequencing the exomes of 15 gastric adenocarcinomas and their matched normal DNAs. Frequently mutated genes in the adenocarcinomas included TP53 (11/15 tumors), PIK3CA (3/15) and ARID1A (3/15). Cell adhesion was the most enriched biological pathway among the frequently mutated genes. A prevalence screening confirmed mutations in FAT4, a cadherin family gene, in 5% of gastric cancers (6/110) and FAT4 genomic deletions in 4% (3/83) of gastric tumors. Frequent mutations in chromatin remodeling genes (ARID1A, MLL3 and MLL) also occurred in 47% of the gastric cancers. We detected ARID1A mutations in 8% of tumors (9/110), which were associated with concurrent PIK3CA mutations and microsatellite instability. In functional assays, we observed both FAT4 and ARID1A to exert tumor-suppressor activity. Somatic inactivation of FAT4 and ARID1A may thus be key tumorigenic events in a subset of gastric cancers.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2012</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">570-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22484628">22484628</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2246">10.1038/ng.2246</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metagenomics with guts.</h4>
<p class="authors"><span class="author">Zarowiecki M</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="abstract">This month's Genome Watch describes two recent studies that used metagenomic data to characterize the role of the human gut microbiota in disease.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22961340" rel="external">22961340</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2879" rel="external">10.1038/nrmicro2879</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SchistoDB: an updated genome resource for the three key schistosomes of humans.</h4>
<p class="authors"><span class="author">Zerlotini A</span>, <span class="author">Aguiar ER</span>, <span class="author">Yu F</span>, <span class="author">Xu H</span>, <span class="author">Li Y</span>, <span class="author">Young ND</span>, <span class="author">Gasser RB</span>, <span class="author">Protasio AV</span>, <span class="author">Berriman M</span>, <span class="author">Roos DS</span>, <span class="author">Kissinger JC</span> and <span class="author">Oliveira G</span></p>
<p class="affiliation">Centro de Excelência em Bioinformática, National Institute for Science and Technology in Tropical Diseases FIOCRUZ-Minas, Belo Horizonte, MG, 30190-002, Laboratório Multiusuário de Bioinformática, Embrapa Informática Agropecuária, Campinas, SP, 13083-970, Brazil, Shanghai Center for Bioinformation Technology, Shanghai, China, School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China, Faculty of Veterinary Science, University of Melbourne, Parkville, Victoria, 3010, Australia, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK, Department of Biology, University of Pennsylvania, PA and Center for Tropical and Emerging Global Diseases, University of Georgia, GA, 30602-7399, USA.</p>
<p class="abstract">The new release of SchistoDB (http://SchistoDB.net) provides a rich resource of genomic data for key blood flukes (genus Schistosoma) which cause disease in hundreds of millions of people worldwide. SchistoDB integrates whole-genome sequence and annotation of three species of the genus and provides enhanced bioinformatics analyses and data-mining tools. A simple, yet comprehensive web interface provided through the Strategies Web Development Kit is available for the mining and visualization of the data. Genomic scale data can be queried based on BLAST searches, annotation keywords and gene ID searches, gene ontology terms, sequence motifs, protein characteristics and phylogenetic relationships. Search strategies can be saved within a user's profile for future retrieval and may also be shared with other researchers using a unique web address.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2012</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/23161692" rel="external">23161692</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgks1087" rel="external">10.1093/nar/gks1087</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multi-isotope imaging mass spectrometry reveals slow protein turnover in hair-cell stereocilia.</h4>
<p class="authors"><span class="author">Zhang DS</span>, <span class="author">Piazza V</span>, <span class="author">Perrin BJ</span>, <span class="author">Rzadzinska AK</span>, <span class="author">Poczatek JC</span>, <span class="author">Wang M</span>, <span class="author">Prosser HM</span>, <span class="author">Ervasti JM</span>, <span class="author">Corey DP</span> and <span class="author">Lechene CP</span></p>
<p class="affiliation">Department of Neurobiology, Harvard Medical School and Howard Hughes Medical Institute, Boston, Massachusetts 02115, USA.</p><p class="abstract">Hair cells of the inner ear are not normally replaced during an animal&#39;s life, and must continually renew components of their various organelles. Among these are the stereocilia, each with a core of several hundred actin filaments that arise from their apical surfaces and that bear the mechanotransduction apparatus at their tips. Actin turnover in stereocilia has previously been studied by transfecting neonatal rat hair cells in culture with a &beta;-actin-GFP fusion, and evidence was found that actin is replaced, from the top down, in 2-3 days. Overexpression of the actin-binding protein espin causes elongation of stereocilia within 12-24 hours, also suggesting rapid regulation of stereocilia lengths. Similarly, the mechanosensory &#39;tip links&#39; are replaced in 5-10 hours after cleavage in chicken and mammalian hair cells. In contrast, turnover in chick stereocilia in vivo is much slower. It might be that only certain components of stereocilia turn over quickly, that rapid turnover occurs only in neonatal animals, only in culture, or only in response to a challenge like breakage or actin overexpression. Here we quantify protein turnover by feeding animals with a (15)N-labelled precursor amino acid and using multi-isotope imaging mass spectrometry to measure appearance of new protein. Surprisingly, in adult frogs and mice and in neonatal mice, in vivo and in vitro, the stereocilia were remarkably stable, incorporating newly synthesized protein at &lt;10% per day. Only stereocilia tips had rapid turnover and no treadmilling was observed. Other methods confirmed this: in hair cells expressing &beta;-actin-GFP we bleached fiducial lines across hair bundles, but they did not move in 6 days. When we stopped expression of &beta;- or &gamma;-actin with tamoxifen-inducible recombination, neither actin isoform left the stereocilia, except at the tips. Thus, rapid turnover in stereocilia occurs only at the tips and not by a treadmilling process.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCRR NIH HHS: <span class="grant">2P41RR0112553-12</span>, <span class="grant">P41RR14579</span>; NEI NIH HHS: <span class="grant">R01EY12963</span>; NIAMS NIH HHS: <span class="grant">R01AR049899</span>; NIBIB NIH HHS: <span class="grant">P41EB001974</span>; NIDCD NIH HHS: <span class="grant">F32DC009539</span>, <span class="grant">R01DC00033</span>, <span class="grant">R01DC02281</span>, <span class="grant">R01DC03463</span>, <span class="grant">R01DC04179</span>; NIDDK NIH HHS: <span class="grant">R37DK39773</span>; NIGMS NIH HHS: <span class="grant">R01GM47214</span>; PHS HHS: <span class="grant">R01D K58762</span>; Wellcome Trust: <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2012</span>;<span class="volume">481</span>;<span class="number">7382</span>;<span class="pagerange">520-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22246323" rel="external">22246323</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3267870" rel="external">3267870</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature10745" rel="external">10.1038/nature10745</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PASSion: a pattern growth algorithm-based pipeline for splice junction detection in paired-end RNA-Seq data.</h4>
<p class="authors"><span class="author">Zhang Y</span>, <span class="author">Lameijer EW</span>, <span class="author">&#39;t Hoen PA</span>, <span class="author">Ning Z</span>, <span class="author">Slagboom PE</span> and <span class="author">Ye K</span></p>
<p class="affiliation">Department of Molecular Epidemiology, Medical Statistics and Bioinformatics, Leiden University Medical Center, Leiden, The Netherlands. <a href="mailto:%79.%7A%68%61%6E%67%40%6C%75%6D%63.%6E%6C">&#121;&#46;&#122;&#104;&#97;&#110;&#103;&#64;&#108;&#117;&#109;&#99;&#46;&#110;&#108;</a></p><p class="abstract"><strong>Motivation</strong>: RNA-seq is a powerful technology for the study of transcriptome profiles that uses deep-sequencing technologies. Moreover, it may be used for cellular phenotyping and help establishing the etiology of diseases characterized by abnormal splicing patterns. In RNA-Seq, the exact nature of splicing events is buried in the reads that span exon-exon boundaries. The accurate and efficient mapping of these reads to the reference genome is a major challenge.</p><p class="abstract"><strong>Results</strong>: We developed PASSion, a pattern growth algorithm-based pipeline for splice site detection in paired-end RNA-Seq reads. Comparing the performance of PASSion to three existing RNA-Seq analysis pipelines, TopHat, MapSplice and HMMSplicer, revealed that PASSion is competitive with these packages. Moreover, the performance of PASSion is not affected by read length and coverage. It performs better than the other three approaches when detecting junctions in highly abundant transcripts. PASSion has the ability to detect junctions that do not have known splicing motifs, which cannot be found by the other tools. Of the two public RNA-Seq datasets, PASSion predicted &asymp; 137,000 and 173,000 splicing events, of which on average 82 are known junctions annotated in the Ensembl transcript database and 18% are novel. In addition, our package can discover differential and shared splicing patterns among multiple samples.</p><p class="abstract"><strong>Availability</strong>: The code and utilities can be freely downloaded from https://trac.nbic.nl/passion and ftp://ftp.sanger.ac.uk/pub/zn1/passion.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2012</span>;<span class="volume">28</span>;<span class="number">4</span>;<span class="pagerange">479-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/22219203" rel="external">22219203</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3278765" rel="external">3278765</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr712" rel="external">10.1093/bioinformatics/btr712</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/qkyw4rdc"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/qkyw4rdc" src="/qr/qkyw4rdc.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:47 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
