<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2010 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2010.html 2831 2011-01-25 13:49:19Z ab6 $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2010</h1><p>Number of papers published in 2010: <strong>438</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A map of human genome variation from population-scale sequencing.</h4>
<p class="authors"><span class="author">1000 Genomes Project Consortium</span>, <span class="author">Abecasis GR</span>, <span class="author">Altshuler D</span>, <span class="author">Auton A</span>, <span class="author">Brooks LD</span>, <span class="author">Durbin RM</span>, <span class="author">Gibbs RA</span>, <span class="author">Hurles ME</span> and <span class="author">McVean GA</span></p><p class="abstract">The 1000 Genomes Project aims to provide a deep characterization of human genome sequence variation as a foundation for investigating the relationship between genotype and phenotype. Here we present results of the pilot phase of the project, designed to develop and compare different strategies for genome-wide sequencing with high-throughput platforms. We undertook three projects: low-coverage whole-genome sequencing of 179 individuals from four populations; high-coverage sequencing of two mother-father-child trios; and exon-targeted sequencing of 697 individuals from seven populations. We describe the location, allele frequency and local haplotype structure of approximately 15 million single nucleotide polymorphisms, 1 million short insertions and deletions, and 20,000 structural variants, most of which were previously undescribed. We show that, because we have catalogued the vast majority of common variation, over 95% of the currently accessible variants found in any individual are present in this data set. On average, each person is found to carry approximately 250 to 300 loss-of-function variants in annotated genes and 50 to 100 variants previously implicated in inherited disorders. We demonstrate how these results can be used to inform association and functional studies. From the two trios, we directly estimate the rate of de novo germline base substitution mutations to be approximately 10(-8) per base pair per generation. We explore the data with regard to signatures of natural selection, and identify a marked reduction of genetic variation in the neighbourhood of genes, due to selection at linked sites. These methods and public data will support the next phase of human genetic research.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">G0801823</span>, <span class="grant">G0801823(89305)</span>; NCRR NIH HHS: <span class="grant">S10RR025056</span>; NHGRI NIH HHS: <span class="grant">01HG3229</span>, <span class="grant">N01HG62088</span>, <span class="grant">P01 HG004120</span>, <span class="grant">P01HG4120</span>, <span class="grant">P41HG2371</span>, <span class="grant">P41HG4221</span>, <span class="grant">P41HG4222</span>, <span class="grant">P50HG2357</span>, <span class="grant">R01 HG003229</span>, <span class="grant">R01 HG003229-05</span>, <span class="grant">R01 HG004719</span>, <span class="grant">R01 HG004719-01</span>, <span class="grant">R01 HG004719-02</span>, <span class="grant">R01 HG004719-02S1</span>, <span class="grant">R01 HG004719-03</span>, <span class="grant">R01 HG004719-04</span>, <span class="grant">R01HG2651</span>, <span class="grant">R01HG3698</span>, <span class="grant">R01HG4333</span>, <span class="grant">R01HG4719</span>, <span class="grant">R01HG4960</span>, <span class="grant">RC2 HG005552</span>, <span class="grant">RC2 HG005552-01</span>, <span class="grant">RC2 HG005552-02</span>, <span class="grant">RC2HG5552</span>, <span class="grant">U01HG5208</span>, <span class="grant">U01HG5209</span>, <span class="grant">U01HG5210</span>, <span class="grant">U01HG5211</span>, <span class="grant">U01HG5214</span>, <span class="grant">U41HG4568</span>, <span class="grant">U54 HG003273</span>, <span class="grant">U54HG2750</span>, <span class="grant">U54HG2757</span>, <span class="grant">U54HG3067</span>, <span class="grant">U54HG3079</span>, <span class="grant">U54HG3273</span>; NIGMS NIH HHS: <span class="grant">R01GM59290</span>, <span class="grant">R01GM72861</span>, <span class="grant">T32 GM007753</span>; NIMH NIH HHS: <span class="grant">01MH84698</span>; Wellcome Trust: <span class="grant">075491</span>, <span class="grant">077009</span>, <span class="grant">077014</span>, <span class="grant">077192</span>, <span class="grant">081407</span>, <span class="grant">085532</span>, <span class="grant">086084</span>, <span class="grant">089061</span>, <span class="grant">089062</span>, <span class="grant">089088</span>, <span class="grant">WT075491/Z/04</span>, <span class="grant">WT077009</span>, <span class="grant">WT081407/Z/06/Z</span>, <span class="grant">WT085532AIA</span>, <span class="grant">WT086084/Z/08/Z</span>, <span class="grant">WT089088/Z/09/Z</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">467</span>;<span class="number">7319</span>;<span class="pagerange">1061-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20981092">20981092</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3042601">3042601</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09534">10.1038/nature09534</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Statistical comparison of methods to estimate the error probability in short-read Illumina sequencing.</h4>
<p class="authors"><span class="author">Abnizova I</span>, <span class="author">Skelly T</span>, <span class="author">Naumenko F</span>, <span class="author">Whiteford N</span>, <span class="author">Brown C</span> and <span class="author">Cox T</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%69%61%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#97;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">As was the case in the beginning of the sequencing era, the new generation of short-read sequencing technologies still requires both accuracy of data processing methods and reliable measures of that accuracy. Inspired by the classic of the genre, the Phred method, we generalized those findings in the area of base quality value calibration. We introduce a simple, straightforward statistically established way to measure the performance of a calibrator, and to find an optimal way to assess its reliability. We illustrate the method by assessing the performance of several calibrators/predictors for Illumina, Genome Analyser 2 (GA2) data. The choice of the best predictor is based on optimization of validity, discriminative ability and discrimination power for several candidate predictors. We applied the method on data from one experimental run for genome of the phage varphiX, and found the best predictor out of ten candidates to be &#39;Purity&#39;, a statistics derived from corrected cluster intensities. The source code for the comparison of the predictors is available from the authors by request.</p>
<p class="publication"><span class="title">Journal of bioinformatics and computational biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">579-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20556863" rel="external">20556863</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic evidence of multiple loci in dystocia--difficult labour.</h4>
<p class="authors"><span class="author">Algovik M</span>, <span class="author">Kivinen K</span>, <span class="author">Peterson H</span>, <span class="author">Westgren M</span> and <span class="author">Kere J</span></p>
<p class="affiliation">Department of Clinical Science, Intervention and Technology, Karolinska Institutet, Stockholm, Sweden. <a href="mailto:%6D%69%63%68%61%65%6C.%61%6C%67%6F%76%69%6B%40%6C%74%6B%61%6C%6D%61%72.%73%65">&#109;&#105;&#99;&#104;&#97;&#101;&#108;&#46;&#97;&#108;&#103;&#111;&#118;&#105;&#107;&#64;&#108;&#116;&#107;&#97;&#108;&#109;&#97;&#114;&#46;&#115;&#101;</a></p><p class="abstract"><strong>Background</strong>: Dystocia, difficult labour, is a common but also complex problem during childbirth. It can be attributed to either weak contractions of the uterus, a large infant, reduced capacity of the pelvis or combinations of these. Previous studies have indicated that there is a genetic component in the susceptibility of experiencing dystocia. The purpose of this study was to identify susceptibility genes in dystocia.</p><p class="abstract"><strong>Methods</strong>: A total of 104 women in 47 families were included where at least two sisters had undergone caesarean section at a gestational length of 286 days or more at their first delivery. Study of medical records and a telephone interview was performed to identify subjects with dystocia. Whole-genome scanning using Affymetrix genotyping-arrays and non-parametric linkage (NPL) analysis was made in 39 women exhibiting the phenotype of dystocia from 19 families. In 68 women re-sequencing was performed of candidate genes showing suggestive linkage: oxytocin (OXT) on chromosome 20 and oxytocin-receptor (OXTR) on chromosome 3.</p><p class="abstract"><strong>Results</strong>: We found a trend towards linkage with suggestive NPL-score (3.15) on chromosome 12p12. Suggestive linkage peaks were observed on chromosomes 3, 4, 6, 10, 20. Re-sequencing of OXT and OXTR did not reveal any causal variants.</p><p class="abstract"><strong>Conclusions</strong>: Dystocia is likely to have a genetic component with variations in multiple genes affecting the patient outcome. We found 6 loci that could be re-evaluated in larger patient cohorts.</p>
<p class="publication"><span class="title">BMC medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">105</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20587075">20587075</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2914646">2914646</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2350-11-105">10.1186/1471-2350-11-105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An insight into the sialome of Glossina morsitans morsitans.</h4>
<p class="authors"><span class="author">Alves-Silva J</span>, <span class="author">Ribeiro JM</span>, <span class="author">Van Den Abbeele J</span>, <span class="author">Attardo G</span>, <span class="author">Hao Z</span>, <span class="author">Haines LR</span>, <span class="author">Soares MB</span>, <span class="author">Berriman M</span>, <span class="author">Aksoy S</span> and <span class="author">Lehane MJ</span></p>
<p class="affiliation">Vector Group, Liverpool School of Tropical Medicine, Liverpool, L3 5QA, UK.</p><p class="abstract"><strong>Background</strong>: Blood feeding evolved independently in worms, arthropods and mammals. Among the adaptations to this peculiar diet, these animals developed an armament of salivary molecules that disarm their host&#39;s anti-bleeding defenses (hemostasis), inflammatory and immune reactions. Recent sialotranscriptome analyses (from the Greek sialo = saliva) of blood feeding insects and ticks have revealed that the saliva contains hundreds of polypeptides, many unique to their genus or family. Adult tsetse flies feed exclusively on vertebrate blood and are important vectors of human and animal diseases. Thus far, only limited information exists regarding the Glossina sialome, or any other fly belonging to the Hippoboscidae.</p><p class="abstract"><strong>Results</strong>: As part of the effort to sequence the genome of Glossina morsitans morsitans, several organ specific, high quality normalized cDNA libraries have been constructed, from which over 20,000 ESTs from an adult salivary gland library were sequenced. These ESTs have been assembled using previously described ESTs from the fat body and midgut libraries of the same fly, thus totaling 62,251 ESTs, which have been assembled into 16,743 clusters (8,506 of which had one or more EST from the salivary gland library). Coding sequences were obtained for 2,509 novel proteins, 1,792 of which had at least one EST expressed in the salivary glands. Despite library normalization, 59 transcripts were overrepresented in the salivary library indicating high levels of expression. This work presents a detailed analysis of the salivary protein families identified. Protein expression was confirmed by 2D gel electrophoresis, enzymatic digestion and mass spectrometry. Concurrently, an initial attempt to determine the immunogenic properties of selected salivary proteins was undertaken.</p><p class="abstract"><strong>Conclusions</strong>: The sialome of G. m. morsitans contains over 250 proteins that are possibly associated with blood feeding. This set includes alleles of previously described gene products, reveals new evidence that several salivary proteins are multigenic and identifies at least seven new polypeptide families unique to Glossina. Most of these proteins have no known function and thus, provide a discovery platform for the identification of novel pharmacologically active compounds, innovative vector-based vaccine targets, and immunological markers of vector exposure.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI51584</span>, <span class="grant">R21 AI076879-01A1</span>, <span class="grant">R21 AI076879-02</span>; NIGMS NIH HHS: <span class="grant">F32 GM077964</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">213</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20353571" rel="external">20353571</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2853526" rel="external">2853526</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-213" rel="external">10.1186/1471-2164-11-213</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data quality control in genetic case-control association studies.</h4>
<p class="authors"><span class="author">Anderson CA</span>, <span class="author">Pettersson FH</span>, <span class="author">Clarke GM</span>, <span class="author">Cardon LR</span>, <span class="author">Morris AP</span> and <span class="author">Zondervan KT</span></p>
<p class="affiliation">Genetic and Genomic Epidemiology Unit, Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <a href="mailto:%63%61%72%6C.%61%6E%64%65%72%73%6F%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#114;&#108;&#46;&#97;&#110;&#100;&#101;&#114;&#115;&#111;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This protocol details the steps for data quality assessment and control that are typically carried out during case-control association studies. The steps described involve the identification and removal of DNA samples and markers that introduce bias. These critical steps are paramount to the success of a case-control study and are necessary before statistically testing for association. We describe how to use PLINK, a tool for handling SNP data, to perform assessments of failure rate per individual and per SNP and to assess the degree of relatedness between individuals. We also detail other quality-control procedures, including the use of SMARTPCA software for the identification of ancestral outliers. These platforms were selected because they are user-friendly, widely used and computationally efficient. Steps needed to detect and establish a disease association using case-control data are not discussed here. Issues concerning study design and marker selection in case-control studies have been discussed in our earlier protocols. This protocol, which is routinely used in our labs, should take approximately 8 h to complete.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">081682</span>, <span class="grant">085235</span>, <span class="grant">WT91745/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">9</span>;<span class="pagerange">1564-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21085122" rel="external">21085122</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3025522" rel="external">3025522</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2010.116" rel="external">10.1038/nprot.2010.116</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data quality control in genetic case-control association studies</h4>
<p class="authors"><span class="author">ANDERSON CA</span>, <span class="author">Pettersson FH</span>, <span class="author">Clarke GM</span>, <span class="author">Cardon LR</span>, <span class="author">Morris AP</span>, <span class="author">Zondervan KT</span></p>
<p class="publication"><span class="title">Nature Protocols.</span> <span class="year">2010</span>;<span class="volume">5</span>;<span class="pagerange">1564-73</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of migraine implicates a common susceptibility variant on 8q22.1.</h4>
<p class="authors"><span class="author">Anttila V</span>, <span class="author">Stefansson H</span>, <span class="author">Kallela M</span>, <span class="author">Todt U</span>, <span class="author">Terwindt GM</span>, <span class="author">Calafato MS</span>, <span class="author">Nyholt DR</span>, <span class="author">Dimas AS</span>, <span class="author">Freilinger T</span>, <span class="author">M&uuml;ller-Myhsok B</span>, <span class="author">Artto V</span>, <span class="author">Inouye M</span>, <span class="author">Alakurtti K</span>, <span class="author">Kaunisto MA</span>, <span class="author">H&auml;m&auml;l&auml;inen E</span>, <span class="author">de Vries B</span>, <span class="author">Stam AH</span>, <span class="author">Weller CM</span>, <span class="author">Heinze A</span>, <span class="author">Heinze-Kuhn K</span>, <span class="author">Goebel I</span>, <span class="author">Borck G</span>, <span class="author">G&ouml;bel H</span>, <span class="author">Steinberg S</span>, <span class="author">Wolf C</span>, <span class="author">Bj&ouml;rnsson A</span>, <span class="author">Gudmundsson G</span>, <span class="author">Kirchmann M</span>, <span class="author">Hauge A</span>, <span class="author">Werge T</span>, <span class="author">Schoenen J</span>, <span class="author">Eriksson JG</span>, <span class="author">Hagen K</span>, <span class="author">Stovner L</span>, <span class="author">Wichmann HE</span>, <span class="author">Meitinger T</span>, <span class="author">Alexander M</span>, <span class="author">Moebus S</span>, <span class="author">Schreiber S</span>, <span class="author">Aulchenko YS</span>, <span class="author">Breteler MM</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">van Duijn CM</span>, <span class="author">Tikka-Kleemola P</span>, <span class="author">Veps&auml;l&auml;inen S</span>, <span class="author">Lucae S</span>, <span class="author">Tozzi F</span>, <span class="author">Muglia P</span>, <span class="author">Barrett J</span>, <span class="author">Kaprio J</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Peltonen L</span>, <span class="author">Stefansson K</span>, <span class="author">Zwart JA</span>, <span class="author">Ferrari MD</span>, <span class="author">Olesen J</span>, <span class="author">Daly M</span>, <span class="author">Wessman M</span>, <span class="author">van den Maagdenberg AM</span>, <span class="author">Dichgans M</span>, <span class="author">Kubisch C</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Frants RR</span>, <span class="author">Palotie A</span> and <span class="author">International Headache Genetics Consortium</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%76%65%72%6E%65%72%69.%61%6E%74%74%69%6C%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#118;&#101;&#114;&#110;&#101;&#114;&#105;&#46;&#97;&#110;&#116;&#116;&#105;&#108;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Migraine is a common episodic neurological disorder, typically presenting with recurrent attacks of severe headache and autonomic dysfunction. Apart from rare monogenic subtypes, no genetic or molecular markers for migraine have been convincingly established. We identified the minor allele of rs1835740 on chromosome 8q22.1 to be associated with migraine (P = 5.38 &times; 10&#x207B;&#x2079;, odds ratio = 1.23, 95% CI 1.150-1.324) in a genome-wide association study of 2,731 migraine cases ascertained from three European headache clinics and 10,747 population-matched controls. The association was replicated in 3,202 cases and 40,062 controls for an overall meta-analysis P value of 1.69 &times; 10&#x207B;&sup1;&sup1; (odds ratio = 1.18, 95% CI 1.127-1.244). rs1835740 is located between MTDH (astrocyte elevated gene 1, also known as AEG-1) and PGCP (encoding plasma glutamate carboxypeptidase). In an expression quantitative trait study in lymphoblastoid cell lines, transcript levels of the MTDH were found to have a significant correlation to rs1835740 (P = 3.96 &times; 10&#x207B;&#x2075;, permuted threshold for genome-wide significance 7.7 &times; 10&#x207B;&#x2075;. To our knowledge, our data establish rs1835740 as the first genetic risk factor for migraine.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089062</span>, <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">10</span>;<span class="pagerange">869-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20802479">20802479</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2948563">2948563</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.652">10.1038/ng.652</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The SH3 domain of postsynaptic density 95 mediates inflammatory pain through phosphatidylinositol-3-kinase recruitment.</h4>
<p class="authors"><span class="author">Arbuckle MI</span>, <span class="author">Komiyama NH</span>, <span class="author">Delaney A</span>, <span class="author">Coba M</span>, <span class="author">Garry EM</span>, <span class="author">Rosie R</span>, <span class="author">Allchorne AJ</span>, <span class="author">Forsyth LH</span>, <span class="author">Bence M</span>, <span class="author">Carlisle HJ</span>, <span class="author">O&#39;Dell TJ</span>, <span class="author">Mitchell R</span>, <span class="author">Fleetwood-Walker SM</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Centre for Neuroregeneration, The University of Edinburgh, Institute of Immunology and Infection, Ashworth Buildings, Kings Buildings, Edinburgh EH9 3JT, UK.</p><p class="abstract">Sensitization to inflammatory pain is a pathological form of neuronal plasticity that is poorly understood and treated. Here we examine the role of the SH3 domain of postsynaptic density 95 (PSD95) by using mice that carry a single amino-acid substitution in the polyproline-binding site. Testing multiple forms of plasticity we found sensitization to inflammation was specifically attenuated. The inflammatory response required recruitment of phosphatidylinositol-3-kinase-C2alpha to the SH3-binding site of PSD95. In wild-type mice, wortmannin or peptide competition attenuated the sensitization. These results show that different types of behavioural plasticity are mediated by specific domains of PSD95 and suggest novel therapeutic avenues for reducing inflammatory pain.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">EMBO reports</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">6</span>;<span class="pagerange">473-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20467438" rel="external">20467438</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2892321" rel="external">2892321</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fembor.2010.63" rel="external">10.1038/embor.2010.63</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pig genome sequence--analysis and publication strategy.</h4>
<p class="authors"><span class="author">Archibald AL</span>, <span class="author">Bolund L</span>, <span class="author">Churcher C</span>, <span class="author">Fredholm M</span>, <span class="author">Groenen MA</span>, <span class="author">Harlizius B</span>, <span class="author">Lee KT</span>, <span class="author">Milan D</span>, <span class="author">Rogers J</span>, <span class="author">Rothschild MF</span>, <span class="author">Uenishi H</span>, <span class="author">Wang J</span>, <span class="author">Schook LB</span> and <span class="author">Swine Genome Sequencing Consortium</span></p><p class="abstract"><strong>Background</strong>: The pig genome is being sequenced and characterised under the auspices of the Swine Genome Sequencing Consortium. The sequencing strategy followed a hybrid approach combining hierarchical shotgun sequencing of BAC clones and whole genome shotgun sequencing.</p><p class="abstract"><strong>Results</strong>: Assemblies of the BAC clone derived genome sequence have been annotated using the Pre-Ensembl and Ensembl automated pipelines and made accessible through the Pre-Ensembl/Ensembl browsers. The current annotated genome assembly (Sscrofa9) was released with Ensembl 56 in September 2009. A revised assembly (Sscrofa10) is under construction and will incorporate whole genome shotgun sequence (WGS) data providing &gt; 30x genome coverage. The WGS sequence, most of which comprise short Illumina/Solexa reads, were generated from DNA from the same single Duroc sow as the source of the BAC library from which clones were preferentially selected for sequencing. In accordance with the Bermuda and Fort Lauderdale agreements and the more recent Toronto Statement the data have been released into public sequence repositories (Genbank/EMBL, NCBI/Ensembl trace repositories) in a timely manner and in advance of publication.</p><p class="abstract"><strong>Conclusions</strong>: In this marker paper, the Swine Genome Sequencing Consortium (SGSC) sets outs its plans for analysis of the pig genome sequence, for the application and publication of the results.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">438</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20642822" rel="external">20642822</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3017778" rel="external">3017778</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-438" rel="external">10.1186/1471-2164-11-438</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare variant association analysis methods for complex traits.</h4>
<p class="authors"><span class="author">Asimit J</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1SA, United Kingdom.</p><p class="abstract">There has been increasing interest in rare variants and their association with disease, and several rare variant-disease associations have already been detected. The usual association tests for common variants are underpowered for detecting variants of lower frequency, so alternative approaches are required. In addition to reviewing the association analysis methods for rare variants, we discuss the limitations of genome-wide association studies in identifying rare variants and the problems that arise in the imputation of rare variants.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Annual review of genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">44</span>;<span class="pagerange">293-308</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21047260">21047260</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev-genet-102209-163421">10.1146/annurev-genet-102209-163421</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TriTrypDB: a functional genomic resource for the Trypanosomatidae.</h4>
<p class="authors"><span class="author">Aslett M</span>, <span class="author">Aurrecoechea C</span>, <span class="author">Berriman M</span>, <span class="author">Brestelli J</span>, <span class="author">Brunk BP</span>, <span class="author">Carrington M</span>, <span class="author">Depledge DP</span>, <span class="author">Fischer S</span>, <span class="author">Gajria B</span>, <span class="author">Gao X</span>, <span class="author">Gardner MJ</span>, <span class="author">Gingle A</span>, <span class="author">Grant G</span>, <span class="author">Harb OS</span>, <span class="author">Heiges M</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Houston R</span>, <span class="author">Innamorato F</span>, <span class="author">Iodice J</span>, <span class="author">Kissinger JC</span>, <span class="author">Kraemer E</span>, <span class="author">Li W</span>, <span class="author">Logan FJ</span>, <span class="author">Miller JA</span>, <span class="author">Mitra S</span>, <span class="author">Myler PJ</span>, <span class="author">Nayak V</span>, <span class="author">Pennington C</span>, <span class="author">Phan I</span>, <span class="author">Pinney DF</span>, <span class="author">Ramasamy G</span>, <span class="author">Rogers MB</span>, <span class="author">Roos DS</span>, <span class="author">Ross C</span>, <span class="author">Sivam D</span>, <span class="author">Smith DF</span>, <span class="author">Srinivasamoorthy G</span>, <span class="author">Stoeckert CJ</span>, <span class="author">Subramanian S</span>, <span class="author">Thibodeau R</span>, <span class="author">Tivey A</span>, <span class="author">Treatman C</span>, <span class="author">Velarde G</span> and <span class="author">Wang H</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">TriTrypDB (http://tritrypdb.org) is an integrated database providing access to genome-scale datasets for kinetoplastid parasites, and supporting a variety of complex queries driven by research and development needs. TriTrypDB is a collaborative project, utilizing the GUS/WDK computational infrastructure developed by the Eukaryotic Pathogen Bioinformatics Resource Center (EuPathDB.org) to integrate genome annotation and analyses from GeneDB and elsewhere with a wide variety of functional genomics datasets made available by members of the global research community, often pre-publication. Currently, TriTrypDB integrates datasets from Leishmania braziliensis, L. infantum, L. major, L. tarentolae, Trypanosoma brucei and T. cruzi. Users may examine individual genes or chromosomal spans in their genomic context, including syntenic alignments with other kinetoplastid organisms. Data within TriTrypDB can be interrogated utilizing a sophisticated search strategy system that enables a user to construct complex queries combining multiple data types. All search strategies are stored, allowing future access and integrated searches. &#39;User Comments&#39; may be added to any gene page, enhancing available annotation; such comments become immediately searchable via the text search, and are forwarded to curators for incorporation into the reference annotation when appropriate.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">HHSN266200400037C</span>; Wellcome Trust: <span class="grant">085822</span>, <span class="grant">WT085775</span>, <span class="grant">WT085775/Z/08/Z</span>, <span class="grant">WT085822MA</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D457-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19843604" rel="external">19843604</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808979" rel="external">2808979</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp851" rel="external">10.1093/nar/gkp851</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lack of association between the Trp719Arg polymorphism in kinesin-like protein-6 and coronary artery disease in 19 case-control studies.</h4>
<p class="authors"><span class="author">Assimes TL</span>, <span class="author">H&oacute;lm H</span>, <span class="author">Kathiresan S</span>, <span class="author">Reilly MP</span>, <span class="author">Thorleifsson G</span>, <span class="author">Voight BF</span>, <span class="author">Erdmann J</span>, <span class="author">Willenborg C</span>, <span class="author">Vaidya D</span>, <span class="author">Xie C</span>, <span class="author">Patterson CC</span>, <span class="author">Morgan TM</span>, <span class="author">Burnett MS</span>, <span class="author">Li M</span>, <span class="author">Hlatky MA</span>, <span class="author">Knowles JW</span>, <span class="author">Thompson JR</span>, <span class="author">Absher D</span>, <span class="author">Iribarren C</span>, <span class="author">Go A</span>, <span class="author">Fortmann SP</span>, <span class="author">Sidney S</span>, <span class="author">Risch N</span>, <span class="author">Tang H</span>, <span class="author">Myers RM</span>, <span class="author">Berger K</span>, <span class="author">Stoll M</span>, <span class="author">Shah SH</span>, <span class="author">Thorgeirsson G</span>, <span class="author">Andersen K</span>, <span class="author">Havulinna AS</span>, <span class="author">Herrera JE</span>, <span class="author">Faraday N</span>, <span class="author">Kim Y</span>, <span class="author">Kral BG</span>, <span class="author">Mathias RA</span>, <span class="author">Ruczinski I</span>, <span class="author">Suktitipat B</span>, <span class="author">Wilson AF</span>, <span class="author">Yanek LR</span>, <span class="author">Becker LC</span>, <span class="author">Linsel-Nitschke P</span>, <span class="author">Lieb W</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Hengstenberg C</span>, <span class="author">Fischer M</span>, <span class="author">Stark K</span>, <span class="author">Reinhard W</span>, <span class="author">Winogradow J</span>, <span class="author">Grassl M</span>, <span class="author">Grosshennig A</span>, <span class="author">Preuss M</span>, <span class="author">Schreiber S</span>, <span class="author">Wichmann HE</span>, <span class="author">Meisinger C</span>, <span class="author">Yee J</span>, <span class="author">Friedlander Y</span>, <span class="author">Do R</span>, <span class="author">Meigs JB</span>, <span class="author">Williams G</span>, <span class="author">Nathan DM</span>, <span class="author">MacRae CA</span>, <span class="author">Qu L</span>, <span class="author">Wilensky RL</span>, <span class="author">Matthai WH</span>, <span class="author">Qasim AN</span>, <span class="author">Hakonarson H</span>, <span class="author">Pichard AD</span>, <span class="author">Kent KM</span>, <span class="author">Satler L</span>, <span class="author">Lindsay JM</span>, <span class="author">Waksman R</span>, <span class="author">Knouff CW</span>, <span class="author">Waterworth DM</span>, <span class="author">Walker MC</span>, <span class="author">Mooser VE</span>, <span class="author">Marrugat J</span>, <span class="author">Lucas G</span>, <span class="author">Subirana I</span>, <span class="author">Sala J</span>, <span class="author">Ramos R</span>, <span class="author">Martinelli N</span>, <span class="author">Olivieri O</span>, <span class="author">Trabetti E</span>, <span class="author">Malerba G</span>, <span class="author">Pignatti PF</span>, <span class="author">Guiducci C</span>, <span class="author">Mirel D</span>, <span class="author">Parkin M</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Asselta R</span>, <span class="author">Duga S</span>, <span class="author">Musunuru K</span>, <span class="author">Daly MJ</span>, <span class="author">Purcell S</span>, <span class="author">Eifert S</span>, <span class="author">Braund PS</span>, <span class="author">Wright BJ</span>, <span class="author">Balmforth AJ</span>, <span class="author">Ball SG</span>, <span class="author">Myocardial Infarction Genetics Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Cardiogenics</span>, <span class="author">Ouwehand WH</span>, <span class="author">Deloukas P</span>, <span class="author">Scholz M</span>, <span class="author">Cambien F</span>, <span class="author">Huge A</span>, <span class="author">Scheffold T</span>, <span class="author">Salomaa V</span>, <span class="author">Girelli D</span>, <span class="author">Granger CB</span>, <span class="author">Peltonen L</span>, <span class="author">McKeown PP</span>, <span class="author">Altshuler D</span>, <span class="author">Melander O</span>, <span class="author">Devaney JM</span>, <span class="author">Epstein SE</span>, <span class="author">Rader DJ</span>, <span class="author">Elosua R</span>, <span class="author">Engert JC</span>, <span class="author">Anand SS</span>, <span class="author">Hall AS</span>, <span class="author">Ziegler A</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Spertus JA</span>, <span class="author">Siscovick D</span>, <span class="author">Schwartz SM</span>, <span class="author">Becker D</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">Schunkert H</span>, <span class="author">Samani NJ</span> and <span class="author">Quertermous T</span></p>
<p class="affiliation">Department of Medicine, Stanford University School of Medicine, Stanford, California 94304-1334, USA. <a href="mailto:%74%61%73%73%69%6D%65%73%40%73%74%61%6E%66%6F%72%64.%65%64%75">&#116;&#97;&#115;&#115;&#105;&#109;&#101;&#115;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Objectives</strong>: We sought to replicate the association between the kinesin-like protein 6 (KIF6) Trp719Arg polymorphism (rs20455), and clinical coronary artery disease (CAD).</p><p class="abstract"><strong>Background</strong>: Recent prospective studies suggest that carriers of the 719Arg allele in KIF6 are at increased risk of clinical CAD compared with noncarriers.</p><p class="abstract"><strong>Methods</strong>: The KIF6 Trp719Arg polymorphism (rs20455) was genotyped in 19 case-control studies of nonfatal CAD either as part of a genome-wide association study or in a formal attempt to replicate the initial positive reports.</p><p class="abstract"><strong>Results</strong>: A total of 17,000 cases and 39,369 controls of European descent as well as a modest number of South Asians, African Americans, Hispanics, East Asians, and admixed cases and controls were successfully genotyped. None of the 19 studies demonstrated an increased risk of CAD in carriers of the 719Arg allele compared with noncarriers. Regression analyses and fixed-effects meta-analyses ruled out with high degree of confidence an increase of &ge;2% in the risk of CAD among European 719Arg carriers. We also observed no increase in the risk of CAD among 719Arg carriers in the subset of Europeans with early-onset disease (younger than 50 years of age for men and younger than 60 years of age for women) compared with similarly aged controls as well as all non-European subgroups.</p><p class="abstract"><strong>Conclusions</strong>: The KIF6 Trp719Arg polymorphism was not associated with the risk of clinical CAD in this large replication study.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01 HL056931-04</span>, <span class="grant">R01 HL087676-03</span></p>
<p class="publication"><span class="title">Journal of the American College of Cardiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">56</span>;<span class="number">19</span>;<span class="pagerange">1552-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20933357" rel="external">20933357</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3084526" rel="external">3084526</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jacc.2010.06.022" rel="external">10.1016/j.jacc.2010.06.022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene variants associated with schizophrenia in a Norwegian genome-wide study are replicated in a large European cohort.</h4>
<p class="authors"><span class="author">Athanasiu L</span>, <span class="author">Mattingsdal M</span>, <span class="author">K&auml;hler AK</span>, <span class="author">Brown A</span>, <span class="author">Gustafsson O</span>, <span class="author">Agartz I</span>, <span class="author">Giegling I</span>, <span class="author">Muglia P</span>, <span class="author">Cichon S</span>, <span class="author">Rietschel M</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Peltonen L</span>, <span class="author">Bramon E</span>, <span class="author">Collier D</span>, <span class="author">Clair DS</span>, <span class="author">Sigurdsson E</span>, <span class="author">Petursson H</span>, <span class="author">Rujescu D</span>, <span class="author">Melle I</span>, <span class="author">Steen VM</span>, <span class="author">Djurovic S</span> and <span class="author">Andreassen OA</span></p>
<p class="affiliation">Institute of Psychiatry, University of Oslo, P.O. 1130, Blindern, N-0318 Oslo, Norway</p><p class="abstract">We have performed a genome-wide association study (GWAS) of schizophrenia in a Norwegian discovery sample of 201 cases and 305 controls (TOP study) with a focused replication analysis in a larger European sample of 2663 cases and 13,780 control subjects (SGENE-plus study). Firstly, the discovery sample was genotyped with Affymetrix Genome-Wide Human SNP Array 6.0 and 572,888 markers were tested for schizophrenia association. No SNPs in the discovery sample attained genome-wide significance (P&lt;8.7 x 10(-8)). Secondly, based on the GWAS data, we selected 1000 markers with the lowest P values in the discovery TOP sample, and tested these (or HapMap-based surrogates) for association in the replication sample. Sixteen loci were associated with schizophrenia (nominal P value&lt;0.05 and concurring OR) in the replication sample. As a next step, we performed a combined analysis of the findings from these two studies, and the strongest evidence for association with schizophrenia was provided for markers rs7045881 on 9p21, rs433598 on 16p12 and rs10761482 on 10q21. The markers are located in PLAA, ACSM1 and ANK3, respectively. PLAA has not previously been described as a susceptibility gene, but 9p21 is implied as a schizophrenia linkage region. ACSM1 has been identified as a susceptibility gene in a previous schizophrenia GWAS study. The association of ANK3 with schizophrenia is intriguing in light of recent associations of ANK3 with bipolar disorder, thereby supporting the hypothesis of an overlap in genetic susceptibility between these psychopathological entities.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Journal of psychiatric research</span>&nbsp;<span class="year">2010</span>;<span class="volume">44</span>;<span class="number">12</span>;<span class="pagerange">748-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20185149" rel="external">20185149</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3224994" rel="external">3224994</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jpsychires.2010.02.002" rel="external">10.1016/j.jpsychires.2010.02.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptome analysis of reproductive tissue and intrauterine developmental stages of the tsetse fly (Glossina morsitans morsitans).</h4>
<p class="authors"><span class="author">Attardo GM</span>, <span class="author">Ribeiro JM</span>, <span class="author">Wu Y</span>, <span class="author">Berriman M</span> and <span class="author">Aksoy S</span></p>
<p class="affiliation">Department of Epidemiology and Public Health, Yale University School of Medicine, New Haven, CT, USA.</p><p class="abstract"><strong>Background</strong>: Tsetse flies, vectors of African trypanosomes, undergo viviparous reproduction (the deposition of live offspring). This reproductive strategy results in a large maternal investment and the deposition of a small number of progeny during a female&#39;s lifespan. The reproductive biology of tsetse has been studied on a physiological level; however the molecular analysis of tsetse reproduction requires deeper investigation. To build a foundation from which to base molecular studies of tsetse reproduction, a cDNA library was generated from female tsetse (Glossina morsitans morsitans) reproductive tissues and the intrauterine developmental stages. 3438 expressed sequence tags were sequenced and analyzed.</p><p class="abstract"><strong>Results</strong>: Analysis of a nonredundant catalogue of 1391 contigs resulted in 520 predicted proteins. 475 of these proteins were full length. We predict that 412 of these represent cytoplasmic proteins while 57 are secreted. Comparison of these proteins with other tissue specific tsetse cDNA libraries (salivary gland, fat body/milk gland, and midgut) identified 51 that are unique to the reproductive/immature cDNA library. 11 unique proteins were homologous to uncharacterized putative proteins within the NR database suggesting the identification of novel genes associated with reproductive functions in other insects (hypothetical conserved). The analysis also yielded seven putative proteins without significant homology to sequences present in the public database (unknown genes). These proteins may represent unique functions associated with tsetse&#39;s viviparous reproductive cycle. RT-PCR analysis of hypothetical conserved and unknown contigs was performed to determine basic tissue and stage specificity of the expression of these genes.</p><p class="abstract"><strong>Conclusion</strong>: This paper identifies 51 putative proteins specific to a tsetse reproductive/immature EST library. 11 of these proteins correspond to hypothetical conserved genes and 7 proteins are tsetse specific.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI081774</span>, <span class="grant">AI51584</span>, <span class="grant">R01 AI081774-02</span>, <span class="grant">R01 AI081774-03</span>, <span class="grant">R01 AI081774-04</span>, <span class="grant">R21 AI076879-01A1</span>, <span class="grant">R21 AI076879-02</span>; NIGMS NIH HHS: <span class="grant">F32 GM077964</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">160</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20214793" rel="external">20214793</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2846916" rel="external">2846916</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-160" rel="external">10.1186/1471-2164-11-160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Further evidence supporting a role for gs signal transduction in severe malaria pathogenesis.</h4>
<p class="authors"><span class="author">Auburn S</span>, <span class="author">Fry AE</span>, <span class="author">Clark TG</span>, <span class="author">Campino S</span>, <span class="author">Diakite M</span>, <span class="author">Green A</span>, <span class="author">Richardson A</span>, <span class="author">Jallow M</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Pinder M</span>, <span class="author">Molyneux ME</span>, <span class="author">Taylor TE</span>, <span class="author">Haldar K</span>, <span class="author">Rockett KA</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Oxford, United Kingdom. <a href="mailto:%73%61%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#97;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">With the functional demonstration of a role in erythrocyte invasion by Plasmodium falciparum parasites, implications in the aetiology of common conditions that prevail in individuals of African origin, and a wealth of pharmacological knowledge, the stimulatory G protein (Gs) signal transduction pathway presents an exciting target for anti-malarial drug intervention. Having previously demonstrated a role for the G-alpha-s gene, GNAS, in severe malaria disease, we sought to identify other important components of the Gs pathway. Using meta-analysis across case-control and family trio (affected child and parental controls) studies of severe malaria from The Gambia and Malawi, we sought evidence of association in six Gs pathway candidate genes: adenosine receptor 2A (ADORA2A) and 2B (ADORA2B), beta-adrenergic receptor kinase 1 (ADRBK1), adenylyl cyclase 9 (ADCY9), G protein beta subunit 3 (GNB3), and regulator of G protein signalling 2 (RGS2). Our study amassed a total of 2278 cases and 2364 controls. Allele-based models of association were investigated in all genes, and genotype and haplotype-based models were investigated where significant allelic associations were identified. Although no significant associations were observed in the other genes, several were identified in ADORA2A. The most significant association was observed at the rs9624472 locus, where the G allele (approximately 20% frequency) appeared to confer enhanced risk to severe malaria [OR = 1.22 (1.09-1.37); P = 0.001]. Further investigation of the ADORA2A gene region is required to validate the associations identified here, and to identify and functionally characterize the responsible causal variant(s). Our results provide further evidence supporting a role of the Gs signal transduction pathway in the regulation of severe malaria, and request further exploration of this pathway in future studies.</p>
<p class="grants">Funded by: Medical Research Council; NIAID NIH HHS: <span class="grant">5R01AI034969-09</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">e10017</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20386734" rel="external">20386734</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2850389" rel="external">2850389</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0010017" rel="external">10.1371/journal.pone.0010017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of ankylosing spondylitis identifies non-MHC susceptibility loci.</h4>
<p class="authors"><span class="author">Australo-Anglo-American Spondyloarthritis Consortium (TASC)</span>, <span class="author">Reveille JD</span>, <span class="author">Sims AM</span>, <span class="author">Danoy P</span>, <span class="author">Evans DM</span>, <span class="author">Leo P</span>, <span class="author">Pointon JJ</span>, <span class="author">Jin R</span>, <span class="author">Zhou X</span>, <span class="author">Bradbury LA</span>, <span class="author">Appleton LH</span>, <span class="author">Davis JC</span>, <span class="author">Diekman L</span>, <span class="author">Doan T</span>, <span class="author">Dowling A</span>, <span class="author">Duan R</span>, <span class="author">Duncan EL</span>, <span class="author">Farrar C</span>, <span class="author">Hadler J</span>, <span class="author">Harvey D</span>, <span class="author">Karaderi T</span>, <span class="author">Mogg R</span>, <span class="author">Pomeroy E</span>, <span class="author">Pryce K</span>, <span class="author">Taylor J</span>, <span class="author">Savage L</span>, <span class="author">Deloukas P</span>, <span class="author">Kumanduri V</span>, <span class="author">Peltonen L</span>, <span class="author">Ring SM</span>, <span class="author">Whittaker P</span>, <span class="author">Glazov E</span>, <span class="author">Thomas GP</span>, <span class="author">Maksymowych WP</span>, <span class="author">Inman RD</span>, <span class="author">Ward MM</span>, <span class="author">Stone MA</span>, <span class="author">Weisman MH</span>, <span class="author">Wordsworth BP</span> and <span class="author">Brown MA</span></p>
<p class="affiliation">University of Texas Health Science Center at Houston, USA.</p><p class="abstract">To identify susceptibility loci for ankylosing spondylitis, we undertook a genome-wide association study in 2,053 unrelated ankylosing spondylitis cases among people of European descent and 5,140 ethnically matched controls, with replication in an independent cohort of 898 ankylosing spondylitis cases and 1,518 controls. Cases were genotyped with Illumina HumHap370 genotyping chips. In addition to strong association with the major histocompatibility complex (MHC; P &lt; 10(-800)), we found association with SNPs in two gene deserts at 2p15 (rs10865331; combined P = 1.9 x 10(-19)) and 21q22 (rs2242944; P = 8.3 x 10(-20)), as well as in the genes ANTXR2 (rs4333130; P = 9.3 x 10(-8)) and IL1R2 (rs2310173; P = 4.8 x 10(-7)). We also replicated previously reported associations at IL23R (rs11209026; P = 9.1 x 10(-14)) and ERAP1 (rs27434; P = 5.3 x 10(-12)). This study reports four genetic loci associated with ankylosing spondylitis risk and identifies a major role for the interleukin (IL)-23 and IL-1 cytokine pathways in disease susceptibility.</p>
<p class="grants">Funded by: Arthritis Research UK; Medical Research Council; NCRR NIH HHS: <span class="grant">MO1-RR00425</span>, <span class="grant">UL1RR024188</span>; NIAMS NIH HHS: <span class="grant">R01-AR046208</span>; PHS HHS: <span class="grant">P01-052915</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">2</span>;<span class="pagerange">123-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20062062" rel="external">20062062</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3224997" rel="external">3224997</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.513" rel="external">10.1038/ng.513</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RamA, a member of the AraC/XylS family, influences both virulence and efflux in Salmonella enterica serovar Typhimurium.</h4>
<p class="authors"><span class="author">Bailey AM</span>, <span class="author">Ivens A</span>, <span class="author">Kingsley R</span>, <span class="author">Cottell JL</span>, <span class="author">Wain J</span> and <span class="author">Piddock LJ</span></p>
<p class="affiliation">Antimicrobial Agents Research Group, Department of Immunity and Infection, The Medical School, The University of Birmingham, Birmingham, B15 2TT, United Kingdom.</p><p class="abstract">The transcriptomes of Salmonella enterica serovar Typhimurium SL1344 lacking a functional ramA or ramR or with plasmid-mediated high-level overexpression of ramA were compared to those of the wild-type parental strain. Inactivation of ramA led to increased expression of 14 SPI-1 genes and decreased expression of three SPI-2 genes, and it altered expression of ribosomal biosynthetic genes and several amino acid biosynthetic pathways. Furthermore, disruption of ramA led to decreased survival within RAW 264.7 mouse macrophages and attenuation within the BALB/c ByJ mouse model. Highly overexpressed ramA led to increased expression of genes encoding multidrug resistance (MDR) efflux pumps, including acrAB, acrEF, and tolC. Decreased expression of 34 Salmonella pathogenicity island (SPI) 1 and 2 genes, decreased SipC production, decreased adhesion to and survival within macrophages, and decreased colonization of Caenorhabditis elegans were also seen. Disruption of ramR led to the increased expression of ramA, acrAB, and tolC, but not to the same level as when ramA was overexpressed on a plasmid. Inactivation of ramR had a more limited effect on pathogenicity gene expression. In silico analysis of a suggested RamA-binding consensus sequence identified target genes, including ramR, acrA, tolC, sipABC, and ssrA. This study demonstrates that the regulation of a mechanism of MDR and expression of virulence genes show considerable overlap, and we postulate that such a mechanism is dependent on transcriptional activator concentration and promoter sensitivity. However, we have no evidence to support the hypothesis that increased MDR via RamA regulation of AcrAB-TolC gives rise to a hypervirulent strain.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">GO501415</span>, <span class="grant">GO801977</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">6</span>;<span class="pagerange">1607-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20081028" rel="external">20081028</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2832520" rel="external">2832520</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01517-09" rel="external">10.1128/JB.01517-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Searching for the elusive typhoid diagnostic.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Favorov M</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam. <a href="mailto:%73%62%61%6B%65%72%40%6F%75%63%72%75.%6F%72%67">&#115;&#98;&#97;&#107;&#101;&#114;&#64;&#111;&#117;&#99;&#114;&#117;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Typhoid (enteric) fever is still a common disease in many developing countries but current diagnostic tests are inadequate. Studies on pathogenesis and genomics have provided new insight into the organisms that cause enteric fever. Better understanding of the microorganisms explains, in part, why our current typhoid methodologies are limited in their diagnostic information and why developing new strategies may be a considerable challenge. Here we discuss the current position of typhoid diagnostics, highlight the need for technological improvements and suggest potential ways of advancing this area.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC infectious diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">10</span>;<span class="pagerange">45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20205702" rel="external">20205702</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2846943" rel="external">2846943</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2334-10-45" rel="external">10.1186/1471-2334-10-45</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The structure of Jann_2411 (DUF1470) from Jannaschia sp. at 1.45&#x2005;&Aring; resolution reveals a new fold (the ABATE domain) and suggests its possible role as a transcription regulator.</h4>
<p class="authors"><span class="author">Bakolitsa C</span>, <span class="author">Bateman A</span>, <span class="author">Jin KK</span>, <span class="author">McMullan D</span>, <span class="author">Krishna SS</span>, <span class="author">Miller MD</span>, <span class="author">Abdubek P</span>, <span class="author">Acosta C</span>, <span class="author">Astakhova T</span>, <span class="author">Axelrod HL</span>, <span class="author">Burra P</span>, <span class="author">Carlton D</span>, <span class="author">Chiu HJ</span>, <span class="author">Clayton T</span>, <span class="author">Das D</span>, <span class="author">Deller MC</span>, <span class="author">Duan L</span>, <span class="author">Elias Y</span>, <span class="author">Feuerhelm J</span>, <span class="author">Grant JC</span>, <span class="author">Grzechnik A</span>, <span class="author">Grzechnik SK</span>, <span class="author">Han GW</span>, <span class="author">Jaroszewski L</span>, <span class="author">Klock HE</span>, <span class="author">Knuth MW</span>, <span class="author">Kozbial P</span>, <span class="author">Kumar A</span>, <span class="author">Marciano D</span>, <span class="author">Morse AT</span>, <span class="author">Murphy KD</span>, <span class="author">Nigoghossian E</span>, <span class="author">Okach L</span>, <span class="author">Oommachen S</span>, <span class="author">Paulsen J</span>, <span class="author">Reyes R</span>, <span class="author">Rife CL</span>, <span class="author">Sefcovic N</span>, <span class="author">Tien H</span>, <span class="author">Trame CB</span>, <span class="author">Trout CV</span>, <span class="author">van den Bedem H</span>, <span class="author">Weekes D</span>, <span class="author">White A</span>, <span class="author">Xu Q</span>, <span class="author">Hodgson KO</span>, <span class="author">Wooley J</span>, <span class="author">Elsliger MA</span>, <span class="author">Deacon AM</span>, <span class="author">Godzik A</span>, <span class="author">Lesley S</span> and <span class="author">Wilson IA</span></p>
<p class="affiliation">Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, Menlo Park, CA, USA.</p><p class="abstract">The crystal structure of Jann_2411 from Jannaschia sp. strain CCS1, a member of&nbsp;the Pfam PF07336 family classified as a domain of unknown function (DUF1470), was solved to a resolution of 1.45&#x2005;&Aring; by multiple-wavelength anomalous dispersion (MAD). This protein is the first structural representative of the DUF1470 Pfam family. Structural analysis revealed a two-domain organization, with the N-terminal domain presenting a new fold called the ABATE domain that may bind an as yet unknown ligand. The C-terminal domain forms a treble-clef zinc finger that is likely to be involved in DNA binding. Analysis of the Jann_2411 protein and the broader ABATE-domain family suggests a role as stress-induced transcriptional regulators.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">P50 GM62411</span>, <span class="grant">U54 GM074898</span>; Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Acta crystallographica. Section F, Structural biology and crystallization communications</span>&nbsp;<span class="year">2010</span>;<span class="volume">66</span>;<span class="number">Pt 10</span>;<span class="pagerange">1198-204</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20944211" rel="external">20944211</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2954205" rel="external">2954205</a>; DOI: <a href="http://dx.doi.org/10.1107%2FS1744309109025196" rel="external">10.1107/S1744309109025196</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A predominantly neolithic origin for European paternal lineages.</h4>
<p class="authors"><span class="author">Balaresque P</span>, <span class="author">Bowden GR</span>, <span class="author">Adams SM</span>, <span class="author">Leung HY</span>, <span class="author">King TE</span>, <span class="author">Rosser ZH</span>, <span class="author">Goodwin J</span>, <span class="author">Moisan JP</span>, <span class="author">Richard C</span>, <span class="author">Millward A</span>, <span class="author">Demaine AG</span>, <span class="author">Barbujani G</span>, <span class="author">Previder&egrave; C</span>, <span class="author">Wilson IJ</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Jobling MA</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, Leicester, United Kingdom.</p><p class="abstract">The relative contributions to modern European populations of Paleolithic hunter-gatherers and Neolithic farmers from the Near East have been intensely debated. Haplogroup R1b1b2 (R-M269) is the commonest European Y-chromosomal lineage, increasing in frequency from east to west, and carried by 110 million European men. Previous studies suggested a Paleolithic origin, but here we show that the geographical distribution of its microsatellite diversity is best explained by spread from a single source in the Near East via Anatolia during the Neolithic. Taken with evidence on the origins of other haplogroups, this indicates that most European Y chromosomes originate in the Neolithic expansion. This reinterpretation makes Europe a prime example of how technological and cultural change is linked with the expansion of a Y-chromosomal lineage, and the contrast of this pattern with that shown by maternally inherited mitochondrial DNA suggests a unique role for males in the transition.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span>, <span class="grant">065569</span>, <span class="grant">084060</span>, <span class="grant">087576</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">e1000285</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20087410">20087410</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2799514">2799514</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1000285">10.1371/journal.pbio.1000285</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Attack of the clones.</h4>
<p class="authors"><span class="author">Baldry S</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">390</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20467444" rel="external">20467444</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2369" rel="external">10.1038/nrmicro2369</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of prevaccination and modern Bordetella pertussis strains.</h4>
<p class="authors"><span class="author">Bart MJ</span>, <span class="author">van Gent M</span>, <span class="author">van der Heide HG</span>, <span class="author">Boekhorst J</span>, <span class="author">Hermans P</span>, <span class="author">Parkhill J</span> and <span class="author">Mooi FR</span></p>
<p class="affiliation">Laboratory for Infectious Diseases and Screening, Netherlands Centre for Infectious Diseases Control, RIVM, Bilthoven, Netherlands.</p><p class="abstract"><strong>Background</strong>: Despite vaccination since the 1950s, pertussis has persisted and resurged. It remains a major cause of infant death worldwide and is the most prevalent vaccine-preventable disease in developed countries. The resurgence of pertussis has been associated with the expansion of Bordetella pertussis strains with a novel allele for the pertussis toxin (Ptx) promoter, ptxP3, which have replaced resident ptxP1 strains. Compared to ptxP1 strains, ptxP3 produce more Ptx resulting in increased virulence and immune suppression. To elucidate how B. pertussis has adapted to vaccination, we compared genome sequences of two ptxP3 strains with four strains isolated before and after the introduction vaccination.</p><p class="abstract"><strong>Results</strong>: The distribution of SNPs in regions involved in transcription and translation suggested that changes in gene regulation play an important role in adaptation. No evidence was found for acquisition of novel genes. Modern strains differed significantly from prevaccination strains, both phylogenetically and with respect to particular alleles. The ptxP3 strains were found to have diverged recently from modern ptxP1 strains. Differences between ptxP3 and modern ptxP1 strains included SNPs in a number of pathogenicity-associated genes. Further, both gene inactivation and reactivation was observed in ptxP3 strains relative to modern ptxP1 strains.</p><p class="abstract"><strong>Conclusions</strong>: Our work suggests that B. pertussis adapted by successive accumulation of SNPs and by gene (in)activation. In particular changes in gene regulation may have played a role in adaptation.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">627</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21070624" rel="external">21070624</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3018138" rel="external">3018138</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-627" rel="external">10.1186/1471-2164-11-627</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Curators of the world unite: the International Society of Biocuration.</h4>
<p class="authors"><span class="author">Bateman A</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">8</span>;<span class="pagerange">991</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20305270">20305270</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq101">10.1093/bioinformatics/btq101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Time to underpin Wikipedia wisdom.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Logan DW</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">468</span>;<span class="number">7325</span>;<span class="pagerange">765</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21150981" rel="external">21150981</a>; DOI: <a href="http://dx.doi.org/10.1038%2F468765c" rel="external">10.1038/468765c</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DUFs: families in search of function.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Coggill P</span> and <span class="author">Finn RD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, England. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Domains of unknown function (DUFs) are a large set of uncharacterized protein families that are found in the Pfam database. Here, the scale and growth of functionally uncharacterized families in biological databases are surveyed and the prospects for discovering their function are examined. In particular, the important role that structural genomics can play in identifying potential function is evaluated.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Acta crystallographica. Section F, Structural biology and crystallization communications</span>&nbsp;<span class="year">2010</span>;<span class="volume">66</span>;<span class="number">Pt 10</span>;<span class="pagerange">1148-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20944204">20944204</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2954198">2954198</a>; DOI: <a href="http://dx.doi.org/10.1107%2FS1744309110001685">10.1107/S1744309110001685</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Signatures of adaptation to obligate biotrophy in the Hyaloperonospora arabidopsidis genome.</h4>
<p class="authors"><span class="author">Baxter L</span>, <span class="author">Tripathy S</span>, <span class="author">Ishaque N</span>, <span class="author">Boot N</span>, <span class="author">Cabral A</span>, <span class="author">Kemen E</span>, <span class="author">Thines M</span>, <span class="author">Ah-Fong A</span>, <span class="author">Anderson R</span>, <span class="author">Badejoko W</span>, <span class="author">Bittner-Eddy P</span>, <span class="author">Boore JL</span>, <span class="author">Chibucos MC</span>, <span class="author">Coates M</span>, <span class="author">Dehal P</span>, <span class="author">Delehaunty K</span>, <span class="author">Dong S</span>, <span class="author">Downton P</span>, <span class="author">Dumas B</span>, <span class="author">Fabro G</span>, <span class="author">Fronick C</span>, <span class="author">Fuerstenberg SI</span>, <span class="author">Fulton L</span>, <span class="author">Gaulin E</span>, <span class="author">Govers F</span>, <span class="author">Hughes L</span>, <span class="author">Humphray S</span>, <span class="author">Jiang RH</span>, <span class="author">Judelson H</span>, <span class="author">Kamoun S</span>, <span class="author">Kyung K</span>, <span class="author">Meijer H</span>, <span class="author">Minx P</span>, <span class="author">Morris P</span>, <span class="author">Nelson J</span>, <span class="author">Phuntumart V</span>, <span class="author">Qutob D</span>, <span class="author">Rehmany A</span>, <span class="author">Rougon-Cardoso A</span>, <span class="author">Ryden P</span>, <span class="author">Torto-Alalibo T</span>, <span class="author">Studholme D</span>, <span class="author">Wang Y</span>, <span class="author">Win J</span>, <span class="author">Wood J</span>, <span class="author">Clifton SW</span>, <span class="author">Rogers J</span>, <span class="author">Van den Ackerveken G</span>, <span class="author">Jones JD</span>, <span class="author">McDowell JM</span>, <span class="author">Beynon J</span> and <span class="author">Tyler BM</span></p>
<p class="affiliation">School of Life Sciences, Warwick University, Wellesbourne, CV35 9EF, UK.</p><p class="abstract">Many oomycete and fungal plant pathogens are obligate biotrophs, which extract nutrients only from living plant tissue and cannot grow apart from their hosts. Although these pathogens cause substantial crop losses, little is known about the molecular basis or evolution of obligate biotrophy. Here, we report the genome sequence of the oomycete Hyaloperonospora arabidopsidis (Hpa), an obligate biotroph and natural pathogen of Arabidopsis thaliana. In comparison with genomes of related, hemibiotrophic Phytophthora species, the Hpa genome exhibits dramatic reductions in genes encoding (i) RXLR effectors and other secreted pathogenicity proteins, (ii) enzymes for assimilation of inorganic nitrogen and sulfur, and (iii) proteins associated with zoospore formation and motility. These attributes comprise a genomic signature of evolution toward obligate biotrophy.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C509123/1</span>, <span class="grant">BB/E024815/1</span>, <span class="grant">BB/E024882/1</span>, <span class="grant">BB/F0161901</span>, <span class="grant">EP/F500025/1</span>, <span class="grant">T12144</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">330</span>;<span class="number">6010</span>;<span class="pagerange">1549-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21148394">21148394</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1195203">10.1126/science.1195203</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic hotspots for adaptation: the population genetics of M&uuml;llerian mimicry in the Heliconius melpomene clade.</h4>
<p class="authors"><span class="author">Baxter SW</span>, <span class="author">Nadeau NJ</span>, <span class="author">Maroja LS</span>, <span class="author">Wilkinson P</span>, <span class="author">Counterman BA</span>, <span class="author">Dawson A</span>, <span class="author">Beltran M</span>, <span class="author">Perez-Espona S</span>, <span class="author">Chamberlain N</span>, <span class="author">Ferguson L</span>, <span class="author">Clark R</span>, <span class="author">Davidson C</span>, <span class="author">Glithero R</span>, <span class="author">Mallet J</span>, <span class="author">McMillan WO</span>, <span class="author">Kronforst M</span>, <span class="author">Joron M</span>, <span class="author">Ffrench-Constant RH</span> and <span class="author">Jiggins CD</span></p>
<p class="affiliation">Department of Zoology, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">Wing patterning in Heliconius butterflies is a longstanding example of both M&uuml;llerian mimicry and phenotypic radiation under strong natural selection. The loci controlling such patterns are &quot;hotspots&quot; for adaptive evolution with great allelic diversity across different species in the genus. We characterise nucleotide variation, genotype-by-phenotype associations, linkage disequilibrium, and candidate gene expression at two loci and across multiple hybrid zones in Heliconius melpomene and relatives. Alleles at HmB control the presence or absence of the red forewing band, while alleles at HmYb control the yellow hindwing bar. Across HmYb two regions, separated by approximately 100 kb, show significant genotype-by-phenotype associations that are replicated across independent hybrid zones. In contrast, at HmB a single peak of association indicates the likely position of functional sites at three genes, encoding a kinesin, a G-protein coupled receptor, and an mRNA splicing factor. At both HmYb and HmB there is evidence for enhanced linkage disequilibrium (LD) between associated sites separated by up to 14 kb, suggesting that multiple sites are under selection. However, there was no evidence for reduced variation or deviations from neutrality that might indicate a recent selective sweep, consistent with these alleles being relatively old. Of the three genes showing an association with the HmB locus, the kinesin shows differences in wing disc expression between races that are replicated in the co-mimic, Heliconius erato, providing striking evidence for parallel changes in gene expression between M&uuml;llerian co-mimics. Wing patterning loci in Heliconius melpomene therefore show a haplotype structure maintained by selection, but no evidence for a recent selective sweep. The complex genetic pattern contrasts with the simple genetic basis of many adaptive traits studied previously, but may provide a better model for most adaptation in natural populations that has arisen over millions rather than tens of years.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">011845</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">e1000794</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20140188" rel="external">20140188</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2816687" rel="external">2816687</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000794" rel="external">10.1371/journal.pgen.1000794</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of different criteria for the diagnosis of primary myelofibrosis reveals limited clinical utility for measurement of serum lactate dehydrogenase.</h4>
<p class="authors"><span class="author">Beer PA</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, 1Cambridge Institute for Medical Research, Cambridge, United Kingdom.</p><p class="abstract">Primary myelofibrosis shows histological and pathogenetic overlap with essential thrombocythemia and polycythemia vera. Several diagnostic classifications have been proposed for primary myelofibrosis, although little is known about their clinical utility. In a comparison of three recent classifications, overall concordance was 79%. Inclusion of raised serum lactate dehydrogenase categorized 9% of patients as primary myelofibrosis when other criteria were not met. Although mean serum lactate dehydrogenase levels were higher in patients with primary myelofibrosis, levels were also increased in the majority of patients with essential thrombocythemia or polycythemia vera, and significant overlap was observed. A positive correlation with higher leukocyte and platelet count, and disease duration in primary myelofibrosis, suggests that serum lactate dehydrogenase is a biomarker for disease bulk and/or cellular proliferation. In conclusion, raised lactate dehydrogenase lacks specificity for primary myelofibrosis, consistent with the concept of a phenotypic continuum between essential thrombocythemia, polycythemia vera and primary myelofibrosis.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Haematologica</span>&nbsp;<span class="year">2010</span>;<span class="volume">95</span>;<span class="number">11</span>;<span class="pagerange">1960-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20534696" rel="external">20534696</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2966920" rel="external">2966920</a>; DOI: <a href="http://dx.doi.org/10.3324%2Fhaematol.2010.026708" rel="external">10.3324/haematol.2010.026708</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Independently acquired biallelic JAK2 mutations are present in a minority of patients with essential thrombocythemia.</h4>
<p class="authors"><span class="author">Beer PA</span>, <span class="author">Ortmann CA</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2010</span>;<span class="volume">116</span>;<span class="number">6</span>;<span class="pagerange">1013-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20705770" rel="external">20705770</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145103" rel="external">3145103</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2010-05-284356" rel="external">10.1182/blood-2010-05-284356</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mobile genetic element proliferation and gene inactivation impact over the genome structure and metabolic capabilities of Sodalis glossinidius, the secondary endosymbiont of tsetse flies.</h4>
<p class="authors"><span class="author">Belda E</span>, <span class="author">Moya A</span>, <span class="author">Bentley S</span> and <span class="author">Silva FJ</span></p>
<p class="affiliation">Institut Cavanilles de Biodiversitat i Biologia Evolutiva, Universitat de Val&egrave;ncia, Spain.</p><p class="abstract"><strong>Background</strong>: Genome reduction is a common evolutionary process in symbiotic and pathogenic bacteria. This process has been extensively characterized in bacterial endosymbionts of insects, where primary mutualistic bacteria represent the most extreme cases of genome reduction consequence of a massive process of gene inactivation and loss during their evolution from free-living ancestors. Sodalis glossinidius, the secondary endosymbiont of tsetse flies, contains one of the few complete genomes of bacteria at the very beginning of the symbiotic association, allowing to evaluate the relative impact of mobile genetic element proliferation and gene inactivation over the structure and functional capabilities of this bacterial endosymbiont during the transition to a host dependent lifestyle.</p><p class="abstract"><strong>Results</strong>: A detailed characterization of mobile genetic elements and pseudogenes reveals a massive presence of different types of prophage elements together with five different families of IS elements that have proliferated across the genome of Sodalis glossinidius at different levels. In addition, a detailed survey of intergenic regions allowed the characterization of 1501 pseudogenes, a much higher number than the 972 pseudogenes described in the original annotation. Pseudogene structure reveals a minor impact of mobile genetic element proliferation in the process of gene inactivation, with most of pseudogenes originated by multiple frameshift mutations and premature stop codons. The comparison of metabolic profiles of Sodalis glossinidius and tsetse fly primary endosymbiont Wiglesworthia glossinidia based on their whole gene and pseudogene repertoires revealed a novel case of pathway inactivation, the arginine biosynthesis, in Sodalis glossinidius together with a possible case of metabolic complementation with Wigglesworthia glossinidia for thiamine biosynthesis.</p><p class="abstract"><strong>Conclusions</strong>: The complete re-analysis of the genome sequence of Sodalis glossinidius reveals novel insights in the evolutionary transition from a free-living ancestor to a host-dependent lifestyle, with a massive proliferation of mobile genetic elements mainly of phage origin although with minor impact in the process of gene inactivation that is taking place in this bacterial genome. The metabolic analysis of the whole endosymbiotic consortia of tsetse flies have revealed a possible phenomenon of metabolic complementation between primary and secondary endosymbionts that can contribute to explain the co-existence of both bacterial endosymbionts in the context of the tsetse host.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">449</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20649993" rel="external">20649993</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3091646" rel="external">3091646</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-449" rel="external">10.1186/1471-2164-11-449</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrated genetic and epigenetic analysis identifies haplotype-specific methylation in the FTO type 2 diabetes and obesity susceptibility locus.</h4>
<p class="authors"><span class="author">Bell CG</span>, <span class="author">Finer S</span>, <span class="author">Lindgren CM</span>, <span class="author">Wilson GA</span>, <span class="author">Rakyan VK</span>, <span class="author">Teschendorff AE</span>, <span class="author">Akan P</span>, <span class="author">Stupka E</span>, <span class="author">Down TA</span>, <span class="author">Prokopenko I</span>, <span class="author">Morison IM</span>, <span class="author">Mill J</span>, <span class="author">Pidsley R</span>, <span class="author">International Type 2 Diabetes 1q Consortium</span>, <span class="author">Deloukas P</span>, <span class="author">Frayling TM</span>, <span class="author">Hattersley AT</span>, <span class="author">McCarthy MI</span>, <span class="author">Beck S</span> and <span class="author">Hitman GA</span></p>
<p class="affiliation">Medical Genomics, UCL Cancer Institute, University College London, London, United Kingdom. <a href="mailto:%63%68%72%69%73%74%6F%70%68%65%72.%62%65%6C%6C%40%75%63%6C.%61%63.%75%6B">&#99;&#104;&#114;&#105;&#115;&#116;&#111;&#112;&#104;&#101;&#114;&#46;&#98;&#101;&#108;&#108;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Recent multi-dimensional approaches to the study of complex disease have revealed powerful insights into how genetic and epigenetic factors may underlie their aetiopathogenesis. We examined genotype-epigenotype interactions in the context of Type 2 Diabetes (T2D), focussing on known regions of genomic susceptibility. We assayed DNA methylation in 60 females, stratified according to disease susceptibility haplotype using previously identified association loci. CpG methylation was assessed using methylated DNA immunoprecipitation on a targeted array (MeDIP-chip) and absolute methylation values were estimated using a Bayesian algorithm (BATMAN). Absolute methylation levels were quantified across LD blocks, and we identified increased DNA methylation on the FTO obesity susceptibility haplotype, tagged by the rs8050136 risk allele A (p&#x200A;=&#x200A;9.40&times;10(-4), permutation p&#x200A;=&#x200A;1.0&times;10(-3)). Further analysis across the 46 kb LD block using sliding windows localised the most significant difference to be within a 7.7 kb region (p&#x200A;=&#x200A;1.13&times;10(-7)). Sequence level analysis, followed by pyrosequencing validation, revealed that the methylation difference was driven by the co-ordinated phase of CpG-creating SNPs across the risk haplotype. This 7.7 kb region of haplotype-specific methylation (HSM), encapsulates a Highly Conserved Non-Coding Element (HCNE) that has previously been validated as a long-range enhancer, supported by the histone H3K4me1 enhancer signature. This study demonstrates that integration of Genome-Wide Association (GWA) SNP and epigenomic DNA methylation data can identify potential novel genotype-epigenotype interactions within disease-associated loci, thus providing a novel route to aid unravelling common complex diseases.</p>
<p class="grants">Funded by: Medical Research Council; NIDDK NIH HHS: <span class="grant">DK-073490</span>; Wellcome Trust: <span class="grant">084071</span>, <span class="grant">WT086596/Z/08/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">e14040</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21124985" rel="external">21124985</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987816" rel="external">2987816</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0014040" rel="external">10.1371/journal.pone.0014040</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Independent evolution of the core and accessory gene sets in the genus Neisseria: insights gained from the genome of Neisseria lactamica isolate 020-06.</h4>
<p class="authors"><span class="author">Bennett JS</span>, <span class="author">Bentley SD</span>, <span class="author">Vernikos GS</span>, <span class="author">Quail MA</span>, <span class="author">Cherevach I</span>, <span class="author">White B</span>, <span class="author">Parkhill J</span> and <span class="author">Maiden MC</span></p>
<p class="affiliation">Department of Zoology, University of Oxford, UK. <a href="mailto:%6A%75%6C%69%61.%62%65%6E%6E%65%74%74%40%7A%6F%6F.%6F%78.%61%63.%75%6B">&#106;&#117;&#108;&#105;&#97;&#46;&#98;&#101;&#110;&#110;&#101;&#116;&#116;&#64;&#122;&#111;&#111;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The genus Neisseria contains two important yet very different pathogens, N. meningitidis and N. gonorrhoeae, in addition to non-pathogenic species, of which N. lactamica is the best characterized. Genomic comparisons of these three bacteria will provide insights into the mechanisms and evolution of pathogenesis in this group of organisms, which are applicable to understanding these processes more generally.</p><p class="abstract"><strong>Results</strong>: Non-pathogenic N. lactamica exhibits very similar population structure and levels of diversity to the meningococcus, whilst gonococci are essentially recent descendents of a single clone. All three species share a common core gene set estimated to comprise around 1190 CDSs, corresponding to about 60% of the genome. However, some of the nucleotide sequence diversity within this core genome is particular to each group, indicating that cross-species recombination is rare in this shared core gene set. Other than the meningococcal cps region, which encodes the polysaccharide capsule, relatively few members of the large accessory gene pool are exclusive to one species group, and cross-species recombination within this accessory genome is frequent.</p><p class="abstract"><strong>Conclusion</strong>: The three Neisseria species groups represent coherent biological and genetic groupings which appear to be maintained by low rates of inter-species horizontal genetic exchange within the core genome. There is extensive evidence for exchange among positively selected genes and the accessory genome and some evidence of hitch-hiking of housekeeping genes with other loci. It is not possible to define a &#39;pathogenome&#39; for this group of organisms and the disease causing phenotypes are therefore likely to be complex, polygenic, and different among the various disease-associated phenotypes observed.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087622</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">652</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21092259">21092259</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3091772">3091772</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-652">10.1186/1471-2164-11-652</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Taming the next-gen beast.</h4>
<p class="authors"><span class="author">Bentley S</span></p>
<p class="affiliation">Stephen Bentley is at the Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This month&#39;s Genome Watch discusses how alternative approaches to using second-generation sequencing technologies are powerful tools for the analysis of common pathogenic bacteria.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">161</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20157335" rel="external">20157335</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2322" rel="external">10.1038/nrmicro2322</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Study of smell and reproductive organs in a mouse model for CHARGE syndrome.</h4>
<p class="authors"><span class="author">Bergman JE</span>, <span class="author">Bosman EA</span>, <span class="author">van Ravenswaaij-Arts CM</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Department of Genetics, University Medical Centre Groningen, University of Groningen, Groningen, The Netherlands.</p><p class="abstract">CHARGE syndrome is a multiple congenital anomaly syndrome characterised by Coloboma, Heart defects, Atresia of choanae, Retardation of growth and/or development, Genital hypoplasia, and Ear anomalies often associated with deafness. It is caused by heterozygous mutations in the CHD7 gene and shows a highly variable phenotype. Anosmia and hypogonadotropic hypogonadism occur in the majority of the CHARGE patients, but the underlying pathogenesis is unknown. Therefore, we studied the ability to smell and aspects of the reproductive system (reproductive performance, gonadotropin-releasing hormone (GnRH) neurons and anatomy of testes and uteri) in a mouse model for CHARGE syndrome, the whirligig mouse (Chd7(Whi/+)). We showed that Chromodomain Helicase DNA-binding protein 7 (Chd7) is expressed in brain areas involved in olfaction and reproduction during embryonic development. We observed poorer performance in the smell test in adult Chd7(Whi/+) mice, secondary either to olfactory dysfunction or to balance disturbances. Olfactory bulb and reproductive organ abnormalities were observed in a proportion of Chd7(Whi/+) mice. Hypothalamic GnRH neurons were slightly reduced in Chd7(Whi/+) females and reproductive performance was slightly less in Chd7(Whi/+) mice. This study shows that the penetrance of anosmia and hypogonadotropic hypogonadism is lower in Chd7(Whi/+) mice than in CHARGE patients. Interestingly, many phenotypic features of the Chd7 mutation showed incomplete penetrance in our model mice, despite the use of inbred, genetically identical mice. This supports the theory that the extreme variability of the CHARGE phenotype in both humans and mice might be attributed to variations in the fetal microenvironment or to purely stochastic events.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">2</span>;<span class="pagerange">171-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19809474" rel="external">19809474</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987182" rel="external">2987182</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2009.158" rel="external">10.1038/ejhg.2009.158</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prevalence of Salmonella enterica in poultry and eggs in Uruguay during an epidemic due to Salmonella enterica serovar Enteritidis.</h4>
<p class="authors"><span class="author">Betancor L</span>, <span class="author">Pereira M</span>, <span class="author">Martinez A</span>, <span class="author">Giossa G</span>, <span class="author">Fookes M</span>, <span class="author">Flores K</span>, <span class="author">Barrios P</span>, <span class="author">Repiso V</span>, <span class="author">Vignoli R</span>, <span class="author">Cordeiro N</span>, <span class="author">Algorta G</span>, <span class="author">Thomson N</span>, <span class="author">Maskell D</span>, <span class="author">Schelotto F</span> and <span class="author">Chabalgoity JA</span></p>
<p class="affiliation">Bacteriology and Virology Department, Instituto de Higiene, Facultad de Medicina, Universidad de la Rep&uacute;blica, Montevideo, Uruguay.</p><p class="abstract">Salmonella enterica serovar Enteritidis (S. Enteritidis) is frequently associated with food-borne disease worldwide. Poultry-derived products are a major source. An epidemic of human infection with S. Enteritidis occurred in Uruguay, and to evaluate the extent of poultry contamination, we conducted a nationwide survey over 2 years that included the analysis of sera from 5,751 birds and 12,400 eggs. Serological evidence of infection with Salmonella group O:9 was found in 24.4% of the birds. All positive sera were retested with a gm flagellum-based enzyme-linked immunosorbent assay, and based on these results, the national prevalence of S. Enteritidis infection was estimated to be 6.3%. Salmonellae were recovered from 58 of 620 pools made up of 20 eggs each, demonstrating a prevalence of at least 1 in every 214 eggs. Surprisingly, the majority of the isolates were not S. Enteritidis. Thirty-nine isolates were typed as S. Derby, 9 as S. Gallinarum, 8 as S. Enteritidis, and 2 as S. Panama. Despite the highest prevalence in eggs, S. Derby was not isolated from humans in the period of analysis, suggesting a low capacity to infect humans. Microarray-based comparative genomic hybridization analysis of S. Derby and S. Enteritidis revealed more than 350 genetic differences. S. Derby lacked pathogenicity islands 13 and 14, the fimbrial lpf operon, and other regions encoding metabolic functions. Several of these regions are present not only in serovar Enteritidis but also in all sequenced strains of S. Typhimurium, suggesting that these regions might be related to the capacity of Salmonella to cause food-borne disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">078168/Z/05/Z</span></p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">48</span>;<span class="number">7</span>;<span class="pagerange">2413-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20484605" rel="external">20484605</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2897505" rel="external">2897505</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.02137-09" rel="external">10.1128/JCM.02137-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variants in ACAD10 are associated with type 2 diabetes, insulin resistance and lipid oxidation in Pima Indians.</h4>
<p class="authors"><span class="author">Bian L</span>, <span class="author">Hanson RL</span>, <span class="author">Muller YL</span>, <span class="author">Ma L</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Kobes S</span>, <span class="author">Knowler WC</span>, <span class="author">Bogardus C</span> and <span class="author">Baier LJ</span></p>
<p class="affiliation">Phoenix Epidemiology and Clinical Research Branch, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, 445 N. 5th Street, Suite 210, Phoenix, AZ 85004, USA.</p><p class="abstract">A prior genome-wide association study in Pima Indians identified a variant within the ACAD10 gene that is associated with early-onset type 2 diabetes. Acylcoenzyme A dehydrogenase 10 (ACAD10) catalyses mitochondrial fatty acid beta-oxidation, which plays a pivotal role in developing insulin resistance and type 2 diabetes. Therefore, ACAD10 was analysed as a positional and biological candidate for type 2 diabetes.</p><p class="abstract"><strong>Methods</strong>: Twenty-three SNPs were genotyped in 1,500 Pima Indians to determine the linkage disequilibrium pattern across ACAD10. Association with type 2 diabetes was determined by genotyping four tag single nucleotide polymorphisms (SNPs) in a population-based sample of 3,501 full-heritage Pima Indians; two associated SNPs were further genotyped in a second population-based sample of 3,723 American Indians. Associations with quantitative traits were assessed in 415 non-diabetic full heritage Pima individuals who had been metabolically phenotyped.</p><p class="abstract"><strong>Results</strong>: SNPs rs601663 and rs659964 were associated with type 2 diabetes in the full-heritage Pima Indian sample (p=0.04 and 0.0006, respectively), and rs659964 was further associated with type 2 diabetes in the second American Indian sample (p=0.04). Combination of these two samples provided the strongest evidence for association (p=0.009 and 0.00007, for rs601663 and rs659964, respectively). Quantitative trait analyses identified nominal associations with both lower lipid oxidation rate and larger subcutaneous abdominal adipocyte size, which is consistent with the known physiology of ACAD10, and also identified associations with increased insulin resistance.</p><p class="abstract">We propose that ACAD10 variation may increase type 2 diabetes susceptibility by impairing insulin sensitivity via abnormal lipid oxidation.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">ZIA DK075012-04</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2010</span>;<span class="volume">53</span>;<span class="number">7</span>;<span class="pagerange">1349-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20390405">20390405</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2947857">2947857</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-010-1695-y">10.1007/s00125-010-1695-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Signatures of mutation and selection in the cancer genome.</h4>
<p class="authors"><span class="author">Bignell GR</span>, <span class="author">Greenman CD</span>, <span class="author">Davies H</span>, <span class="author">Butler AP</span>, <span class="author">Edkins S</span>, <span class="author">Andrews JM</span>, <span class="author">Buck G</span>, <span class="author">Chen L</span>, <span class="author">Beare D</span>, <span class="author">Latimer C</span>, <span class="author">Widaa S</span>, <span class="author">Hinton J</span>, <span class="author">Fahey C</span>, <span class="author">Fu B</span>, <span class="author">Swamy S</span>, <span class="author">Dalgliesh GL</span>, <span class="author">Teh BT</span>, <span class="author">Deloukas P</span>, <span class="author">Yang F</span>, <span class="author">Campbell PJ</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The cancer genome is moulded by the dual processes of somatic mutation and selection. Homozygous deletions in cancer genomes occur over recessive cancer genes, where they can confer selective growth advantage, and over fragile sites, where they are thought to reflect an increased local rate of DNA breakage. However, most homozygous deletions in cancer genomes are unexplained. Here we identified 2,428 somatic homozygous deletions in 746 cancer cell lines. These overlie 11% of protein-coding genes that, therefore, are not mandatory for survival of human cells. We derived structural signatures that distinguish between homozygous deletions over recessive cancer genes and fragile sites. Application to clusters of unexplained homozygous deletions suggests that many are in regions of inherent fragility, whereas a small subset overlies recessive cancer genes. The results illustrate how structural signatures can be used to distinguish between the influences of mutation and selection in cancer genomes. The extensive copy number, genotyping, sequence and expression data available for this large series of publicly available cancer cell lines renders them informative reagents for future studies of cancer biology and drug discovery.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01 CA155258</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">463</span>;<span class="number">7283</span>;<span class="pagerange">893-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20164919">20164919</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145113">3145113</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08768">10.1038/nature08768</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sex determination in the social amoeba Dictyostelium discoideum.</h4>
<p class="authors"><span class="author">Bloomfield G</span>, <span class="author">Skelton J</span>, <span class="author">Ivens A</span>, <span class="author">Tanaka Y</span> and <span class="author">Kay RR</span></p>
<p class="affiliation">Medical Research Council Laboratory of Molecular Biology, Hills Road, Cambridge CB2 0QH, UK. <a href="mailto:%67%61%72%65%74%68%62%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#103;&#97;&#114;&#101;&#116;&#104;&#98;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The genetics of sex determination remain mysterious in many organisms, including some that are otherwise well studied. Here we report the discovery and analysis of the mating-type locus of the model organism Dictyostelium discoideum. Three forms of a single genetic locus specify this species&#39; three mating types: two versions of the locus are entirely different in sequence, and the third resembles a composite of the other two. Single, unrelated genes are sufficient to determine two of the mating types, whereas homologs of both these genes are required in the composite type. The key genes encode polypeptides that possess no recognizable similarity to established protein families. Sex determination in the social amoebae thus appears to use regulators that are unrelated to any others currently known.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">06724</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">330</span>;<span class="number">6010</span>;<span class="pagerange">1533-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21148389">21148389</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1197423">10.1126/science.1197423</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large, rare chromosomal deletions associated with severe early-onset obesity.</h4>
<p class="authors"><span class="author">Bochukova EG</span>, <span class="author">Huang N</span>, <span class="author">Keogh J</span>, <span class="author">Henning E</span>, <span class="author">Purmann C</span>, <span class="author">Blaszczyk K</span>, <span class="author">Saeed S</span>, <span class="author">Hamilton-Shield J</span>, <span class="author">Clayton-Smith J</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Hurles ME</span> and <span class="author">Farooqi IS</span></p>
<p class="affiliation">University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge CB2 0QQ, UK.</p><p class="abstract">Obesity is a highly heritable and genetically heterogeneous disorder. Here we investigated the contribution of copy number variation to obesity in 300 Caucasian patients with severe early-onset obesity, 143 of whom also had developmental delay. Large (&gt;500 kilobases), rare (&lt;1%) deletions were significantly enriched in patients compared to 7,366 controls (P &lt; 0.001). We identified several rare copy number variants that were recurrent in patients but absent or at much lower prevalence in controls. We identified five patients with overlapping deletions on chromosome 16p11.2 that were found in 2 out of 7,366 controls (P &lt; 5 x 10(-5)). In three patients the deletion co-segregated with severe obesity. Two patients harboured a larger de novo 16p11.2 deletion, extending through a 593-kilobase region previously associated with autism and mental retardation; both of these patients had mild developmental delay in addition to severe obesity. In an independent sample of 1,062 patients with severe obesity alone, the smaller 16p11.2 deletion was found in an additional two patients. All 16p11.2 deletions encompass several genes but include SH2B1, which is known to be involved in leptin and insulin signalling. Deletion carriers exhibited hyperphagia and severe insulin resistance disproportionate for the degree of obesity. We show that copy number variation contributes significantly to the genetic architecture of human obesity.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0900554</span>; Wellcome Trust: <span class="grant">077014</span>, <span class="grant">077014/Z/05/0Z</span>, <span class="grant">082390</span>, <span class="grant">082390/Z/07/Z)</span>, <span class="grant">085475</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">463</span>;<span class="number">7281</span>;<span class="pagerange">666-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19966786">19966786</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3108883">3108883</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08689">10.1038/nature08689</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variants at DGKB/TMEM195, ADRA2A, GLIS3 and C2CD4B loci are associated with reduced glucose-stimulated beta cell function in middle-aged Danish people.</h4>
<p class="authors"><span class="author">Boesgaard TW</span>, <span class="author">Grarup N</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Borch-Johnsen K</span>, <span class="author">Meta-Analysis of Glucose and Insulin-Related Trait Consortium (MAGIC)</span>, <span class="author">Hansen T</span> and <span class="author">Pedersen O</span></p>
<p class="affiliation">Hagedorn Research Institute, Niels Steensens Vej 2, 2820 Gentofte, Denmark.</p><p class="abstract">A meta-analysis of 21 genome-wide association studies identified 11 novel genetic loci implicated in fasting glucose homeostasis. We aimed to evaluate the impact of these variants on insulin release and insulin sensitivity estimated from OGTTs.</p><p class="abstract"><strong>Methods</strong>: Eleven variants in or near DGKB/TMEM195, ADCY5, MADD, ADRA2A, FADS1, CRY2, SLC2A2, GLIS3, PROX1, C2CD4B and IGF1 were genotyped in 6,784 middle-aged participants of the population-based Inter99 cohort. Association studies of quantitative estimates of insulin release and insulin sensitivity were performed in 5,722 non-diabetic Danish participants on whom an OGTT was performed.</p><p class="abstract"><strong>Results</strong>: Assuming an additive genetic model, carriers of the alleles increasing fasting glucose in DGKB/TMEM195, ADRA2A, GLIS3 and C2CD4B showed decreased glucose-stimulated insulin release as assessed by the BIGTT-acute insulin response index (2.7-3.5%; p &lt; 0.005 for all) and by corrected insulin response (2.8-5.9%; p &lt; 0.03 for all). In addition, the PROX1 glucose-raising allele showed a 2.9% decreased corrected insulin response (p = 0.03), while the hyperglycaemic allele of variants in or near ADRA2A, FADS1, CRY2 and C2CD4B were associated with a 2.6% to 9.3% decrease in one or both of two different OGTT-based disposition indices (p &lt; 0.02 for all). After correction for multiple testing, variants in the DGKB/TMEM195, ADRA2A, GLIS3 and C2CD4B loci were associated with estimates of beta cell function.</p><p class="abstract">We found that the lead variants at the DGKB/TMEM195, ADRA2A, GLIS3 and C2CD4B loci were associated with decreased glucose-stimulated insulin response. This association underlines the importance of pancreatic beta cell dysfunction in the genetic predisposition to hyperglycaemia and type 2 diabetes.</p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2010</span>;<span class="volume">53</span>;<span class="number">8</span>;<span class="pagerange">1647-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20419449">20419449</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-010-1753-5">10.1007/s00125-010-1753-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gap5--editing the billion fragment sequence assembly.</h4>
<p class="authors"><span class="author">Bonfield JK</span> and <span class="author">Whitwham A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK. <a href="mailto:%6A%6B%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#107;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">MOTIVATION: Existing sequence assembly editors struggle with the volumes of data now readily available from the latest generation of DNA sequencing instruments. RESULTS: We describe the Gap5 software along with the data structures and algorithms used that allow it to be scalable. We demonstrate this with an assembly of 1.1 billion sequence fragments and compare the performance with several other programs. We analyse the memory, CPU, I/O usage and file sizes used by Gap5. Availability and Implementation: Gap5 is part of the Staden Package and is available under an Open Source licence from http://staden.sourceforge.net. It is implemented in C and Tcl/Tk. Currently it works on Unix systems only.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">077200/Z/05/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">14</span>;<span class="pagerange">1699-703</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20513662" rel="external">20513662</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2894512" rel="external">2894512</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq268" rel="external">10.1093/bioinformatics/btq268</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale association analysis of TNF/LTA gene region polymorphisms in type 2 diabetes.</h4>
<p class="authors"><span class="author">Boraska V</span>, <span class="author">Rayner NW</span>, <span class="author">Groves CJ</span>, <span class="author">Frayling TM</span>, <span class="author">Diakite M</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Day-Williams AG</span>, <span class="author">McCarthy MI</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Department of Medical Biology, University of Split School of Medicine, Split, Croatia. <a href="mailto:%76%62%6F%72%61%73%6B%61%40%6D%65%66%73%74.%68%72">&#118;&#98;&#111;&#114;&#97;&#115;&#107;&#97;&#64;&#109;&#101;&#102;&#115;&#116;&#46;&#104;&#114;</a></p><p class="abstract"><strong>Background</strong>: The TNF/LTA locus has been a long-standing T2D candidate gene. Several studies have examined association of TNF/LTA SNPs with T2D but the majority have been small-scale and produced no convincing evidence of association. The purpose of this study is to examine T2D association of tag SNPs in the TNF/LTA region capturing the majority of common variation in a large-scale sample set of UK/Irish origin.</p><p class="abstract"><strong>Methods</strong>: This study comprised a case-control (1520 cases and 2570 control samples) and a family-based component (423 parent-offspring trios). Eleven tag SNPs (rs928815, rs909253, rs746868, rs1041981 (T60N), rs1800750, rs1800629 (G-308A), rs361525 (G-238A), rs3093662, rs3093664, rs3093665, and rs3093668) were selected across the TNF/LTA locus and genotyped using a fluorescence-based competitive allele specific assay. Quality control of the obtained genotypes was performed prior to single- and multi-point association analyses under the additive model.</p><p class="abstract"><strong>Results</strong>: We did not find any consistent SNP associations with T2D in the case-control or family-based datasets.</p><p class="abstract"><strong>Conclusions</strong>: The present study, designed to analyse a set of tag SNPs specifically selected to capture the majority of common variation in the TNF/LTA gene region, found no robust evidence for association with T2D. To investigate the presence of smaller effects of TNF/LTA gene variation with T2D, a large-scale meta-analysis will be required.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">BMC medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20459604">20459604</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873325">2873325</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2350-11-69">10.1186/1471-2350-11-69</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">53BP1 loss rescues BRCA1 deficiency and is associated with triple-negative and BRCA-mutated breast cancers.</h4>
<p class="authors"><span class="author">Bouwman P</span>, <span class="author">Aly A</span>, <span class="author">Escandell JM</span>, <span class="author">Pieterse M</span>, <span class="author">Bartkova J</span>, <span class="author">van der Gulden H</span>, <span class="author">Hiddingh S</span>, <span class="author">Thanasoula M</span>, <span class="author">Kulkarni A</span>, <span class="author">Yang Q</span>, <span class="author">Haffty BG</span>, <span class="author">Tommiska J</span>, <span class="author">Blomqvist C</span>, <span class="author">Drapkin R</span>, <span class="author">Adams DJ</span>, <span class="author">Nevanlinna H</span>, <span class="author">Bartek J</span>, <span class="author">Tarsounas M</span>, <span class="author">Ganesan S</span> and <span class="author">Jonkers J</span></p>
<p class="affiliation">Division of Molecular Biology, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</p><p class="abstract">Germ-line mutations in breast cancer 1, early onset (BRCA1) result in predisposition to breast and ovarian cancer. BRCA1-mutated tumors show genomic instability, mainly as a consequence of impaired recombinatorial DNA repair. Here we identify p53-binding protein 1 (53BP1) as an essential factor for sustaining the growth arrest induced by Brca1 deletion. Depletion of 53BP1 abrogates the ATM-dependent checkpoint response and G2 cell-cycle arrest triggered by the accumulation of DNA breaks in Brca1-deleted cells. This effect of 53BP1 is specific to BRCA1 function, as 53BP1 depletion did not alleviate proliferation arrest or checkpoint responses in Brca2-deleted cells. Notably, loss of 53BP1 partially restores the homologous-recombination defect of Brca1-deleted cells and reverts their hypersensitivity to DNA-damaging agents. We find reduced 53BP1 expression in subsets of sporadic triple-negative and BRCA-associated breast cancers, indicating the potential clinical implications of our findings.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6997</span>, <span class="grant">A8784</span>; Wellcome Trust: <span class="grant">082356</span></p>
<p class="publication"><span class="title">Nature structural &amp; molecular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">688-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20453858">20453858</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2912507">2912507</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnsmb.1831">10.1038/nsmb.1831</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare variation at the TNFAIP3 locus and susceptibility to rheumatoid arthritis.</h4>
<p class="authors"><span class="author">Bowes J</span>, <span class="author">Lawrence R</span>, <span class="author">Eyre S</span>, <span class="author">Panoutsopoulou K</span>, <span class="author">Orozco G</span>, <span class="author">Elliott KS</span>, <span class="author">Ke X</span>, <span class="author">Morris AP</span>, <span class="author">UKRAG</span>, <span class="author">Thomson W</span>, <span class="author">Worthington J</span>, <span class="author">Barton A</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Arthritis Research UK, Epidemiology Unit, University of Manchester, Manchester, UK.</p><p class="abstract">Genome-wide association studies (GWAS) conducted using commercial single nucleotide polymorphisms (SNP) arrays have proven to be a powerful tool for the detection of common disease susceptibility variants. However, their utility for the detection of lower frequency variants is yet to be practically investigated. Here we describe the application of a rare variant collapsing method to a large genome-wide SNP dataset, the Wellcome Trust Case Control Consortium rheumatoid arthritis (RA) GWAS. We partitioned the data into gene-centric bins and collapsed genotypes of low frequency variants (defined here as MAF &le; 0.05) into a single count coupled with univariate analysis. We then prioritized gene regions for further investigation in an independent cohort of 3,355 cases and 2,427 controls based on rare variant signal p value and prior evidence to support involvement in RA. A total of 14,536 gene bins were investigated in the primary analysis and signals mapping to the TNFAIP3 and chr17q24 loci were selected for further investigation. We detected replicating association to low frequency variants in the TNFAIP3 gene (combined p = 6.6 &times; 10(-6)). Even though rare variants are not well-represented and can be difficult to genotype in GWAS, our study supports the application of low frequency variant collapsing methods to genome-wide SNP datasets as a means of exploiting data that are routinely ignored.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>, <span class="grant">18475</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">081682</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">128</span>;<span class="number">6</span>;<span class="pagerange">627-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20852893">20852893</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2978888">2978888</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-010-0889-1">10.1007/s00439-010-0889-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Towards Viral Genome Annotation Standards, Report from the 2010 NCBI Annotation Workshop</h4>
<p class="authors"><span class="author">Brister JR</span>, <span class="author">Bao Y</span>, <span class="author">Kuiken C</span>, <span class="author">Lefkowitz EJ</span>, <span class="author">Le Mercier P</span>, <span class="author">Leplae R</span>, <span class="author">Madupu R</span>, <span class="author">Scheuermann RH</span>, <span class="author">Schobel S</span>, <span class="author">Seto D</span>, <span class="author">Shrivastava S</span>, <span class="author">STERK P</span>, <span class="author">Zeng Q</span>, <span class="author">Klimke W</span>, <span class="author">Tatusova T</span></p>
<p class="publication"><span class="title">Viruses-Basel.</span> <span class="year">2010</span>;<span class="volume">2</span>;<span class="pagerange">2258-68</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Scoring and validation of tandem MS peptide identification methods.</h4>
<p class="authors"><span class="author">Brosch M</span> and <span class="author">Choudhary J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">A variety of methods are described in the literature to assign peptide sequences to observed tandem MS data. Typically, the identified peptides are associated only with an arbitrary score that reflects the quality of the peptide-spectrum match but not with a statistically meaningful significance measure. In this chapter, we discuss why statistical significance measures can simplify and unify the interpretation of MS-based proteomic experiments. In addition, we also present available software solutions that convert scores into sound statistical measures.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">604</span>;<span class="pagerange">43-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20013363">20013363</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-60761-444-9_4">10.1007/978-1-60761-444-9_4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quantifying the mechanisms of domain gain in animal proteins.</h4>
<p class="authors"><span class="author">Buljan M</span>, <span class="author">Frankish A</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%62%36%31%33%40%63%61%6D.%61%63.%75%6B">&#109;&#98;&#54;&#49;&#51;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Protein domains are protein regions that are shared among different proteins and are frequently functionally and structurally independent from the rest of the protein. Novel domain combinations have a major role in evolutionary innovation. However, the relative contributions of the different molecular mechanisms that underlie domain gains in animals are still unknown. By using animal gene phylogenies we were able to identify a set of high confidence domain gain events and by looking at their coding DNA investigate the causative mechanisms.</p><p class="abstract"><strong>Results</strong>: Here we show that the major mechanism for gains of new domains in metazoan proteins is likely to be gene fusion through joining of exons from adjacent genes, possibly mediated by non-allelic homologous recombination. Retroposition and insertion of exons into ancestral introns through intronic recombination are, in contrast to previous expectations, only minor contributors to domain gains and have accounted for less than 1% and 10% of high confidence domain gain events, respectively. Additionally, exonization of previously non-coding regions appears to be an important mechanism for addition of disordered segments to proteins. We observe that gene duplication has preceded domain gain in at least 80% of the gain events.</p><p class="abstract"><strong>Conclusions</strong>: The interplay of gene duplication and domain gain demonstrates an important mechanism for fast neofunctionalization of genes.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">7</span>;<span class="pagerange">R74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20633280">20633280</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2926785">2926785</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-7-r74">10.1186/gb-2010-11-7-r74</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New views on natural killer cell-based immunotherapy for melanoma treatment.</h4>
<p class="authors"><span class="author">Burke S</span>, <span class="author">Lakshmikanth T</span>, <span class="author">Colucci F</span> and <span class="author">Carbone E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Natural killer (NK) cell-based immunotherapies treat hematopoietic malignancies, but are less effective against solid tumors. Here, we review recent data on NK cell recognition of melanoma at various stages of the disease and propose a combinatorial strategy to exploit fully the potential of NK cells. Depending on the stage of melanoma progression, NK cell-based therapies could be combined with pharmacological and T cell-based immunotherapies, to: (i) prevent lymph node metastases by redistributing cytotoxic NK cells; (ii) boost NK cell activity using chemotherapy to upregulate activating ligands on tumor cells; and (iii) target visceral metastases by transfer of autologous or allogeneic NK cells.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Trends in immunology</span>&nbsp;<span class="year">2010</span>;<span class="volume">31</span>;<span class="number">9</span>;<span class="pagerange">339-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20655806" rel="external">20655806</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.it.2010.06.003" rel="external">10.1016/j.it.2010.06.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The patterns and dynamics of genomic instability in metastatic pancreatic cancer.</h4>
<p class="authors"><span class="author">Campbell PJ</span>, <span class="author">Yachida S</span>, <span class="author">Mudie LJ</span>, <span class="author">Stephens PJ</span>, <span class="author">Pleasance ED</span>, <span class="author">Stebbings LA</span>, <span class="author">Morsberger LA</span>, <span class="author">Latimer C</span>, <span class="author">McLaren S</span>, <span class="author">Lin ML</span>, <span class="author">McBride DJ</span>, <span class="author">Varela I</span>, <span class="author">Nik-Zainal SA</span>, <span class="author">Leroy C</span>, <span class="author">Jia M</span>, <span class="author">Menzies A</span>, <span class="author">Butler AP</span>, <span class="author">Teague JW</span>, <span class="author">Griffin CA</span>, <span class="author">Burton J</span>, <span class="author">Swerdlow H</span>, <span class="author">Quail MA</span>, <span class="author">Stratton MR</span>, <span class="author">Iacobuzio-Donahue C</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Pancreatic cancer is an aggressive malignancy with a five-year mortality of 97-98%, usually due to widespread metastatic disease. Previous studies indicate that this disease has a complex genomic landscape, with frequent copy number changes and point mutations, but genomic rearrangements have not been characterized in detail. Despite the clinical importance of metastasis, there remain fundamental questions about the clonal structures of metastatic tumours, including phylogenetic relationships among metastases, the scale of ongoing parallel evolution in metastatic and primary sites, and how the tumour disseminates. Here we harness advances in DNA sequencing to annotate genomic rearrangements in 13 patients with pancreatic cancer and explore clonal relationships among metastases. We find that pancreatic cancer acquires rearrangements indicative of telomere dysfunction and abnormal cell-cycle control, namely dysregulated G1-to-S-phase transition with intact G2-M checkpoint. These initiate amplification of cancer genes and occur predominantly in early cancer development rather than the later stages of the disease. Genomic instability frequently persists after cancer dissemination, resulting in ongoing, parallel and even convergent evolution among different metastases. We find evidence that there is genetic heterogeneity among metastasis-initiating cells, that seeding metastasis may require driver mutations beyond those required for primary tumours, and that phylogenetic trees across metastases show organ-specific branches. These data attest to the richness of genetic variation in cancer, brought about by the tandem forces of genomic instability and evolutionary selection.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA106610</span>, <span class="grant">CA140599</span>, <span class="grant">K08 CA106610</span>, <span class="grant">K08 CA106610-03</span>, <span class="grant">K08 CA106610-04</span>, <span class="grant">K08 CA106610-05</span>, <span class="grant">R01 CA140599</span>, <span class="grant">R01 CA140599-01</span>, <span class="grant">R01 CA140599-02</span>, <span class="grant">R01 CA140599-03</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span>, <span class="grant">WT088340MA</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">467</span>;<span class="number">7319</span>;<span class="pagerange">1109-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20981101">20981101</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3137369">3137369</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09460">10.1038/nature09460</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural modelling and comparative analysis of homologous, analogous and specific proteins from Trypanosoma cruzi versus Homo sapiens: putative drug targets for chagas&#39; disease treatment.</h4>
<p class="authors"><span class="author">Capriles PV</span>, <span class="author">Guimar&atilde;es AC</span>, <span class="author">Otto TD</span>, <span class="author">Miranda AB</span>, <span class="author">Dardenne LE</span> and <span class="author">Degrave WM</span></p>
<p class="affiliation">Grupo de Modelagem Molecular de Sistemas Biol&oacute;gicos, Laborat&oacute;rio Nacional de Computa&ccedil;&atilde;o Cient&iacute;fica, LNCC/MCT, Petr&oacute;polis, CEP 25651-075, Brazil. <a href="mailto:%63%61%70%72%69%6C%65%73%40%6C%6E%63%63.%62%72">&#99;&#97;&#112;&#114;&#105;&#108;&#101;&#115;&#64;&#108;&#110;&#99;&#99;&#46;&#98;&#114;</a></p><p class="abstract"><strong>Background</strong>: Trypanosoma cruzi is the etiological agent of Chagas&#39; disease, an endemic infection that causes thousands of deaths every year in Latin America. Therapeutic options remain inefficient, demanding the search for new drugs and/or new molecular targets. Such efforts can focus on proteins that are specific to the parasite, but analogous enzymes and enzymes with a three-dimensional (3D) structure sufficiently different from the corresponding host proteins may represent equally interesting targets. In order to find these targets we used the workflows MHOLline and AnEn&Pi; obtaining 3D models from homologous, analogous and specific proteins of Trypanosoma cruzi versus Homo sapiens.</p><p class="abstract"><strong>Results</strong>: We applied genome wide comparative modelling techniques to obtain 3D models for 3,286 predicted proteins of T. cruzi. In combination with comparative genome analysis to Homo sapiens, we were able to identify a subset of 397 enzyme sequences, of which 356 are homologous, 3 analogous and 38 specific to the parasite.</p><p class="abstract"><strong>Conclusions</strong>: In this work, we present a set of 397 enzyme models of T. cruzi that can constitute potential structure-based drug targets to be investigated for the development of new strategies to fight Chagas&#39; disease. The strategies presented here support the concept of structural analysis in conjunction with protein functional analysis as an interesting computational methodology to detect potential targets for structure-based rational drug design. For example, 2,4-dienoyl-CoA reductase (EC 1.3.1.34) and triacylglycerol lipase (EC 3.1.1.3), classified as analogous proteins in relation to H. sapiens enzymes, were identified as new potential molecular targets.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">610</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21034488" rel="external">21034488</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3091751" rel="external">3091751</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-610" rel="external">10.1186/1471-2164-11-610</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BamView: viewing mapped read alignment data in the context of the reference sequence.</h4>
<p class="authors"><span class="author">Carver T</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Otto TD</span>, <span class="author">Parkhill J</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%72%74%65%6D%69%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#116;&#101;&#109;&#105;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">SUMMARY: BamView is an interactive Java application for visualizing the large amounts of data stored for sequence reads which are aligned against a reference genome sequence. It supports the BAM (Binary Alignment/Map) format. It can be used in a number of contexts including SNP calling and structural annotation. BamView has also been integrated into Artemis so that the reads can be viewed in the context of the nucleotide sequence and genomic features. AVAILABILITY: BamView and Artemis are freely available (under a GPL licence) for download (for MacOSX, UNIX and Windows) at: http://bamview.sourceforge.net/</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">5</span>;<span class="pagerange">676-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20071372" rel="external">20071372</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2828118" rel="external">2828118</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq010" rel="external">10.1093/bioinformatics/btq010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PLA2G7 genotype, lipoprotein-associated phospholipase A2 activity, and coronary heart disease risk in 10 494 cases and 15 624 controls of European Ancestry.</h4>
<p class="authors"><span class="author">Casas JP</span>, <span class="author">Ninio E</span>, <span class="author">Panayiotou A</span>, <span class="author">Palmen J</span>, <span class="author">Cooper JA</span>, <span class="author">Ricketts SL</span>, <span class="author">Sofat R</span>, <span class="author">Nicolaides AN</span>, <span class="author">Corsetti JP</span>, <span class="author">Fowkes FG</span>, <span class="author">Tzoulaki I</span>, <span class="author">Kumari M</span>, <span class="author">Brunner EJ</span>, <span class="author">Kivimaki M</span>, <span class="author">Marmot MG</span>, <span class="author">Hoffmann MM</span>, <span class="author">Winkler K</span>, <span class="author">M&auml;rz W</span>, <span class="author">Ye S</span>, <span class="author">Stirnadel HA</span>, <span class="author">Boekholdt SM</span>, <span class="author">Khaw KT</span>, <span class="author">Humphries SE</span>, <span class="author">Sandhu MS</span>, <span class="author">Hingorani AD</span> and <span class="author">Talmud PJ</span></p>
<p class="affiliation">Department of Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, UK. <a href="mailto:%4A%75%61%6E.%70%61%62%6C%6F-%63%61%73%61%73%40%6C%73%68%74%6D.%61%63.%75%6B">&#74;&#117;&#97;&#110;&#46;&#112;&#97;&#98;&#108;&#111;&#45;&#99;&#97;&#115;&#97;&#115;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a> &lt;<a href="mailto:%4A%75%61%6E.%70%61%62%6C%6F-%63%61%73%61%73%40%6C%73%68%74%6D.%61%63.%75%6B&%67%74">&#74;&#117;&#97;&#110;&#46;&#112;&#97;&#98;&#108;&#111;&#45;&#99;&#97;&#115;&#97;&#115;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: Higher lipoprotein-associated phospholipase A(2)(Lp-PLA2) activity is associated with increased risk of coronary heart disease (CHD), making Lp-PLA2 a potential therapeutic target. PLA2G7 variants associated with Lp-PLA2 activity could evaluate whether this relationship is causal.</p><p class="abstract">A meta-analysis including a total of 12 studies (5 prospective, 4 case-control, 1 case-only, and 2 cross-sectional studies; n=26 118) was undertaken to examine the association of the following: (1) Lp-PLA2 activity versus cardiovascular biomarkers and risk factors and CHD events (2 prospective studies; n=4884); (2) PLA2G7 single-nucleotide polymorphisms and Lp-PLA2 activity (3 prospective, 2 case-control, 2 cross-sectional studies; up to n=6094); and (3) PLA2G7 single-nucleotide polymorphisms and angiographic coronary artery disease (2 case-control, 1 case-only study; n=4971 cases) and CHD events (5 prospective, 2 case-control studies; n=5523). Lp-PLA2 activity correlated with several CHD risk markers. Hazard ratios for CHD events for the top versus bottom quartile of Lp-PLA2 activity were 1.61 (95% confidence interval, 1.31 to 1.99) and 1.17 (95% confidence interval, 0.91 to 1.51) after adjustment for baseline traits. Of 7 single-nucleotide polymorphisms, rs1051931 (A379V) showed the strongest association with Lp-PLA2 activity, with VV subjects having 7.2% higher activity than AAs. Genotype was not associated with risk markers, angiographic coronary disease (odds ratio, 1.03; 95% confidence interval, 0.80 to 1.32), or CHD events (odds ratio, 0.98; 95% confidence interval, 0.82 to 1.17).</p><p class="abstract"><strong>Conclusions</strong>: Unlike Lp-PLA2 activity, PLA2G7 variants associated with modest effects on Lp-PLA2 activity were not associated with cardiovascular risk markers, coronary atheroma, or CHD. Larger association studies, identification of single-nucleotide polymorphisms with larger effects, or randomized trials of specific Lp-PLA2 inhibitors are needed to confirm or refute a contributory role for Lp-PLA2 in CHD.</p>
<p class="grants">Funded by: AHRQ HHS: <span class="grant">HS06516</span>; British Heart Foundation: <span class="grant">FS/07/011</span>, <span class="grant">PG98/183</span>, <span class="grant">RG05/014</span>; NIA NIH HHS: <span class="grant">AG13196</span>; Wellcome Trust: <span class="grant">085475</span></p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2010</span>;<span class="volume">121</span>;<span class="number">21</span>;<span class="pagerange">2284-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20479152" rel="external">20479152</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.109.923383" rel="external">10.1161/CIRCULATIONAHA.109.923383</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Beyond the Genome: genomics research ten years after the human genome sequence.</h4>
<p class="authors"><span class="author">Casto AM</span> and <span class="author">Amid C</span></p>
<p class="affiliation">Department of Genetics, Stanford University, Stanford, CA 94305, USA. <a href="mailto:%6D%6F%72%67%61%6E%32%31%40%73%74%61%6E%66%6F%72%64.%65%64%75">&#109;&#111;&#114;&#103;&#97;&#110;&#50;&#49;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">A report on the meeting &#39;Beyond the Genome&#39;, Boston, USA, 11-13 October 2010.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">11</span>;<span class="pagerange">309</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21122162">21122162</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3156947">3156947</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-11-309">10.1186/gb-2010-11-11-309</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular and physiological analysis of three Pseudomonas aeruginosa phages belonging to the &quot;N4-like viruses&quot;.</h4>
<p class="authors"><span class="author">Ceyssens PJ</span>, <span class="author">Brabban A</span>, <span class="author">Rogge L</span>, <span class="author">Lewis MS</span>, <span class="author">Pickard D</span>, <span class="author">Goulding D</span>, <span class="author">Dougan G</span>, <span class="author">Noben JP</span>, <span class="author">Kropinski A</span>, <span class="author">Kutter E</span> and <span class="author">Lavigne R</span></p>
<p class="affiliation">Division of Gene Technology, Katholieke Universiteit Leuven, Kasteelpark Arenberg, Leuven, B-3001, Belgium.</p><p class="abstract">We present a detailed analysis of the genome architecture, structural proteome and infection-related properties of three Pseudomonas phages, designated LUZ7, LIT1 and PEV2. These podoviruses encapsulate 72.5 to 74.9 kb genomes and lyse their host after 25 min aerobic infection. PEV2 can successfully infect under anaerobic conditions, but its latent period is tripled, the lysis proceeds far slower and the burst size decreases significantly. While the overall genome structure of these phages resembles the well-studied coliphage N4, these Pseudomonas phages encode a cluster of tail genes which displays significant similarity to a Pseudomonasaeruginosa (cryptic) prophage region. Using ESI-MS/MS, these tail proteins were shown to be part of the phage particle, as well as ten other proteins including a giant 370 kDa virion RNA polymerase. These phages are the first described representatives of a novel kind of obligatory lytic P. aeruginosa-infecting phages, belonging to the widespread &quot;N4-like viruses&quot; genus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Virology</span>&nbsp;<span class="year">2010</span>;<span class="volume">405</span>;<span class="number">1</span>;<span class="pagerange">26-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20619867">20619867</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3715699">3715699</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.virol.2010.06.011">10.1016/j.virol.2010.06.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic loci influencing kidney function and chronic kidney disease.</h4>
<p class="authors"><span class="author">Chambers JC</span>, <span class="author">Zhang W</span>, <span class="author">Lord GM</span>, <span class="author">van der Harst P</span>, <span class="author">Lawlor DA</span>, <span class="author">Sehmi JS</span>, <span class="author">Gale DP</span>, <span class="author">Wass MN</span>, <span class="author">Ahmadi KR</span>, <span class="author">Bakker SJ</span>, <span class="author">Beckmann J</span>, <span class="author">Bilo HJ</span>, <span class="author">Bochud M</span>, <span class="author">Brown MJ</span>, <span class="author">Caulfield MJ</span>, <span class="author">Connell JM</span>, <span class="author">Cook HT</span>, <span class="author">Cotlarciuc I</span>, <span class="author">Davey Smith G</span>, <span class="author">de Silva R</span>, <span class="author">Deng G</span>, <span class="author">Devuyst O</span>, <span class="author">Dikkeschei LD</span>, <span class="author">Dimkovic N</span>, <span class="author">Dockrell M</span>, <span class="author">Dominiczak A</span>, <span class="author">Ebrahim S</span>, <span class="author">Eggermann T</span>, <span class="author">Farrall M</span>, <span class="author">Ferrucci L</span>, <span class="author">Floege J</span>, <span class="author">Forouhi NG</span>, <span class="author">Gansevoort RT</span>, <span class="author">Han X</span>, <span class="author">Hedblad B</span>, <span class="author">Homan van der Heide JJ</span>, <span class="author">Hepkema BG</span>, <span class="author">Hernandez-Fuentes M</span>, <span class="author">Hypponen E</span>, <span class="author">Johnson T</span>, <span class="author">de Jong PE</span>, <span class="author">Kleefstra N</span>, <span class="author">Lagou V</span>, <span class="author">Lapsley M</span>, <span class="author">Li Y</span>, <span class="author">Loos RJ</span>, <span class="author">Luan J</span>, <span class="author">Luttropp K</span>, <span class="author">Mar&eacute;chal C</span>, <span class="author">Melander O</span>, <span class="author">Munroe PB</span>, <span class="author">Nordfors L</span>, <span class="author">Parsa A</span>, <span class="author">Peltonen L</span>, <span class="author">Penninx BW</span>, <span class="author">Perucha E</span>, <span class="author">Pouta A</span>, <span class="author">Prokopenko I</span>, <span class="author">Roderick PJ</span>, <span class="author">Ruokonen A</span>, <span class="author">Samani NJ</span>, <span class="author">Sanna S</span>, <span class="author">Schalling M</span>, <span class="author">Schlessinger D</span>, <span class="author">Schlieper G</span>, <span class="author">Seelen MA</span>, <span class="author">Shuldiner AR</span>, <span class="author">Sj&ouml;gren M</span>, <span class="author">Smit JH</span>, <span class="author">Snieder H</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Stenvinkel P</span>, <span class="author">Sternberg MJ</span>, <span class="author">Swaminathan R</span>, <span class="author">Tanaka T</span>, <span class="author">Ubink-Veltmaat LJ</span>, <span class="author">Uda M</span>, <span class="author">Vollenweider P</span>, <span class="author">Wallace C</span>, <span class="author">Waterworth D</span>, <span class="author">Zerres K</span>, <span class="author">Waeber G</span>, <span class="author">Wareham NJ</span>, <span class="author">Maxwell PH</span>, <span class="author">McCarthy MI</span>, <span class="author">Jarvelin MR</span>, <span class="author">Mooser V</span>, <span class="author">Abecasis GR</span>, <span class="author">Lightstone L</span>, <span class="author">Scott J</span>, <span class="author">Navis G</span>, <span class="author">Elliott P</span> and <span class="author">Kooner JS</span></p>
<p class="affiliation">Department of Epidemiology and Biostatistics, School of Public Health, Imperial College of London, London, UK. <a href="mailto:%6A%6F%68%6E.%63%68%61%6D%62%65%72%73%40%69%63.%61%63.%75%6B">&#106;&#111;&#104;&#110;&#46;&#99;&#104;&#97;&#109;&#98;&#101;&#114;&#115;&#64;&#105;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Using genome-wide association, we identify common variants at 2p12-p13, 6q26, 17q23 and 19q13 associated with serum creatinine, a marker of kidney function (P = 10(-10) to 10(-15)). Of these, rs10206899 (near NAT8, 2p12-p13) and rs4805834 (near SLC7A9, 19q13) were also associated with chronic kidney disease (P = 5.0 x 10(-5) and P = 3.6 x 10(-4), respectively). Our findings provide insight into metabolic, solute and drug-transport pathways underlying susceptibility to chronic kidney disease.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">373-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20383145">20383145</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.566">10.1038/ng.566</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&alpha;8-integrins are required for hippocampal long-term potentiation but not for hippocampal-dependent learning.</h4>
<p class="authors"><span class="author">Chan CS</span>, <span class="author">Chen H</span>, <span class="author">Bradley A</span>, <span class="author">Dragatsis I</span>, <span class="author">Rosenmund C</span> and <span class="author">Davis RL</span></p>
<p class="affiliation">Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX, USA.</p><p class="abstract">Integrins are heterodimeric transmembrane cell adhesion receptors that are essential for a wide range of biological functions via cell-matrix and cell-cell interactions. Recent studies have provided evidence that some of the subunits in the integrin family are involved in synaptic and behavioral plasticity. To further understand the role of integrins in the mammalian central nervous system, we generated a postnatal forebrain and excitatory neuron-specific knockout of alpha8-integrin in the mouse. Behavioral studies showed that the mutant mice are normal in multiple hippocampal-dependent learning tasks, including a T-maze, non-match-to-place working memory task for which other integrin subunits like alpha3- and beta1-integrin are required. In contrast, mice mutant for alpha8-integrin exhibited a specific impairment of long-term potentiation (LTP) at Schaffer collateral-CA1 synapses, whereas basal synaptic transmission, paired-pulse facilitation and long-term depression (LTD) remained unaffected. Because LTP is also impaired in the absence of alpha3-integrin, our results indicate that multiple integrin molecules are required for the normal expression of LTP, and different integrins display distinct roles in behavioral and neurophysiological processes like synaptic plasticity.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD24064</span>; NIMH NIH HHS: <span class="grant">MH60420</span>, <span class="grant">R01 MH060420-10</span></p>
<p class="publication"><span class="title">Genes, brain, and behavior</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="number">4</span>;<span class="pagerange">402-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20132319" rel="external">20132319</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2925190" rel="external">2925190</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1601-183X.2010.00569.x" rel="external">10.1111/j.1601-183X.2010.00569.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The impact of gene expression regulation on evolution of extracellular signaling pathways.</h4>
<p class="authors"><span class="author">Charoensawan V</span>, <span class="author">Adryan B</span>, <span class="author">Martin S</span>, <span class="author">S&ouml;llner C</span>, <span class="author">Thisse B</span>, <span class="author">Thisse C</span>, <span class="author">Wright GJ</span> and <span class="author">Teichmann SA</span></p>
<p class="affiliation">Medical Research Council Laboratory of Molecular Biology, Cambridge CB20QH, United Kingdom. <a href="mailto:%76%61%72%6F%64%6F%6D%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#118;&#97;&#114;&#111;&#100;&#111;&#109;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Extracellular protein interactions are crucial to the development of multicellular organisms because they initiate signaling pathways and enable cellular recognition cues. Despite their importance, extracellular protein interactions are often under-represented in large scale protein interaction data sets because most high throughput assays are not designed to detect low affinity extracellular interactions. Due to the lack of a comprehensive data set, the evolution of extracellular signaling pathways has remained largely a mystery. We investigated this question using a combined data set of physical pairwise interactions between zebrafish extracellular proteins, mainly from the immunoglobulin superfamily and leucine-rich repeat families, and their spatiotemporal expression profiles. We took advantage of known homology between proteins to estimate the relative rates of changes of four parameters after gene duplication, namely extracellular protein interaction, expression pattern, and the divergence of extracellular and intracellular protein sequences. We showed that change in expression profile is a major contributor to the evolution of signaling pathways followed by divergence in intracellular protein sequence, whereas extracellular sequence and interaction profiles were relatively more conserved. Rapidly evolving expression profiles will eventually drive other parameters to diverge more quickly because differentially expressed proteins get exposed to different environments and potential binding partners. This allows homologous extracellular receptors to attain specialized functions and become specific to tissues and/or developmental stages.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105161047</span>; Wellcome Trust: <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">2666-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20935258">20935258</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3101855">3101855</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M110.003020">10.1074/mcp.M110.003020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence and comparative metabolic profiling of the prototypical enteroaggregative Escherichia coli strain 042.</h4>
<p class="authors"><span class="author">Chaudhuri RR</span>, <span class="author">Sebaihia M</span>, <span class="author">Hobman JL</span>, <span class="author">Webber MA</span>, <span class="author">Leyton DL</span>, <span class="author">Goldberg MD</span>, <span class="author">Cunningham AF</span>, <span class="author">Scott-Tucker A</span>, <span class="author">Ferguson PR</span>, <span class="author">Thomas CM</span>, <span class="author">Frankel G</span>, <span class="author">Tang CM</span>, <span class="author">Dudley EG</span>, <span class="author">Roberts IS</span>, <span class="author">Rasko DA</span>, <span class="author">Pallen MJ</span>, <span class="author">Parkhill J</span>, <span class="author">Nataro JP</span>, <span class="author">Thomson NR</span> and <span class="author">Henderson IR</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Escherichia coli can experience a multifaceted life, in some cases acting as a commensal while in other cases causing intestinal and/or extraintestinal disease. Several studies suggest enteroaggregative E. coli are the predominant cause of E. coli-mediated diarrhea in the developed world and are second only to Campylobacter sp. as a cause of bacterial-mediated diarrhea. Furthermore, enteroaggregative E. coli are a predominant cause of persistent diarrhea in the developing world where infection has been associated with malnourishment and growth retardation.</p><p class="abstract"><strong>Methods</strong>: In this study we determined the complete genomic sequence of E. coli 042, the prototypical member of the enteroaggregative E. coli, which has been shown to cause disease in volunteer studies. We performed genomic and phylogenetic comparisons with other E. coli strains revealing previously uncharacterised virulence factors including a variety of secreted proteins and a capsular polysaccharide biosynthetic locus. In addition, by using Biolog Phenotype Microarrays we have provided a full metabolic profiling of E. coli 042 and the non-pathogenic lab strain E. coli K-12. We have highlighted the genetic basis for many of the metabolic differences between E. coli 042 and E. coli K-12.</p><p class="abstract"><strong>Conclusion</strong>: This study provides a genetic context for the vast amount of experimental and epidemiological data published thus far and provides a template for future diagnostic and intervention strategies.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801209</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">e8801</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20098708">20098708</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808357">2808357</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0008801">10.1371/journal.pone.0008801</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinct clinical phenotypes associated with JAK2V617F reflect differential STAT1 signaling.</h4>
<p class="authors"><span class="author">Chen E</span>, <span class="author">Beer PA</span>, <span class="author">Godfrey AL</span>, <span class="author">Ortmann CA</span>, <span class="author">Li J</span>, <span class="author">Costa-Pereira AP</span>, <span class="author">Ingle CE</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Campbell PJ</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Cambridge Institute for Medical Research and Department of Haematology, University of Cambridge, Hills Road, Cambridge, CB2 0XY, UK.</p><p class="abstract">The JAK2V617F mutation is associated with distinct myeloproliferative neoplasms, including polycythemia vera (PV) and essential thrombocythemia (ET), but it remains unclear how it generates disparate disorders. By comparing clonally-derived mutant and wild-type cells from individual patients, we demonstrate that the transcriptional consequences of JAK2V617F are subtle, and that JAK2V617F-heterozygous erythroid cells from ET and PV patients exhibit differential interferon signaling and STAT1 phosphorylation. Increased STAT1 activity in normal CD34-positive progenitors produces an ET-like phenotype, whereas downregulation of STAT1 activity in JAK2V617F-heterozygous ET progenitors produces a PV-like phenotype. Our results illustrate the power of clonal analysis, indicate that the consequences of JAK2V617F reflect a balance between STAT5 and STAT1 activation and are relevant for other neoplasms associated with signaling pathway mutations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">5</span>;<span class="pagerange">524-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21074499" rel="external">21074499</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2996868" rel="external">2996868</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ccr.2010.10.013" rel="external">10.1016/j.ccr.2010.10.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl variation resources.</h4>
<p class="authors"><span class="author">Chen Y</span>, <span class="author">Cunningham F</span>, <span class="author">Rios D</span>, <span class="author">McLaren WM</span>, <span class="author">Smith J</span>, <span class="author">Pritchard B</span>, <span class="author">Spudich GM</span>, <span class="author">Brent S</span>, <span class="author">Kulesha E</span>, <span class="author">Marin-Garcia P</span>, <span class="author">Smedley D</span>, <span class="author">Birney E</span> and <span class="author">Flicek P</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK.</p><p class="abstract"><strong>Background</strong>: The maturing field of genomics is rapidly increasing the number of sequenced genomes and producing more information from those previously sequenced. Much of this additional information is variation data derived from sampling multiple individuals of a given species with the goal of discovering new variants and characterising the population frequencies of the variants that are already known. These data have immense value for many studies, including those designed to understand evolution and connect genotype to phenotype. Maximising the utility of the data requires that it be stored in an accessible manner that facilitates the integration of variation data with other genome resources such as gene annotation and comparative genomics.</p><p class="abstract"><strong>Description</strong>: The Ensembl project provides comprehensive and integrated variation resources for a wide variety of chordate genomes. This paper provides a detailed description of the sources of data and the methods for creating the Ensembl variation databases. It also explores the utility of the information by explaining the range of query options available, from using interactive web displays, to online data mining tools and connecting directly to the data servers programmatically. It gives a good overview of the variation resources and future plans for expanding the variation data within Ensembl.</p><p class="abstract"><strong>Conclusions</strong>: Variation data is an important key to understanding the functional and phenotypic differences between individuals. The development of new sequencing and genotyping technologies is greatly increasing the amount of variation data known for almost all genomes. The Ensembl variation resources are integrated into the Ensembl genome browser and provide a comprehensive way to access this data in the context of a widely used genome bioinformatics system. All Ensembl data is freely available at http://www.ensembl.org and from the public MySQL database server at ensembldb.ensembl.org.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">293</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20459805">20459805</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2894800">2894800</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-293">10.1186/1471-2164-11-293</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A cancer-derived mutation in the PSTAIRE helix of cyclin-dependent kinase 2 alters the stability of cyclin binding.</h4>
<p class="authors"><span class="author">Child ES</span>, <span class="author">Hendrychov&aacute; T</span>, <span class="author">McCague K</span>, <span class="author">Futreal A</span>, <span class="author">Otyepka M</span> and <span class="author">Mann DJ</span></p>
<p class="affiliation">Department of Life Sciences, Imperial College, South Kensington, London SW72AZ, UK.</p><p class="abstract">Cyclin-dependent kinase 2 (cdk2) is a central regulator of the mammalian cell cycle. Here we describe the properties of a mutant form of cdk2 identified during large-scale sequencing of protein kinases from cancerous tissue. The mutation substituted a leucine for a proline in the PSTAIRE helix, the central motif in the interaction of the cdk with its regulatory cyclin subunit. We demonstrate that whilst the mutant cdk2 is considerably impaired in stable cyclin association, it is still able to generate an active kinase that can functionally complement defective cdks in vivo. Molecular dynamic simulations and biophysical measurements indicate that the observed biochemical properties likely stem from increased flexibility within the cyclin-binding helix.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C510859/1</span>, <span class="grant">BB/D524208/1</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Biochimica et biophysica acta</span>&nbsp;<span class="year">2010</span>;<span class="volume">1803</span>;<span class="number">7</span>;<span class="pagerange">858-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20399812" rel="external">20399812</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3111755" rel="external">3111755</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.bbamcr.2010.04.004" rel="external">10.1016/j.bbamcr.2010.04.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Bayesian approach using covariance of single nucleotide polymorphism data to detect differences in linkage disequilibrium patterns between groups of individuals.</h4>
<p class="authors"><span class="author">Clark TG</span>, <span class="author">Campino SG</span>, <span class="author">Anastasi E</span>, <span class="author">Auburn S</span>, <span class="author">Teo YY</span>, <span class="author">Small K</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Holmes CC</span></p>
<p class="affiliation">Department of Epidemiology and Public Health, London School of Hygiene and Tropical Medicine, London, UK. <a href="mailto:%74%61%61%6E%65.%63%6C%61%72%6B%40%6C%73%68%74%6D.%61%63.%75%6B">&#116;&#97;&#97;&#110;&#101;&#46;&#99;&#108;&#97;&#114;&#107;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">MOTIVATION: Quantifying differences in linkage disequilibrium (LD) between sub-groups can highlight genetic regions or sites under selection and/or associated with disease, and may have utility in trans-ethnic mapping studies. RESULTS: We present a novel pseudo Bayes factor (PBF) approach that assess differences in covariance of genotype frequencies from single nucleotide polymorphism (SNP) data from a genome-wide study. The magnitude of the PBF reflects the strength of evidence for a difference, while accounting for the sample size and number of SNPs, without the requirement for permutation testing to establish statistical significance. Application of the PBF to HapMap and Gambian malaria SNP data reveals regional LD differences, some known to be under selection. AVAILABILITY AND IMPLEMENTATION: The PBF approach has been implemented in the BALD (Bayesian analysis of LD differences) C++ software, and is available from http://homepages.lshtm.ac.uk/tgclark/downloads.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">16</span>;<span class="pagerange">1999-2003</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20554688" rel="external">20554688</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2916719" rel="external">2916719</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq327" rel="external">10.1093/bioinformatics/btq327</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants near TERC are associated with mean telomere length.</h4>
<p class="authors"><span class="author">Codd V</span>, <span class="author">Mangino M</span>, <span class="author">van der Harst P</span>, <span class="author">Braund PS</span>, <span class="author">Kaiser M</span>, <span class="author">Beveridge AJ</span>, <span class="author">Rafelt S</span>, <span class="author">Moore J</span>, <span class="author">Nelson C</span>, <span class="author">Soranzo N</span>, <span class="author">Zhai G</span>, <span class="author">Valdes AM</span>, <span class="author">Blackburn H</span>, <span class="author">Mateo Leach I</span>, <span class="author">de Boer RA</span>, <span class="author">Kimura M</span>, <span class="author">Aviv A</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Goodall AH</span>, <span class="author">Ouwehand W</span>, <span class="author">van Veldhuisen DJ</span>, <span class="author">van Gilst WH</span>, <span class="author">Navis G</span>, <span class="author">Burton PR</span>, <span class="author">Tobin MD</span>, <span class="author">Hall AS</span>, <span class="author">Thompson JR</span>, <span class="author">Spector T</span> and <span class="author">Samani NJ</span></p>
<p class="affiliation">Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Leicester, UK.</p><p class="abstract">We conducted genome-wide association analyses of mean leukocyte telomere length in 2,917 individuals, with follow-up replication in 9,492 individuals. We identified an association with telomere length on 3q26 (rs12696304, combined P = 3.72 x 10(-14)) at a locus that includes TERC, which encodes the telomerase RNA component. Each copy of the minor allele of rs12696304 was associated with an approximately 75-base-pair reduction in mean telomere length, equivalent to approximately 3.6 years of age-related telomere-length attrition.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">3</span>;<span class="pagerange">197-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20139977">20139977</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.532">10.1038/ng.532</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The dopamine &beta;-hydroxylase -1021C/T polymorphism is associated with the risk of Alzheimer&#39;s disease in the Epistasis Project.</h4>
<p class="authors"><span class="author">Combarros O</span>, <span class="author">Warden DR</span>, <span class="author">Hammond N</span>, <span class="author">Cortina-Borja M</span>, <span class="author">Belbin O</span>, <span class="author">Lehmann MG</span>, <span class="author">Wilcock GK</span>, <span class="author">Brown K</span>, <span class="author">Kehoe PG</span>, <span class="author">Barber R</span>, <span class="author">Coto E</span>, <span class="author">Alvarez V</span>, <span class="author">Deloukas P</span>, <span class="author">Gwilliam R</span>, <span class="author">Heun R</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">Mateo I</span>, <span class="author">Oulhaj A</span>, <span class="author">Arias-V&aacute;squez A</span>, <span class="author">Schuur M</span>, <span class="author">Aulchenko YS</span>, <span class="author">Ikram MA</span>, <span class="author">Breteler MM</span>, <span class="author">van Duijn CM</span>, <span class="author">Morgan K</span>, <span class="author">Smith AD</span> and <span class="author">Lehmann DJ</span></p>
<p class="affiliation">Neurology Service and Centro de Investigaci&oacute;n Biom&eacute;dica en Red sobre Enfermedades Neurodegenerativas, Marqu&eacute;s de Valdecilla University Hospital (University of Cantabria), 39008 Santander, Spain. <a href="mailto:%63%6F%6D%62%61%72%72%6F%40%75%6E%69%63%61%6E.%65%73">&#99;&#111;&#109;&#98;&#97;&#114;&#114;&#111;&#64;&#117;&#110;&#105;&#99;&#97;&#110;&#46;&#101;&#115;</a></p><p class="abstract"><strong>Background</strong>: The loss of noradrenergic neurones of the locus coeruleus is a major feature of Alzheimer&#39;s disease (AD). Dopamine &beta;-hydroxylase (DBH) catalyses the conversion of dopamine to noradrenaline. Interactions have been reported between the low-activity -1021T allele (rs1611115) of DBH and polymorphisms of the pro-inflammatory cytokine genes, IL1A and IL6, contributing to the risk of AD. We therefore examined the associations with AD of the DBH -1021T allele and of the above interactions in the Epistasis Project, with 1757 cases of AD and 6294 elderly controls.</p><p class="abstract"><strong>Methods</strong>: We genotyped eight single nucleotide polymorphisms (SNPs) in the three genes, DBH, IL1A and IL6. We used logistic regression models and synergy factor analysis to examine potential interactions and associations with AD.</p><p class="abstract"><strong>Results</strong>: We found that the presence of the -1021T allele was associated with AD: odds ratio = 1.2 (95% confidence interval: 1.06-1.4, p = 0.005). This association was nearly restricted to men &lt; 75 years old: odds ratio = 2.2 (1.4-3.3, 0.0004). We also found an interaction between the presence of DBH -1021T and the -889TT genotype (rs1800587) of IL1A: synergy factor = 1.9 (1.2-3.1, 0.005). All these results were consistent between North Europe and North Spain.</p><p class="abstract"><strong>Conclusions</strong>: Extensive, previous evidence (reviewed here) indicates an important role for noradrenaline in the control of inflammation in the brain. Thus, the -1021T allele with presumed low activity may be associated with misregulation of inflammation, which could contribute to the onset of AD. We suggest that such misregulation is the predominant mechanism of the association we report here.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0400546</span></p>
<p class="publication"><span class="title">BMC medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">162</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21070631">21070631</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2994840">2994840</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2350-11-162">10.1186/1471-2350-11-162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation spectrum revealed by breakpoint sequencing of human germline CNVs.</h4>
<p class="authors"><span class="author">Conrad DF</span>, <span class="author">Bird C</span>, <span class="author">Blackburne B</span>, <span class="author">Lindsay S</span>, <span class="author">Mamanova L</span>, <span class="author">Lee C</span>, <span class="author">Turner DJ</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Precisely characterizing the breakpoints of copy number variants (CNVs) is crucial for assessing their functional impact. However, fewer than 10% of known germline CNVs have been mapped to the single-nucleotide level. We characterized the sequence breakpoints from a dataset of all CNVs detected in three unrelated individuals in previous array-based CNV discovery experiments. We used targeted hybridization-based DNA capture and 454 sequencing to sequence 324 CNV breakpoints, including 315 deletions. We observed two major breakpoint signatures: 70% of the deletion breakpoints have 1-30 bp of microhomology, whereas 33% of deletion breakpoints contain 1-367 bp of inserted sequence. The co-occurrence of microhomology and inserted sequence is low (10%), suggesting that there are at least two different mutational mechanisms. Approximately 5% of the breakpoints represent more complex rearrangements, including local microinversions, suggesting a replication-based strand switching mechanism. Despite a rich literature on DNA repair processes, reconstruction of the molecular events generating each of these mutations is not yet possible.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077014</span>, <span class="grant">077014/Z/05/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">385-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20364136">20364136</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3428939">3428939</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.564">10.1038/ng.564</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Origins and functional impact of copy number variation in the human genome.</h4>
<p class="authors"><span class="author">Conrad DF</span>, <span class="author">Pinto D</span>, <span class="author">Redon R</span>, <span class="author">Feuk L</span>, <span class="author">Gokcumen O</span>, <span class="author">Zhang Y</span>, <span class="author">Aerts J</span>, <span class="author">Andrews TD</span>, <span class="author">Barnes C</span>, <span class="author">Campbell P</span>, <span class="author">Fitzgerald T</span>, <span class="author">Hu M</span>, <span class="author">Ihm CH</span>, <span class="author">Kristiansson K</span>, <span class="author">Macarthur DG</span>, <span class="author">Macdonald JR</span>, <span class="author">Onyiah I</span>, <span class="author">Pang AW</span>, <span class="author">Robson S</span>, <span class="author">Stirrups K</span>, <span class="author">Valsesia A</span>, <span class="author">Walter K</span>, <span class="author">Wei J</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Carter NP</span>, <span class="author">Lee C</span>, <span class="author">Scherer SW</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA UK.</p><p class="abstract">Structural variations of DNA greater than 1 kilobase in size account for most bases that vary among human genomes, but are still relatively under-ascertained. Here we use tiling oligonucleotide microarrays, comprising 42 million probes, to generate a comprehensive map of 11,700 copy number variations (CNVs) greater than 443 base pairs, of which most (8,599) have been validated independently. For 4,978 of these CNVs, we generated reference genotypes from 450 individuals of European, African or East Asian ancestry. The predominant mutational mechanisms differ among CNV size classes. Retrotransposition has duplicated and inserted some coding and non-coding DNA segments randomly around the genome. Furthermore, by correlation with known trait-associated single nucleotide polymorphisms (SNPs), we identified 30 loci with CNVs that are candidates for influencing disease susceptibility. Despite this, having assessed the completeness of our map and the patterns of linkage disequilibrium between CNVs and SNPs, we conclude that, for complex traits, the heritability void left by genome-wide association studies will not be accounted for by common CNVs.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; NHGRI NIH HHS: <span class="grant">HG004221</span>; NIGMS NIH HHS: <span class="grant">GM081533</span>; Wellcome Trust: <span class="grant">077006/Z/05/Z</span>, <span class="grant">077008</span>, <span class="grant">077009</span>, <span class="grant">077014</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7289</span>;<span class="pagerange">704-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19812545">19812545</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3330748">3330748</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08516">10.1038/nature08516</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Community-associated methicillin-resistant Staphylococcus aureus infections.</h4>
<p class="authors"><span class="author">Cooke FJ</span> and <span class="author">Brown NM</span></p>
<p class="affiliation">Clinical Microbiology and Public Health Laboratory, Health Protection Agency, Addenbrooke&#39;s Hospital, Cambridge CB2 0QW, UK.</p><p class="abstract">Community-associated methicillin-resistant Staphylococcus aureus (CA-MRSA) has been recognized for over a decade, and usually refers to MRSA identified in previously healthy individuals with no recognized MRSA risk factors. Infections range from minor skin and soft tissue infections, through to severe pneumonia and necrotizing fasciitis. This review summarizes the current data on the epidemiology and molecular features of CA-MRSA, in addition to diagnosis and therapeutic measures. We also refer to current national guidelines for the management of these infections. Areas of agreement include the important genotypic and phenotypic differences of community MRSA strains compared with hospital strains. Areas of controversy include the precise epidemiological definition of community-acquired/associated MRSA. Fortunately, true CA-MRSA can be differentiated from hospital MRSA by molecular techniques, as discussed herein. Recent interest has focused on the changing epidemiology of CA-MRSA. Worldwide, CA-MRSA is now seen outside of the initial specific population groups, and in the USA, the successful USA300 community strain is beginning to spread back into hospitals. Reasons why USA300 remains relatively uncommon in Europe are unclear. Topics timely for research include the investigation of the epidemiology of infections and evolutionary genomics.</p>
<p class="publication"><span class="title">British medical bulletin</span>&nbsp;<span class="year">2010</span>;<span class="volume">94</span>;<span class="pagerange">215-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20360061" rel="external">20360061</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbmb%2Fldq010" rel="external">10.1093/bmb/ldq010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microfluidics for the upstream pipeline of DNA sequencing--a worthy application?</h4>
<p class="authors"><span class="author">Coupland P</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK CB10 1SA. <a href="mailto:%70%63%31%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#99;&#49;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Technological advances and economic investment into DNA sequencing during this decade has provided the industry of genome sequencing with a suite of dedicated sequencing machines capable of rapidly generating vast quantities of sequence data. This next generation of equipment for DNA sequencing is freely available and is utilised more commonly; this has lead to the traditional bottle-neck in the sequencing pipeline transferring from the sequencing process, i.e. reading the bases on the older capillary based machines, to the upstream processes of sample preparation, i.e. creating the DNA libraries that are to be read. Essentially, advancement in sequencing technology is running faster than the equivalent for sample preparation technology and, without a remedy, we will no longer be able to provide samples quick enough to keep the sequencing machines running at full capacity.</p>
<p class="publication"><span class="title">Lab on a chip</span>&nbsp;<span class="year">2010</span>;<span class="volume">10</span>;<span class="number">5</span>;<span class="pagerange">544-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20162226" rel="external">20162226</a>; DOI: <a href="http://dx.doi.org/10.1039%2Fb917560a" rel="external">10.1039/b917560a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Strong genetic evidence for a selective influence of GABAA receptors on a component of the bipolar disorder phenotype.</h4>
<p class="authors"><span class="author">Craddock N</span>, <span class="author">Jones L</span>, <span class="author">Jones IR</span>, <span class="author">Kirov G</span>, <span class="author">Green EK</span>, <span class="author">Grozeva D</span>, <span class="author">Moskvina V</span>, <span class="author">Nikolov I</span>, <span class="author">Hamshere ML</span>, <span class="author">Vukcevic D</span>, <span class="author">Caesar S</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Fraser C</span>, <span class="author">Russell E</span>, <span class="author">Norton N</span>, <span class="author">Breen G</span>, <span class="author">St Clair D</span>, <span class="author">Collier DA</span>, <span class="author">Young AH</span>, <span class="author">Ferrier IN</span>, <span class="author">Farmer A</span>, <span class="author">McGuffin P</span>, <span class="author">Holmans PA</span>, <span class="author">Wellcome Trust Case Control Consortium (WTCCC)</span>, <span class="author">Donnelly P</span>, <span class="author">Owen MJ</span> and <span class="author">O&#39;Donovan MC</span></p>
<p class="affiliation">Department of Psychological Medicine, School of Medicine, Cardiff University, Cardiff, UK. <a href="mailto:%63%72%61%64%64%6F%63%6B%6E%40%63%61%72%64%69%66%66.%61%63.%75%6B">&#99;&#114;&#97;&#100;&#100;&#111;&#99;&#107;&#110;&#64;&#99;&#97;&#114;&#100;&#105;&#102;&#102;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Despite compelling evidence for a major genetic contribution to risk of bipolar mood disorder, conclusive evidence implicating specific genes or pathophysiological systems has proved elusive. In part this is likely to be related to the unknown validity of current phenotype definitions and consequent aetiological heterogeneity of samples. In the recent Wellcome Trust Case Control Consortium genome-wide association analysis of bipolar disorder (1868 cases, 2938 controls) one of the most strongly associated polymorphisms lay within the gene encoding the GABA(A) receptor beta1 subunit, GABRB1. Aiming to increase biological homogeneity, we sought the diagnostic subset that showed the strongest signal at this polymorphism and used this to test for independent evidence of association with other members of the GABA(A) receptor gene family. The index signal was significantly enriched in the 279 cases meeting Research Diagnostic Criteria for schizoaffective disorder, bipolar type (P=3.8 x 10(-6)). Independently, these cases showed strong evidence that variation in GABA(A) receptor genes influences risk for this phenotype (independent system-wide P=6.6 x 10(-5)) with association signals also at GABRA4, GABRB3, GABRA5 and GABRR3. [corrected] Our findings have the potential to inform understanding of presentation, pathogenesis and nosology of bipolar disorders. Our method of phenotype refinement may be useful in studies of other complex psychiatric and non-psychiatric disorders.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">15</span>;<span class="number">2</span>;<span class="pagerange">146-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19078961">19078961</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3967096">3967096</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2008.66">10.1038/mp.2008.66</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Automated design of genomic Southern blot probes.</h4>
<p class="authors"><span class="author">Croning MD</span>, <span class="author">Fricker DG</span>, <span class="author">Komiyama NH</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB101SA, UK.</p><p class="abstract"><strong>Background</strong>: Sothern blotting is a DNA analysis technique that has found widespread application in molecular biology. It has been used for gene discovery and mapping and has diagnostic and forensic applications, including mutation detection in patient samples and DNA fingerprinting in criminal investigations. Southern blotting has been employed as the definitive method for detecting transgene integration, and successful homologous recombination in gene targeting experiments.The technique employs a labeled DNA probe to detect a specific DNA sequence in a complex DNA sample that has been separated by restriction-digest and gel electrophoresis. Critically for the technique to succeed the probe must be unique to the target locus so as not to cross-hybridize to other endogenous DNA within the sample.Investigators routinely employ a manual approach to probe design. A genome browser is used to extract DNA sequence from the locus of interest, which is searched against the target genome using a BLAST-like tool. Ideally a single perfect match is obtained to the target, with little cross-reactivity caused by homologous DNA sequence present in the genome and/or repetitive and low-complexity elements in the candidate probe. This is a labor intensive process often requiring several attempts to find a suitable probe for laboratory testing.</p><p class="abstract"><strong>Results</strong>: We have written an informatic pipeline to automatically design genomic Sothern blot probes that specifically attempts to optimize the resultant probe, employing a brute-force strategy of generating many candidate probes of acceptable length in the user-specified design window, searching all against the target genome, then scoring and ranking the candidates by uniqueness and repetitive DNA element content. Using these in silico measures we can automatically design probes that we predict to perform as well, or better, than our previous manual designs, while considerably reducing design time.We went on to experimentally validate a number of these automated designs by Southern blotting. The majority of probes we tested performed well confirming our in silico prediction methodology and the general usefulness of the software for automated genomic Southern probe design.</p><p class="abstract"><strong>Conclusions</strong>: Software and supplementary information are freely available at: http://www.genes2cognition.org/software/southern_blot.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20113467" rel="external">20113467</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2830989" rel="external">2830989</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-74" rel="external">10.1186/1471-2164-11-74</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A rapid and scalable method for selecting recombinant mouse monoclonal antibodies.</h4>
<p class="authors"><span class="author">Crosnier C</span>, <span class="author">Staudt N</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Cambridge CB10 1HH, UK.</p><p class="abstract"><strong>Background</strong>: Monoclonal antibodies with high affinity and selectivity that work on wholemount fixed tissues are valuable reagents to the cell and developmental biologist, and yet isolating them remains a long and unpredictable process. Here we report a rapid and scalable method to select and express recombinant mouse monoclonal antibodies that are essentially equivalent to those secreted by parental IgG-isotype hybridomas.</p><p class="abstract"><strong>Results</strong>: Increased throughput was achieved by immunizing mice with pools of antigens and cloning - from small numbers of hybridoma cells - the functionally rearranged light and heavy chains into a single expression plasmid. By immunizing with the ectodomains of zebrafish cell surface receptor proteins expressed in mammalian cells and screening for formalin-resistant epitopes, we selected antibodies that gave expected staining patterns on wholemount fixed zebrafish embryos.</p><p class="abstract"><strong>Conclusions</strong>: This method can be used to quickly select several high quality monoclonal antibodies from a single immunized mouse and facilitates their distribution using plasmids.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">R01NS063400</span>; Wellcome Trust: <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">BMC biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="pagerange">76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20525357">20525357</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2898661">2898661</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1741-7007-8-76">10.1186/1741-7007-8-76</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A commensal gone bad: complete genome sequence of the prototypical enterotoxigenic Escherichia coli strain H10407.</h4>
<p class="authors"><span class="author">Crossman LC</span>, <span class="author">Chaudhuri RR</span>, <span class="author">Beatson SA</span>, <span class="author">Wells TJ</span>, <span class="author">Desvaux M</span>, <span class="author">Cunningham AF</span>, <span class="author">Petty NK</span>, <span class="author">Mahon V</span>, <span class="author">Brinkley C</span>, <span class="author">Hobman JL</span>, <span class="author">Savarino SJ</span>, <span class="author">Turner SM</span>, <span class="author">Pallen MJ</span>, <span class="author">Penn CW</span>, <span class="author">Parkhill J</span>, <span class="author">Turner AK</span>, <span class="author">Johnson TJ</span>, <span class="author">Thomson NR</span>, <span class="author">Smith SG</span> and <span class="author">Henderson IR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">In most cases, Escherichia coli exists as a harmless commensal organism, but it may on occasion cause intestinal and/or extraintestinal disease. Enterotoxigenic E. coli (ETEC) is the predominant cause of E. coli-mediated diarrhea in the developing world and is responsible for a significant portion of pediatric deaths. In this study, we determined the complete genomic sequence of E. coli H10407, a prototypical strain of enterotoxigenic E. coli, which reproducibly elicits diarrhea in human volunteer studies. We performed genomic and phylogenetic comparisons with other E. coli strains, revealing that the chromosome is closely related to that of the nonpathogenic commensal strain E. coli HS and to those of the laboratory strains E. coli K-12 and C. Furthermore, these analyses demonstrated that there were no chromosomally encoded factors unique to any sequenced ETEC strains. Comparison of the E. coli H10407 plasmids with those from several ETEC strains revealed that the plasmids had a mosaic structure but that several loci were conserved among ETEC strains. This study provides a genetic context for the vast amount of experimental and epidemiological data that have been published.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C510075/1</span>; Medical Research Council: <span class="grant">G0801209</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">21</span>;<span class="pagerange">5822-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20802035">20802035</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2953697">2953697</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00710-10">10.1128/JB.00710-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Studying bacterial transcriptomes using RNA-seq.</h4>
<p class="authors"><span class="author">Croucher NJ</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Genome-wide studies of bacterial gene expression are shifting from microarray technology to second generation sequencing platforms. RNA-seq has a number of advantages over hybridization-based techniques, such as annotation-independent detection of transcription, improved sensitivity and increased dynamic range. Early studies have uncovered a wealth of novel coding sequences and non-coding RNA, and are revealing a transcriptional landscape that increasingly mirrors that of eukaryotes. Already basic RNA-seq protocols have been improved and adapted to looking at particular aspects of RNA biology, often with an emphasis on non-coding RNAs, and further refinements to current techniques will improve our understanding of gene expression, and genome content, in the future.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">13</span>;<span class="number">5</span>;<span class="pagerange">619-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20888288">20888288</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3025319">3025319</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mib.2010.09.009">10.1016/j.mib.2010.09.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of attractive drug targets in neglected-disease pathogens using an in silico approach.</h4>
<p class="authors"><span class="author">Crowther GJ</span>, <span class="author">Shanmugam D</span>, <span class="author">Carmona SJ</span>, <span class="author">Doyle MA</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Berriman M</span>, <span class="author">Nwaka S</span>, <span class="author">Ralph SA</span>, <span class="author">Roos DS</span>, <span class="author">Van Voorhis WC</span> and <span class="author">Ag&uuml;ero F</span></p>
<p class="affiliation">Division of Allergy and Infectious Diseases, Department of Medicine, University of Washington, Seattle, Washington, United States of America. <a href="mailto:%63%72%6F%77%74%68%65%72%40%75.%77%61%73%68%69%6E%67%74%6F%6E.%65%64%75">&#99;&#114;&#111;&#119;&#116;&#104;&#101;&#114;&#64;&#117;&#46;&#119;&#97;&#115;&#104;&#105;&#110;&#103;&#116;&#111;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: The increased sequencing of pathogen genomes and the subsequent availability of genome-scale functional datasets are expected to guide the experimental work necessary for target-based drug discovery. However, a major bottleneck in this has been the difficulty of capturing and integrating relevant information in an easily accessible format for identifying and prioritizing potential targets. The open-access resource TDRtargets.org facilitates drug target prioritization for major tropical disease pathogens such as the mycobacteria Mycobacterium leprae and Mycobacterium tuberculosis; the kinetoplastid protozoans Leishmania major, Trypanosoma brucei, and Trypanosoma cruzi; the apicomplexan protozoans Plasmodium falciparum, Plasmodium vivax, and Toxoplasma gondii; and the helminths Brugia malayi and Schistosoma mansoni.</p><p class="abstract">Here we present strategies to prioritize pathogen proteins based on whether their properties meet criteria considered desirable in a drug target. These criteria are based upon both sequence-derived information (e.g., molecular mass) and functional data on expression, essentiality, phenotypes, metabolic pathways, assayability, and druggability. This approach also highlights the fact that data for many relevant criteria are lacking in less-studied pathogens (e.g., helminths), and we demonstrate how this can be partially overcome by mapping data from homologous genes in well-studied organisms. We also show how individual users can easily upload external datasets and integrate them with existing data in TDRtargets.org to generate highly customized ranked lists of potential targets.</p><p class="abstract">Using the datasets and the tools available in TDRtargets.org, we have generated illustrative lists of potential drug targets in seven tropical disease pathogens. While these lists are broadly consistent with the research community&#39;s current interest in certain specific proteins, and suggest novel target candidates that may merit further study, the lists can easily be modified in a user-specific manner, either by adjusting the weights for chosen criteria or by changing the criteria that are included.</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="number">8</span>;<span class="pagerange">e804</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20808766" rel="external">20808766</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2927427" rel="external">2927427</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000804" rel="external">10.1371/journal.pntd.0000804</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput analysis of candidate imprinted genes and allele-specific gene expression in the human term placenta.</h4>
<p class="authors"><span class="author">Daelemans C</span>, <span class="author">Ritchie ME</span>, <span class="author">Smits G</span>, <span class="author">Abu-Amero S</span>, <span class="author">Sudbery IM</span>, <span class="author">Forrest MS</span>, <span class="author">Campino S</span>, <span class="author">Clark TG</span>, <span class="author">Stanier P</span>, <span class="author">Kwiatkowski D</span>, <span class="author">Deloukas P</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Tavar&eacute; S</span>, <span class="author">Moore GE</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1 SA, UK.</p><p class="abstract"><strong>Background</strong>: Imprinted genes show expression from one parental allele only and are important for development and behaviour. This extreme mode of allelic imbalance has been described for approximately 56 human genes. Imprinting status is often disrupted in cancer and dysmorphic syndromes. More subtle variation of gene expression, that is not parent-of-origin specific, termed &#39;allele-specific gene expression&#39; (ASE) is more common and may give rise to milder phenotypic differences. Using two allele-specific high-throughput technologies alongside bioinformatics predictions, normal term human placenta was screened to find new imprinted genes and to ascertain the extent of ASE in this tissue.</p><p class="abstract"><strong>Results</strong>: Twenty-three family trios of placental cDNA, placental genomic DNA (gDNA) and gDNA from both parents were tested for 130 candidate genes with the Sequenom MassArray system. Six genes were found differentially expressed but none imprinted. The Illumina ASE BeadArray platform was then used to test 1536 SNPs in 932 genes. The array was enriched for the human orthologues of 124 mouse candidate genes from bioinformatics predictions and 10 human candidate imprinted genes from EST database mining. After quality control pruning, a total of 261 informative SNPs (214 genes) remained for analysis. Imprinting with maternal expression was demonstrated for the lymphocyte imprinted gene ZNF331 in human placenta. Two potential differentially methylated regions (DMRs) were found in the vicinity of ZNF331. None of the bioinformatically predicted candidates tested showed imprinting except for a skewed allelic expression in a parent-specific manner observed for PHACTR2, a neighbour of the imprinted PLAGL1 gene. ASE was detected for two or more individuals in 39 candidate genes (18%).</p><p class="abstract"><strong>Conclusions</strong>: Both Sequenom and Illumina assays were sensitive enough to study imprinting and strong allelic bias. Previous bioinformatics approaches were not predictive of new imprinted genes in the human term placenta. ZNF331 is imprinted in human term placenta and might be a new ubiquitously imprinted gene, part of a primate-specific locus. Demonstration of partial imprinting of PHACTR2 calls for re-evaluation of the allelic pattern of expression for the PHACTR2-PLAGL1 locus. ASE was common in human term placenta.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">BMC genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20403199" rel="external">20403199</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871261" rel="external">2871261</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2156-11-25" rel="external">10.1186/1471-2156-11-25</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic sequencing of renal carcinoma reveals inactivation of histone modifying genes.</h4>
<p class="authors"><span class="author">Dalgliesh GL</span>, <span class="author">Furge K</span>, <span class="author">Greenman C</span>, <span class="author">Chen L</span>, <span class="author">Bignell G</span>, <span class="author">Butler A</span>, <span class="author">Davies H</span>, <span class="author">Edkins S</span>, <span class="author">Hardy C</span>, <span class="author">Latimer C</span>, <span class="author">Teague J</span>, <span class="author">Andrews J</span>, <span class="author">Barthorpe S</span>, <span class="author">Beare D</span>, <span class="author">Buck G</span>, <span class="author">Campbell PJ</span>, <span class="author">Forbes S</span>, <span class="author">Jia M</span>, <span class="author">Jones D</span>, <span class="author">Knott H</span>, <span class="author">Kok CY</span>, <span class="author">Lau KW</span>, <span class="author">Leroy C</span>, <span class="author">Lin ML</span>, <span class="author">McBride DJ</span>, <span class="author">Maddison M</span>, <span class="author">Maguire S</span>, <span class="author">McLay K</span>, <span class="author">Menzies A</span>, <span class="author">Mironenko T</span>, <span class="author">Mulderrig L</span>, <span class="author">Mudie L</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Pleasance E</span>, <span class="author">Rajasingham A</span>, <span class="author">Shepherd R</span>, <span class="author">Smith R</span>, <span class="author">Stebbings L</span>, <span class="author">Stephens P</span>, <span class="author">Tang G</span>, <span class="author">Tarpey PS</span>, <span class="author">Turrell K</span>, <span class="author">Dykema KJ</span>, <span class="author">Khoo SK</span>, <span class="author">Petillo D</span>, <span class="author">Wondergem B</span>, <span class="author">Anema J</span>, <span class="author">Kahnoski RJ</span>, <span class="author">Teh BT</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Clear cell renal cell carcinoma (ccRCC) is the most common form of adult kidney cancer, characterized by the presence of inactivating mutations in the VHL gene in most cases, and by infrequent somatic mutations in known cancer genes. To determine further the genetics of ccRCC, we have sequenced 101 cases through 3,544 protein-coding genes. Here we report the identification of inactivating mutations in two genes encoding enzymes involved in histone modification-SETD2, a histone H3 lysine 36 methyltransferase, and JARID1C (also known as KDM5C), a histone H3 lysine 4 demethylase-as well as mutations in the histone H3 lysine 27 demethylase, UTX (KMD6A), that we recently reported. The results highlight the role of mutations in components of the chromatin modification machinery in human cancer. Furthermore, NF2 mutations were found in non-VHL mutated ccRCC, and several other probable cancer genes were identified. These results indicate that substantial genetic heterogeneity exists in a cancer type dominated by mutations in a single gene, and that systematic screens will be key to fully determining the somatic genetic architecture of cancer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">082359</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">463</span>;<span class="number">7279</span>;<span class="pagerange">360-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20054297">20054297</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2820242">2820242</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08672">10.1038/nature08672</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multi-platform next-generation sequencing of the domestic turkey (Meleagris gallopavo): genome assembly and analysis.</h4>
<p class="authors"><span class="author">Dalloul RA</span>, <span class="author">Long JA</span>, <span class="author">Zimin AV</span>, <span class="author">Aslam L</span>, <span class="author">Beal K</span>, <span class="author">Blomberg Le Ann</span>, <span class="author">Bouffard P</span>, <span class="author">Burt DW</span>, <span class="author">Crasta O</span>, <span class="author">Crooijmans RP</span>, <span class="author">Cooper K</span>, <span class="author">Coulombe RA</span>, <span class="author">De S</span>, <span class="author">Delany ME</span>, <span class="author">Dodgson JB</span>, <span class="author">Dong JJ</span>, <span class="author">Evans C</span>, <span class="author">Frederickson KM</span>, <span class="author">Flicek P</span>, <span class="author">Florea L</span>, <span class="author">Folkerts O</span>, <span class="author">Groenen MA</span>, <span class="author">Harkins TT</span>, <span class="author">Herrero J</span>, <span class="author">Hoffmann S</span>, <span class="author">Megens HJ</span>, <span class="author">Jiang A</span>, <span class="author">de Jong P</span>, <span class="author">Kaiser P</span>, <span class="author">Kim H</span>, <span class="author">Kim KW</span>, <span class="author">Kim S</span>, <span class="author">Langenberger D</span>, <span class="author">Lee MK</span>, <span class="author">Lee T</span>, <span class="author">Mane S</span>, <span class="author">Marcais G</span>, <span class="author">Marz M</span>, <span class="author">McElroy AP</span>, <span class="author">Modise T</span>, <span class="author">Nefedov M</span>, <span class="author">Notredame C</span>, <span class="author">Paton IR</span>, <span class="author">Payne WS</span>, <span class="author">Pertea G</span>, <span class="author">Prickett D</span>, <span class="author">Puiu D</span>, <span class="author">Qioa D</span>, <span class="author">Raineri E</span>, <span class="author">Ruffier M</span>, <span class="author">Salzberg SL</span>, <span class="author">Schatz MC</span>, <span class="author">Scheuring C</span>, <span class="author">Schmidt CJ</span>, <span class="author">Schroeder S</span>, <span class="author">Searle SM</span>, <span class="author">Smith EJ</span>, <span class="author">Smith J</span>, <span class="author">Sonstegard TS</span>, <span class="author">Stadler PF</span>, <span class="author">Tafer H</span>, <span class="author">Tu ZJ</span>, <span class="author">Van Tassell CP</span>, <span class="author">Vilella AJ</span>, <span class="author">Williams KP</span>, <span class="author">Yorke JA</span>, <span class="author">Zhang L</span>, <span class="author">Zhang HB</span>, <span class="author">Zhang X</span>, <span class="author">Zhang Y</span> and <span class="author">Reed KM</span></p>
<p class="affiliation">Avian Immunobiology Laboratory, Department of Animal and Poultry Sciences, Virginia Tech, Blacksburg, Virginia, United States of America.</p><p class="abstract">A synergistic combination of two next-generation sequencing platforms with a detailed comparative BAC physical contig map provided a cost-effective assembly of the genome sequence of the domestic turkey (Meleagris gallopavo). Heterozygosity of the sequenced source genome allowed discovery of more than 600,000 high quality single nucleotide variants. Despite this heterozygosity, the current genome assembly (&sim;1.1 Gb) includes 917 Mb of sequence assigned to specific turkey chromosomes. Annotation identified nearly 16,000 genes, with 15,093 recognized as protein coding and 611 as non-coding RNA genes. Comparative analysis of the turkey, chicken, and zebra finch genomes, and comparing avian to mammalian species, supports the characteristic stability of avian genomes and identifies genes unique to the avian lineage. Clear differences are seen in number and variety of genes of the avian immune system where expansions and novel genes are less frequent than examples of gene loss. The turkey genome sequence provides resources to further understand the evolution of vertebrate genomes and genetic variation underlying economically important quantitative traits in poultry. This integrated approach may be a model for providing both gene and chromosome level assemblies of other species with agricultural, ecological, and evolutionary interest.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">#R01-HG002945</span>, <span class="grant">R01 HG006677-12</span>; NIGMS NIH HHS: <span class="grant">R01 GM083873-08</span>, <span class="grant">R01 GM083873-09</span>; NLM NIH HHS: <span class="grant">#R01-LM006845</span>, <span class="grant">R01 LM006845-09</span>, <span class="grant">R01 LM006845-10</span>, <span class="grant">R01 LM006845-11</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20838655" rel="external">20838655</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2935454" rel="external">2935454</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1000475" rel="external">10.1371/journal.pbio.1000475</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FLU, an amino acid substitution model for influenza proteins.</h4>
<p class="authors"><span class="author">Dang CC</span>, <span class="author">Le QS</span>, <span class="author">Gascuel O</span> and <span class="author">Le VS</span></p>
<p class="affiliation">College of Technology, Vietnam National University Hanoi, Cau Giay, Hanoi, Vietnam.</p><p class="abstract"><strong>Background</strong>: The amino acid substitution model is the core component of many protein analysis systems such as sequence similarity search, sequence alignment, and phylogenetic inference. Although several general amino acid substitution models have been estimated from large and diverse protein databases, they remain inappropriate for analyzing specific species, e.g., viruses. Emerging epidemics of influenza viruses raise the need for comprehensive studies of these dangerous viruses. We propose an influenza-specific amino acid substitution model to enhance the understanding of the evolution of influenza viruses.</p><p class="abstract"><strong>Results</strong>: A maximum likelihood approach was applied to estimate an amino acid substitution model (FLU) from approximately 113,000 influenza protein sequences, consisting of approximately 20 million residues. FLU outperforms 14 widely used models in constructing maximum likelihood phylogenetic trees for the majority of influenza protein alignments. On average, FLU gains approximately 42 log likelihood points with an alignment of 300 sites. Moreover, topologies of trees constructed using FLU and other models are frequently different. FLU does indeed have an impact on likelihood improvement as well as tree topologies. It was implemented in PhyML and can be downloaded from ftp://ftp.sanger.ac.uk/pub/1000genomes/lsq/FLU or included in PhyML 3.0 server at http://www.atgc-montpellier.fr/phyml/.</p><p class="abstract"><strong>Conclusions</strong>: FLU should be useful for any influenza protein analysis system which requires an accurate description of amino acid substitutions.</p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">10</span>;<span class="pagerange">99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20384985" rel="external">20384985</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873421" rel="external">2873421</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-10-99" rel="external">10.1186/1471-2148-10-99</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The crystal structure of a bacterial Sufu-like protein defines a novel group of bacterial proteins that are similar to the N-terminal domain of human Sufu.</h4>
<p class="authors"><span class="author">Das D</span>, <span class="author">Finn RD</span>, <span class="author">Abdubek P</span>, <span class="author">Astakhova T</span>, <span class="author">Axelrod HL</span>, <span class="author">Bakolitsa C</span>, <span class="author">Cai X</span>, <span class="author">Carlton D</span>, <span class="author">Chen C</span>, <span class="author">Chiu HJ</span>, <span class="author">Chiu M</span>, <span class="author">Clayton T</span>, <span class="author">Deller MC</span>, <span class="author">Duan L</span>, <span class="author">Ellrott K</span>, <span class="author">Farr CL</span>, <span class="author">Feuerhelm J</span>, <span class="author">Grant JC</span>, <span class="author">Grzechnik A</span>, <span class="author">Han GW</span>, <span class="author">Jaroszewski L</span>, <span class="author">Jin KK</span>, <span class="author">Klock HE</span>, <span class="author">Knuth MW</span>, <span class="author">Kozbial P</span>, <span class="author">Krishna SS</span>, <span class="author">Kumar A</span>, <span class="author">Lam WW</span>, <span class="author">Marciano D</span>, <span class="author">Miller MD</span>, <span class="author">Morse AT</span>, <span class="author">Nigoghossian E</span>, <span class="author">Nopakun A</span>, <span class="author">Okach L</span>, <span class="author">Puckett C</span>, <span class="author">Reyes R</span>, <span class="author">Tien HJ</span>, <span class="author">Trame CB</span>, <span class="author">van den Bedem H</span>, <span class="author">Weekes D</span>, <span class="author">Wooten T</span>, <span class="author">Xu Q</span>, <span class="author">Yeh A</span>, <span class="author">Zhou J</span>, <span class="author">Hodgson KO</span>, <span class="author">Wooley J</span>, <span class="author">Elsliger MA</span>, <span class="author">Deacon AM</span>, <span class="author">Godzik A</span>, <span class="author">Lesley SA</span> and <span class="author">Wilson IA</span></p>
<p class="affiliation">Joint Center for Structural Genomics. http://www.jcsg.org</p><p class="abstract">Sufu (Suppressor of Fused), a two-domain protein, plays a critical role in regulating Hedgehog signaling and is conserved from flies to humans. A few bacterial Sufu-like proteins have previously been identified based on sequence similarity to the N-terminal domain of eukaryotic Sufu proteins, but none have been structurally or biochemically characterized and their function in bacteria is unknown. We have determined the crystal structure of a more distantly related Sufu-like homolog, NGO1391 from Neisseria gonorrhoeae, at 1.4 &Aring; resolution, which provides the first biophysical characterization of a bacterial Sufu-like protein. The structure revealed a striking similarity to the N-terminal domain of human Sufu (r.m.s.d. of 2.6 &Aring; over 93% of the NGO1391 protein), despite an extremely low sequence identity of &sim;15%. Subsequent sequence analysis revealed that NGO1391 defines a new subset of smaller, Sufu-like proteins that are present in &sim;200 bacterial species and has resulted in expansion of the SUFU (PF05076) family in Pfam.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54 GM074898</span>; Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Protein science : a publication of the Protein Society</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">11</span>;<span class="pagerange">2131-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20836087" rel="external">20836087</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3005784" rel="external">3005784</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpro.497" rel="external">10.1002/pro.497</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The structure of BVU2987 from Bacteroides vulgatus reveals a superfamily of bacterial periplasmic proteins with possible inhibitory function.</h4>
<p class="authors"><span class="author">Das D</span>, <span class="author">Finn RD</span>, <span class="author">Carlton D</span>, <span class="author">Miller MD</span>, <span class="author">Abdubek P</span>, <span class="author">Astakhova T</span>, <span class="author">Axelrod HL</span>, <span class="author">Bakolitsa C</span>, <span class="author">Chen C</span>, <span class="author">Chiu HJ</span>, <span class="author">Chiu M</span>, <span class="author">Clayton T</span>, <span class="author">Deller MC</span>, <span class="author">Duan L</span>, <span class="author">Ellrott K</span>, <span class="author">Ernst D</span>, <span class="author">Farr CL</span>, <span class="author">Feuerhelm J</span>, <span class="author">Grant JC</span>, <span class="author">Grzechnik A</span>, <span class="author">Han GW</span>, <span class="author">Jaroszewski L</span>, <span class="author">Jin KK</span>, <span class="author">Klock HE</span>, <span class="author">Knuth MW</span>, <span class="author">Kozbial P</span>, <span class="author">Krishna SS</span>, <span class="author">Kumar A</span>, <span class="author">Marciano D</span>, <span class="author">McMullan D</span>, <span class="author">Morse AT</span>, <span class="author">Nigoghossian E</span>, <span class="author">Nopakun A</span>, <span class="author">Okach L</span>, <span class="author">Puckett C</span>, <span class="author">Reyes R</span>, <span class="author">Rife CL</span>, <span class="author">Sefcovic N</span>, <span class="author">Tien HJ</span>, <span class="author">Trame CB</span>, <span class="author">van den Bedem H</span>, <span class="author">Weekes D</span>, <span class="author">Wooten T</span>, <span class="author">Xu Q</span>, <span class="author">Hodgson KO</span>, <span class="author">Wooley J</span>, <span class="author">Elsliger MA</span>, <span class="author">Deacon AM</span>, <span class="author">Godzik A</span>, <span class="author">Lesley SA</span> and <span class="author">Wilson IA</span></p>
<p class="affiliation">Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, Menlo Park, CA, USA.</p><p class="abstract">Proteins that contain the DUF2874 domain constitute a new Pfam family PF11396. Members of this family have predominantly been identified in microbes found in the human gut and oral cavity. The crystal structure of one member of this family, BVU2987 from Bacteroides vulgatus, has been determined, revealing a &beta;-lactamase inhibitor protein-like structure with a tandem repeat of domains. Sequence analysis and structural comparisons reveal that BVU2987 and other DUF2874 proteins are related to &beta;-lactamase inhibitor protein, PepSY and SmpA_OmlA proteins and hence are likely to function as inhibitory proteins.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54 GM074898</span>; Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Acta crystallographica. Section F, Structural biology and crystallization communications</span>&nbsp;<span class="year">2010</span>;<span class="volume">66</span>;<span class="number">Pt 10</span>;<span class="pagerange">1265-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20944221" rel="external">20944221</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2954215" rel="external">2954215</a>; DOI: <a href="http://dx.doi.org/10.1107%2FS1744309109046788" rel="external">10.1107/S1744309109046788</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of TBC1D4 in patients with severe insulin resistance.</h4>
<p class="authors"><span class="author">Dash S</span>, <span class="author">Langenberg C</span>, <span class="author">Fawcett KA</span>, <span class="author">Semple RK</span>, <span class="author">Romeo S</span>, <span class="author">Sharp S</span>, <span class="author">Sano H</span>, <span class="author">Lienhard GE</span>, <span class="author">Rochford JJ</span>, <span class="author">Howlett T</span>, <span class="author">Massoud AF</span>, <span class="author">Hindmarsh P</span>, <span class="author">Howell SJ</span>, <span class="author">Wilkinson RJ</span>, <span class="author">Lyssenko V</span>, <span class="author">Groop L</span>, <span class="author">Baroni MG</span>, <span class="author">Barroso I</span>, <span class="author">Wareham NJ</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Savage DB</span></p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600414</span>, <span class="grant">G0800203</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U117588499</span>; NIDDK NIH HHS: <span class="grant">DK25336</span>, <span class="grant">R01 DK025336</span>, <span class="grant">R56 DK025336</span>; Wellcome Trust: <span class="grant">072070</span>, <span class="grant">077016</span>, <span class="grant">088316</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2010</span>;<span class="volume">53</span>;<span class="number">6</span>;<span class="pagerange">1239-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20349035">20349035</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2860565">2860565</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-010-1724-x">10.1007/s00125-010-1724-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proapoptotic Rassf1A/Mst1 signaling in cardiac fibroblasts is protective against pressure overload in mice.</h4>
<p class="authors"><span class="author">Del Re DP</span>, <span class="author">Matsuda T</span>, <span class="author">Zhai P</span>, <span class="author">Gao S</span>, <span class="author">Clark GJ</span>, <span class="author">Van Der Weyden L</span> and <span class="author">Sadoshima J</span></p>
<p class="affiliation">Department of Cell Biology and Molecular Medicine, Cardiovascular Research Institute, New Jersey Medical School, University of Medicine and Dentistry of New Jersey, Newark, New Jersey 07103-2714, USA.</p><p class="abstract">Mammalian sterile 20-like kinase 1 (Mst1) is a mammalian homolog of Drosophila Hippo, the master regulator of cell death, proliferation, and organ size in flies. It is the chief component of the mammalian Hippo pathway and promotes apoptosis and inhibits compensatory cardiac hypertrophy, playing a critical role in mediating heart failure. How Mst1 is regulated, however, remains unclear. Using genetically altered mice in which expression of the tumor suppressor Ras-association domain family 1 isoform A (Rassf1A) was modulated in a cell type-specific manner, we demonstrate here that Rassf1A is an endogenous activator of Mst1 in the heart. Although the Rassf1A/Mst1 pathway promoted apoptosis in cardiomyocytes, thereby playing a detrimental role, the same pathway surprisingly inhibited fibroblast proliferation and cardiac hypertrophy through both cell-autonomous and autocrine/paracrine mechanisms, playing a protective role during pressure overload. In cardiac fibroblasts, the Rassf1A/Mst1 pathway negatively regulated TNF-&alpha;, a key mediator of hypertrophy, fibrosis, and resulting cardiac dysfunction. These results suggest that the functional consequence of activating the proapoptotic Rassf1A/Mst1 pathway during pressure overload is cell type dependent in the heart and that suppressing this mechanism in cardiac fibroblasts could be detrimental.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL099148</span>, <span class="grant">HL59139</span>, <span class="grant">HL67724</span>, <span class="grant">HL69020</span>, <span class="grant">HL91469</span>, <span class="grant">P01 HL059139-060011</span>, <span class="grant">P01 HL069020-080007</span>, <span class="grant">P01 HL069020-090007</span>, <span class="grant">P01 HL069020-100007</span>, <span class="grant">R01 HL067724-08</span>, <span class="grant">R01 HL067724-09</span>, <span class="grant">R01 HL067724-10</span>, <span class="grant">R01 HL067724-11</span>, <span class="grant">R01 HL091469-02</span>, <span class="grant">R01 HL091469-04</span>, <span class="grant">R01 HL102738-02</span>; NIA NIH HHS: <span class="grant">AG27211</span>, <span class="grant">R01 AG023039-08</span></p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2010</span>;<span class="volume">120</span>;<span class="number">10</span>;<span class="pagerange">3555-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20890045" rel="external">20890045</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2947240" rel="external">2947240</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI43569" rel="external">10.1172/JCI43569</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In vivo composition of NMDA receptor signaling complexes differs between membrane subdomains and is modulated by PSD-95 and PSD-93.</h4>
<p class="authors"><span class="author">Delint-Ramirez I</span>, <span class="author">Fern&aacute;ndez E</span>, <span class="author">Bay&eacute;s A</span>, <span class="author">Kicsi E</span>, <span class="author">Komiyama NH</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom.</p><p class="abstract">Lipid rafts are dynamic membrane microdomains enriched in cholesterol and sphingolipids involved in the compartmentalization of signaling pathways, trafficking and sorting of proteins. At synapses, the glutamatergic NMDA receptor and its cytoplasmic scaffold protein PSD-95 move between postsynaptic density (PSD) and rafts following learning or ischemia. However it is not known whether the signaling complexes formed by these proteins are different in rafts nor the molecular mechanisms that govern their localization. To examine these issues in vivo we used mice carrying genetically encoded tags for purification of protein complexes and specific mutations in NMDA receptors, PSD-95 and other postsynaptic scaffold proteins. Isolation of PSD-95 complexes from mice carrying tandem affinity purification tags showed differential composition in lipid rafts, postsynaptic density and detergent-soluble fractions. Raft PSD-95 complexes showed less CaMKIIalpha and SynGAP and enrichment in Src and Arc/Arg3.1 compared with PSD complexes. Mice carrying knock-outs of PSD-95 or PSD-93 show a key role for PSD-95 in localizing NR2A-containing NMDA receptor complexes to rafts. Deletion of the NR2A C terminus or the C-terminal valine residue of NR2B, which prevents all PDZ interactions, reduced the NR1 association with rafts. Interestingly, the deletion of the NR2B valine residue increased the total amount of lipid rafts. These data show critical roles for scaffold proteins and their interactions with NMDA receptor subunits in organizing the differential expression in rafts and postsynaptic densities of synaptic signaling complexes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">066717</span></p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">24</span>;<span class="pagerange">8162-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20554866" rel="external">20554866</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2912510" rel="external">2912510</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.1792-10.2010" rel="external">10.1523/JNEUROSCI.1792-10.2010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential DNA methylation as a tool for noninvasive prenatal diagnosis (NIPD) of X chromosome aneuploidies.</h4>
<p class="authors"><span class="author">Della Ragione F</span>, <span class="author">Mastrovito P</span>, <span class="author">Campanile C</span>, <span class="author">Conti A</span>, <span class="author">Papageorgiou EA</span>, <span class="author">Hult&eacute;n MA</span>, <span class="author">Patsalis PC</span>, <span class="author">Carter NP</span> and <span class="author">D&#39;Esposito M</span></p>
<p class="affiliation">Institute of Genetics and Biophysics A. Buzzati Traverso, Naples, Italy.</p><p class="abstract">The demographic tendency in industrial countries to delay childbearing, coupled with the maternal age effect in common chromosomal aneuploidies and the risk to the fetus of invasive prenatal diagnosis, are potent drivers for the development of strategies for noninvasive prenatal diagnosis. One breakthrough has been the discovery of differentially methylated cell-free fetal DNA in the maternal circulation. We describe novel bisulfite conversion- and methylation-sensitive enzyme digestion DNA methylation-related approaches that we used to diagnose Turner syndrome from first trimester samples. We used an X-linked marker, EF3, and an autosomal marker, RASSF1A, to discriminate between placental and maternal blood cell DNA using real-time methylation-specific PCR after bisulfite conversion and real-time PCR after methylation-sensitive restriction digestion. By normalizing EF3 amplifications versus RASSF1A outputs, we were able to calculate sex chromosome/autosome ratios in chorionic villus samples, thus permitting us to correctly diagnose Turner syndrome. The identification of this new marker coupled with the strategy outlined here may be instrumental in the development of an efficient, noninvasive method of diagnosis of sex chromosome aneuploidies in plasma samples.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">The Journal of molecular diagnostics : JMD</span>&nbsp;<span class="year">2010</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">797-807</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20847278" rel="external">20847278</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2963918" rel="external">2963918</a>; DOI: <a href="http://dx.doi.org/10.2353%2Fjmoldx.2010.090199" rel="external">10.2353/jmoldx.2010.090199</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Protein variation in blood-dwelling schistosome worms generated by differential splicing of micro-exon gene transcripts.</h4>
<p class="authors"><span class="author">DeMarco R</span>, <span class="author">Mathieson W</span>, <span class="author">Manuel SJ</span>, <span class="author">Dillon GP</span>, <span class="author">Curwen RS</span>, <span class="author">Ashton PD</span>, <span class="author">Ivens AC</span>, <span class="author">Berriman M</span>, <span class="author">Verjovski-Almeida S</span> and <span class="author">Wilson RA</span></p>
<p class="affiliation">Department of Biology, University of York, York YO10 5YW, United Kingdom. <a href="mailto:%72%64%65%6D%61%72%63%6F%40%69%66%73%63.%75%73%70.%62%72">&#114;&#100;&#101;&#109;&#97;&#114;&#99;&#111;&#64;&#105;&#102;&#115;&#99;&#46;&#117;&#115;&#112;&#46;&#98;&#114;</a></p><p class="abstract">Schistosoma mansoni is a well-adapted blood-dwelling parasitic helminth, persisting for decades in its human host despite being continually exposed to potential immune attack. Here, we describe in detail micro-exon genes (MEG) in S. mansoni, some present in multiple copies, which represent a novel molecular system for creating protein variation through the alternate splicing of short (&lt; or =36 bp) symmetric exons organized in tandem. Analysis of three closely related copies of one MEG family allowed us to trace several evolutionary events and propose a mechanism for micro-exon generation and diversification. Microarray experiments show that the majority of MEGs are up-regulated in life cycle stages associated with establishment in the mammalian host after skin penetration. Sequencing of RT-PCR products allowed the description of several alternate splice forms of micro-exon genes, highlighting the potential use of these transcripts to generate a complex pool of protein variants. We obtained direct evidence for the existence of such pools by proteomic analysis of secretions from migrating schistosomula and mature eggs. Whole-mount in situ hybridization and immunolocalization showed that MEG transcripts and proteins were restricted to glands or epithelia exposed to the external environment. The ability of schistosomes to produce a complex pool of variant proteins aligns them with the other major groups of blood parasites, but using a completely different mechanism. We believe that our data open a new chapter in the study of immune evasion by schistosomes, and their ability to generate variant proteins could represent a significant obstacle to vaccine development.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; NIAID NIH HHS: <span class="grant">AI054711-01A2</span>; Wellcome Trust: <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2010</span>;<span class="volume">20</span>;<span class="number">8</span>;<span class="pagerange">1112-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20606017" rel="external">20606017</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2909574" rel="external">2909574</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.100099.109" rel="external">10.1101/gr.100099.109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Leishmania-specific surface antigens show sub-genus sequence variation and immune recognition.</h4>
<p class="authors"><span class="author">Depledge DP</span>, <span class="author">MacLean LM</span>, <span class="author">Hodgkinson MR</span>, <span class="author">Smith BA</span>, <span class="author">Jackson AP</span>, <span class="author">Ma S</span>, <span class="author">Uliana SR</span> and <span class="author">Smith DF</span></p>
<p class="affiliation">Centre for Immunology and Infection, Department of Biology, Hull York Medical School, University of York, York, United Kingdom.</p><p class="abstract"><strong>Background</strong>: A family of hydrophilic acylated surface (HASP) proteins, containing extensive and variant amino acid repeats, is expressed at the plasma membrane in infective extracellular (metacyclic) and intracellular (amastigote) stages of Old World Leishmania species. While HASPs are antigenic in the host and can induce protective immune responses, the biological functions of these Leishmania-specific proteins remain unresolved. Previous genome analysis has suggested that parasites of the sub-genus Leishmania (Viannia) have lost HASP genes from their genomes.</p><p class="abstract">We have used molecular and cellular methods to analyse HASP expression in New World Leishmania mexicana complex species and show that, unlike in L. major, these proteins are expressed predominantly following differentiation into amastigotes within macrophages. Further genome analysis has revealed that the L. (Viannia) species, L. (V.) braziliensis, does express HASP-like proteins of low amino acid similarity but with similar biochemical characteristics, from genes present on a region of chromosome 23 that is syntenic with the HASP/SHERP locus in Old World Leishmania species and the L. (L.) mexicana complex. A related gene is also present in Leptomonas seymouri and this may represent the ancestral copy of these Leishmania-genus specific sequences. The L. braziliensis HASP-like proteins (named the orthologous (o) HASPs) are predominantly expressed on the plasma membrane in amastigotes and are recognised by immune sera taken from 4 out of 6 leishmaniasis patients tested in an endemic region of Brazil. Analysis of the repetitive domains of the oHASPs has shown considerable genetic variation in parasite isolates taken from the same patients, suggesting that antigenic change may play a role in immune recognition of this protein family.</p><p class="abstract">These findings confirm that antigenic hydrophilic acylated proteins are expressed from genes in the same chromosomal region in species across the genus Leishmania. These proteins are surface-exposed on amastigotes (although L. (L.) major parasites also express HASPB on the metacyclic plasma membrane). The central repetitive domains of the HASPs are highly variant in their amino acid sequences, both within and between species, consistent with a role in immune recognition in the host.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council: <span class="grant">G0900950</span>, <span class="grant">G9721629</span>; Wellcome Trust: <span class="grant">048615</span>, <span class="grant">076355</span>, <span class="grant">077503</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="number">9</span>;<span class="pagerange">e829</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20927190">20927190</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2946902">2946902</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000829">10.1371/journal.pntd.0000829</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic approaches uncover increasing complexities in the regulatory landscape at the human SCL (TAL1) locus.</h4>
<p class="authors"><span class="author">Dhami P</span>, <span class="author">Bruce AW</span>, <span class="author">Jim JH</span>, <span class="author">Dillon SC</span>, <span class="author">Hall A</span>, <span class="author">Cooper JL</span>, <span class="author">Bonhoure N</span>, <span class="author">Chiang K</span>, <span class="author">Ellis PD</span>, <span class="author">Langford C</span>, <span class="author">Andrews RM</span> and <span class="author">Vetrie D</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">The SCL (TAL1) transcription factor is a critical regulator of haematopoiesis and its expression is tightly controlled by multiple cis-acting regulatory elements. To elaborate further the DNA elements which control its regulation, we used genomic tiling microarrays covering 256 kb of the human SCL locus to perform a concerted analysis of chromatin structure and binding of regulatory proteins in human haematopoietic cell lines. This approach allowed us to characterise further or redefine known human SCL regulatory elements and led to the identification of six novel elements with putative regulatory function both up and downstream of the SCL gene. They bind a number of haematopoietic transcription factors (GATA1, E2A LMO2, SCL, LDB1), CTCF or components of the transcriptional machinery and are associated with relevant histone modifications, accessible chromatin and low nucleosomal density. Functional characterisation shows that these novel elements are able to enhance or repress SCL promoter activity, have endogenous promoter function or enhancer-blocking insulator function. Our analysis opens up several areas for further investigation and adds new layers of complexity to our understanding of the regulation of SCL expression.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">2</span>;<span class="pagerange">e9059</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20140202" rel="external">20140202</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2816701" rel="external">2816701</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0009059" rel="external">10.1371/journal.pone.0009059</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complex exon-intron marking by histone modifications is not determined solely by nucleosome distribution.</h4>
<p class="authors"><span class="author">Dhami P</span>, <span class="author">Saffrey P</span>, <span class="author">Bruce AW</span>, <span class="author">Dillon SC</span>, <span class="author">Chiang K</span>, <span class="author">Bonhoure N</span>, <span class="author">Koch CM</span>, <span class="author">Bye J</span>, <span class="author">James K</span>, <span class="author">Foad NS</span>, <span class="author">Ellis P</span>, <span class="author">Watkins NA</span>, <span class="author">Ouwehand WH</span>, <span class="author">Langford C</span>, <span class="author">Andrews RM</span>, <span class="author">Dunham I</span> and <span class="author">Vetrie D</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">It has recently been shown that nucleosome distribution, histone modifications and RNA polymerase II (Pol II) occupancy show preferential association with exons (&quot;exon-intron marking&quot;), linking chromatin structure and function to co-transcriptional splicing in a variety of eukaryotes. Previous ChIP-sequencing studies suggested that these marking patterns reflect the nucleosomal landscape. By analyzing ChIP-chip datasets across the human genome in three cell types, we have found that this marking system is far more complex than previously observed. We show here that a range of histone modifications and Pol II are preferentially associated with exons. However, there is noticeable cell-type specificity in the degree of exon marking by histone modifications and, surprisingly, this is also reflected in some histone modifications patterns showing biases towards introns. Exon-intron marking is laid down in the absence of transcription on silent genes, with some marking biases changing or becoming reversed for genes expressed at different levels. Furthermore, the relationship of this marking system with splicing is not simple, with only some histone modifications reflecting exon usage/inclusion, while others mirror patterns of exon exclusion. By examining nucleosomal distributions in all three cell types, we demonstrate that these histone modification patterns cannot solely be accounted for by differences in nucleosome levels between exons and introns. In addition, because of inherent differences between ChIP-chip array and ChIP-sequencing approaches, these platforms report different nucleosome distribution patterns across the human genome. Our findings confound existing views and point to active cellular mechanisms which dynamically regulate histone modification levels and account for exon-intron marking. We believe that these histone modification patterns provide links between chromatin accessibility, Pol II movement and co-transcriptional splicing.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01HG003168</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">8</span>;<span class="pagerange">e12339</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20808788" rel="external">20808788</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2925886" rel="external">2925886</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0012339" rel="external">10.1371/journal.pone.0012339</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ectodomains of the LDL receptor-related proteins LRP1b and LRP4 have anchorage independent functions in vivo.</h4>
<p class="authors"><span class="author">Dietrich MF</span>, <span class="author">van der Weyden L</span>, <span class="author">Prosser HM</span>, <span class="author">Bradley A</span>, <span class="author">Herz J</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Department of Molecular Genetics, UT Southwestern, Dallas, Texas, United States of America.</p><p class="abstract"><strong>Background</strong>: The low-density lipoprotein (LDL) receptor gene family is a highly conserved group of membrane receptors with diverse functions in developmental processes, lipoprotein trafficking, and cell signaling. The low-density lipoprotein (LDL) receptor-related protein 1b (LRP1B) was reported to be deleted in several types of human malignancies, including non-small cell lung cancer. Our group has previously reported that a distal extracellular truncation of murine Lrp1b that is predicted to secrete the entire intact extracellular domain (ECD) is fully viable with no apparent phenotype.</p><p class="abstract">Here, we have used a gene targeting approach to create two mouse lines carrying internally rearranged exons of Lrp1b that are predicted to truncate the protein closer to the N-terminus and to prevent normal trafficking through the secretary pathway. Both mutations result in early embryonic lethality, but, as expected from the restricted expression pattern of LRP1b in vivo, loss of Lrp1b does not cause cellular lethality as homozygous Lrp1b-deficient blastocysts can be propagated normally in culture. This is similar to findings for another LDL receptor family member, Lrp4. We provide in vitro evidence that Lrp4 undergoes regulated intramembraneous processing through metalloproteases and gamma-secretase cleavage. We further demonstrate negative regulation of the Wnt signaling pathway by the soluble extracellular domain.</p><p class="abstract">Our results underline a crucial role for Lrp1b in development. The expression in mice of truncated alleles of Lrp1b and Lrp4 with deletions of the transmembrane and intracellular domains leads to release of the extracellular domain into the extracellular space, which is sufficient to confer viability. In contrast, null mutations are embryonically (Lrp1b) or perinatally (Lrp4) lethal. These findings suggest that the extracellular domains of both proteins may function as a scavenger for signaling ligands or signal modulators in the extracellular space, thereby preserving signaling thresholds that are critical for embryonic development, as well as for the clear, but poorly understood role of LRP1b in cancer.</p>
<p class="grants">Funded by: Cancer Research UK; NHLBI NIH HHS: <span class="grant">R37 HL063762</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">e9960</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20383322">20383322</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2850915">2850915</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0009960">10.1371/journal.pone.0009960</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A parasite calcium switch and Achilles&#39; heel revealed.</h4>
<p class="authors"><span class="author">Doerig C</span> and <span class="author">Billker O</span></p>
<p class="affiliation">Institut National de la Sant&eacute; et de la Recherche M&eacute;dicale (INSERM)-Ecole Polytechnique F&eacute;d&eacute;rale de Lausanne (EPFL) Joint Laboratory, INSERM U609, Global Health Institute, Ecole Polytechnique F&eacute;d&eacute;rale de Lausanne, Lausanne, Switzerland. <a href="mailto:%63%68%72%69%73%74%69%61%6E.%64%6F%65%72%69%67%40%65%70%6C%66.%63%68">&#99;&#104;&#114;&#105;&#115;&#116;&#105;&#97;&#110;&#46;&#100;&#111;&#101;&#114;&#105;&#103;&#64;&#101;&#112;&#108;&#102;&#46;&#99;&#104;</a></p>
<p class="publication"><span class="title">Nature structural &amp; molecular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">17</span>;<span class="number">5</span>;<span class="pagerange">541-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20442739" rel="external">20442739</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnsmb0510-541" rel="external">10.1038/nsmb0510-541</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessment of the neuropeptide S system in anxiety disorders.</h4>
<p class="authors"><span class="author">Donner J</span>, <span class="author">Haapakoski R</span>, <span class="author">Ezer S</span>, <span class="author">Mel&eacute;n E</span>, <span class="author">Pirkola S</span>, <span class="author">Gratac&ograve;s M</span>, <span class="author">Zucchelli M</span>, <span class="author">Anedda F</span>, <span class="author">Johansson LE</span>, <span class="author">S&ouml;derh&auml;ll C</span>, <span class="author">Orsmark-Pietras C</span>, <span class="author">Suvisaari J</span>, <span class="author">Mart&iacute;n-Santos R</span>, <span class="author">Torrens M</span>, <span class="author">Silander K</span>, <span class="author">Terwilliger JD</span>, <span class="author">Wickman M</span>, <span class="author">Pershagen G</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Peltonen L</span>, <span class="author">Estivill X</span>, <span class="author">D&#39;Amato M</span>, <span class="author">Kere J</span>, <span class="author">Alenius H</span> and <span class="author">Hovatta I</span></p>
<p class="affiliation">Research Program of Molecular Neurology, Biomedicum Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: The G protein-coupled receptor neuropeptide S receptor 1 (NPSR1) and its ligand neuropeptide S (NPS) form a signaling system mainly implicated in susceptibility to asthma and inflammatory disorders in humans and regulation of anxiety and arousal in rodents. We addressed here the role of NPS and NPSR1 as susceptibility genes for human anxiety disorders.</p><p class="abstract"><strong>Methods</strong>: We performed comprehensive association analysis of genetic variants in NPS and NPSR1 in three independent study samples. We first studied a population-based sample (Health 2000, Finland) of 321 anxiety disorder patients and 1317 control subjects and subsequently a Spanish clinical panic disorder sample consisting of 188 cases and 315 control subjects. In addition, we examined a birth cohort of 2020 children (Barn Allergi Milj&ouml; Stockholm Epidemiologi [BAMSE], Sweden). We then tested whether alleles of the most significantly associated single nucleotide polymorphisms alter DNA-protein complex formation in electrophoretic mobility shift assays. Finally, we compared acute stress responses on the gene expression level in wild-type and Npsr1(-/-) mice.</p><p class="abstract"><strong>Results</strong>: We confirmed previously observed epidemiological association between anxiety and asthma in two population-based cohorts. Single nucleotide polymorphisms within NPS and NPSR1 associated with panic disorder diagnosis in the Finnish and Spanish samples and with parent-reported anxiety/depression in the BAMSE sample. Moreover, some of the implicated single nucleotide polymorphisms potentially affect transcription factor binding. Expression of neurotrophin-3, a neurotrophic factor connected to stress and panic reaction, was significantly downregulated in brain regions of stressed Npsr1(-/-) mice, whereas interleukin-1 beta, an active stress-related immunotransmitter, was upregulated.</p><p class="abstract"><strong>Conclusions</strong>: Our results suggest that NPS-NPSR1 signaling is likely involved in anxiety.</p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">68</span>;<span class="number">5</span>;<span class="pagerange">474-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20705147">20705147</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2010.05.039">10.1016/j.biopsych.2010.05.039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new era in the genomics of bacteria.</h4>
<p class="authors"><span class="author">Dougan G</span> and <span class="author">Weinstock GM</span></p>
<p class="publication"><span class="title">Current opinion in microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">13</span>;<span class="number">5</span>;<span class="pagerange">616-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20846901" rel="external">20846901</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mib.2010.08.008" rel="external">10.1016/j.mib.2010.08.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide analysis reveals loci encoding anti-macrophage factors in the human pathogen Burkholderia pseudomallei K96243.</h4>
<p class="authors"><span class="author">Dowling AJ</span>, <span class="author">Wilkinson PA</span>, <span class="author">Holden MT</span>, <span class="author">Quail MA</span>, <span class="author">Bentley SD</span>, <span class="author">Reger J</span>, <span class="author">Waterfield NR</span>, <span class="author">Titball RW</span> and <span class="author">Ffrench-Constant RH</span></p>
<p class="affiliation">Biosciences, University of Exeter, Penryn, United Kingdom. <a href="mailto:%61%6A%64%32%31%31%40%65%78%65%74%65%72.%61%63.%75%6B">&#97;&#106;&#100;&#50;&#49;&#49;&#64;&#101;&#120;&#101;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Burkholderia pseudomallei is an important human pathogen whose infection biology is still poorly understood. The bacterium is endemic to tropical regions, including South East Asia and Northern Australia, where it causes melioidosis, a serious disease associated with both high mortality and antibiotic resistance. B. pseudomallei is a Gram-negative facultative intracellular pathogen that is able to replicate in macrophages. However despite the critical nature of its interaction with macrophages, few anti-macrophage factors have been characterized to date. Here we perform a genome-wide gain of function screen of B. pseudomallei strain K96243 to identify loci encoding factors with anti-macrophage activity. We identify a total of 113 such loci scattered across both chromosomes, with positive gene clusters encoding transporters and secretion systems, enzymes/toxins, secondary metabolite, biofilm, adhesion and signal response related factors. Further phenotypic analysis of four of these regions shows that the encoded factors cause striking cellular phenotypes relevant to infection biology, including apoptosis, formation of actin &#39;tails&#39; and multi-nucleation within treated macrophages. The detailed analysis of the remaining host of loci will facilitate genetic dissection of the interaction of this important pathogen with host macrophages and thus further elucidate this critical part of its infection cycle.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E021182/1</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">12</span>;<span class="pagerange">e15693</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21203527" rel="external">21203527</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3008741" rel="external">3008741</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0015693" rel="external">10.1371/journal.pone.0015693</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple common variants for celiac disease influencing immune gene expression.</h4>
<p class="authors"><span class="author">Dubois PC</span>, <span class="author">Trynka G</span>, <span class="author">Franke L</span>, <span class="author">Hunt KA</span>, <span class="author">Romanos J</span>, <span class="author">Curtotti A</span>, <span class="author">Zhernakova A</span>, <span class="author">Heap GA</span>, <span class="author">Ad&aacute;ny R</span>, <span class="author">Aromaa A</span>, <span class="author">Bardella MT</span>, <span class="author">van den Berg LH</span>, <span class="author">Bockett NA</span>, <span class="author">de la Concha EG</span>, <span class="author">Dema B</span>, <span class="author">Fehrmann RS</span>, <span class="author">Fern&aacute;ndez-Arquero M</span>, <span class="author">Fiatal S</span>, <span class="author">Grandone E</span>, <span class="author">Green PM</span>, <span class="author">Groen HJ</span>, <span class="author">Gwilliam R</span>, <span class="author">Houwen RH</span>, <span class="author">Hunt SE</span>, <span class="author">Kaukinen K</span>, <span class="author">Kelleher D</span>, <span class="author">Korponay-Szabo I</span>, <span class="author">Kurppa K</span>, <span class="author">MacMathuna P</span>, <span class="author">M&auml;ki M</span>, <span class="author">Mazzilli MC</span>, <span class="author">McCann OT</span>, <span class="author">Mearin ML</span>, <span class="author">Mein CA</span>, <span class="author">Mirza MM</span>, <span class="author">Mistry V</span>, <span class="author">Mora B</span>, <span class="author">Morley KI</span>, <span class="author">Mulder CJ</span>, <span class="author">Murray JA</span>, <span class="author">N&uacute;&ntilde;ez C</span>, <span class="author">Oosterom E</span>, <span class="author">Ophoff RA</span>, <span class="author">Polanco I</span>, <span class="author">Peltonen L</span>, <span class="author">Platteel M</span>, <span class="author">Rybak A</span>, <span class="author">Salomaa V</span>, <span class="author">Schweizer JJ</span>, <span class="author">Sperandeo MP</span>, <span class="author">Tack GJ</span>, <span class="author">Turner G</span>, <span class="author">Veldink JH</span>, <span class="author">Verbeek WH</span>, <span class="author">Weersma RK</span>, <span class="author">Wolters VM</span>, <span class="author">Urcelay E</span>, <span class="author">Cukrowska B</span>, <span class="author">Greco L</span>, <span class="author">Neuhausen SL</span>, <span class="author">McManus R</span>, <span class="author">Barisani D</span>, <span class="author">Deloukas P</span>, <span class="author">Barrett JC</span>, <span class="author">Saavalainen P</span>, <span class="author">Wijmenga C</span> and <span class="author">van Heel DA</span></p>
<p class="affiliation">Blizard Institute of Cell and Molecular Science, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London, UK.</p><p class="abstract">We performed a second-generation genome-wide association study of 4,533 individuals with celiac disease (cases) and 10,750 control subjects. We genotyped 113 selected SNPs with P(GWAS) &lt; 10(-4) and 18 SNPs from 14 known loci in a further 4,918 cases and 5,684 controls. Variants from 13 new regions reached genome-wide significance (P(combined) &lt; 5 x 10(-8)); most contain genes with immune functions (BACH2, CCR4, CD80, CIITA-SOCS1-CLEC16A, ICOSLG and ZMIZ1), with ETS1, RUNX3, THEMIS and TNFRSF14 having key roles in thymic T-cell selection. There was evidence to suggest associations for a further 13 regions. In an expression quantitative trait meta-analysis of 1,469 whole blood samples, 20 of 38 (52.6%) tested loci had celiac risk variants correlated (P &lt; 0.0028, FDR 5%) with cis gene expression.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700545</span>, <span class="grant">G0700545(82277)</span>; NIDDK NIH HHS: <span class="grant">DK050678</span>, <span class="grant">DK071003</span>, <span class="grant">DK081645</span>, <span class="grant">DK57892</span>, <span class="grant">R01 DK081645</span>; NINDS NIH HHS: <span class="grant">NS058980</span>; Wellcome Trust: <span class="grant">084743</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">4</span>;<span class="pagerange">295-302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20190752">20190752</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2847618">2847618</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.543">10.1038/ng.543</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk.</h4>
<p class="authors"><span class="author">Dupuis J</span>, <span class="author">Langenberg C</span>, <span class="author">Prokopenko I</span>, <span class="author">Saxena R</span>, <span class="author">Soranzo N</span>, <span class="author">Jackson AU</span>, <span class="author">Wheeler E</span>, <span class="author">Glazer NL</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Gloyn AL</span>, <span class="author">Lindgren CM</span>, <span class="author">M&auml;gi R</span>, <span class="author">Morris AP</span>, <span class="author">Randall J</span>, <span class="author">Johnson T</span>, <span class="author">Elliott P</span>, <span class="author">Rybin D</span>, <span class="author">Thorleifsson G</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Henneman P</span>, <span class="author">Grallert H</span>, <span class="author">Dehghan A</span>, <span class="author">Hottenga JJ</span>, <span class="author">Franklin CS</span>, <span class="author">Navarro P</span>, <span class="author">Song K</span>, <span class="author">Goel A</span>, <span class="author">Perry JR</span>, <span class="author">Egan JM</span>, <span class="author">Lajunen T</span>, <span class="author">Grarup N</span>, <span class="author">Spars&oslash; T</span>, <span class="author">Doney A</span>, <span class="author">Voight BF</span>, <span class="author">Stringham HM</span>, <span class="author">Li M</span>, <span class="author">Kanoni S</span>, <span class="author">Shrader P</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Kumari M</span>, <span class="author">Qi L</span>, <span class="author">Timpson NJ</span>, <span class="author">Gieger C</span>, <span class="author">Zabena C</span>, <span class="author">Rocheleau G</span>, <span class="author">Ingelsson E</span>, <span class="author">An P</span>, <span class="author">O&#39;Connell J</span>, <span class="author">Luan J</span>, <span class="author">Elliott A</span>, <span class="author">McCarroll SA</span>, <span class="author">Payne F</span>, <span class="author">Roccasecca RM</span>, <span class="author">Pattou F</span>, <span class="author">Sethupathy P</span>, <span class="author">Ardlie K</span>, <span class="author">Ariyurek Y</span>, <span class="author">Balkau B</span>, <span class="author">Barter P</span>, <span class="author">Beilby JP</span>, <span class="author">Ben-Shlomo Y</span>, <span class="author">Benediktsson R</span>, <span class="author">Bennett AJ</span>, <span class="author">Bergmann S</span>, <span class="author">Bochud M</span>, <span class="author">Boerwinkle E</span>, <span class="author">Bonnefond A</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Borch-Johnsen K</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Brunner E</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Charpentier G</span>, <span class="author">Chen YD</span>, <span class="author">Chines P</span>, <span class="author">Clarke R</span>, <span class="author">Coin LJ</span>, <span class="author">Cooper MN</span>, <span class="author">Cornelis M</span>, <span class="author">Crawford G</span>, <span class="author">Crisponi L</span>, <span class="author">Day IN</span>, <span class="author">de Geus EJ</span>, <span class="author">Delplanque J</span>, <span class="author">Dina C</span>, <span class="author">Erdos MR</span>, <span class="author">Fedson AC</span>, <span class="author">Fischer-Rosinsky A</span>, <span class="author">Forouhi NG</span>, <span class="author">Fox CS</span>, <span class="author">Frants R</span>, <span class="author">Franzosi MG</span>, <span class="author">Galan P</span>, <span class="author">Goodarzi MO</span>, <span class="author">Graessler J</span>, <span class="author">Groves CJ</span>, <span class="author">Grundy S</span>, <span class="author">Gwilliam R</span>, <span class="author">Gyllensten U</span>, <span class="author">Hadjadj S</span>, <span class="author">Hallmans G</span>, <span class="author">Hammond N</span>, <span class="author">Han X</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hassanali N</span>, <span class="author">Hayward C</span>, <span class="author">Heath SC</span>, <span class="author">Hercberg S</span>, <span class="author">Herder C</span>, <span class="author">Hicks AA</span>, <span class="author">Hillman DR</span>, <span class="author">Hingorani AD</span>, <span class="author">Hofman A</span>, <span class="author">Hui J</span>, <span class="author">Hung J</span>, <span class="author">Isomaa B</span>, <span class="author">Johnson PR</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Jula A</span>, <span class="author">Kaakinen M</span>, <span class="author">Kaprio J</span>, <span class="author">Kesaniemi YA</span>, <span class="author">Kivimaki M</span>, <span class="author">Knight B</span>, <span class="author">Koskinen S</span>, <span class="author">Kovacs P</span>, <span class="author">Kyvik KO</span>, <span class="author">Lathrop GM</span>, <span class="author">Lawlor DA</span>, <span class="author">Le Bacquer O</span>, <span class="author">Lecoeur C</span>, <span class="author">Li Y</span>, <span class="author">Lyssenko V</span>, <span class="author">Mahley R</span>, <span class="author">Mangino M</span>, <span class="author">Manning AK</span>, <span class="author">Mart&iacute;nez-Larrad MT</span>, <span class="author">McAteer JB</span>, <span class="author">McCulloch LJ</span>, <span class="author">McPherson R</span>, <span class="author">Meisinger C</span>, <span class="author">Melzer D</span>, <span class="author">Meyre D</span>, <span class="author">Mitchell BD</span>, <span class="author">Morken MA</span>, <span class="author">Mukherjee S</span>, <span class="author">Naitza S</span>, <span class="author">Narisu N</span>, <span class="author">Neville MJ</span>, <span class="author">Oostra BA</span>, <span class="author">Orr&ugrave; M</span>, <span class="author">Pakyz R</span>, <span class="author">Palmer CN</span>, <span class="author">Paolisso G</span>, <span class="author">Pattaro C</span>, <span class="author">Pearson D</span>, <span class="author">Peden JF</span>, <span class="author">Pedersen NL</span>, <span class="author">Perola M</span>, <span class="author">Pfeiffer AF</span>, <span class="author">Pichler I</span>, <span class="author">Polasek O</span>, <span class="author">Posthuma D</span>, <span class="author">Potter SC</span>, <span class="author">Pouta A</span>, <span class="author">Province MA</span>, <span class="author">Psaty BM</span>, <span class="author">Rathmann W</span>, <span class="author">Rayner NW</span>, <span class="author">Rice K</span>, <span class="author">Ripatti S</span>, <span class="author">Rivadeneira F</span>, <span class="author">Roden M</span>, <span class="author">Rolandsson O</span>, <span class="author">Sandbaek A</span>, <span class="author">Sandhu M</span>, <span class="author">Sanna S</span>, <span class="author">Sayer AA</span>, <span class="author">Scheet P</span>, <span class="author">Scott LJ</span>, <span class="author">Seedorf U</span>, <span class="author">Sharp SJ</span>, <span class="author">Shields B</span>, <span class="author">Sigurethsson G</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Silveira A</span>, <span class="author">Simpson L</span>, <span class="author">Singleton A</span>, <span class="author">Smith NL</span>, <span class="author">Sovio U</span>, <span class="author">Swift A</span>, <span class="author">Syddall H</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Tanaka T</span>, <span class="author">Thorand B</span>, <span class="author">Tichet J</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Tuomi T</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Dijk KW</span>, <span class="author">van Hoek M</span>, <span class="author">Varma D</span>, <span class="author">Visvikis-Siest S</span>, <span class="author">Vitart V</span>, <span class="author">Vogelzangs N</span>, <span class="author">Waeber G</span>, <span class="author">Wagner PJ</span>, <span class="author">Walley A</span>, <span class="author">Walters GB</span>, <span class="author">Ward KL</span>, <span class="author">Watkins H</span>, <span class="author">Weedon MN</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Witteman JC</span>, <span class="author">Yarnell JW</span>, <span class="author">Zeggini E</span>, <span class="author">Zelenika D</span>, <span class="author">Zethelius B</span>, <span class="author">Zhai G</span>, <span class="author">Zhao JH</span>, <span class="author">Zillikens MC</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">GIANT Consortium</span>, <span class="author">Global BPgen Consortium</span>, <span class="author">Borecki IB</span>, <span class="author">Loos RJ</span>, <span class="author">Meneton P</span>, <span class="author">Magnusson PK</span>, <span class="author">Nathan DM</span>, <span class="author">Williams GH</span>, <span class="author">Hattersley AT</span>, <span class="author">Silander K</span>, <span class="author">Salomaa V</span>, <span class="author">Smith GD</span>, <span class="author">Bornstein SR</span>, <span class="author">Schwarz P</span>, <span class="author">Spranger J</span>, <span class="author">Karpe F</span>, <span class="author">Shuldiner AR</span>, <span class="author">Cooper C</span>, <span class="author">Dedoussis GV</span>, <span class="author">Serrano-R&iacute;os M</span>, <span class="author">Morris AD</span>, <span class="author">Lind L</span>, <span class="author">Palmer LJ</span>, <span class="author">Hu FB</span>, <span class="author">Franks PW</span>, <span class="author">Ebrahim S</span>, <span class="author">Marmot M</span>, <span class="author">Kao WH</span>, <span class="author">Pankow JS</span>, <span class="author">Sampson MJ</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Hansen T</span>, <span class="author">Pedersen O</span>, <span class="author">Pramstaller PP</span>, <span class="author">Wichmann HE</span>, <span class="author">Illig T</span>, <span class="author">Rudan I</span>, <span class="author">Wright AF</span>, <span class="author">Stumvoll M</span>, <span class="author">Campbell H</span>, <span class="author">Wilson JF</span>, <span class="author">Anders Hamsten on behalf of Procardis Consortium</span>, <span class="author">MAGIC investigators</span>, <span class="author">Bergman RN</span>, <span class="author">Buchanan TA</span>, <span class="author">Collins FS</span>, <span class="author">Mohlke KL</span>, <span class="author">Tuomilehto J</span>, <span class="author">Valle TT</span>, <span class="author">Altshuler D</span>, <span class="author">Rotter JI</span>, <span class="author">Siscovick DS</span>, <span class="author">Penninx BW</span>, <span class="author">Boomsma DI</span>, <span class="author">Deloukas P</span>, <span class="author">Spector TD</span>, <span class="author">Frayling TM</span>, <span class="author">Ferrucci L</span>, <span class="author">Kong A</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">van Duijn CM</span>, <span class="author">Aulchenko YS</span>, <span class="author">Cao A</span>, <span class="author">Scuteri A</span>, <span class="author">Schlessinger D</span>, <span class="author">Uda M</span>, <span class="author">Ruokonen A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Waterworth DM</span>, <span class="author">Vollenweider P</span>, <span class="author">Peltonen L</span>, <span class="author">Mooser V</span>, <span class="author">Abecasis GR</span>, <span class="author">Wareham NJ</span>, <span class="author">Sladek R</span>, <span class="author">Froguel P</span>, <span class="author">Watanabe RM</span>, <span class="author">Meigs JB</span>, <span class="author">Groop L</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span>, <span class="author">Florez JC</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Department of Biostatistics, Boston University School of Public Health, Massachusetts, USA.</p><p class="abstract">Levels of circulating glucose are tightly regulated. To identify new loci influencing glycemic traits, we performed meta-analyses of 21 genome-wide association studies informative for fasting glucose, fasting insulin and indices of beta-cell function (HOMA-B) and insulin resistance (HOMA-IR) in up to 46,186 nondiabetic participants. Follow-up of 25 loci in up to 76,558 additional subjects identified 16 loci associated with fasting glucose and HOMA-B and two loci associated with fasting insulin and HOMA-IR. These include nine loci newly associated with fasting glucose (in or near ADCY5, MADD, ADRA2A, CRY2, FADS1, GLIS3, SLC2A2, PROX1 and C2CD4B) and one influencing fasting insulin and HOMA-IR (near IGF1). We also demonstrated association of ADCY5, PROX1, GCK, GCKR and DGKB-TMEM195 with type 2 diabetes. Within these loci, likely biological candidate genes influence signal transduction, cell proliferation, development, glucose-sensing and circadian regulation. Our results demonstrate that genetic studies of glycemic traits can identify type 2 diabetes risk loci, as well as loci containing gene variants that are associated with a modest elevation in glucose levels but are not associated with overt diabetes.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/07/008/23674</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0100222</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G0700222</span>, <span class="grant">G0700222(81696)</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0902037</span>, <span class="grant">G19/35</span>, <span class="grant">G8802774</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U127592696</span>, <span class="grant">MC_U137686857</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NIDDK NIH HHS: <span class="grant">K24 DK080140</span>, <span class="grant">P30 DK040561</span>, <span class="grant">P30 DK040561-14</span>, <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK029867</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK078616</span>, <span class="grant">R01 DK078616-01A1</span>; The Dunhill Medical Trust: <span class="grant">R69/0208</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">077011</span>, <span class="grant">077016</span>, <span class="grant">081682</span>, <span class="grant">088885</span>, <span class="grant">089061</span>, <span class="grant">091746</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">2</span>;<span class="pagerange">105-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20081858">20081858</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3018764">3018764</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.520">10.1038/ng.520</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Traces of sub-Saharan and Middle Eastern lineages in Indian Muslim populations.</h4>
<p class="authors"><span class="author">Eaaswarkhanth M</span>, <span class="author">Haque I</span>, <span class="author">Ravesh Z</span>, <span class="author">Romero IG</span>, <span class="author">Meganathan PR</span>, <span class="author">Dubey B</span>, <span class="author">Khan FA</span>, <span class="author">Chaubey G</span>, <span class="author">Kivisild T</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Singh L</span> and <span class="author">Thangaraj K</span></p>
<p class="affiliation">National DNA Analysis Centre, Central Forensic Science Laboratory, Kolkata, India.</p><p class="abstract">Islam is the second most practiced religion in India, next to Hinduism. It is still unclear whether the spread of Islam in India has been only a cultural transformation or is associated with detectable levels of gene flow. To estimate the contribution of West Asian and Arabian admixture to Indian Muslims, we assessed genetic variation in mtDNA, Y-chromosomal and LCT/MCM6 markers in 472, 431 and 476 samples, respectively, representing six Muslim communities from different geographical regions of India. We found that most of the Indian Muslim populations received their major genetic input from geographically close non-Muslim populations. However, low levels of likely sub-Saharan African, Arabian and West Asian admixture were also observed among Indian Muslims in the form of L0a2a2 mtDNA and E1b1b1a and J(*)(xJ2) Y-chromosomal lineages. The distinction between Iranian and Arabian sources was difficult to make with mtDNA and the Y chromosome, as the estimates were highly correlated because of similar gene pool compositions in the sources. In contrast, the LCT/MCM6 locus, which shows a clear distinction between the two sources, enabled us to rule out significant gene flow from Arabia. Overall, our results support a model according to which the spread of Islam in India was predominantly cultural conversion associated with minor but still detectable levels of gene flow from outside, primarily from Iran and Central Asia, rather than directly from the Arabian Peninsula.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">3</span>;<span class="pagerange">354-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19809480">19809480</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2859343">2859343</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2009.168">10.1038/ejhg.2009.168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prostate cancer in BRCA2 germline mutation carriers is associated with poorer prognosis.</h4>
<p class="authors"><span class="author">Edwards SM</span>, <span class="author">Evans DG</span>, <span class="author">Hope Q</span>, <span class="author">Norman AR</span>, <span class="author">Barbachano Y</span>, <span class="author">Bullock S</span>, <span class="author">Kote-Jarai Z</span>, <span class="author">Meitz J</span>, <span class="author">Falconer A</span>, <span class="author">Osin P</span>, <span class="author">Fisher C</span>, <span class="author">Guy M</span>, <span class="author">Jhavar SG</span>, <span class="author">Hall AL</span>, <span class="author">O&#39;Brien LT</span>, <span class="author">Gehr-Swain BN</span>, <span class="author">Wilkinson RA</span>, <span class="author">Forrest MS</span>, <span class="author">Dearnaley DP</span>, <span class="author">Ardern-Jones AT</span>, <span class="author">Page EC</span>, <span class="author">Easton DF</span>, <span class="author">Eeles RA</span> and <span class="author">UK Genetic Prostate Cancer Study Collaborators and BAUS Section of Oncology</span></p>
<p class="affiliation">Oncogenetics team, Section of Cancer Genetics, Institute of Cancer Research, Sutton SM2 5PT, UK.</p><p class="abstract"><strong>Background</strong>: The germline BRCA2 mutation is associated with increased prostate cancer (PrCa) risk. We have assessed survival in young PrCa cases with a germline mutation in BRCA2 and investigated loss of heterozygosity at BRCA2 in their tumours.</p><p class="abstract"><strong>Methods</strong>: Two cohorts were compared: one was a group with young-onset PrCa, tested for germline BRCA2 mutations (6 of 263 cases had a germline BRAC2 mutation), and the second was a validation set consisting of a clinical set from Manchester of known BRCA2 mutuation carriers (15 cases) with PrCa. Survival data were compared with a control series of patients in a single clinic as determined by Kaplan-Meier estimates. Loss of heterozygosity was tested for in the DNA of tumour tissue of the young-onset group by typing four microsatellite markers that flanked the BRCA2 gene, followed by sequencing.</p><p class="abstract"><strong>Results</strong>: Median survival of all PrCa cases with a germline BRCA2 mutation was shorter at 4.8 years than was survival in controls at 8.5 years (P=0.002). Loss of heterozygosity was found in the majority of tumours of BRCA2 mutation carriers. Multivariate analysis confirmed that the poorer survival of PrCa in BRCA2 mutation carriers is associated with the germline BRCA2 mutation per se.</p><p class="abstract"><strong>Conclusion</strong>: BRCA2 germline mutation is an independent prognostic factor for survival in PrCa. Such patients should not be managed with active surveillance as they have more aggressive disease.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A3354</span>, <span class="grant">C5047/A3354</span></p>
<p class="publication"><span class="title">British journal of cancer</span>&nbsp;<span class="year">2010</span>;<span class="volume">103</span>;<span class="number">6</span>;<span class="pagerange">918-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20736950" rel="external">20736950</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2948551" rel="external">2948551</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.bjc.6605822" rel="external">10.1038/sj.bjc.6605822</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate malaria susceptibility/protective SNPs in hospital and population-based studies: the effect of sub-structuring.</h4>
<p class="authors"><span class="author">Eid NA</span>, <span class="author">Hussein AA</span>, <span class="author">Elzein AM</span>, <span class="author">Mohamed HS</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Ibrahim ME</span></p>
<p class="affiliation">Department of Molecular Biology, Institute of Endemic Diseases, Medical Campus, Qasser Street, University of Khartoum, Khartoum, Sudan.</p><p class="abstract"><strong>Background</strong>: Populations of East Africa including Sudan, exhibit some of the highest indices of genetic diversity in the continent and worldwide. The current study aims to address the possible impact of population structure and population stratification on the outcome of case-control association-analysis of malaria candidate-genes in different Sudanese populations, where the pronounced genetic heterogeneity becomes a source of concern for the potential effect on the studies outcome.</p><p class="abstract"><strong>Methods</strong>: A total of 72 SNPs were genotyped using the Sequenom iPLEX Gold assay in 449 DNA samples that included; cases and controls from two village populations, malaria patients and out-patients from the area of Sinnar and additional controls consisting of healthy Nilo-Saharan speaking individuals. The population substructure was estimated using the Structure 2.2 programme.</p><p class="abstract">The Hardy-Weinberg Equilibrium values were generally within expectation in Hausa and Massalit. However, in the Sinnar area there was a notable excess of homozygosity, which was attributed to the Whalund effect arising from population amalgamation within the sample. The programme STRUCTURE revealed a division of both Hausa and Massalit into two substructures with the partition in Hausa more pronounced than in Massalit; In Sinnar there was no defined substructure. More than 25 of the 72 SNPs assayed were informative in all areas. Some important SNPs were not differentially distributed between malaria cases and controls, including SNPs in CD36 and NOS2. A number of SNPs showed significant p-values for differences in distribution of genotypes between cases and controls including: rs1805015 (in IL4R1) (P = 0.001), rs17047661 (in CR1) (P = 0.02) and rs1800750 (TNF-376)(P = 0.01) in the hospital samples; rs1050828 (G6PD+202) (P = 0.02) and rs1800896 (IL10-1082) (P = 0.04) in Massalit and rs2243250 (IL4-589) (P = 0.04) in Hausa.</p><p class="abstract"><strong>Conclusions</strong>: The difference in population structure partly accounts for some of these significant associations, and the strength of association proved to be sensitive to all levels of sub-structuring whether in the hospital or population-based study.</p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="pagerange">119</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20459687" rel="external">20459687</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2877684" rel="external">2877684</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-9-119" rel="external">10.1186/1475-2875-9-119</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Thirty new loci for age at menarche identified by a meta-analysis of genome-wide association studies.</h4>
<p class="authors"><span class="author">Elks CE</span>, <span class="author">Perry JR</span>, <span class="author">Sulem P</span>, <span class="author">Chasman DI</span>, <span class="author">Franceschini N</span>, <span class="author">He C</span>, <span class="author">Lunetta KL</span>, <span class="author">Visser JA</span>, <span class="author">Byrne EM</span>, <span class="author">Cousminer DL</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Esko T</span>, <span class="author">Feenstra B</span>, <span class="author">Hottenga JJ</span>, <span class="author">Koller DL</span>, <span class="author">Kutalik Z</span>, <span class="author">Lin P</span>, <span class="author">Mangino M</span>, <span class="author">Marongiu M</span>, <span class="author">McArdle PF</span>, <span class="author">Smith AV</span>, <span class="author">Stolk L</span>, <span class="author">van Wingerden SH</span>, <span class="author">Zhao JH</span>, <span class="author">Albrecht E</span>, <span class="author">Corre T</span>, <span class="author">Ingelsson E</span>, <span class="author">Hayward C</span>, <span class="author">Magnusson PK</span>, <span class="author">Smith EN</span>, <span class="author">Ulivi S</span>, <span class="author">Warrington NM</span>, <span class="author">Zgaga L</span>, <span class="author">Alavere H</span>, <span class="author">Amin N</span>, <span class="author">Aspelund T</span>, <span class="author">Bandinelli S</span>, <span class="author">Barroso I</span>, <span class="author">Berenson GS</span>, <span class="author">Bergmann S</span>, <span class="author">Blackburn H</span>, <span class="author">Boerwinkle E</span>, <span class="author">Buring JE</span>, <span class="author">Busonero F</span>, <span class="author">Campbell H</span>, <span class="author">Chanock SJ</span>, <span class="author">Chen W</span>, <span class="author">Cornelis MC</span>, <span class="author">Couper D</span>, <span class="author">Coviello AD</span>, <span class="author">d&#39;Adamo P</span>, <span class="author">de Faire U</span>, <span class="author">de Geus EJ</span>, <span class="author">Deloukas P</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Smith GD</span>, <span class="author">Easton DF</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Emilsson V</span>, <span class="author">Eriksson J</span>, <span class="author">Ferrucci L</span>, <span class="author">Folsom AR</span>, <span class="author">Foroud T</span>, <span class="author">Garcia M</span>, <span class="author">Gasparini P</span>, <span class="author">Geller F</span>, <span class="author">Gieger C</span>, <span class="author">GIANT Consortium</span>, <span class="author">Gudnason V</span>, <span class="author">Hall P</span>, <span class="author">Hankinson SE</span>, <span class="author">Ferreli L</span>, <span class="author">Heath AC</span>, <span class="author">Hernandez DG</span>, <span class="author">Hofman A</span>, <span class="author">Hu FB</span>, <span class="author">Illig T</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Johnson AD</span>, <span class="author">Karasik D</span>, <span class="author">Khaw KT</span>, <span class="author">Kiel DP</span>, <span class="author">Kilpel&auml;inen TO</span>, <span class="author">Kolcic I</span>, <span class="author">Kraft P</span>, <span class="author">Launer LJ</span>, <span class="author">Laven JS</span>, <span class="author">Li S</span>, <span class="author">Liu J</span>, <span class="author">Levy D</span>, <span class="author">Martin NG</span>, <span class="author">McArdle WL</span>, <span class="author">Melbye M</span>, <span class="author">Mooser V</span>, <span class="author">Murray JC</span>, <span class="author">Murray SS</span>, <span class="author">Nalls MA</span>, <span class="author">Navarro P</span>, <span class="author">Nelis M</span>, <span class="author">Ness AR</span>, <span class="author">Northstone K</span>, <span class="author">Oostra BA</span>, <span class="author">Peacock M</span>, <span class="author">Palmer LJ</span>, <span class="author">Palotie A</span>, <span class="author">Par&eacute; G</span>, <span class="author">Parker AN</span>, <span class="author">Pedersen NL</span>, <span class="author">Peltonen L</span>, <span class="author">Pennell CE</span>, <span class="author">Pharoah P</span>, <span class="author">Polasek O</span>, <span class="author">Plump AS</span>, <span class="author">Pouta A</span>, <span class="author">Porcu E</span>, <span class="author">Rafnar T</span>, <span class="author">Rice JP</span>, <span class="author">Ring SM</span>, <span class="author">Rivadeneira F</span>, <span class="author">Rudan I</span>, <span class="author">Sala C</span>, <span class="author">Salomaa V</span>, <span class="author">Sanna S</span>, <span class="author">Schlessinger D</span>, <span class="author">Schork NJ</span>, <span class="author">Scuteri A</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">Shuldiner AR</span>, <span class="author">Soranzo N</span>, <span class="author">Sovio U</span>, <span class="author">Srinivasan SR</span>, <span class="author">Strachan DP</span>, <span class="author">Tammesoo ML</span>, <span class="author">Tikkanen E</span>, <span class="author">Toniolo D</span>, <span class="author">Tsui K</span>, <span class="author">Tryggvadottir L</span>, <span class="author">Tyrer J</span>, <span class="author">Uda M</span>, <span class="author">van Dam RM</span>, <span class="author">van Meurs JB</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Wareham NJ</span>, <span class="author">Waterworth DM</span>, <span class="author">Weedon MN</span>, <span class="author">Wichmann HE</span>, <span class="author">Willemsen G</span>, <span class="author">Wilson JF</span>, <span class="author">Wright AF</span>, <span class="author">Young L</span>, <span class="author">Zhai G</span>, <span class="author">Zhuang WV</span>, <span class="author">Bierut LJ</span>, <span class="author">Boomsma DI</span>, <span class="author">Boyd HA</span>, <span class="author">Crisponi L</span>, <span class="author">Demerath EW</span>, <span class="author">van Duijn CM</span>, <span class="author">Econs MJ</span>, <span class="author">Harris TB</span>, <span class="author">Hunter DJ</span>, <span class="author">Loos RJ</span>, <span class="author">Metspalu A</span>, <span class="author">Montgomery GW</span>, <span class="author">Ridker PM</span>, <span class="author">Spector TD</span>, <span class="author">Streeten EA</span>, <span class="author">Stefansson K</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Widen E</span>, <span class="author">Murabito JM</span>, <span class="author">Ong KK</span> and <span class="author">Murray A</span></p>
<p class="affiliation">Medical Research Council (MRC) Epidemiology Unit, Institute of Metabolic Science, Addenbrooke&#39;s Hospital, Cambridge, UK.</p><p class="abstract">To identify loci for age at menarche, we performed a meta-analysis of 32 genome-wide association studies in 87,802 women of European descent, with replication in up to 14,731 women. In addition to the known loci at LIN28B (P = 5.4 &times; 10&#x207B;&#x2076;&#x2070;) and 9q31.2 (P = 2.2 &times; 10&#x207B;&sup3;&sup3;), we identified 30 new menarche loci (all P &lt; 5 &times; 10&#x207B;&#x2078;) and found suggestive evidence for a further 10 loci (P &lt; 1.9 &times; 10&#x207B;&#x2076;). The new loci included four previously associated with body mass index (in or near FTO, SEC16B, TRA2B and TMEM18), three in or near other genes implicated in energy homeostasis (BSX, CRTC1 and MCHR2) and three in or near genes implicated in hormonal regulation (INHBA, PCSK2 and RXRG). Ingenuity and gene-set enrichment pathway analyses identified coenzyme A and fatty acid biosynthesis as biological processes related to menarche timing.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research: <span class="grant">166067</span>; Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11022</span>, <span class="grant">A10119</span>, <span class="grant">A10124</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401527</span>, <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">G0701863</span>, <span class="grant">G9815508</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>; NCI NIH HHS: <span class="grant">CA047988</span>, <span class="grant">CA089392</span>, <span class="grant">CA104021</span>, <span class="grant">CA136792</span>, <span class="grant">CA40356</span>, <span class="grant">CA54281</span>, <span class="grant">CA63464</span>, <span class="grant">CA98233</span>, <span class="grant">P01 CA055075</span>, <span class="grant">P01 CA055075-17</span>, <span class="grant">P01 CA087969</span>, <span class="grant">P01 CA087969-13</span>, <span class="grant">P01 CA089392</span>, <span class="grant">P01 CA089392-08</span>, <span class="grant">P01 CA089392-09</span>, <span class="grant">P01CA055075</span>, <span class="grant">P01CA087969</span>, <span class="grant">R01 CA040356-15S1</span>, <span class="grant">R01 CA047988</span>, <span class="grant">R01 CA047988-20</span>, <span class="grant">R01 CA063464</span>, <span class="grant">R01 CA063464-10</span>, <span class="grant">R01 CA104021-05</span>, <span class="grant">R37 CA054281</span>, <span class="grant">R37 CA054281-17</span>, <span class="grant">U01 CA098233</span>, <span class="grant">U01 CA098233-08</span>, <span class="grant">U01 CA136792</span>, <span class="grant">U01 CA136792-03</span>, <span class="grant">Z01 CP010200-03</span>, <span class="grant">Z01CP010200</span>; NCRR NIH HHS: <span class="grant">M01 RR 16500</span>, <span class="grant">M01 RR-00750</span>, <span class="grant">M01 RR000750-31</span>, <span class="grant">M01 RR016500-04</span>, <span class="grant">U54RR025204-01</span>, <span class="grant">UL1 RR025005</span>, <span class="grant">UL1 RR025005-05</span>, <span class="grant">UL1 RR025774</span>, <span class="grant">UL1 RR025774-05</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01 HG004399</span>, <span class="grant">U01 HG004399-02</span>, <span class="grant">U01 HG004402</span>, <span class="grant">U01 HG004402-02</span>, <span class="grant">U01 HG004415-02</span>, <span class="grant">U01 HG004422</span>, <span class="grant">U01 HG004422-01</span>, <span class="grant">U01 HG004422-02</span>, <span class="grant">U01 HG004423</span>, <span class="grant">U01 HG004423-01</span>, <span class="grant">U01 HG004424-04</span>, <span class="grant">U01 HG004436</span>, <span class="grant">U01 HG004436-02</span>, <span class="grant">U01 HG004438</span>, <span class="grant">U01 HG004438-04</span>, <span class="grant">U01 HG004446</span>, <span class="grant">U01 HG004446-04</span>, <span class="grant">U01 HG004726</span>, <span class="grant">U01 HG004726-02</span>, <span class="grant">U01 HG004728</span>, <span class="grant">U01 HG004728-01</span>, <span class="grant">U01 HG004729</span>, <span class="grant">U01 HG004729-02</span>, <span class="grant">U01 HG004735</span>, <span class="grant">U01 HG004735-02</span>, <span class="grant">U01 HG004738</span>, <span class="grant">U01 HG004738-02</span>, <span class="grant">U01HG004399</span>, <span class="grant">U01HG004402</span>, <span class="grant">U01HG004415</span>, <span class="grant">U01HG004422</span>, <span class="grant">U01HG004423</span>, <span class="grant">U01HG004436</span>, <span class="grant">U01HG004438</span>, <span class="grant">U01HG004446</span>, <span class="grant">U01HG004728</span>, <span class="grant">U01HG004729</span>, <span class="grant">U01HG004735</span>, <span class="grant">U01HG004738</span>, <span class="grant">U01HG04424</span>; NHLBI NIH HHS: <span class="grant">HL 043851</span>, <span class="grant">HL087679</span>, <span class="grant">HL69757</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC055016</span>, <span class="grant">N01 HC055018</span>, <span class="grant">N01 HC055019</span>, <span class="grant">N01 HC055020</span>, <span class="grant">N01 HC055021</span>, <span class="grant">N01 HC055022</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N02 HL64278</span>, <span class="grant">R01 HL043851</span>, <span class="grant">R01 HL043851-10</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL059367-11</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL086694-03</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087641-03</span>, <span class="grant">R01 HL087679-03</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL088119-04</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>, <span class="grant">RC2 HL102419</span>, <span class="grant">RC2 HL102419-02</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U01 HL084756-03</span>, <span class="grant">U01 HL84756</span>, <span class="grant">U01HL72515</span>, <span class="grant">U19 HL069757</span>, <span class="grant">U19 HL069757-11</span>; NIA NIH HHS: <span class="grant">AG-16592</span>, <span class="grant">N.1-AG-1-1</span>, <span class="grant">N.1-AG-1-2111</span>, <span class="grant">N01 AG012100</span>, <span class="grant">N01 AG012109</span>, <span class="grant">N01 AG050002</span>, <span class="grant">N01-AG-1-2109</span>, <span class="grant">N01-AG-12100</span>, <span class="grant">N01-AG-5-0002</span>, <span class="grant">P01 AG018397</span>, <span class="grant">P01 AG018397-08</span>, <span class="grant">P01 AG025204-03</span>, <span class="grant">P01-AG-18397</span>, <span class="grant">R01 AG016592</span>, <span class="grant">R01 AG016592-10</span>, <span class="grant">R01 AG041517</span>, <span class="grant">R01 AR/AG 41398</span>, <span class="grant">R21 AG032598</span>, <span class="grant">R21 AG032598-02</span>, <span class="grant">R21AG032598</span>; NIAAA NIH HHS: <span class="grant">AA07535</span>, <span class="grant">AA10248</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">AA14041</span>, <span class="grant">K05 AA017688</span>, <span class="grant">R01 AA007535</span>, <span class="grant">R01 AA007535-08</span>, <span class="grant">R01 AA013320</span>, <span class="grant">R01 AA013320-05</span>, <span class="grant">R01 AA013321</span>, <span class="grant">R01 AA013321-05</span>, <span class="grant">R01 AA013326-05</span>, <span class="grant">R01 AA014041-05</span>, <span class="grant">U10 AA008401</span>, <span class="grant">U10 AA008401-23</span>, <span class="grant">U10AA008401</span>; NIAMS NIH HHS: <span class="grant">R01 AR041398</span>, <span class="grant">R01 AR041398-15</span>, <span class="grant">R01 AR041398-20</span>; NICHD NIH HHS: <span class="grant">HD-061437</span>, <span class="grant">R03 HD061437</span>, <span class="grant">R03 HD061437-02</span>; NIDA NIH HHS: <span class="grant">R01 DA012854</span>, <span class="grant">R01 DA012854-09</span>, <span class="grant">R01 DA013423</span>, <span class="grant">R01 DA013423-05</span>, <span class="grant">R01 DA019963</span>, <span class="grant">R01 DA019963-01A2</span>, <span class="grant">R01 DA019963-02</span>, <span class="grant">R01 DA019963-03</span>, <span class="grant">R01-DA013423</span>; NIDCR NIH HHS: <span class="grant">U01 DE018903</span>, <span class="grant">U01 DE018903-02</span>, <span class="grant">U01 DE018993</span>, <span class="grant">U01 DE018993-01</span>, <span class="grant">U01DE018903</span>, <span class="grant">U01DE018993</span>; NIDDK NIH HHS: <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK058845</span>, <span class="grant">R01 DK058845-11</span>, <span class="grant">R01DK058845</span>, <span class="grant">U01 DK062418</span>, <span class="grant">U01 DK062418-06</span>; NIMH NIH HHS: <span class="grant">MH66206</span>, <span class="grant">R01 MH066206</span>, <span class="grant">R01 MH066206-05</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>, <span class="grant">263 MD821336</span>, <span class="grant">263 MD9164 13</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>, <span class="grant">R01-088119</span>, <span class="grant">RFAHG006033</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076467/Z/05/Z</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span>, <span class="grant">89061/Z/09/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">12</span>;<span class="pagerange">1077-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21102462">21102462</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3140055">3140055</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.714">10.1038/ng.714</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluation of association of HNF1B variants with diverse cancers: collaborative analysis of data from 19 genome-wide association studies.</h4>
<p class="authors"><span class="author">Elliott KS</span>, <span class="author">Zeggini E</span>, <span class="author">McCarthy MI</span>, <span class="author">Gudmundsson J</span>, <span class="author">Sulem P</span>, <span class="author">Stacey SN</span>, <span class="author">Thorlacius S</span>, <span class="author">Amundadottir L</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Xu J</span>, <span class="author">Gaborieau V</span>, <span class="author">Eeles RA</span>, <span class="author">Neal DE</span>, <span class="author">Donovan JL</span>, <span class="author">Hamdy FC</span>, <span class="author">Muir K</span>, <span class="author">Hwang SJ</span>, <span class="author">Spitz MR</span>, <span class="author">Zanke B</span>, <span class="author">Carvajal-Carmona L</span>, <span class="author">Brown KM</span>, <span class="author">Australian Melanoma Family Study Investigators</span>, <span class="author">Hayward NK</span>, <span class="author">Macgregor S</span>, <span class="author">Tomlinson IP</span>, <span class="author">Lemire M</span>, <span class="author">Amos CI</span>, <span class="author">Murabito JM</span>, <span class="author">Isaacs WB</span>, <span class="author">Easton DF</span>, <span class="author">Brennan P</span>, <span class="author">PanScan Consortium</span>, <span class="author">Barkardottir RB</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Rafnar T</span>, <span class="author">Hunter DJ</span>, <span class="author">Chanock SJ</span>, <span class="author">Stefansson K</span> and <span class="author">Ioannidis JP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Genome-wide association studies have found type 2 diabetes-associated variants in the HNF1B gene to exhibit reciprocal associations with prostate cancer risk. We aimed to identify whether these variants may have an effect on cancer risk in general versus a specific effect on prostate cancer only.</p><p class="abstract">In a collaborative analysis, we collected data from GWAS of cancer phenotypes for the frequently reported variants of HNF1B, rs4430796 and rs7501939, which are in linkage disequilibrium (r(2) = 0.76, HapMap CEU). Overall, the analysis included 16 datasets on rs4430796 with 19,640 cancer cases and 21,929 controls; and 21 datasets on rs7501939 with 26,923 cases and 49,085 controls. Malignancies other than prostate cancer included colorectal, breast, lung and pancreatic cancers, and melanoma. Meta-analysis showed large between-dataset heterogeneity that was driven by different effects in prostate cancer and other cancers. The per-T2D-risk-allele odds ratios (95% confidence intervals) for rs4430796 were 0.79 (0.76, 0.83)] per G allele for prostate cancer (p&lt;10(-15) for both); and 1.03 (0.99, 1.07) for all other cancers. Similarly for rs7501939 the per-T2D-risk-allele odds ratios (95% confidence intervals) were 0.80 (0.77, 0.83) per T allele for prostate cancer (p&lt;10(-15) for both); and 1.00 (0.97, 1.04) for all other cancers. No malignancy other than prostate cancer had a nominally statistically significant association.</p><p class="abstract">The examined HNF1B variants have a highly specific effect on prostate cancer risk with no apparent association with any of the other studied cancer types.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01-CA83115</span>; NHLBI NIH HHS: <span class="grant">N01-HC-25195</span>, <span class="grant">N02-HL-6-4278</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">e10858</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20526366" rel="external">20526366</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2878330" rel="external">2878330</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0010858" rel="external">10.1371/journal.pone.0010858</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high-throughput pharmaceutical screen identifies compounds with specific toxicity against BRCA2-deficient tumors.</h4>
<p class="authors"><span class="author">Evers B</span>, <span class="author">Schut E</span>, <span class="author">van der Burg E</span>, <span class="author">Braumuller TM</span>, <span class="author">Egan DA</span>, <span class="author">Holstege H</span>, <span class="author">Edser P</span>, <span class="author">Adams DJ</span>, <span class="author">Wade-Martins R</span>, <span class="author">Bouwman P</span> and <span class="author">Jonkers J</span></p>
<p class="affiliation">Division of Molecular Biology, The Netherlands Cancer Institute, Amsterdam, the Netherlands.</p><p class="abstract"><strong>Purpose</strong>: Hereditary breast cancer is partly explained by germline mutations in BRCA1 and BRCA2. Although patients carry heterozygous mutations, their tumors have typically lost the remaining wild-type allele. Selectively targeting BRCA deficiency may therefore constitute an important therapeutic approach. Clinical trials applying this principle are underway, but it is unknown whether the compounds tested are optimal. It is therefore important to identify alternative compounds that specifically target BRCA deficiency and to test new combination therapies to establish optimal treatment strategies.</p><p class="abstract">We did a high-throughput pharmaceutical screen on BRCA2-deficient mouse mammary tumor cells and isogenic controls with restored BRCA2 function. Subsequently, we validated positive hits in vitro and in vivo using mice carrying BRCA2-deficient mammary tumors.</p><p class="abstract"><strong>Results</strong>: Three alkylators-chlorambucil, melphalan, and nimustine-displayed strong and specific toxicity against BRCA2-deficient cells. In vivo, these showed heterogeneous but generally strong BRCA2-deficient antitumor activity, with melphalan and nimustine doing better than cisplatin and the poly-(ADP-ribose)-polymerase inhibitor olaparib (AZD2281) in this small study. In vitro drug combination experiments showed synergistic interactions between the alkylators and olaparib. Tumor intervention studies combining nimustine and olaparib resulted in recurrence-free survival exceeding 330 days in 3 of 5 animals tested.</p><p class="abstract"><strong>Conclusions</strong>: We generated and validated a platform for identification of compounds with specific activity against BRCA2-deficient cells that translates well to the preclinical setting. Our data call for the re-evaluation of alkylators, especially melphalan and nimustine, alone or in combination with the poly-(ADP-ribose)-polymerase inhibitors, for the treatment of breast cancers with a defective BRCA pathway.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/D012910/1</span>; Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">Clinical cancer research : an official journal of the American Association for Cancer Research</span>&nbsp;<span class="year">2010</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">99-108</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20008842">20008842</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2802735">2802735</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1078-0432.CCR-09-2434">10.1158/1078-0432.CCR-09-2434</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genetics of obesity: FTO leads the way.</h4>
<p class="authors"><span class="author">Fawcett KA</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Metabolic Disease Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">In 2007, an association of single nucleotide polymorphisms (SNPs) in the fat mass and obesity-associated (FTO) gene region with body mass index (BMI) and risk of obesity was identified in multiple populations, making FTO the first locus unequivocally associated with adiposity. At the time, FTO was a gene of unknown function and it was not known whether these SNPs exerted their effect on adiposity by affecting FTO or neighboring genes. Therefore, this breakthrough association inspired a wealth of in silico, in vitro, and in vivo analyses in model organisms and humans to improve knowledge of FTO function. These studies suggested that FTO plays a role in controlling feeding behavior and energy expenditure. Here, we review the approaches taken that provide a blueprint for the study of other obesity-associated genes in the hope that this strategy will result in increased understanding of the biological mechanisms underlying body weight regulation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077016/Z/05/Z</span></p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">6</span>;<span class="pagerange">266-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20381893">20381893</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2906751">2906751</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2010.02.006">10.1016/j.tig.2010.02.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detailed investigation of the role of common and low-frequency WFS1 variants in type 2 diabetes risk.</h4>
<p class="authors"><span class="author">Fawcett KA</span>, <span class="author">Wheeler E</span>, <span class="author">Morris AP</span>, <span class="author">Ricketts SL</span>, <span class="author">Hallmans G</span>, <span class="author">Rolandsson O</span>, <span class="author">Daly A</span>, <span class="author">Wasson J</span>, <span class="author">Permutt A</span>, <span class="author">Hattersley AT</span>, <span class="author">Glaser B</span>, <span class="author">Franks PW</span>, <span class="author">McCarthy MI</span>, <span class="author">Wareham NJ</span>, <span class="author">Sandhu MS</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Metabolic Disease Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK.</p><p class="abstract"><strong>Objective</strong>: Wolfram syndrome 1 (WFS1) single nucleotide polymorphisms (SNPs) are associated with risk of type 2 diabetes. In this study we aimed to refine this association and investigate the role of low-frequency WFS1 variants in type 2 diabetes risk.</p><p class="abstract">For fine-mapping, we sequenced WFS1 exons, splice junctions, and conserved noncoding sequences in samples from 24 type 2 diabetic case and 68 control subjects, selected tagging SNPs, and genotyped these in 959 U.K. type 2 diabetic case and 1,386 control subjects. The same genomic regions were sequenced in samples from 1,235 type 2 diabetic case and 1,668 control subjects to compare the frequency of rarer variants between case and control subjects.</p><p class="abstract"><strong>Results</strong>: Of 31 tagging SNPs, the strongest associated was the previously untested 3&#39; untranslated region rs1046320 (P = 0.008); odds ratio 0.84 and P = 6.59 x 10(-7) on further replication in 3,753 case and 4,198 control subjects. High correlation between rs1046320 and the original strongest SNP (rs10010131) (r2 = 0.92) meant that we could not differentiate between their effects in our samples. There was no difference in the cumulative frequency of 82 rare (minor allele frequency [MAF] &lt;0.01) nonsynonymous variants between type 2 diabetic case and control subjects (P = 0.79). Two intermediate frequency (MAF 0.01-0.05) nonsynonymous changes also showed no statistical association with type 2 diabetes.</p><p class="abstract"><strong>Conclusions</strong>: We identified six highly correlated SNPs that show strong and comparable associations with risk of type 2 diabetes, but further refinement of these associations will require large sample sizes (&gt;100,000) or studies in ethnically diverse populations. Low frequency variants in WFS1 are unlikely to have a large impact on type 2 diabetes risk in white U.K. populations, highlighting the complexities of undertaking association studies with low-frequency variants identified by resequencing.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council: <span class="grant">MC_U106179471</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">081682</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2010</span>;<span class="volume">59</span>;<span class="number">3</span>;<span class="pagerange">741-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20028947">20028947</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2828659">2828659</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb09-0920">10.2337/db09-0920</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a hotspot for mimicry: assembly of a butterfly wing transcriptome to genomic sequence at the HmYb/Sb locus.</h4>
<p class="authors"><span class="author">Ferguson L</span>, <span class="author">Lee SF</span>, <span class="author">Chamberlain N</span>, <span class="author">Nadeau N</span>, <span class="author">Joron M</span>, <span class="author">Baxter S</span>, <span class="author">Wilkinson P</span>, <span class="author">Papanicolaou A</span>, <span class="author">Kumar S</span>, <span class="author">Kee TJ</span>, <span class="author">Clark R</span>, <span class="author">Davidson C</span>, <span class="author">Glithero R</span>, <span class="author">Beasley H</span>, <span class="author">Vogel H</span>, <span class="author">Ffrench-Constant R</span> and <span class="author">Jiggins C</span></p>
<p class="affiliation">Department of Zoology, University of Cambridge, UK.</p><p class="abstract">The mimetic wing patterns of Heliconius butterflies are an excellent example of both adaptive radiation and convergent evolution. Alleles at the HmYb and HmSb loci control the presence/absence of hindwing bar and hindwing margin phenotypes respectively between divergent races of Heliconius melpomene, and also between sister species. Here, we used fine-scale linkage mapping to identify and sequence a BAC tilepath across the HmYb/Sb loci. We also generated transcriptome sequence data for two wing pattern forms of H. melpomene that differed in HmYb/Sb alleles using 454 sequencing technology. Custom scripts were used to process the sequence traces and generate transcriptome assemblies. Genomic sequence for the HmYb/Sb candidate region was annotated both using the MAKER pipeline and manually using transcriptome sequence reads. In total, 28 genes were identified in the HmYb/Sb candidate region, six of which have alternative splice forms. None of these are orthologues of genes previously identified as being expressed in butterfly wing pattern development, implying previously undescribed molecular mechanisms of pattern determination on Heliconius wings. The use of next-generation sequencing has therefore facilitated DNA annotation of a poorly characterized genome, and generated hypotheses regarding the identity of wing pattern at the HmYb/Sb loci.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E008836/1</span>, <span class="grant">BB/E011845/1</span>, <span class="grant">BB/G00661X/1</span>; Medical Research Council: <span class="grant">G0900740</span></p>
<p class="publication"><span class="title">Molecular ecology</span>&nbsp;<span class="year">2010</span>;<span class="volume">19 Suppl 1</span>;<span class="pagerange">240-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20331783">20331783</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-294X.2009.04475.x">10.1111/j.1365-294X.2009.04475.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of mouse Dlg4 (PSD-95) gene deletion and human DLG4 gene variation with phenotypes relevant to autism spectrum disorders and Williams&#39; syndrome.</h4>
<p class="authors"><span class="author">Feyder M</span>, <span class="author">Karlsson RM</span>, <span class="author">Mathur P</span>, <span class="author">Lyman M</span>, <span class="author">Bock R</span>, <span class="author">Momenan R</span>, <span class="author">Munasinghe J</span>, <span class="author">Scattoni ML</span>, <span class="author">Ihne J</span>, <span class="author">Camp M</span>, <span class="author">Graybeal C</span>, <span class="author">Strathdee D</span>, <span class="author">Begg A</span>, <span class="author">Alvarez VA</span>, <span class="author">Kirsch P</span>, <span class="author">Rietschel M</span>, <span class="author">Cichon S</span>, <span class="author">Walter H</span>, <span class="author">Meyer-Lindenberg A</span>, <span class="author">Grant SG</span> and <span class="author">Holmes A</span></p>
<p class="affiliation">Section on Behavioral Science and Genetics, Laboratory for Integrative Neuroscience, National Institute on Alcoholism and Alcohol Abuse, Rockville, MD 20852-9411, USA. <a href="mailto:%6D%69%63%68%61%65%6C.%66%65%79%64%65%72%40%6B%69.%73%65">&#109;&#105;&#99;&#104;&#97;&#101;&#108;&#46;&#102;&#101;&#121;&#100;&#101;&#114;&#64;&#107;&#105;&#46;&#115;&#101;</a></p><p class="abstract"><strong>Objective</strong>: Research is increasingly linking autism spectrum disorders and other neurodevelopmental disorders to synaptic abnormalities (&quot;synaptopathies&quot;). PSD-95 (postsynaptic density-95, DLG4) orchestrates protein-protein interactions at excitatory synapses and is a major functional bridge interconnecting a neurexinneuroligin-SHANK pathway implicated in autism spectrum disorders.</p><p class="abstract"><strong>Method</strong>: The authors characterized behavioral, dendritic, and molecular phenotypic abnormalities relevant to autism spectrum disorders in mice with PSD-95 deletion (Dlg4&#x207B;(/)&#x207B;). The data from mice led to the identification of single-nucleotide polymorphisms (SNPs) in human DLG4 and the examination of associations between these variants and neural signatures of Williams&#39; syndrome in a normal population, using functional and structural neuroimaging.</p><p class="abstract"><strong>Results</strong>: Dlg4&#x207B;(/)&#x207B; showed increased repetitive behaviors, abnormal communication and social behaviors, impaired motor coordination, and increased stress reactivity and anxiety-related responses. Dlg4&#x207B;(/)&#x207B; had subtle dysmorphology of amygdala dendritic spines and altered forebrain expression of various synaptic genes, including Cyln2, which regulates cytoskeletal dynamics and is a candidate gene for Williams&#39; syndrome. A signifi-cant association was observed between variations in two human DLG4 SNPs and reduced intraparietal sulcus volume and abnormal cortico-amygdala coupling, both of which characterize Williams&#39; syndrome.</p><p class="abstract"><strong>Conclusions</strong>: These findings demonstrate that DLG4 gene disruption in mice produces a complex range of behavioral and molecular abnormalities relevant to autism spectrum disorders and Williams&#39; syndrome. The study provides an initial link between human DLG4 gene variation and key neural endophenotypes of Williams&#39; syndrome and perhaps corticoamygdala regulation of emotional and social processes more generally.</p>
<p class="grants">Funded by: NIAAA NIH HHS: <span class="grant">ZIA AA000411-07</span>, <span class="grant">ZIA AA000421-02</span>, <span class="grant">ZIA AA000421-03</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The American journal of psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">167</span>;<span class="number">12</span>;<span class="pagerange">1508-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20952458" rel="external">20952458</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3146008" rel="external">3146008</a>; DOI: <a href="http://dx.doi.org/10.1176%2Fappi.ajp.2010.10040484" rel="external">10.1176/appi.ajp.2010.10040484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The first special issue of Standards in Genomic Sciences from the Genomic Standards Consortium.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Kottmann R</span> and <span class="author">Sterk P</span></p>
<p class="publication"><span class="title">Standards in genomic sciences</span>&nbsp;<span class="year">2010</span>;<span class="volume">3</span>;<span class="number">3</span>;<span class="pagerange">214-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21304721" rel="external">21304721</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3035305" rel="external">3035305</a>; DOI: <a href="http://dx.doi.org/10.4056%2Fsigs.1493697" rel="external">10.4056/sigs.1493697</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Pfam protein families database.</h4>
<p class="authors"><span class="author">Finn RD</span>, <span class="author">Mistry J</span>, <span class="author">Tate J</span>, <span class="author">Coggill P</span>, <span class="author">Heger A</span>, <span class="author">Pollington JE</span>, <span class="author">Gavin OL</span>, <span class="author">Gunasekaran P</span>, <span class="author">Ceric G</span>, <span class="author">Forslund K</span>, <span class="author">Holm L</span>, <span class="author">Sonnhammer EL</span>, <span class="author">Eddy SR</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%72%64%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#100;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Pfam is a widely used database of protein families and domains. This article describes a set of major updates that we have implemented in the latest release (version 24.0). The most important change is that we now use HMMER3, the latest version of the popular profile hidden Markov model package. This software is approximately 100 times faster than HMMER2 and is more sensitive due to the routine use of the forward algorithm. The move to HMMER3 has necessitated numerous changes to Pfam that are described in detail. Pfam release 24.0 contains 11,912 families, of which a large number have been significantly updated during the past two years. Pfam is available via servers in the UK (http://pfam.sanger.ac.uk/), the USA (http://pfam.janelia.org/) and Sweden (http://pfam.sbc.su.se/).</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F010435/1</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">MC_U137761446</span>; Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D211-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19920124">19920124</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808889">2808889</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp985">10.1093/nar/gkp985</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Additional sex combs-like 1 belongs to the enhancer of trithorax and polycomb group and genetically interacts with Cbx2 in mice.</h4>
<p class="authors"><span class="author">Fisher CL</span>, <span class="author">Lee I</span>, <span class="author">Bloyer S</span>, <span class="author">Bozza S</span>, <span class="author">Chevalier J</span>, <span class="author">Dahl A</span>, <span class="author">Bodner C</span>, <span class="author">Helgason CD</span>, <span class="author">Hess JL</span>, <span class="author">Humphries RK</span> and <span class="author">Brock HW</span></p>
<p class="affiliation">Department of Zoology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, British Columbia V6T1Z3, Canada.</p><p class="abstract">The Additional sex combs (Asx) gene of Drosophila behaves genetically as an enhancer of trithorax and polycomb (ETP) in displaying bidirectional homeotic phenotypes, suggesting that is required for maintenance of both activation and silencing of Hox genes. There are three murine homologs of Asx called Additional sex combs-like1, 2, and 3. Asxl1 is required for normal adult hematopoiesis; however, its embryonic function is unknown. We used a targeted mouse mutant line Asxl1(tm1Bc) to determine if Asxl1 is required to silence and activate Hox genes in mice during axial patterning. The mutant embryos exhibit simultaneous anterior and posterior transformations of the axial skeleton, consistent with a role for Asxl1 in activation and silencing of Hox genes. Transformations of the axial skeleton are enhanced in compound mutant embryos for the polycomb group gene M33/Cbx2. Hoxa4, Hoxa7, and Hoxc8 are derepressed in Asxl1(tm1Bc) mutants in the antero-posterior axis, but Hoxc8 expression is reduced in the brain of mutants, consistent with Asxl1 being required both for activation and repression of Hox genes. We discuss the genetic and molecular definition of ETPs, and suggest that the function of Asxl1 depends on its cellular context.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; NCI NIH HHS: <span class="grant">R01 CA078815-03</span>, <span class="grant">R01 CA092251-11</span>, <span class="grant">R01-CA-0078815</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">337</span>;<span class="number">1</span>;<span class="pagerange">9-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19833123" rel="external">19833123</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2807749" rel="external">2807749</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2009.10.004" rel="external">10.1016/j.ydbio.2009.10.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl&#39;s 10th year.</h4>
<p class="authors"><span class="author">Flicek P</span>, <span class="author">Aken BL</span>, <span class="author">Ballester B</span>, <span class="author">Beal K</span>, <span class="author">Bragin E</span>, <span class="author">Brent S</span>, <span class="author">Chen Y</span>, <span class="author">Clapham P</span>, <span class="author">Coates G</span>, <span class="author">Fairley S</span>, <span class="author">Fitzgerald S</span>, <span class="author">Fernandez-Banet J</span>, <span class="author">Gordon L</span>, <span class="author">Gr&auml;f S</span>, <span class="author">Haider S</span>, <span class="author">Hammond M</span>, <span class="author">Howe K</span>, <span class="author">Jenkinson A</span>, <span class="author">Johnson N</span>, <span class="author">K&auml;h&auml;ri A</span>, <span class="author">Keefe D</span>, <span class="author">Keenan S</span>, <span class="author">Kinsella R</span>, <span class="author">Kokocinski F</span>, <span class="author">Koscielny G</span>, <span class="author">Kulesha E</span>, <span class="author">Lawson D</span>, <span class="author">Longden I</span>, <span class="author">Massingham T</span>, <span class="author">McLaren W</span>, <span class="author">Megy K</span>, <span class="author">Overduin B</span>, <span class="author">Pritchard B</span>, <span class="author">Rios D</span>, <span class="author">Ruffier M</span>, <span class="author">Schuster M</span>, <span class="author">Slater G</span>, <span class="author">Smedley D</span>, <span class="author">Spudich G</span>, <span class="author">Tang YA</span>, <span class="author">Trevanion S</span>, <span class="author">Vilella A</span>, <span class="author">Vogel J</span>, <span class="author">White S</span>, <span class="author">Wilder SP</span>, <span class="author">Zadissa A</span>, <span class="author">Birney E</span>, <span class="author">Cunningham F</span>, <span class="author">Dunham I</span>, <span class="author">Durbin R</span>, <span class="author">Fern&aacute;ndez-Suarez XM</span>, <span class="author">Herrero J</span>, <span class="author">Hubbard TJ</span>, <span class="author">Parker A</span>, <span class="author">Proctor G</span>, <span class="author">Smith J</span> and <span class="author">Searle SM</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%66%6C%69%63%65%6B%40%65%62%69.%61%63.%75%6B">&#102;&#108;&#105;&#99;&#101;&#107;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ensembl (http://www.ensembl.org) integrates genomic information for a comprehensive set of chordate genomes with a particular focus on resources for human, mouse, rat, zebrafish and other high-value sequenced genomes. We provide complete gene annotations for all supported species in addition to specific resources that target genome variation, function and evolution. Ensembl data is accessible in a variety of formats including via our genome browser, API and BioMart. This year marks the tenth anniversary of Ensembl and in that time the project has grown with advances in genome technology. As of release 56 (September 2009), Ensembl supports 51 species including marmoset, pig, zebra finch, lizard, gorilla and wallaby, which were added in the past year. Major additions and improvements to Ensembl since our previous report include the incorporation of the human GRCh37 assembly, enhanced visualisation and data-mining options for the Ensembl regulatory features and continued development of our software infrastructure.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBE0116401</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D557-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19906699">19906699</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808936">2808936</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp972">10.1093/nar/gkp972</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Special issue: The Human Intestinal Microbiota.</h4>
<p class="authors"><span class="author">Flint HJ</span>, <span class="author">O&#39;Toole PW</span> and <span class="author">Walker AW</span></p>
<p class="affiliation">Microbial Ecology Group, Rowett Institute of Nutrition and Health, University of Aberdeen, Bucksburn, Aberdeen AB21 9SB, UK. <a href="mailto:%68.%66%6C%69%6E%74%40%72%6F%77%65%74%74.%61%63.%75%6B">&#104;&#46;&#102;&#108;&#105;&#110;&#116;&#64;&#114;&#111;&#119;&#101;&#116;&#116;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">156</span>;<span class="number">Pt 11</span>;<span class="pagerange">3203-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21045216" rel="external">21045216</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.045443-0" rel="external">10.1099/mic.0.045443-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CINister thoughts.</h4>
<p class="authors"><span class="author">Foijer F</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%66%6C%6F%72%69%73.%66%6F%69%6A%65%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#108;&#111;&#114;&#105;&#115;&#46;&#102;&#111;&#105;&#106;&#101;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chromosome instability (CIN) is the process that leads to aneuploidy, a known hallmark of human tumours for over a century. Nowadays, it is believed that CIN promotes tumorigenesis by shuffling the genome into a malignant order through translocations, amplifications, deletions (structural CIN), and gains and losses of whole chromosomes (numerical CIN or nCIN). The present review focuses on the causes and consequences of nCIN. Several roads can lead to nCIN, including a compromised spindle assembly checkpoint, cohesion defects, p53 deficiency and flawed microtubule-kinetochore attachments. Whereas the link between nCIN and tumorigenesis is becoming more evident, indications have emerged recently that nCIN can suppress tumour formation as well. To understand these paradoxical findings, novel reagents and more sophisticated mouse models are needed. This will provide us with a better understanding of nCIN and eventually with therapies that exploit this characteristic of human tumours.</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">6</span>;<span class="pagerange">1715-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21118153" rel="external">21118153</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0381715" rel="external">10.1042/BST0381715</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Major depression and the metabolic syndrome.</h4>
<p class="authors"><span class="author">Foley DL</span>, <span class="author">Morley KI</span>, <span class="author">Madden PA</span>, <span class="author">Heath AC</span>, <span class="author">Whitfield JB</span> and <span class="author">Martin NG</span></p>
<p class="affiliation">Biostatistics Unit, Orygen Youth Health Research Centre &amp; Centre for Youth Mental Health, The University of Melbourne, Australia. <a href="mailto:%64%66%6F%6C%65%79%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#100;&#102;&#111;&#108;&#101;&#121;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">The aim of this study is to characterize the relationship between major depression and the metabolic syndrome in a large community based sample of Australian men and women aged 26-90 years. A lifetime history of major depression was assessed by telephone interview following the DSM-III-R. A current history of metabolic syndrome was assessed following the United States National Cholesterol Education Program Adult Treatment Panel III (NCEP AP-III) guidelines 1 to 3 years later. Logistic regression was used to estimate the association between depression and the metabolic syndrome, and its component criteria, controlling for age, sex and alcohol dependence. There was no association between a lifetime history of major depression and the presence of the metabolic syndrome. There was a weak association between depression and low high-density lipoprotein cholesterol but not with other component criteria of the metabolic syndrome. Despite calls for interventions directed at depression to reduce the onset of the metabolic syndrome there are important failures to replicate in large samples such as this, no consensus regarding the threshold at which depression may pose a significant risk even allowing for heterogeneity across populations, and no consensus regarding confounders that may explain inter-study differences. The absence of any dosage effect of depression on the associated risk for the metabolic syndrome in other unselected samples does not support a direct causal relationship. The call for intervention studies on the basis of the currently published evidence base is unwarranted.</p>
<p class="grants">Funded by: Medical Research Council; NIAAA NIH HHS: <span class="grant">AA014041</span>, <span class="grant">AA07535</span>, <span class="grant">AA13320</span>, <span class="grant">AA13326</span>, <span class="grant">K05 AA017688-04</span>, <span class="grant">R01 AA007535-08</span>, <span class="grant">R01 AA013320-05</span>, <span class="grant">R01 AA013326-05</span>, <span class="grant">R01 AA014041-05</span>; NIDA NIH HHS: <span class="grant">R01 DA012854-08</span></p>
<p class="publication"><span class="title">Twin research and human genetics : the official journal of the International Society for Twin Studies</span>&nbsp;<span class="year">2010</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">347-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20707705" rel="external">20707705</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3150840" rel="external">3150840</a>; DOI: <a href="http://dx.doi.org/10.1375%2Ftwin.13.4.347" rel="external">10.1375/twin.13.4.347</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluating the discriminative power of multi-trait genetic risk scores for type 2 diabetes in a northern Swedish population.</h4>
<p class="authors"><span class="author">Fontaine-Bisson B</span>, <span class="author">Renstr&ouml;m F</span>, <span class="author">Rolandsson O</span>, <span class="author">MAGIC</span>, <span class="author">Payne F</span>, <span class="author">Hallmans G</span>, <span class="author">Barroso I</span> and <span class="author">Franks PW</span></p>
<p class="affiliation">Department of Nutrition Sciences, University of Ottawa, Ottawa, ON, Canada.</p><p class="abstract">We determined whether single nucleotide polymorphisms (SNPs) previously associated with diabetogenic traits improve the discriminative power of a type 2 diabetes genetic risk score.</p><p class="abstract"><strong>Methods</strong>: Participants (n = 2,751) were genotyped for 73 SNPs previously associated with type 2 diabetes, fasting glucose/insulin concentrations, obesity or lipid levels, from which five genetic risk scores (one for each of the four traits and one combining all SNPs) were computed. Type 2 diabetes patients and non-diabetic controls (n = 1,327/1,424) were identified using medical records in addition to an independent oral glucose tolerance test.</p><p class="abstract"><strong>Results</strong>: Model 1, including only SNPs associated with type 2 diabetes, had a discriminative power of 0.591 (p &lt; 1.00 x 10(-20) vs null model) as estimated by the area under the receiver operator characteristic curve (ROC AUC). Model 2, including only fasting glucose/insulin SNPs, had a significantly higher discriminative power than the null model (ROC AUC 0.543; p = 9.38 x 10(-6) vs null model), but lower discriminative power than model 1 (p = 5.92 x 10(-5)). Model 3, with only lipid-associated SNPs, had significantly higher discriminative power than the null model (ROC AUC 0.565; p = 1.44 x 10(-9)) and was not statistically different from model 1 (p = 0.083). The ROC AUC of model 4, which included only obesity SNPs, was 0.557 (p = 2.30 x 10(-7) vs null model) and smaller than model 1 (p = 0.025). Finally, the model including all SNPs yielded a significant improvement in discriminative power compared with the null model (p &lt; 1.0 x 10(-20)) and model 1 (p = 1.32 x 10(-5)); its ROC AUC was 0.626.</p><p class="abstract">Adding SNPs previously associated with fasting glucose, insulin, lipids or obesity to a genetic risk score for type 2 diabetes significantly increases the power to discriminate between people with and without clinically manifest type 2 diabetes compared with a model including only conventional type 2 diabetes loci.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077016/Z/05/Z</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2010</span>;<span class="volume">53</span>;<span class="number">10</span>;<span class="pagerange">2155-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20571754">20571754</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2931645">2931645</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-010-1792-y">10.1007/s00125-010-1792-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">COSMIC (the Catalogue of Somatic Mutations in Cancer): a resource to investigate acquired mutations in human cancer.</h4>
<p class="authors"><span class="author">Forbes SA</span>, <span class="author">Tang G</span>, <span class="author">Bindal N</span>, <span class="author">Bamford S</span>, <span class="author">Dawson E</span>, <span class="author">Cole C</span>, <span class="author">Kok CY</span>, <span class="author">Jia M</span>, <span class="author">Ewing R</span>, <span class="author">Menzies A</span>, <span class="author">Teague JW</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The catalogue of Somatic Mutations in Cancer (COSMIC) (http://www.sanger.ac.uk/cosmic/) is the largest public resource for information on somatically acquired mutations in human cancer and is available freely without restrictions. Currently (v43, August 2009), COSMIC contains details of 1.5-million experiments performed through 13,423 genes in almost 370,000 tumours, describing over 90,000 individual mutations. Data are gathered from two sources, publications in the scientific literature, (v43 contains 7797 curated articles) and the full output of the genome-wide screens from the Cancer Genome Project (CGP) at the Sanger Institute, UK. Most of the world&#39;s literature on point mutations in human cancer has now been curated into COSMIC and while this is continually updated, a greater emphasis on curating fusion gene mutations is driving the expansion of this information; over 2700 fusion gene mutations are now described. Whole-genome sequencing screens are now identifying large numbers of genomic rearrangements in cancer and COSMIC is now displaying details of these analyses also. Examination of COSMIC&#39;s data is primarily web-driven, focused on providing mutation range and frequency statistics based upon a choice of gene and/or cancer phenotype. Graphical views provide easily interpretable summaries of large quantities of data, and export functions can provide precise details of user-selected data.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D652-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19906727">19906727</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808858">2808858</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp995">10.1093/nar/gkp995</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quality, quantity and harmony: the DataSHaPER approach to integrating data across bioclinical studies.</h4>
<p class="authors"><span class="author">Fortier I</span>, <span class="author">Burton PR</span>, <span class="author">Robson PJ</span>, <span class="author">Ferretti V</span>, <span class="author">Little J</span>, <span class="author">L&#39;Heureux F</span>, <span class="author">Desch&ecirc;nes M</span>, <span class="author">Knoppers BM</span>, <span class="author">Doiron D</span>, <span class="author">Keers JC</span>, <span class="author">Linksted P</span>, <span class="author">Harris JR</span>, <span class="author">Lachance G</span>, <span class="author">Boileau C</span>, <span class="author">Pedersen NL</span>, <span class="author">Hamilton CM</span>, <span class="author">Hveem K</span>, <span class="author">Borugian MJ</span>, <span class="author">Gallagher RP</span>, <span class="author">McLaughlin J</span>, <span class="author">Parker L</span>, <span class="author">Potter JD</span>, <span class="author">Gallacher J</span>, <span class="author">Kaaks R</span>, <span class="author">Liu B</span>, <span class="author">Sprosen T</span>, <span class="author">Vilain A</span>, <span class="author">Atkinson SA</span>, <span class="author">Rengifo A</span>, <span class="author">Morton R</span>, <span class="author">Metspalu A</span>, <span class="author">Wichmann HE</span>, <span class="author">Tremblay M</span>, <span class="author">Chisholm RL</span>, <span class="author">Garcia-Montero A</span>, <span class="author">Hillege H</span>, <span class="author">Litton JE</span>, <span class="author">Palmer LJ</span>, <span class="author">Perola M</span>, <span class="author">Wolffenbuttel BH</span>, <span class="author">Peltonen L</span> and <span class="author">Hudson TJ</span></p>
<p class="affiliation">Public Population Project in Genomics (P&sup3;G), Montreal, QC, Canada. <a href="mailto:%69%66%6F%72%74%69%65%72%40%70%33%67.%6F%72%67">&#105;&#102;&#111;&#114;&#116;&#105;&#101;&#114;&#64;&#112;&#51;&#103;&#46;&#111;&#114;&#103;</a></p><p class="abstract"><strong>Background</strong>: Vast sample sizes are often essential in the quest to disentangle the complex interplay of the genetic, lifestyle, environmental and social factors that determine the aetiology and progression of chronic diseases. The pooling of information between studies is therefore of central importance to contemporary bioscience. However, there are many technical, ethico-legal and scientific challenges to be overcome if an effective, valid, pooled analysis is to be achieved. Perhaps most critically, any data that are to be analysed in this way must be adequately &#39;harmonized&#39;. This implies that the collection and recording of information and data must be done in a manner that is sufficiently similar in the different studies to allow valid synthesis to take place.</p><p class="abstract"><strong>Methods</strong>: This conceptual article describes the origins, purpose and scientific foundations of the DataSHaPER (DataSchema and Harmonization Platform for Epidemiological Research; http://www.datashaper.org), which has been created by a multidisciplinary consortium of experts that was pulled together and coordinated by three international organizations: P&sup3;G (Public Population Project in Genomics), PHOEBE (Promoting Harmonization of Epidemiological Biobanks in Europe) and CPT (Canadian Partnership for Tomorrow Project).</p><p class="abstract"><strong>Results</strong>: The DataSHaPER provides a flexible, structured approach to the harmonization and pooling of information between studies. Its two primary components, the &#39;DataSchema&#39; and &#39;Harmonization Platforms&#39;, together support the preparation of effective data-collection protocols and provide a central reference to facilitate harmonization. The DataSHaPER supports both &#39;prospective&#39; and &#39;retrospective&#39; harmonization.</p><p class="abstract"><strong>Conclusion</strong>: It is hoped that this article will encourage readers to investigate the project further: the more the research groups and studies are actively involved, the more effective the DataSHaPER programme will ultimately be.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">086160/Z/08/A</span></p>
<p class="publication"><span class="title">International journal of epidemiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">39</span>;<span class="number">5</span>;<span class="pagerange">1383-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20813861" rel="external">20813861</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2972444" rel="external">2972444</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fije%2Fdyq139" rel="external">10.1093/ije/dyq139</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lysine-specific demethylase 1 regulates the embryonic transcriptome and CoREST stability.</h4>
<p class="authors"><span class="author">Foster CT</span>, <span class="author">Dovey OM</span>, <span class="author">Lezina L</span>, <span class="author">Luo JL</span>, <span class="author">Gant TW</span>, <span class="author">Barlev N</span>, <span class="author">Bradley A</span> and <span class="author">Cowley SM</span></p>
<p class="affiliation">Department of Biochemistry, University of Leicester, Leicester, United Kingdom.</p><p class="abstract">Lysine-specific demethylase 1 (LSD1), which demethylates mono- and dimethylated histone H3-Lys4 as part of a complex including CoREST and histone deacetylases (HDACs), is essential for embryonic development in the mouse beyond embryonic day 6.5 (e6.5). To determine the role of LSD1 during this early period of embryogenesis, we have generated loss-of-function gene trap mice and conditional knockout embryonic stem (ES) cells. Analysis of postimplantation gene trap embryos revealed that LSD1 expression, and therefore function, is restricted to the epiblast. Conditional deletion of LSD1 in mouse ES cells, the in vitro counterpart of the epiblast, revealed a reduction in CoREST protein and associated HDAC activity, resulting in a global increase in histone H3-Lys56 acetylation, but not H3-Lys4 methylation. Despite this biochemical perturbation, ES cells with LSD1 deleted proliferate normally and retain stem cell characteristics. Loss of LSD1 causes the aberrant expression of 588 genes, including those coding for transcription factors with roles in anterior/posterior patterning and limb development, such as brachyury, Hoxb7, Hoxd8, and retinoic acid receptor &gamma; (RAR&gamma;). The gene coding for brachyury, a key regulator of mesodermal differentiation, is a direct target gene of LSD1 and is overexpressed in e6.5 Lsd1 gene trap embryos. Thus, LSD1 regulates the expression and appropriate timing of key developmental regulators, as part of the LSD1/CoREST/HDAC complex, during early embryonic development.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600135</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">20</span>;<span class="pagerange">4851-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20713442" rel="external">20713442</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2950538" rel="external">2950538</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.00521-10" rel="external">10.1128/MCB.00521-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide meta-analysis increases to 71 the number of confirmed Crohn&#39;s disease susceptibility loci.</h4>
<p class="authors"><span class="author">Franke A</span>, <span class="author">McGovern DP</span>, <span class="author">Barrett JC</span>, <span class="author">Wang K</span>, <span class="author">Radford-Smith GL</span>, <span class="author">Ahmad T</span>, <span class="author">Lees CW</span>, <span class="author">Balschun T</span>, <span class="author">Lee J</span>, <span class="author">Roberts R</span>, <span class="author">Anderson CA</span>, <span class="author">Bis JC</span>, <span class="author">Bumpstead S</span>, <span class="author">Ellinghaus D</span>, <span class="author">Festen EM</span>, <span class="author">Georges M</span>, <span class="author">Green T</span>, <span class="author">Haritunians T</span>, <span class="author">Jostins L</span>, <span class="author">Latiano A</span>, <span class="author">Mathew CG</span>, <span class="author">Montgomery GW</span>, <span class="author">Prescott NJ</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Rotter JI</span>, <span class="author">Schumm P</span>, <span class="author">Sharma Y</span>, <span class="author">Simms LA</span>, <span class="author">Taylor KD</span>, <span class="author">Whiteman D</span>, <span class="author">Wijmenga C</span>, <span class="author">Baldassano RN</span>, <span class="author">Barclay M</span>, <span class="author">Bayless TM</span>, <span class="author">Brand S</span>, <span class="author">B&uuml;ning C</span>, <span class="author">Cohen A</span>, <span class="author">Colombel JF</span>, <span class="author">Cottone M</span>, <span class="author">Stronati L</span>, <span class="author">Denson T</span>, <span class="author">De Vos M</span>, <span class="author">D&#39;Inca R</span>, <span class="author">Dubinsky M</span>, <span class="author">Edwards C</span>, <span class="author">Florin T</span>, <span class="author">Franchimont D</span>, <span class="author">Gearry R</span>, <span class="author">Glas J</span>, <span class="author">Van Gossum A</span>, <span class="author">Guthery SL</span>, <span class="author">Halfvarson J</span>, <span class="author">Verspaget HW</span>, <span class="author">Hugot JP</span>, <span class="author">Karban A</span>, <span class="author">Laukens D</span>, <span class="author">Lawrance I</span>, <span class="author">Lemann M</span>, <span class="author">Levine A</span>, <span class="author">Libioulle C</span>, <span class="author">Louis E</span>, <span class="author">Mowat C</span>, <span class="author">Newman W</span>, <span class="author">Pan&eacute;s J</span>, <span class="author">Phillips A</span>, <span class="author">Proctor DD</span>, <span class="author">Regueiro M</span>, <span class="author">Russell R</span>, <span class="author">Rutgeerts P</span>, <span class="author">Sanderson J</span>, <span class="author">Sans M</span>, <span class="author">Seibold F</span>, <span class="author">Steinhart AH</span>, <span class="author">Stokkers PC</span>, <span class="author">Torkvist L</span>, <span class="author">Kullak-Ublick G</span>, <span class="author">Wilson D</span>, <span class="author">Walters T</span>, <span class="author">Targan SR</span>, <span class="author">Brant SR</span>, <span class="author">Rioux JD</span>, <span class="author">D&#39;Amato M</span>, <span class="author">Weersma RK</span>, <span class="author">Kugathasan S</span>, <span class="author">Griffiths AM</span>, <span class="author">Mansfield JC</span>, <span class="author">Vermeire S</span>, <span class="author">Duerr RH</span>, <span class="author">Silverberg MS</span>, <span class="author">Satsangi J</span>, <span class="author">Schreiber S</span>, <span class="author">Cho JH</span>, <span class="author">Annese V</span>, <span class="author">Hakonarson H</span>, <span class="author">Daly MJ</span> and <span class="author">Parkes M</span></p>
<p class="affiliation">Institute of Clinical Molecular Biology, Christian-Albrechts-University Kiel, Kiel, Germany.</p><p class="abstract">We undertook a meta-analysis of six Crohn&#39;s disease genome-wide association studies (GWAS) comprising 6,333 affected individuals (cases) and 15,056 controls and followed up the top association signals in 15,694 cases, 14,026 controls and 414 parent-offspring trios. We identified 30 new susceptibility loci meeting genome-wide significance (P &lt; 5 &times; 10&#x207B;&#x2078;). A series of in silico analyses highlighted particular genes within these loci and, together with manual curation, implicated functionally interesting candidate genes including SMAD3, ERAP2, IL10, IL2RA, TYK2, FUT2, DNMT3A, DENND1B, BACH2 and TAGAP. Combined with previously confirmed loci, these results identify 71 distinct loci with genome-wide significant evidence for association with Crohn&#39;s disease.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0600329</span>, <span class="grant">G0800675</span>, <span class="grant">G0800759</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>; NHLBI NIH HHS: <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">R01 HL087652</span>, <span class="grant">U01 HL080295</span>; NIAMS NIH HHS: <span class="grant">K08 AR055688</span>, <span class="grant">K08 AR055688-01A1S1</span>, <span class="grant">K08 AR055688-03</span>, <span class="grant">K08 AR055688-04</span>; NIDDK NIH HHS: <span class="grant">DK 063491</span>, <span class="grant">DK062413</span>, <span class="grant">DK062420</span>, <span class="grant">DK062422</span>, <span class="grant">DK062423</span>, <span class="grant">DK062429</span>, <span class="grant">DK062431</span>, <span class="grant">DK062432</span>, <span class="grant">DK064869</span>, <span class="grant">DK069513</span>, <span class="grant">DK084554</span>, <span class="grant">DK76984</span>, <span class="grant">P01-DK046763</span>, <span class="grant">P30 DK043351</span>, <span class="grant">R01 DK064869</span>; Wellcome Trust: <span class="grant">089120</span>, <span class="grant">WT089120/Z/09/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">12</span>;<span class="pagerange">1118-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21102463">21102463</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3299551">3299551</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.717">10.1038/ng.717</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nonobese diabetic congenic strain analysis of autoimmune diabetes reveals genetic complexity of the Idd18 locus and identifies Vav3 as a candidate gene.</h4>
<p class="authors"><span class="author">Fraser HI</span>, <span class="author">Dendrou CA</span>, <span class="author">Healy B</span>, <span class="author">Rainbow DB</span>, <span class="author">Howlett S</span>, <span class="author">Smink LJ</span>, <span class="author">Gregory S</span>, <span class="author">Steward CA</span>, <span class="author">Todd JA</span>, <span class="author">Peterson LB</span> and <span class="author">Wicker LS</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Cambridge.</p><p class="abstract">We have used the public sequencing and annotation of the mouse genome to delimit the previously resolved type 1 diabetes (T1D) insulin-dependent diabetes (Idd)18 interval to a region on chromosome 3 that includes the immunologically relevant candidate gene, Vav3. To test the candidacy of Vav3, we developed a novel congenic strain that enabled the resolution of Idd18 to a 604-kb interval, designated Idd18.1, which contains only two annotated genes: the complete sequence of Vav3 and the last exon of the gene encoding NETRIN G1, Ntng1. Targeted sequencing of Idd18.1 in the NOD mouse strain revealed that allelic variation between NOD and C57BL/6J (B6) occurs in noncoding regions with 138 single nucleotide polymorphisms concentrated in the introns between exons 20 and 27 and immediately after the 3&#39; untranslated region. We observed differential expression of VAV3 RNA transcripts in thymocytes when comparing congenic mouse strains with B6 or NOD alleles at Idd18.1. The T1D protection associated with B6 alleles of Idd18.1/Vav3 requires the presence of B6 protective alleles at Idd3, which are correlated with increased IL-2 production and regulatory T cell function. In the absence of B6 protective alleles at Idd3, we detected a second T1D protective B6 locus, Idd18.3, which is closely linked to, but distinct from, Idd18.1. Therefore, genetic mapping, sequencing, and gene expression evidence indicate that alteration of VAV3 expression is an etiological factor in the development of autoimmune beta-cell destruction in NOD mice. This study also demonstrates that a congenic strain mapping approach can isolate closely linked susceptibility genes.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 15416</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">061859</span>, <span class="grant">079895</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2010</span>;<span class="volume">184</span>;<span class="number">9</span>;<span class="pagerange">5075-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20363978">20363978</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2886967">2886967</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.0903734">10.4049/jimmunol.0903734</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variants in ADCY5 and near CCNL1 are associated with fetal growth and birth weight.</h4>
<p class="authors"><span class="author">Freathy RM</span>, <span class="author">Mook-Kanamori DO</span>, <span class="author">Sovio U</span>, <span class="author">Prokopenko I</span>, <span class="author">Timpson NJ</span>, <span class="author">Berry DJ</span>, <span class="author">Warrington NM</span>, <span class="author">Widen E</span>, <span class="author">Hottenga JJ</span>, <span class="author">Kaakinen M</span>, <span class="author">Lange LA</span>, <span class="author">Bradfield JP</span>, <span class="author">Kerkhof M</span>, <span class="author">Marsh JA</span>, <span class="author">M&auml;gi R</span>, <span class="author">Chen CM</span>, <span class="author">Lyon HN</span>, <span class="author">Kirin M</span>, <span class="author">Adair LS</span>, <span class="author">Aulchenko YS</span>, <span class="author">Bennett AJ</span>, <span class="author">Borja JB</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Charoen P</span>, <span class="author">Coin LJ</span>, <span class="author">Cousminer DL</span>, <span class="author">de Geus EJ</span>, <span class="author">Deloukas P</span>, <span class="author">Elliott P</span>, <span class="author">Evans DM</span>, <span class="author">Froguel P</span>, <span class="author">Genetic Investigation of ANthropometric Traits (GIANT) Consortium</span>, <span class="author">Glaser B</span>, <span class="author">Groves CJ</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hassanali N</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Hofman A</span>, <span class="author">Holly JM</span>, <span class="author">Hypp&ouml;nen E</span>, <span class="author">Kanoni S</span>, <span class="author">Knight BA</span>, <span class="author">Laitinen J</span>, <span class="author">Lindgren CM</span>, <span class="author">Meta-Analyses of Glucose and Insulin-related traits Consortium</span>, <span class="author">McArdle WL</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Pennell CE</span>, <span class="author">Postma DS</span>, <span class="author">Pouta A</span>, <span class="author">Ramasamy A</span>, <span class="author">Rayner NW</span>, <span class="author">Ring SM</span>, <span class="author">Rivadeneira F</span>, <span class="author">Shields BM</span>, <span class="author">Strachan DP</span>, <span class="author">Surakka I</span>, <span class="author">Taanila A</span>, <span class="author">Tiesler C</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Wijga AH</span>, <span class="author">Willemsen G</span>, <span class="author">Zhang H</span>, <span class="author">Zhao J</span>, <span class="author">Wilson JF</span>, <span class="author">Steegers EA</span>, <span class="author">Hattersley AT</span>, <span class="author">Eriksson JG</span>, <span class="author">Peltonen L</span>, <span class="author">Mohlke KL</span>, <span class="author">Grant SF</span>, <span class="author">Hakonarson H</span>, <span class="author">Koppelman GH</span>, <span class="author">Dedoussis GV</span>, <span class="author">Heinrich J</span>, <span class="author">Gillman MW</span>, <span class="author">Palmer LJ</span>, <span class="author">Frayling TM</span>, <span class="author">Boomsma DI</span>, <span class="author">Davey Smith G</span>, <span class="author">Power C</span>, <span class="author">Jaddoe VW</span>, <span class="author">Jarvelin MR</span>, <span class="author">Early Growth Genetics (EGG) Consortium</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">Genetics of Complex Traits, Peninsula College of Medicine and Dentistry, University of Exeter, Exeter, UK.</p><p class="abstract">To identify genetic variants associated with birth weight, we meta-analyzed six genome-wide association (GWA) studies (n = 10,623 Europeans from pregnancy/birth cohorts) and followed up two lead signals in 13 replication studies (n = 27,591). rs900400 near LEKR1 and CCNL1 (P = 2 x 10(-35)) and rs9883204 in ADCY5 (P = 7 x 10(-15)) were robustly associated with birth weight. Correlated SNPs in ADCY5 were recently implicated in regulation of glucose levels and susceptibility to type 2 diabetes, providing evidence that the well-described association between lower birth weight and subsequent type 2 diabetes has a genetic component, distinct from the proposed role of programming by maternal nutrition. Using data from both SNPs, we found that the 9% of Europeans carrying four birth weight-lowering alleles were, on average, 113 g (95% CI 89-137 g) lighter at birth than the 24% with zero or one alleles (P(trend) = 7 x 10(-30)). The impact on birth weight is similar to that of a mother smoking 4-5 cigarettes per day in the third trimester of pregnancy.</p>
<p class="grants">Funded by: British Heart Foundation; Canadian Institutes of Health Research: <span class="grant">MOP 82893</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Department of Health; Diabetes UK: <span class="grant">08/0003692</span>; FIC NIH HHS: <span class="grant">TW05596</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0500070</span>, <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G0601653</span>, <span class="grant">G0800582</span>, <span class="grant">G0801056</span>, <span class="grant">G9815508</span>; NCRR NIH HHS: <span class="grant">RR20649</span>; NHLBI NIH HHS: <span class="grant">HL068041</span>, <span class="grant">HL085144</span>, <span class="grant">HL0876792</span>; NICHD NIH HHS: <span class="grant">HD034568</span>, <span class="grant">HD05450</span>, <span class="grant">HD056465</span>, <span class="grant">R24 HD050924</span>; NIDDK NIH HHS: <span class="grant">1R01DK075787</span>, <span class="grant">DK075787</span>, <span class="grant">DK078150</span>, <span class="grant">DK56350</span>; NIEHS NIH HHS: <span class="grant">ES10126</span>; NIMH NIH HHS: <span class="grant">MH083268</span>, <span class="grant">MH63706</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">085301</span>, <span class="grant">085541</span>, <span class="grant">89061/Z/09/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">430-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20372150">20372150</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2862164">2862164</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.567">10.1038/ng.567</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mouse welfare terms</h4>
<p class="authors"><span class="author">Gardiner M</span>, <span class="author">Wells S</span>, <span class="author">Trower C</span>, <span class="author">SALISBURY J</span>, <span class="author">Mallon AM</span>, <span class="author">Beck T</span>, <span class="author">MELVIN D</span>, <span class="author">Bussell J</span></p>
<p class="publication"><span class="title">Animal Technology and Welfare.</span> <span class="year">2010</span>;<span class="volume">9</span>;<span class="pagerange">175</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SnoPatrol: how many snoRNA genes are there?</h4>
<p class="authors"><span class="author">Gardner PP</span>, <span class="author">Bateman A</span> and <span class="author">Poole AM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB101SA, UK. <a href="mailto:%70%67%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#103;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Small nucleolar RNAs (snoRNAs) are among the most evolutionarily ancient classes of small RNA. Two experimental screens published in BMC Genomics expand the eukaryotic snoRNA catalog, but many more snoRNAs remain to be found.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077044</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Journal of biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20122292" rel="external">20122292</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871523" rel="external">2871523</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fjbiol211" rel="external">10.1186/jbiol211</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Gene Ontology in 2010: extensions and refinements.</h4>
<p class="authors"><span class="author">Gene Ontology Consortium</span></p><p class="abstract">The Gene Ontology (GO) Consortium (http://www.geneontology.org) (GOC) continues to develop, maintain and use a set of structured, controlled vocabularies for the annotation of genes, gene products and sequences. The GO ontologies are expanding both in content and in structure. Several new relationship types have been introduced and used, along with existing relationships, to create links between and within the GO domains. These improve the representation of biology, facilitate querying, and allow GO developers to systematically check for and correct inconsistencies within the GO. Gene product annotation using GO continues to increase both in the number of total annotations and in species coverage. GO tools, such as OBO-Edit, an ontology-editing tool, and AmiGO, the GOC ontology browser, have seen major improvements in functionality, speed and ease of use.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">SP/07/007/23671</span>; Medical Research Council: <span class="grant">G0500293</span>; NHGRI NIH HHS: <span class="grant">#P41HG02273</span>, <span class="grant">HG000330</span>, <span class="grant">HG002659</span>, <span class="grant">HG003751</span>, <span class="grant">HG004341</span>, <span class="grant">HG01315</span>, <span class="grant">HG02223</span>, <span class="grant">P41 HG02273</span>; NHLBI NIH HHS: <span class="grant">HL64541</span>; NIGMS NIH HHS: <span class="grant">U24GM077905</span>, <span class="grant">U24GM088849</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D331-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19920128" rel="external">19920128</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808930" rel="external">2808930</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp1018" rel="external">10.1093/nar/gkp1018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1.</h4>
<p class="authors"><span class="author">Genetic Analysis of Psoriasis Consortium &amp;amp; the Wellcome Trust Case Control Consortium 2</span>, <span class="author">Strange A</span>, <span class="author">Capon F</span>, <span class="author">Spencer CC</span>, <span class="author">Knight J</span>, <span class="author">Weale ME</span>, <span class="author">Allen MH</span>, <span class="author">Barton A</span>, <span class="author">Band G</span>, <span class="author">Bellenguez C</span>, <span class="author">Bergboer JG</span>, <span class="author">Blackwell JM</span>, <span class="author">Bramon E</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Casas JP</span>, <span class="author">Cork MJ</span>, <span class="author">Corvin A</span>, <span class="author">Deloukas P</span>, <span class="author">Dilthey A</span>, <span class="author">Duncanson A</span>, <span class="author">Edkins S</span>, <span class="author">Estivill X</span>, <span class="author">Fitzgerald O</span>, <span class="author">Freeman C</span>, <span class="author">Giardina E</span>, <span class="author">Gray E</span>, <span class="author">Hofer A</span>, <span class="author">H&uuml;ffmeier U</span>, <span class="author">Hunt SE</span>, <span class="author">Irvine AD</span>, <span class="author">Jankowski J</span>, <span class="author">Kirby B</span>, <span class="author">Langford C</span>, <span class="author">Lascorz J</span>, <span class="author">Leman J</span>, <span class="author">Leslie S</span>, <span class="author">Mallbris L</span>, <span class="author">Markus HS</span>, <span class="author">Mathew CG</span>, <span class="author">McLean WH</span>, <span class="author">McManus R</span>, <span class="author">M&ouml;ssner R</span>, <span class="author">Moutsianas L</span>, <span class="author">Naluai AT</span>, <span class="author">Nestle FO</span>, <span class="author">Novelli G</span>, <span class="author">Onoufriadis A</span>, <span class="author">Palmer CN</span>, <span class="author">Perricone C</span>, <span class="author">Pirinen M</span>, <span class="author">Plomin R</span>, <span class="author">Potter SC</span>, <span class="author">Pujol RM</span>, <span class="author">Rautanen A</span>, <span class="author">Riveira-Munoz E</span>, <span class="author">Ryan AW</span>, <span class="author">Salmhofer W</span>, <span class="author">Samuelsson L</span>, <span class="author">Sawcer SJ</span>, <span class="author">Schalkwijk J</span>, <span class="author">Smith CH</span>, <span class="author">St&aring;hle M</span>, <span class="author">Su Z</span>, <span class="author">Tazi-Ahnini R</span>, <span class="author">Traupe H</span>, <span class="author">Viswanathan AC</span>, <span class="author">Warren RB</span>, <span class="author">Weger W</span>, <span class="author">Wolk K</span>, <span class="author">Wood N</span>, <span class="author">Worthington J</span>, <span class="author">Young HS</span>, <span class="author">Zeeuwen PL</span>, <span class="author">Hayday A</span>, <span class="author">Burden AD</span>, <span class="author">Griffiths CE</span>, <span class="author">Kere J</span>, <span class="author">Reis A</span>, <span class="author">McVean G</span>, <span class="author">Evans DM</span>, <span class="author">Brown MA</span>, <span class="author">Barker JN</span>, <span class="author">Peltonen L</span>, <span class="author">Donnelly P</span> and <span class="author">Trembath RC</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Oxford, UK.</p><p class="abstract">To identify new susceptibility loci for psoriasis, we undertook a genome-wide association study of 594,224 SNPs in 2,622 individuals with psoriasis and 5,667 controls. We identified associations at eight previously unreported genomic loci. Seven loci harbored genes with recognized immune functions (IL28RA, REL, IFIH1, ERAP1, TRAF3IP2, NFKBIA and TYK2). These associations were replicated in 9,079 European samples (six loci with a combined P &lt; 5 &times; 10&#x207B;&#x2078; and two loci with a combined P &lt; 5 &times; 10&#x207B;&#x2077;). We also report compelling evidence for an interaction between the HLA-C and ERAP1 loci (combined P = 6.95 &times; 10&#x207B;&#x2076;). ERAP1 plays an important role in MHC class I peptide processing. ERAP1 variants only influenced psoriasis susceptibility in individuals carrying the HLA-C risk allele. Our findings implicate pathways that integrate epidermal barrier dysfunction with innate and adaptive immune dysregulation in psoriasis pathogenesis.</p>
<p class="grants">Funded by: Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0601387</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">083948/Z/07/Z</span>, <span class="grant">084726</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">11</span>;<span class="pagerange">985-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20953190">20953190</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.694">10.1038/ng.694</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cell-type-specific long-range looping interactions identify distant regulatory elements of the CFTR gene.</h4>
<p class="authors"><span class="author">Gheldof N</span>, <span class="author">Smith EM</span>, <span class="author">Tabuchi TM</span>, <span class="author">Koch CM</span>, <span class="author">Dunham I</span>, <span class="author">Stamatoyannopoulos JA</span> and <span class="author">Dekker J</span></p>
<p class="affiliation">Program in Gene Function and Expression and Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, 364 Plantation Street, Worcester, MA 01605-0103, USA.</p><p class="abstract">Identification of regulatory elements and their target genes is complicated by the fact that regulatory elements can act over large genomic distances. Identification of long-range acting elements is particularly important in the case of disease genes as mutations in these elements can result in human disease. It is becoming increasingly clear that long-range control of gene expression is facilitated by chromatin looping interactions. These interactions can be detected by chromosome conformation capture (3C). Here, we employed 3C as a discovery tool for identification of long-range regulatory elements that control the cystic fibrosis transmembrane conductance regulator gene, CFTR. We identified four elements in a 460-kb region around the locus that loop specifically to the CFTR promoter exclusively in CFTR expressing cells. The elements are located 20 and 80 kb upstream; and 109 and 203 kb downstream of the CFTR promoter. These elements contain DNase I hypersensitive sites and histone modification patterns characteristic of enhancers. The elements also interact with each other and the latter two activate the CFTR promoter synergistically in reporter assays. Our results reveal novel long-range acting elements that control expression of CFTR and suggest that 3C-based approaches can be used for discovery of novel regulatory elements.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG003143</span>, <span class="grant">HG004592</span>, <span class="grant">R01 HG003143-06</span>, <span class="grant">U01 HG003168</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">13</span>;<span class="pagerange">4325-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20360044" rel="external">20360044</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2910055" rel="external">2910055</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq175" rel="external">10.1093/nar/gkq175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">No correlation between childhood maltreatment and telomere length.</h4>
<p class="authors"><span class="author">Glass D</span>, <span class="author">Parts L</span>, <span class="author">Knowles D</span>, <span class="author">Aviv A</span> and <span class="author">Spector TD</span></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">68</span>;<span class="number">6</span>;<span class="pagerange">e21-2; author reply e23-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20678755" rel="external">20678755</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2930212" rel="external">2930212</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2010.02.026" rel="external">10.1016/j.biopsych.2010.02.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcription profiling in human platelets reveals LRRFIP1 as a novel protein regulating platelet function.</h4>
<p class="authors"><span class="author">Goodall AH</span>, <span class="author">Burns P</span>, <span class="author">Salles I</span>, <span class="author">Macaulay IC</span>, <span class="author">Jones CI</span>, <span class="author">Ardissino D</span>, <span class="author">de Bono B</span>, <span class="author">Bray SL</span>, <span class="author">Deckmyn H</span>, <span class="author">Dudbridge F</span>, <span class="author">Fitzgerald DJ</span>, <span class="author">Garner SF</span>, <span class="author">Gusnanto A</span>, <span class="author">Koch K</span>, <span class="author">Langford C</span>, <span class="author">O&#39;Connor MN</span>, <span class="author">Rice CM</span>, <span class="author">Stemple D</span>, <span class="author">Stephens J</span>, <span class="author">Trip MD</span>, <span class="author">Zwaginga JJ</span>, <span class="author">Samani NJ</span>, <span class="author">Watkins NA</span>, <span class="author">Maguire PB</span>, <span class="author">Ouwehand WH</span> and <span class="author">Bloodomics Consortium</span></p>
<p class="affiliation">Department of Cardiovascular Science, University of Leicester, Clinical Sciences Wing, Glenfield Hospital, Leicester, UK. <a href="mailto:%61%68%67%35%40%6C%65.%61%63.%75%6B">&#97;&#104;&#103;&#53;&#64;&#108;&#101;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Within the healthy population, there is substantial, heritable, and interindividual variability in the platelet response. We explored whether a proportion of this variability could be accounted for by interindividual variation in gene expression. Through a correlative analysis of genome-wide platelet RNA expression data from 37 subjects representing the normal range of platelet responsiveness within a cohort of 500 subjects, we identified 63 genes in which transcript levels correlated with variation in the platelet response to adenosine diphosphate and/or the collagen-mimetic peptide, cross-linked collagen-related peptide. Many of these encode proteins with no reported function in platelets. An association study of 6 of the 63 genes in 4235 cases and 6379 controls showed a putative association with myocardial infarction for COMMD7 (COMM domain-containing protein 7) and a major deviation from the null hypo thesis for LRRFIP1 [leucine-rich repeat (in FLII) interacting protein 1]. Morpholino-based silencing in Danio rerio identified a modest role for commd7 and a significant effect for lrrfip1 as positive regulators of thrombus formation. Proteomic analysis of human platelet LRRFIP1-interacting proteins indicated that LRRFIP1 functions as a component of the platelet cytoskeleton, where it interacts with the actin-remodeling proteins Flightless-1 and Drebrin. Taken together, these data reveal novel proteins regulating the platelet response.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Medical Research Council: <span class="grant">MC_U105292688</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2010</span>;<span class="volume">116</span>;<span class="number">22</span>;<span class="pagerange">4646-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20833976">20833976</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2996120">2996120</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2010-04-280925">10.1182/blood-2010-04-280925</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BioRuby: bioinformatics software for the Ruby programming language.</h4>
<p class="authors"><span class="author">Goto N</span>, <span class="author">Prins P</span>, <span class="author">Nakao M</span>, <span class="author">Bonnal R</span>, <span class="author">Aerts J</span> and <span class="author">Katayama T</span></p>
<p class="affiliation">Department of Genome Informatics, Genome Information Research Center, Research Institute for Microbial Diseases, Osaka University, Japan.</p><p class="abstract"><strong>Summary</strong>: The BioRuby software toolkit contains a comprehensive set of free development tools and libraries for bioinformatics and molecular biology, written in the Ruby programming language. BioRuby has components for sequence analysis, pathway analysis, protein modelling and phylogenetic analysis; it supports many widely used data formats and provides easy access to databases, external programs and public web services, including BLAST, KEGG, GenBank, MEDLINE and GO. BioRuby comes with a tutorial, documentation and an interactive environment, which can be used in the shell, and in the web browser.</p><p class="abstract"><strong>Availability</strong>: BioRuby is free and open source software, made available under the Ruby license. BioRuby runs on all platforms that support Ruby, including Linux, Mac OS X and Windows. And, with JRuby, BioRuby runs on the Java Virtual Machine. The source code is available from http://www.bioruby.org/.</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%6B%61%74%61%79%61%6D%61%40%62%69%6F%72%75%62%79.%6F%72%67">&#107;&#97;&#116;&#97;&#121;&#97;&#109;&#97;&#64;&#98;&#105;&#111;&#114;&#117;&#98;&#121;&#46;&#111;&#114;&#103;</a></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">20</span>;<span class="pagerange">2617-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20739307" rel="external">20739307</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2951089" rel="external">2951089</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq475" rel="external">10.1093/bioinformatics/btq475</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeted TAP tags, phosphoproteomes and the biology of thought.</h4>
<p class="authors"><span class="author">Grant SG</span></p>
<p class="publication"><span class="title">Expert review of proteomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">169-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20377382" rel="external">20377382</a>; DOI: <a href="http://dx.doi.org/10.1586%2Fepr.09.109" rel="external">10.1586/epr.09.109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Computing behaviour in complex synapses</h4>
<p class="authors"><span class="author">Grant</span>, <span class="author">S.G</span></p>
<p class="publication"><span class="title">Biochemist.</span> <span class="year">2010</span>;<span class="volume">32</span>;<span class="pagerange">6-9</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PICNIC: an algorithm to predict absolute allelic copy number variation with microarray cancer data.</h4>
<p class="authors"><span class="author">Greenman CD</span>, <span class="author">Bignell G</span>, <span class="author">Butler A</span>, <span class="author">Edkins S</span>, <span class="author">Hinton J</span>, <span class="author">Beare D</span>, <span class="author">Swamy S</span>, <span class="author">Santarius T</span>, <span class="author">Chen L</span>, <span class="author">Widaa S</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%63%64%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#100;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">High-throughput oligonucleotide microarrays are commonly employed to investigate genetic disease, including cancer. The algorithms employed to extract genotypes and copy number variation function optimally for diploid genomes usually associated with inherited disease. However, cancer genomes are aneuploid in nature leading to systematic errors when using these techniques. We introduce a preprocessing transformation and hidden Markov model algorithm bespoke to cancer. This produces genotype classification, specification of regions of loss of heterozygosity, and absolute allelic copy number segmentation. Accurate prediction is demonstrated with a combination of independent experimental techniques. These methods are exemplified with affymetrix genome-wide SNP6.0 data from 755 cancer cell lines, enabling inference upon a number of features of biological interest. These data and the coded algorithm are freely available for download.</p>
<p class="publication"><span class="title">Biostatistics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">1</span>;<span class="pagerange">164-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19837654" rel="external">19837654</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2800165" rel="external">2800165</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbiostatistics%2Fkxp045" rel="external">10.1093/biostatistics/kxp045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rare copy number variants: a point of rarity in genetic risk for bipolar disorder and schizophrenia.</h4>
<p class="authors"><span class="author">Grozeva D</span>, <span class="author">Kirov G</span>, <span class="author">Ivanov D</span>, <span class="author">Jones IR</span>, <span class="author">Jones L</span>, <span class="author">Green EK</span>, <span class="author">St Clair DM</span>, <span class="author">Young AH</span>, <span class="author">Ferrier N</span>, <span class="author">Farmer AE</span>, <span class="author">McGuffin P</span>, <span class="author">Holmans PA</span>, <span class="author">Owen MJ</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Craddock N</span> and <span class="author">Wellcome Trust Case Control Consortium</span></p>
<p class="affiliation">Medical Research Council Centre for Neuropsychiatric Genetics and Genomics, School of Medicine, Cardiff University, Cardiff CF14 4XN, Wales, UK.</p><p class="abstract"><strong>Context</strong>: Recent studies suggest that copy number variation in the human genome is extensive and may play an important role in susceptibility to disease, including neuropsychiatric disorders such as schizophrenia and autism. The possible involvement of copy number variants (CNVs) in bipolar disorder has received little attention to date.</p><p class="abstract"><strong>Objectives</strong>: To determine whether large (&gt;100,000 base pairs) and rare (found in &lt;1% of the population) CNVs are associated with susceptibility to bipolar disorder and to compare with findings in schizophrenia.</p><p class="abstract"><strong>Design</strong>: A genome-wide survey of large, rare CNVs in a case-control sample using a high-density microarray.</p><p class="abstract"><strong>Setting</strong>: The Wellcome Trust Case Control Consortium.</p><p class="abstract"><strong>Participants</strong>: There were 1697 cases of bipolar disorder and 2806 nonpsychiatric controls. All participants were white UK residents.</p><p class="abstract">Overall load of CNVs and presence of rare CNVs.</p><p class="abstract"><strong>Results</strong>: The burden of CNVs in bipolar disorder was not increased compared with controls and was significantly less than in schizophrenia cases. The CNVs previously implicated in the etiology of schizophrenia were not more common in cases with bipolar disorder.</p><p class="abstract"><strong>Conclusions</strong>: Schizophrenia and bipolar disorder differ with respect to CNV burden in general and association with specific CNVs in particular. Our data are consistent with the possibility that possession of large, rare deletions may modify the phenotype in those at risk of psychosis: those possessing such events are more likely to be diagnosed as having schizophrenia, and those without them are more likely to be diagnosed as having bipolar disorder.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0600329</span>, <span class="grant">G0701003</span>, <span class="grant">G0701420</span>, <span class="grant">G0800509</span>, <span class="grant">G0800759</span>, <span class="grant">G90/106</span>; Wellcome Trust: <span class="grant">061858</span></p>
<p class="publication"><span class="title">Archives of general psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">67</span>;<span class="number">4</span>;<span class="pagerange">318-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20368508">20368508</a>; DOI: <a href="http://dx.doi.org/10.1001%2Farchgenpsychiatry.2010.25">10.1001/archgenpsychiatry.2010.25</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CASK mutations are frequent in males and cause X-linked nystagmus and variable XLMR phenotypes.</h4>
<p class="authors"><span class="author">Hackett A</span>, <span class="author">Tarpey PS</span>, <span class="author">Licata A</span>, <span class="author">Cox J</span>, <span class="author">Whibley A</span>, <span class="author">Boyle J</span>, <span class="author">Rogers C</span>, <span class="author">Grigg J</span>, <span class="author">Partington M</span>, <span class="author">Stevenson RE</span>, <span class="author">Tolmie J</span>, <span class="author">Yates JR</span>, <span class="author">Turner G</span>, <span class="author">Wilson M</span>, <span class="author">Futreal AP</span>, <span class="author">Corbett M</span>, <span class="author">Shaw M</span>, <span class="author">Gecz J</span>, <span class="author">Raymond FL</span>, <span class="author">Stratton MR</span>, <span class="author">Schwartz CE</span> and <span class="author">Abidi FE</span></p>
<p class="affiliation">Genetics of Learning Disability Service, Hunter Genetics, Waratah, New South Wales, Australia. <a href="mailto:%61%6E%6E%61.%68%61%63%6B%65%74%74%40%68%6E%65%68%65%61%6C%74%68.%6E%73%77.%67%6F%76.%61%75">&#97;&#110;&#110;&#97;&#46;&#104;&#97;&#99;&#107;&#101;&#116;&#116;&#64;&#104;&#110;&#101;&#104;&#101;&#97;&#108;&#116;&#104;&#46;&#110;&#115;&#119;&#46;&#103;&#111;&#118;&#46;&#97;&#117;</a></p><p class="abstract">Mutations of the calcium/calmodulin-dependent serine protein kinase (CASK) gene have recently been associated with X-linked mental retardation (XLMR) with microcephaly, optic atrophy and brainstem and cerebellar hypoplasia, as well as with an X-linked syndrome having some FG-like features. Our group has recently identified four male probands from 358 probable XLMR families with missense mutations (p.Y268H, p.P396S, p.D710G and p.W919R) in the CASK gene. Congenital nystagmus, a rare and striking feature, was present in two of these families. We screened a further 45 probands with either nystagmus or microcephaly and mental retardation (MR), and identified two further mutations, a missense mutation (p.Y728C) and a splice mutation (c.2521-2A&gt;T) in two small families with nystagmus and MR. Detailed clinical examinations of all six families, including an ophthalmological review in four families, were undertaken to further characterise the phenotype. We report on the clinical features of 24 individuals, mostly male, from six families with CASK mutations. The phenotype was variable, ranging from non-syndromic mild MR to severe MR associated with microcephaly and dysmorphic facial features. Carrier females were variably affected. Congenital nystagmus was found in members of four of the families. Our findings reinforce the CASK gene as a relatively frequent cause of XLMR in females and males. We further define the phenotypic spectrum and demonstrate that affected males with missense mutations or in-frame deletions in CASK are frequently associated with congenital nystagmus and XLMR, a striking feature not previously reported.</p>
<p class="grants">Funded by: Medical Research Council; NICHD NIH HHS: <span class="grant">HD260202</span>; Wellcome Trust</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">5</span>;<span class="pagerange">544-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20029458" rel="external">20029458</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987321" rel="external">2987321</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2009.220" rel="external">10.1038/ejhg.2009.220</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NK cells influence both innate and adaptive immune responses after mucosal immunization with antigen and mucosal adjuvant.</h4>
<p class="authors"><span class="author">Hall LJ</span>, <span class="author">Clare S</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%6C.%68%61%6C%6C%40%75%63%63.%69%65">&#108;&#46;&#104;&#97;&#108;&#108;&#64;&#117;&#99;&#99;&#46;&#105;&#101;</a></p><p class="abstract">NK cells were found to be recruited in a temporally controlled manner to the nasal-associated lymphoid tissue and the cervical lymph nodes of mice after intranasal immunization with Ag85B-early secreted antigenic target 6 kDa from Mycobacterium tuberculosis mixed with Escherichia coli heat-labile toxin as adjuvant. These NK cells were activated and secreted a diverse range of cytokines and other immunomodulators. Using Ab depletion targeting anti-asialo GM1, we found evidence for altered trafficking, impaired activation, and cytokine secretion of dendritic cells, macrophages, and neutrophils in immunized NK cell-depleted mice compared with control animals. Analysis of Ag-specific immune responses revealed an attenuated Ab and cytokine response in immunized NK cell-depleted animals. Systemic administration of rIL-6 but not rIFN-gamma significantly restored immune responses in mice depleted of NK cells. In conclusion, cytokine production, particularly IL-6, via NK cells and NK cell-activated immune populations plays an important role in the establishment of local innate immune responses and the consequent development of adaptive immunity after mucosal immunization.</p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2010</span>;<span class="volume">184</span>;<span class="number">8</span>;<span class="pagerange">4327-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20220095" rel="external">20220095</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.0903357" rel="external">10.4049/jimmunol.0903357</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Probing local innate immune responses after mucosal immunisation.</h4>
<p class="authors"><span class="author">Hall LJ</span>, <span class="author">Clare S</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%6C.%68%61%6C%6C%40%75%63%63.%69%65">&#108;&#46;&#104;&#97;&#108;&#108;&#64;&#117;&#99;&#99;&#46;&#105;&#101;</a>.</p><p class="abstract"><strong>Background</strong>: Intranasal immunisation is potentially a very effective route for inducing both mucosal and systemic immunity to an infectious agent.</p><p class="abstract"><strong>Methods</strong>: Balb/c mice were intranasally immunised with the mucosal adjuvant heat labile toxin and the Mycobacterium tuberculosis fusion protein Ag85B-ESAT6 and early changes in innate immune responses within local mucosal tissues were examined using flow cytometry and confocal microscopy. Antigen-specific humoral and cellular immune responses were also evaluated.</p><p class="abstract"><strong>Results</strong>: Intranasal immunisation induced significant changes in both number and distribution of dendritic cells, macrophages and neutrophils within the nasal-associated lymphoid tissue and cervical lymph nodes in comparison to controls as early as 5 h post immunisation. Immunisation also resulted in a rapid and transient increase in activation marker expression first in the nasal-associated lymphoid tissue, and then in the cervical lymph nodes. This heightened activation status was also apparent from the pro-inflammatory cytokine profiles of these innate populations. In addition we also showed increased expression and distribution of a number of different cell adhesion molecules early after intranasal immunisation within these lymphoid tissues. These observed early changes correlated with the induction of a TH1 type immune response.</p><p class="abstract"><strong>Conclusions</strong>: These data provide insights into the complex nature of innate immune responses induced following intranasal immunisation within the upper respiratory tract, and may help clarify the concepts and provide the tools that are needed to exploit the full potential of mucosal vaccines.</p>
<p class="publication"><span class="title">Journal of immune based therapies and vaccines</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="pagerange">5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20836885" rel="external">20836885</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2945349" rel="external">2945349</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1476-8518-8-5" rel="external">10.1186/1476-8518-8-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Being more realistic about the public health impact of genomic medicine.</h4>
<p class="authors"><span class="author">Hall WD</span>, <span class="author">Mathews R</span> and <span class="author">Morley KI</span></p>
<p class="affiliation">University of Queensland Centre for Clinical Research, The University of Queensland, Herston, Queensland, Australia. <a href="mailto:%77.%68%61%6C%6C%40%75%71.%65%64%75.%61%75">&#119;&#46;&#104;&#97;&#108;&#108;&#64;&#117;&#113;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p>
<p class="publication"><span class="title">PLoS medicine</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20967240">20967240</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2953533">2953533</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pmed.1000347">10.1371/journal.pmed.1000347</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A pharmacometric model describing the relationship between warfarin dose and INR response with respect to variations in CYP2C9, VKORC1, and age.</h4>
<p class="authors"><span class="author">Hamberg AK</span>, <span class="author">Wadelius M</span>, <span class="author">Lindh JD</span>, <span class="author">Dahl ML</span>, <span class="author">Padrini R</span>, <span class="author">Deloukas P</span>, <span class="author">Rane A</span> and <span class="author">Jonsson EN</span></p>
<p class="affiliation">Department of Medical Sciences, Clinical Pharmacology, Uppsala University Hospital, Uppsala, Sweden. <a href="mailto:%61%6E%6E%61-%6B%61%72%69%6E.%68%61%6D%62%65%72%67%40%6D%65%64%73%63%69.%75%75.%73%65">&#97;&#110;&#110;&#97;&#45;&#107;&#97;&#114;&#105;&#110;&#46;&#104;&#97;&#109;&#98;&#101;&#114;&#103;&#64;&#109;&#101;&#100;&#115;&#99;&#105;&#46;&#117;&#117;&#46;&#115;&#101;</a></p><p class="abstract">The objective of the study was to update a previous NONMEM model to describe the relationship between warfarin dose and international normalized ratio (INR) response, to decrease the dependence of the model on pharmacokinetic (PK) data, and to improve the characterization of rare genotype combinations. The effects of age and CYP2C9 genotype on S-warfarin clearance were estimated from high-quality PK data. Thereafter, a temporal dose-response (K-PD) model was developed from information on dose, INR, age, and CYP2C9 and VKORC1 genotype, with drug clearance as a covariate. Two transit compartment chains accounted for the delay between exposure and response. CYP2C9 genotype was identified as the single most important predictor of required dose, causing a difference of up to 4.2-fold in the maintenance dose. VKORC1 accounted for a difference of up to 2.1-fold in dose, and age reduced the dose requirement by ~6% per decade. This reformulated K-PD model decreases dependence on PK data and enables robust assessment of INR response and dose predictions, even in individuals with rare genotype combinations.</p>
<p class="publication"><span class="title">Clinical pharmacology and therapeutics</span>&nbsp;<span class="year">2010</span>;<span class="volume">87</span>;<span class="number">6</span>;<span class="pagerange">727-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20410877">20410877</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fclpt.2010.37">10.1038/clpt.2010.37</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">KSHV-encoded miRNAs target MAF to induce endothelial cell reprogramming.</h4>
<p class="authors"><span class="author">Hansen A</span>, <span class="author">Henderson S</span>, <span class="author">Lagos D</span>, <span class="author">Nikitenko L</span>, <span class="author">Coulter E</span>, <span class="author">Roberts S</span>, <span class="author">Gratrix F</span>, <span class="author">Plaisance K</span>, <span class="author">Renne R</span>, <span class="author">Bower M</span>, <span class="author">Kellam P</span> and <span class="author">Boshoff C</span></p>
<p class="affiliation">Cancer Research UK Viral Oncology Group, University College London Cancer Institute, University College London, London WC1E 6BT, United Kingdom.</p><p class="abstract">Kaposi sarcoma herpesvirus (KSHV) induces transcriptional reprogramming of endothelial cells. In particular, KSHV-infected lymphatic endothelial cells (LECs) show an up-regulation of genes associated with blood vessel endothelial cells (BECs). Consequently, KSHV-infected tumor cells in Kaposi sarcoma are poorly differentiated endothelial cells, expressing markers of both LECs and BECs. MicroRNAs (miRNAs) are short noncoding RNA molecules that act post-transcriptionally to negatively regulate gene expression. Here we validate expression of the KSHV-encoded miRNAs in Kaposi sarcoma lesions and demonstrate that these miRNAs contribute to viral-induced reprogramming by silencing the cellular transcription factor MAF (musculoaponeurotic fibrosarcoma oncogene homolog). MAF is expressed in LECs but not in BECs. We identify a novel role for MAF as a transcriptional repressor, preventing expression of BEC-specific genes, thereby maintaining the differentiation status of LECs. These findings demonstrate that viral miRNAs could influence the differentiation status of infected cells, and thereby contribute to KSHV-induced oncogenesis.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800168</span></p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2010</span>;<span class="volume">24</span>;<span class="number">2</span>;<span class="pagerange">195-205</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20080955">20080955</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2807354">2807354</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.553410">10.1101/gad.553410</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of MRSA during hospital transmission and intercontinental spread.</h4>
<p class="authors"><span class="author">Harris SR</span>, <span class="author">Feil EJ</span>, <span class="author">Holden MT</span>, <span class="author">Quail MA</span>, <span class="author">Nickerson EK</span>, <span class="author">Chantratita N</span>, <span class="author">Gardete S</span>, <span class="author">Tavares A</span>, <span class="author">Day N</span>, <span class="author">Lindsay JA</span>, <span class="author">Edgeworth JD</span>, <span class="author">de Lencastre H</span>, <span class="author">Parkhill J</span>, <span class="author">Peacock SJ</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 15A, UK.</p><p class="abstract">Current methods for differentiating isolates of predominant lineages of pathogenic bacteria often do not provide sufficient resolution to define precise relationships. Here, we describe a high-throughput genomics approach that provides a high-resolution view of the epidemiology and microevolution of a dominant strain of methicillin-resistant Staphylococcus aureus (MRSA). This approach reveals the global geographic structure within the lineage, its intercontinental transmission through four decades, and the potential to trace person-to-person transmission within a hospital environment. The ability to interrogate and resolve bacterial populations is applicable to a range of infectious diseases, as well as microbial ecology.</p>
<p class="grants">Funded by: Department of Health; Wellcome Trust: <span class="grant">076964</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">327</span>;<span class="number">5964</span>;<span class="pagerange">469-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20093474">20093474</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2821690">2821690</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1182395">10.1126/science.1182395</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: a comprehensive resource for nematode research.</h4>
<p class="authors"><span class="author">Harris TW</span>, <span class="author">Antoshechkin I</span>, <span class="author">Bieri T</span>, <span class="author">Blasiar D</span>, <span class="author">Chan J</span>, <span class="author">Chen WJ</span>, <span class="author">De La Cruz N</span>, <span class="author">Davis P</span>, <span class="author">Duesbury M</span>, <span class="author">Fang R</span>, <span class="author">Fernandes J</span>, <span class="author">Han M</span>, <span class="author">Kishore R</span>, <span class="author">Lee R</span>, <span class="author">M&uuml;ller HM</span>, <span class="author">Nakamura C</span>, <span class="author">Ozersky P</span>, <span class="author">Petcherski A</span>, <span class="author">Rangarajan A</span>, <span class="author">Rogers A</span>, <span class="author">Schindelman G</span>, <span class="author">Schwarz EM</span>, <span class="author">Tuli MA</span>, <span class="author">Van Auken K</span>, <span class="author">Wang D</span>, <span class="author">Wang X</span>, <span class="author">Williams G</span>, <span class="author">Yook K</span>, <span class="author">Durbin R</span>, <span class="author">Stein LD</span>, <span class="author">Spieth J</span> and <span class="author">Sternberg PW</span></p>
<p class="affiliation">Ontario Institute For Cancer Research, Toronto, ON, Canada. <a href="mailto:%69%6E%66%6F%40%74%6F%64%64%68%61%72%72%69%73.%6E%65%74">&#105;&#110;&#102;&#111;&#64;&#116;&#111;&#100;&#100;&#104;&#97;&#114;&#114;&#105;&#115;&#46;&#110;&#101;&#116;</a></p><p class="abstract">WormBase (http://www.wormbase.org) is a central data repository for nematode biology. Initially created as a service to the Caenorhabditis elegans research field, WormBase has evolved into a powerful research tool in its own right. In the past 2 years, we expanded WormBase to include the complete genomic sequence, gene predictions and orthology assignments from a range of related nematodes. This comparative data enrich the C. elegans data with improved gene predictions and a better understanding of gene function. In turn, they bring the wealth of experimental knowledge of C. elegans to other systems of medical and agricultural importance. Here, we describe new species and data types now available at WormBase. In addition, we detail enhancements to our curatorial pipeline and website infrastructure to accommodate new genomes and an extensive user base.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">P41-HG02223</span>, <span class="grant">P41HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D463-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19910365" rel="external">19910365</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808986" rel="external">2808986</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp952" rel="external">10.1093/nar/gkp952</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">no tail integrates two modes of mesoderm induction.</h4>
<p class="authors"><span class="author">Harvey SA</span>, <span class="author">T&uuml;mpel S</span>, <span class="author">Dubrulle J</span>, <span class="author">Schier AF</span> and <span class="author">Smith JC</span></p>
<p class="affiliation">Wellcome Trust and Cancer Research UK, Gurdon Institute and Department of Zoology, University of Cambridge, Tennis Court Road, Cambridge CB2 1QN, UK. <a href="mailto:%73%68%31%39%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#104;&#49;&#57;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">During early zebrafish development the nodal signalling pathway patterns the embryo into three germ layers, in part by inducing the expression of no tail (ntl), which is essential for correct mesoderm formation. When nodal signalling is inhibited ntl fails to be expressed in the dorsal margin, but ventral ntl expression is unaffected. These observations indicate that ntl transcription is under both nodal-dependent and nodal-independent regulation. Consistent with these observations and with a role for ntl in mesoderm formation, some somites form within the tail region of embryos lacking nodal signalling. In an effort to understand how ntl is regulated and thus how mesoderm forms, we have mapped the elements responsible for nodal-dependent and nodal-independent expression of ntl in the margin of the embryo. Our work demonstrates that expression of ntl in the margin is the consequence of two separate enhancers, which act to mediate different mechanisms of mesoderm formation. One of these enhancers responds to nodal signalling, and the other to Wnt and BMP signalling. We demonstrate that the nodal-independent regulation of ntl is essential for tail formation. Misexpression of Wnt and BMP ligands can induce the formation of an ectopic tail, which contains somites, in embryos devoid of nodal signalling, and this tail formation is dependent on ntl function. Similarly, nodal-independent tail somite formation requires ntl. At later stages in development ntl is required for notochord formation, and our analysis has also led to the identification of the enhancer required for ntl expression in the developing notochord.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">137</span>;<span class="number">7</span>;<span class="pagerange">1127-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20215349" rel="external">20215349</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2835328" rel="external">2835328</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.046318" rel="external">10.1242/dev.046318</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary dynamics of Clostridium difficile over short and long time scales.</h4>
<p class="authors"><span class="author">He M</span>, <span class="author">Sebaihia M</span>, <span class="author">Lawley TD</span>, <span class="author">Stabler RA</span>, <span class="author">Dawson LF</span>, <span class="author">Martin MJ</span>, <span class="author">Holt KE</span>, <span class="author">Seth-Smith HM</span>, <span class="author">Quail MA</span>, <span class="author">Rance R</span>, <span class="author">Brooks K</span>, <span class="author">Churcher C</span>, <span class="author">Harris D</span>, <span class="author">Bentley SD</span>, <span class="author">Burrows C</span>, <span class="author">Clark L</span>, <span class="author">Corton C</span>, <span class="author">Murray V</span>, <span class="author">Rose G</span>, <span class="author">Thurston S</span>, <span class="author">van Tonder A</span>, <span class="author">Walker D</span>, <span class="author">Wren BW</span>, <span class="author">Dougan G</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom.</p><p class="abstract">Clostridium difficile has rapidly emerged as the leading cause of antibiotic-associated diarrheal disease, with the transcontinental spread of various PCR ribotypes, including 001, 017, 027 and 078. However, the genetic basis for the emergence of C. difficile as a human pathogen is unclear. Whole genome sequencing was used to analyze genetic variation and virulence of a diverse collection of thirty C. difficile isolates, to determine both macro and microevolution of the species. Horizontal gene transfer and large-scale recombination of core genes has shaped the C. difficile genome over both short and long time scales. Phylogenetic analysis demonstrates C. difficile is a genetically diverse species, which has evolved within the last 1.1-85 million years. By contrast, the disease-causing isolates have arisen from multiple lineages, suggesting that virulence evolved independently in the highly epidemic lineages.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2010</span>;<span class="volume">107</span>;<span class="number">16</span>;<span class="pagerange">7527-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20368420">20368420</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2867753">2867753</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0914322107">10.1073/pnas.0914322107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clear detection of ADIPOQ locus as the major gene for plasma adiponectin: results of genome-wide association analyses including 4659 European individuals.</h4>
<p class="authors"><span class="author">Heid IM</span>, <span class="author">Henneman P</span>, <span class="author">Hicks A</span>, <span class="author">Coassin S</span>, <span class="author">Winkler T</span>, <span class="author">Aulchenko YS</span>, <span class="author">Fuchsberger C</span>, <span class="author">Song K</span>, <span class="author">Hivert MF</span>, <span class="author">Waterworth DM</span>, <span class="author">Timpson NJ</span>, <span class="author">Richards JB</span>, <span class="author">Perry JR</span>, <span class="author">Tanaka T</span>, <span class="author">Amin N</span>, <span class="author">Kollerits B</span>, <span class="author">Pichler I</span>, <span class="author">Oostra BA</span>, <span class="author">Thorand B</span>, <span class="author">Frants RR</span>, <span class="author">Illig T</span>, <span class="author">Dupuis J</span>, <span class="author">Glaser B</span>, <span class="author">Spector T</span>, <span class="author">Guralnik J</span>, <span class="author">Egan JM</span>, <span class="author">Florez JC</span>, <span class="author">Evans DM</span>, <span class="author">Soranzo N</span>, <span class="author">Bandinelli S</span>, <span class="author">Carlson OD</span>, <span class="author">Frayling TM</span>, <span class="author">Burling K</span>, <span class="author">Smith GD</span>, <span class="author">Mooser V</span>, <span class="author">Ferrucci L</span>, <span class="author">Meigs JB</span>, <span class="author">Vollenweider P</span>, <span class="author">Dijk KW</span>, <span class="author">Pramstaller P</span>, <span class="author">Kronenberg F</span> and <span class="author">van Duijn CM</span></p>
<p class="affiliation">Department of Epidemiology and Preventive Medicine, Regensburg University Medical Center, Regensburg, Germany.</p><p class="abstract"><strong>Objective</strong>: Plasma adiponectin is strongly associated with various components of metabolic syndrome, type 2 diabetes and cardiovascular outcomes. Concentrations are highly heritable and differ between men and women. We therefore aimed to investigate the genetics of plasma adiponectin in men and women.</p><p class="abstract"><strong>Methods</strong>: We combined genome-wide association scans of three population-based studies including 4659 persons. For the replication stage in 13795 subjects, we selected the 20 top signals of the combined analysis, as well as the 10 top signals with p-values less than 1.0 x 10(-4) for each the men- and the women-specific analyses. We further selected 73 SNPs that were consistently associated with metabolic syndrome parameters in previous genome-wide association studies to check for their association with plasma adiponectin.</p><p class="abstract"><strong>Results</strong>: The ADIPOQ locus showed genome-wide significant p-values in the combined (p=4.3 x 10(-24)) as well as in both women- and men-specific analyses (p=8.7 x 10(-17) and p=2.5 x 10(-11), respectively). None of the other 39 top signal SNPs showed evidence for association in the replication analysis. None of 73 SNPs from metabolic syndrome loci exhibited association with plasma adiponectin (p&gt;0.01).</p><p class="abstract"><strong>Conclusions</strong>: We demonstrated the ADIPOQ gene as the only major gene for plasma adiponectin, which explains 6.7% of the phenotypic variance. We further found that neither this gene nor any of the metabolic syndrome loci explained the sex differences observed for plasma adiponectin. Larger studies are needed to identify more moderate genetic determinants of plasma adiponectin.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">N01 HC025195</span>; NIDDK NIH HHS: <span class="grant">R01 DK075787-01A1</span></p>
<p class="publication"><span class="title">Atherosclerosis</span>&nbsp;<span class="year">2010</span>;<span class="volume">208</span>;<span class="number">2</span>;<span class="pagerange">412-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20018283" rel="external">20018283</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2845297" rel="external">2845297</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.atherosclerosis.2009.11.035" rel="external">10.1016/j.atherosclerosis.2009.11.035</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis identifies 13 new loci associated with waist-hip ratio and reveals sexual dimorphism in the genetic basis of fat distribution.</h4>
<p class="authors"><span class="author">Heid IM</span>, <span class="author">Jackson AU</span>, <span class="author">Randall JC</span>, <span class="author">Winkler TW</span>, <span class="author">Qi L</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Thorleifsson G</span>, <span class="author">Zillikens MC</span>, <span class="author">Speliotes EK</span>, <span class="author">M&auml;gi R</span>, <span class="author">Workalemahu T</span>, <span class="author">White CC</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Harris TB</span>, <span class="author">Berndt SI</span>, <span class="author">Ingelsson E</span>, <span class="author">Willer CJ</span>, <span class="author">Weedon MN</span>, <span class="author">Luan J</span>, <span class="author">Vedantam S</span>, <span class="author">Esko T</span>, <span class="author">Kilpel&auml;inen TO</span>, <span class="author">Kutalik Z</span>, <span class="author">Li S</span>, <span class="author">Monda KL</span>, <span class="author">Dixon AL</span>, <span class="author">Holmes CC</span>, <span class="author">Kaplan LM</span>, <span class="author">Liang L</span>, <span class="author">Min JL</span>, <span class="author">Moffatt MF</span>, <span class="author">Molony C</span>, <span class="author">Nicholson G</span>, <span class="author">Schadt EE</span>, <span class="author">Zondervan KT</span>, <span class="author">Feitosa MF</span>, <span class="author">Ferreira T</span>, <span class="author">Lango Allen H</span>, <span class="author">Weyant RJ</span>, <span class="author">Wheeler E</span>, <span class="author">Wood AR</span>, <span class="author">MAGIC</span>, <span class="author">Estrada K</span>, <span class="author">Goddard ME</span>, <span class="author">Lettre G</span>, <span class="author">Mangino M</span>, <span class="author">Nyholt DR</span>, <span class="author">Purcell S</span>, <span class="author">Smith AV</span>, <span class="author">Visscher PM</span>, <span class="author">Yang J</span>, <span class="author">McCarroll SA</span>, <span class="author">Nemesh J</span>, <span class="author">Voight BF</span>, <span class="author">Absher D</span>, <span class="author">Amin N</span>, <span class="author">Aspelund T</span>, <span class="author">Coin L</span>, <span class="author">Glazer NL</span>, <span class="author">Hayward C</span>, <span class="author">Heard-Costa NL</span>, <span class="author">Hottenga JJ</span>, <span class="author">Johansson A</span>, <span class="author">Johnson T</span>, <span class="author">Kaakinen M</span>, <span class="author">Kapur K</span>, <span class="author">Ketkar S</span>, <span class="author">Knowles JW</span>, <span class="author">Kraft P</span>, <span class="author">Kraja AT</span>, <span class="author">Lamina C</span>, <span class="author">Leitzmann MF</span>, <span class="author">McKnight B</span>, <span class="author">Morris AP</span>, <span class="author">Ong KK</span>, <span class="author">Perry JR</span>, <span class="author">Peters MJ</span>, <span class="author">Polasek O</span>, <span class="author">Prokopenko I</span>, <span class="author">Rayner NW</span>, <span class="author">Ripatti S</span>, <span class="author">Rivadeneira F</span>, <span class="author">Robertson NR</span>, <span class="author">Sanna S</span>, <span class="author">Sovio U</span>, <span class="author">Surakka I</span>, <span class="author">Teumer A</span>, <span class="author">van Wingerden S</span>, <span class="author">Vitart V</span>, <span class="author">Zhao JH</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Chines PS</span>, <span class="author">Fisher E</span>, <span class="author">Kulzer JR</span>, <span class="author">Lecoeur C</span>, <span class="author">Narisu N</span>, <span class="author">Sandholt C</span>, <span class="author">Scott LJ</span>, <span class="author">Silander K</span>, <span class="author">Stark K</span>, <span class="author">Tammesoo ML</span>, <span class="author">Teslovich TM</span>, <span class="author">Timpson NJ</span>, <span class="author">Watanabe RM</span>, <span class="author">Welch R</span>, <span class="author">Chasman DI</span>, <span class="author">Cooper MN</span>, <span class="author">Jansson JO</span>, <span class="author">Kettunen J</span>, <span class="author">Lawrence RW</span>, <span class="author">Pellikka N</span>, <span class="author">Perola M</span>, <span class="author">Vandenput L</span>, <span class="author">Alavere H</span>, <span class="author">Almgren P</span>, <span class="author">Atwood LD</span>, <span class="author">Bennett AJ</span>, <span class="author">Biffar R</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bornstein SR</span>, <span class="author">Buchanan TA</span>, <span class="author">Campbell H</span>, <span class="author">Day IN</span>, <span class="author">Dei M</span>, <span class="author">D&ouml;rr M</span>, <span class="author">Elliott P</span>, <span class="author">Erdos MR</span>, <span class="author">Eriksson JG</span>, <span class="author">Freimer NB</span>, <span class="author">Fu M</span>, <span class="author">Gaget S</span>, <span class="author">Geus EJ</span>, <span class="author">Gjesing AP</span>, <span class="author">Grallert H</span>, <span class="author">Gr&auml;ssler J</span>, <span class="author">Groves CJ</span>, <span class="author">Guiducci C</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hassanali N</span>, <span class="author">Havulinna AS</span>, <span class="author">Herzig KH</span>, <span class="author">Hicks AA</span>, <span class="author">Hui J</span>, <span class="author">Igl W</span>, <span class="author">Jousilahti P</span>, <span class="author">Jula A</span>, <span class="author">Kajantie E</span>, <span class="author">Kinnunen L</span>, <span class="author">Kolcic I</span>, <span class="author">Koskinen S</span>, <span class="author">Kovacs P</span>, <span class="author">Kroemer HK</span>, <span class="author">Krzelj V</span>, <span class="author">Kuusisto J</span>, <span class="author">Kvaloy K</span>, <span class="author">Laitinen J</span>, <span class="author">Lantieri O</span>, <span class="author">Lathrop GM</span>, <span class="author">Lokki ML</span>, <span class="author">Luben RN</span>, <span class="author">Ludwig B</span>, <span class="author">McArdle WL</span>, <span class="author">McCarthy A</span>, <span class="author">Morken MA</span>, <span class="author">Nelis M</span>, <span class="author">Neville MJ</span>, <span class="author">Par&eacute; G</span>, <span class="author">Parker AN</span>, <span class="author">Peden JF</span>, <span class="author">Pichler I</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Platou CG</span>, <span class="author">Pouta A</span>, <span class="author">Ridderstr&aring;le M</span>, <span class="author">Samani NJ</span>, <span class="author">Saramies J</span>, <span class="author">Sinisalo J</span>, <span class="author">Smit JH</span>, <span class="author">Strawbridge RJ</span>, <span class="author">Stringham HM</span>, <span class="author">Swift AJ</span>, <span class="author">Teder-Laving M</span>, <span class="author">Thomson B</span>, <span class="author">Usala G</span>, <span class="author">van Meurs JB</span>, <span class="author">van Ommen GJ</span>, <span class="author">Vatin V</span>, <span class="author">Volpato CB</span>, <span class="author">Wallaschofski H</span>, <span class="author">Walters GB</span>, <span class="author">Widen E</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Witte DR</span>, <span class="author">Zgaga L</span>, <span class="author">Zitting P</span>, <span class="author">Beilby JP</span>, <span class="author">James AL</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Nieminen MS</span>, <span class="author">Ohlsson C</span>, <span class="author">Palmer LJ</span>, <span class="author">Raitakari O</span>, <span class="author">Ridker PM</span>, <span class="author">Stumvoll M</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Viikari J</span>, <span class="author">Balkau B</span>, <span class="author">Ben-Shlomo Y</span>, <span class="author">Bergman RN</span>, <span class="author">Boeing H</span>, <span class="author">Smith GD</span>, <span class="author">Ebrahim S</span>, <span class="author">Froguel P</span>, <span class="author">Hansen T</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hveem K</span>, <span class="author">Isomaa B</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Karpe F</span>, <span class="author">Khaw KT</span>, <span class="author">Laakso M</span>, <span class="author">Lawlor DA</span>, <span class="author">Marre M</span>, <span class="author">Meitinger T</span>, <span class="author">Metspalu A</span>, <span class="author">Midthjell K</span>, <span class="author">Pedersen O</span>, <span class="author">Salomaa V</span>, <span class="author">Schwarz PE</span>, <span class="author">Tuomi T</span>, <span class="author">Tuomilehto J</span>, <span class="author">Valle TT</span>, <span class="author">Wareham NJ</span>, <span class="author">Arnold AM</span>, <span class="author">Beckmann JS</span>, <span class="author">Bergmann S</span>, <span class="author">Boerwinkle E</span>, <span class="author">Boomsma DI</span>, <span class="author">Caulfield MJ</span>, <span class="author">Collins FS</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Gudnason V</span>, <span class="author">Gyllensten U</span>, <span class="author">Hamsten A</span>, <span class="author">Hattersley AT</span>, <span class="author">Hofman A</span>, <span class="author">Hu FB</span>, <span class="author">Illig T</span>, <span class="author">Iribarren C</span>, <span class="author">Jarvelin MR</span>, <span class="author">Kao WH</span>, <span class="author">Kaprio J</span>, <span class="author">Launer LJ</span>, <span class="author">Munroe PB</span>, <span class="author">Oostra B</span>, <span class="author">Penninx BW</span>, <span class="author">Pramstaller PP</span>, <span class="author">Psaty BM</span>, <span class="author">Quertermous T</span>, <span class="author">Rissanen A</span>, <span class="author">Rudan I</span>, <span class="author">Shuldiner AR</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Syvanen AC</span>, <span class="author">Uda M</span>, <span class="author">Uitterlinden A</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Vollenweider P</span>, <span class="author">Wilson JF</span>, <span class="author">Witteman JC</span>, <span class="author">Wright AF</span>, <span class="author">Abecasis GR</span>, <span class="author">Boehnke M</span>, <span class="author">Borecki IB</span>, <span class="author">Deloukas P</span>, <span class="author">Frayling TM</span>, <span class="author">Groop LC</span>, <span class="author">Haritunians T</span>, <span class="author">Hunter DJ</span>, <span class="author">Kaplan RC</span>, <span class="author">North KE</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Peltonen L</span>, <span class="author">Schlessinger D</span>, <span class="author">Strachan DP</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Assimes TL</span>, <span class="author">Wichmann HE</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">van Duijn CM</span>, <span class="author">Stefansson K</span>, <span class="author">Cupples LA</span>, <span class="author">Loos RJ</span>, <span class="author">Barroso I</span>, <span class="author">McCarthy MI</span>, <span class="author">Fox CS</span>, <span class="author">Mohlke KL</span> and <span class="author">Lindgren CM</span></p>
<p class="affiliation">Regensburg University Medical Center, Department of Epidemiology and Preventive Medicine, Regensburg, Germany.</p><p class="abstract">Waist-hip ratio (WHR) is a measure of body fat distribution and a predictor of metabolic consequences independent of overall adiposity. WHR is heritable, but few genetic variants influencing this trait have been identified. We conducted a meta-analysis of 32 genome-wide association studies for WHR adjusted for body mass index (comprising up to 77,167 participants), following up 16 loci in an additional 29 studies (comprising up to 113,636 subjects). We identified 13 new loci in or near RSPO3, VEGFA, TBX15-WARS2, NFE2L3, GRB14, DNM3-PIGC, ITPR2-SSPN, LY86, HOXC13, ADAMTS9, ZNRF3-KREMEN1, NISCH-STAB1 and CPEB4 (P = 1.9 &times; 10&#x207B;&#x2079; to P = 1.8 &times; 10&#x207B;&#x2074;&#x2070;) and the known signal at LYPLAL1. Seven of these loci exhibited marked sexual dimorphism, all with a stronger effect on WHR in women than men (P for sex difference = 1.9 &times; 10&#x207B;&sup3; to P = 1.2 &times; 10&#x207B;&sup1;&sup3;). These findings provide evidence for multiple loci that modulate body fat distribution independent of overall adiposity and reveal strong gene-by-sex interactions.</p>
<p class="grants">Funded by: British Heart Foundation; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401527</span>, <span class="grant">G0500115</span>, <span class="grant">G0501184</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G9521010</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A390_1107</span>; NCI NIH HHS: <span class="grant">CA047988</span>, <span class="grant">CA49449</span>, <span class="grant">CA50385</span>, <span class="grant">CA65725</span>, <span class="grant">CA67262</span>, <span class="grant">CA87969</span>, <span class="grant">P01 CA087969</span>, <span class="grant">P01 CA087969-12</span>, <span class="grant">R01 CA047988</span>, <span class="grant">R01 CA047988-20</span>, <span class="grant">R01 CA050385</span>, <span class="grant">R01 CA050385-20</span>, <span class="grant">R01 CA065725</span>, <span class="grant">R01 CA065725-14</span>, <span class="grant">R01 CA067262</span>, <span class="grant">R01 CA067262-14</span>, <span class="grant">U01 CA049449</span>, <span class="grant">U01 CA049449-21</span>, <span class="grant">U01 CA098233</span>, <span class="grant">U01 CA098233-08</span>, <span class="grant">&shy;U01-CA098233</span>; NCRR NIH HHS: <span class="grant">UL1 RR025005</span>, <span class="grant">UL1 RR025005-04</span>, <span class="grant">UL1-RR025005</span>, <span class="grant">&shy;UL1-RR025005</span>; NHGRI NIH HHS: <span class="grant">HG002651</span>, <span class="grant">HG005581</span>, <span class="grant">N01 HG065403</span>, <span class="grant">N01-HG-65403</span>, <span class="grant">R01 HG002651</span>, <span class="grant">R01 HG002651-05</span>, <span class="grant">RC2 HG005581</span>, <span class="grant">RC2 HG005581-02</span>, <span class="grant">T32 HG000040</span>, <span class="grant">T32 HG000040-14</span>, <span class="grant">U01 HG004399</span>, <span class="grant">U01 HG004399-02</span>, <span class="grant">U01 HG004402</span>, <span class="grant">U01 HG004402-02</span>, <span class="grant">Z01 HG000024-14</span>, <span class="grant">&shy;T32-HG00040</span>, <span class="grant">&shy;U01-HG004399</span>, <span class="grant">&shy;U01-HG004402</span>; NHLBI NIH HHS: <span class="grant">HL043851</span>, <span class="grant">HL084729</span>, <span class="grant">HL71981</span>, <span class="grant">K99 HL094535</span>, <span class="grant">K99 HL094535-02</span>, <span class="grant">N01 HC015103</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC035129</span>, <span class="grant">N01 HC045133</span>, <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC055016</span>, <span class="grant">N01 HC055018</span>, <span class="grant">N01 HC055019</span>, <span class="grant">N01 HC055020</span>, <span class="grant">N01 HC055021</span>, <span class="grant">N01 HC055022</span>, <span class="grant">N01 HC055222</span>, <span class="grant">N01 HC075150</span>, <span class="grant">N01 HC085079</span>, <span class="grant">N01 HC085080</span>, <span class="grant">N01 HC085081</span>, <span class="grant">N01 HC085082</span>, <span class="grant">N01 HC085083</span>, <span class="grant">N01 HC085084</span>, <span class="grant">N01 HC085085</span>, <span class="grant">N01 HC085086</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC55222</span>, <span class="grant">R01 HL043851</span>, <span class="grant">R01 HL043851-10</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL059367-10</span>, <span class="grant">R01 HL071981</span>, <span class="grant">R01 HL071981-07</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL086694-03</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087641-03</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL087647-03</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL087652-03</span>, <span class="grant">R01 HL087679-03</span>, <span class="grant">R01 HL087700</span>, <span class="grant">R01 HL087700-03</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL088119-04</span>, <span class="grant">R01-HL087647</span>, <span class="grant">R01-HL59367</span>, <span class="grant">U01 HL054527</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL080295</span>, <span class="grant">U01 HL080295-04</span>, <span class="grant">U01 HL084729</span>, <span class="grant">U01 HL084729-03</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U01 HL084756-03</span>, <span class="grant">U01-HL72515</span>, <span class="grant">&shy;K99HL094535</span>, <span class="grant">&shy;N01-HC-25195</span>, <span class="grant">&shy;N01-HC-55019</span>, <span class="grant">&shy;N01-HC-55020</span>, <span class="grant">&shy;N01-HC-55021</span>, <span class="grant">&shy;N01-HC-55022</span>, <span class="grant">&shy;N01-HC15103</span>, <span class="grant">&shy;N01-HC35129</span>, <span class="grant">&shy;N01-HC45133</span>, <span class="grant">&shy;N01-HC55015</span>, <span class="grant">&shy;N01-HC55016</span>, <span class="grant">&shy;N01-HC75150</span>, <span class="grant">&shy;N01-HC85079</span>, <span class="grant">&shy;N01-HC85080</span>, <span class="grant">&shy;N01-HC85081</span>, <span class="grant">&shy;N01-HC85082</span>, <span class="grant">&shy;N01-HC85083</span>, <span class="grant">&shy;N01-HC85084</span>, <span class="grant">&shy;N01-HC85085</span>, <span class="grant">&shy;N01-HC85086</span>, <span class="grant">&shy;R01-HL086694</span>, <span class="grant">&shy;R01-HL087641</span>, <span class="grant">&shy;R01-HL087679</span>, <span class="grant">&shy;R01-HL087700</span>, <span class="grant">&shy;R01-HL088119</span>, <span class="grant">&shy;R01&shy;HL087652</span>, <span class="grant">&shy;U01-HL084756</span>; NIA NIH HHS: <span class="grant">N01 AG012100</span>, <span class="grant">N01 AG012109</span>, <span class="grant">N01-AG-1-2109</span>, <span class="grant">R01 AG031890-02</span>, <span class="grant">&shy;N01-AG-12100</span>, <span class="grant">&shy;R01-AG031890</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK072193</span>, <span class="grant">DK075787</span>, <span class="grant">DK58845</span>, <span class="grant">F32 DK079466</span>, <span class="grant">F32 DK079466-01</span>, <span class="grant">K23 DK080145</span>, <span class="grant">K23 DK080145-01</span>, <span class="grant">K23-DK080145</span>, <span class="grant">P30 DK046200</span>, <span class="grant">P30 DK046200-14</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30 DK072488-06</span>, <span class="grant">P60 DK020541</span>, <span class="grant">R01 DK056690</span>, <span class="grant">R01 DK058845</span>, <span class="grant">R01 DK058845-11</span>, <span class="grant">R01 DK068336</span>, <span class="grant">R01 DK068336-03</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK072193-05</span>, <span class="grant">R01 DK073490</span>, <span class="grant">R01 DK073490-05</span>, <span class="grant">R01 DK075681</span>, <span class="grant">R01 DK075681-04</span>, <span class="grant">R01 DK075787</span>, <span class="grant">R01 DK075787-05</span>, <span class="grant">R01 DK089256</span>, <span class="grant">R01-DK068336</span>, <span class="grant">R01-DK075787</span>, <span class="grant">U01 DK062370</span>, <span class="grant">U01 DK062370-08</span>, <span class="grant">U01 DK062418</span>, <span class="grant">U01 DK062418-06</span>, <span class="grant">&shy;K23-DK080145</span>, <span class="grant">&shy;P30-DK072488</span>, <span class="grant">&shy;R01-DK-073490</span>, <span class="grant">&shy;R01-DK075681</span>, <span class="grant">&shy;R01-DK075787</span>, <span class="grant">&shy;U01-DK062418</span>; NIGMS NIH HHS: <span class="grant">U01 GM074518</span>, <span class="grant">U01 GM074518-05</span>, <span class="grant">&shy;U01-GM074518</span>; NIMH NIH HHS: <span class="grant">R01 MH063706-05</span>, <span class="grant">R01 MH084698</span>, <span class="grant">R01 MH084698-03</span>, <span class="grant">RL1 MH083268</span>, <span class="grant">RL1 MH083268-05</span>, <span class="grant">&shy;1RL1-MH083268-01</span>, <span class="grant">&shy;MH084698</span>, <span class="grant">&shy;R01-MH63706</span>; PHS HHS: <span class="grant">&shy;263-MA-410953</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">068545</span>, <span class="grant">072960</span>, <span class="grant">075491</span>, <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079557</span>, <span class="grant">079895</span>, <span class="grant">081682</span>, <span class="grant">083270</span>, <span class="grant">085235</span>, <span class="grant">085301</span>, <span class="grant">086596</span>, <span class="grant">088885</span>, <span class="grant">089061</span>, <span class="grant">091746</span>, <span class="grant">&shy;068545/Z/02</span>, <span class="grant">&shy;072960</span>, <span class="grant">&shy;076113/B/04/Z</span>, <span class="grant">&shy;091746/Z/10/Z</span>, <span class="grant">&shy;WT086596/Z/08/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">11</span>;<span class="pagerange">949-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20935629">20935629</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3000924">3000924</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.685">10.1038/ng.685</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A trans-acting locus regulates an anti-viral expression network and type 1 diabetes risk.</h4>
<p class="authors"><span class="author">Heinig M</span>, <span class="author">Petretto E</span>, <span class="author">Wallace C</span>, <span class="author">Bottolo L</span>, <span class="author">Rotival M</span>, <span class="author">Lu H</span>, <span class="author">Li Y</span>, <span class="author">Sarwar R</span>, <span class="author">Langley SR</span>, <span class="author">Bauerfeind A</span>, <span class="author">Hummel O</span>, <span class="author">Lee YA</span>, <span class="author">Paskas S</span>, <span class="author">Rintisch C</span>, <span class="author">Saar K</span>, <span class="author">Cooper J</span>, <span class="author">Buchan R</span>, <span class="author">Gray EE</span>, <span class="author">Cyster JG</span>, <span class="author">Cardiogenics Consortium</span>, <span class="author">Erdmann J</span>, <span class="author">Hengstenberg C</span>, <span class="author">Maouche S</span>, <span class="author">Ouwehand WH</span>, <span class="author">Rice CM</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Goodall AH</span>, <span class="author">Schulz H</span>, <span class="author">Roider HG</span>, <span class="author">Vingron M</span>, <span class="author">Blankenberg S</span>, <span class="author">M&uuml;nzel T</span>, <span class="author">Zeller T</span>, <span class="author">Szymczak S</span>, <span class="author">Ziegler A</span>, <span class="author">Tiret L</span>, <span class="author">Smyth DJ</span>, <span class="author">Pravenec M</span>, <span class="author">Aitman TJ</span>, <span class="author">Cambien F</span>, <span class="author">Clayton D</span>, <span class="author">Todd JA</span>, <span class="author">Hubner N</span> and <span class="author">Cook SA</span></p>
<p class="affiliation">Max-Delbr&uuml;ck-Center for Molecular Medicine (MDC), Berlin, Germany.</p><p class="abstract">Combined analyses of gene networks and DNA sequence variation can provide new insights into the aetiology of common diseases that may not be apparent from genome-wide association studies alone. Recent advances in rat genomics are facilitating systems-genetics approaches. Here we report the use of integrated genome-wide approaches across seven rat tissues to identify gene networks and the loci underlying their regulation. We defined an interferon regulatory factor 7 (IRF7)-driven inflammatory network (IDIN) enriched for viral response genes, which represents a molecular biomarker for macrophages and which was regulated in multiple tissues by a locus on rat chromosome 15q25. We show that Epstein-Barr virus induced gene 2 (Ebi2, also known as Gpr183), which lies at this locus and controls B lymphocyte migration, is expressed in macrophages and regulates the IDIN. The human orthologous locus on chromosome 13q32 controlled the human equivalent of the IDIN, which was conserved in monocytes. IDIN genes were more likely to associate with susceptibility to type 1 diabetes (T1D)-a macrophage-associated autoimmune disease-than randomly selected immune response genes (P = 8.85&thinsp;&times;&thinsp;10(-6)). The human locus controlling the IDIN was associated with the risk of T1D at single nucleotide polymorphism rs9585056 (P = 7.0&thinsp;&times;&thinsp;10(-10); odds ratio, 1.15), which was one of five single nucleotide polymorphisms in this region associated with EBI2 (GPR183) expression. These data implicate IRF7 network genes and their regulatory locus in the pathogenesis of T1D.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">P301/10/0290</span>; Medical Research Council: <span class="grant">MC_U120061454</span>, <span class="grant">MC_U120085815</span>, <span class="grant">MC_U120097112</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">076113</span>, <span class="grant">089989</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">467</span>;<span class="number">7314</span>;<span class="pagerange">460-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20827270">20827270</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3657719">3657719</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09386">10.1038/nature09386</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting Report: &quot;Metagenomics, Metadata and Meta-analysis&quot; (M3) Workshop at the Pacific Symposium on Biocomputing 2010.</h4>
<p class="authors"><span class="author">Hirschman L</span>, <span class="author">Sterk P</span>, <span class="author">Field D</span>, <span class="author">Wooley J</span>, <span class="author">Cochrane G</span>, <span class="author">Gilbert J</span>, <span class="author">Kolker E</span>, <span class="author">Kyrpides N</span>, <span class="author">Meyer F</span>, <span class="author">Mizrachi I</span>, <span class="author">Nakamura Y</span>, <span class="author">Sansone SA</span>, <span class="author">Schriml L</span>, <span class="author">Tatusova T</span>, <span class="author">White O</span> and <span class="author">Yilmaz P</span></p><p class="abstract">This report summarizes the M3 Workshop held at the January 2010 Pacific Symposium on Biocomputing. The workshop, organized by Genomic Standards Consortium members, included five contributed talks, a series of short presentations from stakeholders in the genomics standards community, a poster session, and, in the evening, an open discussion session to review current projects and examine future directions for the GSC and its stakeholders.</p>
<p class="publication"><span class="title">Standards in genomic sciences</span>&nbsp;<span class="year">2010</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">357-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21304719" rel="external">21304719</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3035291" rel="external">3035291</a>; DOI: <a href="http://dx.doi.org/10.4056%2Fsigs.802738" rel="external">10.4056/sigs.802738</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intrahost evolutionary dynamics of canine influenza virus in naive and partially immune dogs.</h4>
<p class="authors"><span class="author">Hoelzer K</span>, <span class="author">Murcia PR</span>, <span class="author">Baillie GJ</span>, <span class="author">Wood JL</span>, <span class="author">Metzger SM</span>, <span class="author">Osterrieder N</span>, <span class="author">Dubovi EJ</span>, <span class="author">Holmes EC</span> and <span class="author">Parrish CR</span></p>
<p class="affiliation">Baker Institute for Animal Health, Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University, Hungerford Hill Road, Ithaca, NY 14853, USA.</p><p class="abstract">The patterns and dynamics of evolution in acutely infecting viruses within individual hosts are largely unknown. To this end, we investigated the intrahost variation of canine influenza virus (CIV) during the course of experimental infections in na&iuml;ve and partially immune dogs and in naturally infected dogs. Tracing sequence diversity in the gene encoding domain 1 of the hemagglutinin (HA1) protein over the time course of infection provided information on the patterns and processes of intrahost viral evolution and revealed some of the effects of partial host immunity. Viral populations sampled on any given day were generally characterized by mean pairwise genetic diversities between 0.1 and 0.2% and by mutational spectra that changed considerably on different days. Some observed mutations may have affected antigenicity or host range, including reversions of CIV host-associated mutations. Patterns of sequence diversity differed between na&iuml;ve and vaccinated dogs, with some presumably antigenic mutations transiently reaching high frequency in the latter. CIV populations are therefore characterized by the rapid generation and clearance of genetic diversity. Potentially advantageous mutations arise readily during the course of single infections and may give rise to antigenic escape or host range variants.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM080533</span></p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2010</span>;<span class="volume">84</span>;<span class="number">10</span>;<span class="pagerange">5329-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20219907" rel="external">20219907</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2863817" rel="external">2863817</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.02469-09" rel="external">10.1128/JVI.02469-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Platelets release novel thiol isomerase enzymes which are recruited to the cell surface following activation.</h4>
<p class="authors"><span class="author">Holbrook LM</span>, <span class="author">Watkins NA</span>, <span class="author">Simmonds AD</span>, <span class="author">Jones CI</span>, <span class="author">Ouwehand WH</span> and <span class="author">Gibbins JM</span></p>
<p class="affiliation">Institute for Cardiovascular and Metabolic Research, School of Biological Sciences, University of Reading, Whiteknights, Reading, Berkshire.</p><p class="abstract">The thiol isomerase enzymes protein disulphide isomerase (PDI) and endoplasmic reticulum protein 5 (ERp5) are released by resting and activated platelets. These re-associate with the cell surface where they modulate a range of platelet responses including adhesion, secretion and aggregation. Recent studies suggest the existence of yet uncharacterised platelet thiol isomerase proteins. This study aimed to identify which other thiol isomerase enzymes are present in human platelets. Through the use of immunoblotting, flow cytometry, cell-surface biotinylation and gene array analysis, we report the presence of five additional thiol isomerases in human and mouse platelets and megakaryocytes, namely; ERp57, ERp72, ERp44, ERp29 and TMX3. ERp72, ERp57, ERp44 and ERp29 are released by platelets and relocate to the cell surface following platelet activation. The transmembrane thiol isomerase TMX3 was also detected on the platelet surface but does not increase following activation. Extracellular PDI is also implicated in the regulation of coagulation by the modulation of tissue factor activity. ERp57 was identified within platelet-derived microparticle fractions, suggesting that ERp57 may also be involved in the regulation of coagulation as well as platelet function. These data collectively implicate the expanding family of platelet-surface thiol isomerases in the regulation of haemostasis.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council</p>
<p class="publication"><span class="title">British journal of haematology</span>&nbsp;<span class="year">2010</span>;<span class="volume">148</span>;<span class="number">4</span>;<span class="pagerange">627-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19995400">19995400</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2141.2009.07994.x">10.1111/j.1365-2141.2009.07994.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cooking with GAS.</h4>
<p class="authors"><span class="author">Holden M</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">249</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20234375" rel="external">20234375</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2342" rel="external">10.1038/nrmicro2342</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of a recently emerged, highly transmissible, multi-antibiotic- and antiseptic-resistant variant of methicillin-resistant Staphylococcus aureus, sequence type 239 (TW).</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Lindsay JA</span>, <span class="author">Corton C</span>, <span class="author">Quail MA</span>, <span class="author">Cockfield JD</span>, <span class="author">Pathak S</span>, <span class="author">Batra R</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span> and <span class="author">Edgeworth JD</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6D%68%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#104;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The 3.1-Mb genome of an outbreak methicillin-resistant Staphylococcus aureus (MRSA) strain (TW20) contains evidence of recently acquired DNA, including two large regions (635 kb and 127 kb). The strain is resistant to a wide range of antibiotics, antiseptics, and heavy metals due to resistance genes encoded on mobile genetic elements and also mutations in housekeeping genes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">3</span>;<span class="pagerange">888-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19948800">19948800</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2812470">2812470</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01255-09">10.1128/JB.01255-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Emx2 and early hair cell development in the mouse inner ear.</h4>
<p class="authors"><span class="author">Holley M</span>, <span class="author">Rhodes C</span>, <span class="author">Kneebone A</span>, <span class="author">Herde MK</span>, <span class="author">Fleming M</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Department of Biomedical Science, Addison Building, Western Bank, Sheffield S10 2TN, UK. <a href="mailto:%6D.%63.%68%6F%6C%6C%65%79%40%73%68%65%66%66%69%65%6C%64.%61%63.%75%6B">&#109;&#46;&#99;&#46;&#104;&#111;&#108;&#108;&#101;&#121;&#64;&#115;&#104;&#101;&#102;&#102;&#105;&#101;&#108;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Emx2 is a homeodomain protein that plays a critical role in inner ear development. Homozygous null mice die at birth with a range of defects in the CNS, renal system and skeleton. The cochlea is shorter than normal with about 60% fewer auditory hair cells. It appears to lack outer hair cells and some supporting cells are either absent or fail to differentiate. Many of the hair cells differentiate in pairs and although their hair bundles develop normally their planar cell polarity is compromised. Measurements of cell polarity suggest that classic planar cell polarity molecules are not directly influenced by Emx2 and that polarity is compromised by developmental defects in the sensory precursor population or by defects in epithelial cues for cell alignment. Planar cell polarity is normal in the vestibular epithelia although polarity reversal across the striola is absent in both the utricular and saccular maculae. In contrast, cochlear hair cell polarity is disorganized. The expression domain for Bmp4 is expanded and Fgfr1 and Prox1 are expressed in fewer cells in the cochlear sensory epithelium of Emx2 null mice. We conclude that Emx2 regulates early developmental events that balance cell proliferation and differentiation in the sensory precursor population.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">340</span>;<span class="number">2</span>;<span class="pagerange">547-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20152827">20152827</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2877772">2877772</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2010.02.004">10.1016/j.ydbio.2010.02.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput bacterial SNP typing identifies distinct clusters of Salmonella Typhi causing typhoid in Nepalese children.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Baker S</span>, <span class="author">Dongol S</span>, <span class="author">Basnyat B</span>, <span class="author">Adhikari N</span>, <span class="author">Thorson S</span>, <span class="author">Pulickal AS</span>, <span class="author">Song Y</span>, <span class="author">Parkhill J</span>, <span class="author">Farrar JJ</span>, <span class="author">Murdoch DR</span>, <span class="author">Kelly DF</span>, <span class="author">Pollard AJ</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6B%68%6F%6C%74%40%75%6E%69%6D%65%6C%62.%65%64%75.%61%75">&#107;&#104;&#111;&#108;&#116;&#64;&#117;&#110;&#105;&#109;&#101;&#108;&#98;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract"><strong>Background</strong>: Salmonella Typhi (S. Typhi) causes typhoid fever, which remains an important public health issue in many developing countries. Kathmandu, the capital of Nepal, is an area of high incidence and the pediatric population appears to be at high risk of exposure and infection.</p><p class="abstract"><strong>Methods</strong>: We recently defined the population structure of S. Typhi, using new sequencing technologies to identify nearly 2,000 single nucleotide polymorphisms (SNPs) that can be used as unequivocal phylogenetic markers. Here we have used the GoldenGate (Illumina) platform to simultaneously type 1,500 of these SNPs in 62 S. Typhi isolates causing severe typhoid in children admitted to Patan Hospital in Kathmandu.</p><p class="abstract"><strong>Results</strong>: Eight distinct S. Typhi haplotypes were identified during the 20-month study period, with 68% of isolates belonging to a subclone of the previously defined H58 S. Typhi. This subclone was closely associated with resistance to nalidixic acid, with all isolates from this group demonstrating a resistant phenotype and harbouring the same resistance-associated SNP in GyrA (Phe83). A secondary clone, comprising 19% of isolates, was observed only during the second half of the study.</p><p class="abstract"><strong>Conclusions</strong>: Our data demonstrate the utility of SNP typing for monitoring bacterial populations over a defined period in a single endemic setting. We provide evidence for genotype introduction and define a nalidixic acid resistant subclone of S. Typhi, which appears to be the dominant cause of severe pediatric typhoid in Kathmandu during the study period.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC infectious diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">10</span>;<span class="pagerange">144</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20509974" rel="external">20509974</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2897797" rel="external">2897797</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2334-10-144" rel="external">10.1186/1471-2334-10-144</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Disease-associated XMRV sequences are consistent with laboratory contamination.</h4>
<p class="authors"><span class="author">Hu&eacute; S</span>, <span class="author">Gray ER</span>, <span class="author">Gall A</span>, <span class="author">Katzourakis A</span>, <span class="author">Tan CP</span>, <span class="author">Houldcroft CJ</span>, <span class="author">McLaren S</span>, <span class="author">Pillay D</span>, <span class="author">Futreal A</span>, <span class="author">Garson JA</span>, <span class="author">Pybus OG</span>, <span class="author">Kellam P</span> and <span class="author">Towers GJ</span></p>
<p class="affiliation">MRC Centre for Medical Molecular Virology, Division of Infection and Immunity, University College London, 46 Cleveland St, London W1T 4JF, UK.</p><p class="abstract"><strong>Background</strong>: Xenotropic murine leukaemia viruses (MLV-X) are endogenous gammaretroviruses that infect cells from many species, including humans. Xenotropic murine leukaemia virus-related virus (XMRV) is a retrovirus that has been the subject of intense debate since its detection in samples from humans with prostate cancer (PC) and chronic fatigue syndrome (CFS). Controversy has arisen from the failure of some studies to detect XMRV in PC or CFS patients and from inconsistent detection of XMRV in healthy controls.</p><p class="abstract"><strong>Results</strong>: Here we demonstrate that Taqman PCR primers previously described as XMRV-specific can amplify common murine endogenous viral sequences from mouse suggesting that mouse DNA can contaminate patient samples and confound specific XMRV detection. To consider the provenance of XMRV we sequenced XMRV from the cell line 22Rv1, which is infected with an MLV-X that is indistinguishable from patient derived XMRV. Bayesian phylogenies clearly show that XMRV sequences reportedly derived from unlinked patients form a monophyletic clade with interspersed 22Rv1 clones (posterior probability &gt;0.99). The cell line-derived sequences are ancestral to the patient-derived sequences (posterior probability &gt;0.99). Furthermore, pol sequences apparently amplified from PC patient material (VP29 and VP184) are recombinants of XMRV and Moloney MLV (MoMLV) a virus with an envelope that lacks tropism for human cells. Considering the diversity of XMRV we show that the mean pairwise genetic distance among env and pol 22Rv1-derived sequences exceeds that of patient-associated sequences (Wilcoxon rank sum test: p = 0.005 and p &lt; 0.001 for pol and env, respectively). Thus XMRV sequences acquire diversity in a cell line but not in patient samples. These observations are difficult to reconcile with the hypothesis that published XMRV sequences are related by a process of infectious transmission.</p><p class="abstract"><strong>Conclusions</strong>: We provide several independent lines of evidence that XMRV detected by sensitive PCR methods in patient samples is the likely result of PCR contamination with mouse DNA and that the described clones of XMRV arose from the tumour cell line 22Rv1, which was probably infected with XMRV during xenografting in mice. We propose that XMRV might not be a genuine human pathogen.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801172</span>, <span class="grant">G0801172(87743)</span>, <span class="grant">G9721629</span>; Wellcome Trust: <span class="grant">090940</span>, <span class="grant">WT076608</span>, <span class="grant">WT090940</span></p>
<p class="publication"><span class="title">Retrovirology</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">111</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21171979">21171979</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3018392">3018392</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1742-4690-7-111">10.1186/1742-4690-7-111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interleukin-8 mediates resistance to antiangiogenic agent sunitinib in renal cell carcinoma.</h4>
<p class="authors"><span class="author">Huang D</span>, <span class="author">Ding Y</span>, <span class="author">Zhou M</span>, <span class="author">Rini BI</span>, <span class="author">Petillo D</span>, <span class="author">Qian CN</span>, <span class="author">Kahnoski R</span>, <span class="author">Futreal PA</span>, <span class="author">Furge KA</span> and <span class="author">Teh BT</span></p>
<p class="affiliation">Laboratory of Cancer Genetics, Laboratory of Computational Biology, Van Andel Research Institute, Grand Rapids, Michigan 49503, USA.</p><p class="abstract">The broad spectrum kinase inhibitor sunitinib is a first-line therapy for advanced clear cell renal cell carcinoma (ccRCC), a deadly form of kidney cancer. Unfortunately, most patients develop sunitinib resistance and progressive disease after about 1 year of treatment. In this study, we evaluated the mechanisms of resistance to sunitinib to identify the potential tactics to overcome it. Xenograft models were generated that mimicked clinical resistance to sunitinib. Higher microvessel density was found in sunitinib-resistant tumors, indicating that an escape from antiangiogenesis occurred. Notably, escape coincided with increased secretion of interleukin-8 (IL-8) from tumors into the plasma, and coadministration of an IL-8 neutralizing antibody resensitized tumors to sunitinib treatment. In patients who were refractory to sunitinib treatment, IL-8 expression was elevated in ccRCC tumors, supporting the concept that IL-8 levels might predict clinical response to sunitinib. Our results reveal IL-8 as an important contributor to sunitinib resistance in ccRCC and a candidate therapeutic target to reverse acquired or intrinsic resistance to sunitinib in this malignancy.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2010</span>;<span class="volume">70</span>;<span class="number">3</span>;<span class="pagerange">1063-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20103651">20103651</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3719378">3719378</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-09-3965">10.1158/0008-5472.CAN-09-3965</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterising and predicting haploinsufficiency in the human genome.</h4>
<p class="authors"><span class="author">Huang N</span>, <span class="author">Lee I</span>, <span class="author">Marcotte EM</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p><p class="abstract">Haploinsufficiency, wherein a single functional copy of a gene is insufficient to maintain normal function, is a major cause of dominant disease. Human disease studies have identified several hundred haploinsufficient (HI) genes. We have compiled a map of 1,079 haplosufficient (HS) genes by systematic identification of genes unambiguously and repeatedly compromised by copy number variation among 8,458 apparently healthy individuals and contrasted the genomic, evolutionary, functional, and network properties between these HS genes and known HI genes. We found that HI genes are typically longer and have more conserved coding sequences and promoters than HS genes. HI genes exhibit higher levels of expression during early development and greater tissue specificity. Moreover, within a probabilistic human functional interaction network HI genes have more interaction partners and greater network proximity to other known HI genes. We built a predictive model on the basis of these differences and annotated 12,443 genes with their predicted probability of being haploinsufficient. We validated these predictions of haploinsufficiency by demonstrating that genes with a high predicted probability of exhibiting haploinsufficiency are enriched among genes implicated in human dominant diseases and among genes causing abnormal phenotypes in heterozygous knockout mice. We have transformed these gene-based haploinsufficiency predictions into haploinsufficiency scores for genic deletions, which we demonstrate to better discriminate between pathogenic and benign deletions than consideration of the deletion size or numbers of genes deleted. These robust predictions of haploinsufficiency support clinical interpretation of novel loss-of-function variants and prioritization of variants and genes for follow-up studies.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM067779</span>; Wellcome Trust: <span class="grant">077014/Z/05/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">e1001154</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20976243">20976243</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2954820">2954820</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001154">10.1371/journal.pgen.1001154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a novel wood mouse virus related to murid herpesvirus 4.</h4>
<p class="authors"><span class="author">Hughes DJ</span>, <span class="author">Kipar A</span>, <span class="author">Milligan SG</span>, <span class="author">Cunningham C</span>, <span class="author">Sanders M</span>, <span class="author">Quail MA</span>, <span class="author">Rajandream MA</span>, <span class="author">Efstathiou S</span>, <span class="author">Bowden RJ</span>, <span class="author">Chastel C</span>, <span class="author">Bennett M</span>, <span class="author">Sample JT</span>, <span class="author">Barrell B</span>, <span class="author">Davison AJ</span> and <span class="author">Stewart JP</span></p>
<p class="affiliation">School of Infection and Host Defence, University of Liverpool, Liverpool L69 3GA, UK.</p><p class="abstract">Two novel gammaherpesviruses were isolated, one from a field vole (Microtus agrestis) and the other from wood mice (Apodemus sylvaticus). The genome of the latter, designated wood mouse herpesvirus (WMHV), was completely sequenced. WMHV had the same genome structure and predicted gene content as murid herpesvirus 4 (MuHV4; murine gammaherpesvirus 68). Overall nucleotide sequence identity between WMHV and MuHV4 was 85 % and most of the 10 kb region at the left end of the unique region was particularly highly conserved, especially the viral tRNA-like sequences and the coding regions of genes M1 and M4. The partial sequence (71 913 bp) of another gammaherpesvirus, Brest herpesvirus (BRHV), which was isolated ostensibly from a white-toothed shrew (Crocidura russula), was also determined. The BRHV sequence was 99.2 % identical to the corresponding portion of the WMHV genome. Thus, WMHV and BRHV appeared to be strains of a new virus species. Biological characterization of WMHV indicated that it grew with similar kinetics to MuHV4 in cell culture. The pathogenesis of WMHV in wood mice was also extremely similar to that of MuHV4, except for the absence of inducible bronchus-associated lymphoid tissue at day 14 post-infection and a higher load of latently infected cells at 21 days post-infection.</p>
<p class="grants">Funded by: Medical Research Council; NCI NIH HHS: <span class="grant">CA090208</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of general virology</span>&nbsp;<span class="year">2010</span>;<span class="volume">91</span>;<span class="number">Pt 4</span>;<span class="pagerange">867-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19940063" rel="external">19940063</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2888160" rel="external">2888160</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fvir.0.017327-0" rel="external">10.1099/vir.0.017327-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Experimental evolution, genetic analysis and genome re-sequencing reveal the mutation conferring artemisinin resistance in an isogenic lineage of malaria parasites.</h4>
<p class="authors"><span class="author">Hunt P</span>, <span class="author">Martinelli A</span>, <span class="author">Modrzynska K</span>, <span class="author">Borges S</span>, <span class="author">Creasey A</span>, <span class="author">Rodrigues L</span>, <span class="author">Beraldi D</span>, <span class="author">Loewe L</span>, <span class="author">Fawcett R</span>, <span class="author">Kumar S</span>, <span class="author">Thomson M</span>, <span class="author">Trivedi U</span>, <span class="author">Otto TD</span>, <span class="author">Pain A</span>, <span class="author">Blaxter M</span> and <span class="author">Cravo P</span></p>
<p class="affiliation">Institute for Immunology and Infection Research, School of Biological Sciences, University of Edinburgh, Edinburgh, UK. <a href="mailto:%50%61%75%6C.%48%75%6E%74%40%65%64.%61%63.%75%6B">&#80;&#97;&#117;&#108;&#46;&#72;&#117;&#110;&#116;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Classical and quantitative linkage analyses of genetic crosses have traditionally been used to map genes of interest, such as those conferring chloroquine or quinine resistance in malaria parasites. Next-generation sequencing technologies now present the possibility of determining genome-wide genetic variation at single base-pair resolution. Here, we combine in vivo experimental evolution, a rapid genetic strategy and whole genome re-sequencing to identify the precise genetic basis of artemisinin resistance in a lineage of the rodent malaria parasite, Plasmodium chabaudi. Such genetic markers will further the investigation of resistance and its control in natural infections of the human malaria, P. falciparum.</p><p class="abstract"><strong>Results</strong>: A lineage of isogenic in vivo drug-selected mutant P. chabaudi parasites was investigated. By measuring the artemisinin responses of these clones, the appearance of an in vivo artemisinin resistance phenotype within the lineage was defined. The underlying genetic locus was mapped to a region of chromosome 2 by Linkage Group Selection in two different genetic crosses. Whole-genome deep coverage short-read re-sequencing (Illumina Solexa) defined the point mutations, insertions, deletions and copy-number variations arising in the lineage. Eight point mutations arise within the mutant lineage, only one of which appears on chromosome 2. This missense mutation arises contemporaneously with artemisinin resistance and maps to a gene encoding a de-ubiquitinating enzyme.</p><p class="abstract"><strong>Conclusions</strong>: This integrated approach facilitates the rapid identification of mutations conferring selectable phenotypes, without prior knowledge of biological and molecular mechanisms. For malaria, this model can identify candidate genes before resistant parasites are commonly observed in natural human malaria populations.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/D019621/1</span>; Medical Research Council: <span class="grant">G0400476</span>, <span class="grant">G0900740</span>; Wellcome Trust: <span class="grant">082611/Z/07/Z</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">499</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20846421">20846421</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2996995">2996995</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-499">10.1186/1471-2164-11-499</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic analysis of human protein complexes identifies chromosome segregation proteins.</h4>
<p class="authors"><span class="author">Hutchins JR</span>, <span class="author">Toyoda Y</span>, <span class="author">Hegemann B</span>, <span class="author">Poser I</span>, <span class="author">H&eacute;rich&eacute; JK</span>, <span class="author">Sykora MM</span>, <span class="author">Augsburg M</span>, <span class="author">Hudecz O</span>, <span class="author">Buschhorn BA</span>, <span class="author">Bulkescher J</span>, <span class="author">Conrad C</span>, <span class="author">Comartin D</span>, <span class="author">Schleiffer A</span>, <span class="author">Sarov M</span>, <span class="author">Pozniakovsky A</span>, <span class="author">Slabicki MM</span>, <span class="author">Schloissnig S</span>, <span class="author">Steinmacher I</span>, <span class="author">Leuschner M</span>, <span class="author">Ssykor A</span>, <span class="author">Lawo S</span>, <span class="author">Pelletier L</span>, <span class="author">Stark H</span>, <span class="author">Nasmyth K</span>, <span class="author">Ellenberg J</span>, <span class="author">Durbin R</span>, <span class="author">Buchholz F</span>, <span class="author">Mechtler K</span>, <span class="author">Hyman AA</span> and <span class="author">Peters JM</span></p>
<p class="affiliation">Research Institute of Molecular Pathology (IMP), Dr. Bohr-Gasse 7, A-1030 Vienna, Austria.</p><p class="abstract">Chromosome segregation and cell division are essential, highly ordered processes that depend on numerous protein complexes. Results from recent RNA interference screens indicate that the identity and composition of these protein complexes is incompletely understood. Using gene tagging on bacterial artificial chromosomes, protein localization, and tandem-affinity purification-mass spectrometry, the MitoCheck consortium has analyzed about 100 human protein complexes, many of which had not or had only incompletely been characterized. This work has led to the discovery of previously unknown, evolutionarily conserved subunits of the anaphase-promoting complex and the gamma-tubulin ring complex--large complexes that are essential for spindle assembly and chromosome segregation. The approaches we describe here are generally applicable to high-throughput follow-up analyses of phenotypic screens in mammalian cells.</p>
<p class="grants">Funded by: Austrian Science Fund FWF: <span class="grant">F 3407-B03</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">328</span>;<span class="number">5978</span>;<span class="pagerange">593-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20360068">20360068</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2989461">2989461</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1181348">10.1126/science.1181348</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epilepsy and mental retardation limited to females with PCDH19 mutations can present de novo or in single generation families.</h4>
<p class="authors"><span class="author">Hynes K</span>, <span class="author">Tarpey P</span>, <span class="author">Dibbens LM</span>, <span class="author">Bayly MA</span>, <span class="author">Berkovic SF</span>, <span class="author">Smith R</span>, <span class="author">Raisi ZA</span>, <span class="author">Turner SJ</span>, <span class="author">Brown NJ</span>, <span class="author">Desai TD</span>, <span class="author">Haan E</span>, <span class="author">Turner G</span>, <span class="author">Christodoulou J</span>, <span class="author">Leonard H</span>, <span class="author">Gill D</span>, <span class="author">Stratton MR</span>, <span class="author">Gecz J</span> and <span class="author">Scheffer IE</span></p>
<p class="affiliation">SA Pathology, Women&#39;s and Children&#39;s Hospital, 72 King William Road, North Adelaide, SA 5006, Australia.</p><p class="abstract"><strong>Background</strong>: Epilepsy and mental retardation limited to females (EFMR) is an intriguing X-linked disorder affecting heterozygous females and sparing hemizygous males. Mutations in the protocadherin 19 (PCDH19) gene have been identified in seven unrelated families with EFMR.</p><p class="abstract">Here, we assessed the frequency of PCDH19 mutations in individuals with clinical features which overlap those of EFMR. We analysed 185 females from three cohorts: 42 with Rett syndrome who were negative for MECP2 and CDKL5 mutations, 57 with autism spectrum disorders, and 86 with epilepsy with or without intellectual disability. No mutations were identified in the Rett syndrome and autism spectrum disorders cohorts suggesting that despite sharing similar clinical characteristics with EFMR, PCDH19 mutations are not generally associated with these disorders. Among the 86 females with epilepsy (of whom 51 had seizure onset before 3 years), with or without intellectual disability, we identified two (2.3%) missense changes. One (c.1671C--&gt;G, p.N557K), reported previously without clinical data, was found in two affected sisters, the first EFMR family without a multigenerational family history of affected females. The second, reported here, is a novel de novo missense change identified in a sporadic female. The change, p.S276P, is predicted to result in functional disturbance of PCDH19 as it affects a highly conserved residue adjacent to the adhesion interface of EC3 of PCDH19.</p><p class="abstract"><strong>Conclusions</strong>: This de novo PCDH19 mutation in a sporadic female highlights that mutational analysis should be considered in isolated instances of girls with infantile onset seizures and developmental delay, in addition to those with the characteristic family history of EFMR.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">47</span>;<span class="number">3</span>;<span class="pagerange">211-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19752159">19752159</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2009.068817">10.1136/jmg.2009.068817</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Four novel Loci (19q13, 6q24, 12q24, and 5q14) influence the microcirculation in vivo.</h4>
<p class="authors"><span class="author">Ikram MK</span>, <span class="author">Sim X</span>, <span class="author">Xueling S</span>, <span class="author">Jensen RA</span>, <span class="author">Cotch MF</span>, <span class="author">Hewitt AW</span>, <span class="author">Ikram MA</span>, <span class="author">Wang JJ</span>, <span class="author">Klein R</span>, <span class="author">Klein BE</span>, <span class="author">Breteler MM</span>, <span class="author">Cheung N</span>, <span class="author">Liew G</span>, <span class="author">Mitchell P</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Rivadeneira F</span>, <span class="author">Hofman A</span>, <span class="author">de Jong PT</span>, <span class="author">van Duijn CM</span>, <span class="author">Kao L</span>, <span class="author">Cheng CY</span>, <span class="author">Smith AV</span>, <span class="author">Glazer NL</span>, <span class="author">Lumley T</span>, <span class="author">McKnight B</span>, <span class="author">Psaty BM</span>, <span class="author">Jonasson F</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Aspelund T</span>, <span class="author">Global BPgen Consortium</span>, <span class="author">Harris TB</span>, <span class="author">Launer LJ</span>, <span class="author">Taylor KD</span>, <span class="author">Li X</span>, <span class="author">Iyengar SK</span>, <span class="author">Xi Q</span>, <span class="author">Sivakumaran TA</span>, <span class="author">Mackey DA</span>, <span class="author">Macgregor S</span>, <span class="author">Martin NG</span>, <span class="author">Young TL</span>, <span class="author">Bis JC</span>, <span class="author">Wiggins KL</span>, <span class="author">Heckbert SR</span>, <span class="author">Hammond CJ</span>, <span class="author">Andrew T</span>, <span class="author">Fahy S</span>, <span class="author">Attia J</span>, <span class="author">Holliday EG</span>, <span class="author">Scott RJ</span>, <span class="author">Islam FM</span>, <span class="author">Rotter JI</span>, <span class="author">McAuley AK</span>, <span class="author">Boerwinkle E</span>, <span class="author">Tai ES</span>, <span class="author">Gudnason V</span>, <span class="author">Siscovick DS</span>, <span class="author">Vingerling JR</span> and <span class="author">Wong TY</span></p>
<p class="affiliation">Department of Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">There is increasing evidence that the microcirculation plays an important role in the pathogenesis of cardiovascular diseases. Changes in retinal vascular caliber reflect early microvascular disease and predict incident cardiovascular events. We performed a genome-wide association study to identify genetic variants associated with retinal vascular caliber. We analyzed data from four population-based discovery cohorts with 15,358 unrelated Caucasian individuals, who are members of the Cohort for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium, and replicated findings in four independent Caucasian cohorts (n &#x200A;= &#x200A;6,652). All participants had retinal photography and retinal arteriolar and venular caliber measured from computer software. In the discovery cohorts, 179 single nucleotide polymorphisms (SNP) spread across five loci were significantly associated (p&lt;5.0&times;10(-8)) with retinal venular caliber, but none showed association with arteriolar caliber. Collectively, these five loci explain 1.0%-3.2% of the variation in retinal venular caliber. Four out of these five loci were confirmed in independent replication samples. In the combined analyses, the top SNPs at each locus were: rs2287921 (19q13; p&#x200A; =&#x200A; 1.61&times;10(-25), within the RASIP1 locus), rs225717 (6q24; p&#x200A;=&#x200A;1.25&times;10(-16), adjacent to the VTA1 and NMBR loci), rs10774625 (12q24; p &#x200A;=&#x200A; 2.15&times;10(-13), in the region of ATXN2,SH2B3 and PTPN11 loci), and rs17421627 (5q14; p&#x200A;=&#x200A;7.32&times;10(-16), adjacent to the MEF2C locus). In two independent samples, locus 12q24 was also associated with coronary heart disease and hypertension. Our population-based genome-wide association study demonstrates four novel loci associated with retinal venular caliber, an endophenotype of the microcirculation associated with clinical cardiovascular disease. These data provide further insights into the contribution and biological mechanisms of microcirculatory changes that underlie cardiovascular disease.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_UP_A100_1003</span>; NCRR NIH HHS: <span class="grant">M01RR00069</span>, <span class="grant">UL1RR025005</span>; NEI NIH HHS: <span class="grant">R01 EY018246</span>, <span class="grant">Z01 EY000401-06</span>, <span class="grant">Z01 EY000401-07</span>, <span class="grant">Z01 EY000426-04</span>, <span class="grant">Z01 EY000426-05</span>, <span class="grant">Z01EY000401</span>, <span class="grant">Z01EY000426</span>, <span class="grant">Z99 EY999999</span>, <span class="grant">ZIA EY000401-08</span>, <span class="grant">ZIA EY000401-09</span>, <span class="grant">ZIA EY000401-10</span>, <span class="grant">ZIA EY000403-09</span>, <span class="grant">ZIA EY000403-10</span>, <span class="grant">ZIA EY000426-06</span>, <span class="grant">ZIA EY000426-07</span>, <span class="grant">ZIA EY000426-08</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">N01 HC-15103</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N01HC-55222</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01HL087641</span>, <span class="grant">T32HL007902</span>, <span class="grant">U01 HL080295</span>; NIA NIH HHS: <span class="grant">N01-AG-12100</span>, <span class="grant">Z01AG007380</span>; NIDDK NIH HHS: <span class="grant">DK063491</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">e1001184</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21060863">21060863</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2965750">2965750</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001184">10.1371/journal.pgen.1001184</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide perspective of genetic variation in human metabolism.</h4>
<p class="authors"><span class="author">Illig T</span>, <span class="author">Gieger C</span>, <span class="author">Zhai G</span>, <span class="author">R&ouml;misch-Margl W</span>, <span class="author">Wang-Sattler R</span>, <span class="author">Prehn C</span>, <span class="author">Altmaier E</span>, <span class="author">Kastenm&uuml;ller G</span>, <span class="author">Kato BS</span>, <span class="author">Mewes HW</span>, <span class="author">Meitinger T</span>, <span class="author">de Angelis MH</span>, <span class="author">Kronenberg F</span>, <span class="author">Soranzo N</span>, <span class="author">Wichmann HE</span>, <span class="author">Spector TD</span>, <span class="author">Adamski J</span> and <span class="author">Suhre K</span></p>
<p class="affiliation">Institute of Epidemiology, Helmholtz Zentrum M&uuml;nchen, German Research Center for Environmental Health, Neuherberg, Germany.</p><p class="abstract">Serum metabolite concentrations provide a direct readout of biological processes in the human body, and they are associated with disorders such as cardiovascular and metabolic diseases. We present a genome-wide association study (GWAS) of 163 metabolic traits measured in human blood from 1,809 participants from the KORA population, with replication in 422 participants of the TwinsUK cohort. For eight out of nine replicated loci (FADS1, ELOVL2, ACADS, ACADM, ACADL, SPTLC3, ETFDH and SLC16A9), the genetic variant is located in or near genes encoding enzymes or solute carriers whose functions match the associating metabolic traits. In our study, the use of metabolite concentration ratios as proxies for enzymatic reaction rates reduced the variance and yielded robust statistical associations with P values ranging from 3 x 10(-24) to 6.5 x 10(-179). These loci explained 5.6%-36.3% of the observed variance in metabolite concentrations. For several loci, associations with clinically relevant parameters have been reported previously.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>; Wellcome Trust: <span class="grant">091746</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">2</span>;<span class="pagerange">137-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20037589">20037589</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3773904">3773904</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.507">10.1038/ng.507</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Orphan CpG islands identify numerous conserved promoters in the mammalian genome.</h4>
<p class="authors"><span class="author">Illingworth RS</span>, <span class="author">Gruenewald-Schneider U</span>, <span class="author">Webb S</span>, <span class="author">Kerr AR</span>, <span class="author">James KD</span>, <span class="author">Turner DJ</span>, <span class="author">Smith C</span>, <span class="author">Harrison DJ</span>, <span class="author">Andrews R</span> and <span class="author">Bird AP</span></p>
<p class="affiliation">Wellcome Trust Centre for Cell Biology, University of Edinburgh, Edinburgh, United Kingdom.</p><p class="abstract">CpG islands (CGIs) are vertebrate genomic landmarks that encompass the promoters of most genes and often lack DNA methylation. Querying their apparent importance, the number of CGIs is reported to vary widely in different species and many do not co-localise with annotated promoters. We set out to quantify the number of CGIs in mouse and human genomes using CXXC Affinity Purification plus deep sequencing (CAP-seq). We also asked whether CGIs not associated with annotated transcripts share properties with those at known promoters. We found that, contrary to previous estimates, CGI abundance in humans and mice is very similar and many are at conserved locations relative to genes. In each species CpG density correlates positively with the degree of H3K4 trimethylation, supporting the hypothesis that these two properties are mechanistically interdependent. Approximately half of mammalian CGIs (&gt;10,000) are &quot;orphans&quot; that are not associated with annotated promoters. Many orphan CGIs show evidence of transcriptional initiation and dynamic expression during development. Unlike CGIs at known promoters, orphan CGIs are frequently subject to DNA methylation during development, and this is accompanied by loss of their active promoter features. In colorectal tumors, however, orphan CGIs are not preferentially methylated, suggesting that cancer does not recapitulate a developmental program. Human and mouse genomes have similar numbers of CGIs, over half of which are remote from known promoters. Orphan CGIs nevertheless have the characteristics of functional promoters, though they are much more likely than promoter CGIs to become methylated during development and hence lose these properties. The data indicate that orphan CGIs correspond to previously undetected promoters whose transcriptional activity may play a functional role during development.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800026</span>, <span class="grant">G0900627</span>; Wellcome Trust: <span class="grant">077224</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span>;<span class="pagerange">e1001134</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20885785">20885785</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2944787">2944787</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001134">10.1371/journal.pgen.1001134</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A large replication study and meta-analysis in European samples provides further support for association of AHI1 markers with schizophrenia.</h4>
<p class="authors"><span class="author">Ingason A</span>, <span class="author">Giegling I</span>, <span class="author">Cichon S</span>, <span class="author">Hansen T</span>, <span class="author">Rasmussen HB</span>, <span class="author">Nielsen J</span>, <span class="author">J&uuml;rgens G</span>, <span class="author">Muglia P</span>, <span class="author">Hartmann AM</span>, <span class="author">Strengman E</span>, <span class="author">Vasilescu C</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">Djurovic S</span>, <span class="author">Melle I</span>, <span class="author">Lerer B</span>, <span class="author">M&ouml;ller HJ</span>, <span class="author">Francks C</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Lonnqvist J</span>, <span class="author">Suvisaari J</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Walshe M</span>, <span class="author">Vassos E</span>, <span class="author">Di Forti M</span>, <span class="author">Murray R</span>, <span class="author">Bonetto C</span>, <span class="author">Tosato S</span>, <span class="author">GROUP Investigators</span>, <span class="author">Cantor RM</span>, <span class="author">Rietschel M</span>, <span class="author">Craddock N</span>, <span class="author">Owen MJ</span>, <span class="author">Peltonen L</span>, <span class="author">Andreassen OA</span>, <span class="author">N&ouml;then MM</span>, <span class="author">St Clair D</span>, <span class="author">Ophoff RA</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Collier DA</span>, <span class="author">Werge T</span> and <span class="author">Rujescu D</span></p>
<p class="affiliation">Research Institute of Biological Psychiatry, Copenhagen University Hospital, Roskilde, Denmark. <a href="mailto:%61%6E%64%72%65%73.%69%6E%67%61%73%6F%6E%40%73%68%68.%72%65%67%69%6F%6E%68.%64%6B">&#97;&#110;&#100;&#114;&#101;&#115;&#46;&#105;&#110;&#103;&#97;&#115;&#111;&#110;&#64;&#115;&#104;&#104;&#46;&#114;&#101;&#103;&#105;&#111;&#110;&#104;&#46;&#100;&#107;</a></p><p class="abstract">The Abelson helper integration site 1 (AHI1) gene locus on chromosome 6q23 is among a group of candidate loci for schizophrenia susceptibility that were initially identified by linkage followed by linkage disequilibrium mapping, and subsequent replication of the association in an independent sample. Here, we present results of a replication study of AHI1 locus markers, previously implicated in schizophrenia, in a large European sample (in total 3907 affected and 7429 controls). Furthermore, we perform a meta-analysis of the implicated markers in 4496 affected and 18,920 controls. Both the replication study of new samples and the meta-analysis show evidence for significant overrepresentation of all tested alleles in patients compared with controls (meta-analysis; P = 8.2 x 10(-5)-1.7 x 10(-3), common OR = 1.09-1.11). The region contains two genes, AHI1 and C6orf217, and both genes-as well as the neighbouring phosphodiesterase 7B (PDE7B)-may be considered candidates for involvement in the genetic aetiology of schizophrenia.</p>
<p class="grants">Funded by: Medical Research Council; NIMH NIH HHS: <span class="grant">R01 MH078075</span>; Wellcome Trust: <span class="grant">076113</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">7</span>;<span class="pagerange">1379-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20071346" rel="external">20071346</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2838541" rel="external">2838541</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq009" rel="external">10.1093/hmg/ddq009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detailed physiologic characterization reveals diverse mechanisms for novel genetic Loci regulating glucose and insulin metabolism in humans.</h4>
<p class="authors"><span class="author">Ingelsson E</span>, <span class="author">Langenberg C</span>, <span class="author">Hivert MF</span>, <span class="author">Prokopenko I</span>, <span class="author">Lyssenko V</span>, <span class="author">Dupuis J</span>, <span class="author">M&auml;gi R</span>, <span class="author">Sharp S</span>, <span class="author">Jackson AU</span>, <span class="author">Assimes TL</span>, <span class="author">Shrader P</span>, <span class="author">Knowles JW</span>, <span class="author">Zethelius B</span>, <span class="author">Abbasi FA</span>, <span class="author">Bergman RN</span>, <span class="author">Bergmann A</span>, <span class="author">Berne C</span>, <span class="author">Boehnke M</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bornstein SR</span>, <span class="author">Buchanan TA</span>, <span class="author">Bumpstead SJ</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Chines P</span>, <span class="author">Collins FS</span>, <span class="author">Cooper CC</span>, <span class="author">Dennison EM</span>, <span class="author">Erdos MR</span>, <span class="author">Ferrannini E</span>, <span class="author">Fox CS</span>, <span class="author">Graessler J</span>, <span class="author">Hao K</span>, <span class="author">Isomaa B</span>, <span class="author">Jameson KA</span>, <span class="author">Kovacs P</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Ladenvall C</span>, <span class="author">Mohlke KL</span>, <span class="author">Morken MA</span>, <span class="author">Narisu N</span>, <span class="author">Nathan DM</span>, <span class="author">Pascoe L</span>, <span class="author">Payne F</span>, <span class="author">Petrie JR</span>, <span class="author">Sayer AA</span>, <span class="author">Schwarz PE</span>, <span class="author">Scott LJ</span>, <span class="author">Stringham HM</span>, <span class="author">Stumvoll M</span>, <span class="author">Swift AJ</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Tuomi T</span>, <span class="author">Tuomilehto J</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Valle TT</span>, <span class="author">Williams GH</span>, <span class="author">Lind L</span>, <span class="author">Barroso I</span>, <span class="author">Quertermous T</span>, <span class="author">Walker M</span>, <span class="author">Wareham NJ</span>, <span class="author">Meigs JB</span>, <span class="author">McCarthy MI</span>, <span class="author">Groop L</span>, <span class="author">Watanabe RM</span>, <span class="author">Florez JC</span> and <span class="author">MAGIC investigators</span></p>
<p class="affiliation">Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden. <a href="mailto:%65%72%69%6B.%69%6E%67%65%6C%73%73%6F%6E%40%6B%69.%73%65">&#101;&#114;&#105;&#107;&#46;&#105;&#110;&#103;&#101;&#108;&#115;&#115;&#111;&#110;&#64;&#107;&#105;&#46;&#115;&#101;</a></p><p class="abstract">OBJECTIVE Recent genome-wide association studies have revealed loci associated with glucose and insulin-related traits. We aimed to characterize 19 such loci using detailed measures of insulin processing, secretion, and sensitivity to help elucidate their role in regulation of glucose control, insulin secretion and/or action. RESEARCH DESIGN AND METHODS We investigated associations of loci identified by the Meta-Analyses of Glucose and Insulin-related traits Consortium (MAGIC) with circulating proinsulin, measures of insulin secretion and sensitivity from oral glucose tolerance tests (OGTTs), euglycemic clamps, insulin suppression tests, or frequently sampled intravenous glucose tolerance tests in nondiabetic humans (n = 29,084). RESULTS The glucose-raising allele in MADD was associated with abnormal insulin processing (a dramatic effect on higher proinsulin levels, but no association with insulinogenic index) at extremely persuasive levels of statistical significance (P = 2.1 x 10(-71)). Defects in insulin processing and insulin secretion were seen in glucose-raising allele carriers at TCF7L2, SCL30A8, GIPR, and C2CD4B. Abnormalities in early insulin secretion were suggested in glucose-raising allele carriers at MTNR1B, GCK, FADS1, DGKB, and PROX1 (lower insulinogenic index; no association with proinsulin or insulin sensitivity). Two loci previously associated with fasting insulin (GCKR and IGF1) were associated with OGTT-derived insulin sensitivity indices in a consistent direction. CONCLUSIONS Genetic loci identified through their effect on hyperglycemia and/or hyperinsulinemia demonstrate considerable heterogeneity in associations with measures of insulin processing, secretion, and sensitivity. Our findings emphasize the importance of detailed physiological characterization of such loci for improved understanding of pathways associated with alterations in glucose homeostasis and eventually type 2 diabetes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701863</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U147574213</span>, <span class="grant">MC_U147574239</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NHLBI NIH HHS: <span class="grant">R01 HL087647</span>; NIDDK NIH HHS: <span class="grant">R01 DK029867</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2010</span>;<span class="volume">59</span>;<span class="number">5</span>;<span class="pagerange">1266-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20185807">20185807</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2857908">2857908</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb09-1568">10.2337/db09-1568</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metabonomic, transcriptomic, and genomic variation of a population cohort.</h4>
<p class="authors"><span class="author">Inouye M</span>, <span class="author">Kettunen J</span>, <span class="author">Soininen P</span>, <span class="author">Silander K</span>, <span class="author">Ripatti S</span>, <span class="author">Kumpula LS</span>, <span class="author">H&auml;m&auml;l&auml;inen E</span>, <span class="author">Jousilahti P</span>, <span class="author">Kangas AJ</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">Savolainen MJ</span>, <span class="author">Jula A</span>, <span class="author">Leivisk&auml; J</span>, <span class="author">Palotie A</span>, <span class="author">Salomaa V</span>, <span class="author">Perola M</span>, <span class="author">Ala-Korpela M</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Immunology Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia. <a href="mailto:%69%6E%6F%75%79%65%40%77%65%68%69.%65%64%75.%61%75">&#105;&#110;&#111;&#117;&#121;&#101;&#64;&#119;&#101;&#104;&#105;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Comprehensive characterization of human tissues promises novel insights into the biological architecture of human diseases and traits. We assessed metabonomic, transcriptomic, and genomic variation for a large population-based cohort from the capital region of Finland. Network analyses identified a set of highly correlated genes, the lipid-leukocyte (LL) module, as having a prominent role in over 80 serum metabolites (of 134 measures quantified), including lipoprotein subclasses, lipids, and amino acids. Concurrent association with immune response markers suggested the LL module as a possible link between inflammation, metabolism, and adiposity. Further, genomic variation was used to generate a directed network and infer LL module&#39;s largely reactive nature to metabolites. Finally, gene co-expression in circulating leukocytes was shown to be dependent on serum metabolite concentrations, providing evidence for the hypothesis that the coherence of molecular networks themselves is conditional on environmental factors. These findings show the importance and opportunity of systematic molecular investigation of human population samples. To facilitate and encourage this investigation, the metabonomic, transcriptomic, and genomic data used in this study have been made available as a resource for the research community.</p>
<p class="publication"><span class="title">Molecular systems biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="pagerange">441</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21179014" rel="external">21179014</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3018170" rel="external">3018170</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmsb.2010.93" rel="external">10.1038/msb.2010.93</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An immune response network associated with blood lipid levels.</h4>
<p class="authors"><span class="author">Inouye M</span>, <span class="author">Silander K</span>, <span class="author">Hamalainen E</span>, <span class="author">Salomaa V</span>, <span class="author">Harald K</span>, <span class="author">Jousilahti P</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">Eriksson JG</span>, <span class="author">Saarela J</span>, <span class="author">Ripatti S</span>, <span class="author">Perola M</span>, <span class="author">van Ommen GJ</span>, <span class="author">Taskinen MR</span>, <span class="author">Palotie A</span>, <span class="author">Dermitzakis ET</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom. <a href="mailto:%69%6E%6F%75%79%65%40%77%65%68%69.%65%64%75.%61%75">&#105;&#110;&#111;&#117;&#121;&#101;&#64;&#119;&#101;&#104;&#105;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">While recent scans for genetic variation associated with human disease have been immensely successful in uncovering large numbers of loci, far fewer studies have focused on the underlying pathways of disease pathogenesis. Many loci which are associated with disease and complex phenotypes map to non-coding, regulatory regions of the genome, indicating that modulation of gene transcription plays a key role. Thus, this study generated genome-wide profiles of both genetic and transcriptional variation from the total blood extracts of over 500 randomly-selected, unrelated individuals. Using measurements of blood lipids, key players in the progression of atherosclerosis, three levels of biological information are integrated in order to investigate the interactions between circulating leukocytes and proximal lipid compounds. Pair-wise correlations between gene expression and lipid concentration indicate a prominent role for basophil granulocytes and mast cells, cell types central to powerful allergic and inflammatory responses. Network analysis of gene co-expression showed that the top associations function as part of a single, previously unknown gene module, the Lipid Leukocyte (LL) module. This module replicated in T cells from an independent cohort while also displaying potential tissue specificity. Further, genetic variation driving LL module expression included the single nucleotide polymorphism (SNP) most strongly associated with serum immunoglobulin E (IgE) levels, a key antibody in allergy. Structural Equation Modeling (SEM) indicated that LL module is at least partially reactive to blood lipid levels. Taken together, this study uncovers a gene network linking blood lipids and circulating cell types and offers insight into the hypothesis that the inflammatory response plays a prominent role in metabolism and the potential control of atherogenesis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT089061</span>, <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span>;<span class="pagerange">e1001113</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20844574">20844574</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2936545">2936545</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001113">10.1371/journal.pgen.1001113</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">International network of cancer genome projects.</h4>
<p class="authors"><span class="author">International Cancer Genome Consortium</span>, <span class="author">Hudson TJ</span>, <span class="author">Anderson W</span>, <span class="author">Artez A</span>, <span class="author">Barker AD</span>, <span class="author">Bell C</span>, <span class="author">Bernab&eacute; RR</span>, <span class="author">Bhan MK</span>, <span class="author">Calvo F</span>, <span class="author">Eerola I</span>, <span class="author">Gerhard DS</span>, <span class="author">Guttmacher A</span>, <span class="author">Guyer M</span>, <span class="author">Hemsley FM</span>, <span class="author">Jennings JL</span>, <span class="author">Kerr D</span>, <span class="author">Klatt P</span>, <span class="author">Kolar P</span>, <span class="author">Kusada J</span>, <span class="author">Lane DP</span>, <span class="author">Laplace F</span>, <span class="author">Youyong L</span>, <span class="author">Nettekoven G</span>, <span class="author">Ozenberger B</span>, <span class="author">Peterson J</span>, <span class="author">Rao TS</span>, <span class="author">Remacle J</span>, <span class="author">Schafer AJ</span>, <span class="author">Shibata T</span>, <span class="author">Stratton MR</span>, <span class="author">Vockley JG</span>, <span class="author">Watanabe K</span>, <span class="author">Yang H</span>, <span class="author">Yuen MM</span>, <span class="author">Knoppers BM</span>, <span class="author">Bobrow M</span>, <span class="author">Cambon-Thomsen A</span>, <span class="author">Dressler LG</span>, <span class="author">Dyke SO</span>, <span class="author">Joly Y</span>, <span class="author">Kato K</span>, <span class="author">Kennedy KL</span>, <span class="author">Nicol&aacute;s P</span>, <span class="author">Parker MJ</span>, <span class="author">Rial-Sebbag E</span>, <span class="author">Romeo-Casabona CM</span>, <span class="author">Shaw KM</span>, <span class="author">Wallace S</span>, <span class="author">Wiesner GL</span>, <span class="author">Zeps N</span>, <span class="author">Lichter P</span>, <span class="author">Biankin AV</span>, <span class="author">Chabannon C</span>, <span class="author">Chin L</span>, <span class="author">Cl&eacute;ment B</span>, <span class="author">de Alava E</span>, <span class="author">Degos F</span>, <span class="author">Ferguson ML</span>, <span class="author">Geary P</span>, <span class="author">Hayes DN</span>, <span class="author">Hudson TJ</span>, <span class="author">Johns AL</span>, <span class="author">Kasprzyk A</span>, <span class="author">Nakagawa H</span>, <span class="author">Penny R</span>, <span class="author">Piris MA</span>, <span class="author">Sarin R</span>, <span class="author">Scarpa A</span>, <span class="author">Shibata T</span>, <span class="author">van de Vijver M</span>, <span class="author">Futreal PA</span>, <span class="author">Aburatani H</span>, <span class="author">Bay&eacute;s M</span>, <span class="author">Botwell DD</span>, <span class="author">Campbell PJ</span>, <span class="author">Estivill X</span>, <span class="author">Gerhard DS</span>, <span class="author">Grimmond SM</span>, <span class="author">Gut I</span>, <span class="author">Hirst M</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span>, <span class="author">Majumder P</span>, <span class="author">Marra M</span>, <span class="author">McPherson JD</span>, <span class="author">Nakagawa H</span>, <span class="author">Ning Z</span>, <span class="author">Puente XS</span>, <span class="author">Ruan Y</span>, <span class="author">Shibata T</span>, <span class="author">Stratton MR</span>, <span class="author">Stunnenberg HG</span>, <span class="author">Swerdlow H</span>, <span class="author">Velculescu VE</span>, <span class="author">Wilson RK</span>, <span class="author">Xue HH</span>, <span class="author">Yang L</span>, <span class="author">Spellman PT</span>, <span class="author">Bader GD</span>, <span class="author">Boutros PC</span>, <span class="author">Campbell PJ</span>, <span class="author">Flicek P</span>, <span class="author">Getz G</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Guo G</span>, <span class="author">Haussler D</span>, <span class="author">Heath S</span>, <span class="author">Hubbard TJ</span>, <span class="author">Jiang T</span>, <span class="author">Jones SM</span>, <span class="author">Li Q</span>, <span class="author">L&oacute;pez-Bigas N</span>, <span class="author">Luo R</span>, <span class="author">Muthuswamy L</span>, <span class="author">Ouellette BF</span>, <span class="author">Pearson JV</span>, <span class="author">Puente XS</span>, <span class="author">Quesada V</span>, <span class="author">Raphael BJ</span>, <span class="author">Sander C</span>, <span class="author">Shibata T</span>, <span class="author">Speed TP</span>, <span class="author">Stein LD</span>, <span class="author">Stuart JM</span>, <span class="author">Teague JW</span>, <span class="author">Totoki Y</span>, <span class="author">Tsunoda T</span>, <span class="author">Valencia A</span>, <span class="author">Wheeler DA</span>, <span class="author">Wu H</span>, <span class="author">Zhao S</span>, <span class="author">Zhou G</span>, <span class="author">Stein LD</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Hubbard TJ</span>, <span class="author">Joly Y</span>, <span class="author">Jones SM</span>, <span class="author">Kasprzyk A</span>, <span class="author">Lathrop M</span>, <span class="author">L&oacute;pez-Bigas N</span>, <span class="author">Ouellette BF</span>, <span class="author">Spellman PT</span>, <span class="author">Teague JW</span>, <span class="author">Thomas G</span>, <span class="author">Valencia A</span>, <span class="author">Yoshida T</span>, <span class="author">Kennedy KL</span>, <span class="author">Axton M</span>, <span class="author">Dyke SO</span>, <span class="author">Futreal PA</span>, <span class="author">Gerhard DS</span>, <span class="author">Gunter C</span>, <span class="author">Guyer M</span>, <span class="author">Hudson TJ</span>, <span class="author">McPherson JD</span>, <span class="author">Miller LJ</span>, <span class="author">Ozenberger B</span>, <span class="author">Shaw KM</span>, <span class="author">Kasprzyk A</span>, <span class="author">Stein LD</span>, <span class="author">Zhang J</span>, <span class="author">Haider SA</span>, <span class="author">Wang J</span>, <span class="author">Yung CK</span>, <span class="author">Cros A</span>, <span class="author">Cross A</span>, <span class="author">Liang Y</span>, <span class="author">Gnaneshan S</span>, <span class="author">Guberman J</span>, <span class="author">Hsu J</span>, <span class="author">Bobrow M</span>, <span class="author">Chalmers DR</span>, <span class="author">Hasel KW</span>, <span class="author">Joly Y</span>, <span class="author">Kaan TS</span>, <span class="author">Kennedy KL</span>, <span class="author">Knoppers BM</span>, <span class="author">Lowrance WW</span>, <span class="author">Masui T</span>, <span class="author">Nicol&aacute;s P</span>, <span class="author">Rial-Sebbag E</span>, <span class="author">Rodriguez LL</span>, <span class="author">Vergely C</span>, <span class="author">Yoshida T</span>, <span class="author">Grimmond SM</span>, <span class="author">Biankin AV</span>, <span class="author">Bowtell DD</span>, <span class="author">Cloonan N</span>, <span class="author">deFazio A</span>, <span class="author">Eshleman JR</span>, <span class="author">Etemadmoghadam D</span>, <span class="author">Gardiner BB</span>, <span class="author">Gardiner BA</span>, <span class="author">Kench JG</span>, <span class="author">Scarpa A</span>, <span class="author">Sutherland RL</span>, <span class="author">Tempero MA</span>, <span class="author">Waddell NJ</span>, <span class="author">Wilson PJ</span>, <span class="author">McPherson JD</span>, <span class="author">Gallinger S</span>, <span class="author">Tsao MS</span>, <span class="author">Shaw PA</span>, <span class="author">Petersen GM</span>, <span class="author">Mukhopadhyay D</span>, <span class="author">Chin L</span>, <span class="author">DePinho RA</span>, <span class="author">Thayer S</span>, <span class="author">Muthuswamy L</span>, <span class="author">Shazand K</span>, <span class="author">Beck T</span>, <span class="author">Sam M</span>, <span class="author">Timms L</span>, <span class="author">Ballin V</span>, <span class="author">Lu Y</span>, <span class="author">Ji J</span>, <span class="author">Zhang X</span>, <span class="author">Chen F</span>, <span class="author">Hu X</span>, <span class="author">Zhou G</span>, <span class="author">Yang Q</span>, <span class="author">Tian G</span>, <span class="author">Zhang L</span>, <span class="author">Xing X</span>, <span class="author">Li X</span>, <span class="author">Zhu Z</span>, <span class="author">Yu Y</span>, <span class="author">Yu J</span>, <span class="author">Yang H</span>, <span class="author">Lathrop M</span>, <span class="author">Tost J</span>, <span class="author">Brennan P</span>, <span class="author">Holcatova I</span>, <span class="author">Zaridze D</span>, <span class="author">Brazma A</span>, <span class="author">Egevard L</span>, <span class="author">Prokhortchouk E</span>, <span class="author">Banks RE</span>, <span class="author">Uhl&eacute;n M</span>, <span class="author">Cambon-Thomsen A</span>, <span class="author">Viksna J</span>, <span class="author">Ponten F</span>, <span class="author">Skryabin K</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Birney E</span>, <span class="author">Borg A</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">Caldas C</span>, <span class="author">Foekens JA</span>, <span class="author">Martin S</span>, <span class="author">Reis-Filho JS</span>, <span class="author">Richardson AL</span>, <span class="author">Sotiriou C</span>, <span class="author">Stunnenberg HG</span>, <span class="author">Thoms G</span>, <span class="author">van de Vijver M</span>, <span class="author">van&#39;t Veer L</span>, <span class="author">Calvo F</span>, <span class="author">Birnbaum D</span>, <span class="author">Blanche H</span>, <span class="author">Boucher P</span>, <span class="author">Boyault S</span>, <span class="author">Chabannon C</span>, <span class="author">Gut I</span>, <span class="author">Masson-Jacquemier JD</span>, <span class="author">Lathrop M</span>, <span class="author">Pauport&eacute; I</span>, <span class="author">Pivot X</span>, <span class="author">Vincent-Salomon A</span>, <span class="author">Tabone E</span>, <span class="author">Theillet C</span>, <span class="author">Thomas G</span>, <span class="author">Tost J</span>, <span class="author">Treilleux I</span>, <span class="author">Calvo F</span>, <span class="author">Bioulac-Sage P</span>, <span class="author">Cl&eacute;ment B</span>, <span class="author">Decaens T</span>, <span class="author">Degos F</span>, <span class="author">Franco D</span>, <span class="author">Gut I</span>, <span class="author">Gut M</span>, <span class="author">Heath S</span>, <span class="author">Lathrop M</span>, <span class="author">Samuel D</span>, <span class="author">Thomas G</span>, <span class="author">Zucman-Rossi J</span>, <span class="author">Lichter P</span>, <span class="author">Eils R</span>, <span class="author">Brors B</span>, <span class="author">Korbel JO</span>, <span class="author">Korshunov A</span>, <span class="author">Landgraf P</span>, <span class="author">Lehrach H</span>, <span class="author">Pfister S</span>, <span class="author">Radlwimmer B</span>, <span class="author">Reifenberger G</span>, <span class="author">Taylor MD</span>, <span class="author">von Kalle C</span>, <span class="author">Majumder PP</span>, <span class="author">Sarin R</span>, <span class="author">Rao TS</span>, <span class="author">Bhan MK</span>, <span class="author">Scarpa A</span>, <span class="author">Pederzoli P</span>, <span class="author">Lawlor RA</span>, <span class="author">Delledonne M</span>, <span class="author">Bardelli A</span>, <span class="author">Biankin AV</span>, <span class="author">Grimmond SM</span>, <span class="author">Gress T</span>, <span class="author">Klimstra D</span>, <span class="author">Zamboni G</span>, <span class="author">Shibata T</span>, <span class="author">Nakamura Y</span>, <span class="author">Nakagawa H</span>, <span class="author">Kusada J</span>, <span class="author">Tsunoda T</span>, <span class="author">Miyano S</span>, <span class="author">Aburatani H</span>, <span class="author">Kato K</span>, <span class="author">Fujimoto A</span>, <span class="author">Yoshida T</span>, <span class="author">Campo E</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span>, <span class="author">Estivill X</span>, <span class="author">Guig&oacute; R</span>, <span class="author">de Sanjos&eacute; S</span>, <span class="author">Piris MA</span>, <span class="author">Montserrat E</span>, <span class="author">Gonz&aacute;lez-D&iacute;az M</span>, <span class="author">Puente XS</span>, <span class="author">Jares P</span>, <span class="author">Valencia A</span>, <span class="author">Himmelbauer H</span>, <span class="author">Himmelbaue H</span>, <span class="author">Quesada V</span>, <span class="author">Bea S</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Campbell PJ</span>, <span class="author">Vincent-Salomon A</span>, <span class="author">Richardson AL</span>, <span class="author">Reis-Filho JS</span>, <span class="author">van de Vijver M</span>, <span class="author">Thomas G</span>, <span class="author">Masson-Jacquemier JD</span>, <span class="author">Aparicio S</span>, <span class="author">Borg A</span>, <span class="author">B&oslash;rresen-Dale AL</span>, <span class="author">Caldas C</span>, <span class="author">Foekens JA</span>, <span class="author">Stunnenberg HG</span>, <span class="author">van&#39;t Veer L</span>, <span class="author">Easton DF</span>, <span class="author">Spellman PT</span>, <span class="author">Martin S</span>, <span class="author">Barker AD</span>, <span class="author">Chin L</span>, <span class="author">Collins FS</span>, <span class="author">Compton CC</span>, <span class="author">Ferguson ML</span>, <span class="author">Gerhard DS</span>, <span class="author">Getz G</span>, <span class="author">Gunter C</span>, <span class="author">Guttmacher A</span>, <span class="author">Guyer M</span>, <span class="author">Hayes DN</span>, <span class="author">Lander ES</span>, <span class="author">Ozenberger B</span>, <span class="author">Penny R</span>, <span class="author">Peterson J</span>, <span class="author">Sander C</span>, <span class="author">Shaw KM</span>, <span class="author">Speed TP</span>, <span class="author">Spellman PT</span>, <span class="author">Vockley JG</span>, <span class="author">Wheeler DA</span>, <span class="author">Wilson RK</span>, <span class="author">Hudson TJ</span>, <span class="author">Chin L</span>, <span class="author">Knoppers BM</span>, <span class="author">Lander ES</span>, <span class="author">Lichter P</span>, <span class="author">Stein LD</span>, <span class="author">Stratton MR</span>, <span class="author">Anderson W</span>, <span class="author">Barker AD</span>, <span class="author">Bell C</span>, <span class="author">Bobrow M</span>, <span class="author">Burke W</span>, <span class="author">Collins FS</span>, <span class="author">Compton CC</span>, <span class="author">DePinho RA</span>, <span class="author">Easton DF</span>, <span class="author">Futreal PA</span>, <span class="author">Gerhard DS</span>, <span class="author">Green AR</span>, <span class="author">Guyer M</span>, <span class="author">Hamilton SR</span>, <span class="author">Hubbard TJ</span>, <span class="author">Kallioniemi OP</span>, <span class="author">Kennedy KL</span>, <span class="author">Ley TJ</span>, <span class="author">Liu ET</span>, <span class="author">Lu Y</span>, <span class="author">Majumder P</span>, <span class="author">Marra M</span>, <span class="author">Ozenberger B</span>, <span class="author">Peterson J</span>, <span class="author">Schafer AJ</span>, <span class="author">Spellman PT</span>, <span class="author">Stunnenberg HG</span>, <span class="author">Wainwright BJ</span>, <span class="author">Wilson RK</span> and <span class="author">Yang H</span></p><p class="abstract">The International Cancer Genome Consortium (ICGC) was launched to coordinate large-scale cancer genome studies in tumours from 50 different cancer types and/or subtypes that are of clinical and societal importance across the globe. Systematic studies of more than 25,000 cancer genomes at the genomic, epigenomic and transcriptomic levels will reveal the repertoire of oncogenic mutations, uncover traces of the mutagenic influences, define clinically relevant subtypes for prognosis and therapeutic management, and enable the development of new cancer therapies.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">6613</span>; NCI NIH HHS: <span class="grant">P01 CA117969</span>, <span class="grant">P01 CA117969-04S1</span>, <span class="grant">P01 CA117969-05</span>, <span class="grant">P50 CA102701</span>, <span class="grant">P50 CA102701-08</span>, <span class="grant">P50 CA127003</span>, <span class="grant">P50 CA127003-04</span>, <span class="grant">P50 CA127003-05</span>; NHGRI NIH HHS: <span class="grant">R01 HG001806-02</span>; NIDDK NIH HHS: <span class="grant">K08 DK071329</span>, <span class="grant">K08 DK071329-04</span>, <span class="grant">K08 DK071329-05</span>; Wellcome Trust: <span class="grant">077198</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7291</span>;<span class="pagerange">993-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20393554">20393554</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2902243">2902243</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08987">10.1038/nature08987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrating common and rare genetic variation in diverse human populations.</h4>
<p class="authors"><span class="author">International HapMap 3 Consortium</span>, <span class="author">Altshuler DM</span>, <span class="author">Gibbs RA</span>, <span class="author">Peltonen L</span>, <span class="author">Altshuler DM</span>, <span class="author">Gibbs RA</span>, <span class="author">Peltonen L</span>, <span class="author">Dermitzakis E</span>, <span class="author">Schaffner SF</span>, <span class="author">Yu F</span>, <span class="author">Peltonen L</span>, <span class="author">Dermitzakis E</span>, <span class="author">Bonnen PE</span>, <span class="author">Altshuler DM</span>, <span class="author">Gibbs RA</span>, <span class="author">de Bakker PI</span>, <span class="author">Deloukas P</span>, <span class="author">Gabriel SB</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt S</span>, <span class="author">Inouye M</span>, <span class="author">Jia X</span>, <span class="author">Palotie A</span>, <span class="author">Parkin M</span>, <span class="author">Whittaker P</span>, <span class="author">Yu F</span>, <span class="author">Chang K</span>, <span class="author">Hawes A</span>, <span class="author">Lewis LR</span>, <span class="author">Ren Y</span>, <span class="author">Wheeler D</span>, <span class="author">Gibbs RA</span>, <span class="author">Muzny DM</span>, <span class="author">Barnes C</span>, <span class="author">Darvishi K</span>, <span class="author">Hurles M</span>, <span class="author">Korn JM</span>, <span class="author">Kristiansson K</span>, <span class="author">Lee C</span>, <span class="author">McCarrol SA</span>, <span class="author">Nemesh J</span>, <span class="author">Dermitzakis E</span>, <span class="author">Keinan A</span>, <span class="author">Montgomery SB</span>, <span class="author">Pollack S</span>, <span class="author">Price AL</span>, <span class="author">Soranzo N</span>, <span class="author">Bonnen PE</span>, <span class="author">Gibbs RA</span>, <span class="author">Gonzaga-Jauregui C</span>, <span class="author">Keinan A</span>, <span class="author">Price AL</span>, <span class="author">Yu F</span>, <span class="author">Anttila V</span>, <span class="author">Brodeur W</span>, <span class="author">Daly MJ</span>, <span class="author">Leslie S</span>, <span class="author">McVean G</span>, <span class="author">Moutsianas L</span>, <span class="author">Nguyen H</span>, <span class="author">Schaffner SF</span>, <span class="author">Zhang Q</span>, <span class="author">Ghori MJ</span>, <span class="author">McGinnis R</span>, <span class="author">McLaren W</span>, <span class="author">Pollack S</span>, <span class="author">Price AL</span>, <span class="author">Schaffner SF</span>, <span class="author">Takeuchi F</span>, <span class="author">Grossman SR</span>, <span class="author">Shlyakhter I</span>, <span class="author">Hostetter EB</span>, <span class="author">Sabeti PC</span>, <span class="author">Adebamowo CA</span>, <span class="author">Foster MW</span>, <span class="author">Gordon DR</span>, <span class="author">Licinio J</span>, <span class="author">Manca MC</span>, <span class="author">Marshall PA</span>, <span class="author">Matsuda I</span>, <span class="author">Ngare D</span>, <span class="author">Wang VO</span>, <span class="author">Reddy D</span>, <span class="author">Rotimi CN</span>, <span class="author">Royal CD</span>, <span class="author">Sharp RR</span>, <span class="author">Zeng C</span>, <span class="author">Brooks LD</span> and <span class="author">McEwen JE</span></p>
<p class="affiliation">Broad Institute, 7 Cambridge Center, Cambridge, Massachusetts 02138, USA. <a href="mailto:%61%6C%74%73%68%75%6C%65%72%40%6D%6F%6C%62%69%6F.%6D%67%68.%68%61%72%76%61%72%64.%65%64%75">&#97;&#108;&#116;&#115;&#104;&#117;&#108;&#101;&#114;&#64;&#109;&#111;&#108;&#98;&#105;&#111;&#46;&#109;&#103;&#104;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Despite great progress in identifying genetic variants that influence human disease, most inherited risk remains unexplained. A more complete understanding requires genome-wide studies that fully examine less common alleles in populations with a wide range of ancestry. To inform the design and interpretation of such studies, we genotyped 1.6 million common single nucleotide polymorphisms (SNPs) in 1,184 reference individuals from 11 global populations, and sequenced ten 100-kilobase regions in 692 of these individuals. This integrated data set of common and rare alleles, called &#39;HapMap 3&#39;, includes both SNPs and copy number polymorphisms (CNPs). We characterized population-specific differences among low-frequency variants, measured the improvement in imputation accuracy afforded by the larger reference panel, especially in imputing SNPs with a minor allele frequency of &lt;or=5%, and demonstrated the feasibility of imputing newly discovered CNPs and SNPs. This expanded public resource of genome variants in global populations supports deeper interrogation of genomic variation and its role in human disease, and serves as a step towards a high-resolution map of the landscape of human genetic variation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; NHGRI NIH HHS: <span class="grant">U54 HG003273</span>; NIDDK NIH HHS: <span class="grant">P30 DK043351</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">077014</span>, <span class="grant">082371</span>, <span class="grant">089061</span>, <span class="grant">089062</span>, <span class="grant">091746</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">467</span>;<span class="number">7311</span>;<span class="pagerange">52-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20811451">20811451</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3173859">3173859</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09298">10.1038/nature09298</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lack of support for association between the KIF1B rs10492972[C] variant and multiple sclerosis.</h4>
<p class="authors"><span class="author">International Multiple Sclerosis Genetics Consortium (IMSGC)</span>, <span class="author">Booth DR</span>, <span class="author">Heard RN</span>, <span class="author">Stewart GJ</span>, <span class="author">Cox M</span>, <span class="author">Scott RJ</span>, <span class="author">Lechner-Scott J</span>, <span class="author">Goris A</span>, <span class="author">Dobosi R</span>, <span class="author">Dubois B</span>, <span class="author">Saarela J</span>, <span class="author">Lepp&auml; V</span>, <span class="author">Peltonen L</span>, <span class="author">Pirttila T</span>, <span class="author">Cournu-Rebeix I</span>, <span class="author">Fontaine B</span>, <span class="author">Bergamaschi L</span>, <span class="author">D&#39;Alfonso S</span>, <span class="author">Leone M</span>, <span class="author">Lorentzen AR</span>, <span class="author">Harbo HF</span>, <span class="author">Celius EG</span>, <span class="author">Spurkland A</span>, <span class="author">Link J</span>, <span class="author">Kockum I</span>, <span class="author">Olsson T</span>, <span class="author">Hillert J</span>, <span class="author">Ban M</span>, <span class="author">Baker A</span>, <span class="author">Kemppinen A</span>, <span class="author">Sawcer S</span>, <span class="author">Compston A</span>, <span class="author">Robertson NP</span>, <span class="author">De Jager PL</span>, <span class="author">Hafler DA</span>, <span class="author">Barcellos LF</span>, <span class="author">Ivinson AJ</span>, <span class="author">McCauley JL</span>, <span class="author">Pericak-Vance MA</span>, <span class="author">Oksenberg JR</span>, <span class="author">Hauser SL</span>, <span class="author">Sexton D</span> and <span class="author">Haines J</span></p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">R01 NS049477-01A1</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">6</span>;<span class="pagerange">469-70; author reply 470-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20502484" rel="external">20502484</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3109873" rel="external">3109873</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0610-469" rel="external">10.1038/ng0610-469</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">IL12A, MPHOSPH9/CDK2AP1 and RGS1 are novel multiple sclerosis susceptibility loci.</h4>
<p class="authors"><span class="author">International Multiple Sclerosis Genetics Conssortium (IMSGC)</span></p><p class="abstract">A recent meta-analysis identified seven single-nucleotide polymorphisms (SNPs) with suggestive evidence of association with multiple sclerosis (MS). We report an analysis of these polymorphisms in a replication study that includes 8,085 cases and 7,777 controls. A meta-analysis across the replication collections and a joint analysis with the discovery data set were performed. The possible functional consequences of the validated susceptibility loci were explored using RNA expression data. For all of the tested SNPs, the effect observed in the replication phase involved the same allele and the same direction of effect observed in the discovery phase. Three loci exceeded genome-wide significance in the joint analysis: RGS1 (P value=3.55 x 10(-9)), IL12A (P=3.08 x 10(-8)) and MPHOSPH9/CDK2AP1 (P=3.96 x 10(-8)). The RGS1 risk allele is shared with celiac disease (CD), and the IL12A risk allele seems to be protective for celiac disease. Within the MPHOSPH9/CDK2AP1 locus, the risk allele correlates with diminished RNA expression of the cell cycle regulator CDK2AP1; this effect is seen in both lymphoblastic cell lines (P=1.18 x 10(-5)) and in peripheral blood mononuclear cells from subjects with MS (P=0.01). Thus, we report three new MS susceptibility loci, including a novel inflammatory disease locus that could affect autoreactive cell proliferation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700061</span>; NINDS NIH HHS: <span class="grant">R01NS049477</span></p>
<p class="publication"><span class="title">Genes and immunity</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">5</span>;<span class="pagerange">397-405</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20555355">20555355</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fgene.2010.28">10.1038/gene.2010.28</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Failure to validate association between 12p13 variants and ischemic stroke.</h4>
<p class="authors"><span class="author">International Stroke Genetics Consortium</span> and <span class="author">Wellcome Trust Case-Control Consortium 2</span></p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0701075</span>; NCI NIH HHS: <span class="grant">CA 047988</span>; NCRR NIH HHS: <span class="grant">M01 RR 165001</span>, <span class="grant">M01 RR07122</span>, <span class="grant">R54 RR020278</span>; NHGRI NIH HHS: <span class="grant">U01 HG004436</span>; NHLBI NIH HHS: <span class="grant">HL 043851</span>, <span class="grant">HL69757</span>, <span class="grant">R01 HL087676</span>, <span class="grant">R25 HL088724</span>; NIDDK NIH HHS: <span class="grant">P30 DK072488</span>; NINDS NIH HHS: <span class="grant">1R01 NS059727</span>, <span class="grant">K08 NS045802</span>, <span class="grant">NS056302</span>, <span class="grant">NS30678</span>, <span class="grant">NS34447</span>, <span class="grant">NS36695</span>, <span class="grant">R01 NS 42733</span>, <span class="grant">R01 NS059727</span>, <span class="grant">R01 NS059727-01A1</span>, <span class="grant">R01 NS45012</span>, <span class="grant">R21NS064908</span>; PHS HHS: <span class="grant">P60 12583</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2010</span>;<span class="volume">362</span>;<span class="number">16</span>;<span class="pagerange">1547-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20410525">20410525</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2978046">2978046</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMc0910050">10.1056/NEJMc0910050</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of Trypanosoma brucei gambiense, causative agent of chronic human african trypanosomiasis.</h4>
<p class="authors"><span class="author">Jackson AP</span>, <span class="author">Sanders M</span>, <span class="author">Berry A</span>, <span class="author">McQuillan J</span>, <span class="author">Aslett MA</span>, <span class="author">Quail MA</span>, <span class="author">Chukualim B</span>, <span class="author">Capewell P</span>, <span class="author">MacLeod A</span>, <span class="author">Melville SE</span>, <span class="author">Gibson W</span>, <span class="author">Barry JD</span>, <span class="author">Berriman M</span> and <span class="author">Hertz-Fowler C</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Trypanosoma brucei gambiense is the causative agent of chronic Human African Trypanosomiasis or sleeping sickness, a disease endemic across often poor and rural areas of Western and Central Africa. We have previously published the genome sequence of a T. b. brucei isolate, and have now employed a comparative genomics approach to understand the scale of genomic variation between T. b. gambiense and the reference genome. We sought to identify features that were uniquely associated with T. b. gambiense and its ability to infect humans.</p><p class="abstract">An improved high-quality draft genome sequence for the group 1 T. b. gambiense DAL 972 isolate was produced using a whole-genome shotgun strategy. Comparison with T. b. brucei showed that sequence identity averages 99.2% in coding regions, and gene order is largely collinear. However, variation associated with segmental duplications and tandem gene arrays suggests some reduction of functional repertoire in T. b. gambiense DAL 972. A comparison of the variant surface glycoproteins (VSG) in T. b. brucei with all T. b. gambiense sequence reads showed that the essential structural repertoire of VSG domains is conserved across T. brucei.</p><p class="abstract"><strong>Conclusions</strong>: This study provides the first estimate of intraspecific genomic variation within T. brucei, and so has important consequences for future population genomics studies. We have shown that the T. b. gambiense genome corresponds closely with the reference, which should therefore be an effective scaffold for any T. brucei genome sequence data. As VSG repertoire is also well conserved, it may be feasible to describe the total diversity of variant antigens. While we describe several as yet uncharacterized gene families with predicted cell surface roles that were expanded in number in T. b. brucei, no T. b. gambiense-specific gene was identified outside of the subtelomeres that could explain the ability to infect humans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079703</span>, <span class="grant">095201</span>, <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="number">4</span>;<span class="pagerange">e658</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20404998">20404998</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2854126">2854126</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000658">10.1371/journal.pntd.0000658</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study in a high-risk isolate for multiple sclerosis reveals associated variants in STAT3 gene.</h4>
<p class="authors"><span class="author">Jakkula E</span>, <span class="author">Lepp&auml; V</span>, <span class="author">Sulonen AM</span>, <span class="author">Varilo T</span>, <span class="author">Kallio S</span>, <span class="author">Kemppinen A</span>, <span class="author">Purcell S</span>, <span class="author">Koivisto K</span>, <span class="author">Tienari P</span>, <span class="author">Sumelahti ML</span>, <span class="author">Elovaara I</span>, <span class="author">Pirttil&auml; T</span>, <span class="author">Reunanen M</span>, <span class="author">Aromaa A</span>, <span class="author">Oturai AB</span>, <span class="author">S&oslash;ndergaard HB</span>, <span class="author">Harbo HF</span>, <span class="author">Mero IL</span>, <span class="author">Gabriel SB</span>, <span class="author">Mirel DB</span>, <span class="author">Hauser SL</span>, <span class="author">Kappos L</span>, <span class="author">Polman C</span>, <span class="author">De Jager PL</span>, <span class="author">Hafler DA</span>, <span class="author">Daly MJ</span>, <span class="author">Palotie A</span>, <span class="author">Saarela J</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Genetic risk for multiple sclerosis (MS) is thought to involve both common and rare risk alleles. Recent GWAS and subsequent meta-analysis have established the critical role of the HLA locus and identified new common variants associated to MS. These variants have small odds ratios (ORs) and explain only a fraction of the genetic risk. To expose potentially rare, high-impact alleles, we conducted a GWAS of 68 distantly related cases and 136 controls from a high-risk internal isolate of Finland with increased prevalence and familial occurrence of MS. The top 27 loci with p &lt; 10(-4) were tested in 711 cases and 1029 controls from Finland, and the top two findings were validated in 3859 cases and 9110 controls from more heterogeneous populations. SNP (rs744166) within the STAT3 gene was associated to MS (p = 2.75 x 10(-10), OR 0.87, confidence interval 0.83-0.91). The protective haplotype for MS in STAT3 is a risk allele for Crohn disease, implying that STAT3 represents a shared risk locus for at least two autoimmune diseases. This study also demonstrates the potential of special isolated populations in search for variants contributing to complex traits.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">U54 RR020278</span>; NINDS NIH HHS: <span class="grant">R01 NS 43559</span>; Wellcome Trust: <span class="grant">089061/Z/09/Z</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">86</span>;<span class="number">2</span>;<span class="pagerange">285-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20159113" rel="external">20159113</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2820168" rel="external">2820168</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.01.017" rel="external">10.1016/j.ajhg.2010.01.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic diversity of two haploid markers in the Udegey population from southeastern Siberia.</h4>
<p class="authors"><span class="author">Jin HJ</span>, <span class="author">Kim KC</span> and <span class="author">Kim W</span></p>
<p class="affiliation">Department of Biological Sciences, Dankook University, Cheonan 330-714, Korea.</p><p class="abstract">The Udegeys are a small ethnic group who live along the tributaries of the Amur River Basin of southeastern Siberia in Russia. They are thought to speak a language belonging to a subdivision of the Tungusic-Manchu branch of the Altaic family. To understand the genetic features and genetic history of the Udegeys, we analyzed two haploid markers, mitochondrial DNA (mtDNA), and Y-chromosomal variation, in 51 individuals (including 21 males) from the Udegey population. In general, the Udegeys&#39; mtDNA profiles revealed similarities to Siberians and other northeastern Asian populations, although a moderate European contribution was also detected. Interestingly, pairwise values of F(ST) and the MDS plots based on the mtDNA variation showed that the Orok and Nivkh inhabiting the very same region of the Udegey were significantly different from the Udegey, implying that they may have been isolated and undergone substantial genetic drift. The Udegeys were characterized by a high frequency (66.7%) of Y chromosome haplogroup C, indicating a close genetic relationship with Mongolians and Siberians. On the paternal side, however, very little admixture was observed between the Udegeys and Europeans. Thus, the combined haploid genetic markers of both mtDNA and the Y chromosome imply that the Udegeys are overall closest to Siberians and northeast Asians of the Altaic linguistic family, with a minor maternal contribution from the European part of the continent.</p>
<p class="publication"><span class="title">American journal of physical anthropology</span>&nbsp;<span class="year">2010</span>;<span class="volume">142</span>;<span class="number">2</span>;<span class="pagerange">303-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19953529" rel="external">19953529</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajpa.21232" rel="external">10.1002/ajpa.21232</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The JAK2 46/1 haplotype predisposes to MPL-mutated myeloproliferative neoplasms.</h4>
<p class="authors"><span class="author">Jones AV</span>, <span class="author">Campbell PJ</span>, <span class="author">Beer PA</span>, <span class="author">Schnittger S</span>, <span class="author">Vannucchi AM</span>, <span class="author">Zoi K</span>, <span class="author">Percy MJ</span>, <span class="author">McMullin MF</span>, <span class="author">Scott LM</span>, <span class="author">Tapper W</span>, <span class="author">Silver RT</span>, <span class="author">Oscier D</span>, <span class="author">Harrison CN</span>, <span class="author">Grallert H</span>, <span class="author">Kisialiou A</span>, <span class="author">Strike P</span>, <span class="author">Chase AJ</span>, <span class="author">Green AR</span> and <span class="author">Cross NC</span></p>
<p class="affiliation">Wessex Regional Genetics Laboratory, Salisbury District Hospital, Salisbury, United Kingdom.</p><p class="abstract">The 46/1 JAK2 haplotype predisposes to V617F-positive myeloproliferative neoplasms, but the underlying mechanism is obscure. We analyzed essential thrombocythemia patients entered into the PT-1 studies and, as expected, found that 46/1 was overrepresented in V617F-positive cases (n = 404) versus controls (n = 1492, P = 3.9 x 10(-11)). The 46/1 haplotype was also overrepresented in cases without V617F (n = 347, P = .009), with an excess seen for both MPL exon 10 mutated and V617F, MPL exon 10 nonmutated cases. Analysis of further MPL-positive, V617F-negative cases confirmed an excess of 46/1 (n = 176, P = .002), but no association between MPL mutations and MPL haplotype was seen. An excess of 46/1 was also seen in JAK2 exon 12 mutated cases (n = 69, P = .002), and these mutations preferentially arose on the 46/1 chromosome (P = .029). No association between 46/1 and clinical or laboratory features was seen in the PT-1 cohort either with or without V617F. The excess of 46/1 in JAK2 exon 12 cases is compatible with both the &quot;hypermutability&quot; and &quot;fertile ground&quot; hypotheses, but the excess in MPL-mutated cases argues against the former. No difference in sequence, splicing, or expression of JAK2 was found on 46/1 compared with other haplotypes, suggesting that any functional difference of JAK2 on 46/1, if it exists, must be relatively subtle.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">N01-AG-1-1</span>, <span class="grant">N01-AG-1-2111</span>, <span class="grant">N01-AG-5-0002</span>; NIMHD NIH HHS: <span class="grant">263 MD 821336</span>, <span class="grant">263 MD 9164</span>; Wellcome Trust: <span class="grant">07611</span>, <span class="grant">088340</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2010</span>;<span class="volume">115</span>;<span class="number">22</span>;<span class="pagerange">4517-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20304805" rel="external">20304805</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145114" rel="external">3145114</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2009-08-236448" rel="external">10.1182/blood-2009-08-236448</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic evidence implicates the immune system and cholesterol metabolism in the aetiology of Alzheimer&#39;s disease.</h4>
<p class="authors"><span class="author">Jones L</span>, <span class="author">Holmans PA</span>, <span class="author">Hamshere ML</span>, <span class="author">Harold D</span>, <span class="author">Moskvina V</span>, <span class="author">Ivanov D</span>, <span class="author">Pocklington A</span>, <span class="author">Abraham R</span>, <span class="author">Hollingworth P</span>, <span class="author">Sims R</span>, <span class="author">Gerrish A</span>, <span class="author">Pahwa JS</span>, <span class="author">Jones N</span>, <span class="author">Stretton A</span>, <span class="author">Morgan AR</span>, <span class="author">Lovestone S</span>, <span class="author">Powell J</span>, <span class="author">Proitsi P</span>, <span class="author">Lupton MK</span>, <span class="author">Brayne C</span>, <span class="author">Rubinsztein DC</span>, <span class="author">Gill M</span>, <span class="author">Lawlor B</span>, <span class="author">Lynch A</span>, <span class="author">Morgan K</span>, <span class="author">Brown KS</span>, <span class="author">Passmore PA</span>, <span class="author">Craig D</span>, <span class="author">McGuinness B</span>, <span class="author">Todd S</span>, <span class="author">Holmes C</span>, <span class="author">Mann D</span>, <span class="author">Smith AD</span>, <span class="author">Love S</span>, <span class="author">Kehoe PG</span>, <span class="author">Mead S</span>, <span class="author">Fox N</span>, <span class="author">Rossor M</span>, <span class="author">Collinge J</span>, <span class="author">Maier W</span>, <span class="author">Jessen F</span>, <span class="author">Sch&uuml;rmann B</span>, <span class="author">Heun R</span>, <span class="author">K&ouml;lsch H</span>, <span class="author">van den Bussche H</span>, <span class="author">Heuser I</span>, <span class="author">Peters O</span>, <span class="author">Kornhuber J</span>, <span class="author">Wiltfang J</span>, <span class="author">Dichgans M</span>, <span class="author">Fr&ouml;lich L</span>, <span class="author">Hampel H</span>, <span class="author">H&uuml;ll M</span>, <span class="author">Rujescu D</span>, <span class="author">Goate AM</span>, <span class="author">Kauwe JS</span>, <span class="author">Cruchaga C</span>, <span class="author">Nowotny P</span>, <span class="author">Morris JC</span>, <span class="author">Mayo K</span>, <span class="author">Livingston G</span>, <span class="author">Bass NJ</span>, <span class="author">Gurling H</span>, <span class="author">McQuillin A</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">Al-Chalabi A</span>, <span class="author">Shaw CE</span>, <span class="author">Singleton AB</span>, <span class="author">Guerreiro R</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Moebus S</span>, <span class="author">J&ouml;ckel KH</span>, <span class="author">Klopp N</span>, <span class="author">Wichmann HE</span>, <span class="author">R&uuml;ther E</span>, <span class="author">Carrasquillo MM</span>, <span class="author">Pankratz VS</span>, <span class="author">Younkin SG</span>, <span class="author">Hardy J</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Owen MJ</span> and <span class="author">Williams J</span></p>
<p class="affiliation">Department of Psychological Medicine and Neurology, School of Medicine, Cardiff University, Medical Research Council Centre for Neuropsychiatric Genetics and Genomics, Cardiff, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Late Onset Alzheimer&#39;s disease (LOAD) is the leading cause of dementia. Recent large genome-wide association studies (GWAS) identified the first strongly supported LOAD susceptibility genes since the discovery of the involvement of APOE in the early 1990s. We have now exploited these GWAS datasets to uncover key LOAD pathophysiological processes.</p><p class="abstract"><strong>Methodology</strong>: We applied a recently developed tool for mining GWAS data for biologically meaningful information to a LOAD GWAS dataset. The principal findings were then tested in an independent GWAS dataset.</p><p class="abstract">We found a significant overrepresentation of association signals in pathways related to cholesterol metabolism and the immune response in both of the two largest genome-wide association studies for LOAD.</p><p class="abstract"><strong>Significance</strong>: Processes related to cholesterol metabolism and the innate immune response have previously been implicated by pathological and epidemiological studies of Alzheimer&#39;s disease, but it has been unclear whether those findings reflected primary aetiological events or consequences of the disease process. Our independent evidence from two large studies now demonstrates that these processes are aetiologically relevant, and suggests that they may be suitable targets for novel and existing therapeutic approaches.</p>
<p class="grants">Funded by: Medical Research Council; NIA NIH HHS: <span class="grant">Z01 AG000950-06</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">e13950</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21085570" rel="external">21085570</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2981526" rel="external">2981526</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0013950" rel="external">10.1371/journal.pone.0013950</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reverse engineering a gene network using an asynchronous parallel evolution strategy.</h4>
<p class="authors"><span class="author">Jostins L</span> and <span class="author">Jaeger J</span></p>
<p class="affiliation">Laboratory for Development &amp; Evolution, University Museum of Zoology, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, UK.</p><p class="abstract"><strong>Background</strong>: The use of reverse engineering methods to infer gene regulatory networks by fitting mathematical models to gene expression data is becoming increasingly popular and successful. However, increasing model complexity means that more powerful global optimisation techniques are required for model fitting. The parallel Lam Simulated Annealing (pLSA) algorithm has been used in such approaches, but recent research has shown that island Evolutionary Strategies can produce faster, more reliable results. However, no parallel island Evolutionary Strategy (piES) has yet been demonstrated to be effective for this task.</p><p class="abstract"><strong>Results</strong>: Here, we present synchronous and asynchronous versions of the piES algorithm, and apply them to a real reverse engineering problem: inferring parameters in the gap gene network. We find that the asynchronous piES exhibits very little communication overhead, and shows significant speed-up for up to 50 nodes: the piES running on 50 nodes is nearly 10 times faster than the best serial algorithm. We compare the asynchronous piES to pLSA on the same test problem, measuring the time required to reach particular levels of residual error, and show that it shows much faster convergence than pLSA across all optimisation conditions tested.</p><p class="abstract"><strong>Conclusions</strong>: Our results demonstrate that the piES is consistently faster and more reliable than the pLSA algorithm on this problem, and scales better with increasing numbers of nodes. In addition, the piES is especially well suited to further improvements and adaptations: Firstly, the algorithm&#39;s fast initial descent speed and high reliability make it a good candidate for being used as part of a global/local search hybrid algorithm. Secondly, it has the potential to be used as part of a hierarchical evolutionary algorithm, which takes advantage of modern multi-core computing architectures.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/D000513/1</span>, <span class="grant">BB/D00513</span></p>
<p class="publication"><span class="title">BMC systems biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="pagerange">17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20196855">20196855</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2850326">2850326</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1752-0509-4-17">10.1186/1752-0509-4-17</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">iCLIP reveals the function of hnRNP particles in splicing at individual nucleotide resolution.</h4>
<p class="authors"><span class="author">K&ouml;nig J</span>, <span class="author">Zarnack K</span>, <span class="author">Rot G</span>, <span class="author">Curk T</span>, <span class="author">Kayikci M</span>, <span class="author">Zupan B</span>, <span class="author">Turner DJ</span>, <span class="author">Luscombe NM</span> and <span class="author">Ule J</span></p>
<p class="affiliation">Medical Research Council Laboratory of Molecular Biology, Cambridge, UK.</p><p class="abstract">In the nucleus of eukaryotic cells, nascent transcripts are associated with heterogeneous nuclear ribonucleoprotein (hnRNP) particles that are nucleated by hnRNP C. Despite their abundance, however, it remained unclear whether these particles control pre-mRNA processing. Here, we developed individual-nucleotide resolution UV cross-linking and immunoprecipitation (iCLIP) to study the role of hnRNP C in splicing regulation. iCLIP data show that hnRNP C recognizes uridine tracts with a defined long-range spacing consistent with hnRNP particle organization. hnRNP particles assemble on both introns and exons but remain generally excluded from splice sites. Integration of transcriptome-wide iCLIP data and alternative splicing profiles into an &#39;RNA map&#39; indicates how the positioning of hnRNP particles determines their effect on the inclusion of alternative exons. The ability of high-resolution iCLIP data to provide insights into the mechanism of this regulation holds promise for studies of other higher-order ribonucleoprotein complexes.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E01075X/1</span>; Medical Research Council: <span class="grant">U.1051.04.028.00001.01 (85858)</span></p>
<p class="publication"><span class="title">Nature structural &amp; molecular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">17</span>;<span class="number">7</span>;<span class="pagerange">909-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20601959" rel="external">20601959</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3000544" rel="external">3000544</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnsmb.1838" rel="external">10.1038/nsmb.1838</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Life-course analysis of a fat mass and obesity-associated (FTO) gene variant and body mass index in the Northern Finland Birth Cohort 1966 using structural equation modeling.</h4>
<p class="authors"><span class="author">Kaakinen M</span>, <span class="author">L&auml;&auml;r&auml; E</span>, <span class="author">Pouta A</span>, <span class="author">Hartikainen AL</span>, <span class="author">Laitinen J</span>, <span class="author">Tammelin TH</span>, <span class="author">Herzig KH</span>, <span class="author">Sovio U</span>, <span class="author">Bennett AJ</span>, <span class="author">Peltonen L</span>, <span class="author">McCarthy MI</span>, <span class="author">Elliott P</span>, <span class="author">De Stavola B</span> and <span class="author">J&auml;rvelin MR</span></p>
<p class="affiliation">Institute of Health Sciences, Faculty of Medicine, University of Oulu, Oulu, Finland.</p><p class="abstract">The association between variation in the fat mass and obesity-associated (FTO) gene and adulthood body mass index (BMI; weight (kg)/height (m)(2)) is well-replicated. More thorough analyses utilizing phenotypic data over the life course may deepen our understanding of the development of BMI and thus help in the prevention of obesity. The authors used a structural equation modeling approach to explore the network of variables associated with BMI from the prenatal period to age 31 years (1965-1997) in 4,435 subjects from the Northern Finland Birth Cohort 1966. The use of structural equation modeling permitted the easy inclusion of variables with missing values in the analyses without separate imputation steps, as well as differentiation between direct and indirect effects. There was an association between the FTO single nucleotide polymorphism rs9939609 and BMI at age 31 years that persisted after controlling for several relevant factors during the life course. The total effect of the FTO variant on adult BMI was mostly composed of the direct effect, but a notable part was also arising indirectly via its effects on earlier BMI development. In addition to well-established genetic determinants, many life-course factors such as physical activity, in spite of not showing mediation or interaction, had a strong independent effect on BMI.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span></p>
<p class="publication"><span class="title">American journal of epidemiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">172</span>;<span class="number">6</span>;<span class="pagerange">653-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20702506" rel="external">20702506</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2938267" rel="external">2938267</a>; DOI: <a href="http://dx.doi.org/10.1093%2Faje%2Fkwq178" rel="external">10.1093/aje/kwq178</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Childhood adversities are associated with shorter telomere length at adult age both in individuals with an anxiety disorder and controls.</h4>
<p class="authors"><span class="author">Kananen L</span>, <span class="author">Surakka I</span>, <span class="author">Pirkola S</span>, <span class="author">Suvisaari J</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Peltonen L</span>, <span class="author">Ripatti S</span> and <span class="author">Hovatta I</span></p>
<p class="affiliation">Research Program of Molecular Neurology, Faculty of Medicine, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Accelerated leukocyte telomere shortening has been previously associated to self-perceived stress and psychiatric disorders, including schizophrenia and mood disorders. We set out to investigate whether telomere length is affected in patients with anxiety disorders in which stress is a known risk factor. We also studied the effects of childhood and recent psychological distress on telomere length. We utilized samples from the nationally representative population-based Health 2000 Survey that was carried out between 2000-2001 in Finland to assess major public health problems and their determinants. We measured the relative telomere length of the peripheral blood cells by quantitative real-time PCR from 321 individuals with DSM-IV anxiety disorder or subthreshold diagnosis and 653 matched controls aged 30-87 years, who all had undergone the Composite International Diagnostic Interview. While telomere length did not differ significantly between cases and controls in the entire cohort, the older half of the anxiety disorder patients (48-87 years) exhibited significantly shorter telomeres than healthy controls of the same age (P = 0.013). Interestingly, shorter telomere length was also associated with a greater number of reported childhood adverse life events, among both the anxiety disorder cases and controls (P = 0.005). Childhood chronic or serious illness was the most significantly associated single event affecting telomere length at the adult age (P = 0.004). Self-reported current psychological distress did not affect telomere length. Our results suggest that childhood stress might lead to accelerated telomere shortening seen at the adult age. This finding has potentially important implications supporting the view that childhood adversities might have a considerable impact on well being later in life.</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">e10826</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20520834" rel="external">20520834</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2876034" rel="external">2876034</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0010826" rel="external">10.1371/journal.pone.0010826</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prokaryote-derived protein inhibitors of peptidases: A sketchy occurrence and mostly unknown function.</h4>
<p class="authors"><span class="author">Kantyka T</span>, <span class="author">Rawlings ND</span> and <span class="author">Potempa J</span></p>
<p class="affiliation">Department of Microbiology, Jagiellonian University, Krakow, Poland.</p><p class="abstract">In metazoan organisms protein inhibitors of peptidases are important factors essential for regulation of proteolytic activity. In vertebrates genes encoding peptidase inhibitors constitute up to 1% of genes reflecting a need for tight and specific control of proteolysis especially in extracellular body fluids. In stark contrast unicellular organisms, both prokaryotic and eukaryotic consistently contain only few, if any, genes coding for putative peptidase inhibitors. This may seem perplexing in the light of the fact that these organisms produce large numbers of proteases of different catalytic classes with the genes constituting up to 6% of the total gene count with the average being about 3%. Apparently, however, a unicellular life-style is fully compatible with other mechanisms of regulation of proteolysis and does not require protein inhibitors to control their intracellular and extracellular proteolytic activity. So in prokaryotes occurrence of genes encoding different types of peptidase inhibitors is infrequent and often scattered among phylogenetically distinct orders or even phyla of microbiota. Genes encoding proteins homologous to alpha-2-macroglobulin (family I39), serine carboxypeptidase Y inhibitor (family I51), alpha-1-peptidase inhibitor (family I4) and ecotin (family I11) are the most frequently represented in Bacteria. Although several of these gene products were shown to possess inhibitory activity, with an exception of ecotin and staphostatins, the biological function of microbial inhibitors is unclear. In this review we present distribution of protein inhibitors from different families among prokaryotes, describe their mode of action and hypothesize on their role in microbial physiology and interactions with hosts and environment.</p>
<p class="grants">Funded by: NIDCR NIH HHS: <span class="grant">R01 DE009761-18</span>; Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Biochimie</span>&nbsp;<span class="year">2010</span>;<span class="volume">92</span>;<span class="number">11</span>;<span class="pagerange">1644-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20558234" rel="external">20558234</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2952067" rel="external">2952067</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biochi.2010.06.004" rel="external">10.1016/j.biochi.2010.06.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Typhoid in Kenya is associated with a dominant multidrug-resistant Salmonella enterica serovar Typhi haplotype that is also widespread in Southeast Asia.</h4>
<p class="authors"><span class="author">Kariuki S</span>, <span class="author">Revathi G</span>, <span class="author">Kiiru J</span>, <span class="author">Mengo DM</span>, <span class="author">Mwituria J</span>, <span class="author">Muyodi J</span>, <span class="author">Munyalo A</span>, <span class="author">Teo YY</span>, <span class="author">Holt KE</span>, <span class="author">Kingsley RA</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Centre for Microbiology Research, Kenya Medical Research Institute, P.O. Box 43640-00100, Nairobi, Kenya. <a href="mailto:%73%6B%61%72%69%75%6B%69%40%6B%65%6D%72%69.%6F%72%67">&#115;&#107;&#97;&#114;&#105;&#117;&#107;&#105;&#64;&#107;&#101;&#109;&#114;&#105;&#46;&#111;&#114;&#103;</a></p><p class="abstract">In sub-Saharan Africa, the burden of typhoid fever, caused by Salmonella enterica serovar Typhi, remains largely unknown, in part because of a lack of blood or bone marrow culture facilities. We characterized a total of 323 S. Typhi isolates from outbreaks in Kenya over the period 1988 to 2008 for antimicrobial susceptibilities and phylogenetic relationships using single-nucleotide polymorphism (SNP) analysis. There was a dramatic increase in the number and percentage of multidrug-resistant (MDR) S. Typhi isolates over the study period. Overall, only 54 (16.7%) S. Typhi isolates were fully sensitive, while the majority, 195 (60.4%), were multiply resistant to most commonly available drugs-ampicillin, chloramphenicol, tetracycline, and cotrimoxazole; 74 (22.9%) isolates were resistant to a single antimicrobial, usually ampicillin, cotrimoxazole, or tetracycline. Resistance to these antibiotics was encoded on self-transferrable IncHI1 plasmids of the ST6 sequence type. Of the 94 representative S. Typhi isolates selected for genome-wide haplotype analysis, sensitive isolates fell into several phylogenetically different groups, whereas MDR isolates all belonged to a single haplotype, H58, associated with MDR and decreased ciprofloxacin susceptibility, which is also dominant in many parts of Southeast Asia. Derivatives of the same S. Typhi lineage, H58, are responsible for multidrug resistance in Kenya and parts of Southeast Asia, suggesting intercontinental spread of a single MDR clone. Given the emergence of this aggressive MDR haplotype, careful selection and monitoring of antibiotic usage will be required in Kenya, and potentially other regions of sub-Saharan Africa.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">064616/01/Z.</span></p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">48</span>;<span class="number">6</span>;<span class="pagerange">2171-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20392916">20392916</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2884483">2884483</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.01983-09">10.1128/JCM.01983-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The burden and characteristics of enteric fever at a healthcare facility in a densely populated area of Kathmandu.</h4>
<p class="authors"><span class="author">Karkey A</span>, <span class="author">Arjyal A</span>, <span class="author">Anders KL</span>, <span class="author">Boni MF</span>, <span class="author">Dongol S</span>, <span class="author">Koirala S</span>, <span class="author">My PV</span>, <span class="author">Nga TV</span>, <span class="author">Clements AC</span>, <span class="author">Holt KE</span>, <span class="author">Duy PT</span>, <span class="author">Day JN</span>, <span class="author">Campbell JI</span>, <span class="author">Dougan G</span>, <span class="author">Dolecek C</span>, <span class="author">Farrar J</span>, <span class="author">Basnyat B</span> and <span class="author">Baker S</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Patan Academy of Health Sciences, Lagankhel, Kathmandu, Nepal.</p><p class="abstract">Enteric fever, caused by Salmonella enterica serovars Typhi and Paratyphi A (S. Typhi and S. Paratyphi A) remains a major public health problem in many settings. The disease is limited to locations with poor sanitation which facilitates the transmission of the infecting organisms. Efficacious and inexpensive vaccines are available for S. Typhi, yet are not commonly deployed to control the disease. Lack of vaccination is due partly to uncertainty of the disease burden arising from a paucity of epidemiological information in key locations. We have collected and analyzed data from 3,898 cases of blood culture-confirmed enteric fever from Patan Hospital in Lalitpur Sub-Metropolitan City (LSMC), between June 2005 and May 2009. Demographic data was available for a subset of these patients (n = 527) that were resident in LSMC and who were enrolled in trials. We show a considerable burden of enteric fever caused by S. Typhi (2,672; 68.5%) and S. Paratyphi A (1,226; 31.5%) at this Hospital over a four year period, which correlate with seasonal fluctuations in rainfall. We found that local population density was not related to incidence and we identified a focus of infections in the east of LSMC. With data from patients resident in LSMC we found that the median age of those with S. Typhi (16 years) was significantly less than S. Paratyphi A (20 years) and that males aged 15 to 25 were disproportionately infected. Our findings provide a snapshot into the epidemiological patterns of enteric fever in Kathmandu. The uneven distribution of enteric fever patients within the population suggests local variation in risk factors, such as contaminated drinking water. These findings are important for initiating a vaccination scheme and improvements in sanitation. We suggest any such intervention should be implemented throughout the LSMC area.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600718</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">e13988</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21085575">21085575</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2981554">2981554</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0013988">10.1371/journal.pone.0013988</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Optimising experimental design for high-throughput phenotyping in mice: a case study.</h4>
<p class="authors"><span class="author">Karp NA</span>, <span class="author">Baker LA</span>, <span class="author">Gerdin AK</span>, <span class="author">Adams NC</span>, <span class="author">Ram&iacute;rez-Solis R</span> and <span class="author">White JK</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">To further the functional annotation of the mammalian genome, the Sanger Mouse Genetics Programme aims to generate and characterise knockout mice in a high-throughput manner. Annually, approximately 200 lines of knockout mice will be characterised using a standardised battery of phenotyping tests covering key disease indications ranging from obesity to sensory acuity. From these findings secondary centres will select putative mutants of interest for more in-depth, confirmatory experiments. Optimising experimental design and data analysis is essential to maximise output using the resources with greatest efficiency, thereby attaining our biological objective of understanding the role of genes in normal development and disease. This study uses the example of the noninvasive blood pressure test to demonstrate how statistical investigation is important for generating meaningful, reliable results and assessing the design for the defined research objectives. The analysis adjusts for the multiple-testing problem by applying the false discovery rate, which controls the number of false calls within those highlighted as significant. A variance analysis finds that the variation between mice dominates this assay. These variance measures were used to examine the interplay between days, readings, and number of mice on power, the ability to detect change. If an experiment is underpowered, we cannot conclude whether failure to detect a biological difference arises from low power or lack of a distinct phenotype, hence the mice are subjected to testing without gain. Consequently, in confirmatory studies, a power analysis along with the 3Rs can provide justification to increase the number of mice used.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077157/Z/05/Z</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2010</span>;<span class="volume">21</span>;<span class="number">9-10</span>;<span class="pagerange">467-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20799038" rel="external">20799038</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2974211" rel="external">2974211</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-010-9279-1" rel="external">10.1007/s00335-010-9279-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Addressing accuracy and precision issues in iTRAQ quantitation.</h4>
<p class="authors"><span class="author">Karp NA</span>, <span class="author">Huber W</span>, <span class="author">Sadowski PG</span>, <span class="author">Charles PD</span>, <span class="author">Hester SV</span> and <span class="author">Lilley KS</span></p>
<p class="affiliation">European Bioinformatics Institute, European Molecular Biology Laboratory Outstation, Hinxton, UK.</p><p class="abstract">iTRAQ (isobaric tags for relative or absolute quantitation) is a mass spectrometry technology that allows quantitative comparison of protein abundance by measuring peak intensities of reporter ions released from iTRAQ-tagged peptides by fragmentation during MS/MS. However, current data analysis techniques for iTRAQ struggle to report reliable relative protein abundance estimates and suffer with problems of precision and accuracy. The precision of the data is affected by variance heterogeneity: low signal data have higher relative variability; however, low abundance peptides dominate data sets. Accuracy is compromised as ratios are compressed toward 1, leading to underestimation of the ratio. This study investigated both issues and proposed a methodology that combines the peptide measurements to give a robust protein estimate even when the data for the protein are sparse or at low intensity. Our data indicated that ratio compression arises from contamination during precursor ion selection, which occurs at a consistent proportion within an experiment and thus results in a linear relationship between expected and observed ratios. We proposed that a correction factor can be calculated from spiked proteins at known ratios. Then we demonstrated that variance heterogeneity is present in iTRAQ data sets irrespective of the analytical packages, LC-MS/MS instrumentation, and iTRAQ labeling kit (4-plex or 8-plex) used. We proposed using an additive-multiplicative error model for peak intensities in MS/MS quantitation and demonstrated that a variance-stabilizing normalization is able to address the error structure and stabilize the variance across the entire intensity range. The resulting uniform variance structure simplifies the downstream analysis. Heterogeneity of variance consistent with an additive-multiplicative model has been reported in other MS-based quantitation including fields outside of proteomics; consequently the variance-stabilizing normalization methodology has the potential to increase the capabilities of MS in quantitation across diverse areas of biology and chemistry.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C50694/1</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="number">9</span>;<span class="pagerange">1885-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20382981" rel="external">20382981</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2938101" rel="external">2938101</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M900628-MCP200" rel="external">10.1074/mcp.M900628-MCP200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mass Spectrometry for Microbial Proteomics: Issues in Data Analysis with Electrophoretic or Mass Spectrometric Expression Proteomic Data</h4>
<p class="authors"><span class="author">Karp, N.</span></p>
<p class="publication"><span class="title">Mass Spectrometry for Microbial Proteomics</span>&nbsp;<span class="year">2010</span>;<span class="number">Chapter 18</span>;<span class="pagerange">423-40</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1002%2F9780470665497.ch18">10.1002/9780470665497.ch18</a>; URL: <a title="http://onlinelibrary.wiley.com/doi/10.1002/9780470665497.ch18/summary;jsessionid=C18EE48CEA80BB29BA75600B58FD5E7C.f02t03" href="http://onlinelibrary.wiley.com/doi/10.1002/9780470665497.ch18/summary;jsessionid=C18EE48CEA80BB29BA75600B58FD5E7C.f02t03">http://onlinelibrary.wiley.com/doi/<span class="print-hide">...</span><span class="print-show">10.1002/9780470665497.ch18/summary;jsessionid=C18EE48CEA80BB29BA75600B58FD5E7C.f</span>02t03</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">European lactase persistence genotype shows evidence of association with increase in body mass index.</h4>
<p class="authors"><span class="author">Kettunen J</span>, <span class="author">Silander K</span>, <span class="author">Saarela O</span>, <span class="author">Amin N</span>, <span class="author">M&uuml;ller M</span>, <span class="author">Timpson N</span>, <span class="author">Surakka I</span>, <span class="author">Ripatti S</span>, <span class="author">Laitinen J</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">Lahermo P</span>, <span class="author">Anttila V</span>, <span class="author">M&auml;nnist&ouml; S</span>, <span class="author">Jula A</span>, <span class="author">Virtamo J</span>, <span class="author">Salomaa V</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Raitakari O</span>, <span class="author">Gieger C</span>, <span class="author">Wichmann EH</span>, <span class="author">Van Duijn CM</span>, <span class="author">Smith GD</span>, <span class="author">McCarthy MI</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Perola M</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1HH, UK. <a href="mailto:%6A%6F%68%61%6E%6E%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#111;&#104;&#97;&#110;&#110;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The global prevalence of obesity has increased significantly in recent decades, mainly due to excess calorie intake and increasingly sedentary lifestyle. Here, we test the association between obesity measured by body mass index (BMI) and one of the best-known genetic variants showing strong selective pressure: the functional variant in the cis-regulatory element of the lactase gene. We tested this variant since it is presumed to provide nutritional advantage in specific physical and cultural environments. We genetically defined lactase persistence (LP) in 31 720 individuals from eight European population-based studies and one family study by genotyping or imputing the European LP variant (rs4988235). We performed a meta-analysis by pooling the beta-coefficient estimates of the relationship between rs4988235 and BMI from the nine studies and found that the carriers of the allele responsible for LP among Europeans showed higher BMI (P = 7.9 x 10(-5)). Since this locus has been shown to be prone to population stratification, we paid special attention to reveal any population substructure which might be responsible for the association signal. The best evidence of exclusion of stratification came from the Dutch family sample which is robust for stratification. In this study, we highlight issues in model selection in the genome-wide association studies and problems in imputation of these special genomic regions.</p>
<p class="grants">Funded by: CCR NIH HHS: <span class="grant">N01-RC-37004</span>, <span class="grant">N01-RC-45035</span>; Medical Research Council: <span class="grant">G0600705</span>; NCI NIH HHS: <span class="grant">N01-CN-45165</span>; NHLBI NIH HHS: <span class="grant">1-R01-HL087679-01</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">6</span>;<span class="pagerange">1129-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20015952">20015952</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2830824">2830824</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp561">10.1093/hmg/ddp561</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Specific replication origins promote DNA amplification in fission yeast.</h4>
<p class="authors"><span class="author">Kiang L</span>, <span class="author">Heichinger C</span>, <span class="author">Watt S</span>, <span class="author">B&auml;hler J</span> and <span class="author">Nurse P</span></p>
<p class="affiliation">Laboratory of Yeast Genetics and Cell Biology, The Rockefeller University, 1230 York Avenue, Box 5, New York, NY 10065, USA. <a href="mailto:%6C%6B%69%61%6E%67%40%72%6F%63%6B%65%66%65%6C%6C%65%72.%65%64%75">&#108;&#107;&#105;&#97;&#110;&#103;&#64;&#114;&#111;&#99;&#107;&#101;&#102;&#101;&#108;&#108;&#101;&#114;&#46;&#101;&#100;&#117;</a></p><p class="abstract">To ensure equal replication of the genome in every eukaryotic cell cycle, replication origins fire only once each S phase and do not fire after passive replication. Failure in these controls can lead to local amplification, contributing to genome instability and the development of cancer. To identify features of replication origins important for such amplification, we have investigated origin firing and local genome amplification in the presence of excess helicase loaders Cdc18 and Cdt1 in fission yeast. We find that S phase controls are attenuated and coordination of origin firing is lost, resulting in local amplification. Specific origins are necessary for amplification but act only within a permissive chromosomal context. Origins associated with amplification are highly AT-rich, fire efficiently and early during mitotic S phase, and are located in large intergenic regions. We propose that these features predispose replication origins to re-fire within a single S phase, or to remain active after passive replication.</p>
<p class="grants">Funded by: Cancer Research UK; NIGMS NIH HHS: <span class="grant">GM07739</span></p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2010</span>;<span class="volume">123</span>;<span class="number">Pt 18</span>;<span class="pagerange">3047-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20736315" rel="external">20736315</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2931602" rel="external">2931602</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fjcs.067470" rel="external">10.1242/jcs.067470</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe.</h4>
<p class="authors"><span class="author">Kim DU</span>, <span class="author">Hayles J</span>, <span class="author">Kim D</span>, <span class="author">Wood V</span>, <span class="author">Park HO</span>, <span class="author">Won M</span>, <span class="author">Yoo HS</span>, <span class="author">Duhig T</span>, <span class="author">Nam M</span>, <span class="author">Palmer G</span>, <span class="author">Han S</span>, <span class="author">Jeffery L</span>, <span class="author">Baek ST</span>, <span class="author">Lee H</span>, <span class="author">Shim YS</span>, <span class="author">Lee M</span>, <span class="author">Kim L</span>, <span class="author">Heo KS</span>, <span class="author">Noh EJ</span>, <span class="author">Lee AR</span>, <span class="author">Jang YJ</span>, <span class="author">Chung KS</span>, <span class="author">Choi SJ</span>, <span class="author">Park JY</span>, <span class="author">Park Y</span>, <span class="author">Kim HM</span>, <span class="author">Park SK</span>, <span class="author">Park HJ</span>, <span class="author">Kang EJ</span>, <span class="author">Kim HB</span>, <span class="author">Kang HS</span>, <span class="author">Park HM</span>, <span class="author">Kim K</span>, <span class="author">Song K</span>, <span class="author">Song KB</span>, <span class="author">Nurse P</span> and <span class="author">Hoe KL</span></p>
<p class="affiliation">Integrative Omics Research Centre, Korea Research Institute of Bioscience and Biotechnology, Yuseong, Daejeon, Korea.</p><p class="abstract">We report the construction and analysis of 4,836 heterozygous diploid deletion mutants covering 98.4% of the fission yeast genome providing a tool for studying eukaryotic biology. Comprehensive gene dispensability comparisons with budding yeast--the only other eukaryote for which a comprehensive knockout library exists--revealed that 83% of single-copy orthologs in the two yeasts had conserved dispensability. Gene dispensability differed for certain pathways between the two yeasts, including mitochondrial translation and cell cycle checkpoint control. We show that fission yeast has more essential genes than budding yeast and that essential genes are more likely than nonessential genes to be present in a single copy, to be broadly conserved and to contain introns. Growth fitness analyses determined sets of haploinsufficient and haploproficient genes for fission yeast, and comparisons with budding yeast identified specific ribosomal proteins and RNA polymerase subunits, which may act more generally to regulate eukaryotic cell growth.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust: <span class="grant">093917</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2010</span>;<span class="volume">28</span>;<span class="number">6</span>;<span class="pagerange">617-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20473289">20473289</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.1628">10.1038/nbt.1628</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss of NPC1 function in a patient with a co-inherited novel insulin receptor mutation does not grossly modify the severity of the associated insulin resistance.</h4>
<p class="authors"><span class="author">Kirk J</span>, <span class="author">Porter KM</span>, <span class="author">Parker V</span>, <span class="author">Barroso I</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Hendriksz C</span> and <span class="author">Semple RK</span></p>
<p class="affiliation">Department of Endocrinology, Birmingham Children&#39;s Hospital, Steelhouse Lane, Birmingham B4 6NH, United Kingdom.</p><p class="abstract">In Npc1 null mice, a model for Niemann Pick Disease Type C1, it has been reported that hepatocyte insulin receptor function is significantly impaired, consistent with growing evidence that membrane fluidity and microdomain structure have an important role in insulin signal transduction. However, whether insulin receptor function is also compromised in human Niemann Pick disease Type C1 is unclear. We now report a girl who developed progressive dementia, ataxia and opthalmoplegia from 9 years old, followed by severe acanthosis nigricans, hirsutism and acne at 11 years old. She was diagnosed with Niemann Pick Disease type C1 (OMIM#257220) based on positive filipin staining and reduced cholesterol-esterifying activity in dermal fibroblasts, and homozygosity for the p.Ile1061Thr NPC1 mutation. Further analysis revealed her also to be heterozygous for a novel trinucleotide deletion (c.3659 + 1_3659 + 3delGTG) at the end of exon 20 of INSR, encoding the insulin receptor, leading to deletion of Trp1193 in the intracellular tyrosine kinase domain. INSR mRNA and protein levels were normal in dermal fibroblasts, consistent with a primary signal transduction defect in the mutant receptor. Although the proband was significantly more insulin resistant than her father, who carried the INSR mutation but was only heterozygous for the NPC1 variant, their respective degrees of IR were very similar to those previously reported in a father-daughter pair with the closely related p.Trp1193Leu INSR mutation. This suggests that loss of NPC1 function, with attendant changes in membrane cholesterol composition, does not significantly modify the IR phenotype, even in the context of severely impaired INSR function.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">078986</span>, <span class="grant">078986/Z/06/Z</span>, <span class="grant">080952</span>, <span class="grant">080952/Z/06/Z</span></p>
<p class="publication"><span class="title">Journal of inherited metabolic disease</span>&nbsp;<span class="year">2010</span>;<span class="volume">33 Suppl 3</span>;<span class="pagerange">S227-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20521171">20521171</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3757264">3757264</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10545-010-9107-5">10.1007/s10545-010-9107-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of networks of co-occurring, tumor-related DNA copy number changes using a genome-wide scoring approach.</h4>
<p class="authors"><span class="author">Klijn C</span>, <span class="author">Bot J</span>, <span class="author">Adams DJ</span>, <span class="author">Reinders M</span>, <span class="author">Wessels L</span> and <span class="author">Jonkers J</span></p>
<p class="affiliation">Division of Molecular Biology, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</p><p class="abstract">Tumorigenesis is a multi-step process in which normal cells transform into malignant tumors following the accumulation of genetic mutations that enable them to evade the growth control checkpoints that would normally suppress their growth or result in apoptosis. It is therefore important to identify those combinations of mutations that collaborate in cancer development and progression. DNA copy number alterations (CNAs) are one of the ways in which cancer genes are deregulated in tumor cells. We hypothesized that synergistic interactions between cancer genes might be identified by looking for regions of co-occurring gain and/or loss. To this end we developed a scoring framework to separate truly co-occurring aberrations from passenger mutations and dominant single signals present in the data. The resulting regions of high co-occurrence can be investigated for between-region functional interactions. Analysis of high-resolution DNA copy number data from a panel of 95 hematological tumor cell lines correctly identified co-occurring recombinations at the T-cell receptor and immunoglobulin loci in T- and B-cell malignancies, respectively, showing that we can recover truly co-occurring genomic alterations. In addition, our analysis revealed networks of co-occurring genomic losses and gains that are enriched for cancer genes. These networks are also highly enriched for functional relationships between genes. We further examine sub-networks of these networks, core networks, which contain many known cancer genes. The core network for co-occurring DNA losses we find seems to be independent of the canonical cancer genes within the network. Our findings suggest that large-scale, low-intensity copy number alterations may be an important feature of cancer development or maintenance by affecting gene dosage of a large interconnected network of functionally related genes.</p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">e1000631</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20052266">20052266</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2791203">2791203</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.1000631">10.1371/journal.pcbi.1000631</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">AnnoTrack--a tracking system for genome annotation.</h4>
<p class="authors"><span class="author">Kokocinski F</span>, <span class="author">Harrow J</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">Vertebrate Genome Analysis, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB101HH, UK. <a href="mailto:%66%73%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#115;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: As genome sequences are determined for increasing numbers of model organisms, demand has grown for better tools to facilitate unified genome annotation efforts by communities of biologists. Typically this process involves numerous experts from the field and the use of data from dispersed sources as evidence. This kind of collaborative annotation project requires specialized software solutions for efficient data tracking and processing.</p><p class="abstract"><strong>Results</strong>: As part of the scale-up phase of the ENCODE project (Encyclopedia of DNA Elements), the aim of the GENCODE project is to produce a highly accurate evidence-based reference gene annotation for the human genome. The AnnoTrack software system was developed to aid this effort. It integrates data from multiple distributed sources, highlights conflicts and facilitates the quick identification, prioritisation and resolution of problems during the process of genome annotation.</p><p class="abstract"><strong>Conclusions</strong>: AnnoTrack has been in use for the last year and has proven a very valuable tool for large-scale genome annotation. Designed to interface with standard bioinformatics components, such as DAS servers and Ensembl databases, it is easy to setup and configure for different genome projects. The source code is available at http://annotrack.sanger.ac.uk.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5U54HG004555</span>; Wellcome Trust: <span class="grant">077198</span>, <span class="grant">WT077198/Z/05/Z</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">538</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20923551">20923551</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3091687">3091687</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-538">10.1186/1471-2164-11-538</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Slingshot: a PiggyBac based transposon system for tamoxifen-inducible &#39;self-inactivating&#39; insertional mutagenesis.</h4>
<p class="authors"><span class="author">Kong J</span>, <span class="author">Wang F</span>, <span class="author">Brenton JD</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Welcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We have developed a self-inactivating PiggyBac transposon system for tamoxifen inducible insertional mutagenesis from a stably integrated chromosomal donor. This system, which we have named &#39;Slingshot&#39;, utilizes a transposon carrying elements for both gain- and loss-of-function screens in vitro. We show that the Slingshot transposon can be efficiently mobilized from a range of chromosomal loci with high inducibility and low background generating insertions that are randomly dispersed throughout the genome. Furthermore, we show that once the Slingshot transposon has been mobilized it is not remobilized producing stable clonal integrants in all daughter cells. To illustrate the efficacy of Slingshot as a screening tool we set out to identify mediators of resistance to puromycin and the chemotherapeutic drug vincristine by performing genetrap screens in mouse embryonic stem cells. From these genome-wide screens we identified multiple independent insertions in the multidrug resistance transporter genes Abcb1a/b and Abcg2 conferring resistance to drug treatment. Importantly, we also show that the Slingshot transposon system is functional in other mammalian cell lines such as human HEK293, OVCAR-3 and PE01 cells suggesting that it may be used in a range of cell culture systems. Slingshot represents a flexible and potent system for genome-wide transposon-mediated mutagenesis with many potential applications.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">18</span>;<span class="pagerange">e173</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20688953">20688953</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2952874">2952874</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq658">10.1093/nar/gkq658</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insertional mutagenesis in mice deficient for p15Ink4b, p16Ink4a, p21Cip1, and p27Kip1 reveals cancer gene interactions and correlations with tumor phenotypes.</h4>
<p class="authors"><span class="author">Kool J</span>, <span class="author">Uren AG</span>, <span class="author">Martins CP</span>, <span class="author">Sie D</span>, <span class="author">de Ridder J</span>, <span class="author">Turner G</span>, <span class="author">van Uitert M</span>, <span class="author">Matentzoglu K</span>, <span class="author">Lagcher W</span>, <span class="author">Krimpenfort P</span>, <span class="author">Gadiot J</span>, <span class="author">Pritchard C</span>, <span class="author">Lenz J</span>, <span class="author">Lund AH</span>, <span class="author">Jonkers J</span>, <span class="author">Rogers J</span>, <span class="author">Adams DJ</span>, <span class="author">Wessels L</span>, <span class="author">Berns A</span> and <span class="author">van Lohuizen M</span></p>
<p class="affiliation">Division of Molecular Genetics, The Centre of Biomedical Genetics, Academic Medical Center and Cancer Genomics Centre, Netherlands Cancer Institute, 1066CX, Amsterdam, the Netherlands.</p><p class="abstract">The cyclin dependent kinase (CDK) inhibitors p15, p16, p21, and p27 are frequently deleted, silenced, or downregulated in many malignancies. Inactivation of CDK inhibitors predisposes mice to tumor development, showing that these genes function as tumor suppressors. Here, we describe high-throughput murine leukemia virus insertional mutagenesis screens in mice that are deficient for one or two CDK inhibitors. We retrieved 9,117 retroviral insertions from 476 lymphomas to define hundreds of loci that are mutated more frequently than expected by chance. Many of these loci are skewed toward a specific genetic context of predisposing germline and somatic mutations. We also found associations between these loci with gender, age of tumor onset, and lymphocyte lineage (B or T cell). Comparison of retroviral insertion sites with single nucleotide polymorphisms associated with chronic lymphocytic leukemia revealed a significant overlap between the datasets. Together, our findings highlight the importance of genetic context within large-scale mutation detection studies, and they show a novel use for insertional mutagenesis data in prioritizing disease-associated genes that emerge from genome-wide association studies.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6997</span>, <span class="grant">A8784</span>; Wellcome Trust: <span class="grant">082356</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2010</span>;<span class="volume">70</span>;<span class="number">2</span>;<span class="pagerange">520-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20068150">20068150</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2875110">2875110</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-09-2736">10.1158/0008-5472.CAN-09-2736</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microindel detection in short-read sequence data.</h4>
<p class="authors"><span class="author">Krawitz P</span>, <span class="author">R&ouml;delsperger C</span>, <span class="author">J&auml;ger M</span>, <span class="author">Jostins L</span>, <span class="author">Bauer S</span> and <span class="author">Robinson PN</span></p>
<p class="affiliation">Institute for Medical Genetics, Charit&eacute;-Universit&auml;tsmedizin Berlin, 13353 Berlin. <a href="mailto:%70%65%74%65%72.%6B%72%61%77%69%74%7A%40%67%6F%6F%67%6C%65%6D%61%69%6C.%63%6F%6D">&#112;&#101;&#116;&#101;&#114;&#46;&#107;&#114;&#97;&#119;&#105;&#116;&#122;&#64;&#103;&#111;&#111;&#103;&#108;&#101;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Motivation</strong>: Several recent studies have demonstrated the effectiveness of resequencing and single nucleotide variant (SNV) detection by deep short-read sequencing platforms. While several reliable algorithms are available for automated SNV detection, the automated detection of microindels in deep short-read data presents a new bioinformatics challenge.</p><p class="abstract"><strong>Results</strong>: We systematically analyzed how the short-read mapping tools MAQ, Bowtie, Burrows-Wheeler alignment tool (BWA), Novoalign and RazerS perform on simulated datasets that contain indels and evaluated how indels affect error rates in SNV detection. We implemented a simple algorithm to compute the equivalent indel region eir, which can be used to process the alignments produced by the mapping tools in order to perform indel calling. Using simulated data that contains indels, we demonstrate that indel detection works well on short-read data: the detection rate for microindels (&lt;4 bp) is &gt;90%. Our study provides insights into systematic errors in SNV detection that is based on ungapped short sequence read alignments. Gapped alignments of short sequence reads can be used to reduce this error and to detect microindels in simulated short-read data. A comparison with microindels automatically identified on the ABI Sanger and Roche 454 platform indicates that microindel detection from short sequence reads identifies both overlapping and distinct indels.</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%70%65%74%65%72.%6B%72%61%77%69%74%7A%40%67%6F%6F%67%6C%65%6D%61%69%6C.%63%6F%6D">&#112;&#101;&#116;&#101;&#114;&#46;&#107;&#114;&#97;&#119;&#105;&#116;&#122;&#64;&#103;&#111;&#111;&#103;&#108;&#101;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a>; <a href="mailto:%70%65%74%65%72.%72%6F%62%69%6E%73%6F%6E%40%63%68%61%72%69%74%65.%64%65">&#112;&#101;&#116;&#101;&#114;&#46;&#114;&#111;&#98;&#105;&#110;&#115;&#111;&#110;&#64;&#99;&#104;&#97;&#114;&#105;&#116;&#101;&#46;&#100;&#101;</a></p><p class="abstract">Supplementary data are available at Bioinformatics online.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">6</span>;<span class="pagerange">722-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20144947">20144947</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq027">10.1093/bioinformatics/btq027</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of JAG1 with bone mineral density and osteoporotic fractures: a genome-wide association study and follow-up replication studies.</h4>
<p class="authors"><span class="author">Kung AW</span>, <span class="author">Xiao SM</span>, <span class="author">Cherny S</span>, <span class="author">Li GH</span>, <span class="author">Gao Y</span>, <span class="author">Tso G</span>, <span class="author">Lau KS</span>, <span class="author">Luk KD</span>, <span class="author">Liu JM</span>, <span class="author">Cui B</span>, <span class="author">Zhang MJ</span>, <span class="author">Zhang ZL</span>, <span class="author">He JW</span>, <span class="author">Yue H</span>, <span class="author">Xia WB</span>, <span class="author">Luo LM</span>, <span class="author">He SL</span>, <span class="author">Kiel DP</span>, <span class="author">Karasik D</span>, <span class="author">Hsu YH</span>, <span class="author">Cupples LA</span>, <span class="author">Demissie S</span>, <span class="author">Styrkarsdottir U</span>, <span class="author">Halldorsson BV</span>, <span class="author">Sigurdsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">Richards JB</span>, <span class="author">Zhai G</span>, <span class="author">Soranzo N</span>, <span class="author">Valdes A</span>, <span class="author">Spector TD</span> and <span class="author">Sham PC</span></p>
<p class="affiliation">Department of Medicine, Research Centre of Heart, Brain, Hormone &amp; Healthy Aging, Faculty of Medicine, The University of Hong Kong, Hong Kong, China. <a href="mailto:%61%77%63%6B%75%6E%67%40%68%6B%75%63%63.%68%6B%75.%68%6B">&#97;&#119;&#99;&#107;&#117;&#110;&#103;&#64;&#104;&#107;&#117;&#99;&#99;&#46;&#104;&#107;&#117;&#46;&#104;&#107;</a></p><p class="abstract">Bone mineral density (BMD), a diagnostic parameter for osteoporosis and a clinical predictor of fracture, is a polygenic trait with high heritability. To identify genetic variants that influence BMD in different ethnic groups, we performed a genome-wide association study (GWAS) on 800 unrelated Southern Chinese women with extreme BMD and carried out follow-up replication studies in six independent study populations of European descent and Asian populations including 18,098 subjects. In the meta-analysis, rs2273061 of the Jagged1 (JAG1) gene was associated with high BMD (p = 5.27 x 10(-8) for lumbar spine [LS] and p = 4.15 x 10(-5) for femoral neck [FN], n = 18,898). This SNP was further found to be associated with the low risk of osteoporotic fracture (p = 0.009, OR = 0.7, 95% CI 0.57-0.93, n = 1881). Region-wide and haplotype analysis showed that the strongest association evidence was from the linkage disequilibrium block 5, which included rs2273061 of the JAG1 gene (p = 8.52 x 10(-9) for LS and 3.47 x 10(-5) at FN). To assess the function of identified variants, an electrophoretic mobility shift assay demonstrated the binding of c-Myc to the &quot;G&quot; but not &quot;A&quot; allele of rs2273061. A mRNA expression study in both human bone-derived cells and peripheral blood mononuclear cells confirmed association of the high BMD-related allele G of rs2273061 with higher JAG1 expression. Our results identify the JAG1 gene as a candidate for BMD regulation in different ethnic groups, and it is a potential key factor for fracture pathogenesis.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">(N02-HL-64278</span>, <span class="grant">N01-HC-25195</span>; NIA NIH HHS: <span class="grant">R01 AR/AG 41398</span>; NIAMS NIH HHS: <span class="grant">R01 AR 050066</span>, <span class="grant">R01 AR050066-04</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">86</span>;<span class="number">2</span>;<span class="pagerange">229-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20096396" rel="external">20096396</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2820171" rel="external">2820171</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2009.12.014" rel="external">10.1016/j.ajhg.2009.12.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular analysis of tumor-promoting CD8+ T cells in two-stage cutaneous chemical carcinogenesis.</h4>
<p class="authors"><span class="author">Kwong BY</span>, <span class="author">Roberts SJ</span>, <span class="author">Silberzahn T</span>, <span class="author">Filler RB</span>, <span class="author">Neustadter JH</span>, <span class="author">Galan A</span>, <span class="author">Reddy S</span>, <span class="author">Lin WM</span>, <span class="author">Ellis PD</span>, <span class="author">Langford CF</span>, <span class="author">Hayday AC</span> and <span class="author">Girardi M</span></p>
<p class="affiliation">Department of Dermatology and Skin Diseases Research Center, Yale University School of Medicine, 333 Cedar Street, New Haven, CT 06520, USA.</p><p class="abstract">T-pro are tumor-infiltrating TCRalphabeta(+)CD8(+) cells of reduced cytotoxic potential that promote experimental two-stage chemical cutaneous carcinogenesis. Toward understanding their mechanism of action, this study uses whole-genome expression analysis to compare T-pro with systemic CD8(+) T cells from multiple groups of tumor-bearing mice. T-pro show an overt T helper 17-like profile (high retinoic acid-related orphan receptor-(ROR)gammat, IL-17A, IL-17F; low T-bet and eomesodermin), regulatory potential (high FoxP3, IL-10, Tim-3), and transcripts encoding epithelial growth factors (amphiregulin, Gro-1, Gro-2). Tricolor flow cytometry subsequently confirmed the presence of TCRbeta(+) CD8(+) IL-17(+) T cells among tumor-infiltrating lymphocytes (TILs). Moreover, a time-course analysis of independent TIL isolates from papillomas versus carcinomas exposed a clear association of the &quot;T-pro phenotype&quot; with malignant progression. This molecular characterization of T-pro builds a foundation for elucidating the contributions of inflammation to cutaneous carcinogenesis, and may provide useful biomarkers for cancer immunotherapy in which the widely advocated use of tumor-specific CD8(+) cytolytic T cells should perhaps accommodate the cells&#39; potential corruption toward the T-pro phenotype. The data are also likely germane to psoriasis, in which the epidermis may be infiltrated by CD8(+) IL-17-producing T cells.</p>
<p class="grants">Funded by: Medical Research Council; NCI NIH HHS: <span class="grant">P50 CA121974</span>, <span class="grant">P50 CA121974-01</span>, <span class="grant">R01 CA102703</span>, <span class="grant">R01 CA102703-05</span>; NIAMS NIH HHS: <span class="grant">P30 AR041942-07</span>, <span class="grant">P30 AR41942</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of investigative dermatology</span>&nbsp;<span class="year">2010</span>;<span class="volume">130</span>;<span class="number">6</span>;<span class="pagerange">1726-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19924136" rel="external">19924136</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2920801" rel="external">2920801</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fjid.2009.362" rel="external">10.1038/jid.2009.362</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting Report from the Genomic Standards Consortium (GSC) Workshop 8.</h4>
<p class="authors"><span class="author">Kyrpides N</span>, <span class="author">Field D</span>, <span class="author">Sterk P</span>, <span class="author">Kottmann R</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Hirschman L</span>, <span class="author">Garrity GM</span>, <span class="author">Cochrane G</span> and <span class="author">Wooley J</span></p><p class="abstract">This report summarizes the proceedings of the 8th meeting of the Genomic Standards Consortium held at the Department of Energy Joint Genome Institute in Walnut Creek, CA, USA on September 9-11, 2009. This three-day workshop marked the maturing of Genomic Standards Consortium from an informal gathering of researchers interested in developing standards in the field of genomic and metagenomics to an established community with a defined governance mechanism, its own open access journal, and a family of established standards for describing genomes, metagenomes and marker studies (i.e. ribosomal RNA gene surveys). There will be increased efforts within the GSC to reach out to the wider scientific community via a range of new projects. Further information about the GSC and its activities can be found at http://gensc.org/.</p>
<p class="publication"><span class="title">Standards in genomic sciences</span>&nbsp;<span class="year">2010</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">93-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21304696" rel="external">21304696</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3035271" rel="external">3035271</a>; DOI: <a href="http://dx.doi.org/10.4056%2Fsigs.1022942" rel="external">10.4056/sigs.1022942</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Jarid2 is a PRC2 component in embryonic stem cells required for multi-lineage differentiation and recruitment of PRC1 and RNA Polymerase II to developmental regulators.</h4>
<p class="authors"><span class="author">Landeira D</span>, <span class="author">Sauer S</span>, <span class="author">Poot R</span>, <span class="author">Dvorkina M</span>, <span class="author">Mazzarella L</span>, <span class="author">J&oslash;rgensen HF</span>, <span class="author">Pereira CF</span>, <span class="author">Leleu M</span>, <span class="author">Piccolo FM</span>, <span class="author">Spivakov M</span>, <span class="author">Brookes E</span>, <span class="author">Pombo A</span>, <span class="author">Fisher C</span>, <span class="author">Skarnes WC</span>, <span class="author">Snoek T</span>, <span class="author">Bezstarosti K</span>, <span class="author">Demmers J</span>, <span class="author">Klose RJ</span>, <span class="author">Casanova M</span>, <span class="author">Tavares L</span>, <span class="author">Brockdorff N</span>, <span class="author">Merkenschlager M</span> and <span class="author">Fisher AG</span></p>
<p class="affiliation">Lymphocyte Development Group, MRC Clinical Sciences Centre, Imperial College School of Medicine, Hammersmith Hospital Campus, Du Cane Road, London, W12 0NN UK.</p><p class="abstract">Polycomb Repressor Complexes (PRCs) are important regulators of embryogenesis. In embryonic stem (ES) cells many genes that regulate subsequent stages in development are enriched at their promoters for PRC1, PRC2 and Ser 5-phosphorylated RNA Polymerase II (RNAP), and contain domains of &#39;bivalent&#39; chromatin (enriched for H3K4me3; histone H3 di- or trimethylated at Lys 4 and H3K27me3; histone H3 trimethylated at Lys 27). Loss of individual PRC components in ES cells can lead to gene de-repression and to unscheduled differentiation. Here we show that Jarid2 is a novel subunit of PRC2 that is required for the co-recruitment of PRC1 and RNAP to genes that regulate development in ES cells. Jarid2-deficient ES cells showed reduced H3K4me2/me3 and H3K27me3 marking and PRC1/PRC2 recruitment, and did not efficiently establish Ser 5-phosporylated RNAP at target genes. ES cells lacking Jarid2, in contrast to previously characterized PRC1 and PRC2 mutants, did not inappropriately express PRC2 target genes. Instead, they show a severely compromised capacity for successful differentiation towards neural or mesodermal fates and failed to correctly initiate lineage-specific gene expression in vitro. Collectively, these data indicate that transcriptional priming of bivalent genes in pluripotent ES cells is Jarid2-dependent, and suggests that priming is critical for subsequent multi-lineage differentiation.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">Nature cell biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">618-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20473294">20473294</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fncb2065">10.1038/ncb2065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hundreds of variants clustered in genomic loci and biological pathways affect human height.</h4>
<p class="authors"><span class="author">Lango Allen H</span>, <span class="author">Estrada K</span>, <span class="author">Lettre G</span>, <span class="author">Berndt SI</span>, <span class="author">Weedon MN</span>, <span class="author">Rivadeneira F</span>, <span class="author">Willer CJ</span>, <span class="author">Jackson AU</span>, <span class="author">Vedantam S</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Ferreira T</span>, <span class="author">Wood AR</span>, <span class="author">Weyant RJ</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">Speliotes EK</span>, <span class="author">Wheeler E</span>, <span class="author">Soranzo N</span>, <span class="author">Park JH</span>, <span class="author">Yang J</span>, <span class="author">Gudbjartsson D</span>, <span class="author">Heard-Costa NL</span>, <span class="author">Randall JC</span>, <span class="author">Qi L</span>, <span class="author">Vernon Smith A</span>, <span class="author">M&auml;gi R</span>, <span class="author">Pastinen T</span>, <span class="author">Liang L</span>, <span class="author">Heid IM</span>, <span class="author">Luan J</span>, <span class="author">Thorleifsson G</span>, <span class="author">Winkler TW</span>, <span class="author">Goddard ME</span>, <span class="author">Sin Lo K</span>, <span class="author">Palmer C</span>, <span class="author">Workalemahu T</span>, <span class="author">Aulchenko YS</span>, <span class="author">Johansson A</span>, <span class="author">Zillikens MC</span>, <span class="author">Feitosa MF</span>, <span class="author">Esko T</span>, <span class="author">Johnson T</span>, <span class="author">Ketkar S</span>, <span class="author">Kraft P</span>, <span class="author">Mangino M</span>, <span class="author">Prokopenko I</span>, <span class="author">Absher D</span>, <span class="author">Albrecht E</span>, <span class="author">Ernst F</span>, <span class="author">Glazer NL</span>, <span class="author">Hayward C</span>, <span class="author">Hottenga JJ</span>, <span class="author">Jacobs KB</span>, <span class="author">Knowles JW</span>, <span class="author">Kutalik Z</span>, <span class="author">Monda KL</span>, <span class="author">Polasek O</span>, <span class="author">Preuss M</span>, <span class="author">Rayner NW</span>, <span class="author">Robertson NR</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Tyrer JP</span>, <span class="author">Voight BF</span>, <span class="author">Wiklund F</span>, <span class="author">Xu J</span>, <span class="author">Zhao JH</span>, <span class="author">Nyholt DR</span>, <span class="author">Pellikka N</span>, <span class="author">Perola M</span>, <span class="author">Perry JR</span>, <span class="author">Surakka I</span>, <span class="author">Tammesoo ML</span>, <span class="author">Altmaier EL</span>, <span class="author">Amin N</span>, <span class="author">Aspelund T</span>, <span class="author">Bhangale T</span>, <span class="author">Boucher G</span>, <span class="author">Chasman DI</span>, <span class="author">Chen C</span>, <span class="author">Coin L</span>, <span class="author">Cooper MN</span>, <span class="author">Dixon AL</span>, <span class="author">Gibson Q</span>, <span class="author">Grundberg E</span>, <span class="author">Hao K</span>, <span class="author">Juhani Junttila M</span>, <span class="author">Kaplan LM</span>, <span class="author">Kettunen J</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Kwan T</span>, <span class="author">Lawrence RW</span>, <span class="author">Levinson DF</span>, <span class="author">Lorentzon M</span>, <span class="author">McKnight B</span>, <span class="author">Morris AP</span>, <span class="author">M&uuml;ller M</span>, <span class="author">Suh Ngwa J</span>, <span class="author">Purcell S</span>, <span class="author">Rafelt S</span>, <span class="author">Salem RM</span>, <span class="author">Salvi E</span>, <span class="author">Sanna S</span>, <span class="author">Shi J</span>, <span class="author">Sovio U</span>, <span class="author">Thompson JR</span>, <span class="author">Turchin MC</span>, <span class="author">Vandenput L</span>, <span class="author">Verlaan DJ</span>, <span class="author">Vitart V</span>, <span class="author">White CC</span>, <span class="author">Ziegler A</span>, <span class="author">Almgren P</span>, <span class="author">Balmforth AJ</span>, <span class="author">Campbell H</span>, <span class="author">Citterio L</span>, <span class="author">De Grandi A</span>, <span class="author">Dominiczak A</span>, <span class="author">Duan J</span>, <span class="author">Elliott P</span>, <span class="author">Elosua R</span>, <span class="author">Eriksson JG</span>, <span class="author">Freimer NB</span>, <span class="author">Geus EJ</span>, <span class="author">Glorioso N</span>, <span class="author">Haiqing S</span>, <span class="author">Hartikainen AL</span>, <span class="author">Havulinna AS</span>, <span class="author">Hicks AA</span>, <span class="author">Hui J</span>, <span class="author">Igl W</span>, <span class="author">Illig T</span>, <span class="author">Jula A</span>, <span class="author">Kajantie E</span>, <span class="author">Kilpel&auml;inen TO</span>, <span class="author">Koiranen M</span>, <span class="author">Kolcic I</span>, <span class="author">Koskinen S</span>, <span class="author">Kovacs P</span>, <span class="author">Laitinen J</span>, <span class="author">Liu J</span>, <span class="author">Lokki ML</span>, <span class="author">Marusic A</span>, <span class="author">Maschio A</span>, <span class="author">Meitinger T</span>, <span class="author">Mulas A</span>, <span class="author">Par&eacute; G</span>, <span class="author">Parker AN</span>, <span class="author">Peden JF</span>, <span class="author">Petersmann A</span>, <span class="author">Pichler I</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Pouta A</span>, <span class="author">Ridderstr&aring;le M</span>, <span class="author">Rotter JI</span>, <span class="author">Sambrook JG</span>, <span class="author">Sanders AR</span>, <span class="author">Schmidt CO</span>, <span class="author">Sinisalo J</span>, <span class="author">Smit JH</span>, <span class="author">Stringham HM</span>, <span class="author">Bragi Walters G</span>, <span class="author">Widen E</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Zagato L</span>, <span class="author">Zgaga L</span>, <span class="author">Zitting P</span>, <span class="author">Alavere H</span>, <span class="author">Farrall M</span>, <span class="author">McArdle WL</span>, <span class="author">Nelis M</span>, <span class="author">Peters MJ</span>, <span class="author">Ripatti S</span>, <span class="author">van Meurs JB</span>, <span class="author">Aben KK</span>, <span class="author">Ardlie KG</span>, <span class="author">Beckmann JS</span>, <span class="author">Beilby JP</span>, <span class="author">Bergman RN</span>, <span class="author">Bergmann S</span>, <span class="author">Collins FS</span>, <span class="author">Cusi D</span>, <span class="author">den Heijer M</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Gejman PV</span>, <span class="author">Hall AS</span>, <span class="author">Hamsten A</span>, <span class="author">Huikuri HV</span>, <span class="author">Iribarren C</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kaprio J</span>, <span class="author">Kathiresan S</span>, <span class="author">Kiemeney L</span>, <span class="author">Kocher T</span>, <span class="author">Launer LJ</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Melander O</span>, <span class="author">Mosley TH</span>, <span class="author">Musk AW</span>, <span class="author">Nieminen MS</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Ohlsson C</span>, <span class="author">Oostra B</span>, <span class="author">Palmer LJ</span>, <span class="author">Raitakari O</span>, <span class="author">Ridker PM</span>, <span class="author">Rioux JD</span>, <span class="author">Rissanen A</span>, <span class="author">Rivolta C</span>, <span class="author">Schunkert H</span>, <span class="author">Shuldiner AR</span>, <span class="author">Siscovick DS</span>, <span class="author">Stumvoll M</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Tuomilehto J</span>, <span class="author">van Ommen GJ</span>, <span class="author">Viikari J</span>, <span class="author">Heath AC</span>, <span class="author">Martin NG</span>, <span class="author">Montgomery GW</span>, <span class="author">Province MA</span>, <span class="author">Kayser M</span>, <span class="author">Arnold AM</span>, <span class="author">Atwood LD</span>, <span class="author">Boerwinkle E</span>, <span class="author">Chanock SJ</span>, <span class="author">Deloukas P</span>, <span class="author">Gieger C</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Hall P</span>, <span class="author">Hattersley AT</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hoffman W</span>, <span class="author">Lathrop GM</span>, <span class="author">Salomaa V</span>, <span class="author">Schreiber S</span>, <span class="author">Uda M</span>, <span class="author">Waterworth D</span>, <span class="author">Wright AF</span>, <span class="author">Assimes TL</span>, <span class="author">Barroso I</span>, <span class="author">Hofman A</span>, <span class="author">Mohlke KL</span>, <span class="author">Boomsma DI</span>, <span class="author">Caulfield MJ</span>, <span class="author">Cupples LA</span>, <span class="author">Erdmann J</span>, <span class="author">Fox CS</span>, <span class="author">Gudnason V</span>, <span class="author">Gyllensten U</span>, <span class="author">Harris TB</span>, <span class="author">Hayes RB</span>, <span class="author">Jarvelin MR</span>, <span class="author">Mooser V</span>, <span class="author">Munroe PB</span>, <span class="author">Ouwehand WH</span>, <span class="author">Penninx BW</span>, <span class="author">Pramstaller PP</span>, <span class="author">Quertermous T</span>, <span class="author">Rudan I</span>, <span class="author">Samani NJ</span>, <span class="author">Spector TD</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Watkins H</span>, <span class="author">Wilson JF</span>, <span class="author">Groop LC</span>, <span class="author">Haritunians T</span>, <span class="author">Hu FB</span>, <span class="author">Kaplan RC</span>, <span class="author">Metspalu A</span>, <span class="author">North KE</span>, <span class="author">Schlessinger D</span>, <span class="author">Wareham NJ</span>, <span class="author">Hunter DJ</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Strachan DP</span>, <span class="author">Wichmann HE</span>, <span class="author">Borecki IB</span>, <span class="author">van Duijn CM</span>, <span class="author">Schadt EE</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Peltonen L</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Visscher PM</span>, <span class="author">Chatterjee N</span>, <span class="author">Loos RJ</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span>, <span class="author">Ingelsson E</span>, <span class="author">Lindgren CM</span>, <span class="author">Abecasis GR</span>, <span class="author">Stefansson K</span>, <span class="author">Frayling TM</span> and <span class="author">Hirschhorn JN</span></p>
<p class="affiliation">Genetics of Complex Traits, Peninsula College of Medicine and Dentistry, University of Exeter, Exeter EX1 2LU, UK.</p><p class="abstract">Most common human traits and diseases have a polygenic pattern of inheritance: DNA sequence variants at many genetic loci influence the phenotype. Genome-wide association (GWA) studies have identified more than 600 variants associated with human traits, but these typically explain small fractions of phenotypic variation, raising questions about the use of further studies. Here, using 183,727 individuals, we show that hundreds of genetic variants, in at least 180 loci, influence adult height, a highly heritable and classic polygenic trait. The large number of loci reveals patterns with important implications for genetic studies of common human diseases and traits. First, the 180 loci are not random, but instead are enriched for genes that are connected in biological pathways (P = 0.016) and that underlie skeletal growth defects (P&thinsp;&lt;&thinsp;0.001). Second, the likely causal gene is often located near the most strongly associated variant: in 13 of 21 loci containing a known skeletal growth gene, that gene was closest to the associated variant. Third, at least 19 loci have multiple independently associated variants, suggesting that allelic heterogeneity is a frequent feature of polygenic traits, that comprehensive explorations of already-discovered loci should discover additional variants and that an appreciable fraction of associated loci may have been identified. Fourth, associated variants are enriched for likely functional effects on genes, being over-represented among variants that alter amino-acid structure of proteins and expression levels of nearby genes. Our data explain approximately 10% of the phenotypic variation in height, and we estimate that unidentified common variants of similar effect sizes would increase this figure to approximately 16% of phenotypic variation (approximately 20% of heritable variation). Although additional approaches are needed to dissect the genetic architecture of polygenic human traits fully, our findings indicate that GWA studies can identify large numbers of loci that implicate biologically relevant genes and pathways.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/02/128</span>, <span class="grant">PG/02/128/14470</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/276</span>, <span class="grant">CZB/4/279</span>, <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0000649</span>, <span class="grant">G0000934</span>, <span class="grant">G0500539</span>, <span class="grant">G0600331</span>, <span class="grant">G0600331(77796)</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010(63660)</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>; NCI NIH HHS: <span class="grant">CA047988</span>, <span class="grant">CA49449</span>, <span class="grant">CA50385</span>, <span class="grant">CA65725</span>, <span class="grant">CA67262</span>, <span class="grant">CA87969</span>, <span class="grant">P01 CA087969</span>, <span class="grant">P01 CA087969-12</span>, <span class="grant">R01 CA047988</span>, <span class="grant">R01 CA047988-20</span>, <span class="grant">R01 CA050385</span>, <span class="grant">R01 CA050385-20</span>, <span class="grant">R01 CA065725</span>, <span class="grant">R01 CA065725-14</span>, <span class="grant">R01 CA067262</span>, <span class="grant">R01 CA067262-14</span>, <span class="grant">R01 CA104021</span>, <span class="grant">R01 CA104021-02</span>, <span class="grant">U01 CA049449</span>, <span class="grant">U01 CA049449-21</span>, <span class="grant">U01 CA098233</span>, <span class="grant">U01 CA098233-08</span>, <span class="grant">U01-CA098233</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>, <span class="grant">U54-RR020278</span>, <span class="grant">UL1-RR025005</span>; NHGRI NIH HHS: <span class="grant">HG002651</span>, <span class="grant">HG005214</span>, <span class="grant">HG005581</span>, <span class="grant">R01 HG002651</span>, <span class="grant">R01 HG002651-05</span>, <span class="grant">RC2 HG005581</span>, <span class="grant">RC2 HG005581-02</span>, <span class="grant">T32-HG00040</span>, <span class="grant">U01 HG004399</span>, <span class="grant">U01 HG004399-02</span>, <span class="grant">U01 HG004402</span>, <span class="grant">U01 HG004402-02</span>, <span class="grant">U01 HG005214</span>, <span class="grant">U01 HG005214-02</span>, <span class="grant">U01-HG004399</span>, <span class="grant">U01-HG004402</span>, <span class="grant">Z01-HG000024</span>; NHLBI NIH HHS: <span class="grant">HL043851</span>, <span class="grant">HL084729</span>, <span class="grant">HL69757</span>, <span class="grant">HL71981</span>, <span class="grant">K99-HL094535</span>, <span class="grant">N01-HC15103</span>, <span class="grant">N01-HC25195</span>, <span class="grant">N01-HC35129</span>, <span class="grant">N01-HC45133</span>, <span class="grant">N01-HC55015</span>, <span class="grant">N01-HC55016</span>, <span class="grant">N01-HC55018</span>, <span class="grant">N01-HC55019</span>, <span class="grant">N01-HC55020</span>, <span class="grant">N01-HC55021</span>, <span class="grant">N01-HC55022</span>, <span class="grant">N01-HC55222</span>, <span class="grant">N01-HC75150</span>, <span class="grant">N01-HC85079</span>, <span class="grant">N01-HC85080</span>, <span class="grant">N01-HC85081</span>, <span class="grant">N01-HC85082</span>, <span class="grant">N01-HC85083</span>, <span class="grant">N01-HC85084</span>, <span class="grant">N01-HC85085</span>, <span class="grant">N01-HC85086</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL043851</span>, <span class="grant">R01 HL043851-10</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL059367-10</span>, <span class="grant">R01 HL071981</span>, <span class="grant">R01 HL071981-07</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL086694-02</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087641-01</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL087647-01</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL087652-01</span>, <span class="grant">R01 HL087676</span>, <span class="grant">R01 HL087676-01</span>, <span class="grant">R01 HL087679-01</span>, <span class="grant">R01 HL087700</span>, <span class="grant">R01 HL087700-03</span>, <span class="grant">R01 HL088119</span>, <span class="grant">R01 HL088119-01</span>, <span class="grant">R01-HL086694</span>, <span class="grant">R01-HL087641</span>, <span class="grant">R01-HL087647</span>, <span class="grant">R01-HL087652</span>, <span class="grant">R01-HL087676</span>, <span class="grant">R01-HL087679</span>, <span class="grant">R01-HL087700</span>, <span class="grant">R01-HL088119</span>, <span class="grant">R01-HL59367</span>, <span class="grant">U01 HL069757</span>, <span class="grant">U01 HL069757-10</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL080295</span>, <span class="grant">U01 HL080295-04</span>, <span class="grant">U01 HL084729</span>, <span class="grant">U01 HL084729-03</span>, <span class="grant">U01 HL084756</span>, <span class="grant">U01 HL084756-03</span>, <span class="grant">U01-HL080295</span>, <span class="grant">U01-HL084756</span>, <span class="grant">U01-HL72515</span>; NIA NIH HHS: <span class="grant">N01-AG12100</span>, <span class="grant">N01-AG12109</span>, <span class="grant">R01 AG031890-02</span>, <span class="grant">R01-AG031890</span>, <span class="grant">Z01-AG00675</span>, <span class="grant">Z01-AG007380</span>; NIAAA NIH HHS: <span class="grant">AA014041</span>, <span class="grant">AA07535</span>, <span class="grant">AA10248</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">K05 AA017688</span>, <span class="grant">R01 AA007535</span>, <span class="grant">R01 AA007535-08</span>, <span class="grant">R01 AA013320-04</span>, <span class="grant">R01 AA013321</span>, <span class="grant">R01 AA013321-05</span>, <span class="grant">R01 AA013326-05</span>, <span class="grant">R01 AA014041-05</span>; NIAMS NIH HHS: <span class="grant">K08 AR055688</span>, <span class="grant">K08 AR055688-03</span>, <span class="grant">K08 AR055688-04</span>, <span class="grant">K08-AR055688</span>; NIDA NIH HHS: <span class="grant">DA12854</span>, <span class="grant">R01 DA012854</span>, <span class="grant">R01 DA012854-09</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK063491</span>, <span class="grant">DK072193</span>, <span class="grant">DK079466</span>, <span class="grant">DK080145</span>, <span class="grant">DK46200</span>, <span class="grant">DK58845</span>, <span class="grant">F32 DK079466</span>, <span class="grant">F32 DK079466-01</span>, <span class="grant">K23 DK080145</span>, <span class="grant">K23 DK080145-01</span>, <span class="grant">K23-DK080145</span>, <span class="grant">P30 DK072488</span>, <span class="grant">R01 DK058845</span>, <span class="grant">R01 DK058845-11</span>, <span class="grant">R01 DK068336</span>, <span class="grant">R01 DK068336-01</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK072193-05</span>, <span class="grant">R01 DK073490</span>, <span class="grant">R01 DK073490-01</span>, <span class="grant">R01 DK075681</span>, <span class="grant">R01 DK075681-02</span>, <span class="grant">R01 DK075787</span>, <span class="grant">R01 DK075787-03</span>, <span class="grant">R01 DK089256</span>, <span class="grant">R01 DK091718</span>, <span class="grant">R01-DK068336</span>, <span class="grant">R01-DK073490</span>, <span class="grant">R01-DK075681</span>, <span class="grant">R01-DK075787</span>, <span class="grant">U01 DK062370</span>, <span class="grant">U01 DK062370-08</span>, <span class="grant">U01 DK062418</span>; NIGMS NIH HHS: <span class="grant">U01 GM074518</span>, <span class="grant">U01 GM074518-05</span>, <span class="grant">U01-GM074518</span>; NIMH NIH HHS: <span class="grant">MH084698</span>, <span class="grant">R01 MH059160</span>, <span class="grant">R01 MH059160-04</span>, <span class="grant">R01 MH059565</span>, <span class="grant">R01 MH059565-06</span>, <span class="grant">R01 MH059566</span>, <span class="grant">R01 MH059566-08</span>, <span class="grant">R01 MH059571</span>, <span class="grant">R01 MH059571-05</span>, <span class="grant">R01 MH059586</span>, <span class="grant">R01 MH059586-08</span>, <span class="grant">R01 MH059587-09</span>, <span class="grant">R01 MH059588-08</span>, <span class="grant">R01 MH060870-09</span>, <span class="grant">R01 MH060879-08</span>, <span class="grant">R01 MH061675</span>, <span class="grant">R01 MH061675-09</span>, <span class="grant">R01 MH067257-04</span>, <span class="grant">R01 MH081800</span>, <span class="grant">R01 MH081800-01</span>, <span class="grant">R01-MH059160</span>, <span class="grant">R01-MH59565</span>, <span class="grant">R01-MH59566</span>, <span class="grant">R01-MH59571</span>, <span class="grant">R01-MH59586</span>, <span class="grant">R01-MH59587</span>, <span class="grant">R01-MH59588</span>, <span class="grant">R01-MH60870</span>, <span class="grant">R01-MH60879</span>, <span class="grant">R01-MH61675</span>, <span class="grant">R01-MH63706</span>, <span class="grant">R01-MH67257</span>, <span class="grant">R01-MH79469</span>, <span class="grant">R01-MH81800</span>, <span class="grant">RL1 MH083268</span>, <span class="grant">RL1 MH083268-05</span>, <span class="grant">RL1-MH083268</span>, <span class="grant">U01 MH079469</span>, <span class="grant">U01 MH079469-03</span>, <span class="grant">U01 MH079470</span>, <span class="grant">U01 MH079470-03</span>, <span class="grant">U01-MH79469</span>, <span class="grant">U01-MH79470</span>; PHS HHS: <span class="grant">263-MA-410953</span>, <span class="grant">HHSN268200625226C</span>, <span class="grant">N01-G65403</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">072856</span>, <span class="grant">072960</span>, <span class="grant">075491</span>, <span class="grant">076113</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">076113/C/04/Z</span>, <span class="grant">077016</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079557</span>, <span class="grant">079771</span>, <span class="grant">079895</span>, <span class="grant">081682</span>, <span class="grant">081682/Z/06/Z</span>, <span class="grant">083270</span>, <span class="grant">084183/Z/07/Z</span>, <span class="grant">085301</span>, <span class="grant">085301/Z/08/Z</span>, <span class="grant">086596</span>, <span class="grant">086596/Z/08/Z</span>, <span class="grant">088885</span>, <span class="grant">091746</span>, <span class="grant">091746/Z/10/Z</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">467</span>;<span class="number">7317</span>;<span class="pagerange">832-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20881960">20881960</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2955183">2955183</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09410">10.1038/nature09410</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic association and interaction analysis of USF1 and APOA5 on lipid levels and atherosclerosis.</h4>
<p class="authors"><span class="author">Laurila PP</span>, <span class="author">Naukkarinen J</span>, <span class="author">Kristiansson K</span>, <span class="author">Ripatti S</span>, <span class="author">Kauttu T</span>, <span class="author">Silander K</span>, <span class="author">Salomaa V</span>, <span class="author">Perola M</span>, <span class="author">Karhunen PJ</span>, <span class="author">Barter PJ</span>, <span class="author">Ehnholm C</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Public Health Genomics Unit, National Institute for Health and Welfare and Institute for Molecular Medicine, Helsinki, Finland.</p><p class="abstract"><strong>Objective</strong>: USF1 is a ubiquitous transcription factor governing the expression of numerous genes of lipid and glucose metabolism. APOA5 is a well-established candidate gene regulating triglyceride (TG) levels and has been identified as a downstream target of upstream stimulatory factor. No detailed studies about the effect of APOA5 on atherosclerotic lesion formation have been conducted, nor has its potential interaction with USF1 been examined.</p><p class="abstract">We analyzed allelic variants of USF1 and APOA5 in families (n=516) ascertained for atherogenic dyslipidemia and in an autopsy series of middle-aged men (n=300) with precise quantitative measurements of atherosclerotic lesions. The impact of previously associated APOA5 variants on TGs was observed in the dyslipidemic families, and variant rs3135506 was associated with size of fibrotic aortic lesions in the autopsy series. The USF1 variant rs2516839, associated previously with atherosclerotic lesions, showed an effect on TGs in members of the dyslipidemic families with documented coronary artery disease. We provide preliminary evidence of gene-gene interaction between these variants in an autopsy series with a fibrotic lesion area in the abdominal aorta (P=0.0028), with TGs in dyslipidemic coronary artery disease subjects (P=0.03), and with high-density lipoprotein cholesterol (P=0.008) in a large population cohort of coronary artery disease patients (n=1065) in which the interaction for TGs was not replicated.</p><p class="abstract"><strong>Conclusions</strong>: Our findings in these unique samples reinforce the roles of APOA5 and USF1 variants on cardiovascular phenotypes and suggest that both genes contribute to lipid levels and aortic atherosclerosis individually and possibly through epistatic effects.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">2</span>;<span class="pagerange">346-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19910639" rel="external">19910639</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3224996" rel="external">3224996</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.109.188912" rel="external">10.1161/ATVBAHA.109.188912</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of purified Clostridium difficile spores to facilitate evaluation of health care disinfection regimens.</h4>
<p class="authors"><span class="author">Lawley TD</span>, <span class="author">Clare S</span>, <span class="author">Deakin LJ</span>, <span class="author">Goulding D</span>, <span class="author">Yen JL</span>, <span class="author">Raisen C</span>, <span class="author">Brandt C</span>, <span class="author">Lovell J</span>, <span class="author">Cooke F</span>, <span class="author">Clark TG</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Microbial Pathogenesis Laboratory, Wellcome Trust, Hinxton, Cambridgeshire CB10 1SA, United Kingdom. <a href="mailto:%74%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Clostridium difficile is a major cause of antibiotic-associated diarrheal disease in many parts of the world. In recent years, distinct genetic variants of C. difficile that cause severe disease and persist within health care settings have emerged. Highly resistant and infectious C. difficile spores are proposed to be the main vectors of environmental persistence and host transmission, so methods to accurately monitor spores and their inactivation are urgently needed. Here we describe simple quantitative methods, based on purified C. difficile spores and a murine transmission model, for evaluating health care disinfection regimens. We demonstrate that disinfectants that contain strong oxidizing active ingredients, such as hydrogen peroxide, are very effective in inactivating pure spores and blocking spore-mediated transmission. Complete inactivation of 10&#x2076; pure C. difficile spores on indicator strips, a six-log reduction, and a standard measure of stringent disinfection regimens require at least 5 min of exposure to hydrogen peroxide vapor (HPV; 400 ppm). In contrast, a 1-min treatment with HPV was required to disinfect an environment that was heavily contaminated with C. difficile spores (17 to 29 spores/cm&sup2;) and block host transmission. Thus, pure C. difficile spores facilitate practical methods for evaluating the efficacy of C. difficile spore disinfection regimens and bringing scientific acumen to C. difficile infection control.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0901743</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Applied and environmental microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">76</span>;<span class="number">20</span>;<span class="pagerange">6895-900</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20802075">20802075</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2953018">2953018</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAEM.00718-10">10.1128/AEM.00718-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CCRaVAT and QuTie-enabling analysis of rare variants in large-scale case control and quantitative trait association studies.</h4>
<p class="authors"><span class="author">Lawrence R</span>, <span class="author">Day-Williams AG</span>, <span class="author">Elliott KS</span>, <span class="author">Morris AP</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract"><strong>Background</strong>: Genome-wide association studies have been successful in finding common variants influencing common traits. However, these associations only account for a fraction of trait heritability. There has been a shift in the field towards studying low frequency and rare variants, which are now widely recognised as putative complex trait determinants. Despite this increasing focus on examining the role of low frequency and rare variants in complex disease susceptibility, there is a lack of user-friendly analytical packages implementing powerful association tests for the analysis of rare variants.</p><p class="abstract"><strong>Results</strong>: We have developed two software tools, CCRaVAT (Case-Control Rare Variant Analysis Tool) and QuTie (Quantitative Trait), which enable efficient large-scale analysis of low frequency and rare variants. Both programs implement a collapsing method examining the accumulation of low frequency and rare variants across a locus of interest that has more power than single variant analysis. CCRaVAT carries out case-control analyses whereas QuTie has been developed for continuous trait analysis.</p><p class="abstract"><strong>Conclusions</strong>: CCRaVAT and QuTie are easy to use software tools that allow users to perform genome-wide association analysis on low frequency and rare variants for both binary and quantitative traits. The software is freely available and provides the genetics community with a resource to perform association analysis on rarer genetic variants.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">064890</span>, <span class="grant">079557</span>, <span class="grant">079557MA</span>, <span class="grant">081682</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">527</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20964851">20964851</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2973964">2973964</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-11-527">10.1186/1471-2105-11-527</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improvements to services at the European Nucleotide Archive.</h4>
<p class="authors"><span class="author">Leinonen R</span>, <span class="author">Akhtar R</span>, <span class="author">Birney E</span>, <span class="author">Bonfield J</span>, <span class="author">Bower L</span>, <span class="author">Corbett M</span>, <span class="author">Cheng Y</span>, <span class="author">Demiralp F</span>, <span class="author">Faruque N</span>, <span class="author">Goodgame N</span>, <span class="author">Gibson R</span>, <span class="author">Hoad G</span>, <span class="author">Hunter C</span>, <span class="author">Jang M</span>, <span class="author">Leonard S</span>, <span class="author">Lin Q</span>, <span class="author">Lopez R</span>, <span class="author">Maguire M</span>, <span class="author">McWilliam H</span>, <span class="author">Plaister S</span>, <span class="author">Radhakrishnan R</span>, <span class="author">Sobhany S</span>, <span class="author">Slater G</span>, <span class="author">Ten Hoopen P</span>, <span class="author">Valentin F</span>, <span class="author">Vaughan R</span>, <span class="author">Zalunin V</span>, <span class="author">Zerbino D</span> and <span class="author">Cochrane G</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%72%61%73%6B%6F%40%65%62%69.%61%63.%75%6B">&#114;&#97;&#115;&#107;&#111;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The European Nucleotide Archive (ENA; http://www.ebi.ac.uk/ena) is Europe&#39;s primary nucleotide sequence archival resource, safeguarding open nucleotide data access, engaging in worldwide collaborative data exchange and integrating with the scientific publication process. ENA has made significant contributions to the collaborative nucleotide archival arena as an active proponent of extending the traditional collaboration to cover capillary and next-generation sequencing information. We have continued to co-develop data and metadata representation formats with our collaborators for both data exchange and public data dissemination. In addition to the DDBJ/EMBL/GenBank feature table format, we share metadata formats for capillary and next-generation sequencing traces and are using and contributing to the NCBI SRA Toolkit for the long-term storage of the next-generation sequence traces. During the course of 2009, ENA has significantly improved sequence submission, search and access functionalities provided at EMBL-EBI. In this article, we briefly describe the content and scope of our archive and introduce major improvements to our services.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D39-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19906712" rel="external">19906712</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808951" rel="external">2808951</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp998" rel="external">10.1093/nar/gkp998</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phylogenetic analysis of gene structure and alternative splicing in alpha-actinins.</h4>
<p class="authors"><span class="author">Lek M</span>, <span class="author">MacArthur DG</span>, <span class="author">Yang N</span> and <span class="author">North KN</span></p>
<p class="affiliation">Institute for Neuroscience and Muscle Research, The Children&#39;s Hospital at Westmead, Sydney, NSW, Australia.</p><p class="abstract">The alpha-actinins are an important family of actin-binding proteins with the ability to cross-link actin filaments when in dimer form. Members of the alpha-actinin family share a domain topology composed of highly conserved actin-binding and EF-hand domains separated by a rod domain composed of spectrin-like repeats. Functional diversity within this family has arisen through exon duplication and the formation of alternate splice isoforms as well as gene duplications during the evolution of vertebrates. In addition to the known functional domains, alpha-actinins also contain a consensus PDZ-binding site. The completed genome sequence of over 32 invertebrate species has allowed the analysis of gene structure and exon-gene duplication over a diverse range of phyla. Our analysis shows that relative to early branching metazoans, there has been considerable intron loss especially in arthropods with few cases of intron gains. The C-terminal PDZ-binding site is conserved in nearly all invertebrates but is missing in some nematodes and platyhelminths. Alternative splicing in the actin-binding domain is conserved in chordates, arthropods, and some nematodes and platyhelminths. In contrast, alternative splicing of the EF-hand domain is only observed in chordates. Finally, given the prevalence of exon duplications seen in the actin-binding domain, this may act as a significant mechanism in the modification of actin-binding properties.</p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2010</span>;<span class="volume">27</span>;<span class="number">4</span>;<span class="pagerange">773-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19897525">19897525</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4125621">4125621</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsp268">10.1093/molbev/msp268</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integration of genetic, clinical, and INR data to refine warfarin dosing.</h4>
<p class="authors"><span class="author">Lenzini P</span>, <span class="author">Wadelius M</span>, <span class="author">Kimmel S</span>, <span class="author">Anderson JL</span>, <span class="author">Jorgensen AL</span>, <span class="author">Pirmohamed M</span>, <span class="author">Caldwell MD</span>, <span class="author">Limdi N</span>, <span class="author">Burmester JK</span>, <span class="author">Dowd MB</span>, <span class="author">Angchaisuksiri P</span>, <span class="author">Bass AR</span>, <span class="author">Chen J</span>, <span class="author">Eriksson N</span>, <span class="author">Rane A</span>, <span class="author">Lindh JD</span>, <span class="author">Carlquist JF</span>, <span class="author">Horne BD</span>, <span class="author">Grice G</span>, <span class="author">Milligan PE</span>, <span class="author">Eby C</span>, <span class="author">Shin J</span>, <span class="author">Kim H</span>, <span class="author">Kurnik D</span>, <span class="author">Stein CM</span>, <span class="author">McMillin G</span>, <span class="author">Pendleton RC</span>, <span class="author">Berg RL</span>, <span class="author">Deloukas P</span> and <span class="author">Gage BF</span></p>
<p class="affiliation">Department of Internal Medicine, Washington University, St Louis, Missouri, USA.</p><p class="abstract">Well-characterized genes that affect warfarin metabolism (cytochrome P450 (CYP) 2C9) and sensitivity (vitamin K epoxide reductase complex 1 (VKORC1)) explain one-third of the variability in therapeutic dose before the international normalized ratio (INR) is measured. To determine genotypic relevance after INR becomes available, we derived clinical and pharmacogenetic refinement algorithms on the basis of INR values (on day 4 or 5 of therapy), clinical factors, and genotype. After adjusting for INR, CYP2C9 and VKORC1 genotypes remained significant predictors (P &lt; 0.001) of warfarin dose. The clinical algorithm had an R(2) of 48% (median absolute error (MAE): 7.0 mg/week) and the pharmacogenetic algorithm had an R(2) of 63% (MAE: 5.5 mg/week) in the derivation set (N = 969). In independent validation sets, the R(2) was 26-43% with the clinical algorithm and 42-58% when genotype was added (P = 0.002). After several days of therapy, a pharmacogenetic algorithm estimates the therapeutic warfarin dose more accurately than one using clinical factors and INR response alone.</p>
<p class="grants">Funded by: Department of Health; NHLBI NIH HHS: <span class="grant">HL097036</span>, <span class="grant">R01 HL074724-01</span>, <span class="grant">R01 HL074724-02</span>, <span class="grant">R01 HL074724-03</span>, <span class="grant">R01 HL074724-04</span>, <span class="grant">R01 HL092173-03</span>, <span class="grant">R01 HL092173-04</span>, <span class="grant">R01 HL092173-05</span>, <span class="grant">R01 HL097036-01</span>, <span class="grant">R01S HL074724</span>; NINDS NIH HHS: <span class="grant">K23 NS045598-05</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Clinical pharmacology and therapeutics</span>&nbsp;<span class="year">2010</span>;<span class="volume">87</span>;<span class="number">5</span>;<span class="pagerange">572-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20375999" rel="external">20375999</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2858245" rel="external">2858245</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fclpt.2010.13" rel="external">10.1038/clpt.2010.13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of a pathogenic rhodococcus: cooptive virulence underpinned by key gene acquisitions.</h4>
<p class="authors"><span class="author">Letek M</span>, <span class="author">Gonz&aacute;lez P</span>, <span class="author">Macarthur I</span>, <span class="author">Rodr&iacute;guez H</span>, <span class="author">Freeman TC</span>, <span class="author">Valero-Rello A</span>, <span class="author">Blanco M</span>, <span class="author">Buckley T</span>, <span class="author">Cherevach I</span>, <span class="author">Fahey R</span>, <span class="author">Hapeshi A</span>, <span class="author">Holdstock J</span>, <span class="author">Leadon D</span>, <span class="author">Navas J</span>, <span class="author">Ocampo A</span>, <span class="author">Quail MA</span>, <span class="author">Sanders M</span>, <span class="author">Scortti MM</span>, <span class="author">Prescott JF</span>, <span class="author">Fogarty U</span>, <span class="author">Meijer WG</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span> and <span class="author">V&aacute;zquez-Boland JA</span></p>
<p class="affiliation">Microbial Pathogenesis Unit, Centres for Infectious Diseases and Immunity, Infection, and Evolution, University of Edinburgh, Edinburgh, United Kingdom.</p><p class="abstract">We report the genome of the facultative intracellular parasite Rhodococcus equi, the only animal pathogen within the biotechnologically important actinobacterial genus Rhodococcus. The 5.0-Mb R. equi 103S genome is significantly smaller than those of environmental rhodococci. This is due to genome expansion in nonpathogenic species, via a linear gain of paralogous genes and an accelerated genetic flux, rather than reductive evolution in R. equi. The 103S genome lacks the extensive catabolic and secondary metabolic complement of environmental rhodococci, and it displays unique adaptations for host colonization and competition in the short-chain fatty acid-rich intestine and manure of herbivores--two main R. equi reservoirs. Except for a few horizontally acquired (HGT) pathogenicity loci, including a cytoadhesive pilus determinant (rpl) and the virulence plasmid vap pathogenicity island (PAI) required for intramacrophage survival, most of the potential virulence-associated genes identified in R. equi are conserved in environmental rhodococci or have homologs in nonpathogenic Actinobacteria. This suggests a mechanism of virulence evolution based on the cooption of existing core actinobacterial traits, triggered by key host niche-adaptive HGT events. We tested this hypothesis by investigating R. equi virulence plasmid-chromosome crosstalk, by global transcription profiling and expression network analysis. Two chromosomal genes conserved in environmental rhodococci, encoding putative chorismate mutase and anthranilate synthase enzymes involved in aromatic amino acid biosynthesis, were strongly coregulated with vap PAI virulence genes and required for optimal proliferation in macrophages. The regulatory integration of chromosomal metabolic genes under the control of the HGT-acquired plasmid PAI is thus an important element in the cooptive virulence of R. equi.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F003722/1</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span>;<span class="pagerange">e1001145</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20941392">20941392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2947987">2947987</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001145">10.1371/journal.pgen.1001145</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NordicDB: a Nordic pool and portal for genome-wide control data.</h4>
<p class="authors"><span class="author">Leu M</span>, <span class="author">Humphreys K</span>, <span class="author">Surakka I</span>, <span class="author">Rehnberg E</span>, <span class="author">Muilu J</span>, <span class="author">Rosenstr&ouml;m P</span>, <span class="author">Almgren P</span>, <span class="author">J&auml;&auml;skel&auml;inen J</span>, <span class="author">Lifton RP</span>, <span class="author">Kyvik KO</span>, <span class="author">Kaprio J</span>, <span class="author">Pedersen NL</span>, <span class="author">Palotie A</span>, <span class="author">Hall P</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Groop L</span>, <span class="author">Peltonen L</span>, <span class="author">Palmgren J</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden. <a href="mailto:%6D%6F%6E%69%63%61.%6C%65%75%40%6B%69.%73%65">&#109;&#111;&#110;&#105;&#99;&#97;&#46;&#108;&#101;&#117;&#64;&#107;&#105;&#46;&#115;&#101;</a></p><p class="abstract">A cost-efficient way to increase power in a genetic association study is to pool controls from different sources. The genotyping effort can then be directed to large case series. The Nordic Control database, NordicDB, has been set up as a unique resource in the Nordic area and the data are available for authorized users through the web portal (http://www.nordicdb.org). The current version of NordicDB pools together high-density genome-wide SNP information from &sim;5000 controls originating from Finnish, Swedish and Danish studies and shows country-specific allele frequencies for SNP markers. The genetic homogeneity of the samples was investigated using multidimensional scaling (MDS) analysis and pairwise allele frequency differences between the studies. The plot of the first two MDS components showed excellent resemblance to the geographical placement of the samples, with a clear NW-SE gradient. We advise researchers to assess the impact of population structure when incorporating NordicDB controls in association studies. This harmonized Nordic database presents a unique genome-wide resource for future genetic association studies in the Nordic countries.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">12</span>;<span class="pagerange">1322-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20664631" rel="external">20664631</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3002853" rel="external">3002853</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2010.112" rel="external">10.1038/ejhg.2010.112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association identifies OBFC1 as a locus involved in human leukocyte telomere biology.</h4>
<p class="authors"><span class="author">Levy D</span>, <span class="author">Neuhausen SL</span>, <span class="author">Hunt SC</span>, <span class="author">Kimura M</span>, <span class="author">Hwang SJ</span>, <span class="author">Chen W</span>, <span class="author">Bis JC</span>, <span class="author">Fitzpatrick AL</span>, <span class="author">Smith E</span>, <span class="author">Johnson AD</span>, <span class="author">Gardner JP</span>, <span class="author">Srinivasan SR</span>, <span class="author">Schork N</span>, <span class="author">Rotter JI</span>, <span class="author">Herbig U</span>, <span class="author">Psaty BM</span>, <span class="author">Sastrasinh M</span>, <span class="author">Murray SS</span>, <span class="author">Vasan RS</span>, <span class="author">Province MA</span>, <span class="author">Glazer NL</span>, <span class="author">Lu X</span>, <span class="author">Cao X</span>, <span class="author">Kronmal R</span>, <span class="author">Mangino M</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Berenson GS</span> and <span class="author">Aviv A</span></p>
<p class="affiliation">National Heart, Lung, and Blood Institute&#39;s Framingham Heart Study, Framingham, MA, USA. <a href="mailto:%4C%65%76%79%64%40%6E%69%68.%67%6F%76">&#76;&#101;&#118;&#121;&#100;&#64;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">Telomeres are engaged in a host of cellular functions, and their length is regulated by multiple genes. Telomere shortening, in the course of somatic cell replication, ultimately leads to replicative senescence. In humans, rare mutations in genes that regulate telomere length have been identified in monogenic diseases such as dyskeratosis congenita and idiopathic pulmonary fibrosis, which are associated with shortened leukocyte telomere length (LTL) and increased risk for aplastic anemia. Shortened LTL is observed in a host of aging-related complex genetic diseases and is associated with diminished survival in the elderly. We report results of a genome-wide association study of LTL in a consortium of four observational studies (n = 3,417 participants with LTL and genome-wide genotyping). SNPs in the regions of the oligonucleotide/oligosaccharide-binding folds containing one gene (OBFC1; rs4387287; P = 3.9 x 10(-9)) and chemokine (C-X-C motif) receptor 4 gene (CXCR4; rs4452212; P = 2.9 x 10(-8)) were associated with LTL at a genome-wide significance level (P &lt; 5 x 10(-8)). We attempted replication of the top SNPs at these loci through de novo genotyping of 1,893 additional individuals and in silico lookup in another observational study (n = 2,876), and we confirmed the association findings for OBFC1 but not CXCR4. In addition, we confirmed the telomerase RNA component (TERC) as a gene associated with LTL (P = 1.1 x 10(-5)). The identification of OBFC1 through genome-wide association as a locus for interindividual variation in LTL in the general population advances the understanding of telomere biology in humans and may provide insights into aging-related disorders linked to altered LTL dynamics.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2010</span>;<span class="volume">107</span>;<span class="number">20</span>;<span class="pagerange">9293-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20421499" rel="external">20421499</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2889047" rel="external">2889047</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0911494107" rel="external">10.1073/pnas.0911494107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MicroRNAs in mouse development and disease.</h4>
<p class="authors"><span class="author">Lewis MA</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">MicroRNAs, small non-coding RNAs which act as repressors of target genes, were discovered in 1993, and since then have been shown to play important roles in the development of numerous systems. Consistent with this role, they are also implicated in the pathogenesis of multiple diseases. Here we review the involvement of microRNAs in mouse development and disease, with particular reference to deafness as an example.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Seminars in cell &amp; developmental biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">21</span>;<span class="number">7</span>;<span class="pagerange">774-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20152923">20152923</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2938480">2938480</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.semcdb.2010.02.004">10.1016/j.semcdb.2010.02.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fast and accurate long-read alignment with Burrows-Wheeler transform.</h4>
<p class="authors"><span class="author">Li H</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Genome Campus, Cambridge, CB10 1SA, UK.</p><p class="abstract"><strong>Motivation</strong>: Many programs for aligning short sequencing reads to a reference genome have been developed in the last 2 years. Most of them are very efficient for short reads but inefficient or not applicable for reads &gt;200 bp because the algorithms are heavily and specifically tuned for short queries with low sequencing error rate. However, some sequencing platforms already produce longer reads and others are expected to become available soon. For longer reads, hashing-based software such as BLAT and SSAHA2 remain the only choices. Nonetheless, these methods are substantially slower than short-read aligners in terms of aligned bases per unit time.</p><p class="abstract"><strong>Results</strong>: We designed and implemented a new algorithm, Burrows-Wheeler Aligner&#39;s Smith-Waterman Alignment (BWA-SW), to align long sequences up to 1 Mb against a large sequence database (e.g. the human genome) with a few gigabytes of memory. The algorithm is as accurate as SSAHA2, more accurate than BLAT, and is several to tens of times faster than both.</p><p class="abstract"><strong>Availability</strong>: http://bio-bwa.sourceforge.net</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077192/Z/05/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">5</span>;<span class="pagerange">589-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20080505" rel="external">20080505</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2828108" rel="external">2828108</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp698" rel="external">10.1093/bioinformatics/btp698</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association scan on estrogen receptor-negative breast cancer.</h4>
<p class="authors"><span class="author">Li J</span>, <span class="author">Humphreys K</span>, <span class="author">Darabi H</span>, <span class="author">Rosin G</span>, <span class="author">Hannelius U</span>, <span class="author">Heikkinen T</span>, <span class="author">Aittom&auml;ki K</span>, <span class="author">Blomqvist C</span>, <span class="author">Pharoah PD</span>, <span class="author">Dunning AM</span>, <span class="author">Ahmed S</span>, <span class="author">Hooning MJ</span>, <span class="author">Hollestelle A</span>, <span class="author">Oldenburg RA</span>, <span class="author">Alfredsson L</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen-Palotie L</span>, <span class="author">Irwanto A</span>, <span class="author">Low HQ</span>, <span class="author">Teoh GH</span>, <span class="author">Thalamuthu A</span>, <span class="author">Kere J</span>, <span class="author">D&#39;Amato M</span>, <span class="author">Easton DF</span>, <span class="author">Nevanlinna H</span>, <span class="author">Liu J</span>, <span class="author">Czene K</span> and <span class="author">Hall P</span></p>
<p class="affiliation">Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm 17177, Sweden. <a href="mailto:%6A%69%6E%67%6D%65%69.%6C%69%40%6B%69.%73%65">&#106;&#105;&#110;&#103;&#109;&#101;&#105;&#46;&#108;&#105;&#64;&#107;&#105;&#46;&#115;&#101;</a></p><p class="abstract"><strong>Introduction</strong>: Breast cancer is a heterogeneous disease and may be characterized on the basis of whether estrogen receptors (ER) are expressed in the tumour cells. ER status of breast cancer is important clinically, and is used both as a prognostic indicator and treatment predictor. In this study, we focused on identifying genetic markers associated with ER-negative breast cancer risk.</p><p class="abstract"><strong>Methods</strong>: We conducted a genome-wide association analysis of 285,984 single nucleotide polymorphisms (SNPs) genotyped in 617 ER-negative breast cancer cases and 4,583 controls. We also conducted a genome-wide pathway analysis on the discovery dataset using permutation-based tests on pre-defined pathways. The extent of shared polygenic variation between ER-negative and ER-positive breast cancers was assessed by relating risk scores, derived using ER-positive breast cancer samples, to disease state in independent, ER-negative breast cancer cases.</p><p class="abstract"><strong>Results</strong>: Association with ER-negative breast cancer was not validated for any of the five most strongly associated SNPs followed up in independent studies (1,011 ER-negative breast cancer cases, 7,604 controls). However, an excess of small P-values for SNPs with known regulatory functions in cancer-related pathways was found (global P = 0.052). We found no evidence to suggest that ER-negative breast cancer shares a polygenic basis to disease with ER-positive breast cancer.</p><p class="abstract"><strong>Conclusions</strong>: ER-negative breast cancer is a distinct breast cancer subtype that merits independent analyses. Given the clinical importance of this phenotype and the likelihood that genetic effect sizes are small, greater sample sizes and further studies are required to understand the etiology of ER-negative breast cancers.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A10119</span>, <span class="grant">A10124</span>; NCI NIH HHS: <span class="grant">R01 CA58427</span></p>
<p class="publication"><span class="title">Breast cancer research : BCR</span>&nbsp;<span class="year">2010</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">R93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21062454" rel="external">21062454</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3046434" rel="external">3046434</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fbcr2772" rel="external">10.1186/bcr2772</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">JAK2 V617F impairs hematopoietic stem cell function in a conditional knock-in mouse model of JAK2 V617F-positive essential thrombocythemia.</h4>
<p class="authors"><span class="author">Li J</span>, <span class="author">Spensberger D</span>, <span class="author">Ahn JS</span>, <span class="author">Anand S</span>, <span class="author">Beer PA</span>, <span class="author">Ghevaert C</span>, <span class="author">Chen E</span>, <span class="author">Forrai A</span>, <span class="author">Scott LM</span>, <span class="author">Ferreira R</span>, <span class="author">Campbell PJ</span>, <span class="author">Watson SP</span>, <span class="author">Liu P</span>, <span class="author">Erber WN</span>, <span class="author">Huntly BJ</span>, <span class="author">Ottersbach K</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Department of Haematology, University of Cambridge, United Kingdom.</p><p class="abstract">The JAK2 V617F mutation is found in most patients with a myeloproliferative neoplasm and is sufficient to produce a myeloproliferative phenotype in murine retroviral transplantation or transgenic models. However, several lines of evidence suggest that disease phenotype is influenced by the level of mutant JAK2 signaling, and we have therefore generated a conditional knock-in mouse in which a human JAK2 V617F is expressed under the control of the mouse Jak2 locus. Human and murine Jak2 transcripts are expressed at similar levels, and mice develop modest increases in hemoglobin and platelet levels reminiscent of human JAK2 V617F-positive essential thrombocythemia. The phenotype is transplantable and accompanied by increased terminal erythroid and megakaryocyte differentiation together with increased numbers of clonogenic progenitors, including erythropoietin-independent erythroid colonies. Unexpectedly, JAK2(V617F) mice develop reduced numbers of lineage(-)Sca-1(+)c-Kit(+) cells, which exhibit increased DNA damage, reduced apoptosis, and reduced cell cycling. Moreover, competitive bone marrow transplantation studies demonstrated impaired hematopoietic stem cell function in JAK2(V617F) mice. These results suggest that the chronicity of human myeloproliferative neoplasms may reflect a balance between impaired hematopoietic stem cell function and the accumulation of additional mutations.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS09039</span>; Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2010</span>;<span class="volume">116</span>;<span class="number">9</span>;<span class="pagerange">1528-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20489053" rel="external">20489053</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145111" rel="external">3145111</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2009-12-259747" rel="external">10.1182/blood-2009-12-259747</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide forward genetic screens in mouse ES cells.</h4>
<p class="authors"><span class="author">Li MA</span>, <span class="author">Pettitt SJ</span>, <span class="author">Yusa K</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Mouse embryonic stem (ES) cells are an attractive model system for investigating mammalian biology. Their relatively stable genome and high amenability to genome modification enables the generation of large-scale mutant libraries, which can be subsequently used for phenotype-driven genetic screens. While retroviral vectors have traditionally been used to generate insertional mutations in ES cells, their severe distribution-bias in the mammalian genome substantially limits genome-wide mutagenesis. The recent development of the DNA transposon piggyBac offers an efficient and highly versatile alternative for achieving more unbiased mutagenesis. Furthermore, heterozygous mutations created by insertional mutagens can be converted in parallel to homozygosity by using Blm-deficient ES cells, allowing genome-wide loss-of-function screens to be conducted. In this chapter, we describe the principles underpinning genetic screens in mouse ES cells with examples of previously successful screens. Protocols are provided for piggyBac transposon-mediated mutagenesis, production of the corresponding homozygous mutants in a Blm-deficient genetic background, and methods for mapping and validation of mutations recovered from screens of such libraries.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Methods in enzymology</span>&nbsp;<span class="year">2010</span>;<span class="volume">477</span>;<span class="pagerange">217-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20699144" rel="external">20699144</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0076-6879(10)77012-9" rel="external">10.1016/S0076-6879(10)77012-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reprogramming of T cells to natural killer-like cells upon Bcl11b deletion.</h4>
<p class="authors"><span class="author">Li P</span>, <span class="author">Burke S</span>, <span class="author">Wang J</span>, <span class="author">Chen X</span>, <span class="author">Ortiz M</span>, <span class="author">Lee SC</span>, <span class="author">Lu D</span>, <span class="author">Campos L</span>, <span class="author">Goulding D</span>, <span class="author">Ng BL</span>, <span class="author">Dougan G</span>, <span class="author">Huntly B</span>, <span class="author">Gottgens B</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Colucci F</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">T cells develop in the thymus and are critical for adaptive immunity. Natural killer (NK) lymphocytes constitute an essential component of the innate immune system in tumor surveillance, reproduction, and defense against microbes and viruses. Here, we show that the transcription factor Bcl11b was expressed in all T cell compartments and was indispensable for T lineage development. When Bcl11b was deleted, T cells from all developmental stages acquired NK cell properties and concomitantly lost or decreased T cell-associated gene expression. These induced T-to-natural killer (ITNK) cells, which were morphologically and genetically similar to conventional NK cells, killed tumor cells in vitro, and effectively prevented tumor metastasis in vivo. Therefore, ITNKs may represent a new cell source for cell-based therapies.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council: <span class="grant">G0501150</span>, <span class="grant">G0800784</span>, <span class="grant">G116/187</span>; Wellcome Trust: <span class="grant">076962</span>, <span class="grant">077186</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">329</span>;<span class="number">5987</span>;<span class="pagerange">85-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20538915">20538915</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3628452">3628452</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1188063">10.1126/science.1188063</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis and imputation refines the association of 15q25 with smoking quantity.</h4>
<p class="authors"><span class="author">Liu JZ</span>, <span class="author">Tozzi F</span>, <span class="author">Waterworth DM</span>, <span class="author">Pillai SG</span>, <span class="author">Muglia P</span>, <span class="author">Middleton L</span>, <span class="author">Berrettini W</span>, <span class="author">Knouff CW</span>, <span class="author">Yuan X</span>, <span class="author">Waeber G</span>, <span class="author">Vollenweider P</span>, <span class="author">Preisig M</span>, <span class="author">Wareham NJ</span>, <span class="author">Zhao JH</span>, <span class="author">Loos RJ</span>, <span class="author">Barroso I</span>, <span class="author">Khaw KT</span>, <span class="author">Grundy S</span>, <span class="author">Barter P</span>, <span class="author">Mahley R</span>, <span class="author">Kesaniemi A</span>, <span class="author">McPherson R</span>, <span class="author">Vincent JB</span>, <span class="author">Strauss J</span>, <span class="author">Kennedy JL</span>, <span class="author">Farmer A</span>, <span class="author">McGuffin P</span>, <span class="author">Day R</span>, <span class="author">Matthews K</span>, <span class="author">Bakke P</span>, <span class="author">Gulsvik A</span>, <span class="author">Lucae S</span>, <span class="author">Ising M</span>, <span class="author">Brueckl T</span>, <span class="author">Horstmann S</span>, <span class="author">Wichmann HE</span>, <span class="author">Rawal R</span>, <span class="author">Dahmen N</span>, <span class="author">Lamina C</span>, <span class="author">Polasek O</span>, <span class="author">Zgaga L</span>, <span class="author">Huffman J</span>, <span class="author">Campbell S</span>, <span class="author">Kooner J</span>, <span class="author">Chambers JC</span>, <span class="author">Burnett MS</span>, <span class="author">Devaney JM</span>, <span class="author">Pichard AD</span>, <span class="author">Kent KM</span>, <span class="author">Satler L</span>, <span class="author">Lindsay JM</span>, <span class="author">Waksman R</span>, <span class="author">Epstein S</span>, <span class="author">Wilson JF</span>, <span class="author">Wild SH</span>, <span class="author">Campbell H</span>, <span class="author">Vitart V</span>, <span class="author">Reilly MP</span>, <span class="author">Li M</span>, <span class="author">Qu L</span>, <span class="author">Wilensky R</span>, <span class="author">Matthai W</span>, <span class="author">Hakonarson HH</span>, <span class="author">Rader DJ</span>, <span class="author">Franke A</span>, <span class="author">Wittig M</span>, <span class="author">Sch&auml;fer A</span>, <span class="author">Uda M</span>, <span class="author">Terracciano A</span>, <span class="author">Xiao X</span>, <span class="author">Busonero F</span>, <span class="author">Scheet P</span>, <span class="author">Schlessinger D</span>, <span class="author">St Clair D</span>, <span class="author">Rujescu D</span>, <span class="author">Abecasis GR</span>, <span class="author">Grabe HJ</span>, <span class="author">Teumer A</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Petersmann A</span>, <span class="author">John U</span>, <span class="author">Rudan I</span>, <span class="author">Hayward C</span>, <span class="author">Wright AF</span>, <span class="author">Kolcic I</span>, <span class="author">Wright BJ</span>, <span class="author">Thompson JR</span>, <span class="author">Balmforth AJ</span>, <span class="author">Hall AS</span>, <span class="author">Samani NJ</span>, <span class="author">Anderson CA</span>, <span class="author">Ahmad T</span>, <span class="author">Mathew CG</span>, <span class="author">Parkes M</span>, <span class="author">Satsangi J</span>, <span class="author">Caulfield M</span>, <span class="author">Munroe PB</span>, <span class="author">Farrall M</span>, <span class="author">Dominiczak A</span>, <span class="author">Worthington J</span>, <span class="author">Thomson W</span>, <span class="author">Eyre S</span>, <span class="author">Barton A</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Mooser V</span>, <span class="author">Francks C</span> and <span class="author">Marchini J</span></p>
<p class="affiliation">Department of Statistics, University of Oxford, Oxford, UK.</p><p class="abstract">Smoking is a leading global cause of disease and mortality. We established the Oxford-GlaxoSmithKline study (Ox-GSK) to perform a genome-wide meta-analysis of SNP association with smoking-related behavioral traits. Our final data set included 41,150 individuals drawn from 20 disease, population and control cohorts. Our analysis confirmed an effect on smoking quantity at a locus on 15q25 (P = 9.45 x 10(-19)) that includes CHRNA5, CHRNA3 and CHRNB4, three genes encoding neuronal nicotinic acetylcholine receptor subunits. We used data from the 1000 Genomes project to investigate the region using imputation, which allowed for analysis of virtually all common SNPs in the region and offered a fivefold increase in marker density over HapMap2 (ref. 2) as an imputation reference panel. Our fine-mapping approach identified a SNP showing the highest significance, rs55853698, located within the promoter region of CHRNA5. Conditional analysis also identified a secondary locus (rs6495308) in CHRNA3.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">CZB/4/710</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0600329</span>, <span class="grant">G0701863</span>, <span class="grant">G0800759</span>, <span class="grant">G9521010</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>; NIA NIH HHS: <span class="grant">Z99 AG999999</span>, <span class="grant">ZIA AG000196-03</span>, <span class="grant">ZIA AG000196-04</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">436-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20418889">20418889</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3612983">3612983</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.572">10.1038/ng.572</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical roles of Bcl11b in T-cell development and maintenance of T-cell identity.</h4>
<p class="authors"><span class="author">Liu P</span>, <span class="author">Li P</span> and <span class="author">Burke S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%70%6C%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#108;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">T-cell development primarily occurs in the thymus and involves in the interactions of many important transcription factors. Until recently, no single transcription factor has been identified to be essential for T-cell lineage commitment or maintenance of T-cell identity. Recent studies have now identified the zinc finger transcription factor Bcl11b to be essential for T-cell development and for maintenance of T-cell identity. Remarkably, T cells acquire NK cell properties upon Bcl11b deletion. These reprogrammed cells have unique properties in proliferation, cytokine dependency and killing target cells, and may therefore provide a new cell source for some cell-based therapies.</p>
<p class="publication"><span class="title">Immunological reviews</span>&nbsp;<span class="year">2010</span>;<span class="volume">238</span>;<span class="number">1</span>;<span class="pagerange">138-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20969590" rel="external">20969590</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1600-065X.2010.00953.x" rel="external">10.1111/j.1600-065X.2010.00953.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterisations of odorant-binding proteins in the tsetse fly Glossina morsitans morsitans.</h4>
<p class="authors"><span class="author">Liu R</span>, <span class="author">Lehane S</span>, <span class="author">He X</span>, <span class="author">Lehane M</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Berriman M</span>, <span class="author">Pickett JA</span>, <span class="author">Field LM</span> and <span class="author">Zhou JJ</span></p>
<p class="affiliation">Department of Biological Chemistry, Harpenden, UK.</p><p class="abstract">Odorant-binding proteins (OBPs) play an important role in insect olfaction by mediating interactions between odorants and odorant receptors. We report for the first time 20 OBP genes in the tsetse fly Glossina morsitans morsitans. qRT-PCR revealed that 8 of these genes were highly transcribed in the antennae. The transcription of these genes in the antennae was significantly lower in males than in females and there was a clear correlation between OBP gene transcription and feeding status. Starvation over 72 h post-blood meal (PBM) did not significantly affect the transcription. However, the transcription in the antennae of 10-week-old flies was much higher than in 3-day-old flies at 48 h PBM and decreased sharply after 72 h starvation, suggesting that the OBP gene expression is affected by the insect&#39;s nutritional status. Sequence comparisons with OBPs of other Dipterans identified several homologs to sex pheromone-binding proteins and OBPs of Drosophila melanogaster.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Cellular and molecular life sciences : CMLS</span>&nbsp;<span class="year">2010</span>;<span class="volume">67</span>;<span class="number">6</span>;<span class="pagerange">919-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20012146">20012146</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00018-009-0221-1">10.1007/s00018-009-0221-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Origin of the human malaria parasite Plasmodium falciparum in gorillas.</h4>
<p class="authors"><span class="author">Liu W</span>, <span class="author">Li Y</span>, <span class="author">Learn GH</span>, <span class="author">Rudicell RS</span>, <span class="author">Robertson JD</span>, <span class="author">Keele BF</span>, <span class="author">Ndjango JB</span>, <span class="author">Sanz CM</span>, <span class="author">Morgan DB</span>, <span class="author">Locatelli S</span>, <span class="author">Gonder MK</span>, <span class="author">Kranzusch PJ</span>, <span class="author">Walsh PD</span>, <span class="author">Delaporte E</span>, <span class="author">Mpoudi-Ngole E</span>, <span class="author">Georgiev AV</span>, <span class="author">Muller MN</span>, <span class="author">Shaw GM</span>, <span class="author">Peeters M</span>, <span class="author">Sharp PM</span>, <span class="author">Rayner JC</span> and <span class="author">Hahn BH</span></p>
<p class="affiliation">Department of Medicine, University of Alabama at Birmingham, Birmingham, Alabama 35294, USA.</p><p class="abstract">Plasmodium falciparum is the most prevalent and lethal of the malaria parasites infecting humans, yet the origin and evolutionary history of this important pathogen remain controversial. Here we develop a single-genome amplification strategy to identify and characterize Plasmodium spp. DNA sequences in faecal samples from wild-living apes. Among nearly 3,000 specimens collected from field sites throughout central Africa, we found Plasmodium infection in chimpanzees (Pan troglodytes) and western gorillas (Gorilla gorilla), but not in eastern gorillas (Gorilla beringei) or bonobos (Pan paniscus). Ape plasmodial infections were highly prevalent, widely distributed and almost always made up of mixed parasite species. Analysis of more than 1,100 mitochondrial, apicoplast and nuclear gene sequences from chimpanzees and gorillas revealed that 99% grouped within one of six host-specific lineages representing distinct Plasmodium species within the subgenus Laverania. One of these from western gorillas comprised parasites that were nearly identical to P. falciparum. In phylogenetic analyses of full-length mitochondrial sequences, human P. falciparum formed a monophyletic lineage within the gorilla parasite radiation. These findings indicate that P. falciparum is of gorilla origin and not of chimpanzee, bonobo or ancient human origin.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NIAID NIH HHS: <span class="grant">P30 AI 7767</span>, <span class="grant">P30 AI027767</span>, <span class="grant">P30 AI027767-21A1</span>, <span class="grant">R01 AI058715</span>, <span class="grant">R01 AI058715-06A1</span>, <span class="grant">R01 AI058715-07</span>, <span class="grant">R01 AI50529</span>, <span class="grant">R03 AI074778</span>, <span class="grant">R03 AI074778-02</span>, <span class="grant">R37 AI050529</span>, <span class="grant">R37 AI050529-07</span>, <span class="grant">R37 AI050529-08</span>, <span class="grant">T32 AI007245</span>, <span class="grant">T32 AI007245-26</span>, <span class="grant">U19 AI 067854</span>, <span class="grant">U19 AI067854</span>, <span class="grant">U19 AI067854-06</span>; NIGMS NIH HHS: <span class="grant">T32 GM008111</span>, <span class="grant">T32 GM008111-13</span>; PHS HHS: <span class="grant">R01 I58715</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">467</span>;<span class="number">7314</span>;<span class="pagerange">420-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20864995">20864995</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2997044">2997044</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09442">10.1038/nature09442</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single genome amplification and direct amplicon sequencing of Plasmodium spp. DNA from ape fecal specimens.</h4>
<p class="authors"><span class="author">Liu W</span>, <span class="author">Li Y</span>, <span class="author">Peeters M</span>, <span class="author">Rayner J</span>, <span class="author">Sharp P</span>, <span class="author">Shaw G</span> and <span class="author">Hahn B</span></p>
<p class="affiliation">Department of Medicine, University of Alabama at Birmingham.</p><p class="abstract">Conventional PCR followed by molecular cloning and sequencing of amplified products is commonly used to test clinical specimens for target sequences of interest, such as viral, bacterial or parasite nucleic acids. However, this approach has serious limitations when used to analyze mixtures of genetically divergent templates(1-9). This is because Taq polymerase is prone to switch templates during the amplification process, thereby generating recombinants that do not exist in vivo (4). When amplicons are cloned prior to sequence analysis, the resulting sequences may also contain a substantial number of Tag-induced substitutions(1-4). Finally, cloning of amplicons can lead to a non-proportional representation of sequences due to the re-sampling of only certain templates(1-4). These confounders can be avoided by using single genome amplification (SGA) followed by direct sequencing of SGA amplicons(1-5). While SGA is not required for many research applications, we have shown it to be essential for deciphering the diversification pathways of human and simian immunodeficiency viruses (HIV/SIV) in acute and chronic infection(4-7), the detection of simian foamy virus (SFVCPZ) super-infection in wild-living chimpanzees(8), and most recently, the molecular identification and characterization of Plasmodium spp. infections in wild-living apes(9). Here, we describe SGA-direct amplicon sequencing of Plasmodium spp. DNA from ape fecal samples.</p>
<p class="publication"><span class="title">Protocol exchange</span>&nbsp;<span class="year">2010</span>;<span class="volume">2010</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/25221670">25221670</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2010.156">10.1038/nprot.2010.156</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ten simple rules for editing Wikipedia.</h4>
<p class="authors"><span class="author">Logan DW</span>, <span class="author">Sandal M</span>, <span class="author">Gardner PP</span>, <span class="author">Manske M</span> and <span class="author">Bateman A</span></p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20941386">20941386</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2947980">2947980</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.1000941">10.1371/journal.pcbi.1000941</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss-of-function variants in the genomes of healthy humans.</h4>
<p class="authors"><span class="author">MacArthur DG</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%64%6D%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#109;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genetic variants predicted to seriously disrupt the function of human protein-coding genes-so-called loss-of-function (LOF) variants-have traditionally been viewed in the context of severe Mendelian disease. However, recent large-scale sequencing and genotyping projects have revealed a surprisingly large number of these variants in the genomes of apparently healthy individuals--at least 100 per genome, including more than 30 in a homozygous state--suggesting a previously unappreciated level of variation in functional gene content between humans. These variants are mostly found at low frequency, suggesting that they are enriched for mildly deleterious polymorphisms suppressed by negative natural selection, and thus represent an attractive set of candidate variants for complex disease susceptibility. However, they are also enriched for sequencing and annotation artefacts, so overall present serious challenges for clinical sequencing projects seeking to identify severe disease genes amidst the &#39;noise&#39; of technical error and benign genetic polymorphism. Systematic, high-quality catalogues of LOF variants present in the genomes of healthy individuals, built from the output of large-scale sequencing studies such as the 1000 Genomes Project, will help to distinguish between benign and disease-causing LOF variants, and will provide valuable resources for clinical genomics.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">R2</span>;<span class="pagerange">R125-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20805107">20805107</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2953739">2953739</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq365">10.1093/hmg/ddq365</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dysregulated humoral immunity to nontyphoidal Salmonella in HIV-infected African adults.</h4>
<p class="authors"><span class="author">MacLennan CA</span>, <span class="author">Gilchrist JJ</span>, <span class="author">Gordon MA</span>, <span class="author">Cunningham AF</span>, <span class="author">Cobbold M</span>, <span class="author">Goodall M</span>, <span class="author">Kingsley RA</span>, <span class="author">van Oosterhout JJ</span>, <span class="author">Msefula CL</span>, <span class="author">Mandala WL</span>, <span class="author">Leyton DL</span>, <span class="author">Marshall JL</span>, <span class="author">Gondwe EN</span>, <span class="author">Bobat S</span>, <span class="author">L&oacute;pez-Mac&iacute;as C</span>, <span class="author">Doffinger R</span>, <span class="author">Henderson IR</span>, <span class="author">Zijlstra EE</span>, <span class="author">Dougan G</span>, <span class="author">Drayson MT</span>, <span class="author">MacLennan IC</span> and <span class="author">Molyneux ME</span></p>
<p class="affiliation">Medical Research Council Centre for Immune Regulation and Clinical Immunology Service, Institute of Biomedical Research, School of Immunity and Infection, University of Birmingham, Birmingham, UK. <a href="mailto:%63.%6D%61%63%6C%65%6E%6E%61%6E%40%62%68%61%6D.%61%63.%75%6B">&#99;&#46;&#109;&#97;&#99;&#108;&#101;&#110;&#110;&#97;&#110;&#64;&#98;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Nontyphoidal Salmonellae are a major cause of life-threatening bacteremia among HIV-infected individuals. Although cell-mediated immunity controls intracellular infection, antibodies protect against Salmonella bacteremia. We report that high-titer antibodies specific for Salmonella lipopolysaccharide (LPS) are associated with a lack of Salmonella-killing in HIV-infected African adults. Killing was restored by genetically shortening LPS from the target Salmonella or removing LPS-specific antibodies from serum. Complement-mediated killing of Salmonella by healthy serum is shown to be induced specifically by antibodies against outer membrane proteins. This killing is lost when excess antibody against Salmonella LPS is added. Thus, our study indicates that impaired immunity against nontyphoidal Salmonella bacteremia in HIV infection results from excess inhibitory antibodies against Salmonella LPS, whereas serum killing of Salmonella is induced by antibodies against outer membrane proteins.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">328</span>;<span class="number">5977</span>;<span class="pagerange">508-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20413503">20413503</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1180346">10.1126/science.1180346</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting report: a workshop on Best Practices in Genome Annotation.</h4>
<p class="authors"><span class="author">Madupu R</span>, <span class="author">Brinkac LM</span>, <span class="author">Harrow J</span>, <span class="author">Wilming LG</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Lamesch P</span> and <span class="author">Hannick LI</span></p>
<p class="affiliation">Informatics, J. Craig Venter Institute, Rockville, MD 20850 USA, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK and The Arabidopsis Information Resource, Carnegie Institution of Washington, Stanford, CA 94305 USA.</p><p class="abstract">Efforts to annotate the genomes of a wide variety of model organisms are currently carried out by sequencing centers, model organism databases and academic/institutional laboratories around the world. Different annotation methods and tools have been developed over time to meet the needs of biologists faced with the task of annotating biological data. While standardized methods are essential for consistent curation within each annotation group, methods and tools can differ between groups, especially when the groups are curating different organisms. Biocurators from several institutes met at the Third International Biocuration Conference in Berlin, Germany, April 2009 and hosted the &#39;Best Practices in Genome Annotation: Inference from Evidence&#39; workshop to share their strategies, pipelines, standards and tools. This article documents the material presented in the workshop.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Database : the journal of biological databases and curation</span>&nbsp;<span class="year">2010</span>;<span class="volume">2010</span>;<span class="pagerange">baq001</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20428316">20428316</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2860899">2860899</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fdatabase%2Fbaq001">10.1093/database/baq001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinct effects of allelic NFIX mutations on nonsense-mediated mRNA decay engender either a Sotos-like or a Marshall-Smith syndrome.</h4>
<p class="authors"><span class="author">Malan V</span>, <span class="author">Rajan D</span>, <span class="author">Thomas S</span>, <span class="author">Shaw AC</span>, <span class="author">Louis Dit Picard H</span>, <span class="author">Layet V</span>, <span class="author">Till M</span>, <span class="author">van Haeringen A</span>, <span class="author">Mortier G</span>, <span class="author">Nampoothiri S</span>, <span class="author">Puselji&#x107; S</span>, <span class="author">Legeai-Mallet L</span>, <span class="author">Carter NP</span>, <span class="author">Vekemans M</span>, <span class="author">Munnich A</span>, <span class="author">Hennekam RC</span>, <span class="author">Colleaux L</span> and <span class="author">Cormier-Daire V</span></p>
<p class="affiliation">D&eacute;partement de G&eacute;n&eacute;tique, Universit&eacute; Paris Descartes, H&ocirc;pital Necker-Enfants Malades, Paris 75015, France.</p><p class="abstract">By using a combination of array comparative genomic hybridization and a candidate gene approach, we identified nuclear factor I/X (NFIX) deletions or nonsense mutation in three sporadic cases of a Sotos-like overgrowth syndrome with advanced bone age, macrocephaly, developmental delay, scoliosis, and unusual facies. Unlike the aforementioned human syndrome, Nfix-deficient mice are unable to gain weight and die in the first 3 postnatal weeks, while they also present with a spinal deformation and decreased bone mineralization. These features prompted us to consider NFIX as a candidate gene for Marshall-Smith syndrome (MSS), a severe malformation syndrome characterized by failure to thrive, respiratory insufficiency, accelerated osseous maturation, kyphoscoliosis, osteopenia, and unusual facies. Distinct frameshift and splice NFIX mutations that escaped nonsense-mediated mRNA decay (NMD) were identified in nine MSS subjects. NFIX belongs to the Nuclear factor one (NFI) family of transcription factors, but its specific function is presently unknown. We demonstrate that NFIX is normally expressed prenatally during human brain development and skeletogenesis. These findings demonstrate that allelic NFIX mutations trigger distinct phenotypes, depending specifically on their impact on NMD.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">87</span>;<span class="number">2</span>;<span class="pagerange">189-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20673863" rel="external">20673863</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2917711" rel="external">2917711</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.07.001" rel="external">10.1016/j.ajhg.2010.07.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Butyrate greatly enhances derivation of human induced pluripotent stem cells by promoting epigenetic remodeling and the expression of pluripotency-associated genes.</h4>
<p class="authors"><span class="author">Mali P</span>, <span class="author">Chou BK</span>, <span class="author">Yen J</span>, <span class="author">Ye Z</span>, <span class="author">Zou J</span>, <span class="author">Dowey S</span>, <span class="author">Brodsky RA</span>, <span class="author">Ohm JE</span>, <span class="author">Yu W</span>, <span class="author">Baylin SB</span>, <span class="author">Yusa K</span>, <span class="author">Bradley A</span>, <span class="author">Meyers DJ</span>, <span class="author">Mukherjee C</span>, <span class="author">Cole PA</span> and <span class="author">Cheng L</span></p>
<p class="affiliation">Stem Cell Program, Institute for Cell Engineering, Johns Hopkins University School of Medicine, Baltimore, Maryland, USA.</p><p class="abstract">We report here that butyrate, a naturally occurring fatty acid commonly used as a nutritional supplement and differentiation agent, greatly enhances the efficiency of induced pluripotent stem (iPS) cell derivation from human adult or fetal fibroblasts. After transient butyrate treatment, the iPS cell derivation efficiency is enhanced by 15- to 51-fold using either retroviral or piggyBac transposon vectors expressing 4 to 5 reprogramming genes. Butyrate stimulation is more remarkable (&gt;100- to 200-fold) on reprogramming in the absence of either KLF4 or MYC transgene. Butyrate treatment did not negatively affect properties of iPS cell lines established by either 3 or 4 retroviral vectors or a single piggyBac DNA transposon vector. These characterized iPS cell lines, including those derived from an adult patient with sickle cell disease by either the piggyBac or retroviral vectors, show normal karyotypes and pluripotency. To gain insights into the underlying mechanisms of butyrate stimulation, we conducted genome-wide gene expression and promoter DNA methylation microarrays and other epigenetic analyses on established iPS cells and cells from intermediate stages of the reprogramming process. By days 6 to 12 during reprogramming, butyrate treatment enhanced histone H3 acetylation, promoter DNA demethylation, and the expression of endogenous pluripotency-associated genes, including DPPA2, whose overexpression partially substitutes for butyrate stimulation. Thus, butyrate as a cell permeable small molecule provides a simple tool to further investigate molecular mechanisms of cellular reprogramming. Moreover, butyrate stimulation provides an efficient method for reprogramming various human adult somatic cells, including cells from patients that are more refractory to reprogramming.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01 HL073781-06</span>, <span class="grant">RC2 HL101582</span>, <span class="grant">RC2 HL101582-01</span>; NIGMS NIH HHS: <span class="grant">GM62437</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Stem cells (Dayton, Ohio)</span>&nbsp;<span class="year">2010</span>;<span class="volume">28</span>;<span class="number">4</span>;<span class="pagerange">713-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20201064" rel="external">20201064</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3015217" rel="external">3015217</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fstem.402" rel="external">10.1002/stem.402</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FRT-seq: amplification-free, strand-specific transcriptome sequencing.</h4>
<p class="authors"><span class="author">Mamanova L</span>, <span class="author">Andrews RM</span>, <span class="author">James KD</span>, <span class="author">Sheridan EM</span>, <span class="author">Ellis PD</span>, <span class="author">Langford CF</span>, <span class="author">Ost TW</span>, <span class="author">Collins JE</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We report an alternative approach to transcriptome sequencing for the Illumina Genome Analyzer, in which the reverse transcription reaction takes place on the flowcell. No amplification is performed during the library preparation, so PCR biases and duplicates are avoided, and because the template is poly(A)(+) RNA rather than cDNA, the resulting sequences are necessarily strand-specific. The method is compatible with paired- or single-end sequencing.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span>, <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">130-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20081834">20081834</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2861772">2861772</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1417">10.1038/nmeth.1417</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Target-enrichment strategies for next-generation sequencing.</h4>
<p class="authors"><span class="author">Mamanova L</span>, <span class="author">Coffey AJ</span>, <span class="author">Scott CE</span>, <span class="author">Kozarewa I</span>, <span class="author">Turner EH</span>, <span class="author">Kumar A</span>, <span class="author">Howard E</span>, <span class="author">Shendure J</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We have not yet reached a point at which routine sequencing of large numbers of whole eukaryotic genomes is feasible, and so it is often necessary to select genomic regions of interest and to enrich these regions before sequencing. There are several enrichment approaches, each with unique advantages and disadvantages. Here we describe our experiences with the leading target-enrichment technologies, the optimizations that we have performed and typical results that can be obtained using each. We also provide detailed protocols for each technology so that end users can find the best compromise between sensitivity, specificity and uniformity for their particular project.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5R21HG004749</span>, <span class="grant">R21 HG004749</span>; NHLBI NIH HHS: <span class="grant">5R01HL094976</span>, <span class="grant">R01 HL094976</span>; Wellcome Trust: <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">111-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20111037">20111037</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1419">10.1038/nmeth.1419</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction of a large extracellular protein interaction network and its resolution by spatiotemporal expression profiling.</h4>
<p class="authors"><span class="author">Martin S</span>, <span class="author">S&ouml;llner C</span>, <span class="author">Charoensawan V</span>, <span class="author">Adryan B</span>, <span class="author">Thisse B</span>, <span class="author">Thisse C</span>, <span class="author">Teichmann S</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Cambridge CB101HH, United Kingdom.</p><p class="abstract">Extracellular interactions involving both secreted and membrane-tethered receptor proteins are essential to initiate signaling pathways that orchestrate cellular behaviors within biological systems. Because of the biochemical properties of these proteins and their interactions, identifying novel extracellular interactions remains experimentally challenging. To address this, we have recently developed an assay, AVEXIS (avidity-based extracellular interaction screen) to detect low affinity extracellular interactions on a large scale and have begun to construct interaction networks between zebrafish receptors belonging to the immunoglobulin and leucine-rich repeat protein families to identify novel signaling pathways important for early development. Here, we expanded our zebrafish protein library to include other domain families and many more secreted proteins and performed our largest screen to date totaling 16,544 potential unique interactions. We report 111 interactions of which 96 are novel and include the first documented extracellular ligands for 15 proteins. By including 77 interactions from previous screens, we assembled an expanded network of 188 extracellular interactions between 92 proteins and used it to show that secreted proteins have twice as many interaction partners as membrane-tethered receptors and that the connectivity of the extracellular network behaves as a power law. To try to understand the functional role of these interactions, we determined new expression patterns for 164 genes within our clone library by using whole embryo in situ hybridization at five key stages of zebrafish embryonic development. These expression data were integrated with the binding network to reveal where each interaction was likely to function within the embryo and were used to resolve the static interaction network into dynamic tissue- and stage-specific subnetworks within the developing zebrafish embryo. All these data were organized into a freely accessible on-line database called ARNIE (AVEXIS Receptor Network with Integrated Expression; www.sanger.ac.uk/arnie) and provide a valuable resource of new extracellular signaling interactions for developmental biology.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="number">12</span>;<span class="pagerange">2654-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20802085">20802085</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3101854">3101854</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M110.004119">10.1074/mcp.M110.004119</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Review of genetic factors in intestinal malrotation.</h4>
<p class="authors"><span class="author">Martin V</span> and <span class="author">Shaw-Smith C</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Intestinal malrotation is well covered in the surgical literature from the point of view of operative management, but few reviews to date have attempted to provide a comprehensive examination of the topic from the point of view of aetiology, in particular genetic aetiology. Following a brief overview of molecular embryology of midgut rotation, we present in this article instances of and case reports and case series of intestinal malrotation in which a genetic aetiology is likely. Autosomal dominant, autosomal recessive, X-linked and chromosomal forms of the disorder are represented. Most occur in syndromic form, that is to say, in association with other malformations. In many instances, recognition of a specific syndrome is possible, one of several examples discussed being the recently described association of intestinal malrotation with alveolar capillary dysplasia, due to mutations in the forkhead box transcription factor FOXF1. New advances in sequencing technology mean that the identification of the genes mutated in these disorders is more accessible than ever, and paediatric surgeons are encouraged to refer to their colleagues in clinical genetics where a genetic aetiology seems likely.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Pediatric surgery international</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">8</span>;<span class="pagerange">769-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20549505" rel="external">20549505</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2908440" rel="external">2908440</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00383-010-2622-5" rel="external">10.1007/s00383-010-2622-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies two novel regions at 11p15.5-p13 and 1p31 with major impact on acute-phase serum amyloid A.</h4>
<p class="authors"><span class="author">Marzi C</span>, <span class="author">Albrecht E</span>, <span class="author">Hysi PG</span>, <span class="author">Lagou V</span>, <span class="author">Waldenberger M</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Prokopenko I</span>, <span class="author">Heim K</span>, <span class="author">Blackburn H</span>, <span class="author">Ried JS</span>, <span class="author">Kleber ME</span>, <span class="author">Mangino M</span>, <span class="author">Thorand B</span>, <span class="author">Peters A</span>, <span class="author">Hammond CJ</span>, <span class="author">Grallert H</span>, <span class="author">Boehm BO</span>, <span class="author">Kovacs P</span>, <span class="author">Geistlinger L</span>, <span class="author">Prokisch H</span>, <span class="author">Winkelmann BR</span>, <span class="author">Spector TD</span>, <span class="author">Wichmann HE</span>, <span class="author">Stumvoll M</span>, <span class="author">Soranzo N</span>, <span class="author">M&auml;rz W</span>, <span class="author">Koenig W</span>, <span class="author">Illig T</span> and <span class="author">Gieger C</span></p>
<p class="affiliation">Institute of Epidemiology, Helmholtz Zentrum M&uuml;nchen, German Research Center for Environmental Health, Neuherberg, Germany.</p><p class="abstract">Elevated levels of acute-phase serum amyloid A (A-SAA) cause amyloidosis and are a risk factor for atherosclerosis and its clinical complications, type 2 diabetes, as well as various malignancies. To investigate the genetic basis of A-SAA levels, we conducted the first genome-wide association study on baseline A-SAA concentrations in three population-based studies (KORA, TwinsUK, Sorbs) and one prospective case cohort study (LURIC), including a total of 4,212 participants of European descent, and identified two novel genetic susceptibility regions at 11p15.5-p13 and 1p31. The region at 11p15.5-p13 (rs4150642; p&#x200A;=&#x200A;3.20&times;10(-111)) contains serum amyloid A1 (SAA1) and the adjacent general transcription factor 2 H1 (GTF2H1), Hermansky-Pudlak Syndrome 5 (HPS5), lactate dehydrogenase A (LDHA), and lactate dehydrogenase C (LDHC). This region explains 10.84% of the total variation of A-SAA levels in our data, which makes up 18.37% of the total estimated heritability. The second region encloses the leptin receptor (LEPR) gene at 1p31 (rs12753193; p&#x200A;=&#x200A;1.22&times;10(-11)) and has been found to be associated with CRP and fibrinogen in previous studies. Our findings demonstrate a key role of the 11p15.5-p13 region in the regulation of baseline A-SAA levels and provide confirmative evidence of the importance of the 1p31 region for inflammatory processes and the close interplay between A-SAA, leptin, and other acute-phase proteins.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e1001213</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21124955" rel="external">21124955</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987930" rel="external">2987930</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001213" rel="external">10.1371/journal.pgen.1001213</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ten years experience of Salmonella infections in Cambridge, UK.</h4>
<p class="authors"><span class="author">Matheson N</span>, <span class="author">Kingsley RA</span>, <span class="author">Sturgess K</span>, <span class="author">Aliyu SH</span>, <span class="author">Wain J</span>, <span class="author">Dougan G</span> and <span class="author">Cooke FJ</span></p>
<p class="affiliation">Clinical Microbiology and Public Health Laboratory, Health Protection Agency, Box 236, Addenbrooke&#39;s Hospital, Hills Road, Cambridge CB20QW, UK.</p><p class="abstract"><strong>Objectives</strong>: Review of all Salmonella infections diagnosed in the Cambridge area over 10 years.</p><p class="abstract"><strong>Methods</strong>: All Salmonella enterica isolated in the Clinical Microbiology Laboratory, Addenbrooke&#39;s Hospital between 1.1.1999 and 31.12.2008 were included. Patient demographics, serotype and additional relevant details (travel history, resistance-type, phage-type) were recorded.</p><p class="abstract"><strong>Results</strong>: 1003 episodes of Salmonella gastroenteritis were confirmed by stool culture, representing 88 serotypes. Serotypes Enteritidis (59%), Typhimurium (4.7%), Virchow (2.6%), Newport (1.8%) and Braenderup (1.7%) were the 5 most common isolates. There were an additional 37 invasive Salmonella infections (32 blood cultures, 4 tissue samples, 1 CSF). 13/15 patients with Salmonella Typhi or Salmonella Paratyphi isolated from blood or faeces with an available travel history had returned from the Indian subcontinent. 8/10 S. Typhi or Paratyphi isolates tested had reduced susceptibility to fluoroquinolones (MIC &gt; or = 0.125 mg/L). 7/21 patients with non-typhoidal Salmonella bacteraemia were known to be immunosuppressed.</p><p class="abstract"><strong>Conclusion</strong>: This study describes Salmonella serotypes circulating within a defined geographical area over a decade. Prospective molecular analysis of isolates of S. enterica by multi-locus sequence typing (MLST) and single nucleotide polymorphism (SNP) detection will determine the geo-phylogenetic relationship of isolates within our region.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of infection</span>&nbsp;<span class="year">2010</span>;<span class="volume">60</span>;<span class="number">1</span>;<span class="pagerange">21-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19819256" rel="external">19819256</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jinf.2009.09.016" rel="external">10.1016/j.jinf.2009.09.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel candidate cancer genes identified by a large-scale cross-species comparative oncogenomics approach.</h4>
<p class="authors"><span class="author">Mattison J</span>, <span class="author">Kool J</span>, <span class="author">Uren AG</span>, <span class="author">de Ridder J</span>, <span class="author">Wessels L</span>, <span class="author">Jonkers J</span>, <span class="author">Bignell GR</span>, <span class="author">Butler A</span>, <span class="author">Rust AG</span>, <span class="author">Brosch M</span>, <span class="author">Wilson CH</span>, <span class="author">van der Weyden L</span>, <span class="author">Largaespada DA</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">van Lohuizen M</span>, <span class="author">Berns A</span>, <span class="author">Collier LS</span>, <span class="author">Hubbard T</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Comparative genomic hybridization (CGH) can reveal important disease genes but the large regions identified could sometimes contain hundreds of genes. Here we combine high-resolution CGH analysis of 598 human cancer cell lines with insertion sites isolated from 1,005 mouse tumors induced with the murine leukemia virus (MuLV). This cross-species oncogenomic analysis revealed candidate tumor suppressor genes and oncogenes mutated in both human and mouse tumors, making them strong candidates for novel cancer genes. A significant number of these genes contained binding sites for the stem cell transcription factors Oct4 and Nanog. Notably, mice carrying tumors with insertions in or near stem cell module genes, which are thought to participate in cell self-renewal, died significantly faster than mice without these insertions. A comparison of the profile we identified to that induced with the Sleeping Beauty (SB) transposon system revealed significant differences in the profile of recurrently mutated genes. Collectively, this work provides a rich catalogue of new candidate cancer genes for functional analysis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6997</span>, <span class="grant">A8784</span>; NCI NIH HHS: <span class="grant">K01 CA122183</span>, <span class="grant">K01CA122183</span>, <span class="grant">R01 CA113636</span>, <span class="grant">R01 CA134759</span>; Wellcome Trust: <span class="grant">077198</span>, <span class="grant">082356</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2010</span>;<span class="volume">70</span>;<span class="number">3</span>;<span class="pagerange">883-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20103622">20103622</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2880710">2880710</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-09-1737">10.1158/0008-5472.CAN-09-1737</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Notch2 is required for progression of pancreatic intraepithelial neoplasia and development of pancreatic ductal adenocarcinoma.</h4>
<p class="authors"><span class="author">Mazur PK</span>, <span class="author">Einw&auml;chter H</span>, <span class="author">Lee M</span>, <span class="author">Sipos B</span>, <span class="author">Nakhai H</span>, <span class="author">Rad R</span>, <span class="author">Zimber-Strobl U</span>, <span class="author">Strobl LJ</span>, <span class="author">Radtke F</span>, <span class="author">Kl&ouml;ppel G</span>, <span class="author">Schmid RM</span> and <span class="author">Siveke JT</span></p>
<p class="affiliation">Second Department of Internal Medicine and Institute of Pathology, Technical University of Munich, 81675 Munich, Germany.</p><p class="abstract">Pancreatic cancer is one of the most fatal malignancies lacking effective therapies. Notch signaling is a key regulator of cell fate specification and pancreatic cancer development; however, the role of individual Notch receptors and downstream signaling is largely unknown. Here, we show that Notch2 is predominantly expressed in ductal cells and pancreatic intraepithelial neoplasia (PanIN) lesions. Using genetically engineered mice, we demonstrate the effect of conditional Notch receptor ablation in KrasG12D-driven pancreatic carcinogenesis. Deficiency of Notch2 but not Notch1 stops PanIN progression, prolongs survival, and leads to a phenotypical switch toward anaplastic pancreatic cancer with epithelial-mesenchymal transition. By expression profiling, we identified increased Myc signaling regulated by Notch2 during tumor development, placing Notch2 as a central regulator of PanIN progression and malignant transformation. Our study supports the concept of distinctive roles of individual Notch receptors in cancer development.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2010</span>;<span class="volume">107</span>;<span class="number">30</span>;<span class="pagerange">13438-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20624967" rel="external">20624967</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2922150" rel="external">2922150</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1002423107" rel="external">10.1073/pnas.1002423107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of cancer-specific genomic rearrangements to quantify disease burden in plasma from patients with solid tumors.</h4>
<p class="authors"><span class="author">McBride DJ</span>, <span class="author">Orpana AK</span>, <span class="author">Sotiriou C</span>, <span class="author">Joensuu H</span>, <span class="author">Stephens PJ</span>, <span class="author">Mudie LJ</span>, <span class="author">H&auml;m&auml;l&auml;inen E</span>, <span class="author">Stebbings LA</span>, <span class="author">Andersson LC</span>, <span class="author">Flanagan AM</span>, <span class="author">Durbecq V</span>, <span class="author">Ignatiadis M</span>, <span class="author">Kallioniemi O</span>, <span class="author">Heckman CA</span>, <span class="author">Alitalo K</span>, <span class="author">Edgren H</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.</p><p class="abstract">Detection of recurrent somatic rearrangements routinely allows monitoring of residual disease burden in leukemias, but is not used for most solid tumors. However, next-generation sequencing now allows rapid identification of patient-specific rearrangements in solid tumors. We mapped genomic rearrangements in three cancers and showed that PCR assays for rearrangements could detect a single copy of the tumor genome in plasma without false positives. Disease status, drug responsiveness, and incipient relapse could be serially assessed. In future, this strategy could be readily established in diagnostic laboratories, with major impact on monitoring of disease status and personalizing treatment of solid tumors.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">088340/Z/09/Z</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2010</span>;<span class="volume">49</span>;<span class="number">11</span>;<span class="pagerange">1062-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20725990" rel="external">20725990</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145117" rel="external">3145117</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20815" rel="external">10.1002/gcc.20815</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulation of the Epstein-Barr virus Zp promoter in B lymphocytes during reactivation from latency.</h4>
<p class="authors"><span class="author">McDonald C</span>, <span class="author">Karstegl CE</span>, <span class="author">Kellam P</span> and <span class="author">Farrell PJ</span></p>
<p class="affiliation">Department of Virology, Imperial College Faculty of Medicine, St Mary&#39;s Campus, London W2 1PG, UK.</p><p class="abstract">Ten novel mutations were introduced into the Zp promoter to test the role of sequences outside the established transcription factor-binding sites in Epstein-Barr virus (EBV) reactivation. Most of these had only small effects, but mutations in the ZID site were shown to reduce Zp activity strongly at early times after induction by anti-immunoglobulin (anti-Ig). The binding of MEF2 transcription factor to ZID was characterized in detail and linked functionally to Zp promoter activity. The presence of XBP-1s, the active form of XBP-1, after administration of anti-Ig to Akata Burkitt&#39;s lymphoma cells is consistent with a role for this factor in reactivation of the EBV lytic cycle, although signalling through MEF2D was quantitatively much more significant in activation of Zp. Silencing of Zp during latency is thought to be primarily a consequence of a repressive chromatin structure on Zp, and this aspect of Zp regulation can be observed in the Akata genome through protection of Zp from activation by BZLF1 in the absence of signalling from the B-cell receptor.</p>
<p class="publication"><span class="title">The Journal of general virology</span>&nbsp;<span class="year">2010</span>;<span class="volume">91</span>;<span class="number">Pt 3</span>;<span class="pagerange">622-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19923262">19923262</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fvir.0.017277-0">10.1099/vir.0.017277-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Visualizing chromosome mosaicism and detecting ethnic outliers by the method of &quot;rare&quot; heterozygotes and homozygotes (RHH).</h4>
<p class="authors"><span class="author">McGinnis RE</span>, <span class="author">Deloukas P</span>, <span class="author">McLaren WM</span> and <span class="author">Inouye M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%72%6D%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#109;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We describe a novel approach for evaluating SNP genotypes of a genome-wide association scan to identify &quot;ethnic outlier&quot; subjects whose ethnicity is different or admixed compared to most other subjects in the genotyped sample set. Each ethnic outlier is detected by counting a genomic excess of &quot;rare&quot; heterozygotes and/or homozygotes whose frequencies are low (&lt;1%) within genotypes of the sample set being evaluated. This method also enables simple and striking visualization of non-Caucasian chromosomal DNA segments interspersed within the chromosomes of ethnically admixed individuals. We show that this visualization of the mosaic structure of admixed human chromosomes gives results similar to another visualization method (SABER) but with much less computational time and burden. We also show that other methods for detecting ethnic outliers are enhanced by evaluating only genomic regions of visualized admixture rather than diluting outlier ancestry by evaluating the entire genome considered in aggregate. We have validated our method in the Wellcome Trust Case Control Consortium (WTCCC) study of 17,000 subjects as well as in HapMap subjects and simulated outliers of known ethnicity and admixture. The method&#39;s ability to precisely delineate chromosomal segments of non-Caucasian ethnicity has enabled us to demonstrate previously unreported non-Caucasian admixture in two HapMap Caucasian parents and in a number of WTCCC subjects. Its sensitive detection of ethnic outliers and simple visual discrimination of discrete chromosomal segments of different ethnicity implies that this method of rare heterozygotes and homozygotes (RHH) is likely to have diverse and important applications in humans and other species.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">13</span>;<span class="pagerange">2539-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20211853">20211853</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2883336">2883336</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq102">10.1093/hmg/ddq102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.</h4>
<p class="authors"><span class="author">McLaren W</span>, <span class="author">Pritchard B</span>, <span class="author">Rios D</span>, <span class="author">Chen Y</span>, <span class="author">Flicek P</span> and <span class="author">Cunningham F</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%77%6D%32%40%65%62%69.%61%63.%75%6B">&#119;&#109;&#50;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Summary</strong>: A tool to predict the effect that newly discovered genomic variants have on known transcripts is indispensible in prioritizing and categorizing such variants. In Ensembl, a web-based tool (the SNP Effect Predictor) and API interface can now functionally annotate variants in all Ensembl and Ensembl Genomes supported species.</p><p class="abstract"><strong>Availability</strong>: The Ensembl SNP Effect Predictor can be accessed via the Ensembl website at http://www.ensembl.org/. The Ensembl API (http://www.ensembl.org/info/docs/api/api_installation.html for installation instructions) is open source software.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">16</span>;<span class="pagerange">2069-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20562413">20562413</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2916720">2916720</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq330">10.1093/bioinformatics/btq330</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hypomorphic temperature-sensitive alleles of NSDHL cause CK syndrome.</h4>
<p class="authors"><span class="author">McLarren KW</span>, <span class="author">Severson TM</span>, <span class="author">du Souich C</span>, <span class="author">Stockton DW</span>, <span class="author">Kratz LE</span>, <span class="author">Cunningham D</span>, <span class="author">Hendson G</span>, <span class="author">Morin RD</span>, <span class="author">Wu D</span>, <span class="author">Paul JE</span>, <span class="author">An J</span>, <span class="author">Nelson TN</span>, <span class="author">Chou A</span>, <span class="author">DeBarber AE</span>, <span class="author">Merkens LS</span>, <span class="author">Michaud JL</span>, <span class="author">Waters PJ</span>, <span class="author">Yin J</span>, <span class="author">McGillivray B</span>, <span class="author">Demos M</span>, <span class="author">Rouleau GA</span>, <span class="author">Grzeschik KH</span>, <span class="author">Smith R</span>, <span class="author">Tarpey PS</span>, <span class="author">Shears D</span>, <span class="author">Schwartz CE</span>, <span class="author">Gecz J</span>, <span class="author">Stratton MR</span>, <span class="author">Arbour L</span>, <span class="author">Hurlburt J</span>, <span class="author">Van Allen MI</span>, <span class="author">Herman GE</span>, <span class="author">Zhao Y</span>, <span class="author">Moore R</span>, <span class="author">Kelley RI</span>, <span class="author">Jones SJ</span>, <span class="author">Steiner RD</span>, <span class="author">Raymond FL</span>, <span class="author">Marra MA</span> and <span class="author">Boerkoel CF</span></p>
<p class="affiliation">Department of Medical Genetics, Vancouver, Canada.</p><p class="abstract">CK syndrome (CKS) is an X-linked recessive intellectual disability syndrome characterized by dysmorphism, cortical brain malformations, and an asthenic build. Through an X chromosome single-nucleotide variant scan in the first reported family, we identified linkage to a 5 Mb region on Xq28. Sequencing of this region detected a segregating 3 bp deletion (c.696_698del [p.Lys232del]) in exon 7 of NAD(P) dependent steroid dehydrogenase-like (NSDHL), a gene that encodes an enzyme in the cholesterol biosynthesis pathway. We also found that males with intellectual disability in another reported family with an NSDHL mutation (c.1098 dup [p.Arg367SerfsX33]) have CKS. These two mutations, which alter protein folding, show temperature-sensitive protein stability and complementation in Erg26-deficient yeast. As described for the allelic disorder CHILD syndrome, cells and cerebrospinal fluid from CKS patients have increased methyl sterol levels. We hypothesize that methyl sterol accumulation, not only cholesterol deficiency, causes CKS, given that cerebrospinal fluid cholesterol, plasma cholesterol, and plasma 24S-hydroxycholesterol levels are normal in males with CKS. In summary, CKS expands the spectrum of cholesterol-related disorders and insight into the role of cholesterol in human development.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">87</span>;<span class="number">6</span>;<span class="pagerange">905-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21129721" rel="external">21129721</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2997364" rel="external">2997364</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.11.004" rel="external">10.1016/j.ajhg.2010.11.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The bacteriology of pouchitis: a molecular phylogenetic analysis using 16S rRNA gene cloning and sequencing.</h4>
<p class="authors"><span class="author">McLaughlin SD</span>, <span class="author">Walker AW</span>, <span class="author">Churcher C</span>, <span class="author">Clark SK</span>, <span class="author">Tekkis PP</span>, <span class="author">Johnson MW</span>, <span class="author">Parkhill J</span>, <span class="author">Ciclitira PJ</span>, <span class="author">Dougan G</span>, <span class="author">Nicholls RJ</span> and <span class="author">Petrovska L</span></p>
<p class="affiliation">Department of Biosurgery and Surgical Technology, St Mary&#39;s Hospital, Imperial College London, London, United Kingdom.</p><p class="abstract"><strong>Objective</strong>: To identify, compare, and contrast the microbiota in patients with and without pouchitis after restorative proctocolectomy (RPC) for ulcerative colitis (UC) and familial adenomatous polyposis (FAP).</p><p class="abstract">Pouchitis is the most common complication following RPC. An abnormal host-microbial interaction has been implicated. We investigated the pouch microbiota in patients with and without pouchitis undergoing restorative proctocolectomy for UC and FAP.</p><p class="abstract"><strong>Methods</strong>: Mucosal pouch biopsies, taken from 16 UC (pouchitis 8) and 8 FAP (pouchitis 3) patients were analyzed to the species (or phylotype) level by cloning and sequencing of 3184 full-length bacterial 16S rRNA genes.</p><p class="abstract"><strong>Results</strong>: There was a significant increase in Proteobacteria (P = 0.019) and a significant decrease in Bacteroidetes (P = 0.001) and Faecalibacterium prausnitzii (P = 0.029) in the total UC compared with the total FAP cohort, but only limited differences were found between the UC nonpouchitis and pouchitis groups and the FAP pouchitis and nonpouchitis groups. Bacterial diversity in the FAP nonpouchitis group was significantly greater than in UC nonpouchitis (P = 0.019) and significantly greater in UC nonpouchitis compared with UC pouchitis (P = 0.009). No individual species or phylotype specifically associated with either UC or FAP pouchitis was found.</p><p class="abstract"><strong>Conclusions</strong>: UC pouch patients have a different, less diverse, gut microbiota than FAP patients. A further reduction in bacterial diversity but no significant dysbiosis occurs in those with pouchitis. The study suggests that a dysbiosis occurs in the ileal pouch of UC RPC patients which predisposes to, but may not directly cause, pouchitis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076964</span></p>
<p class="publication"><span class="title">Annals of surgery</span>&nbsp;<span class="year">2010</span>;<span class="volume">252</span>;<span class="number">1</span>;<span class="pagerange">90-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20562611" rel="external">20562611</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3292798" rel="external">3292798</a>; DOI: <a href="http://dx.doi.org/10.1097%2FSLA.0b013e3181e3dc8b" rel="external">10.1097/SLA.0b013e3181e3dc8b</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A variant in LIN28B is associated with 2D:4D finger-length ratio, a putative retrospective biomarker of prenatal testosterone exposure.</h4>
<p class="authors"><span class="author">Medland SE</span>, <span class="author">Zayats T</span>, <span class="author">Glaser B</span>, <span class="author">Nyholt DR</span>, <span class="author">Gordon SD</span>, <span class="author">Wright MJ</span>, <span class="author">Montgomery GW</span>, <span class="author">Campbell MJ</span>, <span class="author">Henders AK</span>, <span class="author">Timpson NJ</span>, <span class="author">Peltonen L</span>, <span class="author">Wolke D</span>, <span class="author">Ring SM</span>, <span class="author">Deloukas P</span>, <span class="author">Martin NG</span>, <span class="author">Smith GD</span> and <span class="author">Evans DM</span></p>
<p class="affiliation">Genetic Epidemiology, Queensland Institute of Medical Research, Australia.</p><p class="abstract">The ratio of the lengths of an individual&#39;s second to fourth digit (2D:4D) is commonly used as a noninvasive retrospective biomarker for prenatal androgen exposure. In order to identify the genetic determinants of 2D:4D, we applied a genome-wide association approach to 1507 11-year-old children from the Avon Longitudinal Study of Parents and Children (ALSPAC) in whom 2D:4D ratio had been measured, as well as a sample of 1382 12- to 16-year-olds from the Brisbane Adolescent Twin Study. A meta-analysis of the two scans identified a single variant in the LIN28B gene that was strongly associated with 2D:4D (rs314277: p = 4.1 x 10(-8)) and was subsequently independently replicated in an additional 3659 children from the ALSPAC cohort (p = 1.53 x 10(-6)). The minor allele of the rs314277 variant has previously been linked to increased height and delayed age at menarche, but in our study it was associated with increased 2D:4D in the direction opposite to that of previous reports on the correlation between 2D:4D and age at menarche. Our findings call into question the validity of 2D:4D as a simplistic retrospective biomarker for prenatal testosterone exposure.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">90600705</span>, <span class="grant">G0800582</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">86</span>;<span class="number">4</span>;<span class="pagerange">519-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20303062" rel="external">20303062</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2850436" rel="external">2850436</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.02.017" rel="external">10.1016/j.ajhg.2010.02.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association studies of serum magnesium, potassium, and sodium concentrations identify six Loci influencing serum magnesium levels.</h4>
<p class="authors"><span class="author">Meyer TE</span>, <span class="author">Verwoert GC</span>, <span class="author">Hwang SJ</span>, <span class="author">Glazer NL</span>, <span class="author">Smith AV</span>, <span class="author">van Rooij FJ</span>, <span class="author">Ehret GB</span>, <span class="author">Boerwinkle E</span>, <span class="author">Felix JF</span>, <span class="author">Leak TS</span>, <span class="author">Harris TB</span>, <span class="author">Yang Q</span>, <span class="author">Dehghan A</span>, <span class="author">Aspelund T</span>, <span class="author">Katz R</span>, <span class="author">Homuth G</span>, <span class="author">Kocher T</span>, <span class="author">Rettig R</span>, <span class="author">Ried JS</span>, <span class="author">Gieger C</span>, <span class="author">Prucha H</span>, <span class="author">Pfeufer A</span>, <span class="author">Meitinger T</span>, <span class="author">Coresh J</span>, <span class="author">Hofman A</span>, <span class="author">Sarnak MJ</span>, <span class="author">Chen YD</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Chakravarti A</span>, <span class="author">Psaty BM</span>, <span class="author">van Duijn CM</span>, <span class="author">Kao WH</span>, <span class="author">Witteman JC</span>, <span class="author">Gudnason V</span>, <span class="author">Siscovick DS</span>, <span class="author">Fox CS</span>, <span class="author">K&ouml;ttgen A</span>, <span class="author">Genetic Factors for Osteoporosis Consortium</span> and <span class="author">Meta Analysis of Glucose and Insulin Related Traits Consortium</span></p>
<p class="affiliation">Human Genetics Center and Division of Epidemiology, The University of Texas Health Science Center at Houston, School of Public Health, Houston, Texas, USA.</p><p class="abstract">Magnesium, potassium, and sodium, cations commonly measured in serum, are involved in many physiological processes including energy metabolism, nerve and muscle function, signal transduction, and fluid and blood pressure regulation. To evaluate the contribution of common genetic variation to normal physiologic variation in serum concentrations of these cations, we conducted genome-wide association studies of serum magnesium, potassium, and sodium concentrations using approximately 2.5 million genotyped and imputed common single nucleotide polymorphisms (SNPs) in 15,366 participants of European descent from the international CHARGE Consortium. Study-specific results were combined using fixed-effects inverse-variance weighted meta-analysis. SNPs demonstrating genome-wide significant (p&lt;5 x 10(-8)) or suggestive associations (p&lt;4 x 10(-7)) were evaluated for replication in an additional 8,463 subjects of European descent. The association of common variants at six genomic regions (in or near MUC1, ATP2B1, DCDC5, TRPM6, SHROOM3, and MDS1) with serum magnesium levels was genome-wide significant when meta-analyzed with the replication dataset. All initially significant SNPs from the CHARGE Consortium showed nominal association with clinically defined hypomagnesemia, two showed association with kidney function, two with bone mineral density, and one of these also associated with fasting glucose levels. Common variants in CNNM2, a magnesium transporter studied only in model systems to date, as well as in CNNM3 and CNNM4, were also associated with magnesium concentrations in this study. We observed no associations with serum sodium or potassium levels exceeding p&lt;4 x 10(-7). Follow-up studies of newly implicated genomic loci may provide additional insights into the regulation and homeostasis of human serum magnesium levels.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">M01-RR00425</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01HL087641</span>, <span class="grant">U01 HL080295</span>; NIA NIH HHS: <span class="grant">N01-AG-12100</span>; NIDDK NIH HHS: <span class="grant">DK063491</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20700443">20700443</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2916845">2916845</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001045">10.1371/journal.pgen.1001045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Consensus statement: chromosomal microarray is a first-tier clinical diagnostic test for individuals with developmental disabilities or congenital anomalies.</h4>
<p class="authors"><span class="author">Miller DT</span>, <span class="author">Adam MP</span>, <span class="author">Aradhya S</span>, <span class="author">Biesecker LG</span>, <span class="author">Brothman AR</span>, <span class="author">Carter NP</span>, <span class="author">Church DM</span>, <span class="author">Crolla JA</span>, <span class="author">Eichler EE</span>, <span class="author">Epstein CJ</span>, <span class="author">Faucett WA</span>, <span class="author">Feuk L</span>, <span class="author">Friedman JM</span>, <span class="author">Hamosh A</span>, <span class="author">Jackson L</span>, <span class="author">Kaminsky EB</span>, <span class="author">Kok K</span>, <span class="author">Krantz ID</span>, <span class="author">Kuhn RM</span>, <span class="author">Lee C</span>, <span class="author">Ostell JM</span>, <span class="author">Rosenberg C</span>, <span class="author">Scherer SW</span>, <span class="author">Spinner NB</span>, <span class="author">Stavropoulos DJ</span>, <span class="author">Tepperberg JH</span>, <span class="author">Thorland EC</span>, <span class="author">Vermeesch JR</span>, <span class="author">Waggoner DJ</span>, <span class="author">Watson MS</span>, <span class="author">Martin CL</span> and <span class="author">Ledbetter DH</span></p>
<p class="affiliation">Division of Genetics and Department of Laboratory Medicine, Children&#39;s Hospital Boston and Harvard Medical School, Boston, MA, USA. <a href="mailto:%64%61%76%69%64.%6D%69%6C%6C%65%72%32%40%63%68%69%6C%64%72%65%6E%73.%68%61%72%76%61%72%64.%65%64%75">&#100;&#97;&#118;&#105;&#100;&#46;&#109;&#105;&#108;&#108;&#101;&#114;&#50;&#64;&#99;&#104;&#105;&#108;&#100;&#114;&#101;&#110;&#115;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Chromosomal microarray (CMA) is increasingly utilized for genetic testing of individuals with unexplained developmental delay/intellectual disability (DD/ID), autism spectrum disorders (ASD), or multiple congenital anomalies (MCA). Performing CMA and G-banded karyotyping on every patient substantially increases the total cost of genetic testing. The International Standard Cytogenomic Array (ISCA) Consortium held two international workshops and conducted a literature review of 33 studies, including 21,698 patients tested by CMA. We provide an evidence-based summary of clinical cytogenetic testing comparing CMA to G-banded karyotyping with respect to technical advantages and limitations, diagnostic yield for various types of chromosomal aberrations, and issues that affect test interpretation. CMA offers a much higher diagnostic yield (15%-20%) for genetic testing of individuals with unexplained DD/ID, ASD, or MCA than a G-banded karyotype ( approximately 3%, excluding Down syndrome and other recognizable chromosomal syndromes), primarily because of its higher sensitivity for submicroscopic deletions and duplications. Truly balanced rearrangements and low-level mosaicism are generally not detectable by arrays, but these are relatively infrequent causes of abnormal phenotypes in this population (&lt;1%). Available evidence strongly supports the use of CMA in place of G-banded karyotyping as the first-tier cytogenetic diagnostic test for patients with DD/ID, ASD, or MCA. G-banded karyotype analysis should be reserved for patients with obvious chromosomal syndromes (e.g., Down syndrome), a family history of chromosomal rearrangement, or a history of multiple miscarriages.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NICHD NIH HHS: <span class="grant">RC2 HD064525</span>; NIMH NIH HHS: <span class="grant">MH074090</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">86</span>;<span class="number">5</span>;<span class="pagerange">749-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20466091" rel="external">20466091</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2869000" rel="external">2869000</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.04.006" rel="external">10.1016/j.ajhg.2010.04.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Annotating the regulatory genome.</h4>
<p class="authors"><span class="author">Montgomery SB</span>, <span class="author">Kasaian K</span>, <span class="author">Jones SJ</span> and <span class="author">Griffith OL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%73%74%65%70%68%65%6E.%6D%6F%6E%74%67%6F%6D%65%72%79%40%75%6E%69%67%65.%63%68">&#115;&#116;&#101;&#112;&#104;&#101;&#110;&#46;&#109;&#111;&#110;&#116;&#103;&#111;&#109;&#101;&#114;&#121;&#64;&#117;&#110;&#105;&#103;&#101;&#46;&#99;&#104;</a></p><p class="abstract">Determining the timing and molecular repertoire responsible for gene expression is fundamental to understanding a gene&#39;s function. Heritable differences in this character are increasingly regarded as explanatory for complex and common traits. For many known trait-predisposing genes, studies have sought to elucidate the associated logic behind gene regulation. However, there exist many challenges in deciphering these mechanisms. Among them, it is recognized that we have limited understanding of regulatory complexity, the current models of gene regulation have low specificity and any gene&#39;s regulatory logic is dependent on biological context. Addressing these limitations and defining the regulatory genome is an ongoing challenge for molecular biology. We discuss current efforts to define and annotate the regulatory genome by focusing on curation and text-mining activities. We further highlight the type of information and curation process for describing regulatory elements within the ORegAnno database ( www.oreganno.org ) and how the general standards for such information are changing.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">674</span>;<span class="pagerange">313-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20827601" rel="external">20827601</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-60761-854-6_20" rel="external">10.1007/978-1-60761-854-6_20</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptome genetics using second generation sequencing in a Caucasian population.</h4>
<p class="authors"><span class="author">Montgomery SB</span>, <span class="author">Sammeth M</span>, <span class="author">Gutierrez-Arcelus M</span>, <span class="author">Lach RP</span>, <span class="author">Ingle C</span>, <span class="author">Nisbett J</span>, <span class="author">Guigo R</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Department of Genetic Medicine and Development, University of Geneva Medical School, Geneva, 1211 Switzerland. <a href="mailto:%73%74%65%70%68%65%6E.%6D%6F%6E%74%67%6F%6D%65%72%79%40%75%6E%69%67%65.%63%68">&#115;&#116;&#101;&#112;&#104;&#101;&#110;&#46;&#109;&#111;&#110;&#116;&#103;&#111;&#109;&#101;&#114;&#121;&#64;&#117;&#110;&#105;&#103;&#101;&#46;&#99;&#104;</a></p><p class="abstract">Gene expression is an important phenotype that informs about genetic and environmental effects on cellular state. Many studies have previously identified genetic variants for gene expression phenotypes using custom and commercially available microarrays. Second generation sequencing technologies are now providing unprecedented access to the fine structure of the transcriptome. We have sequenced the mRNA fraction of the transcriptome in 60 extended HapMap individuals of European descent and have combined these data with genetic variants from the HapMap3 project. We have quantified exon abundance based on read depth and have also developed methods to quantify whole transcript abundance. We have found that approximately 10 million reads of sequencing can provide access to the same dynamic range as arrays with better quantification of alternative and highly abundant transcripts. Correlation with SNPs (small nucleotide polymorphisms) leads to a larger discovery of eQTLs (expression quantitative trait loci) than with arrays. We also detect a substantial number of variants that influence the structure of mature transcripts indicating variants responsible for alternative splicing. Finally, measures of allele-specific expression allowed the identification of rare eQTLs and allelic differences in transcript structure. This analysis shows that high throughput sequencing technologies reveal new properties of genetic effects on the transcriptome and allow the exploration of genetic effects in cellular processes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077046</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7289</span>;<span class="pagerange">773-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20220756">20220756</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3836232">3836232</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08903">10.1038/nature08903</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Yersinia pestis genome sequencing identifies patterns of global phylogenetic diversity.</h4>
<p class="authors"><span class="author">Morelli G</span>, <span class="author">Song Y</span>, <span class="author">Mazzoni CJ</span>, <span class="author">Eppinger M</span>, <span class="author">Roumagnac P</span>, <span class="author">Wagner DM</span>, <span class="author">Feldkamp M</span>, <span class="author">Kusecek B</span>, <span class="author">Vogler AJ</span>, <span class="author">Li Y</span>, <span class="author">Cui Y</span>, <span class="author">Thomson NR</span>, <span class="author">Jombart T</span>, <span class="author">Leblois R</span>, <span class="author">Lichtner P</span>, <span class="author">Rahalison L</span>, <span class="author">Petersen JM</span>, <span class="author">Balloux F</span>, <span class="author">Keim P</span>, <span class="author">Wirth T</span>, <span class="author">Ravel J</span>, <span class="author">Yang R</span>, <span class="author">Carniel E</span> and <span class="author">Achtman M</span></p>
<p class="affiliation">Max-Planck-Institut f&uuml;r Infektionsbiologie, Department of Molecular Biology, Berlin, Germany.</p><p class="abstract">Plague is a pandemic human invasive disease caused by the bacterial agent Yersinia pestis. We here report a comparison of 17 whole genomes of Y. pestis isolates from global sources. We also screened a global collection of 286 Y. pestis isolates for 933 SNPs using Sequenom MassArray SNP typing. We conducted phylogenetic analyses on this sequence variation dataset, assigned isolates to populations based on maximum parsimony and, from these results, made inferences regarding historical transmission routes. Our phylogenetic analysis suggests that Y. pestis evolved in or near China and spread through multiple radiations to Europe, South America, Africa and Southeast Asia, leading to country-specific lineages that can be traced by lineage-specific SNPs. All 626 current isolates from the United States reflect one radiation, and 82 isolates from Madagascar represent a second radiation. Subsequent local microevolution of Y. pestis is marked by sequential, geographically specific SNPs.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI065359</span>, <span class="grant">N01 AI-30071</span>; Science Foundation Ireland: <span class="grant">05/FE1/B882</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">12</span>;<span class="pagerange">1140-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21037571" rel="external">21037571</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2999892" rel="external">2999892</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.705" rel="external">10.1038/ng.705</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EuroPhenome: a repository for high-throughput mouse phenotyping data.</h4>
<p class="authors"><span class="author">Morgan H</span>, <span class="author">Beck T</span>, <span class="author">Blake A</span>, <span class="author">Gates H</span>, <span class="author">Adams N</span>, <span class="author">Debouzy G</span>, <span class="author">Leblanc S</span>, <span class="author">Lengger C</span>, <span class="author">Maier H</span>, <span class="author">Melvin D</span>, <span class="author">Meziane H</span>, <span class="author">Richardson D</span>, <span class="author">Wells S</span>, <span class="author">White J</span>, <span class="author">Wood J</span>, <span class="author">EUMODIC Consortium</span>, <span class="author">de Angelis MH</span>, <span class="author">Brown SD</span>, <span class="author">Hancock JM</span> and <span class="author">Mallon AM</span></p>
<p class="affiliation">MRC Harwell, Mammalian Genetics Unit, MRC Harwell, Mary Lyon Centre, Harwell Science and Innovation Campus, Oxfordshire OX11 0RD, UK.</p><p class="abstract">The broad aim of biomedical science in the postgenomic era is to link genomic and phenotype information to allow deeper understanding of the processes leading from genomic changes to altered phenotype and disease. The EuroPhenome project (http://www.EuroPhenome.org) is a comprehensive resource for raw and annotated high-throughput phenotyping data arising from projects such as EUMODIC. EUMODIC is gathering data from the EMPReSSslim pipeline (http://www.empress.har.mrc.ac.uk/) which is performed on inbred mouse strains and knock-out lines arising from the EUCOMM project. The EuroPhenome interface allows the user to access the data via the phenotype or genotype. It also allows the user to access the data in a variety of ways, including graphical display, statistical analysis and access to the raw data via web services. The raw phenotyping data captured in EuroPhenome is annotated by an annotation pipeline which automatically identifies statistically different mutants from the appropriate baseline and assigns ontology terms for that specific test. Mutant phenotypes can be quickly identified using two EuroPhenome tools: PhenoMap, a graphical representation of statistically relevant phenotypes, and mining for a mutant using ontology terms. To assist with data definition and cross-database comparisons, phenotype data is annotated using combinations of terms from biological ontologies.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D577-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19933761" rel="external">19933761</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808931" rel="external">2808931</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp1007" rel="external">10.1093/nar/gkp1007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in SLC29A3, encoding an equilibrative nucleoside transporter ENT3, cause a familial histiocytosis syndrome (Faisalabad histiocytosis) and familial Rosai-Dorfman disease.</h4>
<p class="authors"><span class="author">Morgan NV</span>, <span class="author">Morris MR</span>, <span class="author">Cangul H</span>, <span class="author">Gleeson D</span>, <span class="author">Straatman-Iwanowska A</span>, <span class="author">Davies N</span>, <span class="author">Keenan S</span>, <span class="author">Pasha S</span>, <span class="author">Rahman F</span>, <span class="author">Gentle D</span>, <span class="author">Vreeswijk MP</span>, <span class="author">Devilee P</span>, <span class="author">Knowles MA</span>, <span class="author">Ceylaner S</span>, <span class="author">Trembath RC</span>, <span class="author">Dalence C</span>, <span class="author">Kismet E</span>, <span class="author">K&ouml;seo&#x11F;lu V</span>, <span class="author">Rossbach HC</span>, <span class="author">Gissen P</span>, <span class="author">Tannahill D</span> and <span class="author">Maher ER</span></p>
<p class="affiliation">Wellchild Paediatric Research Centre and Department of Medical and Molecular Genetics, University of Birmingham College of Medical and Dental Sciences, Edgbaston, Birmingham, United Kingdom.</p><p class="abstract">The histiocytoses are a heterogeneous group of disorders characterised by an excessive number of histiocytes. In most cases the pathophysiology is unclear and treatment is nonspecific. Faisalabad histiocytosis (FHC) (MIM 602782) has been classed as an autosomal recessively inherited form of histiocytosis with similarities to Rosai-Dorfman disease (RDD) (also known as sinus histiocytosis with massive lymphadenopathy (SHML)). To elucidate the molecular basis of FHC, we performed autozygosity mapping studies in a large consanguineous family and identified a novel locus at chromosome 10q22.1. Mutation analysis of candidate genes within the target interval identified biallelic germline mutations in SLC29A3 in the FHC kindred and in two families reported to have familial RDD. Analysis of SLC29A3 expression during mouse embryogenesis revealed widespread expression by e14.5 with prominent expression in the central nervous system, eye, inner ear, and epithelial tissues including the gastrointestinal tract. SLC29A3 encodes an intracellular equilibrative nucleoside transporter (hENT3) with affinity for adenosine. Recently germline mutations in SLC29A3 were also described in two rare autosomal recessive disorders with overlapping phenotypes: (a) H syndrome (MIM 612391) that is characterised by cutaneous hyperpigmentation and hypertrichosis, hepatomegaly, heart anomalies, hearing loss, and hypogonadism; and (b) PHID (pigmented hypertrichosis with insulin-dependent diabetes mellitus) syndrome. Our findings suggest that a variety of clinical diagnoses (H and PHID syndromes, FHC, and familial RDD) can be included in a new diagnostic category of SLC29A3 spectrum disorder.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">e1000833</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20140240" rel="external">20140240</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2816679" rel="external">2816679</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000833" rel="external">10.1371/journal.pgen.1000833</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of protective and broadly conserved vaccine antigens from the genome of extraintestinal pathogenic Escherichia coli.</h4>
<p class="authors"><span class="author">Moriel DG</span>, <span class="author">Bertoldi I</span>, <span class="author">Spagnuolo A</span>, <span class="author">Marchi S</span>, <span class="author">Rosini R</span>, <span class="author">Nesta B</span>, <span class="author">Pastorello I</span>, <span class="author">Corea VA</span>, <span class="author">Torricelli G</span>, <span class="author">Cartocci E</span>, <span class="author">Savino S</span>, <span class="author">Scarselli M</span>, <span class="author">Dobrindt U</span>, <span class="author">Hacker J</span>, <span class="author">Tettelin H</span>, <span class="author">Tallon LJ</span>, <span class="author">Sullivan S</span>, <span class="author">Wieler LH</span>, <span class="author">Ewers C</span>, <span class="author">Pickard D</span>, <span class="author">Dougan G</span>, <span class="author">Fontana MR</span>, <span class="author">Rappuoli R</span>, <span class="author">Pizza M</span> and <span class="author">Serino L</span></p>
<p class="affiliation">Novartis Vaccines and Diagnostics, 53100 Siena, Italy.</p><p class="abstract">Extraintestinal pathogenic Escherichia coli (ExPEC) are a common cause of disease in both mammals and birds. A vaccine to prevent such infections would be desirable given the increasing antibiotic resistance of these bacteria. We have determined the genome sequence of ExPEC IHE3034 (ST95) isolated from a case of neonatal meningitis and compared this to available genome sequences of other ExPEC strains and a few nonpathogenic E. coli. We found 19 genomic islands present in the genome of IHE3034, which are absent in the nonpathogenic E. coli isolates. By using subtractive reverse vaccinology we identified 230 antigens present in ExPEC but absent (or present with low similarity) in nonpathogenic strains. Nine antigens were protective in a mouse challenge model. Some of them were also present in other pathogenic non-ExPEC strains, suggesting that a broadly protective E. coli vaccine may be possible. The gene encoding the most protective antigen was detected in most of the E. coli isolates, highly conserved in sequence and found to be exported by a type II secretion system which seems to be nonfunctional in nonpathogenic strains.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2010</span>;<span class="volume">107</span>;<span class="number">20</span>;<span class="pagerange">9072-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20439758" rel="external">20439758</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2889118" rel="external">2889118</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0915077107" rel="external">10.1073/pnas.0915077107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessment of protein domain fusions in human protein interaction networks prediction: application to the human kinetochore model.</h4>
<p class="authors"><span class="author">Morilla I</span>, <span class="author">Lees JG</span>, <span class="author">Reid AJ</span>, <span class="author">Orengo C</span> and <span class="author">Ranea JA</span></p>
<p class="affiliation">Department of Molecular Biology and Biochemistry, University of Malaga, Malaga, Spain. <a href="mailto:%69%61%6E.%6D%6F%72%69%6C%6C%61%40%75%6D%61.%65%73">&#105;&#97;&#110;&#46;&#109;&#111;&#114;&#105;&#108;&#108;&#97;&#64;&#117;&#109;&#97;&#46;&#101;&#115;</a></p><p class="abstract">In order to understand how biological systems function it is necessary to determine the interactions and associations between proteins. Some proteins, involved in a common biological process and encoded by separate genes in one organism, can be found fused within a single protein chain in other organisms. By detecting these triplets, a functional relationship can be established between the unfused proteins. Here we use a domain fusion prediction method to predict these protein interactions for the human interactome. We observed that gene fusion events are more related to physical interaction between proteins than to other weaker functional relationships such as participation in a common biological pathway. These results suggest that domain fusion is an appropriate method for predicting protein complexes. The most reliable fused domain predictions were used to build protein-protein interaction (PPI) networks. These predicted PPI network models showed the same topological features as real biological networks and different features from random behaviour. We built the PPI domain fusion sub-network model of the human kinetochore and observed that the majority of the predicted interactions have not yet been experimentally characterised in the publicly available PPI repositories. The study of the human kinetochore domain fusion sub-network reveals undiscovered kinetochore proteins with presumably relevant functions, such as hubs with many connections in the kinetochore sub-network. These results suggest that experimentally hidden regions in the predicted PPI networks contain key functional elements, associated with important functional areas, still undiscovered in the human interactome. Until novel experiments shed light on these hidden regions; domain fusion predictions provide a valuable approach for exploring them.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">New biotechnology</span>&nbsp;<span class="year">2010</span>;<span class="volume">27</span>;<span class="number">6</span>;<span class="pagerange">755-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20851221" rel="external">20851221</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.nbt.2010.09.005" rel="external">10.1016/j.nbt.2010.09.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evaluation of statistical approaches to rare variant analysis in genetic association studies.</h4>
<p class="authors"><span class="author">Morris AP</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, United Kingdom. <a href="mailto:%61%6D%6F%72%72%69%73%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#97;&#109;&#111;&#114;&#114;&#105;&#115;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genome-wide association (GWA) studies have proved to be extremely successful in identifying novel common polymorphisms contributing effects to the genetic component underlying complex traits. Nevertheless, one source of, as yet, undiscovered genetic determinants of complex traits are those mediated through the effects of rare variants. With the increasing availability of large-scale re-sequencing data for rare variant discovery, we have developed a novel statistical method for the detection of complex trait associations with these loci, based on searching for accumulations of minor alleles within the same functional unit. We have undertaken simulations to evaluate strategies for the identification of rare variant associations in population-based genetic studies when data are available from re-sequencing discovery efforts or from commercially available GWA chips. Our results demonstrate that methods based on accumulations of rare variants discovered through re-sequencing offer substantially greater power than conventional analysis of GWA data, and thus provide an exciting opportunity for future discovery of genetic determinants of complex traits.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">064890</span>, <span class="grant">081682</span>, <span class="grant">WT081682/Z/06/Z</span>, <span class="grant">WT088885/Z/09/Z</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">34</span>;<span class="number">2</span>;<span class="pagerange">188-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19810025">19810025</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2962811">2962811</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20450">10.1002/gepi.20450</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A powerful approach to sub-phenotype analysis in population-based genetic association studies.</h4>
<p class="authors"><span class="author">Morris AP</span>, <span class="author">Lindgren CM</span>, <span class="author">Zeggini E</span>, <span class="author">Timpson NJ</span>, <span class="author">Frayling TM</span>, <span class="author">Hattersley AT</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, United Kingdom. <a href="mailto:%61%6D%6F%72%72%69%73%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#97;&#109;&#111;&#114;&#114;&#105;&#115;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The ultimate goal of genome-wide association (GWA) studies is to identify genetic variants contributing effects to complex phenotypes in order to improve our understanding of the biological architecture underlying the trait. One approach to allow us to meet this challenge is to consider more refined sub-phenotypes of disease, defined by pattern of symptoms, for example, which may be physiologically distinct, and thus may have different underlying genetic causes. The disadvantage of sub-phenotype analysis is that large disease cohorts are sub-divided into smaller case categories, thus reducing power to detect association. To address this issue, we have developed a novel test of association within a multinomial regression modeling framework, allowing for heterogeneity of genetic effects between sub-phenotypes. The modeling framework is extremely flexible, and can be generalized to any number of distinct sub-phenotypes. Simulations demonstrate the power of the multinomial regression-based analysis over existing methods when genetic effects differ between sub-phenotypes, with minimal loss of power when these effects are homogenous for the unified phenotype. Application of the multinomial regression analysis to a genome-wide association study of type 2 diabetes, with cases categorized according to body mass index, highlights previously recognized differential mechanisms underlying obese and non-obese forms of the disease, and provides evidence of a potential novel association that warrants follow-up in independent replication cohorts.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076113</span>, <span class="grant">081682</span>, <span class="grant">WT081682/Z/06/Z</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">34</span>;<span class="number">4</span>;<span class="pagerange">335-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20039379" rel="external">20039379</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2964510" rel="external">2964510</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20486" rel="external">10.1002/gepi.20486</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evoker: a visualization tool for genotype intensity data.</h4>
<p class="authors"><span class="author">Morris JA</span>, <span class="author">Randall JC</span>, <span class="author">Maller JB</span> and <span class="author">Barrett JC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1HH, UK.</p><p class="abstract"><strong>Summary</strong>: Genome-wide association studies (GWAS), which produce huge volumes of data, are now being carried out by many groups around the world, creating a need for user-friendly tools for data quality control (QC) and analysis. One critical aspect of GWAS QC is evaluating genotype cluster plots to verify sensible genotype calling in putatively associated single nucleotide polymorphisms (SNPs). Evoker is a tool for visualizing genotype cluster plots, and provides a solution to the computational and storage problems related to working with such large datasets.</p><p class="abstract"><strong>Availability</strong>: http://www.sanger.ac.uk/resources/software/evoker/</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089120</span>, <span class="grant">WT08912/Z/09/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">14</span>;<span class="pagerange">1786-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20507892">20507892</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3073232">3073232</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq280">10.1093/bioinformatics/btq280</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">C-kit gene mutations in adenoid cystic carcinoma are rare.</h4>
<p class="authors"><span class="author">Moskaluk CA</span>, <span class="author">Frierson HF</span>, <span class="author">El-Naggar AK</span> and <span class="author">Futreal PA</span></p>
<p class="publication"><span class="title">Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc</span>&nbsp;<span class="year">2010</span>;<span class="volume">23</span>;<span class="number">6</span>;<span class="pagerange">905-6; author reply 906-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20514080" rel="external">20514080</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmodpathol.2010.61" rel="external">10.1038/modpathol.2010.61</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Methods for Improving Genome Annotation</h4>
<p class="authors"><span class="author">Mudge,J.</span> and <span class="author">Harrow,J.</span></p>
<p class="publication"><span class="title">Knowledge-Based Bioinformatics: From Analysis to Interpretation</span>&nbsp;<span class="year">2010</span>;<span class="number">Chapter 9</span>;<span class="pagerange">209-32</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1002%2F9780470669716.ch9">10.1002/9780470669716.ch9</a>; URL: <a title="http://onlinelibrary.wiley.com/doi/10.1002/9780470669716.ch9/summary" href="http://onlinelibrary.wiley.com/doi/10.1002/9780470669716.ch9/summary">http://onlinelibrary.wiley.com/doi/<span class="print-hide">...</span><span class="print-show">10.1002/9780470669716.ch9/su</span>mmary</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intra- and interhost evolutionary dynamics of equine influenza virus.</h4>
<p class="authors"><span class="author">Murcia PR</span>, <span class="author">Baillie GJ</span>, <span class="author">Daly J</span>, <span class="author">Elton D</span>, <span class="author">Jervis C</span>, <span class="author">Mumford JA</span>, <span class="author">Newton R</span>, <span class="author">Parrish CR</span>, <span class="author">Hoelzer K</span>, <span class="author">Dougan G</span>, <span class="author">Parkhill J</span>, <span class="author">Lennard N</span>, <span class="author">Ormond D</span>, <span class="author">Moule S</span>, <span class="author">Whitwham A</span>, <span class="author">McCauley JW</span>, <span class="author">McKinley TJ</span>, <span class="author">Holmes EC</span>, <span class="author">Grenfell BT</span> and <span class="author">Wood JL</span></p>
<p class="affiliation">Cambridge Infectious Diseases Consortium, Department of Veterinary Medicine, University of Cambridge, Madingley Road, CB3 0ES Cambridge, England, United Kingdom.</p><p class="abstract">Determining the evolutionary basis of cross-species transmission and immune evasion is key to understanding the mechanisms that control the emergence of either new viruses or novel antigenic variants with pandemic potential. The hemagglutinin glycoprotein of influenza A viruses is a critical host range determinant and a major target of neutralizing antibodies. Equine influenza virus (EIV) is a significant pathogen of the horse that causes periodical outbreaks of disease even in populations with high vaccination coverage. EIV has also jumped the species barrier and emerged as a novel respiratory pathogen in dogs, canine influenza virus. We studied the dynamics of equine influenza virus evolution in horses at the intrahost level and how this evolutionary process is affected by interhost transmission in a natural setting. To this end, we performed clonal sequencing of the hemagglutinin 1 gene derived from individual animals at different times postinfection. Our results show that despite the population consensus sequence remaining invariant, genetically distinct subpopulations persist during the course of infection and are also transmitted, with some variants likely to change antigenicity. We also detected a natural case of mixed infection in an animal infected during an outbreak of equine influenza, raising the possibility of reassortment between different strains of virus. In sum, our data suggest that transmission bottlenecks may not be as narrow as originally perceived and that the genetic diversity required to adapt to new host species may be partially present in the donor host and potentially transmitted to the recipient host.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R24 HD047879</span>; NIGMS NIH HHS: <span class="grant">R01 GM080533</span>, <span class="grant">R01 GM083983-01</span>, <span class="grant">R01 GM083983-05</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">2010</span>;<span class="volume">84</span>;<span class="number">14</span>;<span class="pagerange">6943-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20444896" rel="external">20444896</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2898244" rel="external">2898244</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJVI.00112-10" rel="external">10.1128/JVI.00112-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The two most common histological subtypes of malignant germ cell tumour are distinguished by global microRNA profiles, associated with differential transcription factor expression.</h4>
<p class="authors"><span class="author">Murray MJ</span>, <span class="author">Saini HK</span>, <span class="author">van Dongen S</span>, <span class="author">Palmer RD</span>, <span class="author">Muralidhar B</span>, <span class="author">Pett MR</span>, <span class="author">Piipari M</span>, <span class="author">Thornton CM</span>, <span class="author">Nicholson JC</span>, <span class="author">Enright AJ</span> and <span class="author">Coleman N</span></p>
<p class="affiliation">Medical Research Council Cancer Cell Unit, Cambridge, CB2 0XZ, UK. <a href="mailto:%6D%6A%6D%31%36%40%68%75%74%63%68%69%73%6F%6E-%6D%72%63.%63%61%6D.%61%63.%75%6B">&#109;&#106;&#109;&#49;&#54;&#64;&#104;&#117;&#116;&#99;&#104;&#105;&#115;&#111;&#110;&#45;&#109;&#114;&#99;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: We hypothesised that differences in microRNA expression profiles contribute to the contrasting natural history and clinical outcome of the two most common types of malignant germ cell tumour (GCT), yolk sac tumours (YSTs) and germinomas.</p><p class="abstract"><strong>Results</strong>: By direct comparison, using microarray data for paediatric GCT samples and published qRT-PCR data for adult samples, we identified microRNAs significantly up-regulated in YSTs (n = 29 paediatric, 26 adult, 11 overlapping) or germinomas (n = 37 paediatric). By Taqman qRT-PCR we confirmed differential expression of 15 of 16 selected microRNAs and further validated six of these (miR-302b, miR-375, miR-200b, miR-200c, miR-122, miR-205) in an independent sample set. Interestingly, the miR-302 cluster, which is over-expressed in all malignant GCTs, showed further over-expression in YSTs versus germinomas, representing six of the top eight microRNAs over-expressed in paediatric YSTs and seven of the top 11 in adult YSTs. To explain this observation, we used mRNA expression profiles of paediatric and adult malignant GCTs to identify 10 transcription factors (TFs) consistently over-expressed in YSTs versus germinomas, followed by linear regression to confirm associations between TF and miR-302 cluster expression levels. Using the sequence motif analysis environment iMotifs, we identified predicted binding sites for four of the 10 TFs (GATA6, GATA3, TCF7L2 and MAF) in the miR-302 cluster promoter region. Finally, we showed that miR-302 family over-expression in YST is likely to be functionally significant, as mRNAs down-regulated in YSTs were enriched for 3&#39; untranslated region sequences complementary to the common seed of miR-302a~miR-302d. Such mRNAs included mediators of key cancer-associated processes, including tumour suppressor genes, apoptosis regulators and TFs.</p><p class="abstract"><strong>Conclusions</strong>: Differential microRNA expression is likely to contribute to the relatively aggressive behaviour of YSTs and may enable future improvements in clinical diagnosis and/or treatment.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">Molecular cancer</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="pagerange">290</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21059207" rel="external">21059207</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2993676" rel="external">2993676</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1476-4598-9-290" rel="external">10.1186/1476-4598-9-290</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Allelic variants of IL1R1 gene associate with severe hand osteoarthritis.</h4>
<p class="authors"><span class="author">N&auml;kki A</span>, <span class="author">Kouhia ST</span>, <span class="author">Saarela J</span>, <span class="author">Harilainen A</span>, <span class="author">Tallroth K</span>, <span class="author">Videman T</span>, <span class="author">Batti&eacute; MC</span>, <span class="author">Kaprio J</span>, <span class="author">Peltonen L</span> and <span class="author">Kujala UM</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: In search for genes predisposing to osteoarthritis (OA), several genome wide scans have provided evidence for linkage on 2q. In this study we targeted a 470 kb region on 2q11.2 presenting the locus with most evidence for linkage to severe OA of distal interphalangeal joints (DIP) in our genome wide scan families.</p><p class="abstract"><strong>Methods</strong>: We genotyped 32 single nucleotide polymorphisms (SNPs) in this 470 kb region comprising six genes belonging to the interleukin 1 superfamily and monitored for association with individual SNPs and SNP haplotypes among severe familial hand OA cases (material extended from our previous linkage study; n = 134), unrelated end-stage bilateral primary knee OA cases (n = 113), and population based controls (n = 436).</p><p class="abstract"><strong>Results</strong>: Four SNPs in the IL1R1 gene, mapping to a 125 kb LD block, provided evidence for association with hand OA in family-based and case-control analysis, the strongest association being with SNP rs2287047 (p-value = 0.0009).</p><p class="abstract"><strong>Conclusions</strong>: This study demonstrates an association between severe hand OA and IL1R1 gene. This gene represents a highly relevant biological candidate since it encodes protein that is a known modulator of inflammatory processes associated with joint destruction and resides within a locus providing consistent evidence for linkage to hand OA. As the observed association did not fully explain the linkage obtained in the previous study, it is plausible that also other variants in this genome region predispose to hand OA.</p>
<p class="publication"><span class="title">BMC medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20353565" rel="external">20353565</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2859817" rel="external">2859817</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2350-11-50" rel="external">10.1186/1471-2350-11-50</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Limited variation in vaccine candidate Plasmodium falciparum Merozoite Surface Protein-6 over multiple transmission seasons.</h4>
<p class="authors"><span class="author">Neal AT</span>, <span class="author">Jordan SJ</span>, <span class="author">Oliveira AL</span>, <span class="author">Hernandez JN</span>, <span class="author">Branch OH</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">William C Gorgas Center for Geographic Medicine, Division of Infectious Diseases, Department of Medicine, University of Alabama at Birmingham, Birmingham, AL 35294-2170, USA.</p><p class="abstract"><strong>Background</strong>: Plasmodium falciparum Merozoite Surface Protein-6 (PfMSP6) is a component of the complex proteinacious coat that surrounds P. falciparum merozoites. This location, and the presence of anti-PfMSP6 antibodies in P. falciparum-exposed individuals, makes PfMSP6 a potential blood stage vaccine target. However, genetic diversity has proven to be a major hurdle for vaccines targeting other blood stage P. falciparum antigens, and few endemic field studies assessing PfMSP6 gene diversity have been conducted. This study follows PfMSP6 diversity in the Peruvian Amazon from 2003 to 2006 and is the first longitudinal assessment of PfMSP6 sequence dynamics.</p><p class="abstract"><strong>Methods</strong>: Parasite DNA was extracted from 506 distinct P. falciparum infections spanning the transmission seasons from 2003 to 2006 as part of the Malaria Immunology and Genetics in the Amazon (MIGIA) cohort study near Iquitos, Peru. PfMSP6 was amplified from each sample using a nested PCR protocol, genotyped for allele class by agarose gel electrophoresis, and sequenced to detect diversity. Allele frequencies were analysed using JMP v.8.0.1.0 and correlated with clinical and epidemiological data collected as part of the MIGIA project.</p><p class="abstract"><strong>Results</strong>: Both PfMSP6 allele classes, K1-like and 3D7-like, were detected at the study site, confirming that both are globally distributed. Allele frequencies varied significantly between transmission seasons, with 3D7-class alleles dominating and K1-class alleles nearly disappearing in 2005 and 2006. There was a significant association between allele class and village location (p-value = 0.0008), but no statistically significant association between allele class and age, sex, or symptom status. No intra-allele class sequence diversity was detected.</p><p class="abstract"><strong>Conclusions</strong>: Both PfMSP6 allele classes are globally distributed, and this study shows that allele frequencies can fluctuate significantly between communities separated by only a few kilometres, and over time in the same community. By contrast, PfMSP6 was highly stable at the sequence level, with no SNPs detected in the 506 samples analysed. This limited diversity supports further investigation of PfMSP6 as a blood stage vaccine candidate, with the clear caveat that any such vaccine must either contain both alleles or generate cross-protective responses that react against both allele classes. Detailed immunoepidemiology studies are needed to establish the viability of these approaches before PfMSP6 advances further down the vaccine development pipeline.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI064831</span>, <span class="grant">R21 AI072421</span>, <span class="grant">R21 AI072421-02</span></p>
<p class="publication"><span class="title">Malaria journal</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="pagerange">138</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20497564" rel="external">20497564</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2881939" rel="external">2881939</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1475-2875-9-138" rel="external">10.1186/1475-2875-9-138</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interactions of dietary whole-grain intake with fasting glucose- and insulin-related genetic loci in individuals of European descent: a meta-analysis of 14 cohort studies.</h4>
<p class="authors"><span class="author">Nettleton JA</span>, <span class="author">McKeown NM</span>, <span class="author">Kanoni S</span>, <span class="author">Lemaitre RN</span>, <span class="author">Hivert MF</span>, <span class="author">Ngwa J</span>, <span class="author">van Rooij FJ</span>, <span class="author">Sonestedt E</span>, <span class="author">Wojczynski MK</span>, <span class="author">Ye Z</span>, <span class="author">Tanaka T</span>, <span class="author">Garcia M</span>, <span class="author">Anderson JS</span>, <span class="author">Follis JL</span>, <span class="author">Djousse L</span>, <span class="author">Mukamal K</span>, <span class="author">Papoutsakis C</span>, <span class="author">Mozaffarian D</span>, <span class="author">Zillikens MC</span>, <span class="author">Bandinelli S</span>, <span class="author">Bennett AJ</span>, <span class="author">Borecki IB</span>, <span class="author">Feitosa MF</span>, <span class="author">Ferrucci L</span>, <span class="author">Forouhi NG</span>, <span class="author">Groves CJ</span>, <span class="author">Hallmans G</span>, <span class="author">Harris T</span>, <span class="author">Hofman A</span>, <span class="author">Houston DK</span>, <span class="author">Hu FB</span>, <span class="author">Johansson I</span>, <span class="author">Kritchevsky SB</span>, <span class="author">Langenberg C</span>, <span class="author">Launer L</span>, <span class="author">Liu Y</span>, <span class="author">Loos RJ</span>, <span class="author">Nalls M</span>, <span class="author">Orho-Melander M</span>, <span class="author">Renstrom F</span>, <span class="author">Rice K</span>, <span class="author">Riserus U</span>, <span class="author">Rolandsson O</span>, <span class="author">Rotter JI</span>, <span class="author">Saylor G</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Sjogren P</span>, <span class="author">Smith A</span>, <span class="author">Steingr&iacute;msd&oacute;ttir L</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Wareham NJ</span>, <span class="author">Prokopenko I</span>, <span class="author">Pankow JS</span>, <span class="author">van Duijn CM</span>, <span class="author">Florez JC</span>, <span class="author">Witteman JC</span>, <span class="author">MAGIC Investigators</span>, <span class="author">Dupuis J</span>, <span class="author">Dedoussis GV</span>, <span class="author">Ordovas JM</span>, <span class="author">Ingelsson E</span>, <span class="author">Cupples L</span>, <span class="author">Siscovick DS</span>, <span class="author">Franks PW</span> and <span class="author">Meigs JB</span></p>
<p class="affiliation">Division of Epidemiology, Human Genetics, and Environmental Sciences, University of Texas Health Sciences Center, Houston, Houston, Texas, USA. <a href="mailto:%6A%65%6E%6E%69%66%65%72.%61.%6E%65%74%74%6C%65%74%6F%6E%40%75%74%68.%74%6D%63.%65%64%75">&#106;&#101;&#110;&#110;&#105;&#102;&#101;&#114;&#46;&#97;&#46;&#110;&#101;&#116;&#116;&#108;&#101;&#116;&#111;&#110;&#64;&#117;&#116;&#104;&#46;&#116;&#109;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Objective</strong>: Whole-grain foods are touted for multiple health benefits, including enhancing insulin sensitivity and reducing type 2 diabetes risk. Recent genome-wide association studies (GWAS) have identified several single nucleotide polymorphisms (SNPs) associated with fasting glucose and insulin concentrations in individuals free of diabetes. We tested the hypothesis that whole-grain food intake and genetic variation interact to influence concentrations of fasting glucose and insulin.</p><p class="abstract">Via meta-analysis of data from 14 cohorts comprising &sim; 48,000 participants of European descent, we studied interactions of whole-grain intake with loci previously associated in GWAS with fasting glucose (16 loci) and/or insulin (2 loci) concentrations. For tests of interaction, we considered a P value &lt;0.0028 (0.05 of 18 tests) as statistically significant.</p><p class="abstract"><strong>Results</strong>: Greater whole-grain food intake was associated with lower fasting glucose and insulin concentrations independent of demographics, other dietary and lifestyle factors, and BMI (&beta; [95% CI] per 1-serving-greater whole-grain intake: -0.009 mmol/l glucose [-0.013 to -0.005], P &lt; 0.0001 and -0.011 pmol/l [ln] insulin [-0.015 to -0.007], P = 0.0003). No interactions met our multiple testing-adjusted statistical significance threshold. The strongest SNP interaction with whole-grain intake was rs780094 (GCKR) for fasting insulin (P = 0.006), where greater whole-grain intake was associated with a smaller reduction in fasting insulin concentrations in those with the insulin-raising allele.</p><p class="abstract"><strong>Conclusions</strong>: Our results support the favorable association of whole-grain intake with fasting glucose and insulin and suggest a potential interaction between variation in GCKR and whole-grain intake in influencing fasting insulin concentrations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701863</span>, <span class="grant">G0902037</span>, <span class="grant">G19/35</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A100_1003</span>, <span class="grant">MC_UP_A620_1015</span>, <span class="grant">U1475000002</span>; NIA NIH HHS: <span class="grant">R01 AG032098</span></p>
<p class="publication"><span class="title">Diabetes care</span>&nbsp;<span class="year">2010</span>;<span class="volume">33</span>;<span class="number">12</span>;<span class="pagerange">2684-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20693352">20693352</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2992213">2992213</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdc10-1150">10.2337/dc10-1150</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phenotypic profiling of the human genome by time-lapse microscopy reveals cell division genes.</h4>
<p class="authors"><span class="author">Neumann B</span>, <span class="author">Walter T</span>, <span class="author">H&eacute;rich&eacute; JK</span>, <span class="author">Bulkescher J</span>, <span class="author">Erfle H</span>, <span class="author">Conrad C</span>, <span class="author">Rogers P</span>, <span class="author">Poser I</span>, <span class="author">Held M</span>, <span class="author">Liebel U</span>, <span class="author">Cetin C</span>, <span class="author">Sieckmann F</span>, <span class="author">Pau G</span>, <span class="author">Kabbe R</span>, <span class="author">W&uuml;nsche A</span>, <span class="author">Satagopam V</span>, <span class="author">Schmitz MH</span>, <span class="author">Chapuis C</span>, <span class="author">Gerlich DW</span>, <span class="author">Schneider R</span>, <span class="author">Eils R</span>, <span class="author">Huber W</span>, <span class="author">Peters JM</span>, <span class="author">Hyman AA</span>, <span class="author">Durbin R</span>, <span class="author">Pepperkok R</span> and <span class="author">Ellenberg J</span></p>
<p class="affiliation">MitoCheck Project Group, European Molecular Biology Laboratory (EMBL), Meyerhofstrasse 1, D-69117 Heidelberg, Germany.</p><p class="abstract">Despite our rapidly growing knowledge about the human genome, we do not know all of the genes required for some of the most basic functions of life. To start to fill this gap we developed a high-throughput phenotypic screening platform combining potent gene silencing by RNA interference, time-lapse microscopy and computational image processing. We carried out a genome-wide phenotypic profiling of each of the approximately 21,000 human protein-coding genes by two-day live imaging of fluorescently labelled chromosomes. Phenotypes were scored quantitatively by computational image processing, which allowed us to identify hundreds of human genes involved in diverse biological functions including cell division, migration and survival. As part of the Mitocheck consortium, this study provides an in-depth analysis of cell division phenotypes and makes the entire high-content data set available as a resource to the community.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077192</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7289</span>;<span class="pagerange">721-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20360735">20360735</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3108885">3108885</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08869">10.1038/nature08869</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Laser excitation power and the flow cytometric resolution of complex karyotypes.</h4>
<p class="authors"><span class="author">Ng BL</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kingdom. <a href="mailto:%62%6C%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#108;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The analytical resolution of individual chromosome peaks in the flow karyotype of cell lines is dependent on sample preparation and the detection sensitivity of the flow cytometer. We have investigated the effect of laser power on the resolution of chromosome peaks in cell lines with complex karyotypes. Chromosomes were prepared from a human gastric cancer cell line and a cell line from a patient with an abnormal phenotype using a modified polyamine isolation buffer. The stained chromosome suspensions were analyzed on a MoFlo sorter (Beckman Coulter) equipped with two water-cooled lasers (Coherent). A bivariate flow karyotype was obtained from each of the cell lines at various laser power settings and compared to a karyotype generated using laser power settings of 300 mW. The best separation of chromosome peaks was obtained with laser powers of 300 mW. This study demonstrates the requirement for high-laser powers for the accurate detection and purification of chromosomes, particularly from complex karyotypes, using a conventional flow cytometer.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span>, <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Cytometry. Part A : the journal of the International Society for Analytical Cytology</span>&nbsp;<span class="year">2010</span>;<span class="volume">77</span>;<span class="number">6</span>;<span class="pagerange">585-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20506467">20506467</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC4131141">4131141</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fcyto.a.20904">10.1002/cyto.a.20904</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The sudden dominance of blaCTX-M harbouring plasmids in Shigella spp. Circulating in Southern Vietnam.</h4>
<p class="authors"><span class="author">Nguyen NT</span>, <span class="author">Ha V</span>, <span class="author">Tran NV</span>, <span class="author">Stabler R</span>, <span class="author">Pham DT</span>, <span class="author">Le TM</span>, <span class="author">van Doorn HR</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Thomson N</span>, <span class="author">Campbell J</span>, <span class="author">Nguyen VM</span>, <span class="author">Tran TT</span>, <span class="author">Pham MV</span>, <span class="author">Cao TT</span>, <span class="author">Wren B</span>, <span class="author">Farrar J</span> and <span class="author">Baker S</span></p>
<p class="affiliation">The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam.</p><p class="abstract"><strong>Background</strong>: Plasmid mediated antimicrobial resistance in the Enterobacteriaceae is a global problem. The rise of CTX-M class extended spectrum beta lactamases (ESBLs) has been well documented in industrialized countries. Vietnam is representative of a typical transitional middle income country where the spectrum of infectious diseases combined with the spread of drug resistance is shifting and bringing new healthcare challenges.</p><p class="abstract"><strong>Methodology</strong>: We collected hospital admission data from the pediatric population attending the hospital for tropical diseases in Ho Chi Minh City with Shigella infections. Organisms were cultured from all enrolled patients and subjected to antimicrobial susceptibility testing. Those that were ESBL positive were subjected to further investigation. These investigations included PCR amplification for common ESBL genes, plasmid investigation, conjugation, microarray hybridization and DNA sequencing of a bla(CTX-M) encoding plasmid.</p><p class="abstract">We show that two different bla(CTX-M) genes are circulating in this bacterial population in this location. Sequence of one of the ESBL plasmids shows that rather than the gene being integrated into a preexisting MDR plasmid, the bla(CTX-M) gene is located on relatively simple conjugative plasmid. The sequenced plasmid (pEG356) carried the bla(CTX-M-24) gene on an ISEcp1 element and demonstrated considerable sequence homology with other IncFI plasmids.</p><p class="abstract"><strong>Significance</strong>: The rapid dissemination, spread of antimicrobial resistance and changing population of Shigella spp. concurrent with economic growth are pertinent to many other countries undergoing similar development. Third generation cephalosporins are commonly used empiric antibiotics in Ho Chi Minh City. We recommend that these agents should not be considered for therapy of dysentery in this setting.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300020</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="number">6</span>;<span class="pagerange">e702</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20544028">20544028</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2882334">2882334</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000702">10.1371/journal.pntd.0000702</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate causal regulatory effects by integration of expression QTLs with complex trait genetic associations.</h4>
<p class="authors"><span class="author">Nica AC</span>, <span class="author">Montgomery SB</span>, <span class="author">Dimas AS</span>, <span class="author">Stranger BE</span>, <span class="author">Beazley C</span>, <span class="author">Barroso I</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The recent success of genome-wide association studies (GWAS) is now followed by the challenge to determine how the reported susceptibility variants mediate complex traits and diseases. Expression quantitative trait loci (eQTLs) have been implicated in disease associations through overlaps between eQTLs and GWAS signals. However, the abundance of eQTLs and the strong correlation structure (LD) in the genome make it likely that some of these overlaps are coincidental and not driven by the same functional variants. In the present study, we propose an empirical methodology, which we call Regulatory Trait Concordance (RTC) that accounts for local LD structure and integrates eQTLs and GWAS results in order to reveal the subset of association signals that are due to cis eQTLs. We simulate genomic regions of various LD patterns with both a single or two causal variants and show that our score outperforms SNP correlation metrics, be they statistical (r(2)) or historical (D&#39;). Following the observation of a significant abundance of regulatory signals among currently published GWAS loci, we apply our method with the goal to prioritize relevant genes for each of the respective complex traits. We detect several potential disease-causing regulatory effects, with a strong enrichment for immunity-related conditions, consistent with the nature of the cell line tested (LCLs). Furthermore, we present an extension of the method in trans, where interrogating the whole genome for downstream effects of the disease variant can be informative regarding its unknown primary biological effect. We conclude that integrating cellular phenotype associations with organismal complex traits will facilitate the biological interpretation of the genetic effects on these traits.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">e1000895</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20369022">20369022</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2848550">2848550</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000895">10.1371/journal.pgen.1000895</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Out of the sequencer and into the wiki as we face new challenges in genome informatics.</h4>
<p class="authors"><span class="author">Ning Z</span> and <span class="author">Montgomery SB</span></p>
<p class="affiliation">Sequencing Informatics, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB101SA, UK. <a href="mailto:%7A%6E%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#122;&#110;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A report on the joint Cold Spring Harbor Laboratory/Wellcome Trust Conference &#39;Genome Informatics&#39;, 15-19 September 2010, Hinxton, Cambridge, UK.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">308</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21067526" rel="external">21067526</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3218654" rel="external">3218654</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-10-308" rel="external">10.1186/gb-2010-11-10-308</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Salmonella enterica serovar Typhimurium mutants completely lacking the F(0)F(1) ATPase are novel live attenuated vaccine strains.</h4>
<p class="authors"><span class="author">Northen H</span>, <span class="author">Paterson GK</span>, <span class="author">Constantino-Casas F</span>, <span class="author">Bryant CE</span>, <span class="author">Clare S</span>, <span class="author">Mastroeni P</span>, <span class="author">Peters SE</span> and <span class="author">Maskell DJ</span></p>
<p class="affiliation">Department of Veterinary Medicine, University of Cambridge, Cambridge, UK.</p><p class="abstract">The F(0)F(1) ATPase plays a central role in both the generation of ATP and the utilisation of ATP for cellular processes such as rotation of bacterial flagella. We have deleted the entire operon encoding the F(0)F(1) ATPase, as well as genes encoding individual F(0) or F(1) subunits, in Salmonella enteric serovar Typhimurium. These mutants were attenuated for virulence, as assessed by bacterial counts in the livers and spleens of intravenously infected mice. The attenuated in vivo growth of the entire atp operon mutant was complemented by the insertion of the atp operon into the malXY pseudogene region. Following clearance of the attenuated mutants from the organs, mice were protected against challenge with the virulent wild type parent strain. We have shown that the F(0)F(1) ATPase is important for bacterial growth in vivo and that atp mutants are effective live attenuated vaccines against Salmonella infection.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBS/S/N/2006/13095</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Vaccine</span>&nbsp;<span class="year">2010</span>;<span class="volume">28</span>;<span class="number">4</span>;<span class="pagerange">940-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19925904">19925904</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3898827">3898827</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vaccine.2009.10.146">10.1016/j.vaccine.2009.10.146</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Long- and short-term selective forces on malaria parasite genomes.</h4>
<p class="authors"><span class="author">Nygaard S</span>, <span class="author">Braunstein A</span>, <span class="author">Malsen G</span>, <span class="author">Van Dongen S</span>, <span class="author">Gardner PP</span>, <span class="author">Krogh A</span>, <span class="author">Otto TD</span>, <span class="author">Pain A</span>, <span class="author">Berriman M</span>, <span class="author">McAuliffe J</span>, <span class="author">Dermitzakis ET</span> and <span class="author">Jeffares DC</span></p>
<p class="affiliation">Bioinformatics Centre, University of Copenhagen, Copenhagen, Denmark.</p><p class="abstract">Plasmodium parasites, the causal agents of malaria, result in more than 1 million deaths annually. Plasmodium are unicellular eukaryotes with small &sim;23 Mb genomes encoding &sim;5200 protein-coding genes. The protein-coding genes comprise about half of these genomes. Although evolutionary processes have a significant impact on malaria control, the selective pressures within Plasmodium genomes are poorly understood, particularly in the non-protein-coding portion of the genome. We use evolutionary methods to describe selective processes in both the coding and non-coding regions of these genomes. Based on genome alignments of seven Plasmodium species, we show that protein-coding, intergenic and intronic regions are all subject to purifying selection and we identify 670 conserved non-genic elements. We then use genome-wide polymorphism data from P. falciparum to describe short-term selective processes in this species and identify some candidate genes for balancing (diversifying) selection. Our analyses suggest that there are many functional elements in the non-genic regions of these genomes and that adaptive evolution has occurred more frequently in the protein-coding regions of the genome.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085775/Z/08/Z</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20838588" rel="external">20838588</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2936524" rel="external">2936524</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001099" rel="external">10.1371/journal.pgen.1001099</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multi-locus sequence typing of enteroaggregative Escherichia coli isolates from Nigerian children uncovers multiple lineages.</h4>
<p class="authors"><span class="author">Okeke IN</span>, <span class="author">Wallace-Gadsden F</span>, <span class="author">Simons HR</span>, <span class="author">Matthews N</span>, <span class="author">Labar AS</span>, <span class="author">Hwang J</span> and <span class="author">Wain J</span></p>
<p class="affiliation">Department of Biology, Haverford College, Haverford, Pennsylvania, USA. <a href="mailto:%69%6F%6B%65%6B%65%40%68%61%76%65%72%66%6F%72%64.%65%64%75">&#105;&#111;&#107;&#101;&#107;&#101;&#64;&#104;&#97;&#118;&#101;&#114;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Enteroaggregative Escherichia coli (EAEC) are defined by their stacked-brick adherence pattern to human epithelial cells. There is no all-encompassing genetic marker for EAEC. The category is commonly implicated in diarrhea but research is hampered by perplexing heterogeneity.</p><p class="abstract">To identify key EAEC lineages, we applied multilocus sequence typing to 126 E. coli isolates from a Nigerian case-control study that showed aggregative adherence in the HEp-2 adherence assay, and 24 other EAEC strains from diverse locations. EAEC largely belonged to the A, B1 and D phylogenetic groups and only 7 (4.6%) isolates were in the B2 cluster. As many as 96 sequence types (STs) were identified but 60 (40%) of the EAEC strains belong to or are double locus variants of STs 10, 31, and 394. The remainder did not belong to predominant complexes. The most common ST complex, with predicted ancestor ST10, included 32 (21.3%) of the isolates. Significant age-related distribution suggests that weaned children in Nigeria are at risk for diarrhea from of ST10-complex EAEC. Phylogenetic group D EAEC strains, predominantly from ST31- and ST394 complexes, represented 38 (25.3%) of all isolates, include genome-sequenced strain 042, and possessed conserved chromosomal loci.</p><p class="abstract">We have developed a molecular phylogenetic framework, which demonstrates that although grouped by a shared phenotype, the category of &#39;EAEC&#39; encompasses multiple pathogenic lineages. Principal among isolates from Nigeria were ST10-complex EAEC that were associated with diarrhea in children over one year and ECOR D strains that share horizontally acquired loci.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">e14093</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21124856" rel="external">21124856</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2990770" rel="external">2990770</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0014093" rel="external">10.1371/journal.pone.0014093</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lechevalieria atacamensis sp. nov., Lechevalieria deserti sp. nov. and Lechevalieria roselyniae sp. nov., isolated from hyperarid soils.</h4>
<p class="authors"><span class="author">Okoro CK</span>, <span class="author">Bull AT</span>, <span class="author">Mutreja A</span>, <span class="author">Rong X</span>, <span class="author">Huang Y</span> and <span class="author">Goodfellow M</span></p>
<p class="affiliation">School of Biology, University of Newcastle, Newcastle-upon-Tyne, UK.</p><p class="abstract">The taxonomic positions of three Lechevalieria-like strains isolated from hyperarid soils of the Atacama Desert, Chile, were established by using a polyphasic approach. The organisms had chemical and morphological properties consistent with their classification in the genus Lechevalieria. They formed a distinct subclade in the Lechevalieria 16S rRNA gene clade and were most closely related to the type strain of Lechevalieria xinjiangensis. DNA-DNA relatedness data showed that each of the isolates and Lechevalieria xinjiangensis DSM 45081(T) belong to distinct genomic species. The new isolates and the type strains of recognized Lechevalieria species were readily distinguished based on a number of phenotypic properties. A combination of the genotypic and phenotypic data showed that the three isolates represent three novel species of the genus Lechevalieria. The names proposed for these taxa are Lechevalieria atacamensis sp. nov. (type strain C61(T) =CGMCC 4.5536(T) =NRRL B-24706(T)), Lechevalieria deserti sp. nov. (type strain C68(T) =CGMCC 4.5535(T) =NRRL B-24707(T)) and Lechevalieria roselyniae sp. nov. (type strain C81(T) =CGMCC 4.5537(T) =NRRL B-24708(T)).</p>
<p class="publication"><span class="title">International journal of systematic and evolutionary microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">60</span>;<span class="number">Pt 2</span>;<span class="pagerange">296-300</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19651742" rel="external">19651742</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fijs.0.009985-0" rel="external">10.1099/ijs.0.009985-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Synthetic associations in the context of genome-wide association scan signals.</h4>
<p class="authors"><span class="author">Orozco G</span>, <span class="author">Barrett JC</span> and <span class="author">Zeggini E</span></p>
<p class="affiliation">Arthritis Research UK Epidemiology Unit, University of Manchester, Manchester, UK.</p><p class="abstract">Genome-wide association studies (GWAS) have successfully identified a large number of genetic variants associated with complex traits, but these only explain a small proportion of the total heritability. It has been recently proposed that rare variants can create &#39;synthetic association&#39; signals in GWAS, by occurring more often in association with one of the alleles of a common tag single nucleotide polymorphism. While the ultimate evaluation of this hypothesis will require the completion of large-scale sequencing studies, it is informative to place it in the broader context of what is known about the genetic architecture of complex disease. In this review, we draw from empirical and theoretical data to summarize evidence showing that synthetic associations do not underlie many reported GWAS associations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT088885/Z/09/Z</span>, <span class="grant">WT089120/Z/09/Z</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">R2</span>;<span class="pagerange">R137-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20805105">20805105</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2953742">2953742</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq368">10.1093/hmg/ddq368</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dual RMCE for efficient re-engineering of mouse mutant alleles.</h4>
<p class="authors"><span class="author">Osterwalder M</span>, <span class="author">Galli A</span>, <span class="author">Rosen B</span>, <span class="author">Skarnes WC</span>, <span class="author">Zeller R</span> and <span class="author">Lopez-Rios J</span></p>
<p class="affiliation">Developmental Genetics, Department of Biomedicine, University of Basel, Basel, Switzerland.</p><p class="abstract">We have developed dual recombinase-mediated cassette exchange (dRMCE) to efficiently re-engineer the thousands of available conditional alleles in mouse embryonic stem cells. dRMCE takes advantage of the wild-type loxP and FRT sites present in these conditional alleles and in many gene-trap lines. dRMCE is a scalable, flexible tool to introduce tags, reporters and mutant coding regions into an endogenous locus of interest in an easy and highly efficient manner.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077188</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">11</span>;<span class="pagerange">893-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20953177">20953177</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3576631">3576631</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1521">10.1038/nmeth.1521</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Thioredoxin and glutathione systems differ in parasitic and free-living platyhelminths.</h4>
<p class="authors"><span class="author">Otero L</span>, <span class="author">Bonilla M</span>, <span class="author">Protasio AV</span>, <span class="author">Fern&aacute;ndez C</span>, <span class="author">Gladyshev VN</span> and <span class="author">Salinas G</span></p>
<p class="affiliation">C&aacute;tedra de Inmunolog&iacute;a, Facultad de Qu&iacute;mica, Instituto de Higiene, Universidad de la Rep&uacute;blica, Avda, A, Navarro 3051, Montevideo, Uruguay.</p><p class="abstract"><strong>Background</strong>: The thioredoxin and/or glutathione pathways occur in all organisms. They provide electrons for deoxyribonucleotide synthesis, function as antioxidant defenses, in detoxification, Fe/S biogenesis and participate in a variety of cellular processes. In contrast to their mammalian hosts, platyhelminth (flatworm) parasites studied so far, lack conventional thioredoxin and glutathione systems. Instead, they possess a linked thioredoxin-glutathione system with the selenocysteine-containing enzyme thioredoxin glutathione reductase (TGR) as the single redox hub that controls the overall redox homeostasis. TGR has been recently validated as a drug target for schistosomiasis and new drug leads targeting TGR have recently been identified for these platyhelminth infections that affect more than 200 million people and for which a single drug is currently available. Little is known regarding the genomic structure of flatworm TGRs, the expression of TGR variants and whether the absence of conventional thioredoxin and glutathione systems is a signature of the entire platyhelminth phylum.</p><p class="abstract"><strong>Results</strong>: We examine platyhelminth genomes and transcriptomes and find that all platyhelminth parasites (from classes Cestoda and Trematoda) conform to a biochemical scenario involving, exclusively, a selenium-dependent linked thioredoxin-glutathione system having TGR as a central redox hub. In contrast, the free-living platyhelminth Schmidtea mediterranea (Class Turbellaria) possesses conventional and linked thioredoxin and glutathione systems. We identify TGR variants in Schistosoma spp. derived from a single gene, and demonstrate their expression. We also provide experimental evidence that alternative initiation of transcription and alternative transcript processing contribute to the generation of TGR variants in platyhelminth parasites.</p><p class="abstract"><strong>Conclusions</strong>: Our results indicate that thioredoxin and glutathione pathways differ in parasitic and free-living flatworms and that canonical enzymes were specifically lost in the parasitic lineage. Platyhelminth parasites possess a unique and simplified redox system for diverse essential processes, and thus TGR is an excellent drug target for platyhelminth infections. Inhibition of the central redox wire hub would lead to overall disruption of redox homeostasis and disable DNA synthesis.</p>
<p class="grants">Funded by: FIC NIH HHS: <span class="grant">TW006959</span>; NIGMS NIH HHS: <span class="grant">GM065204</span>; Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">237</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20385027">20385027</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873472">2873472</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-237">10.1186/1471-2164-11-237</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Seeking perfection.</h4>
<p class="authors"><span class="author">Otto TD</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">681</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20844555" rel="external">20844555</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2446" rel="external">10.1038/nrmicro2446</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Iterative Correction of Reference Nucleotides (iCORN) using second generation sequencing technology.</h4>
<p class="authors"><span class="author">Otto TD</span>, <span class="author">Sanders M</span>, <span class="author">Berriman M</span> and <span class="author">Newbold C</span></p>
<p class="affiliation">Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK. <a href="mailto:%74%64%6F%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#100;&#111;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: The accuracy of reference genomes is important for downstream analysis but a low error rate requires expensive manual interrogation of the sequence. Here, we describe a novel algorithm (Iterative Correction of Reference Nucleotides) that iteratively aligns deep coverage of short sequencing reads to correct errors in reference genome sequences and evaluate their accuracy.</p><p class="abstract"><strong>Results</strong>: Using Plasmodium falciparum (81% A + T content) as an extreme example, we show that the algorithm is highly accurate and corrects over 2000 errors in the reference sequence. We give examples of its application to numerous other eukaryotic and prokaryotic genomes and suggest additional applications.</p><p class="abstract"><strong>Availability</strong>: The software is available at http://icorn.sourceforge.net</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">14</span>;<span class="pagerange">1704-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20562415">20562415</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2894513">2894513</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq269">10.1093/bioinformatics/btq269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New insights into the blood-stage transcriptome of Plasmodium falciparum using RNA-Seq.</h4>
<p class="authors"><span class="author">Otto TD</span>, <span class="author">Wilinski D</span>, <span class="author">Assefa S</span>, <span class="author">Keane TM</span>, <span class="author">Sarry LR</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Lemieux J</span>, <span class="author">Barrell B</span>, <span class="author">Pain A</span>, <span class="author">Berriman M</span>, <span class="author">Newbold C</span> and <span class="author">Llin&aacute;s M</span></p>
<p class="affiliation">Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Recent advances in high-throughput sequencing present a new opportunity to deeply probe an organism&#39;s transcriptome. In this study, we used Illumina-based massively parallel sequencing to gain new insight into the transcriptome (RNA-Seq) of the human malaria parasite, Plasmodium falciparum. Using data collected at seven time points during the intraerythrocytic developmental cycle, we (i) detect novel gene transcripts; (ii) correct hundreds of gene models; (iii) propose alternative splicing events; and (iv) predict 5&#39; and 3&#39; untranslated regions. Approximately 70% of the unique sequencing reads map to previously annotated protein-coding genes. The RNA-Seq results greatly improve existing annotation of the P. falciparum genome with over 10% of gene models modified. Our data confirm 75% of predicted splice sites and identify 202 new splice sites, including 84 previously uncharacterized alternative splicing events. We also discovered 107 novel transcripts and expression of 38 pseudogenes, with many demonstrating differential expression across the developmental time series. Our RNA-Seq results correlate well with DNA microarray analysis performed in parallel on the same samples, and provide improved resolution over the microarray-based method. These data reveal new features of the P. falciparum transcriptional landscape and significantly advance our understanding of the parasite&#39;s red blood cell-stage transcriptome.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">P50 GM071508</span>; Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">76</span>;<span class="number">1</span>;<span class="pagerange">12-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20141604">20141604</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2859250">2859250</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2009.07026.x">10.1111/j.1365-2958.2009.07026.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of blood pressure extremes identifies variant near UMOD associated with hypertension.</h4>
<p class="authors"><span class="author">Padmanabhan S</span>, <span class="author">Melander O</span>, <span class="author">Johnson T</span>, <span class="author">Di Blasio AM</span>, <span class="author">Lee WK</span>, <span class="author">Gentilini D</span>, <span class="author">Hastie CE</span>, <span class="author">Menni C</span>, <span class="author">Monti MC</span>, <span class="author">Delles C</span>, <span class="author">Laing S</span>, <span class="author">Corso B</span>, <span class="author">Navis G</span>, <span class="author">Kwakernaak AJ</span>, <span class="author">van der Harst P</span>, <span class="author">Bochud M</span>, <span class="author">Maillard M</span>, <span class="author">Burnier M</span>, <span class="author">Hedner T</span>, <span class="author">Kjeldsen S</span>, <span class="author">Wahlstrand B</span>, <span class="author">Sj&ouml;gren M</span>, <span class="author">Fava C</span>, <span class="author">Montagnana M</span>, <span class="author">Danese E</span>, <span class="author">Torffvit O</span>, <span class="author">Hedblad B</span>, <span class="author">Snieder H</span>, <span class="author">Connell JM</span>, <span class="author">Brown M</span>, <span class="author">Samani NJ</span>, <span class="author">Farrall M</span>, <span class="author">Cesana G</span>, <span class="author">Mancia G</span>, <span class="author">Signorini S</span>, <span class="author">Grassi G</span>, <span class="author">Eyheramendy S</span>, <span class="author">Wichmann HE</span>, <span class="author">Laan M</span>, <span class="author">Strachan DP</span>, <span class="author">Sever P</span>, <span class="author">Shields DC</span>, <span class="author">Stanton A</span>, <span class="author">Vollenweider P</span>, <span class="author">Teumer A</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Rettig R</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Arora P</span>, <span class="author">Zhang F</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Lucas G</span>, <span class="author">Kathiresan S</span>, <span class="author">Siscovick DS</span>, <span class="author">Luan J</span>, <span class="author">Loos RJ</span>, <span class="author">Wareham NJ</span>, <span class="author">Penninx BW</span>, <span class="author">Nolte IM</span>, <span class="author">McBride M</span>, <span class="author">Miller WH</span>, <span class="author">Nicklin SA</span>, <span class="author">Baker AH</span>, <span class="author">Graham D</span>, <span class="author">McDonald RA</span>, <span class="author">Pell JP</span>, <span class="author">Sattar N</span>, <span class="author">Welsh P</span>, <span class="author">Global BPgen Consortium</span>, <span class="author">Munroe P</span>, <span class="author">Caulfield MJ</span>, <span class="author">Zanchetti A</span> and <span class="author">Dominiczak AF</span></p>
<p class="affiliation">Institute of Cardiovascular and Medical Sciences, College of Medical Veterinary and Life Sciences, University of Glasgow, Glasgow, United Kingdom.</p><p class="abstract">Hypertension is a heritable and major contributor to the global burden of disease. The sum of rare and common genetic variants robustly identified so far explain only 1%-2% of the population variation in BP and hypertension. This suggests the existence of more undiscovered common variants. We conducted a genome-wide association study in 1,621 hypertensive cases and 1,699 controls and follow-up validation analyses in 19,845 cases and 16,541 controls using an extreme case-control design. We identified a locus on chromosome 16 in the 5&#39; region of Uromodulin (UMOD; rs13333226, combined P value of 3.6 &times; 10&#x207B;&sup1;&sup1;). The minor G allele is associated with a lower risk of hypertension (OR [95%CI]: 0.87 [0.84-0.91]), reduced urinary uromodulin excretion, better renal function; and each copy of the G allele is associated with a 7.7% reduction in risk of CVD events after adjusting for age, sex, BMI, and smoking status (H.R.&#x200A;=&#x200A;0.923, 95% CI 0.860-0.991; p&#x200A;=&#x200A;0.027). In a subset of 13,446 individuals with estimated glomerular filtration rate (eGFR) measurements, we show that rs13333226 is independently associated with hypertension (unadjusted for eGFR: 0.89 [0.83-0.96], p&#x200A;=&#x200A;0.004; after eGFR adjustment: 0.89 [0.83-0.96], p&#x200A;=&#x200A;0.003). In clinical functional studies, we also consistently show the minor G allele is associated with lower urinary uromodulin excretion. The exclusive expression of uromodulin in the thick portion of the ascending limb of Henle suggests a putative role of this variant in hypertension through an effect on sodium homeostasis. The newly discovered UMOD locus for hypertension has the potential to give new insights into the role of uromodulin in BP regulation and to identify novel drugable targets for reducing cardiovascular risk.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">CH/98001</span>, <span class="grant">FS/05/095/19937</span>, <span class="grant">RG/07/005/23633</span>, <span class="grant">SP/08/005/25115</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">e1001177</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21082022" rel="external">21082022</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2965757" rel="external">2965757</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001177" rel="external">10.1371/journal.pgen.1001177</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multi-heuristic dynamic task allocation using genetic algorithms in a heterogeneous distributed system.</h4>
<p class="authors"><span class="author">Page AJ</span>, <span class="author">Keane TM</span> and <span class="author">Naughton TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK.</p>
<p class="abstract">We present a multi-heuristic evolutionary task allocation algorithm to dynamically map tasks to processors in a heterogeneous distributed system. It utilizes a genetic algorithm, combined with eight common heuristics, in an effort to minimize the total execution time. It operates on batches of unmapped tasks and can preemptively remap tasks to processors. The algorithm has been implemented on a Java distributed system and evaluated with a set of six problems from the areas of bioinformatics, biomedical engineering, computer science and cryptography. Experiments using up to 150 heterogeneous processors show that the algorithm achieves better efficiency than other state-of-the-art heuristic algorithms.</p>
<p class="publication"><span class="title">Journal of parallel and distributed computing</span>&nbsp;<span class="year">2010</span>;<span class="volume">70</span>;<span class="number">7</span>;<span class="pagerange">758-766</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20862190">20862190</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2927021">2927021</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jpdc.2010.03.011">10.1016/j.jpdc.2010.03.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a family with rare deletions in CNTNAP5 and DOCK4 suggests novel risk loci for autism and dyslexia.</h4>
<p class="authors"><span class="author">Pagnamenta AT</span>, <span class="author">Bacchelli E</span>, <span class="author">de Jonge MV</span>, <span class="author">Mirza G</span>, <span class="author">Scerri TS</span>, <span class="author">Minopoli F</span>, <span class="author">Chiocchetti A</span>, <span class="author">Ludwig KU</span>, <span class="author">Hoffmann P</span>, <span class="author">Paracchini S</span>, <span class="author">Lowy E</span>, <span class="author">Harold DH</span>, <span class="author">Chapman JA</span>, <span class="author">Klauck SM</span>, <span class="author">Poustka F</span>, <span class="author">Houben RH</span>, <span class="author">Staal WG</span>, <span class="author">Ophoff RA</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Williams J</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Schulte-K&ouml;rne G</span>, <span class="author">Deloukas P</span>, <span class="author">Ragoussis J</span>, <span class="author">Bailey AJ</span>, <span class="author">Maestrini E</span>, <span class="author">Monaco AP</span> and <span class="author">International Molecular Genetic Study Of Autism Consortium</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Autism spectrum disorders (ASDs) are characterized by social, communication, and behavioral deficits and complex genetic etiology. A recent study of 517 ASD families implicated DOCK4 by single nucleotide polymorphism (SNP) association and a microdeletion in an affected sibling pair.</p><p class="abstract"><strong>Methods</strong>: The DOCK4 microdeletion on 7q31.1 was further characterized in this family using QuantiSNP analysis of 1M SNP array data and reverse transcription polymerase chain reaction. Extended family members were tested by polymerase chain reaction amplification of junction fragments. DOCK4 dosage was measured in additional samples using SNP arrays. Since QuantiSNP analysis identified a novel CNTNAP5 microdeletion in the same affected sibling pair, this gene was sequenced in 143 additional ASD families. Further polymerase chain reaction-restriction fragment length polymorphism analysis included 380 ASD cases and suitable control subjects.</p><p class="abstract"><strong>Results</strong>: The maternally inherited microdeletion encompassed chr7:110,663,978-111,257,682 and led to a DOCK4-IMMP2L fusion transcript. It was also detected in five extended family members with no ASD. However, six of nine individuals with this microdeletion had poor reading ability, which prompted us to screen 606 other dyslexia cases. This led to the identification of a second DOCK4 microdeletion co-segregating with dyslexia. Assessment of genomic background in the original ASD family detected a paternal 2q14.3 microdeletion disrupting CNTNAP5 that was also transmitted to both affected siblings. Analysis of other ASD cohorts revealed four additional rare missense changes in CNTNAP5. No exonic deletions of DOCK4 or CNTNAP5 were seen in 2091 control subjects.</p><p class="abstract"><strong>Conclusions</strong>: This study highlights two new risk factors for ASD and dyslexia and demonstrates the importance of performing a high-resolution assessment of genomic background, even after detection of a rare and likely damaging microdeletion using a targeted approach.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02.</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">68</span>;<span class="number">4</span>;<span class="pagerange">320-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20346443" rel="external">20346443</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2941017" rel="external">2941017</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2010.02.002" rel="external">10.1016/j.biopsych.2010.02.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of three novel superantigen-encoding genes in Streptococcus equi subsp. zooepidemicus, szeF, szeN, and szeP.</h4>
<p class="authors"><span class="author">Paillot R</span>, <span class="author">Darby AC</span>, <span class="author">Robinson C</span>, <span class="author">Wright NL</span>, <span class="author">Steward KF</span>, <span class="author">Anderson E</span>, <span class="author">Webb K</span>, <span class="author">Holden MT</span>, <span class="author">Efstratiou A</span>, <span class="author">Broughton K</span>, <span class="author">Jolley KA</span>, <span class="author">Priestnall SL</span>, <span class="author">Marotti Campi MC</span>, <span class="author">Hughes MA</span>, <span class="author">Radford A</span>, <span class="author">Erles K</span> and <span class="author">Waller AS</span></p>
<p class="affiliation">Centre for Preventive Medicine, Animal Health Trust, Kentford, Newmarket, Suffolk, United Kingdom.</p><p class="abstract">The acquisition of superantigen-encoding genes by Streptococcus pyogenes has been associated with increased morbidity and mortality in humans, and the gain of four superantigens by Streptococcus equi is linked to the evolution of this host-restricted pathogen from an ancestral strain of the opportunistic pathogen Streptococcus equi subsp. zooepidemicus. A recent study determined that the culture supernatants of several S. equi subsp. zooepidemicus strains possessed mitogenic activity but lacked known superantigen-encoding genes. Here, we report the identification and activities of three novel superantigen-encoding genes. The products of szeF, szeN, and szeP share 59%, 49%, and 34% amino acid sequence identity with SPEH, SPEM, and SPEL, respectively. Recombinant SzeF, SzeN, and SzeP stimulated the proliferation of equine peripheral blood mononuclear cells, and tumor necrosis factor alpha (TNF-&alpha;) and gamma interferon (IFN-&gamma;) production, in vitro. Although none of these superantigen genes were encoded within functional prophage elements, szeN and szeP were located next to a prophage remnant, suggesting that they were acquired by horizontal transfer. Eighty-one of 165 diverse S. equi subsp. zooepidemicus strains screened, including 7 out of 15 isolates from cases of disease in humans, contained at least one of these new superantigen-encoding genes. The presence of szeN or szeP, but not szeF, was significantly associated with mitogenic activity in the S. equi subsp. zooepidemicus population (P &lt; 0.000001, P &lt; 0.000001, and P = 0.104, respectively). We conclude that horizontal transfer of these novel superantigens from and within the diverse S. equi subsp. zooepidemicus population is likely to have implications for veterinary and human disease.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2010</span>;<span class="volume">78</span>;<span class="number">11</span>;<span class="pagerange">4817-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20713629" rel="external">20713629</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2976332" rel="external">2976332</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00751-10" rel="external">10.1128/IAI.00751-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of susceptibility loci at 7q31 and 9p13 for bipolar disorder in an isolated population.</h4>
<p class="authors"><span class="author">Palo OM</span>, <span class="author">Soronen P</span>, <span class="author">Silander K</span>, <span class="author">Varilo T</span>, <span class="author">Tuononen K</span>, <span class="author">Kiesepp&auml; T</span>, <span class="author">Partonen T</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Paunio T</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">FIMM, Institute for Molecular Medicine and National Institute for Health and Welfare, Helsinki, Finland.</p><p class="abstract">We performed a linkage analysis on 23 Finnish families with bipolar disorder and originating from the North-Eastern region of Finland, using the Illumina Linkage Panel IV (6K) Array with an average intermarker spacing of 0.65 cM across the genome. We detected genome-wide significant evidence for linkage of mood disorder (bipolar disorder type I, II, or not otherwise specified, manic type of schizoaffective psychosis, cyclothymia, or recurrent depression) to chromosomes 7q31 (LOD = 3.20) and 9p13.1 (LOD = 4.02). Analyzing the best markers on the complete set of 179 Finnish bipolar families supported the findings on chromosome 9p13 (maximum LOD score of 3.02 at position 383 Mb, immediately upstream of the centromere). This region harbors several interesting candidate genes, including contactin associated protein-like 3 (CNTNAP3) and aldehyde dehydrogenase 1 (ALDH1B1). For the 7q31 locus, only one extended pedigree and ten families originating from the same late settlement region in North-Eastern Finland provided evidence for linkage, suggesting that a gene predisposing to bipolar disorder is enriched in that region. Candidate genes of interest in this locus include potassium-voltage-gated channel, member 2 (KCND2) and calcium-dependent activator protein for secretion 2 (CADPS2). The loci on the centromeric region of 9p13 and the telomeric region of 7q31 may represent susceptibility loci for mood disorder in the Finnish population.</p>
<p class="publication"><span class="title">American journal of medical genetics. Part B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric Genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">153B</span>;<span class="number">3</span>;<span class="pagerange">723-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19851985">19851985</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.b.31039">10.1002/ajmg.b.31039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Towards a comprehensive structural variation map of an individual human genome.</h4>
<p class="authors"><span class="author">Pang AW</span>, <span class="author">MacDonald JR</span>, <span class="author">Pinto D</span>, <span class="author">Wei J</span>, <span class="author">Rafiq MA</span>, <span class="author">Conrad DF</span>, <span class="author">Park H</span>, <span class="author">Hurles ME</span>, <span class="author">Lee C</span>, <span class="author">Venter JC</span>, <span class="author">Kirkness EF</span>, <span class="author">Levy S</span>, <span class="author">Feuk L</span> and <span class="author">Scherer SW</span></p>
<p class="affiliation">Department of Molecular Genetics, University of Toronto, 1 King&#39;s College Circle, Toronto, Ontario M5S 1A8, Canada. <a href="mailto:%61%6E%64%79%70%61%6E%67%40%73%69%63%6B%6B%69%64%73.%63%61">&#97;&#110;&#100;&#121;&#112;&#97;&#110;&#103;&#64;&#115;&#105;&#99;&#107;&#107;&#105;&#100;&#115;&#46;&#99;&#97;</a></p><p class="abstract"><strong>Background</strong>: Several genomes have now been sequenced, with millions of genetic variants annotated. While significant progress has been made in mapping single nucleotide polymorphisms (SNPs) and small (&lt;10 bp) insertion/deletions (indels), the annotation of larger structural variants has been less comprehensive. It is still unclear to what extent a typical genome differs from the reference assembly, and the analysis of the genomes sequenced to date have shown varying results for copy number variation (CNV) and inversions.</p><p class="abstract"><strong>Results</strong>: We have combined computational re-analysis of existing whole genome sequence data with novel microarray-based analysis, and detect 12,178 structural variants covering 40.6 Mb that were not reported in the initial sequencing of the first published personal genome. We estimate a total non-SNP variation content of 48.8 Mb in a single genome. Our results indicate that this genome differs from the consensus reference sequence by approximately 1.2% when considering indels/CNVs, 0.1% by SNPs and approximately 0.3% by inversions. The structural variants impact 4,867 genes, and &gt;24% of structural variants would not be imputed by SNP-association.</p><p class="abstract"><strong>Conclusions</strong>: Our results indicate that a large number of structural variants have been unreported in the individual genomes published to date. This significant extent and complexity of structural variants, as well as the growing recognition of their medical relevance, necessitate they be actively studied in health-related analyses of personal genomes. The new catalogue of structural variants generated for this genome provides a crucial resource for future comparison studies.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">5</span>;<span class="pagerange">R52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20482838" rel="external">20482838</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2898065" rel="external">2898065</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-5-r52" rel="external">10.1186/gb-2010-11-5-r52</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A locus on chromosome 1p36 is associated with thyrotropin and thyroid function as identified by genome-wide association study.</h4>
<p class="authors"><span class="author">Panicker V</span>, <span class="author">Wilson SG</span>, <span class="author">Walsh JP</span>, <span class="author">Richards JB</span>, <span class="author">Brown SJ</span>, <span class="author">Beilby JP</span>, <span class="author">Bremner AP</span>, <span class="author">Surdulescu GL</span>, <span class="author">Qweitin E</span>, <span class="author">Gillham-Nasenya I</span>, <span class="author">Soranzo N</span>, <span class="author">Lim EM</span>, <span class="author">Fletcher SJ</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Department of Endocrinology and Diabetes, Sir Charles Gairdner Hospital, Nedlands, Western Australia.</p><p class="abstract">Thyroid hormones are key regulators of cellular growth, development, and metabolism, and thyroid disorders are a common cause of ill health in the community. Circulating concentrations of thyrotropin (TSH), thyroxine (T4) and triiodothyronine (T3) have a strong heritable component and are thought to be under polygenic control, but the genes responsible are mostly unknown. In order to identify genetic loci associated with these metabolic phenotypes, we performed a genome-wide association study of 2,120,505 SNPs in 2014 female twins from the TwinsUK study and found a significant association between rs10917469 on chromosome 1p36.13 and serum TSH (p = 3.2 &times; 10(-8)). The association of rs10917469 with serum TSH was replicated (p = 2.0 &times; 10(-4)) in an independent community-based sample of 1154 participants in the Busselton Health Study. This SNP is located near CAPZB, which might be a regulator of TSH secretion and thus of pituitary-thyroid axis function. Twenty-nine percent of white individuals carry the variant, and the difference in mean TSH concentrations between wild-type individuals and those homozygous for the minor G allele was 0.5 mU/l, which is likely to be clinically relevant. We also provide evidence of suggestive association (p &lt; 5.0 &times; 10(-6)) of other SNPs with serum TSH, free T4, and free T3 concentrations, and these SNPs might be good targets for further studies. These results advance understanding of the genetic basis of pituitary-thyroid axis function and metabolic regulation.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">87</span>;<span class="number">3</span>;<span class="pagerange">430-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20826269" rel="external">20826269</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2933351" rel="external">2933351</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.08.005" rel="external">10.1016/j.ajhg.2010.08.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The RING-CH ligase K5 antagonizes restriction of KSHV and HIV-1 particle release by mediating ubiquitin-dependent endosomal degradation of tetherin.</h4>
<p class="authors"><span class="author">Pardieu C</span>, <span class="author">Vigan R</span>, <span class="author">Wilson SJ</span>, <span class="author">Calvi A</span>, <span class="author">Zang T</span>, <span class="author">Bieniasz P</span>, <span class="author">Kellam P</span>, <span class="author">Towers GJ</span> and <span class="author">Neil SJ</span></p>
<p class="affiliation">MRC Centre for Medical Molecular Virology, University College London, London, United Kingdom.</p><p class="abstract">Tetherin (CD317/BST2) is an interferon-induced membrane protein that inhibits the release of diverse enveloped viral particles. Several mammalian viruses have evolved countermeasures that inactivate tetherin, with the prototype being the HIV-1 Vpu protein. Here we show that the human herpesvirus Kaposi&#39;s sarcoma-associated herpesvirus (KSHV) is sensitive to tetherin restriction and its activity is counteracted by the KSHV encoded RING-CH E3 ubiquitin ligase K5. Tetherin expression in KSHV-infected cells inhibits viral particle release, as does depletion of K5 protein using RNA interference. K5 induces a species-specific downregulation of human tetherin from the cell surface followed by its endosomal degradation. We show that K5 targets a single lysine (K18) in the cytoplasmic tail of tetherin for ubiquitination, leading to relocalization of tetherin to CD63-positive endosomal compartments. Tetherin degradation is dependent on ESCRT-mediated endosomal sorting, but does not require a tyrosine-based sorting signal in the tetherin cytoplasmic tail. Importantly, we also show that the ability of K5 to substitute for Vpu in HIV-1 release is entirely dependent on K18 and the RING-CH domain of K5. By contrast, while Vpu induces ubiquitination of tetherin cytoplasmic tail lysine residues, mutation of these positions has no effect on its antagonism of tetherin function, and residual tetherin is associated with the trans-Golgi network (TGN) in Vpu-expressing cells. Taken together our results demonstrate that K5 is a mechanistically distinct viral countermeasure to tetherin-mediated restriction, and that herpesvirus particle release is sensitive to this mode of antiviral inhibition.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0801172</span>, <span class="grant">G0801172(87743)</span>, <span class="grant">G0801937</span>, <span class="grant">G9721629</span>; Wellcome Trust: <span class="grant">076608</span>, <span class="grant">WT082274MA</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">e1000843</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20419159">20419159</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2855335">2855335</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.1000843">10.1371/journal.ppat.1000843</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An expanded Oct4 interaction network: implications for stem cell biology, development, and disease.</h4>
<p class="authors"><span class="author">Pardo M</span>, <span class="author">Lang B</span>, <span class="author">Yu L</span>, <span class="author">Prosser H</span>, <span class="author">Bradley A</span>, <span class="author">Babu MM</span> and <span class="author">Choudhary J</span></p>
<p class="affiliation">Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK. <a href="mailto:%6D%70%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#112;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The transcription factor Oct4 is key in embryonic stem cell identity and reprogramming. Insight into its partners should illuminate how the pluripotent state is established and regulated. Here, we identify a considerably expanded set of Oct4-binding proteins in mouse embryonic stem cells. We find that Oct4 associates with a varied set of proteins including regulators of gene expression and modulators of Oct4 function. Half of its partners are transcriptionally regulated by Oct4 itself or other stem cell transcription factors, whereas one-third display a significant change in expression upon cell differentiation. The majority of Oct4-associated proteins studied to date show an early lethal phenotype when mutated. A fraction of the human orthologs is associated with inherited developmental disorders or causative of cancer. The Oct4 interactome provides a resource for dissecting mechanisms of Oct4 function, enlightening the basis of pluripotency and development, and identifying potential additional reprogramming factors.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105185859</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cell stem cell</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">382-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20362542">20362542</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2860244">2860244</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.stem.2010.03.004">10.1016/j.stem.2010.03.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovery of common Asian copy number variants using integrated high-resolution array CGH and massively parallel DNA sequencing.</h4>
<p class="authors"><span class="author">Park H</span>, <span class="author">Kim JI</span>, <span class="author">Ju YS</span>, <span class="author">Gokcumen O</span>, <span class="author">Mills RE</span>, <span class="author">Kim S</span>, <span class="author">Lee S</span>, <span class="author">Suh D</span>, <span class="author">Hong D</span>, <span class="author">Kang HP</span>, <span class="author">Yoo YJ</span>, <span class="author">Shin JY</span>, <span class="author">Kim HJ</span>, <span class="author">Yavartanoo M</span>, <span class="author">Chang YW</span>, <span class="author">Ha JS</span>, <span class="author">Chong W</span>, <span class="author">Hwang GR</span>, <span class="author">Darvishi K</span>, <span class="author">Kim H</span>, <span class="author">Yang SJ</span>, <span class="author">Yang KS</span>, <span class="author">Kim H</span>, <span class="author">Hurles ME</span>, <span class="author">Scherer SW</span>, <span class="author">Carter NP</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Lee C</span> and <span class="author">Seo JS</span></p>
<p class="affiliation">Genomic Medicine Institute, Medical Research Center, Seoul National University, Seoul, Korea.</p><p class="abstract">Copy number variants (CNVs) account for the majority of human genomic diversity in terms of base coverage. Here, we have developed and applied a new method to combine high-resolution array comparative genomic hybridization (CGH) data with whole-genome DNA sequencing data to obtain a comprehensive catalog of common CNVs in Asian individuals. The genomes of 30 individuals from three Asian populations (Korean, Chinese and Japanese) were interrogated with an ultra-high-resolution array CGH platform containing 24 million probes. Whole-genome sequencing data from a reference genome (NA10851, with 28.3x coverage) and two Asian genomes (AK1, with 27.8x coverage and AK2, with 32.0x coverage) were used to transform the relative copy number information obtained from array CGH experiments into absolute copy number values. We discovered 5,177 CNVs, of which 3,547 were putative Asian-specific CNVs. These common CNVs in Asian populations will be a useful resource for subsequent genetic studies in these populations, and the new method of calling absolute CNVs will be essential for applying CNV data to personalized medicine.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG004221</span>; Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077009</span>, <span class="grant">077014</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">400-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20364138">20364138</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3329635">3329635</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.555">10.1038/ng.555</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using caching and optimization techniques to improve performance of the Ensembl website.</h4>
<p class="authors"><span class="author">Parker A</span>, <span class="author">Bragin E</span>, <span class="author">Brent S</span>, <span class="author">Pritchard B</span>, <span class="author">Smith JA</span> and <span class="author">Trevanion S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs CB10 1SA, UK. <a href="mailto:%61%70%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#112;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The Ensembl web site has provided access to genomic information for almost 10 years. During this time the amount of data available through Ensembl has grown dramatically. At the same time, the World Wide Web itself has become a dramatically more important component of the scientific workflow and the way that scientists share and access data and scientific information. Since 2000, the Ensembl web interface has had three major updates and numerous smaller updates. These have largely been in response to expanding data types and valuable representations of existing data types. In 2007 it was realised that a radical new approach would be required in order to serve the project&#39;s future requirements, and development therefore focused on identifying suitable web technologies for implementation in the 2008 site redesign.</p><p class="abstract"><strong>Results</strong>: By comparing the Ensembl website to well-known &quot;Web 2.0&quot; sites, we were able to identify two main areas in which cutting-edge technologies could be advantageously deployed: server efficiency and interface latency. We then evaluated the performance of the existing site using browser-based tools and Apache benchmarking, and selected appropriate technologies to overcome any issues found. Solutions included optimization of the Apache web server, introduction of caching technologies and widespread implementation of AJAX code. These improvements were successfully deployed on the Ensembl website in late 2008 and early 2009.</p><p class="abstract"><strong>Conclusions</strong>: Web 2.0 technologies provide a flexible and efficient way to access the terabytes of data now available from Ensembl, enhancing the user experience through improved website responsiveness and a rich, interactive interface.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">239</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20459812">20459812</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3098060">3098060</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-11-239">10.1186/1471-2105-11-239</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic information infrastructure after the deluge.</h4>
<p class="authors"><span class="author">Parkhill J</span>, <span class="author">Birney E</span> and <span class="author">Kersey P</span></p><p class="abstract">Maintaining up-to-date annotation on reference genomes is becoming more important, not less, as the ability to rapidly and cheaply resequence genomes expands.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">7</span>;<span class="pagerange">402</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20670392" rel="external">20670392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2926780" rel="external">2926780</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-7-402" rel="external">10.1186/gb-2010-11-7-402</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association meta-analysis of cortical bone mineral density unravels allelic heterogeneity at the RANKL locus and potential pleiotropic effects on bone.</h4>
<p class="authors"><span class="author">Paternoster L</span>, <span class="author">Lorentzon M</span>, <span class="author">Vandenput L</span>, <span class="author">Karlsson MK</span>, <span class="author">Ljunggren O</span>, <span class="author">Kindmark A</span>, <span class="author">Mellstrom D</span>, <span class="author">Kemp JP</span>, <span class="author">Jarett CE</span>, <span class="author">Holly JM</span>, <span class="author">Sayers A</span>, <span class="author">St Pourcain B</span>, <span class="author">Timpson NJ</span>, <span class="author">Deloukas P</span>, <span class="author">Davey Smith G</span>, <span class="author">Ring SM</span>, <span class="author">Evans DM</span>, <span class="author">Tobias JH</span> and <span class="author">Ohlsson C</span></p>
<p class="affiliation">University of Bristol, Bristol, UK. <a href="mailto:%6C.%70%61%74%65%72%6E%6F%73%74%65%72%40%62%72%69%73%74%6F%6C.%61%63.%75%6B">&#108;&#46;&#112;&#97;&#116;&#101;&#114;&#110;&#111;&#115;&#116;&#101;&#114;&#64;&#98;&#114;&#105;&#115;&#116;&#111;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Previous genome-wide association (GWA) studies have identified SNPs associated with areal bone mineral density (aBMD). However, this measure is influenced by several different skeletal parameters, such as periosteal expansion, cortical bone mineral density (BMD(C)) cortical thickness, trabecular number, and trabecular thickness, which may be under distinct biological and genetic control. We have carried out a GWA and replication study of BMD(C), as measured by peripheral quantitative computed tomography (pQCT), a more homogenous and valid measure of actual volumetric bone density. After initial GWA meta-analysis of two cohorts (ALSPAC n&#x200A;=&#x200A;999, aged &sim;15 years and GOOD n&#x200A;=&#x200A;935, aged &sim;19 years), we attempted to replicate the BMD(C) associations that had p&lt;1&times;10(-5) in an independent sample of ALSPAC children (n&#x200A;=&#x200A;2803) and in a cohort of elderly men (MrOS Sweden, n&#x200A;=&#x200A;1052). The rs1021188 SNP (near RANKL) was associated with BMD(C) in all cohorts (overall p&#x200A;=&#x200A;2&times;10(-14), n&#x200A;=&#x200A;5739). Each minor allele was associated with a decrease in BMD(C) of &sim;0.14SD. There was also evidence for an interaction between this variant and sex (p&#x200A;=&#x200A;0.01), with a stronger effect in males than females (at age 15, males -6.77mg/cm(3) per C allele, p&#x200A;=&#x200A;2&times;10(-6); females -2.79 mg/cm(3) per C allele, p&#x200A;=&#x200A;0.004). Furthermore, in a preliminary analysis, the rs1021188 minor C allele was associated with higher circulating levels of sRANKL (p&lt;0.005). We show this variant to be independent from the previously aBMD associated SNP (rs9594738) and possibly from a third variant in the same RANKL region, which demonstrates important allelic heterogeneity at this locus. Associations with skeletal parameters reflecting bone dimensions were either not found or were much less pronounced. This finding implicates RANKL as a locus containing variation associated with volumetric bone density and provides further insight into the mechanism by which the RANK/RANKL/OPG pathway may be involved in skeletal development.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">74882</span>, <span class="grant">G0800582</span>; Wellcome Trust: <span class="grant">076467</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">e1001217</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21124946" rel="external">21124946</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987837" rel="external">2987837</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001217" rel="external">10.1371/journal.pgen.1001217</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study identifies a novel major locus for glycemic control in type 1 diabetes, as measured by both A1C and glucose.</h4>
<p class="authors"><span class="author">Paterson AD</span>, <span class="author">Waggott D</span>, <span class="author">Boright AP</span>, <span class="author">Hosseini SM</span>, <span class="author">Shen E</span>, <span class="author">Sylvestre MP</span>, <span class="author">Wong I</span>, <span class="author">Bharaj B</span>, <span class="author">Cleary PA</span>, <span class="author">Lachin JM</span>, <span class="author">MAGIC (Meta-Analyses of Glucose and Insulin-related traits Consortium)</span>, <span class="author">Below JE</span>, <span class="author">Nicolae D</span>, <span class="author">Cox NJ</span>, <span class="author">Canty AJ</span>, <span class="author">Sun L</span>, <span class="author">Bull SB</span> and <span class="author">Diabetes Control and Complications Trial/Epidemiology of Diabetes Interventions and Complications Research Group</span></p>
<p class="affiliation">Program in Genetics and Genome Biology, Hospital for Sick Children, Toronto, Canada. <a href="mailto:%61%6E%64%72%65%77.%70%61%74%65%72%73%6F%6E%40%75%74%6F%72%6F%6E%74%6F.%63%61">&#97;&#110;&#100;&#114;&#101;&#119;&#46;&#112;&#97;&#116;&#101;&#114;&#115;&#111;&#110;&#64;&#117;&#116;&#111;&#114;&#111;&#110;&#116;&#111;&#46;&#99;&#97;</a></p><p class="abstract"><strong>Objective</strong>: Glycemia is a major risk factor for the development of long-term complications in type 1 diabetes; however, no specific genetic loci have been identified for glycemic control in individuals with type 1 diabetes. To identify such loci in type 1 diabetes, we analyzed longitudinal repeated measures of A1C from the Diabetes Control and Complications Trial.</p><p class="abstract">We performed a genome-wide association study using the mean of quarterly A1C values measured over 6.5 years, separately in the conventional (n = 667) and intensive (n = 637) treatment groups of the DCCT. At loci of interest, linear mixed models were used to take advantage of all the repeated measures. We then assessed the association of these loci with capillary glucose and repeated measures of multiple complications of diabetes.</p><p class="abstract"><strong>Results</strong>: We identified a major locus for A1C levels in the conventional treatment group near SORCS1 (10q25.1, P = 7 x 10(-10)), which was also associated with mean glucose (P = 2 x 10(-5)). This was confirmed using A1C in the intensive treatment group (P = 0.01). Other loci achieved evidence close to genome-wide significance: 14q32.13 (GSC) and 9p22 (BNC2) in the combined treatment groups and 15q21.3 (WDR72) in the intensive group. Further, these loci gave evidence for association with diabetic complications, specifically SORCS1 with hypoglycemia and BNC2 with renal and retinal complications. We replicated the SORCS1 association in Genetics of Diabetes in Kidneys (GoKinD) study control subjects (P = 0.01) and the BNC2 association with A1C in nondiabetic individuals.</p><p class="abstract"><strong>Conclusions</strong>: A major locus for A1C and glucose in individuals with diabetes is near SORCS1. This may influence the design and analysis of genetic studies attempting to identify risk factors for long-term diabetic complications.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; NIDDK NIH HHS: <span class="grant">N01-DK-6-2204</span>, <span class="grant">P60-DK20595</span>, <span class="grant">R01-DK-077510</span>, <span class="grant">R01-DK077489</span>; NIGMS NIH HHS: <span class="grant">T32 GM007197</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2010</span>;<span class="volume">59</span>;<span class="number">2</span>;<span class="pagerange">539-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19875614">19875614</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2809960">2809960</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb09-0653">10.2337/db09-0653</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antagonistic coevolution accelerates molecular evolution.</h4>
<p class="authors"><span class="author">Paterson S</span>, <span class="author">Vogwill T</span>, <span class="author">Buckling A</span>, <span class="author">Benmayor R</span>, <span class="author">Spiers AJ</span>, <span class="author">Thomson NR</span>, <span class="author">Quail M</span>, <span class="author">Smith F</span>, <span class="author">Walker D</span>, <span class="author">Libberton B</span>, <span class="author">Fenton A</span>, <span class="author">Hall N</span> and <span class="author">Brockhurst MA</span></p>
<p class="affiliation">School of Biological Sciences, Biosciences Building, University of Liverpool, Crown Street, Liverpool L69 7ZB, UK.</p><p class="abstract">The Red Queen hypothesis proposes that coevolution of interacting species (such as hosts and parasites) should drive molecular evolution through continual natural selection for adaptation and counter-adaptation. Although the divergence observed at some host-resistance and parasite-infectivity genes is consistent with this, the long time periods typically required to study coevolution have so far prevented any direct empirical test. Here we show, using experimental populations of the bacterium Pseudomonas fluorescens SBW25 and its viral parasite, phage Phi2 (refs 10, 11), that the rate of molecular evolution in the phage was far higher when both bacterium and phage coevolved with each other than when phage evolved against a constant host genotype. Coevolution also resulted in far greater genetic divergence between replicate populations, which was correlated with the range of hosts that coevolved phage were able to infect. Consistent with this, the most rapidly evolving phage genes under coevolution were those involved in host infection. These results demonstrate, at both the genomic and phenotypic level, that antagonistic coevolution is a cause of rapid and divergent evolution, and is likely to be a major driver of evolutionary change within species.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7286</span>;<span class="pagerange">275-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20182425">20182425</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3717453">3717453</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08798">10.1038/nature08798</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Twenty-eight divergent polysaccharide loci specifying within- and amongst-strain capsule diversity in three strains of Bacteroides fragilis.</h4>
<p class="authors"><span class="author">Patrick S</span>, <span class="author">Blakely GW</span>, <span class="author">Houston S</span>, <span class="author">Moore J</span>, <span class="author">Abratt VR</span>, <span class="author">Bertalan M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Quail MA</span>, <span class="author">Corton N</span>, <span class="author">Corton C</span>, <span class="author">Bignell A</span>, <span class="author">Barron A</span>, <span class="author">Clark L</span>, <span class="author">Bentley SD</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Centre for Infection and Immunity, School of Medicine, Dentistry and Biomedical Sciences, Queen&#39;s University Belfast, Medical Biology Centre, 97 Lisburn Road, Belfast BT9 7BL, UK. <a href="mailto:%73.%70%61%74%72%69%63%6B%40%71%75%62.%61%63.%75%6B">&#115;&#46;&#112;&#97;&#116;&#114;&#105;&#99;&#107;&#64;&#113;&#117;&#98;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Comparison of the complete genome sequence of Bacteroides fragilis 638R, originally isolated in the USA, was made with two previously sequenced strains isolated in the UK (NCTC 9343) and Japan (YCH46). The presence of 10 loci containing genes associated with polysaccharide (PS) biosynthesis, each including a putative Wzx flippase and Wzy polymerase, was confirmed in all three strains, despite a lack of cross-reactivity between NCTC 9343 and 638R surface PS-specific antibodies by immunolabelling and microscopy. Genomic comparisons revealed an exceptional level of PS biosynthesis locus diversity. Of the 10 divergent PS-associated loci apparent in each strain, none is similar between NCTC 9343 and 638R. YCH46 shares one locus with NCTC 9343, confirmed by mAb labelling, and a second different locus with 638R, making a total of 28 divergent PS biosynthesis loci amongst the three strains. The lack of expression of the phase-variable large capsule (LC) in strain 638R, observed in NCTC 9343, is likely to be due to a point mutation that generates a stop codon within a putative initiating glycosyltransferase, necessary for the expression of the LC in NCTC 9343. Other major sequence differences were observed to arise from different numbers and variety of inserted extra-chromosomal elements, in particular prophages. Extensive horizontal gene transfer has occurred within these strains, despite the presence of a significant number of divergent DNA restriction and modification systems that act to prevent acquisition of foreign DNA. The level of amongst-strain diversity in PS biosynthesis loci is unprecedented.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">061696</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">156</span>;<span class="number">Pt 11</span>;<span class="pagerange">3255-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20829291">20829291</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3090145">3090145</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.042978-0">10.1099/mic.0.042978-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic evidence that raised sex hormone binding globulin (SHBG) levels reduce the risk of type 2 diabetes.</h4>
<p class="authors"><span class="author">Perry JR</span>, <span class="author">Weedon MN</span>, <span class="author">Langenberg C</span>, <span class="author">Jackson AU</span>, <span class="author">Lyssenko V</span>, <span class="author">Spars&oslash; T</span>, <span class="author">Thorleifsson G</span>, <span class="author">Grallert H</span>, <span class="author">Ferrucci L</span>, <span class="author">Maggio M</span>, <span class="author">Paolisso G</span>, <span class="author">Walker M</span>, <span class="author">Palmer CN</span>, <span class="author">Payne F</span>, <span class="author">Young E</span>, <span class="author">Herder C</span>, <span class="author">Narisu N</span>, <span class="author">Morken MA</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Owen KR</span>, <span class="author">Shields B</span>, <span class="author">Knight B</span>, <span class="author">Bennett A</span>, <span class="author">Groves CJ</span>, <span class="author">Ruokonen A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Pearson E</span>, <span class="author">Pascoe L</span>, <span class="author">Ferrannini E</span>, <span class="author">Bornstein SR</span>, <span class="author">Stringham HM</span>, <span class="author">Scott LJ</span>, <span class="author">Kuusisto J</span>, <span class="author">Nilsson P</span>, <span class="author">Neptin M</span>, <span class="author">Gjesing AP</span>, <span class="author">Pisinger C</span>, <span class="author">Lauritzen T</span>, <span class="author">Sandbaek A</span>, <span class="author">Sampson M</span>, <span class="author">MAGIC</span>, <span class="author">Zeggini E</span>, <span class="author">Lindgren CM</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Hansen T</span>, <span class="author">Schwarz P</span>, <span class="author">Illig T</span>, <span class="author">Laakso M</span>, <span class="author">Stefansson K</span>, <span class="author">Morris AD</span>, <span class="author">Groop L</span>, <span class="author">Pedersen O</span>, <span class="author">Boehnke M</span>, <span class="author">Barroso I</span>, <span class="author">Wareham NJ</span>, <span class="author">Hattersley AT</span>, <span class="author">McCarthy MI</span> and <span class="author">Frayling TM</span></p>
<p class="affiliation">Genetics of Complex Traits, Peninsula College of Medicine and Dentistry, University of Exeter, Magdalen Road, Exeter, UK.</p><p class="abstract">Epidemiological studies consistently show that circulating sex hormone binding globulin (SHBG) levels are lower in type 2 diabetes patients than non-diabetic individuals, but the causal nature of this association is controversial. Genetic studies can help dissect causal directions of epidemiological associations because genotypes are much less likely to be confounded, biased or influenced by disease processes. Using this Mendelian randomization principle, we selected a common single nucleotide polymorphism (SNP) near the SHBG gene, rs1799941, that is strongly associated with SHBG levels. We used data from this SNP, or closely correlated SNPs, in 27 657 type 2 diabetes patients and 58 481 controls from 15 studies. We then used data from additional studies to estimate the difference in SHBG levels between type 2 diabetes patients and controls. The SHBG SNP rs1799941 was associated with type 2 diabetes [odds ratio (OR) 0.94, 95% CI: 0.91, 0.97; P = 2 x 10(-5)], with the SHBG raising allele associated with reduced risk of type 2 diabetes. This effect was very similar to that expected (OR 0.92, 95% CI: 0.88, 0.96), given the SHBG-SNP versus SHBG levels association (SHBG levels are 0.2 standard deviations higher per copy of the A allele) and the SHBG levels versus type 2 diabetes association (SHBG levels are 0.23 standard deviations lower in type 2 diabetic patients compared to controls). Results were very similar in men and women. There was no evidence that this variant is associated with diabetes-related intermediate traits, including several measures of insulin secretion and resistance. Our results, together with those from another recent genetic study, strengthen evidence that SHBG and sex hormones are involved in the aetiology of type 2 diabetes.</p>
<p class="grants">Funded by: Department of Health: <span class="grant">DHCS/07/07/008</span>; Medical Research Council: <span class="grant">G0000649</span>, <span class="grant">G016121</span>, <span class="grant">G0601261</span>, <span class="grant">MC_U106179471</span>; NHGRI NIH HHS: <span class="grant">1 Z01 HG000024</span>; NIA NIH HHS: <span class="grant">R01 AG24233-0</span>; NIDA NIH HHS: <span class="grant">U54 DA021519</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK069922</span>, <span class="grant">DK072193</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">083270/Z/07/Z</span>, <span class="grant">GR072960</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">3</span>;<span class="pagerange">535-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19933169">19933169</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2798726">2798726</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddp522">10.1093/hmg/ddp522</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome annotation: man versus machine.</h4>
<p class="authors"><span class="author">Petty NK</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">762</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20948549" rel="external">20948549</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2462" rel="external">10.1038/nrmicro2462</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Citrobacter rodentium genome sequence reveals convergent evolution with human pathogenic Escherichia coli.</h4>
<p class="authors"><span class="author">Petty NK</span>, <span class="author">Bulgin R</span>, <span class="author">Crepin VF</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Schroeder GN</span>, <span class="author">Quail MA</span>, <span class="author">Lennard N</span>, <span class="author">Corton C</span>, <span class="author">Barron A</span>, <span class="author">Clark L</span>, <span class="author">Toribio AL</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span>, <span class="author">Frankel G</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">Citrobacter rodentium (formally Citrobacter freundii biotype 4280) is a highly infectious pathogen that causes colitis and transmissible colonic hyperplasia in mice. In common with enteropathogenic and enterohemorrhagic Escherichia coli (EPEC and EHEC, respectively), C. rodentium exploits a type III secretion system (T3SS) to induce attaching and effacing (A/E) lesions that are essential for virulence. Here, we report the fully annotated genome sequence of the 5.3-Mb chromosome and four plasmids harbored by C. rodentium strain ICC168. The genome sequence revealed key information about the phylogeny of C. rodentium and identified 1,585 C. rodentium-specific (without orthologues in EPEC or EHEC) coding sequences, 10 prophage-like regions, and 17 genomic islands, including the locus for enterocyte effacement (LEE) region, which encodes a T3SS and effector proteins. Among the 29 T3SS effectors found in C. rodentium are all 22 of the core effectors of EPEC strain E2348/69. In addition, we identified a novel C. rodentium effector, named EspS. C. rodentium harbors two type VI secretion systems (T6SS) (CTS1 and CTS2), while EHEC contains only one T6SS (EHS). Our analysis suggests that C. rodentium and EPEC/EHEC have converged on a common host infection strategy through access to a common pool of mobile DNA and that C. rodentium has lost gene functions associated with a previous pathogenic niche.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700823</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">2</span>;<span class="pagerange">525-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19897651">19897651</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2805327">2805327</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01144-09">10.1128/JB.01144-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A conserved acetyl esterase domain targets diverse bacteriophages to the Vi capsular receptor of Salmonella enterica serovar Typhi.</h4>
<p class="authors"><span class="author">Pickard D</span>, <span class="author">Toribio AL</span>, <span class="author">Petty NK</span>, <span class="author">van Tonder A</span>, <span class="author">Yu L</span>, <span class="author">Goulding D</span>, <span class="author">Barrell B</span>, <span class="author">Rance R</span>, <span class="author">Harris D</span>, <span class="author">Wetter M</span>, <span class="author">Wain J</span>, <span class="author">Choudhary J</span>, <span class="author">Thomson N</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Sulston Building, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%64%6A%70%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#106;&#112;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A number of bacteriophages have been identified that target the Vi capsular antigen of Salmonella enterica serovar Typhi. Here we show that these Vi phages represent a remarkably diverse set of phages belonging to three phage families, including Podoviridae and Myoviridae. Genome analysis facilitated the further classification of these phages and highlighted aspects of their independent evolution. Significantly, a conserved protein domain carrying an acetyl esterase was found to be associated with at least one tail fiber gene for all Vi phages, and the presence of this domain was confirmed in representative phage particles by mass spectrometric analysis. Thus, we provide a simple explanation and paradigm of how a diverse group of phages target a single key virulence antigen associated with this important human-restricted pathogen.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">21</span>;<span class="pagerange">5746-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20817773">20817773</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2953684">2953684</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00659-10">10.1128/JB.00659-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Metamotifs--a generative model for building families of nucleotide position weight matrices.</h4>
<p class="authors"><span class="author">Piipari M</span>, <span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK. <a href="mailto:%6D%61%74%69%61%73.%70%69%69%70%61%72%69%40%67%6D%61%69%6C.%63%6F%6D">&#109;&#97;&#116;&#105;&#97;&#115;&#46;&#112;&#105;&#105;&#112;&#97;&#114;&#105;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: Development of high-throughput methods for measuring DNA interactions of transcription factors together with computational advances in short motif inference algorithms is expanding our understanding of transcription factor binding site motifs. The consequential growth of sequence motif data sets makes it important to systematically group and categorise regulatory motifs. It has been shown that there are familial tendencies in DNA sequence motifs that are predictive of the family of factors that binds them. Further development of methods that detect and describe familial motif trends has the potential to help in measuring the similarity of novel computational motif predictions to previously known data and sensitively detecting regulatory motifs similar to previously known ones from novel sequence.</p><p class="abstract"><strong>Results</strong>: We propose a probabilistic model for position weight matrix (PWM) sequence motif families. The model, which we call the &#39;metamotif&#39; describes recurring familial patterns in a set of motifs. The metamotif framework models variation within a family of sequence motifs. It allows for simultaneous estimation of a series of independent metamotifs from input position weight matrix (PWM) motif data and does not assume that all input motif columns contribute to a familial pattern. We describe an algorithm for inferring metamotifs from weight matrix data. We then demonstrate the use of the model in two practical tasks: in the Bayesian NestedMICA model inference algorithm as a PWM prior to enhance motif inference sensitivity, and in a motif classification task where motifs are labelled according to their interacting DNA binding domain.</p><p class="abstract"><strong>Conclusions</strong>: We show that metamotifs can be used as PWM priors in the NestedMICA motif inference algorithm to dramatically increase the sensitivity to infer motifs. Metamotifs were also successfully applied to a motif classification problem where sequence motif features were used to predict the family of protein DNA binding domains that would interact with it. The metamotif based classifier is shown to compare favourably to previous related methods. The metamotif has great potential for further use in machine learning tasks related to especially de novo computational sequence motif inference. The metamotif methods presented have been incorporated into the NestedMICA suite.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span>, <span class="grant">077198/Z/05/Z</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">348</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20579334">20579334</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2906491">2906491</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-11-348">10.1186/1471-2105-11-348</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">iMotifs: an integrated sequence motif visualization and analysis environment.</h4>
<p class="authors"><span class="author">Piipari M</span>, <span class="author">Down TA</span>, <span class="author">Saini H</span>, <span class="author">Enright A</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK. <a href="mailto:%6D%61%74%69%61%73.%70%69%69%70%61%72%69%40%67%6D%61%69%6C.%63%6F%6D">&#109;&#97;&#116;&#105;&#97;&#115;&#46;&#112;&#105;&#105;&#112;&#97;&#114;&#105;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Motivation</strong>: Short sequence motifs are an important class of models in molecular biology, used most commonly for describing transcription factor binding site specificity patterns. High-throughput methods have been recently developed for detecting regulatory factor binding sites in vivo and in vitro and consequently high-quality binding site motif data are becoming available for increasing number of organisms and regulatory factors. Development of intuitive tools for the study of sequence motifs is therefore important. iMotifs is a graphical motif analysis environment that allows visualization of annotated sequence motifs and scored motif hits in sequences. It also offers motif inference with the sensitive NestedMICA algorithm, as well as overrepresentation and pairwise motif matching capabilities. All of the analysis functionality is provided without the need to convert between file formats or learn different command line interfaces. The application includes a bundled and graphically integrated version of the NestedMICA motif inference suite that has no outside dependencies. Problems associated with local deployment of software are therefore avoided.</p><p class="abstract"><strong>Availability</strong>: iMotifs is licensed with the GNU Lesser General Public License v2.0 (LGPL 2.0). The software and its source is available at http://wiki.github.com/mz2/imotifs and can be run on Mac OS X Leopard (Intel/PowerPC). We also provide a cross-platform (Linux, OS X, Windows) LGPL 2.0 licensed library libxms for the Perl, Ruby, R and Objective-C programming languages for input and output of XMS formatted annotated sequence motif set files.</p><p class="abstract"><strong>Contact</strong>: <a href="mailto:%6D%61%74%69%61%73.%70%69%69%70%61%72%69%40%67%6D%61%69%6C.%63%6F%6D">&#109;&#97;&#116;&#105;&#97;&#115;&#46;&#112;&#105;&#105;&#112;&#97;&#114;&#105;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a>; <a href="mailto:%69%6D%6F%74%69%66%73%40%67%6F%6F%67%6C%65%67%72%6F%75%70%73.%63%6F%6D">&#105;&#109;&#111;&#116;&#105;&#102;&#115;&#64;&#103;&#111;&#111;&#103;&#108;&#101;&#103;&#114;&#111;&#117;&#112;&#115;&#46;&#99;&#111;&#109;</a>.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span>, <span class="grant">077198/Z/05/Z</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">6</span>;<span class="pagerange">843-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20106815">20106815</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2832821">2832821</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq026">10.1093/bioinformatics/btq026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study reveals multiple loci associated with primary tooth development during infancy.</h4>
<p class="authors"><span class="author">Pillas D</span>, <span class="author">Hoggart CJ</span>, <span class="author">Evans DM</span>, <span class="author">O&#39;Reilly PF</span>, <span class="author">Sipil&auml; K</span>, <span class="author">L&auml;hdesm&auml;ki R</span>, <span class="author">Millwood IY</span>, <span class="author">Kaakinen M</span>, <span class="author">Netuveli G</span>, <span class="author">Blane D</span>, <span class="author">Charoen P</span>, <span class="author">Sovio U</span>, <span class="author">Pouta A</span>, <span class="author">Freimer N</span>, <span class="author">Hartikainen AL</span>, <span class="author">Laitinen J</span>, <span class="author">Vaara S</span>, <span class="author">Glaser B</span>, <span class="author">Crawford P</span>, <span class="author">Timpson NJ</span>, <span class="author">Ring SM</span>, <span class="author">Deng G</span>, <span class="author">Zhang W</span>, <span class="author">McCarthy MI</span>, <span class="author">Deloukas P</span>, <span class="author">Peltonen L</span>, <span class="author">Elliott P</span>, <span class="author">Coin LJ</span>, <span class="author">Smith GD</span> and <span class="author">Jarvelin MR</span></p>
<p class="affiliation">Department of Epidemiology and Public Health, Imperial College London, London, United Kingdom.</p><p class="abstract">Tooth development is a highly heritable process which relates to other growth and developmental processes, and which interacts with the development of the entire craniofacial complex. Abnormalities of tooth development are common, with tooth agenesis being the most common developmental anomaly in humans. We performed a genome-wide association study of time to first tooth eruption and number of teeth at one year in 4,564 individuals from the 1966 Northern Finland Birth Cohort (NFBC1966) and 1,518 individuals from the Avon Longitudinal Study of Parents and Children (ALSPAC). We identified 5 loci at P&lt;5x10(-8), and 5 with suggestive association (P&lt;5x10(-6)). The loci included several genes with links to tooth and other organ development (KCNJ2, EDA, HOXB2, RAD51L1, IGF2BP1, HMGA2, MSRB3). Genes at four of the identified loci are implicated in the development of cancer. A variant within the HOXB gene cluster associated with occlusion defects requiring orthodontic treatment by age 31 years.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600705</span>, <span class="grant">G0800582</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">e1000856</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20195514" rel="external">20195514</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2829062" rel="external">2829062</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000856" rel="external">10.1371/journal.pgen.1000856</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comprehensive catalogue of somatic mutations from a human cancer genome.</h4>
<p class="authors"><span class="author">Pleasance ED</span>, <span class="author">Cheetham RK</span>, <span class="author">Stephens PJ</span>, <span class="author">McBride DJ</span>, <span class="author">Humphray SJ</span>, <span class="author">Greenman CD</span>, <span class="author">Varela I</span>, <span class="author">Lin ML</span>, <span class="author">Ord&oacute;&ntilde;ez GR</span>, <span class="author">Bignell GR</span>, <span class="author">Ye K</span>, <span class="author">Alipaz J</span>, <span class="author">Bauer MJ</span>, <span class="author">Beare D</span>, <span class="author">Butler A</span>, <span class="author">Carter RJ</span>, <span class="author">Chen L</span>, <span class="author">Cox AJ</span>, <span class="author">Edkins S</span>, <span class="author">Kokko-Gonzales PI</span>, <span class="author">Gormley NA</span>, <span class="author">Grocock RJ</span>, <span class="author">Haudenschild CD</span>, <span class="author">Hims MM</span>, <span class="author">James T</span>, <span class="author">Jia M</span>, <span class="author">Kingsbury Z</span>, <span class="author">Leroy C</span>, <span class="author">Marshall J</span>, <span class="author">Menzies A</span>, <span class="author">Mudie LJ</span>, <span class="author">Ning Z</span>, <span class="author">Royce T</span>, <span class="author">Schulz-Trieglaff OB</span>, <span class="author">Spiridou A</span>, <span class="author">Stebbings LA</span>, <span class="author">Szajkowski L</span>, <span class="author">Teague J</span>, <span class="author">Williamson D</span>, <span class="author">Chin L</span>, <span class="author">Ross MT</span>, <span class="author">Campbell PJ</span>, <span class="author">Bentley DR</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">All cancers carry somatic mutations. A subset of these somatic alterations, termed driver mutations, confer selective growth advantage and are implicated in cancer development, whereas the remainder are passengers. Here we have sequenced the genomes of a malignant melanoma and a lymphoblastoid cell line from the same person, providing the first comprehensive catalogue of somatic mutations from an individual cancer. The catalogue provides remarkable insights into the forces that have shaped this cancer genome. The dominant mutational signature reflects DNA damage due to ultraviolet light exposure, a known risk factor for malignant melanoma, whereas the uneven distribution of mutations across the genome, with a lower prevalence in gene footprints, indicates that DNA repair has been preferentially deployed towards transcribed regions. The results illustrate the power of a cancer genome sequence to reveal traces of the DNA damage, repair, mutation and selection processes that were operative years before the cancer became symptomatic.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">463</span>;<span class="number">7278</span>;<span class="pagerange">191-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20016485">20016485</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3145108">3145108</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08658">10.1038/nature08658</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A small-cell lung cancer genome with complex signatures of tobacco exposure.</h4>
<p class="authors"><span class="author">Pleasance ED</span>, <span class="author">Stephens PJ</span>, <span class="author">O&#39;Meara S</span>, <span class="author">McBride DJ</span>, <span class="author">Meynert A</span>, <span class="author">Jones D</span>, <span class="author">Lin ML</span>, <span class="author">Beare D</span>, <span class="author">Lau KW</span>, <span class="author">Greenman C</span>, <span class="author">Varela I</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Davies HR</span>, <span class="author">Ordo&ntilde;ez GR</span>, <span class="author">Mudie LJ</span>, <span class="author">Latimer C</span>, <span class="author">Edkins S</span>, <span class="author">Stebbings L</span>, <span class="author">Chen L</span>, <span class="author">Jia M</span>, <span class="author">Leroy C</span>, <span class="author">Marshall J</span>, <span class="author">Menzies A</span>, <span class="author">Butler A</span>, <span class="author">Teague JW</span>, <span class="author">Mangion J</span>, <span class="author">Sun YA</span>, <span class="author">McLaughlin SF</span>, <span class="author">Peckham HE</span>, <span class="author">Tsung EF</span>, <span class="author">Costa GL</span>, <span class="author">Lee CC</span>, <span class="author">Minna JD</span>, <span class="author">Gazdar A</span>, <span class="author">Birney E</span>, <span class="author">Rhodes MD</span>, <span class="author">McKernan KJ</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span> and <span class="author">Campbell PJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Cancer is driven by mutation. Worldwide, tobacco smoking is the principal lifestyle exposure that causes cancer, exerting carcinogenicity through &gt;60 chemicals that bind and mutate DNA. Using massively parallel sequencing technology, we sequenced a small-cell lung cancer cell line, NCI-H209, to explore the mutational burden associated with tobacco smoking. A total of 22,910 somatic substitutions were identified, including 134 in coding exons. Multiple mutation signatures testify to the cocktail of carcinogens in tobacco smoke and their proclivities for particular bases and surrounding sequence context. Effects of transcription-coupled repair and a second, more general, expression-linked repair pathway were evident. We identified a tandem duplication that duplicates exons 3-8 of CHD7 in frame, and another two lines carrying PVT1-CHD7 fusion genes, indicating that CHD7 may be recurrently rearranged in this disease. These findings illustrate the potential for next-generation sequencing to provide unprecedented insights into mutational processes, cellular repair pathways and gene networks associated with cancer.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P50CA70907</span>; Wellcome Trust: <span class="grant">077012</span>, <span class="grant">077012/Z/05/Z</span>, <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">463</span>;<span class="number">7278</span>;<span class="pagerange">184-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20016488">20016488</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2880489">2880489</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08629">10.1038/nature08629</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Telomere length in prospective and retrospective cancer case-control studies.</h4>
<p class="authors"><span class="author">Pooley KA</span>, <span class="author">Sandhu MS</span>, <span class="author">Tyrer J</span>, <span class="author">Shah M</span>, <span class="author">Driver KE</span>, <span class="author">Luben RN</span>, <span class="author">Bingham SA</span>, <span class="author">Ponder BA</span>, <span class="author">Pharoah PD</span>, <span class="author">Khaw KT</span>, <span class="author">Easton DF</span> and <span class="author">Dunning AM</span></p>
<p class="affiliation">Cancer Research UK Genetic Epidemiology Unit, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Li Ka Shing Centre, Cambridge, United Kingdom.</p><p class="abstract">Previous studies have reported that shorter mean telomere length in lymphocytes was associated with increased susceptibility to common diseases of aging, and may be predictive of cancer risk. However, most analyses have examined retrospectively collected case-control studies. Mean telomere length was measured using high-throughput quantitative real-time PCR. Blood for DNA extraction was collected after cancer diagnosis in the East Anglian SEARCH Breast (2,243 cases and 2,181 controls) and SEARCH Colorectal (2,249 cases and 2,161 controls) studies. Prospective case-control studies were conducted for breast cancer (199 cases) and colorectal cancer (185 cases), nested within the EPIC-Norfolk cohort. Blood was collected at least 6 months prior to diagnosis, and was matched to DNA from two cancer-free controls per case. In the retrospective SEARCH studies, the age-adjusted odds ratios for shortest (Q4) versus longest (Q1) quartile of mean telomere length was 15.5 [95% confidence intervals (CI), 11.6-20.8; p-het = 5.7 x 10(-75)], with a &quot;per quartile&quot; P-trend = 2.1 x 10(-80) for breast cancer; and 2.14 (95% CI, 1.77-2.59; p-het = 7.3 x 10(-15)), with a per quartile P-trend = 1.8 x 10(-13) for colorectal cancer. In the prospective EPIC study, the comparable odds ratios (Q4 versus Q1) were 1.58 (95% CI, 0.75-3.31; p-het = 0.23) for breast cancer and 1.13 (95% CI, 0.54-2.36; p-het = 0.75) for colorectal cancer risk. Mean telomere length was shorter in retrospectively collected cases than in controls but the equivalent association was markedly weaker in the prospective studies. This suggests that telomere shortening largely occurs after diagnosis, and therefore, might not be of value in cancer prediction.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A10119</span>, <span class="grant">A10123</span>, <span class="grant">A10124</span>, <span class="grant">A9540</span>, <span class="grant">C1287/A9540</span>; Medical Research Council</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2010</span>;<span class="volume">70</span>;<span class="number">8</span>;<span class="pagerange">3170-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20395204" rel="external">20395204</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2855947" rel="external">2855947</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-09-4595" rel="external">10.1158/0008-5472.CAN-09-4595</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PARK2 deletions occur frequently in sporadic colorectal cancer and accelerate adenoma development in Apc mutant mice.</h4>
<p class="authors"><span class="author">Poulogiannis G</span>, <span class="author">McIntyre RE</span>, <span class="author">Dimitriadi M</span>, <span class="author">Apps JR</span>, <span class="author">Wilson CH</span>, <span class="author">Ichimura K</span>, <span class="author">Luo F</span>, <span class="author">Cantley LC</span>, <span class="author">Wyllie AH</span>, <span class="author">Adams DJ</span> and <span class="author">Arends MJ</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Cambridge CB2 0QQ, United Kingdom.</p><p class="abstract">In 100 primary colorectal carcinomas, we demonstrate by array comparative genomic hybridization (aCGH) that 33% show DNA copy number (DCN) loss involving PARK2, the gene encoding PARKIN, the E3 ubiquitin ligase whose deficiency is responsible for a form of autosomal recessive juvenile parkinsonism. PARK2 is located on chromosome 6 (at 6q25-27), a chromosome with one of the lowest overall frequencies of DNA copy number alterations recorded in colorectal cancers. The PARK2 deletions are mostly focal (31% approximately 0.5 Mb on average), heterozygous, and show maximum incidence in exons 3 and 4. As PARK2 lies within FRA6E, a large common fragile site, it has been argued that the observed DCN losses in PARK2 in cancer may represent merely the result of enforced replication of locally vulnerable DNA. However, we show that deficiency in expression of PARK2 is significantly associated with adenomatous polyposis coli (APC) deficiency in human colorectal cancer. Evidence of some PARK2 mutations and promoter hypermethylation is described. PARK2 overexpression inhibits cell proliferation in vitro. Moreover, interbreeding of Park2 heterozygous knockout mice with Apc(Min) mice resulted in a dramatic acceleration of intestinal adenoma development and increased polyp multiplicity. We conclude that PARK2 is a tumor suppressor gene whose haploinsufficiency cooperates with mutant APC in colorectal carcinogenesis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">12401</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2010</span>;<span class="volume">107</span>;<span class="number">34</span>;<span class="pagerange">15145-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20696900">20696900</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2930574">2930574</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.1009941107">10.1073/pnas.1009941107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Independent and population-specific association of risk variants at the IRGM locus with Crohn&#39;s disease.</h4>
<p class="authors"><span class="author">Prescott NJ</span>, <span class="author">Dominy KM</span>, <span class="author">Kubo M</span>, <span class="author">Lewis CM</span>, <span class="author">Fisher SA</span>, <span class="author">Redon R</span>, <span class="author">Huang N</span>, <span class="author">Stranger BE</span>, <span class="author">Blaszczyk K</span>, <span class="author">Hudspith B</span>, <span class="author">Parkes G</span>, <span class="author">Hosono N</span>, <span class="author">Yamazaki K</span>, <span class="author">Onnie CM</span>, <span class="author">Forbes A</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Nakamura Y</span>, <span class="author">Mansfield JC</span>, <span class="author">Sanderson J</span>, <span class="author">Hurles ME</span>, <span class="author">Roberts RG</span> and <span class="author">Mathew CG</span></p>
<p class="affiliation">Department of Medical and Molecular Genetics, King&#39;s College London School of Medicine, Guy&#39;s Hospital, London SE1 9RT, UK. <a href="mailto:%6E%61%74%61%6C%69%65.%70%72%65%73%63%6F%74%74%40%67%65%6E%65%74%69%63%73.%6B%63%6C.%61%63.%75%6B">&#110;&#97;&#116;&#97;&#108;&#105;&#101;&#46;&#112;&#114;&#101;&#115;&#99;&#111;&#116;&#116;&#64;&#103;&#101;&#110;&#101;&#116;&#105;&#99;&#115;&#46;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">DNA polymorphisms in a region on chromosome 5q33.1 which contains two genes, immunity related GTPase related family, M (IRGM) and zinc finger protein 300 (ZNF300), are associated with Crohn&#39;s disease (CD). The deleted allele of a 20 kb copy number variation (CNV) upstream of IRGM was recently shown to be in strong linkage disequilibrium (LD) with the CD-associated single nucleotide polymorphisms and is itself associated with CD (P &lt; 0.01). The deletion was correlated with increased or reduced expression of IRGM in transformed cells in a cell line-dependent manner, and has been proposed as a likely causal variant. We report here that small insertion/deletion polymorphisms in the promoter and 5&#39; untranslated region of IRGM are, together with the CNV, strongly associated with CD (P = 1.37 x 10(-5) to 1.40 x 10(-9)), and that the CNV and the 5&#39;-untranslated region variant -308(GTTT)(5) contribute independently to CD susceptibility (P = 2.6 x 10(-7) and P = 2 x 10(-5), respectively). We also show that the CD risk haplotype is associated with a significant decrease in IRGM expression (P &lt; 10(-12)) in untransformed lymphocytes from CD patients. Further analysis of these variants in a Japanese CD case-control sample and of IRGM expression in HapMap populations revealed that neither the IRGM insertion/deletion polymorphisms nor the CNV was associated with CD or with altered IRGM expression in the Asian population. This suggests that the involvement of the IRGM risk haplotype in the pathogenesis of CD requires gene-gene or gene-environment interactions which are absent in Asian populations, or that none of the variants analysed are causal, and that the true causal variants arose after the European-Asian split.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">081808/</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">9</span>;<span class="pagerange">1828-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20106866" rel="external">20106866</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2850616" rel="external">2850616</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq041" rel="external">10.1093/hmg/ddq041</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of pneumonia due to Streptococcus equi subsp. zooepidemicus in dogs.</h4>
<p class="authors"><span class="author">Priestnall SL</span>, <span class="author">Erles K</span>, <span class="author">Brooks HW</span>, <span class="author">Cardwell JM</span>, <span class="author">Waller AS</span>, <span class="author">Paillot R</span>, <span class="author">Robinson C</span>, <span class="author">Darby AC</span>, <span class="author">Holden MT</span> and <span class="author">Sch&ouml;niger S</span></p>
<p class="affiliation">Department of Pathology and Infectious Diseases, Royal Veterinary College, Hawkshead Lane, North Mymms, Hatfield, Hertfordshire AL9 7TA, United Kingdom. <a href="mailto:%73%70%72%69%65%73%74%6E%61%6C%6C%40%72%76%63.%61%63.%75%6B">&#115;&#112;&#114;&#105;&#101;&#115;&#116;&#110;&#97;&#108;&#108;&#64;&#114;&#118;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Streptococcus equi subsp. zooepidemicus has been linked to cases of acute fatal pneumonia in dogs in several countries. Outbreaks can occur in kenneled dog populations and result in significant levels of morbidity and mortality. This highly contagious disease is characterized by the sudden onset of clinical signs, including pyrexia, dyspnea, and hemorrhagic nasal discharge. The pathogenesis of S. equi subsp. zooepidemicus infection in dogs is poorly understood. This study systematically characterized the histopathological changes in the lungs of 39 dogs from a large rehoming shelter in London, United Kingdom; the dogs were infected with S. equi subsp. zooepidemicus. An objective scoring system demonstrated that S. equi subsp. zooepidemicus caused pneumonia in 26/39 (66.7%) dogs, and most of these dogs (17/26 [65.4%]) were classified as severe fibrino-suppurative, necrotizing, and hemorrhagic. Three recently described superantigen genes (szeF, szeN, and szeP) were detected by PCR in 17/47 (36.2%) of the S. equi subsp. zooepidemicus isolates; however, there was no association between the presence of these genes and the histopathological score. The lungs of S. equi subsp. zooepidemicus-infected dogs with severe respiratory signs and lung pathology did however have significantly higher mRNA levels of the proinflammatory cytokines tumor necrosis factor alpha (TNF-&alpha;), interleukin 6 (IL-6), and interleukin 8 (IL-8) than in uninfected controls, suggesting a role for an exuberant host immune response in the pathogenesis of this disease.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Clinical and vaccine immunology : CVI</span>&nbsp;<span class="year">2010</span>;<span class="volume">17</span>;<span class="number">11</span>;<span class="pagerange">1790-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20861329" rel="external">20861329</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2976090" rel="external">2976090</a>; DOI: <a href="http://dx.doi.org/10.1128%2FCVI.00188-10" rel="external">10.1128/CVI.00188-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variants at 2q24 are associated with susceptibility to type 2 diabetes.</h4>
<p class="authors"><span class="author">Qi L</span>, <span class="author">Cornelis MC</span>, <span class="author">Kraft P</span>, <span class="author">Stanya KJ</span>, <span class="author">Linda Kao WH</span>, <span class="author">Pankow JS</span>, <span class="author">Dupuis J</span>, <span class="author">Florez JC</span>, <span class="author">Fox CS</span>, <span class="author">Par&eacute; G</span>, <span class="author">Sun Q</span>, <span class="author">Girman CJ</span>, <span class="author">Laurie CC</span>, <span class="author">Mirel DB</span>, <span class="author">Manolio TA</span>, <span class="author">Chasman DI</span>, <span class="author">Boerwinkle E</span>, <span class="author">Ridker PM</span>, <span class="author">Hunter DJ</span>, <span class="author">Meigs JB</span>, <span class="author">Lee CH</span>, <span class="author">Hu FB</span>, <span class="author">van Dam RM</span>, <span class="author">Meta-Analysis of Glucose and Insulin-related traits Consortium (MAGIC)</span> and <span class="author">Diabetes Genetics Replication and Meta-analysis (DIAGRAM) Consortium</span></p>
<p class="affiliation">Department of Nutrition, Harvard School of Public Health, and Brigham and Women&#39;s Hospital, Boston, MA, USA. <a href="mailto:%6E%68%6C%71%69%40%63%68%61%6E%6E%69%6E%67.%68%61%72%76%61%72%64.%65%64%75">&#110;&#104;&#108;&#113;&#105;&#64;&#99;&#104;&#97;&#110;&#110;&#105;&#110;&#103;&#46;&#104;&#97;&#114;&#118;&#97;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">To identify type 2 diabetes (T2D) susceptibility loci, we conducted genome-wide association (GWA) scans in nested case-control samples from two prospective cohort studies, including 2591 patients and 3052 controls of European ancestry. Validation was performed in 11 independent GWA studies of 10,870 cases and 73,735 controls. We identified significantly associated variants near RBMS1 and ITGB6 genes at 2q24, best-represented by SNP rs7593730 (combined OR=0.90, 95% CI=0.86-0.93; P=3.7x10(-8)). The frequency of the risk-lowering allele T is 0.23. Variants in this region were nominally related to lower fasting glucose and HOMA-IR in the MAGIC consortium (P&lt;0.05). These data suggest that the 2q24 locus may influence the T2D risk by affecting glucose metabolism and insulin resistance.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA047988</span>, <span class="grant">CA1367 92</span>, <span class="grant">CA54281</span>, <span class="grant">CA63464</span>, <span class="grant">P01CA 089392</span>, <span class="grant">P01CA055075</span>, <span class="grant">P01CA087969</span>, <span class="grant">Z01CP010200</span>; NCRR NIH HHS: <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">U01HG0 04436</span>, <span class="grant">U01HG004399</span>, <span class="grant">U01HG004402</span>, <span class="grant">U01HG004415</span>, <span class="grant">U01HG004422</span>, <span class="grant">U01HG004423</span>, <span class="grant">U01HG004438</span>, <span class="grant">U01HG004446</span>, <span class="grant">U01HG0047 29</span>, <span class="grant">U01HG004726</span>, <span class="grant">U01HG004728</span>, <span class="grant">U01HG004735</span>, <span class="grant">U01HG004738</span>, <span class="grant">U01HG04424</span>; NHLBI NIH HHS: <span class="grant">HL043851</span>, <span class="grant">HL69757</span>, <span class="grant">N01- HC-55022</span>, <span class="grant">N01-HC- 55018</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N02-HL-6-427</span>, <span class="grant">R01 HL071981</span>, <span class="grant">R01 HL71981</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>; NIAAA NIH HHS: <span class="grant">U10AA008401</span>; NIDA NIH HHS: <span class="grant">R01DA013423</span>; NIDCR NIH HHS: <span class="grant">U01DE018 993</span>, <span class="grant">U01DE018903</span>; NIDDK NIH HHS: <span class="grant">DK46200</span>, <span class="grant">K01- DK067207</span>, <span class="grant">K23 DK65978</span>, <span class="grant">K24 DK080140</span>, <span class="grant">R01DK058845</span>, <span class="grant">R01DK075046</span>, <span class="grant">R01DK078616</span>, <span class="grant">R90DK071507</span>, <span class="grant">T90 DK070078</span>, <span class="grant">T90 DK070078-05</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>, <span class="grant">HHSN268200782096C</span>, <span class="grant">RFAHG006033</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">13</span>;<span class="pagerange">2706-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20418489">20418489</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2883345">2883345</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq156">10.1093/hmg/ddq156</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A human gut microbial gene catalogue established by metagenomic sequencing.</h4>
<p class="authors"><span class="author">Qin J</span>, <span class="author">Li R</span>, <span class="author">Raes J</span>, <span class="author">Arumugam M</span>, <span class="author">Burgdorf KS</span>, <span class="author">Manichanh C</span>, <span class="author">Nielsen T</span>, <span class="author">Pons N</span>, <span class="author">Levenez F</span>, <span class="author">Yamada T</span>, <span class="author">Mende DR</span>, <span class="author">Li J</span>, <span class="author">Xu J</span>, <span class="author">Li S</span>, <span class="author">Li D</span>, <span class="author">Cao J</span>, <span class="author">Wang B</span>, <span class="author">Liang H</span>, <span class="author">Zheng H</span>, <span class="author">Xie Y</span>, <span class="author">Tap J</span>, <span class="author">Lepage P</span>, <span class="author">Bertalan M</span>, <span class="author">Batto JM</span>, <span class="author">Hansen T</span>, <span class="author">Le Paslier D</span>, <span class="author">Linneberg A</span>, <span class="author">Nielsen HB</span>, <span class="author">Pelletier E</span>, <span class="author">Renault P</span>, <span class="author">Sicheritz-Ponten T</span>, <span class="author">Turner K</span>, <span class="author">Zhu H</span>, <span class="author">Yu C</span>, <span class="author">Li S</span>, <span class="author">Jian M</span>, <span class="author">Zhou Y</span>, <span class="author">Li Y</span>, <span class="author">Zhang X</span>, <span class="author">Li S</span>, <span class="author">Qin N</span>, <span class="author">Yang H</span>, <span class="author">Wang J</span>, <span class="author">Brunak S</span>, <span class="author">Dor&eacute; J</span>, <span class="author">Guarner F</span>, <span class="author">Kristiansen K</span>, <span class="author">Pedersen O</span>, <span class="author">Parkhill J</span>, <span class="author">Weissenbach J</span>, <span class="author">MetaHIT Consortium</span>, <span class="author">Bork P</span>, <span class="author">Ehrlich SD</span> and <span class="author">Wang J</span></p>
<p class="affiliation">BGI-Shenzhen, Shenzhen 518083, China.</p><p class="abstract">To understand the impact of gut microbes on human health and well-being it is crucial to assess their genetic potential. Here we describe the Illumina-based metagenomic sequencing, assembly and characterization of 3.3 million non-redundant microbial genes, derived from 576.7 gigabases of sequence, from faecal samples of 124 European individuals. The gene set, approximately 150 times larger than the human gene complement, contains an overwhelming majority of the prevalent (more frequent) microbial genes of the cohort and probably includes a large proportion of the prevalent human intestinal microbial genes. The genes are largely shared among individuals of the cohort. Over 99% of the genes are bacterial, indicating that the entire cohort harbours between 1,000 and 1,150 prevalent bacterial species and each individual at least 160 such species, which are also largely shared. We define and describe the minimal gut metagenome and the minimal gut bacterial genome in terms of functions present in all individuals and most bacteria, respectively.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7285</span>;<span class="pagerange">59-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20203603">20203603</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08821">10.1038/nature08821</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.</h4>
<p class="authors"><span class="author">Quinlan AR</span>, <span class="author">Clark RA</span>, <span class="author">Sokolova S</span>, <span class="author">Leibowitz ML</span>, <span class="author">Zhang Y</span>, <span class="author">Hurles ME</span>, <span class="author">Mell JC</span> and <span class="author">Hall IM</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Genetics, University of Virginia School of Medicine, Charlottesville, Virginia 22908, USA.</p><p class="abstract">Structural variation (SV) is a rich source of genetic diversity in mammals, but due to the challenges associated with mapping SV in complex genomes, basic questions regarding their genomic distribution and mechanistic origins remain unanswered. We have developed an algorithm (HYDRA) to localize SV breakpoints by paired-end mapping, and a general approach for the genome-wide assembly and interpretation of breakpoint sequences. We applied these methods to two inbred mouse strains: C57BL/6J and DBA/2J. We demonstrate that HYDRA accurately maps diverse classes of SV, including those involving repetitive elements such as transposons and segmental duplications; however, our analysis of the C57BL/6J reference strain shows that incomplete reference genome assemblies are a major source of noise. We report 7196 SVs between the two strains, more than two-thirds of which are due to transposon insertions. Of the remainder, 59% are deletions (relative to the reference), 26% are insertions of unlinked DNA, 9% are tandem duplications, and 6% are inversions. To investigate the origins of SV, we characterized 3316 breakpoint sequences at single-nucleotide resolution. We find that approximately 16% of non-transposon SVs have complex breakpoint patterns consistent with template switching during DNA replication or repair, and that this process appears to preferentially generate certain classes of complex variants. Moreover, we find that SVs are significantly enriched in regions of segmental duplication, but that this effect is largely independent of DNA sequence homology and thus cannot be explained by non-allelic homologous recombination (NAHR) alone. This result suggests that the genetic instability of such regions is often the cause rather than the consequence of duplicated genomic architecture.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1F32HG005197-01</span>; NIH HHS: <span class="grant">DP2OD006493-01</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2010</span>;<span class="volume">20</span>;<span class="number">5</span>;<span class="pagerange">623-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20308636" rel="external">20308636</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2860164" rel="external">2860164</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.102970.109" rel="external">10.1101/gr.102970.109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PiggyBac transposon mutagenesis: a tool for cancer gene discovery in mice.</h4>
<p class="authors"><span class="author">Rad R</span>, <span class="author">Rad L</span>, <span class="author">Wang W</span>, <span class="author">Cadinanos J</span>, <span class="author">Vassiliou G</span>, <span class="author">Rice S</span>, <span class="author">Campos LS</span>, <span class="author">Yusa K</span>, <span class="author">Banerjee R</span>, <span class="author">Li MA</span>, <span class="author">de la Rosa J</span>, <span class="author">Strong A</span>, <span class="author">Lu D</span>, <span class="author">Ellis P</span>, <span class="author">Conte N</span>, <span class="author">Yang FT</span>, <span class="author">Liu P</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton-Cambridge CB10 1SA, UK.</p><p class="abstract">Transposons are mobile DNA segments that can disrupt gene function by inserting in or near genes. Here, we show that insertional mutagenesis by the PiggyBac transposon can be used for cancer gene discovery in mice. PiggyBac transposition in genetically engineered transposon-transposase mice induced cancers whose type (hematopoietic versus solid) and latency were dependent on the regulatory elements introduced into transposons. Analysis of 63 hematopoietic tumors revealed that PiggyBac is capable of genome-wide mutagenesis. The PiggyBac screen uncovered many cancer genes not identified in previous retroviral or Sleeping Beauty transposon screens, including Spic, which encodes a PU.1-related transcription factor, and Hdac7, a histone deacetylase gene. PiggyBac and Sleeping Beauty have different integration preferences. To maximize the utility of the tool, we engineered 21 mouse lines to be compatible with both transposon systems in constitutive, tissue- or temporal-specific mutagenesis. Mice with different transposon types, copy numbers, and chromosomal locations support wide applicability.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077186</span>, <span class="grant">079643</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">330</span>;<span class="number">6007</span>;<span class="pagerange">1104-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20947725">20947725</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3719098">3719098</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1193004">10.1126/science.1193004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains.</h4>
<p class="authors"><span class="author">Rakyan VK</span>, <span class="author">Down TA</span>, <span class="author">Maslau S</span>, <span class="author">Andrew T</span>, <span class="author">Yang TP</span>, <span class="author">Beyan H</span>, <span class="author">Whittaker P</span>, <span class="author">McCann OT</span>, <span class="author">Finer S</span>, <span class="author">Valdes AM</span>, <span class="author">Leslie RD</span>, <span class="author">Deloukas P</span> and <span class="author">Spector TD</span></p>
<p class="affiliation">Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London E1 2AT, UK. <a href="mailto:%76.%72%61%6B%79%61%6E%40%71%6D%75%6C.%61%63.%75%6B">&#118;&#46;&#114;&#97;&#107;&#121;&#97;&#110;&#64;&#113;&#109;&#117;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">There is a growing realization that some aging-associated phenotypes/diseases have an epigenetic basis. Here, we report the first genome-scale study of epigenomic dynamics during normal human aging. We identify aging-associated differentially methylated regions (aDMRs) in whole blood in a discovery cohort, and then replicate these aDMRs in sorted CD4(+) T-cells and CD14(+) monocytes in an independent cohort, suggesting that aDMRs occur in precursor haematopoietic cells. Further replication of the aDMRs in buccal cells, representing a tissue that originates from a different germ layer compared with blood, demonstrates that the aDMR signature is a multitissue phenomenon. Moreover, we demonstrate that aging-associated DNA hypermethylation occurs predominantly at bivalent chromatin domain promoters. This same category of promoters, associated with key developmental genes, is frequently hypermethylated in cancers and in vitro cell culture, pointing to a novel mechanistic link between aberrant hypermethylation in cancer, aging, and cell culture.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2010</span>;<span class="volume">20</span>;<span class="number">4</span>;<span class="pagerange">434-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20219945" rel="external">20219945</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2847746" rel="external">2847746</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.103101.109" rel="external">10.1101/gr.103101.109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Peptidase inhibitors in the MEROPS database.</h4>
<p class="authors"><span class="author">Rawlings ND</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The MEROPS website (http://merops.sanger.ac.uk) includes information on peptidase inhibitors as well as on peptidases and their substrates. Displays have been put in place to link peptidases and inhibitors together. The classification of protein peptidase inhibitors is continually being revised, and currently inhibitors are grouped into 67 families based on comparisons of protein sequences. These families can be further grouped into 38 clans based on comparisons of tertiary structure. Small molecule inhibitors are important reagents for peptidase characterization and, with the increasing importance of peptidases as drug targets, they are also important to the pharmaceutical industry. Small molecule inhibitors are now included in MEROPS and over 160 summaries have been written.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Biochimie</span>&nbsp;<span class="year">2010</span>;<span class="volume">92</span>;<span class="number">11</span>;<span class="pagerange">1463-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20430064" rel="external">20430064</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biochi.2010.04.013" rel="external">10.1016/j.biochi.2010.04.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MEROPS: the peptidase database.</h4>
<p class="authors"><span class="author">Rawlings ND</span>, <span class="author">Barrett AJ</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Peptidases, their substrates and inhibitors are of great relevance to biology, medicine and biotechnology. The MEROPS database (http://merops.sanger.ac.uk) aims to fulfil the need for an integrated source of information about these. The database has a hierarchical classification in which homologous sets of peptidases and protein inhibitors are grouped into protein species, which are grouped into families, which are in turn grouped into clans. The classification framework is used for attaching information at each level. An important focus of the database has become distinguishing one peptidase from another through identifying the specificity of the peptidase in terms of where it will cleave substrates and with which inhibitors it will interact. We have collected over 39,000 known cleavage sites in proteins, peptides and synthetic substrates. These allow us to display peptidase specificity and alignments of protein substrates to give an indication of how well a cleavage site is conserved, and thus its probable physiological relevance. While the number of new peptidase families and clans has only grown slowly the number of complete genomes has greatly increased. This has allowed us to add an analysis tool to the relevant species pages to show significant gains and losses of peptidase genes relative to related species.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">Database issue</span>;<span class="pagerange">D227-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19892822">19892822</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2808883">2808883</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkp971">10.1093/nar/gkp971</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">It&#39;s alive!</h4>
<p class="authors"><span class="author">Reid AJ</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">468</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20551972" rel="external">20551972</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2399" rel="external">10.1038/nrmicro2399</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CODA: accurate detection of functional associations between proteins in eukaryotic genomes using domain fusion.</h4>
<p class="authors"><span class="author">Reid AJ</span>, <span class="author">Ranea JA</span>, <span class="author">Clegg AB</span> and <span class="author">Orengo CA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, United Kingdom. <a href="mailto:%61%72%31%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#49;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: In order to understand how biological systems function it is necessary to determine the interactions and associations between proteins. Gene fusion prediction is one approach to detection of such functional relationships. Its use is however known to be problematic in higher eukaryotic genomes due to the presence of large homologous domain families. Here we introduce CODA (Co-Occurrence of Domains Analysis), a method to predict functional associations based on the gene fusion idiom.</p><p class="abstract">We apply a novel scoring scheme which takes account of the genome-specific size of homologous domain families involved in fusion to improve accuracy in predicting functional associations. We show that CODA is able to accurately predict functional similarities in human with comparison to state-of-the-art methods and show that different methods can be complementary. CODA is used to produce evidence that a currently uncharacterised human protein may be involved in pathways related to depression and that another is involved in DNA replication.</p><p class="abstract">The relative performance of different gene fusion methodologies has not previously been explored. We find that they are largely complementary, with different methods being more or less appropriate in different genomes. Our method is the only one currently available for download and can be run on an arbitrary dataset by the user. The CODA software and datasets are freely available from ftp://ftp.biochem.ucl.ac.uk/pub/gene3d_data/v6.1.0/CODA/. Predictions are also available via web services from http://funcnet.eu/.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e10908</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20532224">20532224</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2879367">2879367</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0010908">10.1371/journal.pone.0010908</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies five loci associated with lung function.</h4>
<p class="authors"><span class="author">Repapi E</span>, <span class="author">Sayers I</span>, <span class="author">Wain LV</span>, <span class="author">Burton PR</span>, <span class="author">Johnson T</span>, <span class="author">Obeidat M</span>, <span class="author">Zhao JH</span>, <span class="author">Ramasamy A</span>, <span class="author">Zhai G</span>, <span class="author">Vitart V</span>, <span class="author">Huffman JE</span>, <span class="author">Igl W</span>, <span class="author">Albrecht E</span>, <span class="author">Deloukas P</span>, <span class="author">Henderson J</span>, <span class="author">Granell R</span>, <span class="author">McArdle WL</span>, <span class="author">Rudnicka AR</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Barroso I</span>, <span class="author">Loos RJ</span>, <span class="author">Wareham NJ</span>, <span class="author">Mustelin L</span>, <span class="author">Rantanen T</span>, <span class="author">Surakka I</span>, <span class="author">Imboden M</span>, <span class="author">Wichmann HE</span>, <span class="author">Grkovic I</span>, <span class="author">Jankovic S</span>, <span class="author">Zgaga L</span>, <span class="author">Hartikainen AL</span>, <span class="author">Peltonen L</span>, <span class="author">Gyllensten U</span>, <span class="author">Johansson A</span>, <span class="author">Zaboli G</span>, <span class="author">Campbell H</span>, <span class="author">Wild SH</span>, <span class="author">Wilson JF</span>, <span class="author">Gl&auml;ser S</span>, <span class="author">Homuth G</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Mangino M</span>, <span class="author">Soranzo N</span>, <span class="author">Spector TD</span>, <span class="author">Polasek O</span>, <span class="author">Rudan I</span>, <span class="author">Wright AF</span>, <span class="author">Heli&ouml;vaara M</span>, <span class="author">Ripatti S</span>, <span class="author">Pouta A</span>, <span class="author">Naluai AT</span>, <span class="author">Olin AC</span>, <span class="author">Tor&eacute;n K</span>, <span class="author">Cooper MN</span>, <span class="author">James AL</span>, <span class="author">Palmer LJ</span>, <span class="author">Hingorani AD</span>, <span class="author">Wannamethee SG</span>, <span class="author">Whincup PH</span>, <span class="author">Smith GD</span>, <span class="author">Ebrahim S</span>, <span class="author">McKeever TM</span>, <span class="author">Pavord ID</span>, <span class="author">MacLeod AK</span>, <span class="author">Morris AD</span>, <span class="author">Porteous DJ</span>, <span class="author">Cooper C</span>, <span class="author">Dennison E</span>, <span class="author">Shaheen S</span>, <span class="author">Karrasch S</span>, <span class="author">Schnabel E</span>, <span class="author">Schulz H</span>, <span class="author">Grallert H</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Delplanque J</span>, <span class="author">Froguel P</span>, <span class="author">Blakey JD</span>, <span class="author">NSHD Respiratory Study Team</span>, <span class="author">Britton JR</span>, <span class="author">Morris RW</span>, <span class="author">Holloway JW</span>, <span class="author">Lawlor DA</span>, <span class="author">Hui J</span>, <span class="author">Nyberg F</span>, <span class="author">Jarvelin MR</span>, <span class="author">Jackson C</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kaprio J</span>, <span class="author">Probst-Hensch NM</span>, <span class="author">Koch B</span>, <span class="author">Hayward C</span>, <span class="author">Evans DM</span>, <span class="author">Elliott P</span>, <span class="author">Strachan DP</span>, <span class="author">Hall IP</span> and <span class="author">Tobin MD</span></p>
<p class="affiliation">Departments of Health Sciences and Genetics, Adrian Building, University of Leicester, Leicester, UK.</p><p class="abstract">Pulmonary function measures are heritable traits that predict morbidity and mortality and define chronic obstructive pulmonary disease (COPD). We tested genome-wide association with forced expiratory volume in 1 s (FEV(1)) and the ratio of FEV(1) to forced vital capacity (FVC) in the SpiroMeta consortium (n = 20,288 individuals of European ancestry). We conducted a meta-analysis of top signals with data from direct genotyping (n &lt; or = 32,184 additional individuals) and in silico summary association data from the CHARGE Consortium (n = 21,209) and the Health 2000 survey (n &lt; or = 883). We confirmed the reported locus at 4q31 and identified associations with FEV(1) or FEV(1)/FVC and common variants at five additional loci: 2q35 in TNS1 (P = 1.11 x 10(-12)), 4q24 in GSTCD (2.18 x 10(-23)), 5q33 in HTR4 (P = 4.29 x 10(-9)), 6p21 in AGER (P = 3.07 x 10(-15)) and 15q23 in THSD4 (P = 7.24 x 10(-15)). mRNA analyses showed expression of TNS1, GSTCD, AGER, HTR4 and THSD4 in human lung tissue. These associations offer mechanistic insight into pulmonary function regulation and indicate potential targets for interventions to alleviate respiratory disease.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; British Heart Foundation: <span class="grant">PG/06/154/22043</span>, <span class="grant">PG/97012</span>, <span class="grant">RG/08/013/25942</span>; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/710</span>, <span class="grant">CZD/16/6/2</span>, <span class="grant">CZD/16/6/4</span>; Department of Health: <span class="grant">0020029</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0000943</span>, <span class="grant">G0401540</span>, <span class="grant">G0500539</span>, <span class="grant">G0501942</span>, <span class="grant">G0600705</span>, <span class="grant">G0800582</span>, <span class="grant">G0801056</span>, <span class="grant">G0902125</span>, <span class="grant">G9815508</span>, <span class="grant">G990146</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U123092720</span>, <span class="grant">MC_U123092721</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">U.1230.00.008.00005.02</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIDDK NIH HHS: <span class="grant">U01 DK062418</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">075883</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span>, <span class="grant">086160/Z/08/A</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">1</span>;<span class="pagerange">36-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20010834">20010834</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2862965">2862965</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.501">10.1038/ng.501</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using randomised vectors in transcription factor binding site predictions</h4>
<p class="authors"><span class="author">Rezwan F</span>, <span class="author">Sun Y</span>, <span class="author">Davey N</span>, <span class="author">Adams R</span>, <span class="author">RUST AG</span>, <span class="author">Robinson M</span></p>
<p class="publication"><span class="title">Proceedings - 9th International Conference on Machine Learning and Applications, ICMLA</span> <span class="year">2010</span>;<span class="volume">5708880</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cross-species chromosome painting in bats from Madagascar: the contribution of Myzopodidae to revealing ancestral syntenies in Chiroptera.</h4>
<p class="authors"><span class="author">Richards LR</span>, <span class="author">Rambau RV</span>, <span class="author">Lamb JM</span>, <span class="author">Taylor PJ</span>, <span class="author">Yang F</span>, <span class="author">Schoeman MC</span> and <span class="author">Goodman SM</span></p>
<p class="affiliation">School of Biological and Conservation Sciences, University of KwaZulu-Natal, Westville Campus, Durban, 4001, South Africa. <a href="mailto:%6C%65%69%67%68%72%72%69%63%68%40%67%6D%61%69%6C.%63%6F%6D">&#108;&#101;&#105;&#103;&#104;&#114;&#114;&#105;&#99;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">The chiropteran fauna of Madagascar comprises eight of the 19 recognized families of bats, including the endemic Myzopodidae. While recent systematic studies of Malagasy bats have contributed to our understanding of the morphological and genetic diversity of the island&#39;s fauna, little is known about their cytosystematics. Here we investigate karyotypic relationships among four species, representing four families of Chiroptera endemic to the Malagasy region using cross-species chromosome painting with painting probes of Myotis myotis: Myzopodidae (Myzopoda aurita, 2n&thinsp;=&thinsp;26), Molossidae (Mormopterus jugularis, 2n&thinsp;=&thinsp;48), Miniopteridae (Miniopterus griveaudi, 2n&thinsp;=&thinsp;46), and Vespertilionidae (Myotis goudoti, 2n&thinsp;=&thinsp;44). This study represents the first time a member of the family Myzopodidae has been investigated using chromosome painting. Painting probes of M. myotis were used to delimit 29, 24, 23, and 22 homologous chromosomal segments in the genomes of M. aurita, M. jugularis, M. griveaudi, and M. goudoti, respectively. Comparison of GTG-banded homologous chromosomes/chromosomal segments among the four species revealed the genome of M. aurita has been structured through 14 fusions of chromosomes and chromosomal segments of M. myotis chromosomes leading to a karyotype consisting solely of bi-armed chromosomes. In addition, chromosome painting revealed a novel X-autosome translocation in M. aurita. Comparison of our results with published chromosome maps provided further evidence for karyotypic conservatism within the genera Mormopterus, Miniopterus, and Myotis. Mapping of chromosomal rearrangements onto a molecular consensus phylogeny revealed ancestral syntenies shared between Myzopoda and other bat species of the infraorders Pteropodiformes and Vespertilioniformes. Our study provides further evidence for the involvement of Robertsonian (Rb) translocations and fusions/fissions in chromosomal evolution within Chiroptera.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">6</span>;<span class="pagerange">635-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20596765" rel="external">20596765</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-010-9139-6" rel="external">10.1007/s10577-010-9139-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A multilocus genetic risk score for coronary heart disease: case-control and prospective cohort analyses.</h4>
<p class="authors"><span class="author">Ripatti S</span>, <span class="author">Tikkanen E</span>, <span class="author">Orho-Melander M</span>, <span class="author">Havulinna AS</span>, <span class="author">Silander K</span>, <span class="author">Sharma A</span>, <span class="author">Guiducci C</span>, <span class="author">Perola M</span>, <span class="author">Jula A</span>, <span class="author">Sinisalo J</span>, <span class="author">Lokki ML</span>, <span class="author">Nieminen MS</span>, <span class="author">Melander O</span>, <span class="author">Salomaa V</span>, <span class="author">Peltonen L</span> and <span class="author">Kathiresan S</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland. <a href="mailto:%73%61%6D%75%6C%69.%72%69%70%61%74%74%69%40%66%69">&#115;&#97;&#109;&#117;&#108;&#105;&#46;&#114;&#105;&#112;&#97;&#116;&#116;&#105;&#64;&#102;&#105;</a> mm.fi</p><p class="abstract"><strong>Background</strong>: Comparison of patients with coronary heart disease and controls in genome-wide association studies has revealed several single nucleotide polymorphisms (SNPs) associated with coronary heart disease. We aimed to establish the external validity of these findings and to obtain more precise risk estimates using a prospective cohort design.</p><p class="abstract"><strong>Methods</strong>: We tested 13 recently discovered SNPs for association with coronary heart disease in a case-control design including participants differing from those in the discovery samples (3829 participants with prevalent coronary heart disease and 48,897 controls free of the disease) and a prospective cohort design including 30,725 participants free of cardiovascular disease from Finland and Sweden. We modelled the 13 SNPs as a multilocus genetic risk score and used Cox proportional hazards models to estimate the association of genetic risk score with incident coronary heart disease. For case-control analyses we analysed associations between individual SNPs and quintiles of genetic risk score using logistic regression.</p><p class="abstract"><strong>Findings</strong>: In prospective cohort analyses, 1264 participants had a first coronary heart disease event during a median 10&middot;7 years&#39; follow-up (IQR 6&middot;7-13&middot;6). Genetic risk score was associated with a first coronary heart disease event. When compared with the bottom quintile of genetic risk score, participants in the top quintile were at 1&middot;66-times increased risk of coronary heart disease in a model adjusting for traditional risk factors (95% CI 1&middot;35-2&middot;04, p value for linear trend=7&middot;3&times;10(-10)). Adjustment for family history did not change these estimates. Genetic risk score did not improve C index over traditional risk factors and family history (p=0&middot;19), nor did it have a significant effect on net reclassification improvement (2&middot;2%, p=0&middot;18); however, it did have a small effect on integrated discrimination index (0&middot;004, p=0&middot;0006). Results of the case-control analyses were similar to those of the prospective cohort analyses.</p><p class="abstract"><strong>Interpretation</strong>: Using a genetic risk score based on 13 SNPs associated with coronary heart disease, we can identify the 20% of individuals of European ancestry who are at roughly 70% increased risk of a first coronary heart disease event. The potential clinical use of this panel of SNPs remains to be defined.</p><p class="abstract"><strong>Funding</strong>: The Wellcome Trust; Academy of Finland Center of Excellence for Complex Disease Genetics; US National Institutes of Health; the Donovan Family Foundation.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">(R01 HL087676</span>; Wellcome Trust: <span class="grant">WT089061/Z/09/Z</span>, <span class="grant">WT089062/Z/09/Z</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2010</span>;<span class="volume">376</span>;<span class="number">9750</span>;<span class="pagerange">1393-400</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20971364">20971364</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2965351">2965351</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2810%2961267-6">10.1016/S0140-6736(10)61267-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data analysis issues for allele-specific expression using Illumina&#39;s GoldenGate assay.</h4>
<p class="authors"><span class="author">Ritchie ME</span>, <span class="author">Forrest MS</span>, <span class="author">Dimas AS</span>, <span class="author">Daelemans C</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Deloukas P</span> and <span class="author">Tavar&eacute; S</span></p>
<p class="affiliation">Bioinformatics Division, The Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, Victoria, 3052, Australia. <a href="mailto:%6D%72%69%74%63%68%69%65%40%77%65%68%69.%65%64%75.%61%75">&#109;&#114;&#105;&#116;&#99;&#104;&#105;&#101;&#64;&#119;&#101;&#104;&#105;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract"><strong>Background</strong>: High-throughput measurement of allele-specific expression (ASE) is a relatively new and exciting application area for array-based technologies. In this paper, we explore several data sets which make use of Illumina&#39;s GoldenGate BeadArray technology to measure ASE. This platform exploits coding SNPs to obtain relative expression measurements for alleles at approximately 1500 positions in the genome.</p><p class="abstract"><strong>Results</strong>: We analyze data from a mixture experiment where genomic DNA samples from pairs of individuals of known genotypes are pooled to create allelic imbalances at varying levels for the majority of SNPs on the array. We observe that GoldenGate has less sensitivity at detecting subtle allelic imbalances (around 1.3 fold) compared to extreme imbalances, and note the benefit of applying local background correction to the data. Analysis of data from a dye-swap control experiment allowed us to quantify dye-bias, which can be reduced considerably by careful normalization. The need to filter the data before carrying out further downstream analysis to remove non-responding probes, which show either weak, or non-specific signal for each allele, was also demonstrated. Throughout this paper, we find that a linear model analysis of the data from each SNP is a flexible modelling strategy that allows for testing of allelic imbalances in each sample when replicate hybridizations are available.</p><p class="abstract"><strong>Conclusions</strong>: Our analysis shows that local background correction carried out by Illumina&#39;s software, together with quantile normalization of the red and green channels within each array, provides optimal performance in terms of false positive rates. In addition, we strongly encourage intensity-based filtering to remove SNPs which only measure non-specific signal. We anticipate that a similar analysis strategy will prove useful when quantifying ASE on Illumina&#39;s higher density Infinium BeadChips.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">280</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20504309" rel="external">20504309</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2887809" rel="external">2887809</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-11-280" rel="external">10.1186/1471-2105-11-280</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ISA software suite: supporting standards-compliant experimental annotation and enabling curation at the community level.</h4>
<p class="authors"><span class="author">Rocca-Serra P</span>, <span class="author">Brandizi M</span>, <span class="author">Maguire E</span>, <span class="author">Sklyar N</span>, <span class="author">Taylor C</span>, <span class="author">Begley K</span>, <span class="author">Field D</span>, <span class="author">Harris S</span>, <span class="author">Hide W</span>, <span class="author">Hofmann O</span>, <span class="author">Neumann S</span>, <span class="author">Sterk P</span>, <span class="author">Tong W</span> and <span class="author">Sansone SA</span></p>
<p class="affiliation">The European Bioinformatics Institute, Wellcome Trust Genome Campus, Cambridge, UK.</p><p class="abstract">The first open source software suite for experimentalists and curators that (i) assists in the annotation and local management of experimental metadata from high-throughput studies employing one or a combination of omics and other technologies; (ii) empowers users to uptake community-defined checklists and ontologies; and (iii) facilitates submission to international public repositories. Availability and Implementation: Software, documentation, case studies and implementations at http://www.isa-tools.org.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E025080/1</span>, <span class="grant">BB/G000638/1</span>, <span class="grant">BB/I000917/1</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">18</span>;<span class="pagerange">2354-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20679334" rel="external">20679334</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2935443" rel="external">2935443</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq415" rel="external">10.1093/bioinformatics/btq415</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lung infections in cystic fibrosis: deriving clinical insight from microbial complexity.</h4>
<p class="authors"><span class="author">Rogers GB</span>, <span class="author">Stressmann FA</span>, <span class="author">Walker AW</span>, <span class="author">Carroll MP</span> and <span class="author">Bruce KD</span></p>
<p class="affiliation">Molecular Microbiology Research Laboratory, Pharmaceutical Science Division, 150 Stamford Street, Franklin-Wilkins Building, King&#39;s College London, London, SE1 9NH, UK. <a href="mailto:%67%65%72%61%69%6E%74.%72%6F%67%65%72%73%40%6B%63%6C.%61%63.%75%6B">&#103;&#101;&#114;&#97;&#105;&#110;&#116;&#46;&#114;&#111;&#103;&#101;&#114;&#115;&#64;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Lower respiratory tract bacterial infections, such as those associated with cystic fibrosis lung disease, represent a major healthcare burden. Treatment strategies are currently informed by culture-based routine diagnostics whose limitations, including an inability to isolate all potentially clinically significant bacterial species present in a sample, are well documented. Some advances have resulted from the introduction of culture-independent molecular assays for the detection of specific pathogens. However, the application of bacterial community profiling techniques to the characterization of these infections has revealed much higher levels of microbial diversity than previously recognized. These findings are leading to a fundamental shift in the way such infections are considered. Increasingly, polymicrobial infections are being viewed as complex communities of interacting organisms, with dynamic processes key to their pathogenicity. Such a model requires an analytical strategy that provides insight into the interactions of all members of the infective community. The rapid advance in sequencing technology, along with protocols that limit analysis to viable bacterial cells, are for the first time providing an opportunity to gain such insight.</p>
<p class="publication"><span class="title">Expert review of molecular diagnostics</span>&nbsp;<span class="year">2010</span>;<span class="volume">10</span>;<span class="number">2</span>;<span class="pagerange">187-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20214537" rel="external">20214537</a>; DOI: <a href="http://dx.doi.org/10.1586%2Ferm.09.81" rel="external">10.1586/erm.09.81</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and characterization of two novel JARID1C mutations: suggestion of an emerging genotype-phenotype correlation.</h4>
<p class="authors"><span class="author">Rujirabanjerd S</span>, <span class="author">Nelson J</span>, <span class="author">Tarpey PS</span>, <span class="author">Hackett A</span>, <span class="author">Edkins S</span>, <span class="author">Raymond FL</span>, <span class="author">Schwartz CE</span>, <span class="author">Turner G</span>, <span class="author">Iwase S</span>, <span class="author">Shi Y</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span> and <span class="author">Gecz J</span></p>
<p class="affiliation">Department of Pathology, Faculty of Medicine, Prince of Songkla University, Songkhla, Thailand.</p><p class="abstract">Mental retardation (MR) is characterized by cognitive impairment with an IQ &lt;70. Many of the major causes are genetically determined and the approximately 30% male excess suggests that mutations in genes carried on the X chromosome are disproportionably represented. One such gene, jumonji AT-rich interactive domain 1C (JARID1C) on Xp11.2, has been identified in families with X-linked MR (XLMR), with 18 different mutations reported to date. As part of a systematic resequencing of 720 genes in 208 XLMR families of the International Genetic of Learning Disability (IGOLD) consortium, two novel nucleotide changes in the JARID1C coding region were identified, with the nucleotide changes segregating with the disease phenotype in the two families. The first mutation is a single-nucleotide insertion in exon 21 (c.3258_3259insC p.K1087fs(*)43) causing a frameshift and resulting in a premature termination codon (PTC). Such PTC-containing mRNAs are generally degraded by nonsense-mediated mRNA decay (NMD) surveillance, but our results show that this is not the case with this mutation. The other change is a single-nucleotide substitution in exon 12 (c.1160C&gt;A) in a published family with nonsyndromic MR, MRX13. This change occurs in a highly conserved amino acid, with proline (P) being substituted by threonine (T) (p.P544T). Functional analysis shows that this amino-acid substitution compromises both tri- and didemethylase activity of the JARID1C protein. We conclude that the two novel changes impair JARID1C protein function and are disease-causing mutations in these families.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">3</span>;<span class="pagerange">330-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19826449" rel="external">19826449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987212" rel="external">2987212</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2009.175" rel="external">10.1038/ejhg.2009.175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic architecture characterizes tumor progression paths and fate in breast cancer patients.</h4>
<p class="authors"><span class="author">Russnes HG</span>, <span class="author">Vollan HK</span>, <span class="author">Lingjaerde OC</span>, <span class="author">Krasnitz A</span>, <span class="author">Lundin P</span>, <span class="author">Naume B</span>, <span class="author">S&oslash;rlie T</span>, <span class="author">Borgen E</span>, <span class="author">Rye IH</span>, <span class="author">Langer&oslash;d A</span>, <span class="author">Chin SF</span>, <span class="author">Teschendorff AE</span>, <span class="author">Stephens PJ</span>, <span class="author">M&aring;n&eacute;r S</span>, <span class="author">Schlichting E</span>, <span class="author">Baumbusch LO</span>, <span class="author">K&aring;resen R</span>, <span class="author">Stratton MP</span>, <span class="author">Wigler M</span>, <span class="author">Caldas C</span>, <span class="author">Zetterberg A</span>, <span class="author">Hicks J</span> and <span class="author">B&oslash;rresen-Dale AL</span></p>
<p class="affiliation">Department of Genetics, Institute for Cancer Research, Oslo University Hospital Radiumhospitalet, 0310 Oslo, Norway.</p><p class="abstract">Distinct molecular subtypes of breast carcinomas have been identified, but translation into clinical use has been limited. We have developed two platform-independent algorithms to explore genomic architectural distortion using array comparative genomic hybridization data to measure (i) whole-arm gains and losses [whole-arm aberration index (WAAI)] and (ii) complex rearrangements [complex arm aberration index (CAAI)]. By applying CAAI and WAAI to data from 595 breast cancer patients, we were able to separate the cases into eight subgroups with different distributions of genomic distortion. Within each subgroup data from expression analyses, sequencing and ploidy indicated that progression occurs along separate paths into more complex genotypes. Histological grade had prognostic impact only in the luminal-related groups, whereas the complexity identified by CAAI had an overall independent prognostic power. This study emphasizes the relation among structural genomic alterations, molecular subtype, and clinical behavior and shows that objective score of genomic complexity (CAAI) is an independent prognostic marker in breast cancer.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C507/A3086</span></p>
<p class="publication"><span class="title">Science translational medicine</span>&nbsp;<span class="year">2010</span>;<span class="volume">2</span>;<span class="number">38</span>;<span class="pagerange">38ra47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20592421">20592421</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscitranslmed.3000611">10.1126/scitranslmed.3000611</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple independent loci at chromosome 15q25.1 affect smoking quantity: a meta-analysis and comparison with lung cancer and COPD.</h4>
<p class="authors"><span class="author">Saccone NL</span>, <span class="author">Culverhouse RC</span>, <span class="author">Schwantes-An TH</span>, <span class="author">Cannon DS</span>, <span class="author">Chen X</span>, <span class="author">Cichon S</span>, <span class="author">Giegling I</span>, <span class="author">Han S</span>, <span class="author">Han Y</span>, <span class="author">Keskitalo-Vuokko K</span>, <span class="author">Kong X</span>, <span class="author">Landi MT</span>, <span class="author">Ma JZ</span>, <span class="author">Short SE</span>, <span class="author">Stephens SH</span>, <span class="author">Stevens VL</span>, <span class="author">Sun L</span>, <span class="author">Wang Y</span>, <span class="author">Wenzlaff AS</span>, <span class="author">Aggen SH</span>, <span class="author">Breslau N</span>, <span class="author">Broderick P</span>, <span class="author">Chatterjee N</span>, <span class="author">Chen J</span>, <span class="author">Heath AC</span>, <span class="author">Heli&ouml;vaara M</span>, <span class="author">Hoft NR</span>, <span class="author">Hunter DJ</span>, <span class="author">Jensen MK</span>, <span class="author">Martin NG</span>, <span class="author">Montgomery GW</span>, <span class="author">Niu T</span>, <span class="author">Payne TJ</span>, <span class="author">Peltonen L</span>, <span class="author">Pergadia ML</span>, <span class="author">Rice JP</span>, <span class="author">Sherva R</span>, <span class="author">Spitz MR</span>, <span class="author">Sun J</span>, <span class="author">Wang JC</span>, <span class="author">Weiss RB</span>, <span class="author">Wheeler W</span>, <span class="author">Witt SH</span>, <span class="author">Yang BZ</span>, <span class="author">Caporaso NE</span>, <span class="author">Ehringer MA</span>, <span class="author">Eisen T</span>, <span class="author">Gapstur SM</span>, <span class="author">Gelernter J</span>, <span class="author">Houlston R</span>, <span class="author">Kaprio J</span>, <span class="author">Kendler KS</span>, <span class="author">Kraft P</span>, <span class="author">Leppert MF</span>, <span class="author">Li MD</span>, <span class="author">Madden PA</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Pillai S</span>, <span class="author">Rietschel M</span>, <span class="author">Rujescu D</span>, <span class="author">Schwartz A</span>, <span class="author">Amos CI</span> and <span class="author">Bierut LJ</span></p>
<p class="affiliation">Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, USA. <a href="mailto:%6E%6C%69%6D%73%40%67%65%6E%65%74%69%63%73.%77%75%73%74%6C.%65%64%75">&#110;&#108;&#105;&#109;&#115;&#64;&#103;&#101;&#110;&#101;&#116;&#105;&#99;&#115;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Recently, genetic association findings for nicotine dependence, smoking behavior, and smoking-related diseases converged to implicate the chromosome 15q25.1 region, which includes the CHRNA5-CHRNA3-CHRNB4 cholinergic nicotinic receptor subunit genes. In particular, association with the nonsynonymous CHRNA5 SNP rs16969968 and correlates has been replicated in several independent studies. Extensive genotyping of this region has suggested additional statistically distinct signals for nicotine dependence, tagged by rs578776 and rs588765. One goal of the Consortium for the Genetic Analysis of Smoking Phenotypes (CGASP) is to elucidate the associations among these markers and dichotomous smoking quantity (heavy versus light smoking), lung cancer, and chronic obstructive pulmonary disease (COPD). We performed a meta-analysis across 34 datasets of European-ancestry subjects, including 38,617 smokers who were assessed for cigarettes-per-day, 7,700 lung cancer cases and 5,914 lung-cancer-free controls (all smokers), and 2,614 COPD cases and 3,568 COPD-free controls (all smokers). We demonstrate statistically independent associations of rs16969968 and rs588765 with smoking (mutually adjusted p-values&lt;10(-35) and &lt;10(-8) respectively). Because the risk alleles at these loci are negatively correlated, their association with smoking is stronger in the joint model than when each SNP is analyzed alone. Rs578776 also demonstrates association with smoking after adjustment for rs16969968 (p&lt;10(-6)). In models adjusting for cigarettes-per-day, we confirm the association between rs16969968 and lung cancer (p&lt;10(-20)) and observe a nominally significant association with COPD (p = 0.01); the other loci are not significantly associated with either lung cancer or COPD after adjusting for rs16969968. This study provides strong evidence that multiple statistically distinct loci in this region affect smoking behavior. This study is also the first report of association between rs588765 (and correlates) and smoking that achieves genome-wide significance; these SNPs have previously been associated with mRNA levels of CHRNA5 in brain and lung tissue.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA016672</span>, <span class="grant">CA121197</span>, <span class="grant">CA127219</span>, <span class="grant">CA133996</span>, <span class="grant">CA55769</span>, <span class="grant">N01 CN-25524</span>, <span class="grant">N01 CN-75022</span>, <span class="grant">N01 PC35145</span>, <span class="grant">N01-CN-25404</span>, <span class="grant">N01-CN-25476</span>, <span class="grant">N01-CN-25511</span>, <span class="grant">N01-CN-25512</span>, <span class="grant">N01-CN-25513</span>, <span class="grant">N01-CN-25514</span>, <span class="grant">N01-CN-25515</span>, <span class="grant">N01-CN-25516</span>, <span class="grant">N01-CN-25518</span>, <span class="grant">N01-CN-25522</span>, <span class="grant">P01 CA089392</span>, <span class="grant">P01 CA089392-08</span>, <span class="grant">P01CA087969</span>, <span class="grant">P30 CA22453</span>, <span class="grant">R01 CA60691</span>, <span class="grant">R01 CA87895</span>; NHGRI NIH HHS: <span class="grant">5U01HG004399</span>, <span class="grant">U01 HG004422-02</span>, <span class="grant">U01 HG004446</span>, <span class="grant">U01 HG04422-01</span>, <span class="grant">U01HG004438</span>, <span class="grant">U01HG04424</span>; NHLBI NIH HHS: <span class="grant">5R01HL035464</span>, <span class="grant">P01-HL72903</span>; NIAAA NIH HHS: <span class="grant">AA11330</span>, <span class="grant">K01 AA015336</span>, <span class="grant">K05 AA017688-04</span>, <span class="grant">R01 AA011949</span>, <span class="grant">R01 AA017889</span>, <span class="grant">R01 AA017889-01A1</span>, <span class="grant">R01 AA017889-02</span>, <span class="grant">R01 AA017889-03</span>, <span class="grant">R01 AA017889-04</span>, <span class="grant">R01 AA11998</span>, <span class="grant">R01 AA13320</span>, <span class="grant">R01 AA13321</span>, <span class="grant">T32 AA007464</span>; NICHD NIH HHS: <span class="grant">P01 HD31921</span>; NIDA NIH HHS: <span class="grant">DA-13783</span>, <span class="grant">DA12849</span>, <span class="grant">K01DA019498</span>, <span class="grant">K01DA024758</span>, <span class="grant">K02 DA021237</span>, <span class="grant">K02 DA021237-05</span>, <span class="grant">K08 DA019951</span>, <span class="grant">K08 DA019951-05</span>, <span class="grant">P60 DA011015</span>, <span class="grant">R01 DA-12844</span>, <span class="grant">R01 DA012845</span>, <span class="grant">R01 DA021913</span>, <span class="grant">R01 DA021913-01A1</span>, <span class="grant">R01 DA021913-02</span>, <span class="grant">R01 DA021913-03</span>, <span class="grant">R01 DA021913-04</span>, <span class="grant">R01 DA026911</span>, <span class="grant">R01 DA12854</span>, <span class="grant">R01 DA12890</span>, <span class="grant">R03 DA023166</span>, <span class="grant">R21 DA026901</span>, <span class="grant">R21DA027070</span>, <span class="grant">R56 DA12854</span>; NIDDK NIH HHS: <span class="grant">5P01DK070756</span>; NIEHS NIH HHS: <span class="grant">P30ES007784</span>; NIGMS NIH HHS: <span class="grant">K25 GM69590</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20700436" rel="external">20700436</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2916847" rel="external">2916847</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001053" rel="external">10.1371/journal.pgen.1001053</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Three authors reply</h4>
<p class="authors"><span class="author">Salanti G</span>, <span class="author">ZEGGINI E</span>, <span class="author">Ioannidis JPA</span></p>
<p class="publication"><span class="title">American Journal of Epidemiology.</span> <span class="year">2010</span>;<span class="volume">171</span>;<span class="pagerange">1155-6</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of the 9p21.3 locus with risk of first-ever myocardial infarction in Pakistanis: case-control study in South Asia and updated meta-analysis of Europeans.</h4>
<p class="authors"><span class="author">Saleheen D</span>, <span class="author">Alexander M</span>, <span class="author">Rasheed A</span>, <span class="author">Wormser D</span>, <span class="author">Soranzo N</span>, <span class="author">Hammond N</span>, <span class="author">Butterworth A</span>, <span class="author">Zaidi M</span>, <span class="author">Haycock P</span>, <span class="author">Bumpstead S</span>, <span class="author">Potter S</span>, <span class="author">Blackburn H</span>, <span class="author">Gray E</span>, <span class="author">Di Angelantonio E</span>, <span class="author">Kaptoge S</span>, <span class="author">Shah N</span>, <span class="author">Samuel M</span>, <span class="author">Janjua A</span>, <span class="author">Sheikh N</span>, <span class="author">Haider SR</span>, <span class="author">Murtaza M</span>, <span class="author">Ahmad U</span>, <span class="author">Hakeem A</span>, <span class="author">Memon MA</span>, <span class="author">Mallick NH</span>, <span class="author">Azhar M</span>, <span class="author">Samad A</span>, <span class="author">Rasheed SZ</span>, <span class="author">Gardezi AR</span>, <span class="author">Memon NA</span>, <span class="author">Ghaffar A</span>, <span class="author">Memon FU</span>, <span class="author">Zaman KS</span>, <span class="author">Kundi A</span>, <span class="author">Yaqoob Z</span>, <span class="author">Cheema LA</span>, <span class="author">Qamar N</span>, <span class="author">Faruqui A</span>, <span class="author">Jooma R</span>, <span class="author">Niazi JH</span>, <span class="author">Hussain M</span>, <span class="author">Kumar K</span>, <span class="author">Saleem A</span>, <span class="author">Kumar K</span>, <span class="author">Daood MS</span>, <span class="author">Memon F</span>, <span class="author">Gul AA</span>, <span class="author">Abbas S</span>, <span class="author">Zafar J</span>, <span class="author">Shahid F</span>, <span class="author">Memon Z</span>, <span class="author">Bhatti SM</span>, <span class="author">Kayani W</span>, <span class="author">Ali SS</span>, <span class="author">Fahim M</span>, <span class="author">Ishaq M</span>, <span class="author">Frossard P</span>, <span class="author">Deloukas P</span> and <span class="author">Danesh J</span></p>
<p class="affiliation">Center for Non-Communicable Diseases, Karachi, Pakistan. <a href="mailto:%64%61%6E%69%73%68.%73%61%6C%65%68%65%65%6E%40%63%6E%63%64%70%6B.%63%6F%6D">&#100;&#97;&#110;&#105;&#115;&#104;&#46;&#115;&#97;&#108;&#101;&#104;&#101;&#101;&#110;&#64;&#99;&#110;&#99;&#100;&#112;&#107;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Objective</strong>: To examine variants at the 9p21 locus in a case-control study of acute myocardial infarction (MI) in Pakistanis and to perform an updated meta-analysis of published studies in people of European ancestry.</p><p class="abstract">A total of 1851 patients with first-ever confirmed MI and 1903 controls were genotyped for 89 tagging single-nucleotide polymorphisms at locus 9p21, including the lead variant (rs1333049) identified by the Wellcome Trust Case Control Consortium. Minor allele frequencies and extent of linkage disequilibrium observed in Pakistanis were broadly similar to those seen in Europeans. In the Pakistani study, 6 variants were associated with MI (P&lt;10(-2)) in the initial sample set, and in an additional 741 cases and 674 controls in whom further genotyping was performed for these variants. For Pakistanis, the odds ratio for MI was 1.13 (95% CI, 1.05 to 1.22; P=2 x 10(-3)) for each copy of the C allele at rs1333049. In comparison, a meta-analysis of studies in Europeans yielded an odds ratio of 1.31 (95% CI, 1.26 to 1.37) for the same variant (P=1 x 10(-3) for heterogeneity). Meta-analyses of 23 variants, in up to 38,250 cases and 84,820 controls generally yielded higher values in Europeans than in Pakistanis.</p><p class="abstract"><strong>Conclusions</strong>: To our knowledge, this study provides the first demonstration that variants at the 9p21 locus are significantly associated with MI risk in Pakistanis. However, association signals at this locus were weaker in Pakistanis than those in European studies.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">7</span>;<span class="pagerange">1467-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20395598">20395598</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.109.197210">10.1161/ATVBAHA.109.197210</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic determinants of major blood lipids in Pakistanis compared with Europeans.</h4>
<p class="authors"><span class="author">Saleheen D</span>, <span class="author">Soranzo N</span>, <span class="author">Rasheed A</span>, <span class="author">Scharnagl H</span>, <span class="author">Gwilliam R</span>, <span class="author">Alexander M</span>, <span class="author">Inouye M</span>, <span class="author">Zaidi M</span>, <span class="author">Potter S</span>, <span class="author">Haycock P</span>, <span class="author">Bumpstead S</span>, <span class="author">Kaptoge S</span>, <span class="author">Di Angelantonio E</span>, <span class="author">Sarwar N</span>, <span class="author">Hunt SE</span>, <span class="author">Sheikh N</span>, <span class="author">Shah N</span>, <span class="author">Samuel M</span>, <span class="author">Haider SR</span>, <span class="author">Murtaza M</span>, <span class="author">Thompson A</span>, <span class="author">Gobin R</span>, <span class="author">Butterworth A</span>, <span class="author">Ahmad U</span>, <span class="author">Hakeem A</span>, <span class="author">Zaman KS</span>, <span class="author">Kundi A</span>, <span class="author">Yaqoob Z</span>, <span class="author">Cheema LA</span>, <span class="author">Qamar N</span>, <span class="author">Faruqui A</span>, <span class="author">Mallick NH</span>, <span class="author">Azhar M</span>, <span class="author">Samad A</span>, <span class="author">Ishaq M</span>, <span class="author">Rasheed SZ</span>, <span class="author">Jooma R</span>, <span class="author">Niazi JH</span>, <span class="author">Gardezi AR</span>, <span class="author">Memon NA</span>, <span class="author">Ghaffar A</span>, <span class="author">Rehman FU</span>, <span class="author">Hoffmann MM</span>, <span class="author">Renner W</span>, <span class="author">Kleber ME</span>, <span class="author">Grammer TB</span>, <span class="author">Stephens J</span>, <span class="author">Attwood A</span>, <span class="author">Koch K</span>, <span class="author">Hussain M</span>, <span class="author">Kumar K</span>, <span class="author">Saleem A</span>, <span class="author">Kumar K</span>, <span class="author">Daood MS</span>, <span class="author">Gul AA</span>, <span class="author">Abbas S</span>, <span class="author">Zafar J</span>, <span class="author">Shahid F</span>, <span class="author">Bhatti SM</span>, <span class="author">Ali SS</span>, <span class="author">Muhammad F</span>, <span class="author">Sagoo G</span>, <span class="author">Bray S</span>, <span class="author">McGinnis R</span>, <span class="author">Dudbridge F</span>, <span class="author">Winkelmann BR</span>, <span class="author">B&ouml;ehm B</span>, <span class="author">Thompson S</span>, <span class="author">Ouwehand W</span>, <span class="author">M&auml;rz W</span>, <span class="author">Frossard P</span>, <span class="author">Danesh J</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Center for Non-Communicable Diseases Karachi, Pakistan. <a href="mailto:%64%61%6E%69%73%68.%73%61%6C%65%68%65%65%6E%40%63%6E%63%64%70%6B.%63%6F%6D">&#100;&#97;&#110;&#105;&#115;&#104;&#46;&#115;&#97;&#108;&#101;&#104;&#101;&#101;&#110;&#64;&#99;&#110;&#99;&#100;&#112;&#107;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: Evidence is sparse about the genetic determinants of major lipids in Pakistanis.</p><p class="abstract">Variants (n=45 000) across 2000 genes were assessed in 3200 Pakistanis and compared with 2450 Germans using the same gene array and similar lipid assays. We also did a meta-analysis of selected lipid-related variants in Europeans. Pakistani genetic architecture was distinct from that of several ethnic groups represented in international reference samples. Forty-one variants at 14 loci were significantly associated with levels of HDL-C, triglyceride, or LDL-C. The most significant lipid-related variants identified among Pakistanis corresponded to genes previously shown to be relevant to Europeans, such as CETP associated with HDL-C levels (rs711752; P&lt;10(-13)), APOA5/ZNF259 (rs651821; P&lt;10(-13)) and GCKR (rs1260326; P&lt;10(-13)) with triglyceride levels; and CELSR2 variants with LDL-C levels (rs646776; P&lt;10(-9)). For Pakistanis, these 41 variants explained 6.2%, 7.1%, and 0.9% of the variation in HDL-C, triglyceride, and LDL-C, respectively. Compared with Europeans, the allele frequency of rs662799 in APOA5 among Pakistanis was higher and its impact on triglyceride concentration was greater (P-value for difference &lt;10(-4)).</p><p class="abstract"><strong>Conclusions</strong>: Several lipid-related genetic variants are common to Pakistanis and Europeans, though they explain only a modest proportion of population variation in lipid concentration. Allelic frequencies and effect sizes of lipid-related variants can differ between Pakistanis and Europeans.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Circulation. Cardiovascular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">3</span>;<span class="number">4</span>;<span class="pagerange">348-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20570915">20570915</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCGENETICS.109.906180">10.1161/CIRCGENETICS.109.906180</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic risk prediction.</h4>
<p class="authors"><span class="author">Sandhu M</span>, <span class="author">Wood A</span> and <span class="author">Young E</span></p>
<p class="affiliation">Department of Public Health and Primary Care, Strangeways Research Laboratory, University of Cambridge, Cambridge CB1 8RN, UK. <a href="mailto:%6D%61%6E%6A.%73%61%6E%64%68%75%40%73%72%6C.%63%61%6D.%61%63.%75%6B">&#109;&#97;&#110;&#106;&#46;&#115;&#97;&#110;&#100;&#104;&#117;&#64;&#115;&#114;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2010</span>;<span class="volume">376</span>;<span class="number">9750</span>;<span class="pagerange">1366-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20971347" rel="external">20971347</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736(10)61921-6" rel="external">10.1016/S0140-6736(10)61921-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GLO1-A novel amplified gene in human cancer.</h4>
<p class="authors"><span class="author">Santarius T</span>, <span class="author">Bignell GR</span>, <span class="author">Greenman CD</span>, <span class="author">Widaa S</span>, <span class="author">Chen L</span>, <span class="author">Mahoney CL</span>, <span class="author">Butler A</span>, <span class="author">Edkins S</span>, <span class="author">Waris S</span>, <span class="author">Thornalley PJ</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%74%73%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#115;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">To identify a novel amplified cancer gene a systematic screen of 975 human cancer DNA samples, 750 cell lines and 225 primary tumors, using the Affymetrix 10K SNP microarray was undertaken. The screen identified 193 amplicons. A previously uncharacterized amplicon located on 6p21.2 whose 1 Mb minimal common amplified region contained eight genes (GLO1, DNAH8, GLP1R, C6orf64, KCNK5, KCNK17, KCNK16, and C6orf102) was further investigated to determine which gene(s) are the biological targets of this amplicon. Real time quantitative PCR (qPCR) analysis of all amplicon 6p21.2 genes in 618 human cancer cell lines identified GLO1, encoding glyoxalase 1, to be the most frequently amplified gene [twofold or greater amplification in 8.4% (49/536) of cancers]. Also the association between amplification and overexpression was greatest for GLO1. RNAi knockdown of GLO1 had the greatest and most consistent impact on cell accumulation and apoptosis. Cell lines with GLO1 amplification were more sensitive to inhibition of GLO1 by bromobenzylglutathione cyclopentyl diester (BBGC). Subsequent qPCR of 520 primary tumor samples identified twofold and greater amplification of GLO1 in 8/37 (22%) of breast, 12/71 (17%) of sarcomas, 6/53 (11.3%) of nonsmall cell lung, 2/23 (8.7%) of bladder, 6/93 (6.5%) of renal and 5/83 (6%) of gastric cancers. Amplification of GLO1 was rare in colon cancer (1/35) and glioma (1/94). Collectively the results indicate that GLO1 is at least one of the targets of gene amplification on 6p21.2 and may represent a useful target for therapy in cancers with GLO1 amplification.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2010</span>;<span class="volume">49</span>;<span class="number">8</span>;<span class="pagerange">711-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20544845" rel="external">20544845</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3398139" rel="external">3398139</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20784" rel="external">10.1002/gcc.20784</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A census of amplified and overexpressed human cancer genes.</h4>
<p class="authors"><span class="author">Santarius T</span>, <span class="author">Shipley J</span>, <span class="author">Brewer D</span>, <span class="author">Stratton MR</span> and <span class="author">Cooper CS</span></p>
<p class="affiliation">The Wellcome Trust Sanger Centre, Wellcome Trust Genome Campus Hinxton, Cambridge, UK.</p><p class="abstract">Integrated genome-wide screens of DNA copy number and gene expression in human cancers have accelerated the rate of discovery of amplified and overexpressed genes. However, the biological importance of most of the genes identified in such studies remains unclear. In this Analysis, we propose a weight-of-evidence based classification system for identifying individual genes in amplified regions that are selected for during tumour development. In a census of the published literature we have identified 77 genes for which there is good evidence of involvement in the development of human cancer.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature reviews. Cancer</span>&nbsp;<span class="year">2010</span>;<span class="volume">10</span>;<span class="number">1</span>;<span class="pagerange">59-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20029424" rel="external">20029424</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrc2771" rel="external">10.1038/nrc2771</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Haptoglobin and sickle cell polymorphisms and risk of active trachoma in Gambian children.</h4>
<p class="authors"><span class="author">Savy M</span>, <span class="author">Hennig BJ</span>, <span class="author">Doherty CP</span>, <span class="author">Fulford AJ</span>, <span class="author">Bailey R</span>, <span class="author">Holland MJ</span>, <span class="author">Sirugo G</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">Prentice AM</span> and <span class="author">Cox SE</span></p>
<p class="affiliation">MRC International Nutrition Group, London School of Hygiene &amp; Tropical Medicine, London, UK.</p><p class="abstract"><strong>Background</strong>: Susceptibility and resistance to trachoma, the leading infectious cause of blindness, have been associated with a range of host genetic factors. In vitro studies of the causative organism, Chlamydia trachomatis, demonstrate that iron availability regulates its growth, suggesting that host genes involved in regulating iron status and/or availability may modulate the risk of trachoma. The objective was to investigate whether haptoglobin (Hp) haplotypes constructed from the functional polymorphism (Hp1/Hp2) plus the functional promoter SNPs -61A-C (rs5471) and -101C-G (rs5470), or sickle cell trait (HbAS, rs334) were associated with risk of active trachoma when stratified by age and sex, in rural Gambian children.</p><p class="abstract">In two cross sectional surveys of children aged 6-78 months (n = 836), the prevalence of the clinical signs of active trachoma was 21.4%. Within boys, haplotype E (-101G, -61A, Hp1), containing the variant allele of the -101C-G promoter SNP, was associated with a two-fold increased risk of active trachoma (OR = 2.0 [1.17-3.44]). Within girls, an opposite association was non-significant (OR = 0.58 [0.32-1.04]; P = 0.07) and the interaction by sex was statistically significant (P = 0.001). There was no association between trachoma and HbAS.</p><p class="abstract"><strong>Conclusions</strong>: These data indicate that genetic variation in Hp may affect susceptibility to active trachoma differentially by sex in The Gambia.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e11075</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20552021" rel="external">20552021</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2884021" rel="external">2884021</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0011075" rel="external">10.1371/journal.pone.0011075</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variation in GIPR influences the glucose and insulin responses to an oral glucose challenge.</h4>
<p class="authors"><span class="author">Saxena R</span>, <span class="author">Hivert MF</span>, <span class="author">Langenberg C</span>, <span class="author">Tanaka T</span>, <span class="author">Pankow JS</span>, <span class="author">Vollenweider P</span>, <span class="author">Lyssenko V</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Dupuis J</span>, <span class="author">Jackson AU</span>, <span class="author">Kao WH</span>, <span class="author">Li M</span>, <span class="author">Glazer NL</span>, <span class="author">Manning AK</span>, <span class="author">Luan J</span>, <span class="author">Stringham HM</span>, <span class="author">Prokopenko I</span>, <span class="author">Johnson T</span>, <span class="author">Grarup N</span>, <span class="author">Boesgaard TW</span>, <span class="author">Lecoeur C</span>, <span class="author">Shrader P</span>, <span class="author">O&#39;Connell J</span>, <span class="author">Ingelsson E</span>, <span class="author">Couper DJ</span>, <span class="author">Rice K</span>, <span class="author">Song K</span>, <span class="author">Andreasen CH</span>, <span class="author">Dina C</span>, <span class="author">K&ouml;ttgen A</span>, <span class="author">Le Bacquer O</span>, <span class="author">Pattou F</span>, <span class="author">Taneera J</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Rybin D</span>, <span class="author">Ardlie K</span>, <span class="author">Sampson M</span>, <span class="author">Qi L</span>, <span class="author">van Hoek M</span>, <span class="author">Weedon MN</span>, <span class="author">Aulchenko YS</span>, <span class="author">Voight BF</span>, <span class="author">Grallert H</span>, <span class="author">Balkau B</span>, <span class="author">Bergman RN</span>, <span class="author">Bielinski SJ</span>, <span class="author">Bonnefond A</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Borch-Johnsen K</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Brunner E</span>, <span class="author">Buchanan TA</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Charpentier G</span>, <span class="author">Chen YD</span>, <span class="author">Chines PS</span>, <span class="author">Collins FS</span>, <span class="author">Cornelis M</span>, <span class="author">J Crawford G</span>, <span class="author">Delplanque J</span>, <span class="author">Doney A</span>, <span class="author">Egan JM</span>, <span class="author">Erdos MR</span>, <span class="author">Firmann M</span>, <span class="author">Forouhi NG</span>, <span class="author">Fox CS</span>, <span class="author">Goodarzi MO</span>, <span class="author">Graessler J</span>, <span class="author">Hingorani A</span>, <span class="author">Isomaa B</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Kivimaki M</span>, <span class="author">Kovacs P</span>, <span class="author">Krohn K</span>, <span class="author">Kumari M</span>, <span class="author">Lauritzen T</span>, <span class="author">L&eacute;vy-Marchal C</span>, <span class="author">Mayor V</span>, <span class="author">McAteer JB</span>, <span class="author">Meyre D</span>, <span class="author">Mitchell BD</span>, <span class="author">Mohlke KL</span>, <span class="author">Morken MA</span>, <span class="author">Narisu N</span>, <span class="author">Palmer CN</span>, <span class="author">Pakyz R</span>, <span class="author">Pascoe L</span>, <span class="author">Payne F</span>, <span class="author">Pearson D</span>, <span class="author">Rathmann W</span>, <span class="author">Sandbaek A</span>, <span class="author">Sayer AA</span>, <span class="author">Scott LJ</span>, <span class="author">Sharp SJ</span>, <span class="author">Sijbrands E</span>, <span class="author">Singleton A</span>, <span class="author">Siscovick DS</span>, <span class="author">Smith NL</span>, <span class="author">Spars&oslash; T</span>, <span class="author">Swift AJ</span>, <span class="author">Syddall H</span>, <span class="author">Thorleifsson G</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Tuomi T</span>, <span class="author">Tuomilehto J</span>, <span class="author">Valle TT</span>, <span class="author">Waeber G</span>, <span class="author">Walley A</span>, <span class="author">Waterworth DM</span>, <span class="author">Zeggini E</span>, <span class="author">Zhao JH</span>, <span class="author">GIANT consortium</span>, <span class="author">MAGIC investigators</span>, <span class="author">Illig T</span>, <span class="author">Wichmann HE</span>, <span class="author">Wilson JF</span>, <span class="author">van Duijn C</span>, <span class="author">Hu FB</span>, <span class="author">Morris AD</span>, <span class="author">Frayling TM</span>, <span class="author">Hattersley AT</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Stefansson K</span>, <span class="author">Nilsson P</span>, <span class="author">Syv&auml;nen AC</span>, <span class="author">Shuldiner AR</span>, <span class="author">Walker M</span>, <span class="author">Bornstein SR</span>, <span class="author">Schwarz P</span>, <span class="author">Williams GH</span>, <span class="author">Nathan DM</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Cooper C</span>, <span class="author">Marmot M</span>, <span class="author">Ferrucci L</span>, <span class="author">Mooser V</span>, <span class="author">Stumvoll M</span>, <span class="author">Loos RJ</span>, <span class="author">Altshuler D</span>, <span class="author">Psaty BM</span>, <span class="author">Rotter JI</span>, <span class="author">Boerwinkle E</span>, <span class="author">Hansen T</span>, <span class="author">Pedersen O</span>, <span class="author">Florez JC</span>, <span class="author">McCarthy MI</span>, <span class="author">Boehnke M</span>, <span class="author">Barroso I</span>, <span class="author">Sladek R</span>, <span class="author">Froguel P</span>, <span class="author">Meigs JB</span>, <span class="author">Groop L</span>, <span class="author">Wareham NJ</span> and <span class="author">Watanabe RM</span></p>
<p class="affiliation">Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA.</p><p class="abstract">Glucose levels 2 h after an oral glucose challenge are a clinical measure of glucose tolerance used in the diagnosis of type 2 diabetes. We report a meta-analysis of nine genome-wide association studies (n = 15,234 nondiabetic individuals) and a follow-up of 29 independent loci (n = 6,958-30,620). We identify variants at the GIPR locus associated with 2-h glucose level (rs10423928, beta (s.e.m.) = 0.09 (0.01) mmol/l per A allele, P = 2.0 x 10(-15)). The GIPR A-allele carriers also showed decreased insulin secretion (n = 22,492; insulinogenic index, P = 1.0 x 10(-17); ratio of insulin to glucose area under the curve, P = 1.3 x 10(-16)) and diminished incretin effect (n = 804; P = 4.3 x 10(-4)). We also identified variants at ADCY5 (rs2877716, P = 4.2 x 10(-16)), VPS13C (rs17271305, P = 4.1 x 10(-8)), GCKR (rs1260326, P = 7.1 x 10(-11)) and TCF7L2 (rs7903146, P = 4.2 x 10(-10)) associated with 2-h glucose. Of the three newly implicated loci (GIPR, ADCY5 and VPS13C), only ADCY5 was found to be associated with type 2 diabetes in collaborating studies (n = 35,869 cases, 89,798 controls, OR = 1.12, 95% CI 1.09-1.15, P = 4.8 x 10(-18)).</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0701863</span>, <span class="grant">G19/35</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">MC_UP_A620_1015</span>; NCI NIH HHS: <span class="grant">P01 CA087969</span>, <span class="grant">P01 CA087969-12</span>; NCRR NIH HHS: <span class="grant">M01 RR000052</span>, <span class="grant">M01 RR000052-46</span>, <span class="grant">M01 RR001066-26</span>, <span class="grant">M01 RR016500-08</span>; NHGRI NIH HHS: <span class="grant">U01 HG004399</span>, <span class="grant">U01 HG004399-02</span>, <span class="grant">U01 HG004402</span>, <span class="grant">U01 HG004402-02</span>, <span class="grant">Z01 HG000024-14</span>; NHLBI NIH HHS: <span class="grant">N01 HC015103</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC035129</span>, <span class="grant">N01 HC045133</span>, <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC055016</span>, <span class="grant">N01 HC055018</span>, <span class="grant">N01 HC055019</span>, <span class="grant">N01 HC055020</span>, <span class="grant">N01 HC055021</span>, <span class="grant">N01 HC055022</span>, <span class="grant">N01 HC055222</span>, <span class="grant">N01 HC075150</span>, <span class="grant">N01 HC085079</span>, <span class="grant">N01 HC085080</span>, <span class="grant">N01 HC085081</span>, <span class="grant">N01 HC085082</span>, <span class="grant">N01 HC085083</span>, <span class="grant">N01 HC085084</span>, <span class="grant">N01 HC085085</span>, <span class="grant">N01 HC085086</span>, <span class="grant">N02 HL64278</span>, <span class="grant">R01 HL036310</span>, <span class="grant">R01 HL036310-21</span>, <span class="grant">R01 HL059367</span>, <span class="grant">R01 HL059367-10</span>, <span class="grant">R01 HL086694</span>, <span class="grant">R01 HL086694-03</span>, <span class="grant">R01 HL087641</span>, <span class="grant">R01 HL087641-03</span>, <span class="grant">R01 HL087652</span>, <span class="grant">R01 HL087652-03</span>, <span class="grant">U01 HL072515</span>, <span class="grant">U01 HL072515-06</span>, <span class="grant">U01 HL080295</span>, <span class="grant">U01 HL080295-04</span>; NIA NIH HHS: <span class="grant">R01 AG013196</span>, <span class="grant">R01 AG013196-16</span>; NIDA NIH HHS: <span class="grant">U54 DA021519</span>, <span class="grant">U54 DA021519-04</span>; NIDDK NIH HHS: <span class="grant">K23 DK065978</span>, <span class="grant">K23 DK065978-05</span>, <span class="grant">K24 DK080140</span>, <span class="grant">K24 DK080140-04</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30 DK072488-06</span>, <span class="grant">P30 DK079637</span>, <span class="grant">P60 DK079637</span>, <span class="grant">P60 DK079637-04</span>, <span class="grant">R01 DK029867</span>, <span class="grant">R01 DK054261</span>, <span class="grant">R01 DK054261-09</span>, <span class="grant">R01 DK058845</span>, <span class="grant">R01 DK058845-11</span>, <span class="grant">R01 DK062370</span>, <span class="grant">R01 DK062370-05</span>, <span class="grant">R01 DK069922</span>, <span class="grant">R01 DK069922-03</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK072193-04</span>, <span class="grant">R01 DK078616</span>, <span class="grant">R01 DK078616-03</span>, <span class="grant">R01 DK091718</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">088885</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">2</span>;<span class="pagerange">142-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20081857">20081857</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2922003">2922003</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.521">10.1038/ng.521</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human deafness-associated connexin 30 T5M mutation causes mild hearing loss and reduces biochemical coupling among cochlear non-sensory cells in knock-in mice.</h4>
<p class="authors"><span class="author">Sch&uuml;tz M</span>, <span class="author">Scimemi P</span>, <span class="author">Majumder P</span>, <span class="author">De Siati RD</span>, <span class="author">Crispino G</span>, <span class="author">Rodriguez L</span>, <span class="author">Bortolozzi M</span>, <span class="author">Santarelli R</span>, <span class="author">Seydel A</span>, <span class="author">Sonntag S</span>, <span class="author">Ingham N</span>, <span class="author">Steel KP</span>, <span class="author">Willecke K</span> and <span class="author">Mammano F</span></p>
<p class="affiliation">Institut fuer Genetik, Rheinische Friedrich-Wilhelms-Universitaet Bonn, Roemerstrasse 164, D-53117 Bonn, Germany.</p><p class="abstract">Mutations in the GJB2 and GJB6 genes, respectively, coding for connexin26 (Cx26) and connexin30 (Cx30) proteins, are the most common cause for prelingual non-syndromic deafness in humans. In the inner ear, Cx26 and Cx30 are expressed in different non-sensory cell types, where they largely co-localize and may form heteromeric gap junction channels. Here, we describe the generation and characterization of a mouse model for human bilateral middle/high-frequency hearing loss based on the substitution of an evolutionarily conserved threonine by a methionine residue at position 5 near the N-terminus of Cx30 (Cx30T5M). The mutation was inserted in the mouse genome by homologous recombination in mouse embryonic stem cells. Expression of the mutated Cx30T5M protein in these transgenic mice is under the control of the endogenous Cx30 promoter and was analysed via activation of the lacZ reporter gene. When probed by auditory brainstem recordings, Cx30(T5M/T5M) mice exhibited a mild, but significant increase in their hearing thresholds of about 15 dB at all frequencies. Immunolabelling with antibodies to Cx26 or Cx30 suggested normal location of these proteins in the adult inner ear, but western blot analysis showed significantly down-regulated the expression levels of Cx26 and Cx30. In the developing cochlea, electrical coupling, probed by dual patch-clamp recordings, was normal. However, transfer of the fluorescent tracer calcein between cochlear non-sensory cells was reduced, as was intercellular Ca(2+) signalling due to spontaneous ATP release from connexin hemichannels. Our findings link hearing loss to decreased biochemical coupling due to the point-mutated Cx30 in mice.</p>
<p class="grants">Funded by: Telethon: <span class="grant">GGP09137</span>; Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">24</span>;<span class="pagerange">4759-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20858605" rel="external">20858605</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2989887" rel="external">2989887</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq402" rel="external">10.1093/hmg/ddq402</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neuroligin 1 is dynamically exchanged at postsynaptic sites.</h4>
<p class="authors"><span class="author">Schapitz IU</span>, <span class="author">Behrend B</span>, <span class="author">Pechmann Y</span>, <span class="author">Lappe-Siefke C</span>, <span class="author">Kneussel SJ</span>, <span class="author">Wallace KE</span>, <span class="author">Stempel AV</span>, <span class="author">Buck F</span>, <span class="author">Grant SG</span>, <span class="author">Schweizer M</span>, <span class="author">Schmitz D</span>, <span class="author">Schwarz JR</span>, <span class="author">Holzbaur EL</span> and <span class="author">Kneussel M</span></p>
<p class="affiliation">Center for Molecular Neurobiology (ZMNH), University of Hamburg Medical School, D-20251 Hamburg, Germany.</p><p class="abstract">Neuroligins are postsynaptic cell adhesion molecules that associate with presynaptic neurexins. Both factors form a transsynaptic connection, mediate signaling across the synapse, specify synaptic functions, and play a role in synapse formation. Neuroligin dysfunction impairs synaptic transmission, disrupts neuronal networks, and is thought to participate in cognitive diseases. Here we report that chemical treatment designed to induce long-term potentiation or long-term depression (LTD) induces neuroligin 1/3 turnover, leading to either increased or decreased surface membrane protein levels, respectively. Despite its structural role at a crucial transsynaptic position, GFP-neuroligin 1 leaves synapses in hippocampal neurons over time with chemical LTD-induced neuroligin internalization depending on an intact microtubule cytoskeleton. Accordingly, neuroligin 1 and its binding partner postsynaptic density protein-95 (PSD-95) associate with components of the dynein motor complex and undergo retrograde cotransport with a dynein subunit. Transgenic depletion of dynein function in mice causes postsynaptic NLG1/3 and PSD-95 enrichment. In parallel, PSD lengths and spine head sizes are significantly increased, a phenotype similar to that observed upon transgenic overexpression of NLG1 (Dahlhaus et al., 2010). Moreover, application of a competitive PSD-95 peptide and neuroligin 1 C-terminal mutagenesis each specifically alter neuroligin 1 surface membrane expression and interfere with its internalization. Our data suggest the concept that synaptic plasticity regulates neuroligin turnover through active cytoskeleton transport.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">NS060698</span>, <span class="grant">R01 NS060698-04</span></p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">38</span>;<span class="pagerange">12733-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20861378" rel="external">20861378</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3108891" rel="external">3108891</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.0896-10.2010" rel="external">10.1523/JNEUROSCI.0896-10.2010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Five-vertebrate ChIP-seq reveals the evolutionary dynamics of transcription factor binding.</h4>
<p class="authors"><span class="author">Schmidt D</span>, <span class="author">Wilson MD</span>, <span class="author">Ballester B</span>, <span class="author">Schwalie PC</span>, <span class="author">Brown GD</span>, <span class="author">Marshall A</span>, <span class="author">Kutter C</span>, <span class="author">Watt S</span>, <span class="author">Martinez-Jimenez CP</span>, <span class="author">Mackay S</span>, <span class="author">Talianidis I</span>, <span class="author">Flicek P</span> and <span class="author">Odom DT</span></p>
<p class="affiliation">Cancer Research UK, Cambridge Research Institute, Li Ka Shing Centre, Robinson Way, Cambridge CB2 0RE, UK.</p><p class="abstract">Transcription factors (TFs) direct gene expression by binding to DNA regulatory regions. To explore the evolution of gene regulation, we used chromatin immunoprecipitation with high-throughput sequencing (ChIP-seq) to determine experimentally the genome-wide occupancy of two TFs, CCAAT/enhancer-binding protein alpha and hepatocyte nuclear factor 4 alpha, in the livers of five vertebrates. Although each TF displays highly conserved DNA binding preferences, most binding is species-specific, and aligned binding events present in all five species are rare. Regions near genes with expression levels that are dependent on a TF are often bound by the TF in multiple species yet show no enhanced DNA sequence constraint. Binding divergence between species can be largely explained by sequence changes to the bound motifs. Among the binding events lost in one lineage, only half are recovered by another binding event within 10 kilobases. Our results reveal large interspecies differences in transcriptional regulation and provide insight into regulatory evolution.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">15603</span>, <span class="grant">A15603</span>; European Research Council: <span class="grant">202218</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">079643</span>, <span class="grant">WT062023</span>, <span class="grant">WT079643</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2010</span>;<span class="volume">328</span>;<span class="number">5981</span>;<span class="pagerange">1036-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20378774">20378774</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3008766">3008766</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1186176">10.1126/science.1186176</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CHD7 targets active gene enhancer elements to modulate ES cell-specific gene expression.</h4>
<p class="authors"><span class="author">Schnetz MP</span>, <span class="author">Handoko L</span>, <span class="author">Akhtar-Zaidi B</span>, <span class="author">Bartels CF</span>, <span class="author">Pereira CF</span>, <span class="author">Fisher AG</span>, <span class="author">Adams DJ</span>, <span class="author">Flicek P</span>, <span class="author">Crawford GE</span>, <span class="author">Laframboise T</span>, <span class="author">Tesar P</span>, <span class="author">Wei CL</span> and <span class="author">Scacheri PC</span></p>
<p class="affiliation">Department of Genetics, Case Western Reserve University, Cleveland, Ohio, United States of America.</p><p class="abstract">CHD7 is one of nine members of the chromodomain helicase DNA-binding domain family of ATP-dependent chromatin remodeling enzymes found in mammalian cells. De novo mutation of CHD7 is a major cause of CHARGE syndrome, a genetic condition characterized by multiple congenital anomalies. To gain insights to the function of CHD7, we used the technique of chromatin immunoprecipitation followed by massively parallel DNA sequencing (ChIP-Seq) to map CHD7 sites in mouse ES cells. We identified 10,483 sites on chromatin bound by CHD7 at high confidence. Most of the CHD7 sites show features of gene enhancer elements. Specifically, CHD7 sites are predominantly located distal to transcription start sites, contain high levels of H3K4 mono-methylation, found within open chromatin that is hypersensitive to DNase I digestion, and correlate with ES cell-specific gene expression. Moreover, CHD7 co-localizes with P300, a known enhancer-binding protein and strong predictor of enhancer activity. Correlations with 18 other factors mapped by ChIP-seq in mouse ES cells indicate that CHD7 also co-localizes with ES cell master regulators OCT4, SOX2, and NANOG. Correlations between CHD7 sites and global gene expression profiles obtained from Chd7(+/+), Chd7(+/-), and Chd7(-/-) ES cells indicate that CHD7 functions at enhancers as a transcriptional rheostat to modulate, or fine-tune the expression levels of ES-specific genes. CHD7 can modulate genes in either the positive or negative direction, although negative regulation appears to be the more direct effect of CHD7 binding. These data indicate that enhancer-binding proteins can limit gene expression and are not necessarily co-activators. Although ES cells are not likely to be affected in CHARGE syndrome, we propose that enhancer-mediated gene dysregulation contributes to disease pathogenesis and that the critical CHD7 target genes may be subject to positive or negative regulation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800024</span>, <span class="grant">MC_U120027516</span>; NHGRI NIH HHS: <span class="grant">1U54HG004557-01</span>, <span class="grant">R01 HG004456-01</span>, <span class="grant">R01HG003521-01</span>, <span class="grant">R01HG004722</span>; NICHD NIH HHS: <span class="grant">R01HD056369</span>, <span class="grant">T32 HD007104</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">e1001023</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20657823">20657823</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2904778">2904778</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001023">10.1371/journal.pgen.1001023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An investigation of clinical and immunological events following repeated aerodigestive tract challenge infections with live Mycobacterium bovis Bacille Calmette Gu&eacute;rin.</h4>
<p class="authors"><span class="author">Schreiber F</span>, <span class="author">Huo Z</span>, <span class="author">Giemza R</span>, <span class="author">Woodrow M</span>, <span class="author">Fenner N</span>, <span class="author">Stephens Z</span>, <span class="author">Dougan G</span>, <span class="author">Prideaux S</span>, <span class="author">Castello-Branco LR</span> and <span class="author">Lewis DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom. <a href="mailto:%66%73%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#115;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Bacille Calmette Gu&eacute;rin substrain Moreau Rio de Janeiro is an attenuated strain of Mycobacterium bovis that has been used extensively as an oral tuberculosis vaccine. We assessed its potential as a challenge model to study clinical and immunological events following repeated mycobacterial gut infection. Seven individuals received three oral challenges with approximately 10(7) viable bacilli. Clinical symptoms, T-cell responses and gene expression patterns in peripheral blood were monitored. Clinical symptoms were relatively mild and declined following each oral challenge. Delayed T-cell responses were observed, and limited differential gene expression detected by microarrays. Oral challenge with BCG Moreau Rio de Janeiro vaccine was immunogenic in healthy volunteers, limiting its potential to explore clinical innate immune responses, but with low reactogenicity.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Vaccine</span>&nbsp;<span class="year">2010</span>;<span class="volume">28</span>;<span class="number">33</span>;<span class="pagerange">5427-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20558246">20558246</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3764428">3764428</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vaccine.2010.06.005">10.1016/j.vaccine.2010.06.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Legionella pneumophila strain 130b possesses a unique combination of type IV secretion systems and novel Dot/Icm secretion system effector proteins.</h4>
<p class="authors"><span class="author">Schroeder GN</span>, <span class="author">Petty NK</span>, <span class="author">Mousnier A</span>, <span class="author">Harding CR</span>, <span class="author">Vogrin AJ</span>, <span class="author">Wee B</span>, <span class="author">Fry NK</span>, <span class="author">Harrison TG</span>, <span class="author">Newton HJ</span>, <span class="author">Thomson NR</span>, <span class="author">Beatson SA</span>, <span class="author">Dougan G</span>, <span class="author">Hartland EL</span> and <span class="author">Frankel G</span></p>
<p class="affiliation">Centre for Molecular Microbiology and Infection, Division of Cell and Molecular Biology, Imperial College, London, United Kingdom.</p><p class="abstract">Legionella pneumophila is a ubiquitous inhabitant of environmental water reservoirs. The bacteria infect a wide variety of protozoa and, after accidental inhalation, human alveolar macrophages, which can lead to severe pneumonia. The capability to thrive in phagocytic hosts is dependent on the Dot/Icm type IV secretion system (T4SS), which translocates multiple effector proteins into the host cell. In this study, we determined the draft genome sequence of L. pneumophila strain 130b (Wadsworth). We found that the 130b genome encodes a unique set of T4SSs, namely, the Dot/Icm T4SS, a Trb-1-like T4SS, and two Lvh T4SS gene clusters. Sequence analysis substantiated that a core set of 107 Dot/Icm T4SS effectors was conserved among the sequenced L. pneumophila strains Philadelphia-1, Lens, Paris, Corby, Alcoy, and 130b. We also identified new effector candidates and validated the translocation of 10 novel Dot/Icm T4SS effectors that are not present in L. pneumophila strain Philadelphia-1. We examined the prevalence of the new effector genes among 87 environmental and clinical L. pneumophila isolates. Five of the new effectors were identified in 34 to 62% of the isolates, while less than 15% of the strains tested positive for the other five genes. Collectively, our data show that the core set of conserved Dot/Icm T4SS effector proteins is supplemented by a variable repertoire of accessory effectors that may partly account for differences in the virulences and prevalences of particular L. pneumophila strains.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700823</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">22</span>;<span class="pagerange">6001-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20833813">20833813</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2976443">2976443</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00778-10">10.1128/JB.00778-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Natural history of Christianson syndrome.</h4>
<p class="authors"><span class="author">Schroer RJ</span>, <span class="author">Holden KR</span>, <span class="author">Tarpey PS</span>, <span class="author">Matheus MG</span>, <span class="author">Griesemer DA</span>, <span class="author">Friez MJ</span>, <span class="author">Fan JZ</span>, <span class="author">Simensen RJ</span>, <span class="author">Str&oslash;mme P</span>, <span class="author">Stevenson RE</span>, <span class="author">Stratton MR</span> and <span class="author">Schwartz CE</span></p>
<p class="affiliation">Greenwood Genetic Center, Greenwood, South Carolina, USA. <a href="mailto:%73%63%68%72%6F%65%72%40%67%67%63.%6F%72%67">&#115;&#99;&#104;&#114;&#111;&#101;&#114;&#64;&#103;&#103;&#99;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Christianson syndrome is an X-linked mental retardation syndrome characterized by microcephaly, impaired ocular movement, severe global developmental delay, hypotonia which progresses to spasticity, and early onset seizures of variable types. Gilfillan et al.2008] reported mutations in SLC9A6, the gene encoding the sodium/hydrogen exchanger NHE6, in the family first reported and in three others. They also noted the clinical similarities to Angelman syndrome and found cerebellar atrophy on MRI and elevated glutamate/glutamine in the basal ganglia on MRS. Here we report on nonsense mutations in two additional families. The natural history is detailed in childhood and adult life, the similarities to Angelman syndrome confirmed, and the MRI/MRS findings documented in three affected boys.</p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2010</span>;<span class="volume">152A</span>;<span class="number">11</span>;<span class="pagerange">2775-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20949524">20949524</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.33093">10.1002/ajmg.a.33093</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dosage sensitivity shapes the evolution of copy-number varied regions.</h4>
<p class="authors"><span class="author">Schuster-B&ouml;ckler B</span>, <span class="author">Conrad D</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">European Molecular Biology Laboratory-Centre for Genomic Regulation Systems Biology Unit, Centre for Genomic Regulation, Barcelona, Spain.</p><p class="abstract">Dosage sensitivity is an important evolutionary force which impacts on gene dispensability and duplicability. The newly available data on human copy-number variation (CNV) allow an analysis of the most recent and ongoing evolution. Provided that heterozygous gene deletions and duplications actually change gene dosage, we expect to observe negative selection against CNVs encompassing dosage sensitive genes. In this study, we make use of several sources of population genetic data to identify selection on structural variations of dosage sensitive genes. We show that CNVs can directly affect expression levels of contained genes. We find that genes encoding members of protein complexes exhibit limited expression variation and overlap significantly with a manually derived set of dosage sensitive genes. We show that complexes and other dosage sensitive genes are underrepresented in CNV regions, with a particular bias against frequent variations and duplications. These results suggest that dosage sensitivity is a significant force of negative selection on regions of copy-number variation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">e9474</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20224824" rel="external">20224824</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2835737" rel="external">2835737</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0009474" rel="external">10.1371/journal.pone.0009474</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence of the plant pathogen Erwinia amylovora strain ATCC 49946.</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Bocsanczy AM</span>, <span class="author">Biehl BS</span>, <span class="author">Quail MA</span>, <span class="author">Perna NT</span>, <span class="author">Glasner JD</span>, <span class="author">DeClerck GA</span>, <span class="author">Cartinhour S</span>, <span class="author">Schneider DJ</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span> and <span class="author">Beer SV</span></p>
<p class="affiliation">The Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Erwinia amylovora causes the economically important disease fire blight that affects rosaceous plants, especially pear and apple. Here we report the complete genome sequence and annotation of strain ATCC 49946. The analysis of the sequence and its comparison with sequenced genomes of closely related enterobacteria revealed signs of pathoadaptation to rosaceous hosts.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2010</span>;<span class="volume">192</span>;<span class="number">7</span>;<span class="pagerange">2020-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20118253">20118253</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2838050">2838050</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00022-10">10.1128/JB.00022-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common inherited variation in mitochondrial genes is not enriched for associations with type 2 diabetes or related glycemic traits.</h4>
<p class="authors"><span class="author">Segr&egrave; AV</span>, <span class="author">DIAGRAM Consortium</span>, <span class="author">MAGIC investigators</span>, <span class="author">Groop L</span>, <span class="author">Mootha VK</span>, <span class="author">Daly MJ</span> and <span class="author">Altshuler D</span></p>
<p class="affiliation">Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America.</p><p class="abstract">Mitochondrial dysfunction has been observed in skeletal muscle of people with diabetes and insulin-resistant individuals. Furthermore, inherited mutations in mitochondrial DNA can cause a rare form of diabetes. However, it is unclear whether mitochondrial dysfunction is a primary cause of the common form of diabetes. To date, common genetic variants robustly associated with type 2 diabetes (T2D) are not known to affect mitochondrial function. One possibility is that multiple mitochondrial genes contain modest genetic effects that collectively influence T2D risk. To test this hypothesis we developed a method named Meta-Analysis Gene-set Enrichment of variaNT Associations (MAGENTA; http://www.broadinstitute.org/mpg/magenta). MAGENTA, in analogy to Gene Set Enrichment Analysis, tests whether sets of functionally related genes are enriched for associations with a polygenic disease or trait. MAGENTA was specifically designed to exploit the statistical power of large genome-wide association (GWA) study meta-analyses whose individual genotypes are not available. This is achieved by combining variant association p-values into gene scores and then correcting for confounders, such as gene size, variant number, and linkage disequilibrium properties. Using simulations, we determined the range of parameters for which MAGENTA can detect associations likely missed by single-marker analysis. We verified MAGENTA&#39;s performance on empirical data by identifying known relevant pathways in lipid and lipoprotein GWA meta-analyses. We then tested our mitochondrial hypothesis by applying MAGENTA to three gene sets: nuclear regulators of mitochondrial genes, oxidative phosphorylation genes, and approximately 1,000 nuclear-encoded mitochondrial genes. The analysis was performed using the most recent T2D GWA meta-analysis of 47,117 people and meta-analyses of seven diabetes-related glycemic traits (up to 46,186 non-diabetic individuals). This well-powered analysis found no significant enrichment of associations to T2D or any of the glycemic traits in any of the gene sets tested. These results suggest that common variants affecting nuclear-encoded mitochondrial genes have at most a small genetic contribution to T2D susceptibility.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20714348">20714348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2920848">2920848</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001058">10.1371/journal.pgen.1001058</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&#39;Slick&#39; operation.</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">538</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20634808" rel="external">20634808</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2407" rel="external">10.1038/nrmicro2407</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic factors in esophageal atresia, tracheo-esophageal fistula and the VACTERL association: roles for FOXF1 and the 16q24.1 FOX transcription factor gene cluster, and review of the literature.</h4>
<p class="authors"><span class="author">Shaw-Smith C</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%63%73%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#115;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Esophageal atresia with/without tracheo-esophageal fistula is a relatively common malformation, occurring in around 1 in 3500 births. In around half of cases, additional malformations are present, forming either a syndrome of known genetic aetiology, or a recognised association, of which the VACTERL association (Vertebral anomalies, Anal atresia, Cardiac malformations, Tracheo-Esophageal fistula, Renal and Limb malformations) is the most recognised. Recently, microdeletions of the FOX gene cluster at 16q24.1, comprising four genes, FOXF1, MTHFSD, FOXC2 and FOXL1, were reported to cause a phenotype resembling VACTERL association, with vertebral anomalies, gastro-intestinal atresias (esophageal, duodenal and anal), congenital heart malformations, and urinary tract malformations, as well as a rare lethal developmental anomaly of the lung, alveolar capillary dysplasia. This article reviews these new data alongside other genetic causes of syndromic esophageal atresia, and also highlights information from relevant mouse models, particularly those for genes in the Sonic Hedgehog pathway.</p>
<p class="publication"><span class="title">European journal of medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">53</span>;<span class="number">1</span>;<span class="pagerange">6-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19822228" rel="external">19822228</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2809919" rel="external">2809919</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ejmg.2009.10.001" rel="external">10.1016/j.ejmg.2009.10.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A worldwide survey of human male demographic history based on Y-SNP and Y-STR data from the HGDP-CEPH populations.</h4>
<p class="authors"><span class="author">Shi W</span>, <span class="author">Ayub Q</span>, <span class="author">Vermeulen M</span>, <span class="author">Shao RG</span>, <span class="author">Zuniga S</span>, <span class="author">van der Gaag K</span>, <span class="author">de Knijff P</span>, <span class="author">Kayser M</span>, <span class="author">Xue Y</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambs., United Kingdom.</p><p class="abstract">We have investigated human male demographic history using 590 males from 51 populations in the Human Genome Diversity Project - Centre d&#39;Etude du Polymorphisme Humain worldwide panel, typed with 37 Y-chromosomal Single Nucleotide Polymorphisms and 65 Y-chromosomal Short Tandem Repeats and analyzed with the program Bayesian Analysis of Trees With Internal Node Generation. The general patterns we observe show a gradient from the oldest population time to the most recent common ancestors (TMRCAs) and expansion times together with the largest effective population sizes in Africa, to the youngest times and smallest effective population sizes in the Americas. These parameters are significantly negatively correlated with distance from East Africa, and the patterns are consistent with most other studies of human variation and history. In contrast, growth rate showed a weaker correlation in the opposite direction. Y-lineage diversity and TMRCA also decrease with distance from East Africa, supporting a model of expansion with serial founder events starting from this source. A number of individual populations diverge from these general patterns, including previously documented examples such as recent expansions of the Yoruba in Africa, Basques in Europe, and Yakut in Northern Asia. However, some unexpected demographic histories were also found, including low growth rates in the Hazara and Kalash from Pakistan and recent expansion of the Mozabites in North Africa.</p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2010</span>;<span class="volume">27</span>;<span class="number">2</span>;<span class="pagerange">385-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19822636">19822636</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2806244">2806244</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsp243">10.1093/molbev/msp243</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the guanine nucleotide exchange factor gene IQSEC2 cause nonsyndromic intellectual disability.</h4>
<p class="authors"><span class="author">Shoubridge C</span>, <span class="author">Tarpey PS</span>, <span class="author">Abidi F</span>, <span class="author">Ramsden SL</span>, <span class="author">Rujirabanjerd S</span>, <span class="author">Murphy JA</span>, <span class="author">Boyle J</span>, <span class="author">Shaw M</span>, <span class="author">Gardner A</span>, <span class="author">Proos A</span>, <span class="author">Puusepp H</span>, <span class="author">Raymond FL</span>, <span class="author">Schwartz CE</span>, <span class="author">Stevenson RE</span>, <span class="author">Turner G</span>, <span class="author">Field M</span>, <span class="author">Walikonis RS</span>, <span class="author">Harvey RJ</span>, <span class="author">Hackett A</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span> and <span class="author">G&eacute;cz J</span></p>
<p class="affiliation">Genetics and Molecular Pathology, SA Pathology, Adelaide, Australia. <a href="mailto:%63%68%65%72%79%6C.%73%68%6F%75%62%72%69%64%67%65%40%61%64%65%6C%61%69%64%65.%65%64%75.%61%75">&#99;&#104;&#101;&#114;&#121;&#108;&#46;&#115;&#104;&#111;&#117;&#98;&#114;&#105;&#100;&#103;&#101;&#64;&#97;&#100;&#101;&#108;&#97;&#105;&#100;&#101;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">The first family identified as having a nonsyndromic intellectual disability was mapped in 1988. Here we show that a mutation of IQSEC2, encoding a guanine nucleotide exchange factor for the ADP-ribosylation factor family of small GTPases, caused this disorder. In addition to MRX1, IQSEC2 mutations were identified in three other families with X-linked intellectual disability. This discovery was made possible by systematic and unbiased X chromosome exome resequencing.</p>
<p class="grants">Funded by: Medical Research Council; NICHD NIH HHS: <span class="grant">HD26202</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">6</span>;<span class="pagerange">486-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20473311">20473311</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3632837">3632837</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.588">10.1038/ng.588</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of Fc receptor-like 5 (FCRL5) with Graves&#39; disease is secondary to the effect of FCRL3.</h4>
<p class="authors"><span class="author">Simmonds MJ</span>, <span class="author">Brand OJ</span>, <span class="author">Barrett JC</span>, <span class="author">Newby PR</span>, <span class="author">Franklyn JA</span> and <span class="author">Gough SC</span></p>
<p class="affiliation">Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Headington, Oxford, UK. <a href="mailto:%6D%61%74%74%68%65%77.%73%69%6D%6D%6F%6E%64%73%40%6F%63%64%65%6D.%6F%78.%61%63.%75%6B">&#109;&#97;&#116;&#116;&#104;&#101;&#119;&#46;&#115;&#105;&#109;&#109;&#111;&#110;&#100;&#115;&#64;&#111;&#99;&#100;&#101;&#109;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Objective</strong>: The Fc receptor-like 3 (FCRL3) molecule, involved in controlling B-cell signalling, may contribute to the autoimmune disease process. Recently, a genome-wide screen detected association of neighbouring gene FCRL5 with Graves&#39; disease (GD). To determine whether FCRL5 represents a further independent B-cell signalling GD susceptibility loci, we screened 12 tag SNPs, capturing all known common variation within FCRL5, in 5192 UK Caucasian GD index cases and controls.</p><p class="abstract"><strong>Design</strong>: A case-control association study investigating twelve tag SNPs within FCRL5 which captured the majority of known common variation within this gene region.</p><p class="abstract"><strong>Patients</strong>: A data set comprising 2504 UK Caucasian patients with GD and 2688 geographically matched controls taken from the 1958 British Birth cohort.</p><p class="abstract"><strong>Measurements</strong>: We used the chi-squared test and haplotype analysis to investigate the association between the tag SNPs and GD before performing logistic regression analysis to determine whether association at FCRL5 was independent of the known FCRL3 association.</p><p class="abstract"><strong>Results</strong>: Three of the FCRL5 tag SNPs, rs6667109, rs3811035 and rs6692977 showed association with GD (P = 0&middot;015-0&middot;001, OR = 1&middot;15-1&middot;16). Logistic regression performed on all FCRL5 and, previously screened, FCRL3 tag SNPs revealed that association with FCRL5 was secondary to linkage disequilibrium with the FCRL3, rs11264798 and rs10489678 SNPs.</p><p class="abstract"><strong>Conclusions</strong>: FCRL5 does not appear to be exerting an independent effect on the development of GD in the UK. Fine mapping of the entire FCRL region is required to determine the exact location of the aetiological variant/s present.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">089120</span></p>
<p class="publication"><span class="title">Clinical endocrinology</span>&nbsp;<span class="year">2010</span>;<span class="volume">73</span>;<span class="number">5</span>;<span class="pagerange">654-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20626413" rel="external">20626413</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3299549" rel="external">3299549</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2265.2010.03843.x" rel="external">10.1111/j.1365-2265.2010.03843.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficient construction of an assembly string graph using the FM-index.</h4>
<p class="authors"><span class="author">Simpson JT</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK. <a href="mailto:%6A%73%31%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#115;&#49;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">MOTIVATION: Sequence assembly is a difficult problem whose importance has grown again recently as the cost of sequencing has dramatically dropped. Most new sequence assembly software has started by building a de Bruijn graph, avoiding the overlap-based methods used previously because of the computational cost and complexity of these with very large numbers of short reads. Here, we show how to use suffix array-based methods that have formed the basis of recent very fast sequence mapping algorithms to find overlaps and generate assembly string graphs asymptotically faster than previously described algorithms. RESULTS: Standard overlap assembly methods have time complexity O(N(2)), where N is the sum of the lengths of the reads. We use the Ferragina-Manzini index (FM-index) derived from the Burrows-Wheeler transform to find overlaps of length at least tau among a set of reads. As well as an approach that finds all overlaps then implements transitive reduction to produce a string graph, we show how to output directly only the irreducible overlaps, significantly shrinking memory requirements and reducing compute time to O(N), independent of depth. Overlap-based assembly methods naturally handle mixed length read sets, including capillary reads or long reads promised by the third generation sequencing technologies. The algorithms we present here pave the way for overlap-based assembly approaches to be developed that scale to whole vertebrate genome de novo assembly.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077192</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">12</span>;<span class="pagerange">i367-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20529929" rel="external">20529929</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2881401" rel="external">2881401</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq217" rel="external">10.1093/bioinformatics/btq217</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Copy number variant detection in inbred strains from short read sequence data.</h4>
<p class="authors"><span class="author">Simpson JT</span>, <span class="author">McIntyre RE</span>, <span class="author">Adams DJ</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, CB10 1HH, UK.</p><p class="abstract"><strong>Summary</strong>: We have developed an algorithm to detect copy number variants (CNVs) in homozygous organisms, such as inbred laboratory strains of mice, from short read sequence data. Our novel approach exploits the fact that inbred mice are homozygous at virtually every position in the genome to detect CNVs using a hidden Markov model (HMM). This HMM uses both the density of sequence reads mapped to the genome, and the rate of apparent heterozygous single nucleotide polymorphisms, to determine genomic copy number. We tested our algorithm on short read sequence data generated from re-sequencing chromosome 17 of the mouse strains A/J and CAST/EiJ with the Illumina platform. In total, we identified 118 copy number variants (43 for A/J and 75 for CAST/EiJ). We investigated the performance of our algorithm through comparison to CNVs previously identified by array-comparative genomic hybridization (array CGH). We performed quantitative-PCR validation on a subset of the calls that differed from the array CGH data sets.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">4</span>;<span class="pagerange">565-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20022973">20022973</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2820678">2820678</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp693">10.1093/bioinformatics/btp693</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Floxin, a resource for genetically engineering mouse ESCs.</h4>
<p class="authors"><span class="author">Singla V</span>, <span class="author">Hunkapiller J</span>, <span class="author">Santos N</span>, <span class="author">Seol AD</span>, <span class="author">Norman AR</span>, <span class="author">Wakenight P</span>, <span class="author">Skarnes WC</span> and <span class="author">Reiter JF</span></p>
<p class="affiliation">Department of Biochemistry and Biophysics, Cardiovascular Research Institute, University of California, San Francisco, San Francisco, California, USA.</p><p class="abstract">We describe a method for the highly efficient and precise targeted modification of gene trap loci in mouse embryonic stem cells (ESCs). Through the Floxin method, gene trap mutations were reverted and new DNA sequences inserted using Cre recombinase and a shuttle vector, pFloxin. Floxin technology is applicable to the existing collection of 24,149 compatible gene trap cell lines, which should enable high-throughput modification of many genes in mouse ESCs.</p>
<p class="grants">Funded by: NIAMS NIH HHS: <span class="grant">R01 AR054396</span>, <span class="grant">R01 AR054396-01A1</span>, <span class="grant">R01AR054396</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2010</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">50-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19966808">19966808</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2895430">2895430</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1406">10.1038/nmeth.1406</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An association analysis of circadian genes in anxiety disorders.</h4>
<p class="authors"><span class="author">Sipil&auml; T</span>, <span class="author">Kananen L</span>, <span class="author">Greco D</span>, <span class="author">Donner J</span>, <span class="author">Silander K</span>, <span class="author">Terwilliger JD</span>, <span class="author">Auvinen P</span>, <span class="author">Peltonen L</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Pirkola S</span>, <span class="author">Partonen T</span> and <span class="author">Hovatta I</span></p>
<p class="affiliation">Research Program of Molecular Neurology, University of Helsinki, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: The mammalian circadian system is responsible for controlling daily oscillations in physiology and behavior. Circadian genes contribute to the sleep-wake cycle and mood, and because patients with anxiety disorder often suffer from sleep disturbances, we hypothesized that variants in circadian-clock-related genes might predispose to human anxiety disorders as well. We tested this hypothesis with a genetic association analysis.</p><p class="abstract"><strong>Methods</strong>: We analyzed 131 single nucleotide polymorphisms from 13 circadian-clock-related genes. The study sample consisted of 321 individuals diagnosed with an anxiety disorder and 653 matched healthy controls from a Finnish population-based cohort.</p><p class="abstract"><strong>Results</strong>: Single nucleotide polymorphisms in two genes showed some evidence for association to social phobia: in ARNTL2 rs2306073 (p = .0099) and in DRD2 rs7131056 (p = .0084). BCL2 rs12454712 (p = .0029) and DRD2 rs4245146 (p = .0010) showed evidence for association to generalized anxiety disorder, whereas rs2463107 (p = .0064) in PAWR and rs4245146 (p = .0029) in DRD2 showed evidence for association to the pooled group of all anxiety disorders. Findings in DRD2 became stronger when only anxiety disorder cases with comorbid alcohol use disorder were considered.</p><p class="abstract"><strong>Conclusions</strong>: Genes contributing to circadian rhythms might also play a role in the genetic predisposition to anxiety disorders. In addition, our study provides further support for the association of DRD2 to comorbid anxiety and alcohol use disorder.</p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">67</span>;<span class="number">12</span>;<span class="pagerange">1163-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20122683">20122683</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2009.12.011">10.1016/j.biopsych.2009.12.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Family history of premature coronary heart disease and risk prediction in the EPIC-Norfolk prospective population study.</h4>
<p class="authors"><span class="author">Sivapalaratnam S</span>, <span class="author">Boekholdt SM</span>, <span class="author">Trip MD</span>, <span class="author">Sandhu MS</span>, <span class="author">Luben R</span>, <span class="author">Kastelein JJ</span>, <span class="author">Wareham NJ</span> and <span class="author">Khaw KT</span></p>
<p class="affiliation">Department of Vascular Medicine, Academic Medical Center, Amsterdam, The Netherlands.</p><p class="abstract"><strong>Objective</strong>: The value of a family history for coronary heart disease (CHD) in addition to established cardiovascular risk factors in predicting an individual&#39;s risk of CHD is unclear. In the European Prospective Investigation of Cancer (EPIC)-Norfolk cohort, the authors tested whether adding family history of premature CHD in first-degree relatives improves risk prediction compared with the Framingham risk score (FRS) alone.</p><p class="abstract">This study comprised 10,288 men and 12,553 women aged 40-79 years participating in the EPIC-Norfolk cohort who were followed for a mean of 10.9&plusmn;2.1 years (mean&plusmn;SD). The authors computed the FRS as well as a modified score taking into account family history of premature CHD. A family history of CHD was indeed associated with an increased risk of future CHD, independent of established risk factors (FRS-adjusted HR of 1.74 (95% CI 1.56 to 1.95) for family history of premature CHD). However, adding family history of CHD to the FRS resulted in a negative net reclassification of 2%. In the subgroup of individuals estimated to be at intermediate risk, family history of premature CHD resulted in an increase in net reclassification of 2%. The sensitivity increased with 0.4%, and the specificity decreased 0.8%.</p><p class="abstract"><strong>Conclusion</strong>: Although family history of CHD was an independent risk factor of future CHD, its use did not improve classification of individuals into clinically relevant risk categories based on the FRS. Among study participants at intermediate risk of CHD, adding family history of premature CHD resulted in, at best, a modest improvement in reclassification of individuals into a more accurate risk category.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council</p>
<p class="publication"><span class="title">Heart (British Cardiac Society)</span>&nbsp;<span class="year">2010</span>;<span class="volume">96</span>;<span class="number">24</span>;<span class="pagerange">1985-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20962344">20962344</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fhrt.2010.210740">10.1136/hrt.2010.210740</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neuronal MeCP2 is expressed at near histone-octamer levels and globally alters the chromatin state.</h4>
<p class="authors"><span class="author">Skene PJ</span>, <span class="author">Illingworth RS</span>, <span class="author">Webb S</span>, <span class="author">Kerr AR</span>, <span class="author">James KD</span>, <span class="author">Turner DJ</span>, <span class="author">Andrews R</span> and <span class="author">Bird AP</span></p>
<p class="affiliation">Wellcome Trust Centre for Cell Biology, University of Edinburgh, Edinburgh EH9 3JR, UK.</p><p class="abstract">MeCP2 is a nuclear protein with an affinity for methylated DNA that can recruit histone deacetylases. Deficiency or excess of MeCP2 causes severe neurological problems, suggesting that the number of molecules per cell must be precisely regulated. We quantified MeCP2 in neuronal nuclei and found that it is nearly as abundant as the histone octamer. Despite this high abundance, MeCP2 associates preferentially with methylated regions, and high-throughput sequencing showed that its genome-wide binding tracks methyl-CpG density. MeCP2 deficiency results in global changes in neuronal chromatin structure, including elevated histone acetylation and a doubling of histone H1. Neither change is detectable in glia, where MeCP2 occurs at lower levels. The mutant brain also shows elevated transcription of repetitive elements. Our data argue that MeCP2 may not act as a gene-specific transcriptional repressor in neurons, but might instead dampen transcriptional noise genome-wide in a DNA methylation-dependent manner.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular cell</span>&nbsp;<span class="year">2010</span>;<span class="volume">37</span>;<span class="number">4</span>;<span class="pagerange">457-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20188665">20188665</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molcel.2010.01.030">10.1016/j.molcel.2010.01.030</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Constitutional translocation breakpoint mapping by genome-wide paired-end sequencing identifies HACE1 as a putative Wilms tumour susceptibility gene.</h4>
<p class="authors"><span class="author">Slade I</span>, <span class="author">Stephens P</span>, <span class="author">Douglas J</span>, <span class="author">Barker K</span>, <span class="author">Stebbings L</span>, <span class="author">Abbaszadeh F</span>, <span class="author">Pritchard-Jones K</span>, <span class="author">FACT collaboration</span>, <span class="author">Cole R</span>, <span class="author">Pizer B</span>, <span class="author">Stiller C</span>, <span class="author">Vujanic G</span>, <span class="author">Scott RH</span>, <span class="author">Stratton MR</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Section Chair and Professor of Human Genetics, The Institute of Cancer Research, 15 Cotswold Road, Sutton SM2 5NG, UK.</p><p class="abstract"><strong>Background</strong>: Localisation of the breakpoints of chromosomal translocations has aided the discovery of several disease genes but has traditionally required laborious investigation of chromosomes by fluorescent in situ hybridisation approaches. Here, a strategy that utilises genome-wide paired-end massively parallel DNA sequencing to rapidly map translocation breakpoints is reported. This method was used to fine map a de novo t(5;6)(q21;q21) translocation in a child with bilateral, young-onset Wilms tumour.</p><p class="abstract">Genome-wide paired-end sequencing was performed for approximately 6 million randomly generated approximately 3 kb fragments from constitutional DNA containing the translocation, and six fragments in which one end mapped to chromosome 5 and the other to chromosome 6 were identified. This mapped the translocation breakpoints to within 1.7 kb. Then, PCR assays that amplified across the rearrangement junction were designed to characterise the breakpoints at sequence-level resolution. The 6q21 breakpoint transects and truncates HACE1, an E3 ubiquitin-protein ligase that has been implicated as a somatically inactivated target in Wilms tumourigenesis. To evaluate the contribution of HACE1 to Wilms tumour predisposition, the gene was mutationally screened in 450 individuals with Wilms tumour. One child with unilateral Wilms tumour and a truncating HACE1 mutation was identified.</p><p class="abstract"><strong>Conclusions</strong>: These data indicate that constitutional disruption of HACE1 likely predisposes to Wilms tumour. However, HACE1 mutations are rare and therefore can only make a small contribution to Wilms tumour incidence. More broadly, this study demonstrates the utility of genome-wide paired-end sequencing in the delineation of apparently balanced chromosomal translocations, for which it is likely to become the method of choice.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">11886</span>, <span class="grant">9024</span>, <span class="grant">C8620_A8857</span>, <span class="grant">C8620_A9024</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">47</span>;<span class="number">5</span>;<span class="pagerange">342-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19948536">19948536</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2009.072983">10.1136/jmg.2009.072983</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Longitudinal genome-wide association of cardiovascular disease risk factors in the Bogalusa heart study.</h4>
<p class="authors"><span class="author">Smith EN</span>, <span class="author">Chen W</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Kettunen J</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Peltonen L</span>, <span class="author">Raitakari OT</span>, <span class="author">Salem RM</span>, <span class="author">Schork NJ</span>, <span class="author">Shaw M</span>, <span class="author">Srinivasan SR</span>, <span class="author">Topol EJ</span>, <span class="author">Viikari JS</span>, <span class="author">Berenson GS</span> and <span class="author">Murray SS</span></p>
<p class="affiliation">Scripps Genomic Medicine and Scripps Translational Science Institute, La Jolla, California, United States of America.</p><p class="abstract">Cardiovascular disease (CVD) is the leading cause of death worldwide. Recent genome-wide association (GWA) studies have pinpointed many loci associated with CVD risk factors in adults. It is unclear, however, if these loci predict trait levels at all ages, if they are associated with how a trait develops over time, or if they could be used to screen individuals who are pre-symptomatic to provide the opportunity for preventive measures before disease onset. We completed a genome-wide association study on participants in the longitudinal Bogalusa Heart Study (BHS) and have characterized the association between genetic factors and the development of CVD risk factors from childhood to adulthood. We report 7 genome-wide significant associations involving CVD risk factors, two of which have been previously reported. Top regions were tested for replication in the Young Finns Study (YF) and two associations strongly replicated: rs247616 in CETP with HDL levels (combined P&#x200A;=&#x200A;9.7 x 10(-24)), and rs445925 at APOE with LDL levels (combined P&#x200A;=&#x200A;8.7 x 10(-19)). We show that SNPs previously identified in adult cross-sectional studies tend to show age-independent effects in the BHS with effect sizes consistent with previous reports. Previously identified variants were associated with adult trait levels above and beyond those seen in childhood; however, variants with time-dependent effects were also promising predictors. This is the first GWA study to evaluate the role of common genetic variants in the development of CVD risk factors in children as they advance through adulthood and highlights the utility of using longitudinal studies to identify genetic predictors of adult traits in children.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">1U54RR025204-01</span>; NIA NIH HHS: <span class="grant">AG-16592</span>; NICHD NIH HHS: <span class="grant">HD-061437</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20838585" rel="external">20838585</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2936521" rel="external">2936521</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001094" rel="external">10.1371/journal.pgen.1001094</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel associations of multiple genetic loci with plasma levels of factor VII, factor VIII, and von Willebrand factor: The CHARGE (Cohorts for Heart and Aging Research in Genome Epidemiology) Consortium.</h4>
<p class="authors"><span class="author">Smith NL</span>, <span class="author">Chen MH</span>, <span class="author">Dehghan A</span>, <span class="author">Strachan DP</span>, <span class="author">Basu S</span>, <span class="author">Soranzo N</span>, <span class="author">Hayward C</span>, <span class="author">Rudan I</span>, <span class="author">Sabater-Lleal M</span>, <span class="author">Bis JC</span>, <span class="author">de Maat MP</span>, <span class="author">Rumley A</span>, <span class="author">Kong X</span>, <span class="author">Yang Q</span>, <span class="author">Williams FM</span>, <span class="author">Vitart V</span>, <span class="author">Campbell H</span>, <span class="author">M&auml;larstig A</span>, <span class="author">Wiggins KL</span>, <span class="author">Van Duijn CM</span>, <span class="author">McArdle WL</span>, <span class="author">Pankow JS</span>, <span class="author">Johnson AD</span>, <span class="author">Silveira A</span>, <span class="author">McKnight B</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Wellcome Trust Case Control Consortium;</span>, <span class="author">Aleksic N</span>, <span class="author">Meigs JB</span>, <span class="author">Peters A</span>, <span class="author">Koenig W</span>, <span class="author">Cushman M</span>, <span class="author">Kathiresan S</span>, <span class="author">Rotter JI</span>, <span class="author">Bovill EG</span>, <span class="author">Hofman A</span>, <span class="author">Boerwinkle E</span>, <span class="author">Tofler GH</span>, <span class="author">Peden JF</span>, <span class="author">Psaty BM</span>, <span class="author">Leebeek F</span>, <span class="author">Folsom AR</span>, <span class="author">Larson MG</span>, <span class="author">Spector TD</span>, <span class="author">Wright AF</span>, <span class="author">Wilson JF</span>, <span class="author">Hamsten A</span>, <span class="author">Lumley T</span>, <span class="author">Witteman JC</span>, <span class="author">Tang W</span> and <span class="author">O&#39;Donnell CJ</span></p>
<p class="affiliation">Department of Epidemiology, University of Washington, Seattle, Wash 98101, USA. <a href="mailto:%6E%6C%73%6D%69%74%68%40%75.%77%61%73%68%69%6E%67%74%6F%6E.%65%64%75">&#110;&#108;&#115;&#109;&#105;&#116;&#104;&#64;&#117;&#46;&#119;&#97;&#115;&#104;&#105;&#110;&#103;&#116;&#111;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Plasma levels of coagulation factors VII (FVII), VIII (FVIII), and von Willebrand factor (vWF) influence risk of hemorrhage and thrombosis. We conducted genome-wide association studies to identify new loci associated with plasma levels.</p><p class="abstract">The setting of the study included 5 community-based studies for discovery comprising 23 608 European-ancestry participants: Atherosclerosis Risk In Communities Study, Cardiovascular Health Study, British 1958 Birth Cohort, Framingham Heart Study, and Rotterdam Study. All subjects had genome-wide single-nucleotide polymorphism (SNP) scans and at least 1 phenotype measured: FVII activity/antigen, FVIII activity, and vWF antigen. Each study used its genotype data to impute to HapMap SNPs and independently conducted association analyses of hemostasis measures using an additive genetic model. Study findings were combined by meta-analysis. Replication was conducted in 7604 participants not in the discovery cohort. For FVII, 305 SNPs exceeded the genome-wide significance threshold of 5.0x10(-8) and comprised 5 loci on 5 chromosomes: 2p23 (smallest P value 6.2x10(-24)), 4q25 (3.6x10(-12)), 11q12 (2.0x10(-10)), 13q34 (9.0x10(-259)), and 20q11.2 (5.7x10(-37)). Loci were within or near genes, including 4 new candidate genes and F7 (13q34). For vWF, 400 SNPs exceeded the threshold and marked 8 loci on 6 chromosomes: 6q24 (1.2x10(-22)), 8p21 (1.3x10(-16)), 9q34 (&lt;5.0x10(-324)), 12p13 (1.7x10(-32)), 12q23 (7.3x10(-10)), 12q24.3 (3.8x10(-11)), 14q32 (2.3x10(-10)), and 19p13.2 (1.3x10(-9)). All loci were within genes, including 6 new candidate genes, as well as ABO (9q34) and VWF (12p13). For FVIII, 5 loci were identified and overlapped vWF findings. Nine of the 10 new findings were replicated.</p><p class="abstract"><strong>Conclusions</strong>: New genetic associations were discovered outside previously known biological pathways and may point to novel prevention and treatment targets of hemostasis disorders.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G20234</span>, <span class="grant">G20234/2</span>; Chief Scientist Office; Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0000934(68341)</span>; NCRR NIH HHS: <span class="grant">M01 RR000425-39</span>, <span class="grant">M01-RR00425</span>, <span class="grant">UL1 RR025005-02</span>, <span class="grant">UL1-RR-025005</span>; NHGRI NIH HHS: <span class="grant">U01 HG004402-02S1</span>, <span class="grant">U01-HG-004402</span>; NHLBI NIH HHS: <span class="grant">HL073410</span>, <span class="grant">HL095080</span>, <span class="grant">N01 HC015103</span>, <span class="grant">N01 HC025195</span>, <span class="grant">N01 HC035129</span>, <span class="grant">N01 HC045133</span>, <span class="grant">N01 HC055015</span>, <span class="grant">N01 HC055016</span>, <span class="grant">N01 HC055018</span>, <span class="grant">N01 HC055019</span>, <span class="grant">N01 HC055020</span>, <span class="grant">N01 HC055021</span>, <span class="grant">N01 HC055022</span>, <span class="grant">N01 HC055222</span>, <span class="grant">N01 HC075150</span>, <span class="grant">N01 HC085079</span>, <span class="grant">N01 HC085080</span>, <span class="grant">N01 HC085081</span>, <span class="grant">N01 HC085082</span>, <span class="grant">N01 HC085083</span>, <span class="grant">N01 HC085084</span>, <span class="grant">N01 HC085085</span>, <span class="grant">N01 HC085086</span>, <span class="grant">N01-HC-15103</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-55222</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N02 HL64278</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL 087652</span>, <span class="grant">R01 HL059367-09</span>, <span class="grant">R01 HL073410-05A2</span>, <span class="grant">R01 HL086694-03</span>, <span class="grant">R01 HL087641-03</span>, <span class="grant">R01 HL087652-03</span>, <span class="grant">R01 HL095080-02</span>, <span class="grant">R01-HL-086694</span>, <span class="grant">R01-HL-087641</span>, <span class="grant">R01-HL-59367</span>, <span class="grant">U01 HL080295</span>, <span class="grant">U01 HL080295-04</span>; NIDDK NIH HHS: <span class="grant">DK063491</span>, <span class="grant">K24 DK080140</span>, <span class="grant">K24 DK080140-03</span>, <span class="grant">K24 DK080140-05</span>, <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK063491-07</span>, <span class="grant">R01 DK033651</span>, <span class="grant">R01 DK033651-29</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">079996</span>, <span class="grant">091746</span></p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2010</span>;<span class="volume">121</span>;<span class="number">12</span>;<span class="pagerange">1382-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20231535" rel="external">20231535</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2861278" rel="external">2861278</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.109.869156" rel="external">10.1161/CIRCULATIONAHA.109.869156</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A multiplex single nucleotide polymorphism typing assay for detecting mutations that result in decreased fluoroquinolone susceptibility in Salmonella enterica serovars Typhi and Paratyphi A.</h4>
<p class="authors"><span class="author">Song Y</span>, <span class="author">Roumagnac P</span>, <span class="author">Weill FX</span>, <span class="author">Wain J</span>, <span class="author">Dolecek C</span>, <span class="author">Mazzoni CJ</span>, <span class="author">Holt KE</span> and <span class="author">Achtman M</span></p>
<p class="affiliation">Environmental Research Institute, University College Cork, Lee Road, Cork, Ireland.</p><p class="abstract"><strong>Objectives</strong>: Decreased susceptibility to fluoroquinolones has become a major problem for the successful therapy of human infections caused by Salmonella enterica, especially the life-threatening typhoid and paratyphoid fevers.</p><p class="abstract"><strong>Methods</strong>: By using Luminex xTAG beads, we developed a rapid, reliable and cost-effective multiplexed genotyping assay for simultaneously detecting 11 mutations in gyrA, gyrB and parE of S. enterica serovars Typhi and Paratyphi A that result in nalidixic acid resistance (Nal(R)) and/or decreased susceptibility to fluoroquinolones.</p><p class="abstract"><strong>Results</strong>: This assay yielded unambiguous single nucleotide polymorphism calls on extracted DNA from 292 isolates of Salmonella Typhi (Nal(R) = 223 and Nal(S) = 69) and 106 isolates of Salmonella Paratyphi A (Nal(R) = 24 and Nal(S) = 82). All of the 247 Nal(R) Salmonella Typhi and Salmonella Paratyphi A isolates were found to harbour at least one of the target mutations, with GyrA Phe-83 as the most common one (143/223 for Salmonella Typhi and 18/24 for Salmonella Paratyphi A). We also identified three GyrB mutations in eight Nal(S) Salmonella Typhi isolates (six for GyrB Phe-464, one for GyrB Leu-465 and one for GyrB Asp-466), and mutations GyrB Phe-464 and GyrB Asp-466 seem to be related to the decreased ciprofloxacin susceptibility phenotype in Salmonella Typhi. This assay can also be used directly on boiled single colonies.</p><p class="abstract"><strong>Conclusions</strong>: The assay presented here would be useful for clinical and reference laboratories to rapidly screen quinolone-resistant isolates of Salmonella Typhi and Salmonella Paratyphi A, and decipher the underlying genetic changes for epidemiological purposes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of antimicrobial chemotherapy</span>&nbsp;<span class="year">2010</span>;<span class="volume">65</span>;<span class="number">8</span>;<span class="pagerange">1631-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20511368" rel="external">20511368</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2904664" rel="external">2904664</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fjac%2Fdkq175" rel="external">10.1093/jac/dkq175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High rates of homologous recombination in the mite endosymbiont and opportunistic human pathogen Orientia tsutsugamushi.</h4>
<p class="authors"><span class="author">Sonthayanon P</span>, <span class="author">Peacock SJ</span>, <span class="author">Chierakul W</span>, <span class="author">Wuthiekanun V</span>, <span class="author">Blacksell SD</span>, <span class="author">Holden MT</span>, <span class="author">Bentley SD</span>, <span class="author">Feil EJ</span> and <span class="author">Day NP</span></p>
<p class="affiliation">Department of Clinical Tropical Medicine, Faculty of Tropical Medicine, Mahidol University, Bangkok, Thailand.</p><p class="abstract">Orientia tsutsugamushi is an intracellular alpha-proteobacterium which resides in trombiculid mites, and is the causative agent of scrub typhus in East Asia. The genome sequence of this species has revealed an unprecedented number of repeat sequences, most notably of the genes encoding the conjugative properties of a type IV secretion system (T4SS). Although this observation is consistent with frequent intragenomic recombination, the extent of homologous recombination (gene conversion) in this species is unknown. To address this question, and to provide a protocol for the epidemiological surveillance of this important pathogen, we have developed a multilocus sequence typing (MLST) scheme based on 7 housekeeping genes (gpsA, mdh, nrdB, nuoF, ppdK, sucD, sucB). We applied this scheme to the two published genomes, and to DNA extracted from blood taken from 84 Thai scrub typhus patients, from 20 cultured Thai patient isolates, 1 Australian patient sample, and from 3 cultured type strains. These data demonstrated that the O. tsutsugamushi population was both highly diverse [Simpson&#39;s index (95% CI) = 0.95 (0.92-0.98)], and highly recombinogenic. These results are surprising given the intracellular life-style of this species, but are broadly consistent with results obtained for Wolbachia, which is an alpha-proteobacterial reproductive parasite of arthropods. We also compared the MLST data with ompA sequence data and noted low levels of consistency and much higher discrimination by MLST. Finally, twenty-five percent of patients in this study were simultaneously infected with multiple sequence types, suggesting multiple infection caused by either multiple mite bites, or multiple strains co-existing within individual mites.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">073135</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="number">7</span>;<span class="pagerange">e752</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20651929" rel="external">20651929</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2907413" rel="external">2907413</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000752" rel="external">10.1371/journal.pntd.0000752</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants at 10 genomic loci influence hemoglobin A&#x2081;(C) levels via glycemic and nonglycemic pathways.</h4>
<p class="authors"><span class="author">Soranzo N</span>, <span class="author">Sanna S</span>, <span class="author">Wheeler E</span>, <span class="author">Gieger C</span>, <span class="author">Radke D</span>, <span class="author">Dupuis J</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Langenberg C</span>, <span class="author">Prokopenko I</span>, <span class="author">Stolerman E</span>, <span class="author">Sandhu MS</span>, <span class="author">Heeney MM</span>, <span class="author">Devaney JM</span>, <span class="author">Reilly MP</span>, <span class="author">Ricketts SL</span>, <span class="author">Stewart AF</span>, <span class="author">Voight BF</span>, <span class="author">Willenborg C</span>, <span class="author">Wright B</span>, <span class="author">Altshuler D</span>, <span class="author">Arking D</span>, <span class="author">Balkau B</span>, <span class="author">Barnes D</span>, <span class="author">Boerwinkle E</span>, <span class="author">B&ouml;hm B</span>, <span class="author">Bonnefond A</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Boomsma DI</span>, <span class="author">Bornstein SR</span>, <span class="author">B&ouml;ttcher Y</span>, <span class="author">Bumpstead S</span>, <span class="author">Burnett-Miller MS</span>, <span class="author">Campbell H</span>, <span class="author">Cao A</span>, <span class="author">Chambers J</span>, <span class="author">Clark R</span>, <span class="author">Collins FS</span>, <span class="author">Coresh J</span>, <span class="author">de Geus EJ</span>, <span class="author">Dei M</span>, <span class="author">Deloukas P</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Egan JM</span>, <span class="author">Elosua R</span>, <span class="author">Ferrucci L</span>, <span class="author">Forouhi N</span>, <span class="author">Fox CS</span>, <span class="author">Franklin C</span>, <span class="author">Franzosi MG</span>, <span class="author">Gallina S</span>, <span class="author">Goel A</span>, <span class="author">Graessler J</span>, <span class="author">Grallert H</span>, <span class="author">Greinacher A</span>, <span class="author">Hadley D</span>, <span class="author">Hall A</span>, <span class="author">Hamsten A</span>, <span class="author">Hayward C</span>, <span class="author">Heath S</span>, <span class="author">Herder C</span>, <span class="author">Homuth G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Hunter-Merrill R</span>, <span class="author">Illig T</span>, <span class="author">Jackson AU</span>, <span class="author">Jula A</span>, <span class="author">Kleber M</span>, <span class="author">Knouff CW</span>, <span class="author">Kong A</span>, <span class="author">Kooner J</span>, <span class="author">K&ouml;ttgen A</span>, <span class="author">Kovacs P</span>, <span class="author">Krohn K</span>, <span class="author">K&uuml;hnel B</span>, <span class="author">Kuusisto J</span>, <span class="author">Laakso M</span>, <span class="author">Lathrop M</span>, <span class="author">Lecoeur C</span>, <span class="author">Li M</span>, <span class="author">Li M</span>, <span class="author">Loos RJ</span>, <span class="author">Luan J</span>, <span class="author">Lyssenko V</span>, <span class="author">M&auml;gi R</span>, <span class="author">Magnusson PK</span>, <span class="author">M&auml;larstig A</span>, <span class="author">Mangino M</span>, <span class="author">Mart&iacute;nez-Larrad MT</span>, <span class="author">M&auml;rz W</span>, <span class="author">McArdle WL</span>, <span class="author">McPherson R</span>, <span class="author">Meisinger C</span>, <span class="author">Meitinger T</span>, <span class="author">Melander O</span>, <span class="author">Mohlke KL</span>, <span class="author">Mooser VE</span>, <span class="author">Morken MA</span>, <span class="author">Narisu N</span>, <span class="author">Nathan DM</span>, <span class="author">Nauck M</span>, <span class="author">O&#39;Donnell C</span>, <span class="author">Oexle K</span>, <span class="author">Olla N</span>, <span class="author">Pankow JS</span>, <span class="author">Payne F</span>, <span class="author">Peden JF</span>, <span class="author">Pedersen NL</span>, <span class="author">Peltonen L</span>, <span class="author">Perola M</span>, <span class="author">Polasek O</span>, <span class="author">Porcu E</span>, <span class="author">Rader DJ</span>, <span class="author">Rathmann W</span>, <span class="author">Ripatti S</span>, <span class="author">Rocheleau G</span>, <span class="author">Roden M</span>, <span class="author">Rudan I</span>, <span class="author">Salomaa V</span>, <span class="author">Saxena R</span>, <span class="author">Schlessinger D</span>, <span class="author">Schunkert H</span>, <span class="author">Schwarz P</span>, <span class="author">Seedorf U</span>, <span class="author">Selvin E</span>, <span class="author">Serrano-R&iacute;os M</span>, <span class="author">Shrader P</span>, <span class="author">Silveira A</span>, <span class="author">Siscovick D</span>, <span class="author">Song K</span>, <span class="author">Spector TD</span>, <span class="author">Stefansson K</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Strachan DP</span>, <span class="author">Strawbridge R</span>, <span class="author">Stumvoll M</span>, <span class="author">Surakka I</span>, <span class="author">Swift AJ</span>, <span class="author">Tanaka T</span>, <span class="author">Teumer A</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Usala G</span>, <span class="author">Vitart V</span>, <span class="author">V&ouml;lzke H</span>, <span class="author">Wallaschofski H</span>, <span class="author">Waterworth DM</span>, <span class="author">Watkins H</span>, <span class="author">Wichmann HE</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Williams GH</span>, <span class="author">Wilson JF</span>, <span class="author">Winkelmann J</span>, <span class="author">Wright AF</span>, <span class="author">WTCCC</span>, <span class="author">Zabena C</span>, <span class="author">Zhao JH</span>, <span class="author">Epstein SE</span>, <span class="author">Erdmann J</span>, <span class="author">Hakonarson HH</span>, <span class="author">Kathiresan S</span>, <span class="author">Khaw KT</span>, <span class="author">Roberts R</span>, <span class="author">Samani NJ</span>, <span class="author">Fleming MD</span>, <span class="author">Sladek R</span>, <span class="author">Abecasis G</span>, <span class="author">Boehnke M</span>, <span class="author">Froguel P</span>, <span class="author">Groop L</span>, <span class="author">McCarthy MI</span>, <span class="author">Kao WH</span>, <span class="author">Florez JC</span>, <span class="author">Uda M</span>, <span class="author">Wareham NJ</span>, <span class="author">Barroso I</span> and <span class="author">Meigs JB</span></p>
<p class="affiliation">Human Genetics, Wellcome Trust Sanger Institute, Hinxton, U.K.</p><p class="abstract"><strong>Objective</strong>: Glycated hemoglobin (HbA&#x2081;(c)), used to monitor and diagnose diabetes, is influenced by average glycemia over a 2- to 3-month period. Genetic factors affecting expression, turnover, and abnormal glycation of hemoglobin could also be associated with increased levels of HbA&#x2081;(c). We aimed to identify such genetic factors and investigate the extent to which they influence diabetes classification based on HbA&#x2081;(c) levels.</p><p class="abstract">We studied associations with HbA&#x2081;(c) in up to 46,368 nondiabetic adults of European descent from 23 genome-wide association studies (GWAS) and 8 cohorts with de novo genotyped single nucleotide polymorphisms (SNPs). We combined studies using inverse-variance meta-analysis and tested mediation by glycemia using conditional analyses. We estimated the global effect of HbA&#x2081;(c) loci using a multilocus risk score, and used net reclassification to estimate genetic effects on diabetes screening.</p><p class="abstract"><strong>Results</strong>: Ten loci reached genome-wide significant association with HbA(1c), including six new loci near FN3K (lead SNP/P value, rs1046896/P = 1.6 &times; 10&#x207B;&sup2;&#x2076;), HFE (rs1800562/P = 2.6 &times; 10&#x207B;&sup2;&#x2070;), TMPRSS6 (rs855791/P = 2.7 &times; 10&#x207B;&sup1;&#x2074;), ANK1 (rs4737009/P = 6.1 &times; 10&#x207B;&sup1;&sup2;), SPTA1 (rs2779116/P = 2.8 &times; 10&#x207B;&#x2079;) and ATP11A/TUBGCP3 (rs7998202/P = 5.2 &times; 10&#x207B;&#x2079;), and four known HbA&#x2081;(c) loci: HK1 (rs16926246/P = 3.1 &times; 10&#x207B;&#x2075;&#x2074;), MTNR1B (rs1387153/P = 4.0 &times; 10&#x207B;&sup1;&sup1;), GCK (rs1799884/P = 1.5 &times; 10&#x207B;&sup2;&#x2070;) and G6PC2/ABCB11 (rs552976/P = 8.2 &times; 10&#x207B;&sup1;&#x2078;). We show that associations with HbA&#x2081;(c) are partly a function of hyperglycemia associated with 3 of the 10 loci (GCK, G6PC2 and MTNR1B). The seven nonglycemic loci accounted for a 0.19 (% HbA&#x2081;(c)) difference between the extreme 10% tails of the risk score, and would reclassify &sim;2% of a general white population screened for diabetes with HbA&#x2081;(c).</p><p class="abstract"><strong>Conclusions</strong>: GWAS identified 10 genetic loci reproducibly associated with HbA&#x2081;(c). Six are novel and seven map to loci where rarer variants cause hereditary anemias and iron storage disorders. Common variants at these loci likely influence HbA&#x2081;(c) levels via erythrocyte biology, and confer a small but detectable reclassification of diabetes diagnosis by HbA&#x2081;(c).</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Medical Research Council: <span class="grant">G0401527</span>, <span class="grant">G0701863</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_UP_A100_1003</span>; NIDDK NIH HHS: <span class="grant">R01 DK072193</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2010</span>;<span class="volume">59</span>;<span class="number">12</span>;<span class="pagerange">3229-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20858683">20858683</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2992787">2992787</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb10-0502">10.2337/db10-0502</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of GWAS of bipolar disorder: association of SNPs near CDH7 with bipolar disorder and visual processing.</h4>
<p class="authors"><span class="author">Soronen P</span>, <span class="author">Ollila HM</span>, <span class="author">Antila M</span>, <span class="author">Silander K</span>, <span class="author">Palo OM</span>, <span class="author">Kiesepp&auml; T</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Peltonen L</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Partonen T</span> and <span class="author">Paunio T</span></p>
<p class="publication"><span class="title">Molecular psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">15</span>;<span class="number">1</span>;<span class="pagerange">4-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20029406" rel="external">20029406</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fmp.2009.86" rel="external">10.1038/mp.2009.86</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association analyses of 249,796 individuals reveal 18 new loci associated with body mass index.</h4>
<p class="authors"><span class="author">Speliotes EK</span>, <span class="author">Willer CJ</span>, <span class="author">Berndt SI</span>, <span class="author">Monda KL</span>, <span class="author">Thorleifsson G</span>, <span class="author">Jackson AU</span>, <span class="author">Lango Allen H</span>, <span class="author">Lindgren CM</span>, <span class="author">Luan J</span>, <span class="author">M&auml;gi R</span>, <span class="author">Randall JC</span>, <span class="author">Vedantam S</span>, <span class="author">Winkler TW</span>, <span class="author">Qi L</span>, <span class="author">Workalemahu T</span>, <span class="author">Heid IM</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Stringham HM</span>, <span class="author">Weedon MN</span>, <span class="author">Wheeler E</span>, <span class="author">Wood AR</span>, <span class="author">Ferreira T</span>, <span class="author">Weyant RJ</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">Estrada K</span>, <span class="author">Liang L</span>, <span class="author">Nemesh J</span>, <span class="author">Park JH</span>, <span class="author">Gustafsson S</span>, <span class="author">Kilpel&auml;inen TO</span>, <span class="author">Yang J</span>, <span class="author">Bouatia-Naji N</span>, <span class="author">Esko T</span>, <span class="author">Feitosa MF</span>, <span class="author">Kutalik Z</span>, <span class="author">Mangino M</span>, <span class="author">Raychaudhuri S</span>, <span class="author">Scherag A</span>, <span class="author">Smith AV</span>, <span class="author">Welch R</span>, <span class="author">Zhao JH</span>, <span class="author">Aben KK</span>, <span class="author">Absher DM</span>, <span class="author">Amin N</span>, <span class="author">Dixon AL</span>, <span class="author">Fisher E</span>, <span class="author">Glazer NL</span>, <span class="author">Goddard ME</span>, <span class="author">Heard-Costa NL</span>, <span class="author">Hoesel V</span>, <span class="author">Hottenga JJ</span>, <span class="author">Johansson A</span>, <span class="author">Johnson T</span>, <span class="author">Ketkar S</span>, <span class="author">Lamina C</span>, <span class="author">Li S</span>, <span class="author">Moffatt MF</span>, <span class="author">Myers RH</span>, <span class="author">Narisu N</span>, <span class="author">Perry JR</span>, <span class="author">Peters MJ</span>, <span class="author">Preuss M</span>, <span class="author">Ripatti S</span>, <span class="author">Rivadeneira F</span>, <span class="author">Sandholt C</span>, <span class="author">Scott LJ</span>, <span class="author">Timpson NJ</span>, <span class="author">Tyrer JP</span>, <span class="author">van Wingerden S</span>, <span class="author">Watanabe RM</span>, <span class="author">White CC</span>, <span class="author">Wiklund F</span>, <span class="author">Barlassina C</span>, <span class="author">Chasman DI</span>, <span class="author">Cooper MN</span>, <span class="author">Jansson JO</span>, <span class="author">Lawrence RW</span>, <span class="author">Pellikka N</span>, <span class="author">Prokopenko I</span>, <span class="author">Shi J</span>, <span class="author">Thiering E</span>, <span class="author">Alavere H</span>, <span class="author">Alibrandi MT</span>, <span class="author">Almgren P</span>, <span class="author">Arnold AM</span>, <span class="author">Aspelund T</span>, <span class="author">Atwood LD</span>, <span class="author">Balkau B</span>, <span class="author">Balmforth AJ</span>, <span class="author">Bennett AJ</span>, <span class="author">Ben-Shlomo Y</span>, <span class="author">Bergman RN</span>, <span class="author">Bergmann S</span>, <span class="author">Biebermann H</span>, <span class="author">Blakemore AI</span>, <span class="author">Boes T</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bornstein SR</span>, <span class="author">Brown MJ</span>, <span class="author">Buchanan TA</span>, <span class="author">Busonero F</span>, <span class="author">Campbell H</span>, <span class="author">Cappuccio FP</span>, <span class="author">Cavalcanti-Proen&ccedil;a C</span>, <span class="author">Chen YD</span>, <span class="author">Chen CM</span>, <span class="author">Chines PS</span>, <span class="author">Clarke R</span>, <span class="author">Coin L</span>, <span class="author">Connell J</span>, <span class="author">Day IN</span>, <span class="author">den Heijer M</span>, <span class="author">Duan J</span>, <span class="author">Ebrahim S</span>, <span class="author">Elliott P</span>, <span class="author">Elosua R</span>, <span class="author">Eiriksdottir G</span>, <span class="author">Erdos MR</span>, <span class="author">Eriksson JG</span>, <span class="author">Facheris MF</span>, <span class="author">Felix SB</span>, <span class="author">Fischer-Posovszky P</span>, <span class="author">Folsom AR</span>, <span class="author">Friedrich N</span>, <span class="author">Freimer NB</span>, <span class="author">Fu M</span>, <span class="author">Gaget S</span>, <span class="author">Gejman PV</span>, <span class="author">Geus EJ</span>, <span class="author">Gieger C</span>, <span class="author">Gjesing AP</span>, <span class="author">Goel A</span>, <span class="author">Goyette P</span>, <span class="author">Grallert H</span>, <span class="author">Gr&auml;ssler J</span>, <span class="author">Greenawalt DM</span>, <span class="author">Groves CJ</span>, <span class="author">Gudnason V</span>, <span class="author">Guiducci C</span>, <span class="author">Hartikainen AL</span>, <span class="author">Hassanali N</span>, <span class="author">Hall AS</span>, <span class="author">Havulinna AS</span>, <span class="author">Hayward C</span>, <span class="author">Heath AC</span>, <span class="author">Hengstenberg C</span>, <span class="author">Hicks AA</span>, <span class="author">Hinney A</span>, <span class="author">Hofman A</span>, <span class="author">Homuth G</span>, <span class="author">Hui J</span>, <span class="author">Igl W</span>, <span class="author">Iribarren C</span>, <span class="author">Isomaa B</span>, <span class="author">Jacobs KB</span>, <span class="author">Jarick I</span>, <span class="author">Jewell E</span>, <span class="author">John U</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Jousilahti P</span>, <span class="author">Jula A</span>, <span class="author">Kaakinen M</span>, <span class="author">Kajantie E</span>, <span class="author">Kaplan LM</span>, <span class="author">Kathiresan S</span>, <span class="author">Kettunen J</span>, <span class="author">Kinnunen L</span>, <span class="author">Knowles JW</span>, <span class="author">Kolcic I</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Koskinen S</span>, <span class="author">Kovacs P</span>, <span class="author">Kuusisto J</span>, <span class="author">Kraft P</span>, <span class="author">Kval&oslash;y K</span>, <span class="author">Laitinen J</span>, <span class="author">Lantieri O</span>, <span class="author">Lanzani C</span>, <span class="author">Launer LJ</span>, <span class="author">Lecoeur C</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Lettre G</span>, <span class="author">Liu J</span>, <span class="author">Lokki ML</span>, <span class="author">Lorentzon M</span>, <span class="author">Luben RN</span>, <span class="author">Ludwig B</span>, <span class="author">MAGIC</span>, <span class="author">Manunta P</span>, <span class="author">Marek D</span>, <span class="author">Marre M</span>, <span class="author">Martin NG</span>, <span class="author">McArdle WL</span>, <span class="author">McCarthy A</span>, <span class="author">McKnight B</span>, <span class="author">Meitinger T</span>, <span class="author">Melander O</span>, <span class="author">Meyre D</span>, <span class="author">Midthjell K</span>, <span class="author">Montgomery GW</span>, <span class="author">Morken MA</span>, <span class="author">Morris AP</span>, <span class="author">Mulic R</span>, <span class="author">Ngwa JS</span>, <span class="author">Nelis M</span>, <span class="author">Neville MJ</span>, <span class="author">Nyholt DR</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Ong KK</span>, <span class="author">Oostra B</span>, <span class="author">Par&eacute; G</span>, <span class="author">Parker AN</span>, <span class="author">Perola M</span>, <span class="author">Pichler I</span>, <span class="author">Pietil&auml;inen KH</span>, <span class="author">Platou CG</span>, <span class="author">Polasek O</span>, <span class="author">Pouta A</span>, <span class="author">Rafelt S</span>, <span class="author">Raitakari O</span>, <span class="author">Rayner NW</span>, <span class="author">Ridderstr&aring;le M</span>, <span class="author">Rief W</span>, <span class="author">Ruokonen A</span>, <span class="author">Robertson NR</span>, <span class="author">Rzehak P</span>, <span class="author">Salomaa V</span>, <span class="author">Sanders AR</span>, <span class="author">Sandhu MS</span>, <span class="author">Sanna S</span>, <span class="author">Saramies J</span>, <span class="author">Savolainen MJ</span>, <span class="author">Scherag S</span>, <span class="author">Schipf S</span>, <span class="author">Schreiber S</span>, <span class="author">Schunkert H</span>, <span class="author">Silander K</span>, <span class="author">Sinisalo J</span>, <span class="author">Siscovick DS</span>, <span class="author">Smit JH</span>, <span class="author">Soranzo N</span>, <span class="author">Sovio U</span>, <span class="author">Stephens J</span>, <span class="author">Surakka I</span>, <span class="author">Swift AJ</span>, <span class="author">Tammesoo ML</span>, <span class="author">Tardif JC</span>, <span class="author">Teder-Laving M</span>, <span class="author">Teslovich TM</span>, <span class="author">Thompson JR</span>, <span class="author">Thomson B</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Tuomi T</span>, <span class="author">van Meurs JB</span>, <span class="author">van Ommen GJ</span>, <span class="author">Vatin V</span>, <span class="author">Viikari J</span>, <span class="author">Visvikis-Siest S</span>, <span class="author">Vitart V</span>, <span class="author">Vogel CI</span>, <span class="author">Voight BF</span>, <span class="author">Waite LL</span>, <span class="author">Wallaschofski H</span>, <span class="author">Walters GB</span>, <span class="author">Widen E</span>, <span class="author">Wiegand S</span>, <span class="author">Wild SH</span>, <span class="author">Willemsen G</span>, <span class="author">Witte DR</span>, <span class="author">Witteman JC</span>, <span class="author">Xu J</span>, <span class="author">Zhang Q</span>, <span class="author">Zgaga L</span>, <span class="author">Ziegler A</span>, <span class="author">Zitting P</span>, <span class="author">Beilby JP</span>, <span class="author">Farooqi IS</span>, <span class="author">Hebebrand J</span>, <span class="author">Huikuri HV</span>, <span class="author">James AL</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Levinson DF</span>, <span class="author">Macciardi F</span>, <span class="author">Nieminen MS</span>, <span class="author">Ohlsson C</span>, <span class="author">Palmer LJ</span>, <span class="author">Ridker PM</span>, <span class="author">Stumvoll M</span>, <span class="author">Beckmann JS</span>, <span class="author">Boeing H</span>, <span class="author">Boerwinkle E</span>, <span class="author">Boomsma DI</span>, <span class="author">Caulfield MJ</span>, <span class="author">Chanock SJ</span>, <span class="author">Collins FS</span>, <span class="author">Cupples LA</span>, <span class="author">Smith GD</span>, <span class="author">Erdmann J</span>, <span class="author">Froguel P</span>, <span class="author">Gr&ouml;nberg H</span>, <span class="author">Gyllensten U</span>, <span class="author">Hall P</span>, <span class="author">Hansen T</span>, <span class="author">Harris TB</span>, <span class="author">Hattersley AT</span>, <span class="author">Hayes RB</span>, <span class="author">Heinrich J</span>, <span class="author">Hu FB</span>, <span class="author">Hveem K</span>, <span class="author">Illig T</span>, <span class="author">Jarvelin MR</span>, <span class="author">Kaprio J</span>, <span class="author">Karpe F</span>, <span class="author">Khaw KT</span>, <span class="author">Kiemeney LA</span>, <span class="author">Krude H</span>, <span class="author">Laakso M</span>, <span class="author">Lawlor DA</span>, <span class="author">Metspalu A</span>, <span class="author">Munroe PB</span>, <span class="author">Ouwehand WH</span>, <span class="author">Pedersen O</span>, <span class="author">Penninx BW</span>, <span class="author">Peters A</span>, <span class="author">Pramstaller PP</span>, <span class="author">Quertermous T</span>, <span class="author">Reinehr T</span>, <span class="author">Rissanen A</span>, <span class="author">Rudan I</span>, <span class="author">Samani NJ</span>, <span class="author">Schwarz PE</span>, <span class="author">Shuldiner AR</span>, <span class="author">Spector TD</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uda M</span>, <span class="author">Uitterlinden A</span>, <span class="author">Valle TT</span>, <span class="author">Wabitsch M</span>, <span class="author">Waeber G</span>, <span class="author">Wareham NJ</span>, <span class="author">Watkins H</span>, <span class="author">Procardis Consortium</span>, <span class="author">Wilson JF</span>, <span class="author">Wright AF</span>, <span class="author">Zillikens MC</span>, <span class="author">Chatterjee N</span>, <span class="author">McCarroll SA</span>, <span class="author">Purcell S</span>, <span class="author">Schadt EE</span>, <span class="author">Visscher PM</span>, <span class="author">Assimes TL</span>, <span class="author">Borecki IB</span>, <span class="author">Deloukas P</span>, <span class="author">Fox CS</span>, <span class="author">Groop LC</span>, <span class="author">Haritunians T</span>, <span class="author">Hunter DJ</span>, <span class="author">Kaplan RC</span>, <span class="author">Mohlke KL</span>, <span class="author">O&#39;Connell JR</span>, <span class="author">Peltonen L</span>, <span class="author">Schlessinger D</span>, <span class="author">Strachan DP</span>, <span class="author">van Duijn CM</span>, <span class="author">Wichmann HE</span>, <span class="author">Frayling TM</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Abecasis GR</span>, <span class="author">Barroso I</span>, <span class="author">Boehnke M</span>, <span class="author">Stefansson K</span>, <span class="author">North KE</span>, <span class="author">McCarthy MI</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Ingelsson E</span> and <span class="author">Loos RJ</span></p>
<p class="affiliation">Metabolism Initiative and Program in Medical and Population Genetics, Broad Institute, Cambridge, Massachusetts, USA.</p><p class="abstract">Obesity is globally prevalent and highly heritable, but its underlying genetic factors remain largely elusive. To identify genetic loci for obesity susceptibility, we examined associations between body mass index and &sim; 2.8 million SNPs in up to 123,865 individuals with targeted follow up of 42 SNPs in up to 125,931 additional individuals. We confirmed 14 known obesity susceptibility loci and identified 18 new loci associated with body mass index (P &lt; 5 &times; 10&#x207B;&#x2078;), one of which includes a copy number variant near GPRC5B. Some loci (at MC4R, POMC, SH2B1 and BDNF) map near key hypothalamic regulators of energy balance, and one of these loci is near GIPR, an incretin receptor. Furthermore, genes in other newly associated loci may provide new insights into human body weight regulation.</p>
<p class="grants">Funded by: British Heart Foundation; Cancer Research UK; Chief Scientist Office: <span class="grant">CZB/4/710</span>; Department of Health; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401527</span>, <span class="grant">G0501184</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0900554</span>, <span class="grant">G9521010</span>, <span class="grant">G9824984</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U127561128</span>, <span class="grant">MC_U137686857</span>; NCI NIH HHS: <span class="grant">CA047988</span>, <span class="grant">CA49449</span>, <span class="grant">CA50385</span>, <span class="grant">CA65725</span>, <span class="grant">CA67262</span>, <span class="grant">CA87969</span>, <span class="grant">U01-CA098233</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>, <span class="grant">U54-RR020278</span>, <span class="grant">UL1-RR025005</span>; NHGRI NIH HHS: <span class="grant">HG002651</span>, <span class="grant">N01-HG-65403</span>, <span class="grant">T32 HG000040</span>, <span class="grant">T32 HG000040-17</span>, <span class="grant">T32-HG00040</span>, <span class="grant">U01-HG004399</span>, <span class="grant">U01-HG004402</span>, <span class="grant">Z01-HG000024</span>; NHLBI NIH HHS: <span class="grant">HL084729</span>, <span class="grant">HL71981</span>, <span class="grant">K99-HL094535</span>, <span class="grant">N01-HC15103</span>, <span class="grant">N01-HC25195</span>, <span class="grant">N01-HC35129</span>, <span class="grant">N01-HC45133</span>, <span class="grant">N01-HC55015</span>, <span class="grant">N01-HC55016</span>, <span class="grant">N01-HC55018</span>, <span class="grant">N01-HC55019</span>, <span class="grant">N01-HC55020</span>, <span class="grant">N01-HC55022</span>, <span class="grant">N01-HC55222</span>, <span class="grant">N01-HC75150</span>, <span class="grant">N01-HC85079</span>, <span class="grant">N01-HC85080</span>, <span class="grant">N01-HC85081</span>, <span class="grant">N01-HC85082</span>, <span class="grant">N01-HC85083</span>, <span class="grant">N01-HC85084</span>, <span class="grant">N01-HC85085</span>, <span class="grant">N01-HC85086</span>, <span class="grant">N01-N01HC-55021</span>, <span class="grant">N02-HL64278</span>, <span class="grant">R01 HL071981</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01-HL086694</span>, <span class="grant">R01-HL087641</span>, <span class="grant">R01-HL087647</span>, <span class="grant">R01-HL087652</span>, <span class="grant">R01-HL087676</span>, <span class="grant">R01-HL087679</span>, <span class="grant">R01-HL087700</span>, <span class="grant">R01-HL088119</span>, <span class="grant">R01-HL59367</span>, <span class="grant">U01 HL054527</span>, <span class="grant">U01-HL080295</span>, <span class="grant">U01-HL084756</span>, <span class="grant">U01-HL72515</span>; NIA NIH HHS: <span class="grant">N01-AG12100</span>, <span class="grant">N01-AG12109</span>, <span class="grant">R01-AG031890</span>; NIAAA NIH HHS: <span class="grant">AA014041</span>, <span class="grant">AA07535</span>, <span class="grant">AA10248</span>, <span class="grant">AA13320</span>, <span class="grant">AA13321</span>, <span class="grant">AA13326</span>, <span class="grant">K05 AA017688</span>; NIAMS NIH HHS: <span class="grant">K08 AR055688</span>, <span class="grant">K08 AR055688-03</span>, <span class="grant">K08 AR055688-04</span>; NIDA NIH HHS: <span class="grant">DA12854</span>, <span class="grant">R01 DA012854</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK063491</span>, <span class="grant">DK072193</span>, <span class="grant">DK46200</span>, <span class="grant">DK58845</span>, <span class="grant">F32 DK079466</span>, <span class="grant">F32 DK079466-01</span>, <span class="grant">K23 DK080145</span>, <span class="grant">K23 DK080145-01</span>, <span class="grant">K23-DK080145</span>, <span class="grant">P30 DK072488</span>, <span class="grant">P30-DK072488</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK072193-05</span>, <span class="grant">R01-DK073490</span>, <span class="grant">R01-DK075787</span>, <span class="grant">R01DK068336</span>, <span class="grant">R01DK075681</span>, <span class="grant">U01 DK062370</span>, <span class="grant">U01 DK062370-08</span>, <span class="grant">U01-DK062418</span>; NIGMS NIH HHS: <span class="grant">T32 GM074905</span>, <span class="grant">U01-GM074518</span>; NIMH NIH HHS: <span class="grant">MH084698</span>, <span class="grant">R01-MH59160</span>, <span class="grant">R01-MH59565</span>, <span class="grant">R01-MH59566</span>, <span class="grant">R01-MH59571</span>, <span class="grant">R01-MH59586</span>, <span class="grant">R01-MH59587</span>, <span class="grant">R01-MH59588</span>, <span class="grant">R01-MH60870</span>, <span class="grant">R01-MH60879</span>, <span class="grant">R01-MH61675</span>, <span class="grant">R01-MH63706</span>, <span class="grant">R01-MH67257</span>, <span class="grant">R01-MH79469</span>, <span class="grant">R01-MH79470</span>, <span class="grant">R01-MH81800</span>, <span class="grant">RL1-MH083268</span>; PHS HHS: <span class="grant">263-MA-410953</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">068545</span>, <span class="grant">072960</span>, <span class="grant">075491</span>, <span class="grant">076113</span>, <span class="grant">077016</span>, <span class="grant">079557</span>, <span class="grant">079895</span>, <span class="grant">081682</span>, <span class="grant">083270</span>, <span class="grant">085301</span>, <span class="grant">086596</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">11</span>;<span class="pagerange">937-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20935630">20935630</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3014648">3014648</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.686">10.1038/ng.686</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The ear</h4>
<p class="authors"><span class="author">Spiden,S.L.</span> and <span class="author">Steel,K.P.;</span></p>
<p class="publication"><span class="title">Embryos, Genes and Birth Defects</span>&nbsp;<span class="year">2010</span>;<span class="number">Chapter 10</span>;<span class="pagerange">231-62</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1002%2F9780470090121.ch10">10.1002/9780470090121.ch10</a>; URL: <a title="http://onlinelibrary.wiley.com/doi/10.1002/9780470090121.ch10/summary" href="http://onlinelibrary.wiley.com/doi/10.1002/9780470090121.ch10/summary">http://onlinelibrary.wiley.com/doi/<span class="print-hide">...</span><span class="print-show">10.1002/9780470090121.ch10/su</span>mmary</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pooled analysis indicates that the GSTT1 deletion, GSTM1 deletion, and GSTP1 Ile105Val polymorphisms do not modify breast cancer risk in BRCA1 and BRCA2 mutation carriers.</h4>
<p class="authors"><span class="author">Spurdle AB</span>, <span class="author">Fahey P</span>, <span class="author">Chen X</span>, <span class="author">McGuffog L</span>, <span class="author">kConFab</span>, <span class="author">Easton D</span>, <span class="author">Peock S</span>, <span class="author">Cook M</span>, <span class="author">EMBRACE</span>, <span class="author">Simard J</span>, <span class="author">INHERIT</span>, <span class="author">Rebbeck TR</span>, <span class="author">MAGIC</span>, <span class="author">Antoniou AC</span> and <span class="author">Chenevix-Trench G</span></p>
<p class="affiliation">Division of Genetics and Population Health, Queensland Institute of Medical Research, 300 Herston Rd, Herston 4006, Australia.</p><p class="abstract">The GSTP1, GSTM1, and GSTT1 detoxification genes all have functional polymorphisms that are common in the general population. A single study of 320 BRCA1/2 carriers previously assessed their effect in BRCA1 or BRCA2 mutation carriers. This study showed no evidence for altered risk of breast cancer for individuals with the GSTT1 and GSTM1 deletion variants, but did report that the GSTP1 Ile105Val (rs1695) variant was associated with increased breast cancer risk in carriers. We investigated the association between these three GST polymorphisms and breast cancer risk using existing data from 718 women BRCA1 and BRCA2 mutation carriers from Australia, the UK, Canada, and the USA. Data were analyzed within a proportional hazards framework using Cox regression. There was no evidence to show that any of the polymorphisms modified disease risk for BRCA1 or BRCA2 carriers, and there was no evidence for heterogeneity between sites. These results support the need for replication studies to confirm or refute hypothesis-generating studies.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Cancer Research UK: <span class="grant">10118</span>, <span class="grant">11022</span>, <span class="grant">11174</span>, <span class="grant">C1287/A10118</span>, <span class="grant">C1287/A8874</span>; NCI NIH HHS: <span class="grant">R01 CA083855</span>, <span class="grant">R01 CA083855-01</span>, <span class="grant">R01 CA083855-02</span>, <span class="grant">R01 CA083855-03</span>, <span class="grant">R01 CA083855-04</span>, <span class="grant">R01 CA083855-05</span>, <span class="grant">R01 CA083855-06</span>, <span class="grant">R01 CA083855-07</span>, <span class="grant">R01 CA083855-08</span>, <span class="grant">R01 CA083855-09</span>, <span class="grant">R01 CA083855-10</span>, <span class="grant">R01 CA083855-11</span>, <span class="grant">R01 CA102776</span>, <span class="grant">R01 CA102776-01A1</span>, <span class="grant">R01 CA102776-02</span>, <span class="grant">R01 CA102776-03</span>, <span class="grant">R01 CA102776-04</span>, <span class="grant">R01 CA102776-05</span>, <span class="grant">R01-CA083855</span>, <span class="grant">R01-CA102776</span></p>
<p class="publication"><span class="title">Breast cancer research and treatment</span>&nbsp;<span class="year">2010</span>;<span class="volume">122</span>;<span class="number">1</span>;<span class="pagerange">281-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19921428">19921428</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3074275">3074275</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10549-009-0601-0">10.1007/s10549-009-0601-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Low-density lipoprotein receptor-related protein 5 polymorphisms are associated with bone mineral density in Greek postmenopausal women: an interaction with calcium intake.</h4>
<p class="authors"><span class="author">Stathopoulou MG</span>, <span class="author">Dedoussis GV</span>, <span class="author">Trovas G</span>, <span class="author">Katsalira A</span>, <span class="author">Hammond N</span>, <span class="author">Deloukas P</span> and <span class="author">Lyritis GP</span></p>
<p class="affiliation">Department of Dietetics and Nutrition, Harokopio University, 17671 Athens, Greece.</p><p class="abstract">The low-density lipoprotein receptor-related protein 5 (LRP5) has been shown to play a significant role in bone biology. This study aimed to assess the association of four common polymorphisms of the LRP5 gene with bone mineral density (BMD) and possible genexcalcium intake interactions in Greek postmenopausal women. For this observational cross-sectional association study, healthy postmenopausal women (N=578) were recruited (between December 2006 and January 2008) and genotyped for four polymorphisms (rs1784235, rs491347, rs4988321, and rs4988330) in the LRP5 gene. Measurements of BMD were performed and detailed medical, dietary, and anthropometric data were recorded. Student t tests and multiple linear regression models were applied after controlling for potential covariates (ie, age, weight, height, and calcium intake). None of the polymorphisms was associated with the presence of osteoporosis, fractures, and hip BMD. All polymorphisms were associated with unadjusted spine BMD, with the exception of rs4988330. Only rs4988321 was associated with adjusted spine BMD, where the presence of the A allele was associated with significantly lower spine BMD compared with the GG genotype (P=0.002). An interaction of the rs4988321 polymorphism with calcium intake (P=0.016) was found. The carriers of the A allele demonstrated significantly lower spine BMD compared to GG homozygotes (P=0.001) only in the lowest calcium intake group (&lt;680 mg/day), whereas in the highest calcium intake group no differences were found in BMD between genotypes. These findings demonstrate that both rs4988321 polymorphism and its interaction with calcium intake are associated with BMD, whereas higher calcium intake was shown to decrease the negative effect of this polymorphism on BMD.</p>
<p class="publication"><span class="title">Journal of the American Dietetic Association</span>&nbsp;<span class="year">2010</span>;<span class="volume">110</span>;<span class="number">7</span>;<span class="pagerange">1078-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20630166">20630166</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jada.2010.04.007">10.1016/j.jada.2010.04.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Bayesian framework to account for complex non-genetic factors in gene expression levels greatly increases power in eQTL studies.</h4>
<p class="authors"><span class="author">Stegle O</span>, <span class="author">Parts L</span>, <span class="author">Durbin R</span> and <span class="author">Winn J</span></p>
<p class="affiliation">Max Planck Institutes T&uuml;bingen, T&uuml;bingen, Germany. <a href="mailto:%6F%6C%69%76%65%72.%73%74%65%67%6C%65%40%74%75%65%62%69%6E%67%65%6E.%6D%70%67.%64%65">&#111;&#108;&#105;&#118;&#101;&#114;&#46;&#115;&#116;&#101;&#103;&#108;&#101;&#64;&#116;&#117;&#101;&#98;&#105;&#110;&#103;&#101;&#110;&#46;&#109;&#112;&#103;&#46;&#100;&#101;</a></p><p class="abstract">Gene expression measurements are influenced by a wide range of factors, such as the state of the cell, experimental conditions and variants in the sequence of regulatory regions. To understand the effect of a variable of interest, such as the genotype of a locus, it is important to account for variation that is due to confounding causes. Here, we present VBQTL, a probabilistic approach for mapping expression quantitative trait loci (eQTLs) that jointly models contributions from genotype as well as known and hidden confounding factors. VBQTL is implemented within an efficient and flexible inference framework, making it fast and tractable on large-scale problems. We compare the performance of VBQTL with alternative methods for dealing with confounding variability on eQTL mapping datasets from simulations, yeast, mouse, and human. Employing Bayesian complexity control and joint modelling is shown to result in more precise estimates of the contribution of different confounding factors resulting in additional associations to measured transcript levels compared to alternative approaches. We present a threefold larger collection of cis eQTLs than previously found in a whole-genome eQTL scan of an outbred human population. Altogether, 27% of the tested probes show a significant genetic association in cis, and we validate that the additional eQTLs are likely to be real by replicating them in different sets of individuals. Our method is the next step in the analysis of high-dimensional phenotype data, and its application has revealed insights into genetic regulation of gene expression by demonstrating more abundant cis-acting eQTLs in human than previously shown. Our software is freely available online at http://www.sanger.ac.uk/resources/software/peer/.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077192/Z/05/Z</span></p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">e1000770</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20463871">20463871</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2865505">2865505</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.1000770">10.1371/journal.pcbi.1000770</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide end-sequenced BAC resources for the NOD/MrkTac() and NOD/ShiLtJ() mouse genomes.</h4>
<p class="authors"><span class="author">Steward CA</span>, <span class="author">Humphray S</span>, <span class="author">Plumb B</span>, <span class="author">Jones MC</span>, <span class="author">Quail MA</span>, <span class="author">Rice S</span>, <span class="author">Cox T</span>, <span class="author">Davies R</span>, <span class="author">Bonfield J</span>, <span class="author">Keane TM</span>, <span class="author">Nefedov M</span>, <span class="author">de Jong PJ</span>, <span class="author">Lyons P</span>, <span class="author">Wicker L</span>, <span class="author">Todd J</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Gulban O</span>, <span class="author">Danska J</span>, <span class="author">Harrow J</span>, <span class="author">Hubbard T</span>, <span class="author">Rogers J</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%63%61%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Non-obese diabetic (NOD) mice spontaneously develop type 1 diabetes (T1D) due to the progressive loss of insulin-secreting beta-cells by an autoimmune driven process. NOD mice represent a valuable tool for studying the genetics of T1D and for evaluating therapeutic interventions. Here we describe the development and characterization by end-sequencing of bacterial artificial chromosome (BAC) libraries derived from NOD/MrkTac (DIL NOD) and NOD/ShiLtJ (CHORI-29), two commonly used NOD substrains. The DIL NOD library is composed of 196,032 BACs and the CHORI-29 library is composed of 110,976 BACs. The average depth of genome coverage of the DIL NOD library, estimated from mapping the BAC end-sequences to the reference mouse genome sequence, was 7.1-fold across the autosomes and 6.6-fold across the X chromosome. Clones from this library have an average insert size of 150 kb and map to over 95.6% of the reference mouse genome assembly (NCBIm37), covering 98.8% of Ensembl mouse genes. By the same metric, the CHORI-29 library has an average depth over the autosomes of 5.0-fold and 2.8-fold coverage of the X chromosome, the reduced X chromosome coverage being due to the use of a male donor for this library. Clones from this library have an average insert size of 205 kb and map to 93.9% of the reference mouse genome assembly, covering 95.7% of Ensembl genes. We have identified and validated 191,841 single nucleotide polymorphisms (SNPs) for DIL NOD and 114,380 SNPs for CHORI-29. In total we generated 229,736,133 bp of sequence for the DIL NOD and 121,963,211 bp for the CHORI-29. These BAC libraries represent a powerful resource for functional studies, such as gene targeting in NOD embryonic stem (ES) cell lines, and for sequencing and mapping experiments.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">95</span>;<span class="number">2</span>;<span class="pagerange">105-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19909804">19909804</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2824108">2824108</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ygeno.2009.10.004">10.1016/j.ygeno.2009.10.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple genetic mechanisms lead to loss of functional TbAT1 expression in drug-resistant trypanosomes.</h4>
<p class="authors"><span class="author">Stewart ML</span>, <span class="author">Burchmore RJ</span>, <span class="author">Clucas C</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Brooks K</span>, <span class="author">Tait A</span>, <span class="author">Macleod A</span>, <span class="author">Turner CM</span>, <span class="author">De Koning HP</span>, <span class="author">Wong PE</span> and <span class="author">Barrett MP</span></p>
<p class="affiliation">Division of Infection &amp; Immunity, Faculty of Biomedical and Life Sciences, Glasgow Biomedical Research Centre, University of Glasgow, Glasgow G12 8TA, United Kingdom.</p><p class="abstract">The P2 aminopurine transporter, encoded by TbAT1 in African trypanosomes in the Trypanosoma brucei group, carries melaminophenyl arsenical and diamidine drugs into these parasites. Loss of this transporter contributes to drug resistance. We identified the genomic location of TbAT1 to be in the subtelomeric region of chromosome 5 and determined the status of the TbAT1 gene in two trypanosome lines selected for resistance to the melaminophenyl arsenical, melarsamine hydrochloride (Cymelarsan), and in a Trypanosoma equiperdum clone selected for resistance to the diamidine, diminazene aceturate. In the Trypanosoma brucei gambiense STIB 386 melarsamine hydrochloride-resistant line, TbAT1 is deleted, while in the Trypanosoma brucei brucei STIB 247 melarsamine hydrochloride-resistant and T. equiperdum diminazene-resistant lines, TbAT1 is present, but expression at the RNA level is no longer detectable. Further characterization of TbAT1 in T. equiperdum revealed that a loss of heterozygosity at the TbAT1 locus accompanied loss of expression and that P2-mediated uptake of [(3)H]diminazene is lost in drug-resistant T. equiperdum. Adenine-inhibitable adenosine uptake is still detectable in a DeltaTbat1 T. b. brucei mutant, although at a greatly reduced capacity compared to that of the wild type, indicating that an additional adenine-inhibitable adenosine permease, distinct from P2, is present in these cells.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">C13486</span>; Wellcome Trust: <span class="grant">074732</span>, <span class="grant">079703</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2010</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">336-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19966032" rel="external">19966032</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2823006" rel="external">2823006</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.00200-09" rel="external">10.1128/EC.00200-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sexual dimorphism in olfactory signaling.</h4>
<p class="authors"><span class="author">Stowers L</span> and <span class="author">Logan DW</span></p>
<p class="affiliation">Department of Cell Biology, The Scripps Research Institute, ICND222, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA. <a href="mailto:%73%74%6F%77%65%72%73%40%73%63%72%69%70%70%73.%65%64%75">&#115;&#116;&#111;&#119;&#101;&#114;&#115;&#64;&#115;&#99;&#114;&#105;&#112;&#112;&#115;&#46;&#101;&#100;&#117;</a></p><p class="abstract">What makes males and females behave differently? Although genetic master-regulators commonly underlie physical differences, sexually dimorphic behavior is additionally influenced by sensory input such as olfactory cues. Olfaction requires both ligands for signaling and sensory neural circuits for detection. Specialized subsets of each interact to generate gender-dimorphic behavior. It has long been accepted that males and females emit sex-specific odor compounds that function as pheromones to promote stereotypic behavior. Significant advances have now been made in purifying and isolating several of these sex-specific olfactory ligands. In contrast, the neural mechanisms that enable a gender-dimorphic response to these odors remain largely unknown. However, first progress has been made in identifying components of sexually dimorphic olfactory circuits in both Drosophila and the mouse.</p>
<p class="grants">Funded by: NIDCD NIH HHS: <span class="grant">R01 DC006885-05</span>, <span class="grant">R01 DC009413-03</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in neurobiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">20</span>;<span class="number">6</span>;<span class="pagerange">770-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20833534" rel="external">20833534</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3005963" rel="external">3005963</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.conb.2010.08.015" rel="external">10.1016/j.conb.2010.08.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Leena Peltonen 1952-2010 Obituary</h4>
<p class="authors"><span class="author">STRATTON MR</span>, <span class="author">Lander E</span></p>
<p class="publication"><span class="title">Cell.</span> <span class="year">2010</span>;<span class="volume">141</span>;<span class="pagerange">208-9</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systematic analysis of off-target effects in an RNAi screen reveals microRNAs affecting sensitivity to TRAIL-induced apoptosis.</h4>
<p class="authors"><span class="author">Sudbery I</span>, <span class="author">Enright AJ</span>, <span class="author">Fraser AG</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Work performed at: Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: RNA inhibition by siRNAs is a frequently used approach to identify genes required for specific biological processes. However RNAi screening using siRNAs is hampered by non-specific or off target effects of the siRNAs, making it difficult to separate genuine hits from false positives. It is thought that many of the off-target effects seen in RNAi experiments are due to siRNAs acting as microRNAs (miRNAs), causing a reduction in gene expression of unintended targets via matches to the 6 or 7 nt &#39;seed&#39; sequence. We have conducted a careful examination of off-target effects during an siRNA screen for novel regulators of the TRAIL apoptosis induction pathway(s).</p><p class="abstract"><strong>Results</strong>: We identified 3 hexamers and 3 heptamer seed sequences that appeared multiple times in the top twenty siRNAs in the TRAIL apoptosis screen. Using a novel statistical enrichment approach, we systematically identified a further 17 hexamer and 13 heptamer seed sequences enriched in high scoring siRNAs. The presence of one of these seeds sequences (which could explain 6 of 8 confirmed off-target effects) is sufficient to elicit a phenotype. Three of these seed sequences appear in the human miRNAs miR-26a, miR-145 and miR-384. Transfection of mimics of these miRNAs protects several cell types from TRAIL-induced cell death.</p><p class="abstract"><strong>Conclusions</strong>: We have demonstrated a role for miR-26a, miR-145 and miR-26a in TRAIL-induced apoptosis. Further these results show that RNAi screening enriches for siRNAs with relevant off-target effects. Some of these effects can be identified by the over-representation of certain seed sequences in high-scoring siRNAs and we demonstrate the usefulness of such systematic analysis of enriched seed sequences.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">175</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20230625" rel="external">20230625</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2996961" rel="external">2996961</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-175" rel="external">10.1186/1471-2164-11-175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two novel Src homology 2 domain proteins interact to regulate dictyostelium gene expression during growth and early development.</h4>
<p class="authors"><span class="author">Sugden C</span>, <span class="author">Ross S</span>, <span class="author">Bloomfield G</span>, <span class="author">Ivens A</span>, <span class="author">Skelton J</span>, <span class="author">Mueller-Taubenberger A</span> and <span class="author">Williams JG</span></p>
<p class="affiliation">School of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, Scotland, United Kingdom.</p><p class="abstract">There are 13 Dictyostelium Src homology 2 (SH2) domain proteins, almost 10-fold fewer than in mammals, and only three are functionally unassigned. One of these, LrrB, contains a novel combination of protein interaction domains: an SH2 domain and a leucine-rich repeat domain. Growth and early development appear normal in the mutant, but expression profiling reveals that three genes active at these stages are greatly underexpressed: the ttdA metallohydrolase, the abcG10 small molecule transporter, and the cinB esterase. In contrast, the multigene family encoding the lectin discoidin 1 is overexpressed in the disruptant strain. LrrB binds to 14-3-3 protein, and the level of binding is highest during growth and decreases during early development. Comparative tandem affinity purification tagging shows that LrrB also interacts, via its SH2 domain and in a tyrosine phosphorylation-dependent manner, with two novel proteins: CldA and CldB. Both of these proteins contain a Clu domain, a &gt;200-amino acid sequence present within highly conserved eukaryotic proteins required for correct mitochondrial dispersal. A functional interaction of LrrB with CldA is supported by the fact that a cldA disruptant mutant also underexpresses ttdA, abcG10, and cinB. Significantly, CldA is itself one of the three functionally unassigned SH2 domain proteins. Thus, just as in metazoa, but on a vastly reduced numerical scale, an interacting network of SH2 domain proteins regulates specific Dictyostelium gene expression.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C500428/1</span>; Wellcome Trust: <span class="grant">06724</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2010</span>;<span class="volume">285</span>;<span class="number">30</span>;<span class="pagerange">22927-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20457612" rel="external">20457612</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2906285" rel="external">2906285</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M110.139733" rel="external">10.1074/jbc.M110.139733</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Founder population-specific HapMap panel increases power in GWA studies through improved imputation accuracy and CNV tagging.</h4>
<p class="authors"><span class="author">Surakka I</span>, <span class="author">Kristiansson K</span>, <span class="author">Anttila V</span>, <span class="author">Inouye M</span>, <span class="author">Barnes C</span>, <span class="author">Moutsianas L</span>, <span class="author">Salomaa V</span>, <span class="author">Daly M</span>, <span class="author">Palotie A</span>, <span class="author">Peltonen L</span> and <span class="author">Ripatti S</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, FIMM, University of Helsinki, FI-00014 Helsinki, Finland.</p><p class="abstract">The combining of genome-wide association (GWA) data across populations represents a major challenge for massive global meta-analyses. Genotype imputation using densely genotyped reference samples facilitates the combination of data across different genotyping platforms. HapMap data is typically used as a reference for single nucleotide polymorphism (SNP) imputation and tagging copy number polymorphisms (CNPs). However, the advantage of having population-specific reference panels for founder populations has not been evaluated. We looked at the properties and impact of adding 81 individuals from a founder population to HapMap3 reference data on imputation quality, CNP tagging, and power to detect association in simulations and in an independent cohort of 2138 individuals. The gain in SNP imputation accuracy was highest among low-frequency markers (minor allele frequency [MAF] &lt; 5%), for which adding the population-specific samples to the reference set increased the median R(2) between imputed and genotyped SNPs from 0.90 to 0.94. Accuracy also increased in regions with high recombination rates. Similarly, a reference set with population-specific extension facilitated the identification of better tag-SNPs for a subset of CNPs; for 4% of CNPs the R(2) between SNP genotypes and CNP intensity in the independent population cohort was at least twice as high as without the extension. We conclude that even a relatively small population-specific reference set yields considerable benefits in SNP imputation, CNP tagging accuracy, and the power to detect associations in founder populations and population isolates in particular.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT089061/Z/09/Z</span>, <span class="grant">WT089062/Z/09/Z</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2010</span>;<span class="volume">20</span>;<span class="number">10</span>;<span class="pagerange">1344-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20810666">20810666</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2945183">2945183</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.106534.110">10.1101/gr.106534.110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two nonrecombining sympatric forms of the human malaria parasite Plasmodium ovale occur globally.</h4>
<p class="authors"><span class="author">Sutherland CJ</span>, <span class="author">Tanomsing N</span>, <span class="author">Nolder D</span>, <span class="author">Oguike M</span>, <span class="author">Jennison C</span>, <span class="author">Pukrittayakamee S</span>, <span class="author">Dolecek C</span>, <span class="author">Hien TT</span>, <span class="author">do Ros&aacute;rio VE</span>, <span class="author">Arez AP</span>, <span class="author">Pinto J</span>, <span class="author">Michon P</span>, <span class="author">Escalante AA</span>, <span class="author">Nosten F</span>, <span class="author">Burke M</span>, <span class="author">Lee R</span>, <span class="author">Blaze M</span>, <span class="author">Otto TD</span>, <span class="author">Barnwell JW</span>, <span class="author">Pain A</span>, <span class="author">Williams J</span>, <span class="author">White NJ</span>, <span class="author">Day NP</span>, <span class="author">Snounou G</span>, <span class="author">Lockhart PJ</span>, <span class="author">Chiodini PL</span>, <span class="author">Imwong M</span> and <span class="author">Polley SD</span></p>
<p class="affiliation">Health Protection Agency Malaria Reference Laboratory, Immunology Unit, London School of Hygiene and Tropical Medicine, London, United Kingdom. <a href="mailto:%63%6F%6C%69%6E.%73%75%74%68%65%72%6C%61%6E%64%40%6C%73%68%74%6D.%61%63.%75%6B">&#99;&#111;&#108;&#105;&#110;&#46;&#115;&#117;&#116;&#104;&#101;&#114;&#108;&#97;&#110;&#100;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Malaria in humans is caused by apicomplexan parasites belonging to 5 species of the genus Plasmodium. Infections with Plasmodium ovale are widely distributed but rarely investigated, and the resulting burden of disease is not known. Dimorphism in defined genes has led to P. ovale parasites being divided into classic and variant types. We hypothesized that these dimorphs represent distinct parasite species.</p><p class="abstract"><strong>Methods</strong>: Multilocus sequence analysis of 6 genetic characters was carried out among 55 isolates from 12 African and 3 Asia-Pacific countries.</p><p class="abstract"><strong>Results</strong>: Each genetic character displayed complete dimorphism and segregated perfectly between the 2 types. Both types were identified in samples from Ghana, Nigeria, S&atilde;o Tom&eacute;, Sierra Leone, and Uganda and have been described previously in Myanmar. Splitting of the 2 lineages is estimated to have occurred between 1.0 and 3.5 million years ago in hominid hosts.</p><p class="abstract"><strong>Conclusions</strong>: We propose that P. ovale comprises 2 nonrecombining species that are sympatric in Africa and Asia. We speculate on possible scenarios that could have led to this speciation. Furthermore, the relatively high frequency of imported cases of symptomatic P. ovale infection in the United Kingdom suggests that the morbidity caused by ovale malaria has been underestimated.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">093956</span></p>
<p class="publication"><span class="title">The Journal of infectious diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">201</span>;<span class="number">10</span>;<span class="pagerange">1544-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20380562">20380562</a>; DOI: <a href="http://dx.doi.org/10.1086%2F652240">10.1086/652240</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Predictive biomarker discovery through the parallel integration of clinical trial and functional genomics datasets.</h4>
<p class="authors"><span class="author">Swanton C</span>, <span class="author">Larkin JM</span>, <span class="author">Gerlinger M</span>, <span class="author">Eklund AC</span>, <span class="author">Howell M</span>, <span class="author">Stamp G</span>, <span class="author">Downward J</span>, <span class="author">Gore M</span>, <span class="author">Futreal PA</span>, <span class="author">Escudier B</span>, <span class="author">Andre F</span>, <span class="author">Albiges L</span>, <span class="author">Beuselinck B</span>, <span class="author">Oudard S</span>, <span class="author">Hoffmann J</span>, <span class="author">Gyorffy B</span>, <span class="author">Torrance CJ</span>, <span class="author">Boehme KA</span>, <span class="author">Volkmer H</span>, <span class="author">Toschi L</span>, <span class="author">Nicke B</span>, <span class="author">Beck M</span> and <span class="author">Szallasi Z</span></p>
<p class="affiliation">Cancer Research UK London Research Institute, 44 Lincoln&#39;s Inn Fields, London, WC2A 3PX, UK. <a href="mailto:%63%68%61%72%6C%65%73.%73%77%61%6E%74%6F%6E%40%63%61%6E%63%65%72.%6F%72%67.%75%6B">&#99;&#104;&#97;&#114;&#108;&#101;&#115;&#46;&#115;&#119;&#97;&#110;&#116;&#111;&#110;&#64;&#99;&#97;&#110;&#99;&#101;&#114;&#46;&#111;&#114;&#103;&#46;&#117;&#107;</a>.</p><p class="abstract">The European Union multi-disciplinary Personalised RNA interference to Enhance the Delivery of Individualised Cytotoxic and Targeted therapeutics (PREDICT) consortium has recently initiated a framework to accelerate the development of predictive biomarkers of individual patient response to anti-cancer agents. The consortium focuses on the identification of reliable predictive biomarkers to approved agents with anti-angiogenic activity for which no reliable predictive biomarkers exist: sunitinib, a multi-targeted tyrosine kinase inhibitor and everolimus, a mammalian target of rapamycin (mTOR) pathway inhibitor. Through the analysis of tumor tissue derived from pre-operative renal cell carcinoma (RCC) clinical trials, the PREDICT consortium will use established and novel methods to integrate comprehensive tumor-derived genomic data with personalized tumor-derived small hairpin RNA and high-throughput small interfering RNA screens to identify and validate functionally important genomic or transcriptomic predictive biomarkers of individual drug response in patients. PREDICT&#39;s approach to predictive biomarker discovery differs from conventional associative learning approaches, which can be susceptible to the detection of chance associations that lead to overestimation of true clinical accuracy. These methods will identify molecular pathways important for survival and growth of RCC cells and particular targets suitable for therapeutic development. Importantly, our results may enable individualized treatment of RCC, reducing ineffective therapy in drug-resistant disease, leading to improved quality of life and higher cost efficiency, which in turn should broaden patient access to beneficial therapeutics, thereby enhancing clinical outcome and cancer survival. The consortium will also establish and consolidate a European network providing the technological and clinical platform for large-scale functional genomic biomarker discovery. Here we review our current understanding of molecular mechanisms driving resistance to anti-angiogenesis agents, the current limitations of laboratory and clinical trial strategies and how the PREDICT consortium will endeavor to identify a new generation of predictive biomarkers.</p>
<p class="publication"><span class="title">Genome medicine</span>&nbsp;<span class="year">2010</span>;<span class="volume">2</span>;<span class="number">8</span>;<span class="pagerange">53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20701793" rel="external">20701793</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2945010" rel="external">2945010</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgm174" rel="external">10.1186/gm174</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants in the ATP2B1 gene are associated with susceptibility to hypertension: the Japanese Millennium Genome Project.</h4>
<p class="authors"><span class="author">Tabara Y</span>, <span class="author">Kohara K</span>, <span class="author">Kita Y</span>, <span class="author">Hirawa N</span>, <span class="author">Katsuya T</span>, <span class="author">Ohkubo T</span>, <span class="author">Hiura Y</span>, <span class="author">Tajima A</span>, <span class="author">Morisaki T</span>, <span class="author">Miyata T</span>, <span class="author">Nakayama T</span>, <span class="author">Takashima N</span>, <span class="author">Nakura J</span>, <span class="author">Kawamoto R</span>, <span class="author">Takahashi N</span>, <span class="author">Hata A</span>, <span class="author">Soma M</span>, <span class="author">Imai Y</span>, <span class="author">Kokubo Y</span>, <span class="author">Okamura T</span>, <span class="author">Tomoike H</span>, <span class="author">Iwai N</span>, <span class="author">Ogihara T</span>, <span class="author">Inoue I</span>, <span class="author">Tokunaga K</span>, <span class="author">Johnson T</span>, <span class="author">Caulfield M</span>, <span class="author">Munroe P</span>, <span class="author">Global Blood Pressure Genetics Consortium</span>, <span class="author">Umemura S</span>, <span class="author">Ueshima H</span> and <span class="author">Miki T</span></p>
<p class="affiliation">Department of Basic Medical Research and Education, Ehime University Graduate School of Medicine, Toon-City, Ehime, Japan. <a href="mailto:%74%61%62%61%72%61%40%6D.%65%68%69%6D%65-%75.%61%63.%6A%70">&#116;&#97;&#98;&#97;&#114;&#97;&#64;&#109;&#46;&#101;&#104;&#105;&#109;&#101;&#45;&#117;&#46;&#97;&#99;&#46;&#106;&#112;</a></p><p class="abstract">Hypertension is one of the most common complex genetic disorders. We have described previously 38 single nucleotide polymorphisms (SNPs) with suggestive association with hypertension in Japanese individuals. In this study we extend our previous findings by analyzing a large sample of Japanese individuals (n=14 105) for the most associated SNPs. We also conducted replication analyses in Japanese of susceptibility loci for hypertension identified recently from genome-wide association studies of European ancestries. Association analysis revealed significant association of the ATP2B1 rs2070759 polymorphism with hypertension (P=5.3&times;10(-5); allelic odds ratio: 1.17 [95% CI: 1.09 to 1.26]). Additional SNPs in ATP2B1 were subsequently genotyped, and the most significant association was with rs11105378 (odds ratio: 1.31 [95% CI: 1.21 to 1.42]; P=4.1&times;10(-11)). Association of rs11105378 with hypertension was cross-validated by replication analysis with the Global Blood Pressure Genetics consortium data set (odds ratio: 1.13 [95% CI: 1.05 to 1.21]; P=5.9&times;10(-4)). Mean adjusted systolic blood pressure was highly significantly associated with the same SNP in a meta-analysis with individuals of European descent (P=1.4&times;10(-18)). ATP2B1 mRNA expression levels in umbilical artery smooth muscle cells were found to be significantly different among rs11105378 genotypes. Seven SNPs discovered in published genome-wide association studies were also genotyped in the Japanese population. In the combined analysis with replicated 3 genes, FGF5 rs1458038, CYP17A1, rs1004467, and CSK rs1378942, odds ratio of the highest risk group was 2.27 (95% CI: 1.65 to 3.12; P=4.6&times;10(-7)) compared with the lower risk group. In summary, this study confirmed common genetic variation in ATP2B1, as well as FGF5, CYP17A1, and CSK, to be associated with blood pressure levels and risk of hypertension.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0400874</span>, <span class="grant">G0401527</span>, <span class="grant">G0801056</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_UP_A100_1003</span></p>
<p class="publication"><span class="title">Hypertension</span>&nbsp;<span class="year">2010</span>;<span class="volume">56</span>;<span class="number">5</span>;<span class="pagerange">973-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20921432">20921432</a>; DOI: <a href="http://dx.doi.org/10.1161%2FHYPERTENSIONAHA.110.153429">10.1161/HYPERTENSIONAHA.110.153429</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rec8-containing cohesin maintains bivalents without turnover during the growing phase of mouse oocytes.</h4>
<p class="authors"><span class="author">Tachibana-Konwalski K</span>, <span class="author">Godwin J</span>, <span class="author">van der Weyden L</span>, <span class="author">Champion L</span>, <span class="author">Kudo NR</span>, <span class="author">Adams DJ</span> and <span class="author">Nasmyth K</span></p>
<p class="affiliation">Department of Biochemistry, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">During female meiosis, bivalent chromosomes are thought to be held together from birth until ovulation by sister chromatid cohesion mediated by cohesin complexes whose ring structure depends on kleisin subunits, either Rec8 or Scc1. Because cohesion is established at DNA replication in the embryo, its maintenance for such a long time may require cohesin turnover. To address whether Rec8- or Scc1-containing cohesin holds bivalents together and whether it turns over, we created mice whose kleisin subunits can be cleaved by TEV protease. We show by microinjection experiments and confocal live-cell imaging that Rec8 cleavage triggers chiasmata resolution during meiosis I and sister centromere disjunction during meiosis II, while Scc1 cleavage triggers sister chromatid disjunction in the first embryonic mitosis, demonstrating a dramatic transition from Rec8- to Scc1-containing cohesin at fertilization. Crucially, activation of an ectopic Rec8 transgene during the growing phase of Rec8(TEV)(/TEV) oocytes does not prevent TEV-mediated bivalent destruction, implying little or no cohesin turnover for &ge;2 wk during oocyte growth. We suggest that the inability of oocytes to regenerate cohesion may contribute to age-related meiosis I errors.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0701161</span>, <span class="grant">G0901046</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2010</span>;<span class="volume">24</span>;<span class="number">22</span>;<span class="pagerange">2505-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20971813">20971813</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2975927">2975927</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.605910">10.1101/gad.605910</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Confirmed rare copy number variants implicate novel genes in schizophrenia.</h4>
<p class="authors"><span class="author">Tam GW</span>, <span class="author">van de Lagemaat LN</span>, <span class="author">Redon R</span>, <span class="author">Strathdee KE</span>, <span class="author">Croning MD</span>, <span class="author">Malloy MP</span>, <span class="author">Muir WJ</span>, <span class="author">Pickard BS</span>, <span class="author">Deary IJ</span>, <span class="author">Blackwood DH</span>, <span class="author">Carter NP</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Understanding how cognitive processes including learning, memory, decision making and ideation are encoded by the genome is a key question in biology. Identification of sets of genes underlying human mental disorders is a path towards this objective. Schizophrenia is a common disease with cognitive symptoms, high heritability and complex genetics. We have identified genes involved with schizophrenia by measuring differences in DNA copy number across the entire genome in 91 schizophrenia cases and 92 controls in the Scottish population. Our data reproduce rare and common variants observed in public domain data from &gt;3000 schizophrenia cases, confirming known disease loci as well as identifying novel loci. We found copy number variants in PDE10A (phosphodiesterase 10A), CYFIP1 [cytoplasmic FMR1 (Fragile X mental retardation 1)-interacting protein 1], K(+) channel genes KCNE1 and KCNE2, the Down&#39;s syndrome critical region 1 gene RCAN1 (regulator of calcineurin 1), cell-recognition protein CHL1 (cell adhesion molecule with homology with L1CAM), the transcription factor SP4 (specificity protein 4) and histone deacetylase HDAC9, among others (see http://www.genes2cognition.org/SCZ-CNV). Integrating the function of these many genes into a coherent model of schizophrenia and cognition is a major unanswered challenge.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">066717</span>, <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">2</span>;<span class="pagerange">445-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20298200" rel="external">20298200</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0380445" rel="external">10.1042/BST0380445</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficiency of Xist-mediated silencing on autosomes is linked to chromosomal domain organisation.</h4>
<p class="authors"><span class="author">Tang YA</span>, <span class="author">Huntley D</span>, <span class="author">Montana G</span>, <span class="author">Cerase A</span>, <span class="author">Nesterova TB</span> and <span class="author">Brockdorff N</span></p>
<p class="affiliation">MRC Clinical Sciences Centre, Faculty of Medicine ICSTM, Hammersmith Hospital, London, UK. <a href="mailto:%6E%65%69%6C.%62%72%6F%63%6B%64%6F%72%66%66%40%62%69%6F%63%68.%6F%78.%61%63.%75%6B">&#110;&#101;&#105;&#108;&#46;&#98;&#114;&#111;&#99;&#107;&#100;&#111;&#114;&#102;&#102;&#64;&#98;&#105;&#111;&#99;&#104;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract"><strong>Background</strong>: X chromosome inactivation, the mechanism used by mammals to equalise dosage of X-linked genes in XX females relative to XY males, is triggered by chromosome-wide localisation of a cis-acting non-coding RNA, Xist. The mechanism of Xist RNA spreading and Xist-dependent silencing is poorly understood. A large body of evidence indicates that silencing is more efficient on the X chromosome than on autosomes, leading to the idea that the X chromosome has acquired sequences that facilitate propagation of silencing. LINE-1 (L1) repeats are relatively enriched on the X chromosome and have been proposed as candidates for these sequences. To determine the requirements for efficient silencing we have analysed the relationship of chromosome features, including L1 repeats, and the extent of silencing in cell lines carrying inducible Xist transgenes located on one of three different autosomes.</p><p class="abstract"><strong>Results</strong>: Our results show that the organisation of the chromosome into large gene-rich and L1-rich domains is a key determinant of silencing efficiency. Specifically genes located in large gene-rich domains with low L1 density are relatively resistant to Xist-mediated silencing whereas genes located in gene-poor domains with high L1 density are silenced more efficiently. These effects are observed shortly after induction of Xist RNA expression, suggesting that chromosomal domain organisation influences establishment rather than long-term maintenance of silencing. The X chromosome and some autosomes have only small gene-rich L1-depleted domains and we suggest that this could confer the capacity for relatively efficient chromosome-wide silencing.</p><p class="abstract"><strong>Conclusions</strong>: This study provides insight into the requirements for efficient Xist mediated silencing and specifically identifies organisation of the chromosome into gene-rich L1-depleted and gene-poor L1-dense domains as a major influence on the ability of Xist-mediated silencing to be propagated in a continuous manner in cis.</p>
<p class="publication"><span class="title">Epigenetics &amp; chromatin</span>&nbsp;<span class="year">2010</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20459652" rel="external">20459652</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873326" rel="external">2873326</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1756-8935-3-10" rel="external">10.1186/1756-8935-3-10</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prmt5 is essential for early mouse development and acts in the cytoplasm to maintain ES cell pluripotency.</h4>
<p class="authors"><span class="author">Tee WW</span>, <span class="author">Pardo M</span>, <span class="author">Theunissen TW</span>, <span class="author">Yu L</span>, <span class="author">Choudhary JS</span>, <span class="author">Hajkova P</span> and <span class="author">Surani MA</span></p>
<p class="affiliation">Wellcome Trust, Cancer Research UK, Gurdon Institute of Cancer and Developmental Biology, University of Cambridge, Cambridge CB2 1QN, United Kingdom.</p><p class="abstract">Prmt5, an arginine methyltransferase, has multiple roles in germ cells, and possibly in pluripotency. Here we show that loss of Prmt5 function is early embryonic-lethal due to the abrogation of pluripotent cells in blastocysts. Prmt5 is also up-regulated in the cytoplasm during the derivation of embryonic stem (ES) cells together with Stat3, where they persist to maintain pluripotency. Prmt5 in association with Mep50 methylates cytosolic histone H2A (H2AR3me2s) to repress differentiation genes in ES cells. Loss of Prmt5 or Mep50 results in derepression of differentiation genes, indicating the significance of the Prmt5/Mep50 complex for pluripotency, which may occur in conjunction with the leukemia inhibitory factor (LIF)/Stat3 pathway.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0800784</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2010</span>;<span class="volume">24</span>;<span class="number">24</span>;<span class="pagerange">2772-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21159818">21159818</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3003195">3003195</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.606110">10.1101/gad.606110</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exploratory data analysis in large-scale genetic studies.</h4>
<p class="authors"><span class="author">Teo YY</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. <a href="mailto:%74%65%6F%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#116;&#101;&#111;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genome-wide association studies (GWAS) have become the method of choice for investigating the genetic basis of common diseases and complex traits. The immense scale of these experiments is unprecedented, involving thousands of samples and up to a million variables. The careful execution of exploratory data analysis (EDA) prior to the actual genotype-phenotype association analysis is crucial as this identifies problematic samples and poorly assayed genetic polymorphisms that, if undetected, can compromise the outcome of the experiment. EDA of such large-scale genetic data sets thus requires specialized numerical and graphical strategies, and this article provides a review of the current exploratory tools commonly used in GWAS.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Biostatistics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">1</span>;<span class="pagerange">70-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19828557" rel="external">19828557</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbiostatistics%2Fkxp038" rel="external">10.1093/biostatistics/kxp038</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Methodological challenges of genome-wide association analysis in Africa.</h4>
<p class="authors"><span class="author">Teo YY</span>, <span class="author">Small KS</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7BN, UK.</p><p class="abstract">Medical research in Africa has yet to benefit from the advent of genome-wide association (GWA) analysis, partly because the genotyping tools and statistical methods that have been developed for European and Asian populations struggle to deal with the high levels of genome diversity and population structure in Africa. However, the haplotypic diversity of African populations might help to overcome one of the major roadblocks in GWA research, the fine mapping of causal variants. We review the methodological challenges and consider how GWA studies in Africa will be transformed by new approaches in statistical imputation and large-scale genome sequencing.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G0600718</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">077383</span>, <span class="grant">082370</span></p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">2</span>;<span class="pagerange">149-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20084087">20084087</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3769612">3769612</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg2731">10.1038/nrg2731</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biological, clinical and population relevance of 95 loci for blood lipids.</h4>
<p class="authors"><span class="author">Teslovich TM</span>, <span class="author">Musunuru K</span>, <span class="author">Smith AV</span>, <span class="author">Edmondson AC</span>, <span class="author">Stylianou IM</span>, <span class="author">Koseki M</span>, <span class="author">Pirruccello JP</span>, <span class="author">Ripatti S</span>, <span class="author">Chasman DI</span>, <span class="author">Willer CJ</span>, <span class="author">Johansen CT</span>, <span class="author">Fouchier SW</span>, <span class="author">Isaacs A</span>, <span class="author">Peloso GM</span>, <span class="author">Barbalic M</span>, <span class="author">Ricketts SL</span>, <span class="author">Bis JC</span>, <span class="author">Aulchenko YS</span>, <span class="author">Thorleifsson G</span>, <span class="author">Feitosa MF</span>, <span class="author">Chambers J</span>, <span class="author">Orho-Melander M</span>, <span class="author">Melander O</span>, <span class="author">Johnson T</span>, <span class="author">Li X</span>, <span class="author">Guo X</span>, <span class="author">Li M</span>, <span class="author">Shin Cho Y</span>, <span class="author">Jin Go M</span>, <span class="author">Jin Kim Y</span>, <span class="author">Lee JY</span>, <span class="author">Park T</span>, <span class="author">Kim K</span>, <span class="author">Sim X</span>, <span class="author">Twee-Hee Ong R</span>, <span class="author">Croteau-Chonka DC</span>, <span class="author">Lange LA</span>, <span class="author">Smith JD</span>, <span class="author">Song K</span>, <span class="author">Hua Zhao J</span>, <span class="author">Yuan X</span>, <span class="author">Luan J</span>, <span class="author">Lamina C</span>, <span class="author">Ziegler A</span>, <span class="author">Zhang W</span>, <span class="author">Zee RY</span>, <span class="author">Wright AF</span>, <span class="author">Witteman JC</span>, <span class="author">Wilson JF</span>, <span class="author">Willemsen G</span>, <span class="author">Wichmann HE</span>, <span class="author">Whitfield JB</span>, <span class="author">Waterworth DM</span>, <span class="author">Wareham NJ</span>, <span class="author">Waeber G</span>, <span class="author">Vollenweider P</span>, <span class="author">Voight BF</span>, <span class="author">Vitart V</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Uda M</span>, <span class="author">Tuomilehto J</span>, <span class="author">Thompson JR</span>, <span class="author">Tanaka T</span>, <span class="author">Surakka I</span>, <span class="author">Stringham HM</span>, <span class="author">Spector TD</span>, <span class="author">Soranzo N</span>, <span class="author">Smit JH</span>, <span class="author">Sinisalo J</span>, <span class="author">Silander K</span>, <span class="author">Sijbrands EJ</span>, <span class="author">Scuteri A</span>, <span class="author">Scott J</span>, <span class="author">Schlessinger D</span>, <span class="author">Sanna S</span>, <span class="author">Salomaa V</span>, <span class="author">Saharinen J</span>, <span class="author">Sabatti C</span>, <span class="author">Ruokonen A</span>, <span class="author">Rudan I</span>, <span class="author">Rose LM</span>, <span class="author">Roberts R</span>, <span class="author">Rieder M</span>, <span class="author">Psaty BM</span>, <span class="author">Pramstaller PP</span>, <span class="author">Pichler I</span>, <span class="author">Perola M</span>, <span class="author">Penninx BW</span>, <span class="author">Pedersen NL</span>, <span class="author">Pattaro C</span>, <span class="author">Parker AN</span>, <span class="author">Pare G</span>, <span class="author">Oostra BA</span>, <span class="author">O&#39;Donnell CJ</span>, <span class="author">Nieminen MS</span>, <span class="author">Nickerson DA</span>, <span class="author">Montgomery GW</span>, <span class="author">Meitinger T</span>, <span class="author">McPherson R</span>, <span class="author">McCarthy MI</span>, <span class="author">McArdle W</span>, <span class="author">Masson D</span>, <span class="author">Martin NG</span>, <span class="author">Marroni F</span>, <span class="author">Mangino M</span>, <span class="author">Magnusson PK</span>, <span class="author">Lucas G</span>, <span class="author">Luben R</span>, <span class="author">Loos RJ</span>, <span class="author">Lokki ML</span>, <span class="author">Lettre G</span>, <span class="author">Langenberg C</span>, <span class="author">Launer LJ</span>, <span class="author">Lakatta EG</span>, <span class="author">Laaksonen R</span>, <span class="author">Kyvik KO</span>, <span class="author">Kronenberg F</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Khaw KT</span>, <span class="author">Kaprio J</span>, <span class="author">Kaplan LM</span>, <span class="author">Johansson A</span>, <span class="author">Jarvelin MR</span>, <span class="author">Janssens AC</span>, <span class="author">Ingelsson E</span>, <span class="author">Igl W</span>, <span class="author">Kees Hovingh G</span>, <span class="author">Hottenga JJ</span>, <span class="author">Hofman A</span>, <span class="author">Hicks AA</span>, <span class="author">Hengstenberg C</span>, <span class="author">Heid IM</span>, <span class="author">Hayward C</span>, <span class="author">Havulinna AS</span>, <span class="author">Hastie ND</span>, <span class="author">Harris TB</span>, <span class="author">Haritunians T</span>, <span class="author">Hall AS</span>, <span class="author">Gyllensten U</span>, <span class="author">Guiducci C</span>, <span class="author">Groop LC</span>, <span class="author">Gonzalez E</span>, <span class="author">Gieger C</span>, <span class="author">Freimer NB</span>, <span class="author">Ferrucci L</span>, <span class="author">Erdmann J</span>, <span class="author">Elliott P</span>, <span class="author">Ejebe KG</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Dominiczak AF</span>, <span class="author">Demissie S</span>, <span class="author">Deloukas P</span>, <span class="author">de Geus EJ</span>, <span class="author">de Faire U</span>, <span class="author">Crawford G</span>, <span class="author">Collins FS</span>, <span class="author">Chen YD</span>, <span class="author">Caulfield MJ</span>, <span class="author">Campbell H</span>, <span class="author">Burtt NP</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Boomsma DI</span>, <span class="author">Boekholdt SM</span>, <span class="author">Bergman RN</span>, <span class="author">Barroso I</span>, <span class="author">Bandinelli S</span>, <span class="author">Ballantyne CM</span>, <span class="author">Assimes TL</span>, <span class="author">Quertermous T</span>, <span class="author">Altshuler D</span>, <span class="author">Seielstad M</span>, <span class="author">Wong TY</span>, <span class="author">Tai ES</span>, <span class="author">Feranil AB</span>, <span class="author">Kuzawa CW</span>, <span class="author">Adair LS</span>, <span class="author">Taylor HA</span>, <span class="author">Borecki IB</span>, <span class="author">Gabriel SB</span>, <span class="author">Wilson JG</span>, <span class="author">Holm H</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Gudnason V</span>, <span class="author">Krauss RM</span>, <span class="author">Mohlke KL</span>, <span class="author">Ordovas JM</span>, <span class="author">Munroe PB</span>, <span class="author">Kooner JS</span>, <span class="author">Tall AR</span>, <span class="author">Hegele RA</span>, <span class="author">Kastelein JJ</span>, <span class="author">Schadt EE</span>, <span class="author">Rotter JI</span>, <span class="author">Boerwinkle E</span>, <span class="author">Strachan DP</span>, <span class="author">Mooser V</span>, <span class="author">Stefansson K</span>, <span class="author">Reilly MP</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Cupples LA</span>, <span class="author">Sandhu MS</span>, <span class="author">Ridker PM</span>, <span class="author">Rader DJ</span>, <span class="author">van Duijn CM</span>, <span class="author">Peltonen L</span>, <span class="author">Abecasis GR</span>, <span class="author">Boehnke M</span> and <span class="author">Kathiresan S</span></p>
<p class="affiliation">Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA.</p><p class="abstract">Plasma concentrations of total cholesterol, low-density lipoprotein cholesterol, high-density lipoprotein cholesterol and triglycerides are among the most important risk factors for coronary artery disease (CAD) and are targets for therapeutic intervention. We screened the genome for common variants associated with plasma lipids in &gt;100,000 individuals of European ancestry. Here we report 95 significantly associated loci (P &lt; 5 x 10(-8)), with 59 showing genome-wide significant association with lipid traits for the first time. The newly reported associations include single nucleotide polymorphisms (SNPs) near known lipid regulators (for example, CYP7A1, NPC1L1 and SCARB1) as well as in scores of loci not previously implicated in lipoprotein metabolism. The 95 loci contribute not only to normal variation in lipid traits but also to extreme lipid phenotypes and have an impact on lipid traits in three non-European populations (East Asians, South Asians and African Americans). Our results identify several novel loci associated with plasma lipids that are also associated with CAD. Finally, we validated three of the novel genes-GALNT2, PPP1R3B and TTC39B-with experiments in mouse models. Taken together, our findings provide the foundation to develop a broader biological understanding of lipoprotein metabolism and to identify new therapeutic opportunities for the prevention of CAD.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/02/128</span>, <span class="grant">PG/08/094</span>, <span class="grant">PG/08/094/26019</span>, <span class="grant">RG/07/005/23633</span>, <span class="grant">SP/08/005/25115</span>; Chief Scientist Office: <span class="grant">CZB/4/710</span>; FIC NIH HHS: <span class="grant">TW05596</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401527</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0801056</span>, <span class="grant">G0801566</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NCI NIH HHS: <span class="grant">CA 047988</span>; NCRR NIH HHS: <span class="grant">M01-RR00425</span>, <span class="grant">RR20649</span>, <span class="grant">U54 RR020278</span>, <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">1Z01 HG000024</span>, <span class="grant">N01-HG-65403</span>, <span class="grant">T32 HG00040</span>, <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>, <span class="grant">5R01HL08770003</span>, <span class="grant">5R01HL08821502</span>, <span class="grant">HL 04381</span>, <span class="grant">HL 080467</span>, <span class="grant">HL-54776</span>, <span class="grant">HL085144</span>, <span class="grant">K99 HL098364</span>, <span class="grant">K99 HL098364-01</span>, <span class="grant">K99HL094535</span>, <span class="grant">N01 HC-15103</span>, <span class="grant">N01 HC-55222</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-35129</span>, <span class="grant">N01-HC-45133</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N01-HC-75150</span>, <span class="grant">N01-HC-85079</span>, <span class="grant">N01-HC-85080</span>, <span class="grant">N01-HC-85081</span>, <span class="grant">N01-HC-85082</span>, <span class="grant">N01-HC-85083</span>, <span class="grant">N01-HC-85084</span>, <span class="grant">N01-HC-85085</span>, <span class="grant">N01-HC-85086</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01 HL087647</span>, <span class="grant">R01 HL087676</span>, <span class="grant">R01 HL089650</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL087652</span>, <span class="grant">R01HL59367</span>, <span class="grant">RC1 HL099634</span>, <span class="grant">RC1 HL099634-02</span>, <span class="grant">RC1 HL099793</span>, <span class="grant">RC2 HL101864,</span>, <span class="grant">RC2 HL102419</span>, <span class="grant">T32HL007208</span>, <span class="grant">U01 HL069757</span>, <span class="grant">U01 HL080295</span>; NIA NIH HHS: <span class="grant">N01-AG-12100</span>; NICHD NIH HHS: <span class="grant">R24 HD050924</span>; NIDDK NIH HHS: <span class="grant">5R01DK06833603</span>, <span class="grant">5R01DK07568102</span>, <span class="grant">DK062370</span>, <span class="grant">DK063491</span>, <span class="grant">DK072193</span>, <span class="grant">DK078150</span>, <span class="grant">DK56350</span>, <span class="grant">R01 DK072193</span>, <span class="grant">R01 DK078150</span>, <span class="grant">U01 DK062370</span>, <span class="grant">U01 DK062418</span>; NIEHS NIH HHS: <span class="grant">ES10126</span>; NIGMS NIH HHS: <span class="grant">T32 GM007092</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113/B/04/Z</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">466</span>;<span class="number">7307</span>;<span class="pagerange">707-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20686565">20686565</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3039276">3039276</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature09270">10.1038/nature09270</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The systematic functional analysis of Plasmodium protein kinases identifies essential regulators of mosquito transmission.</h4>
<p class="authors"><span class="author">Tewari R</span>, <span class="author">Straschil U</span>, <span class="author">Bateman A</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Cherevach I</span>, <span class="author">Gong P</span>, <span class="author">Pain A</span> and <span class="author">Billker O</span></p>
<p class="affiliation">Institute of Genetics, QMC, University of Nottingham, Nottingham NG7 2UH, UK. <a href="mailto:%72%69%74%61.%74%65%77%61%72%69%40%6E%6F%74%74%69%6E%67%68%61%6D.%61%63.%75%6B">&#114;&#105;&#116;&#97;&#46;&#116;&#101;&#119;&#97;&#114;&#105;&#64;&#110;&#111;&#116;&#116;&#105;&#110;&#103;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Although eukaryotic protein kinases (ePKs) contribute to many cellular processes, only three Plasmodium falciparum ePKs have thus far been identified as essential for parasite asexual blood stage development. To identify pathways essential for parasite transmission between their mammalian host and mosquito vector, we undertook a systematic functional analysis of ePKs in the genetically tractable rodent parasite Plasmodium berghei. Modeling domain signatures of conventional ePKs identified 66 putative Plasmodium ePKs. Kinomes are highly conserved between Plasmodium species. Using reverse genetics, we show that 23 ePKs are redundant for asexual erythrocytic parasite development in mice. Phenotyping mutants at four life cycle stages in Anopheles stephensi mosquitoes revealed functional clusters of kinases required for sexual development and sporogony. Roles for a putative SR protein kinase (SRPK) in microgamete formation, a conserved regulator of clathrin uncoating (GAK) in ookinete formation, and a likely regulator of energy metabolism (SNF1/KIN) in sporozoite development were identified.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501670</span>, <span class="grant">G0900109</span>; Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT078335MA</span>, <span class="grant">WT089085/Z/09/Z</span></p>
<p class="publication"><span class="title">Cell host &amp; microbe</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">377-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20951971">20951971</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2977076">2977076</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.chom.2010.09.006">10.1016/j.chom.2010.09.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An adaptable two-color flow cytometric assay to quantitate the invasion of erythrocytes by Plasmodium falciparum parasites.</h4>
<p class="authors"><span class="author">Theron M</span>, <span class="author">Hesketh RL</span>, <span class="author">Subramanian S</span> and <span class="author">Rayner JC</span></p>
<p class="affiliation">Sanger Malaria Programme, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, United Kingdom.</p><p class="abstract">Plasmodium falciparum genotyping has recently undergone a revolution, and genome-wide genotype datasets are now being collected for large numbers of parasite isolates. By contrast, phenotyping technologies have lagged behind, with few high throughput phenotyping platforms available. Invasion of human erythrocytes by Plasmodium falciparum is a phenotype of particular interest because of its central role in parasite development. Invasion is a variable phenotype influenced by natural genetic variation in both the parasite and host and is governed by multiple overlapping and in some instances redundant parasite-erythrocyte interactions. To facilitate the scale-up of erythrocyte invasion phenotyping, we have developed a novel platform based on two-color flow cytometry that distinguishes parasite invasion from parasite growth. Target cells that had one or more receptors removed using enzymatic treatment were prelabeled with intracellular dyes CFDA-SE or DDAO-SE, incubated with P. falciparum parasites, and parasites that had invaded either labeled or unlabeled cells were detected with fluorescent DNA-intercalating dyes Hoechst 33342 or SYBR Green I. Neither cell label interfered with erythrocyte invasion, and the combination of cell and parasite dyes recapitulated known invasion phenotypes for three standard laboratory strains. Three different dye combinations with minimal overlap have been validated, meaning the same assay can be adapted to instruments harboring several different combinations of laser lines. The assay is sensitive, operates in a 96-well format, and can be used to quantitate the impact of natural or experimental genetic variation on erythrocyte invasion efficiency.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cytometry. Part A : the journal of the International Society for Analytical Cytology</span>&nbsp;<span class="year">2010</span>;<span class="volume">77</span>;<span class="number">11</span>;<span class="pagerange">1067-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20872885">20872885</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3047707">3047707</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fcyto.a.20972">10.1002/cyto.a.20972</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">De novo apparently balanced translocations in man are predominantly paternal in origin and associated with a significant increase in paternal age.</h4>
<p class="authors"><span class="author">Thomas NS</span>, <span class="author">Morris JK</span>, <span class="author">Baptista J</span>, <span class="author">Ng BL</span>, <span class="author">Crolla JA</span> and <span class="author">Jacobs PA</span></p>
<p class="affiliation">Wessex Regional Genetics Laboratory, Salisbury District Hospital, Salisbury SP2 8TE, UK. <a href="mailto:%73%69%6D%6F%6E.%74%68%6F%6D%61%73%40%73%61%6C%69%73%62%75%72%79.%6E%68%73.%75%6B">&#115;&#105;&#109;&#111;&#110;&#46;&#116;&#104;&#111;&#109;&#97;&#115;&#64;&#115;&#97;&#108;&#105;&#115;&#98;&#117;&#114;&#121;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Congenital chromosome abnormalities are relatively common in our species and among structural abnormalities the most common class is balanced reciprocal translocations. Determining the parental origin of de novo balanced translocations may provide insights into how and when they arise. While there is a general paternal bias in the origin of non-recurrent unbalanced rearrangements, there are few data on parental origin of non-recurrent balanced rearrangements.</p><p class="abstract"><strong>Methods</strong>: The parental origin of a series of de novo balanced reciprocal translocations was determined using DNA from flow sorted derivative chromosomes and linkage analysis.</p><p class="abstract"><strong>Results</strong>: Of 27 translocations, we found 26 to be of paternal origin and only one of maternal origin. We also found the paternally derived translocations to be associated with a significantly increased paternal age (p&lt;0.008).</p><p class="abstract"><strong>Conclusion</strong>: Our results suggest there is a very pronounced paternal bias in the origin of all non-recurrent reciprocal translocations and that they may arise during one of the numerous mitotic divisions that occur in the spermatogonial germ cells prior to meiosis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">47</span>;<span class="number">2</span>;<span class="pagerange">112-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19638350">19638350</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2009.069716">10.1136/jmg.2009.069716</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CpG islands influence chromatin structure via the CpG-binding protein Cfp1.</h4>
<p class="authors"><span class="author">Thomson JP</span>, <span class="author">Skene PJ</span>, <span class="author">Selfridge J</span>, <span class="author">Clouaire T</span>, <span class="author">Guy J</span>, <span class="author">Webb S</span>, <span class="author">Kerr AR</span>, <span class="author">Deaton A</span>, <span class="author">Andrews R</span>, <span class="author">James KD</span>, <span class="author">Turner DJ</span>, <span class="author">Illingworth R</span> and <span class="author">Bird A</span></p>
<p class="affiliation">Wellcome Trust Centre for Cell Biology, Michael Swann Building, University of Edinburgh, Mayfield Road, Edinburgh EH9 3JR, UK.</p><p class="abstract">CpG islands (CGIs) are prominent in the mammalian genome owing to their GC-rich base composition and high density of CpG dinucleotides. Most human gene promoters are embedded within CGIs that lack DNA methylation and coincide with sites of histone H3 lysine 4 trimethylation (H3K4me3), irrespective of transcriptional activity. In spite of these intriguing correlations, the functional significance of non-methylated CGI sequences with respect to chromatin structure and transcription is unknown. By performing a search for proteins that are common to all CGIs, here we show high enrichment for Cfp1, which selectively binds to non-methylated CpGs in vitro. Chromatin immunoprecipitation of a mono-allelically methylated CGI confirmed that Cfp1 specifically associates with non-methylated CpG sites in vivo. High throughput sequencing of Cfp1-bound chromatin identified a notable concordance with non-methylated CGIs and sites of H3K4me3 in the mouse brain. Levels of H3K4me3 at CGIs were markedly reduced in Cfp1-depleted cells, consistent with the finding that Cfp1 associates with the H3K4 methyltransferase Setd1 (refs 7, 8). To test whether non-methylated CpG-dense sequences are sufficient to establish domains of H3K4me3, we analysed artificial CpG clusters that were integrated into the mouse genome. Despite the absence of promoters, the insertions recruited Cfp1 and created new peaks of H3K4me3. The data indicate that a primary function of non-methylated CGIs is to genetically influence the local chromatin modification state by interaction with Cfp1 and perhaps other CpG-binding proteins.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800026</span>; Wellcome Trust: <span class="grant">079643</span>, <span class="grant">091580</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7291</span>;<span class="pagerange">1082-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20393567">20393567</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3730110">3730110</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08924">10.1038/nature08924</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chlamydia trachomatis: small genome, big challenges.</h4>
<p class="authors"><span class="author">Thomson NR</span> and <span class="author">Clarke IN</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%6E%72%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#114;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chlamydial infections are highly newsworthy, but basic research into Chlamydia trachomatis is severely hampered by a series of formidable technical barriers. This has resulted in a paucity of information with respect to the genetics and population structure of these recalcitrant bacteria. Here we present a review of what is currently known about the genomics of C. trachomatis and discuss the usefulness of molecular typing systems and the prospects of developing a pan-chlamydial genome resource.</p>
<p class="publication"><span class="title">Future microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">555-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20353297" rel="external">20353297</a>; DOI: <a href="http://dx.doi.org/10.2217%2Ffmb.10.31" rel="external">10.2217/fmb.10.31</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence variants at CHRNB3-CHRNA6 and CYP2A6 affect smoking behavior.</h4>
<p class="authors"><span class="author">Thorgeirsson TE</span>, <span class="author">Gudbjartsson DF</span>, <span class="author">Surakka I</span>, <span class="author">Vink JM</span>, <span class="author">Amin N</span>, <span class="author">Geller F</span>, <span class="author">Sulem P</span>, <span class="author">Rafnar T</span>, <span class="author">Esko T</span>, <span class="author">Walter S</span>, <span class="author">Gieger C</span>, <span class="author">Rawal R</span>, <span class="author">Mangino M</span>, <span class="author">Prokopenko I</span>, <span class="author">M&auml;gi R</span>, <span class="author">Keskitalo K</span>, <span class="author">Gudjonsdottir IH</span>, <span class="author">Gretarsdottir S</span>, <span class="author">Stefansson H</span>, <span class="author">Thompson JR</span>, <span class="author">Aulchenko YS</span>, <span class="author">Nelis M</span>, <span class="author">Aben KK</span>, <span class="author">den Heijer M</span>, <span class="author">Dirksen A</span>, <span class="author">Ashraf H</span>, <span class="author">Soranzo N</span>, <span class="author">Valdes AM</span>, <span class="author">Steves C</span>, <span class="author">Uitterlinden AG</span>, <span class="author">Hofman A</span>, <span class="author">T&ouml;njes A</span>, <span class="author">Kovacs P</span>, <span class="author">Hottenga JJ</span>, <span class="author">Willemsen G</span>, <span class="author">Vogelzangs N</span>, <span class="author">D&ouml;ring A</span>, <span class="author">Dahmen N</span>, <span class="author">Nitz B</span>, <span class="author">Pergadia ML</span>, <span class="author">Saez B</span>, <span class="author">De Diego V</span>, <span class="author">Lezcano V</span>, <span class="author">Garcia-Prats MD</span>, <span class="author">Ripatti S</span>, <span class="author">Perola M</span>, <span class="author">Kettunen J</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">Laitinen J</span>, <span class="author">Isohanni M</span>, <span class="author">Huei-Yi S</span>, <span class="author">Allen M</span>, <span class="author">Krestyaninova M</span>, <span class="author">Hall AS</span>, <span class="author">Jones GT</span>, <span class="author">van Rij AM</span>, <span class="author">Mueller T</span>, <span class="author">Dieplinger B</span>, <span class="author">Haltmayer M</span>, <span class="author">Jonsson S</span>, <span class="author">Matthiasson SE</span>, <span class="author">Oskarsson H</span>, <span class="author">Tyrfingsson T</span>, <span class="author">Kiemeney LA</span>, <span class="author">Mayordomo JI</span>, <span class="author">Lindholt JS</span>, <span class="author">Pedersen JH</span>, <span class="author">Franklin WA</span>, <span class="author">Wolf H</span>, <span class="author">Montgomery GW</span>, <span class="author">Heath AC</span>, <span class="author">Martin NG</span>, <span class="author">Madden PA</span>, <span class="author">Giegling I</span>, <span class="author">Rujescu D</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Salomaa V</span>, <span class="author">Stumvoll M</span>, <span class="author">Spector TD</span>, <span class="author">Wichmann HE</span>, <span class="author">Metspalu A</span>, <span class="author">Samani NJ</span>, <span class="author">Penninx BW</span>, <span class="author">Oostra BA</span>, <span class="author">Boomsma DI</span>, <span class="author">Tiemeier H</span>, <span class="author">van Duijn CM</span>, <span class="author">Kaprio J</span>, <span class="author">Gulcher JR</span>, <span class="author">ENGAGE Consortium</span>, <span class="author">McCarthy MI</span>, <span class="author">Peltonen L</span>, <span class="author">Thorsteinsdottir U</span> and <span class="author">Stefansson K</span></p>
<p class="affiliation">[1] deCODE Genetics, Reykjavik, Iceland. [2] Center for Biomolecular Science and Engineering, Jack Baskin School of Engineering, University of California, Santa Cruz, California, USA.</p><p class="abstract">Smoking is a common risk factor for many diseases. We conducted genome-wide association meta-analyses for the number of cigarettes smoked per day (CPD) in smokers (n = 31,266) and smoking initiation (n = 46,481) using samples from the ENGAGE Consortium. In a second stage, we tested selected SNPs with in silico replication in the Tobacco and Genetics (TAG) and Glaxo Smith Kline (Ox-GSK) consortia cohorts (n = 45,691 smokers) and assessed some of those in a third sample of European ancestry (n = 9,040). Variants in three genomic regions associated with CPD (P &lt; 5 x 10(-8)), including previously identified SNPs at 15q25 represented by rs1051730[A] (effect size = 0.80 CPD, P = 2.4 x 10(-69)), and SNPs at 19q13 and 8p11, represented by rs4105144[C] (effect size = 0.39 CPD, P = 2.2 x 10(-12)) and rs6474412-T (effect size = 0.29 CPD, P = 1.4 x 10(-8)), respectively. Among the genes at the two newly associated loci are genes encoding nicotine-metabolizing enzymes (CYP2A6 and CYP2B6) and nicotinic acetylcholine receptor subunits (CHRNB3 and CHRNA6), all of which have been highlighted in previous studies of smoking and nicotine dependence. Nominal associations with lung cancer were observed at both 8p11 (rs6474412[T], odds ratio (OR) = 1.09, P = 0.04) and 19q13 (rs4105144[C], OR = 1.12, P = 0.0006).</p>
<p class="grants">Funded by: NIDA NIH HHS: <span class="grant">R01 DA 017932</span>; Wellcome Trust: <span class="grant">072856</span>, <span class="grant">079771</span>, <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">448-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20418888" rel="external">20418888</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3080600" rel="external">3080600</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.573" rel="external">10.1038/ng.573</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Imidazopurinones are markers of physiological genomic damage linked to DNA instability and glyoxalase 1-associated tumour multidrug resistance.</h4>
<p class="authors"><span class="author">Thornalley PJ</span>, <span class="author">Waris S</span>, <span class="author">Fleming T</span>, <span class="author">Santarius T</span>, <span class="author">Larkin SJ</span>, <span class="author">Winklhofer-Roob BM</span>, <span class="author">Stratton MR</span> and <span class="author">Rabbani N</span></p>
<p class="affiliation">Warwick Medical School, Clinical Sciences Research Institute, University of Warwick, University Hospital, Coventry CV2 2DX, UK. <a href="mailto:%70.%6A.%74%68%6F%72%6E%61%6C%6C%65%79%40%77%61%72%77%69%63%6B.%61%63.%75%6B">&#112;&#46;&#106;&#46;&#116;&#104;&#111;&#114;&#110;&#97;&#108;&#108;&#101;&#121;&#64;&#119;&#97;&#114;&#119;&#105;&#99;&#107;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Glyoxal and methylglyoxal are reactive dicarbonyl metabolites formed and metabolized in physiological systems. Increased exposure to these dicarbonyls is linked to mutagenesis and cytotoxicity and enhanced dicarbonyl metabolism by overexpression of glyoxalase 1 is linked to tumour multidrug resistance in cancer chemotherapy. We report herein that glycation of DNA by glyoxal and methylglyoxal produces a quantitatively important class of nucleotide adduct in physiological systems-imidazopurinones. The adduct derived from methylglyoxal-3-(2&#39;-deoxyribosyl)-6,7-dihydro-6,7-dihydroxy-6/7-methylimidazo-[2,3-b]purine-9(8)one isomers-was the major quantitative adduct detected in mononuclear leukocytes in vivo and tumour cell lines in vitro. It was linked to frequency of DNA strand breaks and increased markedly during apoptosis induced by a cell permeable glyoxalase 1 inhibitor. Unexpectedly, the DNA content of methylglyoxal-derived imidazopurinone and oxidative marker 7,8-dihydro-8-oxo-2&#39;-deoxyguanosine were increased moderately in glyoxalase 1-linked multidrug resistant tumour cell lines. Together these findings suggest that imidazopurinones are a major type of endogenous DNA damage and glyoxalase 1 overexpression in tumour cells strives to counter increased imidazopurinone formation in tumour cells likely linked to their high glycolytic activity.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C790/A5147</span>; Wellcome Trust: <span class="grant">077012/Z/05/Z</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">16</span>;<span class="pagerange">5432-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20435681" rel="external">20435681</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2938218" rel="external">2938218</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq306" rel="external">10.1093/nar/gkq306</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A visual migraine aura locus maps to 9q21-q22.</h4>
<p class="authors"><span class="author">Tikka-Kleemola P</span>, <span class="author">Artto V</span>, <span class="author">Veps&auml;l&auml;inen S</span>, <span class="author">Sobel EM</span>, <span class="author">R&auml;ty S</span>, <span class="author">Kaunisto MA</span>, <span class="author">Anttila V</span>, <span class="author">H&auml;m&auml;l&auml;inen E</span>, <span class="author">Sumelahti ML</span>, <span class="author">Ilmavirta M</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Kallela M</span>, <span class="author">Palotie A</span> and <span class="author">Wessman M</span></p>
<p class="affiliation">Folkh&auml;lsan Research Center, Biomedicum Helsinki, PO Box 63, 00014 University of Helsinki, Finland. <a href="mailto:%6D%61%69%6A%61.%77%65%73%73%6D%61%6E%40%68%65%6C%73%69%6E%6B%69.%66%69">&#109;&#97;&#105;&#106;&#97;&#46;&#119;&#101;&#115;&#115;&#109;&#97;&#110;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a></p><p class="abstract"><strong>Objective</strong>: To identify susceptibility loci for visual migraine aura in migraine families primarily affected with scintillating scotoma type of aura.</p><p class="abstract"><strong>Methods</strong>: We included Finnish migraine families with at least 2 affected family members with scintillating scotoma as defined by the International Criteria for Headache Disorders-II. A total of 36 multigenerational families containing 351 individuals were included, 185 of whom have visual aura and 159 have scintillating scotoma. Parametric and nonparametric linkage analyses were performed with 378 microsatellite markers. The most promising linkage loci found were fine-mapped with additional microsatellite markers.</p><p class="abstract"><strong>Results</strong>: A novel locus on chromosome 9q22-q31 for migraine aura was identified (HLOD = 4.7 at 104 cM). Fine-mapping identified a shared haplotype segment of 12 cM (9.8 Mb) on 9q21-q22 among the aura affected. Four other loci showed linkage to aura: a locus on 12p13 showed significant evidence of linkage, and suggestive evidence of linkage was detected to loci on chromosomes 5q13, 6q25, and 13q14.</p><p class="abstract"><strong>Conclusions</strong>: A novel visual migraine aura locus has been mapped to chromosome 9q21-q22. Interestingly, this region has previously been linked to occipitotemporal lobe epilepsy with prominent visual symptoms. Our finding further supports a shared genetic background in migraine and epilepsy and suggests that susceptibility variant(s) to visual aura for both of these traits are located in the 9q21-q22 locus.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM053275</span></p>
<p class="publication"><span class="title">Neurology</span>&nbsp;<span class="year">2010</span>;<span class="volume">74</span>;<span class="number">15</span>;<span class="pagerange">1171-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20385888">20385888</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2865729">2865729</a>; DOI: <a href="http://dx.doi.org/10.1212%2FWNL.0b013e3181d8ffcb">10.1212/WNL.0b013e3181d8ffcb</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evidence for a lineage of virulent bacteriophages that target Campylobacter.</h4>
<p class="authors"><span class="author">Timms AR</span>, <span class="author">Cambray-Young J</span>, <span class="author">Scott AE</span>, <span class="author">Petty NK</span>, <span class="author">Connerton PL</span>, <span class="author">Clarke L</span>, <span class="author">Seeger K</span>, <span class="author">Quail M</span>, <span class="author">Cummings N</span>, <span class="author">Maskell DJ</span>, <span class="author">Thomson NR</span> and <span class="author">Connerton IF</span></p>
<p class="affiliation">Division of Food Sciences, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough, Leicestershire, LE12 5RD, UK. <a href="mailto:%61%6E%64%72%65%77.%74%69%6D%6D%73%40%6E%6F%74%74%69%6E%67%68%61%6D.%61%63.%75%6B">&#97;&#110;&#100;&#114;&#101;&#119;&#46;&#116;&#105;&#109;&#109;&#115;&#64;&#110;&#111;&#116;&#116;&#105;&#110;&#103;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Our understanding of the dynamics of genome stability versus gene flux within bacteriophage lineages is limited. Recently, there has been a renewed interest in the use of bacteriophages as &#39;therapeutic&#39; agents; a prerequisite for their use in such therapies is a thorough understanding of their genetic complement, genome stability and their ecology to avoid the dissemination or mobilisation of phage or bacterial virulence and toxin genes. Campylobacter, a food-borne pathogen, is one of the organisms for which the use of bacteriophage is being considered to reduce human exposure to this organism.</p><p class="abstract"><strong>Results</strong>: Sequencing and genome analysis was performed for two Campylobacter bacteriophages. The genomes were extremely similar at the nucleotide level (&gt; or = 96%) with most differences accounted for by novel insertion sequences, DNA methylases and an approximately 10 kb contiguous region of metabolic genes that were dissimilar at the sequence level but similar in gene function between the two phages. Both bacteriophages contained a large number of radical S-adenosylmethionine (SAM) genes, presumably involved in boosting host metabolism during infection, as well as evidence that many genes had been acquired from a wide range of bacterial species. Further bacteriophages, from the UK Campylobacter typing set, were screened for the presence of bacteriophage structural genes, DNA methylases, mobile genetic elements and regulatory genes identified from the genome sequences. The results indicate that many of these bacteriophages are related, with 10 out of 15 showing some relationship to the sequenced genomes.</p><p class="abstract"><strong>Conclusions</strong>: Two large virulent Campylobacter bacteriophages were found to show very high levels of sequence conservation despite separation in time and place of isolation. The bacteriophages show adaptations to their host and possess genes that may enhance Campylobacter metabolism, potentially advantaging both the bacteriophage and its host. Genetic conservation has been shown to extend to other Campylobacter bacteriophages, forming a highly conserved lineage of bacteriophages that predate upon campylobacters and indicating that highly adapted bacteriophage genomes can be stable over prolonged periods of time.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="pagerange">214</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20353581" rel="external">20353581</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2853527" rel="external">2853527</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-11-214" rel="external">10.1186/1471-2164-11-214</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A randomised trial evaluating the safety and immunogenicity of the novel single oral dose typhoid vaccine M01ZH09 in healthy Vietnamese children.</h4>
<p class="authors"><span class="author">Tran TH</span>, <span class="author">Nguyen TD</span>, <span class="author">Nguyen TT</span>, <span class="author">Ninh TT</span>, <span class="author">Tran NB</span>, <span class="author">Nguyen VM</span>, <span class="author">Tran TT</span>, <span class="author">Cao TT</span>, <span class="author">Pham VM</span>, <span class="author">Nguyen TC</span>, <span class="author">Tran TD</span>, <span class="author">Pham VT</span>, <span class="author">To SD</span>, <span class="author">Campbell JI</span>, <span class="author">Stockwell E</span>, <span class="author">Schultsz C</span>, <span class="author">Simmons CP</span>, <span class="author">Glover C</span>, <span class="author">Lam W</span>, <span class="author">Marques F</span>, <span class="author">May JP</span>, <span class="author">Upton A</span>, <span class="author">Budhram R</span>, <span class="author">Dougan G</span>, <span class="author">Farrar J</span>, <span class="author">Nguyen VV</span> and <span class="author">Dolecek C</span></p>
<p class="affiliation">The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam.</p><p class="abstract"><strong>Background</strong>: The emergence of drug resistant typhoid fever is a major public health problem, especially in Asia. An oral single dose typhoid vaccine would have major advantages. M01ZH09 is a live oral single dose candidate typhoid vaccine containing Salmonella enterica serovar Typhi (Ty2 aroC(-)ssaV(-)) ZH9 with two independently attenuating deletions. Studies in healthy adults demonstrated immunogenicity and an acceptable safety profile.</p><p class="abstract"><strong>Objectives</strong>: We conducted a randomised placebo controlled, single-blind trial to evaluate the safety and immunogenicity of M01ZH09 in healthy Vietnamese children aged 5 to 14 years.</p><p class="abstract"><strong>Methods</strong>: Subjects were randomly assigned to receive either a nominal dose of 5x10(9) CFU of M01ZH09 or placebo and were followed up for 28 days. The primary safety outcome was the proportion of subjects with any adverse event attributed to M01ZH09. The primary immunogenicity endpoint was the proportion of subjects who showed a positive immune response to M01ZH09 in the Salmonella Typhi lipopolysaccharide (LPS) specific serum IgA and IgG ELISA.</p><p class="abstract">One hundred and fifty-one children were enrolled, 101 subjects received M01ZH09 and 50 subjects received placebo. An intention to treat analysis was conducted. There were no serious adverse events and no bacteraemias. In the M01ZH09 group, 26 (26%; 95% CI, 18-5%) of 101 subjects experienced adverse events compared to 11 (22%; 95% CI, 12-36%) of 50 subjects in the placebo group (odds ratio (OR) [95%CI] = 1.23 [0.550-2.747]; p = 0.691). Faecal shedding of S. Typhi (Ty2 aroC(-)ssaV(-)) ZH9 was detected in 51 (51%; 95% CI, 41-61%) of 100 M01ZH09 subjects. No shedding was detected beyond day 3. A positive immune response, defined as 70% increase (1.7 fold change) in LPS specific serum IgG (day 14 or 28) and/or 50% increase (1.5 fold change) in LPS specific serum IgA (day 7 or 14) from baseline was detected in 98 (97%; 95% CI, 92-99%) of 101 M01ZH09 recipients and 8 (16%; 95% CI, 7-29%) of 50 placebo recipients. Twenty-eight (100%; 95% CI, 88-100%) of 28 vaccine recipients who were evaluated in the LPS specific IgA ELISPOT assay showed a positive response compared to none of the 14 placebo recipients tested.</p><p class="abstract"><strong>Conclusions</strong>: This was the first phase II trial of a novel oral candidate typhoid vaccine in children in an endemic country. M01ZH09 had an appropriate safety profile and was immunogenic in children.</p><p class="abstract">Controlled-trials.com ISRCTN91111837.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">084368</span>, <span class="grant">B9RKYTO</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">e11778</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20668668" rel="external">20668668</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2909895" rel="external">2909895</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0011778" rel="external">10.1371/journal.pone.0011778</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improving draft assemblies by iterative mapping and assembly of short reads to eliminate gaps.</h4>
<p class="authors"><span class="author">Tsai IJ</span>, <span class="author">Otto TD</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6A%69%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#105;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Advances in sequencing technology allow genomes to be sequenced at vastly decreased costs. However, the assembled data frequently are highly fragmented with many gaps. We present a practical approach that uses Illumina sequences to improve draft genome assemblies by aligning sequences against contig ends and performing local assemblies to produce gap-spanning contigs. The continuity of a draft genome can thus be substantially improved, often without the need to generate new data.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT 085775/Z/08/Z</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">4</span>;<span class="pagerange">R41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20388197">20388197</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2884544">2884544</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-4-r41">10.1186/gb-2010-11-4-r41</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies five new breast cancer susceptibility loci.</h4>
<p class="authors"><span class="author">Turnbull C</span>, <span class="author">Ahmed S</span>, <span class="author">Morrison J</span>, <span class="author">Pernet D</span>, <span class="author">Renwick A</span>, <span class="author">Maranian M</span>, <span class="author">Seal S</span>, <span class="author">Ghoussaini M</span>, <span class="author">Hines S</span>, <span class="author">Healey CS</span>, <span class="author">Hughes D</span>, <span class="author">Warren-Perry M</span>, <span class="author">Tapper W</span>, <span class="author">Eccles D</span>, <span class="author">Evans DG</span>, <span class="author">Breast Cancer Susceptibility Collaboration (UK)</span>, <span class="author">Hooning M</span>, <span class="author">Schutte M</span>, <span class="author">van den Ouweland A</span>, <span class="author">Houlston R</span>, <span class="author">Ross G</span>, <span class="author">Langford C</span>, <span class="author">Pharoah PD</span>, <span class="author">Stratton MR</span>, <span class="author">Dunning AM</span>, <span class="author">Rahman N</span> and <span class="author">Easton DF</span></p>
<p class="affiliation">Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK.</p><p class="abstract">Breast cancer is the most common cancer in women in developed countries. To identify common breast cancer susceptibility alleles, we conducted a genome-wide association study in which 582,886 SNPs were genotyped in 3,659 cases with a family history of the disease and 4,897 controls. Promising associations were evaluated in a second stage, comprising 12,576 cases and 12,223 controls. We identified five new susceptibility loci, on chromosomes 9, 10 and 11 (P = 4.6 x 10(-7) to P = 3.2 x 10(-15)). We also identified SNPs in the 6q25.1 (rs3757318, P = 2.9 x 10(-6)), 8q24 (rs1562430, P = 5.8 x 10(-7)) and LSP1 (rs909116, P = 7.3 x 10(-7)) regions that showed more significant association with risk than those reported previously. Previously identified breast cancer susceptibility loci were also found to show larger effect sizes in this study of familial breast cancer cases than in previous population-based studies, consistent with polygenic susceptibility to the disease.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A10119</span>, <span class="grant">A10123</span>, <span class="grant">A10124</span>; Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">085475</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">6</span>;<span class="pagerange">504-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20453838">20453838</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3632836">3632836</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.586">10.1038/ng.586</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation and association analysis of GEN1 in breast cancer susceptibility.</h4>
<p class="authors"><span class="author">Turnbull C</span>, <span class="author">Hines S</span>, <span class="author">Renwick A</span>, <span class="author">Hughes D</span>, <span class="author">Pernet D</span>, <span class="author">Elliott A</span>, <span class="author">Seal S</span>, <span class="author">Warren-Perry M</span>, <span class="author">Gareth Evans D</span>, <span class="author">Eccles D</span>, <span class="author">Breast Cancer Susceptibility Collaboration UK</span>, <span class="author">Stratton MR</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Section of Cancer Genetics, the Institute of Cancer Research, 15 Cotswold Road, Sutton, Surrey SM2 5NG, UK.</p><p class="abstract">GEN1 was recently identified as a key Holliday junction resolvase involved in homologous recombination. Somatic truncating GEN1 mutations have been reported in two breast cancers. Together these data led to the proposition that GEN1 is a breast cancer predisposition gene. In this article we have formally investigated this hypothesis. We performed full-gene mutational analysis of GEN1 in 176 BRCA1/2-negative familial breast cancer samples and 159 controls. We genotyped six SNPs tagging the 30 common variants in the transcribed region of GEN1 in 3,750 breast cancer cases and 4,907 controls. Mutation analysis revealed one truncating variant, c.2515_2519delAAGTT, which was present in 4% of cases and 4% of controls. We identified control individuals homozygous for the deletion, demonstrating that the last 69 amino acids of GEN1 are dispensable for its function. We identified 17 other variants, but their frequency did not significantly differ between cases and controls. Analysis of 3,750 breast cancer cases and 4,907 controls demonstrated no evidence of significant association with breast cancer for six SNPs tagging the 30 common GEN1 variants. These data indicate that although it also plays a key role in double-strand DNA break repair, GEN1 does not make an appreciable contribution to breast cancer susceptibility by acting as a high- or intermediate-penetrance breast cancer predisposition gene like BRCA1, BRCA2, CHEK2, ATM, BRIP1 and PALB2 and that common GEN1 variants do not act as low-penetrance susceptibility alleles analogous to SNPs in FGFR2. Furthermore, our analyses demonstrate the importance of undertaking appropriate genetic investigations, typically full gene screening in cases and controls together with large-scale case-control association analyses, to evaluate the contribution of genes to cancer susceptibility.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C8620_A8372</span>; Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Breast cancer research and treatment</span>&nbsp;<span class="year">2010</span>;<span class="volume">124</span>;<span class="number">1</span>;<span class="pagerange">283-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20512659">20512659</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3632835">3632835</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10549-010-0949-1">10.1007/s10549-010-0949-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variants near DMRT1, TERT and ATF7IP are associated with testicular germ cell cancer.</h4>
<p class="authors"><span class="author">Turnbull C</span>, <span class="author">Rapley EA</span>, <span class="author">Seal S</span>, <span class="author">Pernet D</span>, <span class="author">Renwick A</span>, <span class="author">Hughes D</span>, <span class="author">Ricketts M</span>, <span class="author">Linger R</span>, <span class="author">Nsengimana J</span>, <span class="author">Deloukas P</span>, <span class="author">Huddart RA</span>, <span class="author">Bishop DT</span>, <span class="author">Easton DF</span>, <span class="author">Stratton MR</span>, <span class="author">Rahman N</span> and <span class="author">UK Testicular Cancer Collaboration</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK. <a href="mailto:%63%6C%61%72%65.%74%75%72%6E%62%75%6C%6C%40%69%63%72.%61%63.%75%6B">&#99;&#108;&#97;&#114;&#101;&#46;&#116;&#117;&#114;&#110;&#98;&#117;&#108;&#108;&#64;&#105;&#99;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We conducted a genome-wide association study for testicular germ cell tumor, genotyping 298,782 SNPs in 979 affected individuals and 4,947 controls from the UK and replicating associations in a further 664 cases and 3,456 controls. We identified three new susceptibility loci, two of which include genes that are involved in telomere regulation. We identified two independent signals within the TERT-CLPTM1L locus on chromosome 5, which has previously been associated with multiple other cancers (rs4635969, OR=1.54, P=1.14x10(-23); rs2736100, OR=1.33, P=7.55x10(-15)). We also identified a locus on chromosome 12 (rs2900333, OR=1.27, P=6.16x10(-10)) that contains ATF7IP, a regulator of TERT expression. Finally, we identified a locus on chromosome 9 (rs755383, OR=1.37, P=1.12x10(-23)), containing the sex determination gene DMRT1, which has been linked to teratoma susceptibility in mice.</p>
<p class="grants">Funded by: Cancer Research UK; Department of Health; Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">7</span>;<span class="pagerange">604-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20543847">20543847</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.607">10.1038/ng.607</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Separating the post-Glacial coancestry of European and Asian Y chromosomes within haplogroup R1a.</h4>
<p class="authors"><span class="author">Underhill PA</span>, <span class="author">Myres NM</span>, <span class="author">Rootsi S</span>, <span class="author">Metspalu M</span>, <span class="author">Zhivotovsky LA</span>, <span class="author">King RJ</span>, <span class="author">Lin AA</span>, <span class="author">Chow CE</span>, <span class="author">Semino O</span>, <span class="author">Battaglia V</span>, <span class="author">Kutuev I</span>, <span class="author">J&auml;rve M</span>, <span class="author">Chaubey G</span>, <span class="author">Ayub Q</span>, <span class="author">Mohyuddin A</span>, <span class="author">Mehdi SQ</span>, <span class="author">Sengupta S</span>, <span class="author">Rogaev EI</span>, <span class="author">Khusnutdinova EK</span>, <span class="author">Pshenichnov A</span>, <span class="author">Balanovsky O</span>, <span class="author">Balanovska E</span>, <span class="author">Jeran N</span>, <span class="author">Augustin DH</span>, <span class="author">Baldovic M</span>, <span class="author">Herrera RJ</span>, <span class="author">Thangaraj K</span>, <span class="author">Singh V</span>, <span class="author">Singh L</span>, <span class="author">Majumder P</span>, <span class="author">Rudan P</span>, <span class="author">Primorac D</span>, <span class="author">Villems R</span> and <span class="author">Kivisild T</span></p>
<p class="affiliation">Division of Child and Adolescent Psychiatry and Child Development, Department of Psychiatry and Behavioral Sciences, Stanford University School of Medicine, 1201 Welch Road, Stanford, CA 94304-5485, USA. <a href="mailto:%75%6E%64%65%72%40%73%74%61%6E%66%6F%72%64.%65%64%75">&#117;&#110;&#100;&#101;&#114;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Human Y-chromosome haplogroup structure is largely circumscribed by continental boundaries. One notable exception to this general pattern is the young haplogroup R1a that exhibits post-Glacial coalescent times and relates the paternal ancestry of more than 10% of men in a wide geographic area extending from South Asia to Central East Europe and South Siberia. Its origin and dispersal patterns are poorly understood as no marker has yet been described that would distinguish European R1a chromosomes from Asian. Here we present frequency and haplotype diversity estimates for more than 2000 R1a chromosomes assessed for several newly discovered SNP markers that introduce the onset of informative R1a subdivisions by geography. Marker M434 has a low frequency and a late origin in West Asia bearing witness to recent gene flow over the Arabian Sea. Conversely, marker M458 has a significant frequency in Europe, exceeding 30% in its core area in Eastern Europe and comprising up to 70% of all M17 chromosomes present there. The diversity and frequency profiles of M458 suggest its origin during the early Holocene and a subsequent expansion likely related to a number of prehistoric cultural developments in the region. Its primary frequency and diversity distribution correlates well with some of the major Central and East European river basins where settled farming was established before its spread further eastward. Importantly, the virtual absence of M458 chromosomes outside Europe speaks against substantial patrilineal gene flow from East Europe to Asia, including to India, at least since the mid-Holocene.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2010</span>;<span class="volume">18</span>;<span class="number">4</span>;<span class="pagerange">479-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19888303">19888303</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2987245">2987245</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2009.194">10.1038/ejhg.2009.194</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Swedish new variant of Chlamydia trachomatis: genome sequence, morphology, cell tropism and phenotypic characterization.</h4>
<p class="authors"><span class="author">Unemo M</span>, <span class="author">Seth-Smith HM</span>, <span class="author">Cutcliffe LT</span>, <span class="author">Skilton RJ</span>, <span class="author">Barlow D</span>, <span class="author">Goulding D</span>, <span class="author">Persson K</span>, <span class="author">Harris SR</span>, <span class="author">Kelly A</span>, <span class="author">Bjartling C</span>, <span class="author">Fredlund H</span>, <span class="author">Olc&eacute;n P</span>, <span class="author">Thomson NR</span> and <span class="author">Clarke IN</span></p>
<p class="affiliation">National Reference Laboratory for Pathogenic Neisseria, Department of Laboratory Medicine, Orebro University Hospital, Orebro, Sweden. <a href="mailto:%6D%61%67%6E%75%73.%75%6E%65%6D%6F%40%6F%72%65%62%72%6F%6C%6C.%73%65">&#109;&#97;&#103;&#110;&#117;&#115;&#46;&#117;&#110;&#101;&#109;&#111;&#64;&#111;&#114;&#101;&#98;&#114;&#111;&#108;&#108;&#46;&#115;&#101;</a></p><p class="abstract">Chlamydia trachomatis is a major cause of bacterial sexually transmitted infections worldwide. In 2006, a new variant of C. trachomatis (nvCT), carrying a 377 bp deletion within the plasmid, was reported in Sweden. This deletion included the targets used by the commercial diagnostic systems from Roche and Abbott. The nvCT is clonal (serovar/genovar E) and it spread rapidly in Sweden, undiagnosed by these systems. The degree of spread may also indicate an increased biological fitness of nvCT. The aims of this study were to describe the genome of nvCT, to compare the nvCT genome to all available C. trachomatis genome sequences and to investigate the biological properties of nvCT. An early nvCT isolate (Sweden2) was analysed by genome sequencing, growth kinetics, microscopy, cell tropism assay and antimicrobial susceptibility testing. It was compared with relevant C. trachomatis isolates, including a similar serovar E C. trachomatis wild-type strain that circulated in Sweden prior to the initially undetected expansion of nvCT. The nvCT genome does not contain any major genetic polymorphisms - the genes for central metabolism, development cycle and virulence are conserved - or phenotypic characteristics that indicate any altered biological fitness. This is supported by the observations that the nvCT and wild-type C. trachomatis infections are very similar in terms of epidemiological distribution, and that differences in clinical signs are only described, in one study, in women. In conclusion, the nvCT does not appear to have any altered biological fitness. Therefore, the rapid transmission of nvCT in Sweden was due to the strong diagnostic selective advantage and its introduction into a high-frequency transmitting population.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">080348</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">156</span>;<span class="number">Pt 5</span>;<span class="pagerange">1394-404</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20093289">20093289</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3541825">3541825</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.036830-0">10.1099/mic.0.036830-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss of Rb proteins causes genomic instability in the absence of mitogenic signaling.</h4>
<p class="authors"><span class="author">van Harn T</span>, <span class="author">Foijer F</span>, <span class="author">van Vugt M</span>, <span class="author">Banerjee R</span>, <span class="author">Yang F</span>, <span class="author">Oostra A</span>, <span class="author">Joenje H</span> and <span class="author">te Riele H</span></p>
<p class="affiliation">Division of Molecular Biology, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</p><p class="abstract">Loss of G1/S control is a hallmark of cancer, and is often caused by inactivation of the retinoblastoma pathway. However, mouse embryonic fibroblasts lacking the retinoblastoma genes RB1, p107, and p130 (TKO MEFs) are still subject to cell cycle control: Upon mitogen deprivation, they enter and complete S phase, but then firmly arrest in G2. We now show that G2-arrested TKO MEFs have accumulated DNA damage. Upon mitogen readdition, cells resume proliferation, although only part of the damage is repaired. As a result, mitotic cells show chromatid breaks and chromatid cohesion defects. These aberrations lead to aneuploidy in the descendent cell population. Thus, our results demonstrate that unfavorable growth conditions can cause genomic instability in cells lacking G1/S control. This mechanism may allow premalignant tumor cells to acquire additional genetic alterations that promote tumorigenesis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">WT077008</span></p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2010</span>;<span class="volume">24</span>;<span class="number">13</span>;<span class="pagerange">1377-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20551164" rel="external">20551164</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2895197" rel="external">2895197</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.580710" rel="external">10.1101/gad.580710</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chemokine ligand 2 genetic variants, serum monocyte chemoattractant protein-1 levels, and the risk of coronary artery disease.</h4>
<p class="authors"><span class="author">van Wijk DF</span>, <span class="author">van Leuven SI</span>, <span class="author">Sandhu MS</span>, <span class="author">Tanck MW</span>, <span class="author">Hutten BA</span>, <span class="author">Wareham NJ</span>, <span class="author">Kastelein JJ</span>, <span class="author">Stroes ES</span>, <span class="author">Khaw KT</span> and <span class="author">Boekholdt SM</span></p>
<p class="affiliation">Department of Vascular Medicine, Academic Medical Center, Amsterdam, the Netherlands. <a href="mailto:%64.%66.%76%61%6E%77%69%6A%6B%40%61%6D%63.%75%76%61.%6E%6C">&#100;&#46;&#102;&#46;&#118;&#97;&#110;&#119;&#105;&#106;&#107;&#64;&#97;&#109;&#99;&#46;&#117;&#118;&#97;&#46;&#110;&#108;</a></p><p class="abstract"><strong>Objective</strong>: In humans, evidence about the association between levels of monocyte chemoattractant protein-1 (MCP-1), its coding gene chemokine (C-C motif) ligand 2 (CCL2), and risk of coronary artery disease (CAD) is contradictory.</p><p class="abstract">We performed a nested case-control study in the prospective EPIC-Norfolk cohort investigating the relationship between CCL2 single-nucleotide polymorphisms (SNPs), MCP-1 concentrations, and the risk of future CAD. Cases (n=1138) were apparently healthy men and women aged 45 to 79 years who developed fatal or nonfatal CAD during a mean follow-up of 6 years. Controls (n=2237) were matched by age, sex, and enrollment time. Using linear regression analysis no association between CCL2 SNPs and MCP-1 serum concentrations became apparent, nor did we find a significant association between MCP-1 serum levels and risk of future CAD. Finally, Cox regression analysis showed no significant association between CCL2 SNPs and the future CAD risk. In addition, we did not find any robust associations between the CCL2 haplotypes and MCP-1 serum concentration or future CAD risk.</p><p class="abstract"><strong>Conclusions</strong>: Our data do not support previous publications indicating that MCP-1 is involved in the pathogenesis of CAD.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council</p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">7</span>;<span class="pagerange">1460-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20431065">20431065</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.110.205526">10.1161/ATVBAHA.110.205526</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somatic structural rearrangements in genetically engineered mouse mammary tumors.</h4>
<p class="authors"><span class="author">Varela I</span>, <span class="author">Klijn C</span>, <span class="author">Stephens PJ</span>, <span class="author">Mudie LJ</span>, <span class="author">Stebbings L</span>, <span class="author">Galappaththige D</span>, <span class="author">van der Gulden H</span>, <span class="author">Schut E</span>, <span class="author">Klarenbeek S</span>, <span class="author">Campbell PJ</span>, <span class="author">Wessels LF</span>, <span class="author">Stratton MR</span>, <span class="author">Jonkers J</span>, <span class="author">Futreal PA</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB101SA, UK. <a href="mailto:%69%76%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#118;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Here we present the first paired-end sequencing of tumors from genetically engineered mouse models of cancer to determine how faithfully these models recapitulate the landscape of somatic rearrangements found in human tumors. These were models of Trp53-mutated breast cancer, Brca1- and Brca2-associated hereditary breast cancer, and E-cadherin (Cdh1) mutated lobular breast cancer.</p><p class="abstract"><strong>Results</strong>: We show that although Brca1- and Brca2-deficient mouse mammary tumors have a defect in the homologous recombination pathway, there is no apparent difference in the type or frequency of somatic rearrangements found in these cancers when compared to other mouse mammary cancers, and tumors from all genetic backgrounds showed evidence of microhomology-mediated repair and non-homologous end-joining processes. Importantly, mouse mammary tumors were found to carry fewer structural rearrangements than human mammary cancers and expressed in-frame fusion genes. Like the fusion genes found in human mammary tumors, these were not recurrent. One mouse tumor was found to contain an internal deletion of exons of the Lrp1b gene, which led to a smaller in-frame transcript. We found internal in-frame deletions in the human ortholog of this gene in a significant number (4.2%) of human cancer cell lines.</p><p class="abstract"><strong>Conclusions</strong>: Paired-end sequencing of mouse mammary tumors revealed that they display significant heterogeneity in their profiles of somatic rearrangement but, importantly, fewer rearrangements than cognate human mammary tumors, probably because these cancers have been induced by strong driver mutations engineered into the mouse genome. Both human and mouse mammary cancers carry expressed fusion genes and conserved homozygous deletions.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council: <span class="grant">G0800024</span>; Wellcome Trust: <span class="grant">088340</span>, <span class="grant">093867</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">R100</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20942901">20942901</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3218656">3218656</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-10-r100">10.1186/gb-2010-11-10-r100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The use of DNA transposons for cancer gene discovery in mice.</h4>
<p class="authors"><span class="author">Vassiliou G</span>, <span class="author">Rad R</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Insertional mutagenesis in mice is a potent instrument for cancer gene discovery. Until recently, retroviruses were the main experimental tools in this field and application of insertional mutagenesis was limited to tissues for which these agents have tropism, namely hemopoietic cells and mammary epithelium. However, the field has been revolutionized and greatly expanded with the recent reanimation of the transposons, a highly flexible group of insertional mutagens first discovered in maize, which have now been adapted for use in mammalian cells. Transposons do not only extend the application of insertional mutagenesis to any tissue of choice, but also allow a more extensive and unbiased coverage of the genome, can be designed to selectively activate or inactivate genes, and are highly amenable to temporal and spatial control. This chapter gives an overview of the design and application of transposons to cancer gene discovery in mice.</p>
<p class="publication"><span class="title">Methods in enzymology</span>&nbsp;<span class="year">2010</span>;<span class="volume">477</span>;<span class="pagerange">91-106</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20699138">20699138</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0076-6879%2810%2977006-3">10.1016/S0076-6879(10)77006-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Molecular Basis of Leukaemia and Lymphoma</h4>
<p class="authors"><span class="author">Vassiliou,G.S.</span> and <span class="author">Green,A.R.;</span></p>
<p class="publication"><span class="title">Postgraduate Haematology</span>&nbsp;<span class="year">2010</span>;<span class="number">Chapter 21</span>;<span class="pagerange">380-94</span></p>
<p class="links">DOI: <a href="http://dx.doi.org/10.1002%2F9781444323160.ch21">10.1002/9781444323160.ch21</a>; URL: <a title="http://onlinelibrary.wiley.com/doi/10.1002/9781444323160.ch21/summary" href="http://onlinelibrary.wiley.com/doi/10.1002/9781444323160.ch21/summary">http://onlinelibrary.wiley.com/doi/<span class="print-hide">...</span><span class="print-show">10.1002/9781444323160.ch21/su</span>mmary</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variation at the phospholipid transfer protein locus affects its activity and high-density lipoprotein size and is a novel marker of cardiovascular disease susceptibility.</h4>
<p class="authors"><span class="author">Vergeer M</span>, <span class="author">Boekholdt SM</span>, <span class="author">Sandhu MS</span>, <span class="author">Ricketts SL</span>, <span class="author">Wareham NJ</span>, <span class="author">Brown MJ</span>, <span class="author">de Faire U</span>, <span class="author">Leander K</span>, <span class="author">Gigante B</span>, <span class="author">Kavousi M</span>, <span class="author">Hofman A</span>, <span class="author">Uitterlinden AG</span>, <span class="author">van Duijn CM</span>, <span class="author">Witteman JC</span>, <span class="author">Jukema JW</span>, <span class="author">Schadt EE</span>, <span class="author">van der Schoot E</span>, <span class="author">Kastelein JJ</span>, <span class="author">Khaw KT</span>, <span class="author">Dullaart RP</span>, <span class="author">van Tol A</span>, <span class="author">Trip MD</span> and <span class="author">Dallinga-Thie GM</span></p>
<p class="affiliation">Department of Vascular Medicine, Academic Medical Center, 1105 AZ Amsterdam, the Netherlands. <a href="mailto:%6D.%76%65%72%67%65%65%72%40%61%6D%63.%75%76%61.%6E%6C">&#109;&#46;&#118;&#101;&#114;&#103;&#101;&#101;&#114;&#64;&#97;&#109;&#99;&#46;&#117;&#118;&#97;&#46;&#110;&#108;</a></p><p class="abstract"><strong>Background</strong>: In contrast to clear associations between variants in genes participating in low-density lipoprotein metabolism and cardiovascular disease risk, such associations for high-density lipoprotein (HDL)-related genes are not well supported by recent large studies. We aimed to determine whether genetic variants at the locus encoding phospholipid transfer protein (PLTP), a protein involved in HDL remodeling, underlie altered PLTP activity, HDL particle concentration and size, and cardiovascular disease risk.</p><p class="abstract">We assessed associations between 6 PLTP tagging single nucleotide polymorphisms and PLTP activity in 2 studies (combined n=384) and identified 2 variants that show reproducible associations with altered plasma PLTP activity. A gene score based on these variants is associated with lower hepatic PLTP transcription (P=3.2x10(-18)) in a third study (n=957) and with an increased number of HDL particles of smaller size (P=3.4x10(-17)) in a fourth study (n=3375). In a combination of 5 cardiovascular disease case-control studies (n=4658 cases and 11 459 controls), a higher gene score was associated with a lower cardiovascular disease risk (per-allele odds ratio, 0.94; 95% confidence interval, 0.90 to 0.98; P=1.2x10(-3); odds ratio for highest versus lowest gene score, 0.69; 95% confidence interval, 0.55 to 0.86; P=1.0x10(-3)).</p><p class="abstract"><strong>Conclusions</strong>: A gene score based on 2 PLTP single nucleotide polymorphisms is associated with lower PLTP transcription and activity, an increased number of HDL particles, smaller HDL size, and decreased risk of cardiovascular disease. These findings indicate that PLTP is a proatherogenic entity and suggest that modulation of specific elements of HDL metabolism may offer cardiovascular benefit.</p>
<p class="grants">Funded by: British Heart Foundation; Cancer Research UK: <span class="grant">C490/A11021</span>, <span class="grant">C8197/A1012</span></p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2010</span>;<span class="volume">122</span>;<span class="number">5</span>;<span class="pagerange">470-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20644014" rel="external">20644014</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.109.912519" rel="external">10.1161/CIRCULATIONAHA.109.912519</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The pyramid of knowledge.</h4>
<p class="authors"><span class="author">Vernikos GS</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20075924" rel="external">20075924</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2300" rel="external">10.1038/nrmicro2300</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinct patterns of 1p and 19q alterations identify subtypes of human gliomas that have different prognoses.</h4>
<p class="authors"><span class="author">Vogazianou AP</span>, <span class="author">Chan R</span>, <span class="author">B&auml;cklund LM</span>, <span class="author">Pearson DM</span>, <span class="author">Liu L</span>, <span class="author">Langford CF</span>, <span class="author">Gregory SG</span>, <span class="author">Collins VP</span> and <span class="author">Ichimura K</span></p>
<p class="affiliation">Department of Pathology, Division of Molecular Histopathology, University of Cambridge, Level 3, Lab Block, Addenbrooke&#39;s Hospital, Box 231, Cambridge CB2 0QQ, UK.</p><p class="abstract">We studied the status of chromosomes 1 and 19 in 363 astrocytic and oligodendroglial tumors. Whereas the predominant pattern of copy number abnormality was a concurrent loss of the entire 1p and 19q regions (total 1p/19q loss) among oligodendroglial tumors and partial deletions of 1p and/or 19q in astrocytic tumors, a subset of apparently astrocytic tumors also had total 1p/19q loss. The presence of total 1p/19q loss was associated with longer survival of patients with all types of adult gliomas independent of age and diagnosis (P = .041). The most commonly deleted region on 19q in astrocytic tumors spans 885&nbsp;kb in 19q13.33-q13.41, which is telomeric to the previously proposed region. Novel regions of homozygous deletion, including a part of DPYD (1p21.3) or the KLK cluster (19q13.33), were observed in anaplastic oligodendrogliomas. Amplifications encompassing AKT2 (19q13.2) or CCNE1 (19q12) were identified in some glioblastomas. Deletion mapping of the centromeric regions of 1p and 19q in the tumors that had total 1p/19q loss, indicating that the breakpoints lie centromeric to NOTCH2 within the pericentromeric regions of 1p and 19q. Thus, we show that the copy number abnormalities of 1p and 19q in human gliomas are complex and have distinct patterns that are prognostically predictive independent of age and pathological diagnosis. An accurate identification of total 1p/19q loss and discriminating this from other 1p/19q changes is, however, critical when the 1p/19q copy number status is used to stratify patients in clinical trials.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6618</span></p>
<p class="publication"><span class="title">Neuro-oncology</span>&nbsp;<span class="year">2010</span>;<span class="volume">12</span>;<span class="number">7</span>;<span class="pagerange">664-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20164239" rel="external">20164239</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2940668" rel="external">2940668</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fneuonc%2Fnop075" rel="external">10.1093/neuonc/nop075</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.</h4>
<p class="authors"><span class="author">Voight BF</span>, <span class="author">Scott LJ</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Morris AP</span>, <span class="author">Dina C</span>, <span class="author">Welch RP</span>, <span class="author">Zeggini E</span>, <span class="author">Huth C</span>, <span class="author">Aulchenko YS</span>, <span class="author">Thorleifsson G</span>, <span class="author">McCulloch LJ</span>, <span class="author">Ferreira T</span>, <span class="author">Grallert H</span>, <span class="author">Amin N</span>, <span class="author">Wu G</span>, <span class="author">Willer CJ</span>, <span class="author">Raychaudhuri S</span>, <span class="author">McCarroll SA</span>, <span class="author">Langenberg C</span>, <span class="author">Hofmann OM</span>, <span class="author">Dupuis J</span>, <span class="author">Qi L</span>, <span class="author">Segr&egrave; AV</span>, <span class="author">van Hoek M</span>, <span class="author">Navarro P</span>, <span class="author">Ardlie K</span>, <span class="author">Balkau B</span>, <span class="author">Benediktsson R</span>, <span class="author">Bennett AJ</span>, <span class="author">Blagieva R</span>, <span class="author">Boerwinkle E</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Bengtsson Bostr&ouml;m K</span>, <span class="author">Bravenboer B</span>, <span class="author">Bumpstead S</span>, <span class="author">Burtt NP</span>, <span class="author">Charpentier G</span>, <span class="author">Chines PS</span>, <span class="author">Cornelis M</span>, <span class="author">Couper DJ</span>, <span class="author">Crawford G</span>, <span class="author">Doney AS</span>, <span class="author">Elliott KS</span>, <span class="author">Elliott AL</span>, <span class="author">Erdos MR</span>, <span class="author">Fox CS</span>, <span class="author">Franklin CS</span>, <span class="author">Ganser M</span>, <span class="author">Gieger C</span>, <span class="author">Grarup N</span>, <span class="author">Green T</span>, <span class="author">Griffin S</span>, <span class="author">Groves CJ</span>, <span class="author">Guiducci C</span>, <span class="author">Hadjadj S</span>, <span class="author">Hassanali N</span>, <span class="author">Herder C</span>, <span class="author">Isomaa B</span>, <span class="author">Jackson AU</span>, <span class="author">Johnson PR</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Kao WH</span>, <span class="author">Klopp N</span>, <span class="author">Kong A</span>, <span class="author">Kraft P</span>, <span class="author">Kuusisto J</span>, <span class="author">Lauritzen T</span>, <span class="author">Li M</span>, <span class="author">Lieverse A</span>, <span class="author">Lindgren CM</span>, <span class="author">Lyssenko V</span>, <span class="author">Marre M</span>, <span class="author">Meitinger T</span>, <span class="author">Midthjell K</span>, <span class="author">Morken MA</span>, <span class="author">Narisu N</span>, <span class="author">Nilsson P</span>, <span class="author">Owen KR</span>, <span class="author">Payne F</span>, <span class="author">Perry JR</span>, <span class="author">Petersen AK</span>, <span class="author">Platou C</span>, <span class="author">Proen&ccedil;a C</span>, <span class="author">Prokopenko I</span>, <span class="author">Rathmann W</span>, <span class="author">Rayner NW</span>, <span class="author">Robertson NR</span>, <span class="author">Rocheleau G</span>, <span class="author">Roden M</span>, <span class="author">Sampson MJ</span>, <span class="author">Saxena R</span>, <span class="author">Shields BM</span>, <span class="author">Shrader P</span>, <span class="author">Sigurdsson G</span>, <span class="author">Spars&oslash; T</span>, <span class="author">Strassburger K</span>, <span class="author">Stringham HM</span>, <span class="author">Sun Q</span>, <span class="author">Swift AJ</span>, <span class="author">Thorand B</span>, <span class="author">Tichet J</span>, <span class="author">Tuomi T</span>, <span class="author">van Dam RM</span>, <span class="author">van Haeften TW</span>, <span class="author">van Herpt T</span>, <span class="author">van Vliet-Ostaptchouk JV</span>, <span class="author">Walters GB</span>, <span class="author">Weedon MN</span>, <span class="author">Wijmenga C</span>, <span class="author">Witteman J</span>, <span class="author">Bergman RN</span>, <span class="author">Cauchi S</span>, <span class="author">Collins FS</span>, <span class="author">Gloyn AL</span>, <span class="author">Gyllensten U</span>, <span class="author">Hansen T</span>, <span class="author">Hide WA</span>, <span class="author">Hitman GA</span>, <span class="author">Hofman A</span>, <span class="author">Hunter DJ</span>, <span class="author">Hveem K</span>, <span class="author">Laakso M</span>, <span class="author">Mohlke KL</span>, <span class="author">Morris AD</span>, <span class="author">Palmer CN</span>, <span class="author">Pramstaller PP</span>, <span class="author">Rudan I</span>, <span class="author">Sijbrands E</span>, <span class="author">Stein LD</span>, <span class="author">Tuomilehto J</span>, <span class="author">Uitterlinden A</span>, <span class="author">Walker M</span>, <span class="author">Wareham NJ</span>, <span class="author">Watanabe RM</span>, <span class="author">Abecasis GR</span>, <span class="author">Boehm BO</span>, <span class="author">Campbell H</span>, <span class="author">Daly MJ</span>, <span class="author">Hattersley AT</span>, <span class="author">Hu FB</span>, <span class="author">Meigs JB</span>, <span class="author">Pankow JS</span>, <span class="author">Pedersen O</span>, <span class="author">Wichmann HE</span>, <span class="author">Barroso I</span>, <span class="author">Florez JC</span>, <span class="author">Frayling TM</span>, <span class="author">Groop L</span>, <span class="author">Sladek R</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Wilson JF</span>, <span class="author">Illig T</span>, <span class="author">Froguel P</span>, <span class="author">van Duijn CM</span>, <span class="author">Stefansson K</span>, <span class="author">Altshuler D</span>, <span class="author">Boehnke M</span>, <span class="author">McCarthy MI</span>, <span class="author">MAGIC investigators</span> and <span class="author">GIANT Consortium</span></p>
<p class="affiliation">Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA, USA.</p><p class="abstract">By combining genome-wide association data from 8,130 individuals with type 2 diabetes (T2D) and 38,987 controls of European descent and following up previously unidentified meta-analysis signals in a further 34,412 cases and 59,925 controls, we identified 12 new T2D association signals with combined P&lt;5x10(-8). These include a second independent signal at the KCNQ1 locus; the first report, to our knowledge, of an X-chromosomal association (near DUSP9); and a further instance of overlap between loci implicated in monogenic and multifactorial forms of diabetes (at HNF1A). The identified loci affect both beta-cell function and insulin action, and, overall, T2D association signals show evidence of enrichment for genes involved in cell cycle regulation. We also show that a high proportion of T2D susceptibility loci harbor independent association signals influencing apparently unrelated complex traits.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/710</span>; Department of Health: <span class="grant">DHCS/07/07/008</span>; Medical Research Council: <span class="grant">G0601261</span>, <span class="grant">G0700222</span>, <span class="grant">G0700222(81696)</span>, <span class="grant">G0701863</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179474</span>, <span class="grant">MC_U127592696</span>; NCRR NIH HHS: <span class="grant">UL1RR025005</span>; NHGRI NIH HHS: <span class="grant">1 Z01 HG000024</span>, <span class="grant">U01HG004171</span>, <span class="grant">U01HG004399</span>, <span class="grant">U01HG004402</span>; NHLBI NIH HHS: <span class="grant">1K99HL094535-01A1</span>, <span class="grant">N01-HC-25195</span>, <span class="grant">N01-HC-55015</span>, <span class="grant">N01-HC-55016</span>, <span class="grant">N01-HC-55018</span>, <span class="grant">N01-HC-55019</span>, <span class="grant">N01-HC-55020</span>, <span class="grant">N01-HC-55021</span>, <span class="grant">N01-HC-55022</span>, <span class="grant">N02-HL-6-4278</span>, <span class="grant">R01HL086694</span>, <span class="grant">R01HL087641</span>, <span class="grant">R01HL59367</span>; NIAMS NIH HHS: <span class="grant">1K08AR055688</span>, <span class="grant">K08 AR055688</span>, <span class="grant">K08 AR055688-03</span>; NIDA NIH HHS: <span class="grant">U54 DA021519</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK069922</span>, <span class="grant">DK072193</span>, <span class="grant">DK073490</span>, <span class="grant">DK078616</span>, <span class="grant">DK58845</span>, <span class="grant">K23-DK65978</span>, <span class="grant">K24-DK080140</span>, <span class="grant">R01 DK029867</span>, <span class="grant">R01 DK072193</span>; PHS HHS: <span class="grant">HHSN268200625226C</span>; Wellcome Trust: <span class="grant">064890</span>, <span class="grant">072960</span>, <span class="grant">075491</span>, <span class="grant">076113</span>, <span class="grant">077016</span>, <span class="grant">079557</span>, <span class="grant">081682</span>, <span class="grant">083270</span>, <span class="grant">086596</span>, <span class="grant">088885</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">7</span>;<span class="pagerange">579-89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20581827">20581827</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3080658">3080658</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.609">10.1038/ng.609</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Emergence of the Asian 1 genotype of dengue virus serotype 2 in viet nam: in vivo fitness advantage and lineage replacement in South-East Asia.</h4>
<p class="authors"><span class="author">Vu TT</span>, <span class="author">Holmes EC</span>, <span class="author">Duong V</span>, <span class="author">Nguyen TQ</span>, <span class="author">Tran TH</span>, <span class="author">Quail M</span>, <span class="author">Churcher C</span>, <span class="author">Parkhill J</span>, <span class="author">Cardosa J</span>, <span class="author">Farrar J</span>, <span class="author">Wills B</span>, <span class="author">Lennon NJ</span>, <span class="author">Birren BW</span>, <span class="author">Buchy P</span>, <span class="author">Henn MR</span> and <span class="author">Simmons CP</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Hospital for Tropical Diseases, Ho Chi Minh City, Viet Nam.</p><p class="abstract">A better description of the extent and structure of genetic diversity in dengue virus (DENV) in endemic settings is central to its eventual control. To this end we determined the complete coding region sequence of 187 DENV-2 genomes and 68 E genes from viruses sampled from Vietnamese patients between 1995 and 2009. Strikingly, an episode of genotype replacement was observed, with Asian 1 lineage viruses entirely displacing the previously dominant Asian/American lineage viruses. This genotype replacement event also seems to have occurred within DENV-2 in Thailand and Cambodia, suggestive of a major difference in viral fitness. To determine the cause of this major evolutionary event we compared both the infectivity of the Asian 1 and Asian/American genotypes in mosquitoes and their viraemia levels in humans. Although there was little difference in infectivity in mosquitoes, we observed significantly higher plasma viraemia levels in paediatric patients infected with Asian 1 lineage viruses relative to Asian/American viruses, a phenotype that is predicted to result in a higher probability of human-to-mosquito transmission. These results provide a mechanistic basis to a marked change in the genetic structure of DENV-2 and more broadly underscore that an understanding of DENV evolutionary dynamics can inform the development of vaccines and anti-viral drugs.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">HHSN266200400001C</span>, <span class="grant">HHSN272200900006C</span>; Wellcome Trust: <span class="grant">084368</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2010</span>;<span class="volume">4</span>;<span class="number">7</span>;<span class="pagerange">e757</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20651932" rel="external">20651932</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2907417" rel="external">2907417</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000757" rel="external">10.1371/journal.pntd.0000757</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A glut from the gut: metagenomics takes a giant step foward.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">315</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20395965" rel="external">20395965</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2363" rel="external">10.1038/nrmicro2363</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gut metagenomics goes viral.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2010</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">841</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21060317" rel="external">21060317</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2476" rel="external">10.1038/nrmicro2476</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of two DNA microarrays for detection of plasmid-mediated antimicrobial resistance and virulence factor genes in clinical isolates of Enterobacteriaceae and non-Enterobacteriaceae.</h4>
<p class="authors"><span class="author">Walsh F</span>, <span class="author">Cooke NM</span>, <span class="author">Smith SG</span>, <span class="author">Moran GP</span>, <span class="author">Cooke FJ</span>, <span class="author">Ivens A</span>, <span class="author">Wain J</span> and <span class="author">Rogers TR</span></p>
<p class="affiliation">Department of Clinical Microbiology, Sir Patrick Dun Translational Research Laboratory, School of Medicine, University of Dublin, Trinity College, St James&#39;s Hospital Campus, Dublin 8, Ireland. <a href="mailto:%66%69%6F%6E%61%31%77%61%6C%73%68%40%67%6D%61%69%6C.%63%6F%6D">&#102;&#105;&#111;&#110;&#97;&#49;&#119;&#97;&#108;&#115;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">A DNA microarray was developed to detect plasmid-mediated antimicrobial resistance (AR) and virulence factor (VF) genes in clinical isolates of Enterobacteriaceae and non-Enterobacteriaceae. The array was validated with the following bacterial species: Escherichiacoli (n=17); Klebsiellapneumoniae (n=3); Enterobacter spp. (n=6); Acinetobacter genospecies 3 (n=1); Acinetobacterbaumannii (n=1); Pseudomonasaeruginosa (n=2); and Stenotrophomonasmaltophilia (n=2). The AR gene profiles of these isolates were identified by polymerase chain reaction (PCR). The DNA microarray consisted of 155 and 133 AR and VF gene probes, respectively. Results were compared with the commercially available Identibac AMR-ve Array Tube. Hybridisation results indicated that there was excellent correlation between PCR and array results for AR and VF genes. Genes conferring resistance to each antibiotic class were identified by the DNA array. Unusual resistance genes were also identified, such as bla(SHV-5) in a bla(OXA-23)-positive carbapenem-resistant A. baumannii. The phylogenetic group of each E. coli isolate was verified by the array. These data demonstrate that it is possible to screen simultaneously for all important classes of mobile AR and VF genes in Enterobacteriaceae and non-Enterobacteriaceae whilst also assigning a correct phylogenetic group to E. coli isolates. Therefore, it is feasible to test clinical Gram-negative bacteria for all known AR genes and to provide important information regarding pathogenicity simultaneously.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">International journal of antimicrobial agents</span>&nbsp;<span class="year">2010</span>;<span class="volume">35</span>;<span class="number">6</span>;<span class="pagerange">593-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20356716">20356716</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijantimicag.2010.02.011">10.1016/j.ijantimicag.2010.02.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new highly penetrant form of obesity due to deletions on chromosome 16p11.2.</h4>
<p class="authors"><span class="author">Walters RG</span>, <span class="author">Jacquemont S</span>, <span class="author">Valsesia A</span>, <span class="author">de Smith AJ</span>, <span class="author">Martinet D</span>, <span class="author">Andersson J</span>, <span class="author">Falchi M</span>, <span class="author">Chen F</span>, <span class="author">Andrieux J</span>, <span class="author">Lobbens S</span>, <span class="author">Delobel B</span>, <span class="author">Stutzmann F</span>, <span class="author">El-Sayed Moustafa JS</span>, <span class="author">Ch&egrave;vre JC</span>, <span class="author">Lecoeur C</span>, <span class="author">Vatin V</span>, <span class="author">Bouquillon S</span>, <span class="author">Buxton JL</span>, <span class="author">Boute O</span>, <span class="author">Holder-Espinasse M</span>, <span class="author">Cuisset JM</span>, <span class="author">Lemaitre MP</span>, <span class="author">Ambresin AE</span>, <span class="author">Brioschi A</span>, <span class="author">Gaillard M</span>, <span class="author">Giusti V</span>, <span class="author">Fellmann F</span>, <span class="author">Ferrarini A</span>, <span class="author">Hadjikhani N</span>, <span class="author">Campion D</span>, <span class="author">Guilmatre A</span>, <span class="author">Goldenberg A</span>, <span class="author">Calmels N</span>, <span class="author">Mandel JL</span>, <span class="author">Le Caignec C</span>, <span class="author">David A</span>, <span class="author">Isidor B</span>, <span class="author">Cordier MP</span>, <span class="author">Dupuis-Girod S</span>, <span class="author">Labalme A</span>, <span class="author">Sanlaville D</span>, <span class="author">B&eacute;ri-Dexheimer M</span>, <span class="author">Jonveaux P</span>, <span class="author">Leheup B</span>, <span class="author">Ounap K</span>, <span class="author">Bochukova EG</span>, <span class="author">Henning E</span>, <span class="author">Keogh J</span>, <span class="author">Ellis RJ</span>, <span class="author">Macdermot KD</span>, <span class="author">van Haelst MM</span>, <span class="author">Vincent-Delorme C</span>, <span class="author">Plessis G</span>, <span class="author">Touraine R</span>, <span class="author">Philippe A</span>, <span class="author">Malan V</span>, <span class="author">Mathieu-Dramard M</span>, <span class="author">Chiesa J</span>, <span class="author">Blaumeiser B</span>, <span class="author">Kooy RF</span>, <span class="author">Caiazzo R</span>, <span class="author">Pigeyre M</span>, <span class="author">Balkau B</span>, <span class="author">Sladek R</span>, <span class="author">Bergmann S</span>, <span class="author">Mooser V</span>, <span class="author">Waterworth D</span>, <span class="author">Reymond A</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Kurg A</span>, <span class="author">Palta P</span>, <span class="author">Esko T</span>, <span class="author">Metspalu A</span>, <span class="author">Nelis M</span>, <span class="author">Elliott P</span>, <span class="author">Hartikainen AL</span>, <span class="author">McCarthy MI</span>, <span class="author">Peltonen L</span>, <span class="author">Carlsson L</span>, <span class="author">Jacobson P</span>, <span class="author">Sj&ouml;str&ouml;m L</span>, <span class="author">Huang N</span>, <span class="author">Hurles ME</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Farooqi IS</span>, <span class="author">M&auml;nnik K</span>, <span class="author">Jarvelin MR</span>, <span class="author">Pattou F</span>, <span class="author">Meyre D</span>, <span class="author">Walley AJ</span>, <span class="author">Coin LJ</span>, <span class="author">Blakemore AI</span>, <span class="author">Froguel P</span> and <span class="author">Beckmann JS</span></p>
<p class="affiliation">Section of Genomic Medicine, Imperial College London, London W12 0NN, UK.</p><p class="abstract">Obesity has become a major worldwide challenge to public health, owing to an interaction between the Western &#39;obesogenic&#39; environment and a strong genetic contribution. Recent extensive genome-wide association studies (GWASs) have identified numerous single nucleotide polymorphisms associated with obesity, but these loci together account for only a small fraction of the known heritable component. Thus, the &#39;common disease, common variant&#39; hypothesis is increasingly coming under challenge. Here we report a highly penetrant form of obesity, initially observed in 31 subjects who were heterozygous for deletions of at least 593 kilobases at 16p11.2 and whose ascertainment included cognitive deficits. Nineteen similar deletions were identified from GWAS data in 16,053 individuals from eight European cohorts. These deletions were absent from healthy non-obese controls and accounted for 0.7% of our morbid obesity cases (body mass index (BMI) &gt;or= 40 kg m(-2) or BMI standard deviation score &gt;or= 4; P = 6.4 x 10(-8), odds ratio 43.0), demonstrating the potential importance in common disease of rare variants with strong effects. This highlights a promising strategy for identifying missing heritability in obesity and other complex traits: cohorts with extreme phenotypes are likely to be enriched for rare variants, thereby improving power for their discovery. Subsequent analysis of the loci so identified may well reveal additional rare variants that further contribute to the missing heritability, as recently reported for SIM1 (ref. 3). The most productive approach may therefore be to combine the &#39;power of the extreme&#39; in small, well-phenotyped cohorts, with targeted follow-up in case-control and population cohorts.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>, <span class="grant">G0600331(77796)</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>, <span class="grant">5R01MH63706:02</span>; Wellcome Trust: <span class="grant">077014</span>, <span class="grant">079534</span>, <span class="grant">082390</span>, <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">463</span>;<span class="number">7281</span>;<span class="pagerange">671-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20130649" rel="external">20130649</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2880448" rel="external">2880448</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08727" rel="external">10.1038/nature08727</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Irradiated Blm-deficient mice are a highly tumor prone model for analysis of a broad spectrum of hematologic malignancies.</h4>
<p class="authors"><span class="author">Warren M</span>, <span class="author">Chung YJ</span>, <span class="author">Howat WJ</span>, <span class="author">Harrison H</span>, <span class="author">McGinnis R</span>, <span class="author">Hao X</span>, <span class="author">McCafferty J</span>, <span class="author">Fredrickson TN</span>, <span class="author">Bradley A</span> and <span class="author">Morse HC</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%76%77%40%70%61%74%68%6F%6C%6F%67%79%64%69%61%67%6E%6F%73%74%69%63%73.%63%6F%6D">&#109;&#118;&#119;&#64;&#112;&#97;&#116;&#104;&#111;&#108;&#111;&#103;&#121;&#100;&#105;&#97;&#103;&#110;&#111;&#115;&#116;&#105;&#99;&#115;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Mutations in the BLM gene cause human Bloom syndrome (BS), an autosomal recessive disorder of growth retardation, immunodeficiency and cancer predisposition. Homozygous null Blm(m3/m3) mice are cancer prone with a 5-fold increased risk of cancer compared with Blm(m3/+) and Blm(+/+) mice. Irradiation of Blm(m3/m3) mice increased the risk to 28-fold. Tumors occurred mainly in the hematopoietic system and were similar to those in BS based on detailed histologic and immunohistochemical analyses. Irradiated Blm-deficient mice thus provide a novel model for understanding accelerated malignancies in BS and a new platform for investigating the molecular basis for a wide range of hematopoietic neoplasms.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">ZIA AI000858-10</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Leukemia research</span>&nbsp;<span class="year">2010</span>;<span class="volume">34</span>;<span class="number">2</span>;<span class="pagerange">210-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19709744" rel="external">19709744</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2815150" rel="external">2815150</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.leukres.2009.06.007" rel="external">10.1016/j.leukres.2009.06.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of a songbird.</h4>
<p class="authors"><span class="author">Warren WC</span>, <span class="author">Clayton DF</span>, <span class="author">Ellegren H</span>, <span class="author">Arnold AP</span>, <span class="author">Hillier LW</span>, <span class="author">K&uuml;nstner A</span>, <span class="author">Searle S</span>, <span class="author">White S</span>, <span class="author">Vilella AJ</span>, <span class="author">Fairley S</span>, <span class="author">Heger A</span>, <span class="author">Kong L</span>, <span class="author">Ponting CP</span>, <span class="author">Jarvis ED</span>, <span class="author">Mello CV</span>, <span class="author">Minx P</span>, <span class="author">Lovell P</span>, <span class="author">Velho TA</span>, <span class="author">Ferris M</span>, <span class="author">Balakrishnan CN</span>, <span class="author">Sinha S</span>, <span class="author">Blatti C</span>, <span class="author">London SE</span>, <span class="author">Li Y</span>, <span class="author">Lin YC</span>, <span class="author">George J</span>, <span class="author">Sweedler J</span>, <span class="author">Southey B</span>, <span class="author">Gunaratne P</span>, <span class="author">Watson M</span>, <span class="author">Nam K</span>, <span class="author">Backstr&ouml;m N</span>, <span class="author">Smeds L</span>, <span class="author">Nabholz B</span>, <span class="author">Itoh Y</span>, <span class="author">Whitney O</span>, <span class="author">Pfenning AR</span>, <span class="author">Howard J</span>, <span class="author">V&ouml;lker M</span>, <span class="author">Skinner BM</span>, <span class="author">Griffin DK</span>, <span class="author">Ye L</span>, <span class="author">McLaren WM</span>, <span class="author">Flicek P</span>, <span class="author">Quesada V</span>, <span class="author">Velasco G</span>, <span class="author">Lopez-Otin C</span>, <span class="author">Puente XS</span>, <span class="author">Olender T</span>, <span class="author">Lancet D</span>, <span class="author">Smit AF</span>, <span class="author">Hubley R</span>, <span class="author">Konkel MK</span>, <span class="author">Walker JA</span>, <span class="author">Batzer MA</span>, <span class="author">Gu W</span>, <span class="author">Pollock DD</span>, <span class="author">Chen L</span>, <span class="author">Cheng Z</span>, <span class="author">Eichler EE</span>, <span class="author">Stapley J</span>, <span class="author">Slate J</span>, <span class="author">Ekblom R</span>, <span class="author">Birkhead T</span>, <span class="author">Burke T</span>, <span class="author">Burt D</span>, <span class="author">Scharff C</span>, <span class="author">Adam I</span>, <span class="author">Richard H</span>, <span class="author">Sultan M</span>, <span class="author">Soldatov A</span>, <span class="author">Lehrach H</span>, <span class="author">Edwards SV</span>, <span class="author">Yang SP</span>, <span class="author">Li X</span>, <span class="author">Graves T</span>, <span class="author">Fulton L</span>, <span class="author">Nelson J</span>, <span class="author">Chinwalla A</span>, <span class="author">Hou S</span>, <span class="author">Mardis ER</span> and <span class="author">Wilson RK</span></p>
<p class="affiliation">The Genome Center, Washington University School of Medicine, Campus Box 8501, 4444 Forest Park Avenue, St Louis, Missouri 63108, USA. <a href="mailto:%77%77%61%72%72%65%6E%40%77%61%74%73%6F%6E.%77%75%73%74%6C.%65%64%75">&#119;&#119;&#97;&#114;&#114;&#101;&#110;&#64;&#119;&#97;&#116;&#115;&#111;&#110;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The zebra finch is an important model organism in several fields with unique relevance to human neuroscience. Like other songbirds, the zebra finch communicates through learned vocalizations, an ability otherwise documented only in humans and a few other animals and lacking in the chicken-the only bird with a sequenced genome until now. Here we present a structural, functional and comparative analysis of the genome sequence of the zebra finch (Taeniopygia guttata), which is a songbird belonging to the large avian order Passeriformes. We find that the overall structures of the genomes are similar in zebra finch and chicken, but they differ in many intrachromosomal rearrangements, lineage-specific gene family expansions, the number of long-terminal-repeat-based retrotransposons, and mechanisms of sex chromosome dosage compensation. We show that song behaviour engages gene regulatory networks in the zebra finch brain, altering the expression of long non-coding RNAs, microRNAs, transcription factors and their targets. We also show evidence for rapid molecular evolution in the songbird lineage of genes that are regulated during song experience. These results indicate an active involvement of the genome in neural processes underlying vocal communication and identify potential genetic substrates for the evolution and regulation of this behaviour.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBE0175091</span>; Howard Hughes Medical Institute; Medical Research Council: <span class="grant">MC_U137761446</span>; NHGRI NIH HHS: <span class="grant">U54 HG003079</span>; NIDA NIH HHS: <span class="grant">P30 DA018310</span>; NIDCD NIH HHS: <span class="grant">R01 DC007218</span>; NIGMS NIH HHS: <span class="grant">R01 GM059290</span>, <span class="grant">R01 GM085233</span>, <span class="grant">R01 GM59290</span>; NINDS NIH HHS: <span class="grant">R01 NS045264</span>, <span class="grant">R01NS051820</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7289</span>;<span class="pagerange">757-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20360741">20360741</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3187626">3187626</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08819">10.1038/nature08819</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic variants influencing circulating lipid levels and risk of coronary artery disease.</h4>
<p class="authors"><span class="author">Waterworth DM</span>, <span class="author">Ricketts SL</span>, <span class="author">Song K</span>, <span class="author">Chen L</span>, <span class="author">Zhao JH</span>, <span class="author">Ripatti S</span>, <span class="author">Aulchenko YS</span>, <span class="author">Zhang W</span>, <span class="author">Yuan X</span>, <span class="author">Lim N</span>, <span class="author">Luan J</span>, <span class="author">Ashford S</span>, <span class="author">Wheeler E</span>, <span class="author">Young EH</span>, <span class="author">Hadley D</span>, <span class="author">Thompson JR</span>, <span class="author">Braund PS</span>, <span class="author">Johnson T</span>, <span class="author">Struchalin M</span>, <span class="author">Surakka I</span>, <span class="author">Luben R</span>, <span class="author">Khaw KT</span>, <span class="author">Rodwell SA</span>, <span class="author">Loos RJ</span>, <span class="author">Boekholdt SM</span>, <span class="author">Inouye M</span>, <span class="author">Deloukas P</span>, <span class="author">Elliott P</span>, <span class="author">Schlessinger D</span>, <span class="author">Sanna S</span>, <span class="author">Scuteri A</span>, <span class="author">Jackson A</span>, <span class="author">Mohlke KL</span>, <span class="author">Tuomilehto J</span>, <span class="author">Roberts R</span>, <span class="author">Stewart A</span>, <span class="author">Kes&auml;niemi YA</span>, <span class="author">Mahley RW</span>, <span class="author">Grundy SM</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">McArdle W</span>, <span class="author">Cardon L</span>, <span class="author">Waeber G</span>, <span class="author">Vollenweider P</span>, <span class="author">Chambers JC</span>, <span class="author">Boehnke M</span>, <span class="author">Abecasis GR</span>, <span class="author">Salomaa V</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Ruokonen A</span>, <span class="author">Barroso I</span>, <span class="author">Epstein SE</span>, <span class="author">Hakonarson HH</span>, <span class="author">Rader DJ</span>, <span class="author">Reilly MP</span>, <span class="author">Witteman JC</span>, <span class="author">Hall AS</span>, <span class="author">Samani NJ</span>, <span class="author">Strachan DP</span>, <span class="author">Barter P</span>, <span class="author">van Duijn CM</span>, <span class="author">Kooner JS</span>, <span class="author">Peltonen L</span>, <span class="author">Wareham NJ</span>, <span class="author">McPherson R</span>, <span class="author">Mooser V</span> and <span class="author">Sandhu MS</span></p>
<p class="affiliation">Genetics Division, GlaxoSmithKline R&amp;D, King of Prussia, PA, USA.</p><p class="abstract"><strong>Objective</strong>: Genetic studies might provide new insights into the biological mechanisms underlying lipid metabolism and risk of CAD. We therefore conducted a genome-wide association study to identify novel genetic determinants of low-density lipoprotein cholesterol (LDL-C), high-density lipoprotein cholesterol (HDL-C), and triglycerides.</p><p class="abstract">We combined genome-wide association data from 8 studies, comprising up to 17 723 participants with information on circulating lipid concentrations. We did independent replication studies in up to 37 774 participants from 8 populations and also in a population of Indian Asian descent. We also assessed the association between single-nucleotide polymorphisms (SNPs) at lipid loci and risk of CAD in up to 9 633 cases and 38 684 controls. We identified 4 novel genetic loci that showed reproducible associations with lipids (probability values, 1.6&times;10(-8) to 3.1&times;10(-10)). These include a potentially functional SNP in the SLC39A8 gene for HDL-C, an SNP near the MYLIP/GMPR and PPP1R3B genes for LDL-C, and at the AFF1 gene for triglycerides. SNPs showing strong statistical association with 1 or more lipid traits at the CELSR2, APOB, APOE-C1-C4-C2 cluster, LPL, ZNF259-APOA5-A4-C3-A1 cluster and TRIB1 loci were also associated with CAD risk (probability values, 1.1&times;10(-3) to 1.2&times;10(-9)).</p><p class="abstract"><strong>Conclusions</strong>: We have identified 4 novel loci associated with circulating lipids. We also show that in addition to those that are largely associated with LDL-C, genetic loci mainly associated with circulating triglycerides and HDL-C are also associated with risk of CAD. These findings potentially provide new insights into the biological mechanisms underlying lipid metabolism and CAD risk.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">PG/08/094</span>, <span class="grant">PG/08/094/26019</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0401527</span>, <span class="grant">G0500539</span>, <span class="grant">G0601966</span>, <span class="grant">G0700931</span>, <span class="grant">G0701863</span>, <span class="grant">G0801566</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NHLBI NIH HHS: <span class="grant">5R01HL087679-02</span>; NIDDK NIH HHS: <span class="grant">R01 DK062370</span>, <span class="grant">U01 DK062370</span>, <span class="grant">U01 DK062418</span>; NIMH NIH HHS: <span class="grant">1RL1MH083268-01</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">077016/Z/05/Z</span>, <span class="grant">079895</span>, <span class="grant">GR069224</span></p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">30</span>;<span class="number">11</span>;<span class="pagerange">2264-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20864672">20864672</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3891568">3891568</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.109.201020">10.1161/ATVBAHA.109.201020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome sequencing highlights genetic changes associated with laboratory domestication of C. elegans.</h4>
<p class="authors"><span class="author">Weber KP</span>, <span class="author">De S</span>, <span class="author">Kozarewa I</span>, <span class="author">Turner DJ</span>, <span class="author">Babu MM</span> and <span class="author">de Bono M</span></p>
<p class="affiliation">Medical Research Council Laboratory of Molecular Biology, Cambridge, United Kingdom.</p><p class="abstract">Defining the mutational landscape when individuals of a species grow separately and diverge over many generations can provide insights into trait evolution. A specific example of this involves studying changes associated with domestication where different lines of the same wild stock have been cultivated independently in different standard environments. Whole genome sequence comparison of such lines permits estimation of mutation rates, inference of genes&#39; ancestral states and ancestry of existing strains, and correction of sequencing errors in genome databases. Here we study domestication of the C. elegans Bristol strain as a model, and report the genome sequence of LSJ1 (Bristol), a sibling of the standard C. elegans reference wild type N2 (Bristol). The LSJ1 and N2 lines were cultivated separately from shortly after the Bristol strain was isolated until methods to freeze C. elegans were developed. We find that during this time the two strains have accumulated 1208 genetic differences. We describe phenotypic variation between N2 and LSJ1 in the rate at which embryos develop, the rate of production of eggs, the maturity of eggs at laying, and feeding behavior, all the result of post-isolation changes. We infer the ancestral alleles in the original Bristol isolate and highlight 2038 likely sequencing errors in the original N2 reference genome sequence. Many of these changes modify genome annotation. Our study provides a starting point to further investigate genotype-phenotype association and offers insights into the process of selection as a result of laboratory domestication.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2010</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">e13922</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21085631" rel="external">21085631</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2978686" rel="external">2978686</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0013922" rel="external">10.1371/journal.pone.0013922</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of association between working memory and Reelin, a potential modifier gene in schizophrenia.</h4>
<p class="authors"><span class="author">Wedenoja J</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Suvisaari J</span>, <span class="author">Loukola A</span>, <span class="author">Paunio T</span>, <span class="author">Partonen T</span>, <span class="author">Varilo T</span>, <span class="author">L&ouml;nnqvist J</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland FIMM, University of Helsinki and National Institute for Health and Welfare, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: The challenges in gene identification for psychiatric disorders have awakened interest toward quantitative traits and endophenotypes that are potentially more closely related to the underlying biology and provide more power in the linkage and association analyses. Previously, we successfully replicated schizophrenia linkage on chromosome 7q21-32 in Finnish families and demonstrated that an intragenic short tandem repeat (STR) allele of the regional Reelin (RELN) gene is associated with multiple cognitive traits representing central cognitive functions regarded as valid endophenotypes for schizophrenia.</p><p class="abstract"><strong>Methods</strong>: Here, we used an extended sample of 290 Finnish families with schizophrenia and 375 control subjects in an association analysis between 96 SNPs and three STRs in RELN and diagnostic categories, clinical disorder features, as well as central cognitive functions impaired in schizophrenia.</p><p class="abstract"><strong>Results</strong>: We replicated the original association between RELN intragenic STR allele and working memory in individuals (n = 342) not overlapping with the previous study. This risk allele remained central in the whole study sample by being associated with impaired cognitive functioning and more severe positive and negative symptoms of schizophrenia (p = .0005-.00002). Additionally, multiple SNPs indicated association with the severity of positive symptoms of schizophrenia and together showed potential additive effect on the severity of the symptoms (p = .0000001). However, no significant associations with clinical diagnostic categories emerged.</p><p class="abstract"><strong>Conclusions</strong>: The strongest effects on cognitive functions were detected among the affected individuals. We thus propose a particular role for RELN as a modifier gene of the pathogenesis of schizophrenia.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2010</span>;<span class="volume">67</span>;<span class="number">10</span>;<span class="pagerange">983-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19922905" rel="external">19922905</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3083525" rel="external">3083525</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2009.09.026" rel="external">10.1016/j.biopsych.2009.09.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.</h4>
<p class="authors"><span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Craddock N</span>, <span class="author">Hurles ME</span>, <span class="author">Cardin N</span>, <span class="author">Pearson RD</span>, <span class="author">Plagnol V</span>, <span class="author">Robson S</span>, <span class="author">Vukcevic D</span>, <span class="author">Barnes C</span>, <span class="author">Conrad DF</span>, <span class="author">Giannoulatou E</span>, <span class="author">Holmes C</span>, <span class="author">Marchini JL</span>, <span class="author">Stirrups K</span>, <span class="author">Tobin MD</span>, <span class="author">Wain LV</span>, <span class="author">Yau C</span>, <span class="author">Aerts J</span>, <span class="author">Ahmad T</span>, <span class="author">Andrews TD</span>, <span class="author">Arbury H</span>, <span class="author">Attwood A</span>, <span class="author">Auton A</span>, <span class="author">Ball SG</span>, <span class="author">Balmforth AJ</span>, <span class="author">Barrett JC</span>, <span class="author">Barroso I</span>, <span class="author">Barton A</span>, <span class="author">Bennett AJ</span>, <span class="author">Bhaskar S</span>, <span class="author">Blaszczyk K</span>, <span class="author">Bowes J</span>, <span class="author">Brand OJ</span>, <span class="author">Braund PS</span>, <span class="author">Bredin F</span>, <span class="author">Breen G</span>, <span class="author">Brown MJ</span>, <span class="author">Bruce IN</span>, <span class="author">Bull J</span>, <span class="author">Burren OS</span>, <span class="author">Burton J</span>, <span class="author">Byrnes J</span>, <span class="author">Caesar S</span>, <span class="author">Clee CM</span>, <span class="author">Coffey AJ</span>, <span class="author">Connell JM</span>, <span class="author">Cooper JD</span>, <span class="author">Dominiczak AF</span>, <span class="author">Downes K</span>, <span class="author">Drummond HE</span>, <span class="author">Dudakia D</span>, <span class="author">Dunham A</span>, <span class="author">Ebbs B</span>, <span class="author">Eccles D</span>, <span class="author">Edkins S</span>, <span class="author">Edwards C</span>, <span class="author">Elliot A</span>, <span class="author">Emery P</span>, <span class="author">Evans DM</span>, <span class="author">Evans G</span>, <span class="author">Eyre S</span>, <span class="author">Farmer A</span>, <span class="author">Ferrier IN</span>, <span class="author">Feuk L</span>, <span class="author">Fitzgerald T</span>, <span class="author">Flynn E</span>, <span class="author">Forbes A</span>, <span class="author">Forty L</span>, <span class="author">Franklyn JA</span>, <span class="author">Freathy RM</span>, <span class="author">Gibbs P</span>, <span class="author">Gilbert P</span>, <span class="author">Gokumen O</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Gray E</span>, <span class="author">Green E</span>, <span class="author">Groves CJ</span>, <span class="author">Grozeva D</span>, <span class="author">Gwilliam R</span>, <span class="author">Hall A</span>, <span class="author">Hammond N</span>, <span class="author">Hardy M</span>, <span class="author">Harrison P</span>, <span class="author">Hassanali N</span>, <span class="author">Hebaishi H</span>, <span class="author">Hines S</span>, <span class="author">Hinks A</span>, <span class="author">Hitman GA</span>, <span class="author">Hocking L</span>, <span class="author">Howard E</span>, <span class="author">Howard P</span>, <span class="author">Howson JM</span>, <span class="author">Hughes D</span>, <span class="author">Hunt S</span>, <span class="author">Isaacs JD</span>, <span class="author">Jain M</span>, <span class="author">Jewell DP</span>, <span class="author">Johnson T</span>, <span class="author">Jolley JD</span>, <span class="author">Jones IR</span>, <span class="author">Jones LA</span>, <span class="author">Kirov G</span>, <span class="author">Langford CF</span>, <span class="author">Lango-Allen H</span>, <span class="author">Lathrop GM</span>, <span class="author">Lee J</span>, <span class="author">Lee KL</span>, <span class="author">Lees C</span>, <span class="author">Lewis K</span>, <span class="author">Lindgren CM</span>, <span class="author">Maisuria-Armer M</span>, <span class="author">Maller J</span>, <span class="author">Mansfield J</span>, <span class="author">Martin P</span>, <span class="author">Massey DC</span>, <span class="author">McArdle WL</span>, <span class="author">McGuffin P</span>, <span class="author">McLay KE</span>, <span class="author">Mentzer A</span>, <span class="author">Mimmack ML</span>, <span class="author">Morgan AE</span>, <span class="author">Morris AP</span>, <span class="author">Mowat C</span>, <span class="author">Myers S</span>, <span class="author">Newman W</span>, <span class="author">Nimmo ER</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Onipinla A</span>, <span class="author">Onyiah I</span>, <span class="author">Ovington NR</span>, <span class="author">Owen MJ</span>, <span class="author">Palin K</span>, <span class="author">Parnell K</span>, <span class="author">Pernet D</span>, <span class="author">Perry JR</span>, <span class="author">Phillips A</span>, <span class="author">Pinto D</span>, <span class="author">Prescott NJ</span>, <span class="author">Prokopenko I</span>, <span class="author">Quail MA</span>, <span class="author">Rafelt S</span>, <span class="author">Rayner NW</span>, <span class="author">Redon R</span>, <span class="author">Reid DM</span>, <span class="author">Renwick</span>, <span class="author">Ring SM</span>, <span class="author">Robertson N</span>, <span class="author">Russell E</span>, <span class="author">St Clair D</span>, <span class="author">Sambrook JG</span>, <span class="author">Sanderson JD</span>, <span class="author">Schuilenburg H</span>, <span class="author">Scott CE</span>, <span class="author">Scott R</span>, <span class="author">Seal S</span>, <span class="author">Shaw-Hawkins S</span>, <span class="author">Shields BM</span>, <span class="author">Simmonds MJ</span>, <span class="author">Smyth DJ</span>, <span class="author">Somaskantharajah E</span>, <span class="author">Spanova K</span>, <span class="author">Steer S</span>, <span class="author">Stephens J</span>, <span class="author">Stevens HE</span>, <span class="author">Stone MA</span>, <span class="author">Su Z</span>, <span class="author">Symmons DP</span>, <span class="author">Thompson JR</span>, <span class="author">Thomson W</span>, <span class="author">Travers ME</span>, <span class="author">Turnbull C</span>, <span class="author">Valsesia A</span>, <span class="author">Walker M</span>, <span class="author">Walker NM</span>, <span class="author">Wallace C</span>, <span class="author">Warren-Perry M</span>, <span class="author">Watkins NA</span>, <span class="author">Webster J</span>, <span class="author">Weedon MN</span>, <span class="author">Wilson AG</span>, <span class="author">Woodburn M</span>, <span class="author">Wordsworth BP</span>, <span class="author">Young AH</span>, <span class="author">Zeggini E</span>, <span class="author">Carter NP</span>, <span class="author">Frayling TM</span>, <span class="author">Lee C</span>, <span class="author">McVean G</span>, <span class="author">Munroe PB</span>, <span class="author">Palotie A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Scherer SW</span>, <span class="author">Strachan DP</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Brown MA</span>, <span class="author">Burton PR</span>, <span class="author">Caulfield MJ</span>, <span class="author">Compston A</span>, <span class="author">Farrall M</span>, <span class="author">Gough SC</span>, <span class="author">Hall AS</span>, <span class="author">Hattersley AT</span>, <span class="author">Hill AV</span>, <span class="author">Mathew CG</span>, <span class="author">Pembrey M</span>, <span class="author">Satsangi J</span>, <span class="author">Stratton MR</span>, <span class="author">Worthington J</span>, <span class="author">Deloukas P</span>, <span class="author">Duncanson A</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">McCarthy MI</span>, <span class="author">Ouwehand W</span>, <span class="author">Parkes M</span>, <span class="author">Rahman N</span>, <span class="author">Todd JA</span>, <span class="author">Samani NJ</span> and <span class="author">Donnelly P</span></p><p class="abstract">Copy number variants (CNVs) account for a major proportion of human genetic polymorphism and have been predicted to have an important role in genetic susceptibility to common disease. To address this we undertook a large, direct genome-wide study of association between CNVs and eight common human diseases. Using a purpose-designed array we typed approximately 19,000 individuals into distinct copy-number classes at 3,432 polymorphic CNVs, including an estimated approximately 50% of all common CNVs larger than 500 base pairs. We identified several biological artefacts that lead to false-positive associations, including systematic CNV differences between DNAs derived from blood and cell lines. Association testing and follow-up replication analyses confirmed three loci where CNVs were associated with disease-IRGM for Crohn&#39;s disease, HLA for Crohn&#39;s disease, rheumatoid arthritis and type 1 diabetes, and TSPAN8 for type 2 diabetes-although in each case the locus had previously been identified in single nucleotide polymorphism (SNP)-based studies, reflecting our observation that most common CNVs that are well-typed on our array are well tagged by SNPs and so have been indirectly explored through SNP studies. We conclude that common CNVs that can be typed on existing platforms are unlikely to contribute greatly to the genetic basis of common human diseases.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>; British Heart Foundation: <span class="grant">RG/09/012/28096</span>; Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/137</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0400874</span>, <span class="grant">G0500115</span>, <span class="grant">G0501942</span>, <span class="grant">G0600329</span>, <span class="grant">G0600705</span>, <span class="grant">G0700491</span>, <span class="grant">G0701003</span>, <span class="grant">G0701420</span>, <span class="grant">G0701810</span>, <span class="grant">G0701810(85517)</span>, <span class="grant">G0800383</span>, <span class="grant">G0800759</span>, <span class="grant">G19/9</span>, <span class="grant">G90/106</span>, <span class="grant">G9521010</span>, <span class="grant">MC_UP_A390_1107</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">083948</span>, <span class="grant">089989</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2010</span>;<span class="volume">464</span>;<span class="number">7289</span>;<span class="pagerange">713-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20360734">20360734</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2892339">2892339</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature08979">10.1038/nature08979</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Vgl1, a multi-KH domain protein, is a novel component of the fission yeast stress granules required for cell survival under thermal stress.</h4>
<p class="authors"><span class="author">Wen WL</span>, <span class="author">Stevenson AL</span>, <span class="author">Wang CY</span>, <span class="author">Chen HJ</span>, <span class="author">Kearsey SE</span>, <span class="author">Norbury CJ</span>, <span class="author">Watt S</span>, <span class="author">B&auml;hler J</span> and <span class="author">Wang SW</span></p>
<p class="affiliation">Division of Molecular and Genomic Medicine, National Health Research Institutes, 35 Keyan Road, Zhunan Town, Miaoli County 350, Taiwan.</p><p class="abstract">Multiple KH-domain proteins, collectively known as vigilins, are evolutionarily highly conserved proteins that are present in eukaryotic organisms from yeast to metazoa. Proposed roles for vigilins include chromosome segregation, messenger RNA (mRNA) metabolism, translation and tRNA transport. As a step toward understanding its biological function, we have identified the fission yeast vigilin, designated Vgl1, and have investigated its role in cellular response to environmental stress. Unlike its counterpart in Saccharomyces cerevisiae, we found no indication that Vgl1 is required for the maintenance of cell ploidy in Schizosaccharomyces pombe. Instead, Vgl1 is required for cell survival under thermal stress, and vgl1&Delta; mutants lose their viability more rapidly than wild-type cells when incubated at high temperature. As for Scp160 in S. cerevisiae, Vgl1 bound polysomes accumulated at endoplasmic reticulum (ER) but in a microtubule-independent manner. Under thermal stress, Vgl1 is rapidly relocalized from the ER to cytoplasmic foci that are distinct from P-bodies but contain stress granule markers such as poly(A)-binding protein and components of the translation initiation factor eIF3. Together, these observations demonstrated in S. pombe the presence of RNA granules with similar composition as mammalian stress granules and identified Vgl1 as a novel component that required for cell survival under thermal stress.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">19</span>;<span class="pagerange">6555-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20547592" rel="external">20547592</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2965253" rel="external">2965253</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq555" rel="external">10.1093/nar/gkq555</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fine-scale survey of X chromosome copy number variants and indels underlying intellectual disability.</h4>
<p class="authors"><span class="author">Whibley AC</span>, <span class="author">Plagnol V</span>, <span class="author">Tarpey PS</span>, <span class="author">Abidi F</span>, <span class="author">Fullston T</span>, <span class="author">Choma MK</span>, <span class="author">Boucher CA</span>, <span class="author">Shepherd L</span>, <span class="author">Willatt L</span>, <span class="author">Parkin G</span>, <span class="author">Smith R</span>, <span class="author">Futreal PA</span>, <span class="author">Shaw M</span>, <span class="author">Boyle J</span>, <span class="author">Licata A</span>, <span class="author">Skinner C</span>, <span class="author">Stevenson RE</span>, <span class="author">Turner G</span>, <span class="author">Field M</span>, <span class="author">Hackett A</span>, <span class="author">Schwartz CE</span>, <span class="author">Gecz J</span>, <span class="author">Stratton MR</span> and <span class="author">Raymond FL</span></p>
<p class="affiliation">Department of Medical Genetics, Cambridge Institute for Medical Research, Cambridge CB2 0XY, UK.</p><p class="abstract">Copy number variants and indels in 251 families with evidence of X-linked intellectual disability (XLID) were investigated by array comparative genomic hybridization on a high-density oligonucleotide X chromosome array platform. We identified pathogenic copy number variants in 10% of families, with mutations ranging from 2 kb to 11 Mb in size. The challenge of assessing causality was facilitated by prior knowledge of XLID-associated genes and the ability to test for cosegregation of variants with disease through extended pedigrees. Fine-scale analysis of rare variants in XLID families leads us to propose four additional genes, PTCHD1, WDR13, FAAH2, and GSPT2, as candidates for XLID causation and the identification of further deletions and duplications affecting X chromosome genes but without apparent disease consequences. Breakpoints of pathogenic variants were characterized to provide insight into the underlying mutational mechanisms and indicated a predominance of mitotic rather than meiotic events. By effectively bridging the gap between karyotype-level investigations and X chromosome exon resequencing, this study informs discussion of alternative mutational mechanisms, such as noncoding variants and non-X-linked disease, which might explain the shortfall of mutation yield in the well-characterized International Genetics of Learning Disability (IGOLD) cohort, where currently disease remains unexplained in two-thirds of families.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">87</span>;<span class="number">2</span>;<span class="pagerange">173-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20655035" rel="external">20655035</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2917707" rel="external">2917707</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.06.017" rel="external">10.1016/j.ajhg.2010.06.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinct variants at LIN28B influence growth in height from birth to adulthood.</h4>
<p class="authors"><span class="author">Wid&eacute;n E</span>, <span class="author">Ripatti S</span>, <span class="author">Cousminer DL</span>, <span class="author">Surakka I</span>, <span class="author">Lappalainen T</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Eriksson JG</span>, <span class="author">Raitakari O</span>, <span class="author">Salomaa V</span>, <span class="author">Sovio U</span>, <span class="author">Hartikainen AL</span>, <span class="author">Pouta A</span>, <span class="author">McCarthy MI</span>, <span class="author">Osmond C</span>, <span class="author">Kajantie E</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Viikari J</span>, <span class="author">K&auml;h&ouml;nen M</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Freimer N</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Peltonen L</span> and <span class="author">Palotie A</span></p>
<p class="affiliation">Institute for Molecular Medicine Finland, University of Helsinki, Helsinki, Finland. <a href="mailto:%65%6C%69%73%61%62%65%74%68.%77%69%64%65%6E%40%68%65%6C%73%69%6E%6B%69.%66%69">&#101;&#108;&#105;&#115;&#97;&#98;&#101;&#116;&#104;&#46;&#119;&#105;&#100;&#101;&#110;&#64;&#104;&#101;&#108;&#115;&#105;&#110;&#107;&#105;&#46;&#102;&#105;</a></p><p class="abstract">We have studied the largely unknown genetic underpinnings of height growth by using a unique resource of longitudinal childhood height data available in Finnish population cohorts. After applying GWAS mapping of potential genes influencing pubertal height growth followed by further characterization of the genetic effects on complete postnatal growth trajectories, we have identified strong association between variants near LIN28B and pubertal growth (rs7759938; female p = 4.0 x 10(-9), male p = 1.5 x 10(-4), combined p = 5.0 x 10(-11), n = 5038). Analysis of growth during early puberty confirmed an effect on the timing of the growth spurt. Correlated SNPs have previously been implicated as influencing both adult stature and age at menarche, the same alleles associating with taller height and later age of menarche in other studies as with later pubertal growth here. Additionally, a partially correlated LIN28B SNP, rs314277, has been associated previously with final height. Testing both rs7759938 and rs314277 (pairwise r(2) = 0.29) for independent effects on postnatal growth in 8903 subjects indicated that the pubertal timing-associated marker rs7759938 affects prepubertal growth in females (p = 7 x 10(-5)) and final height in males (p = 5 x 10(-4)), whereas rs314277 has sex-specific effects on growth (p for interaction = 0.005) that were distinct from those observed at rs7759938. In conclusion, partially correlated variants at LIN28B tag distinctive, complex, and sex-specific height-growth-regulating effects, influencing the entire period of postnatal growth. These findings imply a critical role for LIN28B in the regulation of human growth.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500539</span>; Wellcome Trust: <span class="grant">89061/Z/09/Z</span>, <span class="grant">WT089062</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">86</span>;<span class="number">5</span>;<span class="pagerange">773-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20398887">20398887</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2869010">2869010</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2010.03.010">10.1016/j.ajhg.2010.03.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The activating mutation R201C in GNAS promotes intestinal tumourigenesis in Apc(Min/+) mice through activation of Wnt and ERK1/2 MAPK pathways.</h4>
<p class="authors"><span class="author">Wilson CH</span>, <span class="author">McIntyre RE</span>, <span class="author">Arends MJ</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, Wellcome Trust, Sanger Institute, Hinxton, UK.</p><p class="abstract">Somatically acquired, activating mutations of GNAS, the gene encoding the stimulatory G-protein Gsalpha subunit, have been identified in kidney, thyroid, pituitary, leydig cell, adrenocortical and, more recently, in colorectal tumours, suggesting that mutations such as R201C may be oncogenic in these tissues. To study the role of GNAS in intestinal tumourigenesis, we placed GNAS R201C under the control of the A33-antigen promoter (Gpa33), which is almost exclusively expressed in the intestines. The GNAS R201C mutation has been shown to result in the constitutive activation of Gsalpha and adenylate cyclase and to lead to the autonomous synthesis of cyclic adenosine monophosphate (cAMP). Gpa33(tm1(GnasR201C)Wtsi/+) mice showed significantly elevated cAMP levels and a compensatory upregulation of cAMP-specific phosphodiesterases in the intestinal epithelium. GNAS R201C alone was not sufficient to induce tumourigenesis by 12 months, but there was a significant increase in adenoma formation when Gpa33(tm1(GnasR201C)Wtsi/+) mice were bred onto an Apc(Min/+) background. GNAS R201C expression was associated with elevated expression of Wnt and extracellular signal-regulated kinase 1/2 mitogen-activated protein kinase (ERK1/2 MAPK) pathway target genes, increased phosphorylation of ERK1/2 MAPK and increased immunostaining for the proliferation marker Ki67. Furthermore, the effects of GNAS R201C on the Wnt pathway were additive to the inactivation of Apc. Our data strongly suggest that activating mutations of GNAS cooperate with inactivation of APC and are likely to contribute to colorectal tumourigenesis.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6997</span>, <span class="grant">A8784</span>; Wellcome Trust: <span class="grant">082356</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2010</span>;<span class="volume">29</span>;<span class="number">32</span>;<span class="pagerange">4567-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20531296">20531296</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2923080">2923080</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fonc.2010.202">10.1038/onc.2010.202</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced structural variant and breakpoint detection using SVMerge by integration of multiple detection methods and local assembly.</h4>
<p class="authors"><span class="author">Wong K</span>, <span class="author">Keane TM</span>, <span class="author">Stalker J</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6B%77%31%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#107;&#119;&#49;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present a pipeline, SVMerge, to detect structural variants by integrating calls from several existing structural variant callers, which are then validated and the breakpoints refined using local de novo assembly. SVMerge is modular and extensible, allowing new callers to be incorporated as they become available. We applied SVMerge to the analysis of a HapMap trio, demonstrating enhanced structural variant detection, breakpoint refinement, and a lower false discovery rate. SVMerge can be downloaded from http://svmerge.sourceforge.net.</p>
<p class="grants">Funded by: Cancer Research UK; Medical Research Council</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">12</span>;<span class="pagerange">R128</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/21194472" rel="external">21194472</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3046488" rel="external">3046488</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-12-r128" rel="external">10.1186/gb-2010-11-12-r128</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comment on &quot;CRTAM confers late-stage activation of CD8+ T cells to regulate retention within lymph node&quot;.</h4>
<p class="authors"><span class="author">Wong SH</span>, <span class="author">Vannberg FO</span>, <span class="author">Spencer AJ</span>, <span class="author">van der Weyden L</span>, <span class="author">Hill AV</span> and <span class="author">Wyllie DH</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2010</span>;<span class="volume">184</span>;<span class="number">8</span>;<span class="pagerange">4052-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20368284" rel="external">20368284</a>; DOI: <a href="http://dx.doi.org/10.4049%2Fjimmunol.1090017" rel="external">10.4049/jimmunol.1090017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using next-generation sequencing for high resolution multiplex analysis of copy number variation from nanogram quantities of DNA from formalin-fixed paraffin-embedded specimens.</h4>
<p class="authors"><span class="author">Wood HM</span>, <span class="author">Belvedere O</span>, <span class="author">Conway C</span>, <span class="author">Daly C</span>, <span class="author">Chalkley R</span>, <span class="author">Bickerdike M</span>, <span class="author">McKinley C</span>, <span class="author">Egan P</span>, <span class="author">Ross L</span>, <span class="author">Hayward B</span>, <span class="author">Morgan J</span>, <span class="author">Davidson L</span>, <span class="author">MacLennan K</span>, <span class="author">Ong TK</span>, <span class="author">Papagiannopoulos K</span>, <span class="author">Cook I</span>, <span class="author">Adams DJ</span>, <span class="author">Taylor GR</span> and <span class="author">Rabbitts P</span></p>
<p class="affiliation">Leeds Institute of Molecular Medicine, St James&#39;s University Hospital, Leeds, UK. <a href="mailto:%68.%6D.%77%6F%6F%64%40%6C%65%65%64%73.%61%63.%75%6B">&#104;&#46;&#109;&#46;&#119;&#111;&#111;&#100;&#64;&#108;&#101;&#101;&#100;&#115;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The use of next-generation sequencing technologies to produce genomic copy number data has recently been described. Most approaches, however, reply on optimal starting DNA, and are therefore unsuitable for the analysis of formalin-fixed paraffin-embedded (FFPE) samples, which largely precludes the analysis of many tumour series. We have sought to challenge the limits of this technique with regards to quality and quantity of starting material and the depth of sequencing required. We confirm that the technique can be used to interrogate DNA from cell lines, fresh frozen material and FFPE samples to assess copy number variation. We show that as little as 5 ng of DNA is needed to generate a copy number karyogram, and follow this up with data from a series of FFPE biopsies and surgical samples. We have used various levels of sample multiplexing to demonstrate the adjustable resolution of the methodology, depending on the number of samples and available resources. We also demonstrate reproducibility by use of replicate samples and comparison with microarray-based comparative genomic hybridization (aCGH) and digital PCR. This technique can be valuable in both the analysis of routine diagnostic samples and in examining large repositories of fixed archival material.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">14</span>;<span class="pagerange">e151</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20525786">20525786</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2919738">2919738</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkq510">10.1093/nar/gkq510</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Plasmodium falciparum ATP6 not under selection during introduction of artemisinin combination therapy in Peru.</h4>
<p class="authors"><span class="author">Woodrow CJ</span> and <span class="author">Bustamante LY</span></p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2010</span>;<span class="volume">54</span>;<span class="number">5</span>;<span class="pagerange">2280; author reply 2280-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20404129">20404129</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2863640">2863640</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.01341-09">10.1128/AAC.01341-09</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput identification of transient extracellular protein interactions.</h4>
<p class="authors"><span class="author">Wright GJ</span>, <span class="author">Martin S</span>, <span class="author">Bushell KM</span> and <span class="author">S&ouml;llner C</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK. <a href="mailto:%67%77%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#119;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Protein interactions are highly diverse in their biochemical nature, varying in affinity and are often dependent on the surrounding biochemical environment. Given this heterogeneity, it seems unlikely that any one method, and particularly those capable of screening for many protein interactions in parallel, will be able to detect all functionally relevant interactions that occur within a living cell. One major class of interactions that are not detected by current popular high-throughput methods are those that occur in the extracellular environment, especially those made by membrane-embedded receptor proteins. In the present article, we discuss some of our recent research in the development of a scalable assay to identify this class of protein interaction and some of the findings from its application in the construction of extracellular protein interaction networks.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077108/Z/05/Z</span></p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2010</span>;<span class="volume">38</span>;<span class="number">4</span>;<span class="pagerange">919-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20658977" rel="external">20658977</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0380919" rel="external">10.1042/BST0380919</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacterial pleckstrin homology domains: a prokaryotic origin for the PH domain.</h4>
<p class="authors"><span class="author">Xu Q</span>, <span class="author">Bateman A</span>, <span class="author">Finn RD</span>, <span class="author">Abdubek P</span>, <span class="author">Astakhova T</span>, <span class="author">Axelrod HL</span>, <span class="author">Bakolitsa C</span>, <span class="author">Carlton D</span>, <span class="author">Chen C</span>, <span class="author">Chiu HJ</span>, <span class="author">Chiu M</span>, <span class="author">Clayton T</span>, <span class="author">Das D</span>, <span class="author">Deller MC</span>, <span class="author">Duan L</span>, <span class="author">Ellrott K</span>, <span class="author">Ernst D</span>, <span class="author">Farr CL</span>, <span class="author">Feuerhelm J</span>, <span class="author">Grant JC</span>, <span class="author">Grzechnik A</span>, <span class="author">Han GW</span>, <span class="author">Jaroszewski L</span>, <span class="author">Jin KK</span>, <span class="author">Klock HE</span>, <span class="author">Knuth MW</span>, <span class="author">Kozbial P</span>, <span class="author">Krishna SS</span>, <span class="author">Kumar A</span>, <span class="author">Marciano D</span>, <span class="author">McMullan D</span>, <span class="author">Miller MD</span>, <span class="author">Morse AT</span>, <span class="author">Nigoghossian E</span>, <span class="author">Nopakun A</span>, <span class="author">Okach L</span>, <span class="author">Puckett C</span>, <span class="author">Reyes R</span>, <span class="author">Rife CL</span>, <span class="author">Sefcovic N</span>, <span class="author">Tien HJ</span>, <span class="author">Trame CB</span>, <span class="author">van den Bedem H</span>, <span class="author">Weekes D</span>, <span class="author">Wooten T</span>, <span class="author">Hodgson KO</span>, <span class="author">Wooley J</span>, <span class="author">Elsliger MA</span>, <span class="author">Deacon AM</span>, <span class="author">Godzik A</span>, <span class="author">Lesley SA</span> and <span class="author">Wilson IA</span></p>
<p class="affiliation">Joint Center for Structural Genomics, Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, Menlo Park, CA 94025, USA.</p><p class="abstract">Pleckstrin homology (PH) domains have been identified only in eukaryotic proteins to date. We have determined crystal structures for three members of an uncharacterized protein family (Pfam PF08000), which provide compelling evidence for the existence of PH-like domains in bacteria (PHb). The first two structures contain a single PHb domain that forms a dome-shaped, oligomeric ring with C(5) symmetry. The third structure has an additional helical hairpin attached at the C-terminus and forms a similar but much larger ring with C(12) symmetry. Thus, both molecular assemblies exhibit rare, higher-order, cyclic symmetry but preserve a similar arrangement of their PHb domains, which gives rise to a conserved hydrophilic surface at the intersection of the beta-strands of adjacent protomers that likely mediates protein-protein interactions. As a result of these structures, additional families of PHb domains were identified, suggesting that PH domains are much more widespread than originally anticipated. Thus, rather than being a eukaryotic innovation, the PH domain superfamily appears to have existed before prokaryotes and eukaryotes diverged.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">U54 GM074898</span>; Wellcome Trust: <span class="grant">087656</span>, <span class="grant">WT077044/Z/05/Z</span></p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">396</span>;<span class="number">1</span>;<span class="pagerange">31-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19913036">19913036</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2817789">2817789</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jmb.2009.11.006">10.1016/j.jmb.2009.11.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Commercially available outbred mice for genome-wide association studies.</h4>
<p class="authors"><span class="author">Yalcin B</span>, <span class="author">Nicod J</span>, <span class="author">Bhomra A</span>, <span class="author">Davidson S</span>, <span class="author">Cleak J</span>, <span class="author">Farinelli L</span>, <span class="author">&Oslash;ster&aring;s M</span>, <span class="author">Whitley A</span>, <span class="author">Yuan W</span>, <span class="author">Gan X</span>, <span class="author">Goodson M</span>, <span class="author">Klenerman P</span>, <span class="author">Satpathy A</span>, <span class="author">Mathis D</span>, <span class="author">Benoist C</span>, <span class="author">Adams DJ</span>, <span class="author">Mott R</span> and <span class="author">Flint J</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">Genome-wide association studies using commercially available outbred mice can detect genes involved in phenotypes of biomedical interest. Useful populations need high-frequency alleles to ensure high power to detect quantitative trait loci (QTLs), low linkage disequilibrium between markers to obtain accurate mapping resolution, and an absence of population structure to prevent false positive associations. We surveyed 66 colonies for inbreeding, genetic diversity, and linkage disequilibrium, and we demonstrate that some have haplotype blocks of less than 100 Kb, enabling gene-level mapping resolution. The same alleles contribute to variation in different colonies, so that when mapping progress stalls in one, another can be used in its stead. Colonies are genetically diverse: 45% of the total genetic variation is attributable to differences between colonies. However, quantitative differences in allele frequencies, rather than the existence of private alleles, are responsible for these population differences. The colonies derive from a limited pool of ancestral haplotypes resembling those found in inbred strains: over 95% of sequence variants segregating in outbred populations are found in inbred strains. Consequently it is possible to impute the sequence of any mouse from a dense SNP map combined with inbred strain sequence data, which opens up the possibility of cataloguing and testing all variants for association, a situation that has so far eluded studies in completely outbred populations. We demonstrate the colonies&#39; potential by identifying a deletion in the promoter of H2-Ea as the molecular change that strongly contributes to setting the ratio of CD4+ and CD8+ lymphocytes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">6</span>;<span class="number">9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20838427" rel="external">20838427</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2932682" rel="external">2932682</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1001085" rel="external">10.1371/journal.pgen.1001085</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new Dictyostelium prestalk cell sub-type.</h4>
<p class="authors"><span class="author">Yamada Y</span>, <span class="author">Kay RR</span>, <span class="author">Bloomfield G</span>, <span class="author">Ross S</span>, <span class="author">Ivens A</span> and <span class="author">Williams JG</span></p>
<p class="affiliation">School of Life Sciences, University of Dundee, Dow St., Dundee DD3 5EH, UK.</p><p class="abstract">The mature fruiting body of Dictyostelium consists of stalk and spore cells but its construction, and the migration of the preceding slug stage, requires a number of specialized sub-types of prestalk cell whose nature and function are not well understood. The prototypic prestalk-specific gene, ecmA, is inducible by the polyketide DIF-1 in a monolayer assay and requires the DimB and MybE transcription factors for full inducibility. We perform genome-wide microarray analyses, on parental, mybE- and dimB- cells, and identify many additional genes that depend on MybE and DimB for their DIF-1 inducibility. Surprisingly, an even larger number of genes are only DIF inducible in mybE- cells, some genes are only inducible in DimB- cells and some are inducible when either transcription factor is absent. Thus in assay conditions where MybE and DimB function as inducers of ecmA these genes fall under negative control by the same two transcription factors. We have studied in detail rtaA, one of the MybE and DimB repressed genes. One especially enigmatic group of prestalk cells is the anterior-like cells (ALCs), which exist intermingled with prespore cells in the slug. A promoter fusion reporter gene, rtaA:gal(u), is expressed in a subset of the ALCs that is distinct from the ALC population detected by a reporter construct containing ecmA and ecmB promoter fragments. At culmination, when the ALC sort out from the prespore cells and differentiate to form three ancillary stalk cell structures: the upper cup, the lower cup and the outer basal disk, the rtaA:gal(u) expressing cells preferentially populate the upper cup region. This fact, and their virtual absence from the anterior and posterior regions of the slug, identifies them as a new prestalk sub-type: the pstU cells. PstU cell differentiation is, as expected, increased in a dimB- mutant during normal development but, surprisingly, they differentiate normally in a mutant lacking DIF. Thus genetic removal of MybE or DimB reveals an alternate DIF-1 activation pathway, for pstU differentiation, that functions under monolayer assay conditions but that is not essential during multicellular development.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">053640/Z</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">339</span>;<span class="number">2</span>;<span class="pagerange">390-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20080085" rel="external">20080085</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2845816" rel="external">2845816</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2009.12.045" rel="external">10.1016/j.ydbio.2009.12.045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Racial/ethnic differences in association of fasting glucose-associated genomic loci with fasting glucose, HOMA-B, and impaired fasting glucose in the U.S. adult population.</h4>
<p class="authors"><span class="author">Yang Q</span>, <span class="author">Liu T</span>, <span class="author">Shrader P</span>, <span class="author">Yesupriya A</span>, <span class="author">Chang MH</span>, <span class="author">Dowling NF</span>, <span class="author">Ned RM</span>, <span class="author">Dupuis J</span>, <span class="author">Florez JC</span>, <span class="author">Khoury MJ</span>, <span class="author">Meigs JB</span> and <span class="author">MAGIC Investigators</span></p>
<p class="affiliation">Office of Public Health Genomics, Centers for Disease Control and Prevention, Atlanta, Georgia, USA. <a href="mailto:%71%61%79%30%40%63%64%63.%67%6F%76">&#113;&#97;&#121;&#48;&#64;&#99;&#100;&#99;&#46;&#103;&#111;&#118;</a></p><p class="abstract"><strong>Objective</strong>: To estimate allele frequencies and the marginal and combined effects of novel fasting glucose (FG)-associated single nucleotide polymorphisms (SNPs) on FG levels and on risk of impaired FG (IFG) among non-Hispanic white, non-Hispanic black, and Mexican Americans.</p><p class="abstract">DNA samples from 3,024 adult fasting participants in the National Health and Nutrition Examination Survey (NHANES) III (1991-1994) were genotyped for 16 novel FG-associated SNPs in multiple genes. We determined the allele frequencies and influence of these SNPs alone and in a weighted genetic risk score on FG, homeostasis model assessment of &beta;-cell function (HOMA-B), and IFG by race/ethnicity, while adjusting for age and sex.</p><p class="abstract"><strong>Results</strong>: All allele frequencies varied significantly by race/ethnicity. A weighted genetic risk score, based on 16 SNPs, was associated with a 0.022 mmol/l (95% CI 0.009-0.035), 0.036 mmol/l (0.019-0.052), and 0.033 mmol/l (0.020-0.046) increase in FG levels per risk allele among non-Hispanic whites, non-Hispanic blacks, and Mexican Americans, respectively. Adjusted odds ratios for IFG were 1.78 for non-Hispanic whites (95% CI 1.00-3.17), 2.40 for non-Hispanic blacks (1.07-5.37), and 2.39 for Mexican Americans (1.37-4.14) when we compared the highest with the lowest quintiles of genetic risk score (P=0.365 for testing heterogeneity of effect across race/ethnicity).</p><p class="abstract"><strong>Conclusions</strong>: We conclude that allele frequencies of 16 novel FG-associated SNPs vary significantly by race/ethnicity, but the influence of these SNPs on FG levels, HOMA-B, and IFG were generally consistent across all racial/ethnic groups.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">K23 DK65978</span>, <span class="grant">K24 DK080140</span>, <span class="grant">R01 DK078616</span></p>
<p class="publication"><span class="title">Diabetes care</span>&nbsp;<span class="year">2010</span>;<span class="volume">33</span>;<span class="number">11</span>;<span class="pagerange">2370-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20805255">20805255</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2963497">2963497</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdc10-0898">10.2337/dc10-0898</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies.</h4>
<p class="authors"><span class="author">Yang TP</span>, <span class="author">Beazley C</span>, <span class="author">Montgomery SB</span>, <span class="author">Dimas AS</span>, <span class="author">Gutierrez-Arcelus M</span>, <span class="author">Stranger BE</span>, <span class="author">Deloukas P</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract"><strong>Unlabelled</strong>: Genevar (GENe Expression VARiation) is a database and Java tool designed to integrate multiple datasets, and provides analysis and visualization of associations between sequence variation and gene expression. Genevar allows researchers to investigate expression quantitative trait loci (eQTL) associations within a gene locus of interest in real time. The database and application can be installed on a standard computer in database mode and, in addition, on a server to share discoveries among affiliations or the broader community over the Internet via web services protocols.</p><p class="abstract"><strong>Availability</strong>: http://www.sanger.ac.uk/resources/software/genevar.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2010</span>;<span class="volume">26</span>;<span class="number">19</span>;<span class="pagerange">2474-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20702402">20702402</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2944204">2944204</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtq452">10.1093/bioinformatics/btq452</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phase I trial of the irreversible EGFR and HER2 kinase inhibitor BIBW 2992 in patients with advanced solid tumors.</h4>
<p class="authors"><span class="author">Yap TA</span>, <span class="author">Vidal L</span>, <span class="author">Adam J</span>, <span class="author">Stephens P</span>, <span class="author">Spicer J</span>, <span class="author">Shaw H</span>, <span class="author">Ang J</span>, <span class="author">Temple G</span>, <span class="author">Bell S</span>, <span class="author">Shahidi M</span>, <span class="author">Uttenreuther-Fischer M</span>, <span class="author">Stopfer P</span>, <span class="author">Futreal A</span>, <span class="author">Calvert H</span>, <span class="author">de Bono JS</span> and <span class="author">Plummer R</span></p>
<p class="affiliation">Drug Development Unit, the Royal Marsden National Health Service Foundation Trust, and The Institute of Cancer Research, Sutton, Surrey, United Kingdom.</p><p class="abstract"><strong>Purpose</strong>: Preclinical data have demonstrated that BIBW 2992 is a potent irreversible inhibitor of ErbB1 (EGFR/HER1) and mutated ErbB1 receptors including the T790M variant, as well as ErbB2 (HER2). A phase I study of continuous once-daily oral BIBW 2992 was conducted to determine safety, maximum-tolerated dose, pharmacokinetics (PK), food effect, and preliminary antitumor efficacy.</p><p class="abstract">Patients with advanced solid tumors were treated. PK evaluation was performed after the first dose and at steady-state.</p><p class="abstract"><strong>Results</strong>: Fifty-three patients received BIBW 2992 at 10 to 50 mg/d. BIBW 2992 was generally well-tolerated. The most common adverse effects included diarrhea, nausea, vomiting, rash, and fatigue. Dose-limiting toxicities included grade 3 rash (n = 2) and reversible dyspnea secondary to pneumonitis (n = 1). The recommended phase II dose was 50 mg/d. PK was dose proportional with a terminal elimination half-life ranging between 21.3 and 27.7 hours on day 1 and between 22.3 and 67.0 hours on day 27; BIBW 2992 exposure decreased after food intake. Three patients with non-small-cell lung carcinoma (NSCLC; two with in-frame exon 19 mutation deletions) experienced confirmed partial responses (PR) sustained for 24, 18, and 34 months, respectively. Two other patients (esophageal carcinoma and NSCLC) had nonconfirmed PRs. A patient with a PR at 10 mg/d progressed and developed symptomatic brain metastases, which subsequently regressed with an increased dose of 40 mg/d of BIBW 2992. A further seven patients had disease stabilization lasting &gt; or = 6 months.</p><p class="abstract"><strong>Conclusion</strong>: Continuous, daily, oral BIBW 2992 is safe and has durable antitumor activity. It is currently being evaluated in phase III trials.</p>
<p class="grants">Funded by: Cancer Research UK</p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2010</span>;<span class="volume">28</span>;<span class="number">25</span>;<span class="pagerange">3965-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20679611" rel="external">20679611</a>; DOI: <a href="http://dx.doi.org/10.1200%2FJCO.2009.26.7278" rel="external">10.1200/JCO.2009.26.7278</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of intracranial aneurysm identifies three new risk loci.</h4>
<p class="authors"><span class="author">Yasuno K</span>, <span class="author">Bilguvar K</span>, <span class="author">Bijlenga P</span>, <span class="author">Low SK</span>, <span class="author">Krischek B</span>, <span class="author">Auburger G</span>, <span class="author">Simon M</span>, <span class="author">Krex D</span>, <span class="author">Arlier Z</span>, <span class="author">Nayak N</span>, <span class="author">Ruigrok YM</span>, <span class="author">Niemel&auml; M</span>, <span class="author">Tajima A</span>, <span class="author">von und zu Fraunberg M</span>, <span class="author">D&oacute;czi T</span>, <span class="author">Wirjatijasa F</span>, <span class="author">Hata A</span>, <span class="author">Blasco J</span>, <span class="author">Oszvald A</span>, <span class="author">Kasuya H</span>, <span class="author">Zilani G</span>, <span class="author">Schoch B</span>, <span class="author">Singh P</span>, <span class="author">St&uuml;er C</span>, <span class="author">Risselada R</span>, <span class="author">Beck J</span>, <span class="author">Sola T</span>, <span class="author">Ricciardi F</span>, <span class="author">Aromaa A</span>, <span class="author">Illig T</span>, <span class="author">Schreiber S</span>, <span class="author">van Duijn CM</span>, <span class="author">van den Berg LH</span>, <span class="author">Perret C</span>, <span class="author">Proust C</span>, <span class="author">Roder C</span>, <span class="author">Ozturk AK</span>, <span class="author">Ga&aacute;l E</span>, <span class="author">Berg D</span>, <span class="author">Geisen C</span>, <span class="author">Friedrich CM</span>, <span class="author">Summers P</span>, <span class="author">Frangi AF</span>, <span class="author">State MW</span>, <span class="author">Wichmann HE</span>, <span class="author">Breteler MM</span>, <span class="author">Wijmenga C</span>, <span class="author">Mane S</span>, <span class="author">Peltonen L</span>, <span class="author">Elio V</span>, <span class="author">Sturkenboom MC</span>, <span class="author">Lawford P</span>, <span class="author">Byrne J</span>, <span class="author">Macho J</span>, <span class="author">Sandalcioglu EI</span>, <span class="author">Meyer B</span>, <span class="author">Raabe A</span>, <span class="author">Steinmetz H</span>, <span class="author">R&uuml;fenacht D</span>, <span class="author">J&auml;&auml;skel&auml;inen JE</span>, <span class="author">Hernesniemi J</span>, <span class="author">Rinkel GJ</span>, <span class="author">Zembutsu H</span>, <span class="author">Inoue I</span>, <span class="author">Palotie A</span>, <span class="author">Cambien F</span>, <span class="author">Nakamura Y</span>, <span class="author">Lifton RP</span> and <span class="author">G&uuml;nel M</span></p>
<p class="affiliation">Department of Neurosurgery, Yale University School of Medicine, New Haven, Connecticut, USA.</p><p class="abstract">Saccular intracranial aneurysms are balloon-like dilations of the intracranial arterial wall; their hemorrhage commonly results in severe neurologic impairment and death. We report a second genome-wide association study with discovery and replication cohorts from Europe and Japan comprising 5,891 cases and 14,181 controls with approximately 832,000 genotyped and imputed SNPs across discovery cohorts. We identified three new loci showing strong evidence for association with intracranial aneurysms in the combined dataset, including intervals near RBBP8 on 18q11.2 (odds ratio (OR) = 1.22, P = 1.1 x 10(-12)), STARD13-KL on 13q13.1 (OR = 1.20, P = 2.5 x 10(-9)) and a gene-rich region on 10q24.32 (OR = 1.29, P = 1.2 x 10(-9)). We also confirmed prior associations near SOX17 (8q11.23-q12.1; OR = 1.28, P = 1.3 x 10(-12)) and CDKN2A-CDKN2B (9p21.3; OR = 1.31, P = 1.5 x 10(-22)). It is noteworthy that several putative risk genes play a role in cell-cycle progression, potentially affecting the proliferation and senescence of progenitor-cell populations that are responsible for vascular formation and repair.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCRR NIH HHS: <span class="grant">RR19895</span>, <span class="grant">UL1 RR024139</span>; NINDS NIH HHS: <span class="grant">R01 NS 057756</span>, <span class="grant">R01 NS057756-03</span>, <span class="grant">U24 NS 051869</span>, <span class="grant">U24 NS051869-05</span>; Wellcome Trust: <span class="grant">089061</span>, <span class="grant">089062</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">42</span>;<span class="number">5</span>;<span class="pagerange">420-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20364137" rel="external">20364137</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2861730" rel="external">2861730</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.563" rel="external">10.1038/ng.563</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deficiencies in the region syntenic to human 21q22.3 cause cognitive deficits in mice.</h4>
<p class="authors"><span class="author">Yu T</span>, <span class="author">Clapcote SJ</span>, <span class="author">Li Z</span>, <span class="author">Liu C</span>, <span class="author">Pao A</span>, <span class="author">Bechard AR</span>, <span class="author">Carattini-Rivera S</span>, <span class="author">Matsui S</span>, <span class="author">Roder JC</span>, <span class="author">Baldini A</span>, <span class="author">Mobley WC</span>, <span class="author">Bradley A</span> and <span class="author">Yu YE</span></p>
<p class="affiliation">Genetics Program and Department of Cancer Genetics, Roswell Park Cancer Institute, Buffalo, NY 14263, USA.</p><p class="abstract">Copy-number variation in the human genome can be disease-causing or phenotypically neutral. This type of genetic rearrangement associated with human chromosome 21 (Hsa21) underlies partial Monosomy 21 and Trisomy 21. Mental retardation is a major clinical manifestation of partial Monosomy 21. To model this human chromosomal deletion disorder, we have generated novel mouse mutants carrying heterozygous deletions of the 2.3- and 1.1-Mb segments on mouse chromosome 10 (Mmu10) and Mmu17, respectively, which are orthologous to the regions on human 21q22.3, using Cre/loxP-mediated chromosome engineering. Alterations of the transcriptional levels of genes within the deleted intervals reflect gene-dosage effects in the mutant mice. The analysis of cognitive behaviors shows that the mutant mice carrying the deletion on either Mmu10 or Mmu17 are impaired in learning and memory. Therefore, these mutants represent mouse models for Monosomy 21-associated mental retardation, which can serve as a powerful tool to study the molecular mechanism underlying the clinical phenotype and should facilitate efforts to identify the haploinsufficient causative genes.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R0 HL 091519</span>, <span class="grant">R01 HL091519-01A1</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2010</span>;<span class="volume">21</span>;<span class="number">5-6</span>;<span class="pagerange">258-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20512340" rel="external">20512340</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3023176" rel="external">3023176</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-010-9262-x" rel="external">10.1007/s00335-010-9262-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A mouse model of Down syndrome trisomic for all human chromosome 21 syntenic regions.</h4>
<p class="authors"><span class="author">Yu T</span>, <span class="author">Li Z</span>, <span class="author">Jia Z</span>, <span class="author">Clapcote SJ</span>, <span class="author">Liu C</span>, <span class="author">Li S</span>, <span class="author">Asrar S</span>, <span class="author">Pao A</span>, <span class="author">Chen R</span>, <span class="author">Fan N</span>, <span class="author">Carattini-Rivera S</span>, <span class="author">Bechard AR</span>, <span class="author">Spring S</span>, <span class="author">Henkelman RM</span>, <span class="author">Stoica G</span>, <span class="author">Matsui S</span>, <span class="author">Nowak NJ</span>, <span class="author">Roder JC</span>, <span class="author">Chen C</span>, <span class="author">Bradley A</span> and <span class="author">Yu YE</span></p>
<p class="affiliation">Genetics Program and Department of Cancer Genetics, Roswell Park Cancer Institute, Buffalo, NY 14263, USA.</p><p class="abstract">Down syndrome (DS) is caused by the presence of an extra copy of human chromosome 21 (Hsa21) and is the most common genetic cause for developmental cognitive disability. The regions on Hsa21 are syntenically conserved with three regions located on mouse chromosome 10 (Mmu10), Mmu16 and Mmu17. In this report, we describe a new mouse model for DS that carries duplications spanning the entire Hsa21 syntenic regions on all three mouse chromosomes. This mouse mutant exhibits DS-related neurological defects, including impaired cognitive behaviors, reduced hippocampal long-term potentiation and hydrocephalus. These results suggest that when all the mouse orthologs of the Hsa21 genes are triplicated, an abnormal cognitively relevant phenotype is the final outcome of the elevated expressions of these orthologs as well as all the possible functional interactions among themselves and/or with other mouse genes. Because of its desirable genotype and phenotype, this mutant may have the potential to serve as one of the reference models for further understanding the developmental cognitive disability associated with DS and may also be used for developing novel therapeutic interventions for this clinical manifestation of the disorder.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01HL091519</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">14</span>;<span class="pagerange">2780-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20442137" rel="external">20442137</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2893810" rel="external">2893810</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddq179" rel="external">10.1093/hmg/ddq179</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic heritage and native identity of the Seaconke Wampanoag tribe of Massachusetts.</h4>
<p class="authors"><span class="author">Zhadanov SI</span>, <span class="author">Dulik MC</span>, <span class="author">Markley M</span>, <span class="author">Jennings GW</span>, <span class="author">Gaieski JB</span>, <span class="author">Elias G</span>, <span class="author">Schurr TG</span> and <span class="author">Genographic Project Consortium</span></p>
<p class="affiliation">Department of Anthropology, University of Pennsylvania, Philadelphia, PA 19104-6398, USA.</p><p class="abstract">The name &quot;Wampanoag&quot; means &quot;Eastern People&quot; or &quot;People of the First Light&quot; in the local dialect of the Algonquian language. Once extensively populating the coastal lands and neighboring islands of the eastern United States, the Wampanoag people now consist of two federally recognized tribes, the Aquinnah and Mashpee, the state-recognized Seaconke Wampanoag tribe, and a number of bands and clans in present-day southern Massachusetts. Because of repeated epidemics and conflicts with English colonists, including King Philip&#39;s War of 1675-76, and subsequent colonial laws forbidding tribal identification, the Wampanoag population was largely decimated, decreasing in size from as many as 12,000 individuals in the 16th century to less than 400, as recorded in 1677. To investigate the influence of the historical past on its biological ancestry and native cultural identity, we analyzed genetic variation in the Seaconke Wampanoag tribe. Our results indicate that the majority of their mtDNA haplotypes belongs to West Eurasian and African lineages, thus reflecting the extent of their contacts and interactions with people of European and African descent. On the paternal side, Y-chromosome analysis identified a range of Native American, West Eurasian, and African haplogroups in the population, and also surprisingly revealed the presence of a paternal lineage that appears at its highest frequencies in New Guinea and Melanesia. Comparison of the genetic data with genealogical and historical information allows us to reconstruct the tribal history of the Seaconke Wampanoag back to at least the early 18th century.</p>
<p class="publication"><span class="title">American journal of physical anthropology</span>&nbsp;<span class="year">2010</span>;<span class="volume">142</span>;<span class="number">4</span>;<span class="pagerange">579-89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20229500">20229500</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajpa.21281">10.1002/ajpa.21281</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Merlin/NF2 tumor suppressor functions through the YAP oncoprotein to regulate tissue homeostasis in mammals.</h4>
<p class="authors"><span class="author">Zhang N</span>, <span class="author">Bai H</span>, <span class="author">David KK</span>, <span class="author">Dong J</span>, <span class="author">Zheng Y</span>, <span class="author">Cai J</span>, <span class="author">Giovannini M</span>, <span class="author">Liu P</span>, <span class="author">Anders RA</span> and <span class="author">Pan D</span></p>
<p class="affiliation">Department of Molecular Biology and Genetics, Howard Hughes Medical Institute, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA.</p><p class="abstract">The conserved Hippo signaling pathway regulates organ size in Drosophila and mammals. While a core kinase cascade leading from the protein kinase Hippo (Hpo) (Mst1 and Mst2 in mammals) to the transcription coactivator Yorkie (Yki) (YAP in mammals) has been established, upstream regulators of the Hippo kinase cascade are less well defined, especially in mammals. Using conditional knockout mice, we demonstrate that the Merlin/NF2 tumor suppressor and the YAP oncoprotein function antagonistically to regulate liver development. While inactivation of Yap led to loss of hepatocytes and biliary epithelial cells, inactivation of Nf2 led to hepatocellular carcinoma and bile duct hamartoma. Strikingly, the Nf2-deficient phenotypes in multiple tissues were largely suppressed by heterozygous deletion of Yap, suggesting that YAP is a major effector of Merlin/NF2 in growth regulation. Our studies link Merlin/NF2 to mammalian Hippo signaling and implicate YAP activation as a mediator of pathologies relevant to Neurofibromatosis 2.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NIDDK NIH HHS: <span class="grant">R01 DK081417</span>, <span class="grant">R01 DK081417-04</span>, <span class="grant">R01 DK081417-05</span></p>
<p class="publication"><span class="title">Developmental cell</span>&nbsp;<span class="year">2010</span>;<span class="volume">19</span>;<span class="number">1</span>;<span class="pagerange">27-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20643348" rel="external">20643348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2925178" rel="external">2925178</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.devcel.2010.06.015" rel="external">10.1016/j.devcel.2010.06.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antigen-specific clonal expansion and cytolytic effector function of CD8+ T lymphocytes depend on the transcription factor Bcl11b.</h4>
<p class="authors"><span class="author">Zhang S</span>, <span class="author">Rozell M</span>, <span class="author">Verma RK</span>, <span class="author">Albu DI</span>, <span class="author">Califano D</span>, <span class="author">VanValkenburgh J</span>, <span class="author">Merchant A</span>, <span class="author">Rangel-Moreno J</span>, <span class="author">Randall TD</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Liu P</span> and <span class="author">Avram D</span></p>
<p class="affiliation">Center for Cell Biology and Cancer Research, Albany Medical College, Albany, NY 12208, USA.</p><p class="abstract">CD8(+) T lymphocytes mediate the immune response to viruses, intracellular bacteria, protozoan parasites, and tumors. We provide evidence that the transcription factor Bcl11b/Ctip2 controls hallmark features of CD8(+) T cell immunity, specifically antigen (Ag)-dependent clonal expansion and cytolytic activity. The reduced clonal expansion in the absence of Bcl11b was caused by altered proliferation during the expansion phase, with survival remaining unaffected. Two genes with critical roles in TCR signaling were deregulated in Bcl11b-deficient CD8(+) T cells, CD8 coreceptor and Plcgamma1, both of which may contribute to the impaired responsiveness. Bcl11b was found to bind the E8I, E8IV, and E8V, but not E8II or E8III, enhancers. Thus, Bcl11b is one of the transcription factors implicated in the maintenance of optimal CD8 coreceptor expression in peripheral CD8(+) T cells through association with specific enhancers. Short-lived Klrg1(hi)CD127(lo) effector CD8(+) T cells were formed during the course of infection in the absence of Bcl11b, albeit in smaller numbers, and their Ag-specific cytolytic activity on a per-cell basis was altered, which was associated with reduced granzyme B and perforin.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">T32-HL-07194</span>; NIAID NIH HHS: <span class="grant">AI61511</span>, <span class="grant">AI67846</span>, <span class="grant">R01AI078273</span></p>
<p class="publication"><span class="title">The Journal of experimental medicine</span>&nbsp;<span class="year">2010</span>;<span class="volume">207</span>;<span class="number">8</span>;<span class="pagerange">1687-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20660613" rel="external">20660613</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2916134" rel="external">2916134</a>; DOI: <a href="http://dx.doi.org/10.1084%2Fjem.20092136" rel="external">10.1084/jem.20092136</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and analysis of unitary pseudogenes: historic and contemporary gene losses in humans and other primates.</h4>
<p class="authors"><span class="author">Zhang ZD</span>, <span class="author">Frankish A</span>, <span class="author">Hunt T</span>, <span class="author">Harrow J</span> and <span class="author">Gerstein M</span></p>
<p class="affiliation">Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT 06520, USA. <a href="mailto:%7A%64%7A%6D%67%40%67%65%72%73%74%65%69%6E%6C%61%62.%6F%72%67">&#122;&#100;&#122;&#109;&#103;&#64;&#103;&#101;&#114;&#115;&#116;&#101;&#105;&#110;&#108;&#97;&#98;&#46;&#111;&#114;&#103;</a></p><p class="abstract"><strong>Background</strong>: Unitary pseudogenes are a class of unprocessed pseudogenes without functioning counterparts in the genome. They constitute only a small fraction of annotated pseudogenes in the human genome. However, as they represent distinct functional losses over time, they shed light on the unique features of humans in primate evolution.</p><p class="abstract"><strong>Results</strong>: We have developed a pipeline to detect human unitary pseudogenes through analyzing the global inventory of orthologs between the human genome and its mammalian relatives. We focus on gene losses along the human lineage after the divergence from rodents about 75 million years ago. In total, we identify 76 unitary pseudogenes, including previously annotated ones, and many novel ones. By comparing each of these to its functioning ortholog in other mammals, we can approximately date the creation of each unitary pseudogene (that is, the gene &#39;death date&#39;) and show that for our group of 76, the functional genes appear to be disabled at a fairly uniform rate throughout primate evolution - not all at once, correlated, for instance, with the &#39;Alu burst&#39;. Furthermore, we identify 11 unitary pseudogenes that are polymorphic - that is, they have both nonfunctional and functional alleles currently segregating in the human population. Comparing them with their orthologs in other primates, we find that two of them are in fact pseudogenes in non-human primates, suggesting that they represent cases of a gene being resurrected in the human lineage.</p><p class="abstract"><strong>Conclusions</strong>: This analysis of unitary pseudogenes provides insights into the evolutionary constraints faced by different organisms and the timescales of functional gene loss in humans.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; NLM NIH HHS: <span class="grant">1K99LM009770-01</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2010</span>;<span class="volume">11</span>;<span class="number">3</span>;<span class="pagerange">R26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/20210993">20210993</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2864566">2864566</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2010-11-3-r26">10.1186/gb-2010-11-3-r26</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/tqc9hh01"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/tqc9hh01" src="/qr/tqc9hh01.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
