<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 1996 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 1996.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 1996</h1><p>Number of papers published in 1996: <strong>36</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA sequencing of the MHC class II region and the chromosome 6 sequencing effort at the Sanger Centre.</h4>
<p class="authors"><span class="author">Abdulla S</span>, <span class="author">Alderton RP</span>, <span class="author">Glynne RJ</span>, <span class="author">Gut IG</span>, <span class="author">Hosking LK</span>, <span class="author">Jackson A</span>, <span class="author">Kelly A</span>, <span class="author">Newell WR</span>, <span class="author">Radley E</span>, <span class="author">Sanseau P</span>, <span class="author">Thorpe KL</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">DNA Sequencing Laboratory, Imperial Cancer Research Fund, London, UK.</p><p class="abstract">The human Major Histocompatibility Complex (MHC) is located on the short arm of chromosome 6 (6p21.3) and spans about 4 Mb. According to different gene families the MHC is subdivided into a class I, class II and class III region and many of its gene products are associated with the immune system and the susceptibility to various diseases. To date, we have sequenced about 40% (400 kb) of the class II region between HLA-DP and HLA-DQ and a coordinated effort to sequence the entire MHC is well underway. Analysis of the sequence revealed several novel genes and provides new insights into the molecular organisation and evolution of the MHC. All our data are publicly available via the MHC database (MHCDB) which allows rapid access, retrieval and display in the context of other MHC associated data. MHCDB is online available at (http:(/)/www.hgmp.mrc.ac.uk/) and, together with all our sequences also via anonymous ftp (ftp.icnet.uk/icrf-public).</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1996</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">5-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9063628" rel="external">9063628</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Naturally occurring nucleosome positioning signals in human exons and introns.</h4>
<p class="authors"><span class="author">Baldi P</span>, <span class="author">Brunak S</span>, <span class="author">Chauvin Y</span> and <span class="author">Krogh A</span></p>
<p class="affiliation">Division of Biology, California Institute of Technology, Pasadena 91125, USA.</p><p class="abstract">We describe the structural implications of a periodic pattern found in human exons and introns by hidden Markov models. We show that exons (besides the reading frame) have a specific sequential structure in the form of a pattern with triplet consensus non-T(A/T)G, and a minimal periodicity of roughly ten nucleotides. The periodic pattern is also present in intron sequences, although the strength per nucleotide is weaker. Using two independent profile methods based on triplet bendability parameters from DNase I experiments and nucleosome positioning data, we show that the pattern in multiple alignments of internal exon and intron sequences corresponds to a periodic &quot;in phase&quot; bending potential towards the major groove of the DNA. The nucleosome positioning data show that the consensus triplets (and their complements) have a preference for locations on a bent double helix where the major groove faces inward and is compressed. The in-phase triplets are located adjacent to GCC/GGC triplets known to have the strongest bias in their positioning on the nuclesome. Analysis of mRNA sequences encoding proteins with known tertiary structure exclude the possibility that the pattern is a consequence of the previously well-known periodicity caused by the encoding of alpha-helices in proteins. Finally, we discuss the relation between the bending potential of coding and non-coding regions and its impact on the translational positioning of nucleosomes and the recognition of genes by the transcriptional machinery.</p>
<p class="grants">Funded by: NLM NIH HHS: <span class="grant">R43 LM05780</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1996</span>;<span class="volume">263</span>;<span class="number">4</span>;<span class="pagerange">503-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8918932" rel="external">8918932</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1996.0592" rel="external">10.1006/jmbi.1996.0592</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Organisation and functions of class II genes and molecules.</h4>
<p class="authors"><span class="author">Beck S</span>, <span class="author">Belich M</span>, <span class="author">Gruneberg U</span>, <span class="author">Jackson A</span>, <span class="author">Kelly A</span>, <span class="author">Sanseau P</span>, <span class="author">Sanderson F</span>, <span class="author">Trowsdale J</span> and <span class="author">Van Ham M</span></p>
<p class="affiliation">DNA Sequencing Laboratory, Imperial Cancer Research Fund, Holborn, London.</p><p class="abstract">The class II region of the human MHC contains all of the known class II genes: as well as antigen processing components and only one gene not obviously associated with the immune system, RING3. As an approach to understanding linkage disequilibrium and recombination in relation to polymorphism of the region we are cloning and sequencing the class II region. To date, the sequence of the DP-DQ region has almost been completed (see Report by S. Beck). Several sets of genes implicated in the immune system, especially in antigen processing and presentation, are clustered together in the MHC: class I (HLA-A, B, C etc) class II (DR, DQ, DP, DN, DO, DM) LMP2 and 7, TAP1 and 2, TNF, C2, C4, Bf, Hsp70. This situation has provoked speculation that the MHC behaves as a gene cluster in which allelic products of polymorphic genes are maintained on a haplotype so as to co-ordinate T cell repertoire development and deployment. The high levels of linkage disequilibrium across the region are consistent with this idea. Functions of the genes in the MHC are being investigated as a step towards gaining insight into antigen processing and presentation as well as understanding MHC-disease associations. We are concentrating on the functions of the class II-related genes, DM and DN/DO as well as the TAP/LMP cluster.</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1996</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">21-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9063631" rel="external">9063631</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PairWise and SearchWise: finding the optimal alignment in a simultaneous comparison of a protein profile against all DNA translation frames.</h4>
<p class="authors"><span class="author">Birney E</span>, <span class="author">Thompson JD</span> and <span class="author">Gibson TJ</span></p>
<p class="affiliation">European Molecular Biology Laboratory, Heidelberg, Germany.</p><p class="abstract">DNA translation frames can be disrupted for several reasons, including: (i) errors in sequence determination; (ii) RNA processing, such as intron removal and guide RNA editing; (iii) less commonly, polymerase frameshifting during transcription or ribosomal frameshifting during translation. Frameshifts frequently confound computational activities involving homologous sequences, such as database searches and inferences on structure, function or phylogeny made from multiple alignments. A dynamic alignment algorithm is reported here which compares a protein profile (a residue scoring matrix for one or more aligned sequences) against the three translation frames of a DNA strand, allowing frameshifting. The algorithm has been incorporated into a new package, WiseTools, for comparison of biological sequences. A protein profile can be compared against either a DNA sequence or a protein sequence. The program PairWise may be used interactively for alignment of any two sequence inputs. SearchWise can perform combinations of searches through DNA or protein databases by a protein profile or DNA sequence. Routine application of the programs has revealed a set of database entries with frameshifts caused by errors in sequence determination.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1996</span>;<span class="volume">24</span>;<span class="number">14</span>;<span class="pagerange">2730-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8759004" rel="external">8759004</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC145991" rel="external">145991</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Caenorhabditis elegans genome project. C. elegans Genome Consortium.</h4>
<p class="authors"><span class="author">Coulson A</span></p>
<p class="affiliation">Sanger Centre, Cambridge, U.K.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">1996</span>;<span class="volume">24</span>;<span class="number">1</span>;<span class="pagerange">289-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8674689" rel="external">8674689</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular cloning of tissue-specific transcripts of a transketolase-related gene: implications for the evolution of new vertebrate genes.</h4>
<p class="authors"><span class="author">Coy JF</span>, <span class="author">D&uuml;bel S</span>, <span class="author">Kioschis P</span>, <span class="author">Thomas K</span>, <span class="author">Micklem G</span>, <span class="author">Delius H</span> and <span class="author">Poustka A</span></p>
<p class="affiliation">Deutsches Krebsforschungszentrum, Heidelberg, Germany.</p><p class="abstract">As part of a systematic search for differentially expressed genes, we have isolated a novel transketolase-related gene (TKR) (HGMW-approved symbol TKT), located between the green color vision pigment gene (GCP) and the ABP-280 filamin gene (FLN1) in Xq28. Transcripts encoding tissue-specific protein isoforms could be isolated. Comparison with known transketolases (TK) demonstrated a TKR-specific deletion mutating one thiamine binding site. Genomic sequencing of the TKR gene revealed the presence of a pseudoexon as well as the acquisition of a tissue-specific spliced exon compared to TK. Since it has been postulated that the vertebrate genome arose by two cycles of tetraploidization from a cephalochordate genome, this could represent an example of the modulation of the function of a preexisting transketolase gene by gene duplication. Thiamine defiency is closely involved with two neurological disorders, Beriberi and Wernicke-Korsakoff syndromes, and in both of these conditions TK with altered activity are found. We discuss the possible involvement of TKR in explaining the observed variant transketolase forms.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1996</span>;<span class="volume">32</span>;<span class="number">3</span>;<span class="pagerange">309-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8838793" rel="external">8838793</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1996.0124" rel="external">10.1006/geno.1996.0124</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tissue distribution of adenosine receptor mRNAs in the rat.</h4>
<p class="authors"><span class="author">Dixon AK</span>, <span class="author">Gubitz AK</span>, <span class="author">Sirinathsinghji DJ</span>, <span class="author">Richardson PJ</span> and <span class="author">Freeman TC</span></p>
<p class="affiliation">Department of Pharmacology, University of Cambridge.</p><p class="abstract">1. A degree of ambiguity and uncertainty exists concerning the distribution of mRNAs encoding the four cloned adenosine receptors. In order to consolidate and extent current understanding in this area, the expression of the adenosine receptors has been examined in the rat by use of in situ hybridisation and the reverse transcription-polymerase chain reaction (RT-PCR). 2. In accordance with earlier studies, in situ hybridisation revealed that the adenosine A1 receptor was widely expressed in the brain, whereas A2A receptor mRNA was restricted to the striatum, nucleus accumbens and olfactory tubercle. In addition, A1 receptor mRNA was detected in large striatal cholinergic interneurones, 26% of these neurones were also found to express the A2A receptor gene. Central levels of mRNAs encoding adenosine A2B and A3 receptors were, however, below the detection limits of in situ hybridisation. 3. The more sensitive technique of RT-PCR was then employed to investigate the distribution of adenosine receptor mRNAs in the central nervous system (CNS) and a wide range of peripheral tissues. As a result, many novel sites of adenosine receptor gene expression were identified. A1 receptor expression has now been found in the heart, aorta, liver, kidney, eye and bladder. These observations are largely consistent with previous functional data. A2A receptor mRNA was detected in all brain regions tested, demonstrating that expression of this receptor is not restricted to the basal ganglia. In the periphery A2A receptor mRNA was also found to be more widely distributed than generally recognised. The ubiquitous distribution of the A2B receptor is shown for the first time, A2B mRNA was detected at various levels in all rat tissues studied. Expression of the gene encoding the adenosine A3 receptor was also found to be widespread in the rat, message detected throughout the CNS and in many peripheral tissues. This pattern of expression is similar to that observed in man and sheep, which had previously been perceived to possess distinct patterns of A3 receptor gene expression in comparison to the rat. 4. In summary, this work has comprehensively studied the expression of all the cloned adenosine receptors in the rat, and in so doing, resolves some of the uncertainty over where these receptors might act to control physiological processes mediated by adenosine.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">British journal of pharmacology</span>&nbsp;<span class="year">1996</span>;<span class="volume">118</span>;<span class="number">6</span>;<span class="pagerange">1461-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8832073" rel="external">8832073</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1909676" rel="external">1909676</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Refined mapping and YAC contig construction of the X-linked cleft palate and ankyloglossia locus (CPX) including the proximal X-Y homology breakpoint within Xq21.3.</h4>
<p class="authors"><span class="author">Forbes SA</span>, <span class="author">Brennan L</span>, <span class="author">Richardson M</span>, <span class="author">Coffey A</span>, <span class="author">Cole CG</span>, <span class="author">Gregory SG</span>, <span class="author">Bentley DR</span>, <span class="author">Mumm S</span>, <span class="author">Moore GE</span> and <span class="author">Stanier P</span></p>
<p class="affiliation">Institute of Obstetrics and Gynaecology, Queen Charlotte&#39;s Hospital, London, United Kingdom.</p><p class="abstract">The gene for X-linked cleft palate (CPX) has previously been mapped in an Icelandic kindred between the unordered proximal markers DXS1002/DXS349/DXS95 and the distal marker DXYS1X, which maps to the proximal end of the X-Y homology region in Xq21.3. Using six sequence-tagged sites (STSs) within the region, a total of 91 yeast artificial chromosome (YAC) clones were isolated and overlapped in a single contig that spans approximately 3.1 Mb between DXS1002 and DXYS1X. The order of microsatellite and STS markers in this was established as DXS1002-DXS1168-DSX349-DXS95-DXS364-DXS 1196-DXS262-DXS110-DXS1066-(DXS1169, DXS1222)-DXS472-DXS1217-DXYS1X. A long-range restriction map of this region was created using eight nonchimeric, overlapping YAC clones. Analysis of newly positioned polymorphic markers in recombinant individuals from the Icelandic family has enabled us to identify DXS1196 and DXS1217 as the flanking markers for CPX. The maximum physical distance containing the CPX gene has been estimated to be 2.0 Mb, which is spanned by a minimum set of five nonchimeric YAC clones. In addition, YAC end clone and STS analyses have pinpointed the location of the proximal boundary of the X-Y homology region within the map.</p>
<p class="grants">Funded by: NIAMS NIH HHS: <span class="grant">5 T32 AR07033-21</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1996</span>;<span class="volume">31</span>;<span class="number">1</span>;<span class="pagerange">36-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8808277" rel="external">8808277</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1996.0006" rel="external">10.1006/geno.1996.0006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Localization of the human kinesin light chain gene (KNS2) to chromosome 14q32.3 by fluorescence in situ hybridization.</h4>
<p class="authors"><span class="author">Goedert M</span>, <span class="author">Marsh S</span> and <span class="author">Carter N</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Cambridge CB2 2QH, United Kingdom.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1996</span>;<span class="volume">32</span>;<span class="number">1</span>;<span class="pagerange">173-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8786116" rel="external">8786116</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1996.0102" rel="external">10.1006/geno.1996.0102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Life with 6000 genes.</h4>
<p class="authors"><span class="author">Goffeau A</span>, <span class="author">Barrell BG</span>, <span class="author">Bussey H</span>, <span class="author">Davis RW</span>, <span class="author">Dujon B</span>, <span class="author">Feldmann H</span>, <span class="author">Galibert F</span>, <span class="author">Hoheisel JD</span>, <span class="author">Jacq C</span>, <span class="author">Johnston M</span>, <span class="author">Louis EJ</span>, <span class="author">Mewes HW</span>, <span class="author">Murakami Y</span>, <span class="author">Philippsen P</span>, <span class="author">Tettelin H</span> and <span class="author">Oliver SG</span></p>
<p class="affiliation">Universit&eacute; Catholique de Louvain, Unit&eacute; de Biochimie Physiologique, Place Croix du Sud, 2/20, 1348 Louvain-la-Neuve, Belgium.</p><p class="abstract">The genome of the yeast Saccharomyces cerevisiae has been completely sequenced through a worldwide collaboration. The sequence of 12,068 kilobases defines 5885 potential protein-encoding genes, approximately 140 genes specifying ribosomal RNA, 40 genes for small nuclear RNA molecules, and 275 transfer RNA genes. In addition, the complete sequence provides information about the higher order organization of yeast&#39;s 16 chromosomes and allows some insight into their evolutionary history. The genome shows a considerable amount of apparent genetic redundancy, and one of the major problems to be tackled during the next stage of the yeast genome project is to elucidate the biological functions of all of these genes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1996</span>;<span class="volume">274</span>;<span class="number">5287</span>;<span class="pagerange">546, 563-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8849441">8849441</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The C. elegans expression pattern database: a beginning.</h4>
<p class="authors"><span class="author">Hope IA</span>, <span class="author">Albertson DG</span>, <span class="author">Martinelli SD</span>, <span class="author">Lynch AS</span>, <span class="author">Sonnhammer E</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Department of Biology, University of Leeds, UK. <a href="mailto:%69.%61.%68%6F%70%65%40%6C%65%65%64%73.%61%63.%75%6B">&#105;&#46;&#97;&#46;&#104;&#111;&#112;&#101;&#64;&#108;&#101;&#101;&#100;&#115;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">1996</span>;<span class="volume">12</span>;<span class="number">9</span>;<span class="pagerange">370-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8855668" rel="external">8855668</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hidden Markov models for sequence analysis: extension and analysis of the basic method.</h4>
<p class="authors"><span class="author">Hughey R</span> and <span class="author">Krogh A</span></p>
<p class="affiliation">University of California, Santa Cruz 95064, USA.</p><p class="abstract">Hidden Markov models (HMMs) are a highly effective means of modeling a family of unaligned sequences or a common motif within a set of unaligned sequences. The trained HMM can then be used for discrimination or multiple alignment. The basic mathematical description of an HMM and its expectation-maximization training procedure is relatively straightforward. In this paper, we review the mathematical extensions and heuristics that move the method from the theoretical to the practical. We then experimentally analyze the effectiveness of model regularization, dynamic model modification and optimization strategies. Finally it is demonstrated on the SH2 domain how a domain can be found from unaligned sequences using a special model type. The experimental work was completed with the aid of the Sequence Alignment and Modeling software suite.</p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1996</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">95-107</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8744772" rel="external">8744772</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and characterization of NF1-related loci on human chromosomes 22, 14 and 2.</h4>
<p class="authors"><span class="author">Hulsebos TJ</span>, <span class="author">Bijleveld EH</span>, <span class="author">Riegman PH</span>, <span class="author">Smink LJ</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Institute of Human Genetics, University of Amsterdam, Faculty of Medicine, The Netherlands. <a href="mailto:%54.%4A.%48%55%4C%53%45%42%4F%53%40%41%4D%43.%55%56%41.%4E%4C">&#84;&#46;&#74;&#46;&#72;&#85;&#76;&#83;&#69;&#66;&#79;&#83;&#64;&#65;&#77;&#67;&#46;&#85;&#86;&#65;&#46;&#78;&#76;</a></p><p class="abstract">Neurofibromatosis type 1 (NF1) is a frequent hereditary disorder. The disease is characterized by a very high mutation rate (up to 1/10000 gametes per generation). NF1-related loci in the human genome have been implicated in the high mutation rate by hypothesizing that these carry disease-causing mutations, which can be transferred to the functional NF1 gene on chromosome arm 17q by interchromosomal gene conversion. To test this hypothesis, we want to identify and characterize the NF1-related loci in the human genome. In this study, we have localized an NF1-related locus in the most centromeric region of the long arm of chromosome 22. We demonstrate that this locus contains sequences homologous to cDNAs that include the GAP-related domain of the functional NF1 gene. However, the GAP-related domain itself is not represented in this locus. In addition, cosmids specific to this locus reveal, by in situ hybridization, NF1-related loci in the pericentromeric region of chromosome arm 14q and in chromosomal band 2q21. These cosmids will enable us to determine whether identified disease-causing mutations are present at the chromosome 22-associated NF1-related locus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">98</span>;<span class="number">1</span>;<span class="pagerange">7-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8682511" rel="external">8682511</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a second human clathrin heavy chain polypeptide gene (CLH-22) from chromosome 22q11.</h4>
<p class="authors"><span class="author">Kedra D</span>, <span class="author">Peyrard M</span>, <span class="author">Fransson I</span>, <span class="author">Collins JE</span>, <span class="author">Dunham I</span>, <span class="author">Roe BA</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Molecular Medicine, Karolinska Hospital, Stockholm, Sweden.</p><p class="abstract">We report cloning and characterization of the second human clathrin heavy chain polypeptide gene (CLH-22) localized to chromosome 22q11. Hence H. sapiens is the first species for which two clathrin heavy chain genes have been reported. We provide 5470 bp cDNA sequence covering the entire open reading frame of the CLH-22 gene. The predicted polypeptide is composed of 1640 amino acids. Its 6 kb transcript is expressed in all of 16 tested human tissues, suggesting it is a housekeeping gene. Skeletal muscle, testis and heart show significantly higher expression levels. Compared to the previously characterized human clathrin heavy chain gene localized on chromosome 17 (CLH-17), CLH-22 shows different transcript size and expression profile in human tissues. Northern analysis of CLH-22 suggests that several alternatively spliced transcripts exist. A presumably single, 171 bp long alternatively spliced exon has been characterized. Amino acid sequence comparison between CLH-22 and CLH-17 shows an overall identify and similarity of 84.7 and 91.1%, respectively. At the nucleic acid level, identity between open reading frames of both genes is 74.3%. Sequence comparison with previously cloned genes in other species suggests that counterparts of the CLH-17 gene have been cloned in B. taurus and R. norvegicus, whereas presumptive mammalian homologues of the CLH-22 gene are yet to be characterized. Our Northern and Southern blot analyses of meningiomas clearly suggest the CLH-22 gene may be involved in the tumor development and can be considered as a candidate for a tumor suppressor.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">625-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8733129" rel="external">8733129</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A bacterial artificial chromosome-based framework contig map of human chromosome 22q.</h4>
<p class="authors"><span class="author">Kim UJ</span>, <span class="author">Shizuya H</span>, <span class="author">Kang HL</span>, <span class="author">Choi SS</span>, <span class="author">Garrett CL</span>, <span class="author">Smink LJ</span>, <span class="author">Birren BW</span>, <span class="author">Korenberg JR</span>, <span class="author">Dunham I</span> and <span class="author">Simon MI</span></p>
<p class="affiliation">Division of Biology, California Institute of Technology, Pasadena, 91125, USA.</p><p class="abstract">We have constructed a physical map of human chromosome 22q using bacterial artificial chromosome (BAC) clones. The map consists of 613 chromosome 22-specific BAC clones that have been localized and assembled into contigs using 452 landmarks, 346 of which were previously ordered and mapped to specific regions of the q arm of the chromosome by means of chromosome 22-specific yeast artificial chromosome clones. The BAC-based map provides immediate access to clones that are stable and convenient for direct genome analysis. The approach to rapidly developing marker-specific BAC contigs is relatively straightforward and can be extended to generate scaffold BAC contig maps of the rest of the chromosomes. These contigs will provide substrates for sequencing the entire human genome. We discuss how to efficiently close contig gaps using the end sequences of BAC clone inserts.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1996</span>;<span class="volume">93</span>;<span class="number">13</span>;<span class="pagerange">6297-301</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8692809" rel="external">8692809</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC39016" rel="external">39016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transposon Tc1-derived, sequence-tagged sites in Caenorhabditis elegans as markers for gene mapping.</h4>
<p class="authors"><span class="author">Korswagen HC</span>, <span class="author">Durbin RM</span>, <span class="author">Smits MT</span> and <span class="author">Plasterk RH</span></p>
<p class="affiliation">Division of Molecular Biology, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</p><p class="abstract">We present an approach to map large numbers of Tc1 transposon insertions in the genome of Caenorhabditis elegans. Strains have been described that contain up to 500 polymorphic Tc1 insertions. From these we have cloned and shotgun sequenced over 2000 Tc1 flanks, resulting in an estimated set of 400 or more distinct Tc1 insertion alleles. Alignment of these sequences revealed a weak Tc1 insertion site consensus sequence that was symmetric around the invariant TA target site and reads CAYATATRTG. The Tc1 flanking sequences were compared with 40 Mbp of a C. elegans genome sequence. We found 151 insertions within the sequenced area, a density of approximately 1 Tc1 insertion in every 265 kb. As the rest of the C. elegans genome sequence is obtained, remaining Tc1 alleles will fall into place. These mapped Tc1 insertions can serve two functions: (i) insertions in or near genes can be used to isolate deletion derivatives that have that gene mutated; and (ii) they represent a dense collection of polymorphic sequence-tagged sites. We demonstrate a strategy to use these Tc1 sequence-tagged sites in fine-mapping mutations.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">R01 RR10082-02</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1996</span>;<span class="volume">93</span>;<span class="number">25</span>;<span class="pagerange">14680-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8962114" rel="external">8962114</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC26195" rel="external">26195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A radiation hybrid map spanning the entire human X chromosome integrating YACs, genes, and STS markers.</h4>
<p class="authors"><span class="author">Kumlien J</span>, <span class="author">Grigoriev A</span>, <span class="author">Roest Crollius H</span>, <span class="author">Ross M</span>, <span class="author">Goodfellow PN</span> and <span class="author">Lehrach H</span></p>
<p class="affiliation">Department of Genome Analysis, Imperial Cancer Research Fund, P.O. Box 123, 44 Lincoln&#39;s Inn Fields, London WC2A 3 PX, UK.</p><p class="abstract">We present a radiation hybrid (RH) map of human Chromosome (Chr) X, using 50 markers on 72 radiation hybrids. The markers, obtained from the consensus map, form a grid spanning the entire chromosome. To check the RH map, the marker order was determined by analysis of presence or absence of retained human DNA fragments in the RHs; the comparison with the consensus showed a similar order. Any STSs, microsatellites, genes, and clones can be positioned and ordered relative to the marker grid. This approach integrates genetic, physical, and large-scale clone mapping and is used to link YAC contigs containing data from various experimental sources.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">1996</span>;<span class="volume">7</span>;<span class="number">10</span>;<span class="pagerange">758-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8854864" rel="external">8854864</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome-specific paints from a high-resolution flow karyotype of the dog.</h4>
<p class="authors"><span class="author">Langford CF</span>, <span class="author">Fischer PE</span>, <span class="author">Binns MM</span>, <span class="author">Holmes NG</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">Using peripheral blood lymphocyte cultures and dual-laser flow cytometry, we have routinely obtained high-resolution bivariate flow karyotypes of the dog in which 32 peaks are resolved. To allow the identification of the chromosome types in each peak, chromosomes were flow sorted, amplified and labelled by polymerase chain reaction with partially degenerate primers and hybridized onto metaphase spreads of a male dog. The chromosome paints from 22 of the 32 peaks each hybridized to single homologue pairs and eight peaks each hybridized to two pairs. Paints from the remaining two peaks hybridized to only one homologue each in the male metaphase spread, thus corresponding to the sex chromosomes X and Y. All of the 38 pairs of autosomes and the two sex chromosomes of the dog could be accounted for in these painting experiments. The positions of chromosomes 1-21 were assigned to the flow karyotype (only chromosomes 1-21 have as yet been officially designated). The high-resolution flow karyotype and the chromosome paints will facilitate further standardization of the dog karyotype. The ability to sort sufficient quantities of dog chromosomes for the production of chromosome-specific DNA libraries has the potential to accelerate the physical and genetic mapping of the dog genome.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">1996</span>;<span class="volume">4</span>;<span class="number">2</span>;<span class="pagerange">115-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8785605" rel="external">8785605</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative analysis and genomic structure of the tuberous sclerosis 2 (TSC2) gene in human and pufferfish.</h4>
<p class="authors"><span class="author">Maheshwar MM</span>, <span class="author">Sandford R</span>, <span class="author">Nellist M</span>, <span class="author">Cheadle JP</span>, <span class="author">Sgotto B</span>, <span class="author">Vaudin M</span> and <span class="author">Sampson JR</span></p>
<p class="affiliation">Institute of Medical Genetics, University of Wales College of Medicine, Cardiff, UK.</p><p class="abstract">Germ-line mutations of the TSC2 tumour suppressor gene have been identified in humans with tuberous sclerosis and in the Eker rat. Tuberin, the human TSC2 gene product, has a small region of homology with rap1GAP and stimulates rap1 GTPase activity in vitro, suggesting that one of its cellular roles is to function as a GTPase activating protein (GAP). We have undertaken a comparative analysis of the TSC2 gene in human and the pufferfish, Fugu rubripes. In addition to the GAP domain, three other regions of the proteins are highly conserved (peptide sequence similarity &gt; 80%). These regions are likely to represent further functional domains. To facilitate analysis of mutations within these domains we have determined the genomic structure of the human TSC2 gene. It comprises 41 exons, including exon 31 which was absent from the originally described spliceoform of the human TSC2 transcript and was identified following exon prediction from Fugu genomic sequence. These findings support the proposal of the Fugu genome as a tool for human gene analysis.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">131-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8789450" rel="external">8789450</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical mapping of chromosome 6: a strategy for the rapid generation of sequence-ready contigs.</h4>
<p class="authors"><span class="author">Mungall AJ</span>, <span class="author">Edwards CA</span>, <span class="author">Ranby SA</span>, <span class="author">Humphray SJ</span>, <span class="author">Heathcott RW</span>, <span class="author">Clee CM</span>, <span class="author">East CL</span>, <span class="author">Holloway E</span>, <span class="author">Butler AP</span>, <span class="author">Langford CF</span>, <span class="author">Gwilliam R</span>, <span class="author">Rice KM</span>, <span class="author">Maslen GL</span>, <span class="author">Carter NP</span>, <span class="author">Ross MT</span>, <span class="author">Deloukas P</span>, <span class="author">Bentley DR</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Sanger Centre, Huxton, Cambridge, UK.</p><p class="abstract">The development of radiation hybrid (RH) mapping (Cox et al., 1990) and the availability of large numbers of STS markers, together with extensive bacterial clone resources provided a means to accelerate the process of mapping a human chromosome and preparing bacterial clone contigs ready to sequence. Our aim is to construct physical clone maps covering those regions of chromosome 6 that are not currently extensively mapped, and use these to determine the DNA sequence of the whole chromosome. We report here a strategy which initially involves establishing a high density framework map using RH mapping. The framework markers are then used for the identification of bacterial genomic clones covering the chromosome. The bacterial clones are analysed by restriction enzyme fingerprinting and STS-content analysis to identify sequence-ready contigs. Contig gap closure will also be performed by clone walking.</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1996</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">47-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9063638" rel="external">9063638</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MHCDB: database of the human MHC (release 2).</h4>
<p class="authors"><span class="author">Newell WR</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Imperial Cancer Research Fund, 44 Lincoln&#39;s Inn Fields, London WC2A 3PX, UK.</p><p class="abstract">The second release of the human major histocompatibility complex (MHC) database is now publicly available. It contains an updated physical map and considerably more genomic sequence. cDNA sequences of all current alleles are accessible as individual sequence entries. The variability of different genes is displayed graphically as static and dynamic images accessible from the database. Known disease-serotype associations have also been incorporated, together with data from the MHCPEP database of eluted peptides.</p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">45</span>;<span class="number">1</span>;<span class="pagerange">6-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8881031" rel="external">8881031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the complete DNA sequence of murine cytomegalovirus.</h4>
<p class="authors"><span class="author">Rawlinson WD</span>, <span class="author">Farrell HE</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">Laboratory of Molecular Biology, Cambridge, United Kingdom. <a href="mailto:%57%44%52%40%77%65%73%74%6D%65%64.%77%68.%73%75.%65%64%75.%61%75">&#87;&#68;&#82;&#64;&#119;&#101;&#115;&#116;&#109;&#101;&#100;&#46;&#119;&#104;&#46;&#115;&#117;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">The complete DNA sequence of the Smith strain of murine cytomegalovirus (MCMV) was determined from virion DNA by using a whole-genome shotgun approach. The genome has an overall G+C content of 58.7%, consists of 230,278 bp, and is arranged as a single unique sequence with short (31-bp) terminal direct repeats and several short internal repeats. Significant similarity to the genome of the sequenced human cytomegalovirus (HCMV) strain AD169 is evident, particularly for 78 open reading frames encoded by the central part of the genome. There is a very similar distribution of G+C content across the two genomes. Sequences toward the ends of the MCMV genome encode tandem arrays of homologous glycoproteins (gps) arranged as two gene families. The left end encodes 15 gps that represent one family, and the right end encodes a different family of 11 gps. A homolog (m144) of cellular major histocompatibility complex (MHC) class I genes is located at the end of the genome opposite the HCMV MHC class I homolog (UL18). G protein-coupled receptor (GCR) homologs (M33 and M78) occur in positions congruent with two (UL33 and UL78) of the four putative HCMV GCR homologs. Counterparts of all of the known enzyme homologs in HCMV are present in the MCMV genome, including the phosphotransferase gene (M97), whose product phosphorylates ganciclovir in HCMV-infected cells, and the assembly protein (M80).</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">1996</span>;<span class="volume">70</span>;<span class="number">12</span>;<span class="pagerange">8833-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8971012" rel="external">8971012</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC190980" rel="external">190980</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improving prediction of protein secondary structure using structured neural networks and multiple sequence alignments.</h4>
<p class="authors"><span class="author">Riis SK</span> and <span class="author">Krogh A</span></p>
<p class="affiliation">Electronics Institute, Technical University of Denmark, Lyngby, Denmark.</p><p class="abstract">The prediction of protein secondary structure by use of carefully structured neural networks and multiple sequence alignments has been investigated. Separate networks are used for predicting the three secondary structures alpha-helix, beta-strand, and coil. The networks are designed using a priori knowledge of amino acid properties with respect to the secondary structure and the characteristic periodicity in alpha-helices. Since these single-structure networks all have less than 600 adjustable weights, overfitting is avoided. To obtain a three-state prediction of alpha-helix, beta-strand, or coil, ensembles of single-structure networks are combined with another neural network. This method gives an overall prediction accuracy of 66.3% when using 7-fold cross-validation on a database of 126 nonhomologous globular proteins. Applying the method to multiple sequence alignments of homologous proteins increases the prediction accuracy significantly to 71.3% with corresponding Matthew&#39;s correlation coefficients C alpha = 0.59, C beta = 0.52, and Cc = 0.50. More than 72% of the residues in the database are predicted with an accuracy of 80%. It is shown that the network outputs can be interpreted as estimated probabilities of correct prediction, and, therefore, these numbers indicate which residues are predicted with high confidence.</p>
<p class="publication"><span class="title">Journal of computational biology : a journal of computational molecular cell biology</span>&nbsp;<span class="year">1996</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">163-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8697234" rel="external">8697234</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of DRP2, a novel human dystrophin homologue.</h4>
<p class="authors"><span class="author">Roberts RG</span>, <span class="author">Freeman TC</span>, <span class="author">Kendall E</span>, <span class="author">Vetrie DL</span>, <span class="author">Dixon AK</span>, <span class="author">Shaw-Smith C</span>, <span class="author">Bone Q</span> and <span class="author">Bobrow M</span></p>
<p class="affiliation">Division of Medical and Molecular Genetics, UMDS, London, UK.</p><p class="abstract">The currently recognised dystrophin protein family comprises the archetype, dystrophin, its close relative, utrophin or dystrophin-related protein (DRP), and a distantly related protein known as the 87K tyrosine kinase substrate. During the course of a phylogenetic study of sequences encoding the characteristic C-terminal domains of dystrophin-related proteins, we identified an unexpected novel class of vertebrate dystrophin-related sequences. We term this class dystrophin-related protein 2 (DRP2), and suggest that utrophin/DRP be renamed DRP1 to simplify future nomenclature. DRP2 is a relatively small protein, encoded in man by a 45 kb gene localized to Xq22. It is expressed principally in the brain and spinal cord, and is similar in overall structure to the Dp116 dystrophin isoform. The discovery of a novel relative of dystrophin substantially broadens the scope for study of this interesting group of proteins and their associated glycoprotein complexes.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">13</span>;<span class="number">2</span>;<span class="pagerange">223-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8640231">8640231</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0696-223">10.1038/ng0696-223</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated YAC map of the human X chromosome.</h4>
<p class="authors"><span class="author">Roest Crollius H</span>, <span class="author">Ross MT</span>, <span class="author">Grigoriev A</span>, <span class="author">Knights CJ</span>, <span class="author">Holloway E</span>, <span class="author">Misfud J</span>, <span class="author">Li K</span>, <span class="author">Playford M</span>, <span class="author">Gregory SG</span>, <span class="author">Humphray SJ</span>, <span class="author">Coffey AJ</span>, <span class="author">See CG</span>, <span class="author">Marsh S</span>, <span class="author">Vatcheva R</span>, <span class="author">Kumlien J</span>, <span class="author">Labella T</span>, <span class="author">Lam V</span>, <span class="author">Rak KH</span>, <span class="author">Todd K</span>, <span class="author">Mott R</span>, <span class="author">Graeser D</span>, <span class="author">Rappold G</span>, <span class="author">Zehetner G</span>, <span class="author">Poustka A</span>, <span class="author">Bentley DR</span>, <span class="author">Monaco AP</span> and <span class="author">Lehrach H</span></p>
<p class="affiliation">Max-Planck-Institut f&uuml;r Molekulare Genetik, Berlin, Germany. <a href="mailto:%72%6F%65%73%74%40%6D%70%69%6D%67-%62%65%72%6C%69%6E-%64%61%68%6C%65%6D.%6D%70%67.%64%65">&#114;&#111;&#101;&#115;&#116;&#64;&#109;&#112;&#105;&#109;&#103;&#45;&#98;&#101;&#114;&#108;&#105;&#110;&#45;&#100;&#97;&#104;&#108;&#101;&#109;&#46;&#109;&#112;&#103;&#46;&#100;&#101;</a></p><p class="abstract">The human X chromosome is associated with a large number of disease phenotypes, principally because of its unique mode of inheritance that tends to reveal all recessive disorders in males. With the longer term goal of identifying and characterizing most of these genes, we have adopted a chromosome-wide strategy to establish a YAC contig map. We have performed &gt; 3250 inter Alu-PCR product hybridizations to identify overlaps between YAC clones. Positional information associated with many of these YAC clones has been derived from our Reference Library Database and a variety of other public sources. We have constructed a YAC contig map of the X chromosome covering 125 Mb of DNA in 25 contigs and containing 906 YAC clones. These contigs have been verified extensively by FISH and by gel and hybridization fingerprinting techniques. This independently derived map exceeds the coverage of recently reported X chromosome maps built as part of whole-genome YAC maps.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1996</span>;<span class="volume">6</span>;<span class="number">10</span>;<span class="pagerange">943-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8908513" rel="external">8908513</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Long-range map of a 3.5-Mb region in Xp11.23-22 with a sequence-ready map from a 1.1-Mb gene-rich interval.</h4>
<p class="authors"><span class="author">Schindelhauer D</span>, <span class="author">Hellebrand H</span>, <span class="author">Grimm L</span>, <span class="author">Bader I</span>, <span class="author">Meitinger T</span>, <span class="author">Wehnert M</span>, <span class="author">Ross M</span> and <span class="author">Meindl A</span></p>
<p class="affiliation">Abteilung f&uuml;r P&auml;diatrische Genetik, Kinderpoliklinik der Universit&auml;t M&uuml;nchen, Germany.</p><p class="abstract">Most of the yeast artificial chromosomes (YACs) isolated from the Xp11.23-22 region have shown instability and chimerism and are not a reliable resource for determining physical distances. We therefore constructed a long-range pulsed-field gel electrophoresis map that encompasses approximately 3.5 Mb of genomic DNA between the loci TIMP and DXS146 including a CpG-rich region around the WASP and TFE-3 gene loci. A combined YAC-cosmid contig was constructed along the genomic map and was used for fine-mapping of 15 polymorphic microsatellites and 30 expressed sequence tags (ESTs) or sequence transcribed sites (STSs), revealing the following order: tel-(SYN-TIMP)-(DXS426-ELK1)-ZNF(CA) n-L1-DXS1367-ZNF81-ZNF21-DXS6616- (HB3-OATL1pseudogenes-DXS6950)-DXS6949-DXS694 1-DXS7464E(MG61)-GW1E(EBP)- DXS7927E(MG81)-RBM- DXS722-DXS7467E(MG21)-DXS1011E-WASP-DXS6940++ +-DXS7466E(MG44)-GF1- DXS226-DXS1126-DXS1240-HB1- DXS7469E-(DXS6665-DXS1470)-TFE3-DXS7468E-+ ++SYP-DXS1208-HB2E-DXS573-DXS1331- DXS6666-DXS1039-DXS 1426-DXS1416-DXS7647-DXS8222-DXS6850-DXS255++ +-CIC-5-DXS146-cen. A sequence-ready map was constructed for an 1100-kb gene-rich interval flanked by the markers HB3 and DXS1039, from which six novel ESTs/STSs were isolated, thus increasing the number of markers used in this interval to thirty. This precise ordering is a prerequisite for the construction of a transcription map of this region that contains numerous disease loci, including those for several forms of retinal degeneration and mental retardation. In addition, the map provides the base to delineate the corresponding syntenic region in the mouse, where the mutants scurfy and tattered are localized.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1996</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">1056-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8938429" rel="external">8938429</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome maps 7. The human transcript map. Wall chart.</h4>
<p class="authors"><span class="author">Schuler GD</span>, <span class="author">Boguski MS</span>, <span class="author">Hudson TJ</span>, <span class="author">Hui L</span>, <span class="author">Ma J</span>, <span class="author">Castle AB</span>, <span class="author">Wu X</span>, <span class="author">Silva J</span>, <span class="author">Nusbaum HC</span>, <span class="author">Birren BB</span>, <span class="author">Slonim DK</span>, <span class="author">Rozen S</span>, <span class="author">Stein LD</span>, <span class="author">Page D</span>, <span class="author">Lander ES</span>, <span class="author">Stewart EA</span>, <span class="author">Aggarwal A</span>, <span class="author">Bajorek E</span>, <span class="author">Brady S</span>, <span class="author">Chu S</span>, <span class="author">Fang N</span>, <span class="author">Hadley D</span>, <span class="author">Harris M</span>, <span class="author">Hussain S</span>, <span class="author">Hudson JR</span> <em>et al.</em></p>
<p class="affiliation">National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894, USA.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1996</span>;<span class="volume">274</span>;<span class="number">5287</span>;<span class="pagerange">547-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8928009">8928009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A gene map of the human genome.</h4>
<p class="authors"><span class="author">Schuler GD</span>, <span class="author">Boguski MS</span>, <span class="author">Stewart EA</span>, <span class="author">Stein LD</span>, <span class="author">Gyapay G</span>, <span class="author">Rice K</span>, <span class="author">White RE</span>, <span class="author">Rodriguez-Tom&eacute; P</span>, <span class="author">Aggarwal A</span>, <span class="author">Bajorek E</span>, <span class="author">Bentolila S</span>, <span class="author">Birren BB</span>, <span class="author">Butler A</span>, <span class="author">Castle AB</span>, <span class="author">Chiannilkulchai N</span>, <span class="author">Chu A</span>, <span class="author">Clee C</span>, <span class="author">Cowles S</span>, <span class="author">Day PJ</span>, <span class="author">Dibling T</span>, <span class="author">Drouot N</span>, <span class="author">Dunham I</span>, <span class="author">Duprat S</span>, <span class="author">East C</span>, <span class="author">Edwards C</span>, <span class="author">Fan JB</span>, <span class="author">Fang N</span>, <span class="author">Fizames C</span>, <span class="author">Garrett C</span>, <span class="author">Green L</span>, <span class="author">Hadley D</span>, <span class="author">Harris M</span>, <span class="author">Harrison P</span>, <span class="author">Brady S</span>, <span class="author">Hicks A</span>, <span class="author">Holloway E</span>, <span class="author">Hui L</span>, <span class="author">Hussain S</span>, <span class="author">Louis-Dit-Sully C</span>, <span class="author">Ma J</span>, <span class="author">MacGilvery A</span>, <span class="author">Mader C</span>, <span class="author">Maratukulam A</span>, <span class="author">Matise TC</span>, <span class="author">McKusick KB</span>, <span class="author">Morissette J</span>, <span class="author">Mungall A</span>, <span class="author">Muselet D</span>, <span class="author">Nusbaum HC</span>, <span class="author">Page DC</span>, <span class="author">Peck A</span>, <span class="author">Perkins S</span>, <span class="author">Piercy M</span>, <span class="author">Qin F</span>, <span class="author">Quackenbush J</span>, <span class="author">Ranby S</span>, <span class="author">Reif T</span>, <span class="author">Rozen S</span>, <span class="author">Sanders C</span>, <span class="author">She X</span>, <span class="author">Silva J</span>, <span class="author">Slonim DK</span>, <span class="author">Soderlund C</span>, <span class="author">Sun WL</span>, <span class="author">Tabar P</span>, <span class="author">Thangarajah T</span>, <span class="author">Vega-Czarny N</span>, <span class="author">Vollrath D</span>, <span class="author">Voyticky S</span>, <span class="author">Wilmer T</span>, <span class="author">Wu X</span>, <span class="author">Adams MD</span>, <span class="author">Auffray C</span>, <span class="author">Walter NA</span>, <span class="author">Brandon R</span>, <span class="author">Dehejia A</span>, <span class="author">Goodfellow PN</span>, <span class="author">Houlgatte R</span>, <span class="author">Hudson JR</span>, <span class="author">Ide SE</span>, <span class="author">Iorio KR</span>, <span class="author">Lee WY</span>, <span class="author">Seki N</span>, <span class="author">Nagase T</span>, <span class="author">Ishikawa K</span>, <span class="author">Nomura N</span>, <span class="author">Phillips C</span>, <span class="author">Polymeropoulos MH</span>, <span class="author">Sandusky M</span>, <span class="author">Schmitt K</span>, <span class="author">Berry R</span>, <span class="author">Swanson K</span>, <span class="author">Torres R</span>, <span class="author">Venter JC</span>, <span class="author">Sikela JM</span>, <span class="author">Beckmann JS</span>, <span class="author">Weissenbach J</span>, <span class="author">Myers RM</span>, <span class="author">Cox DR</span>, <span class="author">James MR</span>, <span class="author">Bentley D</span>, <span class="author">Deloukas P</span>, <span class="author">Lander ES</span> and <span class="author">Hudson TJ</span></p>
<p class="affiliation">National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, 8600 Rockville Pike, Bethesda, MD 20894, USA.</p><p class="abstract">The human genome is thought to harbor 50,000 to 100,000 genes, of which about half have been sampled to date in the form of expressed sequence tags. An international consortium was organized to develop and map gene-based sequence tagged site markers on a set of two radiation hybrid panels and a yeast artificial chromosome library. More than 16,000 human genes have been mapped relative to a framework map that contains about 1000 polymorphic genetic markers. The gene map unifies the existing genetic and physical maps with the nucleotide and protein sequence databases in a fashion that should speed the discovery of genes underlying inherited human disease. The integrated resource is available through a site on the World Wide Web at http://www.ncbi.nlm.nih.gov/SCIENCE96/.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG00098</span>, <span class="grant">HG00206</span>, <span class="grant">HG00835</span>; Wellcome Trust; ...</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1996</span>;<span class="volume">274</span>;<span class="number">5287</span>;<span class="pagerange">540-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8849440">8849440</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The t(X;1)(p11.2;q21.2) translocation in papillary renal cell carcinoma fuses a novel gene PRCC to the TFE3 transcription factor gene.</h4>
<p class="authors"><span class="author">Sidhar SK</span>, <span class="author">Clark J</span>, <span class="author">Gill S</span>, <span class="author">Hamoudi R</span>, <span class="author">Crew AJ</span>, <span class="author">Gwilliam R</span>, <span class="author">Ross M</span>, <span class="author">Linehan WM</span>, <span class="author">Birdsall S</span>, <span class="author">Shipley J</span> and <span class="author">Cooper CS</span></p>
<p class="affiliation">Molecular Carcinogenesis Section, Institute of Cancer Research, Haddow Laboratories, Belmont, UK.</p><p class="abstract">The specific chromosomal translocation t(X;1)(p11.2;q21.2) has been observed in human papillary renal cell carcinomas. In this study we demonstrated that this translocation results in the fusion of a novel gene designated PRCC at 1q21.2 to the TFE3 gene at Xp11.2. TFE3 encodes a member of the basic helix-loop-helix (bHLH) family of transcription factors originally identified by its ability to bind to microE3 elements in the immunoglobin heavy chain intronic enhancer. The translocation is predicted to result in the fusion of the N-terminal region of the PRCC protein, which includes a proline-rich domain, to the entire TFE3 protein. Notably the generation of the chimaeric PRCC-TFE3 gene appears to be accompanied by complete loss of normal TFE3 transcripts. This work establishes that the disruption of transcriptional control by chromosomal translocation is important in the development of kidney carcinoma in addition to its previously established role in the aetiology of sarcomas and leukaemias.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">5</span>;<span class="number">9</span>;<span class="pagerange">1333-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8872474" rel="external">8872474</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dirichlet mixtures: a method for improved detection of weak but significant protein sequence homology.</h4>
<p class="authors"><span class="author">Sj&ouml;lander K</span>, <span class="author">Karplus K</span>, <span class="author">Brown M</span>, <span class="author">Hughey R</span>, <span class="author">Krogh A</span>, <span class="author">Mian IS</span> and <span class="author">Haussler D</span></p>
<p class="affiliation">Baskin Center for Computer Engineering and Information Sciences, University of California at Santa Cruz 95064, USA. <a href="mailto:%6B%69%6D%6D%65%6E%40%63%73%65.%75%63%73%63.%65%64%75">&#107;&#105;&#109;&#109;&#101;&#110;&#64;&#99;&#115;&#101;&#46;&#117;&#99;&#115;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract">We present a method for condensing the information in multiple alignments of proteins into a mixture of Dirichlet densities over amino acid distributions. Dirichlet mixture densities are designed to be combined with observed amino acid frequencies to form estimates of expected amino acid probabilities at each position in a profile, hidden Markov model or other statistical model. These estimates give a statistical model greater generalization capacity, so that remotely related family members can be more reliably recognized by the model. This paper corrects the previously published formula for estimating these expected probabilities, and contains complete derivations of the Dirichlet mixture formulas, methods for optimizing the mixtures to match particular databases, and suggestions for efficient implementation.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM17129</span></p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1996</span>;<span class="volume">12</span>;<span class="number">4</span>;<span class="pagerange">327-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8902360" rel="external">8902360</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A member of the MAP kinase phosphatase gene family in mouse containing a complex trinucleotide repeat in the coding region.</h4>
<p class="authors"><span class="author">Theodosiou AM</span>, <span class="author">Rodrigues NR</span>, <span class="author">Nesbit MA</span>, <span class="author">Ambrose HJ</span>, <span class="author">Paterson H</span>, <span class="author">McLellan-Arnold E</span>, <span class="author">Boyd Y</span>, <span class="author">Leversha MA</span>, <span class="author">Owen N</span>, <span class="author">Blake DJ</span>, <span class="author">Ashworth A</span> and <span class="author">Davies KE</span></p>
<p class="affiliation">Biochemistry Department, University of Oxford, UK.</p><p class="abstract">We have identified a novel mouse gene encoding a protein that shows high homology to the dual-specificity tyrosine/threonine phosphatase family of proteins. The gene encodes a 5 kb transcript which is expressed predominantly in brain and lung and contains a translated complex trinucleotide repeat within the coding region. Using interspecific mouse backcross analysis, the gene has been localised to distal mouse chromosome 7. In human, homologous sequences are located in the syntenic region on distal chromosome 11p as well as to chromosome 10q11.2 and 10q22. The presence of a CG-rich trinucleotide repeat in the coding region provides a target for mutation which might result in loss of function or altered properties of this phosphatase.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">675-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8733137" rel="external">8733137</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Yeasties and beasties: 7 years of genome sequencing.</h4>
<p class="authors"><span class="author">Thomas K</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, UK.</p><p class="abstract">The Saccharomyces cerevisiae genome sequencing project was the first of many projects aimed at sequencing the entire genomes of model organisms. Since its initiation in 1989, there have been numerous debates about the validity of genome sequencing, especially with reference to the model organisms. Seven years on, I hope to satisfy some of the critics by demonstrating that, as a consequence of the mass of data now becoming available from such projects, and the beginning of the major collaborative effort to sequence the human genome, we are now entering an exciting and dynamic time for those involved not only in genome sequencing, but also in all areas of the biological sciences.</p>
<p class="publication"><span class="title">FEBS letters</span>&nbsp;<span class="year">1996</span>;<span class="volume">396</span>;<span class="number">1</span>;<span class="pagerange">1-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8906856" rel="external">8906856</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phylogeny and structure of the RING3 gene.</h4>
<p class="authors"><span class="author">Thorpe KL</span>, <span class="author">Abdulla S</span>, <span class="author">Kaufman J</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">DNA Sequencing Laboratory, Imperial Cancer Research Fund, PO Box 123, London WC2A 3PX, UK.</p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">1996</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">391-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8781126" rel="external">8781126</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The imprint of somatic hypermutation on the repertoire of human germline V genes.</h4>
<p class="authors"><span class="author">Tomlinson IM</span>, <span class="author">Walter G</span>, <span class="author">Jones PT</span>, <span class="author">Dear PH</span>, <span class="author">Sonnhammer EL</span> and <span class="author">Winter G</span></p>
<p class="affiliation">MRC Centre for Protein Engineering, Cambridge, UK.</p><p class="abstract">In the human immune system, antibodies with high affinities for antigen are created in two stages. A diverse primary repertoire of antibody structures is produced by the combinatorial rearrangement of germline V gene segments and antibodies are selected from this repertoire by binding to the antigen. Their affinities are then improved by somatic hypermutation and further rounds of selection. We have dissected the sequence diversity created at each stage in response to a wide range of antigens. In the primary repertoire, diversity is focused at the centre of the binding site. With somatic hypermutation, diversity spreads to regions at the periphery of the binding site that are highly conserved in the primary repertoire. We propose that evolution has favoured this complementarity as an efficient strategy for searching sequence space and that the germline V gene families evolved to exploit the diversity created by somatic hypermutation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1996</span>;<span class="volume">256</span>;<span class="number">5</span>;<span class="pagerange">813-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8601832" rel="external">8601832</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1996.0127" rel="external">10.1006/jmbi.1996.0127</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An Xp22.1-p22.2 YAC contig encompassing the disease loci for RS, KFSD, CLS, HYP and RP15: refined localization of RS.</h4>
<p class="authors"><span class="author">Van de Vosse E</span>, <span class="author">Bergen AA</span>, <span class="author">Meershoek EJ</span>, <span class="author">Oosterwijk JC</span>, <span class="author">Gregory S</span>, <span class="author">Bakker B</span>, <span class="author">Weissenbach J</span>, <span class="author">Coffey AJ</span>, <span class="author">van Ommen GJ</span> and <span class="author">Den Dunnen JT</span></p>
<p class="affiliation">MGC-Department of Human Genetics, Leiden University, UK.</p><p class="abstract">To facilitate the positional cloning of the genes involved in retinoschisis (RS), keratosis follicularis spinulosa decalvans (KFSD), Coffin-Lowry syndrome (CLS), X-linked hypophosphatemic rickets (XLH, locus name HYP) and X-linked dominant cone-rod degeneration (locus name RP15), we have extended the molecular map of the Xp22 region. Screening of several YAC libraries allowed us to identify 156 YACs, 52 of which localize between markers DXS414 (P90) and DXS451 (kQST80H1). Analysis of their marker content facilitated the construction of a YAC contig from the region spanning (in this order): DXS414 - DXS987 - DXS207 - DXS1053 - DXS197 - DXS 43 - DXS1195 - DXS418 - DXS999 - PDHA1 - DXS7161 - DXS443 - DXS 7592 - DXS1229 - DXS365 - DXS7101 - DXS7593 - DXS1052 - DXS274 - DXS989 - DXS451. The region between DXS414 and DXS451 covers about 4.5-5 Mb. Two additional markers (DXS7593 and DXS7592) were placed in the region, thereby increasing the genetic resolution. Using the deduced marker order, the analysis of key recombinants in families segregating RS allowed us to refine the critical region for RS to 0.6 Mb, between DXS418 and DXS7161.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">1996</span>;<span class="volume">4</span>;<span class="number">2</span>;<span class="pagerange">101-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8744027" rel="external">8744027</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Saccharomyces cerevisiae genome on the World Wide Web.</h4>
<p class="authors"><span class="author">Walsh S</span> and <span class="author">Barrell B</span></p>
<p class="affiliation">Sanger Centre, Hinxton Hall, Hinxton, Cambridge, UK. <a href="mailto:%73%76%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#118;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">1996</span>;<span class="volume">12</span>;<span class="number">7</span>;<span class="pagerange">276-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8763500">8763500</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/cpyngqp2"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/cpyngqp2" src="/qr/cpyngqp2.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
