<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2008 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2008.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2008</h1><p>Number of papers published in 2008: <strong>308</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale molecular analysis of a 34 Mb interval on chromosome 6q: major refinement of the RP25 interval.</h4>
<p class="authors"><span class="author">Abd El-Aziz MM</span>, <span class="author">Barragan I</span>, <span class="author">O&#39;Driscoll C</span>, <span class="author">Borrego S</span>, <span class="author">Abu-Safieh L</span>, <span class="author">Pieras JI</span>, <span class="author">El-Ashry MF</span>, <span class="author">Prigmore E</span>, <span class="author">Carter N</span>, <span class="author">Antinolo G</span> and <span class="author">Bhattacharya SS</span></p>
<p class="affiliation">Department of Molecular Genetics, Institute of Ophthalmology, London EC1V 9EL, UK.</p><p class="abstract">A large scale bioinformatics and molecular analysis of a 34 Mb interval on chromosome 6q12 was undertaken as part of our ongoing study to identify the gene responsible for an autosomal recessive retinitis pigmentosa (arRP) locus, RP25. Extensive bioinformatics analysis indicated in excess of 110 genes within the region and we also noted unfinished sequence on chromosome 6q in the Human Genome Database, between 58 and 61.2 Mb. Forty three genes within the RP25 interval were considered as good candidates for mutation screening. Direct sequence analysis of the selected genes in 7 Spanish families with arRP revealed a total of 244 sequence variants, of which 67 were novel but none were pathogenic. This, together with previous reports, excludes 60 genes within the interval ( approximately 55%) as disease causing for RP. To investigate if copy number variation (CNV) exists within RP25, a comparative genomic hybridization (CGH) analysis was performed on a consanguineous family. A clone from the tiling path array, chr6tp-19C7, spanning approximately 100-Kb was found to be deleted in all affected members of the family, leading to a major refinement of the interval. This will eventually have a significant impact on cloning of the RP25 gene.</p>
<p class="publication"><span class="title">Annals of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">72</span>;<span class="number">Pt 4</span>;<span class="pagerange">463-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18510646" rel="external">18510646</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2689154" rel="external">2689154</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1469-1809.2008.00455.x" rel="external">10.1111/j.1469-1809.2008.00455.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EYS, encoding an ortholog of Drosophila spacemaker, is mutated in autosomal recessive retinitis pigmentosa.</h4>
<p class="authors"><span class="author">Abd El-Aziz MM</span>, <span class="author">Barragan I</span>, <span class="author">O&#39;Driscoll CA</span>, <span class="author">Goodstadt L</span>, <span class="author">Prigmore E</span>, <span class="author">Borrego S</span>, <span class="author">Mena M</span>, <span class="author">Pieras JI</span>, <span class="author">El-Ashry MF</span>, <span class="author">Safieh LA</span>, <span class="author">Shah A</span>, <span class="author">Cheetham ME</span>, <span class="author">Carter NP</span>, <span class="author">Chakarova C</span>, <span class="author">Ponting CP</span>, <span class="author">Bhattacharya SS</span> and <span class="author">Antinolo G</span></p>
<p class="affiliation">Department of Molecular Genetics, Institute of Ophthalmology, London EC1V 9EL, UK.</p><p class="abstract">Using a positional cloning approach supported by comparative genomics, we have identified a previously unreported gene, EYS, at the RP25 locus on chromosome 6q12 commonly mutated in autosomal recessive retinitis pigmentosa. Spanning over 2 Mb, this is the largest eye-specific gene identified so far. EYS is independently disrupted in four other mammalian lineages, including that of rodents, but is well conserved from Drosophila to man and is likely to have a role in the modeling of retinal architecture.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U137761446</span>; Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">11</span>;<span class="pagerange">1285-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18836446">18836446</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2719291">2719291</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.241">10.1038/ng.241</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Contemporary approaches for modifying the mouse genome.</h4>
<p class="authors"><span class="author">Adams DJ</span> and <span class="author">van der Weyden L</span></p>
<p class="affiliation">Experimental Cancer Genetics, The Wellcome Trust Sanger Institute, Hinxton, United Kingdom.</p><p class="abstract">The mouse is a premiere experimental organism that has contributed significantly to our understanding of vertebrate biology. Manipulation of the mouse genome via embryonic stem (ES) cell technology makes it possible to engineer an almost limitless repertoire of mutations to model human disease and assess gene function. In this review we outline recent advances in mouse experimental genetics and provide a &quot;how-to&quot; guide for those people wishing to access this technology. We also discuss new technologies, such as transposon-mediated mutagenesis, and resources of targeting vectors and ES cells, which are likely to dramatically accelerate the pace with which we can assess gene function in vivo, and the progress of forward and reverse genetic screens in mice.</p>
<p class="grants">Funded by: Cancer Research UK; Wellcome Trust</p>
<p class="publication"><span class="title">Physiological genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">34</span>;<span class="number">3</span>;<span class="pagerange">225-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18559964" rel="external">18559964</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2519963" rel="external">2519963</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fphysiolgenomics.90242.2008" rel="external">10.1152/physiolgenomics.90242.2008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Text-mining assisted regulatory annotation.</h4>
<p class="authors"><span class="author">Aerts S</span>, <span class="author">Haeussler M</span>, <span class="author">van Vooren S</span>, <span class="author">Griffith OL</span>, <span class="author">Hulpiau P</span>, <span class="author">Jones SJ</span>, <span class="author">Montgomery SB</span>, <span class="author">Bergman CM</span> and <span class="author">Open Regulatory Annotation Consortium</span></p>
<p class="affiliation">Laboratory of Neurogenetics, Department of Molecular and Developmental Genetics, VIB, Leuven, B-3000, Belgium. <a href="mailto:%73%74%65%69%6E.%61%65%72%74%73%40%6D%65%64.%6B%75%6C%65%75%76%65%6E.%62%65">&#115;&#116;&#101;&#105;&#110;&#46;&#97;&#101;&#114;&#116;&#115;&#64;&#109;&#101;&#100;&#46;&#107;&#117;&#108;&#101;&#117;&#118;&#101;&#110;&#46;&#98;&#101;</a></p><p class="abstract"><strong>Background</strong>: Decoding transcriptional regulatory networks and the genomic cis-regulatory logic implemented in their control nodes is a fundamental challenge in genome biology. High-throughput computational and experimental analyses of regulatory networks and sequences rely heavily on positive control data from prior small-scale experiments, but the vast majority of previously discovered regulatory data remains locked in the biomedical literature.</p><p class="abstract"><strong>Results</strong>: We develop text-mining strategies to identify relevant publications and extract sequence information to assist the regulatory annotation process. Using a vector space model to identify Medline abstracts from papers likely to have high cis-regulatory content, we demonstrate that document relevance ranking can assist the curation of transcriptional regulatory networks and estimate that, minimally, 30,000 papers harbor unannotated cis-regulatory data. In addition, we show that DNA sequences can be extracted from primary text with high cis-regulatory content and mapped to genome sequences as a means of identifying the location, organism and target gene information that is critical to the cis-regulatory annotation process.</p><p class="abstract"><strong>Conclusion</strong>: Our results demonstrate that text-mining technologies can be successfully integrated with genome annotation systems, thereby increasing the availability of annotated cis-regulatory data needed to catalyze advances in the field of gene regulation.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">R31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18271954" rel="external">18271954</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2374703" rel="external">2374703</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-2-r31" rel="external">10.1186/gb-2008-9-2-r31</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic-scale prioritization of drug targets: the TDR Targets database.</h4>
<p class="authors"><span class="author">Ag&uuml;ero F</span>, <span class="author">Al-Lazikani B</span>, <span class="author">Aslett M</span>, <span class="author">Berriman M</span>, <span class="author">Buckner FS</span>, <span class="author">Campbell RK</span>, <span class="author">Carmona S</span>, <span class="author">Carruthers IM</span>, <span class="author">Chan AW</span>, <span class="author">Chen F</span>, <span class="author">Crowther GJ</span>, <span class="author">Doyle MA</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Hopkins AL</span>, <span class="author">McAllister G</span>, <span class="author">Nwaka S</span>, <span class="author">Overington JP</span>, <span class="author">Pain A</span>, <span class="author">Paolini GV</span>, <span class="author">Pieper U</span>, <span class="author">Ralph SA</span>, <span class="author">Riechers A</span>, <span class="author">Roos DS</span>, <span class="author">Sali A</span>, <span class="author">Shanmugam D</span>, <span class="author">Suzuki T</span>, <span class="author">Van Voorhis WC</span> and <span class="author">Verlinde CL</span></p>
<p class="affiliation">Instituto de Investigaciones Biotecnol&oacute;gicas, Universidad Nacional de General San Mart&iacute;n, San Mart&iacute;n 1650, Buenos Aires, Argentina. <a href="mailto:%66%65%72%6E%61%6E%40%75%6E%73%61%6D.%65%64%75.%61%72">&#102;&#101;&#114;&#110;&#97;&#110;&#64;&#117;&#110;&#115;&#97;&#109;&#46;&#101;&#100;&#117;&#46;&#97;&#114;</a></p><p class="abstract">The increasing availability of genomic data for pathogens that cause tropical diseases has created new opportunities for drug discovery and development. However, if the potential of such data is to be fully exploited, the data must be effectively integrated and be easy to interrogate. Here, we discuss the development of the TDR Targets database (http://tdrtargets.org), which encompasses extensive genetic, biochemical and pharmacological data related to tropical disease pathogens, as well as computationally predicted druggability for potential targets and compound desirability information. By allowing the integration and weighting of this information, this database aims to facilitate the identification and prioritization of candidate drug targets for pathogens.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM054762</span></p>
<p class="publication"><span class="title">Nature reviews. Drug discovery</span>&nbsp;<span class="year">2008</span>;<span class="volume">7</span>;<span class="number">11</span>;<span class="pagerange">900-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18927591">18927591</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3184002">3184002</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrd2684">10.1038/nrd2684</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA sequence and structural properties as predictors of human and mouse promoters.</h4>
<p class="authors"><span class="author">Akan P</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Promoters play a central role in gene regulation, yet our power to discriminate them from non-promoter sequences in higher eukaryotes is mainly restricted to those associated with CpG islands. Here, we examined in silico the promoters of 30,954 human and 18,083 mouse transcripts in the DBTSS database, to assess the impact of particular sequence and structural features (propeller twist, bendability and nucleosome positioning preference) on promoter classification and prediction. Our analysis showed that a stricter-than-traditional definition of CpG islands captures low and high CpG count promoter classes more accurately than the traditional one. We observed that both human and mouse promoter sequences are flexible with the exception of the TATA box and TSS, which are rigid regions irrespective of association with a CpG island. Therefore varying levels of structural flexibility in promoters may affect their accessibility to proteins, and hence their specificity. For all features investigated, averaged values across core promoters discriminated CpG island associated promoters from background, whereas the same did not hold for promoters without a CpG island. However, local changes around - 34 to - 23 (expected position of TATA box) and the TSS were informative in discriminating promoters (both classes) from non-promoter sequences. Additionally, we investigated ATG deserts and observed that they occur in all promoter sets except those with a TATA-box and without a CpG island in human. Interestingly, all mouse promoter sets showed ATG codon depletion irrespective of the presence of a TATA-box, possibly reflecting a weaker contribution to TSS specificity in mouse.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">2008</span>;<span class="volume">410</span>;<span class="number">1</span>;<span class="pagerange">165-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18234453" rel="external">18234453</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672154" rel="external">2672154</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gene.2007.12.011" rel="external">10.1016/j.gene.2007.12.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Order within a mosaic distribution of mitochondrial c-type cytochrome biogenesis systems?</h4>
<p class="authors"><span class="author">Allen JW</span>, <span class="author">Jackson AP</span>, <span class="author">Rigden DJ</span>, <span class="author">Willis AC</span>, <span class="author">Ferguson SJ</span> and <span class="author">Ginger ML</span></p>
<p class="affiliation">Department of Biochemistry, University of Oxford, UK.</p><p class="abstract">Mitochondrial cytochromes c and c(1) are present in all eukaryotes that use oxygen as the terminal electron acceptor in the respiratory chain. Maturation of c-type cytochromes requires covalent attachment of the heme cofactor to the protein, and there are at least five distinct biogenesis systems that catalyze this post-translational modification in different organisms and organelles. In this study, we use biochemical data, comparative genomic and structural bioinformatics investigations to provide a holistic view of mitochondrial c-type cytochrome biogenesis and its evolution. There are three pathways for mitochondrial c-type cytochrome maturation, only one of which is present in prokaryotes. We analyze the evolutionary distribution of these biogenesis systems, which include the Ccm system (System I) and the enzyme heme lyase (System III). We conclude that heme lyase evolved once and, in many lineages, replaced the multicomponent Ccm system (present in the proto-mitochondrial endosymbiont), probably as a consequence of lateral gene transfer. We find no evidence of a System III precursor in prokaryotes, and argue that System III is incompatible with multi-heme cytochromes common to bacteria, but absent from eukaryotes. The evolution of the eukaryotic-specific protein heme lyase is strikingly unusual, given that this protein provides a function (thioether bond formation) that is also ubiquitous in prokaryotes. The absence of any known c-type cytochrome biogenesis system from the sequenced genomes of various trypanosome species indicates the presence of a third distinct mitochondrial pathway. Interestingly, this system attaches heme to mitochondrial cytochromes c that contain only one cysteine residue, rather than the usual two, within the heme-binding motif. The isolation of single-cysteine-containing mitochondrial cytochromes c from free-living kinetoplastids, Euglena and the marine flagellate Diplonema papillatum suggests that this unique form of heme attachment is restricted to, but conserved throughout, the protist phylum Euglenozoa.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C508118/1</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The FEBS journal</span>&nbsp;<span class="year">2008</span>;<span class="volume">275</span>;<span class="number">10</span>;<span class="pagerange">2385-402</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18393999" rel="external">18393999</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1742-4658.2008.06380.x" rel="external">10.1111/j.1742-4658.2008.06380.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data growth and its impact on the SCOP database: new developments.</h4>
<p class="authors"><span class="author">Andreeva A</span>, <span class="author">Howorth D</span>, <span class="author">Chandonia JM</span>, <span class="author">Brenner SE</span>, <span class="author">Hubbard TJ</span>, <span class="author">Chothia C</span> and <span class="author">Murzin AG</span></p>
<p class="affiliation">MRC Centre for Protein Engineering, Hills Road, Cambridge CB2 0QH, UK.</p><p class="abstract">The Structural Classification of Proteins (SCOP) database is a comprehensive ordering of all proteins of known structure, according to their evolutionary and structural relationships. The SCOP hierarchy comprises the following levels: Species, Protein, Family, Superfamily, Fold and Class. While keeping the original classification scheme intact, we have changed the production of SCOP in order to cope with a rapid growth of new structural data and to facilitate the discovery of new protein relationships. We describe ongoing developments and new features implemented in SCOP. A new update protocol supports batch classification of new protein structures by their detected relationships at Family and Superfamily levels in contrast to our previous sequential handling of new structural data by release date. We introduce pre-SCOP, a preview of the SCOP developmental version that enables earlier access to the information on new relationships. We also discuss the impact of worldwide Structural Genomics initiatives, which are producing new protein structures at an increasing rate, on the rates of discovery and growth of protein families and superfamilies. SCOP can be accessed at http://scop.mrc-lmb.cam.ac.uk/scop.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105192716</span>; NIGMS NIH HHS: <span class="grant">R01-GM073109</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D419-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18000004">18000004</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238974">2238974</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm993">10.1093/nar/gkm993</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Toward an online repository of Standard Operating Procedures (SOPs) for (meta)genomic annotation.</h4>
<p class="authors"><span class="author">Angiuoli SV</span>, <span class="author">Gussman A</span>, <span class="author">Klimke W</span>, <span class="author">Cochrane G</span>, <span class="author">Field D</span>, <span class="author">Garrity G</span>, <span class="author">Kodira CD</span>, <span class="author">Kyrpides N</span>, <span class="author">Madupu R</span>, <span class="author">Markowitz V</span>, <span class="author">Tatusova T</span>, <span class="author">Thomson N</span> and <span class="author">White O</span></p>
<p class="affiliation">Institute for Genome Sciences, University of Maryland School of Medicine, Baltimore, Maryland 21201, USA. <a href="mailto:%73%61%6E%67%69%75%6F%6C%69%40%73%6F%6D.%75%6D%61%72%79%6C%61%6E%64.%65%64%75">&#115;&#97;&#110;&#103;&#105;&#117;&#111;&#108;&#105;&#64;&#115;&#111;&#109;&#46;&#117;&#109;&#97;&#114;&#121;&#108;&#97;&#110;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The methodologies used to generate genome and metagenome annotations are diverse and vary between groups and laboratories. Descriptions of the annotation process are helpful in interpreting genome annotation data. Some groups have produced Standard Operating Procedures (SOPs) that describe the annotation process, but standards are lacking for structure and content of these descriptions. In addition, there is no central repository to store and disseminate procedures and protocols for genome annotation. We highlight the importance of SOPs for genome annotation and endorse an online repository of SOPs.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">HHSN2662004000386</span></p>
<p class="publication"><span class="title">Omics : a journal of integrative biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">137-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18416670" rel="external">18416670</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3196215" rel="external">3196215</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fomi.2008.0017" rel="external">10.1089/omi.2008.0017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Consistently replicating locus linked to migraine on 10q22-q23.</h4>
<p class="authors"><span class="author">Anttila V</span>, <span class="author">Nyholt DR</span>, <span class="author">Kallela M</span>, <span class="author">Artto V</span>, <span class="author">Veps&auml;l&auml;inen S</span>, <span class="author">Jakkula E</span>, <span class="author">Wennerstr&ouml;m A</span>, <span class="author">Tikka-Kleemola P</span>, <span class="author">Kaunisto MA</span>, <span class="author">H&auml;m&auml;l&auml;inen E</span>, <span class="author">Wid&eacute;n E</span>, <span class="author">Terwilliger J</span>, <span class="author">Merikangas K</span>, <span class="author">Montgomery GW</span>, <span class="author">Martin NG</span>, <span class="author">Daly M</span>, <span class="author">Kaprio J</span>, <span class="author">Peltonen L</span>, <span class="author">F&auml;rkkil&auml; M</span>, <span class="author">Wessman M</span> and <span class="author">Palotie A</span></p>
<p class="affiliation">Biomedicum Helsinki, Research Program in Molecular Medicine, University of Helsinki, 00290 Helsinki, Finland.</p><p class="abstract">Here, we present the results of two genome-wide scans in two diverse populations in which a consistent use of recently introduced migraine-phenotyping methods detects and replicates a locus on 10q22-q23, with an additional independent replication. No genetic variants have been convincingly established in migraine, and although several loci have been reported, none of them has been consistently replicated. We employed the three known migraine-phenotyping methods (clinical end diagnosis, latent-class analysis, and trait-component analysis) with robust multiple testing correction in a large sample set of 1675 individuals from 210 migraine families from Finland and Australia. Genome-wide multipoint linkage analysis that used the Kong and Cox exponential model in Finns detected a locus on 10q22-q23 with highly significant evidence of linkage (LOD 7.68 at 103 cM in female-specific analysis). The Australian sample showed a LOD score of 3.50 at the same locus (100 cM), as did the independent Finnish replication study (LOD score 2.41, at 102 cM). In addition, four previously reported loci on 8q21, 14q21, 18q12, and Xp21 were also replicated. A shared-segment analysis of 10q22-q23 linked Finnish families identified a 1.6-9.5 cM segment, centered on 101 cM, which shows in-family homology in 95% of affected Finns. This region was further studied with 1323 SNPs. Although no significant association was observed, four regions warranting follow-up studies were identified. These results support the use of symptomology-based phenotyping in migraine and suggest that the 10q22-q23 locus probably contains one or more migraine susceptibility variants.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">U54 RR020278</span>; NIAAA NIH HHS: <span class="grant">AA007535</span>, <span class="grant">AA013320</span>, <span class="grant">AA013326</span>, <span class="grant">AA014041</span>, <span class="grant">AA07728</span>, <span class="grant">AA10249</span>, <span class="grant">AA11998</span>; NINDS NIH HHS: <span class="grant">R01 NS37675</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">5</span>;<span class="pagerange">1051-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18423523">18423523</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427232">2427232</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.03.003">10.1016/j.ajhg.2008.03.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bortezomib-induced peripheral neuropathy in multiple myeloma: a comprehensive review of the literature.</h4>
<p class="authors"><span class="author">Argyriou AA</span>, <span class="author">Iconomou G</span> and <span class="author">Kalofonos HP</span></p>
<p class="affiliation">Department of Neurology, Saint Andrew&#39;s General Hospital of Patras, Greece.</p><p class="abstract">Bortezomib has demonstrated significant activity in clinical trials, mainly against recurrent or newly diagnosed multiple myeloma (MM). Peripheral neuropathy is a significant toxicity of bortezomib, requiring dose modification and potential changes in the treatment plan when it occurs. The mechanism underlying bortezomib-induced peripheral neuropathy (BIPN) is unknown. Metabolic changes resulting from the accumulation of bortezomib in the dorsal root ganglia cells, mitochondrial-mediated disregulation of Ca(++) homeostasis, and disregulation of neurotrophins may contribute to the pathogenesis of BIPN. It is increasingly recognized that BIPN may be a proteasome inhibitor class effect, producing primarily a small fiber and painful, axonal, sensory distal neuropathy. Incidence of BIPN is mainly related to various risk factors, including cumulative dose and evidence of preexisting neuropathy. Assessment of BIPN is based primarily on neurologic clinical examination and neurophysiologic methods. To date, apart from the use of dose reduction and schedule change algorithm, there is no effective treatment with neuroprotective agents for BIPN. Analgesics, tricyclic antidepressants, anticonvulsants, and vitamin supplements have been used as symptomatic treatment against bortezomib-associated neuropathic pain with some success. This review looks critically at the pathogenesis, incidence, risk factors, diagnosis, characteristics, and management of BIPN, and highlights areas for future research.</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2008</span>;<span class="volume">112</span>;<span class="number">5</span>;<span class="pagerange">1593-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18574024" rel="external">18574024</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2008-04-149385" rel="external">10.1182/blood-2008-04-149385</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tumor necrosis factor SNP haplotypes are associated with iron deficiency anemia in West African children.</h4>
<p class="authors"><span class="author">Atkinson SH</span>, <span class="author">Rockett KA</span>, <span class="author">Morgan G</span>, <span class="author">Bejon PA</span>, <span class="author">Sirugo G</span>, <span class="author">O&#39;Connell MA</span>, <span class="author">Hanchard N</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Prentice AM</span></p>
<p class="affiliation">Medical Research Council (MRC) Keneba and MRC Laboratories, Banjul, The Gambia. <a href="mailto:%73%61%72%61%68.%61%74%6B%69%6E%73%6F%6E%40%70%61%65%64%69%61%74%72%69%63%73.%6F%78.%61%63.%75%6B">&#115;&#97;&#114;&#97;&#104;&#46;&#97;&#116;&#107;&#105;&#110;&#115;&#111;&#110;&#64;&#112;&#97;&#101;&#100;&#105;&#97;&#116;&#114;&#105;&#99;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Plasma levels of tumor necrosis factor-alpha (TNF-alpha) are significantly raised in malaria infection and TNF-alpha is thought to inhibit intestinal iron absorption and macrophage iron release. This study investigated putative functional single nucleotide polymorphisms (SNPs) and haplotypes across the major histocompatibility complex (MHC) class III region, including TNF and its immediate neighbors nuclear factor of kappa light polypeptide gene enhancer in B cells (lkappaBL), inhibitor-like 1 and lymphotoxin alpha (LTA), in relation to nutritional iron status and anemia, in a cohort of 780 children across a malaria season. The prevalence of iron deficiency anemia (IDA) increased over the malaria season (P &lt; .001). The TNF(-308) AA genotype was associated with an increased risk of iron deficiency (adjusted OR 8.1; P = .001) and IDA (adjusted OR 5.1; P = .01) at the end of the malaria season. No genotypes were associated with IDA before the malaria season. Thus, TNF appears to be a risk factor for iron deficiency and IDA in children in a malaria-endemic environment and this is likely to be due to a TNF-alpha-induced block in iron absorption.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2008</span>;<span class="volume">112</span>;<span class="number">10</span>;<span class="pagerange">4276-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18716131" rel="external">18716131</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2581977" rel="external">2581977</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2008-06-162008" rel="external">10.1182/blood-2008-06-162008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenetic marking prepares the human HOXA cluster for activation during differentiation of pluripotent cells.</h4>
<p class="authors"><span class="author">Atkinson SP</span>, <span class="author">Koch CM</span>, <span class="author">Clelland GK</span>, <span class="author">Willcox S</span>, <span class="author">Fowler JC</span>, <span class="author">Stewart R</span>, <span class="author">Lako M</span>, <span class="author">Dunham I</span> and <span class="author">Armstrong L</span></p>
<p class="affiliation">North East Institute for Stem Cell Research, University of Newcastle upon Tyne, International Centre for Life, Newcastle upon Tyne, United Kingdom.</p><p class="abstract">Activation of Hox gene clusters is an early event in embryonic development since individual members play important roles in patterning of the body axis. Their functions require precise control of spatiotemporal expression to provide positional information for the cells of the developing embryo, and the manner by which this control is achieved has generated considerable interest. The situation is different in pluripotent cells, where HOX genes are not expressed but are held in potentio as bivalent chromatin domains, which are resolved upon differentiation to permit HOX cluster activation. In this study we have used differentiation of the pluripotent embryonal carcinoma cell line NTera2SP12 and the human embryonic stem cell line H9 to examine epigenetic changes that accompany activation of the HOXA cluster and show that specific genomic loci are marked by lysine methylation of histone H3 (H3K4 tri- and dimethyl, H3K9 trimethyl) and acetylation of histone H4 even in the undifferentiated cells. The precise locations of such modified histones may be involved in controlling the colinear expression of genes from the cluster.</p>
<p class="publication"><span class="title">Stem cells (Dayton, Ohio)</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">5</span>;<span class="pagerange">1174-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18292213" rel="external">18292213</a>; DOI: <a href="http://dx.doi.org/10.1634%2Fstemcells.2007-0497" rel="external">10.1634/stemcells.2007-0497</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of Staphylococcus aureus strain Newman and comparative analysis of staphylococcal genomes: polymorphism and evolution of two major pathogenicity islands.</h4>
<p class="authors"><span class="author">Baba T</span>, <span class="author">Bae T</span>, <span class="author">Schneewind O</span>, <span class="author">Takeuchi F</span> and <span class="author">Hiramatsu K</span></p>
<p class="affiliation">Department of Microbiology and Infection Control Science, Juntendo University, Tokyo 113-8421, Japan. <a href="mailto:%74%62%61%62%61%40%6D%65%64.%6A%75%6E%74%65%6E%64%6F.%61%63.%6A%70">&#116;&#98;&#97;&#98;&#97;&#64;&#109;&#101;&#100;&#46;&#106;&#117;&#110;&#116;&#101;&#110;&#100;&#111;&#46;&#97;&#99;&#46;&#106;&#112;</a></p><p class="abstract">Strains of Staphylococcus aureus, an important human pathogen, display up to 20% variability in their genome sequence, and most sequence information is available for human clinical isolates that have not been subjected to genetic analysis of virulence attributes. S. aureus strain Newman, which was also isolated from a human infection, displays robust virulence properties in animal models of disease and has already been extensively analyzed for its molecular traits of staphylococcal pathogenesis. We report here the complete genome sequence of S. aureus Newman, which carries four integrated prophages, as well as two large pathogenicity islands. In agreement with the view that S. aureus Newman prophages contribute important properties to pathogenesis, fewer virulence factors are found outside of the prophages than for the highly virulent strain MW2. The absence of drug resistance genes reflects the general antibiotic-susceptible phenotype of S. aureus Newman. Phylogenetic analyses reveal clonal relationships between the staphylococcal strains Newman, COL, NCTC8325, and USA300 and a greater evolutionary distance to strains MRSA252, MW2, MSSA476, N315, Mu50, JH1, JH9, and RF122. However, polymorphism analysis of two large pathogenicity islands distributed among these strains shows that the two islands were acquired independently from the evolutionary pathway of the chromosomal backbones of staphylococcal genomes. Prophages and pathogenicity islands play central roles in S. aureus virulence and evolution.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">1</span>;<span class="pagerange">300-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17951380" rel="external">17951380</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2223734" rel="external">2223734</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01000-07" rel="external">10.1128/JB.01000-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput genotyping of Salmonella enterica serovar Typhi allowing geographical assignment of haplotypes and pathotypes within an urban District of Jakarta, Indonesia.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Holt K</span>, <span class="author">van de Vosse E</span>, <span class="author">Roumagnac P</span>, <span class="author">Whitehead S</span>, <span class="author">King E</span>, <span class="author">Ewels P</span>, <span class="author">Keniry A</span>, <span class="author">Weill FX</span>, <span class="author">Lightfoot D</span>, <span class="author">van Dissel JT</span>, <span class="author">Sanderson KE</span>, <span class="author">Farrar J</span>, <span class="author">Achtman M</span>, <span class="author">Deloukas P</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%73%62%61%6B%65%72%40%6F%75%63%72%75.%6F%72%67">&#115;&#98;&#97;&#107;&#101;&#114;&#64;&#111;&#117;&#99;&#114;&#117;&#46;&#111;&#114;&#103;</a></p><p class="abstract">High-throughput epidemiological typing systems that provide phylogenetic and genotypic information are beneficial for tracking bacterial pathogens in the field. The incidence of Salmonella enterica serovar Typhi infection in Indonesia is high and is associated with atypical phenotypic traits such as expression of the j and the z66 flagellum antigens. Utilizing a high-throughput genotyping platform to investigate known nucleotide polymorphisms dispersed around the genome, we determined the haplotypes of 140 serovar Typhi isolates associated with Indonesia. We identified nine distinct serovar Typhi haplotypes circulating in Indonesia for more than 30 years, with eight of these present in a single Jakarta suburb within a 2-year period. One dominant haplotype, H59, is associated with j and z66 flagellum expression, representing a potential pathotype unique to Indonesia. Phylogenetic analysis suggests that H59 z66(+), j(+) isolates emerged relatively recently in terms of the origin of serovar Typhi and are geographically restricted. These data demonstrate the potential of high-throughput genotyping platforms for analyzing serovar Typhi populations in the field. The study also provides insight into the evolution of serovar Typhi and demonstrates the value of a molecular epidemiological technique that is exchangeable, that is internet friendly, and that has global utility.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">46</span>;<span class="number">5</span>;<span class="pagerange">1741-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18322069" rel="external">18322069</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2395080" rel="external">2395080</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.02249-07" rel="external">10.1128/JCM.02249-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mobilization of the incQ plasmid R300B with a chromosomal conjugation system in Salmonella enterica serovar typhi.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Pickard D</span>, <span class="author">Whitehead S</span>, <span class="author">Farrar J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%73%62%61%6B%65%72%40%6F%75%63%72%75.%6F%72%67">&#115;&#98;&#97;&#107;&#101;&#114;&#64;&#111;&#117;&#99;&#114;&#117;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Salmonella pathogenicity island 7 (SPI-7) in Salmonella enterica serovar Typhi appears to be related to other genomic islands. Evidence suggests that SPI-7 is susceptible to spontaneous circularization, loss, and transposition. Here, we demonstrate that a region within SPI-7 has the ability to mobilize the small incQ plasmid R300B.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">11</span>;<span class="pagerange">4084-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18390666" rel="external">18390666</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2395054" rel="external">2395054</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00065-08" rel="external">10.1128/JB.00065-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamic nature of the proximal AZFc region of the human Y chromosome: multiple independent deletion and duplication events revealed by microsatellite analysis.</h4>
<p class="authors"><span class="author">Balaresque P</span>, <span class="author">Bowden GR</span>, <span class="author">Parkin EJ</span>, <span class="author">Omran GA</span>, <span class="author">Heyer E</span>, <span class="author">Quintana-Murci L</span>, <span class="author">Roewer L</span>, <span class="author">Stoneking M</span>, <span class="author">Nasidze I</span>, <span class="author">Carvalho-Silva DR</span>, <span class="author">Tyler-Smith C</span>, <span class="author">de Knijff P</span> and <span class="author">Jobling MA</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, Leicester, United Kingdom.</p><p class="abstract">The human Y chromosome shows frequent structural variants, some of which are selectively neutral, while others cause impaired fertility due to the loss of spermatogenic genes. The large-scale use of multiple Y-chromosomal microsatellites in forensic and population genetic studies can reveal such variants, through the absence or duplication of specific markers in haplotypes. We describe Y chromosomes in apparently normal males carrying null and duplicated alleles at the microsatellite DYS448, which lies in the proximal part of the azoospermia factor c (AZFc) region, important in spermatogenesis, and made up of &quot;ampliconic&quot; repeats that act as substrates for nonallelic homologous recombination (NAHR). Physical mapping in 26 DYS448 deletion chromosomes reveals that only three cases belong to a previously described class, representing independent occurrences of an approximately 1.5-Mb deletion mediated by recombination between the b1 and b3 repeat units. The remainder belong to five novel classes; none appears to be mediated through homologous recombination, and all remove some genes, but are likely to be compatible with normal fertility. A combination of deletion analysis with binary-marker and microsatellite haplotyping shows that the 26 deletions represent nine independent events. Nine DYS448 duplication chromosomes can be explained by four independent events. Some lineages have risen to high frequency in particular populations, in particular a deletion within haplogroup (hg) C(*)(xC3a,C3c) found in 18 Asian males. The nonrandom phylogenetic distribution of duplication and deletion events suggests possible structural predisposition to such mutations in hgs C and G.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span>, <span class="grant">077009</span></p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2008</span>;<span class="volume">29</span>;<span class="number">10</span>;<span class="pagerange">1171-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18470947">18470947</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2689608">2689608</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.20757">10.1002/humu.20757</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Autosomal-dominant microtia linked to five tandem copies of a copy-number-variable region at chromosome 4p16.</h4>
<p class="authors"><span class="author">Balikova I</span>, <span class="author">Martens K</span>, <span class="author">Melotte C</span>, <span class="author">Amyere M</span>, <span class="author">Van Vooren S</span>, <span class="author">Moreau Y</span>, <span class="author">Vetrie D</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span>, <span class="author">Liehr T</span>, <span class="author">Vikkula M</span>, <span class="author">Matthijs G</span>, <span class="author">Fryns JP</span>, <span class="author">Casteels I</span>, <span class="author">Devriendt K</span> and <span class="author">Vermeesch JR</span></p>
<p class="affiliation">Center for Human Genetics, University of Leuven, 3000 Leuven, Belgium.</p><p class="abstract">Recently, large-scale benign copy-number variations (CNVs)--encompassing over 12% of the genome and containing genes considered to be dosage tolerant for human development--were uncovered in the human population. Here we present a family with a novel autosomal-dominantly inherited syndrome characterized by microtia, eye coloboma, and imperforation of the nasolacrimal duct. This phenotype is linked to a cytogenetically visible alteration at 4pter consisting of five copies of a copy-number-variable region, encompassing a low-copy repeat (LCR)-rich sequence. We demonstrate that the approximately 750 kb amplicon occurs in exact tandem copies. This is the first example of an amplified CNV associated with a Mendelian disorder, a discovery that implies that genome screens for genetic disorders should include the analysis of so-called benign CNVs and LCRs.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">1</span>;<span class="pagerange">181-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18179897" rel="external">18179897</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2253954" rel="external">2253954</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2007.08.001" rel="external">10.1016/j.ajhg.2007.08.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Breakpoint mapping and array CGH in translocations: comparison of a phenotypically normal and an abnormal cohort.</h4>
<p class="authors"><span class="author">Baptista J</span>, <span class="author">Mercer C</span>, <span class="author">Prigmore E</span>, <span class="author">Gribble SM</span>, <span class="author">Carter NP</span>, <span class="author">Maloney V</span>, <span class="author">Thomas NS</span>, <span class="author">Jacobs PA</span> and <span class="author">Crolla JA</span></p>
<p class="affiliation">Wessex Regional Genetics Laboratory, Salisbury District Hospital, Salisbury, Wiltshire, UK.</p><p class="abstract">We report the analyses of breakpoints in 31 phenotypically normal and 14 abnormal carriers of balanced translocations. Our study assesses the differences between balanced translocations in normal carriers and those in abnormal carriers, focusing on the presence of genomic imbalances at the breakpoints or elsewhere in the genome, presence of cryptic chromosome rearrangements, and gene disruption. Our hypothesis is that all four features will be associated with phenotypic abnormalities and absent or much less frequent in a normal population. In the normal cohort, we identified neither genomic imbalances at the breakpoints or elsewhere in the genome nor cryptic chromosome rearrangements. In contrast, we identified candidate disease-causing imbalances in 4/14 abnormal patients. These were three breakpoint associated deletions and three deletions unrelated to the breakpoints. All six de novo deletions originated on the paternally inherited chromosome. Additional complexity was also present in one of these cases. Gene disruption by the breakpoints was present in 16/31 phenotypically normal individuals and in 5/14 phenotypically abnormal patients. Our results show that translocations in phenotypically abnormal patients are molecularly distinct from those in normal individuals: the former are more likely to be associated with genomic imbalances at the breakpoints or elsewhere and with chromosomal complexity, whereas the frequency of gene disruption is similar in both normal and abnormal translocation carriers.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">4</span>;<span class="pagerange">927-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18371933" rel="external">18371933</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427237" rel="external">2427237</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.02.012" rel="external">10.1016/j.ajhg.2008.02.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A robust statistical method for case-control association testing with copy number variation.</h4>
<p class="authors"><span class="author">Barnes C</span>, <span class="author">Plagnol V</span>, <span class="author">Fitzgerald T</span>, <span class="author">Redon R</span>, <span class="author">Marchini J</span>, <span class="author">Clayton D</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Copy number variation (CNV) is pervasive in the human genome and can play a causal role in genetic diseases. The functional impact of CNV cannot be fully captured through linkage disequilibrium with SNPs. These observations motivate the development of statistical methods for performing direct CNV association studies. We show through simulation that current tests for CNV association are prone to false-positive associations in the presence of differential errors between cases and controls, especially if quantitative CNV measurements are noisy. We present a statistical framework for performing case-control CNV association studies that applies likelihood ratio testing of quantitative CNV measurements in cases and controls. We show that our methods are robust to differential errors and noisy data and can achieve maximal theoretical power. We illustrate the power of these methods for testing for association with binary and quantitative traits, and have made this software available as the R package CNVtools.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">061860</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">10</span>;<span class="pagerange">1245-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18776912">18776912</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2784596">2784596</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.206">10.1038/ng.206</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association defines more than 30 distinct susceptibility loci for Crohn&#39;s disease.</h4>
<p class="authors"><span class="author">Barrett JC</span>, <span class="author">Hansoul S</span>, <span class="author">Nicolae DL</span>, <span class="author">Cho JH</span>, <span class="author">Duerr RH</span>, <span class="author">Rioux JD</span>, <span class="author">Brant SR</span>, <span class="author">Silverberg MS</span>, <span class="author">Taylor KD</span>, <span class="author">Barmada MM</span>, <span class="author">Bitton A</span>, <span class="author">Dassopoulos T</span>, <span class="author">Datta LW</span>, <span class="author">Green T</span>, <span class="author">Griffiths AM</span>, <span class="author">Kistner EO</span>, <span class="author">Murtha MT</span>, <span class="author">Regueiro MD</span>, <span class="author">Rotter JI</span>, <span class="author">Schumm LP</span>, <span class="author">Steinhart AH</span>, <span class="author">Targan SR</span>, <span class="author">Xavier RJ</span>, <span class="author">NIDDK IBD Genetics Consortium</span>, <span class="author">Libioulle C</span>, <span class="author">Sandor C</span>, <span class="author">Lathrop M</span>, <span class="author">Belaiche J</span>, <span class="author">Dewit O</span>, <span class="author">Gut I</span>, <span class="author">Heath S</span>, <span class="author">Laukens D</span>, <span class="author">Mni M</span>, <span class="author">Rutgeerts P</span>, <span class="author">Van Gossum A</span>, <span class="author">Zelenika D</span>, <span class="author">Franchimont D</span>, <span class="author">Hugot JP</span>, <span class="author">de Vos M</span>, <span class="author">Vermeire S</span>, <span class="author">Louis E</span>, <span class="author">Belgian-French IBD Consortium</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Cardon LR</span>, <span class="author">Anderson CA</span>, <span class="author">Drummond H</span>, <span class="author">Nimmo E</span>, <span class="author">Ahmad T</span>, <span class="author">Prescott NJ</span>, <span class="author">Onnie CM</span>, <span class="author">Fisher SA</span>, <span class="author">Marchini J</span>, <span class="author">Ghori J</span>, <span class="author">Bumpstead S</span>, <span class="author">Gwilliam R</span>, <span class="author">Tremelling M</span>, <span class="author">Deloukas P</span>, <span class="author">Mansfield J</span>, <span class="author">Jewell D</span>, <span class="author">Satsangi J</span>, <span class="author">Mathew CG</span>, <span class="author">Parkes M</span>, <span class="author">Georges M</span> and <span class="author">Daly MJ</span></p>
<p class="affiliation">Bioinformatics and Statistical Genetics, Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK.</p><p class="abstract">Several risk factors for Crohn&#39;s disease have been identified in recent genome-wide association studies. To advance gene discovery further, we combined data from three studies on Crohn&#39;s disease (a total of 3,230 cases and 4,829 controls) and carried out replication in 3,664 independent cases with a mixture of population-based and family-based controls. The results strongly confirm 11 previously reported loci and provide genome-wide significant evidence for 21 additional loci, including the regions containing STAT3, JAK2, ICOSLG, CDKAL1 and ITLN1. The expanded molecular understanding of the basis of this disease offers promise for informed therapeutic development.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0600329</span>, <span class="grant">G0800759</span>; NIAID NIH HHS: <span class="grant">AI06277</span>, <span class="grant">R01 AI062773</span>, <span class="grant">R01 AI062773-02</span>; NIDDK NIH HHS: <span class="grant">DK064869</span>, <span class="grant">DK62413</span>, <span class="grant">DK62420</span>, <span class="grant">DK62422</span>, <span class="grant">DK62423</span>, <span class="grant">DK62429</span>, <span class="grant">DK62431</span>, <span class="grant">DK62432</span>, <span class="grant">P30 DK040561</span>, <span class="grant">P30 DK040561-13</span>, <span class="grant">P30 DK063491</span>, <span class="grant">P30 DK063491-019004</span>, <span class="grant">P30 DK063491-029004</span>, <span class="grant">P30 DK063491-039004</span>, <span class="grant">P30 DK063491-049004</span>, <span class="grant">P30 DK063491-05</span>, <span class="grant">R01 DK064869</span>, <span class="grant">R01 DK064869-04</span>, <span class="grant">U01 DK062413</span>, <span class="grant">U01 DK062413-06</span>, <span class="grant">U01 DK062420</span>, <span class="grant">U01 DK062420-01</span>, <span class="grant">U01 DK062420-02</span>, <span class="grant">U01 DK062420-03</span>, <span class="grant">U01 DK062420-04</span>, <span class="grant">U01 DK062420-05</span>, <span class="grant">U01 DK062420-06</span>, <span class="grant">U01 DK062422</span>, <span class="grant">U01 DK062422-07</span>, <span class="grant">U01 DK062423</span>, <span class="grant">U01 DK062423-06</span>, <span class="grant">U01 DK062429</span>, <span class="grant">U01 DK062429-07</span>, <span class="grant">U01 DK062431</span>, <span class="grant">U01 DK062431-06</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">8</span>;<span class="pagerange">955-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18587394">18587394</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2574810">2574810</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.175">10.1038/ng.175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population-specific risk of type 2 diabetes conferred by HNF4A P2 promoter variants: a lesson for replication studies.</h4>
<p class="authors"><span class="author">Barroso I</span>, <span class="author">Luan J</span>, <span class="author">Wheeler E</span>, <span class="author">Whittaker P</span>, <span class="author">Wasson J</span>, <span class="author">Zeggini E</span>, <span class="author">Weedon MN</span>, <span class="author">Hunt S</span>, <span class="author">Venkatesh R</span>, <span class="author">Frayling TM</span>, <span class="author">Delgado M</span>, <span class="author">Neuman RJ</span>, <span class="author">Zhao J</span>, <span class="author">Sherva R</span>, <span class="author">Glaser B</span>, <span class="author">Walker M</span>, <span class="author">Hitman G</span>, <span class="author">McCarthy MI</span>, <span class="author">Hattersley AT</span>, <span class="author">Permutt MA</span>, <span class="author">Wareham NJ</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%69%62%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#98;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Objective</strong>: Single nucleotide polymorphisms (SNPs) in the P2 promoter region of HNF4A were originally shown to be associated with predisposition for type 2 diabetes in Finnish, Ashkenazi, and, more recently, Scandinavian populations, but they generated conflicting results in additional populations. We aimed to investigate whether data from a large-scale mapping approach would replicate this association in novel Ashkenazi samples and in U.K. populations and whether these data would allow us to refine the association signal.</p><p class="abstract">Using a dense linkage disequilibrium map of 20q, we selected SNPs from a 10-Mb interval centered on HNF4A. In a staged approach, we first typed 4,608 SNPs in case-control populations from four U.K. populations and an Ashkenazi population (n = 2,516). In phase 2, a subset of 763 SNPs was genotyped in 2,513 additional samples from the same populations.</p><p class="abstract"><strong>Results</strong>: Combined analysis of both phases demonstrated association between HNF4A P2 SNPs (rs1884613 and rs2144908) and type 2 diabetes in the Ashkenazim (n = 991; P &lt; 1.6 x 10(-6)). Importantly, these associations are significant in a subset of Ashkenazi samples (n = 531) not previously tested for association with P2 SNPs (odds ratio [OR] approximately 1.7; P &lt; 0.002), thus providing replication within the Ashkenazim. In the U.K. populations, this association was not significant (n = 4,022; P &gt; 0.5), and the estimate for the OR was much smaller (OR 1.04; [95%CI 0.91-1.19]).</p><p class="abstract"><strong>Conclusions</strong>: These data indicate that the risk conferred by HNF4A P2 is significantly different between U.K. and Ashkenazi populations (P &lt; 0.00007), suggesting that the underlying causal variant remains unidentified. Interactions with other genetic or environmental factors may also contribute to this difference in risk between populations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>; PHS HHS: <span class="grant">R01K049583</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077016</span>, <span class="grant">079557</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2008</span>;<span class="volume">57</span>;<span class="number">11</span>;<span class="pagerange">3161-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18728231">18728231</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2570416">2570416</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb08-0719">10.2337/db08-0719</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Re-evaluation of putative rheumatoid arthritis susceptibility genes in the post-genome wide association study era and hypothesis of a key pathway underlying susceptibility.</h4>
<p class="authors"><span class="author">Barton A</span>, <span class="author">Thomson W</span>, <span class="author">Ke X</span>, <span class="author">Eyre S</span>, <span class="author">Hinks A</span>, <span class="author">Bowes J</span>, <span class="author">Gibbons L</span>, <span class="author">Plant D</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Wilson AG</span>, <span class="author">Marinou I</span>, <span class="author">Morgan A</span>, <span class="author">Emery P</span>, <span class="author">YEAR consortium</span>, <span class="author">Steer S</span>, <span class="author">Hocking L</span>, <span class="author">Reid DM</span>, <span class="author">Wordsworth P</span>, <span class="author">Harrison P</span> and <span class="author">Worthington J</span></p>
<p class="affiliation">Arc-Epidemiology Unit, Stopford Building, The University of Manchester, Manchester, UK. <a href="mailto:%61%6E%6E%65.%62%61%72%74%6F%6E%40%6D%61%6E%63%68%65%73%74%65%72.%61%63.%75%6B">&#97;&#110;&#110;&#101;&#46;&#98;&#97;&#114;&#116;&#111;&#110;&#64;&#109;&#97;&#110;&#99;&#104;&#101;&#115;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Rheumatoid arthritis (RA) is an archetypal, common, complex autoimmune disease with both genetic and environmental contributions to disease aetiology. Two novel RA susceptibility loci have been reported from recent genome-wide and candidate gene association studies. We, therefore, investigated the evidence for association of the STAT4 and TRAF1/C5 loci with RA using imputed data from the Wellcome Trust Case Control Consortium (WTCCC). No evidence for association of variants mapping to the TRAF1/C5 gene was detected in the 1860 RA cases and 2930 control samples tested in that study. Variants mapping to the STAT4 gene did show evidence for association (rs7574865, P = 0.04). Given the association of the TRAF1/C5 locus in two previous large case-control series from populations of European descent and the evidence for association of the STAT4 locus in the WTCCC study, single nucleotide polymorphisms mapping to these loci were tested for association with RA in an independent UK series comprising DNA from &gt;3000 cases with disease and &gt;3000 controls and a combined analysis including the WTCCC data was undertaken. We confirm association of the STAT4 and the TRAF1/C5 loci with RA bringing to 5 the number of confirmed susceptibility loci. The effect sizes are less than those reported previously but are likely to be a more accurate reflection of the true effect size given the larger size of the cohort investigated in the current study.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>, <span class="grant">18475</span>; Medical Research Council: <span class="grant">G0000934</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">15</span>;<span class="pagerange">2274-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18434327">18434327</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2465799">2465799</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddn128">10.1093/hmg/ddn128</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rheumatoid arthritis susceptibility loci at chromosomes 10p15, 12q13 and 22q13.</h4>
<p class="authors"><span class="author">Barton A</span>, <span class="author">Thomson W</span>, <span class="author">Ke X</span>, <span class="author">Eyre S</span>, <span class="author">Hinks A</span>, <span class="author">Bowes J</span>, <span class="author">Plant D</span>, <span class="author">Gibbons LJ</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">YEAR Consortium</span>, <span class="author">BIRAC Consortium</span>, <span class="author">Wilson AG</span>, <span class="author">Bax DE</span>, <span class="author">Morgan AW</span>, <span class="author">Emery P</span>, <span class="author">Steer S</span>, <span class="author">Hocking L</span>, <span class="author">Reid DM</span>, <span class="author">Wordsworth P</span>, <span class="author">Harrison P</span> and <span class="author">Worthington J</span></p>
<p class="affiliation">Arthritis Research Campaign, Epidemiology Unit, The University of Manchester, Manchester, UK. <a href="mailto:%61%6E%6E%65.%62%61%72%74%6F%6E%40%6D%61%6E%63%68%65%73%74%65%72.%61%63.%75%6B">&#97;&#110;&#110;&#101;&#46;&#98;&#97;&#114;&#116;&#111;&#110;&#64;&#109;&#97;&#110;&#99;&#104;&#101;&#115;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The WTCCC study identified 49 SNPs putatively associated with rheumatoid arthritis at P = 1 x 10(-4) - 1 x 10(-5) (tier 3). Here we show that three of these SNPs, mapping to chromosome 10p15 (rs4750316), 12q13 (rs1678542) and 22q13 (rs3218253), are also associated (trend P = 4 x 10(-5), P = 4 x 10(-4) and P = 4 x 10(-4), respectively) in a validation study of 4,106 individuals with rheumatoid arthritis and an expanded reference group of 11,238 subjects, confirming them as true susceptibility loci in individuals of European ancestry.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>, <span class="grant">18475</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0600329</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">10</span>;<span class="pagerange">1156-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18794857">18794857</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2662493">2662493</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.218">10.1038/ng.218</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Convergent evolution in the genetic basis of M&uuml;llerian mimicry in heliconius butterflies.</h4>
<p class="authors"><span class="author">Baxter SW</span>, <span class="author">Papa R</span>, <span class="author">Chamberlain N</span>, <span class="author">Humphray SJ</span>, <span class="author">Joron M</span>, <span class="author">Morrison C</span>, <span class="author">ffrench-Constant RH</span>, <span class="author">McMillan WO</span> and <span class="author">Jiggins CD</span></p>
<p class="affiliation">Department of Zoology, University of Cambridge, Cambridge, United Kingdom. <a href="mailto:%73%77%62%32%39%40%63%61%6D.%61%63.%75%6B">&#115;&#119;&#98;&#50;&#57;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The neotropical butterflies Heliconius melpomene and H. erato are M&uuml;llerian mimics that display the same warningly colored wing patterns in local populations, yet pattern diversity between geographic regions. Linkage mapping has previously shown convergent red wing phenotypes in these species are controlled by loci on homologous chromosomes. Here, AFLP bulk segregant analysis using H. melpomene crosses identified genetic markers tightly linked to two red wing-patterning loci. These markers were used to screen a H. melpomene BAC library and a tile path was assembled spanning one locus completely and part of the second. Concurrently, a similar strategy was used to identify a BAC clone tightly linked to the locus controlling the mimetic red wing phenotypes in H. erato. A methionine rich storage protein (MRSP) gene was identified within this BAC clone, and comparative genetic mapping shows red wing color loci are in homologous regions of the genome of H. erato and H. melpomene. Subtle differences in these convergent phenotypes imply they evolved independently using somewhat different developmental routes, but are nonetheless regulated by the same switch locus. Genetic mapping of MRSP in a third related species, the &quot;tiger&quot; patterned H. numata, has no association with wing patterning and shows no evidence for genomic translocation of wing-patterning loci.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">180</span>;<span class="number">3</span>;<span class="pagerange">1567-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18791259" rel="external">18791259</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2581958" rel="external">2581958</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.107.082982" rel="external">10.1534/genetics.107.082982</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel 154-bp deletion in the human mitochondrial DNA control region in healthy individuals.</h4>
<p class="authors"><span class="author">Behar DM</span>, <span class="author">Blue-Smith J</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Tzur S</span>, <span class="author">Hadid Y</span>, <span class="author">Bormans C</span>, <span class="author">Moen A</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Quintana-Murci L</span>, <span class="author">Wells RS</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Molecular Medicine Laboratory, Rambam Health Care Campus, Haifa, Israel. <a href="mailto:%47%65%6E%6F%50%75%62%73%40%6E%67%73.%6F%72%67">&#71;&#101;&#110;&#111;&#80;&#117;&#98;&#115;&#64;&#110;&#103;&#115;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The biological role of the mitochondrial DNA (mtDNA) control region in mtDNA replication remains unclear. In a worldwide survey of mtDNA variation in the general population, we have identified a novel large control region deletion spanning positions 16154 to 16307 (m.16154_16307del154). The population prevalence of this deletion is low, since it was only observed in 1 out of over 120,000 mtDNA genomes studied. The deletion is present in a nonheteroplasmic state, and was transmitted by a mother to her two sons with no apparent past or present disease conditions. The identification of this large deletion in healthy individuals challenges the current view of the control region as playing a crucial role in the regulation of mtDNA replication, and supports the existence of a more complex system of multiple or epigenetically-determined replication origins.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2008</span>;<span class="volume">29</span>;<span class="number">12</span>;<span class="pagerange">1387-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18629826">18629826</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2697596">2697596</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.20835">10.1002/humu.20835</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The dawn of human matrilineal diversity.</h4>
<p class="authors"><span class="author">Behar DM</span>, <span class="author">Villems R</span>, <span class="author">Soodyall H</span>, <span class="author">Blue-Smith J</span>, <span class="author">Pereira L</span>, <span class="author">Metspalu E</span>, <span class="author">Scozzari R</span>, <span class="author">Makkan H</span>, <span class="author">Tzur S</span>, <span class="author">Comas D</span>, <span class="author">Bertranpetit J</span>, <span class="author">Quintana-Murci L</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Wells RS</span>, <span class="author">Rosset S</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Molecular Medicine Laboratory, Rambam Health Care Campus, Haifa 31096, Israel. <a href="mailto:%62%65%68%61%72%64%6D%40%75%73%65%72%6E%65%74.%63%6F%6D">&#98;&#101;&#104;&#97;&#114;&#100;&#109;&#64;&#117;&#115;&#101;&#114;&#110;&#101;&#116;&#46;&#99;&#111;&#109;</a></p><p class="abstract">The quest to explain demographic history during the early part of human evolution has been limited because of the scarce paleoanthropological record from the Middle Stone Age. To shed light on the structure of the mitochondrial DNA (mtDNA) phylogeny at the dawn of Homo sapiens, we constructed a matrilineal tree composed of 624 complete mtDNA genomes from sub-Saharan Hg L lineages. We paid particular attention to the Khoi and San (Khoisan) people of South Africa because they are considered to be a unique relic of hunter-gatherer lifestyle and to carry paternal and maternal lineages belonging to the deepest clades known among modern humans. Both the tree phylogeny and coalescence calculations suggest that Khoisan matrilineal ancestry diverged from the rest of the human mtDNA pool 90,000-150,000 years before present (ybp) and that at least five additional, currently extant maternal lineages existed during this period in parallel. Furthermore, we estimate that a minimum of 40 other evolutionarily successful lineages flourished in sub-Saharan Africa during the period of modern human dispersal out of Africa approximately 60,000-70,000 ybp. Only much later, at the beginning of the Late Stone Age, about 40,000 ybp, did introgression of additional lineages occur into the Khoisan mtDNA pool. This process was further accelerated during the recent Bantu expansions. Our results suggest that the early settlement of humans in Africa was already matrilineally structured and involved small, separately evolving isolated populations.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">5</span>;<span class="pagerange">1130-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18439549">18439549</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427203">2427203</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.04.002">10.1016/j.ajhg.2008.04.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic &#39;valleys of death&#39;.</h4>
<p class="authors"><span class="author">Bentley S</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">260-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18340971">18340971</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1881">10.1038/nrmicro1881</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome of the actinomycete plant pathogen Clavibacter michiganensis subsp. sepedonicus suggests recent niche adaptation.</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Corton C</span>, <span class="author">Brown SE</span>, <span class="author">Barron A</span>, <span class="author">Clark L</span>, <span class="author">Doggett J</span>, <span class="author">Harris B</span>, <span class="author">Ormond D</span>, <span class="author">Quail MA</span>, <span class="author">May G</span>, <span class="author">Francis D</span>, <span class="author">Knudson D</span>, <span class="author">Parkhill J</span> and <span class="author">Ishimaru CA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, United Kingdom.</p><p class="abstract">Clavibacter michiganensis subsp. sepedonicus is a plant-pathogenic bacterium and the causative agent of bacterial ring rot, a devastating agricultural disease under strict quarantine control and zero tolerance in the seed potato industry. This organism appears to be largely restricted to an endophytic lifestyle, proliferating within plant tissues and unable to persist in the absence of plant material. Analysis of the genome sequence of C. michiganensis subsp. sepedonicus and comparison with the genome sequences of related plant pathogens revealed a dramatic recent evolutionary history. The genome contains 106 insertion sequence elements, which appear to have been active in extensive rearrangement of the chromosome compared to that of Clavibacter michiganensis subsp. michiganensis. There are 110 pseudogenes with overrepresentation in functions associated with carbohydrate metabolism, transcriptional regulation, and pathogenicity. Genome comparisons also indicated that there is substantial gene content diversity within the species, probably due to differential gene acquisition and loss. These genomic features and evolutionary dating suggest that there was recent adaptation for life in a restricted niche where nutrient diversity and perhaps competition are low, correlated with a reduced ability to exploit previously occupied complex niches outside the plant. Toleration of factors such as multiplication and integration of insertion sequence elements, genome rearrangements, and functional disruption of many genes and operons seems to indicate that there has been general relaxation of selective pressure on a large proportion of the genome.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">6</span>;<span class="pagerange">2150-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18192393">18192393</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258862">2258862</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01598-07">10.1128/JB.01598-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Susceptibility loci for intracranial aneurysm in European and Japanese populations.</h4>
<p class="authors"><span class="author">Bilguvar K</span>, <span class="author">Yasuno K</span>, <span class="author">Niemel&auml; M</span>, <span class="author">Ruigrok YM</span>, <span class="author">von Und Zu Fraunberg M</span>, <span class="author">van Duijn CM</span>, <span class="author">van den Berg LH</span>, <span class="author">Mane S</span>, <span class="author">Mason CE</span>, <span class="author">Choi M</span>, <span class="author">Ga&aacute;l E</span>, <span class="author">Bayri Y</span>, <span class="author">Kolb L</span>, <span class="author">Arlier Z</span>, <span class="author">Ravuri S</span>, <span class="author">Ronkainen A</span>, <span class="author">Tajima A</span>, <span class="author">Laakso A</span>, <span class="author">Hata A</span>, <span class="author">Kasuya H</span>, <span class="author">Koivisto T</span>, <span class="author">Rinne J</span>, <span class="author">Ohman J</span>, <span class="author">Breteler MM</span>, <span class="author">Wijmenga C</span>, <span class="author">State MW</span>, <span class="author">Rinkel GJ</span>, <span class="author">Hernesniemi J</span>, <span class="author">J&auml;&auml;skel&auml;inen JE</span>, <span class="author">Palotie A</span>, <span class="author">Inoue I</span>, <span class="author">Lifton RP</span> and <span class="author">G&uuml;nel M</span></p>
<p class="affiliation">Department of Neurosurgery, Neurobiology, Yale Center for Human Genetics and Genomics, Yale University School of Medicine, New Haven, CT 06510, USA.</p><p class="abstract">Stroke is the world&#39;s third leading cause of death. One cause of stroke, intracranial aneurysm, affects approximately 2% of the population and accounts for 500,000 hemorrhagic strokes annually in mid-life (median age 50), most often resulting in death or severe neurological impairment. The pathogenesis of intracranial aneurysm is unknown, and because catastrophic hemorrhage is commonly the first sign of disease, early identification is essential. We carried out a multistage genome-wide association study (GWAS) of Finnish, Dutch and Japanese cohorts including over 2,100 intracranial aneurysm cases and 8,000 controls. Genome-wide genotyping of the European cohorts and replication studies in the Japanese cohort identified common SNPs on chromosomes 2q, 8q and 9p that show significant association with intracranial aneurysm with odds ratios 1.24-1.36. The loci on 2q and 8q are new, whereas the 9p locus was previously found to be associated with arterial diseases, including intracranial aneurysm. Associated SNPs on 8q likely act via SOX17, which is required for formation and maintenance of endothelial cells, suggesting a role in development and repair of the vasculature; CDKN2A at 9p may have a similar role. These findings have implications for the pathophysiology, diagnosis and therapy of intracranial aneurysm.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NINDS NIH HHS: <span class="grant">R01 NS057756</span>, <span class="grant">U24 NS051869</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">12</span>;<span class="pagerange">1472-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18997786">18997786</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682433">2682433</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.240">10.1038/ng.240</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction of Salmonella enterica serovar Typhi with cultured epithelial cells: roles of surface structures in adhesion and invasion.</h4>
<p class="authors"><span class="author">Bishop A</span>, <span class="author">House D</span>, <span class="author">Perkins T</span>, <span class="author">Baker S</span>, <span class="author">Kingsley RA</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Centre for Molecular Microbiology and Infection, Faculty of Life Sciences, Division of Molecular and Cell Biology, Imperial College London, London SW7 2AZ, UK.</p><p class="abstract">In this study we investigate the ability of Salmonella enterica serovar Typhi (S. Typhi) surface structures to influence invasion and adhesion in epithelial cell assay systems. In general, S. Typhi was found to be less adherent, invasive and cytotoxic than S. enterica serovar Typhimurium (S. Typhimurium). Culture conditions had little effect on adhesion of S. Typhi to cultured cells but had a marked influence on invasion. In contrast, bacterial growth conditions did not influence S. Typhi apical invasion of polarized cells. The levels of S. Typhi, but not S. Typhimurium, invasion were increased by application of bacteria to the basolateral surface of polarized cells. Expression of virulence (Vi) capsule by S. Typhi resulted in a modest reduction in adhesion, but profoundly reduced levels of invasion of non-polarized cells. However, Vi capsule expression had no affect on invasion of the apical or basolateral surfaces of polarized cells. Mutation of the staA, tcfA or pilS genes did not affect invasion or adhesion in either the presence or the absence of Vi capsule.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2008</span>;<span class="volume">154</span>;<span class="number">Pt 7</span>;<span class="pagerange">1914-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18599820" rel="external">18599820</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652038" rel="external">2652038</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.2008%2F016998-0" rel="external">10.1099/mic.0.2008/016998-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rebound depolarization in single units of the ventral cochlear nucleus: a contribution to grouping by common onset?</h4>
<p class="authors"><span class="author">Bleeck S</span>, <span class="author">Ingham NJ</span>, <span class="author">Verhey JL</span> and <span class="author">Winter IM</span></p>
<p class="affiliation">Institute of Sound and Vibration Research, University of Southampton, University Road, Southampton, SO17 1 BJ, UK.</p><p class="abstract">Simultaneous grouping by common onset time is believed to be a powerful cue in auditory perception; components that start or stop roughly at the same time are judged as far more likely to have originated from the same source. Here we report a simple experiment designed to simulate a complex psychophysical paradigm first described by Darwin and Sutherland [(1984) Grouping frequency components of vowels. When is a harmonic not a harmonic? Quarterly J of Experimental Psychology: Hum Exp Psychol 36(A):193-208]. It is possible to change the perception of the vowel /I/ to /epsilon/ by manipulating the harmonics around the first formant (F1). Increasing the amplitude of one harmonic around F1 caused the perception of the vowel to change from /I/ to /epsilon/. Extending the increased component before the vowel could, however, greatly reduce this change. The role of neural adaptation in this effect was questioned by repeating the experiment but this time using a &#39;captor&#39; tone which was switched on with the asynchronous harmonic and off when the vowel started. This time the vowel percept did change in a fashion analogous to the effect of an increase in the amplitude of the fourth harmonic (which is close to F1). This effect was explained by assuming that the captor had grouped with the leading portion of the asynchronous component enabling the remainder of the asynchronous component to be grouped with the remainder of the components. We propose a relatively low-level neuronal explanation for this grouping effect: the captor reduces the neural response to the leading segment of the asynchronous component by activating across-frequency suppression, either from the cochlea, or acting via a wideband inhibitor in the ventral cochlear nucleus. The reduction in neural response results in a release from adaptation with the offset of the captor terminating the inhibition, such that the response to the continuation of that component is now enhanced. Using a simplified paradigm we show that both primary-like and chopper units in the ventral cochlear nucleus of the anesthetized guinea pig may show a rebound in excitation when a captor is positioned so as to stimulate the suppressive sidebands in its receptive field. The strength of the rebound was positively correlated with the strength of the suppression. These and other results are consistent with the view that low-level mechanisms underlie the psychophysical captor effect.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Neuroscience</span>&nbsp;<span class="year">2008</span>;<span class="volume">154</span>;<span class="number">1</span>;<span class="pagerange">139-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18479835" rel="external">18479835</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neuroscience.2008.03.020" rel="external">10.1016/j.neuroscience.2008.03.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Widespread duplications in the genomes of laboratory stocks of Dictyostelium discoideum.</h4>
<p class="authors"><span class="author">Bloomfield G</span>, <span class="author">Tanaka Y</span>, <span class="author">Skelton J</span>, <span class="author">Ivens A</span> and <span class="author">Kay RR</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge CB2 0QH, UK. <a href="mailto:%67%61%72%65%74%68%62%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#103;&#97;&#114;&#101;&#116;&#104;&#98;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Duplications of stretches of the genome are an important source of individual genetic variation, but their unrecognized presence in laboratory organisms would be a confounding variable for genetic analysis.</p><p class="abstract"><strong>Results</strong>: We report here that duplications of 15 kb or more are common in the genome of the social amoeba Dictyostelium discoideum. Most stocks of the axenic &#39;workhorse&#39; strains Ax2 and Ax3/4 obtained from different laboratories can be expected to carry different duplications. The auxotrophic strains DH1 and JH10 also bear previously unreported duplications. Strain Ax3/4 is known to carry a large duplication on chromosome 2 and this structure shows evidence of continuing instability; we find a further variable duplication on chromosome 5. These duplications are lacking in Ax2, which has instead a small duplication on chromosome 1. Stocks of the type isolate NC4 are similarly variable, though we have identified some approximating the assumed ancestral genotype. More recent wild-type isolates are almost without large duplications, but we can identify small deletions or regions of high divergence, possibly reflecting responses to local selective pressures. Duplications are scattered through most of the genome, and can be stable enough to reconstruct genealogies spanning decades of the history of the NC4 lineage. The expression level of many duplicated genes is increased with dosage, but for others it appears that some form of dosage compensation occurs.</p><p class="abstract"><strong>Conclusion</strong>: The genetic variation described here must underlie some of the phenotypic variation observed between strains from different laboratories. We suggest courses of action to alleviate the problem.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">06724</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">4</span>;<span class="pagerange">R75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18430225" rel="external">18430225</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2643946" rel="external">2643946</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-4-r75" rel="external">10.1186/gb-2008-9-4-r75</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Contributions of the genome sequence of Erwinia amylovora to the fire blight community</h4>
<p class="authors"><span class="author">Bocsanczy</span>, <span class="author">A. M.</span>, <span class="author">Beer</span>, <span class="author">S. V.</span>, <span class="author">Perna</span>, <span class="author">N. T.</span>, <span class="author">Biehl</span>, <span class="author">B.</span>, <span class="author">Glasner</span>, <span class="author">J. D.</span>, <span class="author">Cartinhour</span>, <span class="author">S. W.</span>, <span class="author">Schneider</span>, <span class="author">D. J.</span>, <span class="author">DeClerck</span>, <span class="author">G. A.</span>, <span class="author">Sebaihia</span>, <span class="author">M.</span>, <span class="author">Parkhill</span>, <span class="author">J.</span>, <span class="author">Bentley</span>, <span class="author">S.</span></p>
<p class="publication"><span class="title">Acta Horticulturae.</span> <span class="year">2008</span>;<span class="volume">793</span>;<span class="pagerange">163-170</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Serine/threonine kinase Stk33 exhibits autophosphorylation and phosphorylates the intermediate filament protein Vimentin.</h4>
<p class="authors"><span class="author">Brauksiepe B</span>, <span class="author">Mujica AO</span>, <span class="author">Herrmann H</span> and <span class="author">Schmidt ER</span></p>
<p class="affiliation">Institute of Molecular Genetics, Johannes Gutenberg-University, Mainz, Germany. <a href="mailto:%62%72%61%75%6B%73%69%62%40%75%6E%69-%6D%61%69%6E%7A.%64%65">&#98;&#114;&#97;&#117;&#107;&#115;&#105;&#98;&#64;&#117;&#110;&#105;&#45;&#109;&#97;&#105;&#110;&#122;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: Colocalization of Stk33 with vimentin by double immunofluorescence in certain cells indicated that vimentin might be a target for phosphorylation by the novel kinase Stk33. We therefore tested in vitro the ability of Stk33 to phosphorylate recombinant full length vimentin and amino-terminal truncated versions thereof. In order to prove that Stk33 and vimentin are also in vivo associated proteins co-immunoprecipitation experiments were carried out. For testing the enzymatic activity of immunoprecipitated Stk33 we incubated precipitated Stk33 with recombinant vimentin proteins. To investigate whether Stk33 binds directly to vimentin, an in vitro co-sedimentation assay was performed.</p><p class="abstract"><strong>Results</strong>: The results of the kinase assays demonstrate that Stk33 is able to specifically phosphorylate the non-alpha-helical amino-terminal domain of vimentin in vitro. Furthermore, co-immunoprecipitation experiments employing cultured cell extracts indicate that Stk33 and vimentin are associated in vivo. Immunoprecipitated Stk33 has enzymatic activity as shown by successful phosphorylation of recombinant vimentin proteins. The results of the co-sedimentation assay suggest that vimentin binds directly to Stk33 and that no additional protein mediates the association.</p><p class="abstract"><strong>Conclusion</strong>: We hypothesize that Stk33 is involved in the in vivo dynamics of the intermediate filament cytoskeleton by phosphorylating vimentin.</p>
<p class="publication"><span class="title">BMC biochemistry</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18811945" rel="external">18811945</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2567967" rel="external">2567967</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2091-9-25" rel="external">10.1186/1471-2091-9-25</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The BioGRID Interaction Database: 2008 update.</h4>
<p class="authors"><span class="author">Breitkreutz BJ</span>, <span class="author">Stark C</span>, <span class="author">Reguly T</span>, <span class="author">Boucher L</span>, <span class="author">Breitkreutz A</span>, <span class="author">Livstone M</span>, <span class="author">Oughtred R</span>, <span class="author">Lackner DH</span>, <span class="author">B&auml;hler J</span>, <span class="author">Wood V</span>, <span class="author">Dolinski K</span> and <span class="author">Tyers M</span></p>
<p class="affiliation">Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Canada.</p><p class="abstract">The Biological General Repository for Interaction Datasets (BioGRID) database (http://www.thebiogrid.org) was developed to house and distribute collections of protein and genetic interactions from major model organism species. BioGRID currently contains over 198 000 interactions from six different species, as derived from both high-throughput studies and conventional focused studies. Through comprehensive curation efforts, BioGRID now includes a virtually complete set of interactions reported to date in the primary literature for both the budding yeast Saccharomyces cerevisiae and the fission yeast Schizosaccharomyces pombe. A number of new features have been added to the BioGRID including an improved user interface to display interactions based on different attributes, a mirror site and a dedicated interaction management system to coordinate curation across different locations. The BioGRID provides interaction data with monthly updates to Saccharomyces Genome Database, Flybase and Entrez Gene. Source code for the BioGRID and the linked Osprey network visualization system is now freely available without restriction.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; NCRR NIH HHS: <span class="grant">1R01RR024031-01</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D637-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18000002" rel="external">18000002</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238873" rel="external">2238873</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm1001" rel="external">10.1093/nar/gkm1001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genomics: modeling the new and a renaissance of the old.</h4>
<p class="authors"><span class="author">Brinkman FS</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Current opinion in microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">11</span>;<span class="number">5</span>;<span class="pagerange">439-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18824130" rel="external">18824130</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mib.2008.09.001" rel="external">10.1016/j.mib.2008.09.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of Mascot and X!Tandem performance for low and high accuracy mass spectrometry and the development of an adjusted Mascot threshold.</h4>
<p class="authors"><span class="author">Brosch M</span>, <span class="author">Swamy S</span>, <span class="author">Hubbard T</span> and <span class="author">Choudhary J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">It is a major challenge to develop effective sequence database search algorithms to translate molecular weight and fragment mass information obtained from tandem mass spectrometry into high quality peptide and protein assignments. We investigated the peptide identification performance of Mascot and X!Tandem for mass tolerance settings common for low and high accuracy mass spectrometry. We demonstrated that sensitivity and specificity of peptide identification can vary substantially for different mass tolerance settings, but this effect was more significant for Mascot. We present an adjusted Mascot threshold, which allows the user to freely select the best trade-off between sensitivity and specificity. The adjusted Mascot threshold was compared with the default Mascot and X!Tandem scoring thresholds and shown to be more sensitive at the same false discovery rates for both low and high accuracy mass spectrometry data.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2008</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">962-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18216375">18216375</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656932">2656932</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M700293-MCP200">10.1074/mcp.M700293-MCP200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replacement of adenylate cyclase toxin in a lineage of Bordetella bronchiseptica.</h4>
<p class="authors"><span class="author">Buboltz AM</span>, <span class="author">Nicholson TL</span>, <span class="author">Parette MR</span>, <span class="author">Hester SE</span>, <span class="author">Parkhill J</span> and <span class="author">Harvill ET</span></p>
<p class="affiliation">Department of Veterinary and Biomedical Sciences, The Pennsylvania State University, University Park, Pennsylvania 16802, USA.</p><p class="abstract">Bordetella bronchiseptica is a gram-negative respiratory pathogen that infects a wide range of hosts and causes a diverse spectrum of disease. This diversity is likely affected by multiple factors, such as host immune status, polymicrobial infection, and strain diversity. In a murine model of infection, we found that the virulence of B. bronchiseptica strains, as measured by the mean lethal dose, varied widely. Strain 253 was less virulent than the typically studied strain, RB50. Transcriptome analysis showed that cyaA, the gene encoding adenylate cyclase toxin (CyaA), was the most downregulated transcript identified in strain 253 compared to that in strain RB50. Comparative genomic hybridization and genome sequencing of strain 253 revealed that the cya locus, which encodes, activates, and secretes CyaA, was replaced by an operon (ptp) predicted to encode peptide transport proteins. Other B. bronchiseptica strains from the same phylogenetic lineage as that of strain 253 also lacked the cya locus, contained the ptp genes, and were less virulent than strain RB50. Although the loss of CyaA would be expected to be counterselected since it is conserved among the classical bordetellae and believed to be important to their success, our data indicate that the loss of this toxin and the gain of the ptp genes occurred in an ancestral strain that then expanded into a lineage. This suggests that there may be ecological niches in which CyaA is not critical for the success of B. bronchiseptica.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI 053075</span>, <span class="grant">AI 065507</span>, <span class="grant">R01 AI053075-01A1</span>, <span class="grant">R01 AI053075-02</span>, <span class="grant">R01 AI053075-03</span>, <span class="grant">R01 AI053075-04</span>, <span class="grant">R01 AI053075-05</span>; NIGMS NIH HHS: <span class="grant">GM083113</span>, <span class="grant">R01 GM083113-01</span>, <span class="grant">R01 GM083113-02</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">15</span>;<span class="pagerange">5502-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18556799" rel="external">18556799</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2493278" rel="external">2493278</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00226-08" rel="external">10.1128/JB.00226-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of variation in the platelet transcriptome associated with glycoprotein 6 haplotype.</h4>
<p class="authors"><span class="author">Burns P</span>, <span class="author">Gusnanto A</span>, <span class="author">Macaulay IC</span>, <span class="author">Rankin A</span>, <span class="author">Tom B</span>, <span class="author">Langford CF</span>, <span class="author">Dudbridge F</span>, <span class="author">Ouwehand WH</span>, <span class="author">Watkins NA</span> and <span class="author">Bloodomics Consortium</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge and National Health Service Blood and Transplant, Cambridge, UK.</p><p class="abstract">Platelet Glycoprotein VI (GPVI) is the activatory collagen signalling receptor that transmits an outside-in signal via the FcR gamma-chain. In Caucasians two GP6 haplotypes have been identified which encode GPVI isoforms that differ by five amino-acids. The minor haplotype is associated with a modest but statistically significant reduction in GPVI abundance and reduced downstream signalling events. As GPVI is also expressed on megakaryocytes, different GPVI isoforms may imprint on the platelet transcriptome. We investigated the association of GP6 haplotype with transcription by comparing the transcriptomes of platelets from individuals homozygous for the major (&#39;a&#39;) and minor (&#39;b&#39;) haplotypes to identify differentially expressed (DE) transcripts. Platelet RNA was isolated from apheresis concentrates from 16 &#39;aa&#39; donors and eight &#39;bb&#39; donors. mRNA was amplified using a template-switching PCR based protocol and fluorescently labelled. Samples were randomly paired both within and between haplotypes and compared on a cDNA microarray. No consistently DE transcripts were identified within the &#39;aa&#39; haplotype but 52 significantly DE transcripts were observed between haplotypes. Generally the fold differences were low (two to four-fold) but were confirmed by qRT-PCR for selected transcripts (TUBB1, P = 0.0004; VWF, P = 0.0126). The results of this study indicate that there are subtle differences between the platelet transcriptomes of individuals who differ by GP6 haplotype. The identification of DE genes may identify critical pathways and nodes not previously known to be involved in platelet development and function.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105261167</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Platelets</span>&nbsp;<span class="year">2008</span>;<span class="volume">19</span>;<span class="number">4</span>;<span class="pagerange">258-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18569861">18569861</a>; DOI: <a href="http://dx.doi.org/10.1080%2F09537100801947434">10.1080/09537100801947434</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale screening for novel low-affinity extracellular protein interactions.</h4>
<p class="authors"><span class="author">Bushell KM</span>, <span class="author">S&ouml;llner C</span>, <span class="author">Schuster-Boeckler B</span>, <span class="author">Bateman A</span> and <span class="author">Wright GJ</span></p>
<p class="affiliation">Cell Surface Signalling Laboratory, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, United Kingdom.</p><p class="abstract">Extracellular protein-protein interactions are essential for both intercellular communication and cohesion within multicellular organisms. Approximately a fifth of human genes encode membrane-tethered or secreted proteins, but they are largely absent from recent large-scale protein interaction datasets, making current interaction networks biased and incomplete. This discrepancy is due to the unsuitability of popular high-throughput methods to detect extracellular interactions because of the biochemical intractability of membrane proteins and their interactions. For example, cell surface proteins contain insoluble hydrophobic transmembrane regions, and their extracellular interactions are often highly transient, having half-lives of less than a second. To detect transient extracellular interactions on a large scale, we developed AVEXIS (avidity-based extracellular interaction screen), a high-throughput assay that overcomes these technical issues and can detect very transient interactions (half-lives &lt;or= 0.1 sec) with a low false-positive rate. We used it to systematically screen for receptor-ligand pairs within the zebrafish immunoglobulin superfamily and identified novel ligands for both well-known and orphan receptors. Genes encoding receptor-ligand pairs were often clustered phylogenetically and expressed in the same or adjacent tissues, immediately implying their involvement in similar biological processes. Using AVEXIS, we have determined the first systematic low-affinity extracellular protein interaction network, supported by independent biological data. This technique will now allow large-scale extracellular protein interaction mapping in a broad range of experimental contexts.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">4</span>;<span class="pagerange">622-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18296487">18296487</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2279249">2279249</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.7187808">10.1101/gr.7187808</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Subclonal phylogenetic structures in cancer revealed by ultra-deep sequencing.</h4>
<p class="authors"><span class="author">Campbell PJ</span>, <span class="author">Pleasance ED</span>, <span class="author">Stephens PJ</span>, <span class="author">Dicks E</span>, <span class="author">Rance R</span>, <span class="author">Goodhead I</span>, <span class="author">Follows GA</span>, <span class="author">Green AR</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1SA, United Kingdom.</p><p class="abstract">During the clonal expansion of cancer from an ancestral cell with an initiating oncogenic mutation to symptomatic neoplasm, the occurrence of somatic mutations (both driver and passenger) can be used to track the on-going evolution of the neoplasm. All subclones within a cancer are phylogenetically related, with the prevalence of each subclone determined by its evolutionary fitness and the timing of its origin relative to other subclones. Recently developed massively parallel sequencing platforms promise the ability to detect rare subclones of genetic variants without a priori knowledge of the mutations involved. We used ultra-deep pyrosequencing to investigate intraclonal diversification at the Ig heavy chain locus in 22 patients with B-cell chronic lymphocytic leukemia. Analysis of a non-polymorphic control locus revealed artifactual insertions and deletions resulting from sequencing errors and base substitutions caused by polymerase misincorporation during PCR amplification. We developed an algorithm to differentiate genuine haplotypes of somatic hypermutations from such artifacts. This proved capable of detecting multiple rare subclones with frequencies as low as 1 in 5000 copies and allowed the characterization of phylogenetic interrelationships among subclones within each patient. This study demonstrates the potential for ultra-deep resequencing to recapitulate the dynamics of clonal evolution in cancer cell populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">088340</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">35</span>;<span class="pagerange">13081-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18723673">18723673</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2529122">2529122</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0801523105">10.1073/pnas.0801523105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of somatically acquired rearrangements in cancer using genome-wide massively parallel paired-end sequencing.</h4>
<p class="authors"><span class="author">Campbell PJ</span>, <span class="author">Stephens PJ</span>, <span class="author">Pleasance ED</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Li H</span>, <span class="author">Santarius T</span>, <span class="author">Stebbings LA</span>, <span class="author">Leroy C</span>, <span class="author">Edkins S</span>, <span class="author">Hardy C</span>, <span class="author">Teague JW</span>, <span class="author">Menzies A</span>, <span class="author">Goodhead I</span>, <span class="author">Turner DJ</span>, <span class="author">Clee CM</span>, <span class="author">Quail MA</span>, <span class="author">Cox A</span>, <span class="author">Brown C</span>, <span class="author">Durbin R</span>, <span class="author">Hurles ME</span>, <span class="author">Edwards PA</span>, <span class="author">Bignell GR</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">Human cancers often carry many somatically acquired genomic rearrangements, some of which may be implicated in cancer development. However, conventional strategies for characterizing rearrangements are laborious and low-throughput and have low sensitivity or poor resolution. We used massively parallel sequencing to generate sequence reads from both ends of short DNA fragments derived from the genomes of two individuals with lung cancer. By investigating read pairs that did not align correctly with respect to each other on the reference human genome, we characterized 306 germline structural variants and 103 somatic rearrangements to the base-pair level of resolution. The patterns of germline and somatic rearrangement were markedly different. Many somatic rearrangements were from amplicons, although rearrangements outside these regions, notably including tandem duplications, were also observed. Some somatic rearrangements led to abnormal transcripts, including two from internal tandem duplications and two fusion transcripts created by interchromosomal rearrangements. Germline variants were predominantly mediated by retrotransposition, often involving AluY and LINE elements. The results demonstrate the feasibility of systematic, genome-wide characterization of rearrangements in complex human cancer genomes, raising the prospect of a new harvest of genes associated with cancer using this strategy.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span>, <span class="grant">088340</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">6</span>;<span class="pagerange">722-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18438408">18438408</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705838">2705838</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.128">10.1038/ng.128</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dictyostelium transcriptional responses to Pseudomonas aeruginosa: common and specific effects from PAO1 and PA14 strains.</h4>
<p class="authors"><span class="author">Carilla-Latorre S</span>, <span class="author">Calvo-Garrido J</span>, <span class="author">Bloomfield G</span>, <span class="author">Skelton J</span>, <span class="author">Kay RR</span>, <span class="author">Ivens A</span>, <span class="author">Martinez JL</span> and <span class="author">Escalante R</span></p>
<p class="affiliation">Instituto de Investigaciones Biom&eacute;dicas Alberto Sols, Universidad Aut&oacute;noma de Madrid-Consejo Superior de Investigaciones Cient&iacute;ficas, Madrid, Spain. <a href="mailto:%73%63%61%72%69%6C%6C%61%40%69%69%62.%75%61%6D.%65%73">&#115;&#99;&#97;&#114;&#105;&#108;&#108;&#97;&#64;&#105;&#105;&#98;&#46;&#117;&#97;&#109;&#46;&#101;&#115;</a></p><p class="abstract"><strong>Background</strong>: Pseudomonas aeruginosa is one of the most relevant human opportunistic bacterial pathogens. Two strains (PAO1 and PA14) have been mainly used as models for studying virulence of P. aeruginosa. The strain PA14 is more virulent than PAO1 in a wide range of hosts including insects, nematodes and plants. Whereas some of the differences might be attributable to concerted action of determinants encoded in pathogenicity islands present in the genome of PA14, a global analysis of the differential host responses to these P. aeruginosa strains has not been addressed. Little is known about the host response to infection with P. aeruginosa and whether or not the global host transcription is being affected as a defense mechanism or altered in the benefit of the pathogen. Since the social amoeba Dictyostelium discoideum is a suitable host to study virulence of P. aeruginosa and other pathogens, we used available genomic tools in this model system to study the transcriptional host response to P. aeruginosa infection.</p><p class="abstract"><strong>Results</strong>: We have compared the virulence of the P. aeruginosa PAO1 and PA14 using D. discoideum and studied the transcriptional response of the amoeba upon infection. Our results showed that PA14 is more virulent in Dictyostelium than PA01using different plating assays. For studying the differential response of the host to infection by these model strains, D. discoideum cells were exposed to either P. aeruginosa PAO1 or P. aeruginosa PA14 (mixed with an excess of the non-pathogenic bacterium Klebsiella aerogenes as food supply) and after 4 hours, cellular RNA extracted. A three-way comparison was made using whole-genome D. discoideum microarrays between RNA samples from cells treated with the two different strains and control cells exposed only to K. aerogenes. The transcriptomic analyses have shown the existence of common and specific responses to infection. The expression of 364 genes changed in a similar way upon infection with one or another strain, whereas 169 genes were differentially regulated depending on whether the infecting strain was either P. aeruginosa PAO1 or PA14. Effects on metabolism, signalling, stress response and cell cycle can be inferred from the genes affected.</p><p class="abstract"><strong>Conclusion</strong>: Our results show that pathogenic Pseudomonas strains invoke both a common transcriptional response from Dictyostelium and a strain specific one, indicating that the infective process of bacterial pathogens can be strain-specific and is more complex than previously thought.</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">8</span>;<span class="pagerange">109</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18590548" rel="external">18590548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2474670" rel="external">2474670</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-8-109" rel="external">10.1186/1471-2180-8-109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Grandest Genetics Experiment Ever Performed on Man? A Y-Chromosomal Perspective on Genetic Variation in India</h4>
<p class="authors"><span class="author">Carvalho-Silva D R</span>, <span class="author">Tyler-Smith</span>, <span class="author">C</span></p>
<p class="publication"><span class="title">International Journal of Human Genetics.</span> <span class="year">2008</span>;<span class="volume">8</span>;<span class="pagerange">21-9</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Artemis and ACT: viewing, annotating and comparing sequences stored in a relational database.</h4>
<p class="authors"><span class="author">Carver T</span>, <span class="author">Berriman M</span>, <span class="author">Tivey A</span>, <span class="author">Patel C</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Barrell BG</span>, <span class="author">Parkhill J</span> and <span class="author">Rajandream MA</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Motivation</strong>: Artemis and Artemis Comparison Tool (ACT) have become mainstream tools for viewing and annotating sequence data, particularly for microbial genomes. Since its first release, Artemis has been continuously developed and supported with additional functionality for editing and analysing sequences based on feedback from an active user community of laboratory biologists and professional annotators. Nevertheless, its utility has been somewhat restricted by its limitation to reading and writing from flat files. Therefore, a new version of Artemis has been developed, which reads from and writes to a relational database schema, and allows users to annotate more complex, often large and fragmented, genome sequences.</p><p class="abstract"><strong>Results</strong>: Artemis and ACT have now been extended to read and write directly to the Generic Model Organism Database (GMOD, http://www.gmod.org) Chado relational database schema. In addition, a Gene Builder tool has been developed to provide structured forms and tables to edit coordinates of gene models and edit functional annotation, based on standard ontologies, controlled vocabularies and free text.</p><p class="abstract"><strong>Availability</strong>: Artemis and ACT are freely available (under a GPL licence) for download (for MacOSX, UNIX and Windows) at the Wellcome Trust Sanger Institute web sites: http://www.sanger.ac.uk/Software/Artemis/ http://www.sanger.ac.uk/Software/ACT/</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">082372</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">23</span>;<span class="pagerange">2672-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18845581">18845581</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2606163">2606163</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtn529">10.1093/bioinformatics/btn529</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel mechanistic spectrum underlies glaucoma-associated chromosome 6p25 copy number variation.</h4>
<p class="authors"><span class="author">Chanda B</span>, <span class="author">Asai-Coakwell M</span>, <span class="author">Ye M</span>, <span class="author">Mungall AJ</span>, <span class="author">Barrow M</span>, <span class="author">Dobyns WB</span>, <span class="author">Behesti H</span>, <span class="author">Sowden JC</span>, <span class="author">Carter NP</span>, <span class="author">Walter MA</span> and <span class="author">Lehmann OJ</span></p>
<p class="affiliation">Departments of Ophthalmology, University of Alberta, Edmonton, Alberta, Canada.</p><p class="abstract">The factors that mediate chromosomal rearrangement remain incompletely defined. Among regions prone to structural variant formation, chromosome 6p25 is one of the few in which disease-associated segmental duplications and segmental deletions have been identified, primarily through gene dosage attributable ocular phenotypes. Using array comparative genome hybridization, we studied ten 6p25 duplication and deletion pedigrees and amplified junction fragments from each. Analysis of the breakpoint architecture revealed that all the rearrangements were non-recurrent, and in contrast to most previous examples the majority of the segmental duplications and deletions utilized coupled homologous and non-homologous recombination mechanisms. One junction fragment exhibited an unprecedented 367 bp insert derived from tandemly arranged breakpoint elements. While this accorded with a recently described replication-based mechanism, it differed from the previous example in being unassociated with template switching, and occurring in a segmental deletion. These results extend the mechanisms involved in structural variant formation, provide strong evidence that a spectrum of recombination, DNA repair and replication underlie 6p25 rearrangements, and have implications for genesis of copy number variations in other genomic regions. These findings highlight the benefits of undertaking the extensive studies necessary to characterize structural variants at the base pair level.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">22</span>;<span class="pagerange">3446-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18694899" rel="external">18694899</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2572693" rel="external">2572693</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddn238" rel="external">10.1093/hmg/ddn238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multiple pathways differentially regulate global oxidative stress responses in fission yeast.</h4>
<p class="authors"><span class="author">Chen D</span>, <span class="author">Wilkinson CR</span>, <span class="author">Watt S</span>, <span class="author">Penkett CJ</span>, <span class="author">Toone WM</span>, <span class="author">Jones N</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, United Kingdom.</p><p class="abstract">Cellular protection against oxidative damage is relevant to ageing and numerous diseases. We analyzed the diversity of genome-wide gene expression programs and their regulation in response to various types and doses of oxidants in Schizosaccharomyces pombe. A small core gene set, regulated by the AP-1-like factor Pap1p and the two-component regulator Prr1p, was universally induced irrespective of oxidant and dose. Strong oxidative stresses led to a much larger transcriptional response. The mitogen-activated protein kinase (MAPK) Sty1p and the bZIP factor Atf1p were critical for the response to hydrogen peroxide. A newly identified zinc-finger protein, Hsr1p, is uniquely regulated by all three major regulatory systems (Sty1p-Atf1p, Pap1p, and Prr1p) and in turn globally supports gene expression in response to hydrogen peroxide. Although the overall transcriptional responses to hydrogen peroxide and t-butylhydroperoxide were similar, to our surprise, Sty1p and Atf1p were less critical for the response to the latter. Instead, another MAPK, Pmk1p, was involved in surviving this stress, although Pmk1p played only a minor role in regulating the transcriptional response. These data reveal a considerable plasticity and differential control of regulatory pathways in distinct oxidative stress conditions, providing both specificity and backup for protection from oxidative damage.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular biology of the cell</span>&nbsp;<span class="year">2008</span>;<span class="volume">19</span>;<span class="number">1</span>;<span class="pagerange">308-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18003976" rel="external">18003976</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2174203" rel="external">2174203</a>; DOI: <a href="http://dx.doi.org/10.1091%2Fmbc.E07-08-0735" rel="external">10.1091/mbc.E07-08-0735</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic analysis of human traits in vitro: drug response and gene expression in lymphoblastoid cell lines.</h4>
<p class="authors"><span class="author">Choy E</span>, <span class="author">Yelensky R</span>, <span class="author">Bonakdar S</span>, <span class="author">Plenge RM</span>, <span class="author">Saxena R</span>, <span class="author">De Jager PL</span>, <span class="author">Shaw SY</span>, <span class="author">Wolfish CS</span>, <span class="author">Slavik JM</span>, <span class="author">Cotsapas C</span>, <span class="author">Rivas M</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Cahir-McFarland E</span>, <span class="author">Kieff E</span>, <span class="author">Hafler D</span>, <span class="author">Daly MJ</span> and <span class="author">Altshuler D</span></p>
<p class="affiliation">Broad Institute of MIT and Harvard, Cambridge, Massachusetts, United States of America.</p><p class="abstract">Lymphoblastoid cell lines (LCLs), originally collected as renewable sources of DNA, are now being used as a model system to study genotype-phenotype relationships in human cells, including searches for QTLs influencing levels of individual mRNAs and responses to drugs and radiation. In the course of attempting to map genes for drug response using 269 LCLs from the International HapMap Project, we evaluated the extent to which biological noise and non-genetic confounders contribute to trait variability in LCLs. While drug responses could be technically well measured on a given day, we observed significant day-to-day variability and substantial correlation to non-genetic confounders, such as baseline growth rates and metabolic state in culture. After correcting for these confounders, we were unable to detect any QTLs with genome-wide significance for drug response. A much higher proportion of variance in mRNA levels may be attributed to non-genetic factors (intra-individual variance--i.e., biological noise, levels of the EBV virus used to transform the cells, ATP levels) than to detectable eQTLs. Finally, in an attempt to improve power, we focused analysis on those genes that had both detectable eQTLs and correlation to drug response; we were unable to detect evidence that eQTL SNPs are convincingly associated with drug response in the model. While LCLs are a promising model for pharmacogenetic experiments, biological noise and in vitro artifacts may reduce power and have the potential to create spurious association due to confounding.</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">11</span>;<span class="pagerange">e1000287</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19043577" rel="external">19043577</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2583954" rel="external">2583954</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000287" rel="external">10.1371/journal.pgen.1000287</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Kinase networks integrate profiles of N-methyl-D-aspartate receptor-mediated gene expression in hippocampus.</h4>
<p class="authors"><span class="author">Coba MP</span>, <span class="author">Valor LM</span>, <span class="author">Kopanitsa MV</span>, <span class="author">Afinowi NO</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The postsynaptic N-methyl-d-aspartate (NMDA) receptor activates multiple kinases and changes the phosphorylation of many postsynaptic proteins organized in signaling networks. Because the NMDA receptor is known to regulate gene expression, it is important to examine whether networks of kinases control signaling to gene expression. We examined the requirement of multiple kinases and NMDA receptor-interacting proteins for gene expression in mouse hippocampal slices. Protocols that induce long-term depression (LTD) and long-term potentiation (LTP) activated common kinases and overlapping gene expression profiles. Combinations of kinases were required for induction of each gene. Distinct combinations of kinases were required to up-regulate Arc, Npas4, Egr2, and Egr4 following either LTP or LTD protocols. Consistent with the combinatorial data, a mouse mutant model of the human cognition disease gene SAP102, which couples ERK kinase to the NMDA receptor, showed deregulated expression of specific genes. These data support a network model of postsynaptic integration where kinase signaling networks are recruited by differential synaptic activity and control both local synaptic events and activity-dependent gene expression.</p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2008</span>;<span class="volume">283</span>;<span class="number">49</span>;<span class="pagerange">34101-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18815127" rel="external">18815127</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2662229" rel="external">2662229</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M804951200" rel="external">10.1074/jbc.M804951200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Priorities for nucleotide trace, sequence and annotation data capture at the Ensembl Trace Archive and the EMBL Nucleotide Sequence Database.</h4>
<p class="authors"><span class="author">Cochrane G</span>, <span class="author">Akhtar R</span>, <span class="author">Aldebert P</span>, <span class="author">Althorpe N</span>, <span class="author">Baldwin A</span>, <span class="author">Bates K</span>, <span class="author">Bhattacharyya S</span>, <span class="author">Bonfield J</span>, <span class="author">Bower L</span>, <span class="author">Browne P</span>, <span class="author">Castro M</span>, <span class="author">Cox T</span>, <span class="author">Demiralp F</span>, <span class="author">Eberhardt R</span>, <span class="author">Faruque N</span>, <span class="author">Hoad G</span>, <span class="author">Jang M</span>, <span class="author">Kulikova T</span>, <span class="author">Labarga A</span>, <span class="author">Leinonen R</span>, <span class="author">Leonard S</span>, <span class="author">Lin Q</span>, <span class="author">Lopez R</span>, <span class="author">Lorenc D</span>, <span class="author">McWilliam H</span>, <span class="author">Mukherjee G</span>, <span class="author">Nardone F</span>, <span class="author">Plaister S</span>, <span class="author">Robinson S</span>, <span class="author">Sobhany S</span>, <span class="author">Vaughan R</span>, <span class="author">Wu D</span>, <span class="author">Zhu W</span>, <span class="author">Apweiler R</span>, <span class="author">Hubbard T</span> and <span class="author">Birney E</span></p>
<p class="affiliation">EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%63%6F%63%68%72%61%6E%65%40%65%62%69.%61%63.%75%6B">&#99;&#111;&#99;&#104;&#114;&#97;&#110;&#101;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl Trace Archive (http://trace.ensembl.org/) and the EMBL Nucleotide Sequence Database (http://www.ebi.ac.uk/embl/), known together as the European Nucleotide Archive, continue to see growth in data volume and diversity. Selected major developments of 2007 are presented briefly, along with data submission and retrieval information. In the face of increasing requirements for nucleotide trace, sequence and annotation data archiving, data capture priority decisions have been taken at the European Nucleotide Archive. Priorities are discussed in terms of how reliably information can be captured, the long-term benefits of its capture and the ease with which it can be captured.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span>, <span class="grant">085532</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D5-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18039715">18039715</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238915">2238915</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm1018">10.1093/nar/gkm1018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identifying protein domains with the Pfam database.</h4>
<p class="authors"><span class="author">Coggill P</span>, <span class="author">Finn RD</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Pfam is a database of protein domain families, with each family represented by multiple sequence alignments and profile hidden Markov models (HMMs). In addition, each family has associated annotation, literature references, and links to other databases. The entries in Pfam are available via the World Wide Web and in flatfile format. This unit contains detailed information on how to access and utilize the information present in the Pfam database, namely the families, multiple alignments, and annotation. Details on running Pfam, both remotely and locally are presented.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Current protocols in bioinformatics / editoral board, Andreas D. Baxevanis ... [et al.]</span>&nbsp;<span class="year">2008</span>;<span class="volume">Chapter 2</span>;<span class="pagerange">Unit 2.5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18819075">18819075</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471250953.bi0205s23">10.1002/0471250953.bi0205s23</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">nGASP--the nematode genome annotation assessment project.</h4>
<p class="authors"><span class="author">Coghlan A</span>, <span class="author">Fiedler TJ</span>, <span class="author">McKay SJ</span>, <span class="author">Flicek P</span>, <span class="author">Harris TW</span>, <span class="author">Blasiar D</span>, <span class="author">nGASP Consortium</span> and <span class="author">Stein LD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%6C%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: While the C. elegans genome is extensively annotated, relatively little information is available for other Caenorhabditis species. The nematode genome annotation assessment project (nGASP) was launched to objectively assess the accuracy of protein-coding gene prediction software in C. elegans, and to apply this knowledge to the annotation of the genomes of four additional Caenorhabditis species and other nematodes. Seventeen groups worldwide participated in nGASP, and submitted 47 prediction sets across 10 Mb of the C. elegans genome. Predictions were compared to reference gene sets consisting of confirmed or manually curated gene models from WormBase.</p><p class="abstract"><strong>Results</strong>: The most accurate gene-finders were &#39;combiner&#39; algorithms, which made use of transcript- and protein-alignments and multi-genome alignments, as well as gene predictions from other gene-finders. Gene-finders that used alignments of ESTs, mRNAs and proteins came in second. There was a tie for third place between gene-finders that used multi-genome alignments and ab initio gene-finders. The median gene level sensitivity of combiners was 78% and their specificity was 42%, which is nearly the same accuracy reported for combiners in the human genome. C. elegans genes with exons of unusual hexamer content, as well as those with unusually many exons, short exons, long introns, a weak translation start signal, weak splice sites, or poorly conserved orthologs posed the greatest difficulty for gene-finders.</p><p class="abstract"><strong>Conclusion</strong>: This experiment establishes a baseline of gene prediction accuracy in Caenorhabditis genomes, and has guided the choice of gene-finders for the annotation of newly sequenced genomes of Caenorhabditis and other nematode species. We have created new gene sets for C. briggsae, C. remanei, C. brenneri, C. japonica, and Brugia malayi using some of the best-performing gene-finders.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41 HG02223</span>; Wellcome Trust</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">549</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19099578">19099578</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2651883">2651883</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-9-549">10.1186/1471-2105-9-549</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Finishing the finished human chromosome 22 sequence.</h4>
<p class="authors"><span class="author">Cole CG</span>, <span class="author">McCann OT</span>, <span class="author">Collins JE</span>, <span class="author">Oliver K</span>, <span class="author">Willey D</span>, <span class="author">Gribble SM</span>, <span class="author">Yang F</span>, <span class="author">McLaren K</span>, <span class="author">Rogers J</span>, <span class="author">Ning Z</span>, <span class="author">Beare DM</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%63%67%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#103;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Although the human genome sequence was declared complete in 2004, the sequence was interrupted by 341 gaps of which 308 lay in an estimated approximately 28 Mb of euchromatin. While these gaps constitute only approximately 1% of the sequence, knowledge of the full complement of human genes and regulatory elements is incomplete without their sequences.</p><p class="abstract"><strong>Results</strong>: We have used a combination of conventional chromosome walking (aided by the availability of end sequences) in fosmid and bacterial artificial chromosome (BAC) libraries, whole chromosome shotgun sequencing, comparative genome analysis and long PCR to finish 8 of the 11 gaps in the initial chromosome 22 sequence. In addition, we have patched four regions of the initial sequence where the original clones were found to be deleted, or contained a deletion allele of a known gene, with a further 126 kb of new sequence. Over 1.018 Mb of new sequence has been generated to extend into and close the gaps, and we have annotated 16 new or extended gene structures and one pseudogene.</p><p class="abstract"><strong>Conclusion</strong>: Thus, we have made significant progress to completing the sequence of the euchromatic regions of human chromosome 22 using a combination of detailed approaches. Our experience suggests that substantial work remains to close the outstanding gaps in the human genome sequence.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">R78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18477386" rel="external">18477386</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2441464" rel="external">2441464</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-5-r78" rel="external">10.1186/gb-2008-9-5-r78</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Subcellular localization of intracellular human proteins by construction of tagged fusion proteins and transient expression in COS-7 Cells.</h4>
<p class="authors"><span class="author">Collins JE</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">Identifying the subcellular compartment of a protein is an important step toward assigning protein function. Starting with a clone containing the open reading frame (ORF) of interest, it is possible to attach a variety of short amino acid tags or fluorescent proteins and detect the location of the protein, after transfection into a cell line, using fluorescent microscopy. By collecting data from various expression clone constructs, using a range of cell lines and double labeling with cellular compartment markers, a picture of the localization of a gene can be built up. This chapter describes how to obtain the ORF clone for your gene of interest, clone it into your choice of mammalian expression vector or vectors, transiently transfect for visualization, and where to get started when interpreting the results.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2008</span>;<span class="volume">439</span>;<span class="pagerange">353-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18370115" rel="external">18370115</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-1-59745-188-8_24" rel="external">10.1007/978-1-59745-188-8_24</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping multiprotein complexes by affinity purification and mass spectrometry.</h4>
<p class="authors"><span class="author">Collins MO</span> and <span class="author">Choudhary JS</span></p>
<p class="affiliation">Proteomic Mass Spectrometry, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The combination of affinity purification and tandem mass spectrometry (MS) has emerged as a powerful approach to delineate biological processes. In particular, the use of epitope tags has allowed this approach to become scaleable and has bypassed difficulties associated with generation of antibodies. Single epitope tags and tandem affinity purification (TAP) tags have been used to systematically map protein complexes generating protein interaction data at a near proteome-wide scale. Recent developments in the design of tags, optimisation of purification conditions, experimental design and data analysis have greatly improved the sensitivity and specificity of this approach. Concomitant developments in MS, including high accuracy and high-throughput instrumentation together with quantitative MS methods, have facilitated large-scale and comprehensive analysis of multiprotein complexes.</p>
<p class="publication"><span class="title">Current opinion in biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">19</span>;<span class="number">4</span>;<span class="pagerange">324-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18598764">18598764</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.copbio.2008.06.002">10.1016/j.copbio.2008.06.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phosphoproteomic analysis of the mouse brain cytosol reveals a predominance of protein phosphorylation in regions of intrinsic sequence disorder.</h4>
<p class="authors"><span class="author">Collins MO</span>, <span class="author">Yu L</span>, <span class="author">Campuzano I</span>, <span class="author">Grant SG</span> and <span class="author">Choudhary JS</span></p>
<p class="affiliation">Proteomic Mass Spectrometry, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB101SA, United Kingdom.</p><p class="abstract">We analyzed the mouse forebrain cytosolic phosphoproteome using sequential (protein and peptide) IMAC purifications, enzymatic dephosphorylation, and targeted tandem mass spectrometry analysis strategies. In total, using complementary phosphoenrichment and LC-MS/MS strategies, 512 phosphorylation sites on 540 non-redundant phosphopeptides from 162 cytosolic phosphoproteins were characterized. Analysis of protein domains and amino acid sequence composition of this data set of cytosolic phosphoproteins revealed that it is significantly enriched in intrinsic sequence disorder, and this enrichment is associated with both cellular location and phosphorylation status. The majority of phosphorylation sites found by MS were located outside of structural protein domains (97%) but were mostly located in regions of intrinsic sequence disorder (86%). 368 phosphorylation sites were located in long regions of disorder (over 40 amino acids long), and 94% of proteins contained at least one such long region of disorder. In addition, we found that 58 phosphorylation sites in this data set occur in 14-3-3 binding consensus motifs, linear motifs that are associated with unstructured regions in proteins. These results demonstrate that in this data set protein phosphorylation is significantly depleted in protein domains and significantly enriched in disordered protein sequences and that enrichment of intrinsic sequence disorder may be a common feature of phosphoproteomes. This supports the hypothesis that disordered regions in proteins allow kinases, phosphatases, and phosphorylation-dependent binding proteins to gain access to target sequences to regulate local protein conformation and activity.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2008</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">1331-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18388127">18388127</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M700564-MCP200">10.1074/mcp.M700564-MCP200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the genomes of a diverse collection of Salmonella enterica serovar Typhimurium definitive phage type 104.</h4>
<p class="authors"><span class="author">Cooke FJ</span>, <span class="author">Brown DJ</span>, <span class="author">Fookes M</span>, <span class="author">Pickard D</span>, <span class="author">Ivens A</span>, <span class="author">Wain J</span>, <span class="author">Roberts M</span>, <span class="author">Kingsley RA</span>, <span class="author">Thomson NR</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom. <a href="mailto:%66%69%6F%6E%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#105;&#111;&#110;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Salmonella enterica serovar Typhimurium definitive phage type 104 (DT104) has caused significant morbidity and mortality in humans and animals for almost three decades. We completed the full DNA sequence of one DT104 strain, NCTC13348, and showed that significant differences between the genome of this isolate and the genome of the previously sequenced strain Salmonella serovar Typhimurium LT2 are due to integrated prophage elements and Salmonella genomic island 1 encoding antibiotic resistance genes. Thirteen isolates of Salmonella serovar Typhimurium DT104 with different pulsed-field gel electrophoresis (PFGE) profiles were analyzed by using multilocus sequence typing (MLST), plasmid profiling, hybridization to a pan-Salmonella DNA microarray, and prophage-based multiplex PCR. All the isolates belonged to a single MLST type, sequence type ST19. Microarray data demonstrated that the gene contents of the 13 DT104 isolates were remarkably conserved. The PFGE DNA fragment size differences in these isolates could be explained to a great extent by differences in the prophage and plasmid contents. Thus, here the nature of variation in different Salmonella serovar Typhimurium DT104 isolates is further defined at the gene and whole-genome levels, illustrating how this phage type evolves over time.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">24</span>;<span class="pagerange">8155-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18849424" rel="external">18849424</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2593209" rel="external">2593209</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00636-08" rel="external">10.1128/JB.00636-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A common genomic framework for a diverse assembly of plasmids in the symbiotic nitrogen fixing bacteria.</h4>
<p class="authors"><span class="author">Crossman LC</span>, <span class="author">Castillo-Ram&iacute;rez S</span>, <span class="author">McAnnula C</span>, <span class="author">Lozano L</span>, <span class="author">Vernikos GS</span>, <span class="author">Acosta JL</span>, <span class="author">Ghazoui ZF</span>, <span class="author">Hern&aacute;ndez-Gonz&aacute;lez I</span>, <span class="author">Meakin G</span>, <span class="author">Walker AW</span>, <span class="author">Hynes MF</span>, <span class="author">Young JP</span>, <span class="author">Downie JA</span>, <span class="author">Romero D</span>, <span class="author">Johnston AW</span>, <span class="author">D&aacute;vila G</span>, <span class="author">Parkhill J</span> and <span class="author">Gonz&aacute;lez V</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%6C%63%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#99;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This work centres on the genomic comparisons of two closely-related nitrogen-fixing symbiotic bacteria, Rhizobium leguminosarum biovar viciae 3841 and Rhizobium etli CFN42. These strains maintain a stable genomic core that is also common to other rhizobia species plus a very variable and significant accessory component. The chromosomes are highly syntenic, whereas plasmids are related by fewer syntenic blocks and have mosaic structures. The pairs of plasmids p42f-pRL12, p42e-pRL11 and p42b-pRL9 as well large parts of p42c with pRL10 are shown to be similar, whereas the symbiotic plasmids (p42d and pRL10) are structurally unrelated and seem to follow distinct evolutionary paths. Even though purifying selection is acting on the whole genome, the accessory component is evolving more rapidly. This component is constituted largely for proteins for transport of diverse metabolites and elements of external origin. The present analysis allows us to conclude that a heterogeneous and quickly diversifying group of plasmids co-exists in a common genomic framework.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">104/P16988</span>, <span class="grant">208/BRE13665</span>, <span class="grant">208/PRS12210</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">7</span>;<span class="pagerange">e2567</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18596979" rel="external">18596979</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2434198" rel="external">2434198</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0002567" rel="external">10.1371/journal.pone.0002567</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complete genome, comparative and functional analysis of Stenotrophomonas maltophilia reveals an organism heavily shielded by drug resistance determinants.</h4>
<p class="authors"><span class="author">Crossman LC</span>, <span class="author">Gould VC</span>, <span class="author">Dow JM</span>, <span class="author">Vernikos GS</span>, <span class="author">Okazaki A</span>, <span class="author">Sebaihia M</span>, <span class="author">Saunders D</span>, <span class="author">Arrowsmith C</span>, <span class="author">Carver T</span>, <span class="author">Peters N</span>, <span class="author">Adlem E</span>, <span class="author">Kerhornou A</span>, <span class="author">Lord A</span>, <span class="author">Murphy L</span>, <span class="author">Seeger K</span>, <span class="author">Squares R</span>, <span class="author">Rutter S</span>, <span class="author">Quail MA</span>, <span class="author">Rajandream MA</span>, <span class="author">Harris D</span>, <span class="author">Churcher C</span>, <span class="author">Bentley SD</span>, <span class="author">Parkhill J</span>, <span class="author">Thomson NR</span> and <span class="author">Avison MB</span></p>
<p class="affiliation">Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract"><strong>Background</strong>: Stenotrophomonas maltophilia is a nosocomial opportunistic pathogen of the Xanthomonadaceae. The organism has been isolated from both clinical and soil environments in addition to the sputum of cystic fibrosis patients and the immunocompromised. Whilst relatively distant phylogenetically, the closest sequenced relatives of S. maltophilia are the plant pathogenic xanthomonads.</p><p class="abstract"><strong>Results</strong>: The genome of the bacteremia-associated isolate S. maltophilia K279a is 4,851,126 bp and of high G+C content. The sequence reveals an organism with a remarkable capacity for drug and heavy metal resistance. In addition to a number of genes conferring resistance to antimicrobial drugs of different classes via alternative mechanisms, nine resistance-nodulation-division (RND)-type putative antimicrobial efflux systems are present. Functional genomic analysis confirms a role in drug resistance for several of the novel RND efflux pumps. S. maltophilia possesses potentially mobile regions of DNA and encodes a number of pili and fimbriae likely to be involved in adhesion and biofilm formation that may also contribute to increased antimicrobial drug resistance.</p><p class="abstract"><strong>Conclusion</strong>: The panoply of antimicrobial drug resistance genes and mobile genetic elements found suggests that the organism can act as a reservoir of antimicrobial drug resistance determinants in a clinical environment, which is an issue of considerable concern.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">4</span>;<span class="pagerange">R74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18419807">18419807</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2643945">2643945</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-4-r74">10.1186/gb-2008-9-4-r74</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Response of Schizosaccharomyces pombe to zinc deficiency.</h4>
<p class="authors"><span class="author">Dainty SJ</span>, <span class="author">Kennedy CA</span>, <span class="author">Watt S</span>, <span class="author">B&auml;hler J</span> and <span class="author">Whitehall SK</span></p>
<p class="affiliation">Institute of Cell and Molecular Biosciences, Newcastle University, Framlington Place, Newcastle upon Tyne NE2 4HH, United Kingdom.</p><p class="abstract">A component of the cellular response to zinc deficiency operates via control of transcript abundance. Therefore, microarray analysis was employed to identify Schizosaccharomyces pombe genes whose mRNA levels are regulated by intracellular zinc status. A set of 57 genes whose mRNA levels were substantially reduced in response to zinc deficiency was identified, while the mRNA levels of 63 genes were increased by this condition. In order to investigate the mechanisms that control these responses, a genetic screen was employed to identify mutants with defective zinc-responsive gene expression. Two strains (II-1 and V7) that were identified by this screen harbor mutations that are linked to zrt1+, which encodes a putative Zrt/IRT-like protein (ZIP) zinc uptake transporter. Importantly, zrt1+ mRNA levels are increased in response to zinc deprivation, and cells lacking functional Zrt1 are highly impaired in their ability to proliferate at limiting zinc concentrations. Furthermore, zrt1 null cells were found to have severely reduced zinc contents, indicating that Zrt1 functions as a key regulator of intracellular zinc levels in fission yeast. The deletion of fet4+, another zinc-responsive gene encoding a putative metal ion transporter, exacerbated the phenotypes associated with the loss of Zrt1, suggesting that Fet4 also plays a role in zinc uptake under limiting conditions.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/C004752/1</span>; Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2008</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">454-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18203864" rel="external">18203864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2268511" rel="external">2268511</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.00408-07" rel="external">10.1128/EC.00408-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The RNA WikiProject: community annotation of RNA families.</h4>
<p class="authors"><span class="author">Daub J</span>, <span class="author">Gardner PP</span>, <span class="author">Tate J</span>, <span class="author">Ramsk&ouml;ld D</span>, <span class="author">Manske M</span>, <span class="author">Scott WG</span>, <span class="author">Weinberg Z</span>, <span class="author">Griffiths-Jones S</span> and <span class="author">Bateman A</span></p><p class="abstract">The online encyclopedia Wikipedia has become one of the most important online references in the world and has a substantial and growing scientific content. A search of Google with many RNA-related keywords identifies a Wikipedia article as the top hit. We believe that the RNA community has an important and timely opportunity to maximize the content and quality of RNA information in Wikipedia. To this end, we have formed the RNA WikiProject (http://en.wikipedia.org/wiki/Wikipedia:WikiProject_RNA) as part of the larger Molecular and Cellular Biology WikiProject. We have created over 600 new Wikipedia articles describing families of noncoding RNAs based on the Rfam database, and invite the community to update, edit, and correct these articles. The Rfam database now redistributes this Wikipedia content as the primary textual annotation of its RNA families. Users can, therefore, for the first time, directly edit the content of one of the major RNA databases. We believe that this Wikipedia/Rfam link acts as a functioning model for incorporating community annotation into molecular biology databases.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; Wellcome Trust: <span class="grant">077044</span></p>
<p class="publication"><span class="title">RNA (New York, N.Y.)</span>&nbsp;<span class="year">2008</span>;<span class="volume">14</span>;<span class="number">12</span>;<span class="pagerange">2462-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18945806">18945806</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2590952">2590952</a>; DOI: <a href="http://dx.doi.org/10.1261%2Frna.1200508">10.1261/rna.1200508</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pathogenomics: an updated European Research Agenda.</h4>
<p class="authors"><span class="author">Demuth A</span>, <span class="author">Aharonowitz Y</span>, <span class="author">Bachmann TT</span>, <span class="author">Blum-Oehler G</span>, <span class="author">Buchrieser C</span>, <span class="author">Covacci A</span>, <span class="author">Dobrindt U</span>, <span class="author">Em&ouml;dy L</span>, <span class="author">van der Ende A</span>, <span class="author">Ewbank J</span>, <span class="author">Fern&aacute;ndez LA</span>, <span class="author">Frosch M</span>, <span class="author">Garc&iacute;a-Del Portillo F</span>, <span class="author">Gilmore MS</span>, <span class="author">Glaser P</span>, <span class="author">Goebel W</span>, <span class="author">Hasnain SE</span>, <span class="author">Heesemann J</span>, <span class="author">Islam K</span>, <span class="author">Korhonen T</span>, <span class="author">Maiden M</span>, <span class="author">Meyer TF</span>, <span class="author">Montecucco C</span>, <span class="author">Oswald E</span>, <span class="author">Parkhill J</span>, <span class="author">Pucciarelli MG</span>, <span class="author">Ron E</span>, <span class="author">Svanborg C</span>, <span class="author">Uhlin BE</span>, <span class="author">Wai SN</span>, <span class="author">Wehland J</span> and <span class="author">Hacker J</span></p>
<p class="affiliation">Institut f&uuml;r Molekulare Infektionsbiologie, R&ouml;ntgenring 11, 97070 W&uuml;rzburg, Germany.</p><p class="abstract">The emerging genomic technologies and bioinformatics provide novel opportunities for studying life-threatening human pathogens and to develop new applications for the improvement of human and animal health and the prevention, treatment, and diagnosis of infections. Based on the ecology and population biology of pathogens and related organisms and their connection to epidemiology, more accurate typing technologies and approaches will lead to better means of disease control. The analysis of the genome plasticity and gene pools of pathogenic bacteria including antigenic diversity and antigenic variation results in more effective vaccines and vaccine implementation programs. The study of newly identified and uncultivated microorganisms enables the identification of new threats. The scrutiny of the metabolism of the pathogen in the host allows the identification of new targets for anti-infectives and therapeutic approaches. The development of modulators of host responses and mediators of host damage will be facilitated by the research on interactions of microbes and hosts, including mechanisms of host damage, acute and chronic relationships as well as commensalisms. The study of multiple pathogenic and non-pathogenic microbes interacting in the host will improve the management of multiple infections and will allow probiotic and prebiotic interventions. Needless to iterate, the application of the results of improved prevention and treatment of infections into clinical tests will have a positive impact on the management of human and animal disease. The Pathogenomics Research Agenda draws on discussions with experts of the Network of Excellence &quot;EuroPathoGenomics&quot; at the management board meeting of the project held during 18-21 April 2007, in the Villa Vigoni, Menaggio, Italy. Based on a proposed European Research Agenda in the field of pathogenomics by the ERA-NET PathoGenoMics the meeting&#39;s participants updated the established list of topics as the research agenda for the future.</p>
<p class="publication"><span class="title">Infection, genetics and evolution : journal of molecular epidemiology and evolutionary genetics in infectious diseases</span>&nbsp;<span class="year">2008</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">386-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18321793" rel="external">18321793</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.meegid.2008.01.005" rel="external">10.1016/j.meegid.2008.01.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From gene expression to disease risk.</h4>
<p class="authors"><span class="author">Dermitzakis ET</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">5</span>;<span class="pagerange">492-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18443581">18443581</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0508-492">10.1038/ng0508-492</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulatory variation and evolution: implications for disease.</h4>
<p class="authors"><span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, CB10 1SA Cambridge, United Kingdom.</p><p class="abstract">In the past few years, it has become apparent that there is a substantial amount of noncoding DNA that contributes to genome function. However, the multidimensionality of noncoding DNA properties does not allow us to readily identify, characterize, and assess the functional impact of mutations, polymorphisms, and interspecific substitutions. In this chapter, we discuss the evolutionary properties of some of the known noncoding genomic elements, namely regulatory regions, and the extensions of this to other potentially functionally important noncoding regions such as conserved noncoding regions. The implications of this analysis for studies looking at molecular phenotypes such as gene expression and whole-organism phenotypes (e.g., disease) are presented in the context of the exploration of noncoding DNA properties. The aim is to take advantage of current and emerging analysis methods for noncoding DNA to elucidate the genetic causes of phenotypic variation.</p>
<p class="publication"><span class="title">Advances in genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">61</span>;<span class="pagerange">295-306</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18282511">18282511</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0065-2660%2807%2900011-9">10.1016/S0065-2660(07)00011-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Minimum information specification for in situ hybridization and immunohistochemistry experiments (MISFISHIE).</h4>
<p class="authors"><span class="author">Deutsch EW</span>, <span class="author">Ball CA</span>, <span class="author">Berman JJ</span>, <span class="author">Bova GS</span>, <span class="author">Brazma A</span>, <span class="author">Bumgarner RE</span>, <span class="author">Campbell D</span>, <span class="author">Causton HC</span>, <span class="author">Christiansen JH</span>, <span class="author">Daian F</span>, <span class="author">Dauga D</span>, <span class="author">Davidson DR</span>, <span class="author">Gimenez G</span>, <span class="author">Goo YA</span>, <span class="author">Grimmond S</span>, <span class="author">Henrich T</span>, <span class="author">Herrmann BG</span>, <span class="author">Johnson MH</span>, <span class="author">Korb M</span>, <span class="author">Mills JC</span>, <span class="author">Oudes AJ</span>, <span class="author">Parkinson HE</span>, <span class="author">Pascal LE</span>, <span class="author">Pollet N</span>, <span class="author">Quackenbush J</span>, <span class="author">Ramialison M</span>, <span class="author">Ringwald M</span>, <span class="author">Salgado D</span>, <span class="author">Sansone SA</span>, <span class="author">Sherlock G</span>, <span class="author">Stoeckert CJ</span>, <span class="author">Swedlow J</span>, <span class="author">Taylor RC</span>, <span class="author">Walashek L</span>, <span class="author">Warford A</span>, <span class="author">Wilkinson DG</span>, <span class="author">Zhou Y</span>, <span class="author">Zon LI</span>, <span class="author">Liu AY</span> and <span class="author">True LD</span></p>
<p class="affiliation">Institute for Systems Biology, 1441 N 34th Street, Seattle, Washington 98103, USA.</p><p class="abstract">One purpose of the biomedical literature is to report results in sufficient detail that the methods of data collection and analysis can be independently replicated and verified. Here we present reporting guidelines for gene expression localization experiments: the minimum information specification for in situ hybridization and immunohistochemistry experiments (MISFISHIE). MISFISHIE is modeled after the Minimum Information About a Microarray Experiment (MIAME) specification for microarray experiments. Both guidelines define what information should be reported without dictating a format for encoding that information. MISFISHIE describes six types of information to be provided for each experiment: experimental design, biomaterials and treatments, reporters, staining, imaging data and image characterizations. This specification has benefited the consortium within which it was developed and is expected to benefit the wider research community. We welcome feedback from the scientific community to help improve our proposal.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U117532048</span>, <span class="grant">MC_U127527203</span>; NHLBI NIH HHS: <span class="grant">R33 HL073712</span>; NIDDK NIH HHS: <span class="grant">DK63328</span>, <span class="grant">DK63400</span>, <span class="grant">DK63481</span>, <span class="grant">DK63483</span>, <span class="grant">DK63630</span>, <span class="grant">R01 DK079798</span>, <span class="grant">R01 DK079798-01A2</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">3</span>;<span class="pagerange">305-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18327244">18327244</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt1391">10.1038/nbt1391</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">X-linked protocadherin 19 mutations cause female-limited epilepsy and cognitive impairment.</h4>
<p class="authors"><span class="author">Dibbens LM</span>, <span class="author">Tarpey PS</span>, <span class="author">Hynes K</span>, <span class="author">Bayly MA</span>, <span class="author">Scheffer IE</span>, <span class="author">Smith R</span>, <span class="author">Bomar J</span>, <span class="author">Sutton E</span>, <span class="author">Vandeleur L</span>, <span class="author">Shoubridge C</span>, <span class="author">Edkins S</span>, <span class="author">Turner SJ</span>, <span class="author">Stevens C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Tofts C</span>, <span class="author">Barthorpe S</span>, <span class="author">Buck G</span>, <span class="author">Cole J</span>, <span class="author">Halliday K</span>, <span class="author">Jones D</span>, <span class="author">Lee R</span>, <span class="author">Madison M</span>, <span class="author">Mironenko T</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Wray P</span>, <span class="author">Teague J</span>, <span class="author">Dicks E</span>, <span class="author">Butler A</span>, <span class="author">Menzies A</span>, <span class="author">Jenkinson A</span>, <span class="author">Shepherd R</span>, <span class="author">Gusella JF</span>, <span class="author">Afawi Z</span>, <span class="author">Mazarib A</span>, <span class="author">Neufeld MY</span>, <span class="author">Kivity S</span>, <span class="author">Lev D</span>, <span class="author">Lerman-Sagie T</span>, <span class="author">Korczyn AD</span>, <span class="author">Derry CP</span>, <span class="author">Sutherland GR</span>, <span class="author">Friend K</span>, <span class="author">Shaw M</span>, <span class="author">Corbett M</span>, <span class="author">Kim HG</span>, <span class="author">Geschwind DH</span>, <span class="author">Thomas P</span>, <span class="author">Haan E</span>, <span class="author">Ryan S</span>, <span class="author">McKee S</span>, <span class="author">Berkovic SF</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Mulley JC</span> and <span class="author">G&eacute;cz J</span></p>
<p class="affiliation">Department of Genetic Medicine, Level 9 Rieger Building, Women&#39;s and Children&#39;s Hospital, 72 King William Road, North Adelaide, South Australia 5006, Australia. <a href="mailto:%6C%65%61%6E%6E%65.%64%69%62%62%65%6E%73%40%63%79%77%68%73.%73%61.%67%6F%76.%61%75">&#108;&#101;&#97;&#110;&#110;&#101;&#46;&#100;&#105;&#98;&#98;&#101;&#110;&#115;&#64;&#99;&#121;&#119;&#104;&#115;&#46;&#115;&#97;&#46;&#103;&#111;&#118;&#46;&#97;&#117;</a></p><p class="abstract">Epilepsy and mental retardation limited to females (EFMR) is a disorder with an X-linked mode of inheritance and an unusual expression pattern. Disorders arising from mutations on the X chromosome are typically characterized by affected males and unaffected carrier females. In contrast, EFMR spares transmitting males and affects only carrier females. Aided by systematic resequencing of 737 X chromosome genes, we identified different protocadherin 19 (PCDH19) gene mutations in seven families with EFMR. Five mutations resulted in the introduction of a premature termination codon. Study of two of these demonstrated nonsense-mediated decay of PCDH19 mRNA. The two missense mutations were predicted to affect adhesiveness of PCDH19 through impaired calcium binding. PCDH19 is expressed in developing brains of human and mouse and is the first member of the cadherin superfamily to be directly implicated in epilepsy or mental retardation.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">N01-HD-4-3368</span>, <span class="grant">N01-HD-4-3383</span>; NIGMS NIH HHS: <span class="grant">GM061354</span>; NIMH NIH HHS: <span class="grant">R01 MH 64547</span>, <span class="grant">R01 MH064547</span>, <span class="grant">R01 MH064547-01</span>, <span class="grant">R01 MH064547-01S1</span>, <span class="grant">R01 MH064547-02</span>, <span class="grant">R01 MH064547-02S1</span>, <span class="grant">R01 MH064547-03</span>, <span class="grant">R01 MH064547-04</span>, <span class="grant">R01 MH064547-05</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">6</span>;<span class="pagerange">776-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18469813">18469813</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2756413">2756413</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.149">10.1038/ng.149</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Altered patterns of gene expression underlying the enhanced immunogenicity of radiation-attenuated schistosomes.</h4>
<p class="authors"><span class="author">Dillon GP</span>, <span class="author">Feltwell T</span>, <span class="author">Skelton J</span>, <span class="author">Coulson PS</span>, <span class="author">Wilson RA</span> and <span class="author">Ivens AC</span></p>
<p class="affiliation">Department of Biology, University of York, York, United Kingdom. <a href="mailto:%67%70%64%31%30%35%40%79%6F%72%6B.%61%63.%75%6B">&#103;&#112;&#100;&#49;&#48;&#53;&#64;&#121;&#111;&#114;&#107;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Schistosome cercariae only elicit high levels of protective immunity against a challenge infection if they are optimally attenuated by exposure to ionising radiation that truncates their migration in the lungs. However, the underlying molecular mechanisms responsible for the altered phenotype of the irradiated parasite that primes for protection have yet to be identified.</p><p class="abstract">We have used a custom microarray comprising probes derived from lung-stage parasites to compare patterns of gene expression in schistosomula derived from normal and irradiated cercariae. These were transformed in vitro and cultured for four, seven, and ten days to correspond in development to the priming parasites, before RNA extraction. At these late times after the radiation insult, transcript suppression was the principal feature of the irradiated larvae. Individual gene analysis revealed that only seven were significantly down-regulated in the irradiated versus normal larvae at the three time-points; notably, four of the protein products are present in the tegument or associated with its membranes, perhaps indicating a perturbed function. Grouping of transcripts using Gene Ontology (GO) and subsequent Gene Set Enrichment Analysis (GSEA) proved more informative in teasing out subtle differences. Deficiencies in signalling pathways involving G-protein-coupled receptors suggest the parasite is less able to sense its environment. Reduction of cytoskeleton transcripts could indicate compromised structure which, coupled with a paucity of neuroreceptor transcripts, may mean the parasite is also unable to respond correctly to external stimuli.</p><p class="abstract">The transcriptional differences observed are concordant with the known extended transit of attenuated parasites through skin-draining lymph nodes and the lungs: prolonged priming of the immune system by the parasite, rather than over-expression of novel antigens, could thus explain the efficacy of the irradiated vaccine.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; NIAID NIH HHS: <span class="grant">AI54711-02</span></p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2008</span>;<span class="volume">2</span>;<span class="number">5</span>;<span class="pagerange">e240</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18493602" rel="external">18493602</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2375114" rel="external">2375114</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000240" rel="external">10.1371/journal.pntd.0000240</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modifier effects between regulatory and protein-coding variation.</h4>
<p class="authors"><span class="author">Dimas AS</span>, <span class="author">Stranger BE</span>, <span class="author">Beazley C</span>, <span class="author">Finn RD</span>, <span class="author">Ingle CE</span>, <span class="author">Forrest MS</span>, <span class="author">Ritchie ME</span>, <span class="author">Deloukas P</span>, <span class="author">Tavar&eacute; S</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK.</p><p class="abstract">Genome-wide associations have shown a lot of promise in dissecting the genetics of complex traits in humans with single variants, yet a large fraction of the genetic effects is still unaccounted for. Analyzing genetic interactions between variants (epistasis) is one of the potential ways forward. We investigated the abundance and functional impact of a specific type of epistasis, namely the interaction between regulatory and protein-coding variants. Using genotype and gene expression data from the 210 unrelated individuals of the original four HapMap populations, we have explored the combined effects of regulatory and protein-coding single nucleotide polymorphisms (SNPs). We predict that about 18% (1,502 out of 8,233 nsSNPs) of protein-coding variants are differentially expressed among individuals and demonstrate that regulatory variants can modify the functional effect of a coding variant in cis. Furthermore, we show that such interactions in cis can affect the expression of downstream targets of the gene containing the protein-coding SNP. In this way, a cis interaction between regulatory and protein-coding variants has a trans impact on gene expression. Given the abundance of both types of variants in human populations, we propose that joint consideration of regulatory and protein-coding variants may reveal additional genetic effects underlying complex traits and disease and may shed light on causes of differential penetrance of known disease variants.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">10</span>;<span class="pagerange">e1000244</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18974877" rel="external">18974877</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2570624" rel="external">2570624</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000244" rel="external">10.1371/journal.pgen.1000244</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficient targeted transcript discovery via array-based normalization of RACE libraries.</h4>
<p class="authors"><span class="author">Djebali S</span>, <span class="author">Kapranov P</span>, <span class="author">Foissac S</span>, <span class="author">Lagarde J</span>, <span class="author">Reymond A</span>, <span class="author">Ucla C</span>, <span class="author">Wyss C</span>, <span class="author">Drenkow J</span>, <span class="author">Dumais E</span>, <span class="author">Murray RR</span>, <span class="author">Lin C</span>, <span class="author">Szeto D</span>, <span class="author">Denoeud F</span>, <span class="author">Calvo M</span>, <span class="author">Frankish A</span>, <span class="author">Harrow J</span>, <span class="author">Makrythanasis P</span>, <span class="author">Vidal M</span>, <span class="author">Salehi-Ashtiani K</span>, <span class="author">Antonarakis SE</span>, <span class="author">Gingeras TR</span> and <span class="author">Guig&oacute; R</span></p>
<p class="affiliation">Grup de Recerca en Inform&agrave;tica Biom&egrave;dica, Institut Municipal d&#39;Investigaci&oacute; M&egrave;dica/Universitat Pompeu Fabra, Dr. Aiguader 88, 08003 Barcelona, Spain.</p><p class="abstract">Rapid amplification of cDNA ends (RACE) is a widely used approach for transcript identification. Random clone selection from the RACE mixture, however, is an ineffective sampling strategy if the dynamic range of transcript abundances is large. To improve sampling efficiency of human transcripts, we hybridized the products of the RACE reaction onto tiling arrays and used the detected exons to delineate a series of reverse-transcriptase (RT)-PCRs, through which the original RACE transcript population was segregated into simpler transcript populations. We independently cloned the products and sequenced randomly selected clones. This approach, RACEarray, is superior to direct cloning and sequencing of RACE products because it specifically targets new transcripts and often results in overall normalization of transcript abundance. We show theoretically and experimentally that this strategy leads indeed to efficient sampling of new transcripts, and we investigated multiplexing the strategy by pooling RACE reactions from multiple interrogated loci before hybridization.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">N01-CO-12400</span>; NHGRI NIH HHS: <span class="grant">U01 HG003147</span>, <span class="grant">U01 HG003147-01</span>, <span class="grant">U01 HG003147-02</span>, <span class="grant">U01 HG003147-02S1</span>, <span class="grant">U01 HG003147-02S2</span>, <span class="grant">U01 HG003147-02S3</span>, <span class="grant">U01 HG003150-01</span>, <span class="grant">U01 HG003150-02</span>, <span class="grant">U01 HG003150-03</span>, <span class="grant">U01 HG003150-03S1</span>, <span class="grant">U01 HG003150-03S2</span>, <span class="grant">U01HG003147</span>, <span class="grant">U01HG003150</span>, <span class="grant">U54 HG004555</span>, <span class="grant">U54 HG004557</span>, <span class="grant">U54 HG004557-01</span>, <span class="grant">U54 HG004557-02</span>, <span class="grant">U54 HG004557-02S1</span>, <span class="grant">U54 HG004557-03</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2008</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">629-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18500348">18500348</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2713501">2713501</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1216">10.1038/nmeth.1216</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NestedMICA as an ab initio protein motif discovery tool.</h4>
<p class="authors"><span class="author">Do&#x11F;ruel M</span>, <span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK. <a href="mailto:%6D%64%35%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#100;&#53;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Discovering overrepresented patterns in amino acid sequences is an important step in protein functional element identification. We adapted and extended NestedMICA, an ab initio motif finder originally developed for finding transcription binding site motifs, to find short protein signals, and compared its performance with another popular protein motif finder, MEME. NestedMICA, an open source protein motif discovery tool written in Java, is driven by a Monte Carlo technique called Nested Sampling. It uses multi-class sequence background models to represent different &quot;uninteresting&quot; parts of sequences that do not contain motifs of interest. In order to assess NestedMICA as a protein motif finder, we have tested it on synthetic datasets produced by spiking instances of known motifs into a randomly selected set of protein sequences. NestedMICA was also tested using a biologically-authentic test set, where we evaluated its performance with respect to varying sequence length.</p><p class="abstract"><strong>Results</strong>: Generally NestedMICA recovered most of the short (3-9 amino acid long) test protein motifs spiked into a test set of sequences at different frequencies. We showed that it can be used to find multiple motifs at the same time, too. In all the assessment experiments we carried out, its overall motif discovery performance was better than that of MEME.</p><p class="abstract"><strong>Conclusion</strong>: NestedMICA proved itself to be a robust and sensitive ab initio protein motif finder, even for relatively short motifs that exist in only a small fraction of sequences.</p><p class="abstract"><strong>Availability</strong>: NestedMICA is available under the Lesser GPL open-source license from: http://www.sanger.ac.uk/Software/analysis/nmica/</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18194537">18194537</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2267705">2267705</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-9-19">10.1186/1471-2105-9-19</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Foreword. Comparative cytogenetics in the genomics era: cytogenomics comes of age.</h4>
<p class="authors"><span class="author">Dobigny G</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Institut de Recherche pour le D&eacute;veloppement, Campus de Baillarguet CS30016, 34988, Montferrier sur Lez, France. <a href="mailto:%64%6F%62%69%67%6E%79%40%6D%70%6C.%69%72%64.%66%72">&#100;&#111;&#98;&#105;&#103;&#110;&#121;&#64;&#109;&#112;&#108;&#46;&#105;&#114;&#100;&#46;&#102;&#114;</a></p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">1-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18293101" rel="external">18293101</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-008-1213-y" rel="external">10.1007/s10577-008-1213-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Protein kinases of malaria parasites: an update.</h4>
<p class="authors"><span class="author">Doerig C</span>, <span class="author">Billker O</span>, <span class="author">Haystead T</span>, <span class="author">Sharma P</span>, <span class="author">Tobin AB</span> and <span class="author">Waters NC</span></p>
<p class="affiliation">INSERM U609, Wellcome Centre for Molecular Parasitology, University of Glasgow Biomedical Research Centre, Glasgow G12 8TA, Scotland, UK. <a href="mailto:%63%64%6F%65%72%30%30%31%40%75%64%63%66.%67%6C%61.%61%63.%75%6B">&#99;&#100;&#111;&#101;&#114;&#48;&#48;&#49;&#64;&#117;&#100;&#99;&#102;&#46;&#103;&#108;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Protein kinases (PKs) play crucial roles in the control of proliferation and differentiation in eukaryotic cells. Research on protein phosphorylation has expanded tremendously in the past few years, in part as a consequence of the realization that PKs represent attractive drug targets in a variety of diseases. Activity in Plasmodium PK research has followed this trend, and several reports on various aspects of this subject were delivered at the Molecular Approaches to Malaria 2008 meeting (MAM2008), a sharp increase from the previous meeting. Here, the authors of most of these communications join to propose an integrated update of the development of the rapidly expanding field of Plasmodium kinomics.</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">12</span>;<span class="pagerange">570-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18845480" rel="external">18845480</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2008.08.007" rel="external">10.1016/j.pt.2008.08.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An association analysis of murine anxiety genes in humans implicates novel candidate genes for anxiety disorders.</h4>
<p class="authors"><span class="author">Donner J</span>, <span class="author">Pirkola S</span>, <span class="author">Silander K</span>, <span class="author">Kananen L</span>, <span class="author">Terwilliger JD</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Peltonen L</span> and <span class="author">Hovatta I</span></p>
<p class="affiliation">Research Program of Molecular Neurology, Biomedicum Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: Human anxiety disorders are complex diseases with largely unknown etiology. We have taken a cross-species approach to identify genes that regulate anxiety-like behavior with inbred mouse strains that differ in their innate anxiety levels as a model. We previously identified 17 genes with expression levels that correlate with anxiety behavior across the studied strains. In the present study, we tested their 13 known human homologues as candidate genes for human anxiety disorders with a genetic association study.</p><p class="abstract"><strong>Methods</strong>: We describe an anxiety disorder study sample derived from a Finnish population-based cohort and consisting of 321 patients and 653 carefully matched control subjects, all interviewed to obtain DSM-IV diagnoses. We genotyped altogether 208 single nucleotide polymorphisms (SNPs) (all non-synonymous SNPs, SNPs that alter potential microRNA binding sites, and gap-filling SNPs selected on the basis of HapMap information) from the investigated anxiety candidate genes.</p><p class="abstract"><strong>Results</strong>: Specific alleles and haplotypes of six of the examined genes revealed some evidence for association (p &lt; or = .01). The most significant evidence for association with different anxiety disorder subtypes were: p = .0009 with ALAD (delta-aminolevulinate dehydratase) in social phobia, p = .009 with DYNLL2 (dynein light chain 2) in generalized anxiety disorder, and p = .004 with PSAP (prosaposin) in panic disorder.</p><p class="abstract"><strong>Conclusions</strong>: Our findings suggest that variants in these genes might predispose to specific human anxiety disorders. These results illustrate the potential utility of cross-species approaches in identification of candidate genes for psychiatric disorders.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2008</span>;<span class="volume">64</span>;<span class="number">8</span>;<span class="pagerange">672-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18639233" rel="external">18639233</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682432" rel="external">2682432</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2008.06.002" rel="external">10.1016/j.biopsych.2008.06.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis.</h4>
<p class="authors"><span class="author">Down TA</span>, <span class="author">Rakyan VK</span>, <span class="author">Turner DJ</span>, <span class="author">Flicek P</span>, <span class="author">Li H</span>, <span class="author">Kulesha E</span>, <span class="author">Gr&auml;f S</span>, <span class="author">Johnson N</span>, <span class="author">Herrero J</span>, <span class="author">Tomazou EM</span>, <span class="author">Thorne NP</span>, <span class="author">B&auml;ckdahl L</span>, <span class="author">Herberth M</span>, <span class="author">Howe KL</span>, <span class="author">Jackson DK</span>, <span class="author">Miretti MM</span>, <span class="author">Marioni JC</span>, <span class="author">Birney E</span>, <span class="author">Hubbard TJ</span>, <span class="author">Durbin R</span>, <span class="author">Tavar&eacute; S</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Cancer Research UK Gurdon Institute, and Department of Genetics, University of Cambridge, Tennis Court Road, Cambridge CB2 1QR, UK. <a href="mailto:%74%68%6F%6D%61%73.%64%6F%77%6E%40%67%75%72%64%6F%6E.%63%61%6D.%61%63.%75%6B">&#116;&#104;&#111;&#109;&#97;&#115;&#46;&#100;&#111;&#119;&#110;&#64;&#103;&#117;&#114;&#100;&#111;&#110;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">DNA methylation is an indispensible epigenetic modification required for regulating the expression of mammalian genomes. Immunoprecipitation-based methods for DNA methylome analysis are rapidly shifting the bottleneck in this field from data generation to data analysis, necessitating the development of better analytical tools. In particular, an inability to estimate absolute methylation levels remains a major analytical difficulty associated with immunoprecipitation-based DNA methylation profiling. To address this issue, we developed a cross-platform algorithm-Bayesian tool for methylation analysis (Batman)-for analyzing methylated DNA immunoprecipitation (MeDIP) profiles generated using oligonucleotide arrays (MeDIP-chip) or next-generation sequencing (MeDIP-seq). We developed the latter approach to provide a high-resolution whole-genome DNA methylation profile (DNA methylome) of a mammalian genome. Strong correlation of our data, obtained using mature human spermatozoa, with those obtained using bisulfite sequencing suggest that combining MeDIP-seq or MeDIP-chip with Batman provides a robust, quantitative and cost-effective functional genomic strategy for elucidating the function of DNA methylation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C14303/A8646</span>; Wellcome Trust: <span class="grant">077198</span>, <span class="grant">083563</span>, <span class="grant">084071</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">7</span>;<span class="pagerange">779-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18612301">18612301</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2644410">2644410</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt1414">10.1038/nbt1414</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression changes linked to antimicrobial resistance, oxidative stress, iron depletion and retained motility are observed when Burkholderia cenocepacia grows in cystic fibrosis sputum.</h4>
<p class="authors"><span class="author">Drevinek P</span>, <span class="author">Holden MT</span>, <span class="author">Ge Z</span>, <span class="author">Jones AM</span>, <span class="author">Ketchell I</span>, <span class="author">Gill RT</span> and <span class="author">Mahenthiralingam E</span></p>
<p class="affiliation">Cardiff School of Biosciences, Cardiff University, Cardiff, UK. <a href="mailto:%70%61%76%65%6C.%64%72%65%76%69%6E%65%6B%40%6C%66%6D%6F%74%6F%6C.%63%75%6E%69.%63%7A">&#112;&#97;&#118;&#101;&#108;&#46;&#100;&#114;&#101;&#118;&#105;&#110;&#101;&#107;&#64;&#108;&#102;&#109;&#111;&#116;&#111;&#108;&#46;&#99;&#117;&#110;&#105;&#46;&#99;&#122;</a></p><p class="abstract"><strong>Background</strong>: Bacteria from the Burkholderia cepacia complex (Bcc) are the only group of cystic fibrosis (CF) respiratory pathogens that may cause death by an invasive infection known as cepacia syndrome. Their large genome (&gt; 7000 genes) and multiple pathways encoding the same putative functions make virulence factor identification difficult in these bacteria.</p><p class="abstract"><strong>Methods</strong>: A novel microarray was designed to the genome of Burkholderia cenocepacia J2315 and transcriptomics used to identify genes that were differentially regulated when the pathogen was grown in a CF sputum-based infection model. Sputum samples from CF individuals infected with the same B. cenocepacia strain as genome isolate were used, hence, other than a dilution into a minimal growth medium (used as the control condition), no further treatment of the sputum was carried out.</p><p class="abstract"><strong>Results</strong>: A total of 723 coding sequences were significantly altered, with 287 upregulated and 436 downregulated; the microarray-observed expression was validated by quantitative PCR on five selected genes. B. cenocepacia genes with putative functions in antimicrobial resistance, iron uptake, protection against reactive oxygen and nitrogen species, secretion and motility were among the most altered in sputum. Novel upregulated genes included: a transmembrane ferric reductase (BCAL0270) implicated in iron metabolism, a novel protease (BCAL0849) that may play a role in host tissue destruction, an organic hydroperoxide resistance gene (BCAM2753), an oxidoreductase (BCAL1107) and a nitrite/sulfite reductase (BCAM1676) that may play roles in resistance to the host defenses. The assumptions of growth under iron-depletion and oxidative stress formulated from the microarray data were tested and confirmed by independent growth of B. cenocepacia under each respective environmental condition.</p><p class="abstract"><strong>Conclusion</strong>: Overall, our first full transcriptomic analysis of B. cenocepacia demonstrated the pathogen alters expression of over 10% of the 7176 genes within its genome when it grows in CF sputum. Novel genetic pathways involved in responses to antimicrobial resistance, oxidative stress, and iron metabolism were revealed by the microarray analysis. Virulence factors such as the cable pilus and Cenocepacia Pathogenicity Island were unaltered in expression. However, B. cenocepacia sustained or increased expression of motility-associated genes in sputum, maintaining a potentially invasive phenotype associated with cepacia syndrome.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">075586</span></p>
<p class="publication"><span class="title">BMC infectious diseases</span>&nbsp;<span class="year">2008</span>;<span class="volume">8</span>;<span class="pagerange">121</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18801206" rel="external">18801206</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2559838" rel="external">2559838</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2334-8-121" rel="external">10.1186/1471-2334-8-121</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Emergence of highly fluoroquinolone-resistant Salmonella enterica serovar Typhi in a community-based fever surveillance from Kolkata, India.</h4>
<p class="authors"><span class="author">Dutta S</span>, <span class="author">Sur D</span>, <span class="author">Manna B</span>, <span class="author">Sen B</span>, <span class="author">Bhattacharya M</span>, <span class="author">Bhattacharya SK</span>, <span class="author">Wain J</span>, <span class="author">Nair S</span>, <span class="author">Clemens JD</span> and <span class="author">Ochiai RL</span></p>
<p class="publication"><span class="title">International journal of antimicrobial agents</span>&nbsp;<span class="year">2008</span>;<span class="volume">31</span>;<span class="number">4</span>;<span class="pagerange">387-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18280709" rel="external">18280709</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijantimicag.2007.12.004" rel="external">10.1016/j.ijantimicag.2007.12.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The evolution of the DLK1-DIO3 imprinted domain in mammals.</h4>
<p class="authors"><span class="author">Edwards CA</span>, <span class="author">Mungall AJ</span>, <span class="author">Matthews L</span>, <span class="author">Ryder E</span>, <span class="author">Gray DJ</span>, <span class="author">Pask AJ</span>, <span class="author">Shaw G</span>, <span class="author">Graves JA</span>, <span class="author">Rogers J</span>, <span class="author">SAVOIR consortium</span>, <span class="author">Dunham I</span>, <span class="author">Renfree MB</span> and <span class="author">Ferguson-Smith AC</span></p>
<p class="affiliation">Department of Physiology, Development, and Neuroscience, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">A comprehensive, domain-wide comparative analysis of genomic imprinting between mammals that imprint and those that do not can provide valuable information about how and why imprinting evolved. The imprinting status, DNA methylation, and genomic landscape of the Dlk1-Dio3 cluster were determined in eutherian, metatherian, and prototherian mammals including tammar wallaby and platypus. Imprinting across the whole domain evolved after the divergence of eutherian from marsupial mammals and in eutherians is under strong purifying selection. The marsupial locus at 1.6 megabases, is double that of eutherians due to the accumulation of LINE repeats. Comparative sequence analysis of the domain in seven vertebrates determined evolutionary conserved regions common to particular sub-groups and to all vertebrates. The emergence of Dlk1-Dio3 imprinting in eutherians has occurred on the maternally inherited chromosome and is associated with region-specific resistance to expansion by repetitive elements and the local introduction of noncoding transcripts including microRNAs and C/D small nucleolar RNAs. A recent mammal-specific retrotransposition event led to the formation of a completely new gene only in the eutherian domain, which may have driven imprinting at the cluster.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0400156</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">e135</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18532878">18532878</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2408620">2408620</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0060135">10.1371/journal.pbio.0060135</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary expansion and anatomical specialization of synapse proteome complexity.</h4>
<p class="authors"><span class="author">Emes RD</span>, <span class="author">Pocklington AJ</span>, <span class="author">Anderson CN</span>, <span class="author">Bayes A</span>, <span class="author">Collins MO</span>, <span class="author">Vickers CA</span>, <span class="author">Croning MD</span>, <span class="author">Malik BR</span>, <span class="author">Choudhary JS</span>, <span class="author">Armstrong JD</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Institute for Science and Technology in Medicine, Keele University, Thornburrow Drive, Hartshill, Stoke-on-Trent ST4 7QB, UK.</p><p class="abstract">Understanding the origins and evolution of synapses may provide insight into species diversity and the organization of the brain. Using comparative proteomics and genomics, we examined the evolution of the postsynaptic density (PSD) and membrane-associated guanylate kinase (MAGUK)-associated signaling complexes (MASCs) that underlie learning and memory. PSD and MASC orthologs found in yeast carry out basic cellular functions to regulate protein synthesis and structural plasticity. We observed marked changes in signaling complexity at the yeast-metazoan and invertebrate-vertebrate boundaries, with an expansion of key synaptic components, notably receptors, adhesion/cytoskeletal proteins and scaffold proteins. A proteomic comparison of Drosophila and mouse MASCs revealed species-specific adaptation with greater signaling complexity in mouse. Although synaptic components were conserved amongst diverse vertebrate species, mapping mRNA and protein expression in the mouse brain showed that vertebrate-specific components preferentially contributed to differences between brain regions. We propose that the evolution of synapse complexity around a core proto-synapse has contributed to invertebrate-vertebrate differences and to brain specialization.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G90/112</span>, <span class="grant">G90/93</span>; Wellcome Trust: <span class="grant">077155</span></p>
<p class="publication"><span class="title">Nature neuroscience</span>&nbsp;<span class="year">2008</span>;<span class="volume">11</span>;<span class="number">7</span>;<span class="pagerange">799-806</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18536710">18536710</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3624047">3624047</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnn.2135">10.1038/nn.2135</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Independent introduction of two lactase-persistence alleles into human populations reflects different history of adaptation to milk culture.</h4>
<p class="authors"><span class="author">Enattah NS</span>, <span class="author">Jensen TG</span>, <span class="author">Nielsen M</span>, <span class="author">Lewinski R</span>, <span class="author">Kuokkanen M</span>, <span class="author">Rasinpera H</span>, <span class="author">El-Shanti H</span>, <span class="author">Seo JK</span>, <span class="author">Alifrangis M</span>, <span class="author">Khalil IF</span>, <span class="author">Natah A</span>, <span class="author">Ali A</span>, <span class="author">Natah S</span>, <span class="author">Comas D</span>, <span class="author">Mehdi SQ</span>, <span class="author">Groop L</span>, <span class="author">Vestergaard EM</span>, <span class="author">Imtiaz F</span>, <span class="author">Rashed MS</span>, <span class="author">Meyer B</span>, <span class="author">Troelsen J</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Molecular Medicine, National Public Health Institute, Biomedicum Helsinki, 00251 Helsinki, Finland.</p><p class="abstract">The T(-13910) variant located in the enhancer element of the lactase (LCT) gene correlates perfectly with lactase persistence (LP) in Eurasian populations whereas the variant is almost nonexistent among Sub-Saharan African populations, showing high prevalence of LP. Here, we report identification of two new mutations among Saudis, also known for the high prevalence of LP. We confirmed the absence of the European T(-13910) and established two new mutations found as a compound allele: T/G(-13915) within the -13910 enhancer region and a synonymous SNP in the exon 17 of the MCM6 gene T/C(-3712), -3712 bp from the LCT gene. The compound allele is driven to a high prevalence among Middle East population(s). Our functional analyses in vitro showed that both SNPs of the compound allele, located 10 kb apart, are required for the enhancer effect, most probably mediated through the binding of the hepatic nuclear factor 1 alpha (HNF1 alpha). High selection coefficient (s) approximately 0.04 for LP phenotype was found for both T(-13910) and the compound allele. The European T(-13910) and the earlier identified East African G(-13907) LP allele share the same ancestral background and most likely the same history, probably related to the same cattle domestication event. In contrast, the compound Arab allele shows a different, highly divergent ancestral haplotype, suggesting that these two major global LP alleles have arisen independently, the latter perhaps in response to camel milk consumption. These results support the convergent evolution of the LP in diverse populations, most probably reflecting different histories of adaptation to milk culture.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">1</span>;<span class="pagerange">57-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18179885" rel="external">18179885</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2253962" rel="external">2253962</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2007.09.012" rel="external">10.1016/j.ajhg.2007.09.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nuclear envelope defects cause stem cell dysfunction in premature-aging mice.</h4>
<p class="authors"><span class="author">Espada J</span>, <span class="author">Varela I</span>, <span class="author">Flores I</span>, <span class="author">Ugalde AP</span>, <span class="author">Cadi&ntilde;anos J</span>, <span class="author">Pend&aacute;s AM</span>, <span class="author">Stewart CL</span>, <span class="author">Tryggvason K</span>, <span class="author">Blasco MA</span>, <span class="author">Freije JM</span> and <span class="author">L&oacute;pez-Ot&iacute;n C</span></p>
<p class="affiliation">Departamento de Bioqu&iacute;mica y Biolog&iacute;a Molecular, Facultad de Medicina, Instituto Universitario de Oncolog&iacute;a, Universidad de Oviedo, 33006 Oviedo, Spain.</p><p class="abstract">Nuclear lamina alterations occur in physiological aging and in premature aging syndromes. Because aging is also associated with abnormal stem cell homeostasis, we hypothesize that nuclear envelope alterations could have an important impact on stem cell compartments. To evaluate this hypothesis, we examined the number and functional competence of stem cells in Zmpste24-null progeroid mice, which exhibit nuclear lamina defects. We show that Zmpste24 deficiency causes an alteration in the number and proliferative capacity of epidermal stem cells. These changes are associated with an aberrant nuclear architecture of bulge cells and an increase in apoptosis of their supporting cells in the hair bulb region. These alterations are rescued in Zmpste24(-/-)Lmna(+/-) mutant mice, which do not manifest progeroid symptoms. We also report that molecular signaling pathways implicated in the regulation of stem cell behavior, such as Wnt and microphthalmia transcription factor, are altered in Zmpste24(-/-) mice. These findings establish a link between age-related nuclear envelope defects and stem cell dysfunction.</p>
<p class="publication"><span class="title">The Journal of cell biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">181</span>;<span class="number">1</span>;<span class="pagerange">27-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18378773" rel="external">18378773</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2287278" rel="external">2287278</a>; DOI: <a href="http://dx.doi.org/10.1083%2Fjcb.200801096" rel="external">10.1083/jcb.200801096</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evaluating the role of LPIN1 variation in insulin resistance, body weight, and human lipodystrophy in U.K. Populations.</h4>
<p class="authors"><span class="author">Fawcett KA</span>, <span class="author">Grimsey N</span>, <span class="author">Loos RJ</span>, <span class="author">Wheeler E</span>, <span class="author">Daly A</span>, <span class="author">Soos M</span>, <span class="author">Semple R</span>, <span class="author">Syddall H</span>, <span class="author">Cooper C</span>, <span class="author">Siniossoglou S</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Wareham NJ</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Metabolic Disease Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, U.K.</p><p class="abstract"><strong>Objective</strong>: Loss of lipin 1 activity causes lipodystrophy and insulin resistance in the fld mouse, and LPIN1 expression and common genetic variation were recently suggested to influence adiposity and insulin sensitivity in humans. We aimed to conduct a comprehensive association study to clarify the influence of common LPIN1 variation on adiposity and insulin sensitivity in U.K. populations and to examine the role of LPIN1 mutations in insulin resistance syndromes.</p><p class="abstract">Twenty-two single nucleotide polymorphisms tagging common LPIN1 variation were genotyped in Medical Research Council (MRC) Ely (n = 1,709) and Hertfordshire (n = 2,901) population-based cohorts. LPIN1 exons, exon/intron boundaries, and 3&#39; untranslated region were sequenced in 158 patients with idiopathic severe insulin resistance (including 23 lipodystrophic patients) and 48 control subjects.</p><p class="abstract"><strong>Results</strong>: We found no association between LPIN1 single nucleotide polymorphisms and fasting insulin but report a nominal association between rs13412852 and BMI (P = 0.042) in a meta-analysis of 8,504 samples from in-house and publicly available studies. Three rare nonsynonymous variants (A353T, R552K, and G582R) were detected in severely insulin-resistant patients. However, these did not cosegregate with disease in affected families, and Lipin1 protein expression and phosphorylation in patients with variants were indistinguishable from those in control subjects.</p><p class="abstract"><strong>Conclusions</strong>: Our data do not support a major effect of common LPIN1 variation on metabolic traits and suggest that mutations in LPIN1 are not a common cause of lipodystrophy in humans. The nominal associations with BMI and other metabolic traits in U.K. cohorts require replication in larger cohorts.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0000934(68341)</span>, <span class="grant">G0701446</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U147574221</span>, <span class="grant">MC_U147585824</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">U.1475.00.002.00001.01 (85824)</span>, <span class="grant">U.1475.00.004.00002.01(74221)</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">077016</span>, <span class="grant">078986</span>, <span class="grant">078986/Z/06/Z</span>, <span class="grant">080952</span>, <span class="grant">080952/Z/06/Z</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2008</span>;<span class="volume">57</span>;<span class="number">9</span>;<span class="pagerange">2527-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18591397">18591397</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2518506">2518506</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb08-0422">10.2337/db08-0422</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Collaborative genome-wide association analysis supports a role for ANK3 and CACNA1C in bipolar disorder.</h4>
<p class="authors"><span class="author">Ferreira MA</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Meng YA</span>, <span class="author">Jones IR</span>, <span class="author">Ruderfer DM</span>, <span class="author">Jones L</span>, <span class="author">Fan J</span>, <span class="author">Kirov G</span>, <span class="author">Perlis RH</span>, <span class="author">Green EK</span>, <span class="author">Smoller JW</span>, <span class="author">Grozeva D</span>, <span class="author">Stone J</span>, <span class="author">Nikolov I</span>, <span class="author">Chambert K</span>, <span class="author">Hamshere ML</span>, <span class="author">Nimgaonkar VL</span>, <span class="author">Moskvina V</span>, <span class="author">Thase ME</span>, <span class="author">Caesar S</span>, <span class="author">Sachs GS</span>, <span class="author">Franklin J</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Ardlie KG</span>, <span class="author">Gabriel SB</span>, <span class="author">Fraser C</span>, <span class="author">Blumenstiel B</span>, <span class="author">Defelice M</span>, <span class="author">Breen G</span>, <span class="author">Gill M</span>, <span class="author">Morris DW</span>, <span class="author">Elkin A</span>, <span class="author">Muir WJ</span>, <span class="author">McGhee KA</span>, <span class="author">Williamson R</span>, <span class="author">MacIntyre DJ</span>, <span class="author">MacLean AW</span>, <span class="author">St CD</span>, <span class="author">Robinson M</span>, <span class="author">Van Beck M</span>, <span class="author">Pereira AC</span>, <span class="author">Kandaswamy R</span>, <span class="author">McQuillin A</span>, <span class="author">Collier DA</span>, <span class="author">Bass NJ</span>, <span class="author">Young AH</span>, <span class="author">Lawrence J</span>, <span class="author">Ferrier IN</span>, <span class="author">Anjorin A</span>, <span class="author">Farmer A</span>, <span class="author">Curtis D</span>, <span class="author">Scolnick EM</span>, <span class="author">McGuffin P</span>, <span class="author">Daly MJ</span>, <span class="author">Corvin AP</span>, <span class="author">Holmans PA</span>, <span class="author">Blackwood DH</span>, <span class="author">Gurling HM</span>, <span class="author">Owen MJ</span>, <span class="author">Purcell SM</span>, <span class="author">Sklar P</span>, <span class="author">Craddock N</span> and <span class="author">Wellcome Trust Case Control Consortium</span></p>
<p class="affiliation">Department of Psychiatry, Massachusetts General Hospital, Boston, Massachusetts 02114, USA.</p><p class="abstract">To identify susceptibility loci for bipolar disorder, we tested 1.8 million variants in 4,387 cases and 6,209 controls and identified a region of strong association (rs10994336, P = 9.1 x 10(-9)) in ANK3 (ankyrin G). We also found further support for the previously reported CACNA1C (alpha 1C subunit of the L-type voltage-gated calcium channel; combined P = 7.0 x 10(-8), rs1006737). Our results suggest that ion channelopathies may be involved in the pathogenesis of bipolar disorder.</p>
<p class="grants">Funded by: Chief Scientist Office; Medical Research Council: <span class="grant">G0500791</span>, <span class="grant">G0701003</span>, <span class="grant">G9309834</span>, <span class="grant">G9623693N</span>; NCRR NIH HHS: <span class="grant">U54 RR020278</span>; NIMH NIH HHS: <span class="grant">MH062137</span>, <span class="grant">MH063445</span>, <span class="grant">MH067288</span>, <span class="grant">MH63420</span>, <span class="grant">N01MH80001</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">082371</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">9</span>;<span class="pagerange">1056-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18711365">18711365</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2703780">2703780</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.209">10.1038/ng.209</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The minimum information about a genome sequence (MIGS) specification.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Garrity G</span>, <span class="author">Gray T</span>, <span class="author">Morrison N</span>, <span class="author">Selengut J</span>, <span class="author">Sterk P</span>, <span class="author">Tatusova T</span>, <span class="author">Thomson N</span>, <span class="author">Allen MJ</span>, <span class="author">Angiuoli SV</span>, <span class="author">Ashburner M</span>, <span class="author">Axelrod N</span>, <span class="author">Baldauf S</span>, <span class="author">Ballard S</span>, <span class="author">Boore J</span>, <span class="author">Cochrane G</span>, <span class="author">Cole J</span>, <span class="author">Dawyndt P</span>, <span class="author">De Vos P</span>, <span class="author">DePamphilis C</span>, <span class="author">Edwards R</span>, <span class="author">Faruque N</span>, <span class="author">Feldman R</span>, <span class="author">Gilbert J</span>, <span class="author">Gilna P</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Goldstein P</span>, <span class="author">Guralnick R</span>, <span class="author">Haft D</span>, <span class="author">Hancock D</span>, <span class="author">Hermjakob H</span>, <span class="author">Hertz-Fowler C</span>, <span class="author">Hugenholtz P</span>, <span class="author">Joint I</span>, <span class="author">Kagan L</span>, <span class="author">Kane M</span>, <span class="author">Kennedy J</span>, <span class="author">Kowalchuk G</span>, <span class="author">Kottmann R</span>, <span class="author">Kolker E</span>, <span class="author">Kravitz S</span>, <span class="author">Kyrpides N</span>, <span class="author">Leebens-Mack J</span>, <span class="author">Lewis SE</span>, <span class="author">Li K</span>, <span class="author">Lister AL</span>, <span class="author">Lord P</span>, <span class="author">Maltsev N</span>, <span class="author">Markowitz V</span>, <span class="author">Martiny J</span>, <span class="author">Methe B</span>, <span class="author">Mizrachi I</span>, <span class="author">Moxon R</span>, <span class="author">Nelson K</span>, <span class="author">Parkhill J</span>, <span class="author">Proctor L</span>, <span class="author">White O</span>, <span class="author">Sansone SA</span>, <span class="author">Spiers A</span>, <span class="author">Stevens R</span>, <span class="author">Swift P</span>, <span class="author">Taylor C</span>, <span class="author">Tateno Y</span>, <span class="author">Tett A</span>, <span class="author">Turner S</span>, <span class="author">Ussery D</span>, <span class="author">Vaughan B</span>, <span class="author">Ward N</span>, <span class="author">Whetzel T</span>, <span class="author">San Gil I</span>, <span class="author">Wilson G</span> and <span class="author">Wipat A</span></p>
<p class="affiliation">Natural Environmental Research Council Centre for Ecology and Hydrology, Oxford OX1 3SR, UK. <a href="mailto:%64%66%69%65%6C%64%40%63%65%68.%61%63.%75%6B">&#100;&#102;&#105;&#101;&#108;&#100;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">With the quantity of genomic data increasing at an exponential rate, it is imperative that these data be captured electronically, in a standard format. Standardization activities must proceed within the auspices of open-access and international working bodies. To tackle the issues surrounding the development of better descriptions of genomic investigations, we have formed the Genomic Standards Consortium (GSC). Here, we introduce the minimum information about a genome sequence (MIGS) specification with the intent of promoting participation in its development and discussing the resources that will be required to develop improved mechanisms of metadata capture and exchange. As part of its wider goals, the GSC also supports improving the &#39;transparency&#39; of the information contained in existing genomic databases.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G8225539</span>; NHGRI NIH HHS: <span class="grant">U54 HG004028</span>; NLM NIH HHS: <span class="grant">Z99 LM999999</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">5</span>;<span class="pagerange">541-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18464787">18464787</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2409278">2409278</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt1360">10.1038/nbt1360</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting report: the fifth Genomic Standards Consortium (GSC) workshop.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Garrity GM</span>, <span class="author">Sansone SA</span>, <span class="author">Sterk P</span>, <span class="author">Gray T</span>, <span class="author">Kyrpides N</span>, <span class="author">Hirschman L</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Kottmann R</span>, <span class="author">Angiuoli S</span>, <span class="author">White O</span>, <span class="author">Dawyndt P</span>, <span class="author">Thomson N</span>, <span class="author">Gil IS</span>, <span class="author">Morrison N</span>, <span class="author">Tatusova T</span>, <span class="author">Mizrachi I</span>, <span class="author">Vaughan R</span>, <span class="author">Cochrane G</span>, <span class="author">Kagan L</span>, <span class="author">Murphy S</span>, <span class="author">Schriml L</span> and <span class="author">Genomic Standards Consortium</span></p>
<p class="affiliation">NERC Center for Ecology and Hydrology, Oxford, United Kingdom. <a href="mailto:%64%66%69%65%6C%64%40%63%65%68.%61%63.%75%6B">&#100;&#102;&#105;&#101;&#108;&#100;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This meeting report summarizes the proceedings of the fifth Genomic Standards Consortium (GSC) workshop held December 12-14, 2007, at the European Bioinformatics Institute (EBI), Cambridge, UK. This fifth workshop served as a milestone event in the evolution of the GSC (launched in September 2005); the key outcome of the workshop was the finalization of a stable version of the MIGS specification (v2.0) for publication. This accomplishment enables, and also in some cases necessitates, downstream activities, which are described in the multiauthor, consensus-driven articles in this special issue of OMICS produced as a direct result of the workshop. This report briefly summarizes the workshop and overviews the special issue. In particular, it aims to explain how the various GSC-led projects are working together to help this community achieve its stated mission of further standardizing the descriptions of genomes and metagenomes and implementing improved mechanisms of data exchange and integration to enable more accurate comparative analyses. Further information about the GSC and its range of activities can be found at http://gensc.org.</p>
<p class="publication"><span class="title">Omics : a journal of integrative biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">109-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18564915" rel="external">18564915</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fomi.2008.A3B3" rel="external">10.1089/omi.2008.A3B3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meeting report: the fourth Genomic Standards Consortium (GSC) workshop.</h4>
<p class="authors"><span class="author">Field D</span>, <span class="author">Gl&ouml;ckner FO</span>, <span class="author">Garrity GM</span>, <span class="author">Gray T</span>, <span class="author">Sterk P</span>, <span class="author">Cochrane G</span>, <span class="author">Vaughan R</span>, <span class="author">Kolker E</span>, <span class="author">Kottmann R</span>, <span class="author">Kyrpides N</span>, <span class="author">Angiuoli S</span>, <span class="author">Dawyndt P</span>, <span class="author">Guralnick R</span>, <span class="author">Goldstein P</span>, <span class="author">Hall N</span>, <span class="author">Hirschman L</span>, <span class="author">Kravitz S</span>, <span class="author">Lister AL</span>, <span class="author">Markowitz V</span>, <span class="author">Thomson N</span> and <span class="author">Whetzel T</span></p>
<p class="affiliation">NERC Centre for Ecology and Hydrology, Mansfield Road, Oxford, OX1 3SR United Kingdom. <a href="mailto:%64%66%69%65%6C%64%40%63%65%68.%61%63.%75%6B">&#100;&#102;&#105;&#101;&#108;&#100;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This meeting report summarizes the proceedings of the &quot;eGenomics: Cataloguing our Complete Genome Collection IV&quot; workshop held June 6-8, 2007, at the National Institute for Environmental eScience (NIEeS), Cambridge, United Kingdom. This fourth workshop of the Genomic Standards Consortium (GSC) was a mix of short presentations, strategy discussions, and technical sessions. Speakers provided progress reports on the development of the &quot;Minimum Information about a Genome Sequence&quot; (MIGS) specification and the closely integrated &quot;Minimum Information about a Metagenome Sequence&quot; (MIMS) specification. The key outcome of the workshop was consensus on the next version of the MIGS/MIMS specification (v1.2). This drove further definition and restructuring of the MIGS/MIMS XML schema (syntax). With respect to semantics, a term vetting group was established to ensure that terms are properly defined and submitted to the appropriate ontology projects. Perhaps the single most important outcome of the workshop was a proposal to move beyond the concept of &quot;minimum&quot; to create a far richer XML schema that would define a &quot;Genomic Contextual Data Markup Language&quot; (GCDML) suitable for wider semantic integration across databases. GCDML will contain not only curated information (e.g., compliant with MIGS/MIMS), but also be extended to include a variety of data processing and calculations. Further information about the Genomic Standards Consortium and its range of activities can be found at http://gensc.org.</p>
<p class="publication"><span class="title">Omics : a journal of integrative biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">101-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18564914" rel="external">18564914</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fomi.2008.0014" rel="external">10.1089/omi.2008.0014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Pfam protein families database.</h4>
<p class="authors"><span class="author">Finn RD</span>, <span class="author">Tate J</span>, <span class="author">Mistry J</span>, <span class="author">Coggill PC</span>, <span class="author">Sammut SJ</span>, <span class="author">Hotz HR</span>, <span class="author">Ceric G</span>, <span class="author">Forslund K</span>, <span class="author">Eddy SR</span>, <span class="author">Sonnhammer EL</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton Hall, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Pfam is a comprehensive collection of protein domains and families, represented as multiple sequence alignments and as profile hidden Markov models. The current release of Pfam (22.0) contains 9318 protein families. Pfam is now based not only on the UniProtKB sequence database, but also on NCBI GenPept and on sequences from selected metagenomics projects. Pfam is available on the web from the consortium members using a new, consistent and improved website design in the UK (http://pfam.sanger.ac.uk/), the USA (http://pfam.janelia.org/) and Sweden (http://pfam.sbc.su.se/), as well as from mirror sites in France (http://pfam.jouy.inra.fr/) and South Korea (http://pfam.ccbb.re.kr/).</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F010435/1</span>; Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D281-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18039703">18039703</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238907">2238907</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm960">10.1093/nar/gkm960</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic determinants of ulcerative colitis include the ECM1 locus and five loci implicated in Crohn&#39;s disease.</h4>
<p class="authors"><span class="author">Fisher SA</span>, <span class="author">Tremelling M</span>, <span class="author">Anderson CA</span>, <span class="author">Gwilliam R</span>, <span class="author">Bumpstead S</span>, <span class="author">Prescott NJ</span>, <span class="author">Nimmo ER</span>, <span class="author">Massey D</span>, <span class="author">Berzuini C</span>, <span class="author">Johnson C</span>, <span class="author">Barrett JC</span>, <span class="author">Cummings FR</span>, <span class="author">Drummond H</span>, <span class="author">Lees CW</span>, <span class="author">Onnie CM</span>, <span class="author">Hanson CE</span>, <span class="author">Blaszczyk K</span>, <span class="author">Inouye M</span>, <span class="author">Ewels P</span>, <span class="author">Ravindrarajah R</span>, <span class="author">Keniry A</span>, <span class="author">Hunt S</span>, <span class="author">Carter M</span>, <span class="author">Watkins N</span>, <span class="author">Ouwehand W</span>, <span class="author">Lewis CM</span>, <span class="author">Cardon L</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Lobo A</span>, <span class="author">Forbes A</span>, <span class="author">Sanderson J</span>, <span class="author">Jewell DP</span>, <span class="author">Mansfield JC</span>, <span class="author">Deloukas P</span>, <span class="author">Mathew CG</span>, <span class="author">Parkes M</span> and <span class="author">Satsangi J</span></p>
<p class="affiliation">Department of Medical and Molecular Genetics, King&#39;s College London School of Medicine, 8th Floor Guy&#39;s Tower, Guy&#39;s Hospital, London SE1 9RT, UK.</p><p class="abstract">We report results of a nonsynonymous SNP scan for ulcerative colitis and identify a previously unknown susceptibility locus at ECM1. We also show that several risk loci are common to ulcerative colitis and Crohn&#39;s disease (IL23R, IL12B, HLA, NKX2-3 and MST1), whereas autophagy genes ATG16L1 and IRGM, along with NOD2 (also known as CARD15), are specific for Crohn&#39;s disease. These data provide the first detailed illustration of the genetic relationship between these common inflammatory bowel diseases.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0400874</span>, <span class="grant">G0600329</span>, <span class="grant">G0800383</span>, <span class="grant">G0800759</span>, <span class="grant">G0802320</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U105260799</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">089120</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">6</span>;<span class="pagerange">710-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18438406">18438406</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2719289">2719289</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.145">10.1038/ng.145</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2008.</h4>
<p class="authors"><span class="author">Flicek P</span>, <span class="author">Aken BL</span>, <span class="author">Beal K</span>, <span class="author">Ballester B</span>, <span class="author">Caccamo M</span>, <span class="author">Chen Y</span>, <span class="author">Clarke L</span>, <span class="author">Coates G</span>, <span class="author">Cunningham F</span>, <span class="author">Cutts T</span>, <span class="author">Down T</span>, <span class="author">Dyer SC</span>, <span class="author">Eyre T</span>, <span class="author">Fitzgerald S</span>, <span class="author">Fernandez-Banet J</span>, <span class="author">Gr&auml;f S</span>, <span class="author">Haider S</span>, <span class="author">Hammond M</span>, <span class="author">Holland R</span>, <span class="author">Howe KL</span>, <span class="author">Howe K</span>, <span class="author">Johnson N</span>, <span class="author">Jenkinson A</span>, <span class="author">K&auml;h&auml;ri A</span>, <span class="author">Keefe D</span>, <span class="author">Kokocinski F</span>, <span class="author">Kulesha E</span>, <span class="author">Lawson D</span>, <span class="author">Longden I</span>, <span class="author">Megy K</span>, <span class="author">Meidl P</span>, <span class="author">Overduin B</span>, <span class="author">Parker A</span>, <span class="author">Pritchard B</span>, <span class="author">Prlic A</span>, <span class="author">Rice S</span>, <span class="author">Rios D</span>, <span class="author">Schuster M</span>, <span class="author">Sealy I</span>, <span class="author">Slater G</span>, <span class="author">Smedley D</span>, <span class="author">Spudich G</span>, <span class="author">Trevanion S</span>, <span class="author">Vilella AJ</span>, <span class="author">Vogel J</span>, <span class="author">White S</span>, <span class="author">Wood M</span>, <span class="author">Birney E</span>, <span class="author">Cox T</span>, <span class="author">Curwen V</span>, <span class="author">Durbin R</span>, <span class="author">Fernandez-Suarez XM</span>, <span class="author">Herrero J</span>, <span class="author">Hubbard TJ</span>, <span class="author">Kasprzyk A</span>, <span class="author">Proctor G</span>, <span class="author">Smith J</span>, <span class="author">Ureta-Vidal A</span> and <span class="author">Searle S</span></p>
<p class="affiliation">European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK. <a href="mailto:%66%6C%69%63%65%6B%40%65%62%69.%61%63.%75%6B">&#102;&#108;&#105;&#99;&#101;&#107;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl project (http://www.ensembl.org) is a comprehensive genome information system featuring an integrated set of genome annotation, databases and other information for chordate and selected model organism and disease vector genomes. As of release 47 (October 2007), Ensembl fully supports 35 species, with preliminary support for six additional species. New species in the past year include platypus and horse. Major additions and improvements to Ensembl since our previous report include extensive support for functional genomics data in the form of a specialized functional genomics database, genome-wide maps of protein-DNA interactions and the Ensembl regulatory build; support for customization of the Ensembl web interface through the addition of user accounts and user groups; and increased support for genome resequencing. We have also introduced new comparative genomics-based data mining options and report on the continued development of our software infrastructure.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E010768/1</span>, <span class="grant">BBE0116401</span>, <span class="grant">BBS/B/13438</span>, <span class="grant">BBS/B/13446</span>, <span class="grant">BBS/B/13462</span>, <span class="grant">BBS/B/13470</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D707-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18000006">18000006</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238821">2238821</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm988">10.1093/nar/gkm988</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Testing of diabetes-associated WFS1 polymorphisms in the Diabetes Prevention Program.</h4>
<p class="authors"><span class="author">Florez JC</span>, <span class="author">Jablonski KA</span>, <span class="author">McAteer J</span>, <span class="author">Sandhu MS</span>, <span class="author">Wareham NJ</span>, <span class="author">Barroso I</span>, <span class="author">Franks PW</span>, <span class="author">Altshuler D</span>, <span class="author">Knowler WC</span> and <span class="author">Diabetes Prevention Program Research Group</span></p>
<p class="affiliation">Center for Human Genetic Research, Massachusetts General Hospital, Boston, MA, USA. <a href="mailto:%6A%63%66%6C%6F%72%65%7A%40%70%61%72%74%6E%65%72%73.%6F%72%67">&#106;&#99;&#102;&#108;&#111;&#114;&#101;&#122;&#64;&#112;&#97;&#114;&#116;&#110;&#101;&#114;&#115;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Wolfram syndrome (diabetes insipidus, diabetes mellitus, optic atrophy and deafness) is caused by mutations in the WFS1 gene. Recently, single nucleotide polymorphisms (SNPs) in WFS1 have been reproducibly associated with type 2 diabetes. We therefore examined the effects of these variants on diabetes incidence and response to interventions in the Diabetes Prevention Program (DPP), in which a lifestyle intervention or metformin treatment was compared with placebo.</p><p class="abstract"><strong>Methods</strong>: We genotyped the WFS1 SNPs rs10010131, rs752854 and rs734312 (H611R) in 3,548 DPP participants and performed Cox regression analysis using genotype, intervention and their interactions as predictors of diabetes incidence. We also evaluated the effect of these SNPs on insulin resistance and beta cell function at 1 year.</p><p class="abstract"><strong>Results</strong>: Although none of the three SNPs was associated with diabetes incidence in the overall cohort, white homozygotes for the previously reported protective alleles appeared less likely to develop diabetes in the lifestyle arm. Examination of the publicly available Diabetes Genetics Initiative genome-wide association dataset revealed that rs10012946, which is in strong linkage disequilibrium with the three WFS1 SNPs (r(2)=0.88-1.0), was associated with type 2 diabetes (allelic odds ratio 0.85, 95% CI 0.75-0.97, p=0.026). In the DPP, we noted a trend towards increased insulin secretion in carriers of the protective variants, although for most SNPs this was seen as compensatory for the diminished insulin sensitivity.</p><p class="abstract">The previously reported protective effect of select WFS1 alleles may be magnified by a lifestyle intervention. These variants appear to confer an improvement in beta cell function.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>; NIDDK NIH HHS: <span class="grant">K23 DK65978-04</span>, <span class="grant">R01 DK072041-02</span>, <span class="grant">U01 DK048489</span>, <span class="grant">U01 DK048489-06</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2008</span>;<span class="volume">51</span>;<span class="number">3</span>;<span class="pagerange">451-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18060660">18060660</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2483955">2483955</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-007-0891-x">10.1007/s00125-007-0891-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Catalogue of Somatic Mutations in Cancer (COSMIC).</h4>
<p class="authors"><span class="author">Forbes SA</span>, <span class="author">Bhamra G</span>, <span class="author">Bamford S</span>, <span class="author">Dawson E</span>, <span class="author">Kok C</span>, <span class="author">Clements J</span>, <span class="author">Menzies A</span>, <span class="author">Teague JW</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">COSMIC is currently the most comprehensive global resource for information on somatic mutations in human cancer, combining curation of the scientific literature with tumor resequencing data from the Cancer Genome Project at the Sanger Institute, U.K. Almost 4800 genes and 250000 tumors have been examined, resulting in over 50000 mutations available for investigation. This information can be accessed in a number of ways, the most convenient being the Web-based system which allows detailed data mining, presenting the results in easily interpretable formats. This unit describes the graphical system in detail, elaborating an example walkthrough and the many ways that the resulting information can be thoroughly investigated by combining data, respecializing the query, or viewing the results in different ways. Alternate protocols overview the available precompiled data files available for download.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Current protocols in human genetics / editorial board, Jonathan L. Haines ... [et al.]</span>&nbsp;<span class="year">2008</span>;<span class="volume">Chapter 10</span>;<span class="pagerange">Unit 10.11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18428421">18428421</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705836">2705836</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471142905.hg1011s57">10.1002/0471142905.hg1011s57</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Off-pathway, oxygen-dependent thiamine radical in the Krebs cycle.</h4>
<p class="authors"><span class="author">Frank RA</span>, <span class="author">Kay CW</span>, <span class="author">Hirst J</span> and <span class="author">Luisi BF</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, 80 Tennis Court Road, Cambridge CB2 1GA, UK.</p><p class="abstract">The catalytic cofactor thiamine diphosphate is found in many enzymes of central metabolism and is essential in all extant forms of life. We demonstrate the presence of an oxygen-dependent free radical in the thiamine diphosphate-dependent Escherichia coli 2-oxoglutarate dehydrogenase, which is a key component of the tricarboxylic acid (Krebs) cycle. The radical was sufficiently long-lived to be trapped by freezing in liquid nitrogen, and its electronic structure was investigated by electron paramagnetic resonance (EPR) and electron-nuclear double resonance (ENDOR). Taken together, the spectroscopic results revealed a delocalized pi radical on the enamine-thiazolium intermediate within the enzyme active site. The radical is generated as an intermediate during substrate turnover by a side reaction with molecular oxygen, resulting in the continuous production of reactive oxygen species under aerobic conditions. This off-pathway reaction may account for metabolic dysfunction associated with several neurodegenerative diseases. The possibility that the on-pathway reaction may proceed via a radical mechanism is discussed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of the American Chemical Society</span>&nbsp;<span class="year">2008</span>;<span class="volume">130</span>;<span class="number">5</span>;<span class="pagerange">1662-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18183975" rel="external">18183975</a>; DOI: <a href="http://dx.doi.org/10.1021%2Fja076468k" rel="external">10.1021/ja076468k</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection, imputation, and association analysis of small deletions and null alleles on oligonucleotide arrays.</h4>
<p class="authors"><span class="author">Franke L</span>, <span class="author">de Kovel CG</span>, <span class="author">Aulchenko YS</span>, <span class="author">Trynka G</span>, <span class="author">Zhernakova A</span>, <span class="author">Hunt KA</span>, <span class="author">Blauw HM</span>, <span class="author">van den Berg LH</span>, <span class="author">Ophoff R</span>, <span class="author">Deloukas P</span>, <span class="author">van Heel DA</span> and <span class="author">Wijmenga C</span></p>
<p class="affiliation">Complex Genetics Section, DBG-Department of Medical Genetics, University Medical Centre Utrecht, 3584 CG Utrecht, The Netherlands.</p><p class="abstract">Copy-number variation (CNV) is a major contributor to human genetic variation. Recently, CNV associations with human disease have been reported. Many genome-wide association (GWA) studies in complex diseases have been performed with sets of biallelic single-nucleotide polymorphisms (SNPs), but the available CNV methods are still limited. We present a new method (TriTyper) that can infer genotypes in case-control data sets for deletion CNVs, or SNPs with an extra, untyped allele at a high-resolution single SNP level. By accounting for linkage disequilibrium (LD), as well as intensity data, calling accuracy is improved. Analysis of 3102 unrelated individuals with European descent, genotyped with Illumina Infinium BeadChips, resulted in the identification of 1880 SNPs with a common untyped allele, and these SNPs are in strong LD with neighboring biallelic SNPs. Simulations indicate our method has superior power to detect associations compared to biallelic SNPs that are in LD with these SNPs, yet without increasing type I errors, as shown in a GWA analysis in celiac disease. Genotypes for 1204 triallelic SNPs could be fully imputed, with only biallelic-genotype calls, permitting association analysis of these SNPs in many published data sets. We estimate that 682 of the 1655 unique loci reflect deletions; this is on average 99 deletions per individual, four times greater than those detected by other methods. Whereas the identified loci are strongly enriched for known deletions, 61% have not been reported before. Genes overlapping with these loci more often have paralogs (p = 0.006) and biologically interact with fewer genes than expected (p = 0.004).</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">GR068094MA</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">6</span>;<span class="pagerange">1316-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18519066" rel="external">18519066</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427186" rel="external">2427186</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.05.008" rel="external">10.1016/j.ajhg.2008.05.008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of the association between variants in WFS1 and risk of type 2 diabetes in European populations.</h4>
<p class="authors"><span class="author">Franks PW</span>, <span class="author">Rolandsson O</span>, <span class="author">Debenham SL</span>, <span class="author">Fawcett KA</span>, <span class="author">Payne F</span>, <span class="author">Dina C</span>, <span class="author">Froguel P</span>, <span class="author">Mohlke KL</span>, <span class="author">Willer C</span>, <span class="author">Olsson T</span>, <span class="author">Wareham NJ</span>, <span class="author">Hallmans G</span>, <span class="author">Barroso I</span> and <span class="author">Sandhu MS</span></p>
<p class="affiliation">Department of Public Health and Clinical Medicine, Ume&aring; University Hospital, Ume&aring;, Sweden. <a href="mailto:%70%61%75%6C.%66%72%61%6E%6B%73%40%6D%65%64%69%63%69%6E.%75%6D%75.%73%65">&#112;&#97;&#117;&#108;&#46;&#102;&#114;&#97;&#110;&#107;&#115;&#64;&#109;&#101;&#100;&#105;&#99;&#105;&#110;&#46;&#117;&#109;&#117;&#46;&#115;&#101;</a></p><p class="abstract">Mutations at the gene encoding wolframin (WFS1) cause Wolfram syndrome, a rare neurological condition. Associations between single nucleotide polymorphisms (SNPs) at WFS1 and type 2 diabetes have recently been reported. Thus, our aim was to replicate those associations in a northern Swedish case-control study of type 2 diabetes. We also performed a meta-analysis of published and previously unpublished data from Sweden, Finland and France, to obtain updated summary effect estimates.</p><p class="abstract"><strong>Methods</strong>: Four WFS1 SNPs (rs10010131, rs6446482, rs752854 and rs734312 [H611R]) were genotyped in a type 2 diabetes case-control study (n = 1,296/1,412) of Swedish adults. Logistic regression was used to assess the association between each WFS1 SNP and type 2 diabetes, following adjustment for age, sex and BMI. We then performed a meta-analysis of 11 studies of type 2 diabetes, comprising up to 14,139 patients and 16,109 controls, to obtain a summary effect estimate for the WFS1 variants.</p><p class="abstract"><strong>Results</strong>: In the northern Swedish study, the minor allele at rs752854 was associated with reduced type 2 diabetes risk [odds ratio (OR) 0.85, 95% CI 0.75-0.96, p=0.010]. Borderline statistical associations were observed for the remaining SNPs. The meta-analysis of the four independent replication studies for SNP rs10010131 and correlated variants showed evidence for statistical association (OR 0.87, 95% CI 0.82-0.93, p=4.5 x 10(-5)). In an updated meta-analysis of all 11 studies, strong evidence of statistical association was also observed (OR 0.89, 95% CI 0.86-0.92; p=4.9 x 10(-11)).</p><p class="abstract">In this study of WFS1 variants and type 2 diabetes risk, we have replicated the previously reported associations between SNPs at this locus and the risk of type 2 diabetes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>; NIDDK NIH HHS: <span class="grant">DK62370</span>, <span class="grant">DK72193</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2008</span>;<span class="volume">51</span>;<span class="number">3</span>;<span class="pagerange">458-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18040659">18040659</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2670195">2670195</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-007-0887-6">10.1007/s00125-007-0887-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Submicroscopic duplications of the hydroxysteroid dehydrogenase HSD17B10 and the E3 ubiquitin ligase HUWE1 are associated with mental retardation.</h4>
<p class="authors"><span class="author">Froyen G</span>, <span class="author">Corbett M</span>, <span class="author">Vandewalle J</span>, <span class="author">Jarvela I</span>, <span class="author">Lawrence O</span>, <span class="author">Meldrum C</span>, <span class="author">Bauters M</span>, <span class="author">Govaerts K</span>, <span class="author">Vandeleur L</span>, <span class="author">Van Esch H</span>, <span class="author">Chelly J</span>, <span class="author">Sanlaville D</span>, <span class="author">van Bokhoven H</span>, <span class="author">Ropers HH</span>, <span class="author">Laumonnier F</span>, <span class="author">Ranieri E</span>, <span class="author">Schwartz CE</span>, <span class="author">Abidi F</span>, <span class="author">Tarpey PS</span>, <span class="author">Futreal PA</span>, <span class="author">Whibley A</span>, <span class="author">Raymond FL</span>, <span class="author">Stratton MR</span>, <span class="author">Fryns JP</span>, <span class="author">Scott R</span>, <span class="author">Peippo M</span>, <span class="author">Sipponen M</span>, <span class="author">Partington M</span>, <span class="author">Mowat D</span>, <span class="author">Field M</span>, <span class="author">Hackett A</span>, <span class="author">Marynen P</span>, <span class="author">Turner G</span> and <span class="author">G&eacute;cz J</span></p>
<p class="affiliation">Human Genome Laboratory, Department for Molecular and Developmental Genetics, VIB, B-3000 Leuven, Belgium. <a href="mailto:%67%75%79.%66%72%6F%79%65%6E%40%6D%65%64.%6B%75%6C%65%75%76%65%6E.%62%65">&#103;&#117;&#121;&#46;&#102;&#114;&#111;&#121;&#101;&#110;&#64;&#109;&#101;&#100;&#46;&#107;&#117;&#108;&#101;&#117;&#118;&#101;&#110;&#46;&#98;&#101;</a></p><p class="abstract">Submicroscopic copy-number imbalances contribute significantly to the genetic etiology of human disease. Here, we report a novel microduplication hot spot at Xp11.22 identified in six unrelated families with predominantly nonsyndromic XLMR. All duplications segregate with the disease, including the large families MRX17 and MRX31. The minimal, commonly duplicated region contains three genes: RIBC1, HSD17B10, and HUWE1. RIBC1 could be excluded on the basis of its absence of expression in the brain and because it escapes X inactivation in females. For the other genes, expression array and quantitative PCR analysis in patient cell lines compared to controls showed a significant upregulation of HSD17B10 and HUWE1 as well as several important genes in their molecular pathways. Loss-of-function mutations of HSD17B10 have previously been associated with progressive neurological disease and XLMR. The E3 ubiquitin ligase HUWE1 has been implicated in TP53-associated regulation of the neuronal cell cycle. Here, we also report segregating sequence changes of highly conserved residues in HUWE1 in three XLMR families; these changes are possibly associated with the phenotype. Our findings demonstrate that an increased gene dosage of HSD17B10, HUWE1, or both contribute to the etiology of XLMR and suggest that point mutations in HUWE1 are associated with this disease too.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD26202</span>; NINDS NIH HHS: <span class="grant">NS31564</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">2</span>;<span class="pagerange">432-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18252223" rel="external">18252223</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2426915" rel="external">2426915</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2007.11.002" rel="external">10.1016/j.ajhg.2007.11.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variation in the ABO glycosyltransferase is associated with susceptibility to severe Plasmodium falciparum malaria.</h4>
<p class="authors"><span class="author">Fry AE</span>, <span class="author">Griffiths MJ</span>, <span class="author">Auburn S</span>, <span class="author">Diakite M</span>, <span class="author">Forton JT</span>, <span class="author">Green A</span>, <span class="author">Richardson A</span>, <span class="author">Wilson J</span>, <span class="author">Jallow M</span>, <span class="author">Sisay-Joof F</span>, <span class="author">Pinder M</span>, <span class="author">Peshu N</span>, <span class="author">Williams TN</span>, <span class="author">Marsh K</span>, <span class="author">Molyneux ME</span>, <span class="author">Taylor TE</span>, <span class="author">Rockett KA</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">The Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7BN, UK. <a href="mailto:%61%66%72%79%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#97;&#102;&#114;&#121;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">There is growing epidemiological and molecular evidence that ABO blood group affects host susceptibility to severe Plasmodium falciparum infection. The high frequency of common ABO alleles means that even modest differences in susceptibility could have a significant impact on the health of people living in malaria endemic regions. We performed an association study, the first to utilize key molecular genetic variation underlying the ABO system, genotyping &gt;9000 individuals across three African populations. Using population- and family-based tests, we demonstrated that alleles producing functional ABO enzymes are associated with greater risk of severe malaria phenotypes (particularly malarial anemia) in comparison with the frameshift deletion underlying blood group O: case-control allelic odds ratio (OR), 1.2; 95% confidence interval (CI), 1.09-1.32; P = 0.0003; family-studies allelic OR, 1.19; 95% CI, 1.08-1.32; P = 0.001; pooled across all studies allelic OR, 1.18; 95% CI, 1.11-1.26; P = 2 x 10(-7). We found suggestive evidence of a parent-of-origin effect at the ABO locus by analyzing the family trios. Non-O haplotypes inherited from mothers, but not fathers, are significantly associated with severe malaria (likelihood ratio test of Weinberg, P = 0.046). Finally, we used HapMap data to demonstrate a region of low F(ST) (-0.001) between the three main HapMap population groups across the ABO locus, an outlier in the empirical distribution of F(ST) across chromosome 9 (approximately 99.5-99.9th centile). This low F(ST) region may be a signal of long-standing balancing selection at the ABO locus, caused by multiple infectious pathogens including P. falciparum.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230(77610)</span>; Wellcome Trust: <span class="grant">074586</span>, <span class="grant">076934</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">4</span>;<span class="pagerange">567-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18003641" rel="external">18003641</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2657867" rel="external">2657867</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddm331" rel="external">10.1093/hmg/ddm331</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Advantages of q-PCR as a method of screening for gene targeting in mammalian cells using conventional and whole BAC-based constructs.</h4>
<p class="authors"><span class="author">G&oacute;mez-Rodr&iacute;guez J</span>, <span class="author">Washington V</span>, <span class="author">Cheng J</span>, <span class="author">Dutra A</span>, <span class="author">Pak E</span>, <span class="author">Liu P</span>, <span class="author">McVicar DW</span> and <span class="author">Schwartzberg PL</span></p>
<p class="affiliation">Genetic Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA.</p><p class="abstract">We evaluate here the use of real-time quantitative PCR (q-PCR) as a method for screening for homologous recombinants generated in mammalian cells from either conventional gene-targeting constructs or whole BAC-based constructs. Using gene-targeted events at different loci, we show that q-PCR is a highly sensitive and accurate method for screening for conventional gene targeting that can reduce the number of clones requiring follow-up screening by Southern blotting. We further compared q-PCR to fluorescent in situ hybridization (FISH) for the detection of gene-targeting events using full-length BAC-based constructs designed to introduce mutations either into one gene or simultaneously into two adjacent genes. We find that although BAC-based constructs appeared to have high rates of homologous recombination when evaluated by FISH, screening by FISH was prone to false positives that were detected by q-PCR. Our results demonstrate the utility of q-PCR as a screening tool for gene targeting and further highlight potential problems with the use of whole BAC-based constructs for homologous recombination.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">18</span>;<span class="pagerange">e117</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18710883" rel="external">18710883</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2566865" rel="external">2566865</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkn523" rel="external">10.1093/nar/gkn523</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SrfB, a member of the Serum Response Factor family of transcription factors, regulates starvation response and early development in Dictyostelium.</h4>
<p class="authors"><span class="author">Galardi-Castilla M</span>, <span class="author">Pergolizzi B</span>, <span class="author">Bloomfield G</span>, <span class="author">Skelton J</span>, <span class="author">Ivens A</span>, <span class="author">Kay RR</span>, <span class="author">Bozzaro S</span> and <span class="author">Sastre L</span></p>
<p class="affiliation">Instituto de Investigaciones Biom&eacute;dicas CSIC/UAM. Arturo Duperier, 4. 28029 Madrid, Spain.</p><p class="abstract">The Serum Response Factor (SRF) is an important regulator of cell proliferation and differentiation. Dictyostelium discoideum srfB gene codes for an SRF homologue and is expressed in vegetative cells and during development under the control of three alternative promoters, which show different cell-type specific patterns of expression. The two more proximal promoters directed gene transcription in prestalk AB, stalk and lower-cup cells. The generation of a strain where the srfB gene has been interrupted (srfB(-)) has shown that this gene is required for regulation of actin-cytoskeleton-related functions, such as cytokinesis and macropinocytosis. The mutant failed to develop well in suspension, but could be rescued by cAMP pulsing, suggesting a defect in cAMP signaling. srfB(-) cells showed impaired chemotaxis to cAMP and defective lateral pseudopodium inhibition. Nevertheless, srfB(-) cells aggregated on agar plates and nitrocellulose filters 2 h earlier than wild type cells, and completed development, showing an increased tendency to form slug structures. Analysis of wild type and srfB(-) strains detected significant differences in the regulation of gene expression upon starvation. Genes coding for lysosomal and ribosomal proteins, developmentally-regulated genes, and some genes coding for proteins involved in cytoskeleton regulation were deregulated during the first stages of development.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">316</span>;<span class="number">2</span>;<span class="pagerange">260-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18339368">18339368</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2008.01.026">10.1016/j.ydbio.2008.01.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ES cell pluripotency and germ-layer formation require the SWI/SNF chromatin remodeling component BAF250a.</h4>
<p class="authors"><span class="author">Gao X</span>, <span class="author">Tate P</span>, <span class="author">Hu P</span>, <span class="author">Tjian R</span>, <span class="author">Skarnes WC</span> and <span class="author">Wang Z</span></p>
<p class="affiliation">Cardiovascular Research Center, Massachusetts General Hospital, Harvard Medical School, Richard Simches Research Center, 185 Cambridge Street, Boston, MA 02114, USA.</p><p class="abstract">ATP-dependent chromatin remodeling complexes are a notable group of epigenetic modifiers that use the energy of ATP hydrolysis to change the structure of chromatin, thereby altering its accessibility to nuclear factors. BAF250a (ARID1a) is a unique and defining subunit of the BAF chromatin remodeling complex with the potential to facilitate chromosome alterations critical during development. Our studies show that ablation of BAF250a in early mouse embryos results in developmental arrest (about embryonic day 6.5) and absence of the mesodermal layer, indicating its critical role in early germ-layer formation. Moreover, BAF250a deficiency compromises ES cell pluripotency, severely inhibits self-renewal, and promotes differentiation into primitive endoderm-like cells under normal feeder-free culture conditions. Interestingly, this phenotype can be partially rescued by the presence of embryonic fibroblast cells. DNA microarray, immunostaining, and RNA analyses revealed that BAF250a-mediated chromatin remodeling contributes to the proper expression of numerous genes involved in ES cell self-renewal, including Sox2, Utf1, and Oct4. Furthermore, the pluripotency defects in BAF250a mutant ES cells appear to be cell lineage-specific. For example, embryoid body-based analyses demonstrated that BAF250a-ablated stem cells are defective in differentiating into fully functional mesoderm-derived cardiomyocytes and adipocytes but are capable of differentiating into ectoderm-derived neurons. Our results suggest that BAF250a is a key component of the gene regulatory machinery in ES cells controlling self-renewal, differentiation, and cell lineage decisions.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">18</span>;<span class="pagerange">6656-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18448678">18448678</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2373334">2373334</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0801802105">10.1073/pnas.0801802105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation of miRNA target sequences during human evolution.</h4>
<p class="authors"><span class="author">Gardner PP</span> and <span class="author">Vinther J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">It has long-been hypothesized that changes in non-protein-coding genes and the regulatory sequences controlling expression could undergo positive selection. Here we identify 402 putative microRNA (miRNA) target sequences that have been mutated specifically in the human lineage and show that genes containing such deletions are more highly expressed than their mouse orthologs. Our findings indicate that some miRNA target mutations are fixed by positive selection and might have been involved in the evolution of human-specific traits.</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">6</span>;<span class="pagerange">262-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18472179" rel="external">18472179</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2008.03.009" rel="external">10.1016/j.tig.2008.03.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Apheresis donors and platelet function: inherent platelet responsiveness influences platelet quality.</h4>
<p class="authors"><span class="author">Garner SF</span>, <span class="author">Jones CI</span>, <span class="author">Stephens J</span>, <span class="author">Burns P</span>, <span class="author">Walton J</span>, <span class="author">Bernard A</span>, <span class="author">Angenent W</span>, <span class="author">Ouwehand WH</span>, <span class="author">Goodall AH</span> and <span class="author">BLOODOMICS Consortium</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: Process-induced platelet (PLT) activation occurs with all production methods, including apheresis. Recent studies have highlighted the range and consistence of interindividual variation in the PLT response, but little is known about the contribution of a donors&#39; inherent PLT responsiveness to the activation state of the apheresis PLTs or the effect of frequent apheresis on donors&#39; PLTs.</p><p class="abstract">The relationship between the donors&#39; PLT response on the apheresis PLTs was studied in 47 individuals selected as having PLTs with inherently low, intermediate, or high responsiveness. Whole-blood flow cytometry was used to measure PLT activation (levels of bound fibrinogen) before donation and in the apheresis PLTs. The effects of regular apheresis on the activation status of donors&#39; PLTs were studied by comparing the in vivo activation status of PLTs from apheresis (n = 349) and whole-blood donors (n = 157), before donation. The effect of apheresis per se on PLT activation was measured in 10 apheresis donors before and after donation.</p><p class="abstract"><strong>Results</strong>: The level of PLT activation in the apheresis packs was generally higher than in the donor, and the most activated PLTs were from high-responder donors. There was no significant difference in PLT activation before donation between the apheresis and whole-blood donors (p = 0.697), and there was no consistent evidence of activation in the donors immediately after apheresis.</p><p class="abstract"><strong>Conclusion</strong>: The most activated apheresis PLTs were obtained from donors with more responsive PLTs. Regular apheresis, however, does not lead to PLT activation in the donors.</p>
<p class="publication"><span class="title">Transfusion</span>&nbsp;<span class="year">2008</span>;<span class="volume">48</span>;<span class="number">4</span>;<span class="pagerange">673-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18194381">18194381</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1537-2995.2007.01591.x">10.1111/j.1537-2995.2007.01591.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Gene Ontology project in 2008.</h4>
<p class="authors"><span class="author">Gene Ontology Consortium</span></p><p class="abstract">The Gene Ontology (GO) project (http://www.geneontology.org/) provides a set of structured, controlled vocabularies for community use in annotating genes, gene products and sequences (also see http://www.sequenceontology.org/). The ontologies have been extended and refined for several biological areas, and improvements to the structure of the ontologies have been implemented. To improve the quantity and quality of gene product annotations available from its public repository, the GO Consortium has launched a focused effort to provide comprehensive and detailed annotation of orthologous genes across a number of &#39;reference&#39; genomes, including human and several key model organisms. Software developments include two releases of the ontology-editing tool OBO-Edit, and improvements to the AmiGO browser interface.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG02273</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D440-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17984083" rel="external">17984083</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238979" rel="external">2238979</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm883" rel="external">10.1093/nar/gkm883</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The H-Invitational Database (H-InvDB), a comprehensive annotation resource for human genes and transcripts.</h4>
<p class="authors"><span class="author">Genome Information Integration Project And H-Invitational 2</span>, <span class="author">Yamasaki C</span>, <span class="author">Murakami K</span>, <span class="author">Fujii Y</span>, <span class="author">Sato Y</span>, <span class="author">Harada E</span>, <span class="author">Takeda J</span>, <span class="author">Taniya T</span>, <span class="author">Sakate R</span>, <span class="author">Kikugawa S</span>, <span class="author">Shimada M</span>, <span class="author">Tanino M</span>, <span class="author">Koyanagi KO</span>, <span class="author">Barrero RA</span>, <span class="author">Gough C</span>, <span class="author">Chun HW</span>, <span class="author">Habara T</span>, <span class="author">Hanaoka H</span>, <span class="author">Hayakawa Y</span>, <span class="author">Hilton PB</span>, <span class="author">Kaneko Y</span>, <span class="author">Kanno M</span>, <span class="author">Kawahara Y</span>, <span class="author">Kawamura T</span>, <span class="author">Matsuya A</span>, <span class="author">Nagata N</span>, <span class="author">Nishikata K</span>, <span class="author">Noda AO</span>, <span class="author">Nurimoto S</span>, <span class="author">Saichi N</span>, <span class="author">Sakai H</span>, <span class="author">Sanbonmatsu R</span>, <span class="author">Shiba R</span>, <span class="author">Suzuki M</span>, <span class="author">Takabayashi K</span>, <span class="author">Takahashi A</span>, <span class="author">Tamura T</span>, <span class="author">Tanaka M</span>, <span class="author">Tanaka S</span>, <span class="author">Todokoro F</span>, <span class="author">Yamaguchi K</span>, <span class="author">Yamamoto N</span>, <span class="author">Okido T</span>, <span class="author">Mashima J</span>, <span class="author">Hashizume A</span>, <span class="author">Jin L</span>, <span class="author">Lee KB</span>, <span class="author">Lin YC</span>, <span class="author">Nozaki A</span>, <span class="author">Sakai K</span>, <span class="author">Tada M</span>, <span class="author">Miyazaki S</span>, <span class="author">Makino T</span>, <span class="author">Ohyanagi H</span>, <span class="author">Osato N</span>, <span class="author">Tanaka N</span>, <span class="author">Suzuki Y</span>, <span class="author">Ikeo K</span>, <span class="author">Saitou N</span>, <span class="author">Sugawara H</span>, <span class="author">O&#39;Donovan C</span>, <span class="author">Kulikova T</span>, <span class="author">Whitfield E</span>, <span class="author">Halligan B</span>, <span class="author">Shimoyama M</span>, <span class="author">Twigger S</span>, <span class="author">Yura K</span>, <span class="author">Kimura K</span>, <span class="author">Yasuda T</span>, <span class="author">Nishikawa T</span>, <span class="author">Akiyama Y</span>, <span class="author">Motono C</span>, <span class="author">Mukai Y</span>, <span class="author">Nagasaki H</span>, <span class="author">Suwa M</span>, <span class="author">Horton P</span>, <span class="author">Kikuno R</span>, <span class="author">Ohara O</span>, <span class="author">Lancet D</span>, <span class="author">Eveno E</span>, <span class="author">Graudens E</span>, <span class="author">Imbeaud S</span>, <span class="author">Debily MA</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Amid C</span>, <span class="author">Han M</span>, <span class="author">Osanger A</span>, <span class="author">Endo T</span>, <span class="author">Thomas MA</span>, <span class="author">Hirakawa M</span>, <span class="author">Makalowski W</span>, <span class="author">Nakao M</span>, <span class="author">Kim NS</span>, <span class="author">Yoo HS</span>, <span class="author">De Souza SJ</span>, <span class="author">Bonaldo Mde F</span>, <span class="author">Niimura Y</span>, <span class="author">Kuryshev V</span>, <span class="author">Schupp I</span>, <span class="author">Wiemann S</span>, <span class="author">Bellgard M</span>, <span class="author">Shionyu M</span>, <span class="author">Jia L</span>, <span class="author">Thierry-Mieg D</span>, <span class="author">Thierry-Mieg J</span>, <span class="author">Wagner L</span>, <span class="author">Zhang Q</span>, <span class="author">Go M</span>, <span class="author">Minoshima S</span>, <span class="author">Ohtsubo M</span>, <span class="author">Hanada K</span>, <span class="author">Tonellato P</span>, <span class="author">Isogai T</span>, <span class="author">Zhang J</span>, <span class="author">Lenhard B</span>, <span class="author">Kim S</span>, <span class="author">Chen Z</span>, <span class="author">Hinz U</span>, <span class="author">Estreicher A</span>, <span class="author">Nakai K</span>, <span class="author">Makalowska I</span>, <span class="author">Hide W</span>, <span class="author">Tiffin N</span>, <span class="author">Wilming L</span>, <span class="author">Chakraborty R</span>, <span class="author">Soares MB</span>, <span class="author">Chiusano ML</span>, <span class="author">Suzuki Y</span>, <span class="author">Auffray C</span>, <span class="author">Yamaguchi-Kabata Y</span>, <span class="author">Itoh T</span>, <span class="author">Hishiki T</span>, <span class="author">Fukuchi S</span>, <span class="author">Nishikawa K</span>, <span class="author">Sugano S</span>, <span class="author">Nomura N</span>, <span class="author">Tateno Y</span>, <span class="author">Imanishi T</span> and <span class="author">Gojobori T</span></p>
<p class="affiliation">Japan Biological Information Research Center, Japan Biological Informatics Consortium, Japan.</p><p class="abstract">Here we report the new features and improvements in our latest release of the H-Invitational Database (H-InvDB; http://www.h-invitational.jp/), a comprehensive annotation resource for human genes and transcripts. H-InvDB, originally developed as an integrated database of the human transcriptome based on extensive annotation of large sets of full-length cDNA (FLcDNA) clones, now provides annotation for 120 558 human mRNAs extracted from the International Nucleotide Sequence Databases (INSD), in addition to 54 978 human FLcDNAs, in the latest release H-InvDB_4.6. We mapped those human transcripts onto the human genome sequences (NCBI build 36.1) and determined 34 699 human gene clusters, which could define 34 057 (98.1%) protein-coding and 642 (1.9%) non-protein-coding loci; 858 (2.5%) transcribed loci overlapped with predicted pseudogenes. For all these transcripts and genes, we provide comprehensive annotation including gene structures, gene functions, alternative splicing variants, functional non-protein-coding RNAs, functional domains, predicted sub cellular localizations, metabolic pathways, predictions of protein 3D structure, mapping of SNPs and microsatellite repeat motifs, co-localization with orphan diseases, gene expression profiles, orthologous genes, protein-protein interactions (PPI) and annotation for gene families. The current H-InvDB annotation resources consist of two main views: Transcript view and Locus view and eight sub-databases: the DiseaseInfo Viewer, H-ANGEL, the Clustering Viewer, G-integra, the TOPO Viewer, Evola, the PPI view and the Gene family/group.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01 HL064541</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D793-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18089548">18089548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238988">2238988</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm999">10.1093/nar/gkm999</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SLC9A6 mutations cause X-linked mental retardation, microcephaly, epilepsy, and ataxia, a phenotype mimicking Angelman syndrome.</h4>
<p class="authors"><span class="author">Gilfillan GD</span>, <span class="author">Selmer KK</span>, <span class="author">Roxrud I</span>, <span class="author">Smith R</span>, <span class="author">Kyllerman M</span>, <span class="author">Eiklid K</span>, <span class="author">Kroken M</span>, <span class="author">Mattingsdal M</span>, <span class="author">Egeland T</span>, <span class="author">Stenmark H</span>, <span class="author">Sj&oslash;holm H</span>, <span class="author">Server A</span>, <span class="author">Samuelsson L</span>, <span class="author">Christianson A</span>, <span class="author">Tarpey P</span>, <span class="author">Whibley A</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Teague J</span>, <span class="author">Edkins S</span>, <span class="author">Gecz J</span>, <span class="author">Turner G</span>, <span class="author">Raymond FL</span>, <span class="author">Schwartz C</span>, <span class="author">Stevenson RE</span>, <span class="author">Undlien DE</span> and <span class="author">Str&oslash;mme P</span></p>
<p class="affiliation">Department of Medical Genetics, Ullev&aring;l University Hospital, NO-0407 Oslo, Norway.</p><p class="abstract">Linkage analysis and DNA sequencing in a family exhibiting an X-linked mental retardation (XLMR) syndrome, characterized by microcephaly, epilepsy, ataxia, and absent speech and resembling Angelman syndrome, identified a deletion in the SLC9A6 gene encoding the Na(+)/H(+) exchanger NHE6. Subsequently, other mutations were found in a male with mental retardation (MR) who had been investigated for Angelman syndrome and in two XLMR families with epilepsy and ataxia, including the family designated as having Christianson syndrome. Therefore, mutations in SLC9A6 cause X-linked mental retardation. Additionally, males with findings suggestive of unexplained Angelman syndrome should be considered as potential candidates for SLC9A6 mutations.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD2606</span>; NINDS NIH HHS: <span class="grant">NS31564</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">4</span>;<span class="pagerange">1003-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18342287" rel="external">18342287</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427207" rel="external">2427207</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.01.013" rel="external">10.1016/j.ajhg.2008.01.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phylogenomics of the dog and fox family (Canidae, Carnivora) revealed by chromosome painting.</h4>
<p class="authors"><span class="author">Graphodatsky AS</span>, <span class="author">Perelman PL</span>, <span class="author">Sokolovskaya NV</span>, <span class="author">Beklemisheva VR</span>, <span class="author">Serdukova NA</span>, <span class="author">Dobigny G</span>, <span class="author">O&#39;Brien SJ</span>, <span class="author">Ferguson-Smith MA</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS, Novosibirsk, 630090, Russia. <a href="mailto:%67%72%61%66%40%62%69%6F%6E%65%74.%6E%73%63.%72%75">&#103;&#114;&#97;&#102;&#64;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;</a></p><p class="abstract">Canid species (dogs and foxes) have highly rearranged karyotypes and thus represent a challenge for conventional comparative cytogenetic studies. Among them, the domestic dog is one of the best-mapped species in mammals, constituting an ideal reference genome for comparative genomic study. Here we report the results of genome-wide comparative mapping of dog chromosome-specific probes onto chromosomes of the dhole, fennec fox, and gray fox, as well as the mapping of red fox chromosome-specific probes onto chromosomes of the corsac fox. We also present an integrated comparative chromosome map between the species studied here and all canids studied previously. The integrated map demonstrates an extensive conservation of whole chromosome arms across different canid species. In addition, we have generated a comprehensive genome phylogeny for the Canidae on the basis of the chromosome rearrangements revealed by comparative painting. This genome phylogeny has provided new insights into the karyotypic relationships among the canids. Our results, together with published data, allow the formulation of a likely Canidae ancestral karyotype (CAK, 2n = 82), and reveal that at least 6-24 chromosomal fission/fusion events are needed to convert the CAK karyotype to that of the modern canids.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">129-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18293108" rel="external">18293108</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1203-5" rel="external">10.1007/s10577-007-1203-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tracking genome organization in rodents by Zoo-FISH.</h4>
<p class="authors"><span class="author">Graphodatsky AS</span>, <span class="author">Yang F</span>, <span class="author">Dobigny G</span>, <span class="author">Romanenko SA</span>, <span class="author">Biltueva LS</span>, <span class="author">Perelman PL</span>, <span class="author">Beklemisheva VR</span>, <span class="author">Alkalaeva EZ</span>, <span class="author">Serdukova NA</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Murphy WJ</span> and <span class="author">Robinson TJ</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS 6300090, Novosibirsk, Russia. <a href="mailto:%67%72%61%66%40%62%69%6F%6E%65%74.%6E%73%63.%72%75">&#103;&#114;&#97;&#102;&#64;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;</a></p><p class="abstract">The number of rodent species examined by modern comparative genomic approaches, particularly chromosome painting, is limited. The use of human whole-chromosome painting probes to detect regions of homology in the karyotypes of the rodent index species, the mouse and rat, has been hindered by the highly rearranged nature of their genomes. In contrast, recent studies have demonstrated that non-murid rodents display more conserved genomes, underscoring their suitability for comparative genomic and higher-order systematic studies. Here we provide the first comparative chromosome maps between human and representative rodents of three major rodent lineages Castoridae, Pedetidae and Dipodidae. A comprehensive analysis of these data and those published for Sciuridae show (1) that Castoridae, Pedetidae and Dipodidae form a monophyletic group, and (2) that the European beaver Castor fiber (Castoridae) and the birch mouse Sicista betulina (Dipodidae) are sister species to the exclusion of the springhare Pedetes capensis (Pedetidae), thus resolving an enduring trifurcation in rodent higher-level systematics. Our results together with published data on the Sciuridae allow the formulation of a putative rodent ancestral karyotype (2n = 50) that is thought to comprise the following 26 human chromosomal segments and/or segmental associations: HSA1pq, 1q/10p, 2pq, 2q, 3a, 3b/19p, 3c/21, 4b, 5, 6, 7a, 7b/16p, 8p/4a/8p, 8q, 9/11, 10q, 12a/22a, 12b/22b, 13, 14/15, 16q/19q, 17, 18, 20, X and Y. These findings provide insights into the likely composition of the ancestral rodent karyotype and an improved understanding of placental genome evolution.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">2</span>;<span class="pagerange">261-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18266061" rel="external">18266061</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1191-5" rel="external">10.1007/s10577-007-1191-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication-timing-correlated spatial chromatin arrangements in cancer and in primate interphase nuclei.</h4>
<p class="authors"><span class="author">Grasser F</span>, <span class="author">Neusser M</span>, <span class="author">Fiegler H</span>, <span class="author">Thormeyer T</span>, <span class="author">Cremer M</span>, <span class="author">Carter NP</span>, <span class="author">Cremer T</span> and <span class="author">M&uuml;ller S</span></p>
<p class="affiliation">Department of Biology II, Human Genetics, Ludwig-Maximilians University Munich, Planegg-Martinsreid, Germany.</p><p class="abstract">Using published high-resolution data on S-phase replication timing, we determined the three-dimensional (3D) nuclear arrangement of 33 very-early-replicating and 31 very-late-replicating loci. We analyzed diploid human, non-human primate and rearranged tumor cells by 3D fluorescence in situ hybridization with the aim of investigating the impact of chromosomal structural changes on the nuclear organization of these loci. Overall, their topology was found to be largely conserved between cell types, species and in tumor cells. Early-replicating loci were localized in the nuclear interior, whereas late-replicating loci showed a broader distribution with a higher preference for the periphery than for late-BrdU-incorporation foci. However, differences in the spatial arrangement of early and late loci of chromosome 2, as compared with those from chromosome 5, 7 and 17, argue against replication timing as a major driving force for the 3D radial genome organization in human lymphoblastoid cell nuclei. Instead, genomic properties, and local gene density in particular, were identified as the decisive parameters. Further detailed comparisons of chromosome 7 loci in primate and tumor cells suggest that the inversions analyzed influence nuclear topology to a greater extent than the translocations, thus pointing to geometrical constraints in the 3D conformation of a chromosome territory.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2008</span>;<span class="volume">121</span>;<span class="number">Pt 11</span>;<span class="pagerange">1876-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18477608" rel="external">18477608</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687722" rel="external">2687722</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fjcs.026989" rel="external">10.1242/jcs.026989</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ORegAnno: an open-access community-driven resource for regulatory annotation.</h4>
<p class="authors"><span class="author">Griffith OL</span>, <span class="author">Montgomery SB</span>, <span class="author">Bernier B</span>, <span class="author">Chu B</span>, <span class="author">Kasaian K</span>, <span class="author">Aerts S</span>, <span class="author">Mahony S</span>, <span class="author">Sleumer MC</span>, <span class="author">Bilenky M</span>, <span class="author">Haeussler M</span>, <span class="author">Griffith M</span>, <span class="author">Gallo SM</span>, <span class="author">Giardine B</span>, <span class="author">Hooghe B</span>, <span class="author">Van Loo P</span>, <span class="author">Blanco E</span>, <span class="author">Ticoll A</span>, <span class="author">Lithwick S</span>, <span class="author">Portales-Casamar E</span>, <span class="author">Donaldson IJ</span>, <span class="author">Robertson G</span>, <span class="author">Wadelius C</span>, <span class="author">De Bleser P</span>, <span class="author">Vlieghe D</span>, <span class="author">Halfon MS</span>, <span class="author">Wasserman W</span>, <span class="author">Hardison R</span>, <span class="author">Bergman CM</span>, <span class="author">Jones SJ</span> and <span class="author">Open Regulatory Annotation Consortium</span></p>
<p class="affiliation">Canada&#39;s Michael Smith Genome Sciences Centre, British Columbia Cancer Agency, Vancouver, BC V5Z 4S6, Canada. <a href="mailto:%6F%62%69%67%40%62%63%67%73%63.%63%61">&#111;&#98;&#105;&#103;&#64;&#98;&#99;&#103;&#115;&#99;&#46;&#99;&#97;</a></p><p class="abstract">ORegAnno is an open-source, open-access database and literature curation system for community-based annotation of experimentally identified DNA regulatory regions, transcription factor binding sites and regulatory variants. The current release comprises 30 145 records curated from 922 publications and describing regulatory sequences for over 3853 genes and 465 transcription factors from 19 species. A new feature called the &#39;publication queue&#39; allows users to input relevant papers from scientific literature as targets for annotation. The queue contains 4438 gene regulation papers entered by experts and another 54 351 identified by text-mining methods. Users can enter or &#39;check out&#39; papers from the queue for manual curation using a series of user-friendly annotation pages. A typical record entry consists of species, sequence type, sequence, target gene, binding factor, experimental outcome and one or more lines of experimental evidence. An evidence ontology was developed to describe and categorize these experiments. Records are cross-referenced to Ensembl or Entrez gene identifiers, PubMed and dbSNP and can be visualized in the Ensembl or UCSC genome browsers. All data are freely available through search pages, XML data dumps or web services at: http://www.oreganno.org.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D107-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18006570" rel="external">18006570</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2239002" rel="external">2239002</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm967" rel="external">10.1093/nar/gkm967</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">miRBase: tools for microRNA genomics.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span>, <span class="author">Saini HK</span>, <span class="author">van Dongen S</span> and <span class="author">Enright AJ</span></p>
<p class="affiliation">Faculty of Life Sciences, University of Manchester, Michael Smith Building, Oxford Road, Manchester, UK. <a href="mailto:%73%61%6D.%67%72%69%66%66%69%74%68%73-%6A%6F%6E%65%73%40%6D%61%6E%63%68%65%73%74%65%72.%61%63.%75%6B">&#115;&#97;&#109;&#46;&#103;&#114;&#105;&#102;&#102;&#105;&#116;&#104;&#115;&#45;&#106;&#111;&#110;&#101;&#115;&#64;&#109;&#97;&#110;&#99;&#104;&#101;&#115;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">miRBase is the central online repository for microRNA (miRNA) nomenclature, sequence data, annotation and target prediction. The current release (10.0) contains 5071 miRNA loci from 58 species, expressing 5922 distinct mature miRNA sequences: a growth of over 2000 sequences in the past 2 years. miRBase provides a range of data to facilitate studies of miRNA genomics: all miRNAs are mapped to their genomic coordinates. Clusters of miRNA sequences in the genome are highlighted, and can be defined and retrieved with any inter-miRNA distance. The overlap of miRNA sequences with annotated transcripts, both protein- and non-coding, are described. Finally, graphical views of the locations of a wide range of genomic features in model organisms allow for the first time the prediction of the likely boundaries of many miRNA primary transcripts. miRBase is available at http://microrna.sanger.ac.uk/.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D154-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17991681" rel="external">17991681</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238936" rel="external">2238936</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm952" rel="external">10.1093/nar/gkm952</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The missing link: Bordetella petrii is endowed with both the metabolic versatility of environmental bacteria and virulence traits of pathogenic Bordetellae.</h4>
<p class="authors"><span class="author">Gross R</span>, <span class="author">Guzman CA</span>, <span class="author">Sebaihia M</span>, <span class="author">dos Santos VA</span>, <span class="author">Pieper DH</span>, <span class="author">Koebnik R</span>, <span class="author">Lechner M</span>, <span class="author">Bartels D</span>, <span class="author">Buhrmester J</span>, <span class="author">Choudhuri JV</span>, <span class="author">Ebensen T</span>, <span class="author">Gaigalat L</span>, <span class="author">Herrmann S</span>, <span class="author">Khachane AN</span>, <span class="author">Larisch C</span>, <span class="author">Link S</span>, <span class="author">Linke B</span>, <span class="author">Meyer F</span>, <span class="author">Mormann S</span>, <span class="author">Nakunst D</span>, <span class="author">R&uuml;ckert C</span>, <span class="author">Schneiker-Bekel S</span>, <span class="author">Schulze K</span>, <span class="author">Vorh&ouml;lter FJ</span>, <span class="author">Yevsa T</span>, <span class="author">Engle JT</span>, <span class="author">Goldman WE</span>, <span class="author">P&uuml;hler A</span>, <span class="author">G&ouml;bel UB</span>, <span class="author">Goesmann A</span>, <span class="author">Bl&ouml;cker H</span>, <span class="author">Kaiser O</span> and <span class="author">Martinez-Arias R</span></p>
<p class="affiliation">Chair of Microbiology, Biocenter, University of W&uuml;rzburg, Am Hubland, D-97074 W&uuml;rzburg, Germany. <a href="mailto:%72%6F%79%40%62%69%6F%7A%65%6E%74%72%75%6D.%75%6E%69-%77%75%65%72%7A%62%75%72%67.%64%65">&#114;&#111;&#121;&#64;&#98;&#105;&#111;&#122;&#101;&#110;&#116;&#114;&#117;&#109;&#46;&#117;&#110;&#105;&#45;&#119;&#117;&#101;&#114;&#122;&#98;&#117;&#114;&#103;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: Bordetella petrii is the only environmental species hitherto found among the otherwise host-restricted and pathogenic members of the genus Bordetella. Phylogenetically, it connects the pathogenic Bordetellae and environmental bacteria of the genera Achromobacter and Alcaligenes, which are opportunistic pathogens. B. petrii strains have been isolated from very different environmental niches, including river sediment, polluted soil, marine sponges and a grass root. Recently, clinical isolates associated with bone degenerative disease or cystic fibrosis have also been described.</p><p class="abstract"><strong>Results</strong>: In this manuscript we present the results of the analysis of the completely annotated genome sequence of the B. petrii strain DSMZ12804. B. petrii has a mosaic genome of 5,287,950 bp harboring numerous mobile genetic elements, including seven large genomic islands. Four of them are highly related to the clc element of Pseudomonas knackmussii B13, which encodes genes involved in the degradation of aromatics. Though being an environmental isolate, the sequenced B. petrii strain also encodes proteins related to virulence factors of the pathogenic Bordetellae, including the filamentous hemagglutinin, which is a major colonization factor of B. pertussis, and the master virulence regulator BvgAS. However, it lacks all known toxins of the pathogenic Bordetellae.</p><p class="abstract"><strong>Conclusion</strong>: The genomic analysis suggests that B. petrii represents an evolutionary link between free-living environmental bacteria and the host-restricted obligate pathogenic Bordetellae. Its remarkable metabolic versatility may enable B. petrii to thrive in very different ecological niches.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">449</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18826580">18826580</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2572626">2572626</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-449">10.1186/1471-2164-9-449</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Systemic spread is an early step in breast cancer.</h4>
<p class="authors"><span class="author">H&uuml;semann Y</span>, <span class="author">Geigl JB</span>, <span class="author">Schubert F</span>, <span class="author">Musiani P</span>, <span class="author">Meyer M</span>, <span class="author">Burghart E</span>, <span class="author">Forni G</span>, <span class="author">Eils R</span>, <span class="author">Fehm T</span>, <span class="author">Riethm&uuml;ller G</span> and <span class="author">Klein CA</span></p>
<p class="affiliation">Department of Pathology, Division of Oncogenomics, University of Regensburg, Regensburg 93053, Germany.</p><p class="abstract">It is widely accepted that metastasis is a late event in cancer progression. Here, however, we show that tumor cells can disseminate systemically from earliest epithelial alterations in HER-2 and PyMT transgenic mice and from ductal carcinoma in situ in women. Wild-type mice transplanted with single premalignant HER-2 transgenic glands displayed disseminated tumor cells and micrometastasis in bone marrow and lungs. The number of disseminated cancer cells and their karyotypic abnormalities were similar for small and large tumors in patients and mouse models. When activated by bone marrow transplantation into wild-type recipients, 80 early-disseminated cancer cells sufficed to induce lethal carcinosis. Therefore, release from dormancy of early-disseminated cancer cells may frequently account for metachronous metastasis.</p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2008</span>;<span class="volume">13</span>;<span class="number">1</span>;<span class="pagerange">58-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18167340" rel="external">18167340</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ccr.2007.12.003" rel="external">10.1016/j.ccr.2007.12.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High divergence in primate-specific duplicated regions: human and chimpanzee chorionic gonadotropin beta genes.</h4>
<p class="authors"><span class="author">Hallast P</span>, <span class="author">Saarela J</span>, <span class="author">Palotie A</span> and <span class="author">Laan M</span></p>
<p class="affiliation">Department of Biotechnology, Institute of Molecular and Cell Biology, University of Tartu, Riia 23, 51010 Tartu, Estonia. <a href="mailto:%70%69%6C%6C%65%68%40%75%74.%65%65">&#112;&#105;&#108;&#108;&#101;&#104;&#64;&#117;&#116;&#46;&#101;&#101;</a></p><p class="abstract"><strong>Background</strong>: Low nucleotide divergence between human and chimpanzee does not sufficiently explain the species-specific morphological, physiological and behavioral traits. As gene duplication is a major prerequisite for the emergence of new genes and novel biological processes, comparative studies of human and chimpanzee duplicated genes may assist in understanding the mechanisms behind primate evolution. We addressed the divergence between human and chimpanzee duplicated genomic regions by using Luteinizing Hormone Beta (LHB)/Chorionic Gonadotropin Beta (CGB) gene cluster as a model. The placental CGB genes that are essential for implantation have evolved from an ancestral pituitary LHB gene by duplications in the primate lineage.</p><p class="abstract"><strong>Results</strong>: We shotgun sequenced and compared the human (45,165 bp) and chimpanzee (39,876 bp) LHB/CGB regions and hereby present evidence for structural variation resulting in discordant number of CGB genes (6 in human, 5 in chimpanzee). The scenario of species-specific parallel duplications was supported (i) as the most parsimonious solution requiring the least rearrangement events to explain the interspecies structural differences; (ii) by the phylogenetic trees constructed with fragments of intergenic regions; (iii) by the sequence similarity calculations. Across the orthologous regions of LHB/CGB cluster, substitutions and indels contributed approximately equally to the interspecies divergence and the distribution of nucleotide identity was correlated with the regional repeat content. Intraspecies gene conversion may have shaped the LHB/CGB gene cluster. The substitution divergence (1.8-2.59%) exceeded two-three fold the estimates for single-copy loci and the fraction of transversional mutations was increased compared to the unique sequences (43% versus approximately 30%). Despite the high sequence identity among LHB/CGB genes, there are signs of functional differentiation among the gene copies. Estimates for dn/ds rate ratio suggested a purifying selection on LHB and CGB8, and a positive evolution of CGB1.</p><p class="abstract"><strong>Conclusion</strong>: If generalized, our data suggests that in addition to species-specific deletions and duplications, parallel duplication events may have contributed to genetic differences separating humans from their closest relatives. Compared to unique genomic segments, duplicated regions are characterized by high divergence promoted by intraspecies gene conversion and species-specific chromosomal rearrangements, including the alterations in gene copy number.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">070191/Z/03/Z</span></p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">8</span>;<span class="pagerange">195</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18606016" rel="external">18606016</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2478647" rel="external">2478647</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-8-195" rel="external">10.1186/1471-2148-8-195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel streptococcal integrative conjugative element involved in iron acquisition.</h4>
<p class="authors"><span class="author">Heather Z</span>, <span class="author">Holden MT</span>, <span class="author">Steward KF</span>, <span class="author">Parkhill J</span>, <span class="author">Song L</span>, <span class="author">Challis GL</span>, <span class="author">Robinson C</span>, <span class="author">Davis-Poynter N</span> and <span class="author">Waller AS</span></p>
<p class="affiliation">Centre for Preventive Medicine, Animal Health Trust, Lanwades Park, Kentford, Newmarket, Suffolk, UK.</p><p class="abstract">In this study, we determined the function of a novel non-ribosomal peptide synthetase (NRPS) system carried by a streptococcal integrative conjugative element (ICE), ICESe2. The NRPS shares similarity with the yersiniabactin system found in the high-pathogenicity island of Yersinia sp. and is the first of its kind to be identified in streptococci. We named the NRPS product &#39;equibactin&#39; and genes of this locus eqbA-N. ICESe2, although absolutely conserved in Streptococcus equi, the causative agent of equine strangles, was absent from all strains of the closely related opportunistic pathogen Streptococcus zooepidemicus. Binding of EqbA, a DtxR-like regulator, to the eqbB promoter was increased in the presence of cations. Deletion of eqbA resulted in a small-colony phenotype. Further deletion of the irp2 homologue eqbE, or the genes eqbH, eqbI and eqbJ encoding a putative ABC transporter, or addition of the iron chelator nitrilotriacetate, reversed this phenotype, implicating iron toxicity. Quantification of (55)Fe accumulation and sensitivity to streptonigrin suggested that equibactin is secreted by S. equi and that the eqbH, eqbI and eqbJ genes are required for its associated iron import. In agreement with a structure-based model of equibactin synthesis, supplementation of chemically defined media with salicylate was required for equibactin production.</p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">70</span>;<span class="number">5</span>;<span class="pagerange">1274-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18990191">18990191</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2008.06481.x">10.1111/j.1365-2958.2008.06481.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence data swell for nematodes.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span> and <span class="author">Pain A</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">With more than 80,000 described species that are extremely diverse in terms of ecology and biology, the Nematoda phylum is one of the most common animal phyla. This month&#39;s Genome Watch describes genomes of several nematodes, including that of the human filarial parasite Brugia malayi.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">800-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18923407">18923407</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2021">10.1038/nrmicro2021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Telomeric expression sites are highly conserved in Trypanosoma brucei.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span>, <span class="author">Figueiredo LM</span>, <span class="author">Quail MA</span>, <span class="author">Becker M</span>, <span class="author">Jackson A</span>, <span class="author">Bason N</span>, <span class="author">Brooks K</span>, <span class="author">Churcher C</span>, <span class="author">Fahkro S</span>, <span class="author">Goodhead I</span>, <span class="author">Heath P</span>, <span class="author">Kartvelishvili M</span>, <span class="author">Mungall K</span>, <span class="author">Harris D</span>, <span class="author">Hauser H</span>, <span class="author">Sanders M</span>, <span class="author">Saunders D</span>, <span class="author">Seeger K</span>, <span class="author">Sharp S</span>, <span class="author">Taylor JE</span>, <span class="author">Walker D</span>, <span class="author">White B</span>, <span class="author">Young R</span>, <span class="author">Cross GA</span>, <span class="author">Rudenko G</span>, <span class="author">Barry JD</span>, <span class="author">Louis EJ</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom. <a href="mailto:%63%68%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#104;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Subtelomeric regions are often under-represented in genome sequences of eukaryotes. One of the best known examples of the use of telomere proximity for adaptive purposes are the bloodstream expression sites (BESs) of the African trypanosome Trypanosoma brucei. To enhance our understanding of BES structure and function in host adaptation and immune evasion, the BES repertoire from the Lister 427 strain of T. brucei were independently tagged and sequenced. BESs are polymorphic in size and structure but reveal a surprisingly conserved architecture in the context of extensive recombination. Very small BESs do exist and many functioning BESs do not contain the full complement of expression site associated genes (ESAGs). The consequences of duplicated or missing ESAGs, including ESAG9, a newly named ESAG12, and additional variant surface glycoprotein genes (VSGs) were evaluated by functional assays after BESs were tagged with a drug-resistance gene. Phylogenetic analysis of constituent ESAG families suggests that BESs are sequence mosaics and that extensive recombination has shaped the evolution of the BES repertoire. This work opens important perspectives in understanding the molecular mechanisms of antigenic variation, a widely used strategy for immune evasion in pathogens, and telomere biology.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01AI021729</span>; Wellcome Trust: <span class="grant">095161</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">10</span>;<span class="pagerange">e3527</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18953401">18953401</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2567434">2567434</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0003527">10.1371/journal.pone.0003527</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Myo6 mutation destroys coordination between the myosin heads, revealing new functions of myosin VI in the stereocilia of mammalian inner ear hair cells.</h4>
<p class="authors"><span class="author">Hertzano R</span>, <span class="author">Shalit E</span>, <span class="author">Rzadzinska AK</span>, <span class="author">Dror AA</span>, <span class="author">Song L</span>, <span class="author">Ron U</span>, <span class="author">Tan JT</span>, <span class="author">Shitrit AS</span>, <span class="author">Fuchs H</span>, <span class="author">Hasson T</span>, <span class="author">Ben-Tal N</span>, <span class="author">Sweeney HL</span>, <span class="author">de Angelis MH</span>, <span class="author">Steel KP</span> and <span class="author">Avraham KB</span></p>
<p class="affiliation">Department of Human Molecular Genetics and Biochemistry, Tel Aviv University, Tel Aviv, Israel.</p><p class="abstract">Myosin VI, found in organisms from Caenorhabditis elegans to humans, is essential for auditory and vestibular function in mammals, since genetic mutations lead to hearing impairment and vestibular dysfunction in both humans and mice. Here, we show that a missense mutation in this molecular motor in an ENU-generated mouse model, Tailchaser, disrupts myosin VI function. Structural changes in the Tailchaser hair bundles include mislocalization of the kinocilia and branching of stereocilia. Transfection of GFP-labeled myosin VI into epithelial cells and delivery of endocytic vesicles to the early endosome revealed that the mutant phenotype displays disrupted motor function. The actin-activated ATPase rates measured for the D179Y mutation are decreased, and indicate loss of coordination of the myosin VI heads or &#39;gating&#39; in the dimer form. Proper coordination is required for walking processively along, or anchoring to, actin filaments, and is apparently destroyed by the proximity of the mutation to the nucleotide-binding pocket. This loss of myosin VI function may not allow myosin VI to transport its cargoes appropriately at the base and within the stereocilia, or to anchor the membrane of stereocilia to actin filaments via its cargos, both of which lead to structural changes in the stereocilia of myosin VI-impaired hair cells, and ultimately leading to deafness.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; NEI NIH HHS: <span class="grant">R01-EY12695</span>; NIDCD NIH HHS: <span class="grant">R01-DC0099100</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">10</span>;<span class="pagerange">e1000207</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18833301">18833301</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2543112">2543112</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000207">10.1371/journal.pgen.1000207</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome sequence of the fish pathogen Aliivibrio salmonicida strain LFI1238 shows extensive evidence of gene decay.</h4>
<p class="authors"><span class="author">Hjerde E</span>, <span class="author">Lorentzen MS</span>, <span class="author">Holden MT</span>, <span class="author">Seeger K</span>, <span class="author">Paulsen S</span>, <span class="author">Bason N</span>, <span class="author">Churcher C</span>, <span class="author">Harris D</span>, <span class="author">Norbertczak H</span>, <span class="author">Quail MA</span>, <span class="author">Sanders S</span>, <span class="author">Thurston S</span>, <span class="author">Parkhill J</span>, <span class="author">Willassen NP</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Department of Molecular Biotechnology, Institute of Medical Biology, Faculty of Medicine, University of Troms&oslash;, N-9037 Troms&oslash;, Norway. <a href="mailto:%65%72%69%6B%68%40%66%61%67%6D%65%64.%75%69%74.%6E%6F">&#101;&#114;&#105;&#107;&#104;&#64;&#102;&#97;&#103;&#109;&#101;&#100;&#46;&#117;&#105;&#116;&#46;&#110;&#111;</a></p><p class="abstract"><strong>Background</strong>: The fish pathogen Aliivibrio salmonicida is the causative agent of cold-water vibriosis in marine aquaculture. The Gram-negative bacterium causes tissue degradation, hemolysis and sepsis in vivo.</p><p class="abstract"><strong>Results</strong>: In total, 4 286 protein coding sequences were identified, and the 4.6 Mb genome of A. salmonicida has a six partite architecture with two chromosomes and four plasmids. Sequence analysis revealed a highly fragmented genome structure caused by the insertion of an extensive number of insertion sequence (IS) elements. The IS elements can be related to important evolutionary events such as gene acquisition, gene loss and chromosomal rearrangements. New A. salmonicida functional capabilities that may have been aquired through horizontal DNA transfer include genes involved in iron-acquisition, and protein secretion and play potential roles in pathogenicity. On the other hand, the degeneration of 370 genes and consequent loss of specific functions suggest that A. salmonicida has a reduced metabolic and physiological capacity in comparison to related Vibrionaceae species.</p><p class="abstract"><strong>Conclusion</strong>: Most prominent is the loss of several genes involved in the utilisation of the polysaccharide chitin. In particular, the disruption of three extracellular chitinases responsible for enzymatic breakdown of chitin makes A. salmonicida unable to grow on the polymer form of chitin. These, and other losses could restrict the variety of carrier organisms A. salmonicida can attach to, and associate with. Gene acquisition and gene loss may be related to the emergence of A. salmonicida as a fish pathogen.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">616</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19099551">19099551</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2627896">2627896</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-616">10.1186/1471-2164-9-616</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-throughput sequencing provides insights into genome variation and evolution in Salmonella Typhi.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Parkhill J</span>, <span class="author">Mazzoni CJ</span>, <span class="author">Roumagnac P</span>, <span class="author">Weill FX</span>, <span class="author">Goodhead I</span>, <span class="author">Rance R</span>, <span class="author">Baker S</span>, <span class="author">Maskell DJ</span>, <span class="author">Wain J</span>, <span class="author">Dolecek C</span>, <span class="author">Achtman M</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6B%68%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#107;&#104;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Isolates of Salmonella enterica serovar Typhi (Typhi), a human-restricted bacterial pathogen that causes typhoid, show limited genetic variation. We generated whole-genome sequences for 19 Typhi isolates using 454 (Roche) and Solexa (Illumina) technologies. Isolates, including the previously sequenced CT18 and Ty2 isolates, were selected to represent major nodes in the phylogenetic tree. Comparative analysis showed little evidence of purifying selection, antigenic variation or recombination between isolates. Rather, evolution in the Typhi population seems to be characterized by ongoing loss of gene function, consistent with a small effective population size. The lack of evidence for antigenic variation driven by immune selection is in contrast to strong adaptive selection for mutations conferring antibiotic resistance in Typhi. The observed patterns of genetic isolation and drift are consistent with the proposed key role of asymptomatic carriers of Typhi as the main reservoir of this pathogen, highlighting the need for identification and treatment of carriers.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">067321</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">8</span>;<span class="pagerange">987-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18660809">18660809</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652037">2652037</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.195">10.1038/ng.195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gastric cancer in Japan.</h4>
<p class="authors"><span class="author">Horowitz RE</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2008</span>;<span class="volume">359</span>;<span class="number">22</span>;<span class="pagerange">2393-4; author reply 2394-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19038888" rel="external">19038888</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMc081797" rel="external">10.1056/NEJMc081797</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation analysis and gene annotation of eight MHC haplotypes: the MHC Haplotype Project.</h4>
<p class="authors"><span class="author">Horton R</span>, <span class="author">Gibson R</span>, <span class="author">Coggill P</span>, <span class="author">Miretti M</span>, <span class="author">Allcock RJ</span>, <span class="author">Almeida J</span>, <span class="author">Forbes S</span>, <span class="author">Gilbert JG</span>, <span class="author">Halls K</span>, <span class="author">Harrow JL</span>, <span class="author">Hart E</span>, <span class="author">Howe K</span>, <span class="author">Jackson DK</span>, <span class="author">Palmer S</span>, <span class="author">Roberts AN</span>, <span class="author">Sims S</span>, <span class="author">Stewart CA</span>, <span class="author">Traherne JA</span>, <span class="author">Trevanion S</span>, <span class="author">Wilming L</span>, <span class="author">Rogers J</span>, <span class="author">de Jong PJ</span>, <span class="author">Elliott JF</span>, <span class="author">Sawcer S</span>, <span class="author">Todd JA</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">The human major histocompatibility complex (MHC) is contained within about 4 Mb on the short arm of chromosome 6 and is recognised as the most variable region in the human genome. The primary aim of the MHC Haplotype Project was to provide a comprehensively annotated reference sequence of a single, human leukocyte antigen-homozygous MHC haplotype and to use it as a basis against which variations could be assessed from seven other similarly homozygous cell lines, representative of the most common MHC haplotypes in the European population. Comparison of the haplotype sequences, including four haplotypes not previously analysed, resulted in the identification of &gt;44,000 variations, both substitutions and indels (insertions and deletions), which have been submitted to the dbSNP database. The gene annotation uncovered haplotype-specific differences and confirmed the presence of more than 300 loci, including over 160 protein-coding genes. Combined analysis of the variation and annotation datasets revealed 122 gene loci with coding substitutions of which 97 were non-synonymous. The haplotype (A3-B7-DR15; PGF cell line) designated as the new MHC reference sequence, has been incorporated into the human genome assembly (NCBI35 and subsequent builds), and constitutes the largest single-haplotype sequence of the human genome to date. The extensive variation and annotation data derived from the analysis of seven further haplotypes have been made publicly available and provide a framework and resource for future association studies of all MHC-associated diseases and transplant medicine.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">048880</span>, <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">60</span>;<span class="number">1</span>;<span class="pagerange">1-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18193213">18193213</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2206249">2206249</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00251-007-0262-2">10.1007/s00251-007-0262-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Array painting reveals a high frequency of balanced translocations in breast cancer cell lines that break in cancer-relevant genes.</h4>
<p class="authors"><span class="author">Howarth KD</span>, <span class="author">Blood KA</span>, <span class="author">Ng BL</span>, <span class="author">Beavis JC</span>, <span class="author">Chua Y</span>, <span class="author">Cooke SL</span>, <span class="author">Raby S</span>, <span class="author">Ichimura K</span>, <span class="author">Collins VP</span>, <span class="author">Carter NP</span> and <span class="author">Edwards PA</span></p>
<p class="affiliation">Department of Pathology, Hutchison-MRC Research Centre, University of Cambridge, Cambridge, UK.</p><p class="abstract">Chromosome translocations in the common epithelial cancers are abundant, yet little is known about them. They have been thought to be almost all unbalanced and therefore dismissed as mostly mediating tumour suppressor loss. We present a comprehensive analysis by array painting of the chromosome translocations of breast cancer cell lines HCC1806, HCC1187 and ZR-75-30. In array painting, chromosomes are isolated by flow cytometry, amplified and hybridized to DNA microarrays. A total of 200 breakpoints were identified and all were mapped to 1 Mb resolution on bacterial artificial chromosome (BAC) arrays, then 40 selected breakpoints, including all balanced breakpoints, were further mapped on tiling-path BAC arrays or to around 2 kb resolution using oligonucleotide arrays. Many more of the translocations were balanced at 1 Mb resolution than expected, either reciprocal (eight in total) or balanced for at least one participating chromosome (19 paired breakpoints). Second, many of the breakpoints were at genes that are plausible targets of oncogenic translocation, including balanced breaks at CTCF, EP300/p300 and FOXP4. Two gene fusions were demonstrated, TAX1BP1-AHCY and RIF1-PKD1L1. Our results support the idea that chromosome rearrangements may play an important role in common epithelial cancers such as breast cancer.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A4392</span>; Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2008</span>;<span class="volume">27</span>;<span class="number">23</span>;<span class="pagerange">3345-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18084325">18084325</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2423006">2423006</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1210993">10.1038/sj.onc.1210993</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">No evidence in a large UK collection for celiac disease risk variants reported by a Spanish study.</h4>
<p class="authors"><span class="author">Hunt KA</span>, <span class="author">Franke L</span>, <span class="author">Deloukas P</span>, <span class="author">Wijmenga C</span> and <span class="author">van Heel DA</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">Gastroenterology</span>&nbsp;<span class="year">2008</span>;<span class="volume">134</span>;<span class="number">5</span>;<span class="pagerange">1629-30; author reply 1630-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18471539" rel="external">18471539</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2696798" rel="external">2696798</a>; DOI: <a href="http://dx.doi.org/10.1053%2Fj.gastro.2008.03.068" rel="external">10.1053/j.gastro.2008.03.068</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Newly identified genetic risk variants for celiac disease related to the immune response.</h4>
<p class="authors"><span class="author">Hunt KA</span>, <span class="author">Zhernakova A</span>, <span class="author">Turner G</span>, <span class="author">Heap GA</span>, <span class="author">Franke L</span>, <span class="author">Bruinenberg M</span>, <span class="author">Romanos J</span>, <span class="author">Dinesen LC</span>, <span class="author">Ryan AW</span>, <span class="author">Panesar D</span>, <span class="author">Gwilliam R</span>, <span class="author">Takeuchi F</span>, <span class="author">McLaren WM</span>, <span class="author">Holmes GK</span>, <span class="author">Howdle PD</span>, <span class="author">Walters JR</span>, <span class="author">Sanders DS</span>, <span class="author">Playford RJ</span>, <span class="author">Trynka G</span>, <span class="author">Mulder CJ</span>, <span class="author">Mearin ML</span>, <span class="author">Verbeek WH</span>, <span class="author">Trimble V</span>, <span class="author">Stevens FM</span>, <span class="author">O&#39;Morain C</span>, <span class="author">Kennedy NP</span>, <span class="author">Kelleher D</span>, <span class="author">Pennington DJ</span>, <span class="author">Strachan DP</span>, <span class="author">McArdle WL</span>, <span class="author">Mein CA</span>, <span class="author">Wapenaar MC</span>, <span class="author">Deloukas P</span>, <span class="author">McGinnis R</span>, <span class="author">McManus R</span>, <span class="author">Wijmenga C</span> and <span class="author">van Heel DA</span></p>
<p class="affiliation">Institute of Cell and Molecular Science, Barts and The London School of Medicine and Dentistry, 4 Newark Street, London E1 2AT, UK.</p><p class="abstract">Our genome-wide association study of celiac disease previously identified risk variants in the IL2-IL21 region. To identify additional risk variants, we genotyped 1,020 of the most strongly associated non-HLA markers in an additional 1,643 cases and 3,406 controls. Through joint analysis including the genome-wide association study data (767 cases, 1,422 controls), we identified seven previously unknown risk regions (P &lt; 5 x 10(-7)). Six regions harbor genes controlling immune responses, including CCR3, IL12A, IL18RAP, RGS1, SH2B3 (nsSNP rs3184504) and TAGAP. Whole-blood IL18RAP mRNA expression correlated with IL18RAP genotype. Type 1 diabetes and celiac disease share HLA-DQ, IL2-IL21, CCR3 and SH2B3 risk regions. Thus, this extensive genome-wide association follow-up study has identified additional celiac disease risk variants in relevant biological pathways.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068094</span>, <span class="grant">068545/Z/02</span>, <span class="grant">084743</span>, <span class="grant">GR068094MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">4</span>;<span class="pagerange">395-402</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18311140">18311140</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2673512">2673512</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.102">10.1038/ng.102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The functional impact of structural variation in humans.</h4>
<p class="authors"><span class="author">Hurles ME</span>, <span class="author">Dermitzakis ET</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%65%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#101;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Structural variation includes many different types of chromosomal rearrangement and encompasses millions of bases in every human genome. Over the past 3 years, the extent and complexity of structural variation has become better appreciated. Diverse approaches have been adopted to explore the functional impact of this class of variation. As disparate indications of the important biological consequences of genome dynamism are accumulating rapidly, we review the evidence that structural variation has an appreciable impact on cellular phenotypes, disease and human evolution.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span>, <span class="grant">077014</span>, <span class="grant">077046</span></p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">5</span>;<span class="pagerange">238-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18378036">18378036</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2869026">2869026</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2008.03.001">10.1016/j.tig.2008.03.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">1p36 is a preferential target of chromosome 1 deletions in astrocytic tumours and homozygously deleted in a subset of glioblastomas.</h4>
<p class="authors"><span class="author">Ichimura K</span>, <span class="author">Vogazianou AP</span>, <span class="author">Liu L</span>, <span class="author">Pearson DM</span>, <span class="author">B&auml;cklund LM</span>, <span class="author">Plant K</span>, <span class="author">Baird K</span>, <span class="author">Langford CF</span>, <span class="author">Gregory SG</span> and <span class="author">Collins VP</span></p>
<p class="affiliation">Department of Pathology, Division of Molecular Histopathology, University of Cambridge, Cambridge, UK. <a href="mailto:%6B%69%32%31%32%40%63%61%6D.%61%63.%75%6B">&#107;&#105;&#50;&#49;&#50;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Astrocytic, oligodendroglial and mixed gliomas are the commonest gliomas in adults. They have distinct phenotypes and clinical courses, but as they exist as a continuous histological spectrum, differentiating them can be difficult. Co-deletions of total 1p and 19q are found in the majority of oligodendrogliomas and considered as a diagnostic marker and a prognostic indicator. The 1p status of astrocytomas has not yet been thoroughly examined. Using a chromosome 1 tile path array, we investigated 108 adult astrocytic tumours for copy number alterations. Total 1p deletions were rare (2%), however partial deletions involving 1p36 were frequently identified in anaplastic astrocytomas (22%) and glioblastomas (34%). Multivariate analysis showed that patients with total 1p deletions had significantly longer survival (P=0.005). In nine glioblastomas homozygous deletions at 1p36 were identified. No somatic mutations were found among the five genes located in the homozygously deleted region. However, the CpG island of TNFRSF9 was hypermethylated in 19% of astrocytic tumours and 87% of glioma cell lines. TNFRSF9 expression was upregulated after demethylation of glioma cell lines. Akt3 amplifications were found in four glioblastomas. Our results indicate that 1p deletions are common anaplastic astrocytomas and glioblastomas but are distinct from the 1p abnormalities in oligodendrogliomas.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6618</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2008</span>;<span class="volume">27</span>;<span class="number">14</span>;<span class="pagerange">2097-108</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17934521" rel="external">17934521</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2650419" rel="external">2650419</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1210848" rel="external">10.1038/sj.onc.1210848</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In vitro differential sensitivity of melanomas to phenothiazines is based on the presence of codon 600 BRAF mutation.</h4>
<p class="authors"><span class="author">Ikediobi ON</span>, <span class="author">Reimers M</span>, <span class="author">Durinck S</span>, <span class="author">Blower PE</span>, <span class="author">Futreal AP</span>, <span class="author">Stratton MR</span> and <span class="author">Weinstein JN</span></p>
<p class="affiliation">Genomics and Bioinformatics Group, Laboratory of Molecular Pharmacology, National Cancer Institute, Bethesda, Maryland, USA. <a href="mailto:%69%6B%65%64%69%6F%62%69%6F%40%70%68%61%72%6D%61%63%79.%75%63%73%66.%65%64%75">&#105;&#107;&#101;&#100;&#105;&#111;&#98;&#105;&#111;&#64;&#112;&#104;&#97;&#114;&#109;&#97;&#99;&#121;&#46;&#117;&#99;&#115;&#102;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The panel of 60 human cancer cell lines (the NCI-60) assembled by the National Cancer Institute for anticancer drug discovery is a widely used resource. We previously sequenced 24 cancer genes in those cell lines. Eleven of the genes were found to be mutated in three or more of the lines. Using a pharmacogenomic approach, we analyzed the relationship between drug activity and mutations in those 11 genes (APC, RB1, KRAS, NRAS, BRAF, PIK3CA, PTEN, STK11, MADH4, TP53, and CDKN2A). That analysis identified an association between mutation in BRAF and the antiproliferative potential of phenothiazine compounds. Phenothiazines have been used as antipsychotics and as adjunct antiemetics during cancer chemotherapy and more recently have been reported to have anticancer properties. However, to date, the anticancer mechanism of action of phenothiazines has not been elucidated. To follow up on the initial pharmacologic observations in the NCI-60 screen, we did pharmacologic experiments on 11 of the NCI-60 cell lines and, prospectively, on an additional 24 lines. The studies provide evidence that BRAF mutation (codon 600) in melanoma as opposed to RAS mutation is predictive of an increase in sensitivity to phenothiazines as determined by 3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium, inner salt assay (Wilcoxon P = 0.007). That pattern of increased sensitivity to phenothiazines based on the presence of codon 600 BRAF mutation may be unique to melanomas, as we do not observe it in a panel of colorectal cancers. The findings reported here have potential implications for the use of phenothiazines in the treatment of V600E BRAF mutant melanoma.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Molecular cancer therapeutics</span>&nbsp;<span class="year">2008</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">1337-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18524847" rel="external">18524847</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705835" rel="external">2705835</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1535-7163.MCT-07-2308" rel="external">10.1158/1535-7163.MCT-07-2308</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel CpG island set identifies tissue-specific methylation at developmental gene loci.</h4>
<p class="authors"><span class="author">Illingworth R</span>, <span class="author">Kerr A</span>, <span class="author">Desousa D</span>, <span class="author">J&oslash;rgensen H</span>, <span class="author">Ellis P</span>, <span class="author">Stalker J</span>, <span class="author">Jackson D</span>, <span class="author">Clee C</span>, <span class="author">Plumb R</span>, <span class="author">Rogers J</span>, <span class="author">Humphray S</span>, <span class="author">Cox T</span>, <span class="author">Langford C</span> and <span class="author">Bird A</span></p>
<p class="affiliation">Wellcome Trust Centre for Cell Biology, University of Edinburgh, Edinburgh, United Kingdom.</p><p class="abstract">CpG islands (CGIs) are dense clusters of CpG sequences that punctuate the CpG-deficient human genome and associate with many gene promoters. As CGIs also differ from bulk chromosomal DNA by their frequent lack of cytosine methylation, we devised a CGI enrichment method based on nonmethylated CpG affinity chromatography. The resulting library was sequenced to define a novel human blood CGI set that includes many that are not detected by current algorithms. Approximately half of CGIs were associated with annotated gene transcription start sites, the remainder being intra- or intergenic. Using an array representing over 17,000 CGIs, we established that 6%-8% of CGIs are methylated in genomic DNA of human blood, brain, muscle, and spleen. Inter- and intragenic CGIs are preferentially susceptible to methylation. CGIs showing tissue-specific methylation were overrepresented at numerous genetic loci that are essential for development, including HOX and PAX family members. The findings enable a comprehensive analysis of the roles played by CGI methylation in normal and diseased human tissues.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">e22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18232738">18232738</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2214817">2214817</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0060022">10.1371/journal.pbio.0060022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Refined mapping of X-linked reticulate pigmentary disorder and sequencing of candidate genes.</h4>
<p class="authors"><span class="author">Jaeckle Santos LJ</span>, <span class="author">Xing C</span>, <span class="author">Barnes RB</span>, <span class="author">Ades LC</span>, <span class="author">Megarbane A</span>, <span class="author">Vidal C</span>, <span class="author">Xuereb A</span>, <span class="author">Tarpey PS</span>, <span class="author">Smith R</span>, <span class="author">Khazab M</span>, <span class="author">Shoubridge C</span>, <span class="author">Partington M</span>, <span class="author">Futreal A</span>, <span class="author">Stratton MR</span>, <span class="author">Gecz J</span> and <span class="author">Zinn AR</span></p>
<p class="affiliation">McDermott Center for Human Growth and Development, The University of Texas Southwestern Medical School, 5323 Harry Hines Boulevard, Dallas, TX 75390-8591, USA. <a href="mailto:%6C%61%6E%65.%6A%61%65%63%6B%6C%65%40%75%74%73%6F%75%74%68%77%65%73%74%65%72%6E.%65%64%75">&#108;&#97;&#110;&#101;&#46;&#106;&#97;&#101;&#99;&#107;&#108;&#101;&#64;&#117;&#116;&#115;&#111;&#117;&#116;&#104;&#119;&#101;&#115;&#116;&#101;&#114;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract">X-linked reticulate pigmentary disorder with systemic manifestations in males (PDR) is very rare. Affected males are characterized by cutaneous and visceral symptoms suggestive of abnormally regulated inflammation. A genetic linkage study of a large Canadian kindred previously mapped the PDR gene to a greater than 40 Mb interval of Xp22-p21. The aim of this study was to identify the causative gene for PDR. The Canadian pedigree was expanded and additional PDR families recruited. Genetic linkage was performed using newer microsatellite markers. Positional and functional candidate genes were screened by PCR and sequencing of coding exons in affected males. The location of the PDR gene was narrowed to a approximately 4.9 Mb interval of Xp22.11-p21.3 between markers DXS1052 and DXS1061. All annotated coding exons within this interval were sequenced in one affected male from each of the three multiplex families as well as one singleton, but no causative mutation was identified. Sequencing of other X-linked genes outside of the linked interval also failed to identify the cause of PDR but revealed a novel nonsynonymous cSNP in the GRPR gene in the Maltese population. PDR is most likely due to a mutation within the linked interval not affecting currently annotated coding exons.</p>
<p class="grants">Funded by: NIAMS NIH HHS: <span class="grant">P30 AR041940</span>; Wellcome Trust: <span class="grant">077010</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">123</span>;<span class="number">5</span>;<span class="pagerange">469-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18404279" rel="external">18404279</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2714970" rel="external">2714970</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-008-0498-4" rel="external">10.1007/s00439-008-0498-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome-wide patterns of variation expose significant substructure in a founder population.</h4>
<p class="authors"><span class="author">Jakkula E</span>, <span class="author">Rehnstr&ouml;m K</span>, <span class="author">Varilo T</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Paunio T</span>, <span class="author">Pedersen NL</span>, <span class="author">deFaire U</span>, <span class="author">J&auml;rvelin MR</span>, <span class="author">Saharinen J</span>, <span class="author">Freimer N</span>, <span class="author">Ripatti S</span>, <span class="author">Purcell S</span>, <span class="author">Collins A</span>, <span class="author">Daly MJ</span>, <span class="author">Palotie A</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Molecular Medicine, National Public Health Institute and Institute for Molecular Medicine Finland (FIMM), Helsinki, Finland.</p><p class="abstract">Although high-density SNP genotyping platforms generate a momentum for detailed genome-wide association (GWA) studies, an offshoot is a new insight into population genetics. Here, we present an example in one of the best-known founder populations by scrutinizing ten distinct Finnish early- and late-settlement subpopulations. By determining genetic distances, homozygosity, and patterns of linkage disequilibrium, we demonstrate that population substructure, and even individual ancestry, is detectable at a very high resolution and supports the concept of multiple historical bottlenecks resulting from consecutive founder effects. Given that genetic studies are currently aiming at identifying smaller and smaller genetic effects, recognizing and controlling for population substructure even at this fine level becomes imperative to avoid confounding and spurious associations. This study provides an example of the power of GWA data sets to demonstrate stratification caused by population history even within a seemingly homogeneous population, like the Finns. Further, the results provide interesting lessons concerning the impact of population history on the genome landscape of humans, as well as approaches to identify rare variants enriched in these subpopulations.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">U54RR020278</span>; NHLBI NIH HHS: <span class="grant">1R01HL087679-01</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">83</span>;<span class="number">6</span>;<span class="pagerange">787-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19061986">19061986</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2668058">2668058</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.11.005">10.1016/j.ajhg.2008.11.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapidly regulated genes are intron poor.</h4>
<p class="authors"><span class="author">Jeffares DC</span>, <span class="author">Penkett CJ</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%63%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#99;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We show that genes with rapidly changing expression levels in response to stress contain significantly lower intron densities in yeasts, thale cress and mice. Therefore, we propose that introns can delay regulatory responses and are selected against in genes whose transcripts require rapid adjustment for survival of environmental challenges. These findings could provide an explanation for the apparent extensive intron loss during the evolution of some eukaryotic lineages.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">8</span>;<span class="pagerange">375-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18586348" rel="external">18586348</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2008.05.006" rel="external">10.1016/j.tig.2008.05.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss of prokineticin receptor 2 signaling predisposes mice to torpor.</h4>
<p class="authors"><span class="author">Jethwa PH</span>, <span class="author">I&#39;Anson H</span>, <span class="author">Warner A</span>, <span class="author">Prosser HM</span>, <span class="author">Hastings MH</span>, <span class="author">Maywood ES</span> and <span class="author">Ebling FJ</span></p>
<p class="affiliation">School of Biomedical Sciences, University of Nottingham Medical School, Queen&#39;s Medical Centre, Nottingham, UK.</p><p class="abstract">The genes encoding prokineticin 2 polypeptide (Prok2) and its cognate receptor (Prokr2/Gpcr73l1) are widely expressed in both the suprachiasmatic nucleus and its hypothalamic targets, and this signaling pathway has been implicated in the circadian regulation of behavior and physiology. We have previously observed that the targeted null mutation of Prokr2 disrupts circadian coordination of cycles of locomotor activity and thermoregulation. We have now observed spontaneous but sporadic bouts of torpor in the majority of these transgenic mice lacking Prokr2 signaling. During these torpor bouts, which lasted for up to 8 h, body temperature and locomotor activity decreased markedly. Oxygen consumption and carbon dioxide production also decreased, and there was a decrease in respiratory quotient. These spontaneous torpor bouts generally began toward the end of the dark phase or in the early light phase when the mice were maintained on a 12:12-h light-dark cycle and persisted when mice were exposed to continuous darkness. Periods of food deprivation (16-24 h) induced a substantial decrease in body temperature in all mice, but the duration and depth of hypothermia was significantly greater in mice lacking Prokr2 signaling compared with heterozygous and wild-type littermates. Likewise, when tested in metabolic cages, food deprivation produced greater decreases in oxygen consumption and carbon dioxide production in the transgenic mice than controls. We conclude that Prokr2 signaling plays a role in hypothalamic regulation of energy balance, and loss of this pathway results in physiological and behavioral responses normally only detected when mice are in negative energy balance.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/D525064/I</span>, <span class="grant">BBS/B/10765</span>; Medical Research Council: <span class="grant">U.1051.02.004(78799)</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of physiology. Regulatory, integrative and comparative physiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">294</span>;<span class="number">6</span>;<span class="pagerange">R1968-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18417646" rel="external">18417646</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2735815" rel="external">2735815</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fajpregu.00778.2007" rel="external">10.1152/ajpregu.00778.2007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Image capture and fusion of 3D surface texture using wavelet transform</h4>
<p class="authors"><span class="author">Jian</span>, <span class="author">M. W.</span>, <span class="author">Dong. J. Y..Wu</span>, <span class="author">J. H.</span></p>
<p class="publication"><span class="title">Proceedings of the 2007 International Conference on Wavelet Analysis and Pattern Recognition, ICWAPR &#39;07.</span> <span class="year">2008</span>;<span class="pagerange">338-343</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of genome-wide polymorphisms in the AT-rich Plasmodium falciparum genome using a high-density microarray.</h4>
<p class="authors"><span class="author">Jiang H</span>, <span class="author">Yi M</span>, <span class="author">Mu J</span>, <span class="author">Zhang L</span>, <span class="author">Ivens A</span>, <span class="author">Klimczak LJ</span>, <span class="author">Huyen Y</span>, <span class="author">Stephens RM</span> and <span class="author">Su XZ</span></p>
<p class="affiliation">Laboratory of Malaria and Vector Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892, USA. <a href="mailto:%68%6F%6A%69%61%6E%67%40%6E%69%61%69%64.%6E%69%68.%67%6F%76">&#104;&#111;&#106;&#105;&#97;&#110;&#103;&#64;&#110;&#105;&#97;&#105;&#100;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract"><strong>Background</strong>: Genetic mapping is a powerful method to identify mutations that cause drug resistance and other phenotypic changes in the human malaria parasite Plasmodium falciparum. For efficient mapping of a target gene, it is often necessary to genotype a large number of polymorphic markers. Currently, a community effort is underway to collect single nucleotide polymorphisms (SNP) from the parasite genome. Here we evaluate polymorphism detection accuracy of a high-density &#39;tiling&#39; microarray with 2.56 million probes by comparing single feature polymorphisms (SFP) calls from the microarray with known SNP among parasite isolates.</p><p class="abstract"><strong>Results</strong>: We found that probe GC content, SNP position in a probe, probe coverage, and signal ratio cutoff values were important factors for accurate detection of SFP in the parasite genome. We established a set of SFP calling parameters that could predict mSFP (SFP called by multiple overlapping probes) with high accuracy (&gt; or = 94%) and identified 121,087 mSFP genome-wide from five parasite isolates including 40,354 unique mSFP (excluding those from multi-gene families) and approximately 18,000 new mSFP, producing a genetic map with an average of one unique mSFP per 570 bp. Genomic copy number variation (CNV) among the parasites was also cataloged and compared.</p><p class="abstract"><strong>Conclusion</strong>: A large number of mSFP were discovered from the P. falciparum genome using a high-density microarray, most of which were in clusters of highly polymorphic genes at chromosome ends. Our method for accurate mSFP detection and the mSFP identified will greatly facilitate large-scale studies of genome variation in the P. falciparum parasite and provide useful resources for mapping important parasite traits.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">N01-CO-12400</span>; Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">398</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18724869" rel="external">18724869</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2543026" rel="external">2543026</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-398" rel="external">10.1186/1471-2164-9-398</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Drug susceptibility testing using molecular techniques can enhance tuberculosis diagnosis in a community with a high tuberculosis incidence</h4>
<p class="authors"><span class="author">Johnson</span>, <span class="author">R</span></p>
<p class="publication"><span class="title">JIDC.</span> <span class="year">2008</span>;<span class="volume">2</span>;<span class="pagerange">40-45</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale population study of human cell lines indicates that dosage compensation is virtually complete.</h4>
<p class="authors"><span class="author">Johnston CM</span>, <span class="author">Lovell FL</span>, <span class="author">Leongamornlert DA</span>, <span class="author">Stranger BE</span>, <span class="author">Dermitzakis ET</span> and <span class="author">Ross MT</span></p>
<p class="affiliation">X Chromosome Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">X chromosome inactivation in female mammals results in dosage compensation of X-linked gene products between the sexes. In humans there is evidence that a substantial proportion of genes escape from silencing. We have carried out a large-scale analysis of gene expression in lymphoblastoid cell lines from four human populations to determine the extent to which escape from X chromosome inactivation disrupts dosage compensation. We conclude that dosage compensation is virtually complete. Overall expression from the X chromosome is only slightly higher in females and can largely be accounted for by elevated female expression of approximately 5% of X-linked genes. We suggest that the potential contribution of escape from X chromosome inactivation to phenotypic differences between the sexes is more limited than previously believed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">1</span>;<span class="pagerange">e9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18208332" rel="external">18208332</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2213701" rel="external">2213701</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0040009" rel="external">10.1371/journal.pgen.0040009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A systematic library for comprehensive overexpression screens in Saccharomyces cerevisiae.</h4>
<p class="authors"><span class="author">Jones GM</span>, <span class="author">Stalker J</span>, <span class="author">Humphray S</span>, <span class="author">West A</span>, <span class="author">Cox T</span>, <span class="author">Rogers J</span>, <span class="author">Dunham I</span> and <span class="author">Prelich G</span></p>
<p class="affiliation">Department of Molecular Genetics, Albert Einstein College of Medicine, 1300 Morris Park Avenue, Bronx, New York 10461, USA.</p><p class="abstract">Modern genetic analysis requires the development of new resources to systematically explore gene function in vivo. Overexpression screens are a powerful method to investigate genetic pathways, but the goal of routine and comprehensive overexpression screens has been hampered by the lack of systematic libraries. Here we describe the construction of a systematic collection of the Saccharomyces cerevisiae genome in a high-copy vector and its validation in two overexpression screens.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM52486</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2008</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">239-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18246075">18246075</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1181">10.1038/nmeth.1181</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans.</h4>
<p class="authors"><span class="author">Kathiresan S</span>, <span class="author">Melander O</span>, <span class="author">Guiducci C</span>, <span class="author">Surti A</span>, <span class="author">Burtt NP</span>, <span class="author">Rieder MJ</span>, <span class="author">Cooper GM</span>, <span class="author">Roos C</span>, <span class="author">Voight BF</span>, <span class="author">Havulinna AS</span>, <span class="author">Wahlstrand B</span>, <span class="author">Hedner T</span>, <span class="author">Corella D</span>, <span class="author">Tai ES</span>, <span class="author">Ordovas JM</span>, <span class="author">Berglund G</span>, <span class="author">Vartiainen E</span>, <span class="author">Jousilahti P</span>, <span class="author">Hedblad B</span>, <span class="author">Taskinen MR</span>, <span class="author">Newton-Cheh C</span>, <span class="author">Salomaa V</span>, <span class="author">Peltonen L</span>, <span class="author">Groop L</span>, <span class="author">Altshuler DM</span> and <span class="author">Orho-Melander M</span></p>
<p class="affiliation">Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA. <a href="mailto:%73%6B%61%74%68%69%72%65%73%61%6E%40%70%61%72%74%6E%65%72%73.%6F%72%67">&#115;&#107;&#97;&#116;&#104;&#105;&#114;&#101;&#115;&#97;&#110;&#64;&#112;&#97;&#114;&#116;&#110;&#101;&#114;&#115;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Blood concentrations of lipoproteins and lipids are heritable risk factors for cardiovascular disease. Using genome-wide association data from three studies (n = 8,816 that included 2,758 individuals from the Diabetes Genetics Initiative specific to the current paper as well as 1,874 individuals from the FUSION study of type 2 diabetes and 4,184 individuals from the SardiNIA study of aging-associated variables reported in a companion paper in this issue) and targeted replication association analyses in up to 18,554 independent participants, we show that common SNPs at 18 loci are reproducibly associated with concentrations of low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol, and/or triglycerides. Six of these loci are new (P &lt; 5 x 10(-8) for each new locus). Of the six newly identified chromosomal regions, two were associated with LDL cholesterol (1p13 near CELSR2, PSRC1 and SORT1 and 19p13 near CILP2 and PBX4), one with HDL cholesterol (1q42 in GALNT2) and five with triglycerides (7q11 near TBL2 and MLXIPL, 8q24 near TRIB1, 1q42 in GALNT2, 19p13 near CILP2 and PBX4 and 1p31 near ANGPTL3). At 1p13, the LDL-associated SNP was also strongly correlated with CELSR2, PSRC1, and SORT1 transcript levels in human liver, and a proxy for this SNP was recently shown to affect risk for coronary artery disease. Understanding the molecular, cellular and clinical consequences of the newly identified loci may inform therapy and clinical care.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">2</span>;<span class="pagerange">189-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18193044">18193044</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682493">2682493</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.75">10.1038/ng.75</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clinical implication of recurrent copy number alterations in hepatocellular carcinoma and putative oncogenes in recurrent gains on 1q.</h4>
<p class="authors"><span class="author">Kim TM</span>, <span class="author">Yim SH</span>, <span class="author">Shin SH</span>, <span class="author">Xu HD</span>, <span class="author">Jung YC</span>, <span class="author">Park CK</span>, <span class="author">Choi JY</span>, <span class="author">Park WS</span>, <span class="author">Kwon MS</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span>, <span class="author">Rhyu MG</span> and <span class="author">Chung YJ</span></p>
<p class="affiliation">Department of Microbiology, The Catholic University of Korea, Seoul, Korea.</p><p class="abstract">To elucidate the pathogenesis of hepatocellular carcinoma (HCC) and develop useful prognosis predictors, it is necessary to identify biologically relevant genomic alterations in HCC. In our study, we defined recurrently altered regions (RARs) common to many cases of HCCs, which may contain tumor-related genes, using whole-genome array-CGH and explored their associations with the clinicopathologic features. Gene set enrichment analysis was performed to investigate functional implication of RARs. On an average, 23.1% of the total probes were altered per case. Mean numbers of altered probes are significantly higher in high-grade, bigger and microvascular invasion (MVI) positive tumors. In total, 32 RARs (14 gains and 18 losses) were defined and 4 most frequent RARs are gains in 1q21.1-q32.1 (64.5%), 1q32.1-q44 (59.2%), 8q11.21-q24.3 (48.7%) and a loss in 17p13.3-p12 (51.3%). Through focusing on RARs, we identified genes and functional pathways likely to be involved in hepatocarcinogenesis. Among genes in the recurrently gained regions on 1q, expression of KIF14 and TPM3 was significantly increased, suggesting their oncogenic potential in HCC. Some RARs showed the significant associations with the clinical features. Especially, the recurrent loss in 9p24.2-p21.1 and gain in 8q11.21-q24.3 are associated with the high tumor grade and MVI, respectively. Functional analysis showed that cytokine receptor binding and defense response to virus pathways are significantly enriched in high grade-related RARs. Taken together, our results and the strategy of analysis will help to elucidate pathogenesis of HCC and to develop biomarkers for predicting behaviors of HCC.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">International journal of cancer. Journal international du cancer</span>&nbsp;<span class="year">2008</span>;<span class="volume">123</span>;<span class="number">12</span>;<span class="pagerange">2808-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18803288" rel="external">18803288</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2698448" rel="external">2698448</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fijc.23901" rel="external">10.1002/ijc.23901</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">iMapper: a web application for the automated analysis and mapping of insertional mutagenesis sequence data against Ensembl genomes.</h4>
<p class="authors"><span class="author">Kong J</span>, <span class="author">Zhu F</span>, <span class="author">Stalker J</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics, The Wellcome Trust Sanger Institute, Hinxton, Cambs CB101HH, UK.</p><p class="abstract">SUMMARY: Insertional mutagenesis is a powerful method for gene discovery. To identify the location of insertion sites in the genome linker based polymerase chain reaction (PCR) methods (such as splinkerette-PCR) may be employed. We have developed a web application called iMapper (Insertional Mutagenesis Mapping and Analysis Tool) for the efficient analysis of insertion site sequence reads against vertebrate and invertebrate Ensembl genomes. Taking linker based sequences as input, iMapper scans and trims the sequence to remove the linker and sequences derived from the insertional mutagen. The software then identifies and removes contaminating sequences derived from chimeric genomic fragments, vector or the transposon concatamer and then presents the clipped sequence reads to a sequence mapping server which aligns them to an Ensembl genome. Insertion sites can then be navigated in Ensembl in the context of genomic features such as gene structures. iMapper also generates test-based format for nucleic acid or protein sequences (FASTA) and generic file format (GFF) files of the clipped sequence reads and provides a graphical overview of the mapped insertion sites against a karyotype. iMapper is designed for high-throughput applications and can efficiently process thousands of DNA sequence reads. AVAILABILITY: iMapper is web based and can be accessed at http://www.sanger.ac.uk/cgi-bin/teams/team113/imapper.cgi.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C20510/A6997</span>; Wellcome Trust: <span class="grant">76943</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">24</span>;<span class="pagerange">2923-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18974167" rel="external">18974167</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2639305" rel="external">2639305</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtn541" rel="external">10.1093/bioinformatics/btn541</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association analysis of allelic variants of USF1 in coronary atherosclerosis.</h4>
<p class="authors"><span class="author">Kristiansson K</span>, <span class="author">Ilveskoski E</span>, <span class="author">Lehtim&auml;ki T</span>, <span class="author">Peltonen L</span>, <span class="author">Perola M</span> and <span class="author">Karhunen PJ</span></p>
<p class="affiliation">Department of Molecular Medicine, National Public Health Institute, Helsinki, Finland.</p><p class="abstract"><strong>Objective</strong>: USF1 regulates the transcription of more than 40 cardiovascular related genes and is well established as a gene associated with familial combined hyperlipidemia, a condition increasing the risk for coronary heart disease. No detailed data, however, exists on the impact of this gene to the critical outcome at the tissue level: different types of atherosclerotic lesions.</p><p class="abstract">We analyzed the USF1 in 2 autopsy series of altogether 700 middle-aged men (the Helsinki Sudden Death Study) with quantitative morphometric measurements of coronary atherosclerosis. SNP rs2516839, tagging common USF1 haplotypes, associated with the presence of several types of atherosclerotic lesions, particularly with the proportion of advanced atherosclerotic plaques (P=0.02) and area of calcified lesions (P&lt;0.001) of the coronary arteries. Importantly, carriers of risk alleles of rs2516839 also showed a 2-fold risk for sudden cardiac death (genotype TT versus CC; OR 2.10, 95% CI 1.17 to 3.75, P=0.04). The risk effect of rs2516839 was present also in aorta samples of the men.</p><p class="abstract"><strong>Conclusions</strong>: Our findings in this unique study sample suggest that USF1 contributes to atherosclerosis, the pathological arterial wall phenotype resulting in coronary heart disease and in its most dramatic consequence-sudden cardiac death.</p>
<p class="publication"><span class="title">Arteriosclerosis, thrombosis, and vascular biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">28</span>;<span class="number">5</span>;<span class="pagerange">983-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18276913" rel="external">18276913</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687549" rel="external">2687549</a>; DOI: <a href="http://dx.doi.org/10.1161%2FATVBAHA.107.156463" rel="external">10.1161/ATVBAHA.107.156463</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Isolated populations and complex disease gene identification.</h4>
<p class="authors"><span class="author">Kristiansson K</span>, <span class="author">Naukkarinen J</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">National Public Health Institute and FIMM, Institute for Molecular Medicine Finland, Helsinki 00300, Finland.</p><p class="abstract">The utility of genetically isolated populations (population isolates) in the mapping and identification of genes is not only limited to the study of rare diseases; isolated populations also provide a useful resource for studies aimed at improved understanding of the biology underlying common diseases and their component traits. Well characterized human populations provide excellent study samples for many different genetic investigations, ranging from genome-wide association studies to the characterization of interactions between genes and the environment.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">8</span>;<span class="pagerange">109</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18771588" rel="external">18771588</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2575505" rel="external">2575505</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-8-109" rel="external">10.1186/gb-2008-9-8-109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular and morphological changes in placenta and embryo development associated with the inhibition of polyamine synthesis during midpregnancy in mice.</h4>
<p class="authors"><span class="author">L&oacute;pez-Garc&iacute;a C</span>, <span class="author">L&oacute;pez-Contreras AJ</span>, <span class="author">Cremades A</span>, <span class="author">Castells MT</span>, <span class="author">Mar&iacute;n F</span>, <span class="author">Schreiber F</span> and <span class="author">Pe&ntilde;afiel R</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology B and Immunology, Faculty of Medicine, University of Murcia, Campus de Espinardo, 30100 Murcia, Spain.</p><p class="abstract">Polyamines play an essential role in murine development, as demonstrated by both gene ablation in ornithine decarboxylase (ODC)-deficient embryos and pharmacological treatments of pregnant mice. However, the molecular and cellular mechanisms by which ODC inhibition affects embryonic development during critical periods of pregnancy are mostly unknown. Our present results demonstrate that the contragestational effect of alpha-difluoromethylornithine (DFMO), a suicide inhibitor of ODC, when given at d 7-9 of pregnancy, is associated with embryo growth arrest and marked alterations in the development of yolk sac and placenta. Blood island formation as well as the transcript levels of embryonary globins alpha-like x chain and beta-like y-chain was markedly decreased in the yolk sac. At the placental level, abnormal chorioallantoic attachment, absence of the spongiotrophoblast layer and a deficient development of the labyrinthine zone were evident. Real-time RT-PCR analysis showed that transcript levels of the steroidogenic genes steroidogenic acute regulatory protein, 3beta-hydroxysteroid dehydrogenase VI, and 17alpha-hydroxylase were markedly decreased by DFMO treatment in the developing placenta at d 9 and 10 of pregnancy. Plasma values of progesterone and androstenedione were also decreased by DFMO treatment. Transcriptomic analysis also detected changes in the expression of several genes involved in placentation and the differentiation of trophoblastic lineages. In conclusion, our results indicate that ODC inhibition at d 8 of pregnancy is related to alterations in yolk sac formation and trophoblast differentiation, affecting processes such as vasculogenesis and steroidogenesis.</p>
<p class="publication"><span class="title">Endocrinology</span>&nbsp;<span class="year">2008</span>;<span class="volume">149</span>;<span class="number">10</span>;<span class="pagerange">5012-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18583422" rel="external">18583422</a>; DOI: <a href="http://dx.doi.org/10.1210%2Fen.2008-0084" rel="external">10.1210/en.2008-0084</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Normal germ line establishment in mice carrying a deletion of the Ifitm/Fragilis gene family cluster.</h4>
<p class="authors"><span class="author">Lange UC</span>, <span class="author">Adams DJ</span>, <span class="author">Lee C</span>, <span class="author">Barton S</span>, <span class="author">Schneider R</span>, <span class="author">Bradley A</span> and <span class="author">Surani MA</span></p>
<p class="affiliation">Wellcome Trust/Cancer Research UK Gurdon Institute of Cancer &amp; Developmental Biology, University of Cambridge, Tennis Court Road, Cambridge CB2 1QN, United Kingdom.</p><p class="abstract">The family of interferon-inducible transmembrane proteins (Ifitm) consists of five highly sequence-related cell surface proteins, which are implicated in diverse cellular processes. Ifitm genes are conserved, widely expressed, and characteristically found in genomic clusters, such as the 67-kb Ifitm family locus on mouse chromosome 7. Recently, Ifitm1 and Ifitm3 have been suggested to mediate migration of early primordial germ cells (PGCs), a process that is little understood. To investigate Ifitm function during germ cell development, we used targeted chromosome engineering to generate mutants which either lack the entire Ifitm locus or carry a disrupted Ifitm3 gene only. Here we show that the mutations have no detectable effects on development of the germ line or on the generation of live young. Hence, contrary to previous reports, Ifitm genes are not essential for PGC migration. The Ifitm family is a striking example of a conserved gene cluster which appears to be functionally redundant during development.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Cancer Research UK; Wellcome Trust: <span class="grant">065601</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">28</span>;<span class="number">15</span>;<span class="pagerange">4688-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18505827">18505827</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2493357">2493357</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.00272-08">10.1128/MCB.00272-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Invasive Salmonellosis in Humans</h4>
<p class="authors"><span class="author">Langridge</span>, <span class="author">G.C.</span>, <span class="author">Wain</span>, <span class="author">J.</span>, <span class="author">Nair</span>, <span class="author">S.</span></p>
<p class="publication"><span class="title">Ecosal.</span> <span class="year">2008</span>;<span class="volume">Module8.6.2.2</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of the Rab protein family in Apicomplexan parasites</h4>
<p class="authors"><span class="author">Langsley G.</span></p>
<p class="publication"><span class="title">Microbes and Infection.</span> <span class="year">2008</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Host transmission of Salmonella enterica serovar Typhimurium is controlled by virulence factors and indigenous intestinal microbiota.</h4>
<p class="authors"><span class="author">Lawley TD</span>, <span class="author">Bouley DM</span>, <span class="author">Hoy YE</span>, <span class="author">Gerke C</span>, <span class="author">Relman DA</span> and <span class="author">Monack DM</span></p>
<p class="affiliation">Department of Microbiology and Immunology, 299 Campus Drive, Stanford University, Stanford, CA 94305, USA.</p><p class="abstract">Transmission is an essential stage of a pathogen&#39;s life cycle and remains poorly understood. We describe here a model in which persistently infected 129X1/SvJ mice provide a natural model of Salmonella enterica serovar Typhimurium transmission. In this model only a subset of the infected mice, termed supershedders, shed high levels (&gt;10(8) CFU/g) of Salmonella serovar Typhimurium in their feces and, as a result, rapidly transmit infection. While most Salmonella serovar Typhimurium-infected mice show signs of intestinal inflammation, only supershedder mice develop colitis. Development of the supershedder phenotype depends on the virulence determinants Salmonella pathogenicity islands 1 and 2, and it is characterized by mucosal invasion and, importantly, high luminal abundance of Salmonella serovar Typhimurium within the colon. Immunosuppression of infected mice does not induce the supershedder phenotype, demonstrating that the immune response is not the main determinant of Salmonella serovar Typhimurium levels within the colon. In contrast, treatment of mice with antibiotics that alter the health-associated indigenous intestinal microbiota rapidly induces the supershedder phenotype in infected mice and predisposes uninfected mice to the supershedder phenotype for several days. These results demonstrate that the intestinal microbiota plays a critical role in controlling Salmonella serovar Typhimurium infection, disease, and transmissibility. This novel model should facilitate the study of host, pathogen, and intestinal microbiota factors that contribute to infectious disease transmission.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI26195</span></p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2008</span>;<span class="volume">76</span>;<span class="number">1</span>;<span class="pagerange">403-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17967858">17967858</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2223630">2223630</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.01189-07">10.1128/IAI.01189-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A single gene network accurately predicts phenotypic effects of gene perturbation in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Lee I</span>, <span class="author">Lehner B</span>, <span class="author">Crombie C</span>, <span class="author">Wong W</span>, <span class="author">Fraser AG</span> and <span class="author">Marcotte EM</span></p>
<p class="affiliation">Center for Systems and Synthetic Biology, Department of Chemistry and Biochemistry, Institute for Cellular and Molecular Biology, University of Texas, 2500 Speedway, MBB 3.210, Austin, Texas 78712, USA.</p><p class="abstract">The fundamental aim of genetics is to understand how an organism&#39;s phenotype is determined by its genotype, and implicit in this is predicting how changes in DNA sequence alter phenotypes. A single network covering all the genes of an organism might guide such predictions down to the level of individual cells and tissues. To validate this approach, we computationally generated a network covering most C. elegans genes and tested its predictive capacity. Connectivity within this network predicts essentiality, identifying this relationship as an evolutionarily conserved biological principle. Critically, the network makes tissue-specific predictions-we accurately identify genes for most systematically assayed loss-of-function phenotypes, which span diverse cellular and developmental processes. Using the network, we identify 16 genes whose inactivation suppresses defects in the retinoblastoma tumor suppressor pathway, and we successfully predict that the dystrophin complex modulates EGF signaling. We conclude that an analogous network for human genes might be similarly predictive and thus facilitate identification of disease genes and rational therapeutic targets.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM06779-01</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">2</span>;<span class="pagerange">181-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18223650">18223650</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2007.70">10.1038/ng.2007.70</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of the Rhodococcus equi vap pathogenicity island seen through comparison of host-associated vapA and vapB virulence plasmids.</h4>
<p class="authors"><span class="author">Letek M</span>, <span class="author">Ocampo-Sosa AA</span>, <span class="author">Sanders M</span>, <span class="author">Fogarty U</span>, <span class="author">Buckley T</span>, <span class="author">Leadon DP</span>, <span class="author">Gonz&aacute;lez P</span>, <span class="author">Scortti M</span>, <span class="author">Meijer WG</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley S</span> and <span class="author">V&aacute;zquez-Boland JA</span></p>
<p class="affiliation">Division of Microbial Pathogenesis, Centre for Infectious Diseases, Ashworth Laboratories, King&#39;s Buildings, University of Edinburgh, Edinburgh EH9 3JT, United Kingdom.</p><p class="abstract">The pathogenic actinomycete Rhodococcus equi harbors different types of virulence plasmids associated with specific nonhuman hosts. We determined the complete DNA sequence of a vapB(+) plasmid, typically associated with pig isolates, and compared it with that of the horse-specific vapA(+) plasmid type. pVAPB1593, a circular 79,251-bp element, had the same housekeeping backbone as the vapA(+) plasmid but differed over an approximately 22-kb region. This variable region encompassed the vap pathogenicity island (PAI), was clearly subject to selective pressures different from those affecting the backbone, and showed major genetic rearrangements involving the vap genes. The pVAPB1593 PAI harbored five different vap genes (vapB and vapJ to -M, with vapK present in two copies), which encoded products differing by 24 to 84% in amino acid sequence from the six full-length vapA(+) plasmid-encoded Vap proteins, consistent with a role for the specific vap gene complement in R. equi host tropism. Sequence analyses, including interpolated variable-order motifs for detection of alien DNA and reconstruction of Vap family phylogenetic relationships, suggested that the vap PAI was acquired by an ancestor plasmid via lateral gene transfer, subsequently evolving by vap gene duplication and sequence diversification to give different (host-adapted) plasmids. The R. equi virulence plasmids belong to a new family of actinobacterial circular replicons characterized by an ancient conjugative backbone and a horizontally acquired niche-adaptive plasticity region.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">17</span>;<span class="pagerange">5797-805</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18606735">18606735</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2519538">2519538</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00468-08">10.1128/JB.00468-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of ten loci associated with height highlights new biological pathways in human growth.</h4>
<p class="authors"><span class="author">Lettre G</span>, <span class="author">Jackson AU</span>, <span class="author">Gieger C</span>, <span class="author">Schumacher FR</span>, <span class="author">Berndt SI</span>, <span class="author">Sanna S</span>, <span class="author">Eyheramendy S</span>, <span class="author">Voight BF</span>, <span class="author">Butler JL</span>, <span class="author">Guiducci C</span>, <span class="author">Illig T</span>, <span class="author">Hackett R</span>, <span class="author">Heid IM</span>, <span class="author">Jacobs KB</span>, <span class="author">Lyssenko V</span>, <span class="author">Uda M</span>, <span class="author">Diabetes Genetics Initiative</span>, <span class="author">FUSION</span>, <span class="author">KORA</span>, <span class="author">Prostate, Lung Colorectal and Ovarian Cancer Screening Trial</span>, <span class="author">Nurses&#39; Health Study</span>, <span class="author">SardiNIA</span>, <span class="author">Boehnke M</span>, <span class="author">Chanock SJ</span>, <span class="author">Groop LC</span>, <span class="author">Hu FB</span>, <span class="author">Isomaa B</span>, <span class="author">Kraft P</span>, <span class="author">Peltonen L</span>, <span class="author">Salomaa V</span>, <span class="author">Schlessinger D</span>, <span class="author">Hunter DJ</span>, <span class="author">Hayes RB</span>, <span class="author">Abecasis GR</span>, <span class="author">Wichmann HE</span>, <span class="author">Mohlke KL</span> and <span class="author">Hirschhorn JN</span></p>
<p class="affiliation">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA.</p><p class="abstract">Height is a classic polygenic trait, reflecting the combined influence of multiple as-yet-undiscovered genetic factors. We carried out a meta-analysis of genome-wide association study data of height from 15,821 individuals at 2.2 million SNPs, and followed up the strongest findings in &gt;10,000 subjects. Ten newly identified and two previously reported loci were strongly associated with variation in height (P values from 4 x 10(-7) to 8 x 10(-22)). Together, these 12 loci account for approximately 2% of the population variation in height. Individuals with &lt; or =8 height-increasing alleles and &gt; or =16 height-increasing alleles differ in height by approximately 3.5 cm. The newly identified loci, along with several additional loci with strongly suggestive associations, encompass both strong biological candidates and unexpected genes, and highlight several pathways (let-7 targets, chromatin remodeling proteins and Hedgehog signaling) as important regulators of human stature. These results expand the picture of the biological regulation of human height and of the genetic architecture of this classical complex trait.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">5P01CA087969</span>, <span class="grant">5U01CA098233</span>, <span class="grant">CA49449</span>; NHGRI NIH HHS: <span class="grant">HG02651</span>; NHLBI NIH HHS: <span class="grant">HL084729</span>; NIDDK NIH HHS: <span class="grant">5 R01 DK 075787</span>, <span class="grant">DK62370</span>, <span class="grant">DK72193</span>, <span class="grant">R01 DK072193</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">5</span>;<span class="pagerange">584-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18391950">18391950</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687076">2687076</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.125">10.1038/ng.125</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping short DNA sequencing reads and calling variants using mapping quality scores.</h4>
<p class="authors"><span class="author">Li H</span>, <span class="author">Ruan J</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton CB10 1SA, United Kingdom.</p><p class="abstract">New sequencing technologies promise a new era in the use of DNA sequence. However, some of these technologies produce very short reads, typically of a few tens of base pairs, and to use these reads effectively requires new algorithms and software. In particular, there is a major issue in efficiently aligning short reads to a reference genome and handling ambiguity or lack of accuracy in this alignment. Here we introduce the concept of mapping quality, a measure of the confidence that a read actually comes from the position it is aligned to by the mapping algorithm. We describe the software MAQ that can build assemblies by mapping shotgun short reads to a reference genome, using quality scores to derive genotype calls of the consensus sequence of a diploid genome, e.g., from a human sample. MAQ makes full use of mate-pair information and estimates the error probability of each read alignment. Error probabilities are also derived for the final genotype calls, using a Bayesian statistical model that incorporates the mapping qualities, error probabilities from the raw sequence quality scores, sampling of the two haplotypes, and an empirical model for correlated errors at a site. Both read mapping and genotype calling are evaluated on simulated data and real data. MAQ is accurate, efficient, versatile, and user-friendly. It is freely available at http://maq.sourceforge.net.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">11</span>;<span class="pagerange">1851-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18714091">18714091</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2577856">2577856</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.078212.108">10.1101/gr.078212.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Curcumin prevents and reverses murine cardiac hypertrophy.</h4>
<p class="authors"><span class="author">Li HL</span>, <span class="author">Liu C</span>, <span class="author">de Couto G</span>, <span class="author">Ouzounian M</span>, <span class="author">Sun M</span>, <span class="author">Wang AB</span>, <span class="author">Huang Y</span>, <span class="author">He CW</span>, <span class="author">Shi Y</span>, <span class="author">Chen X</span>, <span class="author">Nghiem MP</span>, <span class="author">Liu Y</span>, <span class="author">Chen M</span>, <span class="author">Dawood F</span>, <span class="author">Fukuoka M</span>, <span class="author">Maekawa Y</span>, <span class="author">Zhang L</span>, <span class="author">Leask A</span>, <span class="author">Ghosh AK</span>, <span class="author">Kirshenbaum LA</span> and <span class="author">Liu PP</span></p>
<p class="affiliation">Division of Cardiology, Heart and Stroke/Richard Lewar Centre of Excellence, University Health Network, University of Toronto, Toronto, Ontario, Canada.</p><p class="abstract">Chromatin remodeling, particularly histone acetylation, plays a critical role in the progression of pathological cardiac hypertrophy and heart failure. We hypothesized that curcumin, a natural polyphenolic compound abundant in the spice turmeric and a known suppressor of histone acetylation, would suppress cardiac hypertrophy through the disruption of p300 histone acetyltransferase-dependent (p300-HAT-dependent) transcriptional activation. We tested this hypothesis using primary cultured rat cardiac myocytes and fibroblasts as well as two well-established mouse models of cardiac hypertrophy. Curcumin blocked phenylephrin-induced (PE-induced) cardiac hypertrophy in vitro in a dose-dependent manner. Furthermore, curcumin both prevented and reversed mouse cardiac hypertrophy induced by aortic banding (AB) and PE infusion, as assessed by heart weight/BW and lung weight/BW ratios, echocardiographic parameters, and gene expression of hypertrophic markers. Further investigation demonstrated that curcumin abrogated histone acetylation, GATA4 acetylation, and DNA-binding activity through blocking p300-HAT activity. Curcumin also blocked AB-induced inflammation and fibrosis through disrupting p300-HAT-dependent signaling pathways. Our results indicate that curcumin has the potential to protect against cardiac hypertrophy, inflammation, and fibrosis through suppression of p300-HAT activity and downstream GATA4, NF-kappaB, and TGF-beta-Smad signaling pathways.</p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2008</span>;<span class="volume">118</span>;<span class="number">3</span>;<span class="pagerange">879-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18292803" rel="external">18292803</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2248327" rel="external">2248327</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI32865" rel="external">10.1172/JCI32865</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the DND1 gene in men with sporadic and familial testicular germ cell tumors.</h4>
<p class="authors"><span class="author">Linger R</span>, <span class="author">Dudakia D</span>, <span class="author">Huddart R</span>, <span class="author">Tucker K</span>, <span class="author">Friedlander M</span>, <span class="author">Phillips KA</span>, <span class="author">Hogg D</span>, <span class="author">Jewett MA</span>, <span class="author">Lohynska R</span>, <span class="author">Daugaard G</span>, <span class="author">Richard S</span>, <span class="author">Chompret A</span>, <span class="author">Stoppa-Lyonnet D</span>, <span class="author">Bona&iuml;ti-Pelli&eacute; C</span>, <span class="author">Heidenreich A</span>, <span class="author">Albers P</span>, <span class="author">Olah E</span>, <span class="author">Geczi L</span>, <span class="author">Bodrogi I</span>, <span class="author">Daly PA</span>, <span class="author">Guilford P</span>, <span class="author">Foss&aring; SD</span>, <span class="author">Heimdal K</span>, <span class="author">Tjulandin SA</span>, <span class="author">Liubchenko L</span>, <span class="author">Stoll H</span>, <span class="author">Weber W</span>, <span class="author">Einhorn L</span>, <span class="author">McMaster M</span>, <span class="author">Korde L</span>, <span class="author">Greene MH</span>, <span class="author">Nathanson KL</span>, <span class="author">Cortessis V</span>, <span class="author">Easton DF</span>, <span class="author">Bishop DT</span>, <span class="author">Stratton MR</span> and <span class="author">Rapley EA</span></p>
<p class="affiliation">Testicular Cancer Genetics Team, Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK.</p><p class="abstract">A base substitution in the mouse Dnd1 gene resulting in a truncated Dnd protein has been shown to be responsible for germ cell loss and the development of testicular germ cell tumors (TGCT) in the 129 strain of mice. We investigated the human orthologue of this gene in 263 patients (165 with a family history of TGCT and 98 without) and found a rare heterozygous variant, p. Glu86Ala, in a single case. This variant was not present in control chromosomes (0/4,132). Analysis of the variant in an additional 842 index TGCT cases (269 with a family history of TGCT and 573 without) did not reveal any additional instances. The variant, p. Glu86Ala, is within a known functional domain of DND1 and is highly conserved through evolution. Although the variant may be a rare polymorphism, a change at such a highly conserved residue is characteristic of a disease-causing variant. Whether it is disease-causing or not, mutations in DND1 make, at most, a very small contribution to TGCT susceptibility in adults and adolescents.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">1R01 CA102042-01A1</span>, <span class="grant">ZIA CP010144-12</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2008</span>;<span class="volume">47</span>;<span class="number">3</span>;<span class="pagerange">247-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18069663" rel="external">18069663</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3109865" rel="external">3109865</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20526" rel="external">10.1002/gcc.20526</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lifelong reduction of LDL-cholesterol related to a common variant in the LDL-receptor gene decreases the risk of coronary artery disease--a Mendelian Randomisation study.</h4>
<p class="authors"><span class="author">Linsel-Nitschke P</span>, <span class="author">G&ouml;tz A</span>, <span class="author">Erdmann J</span>, <span class="author">Braenne I</span>, <span class="author">Braund P</span>, <span class="author">Hengstenberg C</span>, <span class="author">Stark K</span>, <span class="author">Fischer M</span>, <span class="author">Schreiber S</span>, <span class="author">El Mokhtari NE</span>, <span class="author">Schaefer A</span>, <span class="author">Schrezenmeir J</span>, <span class="author">Schrezenmeier J</span>, <span class="author">Rubin D</span>, <span class="author">Hinney A</span>, <span class="author">Reinehr T</span>, <span class="author">Roth C</span>, <span class="author">Ortlepp J</span>, <span class="author">Hanrath P</span>, <span class="author">Hall AS</span>, <span class="author">Mangino M</span>, <span class="author">Lieb W</span>, <span class="author">Lamina C</span>, <span class="author">Heid IM</span>, <span class="author">Doering A</span>, <span class="author">Gieger C</span>, <span class="author">Peters A</span>, <span class="author">Meitinger T</span>, <span class="author">Wichmann HE</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Ziegler A</span>, <span class="author">Kronenberg F</span>, <span class="author">Samani NJ</span>, <span class="author">Schunkert H</span>, <span class="author">Wellcome Trust Case Control Consortium (WTCCC)</span> and <span class="author">Cardiogenics Consortium</span></p>
<p class="affiliation">Medizinische Klinik II, Universit&auml;t zu L&uuml;beck, L&uuml;beck, Germany.</p><p class="abstract"><strong>Background</strong>: Rare mutations of the low-density lipoprotein receptor gene (LDLR) cause familial hypercholesterolemia, which increases the risk for coronary artery disease (CAD). Less is known about the implications of common genetic variation in the LDLR gene regarding the variability of cholesterol levels and risk of CAD.</p><p class="abstract"><strong>Methods</strong>: Imputed genotype data at the LDLR locus on 1 644 individuals of a population-based sample were explored for association with LDL-C level. Replication of association with LDL-C level was sought for the most significant single nucleotide polymorphism (SNP) within the LDLR gene in three European samples comprising 6 642 adults and 533 children. Association of this SNP with CAD was examined in six case-control studies involving more than 15 000 individuals.</p><p class="abstract"><strong>Findings</strong>: Each copy of the minor T allele of SNP rs2228671 within LDLR (frequency 11%) was related to a decrease of LDL-C levels by 0.19 mmol/L (95% confidence interval (CI) [0.13-0.24] mmol/L, p = 1.5x10(-10)). This association with LDL-C was uniformly found in children, men, and women of all samples studied. In parallel, the T allele of rs2228671 was associated with a significantly lower risk of CAD (Odds Ratio per copy of the T allele: 0.82, 95% CI [0.76-0.89], p = 2.1x10(-7)). Adjustment for LDL-C levels by logistic regression or Mendelian Randomisation models abolished the significant association between rs2228671 with CAD completely, indicating a functional link between the genetic variant at the LDLR gene locus, change in LDL-C and risk of CAD.</p><p class="abstract"><strong>Conclusion</strong>: A common variant at the LDLR gene locus affects LDL-C levels and, thereby, the risk for CAD.</p>
<p class="grants">Funded by: British Heart Foundation; Medical Research Council</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">8</span>;<span class="pagerange">e2986</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18714375">18714375</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2500189">2500189</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0002986">10.1371/journal.pone.0002986</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The conserved plant sterility gene HAP2 functions after attachment of fusogenic membranes in Chlamydomonas and Plasmodium gametes.</h4>
<p class="authors"><span class="author">Liu Y</span>, <span class="author">Tewari R</span>, <span class="author">Ning J</span>, <span class="author">Blagborough AM</span>, <span class="author">Garbom S</span>, <span class="author">Pei J</span>, <span class="author">Grishin NV</span>, <span class="author">Steele RE</span>, <span class="author">Sinden RE</span>, <span class="author">Snell WJ</span> and <span class="author">Billker O</span></p>
<p class="affiliation">Department of Cell Biology, University of Texas Southwestern Medical Center, Dallas, Texas 75390, USA.</p><p class="abstract">The cellular and molecular mechanisms that underlie species-specific membrane fusion between male and female gametes remain largely unknown. Here, by use of gene discovery methods in the green alga Chlamydomonas, gene disruption in the rodent malaria parasite Plasmodium berghei, and distinctive features of fertilization in both organisms, we report discovery of a mechanism that accounts for a conserved protein required for gamete fusion. A screen for fusion mutants in Chlamydomonas identified a homolog of HAP2, an Arabidopsis sterility gene. Moreover, HAP2 disruption in Plasmodium blocked fertilization and thereby mosquito transmission of malaria. HAP2 localizes at the fusion site of Chlamydomonas minus gametes, yet Chlamydomonas minus and Plasmodium hap2 male gametes retain the ability, using other, species-limited proteins, to form tight prefusion membrane attachments with their respective gamete partners. Membrane dye experiments show that HAP2 is essential for membrane merger. Thus, in two distantly related eukaryotes, species-limited proteins govern access to a conserved protein essential for membrane fusion.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council: <span class="grant">G0501670</span>; NIGMS NIH HHS: <span class="grant">GM056778</span>, <span class="grant">R01 GM056778</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2008</span>;<span class="volume">22</span>;<span class="number">8</span>;<span class="pagerange">1051-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18367645">18367645</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2335326">2335326</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.1656508">10.1101/gad.1656508</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of PLCL1 gene for hip bone size variation in females in a genome-wide association study.</h4>
<p class="authors"><span class="author">Liu YZ</span>, <span class="author">Wilson SG</span>, <span class="author">Wang L</span>, <span class="author">Liu XG</span>, <span class="author">Guo YF</span>, <span class="author">Li J</span>, <span class="author">Yan H</span>, <span class="author">Deloukas P</span>, <span class="author">Soranzo N</span>, <span class="author">Chinappen-Horsley U</span>, <span class="author">Chinnapen-Horsley U</span>, <span class="author">Cervino A</span>, <span class="author">Cervino A</span>, <span class="author">Williams FM</span>, <span class="author">Xiong DH</span>, <span class="author">Zhang YP</span>, <span class="author">Jin TB</span>, <span class="author">Levy S</span>, <span class="author">Papasian CJ</span>, <span class="author">Drees BM</span>, <span class="author">Hamilton JJ</span>, <span class="author">Recker RR</span>, <span class="author">Spector TD</span> and <span class="author">Deng HW</span></p>
<p class="affiliation">School of Medicine, University of Missouri - Kansas City, Kansas City, Missouri, United States of America.</p><p class="abstract">Osteoporosis, the most prevalent metabolic bone disease among older people, increases risk for low trauma hip fractures (HF) that are associated with high morbidity and mortality. Hip bone size (BS) has been identified as one of the key measurable risk factors for HF. Although hip BS is highly genetically determined, genetic factors underlying the trait are still poorly defined. Here, we performed the first genome-wide association study (GWAS) of hip BS interrogating approximately 380,000 SNPs on the Affymetrix platform in 1,000 homogeneous unrelated Caucasian subjects, including 501 females and 499 males. We identified a gene, PLCL1 (phospholipase c-like 1), that had four SNPs associated with hip BS at, or approaching, a genome-wide significance level in our female subjects; the most significant SNP, rs7595412, achieved a p value of 3.72x10(-7). The gene&#39;s importance to hip BS was replicated using the Illumina genotyping platform in an independent UK cohort containing 1,216 Caucasian females. Two SNPs of the PLCL1 gene, rs892515 and rs9789480, surrounded by the four SNPs identified in our GWAS, achieved p values of 8.62x10(-3) and 2.44x10(-3), respectively, for association with hip BS. Imputation analyses on our GWAS and the UK samples further confirmed the replication signals; eight SNPs of the gene achieved combined imputed p values&lt;10(-5) in the two samples. The PLCL1 gene&#39;s relevance to HF was also observed in a Chinese sample containing 403 females, including 266 with HF and 177 control subjects. A SNP of the PLCL1 gene, rs3771362 that is only approximately 0.6 kb apart from the most significant SNP detected in our GWAS (rs7595412), achieved a p value of 7.66x10(-3) (odds ratio = 0.26) for association with HF. Additional biological support for the role of PLCL1 in BS comes from previous demonstrations that the PLCL1 protein inhibits IP3 (inositol 1,4,5-trisphosphate)-mediated calcium signaling, an important pathway regulating mechanical sensing of bone cells. Our findings suggest that PLCL1 is a novel gene associated with variation in hip BS, and provide new insights into the pathogenesis of HF.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">R01 AG026564</span>, <span class="grant">R21 AG027110</span>; NIAAA NIH HHS: <span class="grant">R21 AA015973</span>; NIAMS NIH HHS: <span class="grant">P50 AR055081</span>, <span class="grant">R01 AR050496-01</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">9</span>;<span class="pagerange">e3160</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18776929" rel="external">18776929</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2522269" rel="external">2522269</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0003160" rel="external">10.1371/journal.pone.0003160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants near MC4R are associated with fat mass, weight and risk of obesity.</h4>
<p class="authors"><span class="author">Loos RJ</span>, <span class="author">Lindgren CM</span>, <span class="author">Li S</span>, <span class="author">Wheeler E</span>, <span class="author">Zhao JH</span>, <span class="author">Prokopenko I</span>, <span class="author">Inouye M</span>, <span class="author">Freathy RM</span>, <span class="author">Attwood AP</span>, <span class="author">Beckmann JS</span>, <span class="author">Berndt SI</span>, <span class="author">Prostate, Lung, Colorectal, and Ovarian (PLCO) Cancer Screening Trial</span>, <span class="author">Jacobs KB</span>, <span class="author">Chanock SJ</span>, <span class="author">Hayes RB</span>, <span class="author">Bergmann S</span>, <span class="author">Bennett AJ</span>, <span class="author">Bingham SA</span>, <span class="author">Bochud M</span>, <span class="author">Brown M</span>, <span class="author">Cauchi S</span>, <span class="author">Connell JM</span>, <span class="author">Cooper C</span>, <span class="author">Smith GD</span>, <span class="author">Day I</span>, <span class="author">Dina C</span>, <span class="author">De S</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Doney AS</span>, <span class="author">Elliott KS</span>, <span class="author">Elliott P</span>, <span class="author">Evans DM</span>, <span class="author">Sadaf Farooqi I</span>, <span class="author">Froguel P</span>, <span class="author">Ghori J</span>, <span class="author">Groves CJ</span>, <span class="author">Gwilliam R</span>, <span class="author">Hadley D</span>, <span class="author">Hall AS</span>, <span class="author">Hattersley AT</span>, <span class="author">Hebebrand J</span>, <span class="author">Heid IM</span>, <span class="author">KORA</span>, <span class="author">Lamina C</span>, <span class="author">Gieger C</span>, <span class="author">Illig T</span>, <span class="author">Meitinger T</span>, <span class="author">Wichmann HE</span>, <span class="author">Herrera B</span>, <span class="author">Hinney A</span>, <span class="author">Hunt SE</span>, <span class="author">Jarvelin MR</span>, <span class="author">Johnson T</span>, <span class="author">Jolley JD</span>, <span class="author">Karpe F</span>, <span class="author">Keniry A</span>, <span class="author">Khaw KT</span>, <span class="author">Luben RN</span>, <span class="author">Mangino M</span>, <span class="author">Marchini J</span>, <span class="author">McArdle WL</span>, <span class="author">McGinnis R</span>, <span class="author">Meyre D</span>, <span class="author">Munroe PB</span>, <span class="author">Morris AD</span>, <span class="author">Ness AR</span>, <span class="author">Neville MJ</span>, <span class="author">Nica AC</span>, <span class="author">Ong KK</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Owen KR</span>, <span class="author">Palmer CN</span>, <span class="author">Papadakis K</span>, <span class="author">Potter S</span>, <span class="author">Pouta A</span>, <span class="author">Qi L</span>, <span class="author">Nurses&#39; Health Study</span>, <span class="author">Randall JC</span>, <span class="author">Rayner NW</span>, <span class="author">Ring SM</span>, <span class="author">Sandhu MS</span>, <span class="author">Scherag A</span>, <span class="author">Sims MA</span>, <span class="author">Song K</span>, <span class="author">Soranzo N</span>, <span class="author">Speliotes EK</span>, <span class="author">Diabetes Genetics Initiative</span>, <span class="author">Syddall HE</span>, <span class="author">Teichmann SA</span>, <span class="author">Timpson NJ</span>, <span class="author">Tobias JH</span>, <span class="author">Uda M</span>, <span class="author">SardiNIA Study</span>, <span class="author">Vogel CI</span>, <span class="author">Wallace C</span>, <span class="author">Waterworth DM</span>, <span class="author">Weedon MN</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Willer CJ</span>, <span class="author">FUSION</span>, <span class="author">Wraight</span>, <span class="author">Yuan X</span>, <span class="author">Zeggini E</span>, <span class="author">Hirschhorn JN</span>, <span class="author">Strachan DP</span>, <span class="author">Ouwehand WH</span>, <span class="author">Caulfield MJ</span>, <span class="author">Samani NJ</span>, <span class="author">Frayling TM</span>, <span class="author">Vollenweider P</span>, <span class="author">Waeber G</span>, <span class="author">Mooser V</span>, <span class="author">Deloukas P</span>, <span class="author">McCarthy MI</span>, <span class="author">Wareham NJ</span>, <span class="author">Barroso I</span>, <span class="author">Jacobs KB</span>, <span class="author">Chanock SJ</span>, <span class="author">Hayes RB</span>, <span class="author">Lamina C</span>, <span class="author">Gieger C</span>, <span class="author">Illig T</span>, <span class="author">Meitinger T</span>, <span class="author">Wichmann HE</span>, <span class="author">Kraft P</span>, <span class="author">Hankinson SE</span>, <span class="author">Hunter DJ</span>, <span class="author">Hu FB</span>, <span class="author">Lyon HN</span>, <span class="author">Voight BF</span>, <span class="author">Ridderstrale M</span>, <span class="author">Groop L</span>, <span class="author">Scheet P</span>, <span class="author">Sanna S</span>, <span class="author">Abecasis GR</span>, <span class="author">Albai G</span>, <span class="author">Nagaraja R</span>, <span class="author">Schlessinger D</span>, <span class="author">Jackson AU</span>, <span class="author">Tuomilehto J</span>, <span class="author">Collins FS</span>, <span class="author">Boehnke M</span> and <span class="author">Mohlke KL</span></p>
<p class="affiliation">MRC Epidemiology Unit, Addenbrooke&#39;s Hospital, Cambridge CB2 0QQ, UK.</p><p class="abstract">To identify common variants influencing body mass index (BMI), we analyzed genome-wide association data from 16,876 individuals of European descent. After previously reported variants in FTO, the strongest association signal (rs17782313, P = 2.9 x 10(-6)) mapped 188 kb downstream of MC4R (melanocortin-4 receptor), mutations of which are the leading cause of monogenic severe childhood-onset obesity. We confirmed the BMI association in 60,352 adults (per-allele effect = 0.05 Z-score units; P = 2.8 x 10(-15)) and 5,988 children aged 7-11 (0.13 Z-score units; P = 1.5 x 10(-8)). In case-control analyses (n = 10,583), the odds for severe childhood obesity reached 1.30 (P = 8.0 x 10(-11)). Furthermore, we observed overtransmission of the risk allele to obese offspring in 660 families (P (pedigree disequilibrium test average; PDT-avg) = 2.4 x 10(-4)). The SNP location and patterns of phenotypic associations are consistent with effects mediated through altered MC4R function. Our findings establish that common variants near MC4R influence fat mass, weight and obesity risk at the population level and reinforce the need for large-scale data integration to identify variants influencing continuous biomedical traits.</p>
<p class="grants">Funded by: British Heart Foundation; Cancer Research UK; Department of Health: <span class="grant">DHCS/07/07/008</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0000934(68341)</span>, <span class="grant">G0400874</span>, <span class="grant">G0401527</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">G0701863</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010(63660)</span>, <span class="grant">G9824984</span>, <span class="grant">G9828345</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U105161047</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>, <span class="grant">MC_U147585824</span>, <span class="grant">MC_UP_A620_1014</span>, <span class="grant">U.1475.00.002.00001.01 (85824)</span>; NIDDK NIH HHS: <span class="grant">F32 DK079466</span>, <span class="grant">F32 DK079466-01</span>, <span class="grant">K23 DK080145</span>, <span class="grant">K23 DK080145-01</span>, <span class="grant">P30 DK040561</span>, <span class="grant">P30 DK040561-13</span>, <span class="grant">R01 DK072193</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">076113</span>, <span class="grant">077016</span>, <span class="grant">079557</span>, <span class="grant">084713</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">6</span>;<span class="pagerange">768-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18454148">18454148</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2669167">2669167</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.140">10.1038/ng.140</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sites of strong Rec12/Spo11 binding in the fission yeast genome are associated with meiotic recombination and with centromeres.</h4>
<p class="authors"><span class="author">Ludin K</span>, <span class="author">Mata J</span>, <span class="author">Watt S</span>, <span class="author">Lehmann E</span>, <span class="author">B&auml;hler J</span> and <span class="author">Kohli J</span></p>
<p class="affiliation">Institute of Cell Biology, University of Bern, 3012 Bern, Switzerland. <a href="mailto:%6B%61%74%6A%61.%6C%75%64%69%6E%40%69%7A%62.%75%6E%69%62%65%72%6E.%63%68">&#107;&#97;&#116;&#106;&#97;&#46;&#108;&#117;&#100;&#105;&#110;&#64;&#105;&#122;&#98;&#46;&#117;&#110;&#105;&#98;&#101;&#114;&#110;&#46;&#99;&#104;</a></p><p class="abstract">Meiotic recombination arises from Rec12/Spo11-dependent formation of DNA double-strand breaks (DSBs) and their subsequent repair. We identified Rec12-binding peaks across the Schizosaccharomyces pombe genome using chromatin immunoprecipitation after reversible formaldehyde cross-linking combined with whole-genome DNA microarrays. Strong Rec12 binding coincided with previously identified DSBs at the recombination hotspots ura4A, mbs1, and mbs2 and correlated with DSB formation at a new site. In addition, Rec12 binding corresponded to eight novel conversion hotspots and correlated with crossover density in segments of chromosome I. Notably, Rec12 binding inversely correlated with guanine-cytosine (GC) content, contrary to findings in Saccharomyces cerevisiae. Although both replication origins and Rec12-binding sites preferred AT-rich gene-free regions, they seemed to exclude each other. We also uncovered a connection between binding sites of Rec12 and meiotic cohesin Rec8. Rec12-binding peaks lay often within 2.5 kb of a Rec8-binding peak. Rec12 binding showed preference for large intergenic regions and was found to bind preferentially near to genes expressed strongly in meiosis. Surprisingly, Rec12 binding was also detected in centromeric core regions, which raises the intriguing possibility that Rec12 plays additional roles in meiotic chromosome dynamics.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Chromosoma</span>&nbsp;<span class="year">2008</span>;<span class="volume">117</span>;<span class="number">5</span>;<span class="pagerange">431-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18449558">18449558</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3671157">3671157</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00412-008-0159-3">10.1007/s00412-008-0159-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MYO5B mutations cause microvillus inclusion disease and disrupt epithelial cell polarity.</h4>
<p class="authors"><span class="author">M&uuml;ller T</span>, <span class="author">Hess MW</span>, <span class="author">Schiefermeier N</span>, <span class="author">Pfaller K</span>, <span class="author">Ebner HL</span>, <span class="author">Heinz-Erian P</span>, <span class="author">Ponstingl H</span>, <span class="author">Partsch J</span>, <span class="author">R&ouml;llinghoff B</span>, <span class="author">K&ouml;hler H</span>, <span class="author">Berger T</span>, <span class="author">Lenhartz H</span>, <span class="author">Schlenck B</span>, <span class="author">Houwen RJ</span>, <span class="author">Taylor CJ</span>, <span class="author">Zoller H</span>, <span class="author">Lechner S</span>, <span class="author">Goulet O</span>, <span class="author">Utermann G</span>, <span class="author">Ruemmele FM</span>, <span class="author">Huber LA</span> and <span class="author">Janecke AR</span></p>
<p class="affiliation">Department of Pediatrics II, Innsbruck Medical University, 6020 Innsbruck, Austria.</p><p class="abstract">Following homozygosity mapping in a single kindred, we identified nonsense and missense mutations in MYO5B, encoding type Vb myosin motor protein, in individuals with microvillus inclusion disease (MVID). MVID is characterized by lack of microvilli on the surface of enterocytes and occurrence of intracellular vacuolar structures containing microvilli. In addition, mislocalization of transferrin receptor in MVID enterocytes suggests that MYO5B deficiency causes defective trafficking of apical and basolateral proteins in MVID.</p>
<p class="grants">Funded by: Austrian Science Fund FWF: <span class="grant">P 19486-B12</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">10</span>;<span class="pagerange">1163-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18724368">18724368</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.225">10.1038/ng.225</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The neglected role of antibody in protection against bacteremia caused by nontyphoidal strains of Salmonella in African children.</h4>
<p class="authors"><span class="author">MacLennan CA</span>, <span class="author">Gondwe EN</span>, <span class="author">Msefula CL</span>, <span class="author">Kingsley RA</span>, <span class="author">Thomson NR</span>, <span class="author">White SA</span>, <span class="author">Goodall M</span>, <span class="author">Pickard DJ</span>, <span class="author">Graham SM</span>, <span class="author">Dougan G</span>, <span class="author">Hart CA</span>, <span class="author">Molyneux ME</span> and <span class="author">Drayson MT</span></p>
<p class="affiliation">Malawi-Liverpool-Wellcome Trust Clinical Research Programme, College of Medicine, University of Malawi, Blantyre, Malawi. <a href="mailto:%63.%6D%61%63%6C%65%6E%6E%61%6E%40%62%68%61%6D.%61%63.%75%6B">&#99;&#46;&#109;&#97;&#99;&#108;&#101;&#110;&#110;&#97;&#110;&#64;&#98;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Nontyphoidal strains of Salmonella (NTS) are a common cause of bacteremia among African children. Cell-mediated immune responses control intracellular infection, but they do not protect against extracellular growth of NTS in the blood. We investigated whether antibody protects against NTS bacteremia in Malawian children, because we found this condition mainly occurs before 2 years of age, with relative sparing of infants younger than 4 months old. Sera from all healthy Malawian children tested aged more than 16 months contained anti-Salmonella antibody and successfully killed NTS. Killing was mediated by complement membrane attack complex and not augmented in the presence of blood leukocytes. Sera from most healthy children less than 16 months old lacked NTS-specific antibody, and sera lacking antibody did not kill NTS despite normal complement function. Addition of Salmonella-specific antibody, but not mannose-binding lectin, enabled NTS killing. All NTS strains tested had long-chain lipopolysaccharide and the rck gene, features that resist direct complement-mediated killing. Disruption of lipopolysaccharide biosynthesis enabled killing of NTS by serum lacking Salmonella-specific antibody. We conclude that Salmonella-specific antibody that overcomes the complement resistance of NTS develops by 2 years of life in Malawian children. This finding and the age-incidence of NTS bacteremia suggest that antibody protects against NTS bacteremia and support the development of vaccines against NTS that induce protective antibody.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">067902</span></p>
<p class="publication"><span class="title">The Journal of clinical investigation</span>&nbsp;<span class="year">2008</span>;<span class="volume">118</span>;<span class="number">4</span>;<span class="pagerange">1553-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18357343">18357343</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2268878">2268878</a>; DOI: <a href="http://dx.doi.org/10.1172%2FJCI33998">10.1172/JCI33998</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Extensive chromosome rearrangements distinguish the karyotype of the hypovirulent species Candida dubliniensis from the virulent Candida albicans.</h4>
<p class="authors"><span class="author">Magee BB</span>, <span class="author">Sanchez MD</span>, <span class="author">Saunders D</span>, <span class="author">Harris D</span>, <span class="author">Berriman M</span> and <span class="author">Magee PT</span></p>
<p class="affiliation">Department of Genetics, Cell Biology, and Development, University of Minnesota, Minneapolis, MN, USA.</p><p class="abstract">Candida dubliniensis and Candida albicans, the most common human fungal pathogen, have most of the same genes and high sequence similarity, but C. dubliniensis is less virulent. C. albicans causes both mucosal and hematogenously disseminated disease, C. dubliniensis mostly mucosal infections. Pulse-field electrophoresis, genomic restriction enzyme digests, Southern blotting, and the emerging sequence from the Wellcome Trust Sanger Institute were used to determine the karyotype of C. dubliniensis type strain CD36. Three chromosomes have two intact homologues. A translocation in the rDNA repeat on chromosome R exchanges telomere-proximal regions of R and chromosome 5. Translocations involving the remaining chromosomes occur at the Major Repeat Sequence. CD36 lacks an MRS on chromosome R but has one on 3. Of six other C. dubliniensis strains, no two had the same electrophoretic karyotype. Despite extensive chromosome rearrangements, karyotypic differences between C. dubliniensis and C. albicans are unlikely to affect gene expression. Karyotypic instability may account for the diminished pathogenicity of C. dubliniensis.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI16567</span>, <span class="grant">R01 AI016567-21</span>, <span class="grant">R01 AI016567-22</span>, <span class="grant">R01 AI016567-23</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Fungal genetics and biology : FG &amp; B</span>&nbsp;<span class="year">2008</span>;<span class="volume">45</span>;<span class="number">3</span>;<span class="pagerange">338-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17719250" rel="external">17719250</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2277252" rel="external">2277252</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.fgb.2007.07.004" rel="external">10.1016/j.fgb.2007.07.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interferon regulatory factor-1 polymorphisms are associated with the control of Plasmodium falciparum infection.</h4>
<p class="authors"><span class="author">Mangano VD</span>, <span class="author">Luoni G</span>, <span class="author">Rockett KA</span>, <span class="author">Sirima BS</span>, <span class="author">Konat&eacute; A</span>, <span class="author">Forton J</span>, <span class="author">Clark TG</span>, <span class="author">Bancone G</span>, <span class="author">Sadighi Akha E</span>, <span class="author">Akha ES</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Modiano D</span></p>
<p class="affiliation">Dipartimento di Scienze di Sanit&agrave; Pubblica, Sezione di Parassitologia, Universit&agrave; di Roma &#39;La Sapienza&#39;, Rome, Italy. <a href="mailto:%64%61%76%69%64.%6D%6F%64%69%61%6E%6F%40%75%6E%69%72%6F%6D%61%31.%69%74">&#100;&#97;&#118;&#105;&#100;&#46;&#109;&#111;&#100;&#105;&#97;&#110;&#111;&#64;&#117;&#110;&#105;&#114;&#111;&#109;&#97;&#49;&#46;&#105;&#116;</a></p><p class="abstract">We describe the haplotypic structure of the interferon regulatory factor-1 (IRF-1) locus in two West African ethnic groups, Fulani and Mossi, that differ in their susceptibility and immune response to Plasmodium falciparum malaria. Both populations showed significant associations between IRF-1 polymorphisms and carriage of P. falciparum infection, with different patterns of association that may reflect their different haplotypic architecture. Genetic variation at this locus does not therefore account for the Fulani-specific resistance to malaria while it could contribute to parasite clearance&#39;s ability in populations living in endemic areas. We then conducted a case-control study of three haplotype-tagging single nucleotide polymorphisms (htSNPs) in 370 hospitalised malaria patients (160 severe and 210 uncomplicated) and 410 healthy population controls, all from the Mossi ethnic group. All three htSNPs showed correlation with blood infection levels in malaria patients, and the rs10065633 polymorphism was associated with severe disease (P=0.02). These findings provide the first evidence of the involvement in malaria susceptibility of a specific locus within the 5q31 region, previously shown to be linked with P. falciparum infection levels.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">077383</span></p>
<p class="publication"><span class="title">Genes and immunity</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">2</span>;<span class="pagerange">122-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18200030" rel="external">18200030</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2963926" rel="external">2963926</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.gene.6364456" rel="external">10.1038/sj.gene.6364456</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative cytogenetics of bats (Chiroptera): the prevalence of Robertsonian translocations limits the power of chromosomal characters in resolving interfamily phylogenetic relationships.</h4>
<p class="authors"><span class="author">Mao X</span>, <span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Su W</span>, <span class="author">Feng Q</span>, <span class="author">Wang Y</span>, <span class="author">Dobigny G</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Kunming Institute of Zoology, Kunming, Yunnan, People&#39;s Republic of China.</p><p class="abstract">Although the monophyly of Chiroptera is well supported by many independent studies, higher-level systematics, e.g. the monophyly of microbats, remains disputed by morphological and molecular studies. Chromosomal rearrangements, as one type of rare genomic changes, have become increasingly popular in phylogenetic studies as alternatives to molecular and other morphological characters. Here, the representatives of families Megadermatidae and Emballonuridae are studied by comparative chromosome painting for the first time. The results have been integrated into published comparative maps, providing an opportunity to assess genome-wide chromosomal homologies between the representatives of eight bat families. Our results further substantiate the wide occurrence of Robertsonian translocations in bats, with the possible involvement of whole-arm reciprocal translocations (WARTs). In order to search for valid cytogenetic signature(s) for each family and superfamily, evolutionary chromosomal rearrangements identified by chromosomal painting and/or banding comparison are subjected to two independent analyses: (1) a cladistic analysis using parsimony and (2) the mapping of these chromosomal changes onto the molecularly defined phylogenetic tree available from the literature. Both analyses clearly indicate the prevalence of homoplasic events that reduce the reliability of chromosomal characters for resolving interfamily relationships in bats.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">155-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18293110" rel="external">18293110</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1206-2" rel="external">10.1007/s10577-007-1206-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Next-generation sequencing: applications beyond genomes.</h4>
<p class="authors"><span class="author">Marguerat S</span>, <span class="author">Wilhelm BT</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK, Fission Yeast Functional Genomics Group, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK. <a href="mailto:%6A%75%72%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#117;&#114;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The development of DNA sequencing more than 30 years ago has profoundly impacted biological research. In the last couple of years, remarkable technological innovations have emerged that allow the direct and cost-effective sequencing of complex samples at unprecedented scale and speed. These next-generation technologies make it feasible to sequence not only static genomes, but also entire transcriptomes expressed under different conditions. These and other powerful applications of next-generation sequencing are rapidly revolutionizing the way genomic studies are carried out. Below, we provide a snapshot of these exciting new approaches to understanding the properties and functions of genomes. Given that sequencing-based assays may increasingly supersede microarray-based assays, we also compare and contrast data obtained from these distinct approaches.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Pt 5</span>;<span class="pagerange">1091-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18793195" rel="external">18793195</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2563889" rel="external">2563889</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0361091" rel="external">10.1042/BST0361091</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GeConT 2: gene context analysis for orthologous proteins, conserved domains and metabolic pathways.</h4>
<p class="authors"><span class="author">Martinez-Guerrero CE</span>, <span class="author">Ciria R</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">Moreno-Hagelsieb G</span> and <span class="author">Merino E</span></p>
<p class="affiliation">Departmento de Microbiolog&iacute;a Molecular, Instituto de Biotecnolog&iacute;a, Universidad Nacional Aut&oacute;noma de M&eacute;xico, Cuernavaca, Morelos 62210, M&eacute;xico.</p><p class="abstract">The Gene Context Tool (GeConT) allows users to visualize the genomic context of a gene or a group of genes and their orthologous relationships within fully sequenced bacterial genomes. The new version of the server incorporates information from the COG, Pfam and KEGG databases, allowing users to have an integrated graphical representation of the function of genes at multiple levels, their phylogenetic distribution and their genomic context. The sequence of any of the genes can be easily retrieved, as well as the 5&#39; or 3&#39; regulatory regions, greatly facilitating further types of analysis. GeConT 2 is available at: http://bioinfo.ibt.unam.mx/gecont.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Web Server issue</span>;<span class="pagerange">W176-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18511460" rel="external">18511460</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2447741" rel="external">2447741</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkn330" rel="external">10.1093/nar/gkn330</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gametogenesis in malaria parasites is mediated by the cGMP-dependent protein kinase.</h4>
<p class="authors"><span class="author">McRobert L</span>, <span class="author">Taylor CJ</span>, <span class="author">Deng W</span>, <span class="author">Fivelman QL</span>, <span class="author">Cummings RM</span>, <span class="author">Polley SD</span>, <span class="author">Billker O</span> and <span class="author">Baker DA</span></p>
<p class="affiliation">Department of Infectious and Tropical Diseases, London School of Hygiene &amp; Tropical Medicine, London, United Kingdom.</p><p class="abstract">Malaria parasite transmission requires differentiation of male and female gametocytes into gametes within a mosquito following a blood meal. A mosquito-derived molecule, xanthurenic acid (XA), can trigger gametogenesis, but the signalling events controlling this process in the human malaria parasite Plasmodium falciparum remain unknown. A role for cGMP was revealed by our observation that zaprinast (an inhibitor of phosphodiesterases that hydrolyse cGMP) stimulates gametogenesis in the absence of XA. Using cGMP-dependent protein kinase (PKG) inhibitors in conjunction with transgenic parasites expressing an inhibitor-insensitive mutant PKG enzyme, we demonstrate that PKG is essential for XA- and zaprinast-induced gametogenesis. Furthermore, we show that intracellular calcium (Ca2+) is required for differentiation and acts downstream of or in parallel with PKG activation. This work defines a key role for PKG in gametogenesis, elucidates the hierarchy of signalling events governing this process in P. falciparum, and demonstrates the feasibility of selective inhibition of a crucial regulator of the malaria parasite life cycle.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">012174</span>; Wellcome Trust: <span class="grant">066742</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">e139</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18532880" rel="external">18532880</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2408617" rel="external">2408617</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0060139" rel="external">10.1371/journal.pbio.0060139</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microbiology in the post-genomic era.</h4>
<p class="authors"><span class="author">Medini D</span>, <span class="author">Serruto D</span>, <span class="author">Parkhill J</span>, <span class="author">Relman DA</span>, <span class="author">Donati C</span>, <span class="author">Moxon R</span>, <span class="author">Falkow S</span> and <span class="author">Rappuoli R</span></p>
<p class="affiliation">Novartis Vaccines and Diagnostics, 53100 Siena, Italy.</p><p class="abstract">Genomics has revolutionized every aspect of microbiology. Now, 13 years after the first bacterial genome was sequenced, it is important to pause and consider what has changed in microbiology research as a consequence of genomics. In this article, we review the evolving field of bacterial typing and the genomic technologies that enable comparative analysis of multiple genomes and the metagenomes of complex microbial environments, and address the implications of the genomic era for the future of microbiology.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">419-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18475305">18475305</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1901">10.1038/nrmicro1901</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Recurrent rearrangements of chromosome 1q21.1 and variable pediatric phenotypes.</h4>
<p class="authors"><span class="author">Mefford HC</span>, <span class="author">Sharp AJ</span>, <span class="author">Baker C</span>, <span class="author">Itsara A</span>, <span class="author">Jiang Z</span>, <span class="author">Buysse K</span>, <span class="author">Huang S</span>, <span class="author">Maloney VK</span>, <span class="author">Crolla JA</span>, <span class="author">Baralle D</span>, <span class="author">Collins A</span>, <span class="author">Mercer C</span>, <span class="author">Norga K</span>, <span class="author">de Ravel T</span>, <span class="author">Devriendt K</span>, <span class="author">Bongers EM</span>, <span class="author">de Leeuw N</span>, <span class="author">Reardon W</span>, <span class="author">Gimelli S</span>, <span class="author">Bena F</span>, <span class="author">Hennekam RC</span>, <span class="author">Male A</span>, <span class="author">Gaunt L</span>, <span class="author">Clayton-Smith J</span>, <span class="author">Simonic I</span>, <span class="author">Park SM</span>, <span class="author">Mehta SG</span>, <span class="author">Nik-Zainal S</span>, <span class="author">Woods CG</span>, <span class="author">Firth HV</span>, <span class="author">Parkin G</span>, <span class="author">Fichera M</span>, <span class="author">Reitano S</span>, <span class="author">Lo Giudice M</span>, <span class="author">Li KE</span>, <span class="author">Casuga I</span>, <span class="author">Broomer A</span>, <span class="author">Conrad B</span>, <span class="author">Schwerzmann M</span>, <span class="author">R&auml;ber L</span>, <span class="author">Gallati S</span>, <span class="author">Striano P</span>, <span class="author">Coppola A</span>, <span class="author">Tolmie JL</span>, <span class="author">Tobias ES</span>, <span class="author">Lilley C</span>, <span class="author">Armengol L</span>, <span class="author">Spysschaert Y</span>, <span class="author">Verloo P</span>, <span class="author">De Coene A</span>, <span class="author">Goossens L</span>, <span class="author">Mortier G</span>, <span class="author">Speleman F</span>, <span class="author">van Binsbergen E</span>, <span class="author">Nelen MR</span>, <span class="author">Hochstenbach R</span>, <span class="author">Poot M</span>, <span class="author">Gallagher L</span>, <span class="author">Gill M</span>, <span class="author">McClellan J</span>, <span class="author">King MC</span>, <span class="author">Regan R</span>, <span class="author">Skinner C</span>, <span class="author">Stevenson RE</span>, <span class="author">Antonarakis SE</span>, <span class="author">Chen C</span>, <span class="author">Estivill X</span>, <span class="author">Menten B</span>, <span class="author">Gimelli G</span>, <span class="author">Gribble S</span>, <span class="author">Schwartz S</span>, <span class="author">Sutcliffe JS</span>, <span class="author">Walsh T</span>, <span class="author">Knight SJ</span>, <span class="author">Sebat J</span>, <span class="author">Romano C</span>, <span class="author">Schwartz CE</span>, <span class="author">Veltman JA</span>, <span class="author">de Vries BB</span>, <span class="author">Vermeesch JR</span>, <span class="author">Barber JC</span>, <span class="author">Willatt L</span>, <span class="author">Tassabehji M</span> and <span class="author">Eichler EE</span></p>
<p class="affiliation">University of Washington School of Medicine, Seattle 98195, USA.</p><p class="abstract"><strong>Background</strong>: Duplications and deletions in the human genome can cause disease or predispose persons to disease. Advances in technologies to detect these changes allow for the routine identification of submicroscopic imbalances in large numbers of patients.</p><p class="abstract"><strong>Methods</strong>: We tested for the presence of microdeletions and microduplications at a specific region of chromosome 1q21.1 in two groups of patients with unexplained mental retardation, autism, or congenital anomalies and in unaffected persons.</p><p class="abstract"><strong>Results</strong>: We identified 25 persons with a recurrent 1.35-Mb deletion within 1q21.1 from screening 5218 patients. The microdeletions had arisen de novo in eight patients, were inherited from a mildly affected parent in three patients, were inherited from an apparently unaffected parent in six patients, and were of unknown inheritance in eight patients. The deletion was absent in a series of 4737 control persons (P=1.1x10(-7)). We found considerable variability in the level of phenotypic expression of the microdeletion; phenotypes included mild-to-moderate mental retardation, microcephaly, cardiac abnormalities, and cataracts. The reciprocal duplication was enriched in nine children with mental retardation or autism spectrum disorder and other variable features (P=0.02). We identified three deletions and three duplications of the 1q21.1 region in an independent sample of 788 patients with mental retardation and congenital anomalies.</p><p class="abstract"><strong>Conclusions</strong>: We have identified recurrent molecular lesions that elude syndromic classification and whose disease manifestations must be considered in a broader context of development as opposed to being assigned to a specific disease. Clinical diagnosis in patients with these lesions may be most readily achieved on the basis of genotype rather than phenotype.</p>
<p class="grants">Funded by: Department of Health; Howard Hughes Medical Institute; NICHD NIH HHS: <span class="grant">HD043569</span>, <span class="grant">R01 HD043569-06</span>; Wellcome Trust: <span class="grant">061183</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2008</span>;<span class="volume">359</span>;<span class="number">16</span>;<span class="pagerange">1685-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18784092" rel="external">18784092</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2703742" rel="external">2703742</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa0805384" rel="external">10.1056/NEJMoa0805384</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Key function for the CCAAT-binding factor Php4 to regulate gene expression in response to iron deficiency in fission yeast.</h4>
<p class="authors"><span class="author">Mercier A</span>, <span class="author">Watt S</span>, <span class="author">B&auml;hler J</span> and <span class="author">Labb&eacute; S</span></p>
<p class="affiliation">D&eacute;partement de Biochimie, Facult&eacute; de M&eacute;decine, Universit&eacute; de Sherbrooke, 3001, 12e Ave. Nord, Sherbrooke, Quebec J1H 5N4, Canada.</p><p class="abstract">The fission yeast Schizosaccharomyces pombe responds to the deprivation of iron by inducing the expression of the php4+ gene, which encodes a negative regulatory subunit of the heteromeric CCAAT-binding factor. Once formed, the Php2/3/4/5 transcription complex is required to inactivate a subset of genes encoding iron-using proteins. Here, we used a pan-S. pombe microarray to study the transcriptional response to iron starvation and identified 86 genes that exhibit php4+-dependent changes on a genome-wide scale. One of these genes encodes the iron-responsive transcriptional repressor Fep1, whose mRNA levels were decreased after treatment with the permeant iron chelator 2,2&#39;-dipyridyl. In addition, several genes encoding the components of iron-dependent biochemical pathways, including the tricarboxylic acid cycle, mitochondrial respiration, amino acid biosynthesis, and oxidative stress defense, were downregulated in response to iron deficiency. Furthermore, Php4 repressed transcription when brought to a promoter using a yeast DNA-binding domain, and iron deprivation was required for this repression. On the other hand, Php4 was constitutively active when glutathione levels were depleted within the cell. Based on these and previous results, we propose that iron-dependent inactivation of Php4 is regulated at two distinct levels: first, at the transcriptional level by the iron-responsive GATA factor Fep1 and second, at the posttranscriptional level by a mechanism yet to be identified, which inhibits Php4-mediated repressive function when iron is abundant.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2008</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">493-508</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18223116" rel="external">18223116</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2268518" rel="external">2268518</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.00446-07" rel="external">10.1128/EC.00446-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of shotgun proteomics for the identification, confirmation, and correction of C. elegans gene annotations.</h4>
<p class="authors"><span class="author">Merrihew GE</span>, <span class="author">Davis C</span>, <span class="author">Ewing B</span>, <span class="author">Williams G</span>, <span class="author">K&auml;ll L</span>, <span class="author">Frewen BE</span>, <span class="author">Noble WS</span>, <span class="author">Green P</span>, <span class="author">Thomas JH</span> and <span class="author">MacCoss MJ</span></p>
<p class="affiliation">University of Washington, Department of Genome Sciences, Seattle, Washington 98195, USA.</p><p class="abstract">We describe a general mass spectrometry-based approach for gene annotation of any organism and demonstrate its effectiveness using the nematode Caenorhabditis elegans. We detected 6779 C. elegans proteins (67,047 peptides), including 384 that, although annotated in WormBase WS150, lacked cDNA or other prior experimental support. We also identified 429 new coding sequences that were unannotated in WS150. Nearly half (192/429) of the new coding sequences were confirmed with RT-PCR data. Thirty-three (approximately 8%) of the new coding sequences had been predicted to be pseudogenes, 151 (approximately 35%) reveal apparent errors in gene models, and 245 (57%) appear to be novel genes. In addition, we verified 6010 exon-exon splice junctions within existing WormBase gene models. Our work confirms that mass spectrometry is a powerful experimental tool for annotating sequenced genomes. In addition, the collection of identified peptides should facilitate future proteomics experiments targeted at specific proteins of interest.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCRR NIH HHS: <span class="grant">P41-RR011823</span>; NIDDK NIH HHS: <span class="grant">R01-DK069386</span>; NIGMS NIH HHS: <span class="grant">R21-GM074787</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">10</span>;<span class="pagerange">1660-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18653799" rel="external">18653799</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2556273" rel="external">2556273</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.077644.108" rel="external">10.1101/gr.077644.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Williams-Beuren syndrome TRIM50 encodes an E3 ubiquitin ligase.</h4>
<p class="authors"><span class="author">Micale L</span>, <span class="author">Fusco C</span>, <span class="author">Augello B</span>, <span class="author">Napolitano LM</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Meroni G</span>, <span class="author">Merla G</span> and <span class="author">Reymond A</span></p>
<p class="affiliation">Medical Genetics Unit, IRCCS Casa Sollievo della Sofferenza, San Giovanni Rotondo, Italy.</p><p class="abstract">Williams-Beuren syndrome (WBS) is a neurodevelopmental and multisystemic disease that results from hemizygosity of approximately 25 genes mapping to chromosomal region 7q11.23. We report here the preliminary description of eight novel genes mapping within the WBS critical region and/or its syntenic mouse region. Three of these genes, TRIM50, TRIM73 and TRIM74, belong to the TRIpartite motif gene family, members of which were shown to be associated to several human genetic diseases. We describe the preliminary functional characterization of these genes and show that Trim50 encodes an E3 ubiquitin ligase, opening the interesting hypothesis that the ubiquitin-mediated proteasome pathway might be involved in the WBS phenotype.</p>
<p class="grants">Funded by: Telethon: <span class="grant">GGP06122</span>; Wellcome Trust: <span class="grant">077046</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">9</span>;<span class="pagerange">1038-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18398435" rel="external">18398435</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680067" rel="external">2680067</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fejhg.2008.68" rel="external">10.1038/ejhg.2008.68</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic expression patterns in cell separation mutants of Schizosaccharomyces pombe defective in the genes sep10 ( + ) and sep15 ( + ) coding for the Mediator subunits Med31 and Med8.</h4>
<p class="authors"><span class="author">Miklos I</span>, <span class="author">Szilagyi Z</span>, <span class="author">Watt S</span>, <span class="author">Zilahi E</span>, <span class="author">Batta G</span>, <span class="author">Antunovics Z</span>, <span class="author">Enczi K</span>, <span class="author">B&auml;hler J</span> and <span class="author">Sipiczki M</span></p>
<p class="affiliation">Department of Genetics and Applied Microbiology, University of Debrecen, Debrecen, Hungary.</p><p class="abstract">Cell division is controlled by a complex network involving regulated transcription of genes and postranslational modification of proteins. The aim of this study is to demonstrate that the Mediator complex, a general regulator of transcription, is involved in the regulation of the second phase (cell separation) of cell division of the fission yeast Schizosaccharomyces pombe. In previous studies we have found that the fission yeast cell separation genes sep10 ( + ) and sep15 ( + ) code for proteins (Med31 and Med8) associated with the Mediator complex. Here, we show by genome-wide gene expression profiling of mutants defective in these genes that both Med8 and Med31 control large, partially overlapping sets of genes scattered over the entire genome and involved in diverse biological functions. Six cell separation genes controlled by the transcription factors Sep1 and Ace2 are among the target genes. Since neither sep1 ( + ) nor ace2 ( + ) is affected in the mutant cells, we propose that the Med8 and Med31 proteins act as coactivators of the Sep1-Ace2-dependent cell separation genes. The results also indicate that the subunits of Mediator may contribute to the coordination of cellular processes by fine-tuning of the expression of larger sets of genes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular genetics and genomics : MGG</span>&nbsp;<span class="year">2008</span>;<span class="volume">279</span>;<span class="number">3</span>;<span class="pagerange">225-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17922236">17922236</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00438-007-0296-z">10.1007/s00438-007-0296-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Oligosaccharyltransferase-subunit mutations in nonsyndromic mental retardation.</h4>
<p class="authors"><span class="author">Molinari F</span>, <span class="author">Foulquier F</span>, <span class="author">Tarpey PS</span>, <span class="author">Morelle W</span>, <span class="author">Boissel S</span>, <span class="author">Teague J</span>, <span class="author">Edkins S</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Turner G</span>, <span class="author">Matthijs G</span>, <span class="author">Gecz J</span>, <span class="author">Munnich A</span> and <span class="author">Colleaux L</span></p>
<p class="affiliation">Laboratoire de G&eacute;n&eacute;tique et Epig&eacute;n&eacute;tique des Maladies M&eacute;taboliques, Neurosensorielles et du D&eacute;veloppement (INSERM U781), Universit&eacute; Paris Descartes, H&ocirc;pital Necker-Enfants Malades, F-75015 Paris, France.</p><p class="abstract">Mental retardation (MR) is the most frequent handicap among children and young adults. Although a large proportion of X-linked MR genes have been identified, only four genes responsible for autosomal-recessive nonsyndromic MR (AR-NSMR) have been described so far. Here, we report on two genes involved in autosomal-recessive and X-linked NSMR. First, autozygosity mapping in two sibs born to first-cousin French parents led to the identification of a region on 8p22-p23.1. This interval encompasses the gene N33/TUSC3 encoding one subunit of the oligosaccharyltransferase (OTase) complex, which catalyzes the transfer of an oligosaccharide chain on nascent proteins, the key step of N-glycosylation. Sequencing N33/TUSC3 identified a 1 bp insertion, c.787_788insC, resulting in a premature stop codon, p.N263fsX300, and leading to mRNA decay. Surprisingly, glycosylation analyses of patient fibroblasts showed normal N-glycan synthesis and transfer, suggesting that normal N-glycosylation observed in patient fibroblasts may be due to functional compensation. Subsequently, screening of the X-linked N33/TUSC3 paralog, the IAP gene, identified a missense mutation (c.932T--&gt;G, p.V311G) in a family with X-linked NSMR. Recent studies of fucosylation and polysialic-acid modification of neuronal cell-adhesion glycoproteins have shown the critical role of glycosylation in synaptic plasticity. However, our data provide the first demonstration that a defect in N-glycosylation can result in NSMR. Together, our results demonstrate that fine regulation of OTase activity is essential for normal cognitive-function development, providing therefore further insights to understand the pathophysiological bases of MR.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">5</span>;<span class="pagerange">1150-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18455129" rel="external">18455129</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427205" rel="external">2427205</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.03.021" rel="external">10.1016/j.ajhg.2008.03.021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fission yeast SWI/SNF and RSC complexes show compositional and functional differences from budding yeast.</h4>
<p class="authors"><span class="author">Monahan BJ</span>, <span class="author">Vill&eacute;n J</span>, <span class="author">Marguerat S</span>, <span class="author">B&auml;hler J</span>, <span class="author">Gygi SP</span> and <span class="author">Winston F</span></p>
<p class="affiliation">Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA.</p><p class="abstract">SWI/SNF chromatin-remodeling complexes have crucial roles in transcription and other chromatin-related processes. The analysis of the two members of this class in Saccharomyces cerevisiae, SWI/SNF and RSC, has heavily contributed to our understanding of these complexes. To understand the in vivo functions of SWI/SNF and RSC in an evolutionarily distant organism, we have characterized these complexes in Schizosaccharomyces pombe. Although core components are conserved between the two yeasts, the compositions of S. pombe SWI/SNF and RSC differ from their S. cerevisiae counterparts and in some ways are more similar to metazoan complexes. Furthermore, several of the conserved proteins, including actin-like proteins, are markedly different between the two yeasts with respect to their requirement for viability. Finally, phenotypic and microarray analyses identified widespread requirements for SWI/SNF and RSC on transcription including strong evidence that SWI/SNF directly represses iron-transport genes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; NHGRI NIH HHS: <span class="grant">HG3456</span>, <span class="grant">R01 HG003456</span>, <span class="grant">R01 HG003456-03</span>; NIGMS NIH HHS: <span class="grant">GM32967</span>, <span class="grant">R37 GM032967</span>, <span class="grant">R37 GM032967-25</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Nature structural &amp; molecular biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">15</span>;<span class="number">8</span>;<span class="pagerange">873-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18622392">18622392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2559950">2559950</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnsmb.1452">10.1038/nsmb.1452</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide discovery and verification of novel structured RNAs in Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Mourier T</span>, <span class="author">Carret C</span>, <span class="author">Kyes S</span>, <span class="author">Christodoulou Z</span>, <span class="author">Gardner PP</span>, <span class="author">Jeffares DC</span>, <span class="author">Pinches R</span>, <span class="author">Barrell B</span>, <span class="author">Berriman M</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Ivens A</span>, <span class="author">Newbold C</span> and <span class="author">Pain A</span></p>
<p class="affiliation">Ancient DNA and Evolution Group, Department of Biology, University of Copenhagen, Copenhagen DK-2100, Denmark.</p><p class="abstract">We undertook a genome-wide search for novel noncoding RNAs (ncRNA) in the malaria parasite Plasmodium falciparum. We used the RNAz program to predict structures in the noncoding regions of the P. falciparum 3D7 genome that were conserved with at least one of seven other Plasmodium spp. genome sequences. By using Northern blot analysis for 76 high-scoring predictions and microarray analysis for the majority of candidates, we have verified the expression of 33 novel ncRNA transcripts including four members of a ncRNA family in the asexual blood stage. These transcripts represent novel structured ncRNAs in P. falciparum and are not represented in any RNA databases. We provide supporting evidence for purifying selection acting on the experimentally verified ncRNAs by comparing the nucleotide substitutions in the predicted ncRNA candidate structures in P. falciparum with the closely related chimp malaria parasite P. reichenowi. The high confirmation rate within a single parasite life cycle stage suggests that many more of the predictions may be expressed in other stages of the organism&#39;s life cycle.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">2</span>;<span class="pagerange">281-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18096748">18096748</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2203626">2203626</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.6836108">10.1101/gr.6836108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamic instability of the major urinary protein gene family revealed by genomic and phenotypic comparisons between C57 and 129 strain mice.</h4>
<p class="authors"><span class="author">Mudge JM</span>, <span class="author">Armstrong SD</span>, <span class="author">McLaren K</span>, <span class="author">Beynon RJ</span>, <span class="author">Hurst JL</span>, <span class="author">Nicholson C</span>, <span class="author">Robertson DH</span>, <span class="author">Wilming LG</span> and <span class="author">Harrow JL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB101SA, UK. <a href="mailto:%6A%6D%31%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#109;&#49;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The major urinary proteins (MUPs) of Mus musculus domesticus are deposited in urine in large quantities, where they bind and release pheromones and also provide an individual &#39;recognition signal&#39; via their phenotypic polymorphism. Whilst important information about MUP functionality has been gained in recent years, the gene cluster is poorly studied in terms of structure, genic polymorphism and evolution.</p><p class="abstract"><strong>Results</strong>: We combine targeted sequencing, manual genome annotation and phylogenetic analysis to compare the Mup clusters of C57BL/6J and 129 strains of mice. We describe organizational heterogeneity within both clusters: a central array of cassettes containing Mup genes highly similar at the protein level, flanked by regions containing Mup genes displaying significantly elevated divergence. Observed genomic rearrangements in all regions have likely been mediated by endogenous retroviral elements. Mup loci with coding sequences that differ between the strains are identified--including a gene/pseudogene pair--suggesting that these inbred lineages exhibit variation that exists in wild populations. We have characterized the distinct MUP profiles in the urine of both strains by mass spectrometry. The total MUP phenotype data is reconciled with our genomic sequence data, matching all proteins identified in urine to annotated genes.</p><p class="abstract"><strong>Conclusion</strong>: Our observations indicate that the MUP phenotypic polymorphism observed in wild populations results from a combination of Mup gene turnover coupled with currently unidentified mechanisms regulating gene expression patterns. We propose that the structural heterogeneity described within the cluster reflects functional divergence within the Mup gene family.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">R91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18507838">18507838</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2441477">2441477</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-5-r91">10.1186/gb-2008-9-5-r91</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative gene mapping in cattle, Indian muntjac, and Chinese muntjac by fluorescence in situ hybridization.</h4>
<p class="authors"><span class="author">Murmann AE</span>, <span class="author">Mincheva A</span>, <span class="author">Scheuermann MO</span>, <span class="author">Gautier M</span>, <span class="author">Yang F</span>, <span class="author">Buitkamp J</span>, <span class="author">Strissel PL</span>, <span class="author">Strick R</span>, <span class="author">Rowley JD</span> and <span class="author">Lichter P</span></p>
<p class="affiliation">Department of Medicine, Section Hematology/Oncology, University of Chicago, Chicago, IL 60637, USA. <a href="mailto:%61%6D%75%72%6D%61%6E%6E%40%75%63%68%69%63%61%67%6F.%65%64%75">&#97;&#109;&#117;&#114;&#109;&#97;&#110;&#110;&#64;&#117;&#99;&#104;&#105;&#99;&#97;&#103;&#111;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The Indian muntjac (Muntiacus muntjak vaginalis) has a karyotype of 2n = 6 in the female and 2n = 7 in the male. The karyotypic evolution of Indian muntjac via extensive tandem fusions and several centric fusions are well documented by molecular cytogenetic studies mainly utilizing chromosome paints. To achieve higher resolution mapping, a set of 42 different genomic clones coding for 37 genes and the nucleolar organizer region were used to examine homologies between the cattle (2n = 60), human (2n = 46), Indian muntjac (2n = 6/7) and Chinese muntjac (2n = 46) karyotypes. These genomic clones were mapped by fluorescence in situ hybridization (FISH). Localization of genes on all three pairs of M. m. vaginalis chromosomes and on the acrocentric chromosomes of M. reevesi allowed not only the analysis of the evolution of syntenic regions within the muntjac genus but also allowed a broader comparison of synteny with more distantly related species, such as cattle and human, to shed more light onto the evolving genome organization.</p>
<p class="publication"><span class="title">Genetica</span>&nbsp;<span class="year">2008</span>;<span class="volume">134</span>;<span class="number">3</span>;<span class="pagerange">345-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18283540" rel="external">18283540</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10709-008-9242-1" rel="external">10.1007/s10709-008-9242-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Effectiveness of continuous glucose monitoring in pregnant women with diabetes: randomised clinical trial.</h4>
<p class="authors"><span class="author">Murphy HR</span>, <span class="author">Rayman G</span>, <span class="author">Lewis K</span>, <span class="author">Kelly S</span>, <span class="author">Johal B</span>, <span class="author">Duffield K</span>, <span class="author">Fowler D</span>, <span class="author">Campbell PJ</span> and <span class="author">Temple RC</span></p>
<p class="affiliation">Department of Diabetes and Endocrinology, Ipswich Hospital NHS Trust, Ipswich IP4 5PD. <a href="mailto:%48%65%6C%65%6E.%4D%75%72%70%68%79%40%69%70%73%77%69%63%68%68%6F%73%70%69%74%61%6C.%6E%68%73.%75%6B">&#72;&#101;&#108;&#101;&#110;&#46;&#77;&#117;&#114;&#112;&#104;&#121;&#64;&#105;&#112;&#115;&#119;&#105;&#99;&#104;&#104;&#111;&#115;&#112;&#105;&#116;&#97;&#108;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Objective</strong>: To evaluate the effectiveness of continuous glucose monitoring during pregnancy on maternal glycaemic control, infant birth weight, and risk of macrosomia in women with type 1 and type 2 diabetes.</p><p class="abstract"><strong>Design</strong>: Prospective, open label randomised controlled trial.</p><p class="abstract"><strong>Setting</strong>: Two secondary care multidisciplinary obstetric clinics for diabetes in the United Kingdom.</p><p class="abstract"><strong>Participants</strong>: 71 women with type 1 diabetes (n=46) or type 2 diabetes (n=25) allocated to antenatal care plus continuous glucose monitoring (n=38) or to standard antenatal care (n=33).</p><p class="abstract"><strong>Intervention</strong>: Continuous glucose monitoring was used as an educational tool to inform shared decision making and future therapeutic changes at intervals of 4-6 weeks during pregnancy. All other aspects of antenatal care were equal between the groups.</p><p class="abstract">The primary outcome was maternal glycaemic control during the second and third trimesters from measurements of HbA(1c) levels every four weeks. Secondary outcomes were birth weight and risk of macrosomia using birthweight standard deviation scores and customised birthweight centiles. Statistical analyses were done on an intention to treat basis.</p><p class="abstract"><strong>Results</strong>: Women randomised to continuous glucose monitoring had lower mean HbA(1c) levels from 32 to 36 weeks&#39; gestation compared with women randomised to standard antenatal care: 5.8% (SD 0.6) v 6.4% (SD 0.7). Compared with infants of mothers in the control arm those of mothers in the intervention arm had decreased mean birthweight standard deviation scores (0.9 v 1.6; effect size 0.7 SD, 95% confidence interval 0.0 to 1.3), decreased median customised birthweight centiles (69% v 93%), and a reduced risk of macrosomia (odds ratio 0.36, 95% confidence interval 0.13 to 0.98).</p><p class="abstract"><strong>Conclusion</strong>: Continuous glucose monitoring during pregnancy is associated with improved glycaemic control in the third trimester, lower birth weight, and reduced risk of macrosomia.</p><p class="abstract">Current Controlled Trials ISRCTN84461581.</p>
<p class="publication"><span class="title">BMJ (Clinical research ed.)</span>&nbsp;<span class="year">2008</span>;<span class="volume">337</span>;<span class="pagerange">a1680</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18818254" rel="external">18818254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2563261" rel="external">2563261</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromhome: a rich internet application for accessing comparative chromosome homology maps.</h4>
<p class="authors"><span class="author">Nagarajan S</span>, <span class="author">Rens W</span>, <span class="author">Stalker J</span>, <span class="author">Cox T</span> and <span class="author">Ferguson-Smith MA</span></p>
<p class="affiliation">Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Madingley Road Cambridge CB3 0ES, UK. <a href="mailto:%73%6E%33%32%34%40%63%61%6D.%61%63.%75%6B">&#115;&#110;&#51;&#50;&#52;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Comparative genomics has become a significant research area in recent years, following the availability of a number of sequenced genomes. The comparison of genomes is of great importance in the analysis of functionally important genome regions. It can also be used to understand the phylogenetic relationships of species and the mechanisms leading to rearrangement of karyotypes during evolution. Many species have been studied at the cytogenetic level by cross species chromosome painting. With the large amount of such information, it has become vital to computerize the data and make them accessible worldwide. Chromhome http://www.chromhome.org is a comprehensive web application that is designed to provide cytogenetic comparisons among species and to fulfil this need.</p><p class="abstract"><strong>Results</strong>: The Chromhome application architecture is multi-tiered with an interactive client layer, business logic and database layers. Enterprise java platform with open source framework OpenLaszlo is used to implement the Rich Internet Chromhome Application. Cross species comparative mapping raw data are collected and the processed information is stored into MySQL Chromhome database. Chromhome Release 1.0 contains 109 homology maps from 51 species. The data cover species from 14 orders and 30 families. The homology map displays all the chromosomes of the compared species as one image, making comparisons among species easier. Inferred data also provides maps of homologous regions that could serve as a guideline for researchers involved in phylogenetic or evolution based studies.</p><p class="abstract"><strong>Conclusion</strong>: Chromhome provides a useful resource for comparative genomics, holding graphical homology maps of a wide range of species. It brings together cytogenetic data of many genomes under one roof. Inferred painting can often determine the chromosomal homologous regions between two species, if each has been compared with a common third species. Inferred painting greatly reduces the need to map entire genomes and helps focus only on relevant regions of the chromosomes of the species under study. Future releases of Chromhome will accommodate more species and their respective gene and BAC maps, in addition to chromosome painting data. Chromhome application provides a single-page interface (SPI) with desktop style layout, delivering a better and richer user experience.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">168</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18366796" rel="external">18366796</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2323974" rel="external">2323974</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-9-168" rel="external">10.1186/1471-2105-9-168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Susceptibility to sequelae of human ocular chlamydial infection associated with allelic variation in IL10 cis-regulation.</h4>
<p class="authors"><span class="author">Natividad A</span>, <span class="author">Holland MJ</span>, <span class="author">Rockett KA</span>, <span class="author">Forton J</span>, <span class="author">Faal N</span>, <span class="author">Joof HM</span>, <span class="author">Mabey DC</span>, <span class="author">Bailey RL</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Infectious and Tropical Diseases Department, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK. <a href="mailto:%61%6E%67%65%6C%73.%6E%61%74%69%76%69%64%61%64-%73%61%6E%63%68%6F%40%6C%73%68%74%6D.%61%63.%75%6B">&#97;&#110;&#103;&#101;&#108;&#115;&#46;&#110;&#97;&#116;&#105;&#118;&#105;&#100;&#97;&#100;&#45;&#115;&#97;&#110;&#99;&#104;&#111;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Trachoma, an infectious disease of the conjunctiva caused by Chlamydia trachomatis, causes scarring and blindness in some infected individuals but not others. In an African community where trachoma is endemic, we have previously identified an IL10 haplotype that is associated with increased risk of scarring complications. Here we examine the hypothesis that the risk haplotype (H-RISK) affects levels of IL10 expression in the conjunctiva during active trachoma infection. To overcome potential genetic and environmental confounders we used the method of allele-specific quantification, which involved identifying subjects in the community who had active trachoma and were also heterozygous for the H-RISK. We find that there is allelic variation in cis-regulation of IL10 in the conjunctiva during active trachoma, with the H-RISK generating relatively more IL10 transcripts than other haplotypes in this population (average difference in IL10 allelic transcripts in the conjunctiva of heterozygous individuals infected with C. trachomatis of 23% (95% confidence interval: 14-32%, P &lt; 0.0001). These findings provide a plausible functional explanation for the observed genetic association, and support the hypothesis that an excessive IL10 response to C. trachomatis infection is a risk factor for scarring and blindness.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">2</span>;<span class="pagerange">323-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17947295" rel="external">17947295</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddm310" rel="external">10.1093/hmg/ddm310</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of 32 genome-wide linkage studies of schizophrenia</h4>
<p class="authors"><span class="author">Ng</span>, <span class="author">M. Y. M.</span>, <span class="author">Levinson</span>, <span class="author">D. F</span>, <span class="author">Faraone</span>, <span class="author">S. V</span>, <span class="author">Suarez</span>, <span class="author">B. K. et al</span></p>
<p class="publication"><span class="title">Molecular Psychiatry</span> <span class="year">2008</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using gene expression to investigate the genetic basis of complex disorders.</h4>
<p class="authors"><span class="author">Nica AC</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1HH, UK.</p><p class="abstract">The identification of complex disease susceptibility loci through genome-wide association studies (GWAS) has recently become possible and is now a method of choice for investigating the genetic basis of complex traits. The number of results from such studies is constantly increasing but the challenge lying forward is to identify the biological context in which these statistically significant candidate variants act. Regulatory variation plays an important role in shaping phenotypic differences among individuals and thus is very likely to also influence disease susceptibility. As such, integrating gene expression data and other disease relevant intermediate phenotypes with GWAS results could potentially help prioritize fine-mapping efforts and provide a shortcut to disease biology. Combining these different levels of information in a meaningful way is however not trivial. In the present review, we outline the several approaches that have been explored so far in this sense and their achievements. We also discuss the limitations of the methods and how upcoming technological developments could help circumvent these limitations. Overall, such efforts will be very helpful in understanding initially regulatory effects on disease and disease etiology in general.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077046</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">R2</span>;<span class="pagerange">R129-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18852201" rel="external">18852201</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2570059" rel="external">2570059</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddn285" rel="external">10.1093/hmg/ddn285</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Flying lemurs--the &#39;flying tree shrews&#39;? Molecular cytogenetic evidence for a Scandentia-Dermoptera sister clade.</h4>
<p class="authors"><span class="author">Nie W</span>, <span class="author">Fu B</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Wang J</span>, <span class="author">Su W</span>, <span class="author">Tanomtong A</span>, <span class="author">Volobouev V</span>, <span class="author">Ferguson-Smith MA</span> and <span class="author">Yang F</span></p>
<p class="affiliation">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming,Yunnan 650223, People&#39;s Republic of China. <a href="mailto:%77%68%6E%69%65%40%6D%61%69%6C.%6B%69%7A.%61%63.%63%6E">&#119;&#104;&#110;&#105;&#101;&#64;&#109;&#97;&#105;&#108;&#46;&#107;&#105;&#122;&#46;&#97;&#99;&#46;&#99;&#110;</a></p><p class="abstract"><strong>Background</strong>: Flying lemurs or Colugos (order Dermoptera) represent an ancient mammalian lineage that contains only two extant species. Although molecular evidence strongly supports that the orders Dermoptera, Scandentia, Lagomorpha, Rodentia and Primates form a superordinal clade called Supraprimates (or Euarchontoglires), the phylogenetic placement of Dermoptera within Supraprimates remains ambiguous.</p><p class="abstract"><strong>Results</strong>: To search for cytogenetic signatures that could help to clarify the evolutionary affinities within this superordinal group, we have established a genome-wide comparative map between human and the Malayan flying lemur (Galeopterus variegatus) by reciprocal chromosome painting using both human and G. variegatus chromosome-specific probes. The 22 human autosomal paints and the X chromosome paint defined 44 homologous segments in the G. variegatus genome. A putative inversion on GVA 11 was revealed by the hybridization patterns of human chromosome probes 16 and 19. Fifteen associations of human chromosome segments (HSA) were detected in the G. variegatus genome: HSA1/3, 1/10, 2/21, 3/21, 4/8, 4/18, 7/15, 7/16, 7/19, 10/16, 12/22 (twice), 14/15, 16/19 (twice). Reverse painting of G. variegatus chromosome-specific paints onto human chromosomes confirmed the above results, and defined the origin of the homologous human chromosomal segments in these associations. In total, G. variegatus paints revealed 49 homologous chromosomal segments in the HSA genome.</p><p class="abstract"><strong>Conclusion</strong>: Comparative analysis of our map with published maps from representative species of other placental orders, including Scandentia, Primates, Lagomorpha and Rodentia, suggests a signature rearrangement (HSA2q/21 association) that links Scandentia and Dermoptera to one sister clade. Our results thus provide new evidence for the hypothesis that Scandentia and Dermoptera have a closer phylogenetic relationship to each other than either of them has to Primates.</p>
<p class="publication"><span class="title">BMC biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="pagerange">18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18452598" rel="external">18452598</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2386441" rel="external">2386441</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1741-7007-6-18" rel="external">10.1186/1741-7007-6-18</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular pathology of aneurysms.</h4>
<p class="authors"><span class="author">Niemel&auml; M</span>, <span class="author">Fr&ouml;sen J</span>, <span class="author">Hernesniemi J</span>, <span class="author">Dashti R</span> and <span class="author">Palotie A</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089062</span></p>
<p class="publication"><span class="title">Surgical neurology</span>&nbsp;<span class="year">2008</span>;<span class="volume">70</span>;<span class="number">1</span>;<span class="pagerange">36-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18440615" rel="external">18440615</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2719287" rel="external">2719287</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.surneu.2008.02.017" rel="external">10.1016/j.surneu.2008.02.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Organ-specific requirements for Hdac1 in liver and pancreas formation.</h4>
<p class="authors"><span class="author">No&euml;l ES</span>, <span class="author">Casal-Sueiro A</span>, <span class="author">Busch-Nentwich E</span>, <span class="author">Verkade H</span>, <span class="author">Dong PD</span>, <span class="author">Stemple DL</span> and <span class="author">Ober EA</span></p>
<p class="affiliation">National Institute for Medical Research, Division of Developmental Biology, The Ridgeway, Mill Hill, London, UK.</p><p class="abstract">Liver, pancreas and lung originate from the presumptive foregut in temporal and spatial proximity. This requires precisely orchestrated transcriptional activation and repression of organ-specific gene expression within the same cell. Here, we show distinct roles for the chromatin remodelling factor and transcriptional repressor Histone deacetylase 1 (Hdac1) in endodermal organogenesis in zebrafish. Loss of Hdac1 causes defects in timely liver specification and in subsequent differentiation. Mosaic analyses reveal a cell-autonomous requirement for hdac1 within the hepatic endoderm. Our studies further reveal specific functions for Hdac1 in pancreas development. Loss of hdac1 causes the formation of ectopic endocrine clusters anteriorly to the main islet, as well as defects in exocrine pancreas specification and differentiation. In addition, we observe defects in extrahepatopancreatic duct formation and morphogenesis. Finally, loss of hdac1 results in an expansion of the foregut endoderm in the domain from which the liver and pancreas originate. Our genetic studies demonstrate that Hdac1 is crucial for regulating distinct steps in endodermal organogenesis. This suggests a model in which Hdac1 may directly or indirectly restrict foregut fates while promoting hepatic and exocrine pancreatic specification and differentiation, as well as pancreatic endocrine islet morphogenesis. These findings establish zebrafish as a tractable system to investigate chromatin remodelling factor functions in controlling gene expression programmes in vertebrate endodermal organogenesis.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U117581329</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">322</span>;<span class="number">2</span>;<span class="pagerange">237-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18687323">18687323</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3710974">3710974</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2008.06.040">10.1016/j.ydbio.2008.06.040</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcription and chromatin organization of a housekeeping gene cluster containing an integrated beta-globin locus control region.</h4>
<p class="authors"><span class="author">Noordermeer D</span>, <span class="author">Branco MR</span>, <span class="author">Splinter E</span>, <span class="author">Klous P</span>, <span class="author">van Ijcken W</span>, <span class="author">Swagemakers S</span>, <span class="author">Koutsourakis M</span>, <span class="author">van der Spek P</span>, <span class="author">Pombo A</span> and <span class="author">de Laat W</span></p>
<p class="affiliation">Department of Cell Biology and Genetics, Erasmus Medical Center, Rotterdam, The Netherlands.</p><p class="abstract">The activity of locus control regions (LCR) has been correlated with chromatin decondensation, spreading of active chromatin marks, locus repositioning away from its chromosome territory (CT), increased association with transcription factories, and long-range interactions via chromatin looping. To investigate the relative importance of these events in the regulation of gene expression, we targeted the human beta-globin LCR in two opposite orientations to a gene-dense region in the mouse genome containing mostly housekeeping genes. We found that each oppositely oriented LCR influenced gene expression on both sides of the integration site and over a maximum distance of 150 kilobases. A subset of genes was transcriptionally enhanced, some of which in an LCR orientation-dependent manner. The locus resides mostly at the edge of its CT and integration of the LCR in either orientation caused a more frequent positioning of the locus away from its CT. Locus association with transcription factories increased moderately, both for loci at the edge and outside of the CT. These results show that nuclear repositioning is not sufficient to increase transcription of any given gene in this region. We identified long-range interactions between the LCR and two upregulated genes and propose that LCR-gene contacts via chromatin looping determine which genes are transcriptionally enhanced.</p>
<p class="grants">Funded by: Medical Research Council</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">3</span>;<span class="pagerange">e1000016</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18369441" rel="external">18369441</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2265466" rel="external">2265466</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000016" rel="external">10.1371/journal.pgen.1000016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in mRNA export mediator GLE1 result in a fetal motoneuron disease.</h4>
<p class="authors"><span class="author">Nousiainen HO</span>, <span class="author">Kestil&auml; M</span>, <span class="author">Pakkasj&auml;rvi N</span>, <span class="author">Honkala H</span>, <span class="author">Kuure S</span>, <span class="author">Tallila J</span>, <span class="author">Vuopala K</span>, <span class="author">Ignatius J</span>, <span class="author">Herva R</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Department of Molecular Medicine, National Public Health Institute, Helsinki 00290, Finland.</p><p class="abstract">The most severe forms of motoneuron disease manifest in utero are characterized by marked atrophy of spinal cord motoneurons and fetal immobility. Here, we report that the defective gene underlying lethal motoneuron syndrome LCCS1 is the mRNA export mediator GLE1. Our finding of mutated GLE1 exposes a common pathway connecting the genes implicated in LCCS1, LCCS2 and LCCS3 and elucidates mRNA processing as a critical molecular mechanism in motoneuron development and maturation.</p>
<p class="grants">Funded by: NIEHS NIH HHS: <span class="grant">P01 ES11253-03</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">2</span>;<span class="pagerange">155-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18204449">18204449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2684619">2684619</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2007.65">10.1038/ng.2007.65</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Post-genomic challenges for collaborative research in infectious diseases.</h4>
<p class="authors"><span class="author">Okeke IN</span> and <span class="author">Wain J</span></p>
<p class="affiliation">Department of Biology, Haverford College, Haverford, Pennsylvania 19041, USA. <a href="mailto:%69%6F%6B%65%6B%65%40%68%61%76%65%72%66%6F%72%64.%65%64%75">&#105;&#111;&#107;&#101;&#107;&#101;&#64;&#104;&#97;&#118;&#101;&#114;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Although high-burden pathogens have been prioritized for sequencing, genomic research has yet to yield effective vaccines, diagnostics or therapeutics for the infectious diseases that burden developing countries. International research partnerships are needed more today than ever before, and we propose that increased participation by scientists in endemic areas would overcome current roadblocks and is an essential path towards translational research outcomes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">11</span>;<span class="pagerange">858-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18711428">18711428</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1989">10.1038/nrmicro1989</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reconfiguration of genomic anchors upon transcriptional activation of the human major histocompatibility complex.</h4>
<p class="authors"><span class="author">Ottaviani D</span>, <span class="author">Lever E</span>, <span class="author">Mitter R</span>, <span class="author">Jones T</span>, <span class="author">Forshew T</span>, <span class="author">Christova R</span>, <span class="author">Tomazou EM</span>, <span class="author">Rakyan VK</span>, <span class="author">Krawetz SA</span>, <span class="author">Platts AE</span>, <span class="author">Segarane B</span>, <span class="author">Beck S</span> and <span class="author">Sheer D</span></p>
<p class="affiliation">Cancer Research UK London Research Institute, Lincoln&#39;s Inn Fields, London WC2A 3PX, United Kingdom.</p><p class="abstract">The folding of chromatin into topologically constrained loop domains is essential for genomic function. We have identified genomic anchors that define the organization of chromatin loop domains across the human major histocompatibility complex (MHC). This locus contains critical genes for immunity and is associated with more diseases than any other region of the genome. Classical MHC genes are expressed in a cell type-specific pattern and can be induced by cytokines such as interferon-gamma (IFNG). Transcriptional activation of the MHC was associated with a reconfiguration of chromatin architecture resulting from the formation of additional genomic anchors. These findings suggest that the dynamic arrangement of genomic anchors and loops plays a role in transcriptional regulation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A8318</span>, <span class="grant">C5321/A8318</span>; NICHD NIH HHS: <span class="grant">HD36512</span>; Wellcome Trust: <span class="grant">WT084071</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">11</span>;<span class="pagerange">1778-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18849521" rel="external">18849521</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2577859" rel="external">2577859</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.082313.108" rel="external">10.1101/gr.082313.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical immunological pathways are downregulated in APECED patient dendritic cells.</h4>
<p class="authors"><span class="author">P&ouml;ntynen N</span>, <span class="author">Strengell M</span>, <span class="author">Sillanp&auml;&auml; N</span>, <span class="author">Saharinen J</span>, <span class="author">Ulmanen I</span>, <span class="author">Julkunen I</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">National Public Health Institute and FIMM, Institute for Molecular Medicine Finland, Biomedicum, Helsinki, Finland. <a href="mailto:%6E%6F%72%61.%70%6F%6E%74%79%6E%65%6E%40%6B%74%6C.%66%69">&#110;&#111;&#114;&#97;&#46;&#112;&#111;&#110;&#116;&#121;&#110;&#101;&#110;&#64;&#107;&#116;&#108;&#46;&#102;&#105;</a></p><p class="abstract">Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) is a monogenic autoimmune disease caused by mutations in the autoimmune regulator (AIRE) gene. AIRE functions as a transcriptional regulator, and it has a central role in the development of immunological tolerance. AIRE regulates the expression of ectopic antigens in epithelial cells of the thymic medulla and has been shown to participate in the development of peripheral tolerance. However, the mechanism of action of AIRE has remained elusive. To further investigate the role of AIRE in host immune functions, we studied the properties and transcript profiles in in vitro monocyte-differentiated dendritic cells (moDCs) obtained from APECED patients and healthy controls. AIRE-deficient monocytes showed typical DC morphology and expressed DC marker proteins cluster of differentiation 86 and human leukocyte antigen class II. APECED patient-derived moDCs were functionally impaired: the transcriptional response of cytokine genes to pathogens was drastically reduced. Interestingly, some changes were observable already at the immature DC stage. Pathway analyses of transcript profiles revealed that the expression of the components of the host cell signaling pathways involved in cell-cell signalling, innate immune responses, and cytokine activity were reduced in APECED moDCs. Our observations support a role for AIRE in peripheral tolerance and are the first ones to show that AIRE has a critical role in DC responses to microbial stimuli in humans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Journal of molecular medicine (Berlin, Germany)</span>&nbsp;<span class="year">2008</span>;<span class="volume">86</span>;<span class="number">10</span>;<span class="pagerange">1139-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18600308" rel="external">18600308</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2685494" rel="external">2685494</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00109-008-0374-7" rel="external">10.1007/s00109-008-0374-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Scheduling in a dynamic heterogeneous distributed system using estimation error</h4>
<p class="authors"><span class="author">Page</span>, <span class="author">A. J</span>, <span class="author">Keane</span>, <span class="author">T. M</span>, <span class="author">Naughton</span>, <span class="author">T. J.</span></p>
<p class="publication"><span class="title">Journal of Parallel and Distributed Computing.</span> <span class="year">2008</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic adaptation: a fungal perspective.</h4>
<p class="authors"><span class="author">Pain A</span> and <span class="author">Hertz-Fowler C</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">572-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18628767">18628767</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1964">10.1038/nrmicro1964</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of the simian and human malaria parasite Plasmodium knowlesi.</h4>
<p class="authors"><span class="author">Pain A</span>, <span class="author">B&ouml;hme U</span>, <span class="author">Berry AE</span>, <span class="author">Mungall K</span>, <span class="author">Finn RD</span>, <span class="author">Jackson AP</span>, <span class="author">Mourier T</span>, <span class="author">Mistry J</span>, <span class="author">Pasini EM</span>, <span class="author">Aslett MA</span>, <span class="author">Balasubrammaniam S</span>, <span class="author">Borgwardt K</span>, <span class="author">Brooks K</span>, <span class="author">Carret C</span>, <span class="author">Carver TJ</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth T</span>, <span class="author">Clark TG</span>, <span class="author">Galinski MR</span>, <span class="author">Hall N</span>, <span class="author">Harper D</span>, <span class="author">Harris D</span>, <span class="author">Hauser H</span>, <span class="author">Ivens A</span>, <span class="author">Janssen CS</span>, <span class="author">Keane T</span>, <span class="author">Larke N</span>, <span class="author">Lapp S</span>, <span class="author">Marti M</span>, <span class="author">Moule S</span>, <span class="author">Meyer IM</span>, <span class="author">Ormond D</span>, <span class="author">Peters N</span>, <span class="author">Sanders M</span>, <span class="author">Sanders S</span>, <span class="author">Sargeant TJ</span>, <span class="author">Simmonds M</span>, <span class="author">Smith F</span>, <span class="author">Squares R</span>, <span class="author">Thurston S</span>, <span class="author">Tivey AR</span>, <span class="author">Walker D</span>, <span class="author">White B</span>, <span class="author">Zuiderwijk E</span>, <span class="author">Churcher C</span>, <span class="author">Quail MA</span>, <span class="author">Cowman AF</span>, <span class="author">Turner CM</span>, <span class="author">Rajandream MA</span>, <span class="author">Kocken CH</span>, <span class="author">Thomas AW</span>, <span class="author">Newbold CI</span>, <span class="author">Barrell BG</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%61%70%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#112;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Plasmodium knowlesi is an intracellular malaria parasite whose natural vertebrate host is Macaca fascicularis (the &#39;kra&#39; monkey); however, it is now increasingly recognized as a significant cause of human malaria, particularly in southeast Asia. Plasmodium knowlesi was the first malaria parasite species in which antigenic variation was demonstrated, and it has a close phylogenetic relationship to Plasmodium vivax, the second most important species of human malaria parasite (reviewed in ref. 4). Despite their relatedness, there are important phenotypic differences between them, such as host blood cell preference, absence of a dormant liver stage or &#39;hypnozoite&#39; in P. knowlesi, and length of the asexual cycle (reviewed in ref. 4). Here we present an analysis of the P. knowlesi (H strain, Pk1(A+) clone) nuclear genome sequence. This is the first monkey malaria parasite genome to be described, and it provides an opportunity for comparison with the recently completed P. vivax genome and other sequenced Plasmodium genomes. In contrast to other Plasmodium genomes, putative variant antigen families are dispersed throughout the genome and are associated with intrachromosomal telomere repeats. One of these families, the KIRs, contains sequences that collectively match over one-half of the host CD99 extracellular domain, which may represent an unusual form of molecular mimicry.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">085775</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2008</span>;<span class="volume">455</span>;<span class="number">7214</span>;<span class="pagerange">799-803</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18843368">18843368</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656934">2656934</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature07306">10.1038/nature07306</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Estimating the ancestral recombinations graph (ARG) as compatible networks of SNP patterns.</h4>
<p class="authors"><span class="author">Parida L</span>, <span class="author">Mel&eacute; M</span>, <span class="author">Calafell F</span>, <span class="author">Bertranpetit J</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Computational Biology Center, IBM TJ Watson Research, Yorktown Heights, NY 10598, USA. <a href="mailto:%70%61%72%69%64%61%40%75%73.%69%62%6D.%63%6F%6D">&#112;&#97;&#114;&#105;&#100;&#97;&#64;&#117;&#115;&#46;&#105;&#98;&#109;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Traditionally nonrecombinant genome, i.e., mtDNA or Y chromosome, has been used for phylogeography, notably for ease of analysis. The topology of the phylogeny structure in this case is an acyclic graph, which is often a tree, is easy to comprehend and is somewhat easy to infer. However, recombination is an undeniable genetic fact for most part of the genome. Driven by the need for a more complete analysis, we address the problem of estimating the ancestral recombination graph (ARG) from a collection of extant sequences. We exploit the coherence that is observed in the human haplotypes as patterns and present a network model of patterns to reconstruct the ARG. We test our model on simulations that closely mimic the observed haplotypes and observe promising results.</p>
<p class="publication"><span class="title">Journal of computational biology : a journal of computational molecular cell biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">15</span>;<span class="number">9</span>;<span class="pagerange">1133-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18844583" rel="external">18844583</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fcmb.2008.0065" rel="external">10.1089/cmb.2008.0065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">alpha-Isoform of calcium-calmodulin-dependent protein kinase II and postsynaptic density protein 95 differentially regulate synaptic expression of NR2A- and NR2B-containing N-methyl-d-aspartate receptors in hippocampus.</h4>
<p class="authors"><span class="author">Park CS</span>, <span class="author">Elgersma Y</span>, <span class="author">Grant SG</span> and <span class="author">Morrison JH</span></p>
<p class="affiliation">Department of Neuroscience, Mount Sinai School of Medicine, New York, NY 10029, USA. <a href="mailto:%63%68%61%72%6C%65%73.%70%61%72%6B%30%31%40%6D%73%73%6D.%65%64%75">&#99;&#104;&#97;&#114;&#108;&#101;&#115;&#46;&#112;&#97;&#114;&#107;&#48;&#49;&#64;&#109;&#115;&#115;&#109;&#46;&#101;&#100;&#117;</a></p><p class="abstract">N-methyl-d-aspartate receptors (NMDARs) are critical determinants of bidirectional synaptic plasticity, however, studies of NMDAR function have been based primarily on pharmacological and electrophysiological manipulations, and it is still debated whether there are subunit-selective forms of long-term potentiation (LTP) and long-term depression (LTD). Here we provide ultrastructural analyses of axospinous synapses in cornu ammonis field 1 of hippocampus (CA1) stratum radiatum of transgenic mice with mutations to two key underlying postsynaptic density (PSD) proteins, postsynaptic density protein 95 (PSD-95) and the alpha-isoform of calcium-calmodulin-dependent protein kinase II (alphaCaMKII). Distribution profiles of synaptic proteins in these mice reveal very different patterns of subunit-specific NMDAR localization, which may be related to the divergent phenotypes of the two mutants. In PSD-95, Dlg, ZO-1/Dlg-homologous region (PDZ) 3-truncated mutant mice in which LTD could not be induced but LTP was found to be enhanced, we found a subtle, yet preferential displacement of synaptic N-methyl-d-aspartate receptor subunit 2B (NR2B) subunits in lateral regions of the synapse without affecting changes in the localization of N-methyl-d-aspartate receptor subunit 2A (NR2A) subunits. In persistent inhibitory alphaCaMKII Thr305 substituted with Asp in alpha-isoform of calcium-calmodulin kinase II (T305D) mutant mice with severely impaired LTP but stable LTD expression, we found a selective reduction of NR2A subunits at both the synapse and throughout the cytoplasm of the spine without any effect on the NR2B subunit. In an experiment of mutual exclusivity, neither PSD-95 nor alphaCaMKII localization was found to be affected by mutations to the corresponding PSD protein suggesting that they are functionally independent of the other in the regulation of NR2A- and NR2B-containing NMDARs preceding synaptic activity. Consequently, there may exist at least two distinct PSD-95 and alphaCaMKII-specific NMDAR complexes involved in mediating LTP and LTD through opposing signal transduction pathways in synapses of the hippocampus. The contrasting phenotypes of the PSD-95 and alphaCaMKII mutant mice further establish the prospect of an independent and, possibly, competing mechanism for the regulation of NMDAR-dependent bidirectional synaptic plasticity.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">AG06647</span>, <span class="grant">R37 AG006647-20</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Neuroscience</span>&nbsp;<span class="year">2008</span>;<span class="volume">151</span>;<span class="number">1</span>;<span class="pagerange">43-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18082335" rel="external">18082335</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2391003" rel="external">2391003</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.neuroscience.2007.09.075" rel="external">10.1016/j.neuroscience.2007.09.075</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-Genome analysis of pathogen evolution</h4>
<p class="authors"><span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Evolution in Helth and Disease</span> <span class="year">2008</span>;<span class="pagerange">199-213</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome analysis of pathogen evolution</h4>
<p class="authors"><span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Evolution in Health and Disease.</span> <span class="year">2008</span>;<span class="pagerange">199-214</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Time to remove the model organism blinkers.</h4>
<p class="authors"><span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">11</span>;<span class="pagerange">510-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18829323" rel="external">18829323</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tim.2008.08.003" rel="external">10.1016/j.tim.2008.08.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential roles of the PKC novel isoforms, PKCdelta and PKCepsilon, in mouse and human platelets.</h4>
<p class="authors"><span class="author">Pears CJ</span>, <span class="author">Thornber K</span>, <span class="author">Auger JM</span>, <span class="author">Hughes CE</span>, <span class="author">Grygielska B</span>, <span class="author">Protty MB</span>, <span class="author">Pearce AC</span> and <span class="author">Watson SP</span></p>
<p class="affiliation">Department of Biochemistry, University of Oxford, Oxford, United Kingdom. <a href="mailto:%63%61%74%68%65%72%69%6E%65.%70%65%61%72%73%40%62%69%6F%63%68.%6F%78.%61%63.%75%6B">&#99;&#97;&#116;&#104;&#101;&#114;&#105;&#110;&#101;&#46;&#112;&#101;&#97;&#114;&#115;&#64;&#98;&#105;&#111;&#99;&#104;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Increasing evidence suggests that individual isoforms of protein kinase C (PKC) play distinct roles in regulating platelet activation.</p><p class="abstract">In this study, we focus on the role of two novel PKC isoforms, PKCdelta and PKCepsilon, in both mouse and human platelets. PKCdelta is robustly expressed in human platelets and undergoes transient tyrosine phosphorylation upon stimulation by thrombin or the collagen receptor, GPVI, which becomes sustained in the presence of the pan-PKC inhibitor, Ro 31-8220. In mouse platelets, however, PKCdelta undergoes sustained tyrosine phosphorylation upon activation. In contrast the related isoform, PKCepsilon, is expressed at high levels in mouse but not human platelets. There is a marked inhibition in aggregation and dense granule secretion to low concentrations of GPVI agonists in mouse platelets lacking PKCepsilon in contrast to a minor inhibition in response to G protein-coupled receptor agonists. This reduction is mediated by inhibition of tyrosine phosphorylation of the FcRgamma-chain and downstream proteins, an effect also observed in wild-type mouse platelets in the presence of a PKC inhibitor.</p><p class="abstract"><strong>Conclusions</strong>: These results demonstrate a reciprocal relationship in levels of the novel PKC isoforms delta and epsilon in human and mouse platelets and a selective role for PKCepsilon in signalling through GPVI.</p>
<p class="grants">Funded by: British Heart Foundation; Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">11</span>;<span class="pagerange">e3793</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19030108" rel="external">19030108</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2583049" rel="external">2583049</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0003793" rel="external">10.1371/journal.pone.0003793</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence of uropathogenic Proteus mirabilis, a master of both adherence and motility.</h4>
<p class="authors"><span class="author">Pearson MM</span>, <span class="author">Sebaihia M</span>, <span class="author">Churcher C</span>, <span class="author">Quail MA</span>, <span class="author">Seshasayee AS</span>, <span class="author">Luscombe NM</span>, <span class="author">Abdellah Z</span>, <span class="author">Arrosmith C</span>, <span class="author">Atkin B</span>, <span class="author">Chillingworth T</span>, <span class="author">Hauser H</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Norbertczak H</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Walker D</span>, <span class="author">Whithead S</span>, <span class="author">Thomson NR</span>, <span class="author">Rather PN</span>, <span class="author">Parkhill J</span> and <span class="author">Mobley HL</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of Michigan Medical School, Ann Arbor, MI 48109-0620, USA.</p><p class="abstract">The gram-negative enteric bacterium Proteus mirabilis is a frequent cause of urinary tract infections in individuals with long-term indwelling catheters or with complicated urinary tracts (e.g., due to spinal cord injury or anatomic abnormality). P. mirabilis bacteriuria may lead to acute pyelonephritis, fever, and bacteremia. Most notoriously, this pathogen uses urease to catalyze the formation of kidney and bladder stones or to encrust or obstruct indwelling urinary catheters. Here we report the complete genome sequence of P. mirabilis HI4320, a representative strain cultured in our laboratory from the urine of a nursing home patient with a long-term (&gt; or =30 days) indwelling urinary catheter. The genome is 4.063 Mb long and has a G+C content of 38.88%. There is a single plasmid consisting of 36,289 nucleotides. Annotation of the genome identified 3,685 coding sequences and seven rRNA loci. Analysis of the sequence confirmed the presence of previously identified virulence determinants, as well as a contiguous 54-kb flagellar regulon and 17 types of fimbriae. Genes encoding a potential type III secretion system were identified on a low-G+C-content genomic island containing 24 intact genes that appear to encode all components necessary to assemble a type III secretion system needle complex. In addition, the P. mirabilis HI4320 genome possesses four tandem copies of the zapE metalloprotease gene, genes encoding six putative autotransporters, an extension of the atf fimbrial operon to six genes, including an mrpJ homolog, and genes encoding at least five iron uptake mechanisms, two potential type IV secretion systems, and 16 two-component regulators.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI059722</span>, <span class="grant">F32 AI068324</span>, <span class="grant">F32 AI068324-01A2</span>, <span class="grant">T32 AI7528</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">11</span>;<span class="pagerange">4027-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18375554">18375554</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2395036">2395036</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01981-07">10.1128/JB.01981-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Exclusion of biglycan mutations in a cohort of patients with neuromuscular disorders.</h4>
<p class="authors"><span class="author">Peat RA</span>, <span class="author">G&eacute;cz J</span>, <span class="author">Fallon JR</span>, <span class="author">Tarpey PS</span>, <span class="author">Smith R</span>, <span class="author">Futreal A</span>, <span class="author">Stratton MR</span>, <span class="author">Lamand&eacute; SR</span>, <span class="author">Yang N</span> and <span class="author">North KN</span></p>
<p class="affiliation">Institute for Neuromuscular Research, The Children&#39;s Hospital at Westmead, Sydney, Australia.</p><p class="abstract">Biglycan has been considered a good candidate for neuromuscular disease based on direct interactions with collagen VI and alpha-dystroglycan, both of which are linked with congenital muscular dystrophy (CMD). We screened 83 patients with CMD and other neuromuscular disorders and six controls for mutations and variations in the biglycan sequence. We identified a number of novel sequence variations. After family analysis and control screening we found that none of these polymorphisms were disease-causing mutations. Thus mutations in biglycan are not a common cause of neuromuscular disorders in our cohort.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R01 HD023924-17</span>; Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Neuromuscular disorders : NMD</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">8</span>;<span class="pagerange">606-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18602826" rel="external">18602826</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873833" rel="external">2873833</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.nmd.2008.05.013" rel="external">10.1016/j.nmd.2008.05.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome painting shows that skunks (Mephitidae, Carnivora) have highly rearranged karyotypes.</h4>
<p class="authors"><span class="author">Perelman PL</span>, <span class="author">Graphodatsky AS</span>, <span class="author">Dragoo JW</span>, <span class="author">Serdyukova NA</span>, <span class="author">Stone G</span>, <span class="author">Cavagna P</span>, <span class="author">Menotti A</span>, <span class="author">Nie W</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Wang J</span>, <span class="author">Burkett S</span>, <span class="author">Yuki K</span>, <span class="author">Roelke ME</span>, <span class="author">O&#39;Brien SJ</span>, <span class="author">Yang F</span> and <span class="author">Stanyon R</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS 630090, Novosibirsk, Russia. <a href="mailto:%70%6F%6C%40%62%69%6F%6E%65%74.%6E%73%63.%72%75">&#112;&#111;&#108;&#64;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;</a></p><p class="abstract">The karyotypic relationships of skunks (Mephitidae) with other major clades of carnivores are not yet established. Here, multi-directional chromosome painting was used to reveal the karyological relationships among skunks and between Mephitidae (skunks) and Procyonidae (raccoons). Representative species from three genera of Mephitidae (Mephitis mephitis, 2n = 50; Mephitis macroura, 2n = 50; Conepatus leuconotus, 2n = 46; Spilogale gracilis, 2n = 60) and one species of Procyonidae (Procyon lotor, 2n = 38) were studied. Chromosomal homology was mapped by hybridization of five sets of whole-chromosome paints derived from stone marten (Martes foina, 2n = 38), cat, skunks (M. mephitis; M. macroura) and human. The karyotype of the raccoon is highly conserved and identical to the hypothetical ancestral musteloid karyotype, suggesting that procyonids have a particular importance for establishing the karyological evolution within the caniforms. Ten fission events and five fusion events are necessary to generate the ancestral skunk karyotype from the ancestral carnivore karyotype. Our results show that Mephitidae joins Canidae and Ursidae as the third family of carnivores that are characterized by a high rate of karyotype evolution. Shared derived chromosomal fusion of stone marten chromosomes 6 and 14 phylogenetically links the American hog-nosed skunk and eastern spotted skunk.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">8</span>;<span class="pagerange">1215-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19051045" rel="external">19051045</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-008-1270-2" rel="external">10.1007/s10577-008-1270-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Copy number variation and evolution in humans and chimpanzees.</h4>
<p class="authors"><span class="author">Perry GH</span>, <span class="author">Yang F</span>, <span class="author">Marques-Bonet T</span>, <span class="author">Murphy C</span>, <span class="author">Fitzgerald T</span>, <span class="author">Lee AS</span>, <span class="author">Hyland C</span>, <span class="author">Stone AC</span>, <span class="author">Hurles ME</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Eichler EE</span>, <span class="author">Carter NP</span>, <span class="author">Lee C</span> and <span class="author">Redon R</span></p>
<p class="affiliation">School of Human Evolution &amp; Social Change, Arizona State University, Tempe, Arizona 85287, USA.</p><p class="abstract">Copy number variants (CNVs) underlie many aspects of human phenotypic diversity and provide the raw material for gene duplication and gene family expansion. However, our understanding of their evolutionary significance remains limited. We performed comparative genomic hybridization on a single human microarray platform to identify CNVs among the genomes of 30 humans and 30 chimpanzees as well as fixed copy number differences between species. We found that human and chimpanzee CNVs occur in orthologous genomic regions far more often than expected by chance and are strongly associated with the presence of highly homologous intrachromosomal segmental duplications. By adapting population genetic analyses for use with copy number data, we identified functional categories of genes that have likely evolved under purifying or positive selection for copy number changes. In particular, duplications and deletions of genes with inflammatory response and cell proliferation functions may have been fixed by positive selection and involved in the adaptive phenotypic differentiation of humans and chimpanzees.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NCRR NIH HHS: <span class="grant">RR014491</span>, <span class="grant">RR015087</span>, <span class="grant">RR016483</span>; NHGRI NIH HHS: <span class="grant">HG004221</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">11</span>;<span class="pagerange">1698-710</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18775914">18775914</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2577862">2577862</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.082016.108">10.1101/gr.082016.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular characterization of the Salmonella enterica serovar Typhi Vi-typing bacteriophage E1.</h4>
<p class="authors"><span class="author">Pickard D</span>, <span class="author">Thomson NR</span>, <span class="author">Baker S</span>, <span class="author">Wain J</span>, <span class="author">Pardo M</span>, <span class="author">Goulding D</span>, <span class="author">Hamlin N</span>, <span class="author">Choudhary J</span>, <span class="author">Threfall J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%64%6A%70%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#106;&#112;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Some bacteriophages target potentially pathogenic bacteria by exploiting surface-associated virulence factors as receptors. For example, phage have been identified that exhibit specificity for Vi capsule producing Salmonella enterica serovar Typhi. Here we have characterized the Vi-associated E1-typing bacteriophage using a number of molecular approaches. The absolute requirement for Vi capsule expression for infectivity was demonstrated using different Vi-negative S. enterica derivatives. The phage particles were shown to have an icosahedral head and a long noncontractile tail structure. The genome is 45,362 bp in length with defined capsid and tail regions that exhibit significant homology to the S. enterica transducing phage ES18. Mass spectrometry was used to confirm the presence of a number of hypothetical proteins in the Vi phage E1 particle and demonstrate that a number of phage proteins are modified posttranslationally. The genome of the Vi phage E1 is significantly related to other bacteriophages belonging to the same serovar Typhi phage-typing set, and we demonstrate a role for phage DNA modification in determining host specificity.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">7</span>;<span class="pagerange">2580-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18192390">18192390</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2293211">2293211</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01654-07">10.1128/JB.01654-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cytogenetic studies and karyotype nomenclature of three wild canid species: maned wolf (Chrysocyon brachyurus), bat-eared fox (Otocyon megalotis) and fennec fox (Fennecus zerda).</h4>
<p class="authors"><span class="author">Pie&#x144;kowska-Schelling A</span>, <span class="author">Schelling C</span>, <span class="author">Zawada M</span>, <span class="author">Yang F</span>, <span class="author">Bugno M</span> and <span class="author">Ferguson-Smith M</span></p>
<p class="affiliation">Department of Animal Sciences, Swiss Federal Institute of Technology, Zurich, Switzerland.</p><p class="abstract">We have analysed the chromosomes of three wild and endangered canid species: the maned wolf (Chrysocyon brachyurus), the bat-eared fox (Otocyon megalotis) and the fennec fox (Fennecuszerda) using classical and molecular cytogenetic methods. For the first time detailed and encompassing descriptions of the chromosomes are presented including the chromosomal assignment of nucleolar organizer regions and the 5S rRNA gene cluster. We propose a karyotype nomenclature with ideograms including more than 300 bands per haploid set for each of these three species which will form the basis for further research. In addition, we propose four basic different patterns of karyotype organization in the family Canidae. A comparison of these patterns with the most recent molecular phylogeny of Canidae revealed that the karyotype evolution of a species is not always strongly connected with its phylogenetic position. Our findings underline the need and justification for basic cytogenetic work in rare and exotic species.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">121</span>;<span class="number">1</span>;<span class="pagerange">25-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18544923" rel="external">18544923</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000124378" rel="external">10.1159/000124378</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global transcript profiles of fat in monozygotic twins discordant for BMI: pathways behind acquired obesity.</h4>
<p class="authors"><span class="author">Pietil&auml;inen KH</span>, <span class="author">Naukkarinen J</span>, <span class="author">Rissanen A</span>, <span class="author">Saharinen J</span>, <span class="author">Ellonen P</span>, <span class="author">Ker&auml;nen H</span>, <span class="author">Suomalainen A</span>, <span class="author">G&ouml;tz A</span>, <span class="author">Suortti T</span>, <span class="author">Yki-J&auml;rvinen H</span>, <span class="author">Oresic M</span>, <span class="author">Kaprio J</span> and <span class="author">Peltonen L</span></p>
<p class="affiliation">Obesity Research Unit, Department of Psychiatry, Helsinki University Central Hospital, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: The acquired component of complex traits is difficult to dissect in humans. Obesity represents such a trait, in which the metabolic and molecular consequences emerge from complex interactions of genes and environment. With the substantial morbidity associated with obesity, a deeper understanding of the concurrent metabolic changes is of considerable importance. The goal of this study was to investigate this important acquired component and expose obesity-induced changes in biological pathways in an identical genetic background.</p><p class="abstract">We used a special study design of &quot;clonal controls,&quot; rare monozygotic twins discordant for obesity identified through a national registry of 2,453 young, healthy twin pairs. A total of 14 pairs were studied (eight male, six female; white), with a mean +/- standard deviation (SD) age 25.8 +/- 1.4 y and a body mass index (BMI) difference 5.2 +/- 1.8 kg/m(2). Sequence analyses of mitochondrial DNA (mtDNA) in subcutaneous fat and peripheral leukocytes revealed no aberrant heteroplasmy between the co-twins. However, mtDNA copy number was reduced by 47% in the obese co-twin&#39;s fat. In addition, novel pathway analyses of the adipose tissue transcription profiles exposed significant down-regulation of mitochondrial branched-chain amino acid (BCAA) catabolism (p &lt; 0.0001). In line with this finding, serum levels of insulin secretion-enhancing BCAAs were increased in obese male co-twins (9% increase, p = 0.025). Lending clinical relevance to the findings, in both sexes the observed aberrations in mitochondrial amino acid metabolism pathways in fat correlated closely with liver fat accumulation, insulin resistance, and hyperinsulinemia, early aberrations of acquired obesity in these healthy young adults.</p><p class="abstract"><strong>Conclusions</strong>: Our findings emphasize a substantial role of mitochondrial energy- and amino acid metabolism in obesity and development of insulin resistance.</p>
<p class="grants">Funded by: NIAAA NIH HHS: <span class="grant">AA-00145</span>, <span class="grant">AA-08315</span>, <span class="grant">AA-12502</span></p>
<p class="publication"><span class="title">PLoS medicine</span>&nbsp;<span class="year">2008</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">e51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18336063" rel="external">18336063</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2265758" rel="external">2265758</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pmed.0050051" rel="external">10.1371/journal.pmed.0050051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Endoglin expression in blood and endothelium is differentially regulated by modular assembly of the Ets/Gata hemangioblast code.</h4>
<p class="authors"><span class="author">Pimanda JE</span>, <span class="author">Chan WY</span>, <span class="author">Wilson NK</span>, <span class="author">Smith AM</span>, <span class="author">Kinston S</span>, <span class="author">Knezevic K</span>, <span class="author">Janes ME</span>, <span class="author">Landry JR</span>, <span class="author">Kolb-Kokocinski A</span>, <span class="author">Frampton J</span>, <span class="author">Tannahill D</span>, <span class="author">Ottersbach K</span>, <span class="author">Follows GA</span>, <span class="author">Lacaud G</span>, <span class="author">Kouskoff V</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, University of Cambridge, Cambridge, United Kingdom. <a href="mailto:%6A%70%69%6D%61%6E%64%61%40%75%6E%73%77.%65%64%75.%61%75">&#106;&#112;&#105;&#109;&#97;&#110;&#100;&#97;&#64;&#117;&#110;&#115;&#119;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Endoglin is an accessory receptor for TGF-beta signaling and is required for normal hemangioblast, early hematopoietic, and vascular development. We have previously shown that an upstream enhancer, Eng -8, together with the promoter region, mediates robust endothelial expression yet is inactive in blood. To identify hematopoietic regulatory elements, we used array-based methods to determine chromatin accessibility across the entire locus. Subsequent transgenic analysis of candidate elements showed that an endothelial enhancer at Eng +9 when combined with an element at Eng +7 functions as a strong hemato-endothelial enhancer. Chromatin immunoprecipitation (ChIP)-chip analysis demonstrated specific binding of Ets factors to the promoter as well as to the -8, +7+9 enhancers in both blood and endothelial cells. By contrast Pu.1, an Ets factor specific to the blood lineage, and Gata2 binding was only detected in blood. Gata2 was bound only at +7 and GATA motifs were required for hematopoietic activity. This modular assembly of regulators gives blood and endothelial cells the regulatory freedom to independently fine-tune gene expression and emphasizes the role of regulatory divergence in driving functional divergence.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Cancer Research UK</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2008</span>;<span class="volume">112</span>;<span class="number">12</span>;<span class="pagerange">4512-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18805961" rel="external">18805961</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682356" rel="external">2682356</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2008-05-157560" rel="external">10.1182/blood-2008-05-157560</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BAC TransgeneOmics: a high-throughput method for exploration of protein function in mammals.</h4>
<p class="authors"><span class="author">Poser I</span>, <span class="author">Sarov M</span>, <span class="author">Hutchins JR</span>, <span class="author">H&eacute;rich&eacute; JK</span>, <span class="author">Toyoda Y</span>, <span class="author">Pozniakovsky A</span>, <span class="author">Weigl D</span>, <span class="author">Nitzsche A</span>, <span class="author">Hegemann B</span>, <span class="author">Bird AW</span>, <span class="author">Pelletier L</span>, <span class="author">Kittler R</span>, <span class="author">Hua S</span>, <span class="author">Naumann R</span>, <span class="author">Augsburg M</span>, <span class="author">Sykora MM</span>, <span class="author">Hofemeister H</span>, <span class="author">Zhang Y</span>, <span class="author">Nasmyth K</span>, <span class="author">White KP</span>, <span class="author">Dietzel S</span>, <span class="author">Mechtler K</span>, <span class="author">Durbin R</span>, <span class="author">Stewart AF</span>, <span class="author">Peters JM</span>, <span class="author">Buchholz F</span> and <span class="author">Hyman AA</span></p>
<p class="affiliation">Max Planck Institute for Molecular Cell Biology and Genetics, Pfotenhauerstrasse 108, D-01307 Dresden, Germany.</p><p class="abstract">The interpretation of genome sequences requires reliable and standardized methods to assess protein function at high throughput. Here we describe a fast and reliable pipeline to study protein function in mammalian cells based on protein tagging in bacterial artificial chromosomes (BACs). The large size of the BAC transgenes ensures the presence of most, if not all, regulatory elements and results in expression that closely matches that of the endogenous gene. We show that BAC transgenes can be rapidly and reliably generated using 96-well-format recombineering. After stable transfection of these transgenes into human tissue culture cells or mouse embryonic stem cells, the localization, protein-protein and/or protein-DNA interactions of the tagged protein are studied using generic, tag-based assays. The same high-throughput approach will be generally applicable to other model systems.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1R01HG004428-01</span>; Wellcome Trust: <span class="grant">077192</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2008</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">409-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18391959">18391959</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871289">2871289</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1199">10.1038/nmeth.1199</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mosaic complementation demonstrates a regulatory role for myosin VIIa in actin dynamics of stereocilia.</h4>
<p class="authors"><span class="author">Prosser HM</span>, <span class="author">Rzadzinska AK</span>, <span class="author">Steel KP</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%68%6D%70%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#109;&#112;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have developed a bacterial artificial chromosome transgenesis approach that allowed the expression of myosin VIIa from the mouse X chromosome. We demonstrated the complementation of the Myo7a null mutant phenotype producing a fine mosaic of two types of sensory hair cells within inner ear epithelia of hemizygous transgenic females due to X inactivation. Direct comparisons between neighboring auditory hair cells that were different only with respect to myosin VIIa expression revealed that mutant stereocilia are significantly longer than those of their complemented counterparts. Myosin VIIa-deficient hair cells showed an abnormally persistent tip localization of whirlin, a protein directly linked to elongation of stereocilia, in stereocilia. Furthermore, myosin VIIa localized at the tips of all abnormally short stereocilia of mice deficient for either myosin XVa or whirlin. Our results strongly suggest that myosin VIIa regulates the establishment of a setpoint for stereocilium heights, and this novel role may influence their normal staircase-like arrangement within a bundle.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">28</span>;<span class="number">5</span>;<span class="pagerange">1702-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18160714">18160714</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258769">2258769</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.01282-07">10.1128/MCB.01282-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A large genome center&#39;s improvements to the Illumina sequencing system.</h4>
<p class="authors"><span class="author">Quail MA</span>, <span class="author">Kozarewa I</span>, <span class="author">Smith F</span>, <span class="author">Scally A</span>, <span class="author">Stephens PJ</span>, <span class="author">Durbin R</span>, <span class="author">Swerdlow H</span> and <span class="author">Turner DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">The Wellcome Trust Sanger Institute is one of the world&#39;s largest genome centers, and a substantial amount of our sequencing is performed with &#39;next-generation&#39; massively parallel sequencing technologies: in June 2008 the quantity of purity-filtered sequence data generated by our Genome Analyzer (Illumina) platforms reached 1 terabase, and our average weekly Illumina production output is currently 64 gigabases. Here we describe a set of improvements we have made to the standard Illumina protocols to make the library preparation more reliable in a high-throughput environment, to reduce bias, tighten insert size distribution and reliably obtain high yields of data.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701805</span>; Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2008</span>;<span class="volume">5</span>;<span class="number">12</span>;<span class="pagerange">1005-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19034268">19034268</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2610436">2610436</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth.1270">10.1038/nmeth.1270</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A software framework for microarray and gene expression object model (MAGE-OM) array design annotation.</h4>
<p class="authors"><span class="author">Qureshi M</span> and <span class="author">Ivens A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%6D%61%74%6C%6F%6F%62%40%67%6D%61%69%6C.%63%6F%6D">&#109;&#97;&#116;&#108;&#111;&#111;&#98;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: The MIAME and MAGE-OM standards defined by the MGED society provide a specification and implementation of a software infrastructure to facilitate the submission and sharing of data from microarray studies via public repositories. However, although the MAGE object model is flexible enough to support different annotation strategies, the annotation of array descriptions can be complex.</p><p class="abstract"><strong>Results</strong>: We have developed a graphical Java-based application (Adamant) to assist with submission of Microarray designs to public repositories. Output of the application is fully compliant with the standards prescribed by the various public data repositories.</p><p class="abstract"><strong>Conclusion</strong>: Adamant will allow researchers to annotate and submit their own array designs to public repositories without requiring programming expertise, knowledge of the MAGE-OM or XML. The application has been used to submit a number of ArrayDesigns to the Array Express database.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">133</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18366695" rel="external">18366695</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2358904" rel="external">2358904</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-133" rel="external">10.1186/1471-2164-9-133</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated resource for genome-wide identification and analysis of human tissue-specific differentially methylated regions (tDMRs).</h4>
<p class="authors"><span class="author">Rakyan VK</span>, <span class="author">Down TA</span>, <span class="author">Thorne NP</span>, <span class="author">Flicek P</span>, <span class="author">Kulesha E</span>, <span class="author">Gr&auml;f S</span>, <span class="author">Tomazou EM</span>, <span class="author">B&auml;ckdahl L</span>, <span class="author">Johnson N</span>, <span class="author">Herberth M</span>, <span class="author">Howe KL</span>, <span class="author">Jackson DK</span>, <span class="author">Miretti MM</span>, <span class="author">Fiegler H</span>, <span class="author">Marioni JC</span>, <span class="author">Birney E</span>, <span class="author">Hubbard TJ</span>, <span class="author">Carter NP</span>, <span class="author">Tavar&eacute; S</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Institute of Cell and Molecular Science, Barts and the London, London E1 2AT, United Kingdom. <a href="mailto:%76.%72%61%6B%79%61%6E%40%71%6D%75%6C.%61%63.%75%6B">&#118;&#46;&#114;&#97;&#107;&#121;&#97;&#110;&#64;&#113;&#109;&#117;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We report a novel resource (methylation profiles of DNA, or mPod) for human genome-wide tissue-specific DNA methylation profiles. mPod consists of three fully integrated parts, genome-wide DNA methylation reference profiles of 13 normal somatic tissues, placenta, sperm, and an immortalized cell line, a visualization tool that has been integrated with the Ensembl genome browser and a new algorithm for the analysis of immunoprecipitation-based DNA methylation profiles. We demonstrate the utility of our resource by identifying the first comprehensive genome-wide set of tissue-specific differentially methylated regions (tDMRs) that may play a role in cellular identity and the regulation of tissue-specific genome function. We also discuss the implications of our findings with respect to the regulatory potential of regions with varied CpG density, gene expression, transcription factor motifs, gene ontology, and correlation with other epigenetic marks such as histone modifications.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">C14303/A4646</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">9</span>;<span class="pagerange">1518-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18577705">18577705</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2527707">2527707</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.077479.108">10.1101/gr.077479.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The pangenome structure of Escherichia coli: comparative genomic analysis of E. coli commensal and pathogenic isolates.</h4>
<p class="authors"><span class="author">Rasko DA</span>, <span class="author">Rosovitz MJ</span>, <span class="author">Myers GS</span>, <span class="author">Mongodin EF</span>, <span class="author">Fricke WF</span>, <span class="author">Gajer P</span>, <span class="author">Crabtree J</span>, <span class="author">Sebaihia M</span>, <span class="author">Thomson NR</span>, <span class="author">Chaudhuri R</span>, <span class="author">Henderson IR</span>, <span class="author">Sperandio V</span> and <span class="author">Ravel J</span></p>
<p class="affiliation">Institute for Genome Sciences, Department of Microbiology &amp; Immunology, University of Maryland School of Medicine, 20 Penn Street, Baltimore, MD 21201, USA.</p><p class="abstract">Whole-genome sequencing has been skewed toward bacterial pathogens as a consequence of the prioritization of medical and veterinary diseases. However, it is becoming clear that in order to accurately measure genetic variation within and between pathogenic groups, multiple isolates, as well as commensal species, must be sequenced. This study examined the pangenomic content of Escherichia coli. Six distinct E. coli pathovars can be distinguished using molecular or phenotypic markers, but only two of the six pathovars have been subjected to any genome sequencing previously. Thus, this report provides a seminal description of the genomic contents and unique features of three unsequenced pathovars, enterotoxigenic E. coli, enteropathogenic E. coli, and enteroaggregative E. coli. We also determined the first genome sequence of a human commensal E. coli isolate, E. coli HS, which will undoubtedly provide a new baseline from which workers can examine the evolution of pathogenic E. coli. Comparison of 17 E. coli genomes, 8 of which are new, resulted in identification of approximately 2,200 genes conserved in all isolates. We were also able to identify genes that were isolate and pathovar specific. Fewer pathovar-specific genes were identified than anticipated, suggesting that each isolate may have independently developed virulence capabilities. Pangenome calculations indicate that E. coli genomic diversity represents an open pangenome model containing a reservoir of more than 13,000 genes, many of which may be uncharacterized but important virulence factors. This comparative study of the species E. coli, while descriptive, should provide the basis for future functional work on this important group of pathogens.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">N01-AI-30071</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2008</span>;<span class="volume">190</span>;<span class="number">20</span>;<span class="pagerange">6881-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18676672" rel="external">18676672</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2566221" rel="external">2566221</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00619-08" rel="external">10.1128/JB.00619-08</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The MEROPS batch BLAST: a tool to detect peptidases and their non-peptidase homologues in a genome.</h4>
<p class="authors"><span class="author">Rawlings ND</span> and <span class="author">Morton FR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Many of the 181 families of peptidases contain homologues that are known to have functions other than peptide bond hydrolysis. Distinguishing an active peptidase from a homologue that is not a peptidase requires specialist knowledge of the important active site residues, because replacement or lack of one of these catalytic residues is an important clue that the homologue in question is unlikely to hydrolyse peptide bonds. Now that the rate at which proteins are characterized is outstripped by the rate that genome sequences are determined, many genes are being incorrectly annotated because only sequence similarity is taken into consideration. We present a tool called the MEROPS batch BLAST which not only performs a comparison against the MEROPS sequence collection, but also does a pair-wise alignment with the closest homologue detected and calculates the position of the active site residues. A non-peptidase homologue can be distinguished by the absence or unacceptable replacement of any of these residues. An analysis of peptidase homologues in the genome of the bacterium Erythrobacter litoralis is presented as an example.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Biochimie</span>&nbsp;<span class="year">2008</span>;<span class="volume">90</span>;<span class="number">2</span>;<span class="pagerange">243-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17980477" rel="external">17980477</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biochi.2007.09.014" rel="external">10.1016/j.biochi.2007.09.014</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MEROPS: the peptidase database.</h4>
<p class="authors"><span class="author">Rawlings ND</span>, <span class="author">Morton FR</span>, <span class="author">Kok CY</span>, <span class="author">Kong J</span> and <span class="author">Barrett AJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Peptidases (proteolytic enzymes or proteases), their substrates and inhibitors are of great relevance to biology, medicine and biotechnology. The MEROPS database (http://merops.sanger.ac.uk) aims to fulfil the need for an integrated source of information about these. The organizational principle of the database is a hierarchical classification in which homologous sets of peptidases and protein inhibitors are grouped into protein species, which are grouped into families and in turn grouped into clans. Important additions to the database include newly written, concise text annotations for peptidase clans and the small molecule inhibitors that are outside the scope of the standard classification; displays to show peptidase specificity compiled from our collection of known substrate cleavages; tables of peptidase-inhibitor interactions; and dynamically generated alignments of representatives of each protein species at the family level. New ways to compare peptidase and inhibitor complements between any two organisms whose genomes have been completely sequenced, or between different strains or subspecies of the same organism, have been devised.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D320-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17991683">17991683</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238837">2238837</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm954">10.1093/nar/gkm954</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A logic-based diagram of signalling pathways central to macrophage activation.</h4>
<p class="authors"><span class="author">Raza S</span>, <span class="author">Robertson KA</span>, <span class="author">Lacaze PA</span>, <span class="author">Page D</span>, <span class="author">Enright AJ</span>, <span class="author">Ghazal P</span> and <span class="author">Freeman TC</span></p>
<p class="affiliation">Division of Pathway Medicine, University of Edinburgh, The Chancellor&#39;s Building, College of Medicine, 49 Little France Crescent, Edinburgh, EH16 4SB, UK. <a href="mailto:%73%6F%62%69%61.%72%61%7A%61%40%65%64.%61%63.%75%6B">&#115;&#111;&#98;&#105;&#97;&#46;&#114;&#97;&#122;&#97;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The complex yet flexible cellular response to pathogens is orchestrated by the interaction of multiple signalling and metabolic pathways. The molecular regulation of this response has been studied in great detail but comprehensive and unambiguous diagrams describing these events are generally unavailable. Four key signalling cascades triggered early-on in the innate immune response are the toll-like receptor, interferon, NF-kappaB and apoptotic pathways, which co-operate to defend cells against a given pathogen. However, these pathways are commonly viewed as separate entities rather than an integrated network of molecular interactions.</p><p class="abstract"><strong>Results</strong>: Here we describe the construction of a logically represented pathway diagram which attempts to integrate these four pathways central to innate immunity using a modified version of the Edinburgh Pathway Notation. The pathway map is available in a number of electronic formats and editing is supported by yEd graph editor software.</p><p class="abstract"><strong>Conclusion</strong>: The map presents a powerful visual aid for interpreting the available pathway interaction knowledge and underscores the valuable contribution well constructed pathway diagrams make to communicating large amounts of molecular interaction data. Furthermore, we discuss issues with the limitations and scalability of pathways presented in this fashion, explore options for automated layout of large pathway networks and demonstrate how such maps can aid the interpretation of functional studies.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">BMC systems biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">2</span>;<span class="pagerange">36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18433497" rel="external">18433497</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2383880" rel="external">2383880</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1752-0509-2-36" rel="external">10.1186/1752-0509-2-36</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Protein Feature Ontology: a tool for the unification of protein feature annotations.</h4>
<p class="authors"><span class="author">Reeves GA</span>, <span class="author">Eilbeck K</span>, <span class="author">Magrane M</span>, <span class="author">O&#39;Donovan C</span>, <span class="author">Montecchi-Palazzi L</span>, <span class="author">Harris MA</span>, <span class="author">Orchard S</span>, <span class="author">Jimenez RC</span>, <span class="author">Prlic A</span>, <span class="author">Hubbard TJ</span>, <span class="author">Hermjakob H</span> and <span class="author">Thornton JM</span></p>
<p class="affiliation">EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK. <a href="mailto:%67%61%62%62%79%40%65%62%69.%61%63.%75%6B">&#103;&#97;&#98;&#98;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: The advent of sequencing and structural genomics projects has provided a dramatic boost in the number of uncharacterized protein structures and sequences. Consequently, many computational tools have been developed to help elucidate protein function. However, such services are spread throughout the world, often with standalone web pages. Integration of these methods is needed and so far this has not been possible as there was no common vocabulary available that could be used as a standard language.</p><p class="abstract"><strong>Results</strong>: The Protein Feature Ontology has been developed to provide a structured controlled vocabulary for features on a protein sequence or structure and comprises approximately 100 positional terms, now integrated into the Sequence Ontology (SO) and 40 non-positional terms which describe features relating to the whole-protein sequence. In addition, post-translational modifications are described by using a pre-existing ontology, the Protein Modification Ontology (MOD). This ontology is being used to integrate over 150 distinct annotations provided by the BioSapiens Network of Excellence, a consortium comprising 19 partner sites in Europe.</p><p class="abstract"><strong>Availability</strong>: The Protein Feature Ontology can be browsed by accessing the ontology lookup service at the European Bioinformatics Institute (http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=BS).</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">23</span>;<span class="pagerange">2767-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18936051">18936051</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2912506">2912506</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtn528">10.1093/bioinformatics/btn528</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fission yeast MAP kinase Sty1 is recruited to stress-induced genes.</h4>
<p class="authors"><span class="author">Reiter W</span>, <span class="author">Watt S</span>, <span class="author">Dawson K</span>, <span class="author">Lawrence CL</span>, <span class="author">B&auml;hler J</span>, <span class="author">Jones N</span> and <span class="author">Wilkinson CR</span></p>
<p class="affiliation">Paterson Institute for Cancer Research, University of Manchester, Wilmslow Road, Manchester, UK.</p><p class="abstract">The stress-induced expression of many fission yeast genes is dependent upon the Sty1 mitogen-activated protein kinase (MAPK) and Atf1 transcription factor. Atf1 is phosphorylated by Sty1 yet this phosphorylation is not required for stress-induced gene expression, suggesting another mechanism exists whereby Sty1 activates transcription. Here we show that Sty1 associates with Atf1-dependent genes and is recruited to both their promoters and coding regions. This occurs in response to various stress conditions coincident with the kinetics of the activation of Sty1. Association with promoters is not a consequence of increased nuclear accumulation of Sty1 nor does it require the phosphorylation of Atf1. However, recruitment is completely abolished in a mutant lacking Sty1 kinase activity. Both Atf1 and its binding partner Pcr1 are required for association of Sty1 with Atf1-dependent promoters, suggesting that this heterodimer must be intact for optimal recruitment of the MAPK. However, many Atf1-dependent genes are still expressed in a pcr1Delta mutant but with significantly delayed kinetics, thus providing an explanation for the relatively mild stress sensitivity displayed by pcr1Delta. Consistent with this delay, Sty1 and Atf1 cannot be detected at these promoters in this condition, suggesting that their association with chromatin is weak or transient in the absence of Pcr1.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span>, <span class="grant">098051</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2008</span>;<span class="volume">283</span>;<span class="number">15</span>;<span class="pagerange">9945-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18252721">18252721</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3668131">3668131</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M710428200">10.1074/jbc.M710428200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Different pathways to acquiring resistance genes illustrated by the recent evolution of IncW plasmids.</h4>
<p class="authors"><span class="author">Revilla C</span>, <span class="author">Garcill&aacute;n-Barcia MP</span>, <span class="author">Fern&aacute;ndez-L&oacute;pez R</span>, <span class="author">Thomson NR</span>, <span class="author">Sanders M</span>, <span class="author">Cheung M</span>, <span class="author">Thomas CM</span> and <span class="author">de la Cruz F</span></p>
<p class="affiliation">Departamento de Biolog&iacute;a Molecular e Instituto de Biomedicina y Biotecnolog&iacute;a de Cantabria, Universidad de Cantabria-CSIC-IDICAN, C. Herrera Oria s/n, 39011 Santander, Spain.</p><p class="abstract">DNA sequence analysis of five IncW plasmids (R388, pSa, R7K, pIE321, and pIE522) demonstrated that they share a considerable portion of their genomes and allowed us to define the IncW backbone. Among these plasmids, the backbone is stable and seems to have diverged recently, since the overall identity among its members is higher than 95%. The only gene in which significant variation was observed was trwA; the changes in the coding sequence correlated with parallel changes in the corresponding TrwA binding sites at oriT, suggesting a functional connection between both sets of changes. The present IncW plasmid diversity is shaped by the acquisition of antibiotic resistance genes as a consequence of the pressure exerted by antibiotic usage. Sequence comparisons pinpointed the insertion events that differentiated the five plasmids analyzed. Of greatest interest is that a single acquisition of a class I integron platform, into which different gene cassettes were later incorporated, gave rise to plasmids R388, pIE522, and pSa, while plasmids R7K and pIE321 do not contain the integron platform and arose in the antibiotic world because of the insertion of several antibiotic resistance transposons.</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2008</span>;<span class="volume">52</span>;<span class="number">4</span>;<span class="pagerange">1472-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18268088" rel="external">18268088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2292564" rel="external">2292564</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.00982-07" rel="external">10.1128/AAC.00982-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use and misuse of the gene ontology annotations.</h4>
<p class="authors"><span class="author">Rhee SY</span>, <span class="author">Wood V</span>, <span class="author">Dolinski K</span> and <span class="author">Draghici S</span></p>
<p class="affiliation">Carnegie Institution for Science, Department of Plant Biology, 260 Panama Street, Stanford, California 94305, USA. <a href="mailto:%72%68%65%65%40%61%63%6F%6D%61.%73%74%61%6E%66%6F%72%64.%65%64%75">&#114;&#104;&#101;&#101;&#64;&#97;&#99;&#111;&#109;&#97;&#46;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The Gene Ontology (GO) project is a collaboration among model organism databases to describe gene products from all organisms using a consistent and computable language. GO produces sets of explicitly defined, structured vocabularies that describe biological processes, molecular functions and cellular components of gene products in both a computer- and human-readable manner. Here we describe key aspects of GO, which, when overlooked, can cause erroneous results, and address how these pitfalls can be avoided.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">7</span>;<span class="pagerange">509-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18475267">18475267</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg2363">10.1038/nrg2363</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Male-pattern baldness susceptibility locus at 20p11.</h4>
<p class="authors"><span class="author">Richards JB</span>, <span class="author">Yuan X</span>, <span class="author">Geller F</span>, <span class="author">Waterworth D</span>, <span class="author">Bataille V</span>, <span class="author">Glass D</span>, <span class="author">Song K</span>, <span class="author">Waeber G</span>, <span class="author">Vollenweider P</span>, <span class="author">Aben KK</span>, <span class="author">Kiemeney LA</span>, <span class="author">Walters B</span>, <span class="author">Soranzo N</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Kong A</span>, <span class="author">Rafnar T</span>, <span class="author">Deloukas P</span>, <span class="author">Sulem P</span>, <span class="author">Stefansson H</span>, <span class="author">Stefansson K</span>, <span class="author">Spector TD</span> and <span class="author">Mooser V</span></p>
<p class="affiliation">Department of Twin Research and Genetic Epidemiology, King&#39;s College London, London SE1 7EH, UK.</p><p class="abstract">We conducted a genome-wide association study for androgenic alopecia in 1,125 men and identified a newly associated locus at chromosome 20p11.22, confirmed in three independent cohorts (n = 1,650; OR = 1.60, P = 1.1 x 10(-14) for rs1160312). The one man in seven who harbors risk alleles at both 20p11.22 and AR (encoding the androgen receptor) has a sevenfold-increased odds of androgenic alopecia (OR = 7.12, P = 3.7 x 10(-15)).</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">11</span>;<span class="pagerange">1282-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18849991">18849991</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672151">2672151</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.255">10.1038/ng.255</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase 2007.</h4>
<p class="authors"><span class="author">Rogers A</span>, <span class="author">Antoshechkin I</span>, <span class="author">Bieri T</span>, <span class="author">Blasiar D</span>, <span class="author">Bastiani C</span>, <span class="author">Canaran P</span>, <span class="author">Chan J</span>, <span class="author">Chen WJ</span>, <span class="author">Davis P</span>, <span class="author">Fernandes J</span>, <span class="author">Fiedler TJ</span>, <span class="author">Han M</span>, <span class="author">Harris TW</span>, <span class="author">Kishore R</span>, <span class="author">Lee R</span>, <span class="author">McKay S</span>, <span class="author">M&uuml;ller HM</span>, <span class="author">Nakamura C</span>, <span class="author">Ozersky P</span>, <span class="author">Petcherski A</span>, <span class="author">Schindelman G</span>, <span class="author">Schwarz EM</span>, <span class="author">Spooner W</span>, <span class="author">Tuli MA</span>, <span class="author">Van Auken K</span>, <span class="author">Wang D</span>, <span class="author">Wang X</span>, <span class="author">Williams G</span>, <span class="author">Yook K</span>, <span class="author">Durbin R</span>, <span class="author">Stein LD</span>, <span class="author">Spieth J</span> and <span class="author">Sternberg PW</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">WormBase (www.wormbase.org) is the major publicly available database of information about Caenorhabditis elegans, an important system for basic biological and biomedical research. Derived from the initial ACeDB database of C. elegans genetic and sequence information, WormBase now includes the genomic, anatomical and functional information about C. elegans, other Caenorhabditis species and other nematodes. As such, it is a crucial resource not only for C. elegans biologists but the larger biomedical and bioinformatics communities. Coverage of core areas of C. elegans biology will allow the biomedical community to make full use of the results of intensive molecular genetic analysis and functional genomic studies of this organism. Improved search and display tools, wider cross-species comparisons and extended ontologies are some of the features that will help scientists extend their research and take advantage of other nematode species genome sequences.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D612-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17991679" rel="external">17991679</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238927" rel="external">2238927</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm975" rel="external">10.1093/nar/gkm975</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maximum-likelihood estimation of site-specific mutation rates in human mitochondrial DNA from partial phylogenetic classification.</h4>
<p class="authors"><span class="author">Rosset S</span>, <span class="author">Wells RS</span>, <span class="author">Soria-Hernanz DF</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Royyuru AK</span>, <span class="author">Behar DM</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">Department of Statistics and Operations Research, Tel Aviv University, Tel Aviv, Israel. <a href="mailto:%73%61%68%61%72%6F%6E%40%70%6F%73%74.%74%61%75.%61%63.%69%6C">&#115;&#97;&#104;&#97;&#114;&#111;&#110;&#64;&#112;&#111;&#115;&#116;&#46;&#116;&#97;&#117;&#46;&#97;&#99;&#46;&#105;&#108;</a></p><p class="abstract">The mitochondrial DNA hypervariable segment I (HVS-I) is widely used in studies of human evolutionary genetics, and therefore accurate estimates of mutation rates among nucleotide sites in this region are essential. We have developed a novel maximum-likelihood methodology for estimating site-specific mutation rates from partial phylogenetic information, such as haplogroup association. The resulting estimation problem is a generalized linear model, with a nonstandard link function. We develop inference and bias correction tools for our estimates and a hypothesis-testing approach for site independence. We demonstrate our methodology using 16,609 HVS-I samples from the Genographic Project. Our results suggest that mutation rates among nucleotide sites in HVS-I are highly variable. The 16,400-16,500 region exhibits significantly lower rates compared to other regions, suggesting potential functional constraints. Several loci identified in the literature as possible termination-associated sequences (TAS) do not yield statistically slower rates than the rest of HVS-I, casting doubt on their functional importance. Our tests do not reject the null hypothesis of independent mutation rates among nucleotide sites, supporting the use of site-independence assumption for analyzing HVS-I. Potential extensions of our methodology include its application to estimation of mutation rates in other genetic regions, like Y chromosome short tandem repeats.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">180</span>;<span class="number">3</span>;<span class="pagerange">1511-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18791242">18791242</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2581953">2581953</a>; DOI: <a href="http://dx.doi.org/10.1534%2Fgenetics.108.091116">10.1534/genetics.108.091116</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TreeFam: 2008 Update.</h4>
<p class="authors"><span class="author">Ruan J</span>, <span class="author">Li H</span>, <span class="author">Chen Z</span>, <span class="author">Coghlan A</span>, <span class="author">Coin LJ</span>, <span class="author">Guo Y</span>, <span class="author">H&eacute;rich&eacute; JK</span>, <span class="author">Hu Y</span>, <span class="author">Kristiansen K</span>, <span class="author">Li R</span>, <span class="author">Liu T</span>, <span class="author">Moses A</span>, <span class="author">Qin J</span>, <span class="author">Vang S</span>, <span class="author">Vilella AJ</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Bolund L</span>, <span class="author">Wang J</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Beijing Institute of Genomics of the Chinese Academy of Sciences, Beijing Genomics Institute, Beijing 101300, China.</p><p class="abstract">TreeFam (http://www.treefam.org) was developed to provide curated phylogenetic trees for all animal gene families, as well as orthologue and paralogue assignments. Release 4.0 of TreeFam contains curated trees for 1314 families and automatically generated trees for another 14,351 families. We have expanded TreeFam to include 25 fully sequenced animal genomes, as well as four genomes from plant and fungal outgroup species. We have also introduced more accurate approaches for automatically grouping genes into families, for building phylogenetic trees, and for inferring orthologues and paralogues. The user interface for viewing phylogenetic trees and family information has been improved. Furthermore, a new perl API lets users easily extract data from the TreeFam mysql database.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D735-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18056084">18056084</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238856">2238856</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm1005">10.1093/nar/gkm1005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of NMDA receptor cytoplasmic interaction domains: implications for organisation of synaptic signalling complexes.</h4>
<p class="authors"><span class="author">Ryan TJ</span>, <span class="author">Emes RD</span>, <span class="author">Grant SG</span> and <span class="author">Komiyama NH</span></p>
<p class="affiliation">Genes to Cognition Program, Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%74%72%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#114;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Glutamate gated postsynaptic receptors in the central nervous system (CNS) are essential for environmentally stimulated behaviours including learning and memory in both invertebrates and vertebrates. Though their genetics, biochemistry, physiology, and role in behaviour have been intensely studied in vitro and in vivo, their molecular evolution and structural aspects remain poorly understood. To understand how these receptors have evolved different physiological requirements we have investigated the molecular evolution of glutamate gated receptors and ion channels, in particular the N-methyl-D-aspartate (NMDA) receptor, which is essential for higher cognitive function. Studies of rodent NMDA receptors show that the C-terminal intracellular domain forms a signalling complex with enzymes and scaffold proteins, which is important for neuronal and behavioural plasticity</p><p class="abstract"><strong>Results</strong>: The vertebrate NMDA receptor was found to have subunits with C-terminal domains up to 500 amino acids longer than invertebrates. This extension was specific to the NR2 subunit and occurred before the duplication and subsequent divergence of NR2 in the vertebrate lineage. The shorter invertebrate C-terminus lacked vertebrate protein interaction motifs involved with forming a signaling complex although the terminal PDZ interaction domain was conserved. The vertebrate NR2 C-terminal domain was predicted to be intrinsically disordered but with a conserved secondary structure.</p><p class="abstract"><strong>Conclusion</strong>: We highlight an evolutionary adaptation specific to vertebrate NMDA receptor NR2 subunits. Using in silico methods we find that evolution has shaped the NMDA receptor C-terminus into an unstructured but modular intracellular domain that parallels the expansion in complexity of an NMDA receptor signalling complex in the vertebrate lineage. We propose the NR2 C-terminus has evolved to be a natively unstructured yet flexible hub organising postsynaptic signalling. The evolution of the NR2 C-terminus and its associated signalling complex may contribute to species differences in behaviour and in particular cognitive function.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC neuroscience</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18197970" rel="external">18197970</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2257970" rel="external">2257970</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2202-9-6" rel="external">10.1186/1471-2202-9-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Annotation of mammalian primary microRNAs.</h4>
<p class="authors"><span class="author">Saini HK</span>, <span class="author">Enright AJ</span> and <span class="author">Griffiths-Jones S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%68%73%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#115;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: MicroRNAs (miRNAs) are important regulators of gene expression and have been implicated in development, differentiation and pathogenesis. Hundreds of miRNAs have been discovered in mammalian genomes. Approximately 50% of mammalian miRNAs are expressed from introns of protein-coding genes; the primary transcript (pri-miRNA) is therefore assumed to be the host transcript. However, very little is known about the structure of pri-miRNAs expressed from intergenic regions. Here we annotate transcript boundaries of miRNAs in human, mouse and rat genomes using various transcription features. The 5&#39; end of the pri-miRNA is predicted from transcription start sites, CpG islands and 5&#39; CAGE tags mapped in the upstream flanking region surrounding the precursor miRNA (pre-miRNA). The 3&#39; end of the pri-miRNA is predicted based on the mapping of polyA signals, and supported by cDNA/EST and ditags data. The predicted pri-miRNAs are also analyzed for promoter and insulator-associated regulatory regions.</p><p class="abstract"><strong>Results</strong>: We define sets of conserved and non-conserved human, mouse and rat pre-miRNAs using bidirectional BLAST and synteny analysis. Transcription features in their flanking regions are used to demarcate the 5&#39; and 3&#39; boundaries of the pri-miRNAs. The lengths and boundaries of primary transcripts are highly conserved between orthologous miRNAs. A significant fraction of pri-miRNAs have lengths between 1 and 10 kb, with very few introns. We annotate a total of 59 pri-miRNA structures, which include 82 pre-miRNAs. 36 pri-miRNAs are conserved in all 3 species. In total, 18 of the confidently annotated transcripts express more than one pre-miRNA. The upstream regions of 54% of the predicted pri-miRNAs are found to be associated with promoter and insulator regulatory sequences.</p><p class="abstract"><strong>Conclusion</strong>: Little is known about the primary transcripts of intergenic miRNAs. Using comparative data, we are able to identify the boundaries of a significant proportion of human, mouse and rat pri-miRNAs. We confidently predict the transcripts including a total of 77, 58 and 47 human, mouse and rat pre-miRNAs respectively. Our computational annotations provide a basis for subsequent experimental validation of predicted pri-miRNAs.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">564</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19038026" rel="external">19038026</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2632650" rel="external">2632650</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-564" rel="external">10.1186/1471-2164-9-564</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pfam 10 years on: 10,000 families and still growing.</h4>
<p class="authors"><span class="author">Sammut SJ</span>, <span class="author">Finn RD</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton Hall, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Classifications of proteins into groups of related sequences are in some respects like a periodic table for biology, allowing us to understand the underlying molecular biology of any organism. Pfam is a large collection of protein domains and families. Its scientific goal is to provide a complete and accurate classification of protein families and domains. The next release of the database will contain over 10,000 entries, which leads us to reflect on how far we are from completing this work. Currently Pfam matches 72% of known protein sequences, but for proteins with known structure Pfam matches 95%, which we believe represents the likely upper bound. Based on our analysis a further 28,000 families would be required to achieve this level of coverage for the current sequence database. We also show that as more sequences are added to the sequence databases the fraction of sequences that Pfam matches is reduced, suggesting that continued addition of new families is essential to maintain its relevance.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">3</span>;<span class="pagerange">210-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18344544">18344544</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbib%2Fbbn010">10.1093/bib/bbn010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mendelian randomisation studies of type 2 diabetes: future prospects.</h4>
<p class="authors"><span class="author">Sandhu MS</span>, <span class="author">Debenham SL</span>, <span class="author">Barroso I</span> and <span class="author">Loos RJ</span></p>
<p class="affiliation">MRC Epidemiology Unit, Strangeways Research Laboratory, Cambridge, UK. <a href="mailto:%6D%61%6E%6A.%73%61%6E%64%68%75%40%73%72%6C.%63%61%6D.%61%63.%75%6B">&#109;&#97;&#110;&#106;&#46;&#115;&#97;&#110;&#100;&#104;&#117;&#64;&#115;&#114;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2008</span>;<span class="volume">51</span>;<span class="number">2</span>;<span class="pagerange">211-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18094955">18094955</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2681076">2681076</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-007-0903-x">10.1007/s00125-007-0903-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">LDL-cholesterol concentrations: a genome-wide association study.</h4>
<p class="authors"><span class="author">Sandhu MS</span>, <span class="author">Waterworth DM</span>, <span class="author">Debenham SL</span>, <span class="author">Wheeler E</span>, <span class="author">Papadakis K</span>, <span class="author">Zhao JH</span>, <span class="author">Song K</span>, <span class="author">Yuan X</span>, <span class="author">Johnson T</span>, <span class="author">Ashford S</span>, <span class="author">Inouye M</span>, <span class="author">Luben R</span>, <span class="author">Sims M</span>, <span class="author">Hadley D</span>, <span class="author">McArdle W</span>, <span class="author">Barter P</span>, <span class="author">Kes&auml;niemi YA</span>, <span class="author">Mahley RW</span>, <span class="author">McPherson R</span>, <span class="author">Grundy SM</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Bingham SA</span>, <span class="author">Khaw KT</span>, <span class="author">Loos RJ</span>, <span class="author">Waeber G</span>, <span class="author">Barroso I</span>, <span class="author">Strachan DP</span>, <span class="author">Deloukas P</span>, <span class="author">Vollenweider P</span>, <span class="author">Wareham NJ</span> and <span class="author">Mooser V</span></p>
<p class="affiliation">Department of Public Health and Primary Care, Strangeways Research Laboratory, University of Cambridge, Cambridge, UK. <a href="mailto:%6D%61%6E%6A.%73%61%6E%64%68%75%40%73%72%6C.%63%61%6D.%61%63.%75%6B">&#109;&#97;&#110;&#106;&#46;&#115;&#97;&#110;&#100;&#104;&#117;&#64;&#115;&#114;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: LDL cholesterol has a causal role in the development of cardiovascular disease. Improved understanding of the biological mechanisms that underlie the metabolism and regulation of LDL cholesterol might help to identify novel therapeutic targets. We therefore did a genome-wide association study of LDL-cholesterol concentrations.</p><p class="abstract"><strong>Methods</strong>: We used genome-wide association data from up to 11,685 participants with measures of circulating LDL-cholesterol concentrations across five studies, including data for 293 461 autosomal single nucleotide polymorphisms (SNPs) with a minor allele frequency of 5% or more that passed our quality control criteria. We also used data from a second genome-wide array in up to 4337 participants from three of these five studies, with data for 290,140 SNPs. We did replication studies in two independent populations consisting of up to 4979 participants. Statistical approaches, including meta-analysis and linkage disequilibrium plots, were used to refine association signals; we analysed pooled data from all seven populations to determine the effect of each SNP on variations in circulating LDL-cholesterol concentrations.</p><p class="abstract"><strong>Findings</strong>: In our initial scan, we found two SNPs (rs599839 [p=1.7x10(-15)] and rs4970834 [p=3.0x10(-11)]) that showed genome-wide statistical association with LDL cholesterol at chromosomal locus 1p13.3. The second genome screen found a third statistically associated SNP at the same locus (rs646776 [p=4.3x10(-9)]). Meta-analysis of data from all studies showed an association of SNPs rs599839 (combined p=1.2x10(-33)) and rs646776 (p=4.8x10(-20)) with LDL-cholesterol concentrations. SNPs rs599839 and rs646776 both explained around 1% of the variation in circulating LDL-cholesterol concentrations and were associated with about 15% of an SD change in LDL cholesterol per allele, assuming an SD of 1 mmol/L.</p><p class="abstract"><strong>Interpretation</strong>: We found evidence for a novel locus for LDL cholesterol on chromosome 1p13.3. These results potentially provide insight into the biological mechanisms that underlie the regulation of LDL cholesterol and might help in the discovery of novel therapeutic targets for cardiovascular disease.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0701863</span>, <span class="grant">MC_QA137934</span>, <span class="grant">MC_U105630924</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">Lancet</span>&nbsp;<span class="year">2008</span>;<span class="volume">371</span>;<span class="number">9611</span>;<span class="pagerange">483-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18262040">18262040</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2292820">2292820</a>; DOI: <a href="http://dx.doi.org/10.1016%2FS0140-6736%2808%2960208-1">10.1016/S0140-6736(08)60208-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic association analyses of non-synonymous single nucleotide polymorphisms in diabetic nephropathy.</h4>
<p class="authors"><span class="author">Savage DA</span>, <span class="author">Patterson CC</span>, <span class="author">Deloukas P</span>, <span class="author">Whittaker P</span>, <span class="author">McKnight AJ</span>, <span class="author">Morrison J</span>, <span class="author">Boulton AJ</span>, <span class="author">Demaine AG</span>, <span class="author">Marshall SM</span>, <span class="author">Millward BA</span>, <span class="author">Thomas SM</span>, <span class="author">Viberti GC</span>, <span class="author">Walker JD</span>, <span class="author">Sadlier D</span>, <span class="author">Maxwell AP</span> and <span class="author">Bain SC</span></p>
<p class="affiliation">Nephrology Research Laboratory, Queen&#39;s University, Belfast, BT9 7AB, Northern Ireland, UK. <a href="mailto:%64.%73%61%76%61%67%65%40%71%75%62.%61%63.%75%6B">&#100;&#46;&#115;&#97;&#118;&#97;&#103;&#101;&#64;&#113;&#117;&#98;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Diabetic nephropathy, characterised by persistent proteinuria, hypertension and progressive kidney failure, affects a subset of susceptible individuals with diabetes. It is also a leading cause of end-stage renal disease (ESRD). Non-synonymous (ns) single nucleotide polymorphisms (SNPs) have been reported to contribute to genetic susceptibility in both monogenic disorders and common complex diseases. The objective of this study was to investigate whether nsSNPs are involved in susceptibility to diabetic nephropathy using a case-control design.</p><p class="abstract"><strong>Methods</strong>: White type 1 diabetic patients with (cases) and without (controls) nephropathy from eight centres in the UK and Ireland were genotyped for a selected subset of nsSNPs using Illumina&#39;s GoldenGate BeadArray assay. A chi (2) test for trend, stratified by centre, was used to assess differences in genotype distribution between cases and controls. Genomic control was used to adjust for possible inflation of test statistics, and the False Discovery Rate method was used to account for multiple testing.</p><p class="abstract"><strong>Results</strong>: We assessed 1,111 nsSNPs for association with diabetic nephropathy in 1,711 individuals with type 1 diabetes (894 cases, 817 controls). A number of SNPs demonstrated a significant difference in genotype distribution between groups before but not after correction for multiple testing. Furthermore, neither subgroup analysis (diabetic nephropathy with ESRD or diabetic nephropathy without ESRD) nor stratification by duration of diabetes revealed any significant differences between groups.</p><p class="abstract">The nsSNPs investigated in this study do not appear to contribute significantly to the development of diabetic nephropathy in patients with type 1 diabetes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2008</span>;<span class="volume">51</span>;<span class="number">11</span>;<span class="pagerange">1998-2002</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18773191" rel="external">18773191</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687720" rel="external">2687720</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-008-1142-5" rel="external">10.1007/s00125-008-1142-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New case of interstitial deletion 12(q15-q21.2) in a girl with facial dysmorphism and mental retardation.</h4>
<p class="authors"><span class="author">Schluth C</span>, <span class="author">Gesny R</span>, <span class="author">Borck G</span>, <span class="author">Redon R</span>, <span class="author">Abadie V</span>, <span class="author">Kleinfinger P</span>, <span class="author">Munnich A</span>, <span class="author">Lyonnet S</span> and <span class="author">Colleaux L</span></p>
<p class="affiliation">D&eacute;partement de G&eacute;n&eacute;tique, H&ocirc;pital Necker Enfants Malades, Paris, France.</p><p class="abstract">Interstitial deletions of the long arm of chromosome 12 are rare rearrangements with only 15 cases reported in the literature. The phenotype may include facial dysmorphism, developmental delay, ectodermal abnormalities, cardiac and renal malformations depending on breakpoints&#39; position. Here, we describe a third case of 12(q15-q21.2) deletion ascertained through CGH-array analyses and provide a 5-year follow-up. The patient presented with pre- and postnatal growth retardation, congenital heart defect, developmental delay, and facial dysmorphism changing with age, underlining the importance of long-term follow-up. We compared this new case with previous observations of 12q deletions in order to propose phenotype-karyotype correlations.</p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2008</span>;<span class="volume">146A</span>;<span class="number">1</span>;<span class="pagerange">93-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18076123">18076123</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.31869">10.1002/ajmg.a.31869</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Repeated replication and a prospective meta-analysis of the association between chromosome 9p21.3 and coronary artery disease.</h4>
<p class="authors"><span class="author">Schunkert H</span>, <span class="author">G&ouml;tz A</span>, <span class="author">Braund P</span>, <span class="author">McGinnis R</span>, <span class="author">Tregouet DA</span>, <span class="author">Mangino M</span>, <span class="author">Linsel-Nitschke P</span>, <span class="author">Cambien F</span>, <span class="author">Hengstenberg C</span>, <span class="author">Stark K</span>, <span class="author">Blankenberg S</span>, <span class="author">Tiret L</span>, <span class="author">Ducimetiere P</span>, <span class="author">Keniry A</span>, <span class="author">Ghori MJ</span>, <span class="author">Schreiber S</span>, <span class="author">El Mokhtari NE</span>, <span class="author">Hall AS</span>, <span class="author">Dixon RJ</span>, <span class="author">Goodall AH</span>, <span class="author">Liptau H</span>, <span class="author">Pollard H</span>, <span class="author">Schwarz DF</span>, <span class="author">Hothorn LA</span>, <span class="author">Wichmann HE</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Fischer M</span>, <span class="author">Meisinger C</span>, <span class="author">Ouwehand W</span>, <span class="author">Deloukas P</span>, <span class="author">Thompson JR</span>, <span class="author">Erdmann J</span>, <span class="author">Ziegler A</span>, <span class="author">Samani NJ</span> and <span class="author">Cardiogenics Consortium</span></p>
<p class="affiliation">Medizinische Klinik II, Universit&auml;t zu L&uuml;beck, L&uuml;beck, Germany. <a href="mailto:%68%65%72%69%62%65%72%74.%73%63%68%75%6E%6B%65%72%74%40%69%6E%6E%65%72%65%32.%75%6E%69-%6C%75%65%62%65%63%6B.%64%65">&#104;&#101;&#114;&#105;&#98;&#101;&#114;&#116;&#46;&#115;&#99;&#104;&#117;&#110;&#107;&#101;&#114;&#116;&#64;&#105;&#110;&#110;&#101;&#114;&#101;&#50;&#46;&#117;&#110;&#105;&#45;&#108;&#117;&#101;&#98;&#101;&#99;&#107;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: Recently, genome-wide association studies identified variants on chromosome 9p21.3 as affecting the risk of coronary artery disease (CAD). We investigated the association of this locus with CAD in 7 case-control studies and undertook a meta-analysis.</p><p class="abstract">A single-nucleotide polymorphism (SNP), rs1333049, representing the 9p21.3 locus, was genotyped in 7 case-control studies involving a total of 4645 patients with myocardial infarction or CAD and 5177 controls. The mode of inheritance was determined. In addition, in 5 of the 7 studies, we genotyped 3 additional SNPs to assess a risk-associated haplotype (ACAC). Finally, a meta-analysis of the present data and previously published samples was conducted. A limited fine mapping of the locus was performed. The risk allele (C) of the lead SNP, rs1333049, was uniformly associated with CAD in each study (P&lt;0.05). In a pooled analysis, the odds ratio per copy of the risk allele was 1.29 (95% confidence interval, 1.22 to 1.37; P=0.0001). Haplotype analysis further suggested that this effect was not homogeneous across the haplotypic background (test for interaction, P=0.0079). An autosomal-additive mode of inheritance best explained the underlying association. The meta-analysis of the rs1333049 SNP in 12,004 cases and 28,949 controls increased the overall level of evidence for association with CAD to P=6.04x10(-10) (odds ratio, 1.24; 95% confidence interval, 1.20 to 1.29). Genotyping of 31 additional SNPs in the region identified several with a highly significant association with CAD, but none had predictive information beyond that of the rs1333049 SNP.</p><p class="abstract"><strong>Conclusions</strong>: This broad replication provides unprecedented evidence for association between genetic variants at chromosome 9p21.3 and risk of CAD.</p>
<p class="grants">Funded by: British Heart Foundation</p>
<p class="publication"><span class="title">Circulation</span>&nbsp;<span class="year">2008</span>;<span class="volume">117</span>;<span class="number">13</span>;<span class="pagerange">1675-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18362232">18362232</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2689930">2689930</a>; DOI: <a href="http://dx.doi.org/10.1161%2FCIRCULATIONAHA.107.730614">10.1161/CIRCULATIONAHA.107.730614</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Protein interactions in human genetic diseases.</h4>
<p class="authors"><span class="author">Schuster-B&ouml;ckler B</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK. <a href="mailto:%62%73%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#115;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present a novel method that combines protein structure information with protein interaction data to identify residues that form part of an interaction interface. Our prediction method can retrieve interaction hotspots with an accuracy of 60% (at a 20% false positive rate). The method was applied to all mutations in the Online Mendelian Inheritance in Man (OMIM) database, predicting 1,428 mutations to be related to an interaction defect. Combining predicted and hand-curated sets, we discuss how mutations affect protein interactions in general.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">R9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18199329">18199329</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2395246">2395246</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-1-r9">10.1186/gb-2008-9-1-r9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Opportunity knocks</h4>
<p class="authors"><span class="author">Seth-Smith H.</span>, <span class="author">Walker A.</span></p>
<p class="publication"><span class="title">Nat Rev Microbiol.</span> <span class="year">2008</span>;<span class="volume">6</span>;<span class="pagerange">652-3</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Poultry Existence</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p>
<p class="publication"><span class="title">Nat Rev Microbiol</span> <span class="year">2008</span>;<span class="volume">6</span>;<span class="pagerange">8</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide transcriptional changes induced by phagocytosis or growth on bacteria in Dictyostelium.</h4>
<p class="authors"><span class="author">Sillo A</span>, <span class="author">Bloomfield G</span>, <span class="author">Balest A</span>, <span class="author">Balbo A</span>, <span class="author">Pergolizzi B</span>, <span class="author">Peracino B</span>, <span class="author">Skelton J</span>, <span class="author">Ivens A</span> and <span class="author">Bozzaro S</span></p>
<p class="affiliation">Department of Clinical and Biological Sciences, University of Turin, Ospedale S, Luigi, 10043 Orbassano, Torino, Italy. <a href="mailto:%61%6C%65%73%73%69%6F.%73%69%6C%6C%6F%40%75%6E%69%74%6F.%69%74">&#97;&#108;&#101;&#115;&#115;&#105;&#111;&#46;&#115;&#105;&#108;&#108;&#111;&#64;&#117;&#110;&#105;&#116;&#111;&#46;&#105;&#116;</a></p><p class="abstract"><strong>Background</strong>: Phagocytosis plays a major role in the defense of higher organisms against microbial infection and provides also the basis for antigen processing in the immune response. Cells of the model organism Dictyostelium are professional phagocytes that exploit phagocytosis of bacteria as the preferred way to ingest food, besides killing pathogens. We have investigated Dictyostelium differential gene expression during phagocytosis of non-pathogenic bacteria, using DNA microarrays, in order to identify molecular functions and novel genes involved in phagocytosis.</p><p class="abstract"><strong>Results</strong>: The gene expression profiles of cells incubated for a brief time with bacteria were compared with cells either incubated in axenic medium or growing on bacteria. Transcriptional changes during exponential growth in axenic medium or on bacteria were also compared. We recognized 443 and 59 genes that are differentially regulated by phagocytosis or by the different growth conditions (growth on bacteria vs. axenic medium), respectively, and 102 genes regulated by both processes. Roughly one third of the genes are up-regulated compared to macropinocytosis and axenic growth. Functional annotation of differentially regulated genes with different tools revealed that phagocytosis induces profound changes in carbohydrate, amino acid and lipid metabolism, and in cytoskeletal components. Genes regulating translation and mitochondrial biogenesis are mostly up-regulated. Genes involved in sterol biosynthesis are selectively up-regulated, suggesting a shift in membrane lipid composition linked to phagocytosis. Very few changes were detected in genes required for vesicle fission/fusion, indicating that the intracellular traffic machinery is mostly in common between phagocytosis and macropinocytosis. A few putative receptors, including GPCR family 3 proteins, scaffolding and adhesion proteins, components of signal transduction and transcription factors have been identified, which could be part of a signalling complex regulating phagocytosis and adaptational downstream responses.</p><p class="abstract"><strong>Conclusion</strong>: The results highlight differences between phagocytosis and macropinocytosis, and provide the basis for targeted functional analysis of new candidate genes and for comparison studies with transcriptomes during infection with pathogenic bacteria.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">291</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18559084" rel="external">18559084</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2443395" rel="external">2443395</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-291" rel="external">10.1186/1471-2164-9-291</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Polymorphisms in the estrogen receptor 1 and vitamin C and matrix metalloproteinase gene families are associated with susceptibility to lymphoma.</h4>
<p class="authors"><span class="author">Skibola CF</span>, <span class="author">Bracci PM</span>, <span class="author">Halperin E</span>, <span class="author">Nieters A</span>, <span class="author">Hubbard A</span>, <span class="author">Paynter RA</span>, <span class="author">Skibola DR</span>, <span class="author">Agana L</span>, <span class="author">Becker N</span>, <span class="author">Tressler P</span>, <span class="author">Forrest MS</span>, <span class="author">Sankararaman S</span>, <span class="author">Conde L</span>, <span class="author">Holly EA</span> and <span class="author">Smith MT</span></p>
<p class="affiliation">School of Public Health, Division of Environmental Health Sciences, University of California Berkeley, Berkeley, California, USA. <a href="mailto:%63%68%72%69%73%66%73%40%62%65%72%6B%65%6C%65%79.%65%64%75">&#99;&#104;&#114;&#105;&#115;&#102;&#115;&#64;&#98;&#101;&#114;&#107;&#101;&#108;&#101;&#121;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Non-Hodgkin lymphoma (NHL) is the fifth most common cancer in the U.S. and few causes have been identified. Genetic association studies may help identify environmental risk factors and enhance our understanding of disease mechanisms.</p><p class="abstract">768 coding and haplotype tagging SNPs in 146 genes were examined using Illumina GoldenGate technology in a large population-based case-control study of NHL in the San Francisco Bay Area (1,292 cases 1,375 controls are included here). Statistical analyses were restricted to HIV- participants of white non-Hispanic origin. Genes involved in steroidogenesis, immune function, cell signaling, sunlight exposure, xenobiotic metabolism/oxidative stress, energy balance, and uptake and metabolism of cholesterol, folate and vitamin C were investigated. Sixteen SNPs in eight pathways and nine haplotypes were associated with NHL after correction for multiple testing at the adjusted q&lt;0.10 level. Eight SNPs were tested in an independent case-control study of lymphoma in Germany (494 NHL cases and 494 matched controls). Novel associations with common variants in estrogen receptor 1 (ESR1) and in the vitamin C receptor and matrix metalloproteinase gene families were observed. Four ESR1 SNPs were associated with follicular lymphoma (FL) in the U.S. study, with rs3020314 remaining associated with reduced risk of FL after multiple testing adjustments [odds ratio (OR) = 0.42, 95% confidence interval (CI) = 0.23-0.77) and replication in the German study (OR = 0.24, 95% CI = 0.06-0.94). Several SNPs and haplotypes in the matrix metalloproteinase-3 (MMP3) and MMP9 genes and in the vitamin C receptor genes, solute carrier family 23 member 1 (SLC23A1) and SLC23A2, showed associations with NHL risk.</p><p class="abstract">Our findings suggest a role for estrogen, vitamin C and matrix metalloproteinases in the pathogenesis of NHL that will require further validation.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA104862</span>, <span class="grant">CA122663</span>, <span class="grant">CA45614</span>, <span class="grant">CA87014</span>, <span class="grant">CA89745</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">7</span>;<span class="pagerange">e2816</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18636124" rel="external">18636124</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2474696" rel="external">2474696</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0002816" rel="external">10.1371/journal.pone.0002816</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Computational prediction of protein-protein interactions.</h4>
<p class="authors"><span class="author">Skrabanek L</span>, <span class="author">Saini HK</span>, <span class="author">Bader GD</span> and <span class="author">Enright AJ</span></p>
<p class="affiliation">Department of Physiology and Biophysics and Institute for Computational Biomedicine, Weill Medical College of Cornell University, 1300 York Avenue, New York, NY 10021, USA.</p><p class="abstract">Recently a number of computational approaches have been developed for the prediction of protein-protein interactions. Complete genome sequencing projects have provided the vast amount of information needed for these analyses. These methods utilize the structural, genomic, and biological context of proteins and genes in complete genomes to predict protein interaction networks and functional linkages between proteins. Given that experimental techniques remain expensive, time-consuming, and labor-intensive, these methods represent an important advance in proteomics. Some of these approaches utilize sequence data alone to predict interactions, while others combine multiple computational and experimental datasets to accurately build protein interaction maps for complete genomes. These methods represent a complementary approach to current high-throughput projects whose aim is to delineate protein interaction maps in complete genomes. We will describe a number of computational protocols for protein interaction prediction based on the structural, genomic, and biological context of proteins in complete genomes, and detail methods for protein interaction network visualization and analysis.</p>
<p class="publication"><span class="title">Molecular biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">38</span>;<span class="number">1</span>;<span class="pagerange">1-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18095187" rel="external">18095187</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs12033-007-0069-2" rel="external">10.1007/s12033-007-0069-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The effect of temperature on Natural Antisense Transcript (NAT) expression in Aspergillus flavus.</h4>
<p class="authors"><span class="author">Smith CA</span>, <span class="author">Robertson D</span>, <span class="author">Yates B</span>, <span class="author">Nielsen DM</span>, <span class="author">Brown D</span>, <span class="author">Dean RA</span> and <span class="author">Payne GA</span></p>
<p class="affiliation">Department of Entomology and Plant Pathology, Oklahoma State University, Stillwater, OK 74078, USA. <a href="mailto:%63%61%72%72%69%65.%61.%73%6D%69%74%68%40%6F%6B%73%74%61%74%65.%65%64%75">&#99;&#97;&#114;&#114;&#105;&#101;&#46;&#97;&#46;&#115;&#109;&#105;&#116;&#104;&#64;&#111;&#107;&#115;&#116;&#97;&#116;&#101;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Naturally occurring Antisense Transcripts (NATs) compose an emerging group of regulatory RNAs. These regulatory elements appear in all organisms examined, but little is known about global expression of NATs in fungi. Analysis of currently available EST sequences suggests that 352 cis NATs are present in Aspergillus flavus. An Affymetrix GeneChip microarray containing probes for these cis NATs, as well as all predicted genes in A. flavus, allowed a whole genome expression analysis of these elements in response to two ecologically important temperatures for the fungus. RNA expression analysis showed that 32 NATs and 2,709 genes were differentially expressed between 37 degrees C, the optimum temperature for growth, and 28 degrees C, the conducive temperature for the biosynthesis of aflatoxin (AF) and many other secondary metabolites. These NATs correspond to sense genes with diverse functions including transcription initiation, carbohydrate processing and binding, temperature sensitive morphogenesis, and secondary metabolism. This is the first report of a whole genome transcriptional analysis of NAT expression in a fungus.</p>
<p class="publication"><span class="title">Current genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">54</span>;<span class="number">5</span>;<span class="pagerange">241-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18813928" rel="external">18813928</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00294-008-0215-9" rel="external">10.1007/s00294-008-0215-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Understanding how morphogens work.</h4>
<p class="authors"><span class="author">Smith JC</span>, <span class="author">Hagemann A</span>, <span class="author">Saka Y</span> and <span class="author">Williams PH</span></p>
<p class="affiliation">Wellcome Trust/CR-UK Gurdon Institute, Department of Zoology, University of Cambridge, Tennis Court Road, Cambridge CB2 1QN, UK. <a href="mailto:%6A%69%6D%40%67%75%72%64%6F%6E.%63%61%6D.%61%63.%75%6B">&#106;&#105;&#109;&#64;&#103;&#117;&#114;&#100;&#111;&#110;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In this article, we describe the mechanisms by which morphogens in the Xenopus embryo exert their long-range effects. Our results are consistent with the idea that signalling molecules such as activin and the nodal-related proteins traverse responding tissue not by transcytosis or by cytonemes but by movement through the extracellular space. We suggest, however, that additional experiments, involving real-time imaging of morphogens, are required for a real understanding of what influences signalling range and the shape of a morphogen gradient.</p>
<p class="publication"><span class="title">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span>&nbsp;<span class="year">2008</span>;<span class="volume">363</span>;<span class="number">1495</span>;<span class="pagerange">1387-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18198154" rel="external">18198154</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2610127" rel="external">2610127</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frstb.2007.2256" rel="external">10.1098/rstb.2007.2256</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Robust, persistent transgene expression in human embryonic stem cells is achieved with AAVS1-targeted integration.</h4>
<p class="authors"><span class="author">Smith JR</span>, <span class="author">Maguire S</span>, <span class="author">Davis LA</span>, <span class="author">Alexander M</span>, <span class="author">Yang F</span>, <span class="author">Chandran S</span>, <span class="author">ffrench-Constant C</span> and <span class="author">Pedersen RA</span></p>
<p class="affiliation">Department of Surgery, Cambridge Institute for Medical Research, Wellcome Trust/MRC Building, Addenbrooke&#39;s Hospital, Hills Road, Cambridge CB2 OXY, United Kingdom. <a href="mailto:%6A%72%73%36%38%40%63%61%6D.%61%63.%75%6B">&#106;&#114;&#115;&#54;&#56;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Silencing and variegated transgene expression are poorly understood problems that can interfere with gene function studies in human embryonic stem cells (hESCs). We show that transgene expression (enhanced green fluorescent protein [EGFP]) from random integration sites in hESCs is affected by variegation and silencing, with only half of hESCs expressing the transgene, which is gradually lost after withdrawal of selection and differentiation. We tested the hypothesis that a transgene integrated into the adeno-associated virus type 2 (AAV2) target region on chromosome 19, known as the AAVS1 locus, would maintain transgene expression in hESCs. When we used AAV2 technology to target the AAVS1 locus, 4.16% of hESC clones achieved AAVS1-targeted integration. Targeted clones expressed Oct-4, stage-specific embryonic antigen-3 (SSEA3), and Tra-1-60 and differentiated into all three primary germ layers. EGFP expression from the AAVS1 locus showed significantly reduced variegated expression when in selection, with 90% +/- 4% of cells expressing EGFP compared with 57% +/- 32% for randomly integrated controls, and reduced tendency to undergo silencing, with 86% +/- 7% hESCs expressing EGFP 25 days after withdrawal of selection compared with 39% +/- 31% for randomly integrated clones. In addition, quantitative polymerase chain reaction analysis of hESCs also indicated significantly higher levels of EGFP mRNA in AAVS1-targeted clones as compared with randomly integrated clones. Transgene expression from the AAVS1 locus was shown to be stable during hESC differentiation, with more than 90% of cells expressing EGFP after 15 days of differentiation, as compared with approximately 30% for randomly integrated clones. These results demonstrate the utility of transgene integration at the AAVS1 locus in hESCs and its potential clinical application.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300300</span>, <span class="grant">G0300723</span>, <span class="grant">G0600275</span></p>
<p class="publication"><span class="title">Stem cells (Dayton, Ohio)</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">2</span>;<span class="pagerange">496-504</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18024421">18024421</a>; DOI: <a href="http://dx.doi.org/10.1634%2Fstemcells.2007-0039">10.1634/stemcells.2007-0039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conservation of the H19 noncoding RNA and H19-IGF2 imprinting mechanism in therians.</h4>
<p class="authors"><span class="author">Smits G</span>, <span class="author">Mungall AJ</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Smith P</span>, <span class="author">Beury D</span>, <span class="author">Matthews L</span>, <span class="author">Rogers J</span>, <span class="author">Pask AJ</span>, <span class="author">Shaw G</span>, <span class="author">VandeBerg JL</span>, <span class="author">McCarrey JR</span>, <span class="author">SAVOIR Consortium</span>, <span class="author">Renfree MB</span>, <span class="author">Reik W</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Babraham Institute, Laboratory of Developmental Genetics and Imprinting, Cambridge CB22 3AT, UK.</p><p class="abstract">Comparisons between eutherians and marsupials suggest limited conservation of the molecular mechanisms that control genomic imprinting in mammals. We have studied the evolution of the imprinted IGF2-H19 locus in therians. Although marsupial orthologs of protein-coding exons were easily identified, the use of evolutionarily conserved regions and low-stringency Bl2seq comparisons was required to delineate a candidate H19 noncoding RNA sequence. The therian H19 orthologs show miR-675 and exon structure conservation, suggesting functional selection on both features. Transcription start site sequences and poly(A) signals are also conserved. As in eutherians, marsupial H19 is maternally expressed and paternal methylation upstream of the gene originates in the male germline, encompasses a CTCF insulator, and spreads somatically into the H19 gene. The conservation in all therians of the mechanism controlling imprinting of the IGF2-H19 locus suggests a sequential model of imprinting evolution.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0400154</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">8</span>;<span class="pagerange">971-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18587395">18587395</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.168">10.1038/ng.168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of a nonsynonymous variant of DAOA with visuospatial ability in a bipolar family sample.</h4>
<p class="authors"><span class="author">Soronen P</span>, <span class="author">Silander K</span>, <span class="author">Antila M</span>, <span class="author">Palo OM</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Kiesepp&auml; T</span>, <span class="author">Ellonen P</span>, <span class="author">Wedenoja J</span>, <span class="author">Turunen JA</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Hennah W</span>, <span class="author">L&ouml;nnqvist J</span>, <span class="author">Peltonen L</span>, <span class="author">Partonen T</span> and <span class="author">Paunio T</span></p>
<p class="affiliation">Department of Molecular Medicine, National Public Health Institute, Helsinki, Finland.</p><p class="abstract"><strong>Background</strong>: Bipolar disorder and schizophrenia are hypothesized to share some genetic background.</p><p class="abstract"><strong>Methods</strong>: In a two-phase study, we evaluated the effect of five promising candidate genes for psychotic disorders, DAOA, COMT, DTNBP1, NRG1, and AKT1, on bipolar spectrum disorder, psychotic disorder, and related cognitive endophenotypes in a Finnish family-based sample ascertained for bipolar disorder.</p><p class="abstract"><strong>Results</strong>: In initial screening of 362 individuals from 63 families, we found only marginal evidence for association with the diagnosis-based dichotomous classification. Those associations did not strengthen when we genotyped the complete sample of 723 individuals from 180 families. We observed a significant association of DAOA variants rs3916966 and rs2391191 with visuospatial ability (Quantitative Transmission Disequilibrium Test [QTDT]; p = 4 x 10(-6) and 5 x 10(-6), respectively) (n = 159) with the two variants in almost complete linkage disequilibrium. The COMT variant rs165599 also associated with visuospatial ability, and in our dataset, we saw an additive effect of DAOA and COMT variants on this neuropsychological trait.</p><p class="abstract"><strong>Conclusions</strong>: The ancestral allele (Arg) of the nonsynonymous common DAOA variant rs2391191 (Arg30Lys) was found to predispose to impaired performance. The DAOA gene may play a role in predisposing individuals to a mixed phenotype of psychosis and mania and to impairments in related neuropsychological traits.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Biological psychiatry</span>&nbsp;<span class="year">2008</span>;<span class="volume">64</span>;<span class="number">5</span>;<span class="pagerange">438-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18466879" rel="external">18466879</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2685493" rel="external">2685493</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.biopsych.2008.03.028" rel="external">10.1016/j.biopsych.2008.03.028</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The novel mouse mutation Oblivion inactivates the PMCA2 pump and causes progressive hearing loss.</h4>
<p class="authors"><span class="author">Spiden SL</span>, <span class="author">Bortolozzi M</span>, <span class="author">Di Leva F</span>, <span class="author">de Angelis MH</span>, <span class="author">Fuchs H</span>, <span class="author">Lim D</span>, <span class="author">Ortolano S</span>, <span class="author">Ingham NJ</span>, <span class="author">Brini M</span>, <span class="author">Carafoli E</span>, <span class="author">Mammano F</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Progressive hearing loss is common in the human population, but we have few clues to the molecular basis. Mouse mutants with progressive hearing loss offer valuable insights, and ENU (N-ethyl-N-nitrosourea) mutagenesis is a useful way of generating models. We have characterised a new ENU-induced mouse mutant, Oblivion (allele symbol Obl), showing semi-dominant inheritance of hearing impairment. Obl/+ mutants showed increasing hearing impairment from post-natal day (P)20 to P90, and loss of auditory function was followed by a corresponding base to apex progression of hair cell degeneration. Obl/Obl mutants were small, showed severe vestibular dysfunction by 2 weeks of age, and were completely deaf from birth; sensory hair cells were completely degenerate in the basal turn of the cochlea, although hair cells appeared normal in the apex. We mapped the mutation to Chromosome 6. Mutation analysis of Atp2b2 showed a missense mutation (2630C--&gt;T) in exon 15, causing a serine to phenylalanine substitution (S877F) in transmembrane domain 6 of the PMCA2 pump, the resident Ca(2+) pump of hair cell stereocilia. Transmembrane domain mutations in these pumps generally are believed to be incompatible with normal targeting of the protein to the plasma membrane. However, analyses of hair cells in cultured utricular maculae of Obl/Obl mice and of the mutant Obl pump in model cells showed that the protein was correctly targeted to the plasma membrane. Biochemical and biophysical characterisation showed that the pump had lost a significant portion of its non-stimulated Ca(2+) exporting ability. These findings can explain the progressive loss of auditory function, and indicate the limits in our ability to predict mechanism from sequence alone.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Telethon: <span class="grant">GGP04169</span>; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">10</span>;<span class="pagerange">e1000238</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18974863">18974863</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2568954">2568954</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000238">10.1371/journal.pgen.1000238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large recurrent microdeletions associated with schizophrenia.</h4>
<p class="authors"><span class="author">Stefansson H</span>, <span class="author">Rujescu D</span>, <span class="author">Cichon S</span>, <span class="author">Pietil&auml;inen OP</span>, <span class="author">Ingason A</span>, <span class="author">Steinberg S</span>, <span class="author">Fossdal R</span>, <span class="author">Sigurdsson E</span>, <span class="author">Sigmundsson T</span>, <span class="author">Buizer-Voskamp JE</span>, <span class="author">Hansen T</span>, <span class="author">Jakobsen KD</span>, <span class="author">Muglia P</span>, <span class="author">Francks C</span>, <span class="author">Matthews PM</span>, <span class="author">Gylfason A</span>, <span class="author">Halldorsson BV</span>, <span class="author">Gudbjartsson D</span>, <span class="author">Thorgeirsson TE</span>, <span class="author">Sigurdsson A</span>, <span class="author">Jonasdottir A</span>, <span class="author">Jonasdottir A</span>, <span class="author">Bjornsson A</span>, <span class="author">Mattiasdottir S</span>, <span class="author">Blondal T</span>, <span class="author">Haraldsson M</span>, <span class="author">Magnusdottir BB</span>, <span class="author">Giegling I</span>, <span class="author">M&ouml;ller HJ</span>, <span class="author">Hartmann A</span>, <span class="author">Shianna KV</span>, <span class="author">Ge D</span>, <span class="author">Need AC</span>, <span class="author">Crombie C</span>, <span class="author">Fraser G</span>, <span class="author">Walker N</span>, <span class="author">Lonnqvist J</span>, <span class="author">Suvisaari J</span>, <span class="author">Tuulio-Henriksson A</span>, <span class="author">Paunio T</span>, <span class="author">Toulopoulou T</span>, <span class="author">Bramon E</span>, <span class="author">Di Forti M</span>, <span class="author">Murray R</span>, <span class="author">Ruggeri M</span>, <span class="author">Vassos E</span>, <span class="author">Tosato S</span>, <span class="author">Walshe M</span>, <span class="author">Li T</span>, <span class="author">Vasilescu C</span>, <span class="author">M&uuml;hleisen TW</span>, <span class="author">Wang AG</span>, <span class="author">Ullum H</span>, <span class="author">Djurovic S</span>, <span class="author">Melle I</span>, <span class="author">Olesen J</span>, <span class="author">Kiemeney LA</span>, <span class="author">Franke B</span>, <span class="author">GROUP</span>, <span class="author">Sabatti C</span>, <span class="author">Freimer NB</span>, <span class="author">Gulcher JR</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Kong A</span>, <span class="author">Andreassen OA</span>, <span class="author">Ophoff RA</span>, <span class="author">Georgi A</span>, <span class="author">Rietschel M</span>, <span class="author">Werge T</span>, <span class="author">Petursson H</span>, <span class="author">Goldstein DB</span>, <span class="author">N&ouml;then MM</span>, <span class="author">Peltonen L</span>, <span class="author">Collier DA</span>, <span class="author">St Clair D</span> and <span class="author">Stefansson K</span></p>
<p class="affiliation">CNS Division, deCODE genetics, Sturlugata 8, IS-101 Reykjav&iacute;k, Iceland.</p><p class="abstract">Reduced fecundity, associated with severe mental disorders, places negative selection pressure on risk alleles and may explain, in part, why common variants have not been found that confer risk of disorders such as autism, schizophrenia and mental retardation. Thus, rare variants may account for a larger fraction of the overall genetic risk than previously assumed. In contrast to rare single nucleotide mutations, rare copy number variations (CNVs) can be detected using genome-wide single nucleotide polymorphism arrays. This has led to the identification of CNVs associated with mental retardation and autism. In a genome-wide search for CNVs associating with schizophrenia, we used a population-based sample to identify de novo CNVs by analysing 9,878 transmissions from parents to offspring. The 66 de novo CNVs identified were tested for association in a sample of 1,433 schizophrenia cases and 33,250 controls. Three deletions at 1q21.1, 15q11.2 and 15q13.3 showing nominal association with schizophrenia in the first sample (phase I) were followed up in a second sample of 3,285 cases and 7,951 controls (phase II). All three deletions significantly associate with schizophrenia and related psychoses in the combined sample. The identification of these rare, recurrent risk variants, having occurred independently in multiple founders and being subject to negative selection, is important in itself. CNV analysis may also point the way to the identification of additional and more prevalent risk variants in genes and pathways involved in schizophrenia.</p>
<p class="grants">Funded by: Department of Health: <span class="grant">PDA/02/06/016</span>; Medical Research Council: <span class="grant">G0901310</span>; NIMH NIH HHS: <span class="grant">R01 MH078075</span>, <span class="grant">R01MH71425-01A1</span>; Wellcome Trust: <span class="grant">089061</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2008</span>;<span class="volume">455</span>;<span class="number">7210</span>;<span class="pagerange">232-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18668039">18668039</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687075">2687075</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature07229">10.1038/nature07229</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Accounting for non-genetic factors improves the power of eQTL studies</h4>
<p class="authors"><span class="author">Stegle</span>, <span class="author">O</span>, <span class="author">Kannan</span>, <span class="author">A</span>, <span class="author">Durbin</span>, <span class="author">R</span>, <span class="author">Winn</span>, <span class="author">J.</span></p>
<p class="publication"><span class="title">Lecture Notes in Computer Science.</span> <span class="year">2008</span>;<span class="volume">4955</span>;<span class="pagerange">411-422</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insights from the complete genome sequence of Mycobacterium marinum on the evolution of Mycobacterium tuberculosis.</h4>
<p class="authors"><span class="author">Stinear TP</span>, <span class="author">Seemann T</span>, <span class="author">Harrison PF</span>, <span class="author">Jenkin GA</span>, <span class="author">Davies JK</span>, <span class="author">Johnson PD</span>, <span class="author">Abdellah Z</span>, <span class="author">Arrowsmith C</span>, <span class="author">Chillingworth T</span>, <span class="author">Churcher C</span>, <span class="author">Clarke K</span>, <span class="author">Cronin A</span>, <span class="author">Davis P</span>, <span class="author">Goodhead I</span>, <span class="author">Holroyd N</span>, <span class="author">Jagels K</span>, <span class="author">Lord A</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Norbertczak H</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Walker D</span>, <span class="author">White B</span>, <span class="author">Whitehead S</span>, <span class="author">Small PL</span>, <span class="author">Brosch R</span>, <span class="author">Ramakrishnan L</span>, <span class="author">Fischbach MA</span>, <span class="author">Parkhill J</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Department of Microbiology, Monash University, Clayton 3800, Australia. <a href="mailto:%74%69%6D.%73%74%69%6E%65%61%72%40%6D%65%64.%6D%6F%6E%61%73%68.%65%64%75.%61%75">&#116;&#105;&#109;&#46;&#115;&#116;&#105;&#110;&#101;&#97;&#114;&#64;&#109;&#101;&#100;&#46;&#109;&#111;&#110;&#97;&#115;&#104;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Mycobacterium marinum, a ubiquitous pathogen of fish and amphibia, is a near relative of Mycobacterium tuberculosis, the etiologic agent of tuberculosis in humans. The genome of the M strain of M. marinum comprises a 6,636,827-bp circular chromosome with 5424 CDS, 10 prophages, and a 23-kb mercury-resistance plasmid. Prominent features are the very large number of genes (57) encoding polyketide synthases (PKSs) and nonribosomal peptide synthases (NRPSs) and the most extensive repertoire yet reported of the mycobacteria-restricted PE and PPE proteins, and related-ESX secretion systems. Some of the NRPS genes comprise a novel family and seem to have been acquired horizontally. M. marinum is used widely as a model organism to study M. tuberculosis pathogenesis, and genome comparisons confirmed the close genetic relationship between these two species, as they share 3000 orthologs with an average amino acid identity of 85%. Comparisons with the more distantly related Mycobacterium avium subspecies paratuberculosis and Mycobacterium smegmatis reveal how an ancestral generalist mycobacterium evolved into M. tuberculosis and M. marinum. M. tuberculosis has undergone genome downsizing and extensive lateral gene transfer to become a specialized pathogen of humans and other primates without retaining an environmental niche. M. marinum has maintained a large genome so as to retain the capacity for environmental survival while becoming a broad host range pathogen that produces disease strikingly similar to M. tuberculosis. The work described herein provides a foundation for using M. marinum to better understand the determinants of pathogenesis of tuberculosis.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI036396</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">5</span>;<span class="pagerange">729-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18403782">18403782</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2336800">2336800</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.075069.107">10.1101/gr.075069.107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distal transgene insertion affects CpG island maintenance during differentiation.</h4>
<p class="authors"><span class="author">Strathdee D</span>, <span class="author">Whitelaw CB</span> and <span class="author">Clark AJ</span></p>
<p class="affiliation">Division of Gene Function and Development, Roslin Institute, Roslin, Midlothian EH25 9PS, United Kingdom. <a href="mailto:%64%73%31%30%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#49;&#48;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">About half of all genes have a CpG island surrounding the promoter and transcription start site. Most promoter CpG islands are normally unmethylated in all tissues, irrespective of the expression level of the associated gene. Establishment of the appropriate patterns of DNA methylation in the genome is essential for normal development and patterns of gene expression. Aberrant methylation of CpG islands and silencing of the associated genes is frequently observed in cancer. One gene with a 5&#39;-CpG island is cytoplasmic beta-actin, which is an abundantly expressed protein and a major component of microfilaments. Inserting a betageo cassette into the 3&#39;-untranslated region of beta-actin gene led to widespread but not ubiquitous lacZ expression in mice heterozygous for the modified beta-actin allele. Surprisingly, embryos homozygous for this insertion died at mid-gestation. The modified beta-actin allele was expressed in undifferentiated embryonic stem cells but was turned off as these cells differentiate in vitro and in vivo. We demonstrate that the insertion affects the maintenance of the methylation status of the CpG island of the modified beta-actin allele in differentiated but not in undifferentiated embryonic cells. These data suggest that there is a two-step process to defining a CpG island, requiring both embryonic establishment and a signal that maintains the CpG island in differentiated cells. Furthermore, they indicate that features built into the CpG island are not sufficient to direct CpG island maintenance during differentiation.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">Cad04943</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2008</span>;<span class="volume">283</span>;<span class="number">17</span>;<span class="pagerange">11509-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18308728" rel="external">18308728</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M709805200" rel="external">10.1074/jbc.M709805200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome resequencing and genetic variation.</h4>
<p class="authors"><span class="author">Stratton M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%72%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#114;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">1</span>;<span class="pagerange">65-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18183021" rel="external">18183021</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt0108-65" rel="external">10.1038/nbt0108-65</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The emerging landscape of breast cancer susceptibility.</h4>
<p class="authors"><span class="author">Stratton MR</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, 15 Cotswold Road, Sutton, Surrey SM2 5NG, UK.</p><p class="abstract">The genetic basis of inherited predisposition to breast cancer has been assiduously investigated for the past two decades and has been the subject of several recent discoveries. Three reasonably well-defined classes of breast cancer susceptibility alleles with different levels of risk and prevalence in the population have become apparent: rare high-penetrance alleles, rare moderate-penetrance alleles and common low-penetrance alleles. The contribution of each component to breast cancer predisposition is still to be fully explored, as are the phenotypic characteristics of the cancers associated with them, the ways in which they interact, much of their biology and their clinical utility. These recent advances herald a new chapter in the exploration of susceptibility to breast cancer and are likely to provide insights relevant to other common, heterogeneous diseases.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">1</span>;<span class="pagerange">17-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18163131">18163131</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2007.53">10.1038/ng.2007.53</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A DNA transposon-based approach to validate oncogenic mutations in the mouse.</h4>
<p class="authors"><span class="author">Su Q</span>, <span class="author">Prosser HM</span>, <span class="author">Campos LS</span>, <span class="author">Ortiz M</span>, <span class="author">Nakamura T</span>, <span class="author">Warren M</span>, <span class="author">Dupuy AJ</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Bradley A</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, England.</p><p class="abstract">Large-scale cancer genome projects will soon be able to sequence many cancer genomes to comprehensively identify genetic changes in human cancer. Genome-wide association studies have also identified putative cancer associated loci. Functional validation of these genetic mutations in vivo is becoming a challenge. We describe here a DNA transposon-based platform that permits us to explore the oncogenic potential of genetic mutations in the mouse. Briefly, promoter-less human cancer gene cDNAs were first cloned into Sleeping Beauty (SB) transposons. DNA transposition in the mouse that carried both the transposons and the SB transposase made it possible for the cDNAs to be expressed from an appropriate endogenous genomic locus and in the relevant cell types for tumor development. Consequently, these mice developed a broad spectrum of tumors at very early postnatal stages. This technology thus complements the large-scale cancer genome projects.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">50</span>;<span class="pagerange">19904-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19064922">19064922</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2597691">2597691</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0807785105">10.1073/pnas.0807785105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HapMap coverage for SNPs in the Japanese population.</h4>
<p class="authors"><span class="author">Takeuchi F</span>, <span class="author">Serizawa M</span> and <span class="author">Kato N</span></p>
<p class="affiliation">Department of Medical Ecology and Informatics, Research Institute, International Medical Center of Japan, Tokyo, Japan.</p><p class="abstract">The coverage of human genomic variations is known to substantially affect the success of genome-wide association studies. We therefore assessed the SNP coverage in the HapMap database for a total of 1,304 subjects from the Japanese population by combining resequencing and high-density genotyping approaches. First, we resequenced 48 Japanese subjects in 86 genes (572 kb in total), and we then genotyped the subset of tag SNPs and also imputed genotypes for all of the detected SNPs in an additional panel of 1,256 subjects. Subsequently, we genotyped 555,352 tag SNPs selected from the HapMap in 72 Japanese subjects (from the panel of 1,256 subjects) and further imputed genotypes for all SNPs currently included in the HapMap. Of 738 common genic SNPs (1.3 per kb) that we detected by resequencing, 58% had already been genotyped in the HapMap, and 31% were not genotyped but had a proxy SNP in the HapMap with a linkage disequilibrium coefficient r (2) &gt; or = 0.8, whereas 11% were not represented in the current HapMap database. Thus, the HapMap coverage appears to be high although not thorough for SNPs in the Japanese population as compared to its coverage reported in Caucasians, and this needs to be considered when we interpret association results.</p>
<p class="publication"><span class="title">Journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">53</span>;<span class="number">1</span>;<span class="pagerange">96-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18043864" rel="external">18043864</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10038-007-0221-7" rel="external">10.1007/s10038-007-0221-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of CC2D2A as a Meckel syndrome gene adds an important piece to the ciliopathy puzzle.</h4>
<p class="authors"><span class="author">Tallila J</span>, <span class="author">Jakkula E</span>, <span class="author">Peltonen L</span>, <span class="author">Salonen R</span> and <span class="author">Kestil&auml; M</span></p>
<p class="affiliation">National Public Health Institute, Institute for Molecular Medicine Finland, Helsinki 00290, Finland.</p><p class="abstract">Meckel syndrome (MKS) is a lethal malformation disorder characterized classically by encephalocele, polycystic kidneys, and polydactyly. MKS is also one of the major contributors to syndromic neural tube defects (NTDs). Recent findings have shown primary cilia dysfunction in the molecular background of MKS, indicating that cilia are critical for early human development. However, even though four genes behind MKS have been identified to date, they elucidate only a minor proportion of the MKS cases. In this study, instead of traditional linkage analysis, we selected 10 nonrelated affected fetuses and looked for the homozygous regions shared by them. Based on this strategy, we identified the sixth locus and the fifth gene, CC2D2A (MKS6), behind MKS. The biological function of CC2D2A is uncharacterized, but the corresponding polypeptide is predicted to be involved in ciliary functions and it has a calcium binding domain (C2). Immunofluorescence staining of patient&#39;s fibroblast cells demonstrates that the cells lack cilia, providing evidence for the critical role of CC2D2A in cilia formation. Our finding is very significant not only to understand the molecular background of MKS, but also to obtain additional information about the function of the cilia, which can help to understand their significance in normal development and also in other ciliopathies, which are an increasing group of disorders with overlapping phenotypes.</p>
<p class="grants">Funded by: NIEHS NIH HHS: <span class="grant">P01 ES11253-03</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">6</span>;<span class="pagerange">1361-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18513680" rel="external">18513680</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427307" rel="external">2427307</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.05.004" rel="external">10.1016/j.ajhg.2008.05.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Patterns of evolution in the unique tRNA gene arrays of the genus Entamoeba.</h4>
<p class="authors"><span class="author">Tawari B</span>, <span class="author">Ali IK</span>, <span class="author">Scott C</span>, <span class="author">Quail MA</span>, <span class="author">Berriman M</span>, <span class="author">Hall N</span> and <span class="author">Clark CG</span></p>
<p class="affiliation">Department of Infectious and Tropical Diseases, London School of Hygiene and Tropical Medicine, London, United Kingdom.</p><p class="abstract">Genome sequencing of the protistan parasite Entamoeba histolytica HM-1:IMSS revealed that almost all the tRNA genes are organized into tandem arrays that make up over 10% of the genome. The 25 distinct array units contain up to 5 tRNA genes each and some also encode the 5S RNA. Between adjacent genes in array units are complex short tandem repeats (STRs) resembling microsatellites. To investigate the origins and evolution of this unique gene organization, we have undertaken a genome survey to determine the array unit organization in 4 other species of Entamoeba-Entamoeba dispar, Entamoeba moshkovskii, Entamoeba terrapinae, and Entamoeba invadens-and have explored the STR structure in other isolates of E. histolytica. The genome surveys revealed that E. dispar has the same array unit organization as E. histolytica, including the presence and numerical variation of STRs between adjacent genes. However, the individual repeat sequences are completely different to those in E. histolytica. All other species of Entamoeba studied also have tandem arrays of clustered tRNA genes, but the gene composition of the array units often differs from that in E. histolytica/E. dispar. None of the other species&#39; arrays exhibit the complex STRs between adjacent genes although simple tandem duplications are occasionally seen. The degree of similarity in organization reflects the phylogenetic relationships among the species studied. Within individual isolates of E. histolytica most copies of the array unit are uniform in sequence with only minor variation in the number and organization of the STRs. Between isolates, however, substantial differences in STR number and organization can exist although the individual repeat sequences tend to be conserved. The origin of this unique gene organization in the genus Entamoeba clearly predates the common ancestor of the species investigated to date and their function remains unclear.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">G18391</span>; Wellcome Trust: <span class="grant">067314</span>, <span class="grant">085775</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2008</span>;<span class="volume">25</span>;<span class="number">1</span>;<span class="pagerange">187-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17974548" rel="external">17974548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652664" rel="external">2652664</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsm238" rel="external">10.1093/molbev/msm238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Promoting coherent minimum reporting guidelines for biological and biomedical investigations: the MIBBI project.</h4>
<p class="authors"><span class="author">Taylor CF</span>, <span class="author">Field D</span>, <span class="author">Sansone SA</span>, <span class="author">Aerts J</span>, <span class="author">Apweiler R</span>, <span class="author">Ashburner M</span>, <span class="author">Ball CA</span>, <span class="author">Binz PA</span>, <span class="author">Bogue M</span>, <span class="author">Booth T</span>, <span class="author">Brazma A</span>, <span class="author">Brinkman RR</span>, <span class="author">Michael Clark A</span>, <span class="author">Deutsch EW</span>, <span class="author">Fiehn O</span>, <span class="author">Fostel J</span>, <span class="author">Ghazal P</span>, <span class="author">Gibson F</span>, <span class="author">Gray T</span>, <span class="author">Grimes G</span>, <span class="author">Hancock JM</span>, <span class="author">Hardy NW</span>, <span class="author">Hermjakob H</span>, <span class="author">Julian RK</span>, <span class="author">Kane M</span>, <span class="author">Kettner C</span>, <span class="author">Kinsinger C</span>, <span class="author">Kolker E</span>, <span class="author">Kuiper M</span>, <span class="author">Le Nov&egrave;re N</span>, <span class="author">Leebens-Mack J</span>, <span class="author">Lewis SE</span>, <span class="author">Lord P</span>, <span class="author">Mallon AM</span>, <span class="author">Marthandan N</span>, <span class="author">Masuya H</span>, <span class="author">McNally R</span>, <span class="author">Mehrle A</span>, <span class="author">Morrison N</span>, <span class="author">Orchard S</span>, <span class="author">Quackenbush J</span>, <span class="author">Reecy JM</span>, <span class="author">Robertson DG</span>, <span class="author">Rocca-Serra P</span>, <span class="author">Rodriguez H</span>, <span class="author">Rosenfelder H</span>, <span class="author">Santoyo-Lopez J</span>, <span class="author">Scheuermann RH</span>, <span class="author">Schober D</span>, <span class="author">Smith B</span>, <span class="author">Snape J</span>, <span class="author">Stoeckert CJ</span>, <span class="author">Tipton K</span>, <span class="author">Sterk P</span>, <span class="author">Untergasser A</span>, <span class="author">Vandesompele J</span> and <span class="author">Wiemann S</span></p>
<p class="affiliation">European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SD, UK. <a href="mailto:%63%68%72%69%73.%74%61%79%6C%6F%72%40%65%62%69.%61%63.%75%6B">&#99;&#104;&#114;&#105;&#115;&#46;&#116;&#97;&#121;&#108;&#111;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a>, <a href="mailto:%64%66%69%65%6C%64%40%63%65%68.%61%63.%75%6B">&#100;&#102;&#105;&#101;&#108;&#100;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">E025080/1</span>; Medical Research Council: <span class="grant">MC_U142684171</span>; NHGRI NIH HHS: <span class="grant">U54 HG004028</span>; NIBIB NIH HHS: <span class="grant">EB005034-01</span>, <span class="grant">R01 EB005034</span>, <span class="grant">R01 EB005034-04</span></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2008</span>;<span class="volume">26</span>;<span class="number">8</span>;<span class="pagerange">889-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18688244">18688244</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2771753">2771753</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt.1411">10.1038/nbt.1411</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Control of feeding behavior in C. elegans by human G protein-coupled receptors permits screening for agonist-expressing bacteria.</h4>
<p class="authors"><span class="author">Teng MS</span>, <span class="author">Shadbolt P</span>, <span class="author">Fraser AG</span>, <span class="author">Jansen G</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1HH, United Kingdom.</p><p class="abstract">G protein-coupled receptors (GPCRs) have a key role in many biological processes and are important drug targets for many human diseases. Therefore, understanding the molecular interactions between GPCRs and their ligands would improve drug design. Here, we describe an approach that allows the rapid identification of functional agonists expressed in bacteria. Transgenic Caenorhabditis elegans expressing the human chemokine receptor 5 (CCR5) in nociceptive neurons show avoidance behavior on encounter with the ligand MIP-1alpha and avoid feeding on Escherichia coli expressing MIP-1alpha compared with control bacteria. This system allows a simple activity screen, based on the distribution of transgenic worms in a binary food-choice assay, without a requirement for protein purification or tagging. By using this approach, a library of 68 MIP-1alpha variants was screened, and 13 critical agonist residues involved in CCR5 activation were identified, four of which (T8, A9, N22, and A25) have not been described previously, to our knowledge. Identified residues were subsequently validated in receptor binding assays and by calcium flux assays in mammalian cells. This approach serves not only for structure/function studies as demonstrated, but may be used to facilitate the discovery of agonists within bacterial libraries.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">39</span>;<span class="pagerange">14826-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18815363" rel="external">18815363</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2553035" rel="external">2553035</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0803290105" rel="external">10.1073/pnas.0803290105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common statistical issues in genome-wide association studies: a review on power, data quality control, genotype calling and population structure.</h4>
<p class="authors"><span class="author">Teo YY</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, UK. <a href="mailto:%74%65%6F%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#116;&#101;&#111;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genetic association studies which survey the entire genome have become a common design for uncovering the genetic basis of common diseases, including lipid-related traits. Such studies have identified several novel loci which influence blood lipids. The present review highlights the statistical challenges associated with such large-scale genetic studies and discusses the available methodological strategies for handling these issues.</p><p class="abstract">The successful analysis of genome-wide data assayed on commercial genotyping arrays depends on careful exploration of the data. Unaccounted sample failures, genotyping errors and population structure can introduce misleading signals that mimic genuine association. Careful interpretation of useful summary statistics and graphical data displays can minimize the extent of false associations that need to be followed up in replication or fine-mapping experiments.</p><p class="abstract"><strong>Summary</strong>: Recently published genome-wide studies are beginning to yield valuable insights into the importance of well designed methodological and statistical techniques for sensible interpretation of the plethora of genetic data generated.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in lipidology</span>&nbsp;<span class="year">2008</span>;<span class="volume">19</span>;<span class="number">2</span>;<span class="pagerange">133-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18388693" rel="external">18388693</a>; DOI: <a href="http://dx.doi.org/10.1097%2FMOL.0b013e3282f5dd77" rel="external">10.1097/MOL.0b013e3282f5dd77</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome-amplified DNA: insights and imputation.</h4>
<p class="authors"><span class="author">Teo YY</span>, <span class="author">Inouye M</span>, <span class="author">Small KS</span>, <span class="author">Fry AE</span>, <span class="author">Potter SC</span>, <span class="author">Dunstan SJ</span>, <span class="author">Seielstad M</span>, <span class="author">Barroso I</span>, <span class="author">Wareham NJ</span>, <span class="author">Rockett KA</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Deloukas P</span></p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G19/9</span>, <span class="grant">MC_U106179471</span>; Wellcome Trust: <span class="grant">077011</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2008</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">279-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18376389">18376389</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2683750">2683750</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth0408-279">10.1038/nmeth0408-279</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Perturbation analysis: a simple method for filtering SNPs with erroneous genotyping in genome-wide association studies.</h4>
<p class="authors"><span class="author">Teo YY</span>, <span class="author">Small KS</span>, <span class="author">Clark TG</span> and <span class="author">Kwiatkowski DP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, United Kingdom. <a href="mailto:%74%65%6F%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#116;&#101;&#111;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We introduce a simple and yet scientifically objective criterion for identifying SNPs with genotyping errors due to poor clustering. This yields a metric for assessing the stability of the assigned genotypes by evaluating the extent of discordance between the calls made with the unperturbed and perturbed intensities. The efficacy of the metric is evaluated by: (1) estimating the extent of over-dispersion of the Hardy-Weinberg equilibrium chi-square test statistics; (2) an interim case-control study, where we investigated the efficacy of the introduced metric and standard quality control filters in reducing the number of SNPs with evidence of phenotypic association which are attributed to genotyping errors; (3) investigating the call and concordance rates of SNPs identified by perturbation analysis which have been genotyped on both Affymetrix and Illumina platforms. Removing SNPs identified by the extent of discordance can reduce the degree of over-dispersion of the HWE test statistic. Sensible use of perturbation analysis in an association study can correctly identify SNPs with problematic genotyping, reducing the number required for visual inspection. SNPs identified by perturbation analysis had lower call and concordance rates, and removal of these SNPs significantly improved the performance for the remaining SNPs.</p>
<p class="grants">Funded by: Medical Research Council; Wellcome Trust: <span class="grant">082370</span></p>
<p class="publication"><span class="title">Annals of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">72</span>;<span class="number">Pt 3</span>;<span class="pagerange">368-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18261185" rel="external">18261185</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2997476" rel="external">2997476</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1469-1809.2007.00422.x" rel="external">10.1111/j.1469-1809.2007.00422.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maternal footprints of Southeast Asians in North India.</h4>
<p class="authors"><span class="author">Thangaraj K</span>, <span class="author">Chaubey G</span>, <span class="author">Kivisild T</span>, <span class="author">Selvi Rani D</span>, <span class="author">Singh VK</span>, <span class="author">Ismail T</span>, <span class="author">Carvalho-Silva D</span>, <span class="author">Metspalu M</span>, <span class="author">Bhaskar LV</span>, <span class="author">Reddy AG</span>, <span class="author">Chandra S</span>, <span class="author">Pande V</span>, <span class="author">Prathap Naidu B</span>, <span class="author">Adarsh N</span>, <span class="author">Verma A</span>, <span class="author">Jyothi IA</span>, <span class="author">Mallick CB</span>, <span class="author">Shrivastava N</span>, <span class="author">Devasena R</span>, <span class="author">Kumari B</span>, <span class="author">Singh AK</span>, <span class="author">Dwivedi SK</span>, <span class="author">Singh S</span>, <span class="author">Rao G</span>, <span class="author">Gupta P</span>, <span class="author">Sonvane V</span>, <span class="author">Kumari K</span>, <span class="author">Basha A</span>, <span class="author">Bhargavi KR</span>, <span class="author">Lalremruata A</span>, <span class="author">Gupta AK</span>, <span class="author">Kaur G</span>, <span class="author">Reddy KK</span>, <span class="author">Rao AP</span>, <span class="author">Villems R</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Singh L</span></p>
<p class="affiliation">Centre for Cellular and Molecular Biology, Hyderabad, India.</p><p class="abstract">We have analyzed 7,137 samples from 125 different caste, tribal and religious groups of India and 99 samples from three populations of Nepal for the length variation in the COII/tRNA(Lys) region of mtDNA. Samples showing length variation were subjected to detailed phylogenetic analysis based on HVS-I and informative coding region sequence variation. The overall frequencies of the 9-bp deletion and insertion variants in South Asia were 1.9 and 0.6%, respectively. We have also defined a novel deep-rooting haplogroup M43 and identified the rare haplogroup H14 in Indian populations carrying the 9-bp deletion by complete mtDNA sequencing. Moreover, we redefined haplogroup M6 and dissected it into two well-defined subclades. The presence of haplogroups F1 and B5a in Uttar Pradesh suggests minor maternal contribution from Southeast Asia to Northern India. The occurrence of haplogroup F1 in the Nepalese sample implies that Nepal might have served as a bridge for the flow of eastern lineages to India. The presence of R6 in the Nepalese, on the other hand, suggests that the gene flow between India and Nepal has been reciprocal.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Human heredity</span>&nbsp;<span class="year">2008</span>;<span class="volume">66</span>;<span class="number">1</span>;<span class="pagerange">1-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18223312">18223312</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2588665">2588665</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000114160">10.1159/000114160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of 6q deletions in mature B cell lymphomas and childhood acute lymphoblastic leukemia.</h4>
<p class="authors"><span class="author">Thelander EF</span>, <span class="author">Ichimura K</span>, <span class="author">Corcoran M</span>, <span class="author">Barbany G</span>, <span class="author">Nordgren A</span>, <span class="author">Heyman M</span>, <span class="author">Berglund M</span>, <span class="author">Mungall A</span>, <span class="author">Rosenquist R</span>, <span class="author">Collins VP</span>, <span class="author">Grand&eacute;r D</span>, <span class="author">Larsson C</span> and <span class="author">Lagercrantz S</span></p>
<p class="affiliation">Medical Genetics Unit, Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden.</p><p class="abstract">The study was undertaken with the aim to outline deletion patterns involving the long arm of chromosome 6, a common abnormality in lymphoproliferative disorders. Using a chromosome 6 specific tile path array, 60 samples from in total 49 cases with mantle cell lymphoma (MCL), de novo diffuse large B-cell lymphoma (DLBCL), transformed DLBCL as well as preceding follicular lymphoma (FL), and childhood acute lymphoblastic leukemia (ALL), were characterized. Twenty-six of the studied cases, representing all diagnoses, showed a 6q deletion among which 85% involved a 3 Mb region in 6q21. The minimal deleted interval in 6q21 encompasses the FOXO3A, PRDM1 and HACE1 candidate genes. The PRDM1 gene was found homozygously deleted in a case of DLBCL. Moreover, in two DLBCL cases, an overlapping homozygous deletion was identified in 6q23.3 - 24.1, encompassing the TNFAIP3 gene among others. Taken together, we refined the deletion pattern within the long arm of chromosome 6 in four different types of hematological malignances, suggesting the location of tumor suppressor genes involved in the tumor progression.</p>
<p class="publication"><span class="title">Leukemia &amp; lymphoma</span>&nbsp;<span class="year">2008</span>;<span class="volume">49</span>;<span class="number">3</span>;<span class="pagerange">477-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18297524">18297524</a>; DOI: <a href="http://dx.doi.org/10.1080%2F10428190701817282">10.1080/10428190701817282</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Phenotypical characteristics of idiopathic infantile nystagmus with and without mutations in FRMD7.</h4>
<p class="authors"><span class="author">Thomas S</span>, <span class="author">Proudlock FA</span>, <span class="author">Sarvananthan N</span>, <span class="author">Roberts EO</span>, <span class="author">Awan M</span>, <span class="author">McLean R</span>, <span class="author">Surendran M</span>, <span class="author">Kumar AS</span>, <span class="author">Farooq SJ</span>, <span class="author">Degg C</span>, <span class="author">Gale RP</span>, <span class="author">Reinecke RD</span>, <span class="author">Woodruff G</span>, <span class="author">Langmann A</span>, <span class="author">Lindner S</span>, <span class="author">Jain S</span>, <span class="author">Tarpey P</span>, <span class="author">Raymond FL</span> and <span class="author">Gottlob I</span></p>
<p class="affiliation">Ophthalmology Group, University of Leicester, Leicester, UK.</p><p class="abstract">Idiopathic infantile nystagmus (IIN) consists of involuntary oscillations of the eyes. The familial form is most commonly X-linked. We recently found mutations in a novel gene FRMD7 (Xq26.2), which provided an opportunity to investigate a genetically defined and homogeneous group of patients with nystagmus. We compared clinical features and eye movement recordings of 90 subjects with mutation in the gene (FRMD7 group) to 48 subjects without mutations but with clinical IIN (non-FRMD7 group). Fifty-eight female obligate carriers of the mutation were also investigated. The median visual acuity (VA) was 0.2 logMAR (Snellen equivalent 6/9) in both groups and most patients had good stereopsis. The prevalence of strabismus was also similar (FRMD7: 7.8%, non-FRMD7: 10%). The presence of anomalous head posture (AHP) was significantly higher in the non-FRMD7 group (P &lt; 0.0001). The amplitude of nystagmus was more strongly dependent on the direction of gaze in the FRMD7 group being lower at primary position (P &lt; 0.0001), compared to non-FRMD7 group (P = 0.83). Pendular nystagmus waveforms were also more frequent in the FRMD7 group (P = 0.003). Fifty-three percent of the obligate female carriers of an FRMD7 mutation were clinically affected. The VA&#39;s in affected females were slightly better compared to affected males (P = 0.014). Subnormal optokinetic responses were found in a subgroup of obligate unaffected carriers, which may be interpreted as a sub-clinical manifestation. FRMD7 is a major cause of X-linked IIN. Most clinical and eye movement characteristics were similar in the FRMD7 group and non-FRMD7 group with most patients having good VA and stereopsis and low incidence of strabismus. Fewer patients in the FRMD7 group had AHPs, their amplitude of nystagmus being lower in primary position. Our findings are helpful in the clinical identification of IIN and genetic counselling of nystagmus patients.</p>
<p class="publication"><span class="title">Brain : a journal of neurology</span>&nbsp;<span class="year">2008</span>;<span class="volume">131</span>;<span class="number">Pt 5</span>;<span class="pagerange">1259-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18372314">18372314</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbrain%2Fawn046">10.1093/brain/awn046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Vive la diff&eacute;rence.</h4>
<p class="authors"><span class="author">Thomson NR</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">502-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18536729">18536729</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1929">10.1038/nrmicro1929</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genome analysis of Salmonella Enteritidis PT4 and Salmonella Gallinarum 287/91 provides insights into evolutionary and host adaptation pathways.</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Clayton DJ</span>, <span class="author">Windhorst D</span>, <span class="author">Vernikos G</span>, <span class="author">Davidson S</span>, <span class="author">Churcher C</span>, <span class="author">Quail MA</span>, <span class="author">Stevens M</span>, <span class="author">Jones MA</span>, <span class="author">Watson M</span>, <span class="author">Barron A</span>, <span class="author">Layton A</span>, <span class="author">Pickard D</span>, <span class="author">Kingsley RA</span>, <span class="author">Bignell A</span>, <span class="author">Clark L</span>, <span class="author">Harris B</span>, <span class="author">Ormond D</span>, <span class="author">Abdellah Z</span>, <span class="author">Brooks K</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth T</span>, <span class="author">Woodward J</span>, <span class="author">Norberczak H</span>, <span class="author">Lord A</span>, <span class="author">Arrowsmith C</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Sanders M</span>, <span class="author">Whitehead S</span>, <span class="author">Chabalgoity JA</span>, <span class="author">Maskell D</span>, <span class="author">Humphrey T</span>, <span class="author">Roberts M</span>, <span class="author">Barrow PA</span>, <span class="author">Dougan G</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6E%72%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#114;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have determined the complete genome sequences of a host-promiscuous Salmonella enterica serovar Enteritidis PT4 isolate P125109 and a chicken-restricted Salmonella enterica serovar Gallinarum isolate 287/91. Genome comparisons between these and other Salmonella isolates indicate that S. Gallinarum 287/91 is a recently evolved descendent of S. Enteritidis. Significantly, the genome of S. Gallinarum has undergone extensive degradation through deletion and pseudogene formation. Comparison of the pseudogenes in S. Gallinarum with those identified previously in other host-adapted bacteria reveals the loss of many common functional traits and provides insights into possible mechanisms of host and tissue adaptation. We propose that experimental analysis in chickens and mice of S. Enteritidis-harboring mutations in functional homologs of the pseudogenes present in S. Gallinarum could provide an experimentally tractable route toward unraveling the genetic basis of host adaptation in S. enterica.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">10</span>;<span class="pagerange">1624-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18583645">18583645</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2556274">2556274</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.077404.108">10.1101/gr.077404.108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chlamydia trachomatis: genome sequence analysis of lymphogranuloma venereum isolates.</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Holden MT</span>, <span class="author">Carder C</span>, <span class="author">Lennard N</span>, <span class="author">Lockey SJ</span>, <span class="author">Marsh P</span>, <span class="author">Skipp P</span>, <span class="author">O&#39;Connor CD</span>, <span class="author">Goodhead I</span>, <span class="author">Norbertzcak H</span>, <span class="author">Harris B</span>, <span class="author">Ormond D</span>, <span class="author">Rance R</span>, <span class="author">Quail MA</span>, <span class="author">Parkhill J</span>, <span class="author">Stephens RS</span> and <span class="author">Clarke IN</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6E%72%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#114;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chlamydia trachomatis is the most common cause of sexually transmitted infections in the UK, a statistic that is also reflected globally. There are three biovariants of C. trachomatis: trachoma (serotypes A-C) and two sexually transmitted pathovars; serotypes D-K and lymphogranuloma venereum (LGV). Trachoma isolates and the sexually transmitted serotypes D-K are noninvasive, whereas the LGV strains are invasive, causing a disseminating infection of the local draining lymph nodes. Genome sequences are available for single isolates from the trachoma (serotype A) and sexually transmitted (serotype D) biotypes. We sequenced two isolates from the remaining biotype, LGV, a long-term laboratory passaged strain and the recent &quot;epidemic&quot; LGV isolate-causing proctitis. Although the genome of the LGV strain shows no additional genes that could account for the differences in disease outcome, we found evidence of functional gene loss and identified regions of heightened sequence variation that have previously been shown to be important sites for interstrain recombination. We have used new sequencing technologies to show that the recent clinical LGV isolate causing proctitis is unlikely to be a newly emerged strain but is most probably an old strain with relatively new clinical manifestations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">080348</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">1</span>;<span class="pagerange">161-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18032721">18032721</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2134780">2134780</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.7020108">10.1101/gr.7020108</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary plasticity of genetic interaction networks.</h4>
<p class="authors"><span class="author">Tischler J</span>, <span class="author">Lehner B</span> and <span class="author">Fraser AG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Non-additive genetic interactions contribute to many genetic disorders, but they are extremely difficult to predict. Here we show that genetic interactions identified in yeast, unlike gene functions or protein interactions, are not highly conserved in animals. Genetic interactions are therefore unlikely to represent simple redundancy between genes or pathways, and genetic interactions from yeast do not directly predict genetic interactions in higher eukaryotes, including humans.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">4</span>;<span class="pagerange">390-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18362882">18362882</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.114">10.1038/ng.114</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of a genomic tiling array of the human major histocompatibility complex (MHC) and its application for DNA methylation analysis.</h4>
<p class="authors"><span class="author">Tomazou EM</span>, <span class="author">Rakyan VK</span>, <span class="author">Lefebvre G</span>, <span class="author">Andrews R</span>, <span class="author">Ellis P</span>, <span class="author">Jackson DK</span>, <span class="author">Langford C</span>, <span class="author">Francis MD</span>, <span class="author">B&auml;ckdahl L</span>, <span class="author">Miretti M</span>, <span class="author">Coggill P</span>, <span class="author">Ottaviani D</span>, <span class="author">Sheer D</span>, <span class="author">Murrell A</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%65%6D%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#109;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The major histocompatibility complex (MHC) is essential for human immunity and is highly associated with common diseases, including cancer. While the genetics of the MHC has been studied intensively for many decades, very little is known about the epigenetics of this most polymorphic and disease-associated region of the genome.</p><p class="abstract"><strong>Methods</strong>: To facilitate comprehensive epigenetic analyses of this region, we have generated a genomic tiling array of 2 Kb resolution covering the entire 4 Mb MHC region. The array has been designed to be compatible with chromatin immunoprecipitation (ChIP), methylated DNA immunoprecipitation (MeDIP), array comparative genomic hybridization (aCGH) and expression profiling, including of non-coding RNAs. The array comprises 7832 features, consisting of two replicates of both forward and reverse strands of MHC amplicons and appropriate controls.</p><p class="abstract"><strong>Results</strong>: Using MeDIP, we demonstrate the application of the MHC array for DNA methylation profiling and the identification of tissue-specific differentially methylated regions (tDMRs). Based on the analysis of two tissues and two cell types, we identified 90 tDMRs within the MHC and describe their characterisation.</p><p class="abstract"><strong>Conclusion</strong>: A tiling array covering the MHC region was developed and validated. Its successful application for DNA methylation profiling indicates that this array represents a useful tool for molecular analyses of the MHC in the context of medical genomics.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A8318</span></p>
<p class="publication"><span class="title">BMC medical genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">1</span>;<span class="pagerange">19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18513384" rel="external">18513384</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2430202" rel="external">2430202</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1755-8794-1-19" rel="external">10.1186/1755-8794-1-19</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Determination and validation of principal gene products.</h4>
<p class="authors"><span class="author">Tress ML</span>, <span class="author">Wesselink JJ</span>, <span class="author">Frankish A</span>, <span class="author">L&oacute;pez G</span>, <span class="author">Goldman N</span>, <span class="author">L&ouml;ytynoja A</span>, <span class="author">Massingham T</span>, <span class="author">Pardi F</span>, <span class="author">Whelan S</span>, <span class="author">Harrow J</span> and <span class="author">Valencia A</span></p>
<p class="affiliation">Structural Biology and Biocomputing Programme, Spanish National Cancer Research Centre, Madrid, Spain. <a href="mailto:%6D%74%72%65%73%73%40%63%6E%69%6F.%65%73">&#109;&#116;&#114;&#101;&#115;&#115;&#64;&#99;&#110;&#105;&#111;&#46;&#101;&#115;</a></p><p class="abstract"><strong>Motivation</strong>: Alternative splicing has the potential to generate a wide range of protein isoforms. For many computational applications and for experimental research, it is important to be able to concentrate on the isoform that retains the core biological function. For many genes this is far from clear.</p><p class="abstract"><strong>Results</strong>: We have combined five methods into a pipeline that allows us to detect the principal variant for a gene. Most of the methods were based on conservation between species, at the level of both gene and protein. The five methods used were the conservation of exonic structure, the detection of non-neutral evolution, the conservation of functional residues, the existence of a known protein structure and the abundance of vertebrate orthologues. The pipeline was able to determine a principal isoform for 83% of a set of well-annotated genes with multiple variants.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">1</span>;<span class="pagerange">11-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18006548">18006548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2734078">2734078</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm547">10.1093/bioinformatics/btm547</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multidirectional cross-species painting illuminates the history of karyotypic evolution in Perissodactyla.</h4>
<p class="authors"><span class="author">Trifonov VA</span>, <span class="author">Stanyon R</span>, <span class="author">Nesterenko AI</span>, <span class="author">Fu B</span>, <span class="author">Perelman PL</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Stone G</span>, <span class="author">Rubtsova NV</span>, <span class="author">Houck ML</span>, <span class="author">Robinson TJ</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Dobigny G</span>, <span class="author">Graphodatsky AS</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Institute of Cytology and Genetics, Russian Academy of Sciences, Novosibirsk, 630090, Russia. <a href="mailto:%76%6C%61%64%40%62%69%6F%6E%65%74.%6E%73%63.%72%75">&#118;&#108;&#97;&#100;&#64;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;</a></p><p class="abstract">The order Perissodactyla, the group of odd-toed ungulates, includes three extant families: Equidae, Tapiridae, and Rhinocerotidae. The extremely rapid karyotypic diversification in perissodactyls has so far prevented the establishment of genome-wide homology maps between these three families by traditional cytogenetic approaches. Here we report the first genome-wide comparative chromosome maps of African rhinoceroses, four tapir species, four equine species, and humans. These maps were established by multidirectional chromosome painting, with paint probes derived from flow-sorted chromosomes of Equus grevyi, Tapirus indicus, and Ceratotherium simum as well as painting probes from horse and human. The Malayan tapir (Tapirus indicus), Baird&#39;s tapir (T. bairdii), mountain tapir (T. pinchaque), lowland tapir (T. terrestris), and onager (E. hemionus onager), were studied by cross-species chromosome painting for the first time. Our results, when integrated with previously published comparative chromosome maps of the other perissodactyl species, have enabled the reconstruction of perissodactyl, ceratomorph, and equid ancestral karyotypes, and the identification of the defining evolutionary chromosomal rearrangements along each lineage. Our results allow a more reliable estimate of the mode and tempo of evolutionary chromosomal rearrangements, revealing a striking switch between the slowly evolving ceratomorphs and extremely rapidly evolving equids.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">89-107</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18293107" rel="external">18293107</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1201-7" rel="external">10.1007/s10577-007-1201-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Burkholderia pseudomallei genome plasticity associated with genomic island variation.</h4>
<p class="authors"><span class="author">Tumapa S</span>, <span class="author">Holden MT</span>, <span class="author">Vesaratchavest M</span>, <span class="author">Wuthiekanun V</span>, <span class="author">Limmathurotsakul D</span>, <span class="author">Chierakul W</span>, <span class="author">Feil EJ</span>, <span class="author">Currie BJ</span>, <span class="author">Day NP</span>, <span class="author">Nierman WC</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Mahidol-Oxford Tropical Medicine Research Unit, Faculty of Tropical Medicine, Mahidol University, Bangkok, Thailand. <a href="mailto:%6A%61%79%40%74%72%6F%70%6D%65%64%72%65%73.%61%63">&#106;&#97;&#121;&#64;&#116;&#114;&#111;&#112;&#109;&#101;&#100;&#114;&#101;&#115;&#46;&#97;&#99;</a> &lt;<a href="mailto:%6A%61%79%40%74%72%6F%70%6D%65%64%72%65%73.%61%63&%67%74">&#106;&#97;&#121;&#64;&#116;&#114;&#111;&#112;&#109;&#101;&#100;&#114;&#101;&#115;&#46;&#97;&#99;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: Burkholderia pseudomallei is a soil-dwelling saprophyte and the cause of melioidosis. Horizontal gene transfer contributes to the genetic diversity of this pathogen and may be an important determinant of virulence potential. The genome contains genomic island (GI) regions that encode a broad array of functions. Although there is some evidence for the variable distribution of genomic islands in B. pseudomallei isolates, little is known about the extent of variation between related strains or their association with disease or environmental survival.</p><p class="abstract"><strong>Results</strong>: Five islands from B. pseudomallei strain K96243 were chosen as representatives of different types of genomic islands present in this strain, and their presence investigated in other B. pseudomallei. In silico analysis of 10 B. pseudomallei genome sequences provided evidence for the variable presence of these regions, together with micro-evolutionary changes that generate GI diversity. The diversity of GIs in 186 isolates from NE Thailand (83 environmental and 103 clinical isolates) was investigated using multiplex PCR screening. The proportion of all isolates positive by PCR ranged from 12% for a prophage-like island (GI 9), to 76% for a metabolic island (GI 16). The presence of each of the five GIs did not differ between environmental and disease-associated isolates (p &gt; 0.05 for all five islands). The cumulative number of GIs per isolate for the 186 isolates ranged from 0 to 5 (median 2, IQR 1 to 3). The distribution of cumulative GI number did not differ between environmental and disease-associated isolates (p = 0.27). The presence of GIs was defined for the three largest clones in this collection (each defined as a single sequence type, ST, by multilocus sequence typing); these were ST 70 (n = 15 isolates), ST 54 (n = 11), and ST 167 (n = 9). The rapid loss and/or acquisition of gene islands was observed within individual clones. Comparisons were drawn between isolates obtained from the environment and from patients with melioidosis in order to examine the role of genomic islands in virulence and clinical associations. There was no reproducible association between the individual or cumulative presence of five GIs and a range of clinical features in 103 patients with melioidosis.</p><p class="abstract"><strong>Conclusion</strong>: Horizontal gene transfer of mobile genetic elements can rapidly alter the gene repertoire of B. pseudomallei. This study confirms the utility of a range of approaches in defining the presence and significance of genomic variation in natural populations of B. pseudomallei.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">190</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18439288" rel="external">18439288</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2386483" rel="external">2386483</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-190" rel="external">10.1186/1471-2164-9-190</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Germline rates of de novo meiotic deletions and duplications causing several genomic disorders.</h4>
<p class="authors"><span class="author">Turner DJ</span>, <span class="author">Miretti M</span>, <span class="author">Rajan D</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span>, <span class="author">Blayney ML</span>, <span class="author">Beck S</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK.</p><p class="abstract">Meiotic recombination between highly similar duplicated sequences (nonallelic homologous recombination, NAHR) generates deletions, duplications, inversions and translocations, and it is responsible for genetic diseases known as &#39;genomic disorders&#39;, most of which are caused by altered copy number of dosage-sensitive genes. NAHR hot spots have been identified within some duplicated sequences. We have developed sperm-based assays to measure the de novo rate of reciprocal deletions and duplications at four NAHR hot spots. We used these assays to dissect the relative rates of NAHR between different pairs of duplicated sequences. We show that (i) these NAHR hot spots are specific to meiosis, (ii) deletions are generated at a higher rate than their reciprocal duplications in the male germline and (iii) some of these genomic disorders are likely to have been underascertained clinically, most notably that resulting from the duplication of 7q11, the reciprocal of the deletion causing Williams-Beuren syndrome.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077014</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">1</span>;<span class="pagerange">90-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18059269">18059269</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2669897">2669897</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2007.40">10.1038/ng.2007.40</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Long-range, high-throughput haplotype determination via haplotype-fusion PCR and ligation haplotyping.</h4>
<p class="authors"><span class="author">Turner DJ</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%64%6A%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#106;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ligation Haplotyping is a robust, novel method for experimental determination of haplotypes over long distances, which can be applied to assaying both sequence and structural variation. The simplicity and efficacy of the method for genotyping large chromosomal rearrangements and haplotyping SNPs over long distances make it a valuable and powerful addition to the methodological repertoire, which will be beneficial to studies of population genetics and evolution, disease association and inheritance, and genomic variation. We illustrate the versatility of the method both by genotyping a Yp paracentric inversion, found in approximately 60% of Northwest European males, that strongly influences the germline rate of infertility-causing XY translocations and by haplotyping two autosomal SNPs that lie 16.4 kb apart on chromosome 7, and which influence an individual&#39;s susceptibility to systemic lupus erythematosus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">13</span>;<span class="pagerange">e82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18562465">18562465</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2490767">2490767</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkn373">10.1093/nar/gkn373</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An evolutionary perspective on Y-chromosomal variation and male infertility.</h4>
<p class="authors"><span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridgeshire, UK. <a href="mailto:%63%74%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#116;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Genetic variation on the Y chromosome is one of the best-documented causes of male infertility, but the genes responsible have still not been identified. This review discusses how an evolutionary perspective may help with interpretation of the data available and suggest novel approaches to identify key genes. Comparison with the chimpanzee Y chromosome indicates that USP9Y is dispensable in apes, but that multiple copies of TSPY1 may have an important role. Comparisons between infertile and control groups in search of genetic susceptibility factors are more complex for the Y chromosome than for the rest of the genome because of population stratification and require unusual levels of confirmation. But the extreme population stratification exhibited by the Y also allows populations particularly suitable for some studies to be identified, such as the partial AZFc deletions common in Northern European populations where further dissection of this complex structural region would be facilitated.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">International journal of andrology</span>&nbsp;<span class="year">2008</span>;<span class="volume">31</span>;<span class="number">4</span>;<span class="pagerange">376-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18399979">18399979</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2628431">2628431</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2605.2008.00889.x">10.1111/j.1365-2605.2008.00889.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss of Rassf1a cooperates with Apc(Min) to accelerate intestinal tumourigenesis.</h4>
<p class="authors"><span class="author">van der Weyden L</span>, <span class="author">Arends MJ</span>, <span class="author">Dovey OM</span>, <span class="author">Harrison HL</span>, <span class="author">Lefebvre G</span>, <span class="author">Conte N</span>, <span class="author">Gergely FV</span>, <span class="author">Bradley A</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Promoter methylation of the RAS-association domain family 1, isoform A gene (RASSF1A) is one of the most frequent events found in human tumours. In this study we set out to test the hypothesis that loss of Rassf1a can cooperate with inactivation of the adenomatous polyposis coli (Apc) gene to accelerate intestinal tumourigenesis using the Apc-Min (Apc(Min/+)) mouse model, as mutational or deletional inactivation of APC is a frequent early event in the genesis of intestinal cancer. Further, loss of RASSF1A has also been reported to occur in premalignant adenomas of the bowel. RASSF1A has been implicated in an array of pivotal cellular processes, including regulation of the cell cycle, apoptosis, microtubule stability and most recently in the beta-catenin signalling pathway. By interbreeding isoform specific Rassf1a knockout mice with Apc(+/Min) mice, we showed that loss of Rassf1a results in a significant increase in adenomas of the small intestine and accelerated intestinal tumourigenesis leading to the earlier death of adenocarcinoma-bearing mice and decreased overall survival. Comparative genomic hybridization of adenomas from Rassf1a(-/-); Apc(+/Min) mice revealed no evidence of aneuploidy or gross chromosomal instability (no difference to adenomas from Rassf1a(+/+); Apc(+/Min) mice). Immunohistochemical analysis of adenomas revealed increased nuclear beta-catenin accumulation in adenomas from Rassf1a(-/-); Apc(+/Min) mice, compared to those from Rassf1a(+/+); Apc(+/Min) mice, but no differences in proliferation marker (Ki67) staining patterns. Collectively these data demonstrate cooperation between inactivation of Rassf1a and Apc resulting in accelerated intestinal tumourigenesis, with adenomas showing increased nuclear accumulation of beta-catenin, supporting a mechanistic link via loss of the known interaction of Rassf1 with beta-TrCP that usually mediates degradation of beta-catenin.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A8449</span>; Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2008</span>;<span class="volume">27</span>;<span class="number">32</span>;<span class="pagerange">4503-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18391979">18391979</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3706934">3706934</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fonc.2008.94">10.1038/onc.2008.94</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Graph clustering via a discrete uncoupling process</h4>
<p class="authors"><span class="author">Van Dongen</span>, <span class="author">S.</span></p>
<p class="publication"><span class="title">Siam Journal on Matrix Analysis and Applications.</span> <span class="year">2008</span>;<span class="volume">30</span>;<span class="pagerange">121-141</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Resolving the structural features of genomic islands: a machine learning approach.</h4>
<p class="authors"><span class="author">Vernikos GS</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Large inserts of horizontally acquired DNA that contain functionally related genes with limited phylogenetic distribution are often referred to as genomic islands (GIs), and structural definitions of these islands, based on common features, have been proposed. Although a large number of mobile elements fall well within the GI definition, there are several concerns about the structural consensus for GIs: The current GI definition was put forward 10 yr ago when only 12 complete bacterial genomes were available, a large number of GIs deviate from that definition, and in silico predictions assuming a full/partial GI structural model bias the sampling of the GI structural space toward &quot;well-structured&quot; GIs. In this study, the structural features of genomic regions are sampled by a hypothesis-free, bottom-up search, and these are exploited in a machine learning approach with the aim of explicitly quantifying and modeling the contribution of each feature to the GI structure. Performing a whole-genome-based comparative analysis between 37 strains of three different genera and 12 outgroup genomes, 668 genomic regions were sampled and used to train structural GI models. The data show that, overall, GIs from the three different genera fall into distinct, genus-specific structural families. However, decreasing the taxa resolution, by studying GI structures across different genus boundaries, provides models that converge on a fairly similar GI structure, further suggesting that GIs can be seen as a superfamily of mobile elements, with core and variable structural features, rather than a well-defined family.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2008</span>;<span class="volume">18</span>;<span class="number">2</span>;<span class="pagerange">331-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18071028" rel="external">18071028</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2203631" rel="external">2203631</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.7004508" rel="external">10.1101/gr.7004508</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution mapping of expression-QTLs yields insight into human gene regulation.</h4>
<p class="authors"><span class="author">Veyrieras JB</span>, <span class="author">Kudaravalli S</span>, <span class="author">Kim SY</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Gilad Y</span>, <span class="author">Stephens M</span> and <span class="author">Pritchard JK</span></p>
<p class="affiliation">Department of Human Genetics, The University of Chicago, Chicago, IL, USA. <a href="mailto:%6A%62.%76%65%79%72%69%65%72%61%73%40%67%6D%61%69%6C.%63%6F%6D">&#106;&#98;&#46;&#118;&#101;&#121;&#114;&#105;&#101;&#114;&#97;&#115;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Recent studies of the HapMap lymphoblastoid cell lines have identified large numbers of quantitative trait loci for gene expression (eQTLs). Reanalyzing these data using a novel Bayesian hierarchical model, we were able to create a surprisingly high-resolution map of the typical locations of sites that affect mRNA levels in cis. Strikingly, we found a strong enrichment of eQTLs in the 250 bp just upstream of the transcription end site (TES), in addition to an enrichment around the transcription start site (TSS). Most eQTLs lie either within genes or close to genes; for example, we estimate that only 5% of eQTLs lie more than 20 kb upstream of the TSS. After controlling for position effects, SNPs in exons are approximately 2-fold more likely than SNPs in introns to be eQTLs. Our results suggest an important role for mRNA stability in determining steady-state mRNA levels, and highlight the potential of eQTL mapping as a high-resolution tool for studying the determinants of gene regulation.</p>
<p class="grants">Funded by: Howard Hughes Medical Institute; NHGRI NIH HHS: <span class="grant">HG002772</span>, <span class="grant">HG02585-01</span>; NIGMS NIH HHS: <span class="grant">GM077959</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">4</span>;<span class="number">10</span>;<span class="pagerange">e1000214</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18846210">18846210</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2556086">2556086</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000214">10.1371/journal.pgen.1000214</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Habitual energy expenditure modifies the association between NOS3 gene polymorphisms and blood pressure.</h4>
<p class="authors"><span class="author">Vimaleswaran KS</span>, <span class="author">Franks PW</span>, <span class="author">Barroso I</span>, <span class="author">Brage S</span>, <span class="author">Ekelund U</span>, <span class="author">Wareham NJ</span> and <span class="author">Loos RJ</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Institute of Metabolic Science, Cambridge, UK. <a href="mailto:%76%6B%73%32%32%40%6D%65%64%73%63%68%6C.%63%61%6D.%61%63.%75%6B">&#118;&#107;&#115;&#50;&#50;&#64;&#109;&#101;&#100;&#115;&#99;&#104;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The endothelial nitric-oxide synthase (NOS3) gene encodes the enzyme (eNOS) that synthesizes the molecule nitric oxide, which facilitates endothelium-dependent vasodilation in response to physical activity. Thus, energy expenditure may modify the association between the genetic variation at NOS3 and blood pressure.</p><p class="abstract"><strong>Methods</strong>: To test this hypothesis, we genotyped 11 NOS3 polymorphisms, capturing all common variations, in 726 men and women from the Medical Research Council (MRC) Ely Study (age (mean +/- s.d.): 55 +/- 10 years, body mass index: 26.4 +/- 4.1 kg/m(2)). Habitual/non-resting energy expenditure (NREE) was assessed via individually calibrated heart rate monitoring over 4 days.</p><p class="abstract"><strong>Results</strong>: The intronic variant, IVS25+15 [G--&gt;A], was significantly associated with blood pressure; GG homozygotes had significantly lower levels of diastolic blood pressure (DBP) (-2.8 mm Hg; P = 0.016) and systolic blood pressure (SBP) (-1.9 mm Hg; P = 0.018) than A-allele carriers. The interaction between NREE and IVS25+15 was also significant for both DBP (P = 0.006) and SBP (P = 0.026), in such a way that the effect of the GG-genotype on blood pressure was stronger in individuals with higher NREE (DBP: -4.9 mm Hg, P = 0.02. SBP: -3.8 mm Hg, P= 0.03 for the third tertile). Similar results were observed when the outcome was dichotomously defined as hypertension.</p><p class="abstract"><strong>Conclusions</strong>: In summary, the NOS3 IVS25+15 is directly associated with blood pressure and hypertension in white Europeans. However, the associations are most evident in the individuals with the highest NREE. These results need further replication and have to be ideally tested in a trial before being informative for targeted disease prevention. Eventually, the selection of individuals for lifestyle intervention programs could be guided by knowledge of genotype.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179473</span>, <span class="grant">MC_U106188470</span>, <span class="grant">U.1061.00.001 (79471)</span>, <span class="grant">U.1061.00.005(79473)</span>; Wellcome Trust: <span class="grant">077016</span>, <span class="grant">087636</span></p>
<p class="publication"><span class="title">American journal of hypertension</span>&nbsp;<span class="year">2008</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">297-302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18246059">18246059</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2714087">2714087</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fajh.2007.69">10.1038/ajh.2007.69</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Gly482Ser genotype at the PPARGC1A gene and elevated blood pressure: a meta-analysis involving 13,949 individuals.</h4>
<p class="authors"><span class="author">Vimaleswaran KS</span>, <span class="author">Luan J</span>, <span class="author">Andersen G</span>, <span class="author">Muller YL</span>, <span class="author">Wheeler E</span>, <span class="author">Brito EC</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Pedersen O</span>, <span class="author">Baier LJ</span>, <span class="author">Knowler WC</span>, <span class="author">Barroso I</span>, <span class="author">Wareham NJ</span>, <span class="author">Loos RJ</span> and <span class="author">Franks PW</span></p>
<p class="affiliation">Department of Public Health &amp; Clinical Medicine, Ume&aring; University Hospital, Ume&aring;, Sweden.</p><p class="abstract">The protein encoded by the PPARGC1A gene is expressed at high levels in metabolically active tissues and is involved in the control of oxidative stress via reactive oxygen species detoxification. Several recent reports suggest that the PPARGC1A Gly482Ser (rs8192678) missense polymorphism may relate inversely with blood pressure. We used conventional meta-analysis methods to assess the association between Gly482Ser and systolic (SBP) or diastolic blood pressures (DBP) or hypertension in 13,949 individuals from 17 studies, of which 6,042 were previously unpublished observations. The studies comprised cohorts of white European, Asian, and American Indian adults, and adolescents from South America. Stratified analyses were conducted to control for population stratification. Pooled genotype frequencies were 0.47 (Gly482Gly), 0.42 (Gly482Ser), and 0.11 (Ser482Ser). We found no evidence of association between Gly482Ser and SBP [Gly482Gly: mean = 131.0 mmHg, 95% confidence interval (CI) = 130.5-131.5 mmHg; Gly482Ser mean = 133.1 mmHg, 95% CI = 132.6-133.6 mmHg; Ser482Ser: mean = 133.5 mmHg, 95% CI = 132.5-134.5 mmHg; P = 0.409] or DBP (Gly482Gly: mean = 80.3 mmHg, 95% CI = 80.0-80.6 mmHg; Gly482Ser mean = 81.5 mmHg, 95% CI = 81.2-81.8 mmHg; Ser482Ser: mean = 82.1 mmHg, 95% CI = 81.5-82.7 mmHg; P = 0.651). Contrary to previous reports, we did not observe significant effect modification by sex (SBP, P = 0.966; DBP, P = 0.715). We were also unable to confirm the previously reported association between the Ser482 allele and hypertension [odds ratio: 0.97, 95% CI = 0.87-1.08, P = 0.585]. These results were materially unchanged when analyses were focused on whites only. However, statistical evidence of gene-age interaction was apparent for DBP [Gly482Gly: 73.5 (72.8, 74.2), Gly482Ser: 77.0 (76.2, 77.8), Ser482Ser: 79.1 (77.4, 80.9), P = 4.20 x 10(-12)] and SBP [Gly482Gly: 121.4 (120.4, 122.5), Gly482Ser: 125.9 (124.6, 127.1), Ser482Ser: 129.2 (126.5, 131.9), P = 7.20 x 10(-12)] in individuals &lt;50 yr (n = 2,511); these genetic effects were absent in those older than 50 yr (n = 5,088) (SBP, P = 0.41; DBP, P = 0.51). Our findings suggest that the PPARGC1A Ser482 allele may be associated with higher blood pressure, but this is only apparent in younger adults.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Journal of applied physiology (Bethesda, Md. : 1985)</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">4</span>;<span class="pagerange">1352-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18467552">18467552</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2576025">2576025</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fjapplphysiol.90423.2008">10.1152/japplphysiol.90423.2008</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Brain gene expression profiles of Cln1 and Cln5 deficient mice unravels common molecular pathways underlying neuronal degeneration in NCL diseases.</h4>
<p class="authors"><span class="author">von Schantz C</span>, <span class="author">Saharinen J</span>, <span class="author">Kopra O</span>, <span class="author">Cooper JD</span>, <span class="author">Gentile M</span>, <span class="author">Hovatta I</span>, <span class="author">Peltonen L</span> and <span class="author">Jalanko A</span></p>
<p class="affiliation">National Public Health Institute and FIMM, Institute for Molecular Medicine, Helsinki, Finland. <a href="mailto:%63%61%72%69%6E%61.%76%6F%6E.%73%63%68%61%6E%74%7A%40%6B%74%6C.%66%69">&#99;&#97;&#114;&#105;&#110;&#97;&#46;&#118;&#111;&#110;&#46;&#115;&#99;&#104;&#97;&#110;&#116;&#122;&#64;&#107;&#116;&#108;&#46;&#102;&#105;</a></p><p class="abstract"><strong>Background</strong>: The neuronal ceroid lipofuscinoses (NCL) are a group of children&#39;s inherited neurodegenerative disorders, characterized by blindness, early dementia and pronounced cortical atrophy. The similar pathological and clinical profiles of the different forms of NCL suggest that common disease mechanisms may be involved. To explore the NCL-associated disease pathology and molecular pathways, we have previously produced targeted knock-out mice for Cln1 and Cln5. Both mouse-models replicate the NCL phenotype and neuropathology; the Cln1-/- model presents with early onset, severe neurodegenerative disease, whereas the Cln5-/- model produces a milder disease with a later onset.</p><p class="abstract"><strong>Results</strong>: Here we have performed quantitative gene expression profiling of the cortex from 1 and 4 month old Cln1-/- and Cln5-/- mice. Combined microarray datasets from both mouse models exposed a common affected pathway: genes regulating neuronal growth cone stabilization display similar aberrations in both models. We analyzed locus specific gene expression and showed regional clustering of Cln1 and three major genes of this pathway, further supporting a close functional relationship between the corresponding gene products; adenylate cyclase-associated protein 1 (Cap1), protein tyrosine phosphatase receptor type F (Ptprf) and protein tyrosine phosphatase 4a2 (Ptp4a2). The evidence from the gene expression data, indicating changes in the growth cone assembly, was substantiated by the immunofluorescence staining patterns of Cln1-/- and Cln5-/- cortical neurons. These primary neurons displayed abnormalities in cytoskeleton-associated proteins actin and beta-tubulin as well as abnormal intracellular distribution of growth cone associated proteins GAP-43, synapsin and Rab3.</p><p class="abstract"><strong>Conclusion</strong>: Our data provide the first evidence for a common molecular pathogenesis behind neuronal degeneration in INCL and vLINCL. Since CLN1 and CLN5 code for proteins with distinct functional roles these data may have implications for other forms of NCLs as well.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="pagerange">146</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18371231" rel="external">18371231</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2323392" rel="external">2323392</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-9-146" rel="external">10.1186/1471-2164-9-146</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Does my genome look big in this?</h4>
<p class="authors"><span class="author">Walker A</span> and <span class="author">Langridge G</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">12</span>;<span class="pagerange">878-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/19008888" rel="external">19008888</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro2044" rel="external">10.1038/nrmicro2044</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Single-cell genomics.</h4>
<p class="authors"><span class="author">Walker A</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2008</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">176-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18283727">18283727</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1862">10.1038/nrmicro1862</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies genes for biomarkers of cardiovascular disease: serum urate and dyslipidemia.</h4>
<p class="authors"><span class="author">Wallace C</span>, <span class="author">Newhouse SJ</span>, <span class="author">Braund P</span>, <span class="author">Zhang F</span>, <span class="author">Tobin M</span>, <span class="author">Falchi M</span>, <span class="author">Ahmadi K</span>, <span class="author">Dobson RJ</span>, <span class="author">Mar&ccedil;ano AC</span>, <span class="author">Hajat C</span>, <span class="author">Burton P</span>, <span class="author">Deloukas P</span>, <span class="author">Brown M</span>, <span class="author">Connell JM</span>, <span class="author">Dominiczak A</span>, <span class="author">Lathrop GM</span>, <span class="author">Webster J</span>, <span class="author">Farrall M</span>, <span class="author">Spector T</span>, <span class="author">Samani NJ</span>, <span class="author">Caulfield MJ</span> and <span class="author">Munroe PB</span></p>
<p class="affiliation">Clinical Pharmacology and The Genome Centre, Barts and The London, Queen Mary&#39;s School of Medicine and Dentistry, London, EC1M 6BQ, UK.</p><p class="abstract">Many common diseases are accompanied by disturbances in biochemical traits. Identifying the genetic determinants could provide novel insights into disease mechanisms and reveal avenues for developing new therapies. Here, we report a genome-wide association analysis for commonly measured serum and urine biochemical traits. As part of the WTCCC, 500,000 SNPs genome wide were genotyped in 1955 hypertensive individuals characterized for 25 serum and urine biochemical traits. For each trait, we assessed association with individual SNPs, adjusting for age, sex, and BMI. Lipid measurements were further examined in a meta-analysis of genome-wide data from a type 2 diabetes scan. The most promising associations were examined in two epidemiological cohorts. We discovered association between serum urate and SLC2A9, a glucose transporter (p = 2 x 10(-15)) and confirmed this in two independent cohorts, GRAPHIC study (p = 9 x 10(-15)) and TwinsUK (p = 8 x 10(-19)). The odds ratio for hyperuricaemia (defined as urate &gt;0.4 mMol/l) is 1.89 (95% CI = 1.36-2.61) per copy of common allele. We also replicated many genes previously associated with serum lipids and found previously recognized association between LDL levels and SNPs close to genes encoding PSRC1 and CELSR2 (p = 1 x 10(-7)). The common allele was associated with a 6% increase in nonfasting serum LDL. This region showed increased association in the meta-analysis (p = 4 x 10(-14)). This finding provides a potential biological mechanism for the recent association of this same allele of the same SNP with increased risk of coronary disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076113/B/04/Z</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">1</span>;<span class="pagerange">139-49</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18179892" rel="external">18179892</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2253977" rel="external">2253977</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2007.11.001" rel="external">10.1016/j.ajhg.2007.11.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCANPS: a web server for iterative protein sequence database searching by dynamic programing, with display in a hierarchical SCOP browser.</h4>
<p class="authors"><span class="author">Walsh TP</span>, <span class="author">Webber C</span>, <span class="author">Searle S</span>, <span class="author">Sturrock SS</span> and <span class="author">Barton GJ</span></p>
<p class="affiliation">College of Life Sciences, University of Dundee, Dundee, UK.</p><p class="abstract">SCANPS performs iterative profile searching similar to PSI-BLAST but with full dynamic programing on each cycle and on-the-fly estimation of significance. This combination gives good sensitivity and selectivity that outperforms PSI-BLAST in domain-searching benchmarks. Although computationally expensive, SCANPS exploits onchip parallelism (MMX and SSE2 instructions on Intel chips) as well as MPI parallelism to give acceptable turnround times even for large databases. A web server developed to run SCANPS searches is now available at http://www.compbio.dundee.ac.uk/www-scanps. The server interface allows a range of different protein sequence databases to be searched including the SCOP database of protein domains. The server provides the user with regularly updated versions of the main protein sequence databases and is backed up by significant computing resources which ensure that searches are performed rapidly. For SCOP searches, the results may be viewed in a new tree-based representation that reflects the structure of the SCOP hierarchy; this aids the user in placing each hit in the context of its SCOP classification and understanding its relationship to other domains in SCOP.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Web Server issue</span>;<span class="pagerange">W25-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18503088" rel="external">18503088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2447745" rel="external">2447745</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkn320" rel="external">10.1093/nar/gkn320</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome wide RNAi screen by time lapse microscopy in order to identify mitotic genes - Computational aspects and challenges</h4>
<p class="authors"><span class="author">Walter</span>, <span class="author">T</span>, <span class="author">Held</span>, <span class="author">M</span>, <span class="author">Neumann</span>, <span class="author">B</span>, <span class="author">Heriche</span>, <span class="author">J. K</span>, <span class="author">Conrad</span>, <span class="author">C</span>, <span class="author">Pepperkok</span>, <span class="author">R</span>, <span class="author">Ellenberg</span>, <span class="author">J.</span></p>
<p class="publication"><span class="title">2008 5th IEEE International Symposium on Biomedical Imaging.</span> <span class="year">2008</span>;<span class="pagerange">328-331</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the antimicrobial peptide attacin loci from Glossina morsitans.</h4>
<p class="authors"><span class="author">Wang J</span>, <span class="author">Hu C</span>, <span class="author">Wu Y</span>, <span class="author">Stuart A</span>, <span class="author">Amemiya C</span>, <span class="author">Berriman M</span>, <span class="author">Toyoda A</span>, <span class="author">Hattori M</span> and <span class="author">Aksoy S</span></p>
<p class="affiliation">Yale University School of Medicine, Department of Epidemiology and Public Health, 60 College Street, New Haven, CT 06510, USA.</p><p class="abstract">The antimicrobial peptide Attacin is an immune effector molecule that can inhibit the growth of gram-negative bacteria. In Glossina morsitans morsitans, which serves as the sole vectors of African trypanosomes, Attacins also play a role in trypanosome resistance, and in maintaining parasite numbers at homeostatic levels in infected individuals. We characterized the attacin encoding loci from a Bacterial Artificial Chromosome (BAC) library. The attacin genes are organized into three clusters. Cluster 1 contains two attacin (attA) genes located in head-to-head orientation, cluster 2 contains two closely related genes (attA and attB) located in a similar transcriptional orientation, and cluster 3 contains a single attacin gene (attD). Coding and transcription regulatory sequences of attA and attB are nearly identical, but differ significantly from attD. Putative AttA and AttB have signal peptide sequences, but lack the pro domain typically present in insect Attacins. Putative AttD lacks both domains. Analysis of attacin cDNA sequences shows polymorphisms that could arise either from allelic variations or from the presence of additional attacin genomic loci. Real time-PCR analysis reveals that attA and attB expression is induced in the fat body of flies per os challenged with Escherichia coli and parasitized with trypanosomes. In the midgut, expression of these attacins is similarly induced following microbial challenge, but reduced in response to parasite infections. Transcription of AttD is significantly less relative to the other two genes, and is preferentially induced in the fat body of parasitized flies. These results indicate that the different attacin genes may be differentially regulated.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG02526</span>, <span class="grant">U01 HG002526-01</span>, <span class="grant">U01 HG002526-02</span>, <span class="grant">U01 HG002526-03</span>; NIAID NIH HHS: <span class="grant">AI51584</span>, <span class="grant">R01 AI051584-05</span>, <span class="grant">R01 AI068932-01A2</span>; Wellcome Trust: <span class="grant">085775</span></p>
<p class="publication"><span class="title">Insect molecular biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">293-302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18477243" rel="external">18477243</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656931" rel="external">2656931</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2583.2008.00805.x" rel="external">10.1111/j.1365-2583.2008.00805.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The diploid genome sequence of an Asian individual.</h4>
<p class="authors"><span class="author">Wang J</span>, <span class="author">Wang W</span>, <span class="author">Li R</span>, <span class="author">Li Y</span>, <span class="author">Tian G</span>, <span class="author">Goodman L</span>, <span class="author">Fan W</span>, <span class="author">Zhang J</span>, <span class="author">Li J</span>, <span class="author">Zhang J</span>, <span class="author">Guo Y</span>, <span class="author">Feng B</span>, <span class="author">Li H</span>, <span class="author">Lu Y</span>, <span class="author">Fang X</span>, <span class="author">Liang H</span>, <span class="author">Du Z</span>, <span class="author">Li D</span>, <span class="author">Zhao Y</span>, <span class="author">Hu Y</span>, <span class="author">Yang Z</span>, <span class="author">Zheng H</span>, <span class="author">Hellmann I</span>, <span class="author">Inouye M</span>, <span class="author">Pool J</span>, <span class="author">Yi X</span>, <span class="author">Zhao J</span>, <span class="author">Duan J</span>, <span class="author">Zhou Y</span>, <span class="author">Qin J</span>, <span class="author">Ma L</span>, <span class="author">Li G</span>, <span class="author">Yang Z</span>, <span class="author">Zhang G</span>, <span class="author">Yang B</span>, <span class="author">Yu C</span>, <span class="author">Liang F</span>, <span class="author">Li W</span>, <span class="author">Li S</span>, <span class="author">Li D</span>, <span class="author">Ni P</span>, <span class="author">Ruan J</span>, <span class="author">Li Q</span>, <span class="author">Zhu H</span>, <span class="author">Liu D</span>, <span class="author">Lu Z</span>, <span class="author">Li N</span>, <span class="author">Guo G</span>, <span class="author">Zhang J</span>, <span class="author">Ye J</span>, <span class="author">Fang L</span>, <span class="author">Hao Q</span>, <span class="author">Chen Q</span>, <span class="author">Liang Y</span>, <span class="author">Su Y</span>, <span class="author">San A</span>, <span class="author">Ping C</span>, <span class="author">Yang S</span>, <span class="author">Chen F</span>, <span class="author">Li L</span>, <span class="author">Zhou K</span>, <span class="author">Zheng H</span>, <span class="author">Ren Y</span>, <span class="author">Yang L</span>, <span class="author">Gao Y</span>, <span class="author">Yang G</span>, <span class="author">Li Z</span>, <span class="author">Feng X</span>, <span class="author">Kristiansen K</span>, <span class="author">Wong GK</span>, <span class="author">Nielsen R</span>, <span class="author">Durbin R</span>, <span class="author">Bolund L</span>, <span class="author">Zhang X</span>, <span class="author">Li S</span>, <span class="author">Yang H</span> and <span class="author">Wang J</span></p>
<p class="affiliation">Beijing Genomics Institute at Shenzhen, Shenzhen 518000, China. <a href="mailto:%77%61%6E%67%6A%40%67%65%6E%6F%6D%69%63%73.%6F%72%67.%63%6E">&#119;&#97;&#110;&#103;&#106;&#64;&#103;&#101;&#110;&#111;&#109;&#105;&#99;&#115;&#46;&#111;&#114;&#103;&#46;&#99;&#110;</a></p><p class="abstract">Here we present the first diploid genome sequence of an Asian individual. The genome was sequenced to 36-fold average coverage using massively parallel sequencing technology. We aligned the short reads onto the NCBI human reference genome to 99.97% coverage, and guided by the reference genome, we used uniquely mapped reads to assemble a high-quality consensus sequence for 92% of the Asian individual&#39;s genome. We identified approximately 3 million single-nucleotide polymorphisms (SNPs) inside this region, of which 13.6% were not in the dbSNP database. Genotyping analysis showed that SNP identification had high accuracy and consistency, indicating the high sequence quality of this assembly. We also carried out heterozygote phasing and haplotype prediction against HapMap CHB and JPT haplotypes (Chinese and Japanese, respectively), sequence comparison with the two available individual genomes (J. D. Watson and J. C. Venter), and structural variation identification. These variations were considered for their potential biological impact. Our sequence data and analyses demonstrate the potential usefulness of next-generation sequencing technologies for personal genomics.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG003229</span>, <span class="grant">R01 HG003229-04</span>; Wellcome Trust: <span class="grant">077192</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2008</span>;<span class="volume">456</span>;<span class="number">7218</span>;<span class="pagerange">60-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18987735" rel="external">18987735</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716080" rel="external">2716080</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature07484" rel="external">10.1038/nature07484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global role for polyadenylation-assisted nuclear RNA degradation in posttranscriptional gene silencing.</h4>
<p class="authors"><span class="author">Wang SW</span>, <span class="author">Stevenson AL</span>, <span class="author">Kearsey SE</span>, <span class="author">Watt S</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Division of Molecular and Genomic Medicine, National Health Research Institutes, 35 Keyan Road, Zhunan Town, Miaoli County 350, Taiwan. <a href="mailto:%73%68%61%6F%77%69%6E%77%61%6E%67%40%6E%68%72%69.%6F%72%67.%74%77">&#115;&#104;&#97;&#111;&#119;&#105;&#110;&#119;&#97;&#110;&#103;&#64;&#110;&#104;&#114;&#105;&#46;&#111;&#114;&#103;&#46;&#116;&#119;</a></p><p class="abstract">Fission yeast Cid14, a component of the TRAMP (Cid14/Trf4-Air1-Mtr4 polyadenylation) complex, polyadenylates nuclear RNA and stimulates degradation by the exosome for RNA quality control. Here, we analyze patterns of global gene expression in cells lacking the Cid14 or the Dis3/Rpr44 subunit of the nuclear exosome. We found that transcripts from many genes induced during meiosis, including key regulators, accumulated in the absence of Cid14 or Dis3. Moreover, our data suggest that additional substrates include transcripts involved in heterochromatin assembly. Mutant cells lacking Cid14 and/or Dis3 accumulate transcripts corresponding to naturally silenced repeat elements within heterochromatic domains, reflecting defects in centromeric gene silencing and derepression of subtelomeric gene expression. We also uncover roles for Cid14 and Dis3 in maintaining the genomic integrity of ribosomal DNA. Our data indicate that polyadenylation-assisted nuclear RNA turnover functions in eliminating a variety of RNA targets to control diverse processes, such as heterochromatic gene silencing, meiotic differentiation, and maintenance of genomic integrity.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">28</span>;<span class="number">2</span>;<span class="pagerange">656-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18025105" rel="external">18025105</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2223409" rel="external">2223409</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.01531-07" rel="external">10.1128/MCB.01531-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal transposition of PiggyBac in mouse embryonic stem cells.</h4>
<p class="authors"><span class="author">Wang W</span>, <span class="author">Lin C</span>, <span class="author">Lu D</span>, <span class="author">Ning Z</span>, <span class="author">Cox T</span>, <span class="author">Melvin D</span>, <span class="author">Wang X</span>, <span class="author">Bradley A</span> and <span class="author">Liu P</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, United Kingdom.</p><p class="abstract">Transposon systems are widely used for generating mutations in various model organisms. PiggyBac (PB) has recently been shown to transpose efficiently in the mouse germ line and other mammalian cell lines. To facilitate PB&#39;s application in mammalian genetics, we characterized the properties of the PB transposon in mouse embryonic stem (ES) cells. We first measured the transposition efficiencies of PB transposon in mouse embryonic stem cells. We next constructed a PB/SB hybrid transposon to compare PB and Sleeping Beauty (SB) transposon systems and demonstrated that PB transposition was inhibited by DNA methylation. The excision and reintegration rates of a single PB from two independent genomic loci were measured and its ability to mutate genes with gene trap cassettes was tested. We examined PB&#39;s integration site distribution in the mouse genome and found that PB transposition exhibited local hopping. The comprehensive information from this study should facilitate further exploration of the potential of PB and SB DNA transposons in mammalian genetics.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">5R21GM079528</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">27</span>;<span class="pagerange">9290-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18579772">18579772</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2440425">2440425</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0801017105">10.1073/pnas.0801017105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome analysis of the platypus reveals unique signatures of evolution.</h4>
<p class="authors"><span class="author">Warren WC</span>, <span class="author">Hillier LW</span>, <span class="author">Marshall Graves JA</span>, <span class="author">Birney E</span>, <span class="author">Ponting CP</span>, <span class="author">Gr&uuml;tzner F</span>, <span class="author">Belov K</span>, <span class="author">Miller W</span>, <span class="author">Clarke L</span>, <span class="author">Chinwalla AT</span>, <span class="author">Yang SP</span>, <span class="author">Heger A</span>, <span class="author">Locke DP</span>, <span class="author">Miethke P</span>, <span class="author">Waters PD</span>, <span class="author">Veyrunes F</span>, <span class="author">Fulton L</span>, <span class="author">Fulton B</span>, <span class="author">Graves T</span>, <span class="author">Wallis J</span>, <span class="author">Puente XS</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span>, <span class="author">Ord&oacute;&ntilde;ez GR</span>, <span class="author">Eichler EE</span>, <span class="author">Chen L</span>, <span class="author">Cheng Z</span>, <span class="author">Deakin JE</span>, <span class="author">Alsop A</span>, <span class="author">Thompson K</span>, <span class="author">Kirby P</span>, <span class="author">Papenfuss AT</span>, <span class="author">Wakefield MJ</span>, <span class="author">Olender T</span>, <span class="author">Lancet D</span>, <span class="author">Huttley GA</span>, <span class="author">Smit AF</span>, <span class="author">Pask A</span>, <span class="author">Temple-Smith P</span>, <span class="author">Batzer MA</span>, <span class="author">Walker JA</span>, <span class="author">Konkel MK</span>, <span class="author">Harris RS</span>, <span class="author">Whittington CM</span>, <span class="author">Wong ES</span>, <span class="author">Gemmell NJ</span>, <span class="author">Buschiazzo E</span>, <span class="author">Vargas Jentzsch IM</span>, <span class="author">Merkel A</span>, <span class="author">Schmitz J</span>, <span class="author">Zemann A</span>, <span class="author">Churakov G</span>, <span class="author">Kriegs JO</span>, <span class="author">Brosius J</span>, <span class="author">Murchison EP</span>, <span class="author">Sachidanandam R</span>, <span class="author">Smith C</span>, <span class="author">Hannon GJ</span>, <span class="author">Tsend-Ayush E</span>, <span class="author">McMillan D</span>, <span class="author">Attenborough R</span>, <span class="author">Rens W</span>, <span class="author">Ferguson-Smith M</span>, <span class="author">Lef&egrave;vre CM</span>, <span class="author">Sharp JA</span>, <span class="author">Nicholas KR</span>, <span class="author">Ray DA</span>, <span class="author">Kube M</span>, <span class="author">Reinhardt R</span>, <span class="author">Pringle TH</span>, <span class="author">Taylor J</span>, <span class="author">Jones RC</span>, <span class="author">Nixon B</span>, <span class="author">Dacheux JL</span>, <span class="author">Niwa H</span>, <span class="author">Sekita Y</span>, <span class="author">Huang X</span>, <span class="author">Stark A</span>, <span class="author">Kheradpour P</span>, <span class="author">Kellis M</span>, <span class="author">Flicek P</span>, <span class="author">Chen Y</span>, <span class="author">Webber C</span>, <span class="author">Hardison R</span>, <span class="author">Nelson J</span>, <span class="author">Hallsworth-Pepin K</span>, <span class="author">Delehaunty K</span>, <span class="author">Markovic C</span>, <span class="author">Minx P</span>, <span class="author">Feng Y</span>, <span class="author">Kremitzki C</span>, <span class="author">Mitreva M</span>, <span class="author">Glasscock J</span>, <span class="author">Wylie T</span>, <span class="author">Wohldmann P</span>, <span class="author">Thiru P</span>, <span class="author">Nhan MN</span>, <span class="author">Pohl CS</span>, <span class="author">Smith SM</span>, <span class="author">Hou S</span>, <span class="author">Nefedov M</span>, <span class="author">de Jong PJ</span>, <span class="author">Renfree MB</span>, <span class="author">Mardis ER</span> and <span class="author">Wilson RK</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University School of Medicine, Campus Box 8501, 4444 Forest Park Avenue, St Louis, Missouri 63108, USA. <a href="mailto:%77%77%61%72%72%65%6E%40%77%75%73%74%6C.%65%64%75">&#119;&#119;&#97;&#114;&#114;&#101;&#110;&#64;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">We present a draft genome sequence of the platypus, Ornithorhynchus anatinus. This monotreme exhibits a fascinating combination of reptilian and mammalian characters. For example, platypuses have a coat of fur adapted to an aquatic lifestyle; platypus females lactate, yet lay eggs; and males are equipped with venom similar to that of reptiles. Analysis of the first monotreme genome aligned these features with genetic innovations. We find that reptile and platypus venom proteins have been co-opted independently from the same gene families; milk protein genes are conserved despite platypuses laying eggs; and immune gene family expansions are directly related to platypus biology. Expansions of protein, non-protein-coding RNA and microRNA families, as well as repeat elements, are identified. Sequencing of this genome now provides a valuable resource for deep mammalian comparative analyses, as well as for monotreme biology and conservation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U137761446</span>; NCI NIH HHS: <span class="grant">P01 CA013106</span>, <span class="grant">P01 CA013106-37</span>; NHGRI NIH HHS: <span class="grant">HG002238</span>, <span class="grant">R01 HG004037</span>, <span class="grant">R01 HG004037-02</span>, <span class="grant">R01HG02385</span>; NIGMS NIH HHS: <span class="grant">R01 GM59290</span>; Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2008</span>;<span class="volume">453</span>;<span class="number">7192</span>;<span class="pagerange">175-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18464734">18464734</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2803040">2803040</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06936">10.1038/nature06936</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rapid Virulence Annotation (RVA): identification of virulence factors using a bacterial genome library and multiple invertebrate hosts.</h4>
<p class="authors"><span class="author">Waterfield NR</span>, <span class="author">Sanchez-Contreras M</span>, <span class="author">Eleftherianos I</span>, <span class="author">Dowling A</span>, <span class="author">Yang G</span>, <span class="author">Wilkinson P</span>, <span class="author">Parkhill J</span>, <span class="author">Thomson N</span>, <span class="author">Reynolds SE</span>, <span class="author">Bode HB</span>, <span class="author">Dorus S</span> and <span class="author">Ffrench-Constant RH</span></p>
<p class="affiliation">Department of Biology and Biochemistry, University of Bath, Bath BA2 7AY, United Kingdom. <a href="mailto:%62%73%73%6E%77%40%62%61%74%68.%61%63.%75%6B">&#98;&#115;&#115;&#110;&#119;&#64;&#98;&#97;&#116;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Current sequence databases now contain numerous whole genome sequences of pathogenic bacteria. However, many of the predicted genes lack any functional annotation. We describe an assumption-free approach, Rapid Virulence Annotation (RVA), for the high-throughput parallel screening of genomic libraries against four different taxa: insects, nematodes, amoeba, and mammalian macrophages. These hosts represent different aspects of both the vertebrate and invertebrate immune system. Here, we apply RVA to the emerging human pathogen Photorhabdus asymbiotica using &quot;gain of toxicity&quot; assays of recombinant Escherichia coli clones. We describe a wealth of potential virulence loci and attribute biological function to several putative genomic islands, which may then be further characterized using conventional molecular techniques. The application of RVA to other pathogen genomes promises to ascribe biological function to otherwise uncharacterized virulence genes.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2008</span>;<span class="volume">105</span>;<span class="number">41</span>;<span class="pagerange">15967-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18838673" rel="external">18838673</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2572985" rel="external">2572985</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0711114105" rel="external">10.1073/pnas.0711114105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">urg1: a uracil-regulatable promoter system for fission yeast with short induction and repression times.</h4>
<p class="authors"><span class="author">Watt S</span>, <span class="author">Mata J</span>, <span class="author">L&oacute;pez-Maury L</span>, <span class="author">Marguerat S</span>, <span class="author">Burns G</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research United Kingdom Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract"><strong>Background</strong>: The fission yeast Schizosaccharomyces pombe is a popular genetic model organism with powerful experimental tools. The thiamine-regulatable nmt1 promoter and derivatives, which take &gt;15 hours for full induction, are most commonly used for controlled expression of ectopic genes. Given the short cell cycle of fission yeast, however, a promoter system that can be rapidly regulated, similar to the GAL system for budding yeast, would provide a key advantage for many experiments.</p><p class="abstract">We used S. pombe microarrays to identify three neighbouring genes (urg1, urg2, and urg3) whose transcript levels rapidly and strongly increased in response to uracil, a condition which otherwise had little effect on global gene expression. We cloned the promoter of urg1 (uracil-regulatable gene) to create several PCR-based gene targeting modules for replacing native promoters with the urg1 promoter (Purg1) in the normal chromosomal locations of genes of interest. The kanMX6 and natMX6 markers allow selection under urg1 induced and repressed conditions, respectively. Some modules also allow N-terminal tagging of gene products placed under urg1 control. Using pom1 as a proof-of-principle, we observed a maximal increase of Purg1-pom1 transcripts after uracil addition within less than 30 minutes, and a similarly rapid decrease after uracil removal. The induced and repressed transcriptional states remained stable over 24-hour periods. RT-PCR comparisons showed that both induced and repressed Purg1-pom1 transcript levels were lower than corresponding P3nmt1-pom1 levels (wild-type nmt1 promoter) but higher than P81nmt1-pom1 levels (weak nmt1 derivative).</p><p class="abstract">We exploited the urg1 promoter system to rapidly induce pom1 expression at defined cell-cycle stages, showing that ectopic pom1 expression leads to cell branching in G2-phase but much less so in G1-phase. The high temporal resolution provided by the urg1 promoter should facilitate experimental design and improve the genetic toolbox for the fission yeast community.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2008</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">e1428</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18197241">18197241</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2174524">2174524</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0001428">10.1371/journal.pone.0001428</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association analysis identifies 20 loci that influence adult height.</h4>
<p class="authors"><span class="author">Weedon MN</span>, <span class="author">Lango H</span>, <span class="author">Lindgren CM</span>, <span class="author">Wallace C</span>, <span class="author">Evans DM</span>, <span class="author">Mangino M</span>, <span class="author">Freathy RM</span>, <span class="author">Perry JR</span>, <span class="author">Stevens S</span>, <span class="author">Hall AS</span>, <span class="author">Samani NJ</span>, <span class="author">Shields B</span>, <span class="author">Prokopenko I</span>, <span class="author">Farrall M</span>, <span class="author">Dominiczak A</span>, <span class="author">Diabetes Genetics Initiative</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Johnson T</span>, <span class="author">Bergmann S</span>, <span class="author">Beckmann JS</span>, <span class="author">Vollenweider P</span>, <span class="author">Waterworth DM</span>, <span class="author">Mooser V</span>, <span class="author">Palmer CN</span>, <span class="author">Morris AD</span>, <span class="author">Ouwehand WH</span>, <span class="author">Cambridge GEM Consortium</span>, <span class="author">Zhao JH</span>, <span class="author">Li S</span>, <span class="author">Loos RJ</span>, <span class="author">Barroso I</span>, <span class="author">Deloukas P</span>, <span class="author">Sandhu MS</span>, <span class="author">Wheeler E</span>, <span class="author">Soranzo N</span>, <span class="author">Inouye M</span>, <span class="author">Wareham NJ</span>, <span class="author">Caulfield M</span>, <span class="author">Munroe PB</span>, <span class="author">Hattersley AT</span>, <span class="author">McCarthy MI</span> and <span class="author">Frayling TM</span></p>
<p class="affiliation">Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK.</p><p class="abstract">Adult height is a model polygenic trait, but there has been limited success in identifying the genes underlying its normal variation. To identify genetic variants influencing adult human height, we used genome-wide association data from 13,665 individuals and genotyped 39 variants in an additional 16,482 samples. We identified 20 variants associated with adult height (P &lt; 5 x 10(-7), with 10 reaching P &lt; 1 x 10(-10)). Combined, the 20 SNPs explain approximately 3% of height variation, with a approximately 5 cm difference between the 6.2% of people with 17 or fewer &#39;tall&#39; alleles compared to the 5.5% with 27 or more &#39;tall&#39; alleles. The loci we identified implicate genes in Hedgehog signaling (IHH, HHIP, PTCH1), extracellular matrix (EFEMP1, ADAMTSL3, ACAN) and cancer (CDK6, HMGA2, DLEU7) pathways, and provide new insights into human growth and developmental processes. Finally, our results provide insights into the genetic architecture of a classic quantitative trait.</p>
<p class="grants">Funded by: British Heart Foundation: <span class="grant">FS/05/061/19501</span>, <span class="grant">PG/02/128/14470</span>, <span class="grant">PG02/128</span>; Medical Research Council: <span class="grant">G0600705</span>, <span class="grant">G0701863</span>, <span class="grant">G9521010</span>, <span class="grant">G9521010(63660)</span>, <span class="grant">G9521010D</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">5</span>;<span class="pagerange">575-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18391952">18391952</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2681221">2681221</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.121">10.1038/ng.121</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association of the timing of puberty with a chromosome 2 locus.</h4>
<p class="authors"><span class="author">Wehkalampi K</span>, <span class="author">Wid&eacute;n E</span>, <span class="author">Laine T</span>, <span class="author">Palotie A</span> and <span class="author">Dunkel L</span></p>
<p class="affiliation">Hospital for Children and Adolescents, Helsinki University Hospital, 00029 Helsinki, Finland.</p><p class="abstract"><strong>Context</strong>: Twin studies indicate that the timing of pubertal onset is under strong genetic control. However, genes controlling pubertal timing in the general population have not yet been identified.</p><p class="abstract"><strong>Objective</strong>: To facilitate the identification of genes influencing the timing of pubertal growth and maturation, we conducted linkage mapping of constitutional delay of growth and puberty (CDGP), an extreme variant of normal pubertal timing, in extended families.</p><p class="abstract">Fifty-two families multiply affected with CDGP were genotyped with 383 multiallelic markers. CDGP was defined based on growth charts (the age at onset of growth spurt, peak height velocity, or attaining adult height taking place at least 1.5 sd later than average). Chromosomal regions cosegregating with CDGP were identified with parametric affected-only linkage analysis using CDGP as a dichotomized trait.</p><p class="abstract"><strong>Results</strong>: The genome-wide scan detected linkage of CDGP to a region on chromosome 2p13-2q13. The two-point heterogeneity LOD (HLOD) score was 1.62 (alpha = 0.27), and the corresponding multipoint HLOD was 2.54 (alpha = 0.31). Fine-mapping the region at 1 cM resolution increased the multipoint HLOD score to 4.44 (alpha = 0.41). The linkage became weaker if family members diagnosed with CDGP without growth data were also included in the analyses.</p><p class="abstract"><strong>Conclusions</strong>: The pericentromeric region of chromosome 2 harbors a gene predisposing to pubertal delay in multiply affected pedigrees. Our data suggest that this locus may be a component of the internal clock controlling the timing of the onset of puberty.</p>
<p class="publication"><span class="title">The Journal of clinical endocrinology and metabolism</span>&nbsp;<span class="year">2008</span>;<span class="volume">93</span>;<span class="number">12</span>;<span class="pagerange">4833-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18812480" rel="external">18812480</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2685475" rel="external">2685475</a>; DOI: <a href="http://dx.doi.org/10.1210%2Fjc.2008-0882" rel="external">10.1210/jc.2008-0882</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">How degrading: Cyp26s in hindbrain development.</h4>
<p class="authors"><span class="author">White RJ</span> and <span class="author">Schilling TF</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The vitamin A derivative retinoic acid performs many functions in vertebrate development and is thought to act as a diffusible morphogen that patterns the anterior-posterior axis of the hindbrain. Recent work in several systems has led to insights into how the spatial distribution of retinoic acid is regulated. These have shown local control of synthesis and degradation, and computational models suggest that degradation by the Cyp26 enzymes plays a critical role in the formation of a morphogen gradient as well as its ability to compensate for fluctuations in RA levels.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R13 HD057707-01A1</span>; NIDCR NIH HHS: <span class="grant">R01 DE013828-07</span>, <span class="grant">R01 DE013828-08</span>; NIGMS NIH HHS: <span class="grant">P50 GM076516-01A1</span>; NINDS NIH HHS: <span class="grant">NS-41353</span>, <span class="grant">R01 NS041353-06</span>, <span class="grant">R01 NS041353-07</span>, <span class="grant">R01 NS041353-08</span></p>
<p class="publication"><span class="title">Developmental dynamics : an official publication of the American Association of Anatomists</span>&nbsp;<span class="year">2008</span>;<span class="volume">237</span>;<span class="number">10</span>;<span class="pagerange">2775-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18816852" rel="external">18816852</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2880953" rel="external">2880953</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvdy.21695" rel="external">10.1002/dvdy.21695</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Feedback circuit among INK4 tumor suppressors constrains human glioblastoma development.</h4>
<p class="authors"><span class="author">Wiedemeyer R</span>, <span class="author">Brennan C</span>, <span class="author">Heffernan TP</span>, <span class="author">Xiao Y</span>, <span class="author">Mahoney J</span>, <span class="author">Protopopov A</span>, <span class="author">Zheng H</span>, <span class="author">Bignell G</span>, <span class="author">Furnari F</span>, <span class="author">Cavenee WK</span>, <span class="author">Hahn WC</span>, <span class="author">Ichimura K</span>, <span class="author">Collins VP</span>, <span class="author">Chu GC</span>, <span class="author">Stratton MR</span>, <span class="author">Ligon KL</span>, <span class="author">Futreal PA</span> and <span class="author">Chin L</span></p>
<p class="affiliation">Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02115, USA.</p><p class="abstract">We have developed a nonheuristic genome topography scan (GTS) algorithm to characterize the patterns of genomic alterations in human glioblastoma (GBM), identifying frequent p18(INK4C) and p16(INK4A) codeletion. Functional reconstitution of p18(INK4C) in GBM cells null for both p16(INK4A) and p18(INK4C) resulted in impaired cell-cycle progression and tumorigenic potential. Conversely, RNAi-mediated depletion of p18(INK4C) in p16(INK4A)-deficient primary astrocytes or established GBM cells enhanced tumorigenicity in vitro and in vivo. Furthermore, acute suppression of p16(INK4A) in primary astrocytes induced a concomitant increase in p18(INK4C). Together, these findings uncover a feedback regulatory circuit in the astrocytic lineage and demonstrate a bona fide tumor suppressor role for p18(INK4C) in human GBM wherein it functions cooperatively with other INK4 family members to constrain inappropriate proliferation.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01 CA095616-060004</span>, <span class="grant">P01 CA095616-070004</span>, <span class="grant">P01CA95616</span>, <span class="grant">R01 CA099041-05</span>, <span class="grant">R01CA99041</span>; NIAMS NIH HHS: <span class="grant">T32 AR007098</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2008</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">355-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18394558" rel="external">18394558</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2292238" rel="external">2292238</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ccr.2008.02.010" rel="external">10.1016/j.ccr.2008.02.010</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution.</h4>
<p class="authors"><span class="author">Wilhelm BT</span>, <span class="author">Marguerat S</span>, <span class="author">Watt S</span>, <span class="author">Schubert F</span>, <span class="author">Wood V</span>, <span class="author">Goodhead I</span>, <span class="author">Penkett CJ</span>, <span class="author">Rogers J</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Recent data from several organisms indicate that the transcribed portions of genomes are larger and more complex than expected, and that many functional properties of transcripts are based not on coding sequences but on regulatory sequences in untranslated regions or non-coding RNAs. Alternative start and polyadenylation sites and regulation of intron splicing add additional dimensions to the rich transcriptional output. This transcriptional complexity has been sampled mainly using hybridization-based methods under one or few experimental conditions. Here we applied direct high-throughput sequencing of complementary DNAs (RNA-Seq), supplemented with data from high-density tiling arrays, to globally sample transcripts of the fission yeast Schizosaccharomyces pombe, independently from available gene annotations. We interrogated transcriptomes under multiple conditions, including rapid proliferation, meiotic differentiation and environmental stress, as well as in RNA processing mutants to reveal the dynamic plasticity of the transcriptional landscape as a function of environmental, developmental and genetic factors. High-throughput sequencing proved to be a powerful and quantitative method to sample transcriptomes deeply at maximal resolution. In contrast to hybridization, sequencing showed little, if any, background noise and was sensitive enough to detect widespread transcription in &gt;90% of the genome, including traces of RNAs that were not robustly transcribed or rapidly degraded. The combined sequencing and strand-specific array data provide rich condition-specific information on novel, mostly non-coding transcripts, untranslated regions and gene structures, thus improving the existing genome annotation. Sequence reads spanning exon-exon or exon-intron junctions give unique insight into a surprising variability in splicing efficiency across introns, genes and conditions. Splicing efficiency was largely coordinated with transcript levels, and increased transcription led to increased splicing in test genes. Hundreds of introns showed such regulated splicing during cellular proliferation or differentiation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>, <span class="grant">C9546/A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2008</span>;<span class="volume">453</span>;<span class="number">7199</span>;<span class="pagerange">1239-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18488015">18488015</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature07002">10.1038/nature07002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reliability of the long case.</h4>
<p class="authors"><span class="author">Wilkinson TJ</span>, <span class="author">Campbell PJ</span> and <span class="author">Judd SJ</span></p>
<p class="affiliation">Department of Medicine, Christchurch School of Medicine and Health Sciences, University of Otago, Dunedin, New Zealand. <a href="mailto:%74%69%6D.%77%69%6C%6B%69%6E%73%6F%6E%40%6F%74%61%67%6F.%61%63.%6E%7A">&#116;&#105;&#109;&#46;&#119;&#105;&#108;&#107;&#105;&#110;&#115;&#111;&#110;&#64;&#111;&#116;&#97;&#103;&#111;&#46;&#97;&#99;&#46;&#110;&#122;</a></p><p class="abstract"><strong>Objectives</strong>: The use of long cases for summative assessment of clinical competence is limited by concerns about unreliability. This study aims to explore the reliability of long cases and how reliability is affected by supplementation with short cases.</p><p class="abstract"><strong>Methods</strong>: We performed a statistical analysis of examinations held by the Royal Australasian College of Physicians in 2005 and 2006 to determine overall reliability and sources of variance in reliability according to candidate ability, case difficulty and inter-examiner differences.</p><p class="abstract"><strong>Results</strong>: Scores for 546 long cases in 2005 and 773 long cases in 2006 were analysed. In 2006, 38% of the total variation in long case data was explained by variation in candidate ability, with other significant contributors to variance being candidate x case and candidate x examiner interactions. Similar figures were found for the 2005 examinations. A short case is less reliable than a long case, but when examiner time is taken into account, three short cases are as reliable as one long case. Any combination of short and long cases would require 4-5 hours of testing time in order to achieve dependability &gt; 0.7.</p><p class="abstract"><strong>Conclusions</strong>: Long cases can be optimised for reliability but time limits their use as the sole tool in a high-stakes examination. Further examiner training, better case selection, or greater use of short cases would have minimal impact on reliability. Reliability can be improved by either increasing examination time or including additional methods of summative assessment, such as might be provided by workplace assessment.</p>
<p class="publication"><span class="title">Medical education</span>&nbsp;<span class="year">2008</span>;<span class="volume">42</span>;<span class="number">9</span>;<span class="pagerange">887-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18715486" rel="external">18715486</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2923.2008.03129.x" rel="external">10.1111/j.1365-2923.2008.03129.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic resources and microarrays for the common carp Cyprinus carpio L</h4>
<p class="authors"><span class="author">Williams</span>, <span class="author">D. R.</span>, <span class="author">Li</span>, <span class="author">W.</span>, <span class="author">Hughes</span>, <span class="author">M. A.</span>, <span class="author">Gonzalez</span>, <span class="author">S. F.</span>, <span class="author">Vernon</span>, <span class="author">C.</span>, <span class="author">Vidal</span>, <span class="author">M. C.</span>, <span class="author">Jeney</span>, <span class="author">Z.</span>, <span class="author">Jeney</span>, <span class="author">G.</span>, <span class="author">Dixon</span>, <span class="author">P.</span>, <span class="author">McAndrew</span>, <span class="author">B.</span>, <span class="author">Bartfai</span>, <span class="author">R.</span>, <span class="author">Orban</span>, <span class="author">L.</span>, <span class="author">Trudeau</span>, <span class="author">V.</span>, <span class="author">Rogers</span>, <span class="author">J.</span>, <span class="author">Matthews</span>, <span class="author">L.</span>, <span class="author">Fraser</span>, <span class="author">E. J.</span>, <span class="author">Gracey</span>, <span class="author">A. Y.</span>, <span class="author">Cossins</span>, <span class="author">A. R.</span></p>
<p class="publication"><span class="title">J Fish Biol.</span> <span class="year">2008</span>;<span class="volume">72</span>;<span class="pagerange">2095-2117</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The vertebrate genome annotation (Vega) database.</h4>
<p class="authors"><span class="author">Wilming LG</span>, <span class="author">Gilbert JG</span>, <span class="author">Howe K</span>, <span class="author">Trevanion S</span>, <span class="author">Hubbard T</span> and <span class="author">Harrow JL</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%6C%77%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#119;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Vertebrate Genome Annotation (Vega) database (http://vega.sanger.ac.uk) was first made public in 2004 and has been designed to view manual annotation of human, mouse and zebrafish genomic sequences produced at the Wellcome Trust Sanger Institute. Since its initial release, the number of human annotated loci has more than doubled to close to 33 000 and now contains comprehensive annotation on 20 of the 24 human chromosomes, four whole mouse chromosomes and around 40% of the zebrafish Danio rerio genome. In addition, we offer manual annotation of a number of haplotype regions in mouse and human and regions of comparative interest in pig and dog that are unique to Vega.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG004555</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2008</span>;<span class="volume">36</span>;<span class="number">Database issue</span>;<span class="pagerange">D753-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18003653">18003653</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2238886">2238886</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm987">10.1093/nar/gkm987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence differentiation in regions identified by a genome scan for local adaptation.</h4>
<p class="authors"><span class="author">Wood HM</span>, <span class="author">Grahame JW</span>, <span class="author">Humphray S</span>, <span class="author">Rogers J</span> and <span class="author">Butlin RK</span></p>
<p class="affiliation">Institute of Integrative and Comparative Biology, University of Leeds, Leeds LS29JT, UK.</p><p class="abstract">Genome scans using large numbers of randomly selected markers have revealed a small proportion of loci that deviate from neutral expectations and so may mark genomic regions that contribute to local adaptation. Measurements of sequence differentiation and identification of genes in these regions is important but difficult, especially in organisms with limited genetic information available. We have followed up a genome scan in the marine gastropod, Littorina saxatilis, by searching a bacterial artificial chromosome library with differentiated and undifferentiated markers, sequencing four bacterial artificial chromosomes and then analysing sequence variation in population samples for fragments at, and close to the original marker polymorphisms. We show that sequence differentiation follows the patterns expected from the original marker frequencies, that differentiated markers identify independent and highly localized sites and that these sites fall outside coding regions. Two differentiated loci are characterized by insertions of putative transposable elements that appear to have increased in frequency recently and which might influence expression of downstream genes. These results provide strong candidate loci for the study of local adaptation in Littorina. They demonstrate an approach that can be applied to follow up genome scans in other taxa and they show that the genome scan approach can lead rapidly to candidate genes in nonmodel organisms.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council</p>
<p class="publication"><span class="title">Molecular ecology</span>&nbsp;<span class="year">2008</span>;<span class="volume">17</span>;<span class="number">13</span>;<span class="pagerange">3123-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18510587">18510587</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-294X.2008.03755.x">10.1111/j.1365-294X.2008.03755.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Databases, data tombs and dust in the wind.</h4>
<p class="authors"><span class="author">Wren JD</span> and <span class="author">Bateman A</span></p><p class="abstract">As biomedical data accumulates, the need to store, share and organize it grows. Consequently, the number of Internet-accessible databases has been rapidly growing on an annual basis. Bioinformatics regularly publishes descriptions of biomedically relevant databases, Nucleic Acids Research has published an annual database issue since 1996 and now a new open-access journal, Database: The Journal of Biological Databases and Curation, will soon be launched by Oxford University Press in 2009 (http://www.oxfordjournals.org/our_journals/databa/). Since databases can be made publicly available on the Internet without publication, it is worth considering what factors prioritize publication of database descriptions in a peer-reviewed journal. In general, publication of a database description in a journal advertises it as a valuable resource for scientific research. Implicitly, it is assumed that this resource is publicly available (most likely for free) and will be maintained. However, therein lies the problem: Database papers are simply not of the same nature as regular research articles. Over time, some databases simply become inaccessible, some are created but not maintained or updated, and some databases are never used (Galperin, 2006). Thus, for database creators, reviewers and journal editors, there are several additional considerations to judge, prior to publication, how potentially valuable these new databases may be.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2008</span>;<span class="volume">24</span>;<span class="number">19</span>;<span class="pagerange">2127-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18819940" rel="external">18819940</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtn464" rel="external">10.1093/bioinformatics/btn464</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation of the oxytocin/neurophysin I (OXT) gene in four human populations.</h4>
<p class="authors"><span class="author">Xu Y</span>, <span class="author">Xue Y</span>, <span class="author">Asan</span>, <span class="author">Daly A</span>, <span class="author">Wu L</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs., CB10 1SA, UK.</p><p class="abstract">Oxytocin is a short peptide with multiple functions in human biology and has been implicated in autism. We aimed to determine the normal pattern of variation around the oxytocin gene and resequenced it and its flanking regions in 91 individuals from four HapMap populations and one chimpanzee. We identified 14 single nucleotide polymorphisms (SNPs), all noncoding, including eight that were novel. Population genetic analyses were largely consistent with a neutral evolutionary history, but an Hudson-Kreitman-Aguad&eacute; (HKA) test revealed more variation within the human population than expected from the level of chimpanzee-human divergence.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">53</span>;<span class="number">7</span>;<span class="pagerange">637-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18566739">18566739</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680061">2680061</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10038-008-0292-0">10.1007/s10038-008-0292-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adaptive evolution of UGT2B17 copy-number variation.</h4>
<p class="authors"><span class="author">Xue Y</span>, <span class="author">Sun D</span>, <span class="author">Daly A</span>, <span class="author">Yang F</span>, <span class="author">Zhou X</span>, <span class="author">Zhao M</span>, <span class="author">Huang N</span>, <span class="author">Zerjal T</span>, <span class="author">Lee C</span>, <span class="author">Carter NP</span>, <span class="author">Hurles ME</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK.</p><p class="abstract">The human UGT2B17 gene varies in copy number from zero to two per individual and also differs in mean number between populations from Africa, Europe, and East Asia. We show that such a high degree of geographical variation is unusual and investigate its evolutionary history. This required first reinterpreting the reference sequence in this region of the genome, which is misassembled from the two different alleles separated by an artifactual gap. A corrected assembly identifies the polymorphism as a 117 kb deletion arising by nonallelic homologous recombination between approximately 4.9 kb segmental duplications and allows the deletion breakpoint to be identified. We resequenced approximately 12 kb of DNA spanning the breakpoint in 91 humans from three HapMap and one extended HapMap populations and one chimpanzee. Diversity was unusually high and the time to the most recent common ancestor was estimated at approximately 2.4 or approximately 3.0 million years by two different methods, with evidence of balancing selection in Europe. In contrast, diversity was low in East Asia where a single haplotype predominated, suggesting positive selection for the deletion in this part of the world.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">83</span>;<span class="number">3</span>;<span class="pagerange">337-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18760392">18760392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2556428">2556428</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.08.004">10.1016/j.ajhg.2008.08.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y-chromosomal diversity in Lebanon is structured by recent historical events.</h4>
<p class="authors"><span class="author">Zalloua PA</span>, <span class="author">Xue Y</span>, <span class="author">Khalife J</span>, <span class="author">Makhoul N</span>, <span class="author">Debiane L</span>, <span class="author">Platt DE</span>, <span class="author">Royyuru AK</span>, <span class="author">Herrera RJ</span>, <span class="author">Hernanz DF</span>, <span class="author">Blue-Smith J</span>, <span class="author">Wells RS</span>, <span class="author">Comas D</span>, <span class="author">Bertranpetit J</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Genographic Consortium</span></p>
<p class="affiliation">The Lebanese American University, Chouran, Beirut 1102 2801, Lebanon.</p><p class="abstract">Lebanon is an eastern Mediterranean country inhabited by approximately four million people with a wide variety of ethnicities and religions, including Muslim, Christian, and Druze. In the present study, 926 Lebanese men were typed with Y-chromosomal SNP and STR markers, and unusually, male genetic variation within Lebanon was found to be more strongly structured by religious affiliation than by geography. We therefore tested the hypothesis that migrations within historical times could have contributed to this situation. Y-haplogroup J*(xJ2) was more frequent in the putative Muslim source region (the Arabian Peninsula) than in Lebanon, and it was also more frequent in Lebanese Muslims than in Lebanese non-Muslims. Conversely, haplogroup R1b was more frequent in the putative Christian source region (western Europe) than in Lebanon and was also more frequent in Lebanese Christians than in Lebanese non-Christians. The most common R1b STR-haplotype in Lebanese Christians was otherwise highly specific for western Europe and was unlikely to have reached its current frequency in Lebanese Christians without admixture. We therefore suggest that the Islamic expansion from the Arabian Peninsula beginning in the seventh century CE introduced lineages typical of this area into those who subsequently became Lebanese Muslims, whereas the Crusader activity in the 11(th)-13(th) centuries CE introduced western European lineages into Lebanese Christians.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">82</span>;<span class="number">4</span>;<span class="pagerange">873-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18374297">18374297</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2427286">2427286</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ajhg.2008.01.020">10.1016/j.ajhg.2008.01.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes.</h4>
<p class="authors"><span class="author">Zeggini E</span>, <span class="author">Scott LJ</span>, <span class="author">Saxena R</span>, <span class="author">Voight BF</span>, <span class="author">Marchini JL</span>, <span class="author">Hu T</span>, <span class="author">de Bakker PI</span>, <span class="author">Abecasis GR</span>, <span class="author">Almgren P</span>, <span class="author">Andersen G</span>, <span class="author">Ardlie K</span>, <span class="author">Bostr&ouml;m KB</span>, <span class="author">Bergman RN</span>, <span class="author">Bonnycastle LL</span>, <span class="author">Borch-Johnsen K</span>, <span class="author">Burtt NP</span>, <span class="author">Chen H</span>, <span class="author">Chines PS</span>, <span class="author">Daly MJ</span>, <span class="author">Deodhar P</span>, <span class="author">Ding CJ</span>, <span class="author">Doney AS</span>, <span class="author">Duren WL</span>, <span class="author">Elliott KS</span>, <span class="author">Erdos MR</span>, <span class="author">Frayling TM</span>, <span class="author">Freathy RM</span>, <span class="author">Gianniny L</span>, <span class="author">Grallert H</span>, <span class="author">Grarup N</span>, <span class="author">Groves CJ</span>, <span class="author">Guiducci C</span>, <span class="author">Hansen T</span>, <span class="author">Herder C</span>, <span class="author">Hitman GA</span>, <span class="author">Hughes TE</span>, <span class="author">Isomaa B</span>, <span class="author">Jackson AU</span>, <span class="author">J&oslash;rgensen T</span>, <span class="author">Kong A</span>, <span class="author">Kubalanza K</span>, <span class="author">Kuruvilla FG</span>, <span class="author">Kuusisto J</span>, <span class="author">Langenberg C</span>, <span class="author">Lango H</span>, <span class="author">Lauritzen T</span>, <span class="author">Li Y</span>, <span class="author">Lindgren CM</span>, <span class="author">Lyssenko V</span>, <span class="author">Marvelle AF</span>, <span class="author">Meisinger C</span>, <span class="author">Midthjell K</span>, <span class="author">Mohlke KL</span>, <span class="author">Morken MA</span>, <span class="author">Morris AD</span>, <span class="author">Narisu N</span>, <span class="author">Nilsson P</span>, <span class="author">Owen KR</span>, <span class="author">Palmer CN</span>, <span class="author">Payne F</span>, <span class="author">Perry JR</span>, <span class="author">Pettersen E</span>, <span class="author">Platou C</span>, <span class="author">Prokopenko I</span>, <span class="author">Qi L</span>, <span class="author">Qin L</span>, <span class="author">Rayner NW</span>, <span class="author">Rees M</span>, <span class="author">Roix JJ</span>, <span class="author">Sandbaek A</span>, <span class="author">Shields B</span>, <span class="author">Sj&ouml;gren M</span>, <span class="author">Steinthorsdottir V</span>, <span class="author">Stringham HM</span>, <span class="author">Swift AJ</span>, <span class="author">Thorleifsson G</span>, <span class="author">Thorsteinsdottir U</span>, <span class="author">Timpson NJ</span>, <span class="author">Tuomi T</span>, <span class="author">Tuomilehto J</span>, <span class="author">Walker M</span>, <span class="author">Watanabe RM</span>, <span class="author">Weedon MN</span>, <span class="author">Willer CJ</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Illig T</span>, <span class="author">Hveem K</span>, <span class="author">Hu FB</span>, <span class="author">Laakso M</span>, <span class="author">Stefansson K</span>, <span class="author">Pedersen O</span>, <span class="author">Wareham NJ</span>, <span class="author">Barroso I</span>, <span class="author">Hattersley AT</span>, <span class="author">Collins FS</span>, <span class="author">Groop L</span>, <span class="author">McCarthy MI</span>, <span class="author">Boehnke M</span> and <span class="author">Altshuler D</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p class="abstract">Genome-wide association (GWA) studies have identified multiple loci at which common variants modestly but reproducibly influence risk of type 2 diabetes (T2D). Established associations to common and rare variants explain only a small proportion of the heritability of T2D. As previously published analyses had limited power to identify variants with modest effects, we carried out meta-analysis of three T2D GWA scans comprising 10,128 individuals of European descent and approximately 2.2 million SNPs (directly genotyped and imputed), followed by replication testing in an independent sample with an effective sample size of up to 53,975. We detected at least six previously unknown loci with robust evidence for association, including the JAZF1 (P = 5.0 x 10(-14)), CDC123-CAMK1D (P = 1.2 x 10(-10)), TSPAN8-LGR5 (P = 1.1 x 10(-9)), THADA (P = 1.1 x 10(-9)), ADAMTS9 (P = 1.2 x 10(-8)) and NOTCH2 (P = 4.1 x 10(-8)) gene regions. Our results illustrate the value of large discovery and follow-up samples for gaining further insights into the inherited basis of T2D.</p>
<p class="grants">Funded by: Department of Health: <span class="grant">DHCS/07/07/008</span>; Medical Research Council: <span class="grant">G0000649</span>, <span class="grant">G0600705</span>, <span class="grant">G0601261</span>, <span class="grant">MC_U106179471</span>; NCI NIH HHS: <span class="grant">CA87969</span>; NHGRI NIH HHS: <span class="grant">HG002651</span>, <span class="grant">N01-HG-65403</span>, <span class="grant">U01 HG004171</span>, <span class="grant">U01 HG004399</span>; NHLBI NIH HHS: <span class="grant">HL084729</span>, <span class="grant">T32 HL007627</span>; NIDA NIH HHS: <span class="grant">U54 DA021519</span>; NIDDK NIH HHS: <span class="grant">DK062370</span>, <span class="grant">DK072193</span>, <span class="grant">DK58845</span>, <span class="grant">P30 DK040561</span>, <span class="grant">P30 DK040561-12</span>, <span class="grant">R01 DK029867</span>, <span class="grant">R01 DK072193</span>; Wellcome Trust: <span class="grant">072960</span>, <span class="grant">076113</span>, <span class="grant">077016</span>, <span class="grant">079557</span>, <span class="grant">GR072960</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2008</span>;<span class="volume">40</span>;<span class="number">5</span>;<span class="pagerange">638-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18372903">18372903</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672416">2672416</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.120">10.1038/ng.120</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genomic-based approach combining in vivo selection in mice to identify a novel virulence gene in Leishmania.</h4>
<p class="authors"><span class="author">Zhang WW</span>, <span class="author">Peacock CS</span> and <span class="author">Matlashewski G</span></p>
<p class="affiliation">Department of Microbiology and Immunology, McGill University, Montreal, Quebec, Canada.</p><p class="abstract"><strong>Background</strong>: Infection with Leishmania results in a broad spectrum of pathologies where L. infantum and L. donovani cause fatal visceral leishmaniasis and L. major causes destructive cutaneous lesions. The identification and characterization of Leishmania virulence genes may define the genetic basis for these different pathologies.</p><p class="abstract">Comparison of the recently completed L. major and L. infantum genomes revealed a relatively small number of genes that are absent or present as pseudogenes in L. major and potentially encode proteins in L. infantum. To investigate the potential role of genetic differences between species in visceral infection, seven genes initially classified as absent in L. major but present in L. infantum were cloned from the closely related L. donovani genome and introduced into L. major. The transgenic L. major expressing the L. donovani genes were then introduced into BALB/c mice to select for parasites with increased virulence in the spleen to determine whether any of the L. donovani genes increased visceral infection levels. During the course of these experiments, one of the selected genes (LinJ32_V3.1040 (Li1040)) was reclassified as also present in the L. major genome. Interestingly, only the Li1040 gene significantly increased visceral infection in the L. major transfectants. The Li1040 gene encodes a protein containing a putative component of an endosomal protein sorting complex involved with protein transport.</p><p class="abstract"><strong>Conclusions</strong>: These observations demonstrate that the levels of expression and sequence variations in genes ubiquitously shared between Leishmania species have the potential to significantly influence virulence and tissue tropism.</p>
<p class="grants">Funded by: Canadian Institutes of Health Research; Wellcome Trust</p>
<p class="publication"><span class="title">PLoS neglected tropical diseases</span>&nbsp;<span class="year">2008</span>;<span class="volume">2</span>;<span class="number">6</span>;<span class="pagerange">e248</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18545684" rel="external">18545684</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2398785" rel="external">2398785</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pntd.0000248" rel="external">10.1371/journal.pntd.0000248</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neo-sex chromosomes in the black muntjac recapitulate incipient evolution of mammalian sex chromosomes.</h4>
<p class="authors"><span class="author">Zhou Q</span>, <span class="author">Wang J</span>, <span class="author">Huang L</span>, <span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Liu Y</span>, <span class="author">Zhao X</span>, <span class="author">Yang F</span> and <span class="author">Wang W</span></p>
<p class="affiliation">CAS-Max Planck Junior Research Group, State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan, People&#39;s Republic of China.</p><p class="abstract"><strong>Background</strong>: The regular mammalian X and Y chromosomes diverged from each other at least 166 to 148 million years ago, leaving few traces of their early evolution, including degeneration of the Y chromosome and evolution of dosage compensation.</p><p class="abstract"><strong>Results</strong>: We studied the intriguing case of black muntjac, in which a recent X-autosome fusion and a subsequent large autosomal inversion within just the past 0.5 million years have led to inheritance patterns identical to the traditional X-Y (neo-sex chromosomes). We compared patterns of genome evolution in 35-kilobase noncoding regions and 23 gene pairs on the homologous neo-sex chromosomes. We found that neo-Y alleles have accumulated more mutations, comprising a wide variety of mutation types, which indicates cessation of recombination and is consistent with an ongoing neo-Y degeneration process. Putative deleterious mutations were observed in coding regions of eight investigated genes as well as cis-regulatory regions of two housekeeping genes. In vivo assays characterized a neo-Y insertion in the promoter of the CLTC gene that causes a significant reduction in allelic expression. A neo-Y-linked deletion in the 3&#39;-untranslated region of gene SNX22 abolished a microRNA target site. Finally, expression analyses revealed complex patterns of expression divergence between neo-Y and neo-X alleles.</p><p class="abstract"><strong>Conclusion</strong>: The nascent neo-sex chromosome system of black muntjacs is a valuable model in which to study the evolution of sex chromosomes in mammals. Our results illustrate the degeneration scenarios in various genomic regions. Of particular importance, we report--for the first time--that regulatory mutations were probably able to accelerate the degeneration process of Y and contribute to further evolution of dosage compensation.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2008</span>;<span class="volume">9</span>;<span class="number">6</span>;<span class="pagerange">R98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18554412" rel="external">18554412</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2481430" rel="external">2481430</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2008-9-6-r98" rel="external">10.1186/gb-2008-9-6-r98</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Immunoglobulin light chain (IgL) genes in zebrafish: Genomic configurations and inversional rearrangements between (V(L)-J(L)-C(L)) gene clusters.</h4>
<p class="authors"><span class="author">Zimmerman AM</span>, <span class="author">Yeo G</span>, <span class="author">Howe K</span>, <span class="author">Maddox BJ</span> and <span class="author">Steiner LA</span></p>
<p class="affiliation">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA, USA. <a href="mailto:%7A%69%6D%6D%65%72%6D%61%6E%61%40%63%6F%66%63.%65%64%75">&#122;&#105;&#109;&#109;&#101;&#114;&#109;&#97;&#110;&#97;&#64;&#99;&#111;&#102;&#99;&#46;&#101;&#100;&#117;</a> &gt;</p><p class="abstract">In mammals, Immunoglobulin light chain (IgL) are localized to two chromosomal regions (designated kappa and lambda). Here we report a genome-wide survey of IgL genes in the zebrafish revealing (V(L)-J(L)-C(L)) clusters spanning 5 separate chromosomes. To elucidate IgL loci present in the zebrafish genome assembly (Zv6), conventional sequence similarity searches and a novel scanning approach based on recombination signal sequence (RSS) motifs were applied. RT-PCR with zebrafish cDNA was used to confirm annotations, evaluate VJ-rearrangement possibilities and show that each chromosomal locus is expressed. In contrast to other vertebrates in which IgL exon usage has been studied, inversional rearrangement between (V(L)-J(L)-C(L)) clusters were found. Inter-cluster rearrangements may convey a selective advantage for editing self-reactive receptors and poise zebrafish by virtue of their extensive numbers of V(L), J(L) and C(L) to have greater potential for immunoglobulin gene shuffling than traditionally studied mice and human models.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Developmental and comparative immunology</span>&nbsp;<span class="year">2008</span>;<span class="volume">32</span>;<span class="number">4</span>;<span class="pagerange">421-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18022691" rel="external">18022691</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3014032" rel="external">3014032</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.dci.2007.08.005" rel="external">10.1016/j.dci.2007.08.005</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/vh3mkpab"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/vh3mkpab" src="/qr/vh3mkpab.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
