<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2007 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2007.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2007</h1><p>Number of papers published in 2007: <strong>315</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Predicted functions and linkage specificities of the products of the Streptococcus pneumoniae capsular biosynthetic loci.</h4>
<p class="authors"><span class="author">Aanensen DM</span>, <span class="author">Mavroidi A</span>, <span class="author">Bentley SD</span>, <span class="author">Reeves PR</span> and <span class="author">Spratt BG</span></p>
<p class="affiliation">Department of Infectious Disease Epidemiology, Imperial College London, Room G22, Old Medical School Building, St. Mary&#39;s Hospital, Norfolk Place, London W2 1PG, United Kingdom.</p><p class="abstract">The sequences of the capsular biosynthetic (cps) loci of 90 serotypes of Streptococcus pneumoniae have recently been determined. Bioinformatic procedures were used to predict the general functions of 1,973 of the 1,999 gene products and to identify proteins within the same homology group, Pfam family, and CAZy glycosyltransferase family. Correlating cps gene content with the 54 known capsular polysaccharide (CPS) structures provided tentative assignments of the specific functions of the different homology groups of each functional class (regulatory proteins, enzymes for synthesis of CPS constituents, polymerases, flippases, initial sugar transferases, glycosyltransferases [GTs], phosphotransferases, acetyltransferases, and pyruvyltransferases). Assignment of the glycosidic linkages catalyzed by the 342 GTs (92 homology groups) is problematic, but tentative assignments could be made by using this large set of cps loci and CPS structures to correlate the presence of particular GTs with specific glycosidic linkages, by correlating inverting or retaining linkages in CPS repeat units with the inverting or retaining mechanisms of the GTs predicted from their CAZy family membership, and by comparing the CPS structures of serotypes that have very similar cps gene contents. These large-scale comparisons between structure and gene content assigned the linkages catalyzed by 72% of the GTs, and all linkages were assigned in 32 of the serotypes with known repeat unit structures. Clear examples where very similar initial sugar transferases or glycosyltransferases catalyze different linkages in different serotypes were also identified. These assignments should provide a stimulus for biochemical studies to evaluate the reactions that are proposed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">21</span>;<span class="pagerange">7856-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17766420">17766420</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2168755">2168755</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00837-07">10.1128/JB.00837-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WebACT: an online genome comparison suite.</h4>
<p class="authors"><span class="author">Abbott JC</span>, <span class="author">Aanensen DM</span> and <span class="author">Bentley SD</span></p>
<p class="affiliation">Centre for Bioinformatics, Imperial College London, UK.</p><p class="abstract">Comparison of related genomes is an enormously powerful technique for explaining phenotypic differences and revealing recent evolutionary events. Genomes evolve through a host of mechanisms including long- and short-range intragenomic rearrangements, insertion of laterally acquired DNA, gene loss, and single-nucleotide polymorphisms. The Artemis Comparison Tool (ACT) was developed to enable the intuitive visualization of the consequences of such events in the context of two or more aligned genomes. WebACT is an online resource designed to allow the alignment of up to five genomic sequences within the ACT environment without the need for local software installation. Comparisons can be carried out between uploaded sequences, or those selected from the EMBL or RefSeq databases, using BLASTZ, MUMmer, or Basic Local Alignment Search Tool (BLAST). Precomputed comparisons can be selected from a database covering all the completed bacterial chromosome and plasmid sequences in the Genome Reviews database (1). This allows the rapid visualization of regions of interest, without the need to handle the full genome sequences. Here, we describe the process of using WebACT to prepare comparisons for visualization, and the selection of precomputed comparisons from the database. The use of ACT to view the selected comparison is then explored using examples from bacterial genomes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">395</span>;<span class="pagerange">57-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17993667">17993667</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Candidate live, attenuated Salmonella enterica serotype Typhimurium vaccines with reduced fecal shedding are immunogenic and effective oral vaccines.</h4>
<p class="authors"><span class="author">Abd El Ghany M</span>, <span class="author">Jansen A</span>, <span class="author">Clare S</span>, <span class="author">Hall L</span>, <span class="author">Pickard D</span>, <span class="author">Kingsley RA</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Environmental shedding of genetically manipulated microorganisms is an issue impeding the development of new live vaccines. We have investigated the immunogenicity of a number of novel Salmonella enterica serotype Typhimurium oral vaccine candidates that express the fragment C (TetC) component of tetanus toxin and harbor combinations of additional mutations in genes shdA, misL, and ratB that contribute to the persistence of serotype Typhimurium&#39;s colonization of the intestine. Serotype Typhimurium aroA (TetC) derivatives harboring additional mutations in either shdA or misL or combinations of these mutations exhibited a marked decrease in shedding of the vaccine strain in the feces of orally vaccinated mice. However, equivalent levels of anti-TetC and anti-Salmonella lipopolysaccharide immunoglobulin G (IgG), IgG1, IgG2a, and IgA were detected in sera of the vaccinated but not of the control mice. Cellular immune responses to TetC were detected in all vaccinated mice, regardless of the presence of the additional mutations in shdA or misL. Further, immunization with serotype Typhimurium aroA candidate vaccines harboring shdA and misL afforded complete protection against challenge with a virulent strain of serotype Typhimurium.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2007</span>;<span class="volume">75</span>;<span class="number">4</span>;<span class="pagerange">1835-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17296764" rel="external">17296764</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1865686" rel="external">1865686</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.01655-06" rel="external">10.1128/IAI.01655-06</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BCL11B is required for positive selection and survival of double-positive thymocytes.</h4>
<p class="authors"><span class="author">Albu DI</span>, <span class="author">Feng D</span>, <span class="author">Bhattacharya D</span>, <span class="author">Jenkins NA</span>, <span class="author">Copeland NG</span>, <span class="author">Liu P</span> and <span class="author">Avram D</span></p>
<p class="affiliation">Center for Cell Biology and Cancer Research, Albany Medical College, Albany, NY 12208, USA.</p><p class="abstract">Transcriptional control of gene expression in double-positive (DP) thymocytes remains poorly understood. We show that the transcription factor BCL11B plays a critical role in DP thymocytes by controlling positive selection of both CD4 and CD8 lineages. BCL11B-deficient DP thymocytes rearrange T cell receptor (TCR) alpha; however, they display impaired proximal TCR signaling and attenuated extracellular signal-regulated kinase phosphorylation and calcium flux, which are all required for initiation of positive selection. Further, provision of transgenic TCRs did not improve positive selection of BCL11B-deficient DP thymocytes. BCL11B-deficient DP thymocytes have altered expression of genes with a role in positive selection, TCR signaling, and other signaling pathways intersecting the TCR, which may account for the defect. BCL11B-deficient DP thymocytes also presented increased susceptibility to spontaneous apoptosis associated with high levels of cleaved caspase-3 and an altered balance of proapoptotic/prosurvival factors. This latter susceptibility was manifested even in the absence of TCR signaling and was only partially rescued by provision of the BCL2 transgene, indicating that control of DP thymocyte survival by BCL11B is nonredundant and, at least in part, independent of BCL2 prosurvival factors.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">T32-HL-07194</span>; NIAID NIH HHS: <span class="grant">R01 AI067846</span>, <span class="grant">R01 AI067846-01A2</span>; NIAMS NIH HHS: <span class="grant">K01 AR-02194</span></p>
<p class="publication"><span class="title">The Journal of experimental medicine</span>&nbsp;<span class="year">2007</span>;<span class="volume">204</span>;<span class="number">12</span>;<span class="pagerange">3003-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17998389">17998389</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2118514">2118514</a>; DOI: <a href="http://dx.doi.org/10.1084%2Fjem.20070863">10.1084/jem.20070863</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in TCF4, encoding a class I basic helix-loop-helix transcription factor, are responsible for Pitt-Hopkins syndrome, a severe epileptic encephalopathy associated with autonomic dysfunction.</h4>
<p class="authors"><span class="author">Amiel J</span>, <span class="author">Rio M</span>, <span class="author">de Pontual L</span>, <span class="author">Redon R</span>, <span class="author">Malan V</span>, <span class="author">Boddaert N</span>, <span class="author">Plouin P</span>, <span class="author">Carter NP</span>, <span class="author">Lyonnet S</span>, <span class="author">Munnich A</span> and <span class="author">Colleaux L</span></p>
<p class="affiliation">From the Departments of Genetics, Pediatric Radiology and INSERM U-797, Universite Paris-Descartes, Faculte de Medecine, Hopitaux de Paris, Hopital Necker-Enfants Malades, Paris, France. <a href="mailto:%61%6D%69%65%6C%40%6E%65%63%6B%65%72.%66%72">&#97;&#109;&#105;&#101;&#108;&#64;&#110;&#101;&#99;&#107;&#101;&#114;&#46;&#102;&#114;</a></p><p class="abstract">Pitt-Hopkins syndrome (PHS) is a rare syndromic encephalopathy characterized by daily bouts of hyperventilation and a facial gestalt. We report a 1.8-Mb de novo microdeletion on chromosome 18q21.1, identified by array-comparative genomic hybridization in one patient with PHS. We subsequently identified two de novo heterozygous missense mutations of a conserved amino acid in the basic region of the TCF4 gene in three additional subjects with PHS. These findings demonstrate that TCF4 anomalies are responsible for PHS and provide the first evidence of a human disorder related to class I basic helix-loop-helix transcription-factor defects (also known as &quot;E proteins&quot;). Moreover, our data may shed new light on the normal processes underlying autonomic nervous system development and maintenance of an appropriate ventilatory neuronal circuitry.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">80</span>;<span class="number">5</span>;<span class="pagerange">988-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17436254" rel="external">17436254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1852736" rel="external">1852736</a>; DOI: <a href="http://dx.doi.org/10.1086%2F515582" rel="external">10.1086/515582</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SISYPHUS--structural alignments for proteins with non-trivial relationships.</h4>
<p class="authors"><span class="author">Andreeva A</span>, <span class="author">Prli&#x107; A</span>, <span class="author">Hubbard TJ</span> and <span class="author">Murzin AG</span></p>
<p class="affiliation">MRC Centre for Protein Engineering, Hills Road, Cambridge CB2 2QH, UK. <a href="mailto:%74%6F%6E%79%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#116;&#111;&#110;&#121;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">With the increasing amount of structural data, the number of homologous protein structures bearing topological irregularities is steadily growing. These include proteins with circular permutations, segment-swapping, context-dependent folding or chameleon sequences that can adopt alternative secondary structures. Their non-trivial structural relationships are readily identified during expert analysis but their automatic identification using the existing computational tools still remains difficult or impossible. Such non-trivial cases of protein relationships are known to pose a problem to multiple alignment algorithms and to impede comparative modeling studies. They support a new emerging concept of evolutionary changeable protein fold, which creates practical difficulties for the hierarchical classifications of protein structures.To facilitate the understanding of, and to provide a comprehensive annotation of proteins with such non-trivial structural relationships we have created SISYPHUS ([Sigmaomeganuphiomicronzeta]--in Greek crafty), a compendium to the SCOP database. The SISYPHUS database contains a collection of manually curated structural alignments and their inter-relationships. The multiple alignments are constructed for protein structural regions that range from oligomeric biological units, or individual domains to fragments of different size. The SISYPHUS multiple alignments are displayed with SPICE, a browser that provides an integrated view of protein sequences, structures and their annotations. The database is available from http://sisyphus.mrc-cpe.cam.ac.uk.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105192716</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">Database issue</span>;<span class="pagerange">D253-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17068077">17068077</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1635320">1635320</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl746">10.1093/nar/gkl746</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Karyotypic evolution and phylogenetic relationships in the order Chiroptera as revealed by G-banding comparison and chromosome painting.</h4>
<p class="authors"><span class="author">Ao L</span>, <span class="author">Mao X</span>, <span class="author">Nie W</span>, <span class="author">Gu X</span>, <span class="author">Feng Q</span>, <span class="author">Wang J</span>, <span class="author">Su W</span>, <span class="author">Wang Y</span>, <span class="author">Volleth M</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Key Laboratory of Cellular and Molecular Evolution, Kunming Institute of Zoology, and Graduate School of the Chinese Academy of Sciences, Kunming, Yunnan, 650223, P.R. China.</p><p class="abstract">Bats are a unique but enigmatic group of mammals and have a world-wide distribution. The phylogenetic relationships of extant bats are far from being resolved. Here, we investigated the karyotypic relationships of representative species from four families of the order Chiroptera by comparative chromosome painting and banding. A complete set of painting probes derived from flow-sorted chromosomes of Myotis myotis (family Vespertilionidae) were hybridized onto metaphases of Cynopterus sphinx (2n = 34, family Pteropodidae), Rhinolophus sinicus (2n=36, family Rhinolophidae) and Aselliscus stoliczkanus (2n=30, family Hipposideridae) and delimited 27, 30 and 25 conserved chromosomal segments in the three genomes, respectively. The results substantiate that Robertsonian translocation is the main mode of chromosome evolution in the order Chiroptera, with extensive conservation of whole chromosomal arms. The use of M. myotis (2n=44) probes has enabled the integration of C. sphinx, R. sinicus and A. stoliczkanus chromosomes into the previously established comparative maps between human and Eonycteris spelaea (2n=36), Rhinolophus mehelyi (2n=58), Hipposideros larvatus (2n=32), and M. myotis. Our results provide the first cytogenetic signature rearrangement that supports the grouping of Pteropodidae and Rhinolophoidea in a common clade (i.e. Pteropodiformes or Yinpterochiroptera) and thus improve our understanding on the karyotypic relationships and genome phylogeny of these bat species.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">3</span>;<span class="pagerange">257-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17310301" rel="external">17310301</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1120-7" rel="external">10.1007/s10577-007-1120-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional characterization of the Plasmodium falciparum and P. berghei homologues of macrophage migration inhibitory factor.</h4>
<p class="authors"><span class="author">Augustijn KD</span>, <span class="author">Kleemann R</span>, <span class="author">Thompson J</span>, <span class="author">Kooistra T</span>, <span class="author">Crawford CE</span>, <span class="author">Reece SE</span>, <span class="author">Pain A</span>, <span class="author">Siebum AH</span>, <span class="author">Janse CJ</span> and <span class="author">Waters AP</span></p>
<p class="affiliation">Department of Parasitology, LUMC, Albinusdreef 2, Room P4-35, 2333 ZA Leiden, The Netherlands.</p><p class="abstract">Macrophage migration inhibitory factor (MIF) is a mammalian cytokine that participates in innate and adaptive immune responses. Homologues of mammalian MIF have been discovered in parasite species infecting mammalian hosts (nematodes and malaria parasites), which suggests that the parasites express MIF to modulate the host immune response upon infection. Here we report the first biochemical and genetic characterization of a Plasmodium MIF (PMIF). Like human MIF, histidine-tagged purified recombinant PMIF shows tautomerase and oxidoreductase activities (although the activities are reduced compared to those of histidine-tagged human MIF) and efficiently inhibits AP-1 activity in human embryonic kidney cells. Furthermore, we found that Plasmodium berghei MIF is expressed in both a mammalian host and a mosquito vector and that, in blood stages, it is secreted into the infected erythrocytes and released upon schizont rupture. Mutant P. berghei parasites lacking PMIF were able to complete the entire life cycle and exhibited no significant changes in growth characteristics or virulence features during blood stage infection. However, rodent hosts infected with knockout parasites had significantly higher numbers of circulating reticulocytes. Our results suggest that PMIF is produced by the parasite to influence host immune responses and the course of anemia upon infection.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">072171</span></p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2007</span>;<span class="volume">75</span>;<span class="number">3</span>;<span class="pagerange">1116-28</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17158894" rel="external">17158894</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1828592" rel="external">1828592</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.00902-06" rel="external">10.1128/IAI.00902-06</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of Salmonella enterica serovar Typhi.</h4>
<p class="authors"><span class="author">Baker S</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The generation of complete genome sequences provides a blueprint that facilitates the genetic characterization of pathogens and their hosts. The genome of Salmonella enterica serovar Typhi (S. Typhi) harbors ~5 million base pairs encoding some 4000 genes, of which &gt;200 are functionally inactive. Comparison of S. Typhi isolates from around the world indicates that they are highly related (clonal) and that they emerged from a single point of origin ~30,000-50,000 years ago. Evidence suggests that, as well as undergoing gene degradation, S. Typhi has also recently acquired genes, such as those encoding the Vi antigen, by horizontal transfer events.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">45 Suppl 1</span>;<span class="pagerange">S29-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17582565">17582565</a>; DOI: <a href="http://dx.doi.org/10.1086%2F518143">10.1086/518143</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel linear plasmid mediates flagellar variation in Salmonella Typhi.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Hardy J</span>, <span class="author">Sanderson KE</span>, <span class="author">Quail M</span>, <span class="author">Goodhead I</span>, <span class="author">Kingsley RA</span>, <span class="author">Parkhill J</span>, <span class="author">Stocker B</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%73%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Unlike the majority of Salmonella enterica serovars, Salmonella Typhi (S. Typhi), the etiological agent of human typhoid, is monophasic. S. Typhi normally harbours only the phase 1 flagellin gene (fliC), which encodes the H:d antigen. However, some S. Typhi strains found in Indonesia express an additional flagellin antigen termed H:z66. Molecular analysis of H:z66+ S. Typhi revealed that the H:z66 flagellin structural gene (fljB(z66)) is encoded on a linear plasmid that we have named pBSSB1. The DNA sequence of pBSSB1 was determined to be just over 27 kbp, and was predicted to encode 33 coding sequences. To our knowledge, pBSSB1 is the first non-bacteriophage-related linear plasmid to be described in the Enterobacteriaceae.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">NIH-AI034829</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">e59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17500588" rel="external">17500588</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1876496" rel="external">1876496</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.0030059" rel="external">10.1371/journal.ppat.0030059</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A linear plasmid truncation induces unidirectional flagellar phase change in H:z66 positive Salmonella Typhi.</h4>
<p class="authors"><span class="author">Baker S</span>, <span class="author">Holt K</span>, <span class="author">Whitehead S</span>, <span class="author">Goodhead I</span>, <span class="author">Perkins T</span>, <span class="author">Stocker B</span>, <span class="author">Hardy J</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%73%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The process by which bacteria regulate flagellar expression is known as phase variation and in Salmonella enterica this process permits the expression of one of two flagellin genes, fliC or fljB, at any one time. Salmonella Typhi (S. Typhi) is normally not capable of phase variation of flagellar antigen expression as isolates only harbour the fliC gene (H:d) and lacks an equivalent fljB locus. However, some S. Typhi isolates, exclusively from Indonesia, harbour an fljB equivalent encoded on linear plasmid, pBSSB1 that drives the expression of a novel flagellin named H:z66. H:z66+S. Typhi isolates were stimulated to change flagellar phase and genetically analysed for the mechanism of variation. The phase change was demonstrated to be unidirectional, reverting to expression from the resident chromosomal fliC gene. DNA sequencing demonstrated that pBSSB1 linear DNA was still detectable but that these derivatives had undergone deletion and were lacking fljA(z66) (encoding a flagellar repressor) and fljB(z66). The deletion end-point was found to involve one of the plasmid termini and a palindromic repeat sequence within fljB(z66), distinct to that found at the terminus of pBSSB1. These data demonstrate that, like some Streptomyces linear elements, at least one of the terminal inverted repeats of pBSSB1 is non-essential, but that a palindromic repeat sequence may be necessary for replication.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">66</span>;<span class="number">5</span>;<span class="pagerange">1207-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17976161">17976161</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652032">2652032</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2007.05995.x">10.1111/j.1365-2958.2007.05995.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cross-species chromosome painting among camel, cattle, pig and human: further insights into the putative Cetartiodactyla ancestral karyotype.</h4>
<p class="authors"><span class="author">Balmus G</span>, <span class="author">Trifonov VA</span>, <span class="author">Biltueva LS</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Alkalaeva ES</span>, <span class="author">Fu B</span>, <span class="author">Skidmore JA</span>, <span class="author">Allen T</span>, <span class="author">Graphodatsky AS</span>, <span class="author">Yang F</span> and <span class="author">Ferguson-Smith MA</span></p>
<p class="affiliation">Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, Cambridge, UK.</p><p class="abstract">The great karyotypic differences between camel, cattle and pig, three important domestic animals, have been a challenge for comparative cytogenetic studies based on conventional cytogenetic approaches. To construct a genome-wide comparative chromosome map among these artiodactyls, we made a set of chromosome painting probes from the dromedary camel (Camelus dromedarius) by flow sorting and degenerate oligonucleotide primed-PCR. The painting probes were first used to characterize the karyotypes of the dromedary camel (C. dromedarius), the Bactrian camel (C. bactrianus), the guanaco (Lama guanicoe), the alpaca (L. pacos) and dromedary x guanaco hybrid karyotypes (all with 2n = 74). These FISH experiments enabled the establishment of a high-resolution GTG-banded karyotype, together with chromosome nomenclature and idiogram for C. dromedarius, and revealed that these camelid species have almost identical karyotypes, with only slight variations in the amount and distribution patterns of heterochromatin. Further cross-species chromosome painting between camel, cattle, pig and human with painting probes from the camel and human led to the establishment of genome-wide comparative maps. Between human and camel, pig and camel, and cattle and camel 47, 53 and 53 autosomal conserved segments were detected, respectively. Integrated analysis with previously published comparative maps of human/pig/cattle enabled us to propose a Cetartiodactyla ancestral karyotype and to discuss the early karyotype evolution of Cetartiodactyla. Furthermore, these maps will facilitate the positional cloning of genes by aiding the cross-species transfer of mapping information.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">4</span>;<span class="pagerange">499-515</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17671843" rel="external">17671843</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1154-x" rel="external">10.1007/s10577-007-1154-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&#39;Species&#39; of peptidases.</h4>
<p class="authors"><span class="author">Barrett AJ</span> and <span class="author">Rawlings ND</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%61%6C%61%6E.%62%61%72%72%65%74%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#97;&#110;&#46;&#98;&#97;&#114;&#114;&#101;&#116;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A good system for the naming and classification of peptidases can contribute much to the study of these enzymes. Having already described the building of families and clans in the MEROPS system, we here focus on the lowest level in the hierarchy, in which the huge number of individual peptidase proteins are assigned to a lesser number of what we term &#39;species&#39; of peptidases. Just over 2000 peptidase species are recognised today, but we estimate that 25 000 will one day be known. Each species is built around a peptidase protein that has been adequately characterised. The cluster of peptidase proteins that represent the single species is then assembled primarily by analysis of a sequence &#39;tree&#39; for the family. Each peptidase species is given a systematic identifier and a summary page of data regarding it is assembled. Because the characterisation of new peptidases lags far behind the sequencing, the majority of peptidase proteins are so far known only as amino acid sequences and cannot yet be assigned to species. We suggest that new forms of analysis of the sequences of the unassigned peptidases may give early indications of how they will cluster into the new species of the future.</p>
<p class="publication"><span class="title">Biological chemistry</span>&nbsp;<span class="year">2007</span>;<span class="volume">388</span>;<span class="number">11</span>;<span class="pagerange">1151-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17976007" rel="external">17976007</a>; DOI: <a href="http://dx.doi.org/10.1515%2FBC.2007.151" rel="external">10.1515/BC.2007.151</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biallelic mutation of MSH2 in primary human cells is associated with sensitivity to irradiation and altered RAD51 foci kinetics.</h4>
<p class="authors"><span class="author">Barwell J</span>, <span class="author">Pangon L</span>, <span class="author">Hodgson S</span>, <span class="author">Georgiou A</span>, <span class="author">Kesterton I</span>, <span class="author">Slade T</span>, <span class="author">Taylor M</span>, <span class="author">Payne SJ</span>, <span class="author">Brinkman H</span>, <span class="author">Smythe J</span>, <span class="author">Sebire NJ</span>, <span class="author">Solomon E</span>, <span class="author">Docherty Z</span>, <span class="author">Camplejohn R</span>, <span class="author">Homfray T</span> and <span class="author">Morris JR</span></p>
<p class="affiliation">Department of Genetics, St. George&#39;s Medical School, University of London, Cranmer Terrace, London, UK. <a href="mailto:%6A%6F.%6D%6F%72%72%69%73%40%67%65%6E%65%74%69%63%73.%6B%63%6C.%61%63.%75%6B">&#106;&#111;&#46;&#109;&#111;&#114;&#114;&#105;&#115;&#64;&#103;&#101;&#110;&#101;&#116;&#105;&#99;&#115;&#46;&#107;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Reports of differential mutagen sensitivity conferred by a defect in the mismatch repair (MMR) pathway are inconsistent in their conclusions. Previous studies have investigated cells established from immortalised human colorectal tumour lines or cells from animal models.</p><p class="abstract"><strong>Methods</strong>: We examined primary human MSH2-deficient neonatal cells, bearing a biallelic truncating mutation in MSH2, for viability and chromosomal damage after exposure to DNA-damaging agents.</p><p class="abstract"><strong>Results</strong>: MSH2-deficient cells exhibit no response to interstrand DNA cross-linking agents but do show reduced viability in response to irradiation. They also show increased chromosome damage and exhibit altered RAD51 foci kinetics after irradiation exposure, indicating defective homologous recombinational repair.</p><p class="abstract"><strong>Discussion</strong>: The cellular features and sensitivity of MSH2-deficient primary human cells are broadly in agreement with observations of primary murine cells lacking the same gene. The data therefore support the view that the murine model recapitulates early features of MMR deficiency in humans, and implies that the variable data reported for MMR-deficient immortalised human cells may be due to further genetic or epigenetic lesions. We suggest caution in the use of radiotherapy for treatment of malignancies in individuals with functional loss of MSH2.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">44</span>;<span class="number">8</span>;<span class="pagerange">516-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17483304" rel="external">17483304</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2597924" rel="external">2597924</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2006.048660" rel="external">10.1136/jmg.2006.048660</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prediction of noncoding transcripts</h4>
<p class="authors"><span class="author">Bateman A</span></p>
<p class="publication"><span class="title">Bioinformatics.</span> <span class="year">2007</span>;<span class="pagerange">103-116</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCOOP: a simple method for identification of novel protein superfamily relationships.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Finn RD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: Profile searches of sequence databases are a sensitive way to detect sequence relationships. Sophisticated profile-profile comparison algorithms that have been recently introduced increase search sensitivity even further.</p><p class="abstract"><strong>Results</strong>: In this article, a simpler approach than profile-profile comparison is presented that has a comparable performance to state-of-the-art tools such as COMPASS, HHsearch and PRC. This approach is called SCOOP (Simple Comparison Of Outputs Program), and is shown to find known relationships between families in the Pfam database as well as detect novel distant relationships between families. Several novel discoveries are presented including the discovery that a domain of unknown function (DUF283) found in Dicer proteins is related to double-stranded RNA-binding domains.</p><p class="abstract"><strong>Availability</strong>: SCOOP is freely available under a GNU GPL license from http://www.sanger.ac.uk/Users/agb/SCOOP/.</p><p class="abstract">Supplementary data are available at Bioinformatics online.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">7</span>;<span class="pagerange">809-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17277330">17277330</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2603044">2603044</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm034">10.1093/bioinformatics/btm034</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EDITORIAL</h4>
<p class="authors"><span class="author">Bateman</span>, <span class="author">A</span></p>
<p class="publication"><span class="title">Nucleic Acids Res..</span> <span class="year">2007</span>;<span class="volume">35</span>;<span class="pagerange">D1</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Genographic Project public participation mitochondrial DNA database.</h4>
<p class="authors"><span class="author">Behar DM</span>, <span class="author">Rosset S</span>, <span class="author">Blue-Smith J</span>, <span class="author">Balanovsky O</span>, <span class="author">Tzur S</span>, <span class="author">Comas D</span>, <span class="author">Mitchell RJ</span>, <span class="author">Quintana-Murci L</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Wells RS</span> and <span class="author">Genographic Consortium</span></p><p class="abstract">The Genographic Project is studying the genetic signatures of ancient human migrations and creating an open-source research database. It allows members of the public to participate in a real-time anthropological genetics study by submitting personal samples for analysis and donating the genetic results to the database. We report our experience from the first 18 months of public participation in the Genographic Project, during which we have created the largest standardized human mitochondrial DNA (mtDNA) database ever collected, comprising 78,590 genotypes. Here, we detail our genotyping and quality assurance protocols including direct sequencing of the mtDNA HVS-I, genotyping of 22 coding-region SNPs, and a series of computational quality checks based on phylogenetic principles. This database is very informative with respect to mtDNA phylogeny and mutational dynamics, and its size allows us to develop a nearest neighbor-based methodology for mtDNA haplogroup prediction based on HVS-I motifs that is superior to classic rule-based approaches. We make available to the scientific community and general public two new resources: a periodically updated database comprising all data donated by participants, and the nearest neighbor haplogroup prediction tool.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">6</span>;<span class="pagerange">e104</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17604454" rel="external">17604454</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1904368" rel="external">1904368</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0030104" rel="external">10.1371/journal.pgen.0030104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nodal signaling activates differentiation genes during zebrafish gastrulation.</h4>
<p class="authors"><span class="author">Bennett JT</span>, <span class="author">Joubin K</span>, <span class="author">Cheng S</span>, <span class="author">Aanstad P</span>, <span class="author">Herwig R</span>, <span class="author">Clark M</span>, <span class="author">Lehrach H</span> and <span class="author">Schier AF</span></p>
<p class="affiliation">Developmental Genetics Program, Skirball Institute of Biomolecular Medicine, and Department of Cell Biology, New York University School of Medicine, New York, NY 10016, USA.</p><p class="abstract">Nodal signals induce mesodermal and endodermal progenitors during vertebrate development. To determine the role of Nodal signaling at a genomic level, we isolated Nodal-regulated genes by expression profiling using macroarrays and gene expression databases. Putative Nodal-regulated genes were validated by in situ hybridization screening in wild type and Nodal signaling mutants. 46 genes were identified, raising the currently known number of Nodal-regulated genes to 72. Based on their expression patterns along the dorsoventral axis, most of these genes can be classified into two groups. One group is expressed in the dorsal margin, whereas the other group is expressed throughout the margin. In addition to transcription factors and signaling components, the screens identified several new functional classes of Nodal-regulated genes, including cytoskeletal components and molecules involved in protein secretion or endoplasmic reticulum stress. We found that x-box binding protein-1 (xbp1) is a direct target of Nodal signaling and required for the terminal differentiation of the hatching gland, a specialized secretory organ whose specification is also dependent on Nodal signaling. These results indicate that Nodal signaling regulates not only specification genes but also differentiation genes.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM056211-06</span>, <span class="grant">R01 GM056211-07</span>, <span class="grant">R01 GM056211-08</span>, <span class="grant">R01 GM056211-09</span>, <span class="grant">R01 GM056211-10</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">304</span>;<span class="number">2</span>;<span class="pagerange">525-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17306247" rel="external">17306247</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1885460" rel="external">1885460</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2007.01.012" rel="external">10.1016/j.ydbio.2007.01.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacterial therapeutics.</h4>
<p class="authors"><span class="author">Bentley S</span> and <span class="author">Sebaihia M</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">170-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17310535">17310535</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1633">10.1038/nrmicro1633</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meningococcal genetic variation mechanisms viewed through comparative analysis of serogroup C strain FAM18.</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Vernikos GS</span>, <span class="author">Snyder LA</span>, <span class="author">Churcher C</span>, <span class="author">Arrowsmith C</span>, <span class="author">Chillingworth T</span>, <span class="author">Cronin A</span>, <span class="author">Davis PH</span>, <span class="author">Holroyd NE</span>, <span class="author">Jagels K</span>, <span class="author">Maddison M</span>, <span class="author">Moule S</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Sharp S</span>, <span class="author">Unwin L</span>, <span class="author">Whitehead S</span>, <span class="author">Quail MA</span>, <span class="author">Achtman M</span>, <span class="author">Barrell B</span>, <span class="author">Saunders NJ</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, United Kingdom. <a href="mailto:%73%64%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#100;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The bacterium Neisseria meningitidis is commonly found harmlessly colonising the mucosal surfaces of the human nasopharynx. Occasionally strains can invade host tissues causing septicaemia and meningitis, making the bacterium a major cause of morbidity and mortality in both the developed and developing world. The species is known to be diverse in many ways, as a product of its natural transformability and of a range of recombination and mutation-based systems. Previous work on pathogenic Neisseria has identified several mechanisms for the generation of diversity of surface structures, including phase variation based on slippage-like mechanisms and sequence conversion of expressed genes using information from silent loci. Comparison of the genome sequences of two N. meningitidis strains, serogroup B MC58 and serogroup A Z2491, suggested further mechanisms of variation, including C-terminal exchange in specific genes and enhanced localised recombination and variation related to repeat arrays. We have sequenced the genome of N. meningitidis strain FAM18, a representative of the ST-11/ET-37 complex, providing the first genome sequence for the disease-causing serogroup C meningococci; it has 1,976 predicted genes, of which 60 do not have orthologues in the previously sequenced serogroup A or B strains. Through genome comparison with Z2491 and MC58 we have further characterised specific mechanisms of genetic variation in N. meningitidis, describing specialised loci for generation of cell surface protein variants and measuring the association between noncoding repeat arrays and sequence variation in flanking genes. Here we provide a detailed view of novel genetic diversification mechanisms in N. meningitidis. Our analysis provides evidence for the hypothesis that the noncoding repeat arrays in neisserial genomes (neisserial intergenic mosaic elements) provide a crucial mechanism for the generation of surface antigen variants. Such variation will have an impact on the interaction with the host tissues, and understanding these mechanisms is important to aid our understanding of the intimate and complex relationship between the human nasopharynx and the meningococcus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">2</span>;<span class="pagerange">e23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17305430">17305430</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1797815">1797815</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0030023">10.1371/journal.pgen.0030023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interactions among genes in the ErbB-Neuregulin signalling network are associated with increased susceptibility to schizophrenia.</h4>
<p class="authors"><span class="author">Benzel I</span>, <span class="author">Bansal A</span>, <span class="author">Browning BL</span>, <span class="author">Galwey NW</span>, <span class="author">Maycox PR</span>, <span class="author">McGinnis R</span>, <span class="author">Smart D</span>, <span class="author">St Clair D</span>, <span class="author">Yates P</span> and <span class="author">Purvis I</span></p>
<p class="affiliation">Psychiatry CEDD, GlaxoSmithKline, New Frontiers Science Park, Third Avenue, Harlow, Essex, CM19 5AW Harlow, Essex, UK. <a href="mailto:%49%73%61%62%65%6C.%4D.%42%65%6E%7A%65%6C%40%67%73%6B.%63%6F%6D">&#73;&#115;&#97;&#98;&#101;&#108;&#46;&#77;&#46;&#66;&#101;&#110;&#122;&#101;&#108;&#64;&#103;&#115;&#107;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: Evidence of genetic association between the NRG1 (Neuregulin-1) gene and schizophrenia is now well-documented. Furthermore, several recent reports suggest association between schizophrenia and single-nucleotide polymorphisms (SNPs) in ERBB4, one of the receptors for Neuregulin-1. In this study, we have extended the previously published associations by investigating the involvement of all eight genes from the ERBB and NRG families for association with schizophrenia.</p><p class="abstract"><strong>Methods</strong>: Eight genes from the ERBB and NRG families were tested for association to schizophrenia using a collection of 396 cases and 1,342 blood bank controls ascertained from Aberdeen, UK. A total of 365 SNPs were tested. Association testing of both alleles and genotypes was carried out using the fast Fisher&#39;s Exact Test (FET). To understand better the nature of the associations, all pairs of SNPs separated by &gt;or= 0.5 cM with at least nominal evidence of association (P &lt; 0.10) were tested for evidence of pairwise interaction by logistic regression analysis.</p><p class="abstract"><strong>Results</strong>: 42 out of 365 tested SNPs in the eight genes from the ERBB and NRG gene families were significantly associated with schizophrenia (P &lt; 0.05). Associated SNPs were located in ERBB4 and NRG1, confirming earlier reports. However, novel associations were also seen in NRG2, NRG3 and EGFR. In pairwise interaction tests, clear evidence of gene-gene interaction was detected for NRG1-NRG2, NRG1-NRG3 and EGFR-NRG2, and suggestive evidence was also seen for ERBB4-NRG1, ERBB4-NRG2, ERBB4-NRG3 and ERBB4-ERBB2. Evidence of intragenic interaction was seen for SNPs in ERBB4.</p><p class="abstract"><strong>Conclusion</strong>: These new findings suggest that observed associations between NRG1 and schizophrenia may be mediated through functional interaction not just with ERBB4, but with other members of the NRG and ERBB families. There is evidence that genetic interaction among these loci may increase susceptibility to schizophrenia.</p>
<p class="publication"><span class="title">Behavioral and brain functions : BBF</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="pagerange">31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17598910" rel="external">17598910</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1934910" rel="external">1934910</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1744-9081-3-31" rel="external">10.1186/1744-9081-3-31</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Helminth initiative for drug discovery</h4>
<p class="authors"><span class="author">Berriman M</span></p>
<p class="publication"><span class="title">Expert Opinion on Drug Discovery.</span> <span class="year">2007</span>;<span class="volume">2</span>;<span class="pagerange">Suppl 1</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variety is the spice of eukaryotic life.</h4>
<p class="authors"><span class="author">Berriman M</span> and <span class="author">Pain A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">9</span>;<span class="pagerange">660-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17703542">17703542</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1741">10.1038/nrmicro1741</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: new content and better access.</h4>
<p class="authors"><span class="author">Bieri T</span>, <span class="author">Blasiar D</span>, <span class="author">Ozersky P</span>, <span class="author">Antoshechkin I</span>, <span class="author">Bastiani C</span>, <span class="author">Canaran P</span>, <span class="author">Chan J</span>, <span class="author">Chen N</span>, <span class="author">Chen WJ</span>, <span class="author">Davis P</span>, <span class="author">Fiedler TJ</span>, <span class="author">Girard L</span>, <span class="author">Han M</span>, <span class="author">Harris TW</span>, <span class="author">Kishore R</span>, <span class="author">Lee R</span>, <span class="author">McKay S</span>, <span class="author">M&uuml;ller HM</span>, <span class="author">Nakamura C</span>, <span class="author">Petcherski A</span>, <span class="author">Rangarajan A</span>, <span class="author">Rogers A</span>, <span class="author">Schindelman G</span>, <span class="author">Schwarz EM</span>, <span class="author">Spooner W</span>, <span class="author">Tuli MA</span>, <span class="author">Van Auken K</span>, <span class="author">Wang D</span>, <span class="author">Wang X</span>, <span class="author">Williams G</span>, <span class="author">Durbin R</span>, <span class="author">Stein LD</span>, <span class="author">Sternberg PW</span> and <span class="author">Spieth J</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University School of Medicine, St Louis, MO 63108, USA. <a href="mailto:%74%62%69%65%72%69%40%77%61%74%73%6F%6E.%77%75%73%74%6C.%65%64%75">&#116;&#98;&#105;&#101;&#114;&#105;&#64;&#119;&#97;&#116;&#115;&#111;&#110;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">WormBase (http://wormbase.org), a model organism database for Caenorhabditis elegans and other related nematodes, continues to evolve and expand. Over the past year WormBase has added new data on C.elegans, including data on classical genetics, cell biology and functional genomics; expanded the annotation of closely related nematodes with a new genome browser for Caenorhabditis remanei; and deployed new hardware for stronger performance. Several existing datasets including phenotype descriptions and RNAi experiments have seen a large increase in new content. New datasets such as the C.remanei draft assembly and annotations, the Vancouver Fosmid library and TEC-RED 5&#39; end sites are now available as well. Access to and searching WormBase has become more dependable and flexible via multiple mirror sites and indexing through Google.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">Database issue</span>;<span class="pagerange">D506-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17099234" rel="external">17099234</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1669750" rel="external">1669750</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl818" rel="external">10.1093/nar/gkl818</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Architectures of somatic genomic rearrangement in human cancer amplicons at sequence-level resolution.</h4>
<p class="authors"><span class="author">Bignell GR</span>, <span class="author">Santarius T</span>, <span class="author">Pole JC</span>, <span class="author">Butler AP</span>, <span class="author">Perry J</span>, <span class="author">Pleasance E</span>, <span class="author">Greenman C</span>, <span class="author">Menzies A</span>, <span class="author">Taylor S</span>, <span class="author">Edkins S</span>, <span class="author">Campbell P</span>, <span class="author">Quail M</span>, <span class="author">Plumb B</span>, <span class="author">Matthews L</span>, <span class="author">McLay K</span>, <span class="author">Edwards PA</span>, <span class="author">Rogers J</span>, <span class="author">Wooster R</span>, <span class="author">Futreal PA</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, United Kingdom.</p><p class="abstract">For decades, cytogenetic studies have demonstrated that somatically acquired structural rearrangements of the genome are a common feature of most classes of human cancer. However, the characteristics of these rearrangements at sequence-level resolution have thus far been subject to very limited description. One process that is dependent upon somatic genome rearrangement is gene amplification, a mechanism often exploited by cancer cells to increase copy number and hence expression of dominantly acting cancer genes. The mechanisms underlying gene amplification are complex but must involve chromosome breakage and rejoining. We sequenced 133 different genomic rearrangements identified within four cancer amplicons involving the frequently amplified cancer genes MYC, MYCN, and ERBB2. The observed architectures of rearrangement were diverse and highly distinctive, with evidence for sister chromatid breakage-fusion-bridge cycles, formation and reinsertion of double minutes, and the presence of bizarre clusters of small genomic fragments. There were characteristic features of sequences at the breakage-fusion junctions, indicating roles for nonhomologous end joining and homologous recombination-mediated repair mechanisms together with nontemplated DNA synthesis. Evidence was also found for sequence-dependent variation in susceptibility of the genome to somatic rearrangement. The results therefore provide insights into the DNA breakage and repair processes operative in somatic genome rearrangement and illustrate how the evolutionary histories of individual cancers can be reconstructed from large-scale cancer genome sequencing.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">9</span>;<span class="pagerange">1296-303</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17675364" rel="external">17675364</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1950898" rel="external">1950898</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.6522707" rel="external">10.1101/gr.6522707</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RPS6KA2, a putative tumour suppressor gene at 6q27 in sporadic epithelial ovarian cancer.</h4>
<p class="authors"><span class="author">Bignone PA</span>, <span class="author">Lee KY</span>, <span class="author">Liu Y</span>, <span class="author">Emilion G</span>, <span class="author">Finch J</span>, <span class="author">Soosay AE</span>, <span class="author">Charnock FM</span>, <span class="author">Beck S</span>, <span class="author">Dunham I</span>, <span class="author">Mungall AJ</span> and <span class="author">Ganesan TS</span></p>
<p class="affiliation">Cancer Research UK, Molecular Oncology Laboratories, Ovarian Cancer Group, Weatherall Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford, UK.</p><p class="abstract">We had previously defined by allele loss studies a minimal region at 6q27 (between D6S264 and D6S297) to contain a putative tumour suppressor gene. The p90 ribosomal S6 kinase-3 gene (p90 Rsk-3, RPS6KA2) maps in this interval. It is a serine-threonine kinase that signals downstream of the mitogen-activated protein kinase pathway. It is expressed in normal ovarian epithelium, whereas reduced or absent in tumours or cell lines. We show that RPS6KA2 is monoallelically expressed in the ovary suggesting that loss of a single expressed allele is sufficient to cause complete loss of expression in cancer cells. Further, we have identified two new isoforms of RPS6KA2 with an alternative start codon. Homozygous deletions were identified within the RPS6KA2 gene in two cell lines. Re-expression of RPS6KA2 in ovarian cancer cell lines suppressed colony formation. In UCI101 cells, the expression of RPS6KA2 reduced proliferation, caused G1 arrest, increased apoptosis, reduced levels of phosphorylated extracellular signal-regulated kinase and altered other cell cycle proteins. In contrast, small interfering RNA against RPS6KA2 showed the opposite effect in 41M cells. The above results suggest that RPS6KA2 is a putative tumour suppressor gene to explain allele loss at 6q27.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">5</span>;<span class="pagerange">683-700</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16878154" rel="external">16878154</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1209827" rel="external">10.1038/sj.onc.1209827</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fast-evolving noncoding sequences in the human genome.</h4>
<p class="authors"><span class="author">Bird CP</span>, <span class="author">Stranger BE</span>, <span class="author">Liu M</span>, <span class="author">Thomas DJ</span>, <span class="author">Ingle CE</span>, <span class="author">Beazley C</span>, <span class="author">Miller W</span>, <span class="author">Hurles ME</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p><p class="abstract"><strong>Background</strong>: Gene regulation is considered one of the driving forces of evolution. Although protein-coding DNA sequences and RNA genes have been subject to recent evolutionary events in the human lineage, it has been hypothesized that the large phenotypic divergence between humans and chimpanzees has been driven mainly by changes in gene regulation rather than altered protein-coding gene sequences. Comparative analysis of vertebrate genomes has revealed an abundance of evolutionarily conserved but noncoding sequences. These conserved noncoding (CNC) sequences may well harbor critical regulatory variants that have driven recent human evolution.</p><p class="abstract"><strong>Results</strong>: Here we identify 1,356 CNC sequences that appear to have undergone dramatic human-specific changes in selective pressures, at least 15% of which have substitution rates significantly above that expected under neutrality. The 1,356 &#39;accelerated CNC&#39; (ANC) sequences are enriched in recent segmental duplications, suggesting a recent change in selective constraint following duplication. In addition, single nucleotide polymorphisms within ANC sequences have a significant excess of high frequency derived alleles and high F(ST) values relative to controls, indicating that acceleration and positive selection are recent in human populations. Finally, a significant number of single nucleotide polymorphisms within ANC sequences are associated with changes in gene expression. The probability of variation in an ANC sequence being associated with a gene expression phenotype is fivefold higher than variation in a control CNC sequence.</p><p class="abstract"><strong>Conclusion</strong>: Our analysis suggests that ANC sequences have until very recently played a role in human evolution, potentially through lineage-specific changes in gene regulation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">R118</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17578567" rel="external">17578567</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2394770" rel="external">2394770</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-6-r118" rel="external">10.1186/gb-2007-8-6-r118</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cell attachment properties and infectivity of host-adapted and environmentally adapted Citrobacter rodentium.</h4>
<p class="authors"><span class="author">Bishop AL</span>, <span class="author">Wiles S</span>, <span class="author">Dougan G</span> and <span class="author">Frankel G</span></p>
<p class="affiliation">Division of Cell and Molecular Biology, Flowers Building, Imperial College London, Exhibition Road, London SW7 2AZ, UK.</p><p class="abstract">Citrobacter rodentium belongs to a family of extracellular enteric pathogens that include enterohaemorrhagic and enteropathogenic Escherichia coli, which colonises the gastrointestinal mucosa by the attaching and effacing (A/E) mechanism. We previously described the appearance of a &#39;hyper-infectious&#39; state after passage of C. rodentium through the murine gastrointestinal tract. Here we report that host-adapted C. rodentium is able to efficiently adhere and trigger actin polymerisation on cultured epithelial cells. Consistent with these observations we recorded higher levels of expression of genes carried on the LEE pathogenicity island and type III secretion system effector genes carried on prophages compared with in vitro-grown bacteria; importantly, the level of ler gene expression was unchanged. These phenotypes were lost after shed C. rodentium was adapted to the external environment. Upon exposure of C57Bl/6 mice, environmentally adapted C. rodentium was no longer infectious at the low doses associated with host-adapted bacteria and the bacteria were found to be localised in the caecal patch in a similar way to C. rodentium cultured in laboratory media. Thus, the &#39;hyper-infectious&#39; host-adapted state, allowing efficient transmission and colonisation of naive hosts, is transient in nature and gradually lost after shedding into the environment.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">071006</span></p>
<p class="publication"><span class="title">Microbes and infection / Institut Pasteur</span>&nbsp;<span class="year">2007</span>;<span class="volume">9</span>;<span class="number">11</span>;<span class="pagerange">1316-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17890132" rel="external">17890132</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.micinf.2007.06.006" rel="external">10.1016/j.micinf.2007.06.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">My 2,000 best films: parallel phenotyping of Dictyostelium development.</h4>
<p class="authors"><span class="author">Bloomfield G</span> and <span class="author">Kay RR</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge CB2 OQH, UK.</p><p class="abstract">A new study has used parallel filming to record the development of 2,000 Dictyostelium mutants, and clustered them into related groups using morphological staging and wavelet analysis of aggregation patterns.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">220</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17666111" rel="external">17666111</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2323213" rel="external">2323213</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-7-220" rel="external">10.1186/gb-2007-8-7-220</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome plasticity of BCG and impact on vaccine efficacy.</h4>
<p class="authors"><span class="author">Brosch R</span>, <span class="author">Gordon SV</span>, <span class="author">Garnier T</span>, <span class="author">Eiglmeier K</span>, <span class="author">Frigui W</span>, <span class="author">Valenti P</span>, <span class="author">Dos Santos S</span>, <span class="author">Duthoy S</span>, <span class="author">Lacroix C</span>, <span class="author">Garcia-Pelayo C</span>, <span class="author">Inwald JK</span>, <span class="author">Golby P</span>, <span class="author">Garcia JN</span>, <span class="author">Hewinson RG</span>, <span class="author">Behr MA</span>, <span class="author">Quail MA</span>, <span class="author">Churcher C</span>, <span class="author">Barrell BG</span>, <span class="author">Parkhill J</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne, Institut Pasteur, 28 Rue du Docteur Roux, 75724 Paris Cedex 15, France.</p><p class="abstract">To understand the evolution, attenuation, and variable protective efficacy of bacillus Calmette-Gu&eacute;rin (BCG) vaccines, Mycobacterium bovis BCG Pasteur 1173P2 has been subjected to comparative genome and transcriptome analysis. The 4,374,522-bp genome contains 3,954 protein-coding genes, 58 of which are present in two copies as a result of two independent tandem duplications, DU1 and DU2. DU1 is restricted to BCG Pasteur, although four forms of DU2 exist; DU2-I is confined to early BCG vaccines, like BCG Japan, whereas DU2-III and DU2-IV occur in the late vaccines. The glycerol-3-phosphate dehydrogenase gene, glpD2, is one of only three genes common to all four DU2 variants, implying that BCG requires higher levels of this enzyme to grow on glycerol. Further amplification of the DU2 region is ongoing, even within vaccine preparations used to immunize humans. An evolutionary scheme for BCG vaccines was established by analyzing DU2 and other markers. Lesions in genes encoding sigma-factors and pleiotropic transcriptional regulators, like PhoR and Crp, were also uncovered in various BCG strains; together with gene amplification, these affect gene expression levels, immunogenicity, and, possibly, protection against tuberculosis. Furthermore, the combined findings suggest that early BCG vaccines may even be superior to the later ones that are more widely used.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">13</span>;<span class="pagerange">5596-601</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17372194">17372194</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1838518">1838518</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0700869104">10.1073/pnas.0700869104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Avian genomics in the 21st century.</h4>
<p class="authors"><span class="author">Burt DW</span> and <span class="author">White SJ</span></p>
<p class="affiliation">Department of Genomics and Genetics, Roslin Institute (Edinburgh), Roslin, Midlothian, UK. <a href="mailto:%44%61%76%65.%42%75%72%74%40%62%62%73%72%63.%61%63.%75%6B">&#68;&#97;&#118;&#101;&#46;&#66;&#117;&#114;&#116;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The chicken has long been an important model organism for developmental biology, as well as a major source of protein with billions of birds used in meat and egg production each year. Chicken genomics has been transformed in recent years, with the characterisation of large EST collections and most recently with the assembly of the chicken genome sequence. As the first livestock genome to be fully sequenced it leads the way for others to follow--with zebra finch later this year. The genome sequence and the availability of three million genetic polymorphisms are expected to aid the identification of genes that control traits of importance in poultry. As the first bird genome to be sequenced it is a model for the remaining 9,600 species thought to exist today. Many of the features of avian biology and organisation of the chicken genome make it an ideal model organism for phylogenetics and embryology, along with applications in agriculture and medicine. The availability of new tools such as whole-genome gene expression arrays and SNP panels, coupled with information resources on the genes and proteins are likely to enhance this position.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">117</span>;<span class="number">1-4</span>;<span class="pagerange">6-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17675839">17675839</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000103159">10.1159/000103159</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of an inducible and optimized piggyBac transposon system.</h4>
<p class="authors"><span class="author">Cadi&ntilde;anos J</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Genomic studies in the mouse have been slowed by the lack of transposon-mediated mutagenesis. However, since the resurrection of Sleeping Beauty (SB), the possibility of performing forward genetics in mice has been reinforced. Recently, piggyBac (PB), a functional transposon from insects, was also described to work in mammals. As the activity of PB is higher than that of SB11 and SB12, two hyperactive SB transposases, we have characterized and improved the PB system in mouse ES cells. We have generated a mouse codon-optimized version of the PB transposase coding sequence (CDS) which provides transposition levels greater than the original. We have also found that the promoter sequence predicted in the 5&#39;-terminal repeat of the PB transposon is active in the mammalian context. Finally, we have engineered inducible versions of the optimized piggyBac transposase fused with ERT2. One of them, when induced, provides higher levels of transposition than the native piggyBac CDS, whereas in the absence of induction its activity is indistinguishable from background. We expect that these tools, adaptable to perform mouse-germline mutagenesis, will facilitate the identification of genes involved in pathological and physiological processes, such as cancer or ES cell differentiation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">12</span>;<span class="pagerange">e87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17576687">17576687</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1919496">1919496</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm446">10.1093/nar/gkm446</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Loss of the mismatch repair protein MSH6 in human glioblastomas is associated with tumor progression during temozolomide treatment.</h4>
<p class="authors"><span class="author">Cahill DP</span>, <span class="author">Levine KK</span>, <span class="author">Betensky RA</span>, <span class="author">Codd PJ</span>, <span class="author">Romany CA</span>, <span class="author">Reavie LB</span>, <span class="author">Batchelor TT</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Curry WT</span>, <span class="author">Iafrate AJ</span> and <span class="author">Louis DN</span></p>
<p class="affiliation">Molecular Pathology Unit, Neurosurgical Service, Brain Tumor Center, and Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA.</p><p class="abstract"><strong>Purpose</strong>: Glioblastomas are treated by surgical resection followed by radiotherapy [X-ray therapy (XRT)] and the alkylating chemotherapeutic agent temozolomide. Recently, inactivating mutations in the mismatch repair gene MSH6 were identified in two glioblastomas recurrent post-temozolomide. Because mismatch repair pathway inactivation is a known mediator of alkylator resistance in vitro, these findings suggested that MSH6 inactivation was causally linked to these two recurrences. However, the extent of involvement of MSH6 in glioblastoma is unknown. We sought to determine the overall frequency and clinical relevance of MSH6 alterations in glioblastomas.</p><p class="abstract">The MSH6 gene was sequenced in 54 glioblastomas. MSH6 and O(6)-methylguanine methyltransferase (MGMT) immunohistochemistry was systematically scored in a panel of 46 clinically well-characterized glioblastomas, and the corresponding patient response to treatment evaluated.</p><p class="abstract"><strong>Results</strong>: MSH6 mutation was not observed in any pretreatment glioblastoma (0 of 40), whereas 3 of 14 recurrent cases had somatic mutations (P = 0.015). MSH6 protein expression was detected in all pretreatment (17 of 17) cases examined but, notably, expression was lost in 7 of 17 (41%) recurrences from matched post-XRT + temozolomide cases (P = 0.016). Loss of MSH6 was not associated with O(6)-methylguanine methyltransferase status. Measurements of in vivo tumor growth using three-dimensional reconstructed magnetic resonance imaging showed that MSH6-negative glioblastomas had a markedly increased rate of growth while under temozolomide treatment (3.17 versus 0.04 cc/mo for MSH6-positive tumors; P = 0.020).</p><p class="abstract"><strong>Conclusions</strong>: Loss of MSH6 occurs in a subset of post-XRT + temozolomide glioblastoma recurrences and is associated with tumor progression during temozolomide treatment, mirroring the alkylator resistance conferred by MSH6 inactivation in vitro. MSH6 deficiency may therefore contribute to the emergence of recurrent glioblastomas during temozolomide treatment.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Clinical cancer research : an official journal of the American Association for Cancer Research</span>&nbsp;<span class="year">2007</span>;<span class="volume">13</span>;<span class="number">7</span>;<span class="pagerange">2038-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17404084" rel="external">17404084</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873832" rel="external">2873832</a>; DOI: <a href="http://dx.doi.org/10.1158%2F1078-0432.CCR-06-2149" rel="external">10.1158/1078-0432.CCR-06-2149</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Methods and strategies for analyzing copy number variation using DNA microarrays.</h4>
<p class="authors"><span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6E%70%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#112;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The association of DNA copy-number variation (CNV) with specific gene function and human disease has been long known, but the wide scope and prevalence of this form of variation has only recently been fully appreciated. The latest studies using microarray technology have demonstrated that as much as 12% of the human genome and thousands of genes are variable in copy number, and this diversity is likely to be responsible for a significant proportion of normal phenotypic variation. Current challenges involve developing methods not only for detecting and cataloging CNVs in human populations at increasingly higher resolution but also for determining the association of CNVs with biological function, recent human evolution, and common and complex human disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7 Suppl</span>;<span class="pagerange">S16-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17597776">17597776</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2697494">2697494</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2028">10.1038/ng2028</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Altered phenotype and gene transcription in endothelial cells, induced by Plasmodium falciparum-infected red blood cells: pathogenic or protective?</h4>
<p class="authors"><span class="author">Chakravorty SJ</span>, <span class="author">Carret C</span>, <span class="author">Nash GB</span>, <span class="author">Ivens A</span>, <span class="author">Szestak T</span> and <span class="author">Craig AG</span></p>
<p class="affiliation">Molecular &amp; Biochemical Parasitology, Liverpool School of Tropical Medicine, University of Liverpool, Liverpool, L3 5QA, United Kingdom. <a href="mailto:%63%68%61%6B%72%61%73%6A%40%6C%69%76%65%72%70%6F%6F%6C.%61%63.%75%6B">&#99;&#104;&#97;&#107;&#114;&#97;&#115;&#106;&#64;&#108;&#105;&#118;&#101;&#114;&#112;&#111;&#111;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Severe malaria is associated with sequestration of Plasmodium falciparum-infected red blood cells (PRBC) in the microvasculature and elevation of intercellular adhesion molecule-1 (ICAM-1) and TNF. In vitro co-culture of human umbilical vein endothelial cells (HUVEC), with either PRBC or uninfected RBC, required the presence of low level TNF (5pg/ml) for significant up-regulation of ICAM-1, which may contribute to increased cytoadhesion in vivo. These effects were independent of P. falciparum erythrocyte membrane protein-1 (PfEMP-1)-mediated adhesion but critically dependent on cell-cell contact. Further changes included increases in IL8 release and soluble TNF receptor shedding. Microarray analysis of HUVEC transcriptome following co-culture, using a human Affymetrix microarray chip, showed significant differential regulation of genes which defined gene ontologies such as cell communication, cell adhesion, signal transduction and immune response. Our data demonstrate that endothelial cells have the ability to mobilise immune and pro-adhesive responses when exposed to both PRBC and TNF. In addition, there is also a previously un-described positive regulation by RBC and TNF and a concurrent negative regulation of a range of genes involved in inflammation and cell-death, by PRBC and TNF. We propose that the balance between positive and negative regulation demonstrated in our study will determine endothelial pathology during a malaria infection.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2007</span>;<span class="volume">37</span>;<span class="number">8-9</span>;<span class="pagerange">975-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17383656" rel="external">17383656</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1906861" rel="external">1906861</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijpara.2007.02.006" rel="external">10.1016/j.ijpara.2007.02.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A recombineering based approach for high-throughput conditional knockout targeting vector construction.</h4>
<p class="authors"><span class="author">Chan W</span>, <span class="author">Costantino N</span>, <span class="author">Li R</span>, <span class="author">Lee SC</span>, <span class="author">Su Q</span>, <span class="author">Melvin D</span>, <span class="author">Court DL</span> and <span class="author">Liu P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK.</p><p class="abstract">Functional analysis of mammalian genes in vivo is primarily achieved through analysing knockout mice. Now that the sequencing of several mammalian genomes has been completed, understanding functions of all the genes represents the next major challenge in the post-genome era. Generation of knockout mutant mice has currently been achieved by many research groups but only by making individual knockouts, one by one. New technological advances and the refinements of existing technologies are critical for genome-wide targeted mutagenesis in the mouse. We describe here new recombineering reagents and protocols that enable recombineering to be carried out in a 96-well format. Consequently, we are able to construct 96 conditional knockout targeting vectors simultaneously. Our new recombineering system makes it a reality to generate large numbers of precisely engineered DNA constructs for functional genomics studies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">8</span>;<span class="pagerange">e64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17426124">17426124</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1885671">1885671</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm163">10.1093/nar/gkm163</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Serodiagnosis of Salmonella enterica serovar Typhi and S. enterica serovars Paratyphi A, B and C human infections.</h4>
<p class="authors"><span class="author">Chart H</span>, <span class="author">Cheasty T</span>, <span class="author">de Pinna E</span>, <span class="author">Siorvanes L</span>, <span class="author">Wain J</span>, <span class="author">Alam D</span>, <span class="author">Nizami Q</span>, <span class="author">Bhutta Z</span> and <span class="author">Threlfall EJ</span></p>
<p class="affiliation">Laboratory of Enteric Pathogens, Department of Gastrointestinal Infections, Centre for Infections, Health Protection Agency, 61 Colindale Avenue, London NW9 5EQ, UK. <a href="mailto:%48%65%6E%72%69%6B.%43%68%61%72%74%40%68%70%61.%6F%72%67.%75%6B">&#72;&#101;&#110;&#114;&#105;&#107;&#46;&#67;&#104;&#97;&#114;&#116;&#64;&#104;&#112;&#97;&#46;&#111;&#114;&#103;&#46;&#117;&#107;</a></p><p class="abstract">The aim of this study was to evaluate an immunoassay for the detection of human serum antibodies to the LPS and flagellar antigens of Salmonella Typhi and Salmonella Paratyphi A, B and C, and to the Vi capsular polysaccharide of S. Typhi and S. Paratyphi C. A total of 330 sera were used; these originated from 15 patients who were culture-positive for S. Typhi and 15 healthy controls, together with 300 sera submitted to the Laboratory of Enteric Pathogens for Salmonella serodiagnosis. By SDS-PAGE/immunoblotting, all 15 sera from culture-positive patients had serum antibodies to the 9,12 LPS antigens and 10 had antibodies to the &#39;d&#39; flagellar antigens. Of the 300 reference sera, 22 had antibodies to the 9,12 LPS antigens, one to the 1,4,5,12 LPS antigens and 12 to the 6,7 LPS antigens. Only two sera had antibodies to flagellar antigens, one of which bound to the &#39;b&#39; and the other to the &#39;d&#39; antigen. An ELISA was developed that successfully detected serum antibodies to the Vi capsular polysaccharides, but because of the kinetics of serum antibody production to the Vi, these antibodies may be of limited value in the serodiagnosis of acute infection with S. Typhi and S. Paratyphi C. The immunoassays described here provide a sensitive means of detecting serum antibodies to the LPS, flagellar and Vi antigens of S. Typhi and S. Paratyphi, and constitute a viable replacement for the Widal assay for the screening of sera. The Salmonella serodiagnosis protocols described here are the new standard operating procedures used by the Health Protection Agency&#39;s National Salmonella Reference Centre based in the Laboratory of Enteric Pathogens, Colindale, UK.</p>
<p class="publication"><span class="title">Journal of medical microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">Pt 9</span>;<span class="pagerange">1161-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17761477">17761477</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fjmm.0.47197-0">10.1099/jmm.0.47197-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antimicrobial drug resistance of Salmonella enterica serovar typhi in asia and molecular mechanism of reduced susceptibility to the fluoroquinolones.</h4>
<p class="authors"><span class="author">Chau TT</span>, <span class="author">Campbell JI</span>, <span class="author">Galindo CM</span>, <span class="author">Van Minh Hoang N</span>, <span class="author">Diep TS</span>, <span class="author">Nga TT</span>, <span class="author">Van Vinh Chau N</span>, <span class="author">Tuan PQ</span>, <span class="author">Page AL</span>, <span class="author">Ochiai RL</span>, <span class="author">Schultsz C</span>, <span class="author">Wain J</span>, <span class="author">Bhutta ZA</span>, <span class="author">Parry CM</span>, <span class="author">Bhattacharya SK</span>, <span class="author">Dutta S</span>, <span class="author">Agtini M</span>, <span class="author">Dong B</span>, <span class="author">Honghui Y</span>, <span class="author">Anh DD</span>, <span class="author">Canh do G</span>, <span class="author">Naheed A</span>, <span class="author">Albert MJ</span>, <span class="author">Phetsouvanh R</span>, <span class="author">Newton PN</span>, <span class="author">Basnyat B</span>, <span class="author">Arjyal A</span>, <span class="author">La TT</span>, <span class="author">Rang NN</span>, <span class="author">Phuong le T</span>, <span class="author">Van Be Bay P</span>, <span class="author">von Seidlein L</span>, <span class="author">Dougan G</span>, <span class="author">Clemens JD</span>, <span class="author">Vinh H</span>, <span class="author">Hien TT</span>, <span class="author">Chinh NT</span>, <span class="author">Acosta CJ</span>, <span class="author">Farrar J</span> and <span class="author">Dolecek C</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Hospital for Tropical Diseases, 190 Ben Ham Tu, Ho Chi Minh City, Vietnam.</p><p class="abstract">This study describes the pattern and extent of drug resistance in 1,774 strains of Salmonella enterica serovar Typhi isolated across Asia between 1993 and 2005 and characterizes the molecular mechanisms underlying the reduced susceptibilities to fluoroquinolones of these strains. For 1,393 serovar Typhi strains collected in southern Vietnam, the proportion of multidrug resistance has remained high since 1993 (50% in 2004) and there was a dramatic increase in nalidixic acid resistance between 1993 (4%) and 2005 (97%). In a cross-sectional sample of 381 serovar Typhi strains from 8 Asian countries, Bangladesh, China, India, Indonesia, Laos, Nepal, Pakistan, and central Vietnam, collected in 2002 to 2004, various rates of multidrug resistance (16 to 37%) and nalidixic acid resistance (5 to 51%) were found. The eight Asian countries involved in this study are home to approximately 80% of the world&#39;s typhoid fever cases. These results document the scale of drug resistance across Asia. The Ser83--&gt;Phe substitution in GyrA was the predominant alteration in serovar Typhi strains from Vietnam (117/127 isolates; 92.1%). No mutations in gyrB, parC, or parE were detected in 55 of these strains. In vitro time-kill experiments showed a reduction in the efficacy of ofloxacin against strains harboring a single-amino-acid substitution at codon 83 or 87 of GyrA; this effect was more marked against a strain with a double substitution. The 8-methoxy fluoroquinolone gatifloxacin showed rapid killing of serovar Typhi harboring both the single- and double-amino-acid substitutions.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2007</span>;<span class="volume">51</span>;<span class="number">12</span>;<span class="pagerange">4315-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17908946" rel="external">17908946</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2167998" rel="external">2167998</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.00294-07" rel="external">10.1128/AAC.00294-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multilocus sequence typing analysis of Shigella flexneri isolates collected in Asian countries.</h4>
<p class="authors"><span class="author">Choi SY</span>, <span class="author">Jeon YS</span>, <span class="author">Lee JH</span>, <span class="author">Choi B</span>, <span class="author">Moon SH</span>, <span class="author">von Seidlein L</span>, <span class="author">Clemens JD</span>, <span class="author">Dougan G</span>, <span class="author">Wain J</span>, <span class="author">Yu J</span>, <span class="author">Lee JC</span>, <span class="author">Seol SY</span>, <span class="author">Lee BK</span>, <span class="author">Song JH</span>, <span class="author">Song M</span>, <span class="author">Czerkinsky C</span>, <span class="author">Chun J</span> and <span class="author">Kim DW</span></p>
<p class="affiliation">1International Vaccine Institute, San 4-8 Bongcheon 7 dong, Kwanak gu, Seoul 151-818, Republic of Korea.</p><p class="abstract">The multilocus sequence typing scheme used previously for phylogenetic analysis of Escherichia coli was applied to 107 clinical isolates of Shigella flexneri. DNA sequencing of 3423 bp throughout seven housekeeping genes identified eight new allele types and ten new sequence types among the isolates. S. flexneri serotypes 1-5, X and Y were clustered together in a group containing many allelic variants while serotype 6 formed a distinct group, as previously established.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Journal of medical microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">Pt 11</span>;<span class="pagerange">1460-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17965345" rel="external">17965345</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652033" rel="external">2652033</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fjmm.0.47322-0" rel="external">10.1099/jmm.0.47322-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Activating transcription factor 6 (ATF6) sequence polymorphisms in type 2 diabetes and pre-diabetic traits.</h4>
<p class="authors"><span class="author">Chu WS</span>, <span class="author">Das SK</span>, <span class="author">Wang H</span>, <span class="author">Chan JC</span>, <span class="author">Deloukas P</span>, <span class="author">Froguel P</span>, <span class="author">Baier LJ</span>, <span class="author">Jia W</span>, <span class="author">McCarthy MI</span>, <span class="author">Ng MC</span>, <span class="author">Damcott C</span>, <span class="author">Shuldiner AR</span>, <span class="author">Zeggini E</span> and <span class="author">Elbein SC</span></p>
<p class="affiliation">Division of Endocrinology 111J-1/LR, Department of Medicine, University of Arkansas for Medical Sciences, John L. McClellan Memorial Veterans Hospital, 4700 W. 7th Street, Little Rock, AR 72205, USA.</p><p class="abstract">Activating transcription factor 6 (ATF6) is located within the region of linkage to type 2 diabetes on chromosome 1q21-q23 and is a key activator of the endoplasmic reticulum stress response. We evaluated 78 single nucleotide polymorphisms (SNPs) spanning &gt;213 kb in 95 people, from which we selected 64 SNPs for evaluation in 191 Caucasian case subjects from Utah and between 165 and 188 control subjects. Six SNPs showed nominal associations with type 2 diabetes (P = 0.001-0.04), including the nonsynonymous SNP rs1058405 (M67V) in exon 3 and rs11579627 in the 3&#39; flanking region. Only rs1159627 remained significant on permutation testing. The associations were not replicated in 353 African-American case subjects and 182 control subjects, nor were ATF6 SNPs associated with altered insulin secretion or insulin sensitivity in nondiabetic Caucasian individuals. No association with type 2 diabetes was found in a subset of 44 SNPs in Caucasian (n = 2,099), Pima Indian (n = 293), and Chinese (n = 287) samples. Allelic expression imbalance was found in transformed lymphocyte cDNA for 3&#39; untranslated region variants, thus suggesting cis-acting regulatory variants. ATF6 does not appear to play a major role in type 2 diabetes, but further work is required to identify the cause of the allelic expression imbalance.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">R01 DK039311-24</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">079557</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">3</span>;<span class="pagerange">856-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17327457" rel="external">17327457</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672156" rel="external">2672156</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb06-1305" rel="external">10.2337/db06-1305</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Validation of a new prognostic index for advanced epithelial ovarian cancer: results from its application to a UK-based cohort.</h4>
<p class="authors"><span class="author">Clark TG</span>, <span class="author">Stewart M</span>, <span class="author">Rye T</span>, <span class="author">Smyth JF</span> and <span class="author">Gourley C</span></p>
<p class="publication"><span class="title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>&nbsp;<span class="year">2007</span>;<span class="volume">25</span>;<span class="number">35</span>;<span class="pagerange">5669-70; author reply 5670-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18065745" rel="external">18065745</a>; DOI: <a href="http://dx.doi.org/10.1200%2FJCO.2007.14.2976" rel="external">10.1200/JCO.2007.14.2976</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomix: a method for combining gene-finders&#39; predictions, which uses evolutionary conservation of sequence and intron-exon structure.</h4>
<p class="authors"><span class="author">Coghlan A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%61%6C%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: Correct gene predictions are crucial for most analyses of genomes. However, in the absence of transcript data, gene prediction is still challenging. One way to improve gene-finding accuracy in such genomes is to combine the exons predicted by several gene-finders, so that gene-finders that make uncorrelated errors can correct each other.</p><p class="abstract"><strong>Results</strong>: We present a method for combining gene-finders called Genomix. Genomix selects the predicted exons that are best conserved within and/or between species in terms of sequence and intron-exon structure, and combines them into a gene structure. Genomix was used to combine predictions from four gene-finders for Caenorhabditis elegans, by selecting the predicted exons that are best conserved with C.briggsae and C.remanei. On a set of approximately 1500 confirmed C.elegans genes, Genomix increased the exon-level specificity by 10.1% and sensitivity by 2.7% compared to the best input gene-finder.</p><p class="abstract"><strong>Availability</strong>: Scripts and Supplementary Material can be found at http://www.sanger.ac.uk/Software/analysis/genomix</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077192</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">12</span>;<span class="pagerange">1468-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17483502">17483502</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2880447">2880447</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm133">10.1093/bioinformatics/btm133</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Supramolecular signalling complexes in the nervous system.</h4>
<p class="authors"><span class="author">Collins MO</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Proteomic Mass Spectrometry, The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%6D%6F%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#111;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">It is now apparent that multiprotein signalling complexes or &quot;signalling machines&quot; are responsible for orchestrating many complex signalling pathways in the cell. The synapse is a sub-cellular specialisation which transmits and converts patterns of electrical activity into cellular memory. This processing of electrical information is mediated by the protein components of the synapse. The organisation of synaptic proteins has been investigated over the last number of years using proteomic methods and with the application ofbioinformatics; a landscape of modular protein complexes at the synapse is emerging. Many share a common organisation centred on a receptor/channel, a protein scaffold, (in which the signalling molecules are localised) and membrane to cytoskeleton interactions. The use of PDZ-domain based protein scaffolds is a particularly common feature in the construction of neuronal protein complexes and the differential presence of these proteins in complexes can have functional consequences. Here we overview current proteomic methodologies for the analysis of multiprotein complexes. In addition, we describe the characterisation of a number of multiprotein complexes associated with ion channels (NMDAR, P2X7 and Kir2) and GPCRs (5-HT2A/5-HT2C, D2 and mGluR5) and discuss common their common components and organisation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Sub-cellular biochemistry</span>&nbsp;<span class="year">2007</span>;<span class="volume">43</span>;<span class="pagerange">185-207</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17953395" rel="external">17953395</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of protein phosphorylation on a proteome-scale.</h4>
<p class="authors"><span class="author">Collins MO</span>, <span class="author">Yu L</span> and <span class="author">Choudhary JS</span></p>
<p class="affiliation">Proteomic Mass Spectrometry, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%6D%6F%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#111;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Phosphorylation, the most intensively studied and common PTM on proteins, is a complex biological phenomenon. Its complexity manifests itself in the large numbers of proteins that attach it, remove it and recognise it as a protein code. Since the first report of protein phosphorylation on vitellin 100 years ago, a wide variety of biochemical and analytical chemical approaches have been developed to enrich and detect protein phosphorylation. The last 5 years have witnessed a renaissance in methodologies capable of characterising protein phosphorylation on a proteome-scale. These technological advances have allowed identification of hundreds to thousands of phosphorylation sites in a proteome and have resulted in a profound paradigm shift. For the first time, using quantitative MS, the topology and significance of global phosphorylation networks may be investigated, marking a new era of cell signalling research. This review addresses recent technological advances in the purification of phosphorylated proteins and peptides and current MS-based strategies used to qualitatively and quantitatively probe these enriched phosphoproteomes. In addition, we review the application of complementary array-based technologies to derive signalling networks from kinase-substrate interactions and discuss future challenges in the field.</p>
<p class="publication"><span class="title">Proteomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="number">16</span>;<span class="pagerange">2751-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17703509" rel="external">17703509</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpmic.200700145" rel="external">10.1002/pmic.200700145</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Adiponectin receptor genes: mutation screening in syndromes of insulin resistance and association studies for type 2 diabetes and metabolic traits in UK populations.</h4>
<p class="authors"><span class="author">Collins SC</span>, <span class="author">Luan J</span>, <span class="author">Thompson AJ</span>, <span class="author">Daly A</span>, <span class="author">Semple RK</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Wareham NJ</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Metabolic Disease Group, The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Adiponectin is an adipokine with insulin-sensitising and anti-atherogenic properties. Several reports suggest that genetic variants in the adiponectin gene are associated with circulating levels of adiponectin, insulin sensitivity and type 2 diabetes risk. Recently two receptors for adiponectin have been cloned. Genetic studies have yielded conflicting results on the role of these genes and type 2 diabetes predisposition. In this study we aimed to evaluate the potential role of genetic variation in these genes in syndromes of severe insulin resistance, type 2 diabetes and in related metabolic traits in UK Europid populations.</p><p class="abstract">Exons and splice junctions of the adiponectin receptor 1 and 2 genes (ADIPOR1; ADIPOR2) were sequenced in patients from our severe insulin resistance cohort (n=129). Subsequently, 24 polymorphisms were tested for association with type 2 diabetes in population-based type 2 diabetes case-control studies (n=2,127) and with quantitative traits in a population-based longitudinal study (n=1,721).</p><p class="abstract"><strong>Results</strong>: No missense or nonsense mutations in ADIPOR1 and ADIPOR2 were detected in the cohort of patients with severe insulin resistance. None of the 24 polymorphisms (allele frequency 2.3-48.3%) tested was associated with type 2 diabetes in the case-control study. Similarly, none of the polymorphisms was associated with fasting plasma insulin, fasting and 2-h post-load plasma glucose, 30-min insulin increment or BMI.</p><p class="abstract">Genetic variation in ADIPOR1 and ADIPOR2 is not a major cause of extreme insulin resistance in humans, nor does it contribute in a significant manner to type 2 diabetes risk and related traits in UK Europid populations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2007</span>;<span class="volume">50</span>;<span class="number">3</span>;<span class="pagerange">555-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17216283">17216283</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1794135">1794135</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-006-0534-7">10.1007/s00125-006-0534-7</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The population genetics of structural variation.</h4>
<p class="authors"><span class="author">Conrad DF</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Department of Human Genetics, University of Chicago, Chicago, Illinois 60637, USA.</p><p class="abstract">Population genetics is central to our understanding of human variation, and by linking medical and evolutionary themes, it enables us to understand the origins and impacts of our genomic differences. Despite current limitations in our knowledge of the locations, sizes and mutational origins of structural variants, our characterization of their population genetics is developing apace, bringing new insights into recent human adaptation, genome biology and disease. We summarize recent dramatic advances, describe the diverse mutational origins of chromosomal rearrangements and argue that their complexity necessitates a re-evaluation of existing population genetic methods.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077014</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7 Suppl</span>;<span class="pagerange">S30-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17597779">17597779</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716079">2716079</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2042">10.1038/ng2042</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The missing care bundle: antibiotic prescribing in hospitals.</h4>
<p class="authors"><span class="author">Cooke FJ</span> and <span class="author">Holmes AH</span></p>
<p class="affiliation">Department of Infectious Diseases and Immunology, Hammersmith Hospitals NHS Trust, Imperial College, Du Cane Road, London W12 0HS, UK. <a href="mailto:%66%69%6F%6E%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#105;&#111;&#110;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The care bundle involves grouping together key elements of care for procedures and the management of specific diagnoses in order to provide a systematic method to improve and monitor the delivery of clinical care processes. In short, care bundles aim to ensure that all patients consistently receive the best care or treatment, all of the time. This approach has been successfully applied to the management of various conditions, particularly in the critical care setting. The Institute for Healthcare Improvement&#39;s &#39;100K lives campaign&#39; consisted of six care bundles, three of which have addressed preventing hospital-acquired infection. The UK Department of Health&#39;s delivery programme to reduce healthcare-associated infections (HCAIs), including methicillin-resistant Staphylococcus aureus (MRSA), includes six &#39;high-impact interventions&#39;, which are care bundles to reduce HCAIs. However, we suggest that one key intervention is missing, and consider this intervention will be increasingly important if hospitals are to address the rising incidence of Clostridium difficile, to tackle antibiotic resistance and to improve patient care. The missing intervention addresses the process of antibiotic prescribing. We propose that the time is right to consider the application of the care bundle approach to improve the prescribing of antibiotics, both for treatment and prophylaxis.</p>
<p class="publication"><span class="title">International journal of antimicrobial agents</span>&nbsp;<span class="year">2007</span>;<span class="volume">30</span>;<span class="number">1</span>;<span class="pagerange">25-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17499482" rel="external">17499482</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijantimicag.2007.03.003" rel="external">10.1016/j.ijantimicag.2007.03.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cases of typhoid fever imported into England, Scotland and Wales (2000-2003).</h4>
<p class="authors"><span class="author">Cooke FJ</span>, <span class="author">Day M</span>, <span class="author">Wain J</span>, <span class="author">Ward LR</span> and <span class="author">Threlfall EJ</span></p>
<p class="affiliation">Health Protection Agency, Centre for Infections, 61 Colindale Avenue, London, UK.</p><p class="abstract">Although typhoid fever is no longer endemic in most of the developed world, it remains a major infectious disease in less developed regions and imported cases continue to occur in returning travellers, immigrants or migrant workers. We analysed all 692 isolates of Salmonella enterica subspecies enterica serovar Typhi from cases in England, Scotland and Wales that were sent to the Laboratory of Enteric Pathogens at the Health Protection Agency, Centre for Infections, London, UK between 2000 and 2003. The country of acquisition was known for 416 isolates (60%), and the majority of these (70%) came from India or Pakistan. Overall, 24 countries were listed, mainly in Asia and Africa. A total of 48 phage types were detected, 41% of which were Vi-phage type E1. Antimicrobial susceptibility testing revealed that 22% of isolates were multidrug resistant (MDR) (defined as resistance to chloramphenicol, ampicillin and co-trimoxazole) and 39% were quinolone resistant. A significant number of isolates (n=49) were sensitive to nalidixic acid by disk test but exhibited low-level ciprofloxacin resistance, suggesting a novel mechanism of resistance and reinforcing the need for minimum inhibitory concentration determination. Overall, 13% of isolates were both MDR and likely to show a poor response to a fluoroquinolone. A third-generation cephalosporin (e.g. ceftriaxone) should be considered as empirical therapy in regions of the Indian subcontinent where resistance is now at high levels as well as in patients returning from these areas. This study helps to describe the epidemiology of antimicrobial drug resistance in typhoid fever.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Transactions of the Royal Society of Tropical Medicine and Hygiene</span>&nbsp;<span class="year">2007</span>;<span class="volume">101</span>;<span class="number">4</span>;<span class="pagerange">398-404</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17014877" rel="external">17014877</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.trstmh.2006.07.005" rel="external">10.1016/j.trstmh.2006.07.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prophage sequences defining hot spots of genome variation in Salmonella enterica serovar Typhimurium can be used to discriminate between field isolates.</h4>
<p class="authors"><span class="author">Cooke FJ</span>, <span class="author">Wain J</span>, <span class="author">Fookes M</span>, <span class="author">Ivens A</span>, <span class="author">Thomson N</span>, <span class="author">Brown DJ</span>, <span class="author">Threlfall EJ</span>, <span class="author">Gunn G</span>, <span class="author">Foster G</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%66%69%6F%6E%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#102;&#105;&#111;&#110;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Sixty-one Salmonella enterica serovar Typhimurium isolates of animal and human origin, matched by phage type, antimicrobial resistance pattern, and place of isolation, were analyzed by microbiological and molecular techniques, including pulsed-field gel electrophoresis (PFGE) and plasmid profiling. PFGE identified 10 profiles that clustered by phage type and antibiotic resistance pattern with human and animal isolates distributed among different PFGE profiles. Genomic DNA was purified from 23 representative strains and hybridized to the composite Salmonella DNA microarray, and specific genomic regions that exhibited significant variation between isolates were identified. Bioinformatic analysis showed that variable regions of DNA were associated with prophage-like elements. Subsequently, simple multiplex PCR assays were designed on the basis of these variable regions that could be used to discriminate between S. enterica serovar Typhimurium isolates from the same geographical region. These multiplex PCR assays, based on prophage-like elements and Salmonella genomic island 1, provide a simple method for identifying new variants of S. enterica serovar Typhimurium in the field.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of clinical microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">45</span>;<span class="number">8</span>;<span class="pagerange">2590-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17522270" rel="external">17522270</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1951247" rel="external">1951247</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJCM.00729-07" rel="external">10.1128/JCM.00729-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The candidate genes TAF5L, TCF7, PDCD1, IL6 and ICAM1 cannot be excluded from having effects in type 1 diabetes.</h4>
<p class="authors"><span class="author">Cooper JD</span>, <span class="author">Smyth DJ</span>, <span class="author">Bailey R</span>, <span class="author">Payne F</span>, <span class="author">Downes K</span>, <span class="author">Godfrey LM</span>, <span class="author">Masters J</span>, <span class="author">Zeitels LR</span>, <span class="author">Vella A</span>, <span class="author">Walker NM</span> and <span class="author">Todd JA</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, UK. <a href="mailto:%6A%61%73%6F%6E.%63%6F%6F%70%65%72%40%63%69%6D%72.%63%61%6D.%61%63.%75%6B">&#106;&#97;&#115;&#111;&#110;&#46;&#99;&#111;&#111;&#112;&#101;&#114;&#64;&#99;&#105;&#109;&#114;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: As genes associated with immune-mediated diseases have an increased prior probability of being associated with other immune-mediated diseases, we tested three such genes, IL23R, IRF5 and CD40, for an association with type 1 diabetes. In addition, we tested seven genes, TAF5L, PDCD1, TCF7, IL12B, IL6, ICAM1 and TBX21, with published marginal or inconsistent evidence of an association with type 1 diabetes.</p><p class="abstract"><strong>Methods</strong>: We genotyped reported polymorphisms of the ten genes, nonsynonymous SNPs (nsSNPs) and, for the IL12B and IL6 regions, tag SNPs in up to 7,888 case, 8,858 control and 3,142 parent-child trio samples. In addition, we analysed data from the Wellcome Trust Case Control Consortium genome-wide association study to determine whether there was any further evidence of an association in each gene region.</p><p class="abstract"><strong>Results</strong>: We found some evidence of associations between type 1 diabetes and TAF5L, PDCD1, TCF7 and IL6 (ORs = 1.05 - 1.13; P = 0.0291 - 4.16 x 10-4). No evidence of an association was obtained for IL12B, IRF5, IL23R, ICAM1, TBX21 and CD40, although there was some evidence of an association (OR = 1.10; P = 0.0257) from the genome-wide association study for the ICAM1 region.</p><p class="abstract"><strong>Conclusion</strong>: We failed to exclude the possibility of some effect in type 1 diabetes for TAF5L, PDCD1, TCF7, IL6 and ICAM1. Additional studies, of these and other candidate genes, employing much larger sample sizes and analysis of additional polymorphisms in each gene and its flanking region will be required to ascertain their contributions to type 1 diabetes susceptibility.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">076113</span></p>
<p class="publication"><span class="title">BMC medical genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18045485" rel="external">18045485</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2217539" rel="external">2217539</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2350-8-71" rel="external">10.1186/1471-2350-8-71</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenetic silencing of Plasmodium falciparum genes linked to erythrocyte invasion.</h4>
<p class="authors"><span class="author">Cort&eacute;s A</span>, <span class="author">Carret C</span>, <span class="author">Kaneko O</span>, <span class="author">Yim Lim BY</span>, <span class="author">Ivens A</span> and <span class="author">Holder AA</span></p>
<p class="affiliation">Division of Parasitology, Medical Research Council National Institute for Medical Research (NIMR), London, United Kingdom. <a href="mailto:%61%63%6F%72%74%65%73%40%70%63%62.%75%62.%65%73">&#97;&#99;&#111;&#114;&#116;&#101;&#115;&#64;&#112;&#99;&#98;&#46;&#117;&#98;&#46;&#101;&#115;</a></p><p class="abstract">The process of erythrocyte invasion by merozoites of Plasmodium falciparum involves multiple steps, including the formation of a moving junction between parasite and host cell, and it is characterised by the redundancy of many of the receptor-ligand interactions involved. Several parasite proteins that interact with erythrocyte receptors or participate in other steps of invasion are encoded by small subtelomerically located gene families of four to seven members. We report here that members of the eba, rhoph1/clag, acbp, and pfRh multigene families exist in either an active or a silenced state. In the case of two members of the rhoph1/clag family, clag3.1 and clag3.2, expression was mutually exclusive. Silencing was clonally transmitted and occurred in the absence of detectable DNA alterations, suggesting that it is epigenetic. This was demonstrated for eba-140. Our data demonstrate that variant or mutually exclusive expression and epigenetic silencing in Plasmodium are not unique to genes such as var, which encode proteins that are exported to the surface of the erythrocyte, but also occur for genes involved in host cell invasion. Clonal variant expression of invasion-related ligands increases the flexibility of the parasite to adapt to its human host.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL078826</span>; Wellcome Trust: <span class="grant">066742</span></p>
<p class="publication"><span class="title">PLoS pathogens</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">8</span>;<span class="pagerange">e107</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17676953" rel="external">17676953</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1937010" rel="external">1937010</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.ppat.0030107" rel="external">10.1371/journal.ppat.0030107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quorum sensing has an unexpected role in virulence in the model pathogen Citrobacter rodentium.</h4>
<p class="authors"><span class="author">Coulthurst SJ</span>, <span class="author">Clare S</span>, <span class="author">Evans TJ</span>, <span class="author">Foulds IJ</span>, <span class="author">Roberts KJ</span>, <span class="author">Welch M</span>, <span class="author">Dougan G</span> and <span class="author">Salmond GP</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, Tennis Court Road, Cambridge CB2 1QW, UK.</p><p class="abstract">The bacterial mouse pathogen Citrobacter rodentium causes attaching and effacing (AE) lesions in the same manner as pathogenic Escherichia coli, and is an important model for this mode of pathogenesis. Quorum sensing (QS) involves chemical signalling by bacteria to regulate gene expression in response to cell density. E. coli has never been reported to have N-acylhomoserine lactone (AHL) QS, but it does utilize luxS-dependent signalling. We found production of AHL QS signalling molecules by an AE pathogen, C. rodentium. AHL QS is directed by the croIR locus and a croI mutant is affected in its surface attachment, although not in Type III secretion. AHL QS has an important role in virulence in the mouse as, unexpectedly, the QS mutant is hypervirulent; by contrast, we detected no impact of luxS inactivation. Further study of QS in Citrobacter should provide new insights into AE pathogenesis. As the croIR locus might have been horizontally acquired, AHL QS might exist in some strains of pathogenic E. coli.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">EMBO reports</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">698-703</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17557113" rel="external">17557113</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1905894" rel="external">1905894</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.embor.7400984" rel="external">10.1038/sj.embor.7400984</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sink or swim.</h4>
<p class="authors"><span class="author">Crossman LC</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">11</span>;<span class="pagerange">834-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17939228">17939228</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1777">10.1038/nrmicro1777</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">It&#39;s hip to be square!</h4>
<p class="authors"><span class="author">Crossman LC</span> and <span class="author">Walker A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">400-1</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17508437" rel="external">17508437</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1677" rel="external">10.1038/nrmicro1677</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The continuing search for cancer-causing somatic mutations.</h4>
<p class="authors"><span class="author">Dalgliesh GL</span> and <span class="author">Futreal PA</span></p><p class="abstract">It is known that cancer is caused by an accumulation of mutations in DNA. Many genes have been associated with tumour progression either through germline or somatic mutations, but mutations in these genes by no means account for all instances of the disease. The availability of the completed human genome sequence and reduced costs of sequencing have allowed large-scale screens to uncover genes that are somatically mutated in cancer. In this issue, Chanock and colleagues present a screen of 91 breast cancers for somatic variants in a set of 21 genes.</p>
<p class="publication"><span class="title">Breast cancer research : BCR</span>&nbsp;<span class="year">2007</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">101</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17319975" rel="external">17319975</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1851399" rel="external">1851399</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fbcr1654" rel="external">10.1186/bcr1654</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tissue-specific histone modification and transcription factor binding in alpha globin gene expression.</h4>
<p class="authors"><span class="author">De Gobbi M</span>, <span class="author">Anguita E</span>, <span class="author">Hughes J</span>, <span class="author">Sloane-Stanley JA</span>, <span class="author">Sharpe JA</span>, <span class="author">Koch CM</span>, <span class="author">Dunham I</span>, <span class="author">Gibbons RJ</span>, <span class="author">Wood WG</span> and <span class="author">Higgs DR</span></p>
<p class="affiliation">Medical Research Council, Molecular Haematology Unit, Weatherall Institute of Molecular Medicine, Oxford University, Oxford, UK.</p><p class="abstract">To address the mechanism by which the human globin genes are activated during erythropoiesis, we have used a tiled microarray to analyze the pattern of transcription factor binding and associated histone modifications across the telomeric region of human chromosome 16 in primary erythroid and nonerythroid cells. This 220-kb region includes the alpha globin genes and 9 widely expressed genes flanking the alpha globin locus. This un-biased, comprehensive analysis of transcription factor binding and histone modifications (acetylation and methylation) described here not only identified all known cis-acting regulatory elements in the human alpha globin cluster but also demonstrated that there are no additional erythroid-specific regulatory elements in the 220-kb region tested. In addition, the pattern of histone modification distinguished promoter elements from potential enhancer elements across this region. Finally, comparison of the human and mouse orthologous regions in a unique mouse model, with both regions coexpressed in the same animal, showed significant differences that may explain how these 2 clusters are regulated differently in vivo.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U137961145</span>, <span class="grant">MC_U137961147</span>; NHGRI NIH HHS: <span class="grant">U01 HG003168</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2007</span>;<span class="volume">110</span>;<span class="number">13</span>;<span class="pagerange">4503-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17715390">17715390</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2007-06-097964">10.1182/blood-2007-06-097964</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NMDA receptor activation dephosphorylates AMPA receptor glutamate receptor 1 subunits at threonine 840.</h4>
<p class="authors"><span class="author">Delgado JY</span>, <span class="author">Coba M</span>, <span class="author">Anderson CN</span>, <span class="author">Thompson KR</span>, <span class="author">Gray EE</span>, <span class="author">Heusner CL</span>, <span class="author">Martin KC</span>, <span class="author">Grant SG</span> and <span class="author">O&#39;Dell TJ</span></p>
<p class="affiliation">Interdepartmental PhD Program for Neuroscience, University of California, Los Angeles, 90095, USA.</p><p class="abstract">Phosphorylation-dependent changes in AMPA receptor function have a crucial role in activity-dependent forms of synaptic plasticity such as long-term potentiation (LTP) and long-term depression (LTD). Although three previously identified phosphorylation sites in AMPA receptor glutamate receptor 1 (GluR1) subunits (S818, S831, and S845) appear to have important roles in LTP and LTD, little is known about the role of other putative phosphorylation sites in GluR1. Here, we describe the characterization of a recently identified phosphorylation site in GluR1 at threonine 840. The results of in vivo and in vitro phosphorylation assays suggest that T840 is not a substrate for protein kinases known to phosphorylate GluR1 at previously identified phosphorylation sites, such as protein kinase A, protein kinase C, and calcium/calmodulin-dependent kinase II. Instead, in vitro phosphorylation assays suggest that T840 is a substrate for p70S6 kinase. Although LTP-inducing patterns of synaptic stimulation had no effect on GluR1 phosphorylation at T840 in the hippocampal CA1 region, bath application of NMDA induced a strong, protein phosphatase 1- and/or 2A-mediated decrease in T840 phosphorylation. Moreover, GluR1 phosphorylation at T840 was transiently decreased by a chemical LTD induction protocol that induced a short-term depression of synaptic strength and persistently decreased by a chemical LTD induction protocol that induced a lasting depression of synaptic transmission. Together, our results show that GluR1 phosphorylation at T840 is regulated by NMDA receptor activation and suggest that decreases in GluR1 phosphorylation at T840 may have a role in LTD.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">R01 MH060919-06A1</span>; Wellcome Trust: <span class="grant">077155</span></p>
<p class="publication"><span class="title">The Journal of neuroscience : the official journal of the Society for Neuroscience</span>&nbsp;<span class="year">2007</span>;<span class="volume">27</span>;<span class="number">48</span>;<span class="pagerange">13210-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18045915" rel="external">18045915</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2851143" rel="external">2851143</a>; DOI: <a href="http://dx.doi.org/10.1523%2FJNEUROSCI.3056-07.2007" rel="external">10.1523/JNEUROSCI.3056-07.2007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prominent use of distal 5&#39; transcription start sites and discovery of a large number of additional exons in ENCODE regions.</h4>
<p class="authors"><span class="author">Denoeud F</span>, <span class="author">Kapranov P</span>, <span class="author">Ucla C</span>, <span class="author">Frankish A</span>, <span class="author">Castelo R</span>, <span class="author">Drenkow J</span>, <span class="author">Lagarde J</span>, <span class="author">Alioto T</span>, <span class="author">Manzano C</span>, <span class="author">Chrast J</span>, <span class="author">Dike S</span>, <span class="author">Wyss C</span>, <span class="author">Henrichsen CN</span>, <span class="author">Holroyd N</span>, <span class="author">Dickson MC</span>, <span class="author">Taylor R</span>, <span class="author">Hance Z</span>, <span class="author">Foissac S</span>, <span class="author">Myers RM</span>, <span class="author">Rogers J</span>, <span class="author">Hubbard T</span>, <span class="author">Harrow J</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Gingeras TR</span>, <span class="author">Antonarakis SE</span> and <span class="author">Reymond A</span></p>
<p class="affiliation">Grup de Recerca en Inform&aacute;tica Biom&egrave;dica, Institut Municipal d&#39;Investigaci&oacute; M&egrave;dica/Universitat Pompeu Fabra, 08003 Barcelona, Catalonia, Spain.</p><p class="abstract">This report presents systematic empirical annotation of transcript products from 399 annotated protein-coding loci across the 1% of the human genome targeted by the Encyclopedia of DNA elements (ENCODE) pilot project using a combination of 5&#39; rapid amplification of cDNA ends (RACE) and high-density resolution tiling arrays. We identified previously unannotated and often tissue- or cell-line-specific transcribed fragments (RACEfrags), both 5&#39; distal to the annotated 5&#39; terminus and internal to the annotated gene bounds for the vast majority (81.5%) of the tested genes. Half of the distal RACEfrags span large segments of genomic sequences away from the main portion of the coding transcript and often overlap with the upstream-annotated gene(s). Notably, at least 20% of the resultant novel transcripts have changes in their open reading frames (ORFs), most of them fusing ORFs of adjacent transcripts. A significant fraction of distal RACEfrags show expression levels comparable to those of known exons of the same locus, suggesting that they are not part of very minority splice forms. These results have significant implications concerning (1) our current understanding of the architecture of protein-coding genes; (2) our views on locations of regulatory regions in the genome; and (3) the interpretation of sequence polymorphisms mapping to regions hitherto considered to be &quot;noncoding,&quot; ultimately relating to the identification of disease-related sequence alterations.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01HG03147</span>, <span class="grant">U01HG03150</span>; PHS HHS: <span class="grant">N01C012400</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">746-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17567994">17567994</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1891335">1891335</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5660607">10.1101/gr.5660607</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">AutoCSA, an algorithm for high throughput DNA sequence variant detection in cancer genomes.</h4>
<p class="authors"><span class="author">Dicks E</span>, <span class="author">Teague JW</span>, <span class="author">Stephens P</span>, <span class="author">Raine K</span>, <span class="author">Yates A</span>, <span class="author">Mattocks C</span>, <span class="author">Tarpey P</span>, <span class="author">Butler A</span>, <span class="author">Menzies A</span>, <span class="author">Richardson D</span>, <span class="author">Jenkinson A</span>, <span class="author">Davies H</span>, <span class="author">Edkins S</span>, <span class="author">Forbes S</span>, <span class="author">Gray K</span>, <span class="author">Greenman C</span>, <span class="author">Shepherd R</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span> and <span class="author">Wooster R</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">The undertaking of large-scale DNA sequencing screens for somatic variants in human cancers requires accurate and rapid processing of traces for variants. Due to their often aneuploid nature and admixed normal tissue, heterozygous variants found in primary cancers are often subtle and difficult to detect. To address these issues, we have developed a mutation detection algorithm, AutoCSA, specifically optimized for the high throughput screening of cancer samples. Availability: http://www.sanger.ac.uk/genetics/CGP/Software/AutoCSA.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">13</span>;<span class="pagerange">1689-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17485433" rel="external">17485433</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm152" rel="external">10.1093/bioinformatics/btm152</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A bimodal pattern of relatedness between the Salmonella Paratyphi A and Typhi genomes: convergence or divergence by homologous recombination?</h4>
<p class="authors"><span class="author">Didelot X</span>, <span class="author">Achtman M</span>, <span class="author">Parkhill J</span>, <span class="author">Thomson NR</span> and <span class="author">Falush D</span></p>
<p class="affiliation">Department of Statistics, University of Oxford, Oxford OX1 3SY, United Kingdom.</p><p class="abstract">All Salmonella can cause disease but severe systemic infections are primarily caused by a few lineages. Paratyphi A and Typhi are the deadliest human restricted serovars, responsible for approximately 600,000 deaths per annum. We developed a Bayesian changepoint model that uses variation in the degree of nucleotide divergence along two genomes to detect homologous recombination between these strains, and with other lineages of Salmonella enterica. Paratyphi A and Typhi showed an atypical and surprising pattern. For three quarters of their genomes, they appear to be distantly related members of the species S. enterica, both in their gene content and nucleotide divergence. However, the remaining quarter is much more similar in both aspects, with average nucleotide divergence of 0.18% instead of 1.2%. We describe two different scenarios that could have led to this pattern, convergence and divergence, and conclude that the former is more likely based on a variety of criteria. The convergence scenario implies that, although Paratyphi A and Typhi were not especially close relatives within S. enterica, they have gone through a burst of recombination involving more than 100 recombination events. Several of the recombination events transferred novel genes in addition to homologous sequences, resulting in similar gene content in the two lineages. We propose that recombination between Typhi and Paratyphi A has allowed the exchange of gene variants that are important for their adaptation to their common ecological niche, the human host.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">1</span>;<span class="pagerange">61-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17090663" rel="external">17090663</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1716267" rel="external">1716267</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5512906" rel="external">10.1101/gr.5512906</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale discovery of promoter motifs in Drosophila melanogaster.</h4>
<p class="authors"><span class="author">Down TA</span>, <span class="author">Bergman CM</span>, <span class="author">Su J</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%74%64%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#100;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A key step in understanding gene regulation is to identify the repertoire of transcription factor binding motifs (TFBMs) that form the building blocks of promoters and other regulatory elements. Identifying these experimentally is very laborious, and the number of TFBMs discovered remains relatively small, especially when compared with the hundreds of transcription factor genes predicted in metazoan genomes. We have used a recently developed statistical motif discovery approach, NestedMICA, to detect candidate TFBMs from a large set of Drosophila melanogaster promoter regions. Of the 120 motifs inferred in our initial analysis, 25 were statistically significant matches to previously reported motifs, while 87 appeared to be novel. Analysis of sequence conservation and motif positioning suggested that the great majority of these discovered motifs are predictive of functional elements in the genome. Many motifs showed associations with specific patterns of gene expression in the D. melanogaster embryo, and we were able to obtain confident annotation of expression patterns for 25 of our motifs, including eight of the novel motifs. The motifs are available through Tiffin, a new database of DNA sequence motifs. We have discovered many new motifs that are overrepresented in D. melanogaster promoter regions, and offer several independent lines of evidence that these are novel TFBMs. Our motif dictionary provides a solid foundation for further investigation of regulatory elements in Drosophila, and demonstrates techniques that should be applicable in other species. We suggest that further improvements in computational motif discovery should narrow the gap between the set of known motifs and the total number of transcription factors in metazoan genomes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">e7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17238282">17238282</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1779301">1779301</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.0030007">10.1371/journal.pcbi.0030007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An H-NS-like stealth protein aids horizontal DNA transmission in bacteria.</h4>
<p class="authors"><span class="author">Doyle M</span>, <span class="author">Fookes M</span>, <span class="author">Ivens A</span>, <span class="author">Mangan MW</span>, <span class="author">Wain J</span> and <span class="author">Dorman CJ</span></p>
<p class="affiliation">Department of Microbiology, Moyne Institute of Preventive Medicine, Trinity College Dublin, Dublin 2, Ireland. <a href="mailto:%63%6A%64%6F%72%6D%61%6E%40%74%63%64.%69%65">&#99;&#106;&#100;&#111;&#114;&#109;&#97;&#110;&#64;&#116;&#99;&#100;&#46;&#105;&#101;</a></p><p class="abstract">The Sfh protein is encoded by self-transmissible plasmids involved in human typhoid and is closely related to the global regulator H-NS. We have found that Sfh provides a stealth function that allows the plasmids to be transmitted to new bacterial hosts with minimal effects on their fitness. Introducing the plasmid without the sfh gene imposes a mild H-NS(-) phenotype and a severe loss of fitness due to titration of the cellular pool of H-NS by the A+T-rich plasmid. This stealth strategy seems to be used widely to aid horizontal DNA transmission and has important implications for bacterial evolution.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">315</span>;<span class="number">5809</span>;<span class="pagerange">251-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17218529">17218529</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1137550">10.1126/science.1137550</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolution of genes and genomes on the Drosophila phylogeny.</h4>
<p class="authors"><span class="author">Drosophila 12 Genomes Consortium</span>, <span class="author">Clark AG</span>, <span class="author">Eisen MB</span>, <span class="author">Smith DR</span>, <span class="author">Bergman CM</span>, <span class="author">Oliver B</span>, <span class="author">Markow TA</span>, <span class="author">Kaufman TC</span>, <span class="author">Kellis M</span>, <span class="author">Gelbart W</span>, <span class="author">Iyer VN</span>, <span class="author">Pollard DA</span>, <span class="author">Sackton TB</span>, <span class="author">Larracuente AM</span>, <span class="author">Singh ND</span>, <span class="author">Abad JP</span>, <span class="author">Abt DN</span>, <span class="author">Adryan B</span>, <span class="author">Aguade M</span>, <span class="author">Akashi H</span>, <span class="author">Anderson WW</span>, <span class="author">Aquadro CF</span>, <span class="author">Ardell DH</span>, <span class="author">Arguello R</span>, <span class="author">Artieri CG</span>, <span class="author">Barbash DA</span>, <span class="author">Barker D</span>, <span class="author">Barsanti P</span>, <span class="author">Batterham P</span>, <span class="author">Batzoglou S</span>, <span class="author">Begun D</span>, <span class="author">Bhutkar A</span>, <span class="author">Blanco E</span>, <span class="author">Bosak SA</span>, <span class="author">Bradley RK</span>, <span class="author">Brand AD</span>, <span class="author">Brent MR</span>, <span class="author">Brooks AN</span>, <span class="author">Brown RH</span>, <span class="author">Butlin RK</span>, <span class="author">Caggese C</span>, <span class="author">Calvi BR</span>, <span class="author">Bernardo de Carvalho A</span>, <span class="author">Caspi A</span>, <span class="author">Castrezana S</span>, <span class="author">Celniker SE</span>, <span class="author">Chang JL</span>, <span class="author">Chapple C</span>, <span class="author">Chatterji S</span>, <span class="author">Chinwalla A</span>, <span class="author">Civetta A</span>, <span class="author">Clifton SW</span>, <span class="author">Comeron JM</span>, <span class="author">Costello JC</span>, <span class="author">Coyne JA</span>, <span class="author">Daub J</span>, <span class="author">David RG</span>, <span class="author">Delcher AL</span>, <span class="author">Delehaunty K</span>, <span class="author">Do CB</span>, <span class="author">Ebling H</span>, <span class="author">Edwards K</span>, <span class="author">Eickbush T</span>, <span class="author">Evans JD</span>, <span class="author">Filipski A</span>, <span class="author">Findeiss S</span>, <span class="author">Freyhult E</span>, <span class="author">Fulton L</span>, <span class="author">Fulton R</span>, <span class="author">Garcia AC</span>, <span class="author">Gardiner A</span>, <span class="author">Garfield DA</span>, <span class="author">Garvin BE</span>, <span class="author">Gibson G</span>, <span class="author">Gilbert D</span>, <span class="author">Gnerre S</span>, <span class="author">Godfrey J</span>, <span class="author">Good R</span>, <span class="author">Gotea V</span>, <span class="author">Gravely B</span>, <span class="author">Greenberg AJ</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Gross S</span>, <span class="author">Guigo R</span>, <span class="author">Gustafson EA</span>, <span class="author">Haerty W</span>, <span class="author">Hahn MW</span>, <span class="author">Halligan DL</span>, <span class="author">Halpern AL</span>, <span class="author">Halter GM</span>, <span class="author">Han MV</span>, <span class="author">Heger A</span>, <span class="author">Hillier L</span>, <span class="author">Hinrichs AS</span>, <span class="author">Holmes I</span>, <span class="author">Hoskins RA</span>, <span class="author">Hubisz MJ</span>, <span class="author">Hultmark D</span>, <span class="author">Huntley MA</span>, <span class="author">Jaffe DB</span>, <span class="author">Jagadeeshan S</span>, <span class="author">Jeck WR</span>, <span class="author">Johnson J</span>, <span class="author">Jones CD</span>, <span class="author">Jordan WC</span>, <span class="author">Karpen GH</span>, <span class="author">Kataoka E</span>, <span class="author">Keightley PD</span>, <span class="author">Kheradpour P</span>, <span class="author">Kirkness EF</span>, <span class="author">Koerich LB</span>, <span class="author">Kristiansen K</span>, <span class="author">Kudrna D</span>, <span class="author">Kulathinal RJ</span>, <span class="author">Kumar S</span>, <span class="author">Kwok R</span>, <span class="author">Lander E</span>, <span class="author">Langley CH</span>, <span class="author">Lapoint R</span>, <span class="author">Lazzaro BP</span>, <span class="author">Lee SJ</span>, <span class="author">Levesque L</span>, <span class="author">Li R</span>, <span class="author">Lin CF</span>, <span class="author">Lin MF</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Llopart A</span>, <span class="author">Long M</span>, <span class="author">Low L</span>, <span class="author">Lozovsky E</span>, <span class="author">Lu J</span>, <span class="author">Luo M</span>, <span class="author">Machado CA</span>, <span class="author">Makalowski W</span>, <span class="author">Marzo M</span>, <span class="author">Matsuda M</span>, <span class="author">Matzkin L</span>, <span class="author">McAllister B</span>, <span class="author">McBride CS</span>, <span class="author">McKernan B</span>, <span class="author">McKernan K</span>, <span class="author">Mendez-Lago M</span>, <span class="author">Minx P</span>, <span class="author">Mollenhauer MU</span>, <span class="author">Montooth K</span>, <span class="author">Mount SM</span>, <span class="author">Mu X</span>, <span class="author">Myers E</span>, <span class="author">Negre B</span>, <span class="author">Newfeld S</span>, <span class="author">Nielsen R</span>, <span class="author">Noor MA</span>, <span class="author">O&#39;Grady P</span>, <span class="author">Pachter L</span>, <span class="author">Papaceit M</span>, <span class="author">Parisi MJ</span>, <span class="author">Parisi M</span>, <span class="author">Parts L</span>, <span class="author">Pedersen JS</span>, <span class="author">Pesole G</span>, <span class="author">Phillippy AM</span>, <span class="author">Ponting CP</span>, <span class="author">Pop M</span>, <span class="author">Porcelli D</span>, <span class="author">Powell JR</span>, <span class="author">Prohaska S</span>, <span class="author">Pruitt K</span>, <span class="author">Puig M</span>, <span class="author">Quesneville H</span>, <span class="author">Ram KR</span>, <span class="author">Rand D</span>, <span class="author">Rasmussen MD</span>, <span class="author">Reed LK</span>, <span class="author">Reenan R</span>, <span class="author">Reily A</span>, <span class="author">Remington KA</span>, <span class="author">Rieger TT</span>, <span class="author">Ritchie MG</span>, <span class="author">Robin C</span>, <span class="author">Rogers YH</span>, <span class="author">Rohde C</span>, <span class="author">Rozas J</span>, <span class="author">Rubenfield MJ</span>, <span class="author">Ruiz A</span>, <span class="author">Russo S</span>, <span class="author">Salzberg SL</span>, <span class="author">Sanchez-Gracia A</span>, <span class="author">Saranga DJ</span>, <span class="author">Sato H</span>, <span class="author">Schaeffer SW</span>, <span class="author">Schatz MC</span>, <span class="author">Schlenke T</span>, <span class="author">Schwartz R</span>, <span class="author">Segarra C</span>, <span class="author">Singh RS</span>, <span class="author">Sirot L</span>, <span class="author">Sirota M</span>, <span class="author">Sisneros NB</span>, <span class="author">Smith CD</span>, <span class="author">Smith TF</span>, <span class="author">Spieth J</span>, <span class="author">Stage DE</span>, <span class="author">Stark A</span>, <span class="author">Stephan W</span>, <span class="author">Strausberg RL</span>, <span class="author">Strempel S</span>, <span class="author">Sturgill D</span>, <span class="author">Sutton G</span>, <span class="author">Sutton GG</span>, <span class="author">Tao W</span>, <span class="author">Teichmann S</span>, <span class="author">Tobari YN</span>, <span class="author">Tomimura Y</span>, <span class="author">Tsolas JM</span>, <span class="author">Valente VL</span>, <span class="author">Venter E</span>, <span class="author">Venter JC</span>, <span class="author">Vicario S</span>, <span class="author">Vieira FG</span>, <span class="author">Vilella AJ</span>, <span class="author">Villasante A</span>, <span class="author">Walenz B</span>, <span class="author">Wang J</span>, <span class="author">Wasserman M</span>, <span class="author">Watts T</span>, <span class="author">Wilson D</span>, <span class="author">Wilson RK</span>, <span class="author">Wing RA</span>, <span class="author">Wolfner MF</span>, <span class="author">Wong A</span>, <span class="author">Wong GK</span>, <span class="author">Wu CI</span>, <span class="author">Wu G</span>, <span class="author">Yamamoto D</span>, <span class="author">Yang HP</span>, <span class="author">Yang SP</span>, <span class="author">Yorke JA</span>, <span class="author">Yoshida K</span>, <span class="author">Zdobnov E</span>, <span class="author">Zhang P</span>, <span class="author">Zhang Y</span>, <span class="author">Zimin AV</span>, <span class="author">Baldwin J</span>, <span class="author">Abdouelleil A</span>, <span class="author">Abdulkadir J</span>, <span class="author">Abebe A</span>, <span class="author">Abera B</span>, <span class="author">Abreu J</span>, <span class="author">Acer SC</span>, <span class="author">Aftuck L</span>, <span class="author">Alexander A</span>, <span class="author">An P</span>, <span class="author">Anderson E</span>, <span class="author">Anderson S</span>, <span class="author">Arachi H</span>, <span class="author">Azer M</span>, <span class="author">Bachantsang P</span>, <span class="author">Barry A</span>, <span class="author">Bayul T</span>, <span class="author">Berlin A</span>, <span class="author">Bessette D</span>, <span class="author">Bloom T</span>, <span class="author">Blye J</span>, <span class="author">Boguslavskiy L</span>, <span class="author">Bonnet C</span>, <span class="author">Boukhgalter B</span>, <span class="author">Bourzgui I</span>, <span class="author">Brown A</span>, <span class="author">Cahill P</span>, <span class="author">Channer S</span>, <span class="author">Cheshatsang Y</span>, <span class="author">Chuda L</span>, <span class="author">Citroen M</span>, <span class="author">Collymore A</span>, <span class="author">Cooke P</span>, <span class="author">Costello M</span>, <span class="author">D&#39;Aco K</span>, <span class="author">Daza R</span>, <span class="author">De Haan G</span>, <span class="author">DeGray S</span>, <span class="author">DeMaso C</span>, <span class="author">Dhargay N</span>, <span class="author">Dooley K</span>, <span class="author">Dooley E</span>, <span class="author">Doricent M</span>, <span class="author">Dorje P</span>, <span class="author">Dorjee K</span>, <span class="author">Dupes A</span>, <span class="author">Elong R</span>, <span class="author">Falk J</span>, <span class="author">Farina A</span>, <span class="author">Faro S</span>, <span class="author">Ferguson D</span>, <span class="author">Fisher S</span>, <span class="author">Foley CD</span>, <span class="author">Franke A</span>, <span class="author">Friedrich D</span>, <span class="author">Gadbois L</span>, <span class="author">Gearin G</span>, <span class="author">Gearin CR</span>, <span class="author">Giannoukos G</span>, <span class="author">Goode T</span>, <span class="author">Graham J</span>, <span class="author">Grandbois E</span>, <span class="author">Grewal S</span>, <span class="author">Gyaltsen K</span>, <span class="author">Hafez N</span>, <span class="author">Hagos B</span>, <span class="author">Hall J</span>, <span class="author">Henson C</span>, <span class="author">Hollinger A</span>, <span class="author">Honan T</span>, <span class="author">Huard MD</span>, <span class="author">Hughes L</span>, <span class="author">Hurhula B</span>, <span class="author">Husby ME</span>, <span class="author">Kamat A</span>, <span class="author">Kanga B</span>, <span class="author">Kashin S</span>, <span class="author">Khazanovich D</span>, <span class="author">Kisner P</span>, <span class="author">Lance K</span>, <span class="author">Lara M</span>, <span class="author">Lee W</span>, <span class="author">Lennon N</span>, <span class="author">Letendre F</span>, <span class="author">LeVine R</span>, <span class="author">Lipovsky A</span>, <span class="author">Liu X</span>, <span class="author">Liu J</span>, <span class="author">Liu S</span>, <span class="author">Lokyitsang T</span>, <span class="author">Lokyitsang Y</span>, <span class="author">Lubonja R</span>, <span class="author">Lui A</span>, <span class="author">MacDonald P</span>, <span class="author">Magnisalis V</span>, <span class="author">Maru K</span>, <span class="author">Matthews C</span>, <span class="author">McCusker W</span>, <span class="author">McDonough S</span>, <span class="author">Mehta T</span>, <span class="author">Meldrim J</span>, <span class="author">Meneus L</span>, <span class="author">Mihai O</span>, <span class="author">Mihalev A</span>, <span class="author">Mihova T</span>, <span class="author">Mittelman R</span>, <span class="author">Mlenga V</span>, <span class="author">Montmayeur A</span>, <span class="author">Mulrain L</span>, <span class="author">Navidi A</span>, <span class="author">Naylor J</span>, <span class="author">Negash T</span>, <span class="author">Nguyen T</span>, <span class="author">Nguyen N</span>, <span class="author">Nicol R</span>, <span class="author">Norbu C</span>, <span class="author">Norbu N</span>, <span class="author">Novod N</span>, <span class="author">O&#39;Neill B</span>, <span class="author">Osman S</span>, <span class="author">Markiewicz E</span>, <span class="author">Oyono OL</span>, <span class="author">Patti C</span>, <span class="author">Phunkhang P</span>, <span class="author">Pierre F</span>, <span class="author">Priest M</span>, <span class="author">Raghuraman S</span>, <span class="author">Rege F</span>, <span class="author">Reyes R</span>, <span class="author">Rise C</span>, <span class="author">Rogov P</span>, <span class="author">Ross K</span>, <span class="author">Ryan E</span>, <span class="author">Settipalli S</span>, <span class="author">Shea T</span>, <span class="author">Sherpa N</span>, <span class="author">Shi L</span>, <span class="author">Shih D</span>, <span class="author">Sparrow T</span>, <span class="author">Spaulding J</span>, <span class="author">Stalker J</span>, <span class="author">Stange-Thomann N</span>, <span class="author">Stavropoulos S</span>, <span class="author">Stone C</span>, <span class="author">Strader C</span>, <span class="author">Tesfaye S</span>, <span class="author">Thomson T</span>, <span class="author">Thoulutsang Y</span>, <span class="author">Thoulutsang D</span>, <span class="author">Topham K</span>, <span class="author">Topping I</span>, <span class="author">Tsamla T</span>, <span class="author">Vassiliev H</span>, <span class="author">Vo A</span>, <span class="author">Wangchuk T</span>, <span class="author">Wangdi T</span>, <span class="author">Weiand M</span>, <span class="author">Wilkinson J</span>, <span class="author">Wilson A</span>, <span class="author">Yadav S</span>, <span class="author">Young G</span>, <span class="author">Yu Q</span>, <span class="author">Zembek L</span>, <span class="author">Zhong D</span>, <span class="author">Zimmer A</span>, <span class="author">Zwirko Z</span>, <span class="author">Jaffe DB</span>, <span class="author">Alvarez P</span>, <span class="author">Brockman W</span>, <span class="author">Butler J</span>, <span class="author">Chin C</span>, <span class="author">Gnerre S</span>, <span class="author">Grabherr M</span>, <span class="author">Kleber M</span>, <span class="author">Mauceli E</span> and <span class="author">MacCallum I</span></p>
<p class="affiliation">Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853, USA. <a href="mailto:%61%63%33%34%37%40%63%6F%72%6E%65%6C%6C.%65%64%75">&#97;&#99;&#51;&#52;&#55;&#64;&#99;&#111;&#114;&#110;&#101;&#108;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Comparative analysis of multiple genomes in a phylogenetic framework dramatically improves the precision and sensitivity of evolutionary inference, producing more robust results than single-genome analyses can provide. The genomes of 12 Drosophila species, ten of which are presented here for the first time (sechellia, simulans, yakuba, erecta, ananassae, persimilis, willistoni, mojavensis, virilis and grimshawi), illustrate how rates and patterns of sequence divergence across taxa can illuminate evolutionary processes on a genomic scale. These genome sequences augment the formidable genetic tools that have made Drosophila melanogaster a pre-eminent model for animal genetics, and will further catalyse fundamental research on mechanisms of development, cell biology, genetics, disease, neurobiology, behaviour, physiology and evolution. Despite remarkable similarities among these Drosophila species, we identified many putatively non-neutral changes in protein-coding genes, non-coding RNA genes, and cis-regulatory regions. These may prove to underlie differences in the ecology and behaviour of these diverse species.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105161047</span>, <span class="grant">MC_U137761446</span>; NHGRI NIH HHS: <span class="grant">R01 HG000747</span>, <span class="grant">R01 HG000747-16</span>, <span class="grant">R01 HG002779-05</span>, <span class="grant">R01 HG002779-06</span>, <span class="grant">R01 HG004037</span>; NIDDK NIH HHS: <span class="grant">Z01 DK015600-12</span>; NIGMS NIH HHS: <span class="grant">F32 GM067504</span>, <span class="grant">R01 GM074813-04</span>; NLM NIH HHS: <span class="grant">R01 LM006845-08</span>, <span class="grant">R01 LM006845-09</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">450</span>;<span class="number">7167</span>;<span class="pagerange">203-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17994087">17994087</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06341">10.1038/nature06341</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A TNF region haplotype offers protection from typhoid fever in Vietnamese patients.</h4>
<p class="authors"><span class="author">Dunstan SJ</span>, <span class="author">Nguyen TH</span>, <span class="author">Rockett K</span>, <span class="author">Forton J</span>, <span class="author">Morris AP</span>, <span class="author">Diakite M</span>, <span class="author">Mai NL</span>, <span class="author">Le TP</span>, <span class="author">House D</span>, <span class="author">Parry CM</span>, <span class="author">Ha V</span>, <span class="author">Nguyen TH</span>, <span class="author">Dougan G</span>, <span class="author">Tran TH</span>, <span class="author">Kwiatowski D</span> and <span class="author">Farrar JJ</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Hospital for Tropical Diseases, 190 Ben Ham Tu, Quan 5, District 5, Ho Chi Minh City, Vietnam. <a href="mailto:%73%64%75%6E%73%74%61%6E%40%6F%75%63%72%75.%6F%72%67">&#115;&#100;&#117;&#110;&#115;&#116;&#97;&#110;&#64;&#111;&#117;&#99;&#114;&#117;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The genomic region surrounding the TNF locus on human chromosome 6 has previously been associated with typhoid fever in Vietnam (Dunstan et al. in J Infect Dis 183:261-268, 2001). We used a haplotypic approach to understand this association further. Eighty single nucleotide polymorphisms (SNPs) spanning a 150 kb region were genotyped in 95 Vietnamese individuals (typhoid case/mother/father trios). A subset of data from 33 SNPs with a minor allele frequency of &gt;4.3% was used to construct haplotypes. Fifteen SNPs, which tagged the 42 constructed haplotypes were selected. The haplotype tagging SNPs (T1-T15) were genotyped in 380 confirmed typhoid cases and 380 Vietnamese ethnically matched controls. Allelic frequencies of seven SNPs (T1, T2, T3, T5, T6, T7, T8) were significantly different between typhoid cases and controls. Logistic regression results support the hypothesis that there is just one signal associated with disease at this locus. Haplotype-based analysis of the tag SNPs provided positive evidence of association with typhoid (posterior probability 0.821). The analysis highlighted a low-risk cluster of haplotypes that each carry the minor allele of T1 or T7, but not both, and otherwise carry the combination of alleles *12122*1111 at T1-T11, further supporting the one associated signal hypothesis. Finally, individuals that carry the typhoid fever protective haplotype *12122*1111 also produce a relatively low TNF-alpha response to LPS.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span>, <span class="grant">081682</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">122</span>;<span class="number">1</span>;<span class="pagerange">51-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17503085" rel="external">17503085</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656995" rel="external">2656995</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-007-0372-9" rel="external">10.1007/s00439-007-0372-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study identifies novel breast cancer susceptibility loci.</h4>
<p class="authors"><span class="author">Easton DF</span>, <span class="author">Pooley KA</span>, <span class="author">Dunning AM</span>, <span class="author">Pharoah PD</span>, <span class="author">Thompson D</span>, <span class="author">Ballinger DG</span>, <span class="author">Struewing JP</span>, <span class="author">Morrison J</span>, <span class="author">Field H</span>, <span class="author">Luben R</span>, <span class="author">Wareham N</span>, <span class="author">Ahmed S</span>, <span class="author">Healey CS</span>, <span class="author">Bowman R</span>, <span class="author">SEARCH collaborators</span>, <span class="author">Meyer KB</span>, <span class="author">Haiman CA</span>, <span class="author">Kolonel LK</span>, <span class="author">Henderson BE</span>, <span class="author">Le Marchand L</span>, <span class="author">Brennan P</span>, <span class="author">Sangrajrang S</span>, <span class="author">Gaborieau V</span>, <span class="author">Odefrey F</span>, <span class="author">Shen CY</span>, <span class="author">Wu PE</span>, <span class="author">Wang HC</span>, <span class="author">Eccles D</span>, <span class="author">Evans DG</span>, <span class="author">Peto J</span>, <span class="author">Fletcher O</span>, <span class="author">Johnson N</span>, <span class="author">Seal S</span>, <span class="author">Stratton MR</span>, <span class="author">Rahman N</span>, <span class="author">Chenevix-Trench G</span>, <span class="author">Bojesen SE</span>, <span class="author">Nordestgaard BG</span>, <span class="author">Axelsson CK</span>, <span class="author">Garcia-Closas M</span>, <span class="author">Brinton L</span>, <span class="author">Chanock S</span>, <span class="author">Lissowska J</span>, <span class="author">Peplonska B</span>, <span class="author">Nevanlinna H</span>, <span class="author">Fagerholm R</span>, <span class="author">Eerola H</span>, <span class="author">Kang D</span>, <span class="author">Yoo KY</span>, <span class="author">Noh DY</span>, <span class="author">Ahn SH</span>, <span class="author">Hunter DJ</span>, <span class="author">Hankinson SE</span>, <span class="author">Cox DG</span>, <span class="author">Hall P</span>, <span class="author">Wedren S</span>, <span class="author">Liu J</span>, <span class="author">Low YL</span>, <span class="author">Bogdanova N</span>, <span class="author">Sch&uuml;rmann P</span>, <span class="author">D&ouml;rk T</span>, <span class="author">Tollenaar RA</span>, <span class="author">Jacobi CE</span>, <span class="author">Devilee P</span>, <span class="author">Klijn JG</span>, <span class="author">Sigurdson AJ</span>, <span class="author">Doody MM</span>, <span class="author">Alexander BH</span>, <span class="author">Zhang J</span>, <span class="author">Cox A</span>, <span class="author">Brock IW</span>, <span class="author">MacPherson G</span>, <span class="author">Reed MW</span>, <span class="author">Couch FJ</span>, <span class="author">Goode EL</span>, <span class="author">Olson JE</span>, <span class="author">Meijers-Heijboer H</span>, <span class="author">van den Ouweland A</span>, <span class="author">Uitterlinden A</span>, <span class="author">Rivadeneira F</span>, <span class="author">Milne RL</span>, <span class="author">Ribas G</span>, <span class="author">Gonzalez-Neira A</span>, <span class="author">Benitez J</span>, <span class="author">Hopper JL</span>, <span class="author">McCredie M</span>, <span class="author">Southey M</span>, <span class="author">Giles GG</span>, <span class="author">Schroen C</span>, <span class="author">Justenhoven C</span>, <span class="author">Brauch H</span>, <span class="author">Hamann U</span>, <span class="author">Ko YD</span>, <span class="author">Spurdle AB</span>, <span class="author">Beesley J</span>, <span class="author">Chen X</span>, <span class="author">kConFab</span>, <span class="author">AOCS Management Group</span>, <span class="author">Mannermaa A</span>, <span class="author">Kosma VM</span>, <span class="author">Kataja V</span>, <span class="author">Hartikainen J</span>, <span class="author">Day NE</span>, <span class="author">Cox DR</span> and <span class="author">Ponder BA</span></p>
<p class="affiliation">CR-UK Genetic Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge CB1 8RN, UK. <a href="mailto:%64.%65%61%73%74%6F%6E%40%73%72%6C.%63%61%6D.%61%63.%75%6B">&#100;&#46;&#101;&#97;&#115;&#116;&#111;&#110;&#64;&#115;&#114;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Breast cancer exhibits familial aggregation, consistent with variation in genetic susceptibility to the disease. Known susceptibility genes account for less than 25% of the familial risk of breast cancer, and the residual genetic variance is likely to be due to variants conferring more moderate risks. To identify further susceptibility alleles, we conducted a two-stage genome-wide association study in 4,398 breast cancer cases and 4,316 controls, followed by a third stage in which 30 single nucleotide polymorphisms (SNPs) were tested for confirmation in 21,860 cases and 22,578 controls from 22 studies. We used 227,876 SNPs that were estimated to correlate with 77% of known common SNPs in Europeans at r2 &gt; 0.5. SNPs in five novel independent loci exhibited strong and consistent evidence of association with breast cancer (P &lt; 10(-7)). Four of these contain plausible causative genes (FGFR2, TNRC9, MAP3K1 and LSP1). At the second stage, 1,792 SNPs were significant at the P &lt; 0.05 level compared with an estimated 1,343 that would be expected by chance, indicating that many additional common susceptibility alleles may be identifiable by this approach.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A3353</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">447</span>;<span class="number">7148</span>;<span class="pagerange">1087-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17529967">17529967</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2714974">2714974</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature05887">10.1038/nature05887</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The evolution of imprinting: chromosomal mapping of orthologues of mammalian imprinted domains in monotreme and marsupial mammals.</h4>
<p class="authors"><span class="author">Edwards CA</span>, <span class="author">Rens W</span>, <span class="author">Clarke O</span>, <span class="author">Mungall AJ</span>, <span class="author">Hore T</span>, <span class="author">Graves JA</span>, <span class="author">Dunham I</span>, <span class="author">Ferguson-Smith AC</span> and <span class="author">Ferguson-Smith MA</span></p>
<p class="affiliation">Department of Physiology, Development and Neuroscience, University of Cambridge, Downing Street, Cambridge CB2 3DY, UK. <a href="mailto:%63%61%65%32%38%40%63%61%6D.%61%63.%75%6B">&#99;&#97;&#101;&#50;&#56;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The evolution of genomic imprinting, the parental-origin specific expression of genes, is the subject of much debate. There are several theories to account for how the mechanism evolved including the hypothesis that it was driven by the evolution of X-inactivation, or that it arose from an ancestrally imprinted chromosome.</p><p class="abstract"><strong>Results</strong>: Here we demonstrate that mammalian orthologues of imprinted genes are dispersed amongst autosomes in both monotreme and marsupial karyotypes.</p><p class="abstract"><strong>Conclusion</strong>: These data, along with the similar distribution seen in birds, suggest that imprinted genes were not located on an ancestrally imprinted chromosome or associated with a sex chromosome. Our results suggest imprinting evolution was a stepwise, adaptive process, with each gene/cluster independently becoming imprinted as the need arose.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="pagerange">157</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17822525" rel="external">17822525</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2042987" rel="external">2042987</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-7-157" rel="external">10.1186/1471-2148-7-157</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Karyotypic differences in two sibling species of Scotophilus from South Africa (Vespertilionidae, Chiroptera, Mammalia).</h4>
<p class="authors"><span class="author">Eick GN</span>, <span class="author">Jacobs DS</span>, <span class="author">Yang F</span> and <span class="author">Volleth M</span></p>
<p class="affiliation">Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Stellenbosch (South Africa).</p><p class="abstract">Karyotype descriptions are given for Scotophilus dinganii (2n = 36, FNa = 50) and a recently discovered sister-species, Scotophilus sp. nov. (2n = 36, FNa = 52). These two sibling species occur sympatrically and are distinguished by body size, echolocation frequency and cytochrome b sequence. Cytogenetically, both species differ from other Scotophilus species in the subtelocentric morphology of chromosome 2 and a terminal heterochromatic segment on the X chromosome. Further, Scotophilus sp. nov. is characterized by a subtelocentric chromosome 4 not found in any other Scotophilus species. Comparing the Scotophilus karyotype with that of the vespertilionid genus Myotis, extensive conservation of whole chromosome arms has been found recently. However, out of 25 chromosomal arms six could not be identified in Scotophilus. Therefore, in the present study fluorescence in situ hybridization with whole chromosome painting probes from Myotis myotis was carried out on metaphase preparations from Scotophilus dinganii and Scotophilus sp. nov. These experiments revealed that three previously unidentified Scotophilus chromosomes (A, B, C) contain homologous sequences to Myotis chromosomes 18 plus 22, 19 plus 25, and 16/17, respectively.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">118</span>;<span class="number">1</span>;<span class="pagerange">72-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17901703" rel="external">17901703</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000106444" rel="external">10.1159/000106444</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Zebrafish genome project: bringing new biology to the vertebrate genome field.</h4>
<p class="authors"><span class="author">Ekker SC</span>, <span class="author">Stemple DL</span>, <span class="author">Clark M</span>, <span class="author">Chien CB</span>, <span class="author">Rasooly RS</span> and <span class="author">Javois LC</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology, Mayo Clinic Cancer Center, Rochester, Minnesota 55905, USA. <a href="mailto:%45%6B%6B%65%72.%53%74%65%70%68%65%6E%40%6D%61%79%6F.%65%64%75">&#69;&#107;&#107;&#101;&#114;&#46;&#83;&#116;&#101;&#112;&#104;&#101;&#110;&#64;&#109;&#97;&#121;&#111;&#46;&#101;&#100;&#117;</a></p>
<p class="publication"><span class="title">Zebrafish</span>&nbsp;<span class="year">2007</span>;<span class="volume">4</span>;<span class="number">4</span>;<span class="pagerange">239-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18284331" rel="external">18284331</a>; DOI: <a href="http://dx.doi.org/10.1089%2Fzeb.2007.9979" rel="external">10.1089/zeb.2007.9979</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project.</h4>
<p class="authors"><span class="author">ENCODE Project Consortium</span>, <span class="author">Birney E</span>, <span class="author">Stamatoyannopoulos JA</span>, <span class="author">Dutta A</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Gingeras TR</span>, <span class="author">Margulies EH</span>, <span class="author">Weng Z</span>, <span class="author">Snyder M</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Thurman RE</span>, <span class="author">Kuehn MS</span>, <span class="author">Taylor CM</span>, <span class="author">Neph S</span>, <span class="author">Koch CM</span>, <span class="author">Asthana S</span>, <span class="author">Malhotra A</span>, <span class="author">Adzhubei I</span>, <span class="author">Greenbaum JA</span>, <span class="author">Andrews RM</span>, <span class="author">Flicek P</span>, <span class="author">Boyle PJ</span>, <span class="author">Cao H</span>, <span class="author">Carter NP</span>, <span class="author">Clelland GK</span>, <span class="author">Davis S</span>, <span class="author">Day N</span>, <span class="author">Dhami P</span>, <span class="author">Dillon SC</span>, <span class="author">Dorschner MO</span>, <span class="author">Fiegler H</span>, <span class="author">Giresi PG</span>, <span class="author">Goldy J</span>, <span class="author">Hawrylycz M</span>, <span class="author">Haydock A</span>, <span class="author">Humbert R</span>, <span class="author">James KD</span>, <span class="author">Johnson BE</span>, <span class="author">Johnson EM</span>, <span class="author">Frum TT</span>, <span class="author">Rosenzweig ER</span>, <span class="author">Karnani N</span>, <span class="author">Lee K</span>, <span class="author">Lefebvre GC</span>, <span class="author">Navas PA</span>, <span class="author">Neri F</span>, <span class="author">Parker SC</span>, <span class="author">Sabo PJ</span>, <span class="author">Sandstrom R</span>, <span class="author">Shafer A</span>, <span class="author">Vetrie D</span>, <span class="author">Weaver M</span>, <span class="author">Wilcox S</span>, <span class="author">Yu M</span>, <span class="author">Collins FS</span>, <span class="author">Dekker J</span>, <span class="author">Lieb JD</span>, <span class="author">Tullius TD</span>, <span class="author">Crawford GE</span>, <span class="author">Sunyaev S</span>, <span class="author">Noble WS</span>, <span class="author">Dunham I</span>, <span class="author">Denoeud F</span>, <span class="author">Reymond A</span>, <span class="author">Kapranov P</span>, <span class="author">Rozowsky J</span>, <span class="author">Zheng D</span>, <span class="author">Castelo R</span>, <span class="author">Frankish A</span>, <span class="author">Harrow J</span>, <span class="author">Ghosh S</span>, <span class="author">Sandelin A</span>, <span class="author">Hofacker IL</span>, <span class="author">Baertsch R</span>, <span class="author">Keefe D</span>, <span class="author">Dike S</span>, <span class="author">Cheng J</span>, <span class="author">Hirsch HA</span>, <span class="author">Sekinger EA</span>, <span class="author">Lagarde J</span>, <span class="author">Abril JF</span>, <span class="author">Shahab A</span>, <span class="author">Flamm C</span>, <span class="author">Fried C</span>, <span class="author">Hackerm&uuml;ller J</span>, <span class="author">Hertel J</span>, <span class="author">Lindemeyer M</span>, <span class="author">Missal K</span>, <span class="author">Tanzer A</span>, <span class="author">Washietl S</span>, <span class="author">Korbel J</span>, <span class="author">Emanuelsson O</span>, <span class="author">Pedersen JS</span>, <span class="author">Holroyd N</span>, <span class="author">Taylor R</span>, <span class="author">Swarbreck D</span>, <span class="author">Matthews N</span>, <span class="author">Dickson MC</span>, <span class="author">Thomas DJ</span>, <span class="author">Weirauch MT</span>, <span class="author">Gilbert J</span>, <span class="author">Drenkow J</span>, <span class="author">Bell I</span>, <span class="author">Zhao X</span>, <span class="author">Srinivasan KG</span>, <span class="author">Sung WK</span>, <span class="author">Ooi HS</span>, <span class="author">Chiu KP</span>, <span class="author">Foissac S</span>, <span class="author">Alioto T</span>, <span class="author">Brent M</span>, <span class="author">Pachter L</span>, <span class="author">Tress ML</span>, <span class="author">Valencia A</span>, <span class="author">Choo SW</span>, <span class="author">Choo CY</span>, <span class="author">Ucla C</span>, <span class="author">Manzano C</span>, <span class="author">Wyss C</span>, <span class="author">Cheung E</span>, <span class="author">Clark TG</span>, <span class="author">Brown JB</span>, <span class="author">Ganesh M</span>, <span class="author">Patel S</span>, <span class="author">Tammana H</span>, <span class="author">Chrast J</span>, <span class="author">Henrichsen CN</span>, <span class="author">Kai C</span>, <span class="author">Kawai J</span>, <span class="author">Nagalakshmi U</span>, <span class="author">Wu J</span>, <span class="author">Lian Z</span>, <span class="author">Lian J</span>, <span class="author">Newburger P</span>, <span class="author">Zhang X</span>, <span class="author">Bickel P</span>, <span class="author">Mattick JS</span>, <span class="author">Carninci P</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Weissman S</span>, <span class="author">Hubbard T</span>, <span class="author">Myers RM</span>, <span class="author">Rogers J</span>, <span class="author">Stadler PF</span>, <span class="author">Lowe TM</span>, <span class="author">Wei CL</span>, <span class="author">Ruan Y</span>, <span class="author">Struhl K</span>, <span class="author">Gerstein M</span>, <span class="author">Antonarakis SE</span>, <span class="author">Fu Y</span>, <span class="author">Green ED</span>, <span class="author">Kara&ouml;z U</span>, <span class="author">Siepel A</span>, <span class="author">Taylor J</span>, <span class="author">Liefer LA</span>, <span class="author">Wetterstrand KA</span>, <span class="author">Good PJ</span>, <span class="author">Feingold EA</span>, <span class="author">Guyer MS</span>, <span class="author">Cooper GM</span>, <span class="author">Asimenos G</span>, <span class="author">Dewey CN</span>, <span class="author">Hou M</span>, <span class="author">Nikolaev S</span>, <span class="author">Montoya-Burgos JI</span>, <span class="author">L&ouml;ytynoja A</span>, <span class="author">Whelan S</span>, <span class="author">Pardi F</span>, <span class="author">Massingham T</span>, <span class="author">Huang H</span>, <span class="author">Zhang NR</span>, <span class="author">Holmes I</span>, <span class="author">Mullikin JC</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Paten B</span>, <span class="author">Seringhaus M</span>, <span class="author">Church D</span>, <span class="author">Rosenbloom K</span>, <span class="author">Kent WJ</span>, <span class="author">Stone EA</span>, <span class="author">NISC Comparative Sequencing Program</span>, <span class="author">Baylor College of Medicine Human Genome Sequencing Center</span>, <span class="author">Washington University Genome Sequencing Center</span>, <span class="author">Broad Institute</span>, <span class="author">Children&#39;s Hospital Oakland Research Institute</span>, <span class="author">Batzoglou S</span>, <span class="author">Goldman N</span>, <span class="author">Hardison RC</span>, <span class="author">Haussler D</span>, <span class="author">Miller W</span>, <span class="author">Sidow A</span>, <span class="author">Trinklein ND</span>, <span class="author">Zhang ZD</span>, <span class="author">Barrera L</span>, <span class="author">Stuart R</span>, <span class="author">King DC</span>, <span class="author">Ameur A</span>, <span class="author">Enroth S</span>, <span class="author">Bieda MC</span>, <span class="author">Kim J</span>, <span class="author">Bhinge AA</span>, <span class="author">Jiang N</span>, <span class="author">Liu J</span>, <span class="author">Yao F</span>, <span class="author">Vega VB</span>, <span class="author">Lee CW</span>, <span class="author">Ng P</span>, <span class="author">Shahab A</span>, <span class="author">Yang A</span>, <span class="author">Moqtaderi Z</span>, <span class="author">Zhu Z</span>, <span class="author">Xu X</span>, <span class="author">Squazzo S</span>, <span class="author">Oberley MJ</span>, <span class="author">Inman D</span>, <span class="author">Singer MA</span>, <span class="author">Richmond TA</span>, <span class="author">Munn KJ</span>, <span class="author">Rada-Iglesias A</span>, <span class="author">Wallerman O</span>, <span class="author">Komorowski J</span>, <span class="author">Fowler JC</span>, <span class="author">Couttet P</span>, <span class="author">Bruce AW</span>, <span class="author">Dovey OM</span>, <span class="author">Ellis PD</span>, <span class="author">Langford CF</span>, <span class="author">Nix DA</span>, <span class="author">Euskirchen G</span>, <span class="author">Hartman S</span>, <span class="author">Urban AE</span>, <span class="author">Kraus P</span>, <span class="author">Van Calcar S</span>, <span class="author">Heintzman N</span>, <span class="author">Kim TH</span>, <span class="author">Wang K</span>, <span class="author">Qu C</span>, <span class="author">Hon G</span>, <span class="author">Luna R</span>, <span class="author">Glass CK</span>, <span class="author">Rosenfeld MG</span>, <span class="author">Aldred SF</span>, <span class="author">Cooper SJ</span>, <span class="author">Halees A</span>, <span class="author">Lin JM</span>, <span class="author">Shulha HP</span>, <span class="author">Zhang X</span>, <span class="author">Xu M</span>, <span class="author">Haidar JN</span>, <span class="author">Yu Y</span>, <span class="author">Ruan Y</span>, <span class="author">Iyer VR</span>, <span class="author">Green RD</span>, <span class="author">Wadelius C</span>, <span class="author">Farnham PJ</span>, <span class="author">Ren B</span>, <span class="author">Harte RA</span>, <span class="author">Hinrichs AS</span>, <span class="author">Trumbower H</span>, <span class="author">Clawson H</span>, <span class="author">Hillman-Jackson J</span>, <span class="author">Zweig AS</span>, <span class="author">Smith K</span>, <span class="author">Thakkapallayil A</span>, <span class="author">Barber G</span>, <span class="author">Kuhn RM</span>, <span class="author">Karolchik D</span>, <span class="author">Armengol L</span>, <span class="author">Bird CP</span>, <span class="author">de Bakker PI</span>, <span class="author">Kern AD</span>, <span class="author">Lopez-Bigas N</span>, <span class="author">Martin JD</span>, <span class="author">Stranger BE</span>, <span class="author">Woodroffe A</span>, <span class="author">Davydov E</span>, <span class="author">Dimas A</span>, <span class="author">Eyras E</span>, <span class="author">Hallgr&iacute;msd&oacute;ttir IB</span>, <span class="author">Huppert J</span>, <span class="author">Zody MC</span>, <span class="author">Abecasis GR</span>, <span class="author">Estivill X</span>, <span class="author">Bouffard GG</span>, <span class="author">Guan X</span>, <span class="author">Hansen NF</span>, <span class="author">Idol JR</span>, <span class="author">Maduro VV</span>, <span class="author">Maskeri B</span>, <span class="author">McDowell JC</span>, <span class="author">Park M</span>, <span class="author">Thomas PJ</span>, <span class="author">Young AC</span>, <span class="author">Blakesley RW</span>, <span class="author">Muzny DM</span>, <span class="author">Sodergren E</span>, <span class="author">Wheeler DA</span>, <span class="author">Worley KC</span>, <span class="author">Jiang H</span>, <span class="author">Weinstock GM</span>, <span class="author">Gibbs RA</span>, <span class="author">Graves T</span>, <span class="author">Fulton R</span>, <span class="author">Mardis ER</span>, <span class="author">Wilson RK</span>, <span class="author">Clamp M</span>, <span class="author">Cuff J</span>, <span class="author">Gnerre S</span>, <span class="author">Jaffe DB</span>, <span class="author">Chang JL</span>, <span class="author">Lindblad-Toh K</span>, <span class="author">Lander ES</span>, <span class="author">Koriabine M</span>, <span class="author">Nefedov M</span>, <span class="author">Osoegawa K</span>, <span class="author">Yoshinaga Y</span>, <span class="author">Zhu B</span> and <span class="author">de Jong PJ</span></p><p class="abstract">We report the generation and analysis of functional data from multiple, diverse experiments performed on a targeted 1% of the human genome as part of the pilot phase of the ENCODE Project. These data have been further integrated and augmented by a number of evolutionary and computational analyses. Together, our results advance the collective knowledge about human genome function in several major areas. First, our studies provide convincing evidence that the genome is pervasively transcribed, such that the majority of its bases can be found in primary transcripts, including non-protein-coding transcripts, and those that extensively overlap one another. Second, systematic examination of transcriptional regulation has yielded new understanding about transcription start sites, including their relationship to specific regulatory sequences and features of chromatin accessibility and histone modification. Third, a more sophisticated view of chromatin structure has emerged, including its inter-relationship with DNA replication and transcriptional regulation. Finally, integration of these new sources of information, in particular with respect to mammalian evolution based on inter- and intra-species sequence comparisons, has yielded new mechanistic and evolutionary insights concerning the functional landscape of the human genome. Together, these studies are defining a path for pursuit of a more comprehensive characterization of human genome function.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">F32 CA108313</span>; NHGRI NIH HHS: <span class="grant">K22 HG003169</span>, <span class="grant">K22 HG003169-01A1</span>, <span class="grant">P41 HG002371</span>, <span class="grant">P41 HG002371-03S1</span>, <span class="grant">R01 HG002238</span>, <span class="grant">R01 HG002238-15</span>, <span class="grant">R01 HG003110</span>, <span class="grant">R01 HG003110-03</span>, <span class="grant">R01 HG003129-03</span>, <span class="grant">R01 HG003143</span>, <span class="grant">R01 HG003143-04</span>, <span class="grant">R01 HG003521</span>, <span class="grant">R01 HG003521-01</span>, <span class="grant">R01 HG003532</span>, <span class="grant">R01 HG003532-01</span>, <span class="grant">R01 HG003541</span>, <span class="grant">R01 HG003541-03</span>, <span class="grant">U01 HG002523</span>, <span class="grant">U01 HG002523-01</span>, <span class="grant">U01 HG003147</span>, <span class="grant">U01 HG003147-02</span>, <span class="grant">U01 HG003150-03</span>, <span class="grant">U01 HG003151</span>, <span class="grant">U01 HG003151-03</span>, <span class="grant">U01 HG003156</span>, <span class="grant">U01 HG003156-03</span>, <span class="grant">U01 HG003157</span>, <span class="grant">U01 HG003157-03</span>, <span class="grant">U01 HG003161</span>, <span class="grant">U01 HG003161-03</span>, <span class="grant">U01 HG003162</span>, <span class="grant">U01 HG003162-03</span>, <span class="grant">U01 HG003168-02</span>, <span class="grant">U54 HG003067</span>, <span class="grant">U54 HG003067-01</span>, <span class="grant">U54 HG003079</span>, <span class="grant">U54 HG003079-01</span>, <span class="grant">U54 HG003273</span>, <span class="grant">U54 HG003273-01</span>; Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">447</span>;<span class="number">7146</span>;<span class="pagerange">799-816</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17571346">17571346</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2212820">2212820</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature05874">10.1038/nature05874</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HCOP: a searchable database of human orthology predictions.</h4>
<p class="authors"><span class="author">Eyre TA</span>, <span class="author">Wright MW</span>, <span class="author">Lush MJ</span> and <span class="author">Bruford EA</span></p>
<p class="affiliation">HUGO Gene Nomenclature Committee (HGNC), Department of Biology, University College London, Wolfson House, 4 Stephenson Way, London.</p><p class="abstract">The HUGO Gene Nomenclature Committee (HGNC) Comparison of Orthology Predictions (HCOP) search tool combines the human, mouse, rat and chicken orthology assertions made by PhIGs, HomoloGene, Ensembl, Inparanoid, Mouse Genome Informatics (MGI) and HGNC, enabling users to identify predicted ortholog pairs for a specified gene or genes. The HCOP resource provides a useful method to integrate, compare and access a variety of disparate sources of human orthology data. The HCOP search tool, data and documentation are available at http://www.gene.ucl.ac.uk/hcop.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41 HG003345</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">1</span>;<span class="pagerange">2-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16951416" rel="external">16951416</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbib%2Fbbl030" rel="external">10.1093/bib/bbl030</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clinical and molecular genetic spectrum of congenital deficiency of the leptin receptor.</h4>
<p class="authors"><span class="author">Farooqi IS</span>, <span class="author">Wangensteen T</span>, <span class="author">Collins S</span>, <span class="author">Kimber W</span>, <span class="author">Matarese G</span>, <span class="author">Keogh JM</span>, <span class="author">Lank E</span>, <span class="author">Bottomley B</span>, <span class="author">Lopez-Fernandez J</span>, <span class="author">Ferraz-Amaro I</span>, <span class="author">Dattani MT</span>, <span class="author">Ercan O</span>, <span class="author">Myhre AG</span>, <span class="author">Retterstol L</span>, <span class="author">Stanhope R</span>, <span class="author">Edge JA</span>, <span class="author">McKenzie S</span>, <span class="author">Lessan N</span>, <span class="author">Ghodsi M</span>, <span class="author">De Rosa V</span>, <span class="author">Perna F</span>, <span class="author">Fontana S</span>, <span class="author">Barroso I</span>, <span class="author">Undlien DE</span> and <span class="author">O&#39;Rahilly S</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, University Department of Clinical Biochemistry, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom. <a href="mailto:%69%73%66%32%30%40%63%61%6D.%61%63.%75%6B">&#105;&#115;&#102;&#50;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: A single family has been described in which obesity results from a mutation in the leptin-receptor gene (LEPR), but the prevalence of such mutations in severe, early-onset obesity has not been systematically examined.</p><p class="abstract"><strong>Methods</strong>: We sequenced LEPR in 300 subjects with hyperphagia and severe early-onset obesity, including 90 probands from consanguineous families, and investigated the extent to which mutations cosegregated with obesity and affected receptor function. We evaluated metabolic, endocrine, and immune function in probands and affected relatives.</p><p class="abstract"><strong>Results</strong>: Of the 300 subjects, 8 (3%) had nonsense or missense LEPR mutations--7 were homozygotes, and 1 was a compound heterozygote. All missense mutations resulted in impaired receptor signaling. Affected subjects were characterized by hyperphagia, severe obesity, alterations in immune function, and delayed puberty due to hypogonadotropic hypogonadism. Serum leptin levels were within the range predicted by the elevated fat mass in these subjects. Their clinical features were less severe than those of subjects with congenital leptin deficiency.</p><p class="abstract"><strong>Conclusions</strong>: The prevalence of pathogenic LEPR mutations in a cohort of subjects with severe, early-onset obesity was 3%. Circulating levels of leptin were not disproportionately elevated, suggesting that serum leptin cannot be used as a marker for leptin-receptor deficiency. Congenital leptin-receptor deficiency should be considered in the differential diagnosis in any child with hyperphagia and severe obesity in the absence of developmental delay or dysmorphism.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0502115</span>; Telethon: <span class="grant">GJT04008</span>; Wellcome Trust: <span class="grant">067457</span>, <span class="grant">068086</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2007</span>;<span class="volume">356</span>;<span class="number">3</span>;<span class="pagerange">237-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17229951">17229951</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2670197">2670197</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa063988">10.1056/NEJMoa063988</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High resolution array-CGH analysis of single cells.</h4>
<p class="authors"><span class="author">Fiegler H</span>, <span class="author">Geigl JB</span>, <span class="author">Langer S</span>, <span class="author">Rigler D</span>, <span class="author">Porter K</span>, <span class="author">Unger K</span>, <span class="author">Carter NP</span> and <span class="author">Speicher MR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%68%65%69%6B%65%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#101;&#105;&#107;&#101;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Heterogeneity in the genome copy number of tissues is of particular importance in solid tumor biology. Furthermore, many clinical applications such as pre-implantation and non-invasive prenatal diagnosis would benefit from the ability to characterize individual single cells. As the amount of DNA from single cells is so small, several PCR protocols have been developed in an attempt to achieve unbiased amplification. Many of these approaches are suitable for subsequent cytogenetic analyses using conventional methodologies such as comparative genomic hybridization (CGH) to metaphase spreads. However, attempts to harness array-CGH for single-cell analysis to provide improved resolution have been disappointing. Here we describe a strategy that combines single-cell amplification using GenomePlex library technology (GenomePlex) Single Cell Whole Genome Amplification Kit, Sigma-Aldrich, UK) and detailed analysis of genomic copy number changes by high-resolution array-CGH. We show that single copy changes as small as 8.3 Mb in single cells are detected reliably with single cells derived from various tumor cell lines as well as patients presenting with trisomy 21 and Prader-Willi syndrome. Our results demonstrate the potential of this technology for studies of tumor biology and for clinical diagnostics.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">3</span>;<span class="pagerange">e15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17178751">17178751</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1807964">1807964</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl1030">10.1093/nar/gkl1030</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction and use of spotted large-insert clone DNA microarrays for the detection of genomic copy number changes.</h4>
<p class="authors"><span class="author">Fiegler H</span>, <span class="author">Redon R</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%68%65%69%6B%65%40%73%61%6E%67%65%72.%61%63.%75%6B">&#104;&#101;&#105;&#107;&#101;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Microarray-based comparative genomic hybridization has become a widespread method for the analysis of DNA copy number changes across the human genome. Initial methods for microarray construction using large-insert clones required the preparation of DNA from large-scale cultures. This rapidly became an expensive and time-consuming process when expanded to the number of clones needed for higher resolution arrays. To overcome this problem, several PCR-based strategies have been developed to enable array construction from small amounts of cloned DNA. Here, we describe the construction of microarrays composed of human-specific large-insert clones (40-200 kb) using a specific degenerate oligonucleotide PCR strategy. In addition, we also describe array hybridization using manual and automated procedures and methods for array analysis. The technology and protocols described in this article can easily be adapted for other species dependent on the availability of clone libraries. According to our protocols, the procedure will take approximately 3 days from labeling the DNA to scanning the hybridized slides.</p>
<p class="publication"><span class="title">Nature protocols</span>&nbsp;<span class="year">2007</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">577-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17406619">17406619</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2688820">2688820</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnprot.2007.53">10.1038/nprot.2007.53</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the BRWD3 gene cause X-linked mental retardation associated with macrocephaly.</h4>
<p class="authors"><span class="author">Field M</span>, <span class="author">Tarpey PS</span>, <span class="author">Smith R</span>, <span class="author">Edkins S</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Stevens C</span>, <span class="author">Tofts C</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Dicks E</span>, <span class="author">Barthorpe S</span>, <span class="author">Buck G</span>, <span class="author">Cole J</span>, <span class="author">Gray K</span>, <span class="author">Halliday K</span>, <span class="author">Hills K</span>, <span class="author">Jenkinson A</span>, <span class="author">Jones D</span>, <span class="author">Menzies A</span>, <span class="author">Mironenko T</span>, <span class="author">Perry J</span>, <span class="author">Raine K</span>, <span class="author">Richardson D</span>, <span class="author">Shepherd R</span>, <span class="author">Small A</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Mallya U</span>, <span class="author">Wooster R</span>, <span class="author">Moon J</span>, <span class="author">Luo Y</span>, <span class="author">Hughes H</span>, <span class="author">Shaw M</span>, <span class="author">Friend KL</span>, <span class="author">Corbett M</span>, <span class="author">Turner G</span>, <span class="author">Partington M</span>, <span class="author">Mulley J</span>, <span class="author">Bobrow M</span>, <span class="author">Schwartz C</span>, <span class="author">Stevenson R</span>, <span class="author">Gecz J</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span> and <span class="author">Raymond FL</span></p>
<p class="affiliation">GOLD Service, Hunter Genetics, Waratah, Australia.</p><p class="abstract">In the course of systematic screening of the X-chromosome coding sequences in 250 families with nonsyndromic X-linked mental retardation (XLMR), two families were identified with truncating mutations in BRWD3, a gene encoding a bromodomain and WD-repeat domain-containing protein. In both families, the mutation segregates with the phenotype in affected males. Affected males have macrocephaly with a prominent forehead, large cupped ears, and mild-to-moderate intellectual disability. No truncating variants were found in 520 control X chromosomes. BRWD3 is therefore a new gene implicated in the etiology of XLMR associated with macrocephaly and may cause disease by altering intracellular signaling pathways affecting cellular proliferation.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD26202</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">81</span>;<span class="number">2</span>;<span class="pagerange">367-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17668385" rel="external">17668385</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1950797" rel="external">1950797</a>; DOI: <a href="http://dx.doi.org/10.1086%2F520677" rel="external">10.1086/520677</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ProServer: a simple, extensible Perl DAS server.</h4>
<p class="authors"><span class="author">Finn RD</span>, <span class="author">Stalker JW</span>, <span class="author">Jackson DK</span>, <span class="author">Kulesha E</span>, <span class="author">Clements J</span> and <span class="author">Pettett R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Geome Campus, Hinxton, Cambridge, UK.</p><p class="abstract"><strong>Summary</strong>: The increasing size and complexity of biological databases has led to a growing trend to federate rather than duplicate them. In order to share data between federated databases, protocols for the exchange mechanism must be developed. One such data exchange protocol that is widely used is the Distributed Annotation System (DAS). For example, DAS has enabled small experimental groups to integrate their data into the Ensembl genome browser. We have developed ProServer, a simple, lightweight, Perl-based DAS server that does not depend on a separate HTTP server. The ProServer package is easily extensible, allowing data to be served from almost any underlying data model. Recent additions to the DAS protocol have enabled both structure and alignment (sequence and structural) data to be exchanged. ProServer allows both of these data types to be served.</p><p class="abstract"><strong>Availability</strong>: ProServer can be downloaded from http://www.sanger.ac.uk/proserver/ or CPAN http://search.cpan.org/~rpettett/. Details on the system requirements and installation of ProServer can be found at http://www.sanger.ac.uk/proserver/.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">12</span>;<span class="pagerange">1568-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17237073">17237073</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2989875">2989875</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtl650">10.1093/bioinformatics/btl650</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y-chromosomal evidence for a limited Greek contribution to the Pathan population of Pakistan.</h4>
<p class="authors"><span class="author">Firasat S</span>, <span class="author">Khaliq S</span>, <span class="author">Mohyuddin A</span>, <span class="author">Papaioannou M</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Underhill PA</span> and <span class="author">Ayub Q</span></p>
<p class="affiliation">Biomedical and Genetic Engineering Division, Dr. AQ Khan Research Laboratories, Islamabad, Pakistan.</p><p class="abstract">Three Pakistani populations residing in northern Pakistan, the Burusho, Kalash and Pathan claim descent from Greek soldiers associated with Alexander&#39;s invasion of southwest Asia. Earlier studies have excluded a substantial Greek genetic input into these populations, but left open the question of a smaller contribution. We have now typed 90 binary polymorphisms and 16 multiallelic, short-tandem-repeat (STR) loci mapping to the male-specific portion of the human Y chromosome in 952 males, including 77 Greeks in order to re-investigate this question. In pairwise comparisons between the Greeks and the three Pakistani populations using genetic distance measures sensitive to recent events, the lowest distances were observed between the Greeks and the Pathans. Clade E3b1 lineages, which were frequent in the Greeks but not in Pakistan, were nevertheless observed in two Pathan individuals, one of whom shared a 16 Y-STR haplotype with the Greeks. The worldwide distribution of a shortened (9 Y-STR) version of this haplotype, determined from database information, was concentrated in Macedonia and Greece, suggesting an origin there. Although based on only a few unrelated descendants, this provides strong evidence for a European origin for a small proportion of the Pathan Y chromosomes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">1</span>;<span class="pagerange">121-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17047675">17047675</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2588664">2588664</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5201726">10.1038/sj.ejhg.5201726</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a 3;6 translocation associated with renal cell carcinoma.</h4>
<p class="authors"><span class="author">Foster RE</span>, <span class="author">Abdulrahman M</span>, <span class="author">Morris MR</span>, <span class="author">Prigmore E</span>, <span class="author">Gribble S</span>, <span class="author">Ng B</span>, <span class="author">Gentle D</span>, <span class="author">Ready S</span>, <span class="author">Weston PM</span>, <span class="author">Wiesener MS</span>, <span class="author">Kishida T</span>, <span class="author">Yao M</span>, <span class="author">Davison V</span>, <span class="author">Barbero JL</span>, <span class="author">Chu C</span>, <span class="author">Carter NP</span>, <span class="author">Latif F</span> and <span class="author">Maher ER</span></p>
<p class="affiliation">Department of Medical and Molecular Genetics, University of Birmingham, The Medical School, Birmingham B15 2TT, UK.</p><p class="abstract">The most frequent cause of familial clear cell renal cell carcinoma (RCC) is von Hippel-Lindau disease and the VHL tumor suppressor gene (TSG) is inactivated in most sporadic clear cell RCC. Although there is relatively little information on the mechanisms of tumorigenesis of clear cell RCC without VHL inactivation, a subset of familial cases harbors a balanced constitutional chromosome 3 translocation. To date nine different chromosome 3 translocations have been associated with familial or multicentric clear cell RCC; and in three cases chromosome 6 was also involved. To identify candidate genes for renal tumorigenesis we characterized a constitutional translocation, t(3;6)(q22;q16.1) associated with multicentric RCC without evidence of VHL target gene dysregulation. Analysis of breakpoint sequences revealed a 1.3-kb deletion on chromosome 6 within the intron of a 2 exon predicted gene (NT_007299.434). However, RT-PCR analysis failed to detect the expression of this gene in lymphoblast, fibroblast, or kidney tumor cell lines. No known genes were disrupted by the translocation breakpoints but several candidate TSGs (e.g., EPHB1, EPHA7, PPP2R3A RNF184, and STAG1) map within close proximity to the breakpoints.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2007</span>;<span class="volume">46</span>;<span class="number">4</span>;<span class="pagerange">311-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17205537" rel="external">17205537</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2695133" rel="external">2695133</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20403" rel="external">10.1002/gcc.20403</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diffusible signal factor-dependent cell-cell signaling and virulence in the nosocomial pathogen Stenotrophomonas maltophilia.</h4>
<p class="authors"><span class="author">Fouhy Y</span>, <span class="author">Scanlon K</span>, <span class="author">Schouest K</span>, <span class="author">Spillane C</span>, <span class="author">Crossman L</span>, <span class="author">Avison MB</span>, <span class="author">Ryan RP</span> and <span class="author">Dow JM</span></p>
<p class="affiliation">BIOMERIT Research Centre, Department of Microbiology, BioSciences Institute, National University of Ireland, Cork, Ireland.</p><p class="abstract">The genome of Stenotrophomonas maltophilia encodes a cell-cell signaling system that is highly related to the diffusible signal factor (DSF)-dependent system of the phytopathogen Xanthomonas campestris. Here we show that in S. maltophilia, DSF signaling controls factors contributing to the virulence and antibiotic resistance of this important nosocomial pathogen.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">13</span>;<span class="pagerange">4964-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17468254" rel="external">17468254</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1913462" rel="external">1913462</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00310-07" rel="external">10.1128/JB.00310-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structure, mechanism and catalytic duality of thiamine-dependent enzymes.</h4>
<p class="authors"><span class="author">Frank RA</span>, <span class="author">Leeper FJ</span> and <span class="author">Luisi BF</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, Cambridge, UK.</p><p class="abstract">Thiamine is an essential cofactor that is required for processes of general metabolism amongst all organisms, and it is likely to have played a role in the earliest stages of the evolution of life. Here, we review from a structural perspective the enzymatic mechanisms that involve this cofactor. We explore asymmetry within homodimeric thiamine diphosphate (ThDP)-dependent enzyme structures and discuss how this may be correlated with the kinetic properties of half-of-the-sites reactivity, and negative cooperativity. It is likely these structural and kinetic hallmarks may arise through reciprocal coupling of active sites. This mode of communication between distant active sites is not unique to ThDP-dependent enzymes, but is widespread in other classes of oligomeric enzyme. Thus, it appears likely to be a general phenomenon reflecting a powerful mechanism of accelerating the rate of a chemical pathway. Finally, we speculate on the early evolutionary history of the cofactor and its ancient association with protein and RNA.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cellular and molecular life sciences : CMLS</span>&nbsp;<span class="year">2007</span>;<span class="volume">64</span>;<span class="number">7-8</span>;<span class="pagerange">892-905</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17429582" rel="external">17429582</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00018-007-6423-5" rel="external">10.1007/s00018-007-6423-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PPARGC1A coding variation may initiate impaired NEFA clearance during glucose challenge.</h4>
<p class="authors"><span class="author">Franks PW</span>, <span class="author">Ekelund U</span>, <span class="author">Brage S</span>, <span class="author">Luan J</span>, <span class="author">Schafer AJ</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Barroso I</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Genetic Epidemiology and Clinical Research Group, Department of Public Health and Clinical Medicine, Section for Medicine, Ume&aring; University Hospital, Ume&aring;, Sweden. <a href="mailto:%70%61%75%6C.%66%72%61%6E%6B%73%40%6D%65%64%69%63%69%6E.%75%6D%75.%73%65">&#112;&#97;&#117;&#108;&#46;&#102;&#114;&#97;&#110;&#107;&#115;&#64;&#109;&#101;&#100;&#105;&#99;&#105;&#110;&#46;&#117;&#109;&#117;&#46;&#115;&#101;</a></p><p class="abstract">The peroxisome proliferator-activated receptor gamma coactivator 1-alpha protein, encoded by the PPARGC1A gene, transcriptionally activates a complex pathway of lipid and glucose metabolism and is expressed primarily in tissues of high metabolic activity such as liver, heart and exercising oxidative skeletal muscle fibre. Ppargc1a-null mice develop systemic dyslipidaemia and hepatic steatosis. In humans, NEFAs downregulate PPARGC1A expression in skeletal muscle. Furthermore, a common non-synonymous coding variant at PPARGC1A (Gly482Ser, rs8192678) is associated with decreased PPARGC1A mRNA levels and increased type 2 diabetes risk.</p><p class="abstract">In a population-based sample of 691 healthy middle-aged Europids we assessed whether Gly482Ser is associated with levels of NEFA when fasting and in response to an oral glucose challenge. We also assessed the potential effect-modifying role of adipose tissue mass on these phenotypes.</p><p class="abstract"><strong>Results</strong>: After adjustment for age, sex, fat mass and fat-free mass, the Ser482 allele associated with higher NEFA at 30 min and 2 h and with NEFA AUC (all values p&lt;or=0.02). Furthermore, suggestive evidence of interaction between fat mass and Gly482Ser was observed for fasting NEFA (p=0.059). After stratification by level of obesity, genotype associations were observed in the obese for fasting NEFA (p=0.028) and NEFA at 30 min (p=0.013) and 2 h (p=0.002), and with NEFA AUC (p=0.005), but no significant associations were observed in lean individuals (all values p&gt;0.6).</p><p class="abstract">Our observations indicate that NEFA clearance is blunted following a glucose load in carriers of the PPARCG1A Ser482 allele. This association is augmented by obesity.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179473</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Diabetologia</span>&nbsp;<span class="year">2007</span>;<span class="volume">50</span>;<span class="number">3</span>;<span class="pagerange">569-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17216277">17216277</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2682771">2682771</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00125-006-0580-1">10.1007/s00125-006-0580-1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity.</h4>
<p class="authors"><span class="author">Frayling TM</span>, <span class="author">Timpson NJ</span>, <span class="author">Weedon MN</span>, <span class="author">Zeggini E</span>, <span class="author">Freathy RM</span>, <span class="author">Lindgren CM</span>, <span class="author">Perry JR</span>, <span class="author">Elliott KS</span>, <span class="author">Lango H</span>, <span class="author">Rayner NW</span>, <span class="author">Shields B</span>, <span class="author">Harries LW</span>, <span class="author">Barrett JC</span>, <span class="author">Ellard S</span>, <span class="author">Groves CJ</span>, <span class="author">Knight B</span>, <span class="author">Patch AM</span>, <span class="author">Ness AR</span>, <span class="author">Ebrahim S</span>, <span class="author">Lawlor DA</span>, <span class="author">Ring SM</span>, <span class="author">Ben-Shlomo Y</span>, <span class="author">Jarvelin MR</span>, <span class="author">Sovio U</span>, <span class="author">Bennett AJ</span>, <span class="author">Melzer D</span>, <span class="author">Ferrucci L</span>, <span class="author">Loos RJ</span>, <span class="author">Barroso I</span>, <span class="author">Wareham NJ</span>, <span class="author">Karpe F</span>, <span class="author">Owen KR</span>, <span class="author">Cardon LR</span>, <span class="author">Walker M</span>, <span class="author">Hitman GA</span>, <span class="author">Palmer CN</span>, <span class="author">Doney AS</span>, <span class="author">Morris AD</span>, <span class="author">Smith GD</span>, <span class="author">Hattersley AT</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK.</p><p class="abstract">Obesity is a serious international health problem that increases the risk of several common diseases. The genetic factors predisposing to obesity are poorly understood. A genome-wide search for type 2 diabetes-susceptibility genes identified a common variant in the FTO (fat mass and obesity associated) gene that predisposes to diabetes through an effect on body mass index (BMI). An additive association of the variant with BMI was replicated in 13 cohorts with 38,759 participants. The 16% of adults who are homozygous for the risk allele weighed about 3 kilograms more and had 1.67-fold increased odds of obesity when compared with those not inheriting a risk allele. This association was observed from age 7 years upward and reflects a specific increase in fat mass.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0500070</span>, <span class="grant">G0600705</span>, <span class="grant">G9815508</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; NIA NIH HHS: <span class="grant">Z99 AG999999</span>; Wellcome Trust: <span class="grant">079557</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">316</span>;<span class="number">5826</span>;<span class="pagerange">889-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17434869">17434869</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2646098">2646098</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1141634">10.1126/science.1141634</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Definition of the zebrafish genome using flow cytometry and cytogenetic mapping.</h4>
<p class="authors"><span class="author">Freeman JL</span>, <span class="author">Adeniyi A</span>, <span class="author">Banerjee R</span>, <span class="author">Dallaire S</span>, <span class="author">Maguire SF</span>, <span class="author">Chi J</span>, <span class="author">Ng BL</span>, <span class="author">Zepeda C</span>, <span class="author">Scott CE</span>, <span class="author">Humphray S</span>, <span class="author">Rogers J</span>, <span class="author">Zhou Y</span>, <span class="author">Zon LI</span>, <span class="author">Carter NP</span>, <span class="author">Yang F</span> and <span class="author">Lee C</span></p>
<p class="affiliation">Department of Pathology, Brigham and Women&#39;s Hospital, Boston, Massachusetts 02115, USA. <a href="mailto:%6A%66%72%65%65%6D%61%6E%37%40%70%61%72%74%6E%65%72%73.%6F%72%67">&#106;&#102;&#114;&#101;&#101;&#109;&#97;&#110;&#55;&#64;&#112;&#97;&#114;&#116;&#110;&#101;&#114;&#115;&#46;&#111;&#114;&#103;</a> &lt;<a href="mailto:%6A%66%72%65%65%6D%61%6E%37%40%70%61%72%74%6E%65%72%73.%6F%72%67&%67%74">&#106;&#102;&#114;&#101;&#101;&#109;&#97;&#110;&#55;&#64;&#112;&#97;&#114;&#116;&#110;&#101;&#114;&#115;&#46;&#111;&#114;&#103;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: The zebrafish (Danio rerio) is an important vertebrate model organism system for biomedical research. The syntenic conservation between the zebrafish and human genome allows one to investigate the function of human genes using the zebrafish model. To facilitate analysis of the zebrafish genome, genetic maps have been constructed and sequence annotation of a reference zebrafish genome is ongoing. However, the duplicative nature of teleost genomes, including the zebrafish, complicates accurate assembly and annotation of a representative genome sequence. Cytogenetic approaches provide &quot;anchors&quot; that can be integrated with accumulating genomic data.</p><p class="abstract"><strong>Results</strong>: Here, we cytogenetically define the zebrafish genome by first estimating the size of each linkage group (LG) chromosome using flow cytometry, followed by the cytogenetic mapping of 575 bacterial artificial chromosome (BAC) clones onto metaphase chromosomes. Of the 575 BAC clones, 544 clones localized to apparently unique chromosomal locations. 93.8% of these clones were assigned to a specific LG chromosome location using fluorescence in situ hybridization (FISH) and compared to the LG chromosome assignment reported in the zebrafish genome databases. Thirty-one BAC clones localized to multiple chromosomal locations in several different hybridization patterns. From these data, a refined second generation probe panel for each LG chromosome was also constructed.</p><p class="abstract"><strong>Conclusion</strong>: The chromosomal mapping of the 575 large-insert DNA clones allows for these clones to be integrated into existing zebrafish mapping data. An accurately annotated zebrafish reference genome serves as a valuable resource for investigating the molecular basis of human diseases using zebrafish mutant models.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">R01-CA111560</span>; NHLBI NIH HHS: <span class="grant">T32 HL007627</span>; Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">195</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17597531">17597531</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1925092">1925092</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-8-195">10.1186/1471-2164-8-195</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction, visualisation, and clustering of transcription networks from microarray expression data.</h4>
<p class="authors"><span class="author">Freeman TC</span>, <span class="author">Goldovsky L</span>, <span class="author">Brosch M</span>, <span class="author">van Dongen S</span>, <span class="author">Mazi&egrave;re P</span>, <span class="author">Grocock RJ</span>, <span class="author">Freilich S</span>, <span class="author">Thornton J</span> and <span class="author">Enright AJ</span></p>
<p class="affiliation">Division of Pathway Medicine, University of Edinburgh Medical School, Edinburgh, United Kingdom. <a href="mailto:%74%66%72%65%65%6D%61%6E%40%73%74%61%66%66%6D%61%69%6C.%65%64.%61%63.%75%6B">&#116;&#102;&#114;&#101;&#101;&#109;&#97;&#110;&#64;&#115;&#116;&#97;&#102;&#102;&#109;&#97;&#105;&#108;&#46;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Network analysis transcends conventional pairwise approaches to data analysis as the context of components in a network graph can be taken into account. Such approaches are increasingly being applied to genomics data, where functional linkages are used to connect genes or proteins. However, while microarray gene expression datasets are now abundant and of high quality, few approaches have been developed for analysis of such data in a network context. We present a novel approach for 3-D visualisation and analysis of transcriptional networks generated from microarray data. These networks consist of nodes representing transcripts connected by virtue of their expression profile similarity across multiple conditions. Analysing genome-wide gene transcription across 61 mouse tissues, we describe the unusual topography of the large and highly structured networks produced, and demonstrate how they can be used to visualise, cluster, and mine large datasets. This approach is fast, intuitive, and versatile, and allows the identification of biological relationships that may be missed by conventional analysis techniques. This work has been implemented in a freely available open-source application named BioLayout Express(3D).</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">10</span>;<span class="pagerange">2032-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17967053" rel="external">17967053</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2041979" rel="external">2041979</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.0030206" rel="external">10.1371/journal.pcbi.0030206</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Backseat drivers take the wheel.</h4>
<p class="authors"><span class="author">Futreal PA</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK. <a href="mailto:%70%61%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Somatic mutations in human cancers are comprised of those that contribute to the oncogenic phenotype, driver mutations, and those that reflect the general patterns of exposure and disrepair but are otherwise noncontributory, passenger mutations. Distinguishing drivers that can be of low frequency in any given tumor type from often more numerous passengers is a key challenge. In this issue of Cancer Cell, Fr&ouml;hling and colleagues tackle this challenge admirably for the known cancer gene FLT3 in acute myeloid leukemia--undertaking a systematic resequencing and functional validation approach, identifying important rare driver mutations as well as passenger mutations in patients negative for the more common activating mutations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Cancer cell</span>&nbsp;<span class="year">2007</span>;<span class="volume">12</span>;<span class="number">6</span>;<span class="pagerange">493-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18068625" rel="external">18068625</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2705833" rel="external">2705833</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ccr.2007.11.020" rel="external">10.1016/j.ccr.2007.11.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fxna, a novel gene differentially expressed in the rat ovary at the time of folliculogenesis, is required for normal ovarian histogenesis.</h4>
<p class="authors"><span class="author">Garcia-Rudaz C</span>, <span class="author">Luna F</span>, <span class="author">Tapia V</span>, <span class="author">Kerr B</span>, <span class="author">Colgin L</span>, <span class="author">Galimi F</span>, <span class="author">Dissen GA</span>, <span class="author">Rawlings ND</span> and <span class="author">Ojeda SR</span></p>
<p class="affiliation">Division of Neuroscience, Oregon National Primate Research Center, Oregon Health and Science University, Beaverton, OR , USA.</p><p class="abstract">In rodents, the formation of ovarian follicles occurs after birth. In recent years, several factors required for follicular assembly and the growth of the newly formed follicles have been identified. We now describe a novel gene, Fxna, identified by differential display in the neonatal rat ovary. Fxna encodes an mRNA of 5.4 kb, and a protein of 898 amino acids. Fxna is a transmembrane metallopeptidase from family M28, localized to the endoplasmic reticulum. In the ovary, Fxna mRNA is expressed in granulosa cells; its abundance is maximal 48 hours after birth, i.e. during the initiation of follicular assembly. Reducing Fxna mRNA levels via lentiviral-mediated delivery of short hairpin RNAs to neonatal ovaries resulted in substantial loss of primordial, primary and secondary follicles, and structural disorganization of the ovary, with many abnormal follicles containing more than one oocyte and clusters of somatic cells not associated with any oocytes. These abnormalities were not attributable to either increased apoptosis or decreased proliferation of granulosa cells. The results indicate that Fxna is required for the organization of somatic cells and oocytes into discrete follicular structures. As an endoplasmic reticulum-bound peptidase, Fxna may facilitate follicular organization by processing precursor proteins required for intraovarian cell-to-cell communication.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">RR-00163</span>; NICHD NIH HHS: <span class="grant">HD-24870</span>, <span class="grant">TW/HD00668</span>, <span class="grant">U54 HD18185</span></p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">134</span>;<span class="number">5</span>;<span class="pagerange">945-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17267443" rel="external">17267443</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.02795" rel="external">10.1242/dev.02795</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale mapping of mutations affecting zebrafish development.</h4>
<p class="authors"><span class="author">Geisler R</span>, <span class="author">Rauch GJ</span>, <span class="author">Geiger-Rudolph S</span>, <span class="author">Albrecht A</span>, <span class="author">van Bebber F</span>, <span class="author">Berger A</span>, <span class="author">Busch-Nentwich E</span>, <span class="author">Dahm R</span>, <span class="author">Dekens MP</span>, <span class="author">Dooley C</span>, <span class="author">Elli AF</span>, <span class="author">Gehring I</span>, <span class="author">Geiger H</span>, <span class="author">Geisler M</span>, <span class="author">Glaser S</span>, <span class="author">Holley S</span>, <span class="author">Huber M</span>, <span class="author">Kerr A</span>, <span class="author">Kirn A</span>, <span class="author">Knirsch M</span>, <span class="author">Konantz M</span>, <span class="author">K&uuml;chler AM</span>, <span class="author">Maderspacher F</span>, <span class="author">Neuhauss SC</span>, <span class="author">Nicolson T</span>, <span class="author">Ober EA</span>, <span class="author">Praeg E</span>, <span class="author">Ray R</span>, <span class="author">Rentzsch B</span>, <span class="author">Rick JM</span>, <span class="author">Rief E</span>, <span class="author">Schauerte HE</span>, <span class="author">Schepp CP</span>, <span class="author">Sch&ouml;nberger U</span>, <span class="author">Schonthaler HB</span>, <span class="author">Seiler C</span>, <span class="author">Sidi S</span>, <span class="author">S&ouml;llner C</span>, <span class="author">Wehner A</span>, <span class="author">Weiler C</span> and <span class="author">N&uuml;sslein-Volhard C</span></p>
<p class="affiliation">Department 3--Genetics, Max-Planck-Institut f&uuml;r Entwicklungsbiologie, Spemannstr, 35/III, 72076 T&uuml;bingen, Germany. <a href="mailto:%72%6F%62%65%72%74.%67%65%69%73%6C%65%72%40%74%75%65%62%69%6E%67%65%6E.%6D%70%67.%64%65">&#114;&#111;&#98;&#101;&#114;&#116;&#46;&#103;&#101;&#105;&#115;&#108;&#101;&#114;&#64;&#116;&#117;&#101;&#98;&#105;&#110;&#103;&#101;&#110;&#46;&#109;&#112;&#103;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: Large-scale mutagenesis screens in the zebrafish employing the mutagen ENU have isolated several hundred mutant loci that represent putative developmental control genes. In order to realize the potential of such screens, systematic genetic mapping of the mutations is necessary. Here we report on a large-scale effort to map the mutations generated in mutagenesis screening at the Max Planck Institute for Developmental Biology by genome scanning with microsatellite markers.</p><p class="abstract"><strong>Results</strong>: We have selected a set of microsatellite markers and developed methods and scoring criteria suitable for efficient, high-throughput genome scanning. We have used these methods to successfully obtain a rough map position for 319 mutant loci from the T&uuml;bingen I mutagenesis screen and subsequent screening of the mutant collection. For 277 of these the corresponding gene is not yet identified. Mapping was successful for 80 % of the tested loci. By comparing 21 mutation and gene positions of cloned mutations we have validated the correctness of our linkage group assignments and estimated the standard error of our map positions to be approximately 6 cM.</p><p class="abstract"><strong>Conclusion</strong>: By obtaining rough map positions for over 300 zebrafish loci with developmental phenotypes, we have generated a dataset that will be useful not only for cloning of the affected genes, but also to suggest allelism of mutations with similar phenotypes that will be identified in future screens. Furthermore this work validates the usefulness of our methodology for rapid, systematic and inexpensive microsatellite mapping of zebrafish mutations.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17212827" rel="external">17212827</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1781435" rel="external">1781435</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-8-11" rel="external">10.1186/1471-2164-8-11</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNA interference in parasitic helminths: current situation, potential pitfalls and future prospects.</h4>
<p class="authors"><span class="author">Geldhof P</span>, <span class="author">Visser A</span>, <span class="author">Clark D</span>, <span class="author">Saunders G</span>, <span class="author">Britton C</span>, <span class="author">Gilleard J</span>, <span class="author">Berriman M</span> and <span class="author">Knox D</span></p>
<p class="affiliation">Faculty of Veterinary Medicine, Department of Virology, Parasitology and Immunology, Salisburylaan 133, B-9820 Merelbeke, Belgium. <a href="mailto:%50%65%74%65%72.%67%65%6C%64%68%6F%66%40%55%67%65%6E%74.%62%65">&#80;&#101;&#116;&#101;&#114;&#46;&#103;&#101;&#108;&#100;&#104;&#111;&#102;&#64;&#85;&#103;&#101;&#110;&#116;&#46;&#98;&#101;</a></p><p class="abstract">RNA interference (RNAi) has become an invaluable tool for the functional analysis of genes in a wide variety of organisms including the free-living nematode Caenorhabditis elegans. Recently, attempts have been made to apply this technology to parasitic helminths of animals and plants with variable success. Gene knockdown has been reported for Schistosoma mansoni by soaking or electroporating different life-stages in dsRNA. Similar approaches have been tested on parasitic nematodes which clearly showed that, under certain conditions, it was possible to interfere with gene expression. However, despite these successes, the current utility of this technology in parasite research is questionable. First, problems have arisen with the specificity of RNAi. Treatment of the parasites with dsRNA resulted, in many cases, in non-specific effects. Second, the current RNAi methods have a limited efficiency and effects are sometimes difficult to reproduce. This was especially the case in strongylid parasites where only a small number of genes were susceptible to RNAi-mediated gene knockdown. The future application of RNAi in parasite functional genomics will greatly depend on how we can overcome these difficulties. Optimization of the dsRNA delivery methods and in vitro culture conditions will be the major challenges.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Parasitology</span>&nbsp;<span class="year">2007</span>;<span class="volume">134</span>;<span class="number">Pt 5</span>;<span class="pagerange">609-19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17201997" rel="external">17201997</a>; DOI: <a href="http://dx.doi.org/10.1017%2FS0031182006002071" rel="external">10.1017/S0031182006002071</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The obesity-associated FTO gene encodes a 2-oxoglutarate-dependent nucleic acid demethylase.</h4>
<p class="authors"><span class="author">Gerken T</span>, <span class="author">Girard CA</span>, <span class="author">Tung YC</span>, <span class="author">Webby CJ</span>, <span class="author">Saudek V</span>, <span class="author">Hewitson KS</span>, <span class="author">Yeo GS</span>, <span class="author">McDonough MA</span>, <span class="author">Cunliffe S</span>, <span class="author">McNeill LA</span>, <span class="author">Galvanovskis J</span>, <span class="author">Rorsman P</span>, <span class="author">Robins P</span>, <span class="author">Prieur X</span>, <span class="author">Coll AP</span>, <span class="author">Ma M</span>, <span class="author">Jovanovic Z</span>, <span class="author">Farooqi IS</span>, <span class="author">Sedgwick B</span>, <span class="author">Barroso I</span>, <span class="author">Lindahl T</span>, <span class="author">Ponting CP</span>, <span class="author">Ashcroft FM</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Schofield CJ</span></p>
<p class="affiliation">Chemistry Research Laboratory and Oxford Centre for Integrative Systems Biology, University of Oxford, 12 Mansfield Road, Oxford, Oxon OX1 3TA, UK.</p><p class="abstract">Variants in the FTO (fat mass and obesity associated) gene are associated with increased body mass index in humans. Here, we show by bioinformatics analysis that FTO shares sequence motifs with Fe(II)- and 2-oxoglutarate-dependent oxygenases. We find that recombinant murine Fto catalyzes the Fe(II)- and 2OG-dependent demethylation of 3-methylthymine in single-stranded DNA, with concomitant production of succinate, formaldehyde, and carbon dioxide. Consistent with a potential role in nucleic acid demethylation, Fto localizes to the nucleus in transfected cells. Studies of wild-type mice indicate that Fto messenger RNA (mRNA) is most abundant in the brain, particularly in hypothalamic nuclei governing energy balance, and that Fto mRNA levels in the arcuate nucleus are regulated by feeding and fasting. Studies can now be directed toward determining the physiologically relevant FTO substrate and how nucleic acid methylation status is linked to increased fat mass.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G108/617</span>, <span class="grant">G9824984</span>, <span class="grant">MC_U137761446</span>; NIGMS NIH HHS: <span class="grant">U54 GM064346</span>; Wellcome Trust: <span class="grant">068086</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">318</span>;<span class="number">5855</span>;<span class="pagerange">1469-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17991826">17991826</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2668859">2668859</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1151710">10.1126/science.1151710</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Draft genome of the filarial nematode parasite Brugia malayi.</h4>
<p class="authors"><span class="author">Ghedin E</span>, <span class="author">Wang S</span>, <span class="author">Spiro D</span>, <span class="author">Caler E</span>, <span class="author">Zhao Q</span>, <span class="author">Crabtree J</span>, <span class="author">Allen JE</span>, <span class="author">Delcher AL</span>, <span class="author">Guiliano DB</span>, <span class="author">Miranda-Saavedra D</span>, <span class="author">Angiuoli SV</span>, <span class="author">Creasy T</span>, <span class="author">Amedeo P</span>, <span class="author">Haas B</span>, <span class="author">El-Sayed NM</span>, <span class="author">Wortman JR</span>, <span class="author">Feldblyum T</span>, <span class="author">Tallon L</span>, <span class="author">Schatz M</span>, <span class="author">Shumway M</span>, <span class="author">Koo H</span>, <span class="author">Salzberg SL</span>, <span class="author">Schobel S</span>, <span class="author">Pertea M</span>, <span class="author">Pop M</span>, <span class="author">White O</span>, <span class="author">Barton GJ</span>, <span class="author">Carlow CK</span>, <span class="author">Crawford MJ</span>, <span class="author">Daub J</span>, <span class="author">Dimmic MW</span>, <span class="author">Estes CF</span>, <span class="author">Foster JM</span>, <span class="author">Ganatra M</span>, <span class="author">Gregory WF</span>, <span class="author">Johnson NM</span>, <span class="author">Jin J</span>, <span class="author">Komuniecki R</span>, <span class="author">Korf I</span>, <span class="author">Kumar S</span>, <span class="author">Laney S</span>, <span class="author">Li BW</span>, <span class="author">Li W</span>, <span class="author">Lindblom TH</span>, <span class="author">Lustigman S</span>, <span class="author">Ma D</span>, <span class="author">Maina CV</span>, <span class="author">Martin DM</span>, <span class="author">McCarter JP</span>, <span class="author">McReynolds L</span>, <span class="author">Mitreva M</span>, <span class="author">Nutman TB</span>, <span class="author">Parkinson J</span>, <span class="author">Peregr&iacute;n-Alvarez JM</span>, <span class="author">Poole C</span>, <span class="author">Ren Q</span>, <span class="author">Saunders L</span>, <span class="author">Sluder AE</span>, <span class="author">Smith K</span>, <span class="author">Stanke M</span>, <span class="author">Unnasch TR</span>, <span class="author">Ware J</span>, <span class="author">Wei AD</span>, <span class="author">Weil G</span>, <span class="author">Williams DJ</span>, <span class="author">Zhang Y</span>, <span class="author">Williams SA</span>, <span class="author">Fraser-Liggett C</span>, <span class="author">Slatko B</span>, <span class="author">Blaxter ML</span> and <span class="author">Scott AL</span></p>
<p class="affiliation">Division of Infectious Diseases, University of Pittsburgh School of Medicine, Pittsburgh, PA 15261, USA. <a href="mailto:%47%68%65%64%69%6E%45%40%64%6F%6D.%70%69%74%74.%65%64%75">&#71;&#104;&#101;&#100;&#105;&#110;&#69;&#64;&#100;&#111;&#109;&#46;&#112;&#105;&#116;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Parasitic nematodes that cause elephantiasis and river blindness threaten hundreds of millions of people in the developing world. We have sequenced the approximately 90 megabase (Mb) genome of the human filarial parasite Brugia malayi and predict approximately 11,500 protein coding genes in 71 Mb of robustly assembled sequence. Comparative analysis with the free-living, model nematode Caenorhabditis elegans revealed that, despite these genes having maintained little conservation of local synteny during approximately 350 million years of evolution, they largely remain in linkage on chromosomal units. More than 100 conserved operons were identified. Analysis of the predicted proteome provides evidence for adaptations of B. malayi to niches in its human and vector hosts and insights into the molecular basis of a mutualistic relationship with its Wolbachia endosymbiont. These findings offer a foundation for rational drug design.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI048562</span>, <span class="grant">R01 AI048562-09</span>, <span class="grant">U01-AI50903</span>; NIEHS NIH HHS: <span class="grant">R15 ES013128</span>, <span class="grant">R15 ES013128-01</span>; NLM NIH HHS: <span class="grant">R01 LM006845</span>, <span class="grant">R01 LM006845-08</span>, <span class="grant">R01 LM007938</span>, <span class="grant">R01 LM007938-04</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">317</span>;<span class="number">5845</span>;<span class="pagerange">1756-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17885136">17885136</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2613796">2613796</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1145406">10.1126/science.1145406</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal evolution in tenrecs (Microgale and Oryzorictes, Tenrecidae) from the Central Highlands of Madagascar.</h4>
<p class="authors"><span class="author">Gilbert C</span>, <span class="author">Goodman SM</span>, <span class="author">Soarimalala V</span>, <span class="author">Olson LE</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Elder FF</span>, <span class="author">Yang F</span>, <span class="author">Ferguson-Smith MA</span> and <span class="author">Robinson TJ</span></p>
<p class="affiliation">Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Stellenbosch, South Africa.</p><p class="abstract">Tenrecs (Tenrecidae) are a widely diversified assemblage of small eutherian mammals that occur in Madagascar and Western and Central Africa. With the exception of a few early karyotypic descriptions based on conventional staining, nothing is known about the chromosomal evolution of this family. We present a detailed analysis of G-banded and molecularly defined chromosomes based on fluorescence in situ hybridization (FISH) that allows a comprehensive comparison between the karyotypes of 11 species of two closely related Malagasy genera, Microgale (10 species) and Oryzorictes (one species), of the subfamily Oryzorictinae. The karyotypes of Microgale taiva and M. parvula (2n = 32) were found to be identical to that of O. hova (2n = 32) most likely reflecting the ancestral karyotypes of both genera, as well as that of the Oryzorictinae. Parsimony analysis of chromosomal rearrangements that could have arisen following Whole Arm Reciprocal Translocations (WARTs) showed, however, that these are more likely to be the result of Robertsonian translocations. A single most parsimonious tree was obtained that provides strong support for three species associations within Microgale, all of which are consistent with previous molecular and morphological investigations. By expanding on a recently published molecular clock for the Tenrecidae we were able to place our findings in a temporal framework that shows strong chromosomal rate heterogeneity within the Oryzorictinae. We use these data to critically examine the possible role of chromosomal rearrangements in speciation within Microgale.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">8</span>;<span class="pagerange">1075-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18058249">18058249</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1182-6">10.1007/s10577-007-1182-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBook: the online review of Caenorhabditis elegans biology.</h4>
<p class="authors"><span class="author">Girard LR</span>, <span class="author">Fiedler TJ</span>, <span class="author">Harris TW</span>, <span class="author">Carvalho F</span>, <span class="author">Antoshechkin I</span>, <span class="author">Han M</span>, <span class="author">Sternberg PW</span>, <span class="author">Stein LD</span> and <span class="author">Chalfie M</span></p>
<p class="affiliation">Division of Biology, 156-29, Pasadena, CA 91125, USA. <a href="mailto:%6C%67%69%72%61%72%64%40%63%61%6C%74%65%63%68.%65%64%75">&#108;&#103;&#105;&#114;&#97;&#114;&#100;&#64;&#99;&#97;&#108;&#116;&#101;&#99;&#104;&#46;&#101;&#100;&#117;</a></p><p class="abstract">WormBook (www.wormbook.org) is an open-access, online collection of original, peer-reviewed chapters on the biology of Caenorhabditis elegans and related nematodes. Since WormBook was launched in June 2005 with 12 chapters, it has grown to over 100 chapters, covering nearly every aspect of C.elegans research, from Cell Biology and Neurobiology to Evolution and Ecology. WormBook also serves as the text companion to WormBase, the C.elegans model organism database. Objects such as genes, proteins and cells are linked to the relevant pages in WormBase, providing easily accessible background information. Additionally, WormBook chapters contain links to other relevant topics in WormBook, and the in-text citations are linked to their abstracts in PubMed and full-text references, if available. Since WormBook is online, its chapters are able to contain movies and complex images that would not be possible in a print version. WormBook is designed to keep up with the rapid pace of discovery in the field of C.elegans research and continues to grow. WormBook represents a generic publishing infrastructure that is easily adaptable to other research communities to facilitate the dissemination of knowledge in the field.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41 HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">Database issue</span>;<span class="pagerange">D472-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17099225" rel="external">17099225</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1669767" rel="external">1669767</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl894" rel="external">10.1093/nar/gkl894</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Setting the tempo in development: an investigation of the zebrafish somite clock mechanism.</h4>
<p class="authors"><span class="author">Giudicelli F</span>, <span class="author">Ozbudak EM</span>, <span class="author">Wright GJ</span> and <span class="author">Lewis J</span></p>
<p class="affiliation">Vertebrate Development Laboratory, Cancer Research UK London Research Institute, London, United Kingdom.</p><p class="abstract">The somites of the vertebrate embryo are clocked out sequentially from the presomitic mesoderm (PSM) at the tail end of the embryo. Formation of each somite corresponds to one cycle of oscillation of the somite segmentation clock--a system of genes whose expression switches on and off periodically in the cells of the PSM. We have previously proposed a simple mathematical model explaining how the oscillations, in zebrafish at least, may be generated by a delayed negative feedback loop in which the products of two Notch target genes, her1 and her7, directly inhibit their own transcription, as well as that of the gene for the Notch ligand DeltaC; Notch signalling via DeltaC keeps the oscillations of neighbouring cells in synchrony. Here we subject the model to quantitative tests. We show how to read temporal information from the spatial pattern of stripes of gene expression in the anterior PSM and in this way obtain values for the biosynthetic delays and molecular lifetimes on which the model critically depends. Using transgenic lines of zebrafish expressing her1 or her7 under heat-shock control, we confirm the regulatory relationships postulated by the model. From the timing of somite segmentation disturbances following a pulse of her7 misexpression, we deduce that although her7 continues to oscillate in the anterior half of the PSM, it governs the future somite segmentation behaviour of the cells only while they are in the posterior half. In general, the findings strongly support the mathematical model of how the somite clock works, but they do not exclude the possibility that other oscillator mechanisms may operate upstream from the her7/her1 oscillator or in parallel with it.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e150</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17535112" rel="external">17535112</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1877819" rel="external">1877819</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0050150" rel="external">10.1371/journal.pbio.0050150</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cutaneous immune responses in the common carp detected using transcript analysis.</h4>
<p class="authors"><span class="author">Gonzalez SF</span>, <span class="author">Chatziandreou N</span>, <span class="author">Nielsen ME</span>, <span class="author">Li W</span>, <span class="author">Rogers J</span>, <span class="author">Taylor R</span>, <span class="author">Santos Y</span> and <span class="author">Cossins A</span></p>
<p class="affiliation">Department of Veterinary Pathobiology, Laboratory for Fish Diseases, The Royal Veterinary and Agricultural University, Stigb&oslash;jlen 7, DK-1870 Frederiksberg C, Denmark. <a href="mailto:%78%61%6E%74%69%40%75%73%63.%65%73">&#120;&#97;&#110;&#116;&#105;&#64;&#117;&#115;&#99;&#46;&#101;&#115;</a></p><p class="abstract">In order to detect new immune-related genes in common carp (Cyprinus carpio L.) challenged by an ectoparasitic infection, two cDNA libraries were constructed from carp skin sampled at 3 and 72h after infection with Ichthyophthirius multifiliis. In a total of 3500 expressed sequence tags (ESTs) we identified 82 orthologues of genes of immune relevance previously described in other organisms. Of these, 61 have never been described before in C. carpio, thus shedding light on some key components of the defence mechanisms of this species. Among the newly described genes, full-length molecules of prostaglandin D2 synthase (PGDS), the CC chemokine molecule SCYA103, and a second gene for the carp beta(2)-microglobulin (beta(2)m), beta(2)m-2, were described. Transcript amounts of the genes PGDS, interferon (IFN), SCYA103, complement factor 7 (C7), complement factor P (FP), complement factor D (FD) and beta(2)m-2 were evaluated by real-time quantitative PCR (RQ-PCR). Samples from skin, blood and liver from fish challenged with I. multifiliis were taken at 3, 12, 24, 36 and 48h post infection. Higher expression levels of most of these transcripts were observed in skin from uninfected fish, compared to the transcript levels detected in blood and liver from the same animals. Also, there was significant down-regulation of the genes PGDS and beta(2)m-2 in skin, whilst significant up-regulation was observed for the C7 and SCYA103 genes in liver of fish infected with the parasite. These results confirm the active role of fish skin in the immune response against infections, acting as an important site of expression of immune-related molecules.</p>
<p class="publication"><span class="title">Molecular immunology</span>&nbsp;<span class="year">2007</span>;<span class="volume">44</span>;<span class="number">7</span>;<span class="pagerange">1664-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17049603" rel="external">17049603</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molimm.2006.08.016" rel="external">10.1016/j.molimm.2006.08.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide dynamics of SAPHIRE, an essential complex for gene activation and chromatin boundaries.</h4>
<p class="authors"><span class="author">Gordon M</span>, <span class="author">Holt DG</span>, <span class="author">Panigrahi A</span>, <span class="author">Wilhelm BT</span>, <span class="author">Erdjument-Bromage H</span>, <span class="author">Tempst P</span>, <span class="author">B&auml;hler J</span> and <span class="author">Cairns BR</span></p>
<p class="affiliation">University of Utah, Huntsman Cancer Institute, 2000 Circle of Hope, Salt Lake City, UT 84112, USA.</p><p class="abstract">In this study, we characterize a four-protein nucleosome-binding complex from Schizosaccharomyces pombe, termed SAPHIRE, that includes two orthologs of human Lsd1, a histone demethylase. The SAPHIRE complex is essential for cell viability, whereas saphire mutants lacking key conserved catalytic residues are viable but thermosensitive, suggesting that SAPHIRE has both an important enzymatic function and an essential nonenzymatic function. SAPHIRE is present in (or adjacent to) particular heterochromatic loci and also in the transcription start site regions of many highly active polymerase II genes. However, ribosomal protein genes are notably SAPHIRE deficient. SAPHIRE promotes activation, as target genes are selectively attenuated in saphire mutants. Interestingly, saphire mutants display increased histone H3 lysine 4 dimethylation, a modification typically associated with euchromatin. SAPHIRE localization is dynamic, as activated genes rapidly acquire SAPHIRE. Furthermore, saphire mutants dramatically shift a heterochromatin-euchromatin boundary in Chr1, suggesting a novel role in boundary regulation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">27</span>;<span class="number">11</span>;<span class="pagerange">4058-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17371846" rel="external">17371846</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1900013" rel="external">1900013</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.02044-06" rel="external">10.1128/MCB.02044-06</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modulation of steroidogenic gene expression and hormone production of H295R cells by pharmaceuticals and other environmentally active compounds.</h4>
<p class="authors"><span class="author">Gracia T</span>, <span class="author">Hilscherova K</span>, <span class="author">Jones PD</span>, <span class="author">Newsted JL</span>, <span class="author">Higley EB</span>, <span class="author">Zhang X</span>, <span class="author">Hecker M</span>, <span class="author">Murphy MB</span>, <span class="author">Yu RM</span>, <span class="author">Lam PK</span>, <span class="author">Wu RS</span> and <span class="author">Giesy JP</span></p>
<p class="affiliation">Department of Zoology, National Food Safety and Toxicology Center, Center for Integrative Toxicology, Michigan State University, East Lansing, MI 48824, USA. <a href="mailto:%74%67%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#103;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The H295R cell bioassay was used to evaluate the potential endocrine disrupting effects of 18 of the most commonly used pharmaceuticals in the United States. Exposures for 48 h with single pharmaceuticals and binary mixtures were conducted; the expression of five steroidogenic genes, 3betaHSD2, CYP11beta1, CYP11beta2, CYP17 and CYP19, was quantified by Q-RT-PCR. Production of the steroid hormones estradiol (E2), testosterone (T) and progesterone (P) was also evaluated. Antibiotics were shown to modulate gene expression and hormone production. Amoxicillin up-regulated the expression of CYP11beta2 and CYP19 by more than 2-fold and induced estradiol production up to almost 3-fold. Erythromycin significantly increased CYP11beta2 expression and the production of P and E2 by 3.5- and 2.4-fold, respectively, while production of T was significantly decreased. The beta-blocker salbutamol caused the greatest induction of CYP17, more than 13-fold, and significantly decreased E2 production. The binary mixture of cyproterone and salbutamol significantly down-regulated expression of CYP19, while a mixture of ethynylestradiol and trenbolone, increased E2 production 3.7-fold. Estradiol production was significantly affected by changes in concentrations of trenbolone, cyproterone, and ethynylestradiol. Exposures with individual pharmaceuticals showed the possible secondary effects that drugs may exert on steroid production. Results from binary mixture exposures suggested the possible type of interactions that may occur between drugs and the joint effects product of such interactions. Dose-response results indicated that although two chemicals may share a common mechanism of action the concentration effects observed may be significantly different.</p>
<p class="publication"><span class="title">Toxicology and applied pharmacology</span>&nbsp;<span class="year">2007</span>;<span class="volume">225</span>;<span class="number">2</span>;<span class="pagerange">142-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17822730" rel="external">17822730</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.taap.2007.07.013" rel="external">10.1016/j.taap.2007.07.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Toward a molecular catalogue of synapses.</h4>
<p class="authors"><span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Hinxton, Cambridge, Cambridgeshire, UK. <a href="mailto:%73%67%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">1906 was a landmark year in the history of the study of the nervous system, most notably for the first &#39;neuroscience&#39; Nobel prize given to the anatomists Ramon Y Cajal and Camillo Golgi. 1906 is less well known for another event, also of great significance for neuroscience, namely the publication of Charles Sherrington&#39;s book &#39;The Integrative Action of the Nervous system&#39;. It was Cajal and Golgi who debated the anatomical evidence for the synapse and it was Sherrington who laid its foundation in electrophysiological function. In tribute to these pioneers in synaptic biology, this article will address the issue of synapse diversity from the molecular point of view. In particular I will reflect upon efforts to obtain a complete molecular characterisation of the synapse and the unexpectedly high degree of molecular complexity found within it. A case will be made for developing approaches that can be used to generate a general catalogue of synapse types based on molecular markers, which should have wide application.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Brain research reviews</span>&nbsp;<span class="year">2007</span>;<span class="volume">55</span>;<span class="number">2</span>;<span class="pagerange">445-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17572504" rel="external">17572504</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.brainresrev.2007.05.003" rel="external">10.1016/j.brainresrev.2007.05.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ultra-high resolution array painting facilitates breakpoint sequencing.</h4>
<p class="authors"><span class="author">Gribble SM</span>, <span class="author">Kalaitzopoulos D</span>, <span class="author">Burford DC</span>, <span class="author">Prigmore E</span>, <span class="author">Selzer RR</span>, <span class="author">Ng BL</span>, <span class="author">Matthews NS</span>, <span class="author">Porter KM</span>, <span class="author">Curley R</span>, <span class="author">Lindsay SJ</span>, <span class="author">Baptista J</span>, <span class="author">Richmond TA</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Objective</strong>: To describe a considerably advanced method of array painting, which allows the rapid, ultra-high resolution mapping of translocation breakpoints such that rearrangement junction fragments can be amplified directly and sequenced.</p><p class="abstract"><strong>Method</strong>: Ultra-high resolution array painting involves the hybridisation of probes generated by the amplification of small numbers of flow-sorted derivative chromosomes to oligonucleotide arrays designed to tile breakpoint regions at extremely high resolution.</p><p class="abstract">How ultra-high resolution array painting of four balanced translocation cases rapidly and efficiently maps breakpoints to a point where junction fragments can be amplified easily and sequenced is demonstrated. With this new development, breakpoints can be mapped using just two array experiments: the first using whole-genome array painting to tiling resolution large insert clone arrays, the second using ultra-high-resolution oligonucleotide arrays targeted to the breakpoint regions. In this way, breakpoints can be mapped and then sequenced in a few weeks.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">44</span>;<span class="number">1</span>;<span class="pagerange">51-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16971479">16971479</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2597908">2597908</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2006.044909">10.1136/jmg.2006.044909</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Annotating noncoding RNA genes.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom. <a href="mailto:%73%61%6D.%67%72%69%66%66%69%74%68%73-%6A%6F%6E%65%73%40%6D%61%6E%63%68%65%73%74%65%72.%61%63.%75%6B">&#115;&#97;&#109;&#46;&#103;&#114;&#105;&#102;&#102;&#105;&#116;&#104;&#115;&#45;&#106;&#111;&#110;&#101;&#115;&#64;&#109;&#97;&#110;&#99;&#104;&#101;&#115;&#116;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Noncoding RNA genes produce a functional RNA product rather than a translated protein. More than 1500 homologs of known &quot;classical&quot; RNA genes can be annotated in the human genome sequence, and automatic homology-based methods predict up to 5000 related sequences. Methods to predict novel RNA genes on a whole-genome scale are immature at present, but their use hints at tens of thousands of such genes in the human genome. Messenger RNA-like transcripts with no protein-coding potential are routinely discovered by high-throughput transcriptome analyses. Meanwhile, various experimental studies have suggested that the vast majority of the human genome is transcribed, although the proportion of the detected RNAs that is functional remains unknown.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Annual review of genomics and human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">279-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17506659" rel="external">17506659</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev.genom.8.080706.092419" rel="external">10.1146/annurev.genom.8.080706.092419</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FAK is required for axonal sorting by Schwann cells.</h4>
<p class="authors"><span class="author">Grove M</span>, <span class="author">Komiyama NH</span>, <span class="author">Nave KA</span>, <span class="author">Grant SG</span>, <span class="author">Sherman DL</span> and <span class="author">Brophy PJ</span></p>
<p class="affiliation">Centre for Neuroscience Research, University of Edinburgh, Edinburgh EH9 1QH, Scotland, UK.</p><p class="abstract">Signaling by laminins and axonal neuregulin has been implicated in regulating axon sorting by myelin-forming Schwann cells. However, the signal transduction mechanisms are unknown. Focal adhesion kinase (FAK) has been linked to alpha6beta1 integrin and ErbB receptor signaling, and we show that myelination by Schwann cells lacking FAK is severely impaired. Mutant Schwann cells could interdigitate between axon bundles, indicating that FAK signaling was not required for process extension. However, Schwann cell FAK was required to stimulate cell proliferation, suggesting that amyelination was caused by insufficient Schwann cells. ErbB2 receptor and AKT were robustly phosphorylated in mutant Schwann cells, indicating that neuregulin signaling from axons was unimpaired. These findings demonstrate the vital relationship between axon defasciculation and Schwann cell number and show the importance of FAK in regulating cell proliferation in the developing nervous system.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of cell biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">176</span>;<span class="number">3</span>;<span class="pagerange">277-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17242067" rel="external">17242067</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2063954" rel="external">2063954</a>; DOI: <a href="http://dx.doi.org/10.1083%2Fjcb.200609021" rel="external">10.1083/jcb.200609021</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Re-annotation and re-analysis of the Campylobacter jejuni NCTC11168 genome sequence.</h4>
<p class="authors"><span class="author">Gundogdu O</span>, <span class="author">Bentley SD</span>, <span class="author">Holden MT</span>, <span class="author">Parkhill J</span>, <span class="author">Dorrell N</span> and <span class="author">Wren BW</span></p>
<p class="affiliation">Pathogen Molecular Department, London School of Hygiene &amp; Tropical Medicine, UK. <a href="mailto:%4F%7A%61%6E.%67%75%6E%64%6F%67%64%75%40%6C%73%68%74%6D.%61%63.%75%6B">&#79;&#122;&#97;&#110;&#46;&#103;&#117;&#110;&#100;&#111;&#103;&#100;&#117;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a> &lt;<a href="mailto:%4F%7A%61%6E.%67%75%6E%64%6F%67%64%75%40%6C%73%68%74%6D.%61%63.%75%6B&%67%74">&#79;&#122;&#97;&#110;&#46;&#103;&#117;&#110;&#100;&#111;&#103;&#100;&#117;&#64;&#108;&#115;&#104;&#116;&#109;&#46;&#97;&#99;&#46;&#117;&#107;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: Campylobacter jejuni is the leading bacterial cause of human gastroenteritis in the developed world. To improve our understanding of this important human pathogen, the C. jejuni NCTC11168 genome was sequenced and published in 2000. The original annotation was a milestone in Campylobacter research, but is outdated. We now describe the complete re-annotation and re-analysis of the C. jejuni NCTC11168 genome using current database information, novel tools and annotation techniques not used during the original annotation.</p><p class="abstract"><strong>Results</strong>: Re-annotation was carried out using sequence database searches such as FASTA, along with programs such as TMHMM for additional support. The re-annotation also utilises sequence data from additional Campylobacter strains and species not available during the original annotation. Re-annotation was accompanied by a full literature search that was incorporated into the updated EMBL file [EMBL: AL111168]. The C. jejuni NCTC11168 re-annotation reduced the total number of coding sequences from 1654 to 1643, of which 90.0% have additional information regarding the identification of new motifs and/or relevant literature. Re-annotation has led to 18.2% of coding sequence product functions being revised.</p><p class="abstract"><strong>Conclusions</strong>: Major updates were made to genes involved in the biosynthesis of important surface structures such as lipooligosaccharide, capsule and both O- and N-linked glycosylation. This re-annotation will be a key resource for Campylobacter research and will also provide a prototype for the re-annotation and re-interpretation of other bacterial genomes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">162</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17565669" rel="external">17565669</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1899501" rel="external">1899501</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-8-162" rel="external">10.1186/1471-2164-8-162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improving the power to detect differentially expressed genes in comparative microarray experiments by including information from self-self hybridizations.</h4>
<p class="authors"><span class="author">Gusnanto A</span>, <span class="author">Tom B</span>, <span class="author">Burns P</span>, <span class="author">Macaulay I</span>, <span class="author">Thijssen-Timmer DC</span>, <span class="author">Tijssen MR</span>, <span class="author">Langford C</span>, <span class="author">Watkins N</span>, <span class="author">Ouwehand W</span>, <span class="author">Berzuini C</span> and <span class="author">Dudbridge F</span></p>
<p class="affiliation">Medical Research Council-Biostatistics Unit, Institute of Public Health, Cambridge CB2 2SR, UK. <a href="mailto:%41%72%69%65%66.%47%75%73%6E%61%6E%74%6F%40%6D%72%63-%62%73%75.%63%61%6D.%61%63.%75%6B">&#65;&#114;&#105;&#101;&#102;&#46;&#71;&#117;&#115;&#110;&#97;&#110;&#116;&#111;&#64;&#109;&#114;&#99;&#45;&#98;&#115;&#117;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Our ability to detect differentially expressed genes in a microarray experiment can be hampered when the number of biological samples of interest is limited. In this situation, we propose the use of information from self-self hybridizations to acuminate our inference of differential expression. A unified modelling strategy is developed to allow better estimation of the error variance. This principle is similar to the use of a pooled variance estimate in the two-sample t-test. The results from real dataset examples suggest that we can detect more genes that are differentially expressed in the combined models. Our simulation study provides evidence that this method increases sensitivity compared to using the information from comparative hybridizations alone, given the same control for false discovery rate. The largest increase in sensitivity occurs when the amount of information in the comparative hybridization is limited.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105260799</span>, <span class="grant">MC_U105261167</span></p>
<p class="publication"><span class="title">Computational biology and chemistry</span>&nbsp;<span class="year">2007</span>;<span class="volume">31</span>;<span class="number">3</span>;<span class="pagerange">178-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17499550">17499550</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.compbiolchem.2007.03.005">10.1016/j.compbiolchem.2007.03.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Schistosoma mansoni genome: closing in on a final gene set.</h4>
<p class="authors"><span class="author">Haas BJ</span>, <span class="author">Berriman M</span>, <span class="author">Hirai H</span>, <span class="author">Cerqueira GG</span>, <span class="author">Loverde PT</span> and <span class="author">El-Sayed NM</span></p>
<p class="affiliation">The J.C. Venter Institute, Rockville, MD 20850, USA.</p><p class="abstract">The Schistosoma mansoni genome sequencing consortium has recently released the latest versions of the genome assembly as well as an automated preliminary gene structure annotation. The combined datasets constitute a vast resource for researchers to exploit in a variety of post-genomic studies with an emphasis of transcriptomic and proteomic tools. Here we present an innovative method used for combining diverse sources of evidence including ab initio gene predictions, protein and transcript sequence homologies, and cross-genome sequence homologies between S. mansoni and Schistosoma japonicum to define a comprehensive list of protein-coding genes.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI48828</span>; Wellcome Trust: <span class="grant">13557021</span></p>
<p class="publication"><span class="title">Experimental parasitology</span>&nbsp;<span class="year">2007</span>;<span class="volume">117</span>;<span class="number">3</span>;<span class="pagerange">225-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17643433">17643433</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.exppara.2007.06.005">10.1016/j.exppara.2007.06.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lessons learned from the initial sequencing of the pig genome: comparative analysis of an 8 Mb region of pig chromosome 17.</h4>
<p class="authors"><span class="author">Hart EA</span>, <span class="author">Caccamo M</span>, <span class="author">Harrow JL</span>, <span class="author">Humphray SJ</span>, <span class="author">Gilbert JG</span>, <span class="author">Trevanion S</span>, <span class="author">Hubbard T</span>, <span class="author">Rogers J</span> and <span class="author">Rothschild MF</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Tust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%65%61%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#97;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: We describe here the sequencing, annotation and comparative analysis of an 8 Mb region of pig chromosome 17, which provides a useful test region to assess coverage and quality for the pig genome sequencing project. We report our findings comparing the annotation of draft sequence assembled at different depths of coverage.</p><p class="abstract"><strong>Results</strong>: Within this region we annotated 71 loci, of which 53 are orthologous to human known coding genes. When compared to the syntenic regions in human (20q13.13-q13.33) and mouse (chromosome 2, 167.5 Mb-178.3 Mb), this region was found to be highly conserved with respect to gene order. The most notable difference between the three species is the presence of a large expansion of zinc finger coding genes and pseudogenes on mouse chromosome 2 between Edn3 and Phactr3 that is absent from pig and human. All of our annotation has been made publicly available in the Vertebrate Genome Annotation browser, VEGA. We assessed the impact of coverage on sequence assembly across this region and found, as expected, that increased sequence depth resulted in fewer, longer contigs. One-third of our annotated loci could not be fully re-aligned back to the low coverage version of the sequence, principally because the transcripts are fragmented over several contigs.</p><p class="abstract"><strong>Conclusion</strong>: We have demonstrated the considerable advantages of sequencing at increased read depths and discuss the implications that lower coverage sequence may have on subsequent comparative and functional studies, particularly those involving complex loci such as GNAS.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBE0116401</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">8</span>;<span class="pagerange">R168</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17705864">17705864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2374978">2374978</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-8-r168">10.1186/gb-2007-8-8-r168</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Specialist fungi, versatile genomes.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span> and <span class="author">Pain A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">5</span>;<span class="pagerange">332-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17436472">17436472</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1669">10.1038/nrmicro1669</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic analyses of seventeen Streptococcus pneumoniae strains: insights into the pneumococcal supragenome.</h4>
<p class="authors"><span class="author">Hiller NL</span>, <span class="author">Janto B</span>, <span class="author">Hogg JS</span>, <span class="author">Boissy R</span>, <span class="author">Yu S</span>, <span class="author">Powell E</span>, <span class="author">Keefe R</span>, <span class="author">Ehrlich NE</span>, <span class="author">Shen K</span>, <span class="author">Hayes J</span>, <span class="author">Barbadora K</span>, <span class="author">Klimke W</span>, <span class="author">Dernovoy D</span>, <span class="author">Tatusova T</span>, <span class="author">Parkhill J</span>, <span class="author">Bentley SD</span>, <span class="author">Post JC</span>, <span class="author">Ehrlich GD</span> and <span class="author">Hu FZ</span></p>
<p class="affiliation">Allegheny General Hospital, Allegheny-Singer Research Institute, Center for Genomic Sciences, Pittsburgh, PA 15212, USA.</p><p class="abstract">The distributed-genome hypothesis (DGH) states that pathogenic bacteria possess a supragenome that is much larger than the genome of any single bacterium and that these pathogens utilize genetic recombination and a large, noncore set of genes as a means of diversity generation. We sequenced the genomes of eight nasopharyngeal strains of Streptococcus pneumoniae isolated from pediatric patients with upper respiratory symptoms and performed quantitative genomic analyses among these and nine publicly available pneumococcal strains. Coding sequences from all strains were grouped into 3,170 orthologous gene clusters, of which 1,454 (46%) were conserved among all 17 strains. The majority of the gene clusters, 1,716 (54%), were not found in all strains. Genic differences per strain pair ranged from 35 to 629 orthologous clusters, with each strain&#39;s genome containing between 21 and 32% noncore genes. The distribution of the orthologous clusters per genome for the 17 strains was entered into the finite-supragenome model, which predicted that (i) the S. pneumoniae supragenome contains more than 5,000 orthologous clusters and (ii) 99% of the orthologous clusters ( approximately 3,000) that are represented in the S. pneumoniae population at frequencies of &gt;or=0.1 can be identified if 33 representative genomes are sequenced. These extensive genic diversity data support the DGH and provide a basis for understanding the great differences in clinical phenotype associated with various pneumococcal strains. When these findings are taken together with previous studies that demonstrated the presence of a supragenome for Streptococcus agalactiae and Haemophilus influenzae, it appears that the possession of a distributed genome is a common host interaction strategy.</p>
<p class="grants">Funded by: NIDCD NIH HHS: <span class="grant">DC02148</span>, <span class="grant">DC04173</span>, <span class="grant">DC05659</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">22</span>;<span class="pagerange">8186-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17675389">17675389</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2168654">2168654</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00690-07">10.1128/JB.00690-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome of acute rheumatic fever-associated serotype M5 Streptococcus pyogenes strain manfredo.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Scott A</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth T</span>, <span class="author">Churcher C</span>, <span class="author">Cronin A</span>, <span class="author">Dowd L</span>, <span class="author">Feltwell T</span>, <span class="author">Hamlin N</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Quail MA</span>, <span class="author">Price C</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Sharp S</span>, <span class="author">Skelton J</span>, <span class="author">Whitehead S</span>, <span class="author">Barrell BG</span>, <span class="author">Kehoe M</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%6D%68%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#104;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Comparisons of the 1.84-Mb genome of serotype M5 Streptococcus pyogenes strain Manfredo with previously sequenced genomes emphasized the role of prophages in diversification of S. pyogenes and the close relationship between strain Manfredo and MGAS8232, another acute rheumatic fever-associated strain.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">4</span>;<span class="pagerange">1473-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17012393">17012393</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1797351">1797351</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.01227-06">10.1128/JB.01227-06</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Multidrug-resistant Salmonella enterica serovar paratyphi A harbors IncHI1 plasmids similar to those found in serovar typhi.</h4>
<p class="authors"><span class="author">Holt KE</span>, <span class="author">Thomson NR</span>, <span class="author">Wain J</span>, <span class="author">Phan MD</span>, <span class="author">Nair S</span>, <span class="author">Hasan R</span>, <span class="author">Bhutta ZA</span>, <span class="author">Quail MA</span>, <span class="author">Norbertczak H</span>, <span class="author">Walker D</span>, <span class="author">Dougan G</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Salmonella enterica serovars Typhi and Paratyphi A cause systemic infections in humans which are referred to as enteric fever. Multidrug-resistant (MDR) serovar Typhi isolates emerged in the 1980s, and in recent years MDR serovar Paratyphi A infections have become established as a significant problem across Asia. MDR in serovar Typhi is almost invariably associated with IncHI1 plasmids, but the genetic basis of MDR in serovar Paratyphi A has remained predominantly undefined. The DNA sequence of an IncHI1 plasmid, pAKU_1, encoding MDR in a serovar Paratyphi A strain has been determined. Significantly, this plasmid shares a common IncHI1-associated DNA backbone with the serovar Typhi plasmid pHCM1 and an S. enterica serovar Typhimurium plasmid pR27. Plasmids pAKU_1 and pHCM1 share 14 antibiotic resistance genes encoded within similar mobile elements, which appear to form a 24-kb composite transposon that has transferred as a single unit into different positions into their IncHI1 backbones. Thus, these plasmids have acquired similar antibiotic resistance genes independently via the horizontal transfer of mobile DNA elements. Furthermore, two IncHI1 plasmids from a Vietnamese isolate of serovar Typhi were found to contain features of the backbone sequence of pAKU_1 rather than pHCM1, with the composite transposon inserted in the same location as in the pAKU_1 sequence. Our data show that these serovar Typhi and Paratyphi A IncHI1 plasmids share highly conserved core DNA and have acquired similar mobile elements encoding antibiotic resistance genes in past decades.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600805</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">11</span>;<span class="pagerange">4257-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17384186">17384186</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1913411">1913411</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00232-07">10.1128/JB.00232-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Generation of active protein phosphatase 2A is coupled to holoenzyme assembly.</h4>
<p class="authors"><span class="author">Hombauer H</span>, <span class="author">Weismann D</span>, <span class="author">Mudrak I</span>, <span class="author">Stanzel C</span>, <span class="author">Fellner T</span>, <span class="author">Lackner DH</span> and <span class="author">Ogris E</span></p>
<p class="affiliation">Department of Medical Biochemistry, Max F. Perutz Laboratories, Medical University of Vienna, Vienna, Austria.</p><p class="abstract">Protein phosphatase 2A (PP2A) is a prime example of the multisubunit architecture of protein serine/threonine phosphatases. Until substrate-specific PP2A holoenzymes assemble, a constitutively active, but nonspecific, catalytic C subunit would constitute a risk to the cell. While it has been assumed that the severe proliferation impairment of yeast lacking the structural PP2A subunit, TPD3, is due to the unrestricted activity of the C subunit, we recently obtained evidence for the existence of the C subunit in a low-activity conformation that requires the RRD/PTPA proteins for the switch into the active conformation. To study whether and how maturation of the C subunit is coupled with holoenzyme assembly, we analyzed PP2A biogenesis in yeast. Here we show that the generation of the catalytically active C subunit depends on the physical and functional interaction between RRD2 and the structural subunit, TPD3. The phenotype of the tpd3Delta strain is therefore caused by impaired, rather than increased, PP2A activity. TPD3/RRD2-dependent C subunit maturation is under the surveillance of the PP2A methylesterase, PPE1, which upon malfunction of PP2A biogenesis, prevents premature generation of the active C subunit and holoenzyme assembly by counteracting the untimely methylation of the C subunit. We propose a novel model of PP2A biogenesis in which a tightly controlled activation cascade protects cells from untargeted activity of the free catalytic PP2A subunit.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">e155</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17550305" rel="external">17550305</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1885835" rel="external">1885835</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0050155" rel="external">10.1371/journal.pbio.0050155</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2007.</h4>
<p class="authors"><span class="author">Hubbard TJ</span>, <span class="author">Aken BL</span>, <span class="author">Beal K</span>, <span class="author">Ballester B</span>, <span class="author">Caccamo M</span>, <span class="author">Chen Y</span>, <span class="author">Clarke L</span>, <span class="author">Coates G</span>, <span class="author">Cunningham F</span>, <span class="author">Cutts T</span>, <span class="author">Down T</span>, <span class="author">Dyer SC</span>, <span class="author">Fitzgerald S</span>, <span class="author">Fernandez-Banet J</span>, <span class="author">Graf S</span>, <span class="author">Haider S</span>, <span class="author">Hammond M</span>, <span class="author">Herrero J</span>, <span class="author">Holland R</span>, <span class="author">Howe K</span>, <span class="author">Howe K</span>, <span class="author">Johnson N</span>, <span class="author">Kahari A</span>, <span class="author">Keefe D</span>, <span class="author">Kokocinski F</span>, <span class="author">Kulesha E</span>, <span class="author">Lawson D</span>, <span class="author">Longden I</span>, <span class="author">Melsopp C</span>, <span class="author">Megy K</span>, <span class="author">Meidl P</span>, <span class="author">Ouverdin B</span>, <span class="author">Parker A</span>, <span class="author">Prlic A</span>, <span class="author">Rice S</span>, <span class="author">Rios D</span>, <span class="author">Schuster M</span>, <span class="author">Sealy I</span>, <span class="author">Severin J</span>, <span class="author">Slater G</span>, <span class="author">Smedley D</span>, <span class="author">Spudich G</span>, <span class="author">Trevanion S</span>, <span class="author">Vilella A</span>, <span class="author">Vogel J</span>, <span class="author">White S</span>, <span class="author">Wood M</span>, <span class="author">Cox T</span>, <span class="author">Curwen V</span>, <span class="author">Durbin R</span>, <span class="author">Fernandez-Suarez XM</span>, <span class="author">Flicek P</span>, <span class="author">Kasprzyk A</span>, <span class="author">Proctor G</span>, <span class="author">Searle S</span>, <span class="author">Smith J</span>, <span class="author">Ureta-Vidal A</span> and <span class="author">Birney E</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%74%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl (http://www.ensembl.org/) project provides a comprehensive and integrated source of annotation of chordate genome sequences. Over the past year the number of genomes available from Ensembl has increased from 15 to 33, with the addition of sites for the mammalian genomes of elephant, rabbit, armadillo, tenrec, platypus, pig, cat, bush baby, common shrew, microbat and european hedgehog; the fish genomes of stickleback and medaka and the second example of the genomes of the sea squirt (Ciona savignyi) and the mosquito (Aedes aegypti). Some of the major features added during the year include the first complete gene sets for genomes with low-sequence coverage, the introduction of new strain variation data and the introduction of new orthology/paralog annotations based on gene trees.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/E010768/1</span>, <span class="grant">BBS/B/13438</span>, <span class="grant">BBS/B/13446</span>, <span class="grant">BBS/B/13462</span>, <span class="grant">BBS/B/13470</span>; Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">Database issue</span>;<span class="pagerange">D610-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17148474">17148474</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1761443">1761443</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl996">10.1093/nar/gkl996</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of common genetic variation that modulates alternative splicing.</h4>
<p class="authors"><span class="author">Hull J</span>, <span class="author">Campino S</span>, <span class="author">Rowlands K</span>, <span class="author">Chan MS</span>, <span class="author">Copley RR</span>, <span class="author">Taylor MS</span>, <span class="author">Rockett K</span>, <span class="author">Elvidge G</span>, <span class="author">Keating B</span>, <span class="author">Knight J</span> and <span class="author">Kwiatkowski D</span></p>
<p class="affiliation">University Department of Paediatrics, John Radcliffe Hospital, Oxford, United Kingdom. <a href="mailto:%6A%65%72%65%6D%79.%68%75%6C%6C%40%70%61%65%64%69%61%74%72%69%63%73.%6F%78.%61%63.%75%6B">&#106;&#101;&#114;&#101;&#109;&#121;&#46;&#104;&#117;&#108;&#108;&#64;&#112;&#97;&#101;&#100;&#105;&#97;&#116;&#114;&#105;&#99;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Alternative splicing of genes is an efficient means of generating variation in protein function. Several disease states have been associated with rare genetic variants that affect splicing patterns. Conversely, splicing efficiency of some genes is known to vary between individuals without apparent ill effects. What is not clear is whether commonly observed phenotypic variation in splicing patterns, and hence potential variation in protein function, is to a significant extent determined by naturally occurring DNA sequence variation and in particular by single nucleotide polymorphisms (SNPs). In this study, we surveyed the splicing patterns of 250 exons in 22 individuals who had been previously genotyped by the International HapMap Project. We identified 70 simple cassette exon alternative splicing events in our experimental system; for six of these, we detected consistent differences in splicing pattern between individuals, with a highly significant association between splice phenotype and neighbouring SNPs. Remarkably, for five out of six of these events, the strongest correlation was found with the SNP closest to the intron-exon boundary, although the distance between these SNPs and the intron-exon boundary ranged from 2 bp to greater than 1,000 bp. Two of these SNPs were further investigated using a minigene splicing system, and in each case the SNPs were found to exert cis-acting effects on exon splicing efficiency in vitro. The functional consequences of these SNPs could not be predicted using bioinformatic algorithms. Our findings suggest that phenotypic variation in splicing patterns is determined by the presence of SNPs within flanking introns or exons. Effects on splicing may represent an important mechanism by which SNPs influence gene function.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">074318</span></p>
<p class="publication"><span class="title">PLoS genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">6</span>;<span class="pagerange">e99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17571926">17571926</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1904363">1904363</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pgen.0030099">10.1371/journal.pgen.0030099</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Completing the map of human genetic variation.</h4>
<p class="authors"><span class="author">Human Genome Structural Variation Working Group</span>, <span class="author">Eichler EE</span>, <span class="author">Nickerson DA</span>, <span class="author">Altshuler D</span>, <span class="author">Bowcock AM</span>, <span class="author">Brooks LD</span>, <span class="author">Carter NP</span>, <span class="author">Church DM</span>, <span class="author">Felsenfeld A</span>, <span class="author">Guyer M</span>, <span class="author">Lee C</span>, <span class="author">Lupski JR</span>, <span class="author">Mullikin JC</span>, <span class="author">Pritchard JK</span>, <span class="author">Sebat J</span>, <span class="author">Sherry ST</span>, <span class="author">Smith D</span>, <span class="author">Valle D</span> and <span class="author">Waterston RH</span></p>
<p class="affiliation">Department of Genome Sciences, University of Washington, Seattle, Washington 98195, USA.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">447</span>;<span class="number">7141</span>;<span class="pagerange">161-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17495918">17495918</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2685471">2685471</a>; DOI: <a href="http://dx.doi.org/10.1038%2F447161a">10.1038/447161a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high utility integrated map of the pig genome.</h4>
<p class="authors"><span class="author">Humphray SJ</span>, <span class="author">Scott CE</span>, <span class="author">Clark R</span>, <span class="author">Marron B</span>, <span class="author">Bender C</span>, <span class="author">Camm N</span>, <span class="author">Davis J</span>, <span class="author">Jenks A</span>, <span class="author">Noon A</span>, <span class="author">Patel M</span>, <span class="author">Sehra H</span>, <span class="author">Yang F</span>, <span class="author">Rogatcheva MB</span>, <span class="author">Milan D</span>, <span class="author">Chardon P</span>, <span class="author">Rohrer G</span>, <span class="author">Nonneman D</span>, <span class="author">de Jong P</span>, <span class="author">Meyers SN</span>, <span class="author">Archibald A</span>, <span class="author">Beever JE</span>, <span class="author">Schook LB</span> and <span class="author">Rogers J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA UK. <a href="mailto:%73%6A%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#106;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The domestic pig is being increasingly exploited as a system for modeling human disease. It also has substantial economic importance for meat-based protein production. Physical clone maps have underpinned large-scale genomic sequencing and enabled focused cloning efforts for many genomes. Comparative genetic maps indicate that there is more structural similarity between pig and human than, for example, mouse and human, and we have used this close relationship between human and pig as a way of facilitating map construction.</p><p class="abstract"><strong>Results</strong>: Here we report the construction of the most highly continuous bacterial artificial chromosome (BAC) map of any mammalian genome, for the pig (Sus scrofa domestica) genome. The map provides a template for the generation and assembly of high-quality anchored sequence across the genome. The physical map integrates previous landmark maps with restriction fingerprints and BAC end sequences from over 260,000 BACs derived from 4 BAC libraries and takes advantage of alignments to the human genome to improve the continuity and local ordering of the clone contigs. We estimate that over 98% of the euchromatin of the 18 pig autosomes and the X chromosome along with localized coverage on Y is represented in 172 contigs, with chromosome 13 (218 Mb) represented by a single contig. The map is accessible through pre-Ensembl, where links to marker and sequence data can be found.</p><p class="abstract"><strong>Conclusion</strong>: The map will enable immediate electronic positional cloning of genes, benefiting the pig research community and further facilitating use of the pig as an alternative animal model for human disease. The clone map and BAC end sequence data can also help to support the assembly of maps and genome sequences of other artiodactyls.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BBE0116401</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">7</span>;<span class="pagerange">R139</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17625002">17625002</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2323232">2323232</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-7-r139">10.1186/gb-2007-8-7-r139</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">G-quadruplexes in promoters throughout the human genome.</h4>
<p class="authors"><span class="author">Huppert JL</span> and <span class="author">Balasubramanian S</span></p>
<p class="affiliation">Cambridge University Chemical Laboratory, University of Cambridge, Lensfield Road, Cambridge CB2 1EW, UK.</p><p class="abstract">Certain G-rich DNA sequences readily form four-stranded structures called G-quadruplexes. These sequence motifs are located in telomeres as a repeated unit, and elsewhere in the genome, where their function is currently unknown. It has been proposed that G-quadruplexes may be directly involved in gene regulation at the level of transcription. In support of this hypothesis, we show that the promoter regions (1 kb upstream of the transcription start site TSS) of genes are significantly enriched in quadruplex motifs relative to the rest of the genome, with &gt;40% of human gene promoters containing one or more quadruplex motif. Furthermore, these promoter quadruplexes strongly associate with nuclease hypersensitive sites identified throughout the genome via biochemical measurement. Regions of the human genome that are both nuclease hypersensitive and within promoters show a remarkable (230-fold) enrichment of quadruplex elements, compared to the rest of the genome. These quadruplex motifs identified in promoter regions also show an interesting structural bias towards more stable forms. These observations support the proposal that promoter G-quadruplexes are directly involved in the regulation of gene expression.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A5709</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">2</span>;<span class="pagerange">406-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17169996" rel="external">17169996</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1802602" rel="external">1802602</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl1057" rel="external">10.1093/nar/gkl1057</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A second generation human haplotype map of over 3.1 million SNPs.</h4>
<p class="authors"><span class="author">International HapMap Consortium</span>, <span class="author">Frazer KA</span>, <span class="author">Ballinger DG</span>, <span class="author">Cox DR</span>, <span class="author">Hinds DA</span>, <span class="author">Stuve LL</span>, <span class="author">Gibbs RA</span>, <span class="author">Belmont JW</span>, <span class="author">Boudreau A</span>, <span class="author">Hardenbol P</span>, <span class="author">Leal SM</span>, <span class="author">Pasternak S</span>, <span class="author">Wheeler DA</span>, <span class="author">Willis TD</span>, <span class="author">Yu F</span>, <span class="author">Yang H</span>, <span class="author">Zeng C</span>, <span class="author">Gao Y</span>, <span class="author">Hu H</span>, <span class="author">Hu W</span>, <span class="author">Li C</span>, <span class="author">Lin W</span>, <span class="author">Liu S</span>, <span class="author">Pan H</span>, <span class="author">Tang X</span>, <span class="author">Wang J</span>, <span class="author">Wang W</span>, <span class="author">Yu J</span>, <span class="author">Zhang B</span>, <span class="author">Zhang Q</span>, <span class="author">Zhao H</span>, <span class="author">Zhao H</span>, <span class="author">Zhou J</span>, <span class="author">Gabriel SB</span>, <span class="author">Barry R</span>, <span class="author">Blumenstiel B</span>, <span class="author">Camargo A</span>, <span class="author">Defelice M</span>, <span class="author">Faggart M</span>, <span class="author">Goyette M</span>, <span class="author">Gupta S</span>, <span class="author">Moore J</span>, <span class="author">Nguyen H</span>, <span class="author">Onofrio RC</span>, <span class="author">Parkin M</span>, <span class="author">Roy J</span>, <span class="author">Stahl E</span>, <span class="author">Winchester E</span>, <span class="author">Ziaugra L</span>, <span class="author">Altshuler D</span>, <span class="author">Shen Y</span>, <span class="author">Yao Z</span>, <span class="author">Huang W</span>, <span class="author">Chu X</span>, <span class="author">He Y</span>, <span class="author">Jin L</span>, <span class="author">Liu Y</span>, <span class="author">Shen Y</span>, <span class="author">Sun W</span>, <span class="author">Wang H</span>, <span class="author">Wang Y</span>, <span class="author">Wang Y</span>, <span class="author">Xiong X</span>, <span class="author">Xu L</span>, <span class="author">Waye MM</span>, <span class="author">Tsui SK</span>, <span class="author">Xue H</span>, <span class="author">Wong JT</span>, <span class="author">Galver LM</span>, <span class="author">Fan JB</span>, <span class="author">Gunderson K</span>, <span class="author">Murray SS</span>, <span class="author">Oliphant AR</span>, <span class="author">Chee MS</span>, <span class="author">Montpetit A</span>, <span class="author">Chagnon F</span>, <span class="author">Ferretti V</span>, <span class="author">Leboeuf M</span>, <span class="author">Olivier JF</span>, <span class="author">Phillips MS</span>, <span class="author">Roumy S</span>, <span class="author">Sall&eacute;e C</span>, <span class="author">Verner A</span>, <span class="author">Hudson TJ</span>, <span class="author">Kwok PY</span>, <span class="author">Cai D</span>, <span class="author">Koboldt DC</span>, <span class="author">Miller RD</span>, <span class="author">Pawlikowska L</span>, <span class="author">Taillon-Miller P</span>, <span class="author">Xiao M</span>, <span class="author">Tsui LC</span>, <span class="author">Mak W</span>, <span class="author">Song YQ</span>, <span class="author">Tam PK</span>, <span class="author">Nakamura Y</span>, <span class="author">Kawaguchi T</span>, <span class="author">Kitamoto T</span>, <span class="author">Morizono T</span>, <span class="author">Nagashima A</span>, <span class="author">Ohnishi Y</span>, <span class="author">Sekine A</span>, <span class="author">Tanaka T</span>, <span class="author">Tsunoda T</span>, <span class="author">Deloukas P</span>, <span class="author">Bird CP</span>, <span class="author">Delgado M</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt S</span>, <span class="author">Morrison J</span>, <span class="author">Powell D</span>, <span class="author">Stranger BE</span>, <span class="author">Whittaker P</span>, <span class="author">Bentley DR</span>, <span class="author">Daly MJ</span>, <span class="author">de Bakker PI</span>, <span class="author">Barrett J</span>, <span class="author">Chretien YR</span>, <span class="author">Maller J</span>, <span class="author">McCarroll S</span>, <span class="author">Patterson N</span>, <span class="author">Pe&#39;er I</span>, <span class="author">Price A</span>, <span class="author">Purcell S</span>, <span class="author">Richter DJ</span>, <span class="author">Sabeti P</span>, <span class="author">Saxena R</span>, <span class="author">Schaffner SF</span>, <span class="author">Sham PC</span>, <span class="author">Varilly P</span>, <span class="author">Altshuler D</span>, <span class="author">Stein LD</span>, <span class="author">Krishnan L</span>, <span class="author">Smith AV</span>, <span class="author">Tello-Ruiz MK</span>, <span class="author">Thorisson GA</span>, <span class="author">Chakravarti A</span>, <span class="author">Chen PE</span>, <span class="author">Cutler DJ</span>, <span class="author">Kashuk CS</span>, <span class="author">Lin S</span>, <span class="author">Abecasis GR</span>, <span class="author">Guan W</span>, <span class="author">Li Y</span>, <span class="author">Munro HM</span>, <span class="author">Qin ZS</span>, <span class="author">Thomas DJ</span>, <span class="author">McVean G</span>, <span class="author">Auton A</span>, <span class="author">Bottolo L</span>, <span class="author">Cardin N</span>, <span class="author">Eyheramendy S</span>, <span class="author">Freeman C</span>, <span class="author">Marchini J</span>, <span class="author">Myers S</span>, <span class="author">Spencer C</span>, <span class="author">Stephens M</span>, <span class="author">Donnelly P</span>, <span class="author">Cardon LR</span>, <span class="author">Clarke G</span>, <span class="author">Evans DM</span>, <span class="author">Morris AP</span>, <span class="author">Weir BS</span>, <span class="author">Tsunoda T</span>, <span class="author">Mullikin JC</span>, <span class="author">Sherry ST</span>, <span class="author">Feolo M</span>, <span class="author">Skol A</span>, <span class="author">Zhang H</span>, <span class="author">Zeng C</span>, <span class="author">Zhao H</span>, <span class="author">Matsuda I</span>, <span class="author">Fukushima Y</span>, <span class="author">Macer DR</span>, <span class="author">Suda E</span>, <span class="author">Rotimi CN</span>, <span class="author">Adebamowo CA</span>, <span class="author">Ajayi I</span>, <span class="author">Aniagwu T</span>, <span class="author">Marshall PA</span>, <span class="author">Nkwodimmah C</span>, <span class="author">Royal CD</span>, <span class="author">Leppert MF</span>, <span class="author">Dixon M</span>, <span class="author">Peiffer A</span>, <span class="author">Qiu R</span>, <span class="author">Kent A</span>, <span class="author">Kato K</span>, <span class="author">Niikawa N</span>, <span class="author">Adewole IF</span>, <span class="author">Knoppers BM</span>, <span class="author">Foster MW</span>, <span class="author">Clayton EW</span>, <span class="author">Watkin J</span>, <span class="author">Gibbs RA</span>, <span class="author">Belmont JW</span>, <span class="author">Muzny D</span>, <span class="author">Nazareth L</span>, <span class="author">Sodergren E</span>, <span class="author">Weinstock GM</span>, <span class="author">Wheeler DA</span>, <span class="author">Yakub I</span>, <span class="author">Gabriel SB</span>, <span class="author">Onofrio RC</span>, <span class="author">Richter DJ</span>, <span class="author">Ziaugra L</span>, <span class="author">Birren BW</span>, <span class="author">Daly MJ</span>, <span class="author">Altshuler D</span>, <span class="author">Wilson RK</span>, <span class="author">Fulton LL</span>, <span class="author">Rogers J</span>, <span class="author">Burton J</span>, <span class="author">Carter NP</span>, <span class="author">Clee CM</span>, <span class="author">Griffiths M</span>, <span class="author">Jones MC</span>, <span class="author">McLay K</span>, <span class="author">Plumb RW</span>, <span class="author">Ross MT</span>, <span class="author">Sims SK</span>, <span class="author">Willey DL</span>, <span class="author">Chen Z</span>, <span class="author">Han H</span>, <span class="author">Kang L</span>, <span class="author">Godbout M</span>, <span class="author">Wallenburg JC</span>, <span class="author">L&#39;Archev&ecirc;que P</span>, <span class="author">Bellemare G</span>, <span class="author">Saeki K</span>, <span class="author">Wang H</span>, <span class="author">An D</span>, <span class="author">Fu H</span>, <span class="author">Li Q</span>, <span class="author">Wang Z</span>, <span class="author">Wang R</span>, <span class="author">Holden AL</span>, <span class="author">Brooks LD</span>, <span class="author">McEwen JE</span>, <span class="author">Guyer MS</span>, <span class="author">Wang VO</span>, <span class="author">Peterson JL</span>, <span class="author">Shi M</span>, <span class="author">Spiegel J</span>, <span class="author">Sung LM</span>, <span class="author">Zacharia LF</span>, <span class="author">Collins FS</span>, <span class="author">Kennedy K</span>, <span class="author">Jamieson R</span> and <span class="author">Stewart J</span></p>
<p class="affiliation">The Scripps Research Institute, 10550 North Torrey Pines Road MEM275, La Jolla, California 92037, USA.</p><p class="abstract">We describe the Phase II HapMap, which characterizes over 3.1 million human single nucleotide polymorphisms (SNPs) genotyped in 270 individuals from four geographically diverse populations and includes 25-35% of common SNP variation in the populations surveyed. The map is estimated to capture untyped common variation with an average maximum r2 of between 0.9 and 0.96 depending on population. We demonstrate that the current generation of commercial genome-wide genotyping products captures common Phase II SNPs with an average maximum r2 of up to 0.8 in African and up to 0.95 in non-African populations, and that potential gains in power in association studies can be obtained through imputation. These data also reveal novel aspects of the structure of linkage disequilibrium. We show that 10-30% of pairs of individuals within a population share at least one region of extended genetic identity arising from recent ancestry and that up to 1% of all common variants are untaggable, primarily because they lie within recombination hotspots. We show that recombination rates vary systematically around genes and between genes of different function. Finally, we demonstrate increased differentiation at non-synonymous, compared to synonymous, SNPs, resulting from systematic differences in the strength or efficacy of natural selection between populations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077011</span>, <span class="grant">077046</span>, <span class="grant">081682</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">449</span>;<span class="number">7164</span>;<span class="pagerange">851-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17943122">17943122</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2689609">2689609</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06258">10.1038/nature06258</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary consequences of a large duplication event in Trypanosoma brucei: chromosomes 4 and 8 are partial duplicons.</h4>
<p class="authors"><span class="author">Jackson AP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%61%6A%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#106;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Gene order along the genome sequence of the human parasite Trypanosoma brucei provides evidence for a 0.5 Mb duplication, comprising the 3&#39; regions of chromosomes 4 and 8. Here, the principal aim was to examine the contribution made by this duplication event to the T. brucei genome sequence, emphasising the consequences for gene content and the evolutionary change subsequently experienced by paralogous gene copies. The duplicated region may be browsed online at http://www.genedb.org/genedb/tryp/48dup_image.jsp</p><p class="abstract"><strong>Results</strong>: Comparisons of trypanosomatid genomes demonstrated widespread gene loss from each duplicon, but also showed that 47% of duplicated genes were retained on both chromosomes as paralogous loci. Secreted and surface-expressed genes were over-represented among retained paralogs, reflecting a bias towards important factors at the host-parasite interface, and consistent with a dosage-balance hypothesis. Genetic divergence in both coding and regulatory regions of retained paralogs was bimodal, with a deficit in moderately divergent paralogs; in particular, non-coding sequences were either conserved or entirely remodelled. The conserved paralogs included examples of remarkable sequence conservation, but also considerable divergence of both coding and regulatory regions. Sequence divergence typically displayed strong negative selection; but several features, such as asymmetric evolutionary rates, positively-selected codons and other non-neutral substitutions, suggested that divergence of some paralogs was driven by functional change. The absence of orthologs to retained paralogs in T. congolense indicated that the duplication event was specific to T. brucei.</p><p class="abstract"><strong>Conclusion</strong>: The duplication of this chromosomal region doubled the dosage of many genes. Rather than creating &#39;more of the same&#39;, these results show that paralogs were structurally modified according to various evolutionary trajectories. The retention of paralogs, and subsequent elaboration of both their primary structures and regulatory regions, strongly suggests that this duplication was a seminal development, stimulating functional innovation and fundamentally altering the genetic repertoire of T. brucei relative to other trypanosomatids.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">432</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18036214" rel="external">18036214</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2212663" rel="external">2212663</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-8-432" rel="external">10.1186/1471-2164-8-432</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Origins of amino acid transporter loci in trypanosomatid parasites.</h4>
<p class="authors"><span class="author">Jackson AP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%61%6A%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#106;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Large amino acid transporter gene families were identified from the genome sequences of three parasitic protists, Trypanosoma brucei, Trypanosoma cruzi and Leishmania major. These genes encode molecular sensors of the external host environment for trypanosomatid cells and are crucial to modulation of gene expression as the parasite passes through different life stages. This study provides a comprehensive phylogenetic account of the origins of these genes, redefining each locus according to a positional criterion, through the integration of phyletic identity with comparative gene order information.</p><p class="abstract"><strong>Results</strong>: Each locus was individually specified by its surrounding gene order and associated with homologs showing the same position (&#39;homoeologs&#39;) in other species, where available. Bayesian and maximum likelihood phylogenies were in general agreement on systematic relationships and confirmed several &#39;orthology sets&#39; of genes retained since divergence from the common ancestor. Reconciliation analysis quantified the scale of duplication and gene loss, as well as identifying further apparent orthology sets, which lacked conservation of genomic position. These instances suggested substantial genomic restructuring or transposition. Other analyses identified clear instances of evolutionary rate changes post-duplication, the effects of concerted evolution within tandem gene arrays and gene conversion events between syntenic loci.</p><p class="abstract"><strong>Conclusion</strong>: Despite their importance to cell function and parasite development, the repertoires of AAT loci in trypanosomatid parasites are relatively fluid in both complement and gene dosage. Some loci are ubiquitous and, after an ancient origin through transposition, originated through descent from the ancestral trypanosomatid. However, reconciliation analysis demonstrated that unilateral expansions of gene number through tandem gene duplication, transposition of gene duplicates to otherwise well conserved genomic positions, and differential patterns of gene loss have produced largely customised and idiosyncratic AAT repertoires in all three species. Not least in T. brucei, which seems to have retained fewer ancestral loci and has acquired novel loci through a complex mix of tandem and transpositive duplication.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="pagerange">26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17319943" rel="external">17319943</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1810246" rel="external">1810246</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-7-26" rel="external">10.1186/1471-2148-7-26</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tandem gene arrays in Trypanosoma brucei: comparative phylogenomic analysis of duplicate sequence variation.</h4>
<p class="authors"><span class="author">Jackson AP</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK. <a href="mailto:%61%6A%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#106;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The genome sequence of the protistan parasite Trypanosoma brucei contains many tandem gene arrays. Gene duplicates are created through tandem duplication and are expressed through polycistronic transcription, suggesting that the primary purpose of long, tandem arrays is to increase gene dosage in an environment where individual gene promoters are absent. This report presents the first account of the tandem gene arrays in the T. brucei genome, employing several related genome sequences to establish how variation is created and removed.</p><p class="abstract"><strong>Results</strong>: A systematic survey of tandem gene arrays showed that substantial sequence variation existed across the genome; variation from different regions of an array often produced inconsistent phylogenetic affinities. Phylogenetic relationships of gene duplicates were consistent with concerted evolution being a widespread homogenising force. However, tandem duplicates were not usually identical; therefore, any homogenising effect was coincident with divergence among duplicates. Allelic gene conversion was detected using various criteria and was apparently able to both remove and introduce sequence variation. Tandem arrays containing structural heterogeneity demonstrated how sequence homogenisation and differentiation can occur within a single locus.</p><p class="abstract"><strong>Conclusion</strong>: The use of multiple genome sequences in a comparative analysis of tandem gene arrays identified substantial sequence variation among gene duplicates. The distribution of sequence variation is determined by a dynamic balance of conservative and innovative evolutionary forces. Gene trees from various species showed that intraspecific duplicates evolve in concert, perhaps through frequent gene conversion, although this does not prevent sequence divergence, especially where structural heterogeneity physically separates a duplicate from its neighbours. In describing dynamics of sequence variation that have consequences beyond gene dosage, this survey provides a basis for uncovering the hidden functionality within tandem gene arrays in trypanosomatids.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC evolutionary biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="pagerange">54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17408475" rel="external">17408475</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1855330" rel="external">1855330</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2148-7-54" rel="external">10.1186/1471-2148-7-54</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome variation and evolution of the malaria parasite Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Jeffares DC</span>, <span class="author">Pain A</span>, <span class="author">Berry A</span>, <span class="author">Cox AV</span>, <span class="author">Stalker J</span>, <span class="author">Ingle CE</span>, <span class="author">Thomas A</span>, <span class="author">Quail MA</span>, <span class="author">Siebenthall K</span>, <span class="author">Uhlemann AC</span>, <span class="author">Kyes S</span>, <span class="author">Krishna S</span>, <span class="author">Newbold C</span>, <span class="author">Dermitzakis ET</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Informatics Division, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, CB10 1SA Hinxton, UK.</p><p class="abstract">Infections with the malaria parasite Plasmodium falciparum result in more than 1 million deaths each year worldwide. Deciphering the evolutionary history and genetic variation of P. falciparum is critical for understanding the evolution of drug resistance, identifying potential vaccine candidates and appreciating the effect of parasite variation on prevalence and severity of malaria in humans. Most studies of natural variation in P. falciparum have been either in depth over small genomic regions (up to the size of a small chromosome) or genome wide but only at low resolution. In an effort to complement these studies with genome-wide data, we undertook shotgun sequencing of a Ghanaian clinical isolate (with fivefold coverage), the IT laboratory isolate (with onefold coverage) and the chimpanzee parasite P. reichenowi (with twofold coverage). We compared these sequences with the fully sequenced P. falciparum 3D7 isolate genome. We describe the most salient features of P. falciparum polymorphism and adaptive evolution with relation to gene function, transcript and protein expression and cellular localization. This analysis uncovers the primary evolutionary changes that have occurred since the P. falciparum-P. reichenowi speciation and changes that are occurring within P. falciparum.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077046</span>, <span class="grant">079643</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">1</span>;<span class="pagerange">120-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17159978">17159978</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2663918">2663918</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1931">10.1038/ng1931</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In silico functional and structural characterisation of ferlin proteins by mapping disease-causing mutations and evolutionary information onto three-dimensional models of their C2 domains.</h4>
<p class="authors"><span class="author">Jim&eacute;nez JL</span> and <span class="author">Bashir R</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6A%5F%6C%5F%6A%69%6D%65%6E%65%7A%37%31%40%79%61%68%6F%6F.%65%73">&#106;&#95;&#108;&#95;&#106;&#105;&#109;&#101;&#110;&#101;&#122;&#55;&#49;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#101;&#115;</a></p><p class="abstract">Ferlins are C2 domain proteins involved in membrane fusion events, including membrane repair and synaptic exocytosis, and their deficiency can result in muscular dystrophy and deafness. We have undertaken a structural study of their C2 domains by sequence comparison and homology modelling to understand the function of these poorly characterised proteins and to predict the molecular impact of disease-causing mutations. We observe that non-conservative mutations affecting buried residues tend to result in detrimental phenotypes, likely because of decreased protein stability, whereas most variants with replacements in surface residues do not. The few cases of exposed residues altered in variants known to cause diseases are found in conserved areas of functional importance, including essential calcium-binding regions, as deduced by analogy to other characterised C2 domains. Furthermore, we report distinct features of some C2 domains in the two known ferlin subfamilies that correlates with the presence or absence of the DysF domains. Taken altogether, our results highlight potential targets for further experimental analyses to understand the function of ferlin proteins. We believe our modelling data will aid the diagnosis of diseases associated with ferlin mutations and the development of therapeutic strategies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of the neurological sciences</span>&nbsp;<span class="year">2007</span>;<span class="volume">260</span>;<span class="number">1-2</span>;<span class="pagerange">114-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17512949">17512949</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jns.2007.04.016">10.1016/j.jns.2007.04.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A systematic comparative and structural analysis of protein phosphorylation sites based on the mtcPTM database.</h4>
<p class="authors"><span class="author">Jim&eacute;nez JL</span>, <span class="author">Hegemann B</span>, <span class="author">Hutchins JR</span>, <span class="author">Peters JM</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6A%5F%6C%5F%6A%69%6D%65%6E%65%7A%37%31%40%79%61%68%6F%6F.%65%73">&#106;&#95;&#108;&#95;&#106;&#105;&#109;&#101;&#110;&#101;&#122;&#55;&#49;&#64;&#121;&#97;&#104;&#111;&#111;&#46;&#101;&#115;</a></p><p class="abstract">mtcPTM is an online repository of human and mouse phosphosites in which data are hierarchically organized to preserve biologically relevant experimental information, thus allowing straightforward comparisons of phosphorylation patterns found under different conditions. The database also contains the largest available collection of atomic models of phosphorylatable proteins. Detailed analysis of this structural dataset reveals that phosphorylation sites are found in a heterogeneous range of structural and sequence contexts. mtcPTM is available on the web http://www.mitocheck.org/cgi-bin/mtcPTM/search.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">R90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17521420">17521420</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1929158">1929158</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-5-r90">10.1186/gb-2007-8-5-r90</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Epigenetic identification of ADAMTS18 as a novel 16q23.1 tumor suppressor frequently silenced in esophageal, nasopharyngeal and multiple other carcinomas.</h4>
<p class="authors"><span class="author">Jin H</span>, <span class="author">Wang X</span>, <span class="author">Ying J</span>, <span class="author">Wong AH</span>, <span class="author">Li H</span>, <span class="author">Lee KY</span>, <span class="author">Srivastava G</span>, <span class="author">Chan AT</span>, <span class="author">Yeo W</span>, <span class="author">Ma BB</span>, <span class="author">Putti TC</span>, <span class="author">Lung ML</span>, <span class="author">Shen ZY</span>, <span class="author">Xu LY</span>, <span class="author">Langford C</span> and <span class="author">Tao Q</span></p>
<p class="affiliation">Cancer Epigenetics Laboratory, Sir YK Pao Center for Cancer, Department of Clinical Oncology, Hong Kong Cancer Institute, Chinese University of Hong Kong, China.</p><p class="abstract">Tumor suppressor genes (TSGs) often locate at chromosomal regions with frequent deletions in tumors. Loss of 16q23 occurs frequently in multiple tumors, indicating the presence of critical TSGs at this locus, such as the well-studied WWOX. Herein, we found that ADAMTS18, located next to WWOX, was significantly downregulated in multiple carcinoma cell lines. No deletion of ADAMTS18 was detected with multiplex differential DNA-PCR or high-resolution 1-Mb array-based comparative genomic hybridization (CGH) analysis. Instead, methylation of the ADAMTS18 promoter CpG Island was frequently detected with methylation-specific PCR and bisulfite genome sequencing in multiple carcinoma cell lines and primary carcinomas, but not in any nontumor cell line and normal epithelial tissue. Both pharmacological and genetic demethylation dramatically induced the ADAMTS18 expression, indicating that CpG methylation directly contributes to the tumor-specific silencing of ADAMTS18. Ectopic ADAMTS18 expression led to significant inhibition of both anchorage-dependent and -independent growth of carcinoma cells lacking the expression. Thus, through functional epigenetics, we identified ADAMTS18 as a novel functional tumor suppressor, being frequently inactivated epigenetically in multiple carcinomas.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">53</span>;<span class="pagerange">7490-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17546048" rel="external">17546048</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2875853" rel="external">2875853</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1210559" rel="external">10.1038/sj.onc.1210559</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Immunohistochemical characterization of cytokeratins in the abnormal corneal endothelium of posterior polymorphous corneal dystrophy patients.</h4>
<p class="authors"><span class="author">Jirsova K</span>, <span class="author">Merjava S</span>, <span class="author">Martincova R</span>, <span class="author">Gwilliam R</span>, <span class="author">Ebenezer ND</span>, <span class="author">Liskova P</span> and <span class="author">Filipec M</span></p>
<p class="affiliation">Ocular Tissue Bank, General Teaching Hospital and Charles University, U Nemocnice 2, Prague 128 08, Czech Republic. <a href="mailto:%6B%61%74%65%72%69%6E%61.%6A%69%72%73%6F%76%61%40%76%66%6E.%63%7A">&#107;&#97;&#116;&#101;&#114;&#105;&#110;&#97;&#46;&#106;&#105;&#114;&#115;&#111;&#118;&#97;&#64;&#118;&#102;&#110;&#46;&#99;&#122;</a></p><p class="abstract">Posterior polymorphous corneal dystrophy (PPCD) is a hereditary bilateral disorder affecting Descemet&#39;s membrane and the endothelium. The aim of the present study was to determine the spectrum of cytokeratin (CK) expression in cells on the posterior surface of the cornea in PPCD patients. Ten corneal buttons and one specimen of the trabecular meshwork (TM) from PPCD patients who underwent graft or glaucoma surgery were used, as well as six corneal buttons and two TM specimens obtained from healthy donors as controls. Cryosections were fixed and indirect immunofluorescent staining was performed using antibodies directed against a wide spectrum of cytokeratins (CKs). The number of positive cells and the intensity of the staining were assessed using fluorescent microscopy. All 10 PPCD corneal specimens had areas of endothelium displaying typical endothelial morphology as well as areas consisting of layers two to six cells thick with both flat endothelial-like cells and polygonal cells with round nuclei and a large cytoplasm. Both of these morphologically distinct cell types showed strong immunostaining for CK7, CK19, CK8 and CK18, while weaker positive signals were observed for CK1, CK3/12, CK4, CK5/6, CK10, CK10/13, CK14, CK16 and CK17. PPCD endothelium was completely negative for CK2e, CK9, CK15, and CK20. Focal positivity was detected in PPCD TM for CK4, CK7 and CK19. CK8 and CK18 were the only CKs expressed in control endothelium. PPCD and control epithelium displayed similar staining patterns. The distinct positivity for CK3/12, CK4, CK5/6, CK10/13, CK14, CK16 and CK17 was observed in aberrant PPCD endothelium for the first time. We demonstrate that the abnormal endothelium of PPCD patients expresses a mixture of CKs, with CK7 and CK19 predominating. In terms of CK composition, the aberrant PPCD endothelium shares features of both simple and squamous stratified epithelium with a proliferative capacity. The wide spectrum of CK expression is most probably not indicative of the transformation of endothelial cells to a distinct epithelial phenotype, but more likely reflects the modified differentiation of metaplastic epithelium.</p>
<p class="publication"><span class="title">Experimental eye research</span>&nbsp;<span class="year">2007</span>;<span class="volume">84</span>;<span class="number">4</span>;<span class="pagerange">680-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17289024">17289024</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.exer.2006.12.006">10.1016/j.exer.2006.12.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structural variation on the short arm of the human Y chromosome: recurrent multigene deletions encompassing Amelogenin Y.</h4>
<p class="authors"><span class="author">Jobling MA</span>, <span class="author">Lo IC</span>, <span class="author">Turner DJ</span>, <span class="author">Bowden GR</span>, <span class="author">Lee AC</span>, <span class="author">Xue Y</span>, <span class="author">Carvalho-Silva D</span>, <span class="author">Hurles ME</span>, <span class="author">Adams SM</span>, <span class="author">Chang YM</span>, <span class="author">Kraaijenbrink T</span>, <span class="author">Henke J</span>, <span class="author">Guanti G</span>, <span class="author">McKeown B</span>, <span class="author">van Oorschot RA</span>, <span class="author">Mitchell RJ</span>, <span class="author">de Knijff P</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Parkin EJ</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, University Road, Leicester LE1 7RH, UK. <a href="mailto:%6D%61%6A%34%40%6C%65.%61%63.%75%6B">&#109;&#97;&#106;&#52;&#64;&#108;&#101;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Structural polymorphism is increasingly recognized as a major form of human genome variation, and is particularly prevalent on the Y chromosome. Assay of the Amelogenin Y gene (AMELY) on Yp is widely used in DNA-based sex testing, and sometimes reveals males who have interstitial deletions. In a collection of 45 deletion males from 12 populations, we used a combination of sequence-tagged site mapping, and binary-marker and Y-short tandem repeat haplotyping to understand the structural basis of this variation. Of the 45 deletion males, 41 carry indistinguishable deletions, 3.0-3.8 Mb in size. Breakpoint mapping strongly implicates a mechanism of non-allelic homologous recombination between the proximal major array of TSPY gene-containing repeats, and a single distal copy of TSPY; this is supported by the estimation of TSPY copy number in deleted and non-deleted males. The remaining four males carry three distinct non-recurrent deletions (2.5-4.0 Mb), which may be due to non-homologous mechanisms. Haplotyping shows that TSPY-mediated deletions have arisen seven times independently in the sample. One instance, represented by 30 chromosomes mostly of Indian origin within haplogroup J2e1*/M241, has a time-to-most-recent-common-ancestor of approximately 7700+/-1300 years. In addition to AMELY, deletion males all lack the genes PRKY and TBL1Y, and the rarer deletion classes also lack PCDH11Y. The persistence and expansion of deletion lineages, together with direct phenotypic evidence, suggests that absence of these genes has no major deleterious effects.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">16</span>;<span class="number">3</span>;<span class="pagerange">307-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17189292" rel="external">17189292</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2590852" rel="external">2590852</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddl465" rel="external">10.1093/hmg/ddl465</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The nicotinic acetylcholine receptor gene family of the nematode Caenorhabditis elegans: an update on nomenclature.</h4>
<p class="authors"><span class="author">Jones AK</span>, <span class="author">Davis P</span>, <span class="author">Hodgkin J</span> and <span class="author">Sattelle DB</span></p>
<p class="affiliation">Department of Physiology, Anatomy and Genetics, University of Oxford, South Parks Road, Oxford,UK. <a href="mailto:%61%6E%64%72%65%77.%6A%6F%6E%65%73%40%64%70%61%67.%6F%78.%61%63.%75%6B">&#97;&#110;&#100;&#114;&#101;&#119;&#46;&#106;&#111;&#110;&#101;&#115;&#64;&#100;&#112;&#97;&#103;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The simple nematode, Caenorhabditis elegans, possesses the most extensive known gene family of nicotinic acetylcholine receptor (nAChR)-like subunits. Whilst all show greatest similarity with nAChR subunits of both invertebrates and vertebrates, phylogenetic analysis suggests that just over half of these (32) may represent other members of the cys-loop ligand-gated ion channel superfamily. We have introduced a novel nomenclature system for these &quot;Orphan&quot; subunits, designating them as lgc genes (ligand-gated ion channels of the cys-loop superfamily), which can also be applied in future to unnamed and uncharacterised members of the cys-loop ligand-gated ion channel superfamily. We present here the resulting updated version of the C. elegans nAChR gene family and related ligand-gated ion channel genes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0701197(84652)</span></p>
<p class="publication"><span class="title">Invertebrate neuroscience : IN</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">129-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17503100" rel="external">17503100</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2972647" rel="external">2972647</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10158-007-0049-z" rel="external">10.1007/s10158-007-0049-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping the platelet profile for functional genomic studies and demonstration of the effect size of the GP6 locus.</h4>
<p class="authors"><span class="author">Jones CI</span>, <span class="author">Garner SF</span>, <span class="author">Angenent W</span>, <span class="author">Bernard A</span>, <span class="author">Berzuini C</span>, <span class="author">Burns P</span>, <span class="author">Farndale RW</span>, <span class="author">Hogwood J</span>, <span class="author">Rankin A</span>, <span class="author">Stephens JC</span>, <span class="author">Tom BD</span>, <span class="author">Walton J</span>, <span class="author">Dudbridge F</span>, <span class="author">Ouwehand WH</span>, <span class="author">Goodall AH</span> and <span class="author">Bloodomics Consortium</span></p>
<p class="affiliation">Department of Cardiovascular Sciences, University of Leicester, Leicester, UK.</p><p class="abstract"><strong>Background</strong>: Evidence suggests the wide variation in platelet response within the population is genetically controlled. Unraveling the complex relationship between sequence variation and platelet phenotype requires accurate and reproducible measurement of platelet response.</p><p class="abstract"><strong>Objective</strong>: To develop a methodology suitable for measuring signaling pathway-specific platelet phenotype, to use this to measure platelet response in a large cohort, and to demonstrate the effect size of sequence variation in a relevant model gene.</p><p class="abstract"><strong>Methods</strong>: Three established platelet assays were evaluated: mobilization of [Ca(2+)](i), aggregometry and flow cytometry, each in response to adenosine 5&#39;-diphosphate (ADP) or the glycoprotein (GP) VI-specific crosslinked collagen-related peptide (CRP). Flow cytometric measurement of fibrinogen binding and P-selectin expression in response to a single, intermediate dose of each agonist gave the best combination of reproducibility and inter-individual variability and was used to measure the platelet response in 506 healthy volunteers. Pathway specificity was ensured by blocking the main subsidiary signaling pathways.</p><p class="abstract"><strong>Results</strong>: Individuals were identified who were hypo- or hyper-responders for both pathways, or who had differential responses to the two agonists, or between outcomes. 89 individuals, retested three months later using the same methodology, showed high concordance between the two visits in all four assays (r(2) = 0.872, 0.868, 0.766 and 0.549); all subjects retaining their phenotype at recall. The effect of sequence variation at the GP6 locus accounted for approximately 35% of the variation in the CRP-XL response.</p><p class="abstract"><strong>Conclusion</strong>: Genotyping-phenotype association studies in a well-characterized, large cohort provides a powerful strategy to measure the effect of sequence variation in genes regulating the platelet response.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500707</span>, <span class="grant">MC_U105260799</span>, <span class="grant">MC_U105261167</span></p>
<p class="publication"><span class="title">Journal of thrombosis and haemostasis : JTH</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">8</span>;<span class="pagerange">1756-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17663743">17663743</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1538-7836.2007.02632.x">10.1111/j.1538-7836.2007.02632.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Advances in the genomics of ticks and tick-borne pathogens.</h4>
<p class="authors"><span class="author">Jongejan F</span>, <span class="author">Nene V</span>, <span class="author">de la Fuente J</span>, <span class="author">Pain A</span> and <span class="author">Willadsen P</span></p><p class="abstract">Ticks and the diseases for which they are vectors engage in complex interactions with their mammalian hosts. These interactions involve the developmental processes of tick and pathogen, and interplay between the defensive responses and counter responses of host, tick and pathogen. Understanding these interactions has long been an intractable problem, but progress is now being made thanks to the flood of genomic information on host, tick and pathogen, and the attendant, novel experimental tools that have been generated. Each advance reveals new levels of complexity, but there are encouraging signs that genomics is leading to novel means of parasite control.</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">9</span>;<span class="pagerange">391-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17656151" rel="external">17656151</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2007.07.004" rel="external">10.1016/j.pt.2007.07.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence and functional analyses of Haemophilus spp. genomic islands.</h4>
<p class="authors"><span class="author">Juhas M</span>, <span class="author">Power PM</span>, <span class="author">Harding RM</span>, <span class="author">Ferguson DJ</span>, <span class="author">Dimopoulou ID</span>, <span class="author">Elamin AR</span>, <span class="author">Mohd-Zain Z</span>, <span class="author">Hood DW</span>, <span class="author">Adegbola R</span>, <span class="author">Erwin A</span>, <span class="author">Smith A</span>, <span class="author">Munson RS</span>, <span class="author">Harrison A</span>, <span class="author">Mansfield L</span>, <span class="author">Bentley S</span> and <span class="author">Crook DW</span></p>
<p class="affiliation">Clinical Microbiology and Infectious Diseases, NDCLS, University of Oxford, Headley Way, Oxford OX3 9DU, UK. <a href="mailto:%6D%61%72%69%6F.%6A%75%68%61%73%40%6E%64%6C%63%73.%6F%78.%61%63.%75%6B">&#109;&#97;&#114;&#105;&#111;&#46;&#106;&#117;&#104;&#97;&#115;&#64;&#110;&#100;&#108;&#99;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: A major part of horizontal gene transfer that contributes to the diversification and adaptation of bacteria is facilitated by genomic islands. The evolution of these islands is poorly understood. Some progress was made with the identification of a set of phylogenetically related genomic islands among the Proteobacteria, recognized from the investigation of the evolutionary origins of a Haemophilus influenzae antibiotic resistance island, namely ICEHin1056. More clarity comes from this comparative analysis of seven complete sequences of the ICEHin1056 genomic island subfamily.</p><p class="abstract"><strong>Results</strong>: These genomic islands have core and accessory genes in approximately equal proportion, with none demonstrating recent acquisition from other islands. The number of variable sites within core genes is similar to that found in the host bacteria. Furthermore, the GC content of the core genes is similar to that of the host bacteria (38% to 40%). Most of the core gene content is formed by the syntenic type IV secretion system dependent conjugative module and replicative module. GC content and lack of variable sites indicate that the antibiotic resistance genes were acquired relatively recently. An analysis of conjugation efficiency and antibiotic susceptibility demonstrates that phenotypic expression of genomic island-borne genes differs between different hosts.</p><p class="abstract"><strong>Conclusion</strong>: Genomic islands of the ICEHin1056 subfamily have a longstanding relationship with H. influenzae and H. parainfluenzae and are co-evolving as semi-autonomous genomes within the &#39;supragenomes&#39; of their host species. They have promoted bacterial diversity and adaptation through becoming efficient vectors of antibiotic resistance by the recent acquisition of antibiotic resistance transposons.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">R237</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17996041" rel="external">17996041</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258188" rel="external">2258188</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-11-r237" rel="external">10.1186/gb-2007-8-11-r237</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Radial chromatin positioning is shaped by local gene density, not by gene expression.</h4>
<p class="authors"><span class="author">K&uuml;pper K</span>, <span class="author">K&ouml;lbl A</span>, <span class="author">Biener D</span>, <span class="author">Dittrich S</span>, <span class="author">von Hase J</span>, <span class="author">Thormeyer T</span>, <span class="author">Fiegler H</span>, <span class="author">Carter NP</span>, <span class="author">Speicher MR</span>, <span class="author">Cremer T</span> and <span class="author">Cremer M</span></p>
<p class="affiliation">Department of Biology II, Anthropology and Human Genetics, Ludwig Maximilians University, Munich, Germany.</p><p class="abstract">G- and R-bands of metaphase chromosomes are characterized by profound differences in gene density, CG content, replication timing, and chromatin compaction. The preferential localization of gene-dense, transcriptionally active, and early replicating chromatin in the nuclear interior and of gene-poor, later replicating chromatin at the nuclear envelope has been demonstrated to be evolutionary-conserved in various cell types. Yet, the impact of different local chromatin features on the radial nuclear arrangement of chromatin is still not well understood. In particular, it is not known whether radial chromatin positioning is preferentially shaped by local gene density per se or by other related parameters such as replication timing or transcriptional activity. The interdependence of these distinct chromatin features on the linear deoxyribonucleic acid (DNA) sequence precludes a simple dissection of these parameters with respect to their importance for the reorganization of the linear DNA organization into the distinct radial chromatin arrangements observed in the nuclear space. To analyze this problem, we generated probe sets of pooled bacterial artificial chromosome (BAC) clones from HSA 11, 12, 18, and 19 representing R/G-band-assigned chromatin, segments with different gene density and gene loci with different expression levels. Using multicolor 3D flourescent in situ hybridization (FISH) and 3D image analysis, we determined their localization in the nucleus and their positions within or outside the corresponding chromosome territory (CT). For each BAC data on local gene density within 2- and 10-Mb windows, as well as GC (guanine and cytosine) content, replication timing and expression levels were determined. A correlation analysis of these parameters with nuclear positioning revealed regional gene density as the decisive parameter determining the radial positioning of chromatin in the nucleus in contrast to band assignment, replication timing, and transcriptional activity. We demonstrate a polarized distribution of gene-dense vs gene-poor chromatin within CTs with respect to the nuclear border. Whereas we confirm previous reports that a particular gene-dense and transcriptionally highly active region of about 2 Mb on 11p15.5 often loops out from the territory surface, gene-dense and highly expressed sequences were not generally found preferentially at the CT surface as previously suggested.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosoma</span>&nbsp;<span class="year">2007</span>;<span class="volume">116</span>;<span class="number">3</span>;<span class="pagerange">285-306</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17333233" rel="external">17333233</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2688818" rel="external">2688818</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00412-007-0098-4" rel="external">10.1007/s00412-007-0098-4</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Denoising inferred functional association networks obtained by gene fusion analysis.</h4>
<p class="authors"><span class="author">Kamburov A</span>, <span class="author">Goldovsky L</span>, <span class="author">Freilich S</span>, <span class="author">Kapazoglou A</span>, <span class="author">Kunin V</span>, <span class="author">Enright AJ</span>, <span class="author">Tsaftaris A</span> and <span class="author">Ouzounis CA</span></p>
<p class="affiliation">Computational Genomics Group, The European Bioinformatics Institute, EMBL Cambridge Outstation, Cambridge CB10 1SD, UK. <a href="mailto:%61%74%61%6E%61%73.%6B%61%6D%62%75%72%6F%76%40%67%6D%61%69%6C.%63%6F%6D">&#97;&#116;&#97;&#110;&#97;&#115;&#46;&#107;&#97;&#109;&#98;&#117;&#114;&#111;&#118;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: Gene fusion detection - also known as the &#39;Rosetta Stone&#39; method - involves the identification of fused composite genes in a set of reference genomes, which indicates potential interactions between its un-fused counterpart genes in query genomes. The precision of this method typically improves with an ever-increasing number of reference genomes.</p><p class="abstract"><strong>Results</strong>: In order to explore the usefulness and scope of this approach for protein interaction prediction and generate a high-quality, non-redundant set of interacting pairs of proteins across a wide taxonomic range, we have exhaustively performed gene fusion analysis for 184 genomes using an efficient variant of a previously developed protocol. By analyzing interaction graphs and applying a threshold that limits the maximum number of possible interactions within the largest graph components, we show that we can reduce the number of implausible interactions due to the detection of promiscuous domains. With this generally applicable approach, we generate a robust set of over 2 million distinct and testable interactions encompassing 696,894 proteins in 184 species or strains, most of which have never been the subject of high-throughput experimental proteomics. We investigate the cumulative effect of increasing numbers of genomes on the fidelity and quantity of predictions, and show that, for large numbers of genomes, predictions do not become saturated but continue to grow linearly, for the majority of the species. We also examine the percentage of component (and composite) proteins with relation to the number of genes and further validate the functional categories that are highly represented in this robust set of detected genome-wide interactions.</p><p class="abstract"><strong>Conclusion</strong>: We illustrate the phylogenetic and functional diversity of gene fusion events across genomes, and their usefulness for accurate prediction of protein interaction and function.</p>
<p class="publication"><span class="title">BMC genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">460</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18081932" rel="external">18081932</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2248599" rel="external">2248599</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2164-8-460" rel="external">10.1186/1471-2164-8-460</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MultiPhyl: a high-throughput phylogenomics webserver using distributed computing.</h4>
<p class="authors"><span class="author">Keane TM</span>, <span class="author">Naughton TJ</span> and <span class="author">McInerney JO</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SA Hinxton, UK. <a href="mailto:%74%6B%65%61%6E%65%40%63%73.%6E%75%69%6D.%69%65">&#116;&#107;&#101;&#97;&#110;&#101;&#64;&#99;&#115;&#46;&#110;&#117;&#105;&#109;&#46;&#105;&#101;</a></p><p class="abstract">With the number of fully sequenced genomes increasing steadily, there is greater interest in performing large-scale phylogenomic analyses from large numbers of individual gene families. Maximum likelihood (ML) has been shown repeatedly to be one of the most accurate methods for phylogenetic construction. Recently, there have been a number of algorithmic improvements in maximum-likelihood-based tree search methods. However, it can still take a long time to analyse the evolutionary history of many gene families using a single computer. Distributed computing refers to a method of combining the computing power of multiple computers in order to perform some larger overall calculation. In this article, we present the first high-throughput implementation of a distributed phylogenetics platform, MultiPhyl, capable of using the idle computational resources of many heterogeneous non-dedicated machines to form a phylogenetics supercomputer. MultiPhyl allows a user to upload hundreds or thousands of amino acid or nucleotide alignments simultaneously and perform computationally intensive tasks such as model selection, tree searching and bootstrapping of each of the alignments using many desktop machines. The program implements a set of 88 amino acid models and 56 nucleotide maximum likelihood models and a variety of statistical methods for choosing between alternative models. A MultiPhyl webserver is available for public use at: http://www.cs.nuim.ie/distributed/multiphyl.php.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">Web Server issue</span>;<span class="pagerange">W33-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17553837" rel="external">17553837</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1933173" rel="external">1933173</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkm359" rel="external">10.1093/nar/gkm359</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ability of SPI2 mutant of S. typhi to effectively induce antibody responses to the mucosal antigen enterotoxigenic E. coli heat labile toxin B subunit after oral delivery to humans.</h4>
<p class="authors"><span class="author">Khan S</span>, <span class="author">Chatfield S</span>, <span class="author">Stratford R</span>, <span class="author">Bedwell J</span>, <span class="author">Bentley M</span>, <span class="author">Sulsh S</span>, <span class="author">Giemza R</span>, <span class="author">Smith S</span>, <span class="author">Bongard E</span>, <span class="author">Cosgrove CA</span>, <span class="author">Johnson J</span>, <span class="author">Dougan G</span>, <span class="author">Griffin GE</span>, <span class="author">Makin J</span> and <span class="author">Lewis DJ</span></p>
<p class="affiliation">Microscience, Wokingham Berkshire RG41 5TU, UK.</p><p class="abstract">We have evaluated an oral vaccine based on an Salmonella enteric serovar typhi (S. typhi) Ty2 derivative TSB7 harboring deletion mutations in ssaV (SPI-2) and aroC together with a chromosomally integrated copy of eltB encoding the B subunit of enterotoxigenic Escherichia coli heat labile toxin (LT-B) in volunteers. Two oral doses of 10(8) or 10(9)CFU were administered to two groups of volunteers and both doses were well tolerated, with no vaccinemia, and only transient stool shedding. Immune responses to LT-B and S. typhi lipopolysaccharide were demonstrated in 67 and 97% of subjects, respectively, without evidence of anti-carrier immunity preventing boosting of LT-B responses in many cases. Further development of this salmonella-based (spi-VEC) system for oral delivery of heterologous antigens appears warranted.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Vaccine</span>&nbsp;<span class="year">2007</span>;<span class="volume">25</span>;<span class="number">21</span>;<span class="pagerange">4175-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17412462" rel="external">17412462</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652036" rel="external">2652036</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vaccine.2007.03.007" rel="external">10.1016/j.vaccine.2007.03.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates.</h4>
<p class="authors"><span class="author">Kikuta H</span>, <span class="author">Laplante M</span>, <span class="author">Navratilova P</span>, <span class="author">Komisarczuk AZ</span>, <span class="author">Engstr&ouml;m PG</span>, <span class="author">Fredman D</span>, <span class="author">Akalin A</span>, <span class="author">Caccamo M</span>, <span class="author">Sealy I</span>, <span class="author">Howe K</span>, <span class="author">Ghislain J</span>, <span class="author">Pezeron G</span>, <span class="author">Mourrain P</span>, <span class="author">Ellingsen S</span>, <span class="author">Oates AC</span>, <span class="author">Thisse C</span>, <span class="author">Thisse B</span>, <span class="author">Foucher I</span>, <span class="author">Adolf B</span>, <span class="author">Geling A</span>, <span class="author">Lenhard B</span> and <span class="author">Becker TS</span></p>
<p class="affiliation">Sars Centre for Marine Molecular Biology, University of Bergen, 5008 Bergen, Norway.</p><p class="abstract">We report evidence for a mechanism for the maintenance of long-range conserved synteny across vertebrate genomes. We found the largest mammal-teleost conserved chromosomal segments to be spanned by highly conserved noncoding elements (HCNEs), their developmental regulatory target genes, and phylogenetically and functionally unrelated &quot;bystander&quot; genes. Bystander genes are not specifically under the control of the regulatory elements that drive the target genes and are expressed in patterns that are different from those of the target genes. Reporter insertions distal to zebrafish developmental regulatory genes pax6.1/2, rx3, id1, and fgf8 and miRNA genes mirn9-1 and mirn9-5 recapitulate the expression patterns of these genes even if located inside or beyond bystander genes, suggesting that the regulatory domain of a developmental regulatory gene can extend into and beyond adjacent transcriptional units. We termed these chromosomal segments genomic regulatory blocks (GRBs). After whole genome duplication in teleosts, GRBs, including HCNEs and target genes, were often maintained in both copies, while bystander genes were typically lost from one GRB, strongly suggesting that evolutionary pressure acts to keep the single-copy GRBs of higher vertebrates intact. We show that loss of bystander genes and other mutational events suffered by duplicated GRBs in teleost genomes permits target gene identification and HCNE/target gene assignment. These findings explain the absence of evolutionary breakpoints from large vertebrate chromosomal segments and will aid in the recognition of position effect mutations within human GRBs.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">5</span>;<span class="pagerange">545-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17387144" rel="external">17387144</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1855176" rel="external">1855176</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.6086307" rel="external">10.1101/gr.6086307</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reduced ENaC protein abundance contributes to the lower blood pressure observed in pendrin-null mice.</h4>
<p class="authors"><span class="author">Kim YH</span>, <span class="author">Pech V</span>, <span class="author">Spencer KB</span>, <span class="author">Beierwaltes WH</span>, <span class="author">Everett LA</span>, <span class="author">Green ED</span>, <span class="author">Shin W</span>, <span class="author">Verlander JW</span>, <span class="author">Sutliff RL</span> and <span class="author">Wall SM</span></p>
<p class="affiliation">Department of Medicine, Emory University, Atlanta, Georgia, USA.</p><p class="abstract">Pendrin (encoded by Pds, Slc26a4) is a Cl(-)/HCO(3)(-) exchanger expressed in the apical regions of type B and non-A, non-B intercalated cells of kidney and mediates renal Cl(-) absorption, particularly when upregulated. Aldosterone increases blood pressure by increasing absorption of both Na(+) and Cl(-) through increased protein abundance and function of Na(+) transporters, such as the epithelial Na(+) channel (ENaC) and the Na(+)-Cl(-) cotransporter (NCC), as well as Cl(-) transporters, such as pendrin. Because aldosterone analogs do not increase blood pressure in Slc26a4(-/-) mice, we asked whether Na(+) excretion and Na(+) transporter protein abundance are altered in kidneys from these mutant mice. Thus wild-type and Slc26a4-null mice were given a NaCl-replete, a NaCl-restricted, or NaCl-replete diet and aldosterone or aldosterone analogs. Abundance of the major renal Na(+) transporters was examined with immunoblots and immunohistochemistry. Slc26a4-null mice showed an impaired ability to conserve Na(+) during dietary NaCl restriction. Under treatment conditions in which circulating aldosterone is increased, alpha-, beta-, and 85-kDa gamma-ENaC subunit protein abundances were reduced 15-35%, whereas abundance of the 70-kDa fragment of gamma-ENaC was reduced approximately 70% in Slc26a4-null relative to wild-type mice. Moreover, ENaC-dependent changes in transepithelial voltage were much lower in cortical collecting ducts from Slc26a4-null than from wild-type mice. Thus, in kidney, ENaC protein abundance and function are modulated by pendrin or through a pendrin-dependent downstream event. The reduced ENaC protein abundance and function observed in Slc26a4-null mice contribute to their lower blood pressure and reduced ability to conserve Na(+) during NaCl restriction.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">P01 061521</span></p>
<p class="publication"><span class="title">American journal of physiology. Renal physiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">293</span>;<span class="number">4</span>;<span class="pagerange">F1314-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17686956">17686956</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fajprenal.00155.2007">10.1152/ajprenal.00155.2007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Finding cis-regulatory elements using comparative genomics: some lessons from ENCODE data.</h4>
<p class="authors"><span class="author">King DC</span>, <span class="author">Taylor J</span>, <span class="author">Zhang Y</span>, <span class="author">Cheng Y</span>, <span class="author">Lawson HA</span>, <span class="author">Martin J</span>, <span class="author">ENCODE groups for Transcriptional Regulation and Multispecies Sequence Analysis</span>, <span class="author">Chiaromonte F</span>, <span class="author">Miller W</span> and <span class="author">Hardison RC</span></p>
<p class="affiliation">Center for Comparative Genomics and Bioinformatics, The Pennsylvania State University, University Park, Pennsylvania 16802, USA.</p><p class="abstract">Identification of functional genomic regions using interspecies comparison will be most effective when the full span of relationships between genomic function and evolutionary constraint are utilized. We find that sets of putative transcriptional regulatory sequences, defined by ENCODE experimental data, have a wide span of evolutionary histories, ranging from stringent constraint shown by deep phylogenetic comparisons to recent selection on lineage-specific elements. This diversity of evolutionary histories can be captured, at least in part, by the suite of available comparative genomics tools, especially after correction for regional differences in the neutral substitution rate. Putative transcriptional regulatory regions show alignability in different clades, and the genes associated with them are enriched for distinct functions. Some of the putative regulatory regions show evidence for recent selection, including a primate-specific, distal promoter that may play a novel role in regulation.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG002238</span>; NIDDK NIH HHS: <span class="grant">DK65806</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">775-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17567996" rel="external">17567996</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1891337" rel="external">1891337</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5592107" rel="external">10.1101/gr.5592107</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Africans in Yorkshire? The deepest-rooting clade of the Y phylogeny within an English genealogy.</h4>
<p class="authors"><span class="author">King TE</span>, <span class="author">Parkin EJ</span>, <span class="author">Swinfield G</span>, <span class="author">Cruciani F</span>, <span class="author">Scozzari R</span>, <span class="author">Rosa A</span>, <span class="author">Lim SK</span>, <span class="author">Xue Y</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Jobling MA</span></p>
<p class="affiliation">Department of Genetics, University of Leicester, Leicester, UK.</p><p class="abstract">The presence of Africans in Britain has been recorded since Roman times, but has left no apparent genetic trace among modern inhabitants. Y chromosomes belonging to the deepest-rooting clade of the Y phylogeny, haplogroup (hg) A, are regarded as African-specific, and no examples have been reported from Britain or elsewhere in Western Europe. We describe the presence of an hgA1 chromosome in an indigenous British male; comparison with African examples suggests a Western African origin. Seven out of 18 men carrying the same rare east-Yorkshire surname as the original male also carry hgA1 chromosomes, and documentary research resolves them into two genealogies with most-recent-common-ancestors living in Yorkshire in the late 18th century. Analysis using 77 Y-short tandem repeats (STRs) is consistent with coalescence a few generations earlier. Our findings represent the first genetic evidence of Africans among &#39;indigenous&#39; British, and emphasize the complexity of human migration history as well as the pitfalls of assigning geographical origin from Y-chromosomal haplotypes.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">3</span>;<span class="pagerange">288-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17245408" rel="external">17245408</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2590664" rel="external">2590664</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5201771" rel="external">10.1038/sj.ejhg.5201771</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Arginine methylation at histone H3R2 controls deposition of H3K4 trimethylation.</h4>
<p class="authors"><span class="author">Kirmizis A</span>, <span class="author">Santos-Rosa H</span>, <span class="author">Penkett CJ</span>, <span class="author">Singer MA</span>, <span class="author">Vermeulen M</span>, <span class="author">Mann M</span>, <span class="author">B&auml;hler J</span>, <span class="author">Green RD</span> and <span class="author">Kouzarides T</span></p>
<p class="affiliation">Gurdon Institute and Department of Pathology, Tennis Court Road, Cambridge CB2 1QN, UK.</p><p class="abstract">Modifications on histones control important biological processes through their effects on chromatin structure. Methylation at lysine 4 on histone H3 (H3K4) is found at the 5&#39; end of active genes and contributes to transcriptional activation by recruiting chromatin-remodelling enzymes. An adjacent arginine residue (H3R2) is also known to be asymmetrically dimethylated (H3R2me2a) in mammalian cells, but its location within genes and its function in transcription are unknown. Here we show that H3R2 is also methylated in budding yeast (Saccharomyces cerevisiae), and by using an antibody specific for H3R2me2a in a chromatin immunoprecipitation-on-chip analysis we determine the distribution of this modification on the entire yeast genome. We find that H3R2me2a is enriched throughout all heterochromatic loci and inactive euchromatic genes and is present at the 3&#39; end of moderately transcribed genes. In all cases the pattern of H3R2 methylation is mutually exclusive with the trimethyl form of H3K4 (H3K4me3). We show that methylation at H3R2 abrogates the trimethylation of H3K4 by the Set1 methyltransferase. The specific effect on H3K4me3 results from the occlusion of Spp1, a Set1 methyltransferase subunit necessary for trimethylation. Thus, the inability of Spp1 to recognize H3 methylated at R2 prevents Set1 from trimethylating H3K4. These results provide the first mechanistic insight into the function of arginine methylation on chromatin.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span>, <span class="grant">092096</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">449</span>;<span class="number">7164</span>;<span class="pagerange">928-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17898715">17898715</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3350864">3350864</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06160">10.1038/nature06160</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The landscape of histone modifications across 1% of the human genome in five human cell lines.</h4>
<p class="authors"><span class="author">Koch CM</span>, <span class="author">Andrews RM</span>, <span class="author">Flicek P</span>, <span class="author">Dillon SC</span>, <span class="author">Kara&ouml;z U</span>, <span class="author">Clelland GK</span>, <span class="author">Wilcox S</span>, <span class="author">Beare DM</span>, <span class="author">Fowler JC</span>, <span class="author">Couttet P</span>, <span class="author">James KD</span>, <span class="author">Lefebvre GC</span>, <span class="author">Bruce AW</span>, <span class="author">Dovey OM</span>, <span class="author">Ellis PD</span>, <span class="author">Dhami P</span>, <span class="author">Langford CF</span>, <span class="author">Weng Z</span>, <span class="author">Birney E</span>, <span class="author">Carter NP</span>, <span class="author">Vetrie D</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB101SA, United Kingdom.</p><p class="abstract">We generated high-resolution maps of histone H3 lysine 9/14 acetylation (H3ac), histone H4 lysine 5/8/12/16 acetylation (H4ac), and histone H3 at lysine 4 mono-, di-, and trimethylation (H3K4me1, H3K4me2, H3K4me3, respectively) across the ENCODE regions. Studying each modification in five human cell lines including the ENCODE Consortium common cell lines GM06990 (lymphoblastoid) and HeLa-S3, as well as K562, HFL-1, and MOLT4, we identified clear patterns of histone modification profiles with respect to genomic features. H3K4me3, H3K4me2, and H3ac modifications are tightly associated with the transcriptional start sites (TSSs) of genes, while H3K4me1 and H4ac have more widespread distributions. TSSs reveal characteristic patterns of both types of modification present and the position relative to TSSs. These patterns differ between active and inactive genes and in particular the state of H3K4me3 and H3ac modifications is highly predictive of gene activity. Away from TSSs, modification sites are enriched in H3K4me1 and relatively depleted in H3K4me3 and H3ac. Comparison between cell lines identified differences in the histone modification profiles associated with transcriptional differences between the cell lines. These results provide an overview of the functional relationship among histone modifications and gene expression in human cells.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01HG03110</span>, <span class="grant">U01HG003168</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">691-707</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17567990" rel="external">17567990</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1891331" rel="external">1891331</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5704207" rel="external">10.1101/gr.5704207</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Paired-end mapping reveals extensive structural variation in the human genome.</h4>
<p class="authors"><span class="author">Korbel JO</span>, <span class="author">Urban AE</span>, <span class="author">Affourtit JP</span>, <span class="author">Godwin B</span>, <span class="author">Grubert F</span>, <span class="author">Simons JF</span>, <span class="author">Kim PM</span>, <span class="author">Palejev D</span>, <span class="author">Carriero NJ</span>, <span class="author">Du L</span>, <span class="author">Taillon BE</span>, <span class="author">Chen Z</span>, <span class="author">Tanzer A</span>, <span class="author">Saunders AC</span>, <span class="author">Chi J</span>, <span class="author">Yang F</span>, <span class="author">Carter NP</span>, <span class="author">Hurles ME</span>, <span class="author">Weissman SM</span>, <span class="author">Harkins TT</span>, <span class="author">Gerstein MB</span>, <span class="author">Egholm M</span> and <span class="author">Snyder M</span></p>
<p class="affiliation">Molecular Biophysics and Biochemistry Department, Yale University, New Haven, CT 06520, USA.</p><p class="abstract">Structural variation of the genome involves kilobase- to megabase-sized deletions, duplications, insertions, inversions, and complex combinations of rearrangements. We introduce high-throughput and massive paired-end mapping (PEM), a large-scale genome-sequencing method to identify structural variants (SVs) approximately 3 kilobases (kb) or larger that combines the rescue and capture of paired ends of 3-kb fragments, massive 454 sequencing, and a computational approach to map DNA reads onto a reference genome. PEM was used to map SVs in an African and in a putatively European individual and identified shared and divergent SVs relative to the reference genome. Overall, we fine-mapped more than 1000 SVs and documented that the number of SVs among humans is much larger than initially hypothesized; many of the SVs potentially affect gene function. The breakpoint junction sequences of more than 200 SVs were determined with a novel pooling strategy and computational analysis. Our analysis provided insights into the mechanisms of SV formation in humans.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">RR19895</span>; Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077014</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">318</span>;<span class="number">5849</span>;<span class="pagerange">420-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17901297">17901297</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2674581">2674581</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1149504">10.1126/science.1149504</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New tools and expanded data analysis capabilities at the Protein Structure Prediction Center.</h4>
<p class="authors"><span class="author">Kryshtafovych A</span>, <span class="author">Prlic A</span>, <span class="author">Dmytriv Z</span>, <span class="author">Daniluk P</span>, <span class="author">Milostan M</span>, <span class="author">Eyrich V</span>, <span class="author">Hubbard T</span> and <span class="author">Fidelis K</span></p>
<p class="affiliation">Genome Center, University of California, Davis, California 95616, USA.</p><p class="abstract">We outline the main tasks performed by the Protein Structure Prediction Center in support of the CASP7 experiment and provide a brief review of the major measures used in the automatic evaluation of predictions. We describe in more detail the software developed to facilitate analysis of modeling success over and beyond the available templates and the adopted Java-based tool enabling visualization of multiple structural superpositions between target and several models/templates. We also give an overview of the CASP infrastructure provided by the Center and discuss the organization of the results web pages available through http://predictioncenter.org.</p>
<p class="grants">Funded by: NLM NIH HHS: <span class="grant">LM07085-01</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2007</span>;<span class="volume">69 Suppl 8</span>;<span class="pagerange">19-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17705273">17705273</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656758">2656758</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fprot.21653">10.1002/prot.21653</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An RNA G-quadruplex in the 5&#39; UTR of the NRAS proto-oncogene modulates translation.</h4>
<p class="authors"><span class="author">Kumari S</span>, <span class="author">Bugaut A</span>, <span class="author">Huppert JL</span> and <span class="author">Balasubramanian S</span></p>
<p class="affiliation">University Chemical Laboratory, Lensfield Road, Cambridge CB2 1EW, UK.</p><p class="abstract">Guanine-rich nucleic acid sequences can adopt noncanonical four-stranded secondary structures called guanine (G)-quadruplexes. Bioinformatics analysis suggests that G-quadruplex motifs are prevalent in genomes, which raises the need to elucidate their function. There is now evidence for the existence of DNA G-quadruplexes at telomeres with associated biological function. A recent hypothesis supports the notion that gene promoter elements contain DNA G-quadruplex motifs that control gene expression at the transcriptional level. We discovered a highly conserved, thermodynamically stable RNA G-quadruplex in the 5&#39; untranslated region (UTR) of the gene transcript of the human NRAS proto-oncogene. Using a cell-free translation system coupled to a reporter gene assay, we have demonstrated that this NRAS RNA G-quadruplex modulates translation. This is the first example of translational repression by an RNA G-quadruplex. Bioinformatics analysis has revealed 2,922 other 5&#39; UTR RNA G-quadruplex elements in the human genome. We propose that RNA G-quadruplexes in the 5&#39; UTR modulate gene expression at the translational level.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A4081</span></p>
<p class="publication"><span class="title">Nature chemical biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">4</span>;<span class="pagerange">218-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17322877">17322877</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2206252">2206252</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnchembio864">10.1038/nchembio864</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comprehensive antibody panel for immunohistochemical analysis of formalin-fixed, paraffin-embedded hematopoietic neoplasms of mice: analysis of mouse specific and human antibodies cross-reactive with murine tissue.</h4>
<p class="authors"><span class="author">Kunder S</span>, <span class="author">Calzada-Wack J</span>, <span class="author">H&ouml;lzlwimmer G</span>, <span class="author">M&uuml;ller J</span>, <span class="author">Kloss C</span>, <span class="author">Howat W</span>, <span class="author">Schmidt J</span>, <span class="author">H&ouml;fler H</span>, <span class="author">Warren M</span> and <span class="author">Quintanilla-Martinez L</span></p>
<p class="affiliation">GSF Research Center for Environment and Health, Institute of Pathology, Neuherberg 85764, Germany.</p><p class="abstract">Immunohistochemistry is an indispensable tool in human pathology enabling immunophenotypic characterization of tumor cells. Immunohistochemical analyses of mouse models of human hematopoietic neoplasias have become an important aspect for comparison of murine entities with their human counterparts. The aim of this study was to establish a diagnostic antibody panel for analysis of murine lymphomas/leukemias, useful in formalin-fixed/paraffin-embedded tissue. Overall, 48 antibodies (4 rabbit monoclonal, 12 rabbit polyclonal, 2 goat polyclonal, 11 rat, and 19 mouse monoclonal), which were either mouse-specific (14) or cross-reactive with murine tissue (34) were tested for staining quality and diagnostic value in 468 murine hematopoietic neoplasms. Specific staining was achieved with 29 antibodies, of which 18 were human antibodies cross-reactive with murine tissue. Only 23 (B220, BCL-2, BCL-6, CD117, CD138 (2x), CD3 (2x), CD43, CD45, CD5, CD79 alpha cy, cyclin D1, Ki-67 (2x), Mac-3, Mac-2, lysozyme, mast cell tryptase, MPO, Pax-5, TdT, and TER-119) were regarded as valuable for diagnostic evaluation. Immunohistochemistry was also established in an automated immunostainer for high throughput analysis. The antibody panel developed is useful for the classification of murine lymphomas and leukemias analyzed, and a valuable tool for human and veterinary pathologists involved in the diagnostic interpretation of murine models of hematopoietic neoplasias.</p>
<p class="publication"><span class="title">Toxicologic pathology</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">3</span>;<span class="pagerange">366-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17455084">17455084</a>; DOI: <a href="http://dx.doi.org/10.1080%2F01926230701230296">10.1080/01926230701230296</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A network of multiple regulatory layers shapes gene expression in fission yeast.</h4>
<p class="authors"><span class="author">Lackner DH</span>, <span class="author">Beilharz TH</span>, <span class="author">Marguerat S</span>, <span class="author">Mata J</span>, <span class="author">Watt S</span>, <span class="author">Schubert F</span>, <span class="author">Preiss T</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">Gene expression is controlled at multiple layers, and cells may integrate different regulatory steps for coherent production of proper protein levels. We applied various microarray-based approaches to determine key gene-expression intermediates in exponentially growing fission yeast, providing genome-wide data for translational profiles, mRNA steady-state levels, polyadenylation profiles, start-codon sequence context, mRNA half-lives, and RNA polymerase II occupancy. We uncovered widespread and unexpected relationships between distinct aspects of gene expression. Translation and polyadenylation are aligned on a global scale with both the lengths and levels of mRNAs: efficiently translated mRNAs have longer poly(A) tails and are shorter, more stable, and more efficiently transcribed on average. Transcription and translation may be independently but congruently optimized to streamline protein production. These rich data sets, all acquired under a standardized condition, reveal a substantial coordination between regulatory layers and provide a basis for a systems-level understanding of multilayered gene-expression programs.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular cell</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">1</span>;<span class="pagerange">145-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17434133" rel="external">17434133</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1885965" rel="external">1885965</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molcel.2007.03.002" rel="external">10.1016/j.molcel.2007.03.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">hORFeome v3.1: a resource of human open reading frames representing over 10,000 human genes.</h4>
<p class="authors"><span class="author">Lamesch P</span>, <span class="author">Li N</span>, <span class="author">Milstein S</span>, <span class="author">Fan C</span>, <span class="author">Hao T</span>, <span class="author">Szabo G</span>, <span class="author">Hu Z</span>, <span class="author">Venkatesan K</span>, <span class="author">Bethel G</span>, <span class="author">Martin P</span>, <span class="author">Rogers J</span>, <span class="author">Lawlor S</span>, <span class="author">McLaren S</span>, <span class="author">Dricot A</span>, <span class="author">Borick H</span>, <span class="author">Cusick ME</span>, <span class="author">Vandenhaute J</span>, <span class="author">Dunham I</span>, <span class="author">Hill DE</span> and <span class="author">Vidal M</span></p>
<p class="affiliation">Center for Cancer Systems Biology and Department of Cancer Biology, Dana-Farber Cancer Institute, and Department of Genetics, Harvard Medical School, Boston, MA 02115, USA.</p><p class="abstract">Complete sets of cloned protein-encoding open reading frames (ORFs), or ORFeomes, are essential tools for large-scale proteomics and systems biology studies. Here we describe human ORFeome version 3.1 (hORFeome v3.1), currently the largest publicly available resource of full-length human ORFs (available at ). Generated by Gateway recombinational cloning, this collection contains 12,212 ORFs, representing 10,214 human genes, and corresponds to a 51% expansion of the original hORFeome v1.1. An online human ORFeome database, hORFDB, was built and serves as the central repository for all cloned human ORFs (http://horfdb.dfci.harvard.edu). This expansion of the original ORFeome resource greatly increases the potential experimental search space for large-scale proteomics studies, which will lead to the generation of more comprehensive datasets.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">89</span>;<span class="number">3</span>;<span class="pagerange">307-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17207965">17207965</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ygeno.2006.11.012">10.1016/j.ygeno.2006.11.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of neuronal complexes in human X-linked brain diseases.</h4>
<p class="authors"><span class="author">Laumonnier F</span>, <span class="author">Cuthbert PC</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Beyond finding individual genes that are involved in medical disorders, an important challenge is the integration of sets of disease genes with the complexities of basic biological processes. We examine this issue by focusing on neuronal multiprotein complexes and their components encoded on the human X chromosome. Multiprotein signaling complexes in the postsynaptic terminal of central nervous system synapses are essential for the induction of neuronal plasticity and cognitive processes in animals. The prototype complex is the N-methyl-D-aspartate receptor complex/membrane-associated guanylate kinase-associated signaling complex (NRC/MASC) comprising 185 proteins and embedded within the postsynaptic density (PSD), which is a set of complexes totaling approximately 1,100 proteins. It is striking that 86% (6 of 7) of X-linked NRC/MASC genes and 49% (19 of 39) of X-chromosomal PSD genes are already known to be involved in human psychiatric disorders. Moreover, of the 69 known proteins mutated in X-linked mental retardation, 19 (28%) encode postsynaptic proteins. The high incidence of involvement in cognitive disorders is also found in mouse mutants and indicates that the complexes are functioning as integrated entities or molecular machines and that disruption of different components impairs their overall role in cognitive processes. We also noticed that NRC/MASC genes appear to be more strongly associated with mental retardation and autism spectrum disorders. We propose that systematic studies of PSD and NRC/MASC genes in mice and humans will give a high yield of novel genes important for human disease and new mechanistic insights into higher cognitive functions.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">80</span>;<span class="number">2</span>;<span class="pagerange">205-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17236127" rel="external">17236127</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1785339" rel="external">1785339</a>; DOI: <a href="http://dx.doi.org/10.1086%2F511441" rel="external">10.1086/511441</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of a novel tumor transforming gene GAEC1 at 7q22 which encodes a nuclear protein and is frequently amplified and overexpressed in esophageal squamous cell carcinoma.</h4>
<p class="authors"><span class="author">Law FB</span>, <span class="author">Chen YW</span>, <span class="author">Wong KY</span>, <span class="author">Ying J</span>, <span class="author">Tao Q</span>, <span class="author">Langford C</span>, <span class="author">Lee PY</span>, <span class="author">Law S</span>, <span class="author">Cheung RW</span>, <span class="author">Chui CH</span>, <span class="author">Tsao SW</span>, <span class="author">Lam KY</span>, <span class="author">Wong J</span>, <span class="author">Srivastava G</span> and <span class="author">Tang JC</span></p>
<p class="affiliation">Department of Pathology, Li Ka Shing Faculty of Medicine, The University of Hong Kong, Hong Kong, China.</p><p class="abstract">By comparative DNA fingerprinting, we identified a 357-bp DNA fragment frequently amplified in esophageal squamous cell carcinomas (ESCC). This fragment overlaps with an expressed sequence tag mapped to 7q22. Further 5&#39; and 3&#39;-rapid amplification of cDNA ends revealed that it is part of a novel, single-exon gene with full-length mRNA of 2052 bp and encodes a nuclear protein of 109 amino acids ( approximately 15 kDa). This gene, designated as gene amplified in esophageal cancer 1 (GAEC1), was located within a 1-2 Mb amplicon at 7q22.1 identified by high-resolution 1 Mb array-comparative genomic hybridization in 6/10 ESCC cell lines. GAEC1 was ubiquitously expressed in normal tissues including esophageal and gastrointestinal organs; with amplification and overexpression in 6/10 (60%) ESCC cell lines and 34/99 (34%) primary tumors. Overexpression of GAEC1 in 3T3 mouse fibroblasts caused foci formation and colony formation in soft agar, comparable to H-ras and injection of GAEC1-transfected 3T3 cells into athymic nude mice formed undifferentiated sarcoma in vivo, indicating that GAEC1 is a transforming oncogene. Although no significant correlation was observed between GAEC1 amplification and clinicopathological parameters and prognosis, our study demonstrated that overexpressed GAEC1 has tumorigenic potential and suggest that overexpressed GAEC1 may play an important role in ESCC pathogenesis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079643</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">40</span>;<span class="pagerange">5877-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17384685" rel="external">17384685</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2875854" rel="external">2875854</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1210390" rel="external">10.1038/sj.onc.1210390</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Selective colonization of insoluble substrates by human faecal bacteria.</h4>
<p class="authors"><span class="author">Leitch EC</span>, <span class="author">Walker AW</span>, <span class="author">Duncan SH</span>, <span class="author">Holtrop G</span> and <span class="author">Flint HJ</span></p>
<p class="affiliation">Microbial Ecology Group, Aberdeen, UK.</p><p class="abstract">Insoluble plant polysaccharides and endogenous mucin are important energy sources for human colonic microorganisms. The object of this study was to determine whether or not specific communities colonize these substrates. Using faecal samples from four individuals as inocula for an anaerobic in vitro continuous flow system, the colonization of wheat bran, high amylose starch and porcine gastric mucin was examined. Recovered substrates were extensively washed and the remaining tightly attached bacterial communities were identified using polymerase chain reaction-amplified 16S rRNA gene sequences and fluorescent in situ hybridization. The substrate had a major influence on the species of attached bacteria detected. Sequences retrieved from bran were dominated by clostridial cluster XIVa bacteria, including uncultured relatives of Clostridium hathewayi, Eubacterium rectale and Roseburia species. Bacteroides species were also detected. The most abundant sequences recovered from starch were related to the cultured species Ruminococcus bromii, Bifidobacterium adolescentis, Bifidobacterium breve and E. rectale. The most commonly recovered sequences from mucin were from Bifidobacterium bifidum and uncultured bacteria related to Ruminococcus lactaris. This study suggests that a specific subset of bacteria is likely to be the primary colonizers of particular insoluble colonic substrates. For a given substrate, however, the primary colonizing species may vary between host individuals.</p>
<p class="publication"><span class="title">Environmental microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">9</span>;<span class="number">3</span>;<span class="pagerange">667-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17298367" rel="external">17298367</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1462-2920.2006.01186.x" rel="external">10.1111/j.1462-2920.2006.01186.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common ABCB1 polymorphisms are not associated with multidrug resistance in epilepsy using a gene-wide tagging approach.</h4>
<p class="authors"><span class="author">Leschziner GD</span>, <span class="author">Andrew T</span>, <span class="author">Leach JP</span>, <span class="author">Chadwick D</span>, <span class="author">Coffey AJ</span>, <span class="author">Balding DJ</span>, <span class="author">Bentley DR</span>, <span class="author">Pirmohamed M</span> and <span class="author">Johnson MR</span></p>
<p class="affiliation">Imperial College London, London, UK.</p><p class="abstract">P-glycoprotein, the product of the ABCB1 gene, is a proposed mechanism of pharmacoresistance in epilepsy. Previous attempts to correlate the ABCB1 C3435T SNP, or a three-SNP haplotype containing C3435T with epilepsy pharmacoresistance have produced discordant findings. We analysed these single nucleotide polymorphisms (SNPs), plus a more comprehensive set of tagging SNPs describing common variation in ABCB1 in a case-control study. No significant association of C3435T (P=0.55), the three-SNP haplotype (lowest P=0.14) or any gene-wide tagging SNP (lowest P=0.17) with multidrug resistance in epilepsy was identified. Meta-analysis of studies using the same definition of multidrug resistance (n=1064) also demonstrated no significant association of C3435T with multidrug resistance (P=0.31). These findings suggest that C3435T is unlikely to be a marker for epilepsy multidrug resistance. In addition, no evidence for a role of other common ABCB1 polymorphisms was found using a potentially more powerful gene-wide tagging approach.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Pharmacogenetics and genomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">217-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17460550">17460550</a>; DOI: <a href="http://dx.doi.org/10.1097%2F01.fpc.0000230408.23146.b1">10.1097/01.fpc.0000230408.23146.b1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ABCB1 genotype and PGP expression, function and therapeutic drug response: a critical review and recommendations for future research.</h4>
<p class="authors"><span class="author">Leschziner GD</span>, <span class="author">Andrew T</span>, <span class="author">Pirmohamed M</span> and <span class="author">Johnson MR</span></p>
<p class="affiliation">Division of Neurosciences, Imperial College, London, UK. <a href="mailto:%67%6C%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#103;&#108;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The product of the ABCB1 gene, P-glycoprotein (PGP), is a transmembrane active efflux pump for a variety of drugs. It is a putative mechanism of multidrug resistance in a range of diseases. It is postulated that ABCB1 polymorphisms contribute to variability in PGP function, and that therefore multidrug resistance is, at least in part, genetically determined. However, studies of ABCB1 genotype or haplotype and PGP expression, activity or drug response have produced inconsistent results. This critical review of ABCB1 genotype and PGP function, including mRNA expression, PGP-substrate drug pharmacokinetics and drug response, highlights methodological limitations of existing studies, including inadequate power, potential confounding by co-morbidity and co-medication, multiple testing, poor definition of disease phenotype and outcomes, and analysis of multiple drugs that might not be PGP substrates. We have produced recommendations for future research that will aid clarification of the association between ABCB1 genotypes and factors related to PGP activity.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The pharmacogenomics journal</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">154-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16969364" rel="external">16969364</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.tpj.6500413" rel="external">10.1038/sj.tpj.6500413</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The association between polymorphisms in RLIP76 and drug response in epilepsy.</h4>
<p class="authors"><span class="author">Leschziner GD</span>, <span class="author">Jorgensen AL</span>, <span class="author">Andrew T</span>, <span class="author">Williamson PR</span>, <span class="author">Marson AG</span>, <span class="author">Coffey AJ</span>, <span class="author">Middleditch C</span>, <span class="author">Balding DJ</span>, <span class="author">Rogers J</span>, <span class="author">Bentley DR</span>, <span class="author">Chadwick D</span>, <span class="author">Johnson MR</span> and <span class="author">Pirmohamed M</span></p>
<p class="affiliation">Imperial College London, Division of Neuroscience, Charing Cross Campus, Room 10E07, St Dunstan&#39;s Road, London W6 8RF, UK. <a href="mailto:%67%75%79%6C%65%73%63%68%40%67%6D%61%69%6C.%63%6F%6D">&#103;&#117;&#121;&#108;&#101;&#115;&#99;&#104;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Introduction</strong>: Approximately 30% of patients with epilepsy are resistant to treatment with anti-epileptic drugs (AEDs). The ABC drug transporter proteins are hypothesized to mediate drug resistance in epilepsy. More recently, a non-ABC putative transporter, RLIP76, has also been proposed to be involved in the mechanism of pharmacoresistance. One previous association study of six polymorphisms in RLIP76 failed to find any association with drug resistance in a retrospective cohort of epilepsy patients. We aimed to look for an association with outcomes reflecting drug response in a larger prospective cohort, with gene-wide coverage.</p><p class="abstract">We investigated the role of common polymorphisms in RLIP76 in epilepsy pharmacoresistance by genotyping 23 common RLIP76 polymorphisms in a prospective cohort of 503 epilepsy patients, from the standard and new anti-epileptic drugs (SANAD) prospective study of new and old AEDs. A total of 13 of these were tested for association with four outcomes reflecting response to drugs: time to first seizure, time to 12-month remission, time to withdrawal due to inadequate seizure control, and time to withdrawal due to unacceptable adverse drug events.</p><p class="abstract"><strong>Results</strong>: No significant associations, allowing for multiple testing, were found in the whole cohort. There was also no effect in a subgroup of patients on carbamazepine, which is thought to be a RLIP76 substrate, although two polymorphisms were associated with time to first seizure (p = 0.007).</p><p class="abstract"><strong>Discussion</strong>: We failed to demonstrate any association between RLIP76 polymorphisms and four different measures of drug response in the larger cohort, but a subgroup analysis of patients receiving carbamazepine suggested an association that should be investigated further.</p><p class="abstract"><strong>Conclusions</strong>: Our data suggest that common variants in RLIP76 are unlikely to contribute to epilepsy drug response.</p>
<p class="publication"><span class="title">Pharmacogenomics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">12</span>;<span class="pagerange">1715-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18086001">18086001</a>; DOI: <a href="http://dx.doi.org/10.2217%2F14622416.8.12.1715">10.2217/14622416.8.12.1715</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation of 52 new Y-STR loci in the Y Chromosome Consortium worldwide panel of 76 diverse individuals.</h4>
<p class="authors"><span class="author">Lim SK</span>, <span class="author">Xue Y</span>, <span class="author">Parkin EJ</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">We have established 16 small multiplex reactions of two-four loci to amplify 52 recently described single-copy simple Y-STRs and typed these loci in a worldwide panel of 74 diverse men and two women. Two Y-STRs were found to be commonly multicopy in this sample set and were excluded from the study. Of the remaining 50, four (DYS481, DYS570, DYS576 and DYS643) showed higher diversities than the commonly used loci and can potentially provide increased haplotype discrimination in both forensic and anthropological work. Ten loci showed occasional missing alleles, duplicated peaks or intermediate-sized alleles.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">International journal of legal medicine</span>&nbsp;<span class="year">2007</span>;<span class="volume">121</span>;<span class="number">2</span>;<span class="pagerange">124-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17058101" rel="external">17058101</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1794629" rel="external">1794629</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00414-006-0124-8" rel="external">10.1007/s00414-006-0124-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Straphylococci: A Postgenomic View</h4>
<p class="authors"><span class="author">Linday J A</span></p>
<p class="publication"><span class="title">Bacterial Pathogenomics.</span> <span class="year">2007</span>;<span class="pagerange">120-40</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MASTR: multiple alignment and structure prediction of non-coding RNAs using simulated annealing.</h4>
<p class="authors"><span class="author">Lindgreen S</span>, <span class="author">Gardner PP</span> and <span class="author">Krogh A</span></p>
<p class="affiliation">Bioinformatics Centre, Department of Molecular Biology, University of Copenhagen, Ole Maaloes Vej 5, 2200 Copenhagen N, Denmark. <a href="mailto:%73%74%69%6E%75%73%40%62%69%6E%66.%6B%75.%64%6B">&#115;&#116;&#105;&#110;&#117;&#115;&#64;&#98;&#105;&#110;&#102;&#46;&#107;&#117;&#46;&#100;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: As more non-coding RNAs are discovered, the importance of methods for RNA analysis increases. Since the structure of ncRNA is intimately tied to the function of the molecule, programs for RNA structure prediction are necessary tools in this growing field of research. Furthermore, it is known that RNA structure is often evolutionarily more conserved than sequence. However, few existing methods are capable of simultaneously considering multiple sequence alignment and structure prediction.</p><p class="abstract"><strong>Result</strong>: We present a novel solution to the problem of simultaneous structure prediction and multiple alignment of RNA sequences. Using Markov chain Monte Carlo in a simulated annealing framework, the algorithm MASTR (Multiple Alignment of STructural RNAs) iteratively improves both sequence alignment and structure prediction for a set of RNA sequences. This is done by minimizing a combined cost function that considers sequence conservation, covariation and basepairing probabilities. The results show that the method is very competitive to similar programs available today, both in terms of accuracy and computational efficiency.</p><p class="abstract"><strong>Availability</strong>: Source code available from http://mastr.binf.ku.dk/</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">24</span>;<span class="pagerange">3304-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18006551" rel="external">18006551</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm525" rel="external">10.1093/bioinformatics/btm525</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and analysis of chromosome 1 of Eimeria tenella reveals a unique segmental organization.</h4>
<p class="authors"><span class="author">Ling KH</span>, <span class="author">Rajandream MA</span>, <span class="author">Rivailler P</span>, <span class="author">Ivens A</span>, <span class="author">Yap SJ</span>, <span class="author">Madeira AM</span>, <span class="author">Mungall K</span>, <span class="author">Billington K</span>, <span class="author">Yee WY</span>, <span class="author">Bankier AT</span>, <span class="author">Carroll F</span>, <span class="author">Durham AM</span>, <span class="author">Peters N</span>, <span class="author">Loo SS</span>, <span class="author">Isa MN</span>, <span class="author">Novaes J</span>, <span class="author">Quail M</span>, <span class="author">Rosli R</span>, <span class="author">Nor Shamsudin M</span>, <span class="author">Sobreira TJ</span>, <span class="author">Tivey AR</span>, <span class="author">Wai SF</span>, <span class="author">White S</span>, <span class="author">Wu X</span>, <span class="author">Kerhornou A</span>, <span class="author">Blake D</span>, <span class="author">Mohamed R</span>, <span class="author">Shirley M</span>, <span class="author">Gruber A</span>, <span class="author">Berriman M</span>, <span class="author">Tomley F</span>, <span class="author">Dear PH</span> and <span class="author">Wan KL</span></p>
<p class="affiliation">Malaysia Genome Institute, UKM-MTDC Smart Technology Centre, Universiti Kebangsaan Malaysia, 43600 UKM Bangi, Selangor DE, Malaysia.</p><p class="abstract">Eimeria tenella is an intracellular protozoan parasite that infects the intestinal tracts of domestic fowl and causes coccidiosis, a serious and sometimes lethal enteritis. Eimeria falls in the same phylum (Apicomplexa) as several human and animal parasites such as Cryptosporidium, Toxoplasma, and the malaria parasite, Plasmodium. Here we report the sequencing and analysis of the first chromosome of E. tenella, a chromosome believed to carry loci associated with drug resistance and known to differ between virulent and attenuated strains of the parasite. The chromosome--which appears to be representative of the genome--is gene-dense and rich in simple-sequence repeats, many of which appear to give rise to repetitive amino acid tracts in the predicted proteins. Most striking is the segmentation of the chromosome into repeat-rich regions peppered with transposon-like elements and telomere-like repeats, alternating with repeat-free regions. Predicted genes differ in character between the two types of segment, and the repeat-rich regions appear to be associated with strain-to-strain variation.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">S19705</span>; Medical Research Council: <span class="grant">MC_U105131672</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">3</span>;<span class="pagerange">311-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17284678">17284678</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1800922">1800922</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5823007">10.1101/gr.5823007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A physical analysis of the Y chromosome shows no additional deletions, other than Gr/Gr, associated with testicular germ cell tumour.</h4>
<p class="authors"><span class="author">Linger R</span>, <span class="author">Dudakia D</span>, <span class="author">Huddart R</span>, <span class="author">Easton D</span>, <span class="author">Bishop DT</span>, <span class="author">Stratton MR</span> and <span class="author">Rapley EA</span></p>
<p class="affiliation">Testicular Cancer Genetics Team, Section of Cancer Genetics, Institute of Cancer Research, Brookes Lawley Building, Sutton, Surrey SM2 5NG, UK.</p><p class="abstract">Testicular germ cell tumour (TGCT) is the most common malignancy in men aged 15-45 years. A small deletion on the Y chromosome known as &#39;gr/gr&#39; was shown to be associated with a two-fold increased risk of TGCT, increasing to three-fold in cases with a family history of TGCT. Additional deletions of the Y chromosome, known as AZFa, AZFb and AZFc, are described in patients with infertility; however, complete deletions of these regions have not been identified in TGCT patients. We screened the Y chromosome in a series of TGCT cases to evaluate if additional deletions of Y were implicated in TGCT susceptibility. Single copy Y chromosome STS markers with an average inter-marker spacing of 128 kb were examined in constitutional DNA of 271 index TGCT patients. Three markers showed evidence of deletions, sY1291, indicative of &#39;gr/gr&#39; (eight out of 271; 2.9%), Y-DAZ3 contained within &#39;gr/gr&#39; (21 out of 271; 7.7%) and a single deletion of the marker G66152 was identified in one TGCT case. No other markers demonstrated deletions. While several regions of the Y chromosome are known to be deleted and associated with infertility, our study provides no evidence to suggest regions of Y deletion, other than &#39;gr/gr&#39;, are associated with susceptibility to TGCT in UK patients.</p>
<p class="publication"><span class="title">British journal of cancer</span>&nbsp;<span class="year">2007</span>;<span class="volume">96</span>;<span class="number">2</span>;<span class="pagerange">357-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17211466" rel="external">17211466</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2360005" rel="external">2360005</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.bjc.6603557" rel="external">10.1038/sj.bjc.6603557</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular analysis of the VSX1 gene in familial keratoconus.</h4>
<p class="authors"><span class="author">Liskova P</span>, <span class="author">Ebenezer ND</span>, <span class="author">Hysi PG</span>, <span class="author">Gwilliam R</span>, <span class="author">El-Ashry MF</span>, <span class="author">Moodaley LC</span>, <span class="author">Hau S</span>, <span class="author">Twa M</span>, <span class="author">Tuft SJ</span> and <span class="author">Bhatacharya SS</span></p>
<p class="affiliation">Division of Molecular Genetics, Institute of Ophthalmology, UCL, London, UK. <a href="mailto:%70.%6C%69%73%6B%6F%76%61%40%75%63%6C.%61%63.%75%6B">&#112;&#46;&#108;&#105;&#115;&#107;&#111;&#118;&#97;&#64;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Purpose</strong>: To evaluate the role of the visual system homeobox gene 1 (VSX1) in the pathogenesis of familial keratoconus.</p><p class="abstract"><strong>Methods</strong>: Families with two or more individuals with keratoconus were recruited and their members examined. The coding region and intron-exon junctions of the VSX1 gene were sequenced in affected individuals. In cases where there were possible pathogenic changes, segregation within the pedigree was analyzed. Meta analysis of reports on an association of p.D144E change with keratoconus phenotype was performed.</p><p class="abstract"><strong>Results</strong>: Probands from a panel of 85 apparently unrelated keratoconus families were included. Eleven sequence variants were observed, including the previously reported c.432C&gt;G (p.D144E) change and two novel intronic single nucleotide polymorphisms. However, these three changes did not cosegregate with the disease phenotype.</p><p class="abstract"><strong>Conclusions</strong>: We excluded the c.432C&gt;G sequence alteration as the direct cause of the disease. Lack of possibly pathogenic VSX1 sequence variants in the familial panel suggests that involvement of this gene in the pathogenesis of keratoconus is likely to be confined to a small number of pedigrees, at least in the population studied.</p>
<p class="publication"><span class="title">Molecular vision</span>&nbsp;<span class="year">2007</span>;<span class="volume">13</span>;<span class="pagerange">1887-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17960127">17960127</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comment on &quot;A common genetic variant is associated with adult and childhood obesity&quot;.</h4>
<p class="authors"><span class="author">Loos RJ</span>, <span class="author">Barroso I</span>, <span class="author">O&#39;rahilly S</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Cambridge, UK. <a href="mailto:%72%75%74%68.%6C%6F%6F%73%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#114;&#117;&#116;&#104;&#46;&#108;&#111;&#111;&#115;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Herbert et al. (Reports, 14 April 2006, p. 279) found that the rs7566605 genetic variant, located upstream of the INSIG2 gene, was consistently associated with increased body mass index. However, we found no evidence of association between rs7566605 and body mass index in two large ethnically homogeneous population-based cohorts. On the contrary, an opposite tendency was observed.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G9824984</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">315</span>;<span class="number">5809</span>;<span class="pagerange">187; author reply 187</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17218509">17218509</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2719286">2719286</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1130012">10.1126/science.1130012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TCF7L2 polymorphisms modulate proinsulin levels and beta-cell function in a British Europid population.</h4>
<p class="authors"><span class="author">Loos RJ</span>, <span class="author">Franks PW</span>, <span class="author">Francis RW</span>, <span class="author">Barroso I</span>, <span class="author">Gribble FM</span>, <span class="author">Savage DB</span>, <span class="author">Ong KK</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Wareham NJ</span></p>
<p class="affiliation">Medical Research Council Epidemiology Unit, Strangeways Research Laboratory, Cambridge, UK. <a href="mailto:%72%75%74%68.%6C%6F%6F%73%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#114;&#117;&#116;&#104;&#46;&#108;&#111;&#111;&#115;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Rapidly accumulating evidence shows that common T-cell transcription factor (TCF)7L2 polymorphisms confer risk of type 2 diabetes through unknown mechanisms. We examined the association between four TCF7L2 single nucleotide polymorphisms (SNPs), including rs7903146, and measures of insulin sensitivity and insulin secretion in 1,697 Europid men and women of the population-based MRC (Medical Research Council)-Ely study. The T-(minor) allele of rs7903146 was strongly and positively associated with fasting proinsulin (P = 4.55 x 10(-9)) and 32,33 split proinsulin (P = 1.72 x 10(-4)) relative to total insulin levels; i.e., differences between T/T and C/C homozygotes amounted to 21.9 and 18.4% respectively. Notably, the insulin-to-glucose ratio (IGR) at 30-min oral glucose tolerance test (OGTT), a frequently used surrogate of first-phase insulin secretion, was not associated with the TCF7L2 SNP (P &gt; 0.7). However, the insulin response (IGR) at 60-min OGTT was significantly lower in T-allele carriers (P = 3.5 x 10(-3)). The T-allele was also associated with higher A1C concentrations (P = 1.2 x 10(-2)) and reduced beta-cell function, assessed by homeostasis model assessment of beta-cell function (P = 2.8 x 10(-2)). Similar results were obtained for the other TCF7L2 SNPs. Of note, both major genes involved in proinsulin processing (PC1, PC2) contain TCF-binding sites in their promoters. Our findings suggest that the TCF7L2 risk allele may predispose to type 2 diabetes by impairing beta-cell proinsulin processing. The risk allele increases proinsulin levels and diminishes the 60-min but not 30-min insulin response during OGTT. The strong association between the TCF7L2 risk allele and fasting proinsulin but not insulin levels is notable, as, in this unselected and largely normoglycemic population, external influences on beta-cell stress are unlikely to be major factors influencing the efficiency of proinsulin processing.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G9824984</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">071187</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">7</span>;<span class="pagerange">1943-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17416797">17416797</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2668957">2668957</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb07-0055">10.2337/db07-0055</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Altered retinal microRNA expression profile in a mouse model of retinitis pigmentosa.</h4>
<p class="authors"><span class="author">Loscher CJ</span>, <span class="author">Hokamp K</span>, <span class="author">Kenna PF</span>, <span class="author">Ivens AC</span>, <span class="author">Humphries P</span>, <span class="author">Palfi A</span> and <span class="author">Farrar GJ</span></p>
<p class="affiliation">Smurfit Institute of Genetics, Trinity College Dublin, College Green, Dublin 2, Ireland. <a href="mailto:%6C%6F%73%63%68%65%63%6A%40%74%63%64.%69%65">&#108;&#111;&#115;&#99;&#104;&#101;&#99;&#106;&#64;&#116;&#99;&#100;&#46;&#105;&#101;</a>.</p><p class="abstract"><strong>Background</strong>: The role played by microRNAs (miRs) as common regulators in physiologic processes such as development and various disease states was recently highlighted. Retinitis pigmentosa (RP) linked to RHO (which encodes rhodopsin) is the most frequent form of inherited retinal degeneration that leads to blindness, for which there are no current therapies. Little is known about the cellular mechanisms that connect mutations within RHO to eventual photoreceptor cell death by apoptosis.</p><p class="abstract"><strong>Results</strong>: Global miR expression profiling using miR microarray technology and quantitative real-time RT-PCR (qPCR) was performed in mouse retinas. RNA samples from retina of a mouse model of RP carrying a mutant Pro347Ser RHO transgene and from wild-type retina, brain and a whole-body representation (prepared by pooling total RNA from eight different mouse organs) exhibited notably different miR profiles. Expression of retina-specific and recently described retinal miRs was semi-quantitatively demonstrated in wild-type mouse retina. Alterations greater than twofold were found in the expression of nine miRs in Pro347Ser as compared with wild-type retina (P &lt; 0.05). Expression of miR-1 and miR-133 decreased by more than 2.5-fold (P &lt; 0.001), whereas expression of miR-96 and miR-183 increased by more than 3-fold (P &lt; 0.001) in Pro347Ser retinas, as validated by qPCR. Potential retinal targets for these miRs were predicted in silico.</p><p class="abstract"><strong>Conclusion</strong>: This is the first miR microarray study to focus on evaluating altered miR expression in retinal disease. Additionally, novel retinal preference for miR-376a and miR-691 was identified. The results obtained contribute toward elucidating the function of miRs in normal and diseased retina. Modulation of expression of retinal miRs may represent a future therapeutic strategy for retinopathies such as RP.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">R248</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18034880" rel="external">18034880</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258196" rel="external">2258196</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-11-r248" rel="external">10.1186/gb-2007-8-11-r248</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional cell permeable motifs within medically relevant proteins.</h4>
<p class="authors"><span class="author">Low W</span>, <span class="author">Mortlock A</span>, <span class="author">Petrovska L</span>, <span class="author">Dottorini T</span>, <span class="author">Dougan G</span> and <span class="author">Crisanti A</span></p>
<p class="affiliation">Biological Sciences, Imperial College London, Imperial College Road, 5th floor SAF Building, London SW7 2AZ, UK. <a href="mailto:%77.%6C%6F%77%40%69%6D%70%65%72%69%61%6C.%61%63.%75%6B">&#119;&#46;&#108;&#111;&#119;&#64;&#105;&#109;&#112;&#101;&#114;&#105;&#97;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Increasing experimental evidence indicates that short polybasic peptides are able to translocate across the membrane of living cells. However, these peptides, often derived from viruses and insects, may induce unspecific effects that could mask the action of their cargoes. Here, we show that a panel of lysine and/or arginine-rich peptides, derived from human proteins involved in cell signalling pathways leading to inflammation, possess the intrinsic ability to cross intact cellular membranes. These peptides are also capable of carrying a biologically active cargo. One of these peptides, encompassing the cell permeable sequence of the Toll-receptor 4 (TLR4) adaptor protein (TIRAP) and modified to carry a dominant-negative domain of the same TIRAP protein, selectively inhibited the production of pro-inflammatory cytokines upon LPS challenge, in in vitro, ex vivo and in vivo experiments. Docking studies indicated that this inhibition might be mediated by the disruption of the recruitment of downstream effector molecules. These results show for the first time the potential of using for therapy cell permeable peptides derived from human proteins involved in disease.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Journal of biotechnology</span>&nbsp;<span class="year">2007</span>;<span class="volume">129</span>;<span class="number">3</span>;<span class="pagerange">555-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17331607" rel="external">17331607</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656996" rel="external">2656996</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jbiotec.2007.01.019" rel="external">10.1016/j.jbiotec.2007.01.019</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Islands of euchromatin-like sequence and expressed polymorphic sequences within the short arm of human chromosome 21.</h4>
<p class="authors"><span class="author">Lyle R</span>, <span class="author">Prandini P</span>, <span class="author">Osoegawa K</span>, <span class="author">ten Hallers B</span>, <span class="author">Humphray S</span>, <span class="author">Zhu B</span>, <span class="author">Eyras E</span>, <span class="author">Castelo R</span>, <span class="author">Bird CP</span>, <span class="author">Gagos S</span>, <span class="author">Scott C</span>, <span class="author">Cox A</span>, <span class="author">Deutsch S</span>, <span class="author">Ucla C</span>, <span class="author">Cruts M</span>, <span class="author">Dahoun S</span>, <span class="author">She X</span>, <span class="author">Bena F</span>, <span class="author">Wang SY</span>, <span class="author">Van Broeckhoven C</span>, <span class="author">Eichler EE</span>, <span class="author">Guigo R</span>, <span class="author">Rogers J</span>, <span class="author">de Jong PJ</span>, <span class="author">Reymond A</span> and <span class="author">Antonarakis SE</span></p>
<p class="affiliation">Department of Genetic Medicine and Development, University of Geneva Medical School, and University Hospitals, 1211 Geneva, Switzerland. <a href="mailto:%52%6F%62%65%72%74.%4C%79%6C%65%40%6D%65%64%69%73%69%6E.%75%69%6F.%6E%6F">&#82;&#111;&#98;&#101;&#114;&#116;&#46;&#76;&#121;&#108;&#101;&#64;&#109;&#101;&#100;&#105;&#115;&#105;&#110;&#46;&#117;&#105;&#111;&#46;&#110;&#111;</a></p><p class="abstract">The goals of the human genome project did not include sequencing of the heterochromatic regions. We describe here an initial sequence of 1.1 Mb of the short arm of human chromosome 21 (HSA21p), estimated to be 10% of 21p. This region contains extensive euchromatic-like sequence and includes on average one transcript every 100 kb. These transcripts show multiple inter- and intrachromosomal copies, and extensive copy number and sequence variability. The sequencing of the &quot;heterochromatic&quot; regions of the human genome is likely to reveal many additional functional elements and provide important evolutionary information.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG002385</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">11</span>;<span class="pagerange">1690-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17895424" rel="external">17895424</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2045151" rel="external">2045151</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.6675307" rel="external">10.1101/gr.6675307</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative gene expression profiling of in vitro differentiated megakaryocytes and erythroblasts identifies novel activatory and inhibitory platelet membrane proteins.</h4>
<p class="authors"><span class="author">Macaulay IC</span>, <span class="author">Tijssen MR</span>, <span class="author">Thijssen-Timmer DC</span>, <span class="author">Gusnanto A</span>, <span class="author">Steward M</span>, <span class="author">Burns P</span>, <span class="author">Langford CF</span>, <span class="author">Ellis PD</span>, <span class="author">Dudbridge F</span>, <span class="author">Zwaginga JJ</span>, <span class="author">Watkins NA</span>, <span class="author">van der Schoot CE</span> and <span class="author">Ouwehand WH</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, Cambridge, UK.</p><p class="abstract">To identify previously unknown platelet receptors we compared the transcriptomes of in vitro differentiated megakaryocytes (MKs) and erythroblasts (EBs). RNA was obtained from purified, biologically paired MK and EB cultures and compared using cDNA microarrays. Bioinformatical analysis of MK-up-regulated genes identified 151 transcripts encoding transmembrane domain-containing proteins. Although many of these were known platelet genes, a number of previously unidentified or poorly characterized transcripts were also detected. Many of these transcripts, including G6b, G6f, LRRC32, LAT2, and the G protein-coupled receptor SUCNR1, encode proteins with structural features or functions that suggest they may be involved in the modulation of platelet function. Immunoblotting on platelets confirmed the presence of the encoded proteins, and flow cytometric analysis confirmed the expression of G6b, G6f, and LRRC32 on the surface of platelets. Through comparative analysis of expression in platelets and other blood cells we demonstrated that G6b, G6f, and LRRC32 are restricted to the platelet lineage, whereas LAT2 and SUCNR1 were also detected in other blood cells. The identification of the succinate receptor SUCNR1 in platelets is of particular interest, because physiologically relevant concentrations of succinate were shown to potentiate the effect of low doses of a variety of platelet agonists.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U105260799</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2007</span>;<span class="volume">109</span>;<span class="number">8</span>;<span class="pagerange">3260-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17192395">17192395</a>; DOI: <a href="http://dx.doi.org/10.1182%2Fblood-2006-07-036269">10.1182/blood-2006-07-036269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Karyotype evolution in Rhinolophus bats (Rhinolophidae, Chiroptera) illuminated by cross-species chromosome painting and G-banding comparison.</h4>
<p class="authors"><span class="author">Mao X</span>, <span class="author">Nie W</span>, <span class="author">Wang J</span>, <span class="author">Su W</span>, <span class="author">Ao L</span>, <span class="author">Feng Q</span>, <span class="author">Wang Y</span>, <span class="author">Volleth M</span> and <span class="author">Yang F</span></p>
<p class="affiliation">Key Laboratory of Cellular and Molecular Evolution, Kunming Institute of Zoology, Kunming, Yunnan, PR China.</p><p class="abstract">Rhinolophus (Rhinolophidae) is the second most speciose genus in Chiroptera and has extensively diversified diploid chromosome numbers (from 2n = 28 to 62). In spite of many attempts to explore the karyotypic evolution of this genus, most studies have been based on conventional Giemsa staining rather than G-banding. Here we have made a whole set of chromosome-specific painting probes from flow-sorted chromosomes of Aselliscus stoliczkanus (Hipposideridae). These probes have been utilized to establish the first genome-wide homology maps among six Rhinolophus species with four different diploid chromosome numbers (2n = 36, 44, 58, and 62) and three species from other families: Rousettus leschenaulti (2n = 36, Pteropodidae), Hipposideros larvatus (2n = 32, Hipposideridae), and Myotis altarium (2n = 44, Vespertilionidae) by fluorescence in situ hybridization. To facilitate integration with published maps, human paints were also hybridized to A. stoliczkanus chromosomes. Our painting results substantiate the wide occurrence of whole-chromosome arm conservation in Rhinolophus bats and suggest that Robertsonian translocations of different combinations account for their karyotype differences. Parsimony analysis using chromosomal characters has provided some new insights into the Rhinolophus ancestral karyotype and phylogenetic relationships among these Rhinolophus species so far studied. In addition to Robertsonian translocations, our results suggest that whole-arm (reciprocal) translocations involving multiple non-homologous chromosomes as well could have been involved in the karyotypic evolution within Rhinolophus, in particular those bats with low and medium diploid numbers.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">7</span>;<span class="pagerange">835-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17899409" rel="external">17899409</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1167-5" rel="external">10.1007/s10577-007-1167-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shaken not stirred: a global research cocktail served in Hinxton.</h4>
<p class="authors"><span class="author">Marguerat S</span>, <span class="author">Wilhelm BT</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">A report of the 2007 Cold Spring Harbor Laboratory/Wellcome Trust Conference on Functional Genomics and Systems Biology, Hinxton, UK, 10-13 October 2007.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">320</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18036269" rel="external">18036269</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258181" rel="external">2258181</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-11-320" rel="external">10.1186/gb-2007-8-11-320</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Breaking the waves: improved detection of copy number variation from microarray-based comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Marioni JC</span>, <span class="author">Thorne NP</span>, <span class="author">Valsesia A</span>, <span class="author">Fitzgerald T</span>, <span class="author">Redon R</span>, <span class="author">Fiegler H</span>, <span class="author">Andrews TD</span>, <span class="author">Stranger BE</span>, <span class="author">Lynch AG</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Carter NP</span>, <span class="author">Tavar&eacute; S</span> and <span class="author">Hurles ME</span></p>
<p class="affiliation">Computational Biology Group, Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Centre for Mathematical Sciences, Wilberforce Road, Cambridge CB3 0WA, UK. <a href="mailto:%4A.%4D%61%72%69%6F%6E%69%40%64%61%6D%74%70.%63%61%6D.%61%63.%75%6B">&#74;&#46;&#77;&#97;&#114;&#105;&#111;&#110;&#105;&#64;&#100;&#97;&#109;&#116;&#112;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Large-scale high throughput studies using microarray technology have established that copy number variation (CNV) throughout the genome is more frequent than previously thought. Such variation is known to play an important role in the presence and development of phenotypes such as HIV-1 infection and Alzheimer&#39;s disease. However, methods for analyzing the complex data produced and identifying regions of CNV are still being refined.</p><p class="abstract"><strong>Results</strong>: We describe the presence of a genome-wide technical artifact, spatial autocorrelation or &#39;wave&#39;, which occurs in a large dataset used to determine the location of CNV across the genome. By removing this artifact we are able to obtain both a more biologically meaningful clustering of the data and an increase in the number of CNVs identified by current calling methods without a major increase in the number of false positives detected. Moreover, removing this artifact is critical for the development of a novel model-based CNV calling algorithm - CNVmix - that uses cross-sample information to identify regions of the genome where CNVs occur. For regions of CNV that are identified by both CNVmix and current methods, we demonstrate that CNVmix is better able to categorize samples into groups that represent copy number gains or losses.</p><p class="abstract"><strong>Conclusion</strong>: Removing artifactual &#39;waves&#39; (which appear to be a general feature of array comparative genomic hybridization (aCGH) datasets) and using cross-sample information when identifying CNVs enables more biological information to be extracted from aCGH experiments designed to investigate copy number variation in normal individuals.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">R228</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17961237">17961237</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2246302">2246302</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-10-r228">10.1186/gb-2007-8-10-r228</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Renin enhancer is crucial for full response in Renin expression to an in vivo stimulus.</h4>
<p class="authors"><span class="author">Markus MA</span>, <span class="author">Goy C</span>, <span class="author">Adams DJ</span>, <span class="author">Lovicu FJ</span> and <span class="author">Morris BJ</span></p>
<p class="affiliation">Basic &amp; Clinical Genomics Laboratory, School of Medical Sciences and Bosch Institute, Building F13, University of Sydney, NSW 2006, Australia.</p><p class="abstract">We showed recently that deletion of a strong enhancer located 2.7 kb upstream of the renin gene in mice produces a strain with mild hypotension and salt-sensitivity. Here we set out to compare responses in renin expression in kidney and extrarenal tissues in these &quot;REKO&quot; mice. REKO and wild-type mice were placed on a low NaCl/enalapril regimen for 1 week, and then Ren-1(c) mRNA and renin enzyme activities were measured in tissues and plasma. In untreated REKO mice, renin and Ren-1(c) mRNA were reduced significantly in kidney, submandibular gland, adrenal, heart, and brain. In situ hybridization indicated a marked reduction in Ren-1(c) mRNA in juxtaglomerular cells and granular ducts of submandibular gland. After the chronic stimulus response in renal Ren-1(c) mRNA in REKO mice was blunted by 54% compared with wild-type mice, and was accompanied by almost complete exhaustion of renin stores. Response in plasma renin was blunted by 47%, this being mirrored in heart (54% decline), in which renin is derived mostly from the bloodstream. In adrenal a 55% reduction was seen. These data are consistent with inability of REKO mice to adequately replenish renal renin stores during chronic stimulation of renin secretion. In conclusion, the renin enhancer is critical for replenishment of renin stores and response in renin to a chronic in vivo stimulus.</p>
<p class="publication"><span class="title">Hypertension</span>&nbsp;<span class="year">2007</span>;<span class="volume">50</span>;<span class="number">5</span>;<span class="pagerange">933-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17846348" rel="external">17846348</a>; DOI: <a href="http://dx.doi.org/10.1161%2FHYPERTENSIONAHA.107.096891" rel="external">10.1161/HYPERTENSIONAHA.107.096891</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomally unstable mouse tumours have genomic alterations similar to diverse human cancers.</h4>
<p class="authors"><span class="author">Maser RS</span>, <span class="author">Choudhury B</span>, <span class="author">Campbell PJ</span>, <span class="author">Feng B</span>, <span class="author">Wong KK</span>, <span class="author">Protopopov A</span>, <span class="author">O&#39;Neil J</span>, <span class="author">Gutierrez A</span>, <span class="author">Ivanova E</span>, <span class="author">Perna I</span>, <span class="author">Lin E</span>, <span class="author">Mani V</span>, <span class="author">Jiang S</span>, <span class="author">McNamara K</span>, <span class="author">Zaghlul S</span>, <span class="author">Edkins S</span>, <span class="author">Stevens C</span>, <span class="author">Brennan C</span>, <span class="author">Martin ES</span>, <span class="author">Wiedemeyer R</span>, <span class="author">Kabbarah O</span>, <span class="author">Nogueira C</span>, <span class="author">Histen G</span>, <span class="author">Aster J</span>, <span class="author">Mansour M</span>, <span class="author">Duke V</span>, <span class="author">Foroni L</span>, <span class="author">Fielding AK</span>, <span class="author">Goldstone AH</span>, <span class="author">Rowe JM</span>, <span class="author">Wang YA</span>, <span class="author">Look AT</span>, <span class="author">Stratton MR</span>, <span class="author">Chin L</span>, <span class="author">Futreal PA</span> and <span class="author">DePinho RA</span></p>
<p class="affiliation">Department of Medical Oncology, Dana Farber Cancer Institute, Boston, Massachusetts 02115, USA.</p><p class="abstract">Highly rearranged and mutated cancer genomes present major challenges in the identification of pathogenetic events driving the neoplastic transformation process. Here we engineered lymphoma-prone mice with chromosomal instability to assess the usefulness of mouse models in cancer gene discovery and the extent of cross-species overlap in cancer-associated copy number aberrations. Along with targeted re-sequencing, our comparative oncogenomic studies identified FBXW7 and PTEN to be commonly deleted both in murine lymphomas and in human T-cell acute lymphoblastic leukaemia/lymphoma (T-ALL). The murine cancers acquire widespread recurrent amplifications and deletions targeting loci syntenic to those not only in human T-ALL but also in diverse human haematopoietic, mesenchymal and epithelial tumours. These results indicate that murine and human tumours experience common biological processes driven by orthologous genetic events in their malignant evolution. The highly concordant nature of genomic events encourages the use of genomically unstable murine cancer models in the discovery of biological driver events in the human oncogenome.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500389</span>; Wellcome Trust: <span class="grant">077012</span>, <span class="grant">088340</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">447</span>;<span class="number">7147</span>;<span class="pagerange">966-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17515920">17515920</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2714968">2714968</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature05886">10.1038/nature05886</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transcriptional regulatory network for sexual differentiation in fission yeast.</h4>
<p class="authors"><span class="author">Mata J</span>, <span class="author">Wilbrey A</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">Cancer Research UK Fission Yeast Functional Genomics Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1HH, UK. <a href="mailto:%6A%6D%35%39%33%40%63%61%6D.%61%63.%75%6B">&#106;&#109;&#53;&#57;&#51;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Changes in gene expression are hallmarks of cellular differentiation. Sexual differentiation in fission yeast (Schizosaccharomyces pombe) provides a model system for gene expression programs accompanying and driving cellular specialization. The expression of hundreds of genes is modulated in successive waves during meiosis and sporulation in S. pombe, and several known transcription factors are critical for these processes.</p><p class="abstract"><strong>Results</strong>: We used DNA microarrays to investigate meiotic gene regulation by examining transcriptomes after genetic perturbations (gene deletion and/or overexpression) of rep1, mei4, atf21 and atf31, which encode known transcription factors controlling sexual differentiation. This analysis reveals target genes at a genome-wide scale and uncovers combinatorial control by Atf21p and Atf31p. We also studied two transcription factors not previously implicated in sexual differentiation whose meiotic induction depended on Mei4p: Rsv2p induces stress-related genes during spore formation, while Rsv1p represses glucose-metabolism genes. Our data further reveal negative feedback interactions: both Rep1p and Mei4p not only activate specific gene expression waves (early and middle genes, respectively) but are also required for repression of genes induced in the previous waves (Ste11p-dependent and early genes, respectively).</p><p class="abstract"><strong>Conclusion</strong>: These data give insight into regulatory principles controlling the extensive gene expression program driving sexual differentiation and highlight sophisticated interactions and combinatorial control among transcription factors. Besides triggering simultaneous expression of gene waves, transcription factors also repress genes in the previous wave and induce other factors that in turn regulate a subsequent wave. These dependencies ensure an ordered and timely succession of transcriptional waves during cellular differentiation.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">10</span>;<span class="pagerange">R217</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17927811" rel="external">17927811</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2246291" rel="external">2246291</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-10-r217" rel="external">10.1186/gb-2007-8-10-r217</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic relatedness of the Streptococcus pneumoniae capsular biosynthetic loci.</h4>
<p class="authors"><span class="author">Mavroidi A</span>, <span class="author">Aanensen DM</span>, <span class="author">Godoy D</span>, <span class="author">Skovsted IC</span>, <span class="author">Kaltoft MS</span>, <span class="author">Reeves PR</span>, <span class="author">Bentley SD</span> and <span class="author">Spratt BG</span></p>
<p class="affiliation">Department of Infectious Disease Epidemiology, Imperial College London, Room G22, Old Medical School Building, St. Mary&#39;s Hospital, Norfolk Place, London W2 1PG, United Kingdom.</p><p class="abstract">Streptococcus pneumoniae (the pneumococcus) produces 1 of 91 capsular polysaccharides (CPS) that define the serotype. The cps loci of 88 pneumococcal serotypes whose CPS is synthesized by the Wzy-dependent pathway were compared with each other and with additional streptococcal polysaccharide biosynthetic loci and were clustered according to the proportion of shared homology groups (HGs), weighted for the sequence similarities between the genes encoding the shared HGs. The cps loci of the 88 pneumococcal serotypes were distributed into eight major clusters and 21 subclusters. All serotypes within the same serogroup fell into the same major cluster, but in six cases, serotypes within the same serogroup were in different subclusters and, conversely, nine subclusters included completely different serotypes. The closely related cps loci within a subcluster were compared to the known CPS structures to relate gene content to structure. The Streptococcus oralis and Streptococcus mitis polysaccharide biosynthetic loci clustered within the pneumococcal cps loci and were in a subcluster that also included the cps locus of pneumococcal serotype 21, whereas the Streptococcus agalactiae cps loci formed a single cluster that was not closely related to any of the pneumococcal cps clusters.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">21</span>;<span class="pagerange">7841-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17766424">17766424</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2168730">2168730</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00836-07">10.1128/JB.00836-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic and molecular analysis of the central and peripheral circadian clockwork of mice.</h4>
<p class="authors"><span class="author">Maywood ES</span>, <span class="author">O&#39;Neill JS</span>, <span class="author">Reddy AB</span>, <span class="author">Chesham JE</span>, <span class="author">Prosser HM</span>, <span class="author">Kyriacou CP</span>, <span class="author">Godinho SI</span>, <span class="author">Nolan PM</span> and <span class="author">Hastings MH</span></p>
<p class="affiliation">Division of Neurobiology, MRC Laboratory of Molecular Biology, Cambridge CB2 0QH, United Kingdom.</p><p class="abstract">A hierarchy of interacting, tissue-based clocks controls circadian physiology and behavior in mammals. Preeminent are the suprachiasmatic nuclei (SCN): central hypothalamic pacemakers synchronized to solar time via retinal afferents and in turn responsible for internal synchronization of other clocks present in major organ systems. The SCN and peripheral clocks share essentially the same cellular timing mechanism. This consists of autoregulatory transcriptional/posttranslational feedback loops in which the Period (Per) and Cryptochrome (Cry) &quot;clock&quot; genes are negatively regulated by their protein products. Here, we review recent studies directed at understanding the molecular and cellular bases to the mammalian clock. At the cellular level, we demonstrate the role of F-box protein Fbxl3 (characterized by the afterhours mutation) in directing the proteasomal degradation of Cry and thereby controlling negative feedback and circadian period of the molecular loops. Within SCN neural circuitry, we describe how neuropeptidergic signaling by VIP synchronizes and sustains the cellular clocks. At the hypothalamic level, signaling via a different SCN neuropeptide, prokineticin, is not required for pacemaking but is necessary for control of circadian behavior. Finally, we consider how metabolic pathways are coordinated in time, focusing on liver function and the role of glucocorticoid signals in driving the circadian transcriptome and proteome.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council; Medical Research Council; Wellcome Trust</p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">72</span>;<span class="pagerange">85-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18419265" rel="external">18419265</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fsqb.2007.72.005" rel="external">10.1101/sqb.2007.72.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prediction of microRNA targets.</h4>
<p class="authors"><span class="author">Mazi&egrave;re P</span> and <span class="author">Enright AJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Recently, microRNAs (miRNAs) have been shown to be important regulators of genes in many organisms and have already been implicated in a growing number of diseases. MiRNAs are short (21-23 nucleotides) RNAs that bind to the 3&#39; untranslated regions of target genes. This binding event causes translational repression of the target gene and, evidence now suggests, also stimulates rapid degradation of the target transcript. miRNAs represent a new species of regulator, controlling the levels of potentially large numbers of proteins, many of which might be important drug targets. The expression of miRNAs shows that they are highly differentially expressed, with specific miRNAs active in certain tissues at certain times. In many cancers, miRNA expression is significantly altered, and this has been shown to be a useful diagnostic tool. Several computational approaches have been developed for the prediction of miRNA targets.</p>
<p class="publication"><span class="title">Drug discovery today</span>&nbsp;<span class="year">2007</span>;<span class="volume">12</span>;<span class="number">11-12</span>;<span class="pagerange">452-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17532529" rel="external">17532529</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.drudis.2007.04.002" rel="external">10.1016/j.drudis.2007.04.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Spreading of mammalian DNA-damage response factors studied by ChIP-chip at damaged telomeres.</h4>
<p class="authors"><span class="author">Meier A</span>, <span class="author">Fiegler H</span>, <span class="author">Mu&ntilde;oz P</span>, <span class="author">Ellis P</span>, <span class="author">Rigler D</span>, <span class="author">Langford C</span>, <span class="author">Blasco MA</span>, <span class="author">Carter N</span> and <span class="author">Jackson SP</span></p>
<p class="affiliation">The Wellcome Trust and Cancer Research UK Gurdon Institute, Department of Zoology, University of Cambridge, Cambridge, UK.</p><p class="abstract">Phosphorylated histone H2AX (gammaH2AX) is generated in nucleosomes flanking sites of DNA double-strand breaks, triggering the recruitment of DNA-damage response proteins such as MDC1 and 53BP1. Here, we study shortened telomeres in senescent human cells. We show that most telomeres trigger gammaH2AX formation, which spreads up to 570 kb into the subtelomeric regions. Furthermore, we reveal that the spreading patterns of 53BP1 and MDC1 are very similar to that of gammaH2AX, consistent with a structural link between these factors. Moreover, different subsets of telomeres signal in different cell lines, with those that signal tending to equate to the shortest telomeres of the corresponding cell line, thus linking telomere attrition with DNA-damage signalling. Notably, we find that, in some cases, gammaH2AX spreading is modulated in a manner suggesting that H2AX distribution or its ability to be phosphorylated is not uniform along the chromosome. Finally, we observe weak gammaH2AX signals at telomeres of proliferating cells, but not in hTERT immortalised cells, suggesting that low telomerase activity leads to telomere uncapping and senescence in proliferating primary cells.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A5290</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">11</span>;<span class="pagerange">2707-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17491589" rel="external">17491589</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1888678" rel="external">1888678</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.emboj.7601719" rel="external">10.1038/sj.emboj.7601719</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Report of a female patient with mental retardation and tall stature due to a chromosomal rearrangement disrupting the OPHN1 gene on Xq12.</h4>
<p class="authors"><span class="author">Menten B</span>, <span class="author">Buysse K</span>, <span class="author">Vermeulen S</span>, <span class="author">Meersschaut V</span>, <span class="author">Vandesompele J</span>, <span class="author">Ng BL</span>, <span class="author">Carter NP</span>, <span class="author">Mortier GR</span> and <span class="author">Speleman F</span></p>
<p class="affiliation">Center for Medical Genetics, Ghent University Hospital, Ghent, Belgium. <a href="mailto:%62%6A%6F%72%6E.%6D%65%6E%74%65%6E%40%75%67%65%6E%74.%62%65">&#98;&#106;&#111;&#114;&#110;&#46;&#109;&#101;&#110;&#116;&#101;&#110;&#64;&#117;&#103;&#101;&#110;&#116;&#46;&#98;&#101;</a></p><p class="abstract">We report on a patient with mental retardation, seizures and tall stature with advanced bone age in whom a de novo apparently balanced chromosomal rearrangement 46,XX,t(X;9)(q12;p13.3) was identified. Using array CGH on flow-sorted derivative chromosomes (array painting) and subsequent FISH and qPCR analysis, we mapped and sequenced both breakpoints. The Xq12 breakpoint was located within the gene coding for oligophrenin 1 (OPHN1) whereas the 9p13.3 breakpoint was assigned to a non-coding segment within a gene dense region. Disruption of OPHN1 by the Xq12 breakpoint was considered the major cause of the abnormal phenotype observed in the proband.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">European journal of medical genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">50</span>;<span class="number">6</span>;<span class="pagerange">446-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17845870" rel="external">17845870</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2688819" rel="external">2688819</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ejmg.2007.07.003" rel="external">10.1016/j.ejmg.2007.07.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lamin A/C polymorphisms, type 2 diabetes, and the metabolic syndrome: case-control and quantitative trait studies.</h4>
<p class="authors"><span class="author">Mesa JL</span>, <span class="author">Loos RJ</span>, <span class="author">Franks PW</span>, <span class="author">Ong KK</span>, <span class="author">Luan J</span>, <span class="author">O&#39;Rahilly S</span>, <span class="author">Wareham NJ</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Medical Research Center Epidemiology Unit, Cambridge, U.K.</p><p class="abstract">Mutations in the LMNA gene, encoding the nuclear envelope protein lamin A/C, are responsible for a number of distinct disease entities including Dunnigan-type familial partial lipodystrophy. Dunningan-type lipodystrophy is characterized by loss of subcutaneous adipose tissue, insulin resistance, dyslipidemia, and type 2 diabetes and shares many of the features of the metabolic syndrome. Furthermore, several genome-wide linkage scans for type 2 diabetes have found evidence of linkage at chromosome 1q21.2, the region that harbors the LMNA gene. Therefore, LMNA is a biological and positional candidate for type 2 diabetes susceptibility. Previous studies have reported association between a common LMNA variant (1908C&gt;T; rs4641) and adverse metabolic traits in ethnically diverse populations from Asia and North America. In the present study, we characterized the common variation across the LMNA gene (including rs4641) and tested for association with type 2 diabetes in two large case-control studies (n = 2,052) and with features of the metabolic syndrome in a separate cohort study (n = 1,572). Despite our study being sufficiently powered to detect effects similar and even smaller in magnitude than those previously reported, none of the LMNA single nucleotide polymorphisms were statistically significantly associated with type 2 diabetes or the metabolic syndrome. Thus, it appears unlikely that variation at LMNA substantially increases the risk of type 2 diabetes or related traits in U.K. Europids.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106179472</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">3</span>;<span class="pagerange">884-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17327461">17327461</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2668858">2668858</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb06-1055">10.2337/db06-1055</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome of the marsupial Monodelphis domestica reveals innovation in non-coding sequences.</h4>
<p class="authors"><span class="author">Mikkelsen TS</span>, <span class="author">Wakefield MJ</span>, <span class="author">Aken B</span>, <span class="author">Amemiya CT</span>, <span class="author">Chang JL</span>, <span class="author">Duke S</span>, <span class="author">Garber M</span>, <span class="author">Gentles AJ</span>, <span class="author">Goodstadt L</span>, <span class="author">Heger A</span>, <span class="author">Jurka J</span>, <span class="author">Kamal M</span>, <span class="author">Mauceli E</span>, <span class="author">Searle SM</span>, <span class="author">Sharpe T</span>, <span class="author">Baker ML</span>, <span class="author">Batzer MA</span>, <span class="author">Benos PV</span>, <span class="author">Belov K</span>, <span class="author">Clamp M</span>, <span class="author">Cook A</span>, <span class="author">Cuff J</span>, <span class="author">Das R</span>, <span class="author">Davidow L</span>, <span class="author">Deakin JE</span>, <span class="author">Fazzari MJ</span>, <span class="author">Glass JL</span>, <span class="author">Grabherr M</span>, <span class="author">Greally JM</span>, <span class="author">Gu W</span>, <span class="author">Hore TA</span>, <span class="author">Huttley GA</span>, <span class="author">Kleber M</span>, <span class="author">Jirtle RL</span>, <span class="author">Koina E</span>, <span class="author">Lee JT</span>, <span class="author">Mahony S</span>, <span class="author">Marra MA</span>, <span class="author">Miller RD</span>, <span class="author">Nicholls RD</span>, <span class="author">Oda M</span>, <span class="author">Papenfuss AT</span>, <span class="author">Parra ZE</span>, <span class="author">Pollock DD</span>, <span class="author">Ray DA</span>, <span class="author">Schein JE</span>, <span class="author">Speed TP</span>, <span class="author">Thompson K</span>, <span class="author">VandeBerg JL</span>, <span class="author">Wade CM</span>, <span class="author">Walker JA</span>, <span class="author">Waters PD</span>, <span class="author">Webber C</span>, <span class="author">Weidman JR</span>, <span class="author">Xie X</span>, <span class="author">Zody MC</span>, <span class="author">Broad Institute Genome Sequencing Platform</span>, <span class="author">Broad Institute Whole Genome Assembly Team</span>, <span class="author">Graves JA</span>, <span class="author">Ponting CP</span>, <span class="author">Breen M</span>, <span class="author">Samollow PB</span>, <span class="author">Lander ES</span> and <span class="author">Lindblad-Toh K</span></p>
<p class="affiliation">Broad Institute of MIT and Harvard, 7 Cambridge Center, Cambridge, Massachusetts 02142, USA. <a href="mailto:%6B%65%72%73%6C%69%40%62%72%6F%61%64.%6D%69%74.%65%64%75">&#107;&#101;&#114;&#115;&#108;&#105;&#64;&#98;&#114;&#111;&#97;&#100;&#46;&#109;&#105;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">We report a high-quality draft of the genome sequence of the grey, short-tailed opossum (Monodelphis domestica). As the first metatherian (&#39;marsupial&#39;) species to be sequenced, the opossum provides a unique perspective on the organization and evolution of mammalian genomes. Distinctive features of the opossum chromosomes provide support for recent theories about genome evolution and function, including a strong influence of biased gene conversion on nucleotide sequence composition, and a relationship between chromosomal characteristics and X chromosome inactivation. Comparison of opossum and eutherian genomes also reveals a sharp difference in evolutionary innovation between protein-coding and non-coding functional elements. True innovation in protein-coding genes seems to be relatively rare, with lineage-specific differences being largely due to diversification and rapid turnover in gene families involved in environmental interactions. In contrast, about 20% of eutherian conserved non-coding elements (CNEs) are recent inventions that postdate the divergence of Eutheria and Metatheria. A substantial proportion of these eutherian-specific CNEs arose from sequence inserted by transposable elements, pointing to transposons as a major creative force in the evolution of mammalian gene regulation.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U137761446</span>; Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">447</span>;<span class="number">7141</span>;<span class="pagerange">167-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17495919">17495919</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature05805">10.1038/nature05805</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pfam: a domain-centric method for analyzing proteins and proteomes.</h4>
<p class="authors"><span class="author">Mistry J</span> and <span class="author">Finn R</span></p><p class="abstract">The constant deluge of genome sequencing data means that annotating, classifying, and comparing proteins or proteomes can seam like an endless task. Furthermore, discovering and accessing such data is fundamental to biologists. There are, however, databases that perform these tasks. Pfam, a protein families database, is one such database. In this chapter, the use of the web interface to Pfam and the resources provided (annotation, sequence alignments, phylogenetic trees, profile hidden Markov models [HMMs]) are described. The exploitation of tools for searching sequences against the library of Pfam HMMs, searching for domain combinations, searching by taxonomy, browsing proteomes, and comparing proteomes are outlined in detail.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">396</span>;<span class="pagerange">43-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18025685" rel="external">18025685</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Predicting active site residue annotations in the Pfam database.</h4>
<p class="authors"><span class="author">Mistry J</span>, <span class="author">Bateman A</span> and <span class="author">Finn RD</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6A%6D%31%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#109;&#49;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Approximately 5% of Pfam families are enzymatic, but only a small fraction of the sequences within these families (&lt;0.5%) have had the residues responsible for catalysis determined. To increase the active site annotations in the Pfam database, we have developed a strict set of rules, chosen to reduce the rate of false positives, which enable the transfer of experimentally determined active site residue data to other sequences within the same Pfam family.</p><p class="abstract"><strong>Description</strong>: We have created a large database of predicted active site residues. On comparing our active site predictions to those found in UniProtKB, Catalytic Site Atlas, PROSITE and MEROPS we find that we make many novel predictions. On investigating the small subset of predictions made by these databases that are not predicted by us, we found these sequences did not meet our strict criteria for prediction. We assessed the sensitivity and specificity of our methodology and estimate that only 3% of our predicted sequences are false positives.</p><p class="abstract"><strong>Conclusion</strong>: We have predicted 606110 active site residues, of which 94% are not found in UniProtKB, and have increased the active site annotations in Pfam by more than 200 fold. Although implemented for Pfam, the tool we have developed for transferring the data can be applied to any alignment with associated experimental active site data and is available for download. Our active site predictions are re-calculated at each Pfam release to ensure they are comprehensive and up to date. They provide one of the largest available databases of active site annotation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">298</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17688688" rel="external">17688688</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2025603" rel="external">2025603</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-8-298" rel="external">10.1186/1471-2105-8-298</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential var gene expression in the organs of patients dying of falciparum malaria.</h4>
<p class="authors"><span class="author">Montgomery J</span>, <span class="author">Mphande FA</span>, <span class="author">Berriman M</span>, <span class="author">Pain A</span>, <span class="author">Rogerson SJ</span>, <span class="author">Taylor TE</span>, <span class="author">Molyneux ME</span> and <span class="author">Craig A</span></p>
<p class="affiliation">Malawi-Liverpool-Wellcome Programme of Clinical Tropical Research, College of Medicine, Blantyre, Malawi. <a href="mailto:%6A%6D%6F%6E%74%67%6F%6D%65%72%79%40%6D%6C%77.%6D%65%64%63%6F%6C.%6D%77">&#106;&#109;&#111;&#110;&#116;&#103;&#111;&#109;&#101;&#114;&#121;&#64;&#109;&#108;&#119;&#46;&#109;&#101;&#100;&#99;&#111;&#108;&#46;&#109;&#119;</a></p><p class="abstract">Sequestration of parasitized erythrocytes in the microcirculation of tissues is thought to be important in the pathogenesis of severe falciparum malaria. A major variant surface antigen, var/Plasmodium falciparum erythrocyte membrane protein 1, expressed on the surface of the infected erythrocyte, mediates cytoadherence to vascular endothelium. To address the question of tissue-specific accumulation of variant types, we used the unique resource generated by the clinicopathological study of fatal paediatric malaria in Blantyre, Malawi, to analyse var gene transcription in patients dying with falciparum malaria. Despite up to 102 different var genes being expressed by P. falciparum populations in a single host, only one to two of these genes were expressed at high levels in the brains and hearts of these patients. These major var types differed between organs. However, identical var types were expressed in the brains of multiple patients from a single malaria season. These results provide the first evidence of organ-specific accumulation of P. falciparum variant types and suggest that parasitized erythrocytes can exhibit preferential binding in the body, supporting the hypothesis of cytoadherence-linked pathogenesis.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01 AI034969-10A1</span>, <span class="grant">R01 AI34969</span>; Wellcome Trust: <span class="grant">042390</span>, <span class="grant">071376</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">65</span>;<span class="number">4</span>;<span class="pagerange">959-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17617167" rel="external">17617167</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2170262" rel="external">2170262</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2007.05837.x" rel="external">10.1111/j.1365-2958.2007.05837.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A survey of genomic properties for the detection of regulatory polymorphisms.</h4>
<p class="authors"><span class="author">Montgomery SB</span>, <span class="author">Griffith OL</span>, <span class="author">Schuetz JM</span>, <span class="author">Brooks-Wilson A</span> and <span class="author">Jones SJ</span></p>
<p class="affiliation">Canada&#39;s Michael Smith Genome Sciences Centre, British Columbia Cancer Agency, Vancouver, British Columbia, Canada. <a href="mailto:%73%6D%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#109;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Advances in the computational identification of functional noncoding polymorphisms will aid in cataloging novel determinants of health and identifying genetic variants that explain human evolution. To date, however, the development and evaluation of such techniques has been limited by the availability of known regulatory polymorphisms. We have attempted to address this by assembling, from the literature, a computationally tractable set of regulatory polymorphisms within the ORegAnno database (http://www.oreganno.org). We have further used 104 regulatory single-nucleotide polymorphisms from this set and 951 polymorphisms of unknown function, from 2-kb and 152-bp noncoding upstream regions of genes, to investigate the discriminatory potential of 23 properties related to gene regulation and population genetics. Among the most important properties detected in this region are distance to transcription start site, local repetitive content, sequence conservation, minor and derived allele frequencies, and presence of a CpG island. We further used the entire set of properties to evaluate their collective performance in detecting regulatory polymorphisms. Using a 10-fold cross-validation approach, we were able to achieve a sensitivity and specificity of 0.82 and 0.71, respectively, and we show that this performance is strongly influenced by the distance to the transcription start site.</p>
<p class="publication"><span class="title">PLoS computational biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">3</span>;<span class="number">6</span>;<span class="pagerange">e106</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17559298" rel="external">17559298</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1892352" rel="external">1892352</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pcbi.0030106" rel="external">10.1371/journal.pcbi.0030106</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clustering of phosphorylation site recognition motifs can be exploited to predict the targets of cyclin-dependent kinase.</h4>
<p class="authors"><span class="author">Moses AM</span>, <span class="author">H&eacute;rich&eacute; JK</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1HH, UK. <a href="mailto:%61%6D%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#109;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Protein kinases are critical to cellular signalling and post-translational gene regulation, but their biological substrates are difficult to identify. We show that cyclin-dependent kinase (CDK) consensus motifs are frequently clustered in CDK substrate proteins. Based on this, we introduce a new computational strategy to predict the targets of CDKs and use it to identify new biologically interesting candidates. Our data suggest that regulatory modules may exist in protein sequence as clusters of short sequence motifs.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">R23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17316440" rel="external">17316440</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1852407" rel="external">1852407</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-2-r23" rel="external">10.1186/gb-2007-8-2-r23</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulatory evolution in proteins by turnover and lineage-specific changes of cyclin-dependent kinase consensus sites.</h4>
<p class="authors"><span class="author">Moses AM</span>, <span class="author">Liku ME</span>, <span class="author">Li JJ</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1HH, United Kingdom. <a href="mailto:%61%6D%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#109;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Evolutionary change in gene regulation is a key mechanism underlying the genetic component of organismal diversity. Here, we study evolution of regulation at the posttranslational level by examining the evolution of cyclin-dependent kinase (CDK) consensus phosphorylation sites in the protein subunits of the pre-replicative complex (RC). The pre-RC, an assembly of proteins formed during an early stage of DNA replication, is believed to be regulated by CDKs throughout the animals and fungi. Interestingly, although orthologous pre-RC components often contain clusters of CDK consensus sites, the positions and numbers of sites do not seem conserved. By analyzing protein sequences from both distantly and closely related species, we confirm that consensus sites can turn over rapidly even when the local cluster of sites is preserved, consistent with the notion that precise positioning of phosphorylation events is not required for regulation. We also identify evolutionary changes in the clusters of sites and further examine one replication protein, Mcm3, where a cluster of consensus sites near a nucleocytoplasmic transport signal is confined to a specific lineage. We show that the presence or absence of the cluster of sites in different species is associated with differential regulation of the transport signal. These findings suggest that the CDK regulation of MCM nuclear localization was acquired in the lineage leading to Saccharomyces cerevisiae after the divergence with Candida albicans. Our results begin to explore the dynamics of regulatory evolution at the posttranslational level and show interesting similarities to recent observations of regulatory evolution at the level of transcription.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">5F31CA110268-03</span>; NIGMS NIH HHS: <span class="grant">R01 GM59704</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">45</span>;<span class="pagerange">17713-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17978194" rel="external">17978194</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2077061" rel="external">2077061</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0700997104" rel="external">10.1073/pnas.0700997104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical assessment of methods of protein structure prediction-Round VII.</h4>
<p class="authors"><span class="author">Moult J</span>, <span class="author">Fidelis K</span>, <span class="author">Kryshtafovych A</span>, <span class="author">Rost B</span>, <span class="author">Hubbard T</span> and <span class="author">Tramontano A</span></p>
<p class="affiliation">Center for Advanced Research in Biotechnology, University of Maryland Biotechnology Institute, Rockville, Maryland 20850, USA. <a href="mailto:%6D%6F%75%6C%74%40%75%6D%62%69.%75%6D%64.%65%64%75">&#109;&#111;&#117;&#108;&#116;&#64;&#117;&#109;&#98;&#105;&#46;&#117;&#109;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">This paper is an introduction to the supplemental issue of the journal PROTEINS, dedicated to the seventh CASP experiment to assess the state of the art in protein structure prediction. The paper describes the conduct of the experiment, the categories of prediction included, and outlines the evaluation and assessment procedures. Highlights are improvements in model accuracy relative to that obtainable from knowledge of a single best template structure; convergence of the accuracy of models produced by automatic servers toward that produced by human modeling teams; the emergence of methods for predicting the quality of models; and rapidly increasing practical applications of the methods.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM072354</span>; NLM NIH HHS: <span class="grant">LM07085</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2007</span>;<span class="volume">69 Suppl 8</span>;<span class="pagerange">3-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17918729">17918729</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2653632">2653632</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fprot.21767">10.1002/prot.21767</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mouse Phenotype Database Integration Consortium: integration [corrected] of mouse phenome data resources.</h4>
<p class="authors"><span class="author">Mouse Phenotype Database Integration Consortium</span>, <span class="author">Hancock JM</span>, <span class="author">Adams NC</span>, <span class="author">Aidinis V</span>, <span class="author">Blake A</span>, <span class="author">Bogue M</span>, <span class="author">Brown SD</span>, <span class="author">Chesler EJ</span>, <span class="author">Davidson D</span>, <span class="author">Duran C</span>, <span class="author">Eppig JT</span>, <span class="author">Gailus-Durner V</span>, <span class="author">Gates H</span>, <span class="author">Gkoutos GV</span>, <span class="author">Greenaway S</span>, <span class="author">Hrab&eacute; de Angelis M</span>, <span class="author">Kollias G</span>, <span class="author">Leblanc S</span>, <span class="author">Lee K</span>, <span class="author">Lengger C</span>, <span class="author">Maier H</span>, <span class="author">Mallon AM</span>, <span class="author">Masuya H</span>, <span class="author">Melvin DG</span>, <span class="author">M&uuml;ller W</span>, <span class="author">Parkinson H</span>, <span class="author">Proctor G</span>, <span class="author">Reuveni E</span>, <span class="author">Schofield P</span>, <span class="author">Shukla A</span>, <span class="author">Smith C</span>, <span class="author">Toyoda T</span>, <span class="author">Vasseur L</span>, <span class="author">Wakana S</span>, <span class="author">Walling A</span>, <span class="author">White J</span>, <span class="author">Wood J</span> and <span class="author">Zouberakis M</span></p><p class="abstract">Understanding the functions encoded in the mouse genome will be central to an understanding of the genetic basis of human disease. To achieve this it will be essential to be able to characterize the phenotypic consequences of variation and alterations in individual genes. Data on the phenotypes of mouse strains are currently held in a number of different forms (detailed descriptions of mouse lines, first-line phenotyping data on novel mutations, data on the normal features of inbred lines) at many sites worldwide. For the most efficient use of these data sets, we have initiated a process to develop standards for the description of phenotypes (using ontologies) and file formats for the description of phenotyping protocols and phenotype data sets. This process is ongoing and needs to be supported by the wider mouse genetics and phenotyping communities to succeed. We invite interested parties to contact us as we develop this process further.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U127527203</span>, <span class="grant">MC_U142684171</span>, <span class="grant">MC_U142684172</span>, <span class="grant">MC_U142684175</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2007</span>;<span class="volume">18</span>;<span class="number">3</span>;<span class="pagerange">157-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17436037">17436037</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-007-9004-x">10.1007/s00335-007-9004-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New developments in the InterPro database.</h4>
<p class="authors"><span class="author">Mulder NJ</span>, <span class="author">Apweiler R</span>, <span class="author">Attwood TK</span>, <span class="author">Bairoch A</span>, <span class="author">Bateman A</span>, <span class="author">Binns D</span>, <span class="author">Bork P</span>, <span class="author">Buillard V</span>, <span class="author">Cerutti L</span>, <span class="author">Copley R</span>, <span class="author">Courcelle E</span>, <span class="author">Das U</span>, <span class="author">Daugherty L</span>, <span class="author">Dibley M</span>, <span class="author">Finn R</span>, <span class="author">Fleischmann W</span>, <span class="author">Gough J</span>, <span class="author">Haft D</span>, <span class="author">Hulo N</span>, <span class="author">Hunter S</span>, <span class="author">Kahn D</span>, <span class="author">Kanapin A</span>, <span class="author">Kejariwal A</span>, <span class="author">Labarga A</span>, <span class="author">Langendijk-Genevaux PS</span>, <span class="author">Lonsdale D</span>, <span class="author">Lopez R</span>, <span class="author">Letunic I</span>, <span class="author">Madera M</span>, <span class="author">Maslen J</span>, <span class="author">McAnulla C</span>, <span class="author">McDowall J</span>, <span class="author">Mistry J</span>, <span class="author">Mitchell A</span>, <span class="author">Nikolskaya AN</span>, <span class="author">Orchard S</span>, <span class="author">Orengo C</span>, <span class="author">Petryszak R</span>, <span class="author">Selengut JD</span>, <span class="author">Sigrist CJ</span>, <span class="author">Thomas PD</span>, <span class="author">Valentin F</span>, <span class="author">Wilson D</span>, <span class="author">Wu CH</span> and <span class="author">Yeats C</span></p>
<p class="affiliation">EMBL Outstation-European Bioinformatics Institute Hinxton, Cambridge, UK. <a href="mailto:%6D%75%6C%64%65%72%40%65%62%69.%61%63.%75%6B">&#109;&#117;&#108;&#100;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">InterPro is an integrated resource for protein families, domains and functional sites, which integrates the following protein signature databases: PROSITE, PRINTS, ProDom, Pfam, SMART, TIGRFAMs, PIRSF, SUPERFAMILY, Gene3D and PANTHER. The latter two new member databases have been integrated since the last publication in this journal. There have been several new developments in InterPro, including an additional reading field, new database links, extensions to the web interface and additional match XML files. InterPro has always provided matches to UniProtKB proteins on the website and in the match XML file on the FTP site. Additional matches to proteins in UniParc (UniProt archive) are now available for download in the new match XML files only. The latest InterPro release (13.0) contains more than 13 000 entries, covering over 78% of all proteins in UniProtKB. The database is available for text- and sequence-based searches via a webserver (http://www.ebi.ac.uk/interpro), and for download by anonymous FTP (ftp://ftp.ebi.ac.uk/pub/databases/interpro). The InterProScan search tool is now also available via a web service at http://www.ebi.ac.uk/Tools/webservices/WSInterProScan.html.</p>
<p class="grants">Funded by: Biotechnology and Biological Sciences Research Council: <span class="grant">BB/F010435/1</span>; Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2007</span>;<span class="volume">35</span>;<span class="number">Database issue</span>;<span class="pagerange">D224-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17202162">17202162</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1899100">1899100</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkl841">10.1093/nar/gkl841</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Localization of type 1 diabetes susceptibility to the MHC class I genes HLA-B and HLA-A.</h4>
<p class="authors"><span class="author">Nejentsev S</span>, <span class="author">Howson JM</span>, <span class="author">Walker NM</span>, <span class="author">Szeszko J</span>, <span class="author">Field SF</span>, <span class="author">Stevens HE</span>, <span class="author">Reynolds P</span>, <span class="author">Hardy M</span>, <span class="author">King E</span>, <span class="author">Masters J</span>, <span class="author">Hulme J</span>, <span class="author">Maier LM</span>, <span class="author">Smyth D</span>, <span class="author">Bailey R</span>, <span class="author">Cooper JD</span>, <span class="author">Ribas G</span>, <span class="author">Campbell RD</span>, <span class="author">Clayton DG</span>, <span class="author">Todd JA</span> and <span class="author">Wellcome Trust Case Control Consortium</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute of Medical Research, University of Cambridge, CB2 0XY, UK.</p><p class="abstract">The major histocompatibility complex (MHC) on chromosome 6 is associated with susceptibility to more common diseases than any other region of the human genome, including almost all disorders classified as autoimmune. In type 1 diabetes the major genetic susceptibility determinants have been mapped to the MHC class II genes HLA-DQB1 and HLA-DRB1 (refs 1-3), but these genes cannot completely explain the association between type 1 diabetes and the MHC region. Owing to the region&#39;s extreme gene density, the multiplicity of disease-associated alleles, strong associations between alleles, limited genotyping capability, and inadequate statistical approaches and sample sizes, which, and how many, loci within the MHC determine susceptibility remains unclear. Here, in several large type 1 diabetes data sets, we analyse a combined total of 1,729 polymorphisms, and apply statistical methods-recursive partitioning and regression-to pinpoint disease susceptibility to the MHC class I genes HLA-B and HLA-A (risk ratios &gt;1.5; P(combined) = 2.01 x 10(-19) and 2.35 x 10(-13), respectively) in addition to the established associations of the MHC class II genes. Other loci with smaller and/or rarer effects might also be involved, but to find these, future searches must take into account both the HLA class II and class I genes and use even larger samples. Taken together with previous studies, we conclude that MHC-class-I-mediated events, principally involving HLA-B*39, contribute to the aetiology of type 1 diabetes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0600681</span>; Wellcome Trust: <span class="grant">076113</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">450</span>;<span class="number">7171</span>;<span class="pagerange">887-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18004301">18004301</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2703779">2703779</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06406">10.1038/nature06406</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and association analysis of the type 1 diabetes-linked region on chromosome 10p12-q11.</h4>
<p class="authors"><span class="author">Nejentsev S</span>, <span class="author">Smink LJ</span>, <span class="author">Smyth D</span>, <span class="author">Bailey R</span>, <span class="author">Lowe CE</span>, <span class="author">Payne F</span>, <span class="author">Masters J</span>, <span class="author">Godfrey L</span>, <span class="author">Lam A</span>, <span class="author">Burren O</span>, <span class="author">Stevens H</span>, <span class="author">Nutland S</span>, <span class="author">Walker NM</span>, <span class="author">Smith A</span>, <span class="author">Twells R</span>, <span class="author">Barratt BJ</span>, <span class="author">Wright C</span>, <span class="author">French L</span>, <span class="author">Chen Y</span>, <span class="author">Deloukas P</span>, <span class="author">Rogers J</span>, <span class="author">Dunham I</span> and <span class="author">Todd JA</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Cambridge, UK. <a href="mailto:%73%65%72%67%65%79.%6E%65%6A%65%6E%74%73%65%76%40%63%69%6D%72.%63%61%6D.%61%63.%75%6B">&#115;&#101;&#114;&#103;&#101;&#121;&#46;&#110;&#101;&#106;&#101;&#110;&#116;&#115;&#101;&#118;&#64;&#99;&#105;&#109;&#114;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: In an effort to locate susceptibility genes for type 1 diabetes (T1D) several genome-wide linkage scans have been undertaken. A chromosomal region designated IDDM10 retained genome-wide significance in a combined analysis of the main linkage scans. Here, we studied sequence polymorphisms in 23 Mb on chromosome 10p12-q11, including the putative IDDM10 region, to identify genes associated with T1D.</p><p class="abstract"><strong>Results</strong>: Initially, we resequenced the functional candidate genes, CREM and SDF1, located in this region, genotyped 13 tag single nucleotide polymorphisms (SNPs) and found no association with T1D. We then undertook analysis of the whole 23 Mb region. We constructed and sequenced a contig tile path from two bacterial artificial clone libraries. By comparison with a clone library from an unrelated person used in the Human Genome Project, we identified 12,058 SNPs. We genotyped 303 SNPs and 25 polymorphic microsatellite markers in 765 multiplex T1D families and followed up 22 associated polymorphisms in up to 2,857 families. We found nominal evidence of association in six loci (P = 0.05 - 0.0026), located near the PAPD1 gene. Therefore, we resequenced 38.8 kb in this region, found 147 SNPs and genotyped 84 of them in the T1D families. We also tested 13 polymorphisms in the PAPD1 gene and in five other loci in 1,612 T1D patients and 1,828 controls from the UK. Overall, only the D10S193 microsatellite marker located 28 kb downstream of PAPD1 showed nominal evidence of association in both T1D families and in the case-control sample (P = 0.037 and 0.03, respectively).</p><p class="abstract"><strong>Conclusion</strong>: We conclude that polymorphisms in the CREM and SDF1 genes have no major effect on T1D. The weak T1D association that we detected in the association scan near the PAPD1 gene may be either false or due to a small genuine effect, and cannot explain linkage at the IDDM10 region.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">068545/Z/02</span></p>
<p class="publication"><span class="title">BMC genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17509149" rel="external">17509149</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1885446" rel="external">1885446</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2156-8-24" rel="external">10.1186/1471-2156-8-24</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Modeling insertional mutagenesis using gene length and expression in murine embryonic stem cells.</h4>
<p class="authors"><span class="author">Nord AS</span>, <span class="author">Vranizan K</span>, <span class="author">Tingley W</span>, <span class="author">Zambon AC</span>, <span class="author">Hanspers K</span>, <span class="author">Fong LG</span>, <span class="author">Hu Y</span>, <span class="author">Bacchetti P</span>, <span class="author">Ferrin TE</span>, <span class="author">Babbitt PC</span>, <span class="author">Doniger SW</span>, <span class="author">Skarnes WC</span>, <span class="author">Young SG</span> and <span class="author">Conklin BR</span></p>
<p class="affiliation">Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, USA. <a href="mailto:%6E%6F%72%64%61%6C%65%78%40%75.%77%61%73%68%69%6E%67%74%6F%6E.%65%64%75">&#110;&#111;&#114;&#100;&#97;&#108;&#101;&#120;&#64;&#117;&#46;&#119;&#97;&#115;&#104;&#105;&#110;&#103;&#116;&#111;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: High-throughput mutagenesis of the mammalian genome is a powerful means to facilitate analysis of gene function. Gene trapping in embryonic stem cells (ESCs) is the most widely used form of insertional mutagenesis in mammals. However, the rules governing its efficiency are not fully understood, and the effects of vector design on the likelihood of gene-trapping events have not been tested on a genome-wide scale.</p><p class="abstract">In this study, we used public gene-trap data to model gene-trap likelihood. Using the association of gene length and gene expression with gene-trap likelihood, we constructed spline-based regression models that characterize which genes are susceptible and which genes are resistant to gene-trapping techniques. We report results for three classes of gene-trap vectors, showing that both length and expression are significant determinants of trap likelihood for all vectors. Using our models, we also quantitatively identified hotspots of gene-trap activity, which represent loci where the high likelihood of vector insertion is controlled by factors other than length and expression. These formalized statistical models describe a high proportion of the variance in the likelihood of a gene being trapped by expression-dependent vectors and a lower, but still significant, proportion of the variance for vectors that are predicted to be independent of endogenous gene expression.</p><p class="abstract">The findings of significant expression and length effects reported here further the understanding of the determinants of vector insertion. Results from this analysis can be applied to help identify other important determinants of this important biological phenomenon and could assist planning of large-scale mutagenesis efforts.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG002766</span>; NHLBI NIH HHS: <span class="grant">HL66621</span></p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2007</span>;<span class="volume">2</span>;<span class="number">7</span>;<span class="pagerange">e617</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17637833">17637833</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1910612">1910612</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0000617">10.1371/journal.pone.0000617</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A second large plasmid encodes conjugative transfer and antimicrobial resistance in O119:H2 and some typical O111 enteropathogenic Escherichia coli strains.</h4>
<p class="authors"><span class="author">Nwaneshiudu AI</span>, <span class="author">Mucci T</span>, <span class="author">Pickard DJ</span> and <span class="author">Okeke IN</span></p>
<p class="affiliation">Department of Biology, Haverford College, 370 Lancaster Avenue, Haverford, PA 19041, USA.</p><p class="abstract">A novel and functional conjugative transfer system identified in O119:H2 enteropathogenic Escherichia coli (EPEC) strain MB80 by subtractive hybridization is encoded on a large multidrug resistance plasmid, distinct from the well-described EPEC adherence factor (EAF) plasmid. Variants of the MB80 conjugative resistance plasmid were identified in other EPEC strains, including the prototypical O111:NM strain B171, from which the EAF plasmid has been sequenced. This separate large plasmid and the selective advantage that it confers in the antibiotic era have been overlooked because it comigrates with the virulence plasmid on conventional gels.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2007</span>;<span class="volume">189</span>;<span class="number">16</span>;<span class="pagerange">6074-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17573481" rel="external">17573481</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1952026" rel="external">1952026</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.00349-07" rel="external">10.1128/JB.00349-07</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Slicer-independent role for Argonaute 2 in hematopoiesis and the microRNA pathway.</h4>
<p class="authors"><span class="author">O&#39;Carroll D</span>, <span class="author">Mecklenbrauker I</span>, <span class="author">Das PP</span>, <span class="author">Santana A</span>, <span class="author">Koenig U</span>, <span class="author">Enright AJ</span>, <span class="author">Miska EA</span> and <span class="author">Tarakhovsky A</span></p>
<p class="affiliation">The Laboratory for Lymphocyte Signaling, The Rockefeller University, New York, New York 10021, USA. <a href="mailto:%64%6F%6E%61%6C.%6F%63%61%72%72%6F%6C%6C%40%65%6D%62%6C-%6D%6F%6E%74%65%72%6F%74%6F%6E%64%6F.%69%74">&#100;&#111;&#110;&#97;&#108;&#46;&#111;&#99;&#97;&#114;&#114;&#111;&#108;&#108;&#64;&#101;&#109;&#98;&#108;&#45;&#109;&#111;&#110;&#116;&#101;&#114;&#111;&#116;&#111;&#110;&#100;&#111;&#46;&#105;&#116;</a></p><p class="abstract">Binding of microRNA (miRNA) to mRNA within the RNA-induced silencing complex (RISC) leads to either translational inhibition or to destruction of the target mRNA. Both of these functions are executed by Argonaute 2 (Ago2). Using hematopoiesis in mice as a model system to study the physiological function of Ago2 in vivo, we found that Ago2 controls early development of lymphoid and erythroid cells. We show that the unique and defining feature of Ago2, the Slicer endonuclease activity, is dispensable for hematopoiesis. Instead, we identified Ago2 as a key regulator of miRNA homeostasis. Deficiency in Ago2 impairs miRNA biogenesis from precursor-miRNAs followed by a reduction in miRNA expression levels. Collectively, our data identify Ago2 as a highly specialized member of the Argonaute family with an essential nonredundant Slicer-independent function within the mammalian miRNA pathway.</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2007</span>;<span class="volume">21</span>;<span class="number">16</span>;<span class="pagerange">1999-2004</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17626790" rel="external">17626790</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1948855" rel="external">1948855</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.1565607" rel="external">10.1101/gad.1565607</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of secretory granule phosphatidylinositol 4,5-bisphosphate-interacting proteins using an affinity pulldown strategy.</h4>
<p class="authors"><span class="author">Osborne SL</span>, <span class="author">Wallis TP</span>, <span class="author">Jimenez JL</span>, <span class="author">Gorman JJ</span> and <span class="author">Meunier FA</span></p>
<p class="affiliation">Molecular Dynamics of Synaptic Function Laboratory, School of Biomedical Sciences, University of Queensland, St. Lucia, Queensland 4072, Australia.</p><p class="abstract">Phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2) synthesis is required for calcium-dependent exocytosis in neurosecretory cells. We developed a PtdIns(4,5)P2 bead pulldown strategy combined with subcellular fractionation to identify endogenous chromaffin granule proteins that interact with PtdIns(4,5)P2. We identified two synaptotagmin isoforms, synaptotagmins 1 and 7; spectrin; alpha-adaptin; and synaptotagmin-like protein 4 (granuphilin) by mass spectrometry and Western blotting. The interaction between synaptotagmin 7 and PtdIns(4,5)P2 and its functional relevance was investigated. The 45-kDa isoform of synaptotagmin 7 was found to be highly expressed in adrenal chromaffin cells compared with PC12 cells and to mainly localize to secretory granules by subcellular fractionation, immunoisolation, and immunocytochemistry. We demonstrated that synaptotagmin 7 binds PtdIns(4,5)P2 via the C2B domain in the absence of calcium and via both the C2A and C2B domains in the presence of calcium. We mutated the polylysine stretch in synaptotagmin 7 C2B and demonstrated that this mutant domain lacks the calcium-independent PtdIns(4,5)P2 binding. Synaptotagmin 7 C2B domain inhibited catecholamine release from digitonin-permeabilized chromaffin cells, and this inhibition was abrogated with the C2B polylysine mutant. These data indicate that synaptotagmin 7 C2B-effector interactions, which occur via the polylysine stretch, including calcium-independent PtdIns(4,5)P2 binding, are important for chromaffin granule exocytosis.</p>
<p class="publication"><span class="title">Molecular &amp; cellular proteomics : MCP</span>&nbsp;<span class="year">2007</span>;<span class="volume">6</span>;<span class="number">7</span>;<span class="pagerange">1158-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17449848" rel="external">17449848</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fmcp.M600430-MCP200" rel="external">10.1074/mcp.M600430-MCP200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variation in the LMNA gene (encoding lamin A/C) and type 2 diabetes: association analyses in 9,518 subjects.</h4>
<p class="authors"><span class="author">Owen KR</span>, <span class="author">Groves CJ</span>, <span class="author">Hanson RL</span>, <span class="author">Knowler WC</span>, <span class="author">Shuldiner AR</span>, <span class="author">Elbein SC</span>, <span class="author">Mitchell BD</span>, <span class="author">Froguel P</span>, <span class="author">Ng MC</span>, <span class="author">Chan JC</span>, <span class="author">Jia W</span>, <span class="author">Deloukas P</span>, <span class="author">Hitman GA</span>, <span class="author">Walker M</span>, <span class="author">Frayling TM</span>, <span class="author">Hattersley AT</span>, <span class="author">Zeggini E</span> and <span class="author">McCarthy MI</span></p>
<p class="affiliation">Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Old Road, Headington, Oxford OX3 7LJ, U.K. <a href="mailto:%6B%61%74%68%61%72%69%6E%65.%6F%77%65%6E%40%64%72%6C.%6F%78.%61%63.%75%6B">&#107;&#97;&#116;&#104;&#97;&#114;&#105;&#110;&#101;&#46;&#111;&#119;&#101;&#110;&#64;&#100;&#114;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Mutations in the LMNA gene (encoding lamin A/C) underlie familial partial lipodystrophy, a syndrome of monogenic insulin resistance and diabetes. LMNA maps to the well-replicated diabetes-linkage region on chromosome 1q, and there are reported associations between LMNA single nucleotide polymorphisms (SNPs) (particularly rs4641; H566H) and metabolic syndrome components. We examined the relationship between LMNA variation and type 2 diabetes (using six tag SNPs capturing &gt;90% of common variation) in several large datasets. Analysis of 2,490 U.K. diabetic case and 2,556 control subjects revealed no significant associations at either genotype or haplotype level: the minor allele at rs4641 was no more frequent in case subjects (allelic odds ratio [OR] 1.07 [95% CI 0.98-1.17], P = 0.15). In 390 U.K. trios, family-based association analyses revealed nominally significant overtransmission of the major allele at rs12063564 (P = 0.01), which was not corroborated in other samples. Finally, genotypes for 2,817 additional subjects from the International 1q Consortium revealed no consistent case-control or family-based associations with LMNA variants. Across all our data, the OR for the rs4641 minor allele approached but did not attain significance (1.07 [0.99-1.15], P = 0.08). Our data do not therefore support a major effect of LMNA variation on diabetes risk. However, in a meta-analysis including other available data, there is evidence that rs4641 has a modest effect on diabetes susceptibility (1.10 [1.04-1.16], P = 0.001).</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">T32 AG 00219</span>; NIDDK NIH HHS: <span class="grant">K24 DK 2673</span>, <span class="grant">R01 DK 073490</span>, <span class="grant">R01 DK 39311</span>, <span class="grant">R01 DK 54261</span>, <span class="grant">U01 DK 58026</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">079557</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">3</span>;<span class="pagerange">879-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17327460" rel="external">17327460</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672988" rel="external">2672988</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb06-0930" rel="external">10.2337/db06-0930</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome painting among Proboscidea, Hyracoidea and Sirenia: support for Paenungulata (Afrotheria, Mammalia) but not Tethytheria.</h4>
<p class="authors"><span class="author">Pardini AT</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Fu B</span>, <span class="author">Bonde RK</span>, <span class="author">Elder FF</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Yang F</span> and <span class="author">Robinson TJ</span></p>
<p class="affiliation">Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Private Bag X1, Matieland, 7602 Stellenbosch, South Africa.</p><p class="abstract">Despite marked improvements in the interpretation of systematic relationships within Eutheria, particular nodes, including Paenungulata (Hyracoidea, Sirenia and Proboscidea), remain ambiguous. The combination of a rapid radiation, a deep divergence and an extensive morphological diversification has resulted in a limited phylogenetic signal confounding resolution within this clade both at the morphological and nucleotide levels. Cross-species chromosome painting was used to delineate regions of homology between Loxodonta africana (2n=56), Procavia capensis (2n=54), Trichechus manatus latirostris (2n=48) and an outgroup taxon, the aardvark (Orycteropus afer, 2n=20). Changes specific to each lineage were identified and although the presence of a minimum of 11 synapomorphies confirmed the monophyly of Paenungulata, no change characterizing intrapaenungulate relationships was evident. The reconstruction of an ancestral paenungulate karyotype and the estimation of rates of chromosomal evolution indicate a reduced rate of genomic repatterning following the paenungulate radiation. In comparison to data available for other mammalian taxa, the paenungulate rate of chromosomal evolution is slow to moderate. As a consequence, the absence of a chromosomal character uniting two paenungulates (at the level of resolution characterized in this study) may be due to a reduced rate of chromosomal change relative to the length of time separating successive divergence events.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings. Biological sciences / The Royal Society</span>&nbsp;<span class="year">2007</span>;<span class="volume">274</span>;<span class="number">1615</span>;<span class="pagerange">1333-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17374594" rel="external">17374594</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1914331" rel="external">1914331</a>; DOI: <a href="http://dx.doi.org/10.1098%2Frspb.2007.0088" rel="external">10.1098/rspb.2007.0088</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence variants in the autophagy gene IRGM and multiple other replicating loci contribute to Crohn&#39;s disease susceptibility.</h4>
<p class="authors"><span class="author">Parkes M</span>, <span class="author">Barrett JC</span>, <span class="author">Prescott NJ</span>, <span class="author">Tremelling M</span>, <span class="author">Anderson CA</span>, <span class="author">Fisher SA</span>, <span class="author">Roberts RG</span>, <span class="author">Nimmo ER</span>, <span class="author">Cummings FR</span>, <span class="author">Soars D</span>, <span class="author">Drummond H</span>, <span class="author">Lees CW</span>, <span class="author">Khawaja SA</span>, <span class="author">Bagnall R</span>, <span class="author">Burke DA</span>, <span class="author">Todhunter CE</span>, <span class="author">Ahmad T</span>, <span class="author">Onnie CM</span>, <span class="author">McArdle W</span>, <span class="author">Strachan D</span>, <span class="author">Bethel G</span>, <span class="author">Bryan C</span>, <span class="author">Lewis CM</span>, <span class="author">Deloukas P</span>, <span class="author">Forbes A</span>, <span class="author">Sanderson J</span>, <span class="author">Jewell DP</span>, <span class="author">Satsangi J</span>, <span class="author">Mansfield JC</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Cardon L</span> and <span class="author">Mathew CG</span></p>
<p class="affiliation">Inflammatory Bowel Disease Research Group, Addenbrooke&#39;s Hospital, University of Cambridge, Cambridge CB2 2QQ, UK. <a href="mailto:%6D%69%6C%65%73.%70%61%72%6B%65%73%40%61%64%64%65%6E%62%72%6F%6F%6B%65%73.%6E%68%73.%75%6B">&#109;&#105;&#108;&#101;&#115;&#46;&#112;&#97;&#114;&#107;&#101;&#115;&#64;&#97;&#100;&#100;&#101;&#110;&#98;&#114;&#111;&#111;&#107;&#101;&#115;&#46;&#110;&#104;&#115;&#46;&#117;&#107;</a></p><p class="abstract">A genome-wide association scan in individuals with Crohn&#39;s disease by the Wellcome Trust Case Control Consortium detected strong association at four novel loci. We tested 37 SNPs from these and other loci for association in an independent case-control sample. We obtained replication for the autophagy-inducing IRGM gene on chromosome 5q33.1 (replication P = 6.6 x 10(-4), combined P = 2.1 x 10(-10)) and for nine other loci, including NKX2-3, PTPN2 and gene deserts on chromosomes 1q and 5p13.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">072029</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7</span>;<span class="pagerange">830-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17554261">17554261</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2628541">2628541</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2061">10.1038/ng2061</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Randomized controlled comparison of ofloxacin, azithromycin, and an ofloxacin-azithromycin combination for treatment of multidrug-resistant and nalidixic acid-resistant typhoid fever.</h4>
<p class="authors"><span class="author">Parry CM</span>, <span class="author">Ho VA</span>, <span class="author">Phuong le T</span>, <span class="author">Bay PV</span>, <span class="author">Lanh MN</span>, <span class="author">Tung le T</span>, <span class="author">Tham NT</span>, <span class="author">Wain J</span>, <span class="author">Hien TT</span> and <span class="author">Farrar JJ</span></p>
<p class="affiliation">Oxford University Clinical Research Unit, Ho Chi Minh City, Vietnam. <a href="mailto:%63%6D%70%61%72%72%79%40%6C%69%76.%61%63.%75%6B">&#99;&#109;&#112;&#97;&#114;&#114;&#121;&#64;&#108;&#105;&#118;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Isolates of Salmonella enterica serovar Typhi that are multidrug resistant (MDR, resistant to chloramphenicol, ampicillin, and trimethoprim-sulfamethoxazole) and have reduced susceptibility to fluoroquinolones (nalidixic acid resistant, Na(r)) are common in Asia. The optimum treatment for infections caused by such isolates is not established. This study compared different antimicrobial regimens for the treatment of MDR/Na(r) typhoid fever. Vietnamese children and adults with uncomplicated typhoid fever were entered into an open randomized controlled trial. Ofloxacin (20 mg/kg of body weight/day for 7 days), azithromycin (10 mg/kg/day for 7 days), and ofloxacin (15 mg/kg/day for 7 days) combined with azithromycin (10 mg/kg/day for the first 3 days) were compared. Of the 241 enrolled patients, 187 were eligible for analysis (186 S. enterica serovar Typhi, 1 Salmonella enterica serovar Paratyphi A). Eighty-seven percent (163/187) of the patients were children; of the S. enterica serovar Typhi isolates, 88% (165/187) were MDR and 93% (173/187) were Na(r). The clinical cure rate was 64% (40/63) with ofloxacin, 76% (47/62) with ofloxacin-azithromycin, and 82% (51/62) with azithromycin (P = 0.053). The mean (95% confidence interval [CI]) fever clearance time for patients treated with azithromycin (5.8 days [5.1 to 6.5 days]) was shorter than that for patients treated with ofloxacin-azithromycin (7.1 days [6.2 to 8.1 days]) and ofloxacin (8.2 days [7.2 to 9.2 days]) (P &lt; 0.001). Positive fecal carriage immediately posttreatment was detected in 19.4% (12/62) of patients treated with ofloxacin, 6.5% (4/62) of those treated with the combination, and 1.6% (1/62) of those treated with azithromycin (P = 0.006). Both antibiotics were well tolerated. Uncomplicated typhoid fever due to isolates of MDR S. enterica serovar Typhi with reduced susceptibility to fluoroquinolones (Na(r)) can be successfully treated with a 7-day course of azithromycin.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Antimicrobial agents and chemotherapy</span>&nbsp;<span class="year">2007</span>;<span class="volume">51</span>;<span class="number">3</span>;<span class="pagerange">819-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17145784" rel="external">17145784</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1803150" rel="external">1803150</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAAC.00447-06" rel="external">10.1128/AAC.00447-06</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction analysis of the CBLB and CTLA4 genes in type 1 diabetes.</h4>
<p class="authors"><span class="author">Payne F</span>, <span class="author">Cooper JD</span>, <span class="author">Walker NM</span>, <span class="author">Lam AC</span>, <span class="author">Smink LJ</span>, <span class="author">Nutland S</span>, <span class="author">Stevens HE</span>, <span class="author">Hutchings J</span> and <span class="author">Todd JA</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust, Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Addenbrooke&#39;s Hospital, Cambridge, UK.</p><p class="abstract">Gene-gene interaction analyses have been suggested as a potential strategy to help identify common disease susceptibility genes. Recently, evidence of a statistical interaction between polymorphisms in two negative immunoregulatory genes, CBLB and CTLA4, has been reported in type 1 diabetes (T1D). This study, in 480 Danish families, reported an association between T1D and a synonymous coding SNP in exon 12 of the CBLB gene (rs3772534 G&gt;A; minor allele frequency, MAF=0.24; derived relative risk, RR for G allele=1.78; P=0.046). Furthermore, evidence of a statistical interaction with the known T1D susceptibility-associated CTLA4 polymorphism rs3087243 (laboratory name CT60, G&gt;A) was reported (P&lt;0.0001), such that the CBLB SNP rs3772534 G allele was overtransmitted to offspring with the CTLA4 rs3087243 G/G genotype. We have, therefore, attempted to obtain additional support for this finding in both large family and case-control collections. In a primary analysis, no evidence for an association of the CBLB SNP rs3772534 with disease was found in either sample set (2162 parent-child trios, P=0.33; 3453 cases and 3655 controls, P=0.69). In the case-only statistical interaction analysis between rs3772534 and rs3087243, there was also no support for an effect (1994 T1D affected offspring, and 3215 cases, P=0.92). These data highlight the need for large, well-characterized populations, offering the possibility of obtaining additional support for initial observations owing to the low prior probability of identifying reproducible evidence of gene-gene interactions in the analysis of common disease-associated variants in human populations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of leukocyte biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">81</span>;<span class="number">3</span>;<span class="pagerange">581-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17209142">17209142</a>; DOI: <a href="http://dx.doi.org/10.1189%2Fjlb.0906577">10.1189/jlb.0906577</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The practical implications of comparative kinetoplastid genomics.</h4>
<p class="authors"><span class="author">Peacock CS</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, UK.</p>
<p class="publication"><span class="title">SEB experimental biology series</span>&nbsp;<span class="year">2007</span>;<span class="volume">58</span>;<span class="pagerange">25-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17608236" rel="external">17608236</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic analysis of three Leishmania species that cause diverse human disease.</h4>
<p class="authors"><span class="author">Peacock CS</span>, <span class="author">Seeger K</span>, <span class="author">Harris D</span>, <span class="author">Murphy L</span>, <span class="author">Ruiz JC</span>, <span class="author">Quail MA</span>, <span class="author">Peters N</span>, <span class="author">Adlem E</span>, <span class="author">Tivey A</span>, <span class="author">Aslett M</span>, <span class="author">Kerhornou A</span>, <span class="author">Ivens A</span>, <span class="author">Fraser A</span>, <span class="author">Rajandream MA</span>, <span class="author">Carver T</span>, <span class="author">Norbertczak H</span>, <span class="author">Chillingworth T</span>, <span class="author">Hance Z</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Ormond D</span>, <span class="author">Rutter S</span>, <span class="author">Squares R</span>, <span class="author">Whitehead S</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Arrowsmith C</span>, <span class="author">White B</span>, <span class="author">Thurston S</span>, <span class="author">Bringaud F</span>, <span class="author">Baldauf SL</span>, <span class="author">Faulconbridge A</span>, <span class="author">Jeffares D</span>, <span class="author">Depledge DP</span>, <span class="author">Oyola SO</span>, <span class="author">Hilley JD</span>, <span class="author">Brito LO</span>, <span class="author">Tosi LR</span>, <span class="author">Barrell B</span>, <span class="author">Cruz AK</span>, <span class="author">Mottram JC</span>, <span class="author">Smith DF</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%63%73%70%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#115;&#112;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Leishmania parasites cause a broad spectrum of clinical disease. Here we report the sequencing of the genomes of two species of Leishmania: Leishmania infantum and Leishmania braziliensis. The comparison of these sequences with the published genome of Leishmania major reveals marked conservation of synteny and identifies only approximately 200 genes with a differential distribution between the three species. L. braziliensis, contrary to Leishmania species examined so far, possesses components of a putative RNA-mediated interference pathway, telomere-associated transposable elements and spliced leader-associated SLACS retrotransposons. We show that pseudogene formation and gene loss are the principal forces shaping the different genomes. Genes that are differentially distributed between the species encode proteins implicated in host-pathogen interactions and parasite survival in the macrophage.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000508</span>; Wellcome Trust: <span class="grant">076355</span>, <span class="grant">085775</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7</span>;<span class="pagerange">839-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17572675">17572675</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2592530">2592530</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2053">10.1038/ng2053</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Angiotensin II increases chloride absorption in the cortical collecting duct in mice through a pendrin-dependent mechanism.</h4>
<p class="authors"><span class="author">Pech V</span>, <span class="author">Kim YH</span>, <span class="author">Weinstein AM</span>, <span class="author">Everett LA</span>, <span class="author">Pham TD</span> and <span class="author">Wall SM</span></p>
<p class="affiliation">Renal Division, Department of Medicine, Emory University School of Medicine, Atlanta, Georgia 30322, USA. <a href="mailto:%76%70%65%63%68%40%65%6D%6F%72%79.%65%64%75">&#118;&#112;&#101;&#99;&#104;&#64;&#101;&#109;&#111;&#114;&#121;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Pendrin (Slc26a4) localizes to type B and non-A, non-B intercalated cells in the distal convoluted tubule, the connecting tubule, and the cortical collecting duct (CCD), where it mediates apical Cl(-)/HCO(3)(-) exchange. The purpose of this study was to determine whether angiotensin II increases transepithelial net chloride transport, J(Cl), in mouse CCD through a pendrin-dependent mechanism. J(Cl) and transepithelial voltage, V(T), were measured in CCDs perfused in vitro from wild-type and Slc26a4 null mice ingesting a NaCl-replete diet or a NaCl-replete diet and furosemide. In CCDs from wild-type mice ingesting a NaCl-replete diet, V(T) and J(Cl) were not different from zero either in the presence or absence of angiotensin II (10(-8) M) in the bath. Thus further experiments employed mice given the high-NaCl diet and furosemide to upregulate renal pendrin expression. CCDs from furosemide-treated wild-type mice had a lumen-negative V(T) and absorbed Cl(-). With angiotensin II in the bath, Cl(-) absorption doubled although V(T) did not become more lumen negative. In contrast, in CCDs from furosemide-treated Slc26a4 null mice, Cl(-) secretion and a V(T) of approximately 0 were observed, neither of which changed with angiotensin II application. Inhibiting ENaC with benzamil abolished V(T) although J(Cl) fell only approximately 50%. Thus substantial Cl(-) absorption is observed in the absence of an electromotive force. Attenuating apical anion exchange with the peritubular application of the H(+)-ATPase inhibitor bafilomycin abolished benzamil-insensitive Cl(-) absorption. In conclusion, angiotensin II increases transcellular Cl(-) absorption in the CCD through a pendrin- and H(+)-ATPase-dependent process.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK-52935</span></p>
<p class="publication"><span class="title">American journal of physiology. Renal physiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">292</span>;<span class="number">3</span>;<span class="pagerange">F914-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17077386" rel="external">17077386</a>; DOI: <a href="http://dx.doi.org/10.1152%2Fajprenal.00361.2006" rel="external">10.1152/ajprenal.00361.2006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Environmental and genetic modifiers of squint penetrance during zebrafish embryogenesis.</h4>
<p class="authors"><span class="author">Pei W</span>, <span class="author">Williams PH</span>, <span class="author">Clark MD</span>, <span class="author">Stemple DL</span> and <span class="author">Feldman B</span></p>
<p class="affiliation">Medical Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA.</p><p class="abstract">The Nodal-related subgroup of the TGFbeta superfamily of secreted cytokines regulates the specification of the mesodermal and endodermal germ layers during gastrulation. Two Nodal-related proteins - Squint (Sqt) and Cyclops (Cyc) - are expressed during germ-layer specification in zebrafish. Genetic sqt mutant phenotypes have defined a variable requirement for zygotic Sqt, but not for maternal Sqt, in midline mesendoderm development. However a comparison of phenotypes arising from oocytes or zygotes injected with Sqt antisense morpholinos has suggested a novel requirement for maternal Sqt in dorsal specification. In this study we examined maternal-zygotic mutants for each of two sqt alleles and we also compared phenotypes of closely related zygotic and maternal-zygotic sqt mutants. Each of these approaches indicated there is no general requirement for maternal Sqt. To better understand the dispensability of maternal and zygotic Sqt, we sought out developmental contexts that more rigorously demand intact Sqt signalling. We found that sqt penetrance is influenced by genetic modifiers, by environmental temperature, by levels of residual Activin-like activity and by Heat-Shock Protein 90 (HSP90) activity. Therefore, Sqt may confer an evolutionary advantage by protecting early-stage embryos against detrimental interacting alleles and environmental challenges.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">Z01 HG200309-05</span>, <span class="grant">Z99 HG999999</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">308</span>;<span class="number">2</span>;<span class="pagerange">368-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17583692">17583692</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1994576">1994576</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2007.05.026">10.1016/j.ydbio.2007.05.026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Diet and the evolution of human amylase gene copy number variation.</h4>
<p class="authors"><span class="author">Perry GH</span>, <span class="author">Dominy NJ</span>, <span class="author">Claw KG</span>, <span class="author">Lee AS</span>, <span class="author">Fiegler H</span>, <span class="author">Redon R</span>, <span class="author">Werner J</span>, <span class="author">Villanea FA</span>, <span class="author">Mountain JL</span>, <span class="author">Misra R</span>, <span class="author">Carter NP</span>, <span class="author">Lee C</span> and <span class="author">Stone AC</span></p>
<p class="affiliation">School of Human Evolution and Social Change, Arizona State University, Tempe, Arizona 85287, USA.</p><p class="abstract">Starch consumption is a prominent characteristic of agricultural societies and hunter-gatherers in arid environments. In contrast, rainforest and circum-arctic hunter-gatherers and some pastoralists consume much less starch. This behavioral variation raises the possibility that different selective pressures have acted on amylase, the enzyme responsible for starch hydrolysis. We found that copy number of the salivary amylase gene (AMY1) is correlated positively with salivary amylase protein level and that individuals from populations with high-starch diets have, on average, more AMY1 copies than those with traditionally low-starch diets. Comparisons with other loci in a subset of these populations suggest that the extent of AMY1 copy number differentiation is highly unusual. This example of positive selection on a copy number-variable gene is, to our knowledge, one of the first discovered in the human genome. Higher AMY1 copy numbers and protein levels probably improve the digestion of starchy foods and may buffer against the fitness-reducing effects of intestinal disease.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">C06 RR014491-01</span>, <span class="grant">C06 RR016483-01</span>, <span class="grant">RR014491</span>, <span class="grant">RR015087</span>, <span class="grant">RR016483</span>, <span class="grant">U42 RR015087-01</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">10</span>;<span class="pagerange">1256-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17828263">17828263</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2377015">2377015</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2123">10.1038/ng2123</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Noncanonical cell death pathways act during Drosophila oogenesis.</h4>
<p class="authors"><span class="author">Peterson JS</span>, <span class="author">Bass BP</span>, <span class="author">Jue D</span>, <span class="author">Rodriguez A</span>, <span class="author">Abrams JM</span> and <span class="author">McCall K</span></p>
<p class="affiliation">Department of Biology, Boston University, Boston, Massachusetts 02115, USA.</p><p class="abstract">Programmed cell death (PCD) is a highly conserved process that occurs during development and in response to adverse conditions. In Drosophila, most PCDs require the genes within the H99 deficiency, the adaptor molecule Ark, and caspases. Here we investigate 10 cell death genes for their potential roles in two distinct types of PCD that occur in oogenesis: developmental nurse cell PCD and starvation-induced PCD. Most of the genes investigated were found to have little effect on late stage developmental PCD in oogenesis, although ark mutants showed a partial inhibition. Mid-stage starvation-induced germline PCD was found to be independent of the upstream activators and ark although it requires caspases, suggesting an apoptosome-independent mechanism of caspase activation in mid-oogenesis. These results indicate that novel pathways must control PCD in the ovary.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM060574-05</span>, <span class="grant">R01 GM072124</span>, <span class="grant">R01 GM60574</span></p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2007</span>;<span class="volume">45</span>;<span class="number">6</span>;<span class="pagerange">396-404</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17506088" rel="external">17506088</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvg.20306" rel="external">10.1002/dvg.20306</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A generalized transducing phage for the murine pathogen Citrobacter rodentium.</h4>
<p class="authors"><span class="author">Petty NK</span>, <span class="author">Toribio AL</span>, <span class="author">Goulding D</span>, <span class="author">Foulds I</span>, <span class="author">Thomson N</span>, <span class="author">Dougan G</span> and <span class="author">Salmond GP</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, Cambridge CB2 1QW, UK.</p><p class="abstract">A virulent phage (phiCR1) capable of generalized transduction in Citrobacter rodentium was isolated from the environment and characterized. C. rodentium is a natural pathogen of mice, causing transmissible murine colonic hyperplasia. Sequencing of its genome has recently been completed and will soon be fully annotated and published. C. rodentium is an important model organism for infections caused by the human pathogens enteropathogenic and enterohaemorrhagic Escherichia coli (EPEC and EHEC). phiCR1 uses a lipopolysaccharide receptor, has a genome size of approximately 300 kb, and is able to transduce a variety of markers. phiCR1 is the first reported transducing phage for C. rodentium and will be a useful tool for functional genomic analysis of this important natural murine pathogen.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">153</span>;<span class="number">Pt 9</span>;<span class="pagerange">2984-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17768241" rel="external">17768241</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2652034" rel="external">2652034</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.2007%2F008888-0" rel="external">10.1099/mic.0.2007/008888-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The SCL transcriptional network and BMP signaling pathway interact to regulate RUNX1 activity.</h4>
<p class="authors"><span class="author">Pimanda JE</span>, <span class="author">Donaldson IJ</span>, <span class="author">de Bruijn MF</span>, <span class="author">Kinston S</span>, <span class="author">Knezevic K</span>, <span class="author">Huckle L</span>, <span class="author">Piltz S</span>, <span class="author">Landry JR</span>, <span class="author">Green AR</span>, <span class="author">Tannahill D</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 2XY, United Kingdom.</p><p class="abstract">Hematopoietic stem cell (HSC) development is regulated by several signaling pathways and a number of key transcription factors, which include Scl/Tal1, Runx1, and members of the Smad family. However, it remains unclear how these various determinants interact. Using a genome-wide computational screen based on the well characterized Scl +19 HSC enhancer, we have identified a related Smad6 enhancer that also targets expression to blood and endothelial cells in transgenic mice. Smad6, Bmp4, and Runx1 transcripts are concentrated along the ventral aspect of the E10.5 dorsal aorta in the aorta-gonad-mesonephros region from which HSCs originate. Moreover, Smad6, an inhibitor of Bmp4 signaling, binds and inhibits Runx1 activity, whereas Smad1, a positive mediator of Bmp4 signaling, transactivates the Runx1 promoter. Taken together, our results integrate three key determinants of HSC development; the Scl transcriptional network, Runx1 activity, and the Bmp4/Smad signaling pathway.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">3</span>;<span class="pagerange">840-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17213321" rel="external">17213321</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1783401" rel="external">1783401</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0607196104" rel="external">10.1073/pnas.0607196104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gata2, Fli1, and Scl form a recursively wired gene-regulatory circuit during early hematopoietic development.</h4>
<p class="authors"><span class="author">Pimanda JE</span>, <span class="author">Ottersbach K</span>, <span class="author">Knezevic K</span>, <span class="author">Kinston S</span>, <span class="author">Chan WY</span>, <span class="author">Wilson NK</span>, <span class="author">Landry JR</span>, <span class="author">Wood AD</span>, <span class="author">Kolb-Kokocinski A</span>, <span class="author">Green AR</span>, <span class="author">Tannahill D</span>, <span class="author">Lacaud G</span>, <span class="author">Kouskoff V</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, United Kingdom. <a href="mailto:%6A%70%69%6D%61%6E%64%61%40%75%6E%73%77.%65%64%75.%61%75">&#106;&#112;&#105;&#109;&#97;&#110;&#100;&#97;&#64;&#117;&#110;&#115;&#119;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">Conservation of the vertebrate body plan has been attributed to the evolutionary stability of gene-regulatory networks (GRNs). We describe a regulatory circuit made up of Gata2, Fli1, and Scl/Tal1 and their enhancers, Gata2-3, Fli1+12, and Scl+19, that operates during specification of hematopoiesis in the mouse embryo. We show that the Fli1+12 enhancer, like the Gata2-3 and Scl+19 enhancers, targets hematopoietic stem cells (HSCs) and relies on a combination of Ets, Gata, and E-Box motifs. We show that the Gata2-3 enhancer also uses a similar cluster of motifs and that Gata2, Fli1, and Scl are expressed in embryonic day-11.5 dorsal aorta where HSCs originate and in fetal liver where they multiply. The three HSC enhancers in these tissues and in ES cell-derived hemangioblast equivalents are bound by each of these transcription factors (TFs) and form a fully connected triad that constitutes a previously undescribed example of both this network motif in mammalian development and a GRN kernel operating during the specification of a mammalian stem cell.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">45</span>;<span class="pagerange">17692-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17962413" rel="external">17962413</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2077040" rel="external">2077040</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0707045104" rel="external">10.1073/pnas.0707045104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Frequent activating FGFR2 mutations in endometrial carcinomas parallel germline mutations associated with craniosynostosis and skeletal dysplasia syndromes.</h4>
<p class="authors"><span class="author">Pollock PM</span>, <span class="author">Gartside MG</span>, <span class="author">Dejeza LC</span>, <span class="author">Powell MA</span>, <span class="author">Mallon MA</span>, <span class="author">Davies H</span>, <span class="author">Mohammadi M</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Trent JM</span> and <span class="author">Goodfellow PJ</span></p>
<p class="affiliation">Cancer and Cell Biology Division, Translational Genomics Research Institute, Phoenix, AZ, USA. <a href="mailto:%70%70%6F%6C%6C%6F%63%6B%40%74%67%65%6E.%6F%72%67">&#112;&#112;&#111;&#108;&#108;&#111;&#99;&#107;&#64;&#116;&#103;&#101;&#110;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Endometrial carcinoma is the most common gynecological malignancy in the United States. Although most women present with early disease confined to the uterus, the majority of persistent or recurrent tumors are refractory to current chemotherapies. We have identified a total of 11 different FGFR2 mutations in 3/10 (30%) of endometrial cell lines and 19/187 (10%) of primary uterine tumors. Mutations were seen primarily in tumors of the endometrioid histologic subtype (18/115 cases investigated, 16%). The majority of the somatic mutations identified were identical to germline activating mutations in FGFR2 and FGFR3 that cause Apert Syndrome, Beare-Stevenson Syndrome, hypochondroplasia, achondroplasia and SADDAN syndrome. The two most common somatic mutations identified were S252W (in eight tumors) and N550K (in five samples). Four novel mutations were identified, three of which are also likely to result in receptor gain-of-function. Extensive functional analyses have already been performed on many of these mutations, demonstrating they result in receptor activation through a variety of mechanisms. The discovery of activating FGFR2 mutations in endometrial carcinoma raises the possibility of employing anti-FGFR molecularly targeted therapies in patients with advanced or recurrent endometrial carcinoma.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA091842</span>, <span class="grant">R01 CA109544</span>, <span class="grant">R01 CA71754</span>; Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">50</span>;<span class="pagerange">7158-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17525745" rel="external">17525745</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871595" rel="external">2871595</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1210529" rel="external">10.1038/sj.onc.1210529</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">F0 generation mice fully derived from gene-targeted embryonic stem cells allowing immediate phenotypic analyses.</h4>
<p class="authors"><span class="author">Poueymirou WT</span>, <span class="author">Auerbach W</span>, <span class="author">Frendewey D</span>, <span class="author">Hickey JF</span>, <span class="author">Escaravage JM</span>, <span class="author">Esau L</span>, <span class="author">Dor&eacute; AT</span>, <span class="author">Stevens S</span>, <span class="author">Adams NC</span>, <span class="author">Dominguez MG</span>, <span class="author">Gale NW</span>, <span class="author">Yancopoulos GD</span>, <span class="author">DeChiara TM</span> and <span class="author">Valenzuela DM</span></p>
<p class="affiliation">Regeneron Pharmaceuticals, Inc., Tarrytown, New York 10591, USA.</p><p class="abstract">A useful approach for exploring gene function involves generating mutant mice from genetically modified embryonic stem (ES) cells. Recent advances in genetic engineering of ES cells have shifted the bottleneck in this process to the generation of mice. Conventional injections of ES cells into blastocyst hosts produce F0 generation chimeras that are only partially derived from ES cells, requiring additional breeding to obtain mutant mice that can be phenotyped. The tetraploid complementation approach directly yields mice that are almost entirely derived from ES cells, but it is inefficient, works only with certain hybrid ES cell lines and suffers from nonspecific lethality and abnormalities, complicating phenotypic analyses. Here we show that laser-assisted injection of either inbred or hybrid ES cells into eight cell-stage embryos efficiently yields F0 generation mice that are fully ES cell-derived and healthy, exhibit 100% germline transmission and allow immediate phenotypic analysis, greatly accelerating gene function assignment.</p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">2007</span>;<span class="volume">25</span>;<span class="number">1</span>;<span class="pagerange">91-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17187059">17187059</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnbt1263">10.1038/nbt1263</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Speciation in the Genus Bordetella as deduced from comparative genome analyses</h4>
<p class="authors"><span class="author">Preston A</span></p>
<p class="publication"><span class="title">Bordetella: Molecular Microbiology.</span> <span class="year">2007</span>;<span class="pagerange">1-16</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrating sequence and structural biology with DAS.</h4>
<p class="authors"><span class="author">Prli&#x107; A</span>, <span class="author">Down TA</span>, <span class="author">Kulesha E</span>, <span class="author">Finn RD</span>, <span class="author">K&auml;h&auml;ri A</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. <a href="mailto:%61%70%33%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#112;&#51;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The Distributed Annotation System (DAS) is a network protocol for exchanging biological data. It is frequently used to share annotations of genomes and protein sequence.</p><p class="abstract"><strong>Results</strong>: Here we present several extensions to the current DAS 1.5 protocol. These provide new commands to share alignments, three dimensional molecular structure data, add the possibility for registration and discovery of DAS servers, and provide a convention how to provide different types of data plots. We present examples of web sites and applications that use the new extensions. We operate a public registry of DAS sources, which now includes entries for more than 250 distinct sources.</p><p class="abstract"><strong>Conclusion</strong>: Our DAS extensions are essential for the management of the growing number of services and exchange of diverse biological data sets. In addition the extensions allow new types of applications to be developed and scientific questions to be addressed. The registry of DAS sources is available at http://www.dasregistry.org.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">333</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17850653">17850653</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2031907">2031907</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-8-333">10.1186/1471-2105-8-333</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Olfactory bulb hypoplasia in Prokr2 null mice stems from defective neuronal progenitor migration and differentiation.</h4>
<p class="authors"><span class="author">Prosser HM</span>, <span class="author">Bradley A</span> and <span class="author">Caldwell MA</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">New neurons are added on a daily basis to the olfactory bulb (OB) of a mammal, and this phenomenon exists throughout its lifetime. These new cells are born in the subventricular zone and migrate to the OB via the rostral migratory stream (RMS). To examine the role of the prokineticin receptor 2 (Prokr2) in neurogenesis, we created a Prokr2 null mouse, and report a decrease in the volume of its OB and also a decrease in the number of bromodeoxyuridine (BrdU)-positive cells. There is disrupted architecture of the OB, with the glomerular layer containing terminal dUTP nick-end labeling (TUNEL) -positive nuclei and also a decrease in tyrosine hydroxylase-positive neurons in this layer. In addition, there are increased numbers of doublecortin-positive neuroblasts in the RMS and increased PSA-NCAM (polysialylated form of the neural cell adhesion molecule) -positive neuronal progenitors around the olfactory ventricle, indicating their detachment from homotypic chains is compromised. Finally, in support of this, Prokr2-deficient cells expanded in vitro as neurospheres are incapable of migrating towards a source of recombinant human prokineticin 2 (PROK2). Together, these findings suggest an important role for Prokr2 in OB neurogenesis.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The European journal of neuroscience</span>&nbsp;<span class="year">2007</span>;<span class="volume">26</span>;<span class="number">12</span>;<span class="pagerange">3339-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18052978" rel="external">18052978</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2228368" rel="external">2228368</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1460-9568.2007.05958.x" rel="external">10.1111/j.1460-9568.2007.05958.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prokineticin receptor 2 (Prokr2) is essential for the regulation of circadian behavior by the suprachiasmatic nuclei.</h4>
<p class="authors"><span class="author">Prosser HM</span>, <span class="author">Bradley A</span>, <span class="author">Chesham JE</span>, <span class="author">Ebling FJ</span>, <span class="author">Hastings MH</span> and <span class="author">Maywood ES</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The suprachiasmatic nucleus (SCN), the brain&#39;s principal circadian pacemaker, coordinates adaptive daily cycles of behavior and physiology, including the rhythm of sleep and wakefulness. The cellular mechanism sustaining SCN circadian timing is well characterized, but the neurochemical pathways by which SCN neurons coordinate circadian behaviors remain unknown. SCN transplant studies suggest a role for (unidentified) secreted factors, and one potential candidate is the SCN neuropeptide prokineticin 2 (Prok2). Prok2 and its cognate prokineticin receptor 2 (Prokr2/Gpcr73l1) are widely expressed in both the SCN and its neural targets, and Prok2 is light-regulated. Hence, they may contribute to cellular timing within the SCN, entrainment of the clock, and/or they may mediate circadian output. We show that a targeted null mutation of Prokr2 disrupts circadian coordination of the activity cycle and thermoregulation. Specifically, mice lacking Prokr2 lost precision in timing the onset of nocturnal locomotor activity; and under both a light/dark cycle and continuous darkness, there was a pronounced temporal redistribution of activity away from early to late circadian night. Moreover, the coherence of circadian behavior was significantly reduced, and nocturnal body temperature was depressed. Entrainment by light is not, however, dependent on Prokr2, and bioluminescence real-time imaging of organotypical SCN slices showed that the mutant SCN is fully competent as a circadian oscillator. We conclude that Prokr2 is not necessary for SCN cellular timekeeping or entrainment, but it is an essential link for coordination of circadian behavior and physiology by the SCN, especially in defining the onset and maintenance of circadian night.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">2</span>;<span class="pagerange">648-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17202262" rel="external">17202262</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1761911" rel="external">1761911</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0606884104" rel="external">10.1073/pnas.0606884104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Wnt5a functions in planar cell polarity regulation in mice.</h4>
<p class="authors"><span class="author">Qian D</span>, <span class="author">Jones C</span>, <span class="author">Rzadzinska A</span>, <span class="author">Mark S</span>, <span class="author">Zhang X</span>, <span class="author">Steel KP</span>, <span class="author">Dai X</span> and <span class="author">Chen P</span></p>
<p class="affiliation">Department of Cell Biology, Emory University School of Medicine, Atlanta, GA 30322, USA.</p><p class="abstract">Planar cell polarity (PCP) refers to the polarization of cells within the plane of a cell sheet. A distinctive epithelial PCP in vertebrates is the uniform orientation of stereociliary bundles of the sensory hair cells in the mammalian cochlea. In addition to establishing epithelial PCP, planar polarization is also required for convergent extension (CE); a polarized cellular movement that occurs during neural tube closure and cochlear extension. Studies in Drosophila and vertebrates have revealed a conserved PCP pathway, including Frizzled (Fz) receptors. Here we use the cochlea as a model system to explore the involvement of known ligands of Fz, Wnt morphogens, in PCP regulation. We show that Wnt5a forms a reciprocal expression pattern with a Wnt antagonist, the secreted frizzled-related protein 3 (Sfrp3 or Frzb), along the axis of planar polarization in the cochlear epithelium. We further demonstrate that Wnt5a antagonizes Frzb in regulating cochlear extension and stereociliary bundle orientation in vitro, and that Wnt5a(-/-) animals have a shortened and widened cochlea. Finally, we show that Wnt5a is required for proper subcellular distribution of a PCP protein, Ltap/Vangl2, and that Wnt5a interacts genetically with Ltap/Vangl2 for uniform orientation of stereocilia, cochlear extension, and neural tube closure. Together, these findings demonstrate that Wnt5a functions in PCP regulation in mice.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; NIDCD NIH HHS: <span class="grant">R01 DC005213</span>, <span class="grant">R01 DC005213-06</span>, <span class="grant">R01 DC007423</span>, <span class="grant">R01 DC007423-01A2</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">306</span>;<span class="number">1</span>;<span class="pagerange">121-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17433286">17433286</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1978180">1978180</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2007.03.011">10.1016/j.ydbio.2007.03.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Butyrate mediates decrease of histone acetylation centered on transcription start sites and down-regulation of associated genes.</h4>
<p class="authors"><span class="author">Rada-Iglesias A</span>, <span class="author">Enroth S</span>, <span class="author">Ameur A</span>, <span class="author">Koch CM</span>, <span class="author">Clelland GK</span>, <span class="author">Respuela-Alonso P</span>, <span class="author">Wilcox S</span>, <span class="author">Dovey OM</span>, <span class="author">Ellis PD</span>, <span class="author">Langford CF</span>, <span class="author">Dunham I</span>, <span class="author">Komorowski J</span> and <span class="author">Wadelius C</span></p>
<p class="affiliation">Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, Uppsala, SE-751 05 Sweden. <a href="mailto:%61%6C%76%61%72%6F.%72%61%64%61%40%67%65%6E%70%61%74.%75%75.%73%65">&#97;&#108;&#118;&#97;&#114;&#111;&#46;&#114;&#97;&#100;&#97;&#64;&#103;&#101;&#110;&#112;&#97;&#116;&#46;&#117;&#117;&#46;&#115;&#101;</a></p><p class="abstract">Butyrate is a histone deacetylase inhibitor (HDACi) with anti-neoplastic properties, which theoretically reactivates epigenetically silenced genes by increasing global histone acetylation. However, recent studies indicate that a similar number or even more genes are down-regulated than up-regulated by this drug. We treated hepatocarcinoma HepG2 cells with butyrate and characterized the levels of acetylation at DNA-bound histones H3 and H4 by ChIP-chip along the ENCODE regions. In contrast to the global increases of histone acetylation, many genomic regions close to transcription start sites were deacetylated after butyrate exposure. In order to validate these findings, we found that both butyrate and trichostatin A treatment resulted in histone deacetylation at selected regions, while nucleosome loss or changes in histone H3 lysine 4 trimethylation (H3K4me3) did not occur in such locations. Furthermore, similar histone deacetylation events were observed when colon adenocarcinoma HT-29 cells were treated with butyrate. In addition, genes with deacetylated promoters were down-regulated by butyrate, and this was mediated at the transcriptional level by affecting RNA polymerase II (POLR2A) initiation/elongation. Finally, the global increase in acetylated histones was preferentially localized to the nuclear periphery, indicating that it might not be associated to euchromatin. Our results are significant for the evaluation of HDACi as anti-tumourogenic drugs, suggesting that previous models of action might need to be revised, and provides an explanation for the frequently observed repression of many genes during HDACi treatment.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">5 U01 HG003168</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">708-19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17567991" rel="external">17567991</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1891332" rel="external">1891332</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5540007" rel="external">10.1101/gr.5540007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">PALB2, which encodes a BRCA2-interacting protein, is a breast cancer susceptibility gene.</h4>
<p class="authors"><span class="author">Rahman N</span>, <span class="author">Seal S</span>, <span class="author">Thompson D</span>, <span class="author">Kelly P</span>, <span class="author">Renwick A</span>, <span class="author">Elliott A</span>, <span class="author">Reid S</span>, <span class="author">Spanova K</span>, <span class="author">Barfoot R</span>, <span class="author">Chagtai T</span>, <span class="author">Jayatilake H</span>, <span class="author">McGuffog L</span>, <span class="author">Hanks S</span>, <span class="author">Evans DG</span>, <span class="author">Eccles D</span>, <span class="author">Breast Cancer Susceptibility Collaboration (UK)</span>, <span class="author">Easton DF</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Section of Cancer Genetics, Institute of Cancer Research, 15 Cotswold Road, Sutton, Surrey SM2 5NG, UK. <a href="mailto:%6E%61%7A%6E%65%65%6E.%72%61%68%6D%61%6E%40%69%63%72.%61%63.%75%6B">&#110;&#97;&#122;&#110;&#101;&#101;&#110;&#46;&#114;&#97;&#104;&#109;&#97;&#110;&#64;&#105;&#99;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">PALB2 interacts with BRCA2, and biallelic mutations in PALB2 (also known as FANCN), similar to biallelic BRCA2 mutations, cause Fanconi anemia. We identified monoallelic truncating PALB2 mutations in 10/923 individuals with familial breast cancer compared with 0/1,084 controls (P = 0.0004) and show that such mutations confer a 2.3-fold higher risk of breast cancer (95% confidence interval (c.i.) = 1.4-3.9, P = 0.0025). The results show that PALB2 is a breast cancer susceptibility gene and further demonstrate the close relationship of the Fanconi anemia-DNA repair pathway and breast cancer predisposition.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">068545/Z/02</span>, <span class="grant">077012</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">2</span>;<span class="pagerange">165-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17200668">17200668</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2871593">2871593</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1959">10.1038/ng1959</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene Prediction</h4>
<p class="authors"><span class="author">Rajandream M</span></p>
<p class="publication"><span class="title">Bioinformatics</span> <span class="year">2007</span>;<span class="pagerange">71-102</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Downregulation of death-associated protein kinase 1 (DAPK1) in chronic lymphocytic leukemia.</h4>
<p class="authors"><span class="author">Raval A</span>, <span class="author">Tanner SM</span>, <span class="author">Byrd JC</span>, <span class="author">Angerman EB</span>, <span class="author">Perko JD</span>, <span class="author">Chen SS</span>, <span class="author">Hackanson B</span>, <span class="author">Grever MR</span>, <span class="author">Lucas DM</span>, <span class="author">Matkovic JJ</span>, <span class="author">Lin TS</span>, <span class="author">Kipps TJ</span>, <span class="author">Murray F</span>, <span class="author">Weisenburger D</span>, <span class="author">Sanger W</span>, <span class="author">Lynch J</span>, <span class="author">Watson P</span>, <span class="author">Jansen M</span>, <span class="author">Yoshinaga Y</span>, <span class="author">Rosenquist R</span>, <span class="author">de Jong PJ</span>, <span class="author">Coggill P</span>, <span class="author">Beck S</span>, <span class="author">Lynch H</span>, <span class="author">de la Chapelle A</span> and <span class="author">Plass C</span></p>
<p class="affiliation">Department of Molecular Virology, Immunology, and Medical Genetics, Human Cancer Genetics Program, The Comprehensive Cancer Center at The Ohio State University, Columbus, OH 43214, USA.</p><p class="abstract">The heritability of B cell chronic lymphocytic leukemia (CLL) is relatively high; however, no predisposing mutation has been convincingly identified. We show that loss or reduced expression of death-associated protein kinase 1 (DAPK1) underlies cases of heritable predisposition to CLL and the majority of sporadic CLL. Epigenetic silencing of DAPK1 by promoter methylation occurs in almost all sporadic CLL cases. Furthermore, we defined a disease haplotype, which segregates with the CLL phenotype in a large family. DAPK1 expression of the CLL allele is downregulated by 75% in germline cells due to increased HOXB7 binding. In the blood cells from affected family members, promoter methylation results in additional loss of DAPK1 expression. Thus, reduced expression of DAPK1 can result from germline predisposition, as well as epigenetic or somatic events causing or contributing to the CLL phenotype.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">5U01 CA86389</span>, <span class="grant">CA101956</span>, <span class="grant">CA110496</span>, <span class="grant">CA81534</span>, <span class="grant">P30 CA16058</span>, <span class="grant">T32 CA106196</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2007</span>;<span class="volume">129</span>;<span class="number">5</span>;<span class="pagerange">879-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17540169">17540169</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2007.03.043">10.1016/j.cell.2007.03.043</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Unusual phyletic distribution of peptidases as a tool for identifying potential drug targets.</h4>
<p class="authors"><span class="author">Rawlings ND</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Eukaryote homologues of carboxypeptidases Taq have been discovered by Niemirowicz et al. in the protozoan Trypanosoma cruzi, the causative agent of Chagas&#39; disease. This is surprising, because the peptidase family was thought to be restricted to bacteria and archaea. In this issue of the Biochemical Journal, the authors propose that the Trypanosoma carboxypeptidases are potential drug targets for treatment of the disease. The authors also propose that the presence of the genes in the zooflagellates can be explained by a horizontal transfer of an ancestral gene from a prokaryote. Because peptidases are popular drug targets, identifying parasite or pathogen peptidases that have no homologues in their hosts would be a method to select the most promising targets. To understand how unusual this phyletic distribution is among the 183 families of peptidases, several other examples of horizontal transfers are presented, as well as some unusual losses of peptidase genes.</p>
<p class="publication"><span class="title">The Biochemical journal</span>&nbsp;<span class="year">2007</span>;<span class="volume">401</span>;<span class="number">2</span>;<span class="pagerange">e5-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17173540" rel="external">17173540</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1820803" rel="external">1820803</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBJ20061613" rel="external">10.1042/BJ20061613</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in ZDHHC9, which encodes a palmitoyltransferase of NRAS and HRAS, cause X-linked mental retardation associated with a Marfanoid habitus.</h4>
<p class="authors"><span class="author">Raymond FL</span>, <span class="author">Tarpey PS</span>, <span class="author">Edkins S</span>, <span class="author">Tofts C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Stevens C</span>, <span class="author">Barthorpe S</span>, <span class="author">Buck G</span>, <span class="author">Cole J</span>, <span class="author">Dicks E</span>, <span class="author">Gray K</span>, <span class="author">Halliday K</span>, <span class="author">Hills K</span>, <span class="author">Hinton J</span>, <span class="author">Jones D</span>, <span class="author">Menzies A</span>, <span class="author">Perry J</span>, <span class="author">Raine K</span>, <span class="author">Shepherd R</span>, <span class="author">Small A</span>, <span class="author">Varian J</span>, <span class="author">Widaa S</span>, <span class="author">Mallya U</span>, <span class="author">Moon J</span>, <span class="author">Luo Y</span>, <span class="author">Shaw M</span>, <span class="author">Boyle J</span>, <span class="author">Kerr B</span>, <span class="author">Turner G</span>, <span class="author">Quarrell O</span>, <span class="author">Cole T</span>, <span class="author">Easton DF</span>, <span class="author">Wooster R</span>, <span class="author">Bobrow M</span>, <span class="author">Schwartz CE</span>, <span class="author">Gecz J</span>, <span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Cambridge Institute of Medical Research, University of Cambridge, Cambridge, CB2 2XY, UK. <a href="mailto:%66%6C%72%32%34%40%63%61%6D.%61%63.%75%6B">&#102;&#108;&#114;&#50;&#52;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p><p class="abstract">We have identified one frameshift mutation, one splice-site mutation, and two missense mutations in highly conserved residues in ZDHHC9 at Xq26.1 in 4 of 250 families with X-linked mental retardation (XLMR). In three of the families, the mental retardation phenotype is associated with a Marfanoid habitus, although none of the affected individuals meets the Ghent criteria for Marfan syndrome. ZDHHC9 is a palmitoyltransferase that catalyzes the posttranslational modification of NRAS and HRAS. The degree of palmitoylation determines the temporal and spatial location of these proteins in the plasma membrane and Golgi complex. The finding of mutations in ZDHHC9 suggests that alterations in the concentrations and cellular distribution of target proteins are sufficient to cause disease. This is the first XLMR gene to be reported that encodes a posttranslational modification enzyme, palmitoyltransferase. Furthermore, now that the first palmitoyltransferase that causes mental retardation has been identified, defects in other palmitoylation transferases become good candidates for causing other mental retardation syndromes.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD26202</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">80</span>;<span class="number">5</span>;<span class="pagerange">982-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17436253">17436253</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1852737">1852737</a>; DOI: <a href="http://dx.doi.org/10.1086%2F513609">10.1086/513609</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary and biomedical insights from the rhesus macaque genome.</h4>
<p class="authors"><span class="author">Rhesus Macaque Genome Sequencing and Analysis Consortium</span>, <span class="author">Gibbs RA</span>, <span class="author">Rogers J</span>, <span class="author">Katze MG</span>, <span class="author">Bumgarner R</span>, <span class="author">Weinstock GM</span>, <span class="author">Mardis ER</span>, <span class="author">Remington KA</span>, <span class="author">Strausberg RL</span>, <span class="author">Venter JC</span>, <span class="author">Wilson RK</span>, <span class="author">Batzer MA</span>, <span class="author">Bustamante CD</span>, <span class="author">Eichler EE</span>, <span class="author">Hahn MW</span>, <span class="author">Hardison RC</span>, <span class="author">Makova KD</span>, <span class="author">Miller W</span>, <span class="author">Milosavljevic A</span>, <span class="author">Palermo RE</span>, <span class="author">Siepel A</span>, <span class="author">Sikela JM</span>, <span class="author">Attaway T</span>, <span class="author">Bell S</span>, <span class="author">Bernard KE</span>, <span class="author">Buhay CJ</span>, <span class="author">Chandrabose MN</span>, <span class="author">Dao M</span>, <span class="author">Davis C</span>, <span class="author">Delehaunty KD</span>, <span class="author">Ding Y</span>, <span class="author">Dinh HH</span>, <span class="author">Dugan-Rocha S</span>, <span class="author">Fulton LA</span>, <span class="author">Gabisi RA</span>, <span class="author">Garner TT</span>, <span class="author">Godfrey J</span>, <span class="author">Hawes AC</span>, <span class="author">Hernandez J</span>, <span class="author">Hines S</span>, <span class="author">Holder M</span>, <span class="author">Hume J</span>, <span class="author">Jhangiani SN</span>, <span class="author">Joshi V</span>, <span class="author">Khan ZM</span>, <span class="author">Kirkness EF</span>, <span class="author">Cree A</span>, <span class="author">Fowler RG</span>, <span class="author">Lee S</span>, <span class="author">Lewis LR</span>, <span class="author">Li Z</span>, <span class="author">Liu YS</span>, <span class="author">Moore SM</span>, <span class="author">Muzny D</span>, <span class="author">Nazareth LV</span>, <span class="author">Ngo DN</span>, <span class="author">Okwuonu GO</span>, <span class="author">Pai G</span>, <span class="author">Parker D</span>, <span class="author">Paul HA</span>, <span class="author">Pfannkoch C</span>, <span class="author">Pohl CS</span>, <span class="author">Rogers YH</span>, <span class="author">Ruiz SJ</span>, <span class="author">Sabo A</span>, <span class="author">Santibanez J</span>, <span class="author">Schneider BW</span>, <span class="author">Smith SM</span>, <span class="author">Sodergren E</span>, <span class="author">Svatek AF</span>, <span class="author">Utterback TR</span>, <span class="author">Vattathil S</span>, <span class="author">Warren W</span>, <span class="author">White CS</span>, <span class="author">Chinwalla AT</span>, <span class="author">Feng Y</span>, <span class="author">Halpern AL</span>, <span class="author">Hillier LW</span>, <span class="author">Huang X</span>, <span class="author">Minx P</span>, <span class="author">Nelson JO</span>, <span class="author">Pepin KH</span>, <span class="author">Qin X</span>, <span class="author">Sutton GG</span>, <span class="author">Venter E</span>, <span class="author">Walenz BP</span>, <span class="author">Wallis JW</span>, <span class="author">Worley KC</span>, <span class="author">Yang SP</span>, <span class="author">Jones SM</span>, <span class="author">Marra MA</span>, <span class="author">Rocchi M</span>, <span class="author">Schein JE</span>, <span class="author">Baertsch R</span>, <span class="author">Clarke L</span>, <span class="author">Cs&uuml;r&ouml;s M</span>, <span class="author">Glasscock J</span>, <span class="author">Harris RA</span>, <span class="author">Havlak P</span>, <span class="author">Jackson AR</span>, <span class="author">Jiang H</span>, <span class="author">Liu Y</span>, <span class="author">Messina DN</span>, <span class="author">Shen Y</span>, <span class="author">Song HX</span>, <span class="author">Wylie T</span>, <span class="author">Zhang L</span>, <span class="author">Birney E</span>, <span class="author">Han K</span>, <span class="author">Konkel MK</span>, <span class="author">Lee J</span>, <span class="author">Smit AF</span>, <span class="author">Ullmer B</span>, <span class="author">Wang H</span>, <span class="author">Xing J</span>, <span class="author">Burhans R</span>, <span class="author">Cheng Z</span>, <span class="author">Karro JE</span>, <span class="author">Ma J</span>, <span class="author">Raney B</span>, <span class="author">She X</span>, <span class="author">Cox MJ</span>, <span class="author">Demuth JP</span>, <span class="author">Dumas LJ</span>, <span class="author">Han SG</span>, <span class="author">Hopkins J</span>, <span class="author">Karimpour-Fard A</span>, <span class="author">Kim YH</span>, <span class="author">Pollack JR</span>, <span class="author">Vinar T</span>, <span class="author">Addo-Quaye C</span>, <span class="author">Degenhardt J</span>, <span class="author">Denby A</span>, <span class="author">Hubisz MJ</span>, <span class="author">Indap A</span>, <span class="author">Kosiol C</span>, <span class="author">Lahn BT</span>, <span class="author">Lawson HA</span>, <span class="author">Marklein A</span>, <span class="author">Nielsen R</span>, <span class="author">Vallender EJ</span>, <span class="author">Clark AG</span>, <span class="author">Ferguson B</span>, <span class="author">Hernandez RD</span>, <span class="author">Hirani K</span>, <span class="author">Kehrer-Sawatzki H</span>, <span class="author">Kolb J</span>, <span class="author">Patil S</span>, <span class="author">Pu LL</span>, <span class="author">Ren Y</span>, <span class="author">Smith DG</span>, <span class="author">Wheeler DA</span>, <span class="author">Schenck I</span>, <span class="author">Ball EV</span>, <span class="author">Chen R</span>, <span class="author">Cooper DN</span>, <span class="author">Giardine B</span>, <span class="author">Hsu F</span>, <span class="author">Kent WJ</span>, <span class="author">Lesk A</span>, <span class="author">Nelson DL</span>, <span class="author">O&#39;brien WE</span>, <span class="author">Pr&uuml;fer K</span>, <span class="author">Stenson PD</span>, <span class="author">Wallace JC</span>, <span class="author">Ke H</span>, <span class="author">Liu XM</span>, <span class="author">Wang P</span>, <span class="author">Xiang AP</span>, <span class="author">Yang F</span>, <span class="author">Barber GP</span>, <span class="author">Haussler D</span>, <span class="author">Karolchik D</span>, <span class="author">Kern AD</span>, <span class="author">Kuhn RM</span>, <span class="author">Smith KE</span> and <span class="author">Zwieg AS</span></p>
<p class="affiliation">Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX 77030, USA. <a href="mailto:%61%67%69%62%62%73%40%62%63%6D.%65%64%75">&#97;&#103;&#105;&#98;&#98;&#115;&#64;&#98;&#99;&#109;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The rhesus macaque (Macaca mulatta) is an abundant primate species that diverged from the ancestors of Homo sapiens about 25 million years ago. Because they are genetically and physiologically similar to humans, rhesus monkeys are the most widely used nonhuman primate in basic and applied biomedical research. We determined the genome sequence of an Indian-origin Macaca mulatta female and compared the data with chimpanzees and humans to reveal the structure of ancestral primate genomes and to identify evidence for positive selection and lineage-specific expansions and contractions of gene families. A comparison of sequences from individual animals was used to investigate their underlying genetic diversity. The complete description of the macaque genome blueprint enhances the utility of this animal model for biomedical research and improves our understanding of the basic biology of the species.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U54 HG003068</span>, <span class="grant">U54 HG003079</span>, <span class="grant">U54 HG003273</span>; Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">316</span>;<span class="number">5822</span>;<span class="pagerange">222-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17431167">17431167</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1139247">10.1126/science.1139247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The brain-derived neurotrophic factor rs6265 (Val66Met) polymorphism and depression in Mexican-Americans.</h4>
<p class="authors"><span class="author">Ribeiro L</span>, <span class="author">Busnello JV</span>, <span class="author">Cantor RM</span>, <span class="author">Whelan F</span>, <span class="author">Whittaker P</span>, <span class="author">Deloukas P</span>, <span class="author">Wong ML</span> and <span class="author">Licinio J</span></p>
<p class="affiliation">Department of Psychiatry &amp; Behavioral Sciences, Center on Pharmacogenomics, University of Miami Miller School of Medicine, Miami, Florida 33136-1013, USA. <a href="mailto:%6C%72%69%62%65%69%72%6F%40%6D%65%64.%6D%69%61%6D%69.%65%64%75">&#108;&#114;&#105;&#98;&#101;&#105;&#114;&#111;&#64;&#109;&#101;&#100;&#46;&#109;&#105;&#97;&#109;&#105;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The hypothesis that brain-derived neurotrophic factor (BDNF) is involved in the pathogenesis of major depression is supported by several research findings; however, genetic studies assessing the relationship between BDNF and psychiatric disorders have produced conflicting results. We examined the effect of a BDNF polymorphism on depression susceptibility in Mexican-Americans. The single nucleotide polymorphism (Val66Met), which has been shown to have functional and behavioral effects, was genotyped in 284 depressed participants and 331 controls, showing association with depression (P=0.005). Individuals homozygous for the major allele (GG) had an increased chance of being depressed (OR=1.7 95% CI 1.17-2.47). Our findings support the association of BDNF single nucleotide polymorphism rs6265 and depression, suggesting that this polymorphism may increase susceptibility to major depression in Mexican-Americans.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">RR000865</span>, <span class="grant">RR017365</span>, <span class="grant">RR16996</span>; NHGRI NIH HHS: <span class="grant">HG002500</span>; NIDDK NIH HHS: <span class="grant">DK063240</span>; NIGMS NIH HHS: <span class="grant">GM61394</span>; NIMH NIH HHS: <span class="grant">MH062777</span>; Wellcome Trust: <span class="grant">077011</span></p>
<p class="publication"><span class="title">Neuroreport</span>&nbsp;<span class="year">2007</span>;<span class="volume">18</span>;<span class="number">12</span>;<span class="pagerange">1291-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17632285" rel="external">17632285</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2686836" rel="external">2686836</a>; DOI: <a href="http://dx.doi.org/10.1097%2FWNR.0b013e328273bcb0" rel="external">10.1097/WNR.0b013e328273bcb0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Requirement of bic/microRNA-155 for normal immune function.</h4>
<p class="authors"><span class="author">Rodriguez A</span>, <span class="author">Vigorito E</span>, <span class="author">Clare S</span>, <span class="author">Warren MV</span>, <span class="author">Couttet P</span>, <span class="author">Soond DR</span>, <span class="author">van Dongen S</span>, <span class="author">Grocock RJ</span>, <span class="author">Das PP</span>, <span class="author">Miska EA</span>, <span class="author">Vetrie D</span>, <span class="author">Okkenhaug K</span>, <span class="author">Enright AJ</span>, <span class="author">Dougan G</span>, <span class="author">Turner M</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">MicroRNAs are a class of small RNAs that are increasingly being recognized as important regulators of gene expression. Although hundreds of microRNAs are present in the mammalian genome, genetic studies addressing their physiological roles are at an early stage. We have shown that mice deficient for bic/microRNA-155 are immunodeficient and display increased lung airway remodeling. We demonstrate a requirement of bic/microRNA-155 for the function of B and T lymphocytes and dendritic cells. Transcriptome analysis of bic/microRNA-155-deficient CD4+ T cells identified a wide spectrum of microRNA-155-regulated genes, including cytokines, chemokines, and transcription factors. Our work suggests that bic/microRNA-155 plays a key role in the homeostasis and function of the immune system.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G117/424</span>; Wellcome Trust: <span class="grant">077187</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">316</span>;<span class="number">5824</span>;<span class="pagerange">608-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17463290">17463290</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2610435">2610435</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1139253">10.1126/science.1139253</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal evolution of Arvicolinae (Cricetidae, Rodentia). II. The genome homology of two mole voles (genus Ellobius), the field vole and golden hamster revealed by comparative chromosome painting.</h4>
<p class="authors"><span class="author">Romanenko SA</span>, <span class="author">Sitnikova NA</span>, <span class="author">Serdukova NA</span>, <span class="author">Perelman PL</span>, <span class="author">Rubtsova NV</span>, <span class="author">Bakloushinskaya IY</span>, <span class="author">Lyapunova EA</span>, <span class="author">Just W</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Yang F</span> and <span class="author">Graphodatsky AS</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS, Novosibirsk, 630090, Russia.</p><p class="abstract">Using cross-species chromosome painting, we have carried out a comprehensive comparison of the karyotypes of two Ellobius species with unusual sex determination systems: the Transcaucasian mole vole, Ellobius lutescens (2n = 17, X in both sexes), and the northern mole vole, Ellobius talpinus (2n = 54, XX in both sexes). Both Ellobius species have highly rearranged karyotypes. The chromosomal paints from the field vole (Microtus agrestis) detected, in total, 34 and 32 homologous autosomal regions in E. lutescens and E. talpinus karyotypes, respectively. No difference in hybridization pattern of the X paint (as well as Y paint) probes on male and female chromosomes was discovered. The set of golden hamster (Mesocricetus auratus) chromosomal painting probes revealed 44 and 43 homologous autosomal regions in E. lutescens and E. talpinus karyotypes, respectively. A comparative chromosome map was established based on the results of cross-species chromosome painting and a hypothetical ancestral Ellobius karyotype was reconstructed. A considerable number of rearrangements were detected; 31 and 7 fusion/fission rearrangements differentiated the karyotypes of E. lutescens and E. talpinus from the ancestral Ellobius karyotype. It seems that inversions have played a minor role in the genome evolution of these Ellobius species.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">7</span>;<span class="pagerange">891-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17924201" rel="external">17924201</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1171-9" rel="external">10.1007/s10577-007-1171-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Karyotype evolution and phylogenetic relationships of hamsters (Cricetidae, Muroidea, Rodentia) inferred from chromosomal painting and banding comparison.</h4>
<p class="authors"><span class="author">Romanenko SA</span>, <span class="author">Volobouev VT</span>, <span class="author">Perelman PL</span>, <span class="author">Lebedev VS</span>, <span class="author">Serdukova NA</span>, <span class="author">Trifonov VA</span>, <span class="author">Biltueva LS</span>, <span class="author">Nie W</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Bulatova NSh</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Yang F</span> and <span class="author">Graphodatsky AS</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS, Novosibirsk, 630090, Russia. <a href="mailto:%67%72%61%66%40%62%69%6F%6E%65%74.%6E%73%63.%72%75">&#103;&#114;&#97;&#102;&#64;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;</a></p><p class="abstract">The evolutionary success of rodents of the superfamily Muroidea makes this taxon the most interesting for evolution studies, including study at the chromosomal level. Chromosome-specific painting probes from the Chinese hamster and the Syrian (golden) hamster were used to delimit homologous chromosomal segments among 15 hamster species from eight genera: Allocricetulus, Calomyscus, Cricetulus, Cricetus, Mesocricetus, Peromyscus, Phodopus and Tscherskia (Cricetidae, Muroidea, Rodentia). Based on results of chromosome painting and G-banding, comparative maps between 20 rodent species have been established. The integrated maps demonstrate a high level of karyotype conservation among species in the Cricetus group (Cricetus, Cricetulus, Allocricetulus) with Tscherskia as its sister group. Species within the genera Mesocricetus and Phodopus also show a high degree of chromosomal conservation. Our results substantiate many of the conclusions suggested by other data and strengthen the topology of the Muroidea phylogenetic tree through the inclusion of genome-wide chromosome rearrangements. The derivation of the muroids karyotypes from the putative ancestral state involved centric fusions, fissions, addition of heterochromatic arms and a great number of inversions. Our results provide further insights into the karyotype relationships of all species investigated.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">3</span>;<span class="pagerange">283-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17333534" rel="external">17333534</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1124-3" rel="external">10.1007/s10577-007-1124-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global transcriptional responses of fission and budding yeast to changes in copper and iron levels: a comparative study.</h4>
<p class="authors"><span class="author">Rustici G</span>, <span class="author">van Bakel H</span>, <span class="author">Lackner DH</span>, <span class="author">Holstege FC</span>, <span class="author">Wijmenga C</span>, <span class="author">B&auml;hler J</span> and <span class="author">Brazma A</span></p>
<p class="affiliation">EMBL Outstation-Hinxton, European Bioinformatics Institute, Cambridge CB10 1SD, UK. <a href="mailto:%67%61%62%72%79%40%65%62%69.%61%63.%75%6B">&#103;&#97;&#98;&#114;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Recent studies in comparative genomics demonstrate that interspecies comparison represents a powerful tool for identifying both conserved and specialized biologic processes across large evolutionary distances. All cells must adjust to environmental fluctuations in metal levels, because levels that are too low or too high can be detrimental. Here we explore the conservation of metal homoeostasis in two distantly related yeasts.</p><p class="abstract"><strong>Results</strong>: We examined genome-wide gene expression responses to changing copper and iron levels in budding and fission yeast using DNA microarrays. The comparison reveals conservation of only a small core set of genes, defining the copper and iron regulons, with a larger number of additional genes being specific for each species. Novel regulatory targets were identified in Schizosaccharomyces pombe for Cuf1p (pex7 and SPAC3G6.05) and Fep1p (srx1, sib1, sib2, rds1, isu1, SPBC27B12.03c, SPAC1F8.02c, and SPBC947.05c). We also present evidence refuting a direct role of Cuf1p in the repression of genes involved in iron uptake. Remarkable differences were detected in responses of the two yeasts to excess copper, probably reflecting evolutionary adaptation to different environments.</p><p class="abstract"><strong>Conclusion</strong>: The considerable evolutionary distance between budding and fission yeast resulted in substantial diversion in the regulation of copper and iron homeostasis. Despite these differences, the conserved regulation of a core set of genes involved in the uptake of these metals provides valuable clues to key features of metal metabolism.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">R73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17477863" rel="external">17477863</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1929147" rel="external">1929147</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-5-r73" rel="external">10.1186/gb-2007-8-5-r73</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide detection and characterization of positive selection in human populations.</h4>
<p class="authors"><span class="author">Sabeti PC</span>, <span class="author">Varilly P</span>, <span class="author">Fry B</span>, <span class="author">Lohmueller J</span>, <span class="author">Hostetter E</span>, <span class="author">Cotsapas C</span>, <span class="author">Xie X</span>, <span class="author">Byrne EH</span>, <span class="author">McCarroll SA</span>, <span class="author">Gaudet R</span>, <span class="author">Schaffner SF</span>, <span class="author">Lander ES</span>, <span class="author">International HapMap Consortium</span>, <span class="author">Frazer KA</span>, <span class="author">Ballinger DG</span>, <span class="author">Cox DR</span>, <span class="author">Hinds DA</span>, <span class="author">Stuve LL</span>, <span class="author">Gibbs RA</span>, <span class="author">Belmont JW</span>, <span class="author">Boudreau A</span>, <span class="author">Hardenbol P</span>, <span class="author">Leal SM</span>, <span class="author">Pasternak S</span>, <span class="author">Wheeler DA</span>, <span class="author">Willis TD</span>, <span class="author">Yu F</span>, <span class="author">Yang H</span>, <span class="author">Zeng C</span>, <span class="author">Gao Y</span>, <span class="author">Hu H</span>, <span class="author">Hu W</span>, <span class="author">Li C</span>, <span class="author">Lin W</span>, <span class="author">Liu S</span>, <span class="author">Pan H</span>, <span class="author">Tang X</span>, <span class="author">Wang J</span>, <span class="author">Wang W</span>, <span class="author">Yu J</span>, <span class="author">Zhang B</span>, <span class="author">Zhang Q</span>, <span class="author">Zhao H</span>, <span class="author">Zhao H</span>, <span class="author">Zhou J</span>, <span class="author">Gabriel SB</span>, <span class="author">Barry R</span>, <span class="author">Blumenstiel B</span>, <span class="author">Camargo A</span>, <span class="author">Defelice M</span>, <span class="author">Faggart M</span>, <span class="author">Goyette M</span>, <span class="author">Gupta S</span>, <span class="author">Moore J</span>, <span class="author">Nguyen H</span>, <span class="author">Onofrio RC</span>, <span class="author">Parkin M</span>, <span class="author">Roy J</span>, <span class="author">Stahl E</span>, <span class="author">Winchester E</span>, <span class="author">Ziaugra L</span>, <span class="author">Altshuler D</span>, <span class="author">Shen Y</span>, <span class="author">Yao Z</span>, <span class="author">Huang W</span>, <span class="author">Chu X</span>, <span class="author">He Y</span>, <span class="author">Jin L</span>, <span class="author">Liu Y</span>, <span class="author">Shen Y</span>, <span class="author">Sun W</span>, <span class="author">Wang H</span>, <span class="author">Wang Y</span>, <span class="author">Wang Y</span>, <span class="author">Xiong X</span>, <span class="author">Xu L</span>, <span class="author">Waye MM</span>, <span class="author">Tsui SK</span>, <span class="author">Xue H</span>, <span class="author">Wong JT</span>, <span class="author">Galver LM</span>, <span class="author">Fan JB</span>, <span class="author">Gunderson K</span>, <span class="author">Murray SS</span>, <span class="author">Oliphant AR</span>, <span class="author">Chee MS</span>, <span class="author">Montpetit A</span>, <span class="author">Chagnon F</span>, <span class="author">Ferretti V</span>, <span class="author">Leboeuf M</span>, <span class="author">Olivier JF</span>, <span class="author">Phillips MS</span>, <span class="author">Roumy S</span>, <span class="author">Sall&eacute;e C</span>, <span class="author">Verner A</span>, <span class="author">Hudson TJ</span>, <span class="author">Kwok PY</span>, <span class="author">Cai D</span>, <span class="author">Koboldt DC</span>, <span class="author">Miller RD</span>, <span class="author">Pawlikowska L</span>, <span class="author">Taillon-Miller P</span>, <span class="author">Xiao M</span>, <span class="author">Tsui LC</span>, <span class="author">Mak W</span>, <span class="author">Song YQ</span>, <span class="author">Tam PK</span>, <span class="author">Nakamura Y</span>, <span class="author">Kawaguchi T</span>, <span class="author">Kitamoto T</span>, <span class="author">Morizono T</span>, <span class="author">Nagashima A</span>, <span class="author">Ohnishi Y</span>, <span class="author">Sekine A</span>, <span class="author">Tanaka T</span>, <span class="author">Tsunoda T</span>, <span class="author">Deloukas P</span>, <span class="author">Bird CP</span>, <span class="author">Delgado M</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt S</span>, <span class="author">Morrison J</span>, <span class="author">Powell D</span>, <span class="author">Stranger BE</span>, <span class="author">Whittaker P</span>, <span class="author">Bentley DR</span>, <span class="author">Daly MJ</span>, <span class="author">de Bakker PI</span>, <span class="author">Barrett J</span>, <span class="author">Chretien YR</span>, <span class="author">Maller J</span>, <span class="author">McCarroll S</span>, <span class="author">Patterson N</span>, <span class="author">Pe&#39;er I</span>, <span class="author">Price A</span>, <span class="author">Purcell S</span>, <span class="author">Richter DJ</span>, <span class="author">Sabeti P</span>, <span class="author">Saxena R</span>, <span class="author">Schaffner SF</span>, <span class="author">Sham PC</span>, <span class="author">Varilly P</span>, <span class="author">Altshuler D</span>, <span class="author">Stein LD</span>, <span class="author">Krishnan L</span>, <span class="author">Smith AV</span>, <span class="author">Tello-Ruiz MK</span>, <span class="author">Thorisson GA</span>, <span class="author">Chakravarti A</span>, <span class="author">Chen PE</span>, <span class="author">Cutler DJ</span>, <span class="author">Kashuk CS</span>, <span class="author">Lin S</span>, <span class="author">Abecasis GR</span>, <span class="author">Guan W</span>, <span class="author">Li Y</span>, <span class="author">Munro HM</span>, <span class="author">Qin ZS</span>, <span class="author">Thomas DJ</span>, <span class="author">McVean G</span>, <span class="author">Auton A</span>, <span class="author">Bottolo L</span>, <span class="author">Cardin N</span>, <span class="author">Eyheramendy S</span>, <span class="author">Freeman C</span>, <span class="author">Marchini J</span>, <span class="author">Myers S</span>, <span class="author">Spencer C</span>, <span class="author">Stephens M</span>, <span class="author">Donnelly P</span>, <span class="author">Cardon LR</span>, <span class="author">Clarke G</span>, <span class="author">Evans DM</span>, <span class="author">Morris AP</span>, <span class="author">Weir BS</span>, <span class="author">Tsunoda T</span>, <span class="author">Johnson TA</span>, <span class="author">Mullikin JC</span>, <span class="author">Sherry ST</span>, <span class="author">Feolo M</span>, <span class="author">Skol A</span>, <span class="author">Zhang H</span>, <span class="author">Zeng C</span>, <span class="author">Zhao H</span>, <span class="author">Matsuda I</span>, <span class="author">Fukushima Y</span>, <span class="author">Macer DR</span>, <span class="author">Suda E</span>, <span class="author">Rotimi CN</span>, <span class="author">Adebamowo CA</span>, <span class="author">Ajayi I</span>, <span class="author">Aniagwu T</span>, <span class="author">Marshall PA</span>, <span class="author">Nkwodimmah C</span>, <span class="author">Royal CD</span>, <span class="author">Leppert MF</span>, <span class="author">Dixon M</span>, <span class="author">Peiffer A</span>, <span class="author">Qiu R</span>, <span class="author">Kent A</span>, <span class="author">Kato K</span>, <span class="author">Niikawa N</span>, <span class="author">Adewole IF</span>, <span class="author">Knoppers BM</span>, <span class="author">Foster MW</span>, <span class="author">Clayton EW</span>, <span class="author">Watkin J</span>, <span class="author">Gibbs RA</span>, <span class="author">Belmont JW</span>, <span class="author">Muzny D</span>, <span class="author">Nazareth L</span>, <span class="author">Sodergren E</span>, <span class="author">Weinstock GM</span>, <span class="author">Wheeler DA</span>, <span class="author">Yakub I</span>, <span class="author">Gabriel SB</span>, <span class="author">Onofrio RC</span>, <span class="author">Richter DJ</span>, <span class="author">Ziaugra L</span>, <span class="author">Birren BW</span>, <span class="author">Daly MJ</span>, <span class="author">Altshuler D</span>, <span class="author">Wilson RK</span>, <span class="author">Fulton LL</span>, <span class="author">Rogers J</span>, <span class="author">Burton J</span>, <span class="author">Carter NP</span>, <span class="author">Clee CM</span>, <span class="author">Griffiths M</span>, <span class="author">Jones MC</span>, <span class="author">McLay K</span>, <span class="author">Plumb RW</span>, <span class="author">Ross MT</span>, <span class="author">Sims SK</span>, <span class="author">Willey DL</span>, <span class="author">Chen Z</span>, <span class="author">Han H</span>, <span class="author">Kang L</span>, <span class="author">Godbout M</span>, <span class="author">Wallenburg JC</span>, <span class="author">L&#39;Archev&ecirc;que P</span>, <span class="author">Bellemare G</span>, <span class="author">Saeki K</span>, <span class="author">Wang H</span>, <span class="author">An D</span>, <span class="author">Fu H</span>, <span class="author">Li Q</span>, <span class="author">Wang Z</span>, <span class="author">Wang R</span>, <span class="author">Holden AL</span>, <span class="author">Brooks LD</span>, <span class="author">McEwen JE</span>, <span class="author">Guyer MS</span>, <span class="author">Wang VO</span>, <span class="author">Peterson JL</span>, <span class="author">Shi M</span>, <span class="author">Spiegel J</span>, <span class="author">Sung LM</span>, <span class="author">Zacharia LF</span>, <span class="author">Collins FS</span>, <span class="author">Kennedy K</span>, <span class="author">Jamieson R</span> and <span class="author">Stewart J</span></p>
<p class="affiliation">Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02139, USA. <a href="mailto:%70%61%72%64%69%73%40%62%72%6F%61%64.%6D%69%74.%65%64%75">&#112;&#97;&#114;&#100;&#105;&#115;&#64;&#98;&#114;&#111;&#97;&#100;&#46;&#109;&#105;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">With the advent of dense maps of human genetic variation, it is now possible to detect positive natural selection across the human genome. Here we report an analysis of over 3 million polymorphisms from the International HapMap Project Phase 2 (HapMap2). We used &#39;long-range haplotype&#39; methods, which were developed to identify alleles segregating in a population that have undergone recent selection, and we also developed new methods that are based on cross-population comparisons to discover alleles that have swept to near-fixation within a population. The analysis reveals more than 300 strong candidate regions. Focusing on the strongest 22 regions, we develop a heuristic for scrutinizing these regions to identify candidate targets of selection. In a complementary analysis, we identify 26 non-synonymous, coding, single nucleotide polymorphisms showing regional evidence of positive selection. Examination of these candidates highlights three cases in which two genes in a common biological process have apparently undergone positive selection in the same population:LARGE and DMD, both related to infection by the Lassa virus, in West Africa;SLC24A5 and SLC45A2, both involved in skin pigmentation, in Europe; and EDAR and EDA2R, both involved in development of hair follicles, in Asia.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077011</span>, <span class="grant">077046</span>, <span class="grant">081682</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">449</span>;<span class="number">7164</span>;<span class="pagerange">913-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17943131">17943131</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2687721">2687721</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06250">10.1038/nature06250</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic analysis of human microRNA transcripts.</h4>
<p class="authors"><span class="author">Saini HK</span>, <span class="author">Griffiths-Jones S</span> and <span class="author">Enright AJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">MicroRNAs (miRNAs) are important genetic regulators of development, differentiation, growth, and metabolism. The mammalian genome encodes approximately 500 known miRNA genes. Approximately 50% are expressed from non-protein-coding transcripts, whereas the rest are located mostly in the introns of coding genes. Intronic miRNAs are generally transcribed coincidentally with their host genes. However, the nature of the primary transcript of intergenic miRNAs is largely unknown. We have performed a large-scale analysis of transcription start sites, polyadenylation signals, CpG islands, EST data, transcription factor-binding sites, and expression ditag data surrounding intergenic miRNAs in the human genome to improve our understanding of the structure of their primary transcripts. We show that a significant fraction of primary transcripts of intergenic miRNAs are 3-4 kb in length, with clearly defined 5&#39; and 3&#39; boundaries. We provide strong evidence for the complete transcript structure of a small number of human miRNAs.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">45</span>;<span class="pagerange">17719-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17965236" rel="external">17965236</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2077053" rel="external">2077053</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0703890104" rel="external">10.1073/pnas.0703890104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomewide association analysis of coronary artery disease.</h4>
<p class="authors"><span class="author">Samani NJ</span>, <span class="author">Erdmann J</span>, <span class="author">Hall AS</span>, <span class="author">Hengstenberg C</span>, <span class="author">Mangino M</span>, <span class="author">Mayer B</span>, <span class="author">Dixon RJ</span>, <span class="author">Meitinger T</span>, <span class="author">Braund P</span>, <span class="author">Wichmann HE</span>, <span class="author">Barrett JH</span>, <span class="author">K&ouml;nig IR</span>, <span class="author">Stevens SE</span>, <span class="author">Szymczak S</span>, <span class="author">Tregouet DA</span>, <span class="author">Iles MM</span>, <span class="author">Pahlke F</span>, <span class="author">Pollard H</span>, <span class="author">Lieb W</span>, <span class="author">Cambien F</span>, <span class="author">Fischer M</span>, <span class="author">Ouwehand W</span>, <span class="author">Blankenberg S</span>, <span class="author">Balmforth AJ</span>, <span class="author">Baessler A</span>, <span class="author">Ball SG</span>, <span class="author">Strom TM</span>, <span class="author">Braenne I</span>, <span class="author">Gieger C</span>, <span class="author">Deloukas P</span>, <span class="author">Tobin MD</span>, <span class="author">Ziegler A</span>, <span class="author">Thompson JR</span>, <span class="author">Schunkert H</span> and <span class="author">WTCCC and the Cardiogenics Consortium</span></p>
<p class="affiliation">University of Leicester, Leicester, United Kingdom. <a href="mailto:%6E%6A%73%40%6C%65.%61%63.%75%6B">&#110;&#106;&#115;&#64;&#108;&#101;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Modern genotyping platforms permit a systematic search for inherited components of complex diseases. We performed a joint analysis of two genomewide association studies of coronary artery disease.</p><p class="abstract"><strong>Methods</strong>: We first identified chromosomal loci that were strongly associated with coronary artery disease in the Wellcome Trust Case Control Consortium (WTCCC) study (which involved 1926 case subjects with coronary artery disease and 2938 controls) and looked for replication in the German MI [Myocardial Infarction] Family Study (which involved 875 case subjects with myocardial infarction and 1644 controls). Data on other single-nucleotide polymorphisms (SNPs) that were significantly associated with coronary artery disease in either study (P&lt;0.001) were then combined to identify additional loci with a high probability of true association. Genotyping in both studies was performed with the use of the GeneChip Human Mapping 500K Array Set (Affymetrix).</p><p class="abstract"><strong>Results</strong>: Of thousands of chromosomal loci studied, the same locus had the strongest association with coronary artery disease in both the WTCCC and the German studies: chromosome 9p21.3 (SNP, rs1333049) (P=1.80x10(-14) and P=3.40x10(-6), respectively). Overall, the WTCCC study revealed nine loci that were strongly associated with coronary artery disease (P&lt;1.2x10(-5) and less than a 50% chance of being falsely positive). In addition to chromosome 9p21.3, two of these loci were successfully replicated (adjusted P&lt;0.05) in the German study: chromosome 6q25.1 (rs6922269) and chromosome 2q36.3 (rs2943634). The combined analysis of the two studies identified four additional loci significantly associated with coronary artery disease (P&lt;1.3x10(-6)) and a high probability (&gt;80%) of a true association: chromosomes 1p13.3 (rs599839), 1q41 (rs17465637), 10q11.21 (rs501120), and 15q22.33 (rs17228212).</p><p class="abstract"><strong>Conclusions</strong>: We identified several genetic loci that, individually and in aggregate, substantially affect the risk of development of coronary artery disease.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0501942</span>, <span class="grant">G9806740</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2007</span>;<span class="volume">357</span>;<span class="number">5</span>;<span class="pagerange">443-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17634449">17634449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2719290">2719290</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa072366">10.1056/NEJMoa072366</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary vignettes of natural killer cell receptors.</h4>
<p class="authors"><span class="author">Sambrook JG</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The discovery of novel immune receptors has led to a recent renaissance of research into the innate immune system, following decades of intense research of the adaptive immune system. Of particular interest has been the discovery of the natural killer (NK) cell receptors which, depending on type, interact with classical or non-classical MHC class I antigens of the adaptive immune system, thus functioning at the interface of innate and adaptive immunity. Here, we review recent progress with respect to two such families of NK receptors, the killer immunoglobulin-like receptors (KIRs) and the killer cell lectin-like receptors (KLRs), and attempt to trace their evolution across vertebrates.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in immunology</span>&nbsp;<span class="year">2007</span>;<span class="volume">19</span>;<span class="number">5</span>;<span class="pagerange">553-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17884420" rel="external">17884420</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.coi.2007.08.002" rel="external">10.1016/j.coi.2007.08.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Common variants in WFS1 confer risk of type 2 diabetes.</h4>
<p class="authors"><span class="author">Sandhu MS</span>, <span class="author">Weedon MN</span>, <span class="author">Fawcett KA</span>, <span class="author">Wasson J</span>, <span class="author">Debenham SL</span>, <span class="author">Daly A</span>, <span class="author">Lango H</span>, <span class="author">Frayling TM</span>, <span class="author">Neumann RJ</span>, <span class="author">Sherva R</span>, <span class="author">Blech I</span>, <span class="author">Pharoah PD</span>, <span class="author">Palmer CN</span>, <span class="author">Kimber C</span>, <span class="author">Tavendale R</span>, <span class="author">Morris AD</span>, <span class="author">McCarthy MI</span>, <span class="author">Walker M</span>, <span class="author">Hitman G</span>, <span class="author">Glaser B</span>, <span class="author">Permutt MA</span>, <span class="author">Hattersley AT</span>, <span class="author">Wareham NJ</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">UK Medical Research Council (MRC) Epidemiology Unit, Strangeways Research Laboratory, Cambridge CB1 8RN, UK. <a href="mailto:%6D%61%6E%6A.%73%61%6E%64%68%75%40%73%72%6C.%63%61%6D.%61%63.%75%6B">&#109;&#97;&#110;&#106;&#46;&#115;&#97;&#110;&#100;&#104;&#117;&#64;&#115;&#114;&#108;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We studied genes involved in pancreatic beta cell function and survival, identifying associations between SNPs in WFS1 and diabetes risk in UK populations that we replicated in an Ashkenazi population and in additional UK studies. In a pooled analysis comprising 9,533 cases and 11,389 controls, SNPs in WFS1 were strongly associated with diabetes risk. Rare mutations in WFS1 cause Wolfram syndrome; using a gene-centric approach, we show that variation in WFS1 also predisposes to common type 2 diabetes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0500070</span>, <span class="grant">MC_U106179471</span>; Wellcome Trust: <span class="grant">068545/z/02</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">8</span>;<span class="pagerange">951-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17603484">17603484</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672152">2672152</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2067">10.1038/ng2067</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Challenges and standards in integrating surveys of structural variation.</h4>
<p class="authors"><span class="author">Scherer SW</span>, <span class="author">Lee C</span>, <span class="author">Birney E</span>, <span class="author">Altshuler DM</span>, <span class="author">Eichler EE</span>, <span class="author">Carter NP</span>, <span class="author">Hurles ME</span> and <span class="author">Feuk L</span></p>
<p class="affiliation">The Centre for Applied Genomics and Program in Genetics and Genomic Biology, The Hospital for Sick Children, 101 College Street, Room 14-701, Ontario M5G 1L7, Canada. <a href="mailto:%73%74%65%76%65%40%67%65%6E%65%74.%73%69%63%6B%6B%69%64%73.%6F%6E.%63%61">&#115;&#116;&#101;&#118;&#101;&#64;&#103;&#101;&#110;&#101;&#116;&#46;&#115;&#105;&#99;&#107;&#107;&#105;&#100;&#115;&#46;&#111;&#110;&#46;&#99;&#97;</a></p><p class="abstract">There has been an explosion of data describing newly recognized structural variants in the human genome. In the flurry of reporting, there has been no standard approach to collecting the data, assessing its quality or describing identified features. This risks becoming a rampant problem, in particular with respect to surveys of copy number variation and their application to disease studies. Here, we consider the challenges in characterizing and documenting genomic structural variants. From this, we derive recommendations for standards to be adopted, with the aim of ensuring the accurate presentation of this form of genetic variation to facilitate ongoing research.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077008</span>, <span class="grant">077014</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7 Suppl</span>;<span class="pagerange">S7-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17597783">17597783</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2698291">2698291</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2093">10.1038/ng2093</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Application of phage display to high throughput antibody generation and characterization.</h4>
<p class="authors"><span class="author">Schofield DJ</span>, <span class="author">Pope AR</span>, <span class="author">Clementel V</span>, <span class="author">Buckell J</span>, <span class="author">Chapple SDj</span>, <span class="author">Clarke KF</span>, <span class="author">Conquer JS</span>, <span class="author">Crofts AM</span>, <span class="author">Crowther SR</span>, <span class="author">Dyson MR</span>, <span class="author">Flack G</span>, <span class="author">Griffin GJ</span>, <span class="author">Hooks Y</span>, <span class="author">Howat WJ</span>, <span class="author">Kolb-Kokocinski A</span>, <span class="author">Kunze S</span>, <span class="author">Martin CD</span>, <span class="author">Maslen GL</span>, <span class="author">Mitchell JN</span>, <span class="author">O&#39;Sullivan M</span>, <span class="author">Perera RL</span>, <span class="author">Roake W</span>, <span class="author">Shadbolt SP</span>, <span class="author">Vincent KJ</span>, <span class="author">Warford A</span>, <span class="author">Wilson WE</span>, <span class="author">Xie J</span>, <span class="author">Young JL</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">Abcam Ltd, Cambridge Science Park, Cambridge CB4 0FW, UK. <a href="mailto:%44%61%72%72%65%6E.%53%63%68%6F%66%69%65%6C%64%40%61%62%63%61%6D.%63%6F%6D">&#68;&#97;&#114;&#114;&#101;&#110;&#46;&#83;&#99;&#104;&#111;&#102;&#105;&#101;&#108;&#100;&#64;&#97;&#98;&#99;&#97;&#109;&#46;&#99;&#111;&#109;</a></p><p class="abstract">We have created a high quality phage display library containing over 1010 human antibodies and describe its use in the generation of antibodies on an unprecedented scale. We have selected, screened and sequenced over 38,000 recombinant antibodies to 292 antigens, yielding over 7,200 unique clones. 4,400 antibodies were characterized by specificity testing and detailed sequence analysis and the data/clones are available online. Sensitive detection was demonstrated in a bead based flow cytometry assay. Furthermore, positive staining by immunohistochemistry on tissue microarrays was found for 37% (143/381) of antibodies. Thus, we have demonstrated the potential of and illuminated the issues associated with genome-wide monoclonal antibody generation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">R254</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18047641" rel="external">18047641</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258204" rel="external">2258204</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-11-r254" rel="external">10.1186/gb-2007-8-11-r254</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An introduction to hidden Markov models.</h4>
<p class="authors"><span class="author">Schuster-B&ouml;ckler B</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">This unit introduces the concept of hidden Markov models in computational biology. It describes them using simple biological examples, requiring as little mathematical knowledge as possible. The unit also presents a brief history of hidden Markov models and an overview of their current applications before concluding with a discussion of their limitations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">Current protocols in bioinformatics / editoral board, Andreas D. Baxevanis ... [et al.]</span>&nbsp;<span class="year">2007</span>;<span class="volume">Appendix 3</span>;<span class="pagerange">Appendix 3A</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18428778" rel="external">18428778</a>; DOI: <a href="http://dx.doi.org/10.1002%2F0471250953.bia03as18" rel="external">10.1002/0471250953.bia03as18</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reuse of structural domain-domain interactions in protein networks.</h4>
<p class="authors"><span class="author">Schuster-B&ouml;ckler B</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%62%73%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#115;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Protein interactions are thought to be largely mediated by interactions between structural domains. Databases such as iPfam relate interactions in protein structures to known domain families. Here, we investigate how the domain interactions from the iPfam database are distributed in protein interactions taken from the HPRD, MPact, BioGRID, DIP and IntAct databases.</p><p class="abstract"><strong>Results</strong>: We find that known structural domain interactions can only explain a subset of 4-19% of the available protein interactions, nevertheless this fraction is still significantly bigger than expected by chance. There is a correlation between the frequency of a domain interaction and the connectivity of the proteins it occurs in. Furthermore, a large proportion of protein interactions can be attributed to a small number of domain interactions. We conclude that many, but not all, domain interactions constitute reusable modules of molecular recognition. A substantial proportion of domain interactions are conserved between E. coli, S. cerevisiae and H. sapiens. These domains are related to essential cellular functions, suggesting that many domain interactions were already present in the last universal common ancestor.</p><p class="abstract"><strong>Conclusion</strong>: Our results support the concept of domain interactions as reusable, conserved building blocks of protein interactions, but also highlight the limitations currently imposed by the small number of available protein structures.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">087656</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="pagerange">259</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17640363" rel="external">17640363</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1940023" rel="external">1940023</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-8-259" rel="external">10.1186/1471-2105-8-259</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The original Lujan syndrome family has a novel missense mutation (p.N1007S) in the MED12 gene.</h4>
<p class="authors"><span class="author">Schwartz CE</span>, <span class="author">Tarpey PS</span>, <span class="author">Lubs HA</span>, <span class="author">Verloes A</span>, <span class="author">May MM</span>, <span class="author">Risheg H</span>, <span class="author">Friez MJ</span>, <span class="author">Futreal PA</span>, <span class="author">Edkins S</span>, <span class="author">Teague J</span>, <span class="author">Briault S</span>, <span class="author">Skinner C</span>, <span class="author">Bauer-Carlin A</span>, <span class="author">Simensen RJ</span>, <span class="author">Joseph SM</span>, <span class="author">Jones JR</span>, <span class="author">Gecz J</span>, <span class="author">Stratton MR</span>, <span class="author">Raymond FL</span> and <span class="author">Stevenson RE</span></p><p class="abstract">A novel missense mutation in the mediator of RNA polymerase II transcription subunit 12 (MED12) gene has been found in the original family with Lujan syndrome and in a second family (K9359) that was initially considered to have Opitz-Kaveggia (FG) syndrome. A different missense mutation in the MED12 gene has been reported previously in the original family with FG syndrome and in five other families with compatible clinical findings. Neither sequence alteration has been found in over 1400 control X chromosomes. Lujan (Lujan-Fryns) syndrome is characterised by tall stature with asthenic habitus, macrocephaly, a tall narrow face, maxillary hypoplasia, a high narrow palate with dental crowding, a small or receding chin, long hands with hyperextensible digits, hypernasal speech, hypotonia, mild-to-moderate mental retardation, behavioural aberrations and dysgenesis of the corpus callosum. Although Lujan syndrome has not been previously considered to be in the differential diagnosis of FG syndrome, there are some overlapping clinical manifestations. Specifically, these are dysgenesis of the corpus callosum, macrocephaly/relative macrocephaly, a tall forehead, hypotonia, mental retardation and behavioural disturbances. Thus, it seems that these two X-linked mental retardation syndromes are allelic, with mutations in the MED12 gene.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD 26202</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">44</span>;<span class="number">7</span>;<span class="pagerange">472-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17369503" rel="external">17369503</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2597996" rel="external">2597996</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2006.048637" rel="external">10.1136/jmg.2006.048637</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">JAK2 exon 12 mutations in polycythemia vera and idiopathic erythrocytosis.</h4>
<p class="authors"><span class="author">Scott LM</span>, <span class="author">Tong W</span>, <span class="author">Levine RL</span>, <span class="author">Scott MA</span>, <span class="author">Beer PA</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Erber WN</span>, <span class="author">McMullin MF</span>, <span class="author">Harrison CN</span>, <span class="author">Warren AJ</span>, <span class="author">Gilliland DG</span>, <span class="author">Lodish HF</span> and <span class="author">Green AR</span></p>
<p class="affiliation">University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract"><strong>Background</strong>: The V617F mutation, which causes the substitution of phenylalanine for valine at position 617 of the Janus kinase (JAK) 2 gene (JAK2), is often present in patients with polycythemia vera, essential thrombocythemia, and idiopathic myelofibrosis. However, the molecular basis of these myeloproliferative disorders in patients without the V617F mutation is unclear.</p><p class="abstract"><strong>Methods</strong>: We searched for new mutations in members of the JAK and signal transducer and activator of transcription (STAT) gene families in patients with V617F-negative polycythemia vera or idiopathic erythrocytosis. The mutations were characterized biochemically and in a murine model of bone marrow transplantation.</p><p class="abstract"><strong>Results</strong>: We identified four somatic gain-of-function mutations affecting JAK2 exon 12 in 10 V617F-negative patients. Those with a JAK2 exon 12 mutation presented with an isolated erythrocytosis and distinctive bone marrow morphology, and several also had reduced serum erythropoietin levels. Erythroid colonies could be grown from their blood samples in the absence of exogenous erythropoietin. All such erythroid colonies were heterozygous for the mutation, whereas colonies homozygous for the mutation occur in most patients with V617F-positive polycythemia vera. BaF3 cells expressing the murine erythropoietin receptor and also carrying exon 12 mutations could proliferate without added interleukin-3. They also exhibited increased phosphorylation of JAK2 and extracellular regulated kinase 1 and 2, as compared with cells transduced by wild-type JAK2 or V617F JAK2. Three of the exon 12 mutations included a substitution of leucine for lysine at position 539 of JAK2. This mutation resulted in a myeloproliferative phenotype, including erythrocytosis, in a murine model of retroviral bone marrow transplantation.</p><p class="abstract"><strong>Conclusions</strong>: JAK2 exon 12 mutations define a distinctive myeloproliferative syndrome that affects patients who currently receive a diagnosis of polycythemia vera or idiopathic erythrocytosis.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA66996</span>, <span class="grant">K01 CA115679</span>; NHLBI NIH HHS: <span class="grant">P01 HL32262</span>; NIDDK NIH HHS: <span class="grant">DK50654</span>; Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">The New England journal of medicine</span>&nbsp;<span class="year">2007</span>;<span class="volume">356</span>;<span class="number">5</span>;<span class="pagerange">459-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17267906" rel="external">17267906</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2873834" rel="external">2873834</a>; DOI: <a href="http://dx.doi.org/10.1056%2FNEJMoa065202" rel="external">10.1056/NEJMoa065202</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of a proteolytic (Group I) Clostridium botulinum strain Hall A and comparative analysis of the clostridial genomes.</h4>
<p class="authors"><span class="author">Sebaihia M</span>, <span class="author">Peck MW</span>, <span class="author">Minton NP</span>, <span class="author">Thomson NR</span>, <span class="author">Holden MT</span>, <span class="author">Mitchell WJ</span>, <span class="author">Carter AT</span>, <span class="author">Bentley SD</span>, <span class="author">Mason DR</span>, <span class="author">Crossman L</span>, <span class="author">Paul CJ</span>, <span class="author">Ivens A</span>, <span class="author">Wells-Bennik MH</span>, <span class="author">Davis IJ</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Churcher C</span>, <span class="author">Quail MA</span>, <span class="author">Chillingworth T</span>, <span class="author">Feltwell T</span>, <span class="author">Fraser A</span>, <span class="author">Goodhead I</span>, <span class="author">Hance Z</span>, <span class="author">Jagels K</span>, <span class="author">Larke N</span>, <span class="author">Maddison M</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Norbertczak H</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Sanders M</span>, <span class="author">Simmonds M</span>, <span class="author">White B</span>, <span class="author">Whithead S</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, United Kingdom;</p><p class="abstract">Clostridium botulinum is a heterogeneous Gram-positive species that comprises four genetically and physiologically distinct groups of bacteria that share the ability to produce botulinum neurotoxin, the most poisonous toxin known to man, and the causative agent of botulism, a severe disease of humans and animals. We report here the complete genome sequence of a representative of Group I (proteolytic) C. botulinum (strain Hall A, ATCC 3502). The genome consists of a chromosome (3,886,916 bp) and a plasmid (16,344 bp), which carry 3650 and 19 predicted genes, respectively. Consistent with the proteolytic phenotype of this strain, the genome harbors a large number of genes encoding secreted proteases and enzymes involved in uptake and metabolism of amino acids. The genome also reveals a hitherto unknown ability of C. botulinum to degrade chitin. There is a significant lack of recently acquired DNA, indicating a stable genomic content, in strong contrast to the fluid genome of Clostridium difficile, which can form longer-term relationships with its host. Overall, the genome indicates that C. botulinum is adapted to a saprophytic lifestyle both in soil and aquatic environments. This pathogen relies on its toxin to rapidly kill a wide range of prey species, and to gain access to nutrient sources, it releases a large number of extracellular enzymes to soften and destroy rotting or decayed tissues.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700837</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">7</span>;<span class="pagerange">1082-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17519437">17519437</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1899119">1899119</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.6282807">10.1101/gr.6282807</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new environmentally resistant cell type from Dictyostelium.</h4>
<p class="authors"><span class="author">Serafimidis I</span>, <span class="author">Bloomfield G</span>, <span class="author">Skelton J</span>, <span class="author">Ivens A</span> and <span class="author">Kay RR</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge CB2 2QH, UK.</p><p class="abstract">This paper describes the serendipitous discovery and first characterization of a new resistant cell type from Dictyostelium, for which the name aspidocyte (from aspis: Greek for shield) is proposed. These cells are induced from amoebae by a range of toxins including heavy metals and antibiotics, and were first detected by their striking resistance to detergent lysis. Aspidocytes are separate, rounded or irregular-shaped cells, which are immotile but remain fully viable; once the toxic stress is removed, they revert to amoeboid cells within an hour. Induction takes a few hours and is completely blocked by the protein synthesis inhibitor cycloheximide. Aspidocytes lack a cell wall and their resistance to detergent lysis is active, requiring continued energy metabolism, and may be assisted by a complete cessation of endocytosis, as measured by uptake of the dye FM1-43. Microarray analysis shows that aspidocytes have a distinct pattern of gene expression, with a number of genes up-regulated that are predicted to be involved in lipid metabolism. Aspidocytes were initially detected in a hypersensitive mutant, in which the AMP deaminase gene is disrupted, suggesting that the inductive pathway involves AMP levels or metabolism. Since aspidocytes can also be induced from wild-type cells and are much more resistant than amoebae to a membrane-disrupting antibiotic, it is possible that they are an adaptation allowing Dictyostelium cells to survive a sudden onslaught of toxins in the wild.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">066742</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">153</span>;<span class="number">Pt 2</span>;<span class="pagerange">619-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17259634" rel="external">17259634</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2786962" rel="external">2786962</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.2006%2F000562-0" rel="external">10.1099/mic.0.2006/000562-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A more convenient truth.</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">4</span>;<span class="pagerange">248-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17366694">17366694</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1644">10.1038/nrmicro1644</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ocean&#39;s elevenses.</h4>
<p class="authors"><span class="author">Seth-Smith H</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17171852">17171852</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1589">10.1038/nrmicro1589</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Different evolutionary histories of the two classical class I genes BF1 and BF2 illustrate drift and selection within the stable MHC haplotypes of chickens.</h4>
<p class="authors"><span class="author">Shaw I</span>, <span class="author">Powell TJ</span>, <span class="author">Marston DA</span>, <span class="author">Baker K</span>, <span class="author">van Hateren A</span>, <span class="author">Riegert P</span>, <span class="author">Wiles MV</span>, <span class="author">Milne S</span>, <span class="author">Beck S</span> and <span class="author">Kaufman J</span></p>
<p class="affiliation">Institute for Animal Health, Compton, Berkshire, United Kingdom.</p><p class="abstract">Compared with the MHC of typical mammals, the chicken MHC (BF/BL region) of the B12 haplotype is smaller, simpler, and rearranged, with two classical class I genes of which only one is highly expressed. In this study, we describe the development of long-distance PCR to amplify some or all of each class I gene separately, allowing us to make the following points. First, six other haplotypes have the same genomic organization as B12, with a poorly expressed (minor) BF1 gene between DMB2 and TAP2 and a well-expressed (major) BF2 gene between TAP2 and C4. Second, the expression of the BF1 gene is crippled in three different ways in these haplotypes: enhancer A deletion (B12, B19), enhancer A divergence and transcription start site deletion (B2, B4, B21), and insertion/rearrangement leading to pseudogenes (B14, B15). Third, the three kinds of alterations in the BF1 gene correspond to dendrograms of the BF1 and poorly expressed class II B (BLB1) genes reflecting mostly neutral changes, while the dendrograms of the BF2 and well-expressed class II (BLB2) genes each have completely different topologies reflecting selection. The common pattern for the poorly expressed genes reflects the fact the BF/BL region undergoes little recombination and allows us to propose a pattern of descent for these chicken MHC haplotypes from a common ancestor. Taken together, these data explain how stable MHC haplotypes predominantly express a single class I molecule, which in turn leads to striking associations of the chicken MHC with resistance to infectious pathogens and response to vaccines.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2007</span>;<span class="volume">178</span>;<span class="number">9</span>;<span class="pagerange">5744-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17442958">17442958</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genes flanking Xist in mouse and human are separated on the X chromosome in American marsupials.</h4>
<p class="authors"><span class="author">Shevchenko AI</span>, <span class="author">Zakharova IS</span>, <span class="author">Elisaphenko EA</span>, <span class="author">Kolesnikov NN</span>, <span class="author">Whitehead S</span>, <span class="author">Bird C</span>, <span class="author">Ross M</span>, <span class="author">Weidman JR</span>, <span class="author">Jirtle RL</span>, <span class="author">Karamysheva TV</span>, <span class="author">Rubtsov NB</span>, <span class="author">VandeBerg JL</span>, <span class="author">Mazurok NA</span>, <span class="author">Nesterova TB</span>, <span class="author">Brockdorff N</span> and <span class="author">Zakian SM</span></p>
<p class="affiliation">Institute of Cytology and Genetics, Russian Academy of Sciences, Siberian Department, Novosibirsk, Russia.</p><p class="abstract">X inactivation, the transcriptional silencing of one of the two X chromosomes in female mammals, achieves dosage compensation of X-linked genes relative to XY males. In eutherian mammals X inactivation is regulated by the X-inactive specific transcript (Xist), a cis-acting non-coding RNA that triggers silencing of the chromosome from which it is transcribed. Marsupial mammals also undergo X inactivation but the mechanism is relatively poorly understood. We set out to analyse the X chromosome in Monodelphis domestica and Didelphis virginiana, focusing on characterizing the interval defined by the Chic1 and Slc16a2 genes that in eutherians flank the Xist locus. The synteny of this region is retained on chicken chromosome 4 where other loci belonging to the evolutionarily ancient stratum of the human X chromosome, the so-called X conserved region (XCR), are also located. We show that in both M. domestica and D. virginiana an evolutionary breakpoint has separated the Chic1 and Slc16a2 loci. Detailed analysis of opossum genomic sequences revealed linkage of Chic1 with the Lnx3 gene, recently proposed to be the evolutionary precursor of Xist, and Fip1, the evolutionary precursor of Tsx, a gene located immediately downstream of Xist in eutherians. We discuss these findings in relation to the evolution of Xist and X inactivation in mammals.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">067065/Z/02/Z</span></p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">2</span>;<span class="pagerange">127-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17333537" rel="external">17333537</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2797855" rel="external">2797855</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-006-1115-9" rel="external">10.1007/s10577-006-1115-9</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal evolution of Arvicolinae (Cricetidae, Rodentia). I. The genome homology of tundra vole, field vole, mouse and golden hamster revealed by comparative chromosome painting.</h4>
<p class="authors"><span class="author">Sitnikova NA</span>, <span class="author">Romanenko SA</span>, <span class="author">O&#39;Brien PC</span>, <span class="author">Perelman PL</span>, <span class="author">Fu B</span>, <span class="author">Rubtsova NV</span>, <span class="author">Serdukova NA</span>, <span class="author">Golenishchev FN</span>, <span class="author">Trifonov VA</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Yang F</span> and <span class="author">Graphodatsky AS</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS, Novosibirsk, Russia.</p><p class="abstract">Cross-species chromosome painting has become the mainstay of comparative cytogenetic and chromosome evolution studies. Here we have made a set of chromosomal painting probes for the field vole (Microtus agrestis) by DOP-PCR amplification of flow-sorted chromosomes. Together with painting probes of golden hamster (Mesocricetus auratus) and mouse (Mus musculus), the field vole probes have been hybridized onto the metaphases of the tundra vole (Microtus oeconomus). A comparative chromosome map between these two voles, golden hamster and mouse has been established based on the results of cross-species chromosome painting and G-banding comparisons. The sets of paints from the field vole, golden hamster and mouse identified a total of 27, 40 and 47 homologous autosomal regions, respectively, in the genome of tundra vole; 16, 41 and 51 fusion/fission rearrangements differentiate the karyotype of the tundra vole from the karyotypes of the field vole, golden hamster and mouse, respectively.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">4</span>;<span class="pagerange">447-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17497247" rel="external">17497247</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-007-1137-y" rel="external">10.1007/s10577-007-1137-y</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics: from genotype to disease phenotype in the leishmaniases.</h4>
<p class="authors"><span class="author">Smith DF</span>, <span class="author">Peacock CS</span> and <span class="author">Cruz AK</span></p>
<p class="affiliation">Immunology and Infection Unit, Department of Biology/Hull York Medical School, University of York, Heslington, York YO10 5YW, UK. <a href="mailto:%64%66%73%35%30%31%40%79%6F%72%6B.%61%63.%75%6B">&#100;&#102;&#115;&#53;&#48;&#49;&#64;&#121;&#111;&#114;&#107;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Recent progress in sequencing the genomes of several Leishmania species, causative agents of cutaneous, mucocutaneous and visceral leishmaniasis, is revealing unusual features of potential relevance to parasite virulence and pathogenesis in the host. While the genomes of Leishmania major, Leishmania braziliensis and Leishmania infantum are highly similar in content and organisation, species-specific genes and mechanisms distinguish one from another. In particular, the presence of retrotransposons and the components of a putative RNA interference machinery in L. braziliensis suggest the potential for both greater diversity and more tractable experimentation in this Leishmania Viannia species.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076355</span></p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2007</span>;<span class="volume">37</span>;<span class="number">11</span>;<span class="pagerange">1173-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17645880" rel="external">17645880</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2696322" rel="external">2696322</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ijpara.2007.05.015" rel="external">10.1016/j.ijpara.2007.05.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Flowering of strict photoperiodic Nicotiana varieties in non-inductive conditions by transgenic approaches.</h4>
<p class="authors"><span class="author">Smykal P</span>, <span class="author">Gennen J</span>, <span class="author">De Bodt S</span>, <span class="author">Ranganath V</span> and <span class="author">Melzer S</span></p>
<p class="affiliation">Institute of Plant Sciences, ETH Z&uuml;rich, Universitaetstrasse 2, 8092, Zurich, Switzerland.</p><p class="abstract">The genus Nicotiana contains species and varieties that respond differently to photoperiod for flowering time control as day-neutral, short-day and long-day plants. In classical photoperiodism studies, these varieties have been widely used to analyse the physiological nature for floral induction by day length. Since key regulators for flowering time control by day length have been identified in Arabidopsis thaliana by molecular genetic studies, it was intriguing to analyse how closely related plants in the Nicotiana genus with opposite photoperiodic requirements respond to certain flowering time regulators. SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1 (SOC1) and FRUITFULL (FUL) are two MADS box genes that are involved in the regulation of flowering time in Arabidopsis. SOC1 is a central flowering time pathway integrator, whereas the exact role of FUL for floral induction has not been established yet. The putative Nicotiana orthologs of SOC1 and FUL, NtSOC1 and NtFUL, were studied in day-neutral tobacco Nicotiana tabacum cv Hicks, in short-day tobacco N. tabacum cv Hicks Maryland Mammoth (MM) and long-day N. sylvestris plants. Both genes were similarly expressed under short- and long-day conditions in day-neutral and short-day tobaccos, but showed a different expression pattern in N. sylvestris. Overexpression of NtSOC1 and NtFUL caused flowering either in strict short-day (NtSOC1) or long-day (NtFUL) Nicotiana varieties under non-inductive photoperiods, indicating that these genes might be limiting for floral induction under non-inductive conditions in different Nicotiana varieties.</p>
<p class="publication"><span class="title">Plant molecular biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">65</span>;<span class="number">3</span>;<span class="pagerange">233-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17660946" rel="external">17660946</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs11103-007-9211-6" rel="external">10.1007/s11103-007-9211-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pharmacogenomics of non-small cell lung cancer</h4>
<p class="authors"><span class="author">Spicer J</span></p>
<p class="publication"><span class="title">Current Pharmacogenomics.</span> <span class="year">2007</span>;<span class="volume">5</span>;<span class="pagerange">228-234</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Discovery of functional elements in 12 Drosophila genomes using evolutionary signatures.</h4>
<p class="authors"><span class="author">Stark A</span>, <span class="author">Lin MF</span>, <span class="author">Kheradpour P</span>, <span class="author">Pedersen JS</span>, <span class="author">Parts L</span>, <span class="author">Carlson JW</span>, <span class="author">Crosby MA</span>, <span class="author">Rasmussen MD</span>, <span class="author">Roy S</span>, <span class="author">Deoras AN</span>, <span class="author">Ruby JG</span>, <span class="author">Brennecke J</span>, <span class="author">Harvard FlyBase curators</span>, <span class="author">Berkeley Drosophila Genome Project</span>, <span class="author">Hodges E</span>, <span class="author">Hinrichs AS</span>, <span class="author">Caspi A</span>, <span class="author">Paten B</span>, <span class="author">Park SW</span>, <span class="author">Han MV</span>, <span class="author">Maeder ML</span>, <span class="author">Polansky BJ</span>, <span class="author">Robson BE</span>, <span class="author">Aerts S</span>, <span class="author">van Helden J</span>, <span class="author">Hassan B</span>, <span class="author">Gilbert DG</span>, <span class="author">Eastman DA</span>, <span class="author">Rice M</span>, <span class="author">Weir M</span>, <span class="author">Hahn MW</span>, <span class="author">Park Y</span>, <span class="author">Dewey CN</span>, <span class="author">Pachter L</span>, <span class="author">Kent WJ</span>, <span class="author">Haussler D</span>, <span class="author">Lai EC</span>, <span class="author">Bartel DP</span>, <span class="author">Hannon GJ</span>, <span class="author">Kaufman TC</span>, <span class="author">Eisen MB</span>, <span class="author">Clark AG</span>, <span class="author">Smith D</span>, <span class="author">Celniker SE</span>, <span class="author">Gelbart WM</span> and <span class="author">Kellis M</span></p>
<p class="affiliation">The Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02140, USA.</p><p class="abstract">Sequencing of multiple related species followed by comparative genomics analysis constitutes a powerful approach for the systematic understanding of any genome. Here, we use the genomes of 12 Drosophila species for the de novo discovery of functional elements in the fly. Each type of functional element shows characteristic patterns of change, or &#39;evolutionary signatures&#39;, dictated by its precise selective constraints. Such signatures enable recognition of new protein-coding genes and exons, spurious and incorrect gene annotations, and numerous unusual gene structures, including abundant stop-codon readthrough. Similarly, we predict non-protein-coding RNA genes and structures, and new microRNA (miRNA) genes. We provide evidence of miRNA processing and functionality from both hairpin arms and both DNA strands. We identify several classes of pre- and post-transcriptional regulatory motifs, and predict individual motif instances with high confidence. We also study how discovery power scales with the divergence and number of species compared, and we provide general guidelines for comparative studies.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG002779-05</span>, <span class="grant">R01 HG002779-06</span>, <span class="grant">R01 HG004037</span>, <span class="grant">R01 HG004037-01A1</span>; NIGMS NIH HHS: <span class="grant">R01 GM067031</span>, <span class="grant">R01 GM067031-04</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">450</span>;<span class="number">7167</span>;<span class="pagerange">219-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17994088">17994088</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2474711">2474711</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature06340">10.1038/nature06340</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Salmonella enterica serovar typhimurium exploits inflammation to compete with the intestinal microbiota.</h4>
<p class="authors"><span class="author">Stecher B</span>, <span class="author">Robbiani R</span>, <span class="author">Walker AW</span>, <span class="author">Westendorf AM</span>, <span class="author">Barthel M</span>, <span class="author">Kremer M</span>, <span class="author">Chaffron S</span>, <span class="author">Macpherson AJ</span>, <span class="author">Buer J</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span>, <span class="author">von Mering C</span> and <span class="author">Hardt WD</span></p>
<p class="affiliation">Institute of Microbiology, Swiss Institute of Technology Zurich, Zurich, Switzerland.</p><p class="abstract">Most mucosal surfaces of the mammalian body are colonized by microbial communities (&quot;microbiota&quot;). A high density of commensal microbiota inhabits the intestine and shields from infection (&quot;colonization resistance&quot;). The virulence strategies allowing enteropathogenic bacteria to successfully compete with the microbiota and overcome colonization resistance are poorly understood. Here, we investigated manipulation of the intestinal microbiota by the enteropathogenic bacterium Salmonella enterica subspecies 1 serovar Typhimurium (S. Tm) in a mouse colitis model: we found that inflammatory host responses induced by S. Tm changed microbiota composition and suppressed its growth. In contrast to wild-type S. Tm, an avirulent invGsseD mutant failing to trigger colitis was outcompeted by the microbiota. This competitive defect was reverted if inflammation was provided concomitantly by mixed infection with wild-type S. Tm or in mice (IL10(-/-), VILLIN-HA(CL4-CD8)) with inflammatory bowel disease. Thus, inflammation is necessary and sufficient for overcoming colonization resistance. This reveals a new concept in infectious disease: in contrast to current thinking, inflammation is not always detrimental for the pathogen. Triggering the host&#39;s immune defence can shift the balance between the protective microbiota and the pathogen in favour of the pathogen.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">2177-89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17760501">17760501</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1951780">1951780</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0050244">10.1371/journal.pbio.0050244</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Conservation and divergence of gene families encoding components of innate immune response systems in zebrafish.</h4>
<p class="authors"><span class="author">Stein C</span>, <span class="author">Caccamo M</span>, <span class="author">Laird G</span> and <span class="author">Leptin M</span></p>
<p class="affiliation">Institute for Genetics, University of Cologne, Zuelpicher Str, 47, 50674 Cologne, Germany. <a href="mailto:%63%6F%72%6E%65%6C%69%61.%73%74%65%69%6E%40%75%6E%69-%6B%6F%65%6C%6E.%64%65">&#99;&#111;&#114;&#110;&#101;&#108;&#105;&#97;&#46;&#115;&#116;&#101;&#105;&#110;&#64;&#117;&#110;&#105;&#45;&#107;&#111;&#101;&#108;&#110;&#46;&#100;&#101;</a></p><p class="abstract"><strong>Background</strong>: The zebrafish has become a widely used model to study disease resistance and immunity. Although the genes encoding many components of immune signaling pathways have been found in teleost fish, it is not clear whether all components are present or whether the complexity of the signaling mechanisms employed by mammals is similar in fish.</p><p class="abstract"><strong>Results</strong>: We searched the genomes of the zebrafish Danio rerio and two pufferfish for genes encoding components of the Toll-like receptor and interferon signaling pathways, the NLR (NACHT-domain and leucine rich repeat containing) protein family, and related proteins. We find that most of the components known in mammals are also present in fish, with clearly recognizable orthologous relationships. The class II cytokines and their receptors have diverged extensively, obscuring orthologies, but the number of receptors is similar in all species analyzed. In the family of the NLR proteins, the canonical members are conserved. We also found a conserved NACHT-domain protein with WD40 repeats that had previously not been described in mammals. Additionally, we have identified in each of the three fish a large species-specific subgroup of NLR proteins that contain a novel amino-terminal domain that is not found in mammalian genomes.</p><p class="abstract"><strong>Conclusion</strong>: The main innate immune signaling pathways are conserved in mammals and teleost fish. Whereas the components that act downstream of the receptors are highly conserved, with orthologous sets of genes in mammals and teleosts, components that are known or assumed to interact with pathogens are more divergent and have undergone lineage-specific expansions.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">R251</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18039395" rel="external">18039395</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258186" rel="external">2258186</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-11-r251" rel="external">10.1186/gb-2007-8-11-r251</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reductive evolution and niche adaptation inferred from the genome of Mycobacterium ulcerans, the causative agent of Buruli ulcer.</h4>
<p class="authors"><span class="author">Stinear TP</span>, <span class="author">Seemann T</span>, <span class="author">Pidot S</span>, <span class="author">Frigui W</span>, <span class="author">Reysset G</span>, <span class="author">Garnier T</span>, <span class="author">Meurice G</span>, <span class="author">Simon D</span>, <span class="author">Bouchier C</span>, <span class="author">Ma L</span>, <span class="author">Tichit M</span>, <span class="author">Porter JL</span>, <span class="author">Ryan J</span>, <span class="author">Johnson PD</span>, <span class="author">Davies JK</span>, <span class="author">Jenkin GA</span>, <span class="author">Small PL</span>, <span class="author">Jones LM</span>, <span class="author">Tekaia F</span>, <span class="author">Laval F</span>, <span class="author">Daff&eacute; M</span>, <span class="author">Parkhill J</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne, Institut Pasteur, 75725 Paris Cedex 15, France.</p><p class="abstract">Mycobacterium ulcerans is found in aquatic ecosystems and causes Buruli ulcer in humans, a neglected but devastating necrotic disease of subcutaneous tissue that is rampant throughout West and Central Africa. Here, we report the complete 5.8-Mb genome sequence of M. ulcerans and show that it comprises two circular replicons, a chromosome of 5632 kb and a virulence plasmid of 174 kb. The plasmid is required for production of the polyketide toxin mycolactone, which provokes necrosis. Comparisons with the recently completed 6.6-Mb genome of Mycobacterium marinum revealed &gt;98% nucleotide sequence identity and genome-wide synteny. However, as well as the plasmid, M. ulcerans has accumulated 213 copies of the insertion sequence IS2404, 91 copies of IS2606, 771 pseudogenes, two bacteriophages, and multiple DNA deletions and rearrangements. These data indicate that M. ulcerans has recently evolved via lateral gene transfer and reductive evolution from the generalist, more rapid-growing environmental species M. marinum to become a niche-adapted specialist. Predictions based on genome inspection for the production of modified mycobacterial virulence factors, such as the highly abundant phthiodiolone lipids, were confirmed by structural analyses. Similarly, 11 protein-coding sequences identified as M. ulcerans-specific by comparative genomics were verified as such by PCR screening a diverse collection of 33 strains of M. ulcerans and M. marinum. This work offers significant insight into the biology and evolution of mycobacterial pathogens and is an important component of international efforts to counter Buruli ulcer.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">2</span>;<span class="pagerange">192-200</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17210928" rel="external">17210928</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1781351" rel="external">1781351</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5942807" rel="external">10.1101/gr.5942807</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Relative impact of nucleotide and copy number variation on gene expression phenotypes.</h4>
<p class="authors"><span class="author">Stranger BE</span>, <span class="author">Forrest MS</span>, <span class="author">Dunning M</span>, <span class="author">Ingle CE</span>, <span class="author">Beazley C</span>, <span class="author">Thorne N</span>, <span class="author">Redon R</span>, <span class="author">Bird CP</span>, <span class="author">de Grassi A</span>, <span class="author">Lee C</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Carter N</span>, <span class="author">Scherer SW</span>, <span class="author">Tavar&eacute; S</span>, <span class="author">Deloukas P</span>, <span class="author">Hurles ME</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">Extensive studies are currently being performed to associate disease susceptibility with one form of genetic variation, namely, single-nucleotide polymorphisms (SNPs). In recent years, another type of common genetic variation has been characterized, namely, structural variation, including copy number variants (CNVs). To determine the overall contribution of CNVs to complex phenotypes, we have performed association analyses of expression levels of 14,925 transcripts with SNPs and CNVs in individuals who are part of the International HapMap project. SNPs and CNVs captured 83.6% and 17.7% of the total detected genetic variation in gene expression, respectively, but the signals from the two types of variation had little overlap. Interrogation of the genome for both types of variants may be an effective way to elucidate the causes of complex phenotypes and disease in humans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">065535</span>, <span class="grant">076113</span>, <span class="grant">077009</span>, <span class="grant">077014</span>, <span class="grant">077046</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">315</span>;<span class="number">5813</span>;<span class="pagerange">848-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17289997">17289997</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2665772">2665772</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1136678">10.1126/science.1136678</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population genomics of human gene expression.</h4>
<p class="authors"><span class="author">Stranger BE</span>, <span class="author">Nica AC</span>, <span class="author">Forrest MS</span>, <span class="author">Dimas A</span>, <span class="author">Bird CP</span>, <span class="author">Beazley C</span>, <span class="author">Ingle CE</span>, <span class="author">Dunning M</span>, <span class="author">Flicek P</span>, <span class="author">Koller D</span>, <span class="author">Montgomery S</span>, <span class="author">Tavar&eacute; S</span>, <span class="author">Deloukas P</span> and <span class="author">Dermitzakis ET</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Genetic variation influences gene expression, and this variation in gene expression can be efficiently mapped to specific genomic regions and variants. Here we have used gene expression profiling of Epstein-Barr virus-transformed lymphoblastoid cell lines of all 270 individuals genotyped in the HapMap Consortium to elucidate the detailed features of genetic variation underlying gene expression variation. We find that gene expression is heritable and that differentiation between populations is in agreement with earlier small-scale studies. A detailed association analysis of over 2.2 million common SNPs per population (5% frequency in HapMap) with gene expression identified at least 1,348 genes with association signals in cis and at least 180 in trans. Replication in at least one independent population was achieved for 37% of cis signals and 15% of trans signals, respectively. Our results strongly support an abundance of cis-regulatory variation in the human genome. Detection of trans effects is limited but suggests that regulatory variation may be the key primary effect contributing to phenotypic variation in humans. We also explore several methodologies that improve the current state of analysis of gene expression variation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077011</span>, <span class="grant">077046</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">10</span>;<span class="pagerange">1217-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17873874">17873874</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2683249">2683249</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2142">10.1038/ng2142</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proteomic and microarray analyses of the Dictyostelium Zak1-GSK-3 signaling pathway reveal a role in early development.</h4>
<p class="authors"><span class="author">Strmecki L</span>, <span class="author">Bloomfield G</span>, <span class="author">Araki T</span>, <span class="author">Dalton E</span>, <span class="author">Skelton J</span>, <span class="author">Schilde C</span>, <span class="author">Harwood A</span>, <span class="author">Williams JG</span>, <span class="author">Ivens A</span> and <span class="author">Pears C</span></p>
<p class="affiliation">Biochemistry Department, Oxford University, South Parks Rd., Oxford OX1 3QU, United Kingdom.</p><p class="abstract">GskA, the Dictyostelium GSK-3 orthologue, is modified and activated by the dual-specificity tyrosine kinase Zak1, and the two kinases form part of a signaling pathway that responds to extracellular cyclic AMP. We identify potential cellular effectors for the two kinases by analyzing the corresponding null mutants. There are proteins and mRNAs that are altered in abundance in only one or the other of the two mutants, indicating that each kinase has some unique functions. However, proteomic and microarray analyses identified a number of proteins and genes, respectively, that are similarly misregulated in both mutant strains. The positive correlation between the array data and the proteomic data is consistent with the Zak1-GskA signaling pathway&#39;s functioning by directly or indirectly regulating gene expression. The discoidin 1 genes are positively regulated by the pathway, while the abundance of the H5 protein is negatively regulated. Two of the targets, H5 and discoidin 1, are well-characterized markers for early development, indicating that the Zak1-GskA pathway plays a role in development earlier than previously observed.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">053640/Z</span>, <span class="grant">063612</span>, <span class="grant">064724</span></p>
<p class="publication"><span class="title">Eukaryotic cell</span>&nbsp;<span class="year">2007</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">245-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17085634" rel="external">17085634</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1797958" rel="external">1797958</a>; DOI: <a href="http://dx.doi.org/10.1128%2FEC.00204-06" rel="external">10.1128/EC.00204-06</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication timing profile reflects the distinct functional and genomic features of the MHC class II region.</h4>
<p class="authors"><span class="author">Takousis P</span>, <span class="author">Johonnett P</span>, <span class="author">Williamson J</span>, <span class="author">Sasieni P</span>, <span class="author">Warnes G</span>, <span class="author">Forshew T</span>, <span class="author">Azuara V</span>, <span class="author">Fisher A</span>, <span class="author">Wu PJ</span>, <span class="author">Jones T</span>, <span class="author">Vatcheva R</span>, <span class="author">Beck S</span> and <span class="author">Sheer D</span></p>
<p class="affiliation">Human Cytogenetics Laboratory, Cancer Research, UK London Research Institute, London, UK.</p><p class="abstract">The timing of DNA replication generally correlates with transcription, gene density and sequence composition. How is the timing affected if a genomic region has a combination of features that individually correlate with either early or late replication? The major histocompatibility complex (MHC) class II region is an AT-rich isochore that would be expected to replicate late, but it also contains coordinately regulated genes that are highly expressed in antigen-presenting cells and are strongly inducible in other cell types. Using cytological and biochemical assays, we find that the entire MHC replicates within the first half of S-phase, and that the class II region replicates slightly later than the adjacent regions irrespective of gene expression. These data suggest that despite AT-richness, an early-to-middle replication time in the class II region is defined by an open chromatin conformation that allows rapid transcriptional activation as a defence against pathogens.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A8318</span>, <span class="grant">C5321/A8318</span>; Medical Research Council: <span class="grant">MC_U120027516</span></p>
<p class="publication"><span class="title">Cell cycle (Georgetown, Tex.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">6</span>;<span class="number">19</span>;<span class="pagerange">2393-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17938583">17938583</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of genetic variation in Akt2/PKB-beta in severe insulin resistance, lipodystrophy, type 2 diabetes, and related metabolic phenotypes.</h4>
<p class="authors"><span class="author">Tan K</span>, <span class="author">Kimber WA</span>, <span class="author">Luan J</span>, <span class="author">Soos MA</span>, <span class="author">Semple RK</span>, <span class="author">Wareham NJ</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Metabolic Disease Group, Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, U.K.</p><p class="abstract">We previously reported a family in which a heterozygous missense mutation in Akt2 led to a dominantly inherited syndrome of insulin-resistant diabetes and partial lipodystrophy. To determine whether genetic variation in AKT2 plays a broader role in human metabolic disease, we sequenced the entire coding region and splice junctions of AKT2 in 94 unrelated patients with severe insulin resistance, 35 of whom had partial lipodystrophy. Two rare missense mutations (R208K and R467W) were identified in single individuals. However, insulin-stimulated kinase activities of these variants were indistinguishable from wild type. In two large case-control studies (total number of participants 2,200), 0 of 11 common single nucleotide polymorphism (SNPs) in AKT2 showed significant association with type 2 diabetes. In a quantitative trait study of 1,721 extensively phenotyped individuals from the U.K., no association was found with any relevant intermediate metabolic trait. In summary, although heterozygous loss-of- function mutations in AKT2 can cause a syndrome of severe insulin resistance and lipodystrophy in humans, such mutations are uncommon causes of these syndromes. Furthermore, genetic variation in and around the AKT2 locus is unlikely to contribute significantly to the risk of type 2 diabetes or related intermediate metabolic traits in U.K. populations.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U106179471</span>; Wellcome Trust: <span class="grant">077016</span></p>
<p class="publication"><span class="title">Diabetes</span>&nbsp;<span class="year">2007</span>;<span class="volume">56</span>;<span class="number">3</span>;<span class="pagerange">714-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17327441">17327441</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672155">2672155</a>; DOI: <a href="http://dx.doi.org/10.2337%2Fdb06-0921">10.2337/db06-0921</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel Gln358Glu mutation in ectodysplasin A associated with X-linked dominant incisor hypodontia.</h4>
<p class="authors"><span class="author">Tarpey P</span>, <span class="author">Pemberton TJ</span>, <span class="author">Stockton DW</span>, <span class="author">Das P</span>, <span class="author">Ninis V</span>, <span class="author">Edkins S</span>, <span class="author">Andrew Futreal P</span>, <span class="author">Wooster R</span>, <span class="author">Kamath S</span>, <span class="author">Nayak R</span>, <span class="author">Stratton MR</span> and <span class="author">Patel PI</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">C06 CA62528-01</span>; NCRR NIH HHS: <span class="grant">C06 RR10600-01</span>, <span class="grant">C06 RR14514-01</span>; NIDCR NIH HHS: <span class="grant">DE014102</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2007</span>;<span class="volume">143</span>;<span class="number">4</span>;<span class="pagerange">390-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17256800" rel="external">17256800</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.31567" rel="external">10.1002/ajmg.a.31567</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in UPF3B, a member of the nonsense-mediated mRNA decay complex, cause syndromic and nonsyndromic mental retardation.</h4>
<p class="authors"><span class="author">Tarpey PS</span>, <span class="author">Raymond FL</span>, <span class="author">Nguyen LS</span>, <span class="author">Rodriguez J</span>, <span class="author">Hackett A</span>, <span class="author">Vandeleur L</span>, <span class="author">Smith R</span>, <span class="author">Shoubridge C</span>, <span class="author">Edkins S</span>, <span class="author">Stevens C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Tofts C</span>, <span class="author">Barthorpe S</span>, <span class="author">Buck G</span>, <span class="author">Cole J</span>, <span class="author">Halliday K</span>, <span class="author">Hills K</span>, <span class="author">Jones D</span>, <span class="author">Mironenko T</span>, <span class="author">Perry J</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Teague J</span>, <span class="author">Dicks E</span>, <span class="author">Butler A</span>, <span class="author">Menzies A</span>, <span class="author">Richardson D</span>, <span class="author">Jenkinson A</span>, <span class="author">Shepherd R</span>, <span class="author">Raine K</span>, <span class="author">Moon J</span>, <span class="author">Luo Y</span>, <span class="author">Parnau J</span>, <span class="author">Bhat SS</span>, <span class="author">Gardner A</span>, <span class="author">Corbett M</span>, <span class="author">Brooks D</span>, <span class="author">Thomas P</span>, <span class="author">Parkinson-Lawrence E</span>, <span class="author">Porteous ME</span>, <span class="author">Warner JP</span>, <span class="author">Sanderson T</span>, <span class="author">Pearson P</span>, <span class="author">Simensen RJ</span>, <span class="author">Skinner C</span>, <span class="author">Hoganson G</span>, <span class="author">Superneau D</span>, <span class="author">Wooster R</span>, <span class="author">Bobrow M</span>, <span class="author">Turner G</span>, <span class="author">Stevenson RE</span>, <span class="author">Schwartz CE</span>, <span class="author">Futreal PA</span>, <span class="author">Srivastava AK</span>, <span class="author">Stratton MR</span> and <span class="author">G&eacute;cz J</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Nonsense-mediated mRNA decay (NMD) is of universal biological significance. It has emerged as an important global RNA, DNA and translation regulatory pathway. By systematically sequencing 737 genes (annotated in the Vertebrate Genome Annotation database) on the human X chromosome in 250 families with X-linked mental retardation, we identified mutations in the UPF3 regulator of nonsense transcripts homolog B (yeast) (UPF3B) leading to protein truncations in three families: two with the Lujan-Fryns phenotype and one with the FG phenotype. We also identified a missense mutation in another family with nonsyndromic mental retardation. Three mutations lead to the introduction of a premature termination codon and subsequent NMD of mutant UPF3B mRNA. Protein blot analysis using lymphoblastoid cell lines from affected individuals showed an absence of the UPF3B protein in two families. The UPF3B protein is an important component of the NMD surveillance machinery. Our results directly implicate abnormalities of NMD in human disease and suggest at least partial redundancy of NMD pathways.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD26202</span>; Wellcome Trust: <span class="grant">077012</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">9</span>;<span class="pagerange">1127-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17704778">17704778</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2872770">2872770</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2100">10.1038/ng2100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in CUL4B, which encodes a ubiquitin E3 ligase subunit, cause an X-linked mental retardation syndrome associated with aggressive outbursts, seizures, relative macrocephaly, central obesity, hypogonadism, pes cavus, and tremor.</h4>
<p class="authors"><span class="author">Tarpey PS</span>, <span class="author">Raymond FL</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Edkins S</span>, <span class="author">Teague J</span>, <span class="author">Butler A</span>, <span class="author">Dicks E</span>, <span class="author">Stevens C</span>, <span class="author">Tofts C</span>, <span class="author">Avis T</span>, <span class="author">Barthorpe S</span>, <span class="author">Buck G</span>, <span class="author">Cole J</span>, <span class="author">Gray K</span>, <span class="author">Halliday K</span>, <span class="author">Harrison R</span>, <span class="author">Hills K</span>, <span class="author">Jenkinson A</span>, <span class="author">Jones D</span>, <span class="author">Menzies A</span>, <span class="author">Mironenko T</span>, <span class="author">Perry J</span>, <span class="author">Raine K</span>, <span class="author">Richardson D</span>, <span class="author">Shepherd R</span>, <span class="author">Small A</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Mallya U</span>, <span class="author">Moon J</span>, <span class="author">Luo Y</span>, <span class="author">Holder S</span>, <span class="author">Smithson SF</span>, <span class="author">Hurst JA</span>, <span class="author">Clayton-Smith J</span>, <span class="author">Kerr B</span>, <span class="author">Boyle J</span>, <span class="author">Shaw M</span>, <span class="author">Vandeleur L</span>, <span class="author">Rodriguez J</span>, <span class="author">Slaugh R</span>, <span class="author">Easton DF</span>, <span class="author">Wooster R</span>, <span class="author">Bobrow M</span>, <span class="author">Srivastava AK</span>, <span class="author">Stevenson RE</span>, <span class="author">Schwartz CE</span>, <span class="author">Turner G</span>, <span class="author">Gecz J</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span> and <span class="author">Partington M</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We have identified three truncating, two splice-site, and three missense variants at conserved amino acids in the CUL4B gene on Xq24 in 8 of 250 families with X-linked mental retardation (XLMR). During affected subjects&#39; adolescence, a syndrome emerged with delayed puberty, hypogonadism, relative macrocephaly, moderate short stature, central obesity, unprovoked aggressive outbursts, fine intention tremor, pes cavus, and abnormalities of the toes. This syndrome was first described by Cazebas et al., in a family that was included in our study and that carried a CUL4B missense variant. CUL4B is a ubiquitin E3 ligase subunit implicated in the regulation of several biological processes, and CUL4B is the first XLMR gene that encodes an E3 ubiquitin ligase. The relatively high frequency of CUL4B mutations in this series indicates that it is one of the most commonly mutated genes underlying XLMR and suggests that its introduction into clinical diagnostics should be a high priority.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD26202</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">80</span>;<span class="number">2</span>;<span class="pagerange">345-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17236139" rel="external">17236139</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1785336" rel="external">1785336</a>; DOI: <a href="http://dx.doi.org/10.1086%2F511134" rel="external">10.1086/511134</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ProteomeBinders: planning a European resource of affinity reagents for analysis of the human proteome.</h4>
<p class="authors"><span class="author">Taussig MJ</span>, <span class="author">Stoevesandt O</span>, <span class="author">Borrebaeck CA</span>, <span class="author">Bradbury AR</span>, <span class="author">Cahill D</span>, <span class="author">Cambillau C</span>, <span class="author">de Daruvar A</span>, <span class="author">D&uuml;bel S</span>, <span class="author">Eichler J</span>, <span class="author">Frank R</span>, <span class="author">Gibson TJ</span>, <span class="author">Gloriam D</span>, <span class="author">Gold L</span>, <span class="author">Herberg FW</span>, <span class="author">Hermjakob H</span>, <span class="author">Hoheisel JD</span>, <span class="author">Joos TO</span>, <span class="author">Kallioniemi O</span>, <span class="author">Koegl M</span>, <span class="author">Koegll M</span>, <span class="author">Konthur Z</span>, <span class="author">Korn B</span>, <span class="author">Kremmer E</span>, <span class="author">Krobitsch S</span>, <span class="author">Landegren U</span>, <span class="author">van der Maarel S</span>, <span class="author">McCafferty J</span>, <span class="author">Muyldermans S</span>, <span class="author">Nygren PA</span>, <span class="author">Palcy S</span>, <span class="author">Pl&uuml;ckthun A</span>, <span class="author">Polic B</span>, <span class="author">Przybylski M</span>, <span class="author">Saviranta P</span>, <span class="author">Sawyer A</span>, <span class="author">Sherman DJ</span>, <span class="author">Skerra A</span>, <span class="author">Templin M</span>, <span class="author">Ueffing M</span> and <span class="author">Uhl&eacute;n M</span></p>
<p class="affiliation">Technology Research Group, The Babraham Institute, Cambridge CB22 3AT, UK. <a href="mailto:%6D%69%6B%65.%74%61%75%73%73%69%67%40%62%62%73%72%63.%61%63.%75%6B">&#109;&#105;&#107;&#101;&#46;&#116;&#97;&#117;&#115;&#115;&#105;&#103;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">ProteomeBinders is a new European consortium aiming to establish a comprehensive resource of well-characterized affinity reagents, including but not limited to antibodies, for analysis of the human proteome. Given the huge diversity of the proteome, the scale of the project is potentially immense but nevertheless feasible in the context of a pan-European or even worldwide coordination.</p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2007</span>;<span class="volume">4</span>;<span class="number">1</span>;<span class="pagerange">13-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17195019">17195019</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth0107-13">10.1038/nmeth0107-13</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genotype calling algorithm for the Illumina BeadArray platform.</h4>
<p class="authors"><span class="author">Teo YY</span>, <span class="author">Inouye M</span>, <span class="author">Small KS</span>, <span class="author">Gwilliam R</span>, <span class="author">Deloukas P</span>, <span class="author">Kwiatkowski DP</span> and <span class="author">Clark TG</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. <a href="mailto:%74%65%6F%40%77%65%6C%6C.%6F%78.%61%63.%75%6B">&#116;&#101;&#111;&#64;&#119;&#101;&#108;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: Large-scale genotyping relies on the use of unsupervised automated calling algorithms to assign genotypes to hybridization data. A number of such calling algorithms have been recently established for the Affymetrix GeneChip genotyping technology. Here, we present a fast and accurate genotype calling algorithm for the Illumina BeadArray genotyping platforms. As the technology moves towards assaying millions of genetic polymorphisms simultaneously, there is a need for an integrated and easy-to-use software for calling genotypes.</p><p class="abstract"><strong>Results</strong>: We have introduced a model-based genotype calling algorithm which does not rely on having prior training data or require computationally intensive procedures. The algorithm can assign genotypes to hybridization data from thousands of individuals simultaneously and pools information across multiple individuals to improve the calling. The method can accommodate variations in hybridization intensities which result in dramatic shifts of the position of the genotype clouds by identifying the optimal coordinates to initialize the algorithm. By incorporating the process of perturbation analysis, we can obtain a quality metric measuring the stability of the assigned genotype calls. We show that this quality metric can be used to identify SNPs with low call rates and accuracy.</p><p class="abstract"><strong>Availability</strong>: The C++ executable for the algorithm described here is available by request from the authors.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0600230</span>, <span class="grant">G19/9</span>; Wellcome Trust: <span class="grant">077011</span>, <span class="grant">082370</span></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">20</span>;<span class="pagerange">2741-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17846035">17846035</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2666488">2666488</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm443">10.1093/bioinformatics/btm443</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence-based analysis of pQBR103; a representative of a unique, transfer-proficient mega plasmid resident in the microbial community of sugar beet.</h4>
<p class="authors"><span class="author">Tett A</span>, <span class="author">Spiers AJ</span>, <span class="author">Crossman LC</span>, <span class="author">Ager D</span>, <span class="author">Ciric L</span>, <span class="author">Dow JM</span>, <span class="author">Fry JC</span>, <span class="author">Harris D</span>, <span class="author">Lilley A</span>, <span class="author">Oliver A</span>, <span class="author">Parkhill J</span>, <span class="author">Quail MA</span>, <span class="author">Rainey PB</span>, <span class="author">Saunders NJ</span>, <span class="author">Seeger K</span>, <span class="author">Snyder LA</span>, <span class="author">Squares R</span>, <span class="author">Thomas CM</span>, <span class="author">Turner SL</span>, <span class="author">Zhang XX</span>, <span class="author">Field D</span> and <span class="author">Bailey MJ</span></p>
<p class="affiliation">Centre for Ecology and Hydrology-Oxford, Oxford, UK.</p><p class="abstract">The plasmid pQBR103 was found within Pseudomonas populations colonizing the leaf and root surfaces of sugar beet plants growing at Wytham, Oxfordshire, UK. At 425 kb it is the largest self-transmissible plasmid yet sequenced from the phytosphere. It is known to enhance the competitive fitness of its host, and parts of the plasmid are known to be actively transcribed in the plant environment. Analysis of the complete sequence of this plasmid predicts a coding sequence (CDS)-rich genome containing 478 CDSs and an exceptional degree of genetic novelty; 80% of predicted coding sequences cannot be ascribed a function and 60% are orphans. Of those to which function could be assigned, 40% bore greatest similarity to sequences from Pseudomonas spp, and the majority of the remainder showed similarity to other gamma-proteobacterial genera and plasmids. pQBR103 has identifiable regions presumed responsible for replication and partitioning, but despite being tra+ lacks the full complement of any previously described conjugal transfer functions. The DNA sequence provided few insights into the functional significance of plant-induced transcriptional regions, but suggests that 14% of CDSs may be expressed (11 CDSs with functional annotation and 54 without), further highlighting the ecological importance of these novel CDSs. Comparative analysis indicates that pQBR103 shares significant regions of sequence with other plasmids isolated from sugar beet plants grown at the same geographic location. These plasmid sequences indicate there is more novelty in the mobile DNA pool accessible to phytosphere pseudomonas than is currently appreciated or understood.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">082372</span></p>
<p class="publication"><span class="title">The ISME journal</span>&nbsp;<span class="year">2007</span>;<span class="volume">1</span>;<span class="number">4</span>;<span class="pagerange">331-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18043644">18043644</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656933">2656933</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fismej.2007.47">10.1038/ismej.2007.47</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A cytogenetically characterized, genome-anchored 10-Mb BAC set and CGH array for the domestic dog.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Duke SE</span>, <span class="author">Bloom SK</span>, <span class="author">Breen TE</span>, <span class="author">Young AC</span>, <span class="author">Feiste E</span>, <span class="author">Seiser EL</span>, <span class="author">Tsai PC</span>, <span class="author">Langford CF</span>, <span class="author">Ellis P</span>, <span class="author">Karlsson EK</span>, <span class="author">Lindblad-Toh K</span> and <span class="author">Breen M</span></p>
<p class="affiliation">Department of Molecular Biomedical Sciences, College of Veterinary Medicine, North Carolina State University, 4700 Hillsborough Street, Raleigh, NC 27606, USA.</p><p class="abstract">The generation of a 7.5x dog genome assembly provides exciting new opportunities to interpret tumor-associated chromosome aberrations at the biological level. We present a genomic microarray for array comparative genomic hybridization (aCGH) analysis in the dog, comprising 275 bacterial artificial chromosome (BAC) clones spaced at intervals of approximately 10 Mb. Each clone has been positioned accurately within the genome assembly and assigned to a unique chromosome location by fluorescence in situ hybridization (FISH) analysis, both individually and as chromosome-specific BAC pools. The microarray also contains clones representing the dog orthologues of 31 genes implicated in human cancers. FISH analysis of the 10-Mb BAC clone set indicated excellent coverage of each dog chromosome by the genome assembly. The order of clones was consistent with the assembly, but the cytogenetic intervals between clones were variable. We demonstrate the application of the BAC array for aCGH analysis to identify both whole and partial chromosome imbalances using a canine histiocytic sarcoma case. Using BAC clones selected from the array as probes, multicolor FISH analysis was used to further characterize these imbalances, revealing numerous structural chromosome rearrangements. We outline the value of a combined aCGH/FISH approach, together with a well-annotated dog genome assembly, in canine and comparative cancer studies.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">R21 NS051190-01</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of heredity</span>&nbsp;<span class="year">2007</span>;<span class="volume">98</span>;<span class="number">5</span>;<span class="pagerange">474-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17702974" rel="external">17702974</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fjhered%2Fesm053" rel="external">10.1093/jhered/esm053</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genome analyses of the pathogenic Yersiniae based on the genome sequence of Yersinia enterocolitica strain 8081.</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Howard S</span>, <span class="author">Wren BW</span> and <span class="author">Prentice MB</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, Wellcome Trust Genome Campus, The Wellcome Trust Sanger Institute, Cambridge, UK. <a href="mailto:%6E%72%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#114;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This chapter represents a summary of the findings from the Yersinia enterocolitica strain 8081 whole genome sequence and the associated microarray analysis. Section 1 &amp; 2 provide an introduction to the species and an overview of the general features of the genome. Section 3 identifies important regions within the genome which highlight important differences in gene function that separate the three pathogenic Yersinias. Section 4 describes genomic loci conferring important, species-specific, metabolic and virulence traits. Section 5 details extensive microarray data to provide an overview of species-specific core Y. enterocolitica gene functions and important insights into the intra-species differences between the high, low and non-pathogenic Y. enterocolitica biotypes.</p>
<p class="publication"><span class="title">Advances in experimental medicine and biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">603</span>;<span class="pagerange">2-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17966400" rel="external">17966400</a>; DOI: <a href="http://dx.doi.org/10.1007%2F978-0-387-72124-8_1" rel="external">10.1007/978-0-387-72124-8_1</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Rheumatoid arthritis association at 6q23.</h4>
<p class="authors"><span class="author">Thomson W</span>, <span class="author">Barton A</span>, <span class="author">Ke X</span>, <span class="author">Eyre S</span>, <span class="author">Hinks A</span>, <span class="author">Bowes J</span>, <span class="author">Donn R</span>, <span class="author">Symmons D</span>, <span class="author">Hider S</span>, <span class="author">Bruce IN</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Wilson AG</span>, <span class="author">Marinou I</span>, <span class="author">Morgan A</span>, <span class="author">Emery P</span>, <span class="author">YEAR Consortium</span>, <span class="author">Carter A</span>, <span class="author">Steer S</span>, <span class="author">Hocking L</span>, <span class="author">Reid DM</span>, <span class="author">Wordsworth P</span>, <span class="author">Harrison P</span>, <span class="author">Strachan D</span> and <span class="author">Worthington J</span></p>
<p class="affiliation">Arthritis Research Campaign (arc)-Epidemiology Unit, Stopford Building, The University of Manchester, Manchester M13 9PT, UK.</p><p class="abstract">The Wellcome Trust Case Control Consortium (WTCCC) identified nine single SNPs putatively associated with rheumatoid arthritis at P = 1 x 10(-5) - 5 x 10(-7) in a genome-wide association screen. One, rs6920220, was unequivocally replicated (trend P = 1.1 x 10(-8)) in a validation study, as described here. This SNP maps to 6q23, between the genes oligodendrocyte lineage transcription factor 3 (OLIG3) and tumor necrosis factor-alpha-induced protein 3 (TNFAIP3).</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0000934(68341)</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">076113</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">12</span>;<span class="pagerange">1431-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17982455">17982455</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2674282">2674282</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2007.32">10.1038/ng.2007.32</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole-genome array-CGH for detection of submicroscopic chromosomal imbalances in children with mental retardation.</h4>
<p class="authors"><span class="author">Thuresson AC</span>, <span class="author">Bondeson ML</span>, <span class="author">Edeby C</span>, <span class="author">Ellis P</span>, <span class="author">Langford C</span>, <span class="author">Dumanski JP</span> and <span class="author">Anner&eacute;n G</span></p>
<p class="affiliation">Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, Uppsala, Sweden. <a href="mailto:%61%6E%6E-%63%68%61%72%6C%6F%74%74%65.%74%68%75%72%65%73%73%6F%6E%40%67%65%6E%70%61%74.%75%75.%73%65">&#97;&#110;&#110;&#45;&#99;&#104;&#97;&#114;&#108;&#111;&#116;&#116;&#101;&#46;&#116;&#104;&#117;&#114;&#101;&#115;&#115;&#111;&#110;&#64;&#103;&#101;&#110;&#112;&#97;&#116;&#46;&#117;&#117;&#46;&#115;&#101;</a></p><p class="abstract">Chromosomal imbalances are the major cause of mental retardation (MR). Many of these imbalances are caused by submicroscopic deletions or duplications not detected by conventional cytogenetic methods. Microarray-based comparative genomic hybridization (array-CGH) is considered to be superior for the investigation of chromosomal aberrations in children with MR, and has been demonstrated to improve the diagnostic detection rate of these small chromosomal abnormalities. In this study we used 1 Mb genome-wide array-CGH to screen 48 children with MR and congenital malformations for submicroscopic chromosomal imbalances, where the underlying cause was unknown. All children were clinically investigated and subtelomere FISH analysis had been performed in all cases. Suspected microdeletion syndromes such as deletion 22q11.2, Williams-Beuren and Angelman syndromes were excluded before array-CGH analysis was performed. We identified de novo interstitial chromosomal imbalances in two patients (4%), and an interstitial deletion inherited from an affected mother in one patient (2%). In another two of the children (4%), suspected imbalances were detected but were also found in one of the non-affected parents. The yield of identified de novo alterations detected in this study is somewhat less than previously described, and might reflect the importance of which selection criterion of patients to be used before array-CGH analysis is performed. However, array-CGH proved to be a high-quality and reliable tool for genome-wide screening of MR patients of unknown etiology.</p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">118</span>;<span class="number">1</span>;<span class="pagerange">1-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17901693" rel="external">17901693</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2874679" rel="external">2874679</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000106434" rel="external">10.1159/000106434</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Convergent adaptation of human lactase persistence in Africa and Europe.</h4>
<p class="authors"><span class="author">Tishkoff SA</span>, <span class="author">Reed FA</span>, <span class="author">Ranciaro A</span>, <span class="author">Voight BF</span>, <span class="author">Babbitt CC</span>, <span class="author">Silverman JS</span>, <span class="author">Powell K</span>, <span class="author">Mortensen HM</span>, <span class="author">Hirbo JB</span>, <span class="author">Osman M</span>, <span class="author">Ibrahim M</span>, <span class="author">Omar SA</span>, <span class="author">Lema G</span>, <span class="author">Nyambo TB</span>, <span class="author">Ghori J</span>, <span class="author">Bumpstead S</span>, <span class="author">Pritchard JK</span>, <span class="author">Wray GA</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Department of Biology, University of Maryland, College Park, Maryland 20742, USA. <a href="mailto:%54%69%73%68%6B%6F%66%66%40%75%6D%64.%65%64%75">&#84;&#105;&#115;&#104;&#107;&#111;&#102;&#102;&#64;&#117;&#109;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">A SNP in the gene encoding lactase (LCT) (C/T-13910) is associated with the ability to digest milk as adults (lactase persistence) in Europeans, but the genetic basis of lactase persistence in Africans was previously unknown. We conducted a genotype-phenotype association study in 470 Tanzanians, Kenyans and Sudanese and identified three SNPs (G/C-14010, T/G-13915 and C/G-13907) that are associated with lactase persistence and that have derived alleles that significantly enhance transcription from the LCT promoter in vitro. These SNPs originated on different haplotype backgrounds from the European C/T-13910 SNP and from each other. Genotyping across a 3-Mb region demonstrated haplotype homozygosity extending &gt;2.0 Mb on chromosomes carrying C-14010, consistent with a selective sweep over the past approximately 7,000 years. These data provide a marked example of convergent evolution due to strong selective pressure resulting from shared cultural traits-animal domestication and adult milk consumption.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">F32HG03801</span>, <span class="grant">HG002772-1</span>; NIGMS NIH HHS: <span class="grant">R01GM076637</span>; Wellcome Trust: <span class="grant">076113</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">1</span>;<span class="pagerange">31-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17159977">17159977</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2672153">2672153</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1946">10.1038/ng1946</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Burkholderia Hep_Hag autotransporter (BuHA) proteins elicit a strong antibody response during experimental glanders but not human melioidosis.</h4>
<p class="authors"><span class="author">Tiyawisutsri R</span>, <span class="author">Holden MT</span>, <span class="author">Tumapa S</span>, <span class="author">Rengpipat S</span>, <span class="author">Clarke SR</span>, <span class="author">Foster SJ</span>, <span class="author">Nierman WC</span>, <span class="author">Day NP</span> and <span class="author">Peacock SJ</span></p>
<p class="affiliation">Faculty of Tropical Medicine, Mahidol University, Bangkok, Thailand. <a href="mailto:%72%61%63%68%61%6E%65%65%70%6F%72%6E%5F%74%40%68%6F%74%6D%61%69%6C.%63%6F%6D">&#114;&#97;&#99;&#104;&#97;&#110;&#101;&#101;&#112;&#111;&#114;&#110;&#95;&#116;&#64;&#104;&#111;&#116;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a> &lt;<a href="mailto:%72%61%63%68%61%6E%65%65%70%6F%72%6E%5F%74%40%68%6F%74%6D%61%69%6C.%63%6F%6D&%67%74">&#114;&#97;&#99;&#104;&#97;&#110;&#101;&#101;&#112;&#111;&#114;&#110;&#95;&#116;&#64;&#104;&#111;&#116;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;&amp;&#103;&#116;</a>;</p><p class="abstract"><strong>Background</strong>: The bacterial biothreat agents Burkholderia mallei and Burkholderia pseudomallei are the cause of glanders and melioidosis, respectively. Genomic and epidemiological studies have shown that B. mallei is a recently emerged, host restricted clone of B. pseudomallei.</p><p class="abstract"><strong>Results</strong>: Using bacteriophage-mediated immunoscreening we identified genes expressed in vivo during experimental equine glanders infection. A family of immunodominant antigens were identified that share protein domain architectures with hemagglutinins and invasins. These have been designated Burkholderia Hep_Hag autotransporter (BuHA) proteins. A total of 110/207 positive clones (53%) of a B. mallei expression library screened with sera from two infected horses belonged to this family. This contrasted with 6/189 positive clones (3%) of a B. pseudomallei expression library screened with serum from 21 patients with culture-proven melioidosis.</p><p class="abstract"><strong>Conclusion</strong>: Members of the BuHA proteins are found in other Gram-negative bacteria and have been shown to have important roles related to virulence. Compared with other bacterial species, the genomes of both B. mallei and B. pseudomallei contain a relative abundance of this family of proteins. The domain structures of these proteins suggest that they function as multimeric surface proteins that modulate interactions of the cell with the host and environment. Their effect on the cellular immune response to B. mallei and their potential as diagnostics for glanders requires further study.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="pagerange">19</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17362501" rel="external">17362501</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1847439" rel="external">1847439</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-7-19" rel="external">10.1186/1471-2180-7-19</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes.</h4>
<p class="authors"><span class="author">Todd JA</span>, <span class="author">Walker NM</span>, <span class="author">Cooper JD</span>, <span class="author">Smyth DJ</span>, <span class="author">Downes K</span>, <span class="author">Plagnol V</span>, <span class="author">Bailey R</span>, <span class="author">Nejentsev S</span>, <span class="author">Field SF</span>, <span class="author">Payne F</span>, <span class="author">Lowe CE</span>, <span class="author">Szeszko JS</span>, <span class="author">Hafler JP</span>, <span class="author">Zeitels L</span>, <span class="author">Yang JH</span>, <span class="author">Vella A</span>, <span class="author">Nutland S</span>, <span class="author">Stevens HE</span>, <span class="author">Schuilenburg H</span>, <span class="author">Coleman G</span>, <span class="author">Maisuria M</span>, <span class="author">Meadows W</span>, <span class="author">Smink LJ</span>, <span class="author">Healy B</span>, <span class="author">Burren OS</span>, <span class="author">Lam AA</span>, <span class="author">Ovington NR</span>, <span class="author">Allen J</span>, <span class="author">Adlem E</span>, <span class="author">Leung HT</span>, <span class="author">Wallace C</span>, <span class="author">Howson JM</span>, <span class="author">Guja C</span>, <span class="author">Ionescu-T&icirc;rgovi&#x15F;te C</span>, <span class="author">Genetics of Type 1 Diabetes in Finland</span>, <span class="author">Simmonds MJ</span>, <span class="author">Heward JM</span>, <span class="author">Gough SC</span>, <span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Dunger DB</span>, <span class="author">Wicker LS</span> and <span class="author">Clayton DG</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge CB2 0XY, UK. <a href="mailto:%6A%6F%68%6E.%74%6F%64%64%40%63%69%6D%72.%63%61%6D.%61%63.%75%6B">&#106;&#111;&#104;&#110;&#46;&#116;&#111;&#100;&#100;&#64;&#99;&#105;&#109;&#114;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Wellcome Trust Case Control Consortium (WTCCC) primary genome-wide association (GWA) scan on seven diseases, including the multifactorial autoimmune disease type 1 diabetes (T1D), shows associations at P &lt; 5 x 10(-7) between T1D and six chromosome regions: 12q24, 12q13, 16p13, 18p11, 12p13 and 4q27. Here, we attempted to validate these and six other top findings in 4,000 individuals with T1D, 5,000 controls and 2,997 family trios independent of the WTCCC study. We confirmed unequivocally the associations of 12q24, 12q13, 16p13 and 18p11 (P(follow-up) &lt;or= 1.35 x 10(-9); P(overall) &lt;or= 1.15 x 10(-14)), leaving eight regions with small effects or false-positive associations. We also obtained evidence for chromosome 18q22 (P(overall) = 1.38 x 10(-8)) from a GWA study of nonsynonymous SNPs. Several regions, including 18q22 and 18p11, showed association with autoimmune thyroid disease. This study increases the number of T1D loci with compelling evidence from six to at least ten.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">061858</span>, <span class="grant">061859</span>, <span class="grant">089989</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7</span>;<span class="pagerange">857-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17554260">17554260</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2492393">2492393</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2068">10.1038/ng2068</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Look who&#39;s talking too: graduates developing skills through communication.</h4>
<p class="authors"><span class="author">Tomazou EM</span> and <span class="author">Powell GT</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%65%6D%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#109;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Greater opportunities for young scientists to present their doctoral research to large general audiences will encourage development of transferable skills and involvement in the scientific community. We look at ways students communicate their research and explore the benefits of student-led meetings. The organization of the first Sanger-Cambridge Ph.D. Symposium provides an example of how students can act to establish forums for their work and we call on other young scientists to do the same.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">9</span>;<span class="pagerange">724-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17680006">17680006</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg2177">10.1038/nrg2177</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">IL23R variation determines susceptibility but not disease phenotype in inflammatory bowel disease.</h4>
<p class="authors"><span class="author">Tremelling M</span>, <span class="author">Cummings F</span>, <span class="author">Fisher SA</span>, <span class="author">Mansfield J</span>, <span class="author">Gwilliam R</span>, <span class="author">Keniry A</span>, <span class="author">Nimmo ER</span>, <span class="author">Drummond H</span>, <span class="author">Onnie CM</span>, <span class="author">Prescott NJ</span>, <span class="author">Sanderson J</span>, <span class="author">Bredin F</span>, <span class="author">Berzuini C</span>, <span class="author">Forbes A</span>, <span class="author">Lewis CM</span>, <span class="author">Cardon L</span>, <span class="author">Deloukas P</span>, <span class="author">Jewell D</span>, <span class="author">Mathew CG</span>, <span class="author">Parkes M</span> and <span class="author">Satsangi J</span></p>
<p class="affiliation">IBD Research Group, Addenbrooke&#39;s Hospital, University of Cambridge, Cambridge, England, UK</p><p class="abstract">Identification of inflammatory bowel disease (IBD) susceptibility genes is key to understanding pathogenic mechanisms. Recently, the North American IBD Genetics Consortium provided compelling evidence for an association between ileal Crohn&#39;s disease (CD) and the IL23R gene using genome-wide association scanning. External replication is a priority, both to confirm this finding in other populations and to validate this new technique. We tested for association between IL23R and IBD in a large independent UK panel to determine the size of the effect and explore subphenotype correlation and interaction with CARD15.</p><p class="abstract"><strong>Methods</strong>: Eight single nucleotide polymorphism markers in IL23R tested in the North American study were genotyped in 1902 cases of Crohn&#39;s disease (CD), 975 cases of ulcerative colitis (UC), and 1345 controls using MassARRAY. Data were analyzed using chi(2) statistics, and subgroup association was sought.</p><p class="abstract"><strong>Results</strong>: A highly significant association with CD was observed, with the strongest signal at coding variant Arg381Gln (allele frequency, 2.5% in CD vs 6.2% in controls [P = 1.1 x 10(-12)]; odds ratio, 0.38; 95% confidence interval, 0.29-0.50). A weaker effect was seen in UC (allele frequency, 4.6%; odds ratio, 0.73; 95% confidence interval, 0.55-0.96). Analysis accounting for Arg381Gln suggested that other loci within IL23R also influence IBD susceptibility. Within CD, there were no subphenotype associations or evidence of interaction with CARD15.</p><p class="abstract"><strong>Conclusions</strong>: This study shows an association between IL23R and all subphenotypes of CD with a smaller effect on UC. This extends the findings of the North American study, providing clear evidence that genome-wide association scanning can successfully identify true complex disease genes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934(68341)</span>; Wellcome Trust: <span class="grant">068545</span>, <span class="grant">068545/Z/02</span>, <span class="grant">077011</span></p>
<p class="publication"><span class="title">Gastroenterology</span>&nbsp;<span class="year">2007</span>;<span class="volume">132</span>;<span class="number">5</span>;<span class="pagerange">1657-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17484863" rel="external">17484863</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2696256" rel="external">2696256</a>; DOI: <a href="http://dx.doi.org/10.1053%2Fj.gastro.2007.02.051" rel="external">10.1053/j.gastro.2007.02.051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The implications of alternative splicing in the ENCODE protein complement.</h4>
<p class="authors"><span class="author">Tress ML</span>, <span class="author">Martelli PL</span>, <span class="author">Frankish A</span>, <span class="author">Reeves GA</span>, <span class="author">Wesselink JJ</span>, <span class="author">Yeats C</span>, <span class="author">Olason PI</span>, <span class="author">Albrecht M</span>, <span class="author">Hegyi H</span>, <span class="author">Giorgetti A</span>, <span class="author">Raimondo D</span>, <span class="author">Lagarde J</span>, <span class="author">Laskowski RA</span>, <span class="author">L&oacute;pez G</span>, <span class="author">Sadowski MI</span>, <span class="author">Watson JD</span>, <span class="author">Fariselli P</span>, <span class="author">Rossi I</span>, <span class="author">Nagy A</span>, <span class="author">Kai W</span>, <span class="author">St&oslash;rling Z</span>, <span class="author">Orsini M</span>, <span class="author">Assenov Y</span>, <span class="author">Blankenburg H</span>, <span class="author">Huthmacher C</span>, <span class="author">Ram&iacute;rez F</span>, <span class="author">Schlicker A</span>, <span class="author">Denoeud F</span>, <span class="author">Jones P</span>, <span class="author">Kerrien S</span>, <span class="author">Orchard S</span>, <span class="author">Antonarakis SE</span>, <span class="author">Reymond A</span>, <span class="author">Birney E</span>, <span class="author">Brunak S</span>, <span class="author">Casadio R</span>, <span class="author">Guigo R</span>, <span class="author">Harrow J</span>, <span class="author">Hermjakob H</span>, <span class="author">Jones DT</span>, <span class="author">Lengauer T</span>, <span class="author">Orengo CA</span>, <span class="author">Patthy L</span>, <span class="author">Thornton JM</span>, <span class="author">Tramontano A</span> and <span class="author">Valencia A</span></p>
<p class="affiliation">Structural Computational Biology Programme, Spanish National Cancer Research Centre, E-28029 Madrid, Spain. <a href="mailto:%6D%74%72%65%73%73%40%63%6E%69%6F.%65%73">&#109;&#116;&#114;&#101;&#115;&#115;&#64;&#99;&#110;&#105;&#111;&#46;&#101;&#115;</a></p><p class="abstract">Alternative premessenger RNA splicing enables genes to generate more than one gene product. Splicing events that occur within protein coding regions have the potential to alter the biological function of the expressed protein and even to create new protein functions. Alternative splicing has been suggested as one explanation for the discrepancy between the number of human genes and functional complexity. Here, we carry out a detailed study of the alternatively spliced gene products annotated in the ENCODE pilot project. We find that alternative splicing in human genes is more frequent than has commonly been suggested, and we demonstrate that many of the potential alternative gene products will have markedly different structure and function from their constitutively spliced counterparts. For the vast majority of these alternative isoforms, little evidence exists to suggest they have a role as functional proteins, and it seems unlikely that the spectrum of conventional enzymatic or structural functions can be substantially extended through alternative splicing.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062023</span>, <span class="grant">077198</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">13</span>;<span class="pagerange">5495-500</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17372197">17372197</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1838448">1838448</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0700800104">10.1073/pnas.0700800104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y Chromosome</h4>
<p class="authors"><span class="author">Tyler-Smith C</span></p>
<p class="publication"><span class="title">Encyclopedia of Life Sciences.</span> <span class="year">2007</span></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mitochondrial permeabilization relies on BH3 ligands engaging multiple prosurvival Bcl-2 relatives, not Bak.</h4>
<p class="authors"><span class="author">Uren RT</span>, <span class="author">Dewson G</span>, <span class="author">Chen L</span>, <span class="author">Coyne SC</span>, <span class="author">Huang DC</span>, <span class="author">Adams JM</span> and <span class="author">Kluck RM</span></p>
<p class="affiliation">The Walter and Eliza Hall Institute of Medical Research, Melbourne, Victoria 3050, Australia.</p><p class="abstract">The Bcl-2 family regulates apoptosis by controlling mitochondrial integrity. To clarify whether its prosurvival members function by sequestering their Bcl-2 homology 3 (BH3)-only ligands or their multidomain relatives Bak and Bax, we analyzed whether four prosurvival proteins differing in their ability to bind specific BH3 peptides or Bak could protect isolated mitochondria. Most BH3 peptides could induce temperature-dependent cytochrome c release, but permeabilization was prevented by Bcl-x(L), Bcl-w, Mcl-1, or BHRF1. However, their protection correlated with the ability to bind Bak rather than the added BH3 peptide and could be overcome only by BH3 peptides that bind directly to the appropriate prosurvival member. Mitochondria protected by both Bcl-x(L)-like and Mcl-1 proteins were disrupted only by BH3 peptides that engage both. BH3-only reagents freed Bak from Bcl-x(L) and Mcl-1 in mitochondrial and cell lysates. The findings support a model for the control of apoptosis in which certain prosurvival proteins sequester Bak/Bax, and BH3-only proteins must neutralize all protective prosurvival proteins to allow Bak/Bax to induce mitochondrial disruption.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA80188</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of cell biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">177</span>;<span class="number">2</span>;<span class="pagerange">277-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17452531" rel="external">17452531</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2064136" rel="external">2064136</a>; DOI: <a href="http://dx.doi.org/10.1083%2Fjcb.200606065" rel="external">10.1083/jcb.200606065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Clustered gene expression changes flank targeted gene loci in knockout mice.</h4>
<p class="authors"><span class="author">Valor LM</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, United Kingdom.</p><p class="abstract"><strong>Background</strong>: Gene expression profiling using microarrays is a powerful technology widely used to study regulatory networks. Profiling of mRNA levels in mutant organisms has the potential to identify genes regulated by the mutated protein.</p><p class="abstract">Using tissues from multiple lines of knockout mice we have examined genome-wide changes in gene expression. We report that a significant proportion of changed genes were found near the targeted gene.</p><p class="abstract">The apparent clustering of these genes was explained by the presence of flanking DNA from the parental ES cell. We provide recommendations for the analysis and reporting of microarray data from knockout mice.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">PloS one</span>&nbsp;<span class="year">2007</span>;<span class="volume">2</span>;<span class="number">12</span>;<span class="pagerange">e1303</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18074027" rel="external">18074027</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2110885" rel="external">2110885</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0001303" rel="external">10.1371/journal.pone.0001303</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrating synapse proteomics with transcriptional regulation.</h4>
<p class="authors"><span class="author">Valor LM</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">The mammalian postsynaptic proteome (PSP) comprises a highly interconnected set of approximately 1,000 proteins. The PSP is organized into macromolecular complexes that have a modular architecture defined by protein interactions and function. Signals initiated by neurotransmitter receptors are integrated by these complexes and their constituent enzymes to orchestrate multiple downstream cellular changes, including transcriptional regulation of genes at the nucleus. Genome wide transcriptome studies are beginning to map the sets of genes regulated by the synapse proteome. Conversely, understanding the transcriptional regulation of genes encoding the synapse proteome will shed light on synapse formation. Mutations that disrupt synapse signalling complexes result in cognitive impairments in mice and humans, and recent evidence indicates that these mutation change gene expression profiles. We discuss the need for global approaches combining genetics, transcriptomics and proteomics in order to understand cognitive function and disruption in diseases.</p>
<p class="publication"><span class="title">Behavior genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">37</span>;<span class="number">1</span>;<span class="pagerange">18-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16977502" rel="external">16977502</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10519-006-9114-3" rel="external">10.1007/s10519-006-9114-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Network activity-independent coordinated gene expression program for synapse assembly.</h4>
<p class="authors"><span class="author">Valor LM</span>, <span class="author">Charlesworth P</span>, <span class="author">Humphreys L</span>, <span class="author">Anderson CN</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Global biological datasets generated by genomics, transcriptomics, and proteomics provide new approaches to understanding the relationship between the genome and the synapse. Combined transcriptome analysis and multielectrode recordings of neuronal network activity were used in mouse embryonic primary neuronal cultures to examine synapse formation and activity-dependent gene regulation. Evidence for a coordinated gene expression program for assembly of synapses was observed in the expression of 642 genes encoding postsynaptic and plasticity proteins. This synaptogenesis gene expression program preceded protein expression of synapse markers and onset of spiking activity. Continued expression was followed by maturation of morphology and electrical neuronal networks, which was then followed by the expression of activity-dependent genes. Thus, two distinct sequentially active gene expression programs underlie the genomic programs of synapse function.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">11</span>;<span class="pagerange">4658-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17360580">17360580</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1810326">1810326</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0609071104">10.1073/pnas.0609071104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ras-association domain family (RASSF) members and their role in human tumourigenesis.</h4>
<p class="authors"><span class="author">van der Weyden L</span> and <span class="author">Adams DJ</span></p>
<p class="affiliation">Experimental Cancer Genetics Laboratory, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton Cambridge, UK. <a href="mailto:%6C%76%64%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#118;&#100;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ras proteins play a direct causal role in human cancer with activating mutations in Ras occurring in approximately 30% of tumours. Ras effectors also contribute to cancer, as mutations occur in Ras effectors, notably B-Raf and PI3-K, and drugs blocking elements of these pathways are in clinical development. In 2000, a new Ras effector was identified, RAS-association domain family 1 (RASSF1), and expression of the RASSF1A isoform of this gene is silenced in tumours by methylation of its promoter. Since methylation is reversible and demethylating agents are currently being used in clinical trials, detection of RASSF1A silencing by promoter hypermethylation has potential clinical uses in cancer diagnosis, prognosis and treatment. RASSF1A belongs to a new family of RAS effectors, of which there are currently 8 members (RASSF1-8). RASSF1-6 each contain a variable N-terminal segment followed by a Ras-association (RA) domain of the Ral-GDS/AF6 type, and a specialised coiled-coil structure known as a SARAH domain extending to the C-terminus. RASSF7-8 contain an N-terminal RA domain and a variable C-terminus. Members of the RASSF family are thought to function as tumour suppressors by regulating the cell cycle and apoptosis. This review will summarise our current knowledge of each member of the RASSF family and in particular what role they play in tumourigenesis, with a special focus on RASSF1A, whose promoter methylation is one of the most frequent alterations found in human tumours.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6997</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Biochimica et biophysica acta</span>&nbsp;<span class="year">2007</span>;<span class="volume">1776</span>;<span class="number">1</span>;<span class="pagerange">58-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17692468">17692468</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2586335">2586335</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.bbcan.2007.06.003">10.1016/j.bbcan.2007.06.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Subunit vaccines based on intimin and Efa-1 polypeptides induce humoral immunity in cattle but do not protect against intestinal colonisation by enterohaemorrhagic Escherichia coli O157:H7 or O26:H-.</h4>
<p class="authors"><span class="author">van Diemen PM</span>, <span class="author">Dziva F</span>, <span class="author">Abu-Median A</span>, <span class="author">Wallis TS</span>, <span class="author">van den Bosch H</span>, <span class="author">Dougan G</span>, <span class="author">Chanter N</span>, <span class="author">Frankel G</span> and <span class="author">Stevens MP</span></p>
<p class="affiliation">Institute for Animal Health, Compton, Berkshire RG20 7NN, UK. <a href="mailto:%70%61%75%6C%69%6E%65.%76%61%6E%64%69%65%6D%65%6E%40%62%62%73%72%63.%61%63.%75%6B">&#112;&#97;&#117;&#108;&#105;&#110;&#101;&#46;&#118;&#97;&#110;&#100;&#105;&#101;&#109;&#101;&#110;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Enterohaemorrhagic Escherichia coli (EHEC) infections in humans are an important public health concern and are commonly acquired via contact with ruminant faeces. Cattle are a key control point however cross-protective vaccines for the control of EHEC in the bovine reservoir do not yet exist. The EHEC serogroups that are predominantly associated with human infection in Europe and North America are O157 and O26. Intimin and EHEC factor for adherence (Efa-1) play important roles in intestinal colonisation of cattle by EHEC and are thus attractive candidates for the development of subunit vaccines. Immunisation of calves with the cell-binding domain of intimin subtypes beta or gamma via the intramuscular route induced antigen-specific serum IgG1 and, in some cases salivary IgA responses, but did not reduce the magnitude or duration of faecal excretion of EHEC O26:H- (Int(280)-beta) or EHEC O157:H7 (Int(280)-gamma) upon subsequent experimental challenge. Similarly, immunisation of calves via the intramuscular route with the truncated Efa-1 protein (Efa-1&#39;) from EHEC O157:H7 or a mixture of the amino-terminal and central thirds of the full-length protein (Efa-1-N and M) did not protect against intestinal colonisation by EHEC O157:H7 (Efa-1&#39;) or EHEC O26:H- (Efa-1-N and M) despite the induction of humoral immunity. A portion of the serum IgG1 elicited by the truncated recombinant antigens in calves was confirmed to recognise native protein exposed on the bacterial surface. Calves immunised with a mixture of Int(280)-gamma and Efa-1&#39; or an EHEC O157:H7 bacterin via the intramuscular route then boosted via the intranasal route with the same antigens using cholera toxin B subunit as an adjuvant were also not protected against intestinal colonisation by EHEC O157:H7. These studies highlight the need for further studies to develop and test novel vaccines or treatments for control of this important foodborne pathogen.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">076962</span></p>
<p class="publication"><span class="title">Veterinary immunology and immunopathology</span>&nbsp;<span class="year">2007</span>;<span class="volume">116</span>;<span class="number">1-2</span>;<span class="pagerange">47-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17258324" rel="external">17258324</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2656997" rel="external">2656997</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.vetimm.2006.12.009" rel="external">10.1016/j.vetimm.2006.12.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome-wide association study for celiac disease identifies risk variants in the region harboring IL2 and IL21.</h4>
<p class="authors"><span class="author">van Heel DA</span>, <span class="author">Franke L</span>, <span class="author">Hunt KA</span>, <span class="author">Gwilliam R</span>, <span class="author">Zhernakova A</span>, <span class="author">Inouye M</span>, <span class="author">Wapenaar MC</span>, <span class="author">Barnardo MC</span>, <span class="author">Bethel G</span>, <span class="author">Holmes GK</span>, <span class="author">Feighery C</span>, <span class="author">Jewell D</span>, <span class="author">Kelleher D</span>, <span class="author">Kumar P</span>, <span class="author">Travis S</span>, <span class="author">Walters JR</span>, <span class="author">Sanders DS</span>, <span class="author">Howdle P</span>, <span class="author">Swift J</span>, <span class="author">Playford RJ</span>, <span class="author">McLaren WM</span>, <span class="author">Mearin ML</span>, <span class="author">Mulder CJ</span>, <span class="author">McManus R</span>, <span class="author">McGinnis R</span>, <span class="author">Cardon LR</span>, <span class="author">Deloukas P</span> and <span class="author">Wijmenga C</span></p>
<p class="affiliation">Centre for Gastroenterology, Institute of Cell and Molecular Science, Queen Mary University of London, London E1 2AT, UK. <a href="mailto:%64.%76%61%6E%68%65%65%6C%40%71%6D%75%6C.%61%63.%75%6B">&#100;&#46;&#118;&#97;&#110;&#104;&#101;&#101;&#108;&#64;&#113;&#109;&#117;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We tested 310,605 SNPs for association in 778 individuals with celiac disease and 1,422 controls. Outside the HLA region, the most significant finding (rs13119723; P = 2.0 x 10(-7)) was in the KIAA1109-TENR-IL2-IL21 linkage disequilibrium block. We independently confirmed association in two further collections (strongest association at rs6822844, 24 kb 5&#39; of IL21; meta-analysis P = 1.3 x 10(-14), odds ratio = 0.63), suggesting that genetic variation in this region predisposes to celiac disease.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>; Wellcome Trust: <span class="grant">068094</span>, <span class="grant">068545/Z/02</span>, <span class="grant">GR068094MA</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">7</span>;<span class="pagerange">827-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17558408">17558408</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2274985">2274985</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2058">10.1038/ng2058</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assembly of the Candida albicans genome into sixteen supercontigs aligned on the eight chromosomes.</h4>
<p class="authors"><span class="author">van het Hoog M</span>, <span class="author">Rast TJ</span>, <span class="author">Martchenko M</span>, <span class="author">Grindle S</span>, <span class="author">Dignard D</span>, <span class="author">Hogues H</span>, <span class="author">Cuomo C</span>, <span class="author">Berriman M</span>, <span class="author">Scherer S</span>, <span class="author">Magee BB</span>, <span class="author">Whiteway M</span>, <span class="author">Chibana H</span>, <span class="author">Nantel A</span> and <span class="author">Magee PT</span></p>
<p class="affiliation">Biotechnology Research Institute, National Research Council of Canada, Montreal, Quebec, Canada. <a href="mailto:%6D%61%72%63%6F%40%76%61%6E%68%65%74%68%6F%6F%67.%63%6F%6D">&#109;&#97;&#114;&#99;&#111;&#64;&#118;&#97;&#110;&#104;&#101;&#116;&#104;&#111;&#111;&#103;&#46;&#99;&#111;&#109;</a></p><p class="abstract"><strong>Background</strong>: The 10.9x genomic sequence of Candida albicans, the most important human fungal pathogen, was published in 2004. Assembly 19 consisted of 412 supercontigs, of which 266 were a haploid set, since this fungus is diploid and contains an extensive degree of heterozygosity but lacks a complete sexual cycle. However, sequences of specific chromosomes were not determined.</p><p class="abstract"><strong>Results</strong>: Supercontigs from Assembly 19 (183, representing 98.4% of the sequence) were assigned to individual chromosomes purified by pulse-field gel electrophoresis and hybridized to DNA microarrays. Nine Assembly 19 supercontigs were found to contain markers from two different chromosomes. Assembly 21 contains the sequence of each of the eight chromosomes and was determined using a synteny analysis with preliminary versions of the Candida dubliniensis genome assembly, bioinformatics, a sequence tagged site (STS) map of overlapping fosmid clones, and an optical map. The orientation and order of the contigs on each chromosome, repeat regions too large to be covered by a sequence run, such as the ribosomal DNA cluster and the major repeat sequence, and telomere placement were determined using the STS map. Sequence gaps were closed by PCR and sequencing of the products. The overall assembly was compared to an optical map; this identified some misassembled contigs and gave a size estimate for each chromosome.</p><p class="abstract"><strong>Conclusion</strong>: Assembly 21 reveals an ancient chromosome fusion, a number of small internal duplications followed by inversions, and a subtelomeric arrangement, including a new gene family, the TLO genes. Correlations of position with relatedness of gene families imply a novel method of dispersion. The sequence of the individual chromosomes of C. albicans raises interesting biological questions about gene family creation and dispersion, subtelomere organization, and chromosome evolution.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">N01 AI05406</span>, <span class="grant">R01 AI 16567</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">R52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17419877" rel="external">17419877</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1896002" rel="external">1896002</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-4-r52" rel="external">10.1186/gb-2007-8-4-r52</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Definition of a minimal region of deletion of chromosome 7 in uterine leiomyomas by tiling-path microarray CGH and mutation analysis of known genes in this region.</h4>
<p class="authors"><span class="author">Vanharanta S</span>, <span class="author">Wortham NC</span>, <span class="author">Langford C</span>, <span class="author">El-Bahrawy M</span>, <span class="author">van der Spuy Z</span>, <span class="author">Sj&ouml;berg J</span>, <span class="author">Lehtonen R</span>, <span class="author">Karhu A</span>, <span class="author">Tomlinson IP</span> and <span class="author">Aaltonen LA</span></p>
<p class="affiliation">Department of Medical Genetics, Biomedicum Helsinki, University of Helsinki, Helsinki, Finland.</p><p class="abstract">Somatic interstitial deletions of chromosome segment 7q22-q31 in uterine leiomyomas are a frequent event, thought to be indicative of a tumor suppressor gene in the region. Previous LOH and CGH studies have refined this region to 7q22.3-q31, although the target gene has not been identified. Here, we have used tiling-path resolution microarray CGH to further refine the region and to identify homozygous deletions in fibroids. Furthermore, we have screened all manually annotated genes in the region for mutations. We have refined the minimum deleted region at 7q22.3-q31 to 2.79 Mbp and identified a second region of deletion at 7q34. However, we identified no pathogenic coding variation.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2007</span>;<span class="volume">46</span>;<span class="number">5</span>;<span class="pagerange">451-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17285575">17285575</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20427">10.1002/gcc.20427</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parallel evolution of conserved non-coding elements that target a common set of developmental regulatory genes from worms to humans.</h4>
<p class="authors"><span class="author">Vavouri T</span>, <span class="author">Walter K</span>, <span class="author">Gilks WR</span>, <span class="author">Lehner B</span> and <span class="author">Elgar G</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%74%76%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#118;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The human genome contains thousands of non-coding sequences that are often more conserved between vertebrate species than protein-coding exons. These highly conserved non-coding elements (CNEs) are associated with genes that coordinate development, and have been proposed to act as transcriptional enhancers. Despite their extreme sequence conservation in vertebrates, sequences homologous to CNEs have not been identified in invertebrates.</p><p class="abstract"><strong>Results</strong>: Here we report that nematode genomes contain an alternative set of CNEs that share sequence characteristics, but not identity, with their vertebrate counterparts. CNEs thus represent a very unusual class of sequences that are extremely conserved within specific animal lineages yet are highly divergent between lineages. Nematode CNEs are also associated with developmental regulatory genes, and include well-characterized enhancers and transcription factor binding sites, supporting the proposed function of CNEs as cis-regulatory elements. Most remarkably, 40 of 156 human CNE-associated genes with invertebrate orthologs are also associated with CNEs in both worms and flies.</p><p class="abstract"><strong>Conclusion</strong>: A core set of genes that regulate development is associated with CNEs across three animal groups (worms, flies and vertebrates). We propose that these CNEs reflect the parallel evolution of alternative enhancers for a common set of developmental regulatory genes in different animal groups. This &#39;re-wiring&#39; of gene regulatory networks containing key developmental coordinators was probably a driving force during the evolution of animal body plans. CNEs may, therefore, represent the genomic traces of these &#39;hard-wired&#39; core gene regulatory networks that specify the development of each alternative animal body plan.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0401138</span>, <span class="grant">MC_U105260799</span></p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">2</span>;<span class="pagerange">R15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17274809">17274809</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1852409">1852409</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-2-r15">10.1186/gb-2007-8-2-r15</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Guidelines for molecular karyotyping in constitutional genetic diagnosis.</h4>
<p class="authors"><span class="author">Vermeesch JR</span>, <span class="author">Fiegler H</span>, <span class="author">de Leeuw N</span>, <span class="author">Szuhai K</span>, <span class="author">Schoumans J</span>, <span class="author">Ciccone R</span>, <span class="author">Speleman F</span>, <span class="author">Rauch A</span>, <span class="author">Clayton-Smith J</span>, <span class="author">Van Ravenswaaij C</span>, <span class="author">Sanlaville D</span>, <span class="author">Patsalis PC</span>, <span class="author">Firth H</span>, <span class="author">Devriendt K</span> and <span class="author">Zuffardi O</span></p>
<p class="affiliation">Center for Human Genetics, University Hospital Gasthuisberg, Leuven, Belgium. <a href="mailto:%4A%6F%72%69%73.%56%65%72%6D%65%65%73%68%40%6D%65%64.%6B%75%6C%65%75%76%65%6E.%64%65">&#74;&#111;&#114;&#105;&#115;&#46;&#86;&#101;&#114;&#109;&#101;&#101;&#115;&#104;&#64;&#109;&#101;&#100;&#46;&#107;&#117;&#108;&#101;&#117;&#118;&#101;&#110;&#46;&#100;&#101;</a></p><p class="abstract">Array-based whole genome investigation or molecular karyotyping enables the genome-wide detection of submicroscopic imbalances. Proof-of-principle experiments have demonstrated that molecular karyotyping outperforms conventional karyotyping with regard to detection of chromosomal imbalances. This article identifies areas for which the technology seems matured and areas that require more investigations. Molecular karyotyping should be part of the genetic diagnostic work-up of patients with developmental disorders. For the implementation of the technique for other constitutional indications and in prenatal diagnosis, more research is appropriate. Also, the article aims to provide best practice guidelines for the application of array comparative genomic hybridisation to ensure both technical and clinical quality criteria that will optimise and standardise results and reports in diagnostic laboratories. In short, both the specificity and the sensitivity of the arrays should be evaluated in every laboratory offering the diagnostic test. Internal and external quality control programmes are urgently needed to evaluate and standardise the test results between laboratories.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">11</span>;<span class="pagerange">1105-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17637806" rel="external">17637806</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5201896" rel="external">10.1038/sj.ejhg.5201896</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distinct cytokine-driven responses of activated blood gammadelta T cells: insights into unconventional T cell pleiotropy.</h4>
<p class="authors"><span class="author">Vermijlen D</span>, <span class="author">Ellis P</span>, <span class="author">Langford C</span>, <span class="author">Klein A</span>, <span class="author">Engel R</span>, <span class="author">Willimann K</span>, <span class="author">Jomaa H</span>, <span class="author">Hayday AC</span> and <span class="author">Eberl M</span></p>
<p class="affiliation">Peter Gorer Department of Immunobiology, Guy&#39;s, King&#39;s and St. Thomas&#39; Medical School, King&#39;s College London, London, UK.</p><p class="abstract">Human Vgamma9/Vdelta2 T cells comprise a small population of peripheral blood T cells that in many infectious diseases respond to the microbial metabolite, (E)-4-hydroxy-3-methyl-but-2-enyl pyrophosphate (HMB-PP), expanding to up to 50% of CD3(+) cells. This &quot;transitional response,&quot; occurring temporally between the rapid innate and slower adaptive response, is widely viewed as proinflammatory and/or cytolytic. However, increasing evidence that different cytokines drive widely different effector functions in alphabeta T cells provoked us to apply cDNA microarrays to explore the potential pleiotropy of HMB-PP-activated Vgamma9/Vdelta2 T cells. The data and accompanying validations show that the related cytokines, IL-2, IL-4, or IL-21, each drive proliferation and comparable CD69 up-regulation but induce distinct effector responses that differ from prototypic alphabeta T cell responses. For example, the Th1-like response to IL-2 also includes expression of IL-5 and IL-13 that conversely are not induced by IL-4. The data identify specific molecules that may mediate gammadelta T cell effects. Thus, IL-21 induces a lymphoid-homing phenotype and high, unexpected expression of the follicular B cell-attracting chemokine CXCL13/BCA-1, suggesting a novel follicular B-helper-like T cell that may play a hitherto underappreciated role in humoral immunity early in infection. Such broad plasticity emphasizes the capacity of gammadelta T cells to influence the nature of the immune response to different challenges and has implications for the ongoing clinical application of cytokines together with Vgamma9/Vdelta2 TCR agonists.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">071534</span></p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2007</span>;<span class="volume">178</span>;<span class="number">7</span>;<span class="pagerange">4304-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17371987">17371987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic flux over time in the Salmonella lineage.</h4>
<p class="authors"><span class="author">Vernikos GS</span>, <span class="author">Thomson NR</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract"><strong>Background</strong>: DNA sequences that are shared between closely related organisms while being absent from their common ancestor and from sister lineages of that ancestor are likely to have been acquired by horizontal gene transfer. Over time, the composition of those sequences tends to become more similar to the compositional signature of their host (amelioration).</p><p class="abstract"><strong>Results</strong>: From a whole-genome comparative analysis of eleven Salmonella, three Escherichia coli and one Shigella strain, we inferred the relative time of insertion of putative horizontally acquired (PHA) genes in three Salmonella strains on different branches of the S. enterica phylogenetic tree. Compositional analysis suggests that most of the PHA genes are still undergoing an amelioration process and shows a clear correlation between time of insertion and the level of amelioration.</p><p class="abstract"><strong>Conclusion</strong>: The results show that older insertions include almost all functional classes. However, very recent horizontal transfer events in the Salmonella lineage involve primarily prophage elements that are shared only between very recently diverged lineages; despite this, the prophage sequence composition is close to that of the host, indicating that host adaptation, rather than amelioration, is likely to be the source of the compositional similarity. Almost half of the PHA genes were acquired at the base of the Salmonella lineage, whereas nearly three-quarters are shared between most S. enterica subspecies. The numerical distribution of PHA genes in the Salmonella tree topology correlates well with the divergence of the major Salmonella species, highlighting the major impact of horizontal transfer on the evolution of the salmonellae.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">R100</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17547764" rel="external">17547764</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2394748" rel="external">2394748</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-6-r100" rel="external">10.1186/gb-2007-8-6-r100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">microRNA-155 regulates the generation of immunoglobulin class-switched plasma cells.</h4>
<p class="authors"><span class="author">Vigorito E</span>, <span class="author">Perks KL</span>, <span class="author">Abreu-Goodger C</span>, <span class="author">Bunting S</span>, <span class="author">Xiang Z</span>, <span class="author">Kohlhaas S</span>, <span class="author">Das PP</span>, <span class="author">Miska EA</span>, <span class="author">Rodriguez A</span>, <span class="author">Bradley A</span>, <span class="author">Smith KG</span>, <span class="author">Rada C</span>, <span class="author">Enright AJ</span>, <span class="author">Toellner KM</span>, <span class="author">Maclennan IC</span> and <span class="author">Turner M</span></p>
<p class="affiliation">Laboratory of Lymphocyte Signalling and Development, The Babraham Institute, Cambridge, CB22 3AT, UK. <a href="mailto:%65%6C%65%6E%61.%76%69%67%6F%72%69%74%6F%40%62%62%73%72%63.%61%63.%75%6B">&#101;&#108;&#101;&#110;&#97;&#46;&#118;&#105;&#103;&#111;&#114;&#105;&#116;&#111;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">microRNA-155 (miR-155) is expressed by cells of the immune system after activation and has been shown to be required for antibody production after vaccination with attenuated Salmonella. Here we show the intrinsic requirement for miR-155 in B cell responses to thymus-dependent and -independent antigens. B cells lacking miR-155 generated reduced extrafollicular and germinal center responses and failed to produce high-affinity IgG1 antibodies. Gene-expression profiling of activated B cells indicated that miR-155 regulates an array of genes with diverse function, many of which are predicted targets of miR-155. The transcription factor Pu.1 is validated as a direct target of miR155-mediated inhibition. When Pu.1 is overexpressed in wild-type B cells, fewer IgG1 cells are produced, indicating that loss of Pu.1 regulation is a contributing factor to the miR-155-deficient phenotype. Our results implicate post-transcriptional regulation of gene expression for establishing the terminal differentiation program of B cells.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0700287</span>, <span class="grant">G117/424</span>, <span class="grant">G8402371</span>, <span class="grant">MC_U105178806</span></p>
<p class="publication"><span class="title">Immunity</span>&nbsp;<span class="year">2007</span>;<span class="volume">27</span>;<span class="number">6</span>;<span class="pagerange">847-59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/18055230">18055230</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.immuni.2007.10.009">10.1016/j.immuni.2007.10.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Say hello to our little friends.</h4>
<p class="authors"><span class="author">Walker A</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">8</span>;<span class="pagerange">572-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17633442">17633442</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1720">10.1038/nrmicro1720</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">This place is big enough for both of us.</h4>
<p class="authors"><span class="author">Walker A</span> and <span class="author">Crossman LC</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">2</span>;<span class="pagerange">90-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17225354">17225354</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1601">10.1038/nrmicro1601</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Urbane decay.</h4>
<p class="authors"><span class="author">Walker A</span> and <span class="author">Seth-Smith H</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">748-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17867225">17867225</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1763">10.1038/nrmicro1763</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ankyrin repeat domain-encoding genes in the wPip strain of Wolbachia from the Culex pipiens group.</h4>
<p class="authors"><span class="author">Walker T</span>, <span class="author">Klasson L</span>, <span class="author">Sebaihia M</span>, <span class="author">Sanders MJ</span>, <span class="author">Thomson NR</span>, <span class="author">Parkhill J</span> and <span class="author">Sinkins SP</span></p>
<p class="affiliation">Peter Medawar Building for Pathogen Research and Department of Zoology, University of Oxford, South Parks Road, Oxford OX1 3PS, UK. <a href="mailto:%74%68%6F%6D%61%73.%77%61%6C%6B%65%72%40%7A%6F%6F.%6F%78.%61%63.%75%6B">&#116;&#104;&#111;&#109;&#97;&#115;&#46;&#119;&#97;&#108;&#107;&#101;&#114;&#64;&#122;&#111;&#111;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Wolbachia are obligate endosymbiotic bacteria maternally transmitted through the egg cytoplasm that are responsible for several reproductive disorders in their insect hosts, such as cytoplasmic incompatibility (CI) in infected mosquitoes. Species in the Culex pipiens complex display an unusually high number of Wolbachia-induced crossing types, and based on present data, only the wPip strain is present.</p><p class="abstract"><strong>Results</strong>: The sequencing of the wPip strain of Wolbachia revealed the presence of 60 ankyrin repeat domain (ANK) encoding genes and expression studies of these genes were carried out in adult mosquitoes. One of these ANK genes, pk2, is shown to be part of an operon of three prophage-associated genes with sex-specific expression, and is present in two identical copies in the genome. Another homolog of pk2 is also present that is differentially expressed in different Cx. pipiens group strains. A further two ANK genes showed sex-specific regulation in wPip-infected Cx. pipiens group adults.</p><p class="abstract"><strong>Conclusion</strong>: The high number, variability and differential expression of ANK genes in wPip suggest an important role in Wolbachia biology, and the gene family provides both markers and promising candidates for the study of reproductive manipulation.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">079059</span></p>
<p class="publication"><span class="title">BMC biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">5</span>;<span class="pagerange">39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17883830" rel="external">17883830</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2045654" rel="external">2045654</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1741-7007-5-39" rel="external">10.1186/1741-7007-5-39</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enteroaggregative Escherichia coli related to uropathogenic clonal group A.</h4>
<p class="authors"><span class="author">Wallace-Gadsden F</span>, <span class="author">Johnson JR</span>, <span class="author">Wain J</span> and <span class="author">Okeke IN</span></p>
<p class="affiliation">Department of Biology, Haverford College, Haverford, Pennsylvania 19041, USA.</p><p class="abstract">Enteroaggregative Escherichia coli (EAEC) are heterogeneous, diarrheagenic E. coli. Of EAEC strains from Nigeria, 10 independent antimicrobial-resistant isolates belonged to the multilocus sequence type 69 clonal complex, to which uropathogenic E. coli clonal group A belongs. This finding suggests a recent common ancestor for these distinct groups of pathogenic E. coli.</p>
<p class="publication"><span class="title">Emerging infectious diseases</span>&nbsp;<span class="year">2007</span>;<span class="volume">13</span>;<span class="number">5</span>;<span class="pagerange">757-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17553259" rel="external">17553259</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2738470" rel="external">2738470</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A recessive genetic screen for host factors required for retroviral infection in a library of insertionally mutated Blm-deficient embryonic stem cells.</h4>
<p class="authors"><span class="author">Wang W</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Cell Biology and Genetics, College of Life Sciences, Peking University, Beijing, PR China. <a href="mailto:%77%78%77%40%73%61%6E%67%65%72.%61%63.%75%6B">&#119;&#120;&#119;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Host factors required for retroviral infection are potential targets for the modulation of diseases caused by retroviruses. During the retroviral life cycle, numerous cellular factors interact with the virus and play an essential role in infection. Cultured embryonic stem (ES) cells are susceptible to retroviral infection, therefore providing access to all of the genes required for this process to take place. In order to identify the host factors involved in retroviral infection, we designed and implemented a scheme for identifying ES cells that are resistant to retroviral infection and subsequent cloning of the mutated gene.</p><p class="abstract"><strong>Results</strong>: A library of mutant ES cells was established by genome-wide insertional mutagenesis in Blm-deficient ES cells, and a screen was performed by superinfection of the library at high multiplicity with a recombinant retrovirus carrying a positive and negative selection cassette. Stringent negative selection was then used to exclude the infected ES cells. We successfully recovered five independent clones of ES cells that are resistant to retroviral infection. Analysis of the mutations in these clones revealed four different homozygous and one compound heterozygous mutation in the mCat-1 locus, which confirms that mCat-1 is the ecotropic murine leukemia virus receptor in ES cells.</p><p class="abstract"><strong>Conclusion</strong>: We have demonstrated the feasibility and reliability of this recessive genetic approach to identifying critical genes required for retroviral infection in ES cells; the approach provides a unique opportunity to recover other cellular factors required for retroviral infection. The resulting insertionally mutated Blm-deficient ES cell library might also provide access to essential host cell components that are required for infection and replication for other types of virus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2007</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">R48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17407599">17407599</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1895998">1895998</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2007-8-4-r48">10.1186/gb-2007-8-4-r48</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Induced mitotic recombination of p53 in vivo.</h4>
<p class="authors"><span class="author">Wang W</span>, <span class="author">Warren M</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Genetic mosaics produced by FLP/FRT induced mitotic recombination have been widely used in Drosophila to study gene function in development. Recently, the Cre/loxP system has been applied to induce mitotic recombination in mouse embryonic stem cells and in many adult mouse tissues. We have used this strategy to generate a previously undescribed p53 mouse model in which expression of a ubiquitously expressed recombinase in a heterozygous p53 knockout animal produces mitotic recombinant clones homozygous for the p53 mutation. The induction of loss of heterozygosity in a few cells in an otherwise normal tissue mimics genetic aspects of tumorigenesis more closely than existing models and has revealed the possible cell autonomous nature of Wnt3. Our results suggest that inducible mitotic recombination can be used for clonal analysis of mutants in the mouse.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">79643</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">11</span>;<span class="pagerange">4501-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17360553" rel="external">17360553</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1838630" rel="external">1838630</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0607953104" rel="external">10.1073/pnas.0607953104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessing the potential of immunohistochemistry for systematic gene expression profiling.</h4>
<p class="authors"><span class="author">Warford A</span>, <span class="author">Flack G</span>, <span class="author">Conquer JS</span>, <span class="author">Zola H</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%61%77%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#119;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Immunohistochemistry (IHC) is a powerful technique for identifying sites of protein expression in tissues at the cellular and sub-cellular level. Here we have investigated the potential of using IHC for genome-wide expression screening by measuring the success rate and specificity of a panel of 35 monoclonal antibodies recognizing 5 well characterised CD antigens. Antibodies were pre-screened on acetone fixed frozen sections of spleen, tonsil and colon tissues. 19/35 antibodies gave staining with a success rate of 0/7 for JAM-2, 1/4 for CD99, 3/6 for CD138, 5/8 for CD45 and 10/10 for MHC-class II. 16/19 of these antibodies also gave staining on formalin fixed paraffin embedded tissue sections of tonsil and colon. All antibodies that had given staining were then profiled on tissues presented in human tissue microarrays. In the frozen microarrays 216 cores from 29 normal tissue types were present and in the formalin fixed paraffin array 344 cores from 35 normal and 4 cancers were represented. Where multiple antibodies were positive, there was evidence of consistent staining of the same tissues with several antibodies. In some cases differences in staining were observed potentially due to differential splice variants, polymorphisms or protein modification. With some antibodies there was evidence of cross-reactivity to inappropriate cells or structures. In addition the staining intensity with formalin fixation was changed quantitatively for some antibodies and in a few cases qualitatively, representing differential sensitivity of specific and non-specific epitopes to fixation. Accordingly, whilst IHC has potential for describing protein expression of unknown genes, these results emphasise a need to systematically address issues of specificity and sensitivity if appropriate profiles are to be described.</p>
<p class="publication"><span class="title">Journal of immunological methods</span>&nbsp;<span class="year">2007</span>;<span class="volume">318</span>;<span class="number">1-2</span>;<span class="pagerange">125-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17141799" rel="external">17141799</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jim.2006.10.011" rel="external">10.1016/j.jim.2006.10.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Targeting therapeutics: closing the gap between bench and bedside--a focus on tissue microarrays.</h4>
<p class="authors"><span class="author">Warford T</span></p>
<p class="publication"><span class="title">Expert review of molecular diagnostics</span>&nbsp;<span class="year">2007</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">103-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17331058" rel="external">17331058</a>; DOI: <a href="http://dx.doi.org/10.1586%2F14737159.7.2.103" rel="external">10.1586/14737159.7.2.103</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Sall4 mutant mouse model useful for studying the role of Sall4 in early embryonic development and organogenesis.</h4>
<p class="authors"><span class="author">Warren M</span>, <span class="author">Wang W</span>, <span class="author">Spiden S</span>, <span class="author">Chen-Murchie D</span>, <span class="author">Tannahill D</span>, <span class="author">Steel KP</span> and <span class="author">Bradley A</span></p><p class="abstract">SALL4 is a homologue of the Drosophila homeotic gene spalt, a zinc finger transcription factor, required for inner cell mass proliferation in early embryonic development. It also interacts with other transcription factors to control the development of the anorectal region, kidney, heart, limbs, and brain. Truncating mutations in SALL4 cause Okihiro syndrome, manifest as Duane anomaly, radial ray defects and sensorineural and conductive deafness. We report the characterization of a novel murine Sall4 null allele created by bacterial recombineering in ES cells. Homozygous mutant mice exhibit early embryonic lethality. Heterozygous mutant mice recapitulate phenotypic features of Okihiro syndrome including deafness, lower anogenital tract abnormalities, renal hypoplasia, anencephaly, Hirschprung&#39;s disease, and skeletal defects. This phenotype shows important differences in cardiac and ear manifestations to previously characterized Sall4 mutant alleles and should prove useful for the investigation of the influence of modifier alleles and protein interactions on the transcriptional regulatory function of Sall4.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0300212</span>, <span class="grant">MC_QA137918</span>; Wellcome Trust: <span class="grant">077187</span></p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2007</span>;<span class="volume">45</span>;<span class="number">1</span>;<span class="pagerange">51-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17216607">17216607</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2593393">2593393</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvg.20264">10.1002/dvg.20264</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</h4>
<p class="authors"><span class="author">Wellcome Trust Case Control Consortium</span></p><p class="abstract">There is increasing evidence that genome-wide association (GWA) studies represent a powerful approach to the identification of genes involved in common human diseases. We describe a joint GWA study (using the Affymetrix GeneChip 500K Mapping Array Set) undertaken in the British population, which has examined approximately 2,000 individuals for each of 7 major diseases and a shared set of approximately 3,000 controls. Case-control comparisons identified 24 independent association signals at P &lt; 5 x 10(-7): 1 in bipolar disorder, 1 in coronary artery disease, 9 in Crohn&#39;s disease, 3 in rheumatoid arthritis, 7 in type 1 diabetes and 3 in type 2 diabetes. On the basis of prior findings and replication studies thus-far completed, almost all of these signals reflect genuine susceptibility effects. We observed association at many previously identified loci, and found compelling evidence that some loci confer risk for more than one of the diseases studied. Across all diseases, we identified a large number of further signals (including 58 loci with single-point P values between 10(-5) and 5 x 10(-7)) likely to yield additional susceptibility loci. The importance of appropriately large samples was confirmed by the modest effect sizes observed at most loci identified. This study thus represents a thorough validation of the GWA approach. It has also demonstrated that careful use of a shared control group represents a safe and effective approach to GWA analyses of multiple disease phenotypes; has generated a genome-wide genotype database for future studies of common diseases in the British population; and shown that, provided individuals with non-European ancestry are excluded, the extent of population stratification in the British population is generally modest. Our findings offer new avenues for exploring the pathophysiology of these important disorders. We anticipate that our data, results and software, which will be widely available to other investigators, will provide a powerful resource for human genetics research.</p>
<p class="grants">Funded by: Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/137</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0100594</span>, <span class="grant">G0501942</span>, <span class="grant">G0600329</span>, <span class="grant">G0600705</span>, <span class="grant">G0800759</span>, <span class="grant">G0901461</span>, <span class="grant">G19/9</span>, <span class="grant">G90/106</span>, <span class="grant">G9806740</span>, <span class="grant">G9810900</span>; Wellcome Trust: <span class="grant">076113</span>, <span class="grant">077011</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2007</span>;<span class="volume">447</span>;<span class="number">7145</span>;<span class="pagerange">661-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17554300">17554300</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2719288">2719288</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature05911">10.1038/nature05911</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Association scan of 14,500 nonsynonymous SNPs in four diseases identifies autoimmunity variants.</h4>
<p class="authors"><span class="author">Wellcome Trust Case Control Consortium</span>, <span class="author">Australo-Anglo-American Spondylitis Consortium (TASC)</span>, <span class="author">Burton PR</span>, <span class="author">Clayton DG</span>, <span class="author">Cardon LR</span>, <span class="author">Craddock N</span>, <span class="author">Deloukas P</span>, <span class="author">Duncanson A</span>, <span class="author">Kwiatkowski DP</span>, <span class="author">McCarthy MI</span>, <span class="author">Ouwehand WH</span>, <span class="author">Samani NJ</span>, <span class="author">Todd JA</span>, <span class="author">Donnelly P</span>, <span class="author">Barrett JC</span>, <span class="author">Davison D</span>, <span class="author">Easton D</span>, <span class="author">Evans DM</span>, <span class="author">Leung HT</span>, <span class="author">Marchini JL</span>, <span class="author">Morris AP</span>, <span class="author">Spencer CC</span>, <span class="author">Tobin MD</span>, <span class="author">Attwood AP</span>, <span class="author">Boorman JP</span>, <span class="author">Cant B</span>, <span class="author">Everson U</span>, <span class="author">Hussey JM</span>, <span class="author">Jolley JD</span>, <span class="author">Knight AS</span>, <span class="author">Koch K</span>, <span class="author">Meech E</span>, <span class="author">Nutland S</span>, <span class="author">Prowse CV</span>, <span class="author">Stevens HE</span>, <span class="author">Taylor NC</span>, <span class="author">Walters GR</span>, <span class="author">Walker NM</span>, <span class="author">Watkins NA</span>, <span class="author">Winzer T</span>, <span class="author">Jones RW</span>, <span class="author">McArdle WL</span>, <span class="author">Ring SM</span>, <span class="author">Strachan DP</span>, <span class="author">Pembrey M</span>, <span class="author">Breen G</span>, <span class="author">St Clair D</span>, <span class="author">Caesar S</span>, <span class="author">Gordon-Smith K</span>, <span class="author">Jones L</span>, <span class="author">Fraser C</span>, <span class="author">Green EK</span>, <span class="author">Grozeva D</span>, <span class="author">Hamshere ML</span>, <span class="author">Holmans PA</span>, <span class="author">Jones IR</span>, <span class="author">Kirov G</span>, <span class="author">Moskivina V</span>, <span class="author">Nikolov I</span>, <span class="author">O&#39;Donovan MC</span>, <span class="author">Owen MJ</span>, <span class="author">Collier DA</span>, <span class="author">Elkin A</span>, <span class="author">Farmer A</span>, <span class="author">Williamson R</span>, <span class="author">McGuffin P</span>, <span class="author">Young AH</span>, <span class="author">Ferrier IN</span>, <span class="author">Ball SG</span>, <span class="author">Balmforth AJ</span>, <span class="author">Barrett JH</span>, <span class="author">Bishop TD</span>, <span class="author">Iles MM</span>, <span class="author">Maqbool A</span>, <span class="author">Yuldasheva N</span>, <span class="author">Hall AS</span>, <span class="author">Braund PS</span>, <span class="author">Dixon RJ</span>, <span class="author">Mangino M</span>, <span class="author">Stevens S</span>, <span class="author">Thompson JR</span>, <span class="author">Bredin F</span>, <span class="author">Tremelling M</span>, <span class="author">Parkes M</span>, <span class="author">Drummond H</span>, <span class="author">Lees CW</span>, <span class="author">Nimmo ER</span>, <span class="author">Satsangi J</span>, <span class="author">Fisher SA</span>, <span class="author">Forbes A</span>, <span class="author">Lewis CM</span>, <span class="author">Onnie CM</span>, <span class="author">Prescott NJ</span>, <span class="author">Sanderson J</span>, <span class="author">Matthew CG</span>, <span class="author">Barbour J</span>, <span class="author">Mohiuddin MK</span>, <span class="author">Todhunter CE</span>, <span class="author">Mansfield JC</span>, <span class="author">Ahmad T</span>, <span class="author">Cummings FR</span>, <span class="author">Jewell DP</span>, <span class="author">Webster J</span>, <span class="author">Brown MJ</span>, <span class="author">Lathrop MG</span>, <span class="author">Connell J</span>, <span class="author">Dominiczak A</span>, <span class="author">Marcano CA</span>, <span class="author">Burke B</span>, <span class="author">Dobson R</span>, <span class="author">Gungadoo J</span>, <span class="author">Lee KL</span>, <span class="author">Munroe PB</span>, <span class="author">Newhouse SJ</span>, <span class="author">Onipinla A</span>, <span class="author">Wallace C</span>, <span class="author">Xue M</span>, <span class="author">Caulfield M</span>, <span class="author">Farrall M</span>, <span class="author">Barton A</span>, <span class="author">Biologics in RA Genetics and Genomics Study Syndicate (BRAGGS) Steering Committee</span>, <span class="author">Bruce IN</span>, <span class="author">Donovan H</span>, <span class="author">Eyre S</span>, <span class="author">Gilbert PD</span>, <span class="author">Hilder SL</span>, <span class="author">Hinks AM</span>, <span class="author">John SL</span>, <span class="author">Potter C</span>, <span class="author">Silman AJ</span>, <span class="author">Symmons DP</span>, <span class="author">Thomson W</span>, <span class="author">Worthington J</span>, <span class="author">Dunger DB</span>, <span class="author">Widmer B</span>, <span class="author">Frayling TM</span>, <span class="author">Freathy RM</span>, <span class="author">Lango H</span>, <span class="author">Perry JR</span>, <span class="author">Shields BM</span>, <span class="author">Weedon MN</span>, <span class="author">Hattersley AT</span>, <span class="author">Hitman GA</span>, <span class="author">Walker M</span>, <span class="author">Elliott KS</span>, <span class="author">Groves CJ</span>, <span class="author">Lindgren CM</span>, <span class="author">Rayner NW</span>, <span class="author">Timpson NJ</span>, <span class="author">Zeggini E</span>, <span class="author">Newport M</span>, <span class="author">Sirugo G</span>, <span class="author">Lyons E</span>, <span class="author">Vannberg F</span>, <span class="author">Hill AV</span>, <span class="author">Bradbury LA</span>, <span class="author">Farrar C</span>, <span class="author">Pointon JJ</span>, <span class="author">Wordsworth P</span>, <span class="author">Brown MA</span>, <span class="author">Franklyn JA</span>, <span class="author">Heward JM</span>, <span class="author">Simmonds MJ</span>, <span class="author">Gough SC</span>, <span class="author">Seal S</span>, <span class="author">Breast Cancer Susceptibility Collaboration (UK)</span>, <span class="author">Stratton MR</span>, <span class="author">Rahman N</span>, <span class="author">Ban M</span>, <span class="author">Goris A</span>, <span class="author">Sawcer SJ</span>, <span class="author">Compston A</span>, <span class="author">Conway D</span>, <span class="author">Jallow M</span>, <span class="author">Newport M</span>, <span class="author">Sirugo G</span>, <span class="author">Rockett KA</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Chaney A</span>, <span class="author">Downes K</span>, <span class="author">Ghori MJ</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt SE</span>, <span class="author">Inouye M</span>, <span class="author">Keniry A</span>, <span class="author">King E</span>, <span class="author">McGinnis R</span>, <span class="author">Potter S</span>, <span class="author">Ravindrarajah R</span>, <span class="author">Whittaker P</span>, <span class="author">Widden C</span>, <span class="author">Withers D</span>, <span class="author">Cardin NJ</span>, <span class="author">Davison D</span>, <span class="author">Ferreira T</span>, <span class="author">Pereira-Gale J</span>, <span class="author">Hallgrimsdo&#39;ttir IB</span>, <span class="author">Howie BN</span>, <span class="author">Su Z</span>, <span class="author">Teo YY</span>, <span class="author">Vukcevic D</span>, <span class="author">Bentley D</span>, <span class="author">Brown MA</span>, <span class="author">Compston A</span>, <span class="author">Farrall M</span>, <span class="author">Hall AS</span>, <span class="author">Hattersley AT</span>, <span class="author">Hill AV</span>, <span class="author">Parkes M</span>, <span class="author">Pembrey M</span>, <span class="author">Stratton MR</span>, <span class="author">Mitchell SL</span>, <span class="author">Newby PR</span>, <span class="author">Brand OJ</span>, <span class="author">Carr-Smith J</span>, <span class="author">Pearce SH</span>, <span class="author">McGinnis R</span>, <span class="author">Keniry A</span>, <span class="author">Deloukas P</span>, <span class="author">Reveille JD</span>, <span class="author">Zhou X</span>, <span class="author">Sims AM</span>, <span class="author">Dowling A</span>, <span class="author">Taylor J</span>, <span class="author">Doan T</span>, <span class="author">Davis JC</span>, <span class="author">Savage L</span>, <span class="author">Ward MM</span>, <span class="author">Learch TL</span>, <span class="author">Weisman MH</span> and <span class="author">Brown M</span></p>
<p class="affiliation">Genetic Epidemiology Group, Department of Health Sciences, University of Leicester, Adrian Building, University Road, Leicester LE1 7RH, UK.</p><p class="abstract">We have genotyped 14,436 nonsynonymous SNPs (nsSNPs) and 897 major histocompatibility complex (MHC) tag SNPs from 1,000 independent cases of ankylosing spondylitis (AS), autoimmune thyroid disease (AITD), multiple sclerosis (MS) and breast cancer (BC). Comparing these data against a common control dataset derived from 1,500 randomly selected healthy British individuals, we report initial association and independent replication in a North American sample of two new loci related to ankylosing spondylitis, ARTS1 and IL23R, and confirmation of the previously reported association of AITD with TSHR and FCRL3. These findings, enabled in part by increased statistical power resulting from the expansion of the control reference group to include individuals from the other disease groups, highlight notable new possibilities for autoimmune regulation and suggest that IL23R may be a common susceptibility factor for the major &#39;seronegative&#39; diseases.</p>
<p class="grants">Funded by: Arthritis Research UK: <span class="grant">17552</span>; Cancer Research UK: <span class="grant">A4994</span>; Chief Scientist Office: <span class="grant">CZB/4/540</span>, <span class="grant">ETM/137</span>, <span class="grant">ETM/75</span>; Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0501942</span>, <span class="grant">G0600329</span>, <span class="grant">G0600705</span>, <span class="grant">G0701003</span>, <span class="grant">G0800759</span>, <span class="grant">G19/9</span>, <span class="grant">G90/106</span>; Multiple Sclerosis Society: <span class="grant">730</span>; Wellcome Trust: <span class="grant">057097</span>, <span class="grant">076113</span>, <span class="grant">081682</span>, <span class="grant">089989</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">11</span>;<span class="pagerange">1329-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17952073">17952073</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2680141">2680141</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng.2007.17">10.1038/ng.2007.17</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Quantitative trait association in parent offspring trios: Extension of case/pseudocontrol method and comparison of prospective and retrospective approaches.</h4>
<p class="authors"><span class="author">Wheeler E</span> and <span class="author">Cordell HJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Cambridge, UK.</p><p class="abstract">The case/pseudocontrol method provides a convenient framework for family-based association analysis of case-parent trios, incorporating several previously proposed methods such as the transmission/disequilibrium test and log-linear modelling of parent-of-origin effects. The method allows genotype and haplotype analysis at an arbitrary number of linked and unlinked multiallelic loci, as well as modelling of more complex effects such as epistasis, parent-of-origin effects, maternal genotype and mother-child interaction effects, and gene-environment interactions. Here we extend the method for analysis of quantitative as opposed to dichotomous (e.g. disease) traits. The resulting method can be thought of as a retrospective approach, modelling genotype given trait value, in contrast to prospective approaches that model trait given genotype. Through simulations and analytical derivations, we examine the power and properties of our proposed approach, and compare it to several previously proposed single-locus methods for quantitative trait association analysis. We investigate the performance of the different methods when extended to allow analysis of haplotype, maternal genotype and parent-of-origin effects. With randomly ascertained families, with or without population stratification, the prospective approach (modeling trait value given genotype) is found to be generally most effective, although the retrospective approach has some advantages with regard to estimation and interpretability of parameter estimates when applied to selected samples.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">068615</span>, <span class="grant">074524</span></p>
<p class="publication"><span class="title">Genetic epidemiology</span>&nbsp;<span class="year">2007</span>;<span class="volume">31</span>;<span class="number">8</span>;<span class="pagerange">813-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17549757" rel="external">17549757</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2707979" rel="external">2707979</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgepi.20243" rel="external">10.1002/gepi.20243</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Esophageal atresia, hypoplasia of zygomatic complex, microcephaly, cup-shaped ears, congenital heart defect, and mental retardation--new MCA/MR syndrome in two affected sibs and a mildly affected mother?</h4>
<p class="authors"><span class="author">Wieczorek D</span>, <span class="author">Shaw-Smith C</span>, <span class="author">Kohlhase J</span>, <span class="author">Schmitt W</span>, <span class="author">Buiting K</span>, <span class="author">Coffey A</span>, <span class="author">Howard E</span>, <span class="author">Hehr U</span> and <span class="author">Gillessen-Kaesbach G</span></p>
<p class="affiliation">Institut f&uuml;r Humangenetik, Universit&auml;tsklinikum Essen, Germany, and Department of Medical Genetics, Addenbrooke&#39;s Hospital, Cambridge, UK. <a href="mailto:%64%61%67%6D%61%72.%77%69%65%63%7A%6F%72%65%6B%40%75%6E%69-%64%75%65.%64%65">&#100;&#97;&#103;&#109;&#97;&#114;&#46;&#119;&#105;&#101;&#99;&#122;&#111;&#114;&#101;&#107;&#64;&#117;&#110;&#105;&#45;&#100;&#117;&#101;&#46;&#100;&#101;</a></p><p class="abstract">The previously undescribed combination of esophageal atresia, hypoplasia of the zygomatic complex, microcephaly, cup-shaped ears, congenital heart defect, and mental retardation was diagnosed in two siblings of different sexes, with the brother being more severely affected. The mother presented with zygomatic arch hypoplasia of the right side only. We discuss major differential diagnoses: Goldenhar, Feingold, CHARGE, and Treacher Collins syndromes show a few overlapping clinical features, but these diagnoses are unlikely as the clinical findings are unusual for Goldenhar syndrome and mutational screening of the MYCN, the CHD7, and the TCOF1 genes did not reveal any abnormalities. Autosomal recessive oto-facial syndrome, hypomandibular faciocranial dysostosis, and Ozkan syndromes were clinically excluded. A microdeletion 22q11.2 was excluded by FISH analysis, a microdeletion 2p23-p24 by microsatellite analyses, a subtelomeric chromosomal aberration by MLPA, and a small genomic deletion/duplication by CGH array. As X-inactivation studies did not show skewed X-inactivation in the mother, we consider X-chromosomal recessive inheritance of this condition less likely. We discuss autosomal dominant inheritance with variable expressivity or mosaicism in the mother as the likely genetic mechanism in this new multiple congenital anomaly/mental retardation (MCA/MR) syndrome.</p>
<p class="publication"><span class="title">American journal of medical genetics. Part A</span>&nbsp;<span class="year">2007</span>;<span class="volume">143A</span>;<span class="number">11</span>;<span class="pagerange">1135-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17497718">17497718</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajmg.a.31752">10.1002/ajmg.a.31752</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Israeli-Palestinian Science Organization.</h4>
<p class="authors"><span class="author">Wiesel T</span>, <span class="author">Agre P</span>, <span class="author">Arrow KJ</span>, <span class="author">Atiyah M</span>, <span class="author">Br&eacute;zin E</span>, <span class="author">Charfi FF</span>, <span class="author">Cohen-Tanoudji C</span>, <span class="author">Daar A</span>, <span class="author">Jacob F</span>, <span class="author">Kahneman D</span>, <span class="author">Lee YT</span>, <span class="author">Nicolaisen I</span>, <span class="author">Nusseibeh S</span>, <span class="author">Reuter H</span>, <span class="author">Shoham Y</span>, <span class="author">Sulston J</span>, <span class="author">Walzer M</span> and <span class="author">Yaari M</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">315</span>;<span class="number">5808</span>;<span class="pagerange">39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17204624">17204624</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.315.5808.39b">10.1126/science.315.5808.39b</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&#39;Oming in on schistosomes: prospects and limitations for post-genomics.</h4>
<p class="authors"><span class="author">Wilson RA</span>, <span class="author">Ashton PD</span>, <span class="author">Braschi S</span>, <span class="author">Dillon GP</span>, <span class="author">Berriman M</span> and <span class="author">Ivens A</span></p>
<p class="affiliation">Department of Biology, University of York, PO Box 373, York YO10 5YW, UK. <a href="mailto:%72%61%77%33%40%79%6F%72%6B.%61%63.%75%6B">&#114;&#97;&#119;&#51;&#64;&#121;&#111;&#114;&#107;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The recent release of version 3 of the Schistosoma mansoni genome assembly has made a wealth of information available to researchers. Here, progress made in schistosome genomics and post-genomics is considered. The current status of knowledge about the genome, transcriptome, proteome, glycome and immunome is summarized and recent publications briefly reviewed. The prospects for advances in understanding schistosome biology are highlighted. Most importantly, the limitations (which are mostly technical) that need to be addressed before the full potential of the genome database(s) can be realized are emphasized.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">1</span>;<span class="pagerange">14-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17097923" rel="external">17097923</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2006.10.002" rel="external">10.1016/j.pt.2006.10.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Finding cis-regulatory modules in Drosophila using phylogenetic hidden Markov models.</h4>
<p class="authors"><span class="author">Wong WS</span> and <span class="author">Nielsen R</span></p>
<p class="affiliation">Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, NY 14853, USA. <a href="mailto:%73%77%77%38%40%63%6F%72%6E%65%6C%6C.%65%64%75">&#115;&#119;&#119;&#56;&#64;&#99;&#111;&#114;&#110;&#101;&#108;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Motivation</strong>: Finding the regulatory modules for transcription factors binding is an important step in elucidating the complex molecular mechanisms underlying regulation of gene expression. There are numerous methods available for solving this problem, however, very few of them take advantage of the increasing availability of comparative genomic data.</p><p class="abstract"><strong>Results</strong>: We develop a method for finding regulatory modules in Eukaryotic species using phylogenetic data. Using computer simulations and analysis of real data, we show that the use of phylogenetic hidden Markov model can lead to an increase in accuracy of prediction over methods that do not take advantage of the data from multiple species.</p><p class="abstract"><strong>Availability</strong>: The new method is made accessible under GPL in a new publicly available JAVA program: EvoPromoter. It can be downloaded at http://sourceforge.net/projects/evopromoter/.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2007</span>;<span class="volume">23</span>;<span class="number">16</span>;<span class="pagerange">2031-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17550911">17550911</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtm299">10.1093/bioinformatics/btm299</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in ionotropic AMPA receptor 3 alter channel properties and are associated with moderate cognitive impairment in humans.</h4>
<p class="authors"><span class="author">Wu Y</span>, <span class="author">Arai AC</span>, <span class="author">Rumbaugh G</span>, <span class="author">Srivastava AK</span>, <span class="author">Turner G</span>, <span class="author">Hayashi T</span>, <span class="author">Suzuki E</span>, <span class="author">Jiang Y</span>, <span class="author">Zhang L</span>, <span class="author">Rodriguez J</span>, <span class="author">Boyle J</span>, <span class="author">Tarpey P</span>, <span class="author">Raymond FL</span>, <span class="author">Nevelsteen J</span>, <span class="author">Froyen G</span>, <span class="author">Stratton M</span>, <span class="author">Futreal A</span>, <span class="author">Gecz J</span>, <span class="author">Stevenson R</span>, <span class="author">Schwartz CE</span>, <span class="author">Valle D</span>, <span class="author">Huganir RL</span> and <span class="author">Wang T</span></p>
<p class="affiliation">Institute of Genetic Medicine and Department of Pediatrics, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA.</p><p class="abstract">Ionotropic alpha-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptors (iGluRs) mediate the majority of excitatory synaptic transmission in the CNS and are essential for the induction and maintenance of long-term potentiation and long-term depression, two cellular models of learning and memory. We identified a genomic deletion (0.4 Mb) involving the entire GRIA3 (encoding iGluR3) by using an X-array comparative genomic hybridization (CGH) and four missense variants (G833R, M706T, R631S, and R450Q) in functional domains of iGluR3 by sequencing 400 males with X-linked mental retardation (XLMR). Three variants were found in males with moderate MR and were absent in 500 control males. Expression studies in HEK293 cells showed that G833R resulted in a 78% reduction of iGluR3 due to protein misfolding. Whole-cell recording studies of iGluR3 homomers in HEK293 cells revealed that neither iGluR3-M706T (S2 domain) nor iGluR3-R631S (near channel core) had substantial channel function, whereas R450Q (S1 domain) was associated with accelerated receptor desensitization. When forming heteromeric receptors with iGluR2 in HEK293 cells, all four iGluR3 variants had altered desensitization kinetics. Our study provides the genetic and functional evidence that mutant iGluR3 with altered kinetic properties is associated with moderate cognitive impairment in humans.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD044789</span>, <span class="grant">HD24061</span>, <span class="grant">HD26202</span>; NINDS NIH HHS: <span class="grant">NS36715</span>, <span class="grant">NS41020</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2007</span>;<span class="volume">104</span>;<span class="number">46</span>;<span class="pagerange">18163-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17989220" rel="external">17989220</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2084314" rel="external">2084314</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0708699104" rel="external">10.1073/pnas.0708699104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interleukin-2 gene variation impairs regulatory T cell function and causes autoimmunity.</h4>
<p class="authors"><span class="author">Yamanouchi J</span>, <span class="author">Rainbow D</span>, <span class="author">Serra P</span>, <span class="author">Howlett S</span>, <span class="author">Hunter K</span>, <span class="author">Garner VE</span>, <span class="author">Gonzalez-Munoz A</span>, <span class="author">Clark J</span>, <span class="author">Veijola R</span>, <span class="author">Cubbon R</span>, <span class="author">Chen SL</span>, <span class="author">Rosa R</span>, <span class="author">Cumiskey AM</span>, <span class="author">Serreze DV</span>, <span class="author">Gregory S</span>, <span class="author">Rogers J</span>, <span class="author">Lyons PA</span>, <span class="author">Healy B</span>, <span class="author">Smink LJ</span>, <span class="author">Todd JA</span>, <span class="author">Peterson LB</span>, <span class="author">Wicker LS</span> and <span class="author">Santamaria P</span></p>
<p class="affiliation">Julia McFarlane Diabetes Research Centre (JMDRC) and Department of Microbiology and Infectious Diseases, Institute of Inflammation, Infection and Immunity, Faculty of Medicine, The University of Calgary, Calgary, Alberta T2N 4N1, Canada.</p><p class="abstract">Autoimmune diseases are thought to result from imbalances in normal immune physiology and regulation. Here, we show that autoimmune disease susceptibility and resistance alleles on mouse chromosome 3 (Idd3) correlate with differential expression of the key immunoregulatory cytokine interleukin-2 (IL-2). In order to test directly that an approximately twofold reduction in IL-2 underpins the Idd3-linked destabilization of immune homeostasis, we show that engineered haplodeficiency of Il2 gene expression not only reduces T cell IL-2 production by twofold but also mimics the autoimmune dysregulatory effects of the naturally occurring susceptibility alleles of Il2. Reduced IL-2 production achieved by either genetic mechanism correlates with reduced function of CD4(+) CD25(+) regulatory T cells, which are critical for maintaining immune homeostasis.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">061859</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">3</span>;<span class="pagerange">329-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17277778">17277778</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2886969">2886969</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1958">10.1038/ng1958</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Revisiting the molecular evolutionary history of Shigella spp.</h4>
<p class="authors"><span class="author">Yang J</span>, <span class="author">Nie H</span>, <span class="author">Chen L</span>, <span class="author">Zhang X</span>, <span class="author">Yang F</span>, <span class="author">Xu X</span>, <span class="author">Zhu Y</span>, <span class="author">Yu J</span> and <span class="author">Jin Q</span></p>
<p class="affiliation">State Key Laboratory for Molecular Virology and Genetic Engineering, 6 Rongjing East Street, BDA Beijing 100176, PR China.</p><p class="abstract">The theory that Shigella is derived from multiple independent origins of Escherichia coli (Pupo et al. 2000) has been challenged by recent findings that the virulence plasmids (VPs) and the chromosomes share a similar evolutionary history (Escobar-Paramo et al. 2003), which suggests that an ancestral VP entered an E. coli strain only once, which gave rise to Shigella spp. In an attempt to resolve these conflicting theories, we constructed three phylogenetic trees in this study: a robust chromosomal tree using 23 housekeeping genes from 46 strains of Shigella and enteroinvasive E. coli (EIEC), a chromosomal tree using 4 housekeeping genes from 19 EcoR strains and 46 Shigella/EIEC strains, and a VP tree using 5 genes outside of the VP cell-entry region from 38 Shigella/EIEC strains. Both chromosomal trees group Shigella into three main clusters and five outliers, and strongly suggest that Shigella has multiple origins within E. coli. Most strikingly, the VP tree shows that the VPs from two main Shigella clusters, C1 and C2, are more closely related, which contradicts the chromosomal trees that place C2 and C3 next to each other but C1 at a distance. Additionally, we have identified a complete tra operon of the F-plasmid in the genome sequence of an EIEC strain and found that two other EIEC strains are also likely to possess a complete tra operon. All lines of evidence support an alternative multiorigin theory that transferable diverse ancestral VPs entered diverse origins of E. coli multiple times during a prolonged period of time, resulting in Shigella species with diverse genomes but similar pathogenic properties.</p>
<p class="publication"><span class="title">Journal of molecular evolution</span>&nbsp;<span class="year">2007</span>;<span class="volume">64</span>;<span class="number">1</span>;<span class="pagerange">71-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17160643" rel="external">17160643</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00239-006-0052-8" rel="external">10.1007/s00239-006-0052-8</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide association study of prostate cancer identifies a second risk locus at 8q24.</h4>
<p class="authors"><span class="author">Yeager M</span>, <span class="author">Orr N</span>, <span class="author">Hayes RB</span>, <span class="author">Jacobs KB</span>, <span class="author">Kraft P</span>, <span class="author">Wacholder S</span>, <span class="author">Minichiello MJ</span>, <span class="author">Fearnhead P</span>, <span class="author">Yu K</span>, <span class="author">Chatterjee N</span>, <span class="author">Wang Z</span>, <span class="author">Welch R</span>, <span class="author">Staats BJ</span>, <span class="author">Calle EE</span>, <span class="author">Feigelson HS</span>, <span class="author">Thun MJ</span>, <span class="author">Rodriguez C</span>, <span class="author">Albanes D</span>, <span class="author">Virtamo J</span>, <span class="author">Weinstein S</span>, <span class="author">Schumacher FR</span>, <span class="author">Giovannucci E</span>, <span class="author">Willett WC</span>, <span class="author">Cancel-Tassin G</span>, <span class="author">Cussenot O</span>, <span class="author">Valeri A</span>, <span class="author">Andriole GL</span>, <span class="author">Gelmann EP</span>, <span class="author">Tucker M</span>, <span class="author">Gerhard DS</span>, <span class="author">Fraumeni JF</span>, <span class="author">Hoover R</span>, <span class="author">Hunter DJ</span>, <span class="author">Chanock SJ</span> and <span class="author">Thomas G</span></p>
<p class="affiliation">SAIC-Frederick, National Cancer Institute (NCI)-Frederick Cancer Research and Development Center, Frederick, Maryland 21702, USA.</p><p class="abstract">Recently, common variants on human chromosome 8q24 were found to be associated with prostate cancer risk. While conducting a genome-wide association study in the Cancer Genetic Markers of Susceptibility project with 550,000 SNPs in a nested case-control study (1,172 cases and 1,157 controls of European origin), we identified a new association at 8q24 with an independent effect on prostate cancer susceptibility. The most significant signal is 70 kb centromeric to the previously reported SNP, rs1447295, but shows little evidence of linkage disequilibrium with it. A combined analysis with four additional studies (total: 4,296 cases and 4,299 controls) confirms association with prostate cancer for rs6983267 in the centromeric locus (P = 9.42 x 10(-13); heterozygote odds ratio (OR): 1.26, 95% confidence interval (c.i.): 1.13-1.41; homozygote OR: 1.58, 95% c.i.: 1.40-1.78). Each SNP remained significant in a joint analysis after adjusting for the other (rs1447295 P = 1.41 x 10(-11); rs6983267 P = 6.62 x 10(-10)). These observations, combined with compelling evidence for a recombination hotspot between the two markers, indicate the presence of at least two independent loci within 8q24 that contribute to prostate cancer in men of European ancestry. We estimate that the population attributable risk of the new locus, marked by rs6983267, is higher than the locus marked by rs1447295 (21% versus 9%).</p>
<p class="grants">Funded by: CCR NIH HHS: <span class="grant">N01-RC-37004</span>, <span class="grant">N01-RC-45035</span>; NCI NIH HHS: <span class="grant">5U01CA098233-04</span>, <span class="grant">CA55075</span>, <span class="grant">N01-CN-45165</span>, <span class="grant">T32 CA 09001</span>, <span class="grant">U01 CA098710</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">39</span>;<span class="number">5</span>;<span class="pagerange">645-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17401363">17401363</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng2022">10.1038/ng2022</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A second major histocompatibility complex susceptibility locus for multiple sclerosis.</h4>
<p class="authors"><span class="author">Yeo TW</span>, <span class="author">De Jager PL</span>, <span class="author">Gregory SG</span>, <span class="author">Barcellos LF</span>, <span class="author">Walton A</span>, <span class="author">Goris A</span>, <span class="author">Fenoglio C</span>, <span class="author">Ban M</span>, <span class="author">Taylor CJ</span>, <span class="author">Goodman RS</span>, <span class="author">Walsh E</span>, <span class="author">Wolfish CS</span>, <span class="author">Horton R</span>, <span class="author">Traherne J</span>, <span class="author">Beck S</span>, <span class="author">Trowsdale J</span>, <span class="author">Caillier SJ</span>, <span class="author">Ivinson AJ</span>, <span class="author">Green T</span>, <span class="author">Pobywajlo S</span>, <span class="author">Lander ES</span>, <span class="author">Pericak-Vance MA</span>, <span class="author">Haines JL</span>, <span class="author">Daly MJ</span>, <span class="author">Oksenberg JR</span>, <span class="author">Hauser SL</span>, <span class="author">Compston A</span>, <span class="author">Hafler DA</span>, <span class="author">Rioux JD</span> and <span class="author">Sawcer S</span></p>
<p class="affiliation">Department of Clinical Neurosciences, University of Cambridge, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom.</p><p class="abstract"><strong>Objective</strong>: Variation in the major histocompatibility complex (MHC) on chromosome 6p21 is known to influence susceptibility to multiple sclerosis with the strongest effect originating from the HLA-DRB1 gene in the class II region. The possibility that other genes in the MHC independently influence susceptibility to multiple sclerosis has been suggested but remains unconfirmed.</p><p class="abstract"><strong>Methods</strong>: Using a combination of microsatellite, single nucleotide polymorphism, and human leukocyte antigen (HLA) typing, we screened the MHC in trio families looking for evidence of residual association above and beyond that attributable to the established DRB1*1501 risk haplotype. We then refined this analysis by extending the genotyping of classical HLA loci into independent cases and control subjects.</p><p class="abstract"><strong>Results</strong>: Screening confirmed the presence of residual association and suggested that this was maximal in the region of the HLA-C gene. Extending analysis of the classical loci confirmed that this residual association is partly due to allelic heterogeneity at the HLA-DRB1 locus, but also reflects an independent effect from the HLA-C gene. Specifically, the HLA-C*05 allele, or a variant in tight linkage disequilibrium with it, appears to exert a protective effect (p = 3.3 x 10(-5)).</p><p class="abstract"><strong>Interpretation</strong>: Variation in the HLA-C gene influences susceptibility to multiple sclerosis independently of any effect attributable to the nearby HLA-DRB1 gene.</p>
<p class="grants">Funded by: Multiple Sclerosis Society: <span class="grant">588</span>; NINDS NIH HHS: <span class="grant">K08 NS46341</span>, <span class="grant">NS026799</span>, <span class="grant">NS032830</span>, <span class="grant">NS049477</span>; Wellcome Trust: <span class="grant">048880</span>, <span class="grant">057097</span></p>
<p class="publication"><span class="title">Annals of neurology</span>&nbsp;<span class="year">2007</span>;<span class="volume">61</span>;<span class="number">3</span>;<span class="pagerange">228-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17252545">17252545</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2737610">2737610</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fana.21063">10.1002/ana.21063</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insights into modern disease from our distant evolutionary past.</h4>
<p class="authors"><span class="author">Yngvadottir B</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%62%79%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#121;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">An EMBO workshop entitled &#39;Human Evolution and Disease&#39; was held recently (6-9 December 2006, Hyderabad, India) where 141 scientists from many disciplines came together to discuss recent studies of human variation, origins and dispersal, natural selection and disease susceptibility. The meeting tackled the subject of human evolution and disease from the different perspectives of archaeology, linguistics, genetics and genomics based on both new and publicly available data sets. In this report, we highlight the latest fashion crazes in the discipline, in particular, the use of large public data sets and new methods to analyse modern human variation and the links between human evolution and disease susceptibility.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">2007</span>;<span class="volume">15</span>;<span class="number">5</span>;<span class="pagerange">603-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17356547">17356547</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5201811">10.1038/sj.ejhg.5201811</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The V103I polymorphism of the MC4R gene and obesity: population based studies and meta-analysis of 29 563 individuals.</h4>
<p class="authors"><span class="author">Young EH</span>, <span class="author">Wareham NJ</span>, <span class="author">Farooqi S</span>, <span class="author">Hinney A</span>, <span class="author">Hebebrand J</span>, <span class="author">Scherag A</span>, <span class="author">O&#39;rahilly S</span>, <span class="author">Barroso I</span> and <span class="author">Sandhu MS</span></p>
<p class="affiliation">MRC Epidemiology Unit, Strangeways Research Laboratory, Cambridge, UK. <a href="mailto:%65%6C%69%7A%61%62%65%74%68.%79%6F%75%6E%67%40%6D%72%63-%65%70%69%64.%63%61%6D.%61%63.%75%6B">&#101;&#108;&#105;&#122;&#97;&#98;&#101;&#116;&#104;&#46;&#121;&#111;&#117;&#110;&#103;&#64;&#109;&#114;&#99;&#45;&#101;&#112;&#105;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Previous studies have suggested that a variant in the melanocortin-4 receptor (MC4R) gene is important in protecting against common obesity. Larger studies are needed, however, to confirm this relation.</p><p class="abstract"><strong>Methods</strong>: We assessed the association between the V103I polymorphism in the MC4R gene and obesity in three UK population based cohort studies, totalling 8304 individuals. We also did a meta-analysis of relevant studies, involving 10 975 cases and 18 588 controls, to place our findings in context.</p><p class="abstract"><strong>Finding</strong>: In an analysis of all studies, individuals carrying the isoleucine allele had an 18% (95% confidence interval 4-30%, P=0.015) lower risk of obesity compared with non-carriers. There was no heterogeneity among studies and no apparent publication bias.</p><p class="abstract"><strong>Interpretation</strong>: This study confirms that the V103I polymorphism protects against human obesity at a population level. As such it provides proof of principle that specific gene variants may, at least in part, explain susceptibility and resistance to common forms of human obesity. A better understanding of the mechanisms underlying this association will help determine whether changes in MC4R activity have therapeutic potential.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0100103</span>, <span class="grant">G9824984</span>, <span class="grant">MC_U106179471</span>, <span class="grant">MC_U106188470</span>; Wellcome Trust: <span class="grant">068086</span>, <span class="grant">077016</span></p>
<p class="publication"><span class="title">International journal of obesity (2005)</span>&nbsp;<span class="year">2007</span>;<span class="volume">31</span>;<span class="number">9</span>;<span class="pagerange">1437-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17356525">17356525</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2683751">2683751</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ijo.0803609">10.1038/sj.ijo.0803609</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping of KIT adjacent sequences on canid autosomes and B chromosomes.</h4>
<p class="authors"><span class="author">Yudkin DV</span>, <span class="author">Trifonov VA</span>, <span class="author">Kukekova AV</span>, <span class="author">Vorobieva NV</span>, <span class="author">Rubtsova NV</span>, <span class="author">Yang F</span>, <span class="author">Acland GM</span>, <span class="author">Ferguson-Smith MA</span> and <span class="author">Graphodatsky AS</span></p>
<p class="affiliation">Institute of Cytology and Genetics, SB RAS, Novosibirsk, Russia.</p><p class="abstract">B chromosomes are often considered to be one of the most mysterious elements of karyotypes (Camacho, 2004). It is generally believed that mammalian B chromosomes do not contain any protein coding genes. The discovery of a conserved KIT gene in Canidae B chromosomes has changed this view. Here we performed analysis of sequences surrounding KIT in B chromosomes of the fox and raccoon dog. The presence of the RPL23A pseudogene was shown in canid B chromosomes. The 3&#39; end fragment of the KDR gene was found in raccoon dog B chromosomes. The size of the B-specific fragment homologous to the autosome fragment was estimated to be a minimum of 480 kbp in both species. The origin and evolution of B chromosomes in Canidae are discussed.</p>
<p class="grants">Funded by: NIMH NIH HHS: <span class="grant">MH069688</span></p>
<p class="publication"><span class="title">Cytogenetic and genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">116</span>;<span class="number">1-2</span>;<span class="pagerange">100-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17268185" rel="external">17268185</a>; DOI: <a href="http://dx.doi.org/10.1159%2F000097424" rel="external">10.1159/000097424</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new function for the fragile X mental retardation protein in regulation of PSD-95 mRNA stability.</h4>
<p class="authors"><span class="author">Zalfa F</span>, <span class="author">Eleuteri B</span>, <span class="author">Dickson KS</span>, <span class="author">Mercaldo V</span>, <span class="author">De Rubeis S</span>, <span class="author">di Penta A</span>, <span class="author">Tabolacci E</span>, <span class="author">Chiurazzi P</span>, <span class="author">Neri G</span>, <span class="author">Grant SG</span> and <span class="author">Bagni C</span></p>
<p class="affiliation">Dipartimento di Biologia, Universit&agrave; Tor Vergata, Via della Ricerca Scientifica 1, 00133 Rome, Italy.</p><p class="abstract">Fragile X syndrome (FXS) results from the loss of the fragile X mental retardation protein (FMRP), an RNA-binding protein that regulates a variety of cytoplasmic mRNAs. FMRP regulates mRNA translation and may be important in mRNA localization to dendrites. We report a third cytoplasmic regulatory function for FMRP: control of mRNA stability. In mice, we found that FMRP binds, in vivo, the mRNA encoding PSD-95, a key molecule that regulates neuronal synaptic signaling and learning. This interaction occurs through the 3&#39; untranslated region of the PSD-95 (also known as Dlg4) mRNA, increasing message stability. Moreover, stabilization is further increased by mGluR activation. Although we also found that the PSD-95 mRNA is synaptically localized in vivo, localization occurs independently of FMRP. Through our functional analysis of this FMRP target we provide evidence that dysregulation of mRNA stability may contribute to the cognitive impairments in individuals with FXS.</p>
<p class="grants">Funded by: Telethon: <span class="grant">GGP05269</span>; Wellcome Trust: <span class="grant">056523</span>, <span class="grant">077155</span></p>
<p class="publication"><span class="title">Nature neuroscience</span>&nbsp;<span class="year">2007</span>;<span class="volume">10</span>;<span class="number">5</span>;<span class="pagerange">578-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17417632">17417632</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2804293">2804293</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnn1893">10.1038/nn1893</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes.</h4>
<p class="authors"><span class="author">Zeggini E</span>, <span class="author">Weedon MN</span>, <span class="author">Lindgren CM</span>, <span class="author">Frayling TM</span>, <span class="author">Elliott KS</span>, <span class="author">Lango H</span>, <span class="author">Timpson NJ</span>, <span class="author">Perry JR</span>, <span class="author">Rayner NW</span>, <span class="author">Freathy RM</span>, <span class="author">Barrett JC</span>, <span class="author">Shields B</span>, <span class="author">Morris AP</span>, <span class="author">Ellard S</span>, <span class="author">Groves CJ</span>, <span class="author">Harries LW</span>, <span class="author">Marchini JL</span>, <span class="author">Owen KR</span>, <span class="author">Knight B</span>, <span class="author">Cardon LR</span>, <span class="author">Walker M</span>, <span class="author">Hitman GA</span>, <span class="author">Morris AD</span>, <span class="author">Doney AS</span>, <span class="author">Wellcome Trust Case Control Consortium (WTCCC)</span>, <span class="author">McCarthy MI</span> and <span class="author">Hattersley AT</span></p>
<p class="affiliation">Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK.</p><p class="abstract">The molecular mechanisms involved in the development of type 2 diabetes are poorly understood. Starting from genome-wide genotype data for 1924 diabetic cases and 2938 population controls generated by the Wellcome Trust Case Control Consortium, we set out to detect replicated diabetes association signals through analysis of 3757 additional cases and 5346 controls and by integration of our findings with equivalent data from other international consortia. We detected diabetes susceptibility loci in and around the genes CDKAL1, CDKN2A/CDKN2B, and IGF2BP2 and confirmed the recently described associations at HHEX/IDE and SLC30A8. Our findings provide insight into the genetic architecture of type 2 diabetes, emphasizing the contribution of multiple variants of modest effect. The regions identified underscore the importance of pathways influencing pancreatic beta cell development and function in the etiology of type 2 diabetes.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">G0000934</span>, <span class="grant">G0500070</span>; Wellcome Trust: <span class="grant">083948</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2007</span>;<span class="volume">316</span>;<span class="number">5829</span>;<span class="pagerange">1336-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17463249">17463249</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC3772310">3772310</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1142364">10.1126/science.1142364</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Y-chromosomal insights into the genetic impact of the caste system in India.</h4>
<p class="authors"><span class="author">Zerjal T</span>, <span class="author">Pandya A</span>, <span class="author">Thangaraj K</span>, <span class="author">Ling EY</span>, <span class="author">Kearley J</span>, <span class="author">Bertoneri S</span>, <span class="author">Paracchini S</span>, <span class="author">Singh L</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs, CB10 1SA, UK.</p><p class="abstract">The caste system has persisted in Indian Hindu society for around 3,500 years. Like the Y chromosome, caste is defined at birth, and males cannot change their caste. In order to investigate the genetic consequences of this system, we have analysed male-lineage variation in a sample of 227 Indian men of known caste, 141 from the Jaunpur district of Uttar Pradesh and 86 from the rest of India. We typed 131 Y-chromosomal binary markers and 16 microsatellites. We find striking evidence for male substructure: in particular, Brahmins and Kshatriyas (but not other castes) from Jaunpur each show low diversity and the predominance of a single distinct cluster of haplotypes. These findings confirm the genetic isolation and drift within the Jaunpur upper castes, which are likely to result from founder effects and social factors. In the other castes, there may be either larger effective population sizes, or less strict isolation, or both.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077009</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2007</span>;<span class="volume">121</span>;<span class="number">1</span>;<span class="pagerange">137-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17075717" rel="external">17075717</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2590678" rel="external">2590678</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-006-0282-2" rel="external">10.1007/s00439-006-0282-2</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inhibition of the dopamine D1 receptor signaling by PSD-95.</h4>
<p class="authors"><span class="author">Zhang J</span>, <span class="author">Vinuela A</span>, <span class="author">Neely MH</span>, <span class="author">Hallett PJ</span>, <span class="author">Grant SG</span>, <span class="author">Miller GM</span>, <span class="author">Isacson O</span>, <span class="author">Caron MG</span> and <span class="author">Yao WD</span></p>
<p class="affiliation">Department of Psychiatry, Harvard Medical School, New England Primate Research Center, Southborough, Massachusetts 01772, USA.</p><p class="abstract">Dopamine D1 receptors play an important role in movement, reward, and learning and are implicated in a number of neurological and psychiatric disorders. These receptors are concentrated in dendritic spines of neurons, including the spine head and the postsynaptic density. D1 within spines is thought to modulate the local channels and receptors to control the excitability and synaptic properties of spines. The molecular mechanisms mediating D1 trafficking, anchorage, and function in spines remain elusive. Here we show that the synaptic scaffolding protein PSD-95 thought to play a role in stabilizing glutamate receptors in the postsynaptic density, interacts with D1 and regulates its trafficking and function. Interestingly, the D1-PSD-95 interaction does not require the well characterized domains of PSD-95 but is mediated by the carboxyl-terminal tail of D1 and the NH(2) terminus of PSD-95, a region that is recognized only recently to participate in protein-protein interaction. Co-expression of PSD-95 with D1 in mammalian cells inhibits the D1-mediated cAMP accumulation without altering the total expression level or the agonist binding properties of the receptor. The diminished D1 signaling is mediated by reduced D1 expression at the cell surface as a consequence of an enhanced constitutive, dynamin-dependent endocytosis. In addition, genetically engineered mice lacking PSD-95 show a heightened behavioral response to either a D1 agonist or the psychostimulant amphetamine. These studies demonstrate a role for a glutamatergic scaffold in dopamine receptor signaling and trafficking and identify a new potential target for the modulation of abnormal dopaminergic function.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P51 RR000168-430106</span>, <span class="grant">RR00168</span>; NINDS NIH HHS: <span class="grant">P50 NS039793-07</span>, <span class="grant">P50 NS039793-08</span>, <span class="grant">P50 NS39793</span></p>
<p class="publication"><span class="title">The Journal of biological chemistry</span>&nbsp;<span class="year">2007</span>;<span class="volume">282</span>;<span class="number">21</span>;<span class="pagerange">15778-89</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17369255" rel="external">17369255</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2649122" rel="external">2649122</a>; DOI: <a href="http://dx.doi.org/10.1074%2Fjbc.M611485200" rel="external">10.1074/jbc.M611485200</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pseudogenes in the ENCODE regions: consensus annotation, analysis of transcription, and evolution.</h4>
<p class="authors"><span class="author">Zheng D</span>, <span class="author">Frankish A</span>, <span class="author">Baertsch R</span>, <span class="author">Kapranov P</span>, <span class="author">Reymond A</span>, <span class="author">Choo SW</span>, <span class="author">Lu Y</span>, <span class="author">Denoeud F</span>, <span class="author">Antonarakis SE</span>, <span class="author">Snyder M</span>, <span class="author">Ruan Y</span>, <span class="author">Wei CL</span>, <span class="author">Gingeras TR</span>, <span class="author">Guig&oacute; R</span>, <span class="author">Harrow J</span> and <span class="author">Gerstein MB</span></p>
<p class="affiliation">Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06520, USA. <a href="mailto:%64%65%79%6F%75.%7A%68%65%6E%67%40%79%61%6C%65.%65%64%75">&#100;&#101;&#121;&#111;&#117;&#46;&#122;&#104;&#101;&#110;&#103;&#64;&#121;&#97;&#108;&#101;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Arising from either retrotransposition or genomic duplication of functional genes, pseudogenes are &quot;genomic fossils&quot; valuable for exploring the dynamics and evolution of genes and genomes. Pseudogene identification is an important problem in computational genomics, and is also critical for obtaining an accurate picture of a genome&#39;s structure and function. However, no consensus computational scheme for defining and detecting pseudogenes has been developed thus far. As part of the ENCyclopedia Of DNA Elements (ENCODE) project, we have compared several distinct pseudogene annotation strategies and found that different approaches and parameters often resulted in rather distinct sets of pseudogenes. We subsequently developed a consensus approach for annotating pseudogenes (derived from protein coding genes) in the ENCODE regions, resulting in 201 pseudogenes, two-thirds of which originated from retrotransposition. A survey of orthologs for these pseudogenes in 28 vertebrate genomes showed that a significant fraction ( approximately 80%) of the processed pseudogenes are primate-specific sequences, highlighting the increasing retrotransposition activity in primates. Analysis of sequence conservation and variation also demonstrated that most pseudogenes evolve neutrally, and processed pseudogenes appear to have lost their coding potential immediately or soon after their emergence. In order to explore the functional implication of pseudogene prevalence, we have extensively examined the transcriptional activity of the ENCODE pseudogenes. We performed systematic series of pseudogene-specific RACE analyses. These, together with complementary evidence derived from tiling microarrays and high throughput sequencing, demonstrated that at least a fifth of the 201 pseudogenes are transcribed in one or more cell lines or tissues.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01HG03147</span>, <span class="grant">U01HG03150</span>, <span class="grant">U01HG03156</span>; PHS HHS: <span class="grant">N01C012400</span>; Wellcome Trust: <span class="grant">077198</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2007</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">839-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17568002">17568002</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1891343">1891343</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.5586307">10.1101/gr.5586307</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Towards efficient registration of medical images.</h4>
<p class="authors"><span class="author">Zhou H</span>, <span class="author">Liu T</span>, <span class="author">Lin F</span>, <span class="author">Pang Y</span>, <span class="author">Wu J</span> and <span class="author">Wu J</span></p>
<p class="affiliation">Queen Mary College, University of London, London, UK. <a href="mailto:%68%75%69%79%75.%7A%68%6F%75%40%65%6C%65%63.%71%6D%75%6C.%61%63.%75%6B">&#104;&#117;&#105;&#121;&#117;&#46;&#122;&#104;&#111;&#117;&#64;&#101;&#108;&#101;&#99;&#46;&#113;&#109;&#117;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In this paper we propose a Bayesian based mutual information technique for image registration, combined with an established affine transformation model. Classical affine models allow the images to be approximately aligned. However, inefficiency and inaccuracy has appeared when using these affine models in rigorous circumstances, such as low-resolution images. To challenge this problem, we conduct mutual information measures with importance sampling to the images in an attempt to simulate the probability distribution of intensity similarity across the images. The entire registration adopts a stopping criterion as discovered in the context of differential equations. Finally, experimental results demonstrate the favorable performance of the proposed algorithm.</p>
<p class="publication"><span class="title">Computerized medical imaging and graphics : the official journal of the Computerized Medical Imaging Society</span>&nbsp;<span class="year">2007</span>;<span class="volume">31</span>;<span class="number">6</span>;<span class="pagerange">374-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/17386999" rel="external">17386999</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.compmedimag.2007.02.018" rel="external">10.1016/j.compmedimag.2007.02.018</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/rvm0cvfd"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/rvm0cvfd" src="/qr/rvm0cvfd.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
