<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 1997 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 1997.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 1997</h1><p>Number of papers published in 1997: <strong>47</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A 12-cistron Escherichia coli operon (hyf) encoding a putative proton-translocating formate hydrogenlyase system.</h4>
<p class="authors"><span class="author">Andrews SC</span>, <span class="author">Berks BC</span>, <span class="author">McClay J</span>, <span class="author">Ambler A</span>, <span class="author">Quail MA</span>, <span class="author">Golby P</span> and <span class="author">Guest JR</span></p>
<p class="affiliation">Krebs Institute, Department of Molecular Biology &amp; Biotechnology, University of Sheffield, UK. <a href="mailto:%73.%61%6E%64%72%65%77%73%40%73%68%65%66%66%69%65%6C%64.%61%63.%75%6B">&#115;&#46;&#97;&#110;&#100;&#114;&#101;&#119;&#115;&#64;&#115;&#104;&#101;&#102;&#102;&#105;&#101;&#108;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The nucleotide sequence has been determined for a twelve-gene operon of Escherichia coli designated the hyf operon (hyfABCDEFGHIR-focB). The hyf operon is located at 55.8-56.0 min and encodes a putative nine-subunit hydrogenase complex (hydrogenase four or Hyf), a potential formate- and sigma 54-dependent transcriptional activator, HyfR (related to FhlA), and a possible formate transporter, FocB (related to FocA). Five of the nine Hyf-complex subunits are related to subunits of both the E. coli hydrogenase-3 complex (Hyc) and the proton-translocating NADH:quinone oxidoreductases (complex I and Nuo), whereas two Hyf subunits are related solely to NADH:quinone oxidoreductase subunits. The Hyf components include a predicted 523 residue [Ni-Fe] hydrogenase (large subunit) with an N-terminus (residues 1-170) homologous to the 30 kDa or NuoC subunit of complex I. It is proposed that Hyf, in conjunction with formate dehydrogenase H (Fdh-H), forms a hitherto unrecognized respiration-linked proton-translocating formate hydrogenlyase (FHL-2). It is likely that HyfR acts as a formate-dependent regulator of the hyf operon and that FocB provides the Hyf complex with external formate as substrate.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">1997</span>;<span class="volume">143 ( Pt 11)</span>;<span class="pagerange">3633-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9387241" rel="external">9387241</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The chromosome 6 sequencing project at the Sanger Centre.</h4>
<p class="authors"><span class="author">Avis T</span>, <span class="author">Clark EK</span>, <span class="author">Flack TL</span>, <span class="author">Mohammadi M</span>, <span class="author">Milne S</span>, <span class="author">Niblett D</span>, <span class="author">Palmer S</span>, <span class="author">Phillips S</span>, <span class="author">Smalley C</span>, <span class="author">Tagney M</span>, <span class="author">Thorpe KL</span>, <span class="author">Tubby B</span>, <span class="author">Westhorp J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, England.</p><p class="abstract">Chromosome 6 is probably best known for encoding the major histocompatibility complex (MHC) which is essential to the human immune response. In addition, it has been shown to be associated with many diseases such Schizophrenia, Diabetes, Arthritis, Haemochromatosis, Narcolepsy, Epilepsy, Retinitis Pigmentosa, Deafness, Ovarian Cancer, and many more. Chromosome 6 is about 180 Mb in size and is estimated to encode around 3500 genes of which only about 10% are currently known. It is our aim to map, sequence and annotate the entire chromosome in close collaboration with the chromosome 6 community.</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1997</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">131-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10668956" rel="external">10668956</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Third single chromosome 6 workshop: meeting report.</h4>
<p class="authors"><span class="author">Beck S</span>, <span class="author">Cann HM</span>, <span class="author">Campbell RD</span>, <span class="author">Dunham I</span>, <span class="author">Inoko H</span>, <span class="author">Jazwinska EC</span>, <span class="author">Ragoussis J</span>, <span class="author">Trowsdale J</span> and <span class="author">Ziegler A</span></p>
<p class="affiliation">The Sanger Centre, Cambridge, UK. <a href="mailto:%62%65%63%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#101;&#99;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1997</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">113-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10668955" rel="external">10668955</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The BRC repeats are conserved in mammalian BRCA2 proteins.</h4>
<p class="authors"><span class="author">Bignell G</span>, <span class="author">Micklem G</span>, <span class="author">Stratton MR</span>, <span class="author">Ashworth A</span> and <span class="author">Wooster R</span></p>
<p class="affiliation">Section of Molecular Carcinogenesis, Institute of Cancer Research, Sutton, UK.</p><p class="abstract">The breast cancer susceptibility gene BRCA2 encodes a protein of 3418 amino acids which does not exhibit substantial sequence similarity to any other protein in the public databases. A dot matrix comparison of BRCA2 with itself revealed an eight times repeated motif in the segment of the protein encoded by exon 11. As a preliminary test of the hypothesis that these motifs are functionally significant, we have sequenced exon 11 of BRCA2 in six mammals. An alignment of the predicted protein sequences shows that, overall, the motifs have been conserved while much of the intervening sequences has diverged. These data support the notion that the BRC motifs are important in BRCA2 function. There is, however, considerable interspecies variation within certain motif units, raising the possibility of redundancy and that not all of the repeats are required for the normal function of BRCA2.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">53-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9002670" rel="external">9002670</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamite: a flexible code generating language for dynamic programming methods used in sequence comparison.</h4>
<p class="authors"><span class="author">Birney E</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Cambridge, UK. <a href="mailto:%62%69%72%6E%65%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#105;&#114;&#110;&#101;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have developed a code generating language, called Dynamite, specialised for the production and subsequent manipulation of complex dynamic programming methods for biological sequence comparison. From a relatively simple text definition file Dynamite will produce a variety of implementations of a dynamic programming method, including database searches and linear space alignments. The speed of the generated code is comparable to hand written code, and the additional flexibility has proved invaluable in designing and testing new algorithms. An innovation is a flexible labelling system, which can be used to annotate the original sequences with biological information. We illustrate the Dynamite syntax and flexibility by showing definitions for dynamic programming routines (i) to align two protein sequences under the assumption that they are both poly-topic transmembrane proteins, with the simultaneous assignment of transmembrane helices and (ii) to align protein information to genomic DNA, allowing for introns and sequencing error.</p>
<p class="publication"><span class="title">Proceedings / ... International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">5</span>;<span class="pagerange">56-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9322016" rel="external">9322016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The nucleotide sequence of Saccharomyces cerevisiae chromosome XIII.</h4>
<p class="authors"><span class="author">Bowman S</span>, <span class="author">Churcher C</span>, <span class="author">Badcock K</span>, <span class="author">Brown D</span>, <span class="author">Chillingworth T</span>, <span class="author">Connor R</span>, <span class="author">Dedman K</span>, <span class="author">Devlin K</span>, <span class="author">Gentles S</span>, <span class="author">Hamlin N</span>, <span class="author">Hunt S</span>, <span class="author">Jagels K</span>, <span class="author">Lye G</span>, <span class="author">Moule S</span>, <span class="author">Odell C</span>, <span class="author">Pearson D</span>, <span class="author">Rajandream M</span>, <span class="author">Rice P</span>, <span class="author">Skelton J</span>, <span class="author">Walsh S</span>, <span class="author">Whitehead S</span> and <span class="author">Barrell B</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Systematic sequencing of the genome of Saccharomyces cerevisiae has revealed thousands of new predicted genes and allowed analysis of long-range features of chromosomal organization. Generally, genes and predicted genes seem to be distributed evenly throughout the genome, having no overall preference for DNA strand. Apart from the smaller chromosomes, which can have substantially lower gene density in their telomeric regions, there is a consistent average of one open reading frame (ORF) approximately every two kilobases. However, one of the most surprising findings for a eukaryote with approximately 6,000 genes was the amount of apparent redundancy in its genome. This redundancy occurs both between individual ORFs and over more extensive chromosome regions, which have been duplicated preserving gene order and orientation. Here we report the entire nucleotide sequence of chromosome XIII, the sixth-largest S. cerevisiae chromosome, and demonstrate that its features and organization are consistent with those observed for other S. cerevisiae chromosomes. Analysis revealed 459 ORFs, 284 have not been identified previously. Both intra- and interchromosomal duplications of regions of this chromosome have occurred.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1997</span>;<span class="volume">387</span>;<span class="number">6632 Suppl</span>;<span class="pagerange">90-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9169872">9169872</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of equine X chromosome abnormalities in equids using a horse X whole chromosome paint probe (WCPP).</h4>
<p class="authors"><span class="author">Breen M</span>, <span class="author">Langford CF</span>, <span class="author">Carter NP</span>, <span class="author">Fischer PE</span>, <span class="author">Marti E</span>, <span class="author">Gerstenberg C</span>, <span class="author">Allen WR</span>, <span class="author">Lear TL</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Centre for Preventive Medicine, Animal Health Trust, Newmarket, Suffolk, U.K.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Veterinary journal (London, England : 1997)</span>&nbsp;<span class="year">1997</span>;<span class="volume">153</span>;<span class="number">3</span>;<span class="pagerange">235-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9232112" rel="external">9232112</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Population statistics of protein structures: lessons from structural classifications.</h4>
<p class="authors"><span class="author">Brenner SE</span>, <span class="author">Chothia C</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Structural Biology Centre, National Institute for Bioscience and Human-Technology, Ibaraki, Japan. <a href="mailto:%62%72%65%6E%6E%65%72%40%68%79%70%65%72.%73%74%61%6E%66%6F%72%64.%65%64%75">&#98;&#114;&#101;&#110;&#110;&#101;&#114;&#64;&#104;&#121;&#112;&#101;&#114;&#46;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Structural classifications aid the interpretation of proteins by describing degrees of structural and evolutionary relatedness. They have also recently revealed strikingly skewed distributions at all levels; for example, a small number of folds are far more common than others, and just a few superfamilies are known to have diverged widely. The classifications also provide an indication of the total number of superfamilies in nature.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in structural biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">369-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9204279">9204279</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The nucleotide sequence of Saccharomyces cerevisiae chromosome XVI.</h4>
<p class="authors"><span class="author">Bussey H</span>, <span class="author">Storms RK</span>, <span class="author">Ahmed A</span>, <span class="author">Albermann K</span>, <span class="author">Allen E</span>, <span class="author">Ansorge W</span>, <span class="author">Araujo R</span>, <span class="author">Aparicio A</span>, <span class="author">Barrell B</span>, <span class="author">Badcock K</span>, <span class="author">Benes V</span>, <span class="author">Botstein D</span>, <span class="author">Bowman S</span>, <span class="author">Br&uuml;ckner M</span>, <span class="author">Carpenter J</span>, <span class="author">Cherry JM</span>, <span class="author">Chung E</span>, <span class="author">Churcher C</span>, <span class="author">Coster F</span>, <span class="author">Davis K</span>, <span class="author">Davis RW</span>, <span class="author">Dietrich FS</span>, <span class="author">Delius H</span>, <span class="author">DiPaolo T</span>, <span class="author">Hani J</span> <em>et al.</em></p>
<p class="affiliation">Department of Biology, McGill University, Montreal, Canada. <a href="mailto:%68%62%75%73%73%65%79%40%6D%6F%6E%6F%64.%62%69%6F%6C.%6D%63%67%69%6C%6C.%63%61">&#104;&#98;&#117;&#115;&#115;&#101;&#121;&#64;&#109;&#111;&#110;&#111;&#100;&#46;&#98;&#105;&#111;&#108;&#46;&#109;&#99;&#103;&#105;&#108;&#108;&#46;&#99;&#97;</a></p><p class="abstract">The nucleotide sequence of the 948,061 base pairs of chromosome XVI has been determined, completing the sequence of the yeast genome. Chromosome XVI was the last yeast chromosome identified, and some of the genes mapped early to it, such as GAL4, PEP4 and RAD1 (ref. 2) have played important roles in the development of yeast biology. The architecture of this final chromosome seems to be typical of the large yeast chromosomes, and shows large duplications with other yeast chromosomes. Chromosome XVI contains 487 potential protein-encoding genes, 17 tRNA genes and two small nuclear RNA genes; 27% of the genes have significant similarities to human gene products, and 48% are new and of unknown biological function. Systematic efforts to explore gene function have begun.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1997</span>;<span class="volume">387</span>;<span class="number">6632 Suppl</span>;<span class="pagerange">103-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9169875">9169875</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The nucleotide sequence of Saccharomyces cerevisiae chromosome IX.</h4>
<p class="authors"><span class="author">Churcher C</span>, <span class="author">Bowman S</span>, <span class="author">Badcock K</span>, <span class="author">Bankier A</span>, <span class="author">Brown D</span>, <span class="author">Chillingworth T</span>, <span class="author">Connor R</span>, <span class="author">Devlin K</span>, <span class="author">Gentles S</span>, <span class="author">Hamlin N</span>, <span class="author">Harris D</span>, <span class="author">Horsnell T</span>, <span class="author">Hunt S</span>, <span class="author">Jagels K</span>, <span class="author">Jones M</span>, <span class="author">Lye G</span>, <span class="author">Moule S</span>, <span class="author">Odell C</span>, <span class="author">Pearson D</span>, <span class="author">Rajandream M</span>, <span class="author">Rice P</span>, <span class="author">Rowley N</span>, <span class="author">Skelton J</span>, <span class="author">Smith V</span>, <span class="author">Barrell B</span> <em>et al.</em></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Large-scale systematic sequencing has generally depended on the availability of an ordered library of large-insert bacterial or viral genomic clones for the organism under study. The generation of these large insert libraries, and the location of each clone on a genome map, is a laborious and time-consuming process. In an effort to overcome these problems, several groups have successfully demonstrated the viability of the whole-genome random &#39;shotgun&#39; method in large-scale sequencing of both viruses and prokaryotes. Here we report the sequence of Saccharomyces cerevisiae chromosome IX, determined in part by a whole-chromosome &#39;shotgun&#39;, and describe the particular difficulties encountered in the random &#39;shotgun&#39; sequencing of an entire eukaryotic chromosome. Analysis of this sequence shows that chromosome IX contains 221 open reading frames (ORFs), of which approximately 30% have been sequenced previously. This chromosome shows features typical of a small Saccharomyces cerevisiae chromosome.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1997</span>;<span class="volume">387</span>;<span class="number">6632 Suppl</span>;<span class="pagerange">84-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9169870">9169870</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The organization of the gamma-glutamyl transferase genes and other low copy repeats in human chromosome 22q11.</h4>
<p class="authors"><span class="author">Collins JE</span>, <span class="author">Mungall AJ</span>, <span class="author">Badcock KL</span>, <span class="author">Fay JM</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambs, UK. <a href="mailto:%6A%65%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#101;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">A clone map consisting of YACs, cosmids, and fosmids has been constructed covering low copy repeat regions of human chromosome 22q11. A combination of clone restriction digest analysis, single-copy landmark content analysis, HindIII-Sau3AI fingerprinting, and sequencing of PCR products derived from clones was required to resolve the map in this region. Seven repeat-containing contigs were placed in 22q11, five containing gamma-glutamyl transferase (GGT) sequences described previously. In one case, a single interval at the resolution of the YAC map was shown to contain at least three GGT sequences after higher resolution mapping. The sequence information was used to design a rapid PCR/restriction digest technique that distinguishes the GGT loci placed in the YAC map. This approach has allowed us to resolve the previous cDNA and mapping information relating to GGT and link it to the physical map of 22q11.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1997</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">522-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9149947" rel="external">9149947</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cloning, chromosomal mapping and expression pattern of the mouse Brca2 gene.</h4>
<p class="authors"><span class="author">Connor F</span>, <span class="author">Smith A</span>, <span class="author">Wooster R</span>, <span class="author">Stratton M</span>, <span class="author">Dixon A</span>, <span class="author">Campbell E</span>, <span class="author">Tait TM</span>, <span class="author">Freeman T</span> and <span class="author">Ashworth A</span></p>
<p class="affiliation">CRC Centre for Cell and Molecular Biology, Chester Beatty Laboratories, Institute of Cancer Research, London, UK.</p><p class="abstract">A proportion of human breast cancers result from an inherited predisposition to the disease. Mutations in the BRCA2 gene confer a high risk of breast cancer and are responsible for almost half of these cases. The recent cloning of the human BRCA2 gene has revealed that it encodes a large protein having little significant homology to known proteins. Here we describe the mouse Brca2 gene. The gene maps to mouse chromosome 5, consistent with its location on human chromosome 13q12. We have sequenced cDNA for the entire 3329 amino acid Brca2 protein and this has revealed that, like Brca1, Brca2 is relatively poorly conserved between humans and mice. Brca2 is transcribed in a diverse range of mouse tissues, and the pattern of expression is strikingly similar to that of Brca1. Taken together, our data highlight some intriguing similarities between two genes involved in inherited breast cancer susceptibility.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">6</span>;<span class="number">2</span>;<span class="pagerange">291-300</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9063750" rel="external">9063750</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence of the human immunoglobulin diversity (D) segment locus: a systematic analysis provides no evidence for the use of DIR segments, inverted D segments, &quot;minor&quot; D segments or D-D recombination.</h4>
<p class="authors"><span class="author">Corbett SJ</span>, <span class="author">Tomlinson IM</span>, <span class="author">Sonnhammer EL</span>, <span class="author">Buck D</span> and <span class="author">Winter G</span></p>
<p class="affiliation">MRC Centre For Protein Engineering, Hills Road, Cambridge, CB2 2QH, U.K.</p><p class="abstract">We have determined the complete nucleotide sequence of the human immunoglobulin D segment locus on chromosome 14q32.3 and identified a total of 27 D segments, of which nine are new. Comparison with a database of rearranged heavy chain sequences indicates that the human antibody repertoire is created by VDJ recombination involving 25 of these 27 D segments, extensive processing at the V-D and D-J junctions and use of multiple reading frames. We could find no evidence for the proposed use of DIR segments, inverted D segments, &quot;minor&quot; D segments or D-D recombination. Conventional VDJ recombination, which obeys the 12/23 rule, is therefore sufficient to explain the wealth of lengths and sequences for the third hypervariable loop of human heavy chains.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">270</span>;<span class="number">4</span>;<span class="pagerange">587-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9245589" rel="external">9245589</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1997.1141" rel="external">10.1006/jmbi.1997.1141</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of recombination in the HLA class II region.</h4>
<p class="authors"><span class="author">Cullen M</span>, <span class="author">Noble J</span>, <span class="author">Erlich H</span>, <span class="author">Thorpe K</span>, <span class="author">Beck S</span>, <span class="author">Klitz W</span>, <span class="author">Trowsdale J</span> and <span class="author">Carrington M</span></p>
<p class="affiliation">Intramural Research Support Program, SAIC Frederick, National Cancer Institute--Frederick Cancer Research and Development Center, MD 21702, USA.</p><p class="abstract">Studies of linkage disequilibrium across the HLA class II region have been useful in predicting where recombination is most likely to occur. The strong associations between genes within the 85-kb region from DQB1 to DRB1 are consistent with low frequency of recombination in this segment of DNA. Conversely, a lack of association between alleles of TAP1 and TAP2 (approximately 15 kb) has been observed, suggesting that recombination occurs here with relatively high frequency. Much of the HLA class II region has now been sequenced, providing the tools to undertake detailed analysis of recombination. Twenty-seven families containing one or two recombinant chromosomes within the 500-kb interval between the DPB1 and DRB1 genes were used to determine patterns of recombination across this region. SSCP analysis and microsatellite typing yielded identification of 127 novel polymorphic markers distributed throughout the class II region, allowing refinement of the site of crossover in 30 class II recombinant chromosomes. The three regions where recombination was observed most frequently are as follows: the 45-kb interval between HLA-DNA and RING3 (11 cases), the 50-kb interval between DQB3 and DQB1 (6 cases), and an 8.8-kb segment of the TAP2 gene (3 cases). Six of the 10 remaining recombinants await further characterization, pending identification of additional informative markers, while four recombinants were localized to other intervals (outliers). Analysis of association between markers flanking HLA-DNA to RING3 (45 kb), as well as TAP1 to TAP2 (15 kb), by use of independent CEPH haplotypes indicated little or no linkage disequilibrium, supporting the familial recombination data. A notable sequence motif located within a region associated with increased rates of recombination consisted of a (TGGA)12 tandem repeat within the TAP2 gene.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM35326</span>; Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">60</span>;<span class="number">2</span>;<span class="pagerange">397-407</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9012413" rel="external">9012413</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1712393" rel="external">1712393</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and characterization of a G protein-coupled receptor homolog encoded by murine cytomegalovirus.</h4>
<p class="authors"><span class="author">Davis-Poynter NJ</span>, <span class="author">Lynch DM</span>, <span class="author">Vally H</span>, <span class="author">Shellam GR</span>, <span class="author">Rawlinson WD</span>, <span class="author">Barrell BG</span> and <span class="author">Farrell HE</span></p>
<p class="affiliation">Department of Microbiology, University of Western Australia, Queen Elizabeth II Medical Centre, Nedlands, Australia. <a href="mailto:%6E%6A%64%70%40%75%6E%69%77%61.%75%77%61.%65%64%75.%61%75">&#110;&#106;&#100;&#112;&#64;&#117;&#110;&#105;&#119;&#97;&#46;&#117;&#119;&#97;&#46;&#101;&#100;&#117;&#46;&#97;&#117;</a></p><p class="abstract">This report describes the identification of a murine cytomegalovirus (MCMV) G protein-coupled receptor (GCR) homolog. This open reading frame (M33) is most closely related to, and collinear with, human cytomegalovirus UL33, and homologs are also present in human herpesvirus 6 and 7 (U12 for both viruses). Conserved counterparts in the sequenced alpha- or gammaherpesviruses have not been identified to date, suggesting that these genes encode proteins which are important for the biological characteristics of betaherpesviruses. We have detected transcripts for both UL33 and M33 as early as 3 or 4 h postinfection, and these reappear at late times. In addition, we have identified N-terminal splicing for both the UL33 and M33 RNA transcripts. For both open reading frames, splicing results in the introduction of amino acids which are highly conserved among known GCRs. To characterise the function of the M33 in the natural host, two independent MCMV recombinant viruses were prepared, each of which possesses an M33 open reading frame which has been disrupted with the beta-galactosidase gene. While the recombinant M33 null viruses showed no phenotypic differences in replication from wild-type MCMV in primary mouse embryo fibroblasts in vitro, they showed severely restricted growth in the salivary glands of infected mice. These data suggest that M33 plays an important role in vivo, in particular in the dissemination to or replication in the salivary gland, and provide the first evidence for the function of a viral GCR homolog in vivo.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of virology</span>&nbsp;<span class="year">1997</span>;<span class="volume">71</span>;<span class="number">2</span>;<span class="pagerange">1521-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8995678" rel="external">8995678</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC191209" rel="external">191209</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression of the dystrophin-related protein 2 (Drp2) transcript in the mouse.</h4>
<p class="authors"><span class="author">Dixon AK</span>, <span class="author">Tait TM</span>, <span class="author">Campbell EA</span>, <span class="author">Bobrow M</span>, <span class="author">Roberts RG</span> and <span class="author">Freeman TC</span></p>
<p class="affiliation">Human Genetics Group, The Sanger Centre, Wellcome Genome Campus, Hinxton, CB10 1SA, U.K.</p><p class="abstract">We have recently characterised a new member of the dystrophin gene family, DRP2, and its murine counterpart, Drp2, which encode dystrophin-related protein 2 (DRP2). DRP2 is predicted to resemble certain short C-terminal isoforms of dystrophin and dystrophin-related protein 1 (DRP1 or utrophin). We describe here a comprehensive survey of Drp2 expression in the mouse by RT-PCR, and compare the expression profile of Drp2 with that of the related genes Dmd, Drp1 and Dag1 that encode all the known isoforms of dystrophin, DRP1/utrophin and a component of the dystrophin-associated protein complex, dystroglycan, respectively. Drp2 was shown to be expressed throughout the central nervous system (CNS) and in several peripheral tissues including the eye, kidney, teeth, oesophagus, colon, epididymis and ovary. The expression of Drp2 in the CNS was then further defined by in situ hybridization. Overall, the pattern of Drp2 expression corresponds to a subset of the brain regions known to express Dag1, and shows substantial overlap with regions that express various isoforms of dystrophin (particularly in the cerebral cortex, hippocampus and cerebellum). These data define the distribution of Drp2 expression in the mouse, and raise the possibility that in the CNS it may be an important component in neuronal dystrophin-associated complexes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">270</span>;<span class="number">4</span>;<span class="pagerange">551-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9245586" rel="external">9245586</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1997.1138" rel="external">10.1006/jmbi.1997.1138</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence comparison of human and yeast telomeres identifies structurally distinct subtelomeric domains.</h4>
<p class="authors"><span class="author">Flint J</span>, <span class="author">Bates GP</span>, <span class="author">Clark K</span>, <span class="author">Dorman A</span>, <span class="author">Willingham D</span>, <span class="author">Roe BA</span>, <span class="author">Micklem G</span>, <span class="author">Higgs DR</span> and <span class="author">Louis EJ</span></p>
<p class="affiliation">Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford, UK. <a href="mailto:%6A%66%40%77%6F%72%66.%6D%6F%6C%62%69%6F%6C.%6F%78.%61%63.%75%6B">&#106;&#102;&#64;&#119;&#111;&#114;&#102;&#46;&#109;&#111;&#108;&#98;&#105;&#111;&#108;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have sequenced and compared DNA from the ends of three human chromosomes: 4p, 16p and 22q. In all cases the pro-terminal regions are subdivided by degenerate (TTAGGG)n repeats into distal and proximal sub-domains with entirely different patterns of homology to other chromosome ends. The distal regions contain numerous, short (&lt;2 kb) segments of interrupted homology to many other human telomeric regions. The proximal regions show much longer (approximately 10-40 kb) uninterrupted homology to a few chromosome ends. A comparison of all yeast subtelomeric regions indicates that they too are subdivided by degenerate TTAGGG repeats into distal and proximal sub-domains with similarly different patterns of identity to other non-homologous chromosome ends. Sequence comparisons indicate that the distal and proximal sub-domains do not interact with each other and that they interact quite differently with the corresponding regions on other, non-homologous, chromosomes. These findings suggest that the degenerate TTAGGG repeats identify a previously unrecognized, evolutionarily conserved boundary between remarkably different subtelomeric domains.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">6</span>;<span class="number">8</span>;<span class="pagerange">1305-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9259277" rel="external">9259277</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The relationship between chromosome structure and function at a human telomeric region.</h4>
<p class="authors"><span class="author">Flint J</span>, <span class="author">Thomas K</span>, <span class="author">Micklem G</span>, <span class="author">Raynham H</span>, <span class="author">Clark K</span>, <span class="author">Doggett NA</span>, <span class="author">King A</span> and <span class="author">Higgs DR</span></p>
<p class="affiliation">MRC Molecular Haematology Unit, John Radcliffe Hospital, Headington, Oxford, UK.</p><p class="abstract">We have sequenced a contiguous 284,495-bp segment of DNA extending from the terminal (TTAGGG)n repeats of the short arm of chromosome 16, providing a full description of the transition from telomeric through subtelomeric DNA to sequences that are unique to the chromosome. To complement and extend analysis of the primary sequence, we have characterized mRNA transcripts, patterns of DNA methylation and DNase I sensitivity. Together with previous data these studies describe in detail the structural and functional organization of a human telomeric region.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">15</span>;<span class="number">3</span>;<span class="pagerange">252-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9054936">9054936</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0397-252">10.1038/ng0397-252</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Immunoglobulin lambda light chain orphons on human chromosome 8q11.2.</h4>
<p class="authors"><span class="author">Frippiat JP</span>, <span class="author">Dard P</span>, <span class="author">Marsh S</span>, <span class="author">Winter G</span> and <span class="author">Lefranc MP</span></p>
<p class="affiliation">Centre for Protein Engineering, MRC, Cambridge, GB. <a href="mailto:%66%72%69%70%70%69%61%74%40%73%63%62%69%6F%6C.%75-%6E%61%6E%63%79.%66%72">&#102;&#114;&#105;&#112;&#112;&#105;&#97;&#116;&#64;&#115;&#99;&#98;&#105;&#111;&#108;&#46;&#117;&#45;&#110;&#97;&#110;&#99;&#121;&#46;&#102;&#114;</a></p><p class="abstract">We have identified two V lambda genes outside the major lambda locus on chromosome 22q11.2, and shown that they reside on chromosome 8q11.2. One gene (Orph&eacute;e1), hybridizing strongly to the V lambda probes, was sequenced and found to belong to the V lambda 8 family; the other gene (Orph&eacute;e2) only hybridized weakly. Orph&eacute;e1 was present in all individuals tested (140) from three different populations, and was also found in gorillas. We envisage that these genes were generated by duplication and translocation of the V lambda 8a gene (and a V lambda pseudogene) from the major locus, and that this event occurred before the evolutionary divergence of humans and gorillas. As there is no other evidence for V lambda genes outside the major locus, it appears that the human lambda locus has undergone considerably less evolutionary shuffling than either the human light chain kappa locus or the heavy chain locus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">1997</span>;<span class="volume">27</span>;<span class="number">5</span>;<span class="pagerange">1260-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9174619" rel="external">9174619</a>; DOI: <a href="http://dx.doi.org/10.1002%2Feji.1830270530" rel="external">10.1002/eji.1830270530</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Role of DNA mismatch repair in the cytotoxicity of ionizing radiation.</h4>
<p class="authors"><span class="author">Fritzell JA</span>, <span class="author">Narayanan L</span>, <span class="author">Baker SM</span>, <span class="author">Bronner CE</span>, <span class="author">Andrew SE</span>, <span class="author">Prolla TA</span>, <span class="author">Bradley A</span>, <span class="author">Jirik FR</span>, <span class="author">Liskay RM</span> and <span class="author">Glazer PM</span></p>
<p class="affiliation">Department of Therapeutic Radiology, Yale University School of Medicine, New Haven, Connecticut 06520-8040, USA.</p><p class="abstract">The DNA mismatch repair (MMR) system in mammalian cells not only serves to correct base mispairs and other replication errors, but it also influences the cellular response to certain forms of DNA damage. Cells that are deficient in MMR are relatively resistant to alkylation damage because, in wild-type cells, the MMR system is thought to promote toxicity via futile repair of alkylated mispairs. Conversely, MMR-deficient cells are sensitive to UV light, possibly due to the requirement for MMR factors in transcription-coupled repair of active genes. MMR deficiency has been associated with familial and sporadic carcinomas of the colon and other sites, and so, we sought to determine the influence of MMR status on cellular response to ionizing radiation, an agent commonly used for cancer therapy. Fibroblast cell lines were established from transgenic mice carrying targeted disruptions of one of three MMR genes in mammalian cells: Pms2, Mlh1, or Msh2. In comparison to wild-type cell lines from related mice, the Pms2-, Mlh1-, or Msh2-nullizygous cell lines were found to exhibit higher levels of clonogenic survival following exposure to ionizing radiation. Because ionizing radiation generates a variety of lesions in DNA, the differences in survival may reflect a role for MMR in processing a subset of these lesions, such as damaged bases. These results both identify a new class of DNA-damaging agents whose effects are modulated by the MMR system and may help to elucidate pathways of radiation response in cancer cells.</p>
<p class="grants">Funded by: NIEHS NIH HHS: <span class="grant">ES05775</span>; NIGMS NIH HHS: <span class="grant">GM32741</span>, <span class="grant">GM45413</span></p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">1997</span>;<span class="volume">57</span>;<span class="number">22</span>;<span class="pagerange">5143-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9371516" rel="external">9371516</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assignment of the human stress-activated protein kinase-3 gene (SAPK3) to chromosome 22q13.3 by fluorescence in situ hybridization.</h4>
<p class="authors"><span class="author">Goedert M</span>, <span class="author">Hasegawa J</span>, <span class="author">Craxton M</span>, <span class="author">Leversha MA</span> and <span class="author">Clegg S</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Cambridge, United Kingdom.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1997</span>;<span class="volume">41</span>;<span class="number">3</span>;<span class="pagerange">501-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9169156" rel="external">9169156</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1997.4633" rel="external">10.1006/geno.1997.4633</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome mapping by fluorescent fingerprinting.</h4>
<p class="authors"><span class="author">Gregory SG</span>, <span class="author">Howell GR</span> and <span class="author">Bentley DR</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%73%67%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The construction of sequence-ready maps of overlapping genomic clones is central to large-scale genome sequencing. We have implemented a method for fluorescent fingerprinting of bacterial clones to assemble contig maps. The method utilizes three spectrally distinct fluorescently tagged dideoxy ATPs to specifically label the HindIII termini in HindIII and Sau3AI restriction digests of clones that are multiplexed prior to electrophoresis and data collection. There is excellent reproducibility of raw data, improved resolution of large fragments, and concordance between the results obtained using this and the equivalent radioactive protocol. This method also allows detection of smaller overlaps between clones when compared to the analysis of restriction digests on nondenaturing agarose gels.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1997</span>;<span class="volume">7</span>;<span class="number">12</span>;<span class="pagerange">1162-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9414321" rel="external">9414321</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310676" rel="external">310676</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the mouse beta-prime adaptin gene; cDNA sequence, genomic structure, and chromosomal localization.</h4>
<p class="authors"><span class="author">Guilbaud C</span>, <span class="author">Peyrard M</span>, <span class="author">Fransson I</span>, <span class="author">Clifton SW</span>, <span class="author">Roe BA</span>, <span class="author">Carter NP</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Molecular Medicine, Karolinska Hospital, CMM-building L8:00, S-171 76 Stockholm, Sweden.</p><p class="abstract">Adaptins are important subunits of heterotetrameric complexes called adaptors, which participate in the clathrin-coated, vesicle-mediated endocytosis and intracellular receptor transport. The gene family of adaptins is divided into three classes, alpha, beta, and gamma, with further subdivision into beta- and beta-prime components. Two beta-prime adaptins, the rat AP105a and the human BAM22, have previously been characterized. The BAM22 gene is located on human Chromosome (Chr) 22q12 and can be considered a candidate meningioma tumor suppressor gene. We report here the characterization of the mouse ortholog of the BAM22 gene, and we suggest the name adtb1 for the mouse gene. Like the BAM22 gene, the adtb1 transcript is highly and ubiquitously expressed. We provide 3885-bp cDNA sequence, which entirely covers the open reading frame of the adtb1, capable of encoding a protein of 943 amino acids. The adtb1 protein is highly conserved (&gt;96% identity) when compared with AP105a and BAM22 proteins. We also report the genomic organization of adtb1, which is similar to the BAM22 gene. The adtb1 gene has been assigned to mouse Chr 11, band 11A2, which confirms the synteny between human Chr 22q12 and mouse Chr 11.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">1997</span>;<span class="volume">8</span>;<span class="number">9</span>;<span class="pagerange">651-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9271666" rel="external">9271666</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New horizons in sequence analysis.</h4>
<p class="authors"><span class="author">Hubbard TJ</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%74%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">An ever increasing number of protein sequences are being compared, partly because of the availability of full sets of protein sequences from several completed genome-sequencing projects. The resulting problem of scale has shifted the emphasis of sequence analysis method development from sensitivity and flexibility, which relies on manual intervention and interpretation, to the automatic generation of results of known reliability.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Current opinion in structural biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">7</span>;<span class="number">2</span>;<span class="pagerange">190-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9094334">9094334</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The nucleotide sequence of Saccharomyces cerevisiae chromosome IV.</h4>
<p class="authors"><span class="author">Jacq C</span>, <span class="author">Alt-M&ouml;rbe J</span>, <span class="author">Andre B</span>, <span class="author">Arnold W</span>, <span class="author">Bahr A</span>, <span class="author">Ballesta JP</span>, <span class="author">Bargues M</span>, <span class="author">Baron L</span>, <span class="author">Becker A</span>, <span class="author">Biteau N</span>, <span class="author">Bl&ouml;cker H</span>, <span class="author">Blugeon C</span>, <span class="author">Boskovic J</span>, <span class="author">Brandt P</span>, <span class="author">Br&uuml;ckner M</span>, <span class="author">Buitrago MJ</span>, <span class="author">Coster F</span>, <span class="author">Delaveau T</span>, <span class="author">del Rey F</span>, <span class="author">Dujon B</span>, <span class="author">Eide LG</span>, <span class="author">Garcia-Cantalejo JM</span>, <span class="author">Goffeau A</span>, <span class="author">Gomez-Peris A</span>, <span class="author">Zaccaria P</span> <em>et al.</em></p>
<p class="affiliation">Laboratoire de G&eacute;n&eacute;tique Mol&eacute;culaire, URA 1302 du CNRS, Ecole Normale Sup&eacute;rieure, Paris, France. <a href="mailto:%6A%61%63%71%40%62%69%6F%6C%6F%67%69%65.%65%6E%73.%66%72">&#106;&#97;&#99;&#113;&#64;&#98;&#105;&#111;&#108;&#111;&#103;&#105;&#101;&#46;&#101;&#110;&#115;&#46;&#102;&#114;</a></p><p class="abstract">The complete DNA sequence of the yeast Saccharomyces cerevisiae chromosome IV has been determined. Apart from chromosome XII, which contains the 1-2 Mb rDNA cluster, chromosome IV is the longest S. cerevisiae chromosome. It was split into three parts, which were sequenced by a consortium from the European Community, the Sanger Centre, and groups from St Louis and Stanford in the United States. The sequence of 1,531,974 base pairs contains 796 predicted or known genes, 318 (39.9%) of which have been previously identified. Of the 478 new genes, 225 (28.3%) are homologous to previously identified genes and 253 (32%) have unknown functions or correspond to spurious open reading frames (ORFs). On average there is one gene approximately every two kilobases. Superimposed on alternating regional variations in G+C composition, there is a large central domain with a lower G+C content that contains all the yeast transposon (Ty) elements and most of the tRNA genes. Chromosome IV shares with chromosomes II, V, XII, XIII and XV some long clustered duplications which partly explain its origin.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1997</span>;<span class="volume">387</span>;<span class="number">6632 Suppl</span>;<span class="pagerange">75-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9169867">9169867</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two methods for improving performance of an HMM and their application for gene finding.</h4>
<p class="authors"><span class="author">Krogh A</span></p>
<p class="affiliation">Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark. <a href="mailto:%6B%72%6F%67%68%40%63%62%73.%64%74%75.%64%6B">&#107;&#114;&#111;&#103;&#104;&#64;&#99;&#98;&#115;&#46;&#100;&#116;&#117;&#46;&#100;&#107;</a></p><p class="abstract">A hidden Markov model for gene finding consists of submodels for coding regions, splice sites, introns, intergenic regions and possibly more. It is described how to estimate the model as a whole from labeled sequences instead of estimating the individual parts independently from subsequences. It is argued that the standard maximum likelihood estimation criterion is not optimal for training such a model. Instead of maximizing the probability of the DNA sequence, one should maximize the probability of the correct prediction. Such a criterion, called conditional maximum likelihood, is used for the gene finder &#39;HMM-gene&#39;. A new (approximative) algorithm is described, which finds the most probable prediction summed over all paths yielding the same prediction. We show that these methods contribute significantly to the high performance of HMMgene.</p>
<p class="publication"><span class="title">Proceedings / ... International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">5</span>;<span class="pagerange">179-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9322033" rel="external">9322033</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Antidiabetic sulphonylureas stimulate acetylcholine release from striatal cholinergic interneurones through inhibition of K(ATP) channel activity.</h4>
<p class="authors"><span class="author">Lee K</span>, <span class="author">Brownhill V</span> and <span class="author">Richardson PJ</span></p>
<p class="affiliation">Parke Davis Neuroscience Research Centre, Cambridge University Forvie Site, England.</p><p class="abstract">The sulphonylureas tolbutamide and glibenclamide were shown to stimulate acetylcholine release from rat striatal slices. To determine the mechanism of this effect, whole-cell patch-clamp recordings were made from large neurones within the striatum that displayed morphological, electrophysiological, and pharmacological characteristics typical of cholinergic interneurones. Dialysis of these neurones with a pipette solution containing low concentrations of ATP produced a gradual hyperpolarisation that could be reversed by bath application of the sulphonylureas. In voltage-clamp studies, these compounds were shown to act through the inhibition of a potassium conductance. It is concluded that cholinergic interneurones within the rat striatum express sulphonylurea-sensitive ATP-sensitive potassium channel activity. These channels are probably cytoprotective and may prove to be novel sites of therapeutic modulation.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of neurochemistry</span>&nbsp;<span class="year">1997</span>;<span class="volume">69</span>;<span class="number">4</span>;<span class="pagerange">1774-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9326309" rel="external">9326309</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Instability of highly expanded CAG repeats in mice transgenic for the Huntington&#39;s disease mutation.</h4>
<p class="authors"><span class="author">Mangiarini L</span>, <span class="author">Sathasivam K</span>, <span class="author">Mahal A</span>, <span class="author">Mott R</span>, <span class="author">Seller M</span> and <span class="author">Bates GP</span></p>
<p class="affiliation">Division of Medical and Molecular Genetics, Guy&#39;s Hospital, London, UK.</p><p class="abstract">Six inherited neurodegenerative diseases are caused by a CAG/polyglutamine expansion, including spinal and bulbar muscular atrophy (SBMA), Huntington&#39;s disease (HD), spinocerebellar ataxia type 1 (SCA1), dentatorubral pallidoluysian atrophy (DRPLA) Machado-Joseph disease (MJD or SCA3) and SCA2. Normal and expanded HD allele sizes of 6-39 and 35-121 repeats have been reported, and the allele distributions for the other diseases are comparable. Intergenerational instability has been described in all cases, and repeats tend to be more unstable on paternal transmission. This may present as larger increases on paternal inheritance as in HD, or as a tendency to increase on male and decrease on female transmission as in SCA1 (ref. 15). Somatic repeat instability is also apparent and appears most pronounced in the CNS. The major exception is the cerebellum, which in HD, DRPLA, SCA1 and MJD has a smaller repeat relative to the other brain regions tested. Of non-CNS tissues, instability was observed in blood, liver, kidney and colon. A mouse model of CAG repeat instability would be helpful in unravelling its molecular basis although an absence of CAG repeat instability in transgenic mice has so far been reported. These studies include (CAG) in the androgen receptor cDNA, (CAG) in the HD cDNA, (CAG) in the SCA1 cDNA, (CAG) in the SCA3 cDNA and as an isolated (CAG) tract.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">15</span>;<span class="number">2</span>;<span class="pagerange">197-200</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9020849">9020849</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0297-197">10.1038/ng0297-197</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression and development databases for C. elegans</h4>
<p class="authors"><span class="author">Martinelli SD</span>, <span class="author">Brown CG</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK</p><p class="abstract">The nematode worm C. elegans, with its transparent body, is an excellent vehicle for studying developmental gene expression during embryogenesis and throughout its short life. Expression data from in-situ hybridization, immunolocalization and reporter constructs have been put into the ACeDB database, which is used to store and disseminate most types of C. elegans data, and is also widely used for genome-sequencing projects. In the database, the gene-expression patterns are linked to genes, sequences, cells, organs and the developmental stage in which expression occurs. An accessory program &#39;Angler&#39; can be used to browse sectional Nomarski images of the worm embryo during early development, and to relate these images to overlaid cell lineage data and 3-D schematic views of cell positions.Copyright 1997 Academic Press Limited Copyright 1997Academic Press Limited</p>
<p class="publication"><span class="title">Seminars in cell &amp; developmental biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">459-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9441951" rel="external">9441951</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fscdb.1997.0171" rel="external">10.1006/scdb.1997.0171</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cytogenetic analysis of three breast carcinoma cell lines using reverse chromosome painting.</h4>
<p class="authors"><span class="author">Morris JS</span>, <span class="author">Carter NP</span>, <span class="author">Ferguson-Smith MA</span> and <span class="author">Edwards PA</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, England.</p><p class="abstract">Chromosome painting was used to determine the copy number and identity of virtually all the chromosomes in three breast cancer cell lines, T-47D, MDA-MB-361, and ZR-75-1. The karyotypes of all three cell lines were very complex, and were consistent with the monosomic pattern of evolution suggested by Dutrillaux, in which nonreciprocal translocations cause an initial reduction in chromosome number, followed by duplication of the entire genome and further chromosome loss. Twenty distinct abnormal chromosomes were identified in T-47D, seven of which were present as two copies. MDA-MB-361 had 27 abnormal chromosomes each as a single copy. Thirteen abnormal chromosomes in ZR-75-1 occurred singly, two were paired, and one was present as three copies. Most of the aberrant chromosomes were nonreciprocal translocations, although deletions, duplications, isochromosomes, and amplifications (HSR of 1q) were also found. Chromosome arms present in abnormal chromosomes in all three lines were 1q, 6p, 7p, 8p, 8q, 10q, 11p, 11q, 12p, 13q, 14q, 15q, 16p, 16q, 17q, and 20q. The only chromosome arms present in four or more copies in all three lines were 8q and proximal 12p, while 1p, 17p, and bands 11q12--13 were the only chromosome regions consistently reduced to two copies. The most striking feature common to all three lines was a translocation breakpoint on the short arm of chromosome 8 at 8p12.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">1997</span>;<span class="volume">20</span>;<span class="number">2</span>;<span class="pagerange">120-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9331563" rel="external">9331563</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EST_GENOME: a program to align spliced DNA sequences to unspliced genomic DNA.</h4>
<p class="authors"><span class="author">Mott R</span></p>
<p class="affiliation">Informatics Group, Sanger Centre, Cambridge, UK.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1997</span>;<span class="volume">13</span>;<span class="number">4</span>;<span class="pagerange">477-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9283765" rel="external">9283765</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical assessment of methods of protein structure prediction (CASP): round II.</h4>
<p class="authors"><span class="author">Moult J</span>, <span class="author">Hubbard T</span>, <span class="author">Bryant SH</span>, <span class="author">Fidelis K</span> and <span class="author">Pedersen JT</span></p>
<p class="affiliation">Center for Advanced Research in Biotechnology, University of Maryland Biotechnology Institute, Rockville 20850, USA. <a href="mailto:%6A%6D%6F%75%6C%74%40%63%61%72%62.%6E%69%73%74.%67%6F%76">&#106;&#109;&#111;&#117;&#108;&#116;&#64;&#99;&#97;&#114;&#98;&#46;&#110;&#105;&#115;&#116;&#46;&#103;&#111;&#118;</a></p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">1997</span>;<span class="volume">Suppl 1</span>;<span class="pagerange">2-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9485489">9485489</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From long range mapping to sequence-ready contigs on human chromosome 6.</h4>
<p class="authors"><span class="author">Mungall AJ</span>, <span class="author">Humphray SJ</span>, <span class="author">Ranby SA</span>, <span class="author">Edwards CA</span>, <span class="author">Heathcott RW</span>, <span class="author">Clee CM</span>, <span class="author">Holloway E</span>, <span class="author">Peck AI</span>, <span class="author">Harrison P</span>, <span class="author">Green LD</span>, <span class="author">Butler AP</span>, <span class="author">Langford CF</span>, <span class="author">William RG</span>, <span class="author">Huckle EJ</span>, <span class="author">Baron L</span>, <span class="author">Smith A</span>, <span class="author">Leversha MA</span>, <span class="author">Ramsey YH</span>, <span class="author">Clegg SM</span>, <span class="author">Rice CM</span>, <span class="author">Maslen GL</span>, <span class="author">Hunt SE</span>, <span class="author">Scott CE</span>, <span class="author">Soderlund CA</span>, <span class="author">Dunham I</span> <em>et al.</em></p>
<p class="affiliation">The Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">Our aim is to construct physical clone maps covering those regions of chromosome 6 that are not currently extensively mapped, and use these to determine the DNA sequence of the whole chromosome. The strategy we are following involves establishing a high density framework map of the order of 15 markers per Megabase using radiation hybrid (RH) mapping. The markers are then used to identify large-insert genomic bacterial clones covering the chromosome, which are assembled into sequence-ready contigs by restriction enzyme fingerprinting and sequence tagged site (STS) content analysis. Contig gap closure is performed by walking experiments using STSs developed from the end sequences of the clone inserts.</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1997</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">151-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10668960" rel="external">10668960</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Trisomy 5q12--&gt;q13.3 in a patient with add(13q): characterization of an interchromosomal insertion by forward and reverse chromosome painting.</h4>
<p class="authors"><span class="author">Nordgren A</span>, <span class="author">Arver S</span>, <span class="author">Kvist U</span>, <span class="author">Carter N</span> and <span class="author">Blennow E</span></p>
<p class="affiliation">Department of Molecular Medicine, Karolinska Institutet, Stockholm, Sweden.</p><p class="abstract">We report on a patient with azoospermia, mild mental retardation, and minor physical anomalies. Chromosome analysis demonstrated the presence of additional material on the long arm of one chromosome 13. Forward chromosome painting using chromosome-specific libraries showed an insertion of material from chromosome 5. Further characterization with flow sorting of the aberrant chromosome and amplification by DOP-PCR followed by reverse chromosome painting showed specific trisomy of 5q12--&gt;q13.3.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of medical genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">73</span>;<span class="number">3</span>;<span class="pagerange">351-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9415698" rel="external">9415698</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Intermediate sequences increase the detection of homology between sequences.</h4>
<p class="authors"><span class="author">Park J</span>, <span class="author">Teichmann SA</span>, <span class="author">Hubbard T</span> and <span class="author">Chothia C</span></p>
<p class="affiliation">Cambridge Centre for Protein Engineering, UK.</p><p class="abstract">Two homologous sequences, which have diverged beyond the point where their homology can be recognised by a simple direct comparison, can be related through a third sequence that is suitably intermediate between the two. High scores, for a sequence match between the first and third sequences and between the second and the third sequences, imply that the first and second sequences are related even though their own match score is low. We have tested the usefulness of this idea using a database that contains the sequences of 971 protein domains whose structures are known and whose residue identities with each other are some 40% or less (PDB40D). On the basis of sequence and structural information, 2143 pairs of these sequences are known to have an evolutionary relationship. FASTA, in an all-against-all comparison of the sequences in the database, detected 320 (15%) of these relationships as well as three false positive (i.e. 1% error rate). Using intermediate sequences found by FASTA matches of PDB40D sequences to those in the large non-redundant OWL database we could detect 550 evolutionary relationships with an error rate of 1%. This means the intermediate sequence procedure increases the ability to recognise the evolutionary relationships amongst the PDB40D sequences by 70%.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1997</span>;<span class="volume">273</span>;<span class="number">1</span>;<span class="pagerange">349-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9367767">9367767</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1997.1288">10.1006/jmbi.1997.1288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tc7, a Tc1-hitch hiking transposon in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Rezsohazy R</span>, <span class="author">van Luenen HG</span>, <span class="author">Durbin RM</span> and <span class="author">Plasterk RH</span></p>
<p class="affiliation">Division of Molecular Biology, The Netherlands Cancer Institute, Plesmanlaan 121, 1066 CX Amsterdam, The Netherlands.</p><p class="abstract">We have found a novel transposon in the genome of Caenorhabditis elegans. Tc7 is a 921 bp element, made up of two 345 bp inverted repeats separated by a unique, internal sequence. Tc7 does not contain an open reading frame. The outer 38 bp of the inverted repeat show 36 matches with the outer 38 bp of Tc1. This region of Tc1 contains the Tc1-transposase binding site. Furthermore, Tc7 is flanked by TA dinucleotides, just like Tc1, which presumably correspond to the target duplication generated upon integration. Since Tc7 does not encode its own transposase but contains the Tc1-transposase binding site at its extremities, we tested the ability of Tc7 to jump upon forced expression of Tc1 transposase in somatic cells. Under these conditions Tc7 jumps at a frequency similar to Tc1. The target site choice of Tc7 is identical to that of Tc1. These data suggest that Tc7 shares with Tc1 all the sequences minimally required to parasitize upon the Tc1 transposition machinery. The genomic distribution of Tc7 shows a striking clustering on the X chromosome where two thirds of the elements (20 out of 33) are located. Related transposons in C. elegans do not show this asymmetric distribution.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">5 R01 RR10082-02</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1997</span>;<span class="volume">25</span>;<span class="number">20</span>;<span class="pagerange">4048-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9321656" rel="external">9321656</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC147001" rel="external">147001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative analysis of the polycystic kidney disease 1 (PKD1) gene reveals an integral membrane glycoprotein with multiple evolutionary conserved domains.</h4>
<p class="authors"><span class="author">Sandford R</span>, <span class="author">Sgotto B</span>, <span class="author">Aparicio S</span>, <span class="author">Brenner S</span>, <span class="author">Vaudin M</span>, <span class="author">Wilson RK</span>, <span class="author">Chissoe S</span>, <span class="author">Pepin K</span>, <span class="author">Bateman A</span>, <span class="author">Chothia C</span>, <span class="author">Hughes J</span> and <span class="author">Harris P</span></p>
<p class="affiliation">Department of Medicine, Addenbrooke&#39;s Hospital, Cambridge, UK. <a href="mailto:%72%73%61%6E%64%66%6F%72%40%6D%65%64.%63%61%6D.%61%63.%75%6B">&#114;&#115;&#97;&#110;&#100;&#102;&#111;&#114;&#64;&#109;&#101;&#100;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">PKD1 is the major locus of the common genetic disorder autosomal dominant polycystic kidney disease (ADPKD). Analysis of the predicted protein sequence of the human PKD1 gene, polycystin, shows a large molecule with a unique arrangement of extracellular domains and multiple putative transmembrane regions. The precise function of polycystin remains unclear with a paucity of mutations to define key structural and functional domains. To refine the structure of this protein we have cloned the genomic region encoding the Fugu PKD1 gene. Fugu PKD1 spans 36 kb of genomic DNA and has greater complexity with 54 exons compared with 46 in man. Comparative analysis of the predicted protein sequences shows a lower level of homology than in similar studies with identity of 40 and 59% similarity. However key structural motifs including leucine rich repeats (LRR), a C-type lectin and LDL-A like domains and 16 PKD repeats are maintained. A region of homology with the sea urchin REJ protein was also confirmed in Fugu but found to extend over 1000 amino acids. Several highly conserved intra- and extra-cellular regions, with no known sequence homologies, that are likely to be of functional importance were detected. The likely structure of the membrane associated region has been refined with similarity to the PKD2 protein and voltage gated Ca2+ and Na+ channels highlighted over part of this area. The overall protein structure has therefore been clarified and this comparative analysis derived structure will form the basis for the functional study of polycystin and its individual domains.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">6</span>;<span class="number">9</span>;<span class="pagerange">1483-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9285785" rel="external">9285785</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regions of human chromosome 2 (2q32-q35) and mouse chromosome 1 show synteny with the pufferfish genome (Fugu rubripes).</h4>
<p class="authors"><span class="author">Schofield JP</span>, <span class="author">Elgar G</span>, <span class="author">Greystrong J</span>, <span class="author">Lye G</span>, <span class="author">Deadman R</span>, <span class="author">Micklem G</span>, <span class="author">King A</span>, <span class="author">Brenner S</span> and <span class="author">Vaudin M</span></p>
<p class="affiliation">Department of Medicine, University of Cambridge, Addenbrooke&#39;s Hospital, United Kingdom. <a href="mailto:%6A%70%73%31%40%6D%6F%6C%65.%62%69%6F.%63%61%6D.%61%63.%75%6B">&#106;&#112;&#115;&#49;&#64;&#109;&#111;&#108;&#101;&#46;&#98;&#105;&#111;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have isolated and sequenced a cosmid clone from the compact genome of the Japanese pufferfish (Fugu rubripes) containing portions of three genes that have the same order as in human. The gene order is microtubule-associated protein (MAP-2), myosin light chain (MYL-1), and carbamoyl phosphate synthetase (CPS III). The intron-exon organization of Fugu CPS III is identical with that of rat CPS I, although the equivalent genomic fragments of rat and Fugu CPS span 87.9 and 21 kb, respectively. This is the first report of a piscine CPS III genomic structure and predicts a close evolutionary link between CPS III and CPS I. The 8-kb intergenic region between MYL-1 and CPS gave no clear areas of transcription factor-binding sites by pairwise comparison with shark or rat CPS promoter regions. However, there was a match with the rat myosin light chain 2 (MLC-2) gene promoter and a MyoD transcription factor-binding site 874 bp upstream of the MYL-1 gene.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1997</span>;<span class="volume">45</span>;<span class="number">1</span>;<span class="pagerange">158-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9339372" rel="external">9339372</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1997.4913" rel="external">10.1006/geno.1997.4913</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FPC: a system for building contigs from restriction fingerprinted clones.</h4>
<p class="authors"><span class="author">Soderlund C</span>, <span class="author">Longden I</span> and <span class="author">Mott R</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK. <a href="mailto:%63%61%72%69%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#114;&#105;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">MOTIVATION: To meet the demands of large-scale sequencing, thousands of clones must be fingerprinted and assembled into contigs. To determine the order of clones, a typical experiment is to digest the clones with one or more restriction enzymes and measure the resulting fragments. The probability of two clones overlapping is based on the similarity of their fragments. A contig contains two or more overlapping clones and a minimal tiling path of clones is selected to be sequenced. Interactive software with algorithmic support is necessary to assemble the clones into contigs quickly. RESULTS: FPC (fingerprinted contigs) is an interactive program for building contigs from restriction fingerprinted clones. FPC uses an algorithm to cluster clones into contigs based on their probability of coincidence score. For each contig, it builds a consensus band (CB) map which is similar to a restriction map; but it does not try to resolve all the errors. The CB map is used to assign coordinates to the clones based on their alignment to the map and to provide a detailed visualization of the clone overlap. FPC has editing facilities for the user to refine the coordinates and to remove poorly fingerprinted clones. Functions are available for updating an FPC database with new clones. Contigs can easily be merged, split or deleted. Markers can be added to clones and are displayed with the appropriate contig. Sequence-ready clones can be selected and their sequencing status displayed. As such, FPC is an integrated program for the assembly of sequence-ready clones for large-scale sequencing projects.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1997</span>;<span class="volume">13</span>;<span class="number">5</span>;<span class="pagerange">523-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9367125" rel="external">9367125</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of protein domain families in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Sonnhammer EL</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Cambridge, United Kingdom.</p><p class="abstract">The Caenorhabditis elegans genome sequencing project has completed over half of this nematode&#39;s 100-Mb genome. Proteins predicted in the finished sequence have been compiled and released in the data-base Wormpep. Presented here is a comprehensive analysis of protein domain families in Wormpep 11, which comprises 7299 proteins. The relative abundance of common protein domain families was counted by comparing all Wormpep proteins to the Pfam collection of protein families, which is based on recognition by hidden Markov models. This analysis also identified a number of previously unannotated domains. To investigate new apparently nematode-specific protein families, Wormpep was clustered into domain families on the basis of sequence similarity using the Domainer program. The largest clusters that lacked clear homology to proteins outside Nematoda were analyzed in further detail, after which some could be assigned a putative function. We compared all proteins in Wormpep 11 to proteins in the human, Saccharomyces cerevisiae, and Haemophilus influenzae genomes. Among the results are the estimation that over two-thirds of the currently known human proteins are likely to have a homologue in the whole C. elegans genome and that a significant number of proteins are well conserved between C. elegans and H. influenzae, that are not found in S. cerevisiae.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1997</span>;<span class="volume">46</span>;<span class="number">2</span>;<span class="pagerange">200-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9417907" rel="external">9417907</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1997.4989" rel="external">10.1006/geno.1997.4989</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pfam: a comprehensive database of protein domain families based on seed alignments.</h4>
<p class="authors"><span class="author">Sonnhammer EL</span>, <span class="author">Eddy SR</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">Databases of multiple sequence alignments are a valuable aid to protein sequence classification and analysis. One of the main challenges when constructing such a database is to simultaneously satisfy the conflicting demands of completeness on the one hand and quality of alignment and domain definitions on the other. The latter properties are best dealt with by manual approaches, whereas completeness in practice is only amenable to automatic methods. Herein we present a database based on hidden Markov model profiles (HMMs), which combines high quality and completeness. Our database, Pfam, consists of parts A and B. Pfam-A is curated and contains well-characterized protein domain families with high quality alignments, which are maintained by using manually checked seed alignments and HMMs to find and align all members. Pfam-B contains sequence families that were generated automatically by applying the Domainer algorithm to cluster and align the remaining protein sequences after removal of Pfam-A domains. By using Pfam, a large number of previously unannotated proteins from the Caenorhabditis elegans genome project were classified. We have also identified many novel family memberships in known proteins, including new kazal, Fibronectin type III, and response regulator receiver domains. Pfam-A families have permanent accession numbers and form a library of HMMs available for searching and automatic annotation of new protein sequences.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG01363</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">1997</span>;<span class="volume">28</span>;<span class="number">3</span>;<span class="pagerange">405-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9223186" rel="external">9223186</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Chromosome 6 database at the Sanger Centre.</h4>
<p class="authors"><span class="author">Theaker AJ</span>, <span class="author">Maslen GL</span>, <span class="author">Scott CE</span>, <span class="author">Rice CM</span>, <span class="author">Hunt SE</span>, <span class="author">King A</span>, <span class="author">Mungall AJ</span>, <span class="author">Dunham I</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">The Sanger Centre Chromosome 6 Database (6ace) has been developed as the primary means of release of annotated sequencing and mapping information for human chromosome 6 from the Sanger Centre. It is also being used to curate global data from published and unpublished external sources. The rationale behind the development of 6ace is described, together with information as to how to access the database.</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1997</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">167-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10668963" rel="external">10668963</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal localization, gene structure and transcription pattern of the ORFX gene, a homologue of the MHC-linked RING3 gene.</h4>
<p class="authors"><span class="author">Thorpe KL</span>, <span class="author">Gorman P</span>, <span class="author">Thomas C</span>, <span class="author">Sheer D</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">We have mapped the human ORFX gene to chromosome 9q34 and determined its complete gene structure. Comparison with RING3, the human MHC-linked homologue on 6p21.3, shows the two gene structures to be highly conserved but with an approximate threefold expansion in the ORFX introns. RING3 and ORFX are found to be ubiquitously expressed in human adult and foetal tissues. Evidence suggests that the two genes may have arisen from an ancient duplication in a common ancestral chromosome.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A3585</span></p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">1997</span>;<span class="volume">200</span>;<span class="number">1-2</span>;<span class="pagerange">177-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9373153" rel="external">9373153</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Co-ordinate regulation of the cystic fibrosis and multidrug resistance genes in cystic fibrosis knockout mice.</h4>
<p class="authors"><span class="author">Trezise AE</span>, <span class="author">Ratcliff R</span>, <span class="author">Hawkins TE</span>, <span class="author">Evans MJ</span>, <span class="author">Freeman TC</span>, <span class="author">Romano PR</span>, <span class="author">Higgins CF</span> and <span class="author">Colledge WH</span></p>
<p class="affiliation">Nuffield Department of Clinical Biochemistry, University of Oxford, John Radcliffe Hospital, UK.</p><p class="abstract">The cystic fibrosis (Cftr and multidrug resistance (Mdr1) genes encode structurally similar proteins which are members of the ABC transporter superfamily. These genes exhibit complementary patterns of expression in vivo, suggesting that the regulation of their expression may be co-ordinated. We have tested this hypothesis in vivo by examining Cftr and Mdr1 expression in cystic fibrosis knockout transgenic mice (Cftr(tm1CAM)). Cftr mRNA expression in Cftr(tm1CAM)/Cftr(tm1CAM) mice was 4-fold reduced in the intestine, as compared with littermate wild-type mice. All other Cftr(tm1CAM)/Cftr(tm1CAM) mouse tissues examined showed similar reductions in Cftr expression. In contrast, we observed a 4-fold increase in Mdr1 mRNA expression in the intestines of neonatal and 3- to 4-week-old Cftr(tm1CAM)/Cftr(tm1CAM) mice, as compared with age-matched +/+ mice, and an intermediate level of Mdr1 mRNA in heterozygous Cftr(tm1CAM) mice. In 10-week-old, Cftr(tm1CAM)/Cftr(tm1CAM) mice and in contrast to the younger mice, Mdr1 mRNA expression was reduced, by 3-fold. The expression of two control genes, Pgk-1 and Mdr2, was similar in all genotypes, suggesting that the changes in Mdr1 mRNA levels observed in the Cftr(tm1CAM)/Cftr(tm1CAM) mice are specific to the loss of Cftr expression and/or function. These data provide further evidence supporting the hypothesis that the regulation Cftr and Mdr1 expression is co-ordinated in vivo, and that this co-ordinate regulation is influenced by temporal factors.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">6</span>;<span class="number">4</span>;<span class="pagerange">527-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9097955" rel="external">9097955</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Report and abstracts of the third international workshop on human chromosome 1 mapping 1997.</h4>
<p class="authors"><span class="author">Vance JM</span>, <span class="author">Matise TC</span>, <span class="author">Wooster R</span>, <span class="author">Schutte BC</span>, <span class="author">Bruns GA</span>, <span class="author">van Roy N</span>, <span class="author">Brodeur GM</span>, <span class="author">Tao YX</span>, <span class="author">Gregory S</span>, <span class="author">Weith A</span>, <span class="author">Vaudin M</span> and <span class="author">White P</span></p>
<p class="affiliation">Department of Medicine, Duke University Medical Center, Durham NC 27710, USA. <a href="mailto:%6A%65%66%66%40%64%6E%61%64%6F%63.%6D%63.%64%75%6B%65.%65%64%75">&#106;&#101;&#102;&#102;&#64;&#100;&#110;&#97;&#100;&#111;&#99;&#46;&#109;&#99;&#46;&#100;&#117;&#107;&#101;&#46;&#101;&#100;&#117;</a></p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">1-R13-HGO1350-01</span>; PHS HHS: <span class="grant">P01-N5-26630</span></p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">1997</span>;<span class="volume">78</span>;<span class="number">3-4</span>;<span class="pagerange">154-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9465885" rel="external">9465885</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution physical map of the X-linked retinoschisis interval in Xp22.</h4>
<p class="authors"><span class="author">Walpole SM</span>, <span class="author">Nicolaou A</span>, <span class="author">Howell GR</span>, <span class="author">Whittaker A</span>, <span class="author">Bentley DR</span>, <span class="author">Ross MT</span>, <span class="author">Yates JR</span> and <span class="author">Trump D</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Addenbrooke&#39;s Hospital, United Kingdom.</p><p class="abstract">X-linked retinoschisis (RS) is the leading cause of macular degeneration in young males and has been mapped to Xp22 between DXS418 and DXS999. To facilitate identification of the RS gene, we have constructed a yeast artificial chromosome (YAC) contig across this region comprising 28 YACs and 32 sequence-tagged sites including seven novel end clone markers. To establish the definitive marker order, a PAC contig containing 50 clones was also constructed, and all clones were fingerprinted. The marker order is: Xpter-DXS1317-(AFM205yd12-DXS7175-DXS7992) -60N8-T7-DXS1195-DXS7993-DXS7174 -60N8-SP6-DXS418-DXS7994-DXS7995-DXS7996-+ ++HYAT2-25HA10R-HYAT1-DXS7997-DXS7998- DXS257-434E8R-3542R-DXS6762-DXS7999-DXS 6763-434E8L-DXS8000-DXS6760-DXS7176- DXS8001-DXS999-3176R-PHKA2-Xcen. A long-range restriction map was constructed, and the RS region is estimated to be 1300 kb, containing three putative CpG islands. An unstable region was identified between DXS6763 and 434E8L. These data will facilitate positional cloning of RS and other disease genes in Xp22.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1997</span>;<span class="volume">44</span>;<span class="number">3</span>;<span class="pagerange">300-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9325051" rel="external">9325051</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1997.4890" rel="external">10.1006/geno.1997.4890</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Numerical criteria for the evaluation of ab initio predictions of protein structure.</h4>
<p class="authors"><span class="author">Zemla A</span>, <span class="author">Venclovas C</span>, <span class="author">Reinhardt A</span>, <span class="author">Fidelis K</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Biology and Biotechnology Research Program, Lawrence Livermore National Laboratory, Livermore, CA 94551, USA.</p><p class="abstract">As part of the CASP2 protein structure prediction experiment, a set of numerical criteria were defined for the evaluation of &quot;ab initio&quot; predictions. The evaluation package comprises a series of electronic submission formats, a submission validator, evaluation software, and a series of scripts to summarize the results for the CASP2 meeting and for presentation via the World Wide Web (WWW). The evaluation package is accessible for use on new predictions via WWW so that results can be compared to those submitted to CASP2. With further input from the community, the evaluation criteria are expected to evolve into a comprehensive set of measures capturing the overall quality of a prediction as well as critical detail essential for further development of prediction methods. We discuss present measures, limitations of the current criteria, and possible improvements.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">1997</span>;<span class="volume">Suppl 1</span>;<span class="pagerange">140-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9485506">9485506</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/7fth8z10"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/7fth8z10" src="/qr/7fth8z10.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:55:34 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
