<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 1999 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 1999.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 1999</h1><p>Number of papers published in 1999: <strong>69</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The SIS domain: a phosphosugar-binding domain.</h4>
<p class="authors"><span class="author">Bateman A</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Cambridge, UK CB10 <a href="mailto:%31%53%41.%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#49;&#83;&#65;&#46;&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">1999</span>;<span class="volume">24</span>;<span class="number">3</span>;<span class="pagerange">94-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10203754" rel="external">10203754</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The PLAT domain: a new piece in the PKD1 puzzle.</h4>
<p class="authors"><span class="author">Bateman A</span> and <span class="author">Sandford R</span></p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">1999</span>;<span class="volume">9</span>;<span class="number">16</span>;<span class="pagerange">R588-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10469604" rel="external">10469604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pfam 3.1: 1313 multiple alignments and profile HMMs match the majority of proteins.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Birney E</span>, <span class="author">Durbin R</span>, <span class="author">Eddy SR</span>, <span class="author">Finn RD</span> and <span class="author">Sonnhammer EL</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Pfam is a collection of multiple alignments and profile hidden Markov models of protein domain families. Release 3.1 is a major update of the Pfam database and contains 1313 families which are available on the World Wide Web in Europe at http://www.sanger.ac.uk/Software/Pfam/ and http://www.cgr.ki.se/Pfam/, and in the US at http://pfam.wustl.edu/. Over 54% of proteins in SWISS-PROT-35 and SP-TrEMBL-5 match a Pfam family. The primary changes of Pfam since release 2.1 are that we now use the more advanced version 2 of the HMMER software, which is more sensitive and provides expectation values for matches, and that it now includes proteins from both SP-TrEMBL and SWISS-PROT.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1999</span>;<span class="volume">27</span>;<span class="number">1</span>;<span class="pagerange">260-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9847196">9847196</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC148151">148151</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence organisation of the class II region of the human MHC.</h4>
<p class="authors"><span class="author">Beck S</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Sanger Centre, Cambridge, UK. <a href="mailto:%62%65%63%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#101;&#99;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present the genomic organisation of the extended class II region of the human MHC. This initial sequence, which is nearing completion, spans about 1.2 Mbp and is at present a composite of more than one haplotype. The sequencing of single haplotypes is planned for the future. The current sequence encompasses all of the known class II genes at the DP, DO, DM, DQ and DR loci as well as the transporter associated with antigen processing (TAP)/low molecular weight protein (LMP) antigen processing genes and the Tapasin locus, at the extended centromeric end.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Immunological reviews</span>&nbsp;<span class="year">1999</span>;<span class="volume">167</span>;<span class="pagerange">201-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10319262" rel="external">10319262</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From genomics to epigenomics: a loftier view of life.</h4>
<p class="authors"><span class="author">Beck S</span>, <span class="author">Olek A</span> and <span class="author">Walter J</span></p>
<p class="affiliation">The Sanger Centre, Cambridge, UK. <a href="mailto:%62%65%63%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#101;&#99;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature biotechnology</span>&nbsp;<span class="year">1999</span>;<span class="volume">17</span>;<span class="number">12</span>;<span class="pagerange">1144</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10585673">10585673</a>; DOI: <a href="http://dx.doi.org/10.1038%2F70651">10.1038/70651</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">THE CAENORHABDITIS ELEGANS GENOME: A Guide in The Post Genomics Age.</h4>
<p class="authors"><span class="author">Bird DM</span>, <span class="author">Opperman CH</span>, <span class="author">Jones SJ</span> and <span class="author">Baillie DL</span></p>
<p class="affiliation">Plant Nematode Genetics Group, Department of Plant Pathology, North Carolina State University, Box 7616, Raleigh, North Carolina 27695; e-mail: <a href="mailto:%64%61%76%69%64%5F%62%69%72%64%40%6E%63%73%75.%65%64%75">&#100;&#97;&#118;&#105;&#100;&#95;&#98;&#105;&#114;&#100;&#64;&#110;&#99;&#115;&#117;&#46;&#101;&#100;&#117;</a>; <a href="mailto:%77%61%72%74%68%6F%67%40%75%6E%69%74%79.%6E%63%73%75.%65%64%75">&#119;&#97;&#114;&#116;&#104;&#111;&#103;&#64;&#117;&#110;&#105;&#116;&#121;&#46;&#110;&#99;&#115;&#117;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The completion of the entire genome sequence of the free-living nematode, Caenorhabditis elegans is a tremendous milestone in modern biology. Not only will scientists be poring over data mined from this resource, but techniques and methodologies developed along the way have changed the way we can approach biological questions. The completion of the C. elegans genomic sequence will be of particular importance to scientists working on parasitic nematodes. In many cases, these nematode species present intractable challenges to those interested in their biology and genetics. The data already compared from parasites to the C. elegans database reveals a wealth of opportunities for parasite biologists. It is likely that many of the same genes will be present in parasites and that these genes will have similar functions. Additional information regarding differences between free-living and parasitic species will provide insight into the evolution and nature of parasitism. Finally, genetic and genomic approaches to the study of parasitic nematodes now have a clearly marked path to follow.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Annual review of phytopathology</span>&nbsp;<span class="year">1999</span>;<span class="volume">37</span>;<span class="pagerange">247-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11701823" rel="external">11701823</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev.phyto.37.1.247" rel="external">10.1146/annurev.phyto.37.1.247</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complete nucleotide sequence of chromosome 3 of Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Bowman S</span>, <span class="author">Lawson D</span>, <span class="author">Basham D</span>, <span class="author">Brown D</span>, <span class="author">Chillingworth T</span>, <span class="author">Churcher CM</span>, <span class="author">Craig A</span>, <span class="author">Davies RM</span>, <span class="author">Devlin K</span>, <span class="author">Feltwell T</span>, <span class="author">Gentles S</span>, <span class="author">Gwilliam R</span>, <span class="author">Hamlin N</span>, <span class="author">Harris D</span>, <span class="author">Holroyd S</span>, <span class="author">Hornsby T</span>, <span class="author">Horrocks P</span>, <span class="author">Jagels K</span>, <span class="author">Jassal B</span>, <span class="author">Kyes S</span>, <span class="author">McLean J</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Murphy L</span>, <span class="author">Oliver K</span>, <span class="author">Quail MA</span>, <span class="author">Rajandream MA</span>, <span class="author">Rutter S</span>, <span class="author">Skelton J</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Sulston JE</span>, <span class="author">Whitehead S</span>, <span class="author">Woodward JR</span>, <span class="author">Newbold C</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Sanger Centre, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%73%68%61%72%65%6E%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#104;&#97;&#114;&#101;&#110;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Analysis of Plasmodium falciparum chromosome 3, and comparison with chromosome 2, highlights novel features of chromosome organization and gene structure. The sub-telomeric regions of chromosome 3 show a conserved order of features, including repetitive DNA sequences, members of multigene families involved in pathogenesis and antigenic variation, a number of conserved pseudogenes, and several genes of unknown function. A putative centromere has been identified that has a core region of about 2 kilobases with an extremely high (adenine + thymidine) composition and arrays of tandem repeats. We have predicted 215 protein-coding genes and two transfer RNA genes in the 1,060,106-base-pair chromosome sequence. The predicted protein-coding genes can be divided into three main classes: 52.6% are not spliced, 45.1% have a large exon with short additional 5&#39; or 3&#39; exons, and 2.3% have a multiple exon structure more typical of higher eukaryotes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1999</span>;<span class="volume">400</span>;<span class="number">6744</span>;<span class="pagerange">532-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10448855">10448855</a>; DOI: <a href="http://dx.doi.org/10.1038%2F22964">10.1038/22964</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DAPI banded karyotype of the domestic dog (Canis familiaris) generated using chromosome-specific paint probes.</h4>
<p class="authors"><span class="author">Breen M</span>, <span class="author">Bullerdiek J</span> and <span class="author">Langford CF</span></p>
<p class="affiliation">Animal Health Trust, Kentford, Suffolk, UK. <a href="mailto:%6D%74%62%72%65%65%6E%40%68%67%6D%70.%6D%72%63.%61%63.%75%6B">&#109;&#116;&#98;&#114;&#101;&#101;&#110;&#64;&#104;&#103;&#109;&#112;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The domestic dog (Canis familiaris) is widely used as a model in the study of human disease. However, many of the 78 chromosomes comprising the canine karyotype are extremely difficult to identify reliably by classical cytogenetics. This has been a major hindrance to molecular cytogenetic studies of this species. The Animal Health Trust and the Sanger Centre have developed a set of canine whole chromosome-specific fluorescence in situ hybridisation (FISH) probes (chromosome paints). We have used these chromosome paints to identify unequivocally each chromosome in a metaphase spread. An increasing number of laboratories are making use of cooled CCD cameras and sophisticated software for FISH mapping. Consequently, there is a major trend towards the use of DAPI banding for concurrent chromosome identification during FISH analyses in a range of species. Here we present, for the first time, a complete DAPI banded karyotype of the dog in which each chromosome has been accurately placed, together with a 460-band DAPI ideogram. These data will facilitate the accurate assignment of FISH-mapped loci to all chromosomes comprising the karyotype and form the basis for an agreed standard of the dog karyotype.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">1999</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">401-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10515215" rel="external">10515215</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">FISH mapping and identification of canine chromosomes.</h4>
<p class="authors"><span class="author">Breen M</span>, <span class="author">Langford CF</span>, <span class="author">Carter NP</span>, <span class="author">Holmes NG</span>, <span class="author">Dickens HF</span>, <span class="author">Thomas R</span>, <span class="author">Suter N</span>, <span class="author">Ryder EJ</span>, <span class="author">Pope M</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Preventive Medicine, Animal Health Trust, Newmarket, Suffolk, U.K. <a href="mailto:%6D%74%62%72%65%65%6E%40%68%67%6D%70.%6D%72%63.%61%63.%75%6B">&#109;&#116;&#98;&#114;&#101;&#101;&#110;&#64;&#104;&#103;&#109;&#112;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The karyotype of the domestic dog (Canis familiaris) is widely accepted as one of the most difficult mammalian karyotypes to work with. The dog has a total of 78 chromosomes; all 76 autosomes are acrocentric in morphology and show only a gradual decrease in length. Standardization of the canine karyotype has been performed in two stages. The first stage dealt only with chromosomes 1-21 which can be readily identified by conventional G-banding techniques. The remaining 17 autosomal pairs have proven to be very difficult to reliably identify by banding alone. To facilitate the identification of all canine chromosomes, chromosome-specific paint probes have been produced by DOP-PCR from flow-sorted dog chromosomes. Each paint probe has been used for FISH to identify the corresponding chromosome(s), allowing precise identification of all 78 canine chromosomes. The identification of the undesignated 17 autosomal pairs has been agreed upon by the standardization committee during the second stage of their role. Cosmid clones containing microsatellite markers may now be conclusively assigned to their chromosomal origin by simultaneous dual-color FISH with the corresponding paint probe. In this way a collection of chromosome-specific cosmid clones is being constructed, comprising at least one marker per chromosome, which will allow anchoring of existing and future linkage groups to the physical map.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of heredity</span>&nbsp;<span class="year">1999</span>;<span class="volume">90</span>;<span class="number">1</span>;<span class="pagerange">27-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9987898" rel="external">9987898</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reciprocal chromosome painting reveals detailed regions of conserved synteny between the karyotypes of the domestic dog (Canis familiaris) and human.</h4>
<p class="authors"><span class="author">Breen M</span>, <span class="author">Thomas R</span>, <span class="author">Binns MM</span>, <span class="author">Carter NP</span> and <span class="author">Langford CF</span></p>
<p class="affiliation">Centre for Preventive Medicine, Animal Health Trust, Lanwades Park, Kentford, CB8 7UU, United Kingdom. <a href="mailto:%6D%74%62%72%65%65%6E%40%68%67%6D%70.%6D%72%63.%61%63.%75%6B">&#109;&#116;&#98;&#114;&#101;&#101;&#110;&#64;&#104;&#103;&#109;&#112;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The domestic dog is increasingly being recognized as a useful model for human disease. The aim of this study was to conduct the first detailed whole-genome comparison of human and dog using bidirectional heterologous chromosome painting (reciprocal Zoo-FISH) analysis. We used whole-chromosome paint probes produced from degenerate oligonucleotide-primed PCR amplification of high-resolution bivariate flow-sorted human and dog chromosomes. No fewer than 68 evolutionarily conserved segments were identified between the dog and the human karyotypes. The use of elongated metaphase chromosomes for both species allowed the boundaries of each evolutionarily conserved segment to be determined to subband resolution. The distribution of conserved segments is discussed, as are the applications of these data in refining the current status of the dog genome map.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1999</span>;<span class="volume">61</span>;<span class="number">2</span>;<span class="pagerange">145-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10534400" rel="external">10534400</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1999.5947" rel="external">10.1006/geno.1999.5947</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Charting the genome.</h4>
<p class="authors"><span class="author">Brooksbank RA</span></p>
<p class="affiliation"><a href="mailto:%72%61%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#97;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Molecular medicine today</span>&nbsp;<span class="year">1999</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10088122" rel="external">10088122</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The structure of a PKD domain from polycystin-1: implications for polycystic kidney disease.</h4>
<p class="authors"><span class="author">Bycroft M</span>, <span class="author">Bateman A</span>, <span class="author">Clarke J</span>, <span class="author">Hamill SJ</span>, <span class="author">Sandford R</span>, <span class="author">Thomas RL</span> and <span class="author">Chothia C</span></p>
<p class="affiliation">MRC Centre for Protein Engineering, Lensfield Road, Cambridge CB2 1EW. <a href="mailto:%6D%62%31%30%30%33%31%40%63%75%73.%63%61%6D.%61%63.%75%6B">&#109;&#98;&#49;&#48;&#48;&#51;&#49;&#64;&#99;&#117;&#115;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Most cases of autosomal dominant polycystic kidney disease (ADPKD) are the result of mutations in the PKD1 gene. The PKD1 gene codes for a large cell-surface glycoprotein, polycystin-1, of unknown function, which, based on its predicted domain structure, may be involved in protein-protein and protein-carbohydrate interactions. Approximately 30% of polycystin-1 consists of 16 copies of a novel protein module called the PKD domain. Here we show that this domain has a beta-sandwich fold. Although this fold is common to a number of cell-surface modules, the PKD domain represents a distinct protein family. The tenth PKD domain of human and Fugu polycystin-1 show extensive conservation of surface residues suggesting that this region could be a ligand-binding site. This structure will allow the likely effects of missense mutations in a large part of the PKD1 gene to be determined.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">1999</span>;<span class="volume">18</span>;<span class="number">2</span>;<span class="pagerange">297-305</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9889186" rel="external">9889186</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1171124" rel="external">1171124</a>; DOI: <a href="http://dx.doi.org/10.1093%2Femboj%2F18.2.297" rel="external">10.1093/emboj/18.2.297</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The hairless gene of the mouse: relationship of phenotypic effects with expression profile and genotype.</h4>
<p class="authors"><span class="author">Cach&oacute;n-Gonz&aacute;lez MB</span>, <span class="author">San-Jos&eacute; I</span>, <span class="author">Cano A</span>, <span class="author">Vega JA</span>, <span class="author">Garc&iacute;a N</span>, <span class="author">Freeman T</span>, <span class="author">Schimmang T</span> and <span class="author">Stoye JP</span></p>
<p class="affiliation">National Institute for Medical Research, The Ridgeway, Mill Hill, London, United Kingdom. <a href="mailto:%6D%62%63%32%33%40%6D%6F%6C%65.%62%69%6F.%63%61%6D.%61%63.%75%6B">&#109;&#98;&#99;&#50;&#51;&#64;&#109;&#111;&#108;&#101;&#46;&#98;&#105;&#111;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Various mutations of the hairless (hr) gene of mice result in hair loss and other integument defects. To examine the role of the hr gene in mouse development, the expression profile of hr has been determined by in situ hybridisation and correlated to the nature of genetic changes and morphological abnormalities in different mutant animals. Four variant alleles have been characterised at the molecular level. hr/hr mice produce reduced, but significant, levels of hr mRNA whereas other alleles contain mutations which would be expected to preclude the synthesis of functional product, demonstrating a correlation between allelic variation at the hr locus and phenotypic severity. hr expression was shown to be widespread and temporally regulated. It was identified in novel tissues such as cartilage, developing tooth, inner ear, retina, and colon as well as in skin and brain. Analysis of mice homozygous for the rhino allele of hairless revealed that, although no morphological defects were detectable in many tissues normally expressing hr, previously undescribed abnormalities were present in several tissues including inner ear, retina, and colon. These findings indicate that the hairless gene product plays a wider role in development than previously suspected. Dev Dyn 1999;216:113-126.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Developmental dynamics : an official publication of the American Association of Anatomists</span>&nbsp;<span class="year">1999</span>;<span class="volume">216</span>;<span class="number">2</span>;<span class="pagerange">113-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10536052" rel="external">10536052</a>; DOI: <a href="http://dx.doi.org/10.1002%2F(SICI)1097-0177(199910)216%3A2%3C113%3A%3AAID-DVDY3%3E3.0.CO%3B2-M" rel="external">10.1002/(SICI)1097-0177(199910)216:2&lt;113::AID-DVDY3&gt;3.0.CO;2-M</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new member of the IL-1 receptor family highly expressed in hippocampus and involved in X-linked mental retardation.</h4>
<p class="authors"><span class="author">Carri&eacute; A</span>, <span class="author">Jun L</span>, <span class="author">Bienvenu T</span>, <span class="author">Vinet MC</span>, <span class="author">McDonell N</span>, <span class="author">Couvert P</span>, <span class="author">Zemni R</span>, <span class="author">Cardona A</span>, <span class="author">Van Buggenhout G</span>, <span class="author">Frints S</span>, <span class="author">Hamel B</span>, <span class="author">Moraine C</span>, <span class="author">Ropers HH</span>, <span class="author">Strom T</span>, <span class="author">Howell GR</span>, <span class="author">Whittaker A</span>, <span class="author">Ross MT</span>, <span class="author">Kahn A</span>, <span class="author">Fryns JP</span>, <span class="author">Beldjord C</span>, <span class="author">Marynen P</span> and <span class="author">Chelly J</span></p>
<p class="affiliation">INSERM Unit&eacute; 129-ICGM, CHU Cochin, 24 Rue du Faubourg Saint Jacques, 75014 Paris, France.</p><p class="abstract">We demonstrate here the importance of interleukin signalling pathways in cognitive function and the normal physiology of the CNS. Thorough investigation of an MRX critical region in Xp22.1-21.3 enabled us to identify a new gene expressed in brain that is responsible for a non-specific form of X-linked mental retardation. This gene encodes a 696 amino acid protein that has homology to IL-1 receptor accessory proteins. Non-overlapping deletions and a nonsense mutation in this gene were identified in patients with cognitive impairment only. Its high level of expression in post-natal brain structures involved in the hippocampal memory system suggests a specialized role for this new gene in the physiological processes underlying memory and learning abilities.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">23</span>;<span class="number">1</span>;<span class="pagerange">25-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10471494">10471494</a>; DOI: <a href="http://dx.doi.org/10.1038%2F12623">10.1038/12623</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic definition and sequence analysis of Arabidopsis centromeres.</h4>
<p class="authors"><span class="author">Copenhaver GP</span>, <span class="author">Nickel K</span>, <span class="author">Kuromori T</span>, <span class="author">Benito MI</span>, <span class="author">Kaul S</span>, <span class="author">Lin X</span>, <span class="author">Bevan M</span>, <span class="author">Murphy G</span>, <span class="author">Harris B</span>, <span class="author">Parnell LD</span>, <span class="author">McCombie WR</span>, <span class="author">Martienssen RA</span>, <span class="author">Marra M</span> and <span class="author">Preuss D</span></p>
<p class="affiliation">University of Chicago, Department of Molecular Genetics and Cell Biology, 1103 East 57 Street, Chicago, IL 60637, USA.</p><p class="abstract">High-precision genetic mapping was used to define the regions that contain centromere functions on each natural chromosome in Arabidopsis thaliana. These regions exhibited dramatic recombinational repression and contained complex DNA surrounding large arrays of 180-base pair repeats. Unexpectedly, the DNA within the centromeres was not merely structural but also encoded several expressed genes. The regions flanking the centromeres were densely populated by repetitive elements yet experienced normal levels of recombination. The genetically defined centromeres were well conserved among Arabidopsis ecotypes but displayed limited sequence homology between different chromosomes, excluding repetitive DNA. This investigation provides a platform for dissecting the role of individual sequences in centromeres in higher eukaryotes.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1999</span>;<span class="volume">286</span>;<span class="number">5449</span>;<span class="pagerange">2468-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10617454">10617454</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New collaborations make pharmacogenomics a SNP.</h4>
<p class="authors"><span class="author">Dawson E</span></p>
<p class="publication"><span class="title">Molecular medicine today</span>&nbsp;<span class="year">1999</span>;<span class="volume">5</span>;<span class="number">7</span>;<span class="pagerange">280</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10377514" rel="external">10377514</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SNP maps: more markers needed?</h4>
<p class="authors"><span class="author">Dawson E</span></p>
<p class="publication"><span class="title">Molecular medicine today</span>&nbsp;<span class="year">1999</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">419-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10498908" rel="external">10498908</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of cosmid-derived and chromosome-specific canine microsatellites.</h4>
<p class="authors"><span class="author">Dickens HF</span>, <span class="author">Holmes NG</span>, <span class="author">Ryder E</span>, <span class="author">Breen M</span>, <span class="author">Thomas R</span>, <span class="author">Suter N</span>, <span class="author">Sampson J</span>, <span class="author">Langford CF</span>, <span class="author">Ross M</span>, <span class="author">Carter NP</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Centre for Preventive Medicine, Animal Health Trust, Suffolk, England.</p><p class="abstract">The majority of microsatellite markers being used to generate the emerging genetic linkage maps of the dog are derived from small-insert, random clones. While such markers are easy to generate, they have the disadvantage that they cannot easily be physically mapped by fluorescence in situ hybridization (FISH), making it difficult to assess the extent of genome coverage represented by such maps. In contrast, microsatellite markers from large-insert libraries enable the linkage groups within which they fall to be physically anchored to specific chromosomes. One aim of our work is to identify at least one microsatellite-containing cosmid clone for each canine chromosome, to ensure that linkage groups exist for all chromosomes. This is particularly important for a species with as complex a karyotype as the dog. Locating two cosmids on each chromosome would allow the orientation of the linkage groups to be established. Chromosomal locations of cosmid clones containing microsatellites have been determined by FISH and confirmed using canine chromosome-specific paints. Microsatellite sequences have been genotyped on the DogMap reference family. Microsatellites derived from flow-sorted, chromosome-specific libraries represent another source of useful markers. Initial studies have been carried out on the canine X chromosome, on which markers were underrepresented in our initial studies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of heredity</span>&nbsp;<span class="year">1999</span>;<span class="volume">90</span>;<span class="number">1</span>;<span class="pagerange">52-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9987903" rel="external">9987903</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DNA sequence of human chromosome 22.</h4>
<p class="authors"><span class="author">Dunham I</span>, <span class="author">Shimizu N</span>, <span class="author">Roe BA</span>, <span class="author">Chissoe S</span>, <span class="author">Hunt AR</span>, <span class="author">Collins JE</span>, <span class="author">Bruskiewich R</span>, <span class="author">Beare DM</span>, <span class="author">Clamp M</span>, <span class="author">Smink LJ</span>, <span class="author">Ainscough R</span>, <span class="author">Almeida JP</span>, <span class="author">Babbage A</span>, <span class="author">Bagguley C</span>, <span class="author">Bailey J</span>, <span class="author">Barlow K</span>, <span class="author">Bates KN</span>, <span class="author">Beasley O</span>, <span class="author">Bird CP</span>, <span class="author">Blakey S</span>, <span class="author">Bridgeman AM</span>, <span class="author">Buck D</span>, <span class="author">Burgess J</span>, <span class="author">Burrill WD</span>, <span class="author">O&#39;Brien KP</span> <em>et al.</em></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%69%64%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#100;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Knowledge of the complete genomic DNA sequence of an organism allows a systematic approach to defining its genetic components. The genomic sequence provides access to the complete structures of all genes, including those without known function, their control elements, and, by inference, the proteins they encode, as well as all other biologically important sequences. Furthermore, the sequence is a rich and permanent source of information for the design of further biological studies of the organism and for the study of evolution through cross-species sequence comparison. The power of this approach has been amply demonstrated by the determination of the sequences of a number of microbial and model organisms. The next step is to obtain the complete sequence of the entire human genome. Here we report the sequence of the euchromatic part of human chromosome 22. The sequence obtained consists of 12 contiguous segments spanning 33.4 megabases, contains at least 545 genes and 134 pseudogenes, and provides the first view of the complex chromosomal landscapes that will be found in the rest of the genome.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1999</span>;<span class="volume">402</span>;<span class="number">6761</span>;<span class="pagerange">489-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10591208">10591208</a>; DOI: <a href="http://dx.doi.org/10.1038%2F990031">10.1038/990031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of gene expression in single cells.</h4>
<p class="authors"><span class="author">Freeman TC</span>, <span class="author">Lee K</span> and <span class="author">Richardson PJ</span></p>
<p class="affiliation">Gene Expression Group, Wellcome Trust Genome Campus, The Sanger Centre, Hinxton, CB10 1SA, UK.</p><p class="abstract">A cell&#39;s structural and functional characteristics are dependent on the specific complement of genes it expresses. The ability to study and compare gene usage at the cellular level will therefore provide valuable insights into cell physiology. Such analyses are complicated by problems associated with sample collection, sample size and the limited sensitivity of expression assays. Advances have been made in approaches to the collection of cellular material and the performance of single-cell gene expression analysis. Recent development in global amplification of mRNA may soon permit expression analyses of single cells to be performed on DNA microarrays.</p>
<p class="publication"><span class="title">Current opinion in biotechnology</span>&nbsp;<span class="year">1999</span>;<span class="volume">10</span>;<span class="number">6</span>;<span class="pagerange">579-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10600693" rel="external">10600693</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">[WWWMGS: an integrated server for molecular-genetic studies].</h4>
<p class="authors"><span class="author">Frolov AS</span>, <span class="author">Lavriushev SV</span>, <span class="author">Grigorovich DA</span>, <span class="author">Kel AE</span>, <span class="author">Ptitsyn AA</span>, <span class="author">Kolchanov NA</span>, <span class="author">Podkolodny&#x12D; NL</span>, <span class="author">Solov&#39;ev VV</span>, <span class="author">Milanesi L</span>, <span class="author">Bourne P</span> <em>et al.</em></p>
<p class="affiliation">Institute of Cytology and Genetics, Russian Academy of Sciences, Novosibirsk, Russia.</p><p class="abstract">We report an integrative technology for molecular biology studies in the field of transcription regulation by using Internet. A set of databases, programs, and systems are included into WWWMGS Web server. For example, the use of TRRD database information for site prediction is described. Using this method, the computer system SeqAnn was developed. The system performs the &quot;real time&quot; searching for prediction of initiation transcription site position according to database information. WWWMGS is available at URL: http://wwwmgs.bionet.nsc.ru/.</p>
<p class="publication"><span class="title">Biofizika</span>&nbsp;<span class="year">1999</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">832-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10624522" rel="external">10624522</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tdr2, a new zebrafish transposon of the Tc1 family.</h4>
<p class="authors"><span class="author">G&ouml;ttgens B</span>, <span class="author">Barton LM</span>, <span class="author">Grafham D</span>, <span class="author">Vaudin M</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Haematology, Cambridge University, MRC Centre, Hills Road, Cambridge, UK. <a href="mailto:%62%67%32%30%30%40%63%61%6D.%61%63.%75%6B">&#98;&#103;&#50;&#48;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We describe here Tdr2, a new class of Tc1-like transposons in zebrafish. Tdr2 was identified from the genomic sequence of a zebrafish PAC (P1 artificial chromosome) clone, and fragments of Tdr2 were found in several zebrafish EST (expressed sequence tag) sequences. Predicted translation of the Tdr2 transposase gene showed that it was most closely related to Caenorhabditis elegans Tc3A, suggesting an ancient origin of the Tdr2 transposon. Tdr2 spans 1. 1kb and is flanked by inverted repeats of approx. 100bp. The 5&#39; repeat is itself composed of an inverted repeat, raising the possibility of the formation of a cruciform DNA structure. Tdr2 transposons may facilitate the development of novel transposon-based tools for the genetic analysis of zebrafish.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">1999</span>;<span class="volume">239</span>;<span class="number">2</span>;<span class="pagerange">373-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10548740" rel="external">10548740</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Localisation of a gene for transient neonatal diabetes mellitus to an 18.72 cR3000 (approximately 5.4 Mb) interval on chromosome 6q.</h4>
<p class="authors"><span class="author">Gardner RJ</span>, <span class="author">Mungall AJ</span>, <span class="author">Dunham I</span>, <span class="author">Barber JC</span>, <span class="author">Shield JP</span>, <span class="author">Temple IK</span> and <span class="author">Robinson DO</span></p>
<p class="affiliation">Wessex Regional Genetics Laboratory, Salisbury District Hospital, Wiltshire, UK.</p><p class="abstract">Transient neonatal diabetes mellitus (TNDM) is a rare condition which presents with intrauterine growth retardation, dehydration, and failure to thrive. The condition spontaneously resolves before 1 year of age but predisposes patients to type 2 diabetes later in life. We have previously shown that, in some cases, TNDM is associated with paternal uniparental disomy (UPD) of chromosome 6 and suggested that an imprinted gene responsible for TNDM lies within a region of chromosome 6q. By analysing three families, two with duplications (family A and patient C) and one with several affected subjects with normal karyotypes (family B), we have further defined the TNDM critical region. In patient A, polymorphic microsatellite repeat analysis identified a duplicated region of chromosome 6, flanked by markers D6S472 and D6S311. This region was identified on the Sanger Centre&#39;s chromosome 6 radiation hybrid map (http://www.sanger.ac.uk/HGP/Chr6) and spanned approximately 60 cR3000. Using markers within the region, 418 unique P1 derived artificial chromosomes (PACs) have been isolated and used to localise the distal breakpoints of the two duplications. Linkage analysis of the familial case with a normal karyotype identified a recombination within the critical region. This recombination has been identified on the radiation hybrid map and defines the proximal end of the region of interest. We therefore propose that an imprinted gene for TNDM lies within an 18.72 cR3000 (approximately 5.4 Mb) interval on chromosome 6q24.1-q24.3 between markers D6S1699 and D6S1010.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">36</span>;<span class="number">3</span>;<span class="pagerange">192-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10204843" rel="external">10204843</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1734332" rel="external">1734332</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New insertion sequences and a novel repeated sequence in the genome of Mycobacterium tuberculosis H37Rv.</h4>
<p class="authors"><span class="author">Gordon SV</span>, <span class="author">Heym B</span>, <span class="author">Parkhill J</span>, <span class="author">Barrell B</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne, Institut Pasteur, Paris, France.</p><p class="abstract">The genome sequence of Mycobacterium tuberculosis H37Rv was found to contain 56 loci with homology to insertion sequences (ISs). As well as the previously described IS6110, IS1081, IS1547 and IS-like elements, new ISs belonging to the IS3, IS5, IS21, IS30, IS110, IS256 and ISL3 families were identified. In addition, six ISs created a grouping of their own to form a new family (the IS1535 family). Elements with similarity to ISs in other actinomycetes were identified, suggesting the movement of ISs between related genera. The location of ISs on the chromosome revealed that an approximately 600 kb region close to the origin of replication lacks ISs, pointing to the possible detrimental effect of insertions in this area. Analysis of the distribution of ISs through the tubercle strains Mycobacterium africanum, M. microti, M. bovis, M. bovis BCG Pasteur, M. tuberculosis H37Ra, M. tuberculosis CSU#93 and 29 clinical isolates revealed that only IS1532, IS1533, 1S1534, and IS1561&#39; were absent from some of the strains tested. A novel repeated sequence, the REP13E12 family, is described that is present in seven copies on the M. tuberculosis H37Rv chromosome and which contains a probable phage attachment site. This study therefore offers an insight into the possible role of ISs and repetitive elements in the evolution of the M. tuberculosis genome, as well as identifying genetic markers that may be useful for phylogenetic and epidemiological analysis of the tubercle complex.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">1999</span>;<span class="volume">145 ( Pt 4)</span>;<span class="pagerange">881-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10220167" rel="external">10220167</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A combined approach of conventional and molecular cytogenetics for detailed karyotypic analysis of the small cell lung carcinoma cell line U2020.</h4>
<p class="authors"><span class="author">Heppell-Parton AC</span>, <span class="author">Nacheva E</span>, <span class="author">Carter NP</span> and <span class="author">Rabbitts PH</span></p>
<p class="affiliation">Medical Research Council Centre, Addenbrooke&#39;s Hospital, Cambridge, United Kingdom.</p><p class="abstract">Until recently the ability to analyze complex karyotypic rearrangements was totally dependent upon light microscopy of G-banded chromosomes. Developments in the area of molecular cytogenetics have revolutionized such analysis, making it possible to determine the nature of complex rearrangements. An extensive analysis has been made of the small cell lung carcinoma (SCLC) cell line U2020, using a combined approach of conventional and molecular cytogenetics, enabling a highly detailed karyotype to be constructed revealing rearrangements previously undetected by G-banding alone. This approach offers the opportunity to reassess other tumor karyotypes, particularly those of high complexity found in solid tumors, for tumor-specific consistent rearrangements indecipherable by conventional karyotyping.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cancer genetics and cytogenetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">108</span>;<span class="number">2</span>;<span class="pagerange">110-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9973937" rel="external">9973937</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Elucidation of the mechanism of homozygous deletion of 3p12-13 in the U2020 cell line reveals the unexpected involvement of other chromosomes.</h4>
<p class="authors"><span class="author">Heppell-Parton AC</span>, <span class="author">Nacheva E</span>, <span class="author">Carter NP</span>, <span class="author">Bergh J</span>, <span class="author">Ogilvie D</span> and <span class="author">Rabbitts PH</span></p>
<p class="affiliation">Medical Research Council Centre, Cambridge, England, United Kingdom.</p><p class="abstract">Homozygous deletions in tumor cells have been useful in the localization and validation of tumor suppressor genes. We have described a homozygous deletion in a lung cancer cell line (U2020) which is located within the most proximal of the three regions on the short arm of chromosome 3 believed to be lost in lung cancer development. Construction of a YAC contig map indicates that the deletion spans around 8 Mb, but no large deletion was apparent on conventional cytogenetic analysis of the cell line. To investigate this paradox, whole chromosome, arm-specific, and regional paints have been used. This analysis has revealed that genetic loss has occurred by complex rearrangements of chromosomes 3, rather than simple interstitial deletion. These studies emphasize the power of molecular cytogenetics to disclose unsuspected tumor-specific translocations within the extremely complex karyotypes characteristic of solid tumors.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cancer genetics and cytogenetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">111</span>;<span class="number">2</span>;<span class="pagerange">105-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10347545" rel="external">10347545</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulation of hmp gene transcription in Mycobacterium tuberculosis: effects of oxygen limitation and nitrosative and oxidative stress.</h4>
<p class="authors"><span class="author">Hu Y</span>, <span class="author">Butcher PD</span>, <span class="author">Mangan JA</span>, <span class="author">Rajandream MA</span> and <span class="author">Coates AR</span></p>
<p class="affiliation">Department of Medical Microbiology, St. George&#39;s Hospital Medical School, London SW17 0RE, United Kingdom.</p><p class="abstract">The Mycobacterium tuberculosis hmp gene encodes a protein which is homologous to flavohemoglobin in Escherichia coli. Northern blotting analysis demonstrated that hmp transcription increased when a microaerophilic culture became oxygen limited as it entered stationary phase at 20 days. There was a fivefold increase of the hmp transcripts during early stationary phase compared with the value which was observed in the exponential growth phase. This induction of hmp transcription was not due to changes in the mRNA stability since the half-life of hmp mRNA was very short in a 20-day microaerophilic culture. No induction of hmp mRNA was observed during entry into stationary phase when the culture was continuously aerated. hmp transcription was induced after a short exposure of a late-exponential-phase culture to anaerobic conditions. These data indicate that oxygen limitation is the trigger for hmp gene transcription. In addition, when a microaerophilic culture entered into the stationary phase at 20 days, transcription of hmp increased to a small extent after exposure to S-nitrosoglutathione (a nitric oxide [NO] releaser) and sodium nitroprusside (an NO+ donor) and decreased after exposure to paraquat (a superoxide generator) and H2O2. In log phase (4 days) and late stationary phase (40 days), the transcription of hmp was unaffected by nitrosative and oxidative stress. Three primer extension products were observed. The -10 region is 100% identical to that of promoter T3 in mycobacteria and shows a strong similarity to the -10 sequence of hmp and rpoS promoters in E. coli. These observations of hmp mRNA induction in response to O2 limitation and nitrosative stress suggest that the hmp gene of M. tuberculosis may have a role in protection of the organism from NO killing under microaerophilic conditions.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">1999</span>;<span class="volume">181</span>;<span class="number">11</span>;<span class="pagerange">3486-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10348862" rel="external">10348862</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC93817" rel="external">93817</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RMS/coverage graphs: a qualitative method for comparing three-dimensional protein structure predictions.</h4>
<p class="authors"><span class="author">Hubbard TJ</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridgeshire, United Kingdom. <a href="mailto:%74%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Evaluating a set of protein structure predictions is difficult as each prediction may omit different residues and different parts of the structure may have different accuracies. A method is described that captures the best results from a large number of alternative sequence-dependent structural superpositions between a prediction and the experimental structure and represents them as a single line on a graph. Applied to CASP2 and CASP3 data the best predictions stand out visually in most cases, as judged by manual inspection. The results from this method applied to CASP data are available from the URLs http:/(/)PredictionCenter. llnl.gov/casp3/results/th/ and http:/(/)www.sanger.ac.uk/ approximately th/casp/.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">1999</span>;<span class="volume">Suppl 3</span>;<span class="pagerange">15-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10526348">10526348</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCOP: a Structural Classification of Proteins database.</h4>
<p class="authors"><span class="author">Hubbard TJ</span>, <span class="author">Ailey B</span>, <span class="author">Brenner SE</span>, <span class="author">Murzin AG</span> and <span class="author">Chothia C</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The Structural Classification of Proteins (SCOP) database provides a detailed and comprehensive description of the relationships of all known proteins structures. The classification is on hierarchical levels: the first two levels, family and superfamily, describe near and far evolutionary relationships; the third, fold, describes geometrical relationships. The distinction between evolutionary relationships and those that arise from the physics and chemistry of proteins is a feature that is unique to this database, so far. The database can be used as a source of data to calibrate sequence search algorithms and for the generation of population statistics on protein structures. The database and its associated files are freely accessible from a number of WWW sites mirrored from URL http://scop. mrc-lmb.cam.ac.uk/scop/</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1999</span>;<span class="volume">27</span>;<span class="number">1</span>;<span class="pagerange">254-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9847194">9847194</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC148149">148149</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Leishmania genome comes of Age.</h4>
<p class="authors"><span class="author">Ivens AC</span> and <span class="author">Blackwell JM</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%61%6C%69%63%61%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#105;&#99;&#97;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Leishmania Genome Network (LGN) was born in Rio de Janeiro, Brazil in 1994. In the short period that has elapsed since then, the LGN has focused solely on the acquisition of the resources, and hence data, that have enabled a rational approach to genomic sequencing of the reference strain, Leishmania major Friedlin. This has now been achieved. In this review, Alasdair Ivens and Jennie Blackwell, secretary and chairman of the LGN, respectively, re-examine the approaches that were adopted, comment on some of the interesting data that have been obtained and introduce some genome-wide approaches that will facilitate functional studies of the parasite.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Parasitology today (Personal ed.)</span>&nbsp;<span class="year">1999</span>;<span class="volume">15</span>;<span class="number">6</span>;<span class="pagerange">225-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10366828" rel="external">10366828</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative analysis of noncoding regions of 77 orthologous mouse and human gene pairs.</h4>
<p class="authors"><span class="author">Jareborg N</span>, <span class="author">Birney E</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Sanger Centre, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6E%69%63%6C%61%73.%6A%61%72%65%62%6F%72%67%40%63%67%72.%6B%69.%73%65">&#110;&#105;&#99;&#108;&#97;&#115;&#46;&#106;&#97;&#114;&#101;&#98;&#111;&#114;&#103;&#64;&#99;&#103;&#114;&#46;&#107;&#105;&#46;&#115;&#101;</a></p><p class="abstract">A data set of 77 genomic mouse/human gene pairs has been compiled from the EMBL nucleotide database, and their corresponding features determined. This set was used to analyze the degree of conservation of noncoding sequences between mouse and human. A new alignment algorithm was developed to cope with the fact that large parts of noncoding sequences are not alignable in a meaningful way because of genetic drift. This new algorithm, DNA Block Aligner (DBA), finds colinear-conserved blocks that are flanked by nonconserved sequences of varying lengths. The noncoding regions of the data set were aligned with DBA. The proportion of the noncoding regions covered by blocks &gt;60% identical was 36% for upstream regions, 50% for 5&#39; UTRs, 23% for introns, and 56% for 3&#39; UTRs. These blocks of high identity were more or less evenly distributed across the length of the features, except for upstream regions in which the first 100 bp upstream of the transcription start site was covered in up to 70% of the gene pairs. This data set complements earlier sets on the basis of cDNA sequences and will be useful for further comparative studies. [This paper contains supplementary data that can be found at http://www.genome.org [corrected]].</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1999</span>;<span class="volume">9</span>;<span class="number">9</span>;<span class="pagerange">815-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10508839" rel="external">10508839</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310816" rel="external">310816</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human family of Deafness/Dystonia peptide (DDP) related mitochondrial import proteins.</h4>
<p class="authors"><span class="author">Jin H</span>, <span class="author">Kendall E</span>, <span class="author">Freeman TC</span>, <span class="author">Roberts RG</span> and <span class="author">Vetrie DL</span></p>
<p class="affiliation">GKT Medical School, Guy&#39;s Hospital, London, SE1 9RT, United Kingdom.</p><p class="abstract">The gene responsible for the human genetic neurodegenerative disorder DFN-1/MTS encodes a small protein known as deafness/dystonia peptide (DDP). It bears a strong resemblance to a recently characterized set of zinc-binding yeast proteins (Tim8p, Tim9p, Tim10p, Tim12p, and Tim13p) that are implicated in the import of a class of transmembrane carrier proteins from the cytoplasm to the mitochondrial inner membrane. We describe here the human complement of DDP/Tim-like proteins and establish the likely orthologous relationships between sequences from human, yeast, and other organisms. We also describe the expression patterns and chromosomal locations of their genes, which are candidate loci for autosomal recessive neurodegenerative disorders.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1999</span>;<span class="volume">61</span>;<span class="number">3</span>;<span class="pagerange">259-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10552927" rel="external">10552927</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1999.5966" rel="external">10.1006/geno.1999.5966</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The chicken B locus is a minimal essential major histocompatibility complex.</h4>
<p class="authors"><span class="author">Kaufman J</span>, <span class="author">Milne S</span>, <span class="author">G&ouml;bel TW</span>, <span class="author">Walker BA</span>, <span class="author">Jacob JP</span>, <span class="author">Auffray C</span>, <span class="author">Zoorob R</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Institute for Animal Health, Compton, UK. <a href="mailto:%6A%69%6D.%6B%61%75%66%6D%61%6E%40%62%62%73%72%63.%61%63.%75%6B">&#106;&#105;&#109;&#46;&#107;&#97;&#117;&#102;&#109;&#97;&#110;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Here we report the sequence of the region that determines rapid allograft rejection in chickens, the chicken major histocompatibility complex (MHC). This 92-kilobase region of the B locus contains only 19 genes, making the chicken MHC roughly 20-fold smaller than the human MHC. Virtually all the genes have counterparts in the human MHC, defining a minimal essential set of MHC genes conserved over 200 million years of divergence between birds and mammals. They are organized differently, with the class III region genes located outside the class II and class I region genes. The absence of proteasome genes is unexpected and might explain unusual peptide-binding specificities of chicken class I molecules. The presence of putative natural killer receptor gene(s) is unprecedented and might explain the importance of the B locus in the response to the herpes virus responsible for Marek&#39;s diseases. The small size and simplicity of the chicken MHC allows co-evolution of genes as haplotypes over considerable periods of time, and makes it possible to study the striking MHC-determined pathogen-specific disease resistance at the molecular level.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1999</span>;<span class="volume">401</span>;<span class="number">6756</span>;<span class="pagerange">923-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10553909">10553909</a>; DOI: <a href="http://dx.doi.org/10.1038%2F44856">10.1038/44856</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrated databases and computer systems for studying eukaryotic gene expression.</h4>
<p class="authors"><span class="author">Kolchanov NA</span>, <span class="author">Ponomarenko MP</span>, <span class="author">Frolov AS</span>, <span class="author">Ananko EA</span>, <span class="author">Kolpakov FA</span>, <span class="author">Ignatieva EV</span>, <span class="author">Podkolodnaya OA</span>, <span class="author">Goryachkovskaya TN</span>, <span class="author">Stepanenko IL</span>, <span class="author">Merkulova TI</span>, <span class="author">Babenko VV</span>, <span class="author">Ponomarenko YV</span>, <span class="author">Kochetov AV</span>, <span class="author">Podkolodny NL</span>, <span class="author">Vorobiev DV</span>, <span class="author">Lavryushev SV</span>, <span class="author">Grigorovich DA</span>, <span class="author">Kondrakhin YV</span>, <span class="author">Milanesi L</span>, <span class="author">Wingender E</span>, <span class="author">Solovyev V</span> and <span class="author">Overton GC</span></p>
<p class="affiliation">Institute of Cytology &amp; Genetics, Siberian Branch of the Russian Academy of Sciences, Prosp. Lavrentieva 10, Novosibirsk 630090, Russia. <a href="mailto:%6B%6F%6C%40%62%69%6F%6E%65%74.%6E%73%63.%72%75">&#107;&#111;&#108;&#64;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;</a></p><p class="abstract"><strong>Motivation</strong>: The goal of the work was to develop a WWW-oriented computer system providing a maximal integration of informational and software resources on the regulation of gene expression and navigation through them. Rapid growth of the variety and volume of information accumulated in the databases on regulation of gene expression necessarily requires the development of computer systems for automated discovery of the knowledge that can be further used for analysis of regulatory genomic sequences.</p><p class="abstract"><strong>Results</strong>: The GeneExpress system developed includes the following major informational and software modules: (1) Transcription Regulation (TRRD) module, which contains the databases on transcription regulatory regions of eukaryotic genes and TRRD Viewer for data visualization; (2) Site Activity Prediction (ACTIVITY), the module for analysis of functional site activity and its prediction; (3) Site Recognition module, which comprises (a) B-DNA-VIDEO system for detecting the conformational and physicochemical properties of DNA sites significant for their recognition, (b) Consensus and Weight Matrices (ConsFrec) and (c) Transcription Factor Binding Sites Recognition (TFBSR) systems for detecting conservative contextual regions of functional sites and their recognition; (4) Gene Networks (GeneNet), which contains an object-oriented database accumulating the data on gene networks and signal transduction pathways, and the Java-based Viewer for exploration and visualization of the GeneNet information; (5) mRNA Translation (Leader mRNA), designed to analyze structural and contextual properties of mRNA 5&#39;-untranslated regions (5&#39;-UTRs) and predict their translation efficiency; (6) other program modules designed to study the structure-function organization of regulatory genomic sequences and regulatory proteins.</p><p class="abstract"><strong>Availability</strong>: GeneExpress is available at http://wwwmgs.bionet.nsc. ru/systems/GeneExpress/ and the links to the mirror site(s) can be found at http://wwwmgs.bionet.nsc.ru/mgs/links/mirrors.html+ ++.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">5-R01-RR04026-09</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">1999</span>;<span class="volume">15</span>;<span class="number">7-8</span>;<span class="pagerange">669-86</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10487874" rel="external">10487874</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">[GeneExpress: an integrator for databases and computer systems accessible by the Internet and intended for studying eukaryotic gene expression].</h4>
<p class="authors"><span class="author">Kolchanov NA</span>, <span class="author">Ponomarenko MP</span>, <span class="author">Kel AE</span>, <span class="author">Kondrakhin IuV</span>, <span class="author">Frolov AS</span>, <span class="author">Kolpakov FA</span>, <span class="author">Goriachkovskaia TN</span>, <span class="author">Kel OV</span>, <span class="author">Anan&#39;ko EA</span>, <span class="author">Ignat&#39;eva EV</span> <em>et al.</em></p>
<p class="affiliation">Institute of Cytology and Genetics, Russian Academy of Sciences, Novosibirsk, Russia.</p><p class="abstract">We have developed GeneExpress that is the WWW-oriented integrator for the databases and systems supporting the investigation of gene expression. The total number of the Web-based resources integrated is 30. The database GeneNet on molecular events forming gene networks was assigned its integrative core. To navigate all these WWW-available resources, the SRS, HTML, and Java viewers were developed, <a href="mailto:%68%74%74%70%3A%40%77%77%77%6D%67%73.%62%69%6F%6E%65%74.%6E%73%63.%72%75/%73%79%73%74%65%6D%73/%47%65%6E%65%45%78%70%72%65%73%73">&#104;&#116;&#116;&#112;&#58;&#64;&#119;&#119;&#119;&#109;&#103;&#115;&#46;&#98;&#105;&#111;&#110;&#101;&#116;&#46;&#110;&#115;&#99;&#46;&#114;&#117;&#47;&#115;&#121;&#115;&#116;&#101;&#109;&#115;&#47;&#71;&#101;&#110;&#101;&#69;&#120;&#112;&#114;&#101;&#115;&#115;</a>/.</p>
<p class="publication"><span class="title">Biofizika</span>&nbsp;<span class="year">1999</span>;<span class="volume">44</span>;<span class="number">5</span>;<span class="pagerange">837-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10624523" rel="external">10624523</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in SLC19A2 cause thiamine-responsive megaloblastic anaemia associated with diabetes mellitus and deafness.</h4>
<p class="authors"><span class="author">Labay V</span>, <span class="author">Raz T</span>, <span class="author">Baron D</span>, <span class="author">Mandel H</span>, <span class="author">Williams H</span>, <span class="author">Barrett T</span>, <span class="author">Szargel R</span>, <span class="author">McDonald L</span>, <span class="author">Shalata A</span>, <span class="author">Nosaka K</span>, <span class="author">Gregory S</span> and <span class="author">Cohen N</span></p>
<p class="affiliation">Department of Genetics, Tamkin Human Molecular Genetics Research Facility, Technion-Israel Institute of Technology, Bruce Rappaport Faculty of Medicine, Haifa.</p><p class="abstract">Thiamine-responsive megaloblastic anaemia (TRMA), also known as Rogers syndrome, is an early onset, autosomal recessive disorder defined by the occurrence of megaloblastic anaemia, diabetes mellitus and sensorineural deafness, responding in varying degrees to thiamine treatment (MIM 249270). We have previously narrowed the TRMA locus from a 16-cM to a 4-cM interval on chromosomal region 1q23.3 (refs 3,4) and this region has been further refined to a 1.4-cM interval. Previous studies have suggested that deficiency in a high-affinity thiamine transporter may cause this disorder. Here we identify the TRMA gene by positional cloning. We assembled a P1-derived artificial chromosome (PAC) contig spanning the TRMA candidate region. This clarified the order of genetic markers across the TRMA locus, provided 9 new polymorphic markers and narrowed the locus to an approximately 400-kb region. Mutations in a new gene, SLC19A2, encoding a putative transmembrane protein homologous to the reduced folate carrier proteins, were found in all affected individuals in six TRMA families, suggesting that a defective thiamine transporter protein (THTR-1) may underlie the TRMA syndrome.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">22</span>;<span class="number">3</span>;<span class="pagerange">300-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10391221">10391221</a>; DOI: <a href="http://dx.doi.org/10.1038%2F10372">10.1038/10372</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data mining parasite genomes: haystack searching with a computer.</h4>
<p class="authors"><span class="author">Lawson D</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">A number of genomes of parasitic organisms are presently being sequenced in the public domain, including Plasmodium falciparum, Leishmania major and Trypanosoma brucei with the likelihood of at least expressed sequence tag (EST) projects for several filarial and apicomplexan species. The early and timely release of sequence data to the community via the World Wide Web (www), and the public databases, (EMBL and GENBANK), forms an invaluable resource. Data mining, or &#39;haystack searching&#39; this resource is becoming more fruitful to all members of the scientific community as the volume of data, diversity of genomes sampled, and accessibility increase.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Parasitology</span>&nbsp;<span class="year">1999</span>;<span class="volume">118 Suppl</span>;<span class="pagerange">S15-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10466134">10466134</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A molecular cytogenetic clone resource for chromosome 22.</h4>
<p class="authors"><span class="author">Leversha MA</span>, <span class="author">Dunham I</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">1999</span>;<span class="volume">7</span>;<span class="number">7</span>;<span class="pagerange">571-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10598573" rel="external">10598573</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel mode of ligand binding by the SH2 domain of the human XLP disease gene product SAP/SH2D1A.</h4>
<p class="authors"><span class="author">Li SC</span>, <span class="author">Gish G</span>, <span class="author">Yang D</span>, <span class="author">Coffey AJ</span>, <span class="author">Forman-Kay JD</span>, <span class="author">Ernberg I</span>, <span class="author">Kay LE</span> and <span class="author">Pawson T</span></p>
<p class="affiliation">Program in Molecular Biology and Cancer, Department of Molecular and Medical Genetics, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, University of Toronto, Canada.</p><p class="abstract"><strong>Background</strong>: The Src homology 2 (SH2) domains of cytoplasmic signaling proteins generally bind phosphotyrosine (pTyr) sites in the context of carboxy-terminal residues. SAP (also known as SH2D1A or DSHP), the product of the gene that is mutated in human X-linked lymphoproliferative (XLP) disease, comprises almost exclusively a single SH2 domain, which may modulate T-cell signaling by engaging T-cell co-activators such as SLAM, thereby blocking binding of other signaling proteins that contain SH2 domains. The SAP-SLAM interaction can occur in a phosphorylation-independent manner.</p><p class="abstract"><strong>Results</strong>: To characterize the interaction between SAP and SLAM, we synthesized peptides corresponding to the SAP-binding site at residue Y281 in SLAM. Both phosphorylated and non-phosphorylated versions of an 11-residue SLAM peptide bound SAP, with dissociation constants of 150 nM and 330 nM, respectively. SLAM phosphopeptides that were truncated either at the amino or carboxyl terminus bound with high affinity to SAP, suggesting that the SAP SH2 domain recognizes both amino-terminal and carboxy-terminal sequences relative to the pTyr residue. These results were confirmed by nuclear magnetic resonance (NMR) studies on (15)N- and (13)C-labeled SAP complexed with three SLAM peptides: an amino-terminally truncated phosphopeptide, a carboxy-terminally truncated phosphopeptide and a non-phosphorylated Tyr-containing full-length peptide.</p><p class="abstract"><strong>Conclusions</strong>: The SAP SH2 domain has a unique specificity. Not only does it bind peptides in a phosphorylation-independent manner, it also recognizes a pTyr residue either preceded by amino-terminal residues or followed by carboxy-terminal residues. We propose that the three &#39;prongs&#39; of a peptide ligand (the amino and carboxyl termini and the pTyr) can engage the SAP SH2 domain, accounting for its unusual properties. These data point to the flexibility of modular protein-interaction domains.</p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">1999</span>;<span class="volume">9</span>;<span class="number">23</span>;<span class="pagerange">1355-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10607564" rel="external">10607564</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Isolation and characterization of human and mouse ZIRTL, a member of the IRT1 family of transporters, mapping within the epidermal differentiation complex.</h4>
<p class="authors"><span class="author">Lioumi M</span>, <span class="author">Ferguson CA</span>, <span class="author">Sharpe PT</span>, <span class="author">Freeman T</span>, <span class="author">Marenholz I</span>, <span class="author">Mischke D</span>, <span class="author">Heizmann C</span> and <span class="author">Ragoussis J</span></p>
<p class="affiliation">Division of Medical and Molecular Genetics, Department of Craniofacial Development, The Guy&#39;s, King&#39;s College and St. Thomas&#39; Hospitals&#39; Medical and Dental School, London, United Kingdom. <a href="mailto:%6D.%6C%69%6F%75%6D%69%40%75%6D%64%73.%61%63.%75%6B">&#109;&#46;&#108;&#105;&#111;&#117;&#109;&#105;&#64;&#117;&#109;&#100;&#115;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We report the precise mapping and characterization of ZIRTL (zinc-iron regulated transporter-like) gene, the first mammalian member of an extensive family of divalent metal ion transporters, comprising IRT1 and ZIP1, ZIP2, ZIP3, and ZIP4 in plants and ZRT1 and ZRT2 in yeast. The human gene maps at the telomeric end of the epidermal differentiation complex (EDC), within chromosomal band 1q21, while the mouse gene maps within the mouse EDC, on mouse chromosome 3, between S100A9 and S100A13. The structure of the human gene has been determined, and message was detected in most adult and fetal tissues including the epidermis. The mouse gene is developmentally regulated and found expressed in fetal and adult suprabasal epidermis, osteoblasts, small intestine, and salivary gland.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1999</span>;<span class="volume">62</span>;<span class="number">2</span>;<span class="pagerange">272-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10610721" rel="external">10610721</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1999.5993" rel="external">10.1006/geno.1999.5993</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Psoriasis upregulated phorbolin-1 shares structural but not functional similarity to the mRNA-editing protein apobec-1.</h4>
<p class="authors"><span class="author">Madsen P</span>, <span class="author">Anant S</span>, <span class="author">Rasmussen HH</span>, <span class="author">Gromov P</span>, <span class="author">Vorum H</span>, <span class="author">Dumanski JP</span>, <span class="author">Tommerup N</span>, <span class="author">Collins JE</span>, <span class="author">Wright CL</span>, <span class="author">Dunham I</span>, <span class="author">MacGinnitie AJ</span>, <span class="author">Davidson NO</span> and <span class="author">Celis JE</span></p>
<p class="affiliation">Department of Medical Biochemistry, University of Aarhus, Denmark. <a href="mailto:%70%6D%40%62%69%6F%6B%65%6D%69.%61%75.%64%6B">&#112;&#109;&#64;&#98;&#105;&#111;&#107;&#101;&#109;&#105;&#46;&#97;&#117;&#46;&#100;&#107;</a></p><p class="abstract">Earlier studies of psoriatic and normal primary keratinocytes treated with phorbol 12-myristate-1-acetate identified two low-molecular-weight proteins, termed phorbolin-1 (20 kDa; pI 6.6) and phorbolin-2 (17.6 kDa; pI 6.5). As a first step towards elucidating the role of these proteins in psoriasis, we report here the molecular cloning and chromosomal mapping of phorbolin-1 and a related cDNA that codes for a protein exhibiting a similar amino acid sequence. The phorbolins were mapped to position 22q13 immediately centromeric to the c-sis proto-oncogene. Transient expression of the phorbolin-1 cDNA in COS cells and by in vitro transcription/translation, yielded polypeptides that comigrated with phorbolins-1 and -2. Comparative sequence analysis revealed 22% overall identity and a similarity of 44% of the phorbolins to apobec-1, the catalytic subunit of the mammalian apolipoprotein B mRNA editing enzyme; however, recombinant-expressed phorbolin-1 exhibited no cytidine deaminase activity, using either a monomeric nucleoside or apolipoprotein B cRNA as substrate, and failed to bind an AU-rich RNA template. Whereas the precise function of the phorbolins remains to be elucidated, the current data suggest that it is unlikely to include a role in the post-transcriptional modification of RNA in a manner analogous to that described for apobec-1.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">HL-38180</span>; NIDDK NIH HHS: <span class="grant">DK-42030</span>; Wellcome Trust</p>
<p class="publication"><span class="title">The Journal of investigative dermatology</span>&nbsp;<span class="year">1999</span>;<span class="volume">113</span>;<span class="number">2</span>;<span class="pagerange">162-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10469298" rel="external">10469298</a>; DOI: <a href="http://dx.doi.org/10.1046%2Fj.1523-1747.1999.00682.x" rel="external">10.1046/j.1523-1747.1999.00682.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The isolation and high-resolution chromosomal mapping of human SOX14 and SOX21; two members of the SOX gene family related to SOX1, SOX2, and SOX3.</h4>
<p class="authors"><span class="author">Malas S</span>, <span class="author">Duthie S</span>, <span class="author">Deloukas P</span> and <span class="author">Episkopou V</span></p>
<p class="affiliation">MRC Clinical Sciences Centre, Imperial College School of Medicine, Hammersmith Hospital, London W12 ONN, UK.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">1999</span>;<span class="volume">10</span>;<span class="number">9</span>;<span class="pagerange">934-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10441749" rel="external">10441749</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NME6: a new member of the nm23/nucleoside diphosphate kinase gene family located on human chromosome 3p21.3.</h4>
<p class="authors"><span class="author">Mehus JG</span>, <span class="author">Deloukas P</span> and <span class="author">Lambeth DO</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology, University of North Dakota School of Medicine and Health Sciences, Grand Forks 58202, USA.</p><p class="abstract">The NME (nm23/nucleoside diphosphate kinase) gene family in human is involved in the phosphorylation of nucleoside diphosphates and a variety of regulatory phenomena associated with development, oncogenic transformation, and metastasis. Here we report the cDNA sequence for a sixth member of this family, NME6. The cDNA sequence predicts a 186-residue protein that includes the characteristic active site motif of a nucleoside diphosphate (NDP) kinase, as well as the other residues previously identified as crucial for nucleotide binding and catalysis. The NME6 protein sequence is only 34-41% identical to the five previously reported human NME proteins, and is similarly related to prokaryotic and primitive eukaryotic NDP kinases. Compared to typical proteins of this family such as NME1 and NME2, NME6 has three additional residues located in the Kpn loop, and a 22-residue extension at the COOH-terminal. Using radiation hybrid mapping, the NME6 gene was localized to chromosome 3p21.3. The 1.3-kb transcript of NME6 is expressed at a moderately low level in many human tissues, and is most abundant in kidney, prostate, ovary, intestine, and spleen. Homologous cDNAs were also cloned and sequenced for rat and mouse. The sequence of the first 171 residues of the mouse homologue (Nm23-M6) is 94% identical to the deduced human NME6 protein.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM54260</span></p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">104</span>;<span class="number">6</span>;<span class="pagerange">454-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10453732" rel="external">10453732</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Techview: DNA sequencing. Sequencing the genome, fast.</h4>
<p class="authors"><span class="author">Mullikin JC</span> and <span class="author">McMurragy AA</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambs, UK. <a href="mailto:%6A%63%6D%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#99;&#109;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1999</span>;<span class="volume">283</span>;<span class="number">5409</span>;<span class="pagerange">1867-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10206892">10206892</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A YAC-based physical map of the mouse genome.</h4>
<p class="authors"><span class="author">Nusbaum C</span>, <span class="author">Slonim DK</span>, <span class="author">Harris KL</span>, <span class="author">Birren BW</span>, <span class="author">Steen RG</span>, <span class="author">Stein LD</span>, <span class="author">Miller J</span>, <span class="author">Dietrich WF</span>, <span class="author">Nahf R</span>, <span class="author">Wang V</span>, <span class="author">Merport O</span>, <span class="author">Castle AB</span>, <span class="author">Husain Z</span>, <span class="author">Farino G</span>, <span class="author">Gray D</span>, <span class="author">Anderson MO</span>, <span class="author">Devine R</span>, <span class="author">Horton LT</span>, <span class="author">Ye W</span>, <span class="author">Wu X</span>, <span class="author">Kouyoumjian V</span>, <span class="author">Zemsteva IS</span>, <span class="author">Wu Y</span>, <span class="author">Collymore AJ</span>, <span class="author">Courtney DF</span>, <span class="author">Tam J</span>, <span class="author">Cadman M</span>, <span class="author">Haynes AR</span>, <span class="author">Heuston C</span>, <span class="author">Marsland T</span>, <span class="author">Southwell A</span>, <span class="author">Trickett P</span>, <span class="author">Strivens MA</span>, <span class="author">Ross MT</span>, <span class="author">Makalowski W</span>, <span class="author">Xu Y</span>, <span class="author">Boguski MS</span>, <span class="author">Carter NP</span>, <span class="author">Denny P</span>, <span class="author">Brown SD</span>, <span class="author">Hudson TJ</span> and <span class="author">Lander ES</span></p>
<p class="affiliation">Whitehead Institute for Biomedical Research, Cambridge, Massachusetts 02142, USA.</p><p class="abstract">A physical map of the mouse genome is an essential tool for both positional cloning and genomic sequencing in this key model system for biomedical research. Indeed, the construction of a mouse physical map with markers spaced at an average interval of 300 kb is one of the stated goals of the Human Genome Project. Here we report the results of a project at the Whitehead Institute/MIT Center for Genome Research to construct such a physical map of the mouse. We built the map by screening sequenced-tagged sites (STSs) against a large-insert yeast artificial chromosome (YAC) library and then integrating the STS-content information with a dense genetic map. The integrated map shows the location of 9,787 loci, providing landmarks with an average spacing of approximately 300 kb and affording YAC coverage of approximately 92% of the mouse genome. We also report the results of a project at the MRC UK Mouse Genome Centre targeted at chromosome X. The project produced a YAC-based map containing 619 loci (with 121 loci in common with the Whitehead map and 498 additional loci), providing especially dense coverage of this sex chromosome. The YAC-based physical map directly facilitates positional cloning of mouse mutations by providing ready access to most of the genome. More generally, use of this map in addition to a newly constructed radiation hybrid (RH) map provides a comprehensive framework for mouse genomic studies.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">22</span>;<span class="number">4</span>;<span class="pagerange">388-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10431246">10431246</a>; DOI: <a href="http://dx.doi.org/10.1038%2F11967">10.1038/11967</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis and assessment of ab initio three-dimensional prediction, secondary structure, and contacts prediction.</h4>
<p class="authors"><span class="author">Orengo CA</span>, <span class="author">Bray JE</span>, <span class="author">Hubbard T</span>, <span class="author">LoConte L</span> and <span class="author">Sillitoe I</span></p>
<p class="affiliation">Department of Biochemistry and Molecular Biology, University College, London, United Kingdom. <a href="mailto:%6F%72%65%6E%67%6F%40%62%73%6D.%62%69%6F%63.%75%63%6C.%61%63.%75%6B">&#111;&#114;&#101;&#110;&#103;&#111;&#64;&#98;&#115;&#109;&#46;&#98;&#105;&#111;&#99;&#46;&#117;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">CASP3 saw a substantial increase in the volume of ab initio 3D prediction data, with 507 datasets for fifteen selected targets and sixty-one groups participating. As with CASP2, methods ranged from computationally intensive strategies that attempt to recreate the physical and chemical forces involved in protein folding to the more recent knowledge-based approaches. These exploit information from the structure databases, extracting potentially similar fragments and/or distance constraints derived from multiple sequence alignments. The knowledge-based approaches generally gave more consistently successful predictions across the range of targets, particularly that of the Baker group (Bystroff and Baker, J Mol Biol 1998;281:565-577; Simons et al. Proteins Suppl 1999;3:171-176), which used a fragment library. In the secondary structure prediction category, the most successful approaches built on the concepts used in PHD (Rost et al. Comput Appl Biosci 1994;10:53-60), an accepted standard in this field. Like PHD, they exploit neural networks but have different strategies for incorporating multiple sequence data or position-dependent weight matrices for training the networks. Analysis of the contact data, for which only six groups participated, suggested that as yet this data provides a rather weak signal. However, in combination with other types of prediction data it can sometimes be a useful constraint for identifying the correct structure.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">1999</span>;<span class="volume">Suppl 3</span>;<span class="pagerange">149-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10526364">10526364</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">&#39;Going wrong with confidence&#39;: misleading sequence analyses of CiaB and clpX.</h4>
<p class="authors"><span class="author">Pallen M</span>, <span class="author">Wren B</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">1999</span>;<span class="volume">34</span>;<span class="number">1</span>;<span class="pagerange">195</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10540297" rel="external">10540297</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human LARGE gene from 22q12.3-q13.1 is a new, distinct member of the glycosyltransferase gene family.</h4>
<p class="authors"><span class="author">Peyrard M</span>, <span class="author">Seroussi E</span>, <span class="author">Sandberg-Nordqvist AC</span>, <span class="author">Xie YG</span>, <span class="author">Han FY</span>, <span class="author">Fransson I</span>, <span class="author">Collins J</span>, <span class="author">Dunham I</span>, <span class="author">Kost-Alimova M</span>, <span class="author">Imreh S</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Molecular Medicine, Karolinska Hospital, S-171 76 Stockholm, Sweden.</p><p class="abstract">Meningioma, a tumor of the meninges covering the central nervous system, shows frequent loss of material from human chromosome 22. Homozygous and heterozygous deletions in meningiomas defined a candidate region of &gt;1 Mbp in 22q12.3-q13.1 and directed us to gene cloning in this segment. We characterized a new member of the N-acetylglucosaminyltransferase gene family, the LARGE gene. It occupies &gt;664 kilobases and is one of the largest human genes. The predicted 756-aa N-acetylglucosaminyltransferase encoded by LARGE displays features that are absent in other glycosyltransferases. The human like-acetylglucosaminyltransferase polypeptide is much longer and contains putative coiled-coil domains. We characterized the mouse LARGE ortholog, which encodes a protein 97.75% identical with the human counterpart. Both genes reveal ubiquitous expression as assessed by Northern blot analysis and in situ histochemistry. Chromosomal mapping of the mouse gene reveals that mouse chromosome 8C1 corresponds to human 22q12.3-q13.1. Abnormal glycosylation of proteins and glycosphingolipids has been shown as a mechanism behind an increased potential of tumor formation and/or progression. Human tumors overexpress ganglioside GD3 (NeuAcalpha2,8NeuAcalpha2, 3Galbeta1,4Glc-Cer), which in meningiomas correlates with deletions on chromosome 22. It is the first time that a glycosyltransferase gene is involved in tumor-specific genomic rearrangements. An abnormal function of the human like-acetylglucosaminyltransferase protein may be linked to the development/progression of meningioma by altering the composition of gangliosides and/or by effect(s) on other glycosylated molecules in tumor cells.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1999</span>;<span class="volume">96</span>;<span class="number">2</span>;<span class="pagerange">598-603</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9892679" rel="external">9892679</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC15182" rel="external">15182</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic basis for lipopolysaccharide O-antigen biosynthesis in bordetellae.</h4>
<p class="authors"><span class="author">Preston A</span>, <span class="author">Allen AG</span>, <span class="author">Cadisch J</span>, <span class="author">Thomas R</span>, <span class="author">Stevens K</span>, <span class="author">Churcher CM</span>, <span class="author">Badcock KL</span>, <span class="author">Parkhill J</span>, <span class="author">Barrell B</span> and <span class="author">Maskell DJ</span></p>
<p class="affiliation">Centre for Veterinary Science, Department of Clinical Veterinary Medicine, University of Cambridge, Cambridge CB3 0ES, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%61%70%32%34%33%40%6D%6F%6C%65.%62%69%6F.%63%61%6D.%61%63.%75%6B">&#97;&#112;&#50;&#52;&#51;&#64;&#109;&#111;&#108;&#101;&#46;&#98;&#105;&#111;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Bordetella bronchiseptica and Bordetella parapertussis express a surface polysaccharide, attached to a lipopolysaccharide, which has been called O antigen. This structure is absent from Bordetella pertussis. We report the identification of a large genetic locus in B. bronchiseptica and B. parapertussis that is required for O-antigen biosynthesis. The locus is replaced by an insertion sequence in B. pertussis, explaining the lack of O-antigen biosynthesis in this species. The DNA sequence of the B. bronchiseptica locus has been determined and the presence of 21 open reading frames has been revealed. We have ascribed putative functions to many of these open reading frames based on database searches. Mutations in the locus in B. bronchiseptica and B. parapertussis prevent O-antigen biosynthesis and provide tools for the study of the role of O antigen in infections caused by these bacteria.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">1999</span>;<span class="volume">67</span>;<span class="number">8</span>;<span class="pagerange">3763-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10417135" rel="external">10417135</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC96651" rel="external">96651</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing. Gels and genomes.</h4>
<p class="authors"><span class="author">Rogers J</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6A%72%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#114;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1999</span>;<span class="volume">286</span>;<span class="number">5439</span>;<span class="pagerange">429</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10577205">10577205</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular analysis of a spontaneous dystrophin &#39;knockout&#39; dog.</h4>
<p class="authors"><span class="author">Schatzberg SJ</span>, <span class="author">Olby NJ</span>, <span class="author">Breen M</span>, <span class="author">Anderson LV</span>, <span class="author">Langford CF</span>, <span class="author">Dickens HF</span>, <span class="author">Wilton SD</span>, <span class="author">Zeiss CJ</span>, <span class="author">Binns MM</span>, <span class="author">Kornegay JN</span>, <span class="author">Morris GE</span> and <span class="author">Sharp NJ</span></p>
<p class="affiliation">College of Veterinary Medicine, North Carolina State University, Raleigh 27606, USA. <a href="mailto:%73%63%6F%74%74%5F%73%63%68%61%74%7A%62%65%72%67%40%6E%63%73%75.%65%64%75">&#115;&#99;&#111;&#116;&#116;&#95;&#115;&#99;&#104;&#97;&#116;&#122;&#98;&#101;&#114;&#103;&#64;&#110;&#99;&#115;&#117;&#46;&#101;&#100;&#117;</a></p><p class="abstract">We have determined the molecular basis for skeletal myopathy and dilated cardiomyopathy in two male German short-haired pointer (GSHP) littermates. Analysis of skeletal muscle demonstrated a complete absence of dystrophin on Western blot analysis. PCR analysis of genomic DNA revealed a deletion encompassing the entire dystrophin gene. Molecular cytogenetic analysis of lymphocytes from the dam and both dystrophic pups confirmed a visible deletion in the p21 region of the affected canine X chromosome. Utrophin is up-regulated in the skeletal muscle, but does not appear to ameliorate the dystrophic canine phenotype. This new canine model should further our understanding of the physiological and biochemical processes in Duchenne muscular dystrophy.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Neuromuscular disorders : NMD</span>&nbsp;<span class="year">1999</span>;<span class="volume">9</span>;<span class="number">5</span>;<span class="pagerange">289-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10407848" rel="external">10407848</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">[Molecular cloning of some components of the translation apparatus of fission yeast Schizosaccharomyces pombe and a list of its cytoplasm ic proteins genes].</h4>
<p class="authors"><span class="author">Shpakovski&#x12D; GV</span>, <span class="author">Baranova GM</span>, <span class="author">Wood V</span>, <span class="author">Gwilliam RG</span>, <span class="author">Shematorova EK</span>, <span class="author">Korol&#39;chuk OL</span> and <span class="author">Lebedenko EN</span></p>
<p class="affiliation">Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, Russia.</p><p class="abstract">Full-length cDNAs of four new genes encoding cytoplasmic ribosomal proteins L14 and L20 (large ribosomal subunit) and S1 and S27 (small ribosomal subunit) were isolated and sequenced during the analysis of the fission yeast Schizosaccharomyces pombe genome. One of the Sz. pombe genes encoding translation elongation factor EF-2 was also cloned and its precise position on chromosome I established. A unified nomenclature was proposed, and the list of all known genetic determinants encoding cytoplasmic ribosomal proteins of Sz. pombe was compiled. By now, 76 genes/cDNAs encoding different ribosomal proteins have been identified in the fission yeast genome. Among them, 35 genes are duplicated and three homologous genes are identified for each of the ribosomal proteins L2, L16, P1, and P2.</p>
<p class="publication"><span class="title">Bioorganicheskaia khimiia</span>&nbsp;<span class="year">1999</span>;<span class="volume">25</span>;<span class="number">6</span>;<span class="pagerange">450-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10505233" rel="external">10505233</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nonmethylated transposable elements and methylated genes in a chordate genome.</h4>
<p class="authors"><span class="author">Simmen MW</span>, <span class="author">Leitgeb S</span>, <span class="author">Charlton J</span>, <span class="author">Jones SJ</span>, <span class="author">Harris BR</span>, <span class="author">Clark VH</span> and <span class="author">Bird A</span></p>
<p class="affiliation">Institute of Cell and Molecular Biology, University of Edinburgh, The King&#39;s Buildings, Edinburgh EH9 3JR, UK.</p><p class="abstract">The genome of the invertebrate chordate Ciona intestinalis was found to be a stable mosaic of methylated and nonmethylated domains. Multiple copies of an apparently active long terminal repeat retrotransposon and a long interspersed element are nonmethylated and a large fraction of abundant short interspersed elements are also methylation free. Genes, by contrast, are predominantly methylated. These data are incompatible with the genome defense model, which proposes that DNA methylation in animals is primarily targeted to endogenous transposable elements. Cytosine methylation in this urochordate may be preferentially directed to genes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1999</span>;<span class="volume">283</span>;<span class="number">5405</span>;<span class="pagerange">1164-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10024242">10024242</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Caenorhabditis elegans genes egl-27 and egr-1 are similar to MTA1, a member of a chromatin regulatory complex, and are redundantly required for embryonic patterning.</h4>
<p class="authors"><span class="author">Solari F</span>, <span class="author">Bateman A</span> and <span class="author">Ahringer J</span></p>
<p class="affiliation">University of Cambridge, Department of Genetics, Downing Street, Cambridge CB2 3EH, UK.</p><p class="abstract">We show here that two functionally redundant Caenorhabditis elegans genes, egl-27 and egr-1, have a fundamental role in embryonic patterning. When both are inactivated, cells in essentially all regions of the embryo fail to be properly organised. Tissue determination and differentiation are unaffected and many zygotic patterning genes are expressed normally, including HOX genes. However, hlh-8, a target of the HOX gene mab-5, is not expressed. egl-27 and egr-1 are members of a gene family that includes MTA1, a human gene with elevated expression in metastatic carcinomas. MTA1 is a component of a protein complex with histone deacetylase and nucleosome remodelling activities. We propose that EGL-27 and EGR-1 function as part of a chromatin regulatory complex required for the function of regional patterning genes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">1999</span>;<span class="volume">126</span>;<span class="number">11</span>;<span class="pagerange">2483-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10226007" rel="external">10226007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">INFOGENE: a database of known gene structures and predicted genes and proteins in sequences of genome sequencing projects.</h4>
<p class="authors"><span class="author">Solovyev VV</span> and <span class="author">Salamov AA</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%6F%6C%6F%76%79%65%76%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#111;&#108;&#111;&#118;&#121;&#101;&#118;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">INFOGENE is a database of known and predicted gene structures with descriptions of basic functional signals and gene components. It provides a possibility to create compilations of sequences with a given gene feature as well as to accumulate and analyze predicted genes in finished and unfinished sequences from genome sequencing projects. Protein sequence similarity searches in the database of predicted proteins is offered through the BLASTP program. INFOGENE is realized under the Sequence Retrieval System that provides useful links with the other informational databases. The database is available through the WWW server of the Computational Genomics Group at http://genomic.sanger.ac.uk/db.html</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1999</span>;<span class="volume">27</span>;<span class="number">1</span>;<span class="pagerange">248-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9847192" rel="external">9847192</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC148147" rel="external">148147</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative analyses of the Dominant megacolon-SOX10 genomic interval in mouse and human.</h4>
<p class="authors"><span class="author">Southard-Smith EM</span>, <span class="author">Collins JE</span>, <span class="author">Ellison JS</span>, <span class="author">Smith KJ</span>, <span class="author">Baxevanis AD</span>, <span class="author">Touchman JW</span>, <span class="author">Green ED</span>, <span class="author">Dunham I</span> and <span class="author">Pavan WJ</span></p>
<p class="affiliation">Genetic Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, 49 Convent Dr., Bethesda, Maryland 20892-4470, USA.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">1999</span>;<span class="volume">10</span>;<span class="number">7</span>;<span class="pagerange">744-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10384052" rel="external">10384052</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution landmark framework for the sequence-ready mapping of Xq23-q26.1.</h4>
<p class="authors"><span class="author">Steingruber HE</span>, <span class="author">Dunham A</span>, <span class="author">Coffey AJ</span>, <span class="author">Clegg SM</span>, <span class="author">Howell GR</span>, <span class="author">Maslen GL</span>, <span class="author">Scott CE</span>, <span class="author">Gwilliam R</span>, <span class="author">Hunt PJ</span>, <span class="author">Sotheran EC</span>, <span class="author">Huckle EJ</span>, <span class="author">Hunt SE</span>, <span class="author">Dhami P</span>, <span class="author">Soderlund C</span>, <span class="author">Leversha MA</span>, <span class="author">Bentley DR</span> and <span class="author">Ross MT</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, UK.</p><p class="abstract">We have established a landmark framework map over 20-25 Mb of the long arm of the human X chromosome using yeast artificial chromosome (YAC) clones. The map has approximately one landmark per 45 kb of DNA and stretches from DXS7531 in proximal Xq23 to DXS895 in proximal Xq26, connecting to published framework maps on its proximal and distal sides. There are three gaps in the framework map resulting from the failure to obtain clone coverage from the YAC resources available. Estimates of the maximum sizes of these gaps have been obtained. The four YAC contigs have been positioned and oriented using somatic-cell hybrids and fluorescence in situ hybridization, and the largest is estimated to cover approximately 15 Mb of DNA. The framework map is being used to assemble a sequence-ready map in large-insert bacterial clones, as part of an international effort to complete the sequence of the X chromosome. PAC and BAC contigs currently cover 18 Mb of the region, and from these, 12 Mb of finished sequence is available.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1999</span>;<span class="volume">9</span>;<span class="number">8</span>;<span class="pagerange">751-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10447510" rel="external">10447510</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310799" rel="external">310799</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene organisation, sequence variation and isochore structure at the centromeric boundary of the human MHC.</h4>
<p class="authors"><span class="author">Stephens R</span>, <span class="author">Horton R</span>, <span class="author">Humphray S</span>, <span class="author">Rowen L</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Immunology, Department of Pathology, University of Cambridge, Tennis Court Road, Cambridge, CB2 1QP, UK.</p><p class="abstract">We have mapped and sequenced the region immediately centromeric of the human major histocompatibility complex (MHC). A cluster of 13 genes/pseudogenes was identified in a 175 kb PAC linking the TAPASIN locus with the class II region. It includes two novel human genes (BING4 and SACM2L) and a thus far unnoticed human leucocyte antigen (HLA) class II pseudogene, termed HLA-DPA3. Analysis of the G+C content revealed an isochore boundary which, together with the previously reported telomeric boundary, defines the MHC class II region as one of the first completely sequenced isochores in the human genome. Comparison of the sequence with limited sequence from other cell lines shows that the high sequence variation found within the classical class II region extends beyond the identified isochore boundary leading us to propose the concept of an &quot;extended MHC&quot;. By comparative analysis, we have precisely identified the mouse/human synteny breakpoint at the centromeric end of the extended MHC class II region between the genes HSET and PHF1.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1999</span>;<span class="volume">291</span>;<span class="number">4</span>;<span class="pagerange">789-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10452889" rel="external">10452889</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1999.3004" rel="external">10.1006/jmbi.1999.3004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the proteome of Mycobacterium tuberculosis in silico.</h4>
<p class="authors"><span class="author">Tekaia F</span>, <span class="author">Gordon SV</span>, <span class="author">Garnier T</span>, <span class="author">Brosch R</span>, <span class="author">Barrell BG</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Unite de G&eacute;n&eacute;tique Mol&eacute;culaire des Levures, Institut Pasteur, Paris, France.</p><p class="abstract">Novel bioinformatics routines have been used to provide a more detailed definition of the proteome of Mycobacterium tuberculosis H37Rv. Over half of the current proteins result from gene duplication or domain shuffling events while one-sixth show no similarity to polypeptides described in other organisms. Prominent among the genes that appear to have been duplicated on numerous occasions are those involved in fatty acid metabolism, regulation of gene expression, and the unusually glycine-rich PE and PPE proteins. Protein similarity analysis, coupled with inspection of the genetic neighbourhood, was used to explore possible functional relatedness. This uncovered four large mce operons whose proteins may mediate initial interactions between the tubercle bacillus and host cells, together with a cluster of genes that might encode components of a structure required for secretion of ESAT-6 like proteins. Close linkage of the mmpL genes, encoding large membrane proteins, with those required for fatty acid metabolism suggests involvement in lipid transport. Compared to free-living bacteria, M. tuberculosis has a significantly smaller transport protein repertoire and this may reflect its intracellular lifestyle.</p>
<p class="publication"><span class="title">Tubercle and lung disease : the official journal of the International Union against Tuberculosis and Lung Disease</span>&nbsp;<span class="year">1999</span>;<span class="volume">79</span>;<span class="number">6</span>;<span class="pagerange">329-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10694977" rel="external">10694977</a>; DOI: <a href="http://dx.doi.org/10.1054%2Ftuld.1999.0220" rel="external">10.1054/tuld.1999.0220</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Zoo-FISH analysis of dog chromosome 5: identification of conserved synteny with human and cat chromosomes.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Breen M</span>, <span class="author">Langford CF</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Centre for Preventive Medicine, Animal Health Trust, Newmarket, Suffolk, UK. <a href="mailto:%72%74%68%6F%6D%61%73%40%68%67%6D%70.%6D%72%63.%61%63.%75%6B">&#114;&#116;&#104;&#111;&#109;&#97;&#115;&#64;&#104;&#103;&#109;&#112;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Conserved segments of synteny between the human genome and chromosome 5 (CFA 5) of the domestic dog (Canis familiaris) have been identified by reciprocal chromosome painting analysis. A CFA 5 paint probe was applied to human metaphase spreads, revealing distinct hybridisation sites on human (HSA) chromosomes 1, 11, 16, and 17. Paint probes for these human chromosomes were then hybridised to dog metaphase spreads, identifying the regions of CFA 5 with which homology is shared with the corresponding human chromosome. Application of the CFA 5 paint probe to metaphase spreads of the domestic cat (Felis catus, FCA) demonstrated hybridisation to cat chromosomes C1, D1, E1, and E2. Dog PCR primers for type 1 markers known to lie in the corresponding regions of HSA 11, 16, and 17 were used to isolate dog BAC clones representing four genes. Fluorescence in situ hybridisation analysis confirmed their localisation to CFA 5 and suggested that two of the conserved segments lie in opposing orientations on CFA 5, compared to the human chromosome concerned. A third segment appears to lie in the same orientation on both human and dog chromosomes. No suitable gene markers were available for analysis of the fourth segment. The significance of these findings is discussed with reference to current and future dog genome mapping efforts.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">87</span>;<span class="number">1-2</span>;<span class="pagerange">4-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10640803" rel="external">10640803</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of polymorphism in the RING3 gene by high-throughput fluorescent SSCP analysis.</h4>
<p class="authors"><span class="author">Thorpe KL</span>, <span class="author">Schafer AJ</span>, <span class="author">G&eacute;nin E</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">We describe the use of a high-throughput, fluorescent, polymorphism-detection system, based on single-strand conformation polymorphism to screen for polymorphism in the RING3 gene. This is the first extensive mutation screen of this major histocompatibility complex-linked gene, and the entire coding region and intron-exon junctions were examined by multiplexing over 3000 polymerase chain reaction products. These techniques should be applicable for analysis of variation in other human genes. Investigation of DNA from acute lymphoblastic leukemia (ALL) and chronic myeloid leukemia (CML) patients, as well as healthy individuals revealed low levels of polymorphism across the RING3 gene. Comparison of the distribution of genotypes at each polymorphic site between patients and healthy individuals revealed a single site which significantly deviates from Hardy-Weinberg proportions.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">49</span>;<span class="number">4</span>;<span class="pagerange">256-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10079288" rel="external">10079288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mapping and complex expression pattern of the human NPAP60L nucleoporin gene.</h4>
<p class="authors"><span class="author">Trichet V</span>, <span class="author">Shkolny D</span>, <span class="author">Dunham I</span>, <span class="author">Beare D</span> and <span class="author">McDermid HE</span></p>
<p class="affiliation">Department of Biological Sciences, University of Alberta, Edmonton, Canada.</p><p class="abstract">From a clone mapping to human chromosome 22q13.3, we have identified NPAP60L, the human homolog of the rat nuclear pore-associated protein gene, Npap60. The expression pattern of the human copy is much more complex that that of the rat, although conservation of the potential specific function of NPAP60L in male germ cells can be seen for one of the five transcripts. The exon-intron organization of the NPAP60L gene shows the presence of at least three alternate 3&#39; ends, and Northern analysis indicates the possible presence of alternate 5&#39; ends. Somatic cell hybrid mapping revealed additional related copies of NPAP60L on human chromosomes 5, 6, and 14, although it is not known if these are functional genes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">85</span>;<span class="number">3-4</span>;<span class="pagerange">221-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10449902" rel="external">10449902</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the Lmo4 gene encoding a LIM-only protein: genomic organization and comparative chromosomal mapping.</h4>
<p class="authors"><span class="author">Tse E</span>, <span class="author">Grutz G</span>, <span class="author">Garner AA</span>, <span class="author">Ramsey Y</span>, <span class="author">Carter NP</span>, <span class="author">Copeland N</span>, <span class="author">Gilbert DJ</span>, <span class="author">Jenkins NA</span>, <span class="author">Agulnick A</span>, <span class="author">Forster A</span> and <span class="author">Rabbitts TH</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Division of Protein and Nucleic Acid Chemistry, Hills Road, Cambridge CB2 2 QH, UK.</p><p class="abstract">LIM-only (LMO) proteins are transcription regulators that function by mediating protein-protein interaction and include the T cell oncogenes encoding LMO1 and LMO2. The oncogenic functions of LMO1 and LMO2 are thought to be mediated by interaction with LDB1 since they form a multimeric protein complex(es). A new member of the Lmo family, Lmo4, has also recently been identified via its interaction with Ldb1. Sequence analysis of the mouse Lmo4 gene shows that it spans about 18 kb and consists of at least six exons, including two alternatively spliced 5&#39; exons. Unlike Lmo1, the two 5&#39; exons of Lmo4 do not encode protein. Comparison of the Lmo4 gene structure with the other LMO family members shows the exon structure of Lmo4 differs in the position of exon junctions encoding the second LIM domain and in a novel exon-intron junction at the penultimate codon of the gene. Lmo4 is thus the least conserved known member of the LIM-only family in both nucleotide sequence and exon structure. Physical mapping of the Lmo4/LMO4 genes has shown mouse Lmo4 is located on Chromosome (Chr) 3 and human LMO4 on Chr 1p22.3. This chromosome location is of interest as it occurs in a region that is deleted in a number of human cancers, indicating a possible role of LMO4 in tumorigenesis, like its relatives LMO1 and LMO2.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">1999</span>;<span class="volume">10</span>;<span class="number">11</span>;<span class="pagerange">1089-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10556429" rel="external">10556429</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and characterization of the human homologue (RAI2) of a mouse retinoic acid-induced gene in Xp22.</h4>
<p class="authors"><span class="author">Walpole SM</span>, <span class="author">Hiriyana KT</span>, <span class="author">Nicolaou A</span>, <span class="author">Bingham EL</span>, <span class="author">Durham J</span>, <span class="author">Vaudin M</span>, <span class="author">Ross MT</span>, <span class="author">Yates JR</span>, <span class="author">Sieving PA</span> and <span class="author">Trump D</span></p>
<p class="affiliation">Department of Medical Genetics, University of Cambridge, Cambridge Institute for Medical Research, Addenbrooke&#39;s Hospital, United Kingdom.</p><p class="abstract">We have identified a novel human gene during studies of a 1.3-Mb region of Xp22 between DXS418 and DXS999. A PAC contig spanning the region was constructed, sequenced, and analyzed by gene and exon prediction programs and by homology searches. Further investigation of predicted exons from PAC clone 389A20 led to the identification of a single-exon gene, designated RAI2 (retinoic acid-induced 2). RAI2 mapped 28 kb centromeric to marker DXS7996, between DXS7996 and DXS7997, and was transcribed from centromere to telomere. Northern blot analysis and reverse transcription-polymerase chain reaction analysis revealed expression of a 2.5-kb transcript in four fetal tissues (brain, lung, kidney, and heart) and eight adult tissues (heart, brain, placenta, lung, skeletal muscle, kidney, pancreas, and retina) but not in fetal or adult liver. The 530-amino-acid protein (57 kDa predicted mass) displays 94% homology with a mouse retinoic acid-induced gene product and contains a novel proline-rich (39%) domain of 68 amino acids. Retinoic acid is involved in vertebrate anteroposterior axis formation and cellular differentiation and has been shown to modulate gene expression controlling early embryonal development, suggesting a developmental role for RAI2. RAI2 remains a candidate gene for diseases mapping to the Xp22 region.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">R01-EY10259</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1999</span>;<span class="volume">55</span>;<span class="number">3</span>;<span class="pagerange">275-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10049581" rel="external">10049581</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1998.5667" rel="external">10.1006/geno.1998.5667</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Report of the fifth international workshop on human chromosome 1 mapping 1999.</h4>
<p class="authors"><span class="author">White PS</span>, <span class="author">Forus A</span>, <span class="author">Matise TC</span>, <span class="author">Schutte BC</span>, <span class="author">Spieker N</span>, <span class="author">Stanier P</span>, <span class="author">Vance JM</span> and <span class="author">Gregory SG</span></p>
<p class="affiliation">Division of Oncology, The Children&#39;s Hospital of Philadelphia, Philadelphia, PA 19104-4318, USA. <a href="mailto:%77%68%69%74%65%40%67%65%6E%6F%6D%65.%63%68%6F%70.%65%64%75">&#119;&#104;&#105;&#116;&#101;&#64;&#103;&#101;&#110;&#111;&#109;&#101;&#46;&#99;&#104;&#111;&#112;&#46;&#101;&#100;&#117;</a></p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">1999</span>;<span class="volume">87</span>;<span class="number">3-4</span>;<span class="pagerange">143-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10702659" rel="external">10702659</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">How the worm was won. The C. elegans genome sequencing project.</h4>
<p class="authors"><span class="author">Wilson RK</span></p>
<p class="affiliation">Washington University Genome Sequencing Center, St Louis, MO 63108, USA. <a href="mailto:%72%77%69%6C%73%6F%6E%40%77%61%74%73%6F%6E.%77%75%73%74%6C.%65%64%75">&#114;&#119;&#105;&#108;&#115;&#111;&#110;&#64;&#119;&#97;&#116;&#115;&#111;&#110;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The genome sequence of the free-living nematode Caenorhabditis elegans is nearly complete, with resolution of the final difficult regions expected over the next few months. This will represent the first genome of a multicellular organism to be sequenced to completion. The genome is approximately 97 Mb in total, and encodes more than 19,099 proteins, considerably more than expected before sequencing began. The sequencing project--a collaboration between the Genome Sequencing Center in St Louis and the Sanger Centre in Hinxton--has lasted eight years, with the majority of the sequence generated in the past four years. Analysis of the genome sequence is just beginning and represents an effort that will undoubtedly last more than another decade. However, some interesting findings are already apparent, indicating that the scope of the project, the approach taken, and the usefulness of having the genetic blueprint for this small organism have been well worth the effort.</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">1999</span>;<span class="volume">15</span>;<span class="number">2</span>;<span class="pagerange">51-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10098407" rel="external">10098407</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA sequencing and analysis of a 67.4 kb region from the right arm of Schizosaccharomyces pombe chromosome II reveals 28 open reading frames including the genes his5, pol5, ppa2, rip1, rpb8 and skb1.</h4>
<p class="authors"><span class="author">Xiang Z</span>, <span class="author">Lyne MH</span>, <span class="author">Wood V</span>, <span class="author">Rajandream MA</span>, <span class="author">Barrell BG</span> and <span class="author">Aves SJ</span></p>
<p class="affiliation">School of Biological Sciences, University of Exeter, Washington Singer Laboratories, Perry Road, Exeter EX4 4QG, U.K.</p><p class="abstract">67 393 bp of contiguous DNA located between markers cdc18 and cdc14 on the right arm of fission yeast chromosome II has been sequenced as part of the European Union Schizosaccharomyces pombe genome sequencing project. The complete sequence, contained in cosmid clones c15C4 and c21H7, has been determined on both strands. Sequence analysis shows that it contains 28 open reading frames capable of coding for proteins, 16 split by one or more introns, but no tRNA, rRNA or transposon sequences. The gene density is one per 2. 4 kb. Six genes have been previously described (his5, pol5, ppa2, rip1, rpb8 and skb1) and 22 are novel. Of the novel genes, 14 have significant similarity with proteins of known function, three have similarities with proteins of unknown function and five show no extensive similarities with known proteins. Sequence similarities suggest that three of the novel genes encode ATP-dependent RNA helicases, two encode transcription factor components and others encode a G-protein, a dehydrogenase, a Rab escort protein, an Abc1-like protein, a lipase, an ATP-binding transport protein, an amino acid permease, an acid phosphatase and a mannosyltransferase.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Yeast (Chichester, England)</span>&nbsp;<span class="year">1999</span>;<span class="volume">15</span>;<span class="number">10A</span>;<span class="pagerange">893-901</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10407269" rel="external">10407269</a>; DOI: <a href="http://dx.doi.org/10.1002%2F(SICI)1097-0061(199907)15%3A10A%3C893%3A%3AAID-YEA430%3E3.0.CO%3B2-S" rel="external">10.1002/(SICI)1097-0061(199907)15:10A&lt;893::AID-YEA430&gt;3.0.CO;2-S</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete sequence and gene map of a human major histocompatibility complex. The MHC sequencing consortium.</h4>
<p class="authors">No authors listed</p><p class="abstract">Here we report the first complete sequence and gene map of a human major histocompatibility complex (MHC), a region on chromosome 6 which is essential to the immune system. When it was discovered over 50 years ago the region was thought to specify histocompatibility genes, but their nature has been resolved only in the last two decades. Although many of the 224 identified gene loci (128 predicted to be expressed) are still of unknown function, we estimate that about 40% of the expressed genes have immune system function. Over 50% of the MHC has been sequenced twice, in different haplotypes, giving insight into the extraordinary polymorphism and evolution of this region. Several genes, particularly of the MHC class II and III regions, can be traced by sequence similarity and synteny to over 700 million years ago, clearly predating the emergence of the adaptive immune system some 400 million years ago. The sequence is expected to be invaluable for the identification of many common disease loci. In the past, the search for these loci has been hampered by the complexity of high gene density and linkage disequilibrium.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1999</span>;<span class="volume">401</span>;<span class="number">6756</span>;<span class="pagerange">921-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10553908">10553908</a>; DOI: <a href="http://dx.doi.org/10.1038%2F44853">10.1038/44853</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Special issue containing a selection of papers presented at the 1st International Conference on Bioinformatics of Genome Regulation and Structure. BGRS &#39;98. Novosibirsk, Altai Mountains, Russia. 24-31 August 1998.</h4>
<p class="authors">No authors listed</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">1999</span>;<span class="volume">15</span>;<span class="number">7-8</span>;<span class="pagerange">527-714</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10627151" rel="external">10627151</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/6umpwhsb"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/6umpwhsb" src="/qr/6umpwhsb.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
